<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>From Hub Proteins Hub Modules: The Relationship <br /> Between Essentiality Centrality Yeast <br /> Interactome Different Scales Organization <br /> Jimin Song, Mona Singh* <br /> Department Computer Science Lewis-Sigler Institute Integrative Genomics, Princeton University, Princeton, New Jersey, United States America <br />  <br />  <br />  <br />      Abstract <br />      Numerous studies suggested hub proteins S. cerevisiae physical interaction network likely be <br />      essential proteins. The proposed reasons underlying observed relationship topology and <br />      functioning subject controversy, recent work suggesting arises participation of <br />      hub proteins essential complexes processes. However, essential modules distinct network <br />      characteristics, essential proteins differ topological properties non-essential proteins? <br />      We aimed advance understanding protein essentiality analyzing proteins, complexes processes their <br />      broader functional context considering physical interactions complexes biological <br />      processes. In agreement view essentiality modular property, number intracomplex or <br />      intraprocess interactions protein better <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-5'  ></span>indicator</span> essentiality overall number interactions. <br />      Moreover, essential complex, essential proteins average interactions, especially <br />      intracomplex interactions, non-essential proteins. Finally, built module-level interaction network found <br />      essential complexes processes tend higher interaction degrees network non-essential <br />      complexes processes; is, exhibit larger functional cross-talk non-essential counterparts. <br />  <br />   Citation: Song J, Singh M (2013) From Hub Proteins Hub Modules: The Relationship Between Essentiality Centrality Yeast Interactome Different <br />   Scales Organization. PLoS Comput Biol 9(2): e1002910. doi:10.1371/journal.pcbi.1002910 <br />   Editor: Frederick P. Roth, Harvard Medical School, United States America <br />   Received August 27, 2012; Accepted December 21, 2012; Published February 21, 2013 <br />   Copyright: &#223; 2013 Song, Singh. <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported NSF ABI-0850063, NIH GM076275, NIH Center Excellence grant P50 GM071508. JS partially <br />   supported Samsung Scholarship Samsung Foundation Culture. The funders role study design, <span id='am-29' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> collection analysis, decision to <br />   publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: msingh@cs.princeton.edu <br />  <br />  <br />  <br /> Introduction                                                                         proteins essential; is, deletion proteins results <br />                                                                                      cell death, optimal growth conditions [11,12]. Early <br />    High-throughput experimental approaches determining                           computational analysis yeast S. cerevisiae protein-protein <br /> protein interactions resulted large-scale cellular networks                  physical interaction network revealed scale-free topology, where <br /> for numerous organisms. Graph-theoretic analyses                            &#8216;&#8216;hub&#8217;&#8217; proteins interactions, showed <br /> networks great aid advancing understanding                        hub proteins likely essential proteins <br /> of cellular functioning organization (review, [1]). One                   [13]. Numerous subsequent studies confirmed centrality- <br /> most fundamental discoveries strong relationship                  lethality relationship, yeast [14&#8211;19] other <br /> between topological characteristics cellular networks                     organisms [20]. On hand, relationship been <br /> their underlying functioning. For example, cellular networks                         observed weak networks consisting interactions <br /> consist tightly clustered groups interacting proteins,                     determined high-throughput yeast two-<span id='am-34' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-35' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> experiments <br /> these proteins work protein complexes biological                      stronger types networks [16,18,19], has <br /> processes achieve specific biological functions [2&#8211;7]. An                         proposed that, yeast two-hybrid networks, observed <br /> orthogonal decomposition reveals recurring                        relationship study <span id='am-6' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>bias</span> favoring determination of <br /> over-represented topological functional patterns larger                   interactions essential proteins [21]. Nevertheless, positive <br /> cellular networks, network motifs [8,9] network                        correlation protein interaction degree essentiality is <br /> schemas [10] associated dynamic regulatory properties                    generally accepted, numerous reasons proposed the <br /> and shared mechanisms functioning. Here, revisit                       literature explain relationship. <br /> the basic structure-to-function relationship                          Initial work suggested high-degree proteins essential <br /> proposed cellular networks&#8212;that number                            role interaction network connectivity [13]; however, <br /> interactions protein overall functional importance.                    unlikely case subsequently shown that <br />    The importance gene cell organism                          non-essential hubs just important essential hubs for <br /> quantitatively measured considering phenotypic effects                     maintaining connectivity, essentiality better correlated <br /> gene deletion disruption. Experimental studies baker&#8217;s                     local, global, measures connectivity protein- <br /> yeast S. cerevisiae demonstrated approximately 19%                  protein interaction networks [17,18]. It alternatively proposed <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                1                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                         Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />   Author Summary                                                               cerevisiae protein-protein interaction networks considered <br />                                                                                functional modules derived protein complexes as <br />   Network analyses large-scale interactomes                     Gene Ontology (GO) biological process annotations [25] at <br />   great aid advancing understanding cellular                         different levels resolution. Further, address issue study <br />   functioning organization. Here, examine                    bias, performed analysis additional networks which <br />   basic intensely-studied structure-to-function                       removed interactions determined small-scale experiments. <br />   relationships observed cellular networks:                        We began re-examining relationship protein <br />   number interactions protein tendency                    essentiality network modularity. We hypothesized if <br />   essential. We develop new computational                              essentiality modular property, proposed previously <br />   framework systematically analyze essential proteins                       [18], protein&#8217;s intramodular physical interaction degree <br />   cellular context explicitly incorporating <br />                                                                                better predictor protein&#8217;s essentiality its <br />   functional information. We apply framework the <br />                                                                                intermodular physical interaction degree. To test this, utilized <br />   yeast interactome demonstrate previously <br />   observed positive relationship interaction degree                    biological process functional annotations proteins classified <br />   essentiality largely intramodular interactions.                physical interactions intraprocess interactions processes <br />   Further, essentiality appears modular property                    interprocess interactions processes. We that <br />   protein complexes broadly biological                         essential proteins tend interactions proteins <br />   processes. Within essential complex, essential                        functional modules intraprocess <br />   proteins tend interactions, especially intra-                   interaction degree correlated essentiality overall <br />   complex interactions, non-essential proteins.                       degree. Further, relationship overall <br />   Finally, computationally inferred module-level inter-                   degree essentiality significantly weakened controlling <br />   action network, essential complexes processes tend                    intramodular degree, affected controlling for <br />   higher interaction degrees non-essential                     intermodular degree. Thus, direct simple <br />   counterparts. In summary, relationship                             manner that, essential proteins, essentiality likely <br />   connectivity essentiality                        consequence participation essential modules <br />   protein level, modules module                     consisting functionally similar proteins. <br />   level, complexes processes essential                           To ascertain modularity essential <br />   tending interact functional groups.                             proteins roles essential protein complexes <br />                                                                                generally essential biological processes, we <br /> that essentiality property interactions [22]. That is,       repeated analysis exclusively focusing proteins <br /> essential protein interactions, organism               protein complexes focusing proteins that <br /> survive, randomly distributed network;                known protein complexes. We most <br /> hubs tend essential likely participate          essential proteins intraprocess interactions fact <br /> in essential interactions. However, model implies                participate essential protein complexes essential biological <br /> probabilities non-interacting proteins essential              processes include protein complexes; is, the <br /> independent other, case [18]. Instead,             modularity protein essentiality appears consequence of <br /> Zotenko et al. [18] argued correlation degree             protein complexes, broadly biological processes. <br /> essentiality participation essential proteins                  Next, examined complexes contain essential proteins, <br /> essential functional modules consisting groups densely                   compared essential non-essential proteins. We <br /> clustered functionally related proteins. They showed               reasoned relationship essentiality and <br /> that essentiality hubs computationally            interaction degree proteins complexes entirely <br /> extracted modules weakly correlated degree [18].                 consequence complexes essential, then <br /> Indeed, previously essential proteins tended            essential non-essential proteins complex <br /> to densely connected [15] concentrated                 differ respect degree. On contrary, found <br /> complexes [23,24], suggesting essentiality modular                   essential proteins tend interactions, particularly <br /> property property individual proteins. Building               intracomplex interactions, non-essential counterparts <br /> upon this, argued essential complexes tend              protein complexes. That is, essentiality appears be <br /> large, proteins larger number                   modular property, degree protein associated with <br /> interactions, explains hubs tend essential             essentiality essential complexes; hypothesize these <br /> [19].                                                                          essential proteins play important role maintaining <br />    While substantial evidence essentiality modular          structure and/or function complexes. <br /> property protein-protein interaction networks, clear                 Finally, analyzed modules containing essential proteins <br /> that complexes processes consist entirely essential           context functional modules. We inferred <br /> non-essential proteins. Do essential proteins essential              significant &#8216;&#8216;cross-talks&#8217;&#8217; protein complexes biological <br /> complex process differ non-essential ones? Further,            processes used build module-level networks, which <br /> all complexes processes contain essential proteins. Do                complexes processes linked enriched <br /> essential modules distinctive roles cellular networks? In         number physical interactions them. Using these <br /> paper, aimed discover whether, modules,                     module-level networks, uncovered functional modules <br /> essential non-essential proteins differ interaction               essential proteins tend average higher degree; that <br /> properties, global scale, essential non-             is, degree module-level network positively correlated with <br /> essential modules differ network-level properties. To                 module essentiality. <br /> accomplish this, developed computational framework                       Overall, considering proteins functional context of <br /> incorporates information functional modules                   yeast interactome, evidence relationship <br /> context network analysis techniques. To uncover general                 essentiality network topology different levels of <br /> robust principles, performed analysis types S.              cellular organization: protein level, protein <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          2                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> complexes, globally module level,                     intraprocess interactions, interprocess interactions, total anno- <br /> complexes processes essential tending interact             tated interactions. We considered intraprocess, <br /> more functional groups.                                                         interprocess total annotated interaction degrees turn, and <br />                                                                                 ordered proteins high low degrees respect it. As <br /> Results                                                                         varied threshold number proteins considered, we <br />                                                                                 computed fraction essential proteins &#8216;&#8216;high degree&#8217;&#8217; or <br />    We analyzed 5640 proteins tested essentiality [12]          &#8216;&#8216;hub&#8217;&#8217; set. Over large range thresholds, high degree <br /> the context large-scale S. cerevisiae protein physical               proteins, ranked intraprocess degree, higher fraction of <br /> interaction datasets; networks captures different                 essential proteins high degree proteins ranked either <br /> features biological interactions. The network Direct              total annotated degree interprocess degree (Figures 1 (a), S1 (a) <br /> interaction network, interaction proteins                  S2 (a)). For Pull-down Full networks, fraction of <br /> corresponds direct physical contact; network includes                 essential proteins tends decrease threshold intraprocess, <br /> interactions determined yeast two-hybrid method                    interprocess, total degree lowered. In Direct network <br /> other types approaches (see Materials Methods). Next,                 (Figure 1 (a)), trend true intraprocess interaction <br /> considered Pull-down network, interaction                degree notably true total degree; consistent with <br /> proteins corresponds members multi-                  previous work showing relationship essentiality <br /> protein complex. Third, considered Full network consisting               overall interaction degree weak networks consisting of <br /> of physical interactions BioGRID [26]; case,                 interactions determined yeast two-hybrid [16,18,21]. <br /> interactions represent direct indirect interactions. In              To quantify correlation essentiality and <br /> the main body paper, primarily report results              degree, used Spearman&#8217;s rho rank correlation coefficient <br /> Direct interaction network, contains 4031 proteins (898                (SRCC) [29] (Figures 1 (b), S1 (b) S2 (b)), the <br /> which essential) 15,073 interactions. All analysis                  SRCC highest intraprocess degree (0.25 Direct, 0.35 for <br /> described performed Pull-down Full                     networks), lower interprocess degree (0 <br /> networks (see Table S1) reported in Supplementary               Direct, 0.22 Pull-down 0.21 Full). We note since <br /> Material. We considered additional networks interac-                 protein essentiality binary value tied <br /> tions determined small-scale experiments removed;                  values, possible SRCC achieve value 1. For <br /> analysis outlined section high-throughput networks                 example, SRCC essentiality annotated degree <br /> below, detailed figures given Supplementary Material.               Direct network reach maximum value of <br />                                                                                 0.7680 (i.e., case essentiality values proteins are <br /> Categorizing interactions intramodular                                    swapped essential proteins higher degrees all <br /> intermodular                                                                    non-essential proteins). Next, disentangle contributions of <br />    For given interaction network, labeled protein interactions             intraprocess interprocess degree observed correlations, <br /> as &#8216;&#8216;intramodular,&#8217;&#8217; &#8216;&#8216;intermodular&#8217;&#8217; using               computed partial correlations essentiality all <br /> sources functional data. In particular, utilized yeast protein            annotated interactions, controlling intraprocess and <br /> complex data compiled [27] Gene Ontology (GO) Biological                 interprocess degree. For networks when <br /> Process (BP) annotations [25]. Thus, intramodular interactions              controlling intraprocess degree, SRCC total degree <br /> arise intracomplex intraprocess interactions,                essentiality notably diminished, controlling for <br /> intermodular interactions arise intercomplex interpro-             interprocess degree, SRCC remained high (Figures 1 (b), S1 (b) <br /> cess interactions; separately consider types                    S2 (b)), increased Direct network. <br /> intramodular intermodular interactions. For protein complex                    As way looking difference intrapro- <br /> data, &#8216;&#8216;intracomplex&#8217;&#8217; interactions pairs proteins           cess interprocess interaction degree, compared degree <br /> that participate shared complex &#8216;&#8216;intercomplex&#8217;&#8217; interac-              distributions essential proteins non-essential proteins <br /> tions pairs proteins         (Figures 1, S1 S2 (c)&#8211;(e)) using Wilcoxon rank sum test. <br /> complex complex.                                For comparing degree distributions, included proteins with <br />    It complicated characterize interactions intramodular          annotated interaction; proteins zero <br /> or intermodular using GO BP terms, terms hierarchically              intraprocess interprocess interactions. Since number <br /> related annotate different numbers proteins,              proteins considered comparing total, intraprocess, or <br /> general terms. To informative specific terms,                   interprocess degree (Figures 1, S1 S2 (c)&#8211;(e)), p-values <br /> considered GO BP terms annotate 50 proteins                 given comparable. The difference number of <br /> yeast proteome. An interaction unannotated unless proteins              interactions essential non-essential proteins much <br /> are annotated specific GO BP terms. An                    significant intraprocess interactions consid- <br /> interaction &#8216;&#8216;intraprocess&#8217;&#8217; proteins sharing       ered (Figures 1 (d), S1 (d) S2 (d)), compared case <br /> of specific BP terms. If proteins interaction             annotated interactions considered (Figures 1 (c), S1 (c) <br /> annotated specific GO BP terms share them,               S2 (c)) interprocess interactions considered <br /> the interaction &#8216;&#8216;interprocess.&#8217;&#8217; We note physical                (Figures 1 (e), S1 (e) S2 (e)). <br /> interactions largely thought &#8216;&#8216;within process,&#8217;&#8217; especially           As alternative categorizing annotated interactions as <br /> compared types interactions [28], significant fraction         interprocess intraprocess, considered case <br /> physical interactions interprocess (Table S2); true         interactions weighted according semantic <br /> the threshold choosing specific terms increased.                         similarity [30] functional terms annotating two <br />                                                                                 proteins. This weight range 0 1 proteins <br /> Intraprocess interactions main factor relationship                 sharing highly specific functional terms getting higher scores (see <br /> between protein essentiality interaction degree                             Materials Methods details). Thus, semantic <br />  As step relating protein essentiality network               similarity interacting proteins continuous <br /> modularity, protein, computed number                         measure &#8216;&#8216;intramodularity&#8217;&#8217; interaction. Then, the <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           3                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                         Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 1. The intraprocess interaction degree correlated protein essentiality overall interaction degree for <br /> proteins Direct network, interactions categorized specific BP terms, annotates 50 <br /> proteins. (a) The fraction essential proteins hub proteins proteins considered hub proteins; proteins added groups 50 <br /> in non-increasing <span id='am-21' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-22' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> interaction degree. This fraction highest intraprocess degree (green), followed annotated degree (blue) <br /> and interprocess degree (red). (b) The correlations measured SRCCs essentiality intraprocess, annotated or <br /> interprocess degree. The SRCC highest essentiality intraprocess degree. The partial correlation computed all <br /> annotated degree essentiality controlling intraprocess interprocess degree. Starred p-values indicate values w0:05. <br /> (c)&#8211;(e) The degree distribution non-essential proteins compared essential proteins (c) annotated, (d) intraprocess, (e) <br /> interprocess degree, respectively. In box plot, horizontal bar box corresponds median distribution; ends the <br /> box indicate quartiles; small circles outliers 2&#8211;98th percentile range. The significance difference <br /> between degree distributions measured Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g001 <br />  <br />  <br /> semantic similarity degree protein defined sum              contribution intracomplex interactions. In particular, previously <br /> semantic similarity interactions. Across Direct, Pull-down         observed essential proteins tend clustered <br /> Full networks, stronger correlation           essential protein complexes [18,24]. Thus, we <br /> essentiality degree interactions weighted               hypothesized having intracomplex physical interactions a <br /> semantic similarity just counted (Figure S3). In           protein important predicting essentiality having <br /> other words, proteins having interactions similar               types physical interactions. That is, defined <br /> functional context likely essential proteins              them, functional modules comprised protein <br /> having interactions. Altogether, range computational                complexes biological processes corresponding GO BP terms. <br /> analyses shows large portion observed relationship              In previous section, utilized modules derived BP <br /> between essentiality interaction degree explained             terms. We focus modules derived protein complexes, <br /> considering just intraprocess interactions.                                   compiled [27]. We begin observing complexes a <br />                                                                               enriched essential proteins. In particular, whereas <br /> The correlation intramodular degree protein                       18.60% (or 1049=5640) proteins essential yeast <br /> essentiality largely complexes, processes                       genome, 37.54% (or 598=1593) essential considering <br />    Having shown strong correlation intraprocess                   proteins involved set complexes considering. In <br /> interaction degree essentiality, sought characterize            fact, 57.01% (or 598=1049) essential proteins involved in <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         4                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> protein complexes, 28.24% (or 1593=5640)                   dataset, clear positive correlation complex size <br /> proteins set complexes. Thus, conclusions              fraction essential proteins complex (SRCC: <br /> arising analysis protein complexes based                 0.24, p-value: 2e-6). Moreover, strong correlation <br /> interaction properties significant fraction essential proteins.        protein essentiality size largest complex to <br />    For network, derived subnetwork nodes                       belongs, SRCCs 0.25, 0.24 0.24 Direct, <br /> represent proteins involved protein complex edges                   Pull-down Full networks, respectively (Figures 2 (b), S4 (b) and <br /> represent interactions original interactions            S5 (b) ). We found, however, relationship strong as <br /> proteins. In Direct network, 35.66% (or 1364=3825)                      essentiality intracomplex degree networks <br /> interactions intracomplex (Table S2). Repeating analysis               (black vs. green curve Figures 2 (a), S4 (a) S5 (a)). <br /> we performed intraprocess vs. interprocess interactions,                   We computed partial correlations essentiality and <br /> found intracomplex physical interactions correlated              annotated interactions, controlling intracomplex <br /> with protein essentiality annotated physical interactions             degree, intercomplex degree, complex size. We that <br /> (Figures 2 (a), S4 (a) S5 (a)).                                            controlling intracomplex degree, SRCC between <br />    It previously observed strong correlation          total degree essentiality notably diminished (from 0.17, 0.32 <br /> between complex size essentiality [19], argued                    0.32 20.01, 0.13 0.14 Direct, Pull-down Full <br /> essential complexes tend large, proteins                 networks, respectively), controlling intercomplex <br /> tend interactions, driving force             degree complex size, SRCC greatly diminished <br /> relationship essentiality interaction degree. In               (Figures 2 (b), S4 (b) S5 (b)). Further, difference degree <br />  <br />  <br />  <br />  <br /> Figure 2. The intracomplex interaction degree correlated protein essentiality overall interaction degree for <br /> proteins Direct network, interactions categorized using protein complexes. (a) The fraction essential proteins among <br /> hub proteins proteins considered hub proteins; proteins added groups 50 non-increasing order interaction degree or <br /> the size largest complex protein belongs. The relationship protein essentiality interaction degree shown for <br /> intracomplex interactions (green), annotated interactions (blue) intercomplex interactions (red). The relationship protein essentiality <br /> and complex size shown (black). (b) The correlations measured SRCCs essentiality intracomplex degree, annotated <br /> degree, intercomplex degree, complex size. The SRCC highest essentiality intracomplex degree. The partial correlation also <br /> computed annotated degree essentiality controlling intracomplex degree, intercomplex degree, size the <br /> largest complex protein belongs. Starred p-values indicate values w0:05. (c)&#8211;(f) The degree distribution non-essential <br /> proteins compared essential proteins complexes (c) annotated degree, (d) intracomplex degree, (e) intercomplex degree, <br /> and (f) complex size. In box plot, horizontal bar box corresponds median distribution; ends box indicate <br /> the quartiles; small circles outliers 2&#8211;98th percentile range. The significance difference two <br /> degree distributions measured Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g002 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          5                         February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 3. In networks ((a) Direct, (b) Pull-down, (c) Full network), set biological processes filtered remove the <br /> effects complexes, intraprocess interaction degree correlated protein essentiality overall <br /> interaction degree. The fraction essential proteins hub proteins function increasing number proteins considered hub <br /> proteins; adding proteins groups 50 non-increasing order interaction degree. The relationship protein <br /> essentiality interaction degree shown intraprocess interactions (green), annotated interactions (blue) interprocess interactions (red). <br /> Correlations measured SRCCs essentiality type interaction degree given. <br /> doi:10.1371/journal.pcbi.1002910.g003 <br />  <br /> distribution essential non-essential proteins                       degree (Figure 3 (a),(b),(c)). The correlations especially weak in <br /> (Figuress 2, S4 S5 (c)&#8211;(f)) significant considering            Direct network. Moreover, Pull-down Full networks, <br /> intracomplex degree significant considering                      correlation essentiality interprocess filtered <br /> intercomplex degree. We note correlation                interaction degree somewhat higher between <br /> a protein&#8217;s intracomplex degree size complex                  essentiality intraprocess filtered degree. <br /> which belongs (SRCC: 0.3790, 0.7319 0.7809 Direct, <br /> Pull-down Full networks, respectively); stronger                   Essential proteins central essential <br /> correlations Pull-down Full networks compared             protein complexes <br /> Direct network expected networks include                     Having shown global analysis proteins complexes <br /> many indirect (i.e., co-complex) interactions.                                  essential proteins tend intracomplex interactions <br />     Thus far, stronger correlation                      non-essential proteins, considered per-complex <br /> essentiality intramodular degree essentiality                  analysis. We hypothesized that, essential protein complex, <br /> and annotated degree focus biological                     essential proteins central higher <br /> process protein complex derived modules. Instead using                    intracomplex degree non-essential proteins. We tested this <br /> biological process protein complex annotations categorize                 hypothesis subset protein complexes member <br /> interactions intramodular intermodular,                    proteins intracomplex interactions. In particular, included <br /> considered modules derived network clustering approaches.                  complex test essential proteins at <br /> In particular, applied state-of-the-art SPICi network                    non-essential proteins, intracomplex <br /> clustering algorithm [31], categorized interactions                  interactions. Table 1 shows large fraction complexes, <br /> clusters intramodular interactions clusters                   essential proteins tend higher average intracomplex <br /> intermodular. We note clusters uncovered purely                   degree non-essential proteins. In particular, Direct <br /> topological manner correspond protein                         network, 76% complexes, essential proteins have <br /> complexes functional modules. On Direct, Pull-down                   higher      average     intracomplex       degree     (empirical    p- <br /> Full networks, essentiality correlated intramodular                value~7|10{4 ). In Pull-down Full network, the <br /> interaction degree, defined network clustering,             fraction complexes higher average degree essential <br /> either intermodular total degree (Figure S7).                                proteins lower Direct network (58.8% 61.5%, <br />     What happens consider intraprocess interactions                  respectively); expected networks include <br /> excluding intracomplex? That is, biological                 indirect intracomplex interactions. In fact, Pull-down the <br /> processes consist single protein complex                    Full networks, seven &#8216;&#8216;clique&#8217;&#8217; complexes every <br /> protein complexes; cases observed intrapro-                protein intracomplex interactions proteins <br /> cess interactions specifically intracomplex interactions               complex, complexes the <br /> within complexes process. To focus                     Direct network. Without clique complexes, percent of <br /> interactions intracomplex, filtered biological                  complexes higher average intracomplex degree essential <br /> processes remove interactions (see Materials Methods               proteins <span id='am-24' about='obo:NCBITaxon_376232' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_192380 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Goes" datatype="xsd:string"></span>goes</span> 68.2% 71.1% Pull-down Full <br /> for details). Among proteins annotated               networks, respectively. <br /> filtered biological process, 16.52% (or 424=2567) proteins                     By considering complex individually, analysis better <br /> essential, slightly considering                handles proteins involved multiple complexes. Although we <br /> proteins genome. In subnetwork set filtered                 removed highly overlapping complexes (see Materials and <br /> biological processes interaction networks,             Methods), 14% (or 223=1593) proteins belong more <br /> is weaker correlation interaction degree essentiality             complexes. Moreover, proteins tend essential; among <br /> as compared correlation complexes, intraprocess              proteins complex, 53.81% (or 120=223) are <br /> degree correlated essentiality annotated              essential (as opposed 37.54% proteins complexes). <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           6                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                             Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  Table 1. Within essential protein complex, essential                         networks (Figures S8 (a) S9 (a)). Since correlation <br />  proteins tend higher average intracomplex degree.                       complex size fraction essential proteins within <br />                                                                                    complex [19], complex size correlated the <br />                                                                                    intracomplex degree member proteins, possible the <br />              Num                 Num Complexes                                     observed relationship intracomplex degree essential- <br />              Complexes           Higher                    Empirical          ity correlation complex size and <br />              Tested              Avg Essential Degree           p-value            essentiality. To address this, normalized interaction <br />  Direct      38                  29 (76.32%)                    7e-4               degree complex size; is, normalized intracomplex <br />                                                                                    degree protein computed number intracomplex <br />  Pull-down   51                  30 (58.82%)                    3e-3 <br />                                                                                    interactions divided complex size. We the <br />  Full        52                  32 (61.54%)                    1e-3 <br />                                                                                    normalized degree essential proteins tends significantly <br />  Num Complexes Tested gives number complexes considered             greater non-essential proteins (Figures 4 (b), S8 (b) and <br />  networks; complex required        S9 (b)). <br />  essential proteins non-essential proteins, with <br />  intracomplex interactions. Num Complexes Higher Avg Essential <br />  Degree gives number complexes tested complexes             Essential complexes processes tend higher <br />  essential proteins higher intracomplex degree average        cross-talk degree module-level network <br />  non-essential proteins. To determine number significant,          As just shown, essential proteins tend more <br />  randomly permuted essentiality assignments computed number of <br />  complexes higher average intracomplex degrees essential proteins. <br />                                                                                    intramodular interactions, complexes essential <br />  Empirical p-value gives fraction random permutations             proteins, essential proteins tend intracomplex <br />  number complexes greater equal actual number,           interactions non-essential proteins. In contrast, the <br />  computed 10,000 permutations.                                                intermodular interaction degree protein weaker <br />  doi:10.1371/journal.pcbi.1002910.t001 <br />                                                                                    relationship essentiality. Nevertheless, noted earlier, <br />                                                                                    significant number intermodular physical interac- <br /> Thus, possible reason essential proteins tend               tions (see Table S2), presumably physical interactions <br /> have higher intracomplex degree (Figure 2) essentiality               connecting different functional modules network are <br /> is enriched proteins belonging multiple complexes,                   important module accomplish task. <br /> intracomplex degree essential protein computed                      We hypothesized essentiality protein complex or <br /> complexes belongs to; however, looking complex                  functional module related topological prominence <br /> at time alleviate problem.                                           module-level network. To test this, built &#8216;&#8216;module <br />    As way addressing possible bias proteins                  network&#8217;&#8217; nodes modules edges between <br /> participating multiple complexes, protein,                          modules enriched number intermodular cross-talk <br /> computed intracomplex degree using interactions                    interactions. In particular, constructed module network for <br /> the largest complex belongs. Next, compared                     protein complexes filtered biological processes each <br /> proteins complexes, significant                   physical interaction network computing number physical <br /> difference degree distribution essential non-                       interactions modules comparing the <br /> essential proteins (Figure 4 (a)), true          average number randomized networks (see Materials and <br />  <br />  <br />  <br />  <br /> Figure 4. Within essential complexes, essential proteins tend higher intracomplex degree Direct network non- <br /> essential proteins. (a) The intracomplex degree (b) normalized intracomplex degree essential proteins significantly larger of <br /> non-essential proteins. Only protein complexes essential proteins non-essential proteins, with <br /> intracomplex interactions, considered. Outliers 2&#8211;98th percentiles shown. The significance difference two <br /> degree distributions determined Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g004 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                              7                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                   Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  Table 2. Module-level networks protein complexes.                                 Table 3. Module-level networks filtered biological <br />                                                                                        processes. <br />  <br />                                                          Fraction of <br />  Network       Num Cross-talks       Num Modules         Essential Modules                           Num                  Num                Fraction Essential <br />                                                                                        Network       Cross-talks          Modules            Modules <br />  Direct        194                   143                 0.68 <br />  Pull-down     535                   242                 0.60                          Direct        1149                 307                0.79 <br />  <br />  Full          727                   279                 0.56                          Pull-down     1409                 321                0.77 <br />                                                                                        Full          2306                 371                0.74 <br />  A module-level network built protein complexes using three <br />  networks. Num Cross-talks gives number inferred cross-talks. Num               A module-level network built filtered biological processes using of <br />  Modules gives number modules inferred cross-talk.            networks. Num Cross-talks gives number inferred cross-talks. <br />  Fraction Essential Modules gives fraction modules having           Num Modules gives number modules inferred cross- <br />  essential protein, modules cross-talk.                  talk. Fraction Essential Modules gives fraction modules having at <br />  doi:10.1371/journal.pcbi.1002910.t002                                                 essential protein, modules cross-talk. <br />                                                                                        doi:10.1371/journal.pcbi.1002910.t003 <br /> Methods). For network, number cross-talks <br /> uncovered using modules derived protein complexes                         complexes 0.23&#8211;0.28 considering filtered biological <br /> or filtered biological processes Tables 2 3, respectively. We                  processes. Further, significantly positive correlation <br /> note number cross-talks processes higher                      normalized cross-talk degree module, defined as <br /> than complexes relatively higher number                         cross-talk degree divided module size, module <br /> interactions processes intermodular intramod-                     essentiality (Table S3). We compared cross-talk degree <br /> ular (86.98% vs. 64.34%, Table S2).                                                   distribution essential non-essential modules using the <br />    For modules, defined complexes filtered biological                    Wilcoxon rank sum test. In networks, considering <br /> processes, decrease threshold number cross-talks                 protein complexes biological processes, essential modules <br /> required module considered hub module,                   significantly higher cross-talk degree non-essential <br /> the fraction modules contain essential protein tends                    modules (Figure 6). Finally, modules larger number <br /> decline (Figure 5). Further, significant positive correlation              proteins greater chance containing essential protein, <br /> between module contains essential protein                        considered fraction proteins module are <br /> cross-talk degree, SRCCs networks w0:33                        essential. We cross-talk degree module is <br /> considering complexes w0:34 considering filtered                             positively correlated fraction proteins module <br /> biological processes. Since modules proteins                       essential (Table S4), values high as <br /> also larger cross-talk degree, computed partial                      binary essentiality (SRCCs networks w0:27 for <br /> correlation cross-talk degree module essentiality                    complexes w0:17 filtered biological processes). <br /> controlling number proteins module (Table S3);                    We observed cross-talks occur functional <br /> varies networks 0.22&#8211;0.29 considering                          modules functionally related (i.e., a <br />  <br />  <br />  <br />  <br /> Figure 5. Cross-talk degree module-level network correlated module essentiality. For (a) protein complexes (b) <br /> filtered biological processes, fraction modules containing essential protein &#8216;&#8216;hub modules&#8217;&#8217; tends decrease each <br /> network modules considered hubs. For data shown, modules added groups 20 non-increasing order cross-talk degree <br /> in Direct (blue), Pull-down (cyan) Full (purple) networks. Correlations computed using SRCC shown network the <br /> binary essentiality module inferred cross-talk degree. The binary essentiality module 1 module essential <br /> protein, 0 otherwise. <br /> doi:10.1371/journal.pcbi.1002910.g005 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                 8                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                     Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 6. The cross-talk degree distribution non-essential modules compared essential modules (a) Direct, (b) <br /> Pull-down (c) Full networks protein complexes (d) Direct, (e) Pull-down (f) Full networks filtered biological <br /> processes. In case, modules essential proteins significantly higher cross-talk degree modules essential proteins, as <br /> determined Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g006 <br />  <br />  <br /> more general, shared biological process). These types cross-            Analysis high-throughput networks <br /> talks interpreted intraprocess interactions broader            Because low-throughput studies biased studying <br /> level functional similarity. As example, Ndc80p                 essential proteins, essential proteins appear more <br /> complex high cross-talk degree networks studied. In           interactions existing network databases. Further, high-through- <br /> the Direct network, uncover seven cross-talks (Figure 7).               studies utilize select set &#8216;&#8216;bait&#8217;&#8217; proteins <br /> Ndc80p component kinetochore, central              bias degree distribution interaction networks. To <br /> chromosome segregation couples chromosomes microtu-                 address potential concerns, performed additional <br /> bule polymers. Two uncovered cross-talks               network analyses. <br /> DASH MIND complexes, kinetochore                   First, repeated analysis Y2H-union network [21] <br /> associated; cross-talks interpreted                recently built BinaryHQHT network [33], both <br /> &#8216;&#8216;intramodular&#8217;&#8217; interactions higher level organization.           derived high-throughput yeast two-hybrid data. In these <br /> On hand, Ndc80 cross-talks                   networks, interactions experiments involving bait <br /> complexes range distinct biological                 proteins removed, high-quality yeast-two hybrid <br /> processes, including nucleosome remodeling complex SWI/                interactions (found experiments) retained. The <br /> SNF, dynactin microtubule associated complex, MRX                  networks notably smaller size focusing annotated <br /> complex involved DNA damage repair, nuclear condensin               interactions (Tables S9, S10 S11); nevertheless, outline <br /> complex nuclear cohesion complex.                                  below, repeating analysis yields similar results Direct <br />    To essential functional modules cross-talks            network. <br /> with functional modules representing truly different biological               In Y2H-union BinaryHQHT networks, intraprocess <br /> processes, considered set expert-selected GO BP terms           interaction degree protein weak statistically <br /> yeast [32], focused annotate 500                 significant correlation essentiality overall degree <br /> proteins. We considered functional module annotated           protein correlated essentiality Y2H-union <br /> one terms &#167;70% proteins annotated it.          network weakly correlated essentiality the <br /> We ignored cross-talks functional modules              BinaryHQHT network (Figures S10 S11, (a) (b)). That is, <br /> they annotated shared term; case,          protein essentiality reflected intraprocess degree these <br /> found essential functional modules correlated          networks, overall degree. Further, intraprocess degree is <br /> cross-talk degree (Table S5). Thus, range analyses reveals       significantly higher essential proteins non- <br /> there relationship topological importance            essential proteins (Figure S10 (d) S11 (d)), true <br /> functional module tendency contain essential proteins.          overall degree interprocess degree (Figures S10 and <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                      9                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 7. Cross-talk network centered Ndc80 complex. Cross-talk analysis Direct network uncovers seven cross-talks between <br /> the Ndc80 complex annotated complexes. The Ndc80 complex kinetochore, cross-talks other <br /> subcomplexes kinetochore (shown dark green box), remaining cross-talks non-kinetochore complexes. An <br /> individual proteins colored red essential, blue otherwise. <br /> doi:10.1371/journal.pcbi.1002910.g007 <br />  <br /> S11, (c) (e)). For 84.62% 78.57% essential protein                    overall interaction degree essentiality (Figure S16 (a) <br /> complexes Y2H-union BinaryHQHT networks, essential                    (b)). Moreover, relationship overall interaction <br /> proteins tend higher average intracomplex degree                  degree essentiality longer significant controlling for <br /> non-essential proteins (Table S12), essential proteins                  intraprocess degree (Figure S16 (b)). Further, intraprocess <br /> higher intracomplex degree normalized intracomplex degree                    degree bait proteins higher essential proteins <br /> than non-essential proteins (Figures S12 S13). Next,                  non-essential proteins (Figure S16), complexes with <br /> module-level analysis, cross-talk degree                   essential bait non-essential bait proteins, essential proteins <br /> complex-level network significantly correlated complex                   higher intracomplex degree (Figure S17). <br /> essentiality BinaryHQHT network (Figure S14(a)),                     Overall, relationships intraprocess degree and <br /> cross-talk degree process-level network significantly                    protein essentiality, degree complexes essentiality, and <br /> correlated process essentiality networks (Figures                   module essentiality cross-talk degree largely recapitulated <br /> S14 (b) S15 (c) (d)). For Y2H-union network,               smaller networks, Y2H-union BinaryHQHT networks, <br /> complexes cross-talks, relationship                    low-throughput experiments experiments biased <br /> between cross-talk degree complex essentiality weak (Figures              essential proteins specifically excluded. Further, <br /> S14 (a) S15 (a)); small number                     comparisons bait proteins, enriched in <br /> intercomplex interactions network (Table S10).                           interactions, confirms relationship number of <br />    In second analysis, removed interactions uncovered                  (intraprocess) interactions protein essential. <br /> low-throughput experiments (where 50 interactions were <br /> determined) Pull-down network, restricted analysis              Discussion <br /> to interaction properties proteins labelled bait proteins. Bait <br /> proteins higher fraction essential proteins network              A long line previous research studied relationship <br /> (Table S14). When considering just bait proteins, stronger               network topology protein essentiality. Recent work <br /> relationships intraprocess degree essentiality                  argued hubs densely connected essential <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           10                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> complexes processes [18], essential complexes tend                Materials Methods <br /> to large [19]. That is, argued essentiality a <br /> modular property, essential proteins essential modules               Physical interaction datasets <br /> tend interactions modules tend large.                   We performed analysis physical interaction datasets. <br /> Our initial analysis, revealing protein&#8217;s intramodular                   For network, physical interactions gathered from <br /> interaction degree predictive essentiality overall          BioGRID [26], release 3.1.78, using evidence codes indicative <br /> degree, largely supports argument. We                physical interactions &#8216;&#8216;Affinity Capture-RNA&#8217;&#8217; and <br /> focus proteins belong complexes, intraprocess             &#8216;&#8216;Protein-RNA.&#8217;&#8217; For early yeast two-hybrid paper Ito et <br /> interaction degree does correlate essentiality better              al. [42], included core data. To remove artifacts to <br /> than overall interaction degree; suggests observed                &#8216;&#8216;sticky proteins&#8217;&#8217; certain experiments, protein than <br /> network modularity essential proteins largely                      30 interactions single experimental data source, we <br /> complexes, general feature biological                      removed interactions. For second network, extracted <br /> processes.                                                                      direct physical interactions initial network utilizing <br />    The observed positive correlation protein essentiality               interactions determined following <br /> and intramodular degree attributed module-                    experimental systems: Biochemical activity, Co-crystal structure, <br /> level complex essentiality. In particular, essential protein             Far western, FRET, Protein-peptide, Reconstituted complex, and <br /> complexes, essential proteins tend                  Two-hybrid. For network, extracted initial <br /> higher intracomplex degrees non-essential counterparts.              network interactions determined Affinity <br /> That is, essential complexes, topological prominence              capture-Western Affinity capture-MS. We refer three <br /> its constitutent proteins related essentiality;        networks Full, Direct Pull-down, respectively, sizes <br /> the importance proteins maintaining structural                  given Table S1. <br /> integrity complexes. This view consistent                      We considered additional networks, comprised of <br /> relative enrichment essentiality proteins                  interactions determined small-scale experimental <br /> structural interfaces opposed just structural                  assays; manner, attempt minimize effect study <br /> interfaces [34].                                                                bias. The networks, refer Y2H-union, <br />    While intermodular interactions                      built earlier study [21]; included interactions <br /> important intramodular interactions explaining protein                  determined large-scale high-quality yeast two-hybrid studies, <br /> essentiality, observed significant number intermodular             excluded experiment using specific set &#8216;&#8216;bait&#8217;&#8217; proteins <br /> interactions physical interaction networks. We considered              enriched essential proteins [43]. We used the <br /> interactions modular level, demonstrated essential                recently built high-throughput yeast two-hybrid network of <br /> functional modules tend cross-talks                     [33], refer BinaryHQHT. Finally, built high- <br /> functional modules. That is, analysis showed                  throughput network Pull-down network keeping only <br /> correlation network topology essentiality               interactions experiments uncovering at <br /> protein level modular level. Further, observed             50 interactions 10 &#8216;&#8216;bait&#8217;&#8217; <br /> that functionally related modules likely interconnect            proteins. We refer second network Pull-downf network, <br /> other, revealing hierarchical structure physical                 use compare network properties bait proteins with <br /> interaction networks.                                                           respect other. <br />    Overall, work advanced understanding the <br /> relationship essentiality network topology. We                 Protein complexes biological processes <br /> shown importance intramodular interactions, especially                      We used set 430 protein complexes compiled [27], <br /> intracomplex interactions, demonstrated essential mod-                 includes SGD Macromolecular Complex GO standard <br /> ules tend higher cross-talk degree non-essential                 [44], CYC2008 protein complex catalog [45] set of <br /> modules. These findings likely yield improvements                 manually curated complexes. From initial set, removed <br /> ability predict protein essentiality. Indeed, integrative machine            highly overlapping complexes follows. First, proteins <br /> learning approaches use range network sequence                    comprising complex subset proteins comprising <br /> features previously applied predict protein essenti-               complex, smaller complex removed. Next, any <br /> ality (e.g., [35&#8211;41]); based work, information                 complexes, Jaccard index proteins making them <br /> functional modules protein complexes, especially respect               (i.e., number overlapping proteins divided size of <br /> to intramodular cross-talk degree, incorporated              union protein sets) &#167;0:5, removed smaller <br /> within frameworks.                                                        complexes. Additionally, previous work [19], removed the <br />    In future, interesting characterize network           complexes corresponding subunits ribosome, as <br /> properties essential proteins central protein                contain large number proteins; is, four <br /> physical interaction networks. Based current findings,                complexes disproportionately affect per-protein analysis. <br /> can speculate proteins important                 After filters, left 390 complexes. (See the <br /> functioning specific essential modules, reflected            Supplement Figure S6, Text S1 intraprocess interprocess <br /> in interactions proteins modules,             results including ribosomal complexes). <br /> these relationships better represented types                   For functional analysis, worked subset specific <br /> interactions (e.g., regulatory, metabolic genetic). Our frame-               Gene Ontology (GO) Biological Process (BP) terms [25] were <br /> work incorporating functional information network                      derived entire GO (version 1.1.2130) follows. First, we <br /> analysis likely useful establishing             extracted 1418 BP terms, annotates 5 yeast <br /> is case. Finally, performed analysis S.                proteins 50. Next, hone contribution a <br /> cerevisiae, approach applied study essential proteins          specific biological process (as opposed effects arising from <br /> other well-annotated organisms large-scale interaction                     proteins annotated process within <br /> networks genome-scale gene deletion disruption data.                     protein complexes), pruned set proteins are <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          11                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> associated functional terms. More specifically, size          tSS(a,b)~0. Moreover, terms same, then <br /> of intersection biological process                 tSS(a,b)~1. <br /> original set 430 protein complexes &#167;2, proteins                   This measure naturally extended functional relationships <br /> intersection longer associated process. If left           proteins multiple annotations. For protein p, <br /> fewer 2 proteins associated process,            let A(p) set terms p annotated. If term <br /> half number proteins known annotate,             annotates p, parent terms naturally included in <br /> term removed consideration. Finally, highly overlapping                A(p). Then, proteins p q, protein semantic <br /> processes removed manner described                    similarity (pSS) defined follows [7]: <br /> for complexes. This procedure resulted 391 &#8216;&#8216;filtered&#8217;&#8217; processes, <br /> with 2567 proteins associated processes.                                        2: maxa[A(p)\A(q) s(a) <br />                                                                                            pSS(p,q)~                                   : <br />                                                                                                         maxa[A(p) s(a)z maxa[A(q) s(a) <br /> Detecting cross-talk complexes processes <br />    For given network, exhaustively determined pairs of <br /> functional modules enriched number interactions <br /> found [10]. We considered modules arising                     Supporting Information <br /> complexes processes turn (i.e., functional modules consist of <br />                                                                                 Figure S1 The intraprocess interaction degree more <br /> either proteins complex, shared <br />                                                                                 correlated protein essentiality overall <br /> process annotation 391 filtered processes considered). We <br />                                                                                 interaction degree proteins Pull-down net- <br /> considered proteins network associated with <br />                                                                                 work, interactions categorized specific BP <br /> any modules considering, edges <br />                                                                                 terms, annotates 50 proteins. (a) <br /> that correspond intermodular interactions these <br />                                                                                 The fraction essential proteins hub proteins more <br /> proteins. Next, modules c1 c2 counted the <br />                                                                                 proteins considered hub proteins; proteins added groups <br /> number &#8216;&#8216;cross-talk&#8217;&#8217; interactions proteins compris-             50 non-increasing order interaction degree. This <br /> ing modules. Note interactions           fraction highest intraprocess degree (green), followed all <br /> proteins annotated c1 c2 included            annotated degree (blue) interprocess degree (red). (b) <br /> are intramodular interactions. To determine number               The correlations measured SRCCs essentiality and <br /> observed cross-talk interactions pair            intraprocess, annotated interprocess degree. The <br /> expected chance, randomized intermodular interactions                 SRCC highest essentiality intraprocess degree. The <br /> within network 100 times using stub-rewiring (as [8]),           partial correlation computed annotated degree <br /> preserving degree distribution, module annotation, overall              essentiality controlling intraprocess or <br /> number cross-talk interactions. Then, countfc1 ,c2 g number        interprocess degree. Starred p-values indicate values <br /> of cross-talk interactions c1 c2 real network,           w0:05. (c)&#8211;(e) The degree distribution non-essential proteins is <br /> avgfc1 ,c2 g average number corresponding cross-talk                  compared essential proteins (c) annotated, (d) <br /> interactions randomized networks, odds-score module               intraprocess, (e) interprocess degree, respectively. In box <br /> pair defined as:                                                             plot, horizontal bar box corresponds median of <br />                                                                                 distribution; ends box indicate third <br />                           (countfc1 ,c2 g z1)                                   quartiles; small circles outliers 2&#8211;98th <br />                                               :                                 percentile range. The significance difference the <br />                            (avgfc1 ,c2 g z1)                                    degree distributions measured Wilcoxon rank sum <br />                                                                                 test. <br />                                                                                 (TIFF) <br />   The addition pseudocount 1 downweighs the <br /> contribution rare cross-talks obtain               Figure S2 The intraprocess interaction degree more <br /> high scores simply small (or zero) average counts            correlated protein essentiality overall <br /> randomized graphs. In order module pair considered                interaction degree proteins Full network, when <br /> cross-talk, required                       interactions categorized specific BP terms, <br /> independent (i.e., non-overlapping) cross-talk interactions,                annotates 50 proteins. (a) The <br /> that odds-score 2. The observed relationship             fraction essential proteins hub proteins proteins <br /> between module essentiality cross-talk degree persists                considered hub proteins; proteins added groups 50 in <br /> range odds-scores (see Tables S6&#8211;S8).                                        non-increasing order interaction degree. This fraction is <br />                                                                                 highest intraprocess degree (green), followed annotated <br />                                                                                 degree (blue) interprocess degree (red). (b) The <br /> Semantic similarity                                                             correlations measured SRCCs essentiality either <br />    The semantic similarity GO terms                 intraprocess, annotated interprocess degree. The SRCC is <br /> ontology estimate functional similarity                highest essentiality intraprocess degree. The partial <br /> terms. We use semantic similarity measure introduced [30].               correlation computed annotated degree and <br /> In particular, let f (a) fraction proteins yeast annotated         essentiality controlling intraprocess interprocess <br /> with term total number proteins. Then                            degree. Starred p-values indicate values w0:05. (c)&#8211;(e) <br /> s(a)~{log(f (a)) measure specific term is. We                   The degree distribution non-essential proteins compared to <br /> compute term semantic similarity b, tSS(a,b)                    essential proteins (c) annotated, (d) intraprocess, and <br />             2:s(LCA(a,b))                                                       (e) interprocess degree, respectively. In box plot, the <br /> tSS(a,b)~                     , LCA(a,b) common <br />                s(a)zs(b)                                                        horizontal bar box corresponds median the <br /> ancestor b GO ontology. Note LCA                 distribution; ends box indicate third <br /> two terms <span id='am-16' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span>root</span> term (e.g., GO:0008150 &#8216;biological process&#8217;),               quartiles; small circles outliers 2&#8211;98th <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          12                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                       Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> percentile range. The significance difference             essentiality intracomplex degree. The partial <br /> two degree distributions measured Wilcoxon rank sum                correlation computed annotated degree and <br /> test.                                                                        essentiality controlling intracomplex degree, <br /> (TIFF)                                                                       intercomplex degree, size largest complex which <br />                                                                              protein belongs. Starred p-values indicate values <br /> Figure S3 In networks, semantic similarity <br />                                                                              w0:05. (c)&#8211;(f) The degree distribution non-essential proteins is <br /> degree correlated protein essentiality than <br />                                                                              compared essential proteins complexes (c) all <br /> the overall interaction degree. (a)&#8211;(c) The fraction of <br />                                                                              annotated degree, (d) intracomplex degree, (e) intercomplex <br /> essential proteins hub proteins proteins are <br /> considered hub proteins Direct, Pull-down Full networks;         degree, (f) complex size. In box plot, horizontal <br /> proteins added groups 50 non-increasing order          bar box corresponds median distribution; the <br /> semantic similarity degree. For network, SRCC                    ends box indicate quartiles; the <br /> computed protein essentiality semantic                    small circles outliers 2&#8211;98th percentile range. The <br /> similarity annotated degree; values boxed           significance difference degree distributions <br /> panel. (d)&#8211;(f) The semantic similarity weighted degree distribution          measured Wilcoxon rank sum test. <br /> of non-essential proteins compared essential proteins          (TIFF) <br /> for Direct, Pull-down Full networks.                                 Figure S6 The correlations interaction degree <br /> (TIFF)                                                                       essentiality proteins complexes, including <br /> Figure S4 The intracomplex interaction degree                        ribosomal complexes, (a) Direct, (b) Pull-down and <br /> correlated protein essentiality overall                        (c) Full networks. Interactions categorized using protein <br /> interaction degree proteins Pull-down net-                        complexes including ribosomal complexes. The fraction of <br /> work, interactions categorized using protein                        essential proteins hub proteins proteins are <br /> complexes. (a) The fraction essential proteins hub                  considered hub proteins; proteins added groups 50 a <br /> proteins proteins considered hub proteins; proteins          non-increasing order interaction degree size the <br /> added groups 50 non-increasing order interaction           largest complex protein belongs. The relationship <br /> degree size largest complex protein               protein essentiality interaction degree shown for <br /> belongs. The relationship protein essentiality                   intracomplex interactions (green), annotated interactions (blue) <br /> interaction degree shown intracomplex interactions (green),           intercomplex interactions (red). The relationship between <br /> all annotated interactions (blue) intercomplex interactions              protein essentiality complex size shown (black). <br /> (red). The relationship protein essentiality complex             (TIFF) <br /> size shown (black). (b) The correlations measured                 Figure S7 Essentiality correlated intra- <br /> SRCCs essentiality intracomplex degree,               modular interaction degree intermodular <br /> annotated degree, intercomplex degree, complex size. The                  total degree, modules determined each <br /> SRCC highest essentiality intracomplex degree.                network ((a) Direct, (b) Pull-down (c) Full) via <br /> The partial correlation computed annotated               network clustering approaches. To obtain clusters, used <br /> degree essentiality controlling intracomplex             SPICi clustering algorithm, local clustering approach, a <br /> degree, intercomplex degree, size largest complex           density threshold 0.5 minimum increment ratio 0.3. <br /> which protein belongs. Starred p-values indicate              The fraction essential proteins hub proteins more <br /> values w0:05. (c)&#8211;(f) The degree distribution non-essential               proteins considered hub proteins; proteins added groups <br /> proteins compared essential proteins complexes          50 non-increasing order interaction degree. The <br /> for (c) annotated degree, (d) intracomplex degree, (e)                   relationship protein essentiality interaction degree is <br /> intercomplex degree, (f) complex size. In box plot,             shown intramodular interactions (green), total interactions <br /> horizontal bar box corresponds median                 (blue) intermodular interactions (red). <br /> distribution; ends box indicate           (TIFF) <br /> quartiles; small circles outliers 2&#8211;98th <br /> percentile range. The significance difference             Figure S8 Within essential complexes, essential proteins <br /> two degree distributions measured Wilcoxon rank sum                tend higher intracomplex degree Pull- <br /> test.                                                                        network non-essential proteins. (a) The intra- <br /> (TIFF)                                                                       complex degree (b) normalized intracomplex degree of <br />                                                                              essential proteins significantly larger non-essential <br /> Figure S5 The intracomplex interaction degree                        proteins. Only protein complexes essential <br /> correlated protein essentiality overall                        proteins non-essential proteins, with <br /> interaction degree proteins Full network,                    intracomplex interactions considered. Outliers 2&#8211; <br /> interactions categorized using protein complexes. (a)                    98th percentiles shown. The significance difference the <br /> The fraction essential proteins hub proteins                degree distributions determined Wilcoxon rank sum test. <br /> proteins considered hub proteins; proteins added groups <br />                                                                              (TIFF) <br /> of 50 non-increasing order interaction degree size <br /> of largest complex protein belongs. The                     Figure S9 Within essential complexes, essential proteins <br /> relationship protein essentiality interaction degree             tend higher intracomplex degree Full <br /> is shown intracomplex interactions (green), annotated                network non-essential proteins. (a) The intracomplex <br /> interactions (blue) intercomplex interactions (red). The                 degree (b) normalized intracomplex degree essential <br /> relationship protein essentiality complex size           proteins significantly larger non-essential proteins. <br /> shown (black). (b) The correlations measured SRCCs                Only protein complexes essential proteins <br /> essentiality intracomplex degree, annotated degree,           non-essential proteins, intracomplex <br /> intercomplex degree, complex size. The SRCC highest                    interactions considered. Outliers 2&#8211;98th percentiles <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                       13                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> are shown. The significance difference degree                  interprocess degree. Further, essential proteins higher <br /> distributions determined Wilcoxon rank sum test.                       average intraprocess degree non-essential proteins this <br /> (TIFF)                                                                           network (panel (d)), significant difference in <br />                                                                                  interprocess degree (panel (e)). <br /> Figure S10 The intraprocess interaction degree more <br />                                                                                  (TIFF) <br /> correlated protein essentiality overall <br /> interaction degree proteins Y2H-union network,                        Figure S12 Essential proteins tend higher <br /> when interactions categorized specific GO BP                            intracomplex degree non-essential proteins within <br /> terms, annotates 50 proteins. (a) The                      protein complexes Y2H-union network. (a) The <br /> fraction essential proteins hub proteins proteins           intracomplex degree (b) normalized intracomplex degree of <br /> considered hub proteins; proteins added groups size 20 (or             essential proteins significantly greater non-essential <br /> larger proteins degree group).            proteins. Only protein complexes essential <br /> This fraction highest intraprocess degree (green), followed        proteins non-essential proteins intracomplex <br /> annotated degree (blue) interprocess degree (red). (b) The           interactions tested. Outliers 2&#8211;98% shown. The <br /> correlations measured SRCCs essentiality                   significance difference degree distributions is <br /> intraprocess, annotated interprocess degree. The SRCC                  determined Wilcoxon rank sum test. <br /> highest essentiality intraprocess degree. The partial                (TIFF) <br /> correlation computed annotated degree                    Figure S13 Essential proteins tend higher <br /> essentiality controlling intraprocess interprocess            intracomplex degree non-essential proteins within <br /> degree. Starred p-values indicate values w0:05. (c)&#8211;(e)               protein complexes BinaryHQHT network. (a) The <br /> The degree distribution non-essential proteins compared            intracomplex degree (b) normalized intracomplex degree of <br /> of essential proteins (c) annotated, (d) intraprocess, (e)           essential proteins significantly greater non-essential <br /> interprocess degree, respectively. In box plot, horizontal bar          proteins. Only protein complexes essential <br /> within box corresponds median distribution;              proteins non-essential proteins intracomplex <br /> ends box indicate quartiles; small            interactions tested. Outliers 2&#8211;98% shown. The <br /> circles outliers 2&#8211;98th percentile range. The                    significance difference degree distributions is <br /> significance difference degree distributions              determined Wilcoxon rank sum test. <br /> is measured Wilcoxon rank sum test. For Y2H-union                     (TIFF) <br /> network, essentiality intraprocess degree small but <br /> statistically significant correlation. Essentiality significantly         Figure S14 Cross-talk degree module-level network <br /> correlated overall degree interprocess degree. Further,                 correlated module essentiality. For (a) protein <br /> essential proteins higher average intraprocess degree non-             complexes (b) filtered biological processes, fraction of <br /> essential proteins network (panel (d)),             modules containing essential protein &#8216;&#8216;hub <br /> significant difference annotated degree interprocess degree            modules&#8217;&#8217; tends decrease network modules are <br /> (panels (c) (e)).                                                            considered hubs. For data shown, modules added non- <br /> (TIFF)                                                                           increasing order cross-talk degree Y2H-union (blue) and <br />                                                                                  BinaryHQHT (cyan) networks. Correlations binary <br /> Figure S11      The intraprocess interaction degree                      essentiality module inferred cross-talk degree are <br /> correlated protein essentiality overall                            computed using SRCC shown network. The <br /> interaction degree proteins BinaryHQHT                                binary essentiality module 1 module one <br /> network, interactions categorized specific                         essential protein, 0 otherwise. For Y2H-union module <br /> GO BP terms, annotates 50                                  network comprised complexes (panel (a)), correlation not <br /> proteins. (a) The fraction essential proteins hub                       significant uncover complexes crosstalks in <br /> proteins proteins considered hub proteins; proteins              small network (549 edges, Table S10). <br /> added groups size 20 (or larger proteins             (TIFF) <br /> same degree group). This fraction highest for <br /> intraprocess degree (green), followed annotated degree (blue)             Figure S15 The cross-talk degree distribution non- <br /> and interprocess degree (red). (b) The correlations                      essential modules compared essential <br /> measured SRCCs essentiality intraprocess,                  modules (a) Y2H-union (b) BinaryHQHT <br /> all annotated interprocess degree. The SRCC highest                        networks protein complexes (c) Y2H-union and <br /> between essentiality intraprocess degree. The partial correla-               (d) BinaryHQHT networks filtered biological pro- <br /> tion computed annotated degree essentiality              cesses. For Y2H-union BinaryHQHT networks, modules <br /> when controlling intraprocess interprocess degree.                 (derived biological processes) essential proteins have <br /> Starred p-values indicate values w0:05. (c)&#8211;(e) The                   significantly higher cross-talk degree modules without <br /> degree distribution non-essential proteins compared             essential proteins, determined Wilcoxon rank sum test. <br />                                                                                  For modules derived complexes Y2H-union network, <br /> essential proteins (c) annotated, (d) intraprocess, (e) <br />                                                                                  differences essential non-essential modules are <br /> interprocess degree, respectively. In box plot, horizontal <br />                                                                                  significant, modules can <br /> bar box corresponds median distribution; the <br />                                                                                  uncover cross-talks. <br /> two ends box indicate quartiles; the <br />                                                                                  (TIFF) <br /> small circles outliers 2&#8211;98th percentile range. The <br /> significance difference degree distributions              Figure S16 The intraprocess interaction degree is <br /> is measured Wilcoxon rank sum test. For BinaryHQHT                    correlated protein essentiality the <br /> network, essentiality intraprocess degree small                   overall interaction degree bait proteins the <br /> statistically significant correlation. Essentiality smaller                Pull-downf network excluding small-scale experi- <br /> correlation overall degree correlated                       ments, interactions categorized spe- <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           14                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> cific GO BP terms, annotates 50                              Table S6 The significant correlation cross-talk <br /> proteins. All tests bait proteins Pull-                  degree binary module essentiality persists a <br /> downf network. (a) The fraction essential proteins hub                    range odd-scores Direct network. <br /> proteins proteins considered hub proteins; proteins                    (PDF) <br /> are added non-increasing order interaction degree.                     Table S7 The significant correlation cross-talk <br /> This fraction highest intraprocess degree (green),                          degree binary module essentiality persists a <br /> followed annotated degree (blue)                                range odd-scores Pull-down network. <br /> interprocess degree (red). (b) The correlations measured                        (PDF) <br /> SRCCs essentiality intraprocess, all <br /> annotated interprocess degree. The SRCC highest                              Table S8 The significant correlation cross-talk <br /> between essentiality intraprocess degree. The partial                          degree binary module essentiality persists a <br /> correlation computed annotated degree                          range odd-scores Full network. <br /> and essentiality controlling intraprocess                       (PDF) <br /> interprocess degree. Starred p-values indicate values                   Table S9   The number proteins, number of <br /> w0:05. (c)&#8211;(e) The degree distribution non-essential                            interactions fraction essential proteins for <br /> proteins compared essential proteins (c)                     Y2H-union BinaryHQHT physical interaction <br /> annotated, (d) intraprocess, (e) interprocess degree,                          networks. <br /> respectively. In box plot, horizontal bar                        (PDF) <br /> box corresponds median distribution; ends <br /> of box indicate quartiles; small                   Table S10 A substantial fraction physical interactions <br /> circles outliers 2&#8211;98th percentile range. The                      Y2H-union network intermodular. <br /> significance difference degree                              (PDF) <br /> distributions measured Wilcoxon rank sum test.                           Table S11 A substantial fraction physical interactions <br /> (TIFF)                                                                             BinaryHQHT network intermodular. <br /> Figure S17     Within essential complexes, essential bait                          (PDF) <br /> proteins tend higher intracomplex degree                              Table S12   Within essential protein complex, <br /> the Pull-downf network non-essential bait proteins.                           essential proteins tend higher average <br /> (a) The intracomplex degree (b) normalized intracomplex                     intracomplex degree Y2H-union BinaryHQHT <br /> degree essential bait proteins significantly larger             networks. <br /> non-essential bait proteins. Only protein complexes                   (PDF) <br /> least essential bait proteins non-essential bait <br />                                                                                    Table S13Module-level networks Y2H-union and <br /> proteins, intracomplex interactions, considered. <br />                                                                                    BinaryHQHT networks. <br /> Outliers 2&#8211;98th percentiles shown. The significance <br />                                                                                    (PDF) <br /> of difference degree distributions is <br /> determined Wilcoxon rank sum test.                                          Table S14       Numbers bait proteins Pull-downf <br /> (TIFF)                                                                             network. <br />                                                                                    (PDF) <br /> Table S1   The number proteins, number of <br /> interactions fraction essential proteins                            Table S15   Within essential protein complex, <br /> each physical interaction networks consid-                            essential bait proteins tend higher average <br /> ered.                                                                              intracomplex degree non-essential bait proteins in <br /> (PDF)                                                                              Pull-downf network. <br />                                                                                    (PDF) <br /> Table S2 A substantial fraction physical interactions <br /> in Direct network intermodular.                                            Text S1 Analysis correlations interac- <br /> (PDF)                                                                              tion degree essentiality proteins complex- <br />                                                                                    es, including ribosomal complexes, net- <br /> Table S3 Correlation cross-talk (CT) degree                                works. <br /> and binary module essentiality.                                                    (PDF) <br /> (PDF) <br /> Table S4 Correlation cross-talk (CT) degree                                Acknowledgments <br /> and fraction essential proteins module.                              The authors thank members Singh group helpful discussions <br /> (PDF)                                                                              comments manuscript. <br /> Table S5 Correlation cross-talk (CT) degree <br /> and binary module essentiality removing function-                            Author Contributions <br /> ally similar cross-talks.                                                          Conceived designed experiments: JS MS. Performed the <br /> (PDF)                                                                              experiments: JS. Analyzed data: JS MS. Wrote paper: JS MS. <br />  <br /> References <br />  1. Aittokallio T, Schwikowski B (2006) Graph-based methods analysing           3. Rives AW, Galitski T (2003) Modular organization cellular networks. Proc <br />     networks cell biology. Briefings Bioinformatics 7: 243&#8211;255.                  Natl Acad Sci USA 100: 1128&#8211;1133. <br />  2. Hartwell LH, Hopfield JJ, Leibler S, Murray AW (1999) From molecular         4. Spirin V, Mirny LA (2003) Protein complexes functional modules in <br />     modular cell biology. Nature 402: C47&#8211;C52.                                         molecular networks. Proc Natl Acad Sci USA 100: 12123. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                             15                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                              Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  5. Bader GD, Hogue CW (2003) An automated method finding molecular                       26. Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz A, et al. (2006) <br />     complexes large protein interaction networks. BMC Bioinformatics 4: 2.                     BioGRID: general repository interaction datasets. Nucleic Acids Research <br />  6. Pereira-Leal J, Enright AJ, Ouzounis CA (2004) Detection functional modules                34(Database Issue): D535&#8211;D539. <br />     protein interaction networks. Proteins 54: 49&#8211;57.                                    27. Baryshnikova A, Costanzo M, Kim Y, Ding H, Koh J, et al. (2010) Quantitative <br />  7. Song J, Singh M (2009) How interactome-derived clusters used               analysis fitness genetic interactions yeast genome scale. Nature <br />     predict functional modules protein function? Bioinformatics 25: 3143&#8211;3150.             Methods 7: 1017&#8211;1025. <br />  8. Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al. (2002) Network           28. Schwikowski B, Uetz P, Fields S (2000) A network protein-protein interactions <br />     motifs: simple building blocks complex networks. Science 298: 824&#8211;827.                     yeast. Nature Biotechnology 18: 1257&#8211;1261. <br />  9. Luscombe NM, Babu MM, Yu H, Snyder M, Teichmann SA, et al. (2004)                         29. Spearman C (1904) The proof measurement association two <br />     Genomic analysis regulato <br /> </body></html>