<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Evolutionary Bioinformatics <br />  <br />  <br />                                                                                                              Open Access <br />                                                                                                        Full open access and <br />                                                                                                       thousands papers at <br /> O r g nal Resea r c h <br />                                                                                                         http://www.la-press.com. <br />  <br />  <br /> Prediction Protein Essentiality Support Vector <br /> Machine Statistical Tests <br />  <br /> Chiou-Yi Hor, Chang-Biau Yang, Zih-Jie Yang Chiou-Ting Tseng <br /> Department Computer Science Engineering, National Sun Yat-sen University Kaohsiung 80424, Taiwan. <br /> Corresponding author email: cbyang@cse.nsysu.edu.tw <br />  <br />  <br />  <br />  <br /> Abstract: Essential proteins include minimum required set proteins support cell life. Identifying essential proteins important <br /> for understanding cellular processes organism. However, identifying essential proteins experimentally extremely time- <br /> consuming labor-intensive. Alternative methods developed examine essential proteins. There goals <span id='am-38' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> study: identifying important features building learning machines discriminating essential proteins. <span id='am-100' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-101' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Data</span> Saccharomy- <br /> ces cerevisiae <span id='am-70' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> used. We collected information variety sources. We proposed modified <br /> backward feature selection method build support vector machines (SVM) predictors based selected features. To evaluate <br /> the performance, conducted cross-validations originally imbalanced data set down-sampling balanced data set. The <br /> statistical tests applied performance associated obtained feature subsets confirm significance. In data <br /> set, best values F-measure Matthews correlation coefficient (MCC) 0.549 0.495 imbalanced experiments. <br /> For balanced experiment, best values F-measure MCC 0.770 0.545, respectively. In second data set, our <br /> best values F-measure MCC 0.421 0.407 imbalanced experiments. For balanced experiment, best values <br /> of F-measure MCC 0.718 0.448, respectively. The experimental results selected features compact and <br /> the performance improved. Prediction conducted users following internet address: http://bio2.cse.nsysu.edu.tw/ <br /> esspredict.aspx. <br />  <br /> Keywords: support vector machine, feature selection, protein-protein interaction, essential protein, statistical test <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 387&#8211;416 <br />  <br /> doi: 10.4137/<span id='am-105' about='obo:NCBITaxon_1109606' typeof='owl:Thing obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_94020 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_243501 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Ebo" datatype="xsd:string"></span>EBO</span>.S11975 <br />  <br /> This article available http://www.la-press.com. <br />  <br /> &#169; author(s), publisher licensee Libertas Academica Ltd. <br />  <br /> This open access article published Creative Commons CC-BY-NC 3.0 license. <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                  387 <br />  Hor et&#160;al <br />  <br />  <br /> Introduction                                                preliminary properties alone <br /> Identifying essential proteins important under-      somewhat relevant essentiality. However, <br /> standing cellular processes organism      information highly accessible only <br /> no proteins perform functions essen-       sequence information required derive these <br /> tial proteins. Once essential protein removed,        properties. Hence, included properties our <br /> dysfunction cell death results. Thus, stud-      study. For topological properties, addition physi- <br /> ies conducted identify essential proteins.     cal interactions, incorporated variety interac- <br /> Experimental approaches identifying essential           tion information, including metabolic, transcriptional <br /> proteins include gene deletion,1 RNA interference,2         regulation, integrated functional, genomic con- <br /> and conditional knockouts.3 However, meth-            text interactions. Our experimental results revealed <br /> ods labor-intensive time-consuming. Hence,          features provide complement infor- <br /> alternative methods identifying essential proteins      mation essentiality identification provide other <br /> are necessary.                                              biological justification. <br />     The essential protein classification problem               To identify reduced feature subset, is <br /> involves determining necessity protein <span id='am-48' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-    crucial biological processes, previous studies <br /> taining cellular function life. Among methods        used feature selection techniques. The advan- <br /> available identifying essential proteins, machine-      tages method include storage reduction, <br /> learning based methods promising approaches.            performance improvement data interpretation.12 <br /> Therefore, studies conducted          In accordance feature selection <br /> examine effectiveness technique. Chin4         procedure bound predictor, method <br /> proposed double-screening scheme constructed         roughly classified categories: filter, <br /> a framework known hub analyzer (http://hub.iis.     wrapper, embedded. Filter methods pro- <br /> sinica.edu.tw/Hubba/index.php) rank &#173;proteins.      vide complete <span id='am-66' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> available features terms <br /> Acencio Lemke5 used Waikato Environment            relevance measures. Methods Fisher <br /> Knowledge Analysis (WEKA)6 predict essen-           score,12 mutual information, minimal redundancy <br /> tial proteins. Hwang et&#160; al7 applied support vector      maximal relevance (mRMR),13 conditional <br /> machine (SVM) classify proteins.                    mutual information maximization (CMIM),14 and <br />     Protein-protein interactions (PPIs) well-known     minimal relevant redundancy (mRR)15 belong to <br /> to significant characteristics protein function.     category. Both wrapper embedded methods <br /> Several studies attempted predict classify     involve selection process learning <br /> protein function8 analyze protein pheno-        &#173;algorithm. The utilizes learning machine to <br /> type9 studying interactions. A previous study10 fur-     evaluate subsets features according per- <br /> ther suggested essential proteins nonessential     formance measurements. For example, sequential <br /> proteins discriminated means topological       backward forward feature selection12 falls into <br /> properties derived PPI network. In spite        &#173;category. Embedded methods directly perform <br /> the superior properties, however, analyzing PPI       feature selection learning process are <br /> experimentally time-consuming. With advent        usually specific given learning machines. Example <br /> yeast two-hybrid11 high-throughput techniques,        include C4.5,16 Classification Regression Trees <br /> can used identify PPIs experi-         (CART),17 ID3.18 Additionally, research- <br /> ment, obtaining PPI information easier.          ers proposed information-gain based feature <br /> Since PPI network similar social network          selection method,19 examines effective- <br /> in aspects, researchers apply social net-         ness classifier combination. <br /> work techniques analyzing PPI networks. Thus,              In paper, used datasets. The was <br /> several topological properties extensively        <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. The corresponding <br /> explored studied recent years.                       PPI data set Scere20070107, obtained <br />     Fundamental properties, sequence             DIP database. The data set totally contains <br /> protein physiochemical ones, subjected           4873 proteins 17,166&#160;interactions. Our feature set <br /> detailed examination previous studies. This          consisted features obtained extracted from <br />  <br /> 388	                                                                                    Evolutionary Bioinformatics 2013:9 <br />                                                                                             Essentiality prediction SVM <br />  <br />  <br /> the methods proposed Acencio Lemke,5 Chin,4          &#173; atabase (SGD), located http://www. <br />                                                             D <br /> Hwang et&#160; al,7 Lin et&#160; al.20 The second data set        yeastgenome.org/. Since data set used in <br /> was <span id='am-86' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli, compiled         previous studies, obtained incor- <br /> by Gustafson et&#160; al.21 The data set totally contained       porated various related features experiments. <br /> 3569 proteins. The associated network information              The E. coli data set obtained Gustafson <br /> included physical, integrated functional, genomic       et&#160;al.21 It contained 3569 proteins, 611 <br /> context interactions collected Hu et&#160;al.22      essential. Due availability coverage issues, <br /> For data sets, propose modified sequen-           used information additional <br /> tial backward feature selection method selecting        networks: physical interaction (PI), integrated func- <br /> important features.                                         tional interaction, integrated PI genomic <br />     Next, SVM models built using selected          context (GC) network. The information collected <br /> feature subsets. In study, SVM software LIB-       Hu et&#160;al.22 <br /> SVM23 adopted classification models. Each              In data sets, ratio nonessential <br /> model applied imbalanced balanced           proteins essential proteins approximately 4:1 <br /> data sets. The results compared          5:1, respectively. The data imbalance inevi- <br /> previous studies statistical tests conducted    tably led biased fitting nonessential proteins <br /> examine significance. For imbalanced S. &#173;cerevisiae     learning processes. Thus, constructed <br /> data, best results F-measure MCC           balanced data set. Taking data set, for <br /> 0.549 0.495, respectively, outperform         example, randomly selected 975 nonessential pro- <br /> best previous method7 results 0.354 0.36,       teins mixed essential proteins form <br /> respectively. We obtained values 0.770 0.545     balanced data set. In new data set, number <br /> F-measure MCC balanced data experiment,          nonessential data elements essential <br /> which superior best previous method7        elements equal. <br /> 0.737 0.492, respectively. For experiments exam- <br /> ining E. coli data set, best values F-measure   Bootstrap cross-validation <br /> and MCC 0.421 0.407, respectively,          We used bootstrap cross-validation (BCV) compare the <br /> imbalanced data set. In balanced experiment,        performance classifiers using k-fold cross- <br /> best values F-measure MCC 0.718            validation. Assume sample S&#160;=&#160;{(x1, y1), (x2, y2), &#8230;, <br /> 0.448, respectively. The results similar    (xn, yn)} composed n observations, xi represents <br /> Gustafson et&#160; al,21 examined 29 features,        feature vector ith observation yi denotes <br /> our method, seven features used       <span id='am-35' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> <span id='am-93' about='rdfs:label' typeof='owl:Thing'>label</span> associated xi. A bootstrap sample <br /> prediction. To verify improvement                 {                                      } <br />                                                             Sb* = ( x1* , y1* ) , ( x *2 , y2* ) , &#8230;, ( x *n , yn* ) consists n <br /> statistically significant, performed bootstrap cross-    obser&#173;vations sampled S replacement, <br /> validation24 performance measures.                       1 # b # B, B constant 50 and <br />                                                             200. For sample Sb*, k-fold cross-&#173;validation was <br /> Background                                                  carried out. The performance measure cb, error <br /> The data set                                                rate, calculated Sb*. The procedure repeated <br /> In paper, used data sets experiments:       B times average performance measure <br /> S. cerevisiae E. coli. The included PPI          CB&#160; =&#160; &#8721;Bb=1 cb/B evaluated B bootstrap <br /> network data. We downloaded data set           samples. Since distribution bootstrap per- <br /> DIP (http://dip.doe-mbi.ucla.edu/) website.25 The           formance measures approximately normal, the <br /> original data set contained 4873 proteins 17166         confidence interval significance estimated <br /> interactions. To comply previous studies,           accordingly. <br /> also adopted largest connected component the <br /> network data. There total 4815 proteins,          Performance measures <br /> including 975 essential proteins 3840 nonessen-         In study, performance measures included <br /> tial proteins. The information protein essential-        precision, recall, F-measure (F1), Matthews cor- <br /> ity obtained <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> Genome              relation coefficient (MCC), percentage <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                         389 <br />  Hor et&#160;al <br />  <br />  <br /> of essential proteins. Their formulas given             tion, desire protein. We first <br /> follows:                                                       select 2 proteins using ranking method A, <br />                                                                W X. Next, use method B rank <br /> 1.	 Precision: TP&#8201;/&#8201;TP + FP <br />                                                                proteins. The ranks W X 2 <br /> 2.	 Recall: TP&#8201;/&#8201;TP + FN <br />                                                                1, respectively. Hence, iteration, X <br /> 3.	 F-measure: 2 &#215; precision &#215; recall&#8201;/(&#8202;precision + recall) <br />                                                                finally selected. It follows bit M [X, 1] set <br />                         TP &#215; TN &#8722; FP &#215; FN                      1, others, M [W, 1], M [Y, 1] M [Z, 1], <br /> 4.	 MCC: <br />              (TP + FN )(TP + FP)(TN + FP)(TN + FN )            set 0. In second iteration, 2 top-ranking <br />                                                                proteins found. First, proteins W, X, <br /> 5.	 Top     percentage        essential    protein: <br />                                                                Y Z selected, 4 <br />     Number real essential proteins n .               proteins ranking method A. Next, ranking <br />                         n                                      method B, select 2 proteins them, <br />     Here, essential protein represented           X Y. Hence, bits corresponding <br /> positive observation. True positive (TP), true neg-            M [X, 2] M [Y, 2] set 1, others <br /> ative (TN), false positive (FP), false negative            set 0&#160;in iteration. Finally, sum the <br /> (FN) represent numbers true positive, true              bits protein, shown fourth column <br /> negative, false positive, false negative proteins,         Table&#160;3. <br /> &#173;respectively. The value n denotes total num-                  There issue bit <span id='am-69' about='xsd:string' typeof='owl:Thing'>string</span> implemen- <br />  ber predictions. In addition, receiver operating           tation, is, M sparse handled <br />  characteristic (ROC) curve18 area curve             &#173;classifiers. Since number proteins being <br />  (AUC) used evaluate classification                selected n/2, sum n/2&#160; bits is <br />  performance.                                                  close 0. In experience, makes difficult <br />                                                                distinguish proteins. To overcome this <br /> Feature extraction                                             problem, protein, added score <br /> The feature set used included sequence properties           n &#8211; r sum bit string, r rank <br /> (S), amino acid occurrence average amino           protein ranking method B. In study, <br /> acid PSSM; protein properties (P), cell cycle          used DMNC rank A MNC rank B. In <br /> and metabolic process; topological properties (T),        example, n&#160;=&#160;4, values n &#8211; r W, X, Y <br /> as bit string double screening scheme between-          Z 0, 2, 3, 1, respectively. We summed the <br /> ness centrality related physical interactions;    values bit string; hence, final scores are <br /> properties (O), phyletic retention essential       0, 4, 4, 1. The overall procedure given the <br /> index. There total 45&#160;groups 90 features         Procedure bit string implementation DSS. <br /> in S. cerevisiae data set. For E. coli data set, <br /> there 35&#160;groups 80 features. All names            Sequential backward feature <br /> sources shown Table&#160;1. Only Bit string double        selection method <br /> screening scheme presented.                                 SVM well-established tool data analysis which <br />     The remaining features detailed                 shown useful various fields, as <br /> Appendix.                                                      text summarization,27 intrusion detection,28 image <br />     Lin et&#160;al26 Chin4 proposed double screen-          coding.29 In study, utilized SVM software <br /> ing scheme. They used multiple ranking scores sort          developed Chang Lin, called LIBSVM.23 To <br /> essential proteins. The drawback protein          address data imbalance, propose modified <br /> does unique score. Thus, propose bit           sequential backward feature selection method. <br /> string implementation incorporate prop-              Since data nonessential, choosing only <br /> erties single score.                                    accuracy objective adopting conventional fea- <br />     An example bit string implementation             ture ranking schemes favored negative data. As more <br /> shown Tables&#160;2 3. Suppose proteins,           features excluded, overall accuracy <br /> W, X, Y, Z, ranked. In itera-          declined. Since number negative data &#173;elements <br />  <br />  <br /> 390	                                                                                       Evolutionary Bioinformatics 2013:9 <br />                                                                                              Essentiality prediction SVM <br />  <br />  <br /> Table 1. Protein features. <br />  <br /> ID    Property                                                           Type   Size   Sub-names      S. cere   E. coli <br /> 1     Amino acid occurrence           20 <br />                                                                               S      20     A&#8230;Y            &#8226;         &#8226; <br /> 2     Average amino acid PSSM20                                               S      20     A&#8230;Y            &#8226;         &#8226; <br /> 3     Average cysteine position20                                             S      1                     &#8226;         &#8226; <br /> 4     Average distance cysteines20                               S      1                     &#8226;         &#8226; <br /> 5     Average hydrophobic20                                                   S      1                     &#8226;         &#8226; <br /> 6     Average hydrophobicity cysteine20                                S      4      1&#8230;4            &#8226;         &#8226; <br /> 7     Cysteine count20                                                        S      1                     &#8226;         &#8226; <br /> 8     Cysteine location20                                                     S      5      1&#8230;5            &#8226;         &#8226; <br /> 9     Cysteine odd-even index20                                               S      1                     &#8226;         &#8226; <br /> 10    Protein length20                                                        S      1                     &#8226;         &#8226; <br /> 11    Cell cycle5                                                             P      1                     &#8226; <br /> 12    Cytoplasm5                                                              P      1                     &#8226; <br /> 13    Endoplasmic reticulum5                                                  P      1                     &#8226; <br /> 14    Metabolic process5                                                      P      1                     &#8226; <br /> 15    Mitochondrion5                                                          P      1                     &#8226; <br /> 16    Nucleus5                                                                P      1                     &#8226; <br /> 17    Other process5                                                          P      1                     &#8226; <br /> 18    Other localization5                                                     P      1                     &#8226; <br /> 19    Signal transduction5                                                    P      1                     &#8226; <br /> 20    Transport5                                                              P      1                     &#8226; <br /> 21    Transcription5                                                          P      1                     &#8226; <br /> 22    Betweenness centrality related interactions41                    T      1                     &#8226;         &#8226; <br /> 23    Betweenness centrality related metabolic interactions5               T      1                     &#8226; <br /> 24    Betweenness centrality related physical interactions5                T      1                     &#8226;         &#8226; <br /> 25    Betweenness centrality transcriptional regulation interactions5         T      1                     &#8226; <br /> 26    Bit string double screening scheme [this paper]                      T      1                     &#8226;         &#8226; <br /> 27    Bottleneck8,41                                                          T      1                     &#8226;         &#8226; <br /> 28    Clique level7                                                           T      1                     &#8226;         &#8226; <br /> 29    Closeness centrality42                                                  T      1                     &#8226;         &#8226; <br /> 30    Clustering coefficient7                                                 T      1                     &#8226;         &#8226; <br /> 31    Degree related interactions43                                    T      1                     &#8226;         &#8226; <br /> 32    Degree related physical interactions5                                T      1                     &#8226;         &#8226; <br /> 33    Density maximum neighborhood component4                              T      1                     &#8226;         &#8226; <br /> 34    Edge percolated component9                                              T      1                     &#8226;         &#8226; <br /> 35    Indegree related metabolic interaction5                              T      1                     &#8226; <br /> 36    Indegree related transcriptional regulation5                         T      1                     &#8226; <br /> 37    Maximum neighborhood component4                                         T      1                     &#8226;         &#8226; <br /> 38    Neighbors&#8217; intra-degree7                                                T      1                     &#8226;         &#8226; <br /> 39    Outdegree related metabolic interaction5                             T      1                     &#8226; <br /> 40    Outdegree related transcriptional regulation interaction5            T      1                     &#8226; <br /> 41    Betweenness centrality related integrated functional interaction22   T      1                               &#8226; <br /> 42    Betweenness centrality related integrated PI GC network22        T      1                               &#8226; <br /> 43    Degree related integrated functional interaction22                   T      1                               &#8226; <br /> 44    Degree related integrated PI GC network22                        T      1                               &#8226; <br /> 45    Common function degree7                                                 O      1                     &#8226; <br /> 46    Essential index7                                                        O      1                     &#8226; <br /> 47    Identicalness5                                                          O      1                     &#8226; <br /> 48    Open reading frame length7                                              O      1                     &#8226;         &#8226; <br /> 49    Phyletic retention21                                                    O      1                     &#8226;         &#8226; <br /> 50    Number paralagous genes21                                            O      1                               &#8226; <br /> 51    <span id='am-43' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Codon" datatype="xsd:string"></span>Codon</span> Adaptation Index (CAI)21,44                                       O      1                               &#8226; <br /> 52    Codon <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Bias</span> Index (CBI)21,44                                             O      1                               &#8226; <br />                                                                                                                 (Continued) <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                      391 <br />  Hor et&#160;al <br />  <br />  <br /> Table 1. (Continued) <br />  <br /> ID     Property                                                                          Type      Size     Sub-names          S. cere       E. coli <br /> 53     Frequency optimal codons         21,44 <br />                                                                                               O         1                                         &#8226; <br /> 54     Aromaticity score21,44                                                                 O         1                                         &#8226; <br /> 55     Leading strand circular chromosome21                                            O         1                                         &#8226; <br />        Total                                                                                            100                         90            80 <br /> Notes: S. cere E. coli mean Saccharomyces cerevisiae Escherichia coli datasets, respectively. For topological features, particularly <br /> mentioned, related physical interactions. Due coverage availability issue, adopt different features S. cere E. coli datasets. <br /> For example, interactions E. coli data set contain integrated functional, PI, GC network information S. cere include metabolic, <br /> transcriptional regulation PI network information. <br /> Abbreviations: GC, genomic context; PI, physical interactions. <br />  <br />  <br />  <br /> was higher positive factors, true-                              fewer features compete with <br /> &#173;positive rate decreased true-nega-                           features. That is, <br />  tive rate. Thus, features selected most <br />  positive samples correctly classified dete-                       C(S)&#160;=&#160; wP * P(S)&#160;+&#160;wR * R(S)&#160;+&#160;wF * F (S) + wM <br />  riorating overall accuracy much. In sense,                                 * M(S) &#215; (|S| - t) * u(|S| - t) * e, <br />  using accuracy guide feature <br />  selection, used composite score C objec-                            S denotes selected feature subset. |S| t <br />  tive &#173;function. The composite score represented                          denote size S goal number features <br />  terms &#173;precision (P), recall (R), F-measure (F),                         specified user, respectively. The unit step func- <br />  MCC (M) given C =&#160;wP *&#160;P&#160;+&#160;wR&#160;*&#160;R +&#160;wF                               tion u(|S| &#8211; t)&#160;=&#160;0 |S| &#8211; t # 0, u(|S| &#8211; t)&#160;=&#160;1. <br />  * F + wM * M. The adjustable weights, wP, wR, wF,                          Finally, threshold &#65533;? adopted make ensure <br />  wM, used, leading compromise                              improvement feature changes not <br />  associated performance measures. An additional pun-                             random process. The value &#65533;? estimated by <br />  ishment imposed C allow scores associated                             comparing average score difference feature <br />                                                                                  subsets sizes p p&#160; +&#160; 1&#160; preliminary run <br />                                                                                  given different values p. The value e <br />                                                                                  denotes penalty score additional fea- <br />      Procedure: Bit String Implementation DSS <br />                                                                                  ture selected. The score specified the <br />      input: P: proteins ranking, |P|&#160;=&#160;n <br />                                                                                  user slightly larger &#65533;? encour- <br />      &#8195;&#8195;&#8194;&#8201;&#8201;&#8201;A: ranking methods, |A|&#160;=&#160;m&#160;=&#160;2 <br />                                  n                                               age feature subsets smaller sizes. The procedure <br />      &#8195;&#8195;&#8194;&#8201;M: bit matrix n &#215; 2 , initialized 0 <br />                                                                                  feature selection described Procedure back- <br />      output: Protein ranking scores R <br />                                                                                  ward feature selection. <br />      begin <br />      &#8195; i&#160;=&#160;1 n R[i]&#160;=&#160;0; <br />      &#8195; i&#160;=&#160;1 &#63728;n/2&#63739;                                                     Experimental procedure results <br />      &#8195;&#8195; T1&#160;=&#160;top 2i proteins ranked A(1) P;                                For comparison purposes, used feature selec- <br />      &#8195;&#8195; T2&#160;=&#160;top proteins ranked A(2) T1;                                tion methods: mRMR CMIM. In S. cerevisiae <br />      &#8195;&#8195; foreach x &#8712; T2 M [x, i]&#160;=&#160;1; <br />      &#8195; end <br />                                                                                  Table 2. Ranking different methods, smaller <br />      &#8195; i&#160;=&#160;1 n                                                         numbers indicate higher ranks. <br />      &#8195;&#8195; foreach j&#160;=&#160;1 &#63728;n/2&#63739; R[i]&#160;=&#160;R[i] <br />          +&#160;M [i, j];                                                             Protein                     Ranking method <br />      &#8195; end                                                                                                        A (DMNC)                    B (MNC) <br />      &#8195; T&#160;=&#160;protein orders ranked A(m) ;                                       W                                1                           4 <br />      &#8195; foreach i&#160;=&#160;1 n R[i]&#160;=&#160;R[i]&#160;+&#160;(n &#8211; T [i])                           X                                2                           2 <br />      end                                                                         Y                                3                           1 <br />                                                                                  Z                                4                           3 <br />  <br />  <br /> 392	                                                                                                                  Evolutionary Bioinformatics 2013:9 <br />                                                                                                   Essentiality prediction SVM <br />  <br />  <br />                                                           Table 3. Bit strings double screening method. <br />    Procedure: Backward Feature Selection <br />                                                           Protein          th iteration             Sum bit           n-r              Sum <br />    input : e: penalty score                                                                      string <br />                                                                            1st          2nd <br />    	       k: number folds cross-validation <br />    	       r: maximal number rounds retry           W                0            0             0                    0                0 <br />    	 S0: set available features,             X                1            1             2                    2                4 <br />                                                           Y                0            1             1                    3                4 <br />            |S0|&#160;=&#160;n                                       Z                0            0             0                    1                1 <br />    	       t: goal number selected features <br />    	 wP, wR, wF wM: weights perfor- <br />            mance measures                                 used Gustafson&#8217;s feature set benchmark. These <br />    	       &#65533;?: minimal improvement proceed              feature sets considerably effective various <br />    output: Selected feature subset S                      performance measures. <br />    data: 50% available data elements <br />    begin                                                  Stage 2: Tune SVM parameters <br />    &#8195; S&#160;=&#160;S0;                                              best performance <br />    &#8195; t&#160;$&#160;n stop m&#160;=&#160;n &#8211; t;                   For feature sets, ran SVM <br />    &#8195; p&#160;=&#160;1 m                                    software using feature sets Hwang Gustafson <br />    	       improved&#160;=&#160;False ;                             tuned SVM parameters achieve highest <br />    	       i&#160;=&#160;1 r                              average performances. <br />    	&#8195; Conduct k-fold CV calculate C (S); <br />    	&#8195; foreach s &#8712; S                                    Stage 3: Adopt best performances <br />    	&#8195;&#8195; Conduct k-fold CV calculate C                  reference performances <br />                  (S &#8211; {s});                               After determining best SVM parameters the <br />    	&#8195; end                                                 feature sets Hwang Gustafson, recorded the <br />    	&#8195; q&#160;=&#160;argmaxs &#8712; S {C (S &#8211; {s}) &#8211; C (S)};              SVM parameters results. To compare results <br />    	&#8195; C (S - {q}) &#8211; C (S) $ &#65533;?                     models, obtained using our <br />    	&#8195;&#8195; S&#160;=&#160;S - {q};                                       methods, mRMR CMIM, used SVM <br />    	&#8195;&#8195; improved&#160;=&#160;True;                                   software adjust cost parameters SVM in <br />    	&#8195;&#8195; break;                                             order achieve similar levels precision. <br />    	&#8195; end <br />      &#8195;&#8195; end                                               Stage 4: Perform feature selection <br />      &#8195;&#8195; improved stop output S            We randomly chose 50% available data. Next, the <br />    &#8195; end                                                  backward feature selection procedure applied to <br />    end                                                    selected data. In beginning feature selec- <br />                                                           tion procedure, imposed penalty score cal- <br />                                                           culation. Hence, procedure attempts achieve the <br /> data set, results compared      highest score. In subsequent runs, added penal- <br /> Acencio Lemke5 Hwang et&#160;al.7 For E. coli      ties feature sizes score &#173;calculation. &#173;Subsets <br /> data set, compared results      smaller feature size slightly &#173;inferior in <br /> Gustafson et&#160;al.21 <br />  <br /> Experimental procedure                                              Determining <br />                                                                benchmark feature sets <br />                                                                                                  Tuning SVM <br />                                                                                             parameters best <br />                                                                                                                           Adopting best <br />                                                                                                                          performances as <br />  <br /> The overall procedure experiments illus- <br />                                                                                                 performances         reference performances <br />  <br />  <br /> trated Figure&#160;1 described follows.                                                  Performing <br />                                                                      Performing               conventional         Performing feature <br />                                                                   significance tests            bootstraping               selections <br /> Stage 1: Determine benchmark feature set                                                      cross-validations <br />  <br />  <br /> For S. cerevisiae data set, used Hwang&#8217;s fea-      Figure 1. Flowchart construction SVM models performance <br /> ture set benchmark. For E. coli data set,   comparison. <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                           393 <br />  Hor et&#160;al <br />  <br />  <br /> performance selected. To compare results             o&#173; pportunities exploit available feature combina- <br /> with methods, used mRMR             tions achieve best performance. In initial <br /> and CMIM feature ranking methods chose subsets             run, obtained feature subset 18&#160;in size. <br /> as Procedure backward feature selection.                        For subsequent runs, value t was <br />                                                                decreased starting 17 (=&#160; 18 - 1) perfor- <br /> Stage 5: Perform 10-fold bootstrap                         mances significantly worse Hwang <br /> cross-validations                                              et&#160; al.7 In order obtain feature subsets reduced <br /> The data prepared balanced imbal-             sizes, parameters set: k&#160;=&#160;2, wP&#160;=&#160;1.03, wR&#160;=&#160;1, <br /> anced manners. For data set, randomly                  wF&#160;=&#160;1, wM&#160;=&#160;1, &#65533;?&#160;=&#160;0.005, e&#160;=&#160;0.01, r&#160;=&#160;5. The rea- <br /> &#173;partitioned data 10 disjoint groups used         son setting wP&#160;=&#160;1.03 prevent true posi- <br />  feature subsets selected previous stage         tive rate decreasing much. In addition, e . &#65533;? <br />  calculate various performance measures. The data              allow procedure proceeded fewer <br />  prepared balanced imbalanced                 features. The settings used encourage <br />  &#173;manners. The 10-fold cross validation repeated           selection reduced feature subsets. <br />   10 times average performance measures                   For setting t, executed procedure <br />   computed. Next, bootstrap sampling procedure           10 times different k-fold partitions obtained <br />   conducted 200 bootstrap samples produced,           10 feature subsets size. Since 10 <br />   &#173;including balanced imbalanced &#173;samples.            resultant feature subsets slightly different, we <br />    Each bootstrap sample partitioned              performed 5-fold cross-validation these <br />    10-fold cross validations performance measures          feature subsets compared performance <br />    &#173;calculated. Note models examined        scores. The highest score finally <br />    sets data partitions conventional        preserved feature subset. <br />    bootstraping cross-validations.                             In addition methods Hwang et&#160; al or <br />                                                                Gustafson et&#160;al,21 used mRMR13 CMIM14 <br /> Stage 6: Perform significance tests                            feature selection methods comparison. Using <br /> Once bootstrap cross validations carried out,             mRMR example, data used feature <br /> the significance tests adopted accordingly. In            selection procedure input mRMR pro- <br /> addition average values AUC, precision,              gram, produced ranking score fea- <br /> recall, F-measure, MCC, conducted statisti-           ture. The feature score removed <br /> cal significance test performance measures.          subsequent 5-fold cross-validation with <br /> Additionally, calculated ROC curves per-            preserved features performed calculate the <br /> centage values imbalanced experiments.                     composite score C (S). That is, ith iteration, <br />                                                                features lowest ranking scores were <br /> Backward feature selection                                     removed C (S) calculated. The removal and <br /> and mRMR/CMIM feature ranking                                  cross-validation procedure repeated fea- <br /> We used 50% available data elements fea-                ture preserved. The entire process (including ran- <br /> ture selection. Taking S. cerevisiae data set        dom choice 50% data feature removal) was <br /> example, (3840&#160;+&#160;975)&#160;&#215;&#160;50% observations             executed 10 times feature subset same <br /> randomly chosen Backward feature selec-                size highest score recorded. <br /> tion procedure. During procedure, per-                 Table&#160;4&#160;shows selected feature subsets dif- <br /> formance scores calculated means k-fold             ferent sizes S. cerevisiae data. The second column <br /> cross-&#173;validations. In run, parameters           table lists selected features. Each Ni the <br /> were set follows: k&#160; =&#160; 2, wP&#160; =&#160; 1, wR&#160; =&#160; 1, wF&#160; =&#160; 1,    row represents feature subset size i, 4 # <br /> wM&#160;=&#160;1, &#65533;?&#160;=&#160;0.005, e&#160;=&#160;0, t&#160;=&#160;0, r&#160;=&#160;5. Since          # 18, using backward feature <br /> associated weights equal, procedure sought            selection procedure. For feature subset Ni, a <br /> the best compromise performance <span id='am-58' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-59' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mea</span>-                 bullet (&#8226;) mark column used <br /> sures. t&#160; =&#160; e&#160; =&#160; 0, meaning goal number           indicate feature included. The most <br /> selected features imposed, giving procedure            competent feature subsets selected CMIM and <br />  <br /> 394	                                                                                       Evolutionary Bioinformatics 2013:9 <br />                                        Table 4. Selected features S. cerevisiae data set. <br />  <br />                                            Feature                             N5    N6    N7   N8   N9   N10   N11   N12   N13   N14   N15   N16   N17   N18   m31   C32   TOT <br />                                       1    PR (phyletic retention)             &#8226;     &#8226;     &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     16 <br />                                       2    EI (essentiality index)             &#8226;     &#8226;     &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     16 <br />                                       3    Cytoplasm                           &#8226;     &#8226;     &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;           &#8226;     15 <br />                                       4    Nucleus                             &#8226;     &#8226;     &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;           &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     15 <br />                                       5    Occurrence A.A. I                            &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;           &#8226;     13 <br />                                       6    Bit string DSS                         &#8226;     &#8226;    &#8226;    &#8226;    &#8226;     &#8226;     &#8226;           &#8226;     &#8226;     &#8226;     &#8226;     &#8226;                 12 <br />                                       7    Occurrence A.A. W                                 &#8226;    &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;           &#8226;     12 <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	 <br />                                       8    Endoplasmic reticulum                                     &#8226;    &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;           11 <br />                                       9    Other process                                                                          &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     7 <br />                                       10   Occurrence A.A. S                                                       &#8226;           &#8226;           &#8226;     &#8226;     &#8226;     &#8226;     &#8226;     7 <br />                                       11   Occurrence A.A. G                                                 &#8226;           &#8226;           &#8226;     &#8226;     &#8226;     &#8226;                 6 <br />                                       12   KLV (clique level)                                                         &#8226;     &#8226;     &#8226;           &#8226;                 &#8226;     &#8226;     6 <br />                                       13   Cell cycle                                                                 &#8226;           &#8226;           &#8226;                 &#8226;     &#8226;     5 <br />                                       14   Average hydrophobic                                                              &#8226;           &#8226;           &#8226;     &#8226;           &#8226;     5 <br />                                       15   Average PSSM A.A. R                                         &#8226;     &#8226;                       &#8226;           &#8226;     &#8226;                 5 <br />                                       16   B.C. related PI                                                               &#8226;                       &#8226;     &#8226;     &#8226;           4 <br />                                       17   Occurrence A.A. E                                                             &#8226;                 &#8226;                 &#8226;     &#8226;     4 <br />                                       18   Average PSSM A.A. P                                         &#8226;     &#8226;                       &#8226;                             &#8226;     4 <br />                                       19   ID related T.R.                                                                                       &#8226;     &#8226;     &#8226;           3 <br />                                       20   B.C. T.R. interactions                                                                       &#8226;                 &#8226;     &#8226;           3 <br />                                       21   Other localization                                                                                       &#8226;     &#8226;           &#8226;     3 <br />                                       22   DMNC                                &#8226;     &#8226;                                                                                &#8226;     3 <br />                                       23   Average HYD C-2                                                     &#8226;           &#8226;           &#8226;                             3 <br />                                       24   Signal transduction                                                                                      &#8226;     &#8226;                 2 <br />                                       25   Edge percolated component                                                                                            &#8226;     &#8226;     2 <br />                                       26   Occurrence A.A. P                                                                                                 &#8226;     &#8226;     2 <br />                                       27   Occurrence A.A. T                                 &#8226;    &#8226;                                                                      2 <br />                                       28   Occurrence A.A. Y                                                                                                 &#8226;     &#8226;     2 <br />                                       29   Average PSSM A.A. Q                                                           &#8226;                                         &#8226;     2 <br />                                       30   Average PSSM A.A. E                                                                 &#8226;           &#8226;                             2 <br />                                       31   CLC (clustering coefficient)                                                                                         &#8226;     &#8226;     2 <br />                                       32   FunK (common function degree)                                                                                        &#8226;     &#8226;     2 <br />                                       33   OD related T.R. interaction                                                                                       &#8226;           1 <br />                                       34   OD related M.I.                                                                                                   &#8226;           1 <br />                                       35   ID related M.I.                                                                                                   &#8226;           1 <br />                                       36   B.C. related M.I.                                                                                                 &#8226;           1 <br />                                       37   Degree related PI                                                                                                       &#8226;     1 <br />                                       38   Metabolic process                                                                                                    &#8226;           1 <br />                                       39   Bottleneck                                                                                                           &#8226;           1 <br />                                       40   MNC                                                                                                                  &#8226;           1 <br />                                       41   Occurrence A.A. A                                                                                                       &#8226;     1 <br />                                       42   Occurrence A.A. C                                                                                                 &#8226;           1 <br />                                       43   Occurrence A.A. D                                                                                                       &#8226;     1 <br />  <br />  <br />  <br />  <br /> 395 <br />                                                                                                                                                                                     Essentiality prediction SVM <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       (Continued) <br />      Hor et&#160;al <br />  <br />  <br />                                                                                                                                                                                                                                                           mRMR denoted C32 m31, means <br />  <br />  <br />  <br />  <br />                                                              Abbreviations: DSS, double screening scheme; A.A., amino acid; B.C., betweenness centrality; T.R., transcriptional regulation; HYD, hydrophobicity; PI, physical interaction; A &#8230; Y, amino <br />                        TOT <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           32 31 features selected, respectively. <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                                                                                                                                                                                                                                           It observed total 60 features been <br />                        C32 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           selected, represent prominent proteins used <br />                                   &#8226; <br />  <br />                                       &#8226; <br />  <br />  <br />  <br />  <br />                                                      &#8226; <br />                                                      &#8226; <br />                                                      &#8226; <br />                                                      &#8226; <br />                                                      &#8226; <br />                                                      &#8226; <br />                                                                                                                                                                                                                                                           identify essential proteins. <br />                        m31 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                               After feature subsets selected, conduct <br />                                  &#8226; <br />  <br />                                      &#8226; <br />  <br />                                          &#8226; <br />  <br />                                              &#8226; <br />  <br />                                                  &#8226; <br />  <br />  <br />  <br />  <br />                                                         &#8226; <br />                                                                                                                                                                                                                                                           performance comparison cope the <br />                                                                                                                                                                                                                                                           randomness, used Hwang&#8217;s method perform <br />                        N18 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           10 10-fold cross-validations. Here, true positive <br />                                                                                                                                                                                                                                                           rates false positive rates input diferent <br />                        N17 <br />  <br />  <br />  <br />  <br />                                                              acid abbreviation; M.I., metabolic interaction; OD, outdegree; ID, indegree; m31, mRMR31; C32, CMIM32; FunK, Common function degree; TOT, total. <br />                                                                                                                                                                                                                                                           software program calculate ROC curves AUC <br />                                                                                                                                                                                                                                                           values. In study, software package used <br />                        N16 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ROCR, developed Tobias et&#160; al.30,31 <br />                                                                                                                                                                                                                                                           Thus, reported performance measures, includ- <br />                        N15 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ing AUC, F1, MCC, precision, recall values and <br />                                                  &#8226; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ROC curves, averaged 10 10-fold cross- <br />                                                                                                                                                                                                                                                           validations. <br />                        N14 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                               For S. cerevisiae data set, predictor with <br />                                              &#8226; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           Hwang&#8217;s 10 features served benchmark because <br />                        N13 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           yielded distinguished results terms feature size <br />                                                                                                                                                                                                                                                           performance. Additionally, mRMR CMIM <br />                        N12 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           adopted comparison. <br />                                                                                                                                                                                                                                                               We appled procedure E. coli data <br />                        N11 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           set. The selected features shown Table&#160;5, with <br />                                                                                                                                                                                                                                                           total 43 features selected. In table, feature <br />                        N10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           subsets selected CMIM mRMR denoted <br />                                                                                                                                                                                                                                                           C9 m13, meaning 9 13 features were <br />                        N9 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           selected, respectively. <br />                        N8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           Bootstrap cross validations <br />                                                                                                                                                                                                                                                           During bootstrapping stage, bootstrap <br />                        N7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           sample, identical 10-fold partition employed <br />                                              &#8226; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           feature subsets carry cross-validations <br />                        N6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           compute various average performance measures. <br />                                                                                                                                                                                                                                                           The procedure repeated 200 distinct boot- <br />                        N5 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           strap samples. In order perform parametric signifi- <br />                                                                                                                                                                                                                                                           cance tests, evaluated distribution of <br />                                                                                                                                                                                                                                                           resultant performance measures normal and <br />                                                                                                                                                                                                                                                           variances obtained different feature sub- <br />                                  Average HYD C-1 <br />  <br />  <br />  <br />  <br />                                  Average PSSM A.A. W <br />  <br />  <br />                                  CC (closeness centrality) <br />                                  Average PSSM A.A. D <br />                                  Cysteine odd-even index <br />  <br />  <br />  <br />  <br />                                  Average PSSM A.A. S <br />  <br />                                  Average PSSM A.A. Y <br />                                  Average PSSM A.A. A <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           sets similar. Consequently, 200 results each <br />                                  Occurrence A.A. M <br />                                  Occurrence A.A. H <br />                                  Occurrence A.A. K <br />  <br />  <br />  <br />  <br />                                  ORFL (ORF length) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           performance measure feature subset were <br />                                  Cysteine location-1 <br />                                  Average C position <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           subjected Kolmogorov-Smirno test.31 This test <br /> Table 4. (Continued) <br />  <br />  <br />  <br />  <br />                                  Cysteine count <br />                                  Protein length <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           examines null hypothesis systematic differ- <br />                                                                                                                                                                                                                                                           ence exists standard normal distribution <br />                                  BC (B.C.) <br />                        Feature <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           underlying distribution alterna- <br />                                                                                                                                                                                                                                                           tive asserts systematic difference. The <br />                                                                                                                                                                                                                                                           threshold set 0.05. If P-value than <br />                                                                                                                                                                                                                                                           0.05, rejected null hypothesis. For CMIM and <br />                                  44 <br />                                  45 <br />                                  46 <br />                                  47 <br />                                  48 <br />                                  49 <br />                                  50 <br />                                  51 <br />                                  52 <br />                                  53 <br />                                  54 <br />                                  55 <br />                                  56 <br />                                  57 <br />                                  58 <br />                                  59 <br />                                  60 <br />  <br />  <br />  <br />  <br />     396	                                                                                                                                                                                                                                                                               Evolutionary Bioinformatics 2013:9 <br />                                                                                                             Essentiality prediction SVM <br />  <br />  <br /> Table 5. Selected features E. coli data set. <br />  <br />       Feature                                     N4     N5    N6     N7     N8     N9     N10     N11     N12      N13     C9     m13      TOT <br /> 1     PR (phyletic retention)                     &#8226;      &#8226;     &#8226;      &#8226;      &#8226;      &#8226;      &#8226;       &#8226;       &#8226;        &#8226;       &#8226;      &#8226;        12 <br /> 2     Open reading frame length                          &#8226;            &#8226;      &#8226;      &#8226;      &#8226;       &#8226;       &#8226;        &#8226;                       8 <br /> 3     Average PSSM A.A. C                      &#8226;      &#8226;     &#8226;      &#8226;      &#8226;                                      &#8226;                       6 <br /> 4     Degree related F.I.                      &#8226;      &#8226;     &#8226;      &#8226;      &#8226;                                      &#8226;                       6 <br /> 5     Degree related A.I.                                                        &#8226;      &#8226;       &#8226;       &#8226;                &#8226;               5 <br /> 6     Degree related PI                                                          &#8226;      &#8226;       &#8226;       &#8226;                       &#8226;        5 <br /> 7     Average PSSM A.A. A                                          &#8226;      &#8226;                                              &#8226;      &#8226;        4 <br /> 8     Average PSSM A.A. R                                                        &#8226;      &#8226;       &#8226;       &#8226;                                4 <br /> 9     Average hydrophobic                                                           &#8226;      &#8226;       &#8226;       &#8226;                                4 <br /> 10    Bit string DSS PI                                                      &#8226;      &#8226;       &#8226;       &#8226;                                4 <br /> 11    Paralog count                                            &#8226;                                           &#8226;        &#8226;              &#8226;        4 <br /> 12    Occurrence A.A. M                                                   &#8226;      &#8226;                      &#8226;                                3 <br /> 13    Occurrence A.A. W                                                                 &#8226;                        &#8226;       &#8226;               3 <br /> 14    Occurrence A.A. E                               &#8226;     &#8226;                                                                            2 <br /> 15    Occurrence A.A. F                                                                         &#8226;                        &#8226;               2 <br /> 16    Occurrence A.A. G                                                          &#8226;              &#8226;                                        2 <br /> 17    Occurrence A.A. I                                                                                                  &#8226;      &#8226;        2 <br /> 18    Average PSSM A.A. Y                                                 &#8226;                     &#8226;                                        2 <br /> 19    Cysteine location-4                                                                                                   &#8226;      &#8226;        2 <br /> 20    KLV (clique level) PI                                       &#8226;                                             &#8226;                       2 <br /> 21    Degree related PI GC                                                                  &#8226;       &#8226;                                2 <br /> 22    Strand bias                                                                                          &#8226;                       &#8226;        2 <br /> 23    Occurrence A.A. A                                                                 &#8226;                                                1 <br /> 24    Occurrence A.A. C                                                                                                  &#8226;               1 <br /> 25    Occurrence A.A. H                                                                                                  &#8226;               1 <br /> 26    Occurrence A.A. P                                     &#8226;                                                                            1 <br /> 27    Occurrence A.A. S                                                                 &#8226;                                                1 <br /> 28    Average PSSM A.A. N                                                                                        &#8226;                       1 <br /> 29    Average PSSM A.A. G                                                                                        &#8226;                       1 <br /> 30    Average PSSM A.A. K                                          &#8226;                                                                     1 <br /> 31    Average PSSM A.A. F                                                 &#8226;                                                              1 <br /> 32    Average PSSM A.A. T                                                                                        &#8226;                       1 <br /> 33    Average PSSM A.A. V                                                                                        &#8226;                       1 <br /> 34    Average distance Cs                                                                     &#8226;                                1 <br /> 35    Average HYD C-2                                                                                        &#8226;                       1 <br /> 36    Cysteine location-1                                                                                                          &#8226;        1 <br /> 37    Cysteine location-5                                                                                                          &#8226;        1 <br /> 38    Cysteine odd-even index                                                                                                      &#8226;        1 <br /> 39    Protein length                              &#8226;                                                                                         1 <br /> 40    Bottleneck PI                                                                                                            &#8226;        1 <br /> 41    CC (closeness centrality) PI                                                                                             &#8226;        1 <br /> 42    MNC PI                                                                                                    &#8226;                       1 <br /> 43    B.C. related F.I.                                                                                                     &#8226;        1 <br /> Abbreviations: C9, CMIM09; m13, mRMR13; TOT, total; DSS, double screening scheme; F.I., integrated functional interaction; A.I., interactions. <br /> PI, physical interaction; HYD, hydrophobicity; A.A., amino acid; A &#8230; Y, amino acid abbreviation. <br />  <br />  <br /> mRMR, prominent values shown.                             observed AUC N8-imbalanced experiment. <br /> Figures&#160;2 3 illustrate results S. cerevisiae                    Therefore, likely significant <br /> and E. coli data sets, respectively, test                      difference normal distribution the <br /> values recorded according feature subsets,                      distribution performance measure each <br /> performance measures, experiment types. For                         feature subset. For E. coli data set, perfor- <br /> S. cerevisiae data set, lowest P-value, 0.186,                      mance measures normal exception of <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                         397 <br />  Hor et&#160;al <br />  <br />  <br />                                                                                                  AUC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                              0.186 <br />                           0.0 <br />  <br />                                                                                                Precision <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                                                                                                                                          Imbalanced <br />                           0.2 <br />                                                                                            0.239                                         Balanced <br />                           0.0 <br />  <br />                                                                                                 Recall <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                               0.229 <br />                           0.0 <br />  <br />                                                                                                    F1 <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                     0.310 <br />                           0.0 <br />  <br />                                                                                                  MCC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                                                      0.373 <br />                           0.0 <br />                                   CMIM32 <br />  <br />                                            mRMR31 <br />  <br />                                                     Hwang <br />  <br />                                                             Acencio <br />  <br />  <br />  <br />  <br />                                                                                                      N10 <br />  <br />                                                                                                            N11 <br />  <br />                                                                                                                  N12 <br />  <br />                                                                                                                        N13 <br />  <br />                                                                                                                              N14 <br />  <br />                                                                                                                                    N15 <br />  <br />                                                                                                                                          N16 <br />  <br />                                                                                                                                                N17 <br />  <br />                                                                                                                                                        N18 <br />  <br />                                                                                                                                                              N90 <br />                                                                       N4 <br />  <br />                                                                            N5 <br />  <br />                                                                                 N6 <br />  <br />                                                                                      N7 <br />  <br />                                                                                           N8 <br />  <br />                                                                                                 N9 <br />  <br />  <br />  <br />  <br /> Figure 2. The P-value normality test S. cerevisiae data set. <br />  <br />  <br /> the recall values associated N12-balanced (with                                                 Performance comparison <br /> P-value&#160;=&#160;0.035), comparing results                                                   significance tests <br /> not reliable. For performance measures associated                                                In section, compared experimental results <br /> with model, listed confidence inter-                                              associated feature selection <br /> val information odds ratios,32 shown                                                methods previous studies. For conciseness, we <br /> the Appendix.                                                                                        prominent results associated with <br />    For certain performance measure, vari-                                                mRMR CMIM. We observed feature sizes <br /> ances obtained various feature subsets quite                                                 identified methods relatively large. <br /> similar, used analysis variance33 (ANOVA)                                                   To compare feature subsets smaller sizes, their <br /> test examine differences existed                                                    comparison working principles detailed <br /> performance measures different feature subsets.                                                   Appendix. <br /> Here, variance obtained multiple <br /> experiments feature subset. Differences existed                                               S. cerevisiae <br /> according ANOVA. Next, measures                                                  Table&#160;6 lists average values performance <br /> were compared associated benchmark                                                     measures associated variety feature subsets, <br /> calculate performance deviations. The average devia-                                                 obtained 10 10-fold cross-validations <br /> tion corresponding type performance mea-                                                  imbalanced data. We adjusted SVM cost param- <br /> sure evaluated using 95% confidence interval                                                 eters order achieve similar levels precision. The <br /> covering 0 determine significance.                                                                rows results CMIM32 (32 &#173;features), <br />  <br /> 398	                                                                                                                                                 Evolutionary Bioinformatics 2013:9 <br />                                                                                                                              Essentiality prediction SVM <br />  <br />  <br />                                                                                         AUC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                                                                      0.344 <br />                           0.2 <br />                           0.0 <br />  <br />                                                                                     Precision <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4                                                                         Imbalanced <br />                           0.2                                                                         Balanced                   0.270 <br />                           0.0 <br />  <br />                                                                                         Recall <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2 <br />                           0.0                                                                                                0.035 <br />  <br />  <br />                                                                                          F1 <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                                      0.139 <br />                           0.0 <br />  <br />                                                                                         MCC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2 <br />                                                                                            0.141 <br />                           0.0 <br />                                   CMIM09 <br />  <br />  <br />                                            mRMR13 <br />  <br />  <br />                                                     Gustafson <br />  <br />  <br />  <br />  <br />                                                                                                                N10 <br />  <br />  <br />                                                                                                                      N11 <br />  <br />  <br />                                                                                                                            N12 <br />  <br />  <br />                                                                                                                                      N13 <br />  <br />  <br />                                                                                                                                            N80 <br />                                                                 N3 <br />  <br />  <br />                                                                          N4 <br />  <br />  <br />                                                                               N5 <br />  <br />  <br />                                                                                    N6 <br />  <br />  <br />                                                                                           N7 <br />  <br />  <br />                                                                                                  N8 <br />  <br />  <br />                                                                                                         N9 <br />  <br />  <br />  <br />  <br /> Figure 3. The P-value normality test S. cerevisiae data set. <br />  <br />  <br /> mRMR31 (31 features), Hwang&#8217;s (10 features),                                           significantly higher N8. For values the <br /> Acencio&#8217;s (23 features). Values following items                                      performance measure column, best <br /> are enclosed parentheses represent numbers                                      underlined. Values row results <br /> of features. Results produced method listed                                     set 90 features. <br /> in subsequent rows table. &#173;Significance tests                                      Based Table&#160; 6, CMIM32, mRMR31 and <br /> were carried bootstrap cross-validations                                      Hwang&#8217;s predictors outperformed Acencio&#8217;s all <br /> over 200 bootstrap &#173;samples. The symbols,                                      performance measures. For feature subsets, the <br /> which plus (+) minus (-), following                                         performance measures slightly higher than <br /> numerical value represents significantly higher                                         Hwang&#8217;s. For N8, perfor- <br /> lower benchmark results. For serve                                        mance difference Hwang&#8217;s AUC, the <br /> as benchmarks marked star (*) symbols                                           remaining measure values higher Hwang&#8217;s. <br /> clarity. For example, recall N6 significantly                                   When feature size exceeded 8, preci- <br /> higher Hwang, AUC sig-                                          sion &#173;values, improvement Hwang&#8217;s con- <br /> nificantly lower mRMR31 Hwang.                                           sistently significant cases. For comparison <br /> For feature subsets prefix &#8216;N&#8217;,                                      mRMR, method performed nearly as <br /> fourth symbols numerical values used                                         mRMR31 feature size 9 and <br /> indicate significance neighboring                                          13 &#173;exception AUC values. When fea- <br /> rows. For example, N7, AUC significantly                                       ture size ranged 14 18, perfor- <br /> higher N6 recall value                                       mance difference model mRMR31. <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                                    399 <br />  400	 <br />                                                                                                                                                                                                                                      Hor et&#160;al <br />  <br />  <br />  <br />  <br />                                      Table 6. Performance comparison imbalanced S. cerevisiae data set. <br />  <br />                                                           AUC                                Precision                           Recall                             F-measure                          MCC <br />                                      CMIM32               0.825	    *	 (+)	 (+)              0.744	 *		       (+)                0.369	 *		       (+)               0.493	 *		       (+)               0.450	 *		       (+) <br />                                      mRMR31               0.821	   (-)	 &#8201;&#8201;*	 (+)             0.738		      &#8201;*                     0.372		      *	 (+)                0.495		      *	 (+)                0.449		      &#8201;*	 (+) <br />                                      Hwang(10)            0.775	   (-)	 (-)	 &#8201;&#8201;&#8201;*            0.743	 (-)		      &#8201;*                0.343	 (-)	 (-)	 *                 0.469	 (-)	 (-)	 *                 0.432	 (-)	 (-)	 &#8201;* <br />                                      Acencio(23)          0.707	   (-)	 (-)	 (-)             0.675	 (-)	 (-)	 (-)                0.121	 (-)	 (-)	 (-)               0.204	 (-)	 (-)	 (-)               0.228	 (-)	 (-)	 (-) <br />                                      N4                   0.744	   (-)	 (-)	 (-)             0.782                               0.327	 (-)	 (-)	 (-)               0.461	 (-)	 (-)	 (-)               0.439	 (-)	 (-) <br />                                      N5                   0.727	   (-)	 (-)	 (-)             0.741	 (-)			        (-)            0.387	 (-)	 (-)		        (+)       0.509	 (-)	 (-)		         (+)      0.461	 (-)	 (-)		       (+) <br />                                      N6                   0.730	   (-)	 (-)	 (-)             0.752	 (-)                          0.395	 (-)	 (-)	 (+)               0.518	 (-)	 (-)                    0.472	 (-)	 (-) <br />                                      N7                   0.761	   (-)	 (-)		     (+)        0.767	 (-)                          0.386	 (-)	 (-)	 (+)               0.513	 (-)	 (-)	 (+)               0.473	 (-)	 (-) <br />                                      N8                   0.772	   (-)	 (-)                  0.755                               0.371	 (-)	 (-)		        (-)       0.498	 (-)	 (-)	 (+)	     (-)      0.457	 (-)	 (-)		       (-) <br />                                      N9                   0.782	   (-)	 (-)	 (+)             0.749                               0.382	 (-)		     (+)	    (+)       0.506	 (-)		     (+)               0.462	 (-)		     (+) <br />                                      N10                  0.781	   (-)	 (-)	 (+)             0.751                               0.399			         (+)               0.521			         (+)               0.474			         (+) <br />                                      N11                  0.786	   (-)	 (-)	 (+)             0.752                               0.402			         (+)               0.524			         (+)               0.476			         (+) <br />                                      N12                  0.798	   (-)	 (-)	 (+)             0.759                               0.409			         (+)               0.532			         (+)               0.485			         (+) <br />                                      N13                  0.789	   (-)	 (-)	 (+)             0.748                               0.433			         (+)	    (+)       0.549			         (+)               0.495			         (+) <br />                                      N14                  0.802	   (-)		     (+)             0.749                               0.397			         (+)	    (-)       0.519			         (+)	     (-)      0.471			         (+)	   (-) <br />                                      N15                  0.801	   (-)		     (+)             0.763                               0.406			         (+)               0.530			         (+)               0.485			         (+)	   (+) <br />                                      N16                  0.814	   (-)		     (+)	 (+)        0.762                               0.401			         (+)               0.525			         (+)               0.480			         (+) <br />                                      N17                  0.814	   (-)		     (+)             0.761                               0.407			         (+)               0.530			         (+)               0.484			         (+) <br />                                      N18                  0.811	   (-)		     (+)             0.751                               0.411			         (+)               0.531			         (+)               0.482			         (+) <br />                                      N90                  0.829	   (+)	 (+)	 (+)	 (+)        0.738		     (+)	 (+)                0.355		     (+)	 (+)	    (-)       0.479	 (+)	 (+)	 (+)	     (-)      0.438	 (+)	 (+)	 (+)	   (-) <br />                                                                                                                                                                                                                     7 <br />                                      Note: With polynomial kernel function, values precision, recall MCC reported 0.77, 0.23, 0.36, respectively, original paper Hwang et al. <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 <br />                                                                                     Essentiality prediction SVM <br />  <br />  <br /> The prominent predictor CMIM32. Except           higher ours. Table&#160;9, the <br /> for AUC values, results achieved similar levels    effective predictor mRMR13, shows no <br /> performance size features exceeded 14.        performance difference feature subsets <br /> Note number features CMIM32            balanced experiments. For ROC analysis, <br /> mRMR31 32 31, higher              addition CMIM09, mRMR13, Gustafson&#8217;s, <br /> than ours.                                                used N4, N8, N11 N80 draw ROC <br />     Table&#160; 7&#160; shows average performance mea-          curves. This observed performances of <br /> sures balanced experiments S. cerevisiae        insufficient, middle feature sets. <br /> data set, obtained 10 10-fold <br /> &#173;cross-validations. For feature subsets      ROC analysis <br />  size ranging 5 18, nearly performance   S. cerevisiae <br />  measures higher        Figure&#160; 4 illustrates average ROC curves and <br />  Hwang&#8217;s. This shows feature subsets        AUCs various feature subsets imbalanced <br />  sizes exceeding 5 good Hwang&#8217;s.       data experiments. Apart competent <br />  Additionally, 12 features achieved    predictor CMIM32, AUC N5 is <br />  significant improvement. Compared CMIM32            higher Acencio&#8217;s, intersection be <br />  mRMR31, results showed similar levels         observed 0.5 horizontal <span id='am-53' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-54' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>axis</span>. This indicates <br />  performance size features exceeded 15.       N5 better predictor allowed <br />  The results set 90 features shown   maximal false positive rate 0.5. In con- <br />  row, performance measures         trast, allowed false positive rate exceeded <br />  similar N14 N18.                        0.5, Acencio&#8217;s better N5. Comparing <br />     In Table&#160; 6, observe feature subsets      N9 Hwang&#8217;s method, AUC values were <br />  N5, N7, N9, N13, N15, N16&#160;showed significant         &#173;similar. For feature subsets sizes exceed- <br />  improvement performance smaller fea-       ing 8 (not shown figure), true positive <br />  ture sizes compared neighboring rows. In        rates higher close Hwang&#8217;s. <br />  Table&#160;7, significant subsets N5, N6, N9.     This supported significance tests in <br />  In addition, shown Tables&#160;6 7, models       Table&#160; 6 suggests feature subsets with <br />  performed equally CMIM32 mRMR31               sizes exceeding 8 achieved higher performance in <br />  feature size 16 17. We used N5, N9,       AUC Hwang&#8217;s predictor. <br />  N16 draw ROC curves.                                    Figure&#160;5 illustrates average ROC curves and <br />                                                            AUCs various feature subsets balanced <br /> E. coli                                                    data experiments. CMIM32 com- <br /> Tables&#160; 8 9&#160; shows average values          petent predictor. Additionally, N16 achieved <br /> performance measures associated variety          level AUC. For feature subsets of <br /> feature subsets, obtained 10 10-fold         sizes ranging 5 18 (not shown), true <br /> cross-validations imbalanced balanced experi-      positive rates higher close to <br /> ments, respectively. The rows results    Hwang&#8217;s level. Thus, N5, N6, &#8230;, N18 outperformed <br /> CMIM09 (9 features) Gustafson&#8217;s (29 features).         performed equally various combina- <br />    Table&#160; 8&#160; shows Gustafson&#8217;s predictors out-        tions true false positive rates balanced <br /> performed CMIM09&#160; performance measures             &#173;experiments. &#173;Similarly imbalanced data set, the <br /> in imbalanced experiments. For feature sub-             features, higher AUC values. &#173;However, <br /> sets, performance measures slightly higher         improvement AUC feature addition <br /> than CMIM09. When feature size exceeded                 significant imbalanced <br /> 6, improvement CMIM09 consis-                  &#173;experiments. It noted ROC <br /> tently significant. To compare Gustafson&#8217;s method            curve AUC Acencio&#8217;s predictor repro- <br /> with method, performed            duced &#173;experiments slightly <br /> Gustafson&#8217;s feature size 11. Note          different original values reported &#173;Acencio <br /> that number features Gustafson&#8217;s 29,           Lemke.5 <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                            401 <br />  402	 <br />                                                                                                                                                                                                                                Hor et&#160;al <br />  <br />  <br />  <br />  <br />                                      Table 7. Perforamnce comparison balanced S. cerevisiae data set. <br />  <br />                                                          AUC                               Precision                        Recall                            F-measure                         MCC <br />                                      CMIM32              0.842	 *		       (+)              0.772	 *                         0.766	 *		       (+)              0.769	 *		       (+)              0.540	 *		       (+) <br />                                      mRMR31              0.836		      *	 (+)               0.765		      *                   0.741		      *	 (+)               0.752		      *	 (+)               0.513		     *	 (+) <br />                                      Hwang(10)           0.822	 (-)	 (-)	 *                0.778			          *              0.720	 (-)	 (-)	 *                0.748	 (-)	 (-)	 *                0.516	 (-)	 (-)	 * <br />                                      Acencio(23)         0.768	 (-)	 (-)	 (-)              0.696	 (-)	 (-)	 (-)             0.734	 (-)	 (-)                   0.714	 (-)	 (-)	 (-)              0.414	 (-)	 (-)	 (-) <br />                                      N4                  0.811	 (-)	 (-)	 (-)	    (+)      0.777		     (-)		    (+)         0.716	 (-)	 (-)                   0.745	 (-)	 (-)		    (+)          0.512	 (-)	 (-)		    (+) <br />                                      N5                  0.824	 (-)	 (-)		        (+)      0.778		     (-)                  0.735	 (-)	 (-)		     (+)         0.756	 (-)	 (-)                   0.527	 (-)	 (-) <br />                                      N6                  0.827	 (-)	 (-)		        (+)      0.778		     (-)                  0.739	 (-)	 (-)                   0.758	 (-)	 (-)                   0.530	 (-)	 (-) <br />                                      N7                  0.831	 (-)	 (-)                   0.779                            0.733	 (-)	 (-)                   0.755	 (-)	 (-)                   0.526	 (-)	 (-) <br />                                      N8                  0.826	 (-)	 (-)                   0.786                            0.721	 (-)	 (-)                   0.752	 (-)	 (-)                   0.527	 (-)	 (-) <br />                                      N9                  0.833	 (-)	 (-)		        (+)      0.791                            0.735	 (-)	 (-)		     (+)         0.762	 (-)	 (-)                   0.541	 (-)	 (-) <br />                                      N10                 0.834	 (-)	 (-)                   0.789                            0.736	 (-)	 (-)                   0.761	 (-)	 (-)                   0.540	 (-)	 (-) <br />                                      N11                 0.831	 (-)	 (-)                   0.784                            0.737	 (-)	 (-)                   0.760	 (-)	 (-)                   0.535	 (-)	 (-) <br />                                      N12                 0.829	 (-)	 (-)                   0.779                            0.732                             0.755			         (+)              0.526			         (+) <br />                                      N13                 0.834	 (-)	 (-)                   0.788                            0.730	 (-)	 (-)                   0.758	 (-)	 (-)                   0.535	 (-)	 (-) <br />                                      N14                 0.836	 (-)	 (-)	 (+)              0.777                            0.743	 (-)	 (-)	 (+)              0.759	 (-)	 (-)	 (+)              0.530	 (-)	 (-) <br />                                      N15                 0.843	 (-)	 (-)	 (+)              0.784                            0.748	 (-)	 (-)	 (+)              0.766		     (-)	 (+)              0.542		     (-)	 (+) <br />                                      N16                 0.842			         (+)              0.777                            0.756			         (+)              0.767			         (+)              0.540			         (+) <br />                                      N17                 0.847			         (+)              0.778                            0.763			         (+)              0.770			         (+)              0.545			         (+) <br />                                      N18                 0.840	 (-)	 (-)	 (+)	    (-)      0.779		     (-)                  0.740	 (-)	 (-)	 (+)	 (-)         0.759	 (-)	 (-)                   0.531	 (-)	 (-) <br />                                      N90                 0.839	 (+)		     (+)              0.760                            0.753			         (+)              0.757			         (+)              0.516			         (+) <br />                                      Note: In original paper Hwang et al7 values precision, recall, F-measure MCC reported 0.763, 0.713, 0.737, 0.492, respectively, polynomial kernel <br />                                      function. <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 <br />                                                                                                                                                  Essentiality prediction SVM <br />  <br />  <br />                                                                                                                         E. coli <br />  <br />  <br />                                                                                      (-)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                              (+)	 (-) <br />                                                                                                                         For imbalanced data set, Figure&#160;6 illustrates aver- <br />                                                                                      (-) <br />                                                                                      (-) <br />  <br />                                                                                      (-) <br />  <br />  <br />  <br />                                                                                                  (-) <br />                                                                                                                         age ROC curves AUCs various feature subsets. <br />                                                                                      * <br />                                                                                      0.413	 (+)	 (+)	 <br />  <br />  <br />  <br />  <br />                                                                                      0.339	 (+)	 (+)	 <br />                                                                                      0.403	 (+)	 (+) <br />  <br />  <br />                                                                                      0.401	 (+)	 (+) <br />                                                                                      0.401	 (+)	 (+) <br />                                                                                      0.393	 (+)	 (+) <br />                                                                                      0.400	 (+)	 (+) <br />                                                                                      0.400	 (+)	 (+) <br />                                                                                                                         It shows curves similar 10% <br />                                                                                                  *	 <br />                                                                                      0.382	 *		 <br />  <br />  <br />                                                                                      0.391			 <br />                                                                                      0.407	 (+)		 <br />  <br />  <br />                                                                                      0.405	 (+)		 <br />                                                                                                                         horizontal range. This indicates little <br />                                                                                      0.392	 (+)                         difference allowable false positive rate was <br />                                                                                      0.360		 <br />                                                                          MCC <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. For horizontal range 10%, <br />                                                                                                                         N80 highest performer, Gustafson N11 <br />                                                                                                                         secondary, N4 worst. In contrast to <br />                                                                                                                         imbalanced data set, Figure&#160;7 corresponding <br />                                                                                      0.421	 (+)	 (+)	 (-)	 (+) <br />  <br />  <br />                                                                                      0.415	 (+)	 (+)	 (-)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                            (-) <br />                                                                                                                         balanced data set, N4 N8 best <br />                                                                                                       (-) <br />                                                                                                   *	 (-) <br />  <br />                                                                                                       (-) <br />  <br />  <br />                                                                                      0.400	 (+)	 (+)	 (-) <br />                                                                                      0.420	 (+)	 (+)	 * <br />  <br />  <br />  <br />  <br />                                                                                      0.352	 (+)	 (+)		                  performers. The remaining predictors showed few <br />                                                                                      0.414	 (+)	 (+) <br />  <br />  <br />                                                                                      0.417	 (+)	 (+) <br />                                                                                      0.418	 (+)	 (+) <br />                                                                                      0.403	 (+)	 (+) <br />                                                                                      0.416	 (+)	 (+) <br />                                                                                      0.413	 (+)	 (+) <br />  <br />                                                                                                                         differences. <br />                                                                                      0.394	 *		 <br />  <br />  <br />                                                                                      0.404	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                                                         Top percentage analysis <br />                                                                                      0.370		 <br />  <br />  <br />  <br />  <br />                                                                                                                         S. cerevisiae <br />                                                                          F1 <br />  <br />  <br />  <br />  <br />                                                                                                                         Table&#160; 10&#160; shows average percentage infor- <br />                                                                                                                         mation imbalanced data set. The &#952; prob- <br />                                                                                                                         ability defined ratio number truly <br />                                                                                           (-) <br />                                                                                           (+) <br />  <br />  <br />                                                                                                  (+) <br />  <br />  <br />                                                                                                        (-) <br />  <br />  <br />                                                                                                              (-) <br />  <br />  <br />  <br />  <br />                                                                                                                         predicted essential proteins top-ranked &#952; &#215; <br />                                                                                                       (-)	 <br />  <br />  <br />  <br />                                                                                      0.288	 (+)	 (+)	 (-)	 <br />                                                                                                       (-) <br />                                                                                                   *	 (-) <br />  <br />  <br />  <br />  <br />                                                                                      0.275	 (+)	 (+)	 (-) <br />                                                                                      0.290	 (+)	 (+)	 * <br />  <br />  <br />  <br />  <br />                                                                                                                         975 proteins, total number true essen- <br />                                                                                      0.295	 (+)	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                      0.278	 (+)	 (+)		 <br />  <br />  <br />                                                                                      0.237	 (+)	 (+)		 <br />                                                                                      0.287	 (+)	 (+) <br />  <br />  <br />                                                                                      0.293	 (+)	 (+) <br />                                                                                      0.294	 (+)	 (+) <br />  <br />                                                                                      0.292	 (+)	 (+) <br />                                                                                      0.287	 (+)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                                         tial proteins 975. The &#952; probability shows the <br />                                                                                      0.271	 *		 <br />  <br />  <br />                                                                                      0.280	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                                                         likelihood proteins essential user <br />                                                                                      0.250		 <br />  <br />  <br />  <br />  <br />                                                                                                                         decides choose specific number top-ranked <br />                                                                          Recall <br />  <br />  <br />  <br />  <br />                                                                                                                         candidates. It slightly different precision <br />                                                                                                                         top-ranked candidates (or denomina- <br />                                                                                                                         <span id='am-94' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span>) necessary classified essential. <br />                                                                                                            (-) <br />  <br />  <br />  <br />  <br />                                                                                      0.677	 (+)	 (+)	 (+)	 (-) <br />  <br />  <br />  <br />  <br />                                                                                                                         CMIM32, mRMR31 Hwang&#8217;s results again <br />       Table 8. Performance comparison imbalanced E. coli data set. <br />  <br />  <br />  <br />  <br />                                                                                                                         served benchmarks denoted star <br />                                                                                                        * <br />  <br />  <br />  <br />  <br />                                                                                      0.726				 <br />  <br />  <br />  <br />  <br />                                                                                                                         &#8216;*&#8217; symbols table. The minus symbol follow- <br />                                                                                                                         ing value represents value lower <br />                                                                                                  * <br />                                                                                      0.720			 <br />                                                                          Precision <br />  <br />  <br />  <br />  <br />                                                                                                                         benchmark results. <br />                                                                                      0.720	 * <br />                                                                                      0.713		 <br />  <br />  <br />  <br />  <br />                                                                                                                             Both mRMR31 Hwang&#8217;s predictor were <br />                                                                                      0.725 <br />                                                                                      0.737 <br />                                                                                      0.742 <br />                                                                                      0.735 <br />                                                                                      0.742 <br />  <br />                                                                                      0.724 <br />                                                                                      0.732 <br />                                                                                      0.725 <br />                                                                                      0.733 <br />  <br />  <br />  <br />  <br />                                                                                                                         extremely effective 10% range. This <br />                                                                                                                         indicates predictors quite preferable <br />                                                                                                                         total number true essential proteins was <br />                                                                                      (-) <br />                                                                                      (-) <br />  <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />  <br />  <br />                                                                                                              (+) <br />                                                                                       * <br />  <br />  <br />  <br />  <br />                                                                                                                         known allowable top-ranked candidates were <br />                                                                                      0.711	 (+)	 (+)	 <br />                                                                                      0.691	 (-)	 (-)	 <br />                                                                                      0.690	 (-)	 (-)	 <br />  <br />  <br />  <br />  <br />                                                                                      0.716	 (+)	 (+)	 <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. Most predictors outperformed <br />                                                                                                   *	 <br />                                                                                      0.701	 *		 <br />  <br />  <br />  <br />  <br />                                                                                      0.701			 <br />                                                                                      0.714			 <br />                                                                                      0.705			 <br />                                                                                      0.707			 <br />                                                                                      0.711			 <br />                                                                                      0.714			 <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. For CMIM32, predictors out- <br />                                                                                      0.712	 (+) <br />                                                                                      0.714	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                                         performed 30%. Thus, N14&#160;may bet- <br />                                                                                      0.715		 <br />  <br />  <br />  <br />  <br />                                                                                                                         ter choice relatively effective beyond <br />                                                                          AUC <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. Figure&#160; 8 depicts average percentage <br />                                                                                                                         curves. <br />                                                                                      Gustafson(29) <br />  <br />  <br />  <br />  <br />                                                                                                                         E. coli <br />                                                                                      mRMR13 <br />  <br />  <br />  <br />  <br />                                                                                                                         Table&#160;11&#160;shows average percentage informa- <br />                                                                                      CMIM09 <br />  <br />  <br />  <br />  <br />                                                                                                                         tion imbalanced data set. CMIM09, mRMR13, <br />                                                                                      N10 <br />  <br />                                                                                      N12 <br />                                                                                      N13 <br />                                                                                      N80 <br />                                                                                      N11 <br />                                                                                      N4 <br />                                                                                      N5 <br />                                                                                      N6 <br />                                                                                      N7 <br />                                                                                      N8 <br />                                                                                      N9 <br />  <br />  <br />  <br />  <br />                                                                                                                         Gustafson&#8217;s results serve &#173;benchmarks. <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                                                        403 <br />  Hor et&#160;al <br />  <br />  <br /> Table 9. Performance comparison balanced E. coli data set. <br />  <br />                                     AUC                            Precision          Recall                                      F1                              MCC <br /> CMIM09                              0.767&#8194; *&#8194;&#8194; (+)                 0.720&#8194; *           0.700&#8194; *&#8194;&#8194; (+)                              0.710&#8194; *&#8194;&#8194; (+)                  0.421&#8194; * <br /> mRMR13                              0.762&#8194; (-)&#8194; *&#8194; (-)             0.728&#8194;&#8194;&#8194;&#8194; *        0.654&#8194; (-)&#8194; *&#8194; (-)                          0.689&#8194; (-)&#8194; *&#8194; (-)              0.396&#8194;&#8194; *&#8194; (-) <br /> Gustafson(29)                       0.777&#8195;&#8194;&#8194; (+)&#8194; *                0.722&#8194;&#8194;&#8194;&#8194;&#8194;&#8194; *      0.715&#8195;&#8194;&#8194; (+)&#8194; *                             0.719&#8195;&#8194;&#8194; (+)&#8194; *                 0.440&#8194; (+)&#8194; * <br /> N4                                  0.780&#8195;&#8194;&#8194; (+)                   0.733              0.701&#8195;&#8194;&#8194; (+)                                0.717&#8195;&#8194;&#8194; (+)                    0.446&#8194; (+) <br /> N5                                  0.779&#8195;&#8194;&#8194; (+)                   0.730              0.706&#8195;&#8194;&#8194; (+)                                0.718&#8195;&#8194;&#8194; (+)                    0.445&#8194; (+) <br /> N6                                  0.762&#8195;&#8195;&#8195;&#8194; (-) (-)              0.735              0.663&#8195;&#8195;&#8195;&#8194; (-) (-)                           0.696                           0.425 <br /> N7                                  0.783&#8195;&#8194;&#8194; (+)&#8194;&#8194;&#8194; (+)            0.737              0.696&#8195;&#8194;&#8194; (+)                                0.716&#8195;&#8194;&#8194; (+)&#8194;&#8194;&#8194; (+)             0.448&#8194; (+) <br /> N8                                  0.781&#8195;&#8194;&#8194; (+)                   0.723              0.711&#8195;&#8194;&#8194; (+)                                0.717&#8195;&#8194;&#8194; (+)                    0.439&#8194; (+) <br /> N9                                  0.782&#8195;&#8194;&#8194; (+)                   0.715              0.703&#8195;&#8194;&#8194; (+)                                0.709&#8195;&#8194;&#8194; (+)                    0.423 <br /> N10                                 0.781&#8195;&#8194;&#8194; (+)                   0.725              0.702&#8195;&#8194;&#8194; (+)                                0.713&#8195;&#8194;&#8194; (+)                    0.436&#8194; (+) <br /> N11                                 0.777&#8195;&#8194;&#8194; (+)                   0.719              0.700&#8195;&#8194;&#8194; (+)                                0.709&#8195;&#8194;&#8194; (+)                    0.426 <br /> N12                                 0.776&#8195;&#8194;&#8194; (+)                   0.715              0.695&#8195;&#8194;&#8194; (+)                                0.705&#8195;&#8194;&#8194; (+)                    0.418 <br /> N13                                 0.776&#8195;&#8194;&#8194; (+)                   0.731              0.695&#8195;&#8194;&#8194; (+)                                0.712&#8195;&#8194;&#8194; (+)                    0.439&#8194; (+) <br /> N80                                 0.769                          0.711              0.715&#8195;&#8194;&#8194; (+)&#8194;&#8194;&#8194; (+)                         0.713&#8195;&#8194;&#8194; (+)                    0.424 <br />  <br />  <br />  <br /> The CMIM09 predictor effective                                              amino acid occurrence W, bit string double screen- <br /> over entire range. Most predictors outper-                                    ing scheme, cytoplasm, endoplasmic reticulum, EI <br /> formed predictors 15%. N9                                           (essentiality index), nucleus, PR (phyletic reten- <br /> most prominent relatively effective                                    tion) selected 10 times. Two among <br /> the entire range. Figure&#160;9 depicts average per-                                  features, EI PR, included all <br /> centage curves.                                                                          feature subsets regarded most <br />                                                                                          important factors identifying essential proteins. <br /> Discussion                                                                               N9 N8 feature subsets cover of <br /> By inspecting S. cerevisiae feature subsets listed                                   8 features. Their prediction capability asso- <br /> in Table&#160;4, observed prominent fea-                                     ciated feature subsets outperformed <br /> tures come diverse sources. This includes                                    Hwang&#8217;s results performance measures, except <br /> sequence, protein, topology &#173;properties.                                       AUC percentage probability very <br /> Among features, amino acid occurrence I,                                           low value. Furthermore, amino acids, which <br />                                                                                                                 1.0 <br />                        1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                 0.8 <br />                        0.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             CMIM32: 0.842 <br />                                                                                                                                                             mRMR31: 0.836 <br />                                                                                            True positive rate <br />  <br />  <br />  <br />  <br />                                                                                                                                                             Hwang: 0.822 <br />   True positive rate <br />  <br />  <br />  <br />  <br />                                                                CMIM32: 0.825                                                                                Acencio: 0.768 <br />                                                                                                                 0.6 <br />  <br />  <br />  <br />  <br />                                                                mRMR31: 0.821 <br />                        0.6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             N5: 0.824 <br />                                                                Hwang: 0.775                                                                                 N9: 0.833 <br />                                                                Acencio: 0.707                                                                               N16: 0.842 <br />                                                                N5: 0.727 <br />                                                                N9: 0.782 <br />                                                                                                                 0.4 <br />                        0.4 <br />  <br />  <br />  <br />  <br />                                                                N16: 0.814 <br />                                                                                                                 0.2 <br />                        0.2 <br />  <br />  <br />  <br />  <br />                                                                                                                 0.0 <br />                        0.0 <br />  <br />  <br />  <br />  <br />                                                                                                                       0.0   0.2           0.4         0.6        0.8         1.0 <br />                              0.0   0.2      0.4       0.6           0.8         1.0 <br />                                          False positive rate                                                                           False positive rate <br />  <br /> Figure 4. The average ROC curves AUCs imbalanced                             Figure 5. The average ROC curves AUCs balanced <br /> S. cerevisiae data set.                                                                  S. cerevisiae data set. <br />  <br />  <br />  <br /> 404	                                                                                                                                            Evolutionary Bioinformatics 2013:9 <br />                                                                                                                       Essentiality prediction SVM <br />  <br />  <br />                                                                                           N9 N6, choosing additional feature, they <br />                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                           significant higher N8 N5&#160;in per- <br />                                                                                           formance measures. <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                                                For E. coli features listed Table&#160;5, most <br />    True positive rate <br />  <br />  <br />  <br />  <br />                                                                 CMIM09: 0.708             second important features PR (phyletic <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                 mRMR13: 0.709 <br />                                                                 Gustafson: 0.711 <br />                                                                                           retention) open reading frame length. The rest <br />                                                                 N4: 0.691                 important features selected more <br />                                                                 N8: 0.705 <br />                                                                                           times included: average PSSM amino <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                 N11: 0.714 <br />                                                                 N80: 0.716 <br />                                                                                           acid C degree related integrated functional <br />                                                                                           &#173;interactions. N5, N7, N8, N13 covered most <br />                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                            features. Among 43 listed features, <br />                                                                                            21&#160; sequence-related features, amino acid <br />                         0.0 <br />  <br />  <br />  <br />  <br />                                                                                            occurrence average PSSM, selected. In <br />                               0.0   0.2      0.4       0.6          0.8            1.0 <br />                                                                                            data set, recommend feature subsets sizes <br />                                           False positive rate <br />                                                                                            exceeding 11 effectiveness and <br /> Figure 6. The average ROC curves AUCs imbalanced E. coli                       compactness feature size. <br /> data set. <br />                                                                                                With experimental results data sets, we <br />                                                                                            conclude phyletic retention important <br /> were relatively easy extract, included                                    feature identifying essential proteins. It defined as <br /> two feature subsets. For predictors built                                     number present ortholog organisms. Gustafson <br /> the feature subsets 10 features,                                      et&#160; al study21 analyzed different organisms calcu- <br /> consistently superior Hwang&#8217;s nearly perfor-                                     late phyletic retention E. coli S. cerevisiae <br /> mance measures. Interestingly, mRMR CMIM                                               data sets. This sensible different <span id='am-63' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> <br /> selected sequence-derived features,                                        associated different &#173;organisms. From <br /> PSSM amino acid occurrence. It                                        biological view, retention process long <br /> features good essentiality prediction terms                                     evolutionary periods suggests organisms <br /> of relevance feature independence. By analyzing                                        crucial certain cell &#173;functionality. By inspect- <br /> Tables&#160;6 7, recommend using N16, N9,                                            ing 5 occurrences amino acids Tables&#160;4 <br /> N6. N16 performed nearly CMIM32 (or                                             5, Tryptophan (W) <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Glycine</span> (G) were <br /> mRMR31) compact feature size. For                                          top-&#173;ranking features. Since amino <br />                                                                                            acids non-polar hydrophobic, hypoth- <br />                                                                                            esize essentiality related physi- <br />                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                            cochemical properties features possessing <br />                                                                                            discrimination information captured other <br />                                                                                            top-ranking features. <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                  CMIM09: 0.775 <br />                                                                                                In study, compiled various interaction <br />    True positive rate <br />  <br />  <br />  <br />  <br />                                                                  mRMR13: 0.762 <br />                                                                  Gustafson: 0.777 <br />                                                                                            information including physical, metabolic, tran- <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                  N4: 0.780 <br />                                                                  N8: 0.781                 scriptional regulation, integrated functional <br />                                                                  N11: 0.777 <br />                                                                  N80: 0.769                genomic context interactions. The experimen- <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                                            tal results revealed various properties, such <br />                                                                                            degrees, identified impor- <br />                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                            tant features. This implies interaction <br />                                                                                            &#173;information, limited physical interactions, <br />                                                                                             closely related essential properties. <br />                         0.0 <br />  <br />  <br />  <br />  <br />                               0.0   0.2      0.4       0.6           0.8            1.0     According literature, hubs networks, <br />                                           False positive rate                               possessing abundance interaction partners, are <br /> Figure 7. The average ROC curves AUCs balanced E. coli <br />                                                                                             important fact play central <br /> data set.                                                                                   roles mediating interactions numerous <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                               405 <br />  Hor et&#160;al <br />  <br />  <br /> Table 10. Percentage essential proteins imbalanced S. cerevisiae data. <br />  <br />                                 Top 5%       Top 10%        Top 15%         Top 20%      Top 25%       Top 30%      Top 50%      Top 75%       Top 100% <br /> CMIM32                          0.939*- -    0.918*         0.910*          0.892*       0.870*        0.839*       0.743*       0.645*        0.582* <br /> mRMR31                          0.955&#8195; * -   0.905- *&#8194; -    0.884-*         0.862-*      0.834-* -     0.820-*      0.740-*      0.641-*       0.572-* <br /> Hwang(10)                       0.959&#8195;&#8194; *    0.918&#8195;&#8194; *      0.871- - *      0.853- -*    0.843-&#8195; *     0.816- -*    0.720- - *   0.637- - *    0.563- - * <br /> Acencio(23)                     0.800- - -   0.741- - -     0.693- - -      0.661- - -   0.646- - -    0.625- - -   0.578- - -   0.519- - -    0.457- - - <br /> N4                              0.980        0.930          0.905-          0.877-       0.865-        0.850        0.727- -     0.632- - -    0.559- - - <br /> N5                              0.843- - -   0.861- - -     0.859- - -      0.852- - -   0.841- &#8194;&#8201; -   0.827-       0.751        0.641-        0.530- - - <br /> N6                              0.908- - -   0.894- - -     0.875- -        0.857- -     0.850-        0.834-       0.763        0.635- - -    0.526- - - <br /> N7                              0.861- - -   0.892- - -     0.897-          0.885-       0.854-        0.832-       0.770        0.645-        0.570- - <br /> N8                              0.892- - -   0.904- - -     0.895-          0.877-       0.868-        0.850        0.751        0.657         0.574- <br /> N9                              0.880- - -   0.911- &#8194;&#8201; -    0.895-          0.875-       0.860-        0.832-       0.753        0.665         0.585 <br /> N10                             0.882- - -   0.896- - -     0.893-          0.882-       0.858-        0.846        0.762        0.665         0.581- <br /> N11                             0.900- - -   0.900- - -     0.888-          0.875-       0.861-        0.856        0.769        0.667         0.580- <br /> N12                             0.941&#8194; - -   0.924          0.899-          0.872-       0.866-        0.854        0.776        0.664         0.588 <br /> N13                             0.941&#8194; - -   0.910- &#8194;&#8201; -    0.886-          0.870-       0.853-        0.840        0.781        0.672         0.578- <br /> N14                             0.949&#8194; - -   0.932          0.916           0.897        0.867-        0.845        0.759        0.667         0.587 <br /> N15                             0.906- - -   0.894- - -     0.897-          0.884-       0.866-        0.851        0.776        0.672         0.584 <br /> N16                             0.933- - -   0.901- - -     0.895-          0.886-       0.864-        0.851        0.771        0.677         0.599 <br /> N17                             0.943&#8194; - -   0.903- - -     0.879- -        0.871-       0.866-        0.856        0.777        0.679         0.595 <br /> N18                             0.937- - -   0.892- - -     0.880- -        0.870-       0.864-        0.854        0.778        0.683         0.595 <br /> N90                             0.939&#8194; - -   0.911- &#8194;&#8201; -    0.884-          0.869-       0.856-        0.835-       0.728- -     0.639- -      0.572- <br />  <br />  <br />  <br /> less-connected proteins. Thus, proteins involved                                       It takes approximately 1&#160;minute LIBSVM soft- <br /> the complex mediation processes likely                                        ware perform 2-fold cross-validation one <br /> be crucial cellular activity survival.                                             Power5+ processor IBM P595 computer. &#173;Assuming <br />    For feature selection proposed study, let                                  m&#160; =&#160; 90, t&#160; =&#160; 10 r&#160; =&#160; 5, total running time is <br /> the size available target selected features                                 4,000 20,000&#160; minutes. The IBM P595 <br /> m t, maximal retry times r. The num-                                       allows users manually submit processes into <br /> ber SVM cross-validation times 1/2                                          order speed &#173;execution. For <br /> (m&#160;+&#160;t&#160;+&#160;1) &#215; (m - t) 1/2 (m&#160;+&#160;t&#160;+&#160;1) &#215; (m &#8211; t) &#215; r.                                  example, invoke 10 SVM processes <br />                                                                                           simultaneously. Consequently, maximal 10-time <br />                                                                                           speed-up achieved total running time <br />                       1.0 <br />  <br />  <br />  <br />  <br />                                                                    CMIM32                 reduced. <br />                                                                    mRMR31 <br />                                                                    Hwang                      If inspect Tables&#160; 4 5, that <br />                       0.9 <br />  <br />  <br />  <br />  <br />                                                                    Acencio <br />                                                                    N5                     one-third features signifi- <br />                                                                    N9 <br />                                                                    N16                    cantly relevant selected. These <br />                       0.8 <br />    Ess. probability <br />  <br />  <br />  <br />  <br />                                                                                           features relatively easy remove back- <br />                                                                                           ward feature selection procedure beginning <br />                       0.7 <br />  <br />  <br />  <br />  <br />                                                                                           stage. &#173;According authors&#8217; experience, the <br />                                                                                           rounds retry r critical stage. With an <br />                       0.6 <br />  <br />  <br />  <br />  <br />                                                                                           increasing number features removed, required <br />                                                                                           number retry increased identifying <br />                       0.5 <br />  <br />  <br />  <br />  <br />                                                                                           relatively competent features increas- <br />                                                                                           ingly difficult. The number retry r accompanied <br />                       0.4 <br />  <br />  <br />  <br />  <br />                             0       20       40       60      80      100                 rest user-specified parameters (such the <br />                                                   Top (%)                                 minimal improvement &#65533;? et&#160;al) set appropriately <br /> Figure 8. The average percentage curves imbalanced <br />                                                                                           ensure feature selection procedure could <br /> S. cerevisiae data set.                                                                   proceed. <br />  <br />  <br /> 406	                                                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                                                       Essentiality prediction SVM <br />  <br />  <br /> Table 11. Percentage essential proteins imbalanced E. coli experiment. <br />  <br />                                 Top 5%        Top 10%      Top 15%          Top 20%       Top 25%      Top 30% Top 50%           Top 75% Top 100% <br /> CMIM09                          0.745*        0.775*       0.730*           0.730*        0.737*       0.727*       0.644*       0.542*      0.440* - <br /> mRMR13                          0.719-*       0.725-*      0.714-*          0.701-* -     0.690-* -    0.679-* -    0.614-* -    0.531-*     0.446 * <br /> Gustafson(29)                   0.706- -*     0.692- -*    0.705- -*        0.707- &#8194; *    0.695- &#8194; *   0.685- &#8194; *   0.624- &#8194; *   0.522- -*   0.436- -* <br /> N4                              0.610- - -    0.689- - -   0.765            0.760         0.749        0.752        0.649        0.534-      0.449 <br /> N5                              0.655- - -    0.705- -     0.747            0.747         0.745        0.743        0.653        0.535-      0.443&#8194;&#8201;&#8201;- <br /> N6                              0.719-        0.723- -     0.717-           0.736         0.744        0.748        0.658        0.525- -    0.435- - - <br /> N7                              0.568- - -    0.700- -     0.713- -         0.730         0.748        0.762        0.655        0.540-      0.464 <br /> N8                              0.671- - -    0.703- -     0.723-           0.736         0.728-       0.731        0.652        0.535-      0.449 <br /> N9                              0.813         0.785        0.766            0.754         0.734-       0.726-       0.685        0.548       0.459 <br /> N10                             0.794         0.751-       0.728-           0.732         0.741        0.745        0.668        0.550       0.463 <br /> N11                             0.719-        0.721- -     0.734            0.742         0.748        0.748        0.668        0.539-      0.457 <br /> N12                             0.735-        0.738-       0.745            0.750         0.743        0.740        0.667        0.548       0.458 <br /> N13                             0.655- - -    0.672- - -   0.713- -         0.739         0.736-       0.732        0.668        0.548       0.457 <br /> N80                             0.674- - -    0.690- - -   0.703- - -       0.705-&#8194;&#8201;-     0.703-       0.691-       0.632-       0.529- -    0.452 <br />  <br />  <br />  <br /> Conclusion Future Work                                                               method effective term feature size and <br /> In study, incorporated protein proper-                                   performance. This method took both <br /> ties, including sequence, protein, topology,                                   feature size performance consideration and <br /> properties. There total 55&#160; groups 96                                       consequently resultant feature subset consider- <br /> &#173;features. The features included data sets                                   ably compact. We compared experimental results <br />  experiments: S. cerevisiae E. coli. We used                                   carrying significance tests types of <br />  modified sequential backward feature selection iden-                                 performance &#173;measures. Hence, provides poten- <br />  tify good feature subsets used SVM software                                     tial researcher essential proteins practical guide to <br />  tools classifying essential proteins. In addition,                               feature method prominent. <br />  built SVM models imbalanced                                            In imbalanced S. cerevisiae data experiment, <br />  balanced data sets. As experimental results illus-                                  best results F-measure MCC 0.549 <br />  trate, features shown effective                                  0.495, respectively, associated the <br />  essentiality prediction. Feature &#173;subsets selected                               N13 predictor. In contrast, performance <br />                                                                                          measures, achieved 0.77 0.545&#160;in balanced <br />                                                                                          data experiment, associated N17 <br />                                                                 CMIM09 <br />                                                                                          predictor. The experimental results showed the <br />                                                                                          performance models better Hwang&#8217;s <br />                       0.8 <br />  <br />  <br />  <br />  <br />                                                                 mRMR13 <br />                                                                 Gustafson <br />                                                                 N4                       selected 9 features. If achieving <br />                                                                 N8 <br />                                                                                          higher accuracy main issue, recommend the <br />                       0.7 <br />    Ess. probability <br />  <br />  <br />  <br />  <br />                                                                 N11 <br />                                                                 N80                      N16&#160;model (16 features). When prefers compact <br />                                                                                          feature set small size, suggest using N9&#160;model <br />                                                                                          (9 features). We list important features. These fea- <br />                       0.6 <br />  <br />  <br />  <br />  <br />                                                                                          tures crucial identifying essential proteins. <br />                                                                                              For E. coli data set, best values F-measure <br />                       0.5 <br />  <br />  <br />  <br />  <br />                                                                                          MCC 0.421 0.407&#160; imbalanced <br />                                                                                          experiments. In balanced experiment, best <br />                                                                                          values F-measure MCC 0.718 0.448, <br />                       0.4 <br />  <br />  <br />  <br />  <br />                             0    20      40       60       80     100                    respectively. Both best results associated <br />                                               Top (%)                                    N5 predictor. For data set, that <br /> Figure 9. The average percentage curves imbalanced E. coli <br />                                                                                          predictors associated feature size 11 <br /> data set.                                                                                comparable Gustafsons&#8217;. <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                               407 <br />  Hor et&#160;al <br />  <br />  <br />    There possible methods                p&#173; rocess, signal transduction, transcription, trans- <br /> improving prediction capability. Features related         port process). <br /> to protein sequence properties use-           The feature sets obtained from <br /> ful identifying essentiality. Furthermore,       Acencio Lemke.5 <br /> proteins similar primary structures pos- <br />                                                            e.	 Betweenness centrality related integrated func- <br /> sess similar functions, essentiality be <br />                                                                tional, PI GC network: The values defined <br /> addressed sequence motif perspective.34 <br />                                                                identically mentioned the <br /> In addition approaches, performance <br />                                                                paths represented terms integrated <br /> can improved incorporating tools or <br />                                                                functional, PI GC network interactions. <br /> constructing <span id='am-110' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-111' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> predictors. Among these, the <br />                                                            f.	 Degree related integrated functional, PI GC <br /> majority vote35 strategy combining classi- <br />                                                                network: The values defined identically to <br /> fiers. This method represents simplest method <br />                                                                mentioned above, paths are <br /> for categorical data fusion. &#173;According litera- <br />                                                                represented terms integrated functional, PI, <br /> ture,36 prerequisite improvement arises from <br />                                                                GC network interactions. <br /> the fact individual classifier contain <br /> distinct information discrimination. Otherwise,            For feature sets, collected <br /> some negative effects imposed con-           network information Hu et&#160; al22 con- <br /> structed ensemble.                                         ducted calculations using iGraph software.37 <br />                                                            g.	 Maximum neighborhood component density <br /> Appendix                                                       maximum neighborhood component: The maxi- <br /> Feature extraction                                             mum neighborhood component (MNC) density <br /> a.	 Outdegree indegree related transcriptional          maximum neigh-borhood component (DMNC) <br />     regulation interaction: The feature represents             properties proposed Lin et&#160;al26 Chin.4 <br />     number outgoing (or incoming) links              For protein i, let N(i) set neighbors i. <br />     gene g corresponding protein. Links           The MNC connected component N(i) <br />     represented terms transcriptional regulation         maximum size, denoted M N(i). <br />     interactions.                                                  For protein i, number proteins the <br /> b.	 Betweenness centrality transcriptional regulation          number edges M N (i) denoted ni and <br />     interactions: Let &#963;gi gj denote number short-       di, respectively. DMNC di /ni&#945; 1 # <br />     est paths gi gj. The value &#963;gi gj (g)       &#945; # 2. In system, set &#945; 1.7. <br />     defined number shortest paths gi     h.	 Sequence features: We used feature sets from <br />     gj passing g. Paths represented         Lin et&#160;al.20 Let L protein length, Fi the <br />     terms transcriptional regulation interactions.          occurrence number amino acid protein, <br /> c.	 Betweenness centrality related physical inter-          H (i) hydrophobic coefficient38 amino acid <br />     actions: The value &#964;gi gj (g) defined num-       i, Pi,j position jth occurrence amino <br />     ber shortest paths gi gj passing            acid i, A(j) amino acid position j Sm,n <br />     g. The <span id='am-99' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> similar previous       score row m column n PSSM.39 <br />     one. &#173;However, paths represented           The sequence features listed follows. <br />     terms protein physical interactions.                	 1.&#8194; Protein length: L <br /> d.	 Protein properties: Acencio Lemke5 discov-         	 2.&#8194; Cysteine count: FC <br />     ered integration topological properties,   	 3.&#8194; Amino acid occurrence: The composition of <br />     cellular components, biological processes                  amino acid Fi /L, 1 # # 20. <br />     possess good capability predicting essential                                                F <br />                                                            	 4.&#8194; Average cysteine position: &#8721; j C=1 PC , j / FC. <br />     proteins. Hence, features contained cel-      	 5.&#8194; Average distance cysteines: <br />     lular components (cytoplasm, endoplasmic reticu-                   F     F <br />     lum, mitochondrion, nucleus localization)              &#8721; xC=1 &#8721; yC=1 PC , x &#8722; PC , y <br />     biological processes (cell cycle, metabolic                      L &#215; FC <br />  <br />  <br />  <br /> 408	                                                                                     Evolutionary Bioinformatics 2013:9 <br />                                                                                                         Essentiality prediction SVM <br />  <br />  <br /> 	&#8194; 6.&#8194; Cysteine odd-even index: FC mod 2.                         We used informational odds ratios (IOR)32 to <br /> 	&#8194; 7.&#8194; Average hydrophobicity: &#8721; i20=1 Fi &#215; H (i ) / L            represent association essentiality <br /> 	&#8194; 8.&#8194; Average hydrophobicity cysteine:                    &#173;predictions. IOR measures more <br />        The kth values average hydrophobic-                     likely protein essential learn- <br />        ity cysteines defined                   ing machine outputs essentiality non- <br />         &#8721; j C=1 H ( A( PC , j + k )) / FC . Here set k&#160;=&#160;-2, <br />           F <br />                                                                   essentiality. A value 1.0&#160;indicates association <br />        -1, +1, +2.                                                essentiality predictions produced <br /> 	&#8194; 9.&#8194; Cysteine position distribution: For 1 # d # &#65533;?,             learning machines. All confidence intervals of <br />        dth cysteine position distribution                 performance measures informational odds <br />                                                                   ratios corresponding prediction models are <br />               &#63729;&#63732;          PC , j d &#8722; 1        d            &#63740;&#63732; <br />                &#63730;&#945; j &#945; j =       ,      &#57388; &#945; j &#57379; ,1 &#57379; j &#57379; FC &#63741;      shown Tables&#160;12&#8211;15. <br /> 	              &#63732;&#63731;          L      &#65533;?           &#65533;?             &#63732;&#63742; <br />                                        Fc                         Comparison feature <br /> 		       We set &#65533;?&#160;=&#160;5                                             selection methods <br /> 	 10.&#8194; Average PSSM amino acid: The average                    We introduced feature selection meth- <br />                                    L <br />          PSSM residue &#8721; m =1 S m ,i / L .                 ods served benchmarks, mRMR13 and <br /> i.	 Phyletic retention: Gustafson et&#160; al21 discovered             CMIM,14 theoretical methods. <br />     essential proteins generally con-           Next, compared feature selection <br />     served nonessential proteins. Phyletic reten-            method feature subsets equal size were <br />     tion protein number organisms               selected. <br />     ortholog present. The ortholog                Unlike methods select top-ranking fea- <br />     protein obtained Hwang et&#160;al.7                       tures based F-score mutual information without <br /> j.	 Essential index:7 Essential index measures ratio          considering relationships features, mRMR <br />     essential proteins neighbors N(i) node i.        accommodates feature relevance respect to <br />     Essential index node defined p (i ) /di ,          class label dependency selected features. <br />     p(i) number essential proteins             The strategy combines maximal relevance <br />     N(i) di degree node i.                          minimal redundancy criteria. In order take <br /> k.	 Clique level:7 The clique level protein defined    criteria consideration avoid <br />     maximal clique containing i. Here, cliques      exhaustive search, mRMR adopts incremen- <br />     sizes 3 10 taken consideration.         tal search approach. That is, rth selected feature <br /> l.	 Number paralagous genes: It shown genes            satisfy <br />     likely essential dupli- <br />                                                                                         &#63729;&#63732;          1                              &#63740;&#63732; <br />     cate existed genome.21 This feature               X r = arg max &#63730; I ( X j , Y ) -          &#8721;        I ( X j , X )&#63741; 	   (1) <br />                                                                         X j &#8712;X - X r -1 &#63732;         r - 1 X &#8712;X r -1                 &#63742;&#63732; <br />     defined number genes present                                   &#63731; <br />     genome. In addition, BLASTP <br />     E-values 10&#8722;20 ratios            X feature set. Xi feature within <br />     larger gene smaller exceed 1.33.            Xr-1 r - 1 number selected features con- <br /> m.	Open reading frame length: Gustafson et&#160; al21                  tained Xr-1. Xj feature selected. <br />     observed ancestral genes likely           I (Xj, Y) mutual information quantifies the <br />     essential proteins generally larger           dependence (or relevance) feature Xj and <br />     evolution. Consequently, The open                  class label Y. This means rth selected feature <br />     reading frame length indicate essentiality.               relevant class label possible <br />                                                                   possesses dependency averagely the <br /> Confidence intervals performance                               selected features. <br /> measures informational odds ratios                                For CMIM, feature Xj good I (Y, Xj|Xi) <br /> All performance measures multiplied                       large selected feature Xi. I (Y, Xj|Xi) is <br /> 100. The confidence intervals set 95%.                    &#173;conditional mutual information quantifies <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                      409 <br />  Hor et&#160;al <br />  <br />  <br /> Table 12. Confidence intervals performance measures (&#215;100) informational odds ratios models produced the <br /> imbalanced S. cerevisiae data set. <br />  <br />                  AUC                Precision         Recall            F1                 MCC                   IOR <br /> CMIM32           82.5&#160;&#177;&#160;1.2         74.4&#160;&#177;&#160;3.1        36.9&#160;&#177;&#160;4.5        49.3&#160;&#177;&#160;3.8         45.0&#160;&#177;&#160;3.6            5.2&#160;&#177;&#160;0.5 <br /> mRMR31           82.1&#160;&#177;&#160;1.6         73.8&#160;&#177;&#160;3.2        37.2&#160;&#177;&#160;4.3        49.5&#160;&#177;&#160;3.6         44.9&#160;&#177;&#160;3.4            5.2&#160;&#177;&#160;0.5 <br /> Hwang            77.5&#160;&#177;&#160;2.2         74.3&#160;&#177;&#160;3.7        34.3&#160;&#177;&#160;4.3        46.9&#160;&#177;&#160;4.0         43.2&#160;&#177;&#160;3.6            5.1&#160;&#177;&#160;0.4 <br /> Acencio          70.7&#160;&#177;&#160;3.4         67.5&#160;&#177;&#160;6.3        12.1&#160;&#177;&#160;5.5        20.4&#160;&#177;&#160;7.6         22.8&#160;&#177;&#160;6.0            3.7&#160;&#177;&#160;0.4 <br /> N4               74.4&#160;&#177;&#160;2.7         78.2&#160;&#177;&#160;3.7        32.7&#160;&#177;&#160;4.1        46.1&#160;&#177;&#160;4.1         43.9&#160;&#177;&#160;3.5            5.3&#160;&#177;&#160;0.4 <br /> N5               72.7&#160;&#177;&#160;3.6         74.1&#160;&#177;&#160;4.1        38.7&#160;&#177;&#160;4.7        50.9&#160;&#177;&#160;4.1         46.1&#160;&#177;&#160;3.8            5.3&#160;&#177;&#160;0.5 <br /> N6               73.0&#160;&#177;&#160;3.2         75.2&#160;&#177;&#160;4.2        39.5&#160;&#177;&#160;4.4        51.8&#160;&#177;&#160;3.8         47.2&#160;&#177;&#160;3.6            5.5&#160;&#177;&#160;0.5 <br /> N7               76.1&#160;&#177;&#160;2.4         76.7&#160;&#177;&#160;3.7        38.6&#160;&#177;&#160;4.4        51.3&#160;&#177;&#160;3.9         47.3&#160;&#177;&#160;3.6            5.5&#160;&#177;&#160;0.5 <br /> N8               77.2&#160;&#177;&#160;2.4         75.5&#160;&#177;&#160;3.4        37.1&#160;&#177;&#160;4.9        49.8&#160;&#177;&#160;4.3         45.7&#160;&#177;&#160;3.9            5.3&#160;&#177;&#160;0.5 <br /> N9               78.2&#160;&#177;&#160;2.4         74.9&#160;&#177;&#160;3.4        38.2&#160;&#177;&#160;4.5        50.6&#160;&#177;&#160;3.9         46.2&#160;&#177;&#160;3.6            5.4&#160;&#177;&#160;0.5 <br /> N10              78.1&#160;&#177;&#160;2.2         75.1&#160;&#177;&#160;3.5        39.9&#160;&#177;&#160;4.1        52.1&#160;&#177;&#160;3.6         47.4&#160;&#177;&#160;3.5            5.5&#160;&#177;&#160;0.5 <br /> N11              78.6&#160;&#177;&#160;2.1         75.2&#160;&#177;&#160;3.2        40.2&#160;&#177;&#160;4.2        52.4&#160;&#177;&#160;3.6         47.6&#160;&#177;&#160;3.4            5.5&#160;&#177;&#160;0.5 <br /> N12              79.8&#160;&#177;&#160;2.0         75.9&#160;&#177;&#160;3.2        40.9&#160;&#177;&#160;4.2        53.2&#160;&#177;&#160;3.6         48.5&#160;&#177;&#160;3.4            5.7&#160;&#177;&#160;0.5 <br /> N13              78.9&#160;&#177;&#160;1.9         74.8&#160;&#177;&#160;3.2        43.3&#160;&#177;&#160;4.3        54.9&#160;&#177;&#160;3.4         49.5&#160;&#177;&#160;3.4            5.8&#160;&#177;&#160;0.5 <br /> N14              80.2&#160;&#177;&#160;1.8         74.9&#160;&#177;&#160;3.2        39.7&#160;&#177;&#160;4.3        51.9&#160;&#177;&#160;3.5         47.1&#160;&#177;&#160;3.4            5.5&#160;&#177;&#160;0.5 <br /> N15              80.1&#160;&#177;&#160;1.9         76.3&#160;&#177;&#160;3.3        40.6&#160;&#177;&#160;4.2        53.0&#160;&#177;&#160;3.5         48.5&#160;&#177;&#160;3.5            5.7&#160;&#177;&#160;0.5 <br /> N16              81.4&#160;&#177;&#160;1.7         76.2&#160;&#177;&#160;3.2        40.1&#160;&#177;&#160;4.6        52.5&#160;&#177;&#160;3.8         48.0&#160;&#177;&#160;3.6            5.6&#160;&#177;&#160;0.5 <br /> N17              81.4&#160;&#177;&#160;1.7         76.1&#160;&#177;&#160;3.3        40.7&#160;&#177;&#160;4.5        53.0&#160;&#177;&#160;3.8         48.4&#160;&#177;&#160;3.6            5.7&#160;&#177;&#160;0.5 <br /> N18              81.1&#160;&#177;&#160;1.8         75.1&#160;&#177;&#160;3.2        41.1&#160;&#177;&#160;4.3        53.1&#160;&#177;&#160;3.6         48.2&#160;&#177;&#160;3.5            5.6&#160;&#177;&#160;0.5 <br /> N90              82.9&#160;&#177;&#160;1.0         73.8&#160;&#177;&#160;2.8        35.5&#160;&#177;&#160;4.5        47.9&#160;&#177;&#160;3.6         43.8&#160;&#177;&#160;3.4            5.1&#160;&#177;&#160;0.4 <br />  <br />  <br />  <br /> Table 13. Confidence intervals performance measures (&#215;100) informational odds ratios models produced the <br /> balanced S. cerevisiae data set. <br />  <br />                  AUC                Precision         Recall            F1                 MCC                   IOR <br /> CMIM32           84.2&#160;&#177;&#160;1.5         77.2&#160;&#177;&#160;2.2        76.6&#160;&#177;&#160;2.9        76.9&#160;&#177;&#160;2.1         54.0&#160;&#177;&#160;3.9            3.3&#160;&#177;&#160;0.4 <br /> mRMR31           83.6&#160;&#177;&#160;1.6         76.5&#160;&#177;&#160;2.4        74.1&#160;&#177;&#160;3.0        75.2&#160;&#177;&#160;2.1         51.3&#160;&#177;&#160;4.1            3.0&#160;&#177;&#160;0.3 <br /> Hwang            82.2&#160;&#177;&#160;1.7         77.8&#160;&#177;&#160;2.6        72.0&#160;&#177;&#160;3.7        74.8&#160;&#177;&#160;2.3         51.6&#160;&#177;&#160;3.9            3.0&#160;&#177;&#160;0.3 <br /> Acencio          76.8&#160;&#177;&#160;2.2         69.6&#160;&#177;&#160;2.4        73.4&#160;&#177;&#160;4.0        71.4&#160;&#177;&#160;2.3         41.4&#160;&#177;&#160;4.3            2.5&#160;&#177;&#160;0.3 <br /> N4               81.1&#160;&#177;&#160;1.8         77.7&#160;&#177;&#160;2.5        71.6&#160;&#177;&#160;3.6        74.5&#160;&#177;&#160;2.3         51.2&#160;&#177;&#160;3.9            3.0&#160;&#177;&#160;0.3 <br /> N5               82.4&#160;&#177;&#160;1.8         77.8&#160;&#177;&#160;2.6        73.5&#160;&#177;&#160;3.5        75.6&#160;&#177;&#160;2.2         52.7&#160;&#177;&#160;4.1            3.1&#160;&#177;&#160;0.3 <br /> N6               82.7&#160;&#177;&#160;1.8         77.8&#160;&#177;&#160;2.6        73.9&#160;&#177;&#160;3.6        75.8&#160;&#177;&#160;2.3         53.0&#160;&#177;&#160;4.1            3.1&#160;&#177;&#160;0.3 <br /> N7               83.1&#160;&#177;&#160;1.8         77.9&#160;&#177;&#160;2.5        73.3&#160;&#177;&#160;3.5        75.5&#160;&#177;&#160;2.2         52.6&#160;&#177;&#160;4.0            3.1&#160;&#177;&#160;0.3 <br /> N8               82.6&#160;&#177;&#160;1.8         78.6&#160;&#177;&#160;2.5        72.1&#160;&#177;&#160;3.5        75.2&#160;&#177;&#160;2.3         52.7&#160;&#177;&#160;4.0            3.1&#160;&#177;&#160;0.3 <br /> N9               83.3&#160;&#177;&#160;1.8         79.1&#160;&#177;&#160;2.4        73.5&#160;&#177;&#160;3.4        76.2&#160;&#177;&#160;2.2         54.1&#160;&#177;&#160;3.8            3.2&#160;&#177;&#160;0.3 <br /> N10              83.4&#160;&#177;&#160;1.7         78.9&#160;&#177;&#160;2.4        73.6&#160;&#177;&#160;3.3        76.1&#160;&#177;&#160;2.1         54.0&#160;&#177;&#160;3.8            3.2&#160;&#177;&#160;0.3 <br /> N11              83.1&#160;&#177;&#160;1.7         78.4&#160;&#177;&#160;2.4        73.7&#160;&#177;&#160;3.5        76.0&#160;&#177;&#160;2.2         53.5&#160;&#177;&#160;3.8            3.2&#160;&#177;&#160;0.3 <br /> N12              82.9&#160;&#177;&#160;1.8         77.9&#160;&#177;&#160;2.5        73.2&#160;&#177;&#160;3.1        75.5&#160;&#177;&#160;2.2         52.6&#160;&#177;&#160;4.2            3.1&#160;&#177;&#160;0.3 <br /> N13              83.4&#160;&#177;&#160;1.7         78.8&#160;&#177;&#160;2.4        73.0&#160;&#177;&#160;3.5        75.8&#160;&#177;&#160;2.2         53.5&#160;&#177;&#160;3.9            3.1&#160;&#177;&#160;0.3 <br /> N14              83.6&#160;&#177;&#160;1.6         77.7&#160;&#177;&#160;2.3        74.3&#160;&#177;&#160;3.4        75.9&#160;&#177;&#160;2.1         53.0&#160;&#177;&#160;3.8            3.2&#160;&#177;&#160;0.3 <br /> N15              84.3&#160;&#177;&#160;1.7         78.4&#160;&#177;&#160;2.4        74.8&#160;&#177;&#160;3.3        76.6&#160;&#177;&#160;2.1         54.2&#160;&#177;&#160;3.9            3.3&#160;&#177;&#160;0.4 <br /> N16              84.2&#160;&#177;&#160;1.6         77.7&#160;&#177;&#160;2.2        75.6&#160;&#177;&#160;3.1        76.7&#160;&#177;&#160;2.0         54.0&#160;&#177;&#160;3.7            3.3&#160;&#177;&#160;0.4 <br /> N17              84.7&#160;&#177;&#160;1.6         77.8&#160;&#177;&#160;2.3        76.3&#160;&#177;&#160;3.0        77.0&#160;&#177;&#160;2.0         54.5&#160;&#177;&#160;3.8            3.3&#160;&#177;&#160;0.4 <br /> N18              84.0&#160;&#177;&#160;1.6         77.9&#160;&#177;&#160;2.4        74.0&#160;&#177;&#160;3.3        75.9&#160;&#177;&#160;2.0         53.1&#160;&#177;&#160;3.8            3.1&#160;&#177;&#160;0.3 <br /> N90              83.9&#160;&#177;&#160;1.4         76.0&#160;&#177;&#160;2.0        75.3&#160;&#177;&#160;2.7        75.7&#160;&#177;&#160;1.8         51.6&#160;&#177;&#160;3.5            3.1&#160;&#177;&#160;0.3 <br />  <br />  <br /> 410	                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                         Essentiality prediction SVM <br />  <br />  <br /> Table 14. Confidence intervals performance measures (&#215;100) informational odds ratios models produced the <br /> imbalanced E. coli data set. <br />                       AUC             Precision        Recall             F1               MCC                IOR <br /> CMIM09                70.1&#160;&#177;&#160;0.9      72.0&#160;&#177;&#160;1.4       27.1&#160;&#177;&#160;0.7         39.4&#160;&#177;&#160;0.9       38.2&#160;&#177;&#160;0.9         5.2&#160;&#177;&#160;0.6 <br /> mRMR13                71.5&#160;&#177;&#160;2.4      71.3&#160;&#177;&#160;5.4       25.0&#160;&#177;&#160;5.8         37.0&#160;&#177;&#160;6.8       36.0&#160;&#177;&#160;5.8         4.7&#160;&#177;&#160;0.6 <br /> Gustafson             71.1&#160;&#177;&#160;2.3      66.5&#160;&#177;&#160;4.6       25.5&#160;&#177;&#160;5.0         36.8&#160;&#177;&#160;5.2       34.7&#160;&#177;&#160;4.8         4.9&#160;&#177;&#160;0.6 <br /> N4                    69.1&#160;&#177;&#160;1.8      72.5&#160;&#177;&#160;1.0       28.0&#160;&#177;&#160;0.6         40.4&#160;&#177;&#160;0.7       39.1&#160;&#177;&#160;0.7         5.5&#160;&#177;&#160;0.6 <br /> N5                    69.0&#160;&#177;&#160;2.0      73.7&#160;&#177;&#160;1.4       29.5&#160;&#177;&#160;0.8         42.1&#160;&#177;&#160;0.9       40.7&#160;&#177;&#160;1.0         5.7&#160;&#177;&#160;0.6 <br /> N6                    70.1&#160;&#177;&#160;1.7      74.2&#160;&#177;&#160;1.4       28.7&#160;&#177;&#160;0.9         41.4&#160;&#177;&#160;1.1       40.3&#160;&#177;&#160;1.0         5.7&#160;&#177;&#160;0.6 <br /> N7                    71.4&#160;&#177;&#160;1.4      73.5&#160;&#177;&#160;1.3       27.5&#160;&#177;&#160;0.7         40.0&#160;&#177;&#160;0.9       39.2&#160;&#177;&#160;0.9         5.5&#160;&#177;&#160;0.6 <br /> N8                    70.5&#160;&#177;&#160;1.3      74.2&#160;&#177;&#160;1.1       28.8&#160;&#177;&#160;0.8         41.5&#160;&#177;&#160;0.9       40.5&#160;&#177;&#160;0.9         5.7&#160;&#177;&#160;0.6 <br /> N9                    70.7&#160;&#177;&#160;1.5      72.6&#160;&#177;&#160;1.4       29.3&#160;&#177;&#160;1.0         41.7&#160;&#177;&#160;1.2       40.1&#160;&#177;&#160;1.2         5.6&#160;&#177;&#160;0.6 <br /> N10                   71.1&#160;&#177;&#160;1.5      72.4&#160;&#177;&#160;1.6       29.4&#160;&#177;&#160;1.0         41.8&#160;&#177;&#160;1.1       40.1&#160;&#177;&#160;1.2         5.6&#160;&#177;&#160;0.6 <br /> N11                   71.4&#160;&#177;&#160;1.4      73.2&#160;&#177;&#160;1.3       27.8&#160;&#177;&#160;0.8         40.3&#160;&#177;&#160;1.0       39.3&#160;&#177;&#160;1.0         5.5&#160;&#177;&#160;0.6 <br /> N12                   71.2&#160;&#177;&#160;1.4      72.5&#160;&#177;&#160;1.9       29.2&#160;&#177;&#160;1.1         41.6&#160;&#177;&#160;1.2       40.0&#160;&#177;&#160;1.3         5.6&#160;&#177;&#160;0.6 <br /> N13                   71.4&#160;&#177;&#160;1.3      73.3&#160;&#177;&#160;1.8       28.7&#160;&#177;&#160;1.2         41.3&#160;&#177;&#160;1.4       40.0&#160;&#177;&#160;1.4         5.6&#160;&#177;&#160;0.6 <br /> N80                   71.6&#160;&#177;&#160;0.9      67.7&#160;&#177;&#160;2.1       23.7&#160;&#177;&#160;1.3         35.2&#160;&#177;&#160;1.6       33.9&#160;&#177;&#160;1.6         4.9&#160;&#177;&#160;0.6 <br />  <br />  <br /> Table 15. Confidence intervals performance measures (&#215;100) informational odds ratios models produced bal- <br /> anced E. coli data set. <br />                       AUC             Precision       Recall              F1               MCC                IOR <br /> CMIM09                76.7&#160;&#177;&#160;1.7      72.0&#160;&#177;&#160;2.4      70.0&#160;&#177;&#160;3.7          71.0&#160;&#177;&#160;2.2       42.1&#160;&#177;&#160;4.0         2.4&#160;&#177;&#160;0.3 <br /> mRMR13                76.2&#160;&#177;&#160;2.0      72.8&#160;&#177;&#160;3.1      65.4&#160;&#177;&#160;6.4          68.9&#160;&#177;&#160;3.3       39.6&#160;&#177;&#160;3.9         2.2&#160;&#177;&#160;0.2 <br /> Gustafson             77.7&#160;&#177;&#160;2.6      72.2&#160;&#177;&#160;3.3      71.5&#160;&#177;&#160;4.0          71.9&#160;&#177;&#160;2.8       44.0&#160;&#177;&#160;5.5         2.6&#160;&#177;&#160;0.3 <br /> N4                    78.0&#160;&#177;&#160;1.6      73.3&#160;&#177;&#160;2.5      70.1&#160;&#177;&#160;2.7          71.7&#160;&#177;&#160;1.9       44.6&#160;&#177;&#160;3.8         2.6&#160;&#177;&#160;0.3 <br /> N5                    77.9&#160;&#177;&#160;1.7      73.0&#160;&#177;&#160;2.5      70.6&#160;&#177;&#160;2.8          71.8&#160;&#177;&#160;1.8       44.5&#160;&#177;&#160;3.8         2.6&#160;&#177;&#160;0.3 <br /> N6                    76.2&#160;&#177;&#160;1.7      73.5&#160;&#177;&#160;2.6      66.3&#160;&#177;&#160;4.3          69.6&#160;&#177;&#160;2.6       42.5&#160;&#177;&#160;4.1         2.4&#160;&#177;&#160;0.3 <br /> N7                    78.3&#160;&#177;&#160;1.7      73.7&#160;&#177;&#160;2.5      69.6&#160;&#177;&#160;2.7          71.6&#160;&#177;&#160;1.8       44.8&#160;&#177;&#160;3.6         2.6&#160;&#177;&#160;0.3 <br /> N8                    78.1&#160;&#177;&#160;1.7      72.3&#160;&#177;&#160;2.3      71.1&#160;&#177;&#160;3.3          71.7&#160;&#177;&#160;2.1       43.9&#160;&#177;&#160;3.8         2.5&#160;&#177;&#160;0.3 <br /> N9                    78.2&#160;&#177;&#160;1.6      71.5&#160;&#177;&#160;2.2      70.3&#160;&#177;&#160;4.1          70.9&#160;&#177;&#160;2.3       42.3&#160;&#177;&#160;3.8         2.4&#160;&#177;&#160;0.3 <br /> N10                   78.1&#160;&#177;&#160;1.6      72.5&#160;&#177;&#160;2.4      70.2&#160;&#177;&#160;3.3          71.3&#160;&#177;&#160;2.1       43.6&#160;&#177;&#160;3.9         2.5&#160;&#177;&#160;0.3 <br /> N11                   77.7&#160;&#177;&#160;1.7      71.9&#160;&#177;&#160;2.2      70.0&#160;&#177;&#160;3.3          70.9&#160;&#177;&#160;2.0       42.6&#160;&#177;&#160;3.6         2.5&#160;&#177;&#160;0.3 <br /> N12                   77.6&#160;&#177;&#160;1.9      71.5&#160;&#177;&#160;2.3      69.5&#160;&#177;&#160;4.5          70.5&#160;&#177;&#160;2.5       41.8&#160;&#177;&#160;4.0         2.4&#160;&#177;&#160;0.3 <br /> N13                   77.6&#160;&#177;&#160;1.7      73.1&#160;&#177;&#160;2.4      69.5&#160;&#177;&#160;3.0          71.2&#160;&#177;&#160;2.1       43.9&#160;&#177;&#160;3.9         2.5&#160;&#177;&#160;0.3 <br /> N80                   76.9&#160;&#177;&#160;1.8      71.1&#160;&#177;&#160;2.4      71.5&#160;&#177;&#160;2.4          71.3&#160;&#177;&#160;1.8       42.4&#160;&#177;&#160;3.8         2.5&#160;&#177;&#160;0.3 <br />  <br />  <br /> Table 16. Performance comparison method vs. mRMR imbalanced S. cerevisiae data set sizes <br /> of feature subsets, the&#160;. symbol represents values significantly higher. <br />                 AUC                   Precision           Recall                  F-measure             MCC <br /> N4              0.744 0.762           0.782   0.756       0.327 0.331             0.461 0.461           0.439 0.430 <br /> N5              0.727 0.718           0.741   0.740       0.387 0.359             0.509 0.484           0.461 0.442 <br /> N6              0.730 0.753           0.752   0.753       0.395 . 0.333           0.518 . 0.462         0.472 . 0.430 <br /> N7              0.761 0.763           0.767   0.761       0.386 . 0.330           0.513 . 0.460         0.473 . 0.431 <br /> N8              0.772 . 0.771         0.755   0.757       0.371 . 0.326           0.498 . 0.456         0.457 . 0.427 <br /> N9              0.782 0.776           0.749   0.749       0.382 . 0.341           0.506 . 0.469         0.462 . 0.434 <br /> N10             0.781 . 0.778         0.751   0.752       0.399 . 0.340           0.521 . 0.469         0.474 . 0.434 <br /> N11             0.786 . 0.774         0.752   0.750       0.402 . 0.341           0.524 . 0.469         0.476 . 0.434 <br /> N12             0.798 . 0.781         0.759   0.757       0.409 . 0.334           0.532 . 0.463         0.485 . 0.432 <br /> N13             0.789 . 0.774         0.748   0.746       0.433 . 0.342           0.549 . 0.469         0.495 . 0.432 <br /> N14             0.802 . 0.775         0.749   0.750       0.397 . 0.340           0.519 . 0.468         0.471 . 0.433 <br /> N15             0.801 . 0.798         0.763   0.764       0.406 . 0.318           0.530 . 0.449         0.485 . 0.424 <br /> N16             0.814 . 0.799         0.762   0.762       0.401 . 0.318           0.525 . 0.449         0.480 . 0.423 <br /> N17             0.814 . 0.799         0.761   0.759       0.407 . 0.326           0.530 . 0.456         0.484 . 0.427 <br /> N18             0.811 . 0.797         0.751   0.749       0.411 . 0.342           0.531 . 0.469         0.482 . 0.434 <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                 411 <br />  Hor et&#160;al <br />  <br />  <br /> Table 17. Performance comparison new method vs. CMIM imbalanced S. cerevisiae data set identical <br /> number features selected. <br />  <br />              AUC                       Precision               Recall               F1                       MCC <br /> N4           0.744 0.761               0.782     0.762         0.327 0.344          0.461 0.474              0.439 0.442 <br /> N5           0.727 0.735               0.741     0.738         0.387 0.371          0.509 0.494              0.461 0.449 <br /> N6           0.730 0.757               0.752     0.749         0.395 0.359          0.518 0.485              0.472 0.446 <br /> N7           0.761 0.779               0.767     0.763         0.386 . 0.339        0.513 . 0.470            0.473 . 0.439 <br /> N8           0.772 0.779               0.755     0.754         0.371 . 0.350        0.498 . 0.478            0.457 . 0.442 <br /> N9           0.782 0.776               0.749     0.750         0.382 . 0.357        0.506 . 0.483            0.462 . 0.445 <br /> N10          0.781 0.782               0.751     0.751         0.399 . 0.353        0.521 . 0.480            0.474 . 0.443 <br /> N11          0.786 0.786               0.752     0.752         0.402 . 0.363        0.524 . 0.490            0.476 . 0.450 <br /> N12          0.798 0.799               0.759     0.758         0.409 . 0.354        0.532 . 0.483            0.485 . 0.447 <br /> N13          0.789 0.797               0.748     0.750         0.433 . 0.360        0.549 . 0.487            0.495 . 0.447 <br /> N14          0.802 . 0.801             0.749     0.749         0.397 . 0.348        0.519 . 0.475            0.471 . 0.438 <br /> N15          0.801 . 0.797             0.763     0.760         0.406 . 0.330        0.530 . 0.460            0.485 . 0.430 <br /> N16          0.814 . 0.796             0.762     0.759         0.401 . 0.338        0.525 . 0.468            0.480 . 0.436 <br /> N17          0.814 . 0.795             0.761     0.756         0.407 . 0.339        0.530 . 0.469            0.484 . 0.435 <br /> N18          0.811 0.799               0.751     0.756         0.411 . 0.338        0.531 . 0.467            0.482 . 0.435 <br />  <br />  <br /> d&#173; iscrepancy features Xj Xi given               In following paragraph, compare our <br />  class label Y. Consequently, feature selection pro-          method feature selection methods <br />  cedure carried incremental manner             feature subsets equal size selected. <br />  follows:                                                      We ran SVM software Hwang&#8217;s or <br />                                                                   Gustafson feature sets tune SVM param- <br />                 &#965; (1) = arg max I ( X j , Y )	           (2)      eters achieve highest average performances. <br /> 	                               Xj <br />                                                                   To fairly compare methods given feature subsets <br />  <br /> 	                      Xj <br />                             { <br />         &#965; (r) = arg max min I (Y , X j | X &#965; (k ) ) .	 <br />                             k &#8804; r -1 <br />                                                    }     (3) <br />                                                                   sizes obtained methods, mRMR <br />                                                                   CMIM, used SVM software and <br />                                                                   adjust cost parameters order achieve simi- <br /> where v(k) denotes kth selected feature.                      lar levels precision. For S. cerevisiae E. coli <br />                                                                   data set, feature numbers k 4 # k # 18 and <br />  <br /> Table 18. Performance comparison method vs. mRMR balanced S. cerevisiae data set sizes <br /> of feature subsets, the&#160;. symbol indicates values significantly higher. <br />  <br />              AUC                       Precision               Recall               F-measure                MCC <br /> N4           0.811 0.815               0.777     0.770         0.716 0.725          0.745 0.747              0.512 0.510 <br /> N5           0.824 0.818               0.778     0.771         0.735 0.722          0.756 0.745              0.527 0.508 <br /> N6           0.827 0.814               0.778     0.775         0.739 0.709          0.758 0.740              0.530 0.504 <br /> N7           0.831 0.824               0.779     0.779         0.733 0.718          0.755 0.747              0.526 0.516 <br /> N8           0.826 0.827               0.786     0.781         0.721 0.721          0.752 0.750              0.527 0.521 <br /> N9           0.833 0.834               0.791     0.783         0.735 0.734          0.762 0.758              0.541 0.531 <br /> N10          0.834 0.835               0.789     0.783         0.736 0.733          0.761 0.757              0.540 0.531 <br /> N11          0.831 0.834               0.784     0.780         0.737 0.730          0.760 0.754              0.535 0.525 <br /> N12          0.829 0.834               0.779     0.778         0.732 0.734          0.755 0.755              0.526 . 0.525 <br /> N13          0.834 0.834               0.788     0.779         0.730 0.732          0.758 0.754              0.535 0.525 <br /> N14          0.836 0.832               0.777     0.777         0.743 0.731          0.759 0.753              0.530 0.522 <br /> N15          0.843 0.835               0.784     0.778         0.748 0.734          0.766 0.756              0.542 0.526 <br /> N16          0.842 . 0.836             0.777     0.777         0.756 . 0.735        0.767 . 0.755            0.540 . 0.525 <br /> N17          0.847 . 0.834             0.778     0.777         0.763 0.733          0.770 . 0.754            0.545 . 0.523 <br /> N18          0.840 0.835               0.779     0.778         0.740 0.735          0.759 0.756              0.531 0.526 <br />  <br />  <br />  <br />  <br /> 412	                                                                                         Evolutionary Bioinformatics 2013:9 <br />                                                                                        Essentiality prediction SVM <br />  <br />  <br /> Table 19. Performance comparison method vs. CMIM balanced S. cerevisiae data set identical number <br /> of features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.811 0.813           0.777   0.777      0.716 0.724            0.745 0.749            0.512 0.517 <br /> N5              0.824 0.817           0.778   0.775      0.735 0.740            0.756 0.757            0.527 0.526 <br /> N6              0.827 0.821           0.778   0.777      0.739 0.742            0.758 0.759            0.530 0.529 <br /> N7              0.831 0.830           0.779   0.772      0.733 0.744            0.755 0.758            0.526 0.524 <br /> N8              0.826 0.833           0.786   0.776      0.721 0.738            0.752 0.756            0.527 0.525 <br /> N9              0.833 0.834           0.791   0.775      0.735 0.740            0.762 0.757            0.541 0.526 <br /> N10             0.834 0.835           0.789   0.776      0.736 0.739            0.761 0.757            0.540 0.527 <br /> N11             0.831 0.836           0.784   0.778      0.737 0.739            0.760 0.758            0.535 0.528 <br /> N12             0.829 0.838           0.779   0.779      0.732 0.742            0.755 0.760            0.526 0.532 <br /> N13             0.834 0.837           0.788   0.778      0.730 0.741            0.758 0.759            0.535 0.530 <br /> N14             0.836 0.836           0.777   0.777      0.743 0.743            0.759 0.759            0.530 0.530 <br /> N15             0.843 0.836           0.784   0.777      0.748 0.739            0.766 0.758            0.542 0.528 <br /> N16             0.842 . 0.837         0.777   0.776      0.756 0.741            0.767 . 0.758          0.540 . 0.528 <br /> N17             0.847 . 0.838         0.778   0.777      0.763 . 0.744          0.770 . 0.760          0.545 . 0.531 <br /> N18             0.840 0.837           0.779   0.778      0.740 0.746            0.759 0.762            0.531 0.533 <br />  <br />  <br />  <br /> Table 20. Performance comparison method vs. mRMR imbalanced E. coli data set identical numbers <br /> of features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.691 0.651           0.725   0.678      0.280 0.269            0.404 0.385            0.391    0.363 <br /> N5              0.690 0.675           0.737   0.687      0.295 0.254            0.421 0.371            0.407    0.356 <br /> N6              0.701 0.681           0.742   0.708      0.287 0.220            0.414 0.336            0.403    0.338 <br /> N7              0.714 0.686           0.735   0.712      0.275 0.212            0.400 0.326            0.392    0.333 <br /> N8              0.705 0.692           0.742   0.713      0.288 0.209            0.415 0.323            0.405    0.330 <br /> N9              0.707 0.692           0.726   0.713      0.293 . 0.199          0.417 . 0.312          0.401    0.322 <br /> N10             0.711 0.697           0.724   0.703      0.294 . 0.193          0.418 . 0.302          0.401    0.313 <br /> N11             0.714&#160;,&#160;0.702         0.732   0.697      0.278 . 0.187          0.403 0.295            0.393    0.306 <br /> N12             0.712&#160;,&#160;0.704         0.725   0.683      0.292 0.192            0.416 0.300            0.400    0.305 <br /> N13             0.714&#160;,&#160;0.715         0.733   0.713      0.287 0.250            0.413 0.370            0.400    0.360 <br />  <br />  <br />  <br /> Table 21. Performance comparison method vs. CMIM imbalanced E. coli data set identical numbers of <br /> features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.691 . 0.663         0.725   0.717      0.280 0.271            0.404 0.393            0.391 0.381 <br /> N5              0.690 0.686           0.737   0.710      0.295 . 0.264          0.421 . 0.385          0.407 . 0.373 <br /> N6              0.701 0.697           0.742   0.715      0.287 . 0.265          0.414 . 0.387          0.403 . 0.376 <br /> N7              0.714 0.693           0.735   0.711      0.275 . 0.261          0.400 . 0.382          0.392 . 0.371 <br /> N8              0.705 0.690           0.742   0.709      0.288 . 0.254          0.415 . 0.373          0.405 . 0.364 <br /> N9              0.707 0.701           0.726   0.720      0.293 . 0.271          0.417 . 0.394          0.401 . 0.382 <br /> N10             0.711 0.702           0.724   0.692      0.294 . 0.248          0.418 . 0.364          0.401 . 0.353 <br /> N11             0.714 0.698           0.732   0.690      0.278 . 0.247          0.403 . 0.363          0.393 . 0.351 <br /> N12             0.712 0.690           0.725   0.683      0.292 . 0.239          0.416 . 0.353          0.400 . 0.342 <br /> N13             0.714 0.688           0.733   0.678      0.287 . 0.236          0.413 . 0.349          0.400 . 0.337 <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                              413 <br />  Hor et&#160;al <br />  <br />  <br /> Table 22. Performance comparison method vs. mRMR balanced E. coli data set identical numbers of <br /> features selected. <br />  <br />              AUC                     Precision               Recall              F1                       MCC <br /> N4           0.780 . 0.773           0.733   0.726           0.701 . 0.651       0.717 . 0.686            0.446 . 0.407 <br /> N5           0.779 0.772             0.730   0.720           0.706 . 0.654       0.718 . 0.684            0.445 . 0.401 <br /> N6           0.762 0.771             0.735   0.717           0.663 0.649         0.696 0.680              0.425 0.394 <br /> N7           0.783 . 0.768           0.737   0.716           0.696 0.649         0.716 . 0.680            0.448 . 0.394 <br /> N8           0.781 . 0.764           0.723   0.715           0.711 . 0.641       0.717 . 0.675            0.439 . 0.387 <br /> N9           0.782 . 0.764           0.715   0.713           0.703 0.643         0.709 . 0.675            0.423 . 0.386 <br /> N10          0.781 . 0.765           0.725   0.716           0.702 . 0.636       0.713 . 0.673            0.436 . 0.386 <br /> N11          0.777 . 0.766           0.719   0.720           0.700 0.643         0.709 . 0.678            0.426 0.394 <br /> N12          0.776 0.765             0.715   0.714           0.695 0.643         0.705 0.676              0.418 0.388 <br /> N13          0.776 . 0.762           0.731   0.728           0.695 . 0.654       0.712 . 0.689            0.439 . 0.396 <br />  <br />  <br />  <br /> 4&#160;# k #13, respectively, compliant             23, method performed significantly better <br /> in Section V.                                                   mRMR numbers selected features <br />    For S. cerevisiae data set imbalanced exper-      exceeded 7. The experimental results showed almost <br /> iment, Tables&#160;16 17&#160;show performance com-               difference method CMIM except <br /> parison method versus mRMR CMIM. Our                 N6&#160;AUC. <br /> method performed significantly better size             For methods mRMR CMIM, both <br /> a feature subset exceeded 7. For balanced experi-           relevance information redundancy taken <br /> ment, illustrated Tables&#160;18 19, method        consideration. Therefore, obtained feature <br /> significantly better number selected           subsets quite compact effective. <br /> features exceeded 16.                                           However, relevance appropriate <br />    For E. coli data set imbalanced experi-           performance measures, classifica- <br /> ment, Tables&#160;20 21&#160;show performance com-                tion accuracy precision. Our method took both <br /> parison method versus mRMR CMIM                      performance feature size consideration. <br /> with configurations similar S. cerevisiae                Consequently, resultant feature subsets were <br /> data set. Our method performed significantly better             effective performance measures <br /> when size feature subset exceeded 9. For               given equal number features precision <br /> the balanced experiment, illustrated Table&#160; 22            values. <br />  <br />  <br /> Table 23. Performance comparison method vs. CMIM balanced E. coli data set identical numbers of <br /> features selected. <br />  <br />              AUC                     Precision               Recall              F1                       MCC <br /> N4           0.780 0.769             0.733   0.719           0.701    0.696      0.717   0.707            0.446      0.424 <br /> N5           0.779 0.771             0.730   0.715           0.706    0.696      0.718   0.705            0.445      0.419 <br /> N6           0.762 , 0.771           0.735   0.716           0.663    0.684      0.696   0.699            0.425      0.413 <br /> N7           0.783 0.769             0.737   0.711           0.696    0.696      0.716   0.703            0.448      0.413 <br /> N8           0.781 0.767             0.723   0.709           0.711    0.697      0.717   0.703            0.439      0.412 <br /> N9           0.782 0.767             0.715   0.720           0.703    0.700      0.709   0.710            0.423      0.421 <br /> N10          0.781 0.767             0.725   0.705           0.702    0.702      0.713   0.704            0.436      0.409 <br /> N11          0.777 0.765             0.719   0.706           0.700    0.700      0.709   0.703            0.426      0.408 <br /> N12          0.776 0.764             0.715   0.703           0.695    0.698      0.705   0.700            0.418      0.404 <br /> N13          0.776 0.765             0.731   0.704           0.695    0.700      0.712   0.702            0.439      0.406 <br />  <br />  <br />  <br />  <br /> 414	                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                                                                  Essentiality prediction SVM <br />  <br />  <br /> Author Contributions                                                                	 6.	 Witten IH, Frank E, Hall MA. Data Mining:Practical Machine Learn- <br />                                                                                                             ing Tools Techniques Java Implementations. Burlington, MA: <br /> Conceived designed experiments: CBY, CYH.                                                           &#173;Morgann Kaufmann; 2000. <br /> Analysed data: CYH, ZJY. Wrote draft                               	 7.	 Hwang YC, Lin CC, Chang JY, Mori H, Juan HF, Huang HC. &#173;Predicting <br />                                                                                                              essential genes based network sequence analysis. Mol Biosyst. 2009; <br /> the manuscript: CYH, ZJY, CTT. Contributed                                                            5(12):1672&#8211;8. <br /> writing manuscript: CYH, CBY, ZJY. Agree                                     	 8.	 Przulj N, Wigle DA, Jurisica I. Functional topology network protein <br />                                                                                                              interactions. Bioinformatics. 1998;20(3):340&#8211;8. <br /> with manuscript results conclusions: CYH, CBY.                                  	 9.	 Chin CS, Samanta MP. Global snapshot protein interaction network <br /> Jointly developed structure arguments                                                            percolation based approach. Bioinformatics. 2003;19(18):2413&#8211;9. <br /> the paper: CYH, CBY, ZJY, CTT. Made critical revi-                                  	10.	 Barab&#225;si AL, Oltvai ZN. Network biology: understanding cell&#8217;s func- <br />                                                                                                              tional organization. Nat Rev Genet. 2004;5(2):101&#8211;13. <br /> sions approved final version: CBY. All authors                                   	11.	 Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, Sakaki Y. A comprehen- <br /> reviewed approved final manuscript.                                                                  sive two-hybrid analysis explore yeast protein interactome. Proc Natl <br />                                                                                                              Acad Sci U S A. 2001;98(8):4569&#8211;74. <br />                                                                                      	12.	 Lal TN, Chapelle O, Sch&#246;lkopf B. Feature Extraction: Foundations and <br /> Funding                                                                                                      Applications. Guyon S, Gunn M, Nikravesh M, Zadeh LA. Berlin: Springer; <br />                                                                                                              2006. <br /> This research work partially supported                                     	13.	 Peng H, Long F, Ding C. Feature selection based mutual information: <br /> National Science Council Taiwan contract                                                            criteria max-dependency, max-relevance, min-redundancy. IEEE <br /> NSC100-2221-E-110-050.                                                                                       Trans Pattern Anal Mach Intell. 2005;27(8):1226&#8211;38. <br />                                                                                        	14.	 Fleuret F. Fast binary feature selection conditional mutual information. <br />                                                                                                              Journal Machine Learning Research. 2004;5:1531&#8211;55. <br /> Competing Interests                                                                     	15.	 Sotoca JM, Pla SF. Supervised feature selection clustering using condi- <br />                                                                                                              tional mutual information-based distances. Pattern Recognition. 2010;43(6): <br /> Author(s) disclose potential conflicts interest.                                                       2068&#8211;81. <br />                                                                                          	16.	 Quinlan JR. C4.5: Programs Machine Learning. Morgan Kaufmann: <br /> Disclosures Ethics                                                                                       Burlington, MA; 1993. <br />                                                                                           	17.	 Breiman L, Friedman J, Stone CJ, Olshen RA. Classification Regres- <br /> As requirement publication authors pro-                                                        sion Trees. London: Chapman Hall/CRC; 1984. <br /> vided signed confirmation compliance                                         	18.	 Tan PN, Steinbach M, Kumar V. Introduction Data Mining. Boston, MA: <br />                                                                                                              Addison-Wesley; 2005. <br /> ethical legal obligations including limited                                  	19.	 Wang G, Ma J, Yan S. IGF-bagging: Information gain based feature selec- <br /> compliance ICMJE authorship competing inter-                                                        tion bagging. International Journal Innovative Computing, Informa- <br /> ests guidelines, article consider-                                                 tion Control. 2011;7(11):6247&#8211;59. <br />                                                                                              	20.	 Lin CY, Yang CB, Hor CY, Huang KS. Disulfide bonding state prediction <br /> ation publication published elsewhere,                                                      svm based protein types. Bio-Inspired Computing: Theories and <br /> compliance legal ethical guidelines concerning                                                      Applications. 2010:1436&#8211;42. <br />                                                                                               	21.	 Gustafson AM, Snitkin ES, Parker SC, DeLisi C, Kasif S. Towards iden- <br /> human animal research participants (if applicable),                                                      tification essential genes using targeted genome sequencing com- <br /> and permission obtained reproduction                                                       parative analysis. BMC Genomics. 2006;7:265. <br /> of copyrighted material. This article subject                                       	22.	 Hu P, Janga SC, Babu M, et&#160; al. Global functional atlas Escherichia coli <br />                                                                                                              encompassing previously uncharacterized proteins. PLoS Biol. 2009;7(4):e96. <br /> blind, independent, expert peer review. The reviewers                                          	23.	 Chang CC, Lin CJ. LIBSVM: library support vector machines. http:// <br /> reported competing interests. A preliminary version                                                       www.csie.ntu.edu.tw/~cjlin/libsvm. Updated date April 1, 2013. Accessed <br />                                                                                                              date Feb 1, 2012. <br /> of paper presented International Confer-                                        	24.	 Fu WJ, Carroll RJ, Wang S. Estimating misclassification error small sam- <br /> ence Machine Learning Applications.40                                                                 ples bootstrap cross-validation. Bioinformatics. 2005;21(9):1979&#8211;86. <br />                                                                                                  	25.	 Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, Eisenberg D. The <br />                                                                                                              database interacting proteins: 2004 update. Nucleic Acids Res. 2004;32: <br /> References                                                                                                   D449&#8211;51. <br /> 1.	 Cadigan K <br /> </body></html>