<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.13 no.24 pp.6062-6075, 1994 <br />  <br />  <br /> A novel nuclear pore protein Nup133p distinct <br /> roles poly(A)+ RNA transport nuclear pore <br /> distribution <br />  <br /> Valerie Doye, <span id='am-84' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Roger" datatype="xsd:string"></span>Roger</span> Wepf                                     (reviewed Pante Aebi, 1993, 1994). Filaments <br /> Eduard C. Hurt1                                                  protrude rings cytoplasm the <br />                                                                  nucleoplasm. On nucleoplasmic side, filaments <br /> EMBL, Meyerhofstrasse 1, Postfach 1022.09, D-69117 Heidelberg,   end distal ring form basket-like structure which, <br /> Germany                                                          amphibian oocytes, connected nuclear envelope <br /> ICorresponding author                                            lattice (Goldberg Allen, 1992). <br /> Communicated E.C.Hurt                                            In recent years, increasing number NPC com- <br />                                                                  ponents (nucleoporins) identified several <br /> Temperature-sensitive nucleoporin nup49-316 mutant               organisms. However, cloned nucleoporins account <br /> cells accumulate poly(A)+ RNA inside nucleus                 10-20% estimated mass NPCs may <br /> when shifted restrictive temperature. We performed            comprise 100 different polypeptides (Reichelt <br /> a synthetic lethal screen mutant allele             et al., 1990; Rout Blobel, 1993). The two <br /> identify components mRNA export                   nucleoporins identified <span id='am-41' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, <br /> machinery. A synthetic lethal mutant slv21 isolated,         Nuplp (Davis Fink, 1990) Nsplp (Nehrbass <br /> which exhibited ts phenotype showed nuclear                et al., 1990) share numerous degenerate &apos;FSFG&apos; peptides <br /> accumulation poly(A)+ RNA 37&#176;C. The wild-type              nucleoporins higher <br /> gene complementing slv21 cloned sequenced.               eukaryotes (Starr et al., 1990; Carmo-Fonseca et al., <br /> It encodes novel protein Nupl33p located              1991; Hallberg et al., 1993; Sukegawa Blobel, 1993; <br /> at nuclear pore complex. NUPJ33 essential          Kraemer et al., 1994; McMorrow et al., 1994) yeast <br /> gene, cells NUP133 disrupted grow                (Loeb et al., 1993). Members second nucleoporin <br /> slowly permissive temperatures stop growing            <span id='am-1' about='obo:NCBITaxon_subclass' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-2' property="rdfs:label" content="subclass" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subclass</span> exhibit repeat motif GLFG so <br /> 37&#176;C. Concomitant growth inhibition,                    far characterized yeast, monoclonal <br /> nup133- cells accumulate poly(A)+ RNA inside                 antibody mammalian nuclear pore proteins was <br /> nucleus nuclear import karyophilic                  used clone yeast nucleoporins (Wente <br /> reporter protein altered. Strikingly, nupl33-             et al., 1992; Wente Blobel, 1994). Three GLFG <br /> cells display extensive clustering nuclear pore com-          nucleoporins, Nup49p, Nup Il6p (previously designated <br /> plexes sites nuclear envelope. However,          Nsp49p Nsp l 16p) Nup l45p independently <br /> the nuclear pore clustering phenotype intranuclear           identified genetic screen synthetic lethal mutants <br /> accumulation poly(A)+ RNA obligatorily                Nsplp (Wimmer et al., 1992; Fabre et al., 1994). <br /> linked, amino-terminally truncated Nupl33p              Despite fact highly conserved among <br /> allows normal poly(A)+ RNA export, does                  nuclear pore proteins, repetitive domains not <br /> complement clustering phenotype nupl33- cells.            essential protein tested (Nehrbass et al., 1990; Fabre <br /> Key words: nuclear envelope/nuclear pore complex/                et al., 1994). On hand, type repeat <br /> nucleocytoplasmic transport/RNA export/yeast                     sequence obligatory feature NPC proteins <br />                                                                  nuclear pore proteins lacking obvious repeat <br />                                                                  motifs higher eukaryotes (Greber <br />                                                                  et al., 1990; Wozniak Blobel, 1992; Radu et al., 1993) <br /> Introduction                                                     yeast (Grandi et al., 1993; Wozniak et al., <br /> In eukaryotic cells, nuclear envelope acts barrier      1994). Due recent isolation highly purified yeast <br /> between nucleus cytoplasm. Bidirectional             NPCs (Rout Blobel, 1993), development genetic <br /> traffic molecules compartments              screens, yeast genome project, yeast nucleo- <br /> can, however, occur nuclear pore complexes           porins probably cloned sequenced a <br /> (NPCs) elaborate structures embedded            short time. <br /> nuclear envelope. While small molecules diffuse                 Despite increasing number cloned nucleoporins, <br /> through nuclear pore, import proteins snRNPs          precise function remains unknown. The transmem- <br /> into, export different classes RNA of,         brane proteins gp2 10, POM 121 POM 152 supposed <br /> nucleus energy-dependent saturable processes        anchor NPCs nuclear envelope (Greber <br /> require specific signals transported substrate (for       et al., 1990; Wozniak Blobel, 1992; Hallberg et al., <br /> reviews Goldfarb Michaud, 1991; Forbes, 1992;            1993; Wozniak et al., 1994), zinc finger <br /> Bossie Silver, 1992; Izaurralde Mattaj, 1992; Fabre      motif-containing Nupl53p (Sukegawa Blobel, 1993), <br /> and Hurt, 1994). As revealed numerous ultrastructural         localized terminal ring nuclear <br /> studies, NPCs composed cytoplasmic            basket (Pante Aebi, 1993), proposed link the <br /> nuclear ring octagonal symmetry, spokes           nuclear pores chromatin. p62 yeast homologue, <br /> a central structure refered plug transporter            Nsplp, proposed play role nuclear pore <br />  <br />   6 Oxford University Press <br /> 6062 <br />                                                                                     Poly(A)+ RNA distribution NPC clustering <br />  biogenesis (Dabauvalle et al., 1990; Mutvei et al., 1992).                                   -1   c   3WC       37CC <br />  However, nucleoporins shown be <br />  involved nuclear import NLS-containing reporter <br />  proteins vivo vitro (Finlay et al., 1991;                     NIiP49 <br />                                                                                                         ~**       r* <br />  <br />  Nehrbass et al., 1993; Schlenstedt et al., 1993). On the <br />  hand, nupi 16 null mutants shifted 37&#176;C, well <br />  Nupl45p-depleted cells, accumulate poly(A)+ RNA                           nup49-3 13 <br />  inside nucleus (Wente Blobel, 1993; Fabre et al., <br />  1994), mRNA export defect nup 116 mutant                                                  4w <br />  proposed indirect consequence membrane                       nup49.316 <br />  sealing NPCs (Wente Blobel, 1993). <br />     Although export different RNA <span id='am-73' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> involve <br />  common pathway exit NPC, earlier steps       Fig. 1. Growth properties ts nup49 mutant strains. Equivalent <br />  process appear mediated factors specific     cells wild-type (NUP49), nup49-313 nup49-316 <br />  <span id='am-65' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-66' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> RNA (Jarmolowski et al., 1994). Indeed,     cells spotted YPD plates incubated 24, 30 37&#176;C. <br />  specific factors possibly involved export   nup49-313 cells defective growth 30&#176;C not <br />  5S rRNA (Guddat et al., 1990), tRNA (Singh Green,         viable 37&#176;C, nup49-316 cells display weaker temperature- <br />                                                                sensitive   phenotype. <br />  1993; Zasloff, 1983), polymerase II transcripts <br />  (Izaurralde et al., 1992; Pifiol-Roma Dreyfuss, 1992)     growing shifted 37&#176;C, nup49-316 cells <br />  characterized higher eukaryotes. To identify     able grow temperature, a <br /> components involved mRNA export yeast, collections       slow rate (Figure 1). Both nup49 mutant alleles encode <br /> of temperature-sensitive mutants screened reten-      proteins exhibit amino acid substitu- <br /> tion poly(A)+ RNA inside nucleus non-            tion compared wild-type Nup49p (see Materials <br /> permissive temperature. These screens yielded mtr          methods), systematic attempt to <br /> (mRNA transport; Kadowaki et al., 1992) rat             determine mutations contribute ts phenotype. <br /> (ribonucleic acid trafficking; Amberg et al., 1992) mutants.      In <span id='am-76' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-77' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> analyse nuclear protein import two <br /> The MTRI gene (identical PRP20 SRMI) encodes             temperature-sensitive nup49 mutant strains, cells were <br /> the S.cerevisiae homologue guanine nucleotide           transformed plasmid directing expression of <br /> release protein (GNRP) interacts homologues      well-defined nuclear reporter protein, Mata2-lacZ <br /> of small GTPase Ran S.cerevisiae, proposed       (Nehrbass et al., 1993; Hall et al., 1984). The subcellular <br /> to directly involved mRNA export (Amberg et al.,         distribution reporter protein examined by <br />  1993; Kadowaki et al., 1993). Interestingly, RATIO        indirect immunofluorescence permissive versus non- <br /> gene encodes nucleoporin Nup 145p (A.Goldstein,            permissive temperature. In nup49-313 cells grown the <br /> T.C.Dockendorff C.N.Cole, personal communication)          permissive temperature (24&#176;C), Matca2-lacZ fusion <br /> that independently shown affect mRNA export             protein correctly localized inside nucleus; in <br /> (Fabre et al., 1994).                                          contrast, nup49-313 cells shifted 5 h 37&#176;C, the <br />    In study, analysed temperature-sensitive nup49      reporter protein mislocalized cytoplasm (Figure <br /> strains defects nucleocytoplasmic transport. One     2). <span id='am-68' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> vivo inhibition nuclear uptake accordance <br /> these mutant strains, nup49-316, accumulates poly(A)+          vitro <span id='am-89' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-90'  ></span><span id='am-91' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown nuclear protein <br /> RNA inside nucleus restrictive temperature. As       import defective semi-permeabilized nup49-313 <br /> alternative approach identify components RNA         cells (Schlenstedt et al., 1993). On hand, the <br /> export machinery, set synthetic lethal screen          Matx2 - lacZ fusion protein accumulated the <br /> based nup49 mutant allele. This genetic screen         nucleus nup49-316 cells shifted 5 h restrictive <br /> allowed cloning characterization novel nucleo-    temperature (Figure 2). <br /> porin, Nupl33p. Interestingly, mutations Nupl33p lead          To complete study, analysed subcellular <br /> to nuclear accumulation poly(A)+ RNA and/or altered         location poly(A)+ RNA nup49 mutant strains <br /> nuclear pore distribution.                                     situ hybridization using fluorescein isothiocyanate <br />                                                                (FITC)-labeled oligo (dT)50. In nup49-313 nup49-316 <br /> Results                                                        cells grown permissive temperature, poly(A)+ RNA <br />                                                                exported and, accordingly, gives rise cytoplasmic <br /> Two mutant alleles nucleoporin Nup49p                   signal. nup49-316 cells shifted 5 h 37&#176;C accumulated <br /> differently affect nuclear import proteins              poly(A)+ RNA inside nucleus, suggesting the <br /> poly(A)+ RNA distribution                                      nup49-316 allele primarily cause defect mRNA <br /> Nup49p previously shown member                export (see Discussion). On hand, nup49-313 <br /> nuclear pore subcomplex contains Nsplp, Nic96p,           cells shifted 8 h 37&#176;C showed pronounced <br /> Nup57p p80 component (Grandi et al., 1993, 1995).        poly(A)+ RNA signal inside nucleus did nup49- <br /> To analyse role Nup49p nucleoporin          316 cells (Figure 3). <br /> complex, conditionally lethal nup49 alleles isolated <br /> and tested defects nucleocytoplasmic transport.         A synthetic lethal screen using nup49-316 <br /> Several thermo-sensitive alleles NUP49 obtained        allele identifies Nup133p novel nuclear pore <br /> by PCR-directed mutagenesis carboxy-terminal            protein <br /> domain Nup49p. Two these, nup49-313 nup49-           So far, synthetic lethality proven powerful <br /> 316, characterized: nup49-313 cells stopped       approach components nuclear pore complex <br />  <br />                                                                                                                             6063 <br />  V.Doye, R.Wepf E.C.Hurt <br />  <br />                          n up49-31 3 <br />                                            ail ti-I ic /                  1)NAk                      \Noilla rs ki <br />  <br />                                                                                              K                 4 <br />  <br />  <br />                           24; C_ <br />  <br />  <br />  <br />                                                               I <br />                           3 7 Ci                                                               kl- <br />  <br />  <br />  <br />  <br />                                                               I <br />                          nup49-316 <br />                                            ti- L .c-CZ                 DNI\                       aNornaKllski <br />                                            4 <br />                                                                                                        .- <br />  <br />  <br />  <br />                                                                                                      _N&apos; <br />                           24 C <br />                                                                                                        i5            .s <br />  <br />  <br />  <br />  <br />                                                                                                       ).\ <br />                           37 C        U <br />  <br />  <br />  <br />  <br />                                                                   I <br /> Fig. 2. Nuclear import Mata2-LacZ reporter protein inhibited temperature-sensitive nup49-313 nup49-316 strain. The <br /> subcellular distribution Mata2-LacZ reporter analysed indirect immunofluorescence nup49-313 cells (top) nup49-316 cells <br /> (bottom) grown 24&#176;C shifted 5 h 37&#176;C. Cells stained DNA using Hoechst 33258 viewed Nomarski optics. <br />  <br /> which functionally overlap physically interact                      obtained screen. These strains fulfilled all <br /> Nsplp (Wimmer et al., 1992; Fabre et al., 1994; Grandi                      criteria synthetic lethal mutants NUP49 (see <br /> et al., 1995). To identify components possibly involved                     Materials methods). However, mutants <br /> in mRNA export, screened mutations lethal                   displayed allele-specificity nup49-316 mutant <br /> in genetic background nup49-316 allele. For                    allele, synthetic lethal the <br /> genetic screen, red/white colony sectoring assay based                  nup49-313 mutant allele (V.Doye, unpublished data). <br /> on ade2/ade3 mutations adopted (see Materials                     One mutant strains, slv21, further <br /> and methods). The screening strain, called VD 1 RW316,                      characterized exhibited thermo-sensitive <br /> was ade2/ade3, carried disrupted nup49::TRPI gene                     growth defect 37&#176;C (Figure 4) indicating the <br /> was complemented plasmid-borne wild-type NUP49                           mutation cause thermosensitivity presence <br /> and mutant nup49-316 alleles (pCH1 122-ADE3-URA3-                           wild-type NUP49 (see below). slv2l cells accumu- <br /> NUP49 pUN 100-LEU2-nup49-316; Table I). If                          lated poly(A)+ RNA inside nucleus shifted to <br /> grown 32&#176;C, screening strain displayed distinct                   37&#176;C (data shown). To clone wild-type gene which <br /> red/white sectoring phenotype cells afford                     caused synthetic lethality slv2 1, mutant strain was <br /> lose pCH 1122-ADE3-URA3-NUP49 plasmid                               transformed yeast genomic library inserted into <br /> grow pUN100-LEU2-nup49-316. After UV muta-                             single copy ARS/CEN plasmid. Four 5000 <br /> genesis, non-sectoring red mutants, longer                   transformants regained red/white sectoring phenotype. <br /> could lose plasmid carrying wild-type NUP49,                       Of re-isolated <span id='am-94' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>, contained NUP49 and <br />  <br /> 6064 <br />                                                                                                            Poly(A)+   RNA distribution NPC       clustering <br />   \ L&apos;IP49                                                                        n1 Li p49-3 &apos;1 <br />                 O Iii._ f Li             D\E.-N                                                                                            \ ma rx    ki <br />  <br />  <br />    24 C                                                                            24 (. <br />                                                               I       I <br />                                                                                                                                            4             W <br />  <br />                                                         -)4    ii                                                                              40P:Nf# <br />                                                               JJ <br />                                                               j <br />  <br />                                                                   .   . <br />                                                                           4IJ <br />  <br />  <br />    3:7 C                                                                  ,L..      .,7C                                               -Aii.s <br />                                                                                                                                         ) <br />  <br />  <br />  <br />  <br />                                                                                   n u p49- 1 {i <br />                                                                                                    *,   ;fl.,;I <br />  <br />  <br />                                                                                    24 <br />  <br />  <br />  <br />                                                                                                                                       15             4 <br />                                                                                                                                                40 <br />                                                                                                                                                 r            q <br />                                                                                                                                                                  % <br /> Fig. 3. Temperature-sensitive nup49 mutant cells accumulate poly(A)+                                                                  I             ,! .f <br /> RNA inside nucleus restrictive temperature. Wild-type <br /> (NUP49), nup49-313 nup49-316 cells grown overnight at <br /> 24&#176;C shifted 37&#176;C 8 h (NUP49 nup49-313) 5 h <br /> (nup49-316). Subcellular localization poly(A)+ RNA analysed <br /> by situ hybridization FITC-labeled oligo(dT) probe. At 37&#176;C, <br /> nup49-316 and, lesser extent, nup49-313 mutants, accumulate                                                                       j                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.1 <br /> poly(A)+ RNA inside nucleus stained Hoechst <br /> 33258. <br />  <br />  <br />  <br /> Table I. Yeast strains <br /> Strain                     Genotype <br /> RS453                      Mata/a, ade2/ade2, his3/his3, leu2/Ieu2, trpl/trpl, ura3/ura3 <br /> CH1305                     Mata, ade2, ade3, leu2, lvs2, ura3 <br /> VDICH                      Mataz, ade2, his3, leu2, trpl, ura3, nup49::TRPI (pCH1 122-ADE3-URA3-NUP49) <br /> VDIR                       Maita, ade2, ade3, his3, leu2, Ivs2, ura3, tzup49::TRPI (pCH1 122-ADE3-URA3-NUP49) <br /> VDIRW316                   Matca, ade2, ade3, his3, leu2, IYs2, urO3, nup49::TRPI (pCH1 122-ADE3-URA3-NUP49, pUNI00-nup49-316) <br /> NUP49                      Mata, ade2, ade3, his3, leu2, 1vs2, ura3, nup49::TRPI (pUN90-NUP49) <br /> nup49-313                  Mata, ade2, ade3, his3, leu2, lYs2, ura3, nup49::TRPI (pUN90-nup49-313) <br /> nup49-316                  Mata, ade2, ade3, his3, leu2, lvs2, ura3, nup49::TRPI (pUN90-nup49-316) <br /> slv2l                      Mata, ade2, ade3, his3, leu2, lYs2, urca3, nup49::TRPl nupl33-1 (pCHI 122-ADE3-URA3-NUP49) [pUN90-nup49-316] <br /> s1248                      Mata, ade2, ade3, leu2, trpl, ura3, nspl::HIS3, nupI33-2 (pCHI 122-ADE3-URA3-NSPI) [pRS414-TRPl-nspl&apos;s(L640S)] <br /> WT                         Mata, ade2, his3, leu2, trpl, ura3 (haploid derivative RS453) <br /> nupl33-                    Mata, ade2, his3, leu2, trpl, ura3, nupl33::HIS3 <br /> ProtA-NUP133               Mata, ade2, his3, leu2, trpl, ura3, nupl33::HIS3 (pUNIOO-LEU2-ProtA-NUP133) <br /> ProtA-nup I33-AN           Mata, ade2, his3, leu2, trpl, ura3, nupl33::HIS3 (pUNIOO-LEU2-ProtA-nupl33-AN) <br /> ProtA-nup 133AM            Matn, ade2, his3, leu2, trpl, ura3, nupl33::HlS3 (pUNIOO-LEU2-ProtA-nupl33-AM) <br />  <br />  <br /> one  carried unrelated 8 kb genomic DNA fragment.                             kDa, designated Nupl33p (see below). Analysis charge <br /> Subsequently, shorter 3.7 kb BamHI-BgIII restriction                           hydrophobicity distribution Nup l33p <br /> fragment shown contain complementing activity                         sequence did reveal specific features two <br /> both synthetic lethal thermo-sensitive pheno-                        hydrophobic stretches (residues 413-429 641-658) <br /> type slv2 1 (Figure 4). While cloned DNA fragment                        randomly distributed acidic residues. Comparison of <br /> could complement seven remaining mutants                          ORF sequences present EMBL database <br /> that synthetic lethal combination nup49-316,                        did reveal striking homology known proteins. In <br /> it restored red/white sectoring mutant previously                           particular, amino acid sequence Nupl33p does not <br /> found synthetic lethal screen ts tispi (s1248;                         contain &apos;FSFG&apos; &apos;GLFG&apos; repeats. However, during <br /> H.Tekotte, unpublished data). This shows novel                         course work, S.cerevisiae chromo- <br /> gene functionally interacts NUP49 NSPI.                            XI DNA sequence released contained <br />    Sequence analysis complementing 3.7 kb DNA                             DNA sequence (ORF YRK402) 100% identical the <br /> restriction fragment revealed single open reading frame                        sequence NUP133 gene (<span id='am-36' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Garcia" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span>Garcia</span>-Cantalejo et al., <br /> of 1157 amino acids encodes acidic protein 133                        1994). <br />  <br />                                                                                                                                                          6065 <br />  V.Doye, R.Wepf E.C.Hurt <br />  <br />                                                                       1989)], signals overlapped (data shown). This <br />                                                                       shows ProtA-Nupl33p co-localizes other <br />                                          24   24-   377C              nuclear pore proteins nuclear envelope. <br />                                                                       Intranuclear accumulation poly(A)+ RNA and <br />        11L11 1033                                                     nuclear pore clustering consequence the <br />                                                                       nup133 gene disruption <br />                                                                       Mutant nup49-316, synthetic lethal strain <br />             L+       LN                                             slv21 carrying nupJ33-1 allele, display nuclear accu- <br />                  p <br />                           10 <br />                      lINl I00- NLUP133 <br />                                                                       mulation poly(A)+ RNA. We analysed <br />                                                                       poly(A)+ distribution nup 133- strain. When <br />                                                                       nupi 33- cells grown 18&#176;C, temperature which <br /> Fig. 4. The nupJ33-1 mutant allele causes synthetic lethality    growth defect observed compared wild-type <br /> nup49-316 exhibits temperature-sensitive growth inhibition   cells, poly(A)+ RNA exported normally the <br /> 37&#176;C. The growth properties strain VDIRW316 (nup49::TRPJ,          cytoplasm (Figure 6A). Surprisingly, nuclear accumulation <br /> pUN90-nup49-3 16, pCH I 122-ADE3-URA3-NUP49) carrying wild- <br /> type NUP133 allele (NUP133+) derivative strain slv2l          poly(A)+ RNA seen cells were <br /> carrying nupJ33-1 allele transformed plasmid pUNIOO       grown 24&#176;C (Figure 6A). On hand, growth is <br /> pUNIOO-NUP133 compared. Equivalent cells          slowed permissive temperature (see also <br /> spotted <span id='am-4' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-5' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>-containing plates YPD plates      Figure SA). The nuclear poly(A)+ RNA accumulation was <br /> incubated 4 days 24&#176;C 37&#176;C. Both synthetic lethal <br /> phenotype nupJ33-1 nup49-316 alleles (followed     enhanced cells shifted 5 h 37&#176;C <br /> the death FOA-containing plates) thermo-sensitive          (Figure 6A). This suggests growth defects of <br /> phenotype conferred nup133-1 mutant allele (followed    nupl33- cells inhibition mRNA export. <br /> death YPD 37&#176;C) complemented transformation     In contrast, cytoplasmic mislocalization nuclear <br /> pUNIOO-NUP133 plasmid. <br />                                                                       reporter protein Mata2-lacZ observed nupl33 <br />                                                                       null strain, independently cells grown at <br />    To determine NUP133 essential cell                  23&#176;C shifted 5 h 37&#176;C (Figure 6B). This shows <br /> growth, NUP133 gene disrupted diploid yeast              NUP133 gene disruption does alter competence <br /> strain (see Materials methods). After sporulation              nuclear pores nuclear reporter protein. <br /> the heterozygous nupJ33::HIS31NUP133 diploid strain                      It previously reported depletion yeast cells <br /> and tetrad analysis, viable spores recovered            Nsp lp (Mutvei et al., 1992) Nup ll6p (Wente and <br /> of nupl33::HIS3 containing progeny                      Blobel, 1993) mutations NUP145 gene <br /> (further designated nupl 33-) grew slower              (Wente Blobel, 1994; A.Goldstein, T.C.Dockendorff <br /> the NUP133+ progeny. The doubling time nupl33-                     C.N.Cole, personal communication), respectively, <br /> cells grown YPD 30&#176;C 4.8 h, compared                   cause decrease nuclear pore number, generate <br /> 2.0 h wild-type strain. If nupl33- cells                 nuclear pores sealed double membrane, lead to <br /> shifted 37&#176;C undergo divisions               nuclear pore clustering. We interested to <br /> before cell growth stopped completely (Figure 5A). This               biogenesis and/or structural organization <br /> temperature-sensitive phenotype nupl33- strain              nuclear pore complexes abnormal nupl33- <br /> fully rescued transformation plasmid containing             strain. For purpose, immunolocalization other <br /> wild-type NUP133 (data shown).                                    nuclear pore proteins analysed wild-type and <br />    In order localize Nupl33p cell, protein              nupl33- cells grown 24&#176;C using monoclonal <br /> was tagged amino-terminal domain IgG                  mAb414 antibody (Aris Blobel, 1989). Whereas in <br /> binding domains <span id='am-9' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span>Staphylococcus</span> aureus protein A,               wild-type cells mAb414 stains nuclear envelope a <br /> the fusion gene expressed nupl33- strain.                  ring-like manner, immunostaining nupl33- cells was <br /> ProtA-Nupl33p functional comple-                   longer ring-like nucleoporin antigens were <br /> ment temperature-sensitive phenotype nupl33-               clustered regions nuclear <br /> strain (Figure SA) derived phenotypic                  envelope (Figure 7A). <br /> defects. Analysis cell extract prepared                  To exclude certain nuclear pore antigens, <br /> the ProtA-Nupl33p expressing strain SDS-PAGE                       nuclear pore complexes cluster nup 133- <br /> and Western blotting revealed major band migrating               cells, nuclear pore distribution directly analysed by <br /> 150 kDa plus degradation products (Figure SA).                freeze-fracture <span id='am-79' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-83'  ></span>electron</span> microscopy. In wild-type yeast <br /> This apparent molecular weight good agreement              cells, nuclear pores normally distributed entire <br /> the calculated molecular mass 133 kDa Nupl33p                  nuclear envelope little tendency cluster. The <br /> and 14 kDa protein A moiety.                                  average pore density NPCs previously determined <br />    The subcellular location tagged Nup l33p deter-             10-15 pores/gm2 nuclear surface (Moor and <br /> mined indirect immunofluorescence. A punctuate stain-              MUhlethaler, 1963; Mutvei et al., 1992). Both distribu- <br /> ing nuclear envelope previously shown              tion density NPCs similar ts nup49- <br /> be typical nuclear pore labelling seen                  313 mutant strain grown 24&#176;C (Figure 7B). In contrast, <br /> ProtA-Nupl33p (Figure 5C). When double immuno-                        NPCs extensively clustered nupl33- cells grown at <br /> fluorescence staining performed ProtA-                   24&#176;C density high 60 pores/gim2. <br /> Nupl33p known yeast nucleoporins [using mono-                 As consequence clustering, large areas the <br /> clonal mAb414 antibody recognizes yeast nucleo-                  nuclear envelope devoid NPCs (Figure 7B). <br /> porins including Nsplp, Nuplp, Nup2p (Afis Blobel,                   Since freeze-fracture electron microscopy cannot <br />  <br /> 6066 <br />                                                                                                                  Poly(A)+ RNA distribution NPC clustering <br />  <br />  <br />                    A                                                                   11*1   B <br />                                                                     -L   ZN <br />                                                                                  1-1 <br />                                      Is <br />                                       ,,G - %,. <br />                                        ,$                                     .111 <br />                                              I             I   :. <br />  <br />  <br />  <br />                                zia:. !&quot;, <br />                    37 C     k&apos;`;24&apos;LOM. <br />                             wiip7r:qw <br />  <br />  <br />  <br />                                                     o &gt;. <br />  <br />  <br />  <br />  <br />                    C&apos; <br />                             Pio)t-         -N upl31331) <br />                                                   p)                                    l)N&apos;A <br />  <br />  <br />  <br />  <br /> Fig. 5. Expression localization Nupl33p tagged proteini A (A) Complementation ts phenotype nupl33- strain. Cells from <br /> wild-type (WT), nupl33-, ProtA-NUP133, ProtA-nup133-AN ProtA-nup133-AM strains spotted YPD plates incubated 4 <br /> days 24&#176;C 37&#176;C. To follow expression fusion proteins, cell extracts ProtA-Nupl33, ProtA-nupl33-AN and <br /> ProtA-nupl33-AM strains analysed SDS-PAGE immunoblotting using IgG-HRP conjugate. The position fusion proteins is <br /> indicated dot. (B) Amino acid sequence Nupl33p protein. The accession number NUPJ33 DNA sequence X80066. The amino- <br /> terminal domain (residues 44-246) middle domain (residues 592-926), deleted ProtA-nupl33-ANp ProtA-nupl33-AMp, respectively, <br /> are boxed. (C) The subcellular localization ProtA-Nupl33p nupI33- cells analysed indirect immunofluorescence using rabbit anti- <br /> chicken IgG followed goat anti-rabbit IgG coupled Texas red. A punctuate nuclear envelope staining typical nucleoporins observed. Cells <br /> were stained DNA Hoechst 33258 viewed Nomarski optics. <br />  <br />  <br />  <br />  <br /> reveal structural alterations nuclear envelope,                                        However, cases (5-10% sectioned nuclei), <br /> nupl33- cells grown 24&#176;C analysed                                              aberrant nuclear envelope structures containing NPC were <br /> thin section electron microscopy. Whereas wild-type                                            seen, reminiscent grape-like NPC clusters displayed in <br /> cells NPC randomly distributed nuclear                                            nupl45AN mutant cells (Wente Blobel, 1994; Figure <br /> envelope tendency clustering (Figure 8A),                                         8C, D F). Finally, rarer cases, intranuclear <br /> nuclear membrane sectioned nup133- cells                                              paired membranes containing pores observed; these <br /> often devoid NPCs (in 50% examined                                                correspond invaginations nuclear envelope or <br /> pictures; Figure 8C, left cell) or, nuclear pores                                         intranuclear annulate lamellae eukaryotic <br /> seen, clustered site nuclear                                       cells (reviewed Kessel, 1983) NUP116- <br /> envelope (Figure 8B-F). NPC clustering normally                                               disrupted yeast cells (Wente Blobel, 1993). This <br /> not associated altered nuclear envelope structure.                                        ultrastructural analysis nuclear pore clusters in <br />  <br />                                                                                                                                                        6067 <br />  V.Doye, R.Wepf E.C.Hurt <br />  <br />  A              ol   &apos;&apos;14wdl             E.)   1--.                   c.m ars I,                                                   N oinars <br />                                                                                                                                           l&apos;s ki <br />  <br />  <br />  <br />  <br />   &apos;IS C <br />                                                                                                    24 C <br />  <br />  <br />                                                                                                                                             I%; <br />                                                                  iK <br />                                                                                                                                  .. <br />  24 C <br />  <br />  <br />  <br />  <br />      C <br />                                  I                                         f.I <br />                                                                           el             f <br />                                                                                                    ;- C <br />                                                                                                                                  *7&apos; <br />                                                                                                                                   i,,. <br />                                                                                                                                                    C&apos; <br />  <br />                                                          I   I <br />  <br />                                                                  .r                          .-_ <br />  <br />  <br />                                                                       ,             f <br />                                                                                                                                        Th <br />                                                                       /S <br />                                                                                fJ <br />                                                                                                                                 V4                 :% <br />  <br /> Fig. 6. nupl33- cells accumulate poly(A)+ RNA inside nucleus. (A) nupl33- cells grown overnight 18&#176;C 24&#176;C pre-grown <br /> overnight 24&#176;C shifted 5 h 37&#176;C. Subcellular localization poly(A)+ RNA analysed situ hybridization FITC- <br /> labeled oligo(dT) probe. Nuclear accumulation poly(A)+ RNA visible 24&#176;C, enhanced 37&#176;C. (B) Subcellular localization of <br /> the Matax2-LacZ reporter protein analysed indirect immunofluorescence nupl33- cells grown 24&#176;C shifted 5 h 37&#176;C. At both <br /> temperatures, reporter protein accumulates nucleus. <br />  <br /> nupl33- cells revealed NPC clusters                                shows large deletion middle domain of <br /> display strict location respect bud neck                               Nup l33p leads non-functional protein. Thus, essential <br /> SPB (Figure 8C D).                                                                  functional elements Nupl33p, possibly involved in <br />                                                                                         targeting NPC, located middle domain. <br /> Intranuclear accumulation poly(A)+ RNA                                              Conversely, ProtA-nupl33-ANp able comple- <br /> nuclear pore clustering linked nup 133                                       ment growth defect nupi 33- strain 24&#176;C <br /> mutant strains                                                                          and, lesser extent, 37&#176;C (Figure 5A). Concomitantly, <br /> The fact nupl33- cells grown 18&#176;C showed                                     defect causing nuclear accumulation poly(A)+ RNA <br /> nuclear accumulation poly(A)+ RNA,                                     export largely rescued mutant allele (Figure <br /> nuclear pores clustered (data shown),                            10). When analysed nuclear pore distribution, the <br /> indicated functional structural defects                              ProtA-Nupl33-AN strain showed pattern of <br /> not strictly coupled. In order determine                               clustered NPCs, means structural defect <br /> Nup l33p required normal NPC distribution,                                     complemented ProtA-Nupl33-ANp. However, <br /> deletions Nup l33p, mutant proteins                                ProtA-Nup 1 33-ANp itself correctly targeted because <br /> were tested capability complement pheno-                               localized NPC clusters labelled mAb4l4 <br /> typic defects nupl33- strain (Figure 5B). Since                                  (Figure 9). This shows amino-terminal of <br /> analysis Nupl33p sequence revealed specific                                   Nup l33p required nuclear pore targeting, is <br /> structural domains, deletions performed arbitrarily,                               necessary keeping NPCs randomly distributed <br /> exploiting convenient restrictions sites NUP133                              nuclear membrane. Furthermore, experi- <br /> gene. In case, residues 592-926 middle                                   ment suggests intranuclear accumulation poly(A)+ <br /> part Nupl33p (nupl33-AM) deleted,                                       RNA nupl33- strains simply abnormal <br /> removing hydrophobic stretches. The                                      distribution nuclear pore complexes. <br /> second mutation deletion amino-terminal <br /> part Nupl33p residue 44 236 (nup 133-AN). To                                 Discussion <br /> allow detection, truncated proteins tagged <br /> with protein A similar way wild-type Nupl33p.                                   Characterization nup49 mutant alleles revealed <br /> The fusion proteins migrated SDS-polyacrylamide                                  different phenotypic defects nucleocytoplasmic trans- <br /> gels expected molecular size expressed                             port. The stronger nup49-313 allele leads primarily an <br /> a level similar wild-type ProtA-Nupl33p (Figure 5A).                                 inhibition nuclear protein import, weaker <br />    Unlike ProtA-Nupl33p, ProtA-nupl33-AMp did                                       nup49-316 allele causes accumulation poly(A)+ RNA <br /> rescue temperature-sensitive phenotype nup 133 -                             inside nucleus. While data suggest mRNA <br /> strain (Figure 5A) longer targeted                                    export inhibited nup49-316 mutant cells, cannot <br /> nuclear envelope (Figure 9). Accordingly, did                                    far exclude poly(A)+ RNA accumulation nup49- <br /> complement mRNA export defect (data                                      316 mutant cells result impairment in <br /> shown) nuclear pore clustering (Figure 9). This                                  mRNA synthesis, processing stability. The allele- <br />  <br /> 6068 <br />                                                                                                  Poly(A)+ RNA distribution NPC clustering <br />  <br />  <br />  <br />                   A <br />  <br />                   mAb4l4 <br />  <br />  <br />  <br />  <br />                    DNA + <br />                   Nornarski <br />  <br />  <br />  <br />  <br />     B                         11        11)-4   - <br />                                                     1   . <br />  <br />  <br />  <br />  <br />                                    tAp <br />                                    .s <br />                                                                                                                                      .IN    .... <br />                                                                                                                                               ..  . <br />                                    0                                                                                                       N. <br />                                                                                                                                            3&apos;    -\ <br />  <br />  <br />  <br />  <br />                                                               .. <br />  <br />  <br />  <br />  <br />                                                                                                                       I.,   -   -4 <br />  <br />  <br />                                                                        P. <br />              -i <br />                                                                        t. <br />                                                                         V\ <br />        I..                                                                        *1 <br />         e,SI <br />  <br />                   &quot;as, <br />                          f. <br />                   f <br />  <br />  <br />  <br />  <br />                          k. <br />  <br />  <br />  <br />  <br />                                                                                                                                             oft.   -1 &apos; &quot;p <br />  <br /> Fig. 7. NUP133 gene disruption induces nuclear pore clustering. (A) Distribution nuclear pore antigens analysed indirect <br /> immunofluorescence using inonoclonal antibody mAb414. In wild-type cells, typical ring-like staining nuclear envelope observed, <br /> whereas nupl33- cells grown early logarithmic phase 24&#176;C, anti-nucleoporin immunostaining concentrated foci the <br /> nuclear periphery. The coincident Nomarski optics view shown. (B) Nuclear pore distribution analysed freeze-fracture electron <br /> microscopy nup49-313 (left panel) nupl33- cells (right panels) grown early logarithmic phase 24&#176;C. In nup49-313 mutant cells, nuclear <br /> pores randomly distributed entire nuclear envelope. In contrast, nuclear pores clustered nupl33- cells accordingly, extensive <br /> areas nuclear envelope devoid NPC. <br />  <br />  <br />                                                                                                                                                     6069 <br />  V.Doye, R.Wepf E.C.Hurt <br />  <br />  <br />                         A                                                   B                Ai           r <br />                                     44. <br />                                                    p <br />  <br />  <br />  <br />                                           n                                                       n <br />  <br />  <br />                                 A <br />  <br />  <br />  <br />                          C <br />                                                                                                            n <br />                                     n <br />  <br />  <br />                          D                                              E <br />                                               I         s <br />                                                                                                       n <br />  <br />  <br />  <br />                          F <br />  <br />  <br />  <br />  <br /> Fig. 8. Thin-section electron-microscopic analysis nupl33- cells. Wild-type (A) nupI33- (B-F) cells grown 24&#176;C early <br /> logarithmic phase processed section EM. Arrowheads indicate nuclear pore complexes big arrows point nuclear pore <br /> clusters associated abnormal nuclear envelope structure. (n): nucleus; (s): spindle pole body. Bars: (A-E) 0.5 ,um, (F) 0.25 gm. <br />  <br /> specific phenotypic defects observed nup49 mutants                   transport pathway. Functional characterization other <br /> could suggest Nup49p functionally overlaps physic-                  nucleoporins (Nsplp, Nic96p Nup57p) build up <br /> ally interacts distinct nucleocytoplasmic transport                    complex required help understand more <br /> machineries. Alternatively, Nup49p forms NPC                        precisely role Nup49p complex. <br /> subcomplex Nsplp, Nic96p Nup57p                              Through synthetic lethal screen, based nup49- <br /> (Grandi et al., 1993, 1995), various members                     316 mutant allele, novel nuclear pore protein, Nupl33p <br /> complex involved different nucleocytoplasmic                      identified. Unlike yeast nucleoporins <br /> transport processes. Mutations component (e.g.                       cloned far, Nupl33p does contain characteristic <br /> Nup49p) inhibit organization subcom-                  FSFG/GLFG repeat sequences known sequence <br /> plex indirectly interfere                       motifs. Since nupJ33-1 allele carried slv2l <br />  <br /> 6070 <br />                                                                                               Poly(A)+ RNA distribution NPC clustering <br />  <br />  <br />                                                  I         .. <br />                                                            ;- <br />                                                                 1p 33-`4 i1 <br />  <br />  <br />                          nti-ProtA <br />  <br />  <br />  <br />  <br />                            nAb4l4 <br />  <br />  <br />  <br />  <br />                            DNA      + <br />                          \ ornarski <br />                                                                                                                  N <br />  <br />  <br />  <br />  <br /> Fig. 9. Unlike ProtA-nupl33-AMp, ProtA-nupl33-ANp localized nuclear pore clusters. ProtA-nupl33-AM ProtA-nupl33-AN <br /> strains double labelled rabbit anti-chicken IgG detect protein A moiety fusion proteins anti-nucleoporin mAb4l4 <br /> antibody. ProtA-nupl33-AMp mislocalized cells, ProtA-nupl33-ANp co-localizes mAb414-reactive nucleoporins within <br /> the NPC clusters. Cells stained Hoechst 33258 viewed Nomarski optics. <br />  <br />  <br /> strain, disrupted nupI33::HIS3 allele, lead                      subcomplex (Grandi et al., 1993) perform overlapping <br /> poly(A)+ RNA accumulation inside nucleus                         function nuclear pore complex. <br /> restrictive temperature, Nup 133p required                          In addition accumulation poly(A)+ RNA, <br /> efficient mRNA export cytoplasm (see above).                       nupl33- cells striking defect nuclear pore <br /> We know finally causes synthetic lethality                  distribution, i.e. nuclear pores longer randomly <br /> when nupJ33-1 nup49-316 alleles combined,                       distributed nuclear envelope, become <br /> an attractive explanation synergistic                     clustered foci. Normal nuclear pore <br /> inhibition RNA export finally kills cells. The                  distribution appears require amino-terminal <br /> may true nupl33-I/nup49-313 combination,                         Nupl33p ProtA-nupl33-ANp targeted <br /> since nup49-313 strain, primarily defective                    nuclear pores, does restore normal NPC <br /> in protein import, displays moderate intranuclear                    distribution. The co-localization ProtA-nup 133-ANp <br /> accumulation poly(A)+ RNA. On hand,                            nucleoporins clustered nuclear pore <br /> synthetic lethality complex anticipated,                   complexes proves Nupl33p physically <br /> since mutant nspl alleles reveal apparent                   linked structures. As welcome incidental result, <br /> mRNA export defect synthetic lethal                         phenotype NPC clustering nupl33- strain <br /> nup133 mutant allele. Conversely, ProtA-nupI33-                         exploited tool localize putative nuclear <br /> AN mutant allele complements poly(A)+ RNA                          pore protein indirect immunofluorescence, given that <br /> accumulation defect nupI33- cells leads to,                      antibodies tagged form protein available. <br /> synthetic lethal phenotype combined nup49                            Alteration nuclear pore distribution previously <br /> mutant alleles (data shown). Therefore, propose                      reported yeast cells over-expressing HMG-CoA <br /> that Nup133p Nsplp/Nup49p-containing NPC                            reductase, nuclear envelope is <br />  <br />                                                                                                                                            6071 <br />  V.Doye, R.Wepf E.C,Hurt <br />  <br />  <br />  <br />  <br />                     Nomarski <br />  <br />  <br />  <br />  <br /> Fig. 10. Expression ProlA-nup133-4Np complements defect cawing nuclcar accumulation o f ply(^)+ RNA nup133- ?train. <br /> ~ o l y ( d ) +RNA export analysed siru hybridization FITC-lakled oligo(dT) prohe ProtA-nupl.73-AN cells grown 24&#176;C or <br /> shifted 5 h 37OC.The coincident Hwchst 33258 staining Nomatiki optic$ views arc ~ h o w n . <br />  <br />  <br /> covered stacks nuclear envelope designated                         half ~ ~ ~was1deletedldisrupted <br />                                                                                                       4 5 ~          IWente Blobel, <br /> karmellae. In later case, NPC reported                       1944) differ observed nup133- <br /> enriched akarmellar regions, indicating yeast cells                 cells: (i) non-clustered NPCs. detected nupF45-AN <br /> can survive clustered pores (Wright et al., 1988). In                  cells, observed nup133- cells; (ii) of <br /> wild-type yeast cells, derived starved cultures (Moor                  nuclear pore clusters observed nup133- cells <br /> and MiihFethaler, 1963; Severs et a!., 14761, clustering                  associated altered stnrcture nuclear <br /> phenotype wa. reported. The later phenotype is,                        envelope; (iii) nup133- cells displayed a <br /> however, drastic nup133- cells, nuclear                  clustering phenotype associated interconnected <br /> pores reach densities higher 18 pores/pm2                    herniations nuclear envelope, previously described <br /> cells stationary phase (Severs er at., 1976), compared                   grape-like clusters nup145-AN cells; (iv) although <br /> with density 60 poreslpm2 nup133- cells.                      rarely seen nup 133- cells, intranuclear structures, <br /> The regulatory mechanism causes NPC cluster                      reminiscent intranuclear lameIlae observed <br /> stationary old cells known. In view                       NUPIIBdisrupted cells (Wente Blobel, 1993) <br /> results, speculate repression specific                     reported nup145-AN cells. <br /> nuclear pore components Nup133p                             Despite fact mRNA export mutants also <br /> possible mechanism regulate NPC distribution                        share abnormal structure distribution NPCs, several <br /> starvation.                                                                 lines evidence indicate: nup133 mutant <br />    Interestingly, nuclear pore clustering reported                 strains, defects causing nuclear pore clustering and <br /> for mRNA trafficking mutants S-cerevhiae, rat2                       intranuclear accumulation poly(A)+ RNA not <br /> and rat3 (Copeland el al., 1941), nup145 mutant                   obligatorily linked other: (i) nup133- cells. <br /> (Wente Blobel, 1994). allelic rat 10 (A-Goldstein,                   nuclear pore clustering seen temperatures at <br /> T.C.Dockendorffand C.N.Cole, personal communication).                       1 S&#176;C, nuclear accumulation poly(A)+ RNA, <br /> However, nuclear pore clustering phenotype observed                     detected 18&#176;C. progressively <br /> in nup145-AN mutant, amino-terminal                        pronounced growing cells higher temperatures: <br />                                                                                     Poly(A)+ RNA distribution NPC clustering <br />  <br />  (ii) amino-terminal truncation Nupl33p (ProtA-            biogenesis arises division pre-existing NPCs, <br />  nupl33-ANp) partially complement temper-           subsequent migration spreading nuclear <br />  ature-sensitive phenotype defect poly(A)+ RNA        envelope require functional Nupl33 protein. As an <br />  localization nupl33- strain, does restore        alternative hypothesis, pore clustering nupl33- mutant <br />  random NPC distribution.                                       cells consequence insertion newly formed <br />     Taken together, data nuclear            NPCs specific sites nuclear envelope whereas <br />  accumulation poly(A)+ RNA linked tempera-         wild-type cells insertion occurs randomly. Time- <br />  ture-sensitive growth phenotype various nupl33          course analysis GAL::NUP133 mutant strains in <br />  mutant alleles. Whether defect specific mRNA       NPC clustering phenotype induced <br />  reflects general inhibition export         repression Nup l 33p expression allow valida- <br />  various classes RNA remains analysed.             tion models. <br />  In nupl33-cells grown 37&#176;C, drastic structural <br />  abnormalities nuclear pores, like nuclear envelope <br />  seal NPCs described NUP116-disrupted            Material methods <br />  cells (Wente Blobel, 1993), unlikely, protein     Yeast strains, DNA manipulations microbiological <br />  import affected. However, subtle alterations       techniques <br />  nuclear pore structure occurring restrictive     Yeast strains used study listed Table I. DNA manipulations <br />  temperature cause RNA export defects. For     including restriction analysis, fill-in reactions Klenow fragment, <br /> instance, grape-like clusters NPC intranuclear    ligations PCR amplifications essentially according Maniatis <br />                                                                 et al. (1982). Microbiological techniques including yeast growth on <br />  lamellae-like structures observed nupl33- cells        minimal YPD medium, plasmid transformation, plasmid recovery, <br /> grown 24&#176;C correlated intranuclear           gene disruption, sporulation diploid cells tetrad analysis were <br />  accumulation poly(A)+ RNA observed tempera-         performed described Wimmer et al. (1992), most <br />  ture cells (Figure 6A). On hand,      cases minimal SD medium supplemented amino acids and <br />                                                                 nutrients used selection (CSM medium, BIO 101, La <br />  nuclear baskets proposed directly implicated    Jolla, CA). <br />  active transport (Pante Aebi, 1994), could <br />  speculate Nupl33p of, interferes with,    Plasmids <br /> dynamic structure. Alternatively, Nup133p           The following plasmids used study: pUNI0O pUN90: <br /> directly involved RNA export. Unlike Nupl45p            ARSU/CEN4 plasmid respectively LEU2 HIS3 markers <br /> was shown involved mRNA export (Fabre             (Elledge Davis, 1988); pCH1 122: YCp5O derivative (ARS1/CEN4) <br />                                                                 URA3 ADE3 markers (Kranz Holm, 1990); pCHI 122- <br /> et al., 1994), primary sequence Nupl33p does         URA3-ADE3-NUP49: complete NUP49 gene inserted a <br /> contain putative RNA-binding motifs. However, export         2.5 kb XmnI fragment (Wimmer et al., 1992) SmaI site of <br /> various RNA species previously shown involve             pCH 1122; pUN 100-LEU2-NUP49 pUN90-HIS3-NUP49: containing <br /> specific proteinaceous factors (Izaurralde Mattaj,          3.5 kb genomic DNA insert encoding Nup49p; pUN100-LEU2-nup49- <br />                                                                 313 pUN 100-LEU2-nup49-316: pUN 100 containing SacI-HindIll <br />  1992) dock nuclear pore complex.            restriction fragment encoding mutated nup49-313 nup49-313 <br /> RNA export mediated specific              alleles; pUN90-HIS3-nup49-313 pUN90-HIS3-nup49-316: Sacl- <br /> interactions RNA-binding proteins specific          HindIIl blunt-ended restriction fragment plasmids pUN 100-LEU2- <br /> receptors located NPC. Nupl33p             nup49-313 pUNI00-LEU2-nup49-316 encoding mutated nup49 <br /> a NPC component linking RNA transport               alleles inserted pUN90 plasmid restriction sites Sacl-SmaI; <br />                                                                 pRS414-TRPI-ts nspl (L640S): pRS414 containing SphI-HindIII <br /> machinery nuclear pore complex. Further                  restriction fragment encoding mutated Nsplp carboxy-terminal <br /> characterization slv21 strain, including identi-     domain (L640S) control ADHI promoter (Wimmer <br /> fication specific residue(s) mutated nupl33-      et al., 1992); pUNI0O-NUP133: containing 3.8 kb BainHI-BglII <br /> I allele isolation components function-      genomic DNA insert encoding Nupl33p. <br /> ally physically interact Nupl33p facilitate       Generation thermo-sensitive nup49 mutant alleles <br /> the understanding involvement Nup 133p             A DNA fragment (nucleotides 1425-1992) NUP49 encodes <br /> nuclear retention poly(A)+ RNA.                              128 amino acids Nup49p (Wimmer et al., 1992) randomly <br />     Since known mechanism directing        mutagenized PCR according Leung et al. (1989), using 1:10 <br /> the biogenesis distribution NPCs      ratio dATP dNTPs. The PCR product cut at <br />                                                                 XbaI/HindIII restriction sites (the generated PCR) was <br /> nuclear envelope, speculate possible     re-introduced plasmid pUNIOO previously cut same <br /> involvement Nupl33p NPC distribution. As normal           restriction enzymes replacing wild-type DNA sequence <br /> nuclear pore distribution persists removal          1425-1992 NUP49 corresponding mutagenized sequences. <br /> nuclear envelope purified nuclei (Aaronson             This library mutagenized nup49 kept control of <br /> Blobel, 1974), NPC arrangement probably main-            authentic promoter. The DNA sequence analysis randomly picked <br />                                                                 clones revealed frequency 2% point mutations. <br /> tained attachment underlying structures. In higher           The shuffling strain VDIR used identify temperature-sensitive (ts) <br /> eukaryotes, nuclear lamina, nuclear envelope lattice    nup49 mutant alleles constructed follows. Strains CH 1305 (Kranz <br /> connecting nucleoplasmic baskets, cytoplasmic        Holm, 1990) VDICH carrying disrupted chromosomal <br /> &apos;pore connecting&apos; fibrils (Goldberg Allen, 1992, 1993)      TRPJ::nup49 gene complemented pCHI 122-ADE3-URA3- <br />                                                                 NUP49 mated selective SD (-his -ura) plates resulting <br /> are potential structures provide NPC           diploids sporulated YPA plates. Following tetrad analysis, a <br /> anchoring sites. Interestingly, basket-like inter-basket    haploid strain (VD1R) selected ade2 ade3 leu2 ura3 <br /> filaments attached NPCs described yeast (Rout        his3 TRPI::nup49 contained pCHI 122-ADE3-URA3-NUP49 <br /> and Blobel, 1993). Nupl33p com-            plasmid. This shuffling strain VD1R transformed muta- <br />                                                                 genized nup49 plasmid library transformants selected SDC <br /> ponent anchoring structures affect possibly   (-leu) plates. A total 1400 colonies streaked SDC (-leu) <br /> dynamic interaction NPCs. If, suggested             plates replica plated FOA-containing plates. Cells grew <br /> (R.A.Milligan, personal communication), nuclear pore            5-fluoro-orotic acid (FOA) lost pCHI 122-URA3-ADE3-NUP49 <br />  <br />                                                                                                                                  6073 <br />  V.Doye, R.Wepf E.C.Hurt <br />  <br /> plasmid dependent mutagenized nup49 gene.            frame coding sequence NUP133 EcoRI site, generated <br /> After 3 days 23&#176;C, colonies double replica plated YPD plates     PCR ATG NUP133, removing methionine. <br /> and incubated 23 37&#176;C. Plasmid DNA recovered          The ProtA-NUP133 fusion gene inserted plasmid pUNI0O was <br /> two putative ts mutant strains NUP49 sequence determined.       used transform nupl33- (nupl33::HIS3) haploid strain, yielding <br /> The following amino acid substitutions found: nup49-313: K376E,        ProtA-NUP133. Protein A-tagged fusion proteins bearing internal <br /> 1390V, L391P, V3981, L449P; nup49-316: Q355L, D370G, 140(1V, 1439L,         deletions NUP133 ORF generated removing, <br /> F447S, stop codon473R; +VNRPESLYSAYNKYKCLVQM.                               respectively, NcoI-Ncol fragment (nucleotides 360-960) encoding <br />    The ts alleles, nup49-313 nup49-316 excised            residues 44-236 (ProtA-nupl33-ANp) BglII-BamHI fragment <br /> recovered pUN100 plasmids Sac -Hindlll fragments, blunt-ended         (nucleotides 1980-2940) encoding residues 592-926 (ProtA-nupl33- <br /> the HindlIl site ligated HIS3-containing plasmid pUN90         AMp). <br /> previously cut Sacl Sinai. The corresponding pUN90-nup49-             Whole yeast cell extracts subcellular fractionation performed <br /> 313 pUN90-nup49-3 16 plasmids used transform shuffling      previously described (Grandi et al., 1993). Aliquots various <br /> strain VD1R. After loss NUP49-containing plasmid FOA,         fractions analysed SDS-PAGE immunoblotting. The fusion <br /> nup49-313 nup49-316 expressing strains displayed similar ts           proteins detected using IgG coupled horseradish peroxidase <br /> phenotype initially isolated alleles.                                (Dakopatts, Denmark) described (Grandi et al., 1993). <br /> Isolation synthetic lethal mutants NUP49                              Immunofluorescence <br /> The tester strain VDIRW316 used isolation synthetic lethal           To immunolocalize protein A fusion proteins vivo, yeast cells <br /> mutants nup49 obtained transforming VDIR strain          expressing tagged protein fixed 3.7% formaldehyde 1 h, <br /> plasmid pUN100-LEU2-nup49-316. If grown 32&#176;C, VDIRW316                   converted spheroplasts using 0.2 mg/ml zymolyase 100.000 T, <br /> exhibits distinct red/white sectoring phenotype SD (-leu +ura)         processed immunofluorescence analysed conventional <br /> plates, occasional loss ADE3-containing pCH 11 22-NUP49       fluorescence microscope described earlier (Wimmer et al., 1992). <br /> plasmid mitotic growth.                                              Rabbit anti-chicken IgG (Medac, Hamburg, Germany) binds to <br />    For mutagenesis, VDlRW316 cells grown liquid SD (-ura            protein A moiety used antibody (1:50 dilution) followed <br /> -leu) medium OD600 0.8, plated SDC (-leu) plates, UV               goat anti-rabbit IgG coupled Texas Red (1:50 dilution). <br /> irradiated (X = 254 nm) 35 s incubated 32&#176;C. Of 400 000             To stain nucleoporins, mAb4l4 antibody (Aris Blobel, 1989) <br /> plated cells, 55 000 survived mutagenesis, forming         (BabCO, Richemond, USA) used dilution 1:20, followed by <br /> colonies clear red/white sectoring phenotype. Non-sectoring red      FITC-conjugated goat anti-mouse IgG (Medac, Hamburg, Germany), <br /> colonies picked streaked YPD plates (4% glucose). Thirteen      dilution 1:100. <br /> of 42 colonies non-sectoring phenotype stable second <br /> screen did grow FOA-containing plates confirming        In vivo analysis nucleocytoplasmic transport <br /> they afford lose URA3-containing pCH 1122-NUP49            In vivO poly(A)+ RNA export nucleus analysed situ <br /> plasmid. After transformations, 8 synthetic lethal mutants (slv)       hybridization retention poly(A)+ RNA inside nucleus as <br /> remained regained sectoring phenotype transformed        described (Amberg et al., 1992) strains grown 18&#176;C, 24&#176;C 37&#176;C, <br /> pUN90-HIS3-NUP49, pUN90-HIS3-nup49-316.                        (dT)50 probe used directly coupled FITC. <br />                                                                                To analyse protein import, strains transformed Matoc2-lacZ <br /> Cloning NUP133                                                           fusion gene (YEpl3-Matax2(2-135)-lacZ, kindly provided M.Hall, <br /> The slv2l strain carrying pCH1122-NUP49 pUN90-HIS3-nup49-               Basel, Switzerland) grown 23&#176;C liquid SDC (-leu) medium <br /> 316 plasmids transformed yeast genomic DNA library               OD600 nm = 0.2 transferred 5 h 23 or <br /> inserted LEU2-containing plasmid pUN 1(00 (Berges et al., 1994).   37&#176;C. Indirect immunofluorescence yeast spheroplasts previously <br /> Transformants selected SDC (-his -leu) plates. Four 5000         described (Nehrbass et al., 1993), using antibody monoclonal <br /> transformants individually streaked YPD plates (4% glucose)    mouse anti-f-galactosidase antibody (Promega, dilution 1:100). Second- <br /> displayed red/white sectoring phenotype. The plasmid DNA       ary antibody FITC-conjugated goat anti-mouse IgG (Medac, Ham- <br /> transformants recovered. Restriction maps showed          burg, Germany, dilution 1: 100). <br /> that contained genomic insert corresponding NUP49, one <br /> carried different insert 8 kb length. A 3.7 kb BacmHI-BglII         Freeze- fracture electron microscopic analysis yeast cells <br /> restriction fragment shown minimal insert able             Strains ts nup49-313 nupl33- grown overnight 24&#176;C in <br /> complement non-sectoring phenotype slv21.                            YPD medium OD6(0) 0.5. For freeze-fracture EM analysis, cells <br />                                                                             centrifuged washed resuspending distilled water least <br /> DNA sequencing gene disruption                                          times. Dense yeast suspensions (2 pl) frozen two <br /> The 3.7 kb genomic insert containing NUPI33 cut            copper platelets, EM grid spacer, modified propane jet <br /> restriction enzymes subfragments covering entire length      freezer (Bal-Tec Balzers, FL). Freeze-fracturing coating was <br /> genomic insert subcloned pUN100 pBluescript KS. DNA              performed BAF-400 (Bal-Tec, Balzers, FL). The sandwiches were <br /> sequence analysis strands performed according Sanger         loaded double-fracture table submersed liquid nitrogen <br /> et al. (1977) using M13 universal reverse primer internal     transferred immediately precooled (- 150&#176;C) cold stage <br /> oligonucleotide primers. The DNA deduced amino acid sequence            flow dry nitrogen. The frozen sandwiches fractured and <br /> of NUP133 analysed GCG programs. The molecular weight           etched temperature -1 10&#176;C pressure 2x 10-7 mbar for <br /> was determined using program PEPTIDESORT. Amino acid                 1-2 min followed unidirectional shadowing 2 nm Pt/C an <br /> sequences compared using FASTA BTFASTA.                             elevation angle 450 stabilized 20 nm carbon film. <br />    For disruption NUP133 gene, H153 gene, isolated          The single copper platelets replica frozen-etched yeast <br /> blunt-ended 1.15 kb BatmHI fragment plasmid YDpH (Berben               cells dipped water replica floated off. Adhering yeast <br /> et al., 1991), used replace internal 1.9 kb Ncol fragment      cells dissolved away floating replica washing in <br /> NUP133 ORF plasmid pUNIOO-NUP133. By manipulation,                frequently changed fresh 70% sulfuric acid 20 h. After <br /> amino acids 45-708 deleted NUP133 ORF. A linear               washing double distilled water, replica taken naked <br /> nupl33::HIS3 DNA fragment excised recombinant plasmid          400 mesh copper grids. The replicas imaged Philips 400T <br /> and used transform diploid strain RS453 selection SDC (-his)       microscope (Philips, Eindhoven, The Netherlands) magnification of <br /> plates. HIS+ transformants characterized correct integration    28 OOOX Kodak S0163 film plates. <br /> nupl33::HIS3 NUP133 gene locus Southern analysis. Isolation <br /> of total DNA Southern analysis essentially described             Thin section electron microscopic analysis yeast cells <br /> (Sherman, 1990). RS453 diploids heterozygous NUP133                Thin section EM performed essentially described Byers and <br /> sporulated tetrad analysis performed.                               Goetsch (1991) vegetatively grown cells modifications. <br />                                                                             Briefly, cells grown early logarithmic phase rinsed in <br /> Construction ProtA- NUP133 fusion gene detection                     phosphate-magnesium buffer (40 mM K2HPO4-KH2PO4, pH 6.5, <br /> of fusion protein yeast                                              0.5 mM MgCl,) fixed 30 min ice 1 ml phosphate- <br /> To construct ProtA-NUP133 fusion gene, BajmHI-EcoRI DNA               magnesium buffer containing 2% paraformaldehyde/2% glutaraldehyde. <br /> fragment encoding IgG binding domains protein A expressed          The fixed cells washed 0.1 M phosphate-citrate buffer, pH 5.8 <br /> under control NOPI promoter (Berges et al., 1994) fused      resuspended I ml phosphate-citrate buffer plus 1/10 dilution <br />  <br /> 6074 <br />                                                                                                 Poly(A)+   RNA distribution NPC clustering <br />  <br />  <br /> of glusulase (Dupont, NEN) 0.2 mg/ml Zymolyase 20T (Seikagaku            Kraemer,D., Wozniak,R.W., Blobel,G. Radu,A. (1994) Proc. Natl <br /> Corp. Tokyo). After 2 h 30&#176;C, cells washed 0.1 M sodium             Acad. Sci. USA, 91, 1519-1523. <br /> acetate, pH 6.1. After postfixation (15 min 2% osmium tetroxide)      Kranz,J.E. Holm,C. (1990) Proc. Natl Acad. Sci. USA, 87, 6629- <br /> en bloc staining (I h I % uranyl acetate dark), samples      6633. <br /> dehydrated embedded Epon according standard procedures.            Leung,D.W., Chen,E. Goeddel,D.V. (1989) Technique, 1, 11-15. <br /> Thin sections collected nickel grids coated formvar, stabilized      Loeb,J.D.J., Davis,L.I. Fink,G.R. (1993) Mol. Biol. Cell, 4, 209-222. <br /> with carbon, contrasted staining uranyl acetate Renold&apos;s    Maniatis,T., Fritsch,E.T. Sambrook,J. (1982) Molecular Cloning: A <br /> lead. Specimens visualized described above.                             Laboratorv Manual. Cold Spring Harbor Laboratory Press, Cold <br />                                                                                 Spring Harbor, NY. <br /> Gene bank accession number                                                   McMorrow,I., Bastos, R., Horton,H. Burke,B. (1994) Biochim. <br /> The NUP133 sequence appear EMBL, GenBank DDBJ                   Biophvs. Acta, 1217, 219-223. <br /> Nucleotide Sequence Databases accession number X80066.             Moor,H. Muhlethaler,K. (1963) J. Cell Biol., 17, 609-627. <br />                                                                              Mutvei,A., Dihlmann,S., Herth,W. Hurt,E.C. (1992) Eur J. Cell <br />                                                                                 Biol., 59, 280-295. <br /> Acknowledgements                                                             Nehrbass,U., Fabre,E., Dihlmann,S., Herth,W. Hurt,E.C. (1993) Eur <br />                                                                                 J. Cell Biol., 62, 1-12. <br /> We grateful Elmar Schiebel, Monika Matzner (Martinsried,              Nehrbass,U., Kern,H., Mutvei,A., Horstmann,H., Marshallsay,B. and <br /> Munich, Germany) Maria Ericsson (EMBL, Heidelberg)                Hurt,E.C. (1990) Cell, 61, 979-989. <br /> expert assistance EM experiments. We like thank            Pant6,N. Aebi,U. (1993) J. Cell Biol., 122, 977-984. <br /> H.Tekotte providing s1248 mutant strain, Dr A.Tartakoff (Cleveland,      Pante,N. Aebi,U. (1994) Ciurr Opin. Struct. Biol., 4, 187-196. <br /> OH) initial contribution analysis poly(A)+ RNA distribution    Pifiol-Roma,S. Dreyfuss,G. (1992) Nature, 355, 730-732. <br /> in nup49 mutants Dr M.Hall (Basel) providing plasmid            Radu,A., Blobel,G. Wozniak,R.W. (1993) J. Cell Biol., 121, 1-9. <br /> YEpl3::Mata2-iacZ. The critical reading manuscript lain            Reichelt,R., Holzenburg,E.L., Buhle,E.L., Jarnik,M., Engel,A. and <br /> Mattaj, Spyros Georgatos, Bertrand S6raphin various colleagues           Aebi,U. (1990) J. Cell Biol., 110, 883-894. <br /> gratefully acknowledged. V.D. recipient C.E.C. Fellowship     Rout,M.P. Blobel, G. (1993) J. Cell Biol., 123, 771-783. <br /> R.W. E.C.H. grants Deutsche Forschungsgemeinschaft.          Sanger,F., Nicklen,S. Coulson,A.R. (1977) Proc. Natl Acad. Sci. <br />                                                                                 USA, 74, 5466-5467. <br />                                                                              Schlenstedt,G., Hurt,E.C., Doye,V. Silver,P. (1993) J. Cell Biol., <br /> References                                                                      123, 785-798. <br />                                                                              Severs,N.J., Jordan,E.G. Williamson,D.H. (1976) J. Ultrastruct. <br /> Aaronson,R.P. Blobel,G. (1974) J. Cell Biol., 62, 746-754.                  Res., 54, 374-387. <br /> Amberg, D.C., Goldstein,A.L. Cole,C.N. (1992) Genes Dev , 6,             Sherman,F. (1990) Methods Enmvinol., 194, 3-20. <br />    1173-1189.                                                                Singh,R. Green,M. (1993) Science, 259, 365-368. <br /> Amberg,D.C., Fleischmann,M., Stagljar,I., Cole,C.N. Aebi,M. (1993)       Starr,C.M., D&apos;Onofrio,M., Park,M.K. Hanover,J.A. (1990) J. Cell <br />    EMBO J., 12, 233-241.                                                        Biol., 110, 1861-1871. <br /> Aris,J.P. Blobel,G. (1989) J. Cell Biol., 108, 2059-2067.                Sukegawa,J. Blobel,G. (1993) Cell, 72, 29-38. <br /> Berben,G., Dumont,J., Gilliquet,V., Bolle,P.-A. Hilger,F. (1991)         Wente,S.R. Blobel,G. (1993) J. Cell Biol., 123, 275-284. <br />    Yeast, 7, 475-477.                                                        Wente,S.R. Blobel,G. (1994) J. Cell Biol., 125, 955-969. <br /> Berges,T., Petfalski,E., Tollervey,D. Hurt,E.C. (1994) EMBO J., 13,      Wente,S.R., Rout,M.P. Blobel,G. (1992) J. Cell Biol., 119, 705-723. <br />    3136-3148.                                                                Wimmer,C., Doye,V., Grandi,P., Nehrbass,U. Hurt,E. (1992) EMBO <br /> Bossie,M.A. Silver, P.A. (1992) Curr Opin. Geniet. Dev:, 2, 768-774.        J., 11, 5051-5061. <br /> Byers,B. Goetsch,L. (1991) Methods Enzvmol., 194, 602-607.               Wozniak,R.W. Blobel,G. (1992) J. Cell Biol., 119, 1441-1449. <br /> Carmo-Fonseca,M., Kern,H. Hurt,E.C. (1991) Eur J. Cell Biol., 55,        Wozniak,R.W., Blobel,G., Rout,M.P. (1994) J. Cell Biol., 125, 31-42. <br />    17-30.                                                                    Wright,R., Basson,M., D&apos;AriL. Rine,J. (1988) J. Cell Biol., 107, <br /> Copeland,C.S., Amberg,D.C., Cole,C.N. Snyder,M. (1991) J. Cell               101-114. <br />    Biol., 115, 317a.                                                         Zasloff,M. (1983) Proc Natl Acad Sci USA, 80, 6436-6440. <br /> Dabauvalle,M.-C., Loos,K. Scheer,U. (1990) Chromlosoma, 100, <br />    56-66.                                                                    Received   oni July 11, 1994; revised October 10, 1994 <br /> Davis,L.I. Fink,G.R. (1990) Cell, 61, 965-978. <br /> Elledge,S.J. Davis,R.W. (1988) Gene, 70, 303-312. <br /> Fabre,E. Hurt,E.C. (1994) Curr Opin1. Cell Biol., 6, 335-342.            Note added proof <br /> Fabre,E., Boelens,W.C., Wimmer,C., Mattaj,I.W. Hurt,E.C (1994)           During course work learned rat3 allelic to <br />    Cell, 78, 275-289.                                                        NUP133 (C.N.Cole, personal communication). <br /> Finlay,D.R., Meier,E., Bradley,P., Horecka,J. Forbes,D.J. (1991) J. <br />    Cell Biol., 114, 169-183. <br /> Forbes,D.J. (1 <br /> </body></html>