<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                            J. R. Soc. Interface (2012) 9, 3426&#8211;3435 <br />                                                                                                            doi:10.1098/rsif.2012.0490 <br />                                                                                                     Published online 15 August 2012 <br />  <br />  <br />                Artefacts statistical analyses of <br />               network motifs: general framework <br />              application metabolic networks <br />                  Moritz Emanuel Beber1, Christoph Fretter2,*, Shubham Jain1, <br />                    Nikolaus Sonnenschein3, Matthias Mu &#168; ller-Hannemann2 <br />                                  Marc-Thorsten Hu&#168;tt1 <br />                         1 <br />                        School Engineering Science, Jacobs University, Bremen, Germany <br />                  2 <br />                  Institut fu&#168;r Informatik, Martin-Luther-Universita&#168;t Halle-Wittenberg, Halle, Germany <br />                3 <br />                 Department Bioengineering, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, San Diego, La Jolla, CA, USA <br />  <br />              Few-node subgraphs smallest collective units network investigated. <br />              They scale individual nodes local than, example, communities. <br />              When statistically over- under-represented, called network motifs. Network motifs <br />              interpreted building blocks shape dynamic behaviour networks. It is <br />              promise potentially explaining emergent properties complex systems relatively <br />              simple structures led network motifs ever-growing number of <br />              studies disciplines. Here, discuss artefacts analysis network motifs aris- <br />              ing discrepancies network investigation pool random <br />              graphs serving null model. Our aim provide clear accessible catalogue of <br />              incongruities effect motif signature. As case study, explore the <br />              metabolic network <span id='am-25' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> excluding artefacts <br />              motif signature strong plausible correlation essentiality pro&#65533;?le of <br />              metabolic reactions emerges. <br />  <br />  <br />  <br /> 1. INTRODUCTION                                                         affect motif signature are: (i) modularity <br />                                                                         [11&#8211;14] (ii) hierarchical structures [15&#8211;19]. Others <br /> Analysing few-node subgraphs network motifs <br />                                                                         include: (i) degree distribution (reviewed Newman <br /> has indispensable tool understanding <br />                                                                         [20]) (ii) degree correlations [21,22]. <br /> complex networks. Network motifs, statistical <br />                                                                            The crosstalk global local network prop- <br /> over- under-representations few-node subgraphs, <br />                                                                         erties effect network motifs assessed in <br /> give insights network organization the <br />                                                                         recent studies [6,23,24]. The effect motif signatures <br /> trivial scale individual nodes links. <br />                                                                         formally explored scale-free hierarchical <br />    Any study employs perspective network <br />                                                                         graphs [24], crosstalk graph proper- <br /> motifs relate function topology network <br />                                                                         ties (assortativity clustering coef&#65533;?cient) studied <br /> must strong evidence following: (i) sub- <br />                                                                         help biased random walk the <br /> graph composition non-random (compared a <br />                                                                         ensemble graphs &#65533;?xed degree sequence [25]. <br /> suitable pool reference graphs, i.e. &#8216;null model&#8217;) <br />                                                                         Even topological analysis doubt, <br /> and (ii) motifs related function, i.e. over- <br />                                                                         functional relevance motifs biological networks <br /> represented subgraphs assigned plausible functional <br />                                                                         called question [5,7,26]. <br /> roles and/or under-represented subgraphs argued to <br />                                                                            The work Ginoza &amp; Mugler [8] forward a <br /> be detrimental functioning system. <br />                                                                         local argument motif correlations induced prop- <br />    The general concept introduced devel- <br />                                                                         erty randomization scheme. While argument <br /> oped Alon co-workers [1,2], particularly for <br />                                                                         (in form given Ginoza &amp; Mugler [8]) true only <br /> transcriptional regulatory networks [3,4]. Ever the <br />                                                                         graphs bidirectional links (and therefore <br /> early studies network motifs, clear other <br />                                                                         reduced motif inventory) small densities (as <br /> mesoscopic macroscopic network properties affect <br />                                                                         wider range randomization steps be <br /> the perceived motif signature. In particular, randomiz- <br />                                                                         available), argument shows possibility <br /> ation procedure drawn signi&#65533;?cant criticism [5&#8211;10]. <br />                                                                         subtle intrinsic correlations in&#64258;uencing result a <br /> Some examples topological properties considered here <br />                                                                         motif analysis. In Avetisov et al. [27], algorithm con- <br />                                                                         structing modular hierarchical graphs described. The <br /> *Author correspondence (christoph.fretter@gmx.de).                  authors, particular, observe constructed <br /> Electronic supplementary material available http://dx.doi.org/    graphs non-random motif composition, resembling <br /> 10.1098/rsif.2012.0490 http://rsif.royalsocietypublishing.org.   superfamilies [2]. <br />  <br /> Received 19 June 2012 <br /> Accepted 24 July 2012                                               3426                        <span id='am-17' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> journal q 2012 The Royal Society <br />                                                          Artefacts network motif analyses    M. E. Beber et al. 3427 <br />  <br />    Here, extend topological arguments          degree correlations or, alternatively, retaining global <br /> earlier-mentioned list provide clear transparent     network features modularity diameter). <br /> catalogue possible artefacts arising mismatches        However, subsidiary conditions increase risk <br /> between network investigation pool           sampling null model search space non-uniformly. <br /> of randomized graphs serving null model. The func-              There sorting schemes subgraph types in <br /> tional relevance motifs investigated             literature. We employ Milo et al. <br /> combining dynamical <span id='am-64' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-65'  ></span><span id='am-66' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>.                            [1], subgraphs grouped according criteria <br />    Our running example metabolic network             (cyclic versus acyclic; connectivity number of <br /> <span id='am-41' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli; yet, parallel, illustrate         bidirectional links), one, three- <br /> some problems random graphs. We             node subgraphs sorted according &#8216;identi&#65533;?er&#8217; <br /> that successive removal artefacts strongly mod-         (the adjacency matrix subgraph, read binary <br /> i&#65533;?es motif signature. Using &#64258;ux-balance analysis           number [34]). In &#65533;?gures showing TSP, also <br /> (FBA) [28], standard method predicting metabolic         indicate subgraph identi&#65533;?er underneath the <br /> &#64258;ux patterns biomass production,              corresponding subgraph pictogram. <br /> that corrected motif signatures functionally            The important software packages motif <br /> ever plausible.                                           analysis m&#65533;?nder, software used original <br />    The organization text follows: short over-   works motifs [1], FANMOD [35]. MAVisto [36] <br /> view motifs, methods employed        additionally ability highlight motifs labelled <br /> and network representation metabolism, given      networks. Motif &#65533;?nding implemented for <br /> &#167;2; number effects network architecture motif       Pajek network analysis software [37]. <br /> content discussed &#167;3; issues mapping (dynami-            In following, impact several <br /> cal) data motifs focus &#167;4; &#167;5,      large-scale topological properties TSP. To this <br /> implications results analysis complex net-   end, construct random graphs prescribed large- <br /> works, general, described, aspects   scale properties evaluate subgraph composition. <br /> further work.                                                  The random graphs directed Erdo&#180;&#180;s&#8211;Re&#180;nyi (ER) <br />                                                                graphs generated additional parameter pb that <br />                                                                regulates number bidirectional links. The other <br /> 2. METHODS                                                     parameter usual link probability p. In some <br />                                                                places, analyse &#8216;randomization error&#8217;, sum <br /> 2.1. Terminology software packages <br />                                                                squares Z-scores subgraphs. Given <br /> Throughout paper, discuss induced subgraphs,           purely random graphs, &#8216;correct&#8217; TSP must <br /> as constitute objects vast lit-    zero subgraphs; randomization error thus <br /> erature network motifs. By induced, mean            measures deviation correct TSP. <br /> for pair nodes subgraph,                 When appropriate, analytical predic- <br /> (directed) link link appears        tions artefactual TSP. In cases, employ <br /> the original graph.                                            formalism developed Fretter et al. [38]. <br />    The general <span id='am-12' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> motif analysis compare <br /> few-node subgraph counts obtained real net- <br />                                                                2.2. Network representations of <br /> work corresponding counts obtained from <br />                                                                     metabolic systems <br /> randomized versions. In case three-node sub- <br /> graphs, corresponding Z-scores summarized           Metabolic networks condensed, abstract represen- <br /> in triad signi&#65533;?cance pro&#65533;?le (TSP) showing statis-        tation production distribution metabolic <br /> tical over- under-representation of             compounds (metabolites) biochemical reactions <br /> subgraphs. The Z-score subgraph m de&#65533;?ned             occur organism owing catalytic <br /> Zm &#188; &#240;cm   mm &#222;=sm , cm count subgraph         action enzymes. Metabolites reactions belong to <br /> m original network, mm expectation value         distinct sets nodes. The possible connections are <br /> of cm random networks sm standard            links categories within, thus <br /> deviation cm random networks.                        forming bipartite network. More involved represen- <br />    A typical randomization scheme preserves                tations metabolism exist, example, including <br /> number incoming, outgoing bidirectional links           enzymes category nodes [39], choosing <br /> at node. The prominent example switch             hyper-graph structure represent reaction&#8211;compound <br /> randomization, iteratively endpoints randomly         relations [40], representations inaccessible <br /> selected links swapped (ensuring bidirectional        wide range current graph-analytical tools. <br /> and unidirectional links randomized independently              The topology metabolic networks organized a <br /> and parallel links produced way).          complicated way. The degree distribution metabolites <br /> The exact procedure randomizing graph               typically scale-free, architecture modular and <br /> retaining degree sequence graph            layered. Ignoring hierarchy functional modules, <br /> debated &#65533;?elds application [29]          uptake reactions, reactions contributing biomass <br /> mathematical communities [30&#8211; 33]. Apart pure             generation reactions in-between result a <br /> convergence issues iterative randomization pro-         layered organization metabolism. These complica- <br /> cess, individual randomization step interest.        tions present formidable challenge attempt to <br /> In particular, accompany decision sub-          explore interrelations network topology <br /> sidiary conditions (e.g. enhancing positive negative        dynamical function metabolic systems [41,42]. <br />  <br /> J. R. Soc. Interface (2012) <br />  3428 Artefacts network motif analyses               M. E. Beber et al. <br />  <br />     (a)                                                                     (a)                                (b) <br />             C3          C2                                                               R2                               R2 <br />                                       (b) <br />                                                  C4 <br />  <br />  <br />                                                                                          C1                               R1 <br />              R2 <br />  <br />                                        C1         C3 <br />  <br />                                                                                                                      R3 <br />              C5                                                               R4         R3          R1 <br />  <br />                                                   C5 <br />                                                                                                                             R4 <br />                    R1 <br />                                                                       Figure 2. Projection metabolic bipartite network (a) onto <br />                                                                       reaction nodes (b). As &#65533;?gure 1, reverse direction a <br />                                                        C2 <br />                                                                       reaction represented dashed lines. <br />              C4         C1 <br />  <br />  <br /> Figure 1. Projection metabolic bipartite network (a)        nature graph, adequate treatment reversible reac- <br /> its metabolite nodes (b). As discussed &#167;2.2, multiple    tions modularity; (ii) incorporating functional <br /> ways unambiguously storing reaction reversibility information      constraints (such mass balance [47] stoichio- <br /> in network. Here, solid lines identify substrates products   metric constraints [48]), ensuring that, principle, the <br /> forward reactions dashed lines reverse direction.             randomized graph fully functional metabolism. <br />                                                                       In paper, pursue &#65533;?rst aspect. <br />                                                                           Eom et al. [46] neglected important bio- <br />     Because formalisms developed analyse net-                 logical point dealing metabolic network <br /> works focused general graphs                   structures. When think metabolic capabilities <br /> nodes belong mode category, arbi-                  organism, mind pathway <br /> trary links possible, henceforth called unipartite                maps procession biochemical reactions <br /> graphs networks, metabolic networks customarily                required transform substrate taken the <br /> projected sets nodes                environment &#65533;?nal product useful organ- <br /> the projected unipartite network analysed. In                 ism. In <span id='am-22' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-23' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> preserve pathway structure, Ma &amp; <br /> the majority, analyses considered metabolism                 Zeng [49] suggested removal currency metab- <br /> an undirected network (see earlier studies [39,43,44]                 olites network, i.e. compounds that <br /> and critique projection metabolic                     provide energy balance charges, chemical groups <br /> networks, recent studies [41,45]).                                main metabolites transformation. In <br />     A method projection directed metabolic net-            opinion, step necessary reveal struc- <br /> works presented [46]. We retain method                   tures dominated connections formed by <br /> projecting metabolic network metabolite-                 currency metabolites reaction <br /> centric representation, i.e. connecting substrate                nodes network. In study, highlight the <br /> with product certain reaction introducing               various results stem considering technical- <br /> a bidirectional link reaction reversible (see           ities topology-based constraints metabolic <br /> scheme &#65533;?gure 1). For directed representation                     null models. <br /> of reaction-centric network, draw link                   In following, use set metabolic networks <br /> two reaction nodes directed path                     43 different organisms collected the <br /> length source reaction Ri, exactly                 WIT database1 published recent studies [43,46]. <br /> one metabolite Ck, reaction R j=i . The link            They available online bipartite graphs.2 <br /> drawn direction existing path (as depicted                  Our main object study metabolic net- <br /> in &#65533;?gure 2).                                                          work representation E. coli compiled the <br />     Studying few-node subgraph compositions                        cytosolic components genome-scale metabolic <br /> metabolic networks suffers issues. First,                model iAF1260 [50] manually curated such <br /> few-node subgraphs typically de&#65533;?ned unipartite                connections currency metabolites removed, <br /> graphs. Second, choice appropriate null model               cases metabolites main <br /> is unclear. In &#65533;?rst publication TSPs metabolic             reactants as, example, ATP synthesis [51]. In <br /> networks [46], metabolite-centric representations                 addition, functional subsystem categories included <br /> were treated graph, standard switch                  Feist et al. [50], i.e. core, amino acid, lipid, nucleotide <br /> randomization applied, ignoring,                       unspeci&#65533;?c metabolism used module- <br /> others, effects caused projection process itself.              aware randomization described later on. <br />     Attempts de&#65533;?ne realistic null models <span id='am-48' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-49' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>meta</span>-            1 <br />                                                                        The database changed location ERGO, https:// <br /> bolic systems need focus aspects: (i)                       ergo.integratedgenomics.com/ <br />                                                                       2 <br /> incorporating topological constraints bipartite            See http://www.nd.edu/ networks/resources/metabolic/ <br />  <br />  <br /> J. R. Soc. Interface (2012) <br />                                                                     Artefacts network motif analyses                    M. E. Beber et al. 3429 <br />  <br />                        0.6                                                                       0.4 <br />                        0.4 <br />  <br />  <br />  <br />  <br />                                                                            normalized Z-score <br />  normalized Z-score <br />  <br />                                                                                                  0.2 <br />                        0.2 <br />                                                                                                   0 <br />                         0 <br />                                                                                                 &#8211;0.2 <br />                       &#8211;0.2 <br />                                                                                                 &#8211;0.4 <br />                       &#8211;0.4 <br />                                                                                                 &#8211;0.6 <br />  <br />                              6 36 12 74 14 78 38 98 108 46 102 110 238                                 6 36 12 74 14 78 38 98 108 46 102 110 238 <br />  <br /> Figure 3. Mean standard deviation normalized Z-scores              Figure 4. A comparison TSP metabolite-centric <br /> of three-node subgraph frequency metabolic networks                projection metabolic network [46], labelled &#8216;orig- <br /> of 43 different organisms compared standard                   inal&#8217;, TSPs manually curated metabolic <br /> switch randomization metabolite-centric projection.                network [51] using different null models labelled <br /> Three examples well-known organisms enter                     &#8216;unipartite&#8217;, &#8216;bipartite&#8217; &#8216;modular&#8217;. The curve labelled &#8216;uni- <br /> mean shown. The networks used                 partite&#8217; stems switch randomization network <br /> Eom et al. [46]. Raw Z-scores electronic sup-             projected metabolite nodes. The <br /> plementary material, &#65533;?gure S4. Solid line, mean 43                        &#8216;bipartite&#8217; curve denotes randomized networks were <br /> organisms; dashed line, E. coli; dotted line, S. cerevisiae;              generated level bipartite network, treating <br /> dashed-dotted line, A. thaliana. (Online version colour.)              reversible reactions separately counting three-node <br />                                                                           subgraphs metabolite-centric projections. The &#8216;modular&#8217; <br />                                                                           curve combines &#8216;bipartite&#8217; switch randomization bio- <br />                                                                           logical modules. The raw Z-scores seen electronic <br /> 3. FINDING MOTIFS IN METABOLIC <br />                                                                           supplementary material, &#65533;?gure S5. Solid line, original; dashed <br />    TOPOLOGIES                                                             line, unipartite; dotted line, bipartite; dashed-dotted line, mod- <br /> The starting point work TSP published                   ular. (Online version colour.) <br /> Eom et al. [46], recomputed original <br /> networks &#65533;?gure 3 subgraph ordering <br />                                                                           function system. Depending type of <br /> used Milo et al. [1]. Most noteworthy this <br />                                                                           dynamic data available metabolite-centric <br /> result small overall magnitude stan- <br />                                                                           reaction-centric, metabolic network represen- <br /> dard deviation surprisingly small variation <br />                                                                           tation motif signature appropriate <br /> of standard deviation triads. Of course, <br />                                                                           topological information. To best knowledge <br /> these means over-normalized Z-scores; yet <br />                                                                           &#65533;?rst time, TSP reaction- <br /> their triad composition appears remarkably similar for <br />                                                                           centric network E. coli &#65533;?gure 5. The curve labelled <br /> networks vastly different sizes. <br />                                                                           &#8216;unipartite&#8217; deviates extremely used null <br />    There problems attached database <br />                                                                           model. Most strikingly, TSP separates clearly into <br /> of 43 networks used, example, recent studies <br />                                                                           categories. The &#65533;?rst triads (the chain-like sub- <br /> [43,46] computing TSPs shown &#65533;?gure 3. <br />                                                                           graphs) extremely under-represented, whereas <br /> They given bipartite networks compounds <br />                                                                           seven triads (triangular subgraphs) are <br /> and reactions identi&#65533;?ed indices, we <br />                                                                           over-represented. Notably, magnitudes <br /> lack chemical information them. Second, and <br />                                                                           differ, case metabolite-centric <br /> related &#65533;?rst issue, networks contain cur- <br />                                                                           network (&#65533;?gure 4). The extreme values Z-scores <br /> rency metabolites. Third, links compounds <br />                                                                           seen electronic supplementary material, &#65533;?gure S6 <br /> and reactions carry additional information. It there- <br />                                                                           indicate ill-formed null model. <br /> fore impossible determine substrates products for <br /> reversible reactions network. Why these <br /> points matter shown following.                             3.2. Categorizing bidirectional links <br />                                                                           We include pieces biochemical <br />                                                                           information require return bipartite <br /> 3.1. Currency metabolites <br />                                                                           network representation metabolism. First all, <br /> From point forward, use metabolic network                     mounting evidence projecting bipartite <br /> of E. coli described &#167;2.2. previously used                   graph set nodes affects number topo- <br /> Sonnenschein et al. [51]. We compare TSP employ-                      logical quantities. In case clustering, been <br /> ing different null models network used                        remarked metabolism Monta nez et al. [45] and <br /> Eom et al. [46]. In &#65533;?gure 4, curve labelled &#8216;unipar-               discussed general level Latapy et al. [52]. <br /> tite&#8217; shows TSP curated E. coli metabolic                      The simplest way incorporating effects <br /> network projecting compound nodes                       null model perform switch randomization <br /> and using simple switch randomization null model.                    level bipartite graph. We com- <br />    Over- under-representations triads particu-                  pare statistics projected metabolic network with <br /> larly informative related dynamical                           projections randomized bipartite counterparts. <br />  <br /> J. R. Soc. Interface (2012) <br />  3430 Artefacts network motif analyses                    M. E. Beber et al. <br />  <br />  <br />                       0.8                                                            250 <br />                       0.6 <br />  normalized Z-score <br />  <br />  <br />  <br />                                                                                      200 <br />                       0.4                                                            150                     scheme 1 <br />  <br />  <br />  <br />  <br />                                                                            Z-score <br />                       0.2                                                            100 <br />                        0 <br />                                                                                       50 <br />                &#8211;0.2                                                                        scheme 2 <br />                                                                                       0 <br />                &#8211;0.4 <br />  <br />                             6 36 12 74 14 78 38 98 108 46 102 110 238                      6 36 12 74 14 78 38 98 108 46 102 110 238 <br /> Figure 5. A comparison TSP reaction-centric pro-            Figure 6. TSP graph elevated number bidirec- <br /> jection metabolic network [46], labelled &#8216;original&#8217;           tional links compared expected value m. Two <br /> (solid line), TSPs manually curated metabolic             different randomization-schemes applied (i) simple &#64258;ip- <br /> network [51] using different null models labelled          ping link-endpoints, (ii) shuf&#64258;ing uni- and <br /> &#8216;unipartite&#8217;, &#8216;bipartite&#8217; &#8216;modular&#8217;. The curve labelled &#8216;uni-         bidirectional links independently preserving the <br /> partite&#8217; (dashed line) stems switch randomization               number bidirectional links network. As analysed <br /> the network projected reaction                 network random apart number bidirectional <br /> nodes. &#8216;Bipartite&#8217; (dotted line) means randomized net-                links, Z-scores using correct randomization scheme <br /> works generated bipartite network, treating                   expected close zero. Obtained graphs <br /> reversible reactions separately, counting three-node             N &#188; 100 M &#188; 400, number bidirectional <br /> subgraphs reaction-centric projections. The &#8216;modular&#8217;              links forced 90% instead 16% expected ER <br /> (dashed-dotted line) obtained combining &#8216;bipar-               graphs connectivity. (Online version colour.) <br /> tite&#8217; switch randomization biological modules. The <br /> untransformed Z-scores seen electronic supplemen- <br /> tary material, &#65533;?gure S6. (Online version colour.)                      3.3. Modularity <br />                                                                           One key observation biological networks their <br />    Switch randomization level bipartite                     organization modules (communities) related <br /> graph allows address issue                         function robustness. It shown [53&#8211; 55] <br /> received little attention, bidirectional links.                    metabolism exception. We use <br /> Typically, number bidirectional links conserved                 insights modules effect Z-scores <br /> in switch randomization, argument                     described electronic supplementary material, <br /> are unique property network just                     &#167;S1.1 explain effects modules on <br /> co-occurrence unidirectional links. It                   TSP graphs general. <br /> noted real networks, types bidirectional                      We assemble graph dense modules, where <br /> links occur, principle. It plausible, example,              module directed ER graph. Additionally, few <br /> to assume gene regulatory networks bidirec-                     inter-module links introduced. As graphs each <br /> tional link co-occurrence                            module inter-module links constructed <br /> unidirectional links, metabolic networks revers-                 motif-blind way, correct randomization yield <br /> ible reactions certainly constitute example true                    &#64258;at TSP, individual Z-scores close to <br /> bidirectionality individual category. In metab-                     zero. The result applying standard switch randomiz- <br /> olism &#65533;?nd, however, case distinct                      ation seen &#65533;?gure 8. We result a <br /> enzymes responsible opposing direc-                     modularity-aware randomization scheme mixes <br /> tions converting compounds and, hence,                         intra-module links inter-module links separately. In <br /> other interpretation bidirectional links.                              real-world networks, modular structure generally <br />    Does distinction types bidir-                   known, quality modularity- <br /> ectional links related distinction                    aware randomization scheme depend quality <br /> two interpretations classical randomization                        module-detection algorithm employed. In order to <br /> scheme affect observed motif signature? In order                      better understand error randomization <br /> to motivate general point, impact                     scheme module information, perform <br /> bidirectional links TSP ER graphs (&#65533;?gure 6,                     analytical calculation yields prediction the <br /> also electronic supplementary material, &#167;S3                       error signature. To obtain predictions artefactual <br /> more in-depth information).                                               TSP arising switch randomization, use the <br />    Returning metabolic networks, curves labelled                   simple model few-node subgraphs Fretter et al. [38]. <br /> &#8216;bipartite&#8217; &#65533;?gures 4 5 result switch ran-                         It essential note that, modules are <br /> domization level bipartite graph                      destroyed, effective local intra-module density of <br /> subsequent projection respective set nodes.                   network reduced factor number of <br /> Reversible reactions treated separately                     modules. This compared network <br /> others, co-occurrence reactions performing                     N nodes M links network twice size <br /> opposing conversions compounds issue                      N *=2N nodes M *=2M links density of <br /> the bipartite graph.                                                      d   &#188; M   =N   &#240;N     1&#222;   d=2. <br />  <br />  <br /> J. R. Soc. Interface (2012) <br />                                                           Artefacts network motif analyses              M. E. Beber et al. 3431 <br />  <br />  <br />                                                                              15 <br />  <br />                                                                              10 <br />  <br />  <br />  <br />  <br />                                                                   Z-score <br />                                                                               5 <br />  <br />                                                                               0 <br />                                                                                                                scheme 2 <br />                                                                             &#8211;15 <br />                                                                                   prediction <br />                                                                             &#8211;10 <br />                                                                                                scheme 1 <br />                                                                             &#8211;15 <br />  <br />                                                                                    6 36 12 74 14 78 38 98 108 46 102 110 238 <br />  <br />                                                                  Figure 8. TSP graph composed strong modules. <br />                                                                  Two different randomization schemes applied (i) simple <br />                                                                  &#64258;ipping link-endpoints; (ii) &#64258;ipping preser- <br /> Figure 7. A graph composed strong modules. (Online       ving module structure. As analysed network is <br />                     version colour.)                          random apart modularity, Z-score using cor- <br />                                                                  rect randomization scheme 0. Additionally, the <br />                                                                  analytical prediction Z-score drawn. Network par- <br />                                                                  ameters: N &#188; 400, M &#188; 1600, 8% inter-modular links. <br />    Let N M number nodes links of <br />                                                                  (Online version colour.) <br /> the modular network consisting modules, as <br /> shown &#65533;?gure 7. Then, count speci&#65533;?c subgraph <br /> cm original graph estimated                         Each reaction manually assigned module <br /> 4 cm &#240;N =4; M =4&#222;, expectation value count        based biological function. In Ravasz et al. [53], <br /> the random graphs mm=cm(N,M) standard                 shown topological modules largely over- <br /> deviation counts sm sm(N,M). For details,           lap metabolic functional categories. Automated <br /> please refer electronic supplementary material, &#167;S1.1         assignment reactions modules, based, e.g. on <br /> and Fretter et al. [38]. The module-unaware null model cor-      gene ontology classes, plausible. The func- <br /> responds taking subgraph counts               tional groups mentioned &#167;2.2. <br /> modules, modelled independent networks compar-                Comparing TSPs &#65533;?gure 4 &#65533;?gure 5, <br /> ing total network containing nodes         general trends observed. A change from <br /> links. The resulting prediction plotted &#65533;?gure 8        complete network links currency <br /> &#65533;?ts numerical results well. When links          metabolites leads greatly increased absolute Z-scores, <br /> inter-modular, effect modularity vanish;       case reaction-centric projection an <br /> electronic supplementary material, &#65533;?gure S3 shows           inversion general trend curve. Proceeding <br /> this case experiment prediction.   switch randomization level bipartite <br />    The results &#65533;?gure 8 electronic supplemen-              graph causes change trend TSP, <br /> tary material, &#65533;?gure S3 clearly switch                 yet, absolute Z-scores lower unipartite <br /> randomization performed graphs regions                level. Using module-aware randomization scheme then <br /> varying densities leads pathological signals           lowers absolute values Z-scores. <br /> TSP. Community detection algorithms work                   It noted cases modularity <br /> remedy case, assess quality          (corresponding) non-zero TSP observed real <br /> their module partition comparing number                   network, clear to assign causality. The <br /> of links module expectation value as,           statement make observed <br /> for example, de&#65533;?ned eqn 5 Newman &amp; Girvan [12].            motif signature (or TSP) explained modularity. <br /> Thus, quality measure, modularity, detected           It helpful appropriate statistical <br /> communities takes account relevant densities.           methods tests addressing intrinsic causality <br /> However, community detection broad &#65533;?eld                  inter-related network properties: a <br /> network science, perfect algorithms exist. A              non-modular (or modular) graph host same <br /> detailed discussion community detection             motif signature? Does graph&#8217;s modularity explain <br /> the scope study, reader encouraged         &#8216;signal strength&#8217; observed motif sig- <br /> look relevant reviews Fortunato [56].               nature amplitude (or average Z-score) the <br />    We like point Z-scores triads         motif signature fully accounted given modular- <br /> metabolic networks used studies [46,57,58]            ity? In addition, overlapping modules, nodes <br /> have accounted strong modularity                unambiguously assigned individual modules, <br /> was discussed earlier. The curves labelled &#8216;modular&#8217;          different impacts TSP described here. <br /> &#65533;?gures 4 5 respective TSPs using switch <br /> randomization module-aware level of <br />                                                                  3.4. Networks hierarchies <br /> the bipartite network. In &#65533;?gures 4 5, Z-scores <br /> are shifted closer zero, result            Similar modularity, hierarchical organization can <br /> more restricted search space null model.                  cause distortions simple switch randomization. <br />  <br /> J. R. Soc. Interface (2012) <br />  3432 Artefacts network motif analyses       M. E. Beber et al. <br />  <br />  <br />  <br />  <br />                                                                          5 <br />                                                                               scheme 2 <br />  <br />  <br />  <br />  <br />                                                                Z-score <br />                                                                           0 <br />  <br />  <br /> Figure 9. An example layered random network inspired              &#8211;5 <br />       Kaluza et al. [59]. (Online version colour.)                        scheme 1 <br />                                                                    &#8211;10 <br />     Here, provide example toy model that <br /> has basic properties metabolic networks. In                           6 36 12 74 14 78 38 98 108 46 102 110 238 <br /> metabolism, distinguish input layer provided <br /> by set uptake reactions (with input provided       Figure 10. Average Z-scores 1000 layered random net- <br />                                                               works. The networks contain 20 input nodes, 50 middle <br /> by available nutrients environment), set      nodes, 20 output nodes probability link <br /> middle layers, reactions process nutrients      present 0.3. We Z-scores resulting a <br /> an output layer consisting reactions directly con-     random ensemble created using standard switch <br /> tributing cell growth (i.e. &#8216;biomass vector&#8217;     randomization (scheme 1) curve obtained from <br /> encountered constrained-based modelling meta-           switch randomization additional constraint disal- <br /> bolic systems). In following, discuss             lowing direct links input output layer <br /> layered random networks (inspired Kaluza &#8211;             (scheme 2). (Online version colour.) <br /> Mikhailov model evolved &#64258;ow networks; [59]). <br />     The input layer de&#65533;?ned set nodes zero <br /> in-degree; output layer given set nodes      feed-forward loop motif ) explicitly modelling <br /> with zero out-degree. Middle nodes receive link         dynamic processes motif classifying <br /> from input layer. They interconnected         signal processing capacities few-node <br /> with middle nodes connected             device [60,61]. <br /> to output nodes. The topological restriction             A powerful alternative direct modelling to <br /> is input nodes directly connected          analyse systematics dynamic data motif, <br /> output nodes. An example layered random             i.e. individual node motif distributed <br /> network shown &#65533;?gure 9. In &#65533;?gure 10, illustrate       motif occurrences network. This has, <br /> the effect hierarchical structure TSP using       example, performed Krumov et al. [62] cita- <br /> such graph layers. Depicted TSPs         tion frequencies (undirected) co-authorship networks, <br /> standard switch randomization randomization         Marr et al. [63] gene expression data transcrip- <br /> disallowing links input output nodes. Note       tional regulatory networks, Sonnenschein et al. <br /> all randomization methods degree nodes        [64] reaction essentiality categories. <br /> constant, i.e. input nodes (output nodes) retain            In work, apply FBA identify essential reac- <br /> zero in-degree (zero out-degree). Because additional       tions metabolic network E. coli. Within this <br /> motif <span id='am-7' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-8' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11'  ></span>bias</span> (beyond bias introduced layer           elegant framework [28,65], optimal steady-state dis- <br /> structure) entered network generation,            tribution metabolic &#64258;uxes predicted using <br /> correct null model ought yield &#64258;at TSP          linear programming, given structure environ- <br /> Z-scores close zero.                                       ment (i.e. availability nutrients) cellular <br />                                                               objective function (e.g. biomass production ATP <br />                                                               maximization). All reactions elimination proved <br />                                                               lethal wide range different environmental con- <br /> 4. DYNAMIC DATA ON MOTIFS <br />                                                               ditions considered essential. Please refer to <br /> Beyond purely structural issues, completely new         electronic supplementary material, &#167;S1.2 speci&#65533;?cs. <br /> set complications arises dynamic information              We use schemes analysing distribution of <br /> on graphs added. The signi&#65533;?cance pro&#65533;?les discussed         dynamic data motifs: (i) single counting scheme, <br /> so far obtained contrasting few-node sub-        node contribute dynamic information <br /> graph counts given graph subgraph counts          (the value attached node, mapping dynamic <br /> from suitable pool randomized graphs. It yields          data graph) subgraph type or <br /> statistical indicators pointing graph features        (ii) multiple counting scheme, counts node&#8217;s <br /> that functional relevance. In biological          dynamic information multiple times, node part <br /> networks, particularly tangible, deviations        multiple subgraphs type. The normalization <br /> from randomness (with right null model)            differences counting schemes, aver- <br /> interpreted in&#64258;uence evolutionary shaping       aging dynamic information, yield different <br /> of network. However, statistical observation         results, particular cases dynamic infor- <br /> needs linked functional level          mation number subgraphs node participates <br /> more direct manner subsequent investigations. Alon         scale non-trivially node&#8217;s degree. <br /> and co-workers carried convincingly              In &#65533;?gure 11, Z-score probability of <br /> the case three-node subgraphs (in particular,      occurrence essential reaction triad in <br />  <br />  <br /> J. R. Soc. Interface (2012) <br />                                                               Artefacts network motif analyses           M. E. Beber et al. 3433 <br />  <br />      (a)                                                               (b) <br />                                                                         4 <br />                 5 <br />                 4                                                       2 <br />  <br />                 3                                                       0 <br />       Z-score <br />  <br />  <br />  <br />  <br />                 2                                                      &#8211;2 <br />                 1 <br />                                                                        &#8211;4 <br />                 0 <br />                                                                        &#8211;6 <br />                 &#8211;1 <br />                                                                        &#8211;8 <br />  <br />                      6 36 12 74 14 78 38 98 108 46 102 110 238               6 36 12 74 14 78 38 98 108 46 102 110 238 <br /> Figure 11. Using different methods arti&#65533;?cial growth media generation &#64258;ux balance analysis, plots Z-scores of <br /> the probability &#65533;?nding essential reactions certain triads compared 1000 random distributions number <br /> of essential reactions network. The arti&#65533;?cial growth media called &#8216;Random&#8217; viable selections possible <br /> nutrients, &#8216;Combinatorial&#8217; minimal combinations nutrients sustain growth. (a) The frequency exactly <br /> three essential reactions stringent measure occurs random models. The exception the <br /> &#8216;chain&#8217;-like triad id 12. There agglomeration essential reactions triad. (b) Z-scores essential reactions <br /> using single counting scheme shows particular role triad id 12 shows accumulated case, <br /> i.e. exact number essential reactions participating subgraph required, essential reactions occur fre- <br /> quently compared null model. More scenarios seen electronic supplementary material, &#65533;?gure S7. Solid <br /> line, random; dashed line, combinatorial. (Online version colour.) <br />  <br />  <br /> the network compared random distribution                   motif signature. For example, case modu- <br /> of essential reactions network. The surprising                  larity, clear motif signature easily obtained, while <br /> result essential reactions occur frequently               modular structure network more <br /> than expected types three-node subgraphs                dif&#65533;?cult detect understand. The motif signa- <br /> the exception &#8216;chain&#8217; (id 12). This shows essen-           ture serve marker non-random <br /> tial reactions occur non-random situations                   network features diverse topological scales. <br /> metabolic network. The exceptional situation triads                    A major conceptual step systems biology to <br /> with id 12 remains explored further.                             reveal systematic signi&#65533;?cant deviations biologi- <br />    Next, use results electronic supplemen-                 cal randomness, subsequently relate <br /> tary material, &#65533;?gure S7 relate functional role               deviations speci&#65533;?c functional features. Both, <br /> the triads motif signature. The details                analyses attempts lit- <br /> be electronic supplementary material, &#167;S4.                    erature exploring network motifs metabolism show <br /> Compared TSP Eom et al. [46] ( positive                  immense dif&#65533;?culty disentangling contributions to <br /> correlations essentiality distribution curves),               motif signatures coming mere network con- <br /> we &#65533;?nd systematically negative correlations                    struction coming evolutionary <br /> re&#65533;?ned TSPs. The &#8216;unipartite&#8217; TSP (enhanced treat-                     shaping optimized function. <br /> ment currency metabolites) shows strongest                         Even re&#65533;?ned TSPs regarded as <br /> (negative) correlation, &#8216;bipartite&#8217; (ran-                 better approximations true enhancement or <br /> domization level bipartite graph)                    suppression three-node subgraphs metabolic systems <br /> the &#8216;modular&#8217; TSP (module-aware randomization)                         evolutionarily acquired, clear that <br /> display negative correlations.                                         additional re&#65533;?nement steps taken account. <br />                                                                        In recent publications, Basler et al. [42,47] presented <br />                                                                        motivated novel method generating chemically <br />                                                                        sensible, mass-balanced randomized metabolisms. The <br /> 5. CONCLUSIONS <br />                                                                        main idea physico-chemical constraints are <br /> We systematically explored biases introduced                 null model remaining deviations <br /> motif signatures variations random back-                     result evolutionary pressure thus <br /> ground, i.e. set reference graphs serving                 worth studying. Basler et al. [42] discuss number of <br /> null models. Making visible cross-talk                     structural properties changed results when <br /> global local network properties is, opinion,                compared new null model. <br /> an important prerequisite interpretation                        Some dynamical systems highly sensitive motifs <br /> motif signatures.                                                      small functional devices. A range investi- <br />    We want point                        gations identi&#65533;?ed deep relationship between <br /> real artefacts found, time bound-               network motifs robust functioning systemic <br /> ary relevant result artefact                processes [7,59,60,66,67]. In order understand gen- <br /> ambiguous. General features networks                         erality fundamental nature links between <br /> simple explanation visible                   topology dynamics, needs better knowledge of <br />  <br /> J. R. Soc. Interface (2012) <br />  3434 Artefacts network motif analyses          M. E. Beber et al. <br />  <br /> the intrinsic statistical properties few-node subgraphs            (ed. D. Thilikos). Lecture Notes Computer Science, <br /> as minimal dynamical situations,                     no. 6410, pp. 220&#8211;231. Heidelberg, Germany: Springer. <br /> in relationship topology                10   Birmele&#180;, E. 2012 Detecting local network motifs. Elec- <br /> dynamics occur. With present investigation,                tronic J. Stat. 6, 908&#8211;933. (doi:10.1214/12-EJS698) <br />                                                                  11   Girvan, M. &amp; Newman, M. E. J. 2002 Community struc- <br /> want contribute understanding statistical <br />                                                                       ture social biological networks. Proc. Natl Acad. <br /> signals obtained motif analyses. We advocate care-               Sci. USA 99, 7821&#8211;7826. (doi:10.1073/pnas.122653799) <br /> fully chosen null models designed profound grasp          12   Newman, M. E. J. &amp; Girvan, M. 2004 Finding evalu- <br /> of investigation. This means               ating community structure networks. Phys. Rev. E 69, <br /> once facts discovered,                  026113. (doi:10.1103/PhysRevE.69.026113) <br /> chosen null model adapted.                        13   Guimera`, R., Sales-Pardo, M. &amp; Amaral, L. A. N. 2004 <br />    Throughout paper, emphasized impor-                     Modularity &#64258;uctuations random graphs com- <br /> tance considerations understanding                       plex networks. Phys. Rev. E 70, 25101. (doi:10.1103/ <br /> metabolic systems main case study. However,                  PhysRevE.70.025101) <br /> proper treatment cross-talk various topo-           14   Guimera`, R., Sales-Pardo, M. &amp; Amaral, L. A. N. 2007 <br /> logical properties complex networks (and                        Module identi&#65533;?cation bipartite directed networks. <br />                                                                       Phys. Rev. E 76, 036102. (doi:10.1103/PhysRevE.76.036102) <br /> particular, evaluation larger-scale properties in&#64258;u- <br />                                                                  15   Baraba&#180;si, A.-L., Ravasz, E. &amp; Vicsek, T. 2001 Determinis- <br /> encing subgraph statistics) broader range               tic scale-free networks. Phys. A Stat. Mech. Appl. 299, <br /> application. We believe small catalogue                559&#8211; 564. (doi:10.1016/S0378-4371(01)00369-7) <br /> of topological features in&#64258;uencing interpretation         16   Jung, S., Kim, S. &amp; Kahng, B. 2002 Geometric fractal <br /> subgraph counts complex networks used                   growth model scale-free networks. Phys. Rev. E 65, <br /> re&#65533;?ning correcting empirical observations TSPs                 056101. (doi:10.1103/PhysRevE.65.056101) <br /> and starting points expand theoreti-              17   Dorogovtsev, S. N., Goltsev, A. V. &amp; Mendes, J. F. F. 2002 <br /> cal understanding crosstalk different                  Pseudofractal scale-free web. Phys. Rev. E 65, 066122. <br /> topological properties complex networks.                           (doi:10.1103/PhysRevE.65.066122) <br />                                                                  18   Va&#180;zquez, A., Pastor-Satorras, R. &amp; Vespignani, A. 2002 <br /> M.T.H. M.M.H. acknowledge support Volkswagen                 Large-scale topological dynamical properties of <br /> Foundation (grant nos I/82 717 I/83435). M.E.B.               internet. Phys. Rev. E 65, 066130. (doi:10.1103/ <br /> C.F. supported Deutsche Forschungsgemeinschaft                 PhysRevE.65.066130) <br /> grants MTH (grant nos HU-937/6 HU-937/7).                 19   Ravasz, E. &amp; Baraba&#180;si, A. 2003 Hierarchical organization <br />                                                                       complex networks. Phys. Rev. E 67, 026112. (doi:10. <br />                                                                       1103/PhysRevE.67.026112) <br /> REFERENCES                                                       20   Newman, M. E. 2003 The structure function com- <br />                                                                       plex networks. SIAM Rev. 45, 167&#8211; 256. (doi:10.1137/ <br />  1 Milo, R., Shen-Orr, S., Itzkovitz, S., Kashtan, N.,                S003614450342480) <br />    Chklovskii, D. &amp; Alon, U. 2002 Network motifs: simple         21   Callaway, D. S., Hopcroft, J. E., Kleinberg, J. M., <br />    building blocks complex networks. Science 298,                  Newman, M. E. J. &amp; Strogatz, S. H. 2001 Are randomly <br />    824&#8211; 827. (doi:10.1126/science.298.5594.824)                       grown graphs really random? Phys. Rev. E 64, 041902. <br />  2 Milo, R., Itzkovitz, S., Kashtan, N., Levitt, R., Shen-Orr,        (doi:10.1103/PhysRevE.64.041902) <br />    S., Ayzenshtat, I., Sheffer, M. &amp; Alon, U. 2004 Super-        22   Newman, M. E. J. 2002 Assortative mixing networks. <br />    families evolved designed networks. Science 303,            Phys. Rev. Lett. 89, 208701. (doi:10.1103/PhysRevLett. <br />    1538&#8211;1542. (doi:10.1126/science.1089167)                           89.208701) <br />  3 Shen-Orr, S. S., Milo, R., Mangan, S. &amp; Alon, U. 2002         23   Jamakovic, A., Mahadevan, P., Vahdat, A., Boguna, M. &amp; <br />    Network motifs transcriptional regulation                   Krioukov, D. How small building blocks complex <br />    network Escherichia coli. Nat. Genet. 31, 64&#8211;68.                networks? (http://arxiv.org/abs/0908.1143) <br />    (doi:10.1038/ng881)                                           24   Vazquez, A., Dobrin, R., Sergi, D., Eckmann, J. P., Oltvai, <br />  4 Alon, U. 2007 Network motifs: theory experimental              Z. &amp; Baraba&#180;si, A. L. 2004 The topological relationship <br />    approaches. Nat. Rev. Genet. 8, 450 &#8211;461. (doi:10.1038/            large-scale attributes local interaction <br />    nrg2102)                                                           patterns complex networks. Proc. Natl Acad. Sci. <br />  5 Konagurthu, A. &amp; Lesk, A. 2008 On origin distri-            USA 101, 17 940&#8211;17 945. (doi:10.1073/pnas.0406024101) <br />    bution patterns motifs biological networks. BMC         25   Holme, P. &amp; Zhao, J. 2007 Exploring assortativity- <br />    Syst. Biol. 2, 73. (doi:10.1186/1752-0509-2-73)                    clustering space network&#8217;s degree sequence. Phys. <br />  6 Artzy-Randrup, Y., Fleishman, S. J., Ben-Tal, N. &amp; Stone,          Rev. E 75, 046111. (doi:10.1103/PhysRevE.75.046111) <br />    L. 2004 <span id='am-63' about='rdfs:comment' typeof='owl:Thing'>Comment</span> &#8216;network motifs: simple building           26   Ingram, P., Stumpf, M. &amp; Stark, J. 2006 Network motifs: <br />    blocks complex networks&#8217; &#8216;superfamilies evolved          structure does determine function. BMC Genomics 7, <br />    designed networks&#8217;. Science 305, 1107. (doi:10.1126/           108. (doi:10.1186/1471-2164-7-108) <br />    science.1099334)                                              27   Avetisov, V., Nechaev, S. &amp; Shkarin, A. 2010 On motif <br />  7 Mazurie, A., Bottani, S. &amp; Vergassola, M. 2005 An evol-            distribution random block-hierarchical networks. Phys. <br />    utionary functional assessment regulatory                   A Stat. Mech. Appl. 389, 5895&#8211;5902. (doi:10.1016/j. <br />    network motifs. Genome Biol. 6, R35. (doi:10.1186/gb-              <span id='am-74' about='obo:NCBITaxon_109670' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_977780 obo:NCBITaxon_1 obo:NCBITaxon_977779 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_109669 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="rdfs:label" content="Physa" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>physa</span>.2010.09.016) <br />    2005-6-4-r35)                                                 28   Varma, A. &amp; Palsson, B. O. 1994 Metabolic &#64258;ux balan- <br />  8 Ginoza, R. &amp; Mugler, A. 2010 Network motifs come                cing: basic concepts, scienti&#65533;?c practical use. Nat. <br />    sets: correlations randomization process. Phys.             Biotechnol. 12, 994&#8211;998. (doi:10.1038/nbt1094-994) <br />    Rev. E 82, 011921. (doi:10.1103/PhysRevE.82.011921)           29   Del Genio, C. I., Kim, H., Toroczkai, Z. &amp; Bassler, K. E. <br />  9 Berger, A. &amp; Mu&#168;ller-Hannemann, M. 2010 Uniform                    2010 Ef&#65533;?cient exact sampling simple graphs with <br />    sampling digraphs &#65533;?xed degree sequence.                  given arbitrary degree sequence. PLoS ONE 5, e10012. <br />    In Graph theoretic concepts science                    (doi:10.1371/journal.pone.0010012) <br />  <br />  <br />  <br /> J. R. Soc. Interface (2012) <br />                                                            Artefacts network motif analyses       M. E. Beber et al. 3435 <br />  <br /> 30 McKay, B. &amp; Wormald, N. C. 1990 Uniform generation          50 Feist, A. M., Henry, C. S., Reed, J. L., Krummenacker, M., <br />    random regular graphs moderate degree. J. Algorithms           Joyce, A. R., Karp, P. D., Broadbelt, L. J., Hatzimanika- <br />    11, 52&#8211;67.                                                        tis, V. &amp; Palsson, B. &#216;. 2007 A genome-scale metabolic <br /> 31 McKay, B. &amp; Wormald, N. C. 1991 Asymptotic enumer-                reconstruction <span id='am-69' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli K-12</span> MG1655 that <br />    ation degree sequence graphs degrees o(n 1/2).         accounts 1260 ORFs thermodynamic information. <br />    Combinatorica 11, 369 &#8211;382. (doi:10.1007/BF01275671)              Mol. Syst. Biol. 3, 121. (doi:10.1038/msb4100155) <br /> 32 Cooper, C., Dyer, M. &amp; Greenhill, C. 2007 Sampling regu-       51 Sonnenschein, N., Geertz, M., Muskhelishvili, G. &amp; Hu&#168;tt, <br />    lar graphs peer-to-peer network. Comb. Probab.              M. T. 2011 Analog regulation metabolic demand. BMC <br />    Comput. 16, 557 &#8211;593. (doi:10.1017/S0963548306007978)             Syst. Biol. 5, 40. (doi:10.1186/1752-0509-5-40) <br /> 33 Bayati, M., Kim, J. H. &amp; Saberi, A. 2010 A sequential          52 Latapy, M., Magnien, C. &amp; Vecchio, N. D. 2008 Basic <br />    algorithm generating random graphs. Algorithmica              notions analysis large two-mode networks. Soc. <br />    58, 860&#8211; 910. (doi:10.1007/s00453-009-9340-1)                     Netw. 30, 31 &#8211;48. (doi:10.1016/j.socnet.2007.04.006) <br /> 34 Itzkovitz, S., Milo, R., Kashtan, N., Ziv, G. &amp; Alon, U.       53 Ravasz, E., Somera, A. L., Monaru, D. A., Oltvai, Z. N. &amp; <br />    2003 Subgraphs random networks. Phys. Rev. E 68,               Baraba&#180;si, A. 2002 Hierarchical organization modularity <br />    026127. (doi:10.1103/PhysRevE.68.026127)                          metabolic networks. Science 297, 1551&#8211;1555. (doi:10. <br /> 35 Wernicke, S. 2005 FANMOD: tool fast network motif           1126/science.1073374) <br />    detection. Bioinformatics 22, 1152&#8211; 1153. (doi:10.1093/        54 Guimera, R. &amp; Amaral, L. 2005 Functional cartography of <br />    bioinformatics/btl038)                                            complex metabolic networks. Nature 433, 895&#8211;900. <br /> 36 Schreiber, F. &amp; Schwo&#168;bbermeyer, H. 2005 MAVisto: tool          (doi:10.1038/nature03288) <br />    exploration network motifs. Bioinformatics 21,      55 Samal, A., Singh, S., Giri, V., Krishna, S., Raghuram, N. &amp; <br />    3572&#8211;3574. (doi:10.1093/bioinformatics/bti556)                    Jain, S. 2006 Low degree metabolites explain essential reac- <br /> 37 Batagelj, V. &amp; Mrvar, A. 2001 A subquadratic triad census         tions enhance modularity biological networks. BMC <br />    algorithm large sparse networks small maximum            Bioinformatics 7, 118. (doi:10.1186/1471-2105-7-118) <br />    degree. Soc. Netw. 23, 237 &#8211;243. (doi:10.1016/S0378-           56 Fortunato, S. 2010 Community detection graphs. Phys. <br />    8733(01)00035-1)                                                  Rep. 486, 75&#8211; 174. (doi:10.1016/j.physrep.2009.11.002) <br /> 38 Fretter, C., Mu&#168;ller-Hannemann, M. &amp; Hu&#168;tt, M. T. 2012         57 Jeong, H. 2009 Analysis E. coli network. In Systems <br />    Subgraph &#64258;uctuations random graphs. Phys. Rev. E               biology biotechnology Escherichia coli (ed. S.Y. <br />    85, 056119. (doi:10.1103/PhysRevE.85.056119)                      Lee), pp. 113&#8211;132. The Netherlands: Springer. <br /> 39 Albert, R. 2005 Scale-free networks cell biology. J. Cell   58 Chang, X., Wang, Z., Hao, P., Li, Y. &amp; Li, Y. 2010 Exploring <br />    Sci. 118, 4947 &#8211;4957. (doi:10.1242/jcs.02714)                     mitochondrial evolution metabolism organization prin- <br /> 40 Zhou, W. &amp; Nakhleh, L. 2011 Properties metabolic               ciples comparative analysis metabolic networks. <br />    graphs: biological organization representation artifacts?      Genomics 95, 339&#8211;344. (doi:10.1016/j.ygeno.2010.03.006) <br />    BMC Bioinf. 12, 132. (doi:10.1186/1471-2105-12-132)            59 Kaluza, P., Vingron, M. &amp; Mikhailov, A. S. 2008 Self-cor- <br /> 41 Papp, B., Teusink, B. &amp; Notebaart, R. A. 2009 A critical          recting networks: function, robustness, motif <br />    view metabolic network adaptations. HFSP J. 3, 24 &#8211;35.         distributions biological signal processing. <span id='am-53' about='obo:NCBITaxon_160821' typeof='owl:Thing obo:NCBITaxon_555370 obo:NCBITaxon_1 obo:NCBITaxon_180229 obo:NCBITaxon_131567 obo:NCBITaxon_35218 obo:NCBITaxon_555369 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Chaos" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57'  ></span>Chaos</span> 18, <br />    (doi:10.2976/1.3020599)                                           026113. <br /> 42 Basler, G., Grimbs, S., Ebenho&#168;h, O., Selbig, J. &amp;             60 Mangan, S. &amp; Alon, U. 2003 Structure function the <br />    Nikoloski, Z. 2011 Evolutionary signi&#65533;?cance metabolic          feed-forward loop network motif. Proc. Natl Acad. Sci. <br />    network properties. J. R. Soc. Interface 9, 1168 &#8211;1176            USA 100, 11 980&#8211;11 985. (doi:10.1073/pnas.2133841100) <br />    (doi:10.1098/rsif.2011.0652)                                   61 Kaplan, S., Bren, A., Dekel, E. &amp; Alon, U. 2008 The inco- <br /> 43 Jeong, H., Tombor, B., Albert, R., Oltvai, Z. N. &amp;                herent feed-forward loop generate non-monotonic <br />    Barabasi, A. 2000 The large-scale organization meta-           input functions genes. Mol. Syst. Biol. 4, 203. (doi:10. <br />    bolic networks. Nature 407, 651 &#8211;654. (doi:10.1038/               1038/msb.2008.43) <br />    35036627)                                                      62 Krumov, L., Fretter, C., Mu&#168;ller-Hannemann, M., Weihe, <br /> 44 Wagner, A. &amp; Fell, D. A. 2001 The small world inside              K. &amp; Hu&#168;tt, M. T. 2011 Motifs co-authorship networks <br />    large metabolic networks. Proc. R. Soc. Lond. B 268,              relation impact scienti&#65533;?c publications. <br />    1803&#8211;1810. (doi:10.1098/rspb.2001.1711)                           Eur. Phys. J. B 84, 535&#8211;540. (doi:10.1140/epjb/e2011- <br /> 45 Montanez, R., <span id='am-58' about='obo:NCBITaxon_569144' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43755 obo:NCBITaxon_27474 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_43917 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_43742 obo:NCBITaxon_50557 obo:NCBITaxon_179436'><span id='am-59' property="rdfs:label" content="Medina" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Medina</span>, M. A., Sole&#180;, R. V. &amp; Rodr&#305;&#180;guez-           10746-5) <br />    Caso, C. 2010 When metabolism meets topology:                  63 Marr, C., Theis, F. J., Liebovitch, L. S. &amp; Hu&#168;tt, M. 2010 <br />    reconciling metabolite reaction networks. BioEssays           Patterns subnet usage reveal distinct scales regulation <br />    32, 246&#8211; 256. (doi:10.1002/bies.200900145)                        transcriptional regulatory network Escherichia <br /> 46 Eom, Y., Lee, S. &amp; Jeong, H. 2006 Exploring local struc-          coli. PLoS Comput. Biol. 6, e1000836. (doi:10.1371/ <br />    tural organization metabolic networks using subgraph           journal.pcbi.1000836) <br />    patterns. J. Theor. Biol. 241, 823 &#8211;829. (doi:10.1016/j.       64 Sonnenschein, N., Marr, C. &amp; Hu&#168;tt, M. T. Submitted. <br />    jtbi.2006.01.018)                                                 A topological characterization medium-dependent <br /> 47 Basler, G., Ebenho&#168;h, O., Selbig, J. &amp; Nikoloski, Z. 2011         essential metabolic reactions. Metabolites. <br />    JMassBalance: mass-balanced randomization metabolic         65 Price, N. D., Reed, J. L. &amp; Palsson, B. &#216;. 2004 Genome- <br />    networks. Bioinformatics 27, 1397&#8211;1403. (doi:10.1093/             scale models microbial cells: evaluating conse- <br />    bioinformatics/btr145)                                            quences constraints. Nat. Rev. Microbiol. 2, 886&#8211;897. <br /> 48 Riehl, W. J., Krapivsky, P. L., Redner, S. &amp; Segre`, D. 2010      (doi:10.1038/nrmicro1023) <br />    Signatures arithmetic simplicity metabolic network       66 Klemm, K. &amp; Bornholdt, S. 2005 Topology biological <br />    architecture. PLoS Comput. Biol. 6, e1000725. (doi:10.            networks reliability information processing. Proc. <br />    1371/journal.pcbi.1000725)                                        Natl Acad. Sci. USA 102, 18 414&#8211; 18 419. (doi:10.1073/ <br /> 49 Ma, H. &amp; Zeng, A. 2003 The connectivity structure, giant          pnas.0509132102) <br />    strong component centrality metabolic networks.         67 Kittisopikul, M. &amp; Su&#168;el, G. M. 2010 Biological role noise <br />    Bioinformatics 19, 1423 &#8211;1430. (doi:10.1093/bioinfor-             encoded genetic network motif. Proc. Natl Acad. Sci. <br />    matics/btg177)                         <br /> </body></html>