<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Feb. 1987, p. 679-686                                                                    Vol. 7, No. 2 <br /> 0270-7306/87/020679-08$02.00/0 <br /> Copyright C 1987, American Society Microbiology <br />  <br />  <br />           The SPT6 Gene Is Essential Growth Is Required for <br />             5-Mediated Transcription <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />                                     CHRIS D. CLARK-ADAMS AND FRED WINSTON* <br />                         Department Genetics, Harvard Medical School, Boston, Massachusetts 02115 <br />                                            Received 18 July 1986/Accepted 7 November 1986 <br />  <br />              Mutations <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span> cerevisiae SPT6 gene originally identified <span id='am-30' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> extragenic <br />            suppressors Ty 8 insertion mutations 5&apos; noncoding regions HIS4 LYS2. We cloned SPT6 <br />            constructed null allele gene disruption. Haploid spores carrying spt6 null allele inviable, <br />            indicating SPT6 gene essential mitotic growth. SPT6 mapped right arm chromosome <br />            VII, 44 centimorgans (cM) ADE6 9 cM CLY8. We showed spt6 mutations suppress 8 <br />            insertion mutations level transcription qualitative quantitative effect Ty <br />            transcription. In addition, observed interesting SPT6 gene dosage effects. An SPT6 strain containing a <br />            high-copy-number plasmid clone SPT6 showed suppression 8 insertion mutations, diploid strain <br />            half normal dose SPT6 (SPT61spt6 null) exhibited suppression 8 insertion mutations. <br />            Therefore, having too copies SPT6 causes mutant phenotype. Finally, study and <br />            accompanying paper (L. Neigeborn, J. L. Celenza, M. Carlson, Mol. Cell. Biol. 7:6794686, <br />            1986) showed spt6 ssn2O mutations (isolated suppressors snJ2 snJS [sucrose nonfermenting] <br />            mutations) identify gene. SPT6 SSN20 genetic map position share identical <br />            restriction map. Furthermore, spt6 ssn2O mutations fail complement other, ssn2O mutations <br />            suppress solo 8 insertion mutations HIS4 LYS2. These results, taken conjunction SPT6 <br />            dosage effects fact SPT6 essential gene, suggest SPT6 plays fundamental role cellular <br />            transcription, interaction transcription factors. <br />  <br />    Insertion transposable genetic elements adjacent          affect proteins recognize transcription sig- <br /> to gene variety effects expression           nals. <br /> gene. In general, insertion coding region gene               Selection suppressors Ty 8 insertion mutations <br /> will destroy gene function. Insertion 5&apos; noncoding             HIS4 LYS2 genes identified 12 trans-acting <br /> region intron, however, predictable               SPT genes (SPT, suppressor Ty) (48; J. Fassler F. <br /> phenotype. Such insertions place adjacent gene                  Winston, unpublished <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98'  ></span>data</span>). In addition effects on <br /> under control new transcription signals allow modu-              Ty 8 insertions, different classes spt mutations cause <br /> lation gene expression different set gene products.          variety phenotypes include defects in <br /> Analysis suppressors types insertion mutations           sporulation, DNA repair, growth (48). These pleiotropic <br /> has identified genes Saccharomyces cerevisiae (8, 11, 48),           effects suggest SPT genes required different <br /> <span id='am-33' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span> (28, 33-35, 40, 51), mice (46)              aspects normal gene expression S. cerevisiae. <br /> that involved transcriptional regulation.                        Mutations trans-acting SPT gene, SPT6, strongly <br />   Ty elements S. cerevisiae divergent set trans-            suppress solo 8 insertion mutations HIS4 (48) LYS2 <br /> posable elements. They 5.9 kilobases (kb) long                  (unpublished data) weakly suppress Ty <br /> consist long terminal repeats, 8 sequences, 330 base              insertion mutation. In addition, spt6 mutations confer <br /> pairs flanking internal e region. Ty elements strongly              temperature-sensitive lethality (48), suggesting SPT6 is <br /> resemble mammalian retroviral proviruses                  essential mitotic growth. In report demonstrate <br /> shown transpose RNA intermediate (2)                   null allele SPT6 lethal spt6 mutations <br /> encode reverse transcriptase activity (20, 27). Ty elements           suppress 8 insertion mutations transcriptional level. <br /> are highly transcribed; transcription initiates 5&apos; 8             However, spt6 mutations distinct previously stud- <br /> sequence terminates 3&apos; 8 sequence (13).                      ied mutations SPT gene, SPT3, not <br />    Insertion Ty element 5&apos; noncoding region          alter transcription complete Ty elements. We show <br /> gene strikingly alter expression gene. Ty insertion         alteration SPT6 gene dosage causes mutant pheno- <br /> mutations identified prevent adjacent gene               type: having too copies wild-type <br /> expression (12, 37, 43) cause altered regulation gene             SPT6 causes suppression 8 insertion mutations. <br /> expression, placing adjacent gene mating-                 spt6 mutations isolated suppressors 8 insertion <br /> type control (14, 47). The mutant phenotypes caused                  mutations (48). However, unrelated selection scheme has <br /> insertion Ty elements solo 8 derivatives                yielded mutations SPT6 gene. Suppressors snJ2 <br /> manifest transcriptional level. In cases           snJf (sucrose nonfermenting) mutations identify gene, <br /> the phenotypes interference normal expression             SSN20 (31), genetic map position SPT6 <br /> by transcription signals adjacent 8 sequence.              shares identical restriction map (30). We showed that <br /> Thus, unlinked suppressors Ty 8 insertion mutations              spt6 ssn20 mutations fail complement for <br />                                                                         temperature-sensitive lethality ssn20 mutations <br />                                                                         suppress solo 8 insertion mutations HIS4 L YS2. <br />   * <br />       Corresponding author.                                             These results, taken conjunction SPT6 gene <br />                                                                   679 <br />  680        CLARK-ADAMS AND WINSTON                                                                            MOL. CELL. BIOL. <br />  <br />                      TABLE 1. Yeast strains                       transformed ampicillin resistance described previously <br />   Strain                Genotype                     Source       (24). <br />                                                                      Enzymes. Restriction enzymes T4 DNA ligase were <br /> S288C        MA Tx gal2                                           purchased New England BioLabs, Inc. (Beverly, <br /> CC132    MATa/MATcx his4-91281his4-9128                           Mass.). Calf intestinal alkaline phosphatase purchased <br />            lys2-12881lys2-1288                                    Boehringer Mannheim Biochemicals (Indianapolis, <br />            SPT6tspt6-140 ura3-521ura3-52                          Ind.). Enzymes used according instructions of <br /> FW619    MATa his4-9128 lys2-1288 <br />            ura3-52                                                vendor. <br /> JF62     MATa his4-9128 lys2-1288              J. Fassler            DNA preparation analysis. Yeast DNA miniprepara- <br />            spt6-140 ura3-52 trp5-l leul                           tions 10-ml overnight cultures described <br /> K382-19D MATot spoll urq3 canl cyh2            23                 previously (49). E. coli plasmid DNA minipreparations were <br />            ade2 his7 hom3 tyrl                                    prepared alkaline lysis method (1). DNA restriction <br /> K381-9D MATot spoll ura3 ade6 arg4             23                 fragments analyzed electrophoresis 0.6 or <br />            aro7 asp5 metl4 Iys2 petl 7                            1.0% Seakem (FMC Corp., Marine Colloids Div., Rockland, <br />            trpI                                                   Maine) ME agarose gels run 0.5x E buffer (38). <br /> K393-35C MAToa spoll ura3 his2 leul lysi       23                 We purified restriction fragments electroelution using a <br />            met4 pet8 <br /> K396-22B MATa spoll ura3 adel hisl leu2        23                 device designed Larry Peck (personal communication). <br />            Iys7 met3 trpS                                         Southern blot hybridization analysis yeast genomic DNA <br /> FW1228   MATa his4-9128 lys2-1288                                 performed described previously (38). 32P-labeled <br />            ura3-52                                                DNA probes prepared nick translation (36). <br /> CC3      MA Ta spt6-140                                              <span id='am-104' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span>. Plasmid <span id='am-79' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> used SPT6 subcloning were <br /> FW145    MATa his4-9128 spt6-167 cani R                           YCp50 YEp24 (3, 22). Plasmids used probes as <br /> CC8      MATa his4-9178 spt6-140 ura3-52                          follows: SPT6, pCC36, SPT6 internal ClaI-SalI re- <br /> MCY846 MATa ade6 cly8 leu2-3                   M. Carlson         striction fragment cloned pBR322; HIS4, pFW45, a <br /> FW600    MATa his4-9128 lys2-1288                                 HIS4 internal BglII-SalI restriction fragment pBR322 (50); <br /> CC136    MATa his4-9128 lys2-1288 <br />            spt6-140                                               LYS2, pFW47, LYS2 internal BglII-XhoI restriction <br /> FW1238   MATa his4-9128 lys2-1288                                 fragment pBR322, pFW112, LYS2 EcoRI-BglII <br />            ura3-52                                                fragment spanning 5&apos; end L YS2, cloned pBR322; <br /> CC135    MA Ta/MA Ta his4-91281his4-9128                          Ty, B161 (R. Surosky, B.-K. Tye, G. R. Fink, <br />            lys2-12881lys2-1288                                    unpublished data), Tyl BglII restriction fragment in <br />            SPT61spt6-140 URA31ura3-52                             pBR322; pyruvate kinase, pFR2, clone containing the <br /> CC139    MA TalMA Ta his4-91281his4-9128                          pyruvate kinase gene PYKI (kindly provided Pratima <br />                 Iys2-12881lys2-1286                               Sinha). <br />                SPT61spt6:: URA3                                      RNA blot hybridization analysis. Cells RNA isolation <br />                ura3-521ura3-52 <br /> CC73          MATa lys2-1288 spt6-140 ade2-1                      grown supplemented SD medium 107 cells ml, <br />                leu2-3, 112 trplAl ura3-52                         yeast RNA isolated described previously (5). For <br /> MCY938        MATa ssn20-1 his4-539 lys2-801   L. Neigeborn   RNA blots hybridization analysis, followed the <br />                 ade2-101                         M. Carlson       instructions GeneScreen (New England Nuclear Corp., <br /> MCY940        MATa ssn2O-7 his4-539 Iys2-801   L. Neigeborn   Boston, Mass.) using dextran sulfate method. RNA was <br />                 ura3-52                          M. Carlson       cross-linked GeneScreen exposure UV radiation <br />                                                                   (1,200 iiW/cm2 2 min) (7). The RNA lane <br />                                                                   standardized hybridization pFR2. Bands auto- <br /> dosage effects fact SPT6 essential gene,       radiograms quantitated densitometry. Sizes were <br /> suggest SPT6 plays fundamental role cellular            determined RNA size markers Bethesda Research <br /> transcription, interaction transcription    Laboratories, Inc. (Gaithersburg, Md.) according their <br /> factors.                                                          instructions. <br />                                                                       Construction SPT6 null allele. We subcloned 6-kb BglII <br />                  MATERIALS AND METHODS                            restriction fragment containing SPT6 BamHI site of <br />   Strains genetic methods. The S. cerevisiae strains          pFW3. pFW3 derivative YIp5 (45) the <br /> constructed studies derivatives strain S288C     HindIII site destroyed (F. Winston, unpublished <br /> (Table 1). We used standard genetic nomenclature: upper-          data). The resulting subclone, pCC16, unique HindlIl <br /> case letters denote dominant allele, lowercase letters       site SPT6 gene. A 1.2-kb HindlIl restriction <br /> indicate recessive allele. The designation his4-9128 indi-      fragment containing URA3 gene subcloned both <br /> cates solo 8 insertion mutation located base pair -97         orientations single HindlIl site pCC16, creating <br /> relative start normal HIS4 transcription (6, 18).        plasmids pCC41 pCC42. The URA3 gene these <br /> lys2-1288 refers solo 8 insertion mutation located 153        plasmids flanked NheI ClaI restriction sites present <br /> base pairs downstream L YS2 translation initiation          SPT6 sequences. We purified NheI-ClaI restriction <br /> <span id='am-51' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span>codon</span> (43; F. Winston, unpublished data). Standard meth-           fragment pCC41 pCC42 used transform the <br /> ods mating, sporulation, tetrad analysis used         diploid strain CC132. Ura+ transformants selected on <br /> (29, 41).                                                          SC-ura. A double crossover event replace the <br />    Media. All media described previously (41).        genomic SPT6 alleles URA3-disrupted SPT6 allele <br /> These include rich medium (YPD), minimal medium (SD),              (32). To analyze phenotype spt6 null allele in <br /> and sporulation medium. SC-ura synthetic complete me-           haploid cells, sporulated Ura+ transformants and <br /> dium lacking uracil.                                               dissected tetrads. <br />    Transformations. Yeast cells transformed               Genetic mapping. We localized SPT6 chromosome VII <br /> lithium acetate method (21). <span id='am-56' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> HB101 (4)        using 2,um-mapping method (16). <span id='am-40' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> technique relies <br />  VOL. 7, 1987                                          SPT6 AND 5-MEDIATED TRANSCRIPTION IN S. CEREVISIAE                          681 <br />  <br /> on observation integration segment the <br /> yeast 2,um circle chromosome causes instability                                               Q        (-   Qo <br /> markers chromosome (17). The 6-kb BglII fragment                                       II <br />                                                                                                              , <br />  <br />  <br />  <br /> containing SPT6 subcloned 2p.m vector YEp24,                                                   U))       U)r <br /> creating plasmid pCC14. To obtain plasmid integrants the <br /> SPT6 locus, pCC14 digested BamnHI, creating <br /> a linear form SPT6 ends. Transformation DNA <br /> into strain FW619 expected yield integrants SPT6 <br /> (32). Stable Ura+ transformants shown                             56- <br /> plasmid integrated SPT6 locus cross strain                          42- <br /> JF62 Southern hybridization analysis. The transform- <br /> ants mated mapping strains K382-19D, <br /> K381-9D, K393-35C, K396-22B. Prototrophic diploids <br /> were selected minimal medium, grown overnight YPD                             / 7- <br /> medium, plated single colonies YPD plates, then <br /> replica plated SC-ura plates SD plates supple- <br /> mented uracil. This allowed identify segregants <br /> that Ura- segregants gained <br /> another auxotrophy. Linkage SPT6 markers chro- <br /> mosome VII determined tetrad analysis. We scored <br /> spt6-140 temperature-sensitive lethality sup- <br /> pression his4-9128 lys2-1288. <br />                                                                    FIG. 1. Northern analysis SPT6-specific RNA. Total RNA <br />                                                                 isolated, 15 p.g sample run lane. The probe <br />                            RESULTS                              pCC36, internal fragment SPT6 cloned pBR322. The <br />                                                                 faint lower bands caused presence 25S 18S rRNA. <br />    Some spt6 alleles confer temperature-sensitive lethality;    The numbers left indicate size markers obtained reprobing <br /> therefore, wanted determine SPT6 required      filter probes Tyl (B161), LYS2 (pFW47), PYK <br /> only nonpermissive temperature SPT6        (pFR2). These probes hybridize mRNAs 5.6, 4.2, 1.7 kb, <br /> an essential gene conditions. Since vivo-      respectively. The strains represented (left right): S288C, CC3, <br /> isolated spt6 mutations uncharacterized defects,       FW1228, JF62, FW145. <br /> mutant phenotype cause (suppression 5 insertion <br /> mutations) loss, gain, alteration SPT6 <br /> function. To address issues, cloned gene       crossed strain FW619, tetrads dissected. All 20 <br /> constructed null allele.                                      tetrads dissected displayed 0:4 segregation both <br />    Cloning SPT6 identification mRNA. SPT6 gene       His+:His- Lys+:Lys-, indicating plasmid had <br /> clones isolated yeast recombinant library        integrated SPT6 locus. <br /> their ability complement temperature-sensitive spt6           Analysis yeast genomic DNA Southern blot <span id='am-99' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>- <br /> mutation vivo. The library used contains Sau3A partial    ization indicated SPT6 single-copy gene unassoci- <br /> restriction fragments yeast genomic DNA in-         ated repeated sequences (data shown). Genomic <br /> serted unique BamHI site vector YCpSO (M.         DNA digested BglII, HindIII, BamHI then <br /> Rose P. Novick, unpublished data). YCp5O stable,       probed following plasmids: 1-kb ClaI-SalI internal <br /> low-copy-number plasmid carries ampicillin resistance      fragment SPT6 subcloned pBR322 6-kb BglII <br /> and URA3 gene provide selectable markers E. coli      fragment containing SPT6 gene subcloned pBR322. <br /> and S. cerevisiae (22). The recipient strain cloning SPT6   Since hybridization pattern DNA prepared from <br /> was JF62 (MATTa spt6-140 his4-9128 lys2-1288 ura3-52 trp5-1     strains carrying mutations spt6-140 spt6-167 was <br /> leul). The spt6-140 mutation confers temperature-sensitive      indistinguishable SPT6 DNA, mutant <br /> lethality; thus, strain JF62 grow 300C 370C   alleles DNA rearrangements detectable Southern <br /> on rich medium. The his4-9128 lys2-1288 alleles 8       hybridization analysis. <br /> insertion mutations suppressed spt6-140; thus,         The SPT6 mRNA identified Northern blot hybrid- <br /> JF62 His&apos; Lys&apos; phenotype. Since spt6-140 recessive     ization total yeast RNA. The probe, 1-kb internal <br /> to SPT6+, transformants JF62 acquired copy       ClaI-SalI restriction fragment SPT6 subcloned into <br /> of wild-type SPT6 gene grow 370C       pBR322, hybridized single 4.5-kb transcript SPT6 <br /> His- Lys- phenotype. Five candidates         strains (Fig. 1, lanes 1 3). Strains carrying mutations <br /> screening 50,000 Ura+ transformant colonies growth       spt6-140 spt6-167 showed identical 4.5-kb transcript <br /> 37&#176;C. All shared common restriction fragments.             approximately equal intensity SPT6 band <br />    We confirmed cloned SPT6 gene             (Fig. 1, lanes 2, 4, 5). Therefore, mutant alleles do <br /> demonstrating restriction fragment putative     affect level SPT6 transcript. <br /> clones direct integration plasmid SPT6          SPT6 null allele. To construct SPT6 null allele, we <br /> locus. A 6-kb BglII fragment common SPT6 clones          constructed subclones SPT6 region identify restric- <br />  vas subcloned YCp5O shown contain SPT6         tion sites gene (Fig. 2). Three sites were <br /> function transformation S. cerevisiae. The 6-kb      6-kb BglII fragment. Subclones frag- <br /> BglII fragment subcloned integrating          ments HindlIl site, ClaI site, the <br /> plasmid pFW3. This plasmid transformed strain          Sall site nearest ClaI site did retain SPT6 <br /> JF62, selecting Ura+ transformants. All Ura+ transform-     activity, indicating sites SPT6 <br /> ants His- Lys- contained wild-type SPT6      gene. We constructed spt6 insertion mutations ligating a <br /> gene associated plasmid. A Ura+ transformant       1.2-kb HindlIl fragment containing URA3 gene the <br />  682     CLARK-ADAMS AND WINSTON                                                                                              MOL. CELL. BIOL. <br />  <br />                      m <br />                                                  x    4 <br />  <br />                                                  E    E            0-                            0 <br />  <br />                      ._                  c   _ <br />                                                  ao <br />                                                  m    OD   Cn Cn   -C <br />                                                                    cn                           x                 SPT6 <br />                           11        li                    II                                         Subclone    Function <br />                                                                                                        pCCI <br />                                                                                                        pCC17 <br />                                                                                                        pCC 18 <br />                                                                                                        pCC 19 <br />                                                                                                        pCC 10 <br />                                                                                                        pCC32 <br />  <br />  <br />                                Kb <br />   FIG. 2. Restriction map SPT6 locus. The line represents SPT6 DNA restriction sites. The lines represent <br /> restriction fragments subcloned YCp5O. The subclones tested SPT6 function transformation spt6 mutant <br /> strain JF62. <br />  <br /> internal HindIlI site SPT6. A restriction fragment con-              previously reported (42), SPT6 his4-9128 strain which <br /> taining URA3 insertion used replace               His-, predominant HIS4 transcript initiates 8 <br /> genomic SPT6 sequences diploid strain CC132                      longer wild-type HIS4 transcript <br /> (SPT61spt6-140) described Materials Methods. Of               (Fig. 4, lane 3). Based DNA sequence (10, 18), this <br /> eight Ura+ transformants isolated, temperature               transcript AUG UAA codon in <br /> sensitive, indicating spt6 null allele (designated          frame upstream HIS4 AUG. Assuming normal <br /> spt6::URA3) replaced wild-type SPT6 sequence                 HIS4 AUG codon used, longer transcript <br /> yield spt6:: URA3Ispt6-140. The temper-             nonfunctional HIS4 activity (42). In spt6 <br /> ature sensitive presumably SPT61spt6::URA3                  his4-9128 strains, His&apos;, wild-type-size HIS4 <br /> configuration. Southern blot hybridization analysis con-                transcript present addition 8-mediated tran- <br /> firmed Ura+ transformants heterozygous              script (Fig. 4, lanes 4 5). spt6 mutations apparently did <br /> 1.2-kb insertion HindIII site SPT6 region.                significant effect level 8-initiated <br />    To determine phenotype spt6::URA3 haploid                  transcript. Thus, spt6 mutation restores functional <br /> strains, Ura+ diploid transformants sporulated,               HIS4 transcript his4-9128. <br /> tetrads dissected, spores allowed germi-                  For lys2-1288 strains, different type transcriptional <br /> nate 30&#176;C. In 47 48 tetrads dissected,              alteration observed. In contrast his4-9128, which <br /> more viable spores tetrad (29 segregated 2:2           8 sequence inserted 5&apos; noncoding region of <br /> viability:inviability, 14 segregated 1:3, 4 segregated 0:4).        HIS4, 8 sequence lys2-1288 inserted 5&apos; end <br /> Most spores failed form colonies present            LYS2 coding region. We used LYS2 probes to <br /> as single cells dissection plate, divided         study transcription lys2-1288: pFW47, internal LYS2 <br /> up times dying. The tetrad yielded             probe, pFW112, probe containing 5&apos; noncoding and <br /> four viable spores segregated 0:4 Ura+:Ura-                  coding sequences. These probes used individually in <br /> apparently mitotic recombinant. Since Ura+ spores                initial experiments subsequently mixed later <br /> contain spt6:: URA3 inviable, SPT6 essential mi-             experiments. For SPT6 lys2-1288 strain, Lys-, <br /> totic growth.                                                           normal 4.2-kb LYS2 message detected the <br />    Mapping SPT6. SPT6 localized chromosome                  internal L YS2 probe pFW47 (Fig. 5, lane 3). However, new <br /> the 2,um-mapping method (16, 17). In diploids           580-base-pair L YS2 transcript detected 5&apos; probe <br /> had 2p&gt;m plasmid containing URA3 SPT6 genes                 pFW112. The size short transcript consistent with <br /> integrated SPT6 locus, co-loss Ura+              transcript initiates normal LYS2 tran- <br /> phenotype Trp+ phenotype low frequency (five                scription start site terminates 8 sequence. In an <br /> Trp- colonies 180 Ura- segregants). Since mul-                spt6-140 lys2-1288 double mutant (Fig. 5, lane 4), internal <br /> tiply marked parent (K396-22B) used construct dip-              L YS2 probe detects L YS2 transcript comigrates with <br /> loid transformed parent trpS marker,                 wild-type LYS2 message. This transcript func- <br /> co-loss Trp+ Ura+ localizes SPT6 chromosome                  tional strain Lys&apos;. The 5&apos; LYS2 probe detects a <br /> VII. By tetrad analysis, showed SPT6 9                        short L YS2 transcript, similar size 5&apos; transcript seen <br /> centimorgans (cM) cly8 44 cM ade6                  SPT6 lys2-1288 strain. <br /> right arm chromosome VII (Table 2; Fig. 3). <br />    spt6 affects transcription 8 insertion mutations                  TABLE 2. Mapping SPT6 tetrad analysisa <br /> Ty elements. Previous work shown mutations spt6             Segregating markers           PD           NPD             TT           cM <br />  suppressed 8 insertion mutation his4-9128 His-                                           58             0             12            9 <br /> His+ (48). We shown (J. Fassler, C. Clark-Adams,                  spt6, cly8 <br />                                                                             cly8, ade6                25             1             44           36 <br />  F. Winston, unpublished data) spt6 mutations                      spt6, ade6                23             2             45           44 <br />  suppress lys2-1288 Lys- Lys+. We determined the <br />  effect spt6 transcription HIS4 L YS2 8            a <br />                                                                             The tetrads scored corne cross CC8 x MCY846. PD, Parental <br />  insertion mutation alleles Northern hybridization analy-            ditype; NPD, nonparental ditype; TT, tetratype. spt6-140 scored its <br /> sis. <br />                                                                         temperature-sensitive lethality 37&#176;C. cly8 scored temperature- <br />                                                                         sensitive lethality 30&apos;C. Genotypes confirmed mating spore clones <br />   In his4-9128 strains, HIS4 transcription occurs               spt6-140 cly8 strains scoring diploids complementation of <br /> SPT6 spt6 strains, transcripts differ size. As           temperature-sensitive lethal phenotypes. <br />  VOL. 7, 1987                                                      SPT6 AND 8-MEDIATED TRANSCRIPTION IN S. CEREVISIAE                             683 <br />  <br />  <br />                                                         /     &apos;lb <br />                                         ,oe                                4     \e          cobcb.                   ,oe <br />                                     /               /       / /            ,          I/ <br />  <br />   FIG. 3. Genetic map position SPT6. spt6-140 strains crossed strains carrying markers indicated chromosome VII. <br />  <br />    spt6 spt3 mutations suppress his4-9128                         Thus, null allele recessive haploid strain. We <br /> lys2-1288 (43, 48). However, unlike spt3 mutations, spt6                       conclude reduction level SPT6 function causes <br /> mutations qualitatively affect general cellular Ty                      suppression 8 insertion mutations. <br /> transcription (Fig. 6). The observed transcriptional effects                      We determined mechanism suppression in- <br /> for spt6 mutants correspond pattern suppression:                   creased decreased SPT6 gene dosage Northern hybrid- <br /> strong suppression 8 insertion mutations weak                     ization analysis. The transcription patterns appeared identi- <br /> suppression complete Ty insertion mutations.                                cal spt6 mutants (data shown). Thus, solo 8 <br />    SPT6 gene dosage effects. To test increased                      insertion mutations suppressed similar transcrip- <br /> dosage SPT6 causes mutant phenotype, subcloned                         tional mechanism different cases examined. <br /> the 6-kb SPT6 BglII fragment high-copy-number                            SPT6 SSN20 gene. Genetic physical <br /> plasmid YEp24 (3). This subclone used transform                         data shown SPT6 SSN20 gene. <br /> either spt6-140 SPT6 strains, carrying 8 insertion                 The SSN20 gene identified mutations suppress <br /> mutations his4-9128 lys2-1288. The SPT6 high-copy                          sucrose-nonfermenting mutations snJ2 sniS (31). <br /> subclone did complement temperature-sensitive defect                       Both SPT6 SSN20 map near cly8 ade6 chromo- <br /> of spt6-140 strain, surprisingly did complement                    VII, share identical restriction map (30; <br /> the suppression (His&apos; Lys&apos;) phenotype. Furthermore,                        paper). ssn20-1 ssn2O-7, like spt6-140, cause reces- <br /> SPT6 high-copy subclone SPT6 wild-type strain                       sive temperature-sensitive lethality. Therefore, tested <br /> caused suppression 8 insertion mutations (Fig. 7, e f0.                 spt6-140 complement ssn20-1 or <br /> Therefore, increase SPT6 gene dosage causes sup-                       ssn2O-7 temperature-sensitive lethality crossing <br /> pression phenotype.                                                            spt6-140 strains ssn2O strains (CC73 MCY938 JF62 <br />    To test reduced gene dosage SPT6 causes                        MCY940). Zygotes isolated micromanipulation, <br /> mutant phenotype, examined phenotype diploid                       patched YPD plates, screened growth at <br /> strain homozygous his4-9128 lys2-1288                      high temperature (37&#176;C). Diploids cross failed to <br /> contains wild-type SPT6 copy copy                               grow 37&#176;C; thus, spt6-140 does complement either <br /> spt6::URA3. This strain showed suppression 8 inser-                     ssn2O allele. The spt6-1401ssn2O-7 diploids sporulated, <br /> tion mutations HIS4 LYS2 (Fig. 7, g h). That is,                    12 tetrads dissected. All 12 tetrads parental <br /> in diploid strain null allele dominant suppression.               ditypes (0 non-temperature-sensitive spores:4 temperature- <br /> One explanation spt6:: URA3 construction                        sensitive spores). <br /> producing truncated SPT6 protein product causes                            Since SPT6 SSN20 identical physically and <br /> suppression. To rule out, integrated spt6::URA3 <br /> construct haploid strain position adjacent the <br /> wild-type SPT6 gene. This strain did exhibit suppression.                                                     L YS2 /.ys2 -/28 <br />                                                                                                                      Q) <br />                               H/S4* his4-912,                                                                      t.               ;t <br />                                                                                                                           Q.-C: <br />                                                                                                                             .9 <br />                                     t    s Ck Q, <br />                                         to <br />  <br />  <br />  <br />  <br />                                                                                                       LYS2-N          4 <br />  <br />  <br />                      H/S44    _a              s <br />  <br />                      PYKI - <br />                                                                                                       PYKI   --   Q       A*        m <br />  <br />  <br />  <br />                                                                                                       L YS2 -&apos;-                40   40 <br />    FIG. 4. Northern analysis HIS4-specific RNA SPT6                    FIG. 5. Northern analysis LYS2-specific RNA SPT6 and <br /> spt6-140 strains. Total RNA isolated, 4 ,ug sample             spt6 strains. Total RNA isolated, 5 ,ug sample was <br /> was run lane. The probes pFW45 pFR2. HIS4 mRNA                    run lane. The probes pFW47, pFW112, PFR2. LYS2 <br /> is 2.6 kb length. The strains represented (left right): S288C,       mRNA 4.2 kb length. The strains represented (left right): <br /> CC3, FW1228, JF62, FW145.                                              S288C, CC3, FW1228, JF62. <br />  684     CLARK-ADAMS AND WINSTON                                                                                          MOL. CELL. BIOL. <br />  <br />                                                                         suppress solo 8 insertion mutations transcriptional <br />                                                                         level, increasing decreasing wild-type <br />                                                                         SPT6 gene dosage causes suppression 8 insertion muta- <br />                                                                         tions. In addition, studies Neigeborn et al. <br />                                                                         (30) demonstrated spt6 ssn2O mutations identify the <br />                                                                         gene. <br />                                                                            Ty solo 8 insertion mutations interfere expres- <br />                                                                         sion adjacent gene. Previous work (42, 50) suggests <br />                               _ <br />                                    -                                    interference caused presence of <br />                                                                         transcription signals transposon. Therefore, SPT <br />                                                                         genes encode proteins recognize transcription <br />                                                                         signals present Ty elements. <br />                                                                            If different SPT gene products recognize different tran- <br />                                                                         scription signals, expect loss gene prod- <br />                                                                         ucts result different transcription alterations dif- <br />                                                                         ferent suppression patterns. Mutations SPT6 distinct <br />                                                                         previously studied SPT3 these <br />                                                                         criteria. The SPT3 gene required transcription initia- <br />                                                                         tion 8 sequences alters transcription all <br />                                                                         cellular Ty elements. This defect explain spt3 <br />                                                                         mutations suppress Ty 8 insertion mutations (50). In <br />                                                                         contrast, spt6 mutations apparent effect total Ty <br />                                                                         transcription. spt3 spt6 mutations suppress his4- <br />                                                                         9128 lys2-1288 (43, 48). However, transcriptional <br />                                                                         alterations caused spt3 spt6 mutations differ. <br />                                                                         Further studies accurately define possible <br />    FIG. 6. Northern analysis Ty-specific RNA SPT6 spt6        transcriptional differences. <br /> strains. Total RNA isolated S288C CC3. A 5-,ug                spt3 mutations able suppress intact Ty <br /> sample run lane. The probe B161, Tyl              element insertion mutations, spt6 mutations only <br /> fragment cloned pBR322. The Ty mRNA 5.6 kb length.             weakly suppress single Ty insertion mutation. This differ- <br />                                                                         ence ability suppress complete Ty insertion mutations <br />                                                                         reflect differential effects spt3 spt6 total Ty <br /> genetically, examined ssn2O mutations suppress               transcription. Several enhancerlike regulatory sequences <br /> 8 insertion mutations spt6 mutations. We constructed              internal &#163; portion Ty elements <br /> strains containing ssn2O allele his4-9128                distinct transcriptional regulatory se- <br /> Iys2-1288. Both ssn20-1 ssn2O-7, like spt6 mutations,               quences 8 direct repeats (15, 39; P. Farabaugh, <br /> suppress phenotype solo 8 insertion mutations                personal communication). The SPT3 SPT6 gene prod- <br /> from minus plus. Also like spt6 mutations, ssn2O muta-               ucts differ interactions various <br /> tions recessive suppression. In addition, spt6-140          internal enhancer sequences. Since SPT6 essential <br /> spt6-167 suppress invertase derepression defect snJ2             gene, product affects transcription signals pre- <br /> mutants (30).                                                            sent 8 elements Ty specific found <br />                                                                          associated genes. <br />                          DISCUSSION                                        Additional evidence SPT6 affects expression a <br />    Our studies demonstrated SPT6 essential gene               diverse set genes comes observation spt6 <br /> for mitotic growth S. cerevisiae, spt6 mutations                 mutations ssn2O mutations define gene. ssn2O <br />  <br />  <br />  <br />                                            SPT6 +                                          spt 6 140 <br />                                           L YS2 *                                           L YS2 * <br />  <br />                                         SP 76 *                                               5sp6- J40 <br />                                    /ys 2- /283                                                fys2 - 1288 <br />  <br />                           SPT6 + /ys 2 -/288                                                  SPT6     /ys2 -1288 <br />                                (pCC/ /2                                                           ( p CC 14J <br />  <br />                             SP T6 &apos;/ spt6 - /40                                            SFPT6 5/spf6. URA3 <br />                          /ys2- 1288/ y.52 - /288                                          /ys2 - /283/ /ys2 - /288 <br />  <br />  <br />  <br />    FIG. 7. spt6 suppression lys2-1288. The strains indicated figure patched SD minimal plate supplemented all <br />  amino acid requirements replica plated plate shown, SD minimal plate required amino acids lysine. The <br />  strains represented are: a, S288C; b, CC3; c, FW600; d, CC136; e, FW1238 carrying low-copy-number SPT6 subclone pCC11; f, FW619 <br />  carrying high-copy-number SPT6 subclone pCC14; g, CC135; h, CC139. <br />  VOL. 7, 1987                                                  SPT6 AND 8-MEDIATED TRANSCRIPTION IN S. CEREVISIAE                           685 <br />  <br /> sent 8 elements Ty specific                3. Botstein, D., S. C. <span id='am-45' about='obo:NCBITaxon_8952' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8949 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8948 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-46' property="oboInOwl:hasExactSynonym" content="falcons" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Falco" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Falco</span>, S. E. Stewart, M. Brennan, S. Scherer, <br /> associated genes.                                                D. T. Stinchcomb, K. Struhl, R. W. Davis. 1979. Sterile host <br />   Additional evidence SPT6 affects expression                 yeasts (SHY): eukaryotic biological containment for <br />                                                                             recombinant DNA experiments. Gene 8:17-24. <br /> diverse set genes comes observation spt6                4. Boyer, H. W., D. Roulland-Dussoix. 1969. A complementa- <br /> mutations ssn2O mutations define gene. ssn2O                   tion analysis restriction modification DNA E. <br /> mutations isolated suppressors snJ2 snpS                     coli. J. Mol. Biol. 41:458-472. <br /> mutations (sucrose nonfermenters) (31)                5. Carlson, M., D. Botstein. 1982. Two differentially regulated <br /> shown suppress upstream regulatory region deletions                   mRNAs different 5&apos; ends encode secreted intracellular <br /> the SUC2 gene (30), mutations spt6 isolated                 forms yeast invertase. Cell 28:145-154. <br /> as suppressors 8 insertion mutation HIS4 (48). It         6. Chaleff, D. T., G. R. Fink. 1980. Genetic events associated <br /> obvious apparently different selection schemes                      insertion mutation yeast. Cell 21:227-237. <br /> have yielded mutations gene. One simple expla-                7. Church, G. M., W. Gilbert. 1984. Genomic sequencing. <br />                                                                              Proc. Natl. Acad. Sci. USA 81:1991-1995. <br /> nation, snJ2 snpS mutations used isolate ssn2O            8. Ciriacy, M., V. M. Williamson. 1981. Analysis mutations <br /> mutations 8 insertion mutations, ruled                affecting Ty-mediated gene expression Saccharomyces cere- <br /> ssn2O mutations suppress nonsense alleles snJ2 sniS                   i&apos;siae. Mol. Gen. Genet. 182:159-163. <br /> mutations (31). Ty element SUC2 expression share                  9. DeFranco, A. L., D. E. Koshland, Jr. 1981. Molecular <br /> some <span id='am-109' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span>unidentified</span> features.                                           cloning chemotaxis genes overproduction gene prod- <br />    The fact spt6 ssn2O mutations suppress types                 ucts bacterial sensing system. J. Bacteriol. 147:390-400. <br /> of disruptions, 8 insertion mutations HIS4 L YS2              10. Donahue, T. F., R. S. Daves, G. Lucchini, G. R. Fink. 1983. <br /> loci upstream regulatory region deletion mutations                A short nucleotide sequence required regulation HIS4 by <br /> SUC2 gene, suggests SPT6 involved process                  general control yeast. Cell 32:89-98. <br />                                                                          11. Dubois, E., E. Jacobs, J.-C. Jauniaux. 1982. Expression of <br /> transcription initiation. Also, observation alteration              ROAM mutations Saccharomyces cerei&apos;isiae: involve- <br /> of SPT6 gene dosage caused suppression 8 insertion                        ment trans-acting regulatory elements relation the <br /> mutations consistent <span id='am-25' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-26' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29'  ></span>idea</span> SPT6 gene                     Tyl transcription. EMBO J. 1:1133-1139. <br /> product acts complex, transcription              12. Eibel, H., P. Philippsen. 1984. Preferential integration of <br /> factors. Altering level SPT6 gene product relative                yeast transposable element Ty promoter region. Nature <br /> to factors disrupt proper stoichiometry, re-                   (London) 307:386-388. <br /> sulting lower level functional complex. Similar effects          13. Elder, R. T., E. Y. Loh, R. W. Davis. 1983. RNA the <br /> have observed assembly complexes                      yeast transposable element Tyl ends direct <br /> processes, including bacteriophage morphogenesis (19, 44), <br />                                                                              repeats, structure similar retrovirus RNA. Proc. NatI. <br />                                                                              Acad. Sci. USA 80:2432-2436. <br /> chemotaxis (9), mitotic chromosome transmission (25,                 14. Errede, B., T. S. Cardillo, F. Sherman, E. Dubois, J. Deschamps, <br /> 26).                                                                         J. M. Wiame. 1980. Mating signals control expression of <br />    We ruled possibility SPT6 cloned               mutations resulting insertion transposable element <br /> yeast RNA polymerase B subunit genes. The cloned SPT6                        adjacent diverse yeast genes. Cell 22:427-436. <br /> gene failed hybridize clones RPBJ, RPB2,                   15. Errede, B., M. Company, J. D. Ferchak, C. A. Hutchison III, <br /> RPB3, RPB4, RPB7, RPB9 subunits RNA polymerase                        W. S. Yarnell. 1985. Activation regions yeast trans- <br /> B (generously provided Richard Young) (data                           poson homology mating type control sequences to <br /> shown). However, vitro transcription                    mammalian enhancers. Proc. Natl. Acad. Sci. USA 82: <br /> that shows specific initiation available S.                   5423-5427. <br />                                                                          16. Falco, S. C., D. Botstein. 1983. A rapid chromosome- <br /> cerevisiae, likely exist unidentified                  mapping method cloned fragments yeast DNA. Genetics <br /> initiation factors, analogous E. coli factors.              105:857-872. <br /> SPT6 factor.                                             17. Falco, S. C., Y. Li, J. R. Broach, D. Botstein. 1982. Genetic <br />    Since SPT6 essential, play general role               properties chromosomally integrated 2u plasmid DNA in <br /> cellular transcription required expression               yeast. Cell 29:573-584. <br /> only genes, essential. Given          18. Farabaugh, P. J., G. R. Fink. 1980. Insertion the <br /> result spt6 (ssn2O) mutations affect 5-mediated                eukaryotic transposable element Tyl creates 5-base pair <br /> glucose-regulated gene expression, favor idea.                 duplication. Nature (London) 286:352-356. <br /> Further elucidation function SPT6 genes                19. Floor, E. 1970. Interaction morphogenetic genes of <br /> whose expression affects help answer questions.                bacteriophage T4. J. Mol. Biol. 47:293-306. <br />                                                                          20. Garfinkel, D. J., J. D. Boeke, G. R. Fink. 1985. Ty element <br />                                                                              transposition: reverse transcriptase virus-like particles. Cell <br />                       ACKNOWLEDGMENTS                                        42:507-517. <br />                                                                          21. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br />   We thank Lenore Neigeborn, John Celenza, Marian Carlson                mation intact yeast cells treated alkali cations. J. <br /> for communicating unpublished results. We thank Jan Fassler             Bacteriol. 153:163-168. <br /> and Joel Hirschhorn critical reading manuscript.              22. Johnston, M., R. W. Davis. 1984. Sequences regulate <br />   C.D.C. supported funds Lucille P. Markey                   divergent GALI-GALIO promoter        Saccharomyces cere- <br /> Charitable Trust. This work supported Public Health Service           ti&apos;siae. Mol. Cell. Biol. 4:1440-1448. <br /> grant GM32967 National Institutes Health F.W.             23. Klapholz, S., R. Esposito. 1982. A new mapping method <br />                                                                              employing meiotic rec- mutant yeast. Genetics 100: <br />                        LITERATURE CITED                                      387-412. <br />                                                                          24. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />  1. Birnboim, H. C., J. Doly. 1979. A rapid alkaline extraction          cloning, laboratory manual. Cold Spring Harbor Laboratory, <br />     procedure screening recombinant plasmid DNA. Nucleic                 Cold Spring Harbor, N.Y. <br />     Acids Res. 7:1513-1523.                                              25. Meeks-Wagner, D., L. H. Hartwell. 1986. Normal stoichi- <br />  2. Boeke, J. D., D. J. Garfinkel, C. A. Styles, G. R. Fink. 1985.       ometry histone dimer sets necessary high fidelity of <br />     Ty elements transpose RNA intermediate. Cell                  mitotic chromosome transmission. Cell 44:43-52. <br />     40:491-500.                                                          26. Meeks-Wagner, D., J. S. Wood, B. Garvik, L. H. Hartwell. <br />  686     CLARK-ADAMS AND WINSTON                                                                                            MOL. CELL. BIOL. <br />  <br />      1986. Isolation genes affect mitotic chromosome              239-249. <br />      transmission S. cerevisiae. Cell 44:53-63.                        39. Roeder, G. S., A. B. Rose, R. E. Pearlman. 1985. Transpos- <br /> 27. Mellor, J., M. H. Malim, K. Gull, M. F. Tuite, S. McCready, T.            able element sequences involved enhancement yeast <br />      Dibbayawan, S. M. Kingsman, A. J. Kingsman. 1985.                    gene expression. Proc. Natl. Acad. Sci. USA 82:5428-5432. <br />      Reverse transcriptase activity Ty RNA associated        40. Searles, L. L., R. A. Voelker. 1986. Molecular characteriza- <br />      virus-like particles yeast. Nature (London) 318:583-586.              tion vermilion locus suppressible alleles. Proc. <br /> 28. Modolell, J., W. Bender, M. Meselson. 1983. D. melanogas-             NatI. Acad. Sci. USA 83:404408. <br />      ter mutations suppressible suppressor hairy-wing       41. Sherman, F., G. R. Fink, C. W. Lawrence. 1978. Cold <br />      insertions 7.3kb mobile element. Proc. NatI. Acad. Sci.             Spring Harbor Laboratory manual course, Methods in <br />      USA 80:1678-1682.                                                        Yeast Genetics, revised edition. Cold Spring Harbor Labora- <br /> 29. Mortimer, R. K., D. C. Hawthorne. 1969. Yeast genetics, p.            tory, Cold Spring Harbor, N.Y. <br />      385-460. In A. H. Rose J. S. Harrison (ed.), The yeasts, vol.    42. Silverman, S. J., G. R. Fink. 1984. Effects Ty insertions <br />      1. Academic Press, Inc., New York.                                       HIS4 transcription Saccharomyces cerevisiae. Mol. Cell. <br /> 30. Neigeborn, L.,  <br /> </body></html>