<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Weber Hurst Genome Biology 2011, 12:R23 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                    Open Access <br />  <br /> Support multiple classes local expression <br /> clusters <span id='am-51' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-57'  ></span>Drosophila melanogaster</span>, no <br /> evidence gene <span id='am-66' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> conservation <br /> Claudia C Weber Laurence D Hurst* <br />  <br />  <br />   Abstract <br />   Background: Gene order eukaryotic genomes random, genes similar expression profiles <br />   tending cluster. In yeasts, model taxon gene order analysis, syntenic clusters non-homologous <br />   genes tend conserved evolutionary time. Whether similar clusters gene order conservation in <br />   lineages is, however, undecided. Here, examine issue Drosophila <span id='am-82' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-83' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86'  ></span>melanogaster</span> using high- <br />   resolution chromosome rearrangement <span id='am-88' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92'  ></span>data</span>. <br />   Results: We D. melanogaster classes expression clusters: first, observed in <br />   mammals, large clusters functionally unrelated housekeeping genes; second, small clusters functionally related <br />   highly co-expressed genes; finally, previously defined Spellman Rubin, larger domains co- <br />   expressed functionally unrelated genes. The are, however, independent small co-expression <br />   clusters likely reflect methodological artifact. While small co-expression housekeeping/essential gene <br />   clusters resemble observed yeast, contrast yeast, evidence cluster <br />   types preserved synteny blocks. If anything, adjacent co-expressed genes likely become <br />   rearranged expected. Again contrast yeast, D. melanogaster, gene pairs short intergene distance <br />   divergent orientations tend higher rearrangement rates. These findings consistent co- <br />   expression partly shared chromatin environment. <br />   Conclusions: We conclude that, similar terms cluster types, gene order evolution strikingly different <br />   patterns yeasts D. melanogaster, recombination associated gene order rearrangement in <br />   both. <br />  <br />  <br /> Background                                                                          stretches 10 15 genes enriched for <br /> In studied eukaryotic genomes gene order                                essential genes tend highly expressed [8]. <br /> known random, genes similar                                     We use term &#8216;cluster&#8217; refer neighbor- <br /> expression profiles tending cluster (see, example,                           hoods genes defined local expression similarities, <br /> [1-4]). The model organisms used work gene                                   term &#8216;co-expression&#8217; refer highly correlated <br /> order evolution yeasts, high-                             expression patterns multiple conditions a <br /> resolution gene order rearrangement data                                   time course. <br /> group <span id='am-63' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, excellent data numerous                               Do different types clusters similarly expressed <br /> additional parameters (for example, gene expression,                                genes behave evolutionarily conserved units, might <br /> and recombination rates) focal species, Sacchar-                            similar expression profiles merely result of <br /> omyces cerevisiae. In S. cerevisiae observe pairs                             transcriptional noise? In yeast, evidence for <br /> triplets adjacent genes functionally related                            possibility. In addition functional simi- <br /> and highly co-expressed [5-7]. Similarly,                              larities observed small co-expression clusters [6], both <br />                                                                                     essential gene clusters co-expression clusters a <br /> * Correspondence: l.d.hurst@bath.ac.uk                                              tendency preserved syntenic units evolu- <br /> Department Biology Biochemistry, University Bath, Claverton <br /> Down, Bath, BA2 7AY, UK <br />                                                                                     tionary time [8-11]. While genes close <br />  <br />                                        &#169; 2011 Weber Hurst; licensee BioMed Central Ltd. <span id='am-58' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> open access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly cited. <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                  Page 2 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> proximity likely rearranged,   functionally coordinated (assayed concordance of <br /> conservation synteny accounted           Gene Ontology (GO) GO Slim categories) seen in <br /> intergene distance (IGD) [9,11].                       species yeast, <span id='am-1' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span>Arabidopsis</span> <br />    Thus, based findings yeast, tempting       thaliana [20] and, lesser extent, humans [6]. These <br /> speculate eukaryotic genomes consist stretches       shall term type 1 clusters. <br /> of genes coordinated expression profiles         The second category clusters house-keeping [21] <br /> maintained natural selection. Can confident,        and/or highly expressed genes [22] human gen- <br /> however, lessons model species       ome likely equivalent (or closely related <br /> general applicability? If not, need consider     to) similarly sized clusters essential genes seen in <br /> genomes case-by-case basis. It is, example, far     yeast <span id='am-23' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> [8,23]. In of <br /> from obvious comparable clusters species       species clusters essential genes tend to <br /> also gene order conservation, reports        low recombination rates [8,23]. These larger clus- <br /> contradictory. Regions high density essential      ters little sign co-expression little or <br /> genes reportedly associated increased regional      sign functional similarity [8,24]. We term these <br /> linkage conservation mice [12]. Similarly, early          functionally uncoordinated clusters type 2 clusters. We <br /> reports claimed dearth breakpoints clusters       shall assume, defendable [24], housekeep- <br /> house-keeping genes [13] conservation small co-       ing clusters essential gene clusters. <br /> expressed clusters [14]. However, recent analysis       Currently, clear D. melanogaster has <br /> [15] suggests, anything, quite opposite        type 1 and/or type 2 clusters. That clusters of <br /> true, highly co-expressed pairs likely    genes expressed testes [25,26] immune genes <br /> be rearranged. Likewise, functionally coordinated gene       involved interactions pathogens [27] suggests <br /> neighborhoods present humans chimps          small type 1 clusters. <br /> enriched synteny breaks [16].                            We previously shown adjacent genes in <br />    How findings reconciled       D. melanogaster similar terms expression <br /> observed yeast? While conserved synteny seen       breadth expected chance [28]. This suggests that <br /> the highly co-expressed gene pairs yeast,      D. melanogaster type 2 housekeeping clusters. <br /> are pairs showing moderate levels co-             D. melanogaster is, however, unusual having third <br /> expression functional similarity. This likely   form cluster that, date, not, best our <br /> reflect noisy expression associated opening     knowledge, reported elsewhere. Spellman and <br /> closing chromatin [17,18]. Similar broad scale noisy      Rubin [29] identified clusters resembled type 1 clus- <br /> chromatin dynamics explain clus-         ters showing co-expression, resembled type 2 <br /> ters housekeeping genes. Given high rate rear-     clusters large having functional similar- <br /> rangement observed highly co-expressed genes         ity. This simply reflect inability define func- <br /> with short IGDs [15], suggested        tional co-ordination, want evidence shall <br /> noise driven co-expression disadvantageous.           consider clusters observed Spellman Rubin <br />    One problem prior analyses, outside yeasts,   large functionally uncoupled clusters we <br /> is dearth close comparator species, making genome-      SR (for Spellman Rubin) clusters. <br /> wide identification breakpoints difficult. With         Given uncertainty kinds cluster <br /> advent well-sequenced Drosophila genomes           D. melanogaster has, start testing different <br /> can, however, ask lessons           forms cluster. In addition previously identified SR <br /> model species, yeasts, hold true        clusters, provide evidence small clusters highly <br /> group. Here, then, consider evolution main-       co-expressed genes larger clusters housekeeping <br /> tenance clusters D. melanogaster, making use        genes. Given evidence small clusters, ask <br /> recent high-resolution data position gene          SR clusters biologically relevant units or <br /> order rearrangements inferred multiple sequenced        reflect methodological artifact. <br /> Drosophila species [19].                                     SR clusters defined considering genes a <br />    Unfortunately, relatively little known   ten-gene window asking mean level of <br /> the kinds gene clusters described species    co-expression thresh- <br /> also present D. melanogaster. The clusters described      old. A given large &#8216;cluster&#8217; could, however, actually con- <br /> in species appear fall main categories:    tain, example, different small clusters are <br /> small co-expression clusters large housekeeping/         uncorrelated other. While strength of <br /> essential gene clusters. Small clusters      co-expression clusters unre- <br /> genes highly co-expressed (assayed Pearson&#8217;s     markable, correlations unrelated <br /> product moment correlation expression values)         clusters force mean level threshold. Given the <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                    Page 3 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> above scenario, far clear large size    housekeeping genes D. melanogaster cluster (with <br /> the cluster need relevance bet-        exception chromosome 4). <br /> ter considering smaller clusters isolation.       We sought exclude possibility is <br /> If clusters grouped together,           accounted presence duplicates. Using all- <br /> appear co-expression clusters func-          against-all Blastp, duplicate genes detected <br /> tional significance individual cluster func-   using cutoff value e &lt; 10 -7 , Spellman and <br /> tionally coordinated. In principle, cluster      Rubin&#8217;s analysis, pair duplicates was <br /> high co-expression scores push ten-gene           excluded (provided blacklisted). The <br /> window threshold, genes win-         transitions real genome exceeded simula- <br /> dow irrelevant. To overcome problem,             tions (P &lt; 9.999 &#215; 10-5), chromosome 4 (real <br /> establish algorithm define type 1 clusters       median 32; simulated median 32; P = 0.4252). The same <br /> by growing small co-expression cluster              observed genes detected 14 adult <br /> extend local genes co-expressed       tissues included FlyAtlas [32] encoded 1 <br /> core co-expressed set. We consider overlap            despite duplicate removal (for chromosome 4, P = <br /> between co-expression clusters SR clusters.          0.2134, real hits 27, median simulated hits 29; P &lt; 9.999 &#215; <br />   Finally, ask cluster types          10-5 chromosomes). Thus, observe greater <br /> units evolution, sense domains      expected clustering putative housekeeping genes, <br /> preserved synteny, observed yeast [8-11]. Consid-        defined low-specificity genes genes expressed <br /> eration rates synteny preservation needs control      adult tissues. <br /> for background effects. In yeast, example, genes         We defined clusters stretches begin and <br /> with small IGD likely          end broad specificity genes (tau &#8804;0.25) within <br /> be rearranged [9,11]. Intergene distance poten-      fourth gene low specificity. <br /> tially important covariate. Likewise, domains               Cluster span number low-specificity genes per <br /> high recombination rates tend domains high-           cluster recorded. We filtered those <br /> rearrangement rates [30,31]. Any preserved synteny        clusters occurred chance running <br /> may reflect covariance local recombination            10,000 randomly shuffled genomes cluster- <br /> rates.                                                         ing algorithm. Chromosome 4, did show <br />                                                                greater expected local similarities expression <br /> Results                                                        breadth, excluded analysis. Only those <br /> Characterizing housekeeping clusters                           clusters span number low-specificity genes <br /> D. melanogaster clusters housekeeping genes             &lt;5% chance occurring randomly retained. <br /> We previously adjacent genes D. melanogaster     We defined 512 clusters median 5 genes and <br /> are similar terms tissue specificities   spanning, average, 7 genes. Of these, 85 clusters with <br /> randomly selected genes [28]. To determine        median gene number 9 median span 13 <br /> might low tissue specificity genes (that is, puta-   &lt;5% chance occurring by <br /> tive housekeeping genes) clustering, asked          chance. Maximum gene number 40, maximum <br /> broadly expressed genes tend sit         span 70. <br /> more expected chance. We encoded low-              We repeated above, defining housekeeping genes <br /> specificity genes (tau &#8804;0.25) 1 genes 0. For     genes expressed 14 adult tissues, and <br /> each set neighbors chromosome, switch          requiring housekeeping genes assigned cluster be <br /> was recorded transition states         expressed tissues, allowing jumps above. We <br /> [24]. For example, simple array genes     obtained fewer slightly larger clusters. We <br /> five housekeeping not, maximum clustering         detected 421 clusters median 7 genes span <br /> would arrangement 1111100000. This           11, 129 median 12 genes and <br /> has transition (between 1 0). By contrast,        median span 17 remaining filtering. The maxi- <br /> the structured organization 0110010101 seven          mum number housekeeping genes cluster <br /> state changes. The number transitions states        57 maximum span 86. Thus, previously <br /> in real genome lower 10,000           described organisms, D. melanogaster contains <br /> randomized sets gene order shuffled prior            type 2 clusters housekeeping genes. The clus- <br /> to recording number transitions (P &lt; 9.999 &#215;            ters defined tau refer tau clusters, those <br /> 10 -5 ), chromosome 4 (P = 0.4326; median           defined respect proportion tissues show- <br /> observed transitions 32, median expected transitions           ing evidence expression term breadth clusters, <br /> 32). Therefore, clustering low tissue specificity genes     housekeeping clusters. A compendium <br /> exceeds random expectation, indicating putative           identified clusters genes given in <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                     Page 4 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> Additional file 1 (breadth clusters) Additional file 2     rate estimate (ACE; based local relationship <br /> (tau clusters).                                                genetic physical map positions) [35] gives <br />                                                                opposite result, genes clusters defined by <br /> Housekeeping clusters greater                 tau expression breadth exhibiting elevated <br /> expected functional coordination                               recombination rates (for tau median cluster rate = <br /> Next asked housekeeping clusters                 1.2100, overall median rate = 1.1800, Wilcoxon test P = <br /> D. melanogaster functionally coordinated. Sixty-           0.0438; breadth clusters median = 3.021, overall med- <br /> three 88 housekeeping clusters (as defined low           ian = 2.934, Wilcoxon test P = 0.0099). <br /> tau) functional annotation available          We unambiguous evidence clus- <br /> significant enrichment GO Slim term.          ters putative housekeeping genes low recombi- <br /> This expected based 10,000 randomiza-          nation rates. This due, part, limited <br /> tions clusters populated randomly              resolution presently available recombination maps <br /> selected genes tested enrichment (probability       accords previous observation broadly <br /> observing enriched clusters = 0.0002).         expressed genes (that is, genes low tau) be <br /> However, owing presence tandem         conclusively shown elevated decreased <br /> duplicates, enriched genes involved           recombination rates given said maps [28]. <br /> developmental processes transcriptional regulation <br /> [33]. When test repeated duplicated             Characterizing co-expression clusters <br /> genes, 51 clusters significant enrichment        Small co-expression clusters functionally coordinated <br /> least term, significantly different      We characterized gene clusters defined high <br /> random expectation (P = 0.1201). Similar results           levels co-expression using ten-gene sliding <br /> obtained breadth clusters duplicates          frame approach cluster-growing algorithm that <br /> considered. The observed number enriched clusters           does impose fixed window size. A compendium of <br /> (68) greater expected (P = 0.5275). Thus,        identified co-expression clusters genes <br /> in yeast C. elegans, large clusters putative house-     given Additional file 3. The cluster-growing <br /> keeping essential genes filtered duplicates     algorithm finds co-expressed genes are <br /> not appear functional coordination [8].                immediate neighbors separated just other <br />                                                                (non-co-expressed) gene (Figure 1) clusters tend to <br /> The recombination rate housekeeping clusters         small terms span gene number (see <br /> unusual                                                        Figure S1 Additional file 4). As null expectation is <br /> In yeast worm, clusters &#8216;essential&#8217; genes (that is,     skewed detection immediate neighbors, the <br /> knockout inviables) conserved low recombi-        distribution distance co-expressed genes pro- <br /> nation rates [8,23]. Similarly, negative correla-   vides prima facie evidence clusters well <br /> tion crossover rate expression breadth          functionally important. As required <span id='am-87' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, the <br /> humans [34]. We asked genes               median co-expression type 1 clusters high <br /> housekeeping clusters tend lower recombination         (0.8644). <br /> rates rest genome. Since chromosome 4          The relevance SR clusters immediately evi- <br /> does recombine evidence clustering,         dent. As required definition, co-expression between <br /> was considered analysis. When consider          genes SR clusters is, average, higher between <br /> recombination estimated using regression polyno-        random genes (median SR clusters = 0.303; Wil- <br /> mial (RP) method [35], tau clusters observe      coxon test difference = -0.2275, P &lt; 2.2 &#215; 10-16). How- <br /> a reduced recombination rate (median cluster rate =            ever, possible pairwise expression correlations <br /> 2.804; overall median rate = 2.934; Wilcoxon test P =          genes SR cluster, striking 30.06% <br /> 0.0006). However, relationship observed          negative Pearson&#8217;s correlation coefficient. Natu- <br /> consider breadth clusters (P = 0.6421). These clusters         rally, lower expected gene pairs selected <br /> could include genes expressed low           random (10,000 chromosomal <span id='am-35' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span>), where <br /> level leaky transcription,       48.01% negative correlation (median 0.0295), <br /> presumed suitable measure identifying         nonetheless indicates large negative co- <br /> housekeeping clusters. Tau, hand, takes      expression clusters putatively positive co- <br /> account highly expressed gene given tissue       expression domains. <br /> relative maximum expression           Previous evidence various taxa supports view <br /> less susceptible low levels noisy expression. It      gene clusters functionally coordinated are <br /> tempting, therefore, disregard result. How-      similar terms expression patterns. For <br /> ever, adjusted coefficient exchange recombination       instance, multiple species, including D. melanogaster, <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                                    Page 5 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br />                                                  Distances nearest coexpressed <br />                                                           pairs type 1 clusters <br />  <br />  <br />  <br />  <br />                                          0.5 <br />                                          0.4 <br />                                           0.3 <br />                                       Density <br />                                    0.2   0.1 <br />                                          0.0 <br />  <br />  <br />  <br />  <br />                                                 0      1        2       3     4       5 <br />                                                 number intervening non&#8722;coexpressed genes <br />  Figure 1 Frequency distribution number intervening non-coexpressed genes nearest co-expressed pairs within <br />  type 1 co-expression clusters. Close half co-expressed genes clusters immediately adjacent nearest co-expressed <br />  neighbors. <br />  <br />  <br /> co-expression clustered genes belonging                   There might, however, trivial explanation this. <br /> same GO classification reported stronger                Duplicated D. melanogaster genes remained <br /> than co-expression functionally coordinated                    adjacent evolutionary time enriched devel- <br /> genes proximate [16]. Meanwhile, proxi-                   opmental transcriptional regulatory functions, as <br /> mate genes S. cerevisiae tend close pro-               co-expressed [33]. While taxa <br /> tein interaction network, proximity network                 minority duplicated genes co-expressed <br /> predicts co-expression [36]. However, function-                [38,39], rule possibility observations <br /> ally coordinated clusters observed D. melanogaster                  explained tandem duplication removed dupli- <br /> are simultaneously co-expression clusters [37].                        cates cluster. In set, the <br />   We asked clusters genes                    observed number GO-term-enriched clusters still <br /> correlated expression profiles tend enriched                 exceeds expected chance dynamic <br /> genes similar functions. Spellman Rubin [29]                  growth algorithm (454, P &lt; 0.0001) sliding <br /> previously reported number GO terms asso-                  window algorithm (observed 115, P = 0.4720). This sug- <br /> ciated co-expression clusters defined sliding                gests large SR clusters functionally <br /> window approach higher expected                       coordinated appearance contrary is <br /> accounting tandem duplication. However,                   owing tandem duplicates. Equally, evi- <br /> relevant question, perhaps, number                   dence presence duplicates, using cutoff <br /> clusters showing significant GO term enrichment                     defined Spellman Rubin [29], accounts func- <br /> greater expected (with tandem duplicate                tional coordination genes clusters defined our <br /> control). To determine types co-                    algorithm. <br /> expression cluster functionally coordinated,                      It ought noted tandem duplicates in <br /> asked number clusters significantly asso-               D. melanogaster remain adjacent Anopheles gam- <br /> ciated GO Slim term higher                  biae (and elevated co-expression func- <br /> expected. We observed counts exceed                 tional similarity) form small subset tandem <br /> expected number (dynamic growth method observed                        duplicates (13.55%) [33]. We expect all <br /> counts 542, P &lt; 0.0001; SR sliding-window method                       tandem duplicates highly co-expressed and <br /> observed counts 160, P &lt; 0.0001) based 10,000 boot-                 functionally related results need not <br /> straps. Therefore, types cluster significant              conflict findings Quijano et al. [33]. More- <br /> functional enrichment.                                                 over, functionally enriched type 1 clusters, <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                    Page 6 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> functional neighborhoods described multiple species,        overlapping small clusters increases SR cluster size <br /> that is, local enrichments GO terms defined           (Spearman&#8217;s rho = 0.4418, P = 1.348 &#215; 10-12, n = 234). <br /> sliding window algorithm, explained tan-       Altering cluster-growing algorithm allow co- <br /> dem duplication [16].                                          expression clusters gaps does affect the <br />    These results indicate failure evidence   finding SR type I clusters independent <br /> SR co-expression clusters functional coordination         (see Supplementary Information R1 Additional file 4). <br /> may partly way question           Given extent SR clusters overlapped by <br /> asked partly presence duplicates. It         type 1 clusters, likely presence few <br /> should noted Spellman Rubin [29] fil-         strongly co-expressed genes leads sliding frame <br /> tered duplicates redefined co-expression clus-        algorithm picking greater-than-expected correlations <br /> ters. This means clusters comparing          window, large cluster size is <br /> may longer locations. Their test does        artifact employed methodology. This <br /> not directly address question           <span id='am-18' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-19' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22'  ></span>idea</span> supported overall lower (and indeed <br /> duplicates responsible functional coordination         negative) co-expression observed SR clusters com- <br /> within co-expression clusters defined allowing        pared type 1 clusters. Based results, one <br /> for duplicates. Spellman Rubin appear         argue sliding frame clusters contain more <br /> completely deleted duplicate genes main-           &#8216;junk&#8217;, is, fewer co-expressed genes, hence <br /> taining pair.                                    appropriate choice investigating co- <br /> Non-independence large small co-expression clusters     expressed proximate genes functionally coordinated. <br /> As noted above, method define SR clusters            While doubt SR clusters defined, <br /> may inexact method detect small type         evidence suppose that, defined, are <br /> 1 clusters. Moreover, Spellman Rubin [29] suggest          biologically meaningful units. This accords the <br /> that presence large domains coordinated              observations S&#233;mon Duret [14], report that <br /> expression provides evidence chromatin status             mammalian co-expression clusters, defined a <br /> drives co-expression. However, assumes           contiguous group co-expressed genes, contain only <br /> size clusters defined algorithm         genes. <br /> merely result method employed. We              What claim size SR clusters pro- <br /> compare cluster sizes SR clusters chro-       vides prima facie support involvement chroma- <br /> matin-defined units ask large small co-        tin [29]? We observe median cluster size 13 the <br /> expression clusters independent.                           approach Spellman Rubin [29]. The chromatin <br />   What, then, relationship small            domains reported Wit et al. [37] contained med- <br /> large clusters, is, small type 1 clusters tend   ian 26 genes, is, twice many. However, median <br /> subsets larger SR sliding window counterparts?        cluster size 3 type 1 clusters (Figure S1 Addi- <br /> The observed proportion genes small clusters        tional file 4). Although clustering algorithm allows <br /> fall completely large SR clusters 0.3506. To         gaps genes clusters, 47.57% cluster <br /> determine proportion greater             members adjacent cluster member no <br /> expected, locations small clusters held fixed      intervening non-co-expressed genes (Figure 1). For the <br /> while large clusters randomly assigned location         duplicates-removed sets, median cluster sizes 11 and <br /> in genome. The maximum expected proportion              3, respectively. This questions assertion cluster <br /> small-cluster genes falling larger clusters           size is, itself, indicative large open chromatin <br /> 0.2428 based 10,000 randomizations. Therefore,          regions permitting gene expression. <br /> extent overlap small large clusters             However, types co-expression cluster we <br /> greater expected chance. In comparison, type 1         evidence chromatin level effects well <br /> co-expression clusters overlap housekeeping             relevant. If co-expression clusters correlated in <br /> clusters expected chance         terms expression patterns co-regulation <br /> the tau definition (observed proportion genes co-        opposed chromatin effects, expect see <br /> expression clusters fall entirely housekeep-     decrease co-expression scores increasing dis- <br /> ing cluster = 0.1068, P = 0.1292) breadth defini-       tance non-overlapping genes belonging the <br /> tion (observed proportion = 0.2444, P = 0.08).                 cluster. If co-expression owing local relaxa- <br />   As expected, SR clusters picking small        tion chromatin, expect weaker co- <br /> clusters, 79.06% SR clusters contain            expression greater distance. For SR type <br /> small clusters. The majority large clusters             1 clusters, observe negative correlation (assayed by <br /> (56.84%) contain just small cluster (median number         Spearman&#8217;s rho, referred rho) between <br /> of overlapping type 1 clusters = 1). The number             IGD co-expression (rho = -0.1176, P &lt; 2.2 &#215; 10-16, <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                       Page 7 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> n = 31,127 SR clusters; rho = -0.0596, P = 1.102 &#215;           1 clusters (32% divergent clusters) versus genes (29% <br /> 10-5, n = 5,438 type 1 clusters), distance        divergent clusters; chi squared 7.7374, df = 3, P = 0.0518) <br /> between midpoints co-expression (rho = -0.1287, P            sliding window SR clusters (28% divergent clusters) <br /> &lt; 2.2 &#215; 10-16 SR clusters; rho = -0.0733, P = 6.236 &#215;        versus genes (chi squared 5.6318, df = 3, P = 0.1310). <br /> 10-8 type 1 clusters). That co-expression            These numbers roughly accord reported <br /> genes clusters decays distance consistent        31% divergently oriented genes D. melanogaster gene <br /> chromatin effects facilitating co-expression genes       pairs intergenic distance &lt;600 bp [48]. Based the <br /> are close accords previous reports          small difference percentages marginally signifi- <br /> a relationship intergenic distance co-               test, evidence divergently oriented <br /> expression levels [20,40-43].                                    pairs common small co-expression clus- <br />   We examine possibility chromatin          ters. This consistent possibility co-expres- <br /> effects responsible, part, co-expression adja-     sion partly owing divergent transcription, been <br /> cent genes comparing clusters clusters          observed humans [49]. <br /> are likely co-regulation. Mezey et al. [1] <br /> provide list sliding-window-defined clusters             Gene order evolution <br /> genes physically adjacent D. melanogaster            Before assess gene clusters greater <br /> and mean expression levels correlated                  expected evolutionary conservation selec- <br /> between males seven species D. melanogaster            tion, identify potential covariates. Compared to <br /> subgroup, is, genes showing evolution coordi-            mammals [19], <span id='am-69' about='obo:NCBITaxon_genus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="rdfs:label" content="genus" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>genus</span> Drosophila shows high <br /> nated expression. We ask extent small          degree genome rearrangement [50]. Functional con- <br /> co-expression clusters share genes between-species          straint thought responsible 15% <br /> clusters defined Mezey et al. [1] using ten-gene            gene order conservation [19], reuse linkage <br /> window, removing genes withdrawn FlyBase                breakpoints influenced fragility certain types <br /> (FB 2010_03). A small percentage (7%) small                   sequence (see, example, [51,52]). We start by <br /> co-expression clusters share genes          examining role recombination IGD modu- <br /> between-species clusters, 5% genes small              lating rearrangement rates. <br /> co-expression clusters between-            Recombination associated gene order <br /> species cluster. This accords finding              rearrangement <br /> co-expression clusters defined Drosophila simu-           One force affecting gene order conservation recombi- <br /> lans correspond defined species          nation. Recombination associated higher rate <br /> [1] argues co-expression adjacent genes        chromosome rearrangement wheat [30] zebra <br /> a given species co-regulation. By contrast,         finch, suggesting possible role non-allelic homolo- <br /> around half (51%) tau clusters share         gous recombination (also termed ectopic recombination) <br /> gene between-species clusters, 27% genes            structural rearrangement [31]. Consistent this <br /> in tau clusters present between-species          idea, adjacent S. cerevisiae gene pairs specifying proteins <br /> cluster (but 9% between-species cluster genes            network proximity remain linked Kluyvero- <br /> also present tau cluster). This result is, again,     myces lactis recombination loci is <br /> consistent small co-expression housekeeping             unusually low [36]. <br /> clusters comprising different sets genes regu-         We ask recombination also <br /> lated differently, subset putative housekeeping        associated increase linkage breakage in <br /> genes possibly showing coordinated evolution              D. melanogaster. Von Grotthuss et al. [19] provide <br /> multiple Drosophila species.                                     information blocks genes conserved nine <br /> Divergent gene pairs unusually common small            Drosophila species (Figure 2). Linkage blocks were <br /> co-expression clusters                                           defined using gene order orientation synteny <br /> Transcriptional orientation thought influence co-          definition, requires gene order orienta- <br /> expression gene pairs. Conserved divergently paired           tion maintained species tree [19]. The <br /> housekeeping genes reported D. melanogaster         correlation linkage block size recombina- <br /> [44] proposed pairs               tion [53] significant (rho = -0.0268, P = 0.1364 <br /> increased co-expression [20,45] functional coor-      RP; rho = -0.0276, P = 0.1262 ACE; n = 3,080). <br /> dination, bidirectional pairs similar functions        However, non-recombining blocks (defined as <br /> showing strongly correlated expression [46,47]. We          having mean RP rate 0) excluded significant <br /> therefore ask divergently oriented gene pairs        negative relationship mean recombination rate <br /> overrepresented adjacent gene pairs co-expression          block size observed (rho = -0.0411, P = 0.0301 <br /> clusters. Chi squared borderline significant small type   RP; rho = -0.0419, P = 0.0268 ACE; n = 2,788). <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                         Page 8 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br />                                                                   genes pair reside conserved block <br />                                               D. willistoni <br />                                                                   species tree (Figure 2) using gene order <br />                                               D. melanogaster <br />                                                                   orientation synteny definition requires both <br />                               A11             D. erecta <br />                                       A10 <br />                                                                   gene order orientation maintained. If IGD <br />                        A15                    D. yakuba           affects degree conservation providing opportu- <br />          A16    A14                           D. ananassae        nities rearrangement, expect a <br />                                               D. pseudoobscura    greater proportion pairs small IGD con- <br />                                               D. mojavensis       served linkage. <br />                                       A12 <br />                                               D. virilis            Gene pairs separated 30 bins approxi- <br />                                A13 <br />                                               D. grimshawi        mately equal size according IGD proportion <br />  Figure 2 Reconstructed ancestral nodes A16 A13          matched blocks, is, instances linkage conserva- <br />  used define D. melanogaster gene pairs ancient linkage   tion species tree, determined each <br />  rearranged branch leading                 bin. However, positive slope obtained distance <br />  D. mojavensis, D. virilis D. grimshawi.                      versus proportion conserved monotone (Figure 3). <br />                                                                   Pairs separated divergent (n = <br />                                                                   1,656), convergent (n = 1,124) parallel orientations <br /> Hence, conclude conserved blocks regions            (n = 2,132), orientation expected to <br /> low recombination, defined D. melanoga-            influence degree linkage conservation. A negative <br /> ster, tend longer blocks regions high            monotone slope obtained convergently tran- <br /> recombination despite low resolution available             scribed pairs. The negative correlation (rho -0.5413, P = <br /> measures recombination. Additionally, recombination            0.0020) indicates short IGD predicts higher <br /> rates positively correlate average breakpoint            degree linkage conservation subset, consistent <br /> index (rho = 0.0427, P = 0.0179 RP; rho = 0.1017, P           idea longer IGDs provide opportunity <br /> = 1.543 &#215; 10-8 ACE; n = 3,080), is, number           rearrangement disrupting function. However, <br /> of breaks edge conserved block divided          contrary expectation, remaining orientations <br /> by two, indicating process recombination              appear non-monotone positive relationship <br /> might increase rate rearrangement.                         IGD linkage conservation (rho = 0.4316, P = <br />   A decrease block length observed regions            0.0173 orientations; rho = 0.6309, P = 0.0002 for <br /> high recombination regardless conserved blocks             parallel; rho = 0.6227, P = 0.0002 divergent; Figure 3). <br /> are defined. When consider rearrangements                 Note, however, Spearman&#8217;s rho suitable for <br /> that occurred inferred ancestor           detecting non-monotone relationships. <br /> nine species D. melanogaster, rearrange-              Co-expression predicts high rearrangement rates <br /> ments lineage leading Drosophila grim-               How account counterintuitive observa- <br /> shawi, <span id='am-38' about='obo:NCBITaxon_7230' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_198037 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_480118 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_32281 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32324 obo:NCBITaxon_33208 obo:NCBITaxon_32321 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Drosophila mohavensis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Drosophila mojavensis" datatype="xsd:string"></span>Drosophila mojavensis</span> <span id='am-44' about='obo:NCBITaxon_7244' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_32335 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_32281 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Drosophila virilis Sturtevant, 1916" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Drosophila virilis" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Drosophila irilis" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila virilis</span>           tion gene pairs shorter IGDs (for orienta- <br /> blocks conserved ancestor D. melano-              tions convergent) rearranged higher rate <br /> gaster, similar results obtained (Supplementary               despite having &#8216;opportunity&#8217; so? One possibility <br /> Information R2 Additional file 4). We conclude            pairs highly co-expressed acci- <br /> that gene order rearrangement associated ele-             dent open chromatin same <br /> vated recombination rates.                                        time, argued deleterious effect <br /> Short intergene distance predicts high rate gene order       [15]. This observe D. melanogaster. <br /> rearrangement                                                     Intergene distance negatively correlates co-expres- <br /> Given prior observations yeast [11], size         sion (rho = -0.0473, P = 0.0009, n = 4,912), for <br /> space pairs expected influ-           divergently transcribed pairs (rho = -0.1634, P = 2.237 &#215; <br /> ence degree conservation, longer intergene              10-11) convergent (rho = -0.0045, P = 0.8807) or <br /> space provide opportunity rearrangement              parallel pairs (rho = 0.0012, P = 0.9544). This presum- <br /> without disrupting function. Does IGD predict gene                ably owing shared <span id='am-72' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span>-regulatory elements located in <br /> order rearrangement neighbors?                            intergenic regions divergently transcribed genes. <br />   For pair adjacent genes available linkage          Median co-expression lower convergently tran- <br /> information, IGD calculated (including UTRs).                 scribed pairs (median = 0.1180) parallel (median <br /> Overlapping gene pairs pairs intergenic                 = 0.1683) divergent pairs (median = 0.2945). That <br /> region overlapped gene filtered               divergent pairs higher co-expression been <br /> out, rearrangements pairs                  observed mouse human [40], consistent <br /> involve disruption coding region UTRs; 4,912            idea bipromoters facilitate co-expression of <br /> pairs remained. We encoded               linked genes [49]. Additionally, reduced co-expression of <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                                                  Page 9 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br />                                                  GOO                                                               parallel GOO <br />                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                                           1.0 <br />  proportion conserved <br />  <br />  <br />  <br />  <br />                                                                                    proportion conserved <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                                                           0.8 <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                                                           0.6 <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                                                           0.4 <br />                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                                           0.2 <br />                         0.0 <br />  <br />  <br />  <br />  <br />                                                                                                           0.0 <br />                               1.0   1.5    2.0     2.5    3.0   3.5   4.0   4.5                                 2.0   2.5       3.0   3.5   4.0   4.5 <br />  <br />                                                  log10(dist)                                                                log10(dist) <br />  <br />  <br />  <br />                                            divergent GOO                                                              convergent GOO <br />                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                                           1.0 <br />  proportion conserved <br />  <br />  <br />  <br />  <br />                                                                                    proportion conserved <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                                                           0.8 <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                                                           0.6 <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                                                           0.4 <br />                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                                           0.2 <br />                         0.0 <br />  <br />  <br />  <br />  <br />                                                                                                           0.0 <br />  <br />  <br />  <br />  <br />                               1.5    2.0     2.5         3.0    3.5   4.0                                         1         2         3       4 <br />  <br />                                                  log10(dist)                                                                log10(dist) <br />  Figure 3 The proportion gene pairs conserved linkage, defined residing orthologous landmark, increases <br />  increasing IGD, convergently oriented gene pairs, greater IGD predicts lower rate conservation consistent <br />  opportunities rearrangement. GOO, gene order orientation synteny definition. <br />  <br />  <br />  <br />  <br /> convergently oriented genes reported zebra-                             As closely linked pairs appear maintained a <br /> fish [42]. Meanwhile, linkage conservation shows                              lower rate distant pairs, examined <br /> opposite pattern: convergent pairs 78.83% conserved                           effect high co-expression maintenance link- <br /> (238 1,124 rearranged), compared 70.59% (627                             age. Given IGD, 5% highly expressed <br /> 2,132 rearranged) parallel 54.89% (747 1,656                           pairs likely rearranged? We employed <br /> rearranged) divergent pairs. Divergent pairs signif-                      logistic regression model conserved linkage as <br /> icantly overrepresented rearranged set (chi sq =                           outcome IGD high co-expression (defined <br /> 193.8161, df = 2, P &lt; 2.2 &#215; 10-16). These results                         5%) predictors. The best fit- <br /> opposite seen yeast [17].                                           ting model includes predictors, residual deviance <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                  Page 10 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> is 6202.7 model predictors, 6211.1        expressed - gene pairs D. melanogaster likely <br /> co-expression 6208.7 IGD alone,          undergone gene order rearrangement parti- <br /> null deviance 6217.4. Neither IGD (chi-square P =          cular lineage. We considering pairs <br /> 0.0038) co-expression (chi-square P = 0.0149)         adjacent D. melanogaster newly linked, <br /> be removed improve fit. IGD predicts increase       is, adjacent ancestral state, conserved <br /> in proportion matches (estimate 1.264 &#215; 10-5, P =      ancestral state D. melanogaster bro- <br /> 0.00742) co-expression decrease (estimate -3.296        ken lineage leading D. grimshawi, <br /> &#215; 10-1, P = 0.01378). Hence, evidence         D. mojavensis D. virilis does affect conclu- <br /> greater IGD provides opportunity rearrange-          sion (see Supplementary Information R3 Additional <br /> ment, highly co-expressed pairs maintained       file 4; Figure 2). <br /> by selection; opposite trend.               Co-expression directly associated recombination <br />    The overall lower co-expression levels lack cor-    It tempting speculate selection (poten- <br /> relation IGD co-expression explain          tially leaky) co-expression responsible the <br /> why convergently transcribed pairs unique show-        increased rate rearrangement. However, does <br /> ing decrease gene order conservation increas-       necessarily follow closely linked highly <br /> ing IGD, consistent idea longer IGDs            co-expressed pairs rearranged dis- <br /> provide opportunities rearrangement. Indeed,         advantageous. Given establish recombi- <br /> when non-convergently oriented genes consid-         nation associated gene order rearrangement and <br /> ered logistic model, high co-expression longer      co-expressed pairs tend highly rearranged, <br /> predicts conservation independent IGD (estimate =          need rule possibility high recombina- <br /> -1.468 &#215; 10 -1, P = 0.3208 co-expression; estimate =      tion co-expressed genes responsible the <br /> 2.239 &#215; 10 -5 , P = 0.0002 IGD; n = 3,788). On        observed synteny breaks. Indeed, recombination rates <br /> other hand, convergent pairs considered         known elevated highly co-expressed linked <br /> in logistic model, co-expression remains         genes humans [15]. <br /> significant predictor linkage conservation (estimate =       We evidence association co- <br /> -1.048, P = 0.0010 co-expression; estimate = -1.091       expression mean recombination rates adjacent <br /> &#215; 10-5, P = 0.1775 IGD; n = 1,124).                       gene pairs D. melanogaster (rho = -0.0113, P = <br />    Thus, appears genes short IGD tend      0.4317 ACE; rho = 0.0058, P = 0.6838 RP; n = <br /> more highly co-expressed, depending orientation,           4,879). Although failure detect association <br /> and likely rearranged, highly        partly lack high-resolution <br /> co-expressed pairs showing increased rates rearrange-      recombination data, conclude presently <br /> ment, exception convergently oriented pairs       evidence increased recombination responsible <br /> where high co-expression predicts reduced linkage       reduced maintenance co-expression clusters in <br /> conservation.                                                 D. melanogaster disruption adaptive gene order [36]. <br />    The results contrast strikingly prior results <br /> in yeast [11] prior observations      SR clusters clusters broadly expressed genes are <br /> mammals [13,14] plants, short IGD              likely conserved expected <br /> major predictor linkage conservation [54]. However,      In light absence evidence association <br /> more recent study reported, observe, reduced          housekeeping clusters low recombination <br /> rate linkage conservation linked co-expressed gene      rates, expect unusual patterns of <br /> pairs attributed potentially disadvantageous      gene order conservation housekeeping clusters. We <br /> effects leaky expression [15]. The discrepancy             examine clusters likely <br /> between result previous reports         expected increase gene order conser- <br /> be explained lack outgroup earlier        vation expected. We compared cluster to <br /> studies. Without polarizing gene order rearrangements         1,000 randomly selected stretches number <br /> it impossible distinguish formation      genes did overlap cluster counting <br /> new destruction old linkages [15]. Indeed,         number orthologous landmarks present each <br /> when Singer et al.&#8217;s [13] data reanalyzed using        cluster proxy rearrangements. We observed no <br /> outgroup, data longer support notion          increase gene order conservation clusters <br /> co-expressed clusters maintained selection             defined tau (P = 0.2337) number tissues expres- <br /> because advantageous [15].                           sing given gene (P = 0.2503). This case <br />    To address consideration polarity        counted rearrangements A13 (that is, <br /> changes alter results, asked clo-        node leading D. grimshawi, D. mojavensis and <br /> sely linked - presumably highly co-             D. virilis; Figure 2) breaks gene order (P = 0.3842 <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                     Page 11 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> for tau clusters; P = 0.4535 breadth clusters). We           genes regulated specific chromatin regulators <br /> therefore evidence selection acts preserve       conserved insect species [55]. Although Bhut- <br /> clusters putative housekeeping genes.                         kar et al. [56] report 60% genes belonging <br />   Given non-independence SR type 1               synteny block (calculated Droso- <br /> clusters, weak median co-expression levels SR             phila species) positively correlated embryonic <br /> clusters compared type 1 clusters extent           expression, analysis did account relation- <br /> which negative co-expression scores genes             ship physical proximity co-expression docu- <br /> SR clusters detected, concluded SR             mented above. On hand, clusters genes that <br /> clusters likely artifacts underlying            physically adjacent D. melanogaster whose <br /> method true co-expression clusters. Not              expression patterns correlate highly males of <br /> only substantial proportion genes assigned           seven Drosophila species, be <br /> these clusters presumably irrelevant putative clus-       expected co-regulated [1], conserved <br /> ter, given lack conservation highly           expected. Indeed, broken half <br /> co-expressed adjacent genes described above,            cases [19]. Additionally, observe general trend <br /> not predict SR clusters reduced rates linkage         divergently paired closely linked genes to <br /> breakage true co-expression clusters.          rearranged. <br /> This observe repeat                  Our findings consistent notion that <br /> above test. SR clusters rearranged ran-        co-expression linked genes disadvantageous <br /> domly selected stretches genes equivalent size (P =        likely transcriptional interference [15]. The <br /> 0.52 orthologous landmark numbers; P = 0.4749            observation similarity expression patterns within <br /> breaks A13 order). We evidence           type 1 SR clusters tends decay increasing <br /> D. melanogaster genes clusters defined simi-          distance genes consistent with <br /> larity expression patterns conserved units          co-expression being, part, consequence shared <br /> evolution.                                                       chromatin opposed tight co-regulation. That <br />                                                                  housekeeping clusters shown either <br /> Discussion                                                       increased gene order conservation reduced recombi- <br /> In terms types clusters observed, D. melanogaster      nation rates point &#8216;leaky expression&#8217; <br /> appears resemble model organism gene cluster          model. It is, however, likely limited resolution <br /> analysis, S. cerevisiae. We that, like yeast, D. melano-    currently available recombination maps affected our <br /> gaster contains small co-expression clusters           ability detect reduced levels recombination [28]. <br /> evidence functional similarity. D. melanogaster        How unexpected finding pairs neigh- <br /> type 2 (housekeeping/essential) clusters,     boring genes co-expression clusters rearranged <br /> clear housekeeping clusters D. melanogaster         slightly adjacent pairs in <br /> have reduced recombination rates. The previously                 clusters reconciled finding genes in <br /> described larger SR co-expression clusters,              co-expression clusters functional similarities not <br /> described yeast, likely artifact imposing fixed   explained tandem duplication? Recent evidence from <br /> window size presence type 1 clusters         D. melanogaster indicates gene order clusters <br /> fall entirely SR clusters.                                testis-expressed genes nuclear proximity need not <br />   This largely similarities end. Despite        maintained ensure co-expression, it <br /> fact domains high recombination tend          facilitate efficient co-expression [57]. It has <br /> highly rearranged species [36], strik-    recently suggested increased rate of <br /> ing differences yeast D. melanogaster             genome rearrangement does necessarily indicate <br /> terms gene order conservation. Yeast shows reduction        selection break apart gene pairs. Rather being <br /> in gene order rearrangements genes house-             conserved terms gene order, gene neighbor- <br /> keeping co-expression clusters [10,11]. Meanwhile,        hoods homologous functions shared between <br /> find evidence type cluster maintained       humans chimpanzees enriched synteny <br /> at higher rate expected D. melanogaster. On        breaks [16]. While neighborhoods are <br /> contrary, observe highly co-expressed neighboring        conserved species, genes them <br /> genes slightly likely conserved        necessarily orthologous. This appear to <br /> adjacent genes. This contrasts previous reports          suggest selection acts preserve functional neigh- <br /> clusters defined chromatin status D. melanogaster,         borhoods ancestral gene composition and <br /> such described Wit et al. [37],         order, gene order break given neighborhood <br /> enriched co-expression exhibit dearth synteny       followed new rearrangements restore the <br /> breaks, functionally coordinated clusters          functional unit [16]. This model reconciled with <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                    Page 12 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> our observation functionally coordinated co-expres-        order, terms presence co-expression and <br /> sion clusters increased levels gene order rear-         housekeeping/essential clusters similar dimen- <br /> rangement pairs short IGD                sions patterns functional similarity. Nevertheless, <br /> likely originated recently (Supplementary Infor-        striking differences patterns gene order conserva- <br /> mation R3 Additional file 4). Possible support       tion observe strongly caution assump- <br /> notion co-expressed pairs common functions            tion applies well-characterized model <br /> are generally prone gene order disruption comes              organism generalized species taxa. <br /> from S. cerevisiae. It suggested proximity facili-      While yeast clusters conserved syntenic units <br /> tates co-regulation bi-directional promoters com-         expected chance, D. melano- <br /> mon transcription factors, favors increase       gaster opposite seen. The picture D. melanogaster <br /> the recombination rate, eventually leading gene order        consistent co-expression due, part, to <br /> rearrangement [36].                                             shared chromatin environment potentially <br />    The question remains co-expression             disadvantageous. <br /> independent predictor logistic regression models <br /> for recently formed pairs pairs broken A13 (Sup-         Materials methods <br /> plementary Information R3 Additional file 4), unlike         SR co-expression clusters defined fixed-size ten-gene <br /> when breaks species tree consid-           sliding window approach <br /> ered. This lack depth, is,         We defined gene clusters using sliding window <br /> insufficient number synteny breaks closely           approach Spellman Rubin [29]. Using Pear- <br /> related species certain classes rearrange-         son&#8217;s correlation coefficient, mean co-expression all <br /> ment considered. Moreover, lack data species        possible pairwise combinations computed each <br /> other D. melanogaster limits ability assess         window adjacent genes. For chromosomal <br /> whether co-expression levels changed rearran-        class (4, X, 2L, 2R, 3R, 3L), mean Pearson&#8217;s correla- <br /> gements branch leading A13 (Supplemen-             tion coefficient possible pairwise combinations <br /> tary Information R3 Additional file 4),       ten-gene window calculated. A threshold <br /> ability look co-expression clusters func-      value chromosomal class defined calcu- <br /> tional homology D. melanogaster.            lating means 10,000 sets 10 randomly <br /> Expression pattern divergence sex biased drosophilid        selected genes above, setting 97.5th percentile <br /> genes known increase divergence time,           cutoff. Where correlation window <br /> different genes contributing effect differing        exceeded threshold value, genes contained <br /> degrees [58]. We expect expres-        window defined belonging clus- <br /> sion pattern given gene, degree               ter. Adjacent overlapping clusters merged, as <br /> to correlates expression patterns neigh-       Spellman Rubin [29]. <br /> boring genes, changed species             Dynamic co-expression clustering algorithm <br /> D. melanogaster, adding noise gene order            To overcome limitations posed above <br /> rearrangement analysis.                                         approach, developed algorithm does rely <br />    Additional limitations arise limited            fixed window size assign genes co-expression <br /> availability recombination rates species      clusters, allowing gaps genes <br /> D. melanogaster. The extent recombination rate               clusters. <br /> divergence species depends scale, fine-           Pearson&#8217;s correlation coefficient used determine <br /> scale rates tending diverge rapidly broad-         co-expression genes Spellman Rubin <br /> scale rates (see, example, [59-61]). Although          [29] dataset. In order determine cutoff, 100,000 ran- <br /> could introduce noise analysis possi-        dom gene pairs chromosomal class (4, X, 2L, <br /> ble effects recombination gene order rearrange-           2R, 3R, 3L) correlated. For genes more <br /> ment, resolution rates available                 strongly correlated 5% randomized values <br /> D. melanogaster, particularly RP, limited terms        (note use stricter threshold SR clusters), <br /> reflecting fine-scale variation. Additionally,         co-expression considered higher expected by <br /> evidence fine-scale recombination rates            chance. The beginning cluster defined pair <br /> the drosophilid sister species D. pseudoobscura             genes meets exceeds threshold no <br /> D. persimilis highly correlated [62].                       farther genes apart. The cluster expanded <br />                                                                 genes farther away permitted gap <br /> Conclusions                                                     size average correlation all <br /> The genome D. melanogaster bears resemblance            existing cluster members (not including genes not <br /> to S. cerevisiae, model species studying gene   co-expressed others) met exceeded the <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                                Page 13 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> threshold, showed above-threshold correlation           analysis. The influence large differences between <br /> with gene added cluster. Hence,            maximal gene expression tissue expression is <br /> opposed Spellman Rubin&#8217;s [29] algorithm,           reduced logging ratio tissue expression to <br /> genes added cluster required corre-      maximal gene expression. <br /> late genes cluster members. Those       Recombination rates <br /> genes meet criteria considered      Per-gene estimates recombination derived from <br /> part cluster, physically   release 4.3 D. melanogaster genome were <br /> limits cluster.                                          obtained Larracuente et al. [53]. Here use <br /> GO Slim enrichment analysis                                   recombination rates calculated different meth- <br /> As associations likely detected            ods: ACE based relationship genetic <br /> broader categories terms associated small         physical map positions polytene bands, that <br /> set genes, determine co-expression clus-        is, local scale; RP calculated slope of <br /> ters functionally related, GO terms mapped        order regression polynomial midpoint of <br /> GO Slim using map2slim.pl (part go-perl pack-          gene [35,67]. <br /> age). To detect clusters significantly enriched GO        Statistics <br /> Slim terms, cumulative hypergeometric distribution        All statistical analyses performed R. <br /> was employed: <br />                                                               Additional material <br />                  A      G&#8722;A <br />               <br />             k&#8722;1 <br />                       n&#8722;i <br />     p=1&#8722;                                             (1)       Additional file 1: FlyBase identifiers breadth clusters. <br />             i=0 <br />                        G                                       Additional file 2: FlyBase identifiers tau clusters. <br />                        n                                       Additional file 3: FlyBase identifiers small co-expression clusters. <br />                                                                Additional file 4: Supplementary results Figures. <br />   P represents probability finding k or <br /> more genes associated particular GO Slim term <br /> in cluster size n (not including uncorrelated genes), <br /> out total A genes associated term        Abbreviations <br />                                                               ACE: adjusted coefficient exchange recombination rate estimate; GO: <br /> the chromosomal arm, total G genes             Gene Ontology; IGD: intergene distance; RP: regression polynomial <br /> present chromosomal arm [63]. To determine             recombination rate estimate; SR: Spellman Rubin; UTR: untranslated <br /> whether number clusters significantly enriched         region. <br />  <br /> for term exceeded random expectation, all <br />                                                               Acknowledgements <br /> genes clusters replaced randomly selected        CCW funded University Bath. LDH Royal Society Wolfson <br /> genes chromosomal class (10,000                 Research Merit Award holder. We thank Amanda Larracuente providing <br /> replicates).                                                  recombination rate estimates. <br /> Removal duplicated genes                                   Authors&#8217; contributions <br /> Duplicates gene clusters defined compar-       CCW compiled, processed analyzed data. LDH conceived study. <br /> ing protein sequences using Blastp e-value         CCW LDH wrote manuscript. All authors read approved the <br />                                                               final manuscript. <br /> &lt;10-7 (as Spellman Rubin&#8217;s analysis) filtering low <br /> complexity regions [64]. For duplicate pair,         Competing interests <br /> gene randomly deleted. Clusters             The authors declare competing interests. <br /> remaining gene removal duplicates deleted.      Received: 14 January 2011 Revised: 4 March 2011 <br /> Expression breadth                                            Accepted: 17 March 2011 Published: 17 March 2011 <br /> Tau, measure tissue specificity takes account of <br /> how tissues given gene expressed in,        References <br />                                                               1. Mezey JG, Nuzhdin SV, Ye FF, Jones CD: Coordinated evolution co- <br /> as expression level relative maximal gene expres-          expressed gene clusters Drosophila transcriptome. BMC Evol Biol <br /> sion [65,66], calculated using adult tissue expression        2008, 8:2. <br /> data Fly Atlas [32]:                                     2. Michalak P: Coexpression, coregulation, cofunctionality of <br />                                                                   neighboring genes eukaryotic genomes. Genomics 2008, 91:243-248. <br />                              <br />          n         logS(j)                                    3. Oliver B, Misteli T: A non-random walk genome. Genome Biol <br />           j=1  1 &#8722;                                                2005, 6:214. <br />                    logSmax                              (2)   4. Hurst LD, Pal C, Lercher MJ: The evolutionary dynamics eukaryotic <br />    &#964;= <br />                 n&#8722;1                                               gene order. Nat Rev Genet 2004, 5:299-310. <br />                                                               5. Cohen BA, <span id='am-77' about='obo:NCBITaxon_69650' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57612 obo:NCBITaxon_69649 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="rdfs:label" content="Mitra" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Mitra</span> RD, Hughes JD, Church GM: A computational analysis of <br />  logS(j) given tissue set 0 expression         whole-genome expression data reveals chromosomal domains gene <br />                                                                   expression. Nat Genet 2000, 26:183-186. <br /> was detected array tissue. Genes         6. Fukuoka Y, Inaoka H, Kohane IS: Inter-species differences co-expression <br /> where tissues set 0 removed             neighboring genes eukaryotic genomes. BMC Genomics 2004, 5:4. <br />  Weber Hurst Genome Biology 2011, 12:R23                                                                                                           Page 14 15 <br /> http://genomebiology.com/2011/12/3/R23 <br />  <br />  <br />  <br />  <br /> 7.    Cho RJ, Campbell MJ, Winzeler EA, Steinmetz L, Conway A, Wodicka L,            31. Volker M, Backstrom N, Skinner BM, Langley EJ, Bunzey SK, Ellegren H, <br />       Wolfsberg TG, Gabrielian AE, Landsman D, Lockhart DJ, Davis RW: A                  Griffin DK: Copy number variation, chromosome rearrangement, and <br />       genome-wide transcriptional analysis mitotic cell cycle. Mol Cell           association recombination avian evolution. Genome Res <br />       1998, 2:65-73.                                                                     2010, 20:503-511. <br /> 8.    Pal C, Hurst LD: Evidence co-evolution gene order                   32. Chintapalli VR, Wang J, Dow JA: Using FlyAtlas identify better <br />       recombination rate. Nat Genet 2003, 33:392-395.                                    Drosophila melanogaster models human disease. Nat Genet 2007, <br /> 9.    Hurst LD, Williams EJ, Pal C: Natural selection promotes conservation          39:715-720. <br />       linkage co-expressed genes. Trends Genet 2002, 18:604-606.               33. Quijano C, Tomancak P, Lopez-Marti J, Suyama M, Bork P, Milan M, <br /> 10.   Fischer G, Rocha EP, Brunet F, Vergassola M, Dujon B: Highly variable rates        Torrents D, Manzanares M: Selective maintenance Drosophila tandemly <br />       genome rearrangements hemiascomycetous yeast lineages.                  arranged duplicated genes evolution. Genome Biol 2008, 9:R176. <br />       PLoS Genet 2006, 2:e32.                                                        34. Necsulea A, Semon M, Duret L, Hurst LD: Monoallelic expression and <br /> 11.   Poyatos JF, Hurst LD: The determinants gene order conservation               tissue specificity associated high crossover rates. Trends Genet <br />       yeasts. Genome Biol 2007, 8:R233.                                                  2009, 25:519-522. <br /> 12.   Hentges KE, Pollock DD, Liu B, Justice MJ: Regional variation density   35. Hey J, Kliman RM: Interactions natural selection, <br />       essential genes mice. PLoS Genet 2007, 3:e72.                                recombination gene density genes Drosophila. Genetics <br /> 13.   Singer GAC, Lloyd AT, Huminiecki LB, Wolfe KH: Clusters co-expressed            2002, 160:595-608. <br />       genes mammalian genomes conserved natural selection. Mol             36. Poyatos JF, Hurst LD: Is optimal gene order impossible? Trends Genet 2006, <br />       Biol Evol 2005, 22:767-775.                                                        22:420-423. <br /> 14.   Semon M, Duret L: Evolutionary origin maintenance coexpressed           37. Wit E, Braunschweig U, Greil F, Bussemaker HJ, van Steensel B: Global <br />       gene clusters mammals. Mol Biol Evol 2006, 23:1715-1723.                        chromatin domain organization Drosophila genome. PLoS Genet <br /> 15.   Liao BY, Zhang J: Coexpression linked genes mammalian genomes                2008, 4:e1000045. <br />       generally disadvantageous. Mol Biol Evol 2008, 25:1555-1565.                38. Ren XY, Fiers M, Stiekema WJ, Nap JP: Local coexpression domains two <br /> 16.   Al-Shahrour F, Minguez P, Marques-Bonet T, Gazave E, Navarro A, Dopazo J:          genes genome Arabidopsis. Plant Physiol 2005, <br />       Selection genome architecture: conservation functional                     138:923-934. <br />       neighborhoods changing genes. PLoS Comput Biol 2010, 6:e1000953.          39. Ren XY, Stiekema WJ, Nap JP: Local coexpression domains genome <br /> 17.   Batada NN, Urrutia AO, Hurst LD: Chromatin remodelling major                  rice microsynteny Arabidopsis domains. Plant Mol Biol <br />       source coexpression linked genes yeast. Trends Genet 2007,                2007, 65:205-217. <br />       23:480-484.                                                                    40. Woo YH, Walker M, Churchill GA: Coordinated expression domains in <br /> 18.   Raj A, Peskin CS, Tranchina D, Vargas DY, Tyagi S: Stochastic mRNA                 mammalian genomes. PLoS One 2010, 5:e12158. <br />       synthesis mammalian cells. PLoS Biol 2006, 4:e309.                          41. Thygesen H, Zwinderman A: Modelling correlation the <br /> 19.   von Grotthuss M, Ashburner M, Ranz JM: Fragile regions                     activities adjacent genes Drosophila. BMC Bioinformatics 2005, 6:10. <br />       functional constraints predominate shaping gene organization         42. Ng YK, Wu W, Zhang L: Positive correlation gene coexpression <br />       genus Drosophila. Genome Res 2010, 20:1084-1096.                                   positional clustering zebrafish genome. BMC Genomics 2009, <br /> 20.   Williams EJB, Bowles DJ: Coexpression neighboring genes                  10:42. <br />       genome <span id='am-8' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-9' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span>Arabidopsis thaliana</span>. Genome Res 2004, 14:1060-1067.                 43. Kruglyak S, Tang H: Regulation adjacent yeast genes. Trends Genet <br /> 21.   Lercher MJ, Urrutia AO, Hurst LD: Clustering housekeeping genes                 2000, 16:109-111. <br />       provides unified model gene order human genome. Nat                44. Yang L, Yu J: A comparative analysis divergently-paired genes (DPGs) <br />       Genet 2002, 31:180-183.                                                            Drosophila vertebrate genomes. BMC Evol Biol 2009, 9:55. <br /> 22.   Caron H, van Schaik B, van der Mee M, Baas F, Riggins G, van Sluis P,          45. Chen WH, Meaux J, Lercher MJ: Co-expression neighbouring genes <br />       Hermus MC, van Asperen R, Boon K, Voute PA, Heisterkamp S, van                     Arabidopsis: separating chromatin effects direct interactions. <br />       Kampen A, Versteeg R: The human transcriptome map: clustering                   BMC Genomics 2010, 11:178. <br />       highly expressed genes chromosomal domains. Science 2001,                   46. Wang Q, Wan L, Li D, Zhu L, Qian M, Deng M: Searching bidirectional <br />       291:1289-1292.                                                                     promoters Arabidopsis thaliana. BMC Bioinformatics 2009, 10(Suppl 1): <br /> 23.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kanapin A, Le           S29. <br />       Bot N, Moreno S, Sohrmann M, Welchman DP, Zipperlen P, Ahringer J:             47. Li YY, Yu H, Guo ZM, Guo TQ, Tu K, Li YX: Systematic analysis head-to- <br />       Systematic functional analysis <span id='am-30' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-31' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans genome                head gene organization: evolutionary conservation potential <br />       using RNAi. Nature 2003, 421:231-237.                                              biological relevance. PLoS Comput Biol 2006, 2:e74. <br /> 24.   Batada NN, Hurst LD: Evolution chromosome organization driven            48. Franck E, Hulsen T, Huynen MA, Jong WW, Lubsen NH, Madsen O: <br />       selection reduced gene expression noise. Nat Genet 2007, 39:945-949.           Evolution closely linked gene pairs vertebrate genomes. Mol Biol <br /> 25.   Boutanaev AM, Kalmykova AI, Shevelyou YY, Nurminsky DI: Large clusters             Evol 2008, 25:1909-1921. <br />       co-expressed genes Drosophila genome. Nature 2002,                   49. Trinklein ND, Aldred SF, Hartman SJ, Schroeder DI, Otillar RP, Myers RM: An <br />       420:666-669.                                                                       abundance bidirectional promoters human genome. Genome <br /> 26.   Dorus S, Busby SA, Gerike U, Shabanowitz J, Hunt DF, Karr TL: Genomic          Res 2004, 14:62-66. <br />       functional evolution Drosophila melanogaster sperm proteome.            50. Ranz JM, Casals F, Ruiz A: How malleable eukaryotic genome? <br />       Nat Genet 2006, 38:1440-1445.                                                      Extreme rate chromosomal rearrangement genus Drosophila. <br /> 27.   Wegner : Clustering Drosophila melanogaster immune genes                     Genome Res 2001, 11:230-239. <br />       interplay recombination rate. PLoS ONE 2008, 3:e2835.                     51. Strissel PL, Strick R, Rowley JD, Zeleznik-Le NJ: An vivo topoisomerase II <br /> 28.   Weber CC, Hurst LD: Intronic AT Skew defendable proxy germline        <br /> </body></html>