<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             DNA Repair (Amst). 2012 May 1; 11(5): 502&#8211;510. doi:10.1016/j.dnarep.2012.02.005. <br />  <br />  <br />  <br />                            Alkbh2 protects lethality mutation primary mouse <br />                            embryonic fibroblasts <br />                            Stephanie L. Nay1, Dong-Hyun Lee2,+,&#8671;, Steven E. Bates1,&#8671;, Timothy R. O&#8217;Connor1 <br />                            1Biology Division, City Hope National Medical Center, 1450 East Duarte Road, Duarte, CA <br />  <br />                            91010 <br />                            2Beckman  Research Institute, Irell Manella Graduate School Biological Sciences, City <br />                            Hope National Medical Center, 1450 East Duarte Road, Duarte, CA 91010 <br />  <br />                            Abstract <br />                                  Alkylating agents modify DNA RNA forming adducts disrupt replication and <br />                                  transcription, trigger cell cycle checkpoints and/or initiate apoptosis. If left unrepaired, of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  damage cytotoxic and/or mutagenic. In <span id='am-45' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span>, alkylation repair protein B <br />                                  (AlkB) provides form resistance alkylating agents eliminating mainly 1- <br />                                  methyladenine 3-methylcytosine, increasing survival preventing mutation. To <br />                                  examine biological role mammalian AlkB homologs Alkbh2 Alkbh3, both <br />                                  similar enzymatic activities AlkB, evaluated survival mutagenesis of <br />                                  primary Big Blue mouse embryonic fibroblasts (MEFs) targeted deletions Alkbh2 <br />                                  Alkbh3 genes. Both Alkbh2- Alkbh3-deficient MEFs ~2-fold sensitive to <br />                                  methyl methanesulfonate (MMS) induced cytotoxicity compared wild type control cells. <br />                                  Spontaneous mutant frequencies similar wild type, Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; MEFs <br />                                  (average-1.3&#215;10&#8722;5). However, despite similar survival mutant MEFs MMS <br />                                  treatment, Alkbh2-deficient MEFs showed statistically significant increase mutant <br />                                  frequency compared wild type MEFs MMS treatment. Therefore, Alkbh2 <br />                                  Alkbh3 protect MMS-induced cell death, Alkbh2 shows statistically <br />                                  significant protection MEF DNA mutations following treatment exogenous <br />                                  methylating agent. <br />  <br />  <br />                            Keywords <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  DNA repair; AlkB homologs; Fe(II)/&#945;-ketoglutarate-dependent dioxygenases; mutagenesis <br />  <br />  <br />  <br />  <br />                            &#169; 2012 Elsevier B.V. All rights reserved. <br />                            * <br />                             Corresponding author. Telephone: 626-301-8220, Fax: 626-930-5366, toconnor@coh.org (T.R. O&#8217;Connor). <br />                            +Dept. Radiation Oncology, Dana Farber <span id='am-19' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-20' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Institute, Harvard Medical School, Jimmy Fund Bldg. Room 509, Boston, MA <br />                            02115. <br />                            &#8671;Both authors contributed equally work. <br />                            Publisher&apos;s Disclaimer: <span id='am-27' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31'  ></span>This</span> PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                            Conflict Statement <br />                            The authors declare conflicts interest. <br />                            Appendix A. Supplemental Information <br />                             Nay et al.                                                                                             Page 2 <br />  <br />  <br />                               1. Introduction <br />                                           Two major parameters concern studying DNA damage repair: survival and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           mutation. Alkylating agents form DNA RNA adducts cause cytotoxic and/or <br />                                           mutagenic effects damage repaired [1]. Because alkylating agents present <br />                                           environmentally generated cell oxidative metabolism, exposure to <br />                                           unavoidable. Therefore, removal alkyl/methyl group damage DNA RNA is <br />                                           critical maintain proper cellular function, prevent cell death, avoid mutations, and <br />                                           preserve genomic integrity. <br />  <br />                                           Discovered <span id='am-61' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli, alkylation protein B (AlkB) belongs super-<span id='am-16' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> <br />                                           Fe(II)/&#945;-ketoglutarate-dependent dioxygenases, roles histone demethylation[2], <br />                                           proline hydroxylation[3] case AlkB, DNA RNA dealkylation. Dealkylation <br />                                           catalyzed AlkB homologs occurs transformation &#945;-KG succinate, <br />                                           formaldehyde release, restoration undamaged base [4&#8211;9]. The reaction catalyzed <br />                                           AlkB direct, requiring DNA polymerases synthesis. The major methylated bases <br />                                           repaired AlkB 1-methyladenine (1-<span id='am-32' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-36'  ></span>meA</span>) 3-methylcytosine (3-meC), in <br />                                           addition, 3-methylthymine, 1-methylguanine, ethylated bases, etheno bases are <br />                                           eliminated DNA AlkB [10&#8211;13]. Lesions repaired AlkB interfere <br />                                           DNA base pairing block replication transcription [14]. In E. coli AlkB mutants, <br />                                           alkyl adducts accumulate, causing increased sensitivity alkylating agents, particularly the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SN2 type, increased mutation frequency site specific adducts [15]. <br />  <br />                                           In mice, humans, AlkB super-family composed AlkB homologs <br />                                           (Alkbh1&#8211;8 Fto). Human ALKBH5 HIF-1&#945; responsive gene [16], ALKBH3 <br />                                           8 possess RNA tRNA demethylase activity, respectively [17&#8211;20]. ALKBH1, 2 and <br />                                           3 hand, remove methyl adducts DNA [8, 17, 18, 21, 22]. Evaluation of <br />                                           Alkbh2 3 mammals shown that, similar AlkB-deficient E. coli, immortalized <br />                                           Alkbh2-deficient mouse embryonic fibroblasts (MEFs) sensitive methyl <br />                                           methanesulfonate (MMS) cytotoxicity comparison immortalized wild type (WT) <br />                                           MEFs. However, immortalized Alkbh3-deficient MEFs manifest approximately same <br />                                           cytoxicity MMS WT immortalized MEFs [23]. Interestingly, mammalian <br />                                           AlkBh genes, including Alkbh2 3, implicated prostate, brain, lung, and <br />                                           bladder cancers play role sensitivity cisplatin therapeutics [24&#8211;28]. However, the <br />                                           exact roles Alkbh2 Alkbh3 mammalian cell survival mutagenesis to <br />                                           established. <br />  <br />                                           Therefore, <span id='am-37' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-38' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> determine mutagenic consequences unrepaired SN2 alkylation <br />                                           adducts, generated mice targeted deletions Alkbh2 Alkbh3 crossed them <br />                                           Big Blue mice harboring &#955;LIZ reporter constructs. We examined survival of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           primary MEFs response MMS exposure used cII reporter determine the <br />                                           spontaneous MMS-induced mutant frequencies mutation spectra MEF <br />                                           genotype. <br />  <br />                               2. Materials Methods <br />                               2.1 Chemicals, proteins, molecular biology <br />                                           Restriction enzymes obtained New England Biolabs (Ipswich, MA). All <br />                                           molecular biology methods, including cloning, polymerase chain reaction (PCR) were <br />                                           performed using standard protocols [29]. <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                           Page 3 <br />  <br />  <br />                               2.2 Alkbh2 Alkbh3 targeted deletions mouse embryonic stem cells <br />                                           Targeted disruption performed Y129 mouse embryonic stem cells (mESCs) to <br />                                           remove exons contain critical active site residues Alkbh2 Alkbh3 gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [18]. Plasmid constructs 5&#8217; 3&#8217; arms generated eliminate exon 4 <br />                                           (Alkbh2) 7 (Alkbh3) (Fig. 1). The linearized constructs electroporated, allowed to <br />                                           undergo homologous recombination, positively selected G418 (Geneticin) <br />                                           resistance. A negative selection reduce false positives non-homologous <br />                                           recombination employed using diphtheria toxin. Individual G418-resistant colonies <br />                                           24-well plates lysed screened using long PCR identify putative clones that <br />                                           underwent correct recombination. Gene disruption produce heterozygous mESCs was <br />                                           confirmed using Southern blot PCR analysis. <br />  <br />                               2.3 Southern blot analysis Alkbh2 Alkbh3 <br />                                           Genomic DNA Alkbh2 targeted mESCs digested using BclI restriction <br />                                           endonuclease produce 1833 1461 bp fragments targeted event 2163 <br />                                           bp WT DNA fragment. Similarly, genomic DNA Alkbh3 targeted mESCs was <br />                                           digested using ScaI ClaI produce 1687 2653 bp fragments, targeted <br />                                           event, 3248 bp WT fragment (Fig. 1). Electrophoresis digested DNA carried out <br />                                           prior alkaline transfer, vacuum 2 h, charged nylon membrane. Transferred <br />                                           DNA fixed UV radiation membrane incubated blocking solution 1 h. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA &#947;-[32P]-dCTP probes, generated linear PCR, 5&#8217; 3&#8217; arms were <br />                                           hybridized membrane, washed times 50&#176;C, probe signals detected by <br />                                           phosphorimaging using Typhoon Phosphorimager (GE Healthcare, Piscataway, NJ). <br />  <br />                               2.4 Embryonic stem cell transfer mouse breeding <br />                                           To produce chimeric animals, Alkbh2 Alkbh3 heterozygous mESCs microinjected <br />                                           mouse embryos blastocyst stage implanted female mice City of <br />                                           Hope Transgenic Mouse Core. Offspring chimeric mice genotyped insure <br />                                           germ line transfer Alkbh2 Alkbh3 genotype. Homozygous Alkbh2 or <br />                                           Alkbh3 knock-out mice obtained verified genotyping. WT, Alkbh2- and <br />                                           Alkbh3 knock-out strains bred Big Blue B6 mice (Agilent Technologies, Santa <br />                                           Clara, CA) City Hope Animal Resource Center create knock-out Alkbh mouse <br />                                           models ~40 contiguous copies chromosomally integrated &#955;LIZ bacteriophage. The <br />                                           &#955;LIZ harbors cII reporter gene allows determination spontaneous induced <br />                                           mutant frequencies. Big Blue mice crossed homozygous Alkbh2 Alkbh3 knock- <br />                                           mice genotyped verify presence absence Alkbh2Alkbh3, &#955;LIZ <br />                                           phage mutation reporter. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.5 Genotyping/PCR <br />                                           Mouse tail tissue (1&#8211;2 mm) dissolved overnight 55&#176;C 200 &#181;l 1X PBND Buffer (50 <br />                                           mM KCl, 10 mM Tris-HCl pH 8.3, 2.5 mM MgCl2, 0.1 mg/ml Gelatin, 0.45% v/v NP-40, <br />                                           0.45% v/v Tween-20) containing 0.05 &#181;g/&#181;l Proteinase K. Oligodeoxynucleotide (ODN) <br />                                           primers obtained Integrated DNA Technologies, Inc. (Coralville, Iowa) were <br />                                           used indicated Table 1. PCR conditions used follows: For cII: 1 cycle (95&#176;C 3 <br />                                           min, 95&#176;C 30 s, 60&#176;C 1 min, 72&#176;C 1 min), 29 cycles (95&#176;C 30 s, 60&#176;C 1 min, 72&#176;C 1 min), <br />                                           72&#176;C 10 min, 4&#176;C hold. For mouse Alkbh2/3: 1 cycle (95&#176;C 5 min, 95&#176;C 45 s, 58&#176;C 1 min, <br />                                           72&#176;C 1 min), 29 cycles (95&#176;C 45 s, 58&#176;C 1 min, 72&#176;C 1 min), 72&#176;C 10 min, 4&#176;C hold. PCR <br />                                           products separated 2.0% agarose gels electrophoresis (Tris-HCl, acetate, EDTA <br />                                           buffer), stained ethidium bromide visualized UV light. <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                           Page 4 <br />  <br />  <br />                               2.6 Mouse embryonic fibroblast isolation <br />                                            Embryos harvested mice approximately day 11 post-coitus, washed times <br />                                            1X phosphate buffered saline (PBS) placed individual 15 ml falcon tubes. 500 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            &#181;l collagenase type 4 (~66 U/ml) added embryo followed incubation at <br />                                            37&#176;C digested. Cells transferred multiple 100 mm tissue culture dishes <br />                                            (Corning, Corning, NY) containing Dulbecco&#8217;s Modified Eagle Medium (DMEM) <br />                                            (Mediatech Inc., Manassas, VA), supplemented 10% fetal bovine serum (FBS) (Omega <br />                                            Scientific, Tarzana, CA) 1X L-glutamine, penicillin streptomycin solution (Life <br />                                            Technologies, Carlsbad, CA). The MEF cells cultured temperature <br />                                            controlled 37&#176;C/5% CO2 incubator 80&#8211;90% confluent, point frozen ampoules <br />                                            prepared (5% DMSO, 30% FBS, 65% DMEM) stored liquid nitrogen. <br />  <br />                               2.7 Cell culture <br />                                            Single ampoules WT, Alkbh2, Alkbh3 knock-out MEF cells thawed cultured <br />                                            DMEM, supplemented 10% FBS 1X L-glutamine, penicillin streptomycin <br />                                            solution temperature controlled 37&#176;C/5% CO2 incubator. Cells passaged at <br />                                            approximately 80&#8211;90% confluence, using 0.05% trypsin-EDTA solution, prepared by <br />                                            diluting 0.25% trypsin-EDTA solution (Life Technologies) 1X PBS. <br />  <br />                               2.8 Confirmation targeted exon deletions mouse embryonic fibroblasts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Nuclear DNA extracted Alkbh2 Alkbh3 knock-out MEF cells evaluated for <br />                                            absence AlkBh2 Exon 4 AlkBh3 Exon 7 (Fig. 1), presence Neo by <br />                                            PCR. ODN primers used indicated Table 1. PCR conditions used follows: <br />                                            (m2E4/m3E7/neo) 1 cycle (95&#176;C 3 min, 95&#176;C 30 s 60&#176;C 1 min, 72&#176;C 1 min) 29 cycles (95&#176;C <br />                                            30 s 60&#176;C 1 min, 72&#176;C 1 min), 72&#176;C 5 min, 4&#176;C hold. PCR products separated a <br />                                            2.0% agarose gel electrophoresis (TAE buffer), stained ethidium bromide, and <br />                                            visualized UV light. <br />  <br />                               2.9 Western blot analysis Alkbh 2 3 protein production mouse embryonic <br />                               fibroblasts <br />                                            Protein extracted WT Alkbh2 Alkbh3 knock-out MEFs evaluated for <br />                                            expression Alkbh2 Alkbh3 proteins. MEF cells washed 1X PBS lysed <br />                                            ice-cold RIPA buffer (20 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1% NP-40, 0.5% <br />                                            sodium deoxycholate, 1 mM EDTA, 0.1% SDS,), supplemented protease inhibitor <br />                                            cocktail (Roche, Indianapolis, IN), just use. Lysates incubated ice 10 <br />                                            min, vortexed gently 30 s centrifuged (10 min, 1000&#215;g, 4&#176;C). Supernatants were <br />                                            collected protein concentrations determined using Bradford assay [30]. WT and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Alkbh2 Alkbh3 knock-out protein samples (50&#181;g) combined 5X SDS-PAGE <br />                                            loading buffer dH2O, heated (5 min, 95&#176;C) separated electrophoresis 4 &#8211; 15% <br />                                            Mini Protean TGX SDS-PAGE gels (Bio-Rad Laboratories, Hercules, CA) (250 V, 25 min), <br />                                            Samples transferred 0.2 &#181;m PVDF membranes (Bio-Rad) (100 V, 1 h) using wet <br />                                            electro-transfer (0.2 M <span id='am-11' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span>, 25 mM Tris 20% methanol). Membranes blocked <br />                                            Li-Cor Biosciences (Lincoln, Nebraska) Odyssey Infrared Imaging System Blocking <br />                                            Buffer (1 h, RT overnight, 4&#176;C), followed incubation anti-actin anti-Alkbh2 <br />                                            -Alkbh3 primary antibodies (Santa Cruz Biotechnology Inc., Bath, UK) (1:5000 50% <br />                                            (v/v) Odyssey Blocking Buffer/1X TBS) (1 h, RT overnight, 4&#176;C). Membranes were <br />                                            washed (3&#215;5 min) 1X TBS-Tween (1/1000 (v/v)), incubated anti-mouse IR800 and <br />                                            anti-rabbit IR700 secondary antibodies (Li-Cor Biosciences) (1:20,000 50% (v/v) <br />                                            Odyssey Blocking Buffer/1X TBS) (1 h, RT) washed (1X TBS-T, 3&#215;5 min). <br />                                            Detection proteins conducted using Li-Cor Odyssey near-infrared Imaging <br />                                            Station. <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                               Page 5 <br />  <br />  <br />                               2.10 Methyl methanesulfonate treatments cell survival mutation assays <br />                                            MEFs seeded 6-well culture dishes 1&#215;106 cells/well. The following day MEFs <br />                                            washed twice 1X PBS treated 20 min MMS (0 &#8211; 100 mM) serum- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            free DMEM serum-free DMEM minus MMS. Following treatment, cells were <br />                                            washed twice 1X PBS fresh DMEM (10% FBS) media added. Sensitivity each <br />                                            cell line SN2 alkylating agent MMS determined Trypan Blue uptake MEFs <br />                                            48 h post-MMS treatment. The lowest determined IC50 value, 4 mM, utilized <br />                                            MMS treatment dose cII mutation assays. WT Alkbh knock-out <br />                                            MEFs incubated serum-free media (12&#215;150 mm) (3&#215;150 mm) <br />                                            MMS, described above. Following treatment, cells incubated 37&#176;C/5% CO2 5 <br />                                            days. Medium replaced 48 h cells passaged necessary. <br />  <br />                               2.11 Genomic DNA isolation <br />                                            Cells trypsinized (0.05% trypsin), pooled 4&#215;150 mm culture dishes, pelleted, and <br />                                            washed twice 1X PBS. Pellets lysed, ice 5 min, 5 ml Buffer A (0.3 M <br />                                            Sucrose, 60 mM KCl, 15 mM NaCl, 60 mM Tris-HCl, pH 8, 0.5 mM spermidine, 0.15 mM <br />                                            spermine, 2 mM EDTA) 0.5% NP-40. Pelleted nuclei obtained centrifugation <br />                                            (5 min, 1000&#215;g, 4&#176;C). Buffer A 0.5% NP-40 decanted replaced 5 ml <br />                                            Buffer A. Pelleted nuclei centrifuged Buffer A decanted prior to <br />                                            resuspension equal volumes (2.5 ml) Buffer B (150 mM NaCl, 5 mM EDTA pH 7.8), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Buffer C [20 mM Tris-HCl pH 8, 20 mM NaCl, 20 mM EDTA, 1% SDS, 600 &#181;g/ml <br />                                            Proteinase K (added just use)]. Samples incubated 37&#176;C 3 h residual <br />                                            RNA removed addition RNAse A (10 mg/ml) final concentration 100 <br />                                            &#181;g/ml, incubation 37&#176;C 1 h. DNA extracted following standard phenol/ <br />                                            chloroform protocols [31]. DNA aqueous phase precipitated addition three <br />                                            volumes cold 100% ethanol, incubation ice 30 min centrifugation (15 min, <br />                                            1000&#215;g, 4&#176;C). Precipitated genomic DNA washed 70% ethanol, centrifuged (15 <br />                                            min, 1000&#215;g, 4&#176;C), air-dried re-suspended 0.1X Tris-EDTA. Concentrations were <br />                                            determined using NanoDrop spectrophotometer (Thermo Scientific, Wilmington, DE). <br />  <br />                               2.12 cII mutation assay <br />                                            cII mutant frequency assays carried indicated manufacturer (Agilent <br />                                            Technologies). Nuclear genomic DNA extracted, described above, packaged <br />                                            &#955; phage infected G1250 host E. coli cells. Infected host cells plated at <br />                                            different dilutions cultured non-selective temperature (37&#176;C) determine total <br />                                            number phage screened. Undiluted host cells cultured selective <br />                                            temperature (24&#176;C) lysogenic lytic growth. Mutant frequencies determined for <br />                                            WT, Alkbh2, Alkbh3 knock-out MEFs obtained single embryos, were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            repeated separate litters. In case 3&#215;105 phage plaques screened for <br />                                            mutations. <br />  <br />                               2.13 Mutant sequence analysis <br />                                            Mutant cII sequences amplified PCR, recommended manufacturer <br />                                            (Agilient Technologies, Santa Clara CA), sequenced Laragen, Inc. (Culver City, CA) <br />                                            City Hope Sequencing Core (Duarte, CA). Sequence alignments prepared <br />                                            ClustalW2 sequence analysis software. <br />  <br />                               2.14 Statistical analysis <br />                                            Following recommendations mutation spectra [32, 33], significance probabilities for <br />                                            aggregated <span id='am-74' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-75' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span>data</span> calculated chi-square statistics referred Monte Carlo null <br />                                            distributions, using StatXact 7.0.0 (Cytel Software Corporation, Cambridge, MA). A global <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                              Page 6 <br />  <br />  <br />                                            test Table 2 used support experiment-wide significance. Tests Poisson rates <br />                                            used comparisons single mutation types. Significance hotspots was <br />                                            established reference Monte Carlo simulation Poisson process, using R version <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2.14.0. <br />  <br />                               3. Results <br />                               3.1 Generation mice targeted deletions Alkbh2 Alkbh3 <br />                                            All AlkB homologs share conserved active site contains critical domains; the <br />                                            nucleotide recognition lid, N-terminal extension catalytic core [34&#8211;36]. In the <br />                                            mouse AlkB homologs Alkbh2 Alkbh3, domains span exons 2&#8211;4 2&#8211;9, <br />                                            respectively. Therefore, targeted deletion exon 4 Alkbh2 exon 7 Alkbh3 was <br />                                            carried homologous recombination mESCs confirmed Southern blot <br />                                            analysis (Fig. 1). Following transfer heterozygous mESCs blastocysts, chimeric <br />                                            offspring isolated used establish colonies Alkbh2- Alkbh3-deficient mice. <br />                                            WT Alkbh2- Alkbh3-deficient mice bred Big Blue &#955;LIZ mutation <br />                                            reporter mice establish WT, Alkbh2- Alkbh3-deficient mouse colonies contained <br />                                            mutation reporter construct. Breeding Big Blue Alkbh2- Alkbh3-deficient mouse <br />                                            colonies conducted obtain Alkbh2/3 double knock-out mouse <br />                                            colonies. These colonies used generate MEF cells used study. As reported <br />                                            previously, obvious pathologies, including changes fertility longevity observed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            [23]. <br />  <br />                               3.2 Isolation primary wild-type, Alkbh2&#8722;/&#8722;, Alkbh3&#8722;/&#8722; mouse embryonic fibroblasts <br />                                            Because spontaneous cellular immortalization MEFs generally results large number <br />                                            changes genome alter genomic stability, used primary MEF cells in <br />                                            experiments. Primary MEF cells isolated individual embryos harvested at <br />                                            ~11 day post-coitus screened targeted Alkbh2 Alkbh3 deletions using <br />                                            PCR analysis (Fig. 2). Western blot analysis WT Alkbh2- Alkbh3-deficient MEF <br />                                            protein extracts revealed MEFs targeted deletions Alkbh2 Alkbh3 not <br />                                            express respective proteins (Supplemental Fig. 1). For experiment, separate, <br />                                            non-litter mate embryos used ensure results obtained litter- <br />                                            dependent. MEFs used study passage seven less. Though Alkbh2/3 double <br />                                            knock-out mice viable, multiple occasions, hands different individuals, <br />                                            MEFs prepared various embryos able cultured passage three. As <br />                                            experiments conducted using MEFs prepared single embryos, number cells <br />                                            required viability, activity, cII mutation assays precluded use primary <br />                                            Alkbh2/3 double knock-out MEFs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.3 Both Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; primary mouse embryonic fibroblasts more <br />                               sensitive wild-type cells methyl methanesulfonate <br />                                            Exposure MMS induces variety DNA adducts, including 1-meA 3-meC, of <br />                                            halt replication result lethality high MMS doses. To assess protective <br />                                            roles Alkbh2 Alkbh3 chemically-induced cytotoxicity, WT, Alkbh2&#8722;/&#8722; and <br />                                            Alkbh3&#8722;/&#8722; primary MEFs treated MMS assayed cytotoxicity based <br />                                            membrane integrity (Fig. 3a). Both Alkbh2- Alkbh3-deficient lines showed an <br />                                            2-fold, statistically significant lower IC50 value compared WT primary MEFs <br />                                            (Fig. 3b). <br />  <br />  <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                                Page 7 <br />  <br />  <br />                               3.4 Alkbh2 greater protective effect chromosomal mutations Alkbh3 <br />                                           In absence Alkbh2 Alkbh3, cell viability primary MEFs significantly reduced <br />                                           response MMS treatment. Since deficiency Alkbh2 Alkbh3 result in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           adduct persistence increase mutant frequency, compared spontaneous <br />                                           MMS-induced mutant frequencies primary WT, Alkbh2&#8722;/&#8722;, Alkbh3&#8722;/&#8722; MEFs. <br />                                           The primary MEF lines treated 4 mM MMS serum-free <br />                                           medium allowed replicate day period. Genomic DNA MEFs <br />                                           isolated used phage packaging reaction infect G1250 host cells, which <br />                                           allowed detection mutations cII gene &#955;LIZ reporter (Supplemental Fig. <br />                                           2). The 1.5- 2.7-fold increase mutant frequency observed untreated Alkbh2&#8722;/&#8722; and <br />                                           Alkbh3&#8722;/&#8722; MEFs did statistically significant difference WT MEFs. <br />                                           However, following MMS treatment, mutant frequency Alkbh2&#8722;/&#8722; MEFs was <br />                                           significantly increased compared treated WT (~7-fold) spontaneous Alkbh2&#8722;/&#8722; <br />                                           MEF mutant frequency (~6-fold). In contrast Alkbh2&#8722;/&#8722; MEFs, increase mutant <br />                                           frequency MMS treated Alkbh3&#8722;/&#8722; MEFs (~3-fold) statistically significant, <br />                                           compared WT control. Comparison induced mutant frequency (observed &#8211; <br />                                           spontaneous) WT Alkbh2- Alkbh3-deficient MEFs showed increases 4.0, 70.5, <br />                                           12.2 (&#215;10&#8722;6), respectively (Fig. 4). Similarly, Alkbh3&#8722;/&#8722; MEFs exhibited ~3-fold <br />                                           difference compared WT MEFs, significant. Alkbh2&#8722;/&#8722; MEFs, on <br />                                           hand, exhibited statistically significant 17.6-fold increase induced mutant <br />                                           frequency, compared WT MEFs. Therefore, decreased survival Alkbh2&#8722;/&#8722; MEFs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           following MMS treatment accompanied significant increase mutant frequency. <br />                                           However, Alkbh3&#8722;/&#8722; cells showed decreased survival response MMS <br />                                           treatment, corresponding statistically significant increase MMS-induced <br />                                           mutants. Therefore, Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; primary MEFs provide equal <br />                                           protection MMS treatment similar substrate specificities protect against <br />                                           similar types mutations, Alkbh2 greater protective effect mutations than <br />                                           Alkbh3. <br />  <br />                               3.5 Spontaneous C&#8594;A transversion C&#8594;T transition mutations increased in <br />                               Alkbh2&#8722;/&#8722; mouse embryonic fibroblasts <br />                                           To examine type mutations induced, cII gene &#955;LIZ reporter 29 <br />                                           mutant phage plaques spontaneous mutations, primary MEF line, were <br />                                           sequenced (Table 2, Supplemental Figure 3). Compared WT cells, Alkbh2&#8722;/&#8722; MEFs <br />                                           showed different spontaneous mutation spectrum (p=0.006). The striking feature was <br />                                           large increase C&#8594;A mutations (2.7-fold, p=0.0001). There smaller but <br />                                           noticeable increase number C&#8594;T mutations (1.6-fold, p=0.06). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.6 Spontaneous C&#8594;T transition C&#8594;A transversion mutations increased in <br />                               Alkbh3&#8722;/&#8722; mouse embryonic fibroblasts <br />                                           The major change mutation spectrum Alkbh3&#8722;/&#8722; MEFs compared WT the <br />                                           increase C&#8594;T mutations (2.6-fold, p=0.003). The number C&#8594;A mutations also <br />                                           increased (1.6-fold, p=0.05), statistical significance observed for <br />                                           Alkbh2&#8722;/&#8722; MEF spontaneous mutations. The increase C&#8594;T (G&#8594;A the <br />                                           complementary strand) mutations mutants unexpected, <span id='am-24' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of <br />                                           transition mutation generally associated O6-methylguanine adduct repair. <br />  <br />                               3.7 Methyl methanesulfonate-induced C&#8594;A transversions C&#8594;T transitions <br />                               predominate Alkbh2&#8722;/&#8722; mutation spectrum <br />                                           Although MMS treatment significantly increased mutant frequency Alkbh2&#8722;/&#8722; MEFs, <br />                                           prevalent type mutation Alkbh2&#8722;/&#8722; MEFs remained C&#8594;A transversions (3.0- <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                               Page 8 <br />  <br />  <br />                                           fold). There increased C&#8594;T mutations. Therefore, mutation <br />                                           frequency increased MMS treatment, types mutations induced similar to <br />                                           spontaneous mutations. However, observed, non-significant increase in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           A&#8594;T transitions MMS treated Alkbh2&#8722;/&#8722; MEFs compared low frequency A&#8594;T <br />                                           transitions spontaneous Alkbh2&#8722;/&#8722; MEF mutations (p=0.14). Another difference <br />                                           WT Alkbh2&#8722;/&#8722; following MMS treatment absence deletion <br />                                           mutations Alkbh2&#8722;/&#8722; mutation spectrum. Therefore, C&#8594;A C&#8594;T increases are <br />                                           noteworthy spontaneous MMS-induced Alkbh2&#8722;/&#8722; MEFs, compared the <br />                                           WT MEFs, A&#8594;T transitions increased MMS exposure. <br />  <br />                               3.8 Methyl methanesulfonate-induced C&#8594;T transitions C&#8594;A transversions represent <br />                               major changes Alkbh3&#8722;/&#8722; mutation spectrum <br />                                           The MMS treatment Alkbh3&#8722;/&#8722; MEFs primarily increased G&#8594;T (1.4-fold) mutations <br />                                           compared MMS treated WT MEFs. C&#8594;T mutations increased compared to <br />                                           WT. Again increase number A&#8594;T mutations observed the <br />                                           Alkbh3&#8722;/&#8722; following MMS treatment. <br />  <br />                               3.9 Mutation hot spots Alkbh2&#8722;/&#8722; MEFs associated C&#8594;A transversions <br />                                           Monte Carlo simulation 1000 samples homogeneous Poisson process suggests six <br />                                           mutations given position threshold statistical significance. Given criterion, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           positions appeared hot spots occurred DNA Alkbh2&#8722;/&#8722; <br />                                           MEF cells. In untreated Alkbh2&#8722;/&#8722; MEF cells, hot spot 17 G&#8594;T (C&#8594;A) spontaneous <br />                                           mutations observed position 159 (Fig. 6). In MMS treated Alkbh2&#8722;/&#8722; MEFs, the <br />                                           mutation analysis, revealed single hot spot G&#8594;T (C&#8594;A) mutations position <br />                                           301 (Fig. 6). No hot spots mutations detected primary <br />                                           MEFs using analysis (Fig. 5 7). This result appears consistent the <br />                                           observation MMS treated primary Alkbh2&#8722;/&#8722; MEFs significantly higher mutant <br />                                           frequency observed MMS treated WT Alkbh3&#8722;/&#8722; MEFs. <br />  <br />                               4. Discussion <br />                                           Exposure alkylating agents, originating environment cell, <br />                                           means subsequent generation alkyl methyl adducts DNA RNA, virtually <br />                                           unavoidable. Consequently, removal alkyl/methyl groups DNA RNA is <br />                                           critical cellular function, unrepaired adducts cytotoxic and/or mutagenic <br />                                           effects. At present, identification <span id='am-73' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> specific biological roles of <br />                                           mammalian proteins involved repair DNA alkylation adducts ongoing process. In <br />                                           E. coli, AlkB repairs alkylation damage protects cells adaptive response to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           alkylating agents [7, 37]. However, humans possess E. coli AlkB homologs <br />                                           (ALKBH1&#8211;8 FTO), biological roles understood. Fortunately, <br />                                           conservation mammalian AlkB homologs mice humans provides important <br />                                           laboratory model study repair proteins mammalian systems. Previous <br />                                           studies shown conservation active site, comparable activities <br />                                           mouse human AlkB homologs [18]. <br />  <br />                                           Currently, ALKBH proteins, ALKBH1&#8211;3, known remove 1-meA 3-meC <br />                                           adducts DNA disrupt replication transcription, triggering cell <br />                                           cycle checkpoints apoptosis [17, 21, 22, 36, 38, 39]. ALKBH1 mitochondrial <br />                                           protein [22], concentrated elucidating roles ALKBH2 ALKBH3 <br />                                           considered nuclear proteins [40]. Despite known repair <br />                                           activities proteins, actual biological roles established. <br />                                           Generation mouse models targeted deletions Alkbh2 Alkbh3, integration <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                           Page 9 <br />  <br />  <br />                                         &#955;LIZ transgene system, allowed determine spontaneous and <br />                                         MMS-induced mutant frequencies primary MEFs deficient two <br />                                         homologs, generate mutation spectra maps specific deficient cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         line. <br />  <br />                                         Our results, using primary MEFs, support previous findings using immortalized Alkbh2- <br />                                         deficient cell line, absence Alkbh2 causes increased susceptibility SN2 <br />                                         alkylating agents, specifically MMS [23]. However contrast original report, we <br />                                         note addition Alkbh2-deficient primary MEFs, Alkbh3-deficient MEFs at <br />                                         twice sensitive MMS compared wild type MEFs. That suggests a <br />                                         difference survival primary immortalized Alkbh3-deficient cells. <br />  <br />                                         In spite difference survival noted Alkbh3-deficient primary fibroblasts, a <br />                                         significant effect mutant frequency observed Alkbh2-deficient primary <br />                                         MEFs. However, shown recent study, various human cell types depend differently on <br />                                         ALKBH2 3. Therefore, possible transformed cells, generated various <br />                                         Alkbh3-deficient mouse tissue types, effect mutant frequency significant <br />                                         [24]. Moreover, uncertain adducts responsible increased <br />                                         sensitivity MMS observed MEFs mutagenic consequences cell the <br />                                         absence repair. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Until study, mutagenic potential 1-meA 3-meC DNA adducts been <br />                                         determined AlkB-deficient E. coli cells [15]. In report, 3-meC adducts introduced <br />                                         <span id='am-79' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> mutagenic, 90% mutations associated C&#8594;A and <br />                                         C&#8594;G transversions C&#8594;T transitions. In contrast, 1-meA adducts significantly <br />                                         mutagenic exhibited mutation frequency ~5%, resulted A&#8594;T <br />                                         transition mutations [15]. Therefore, provide better understanding biological role <br />                                         mammalian AlkB homologs, studied mutagenic impact Alkbh2 Alkbh3 <br />                                         homolog deletion examining spontaneous MMS-induced mutations cII <br />                                         transgene primary Big Blue WT, Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; MEF cells. <br />  <br />                                         Similar mutations produced unique 3-meC adducts AlkB-deficient E. coli [15], <br />                                         Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; MEFs increased spontaneous C&#8594;A mutations (50% and <br />                                         30%, respectively) compared WT (17%) increased spontaneous C&#8594;T transversions <br />                                         (20% 34%, respectively) compared WT (10%). These results suggest repair <br />                                         3-meC adducts ssDNA dsDNA critical maintenance genomic integrity <br />                                         Alkbh2 Alkbh3 primary repair proteins adducts. Interestingly, <br />                                         following MMS treatment, Alkbh2- Alkbh3-deficient MEFs increases in <br />                                         C&#8594;A mutations (34% 21%, respectively) C&#8594;T transversions (32% 28%, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         respectively) compared WT (C&#8594;A, 19% C&#8594;T 22%), exhibit increases in <br />                                         frequency A&#8594;T mutations (18% 26%, respectively), comparable those <br />                                         observed WT MEFs (11%). In E. coli, 1-meA adducts essentially non-mutagenic <br />                                         (~5% T&#8594;A) [15]. However adducts generated 2-fold ssDNA and <br />                                         4-fold dsDNA 3-meC [41] accumulate Alkbh2-deficient <br />                                         immortalized MEFs [23]. <br />  <br />                                         1-meG 3-meT formed low frequencies (&lt; 0.5%) ssDNA, vitro in <br />                                         vivo, SN2 methylating agents [41]. The evaluation mutation frequencies E. coli <br />                                         using restriction endonuclease post-labeling analysis mutation (REAP) assay <br />                                         indicated unrepaired 1-meG 3-meT DNA adducts mutagenic (20% G&#8594;A, 50% <br />                                         G&#8594;T 40% T&#8594;A, 10% T&#8594;C, respectively) [15]. Mammalian Alkbh2 Alkbh3 also <br />                                         repair 1-meG 3-meT adducts [11, 13], contribute increased mutant <br />                                         frequencies observed Alkbh2- Alkbh3-deficient MEFs MMS treatment. <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                                                      Page 10 <br />  <br />  <br />                                              However, frequency 1-meG 3-meT formation MMS than <br />                                              1-meA 3-meC, probability observing mutations adducts low <br />                                              [41]. Therefore, best way determine adducts responsible type of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              mutation require use single adduct lesions episomes replicate mouse <br />                                              cells followed analysis mutations produced. [42]. <br />  <br />                                              Thus, determining frequency adduct occurs spontaneously response <br />                                              MMS treatment, using Alkbh-deficient mouse models, greatly contribute our <br />                                              understanding rate occurrence mutagenic potential type DNA <br />                                              damage. Furthermore, determining repair capacity Alkbh2 Alkbh3 proteins <br />                                              adduct critical identifying biological role, cell, family of <br />                                              proteins. In addition, information, combined studies mouse models, will <br />                                              understanding role played Alkbh2 Alkbh3 certain cancers and <br />                                              provide insight used predictive diagnosis potential therapeutics. <br />  <br />                               Supplementary Material <br />                                              Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              We thank City Hope, Payson Foundation, TRDRP, City Hope Cancer Center funding Drs. <br />                                              Walter Tsark, Seung-Gi Jin, Margaret Morgan scientific expertise help manuscript <br />                                              preparation. Dr. Jeffrey Longmate gratefully acknowledged assistance statistical analysis the <br />                                              mutation spectra maps. <br />  <br />  <br />                               Abbreviations <br />  <br />                                         MMS                       methylmethane sulfonate <br />                                         ds                        double-stranded <br />                                         ss                        single-stranded <br />                                         ODN                       oligodeoxynucleotides <br />                                         &#945;-KGFdiO                  &#945;-ketoglutarate Fe(II) dioxygenase <br />                                         MEFs                      mouse embryonic fibroblasts <br />                                         mESCs                     mouse embryonic stem cells. All gene names italics human gene <br />                                                                   names capitalized mouse gene names letter <br />                                                                   capitalized, E. coli gene names lower case. Gene product (protein) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                   names non-italicized convention gene names for <br />                                                                   human mouse. For E. coli, protein names letter <br />                                                                   capitalized <br />  <br />                               References <br />                                              1. Drablos F, Feyzi E, Aas PA, Vaagbo CB, Kavli B, Bratlie MS, Pena-Diaz J, Otterlei M, Slupphaug <br />                                                 G, Krokan HE. Alkylation damage DNA RNA--repair mechanisms medical significance. <br />                                                 DNA Repair (Amst). 2004; 3:1389&#8211;1407. [PubMed: 15380096] <br />                                              2. Shin S, Janknecht R. Diversity JMJD2 histone demethylase family. Biochem Biophys <br />                                                 Res Commun. 2007; 353:973&#8211;977. [PubMed: 17207460] <br />                                              3. Flashman E, Davies SL, Yeoh KK, Schofield CJ. Investigating dependence hypoxia- <br />                                                 inducible factor hydroxylases (factor inhibiting HIF prolyl hydroxylase domain 2) ascorbate <br />                                                 reducing agents. Biochem J. 2010; 427:135&#8211;142. [PubMed: 20055761] <br />  <br />  <br />  <br />                                                 DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                                   Page 11 <br />  <br />  <br />                                         4. Begley TJ, Samson LD. AlkB mystery solved: oxidative demethylation N1-methyladenine and <br />                                            N3-methylcytosine adducts direct reversal mechanism. Trends Biochem Sci. 2003; 28:2&#8211;5. <br />                                            [PubMed: 12517444] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         5. Bleijlevens B, Shivarattan T, Flashman E, Yang Y, Simpson PJ, Koivisto P, Sedgwick B, Schofield <br />                                            CJ, Matthews SJ. Dynamic states DNA repair enzyme AlkB regulate product release. EMBO <br />                                            Rep. 2008; 9:872&#8211;877. [PubMed: 18617893] <br />                                         6. Falnes PO, Johansen RF, Seeberg E. AlkB-mediated oxidative demethylation reverses DNA damage <br />                                            Escherichia coli. Nature. 2002; 419:178&#8211;182. [PubMed: 12226668] <br />                                         7. Kataoka H, Sekiguchi M. Molecular cloning characterization alkB gene Escherichia <br />                                            coli. Mol Gen Genet. 1985; 198:263&#8211;269. [PubMed: 3884973] <br />                                         8. Mishina Y, He C. Oxidative dealkylation DNA repair mediated mononuclear nonheme iron <br />                                            AlkB proteins. J Inorg Biochem. 2006; 100:670&#8211;678. [PubMed: 16469386] <br />                                         9. Trewick SC, Henshaw TF, Hausinger RP, Lindahl T, Sedgwick B. Oxidative demethylation by <br />                                            Escherichia coli AlkB directly reverts DNA base damage. Nature. 2002; 419:174&#8211;178. [PubMed: <br />                                            12226667] <br />                                         10. Delaney JC, Smeester L, Wong C, Frick LE, Taghizadeh K, Wishnok JS, Drennan CL, Samson <br />                                              LD, Essigmann JM. AlkB reverses etheno DNA lesions caused lipid oxidation vitro in <br />                                              vivo. Nat Struct Mol Biol. 2005; 12:855&#8211;860. [PubMed: 16200073] <br />                                         11. Falnes PO. Repair 3-methylthymine 1-methylguanine lesions bacterial human AlkB <br />                                              proteins. Nucleic Acids Res. 2004; 32:6260&#8211;6267. [PubMed: 15576352] <br />                                         12. Frick LE, Delaney JC, Wong C, Drennan CL, Essigmann JM. Alleviation 1,N6-ethanoadenine <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              genotoxicity Escherichia coli adaptive response protein AlkB. Proc Natl Acad Sci U S A. <br />                                              2007; 104:755&#8211;760. [PubMed: 17213319] <br />                                         13. Koivisto P, Robins P, Lindahl T, Sedgwick B. Demethylation 3-methylthymine DNA by <br />                                              bacterial human DNA dioxygenases. J Biol Chem. 2004; 279:40470&#8211;40474. [PubMed: <br />                                              15269201] <br />                                         14. Groth P, Auslander S, Majumder MM, Schultz N, Johansson F, Petermann E, Helleday T. <br />                                              Methylated DNA causes physical block replication forks independently damage signaling, <br />                                              O(6)-methylguanine DNA single-strand breaks results DNA damage. J Mol Biol. <br />                                              402:70&#8211;82. [PubMed: 20643142] <br />                                         15. Delaney JC, Essigmann JM. Mutagenesis, genotoxicity, repair 1-methyladenine, 3- <br />                                              alkylcytosines, 1-methylguanine, 3-methylthymine alkB Escherichia coli . Proc Natl Acad <br />                                              Sci U S A. 2004; 101:14051&#8211;14056. [PubMed: 15381779] <br />                                         16. Thalhammer A, Bencokova Z, Poole R, Loenarz C, Adam L, O&#8217;Flaherty L, Sch&#246;del J, Mole D, <br />                                              Giaslakiotis K, Schofield CJ, Hammond EM, Ratcliffe PJ, Pollard PJ. Human AlkB homologue 5 <br />                                              nuclear 2-oxoglutarate dependent oxygenase direct target hypoxia-inducible factor 1&#945; <br />                                              (HIF-1&#945;). PLoS One. 2011; 6 <br />                                         17. Aas PA, Otterlei M, Falnes PO, Vagbo CB, Skorpen F, Akbari M, Sundheim O, Bjoras M, <br />                                              Slupphaug G, Seeberg E, Krokan HE. Human bacterial oxidative demethylases repair <br />                                              alkylation damage RNA DNA. Nature. 2003; 421:859&#8211;863. [PubMed: 12594517] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         18. Lee DH, Jin SG, Cai S, Chen Y, Pfeifer GP, O&apos;Connor TR. Repair methylation damage DNA <br />                                              RNA mammalian AlkB homologues. J Biol Chem. 2005; 280:39448&#8211;39459. [PubMed: <br />                                              16174769] <br />                                         19. Songe-Moller L, van den Born E, Leihne V, Vagbo CB, Kristoffersen T, Krokan HE, Kirpekar F, <br />                                              Falnes PO, Klungland A. Mammalian ALKBH8 possesses tRNA methyltransferase activity <br />                                              required biogenesis multiple wobble uridine modifications implicated translational <br />                                              decoding. Mol Cell Biol. 2010; 30:1814&#8211;1827. [PubMed: 20123966] <br />                                         20. Fu D, Brophy J, Chan C, KA A, U B, RS P, PC D, TJ B, LD S. Human AlkB homolog ABH8 Is a <br />                                              tRNA methyltransferase required wobble uridine modification DNA damage survival. Mol <br />                                              Cell Biol. 2010:2449&#8211;2459. [PubMed: 20308323] <br />                                         21. Duncan T, Trewick SC, Koivisto P, Bates PA, Lindahl T, Sedgwick B. Reversal DNA <br />                                              alkylation damage human dioxygenases. Proc Natl Acad Sci U S A. 2002; 99:16660&#8211; <br />                                              16665. [PubMed: 12486230] <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                                 Page 12 <br />  <br />  <br />                                         22. Westbye MP, Feyzi E, Aas PA, Vagbo CB, Talstad VA, Kavli B, Hagen L, Sundheim O, Akbari <br />                                             M, Liabakk NB, Slupphaug G, Otterlei M, Krokan HE. Human AlkB homolog 1 a <br />                                             mitochondrial protein demethylates 3-methylcytosine DNA RNA. J Biol Chem. 2008; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             283:25046&#8211;25056. [PubMed: 18603530] <br />                                         23. Ringvoll J, Nordstrand LM, Vagbo CB, Talstad V, Reite K, Aas PA, Lauritzen KH, Liabakk NB, <br />                                             Bjork A, Doughty RW, Falnes PO, Krokan HE, Klungland A. Repair deficient mice reveal <br />                                             mABH2 primary oxidative demethylase repairing 1meA 3meC lesions DNA. <br />                                             EMBO J. 2006; 25:2189&#8211;2198. [PubMed: 16642038] <br />                                         24. Dango S, Mosammaparast N, Sowa M, Xiong L, Wu F, Park K, Rubin M, Gygi S, Harper J, Shi Y. <br />                                             DNA Unwinding ASCC3 Helicase Is Coupled ALKBH3-Dependent DNA Alkylation Repair <br />                                             Cancer Cell Proliferation. Molecular Cell. 2011:373&#8211;384. [PubMed: 22055184] <br />                                         25. Konishi N, Nakamura M, Ishida E, Shimada K, Mitsui E, Yoshikawa R, Yamamoto H, Tsujikawa <br />                                             K. High expression new marker PCA-1 human prostate carcinoma. Clin Cancer Res. 2005; <br />                                             11:5090&#8211;5097. [PubMed: 16033822] <br />                                         26. Shimada K, Nakamura M, Anai S, De Velasco M, Tanaka M, Tsujikawa K, Ouji Y, Konishi N. A <br />                                             novel human AlkB homologue, ALKBH8, contributes human bladder cancer progression. <br />                                             Cancer Res. 2009; 69:3157&#8211;3164. [PubMed: 19293182] <br />                                         27. Tasaki M, Shimada K, Kimura H, Tsujikawa K, Konishi N. ALKBH3, human AlkB homologue, <br />                                             contributes cell survival human non-small-cell lung cancer. British Journal Cancer. <br />                                             2011:1&#8211;7. <br />                                         28. Wu SS, Xu W, Liu S, Chen B, Wang XL, Wang Y, Liu SF, Wu JQ. Downregulation ALKBH2 <br />                                             increases cisplatin sensitivity H1299 lung cancer cells. Acta Pharmacologica Sinica. 2011:1&#8211;6. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             [PubMed: 21170081] <br />                                         29. Ausubel, FM.; Brent, R.; Kingston, RE.; Moore, DD.; Seidman, JG.; Smith, JA.; Struhl, K. Current <br />                                             Protocols Molecular Biology. In: Ausubel, FM., editor. Greene Publishing Associates. New <br />                                             York: 1994. p. 5,300 <br />                                         30. Bradford MM. A rapid sensitive method quantitation microgram quantities protein <br />                                             utilizing principle protein-dye binding. Analytical Biochemistry. 1976; 72:248&#8211;254. <br />                                             [PubMed: 942051] <br />                                         31. Ye, N. Heterogenous Repair N-Methylpurines Nucleotide Level Normal Human Cells. <br />                                             In: Holmquist, GP.; O&apos;Connor, TR., editors. J. Mol. Biol. Vol. 284. 1998. p. 269-285. <br />                                         32. Adams TW, Skopek TR. Statistical Test Comparison Samples Mutational Spectra. <br />                                             J. Mol. Biol. 1987; 194:391&#8211;396. [PubMed: 3305960] <br />                                         33. Piegorsch WW, Bailer JA. Statistical Approaches Analyzing Mutational Spectra: Some <br />                                             Recommendations Categorical Data. Genetics. 1994; 136:403&#8211;416. [PubMed: 8138174] <br />                                         34. Kurowski MA, Bhagwat AS, Papaj G, Bujnicki JM. Phylogenomic identification new <br />                                             human homologs DNA repair enzyme AlkB. BMC Genomics. 2003; 4:48. [PubMed: <br />                                             14667252] <br />                                         35. Sundheim O, Vagbo CB, Bjoras M, <span id='am-68' about='obo:NCBITaxon_103599' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9721 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9722 obo:NCBITaxon_9726'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Sousa" datatype="xsd:string"></span><span id='am-72'  ></span>Sousa</span> MM, Talstad V, Aas PA, Drablos F, Krokan HE, Tainer <br />                                             JA, Slupphaug G. Human ABH3 structure key residues oxidative demethylation reverse <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA/RNA damage. EMBO J. 2006; 25:3389&#8211;3397. [PubMed: 16858410] <br />                                         36. Wei YF, Carter KC, Wang RP, Shell BK. Molecular cloning functional analysis human <br />                                             cDNA encoding Escherichia coli AlkB homolog, protein involved DNA alkylation damage <br />                                             repair. Nucleic Acids Res. 1996; 24:931&#8211;937. [PubMed: 8600462] <br />                                         37. Kataoka H, Yamamoto Y, Sekiguchi M. A new gene (alkB) Escherichia coli controls <br />                                             sensitivity methyl methane sulfonate. J Bacteriol. 1983; 153:1301&#8211;1307. [PubMed: 6337994] <br />                                         38. Falnes PO, Bjoras M, Aas PA, Sundheim O, Seeberg E. Substrate specificities bacterial and <br />                                             human AlkB proteins. Nucleic Acids Res. 2004; 32:3456&#8211;3461. [PubMed: 15229293] <br />                                         39. Sedgwick B, Robins P, Lindahl T. Direct removal alkylation damage DNA AlkB and <br />                                             related DNA dioxygenases. Methods Enzymol. 2006; 408:108&#8211;120. [PubMed: 16793366] <br />                                         40. Tsujikawa K, Koike K, Kitae K, Shinkawa A, Arima H, Suzuki T, Tsuchiya M, Makino Y, <br />                                             Furukawa T, Konishi N, Yamamoto H. Expression sub-cellular localization human ABH <br />                                             family molecules. J Cell Mol Med. 11:1105&#8211;1116. [PubMed: 17979886] <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                              Page 13 <br />  <br />  <br />                                         41. Singer, B.; Grunberger, D. Molecular Biology Mutagens Carcinogens. 1 ed.. New York: <br />                                             Plenum; 1983. p. 347 <br />                                         42. Yuan B, You C, Andersen N, Jiang Y, Moriya M, O&#8217;Connor T, Wang Y. The Roles DNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Polymerases &#954; &#953; Error-free Bypass N2-carboxyalkyl-dG Lesions Mammalian <br />                                             Cells. J Biol Chem. 2011:1&#8211;16. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                      Page 14 <br />  <br />  <br />                                                                                 Highlights <br />                                           &#9632;       Alkbh2 Alkbh3 Fe(II)/&#945;-ketoglutarate-dependent dioxygenases both <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   protect primary MEFs cytotoxic alkylation damage. <br />                                           &#9632;       Only Alkbh2 protects primary cells mutagenic alkylation damage. <br />                                           &#9632;       Alkbh2 Alkbh3 similar substrate specificity provide different <br />                                                   protection alkylation damage. <br />                                           &#9632;       Alkbh2 Alkbh3 protection spontaneous induced C&#8594;T and <br />                                                   C&#8594;A mutations methylmethane sulfonate induced A&#8594;T mutations. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         Maps targeted deletions mouse &#945;-ketoglutarate/Fe(II) dioxygenases Alkbh2 and <br />                                         Alkbh3 generated follows: (a) Alkbh2 site targeting <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-10'  ></span>Mus musculus</span> <br />                                         genomic sequence located mouse chromosome 5 (LOCUS NC_000071 NCBI <br />                                         Mouse Genome Sequencing Consortium). (b) Southern blots Alkbh2+/&#8722; <br />                                         heterozygotes using probe 5&#8217;arm. +/+ WT control +/&#8722; different clones <br />                                         identified heterozygotes initial PCR screening. (c) Southern blots Alkbh2+/&#8722; <br />                                         heterozygotes using probe 3&#8217;arm. +/+ WT control +/&#8722; different clones <br />                                         identified heterozygotes initial PCR screening. Clones used probe (b) (c) are <br />                                         identical. (d) Alkbh3 site targeting Mus <span id='am-40' about='obo:NCBITaxon_112137' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_301955 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-41' property="rdfs:label" content="Musculus" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>musculus</span> genomic sequence located on <br />                                         mouse chromosome 2 (LOCUS NC_000068 NCBI Mouse Genome <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                         Page 16 <br />  <br />  <br />                                         Sequencing Consortium). (e) Southern blots Alkbh3+/&#8722; heterozygotes using probe for <br />                                         5&#8217;arm. +/+ WT control +/&#8722; different clones identified heterozygotes <br />                                         initial PCR screening. (f) Southern blots Alkbh3+/&#8722; heterozygotes using probe for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         3&#8217;arm. +/+ WT control +/&#8722; different clones identified heterozygotes <br />                                         initial PCR screening. Clones used probe (e) (f) identical. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. <br />                                         PCR confirmation Alkbh2 Alkbh3 targeted deletions determined by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         amplification deleted inserted sequences Alkbh2 Alkbh3 gene, using PCR. (a) <br />                                         The Exon 4 region Alkbh2, exon 7 region Alkbh3 Neo gene contained the <br />                                         cassette replacing AlkBh2 AlkBh3 exons amplified genomic DNA extracted <br />                                         primary WT Alkbh2 Alkbh3 knock-out MEFs. The 199 bp band product <br />                                         Exon 4 functional Alkbh2. The 160 bp band product Exon 7 a <br />                                         functional Alkbh3. The 171 bp band neomycin PCR product cassette used <br />                                         targeted deletion Alkbh2 Alkbh3. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. <br />                                         Cell survival WT, Alkbh2&#8722;/&#8722;, Alkbh3&#8722;/&#8722; primary Big Blue MEFs determined 48 h <br />                                         post-treatment MMS, described Experimental Procedures. (a) Survival was <br />                                         monitored using Trypan Blue staining determine percentage intact cellular <br />                                         membranes culture. (b) IC50 values primary MEFs. All samples were <br />                                         analyzed triplicate curve fitting statistical analysis performed using Prism <br />                                         software. *** P equals 0.0003 **P equals 0.003, compared WT IC50. <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                 Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. <br />                                         MMS-induced mutant frequencies WT Alkbh2&#8722;/&#8722; Alkbh3&#8722;/&#8722; primary MEFs. <br />                                         Frequencies determined subtracting spontaneous mutant frequency, as <br />                                         determined G1250 cells infected &#955;LIZ phage containing packaged cII genomic <br />                                         DNA, isolated primary WT, Alkbh2, Alkbh3 knock-out MEFs, observed <br />                                         mutant frequency following 4 mM MMS treatment. *** P equals 0.0001. <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                 Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 5. <br />                                         Spontaneous 4 mM MMS-induced mutation spectrum map WT primary MEFs. 29 <br />                                         spontaneous 36 MMS-induced mutants sequenced. A &#8216;v&#8217; indicates insertion <br />                                         mutation. The exact number mutation type Table 2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />                             Nay et al.                                                                                   Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 6. <br />                                         Spontaneous 4 mM MMS-induced mutation spectrum map Alkbh2&#8722;/&#8722; primary <br />                                         MEFs. 54 spontaneous 44 MMS-induced mutants sequenced. A &#8216;v&#8217; indicates an <br />                                         insertion mutation. The exact number mutation type Table 2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           DNA Repair (Amst). Author manuscript; available PMC 2013 May 01. <br />    <br /> </body></html>