<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 18 October 2011                                  Nucleic Acids Research, 2012, Vol. 40, Database issue D687&#8211;D694 <br />                                                                                                               doi:10.1093/nar/gkr827 <br />  <br />  <br /> PhenoM: database morphological phenotypes <br /> caused mutation essential genes in <br /> <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> Ke Jin1,2, Jingjing Li1,2,3, Frederick S. Vizeacoumar4, Zhijian Li1,2,3, Renqiang Min1,5, <br /> Lee Zamparo1,5, Franco J. Vizeacoumar1,2, Alessandro Datti4,6, <span id='am-48' about='obo:NCBITaxon_139583' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30102 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_139578 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33365 obo:NCBITaxon_33342 obo:NCBITaxon_33368 obo:NCBITaxon_50557'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Brenda" datatype="xsd:string"></span>Brenda</span> Andrews1,2,3,*, <br /> Charles Boone1,2,3,* Zhaolei Zhang1,2,3,5,* <br /> 1 <br />  Donnelly Centre Cellular Biomolecular Research, University Toronto, 160 College Street, Toronto, <br /> ON, M5S 3E1, 2Banting Best Department Medical Research, University Toronto, 112 College Street, <br /> Toronto, ON, M5G 1L6, 3Department Molecular Genetics, University Toronto, 1 King&#8217;s College Circle, <br /> Toronto, ON, M5S 1A8, 4Samuel Lunenfeld Research Institute, Mount Sinai Hospital, 600 University Avenue, <br /> Toronto, ON, M5G 1X5, Canada, 5Department Experimental Medicine Biochemical Sciences, University <br /> of Perugia, Via del Giochetto, 06122, Perugia, Italy 6Department Experimental Medicine Biochemical <br /> Sciences, University Perugia, Perugia, Italy <br />  <br /> Received July 22, 2011; Revised September 16, 2011; Accepted September 19, 2011 <br />  <br />  <br />  <br /> ABSTRACT                                                                        enrichment analysis gene sets showing similar <br /> About one-fifth genes budding yeast                               morphological alterations. The current PhenoM <br /> are essential haploid viability                               version 1.0 contains 78 194 morphological images <br /> functionally assessed using standard genetic                                    1 909 914 cells covering subcellular compart- <br /> approaches gene deletion. To facilitate                                 ments structures 775 ts alleles spanning 491 <br /> genetic analysis essential genes,                              essential genes. PhenoM freely available http:// <br /> have assembled collections yeast strains ex-                                 phenom.ccbr.utoronto.ca/. <br /> pressing temperature-sensitive (ts) alleles essen- <br /> tial genes. To explore phenotypes caused                                 INTRODUCTION <br /> essential gene mutation used panel genet-                               Essential genes indispensable survival an <br /> ically engineered fluorescent markers explore                            organism typically involved fundamental bio- <br /> morphology cells ts strain collection using                           logical processes, cell wall membrane biogen- <br /> high-throughput microscopy. Here,                               esis, ribosome biosynthesis, DNA replication and <br /> design implementation online database,                                cytoskeletal functions (1). As thoroughly <br /> PhenoM (Phenomics yeast Mutants), storing,                               characterized model organisms, budding yeast <br /> retrieving, visualizing <span id='am-62' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> mining quantita-                           <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae frequently used study <br /> tive single-cell measurements extracted                                    genes involved conserved biological pathways. In <br /> micrographs ts mutant cells. PhenoM allows                               S. cerevisiae,  18% genes considered es- <br />                                                                                 sential growth rich medium glucose the <br /> users rapidly search retrieve raw images and <br />                                                                                 carbon source (1,2). Essential genes evolutionarily <br /> their quantified morphological data genes                                conserved yeast humans than <br /> interest. The database provides                                    non-essential genes, indicating deeper understanding <br /> data-mining tools, including PhenoBlast module                                functions yeast essential genes extend our <br /> for phenotypic comparison mutant strains                                understanding human gene function (3). Previous <br /> and Gene Ontology module functional                                       studies characterized functions yeast essential <br />  <br />  <br />  <br />  <br /> *To correspondence addressed. Tel: +1 416 978 8562; Fax: +1 416 978 8287; Email: brenda.andrews@utoronto.ca <br /> Correspondence addressed Charles Boone. Tel: + 416 946 7260; Fax: +1 416 978 8287; Email: charlie.boone@utoronto.ca <br /> Correspondence addressed Zhaolei Zhang. Tel: +416 946 0924; Fax: +1 416 978 8287; Email: zhaolei.zhang@utoronto.ca <br />  <br /> &#223; The Author(s) 2011. Published Oxford University Press. <br /> <span id='am-40' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44'  ></span>This</span> Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  D688 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br /> genes using variety alleles allow conditional                   To facilitate access valuable single-cell <span id='am-57' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Morpho" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>morpho</span>- <br /> partial inactivation essential gene function. Many                      logical data including raw images, implemented a <br /> mutant strains constructed genetic analysis                  web server integrated database capability <br /> of essential genes past years, including              allow visualization analysis morphological data. <br /> harbouring tetracycline (tet)-repressible promoter alleles                 PhenoM (Phenomics Yeast Mutants) web-based <br /> (1), decreased abundance mRNA perturbation alleles                      platform contains quantitative single-cell measure- <br /> (4) temperature-sensitive (ts) alleles (5,6). Among               ments morphological images yeast cells carrying <br /> methods, ts alleles considered powerful                   ts alleles essential genes generated HCS technology <br /> genetic analysis perturb speci&#65533;?c aspects es-                (10&#8211;13) [reviewed (9)]. Figure 1 shows detailed record <br /> sential gene function (5&#8211;8).                                               allele smc4-1, describes basic information of <br />    Temperature-sensitive alleles essential genes allow                  gene phenotypic images. We developed a <br /> cell viability permissive temperature                     user-friendly fast accurate search <br /> protein encoded ts allele resembles encoded                    engine embedded data mining tools, PhenoDev <br /> by normal wild-type allele mutant                           PhenoBlast. PhenoDev set tools calculating <br /> cells morphologically indistinguishable                     phenotypic differences mutant strains or <br /> the wild-type cells. However, restrictive (non-                     mutant wild-type strain grown the <br /> permissive) temperature, ts gene product defective                  different temperatures. These tools used <br /> and result loss-of-function. Consequently                     identify genes that, compromised, rise a <br /> cells growth morphological defects re-                 speci&#65533;?c abnormal phenotype interest; conversely, a <br /> strictive temperature. We recently developed                        speci&#65533;?c ts allele, tools quickly identify <br /> method quantify cell morphological changes thou-                    signi&#65533;?cant phenotypic changes associated <br /> sands mutants parallel coupling synthetic genetic                 temperature shift. In contrast, tool, <br /> analysis high-content screening (SGA&#8211;HCS) (9,10).                     PhenoBlast, serves quantify morphological similar- <br /> We used SGA&#8211;HCS study essential yeast genes                         ity multiple ts alleles. This tool best used to <br /> recently published phenotypic pro&#65533;?ling 497 yeast                      discover ts alleles cause abnormal phenotypes <br /> strains carrying ts conditional alleles essential genes (6).            similar ts allele interest. Our provides <br />  <br />  <br />  <br />  <br /> Figure 1. An example allele interface based allele smc4-1. The interface includes basic information allele phenotypic <br /> images allele different subcellular compartments structures. <br />                                                                    Nucleic Acids Research, 2012, Vol. 40, Database issue   D689 <br />  <br />  <br /> tools Gene Ontology (GO) enrichment analysis &#65533;?nd            website (18) customized according gene <br /> common GO annotation(s) top-ranked                       set database. <br /> mutants identi&#65533;?ed PhenoDev PhenoBlast. <br />    Currently, PhenoM version 1.0 holds 78 194 morpho-              DATA MINING <br /> logical images 1 909 914 cells, covering yeast <br /> subcellular compartments structures, including actin &#65533;?la-       The current PhenoM database contains data 775 ts <br /> ments, DNA damage foci, mitochondria, nucleus, plasma              mutants covering 491 yeast essential genes. For each <br /> membrane mitotic spindle. These data assembled            mutant strain, 865 morphological parameters were <br /> by studying strains carrying 775 different ts alleles spanning     catalogued database, corresponding different <br /> 491 essential genes. To knowledge, PhenoM &#65533;?rst          temperatures, cellular compartments cell cycle <br /> database phenotypes caused mutation yeast essen-          stages. For example, following parameters describing <br /> tial genes contains analyses quantitative data            morphology spindle quanti&#65533;?ed stored <br /> derived single-cell morphological images. PhenoM              database: area, perimeter, length orientation. <br /> should valuable resource yeast genomics           To aid biologists extracting biologically useful informa- <br /> communities large volume data ef&#65533;?cient        tion, designed implemented data mining tools, <br /> analysis tools facilitate study essential genes        PhenoDev PhenoBlast, summarize the <br /> function. Images available downloadable              phenotypic deviations genes isogenic cell popu- <br /> format facilitate future re-analysis using                 lations quantify morphological similarities among <br /> sophisticated software developed compu-          different mutants, respectively. <br /> tational experts.                                                  Phenotypic deviation: PhenoDev <br />                                                                    PhenoDev collection analytical tools that <br /> DATA SOURCE                                                        help users evaluate phenotypic deviations a <br />                                                                    given morphological parameter response condition <br /> The PhenoM database contains morphological images              changes. It consists modules: PhenoTempDev, <br /> quantitative single-cell measurements yeast strains             PhenoMutaDev PhenoCycleDev. We provide a <br /> carrying 775 different ts alleles associated 491 essen-       detailed user manual examples using tools <br /> tial genes, covering following subcellular compart-        PhenoM website; tools user-friendly inter- <br /> ments structures: actin &#65533;?laments, DNA damage foci,              faces extensively tested optimized. <br /> mitochondria, nucleus, plasma membrane mitotic                    The phenotypic temperature deviation (PhenoTemp <br /> spindle. As described previous report, MATa                 Dev) module help users identify ts strains have <br /> query strains carrying different &#64258;uorescent markers           signi&#65533;?cant changes speci&#65533;?c morphological parameter <br /> mated ts allele array MATa haploid ts strains           grown permissive restrictive temperature <br /> expressing different Green Fluorescent Protein (GFP) and/          (see Figure 2 example). For example, researcher <br /> or Red Fluorescent Protein (RFP) fusion proteins              interested asking ts allele affects &#8216;spindle <br /> isolated using SGA technology (6,10,14). The following             length&#8217;, use PhenoTempDev extract two <br /> cellular markers used: plasma membrane marker               sets measurements &#8216;spindle length&#8217; two <br /> (Psr1p-GFP), reporter DNA damage (Ddc2p-GFP),                 cell populations. A statistical test allows ranking of <br /> a nuclear marker (Mad1p-Nuclear localization Signal-               ts alleles according signi&#65533;?cance difference <br /> RFP), mitotic spindle reporter (GFP-Tub1), mitochon-           tested strains. In essence, tool help <br /> drial marker (OM45p-GFP) actin reporter                     researchers identify genes implicated particu- <br /> (Sac6p-GFP). Morphological images generated                lar yeast morphological trait. Phenotypic mutation devi- <br /> using SGA-HCS protocol (9,15) quantitative                 ation (PhenoMutaDev) intended quantify the <br /> measurements particular morphological feature            phenotypic deviation wild-type strain a <br /> extracted single-cell level. For reporter,    ts mutant strain selected set morphological par- <br /> independent images captured different tempera-             ameters temperature. In contrast to <br /> tures referred &#8216;sites&#8217; database.        PhenoTempDev, detects morphological changes <br /> The quantitative measurements cell                    caused temperature shift, PhenoMutaDev can <br /> categorized factors temperature          identify mutants signi&#65533;?cantly different pheno- <br /> which yeast strain grown, cell cycle stage      type wild-type cells. <br /> cell subcellular compartment structure                Finally, phenotypic cell cycle deviation (PhenoCycleDev) <br /> which feature extracted. Since yeast cells rela-      allows users examine phenotypic changes single <br /> tively simple ellipsoidal shape, dimensions cells       mutant cell cycle. As described above, we <br /> can serve proxy determine cell cycle phase (16). In        categorized cell cycle phases based on <br /> PhenoM, categorized cells phases            daughter bud mother cell ratios. Our queries <br /> the cell cycle according ratio daughter cell area     contiguous cell cycle transitions: (i) unbudded to <br /> mother cell area: (i) unbudded cell; (ii) small budded cell;       small budded cells (G1-S phase); (ii) small budded to <br /> (iii) medium budded cell; (iv) large budded cell               medium budded cells (S-G2); (iii) medium <br /> (detailed information provided PhenoM website).              budded large budded cells (G2-M). For cell cycle <br /> Gene annotations downloaded SGD (17),                    transition, measurements morphological parameters are <br /> while GO annotations downloaded GO                   extracted compared. The program ranks the <br />  D690 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 2. An example &#8216;PhenoTempDev Detail&#8217; interface. The &#65533;?ve alleles presented &#65533;?gure. <br />  <br />  <br />  <br />  <br /> mutants according changes morphological measure-                   colleagues developed software tool called PhenoBlast, in <br /> ments associated transitions. With tool,             analogy BLAST, gene pairs scored based <br /> for essential mutant, users readily determine              similarities morphological phenotypes caused by <br /> phenotypic features affected particular cell                knockdown gene expression RNAi. With tool, <br /> cycle phase transition.                                                  47 phenotypic parameters used assess morpho- <br />                                                                          logical phenotypes <span id='am-28' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> (19). We <br />                                                                          adopted PhenoBlast comparing yeast morphology <br /> PhenoBlast <br />                                                                          data. In algorithm, directly compare the <br /> The rationale using morphological data study gene                 morphology different mutant strains. <br /> function stems recent observation genes                    Instead, mutant strain, morphological meas- <br /> that annotated similar functions tend                     urements population mutant cells &#65533;?rst <br /> give rise similar morphological defects perturb-                 compared population wild-type cells. We <br /> ation (6). The huge volume morphology data generated                  used Wilcoxon rank-sum test determine whether <br /> from high-content screens catalogued PhenoM                   particular morphological parameter (e.g. spindle length) <br /> database offers unique useful resource search               signi&#65533;?cantly increased (labelled &#8216;+1&#8217;), unchanged <br /> alleles similar morphological phenotypes                   (labelled &#8216;0&#8217;) signi&#65533;?cantly decreased (labelled as <br /> thus similar cellular functions.                                    &#8216; 1&#8217;) mutant strain. In way, obtain a <br />    In contrast searching databases genes                vector 865 values consisting &#8216;+1&#8217;, &#8216;0&#8217; &#8216; 1&#8217; for <br /> sequence similarities, searching morphological                       mutant strain, describes overall morpho- <br /> similarities pair yeast strains computa-               logical change comparison wild-type cells. We can <br /> tionally algorithmically challenging. In                     quantify morphological similarities any <br /> earlier study phenotypic changes <span id='am-35' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span>Caenorhabditis</span>                    mutant strains calculating similarities between <br /> elegans RNAi interference experiments, Gunsalus                   corresponding <span id='am-53' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> (for detailed information, <br />                                                                  Nucleic Acids Research, 2012, Vol. 40, Database issue   D691 <br />  <br />  <br /> please PhenoM server). The algorithm described           mutants number objects single cell a <br /> above implemented optimized                      particular compartment; (iv) searching mutants a <br /> PhenoBlast. Users query single mutant          certain range particular cell shape parameter. All of <br /> and mutants similar phenotypic pro&#65533;?les         options uni&#65533;?ed Morphology Search. <br /> real time. Figure 3 shows example PhenoBlast <br /> comparison used default settings query              GO enrichment analysis <br /> smc4-1 strain. The search identi&#65533;?ed mcd1-73 strain,          Depending users&#8217; objective, running PhenoDev or <br /> which contains ts allele MCD1 gene           PhenoBlast automatically generates list mutants <br /> hit&#8212;the smc4-1 mcd1-73 strains shared similarity          showing phenotypic abnormality. We provide a <br /> 621 morphological parameters [122 parameters signi&#65533;?-             module GO enrichment analysis test whether <br /> cantly increased, 435 unchanged 64 signi&#65533;?cantly              genes associated phenotypic defects are <br /> decreased (Figure 4)].                                           enriched known functions pathways. This function- <br />                                                                  ality particularly useful gauging biological sig- <br /> Search                                                           ni&#65533;?cance highly ranked genes mutation <br />                                                                  causes extreme phenotypic defects. To implement GO <br /> The search engine provides options,            analysis, compared tools GoMiner <br /> Quick Search Morphology Search. In Quick Search,             (20), GOEAST (21) GO::TermFinder (22). We <br /> users directly query mutant         chose use GO::TermFinder package (v0.86) for <br /> database specifying gene name, open reading frame         PhenoM uses modular design was <br /> (ORF) name, aliases. A partial-word-match method           easy integrate system. <br /> also incorporated Quick Search. On hand, <br /> if users want &#65533;?nd mutant strains share morpho- <br /> logical similarity pre-speci&#65533;?ed gene, search         DATABASE IMPLEMENTATION AND USER <br /> morphology pro&#65533;?les performed. The Morphology              INTERFACE <br /> Search utility &#64258;exible allows users conduct <br />                                                                  Database implementation <br /> speci&#65533;?c advanced searches including: (i) searching for <br /> mutants certain cell cycle phase; (ii) searching        We designed implemented PhenoM using Struts- <br /> mutants grown certain temperature; (iii) searching          Spring-Hibernate,   lightweight enterprise-level <br />  <br />  <br />  <br />  <br /> Figure 3. An example &#8216;Search&#8217; interface PhenoBlast. <br />  D692 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 4. An example &#8216;PhenoBlast Detail&#8217; interface. The interface includes GO analysis result section &#8216;Search Result&#8217; section. Smc4-1 <br /> is used query allele 5 23&#8211;25 alleles presented &#65533;?gure. <br />  <br />  <br />  <br />  <br /> developmental framework (23,24). MySQL (v5.0.77)                        charts. All source code development performed in <br /> used underlying relational database store                     Eclipse v3.4.2 (http://www.eclipse.org). <br /> large volume quantitative measurements links <br /> to raw morphological images &#65533;?le system. At                   User interface <br /> front end, Apache Server 2.0 deployed handle user <br />                                                                             The PhenoM server provides following primary web <br /> requests forward requests Java servlets <br />                                                                             interfaces, allow users perform focused query <br /> and Java Server Pages Apache Tomcat 6.0. To <br />                                                                             analysis tasks. <br /> provide user-friendly interface, JavaScript AJAX <br />                                                                                The &#8216;Allele interface&#8217; presents detailed informa- <br /> (Asynchronous Javascript XML) technology were <br />                                                                             tion mutant strain. It consists parts: <br /> adopted program client-side functionality and <br /> Apache TilesTM used create reusable interface                          (i) Basic information cross links databases <br /> components. For online graphical visualization                                SGD (17) BioGRID (25); <br /> image manipulation, ImageJ (v1.44) incorporated                       (ii) Phenotypic images statistical information for <br /> assist users morphological image processing Java                         subcellular compartments struc- <br /> applets, Java open source package JFreeChart                              tures. Several functional links provided, <br /> (http://www.jfree.org/jfreechart/) used plot bar                           option download original <br />                                                                 Nucleic Acids Research, 2012, Vol. 40, Database issue          D693 <br />  <br />  <br />       micrograph TIFF format, viewing image               (P = 0.04455) (Figure 4). The results PhenoBlast <br />       ImageJ, link dataset        consistent known biology (6,29&#8211;31), and <br />       particular image. For ease analysis, images      believe useful tool suggesting <br />       annotated names correspond             new biological gene associations yeast research <br />       mutant name, reporter imaged, temperature         community. <br />       images captured site               Download: PhenoM allows users download either <br />       number (which indicates set independent            data set mutants single <br />       images used produce particular image           selected mutant tab-delimited format. <br />       mutant) and <br /> (iii) The PhenoBlast results list, containing &#65533;?fty <br />       mutants share similar morphological            AVAILABILITY AND FUTURE DIRECTION <br />       pro&#65533;?les (Figures 1 4).                                PhenoM freely accessed http://phenom.ccbr. <br />                                                                 utoronto.ca/. The future development PhenoM will <br />                                                                 include following aspects: (i) PhenoM include <br />    PhenoDev: series tools measure pheno-        phenotypic information additional yeast strains <br /> typic deviations strains carrying different ts alleles       carrying mutant alleles non-essential essential <br /> under varying conditions. We PhenoTempDev               genes data available groups or <br /> an example navigate interfaces function          elsewhere; (ii) data subcellular compartments or <br /> (Figure 2). Multiple options, morphological             structures incorporated database help <br /> features certain compartment, bud size                 biologists uncover association cell morph- <br /> others provided customized. An ana-              ology gene function; (iii) data mining and <br /> lysis default parameters lead              statistical analysis tools developed, especially tools <br /> PhenoTempDev results interface (Figure 2),                integrate types data protein <br /> includes sections: (i) results GO analysis       complexes protein interactions. <br /> (ii) deviation details 50 mutants listed         Our database ultimately aims provide centralized <br /> descending <span id='am-45' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>.                                               platform analysis gene function using quan- <br />    PhenoBlast: comparison tool phenotyp-          titative measurements morphological images from <br /> ic similarity accessed home          yeast mammalian cells. To accomplish goal, <br /> interface PhenoM navigation bar. To              PhenoM provide submission collect data <br /> execute PhenoBlast, query mutant                 image-based HCS experiments future. <br /> relevant morphological features speci&#65533;?ed. Upon <br /> typing characters query allele box, will <br /> automatically suggest list similar mutant         ACKNOWLEDGEMENTS <br /> names. To facilitate customized search, list 865       We like thank Xiaoli Wu helping design the <br /> morphological features divided parts: (i) 386     style interfaces Dr Yufang Zheng Fudan <br /> features phenotypic mutation deviation; (ii) 193           University critical readings early version the <br /> features phenotypic temperature deviation;             article. We wish thank Dr Judice L.Y. Koh her <br /> (iii) 286 features phenotypic cell cycle deviation         helpful suggestions. <br /> (detailed information provided PhenoM <br /> server). At feature chosen conduct <br /> the analysis. The PhenoBlast results interface dis-     FUNDING <br /> played following execution successful query and <br />                                                                 A Fudan University - University Toronto Exchange <br /> contains parts: (i) GO analysis results (ii) the <br />                                                                 Scholarship (in K.J.); postdoctoral fellowship <br /> details 50 mutants similar phenotypes. <br />                                                                 Best Foundation (to F.J.V.); Canadian <br /> Figures 3 4 shows example navigating through <br />                                                                 Institutes Health Research grants (GMX-201237 to <br /> PhenoBlast (Figures 3 4). The SMC4 gene encodes a <br />                                                                 B.A. C.B.) (GMX-211012 B.A., C.B. and <br /> member ubiquitous <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> chromosome-associated <br />                                                                 Z.Z.). Infrastructure automated imaging the <br /> ATPases (26). The Smc4 protein subunit the <br />                                                                 Boone Andrews Lab funded the <br /> condensin complex required chromosome <br />                                                                 Canadian Foundation Innovation (LEF-21475) and <br /> condensation dynamics (27,28). To search for <br />                                                                 Ontario Research Fund. Funding open access <br /> mutants sharing similar morphological pro&#65533;?les the <br />                                                                 charge: Canadian Institutes Health Research <br /> smc4-1 mutant strain, input smc4-1 query <br />                                                                 (GMX-201237) (GMX-211012). <br /> allele box Search interface PhenoBlast (Figure 3) <br /> and press button &#8216;Run PhenoBlast&#8217; using default        Con&#64258;ict statement. None declared. <br /> parameters. The automatically leads Detail <br /> interface PhenoBlast. In search result section, the <br /> &#65533;?rst, &#65533;?fth fourth ranked mutants, mcd1-73,           REFERENCES <br /> smc1-2 smc3-1, respectively, carry mutant alleles <br />                                                                  1. Mnaimneh,S., Davierwala,A.P., Haynes,J., Moffat,J., Peng,W.T., <br /> in genes encoding condensin subunits list                Zhang,W., Yang,X., Pootoolal,J., Chua,G., Lopez,A. et al. (2004) <br /> mutants overall enriched DNA metabolic                   Exploration essential gene functions titratable promoter <br /> process (P = 0.01218) sister chromatid cohesion                 alleles. Cell, 118, 31&#8211;44. <br />  D694 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  2. Giaever,G., Chu,A.M., Ni,L., Connelly,C., Riles,L., Veronneau,S.,       17. Hong,E.L., Balakrishnan,R., Dong,Q., Christie,K.R., Park,J., <br />     Dow,S., Lucau-Danila,A., Anderson,K., Andre,B. et al. (2002)                Binkley,G., Costanzo,M.C., Dwight,S.S., Engel,S.R., Fisk,D.G. <br />     Functional pro&#65533;?ling Saccharomyces cerevisiae genome.                 et al. (2008) Gene Ontology annotations SGD: new data <br />     Nature, 418, 387&#8211;391.                                                       sources annotation methods. Nucleic Acids Res., 36, <br />  3. Hughes,T.R. (2002) Yeast drug discovery. Funct. Integr.                 D577&#8211;D581. <br />     Genomics, 2, 199&#8211;211.                                                   18. Ashburner,M., Ball,C.A., Blake,J.A., Botstein,D., Butler,H., <br />  4. Schuldiner,M., Collins,S.R., Thompson,N.J., Denic,V.,                       Cherry,J.M., Davis,A.P., Dolinski,K., Dwight,S.S., Eppig,J.T. <br />     Bhamidipati,A., Punna,T., Ihmels,J., Andrews,B., Boone,C.,                  et al.; The Gene Ontology Consortium (2000) Gene ontology: <br />     Greenblatt,J.F. et al. (2005) Exploration function               tool uni&#65533;?cation biology. Nat. Genet., 25, 25&#8211;29. <br />     organization yeast early secretory pathway            19. Gunsalus,K.C., Yueh,W.C., MacMenamin,P. Piano,F. (2004) <br />     epistatic miniarray pro&#65533;?le. Cell, 123, 507&#8211;519.                             RNAiDB PhenoBlast: web tools genome-wide phenotypic <br />  5. Ben-Aroya,S., Coombes,C., Kwok,T., O&#8217;Donnell,K.A., Boeke,J.D.               mapping projects. Nucleic Acids Res., 32, D406&#8211;D410. <br />     Hieter,P. (2008) Toward comprehensive                             20. Zeeberg,B.R., Feng,W., Wang,G., Wang,M.D., Fojo,A.T., <br />     temperature-sensitive mutant repository essential genes           Sunshine,M., Narasimhan,S., Kane,D.W., Reinhold,W.C., <br />     Saccharomyces cerevisiae. Mol. Cell, 30, 248&#8211;258.                           Lababidi,S. et al. (2003) GoMiner: resource biological <br />  6. Li,Z., Vizeacoumar,F.J., Bahr,S., Li,J., Warringer,J.,                      interpretation genomic proteomic data. Genome Biol., 4, <br />     Vizeacoumar,F.S., Min,R., Vandersluis,B., Bellay,J., Devit,M.               R28. <br />     et al. (2011) Systematic exploration essential yeast gene            21. Zheng,Q. Wang,X.J. (2008) GOEAST: web-based software <br />     function temperature-sensitive mutants. Nat. Biotechnol., 29,          toolkit Gene Ontology enrichment analysis. Nucleic Acids <br />     361&#8211;367.                                                                    Res., 36, W358&#8211;W363. <br />  7. Hartwell,L.H., Culotti,J. Reid,B. (1970) Genetic control         22. Boyle,E.I., Weng,S., Gollub,J., Jin,H., Botstein,D., Cherry,J.M. <br />     cell-division cycle yeast. I. Detection mutants. Proc. Natl       Sherlock,G. (2004) GO::TermFinder&#8211;open source software for <br />                                                                                 accessing Gene Ontology information &#65533;?nding signi&#65533;?cantly <br />     Acad. Sci. USA, 66, 352&#8211;359. <br />                                                                                 enriched Gene Ontology terms associated list genes. <br />  8. Simchen,G. (1978) Cell cycle mutants. Ann. Rev. Genet., 12, <br />                                                                                 Bioinformatics, 20, 3710&#8211;3715. <br />     161&#8211;191. <br />                                                                             23. Sun,L.V., Jin,K., Liu,Y., Yang,W., Xie,X., Ye,L., Wang,L., <br />  9. Vizeacoumar,F.J., Chong,Y., Boone,C. Andrews,B.J. (2009) <br />                                                                                 Zhu,L., Ding,S., Su,Y. et al. (2008) PBmice: integrated <br />     A picture worth thousand words: genomics phenomics in <br />                                                                                 database piggyBac (PB) insertional mutations their <br />     yeast Saccharomyces cerevisiae. FEBS Lett., 583, 1656&#8211;1661.             characterizations mice. Nucleic Acids Res., 36, D729&#8211;D734. <br /> 10. Vizeacoumar,F.J., van Dyk,N.S., Vizeacoumar,F., Cheung,V.,              24. Yang,W., Jin,K., Xie,X., Li,D., Yang,J., Wang,L., Gu,N., <br />     Li,J., Sydorskyy,Y., Case,N., Li,Z., Datti,A., Nislow,C. et al.             Zhong,Y. Sun,L.V. (2009) Development database system <br />     (2010) Integrating high-throughput genetic interaction mapping              mapping insertional mutations mouse genome with <br />     high-content screening explore yeast spindle                         large-scale experimental data. BMC Genomics, 10(Suppl. 3), S7. <br />     morphogenesis. J. Cell. Biol., 188, 69&#8211;81.                              25. Stark,C., Breitkreutz,B.J., Chatr-Aryamontri,A., Boucher,L., <br /> 11. Bakal,C., Aach,J., Church,G. Perrimon,N. (2007)                         Oughtred,R., Livstone,M.S., Nixon,J., Van Auken,K., Wang,X., <br />     Quantitative morphological signatures de&#65533;?ne local signaling                 Shi,X. et al. <br /> </body></html>