<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>&#214;sterlund et al. BMC Systems Biology 2013, 7:36 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                  Open Access <br />  <br /> Mapping condition-dependent regulation of <br /> metabolism yeast genome-scale <br /> modeling <br /> Tobias &#214;sterlund1, Intawat Nookaew1, <span id='am-1' about='obo:NCBITaxon_338213' typeof='owl:Thing obo:NCBITaxon_177244 obo:NCBITaxon_177236 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_116702 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Sergio" datatype="xsd:string"></span><span id='am-5'  ></span>Sergio</span> Bordel1 Jens Nielsen1,2* <br />  <br />  <br />   Abstract <br />   Background: The genome-scale metabolic model <span id='am-33' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-39'  ></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, presented 2003, first <br />   genome-scale network reconstruction eukaryotic organism. Since continuous efforts in <br />   <span id='am-96' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-97' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> improve expand yeast metabolic network. <br />   Results: Here present iTO977, comprehensive genome-scale metabolic model contains reactions, <br />   metabolites genes previous models. The model constructed based earlier reconstructions, <br />   iIN800 consensus network, improved expanded using gap-filling methods by <br />   introducing new reactions pathways based studies literature databases. The model shown <br />   perform growth simulations different media gene essentiality analysis single double <br />   knock-outs. Further, model used scaffold integrating transcriptomics, flux <span id='am-114' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> four <br />   different conditions order identify transcriptionally controlled reactions, i.e. reactions change flux <br />   transcription compared conditions. <br />   Conclusion: We present new yeast model represents comprehensive up-to-date collection knowledge <br />   yeast metabolism. The model used simulating yeast metabolism different growth <br />   conditions experimental data conditions integrated model. The model together <br />   experimental data useful tool identify condition-dependent changes metabolism different <br />   environmental conditions. <br />   Keywords: <span id='am-50' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, Genome-scale metabolic model, Integrated analysis, Transcriptionally <br />   controlled reactions <br />  <br />  <br /> Background                                                                             applications, reviewed lately [2]. Yeast <br /> The number metabolic network reconstructions                                    example used extensively biotechnology as <br /> microorganisms increased rapidly past                                   workhorse production wide range chemicals, <br /> decade following genome revolution rapid                                   metabolic modeling helped cases guide <br /> increase genome sequencing projects. The yeast                                      strain construction finding strategies improved <br /> Saccharomyces cerevisiae eukaryote                               chemical production [10]. Recently S. cerevisiae <br /> its metabolic network reconstructed metabolism                                 metabolic network used basis for <br /> is well-studied. During 10 years yeast                               constructing metabolic models yeasts, e.g. <br /> metabolic network updated                                 <span id='am-57' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61'  ></span>Schizosaccharomyces</span> pombe [11], <span id='am-12' about='obo:NCBITaxon_4951' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_34353 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Yarrowia" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16'  ></span>Yarrowia</span> lipolytica [12], <br /> reconstructed leading genome-scale models                                   <span id='am-26' about='obo:NCBITaxon_4919' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_1156497 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Pichia" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Hansenula" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:34359" datatype="xsd:string"></span>Pichia</span> pastoris Pichia stipitis [13]. <br /> [1-9]. Yeast genome-scale modeling different                                   The genome-scale metabolic model S. <br />                                                                                        cerevisiae presented 2003 [1] less <br /> * Correspondence: nielsenj@chalmers.se                                                 11 genome-scale reconstructions released <br /> 1 <br />  Novo Nordisk Foundation Center Biosustainability, Chalmers University          [2]. To overcome problem different models <br /> Technology, Gothenburg SE412 96, Sweden <br /> 2 <br />  Novo Nordisk Foundation Center Biosustainability, Technical University of <br />                                                                                        having different metabolite names, different scopes and <br /> Denmark, H&#248;rsholm DK2870, Denmark                                                      representing parts metabolism slightly different <br />                                          &#169; 2013 &#214;sterlund et al.; licensee BioMed Central Ltd. <span id='am-88' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-89' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92'  ></span>This</span> Open Access article distributed terms Creative <br />                                          Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                          reproduction medium, provided original work properly cited. <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                   Page 2 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> ways consensus metabolic network constructed                               Genome-scale metabolic model reconstructions can <br /> the result community jamboree effort [3]. The                               used scaffold integrated data analysis, <br /> consensus network introduced nomenclature standard                         network structure model basis <br /> metabolites following MIRIAM, using ChEBI annotations                            integration omics data order gain more <br /> and InChI codes. The consensus network viewed                          mechanistic knowledge cell&#8217;s behavior different <br /> a genome-scale reconstruction (GENRE) yeast                               environments different conditions. The reporter <br /> metabolism, genome-scale model (GEM) ready                          metabolite algorithm [14] useful tool integration of <br /> for simulations using e.g. constraint-based methods                      transcriptomics data metabolic network. The <br /> flux balance analysis (FBA), missing information                          algorithm identifies key metabolites neighboring <br /> about reversibility reactions missing biomass                        genes metabolic network transcriptionally <br /> equations. The consensus network developed                           changed conditions. However, regulation <br /> and improved updated version called Yeast 4 [4]                       cellular response different perturbations as <br /> a GEM ready simulations. It expanded                         gene deletion changes environmental conditions may <br /> updated leading version called Yeast 5 [5]. The                       occur different levels cell. By using the <br /> relationship yeast genome-scale models                            genome-scale model predict changes metabolic flux <br /> shown pedigree Figure 1A. The model                         different conditions correlate changes <br /> included detailed description lipid metabolism,                         change expression genes involved, reac- <br /> namely iIN800 [8], included consensus                             tions directly controlled transcription be <br /> reconstruction.                                                                  identified. The random sampling algorithm presented by <br />    A genome-scale metabolic model thought 1)                           Bordel et al. [15] framework identification this <br /> being biochemically genetically structured database                        kind trancriptionally controlled reactions using the <br /> of metabolism linking enzymes, metabolites reac-                         genome-scale metabolic model. <br /> tions together, 2) predictive model estimate                         Here present updated genome-scale model (GEM) <br /> the fluxes intracellular reactions predicting                      based consensus network iIN800 <br /> the growth cell constraining extracellular                       model. The goal model provide up-to-date <br /> (measured) fluxes represent growth conditions. The                        comprehensive description yeast metabolism, so <br /> first corresponds GENRE second                          new model contains genes, reactions and <br /> corresponds GEM concept. The genome-scale                                 metabolites previous models. We the <br /> metabolic model applied silico hypothesis                        model performs simulating growth under <br /> testing tool gain insight operations                              different environmental conditions. The model was <br /> metabolism, leading identification targets superior                    used tool integrating large of <br /> strain development.                                                              omics data different conditions. It used <br />  <br />  <br />                              A                      iFF708                                      B <br />                                                                                                               iTO980 <br />  <br />  <br />                                                    iND750                 iLL672 <br />                                                                                                                104 <br />                                                                                             Number genes <br />  <br />  <br />  <br />  <br />                                  iIN800            iMM904 <br />                                                                                                                876 <br />  <br />                                                   Yeast 1.0 <br />                                              (Consensus network) <br />                                                                                                                25 <br />  <br />                                                    Yeast4                Yeast 5 <br />                                                                                                               Yeast 5 <br />                                                              Model expansion <br />                                  iTO980                      Model reduction/modification <br />  <br />  <br />  Figure 1 The reconstruction process iTO977 comparison Yeast 5. (A) Yeast genome-scale metabolic model pedigree. The <br />  iTO977 model (red) reconstructed iIN800 model consensus network (blue) starting points. The starting point <br />  reconstructions merged expanded adding new pathways reactions databases literature. (B) A comparison of <br />  genes included iTO977 Yeast 5. <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                               Page 3 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> together random sampling algorithm correlate                        model compartments. The compartments iTO977 are <br /> the changes fluxes aerobic anaerobic                             Cytoplasm, Mitochondria, Peroxisome Extracellu- <br /> conditions carbon limited nitrogen limited                       lar. By keeping number compartments low the <br /> conditions changes gene expression order                          complexity model reduced simulation <br /> identify condition-dependent transcriptionally controlled                       capabilities improved. Some reactions in <br /> reactions.                                                                      consensus network takes place com- <br />                                                                                 partments, ER nucleus, included in <br /> Results discussion                                                          iTO977 model localized cytoplasm, <br /> In order construct comprehensive genome-scale                              reactions compartments out <br /> metabolic model, consensus network merged                               scope model discarded. The com- <br /> with iIN800 model construct draft starting point                       partments included iTO977 compartments <br /> reconstruction. The model improved adding                           biggest confidence terms localization of <br /> several new pathways, new reactions new metabolites                         proteins Saccharomyces Genome Database <br /> to model based database searches literature evi-                     (SGD, www.yeastgenome.org). <br /> dence. The resulting model includes 977 metabolic genes                            When comparing number metabolites reac- <br /> and called iTO977. Figure 1A shows iTO977                            tions different models important in <br /> model created. The model available SBML                           mind metabolite present times <br /> Excel format (Additional files 1 2                          model, different compartments. In genome- <br /> BioMet Toolbox [16], http://www.sysbio.se/biomet )                        scale metabolic models metabolite considered <br /> list newly added reactions pathways available                         different <span id='am-93' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-94' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> localized different <br /> in Additional file 3.                                                           compartments. When measuring levels intracellular <br />                                                                                 metabolites metabolomics impossible distinguish <br /> iTO977 model properties                                                         metabolites different compartments, e.g. pyruvate the <br /> We present updated yeast model bigger                             cytoplasm pyruvate mitochondria considered <br /> scope earlier models; contains genes,                              species metabolome experiment. Similarly <br /> metabolites reactions previous models. Table 1                         reaction taking place different compartments will <br /> shows statistical summary current model                            counted twice model. Since yeast 5 model <br /> some previous models terms number metabo-                              compartments iTO977 a <br /> lites, genes reactions. The iTO977 model 4                              higher number metabolites reactions taking <br /> compartments included model contrast                           compartmentalization account. However, iTO977 <br /> consensus network Yeast 5 15                                higher number unique metabolites unique <br />  <br />  <br /> Table 1 Summary characteristics different yeast genome-scale metabolic models <br />                                     iFF708           iIN800           iMM904              Consensus             Yeast5           iTO977 (Current model) <br /> Metabolites                         825              985              1228                1168                  1768             1353 <br /> -Unique metabolites                 595              683              713                 664                   779              815 <br /> -Cytosolic metabolites              518              645              634                 590                   579              738 <br /> -Mitochondrial mets                 170              209              241                 235                   308              231 <br /> -Peroxisomal mets                   0                0                80                  80                    75               87 <br /> -Extracellular metabolites          137              131              164                 158                   168              156 <br /> Reactions                           1145             1706             1577                1761                  2034             1566 <br /> -Unique reactions                   780              1041             1053                804                   1068             1207 <br /> -Cytosolic reactions                723              1301             710                 835                   1092             1083 <br /> -Mitochondrial reactions            104              242              306                 330                   334              256 <br /> -Peroxisomal reactions              0                0                109                 121                   111              80 <br /> Genes                               708              800              904                 888                   898              977 <br /> -Percentage1                        10.7%            12.1%            13.7%               13.4%                 13.6%            14.8% <br /> -Cytosolic genes                    532              639              736                 676                   681              800 <br /> -Mitochondrial genes                104              124              199                 190                   204              212 <br /> -Peroxisomal genes                  0                0                19                  24                    23               22 <br /> 1 <br />   Part genome included model. Based updated statistics Saccharomyces Genome Database includes 6607 genes. <br /> (as Aug 29, 2010). <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                 Page 4 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> reactions previous models, i.e. taking        percent wild type growth rate, the <br /> compartmentalization account counting            mutation flagged lethal. Simulations growth <br /> number metabolites reactions.                           phenotypes carbon sources ethanol <br />   The number genes included iTO977 model          galactose using different viability cut-offs are <br /> larger previous models. Figure 1B shows            presented Additional file 6. <br /> comparison iTO977 model Yeast 5 model <br /> in terms number open reading frames. A list       Integrated analysis <br /> ORFs included iTO977 model Yeast 5           The iTO977 model used scaffold integrated <br /> is presented Additional file 4. The 25 ORFs        data analysis. Chemostat cultivations performed at <br /> in yeast 5 iTO977 considered         dilution rate 0.05 h-1 different <br /> out scope iTO977 model, mainly        conditions used simulations (carbon limited, <br /> transporters e.g. Golgi membrane, vacuole         nitrogen limited, aerobic anaerobic) microarray <br /> membrane organelles included         data external intracellular flux measure- <br /> the iTO977 model. Among 104 ORFs                  ments obtained condition [20]. We were <br /> included iTO977, Yeast 5,           particularly interested changes occur the <br /> belong pathways newly introduced        transcript level flux level, i.e. correlations <br /> iTO977 model. These pathways represent                 changes transcriptome fluxome under <br /> biosynthesis lipid-linked oligosaccharides [17]         different conditions. The enzymes direct <br /> Glycosylphosphaditylinositol [18], molecules          link change expression change flux <br /> are used protein modifications. These pathways        good targets over-expression order to <br /> provide direct link iTO977 model         direct flux i.e. desired product, there <br /> recent model protein secretion machinery yeast       coupling expression flux. We therefore <br /> [19] step connecting metabolic            used random sampling algorithm [15] identify <br /> model model protein secretion.                      reactions change flux transcription. These <br />                                                                reactions referred transcriptionally controlled <br /> Model validation                                               reactions seen subset global set of <br /> The predictive power model evaluated         reactions change transcriptionally. The random sam- <br /> ways: 1) comparing simulated growth rate             pling algorithm calculates probability score reac- <br /> experimental growth rate different conditions,       tion change flux conditions using <br /> 2) comparing predicted viability silico           measured fluxes additional constraints model to <br /> single double gene knock-outs experimental        able simulate different conditions. The extra- <br /> observations single double mutant growth.               cellular fluxes used constrain condition-specific <br />    Figure 2 shows results growth rate             models intracellular flux distributions <br /> simulations. Experimental measurements specific        predicted condition-specific models are <br /> growth rate, specific substrate consumption rate           presented Additional file 7. The probability score the <br /> specific product formation rate chemostat cultures    reaction changing flux compared probability <br /> four different conditions taken previous         score transcriptional change involved genes, <br /> study [20] compared simulated data using FBA            calculated microarray data. The transcriptionally <br /> (for information, materials methods           controlled reactions changing carbon limited to <br /> Additional file 5). The conditions consid-      nitrogen limited conditions changing aerobic an- <br /> ered carbon limited aerobic growth, nitrogen limited      aerobic conditions shown heatmap Figure 4. <br /> aerobic growth, carbon limited anaerobic growth            These reactions probability score calculated the <br /> nitrogen limited anaerobic growth. The model shown         random sampling algorithm higher 0.9, indi- <br /> to perform predicting growth rate     cates flux transcript level reactions <br /> four conditions.                                               change direction. Red color corresponds <br />    In order evaluate model&#8217;s capability predicting   up-regulation flux transcription, <br /> gene essentiality performed simulations growth        blue corresponds down-regulation the <br /> single double gene knock-outs. The viability        flux level transcriptional level. <br /> mutants compared experimental observations from <br /> literature [6,21,22] SGD. Figure 3 shows          Transcriptionally controlled reactions glycolysis and <br /> accuracy predictions growth glucose           ergosterol biosynthesis <br /> minimal media (single gene knock-outs) rich YPD            As expected observe big differences glycolytic <br /> media. A knock-out mutant considered viable       activity anaerobic aerobic chemostat <br /> the simulated growth rate mutant 10          growth carbon limited nitrogen limited <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                                  Page 5 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br />                                                      0.5 <br />  <br />                                                     0.45             N&#8722;limited aerobic <br />                                                                      N&#8722;limited anaerobic <br />                                                      0.4 <br />                                                                      C&#8722;limited aerobic <br />                                                                      C&#8722;limited anaerobic <br />                               vivo growth rate   0.35 <br />  <br />                                                      0.3 <br />  <br />                                                     0.25 <br />  <br />                                                      0.2 <br />  <br />                                                     0.15 <br />  <br />                                                      0.1 <br />  <br />                                                     0.05 <br />  <br />                                                       0 <br />                                                            0           0.1              0.2             0.3              0.4         0.5 <br />                                                                                        silico growth rate <br />  Figure 2 Model validation comparing silico prediction specific growth rate experimental data. Growth phenotypes <br />  collected literature compared simulated values chemostat cultivations different conditions, nitrogen limited aerobic <br />  (green) anaerobic (red), carbon limited aerobic (blue) anaerobic (white). <br />  <br />  <br />  <br /> growth. There reactions considered                                        chemostat experiments cells controlled grow <br /> be transcriptionally controlled comparisons                                         specific growth rate conditions. The <br /> (Figure 4), phosphofructokinase, phosphoglycerase                                        cells consume glucose anaerobic <br /> mutase pyruvate kinase. These reactions                                             conditions aerobic conditions order grow <br /> glycolytic reactions, reactions                                           rate, anaerobic growth efficient <br /> in glycolysis considered transcriptionally                                   energetically. Similarly, glycolytic activtity higher <br /> controlled, i.e. change sense flux                                        nitrogen limited (glucose excess) conditions <br /> expression (Figure 5A). The flux reactions                                         compared carbon limited conditions, is <br /> the expression involved enzymes higher                                         glucose available, cell uses glucose to <br /> anaerobic conditions aerobic conditions. In                                            able produce biomass ethanol. The <br />  <br />  <br />                        A                                                     B                                           C <br />  <br />                                                                                                                                69% <br />                           82% <br />                                                                                  92% <br />                                                                                                                                            5% <br />                                                                15% <br />                                                                                                                                      23% <br />  <br />  <br />  <br />  <br />                                                                      TP, predicted growth, experimentally growth <br />                                                                      TN, predicted growth, experimentally growth <br />                                                                      FP, predicted growth, experimentally growth <br />                                                                      FN, predicted growth, experimentally growth <br />  <br />  Figure 3 Model validation comparing simulation single double gene knock-out mutant growth experimental <br />  observations. (A) Single gene knock-out mutants cultivated minimal media. (B) Single gene knock-out mutants cultivated rich YPD media. <br />  (C) Double gene knock-out mutants cultivated rich YPD media. TP = true positive, TN = true negative, FP = false positive and <br />  FN = false negative. <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                           Page 6 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br />                                                                   )  ) <br />                                                                 im im <br />                                                      c)       Nl Cl <br />                                                  o bi ic) ic ( ic ( <br />                                                er ob ob ob <br />                                             na er er er <br />                                           (A (A c-A c-A <br />                                         m     m        i <br />                                      li     li ob b <br />                                    -N -N er <span id='am-104' about='obo:NCBITaxon_1112515' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_336669 obo:NCBITaxon_6854 obo:NCBITaxon_336668 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Ero" datatype="xsd:string"></span>ero</span> <br />                     Rxn id     l im lim na na Reaction (subsystem) <br />                               C C          A A <br />                     ELO2_4                 Elongation fatty acids protein 2 (Fatty acid biosynthesis) <br />                     LYS12                  Homoisocitrate dehydrogenase, mitochondrial (Lysine metabolism) <br />                     FPS1                   Glycerol uptake/efflux facilitator protein (Transport, extracellular) <br />                     IPT1_1                 Inositolphosphotransferase 1 (Sphingoglycolipid metabolism) <br />                     BAT2_1                 Branched-chain-amino-acid aminotransferase, cytosolic (Branched chain amino acid metabolism) <br />                     ADY2                   Acetate transporter (Transport, extracellular) <br />                     ARO9_1                 Aromatic amino acid aminotransferase 2 (Aromatic amino acid biosynthesis) <br />                     LYS20_2                Homocitrate synthase, cytosolic isozyme (Pyruvate metabolism) <br />                     LEU4                   2-isopropylmalate synthase (Branched chain amino acid metabolism) <br />                     INO1                   Inositol-3-phosphate synthase (Phospholipid biosynthesis) <br />                     MEP1                   Ammonium transporter MEP1 (Transport, extracellular) <br />                     ADE8                   Phosphoribosylglycinamide formyltransferase (Purine metabolism) <br />                     ADE1                   Phosphoribosylaminoimidazole-succinocarboxamide synthase (Purine metabolism) <br />                     ADE13_1                Adenylosuccinate lyase (Purine metabolism) <br />                     ADE4                   Amidophosphoribosyltransferase (Purine metabolism) <br />                     URA2_1                 Aspartate carbamoyltransferase (Pyrimidine metabolism) <br />                     ILV5_1                 Ketol-acid reductoisomerase, mitochondrial (Branched chain amino acid metabolism) <br />                     ILV3_1                 Dihydroxy-acid dehydratase, mitochondrial (Branched chain amino acid metabolism) <br />                     ADE2                   Phosphoribosylaminoimidazole carboxylase (Purine metabolism) <br />                     URA10                  Orotate phosphoribosyltransferase 2 (Pyrimidine metabolism) <br />                     TAL1                   Transaldolase (Pentose phosphate pathway) <br />                     TKL1_1                 Transketolase 1 (Pentose phosphate pathway) <br />                     GND1                   6-phosphogluconate dehydrogenase, decarboxylating 1 (Pentose phosphate pathway) <br />                     SOL1                   Probable 6-phosphogluconolactonase 1 (Pentose phosphate pathway) <br />                     JEN1_2                 Carboxylic acid transporter protein homolog (Transport, extracellular) <br />                     SUR2                   Sphingolipid C4-hydroxylase SUR2 (Sphingoglycolipid metabolism) <br />                     ERG2                   C-8 sterol isomerase (Sterol biosynthesis) <br />                     LEU2                   3-isopropylmalate dehydrogenase (Branched chain amino acid metabolism) <br />                     ERG7                   Lanosterol synthase (Sterol biosynthesis) <br />                     ERG6                   Sterol 24-C-methyltransferase (Sterol biosynthesis) <br />                     ERG27_1                3-keto-steroid reductase (Sterol biosynthesis) <br />                     PGI1_2                 Glucose-6-phosphate isomerase (Glycolysis / Gluconeogenesis) <br />                     LAC1_1                 Sphingosine N-acyltransferase LAC1 (Sphingoglycolipid metabolism) <br />                     ADH1                   Alcohol dehydrogenase 1 (Pyruvate metabolism) <br />                     Uelo_4                 Model-specific reaction, involved elongation fatty acids (Fatty acid biosynthesis) <br />                     CSG2_1                 Mannosyl phosphorylinositol ceramide synthase (Sphingoglycolipid metabolism) <br />                     LCB1                   Serine palmitoyltransferase 1 (Sphingoglycolipid metabolism) <br />                     TRP2_1                 Anthranilate synthase component 1 (Aromatic amino acid biosynthesis) <br />                     ARO2                   Chorismate synthase (Aromatic amino acid biosynthesis) <br />                     ARO1_1                 3-dehydroquinate synthase (Aromatic amino acid biosynthesis) <br />                     RNR1_3                 Ribonucleoside-diphosphate reductase large chain 1 (Salvage pathways) <br />                     BNA1                   3-hydroxyanthranilate 3,4-dioxygenase (Aromatic amino acid biosynthesis) <br />                     GLK1_1                 Glucokinase GLK1 (Glycolysis / Gluconeogenesis) <br />                     HOR2                   (DL)-glycerol-3-phosphatase 2 (Aminosugars metabolism) <br />                     PHO84                  Inorganic phosphate transporter (Transport, extracellular) <br />                     MVD1                   Diphosphomevalonate decarboxylase (Sterol biosynthesis) <br />                     TGL5                   Lipase 5 (Glycerol metabolism (glycerolipid metabolism)) <br />                     PMI40                  Mannose-6-phosphate isomerase (Fructose mannose metabolism) <br />                     ERG5                   Cytochrome P450 61 (Sterol biosynthesis) <br />                     ERG25_1                C-4 methylsterol oxidase (Sterol biosynthesis) <br />                     ERG4                   <span id='am-109' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Delta</span>(24(24(1)))-sterol reductase (Sterol biosynthesis) <br />                     DGA1                   Diacylglycerol O-acyltransferase 1 (Glycerol metabolism (glycerolipid metabolism)) <br />                     GPM1_2                 Phosphoglycerate mutase 1 (Glycolysis / Gluconeogenesis) <br />                     CDC19                  Pyruvate kinase 1 (Glycolysis / Gluconeogenesis) <br />                     PFK1_1                 Phosphofructokinase 1 (Glycolysis / Gluconeogenesis) <br />                     ERG26_1                Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating (Sterol biosynthesis) <br />                     ERG1                   Squalene monooxygenase (Sterol biosynthesis) <br />                     PMT1                   Dolichyl-phosphate-mannose--protein mannosyltransferase 1 (Glycoprotein metabolism) <br />                     ERG20_1                Dimethylallyltranstransferase (Sterol biosynthesis) <br />                     PDC1                   Pyruvate decarboxylase isozyme 1 (Pyruvate metabolism) <br />                     ERG3                   C-5 sterol desaturase (Sterol biosynthesis) <br />                     ERG11                  Cytochrome P450 51 (Sterol biosynthesis) <br />                     PGK1                   Phosphoglycerate kinase (Glycolysis / Gluconeogenesis) <br />                     ENO1                   Enolase 1 (Glycolysis / Gluconeogenesis) <br />                     ERG24                  C-14 sterol reductase (Sterol biosynthesis) <br />                     MET22                  3&#8217;(2&#8217;),5&#8217;-bisphosphate nucleotidase (Cysteine metabolism) <br />                     HIS5                   Histidinol-phosphate aminotransferase (Histidine metabolism) <br />                     GLT1                   Glutamate synthase [NADH] (Glutamate metabolism) <br />                     ECM17                  Sulfite reductase [NADPH] subunit <span id='am-6' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Beta" datatype="xsd:string"></span>beta</span> (Cysteine metabolism) <br />                     DUR1_1                 Urea carboxylase (Nitrogen metabolism) <br />                     HIS2                   Histidinol-phosphatase (Histidine metabolism) <br />                     RNR1_2                 Ribonucleoside-diphosphate reductase large chain 1 (Salvage pathways) <br />                     HIS1                   ATP phosphoribosyltransferase (Histidine metabolism) <br />                     ARG4                   Argininosuccinate lyase (Arginine metabolism) <br />                     GLN1                   Glutamine synthetase (Glutamate metabolism) <br />  <br />  Figure 4 Transcriptionally controlled reactions (reactions change flux correlate change expression the <br />  involved genes) comparing carbon limited nitrogen limited growth comparing aerobic anaerobic growth. Red color <br />  corresponds up-regulation flux expression blue color corresponds down-regulation. <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                          Page 7 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br />                          A                                                        B <br />                                             Glucose     Flux      Expression              Acetyl-CoA <br />                                     Glucokinase <br />                                                                                                    Mevalonate pathway <br />  <br />                                                   G6P <br />  <br />                                                                                                  IPP                   Flux Expression <br />                                                   F6P <br />                                                                                                                ERG20 <br />                         Phosphofructokinase                                                DMAPP <br />                                                  F16P <br />                                                                                                               FPP <br />                                                                                                    ERG9 <br />                                                   G3P <br />                                                                                                          Squalene <br />                      Phosphoglycerase kinase <br />                                                                                                        ERG1 <br />                                                   3PG <br />                          Phosphoglycerase <br />                          mutase <br />                                                                                                        ERG11 <br />                                                   2PG <br />                                                                                                        ERG24 <br />                                       Enolase <br />  <br />                                                   PEP                                                  ERG26 <br />                                Pyruvate kinase <br />                                                                                                        ERG25 <br />                                             Pyruvate <br />                                                                                                        ERG3 <br />  <br />  <br />                           Z-score          Anaerobic-Aerobic (Nlim)        Clim-Nlim (Anaerobic)         Ergosterol <br />  <br />                       -10 -5    0   5 10 <br />                                            Anaerobic-Aerobic (Clim)        Clim-Nlim (Aerobic) <br />                            = No significant change <br />  <br />  <br />  Figure 5 Transcriptionally controlled reactions glycolysis FPP branch point. (A) Three glycolytic reactions are <br />  considered transcriptionally controlled conditions. The color indicates up- down- regulation conditions, flux <br />  expression. Only reactions considered transcriptionally controlled shown. White color means no <br />  significant change conditions particular comparison. The comparisons Anaerobic &#8211; Aerobic <br />  nitrogen limitation (upper left corner), Anaerobic &#8211; Aerobic carbon limitation (lower left corner), C-limited &#8211; N-limited under <br />  anaerobic conditions (upper right corner) C-limited &#8211; N-limited aerobic conditions (lower right corner). (B) Transcriptionally controlled <br />  reactions ergosterol biosynthesis pathway. <br />  <br />  <br />  <br />  <br /> observation flux changes direction                     Identification responsible transcription factors <br /> the expression enzymes catalyzing reactions                         In order identify transcription factors responsible the <br /> shown Figure 5A suggests glycolytic flux                        transcriptional control reactions shown Figure 4 <br /> transcriptionally controlled regulation                       Figure 5 performed hypergeometric enrichment <br /> key enzymes cell growth constrained. This                        test comparison transcription factor <br /> contrast studies engineering glycolytic flux                         based known transcription factor &#8211; gene interactions <br /> by over-expression glycolytic genes [23],                           [24]. The transcription factors hypergeometric <br /> shown possible increase glycolytic                       p-value 0.05 shown Table 2. The <br /> flux maximum observed un-limited                                  transcription factor Opi1p, known negative regulator <br /> growth conditions.                                                             phospholipid biosynthesis over-represented all <br />   Many steps ergosterol biosynthesis                             comparisons suggest phospholipid <br /> pathway down-regulated                                metabolism significantly changed comparisons. <br /> flux expression carbon limited conditions                            Phospholipid synthesis activated transcription <br /> compared nitrogen limited conditions (Figure 5B).                         factors Ino2p Ino4p [25] inactivated Opi1p. <br /> Many reactions up-regulated                           Opi1p turn activated protein kinase A (Tpk1p) that <br /> flux expression anaerobic conditions compared                           activated high glucose levels [26]. We most <br /> to aerobic nitrogen limitation. Only reactions                       genes regulated Opi1p up-regulated in <br /> considered transcriptionally controlled presented                    carbon limited conditions compared nitrogen limited <br /> in Figure 5.                                                                   conditions, suggests flux the <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                        Page 8 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> Table 2 Over-represented transcription factor interactions            anaerobic conditions. The change predicted <br /> among significantly changed reactions                                 flux reaction model compared with <br /> Comparison                          TF       Hypergeometric p-value   changes expression involved enzymes <br /> Anaerobic - Aerobic (C-limited)     Opi1p    0.0009                   reactions directly transcriptionally con- <br />                                     Pip2p    0.0168                   trolled identified transcription factors <br />                                                                       controlling reactions. The genome-scale metabolic <br />                                     Gis2p    0.0278 <br />                                                                       model experimental data valuable tool <br /> Anaerobic - Aerobic (N-limited)     Opi1p    0.0215 <br />                                                                       data integration order condition-dependent <br /> C-limited - N-limited (Anaerobic)   Opi1p    0.0070 <br />                                                                       differences yeast metabolism. <br /> C-limited - N-limited (Aerobic)     Yap7p    0.0018 <br />                                     Opi1p    0.0021                   Methods <br />                                     Dal80p   0.0153                   Model reconstruction <br />                                     Dig1p    0.0290                   The step constructing iTO977 model to <br />                                                                       merge iIN800 model [8] consensus network <br />                                                                       [3] form draft model. This importing <br /> phospholipid synthesis pathways repressed Opi1p              SBML files starting point reconstructions <br /> glucose excess conditions, carbon limited                  Matlab format using RAVEN toolbox [28]. <br /> conditions. We genes regulated            Additional pathways biochemical reactions were <br /> by Opi1p transcription factor up-regulated                 added Kyoto Encyclopedia Genes Genomes <br /> anaerobic conditions compared aerobic conditions.                [29], Saccharomyces Genome Database [30] litera- <br />   When comparing carbon limited growth nitrogen                  ture. A list reactions references available <br /> limited growth aerobic conditions identify              Additional file 3. Reactions consensus <br /> transcription factors Opi1p, Yap7p, Dal80p Dig1p               network taking place compartments extracel- <br /> important regulators considering change                  lular, cytoplasm, mitochondria peroxisome moved <br /> transcription change flux (Table 2). At glucose                cytoplasm merging models. Since the <br /> limited conditions yeast <span id='am-99' about='obo:NCBITaxon_376232' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_192380 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-100'  ></span><span id='am-101' property="rdfs:label" content="Goes" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>goes</span> respiration                iIN800 model doesn&#8217;t peroxisome model com- <br /> and glucose excess conditions cell produces                    partment reactions iIN800 involved in <br /> ethanol flux glycolysis controlled             fatty acid synthesis degradation (beta-oxidation) was <br /> by carbon catabolite repression. Similarly, high nitro-            moved cytoplasm peroxisome merging. <br /> gen concentrations nitrogen catabolite pathways               Water hydroxyl ions balanced model, <br /> repressed [27]. The transcription factor Dal80p known              assuming processes cell uses <br /> to involved nitrogen catabolite repression. In             molecules well. The model corrected for <br /> analysis majority genes regulated             inconsistencies network gaps, using gap filling methods <br /> by Dal80p higher expression higher flux                 RAVEN. The reactions identified inconsistent by <br /> carbon limited (excess nitrogen) conditions                Zomorrodi et al. [31] corrected improve ability <br /> nitrogen limited conditions aerobic cultivation,                predict knock-out growth phenotypes. CHEBI identifiers <br /> which indicates pathways transcriptionally             INCHI codes added metabolites following <br /> regulated Dal80p changing nitrogen                       MIRIAM standard reactions model have <br /> limitation carbon limitation.                                      links EC-numbers KEGG reactions. The biomass <br />                                                                       equation taken iIN800 model. The final <br /> Conclusions                                                           model converted Matlab format Excel <br /> In paper present iTO977, new updated genome-                format SBML format. <br /> scale metabolic model yeast. The model bigger in <br /> scope previous models used                   In silico growth simulations <br /> simulations scaffold integrated analysis.            Experimental data collected literature for <br /> The model follows standards introduced               chemostat cultivations performed various dilution rates <br /> the consensus reconstruction regarding annotation                  different conditions: carbon limited <br /> metabolites reactions seen              aerobic [20,32-35], carbon limited anaerobic [36,37], nitro- <br /> well-annotated database representing current know-                gen limited aerobic [35,36,38,39] nitrogen limited <br /> ledge yeast metabolism. By using model                  anaerobic [40,41] (Additional file 5). For simulation <br /> simulate different growth conditions compare                  purposes uptake glucose, NH3 oxygen were <br /> simulated fluxes different conditions using                  constrained model according measured values <br /> random sampling algorithm investigate yeast                    experiment. FBA simulations performed with <br /> metabolism carbon limited, nitrogen limited, aerobic               maximization biomass objective function the <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                  Page 9 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> deviation experimentally measured growth rate          conditions compared. The fraction com- <br /> was identified.                                                 parisons reference higher (or lower) <br />   Anaerobic conditions simulated iTO977           value second condition used probability <br /> model constraining oxygen uptake rate zero,           score flux decreasing increasing. The values of <br /> but allowing uptake sterols simulate             moderated t-statistic transcriptome data were <br /> that compounds supplied media              used calculate probability score P reaction <br /> anaerobic growth.                                               representing probability flux the <br />                                                                 transcription significantly changed direction <br /> Gene essentiality predictions                                   conditions. A hypergeometric enrichment test <br /> In silico simulations single gene knock-out growth           performed identify over-represented transcription <br /> were carried genes model          factor interactions list reactions each <br /> constraining flux corresponding reactions         comparison passing cutoff P &gt; 0.9 based known <br /> zero estimating maximal biomass flux               transcription factor-gene interactions [24]. <br /> using FBA. A relative fitness value calculated for <br /> each mutant f = mutant growth rate/wild type growth          Additional files <br /> rate. A mutation considered lethal relative <br /> fitness smaller cutoff value. Three different         Additional file 1: iTO977 model Excel format including <br /> cutoffs considered, 0.8, 0.9 0.95. The simulations      well-annotated spreadsheets reactions, metabolites genes. <br /> were carried growth glucose,               Additional file 2: iTO977 model SBML format. The model also <br />                                                                  available Biomet toolbox, www.sysbio.se/biomet. <br /> simulated minimal media, allowing uptake glucose, <br />                                                                  Additional file 3: Changes additional reactions pathways <br /> ammonium, oxygen, sulfate phosphate, simu-         iTO977. <br /> lated rich media (YPD media), uptake amino         Additional file 4: ORFs iTO977 Yeast 5 vice versa. <br /> acids nucleotides allowed. The viability          Additional file 5: Growth simulation results. <br /> mutants compared experimental observations             Additional file 6: Single double growth knockout <br /> growth phenotypes 524 single gene knockouts               simulation results. <br /> minimal media YPD media [6,30]. The model           Additional file 7: Additional constraints condition-specific <br /> used simulate relative fitness pairs double    models used random sampling. <br /> gene knock-outs, simulating growth glucose rich <br /> media (YPD) compared experimental observations           Competing interests <br /> for 234,719 double knock-outs [21].                             The authors declare competing interests. <br />  <br />                                                                 Authors&#8217; contributions <br /> Random sampling <br />                                                                 T&#214; reconstructed model, analyzed data, performed simulations and <br /> Experimental data chemostat cultivations         performed random sampling. SB involved manual curation the <br /> different conditions (carbon limited, nitrogen limited, aer-    model. IN, SB JN designed study supervised work. T&#214; wrote <br />                                                                 paper. All authors read approved final manuscript. <br /> obic anaerobic) taken Jewett et al. [20]. The <br /> raw transcriptome data normalized using Probe <br />                                                                 Acknowledgements <br /> Logarithmic Intensity Error (PLIER) method mod-         We acknowledge funding Novo Nordisk Foundation, European <br /> erated t-statistic used identify significantly changed   Research Council (ERC grant no. 247013), Chalmers Foundation Knut and <br /> genes conditions. The upper lower               Alice Wallenberg Foundation. We thank Swedish Bioinformatics <br />                                                                 Infrastructure Life Sciences (BILS). The open access charge funded by <br /> bounds exchange fluxes lipid biomass reac-       Chalmers Library. <br /> tions iTO977 model constrained according to <br /> experimentally measured fluxes (Additional file 7)     Received: 16 November 2012 Accepted: 23 April 2013 <br />                                                                 Published: 30 April 2013 <br /> condition. A set flux distributions compatible the <br /> measured fluxes generated considered        References <br /> conditions previously described [15]. In order   1. Forster J, Famili I, Fu P, Palsson B, Nielsen J: Genome-scale reconstruction <br /> to avoid solutions involving loops set default bounds        Saccharomyces cerevisiae metabolic network. Genome Res 2003, <br />                                                                     13(2):244&#8211;253. <br /> for metabolic fluxes Inf &#8211;Inf instead 1000        2. &#214;sterlund T, Nookaew I, Nielsen J: Fifteen years large scale metabolic <br /> and &#8722;1000 commonly done. For pair                  modeling yeast: developments impacts. Biotechnol Adv 2012, <br /> condition, compared sample reference            30(5):979&#8211;988. <br />                                                                 3. Herrgard M, Swainston N, Dobson P, Dunn W, Arga K, Arvas M, Bluthgen N, <br /> condition samples second condition              Borger S, Costenoble R, Heinemann M: A consensus yeast metabolic <br /> and computed times reference higher              network reconstruction obtained community approach to <br /> (or lower) value reaction. The process                 systems biology. Nat Biotechnol 2008, 26(10):1155&#8211;1160. <br />                                                                 4. Dobson P, Smallbone K, Jameson D, Simeonidis E, Lanthaler K, Pir P, Lu C, <br /> repeated samples reference condition             Swainston N, Dunn W, Fisher P, et al: Further developments a <br /> until possible combinations samples              genome-scale metabolic model yeast. BMC Syst Biol 2010, 4(1):145. <br />  &#214;sterlund et al. BMC Systems Biology 2013, 7:36                                                                                                           Page 10 10 <br /> http://www.biomedcentral.com/1752-0509/7/36 <br />  <br />  <br />  <br />  <br /> 5.    Heavner BD, Smallbone K, Barker B, Mendes P, Walker LP: Yeast 5&#8211;an                 28. Agren R, Liu L, Shoaie S, Vongsangnak W, Nookaew I, Nielsen J: The RAVEN <br />       expanded reconstruction Saccharomyces cerevisiae metabolic                      toolbox use generating genome-scale metabolic model for <br />       network. BMC Syst Biol 2012, 6(1):55.                                                  <span id='am-71' about='obo:NCBITaxon_5076' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_254878 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_5073 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:947851" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:5080" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Peniciliium chrysogenum" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:115827" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Penicillium chrysogenum" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Penicillium notatum" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:947850" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Penicillium chysogenum" datatype="xsd:string"></span>Penicillium chrysogenum</span>. PLoS computational biology 2013, <br /> 6.    Mo ML, Palsson B&#216;, Herrg&#229;rd MJ: Connecting extracellular metabolomic                   9(3):e1002980. <br />       measurements intracellular flux states yeast. BMC Syst Biol 2009, 3(1):37.   29. Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita K, Itoh M, Kawashima S, <br /> 7.    Kuepfer L, Sauer U, Blank LM: Metabolic functions duplicate genes                Katayama T, Araki M, Hirakawa M: From genomics chemical genomics: <br />       Saccharomyces cerevisiae. Genome Res 2005, 15(10):1421&#8211;1430.                           new developments KEGG. Nucleic Acids Res 2006, 34:D354&#8211;D357. <br /> 8.    Nookaew I, Jewett M, Meechai A, Thammarongtham C, Laoteng K,                       30. Cherry JM, Hong EL, Amundsen C, Balakrishnan R, Binkley G, Chan ET, <br />       Cheevadhanarak S, Nielsen J, Bhumiratana S: The genome-scale metabolic                 Christie KR, Costanzo MC, Dwight SS, Engel SR: Saccharomyces Genome <br />       model iIN800 Saccharomyces cerevisiae validation: scaffold                Database: genomics resource budding yeast. Nucleic Acids Res <br />       query lipid metabolism. BMC Syst Biol 2008, 2:71.                                   2012, 40(D1):D700&#8211;D705. <br /> 9.    Duarte N, Herrgard M, Palsson B: Reconstruction validation                  31. Zomorrodi A, Maranas C: Improving iMM904 S. cerevisiae metabolic <br />       Saccharomyces cerevisiae iND750, fully compartmentalized genome-                     model using essentiality synthetic lethality data. BMC Syst Biol 2010, <br />       scale metabolic model. Genome Res 2004, 14(7):1298&#8211;1309.                               4(1):178. <br /> 10.   Kim IK, Rold&#227;o A, Siewers V, Nielsen J: A systems&#65533;?level approach               32. Bakker BM, Bro C, K&#246;tter P, Luttik MAH, Van Dijken JP, Pronk JT: The <br />       metabolic engineering yeast cell factories. FEMS yeast research 2012,               mitochondrial alcohol dehydrogenase Adh3p involved redox <br />       12(2):228&#8211;248.                                                                         shuttle Saccharomyces cerevisiae. J Bacteriol 2000, 182(17):4730&#8211;4737. <br /> 11.   Sohn SB, Kim TY, Lee JH, Lee SY: Genome-scale metabolic model               33. Overkamp KM, Bakker BM, K&#246;tter P, Van Tuijl A, De Vries S, Van Dijken JP, <br />       fission yeast <span id='am-62' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> reconciliation                   Pronk JT: In Vivo Analysis Mechanisms Oxidation Cytosolic <br />       silico/in vivo mutant growth. BMC Syst Biol 2012, 6(1):49.                             NADH Saccharomyces cerevisiaeMitochondria. J Bacteriol 2000, <br /> 12.   Loira N, Dulermo T, Nicaud JM, Sherman D: A genome-scale metabolic                     182(10):2823&#8211;2830. <br />       model lipid-accumulating yeast <span id='am-17' about='obo:NCBITaxon_4952' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_34353 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_4951 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="rdfs:label" content="Yarrowia lipolytica" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida lipolytica" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Mycotorula lipolytica" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Endomycopsis lipolytica" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Yallowia lipolitica" datatype="xsd:string"></span>Yarrowia lipolytica</span>. BMC Syst Biol           34. Gombert AK, Moreira dos Santos M, Christensen B, Nielsen J: Network <br />       2012, 6(1):35.                                                                         identification flux quantification central metabolism of <br /> 13.   Caspeta L, Shoaie S, Agren R, Nookaew I, Nielsen J: Genome-scale                       Saccharomyces cerevisiae different conditions glucose <br />       metabolic reconstructions Pichia stipitis Pichia pastoris                repression. J Bacteriol 2001, 183(4):1441&#8211;1451. <br />       silico evaluation potentials. BMC Syst Biol 2012, 6(1):24.                35. Vemuri G, Eiteman M, McEwen J, Olsson L, Nielsen J: Increasing NADH <br /> 14.   Patil K, Nielsen J: Uncovering transcriptional regulation metabolism             oxidation reduces overflow metabolism Saccharomyces cerevisiae. <br />       using metabolic network topology. Proc Natl Acad Sci 2005,                             Proc Natl Acad Sci 2007, 104(7):2402&#8211;2407. <br />       102(8):2685&#8211;2689.                                                                  36. Tai SL, Daran-Lapujade P, Luttik MAH, Walsh MC, Diderich JA, Krijger GC, van <br /> 15.   Bordel S, Agren R, Nielsen J: Sampling solution space genome-scale              Gulik WM, Pronk JT, Daran JM: Control glycolytic flux in <br />       metabolic networks reveals transcriptional regulation key enzymes.                  Saccharomyces cerevisiae grown low temperature. J Biol Chem 2007, <br />       PLoS Comput Biol 2010, 6(7):e1000859.                                                  282(14):10243&#8211;10251. <br /> 16.   Cvijovic M, Olivares-Hernandez R, Agren R, Dahr N, Vongsangnak W,                  37. Nissen TL, Schulze U, Nielsen J, Villadsen J: Flux distributions anaerobic, <br />       Nookaew I, Patil KR, Nielsen J: BioMet Toolbox: genome-wide analysis                glucose-limited continuous cultures Saccharomyces cerevisiae. <br />       metabolism. Nucleic Acids Res 2010, 38(suppl 2):W144.                                  Microbiology 1997, 143(1):203&#8211;218. <br /> 17.   Burda P, Aebi M: The dol <br /> </body></html>