<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  The suil Suppressor Locus in <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Encodes                                                                        a <br />          Translation Factor That Functions during tRNAMet <br />                    Recognition of the Start <span id='am-64' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68'  ></span>Codon</span> <br />  <br />               We initiated a genetic reversion analysis at the HIS4 locus to identify components of the translation initiation <br />            complex that are important for ribosomal recognition of an initiator codon. Three unlinked suppressor loci, <br />            suil, sui2, and SUI3, that restore expression of both HIS4 and HIS4-lacZ in the absence of an AUG initiator <br />            codon were identified. In previous studies, it was demonstrated that the sui2 and SUI3 genes encode mutated <br />            forms of the a and subunits, respectively, of eukaryotic translation initiation factor 2 (eIF-2). In <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-63'  ></span>this</span> report, <br />            we describe the molecular and biochemical characterizations of the suil suppressor locus. The DNA sequence <br />            of the SUII + gene shows that it encodes a protein of 108 amino acids with a calculated Mr of 12,300. The suil <br />            suppressor genes all contain single base pair changes that alter a single amino acid within this 108-amino-acid <br />            sequence. suil suppressor strains that are temperature sensitive for growth on enriched medium have altered <br />            polysome profiles at the restrictive temperature typical of those caused by alteration of a protein that functions <br />            during the translation initiation process. Gene disruption experiments showed that the SUII + gene encodes an <br />            essential protein, and antibodies directed against the SUII + coding region identified a protein with the <br />            predicted Mr in a ribosomal salt wash fraction. As observed for sui2 and SUI3 suppression events, protein <br />            sequence analysis of His4-4-galactosidase fusion proteins produced by suil suppression events indicated that <br />            a UUG codon is used as the site of translation initiation in the absence of an AUG start codon in HIS4. Changing <br />            the penultimate proline codon 3&apos; to UUG at his4 to a Phe codon (UUC) blocks aminopeptidase cleavage of the <br />            amino-terminal amino acid of the His4-41-galactosidase protein, as noted by the appearance of Met in the first <br />            cycle of the Edman degradation reaction. The appearance of Met in the first cycle, as noted, in either a suil <br />            or a SUI3 suppressor strain showed that the mechanism of suppression is the same for both suppressor genes <br />            and allows the initiator tRNA to mismatch base pair with the UUG codon. This suggests that the Suil gene <br />            product performs a function similar to that of the , subunit of eIF-2 as encoded by the SUI3 gene. However, <br />            the Suil gene product does not appear to be a required subunit of eIF-2 on the basis of purification schemes <br />            designed to identify the GTP-dependent binding activity of eIF-2 for the initiator tRNA. In addition, suppressor <br />            mutations in the suil gene, in contrast to suppressor mutations in the sui2 or SUI3 gene, do not alter the <br />            GTP-dependent binding activity of eIF-2. The simplest interpretation of these studies is that the suil suppressor <br />            gene defines an additional factor that functions in concert with eIF-2 to enable tRNA&quot;et to establish ribosomal <br />            recognition of an AUG initiator codon. <br />  <br />   The ribosomal scanning model has been proposed to                      a  start codon is mediated by the initiator tRNA (5). By <br /> account for fundamental features of eukaryotic mRNAs and                 introducing a mutation in the anticodon of tRNAMet, it was <br /> basic steps during the pathway that leads to initiation of               shown that a mutant tRNA&quot;et was capable of directing the <br /> protein synthesis (17, 18). Briefly, eukaryotic translation              ribosome to initiate translation at the first cognate codon in <br /> initiation factor 2 (eIF-2), which is composed of three protein          the his4 message through a complementary anticodon-codon <br /> subunits, cx, P, and y, binds tRNAMet in the presence of GTP             base pair interaction. However, considering the complexity <br /> to form a ternary complex (15). This ternary complex,                    of the eukaryotic translation initiation process, as reflected <br /> eIF-2-GTP-tRNAMet, associates with the 40S ribosomal                     by the fact that eukaryotic initiation factors are composed of <br /> subunit, which in turn binds the 5&apos; end of the message and               at least 25 proteins (24), it was conceivable that, in addition <br /> migrates in the 5&apos;-to-3&apos; direction until the first AUG codon is          to   tRNA&quot;&apos;t,   other components of the translation initiation <br /> found, where initiation occurs. The basic feature of this                complex play an essential role in the start site selection <br /> model has been corroborated in the yeast <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span>Saccharomyces</span>                   process. Unfortunately, biochemical assays for specific <br /> cerevisiae by mutational studies of the CYC1 (30) and HIS4               steps during ribosomal recognition of a start codon have not <br /> (6, 9) genes. These studies suggest that by virtue of its                been effective in identifying those factors that function in <br /> proximity to the 5&apos; end of an mRNA, an AUG codon is used                 establishing the start site of translation. <br /> as the site of initiation. Therefore, the start site selection <br />                                                                            To identify other components of the initiation complex <br /> mechanism proposed for eukaryotes places the AUG codon                   that are important in defining start site selection in eukary- <br /> and its interaction with the initiator tRNAMet as a fundamen-            otes, we initiated a genetic analysis of the translation initia- <br /> tal event in this process. In fact, it has been previously               tion process in S. cerevisiae (4). It has been demonstrated <br /> demonstrated in our laboratory that ribosomal recognition of             previously that mutations of the AUG start codon at HIS4 <br />                                                                          leads to ribosomal bypass of the early HIS4 coding region, <br />                                                                          resulting in a His- phenotype (9). Therefore, by selecting for <br />   * <br />       Corresponding author.                                              spontaneous     His&apos;   revertants of these initiator codon mutant <br />  <br />                                                                    248 <br />  VOL. 12, 1992                                    suil, A TRANSLATION INITIATION SUPPRESSOR LOCUS IN YEAST                        249 <br />  <br /> strains, mutations that no longer allow the ribosome to                                  TABLE 1. Yeast strains <br /> bypass this region were obtained (4). One type of His&apos;                 Strain                             Genotype <br /> revertant restored the AUG start codon at HIS4. However, <br /> by maintaining a second copy of the his4 initiator codon            TD28 .....        MATa ura3-52 inol-13 <br /> mutation as part of a his4-lacZ gene fusion, extragenic             1172-3B .....     MATa his4-316 ura3-52 <br /> suppressors were isolated on the basis of the ability to allow      BC-64 .....       MATot his4401 ura3-52 leu2-3,-112 <br />                                                                     117-8A .....      MATot his4-303(AUU) <br /> translation initiation to occur from both copies of the mutant                          ura3-52: :his4(AUU)-lacZ(Ura+) <br /> messages, as noted by His&apos; revertants that were blue on             EKP84 .....       MATa/MATTa ura3-521ura3-52 LEU21leu2-3,-112 <br /> 5-bromo-4-chloro-3-indolyl-p-D-galactopyranoside <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>indicator</span>                              HIS4+1his4-306(UUG) <br /> plates. Three suppressor genes were identified by this selec-       117-8AR4 .....    MATa suil-J his4-303(AUU) <br /> tion method (4), i.e., suil, sui2, and SUI3 (suppressor of                              ura3-52: :his4(AUU)-lacZ(Ura+) <br /> initiator codon mutations). The basic assumption was that           301-4D .....      MATa suil-J his4-303(AUU) ura3-52 leu2-3,-112 <br /> these suppressor loci contain mutations in genes that encode        HY32 .....        MATa suil-J his4-303(AUU) ura3-52 leu2-3,-112 <br /> trans-acting factors that alter, and therefore function in                              his4(AUU/Phe)-lacZ(Ura+) <br /> mediating, ribosomal recognition of the initiator region dur-       302-2D .....      MATa suil4 his4-303(AUU) ura3-52 leu2-3,-112 <br />                                                                                         inol-13 <br /> ing the scanning process. That this assumption is correct was       139-3C .....      MATa suil-7 his4-303(AUU) ura3-52 <br /> borne out by the molecular and biochemical analysis of the          138-7B .....      MATa suil-17 his4-301(ACG) ura3-52 <br /> sui2 (7) and SUI3 (10) suppressor genes. The sui2 suppressor        117-8AR20 .....   MATot sui2-1 his4-303(AUU) <br /> gene encodes the a subunit of eIF-2, which is 58% identical                             ura3-52: :his4(AUU)-lacZ(Ura+) <br /> to the human a subunit of eIF-2 (7). Mutations in the sui2          2119-liD .....    MATot SUI3-3 his4-303(AUU) ura3-52 leu2-3, <br /> suppressor genes that restore his4 expression change amino                              -112 inol-13 <br /> acids that are identical at the same position in the human          HY40 .....        MATa SUI3-3 his4-303(AUU) ura3-52 leu2-3, <br /> eIF-2 sequence (7, 12). The SUI3 suppressor gene encodes                                -112 inol-13 his4(AUU/Phe)-lacZ(Ura+) <br /> the 1 subunit of eIF-2 (10) and is 42% identical to the human <br /> eIF-213 subunit (21). Mutations that confer SUI3 suppression <br /> map to a putative zinc finger motif that is also conserved in <br /> the carboxyl region of the human 1 subunit (10). Both the           tion and characterization of the suil suppressor strains have <br /> sui2 and the SUI3 genes are essential for cell viability (7, 10).   been previously reported (4). Aside from the suppressor <br /> Amino-terminal sequence analysis of the His4-4-galactosi-           phenotype, restoration of HIS4 expression (growth on syn- <br /> dase protein produced as a result of either sui2 or SUI3            thetic dextrose medium lacking histidine [SD-histidine]) to <br /> suppression events showed that in the absence of an AUG             yeast strains containing an initiator codon mutation at the <br /> codon at his4, ribosomes no longer bypassed the mutated             HIS4 gene, the suil suppressor strains have a temperature <br /> initiator region but recognized and initiated at a UUG codon        sensitivity (Ts-) phenotype for growth at 37&#176;C on YEPD <br /> in the early His4-coding region. These studies suggested that       medium (yeast extract, peptone, dextrose). The suppressor <br /> in addition to its initiator tRNA binding activity, eIF-2 was       phenotype and the Ts- growth phenotype associated with <br /> also involved in mediating the general mechanism of start           suil suppressor strains cosegregate in crosses (4). <br /> site selection in all eukaryotic organisms.                            Isolation and characterization of SUII alleles. The wild- <br />    In this report, we present the characterization of the suil      type SUIJ gene was cloned from both YEp24 and YCp5O <br /> suppressor gene. Our analysis showed that the suil gene             wild-type genomic clone banks. Positive clones were identi- <br /> encodes a previously <span id='am-124' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span>unidentified</span> 12.3-kDa protein. De-             fied on the basis of complementation of the recessive Ts- <br /> scribed properties of suil mutant strains and the Suil protein      phenotype with a concomitant reduction or loss of the His&apos; <br /> are consistent with the notion that this gene product per-          phenotype associated with suil suppressor strains 302-2D, <br /> forms an essential function during the translation initiation       139-3C, and 138-7B. Restriction analysis of the clones de- <br /> process in S. cerevisiae. In addition, we demonstrated that         rived from the YEp24 and YCp5O banks indicated that the <br /> the effect of a suppressor mutation in the suil gene is similar     inserts were related and derived from the same chromosomal <br /> to that of a suppressor mutation in the a subunit of eIF-2 as       location. <br /> encoded by SUI3 and is to allow the normal translation                 The relationship of the cloned DNA fragment to the SUII <br /> initiation apparatus to initiate at a UUG codon in the early        locus was established by genetic analysis. An 8.2-kb BamHI <br /> His4-coding region. Despite similar mechanisms of suppres-          DNA fragment from the YEp24 clone bank was subcloned <br /> sion, the Suil protein is not a subunit of eIF-2 that is            into the BamHI site of integrating yeast vector YIp5 and <br /> required for ternary complex formation. We suggest that             used to transform yeast strain TD28 to URA3+. Ura+ <br /> suil encodes a translation initiation factor that functions         transformants were then crossed to ura3-52 suil suppressor <br /> with eIF-2 and the initiator tRNA in directing the ribosome         strains 139-3C and 302-2D and analyzed by tetrad analysis. <br /> to the proper start site of translation.                               An approximately 10-kb Sau3A DNA insert was obtained <br />                                                                     from the YCp5O genomic bank, and this clone was used for <br />                MATERIALS AND METHODS                                subsequent characterization of the wild-type SUIJ gene (see <br />                                                                     Fig. 1). Subcloning experiments for localization of the SUIJ <br />    Yeast strains and genetic methods. The yeast strains used        gene were performed with single-copy URA3 yeast vector <br /> in this analysis and their complete genotypes are listed in         YCp5O. The SUIJ gene was localized to a 1.2-kb BglII- <br /> Table 1. All strains are related to TD28, an ascospore              HindIII DNA fragment (see Fig. 1). The complete DNA <br /> derivative of S. cerevisiae S288C (MATa). The standard              sequence of this fragment (see Fig. 2) was determined for <br /> genetic techniques and media used for these studies have            both strands by either the Maxam-and-Gilbert (19) or the <br /> already been described (29).                                        chain termination DNA sequencing method (25). <br />    suil suppressor strains 117-8AR4 (suil-1), 302-2D (suil4),          A YCp5O plasmid, p1199, containing the wild-type SUIJ <br /> 139-3C (suil-7), and 138-7B (suil-17) were used for molecu-         gene as part of a 5.4-kb BgIII-Sall DNA fragment (see Fig. <br /> lar characterization of the suil gene. The genetic identifica-      1; constructed from a partial BglII digest) was used to isolate <br />  250    YOON AND DONAHUE                                                                                        MOL. CELL. BIOL. <br />  <br /> the suil-1, suil4, suil-7, and suil-17 suppressor alleles by       plementary to nucleotide positions +28 to +57 in the SUIJ <br /> the gap duplex repair method (20). For these experiments,          coding region (see Fig. 2). <br /> plasmid p1199 was restricted at the BamHI restriction site            For both Southern and Northern analyses, a 1.4-kb SphI <br /> located within the coding region of the SUIJ gene (see Fig.        restriction fragment containing the entire SUIJ gene was <br /> 1). Ts- suil suppressor strains 301-4D, 302-2D, 139-3C, and        used as a probe. 32P-labelled DNA was prepared by the <br /> 138-7B were transformed with this linearized DNA selecting         random-primer labelling method. <br /> for Ura+ transformants. Ura+ transformants were replica               The proximal BglII-BamHI DNA fragment from the SUIJ <br /> plated and tested for a temperature-sensitive growth pheno-        wild-type gene (see Fig. 1) was inserted into the BamHI site <br /> type on enriched medium and the ability to grow on SD-             of plasmid p349 (9), which contains the E. coli lacZ coding <br /> histidine plates. Ura+ Ts&apos; transformants that showed re-           region as part of YCp5O. This plasmid, p1132, results in an <br /> duced growth on SD-histidine plates indicated that the             in-frame sluil-lacZ fusion and was used to transform yeast <br /> wild-type gene was still present on the plasmid. Ura+ Ts-          strain TD28 to URA3+. Transformants were screened on <br /> transformants that maintained the His&apos; suppressor pheno-           5-bromo-4-chloro-3-indolyl-,B-D-galactopyranoside indicator <br /> type indicated <span id='am-119' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> containing the corresponding suil          plates, and the 3-galactosidase activities of transformants <br /> mutant allele as a result of repair-associated gene conversion     were determined as previously described (9). <br /> events. Plasmids from the Ts- His&apos; transformants were                 Site-directed mutations at the HIS4 gene. Mutations in the <br /> isolated in <span id='am-78' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span>, and the presence of a single         HIS4 coding region were constructed by site-directed muta- <br /> mutation in the coding region of each of the suil suppressor       genesis as previously described (9). The template used was <br /> alleles was confirmed by DNA sequencing.                           bacteriophage vector M13mpll, containing a 1,578-bp Sall <br />    Analysis of SUII expression. For gene disruption experi-        DNA insert from the proximal region of the HIS4 gene (11). <br /> ments (23), a 2.3-kb XhoI-HindIII DNA fragment (see Fig. 1)        The oligonucleotide used to introduce mutations in the HIS4 <br /> containing the intact SUI1 gene was ligated into the Sall and      coding region was 5&apos;-TGAATAATTGTTTTGTTCATTCTA <br /> HindlIl sites of pBR322 to yield plasmid p1134. A 1.1-kb           CC-3&apos;. This oligonucleotide changed the HIS4 AUG start <br />                                                                    codon to AUU and amino acid position 4 in the HIS4 coding <br /> Hindlll fragment containing the URA3 gene was adapted              region from a proline (CCG) to a phenylalanine (UUC) <br /> with BamHI ends and inserted into the BamHI site within            codon. Positive plaques for each construct were identified by <br /> the SUIJ coding region (see Fig. 2) of p1134 to construct          hybridization with the corresponding 32P-labelled oligonu- <br /> plasmid p1209 (SUIJ:: URA3+). As a result of this construc-        cleotide, and the presence of the mutations in the HIS4 <br /> tion, the SUIJ coding region was disrupted. p1209 was              region was confirmed by DNA sequencing. A 763-bp Sau3A <br /> restricted with either SphI or ClaI and KpnI and used to           fragment which contains the full, intact HIS4 promoter <br /> transform diploid strain EKP84 to Ura+. Restriction of             region and the first 11 amino acids of the HIS4 coding region <br /> p1209 with these enzymes generated an approximately                was then subcloned from two independent constructs into <br /> 3.4-kb SUIJ:: URA3+ DNA fragment. Transformants were               the BamHI site of p349 to yield an in-frame his4-lacZ fusion <br /> analyzed by tetrad analysis. Genomic DNA was isolated              plasmid. We refer to this mutated his4-lacZ plasmid con- <br /> from one of these diploid Ura3+ transformants, HJY3, and           struct as his4(AUUIPhe)-lacZ. This plasmid was used to <br /> from SUIJ wild-type haploid strain 117-8A. The genomic             transform yeast strains 301-4D (suil-J) and 2119-liD <br /> DNAs were restricted with Sacll and BglIl (see Fig. 1) and         (SUI3-3) to Ura+, which contain a deletion of the HIS4 gene. <br /> analyzed by Southern blotting.                                     Transformants were then nonselectively screened on <br />    The 1,200-bp BglII-HindlII DNA fragment (see Fig. 1)            5-bromo-4-chloro-3-indolyl-3-D-galactopyranoside plates for <br /> containing the wild-type SUIJ gene was subcloned into the          blue colonies. His&apos; blue colonies from each transformation <br /> BamHI and HindIII sites of YCp5O to produce plasmid                were used for purification and amino-terminal sequence <br /> p1128. p1128 was then restricted with BamHI in the SUIl            analysis of the His4-p-galactosidase protein produced as a <br /> coding region (see Fig. 1), and self-complementary oligonu-        result of suppression events (see below). The corresponding <br /> cleotide 5&apos;-GATCTCTAGA-3&apos; was ligated into the BamHI               yeast strains used for His4-4-galactosidase sequence analy- <br /> site. Insertion of this oligonucleotide resulted in a + 1 frame-   sis are referred to as HY32 [suil-J his4(AUUIPhe)-lacZ] and <br /> shift mutation in the SUIJ coding region and a subsequent          HY40 [SUI3-3 his4(AUUIPhe)-lacZ]. <br /> in-frame translation termination signal, TAG, as part of the          Polysome analysis. Polysome profiles of Ts- suil mutants <br /> oligonucleotide. This plasmid, p1511, was used to transform        117-8AR4 and 138-7B were analyzed from cultures incubated <br /> suil suppressor strains 138-7B, 139-3C, 301-4D, and 302-2D         under permissive (23&#176;C) and restrictive (37&#176;C) growth con- <br /> to Ura+. Transformants were tested for the ability to com-         ditions and compared with the polysome profiles of the <br /> plement the Ts- phenotype associated with these suil sup-          parent SUIl + strain, 117-8A, and a Ts- sui2 mutant strain, <br /> pressor strains.                                                   117-8AR20, incubated at the same temperatures. Two cul- <br />    For Northern (RNA) analysis, total and poly(A)+ RNAs            tures of each strain (200 ml) were grown overnight at 23&#176;C to <br /> were isolated from wild-type and suil mutant strains as            an optical density at 600 nm (OD6.) of 1.0 in liquid YEPD <br /> previously described (6). A 50 p.g sample of total RNA or 10       medium. One of the two cultures from each strain was then <br /> Fg of poly(A)+ RNA was electrophoresed on a 6% acryl-              shifted to 37&#176;C and grown for two more hours. At the end of <br /> amide-8 M urea gel (3). Gels were electroblotted to nylon          the 2-h growth period, cycloheximide (10 mg) was added to <br /> filters in 1 mM Tris-acetate (pH 7.8)-5 mM sodium acetate-         each of the cultures grown at 23 and 37&#176;C and extracts were <br /> 0.5 mM EDTA at 20 V overnight. Blots were fixed by                 prepared as described by Baim et al. (2). Fifty OD260 units of <br /> germicidal lamp exposure (330 ptW/cm2 for 2 min). The              each extract was layered on top of a 35-ml linear sucrose <br /> conditions for prehybridization and hybridization have al-         gradient (7 to 47% [wt/vol]) containing 50 mM Tris-acetate <br /> ready been reported (4). The 5&apos; map position of the SUIl+          (pH 7.0), 50 mM NH4C1, 12 mM MgCI2, 1 mM dithiothreitol, <br /> transcripts was determined by primer extension as previ-           0.1% diethy pyrocabonate (DEPC), and 1 mM phenylmeth- <br /> ously described (6), by using 50 F&#177;g of total RNA isolated         ylsulfonyl fluoride. The gradient was prepared by underlay- <br /> from yeast strain 1172-3B and the oligonucleotide 5&apos;-TTCG          ing in a stepwise fashion 7 ml each of 17, 27, 37, and 47% <br /> TCGTCTCCTGTGTCGGCGAAAGGATC-3&apos;, which is com-                        sucrose below an initial 7 ml of 7% sucrose in a centrifuge <br />  VOL. 12, 1992                                  suil, A TRANSLATION INITIATION SUPPRESSOR LOCUS IN YEAST                     251 <br />  <br /> tube and allowed to equilibrate overnight at 4&#176;C. Sucrose           Protein sequence analysis. His4-3-galactosidase protein <br /> gradients were centrifuged at 24,000 rpm in an SW27 rotor        was purified from yeast strains as previously described (10). <br /> for 4.5 h. Gradients were collected from the top by using an     For some experiments, the His4-,-galactosidase protein <br /> ISCO 640 density gradient fractionator and monitored by the      was purified by a modification of the latter part of the <br /> ISCO UA-5 absorbance-fluorescence detector. The 40S and          published procedure. In these experiments, His4-p-galac- <br /> 60S peaks of the polysome gradients were determined by           tosidase was eluted from the p-aminobenzyl-1-thio-P-D-ga- <br /> Northern analysis. rDNA probes were prepared by labelling        lactopyranoside-agarose column and peak activity fractions <br /> EcoRI fragments restricted from pBR322-derived plasmids          were concentrated and dialyzed with a Centricon 30. Protein <br /> pBR322C and pBR322A, which contain 18S (40S) and 25S             concentrations were determined, and aliquots were run on a <br /> (60S) rDNAs, respectively (22).                                  sodium dodecyl sulfate (SDS)-7.5% polyacrylamide gel that <br />    Immunological methods. Antibodies directed against the        contained 10 ,uM sodium thioglycolate in the electrophoresis <br /> SUIJ gene product were generated from TrpE-Suil fusion           buffer in the upper reservoir. The gel was blotted to a <br /> proteins expressed in E. coli. The distal SUIJ coding region     ProBlot nylon membrane. The transfer buffer contained 10 <br /> from amino acid positions +71 to +108 contained on a             mM CAPS [3-(cyclohexylamino)-1-propanesulfonic acid] <br /> 445-bp BamHI-HindIII DNA fragment (see Fig. 1) was               (pH 11.0) with 10% methanol. Blotting was performed at 300 <br /> ligated in frame to the carboxyl end of the trpE coding region   mA for 1 h, and the membrane was stained with Coomassie <br /> as part of the pATH2 expression vector (8). A second             blue to visualize the His4-p-galactosidase protein. Purified <br /> construct was made by ligating the 496-bp EcoRI-HindIII          His4-,3-galactosidase (Mr, approximately 116,000) was iden- <br /> DNA fragment (see Fig. 1) from the distal end of SUII to         tified as a single protein band that comigrated on gels with E. <br /> pATH1, which results in an in-frame fusion of TrpE to amino      coli ,-galactosidase (Mr, approximately 115,000) which was <br /> acid positions +54 to + 108 of the SUIJ coding region. The       used as a molecular weight standard for each experiment. <br /> methods employed for extraction of the antigen and immu-         The His4-p-galactosidase band identified on the filter was <br /> nizations have already been reported (10). The specificities     then excised and used for amino-terminal sequence analysis. <br /> and titers of the antisera were determined by Western blot       r-Galactosidase specific activities of crude extracts were 4 U <br /> (immunoblot) analysis by using extracts prepared from sin-       for parent strain 117-8A, 22 U for suil-J suppressor strain <br /> gle-copy SUIl strain BC64 and isogenic strain BC61, which        117-8AR4 [his4(AUU)-lacZ], 450 U for HIS4&apos;-lacZ SUII+ <br /> is identical to BC64 but contains the SUII gene on high-copy     wild-type strains, 18 U for HY32 (suil-l AUUIPhe), and 150 <br /> URA3+ vector YEp24. Preparation of cell extracts and the         U for HY40 (SUI3-3 AUU/Phe). The amino-terminal se- <br /> Western blot conditions used have been previously de-            quence of His4-p-galactosidase protein was determined at <br /> scribed (10).                                                    the Northwestern University Biotechnology Center by auto- <br />    Fractionation and assay of eIF-2. eIF-2 was partially puri-   mated Edman degradation using an Applied Biosystems <br /> fied from SUIJ wild-type strain 117-8A and suil suppressor       (Foster City, Calif.) 120A on-line phenylthiohydantoin <br /> strains 117-8AR4 and 138-7B by a modification of the method      amino acid analyzer (16). Approximately 10 to 100 pmol of <br /> previously described for eIF-2 purification from S. cerevi-      His4-3-galactosidase was estimated to have been used for <br /> siae (10). A 12-liters volume of cells was grown at 23&#176;C to an   the different analyses. <br /> OD6. of 1.3. Cells were harvested and washed twice in               Nucleotide sequence accession number. The nucleotide <br /> double-distilled H20. Cells were suspended in a lysis buffer     sequence presented here has been assigned GenBank acces- <br /> containing 20 mM Tris-HCI (pH 7.5), 100 mM KCI, and 30           sion no. M77514. <br /> mM MgSO4 and then broken with glass beads in a Bead <br /> Beater. Extracts were cleared by two sequential centrifuga-                                 RESULTS <br /> tions (5,000 x g for 10 min and 22,000 x g for 30 min), and <br /> ribosomes were isolated by centrifugation at 200,000 x g for        Characterization of the SU11+ gene. The wild-type SUIJ <br /> 2 h. Ribosomal pellets were washed in lysis buffer containing    gene was identified as part of approximately 9- and 10-kb <br /> 500 mM KCI and recentrifuged for 2 h at 200,000 x g. The         DNA inserts from YEp24 and YCpSO wild-type genomic <br /> supernatant was then dialyzed overnight against 20 mM            clone banks, respectively, on the basis of its ability to <br /> Tris-HCl (pH 7.5)-100 mM KCI. This ribosomal salt wash           complement the recessive Ts- phenotype and the poor <br /> was then fractionated in a stepwise fashion with a saturated     growth characteristics at 23&#176;C associated with suil mutant <br /> solution of ammonium sulfate. The 55 to 60% ammonium             strains. In addition, these clones, when present in suil <br /> sulfate pellet was then suspended in 20 mM KPO4 (pH 7.5)         strains, either reduced or abolished the His4+ suppressor <br /> containing 5% glycerol (buffer A) and dialyzed against the       phenotype, indicating that, as observed in suiJISUIlJ dip- <br /> same buffer overnight. The dialyzed sample was loaded onto       loids, the wild-type gene acted in either a codominant or a <br /> a hydroxyapatite (HAP) column (1.2 by 3.3 cm) that was           dominant fashion, depending on the suil allele (4). (We <br /> pre-equilibrated with buffer A. After unbound proteins were      indicate by genetic nomenclature that the suil suppressor <br /> washed from the column with buffer A, eIF-2 was eluted by        alleles are recessive to the wild-type allele [4]. Although the <br /> using a 100 to 600 mM linear gradient of KPO4 (pH 7.5)           Ts- phenotype is recessive, it should be noted that the <br /> containing 5% glycerol. Fractions (2 ml) were concentrated       suppressor phenotype associated with different suil strains <br /> by using a Centricon 10 and assayed for eIF-2 activity. All      behaves as either a recessive or a very weak codominant <br /> eIF-2 purification steps were performed at 4&#176;C, and the          trait in suil/SUIl + heterozygotes. We believe this is due to <br /> buffers used for fractionation contained the protease inhibi-    an altered function of low efficiency that gives rise to suil <br /> tors 1 mM phenylmethylsulfonyl fluoride and 0.7 ,ug of           suppression events [4] and not a reflection of a trait that is <br /> pepstatin A per ml. Protein concentrations were determined       truly recessive. Similar observations have also been made <br /> by the dye-binding method of Bradford (Bio-Rad Laborato-         with the sui2 suppressor locus, which encodes altered forms <br /> ries) with bovine serum albumin as the standard. eIF-2           of the a subunit of eIF-2 [4].) An 8.2-kb BamHI fragment <br /> activity was determined as previously described (10), except     derived from the YEp24 bank was subcloned into the inte- <br /> for a change in GTP concentration being reduced in the           grating YIpS vector and used to transform SUIJ wild-type <br /> assay mixture from 1.2 to 0.12 mM.                               yeast strain TD28 to Ura+. Transformants were purified and <br />  252        YOON AND DONAHUE                                                                                                            MOL. CELL. BIOL. <br />  <br />                                                                                  crossed to Ts- Ura3 suil suppressor strains. Among 21 <br />                                                                                  tetrads analyzed, all Ura+ ascospores were Ts+ whereas all <br />                                                                                  Ura- ascospores were Ts-, indicating that the DNA region <br />                                                                                  was derived from the SUIJ locus. The ability to complement <br />                                                                                  the suil Ts- phenotype was localized to a 1.2-kb BgIII- <br />                                                                                  HindIII DNA fragment (Fig. 1). Southern analysis showed <br />                                                                                  that this region is unique in the haploid genome of S. <br />                                                                                  cerevisiae (<span id='am-114' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116'  ></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> not shown). <br />                                                                                    The DNA sequence of the 1.2-kb BgIII-HindIII region is <br />                                                                                  shown in Fig. 2. An open reading frame of 108 amino acids <br />                                                                                  that begins with an ATG is contained in this sequence and <br />                                                                                  would encode a protein with a calculated Mr of 12,300. <br />                                                                                  Northern analysis of total and poly(A)+ RNAs isolated from <br />       le <br />                                                                                  wild-type and mutant suil strains identified a unique tran- <br />                                                                                  script from this region. On the basis of estimation of an Rf <br />                                                                                  value, this transcript is between 400 and 600 nucleotides <br />   FIG. 1. Restriction map and sequencing strategy for the wild-                  long, which is sufficient to encode a 108-amino-acid protein <br /> type SUJI clone. The approximately 10.1-kb BamHI-BamHl- <br /> Sau3A DNA fragment was identified from a YCp5O clone bank on                     (data not shown). Primer extension analysis (data not <br /> the basis of the ability to complement the temperature-sensitivity               shown) identified major and minor start positions for the <br /> phenotype associated with suil suppressor strains. The SUII gene                 transcription of this region with two major transcripts having <br /> was localized to the 1,200-bp BgIII-HindIII DNA fragment. The                    5&apos; ends that mapped at base pair positions -18 and -20 <br /> direction and extent of DNA sequencing by the dideoxy (25) and                   upstream from the first ATG that defines the 108-amino-acid <br /> Maxam-and-Gilbert (19) methods are indicated as arrows with                      opening reading frame (Fig. 2). <br /> closed circles and asterisks, respectively. Restriction site abbrevia-              To demonstrate that the 108-amino-acid open reading <br /> tions: Bam, BamHI; BS, BssHII; H, Hindlll; K, KpnI; P, PstI; Pv, <br /> PvuII; R, EcoRI; Sac, SacII; Sal, Sall; Sau3A, Sau3AI; Sp, SphI;                 frame contained on the BglII-HindIII SUIl clone (p1128) is <br /> X, XhoI.                                                                         responsible for complementation of the Ts- and His+ sup- <br />                                                                                  pressor phenotype of suil mutant strains, plasmid p1511 was <br />                                                                                  constructed to contain a + 1 frameshift mutation after amino <br />  <br />                                               BglII <br />  <br />                                                                                                                             BssHI I <br />                     tttctattgtttcactattctttatatatatttatatatgtatattttccagcaaaacaccctttttggcattctcctataggacgtcaaacggcgcgcg <br />                     *                     .                           .            .                    .                   Bs   HI <br />  <br />  <br />  <br />  <br />                     accgcgt cat tttat acgagcagccgctgcgt cgtcegtacaaaacegcaccgaccta cat tacaataaaaagta cccgtaaaaggat catgt gtacat ggt <br />                                                                           PstI <br />                     tccagtcttttggttcggtactataactactgtaaacgaccctctcactgcagtttatgatatgtgatcgcgttaaaaatgggtattatcaccaataacg <br />                     *                     .           .              .st                .. <br />  <br />  <br />  <br />  <br />                     ggtcgggtaacagattttttcgatgcgccggcaaaaaattttagtttttttttttttcatcacagggataaaagcaaactgcaaaaggacaaaacaagag <br />  <br />                     aagagctacttgttgttttcgtttcgtgctgggggttttctcctttctattctttttcaaaaaagacaaggaatatattaatatagctgaagcaaatcgt <br />                     1 <br />                     ATG TCC ATT GAG AAT CTG AAA TCA TTT GAT CCT TTC GCC GAC ACA GGA GAC GAC GAA ACC GCC ACT TCA AAC TAT <br />                     Met Ser Ile Glu Asn Leu Lys Ser Phe Asp Pro Phe <span id='am-101' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104'  ></span><span id='am-105' property="rdfs:label" content="Ala" datatype="xsd:string"></span>Ala</span> Asp Thr Gly Asp Asp Glu Thr Ala Thr Ser Asn Tyr <br />                     26 <br />                     ATT CAT ATT CGT ATC CAA CAG AGA AAT GGT AGA AAA ACT TTA ACT ACG GTG CAA GGT GTC CCA GAG GAA TAT GAT <br />                     Ile His Ile Arg Ile Gln Gln Arg Asn Gly Arg Lys Thr Leu Thr Thr Val Gln Gly Val Pro Glu Glu Tyr Asp <br />                     51       EcoRI                                                                BamHI <br />                     TTA AAG AGA ATT CTT AAG GTC CTA AAG AAG GAC TTT GCA TGT AAT GGT AAC ATT GTC AAG GAT CCA GAA ATG GGG <br />                     Leu Lys Arg Ile Leu Lys Val Leu Lys Lys Asp Phe Ala Cys Asn Gly Asn Ile Val Lys Asp Pro Glu Met Gly <br />                     76 <br />                     GAG ATT ATT CAG TTG CAG GGT GAC CAA AGA GCA AAG GTT TGC GAA TTT ATG ATC TCC CAA CTG GGA TTG CAA AAG <br />                     Glu Ile Ile Gin Leu Gln Gly Asp Gln Arg Ala Lys Val Cys Glu Phe Met Ile Ser Gln Leu Gly Leu Gln Lys <br />  <br />                     101 <br />                     AAG AAC ATT AAA ATT CAT GGG TTT TAA gttcaaggcttacgccgagtcgcttactcgctgctcgctttctttcttagtgttgttatcctat <br />                     Lys Asn Ile Lys Ile His Gly Phe Ter <br />  <br />                     ataaaacgtaactgtttagttatttaagtccaagtgcatatttacactataatatataatataacat ctcatctttttttatttctttttcttcttet cc <br />  <br />                     cgccgcgggcttacatacagagatataagaacaagtggtacgtagcatgcagcttctcaaaagagatgagcgttcagttttgagtacagggaagaaaggc <br />                                                                                     HindIII <br />                     caaatcgagagtacattatcatggggtcgggcgatttgaatctgttgaaatcatggaatcccaagctt <br />   FIG. 2. DNA sequence analysis of the wild-type SUIJ gene. The DNA sequence corresponding to the 108-amino-acid sequence of the <br /> SUIJ gene product was confirmed by either the Maxam-and-Gilbert or dideoxy chain termination method (see Fig. 1 for sequencing strategy). <br /> The asterisks (left to right) indicate the nucleotides changed in the mutant suil-4 and suil-7 alleles (GAC [Asp] to TAC [Tyr]) and the suil-) <br /> mutant allele (GAC [Asp] to GGC [Gly]) that altered amino acid position +83 and the suil-17 mutant allele (CAA [Gln] to CCA [Pro]) that <br /> altered amino acid position +84. The large arrowheads designate the major transcriptional start sites that mapped to basepair positions -20 <br /> and -18 (relative to the ATG; + 1 position), and the small arrowheads indicate minor start sites that mapped to base pair positions -26, -24, <br /> and -14, as determined by primer extension analysis. <br />  VOL. 12, 1992                                   suil, A TRANSLATION INITIATION SUPPRESSOR LOCUS IN YEAST                       253 <br />  <br /> acid +71 in the open reading frame and used to transform          mutants were identified as spontaneous His&apos; revertants (4). <br /> suil suppressor strains. Transformants containing plasmid         (ii) The Ts- and His&apos; suppressor phenotypes of suil mutants <br /> p1511 maintained the  same Ts- and His&apos; suppressor phe-           are unstable, and Ts&apos; His- revertants are easily obtained. <br /> notypes as the parent suil suppressor strains. In contrast,       (iii) Integration of the large BamHI DNA fragment from the <br /> the same strains transformed with SUIl+ parent plasmid            wild-type SUII clone (Fig. 1) as part of YIp5 at a number of <br /> p1128 were Ts+ and had either a reduced His+ suppressor           suil suppressor alleles does not alter the Ts- and His&apos; <br /> phenotype or were His-, depending on the suil strain. This        suppressor phenotypes of the suil suppressor mutant strains <br /> indicates that translation of this 108-amino-acid open reading    used for transformation. This latter observation indicates <br /> frame results in ability of this DNA fragment to complement       that each mutation that confers suppression maps either 3&apos; or <br /> suil mutants.                                                     very close to the BamHI site in the 108-amino-acid-coding <br />   To test the expression of this 108-amino-acid open reading      region of suil alleles. The basis for this conclusion is that the <br /> frame in S. cerevisiae, we constructed an in-frame fusion of      BamHI subclone contains only the 5&apos; noncoding region and <br /> the first 71 amino acids (Fig. 2) to the lacZ coding region, as   the first 71 amino acids of the wild-type 108-amino-acid- <br /> part of a 785-bp BglII-BamHI DNA fragment (Fig. 1). This          coding region (Fig. 2). Therefore, if substitution of this <br /> construct, when present in S. cerevisiae, produced a signif-      region by integrative transformation for the corresponding <br /> icant level of 13-galactosidase (258 U), indicating that the      region in the suil alleles has no affect on the Ts- or His&apos; <br /> 785-bp BglII-BamHI DNA fragment was capable of directing          phenotypes, then the mutation(s) that confers suppression is <br /> the transcription and translation of this 108-amino-acid open     not contained in the 5&apos; noncoding region or the first 71 amino <br /> reading frame. Immunoprecipitation of this fusion protein         acids of the coding region. Thus, genetic data suggest that <br /> from cell extracts with antibody raised against P-galactosi-      the Ts- and His&apos; phenotypes associated with suil suppres- <br /> dase identified a protein with the Mr (120,000) predicted for     sor strains are a result of a single base change that maps 3&apos; to <br /> the fusion protein. Gene disruption of this 108-amino-acid        amino acid position +71 in the 108-amino-acid-coding region <br /> open reading frame in a wild-type diploid strain and subse-       and agrees with our gap duplex and DNA sequence analyses <br /> quent genetic analysis of 77 tetrads from eight independent       of suil alleles (Fig. 2). In light of these and the above- <br /> diploid transformants resulted in 53 tetrads showing a two        described studies, we conclude that the 108-amino-acid open <br /> viable-two inviable segregation ratio, while the remaining 24     reading frame encodes the SUIJ gene product. <br /> tetrads had only one viable spore. All viable spores were            suil Ts- mutants have an in vivo defect in translation <br /> Ura-, indicating that disruption of the SUII gene by the          initiation. On the basis of the genetic selection scheme we <br />  URA3+ gene leads to cell inviability. Therefore, this 108-       employed (4) and the molecular analysis of the suil gene, it <br /> amino-acid-coding region is transcribed and translated in S.      seemed logical to assume that the suil gene product per- <br /> cerevisiae and encodes an essential gene product.                 formed an essential function during the translation initiation <br />    Characterization of the suil suppressor genes. suil suppres-   process in S. cerevisiae. In light of this assumption, one <br /> sor mutations were mapped and isolated by the gap duplex          simple interpretation of the Ts- growth phenotype of suil <br /> repair method (20). Plasmid p1199 was used to transform           strains is that at the restrictive temperature (37&#176;C) the Suil <br /> Ts- suil suppressor strains 117-8AR4 (suil-J), 302-2D (suil-      protein is thermolabile. As a result of this thermolability, the <br /> 4), 139-3C (suil-7), and 138-7B (suil-17) to Ura3+. This          Suil protein cannot perform its essential function during the <br /> plasmid contains the intact 108-amino-acid open reading           translation initiation process that leads to the inhibition of <br /> frame as part of a 5.4-kb DNA insert in YCp5O. Restriction        protein synthesis, which results in inhibition of cell growth. <br /> of this plasmid at a unique BamHI restriction site prior to          To test this possibility, duplicate cultures of a Ts- suil <br /> transformation yielded a higher frequency of Ts- transfor-        mutant were grown at 23&#176;C and then one culture was shifted <br /> mants than either the uncut plasmid or the plasmid restricted     to 37&#176;C for 2 h. Extracts were prepared from both of the <br /> at a unique PvuII restriction site. This BamHI site is within     cultures and fractionated by sucrose gradients, and poly- <br /> the 108-amino-acid open reading frame (Fig. 2), while the         some profiles were compared. Figure 3A shows the nearly <br /> PvuII site lies outside the open reading frame (Fig. 1). This     identical polysome profiles from two extracts prepared from <br /> higher incidence of Ts- transformants suggested that the site     a SUIJ+ control strain, one from a culture which had been <br /> of the mutation that confers the Ts- phenotype was nearer         grown only at 23&#176;C and the other extract derived from a <br /> the BamHI site and incorporated into the corresponding            culture that had been grown at 23&#176;C and then shifted to 37&#176;C <br /> DNA insert region of the plasmid during gap duplex repair of      for 2 h. In contrast, the polysome profile of an isogenic suil <br /> the restricted plasmid in S. cerevisiae.                          Ts- mutant grown in an identical fashion at 23&#176;C was similar <br />    Plasmids were isolated from these Ts- transformants, and       to that of the wild type but an aberrant profile was obtained <br /> the DNA sequence of the 108-amino-acid open reading frame         from the extract which was derived from a culture that had <br /> was determined. As expected, the DNA sequence analysis            been shifted to the restrictive temperature for 2 h (Fig. 3B). <br /> identified a single-base change in each allele that altered one   In essence, little remained of the larger polysomes at 37&#176;C <br /> of the amino acids in the 108-amino-acid open reading frame       and a significant increase in the 80S region of the gradient <br /> (Fig. 2). The sequence from the suil-I strain contained an        was observed. (This 80S peak appears to represent an <br /> aspartic acid (GAC)-to-<span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span> (GGC) change at amino acid         mRNA-independent association of 40S and 60S subunits.) <br /> position +83. The suil4 and suil-7 alleles both contained         The same polysome profile was also observed in an indepen- <br /> the identical base mutation that changed the same aspartic        dent Ts- suil mutant (data not shown), as well as a Ts- sui2 <br /> acid (+83) to a tyrosine (TAC). The sequence of the suil-17       strain (Fig. 3C) which encodes a defective a subunit of <br /> allele showed that amino acid position +84 had been               eIF-2, a protein known to be essential for translation initia- <br /> changed from a glutamine (CAA) to a proline (CCA) codon.          tion in S. cerevisiae and other eukaryotic organisms. The <br />    Aside from gap duplex repair and DNA sequence analyses         simplest explanation for the similar polysome profiles ob- <br /> of the mutant suil alleles, other experiments supported the       served in suil and sui2 mutants is that the suil gene also <br /> notion that a single base change in the 108-amino-acid-           encodes a protein that has an essential function during <br /> coding region was responsible for the Ts- and His&apos; suppres-       translation initiation. This product is thermolabile in the suil <br /> sor phenotypes of these suil mutants. (i) All suil suppressor     mutant strain, and therefore upon temperature shift it cannot <br />  254                 YOON AND DONAHUE                                                                                      MOL. CELL. BIOL. <br />  <br />                                                                              strain was purified and subjected to amino-terminal se- <br />                            <span id='am-69' about='obo:NCBITaxon_9903' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_27592 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9895 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="oxen, cattle" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Bos" datatype="xsd:string"></span>BOS</span>                230,                     370   quence analysis. The yields of phenylthiohydantoin amino <br />                                                                              acids from the first five cycles of the protein sequence when <br />                                                                  80S         an AUU initiator codon is present in the his4-lacZ message <br />           0O.5- <br />                                                                              are depicted as a bar graph in Fig. 4A. The predominant <br />                                                                              sequence, Pro-lle-Leu-Pro-Leu, matches the HIS4 coding <br />                                                                              region beginning at codon 4 in the message. These data <br />           01                                                                 parallel the sequence analysis of His4-p-galactosidase pro- <br />                                                                              tein produced from an AUU his4-lacZ message as a result of <br />                                                                              either sui2 or SUI3 suppression (7, 10). <br />            Bi.e <br />                      ~~~~~~~~~30,370                                            One interpretation of our amino-terminal sequence analy- <br />                                                                              sis of the His4-3-galactosidase protein produced from a <br />                                                                              mutant his4-lacZ message is that the standard methionine <br />                                                                              initiator tRNA is capable of initiating translation at a UUG <br />           ~0.5-                                                              codon as a result of the mutation in either a suil (Fig. 4A), a <br />            o                                                                 sui2 (7), or a SUI3 (10) suppressor gene. The methionine <br />           01                                                                 inserted is then cleaved in vivo by an aminopeptidase and <br />                                                                              therefore does not appear in the first cycle of the sequence <br />                                                                              analysis. However, given the facts that the sui2 and SUI3 <br />       C        .0                             2301                      7    genes encode the cx and ,B subunits of eIF-2 and eIF-2 is <br />                                                                              known to be involved in initiator tRNA binding, it is also <br />           0~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~               plausible, mechanistically, that mutations in either sui2 or <br />                                                                              SUJ3 alter the tRNA-binding activity of eIF-2. Perhaps the <br />           ~0.5-                                                              mutant forms of these proteins allow eIF-2 to bind a Leu <br />                                                                              tRNA and use it to initiate at a complementary UUG codon <br />            0                                                                 at amino acid position +3 in the early his4 coding region. <br />                                                                              Again, leucine might not appear in the first cycle of the <br />                     TOP                   Bottom Top                         protein sequence analysis, as an aminopeptidase would <br />                                                                              remove it from the fusion protein in vivo. The significance of <br />   FIG. 3. Polysome analysis. Duplicate cultures of either a wild-            this distinction is that in the former case the sui suppressor <br /> type strain, a Ts- suil suppressor strain, or a Ts- sui2 suppressor <br /> strain were grown to an OD600 of 1.0 at 23&#176;C. One of the duplicate <br />                                                                              genes would be involved in mediating ribosomal recognition <br /> cultures was then shifted to 37&#176;C, and both the 23 and 37&#176;C cultures         of the start codon in conjunction with the initiator tRNA, <br /> were incubated for two more hours. Cells from each culture were <br />                                                                              whereas in the latter case the suppressor genes may function <br /> then harvested, cell extracts were prepared, and 50 OD26, U from             only in binding the initiator tRNA and the complementary <br /> each extract was fractionated on a 7 to 47% sucrose gradient (35 ml)         anticodon-codon interaction would be the primary determi- <br /> by centrifugation in an SW27 rotor at 24,000 rpm for 4.5 h. Gradients        nant in establishing start site selection. <br /> were collected from the top by using an ISCO 640 density gradient               To differentiate between these two different mechanistic <br /> fractionator and monitored by the ISCO UA-5 absorbance-fluores-              interpretations of suppression, we sequenced the His4-3- <br /> cence detector. Gradients are plotted as OD260 units versus fraction         galactosidase fusion proteins produced from a suil strain and <br /> (1.2 ml) numbers from the top to the bottom of the gradient. The             a SUI3 strain that contained an AUU initiator codon muta- <br /> position of 80S ribosomes in each gradient is marked and was <br /> determined by probing the various fractions by Northern analysis as          tion in the his4-lacZ message and an additional codon change <br /> described in Materials and Methods. (A) Polysome profiles prepared           at position +4 in the early coding region of his4. The codon <br /> from SUII+ wild-type parent strain 117-8A grown at 23&#176;C (left) and           change substituted a phenylalanine codon (TTC) for the <br /> 37&#176;C (right). (B) Polysome profile of isogenic Ts- suil suppressor           normal proline codon (CCG) at amino acid position +4. The <br /> strain 117-8AR4 grown at 23&#176;C (left) and 37&#176;C (right). (C) Polysome          basic assumption of this codon alteration was that if methi- <br /> profile of isogenic Ts- sui2 suppressor strain 117-8AR20 grown at            onine was inserted at UUG, then by increasing the Stokes <br /> 23&#176;C (left) and 37&#176;C (right).                                                radius of the penultimate amino acid next to the UUG codon <br />                                                                              the ability of aminopeptidases to cleave methionine would be <br />                                                                              inhibited (28). Therefore, we could differentiate between <br /> perform its function. This leads to a block in translation                   suppression events that allow methionine initiator tRNA to <br /> initiation, which leads to accumulation of 40S and 60S                       initiate translation at UUG from another tRNA <span id='am-75' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. If <br /> subunits (14). However, translation elongation is not af-                    Met is inserted at UUG as a result of suppression, then the <br /> fected by the mutation in suil and therefore polysomes finish                penultimate Phe blocks its cleavage by aminopeptidase. This <br /> translating mRNA during the temperature shift, which leads                   would be noted by the appearance of Met as the predominant <br /> to a substantial reduction in the polysomes, as exhibited in                 amino acid in the first cycle of the protein sequence. <br /> each mutant profile.                                                            As shown in Fig. 4B, when phenylalanine is present at <br />    Mechanism of suil suppression. Previous studies in our                    amino acid position +4 in the his4 coding region in a suil <br /> laboratory demonstrated that the sui2 and SUI3 suppressor                    strain, the amino acid sequence is drastically changed com- <br /> genes that encode mutant ot and     subunits of eIF-2, respec-               pared with an isogenic strain that contains only the AUU <br /> tively, restored translation initiation to his4 and his4-lacZ                mutation at the his4 initiator region (Fig. 4A). The predom- <br /> initiator codon mutant strains by allowing initiation at a                   inant amino acid sequence obtained now reads Met-Phe-Ile- <br /> UUG codon present at amino acid position +3 in the early                     Leu-Pro (Fig. 4B). The identical sequence, Met-Phe-Ile-Leu- <br /> his4 coding region (7, 10). To determine whether mutations                   Pro, is also obtained for the purified His4-4-galactosidase <br /> in the suil suppressor gene also restore His4 expression by                  protein produced in a SUI3 suppressor strain (Fig. 4C) when <br /> a similar mechanism, the His4-3-galactosidase fusion pro-                    a phenylalanine codon is present at the +4 amino acid <br /> tein produced from the mutant his4-lacZ mRNA in a suil-I                     position in the his4 coding region of the same mutant <br />  <br />  <br />    FIG. 4. Amino-terminal sequence analysis of His4-3-galactosidase protein. The His4-p-galactosidase fusion protein synthesized from <br /> initiator codon-defective his4-lacZ suppressor mutants was purified to homogeneity by employing the previously described method (10) or a <br /> modification of this procedure as described in Materials and Methods. Purification of His4-,B-galactosidase on SDS-polyacrylamide gels <br /> resulted in a single protein band with a relative Mr of 116,000 that coelectrophoresed with either E. coli P-galactosidase or His4-1- <br /> galactosidase produced from an in-frame HIS4+-lacZ wild-type yeast strain. Amino acid sequence analysis was determined by automated <br /> Edman degradation using an Applied Biosystems 477A liquid-phase protein-peptide sequencer with an on-line 120A phenylthiohydantoin <br /> (PTH) amino acid analyzer. The results are presented as a bar graph plotted as the total yield of each phenylthiohydantoin amino acid <br /> derivative (single-letter amino acid designations) for the first five sequencing cycles per His4-,B-galactosidase protein sequenced. The results <br /> are interpreted in the lower right-hand quadrant relative to the known amino-terminal sequence Val-Leu-Pro-Ile-Leu (methionine [boxed] is <br /> cleaved posttranslationally) of the HIS4+ wild-type protein. (A) Amino-terminal sequence ( ) of His4-,B-galactosidase produced by suil-l <br /> suppressor strain 117-8AR4, which contains an AUU initiator codon change in the his4-lacZ message. (B) Amino-terminal sequence ( M ) <br /> of His4-p-galactosidase produced by yeast strain HY32. Yeast strain HY32 contains the same suil-l suppressor allele as strain 117-8AR4 (A) <br /> but differs from 117-8AR4 by an additional mutation in the his4-lacZ construct [his4(AUUIPhe)-lacZ] that changes the proline codon (CCG) <br /> normally present at amino acid position +4 in the HIS4 coding region to phenylalanine (UUC). (C) Amino-terminal sequence analysis ( ) <br /> of His4-p-galactosidase produced from yeast strain HY40. Yeast strain HY40 also contains the his4(AUUIPhe)-lacZ allele but differs from <br /> HY32 by being wild type for the SUIl gene and containing the SUI3-3 suppressor allele (10). <br />  <br />  <br /> his4-lacZ message. Obviously, methionine appears in the                      Biochemical characterization of the Suil gene product. As <br /> first cycle of the protein sequence as a result of the presence           an initial biochemical characterization of the Suil protein, <br /> of phenylalanine in the penultimate position relative to the              cell extracts made from SUIl+ and suil suppressor strains <br /> UUG codon at position +3. As a result of the situation of                 were fractionated and probed in Western blots with antibod- <br /> phenylalanine at this position, the methionine inserted at the            ies made from a TrpE-Suil fusion protein. Figure 5A (lane 1) <br /> UUG codon can no longer be cleaved by an aminopeptidase.                  shows that a crude extract prepared from a wild-type strain <br /> The simplest interpretation of these experiments is that the              cross-reacted with anti-TrpE-Suil and identified two protein <br /> effect of mutating either eIF-2P or Suil is to allow the                  species, one with an apparent Mr of less than 17,000 and a <br /> standard methionine initiator tRNA to recognize a UUG                     second with an apparent Mr of 26,500. Neither of these two <br /> codon as the site of translation initiation. Most importantly,            proteins was detected when preimmune serum was substi- <br /> a mutation in suil is qualitatively similar to a mutation in one          tuted for our anti-TrpE-Suil antiserum in Western blots <br /> of the known subunits of eIF-2, further demonstrating the                 (data not shown). By using a crude extract that was made <br /> importance of the Suil gene product for translation initia-               from a yeast strain containing the wild-type SUIJ gene on a <br /> tion, specifically, at the level of the start site selection              high-copy vector, we were able to distinguish which of these <br /> process.                                                                  two protein species corresponded to the Suil protein. As <br />  <br />  <br />  <br />   FIG. 5. Western blot analysis. Protein samples were electrophoresed on SDS-15% polyacrylamide gels with prestained molecular weight <br /> markers. The gels were transferred to nitrocellulose (0.2-.m pore size), blocked. and probed with rabbit antibodies (1:100) raised against <br /> TrpE-Suil protein produced in E. coli. Antigen-antibody complexes were visualized by using a horseradish peroxidase conjugate system. <br /> eIF-2 was partially purified from a wild-type SUII strain by 55 to 60% ammonium sulfate fractionation of a 0.5 M KCl wash of ribosomes and <br /> HAP column chromatography. Similar profiles were obtained for siuil mutants (data not shown). (A) Lanes 1 and 2 contained 150 p.g of crude <br /> extract from single-copy (SC) SUII wild-type strain BC64 and multicopy (HC) SUII wild-type strain BC61, which contains the 10.1-kb <br /> BamHI-BamHI-Sau3A DNA fragment (Fig. 1) in high-copy-number yeast vector YEp24. Lane 3 contained 150 pLg of a 0.5 M KCI ribosomal <br /> wash (RW) fraction from SUIJ wild-type parent strain 117-8A. (B) Lane 4 contained 150 p.g of a 55 to 60% ammonium sulfate fraction of a <br /> ribosomal salt wash from SUII wild-type strain 117-8A. Lanes 5 to 9 contained the HAP elution profile (100 to 600 mM KPO4 linear gradient) <br /> from SUII wild-type strain 117-8A (25 ,ug of protein per lane). For lanes 4 to 9, the filter was also probed with anti-Sui3 serum (1:100) and <br /> the position of the Sui3 protein (apparent Mr, 36,000; reference 10) is noted. <br />  <br />  <br />  <br />  <br /> shown in Fig. 5A, lane 2, the polyclonal antiserum cross-                Suil protein preparation that was used for immunizations. <br /> reacted in a specific fashion with the apparent 17-kDa                   Presumably, this protein contaminant is related in sequence <br /> protein, as the cross-reactivity responds to an increase in              to a 26.5-kDa protein from S. cerevisiae. In contrast, the <br /> SUIJ gene dosage. This is a somewhat higher apparent Mr                  TrpE-Suil fusion protein that is smaller by 17 amino acids <br /> than expected for the Suil protein but nevertheless consis-              and migrates at a slightly different position on polyacryl- <br /> tent with the calculated Mr (12,300) of the 108-amino-acid               amide gel electrophoresis gels is less contaminated with the <br /> sequence derived from the SUII gene (Fig. 2).                            E. coli protein when extracted, and hence, the correspond- <br />    The crude extract prepared from the yeast strain that                 ing antiserum cross-reacts weakly with the 26.5-kDa protein. <br /> contains the single chromosomal copy of SUII (Fig. 5A, lane              In addition, we attempted to determine whether the 26.5- <br /> 1) was further fractionated. The 26.5-kDa protein that cross-            kDa protein detected in Fig. 5A is a modified form of the <br /> reacts with our anti-TrpE-Suil serum (Fig. 5A, lanes 1 and 2)            17-kDa protein. In brief, we saw no evidence that this <br /> remained associated with the ribosomal pellet after a salt               protein is either a glycosylated or ubiquinated form of the <br /> wash of ribosomes and could be extracted from the ribosome               17-kDa protein that might account for the cross-reactivity <br /> by using methods for extracting tightly bound ribosomal                  and the Mr difference between the 17- and 26.5-kDa proteins. <br /> proteins (data not shown). The antiserum used for Fig. 5 was             A search of a DNA sequence bank allowed us to detect a <br /> directed against a TrpE-Suil fusion protein that fused amino             plant gene that is predicted to encode a gene product that is <br /> acid positions +54 to +108 in the distal SUIJ coding region              similar in length and sequence to the Suil gene product (see <br /> in frame to the carboxyl end of the trpE gene. However, a                below). In light of these observations and the ability to <br /> different antiserum made from a smaller TrpE-Suil fusion                 detect increased levels of the 17-kDa protein species in <br /> protein that fused amino acid positions +71 to +108 of the               response to an increased dosage of gene SUIJ, we conclude <br /> distal SUIJ coding region in frame to the carboxyl end of the            that the 17-kDa protein is the primary gene product encoded <br /> trpE coding region weakly cross-reacted with a 26.5-kDa                  by the SUIJ gene. <br /> protein species in crude extracts or ribosomal protein ex-                  As shown in Fig. 5A (lane 3), the 17-kDa Suil protein was <br /> tracts. This latter antiserum did cross-react with the 17-kDa            found in a salt wash fraction of a ribosomal pellet. We <br /> protein, and the cross-reactivity with the 17-kDa protein                estimated that greater than 60% of the Suil protein observed <br /> species also responded to an increase in gene dosage (data               in the crude extract was found in the ribosomal salt wash <br /> not shown). The detection level of the 17-kDa protein                    fraction (Fig. 5A, compare lanes 1 and 3); the remainder of <br /> species with this different antiserum was also comparable to             Suil was found in the postribosomal supernatant, and no <br /> the levels of detection shown in Fig. SA. We assume that the             Suil was detected in the pellet of salt-washed ribosomes <br /> differences in the patterns of cross-reactivity to the 26.5-kDa          (data not shown). For comparison, we estimated on the basis <br /> protein species observed with our two different antisera is              of Western blotting that approximately 10% of Sui2 protein <br /> that the antiserum directed against the larger TrpE-Suil                 (eIF-2a), a known subunit of yeast translation initiation <br /> fusion protein contained higher levels of antibodies against             factor eIF-2, was in a postribosomal supernatant with the <br /> an E. coli protein that contaminated our gel-extracted TrpE-             remainder in the ribosomal salt wash fraction and that <br />  VOL. 12, 1992                                         suil, A TRANSLATION INITIATION SUPPRESSOR LOCUS IN YEAST                                  257 <br />  <br />            20000                                                                Go32   1 tSDLDIQIPTRFDPFRERHRGDSGRARGSKDYUHURIQQ 39 <br />                                                                                 Suil   1 MS ... IEHLKSFDPFRD.   .TGDDE.TRTSH.YIHIHR*IQQ    32 <br />  <br />                                                                                        40 RHGRKSLTTUQGLKKEFSYHK LKDLKKEFCCHGTUUQD 78 <br />            15000 <br />  <br />  <br />                                                                                        33 RNGRKTLTTUQGUPEEYDLKR LKULKKDFRCHGN IUKO 71 <br />     c <br />                                                                                        79 PELGQUIQLQGDQRKHUSHFLU.QRGIUKKEHIKIHGF 115 <br />                                                                                          00 -   000000**   0    0    0   0   *-   000000 <br />     co     10000                                                                       72 PEIGEI IQLQGDQRRKUCEF1IISQLGLQKK.HIKIHGF 108 <br />     z <br />                                                                              FIG. 7. Comparison of the wild-type yeast SUII and rice plant <br />     r <br />      4,)                                                                  GOS2 amino acid sequences (single-letter code). The 108-amino- <br />     II <br />                                                                           acid sequence for the Suil protein is derived from the SUII+ DNA <br />             5000 <br />                                                                           sequence in Fig. 2 and is compared to a 115-amino-acid sequence <br />                                                                           which was translated from the DNA sequence listed in the GenBank <br />                                                                           data base for the GOS2 gene from the rice plant, 0. sativa <br />                                                                           (accession no. X51910) (7a). The amino acid positions are numbered <br />                                                                           relative to the methionine (+ 1) encoded by the predicted AUG start <br />                                                                           codon in both genes. The two proteins were aligned by introducing <br />                                                                           gaps within the sequence to maximize homology. Identical amino <br />                                     ug protein <br />                                                                           acids in the putative Gos2 gene product and the Suil protein are <br />    FIG. 6. In vitro assays of eIF-2 activity in partially purified SUII   designated by closed circles. The two asterisks mark amino acids <br /> wild-type and mutant extracts. Fractions from HAP chromatogra-            that are identical between the two sequences but also highlight the <br /> phy were assayed for the ability to promote GTP-dependent binding         two amino acid positions, +83 and +84, in the SUII sequence which <br /> of [3H]methionyl-labeled initiator tRNA as a function of the amount       were found to be altered by mutation in the various suil suppressor <br /> of protein. Mutant extracts were subjected to the same purification       alleles (see Fig. 2 and the text for details). <br /> scheme as shown for the wild-type extract in Fig. 5. The tRNA <br /> labeling and eIF-2 assay conditions used have been previously <br /> described (10), and the modifications used are mentioned in Mate- <br /> rials and Methods. Protein-[3H]tRNA complex formation was de-             wash fraction (Fig. 5A, lane 3) to determine whether the Suil <br /> tected by nitrocellulose filter binding as measured by liquid scintil- <br /> lation counting. Identical reaction conditions lacking GTP were           protein might be a subunit of eIF-2 in S. cerevisiae and <br /> performed to control for nonspecific or GTP-independent binding           assayed these fractions and corresponding fractions derived <br /> activity at each protein concentration assayed. The number of             from suil mutant strains for ternary complex formation (Fig. <br /> counts per minute in the absence of GTP was subtracted from the           6). As shown in Fig. SB (lane 4), the Suil protein was <br /> number of counts per minute in the presence of GTP to determine           observed in a 55 to 60% NH4SO4 fraction of the ribosomal <br /> GTP-dependent binding activity. Symbols: O, pooled HAP fractions          salt wash. We have previously shown that the 55 to 60% <br /> 12 to 15 (Fig. SB, lane 9) derived from wild-type strain 117-8A; A,       NH4SO4 fraction of a ribosomal salt wash contains eIF-2 as <br /> HAP fraction 13 derived from isogenic suil-I mutant 117-8AR4; *,          determined by Western blotting with anti-Sui2 (7) or anti- <br /> HAP fraction 13 derived from suil-17 mutant 138-7B. For compar-           Sui3 antibodies (Fig. SB; reference 10). Upon further frac- <br /> ison, fraction 10 from the HAP column shown in Fig. SB (lane 7), <br /> which contained most of the Suil protein when assayed for eIF-2           tionation of this 55 to 60% NH4SO4 fraction on a HAP <br /> activity (GTP-dependent binding of the initiator tRNA), had an            column, the Suil protein was no longer observed in the same <br /> activity of 51 cpm/p.g of protein. The activities of the other HAP        fractions that contained most of the Sui3 (eIF-2,) protein. <br /> fractions shown in Fig. SB are as follows: fraction 8 (lane 5), 0         Suil protein eluted slightly earlier from HAP (Fig. SB, lanes <br /> cpm/,ug; fraction 9 (lane 6), 20 cpm/,ug; fraction 11 (lane 8), 206       6 to 8) than did Sui3 (Fig. 5B, lanes 8 and 9). In addition, the <br /> cpm/,ug.                                                                  fractions that contained Sui3 protein also contained Sui2 <br />                                                                           protein whereas no Sui2 protein was detected in the fractions <br />                                                                           that contained the Suil protein as determined by Western <br /> approximately 50% of Sui3 (eIF-21) was in a ribosomal salt                blotting of these fractions with anti-Sui2 serum (data not <br /> wash fraction, with the remainder in the pellet of salt-washed            shown). The HAP fraction containing most of the wild-type <br /> ribosomes (data not shown). Thus, a ribosomal salt wash                   Suil protein (Fig. 5B, lane 7) possessed little or no eIF-2 <br /> fraction is enriched with Suil protein, a property consistent             activity (see the legend to Fig. 6 for details). The eIF-2 <br /> with our finding that the Suil protein functions as a transla-            activity (Fig. 6) was located in the latter fractions, where the <br /> tion factor during the initiation process.                                Sui2 (data not shown) and Sui3 proteins were most abundant <br />    As previously mentioned, aside from the suil suppressor                (Fig. 5B, lane 9). Assays of eIF-2 activity in similar HAP <br /> gene, our genetic selection scheme also identified the sui2               fractions derived from extracts of suil mutants did not result <br /> and SUI3 suppressor genes as capable of restoring HIS4                    in reduced GTP-dependent binding activity for the initiator <br /> expression when no AUG start codon was present (4).                       tRNA (Fig. 6). Although the fractionation profiles we ob- <br /> Previous analyses showed that the sui2 (7) and SUI3 (10)                  served are properties consistent with the <span id='am-54' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> that the SUIJ <br /> genes encoded the a and P subunits of eIF-2, respectively.                gene encodes a protein that is involved in the translation <br /> As part of these studies, we demonstrated in vitro that (i)               process, the Suil protein does not appear to be a subunit of <br /> eIF-2a and eIF-2f copurify and that these fractions contain               eIF-2 that is required for ternary complex formation, as Suil <br /> the well-characterized biochemical activity of eIF-2,                     did not appear in HAP fractions which contain eIF-2 and <br /> namely, GTP-dependent binding of the initiator tRNA (ter-                 mutations in Suil did not alter eIF-2 ternary complex for- <br /> nary complex formation), and (ii) sui2 and SUI3 suppressor                mation activity. This is consistent with the established <br /> mutants confer in vitro defects in ternary complex <span id='am-111' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-112' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>-                 subunit requirement of eIF-2 for ternary complex formation <br /> tion. In light of the genetic relatedness of the suil, sui2, and          (GTP-dependent binding of initiator tRNA; 24). eIF-2 is <br /> SUI3 suppressor mutants and identical mechanisms of sup-                  composed of three subunits, a, ,B, and y, of which the -y <br /> pression (Fig. 4), we further fractionated the ribosomal salt             subunit is believed to be an approximately 49-kDa protein in <br />  258    YOON AND DONAHUE                                                                                         MOL. CELL. BIOL. <br />  <br /> both S. cerevisiae (1) and mammals (24). Therefore, al-            limited and genetically saturated spectrum of suppressor <br /> though the sui2 and SUI3 suppressor loci identified the a and      genes obtained by this genetic reversion scheme, suggested <br /> a subunits of eIF-2 and arose from the same genetic selec-         that this approach was highly specific in identifying gene <br /> tion as suil, the suil suppressor gene does not encode the y       products that function at a related step in the translation <br /> subunit of eIF-2. In light of other studies we have presented,     initiation pathway. <br /> we suggest that the SUIJ gene product encodes an additional           The molecular and biochemical characterization of the <br /> translation initiation factor that normally functions in con-      sui2 and SU13 genes and gene products validated the initial <br /> cert with eIF-2 and the initiator tRNA to arrive at ribosomal      assumptions of our genetic selection scheme and pointed to <br /> recognition of the start codon.                                    the related and specific functions of these gene products <br />   The Suil protein is homologous to an amino acid sequence         during the translation initiation process. The fact that the <br /> derived from the GOS2 DNA sequence. Searches of protein            sui2 (7) and SUI3 (10) genes encode the a and a subunits of <br /> sequence data bases identified no previously characterized         eIF-2, respectively, demonstrated that aside from initiator <br /> gene product related to the SUIJ gene product. However, a          tRNA-binding activity (13), eIF-2 has an additional function <br /> search of the GenBank DNA data base with the SUIJ + DNA            during the translation initiation process, i.e., ribosomal <br /> sequence indicated that the SUIJ DNA sequence was related          recognition of a start codon. This is consistent with the <br /> to the GOS2 gene from the rice plant, <span id='am-35' about='obo:NCBITaxon_4530' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_147367 obo:NCBITaxon_359160 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_147380 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_4527 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:19706531" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:17907810" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Oryza sativa" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:389215" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="red rice" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:17617555" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="rice" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:15654106" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Oryza sativa</span> (acces-         known in vitro properties of eIF-2, which binds the initiator <br /> sion no. X51910) (7a). Translation of the DNA sequence of          tRNA and remains closely associated with the scanning <br /> the GOS2 gene and comparison of this 115-amino-acid open           ribosome up until the time of 80S complex formation. <br /> reading frame to the Suil amino acid sequence indicated that       Furthermore, suppressor mutations in SUI3 were shown to <br /> Suil is 61% identical to the putative GOS2 gene product            map to a putative zinc finger motif, suggesting that a nucleic- <br /> (Fig. 7). Interestingly, the mutations identified in suil sup-     acid-binding domain as part of eIF-2 was involved in stabi- <br /> pressor alleles (Fig. 2) alter amino acids that are identical in   lizing the codon-anticodon interaction between the AUG and <br /> the same relative position in the putative GOS2 protein            the initiator tRNA during the recognition process. The net <br /> sequence (Fig. 7). The observation that GOS2 is similar to         effect of suppressor mutations in SUI3 would then be to <br /> SUIJ in coding length and amino acid sequence further              increase the retention time of the ribosome at the UUG <br /> corroborates our assignment of the 108-amino-acid sequence         codon in the early HIS4 coding region by stabilizing a <br /> and the 17-kDa protein observed in Western blots as the            mismatched codon-anticodon interaction between UUG and <br /> primary Suil gene product, as a similar-size gene product          the initiator tRNA. Suppressor mutations in sui2 might also <br /> must be present in rice and encoded by GOS2. Unfortu-              afford such direct effects or, alternatively, alter a function <br /> nately, aside from the fact that it is a constitutively ex-        through a protein-protein interaction to arrive at non-AUG <br /> pressed gene, the function of GOS2 in rice is not known.           initiation events. <br /> However, given the significant identity and the similarity in         It therefore seemed logical to expect that the third and last <br /> the lengths of these two amino acid sequences, we anticipate       suppressor gene to be characterized from our genetic rever- <br /> that the GOS2 gene product is the plant homolog of the yeast       sion analysis of haploid yeast strains, suil, might encode an <br /> Suil protein and has a similar function during the translation     additional protein required for ribosomal recognition of a <br /> initiation process in plants.                                      start codon, perhaps the y subunit, the third and last subunit <br />                                                                    of eIF-2, as defined by being required in addition to a and ,B <br />                          DISCUSSION                                to promote ternary complex formation (GTP-dependent <br />                                                                    binding of the initiator tRNA). However, the data presented <br />    By using a genetic reversion scheme at the HIS4 locus in        in this report are inconsistent with the notion that the suil <br /> S. cerevisiae, we attempted to identify by mutation compo-         gene encodes the y subunit of eIF-2. (i) The Suil protein is <br /> nents of the translation initiation complex that are important     too small to be the y subunit of eIF-2. The Suil protein has <br /> for ribosomal recognition of an initiator codon. The basic         an apparent Mr of less than 17,000 (Fig. 5), compared with <br /> strategy was that mutant components would confer altered           the apparent Mr of eIF-2-y, which is reported to be in excess <br /> specificities on the translation initiation complex. Thus, the     of 49,000 in both S. cerevisiae (1) and mammalian (24) cells. <br /> ribosome would no longer bypass a mutant his4 initiator            (ii) The Suil protein did not copurify with eIF-2 fractions <br /> region but would recognize and initiate at a non-AUG codon,        that maintain GTP-dependent binding activity for the initia- <br /> restoring a His&apos; phenotype. Genetic analysis of 67 extra-          <span id='am-106' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-107' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-108'  ></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> tRNA (Fig. 5). (iii) eIF-2, when partially purified from <br /> genic suppressor mutants isolated from haploid yeast strains       suil mutants, still maintained wild-type levels of ternary <br /> identified only three unlinked suppressor loci, suil, sui2, and    complex formation activity (Fig. 6), in contrast to eIF-2 <br /> SUI3, each of which, when mutated, was capable of restor-          derived from sui2 (7) or SUI3 (10) mutants, which show <br /> ing HIS4 expression in the absence of an AUG start codon           either a decrease in or complete loss of this activity in vitro. <br /> (4). As part of this analysis, other genetic studies suggested        This leads to the suggestion that the suil gene encodes an <br /> that the gene products encoded by the suil, sui2, and SUI3         additional factor that acts in concert with eIF-2 at a subse- <br /> genes are specific in their suppression mechanism and share        quent step in the translation initiation process, namely, <br /> a related function. (i) Each suppressor mutant was incapable       ribosomal recognition of a start codon. Our data are consis- <br /> of suppressing known frameshift, missense, and nonsense            tent with this notion. Conditional suil suppressor mutants <br /> mutations but was capable of suppressing all possible point        exhibit altered polysome profiles, as expected of a gene <br /> mutations of the AUG start codon at his4. (ii) Each suppres-       product that functions during the early steps of translation <br /> sor was incapable of suppressing a his4 allele that contained      initiation (Fig. 3B). In fact, this profile parallels the poly- <br /> a deletion from positions -4 to +9, suggesting that the site of    some profile of a conditional sui2 mutant which encodes a <br /> suppression immediately surrounds the mutant initiator re-         thermolabile eIF-2a gene product known to function during <br /> gion. (iii) All pairwise combinations of suil, sui2, and SUI3      the initiation step (Fig. 3C). Control experiments (data not <br /> suppressor genes in haploid cells conferred lethality, indi-       shown) demonstrated that under the experimental conditions <br /> cating that all three gene products either functionally interact   described, a Ts- mutant unrelated to translation had identi- <br /> or function in a related pathway. These data, in light of the      cal and normal polysome profiles at both the permissive and <br />  VOL. <br />   V. 12, 19929sil A TRANSLATION INITIATION SUPPRESSOR LOCUS IN YEAST                                                               259 <br />  <br /> restrictive temperatures. This indicates that the alteration in    isms. We have also examined the suiil gene for a number of <br /> polysome profiles observed with suil and sui2 mutants at the       characteristics that are frequently found associated with <br /> restrictive temperature is specific for the functions of the       ribosomal genes (32). The suil gene is not duplicated, nor <br /> encoded gene products and is not an indirect effect of a Ts-       does it contain an intron. It is a small protein, as are many <br /> mutation. Another part of our analysis that supports our           ribosomal proteins, and has a somewhat basic amino acid <br /> suggestion that the suil gene encodes a factor that functions      composition (15% Arg or Lys). The suil gene does appear to <br /> during start codon recognition stems from amino-terminal           possess an Abfl-like upstream activation site sequence in its <br /> sequence analysis of the His4-f-galactosidase protein. As          promoter; this sequence has been found only in the promoter <br /> observed for sui2 (7) and SUI3 (10) mutants, suil suppressor       regions of approximately four yeast ribosomal protein genes <br /> strains result in initiation at a UUG codon in the early his4      (32). There appear to be a number of sequences in the suil 5&apos; <br /> coding region (Fig. 4A). Most importantly, by changing the         noncoding region that could be candidates for the Rapl <br /> penultimate amino acid next to this UUG codon from a Pro           DNA-binding site which is more frequently observed among <br /> to a Phe codon, we were able to demonstrate that the amino         promoter regions of ribosomal protein genes (32). Although <br /> acid corresponds to Met, which indicates that the initiator        it would be most interesting if Suil were a ribosomal protein, <br /> tRNA is used to initiate at the UUG codon. This observation        as no structural component of the eukaryotic ribosome has <br /> is important for a number of reasons. (i) It shows that the        ever been directly implicated in translation initiation, our <br /> effect of mutating suil does not result in an aberrant mech-       initial biochemical analysis of the Suil gene product ap- <br /> anism of translation initiation at his4 but rather the normal      peared to rule out this possibility, as Suil was completely <br /> translation initiation machinery is used. In light of other data   dissociated from the ribosome by a salt wash (Fig. 5A). <br /> presented, this further suggests that the Suil protein func-          This leads us to believe that suil encodes a translation <br /> tions during the mechanism of translation initiation in S.         initiation factor. Given the apparent M, of the Suil protein, <br /> cerevisiae. (ii) The observation that SUI3 suppression             calculated as part of this analysis, we compared this to the <br /> events also utilize the initiator tRNA to initiate at UUG (Fig.    apparent Mrs listed for the approximately 25 proteins be- <br /> 4C) shows that the mechanism of suppression is common to           lieved to compose the various mammalian translation initia- <br /> suil and SUI3 suppression, and therefore, the Suil protein         tion factors (24). Three proteins possess MrS consistent with <br /> appears to have a function related to recognition of a start       that of Suil: eIF-1, 15,000; eIF-lA, 17,600; eIF-5A, 16,700. <br /> codon by eIF-2. This is in complete agreement with our             Of these, we know that Suil is not eIF-5A, as two duplicated <br /> genetic studies of suil, sui2, and SUI3, as mentioned above,       genes that encode yeast eIF-5A, TIFSIA and TIFSIB (26), <br /> which suggested that the suppressors employ a similar              have been isolated and neither gene is the suiil locus. In <br /> mechanism of suppression and functionally interact during a        addition, we probed mammalian eIF-1 (kindly provided by <br /> related steps of the translation initiation process. (iii) This    J. W. B. Hershey) with anti-Suil and found no cross- <br /> amino-terminal sequence analysis makes an important mech-          reactivity, although the absence of cross-reactivity might be <br /> anistic distinction for the role of eIF-2 during the initiation    expected, given the evolutionary distance between S. cere- <br /> process. The observation that a mutation in eIF-2 affords a        visiae and mammals. Therefore, the Suil protein could be <br /> mismatched anticodon-codon interaction between the initia-         the yeast equivalent of eIF-1 or eIF-lA, which may make <br /> tor tRNA and UUG, as opposed to allowing a Leu tRNA to             sense, as both factors have been implicated in stimulation of <br /> initiate at UUG, represents more definitive evidence that          the 40S preinitiation complex (27. 31). If this were the case, <br /> eIF-2 functions during start site selection. Clearly, eIF-2        the observation would prove to be quite interesting, as little <br /> activity is not limited to tRNA binding.                           is known about either of these two factors. Neither has been <br />    One simple interpretation of our analysis is that Suil          considered to be required for in vitro translation initiation, <br /> functions with eIF-2 and the initiator tRNA to establish           whereas Suil clearly performs a required role in vivo. <br /> initiation at an AUG codon. Although our data appear to            Alternatively, Suil may represent a translation initiation <br /> eliminate the possibility that the Suil protein is a subunit of    factor that has not been identified by biochemical studies. In <br /> eIF-2 that is required for ternary complex formation, they do      any event, our analysis strongly implicates a function for the <br /> not rule out the possibility that it is part of a mutimeric        Suil gene product during ribosomal recognition of a start <br /> complex that includes eIF-2 and the initiator tRNA that            codon. These studies further point to the effectiveness of a <br /> dictates the recognition process. Our genetic observations         genetic system for identification of components of the trans- <br /> may, in fact, suggest this; specifically, the data that show       lation initiation complex that act at a specific step during the <br /> that suil sui2 and suil SUI3 double mutants confer lethality       scanning process in S. cerevisiae. <br /> on haploid yeast strains (4). The cofractionation and chro- <br /> matography of Suil relative to eIF-2 (Fig. 5) may indicate a                           <br /> weak interaction that is easily dissociated on a HAP column.         <br /> Our attempts to show a specific interaction by coimmuno-            <br /> precipitation of Suil protein with either anti-eIF-2a or anti-     <br /> eIF-2p serum or coimmunoprecipitation of eIF-2a or eIF-2p          <br /> protein with anti-Suil serum have not been successful (un-       <br /> published data). However, these experiments may have                  <br /> limitations, as our antiserum directed against eIF-2a, does <br /> not coimmunoprecipitate eIF-2p although anti-eIF-2, does <br />                                                                                               <br /> precipitate eIF-2cx. More extensive biochemical studies are          <br /> needed to define the function of Suil and its relationship with        <br /> other translation initiation components.                               <br />    Sequence searches indicate that the suil gene is not one of         <br /> the approximately 35 ribosomal protein genes that have been           <br /> cloned from S. cerevisiae, nor does Suil appear to be related     <br /> to other known ribosomal proteins identified in other organ-         <br /> </body></html>