<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 1994, p. 6361-6371                                                                            Vol. 14, No. 9 <br /> 0270-7306/94/$04.00+0 <br /> Copyright &#169; 1994, American Society Microbiology <br />  <br />  <br />         Identification cut8+ cekl +, Novel Protein Kinase Gene, <br />                Which Complement Fission Yeast Mutation That <br />                                  Blocks Anaphase <br />                                        ITARU SAMEJIMA AND MITSUHIRO YANAGIDA* <br />                      Department Biophysics, Faculty Science, Kyoto University, Sakyo-Ku, Kyoto 606-01, Japan <br />                              Received 28 March 1994/Returned modification 16 May 1994/Accepted 8 June 1994 <br />  <br />                The fission yeast <span id='am-30' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span>Schizosaccharomyces</span> cerevisiae temperature sensitivity cut8-563 mutation causes chromo- <br />             overcondensation short spindle formation absence sister chromatid separation. The <br />             cut8-563 mutation allows cytokinesis completion anaphase, producing cells cut <br />             phenotype. The cut8+ gene product required normal progression anaphase. Diploidization occurs <br />             restrictive temperature, 60 70%o cells surviving generations diploid. These <br />             phenotypes reminiscent budding yeast (<span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>) cdf13 cdf4 (ndclO) <br />             mutations. The cut8+ gene, isolated complementation mutant, predicts 262-amino-acid protein; the <br />             amino carboxy domains hydrophilic, central domain contains hydrophobic stretches. <br />             It weak overall similarity budding yeast DBF8 gene product. DBF8 essential gene whose <br />             mutations result delay mitotic progression chromosome instability. Anti-cut8 antibodies detect a <br />             33-kDa polypeptide. Two multicopy suppressor genes cut8-563 identified. They cut1+ gene <br />             essential nuclear division, new gene (designated cekl+) encodes novel protein kinase. The <br />             cekl + gene product unusually large (1,309 amino acids) 112-amino-acid additional sequence the <br />             kinase domain. The cekl+ gene essential gene. Protein phosphorylation cekl facilitate the <br />             progression anaphase direct indirect interaction cut8 protein. <br />  <br />    DNA replication, nuclear division, cell separation                    spindle extension (12), highly limited sister <br /> well coordinated normal cell cycle. In eukaryotic                      chromatid separation sufficient triggering onset <br /> cells, DNA replication precedes nuclear division fol-                     cytokinesis. As approximately 20 cut&apos; genes, the <br /> lowed cytokinesis. Each daughter cell receive                    causes deregulating cell cycle variable. <br /> one nucleus containing correct set chromosomes. A group                        Some cut&apos; gene products identified (18, 21, <br /> of temperature sensitivity (ts) fission yeast mutations called cut               41, 53, 54). They related DNA replication, DNA <br /> (cell untimely torn [21]) disrupt coordination cell                  topology, mitotic chromosome condensation segregation, <br /> cycle. At restrictive temperature (36&#176;C), nuclear division                spindle formation spindle pole body duplication. In <br /> these mutants defective, cytokinesis bisects                    mitotic mutants nda2, nda3, disl, dis2, dis3, nuc2, <br /> nucleus middle cell follows.                                       sds22, piml, mts2 (14, 22, 23, 32, 37, 38, 46, 50), cells <br />    Nineteen genetic loci displaying cut phenotype                       display defective mitosis cut phenotype. In some, <br /> been identified (21, 43, 53). In cut mutants,                     typified nuc2 mutant (22), septum made; the <br /> mitotic event defective shift restrictive                     septation occur cytokinesis blocked nuc2 <br /> temperature (18, 21, 53, 54). In rad4lcut5 mutant, however,                  mutant. In mutants, inhibition mitotic events causes <br /> the division normal second                      block subsequent cell division. The negative feedback <br /> defective (41). <span id='am-47' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> principal defect progres-                control following postanaphase events appears be <br /> sion S phase occurs cell division                  exerted certain mitotic mutants. <br /> (growing <span id='am-35' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-36' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> cells largely G2).                        Here, report characterization cut+ <br />    There possible ways explain major                      genes, namely, cut8+. The cut8-563 mutant defective the <br /> events coordinated cut mutants. A checkpoint                          progression mitotic anaphase, displaying condensed <br /> control (55), monitors progression mitosis                     chromosomes partially extended spindle. The cut8+ gene <br /> cytokinesis, defective cut mutants. Alternatively,                   required normal progression anaphase. <br /> the loss cut&apos; gene function detected the <br /> checkpoint system. It unknown, however,                        Another phenotype cut8-563 production diploid <br /> checkpoint exists fission yeasts delay cytokinesis                  cells restrictive temperature. An increasing number of <br /> mitotic event disturbed. Another possibility cut&apos;                 diploid cells formed restrictive temperature, suggest- <br /> gene product executes function cells passed                 ing sets daughter chromosomes formed <br /> stage commits cytokinesis (21). For example,                       separated. A genomic DNA clone derived <br /> sister chromatid separation trigger cytokinesis despite                  cut8+ locus complement cut8-563 was <br /> fact separation incomplete. In case DNA                       isolated. Nucleotide sequencing showed cut8+ gene <br /> topoisomerase II mutant (top2), small portion chromo-                       encodes 262-amino-acid protein. It similar budding <br /> some DNAs (that is, centromere DNAs) separated                         yeast DBF8 gene product, mutations increase rates <br />                                                                                  chromosome loss disjunction (24). We identified a <br />                                                                                  multicopy suppressor gene cut8-563 mutation, which <br />                                                                                  encodes novel protein kinase designated cekl+ (cut eight <br />   * <br />       Corresponding author. Phone: 81 75 753 4205. Fax: 81 75 753 4208.          suppressing kinase) large size (1,309 amino acids). It <br />                                                                           6361 <br />  6362     SAMEJIMA AND YANAGIDA                                                                                     MOL. CELL. BIOL. <br />  <br /> may play positive role completing anaphase directly    (47), contained bacteriophage T7 gene 10 promoter, <br /> indirectly control cut8+ gene product. In Discussion,      NdeI, site terminator T7 RNA polymerase. The <br /> we suggest role cut8 protein progression mitotic   resulting plasmid, designated pCUT8-23, produce the <br /> anaphase.                                                          cut8+ gene product, 262-amino-acid polypeptide. The plas- <br />                                                                    mid introduced E. coli BL21(DE3), contained the <br />                 MATERUILS AND METHODS                              T7 RNA polymerase gene control lac UV5 <br />                                                                    promoter. For purification fusion protein, proce- <br />    Strains media. Strains used haploid diploid        dures described Marston et al. (31) followed. E. coli <br /> wild types (16, 29), cut8-563 (21), HM123 (h- leul), YS106-1D      BL21 (DE3) harboring pCUT8-23 grown LB medium <br /> (h- ura4-D18), TP4-5A (h- leul ura4-D18 ade6-210),             containing ampicillin (100 ,ug/ml), induced 1 mM isopropyl- <br /> TP4-1D (h+ leul his2 ura4-D18 ade6-216). The ura4+ gene          3-D-thiogalactopyranoside (IPTG), incubated 2 4 h. <br /> deleted ura4-D18 (15). Diploid strains used gene disrup-    Cells pelleted lysed. The cell lysate fractionated, <br /> tion cross TP4-5A TP4-1D. S.            inclusion body purified. The inclusion body was <br /> pombe grown YPD (complete rich medium; 1% yeast             subjected SDS gel electrophoresis. The gel band containing <br /> extract, 2% polypeptone, 2% glucose) EMM2 (minimal              33-kDa cut8 polypeptide excised, polypeptide <br /> medium [34]). Agar (1.5%) added plates. An SPA plate       electroeluted gel. Rabbits immunized a <br /> (1% glucose, 0.1% KH2PO4, 0.001% biotin, 0.0001% calcium           procedure described (20). Antisera affinity <br /> pantothenate, 0.001% nicotinic acid, 0.001% meso-inositol,         purified antigen immobilized nitrocellulose filter <br /> 3% agar) used sporulation. Phloxine B (10 ,ug/ml;          (45). 12CA5, anti-hemagglutinin antigen (HA) tag mono- <br /> Sigma) included YPD plates. The standard genetic            clonal antibody purchased (BAbCO, Berkeley, Calif.) (11). <br /> procedures described S. pombe Gutz et al. (16)          S. pombe cell extracts prepared described Hirano et <br /> Moreno et al. (36) followed. The <span id='am-60' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-61' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> strain      al. (22). Briefly, cells broken glass beads buffer (20 <br /> used MM294, unless mentioned. E. coli            mM Tris [pH 7.5], 1 mM dithiothreitol, 1 mM phenylmethyl- <br /> grown LB (0.5% yeast extract, 1% NaCl, 1% polypeptone           sulfonyl fluoride), cell debris removed centrif- <br /> [pH 7.5]). Agar (1.5%) added plates. Ampicillin (100       ugation 5,000 rpm 5 min. The whole-cell extract thus <br /> ,ug/ml) included plasmid maintenance.                      prepared run SDS-polyacrylamide gel electrophoresis <br />    DNA. All DNA manipulations performed standard           (PAGE) electrophoretically transferred nitro- <br /> procedures (42). Isolation S. pombe genomic DNA             cellulose filter. The filter used immunoblotting <br /> recovery plasmid DNA S. pombe cells described         (51). <br /> by Moreno et al. (36). The nucleotide sequence deter-             Cloning multicopy suppressor genes. A genomic S. pombe <br /> mined dideoxy method (44).                                  library constructed pDB248. Sau3AI partial digests of <br />    Fluorescence microscopy. The procedure described 4&apos;,6-      genomic DNA (average length, 10 kb) ligated a <br /> diamidino-2-phenylindole (DAPI) staining Adachi             shuttle vector, pDB248, contains pBR322, S. cerevi- <br /> Yanagida (1) followed. Immunofluorescence microscopy           siae LEU2 gene, 2,um DNA (4,5). This genomic DNA <br /> was performed method described Hagan Hyams           library used transformation cut8 mutant strain by <br /> (17). The anti-tubulin monoclonal antibody, TAT1 (56),         lithium acetate method (25). A Leu+ Ts&apos; transformant <br /> used primary antibody microtubule staining.               obtained, Leu+ marker cosegregated Ts&apos;. A <br />    Cloning cut8+ gene. The cut8+ gene cloned,           plasmid (designated pS81) recovered transfor- <br /> taking advantage fact cut8 mutation tight    mant. <br /> linkage nuc2. An S. pombe genomic DNA library                  Mapping cekl+ gene. Chromosomal location the <br /> screened Southern hybridization, using nuc2+ gene      cekl+ gene determined physical mapping method. <br /> probe. The S. pombe cosmid library used Saccha-         Southern blot analysis ordered cosmid library (35) was <br /> romyces cerevisiae LEU2 gene selection marker. Cosmid         probed segment pS81. pS81 mapped contig <br /> clones transferred nitrocellulose membrane         chromosome III nearby ade6. <br /> probed 2.2-kb PstI fragment pNC106 (22) encom-              FACS. Fluorescence-activated cell sorter (FACS) analysis <br /> passing nuc2+ gene. Hybridization performed          performed method Costello et al. (8). Cells fixed <br /> high-stringency conditions (65&#176;C 5 x SSC [lx SSC 0.15 M      ethanol treated RNase prior stained with <br /> NaCl plus 0.015 M sodium citrate]-0.3% sodium dodecyl              propidium iodine. Fluorescence measured Beckton <br /> sulfate [SDS], followed wash 2x SSC-0.5% SDS           Dickinson FACScan apparatus. <br /> three successive washes 0.2x SSC-0.5% SDS). Three               Gene disruption cekl+. The 4-kb BglII fragment from <br /> 5,000 clones positive. The positive clones used      cekl+ DNA replaced 1.8-kb ura4+ gene, a <br /> transformation cut8 mutant lithium acetate           4.8-kb fragment construct used transform a <br /> method (25).                                                       haploid uracil-auxotrophic strain. Eleven transformants <br />    Mutation site determination. The mutant gene ampli-         screened stable Ura+ phenotype, indicating the <br /> fied PCR mutant genomic DNA sequence. The              disrupted gene stably integrated. All strains <br /> PCR primers used 5&apos;-GTCCGACTAGTGGTGCC-3&apos;                      analyzed Southern hybridization carry the <br /> and 5&apos;-TTGCTAGCT`I7CATAAG-3&apos;. The resulting PCR                    disrupted cekl gene. <br /> product used template asymmetric PCR            Nucleotide sequence accession number. The nucleotide se- <br /> was carried primer reduced 1/100 concentra-      quences insert pCUT8-15 cekl + gene product <br /> tion other, <span id='am-57' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> generate templates subsequent   submitted EMBL database accession <br /> dideoxy sequencing.                                                numbers D31772 D31773, respectively. <br />    Preparation antibodies. Plasmid pCUT8-23 for <br /> expression cut8+ gene product bacteria. An NdeI site                                RESULTS <br /> was introduced initiation methionine <span id='am-52' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span>. The 1.0-kb <br /> NdeI-BglII fragment contained entire coding region       Cell inviability cut8-563. The cut8-563 mutant invia- <br /> cut8+ ligated NdeI BamHI sites pAR3040         ble 36&#176;C, small normal-size colonies, respectively, <br />  VOL. 14, 1994     A <br />                                                                                               c                 FISSION YEAST cut8+ GENE                   6363 <br />  <br />  <br />                                                                                 100 <br />  <br />  <br />  <br />                                                                                          _. <br />  <br />  <br />  <br />  <br />                                                                                           * <br />                                                                                   50 <br />                                                                                          0- <br />  <br />  <br />  <br />  <br />                                                                                     0 <br />                               0               5                    10 <br />                                              Time (hr) <br />                  B <br />                     I--, <br />                       0 <br />                     0- <br />  <br />                     C) <br />                      a) <br />                      0 <br />                     LL <br />  <br />  <br />  <br />  <br />                      cr <br />                           1 <br />                     lL <br />  <br />  <br />  <br />  <br />                      0 <br />                      C: <br />                      CD <br />  <br />                     U) <br />  <br />  <br />  <br />                                                 2      3 <br />                                                 Time (hr) <br />    FIG. 1. cut8-563 mutant phenotype. (A) Cell number increase 36&#176;C. cut8-563 wild-type cells grown YPD medium 26&#176;C, then <br /> a portion transferred 36&#176;C cell number cultures measured 1-h intervals. Cells producing cut phenotype counted <br /> as two. Open squares, cut8-563 26&#176;C; filled squares, cut8-563 36&#176;C; open circles, wild type 26&#176;C; filled circles, wild type 36&#176;C. To estimate the <br /> viable cell number shift 36&#176;C, cultures plated 26&#176;C numbers colonies counted (broken line). (B C) Time course <br /> appearance different cell types temperature shift 36&#176;C. cut8-563 cells grown 26&#176;C shifted 36&#176;C. Aliquots cultures <br /> were taken 1-h intervals DAPI stained. Seven different types cells (C, numbers 1 7) scored: 1, interphase cells containing a <br /> single nucleus (filled circles panel B); 2, mitotic cells containing nuclei (filled triangles panel B); 3, postanaphase cells containing nuclei and <br /> septum (filled squares panel B); 4, mitotic cells containing condensed chromosomes (open circles panel B); 5, cells containing condensed (often <br /> extended) chromosomes septum (crosses panel B); 6, cells containing single decondensed nucleus septum (open squares panel B); 7, <br /> cells showing cut phenotype (open triangles panel B); wt, wild-type cell condensed chromosomes. Bar, 10 ,um. <br />  6364     SAMEJIMA AND YANAGIDA                                                                                        MOL. CELL. BIOL. <br />  <br /> formed 30 26&#176;C. In rich YPD liquid culture 26&#176;C,       suggesting cytokinesis occurred cut8-563 while <br /> generation time cut8-563 (3.5 h) significantly longer         spindle microtubules condensed chromosomes still <br /> than wild type (2.5 h) (Fig. 1A). At 36&#176;C,           present. <br /> generation time wild type 2.1 h. After temper-           Diploidization. In presence phloxine B, haploids with <br /> ature shift 36&#176;C, cell numbers cut8-563 continued           2C DNA content form white colonies diploids 4C <br /> increase 4 h, cells ceased divide (Fig. 1A);       DNA form red colonies YPD plates (16). At 26&#176;C, haploid <br /> cells divided twice shift restrictive       cut8-563 cells produced white colonies. While cells are <br /> temperature. Correspondingly, cell viability decreased 8%         inviable 5 h restrictive temperature, that <br /> by 5 h. These results suggested cells largely inviable     survived tend form red colonies plates containing <br /> after second division.                                           phloxine B (Fig. 3A; dark colonies red). We suspected that <br />    Anaphase cytokinesis defects cut8-563. Different           cells contain 4C DNA. <br /> types nuclear structure observed cut8-563 cells          We investigated appearance red colonies the <br /> 36&#176;C liquid YPD cultures DAPI staining (Fig. 1B        shift restrictive temperature. A log-phase culture of <br /> C). The frequency type 1 cells containing single interphase     cut8-563 shifted 36&#176;C plated 26&#176;C 1-h intervals. <br /> nucleus (Fig. 1B, curve 1, Fig. 1C, panel 1) high     As shown Fig. 3B, number white colonies (open <br /> permissive temperature 2 h restrictive      circles) increased 2 h rapidly decreased. Con- <br /> temperature (80 90%) sharply decreased 10%            versely, red colonies (filled circles) did appear 3 <br /> after 3 h. The type 2 cells containing nuclei prior           h accounted approximately 60 70% total <br /> septation type 3 cells septation 5 10%      colonies 5 10 hr. The ratio red total colonies is <br /> 26&#176;C dropped 1 4% 1 h shift          indicated broken line Fig. 3B. <br /> restrictive temperature.                                                 DNA contents cells derived red white colonies <br />    Two type cells (4 5) condensed chromo-          determined FACS analysis Beckton-Dickinson <br /> somes seen. Chromosomes type 4 appeared                FACScan. Results obtained independently <br /> overcondensed, reminiscent metaphase chromosomes.             isolated white red colonies shown Fig. 3C. The DNA <br /> Two percent population displayed metaphase conden-            contents white colonies corresponded 2C in <br /> sation 26&#176;C, number increased 12 15% 2        red colonies corresponded 4C. These red colonies were <br /> to 3 h 36&#176;C. Type S cells observed 26&#176;C           stably maintained 26&#176;C, large cell morphology was <br /> increased 10% 3 h 36&#176;C. These condensed chromo-          indistinguishable wild-type diploid light <br /> somes extended bisected septum          microscopy. <br /> middle. These structures precede cut phenotype (see              Isolation cut8+ gene. In order better understand the <br /> below). Normal anaphase structures hardly ob-             role cut8+ chromosome segregation, attempted to <br /> served cut8-563 36&#176;C 2 h.                                isolate cut8+ gene complementation ts phenotype <br />    Types 6 7 cells accumulated terminally cut8-563. Type      cut8-563. A genomic DNA library S. pombe trans- <br /> 6 cells, containing single displaced nucleus septum         formed cut8-563, Ts&apos; transformants se- <br /> (hereafter designated uncut phenotype), 15%           lected. Plasmid pS81 obtained Ts&apos; transformant <br /> the population. The nucleus decondensed inter-          able suppress ts cut8 mutants. However, <br /> phase nuclei. Eighty percent population displayed         analysis indicated derived locus unlinked <br /> cut phenotype (type 7). The cut phenotype accompanied            cut8+ (see Materials Methods) multicopy <br /> by cell division, uncut phenotype not. Both          suppressor cut8-563 (described below). <br /> types observed permissive temperature,               Previous genetic crosses indicated cut8 locus <br /> appeared 2 h restrictive temperature. The cut           closely linked nuc2 (1.9 centimorgans cut8 and <br /> phenotype make cells lethal, uncut pheno-         nuc2 [22]). We took advantage close linkage isolate <br /> type not.                                                      cut8+ gene. Both nuc2+ cut8+ reside or <br />    Short spindle formation failure chromosome disjunc-        adjacent overlapping cosmid clones. Cosmids hybridizing with <br /> tion. Microtubule distribution cut8-563 examined         nuc2+ probe obtained tested their <br /> monoclonal anti-tubulin antibody TAT1 (Materials Meth-           abilities complement ts phenotype cut8. The cosmid <br /> ods). Normal microtubule patterns observed cells             contained S. cerevisiae LEU2 (2) selection marker so <br /> grown 26&#176;C (Fig. 2A, left panel). After 3 h 36&#176;C, however,     isolated cosmids directly used comple- <br /> more cells displayed short metaphase spindle (Fig. 2A,           mentation cut8. <br /> upper right panel). Chromosomes cells                Three overlapping cosmid clones hybridized the <br /> condensed. In cells, chromosomes extended           nuc2+ probe obtained Fig. 4A), able to <br /> spindle clearly fail separate (Fig. 2A, lower right panel).   complement cut8-563 well. Cosmid Bi subcloned, a <br /> Sister chromatids partly separated cells.         1.5-kb BglII-HindIII fragment (pCut8-15) contained mini- <br />    The spindle length measured cells 26&#176;C (0 h)       mal complementable clone. The cut8+ gene located ap- <br /> at 36&#176;C 3 h (Fig. 2B). The average length 36&#176;C       proximately 15 kb nuc2+ gene. <br /> shorter (3.5 &#177; 1.7 ,um) (4.5 &#177; 2.4 ,um) 26&#176;C;         Predicted cut8 protein. Sequencing 2.7-kb BglII frag- <br /> short metaphase long anaphase spindles              ment containing minimal complementary clone revealed <br /> 26&#176;C, short metaphase spindle predominated 36&#176;C.           open reading frame (ORF) 786 bp (262 amino acids; <br /> These results demonstrated mitotic spindle               calculated molecular weight, 30,260 [Fig. 4B]). No other <br /> formed cut8-563 fully extended. Whether            reading frame present. A search EMBL and <br /> failure anaphase progression defect spindle      GenBank databases did yield known protein with <br /> elongation high frequency condensed chromo-              significant homology. A weak similarity sequence the <br /> somes determined. Furthermore, cells display-         S. cerevisiae DBF8 gene product, however, (12a, 24). <br />  ing X-shape postanaphase microtubule arrays,          The dbf8 mutant cell division cycle mutants <br /> seen prior normal cytokinesis (indicated arrow Fig.      previously identified (DBF stands dumbbell [27]). <br />  2A) (17), infrequent cut8-563 mutant 36&#176;C,            Twenty-five percent residues identical 208 <br />  VOL. 14, 1994                                                                                            FISSION YEAST cut8+ GENE               6365 <br />  <br />  <br />  <br />  <br />     A <br />         TUBULIN                          DAPI                             TUBULIN                            DAPI <br />  <br />                                    I <br />  <br />  <br />  <br />  <br />       B <br />  <br />               30-                                 Ohr <br />           0 <br />         0- <br />  <br />          C) <br />          CD   20- <br />          0r <br />          a) <br />                 10 <br />  <br />  <br />                 O- <br />                       1                    5                         10             1                    5                         10 <br />                                                           Spindle length (gm) <br />    FIG. 2. Short spindle formed cut8-563 36&#176;C. (A) cut8-563 cells grown exponentially 26&#176;C incubated 36&#176;C 3 h. Cells <br /> were collected, fixed, stained monoclonal anti-tubulin antibody TATI (56). DAPI used DNA staining. Left panels, cut8-563 cells <br /> before shift 36&#176;C (from bottom): interphase cell, mitotic cell short spindle, anaphase cell elongating spindle, anaphase <br /> cell fully extended spindle, postanaphase cells X-shape microtubule array. Right panels, cut8-563 cells 36&#176;C 3 h. Top, cells <br /> with short spindle condensed chromosomes: bottom, cells elongated spindle extended chromosomes. Bar, 10 p.m. (B) <br /> Spindle size distribution cut8-563 cells 36&#176;C. Spindle length cut8-563 cells incubated 36&#176;C 0 3 h measured. <br />  6366         SAMEJIMA AND YANAGIDA <br />  <br />  <br />  <br />  <br />              . <br />               * <br />                   ~w.~ ~ ~.~ ~ ~ X. ~ ~. <br />                     Oh r <br />                            ..... <br />  <br />  <br />  <br />  <br />                                        :    :   &apos;   i&apos;:....   &apos;   ... <br />                                                                         .... <br />                                                                                                                                     MOL. CELL. BIOL. <br />  <br />                                                                                amino acids (Fig. 4C). Similarity highest amino half <br />                                                                                protein. <br />                                                                                   The cut8-563 mutation site determined. The mutant <br />                                                                                gene amplified PCR method, entire nucle- <br />                                                                                otide sequence determined directly sequencing the <br />                                                                                PCR product (see Materials Methods). Comparison the <br />                                                                                wild-type sequence indicated nucleotide T postion <br />                                                                                1283 C PCR clone, alter 201st <br />                                                                                residue serine proline. This serine residue con- <br />                                                                                served DBF8 protein. <br />                                                                                   Identification p33C&quot;u8 immunoblotting. To identify the <br />                                                                                product cut8+, polyclonal antiserum raised against <br />                                                                                bacterially expressed full-length cut8 protein. A 33-kDa cut8 <br />                                                                                protein produced E. coli, purified, injected a <br />   B    106 <br />                                                                                rabbit. Immunoblot polyclonal anti-cut8 antibodies de- <br />                                                                                tected cut8 protein (Fig. 5A). The 33-kDa band wild-type <br />                                                                                fission yeast cells (lane 3) increased band intensity (lane 4) <br />                                                                                carrying multicopy <span id='am-93' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> cut8+ gene, <br />                                                                                recognized preimmune serum (lanes <br />        E                                                                       1 2). <br />                                                                                   We overexpressed cut8+ gene placing cut8 ORF <br />        0 <br />                                                                                control S. pombe promoter nmtl (33). In the <br />                                                                                absence thiamine, cut8+ gene overexpressed. A <br />                                                                                monoclonal antibody, 12CA5 (11), detects HA, was <br />        E                                                                       used. We constructed recombinant plasmid, pCUT8-20, that <br />        o <br />                                                                                contained cut8+ gene HA tag (52) carboxy <br />                                                                                terminus. pCUT8-20 rescued cut8-563 tagged cut8 <br />                                                                                protein functional. We examined cells overexpressing the <br />     .0 <br />     C                                                                          HA-tagged cut8+ gene nmt promoter grown EMM2 <br />                                                                                lacking thiamine (Fig. 5B). Those containing vector plas- <br />                                                                                mid (lane 1) plasmids pnmt-cut8 HA tag (lane <br />                                                                                3) gave band, 38-kDa band obtained cells <br />                                    Time (hr)                                   carrying pnmt-cut8 HA tag (lane 2). Thus, monoclonal <br />                                                                                antibody highly specific HA-tagged cut8 protein. <br />        0                                                                       This 38-kDa band recognized polyclonal <br />                                                                                anti-cut8 antibody (<span id='am-83' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown). The molecular mass <br />                        v                                                       difference bands polyclonal (33 kDa) and <br />                                                                                monoclonal (38 kDa) antibodies consistent size of <br />                                                                                HA-tagged cut8 protein. <br />        0 <br />                                                                                   Overexpression phenotype. Plasmid pnmt-cut8HA (pCUT8- <br />        E                                                                       20) used overexpress cut8+ wild-type cells. This <br />                                            White colony                        plasmid rescue cut8-563 36&#176;C presence the <br />                                                                                absence thiamine, confirming coding region repre- <br />                                                                                sented cut8+ gene leaky expression re- <br />                                                                                pressed conditions sufficient complementation (3). On <br />                                                                                EMM2 plates lacking thiamine, smaller-size colonies were <br />                                                                                formed. Cells overproducing cut8 protein largely normal, <br />                                                                                approximately 10% showed anomalous septation <br />                                                                                (data shown); multiple septation occurred elongated <br />                                             FL2                                cells, suggesting cytokinesis cells retarded. <br />                                                                                   Genetic interaction cutl+ cdcll. In addition to <br />                                    V                                           cut phenotype, cutl mutants tend diploid, <br />        t.0 <br />        a)                                                                      similar cut8 (9). We tested cut8+ any <br />        E <br />        n                                   Red colony <br />        0                           /~~~~~.                                     plated incubation 36&#176;C (O h). (B) Aliquots cut8-563 <br />                                                                                incubated 36&#176;C taken intervals plated YPD <br />                                                                                containing phioxine B 26&#176;C. The number red (filled circles) and <br />                                                                                white (open circles) colonies shown. The ratio red total <br />                                                                                colonies indicated broken line increased 3 h. (C) DNA <br />                                            FL2                                 contents cells derived red white colonies <br />                                                                                estimated FACScan. The DNA contents red colonies were <br />   FIG. 3. Diploidization cut8-563 36&#176;C. (A) Cells cut8-563            identical diploid cells. The peak positions wild-type <br /> grown 26&#176;C incubated 36&#176;C 5 h plated 26&#176;C           haploids diploids indicated small large arrowheads, <br /> on rich YPD medium containing phloxine B. Red colonies                respectively. Units cell number FL2 (fluorescent light intensity <br /> produced (5 h), red colony formed cells             DNA content) arbitrary. <br />             A <br />                                                                                                                                                                          10kb <br />                                                                      cut8 +                                    nuc2 + <br />                                                                             - <br />                                                                                                                    _                                                     BarnHI <br />  <br />                                      A7 <br />                                      C2                                                                                                                                                       + <br />                                      B1                                                                                                                                                       + <br />                                   pCUT8-1                                                                                                                                                     + <br />                                   pCUT8-8            -                                - <br />                                   pCUT8-9            - <br />                                   pCUT8-10                                                                                                                                                    + <br />                                                          I           Ii                       BgIII <br />                                                                                                                                                            1kb <br />                                                                          BgI 11                    InIIlKl                             +       Bgl 11 <br />                                                                           I                           ICUt8                                        1 <br />  <br />                                           pCUT8-10                                                                                                                            + <br />                                           pCUT8-14 <br />                                           pCUT8-15                                                                                                                            + <br />                                           pCUT8-21                                                                         .-                                                 + <br />                                           pCUT8-20                                                                                             HA                             + <br />  <br />             B <br />            HindIII 10        20        30        40       50        60        70        80        90       100       110       120 <br />            AAGCTTTGTC   ATTTTAGCATCACCAATAAGTTAGAATTGACTACTGTTACCAACAACAATTAGTTAAATAGTTATTGACTATAACATTGATCCATACGCACARTTACTTCGAAAA <br />            AAATTATTA=TTGCTCTTCCAACTTAACGA=TTCTGTACTGCGt TAACATAtTATTGTAGGCTTGACACTCAGCTGCGGTGACATTATATTAAATAAAATGTAATTTTTTCAGTAAAC <br />            AAATAAGGAATTAAGTAATAGTTGGTAACGTCTCATGAGAMGAAAACTCTTGAGCCACTAAATTGTTCTTACCATTTTCCATTATCTTTTTAAGAAATCGAGGTTCTAATTGAGCTGGT <br />            ATTGCGGGCAATAGAGCTTcTTGTAATATAGAAATTATAATcGAcGGTTrTTATTTCATTGGGcTGAGAAAAAAMTTAAATTrGcTAGCTTTcATAAGTcGATAcAcTGAGcAMTAGccAT <br />            CCTTGCCGGCTTTTAAAAAGGAAGCTCATTAACAAAGTATCTCTATCTTTGTTTATTGTTACTACTTTGAAA=TTTTCCTCTTTCTTTTGGAATATCCATTTCTTTGTAATTTTA <br />            TTTTCTTTGTTATAATTMATCAGACGA=TTACGTTTGAAGTTGGTATCmATTTA                                         TTGTTACTGTTCATTTCCTTAATTCATGGAGACCTTATCTTATTCTCAGATTAAGAAACGCAA                                             7270 <br />                                                                                                                                                        M   E     T   L    S       Y   S   Q       I   K   K   R   K     13 <br />  <br />            AGCTGATTTTGACGAAGACATTTCTAAGCGTGCTCGTCAACTACCTGTGGGTGAGCAACTGCCATTGTCGAGACTGCTACAATATTCAGACAAACAACAGCTTTTTACAATTTTATTACA                                                                                    840 <br />             A    D   F    D   E   D   I    S     K   R       A   R   Q     L      P   V   G   E      Q     L   P       L   S   R   L       L   Q       Y   S     D   K    Q       Q   L   F       T   I   L   L   Q     53 <br />  <br />            ATGTGTAGAAAAGCATCCTGATCTCGCACGTGATATACGGGGCATATTACCGGCGCCGTCCATGGATACTTGTGTTGAAACGTTGCGTAAACTTTTAATTAACTTGAATGACTCATTTCC                                                                                   9960 <br />              C   V   E    K   H   P   D    L     A   R       D   I   R     G I        L   P   A       P    S   M       D   T   C   V       E   T       L   R     K   L    L       I   N   L       N   D   S   F   P     93 <br />                                                                          Eco RI <br />             CTATGGTGGTGACAAGAGGGGTGATTACGCATTTAATAGAATGTGAAAAGTACATGGCTGTCTTACATGCGCTTAATGATATGGTTCCTTGTTATmTACCTCCATACTCTACTTGTTT                                                                                    1080 <br />              Y   G   G    D   K   R   G    D     Y   A       F   N   R     I      R   E   K    Y     M A V             L   H   A   L       N   D       M   V     P   C    Y       L   P   P       Y   S   T   C   F    133 <br />                                                                                                      Eco RI <br />             TGAGAAAAATATTACTTTTCTTGATGCCGCTACAACGTTGTCCACGACTGCCAGATTCCATATCCCAACCATAATGTATAAAAGTCGCGTACTATGAATTAACGGGCGCGTG                                                                                          1200 <br />              E   K   N    I   T   F   L    D     A   A       T   N   V     V      H   E   L    P     E     F   H       N   P   N   H       N   V       Y   K     S   Q    A       Y   Y   E       L   T   G   A   W    173 <br />  <br />             GCTTGTTGTTTTAAGACAACTTGAGGATCGGCCTGTAGTCCCTTTACTACCTTTGGAAGAGTTGGAAGAGCACAATAAAACATCACAAAACAGAATGGAAGAGGCATTAAATTACCTTAA                                                                                  1320 <br />             L    V   V    L   R   Q   L    E     D   R       P   V   V     P      L   L   P    L     E     E   L       E   E   H   N       K   T       S   Q N       R    M       E   E   A       L   N   Y   L   K    213 <br />  <br />             GCAATTACAAAAAAATGAGCCACTTGTACATGAAAGAAGTCATACTTTTCAACAAACAAATCCACAAAACAATTTTCATCGTCATACTAATAGTATGAACATTGGGAATGACAATGGAAT                                                                                  1440 <br />              Q   L   Q    K   N   E   P    L     V   H       E   R   S     H      T   F   Q Q        T     N   P       Q N     N   F       H   R       H   T     N   S    M       N   I   G       N   D   N   G   M    253 <br />  <br />             GGGTTGGCACAGTATGCATCAATATATTTAAAAACTAGAACAGAAGGCCTATAAGGAAACATATAATGGCTAATAGGAAAGCGGTTTGCACAGTAACTGTGTTATGAATTTGAGGAACAA                                                                                  1560 <br />              G   W   H    S   M   H Q Y          I   *                                                                                                                                                                 262 <br />  <br />             CCTTGGCACCACTAGTCGGACGCCCAAAAGACGATGAAACGCGTGCAAGTCCTATTTGATTTCCACTGTTTCCAGCGAACATAGATGGACTCGTAGCTGTTGATTGTCTTGTGTTATTAA                                                                                  1680 <br />             TTACATTAGCGGTAGATCT                                                                                                                                                                                       1699 <br />                          Sgl II <br />  <br />             C <br />                          cut8               1&apos;       METLSYSQIKK                          DFDEDISXRAR                            LQYS T                                                                    QCV <br />                          Dbf 8            46&quot;nBEELPRNX                                    GGVSXRRPQPG                            KELM <br />                                                                                                                                 IQ                                                                        DL w <br />  <br />                                        61&apos; <br />                                       106&quot; <br />                                                      LARDI--E PA <br />                                                      IQQSVH <br />                                                                                            D E4                 GG <br />                                                                                       LDFE QKB DK QXSEAVY EIRSYES <br />                                                                                                                            XLLINLNIE----                                                                  REXYMAV <br />                                                                                                                                                                                                           XPQILEF <br />  <br />                                       115&apos;           WA&amp;&amp;VPCY                                 STCFEKNI                             A                   E       t     NPN                      SQAYYFGAIL <br />                                       166n cUE FILDN                                          LENLHASL                             C&amp;Z 45                            LAS                      D:KCIEQE HVEC <br />  <br />                                       175&apos;           VVLRQLEDRPVVP--------LLPLE                                                     4K+4Q NR IEAIY   EKN 4LVHZI <br />                                       226&quot;           TLIEHVARDRIILLADNSSVWKSHKTH                                                   VXUI     GL LRP9QL S MGSESAAS <br />  <br />                                       227&apos;           HTF cINPQNNFHR 4                                          4DNGNGW4HQYI <br />                                       286&quot;           TLS    SIIYHHDT  N                                        I4 GSAAT iF <br />   FIG. 4. Isolation nucleotide sequencing cut8+ gene. (A) Three cosmids, A7, C2, Bl, hybridized nuc2+ probe, used <br /> for complementation cut8-563 mutation. Two cosmids, C2 Bi, complemented cut8-563 (indicated +). Subsequent subcloning yielded <br /> plasmids pCUT8-1 pCUT8-15 complemented cut8-563. pCUT8-20 pCUT8-21 constructed ligating putative initiation <br /> codon S. pombe inducible promoter nmtl. The termination codon cut8+ pCUT8-20 ligated HA. Both plasmids <br /> complemented cut8-563 presence absence thiamine. (B) Nucleotide sequence insert pCUT8-15 determined. A 786-bp <br /> ORF coding 262 amino acids shown. Some restriction sites indicated. (C) Comparison cut8 amino acid sequence of <br /> budding yeast Dbf8 (24). Identical amino acids designated boxes. The budding yeast Dbf8 sequence kindly provided C. Holm. <br />                                                                                                                6367 <br />  6368        SAMEJIMA AND YANAGIDA                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />             A                              B                               acid sequence outside kinase domain similar to <br />                                                                            known proteins. Two hydrophobic regions were <br />                                             1 2 3                          amino domain (indicated arrowheads Fig. <br />                                                                            6D). <br />                                                        -92 <br />                                                                               cekl kinase unusual large size but <br />                 1   2   3     4                        -66 <br />                                                                            potential role exiting mitosis. In regard, cekl <br />                                                        -45                 similar dskl kinase (48). Sequence similarity the <br />                                                                            kinases, however, high. Interestingly, kinases <br />                                                        -31                 contained insertions kinase coding regions. The <br />                                                                            insertion sequence dskl protein signals cell cycle-dependent <br />                                                        -21                 nuclear localization (48). <br />                                                                               cekl +is nonessential viability. The cek] null strain was <br />                                                                            constructed one-step gene disruption (40). Almost the <br />    FIG. 5. Identification cut8+ gene product. (A) Extracts S.        coding region replaced ura4+ gene (see Materials <br /> pombe wild-type cells carrying vector plasmid (lanes 3)     Methods). Plasmid pS820 ura4+ replacement was <br /> multicopy plasmid cut8-&apos;- gene (lanes 2 4) prepared,     linearized, transformed Ura- haploid, selected on <br /> run SDS-PAGE, immunoblotted preimmune serum (lanes 1             medium lacking uracil. Subsequent genomic Southern analysis <br /> and 2) affinity-purified polyclonal anti-cut8 antibodies (lanes 3   reveals ura4+ replaces cekl+ lira&apos; segregates a <br /> 4). The 33-kDa band increased band intensity              single gene. Haploid cells lacking cekl + gene produced <br /> multicopy plasmid indicated arrowhead. The bands and <br /> below 33-kDa protein cross-reacting bands. (B) Extracts S.      normal colonies 22, 26, 30, 33, 36&#176;C. Hence, cekl + is <br /> pombe wild-type cells carrying vector plasmid (lane 1), plasmid        nonessential viability. <br /> with HA-tagged cut8+ gene (lane 2), plasmid cut8+ <br /> gene tag (lane 3) prepared, run SDS-PAGE,                                   DISCUSSION <br /> immunoblotted monoclonal antibody HA. The 38-kDa <br /> band obtained lane 2. No band detected            In paper, characterization cut8- <br /> extracts.                                                                  563 mutant phenotype identification cut8+ and <br />                                                                            suppressor cek] + genes. Although approximately 20 <br />                                                                            genetic loci generate cut phenotype (21, 43, 53), the <br /> interactions cut] cut&apos; genes. The previously                 cut8-563 phenotype unique regard behavior of <br /> isolated cut+ genes used transformation cut8-563.              chromosomes spindle restrictive temperature. <br /> Among cut&apos; genes examined (cutJ+, cut2+, cut5,                    The predicted amino acid sequences cut8+ cek] + not <br /> cut7+, cut9+), cut]&#177; able suppress cut8-563.               resemble known cut&apos; gene products. cekl first <br /> Since cloned cut8+ gene rescue cut] mutant               protein kinase related function cut&apos; genes. The <br /> alleles, suppression bilateral. The phenotype cutl              product cut8+ similar budding yeast DBF8 <br /> mutant partly similar cut8-563 regard spindle          gene, essential viability (24). Temperature sensi- <br /> formation elongation (54).                                             tivity dbf8 mutations cause increased rates chromosome <br />    In presence cdcl] mutation blocks septum               loss nondisjunction permissive temperature. <br /> formation, cut] mutants undergo rounds spindle pole body                   A cellular defect initially recognized cut8-563 mutant <br /> duplication DNA replication, producing giant cells                shift 36&#176;C formation overcondensed <br /> polyploid nuclei multiple spindle pole bodies (9, 54). To              chromosomes short spindles. In cells, chromosomes <br /> examine cut8 mutant cells exhibit similar phenotype,             abnormally extended spindle did fully <br /> the double mutant cut8-cdcl] constructed genetic                    elongate. This anaphase defect occurred 2 3 h shift <br /> crosses. Unexpectedly, septation cytokinesis took place             restrictive temperature, cells divided <br /> the double mutant, producing cut phenotype (data                   once. The inactivation cut8-563 protein require <br /> shown); phenotype cut8-563 epistatic                 duration certain time 36&#176;C. Alternatively, cells <br /> cdcl]. Although understood cut phenotype                reside G2 pass G,/S <br /> of double mutant cut8-cdcl] produced, modes                 phase 36&#176;C produce mutant phenotype, leads to <br /> interactions cutl+ cdcll] cut8+                    2-h delay. Analysis synchronous culture as <br /> with cdc]]+ clearly different.                                        release G, arrest nitrogen starvation be <br />   Novel protein kinase cekl multicopy suppressor. Mul-                necessary determine likelihood of these <br /> ticopy plasmid pS81 rescued cut8-563 mutation. Subcloning                  possibilities. <br /> revealed 4.5-kb XbaI-BglII fragment able                      In budding yeast dbf8 mutant, cells arrested a <br /> suppress ts phenotype cut8 (Fig. 6A). Its entire nucle-             short spindle (24). Both cut8 dbf8 mutants defective in <br /> otide sequence determined, large ORF coding                sister chromatid separation accompany short spindle. <br /> 1,309-amino-acid protein (Fig. 6B).                              The carboxy domain Dbf8 protein essential the <br />   The central domain ORF similar number                 restrictive temperature. This consistent finding that <br /> protein kinases. This suppressor gene designated cekl+                 putative mutation site cut8-563 resides carboxy- <br /> (cut suppressing kinase). Eleven kinase domains                  terminal domain. Minichromosomes introduced cut8-563 <br /> conserved (Fig. 6C). No kinase showed strong similarity             unstable. The difference dbf8 mutants which <br /> kinase domain cekl. The highest similarity (30 40%)               lose minichromosomes high loss rates possibly the <br /> the kinase region protein kinases C A. The               allele-specific phenotype cut8-563. The cut8+ gene not <br /> kinase consensus sequence spans residue 585 987,              functional homolog budding yeast DBF8 gene, but <br /> an interruption 112-amino-acid intervening sequence                   probably functionally similar proteins belonging the <br /> between domains VII VIII. A similarly positioned insert                gene <span id='am-44' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-45' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. <br /> was described S. cerevisiae CDC7 kinase (19),                     It remained determined cut8+ gene was <br /> overall similarity cekl CDC7 low. The amino                essential viability. The phenotype cut8-563 mutant <br />  VOL. 14, 1994                                                                                                                                              FISSION YEAST cut8+ GENE        6369 <br />  <br />  <br />                                                                                                                                                     1k4b <br />                 A                                                                                            ORF                                - <br />  <br />                                                                    Xbal             BgIII      Hindlil         BgIII           III     Hiill                     Bgil <br />                                                                                                                     &gt;     Prd I            Il <br />                                             ps81 <br />                                             ps8 <br />                                             ps83 <br />                                                                                                                                                                   r- + <br />                                             ps87 <br />                                             ps88 <br />                                             ps810 <br />                                                                     I <br />                                                                     I <br />                                                                                      I <br />                                                                                      I              I <br />                                                                                                                          11 <br />                                                                                                                          11                                       L             + <br />  <br />  <br />  <br />  <br />                 B <br />                 MKHIKNEREEVFLEDDQAQHSQAELLSSKDENLQPSI PLSPVAFELDFSGNFQFI SDNSSELLDI PKDKI IGHSVAEVLGTDGYNAFMRAVNCLLKDDSH                                                             100 <br />                 SYHVRFQHSINANHANQNYYTAKGDLPSDEKITKPFDAIGI LIRHPGSAIPAHTMWVVNPATNSLGSVSPLVTKLLDVIGFGASLLDKYLCDLRTSYHKH                                                                200 <br />                 NSLDALPLPTPEFCQICEREIQSWFFELHSKFCLSTSTYESVVQAAQDSLLYFRSTLLEIQEGMQKDSSLVPVYKNEPLIVDADDYFFTDENKQTLSLCS                                                                 300 <br />                 FLSQVMYYLEVAIDITIPPVKI IVNFDKVDSLRVQSPRSEKATIELDNYNPSLENCSSAVIALWEDIKTAVDTKITGVLRLRNAIYYSER -RLEI DHHVQ                                                              400 <br />                 E IIDDWSNLVTNHSSTSLGHLESKLAPSITFPDACDALEAEECITRPGSATNTPQSDRSLDINDLSRSSSYSRHLSHVSLSNPDFAIGSPMSQDSSNYS                                                                 500 <br />                 SPLHRRKASDSNFSDPRFDDLKYLSPNSSPRFVASDGPNRPASNGRSSLFSRGRASNLGDVGLRLPS PSPRI HT IVPNSAPEH PS,INDYK ILKPI SKGAF                                                          600 <br />                 GSVYLAOKRTTGDYFAIKILKKSNMIAKNOVINVRAERAILMSOGESPFVAKLYYTFOSKDYLYLVMEYLNGGDCGSLLKTMGVLDLDWIRTYIAETVLC                                                                 700 <br />                 LGDLHDRGI IHRDIKPENLLISONGHLKLTDFGLSRVGYMKRHRRKQSSSIPVLDLRDRSSAISDLSLSTASSVLEAQSLITPERPKRPSLNEKLLSLDG                                                                800 <br />                 TSIRLAGQSFNYENSAEDSPTATNTPTSQVDESNI FRSTDSPRVQPFFENKDPSKRF IGTPDY IAPEV ILGNPG IKASDWiWSLGCVVFEFLFGY PPFNAE                                                          900 <br />                 T PDOVFON ILARRINWPAEVFTAESSVALDL IDRLLC MNPANRLGANGVEE IKAH PFFKSVNWDT I LEEDPPFV PK PFSPEDTVYFDSRGLKGFDFSEYY                                                      1000 <br />                 NQPTVTEAQKLEEERPASSIPQHVVGNRKGRLRSNTISTPEFGSFTYRNLDFLNKANRNTIQKLRKEHMAVKSAKTSVDDTFSQYMSRFKAKLSTSQSVG                                                                1100 <br />                 PVKSSRRASMADYEASTTTRVQDITTDSIDSIDDFDSLKEGRMLSFFDNLALEDHKGVSSTMSASQSQSSMHTALPDVTEGTSSDEHTTIQKGRI DNLQA                                                               1200 <br />                 QSLTHKRNAISYPGLFQLDRLQMI I PKDEIELAEILKKIFPKLTLVLIDDPWSILKKLLQNEQFNVVFLHFGNDKVSSSRLMYSVRTSATINSRVPVCIH                                                              1300 <br />                 LRGRDLHSD                                                                                                                                                           1309 <br />  <br />  <br />  <br />                 C <br />                                                                I                               II                                      III                              lU <br />                                  -JEL-- _jw__        --m- .-MLJML <br />                                                                                                __ <br />                                                                                                         __L IUNIARN <br />                 c-lcl   SOS I         Ylt [P IE B           3                  QKRTTGDY             I                          INVR          RFA 11SQON9            AKLYY  B <br />                 TPK1     83 L          Q   TLG             BR                  RBRBNGRY                            IVVRLK       ETN          RL    IV T            IRMWGD A <br />                 PlCa    335 L         PN JV LG                                 DRIGTBBL             I              VVIQDD        CTMV           R      LLDK        LTQLHS%   V <br />  <br />  <br />  <br />                                   U                                                                                 Ul                                     Ull <br />                 DYLYL                  CGSLLKTXGVLDLDWIR4 I   VLc RR ELISQ   SRVGYNKR-// <br />                 QQIFM <br />                    Fl                  LFSLLRKSQRFPNPVA A4VCL* 3 YLLD      I AIYVPDVT <br />                 DRLYF       IV         LMYNIQQVGKFKBPQAV    r BIN R <br />                                                                   FN         CRHMNDGVTT <br />  <br />  <br />                 -RF3 <br />                 YTLC <br />                                            I LC3 <br />                                            V IvT <br />                                                   IcVVFBE <br />                                                    Y&amp;3 I1 <br />                                                              YF  Y:AETPDQVFQMLARRIN&apos;46EVFTAB <br />                                                     r L 3I YBRVd qIY <br />                                                                         [ <br />                                                                       DSNTMaTYEV LNABrLRUPFFNB <br />                                                                                                SS <br />  <br />                 RTFC                         3KVLLYB <br />                                            IAY      IY4            PW8DEo&#176;LFQ 4loBNVS 4SLSK <br />  <br />                                                      HI <br />                 VALDL IDILCXNPA                        4VBhB I IC P I8V                       DTILBND                   DFTIP <br />                                                                                                                            FSPNDT <br />  <br />                 DVKDLLS 4TRDLS                      LQ1 TDVK P1EV                               ILLSRNIE                      IQQGQGDTS <br />                  HAVSICXC4(TKQPA                    GP 4RDVR         RI                        KLENREIQ                 F1W          CGOGA             FFT <br />  <br />  <br />  <br />  <br />                                    Xbal                                     BgIll           Saci Smal                           Hindlil                                 BgIll <br />                 D                      I                                                                II <br />                                                                                                                                       i- <br />  <br />                                       ORF                  L <br />                                                    2. 00 <br />  <br />  <br />  <br />  <br />                                                    0.0 <br />  <br />  <br />  <br />  <br />                                                    -2 .0 <br />                                                            I <br />  <br />  <br />  <br />  <br />   FIG. 6. Isolation nucleotide sequencing cekl + gene. (A) Subcloning ceki + gene. +, suppression cut8-563. The coding region <br /> and direction transcription indicated arrow. Some restriction sites indicated. (B) Predicted amino acid sequences cekl + <br /> gene product. The coding region contains 1,309 amino acids. (C) Comparison cekl kinase domain kinases, TPK1 (7, 30, <br /> 49) protein kinase Ca (PKCa) (39). Identical amino acids boxed. Eleven kinase domains numbered. (D) Schematic representation of <br /> cekl protein hydropathy plot. Hydropathy calculated method Kyte Doolittle (28). The arrowheads indicate the <br /> hydrophobic stretches. The hatched regions represent kinase domains. <br />  6370     SAMEJIMA AND YANAGIDA                                                                                            MOL. CELL. BIOL. <br />  <br /> hence cautiously interpreted phenotype       (21, 54). The mode interaction cut8+ cutl + and <br /> the cut8 null known. Gene disruption different        cut8+ cek] + subject future investiga- <br /> constructs successful. The S. cerevisiae DBF8 gene     tion. <br /> is essential gene viability (24). The recessive cut8-563 <br /> mutation (21) causes decrease cell viability                                 ACKNOWLEDGMENTS <br /> restrictive temperature. This result suggests aberrant           We thank I. Hagan help early stage study, M. Goebl <br /> anaphase subsequent cytokinesis result loss       sequence information DBF8, C. Holm F. Houman for <br /> cut8+ lethal. Not cells inviable,           information prior publication. We grateful S. Su <span id='am-88' about='obo:NCBITaxon_84399' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_87877 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_84398 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Discus" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92'  ></span>discus</span>- <br /> however; approximately 10% cells viable 5 h     sion critical reading manuscript. <br /> at 36&#176;C. We suspected viable cells represented           This work supported grant (specially promoted research) <br /> uncut phenotype containing single nucleus sep-         Ministry Education, Science Culture Japan the <br /> tum. Interestingly, viable cells diploid     RIKEN Biodesign Program. I.S. Research Fellow Japan <br /> were stably maintained homozygous cut8-563. We              Society Promotion Science (JSPS). <br /> hypothesize cells displaced nuclei resulting                                  REFERENCES <br /> missorting chromosomes end spindle             1. Adachi, Y., M. Yanagida. 1989. Higher order chromosome <br /> only nucleus receive chromosomes. Thus, cells           structure affected cold-sensitive mutations S. pombe <br /> contain twice DNA. Fission yeast diploidization           gene cnnr+ encodes 115-kd protein preferentially local- <br /> mutants dip isolated procedure brief heat              ized nucleus periphery. J. Cell Biol. 108:1195- <br /> shock (6), allelic cutl cut7 (43). Hence,            1207. <br /> these cut&apos; genes required stable maintenance          2. Andreadis, A., Y.-P. Hsu, M. Hermodson, G. Kohlaw P. R. <br /> of haploid cells. Improper diploidization appearance              Schimmel. 1984. Yeast LEU2: repression mRNA levels by <br /> of cut phenotype brought                   leucine primary structure gene product. J. Biol. Chem. <br /> common defect.                                                            259:8059-8062. <br />    The phenotypes cut8-563 resemble budding yeast        3.   Basi, G., E. Schmid, K. Maundrell. 1993. Tata box mutations <br /> mutants ctfl3 ctfl4 (identical ndcO/cbf2), defective            Schizosaccharomyces pombe nmtl promoter affect transcrip- <br /> the genes encode essential components kinetochore             tion efficiency transcription start point thiamine <br />                                                                           repressibility. Gene 123:131-136. <br /> (10, 13, 26). ctfl3 mutants missegregate chromosomes             4.   Beach, D., P. Nurse. 1981. High-frequency transformation of <br /> transiently arrest mitosis short spindle. Polyploid         fission yeast Schizosaccharomyces pombe. Nature (London) <br /> nuclei frequently mother cells, probably                 290:140-142. <br /> because impaired interaction chromosomes              5.   Beach, D., M. <span id='am-1' about='obo:NCBITaxon_13215' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_16736 obo:NCBITaxon_131567 obo:NCBITaxon_16739 obo:NCBITaxon_3398 obo:NCBITaxon_232347 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Piper" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Piper</span>, P. Nurse. 1982. Construction a <br /> and microtubules. In cut8-563 mutant cells, interaction               Schizosaccharomyces pombe gene bank yeast bacterial vector <br /> between condensed mitotic chromosomes kinetochore                     use isolate genes complementation. Mol. Gen. Genet. <br /> microtubules defective.                                       187:326-329. <br />    Since high dosage cekl + gene overcome           6.   Broek, D., R. Bartlett, K. Crawford, P. Nurse. 1991. Involve- <br /> defect cut8 mutation, cekl kinase affect directly               ment p34cdc2 establishing dependency S phase on <br />                                                                           mitosis. Nature (London) 349:388-393. <br /> indirectly cut8+ gene function. One features cekl          7.   Cannon, J. F., K. Tatchell. 1987. Characterization Saccha- <br /> kinase 112-amino-acid intervening sequence             romyces cerevisiae genes encoding subunits cyclic AMP-depen- <br /> 7th 8th kinase domains. This sequence rich serine/              dent protein kinase. Mol. Cell. Biol. 7:2653-2663. <br /> threonine, proline, charged amino acids, amino           8.   Costello, G., L. Rodgers, D. Beach. 1986. Fission yeast enters <br /> acid composition similar dskl kinase (48).           stationary phase GO state mitotic Gl G2. Curr. <br /> The dskl kinase isolated multicopy suppressor disi         Genet. 11:119-125. <br /> mutants defective mitotic anaphase (37)            9. Creanor, J., J. M. Mitchison. 1990. Continued DNA synthesis <br /> appears function positive element mitotic exit (48).        mitotic block double mutant cut] cdcll fission <br /> dskl kinase activated metaphase-arrested mutant cells             yeast Schizosaccharomyces pombe. J. Cell Sci. 96:435-438. <br />                                                                     10. Doheny, K. F., P. K. Sorger, A. A. Hyman, S. Tugendreich, F. <br /> and enriched nucleus. The intervening sequence dskl           Spencer, P. Hieter. 1993. Identification essential compo- <br /> kinase responsible cell cycle-dependent nuclear         nents S. cerevisiae kinetochore. Cell 73:761-774. <br /> localization. The cekl sequence similar role. The        11. Field, J., J. Nikawa, D. Broek, B. MacDonald, L. Rodgers, I. <br /> cut8 protein phosphorylated cekl kinase,             Wilson, I. Lerner, M. Wigler. 1988. Purification RAS- <br /> phosphorylation, activated. Alternatively, cut8             responsive adenylyl cyclase complex <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae <br /> might interact cekl positive modulator. Another               use epitope addition method. Mol. Cell. Biol. 8:2159- <br />                                                                         2165. <br /> possibility overproduced cekl kinase bypass       12. Funabiki, H., I. Hagan, S. Uzawa, M. Yanagida. 1993. Cell <br /> cut8+ function.                                                         cycle-dependent specific positioning clustering centromeres <br />    The cut8+ gene appears required normal                 telomeres fission yeast. J. Cell Biol. 121:961-976. <br /> progression anaphase. It apparently implicated        12a.Goebl, M., C. Holm. Personal communication. <br /> restraining cytokinesis certain stage anaphase. Mul-     13. Goh, P.-Y., J. V. Kilmartin. 1993. NDCIO: gene involved in <br /> tiple septa cut8+-overexpressed cells consis-          chromosome segregation Saccharomyces cerevisiae. J. Cell Biol. <br /> tent hypothesis overproduced cut8 protein retarded          121:503-512. <br /> cytokinesis. The cut8+ gene possibly plays dual role           14. Gordon, C., G. McGurk, P. Dillon, C. Rosen, N. D. Hastie. <br /> regulating anaphase cytokinesis. It role            1993. Defective mitosis mutation gene fission <br /> checkpoint control progression anaphase; postan-             yeast 26S protease subunit. Nature (London) 366:355-357. <br /> aphase events restrained cut8+ gene            15. Grimm, C., J. Kohli, J. Murray, K. Maundrell. 1988. Genetic <br /> completion anaphase. We think, however, primary               engineering Schizosaccharomyces pombe: gene <br /> role cut8+ gene likely reside anaphase             disruption replacement using ura4 gene selectable <br />                                                                         marker. Mol. Gen. Genet. 215:81-86. <br /> progression. We speculate partial separation chromo-        16. Gutz, H., H. Heslot, U. Leupold, N. Lopreno. 1974. Schizo- <br /> somes cut8 mutant cells sufficient trigger cytoki-         saccharomyces pombe, p. 395-446. In R. C. King (ed.), Handbook <br /> nesis. Similar phenotypes cut] top2 mutants           genetics, vol. 1. Plenum Press, New York. <br />  VOL. 14, 1994                                                                                          FISSION YEAST cut8+ GENE               6371 <br />  <br /> 17. Hagan, I., J. S. Hyams. 1988. The use cell division cycle            sister chromatid separation mitosis. EMBO J. 7:1465-1473. <br />     mutants investigate control microtubule distribution       38. Ohkura, H., N. Kinoshita, S. Miyatani, T. Toda, M. Yanagida. <br />     fission yeast Schizosaccharomyces pombe. J. Cell Sci. 89:343-           1989. The fission yeast dis2+ gene required chromosome <br />     357.                                                                        disjoining encodes putative typel protein phosphatases. <br /> 18. Hagan, I., M. Yanagida. 1990. Novel potential mitotic motor             Cell 57:997-1007. <br />     protein encoded fission yeast cut7+ gene. Nature (London)        39. Rose-John, S., A. Dietrich, F. Marks. 1988. Molecular cloning <br />     347:563-566.                                                                mouse protein kinase C (PKC) cDNA Swiss 3T3 fibro- <br /> 19. Hanks, S., A. M. Quinn, T. Hunter. 1988. The protein kinase             blasts. Gene 74:465-471. <br />     family: conserved features deduced phylogeny catalytic       40. Rothstein, R. J. 1983. One-step gene disruption yeast. Methods <br />     domains. Science 241:42-52.                                                 Enzymol. 101:202-211. <br /> 20. Harlow, E., D. Lane. 1988. Antibodies. Cold Spring Harbor           41. Saka, Y., M. Yanagida. 1993. Fission yeast cut5 gene <br />     Laboratory Press, Cold Spring Harbor, N.Y.                                  required onset S-phase restraint M-phase is <br /> 21. Hirano, T., S. Funahashi, T. Uemura, M Yanagida. 1986.                  identical radiation-damage repair gene rad4+. Cell 74:383- <br />     Isolation characterization Schizosaccharomyces pombe cut             393. <br />     mutants block nuclear division cytokinesis. EMBO J.        42. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br />     5:2973-2979.                                                                cloning: laboratory manual, 2nd ed. Cold Spring Harbor Labo- <br /> 22. Hirano, T., Y. Hiraoka, M. Yanagida. 1988. A temperature                ratory Press, Cold Spring Harbor, N.Y. <br />     sensitive mutation S. pombe gene nuc2+ encodes            43. Samejima, I., T. Matsumoto, Y. Nakaseko, D. Beach, M. <br />     nuclear scaffold-like protein blocks spindle elongation mitotic          Yanagida. 1993. Identification seven new cut genes involved in <br />     anaphase. J. Cell Biol. 106:1171-1183.                                      Schizosaccharomyces pombe mitosis. J. Cell Sci. 105:135-143. <br /> 23. Hiraoka, Y., T. Toda, M. Yanagida. 1984. The NDA3 gene           44. Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequencing <br />     fission yeast encodes j-tubulin: cold sensitive nda3 mutation             chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA <br />     reversibly blocks spindle formation chromosome movement              74:5463-5467. <br />     mitosis. Cell 39:349-358.                                               45. Smith, D. E., P. A. Fisher. 1984. Identification, developmental <br /> 24. Houman, F., C. Holm. 1994. DBF8, essential gene required             regulation, response heat shock antigenically related <br />     efficient chromosome segregation Saccharomyces cerevisiae.           forms major nuclear envelope protein Drosophila embryos: <br />     Mol. Cell. Biol. 14:6350-6360.                                              application improved method affinity purification of <br /> 25. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1993. Transforma-             antibodies using polypeptides immobilized nitrocellulose blot. <br />     tion intact yeast cells treated alkali cations. J. Bacteriol.       J. Cell Biol. 99:20-28. <br />     153:163-168.                                                            46. Stone, E. M., H. Yamano, N. Kinoshita, M. Yanagida. 1993. <br /> 26. Jiang, W., J. Lechner, J. Carbon. 1993. Isolation charac-           Mitotic regulation protein phosphates fission yeast sds22 <br />     terization gene (CBF2) specifying protein component           protein. Curr. Biol. 3:13-26. <br />     budding yeast kinetochore. J. Cell Biol. 121:513-519.                   47. Studier, F. W., B. A. Moffatt. 1986. Use bacteriophage T7 <br /> 27. Johnston, L. H., A. P. Thomas. 1982. The isolation new               RNA polymerase direct selective high-level expression of <br />     DNA synthesis mutants yeast Saccharomyces cerevisiae. Mol.           cloned genes. J. Mol. Biol. 189:113-130. <br />     Gen. Genet. 186:439-444.                                                48. Takeuchi, M., M. Yanagida. 1993. A mitotic role novel <br /> 28. Kyte, J., R. F. Doolittle. 1982. A simple method displaying         fission yeast protein kinase dskl cell cycle stage dependent <br />     hydropathic character protein. J. Mol. Biol. 157:105-132.          phosphorylation localization. Mol. Biol. Cell 4:247-260. <br /> 29. Leupold, U. 1970. Genetical methods Schizosaccharomyces             49. Toda, T., S. Cameron, P. Sass, M. J. Zoller, M. Wigler. 1987. <br />     pombe. Methods Cell Physiol. 4:169-177.                                     Three different genes S. cerevisiae encode catalytic subunits <br /> 30. Lisziewicz, J., A. Godany, H. H. Foerster, H. Kuentzel. 1987.           cAMP-dependent protein kinase. Cell 50:277-287. <br />     Isolation nucleotide sequence Saccharomyces cerevisiae         50. Toda, T., K. Umesono, A. Hirata, M. Yanagida. 1983. Cold- <br />     protein kinase gene suppressing cell cycle start mutation. J.           sensitive nuclear division arrest mutants fission yeast <br />     Biol. Chem. 262:2549-2553.                                                  Schizosaccharomyces pombe. J. Mol. Biol. 168: <br /> </body></html>