<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 4 February 2014                                               Nucleic Acids Research, 2014, Vol. 42, No. 7 4697&#8211;4711 <br />                                                                                                                  doi:10.1093/nar/gku097 <br />  <br />  <br /> Structure-function analysis Yhc1 subunit of <br /> yeast U1 snRNP genetic interactions Yhc1 <br /> with Mud2, Nam8, Mud1, Tgs1, U1 snRNA, SmD3 <br /> and Prp28 <br /> Beate Schwer1,* Stewart Shuman2,* <br /> 1 <br />     Microbiology Immunology Department, Weill Cornell Medical College, New York, NY 10065, USA and <br /> 2 <br />     Molecular Biology Program, Sloan-Kettering Institute, New York, NY 10065, USA <br />  <br /> Received September 27, 2013; Revised January 3, 2014; Accepted January 9, 2014 <br />  <br />  <br />  <br /> ABSTRACT                                                                        INTRODUCTION <br />  <br /> Yhc1 U1C homologous essential subunits                               Yeast pre-mRNA splicing (1&#8211;3) begins formation <br />                                                                                 complex comprising U1 snRNP bound the <br /> the yeast human U1 snRNP, respectively, that <br />                                                                                 intron 50 splice site (50 SS; 50 -GUAUGU) the <br /> are implicated establishment stability                            Msl5 Mud2 heterodimer engaged intron <br /> the complex U1 bound pre-mRNA 50 splice                               branchpoint (BP; 50 -UACUAAC). Bridging interactions <br /> site (50 SS). Here, conducted mutational                                   U1 snRNP Msl5 Mud2 stabilize the <br /> analysis Yhc1, guided U1C NMR structure                               complex prepare scaffold recruitment the <br /> and low-resolution crystal structure human U1                                U2 snRNP branchpoint. The U1 snRNP ultim- <br /> snRNP. The N-terminal 170-amino acid segment                                 ately ejected pre-mRNA U1 U2-containing <br /> the 231-amino acid Yhc1 polypeptide sufficed                                spliceosome U5 U4 U6 tri-snRNP complex <br /> vegetative    growth. Although changing                                     joins en route forming pre-mRNA U2 U5 U6 <br /> zinc-binding residue Cys6 alanine lethal,                                spliceosome. Dissociation U1 snRNP thought be <br /> alanines zinc-binding residues Cys9, His24                               triggered DEAD-box protein Prp28 (4,5), acting to <br />                                                                                 disrupt short U1:50 SS RNA duplex remodel <br /> His30 not. Benign alanine substitutions at <br />                                                                                 protein-RNA contacts 50 SS (or both). <br /> conserved surface residues elicited mutational                                     The <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> U1 snRNP consists a <br /> synergies splicing components.                                       trimethylguanosine (TMG)-capped 568-nt U1 snRNA, a <br /> YHC1-R21A synthetically lethal absence                               7-subunit Sm protein ring (common U2, U4 U5 <br /> of Mud2 synthetically sick absence                                snRNPs), 10 protein subunits unique yeast U1 <br /> Nam8, Mud1 Tgs1 presence                                       snRNP: Prp39, Prp40, Snu71, Snu56, Snp1, Mud1, Luc7, <br /> variant U1 snRNAs. YHC1 alleles K28A, Y12A,                                     Prp42, Nam8 Yhc1 (6&#8211;9). The composition U1 <br /> T14A, K22A H15A displayed progressively                                   snRNP complex budding yeast humans <br /> narrower range synergies. R21A K28A                                      (10&#8211;12), respect size U1 RNA (568 <br /> bypassed essentiality DEAD-box protein                                   versus 164 nt) number U1-speci&#65533;?c protein <br /> Prp28, suggesting affected U1 50 SS                                   subunits (10 versus 3). The human U1-speci&#65533;?c <br />                                                                                 snRNP subunits&#8212;U1-70K, U1-A U1-C&#8212;are <br /> complex stability. Yhc1 Arg21 fortifies the <br />                                                                                 homologs yeast Snp1, Mud1 Yhc1, respectively. <br /> U1 50 SS complex contacts SmD3 residues                                   The conserved 50 leader sequence yeast human <br /> Glu37/Asp38, mutations synergized                                 U1      RNA&#8212;m2,2,7GpppAUACUUACC&#8212;contains               a <br /> mud2&quot; bypassed prp28&quot;. YHC1-(1-170)                                     hexanucleotide motif (underlined) complementary <br /> synthetically lethal mutations                                      consensus yeast 50 SS. The ACUUAC sequence pairs <br /> components interrogated, exception                                  pre-mRNA nucleate early assembly <br /> Nam8.                                                                           intermediate. Although ACUUAC motif U1 RNA <br />  <br />  <br />  <br />  <br /> *To correspondence addressed. Email: s-shuman@ski.mskcc.org <br /> Correspondence addressed Beate Schwer. Email: bschwer@med.cornell.edu <br />  <br /> &#223; The Author(s) 2014. Published Oxford University Press. <br /> <span id='am-53' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  4698 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br /> is essential yeast viability, half U1     identify benign mutations Yhc1 display allele- <br /> primary structure, including 50 TMG cap,                speci&#65533;?c synergies mutations spliceosome, <br /> dispensable (13&#8211;17). In vein, U1 snRNP            especially absence Mud2, Nam8, Mud1, the <br /> subunits Mud1 Nam8 inessential,              TMG cap speci&#65533;?c perturbations 50 end <br /> Mud2 subunit Msl5 Mud2 branchpoint-binding              U1 snRNA. We identify novel Yhc1 mutations <br /> protein (7,17&#8211;21). Substantial chunks essential U1      bypass essentiality Prp28. We interpret our <br /> snRNP subunits Snp1 Prp40 essential                &#65533;?ndings light 5.5 A&#730; crystal structure the <br /> branchpoint-binding subunit Msl5 deleted           human U1 snRNP includes N-terminal domain <br /> without compromising yeast viability (22&#8211;26).                  U1C (10). Our results add Yhc1 extensive <br />    This say dispensable elements    reticulum intramolecular intermolecular genetic <br /> the yeast U1 snRNP functionally irrelevant. Rather,        interactions U1 snRNP. <br /> traditional genome-wide genetic analyses have <br /> highlighted network genetically buffered functions <br /> during early spliceosome assembly, embracing U1            MATERIALS AND METHODS <br /> snRNP, Msl5 Mud2 branchpoint-binding protein,              Yhc1 expression <span id='am-103' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> mutants <br /> the TMG cap Cbc2 Sto1 nuclear m7G <br /> cap-binding complex (7,17&#8211;34). This network                A 1.7-kb DNA segment bearing YHC1 gene was <br /> de&#65533;?ned numerous instances null alleles         ampli&#65533;?ed S. cerevisiae genomic DNA PCR <br /> of inessential players, benign mutations essential       using forward primer introduced XmaI site <br /> factors, elicited synthetic lethal phenotypes             490-bp upstream Yhc1 translation initiation site <br /> combined benign mutations splicing           reverse primer introduce NotI site 514-bp <br /> machinery. Such genetic interactions actors         downstream stop <span id='am-58' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-59' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span>codon</span>. The PCR product was <br /> common pathway meet operational de&#65533;?nition                digested XmaI Not1 inserted the <br /> redundancy, does necessitate synthetic      XmaI NotI sites centromeric plasmids pRS316 <br /> interactor proteins RNA elements perform           (URA3) pRS413 (HIS3) yield yeast expression <br /> task, suggests spliceosome assembly        plasmids p316-YHC1 p413-YHC1. Single alanine <br /> be accomplished stabilized different sub-pathways.      mutations introduced YHC1 gene plasmid <br />    Although mutational synergies played decisive        two-stage PCR overlap extension mutagenic <br /> role identifying components yeast        primers. The PCR products digested XmaI <br /> spliceosome (7,18,19,27,29), interpretation synthetic   NotI inserted pRS413 generate series <br /> phenotypes elusive, especially atomic          p413-YHC1-<span id='am-76' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ala</span> plasmids. C-terminal truncation <br /> structures biochemical activities genetically        variants generated PCR using reverse primers <br /> interacting factors unknown. However,                 introduced stop codons &#64258;anking SpeI site in <br /> structures available, exploited             lieu Asp213, Thr190, Pro171, Gln151, Ser134, Lys117, <br /> program mutations speci&#65533;?c functional defects          His88 Asn66. The truncated PCR products were <br /> then systematically test allelic series synthetic       inserted pRS413 generate plasmids p413- <br /> genetic interactions spliceosome components         YHC1-(1-212), p413-YHC1-(1-189), p413-YHC1-(1-170), <br /> or splicing factors. This applied              p413-YHC1-(1-150), p413-YHC1-(1-133), p413-YHC1- <br /> m7G-cap binding pocket yeast Cbc2 (34), guided           (1-116), p413-YHC1-(1-87) p413-YHC1-(1-65). The <br /> the crystal structure homologous human                  YHC1 genes sequenced completely con&#65533;?rm that <br /> CBC m7G-cap complex (35,36), branchpoint            unwanted changes acquired ampli&#65533;?cation <br /> RNA-binding site yeast Msl5 (25,26), directed        cloning. <br /> NMR structure human homolog SF1 bound an <br />                                                                Yeast strains tests YHC1 function vivo <br /> RNA containing yeast branchpoint consensus <br /> sequence (37).                                                 To develop plasmid shuf&#64258;e assay gauging mutational <br />    In present study, extend approach            effects Yhc1 function, generated yhc1  strain that <br /> interrogate structure-function relations genetic           relies viability maintenance wild-type YHC1 <br /> interactions essential Yhc1 subunit yeast U1         gene CEN URA3 plasmid, p316-YHC1. In brief, we <br /> snRNP. Yhc1 231-amino acid (<span id='am-82' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>aa</span>) polypeptide               &#65533;?rst replaced YHC1 locus position +1 +688 <br /> (Figure 1). Although N-terminal 40-aa segment           hygMX cassette BY4743 diploid strain and <br /> Yhc1 homologous N-terminus 159-aa             transformed heterozygous diploid with <br /> human U1C (27/40 positions chain identity           p316-YHC1. The diploid sporulated, asci were <br /> similarity; Figure 2B), respective C-terminal              dissected haploid yhc1  Ura+ progeny were <br /> segments little apparent primary structure          recovered. The yhc1  [p316-YHC1] cells resistant <br /> similarity (6,38). By serial deletions, de&#65533;?ne distal    hygromycin unable grow medium containing <br /> margin minimal functional Yhc1 protein. We           0.75 mg/ml 5-&#64258;uoroorotic acid (<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>FOA</span>). To assay the <br /> exploit NMR structure N-terminal domain          function wild-type mutated YHC1 alleles, yhc1  <br /> human U1C (38) (Figure 2A) direct mutations          [p316-YHC1] cells transfected CEN HIS3 <br /> Cys6-Cys9-His24-His30 zinc binding site conserved       YHC1 plasmids. Individual His+ transformants were <br /> surface residues considered candidates interact     selected streaked agar medium containing FOA. <br /> with RNA protein components spliceosome. We          The plates incubated 20, 30 37  C, mutants <br />                                                                                           Nucleic Acids Research, 2014, Vol. 42, No. 7 4699 <br />  <br />  <br />  <br />  <br /> Figure 1. Effects C-terminal truncations Yhc1 activity vivo. (Top panel) The amino acid sequence <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae (Sce) Yhc1 is <br /> aligned homologous protein <span id='am-6' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span>Kluyveromyces</span> lactis (Kla). Positions amino acid site chain identity/similarity denoted   <br /> above sequence. Reverse arrowheads indicate boundaries C terminal truncations Yhc1. (Bottom panel) The wild-type truncated <br /> YHC1 alleles tested activity plasmid shuf&#64258;e described Methods. The growth phenotypes viable FOA-resistant yhc1  p[CEN <br /> HIS3 YHC1] strains bearing indicated YHC1 alleles assessed follows. Liquid cultures grown mid-log phase 30  C adjusted <br /> to A600. Aliquots (3 ml) serial 10-fold dilutions cells spotted YPD agar. The plates incubated indicated temperatures <br /> and photographed 2 d (30, 34 37  C), 3 d (25  C) 4 d (18 20  C). The truncated mutants listed failed complement yhc1  in <br /> a plasmid shuf&#64258;e assay deemed lethal. <br />  <br />  <br /> that failed form macroscopic colonies                               subjected sporulation tetrad dissection, after <br /> temperatures 8 d deemed lethal. Individual                         haploid U1  yhc1  [p316-U1-YHC1] prp28  <br /> FOA-resistant colonies viable YHC1 alleles                          yhc1D [p316-PRP28-YHC1] progeny recovered. <br /> grown mid-log phase YPD broth adjusted                           These cells unable grow FOA medium, but <br /> the A600 values. Aliquots (3 ml) serial 10-fold                       double-deletion strains complemented by <br /> dilutions spotted YPD agar plates,                         co-transformation p[CEN HIS3 YHC1] plus either <br /> then incubated temperatures ranging 18 37  C.                      p[CEN LEU2 U1] p[CEN LEU2 PRP28]. <br />    We developed plasmid shuf&#64258;e assays test                          We used similar strategies establish plasmid shuf&#64258;e <br /> mutational effects Yhc1 function mud2 , nam8 ,                          assays test effects SMD3 mutations mud1 , <br /> mud1  tgs1  cells using standard genetic                                  mud2  prp28  backgrounds. In smd3  <br /> manipulations mating, sporulation dissection. We                       [p316-SMD3] haploids used genetic crosses, the <br /> thereby generated mud2  yhc1  [p316-YHC1], nam8                               SMD3 locus (+1 +303) replaced a <br /> yhc1  [p316-YHC1], mud1  yhc1  [p316-YHC1]                                hygromycin-resistance cassette. The viability the <br /> tgs1  yhc1  [p316-YHC1] cells, unable                           smd3 , smd3  mud1  smd3  mud2  strains <br /> grow FOA-containing medium unless                            depended maintenance CEN URA3 plasmid <br /> transformed wild-type YHC1 functional YHC1                          harboring SMD3 gene (nucleotides  445 +552). <br /> mutant allele CEN HIS3 plasmid FOA selec-                         The smd3  prp28  [p316-PRP28-SMD3] strain harbors <br /> tion. To investigate genetic interactions Yhc1 mutants                     CEN URA3 plasmid PRP28 and <br /> with essential U1 PRP28 genes, &#65533;?rst generated                      SMD3 (nucleotides  445 +552) genes arranged a <br /> heterozygous U1 /U1 yhc1 /YHC1 prp28 /PRP28                               head-to-tail con&#65533;?guration. <br /> yhc1 /YHC1 diploids crossing U1D::kanMX p316-U1 <br /> and     prp28D::natMX        p316-PRP28       cells  with <br /> yhc1D::hygMX p316-YHC1 cells opposite mating                           RESULTS <br /> type, selecting diploids YPD medium containing <br /> hygromycin plus G418 hygromycin plus clonNat,                          C-terminal truncations de&#65533;?ne minimized functional Yhc1 <br /> plating FOA-containing medium. The heterozy-                          domain <br /> gous diploids transfected CEN URA3 U1                          The size differences lack amino acid sequence <br /> YHC1 (p316-U1-YHC1) CEN URA3 PRP28 YHC1                                    similarity S. cerevisiae Yhc1 human U1C <br /> (p316-PRP28-YHC1) plasmids. [In plasmids,                           distal N-terminal zinc-binding domain raised the <br /> YHC1 gene arranged head-to-tail con&#65533;?guration                          prospect yeast human C-termini have <br /> with U1 gene (nucleotides  550 +755)                          diverged parallel differences U1 snRNA size <br /> head-to-head con&#65533;?guration PRP28 gene (nucleo-                        number U1-speci&#65533;?c protein subunits the <br /> tides  520 2120).] Ura+ heterozygous diploids                         respective U1 snRNPs. By contrast, alignment the <br />  4700 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br />  <br />  <br /> Figure 2. Structure-guided mutagenesis Yhc1 N-terminal domain. (A) Stereo view NMR structure N-terminal domain human <br /> U1C (pdb 2VRD), highlighting tetrahedral coordination complex Zn2+ ion (magenta sphere) conserved chains Cys6, Cys9, His24 <br /> and His30 (shown stick models). Other conserved chains U1C subjected mutational analysis Yhc1 shown stick <br /> models labeled regular font. Non-identical chains mutated denoted italics: <span id='am-81' about='rdfs:label' typeof='owl:Thing'>label</span> speci&#65533;?es chain human <br /> U1C, yeast Yhc1 chain parenthesis. (B) Alignment primary structures N-terminal domains S. cerevisiae (Sce) Yhc1 <br /> and human (Hsa) U1C. Positions chain identity/similarity indicated   alignment. The helices depicted the <br /> alignment cylinders. The 13 residues Yhc1 subjected alanine scanning present study highlighted gold shading. (C) <br /> Mutational analysis zinc binding site. The C6A, C9A, H24A H30A mutants tested yhc1  complementation plasmid shuf&#64258;e. <br /> C6A lethal. Viable FOA-resistant C9A, H24A H30A strains spot-tested growth YPD agar temperatures speci&#65533;?ed in <br /> parallel isogenic wild-type YHC1 strain. <br />  <br />  <br /> 231-aa S. cerevisiae Yhc1 homologous 189-aa                         lethal, i.e. failed complement growth the <br /> polypeptide yeast <span id='am-63' about='obo:NCBITaxon_28985' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4910 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces marxianus var. lactis" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces lactis var. lactis" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces marxianus var. drosophilarum" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces drosophilarum" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Candida sphaerica" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces marxianus lactis" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Kluyveromyces lactis" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces lactis var. drosophilarum" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:58628" datatype="xsd:string"></span>Kluyveromyces lactis</span>                            yhc1  p[CEN URA3 YHC1] test strain FOA any <br /> highlighted, addition strong conservation                 temperature tested. We conclude (i) C-terminal 61 <br /> N-terminus (35/40 positions chain identity                      aa dispensable Yhc1 function vivo; (ii) the <br /> similarity), modest conservation distal                   segment aa 151-170 needed optimal Yhc1 <br /> segment respective proteins (64/159 positions                    function; (iii) segment aa 134&#8211;151 essential <br /> side chain identity similarity), punctuated                      cell viability. <br /> peptide inserts 19-aa C-terminal tail unique to <br /> S. cerevisiae Yhc1 (Figure 1). We used alignment                   Effects mutating zinc-binding site Yhc1 <br /> guide serial C-terminal truncations Yhc1, sites <br /> denoted reverse arrows Figure 1. The wild-type                       The NMR structure N-terminal segment human <br /> and truncated alleles placed CEN HIS3                              U1C (38) revealed globular fold nucleated a <br /> plasmids control native YHC1                              Cys6-Cys9-His24-His30 zinc coordination complex that <br /> promoter tested plasmid shuf&#64258;e                                  conserved yeast Yhc1 (Figure 2A B). Mutating <br /> complementation yhc1  p[CEN URA3 YHC1]                                human U1C Cys6 Cys9 serine reported to <br /> strain. The resulting YHC1-(1-212), YHC1-(1-189)                       abolish binding U1C human U1 snRNP, as <br /> YHC1-(1-170) strains viable FOA selection                       did mutating His24 His30 glutamine (39). To probe <br /> and grew wild-type YHC1 cells YPD agar                       contributions zinc-binding chains to <br /> at temperatures tested, gauged colony size                       yeast Yhc1 activity vivo, mutated individually <br /> (Figure 1). The YHC1-(1-150) strain viable sick                    alanine context full-length Yhc1. The C6A, <br /> at temperatures (Figure 1). The YHC1-(1-133), YHC1-                    C9A, H24A H30A alleles tested plasmid <br /> (1-116), YHC1-(1-87) YHC1-(1-65) alleles                          shuf&#64258;e yhc1  complementation. Although C6A was <br />                                                                                    Nucleic Acids Research, 2014, Vol. 42, No. 7 4701 <br />  <br />  <br />  <br />  <br /> Figure 3. Synthetic interactions Yhc1-Ala mutants Mud2 Nam8. Yeast strains bearing indicated YHC1 alleles mud2  (panel <br /> A) nam8  (panel B) backgrounds spot-tested growth temperatures speci&#65533;?ed. YHC1-R21A failed complement mud2  yhc1  a <br /> plasmid shuf&#64258;e assay deemed synthetic lethal. Alleles strong synthetic growth defects indicated o left. <br />  <br />  <br />  <br /> unconditionally lethal, surprised &#65533;?nd C9A,               50 SS, protein components U1 <br /> H24A H30A cells viable. The H24A strain                     snRNP (6,10,12,40,41,42). In view, likely candi- <br /> relatively healthy, growing YPD agar tempera-                  dates mediate RNA protein contacts be <br /> tures colony size slightly smaller wild-type                   set conserved basic hydrophilic <br /> (Figure 2C). The C9A H30A strains grew                       chains emanate surface N- <br /> low temperatures (18&#8211;20  C) did thrive high                   terminal domain (Figure 2A). Therefore, targeted <br /> temperatures (30&#8211;37  C) (Figure 2C). These results                  surface-exposed residues alanine scanning: <br /> that Yhc1 function vivo sustained Cys6                Arg3, Tyr12, Thr14, His15, Arg21, Lys22, Asn29, Lys28 <br /> any zinc-binding residues                     Asp36 (Figure 2A B). The YHC1-Ala alleles were <br /> present. It conceivable that, Cys9, His24                     tested yhc1  complementation plasmid shuf&#64258;e and <br /> His30 replaced alanine, water substitute                retain biological activity; of <br /> the missing amino acid sulfur nitrogen atom                    YHC1-Ala strains grew wild-type YHC1 on <br /> tetrahedral zinc coordination complex, albeit                  YPD agar temperatures tested (Supplementary <br /> loss stability/activity, re&#64258;ected slower growth             Figure S1). <br /> or temperature sensitive (ts) growth respective <br /> alanine mutants.                                                         Synthetic genetic interactions Yhc1-Ala mutants with <br />                                                                          Mud2, Nam8, Mud1 Tgs1 <br /> Effects alanine mutations conserved surface residues <br />                                                                          To survey genetic interactions apparently fully <br /> in N-terminal module <br />                                                                          active Yhc1-Ala mutants, constructed series of <br /> The N-terminal domain Yhc1/U1C imputed make                     yeast strains genes encoding inessential <br /> physical contacts 50 leader sequence U1                      splicing factors Mud2, Nam8 Mud1 deleted in <br /> snRNA, pre-mRNA sequence vicinity                      yhc1  p[CEN URA3 YHC1] background, thereby <br />  4702 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br /> allowing tests synergy plasmid shuf&#64258;e. Although the <br /> ablation Mud2 effect yeast vegetative growth <br /> by itself, mud2  caused synthetic lethality combination <br /> with YHC1-R21A allele (Figure 3A). mud2  elicited <br /> strong &#8216;synthetic sick&#8217; phenotypes combination with <br /> YHC1 mutants K28A Y12A, manifested no <br /> growth 30&#8211;37  C minimal growth  25  C <br /> (Figure 3A). YHC1-T14A cold-sensitive (cs) <br /> growth defect mud2  background. The YHC1 <br /> mutants R3A, H15A, K22A, N29A D36A displayed <br /> no synergy mud2 . <br />    Loss U1 snRNP subunit Nam8 effect yeast <br /> vegetative growth, nam8  caused strong synthetic <br /> growth defect combination YHC1 mutant R21A <br /> allele, YHC1-R21A nam8  strain formed             Figure 4. Synthetic interactions Yhc1-Ala mutants Tgs1. Yeast <br />                                                               tgs1  yhc1  strains bearing indicated YHC1 alleles spot- <br /> pinpoint colonies 37 34  C failed grow       tested growth temperatures speci&#65533;?ed. The stronger synthetic <br />  30  C. By contrast, YHC1-Ala         growth defects indicated o left. Alleles milder synergies <br /> mutants displayed growth defect nam8                 denoted  . <br /> background (Figure 3B). <br />    Two YHC1-Ala mutants conditional <br /> synthetic defects yeast strain lacked Mud1      thrive 25 20  C. The YHC1-Y12A U1-U5C strain <br /> subunit U1 snRNP. YHC1-R21A mud1  cells           displayed mild slow growth phenotype all <br /> ts 37  C cs 18  C. YHC1-Y12A cells ts       temperatures. <br /> 37  C. The seven Ala mutants grew normally          Extending 50 leader sequence adding 25 nt <br /> mud1  background (Supplementary Figure S2).                   upstream native 50 end (as depicted in <br />    Tgs1 enzyme responsible conversion m7G       Supplementary Figure S3) effect se yeast <br /> RNA caps m2,2,7G trimethylguanosine caps      vegetative growth. However, U1+25 allele syn- <br /> a signature feature U1, U2, U4 U5 snRNAs           thetic lethal mud2  synthetic sick nam8  <br /> (17,43,44). Yeast tgs1  cells viable grow     tgs1  (25). Here examined effects otherwise <br /> 30&#8211;37  C, fail grow 18&#8211;20  C. Here,         benign Yhc1-Ala mutants U1+25 background. <br /> constructed tgs1  yhc1D p[CEN URA3 YHC1] strain             Again, YHC1-R21A allele uniquely synthetically <br /> to test synergies absence TMG cap        sick U1+25 (Supplementary Figure S3). <br /> the Yhc1-Ala mutants. We YHC1-R21A                The 50 AUACUUACCU10 single-stranded segment of <br /> tgs1  strain sick, forming pinpoint colonies      U1 snRNA precedes folded U1 RNA tertiary structure <br /> 37 34  C failing thrive 30  C (Figure 4). The   initiates nucleotide U11 (i.e. four-<span id='am-48' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span>helix</span> junction <br /> YHC1-Y12A tgs1  strain similar strong synthetic         depicted Figure 6C). The 8CCU10 segment connecting <br /> phenotype (Figure 4). The T14A K28A alleles elicited      50 SS complementary motif helix 1 interacts the <br /> mild synthetic effects tgs1  background (Figure 4).    U1C subunit U1 snRNP (6,10). We showed <br />                                                               previously spacing 50 SS complemen- <br /> Synthetic genetic interactions Yhc1-Ala mutants       tary motif helix 1 important, insofar insertions of <br /> alterations U1 snRNA                                   5 nucleotides C8 C9 yeast U1 <br /> The base-pairing interaction 50 leader            snRNA uniformly lethal (25). By contrast, insertion <br /> sequence 3ACUUAC8 yeast U1 snRNA                   single G nucleotide, [+1] allele Figure 6C, <br /> consensus 50 SS yeast pre-mRNAs depicted             viable. However, [+1] insertion lethal com- <br /> Figure 5. Recently, reported mutating U1              bination nam8 , mud1 , tgs1  mud2  (25). <br /> snRNA nucleobase U5 C effect yeast               Here Yhc1 mutants R21A, K22A and <br /> vegetative growth, signifying Py5:U mispair     K28A displayed severe synthetic sick phenotypes a <br /> U1 50 SS RNA <span id='am-98' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-99' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> suf&#65533;?ces seemingly normal U1           U1-[+1] background (Figure 6A). The T14A U1-[+1] <br /> snRNA activity (25). Yet, observed strong genetic          H15A U1-[+1] strains milder synthetic <br /> interactions U1-U5C mud2  (unconditional              defect, growing slowly 25&#8211;34  C (Figure 6A). <br /> synthetic lethality) nam8  (slow growth 37            Helix 1 conserved respect presence and <br /> 34  C failure grow 30, 25 20  C) (25).   position yeast human U1 snRNAs. To probe the <br /> Here tested genetic interactions U1-U5C    effects increasing length yeast helix 1, 9CU10 <br /> Yhc1-Ala mutants, constructing U1  yhc1  p[CEN           dinucleotide changed 9UA10, extending the <br /> URA3 U1 YHC1] strain, co-transforming                 base-pairing potential sequence 30 end U1 <br /> YHC1-Ala U1-U5C plasmids, screening growth            RNA, helix 1 span 8 bp instead 6 bp <br /> on FOA spot testing FOA-resistant YHC1-Ala            (Supplementary Figure S4). This U1 allele, named H1L <br /> U1-U5C strains growth YPD agar (Figure 5). The         (&#8216;helix 1 long&#8217;), supported normal yeast growth was <br /> YHC1-R21A U1-U5C strain uniquely sick, forming            synthetic lethal mud2  synthetic sick with <br /> pinpoint colonies 37, 34 30  C, failing         nam8 , mud1  tgs1  (25). Of collection of <br />                                                                                        Nucleic Acids Research, 2014, Vol. 42, No. 7 4703 <br />  <br />  <br />  <br />  <br /> Figure 5. Synthetic interactions Yhc1-Ala mutants U1 snRNA mutant U5C. The U1-U5C yhc1  strains bearing indicated YHC1 alleles <br /> were spot-tested growth temperatures speci&#65533;?ed. The severe synthetic growth defect R21A allele denoted o left. The milder <br /> synergy Y12A denoted  . <br />  <br />  <br />  <br /> Yhc1-Ala mutants, R21A uniquely synthetic sick                       U1 snRNP ejection early spliceosome is <br /> the U1-H1L background (Supplementary Figure S4).                            alleviated certain alterations presumed to <br />    The strongest genetic interactions Yhc1-Ala                       weaken U1 50 SS contacts. <br /> mutants U1 snRNA seen spacing                               Yhc1 &#65533;?gures prominently U1 snRNP-Prp28 <br /> between 50 SS complementary motif helix 1                       dynamic, insofar Chen et al. (4) identi&#65533;?ed genetic <br /> shortened deletion U10 nucleotide, depicted                    screen independent mutations amino acid <br /> in Figure 6C. The U1-DU10 single mutant grows normally                      Yhc1 (Leu13 changed Phe Ser) allowed <br /> at 30&#8211;34  C does thrive cold temperatures                        growth prp28  strain. They proceeded test 19 <br /> (Figure 6B). We reported previously U1-DU10                        possible substitutions Leu13 (i) any <br /> synthetic lethal nam8 , mud1 , tgs1  mud2                          amino acid position 13 suf&#65533;?ced complementation <br /> (25). Here, Yhc1 mutants T14A, R21A                       yhc1  (ii) 13 amino acids allowed some <br /> K28A synthetic lethal U1-DU10, Y12A,                      level growth prp28  (4). They proposed the <br /> H15A, K22A N29A displayed severe synthetic sickness                     conserved Leu13 residue &#8216;makes speci&#65533;?c contact&#8217; stabil- <br /> in U1-DU10 background (Figure 6B). However,                             ize U1 50 SS duplex (4). The subsequent report the <br /> on contrary, YHC1-D36A allele elicited gain-                      NMR structure human U1C (38) revealed that <br /> of-function effect U1-DU10 background, fully                         Leu13 chain points globular core the <br /> suppressing U1-DU10 cs growth defect (Figure 6C,                        N-terminal zinc-binding domain, makes van der <br /> compare WT D36A).                                                       Waals contacts Cd2 His24 chain that <br />                                                                             coordinates zinc ion (Figure 2A). The structure <br /> Bypass essentiality Prp28 Yhc1 R21A                        indicates Leu13 position interact with <br /> K28A mutations                                                              protein RNA, making likely Prp28 <br /> Yeast Prp28 essential pre-mRNA splicing factor                    bypass elicited Leu13 mutations indirect effect of <br /> a member DEAD-box <span id='am-35' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> nucleic                                  conformational change Yhc1 N-terminal domain. <br /> acid-dependent NTPases (45,46). Prp28 implicated                            We envisioned scenario subtraction of <br /> genetically displacing U1 snRNP 50 SS                       surface-exposed chains Yhc1 N-terminal <br /> during transition pre-mRNA U1 U2-contain-                        domain weaken U1 50 SS contacts thereby <br /> ing spliceosome pre-mRNA U2 U4 U6 spliceosome                          bypass requirement Prp28. To <span id='am-38' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-39' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span>idea</span> the <br /> (4,5,47). The strongest evidence idea emerges                 test, constructed yhc1  prp28  p[CEN URA3 <br /> the &#65533;?ndings essentiality Prp28 vegetative                   PRP28 YHC1] strain, transfected wild-type <br /> growth bypassed mutations essential U1                     YHC1 YHC1-Ala plasmids and <br /> snRNP subunits Yhc1, Prp42 Snu71, speci&#65533;?c                        screened transformants growth FOA, i.e. <br /> U1 snRNA mutations located &#64258;anking                           survival absence PRP28. Two YHC1 mutant <br /> segment base pairs intron 50 SS (4,25,47).                    alleles passed test: R21A K28A. The viable <br /> These results highlight need Prp28                      yhc1  prp28  cells bearing bypass alleles were <br />  4704 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br />  <br />  <br /> Figure 6. Genetic interactions Yhc1-Ala mutants U1 snRNA mutants [+1] DU10. Yeast strains bearing indicated YHC1 alleles in <br /> the U1-[+1] (panel A) U1-DU10 (panel B) backgrounds spot-tested growth temperatures speci&#65533;?ed. YHC1 alleles failed to <br /> complement U1-DU10 yhc1  plasmid shuf&#64258;e assay deemed synthetic lethal indicated panel B. Alleles strong synthetic <br /> growth defects indicated o left. Suppression U1-DU10 growth defect YHC1-D36A denoted asterisk left. (Panel C) The <br /> primary predicted secondary structures 50 end wild-type U1 RNA [+1] insertion DU10 deletion mutants shown. The <br /> 3 <br />  ACUUAC8 segment pairs intron 50 SS highlighted light gray. The paired bases comprise helix 1 highlighted dark gray. <br />  <br />  <br />  <br />  <br /> Figure 7. Yhc1 R21A K28A bypass requirement essential PRP28 gene. Yeast prp28  yhc1D cells harboring indicated YHC1 <br /> allele CEN HIS3 plasmid wild-type PRP28 (CEN LEU2) CEN LEU2 plasmid (prp28 ) grown liquid cultures at <br /> 37  C mid-log phase. The cultures adjusted A600 0.1, aliquots serial dilutions spotted YPD agar. Photographs the <br /> plates incubation 2 d (30, 34, 37  C) 4 d (20, 25  C) shown. <br />                                                                                       Nucleic Acids Research, 2014, Vol. 42, No. 7 4705 <br />  <br />  <br />  <br />  <br /> Figure 8. Disposition U1C structure U1 snRNP. (A) Stereo view U1C N-terminal domain (light green surface model) a <br /> nearby segment U1 snRNA 5.5 A&#730; crystal structure human U1 snRNP (pdb 3CW1). The NMR structure U1C-(1-31) (from <br /> pdb 2VRD) superimposed crystal structure, selected chain depicted stick models Zn2+ ion magenta sphere. The <br /> phosphate atoms U1 snRNA shown spheres nucleotide numbers speci&#65533;?ed nearby; 3ACUUAC8 motif complementary the <br /> 50 SS blue. The U1 50 end base-pairs counterpart adjacent U1 snRNP complex crystal, mimicking pairing U1 with <br /> nucleotides 50 SS pre-mRNA intron (colored beige). (B) Stereo view U1C (green surface model) SmD3 (pink surface model) <br /> subunits human U1snRNP, superimposed NMR structure U1C, highlighting proximity Arg21 (green stick model) SmD3 residues <br /> His11 (H), Glu36 (E) Asp37 (D). The U1C Asp36 residue indicated green letter D. <br />  <br />  <br />  <br /> tested growth YPD agar 20, 25, 30, 34 37  C                  required. The YHC1-K28A allele allowed Prp28 <br /> (Figure 7). R21A better Prp28 bypass sup-                   bypass 37 34  C, effective 30  C <br /> pressors, insofar YHC1-R21A prp28  cells grew                     ineffective 25  C (Figure 7). <br /> as wild-type cells 37, 34 30  C, gauged by <br /> colony size (Figure 7). That grew slowly 25  C and <br /> did thrive 20  C signi&#65533;?ed R21A bypass effect               Interactions Yhc1 Arg21 SmD3 subunit the <br />                                                                            Sm ring <br /> is cs (Figure 7). This result suggests higher tempera- <br /> tures help destabilize weakened U1 50 SS pairing                    Inspection low-resolution U1 snRNP structure <br /> YHC1-R21A cells permit U1 snRNP                                containing U1C N-terminal domain (with the <br /> ejection assistance Prp28. However,                    higher-resolution U1C fold superimposed) indicates <br /> the pairing stable lower temperatures, Prp28                 Yhc1/U1C Arg21 residue disposed the <br />  4706 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br />  <br />  <br /> Figure 9. Genetic interactions SmD3 mutants. (A B) Mutational synergies mud1  mud2 . Yeast smd3  strains bearing the <br /> indicated SMD3 allele CEN LEU2 plasmid wild-type (panel A), mud1  (panel B) mud2  (panel B) background were <br /> spot-tested growth YPD agar temperatures speci&#65533;?ed. (C) Bypass prp28 . Yeast smd3  prp28  cells bearing wild-type SMD3 or <br /> SMD3-E37A-D38A CEN LEU2 plasmid p(CEN HIS3 PRP28) CEN HIS3 plasmid grown liquid culture 37  C. <br /> After adjustment cell density (by A600), serial 10-fold dilutions spotted YPD agar. The plates photographed incubation <br /> for 2 d 30, 34 37  C 3 d 25  C. <br />  <br />  <br /> protein surface opposite face contacts                     tested (Figure 9A). We queried SMD3 alleles for <br /> intron 50 SS 50 segment U1 snRNA (Figure 8A).                      ability complement smd3  mud2  smd3  <br /> Rather, view Figure 8B reveals Arg21 likely                 mud1  strains. Although synergies evident for <br /> component interface Yhc1/U1C SmD3                         SMD3-N12A, E37A-D38A allele severely synthet- <br /> subunit Sm protein ring. Although atomic contacts                     ically sick absence Mud2, resulting micro- <br /> cannot surmised structures, proximity                     scopic growth YPD agar 25  C 18  C no <br /> the basic Arg21 residue human SmD3 residues His11,                        growth higher temperatures (Figure 9B). Also, the <br /> Glu36 Asp37 (H, E D Figure 8B) hints                         E37A-D38A allele elicited tight ts growth defect at <br /> Arg21 participate ionic and/or hydrogen bonding                     37  C mud1  background (Figure 9B). Thus, loss <br /> interactions corresponding chains                              SmD3 Glu37 Asp38 moieties, of <br /> (Asn12, Glu37 Asp38 yeast SmD3)                                poised make salt bridge Yhc1 Arg21, <br /> putative Yhc1-SmD3 interface yeast U1 snRNP.                          recapitulated synthetic genetic interactions <br /> Thus, speculated perturbing Yhc1-SmD3                            described R21A, fortifying hypothesis the <br /> interaction      accounts              widespread                    SmD3-Yhc1 interface pertinent genetics of <br /> mutational synergies YHC1-R21A, splicing                          Yhc1 Arg21. <br /> depends Mud2, Mud1, TMG cap forth,                                As test model, asked the <br /> because R21A hypomorph affects formation                       SMD3-E37A-D38A allele bypass prp28  found <br /> and/or stability U1 pre-mRNA [Msl5 Mud2] early                        did, allowing growth SMD3-E37A-D38A <br /> spliceosome intermediate. This view consistent                   prp28  cells 37 34  C (Figure 9C). The SMD3- <br /> &#65533;?nding R21A bypasses lethality prp28 ,                           E37A-D38A prp28  strain grew slowly 30  C did <br /> presumably R21A mutation destabilizes                            thrive  25  C, reaf&#65533;?rming prp28  bypass <br /> the U1 50 SS complex later spliceosome                                mutants generally cs. The SMD3-N12A allele did not <br /> intermediate (pre-mRNA U1 U2 U5 U4 U6)                               bypass prp28 . We infer contacts Yhc1 <br /> the putative substrate Prp28-mediated U1 snRNP                           Arg21 SmD3 Glu37 and/or Asp38 stabilize U1 <br /> ejection.                                                                    snRNP 50 SS complex. <br />    To evaluate scenario, established plasmid <br /> shuf&#64258;e assay gauge effects mutations N12A                          Yhc1-D36A exacerbates cs Prp28 mutations <br /> and E37A-D38A putative interface yeast SmD3                        The &#65533;?nding YHC1-D36A allele suppressed cs <br /> with Yhc1 Arg21. smd3  cells bearing SMD3-N12A                           growth defect U1-DU10 (Figure 6C) suggested the <br /> SMD3-E37A-D38A plasmids viable grew                         D36A change stabilize interactions U1 <br /> as smd3  SMD3 cells YPD agar temperatures                          snRNA snRNP 50 SS. If case, the <br />                                                                                 Nucleic Acids Research, 2014, Vol. 42, No. 7 4707 <br />  <br />  <br />  <br />  <br /> Figure 10. Yhc1-D36A exacerbates cs Prp28 mutations. Cultures yhc1  prp28  cells bearing indicated PRP28 YHC1 alleles CEN <br /> HIS3 CEN LEU2 plasmids, respectively, grown 34  C spot-tested growth YPD agar. The plates photographed after <br /> incubation 3 d 30 34  C 4 d 25  C. <br />  <br />  <br />  <br /> stability-enhancing YHC1-D36A allele ought synergize              231 essential speci&#65533;?c genetic <br /> with cs mutants Prp28, presumed                   circumstance. <br /> defective dissociating U1 snRNP                         At extreme, YHC1-(1-170) allele <br /> spliceosome low temperatures, U1 50 SS                   encoding shortest biologically active Yhc1 <br /> base-pairing interactions stable,             proteins lethal combination mud1 , tgs1  <br /> function higher temperatures destabilize                 mud2  U1 snRNA mutants U5C, +25, <br /> U1 50 SS duplex. As separate effort chart             H1L, [+1] DU10 (Figure 11). Yet, YHC1-(1-170) <br /> the structure-function relations yeast Prp28 alanine           entirely benign nam8  background (+++ <br /> scanning, identi&#65533;?ed mutations conferred cs               growth; Figure 11, panel). This result <br /> growth: K227A R499A. Lys227 located                    diverged sharply strong synergy nam8  <br /> the Prp28 P-loop motif (GSGKT) imputed                     YHC1-R21A (Figure 3B) suggested the <br /> contact ATP b- g-phosphates. Arg499                       N C domains Yhc1 distinctly buffered func- <br /> predicted, based crystal structure homolo-             tions pre-mRNA splicing. <br /> gous DEAD-box protein Vasa (48), make salt bridges                   The intermediate truncation allele YHC1-(1-189) <br /> to Glu342 Asp344 chains Prp28                    displayed spectrum mutational synergies different <br /> DEAD-box. The restrictive temperatures PRP28                 genetic backgrounds. Because YHC1-(1-189) mud1  cells <br /> K227A R499A strains 25 20  C, respectively              grew YHC1 mud1  cells all <br /> (Figure 10 <span id='am-93' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span>data</span> shown). However,                      temperatures tested (Figure 11, panel), whereas <br /> YHC1-D36A background, restrictive temperatures                   YHC1-(1-170) mud1  cells inviable, surmise <br /> were shifted upward, YHC1-D36A PRP28-                      Yhc1 segment aa 171 189 genetically <br /> K227A cells did grow 30  C YHC1-D36A                      redundant Mud1 subunit U1 snRNP. We <br /> PRP28-R499A cells did thrive 25  C (Figure 10).               YHC1-(1-189) mud2  cells grew lower <br /> These results consistent idea                      temperatures (18&#8211;25  C) tightly ts failed to <br /> Yhc1-D36A enhances U1 50 SS stability. Asp36                      thrive 34&#8211;37  C (Figure 11, panel). In <br /> located long helix follows globular            comparison lethality mud2  YHC1-(170) <br /> N-terminal domain U1C protein (Figure 8B).                    normal growth YHC1 mud2  cells, result <br /> Interactions Asp36 surmised                    signi&#65533;?es Yhc1 segment aa 171 189 <br />                                                                      compensates absence Mud2 low <br /> structure. <br />                                                                      temperatures, Yhc1 segment aa 190 to <br />                                                                      212 required buffer Mud2 loss high temperatures. <br /> Genetic interactions Yhc1 C-terminal truncations                  The YHC1-(1-189) tgs1  strain displayed mild <br />                                                                      synthetic defect compared YHC1 tgs1 , seen as <br /> In light allele-speci&#65533;?c mutational synergies                  smaller colony size 34&#8211;37  C (not shown) scored as <br /> involving N-terminal domain Yhc1, proceeded                ++ growth (Figure 11). Given tgs1  lethal with <br /> to survey genetic interactions benign Yhc1                    YHC1-(1-170), invoke genetic redundancy of <br /> C-terminal truncations. The results (compiled Figure              Yhc1-(171-189) TMG cap. <br /> 11, panel) disclosed informative hierarchy syn-               On severe end synergy spectrum, YHC1- <br /> thetic mutational effects. YHC1-(1-212) uniquely                 (1-189) lethal combined U1 snRNA <br /> lethal combination U1 snRNA mutant DU10                  allele U1-[+1]. Thus, Yhc1 segment aa 190 to <br /> had effect growth combination mud1 ,                   212 uniquely essential genetic background, <br /> tgs1  mud2  U1 snRNA mutants U5C, +25                     tested. When combined U1 <br /> or H1L. The YHC1-(1-212) U1-[+1] strain displayed                  snRNA variants, YHC1-(1-189) displayed ts growth <br /> mild synthetic defect compared YHC1 U1-[+1]                     defect U1-H1L (Figure 10, panel) cs <br /> (scored ++ growth). We conclude                slow growth defect U1+25, thriving with <br /> dispensable C-terminal segment Yhc1 aa 213                U1-U5C. Collectively, &#65533;?ndings indicate the <br />  4708 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br />  <br />  <br /> Figure 11. Synthetic genetic interactions Yhc1 C-terminal truncations. (Top panel) Plasmid shuf&#64258;e assays used test truncated <br /> YHC1 alleles functional genetic backgrounds speci&#65533;?ed left column. Synthetic lethality indicated failure form colonies on <br /> FOA agar 8 d 20, 30 37  C. Cultures viable FOA-resistant cells grown YPD broth 30  C growth phenotypes were <br /> assessed spotting serial 10-fold dilutions. Exemplary spot tests shown panel. Growth tabulated panel follows: +++ <br /> denotes growth similar pertinent strains bearing wild-type YHC1; ++ indicates smaller colony size strain bearing wild-type YHC1; ts <br /> signi&#65533;?es temperature-sensitive growth defect; cs indicates cold-sensitive growth defect. <br />  <br />  <br />  <br />  <br /> Yhc1 C-terminal domain buffers impact diverse                          did alanine substitutions conserved <br /> hypomorphic mutations 50 end U1 snRNA.                          surface-exposed amino acids Yhc1 N-terminal <br />    Finally, tested active Yhc1 C-terminal                        domain. Even signature zinc-binding site the <br /> truncation mutants ability bypass prp28                          globular N-terminal domain tolerated non-lethal alanine <br /> and able (not shown).                            changes zinc ligands. <br /> Thus, prp28  bypass effect appears speci&#65533;?c                          The low-resolution crystal structure human U1 <br /> mutations globular N-terminal domain Yhc1.                          snRNP containing N-terminal domain U1C <br />                                                                               subunit (10) revealed globular fold nucleated by <br />                                                                               tetrahedral zinc complex (rendered light green <br />                                                                               surface model Figure 8A) proximity the <br /> DISCUSSION <br />                                                                               conserved single-stranded U1 snRNA 50 leader sequence <br />                                                                               1 <br /> The present study comprises mutational analysis                        AUACUUACCU10. In crystal, U1 leader is <br /> essential U1 snRNP subunit Yhc1, guided large                      annealed RNA strand adjacent U1 <br /> the NMR structure homologous U1C subunit                        RNA form short duplex mimics base-paired <br /> human U1 snRNP. Deletion analysis alanine                                 segment formed conserved U1 snRNA 3ACUUAC8 <br /> scanning revealed surprising (to us) high tolerance                     motif (depicted blue spheres phosphorus atoms in <br /> subtraction Yhc1 structural elements, removal                      Figure 8A) 50 SS pre-mRNA intron <br /> of C-terminal 61 aa impact yeast growth                         (rendered beige spheres phosphorus atoms in <br />                                                                         Nucleic Acids Research, 2014, Vol. 42, No. 7 4709 <br />  <br />  <br /> Figure 8A). In Figure 8, superimposed NMR               junction H1 helix duplex stem the <br /> structure U1C globular fold low-resolution       stem-loop 3 (SL3) secondary structure (10). It makes sense <br /> envelope U1C U1 snRNP crystal structure             loss Tyr12 Thr14 deleterious U1-UD10 <br /> (pdb 3CW1, does include atomic models             background, distance 3ACUUAC8 <br /> the amino acid chains RNA nucleotides). This           motif pairs 50 SS start H1 helix <br /> model implicates conserved chains          shortened. The synergy Y12A T14A mud2D <br /> mutated presently potential contact points           suggests lesions impact cross-intron <br /> intron 50 SS, U1 snRNA TMG cap H1 helix.            interactions U1 snRNP branchpoint- <br /> Apparently, putative interactions essential   binding protein. <br /> per se U1 snRNP function.                                     The YHC1-H15A allele severely sick of <br />                                                                genetic backgrounds tested, U1-DU10, mildly <br /> Novel genetic interactions Yhc1 N-domain                sick U1-[+1] extra nucleotide inserted <br /> mutations                                                      C8 C9 (Supplementary Figure S5). These <br />                                                                results resonate crystal structure U1 <br /> Deeper insights emerged survey collection     snRNP aligned U1C N-domain, places <br /> fully active Yhc1 mutants synthetic loss function       His15 near short U1 snRNA segment connects <br /> phenotypes different genetic contexts:          3ACUUAC8 motif start H1 helix. The <br /> absence inessential U1 snRNP subunits Nam8              structures suggest His15 contacts nucleotide 11 the <br /> and Mud1; absence TMG cap U1                  U1 snRNA (G11 human U1; U11 yeast U1). <br /> snRNA (and snRNAs); absence the <br /> Mud2 subunit Msl5 Mud2 branchpoint binding <br />                                                                Synergies Yhc1 C-domain mutations <br /> protein complex presence &#65533;?ve different <br /> biologically active U1 snRNAs mutations             The C-terminal half U1C present crystal <br /> &#64258;anking U1 segment base-pairs 50 SS       structure human U1 snRNP. Lack conservation <br /> the pre-mRNA intron. The mutational synergies             segment U1C yeast Yhc1 hints they <br /> many notable hierarchy severity allele        distinct functions dictated signi&#65533;?cant <br /> speci&#65533;?city, summarized Supplementary Figure S5.          differences RNA protein composition the <br />    The R21A mutation Yhc1 elicited severe          human yeast U1 snRNPs. By conducting analysis <br /> and widespread effects, lethal absence         effects C-terminal truncations yeast Yhc1, we <br /> Mud2, severely sick absence Nam8              traced wide network genetic interactions in- <br /> TMG cap mildly sick absence Mud1. The            essential segments Yhc1 C-terminal domain all <br /> R21A lesion conferred lethality severe sickness     components early spliceosome were <br /> the presence &#65533;?ve U1 snRNA variants tested. We           interrogated here, notable exception Nam8. <br /> inferred superimposed U1 snRNP U1C                The nam8  allele itself synthetic lethal severely sick <br /> structures Yhc1 Arg21 contacts SmD3 Glu37             absence Mud1, Mud2 TMG caps, when <br /> and/or Asp38 chains, loss contacts      combined benign mutations U1 snRNA an <br /> might account synthetic growth defects              essential splicing factor (17,21,25,26,32&#8211;34,49), including, <br /> YHC1-R21A ability bypass prp28 .                    shown here, N-terminal R21A mutation in <br /> Consistent hypothesis, mutating        Yhc1. The fact nam8  effect combined <br /> SmD3 Glu37 Asp38 alanine effect               YHC1-(1-170), nam8  YHC1-(1- <br /> growth se mimicked phenotypes           170) similar spectrum synergies other <br /> seen YHC1-R21A, including prp28  bypass,                   spliceosome components, suggest epistatic, <br /> near-lethality mud2  failure grow 37  C      i.e. Nam8 Yhc1 C-terminal domain act same <br /> the mud1  background. The genetics implicates              (genetically buffered) pathway spliceosome <br /> Yhc1(R21) SmD3         interface       stabilizing      assembly. <br /> U1 pre-mRNA [Msl5 Mud2]             early       spliceosome <br /> intermediate. <br />    The consequential lesion Yhc1,            SUPPLEMENTARY DATA <br /> respect synergies, K28A, lethal          Supplementary Data available NAR Online. <br /> U1-DU10 severely sick U1-[+1] mud2  <br /> (Supplementary Figure S5). The equivalent Arg28 side <br /> chain human U1C poised interact              ACKNOWLEDGEMENTS <br /> pre-mRNA intron 50 SS:U1 snRNA duplex (Figure 8A). <br /> We attribute various synthetic phenotypes YHC1-         S.S. American <span id='am-43' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Society Research Professor. <br /> K28A, ability bypass lethality prp28 ,    B.S. recipient Hearst Foundation Award. <br /> the loss Yhc1 RNA interaction turn affects <br /> the stability U1 50 SS complex. <br />    Alanine substitutions Tyr12 Thr14 caused strong      FUNDING <br /> synthetic growth defects diverse genetic backgrounds        National Institutes Health (NIH) [GM52470 S.S. <br /> (Supplementary Figure S5). Tyr12 Thr14 human            GM102961 B.S.]. Funding open access <br /> U1C located near human U1 snRNA (Figure 8A),           charge: NIH [GM52470]. <br />  4710 Nucleic Acids Research, 2014, Vol. 42, No. 7 <br />  <br />  <br /> Con&#64258;ict statement. None declared.                              19. Abovich,N., Liao,X.C. Rosbash,M. (1994) The yeast MUD2 <br />                                                                                protein: interaction PRP11 de&#65533;?<span id='am-88' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Nes" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nes</span> bridge between <br />                                                                                commitment complexes U2 snRNP addition. Genes Dev., 8, <br />                                                                                843&#8211;854. <br /> REFERENCES                                                                 20. Chang,J., Schwer,B. Shuman,S. (2010) Mutational analyses of <br />  1. Fabrizio,P., Dannenberg,J., Dube,P., Kastner,B., Stark,H.,                 trimethylguanosine synthase (Tgs1) Mud2: proteins implicated <br />     Urlaub,H. Lu&#168;hrmann,R. (2009) The evolutionarily conserved             pre-mRNA splicing. RNA, 16, 1018&#8211;1031. <br />     core design catalytic activation step yeast              21. Qiu,Z.R., Schwer,B. Shuman,S. (2011) Determinants of <br />     spliceosome. Mol. Cell, 36, 593&#8211;608.                                       Nam8-dependent splicing meiotic pre-mRNAs. Nucleic Acids <br />  2. Warlocki,Z., Odenwa&#168;lder,P., Schitzova,J., Platzmann,F., Stark,H.,         Res., 39, 3427&#8211;3445. <br />     Urlaub,H., Ficner,R., Fabrizio,P. Lu&#168;hrmann,R. (2009)              22. Hilleren,P.J., Kao,H.Y. Siliciano,P.G. (1995) The amino- <br />     Reconstitution steps Saccharomyces cerevisiae splicing          terminal domain yeast U1-70K necessary suf&#65533;?cient for <br />     puri&#65533;?ed spliceosomal components. Nat. Struct. Mol. Biol.,             function. Mol. Cell. Biol., 15, 6341&#8211;6350. <br />     16, 1237&#8211;1243.                                                         23. Ester,C. Uetz,P. (2008) The FF domains yeast U1 snRNP <br />  3. Hoskins,A.A., Friedman,L.J., Gallagher,S.S., Crawford,D.J.,                protein Prp40 mediate interactions Luc7 Snu71. BMC <br />     Anderson,E.G., Wombacher,R., Ramirez,N., Cornish,V.W.,                     Biochem., 9, 29. <br />     Gelles,J. Moore,M.J. (2011) Ordered dynamic assembly           24. Go&#168;rnemann,J., Barrandon,C., Hujer,K., Rutz,B., Rigaut,G., <br />     single spliceosomes. Science, 331, 1289&#8211;1295.                           Kotovic,K.M., Faux,C., Neugebauer,K.M. Se&#180;raphin,B. (2011) <br />  4. Chen,J.Y., Stands,L., Staley,J.P., Jackups,R.R., Latus,L.J.            Cotranscriptional spliceosome assembly splicing are <br />     Chang,T.H. (2001) Speci&#65533;?c alterations U1-C protein U1                independent Prp40p WW domain. RNA, 17, 2119&#8211;2129. <br />     small nuclear RNA eliminate requirement Prp28p,          25. Schwer,B., Chang,J. Shuman,S. (2013) Structure-function <br />     essential DEAD box splicing factor. Mol. Cell, 7, 227&#8211;232.                 analysis 50 end yeast U1 snRNA highlights genetic <br />  5. Staley,J.P. Guthrie,C. (1999) An RNA switch 50 splice           interactions Msl5 Mud2 branchpoint binding complex <br />     site requires ATP DEAD box protein Prp28p. Mol. Cell,              spliceosome assembly factors. Nucleic Acids Res., 41, <br />     3, 55&#8211;64.                                                                  7485&#8211;7500. <br />  6. Tang,J., Abovich,N., Fleming,M.J., Se&#180;raphin,B. Rosbash,M.         26. Chang,J., Schwer,B. Shuman,S. (2012) Structure-function <br />     (1997) Identi&#65533;?cation characterization yeast homolog            analysis genetic interactions yeast branchpoint binding <br />     U1 snRNP-speci&#65533;?c protein C. EMBO J., 13, 4082&#8211;4091.                        protein Msl5. Nucleic Acids Res., 40, 4539&#8211;4552. <br />  7. Gottschalk,A., Tang,J., Puig,O., Salgado,J., Neubauer,G.,              27. Colot,H.V., Stutz,F. Rosbash,M. (1996) The yeast splicing <br />     Colot,H.V., Mann,M., Se&#180;raphin,B., Rosbash,M., Lu&#168;hrmann,R.                factor Mud13p commitment complex component and <br />     et al. (1998) A comprehensive biochemical genetic analysis          corresponds CBP20, small subunit nuclear cap- <br />     yeast U1 snRNP reveals &#65533;?ve novel proteins. RNA, 4, 374&#8211;393.            binding complex. Genes Dev., 10, 1699&#8211;1708. <br />  8. Fortes,P., Bilbao-Corte&#180;s,D., Fornerod,M., Rigaut,G.,                  28. Abovich,N. Rosbash,M. (1997) Cross-intron bridging <br />     Raymond,W., Se&#180;raphin,B. Mattaj,I.W. (1999) Luc7p, novel             interactions yeast commitment complex conserved in <br />     yeast U1 snRNP protein role 50 splice site recognition.          mammals. Cell, 89, 403&#8211;412. <br />     Genes Dev., 13, 2425&#8211;2438.                                             29. Tang,J., Abovich,N. Rosbash,M. (1996) Identi&#65533;?cation and <br />  9. Schwer,B., Erdjument-Bromage,H. Shuman,S. (2011)                       characterization yeast gene encoding U2 small nuclear <br />     Composition yeast snRNPs snoRNPs absence                  ribonucleoprotein particle B&#8217;&#8217; protein. Mol. Cell. Biol., 16, <br />     trimethylguanosine caps reveals nuclear cap binding protein           2787&#8211;2795. <br />     gained U1 component implicated cold-sensitivity tgs1         30. Fortes,P., Kufel,J., Fornerod,M., Polycarpou-Schwarz,M., <br />     cells. Nucleic Acids Res., 39, 6715&#8211;6728.                                  Lafontaine,D., Tollervey,D. Mattaj,I.W. (1999) Genetic and <br /> 10. Pomeranz Krummel,D.A., Oubridge,C., Leung,A.K.W., Li,J.                physical interaction involving yeast nuclear cap-binding <br />     Nagai,K. (2009) Crystal structure human spliceosomal U1                 complex. Mol. Cell. Biol., 19, 6543&#8211;6553. <br />     snRNP 5.5 A&#730; resolution. Nature, 458, 475&#8211;480.                      31. Tong,A.H., Evangelista,M., Parsons,A.B., Xu,H., Bader,G.D., <br /> 11. Weber,G., Trowitzsch,S., Kastner,B., Lu&#168;hrmann,R.                      Page,N., Robinson,M., Raghibizadeh,S., Hogue,C.W., Bussey,H. <br />     Wahl,M.C. (2010) Functional organization Sm core             et al. (2001) Systematic genetic analysis ordered arrays of <br />     crystal structure human U1 snRNP. EMBO J., 29, 4172&#8211;4184.               yeast deletion mutants. Science, 294, 2364&#8211;2368. <br /> 12. Van der Feltz,C., Anthony,K., Brilot,A. Pomeranz                   32. Wilmes,G.M., Bergkessel,M., Bandyopadhyay,S., Shales,M., <br />     Krummel,D.A. (2012) Architecture spliceosome.                       Braberg,H., Cagney,G., Collins,S.R., Whitworth,G.B., Kress,T.L., <br />     Biochemistry, 51, 3321&#8211;3333.                                               Weissman,J.S. et al. (2008) A genetic interaction map RNA- <br /> 13. Siliciano,P.G. Guthrie,C. (1988) 50 splice site selection           processing factors reveals links Sem1/Dss1-containing <br />     yeast: genetic alterations base-pairing U1 reveal additional       complexes mRNA export splicing. Mol. Cell, 32, <br />     requirements. Genes Dev., 2, 1258&#8211;1267.                                     <br /> </body></html>