<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>l l: . 1994 Oxford University Press                                          Nucleic Acids Research, 1994, Vol. 22, No. 25 5555-5564 <br />  <br />  <br /> The budding yeast U5 snRNP Prp8 highly conserved <br /> protein links RNA splicing cell cycle <br /> progression <br /> Jacqueline E.Shea+, Jeremy H.Toyn Leland H.Johnston* <br /> Laboratory Yeast Genetics, National Institute Medical Research, The Ridgeway, Mill Hill, <br /> London NW7 1AA, UK <br />  <br /> Received October 14, 1994; Revised Accepted November 18, 1994                                              GenBank accession no. L29421 <br />  <br />  <br /> ABSTRACT <br /> The dbf3 mutation originally obtained screen                          The U1, U2, U4/U6 U5 snRNPS extensively <br /> for DNA synthesis mutants cell cycle phenotype                          studied, Ul snRNP particle interact the <br /> in budding yeast <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. We                              substrate mRNA precursor, followed U2 <br /> have isolated DBF3 gene                             snRNP form pre-spliceosome complex. <br /> an essential gene ORF 7239 nucleotides,                             Concurrently, U4/U6 U5 snRNPs associate form a <br /> potentially encoding large protein 268 kDa. We                       triple snRNP joins pre-spliceosome complex <br /> obtained allele-specific high copy number                                   form spliceosome [For reviews (1-5)]. <br /> suppressor dbf3-1 allele, encoded known                             Studies budding yeast useful identifying some <br /> SSB1 gene, member Hsp7O <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> heat                                protein components spliceosome include the <br /> shock proteins. The sequence Dbf3 protein                            important prp mutants (precursor mRNA processing) many <br /> 58% identical 2300 amino acid residues                               originally designated ma mutants. These <br /> predicted protein <span id='am-33' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span>Caenorhabditis elegans</span>.                                 mutants defects pre-mRNA splicing cause a <br /> Furthermore, partial sequences 61% amino acid                             temperature sensitive (ts) conditional lethal phenotype (2,6,7). <br /> sequence identity deduced files                               Over 30 PRP genes identified date, the <br /> human cDNA EST nucleotide database                              PRP3, PRP4, PRPS, PRP7, PRP8 PRP11 shown <br /> Dbf3 highly conserved protein. The nucleotide                             vital spliceosome assembly vitro (1,8,9). <br /> sequence DBF3 turned identical                                The PRP8 gene, subject study, encodes protein <br /> yeast gene PRP8, encodes U5 snRNP required                             component U5 snRNP. The Prp8 protein associated with <br /> for pre-mRNA splicing. <span id='am-45' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49'  ></span>This</span> surprising result led                        spliceosome transesterification reactions (10,11). <br /> further characterise phenotype dbf3                               Functional Prp8 protein required formation the <br /> confirmed role cell cycle showed                          U4/U6.U5 tri-snRNP spliceosome form <br /> function early, time S phase. This <span id='am-53' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-57'  ></span>data</span>                          (9). Additionally, immunological UV crosslinking studies <br /> suggests hitherto unexpected link pre-                               shown Prp8 interacts directly pre-mRNA <br /> mRNA splicing cell cycle.                                              substrate possess important structural and/or <br />                                                                                regulatory role yeast spliceosome (12). Moreover, proteins <br /> INTRODUCTION                                                                   related Prp8 identified immunologically in <br />                                                                                mammalian cells eukaryotes emphasises that <br /> The removal introns eukaryotic messenger RNA                           Prp8 plays important role splicing be <br /> precursors requires sequential transesterification reactions               conserved <span id='am-50' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-51' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> boundaries (11,13-15). <br /> which place complex ribonucleoprotein particle,                   Analysis yeast demonstrated genetic interactions between <br /> the spliceosome. Assembly spliceosome involves                          PRP8 number genes. For example, suppressor <br /> interactions large number protein factors                prp8-1 turned dominant mutation SRNl <br /> small nuclear RNA molecules U1, U2, U4, U5 U6                          (suppressor RNA mutations) gene. The SRN-J mutation was <br /> snRNAs. Some protein factors bound snRNAs                     suppress ts defect prp2 prp6 cells can <br /> form small ribonucleoproteins (snRNPs), whilst non-snRNP                 suppress rna2/prp2, rna3, rna4, rnaS, rna6/prp6 and <br /> proteins transient association spliceosome.               rna8/prp8 ts mutations singly pairs (16). Surprisingly, <br />  <br /> *To correspondence addressed <br /> +Present address:   Department Infectious Diseases Bacteriology, Royal Postgraduate Medical School, Hammersmith Hospital, Du Cane Road, London, <br /> UK <br />  5556 Nucleic Acids Research, 1994, Vol. 22, No. 25 <br />  <br />  SRNI gene the HEX2/REGI                JL342       MATa adel ade2 dna39-3 gall his7-2 lys2 tyrl ural <br />  gene acts negative regulator glucose repression          L149a       MATa adel dbJ3-l his7-2 lysi trpl-289 ura3-52 <br />  (17). This finding suggests link, direct      JLIU        MATa dna39-1 his7-2 trpl-289 ura3-52 <br />                                                                         JL2U        MATa adel dna39-2 his7-2 trpl-289 ura3-52 <br />  indirect, carbon catabolite metabolism mRNA                JL3U        MATa adel dna3-3 his7-2 leu2-3 tyri trpl-89 <br /> precursor splicing. In paper unexpected        CG378/CG379 MA Ta/ls ura3/ura3 leu2/leu2 trpl/trpl ade5/+ +/hiss7 <br /> link involving mRNA splicing, connection cell         D273        MA Ta ade hisl trp2 <br /> cycle. We isolated DBF3 gene, mutant allele            SPJ8-31     MATa prp8-1 ura3 leu2 his3 <br /> possesses cell cycle phenotype indicative defect DNA          SPJ8-2      MATa prp8-1 leu2 trpl ura3 <br />                                                                         J16a        MATa prp8-1 adelade2 ural his7 lys2 tyrl gall <br />  synthesis (18,19) surprised identical       JDY8-22     MATae prp8-2 ura3 leu2 lys2 his3 <br /> PRP8 gene.                                                              JDY8-31     MA4Ta prp8-3 ura3 his3 <br />    In budding yeast, genetic approaches understanding        DJY76       MATxi prp8-7 leu2 ura3 his7 tyrl <br /> cell cycle functions regulated began isolation       JDY8-81     MATTa prp8-8 ura3 leu2 trpl his3 <br /> cell division cycle (cdc) mutants (20). Mutations CDC gene <br /> are generally temperature-sensitive (ts) result defect      Strains JL130, JL253 JL342 obtained Alan <br /> a particular cell cycle stage specific function restrictive      Wheals, University Bath. CG378 CG379 Dr <br /> temperature leads accumulation cells uniform      C.Giroux, Detroit. The prp8 strains kindly provided by <br /> morphology [reviewed (21)]. The dbf (dumb-bell former)               Andrew Brown Jean Beggs, University Edinburgh. Strains <br /> mutants cell cycle mutants originally isolated           L149a, JL1U, JL2U, JL3U J219-3D constructed during <br /> attempt identify genes involved control S phase. The       study. Construction spo12A::TRP1 genotype been <br /> three mutants dbJ2, 3 4 possess ts defect DNA synthesis,       described (27). <br /> but apparent defect RNA metabolism (18,19), <br /> although, significantly, mutant, dbfs, allelic      Media general methods <br /> to ma3/prp3 (18). The DBF2 gene isolated encodes           YPD (28) YNB (Difco) media used rich minimal <br /> a cell cycle regulated protein kinase functioning               medium, respectively. Cell numbers determined use of <br /> controlling S phase mitosis (22,23). Its principal role     Coulter Counter (Coulter Electronics, Dunstable, UK). <br /> is pathway bringing end mitosis. The DBF4             <span id='am-58' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> introduced yeast treated lithium acetate <br /> gene expressed cell cycle control acts immediately         (29). Preparation cells flow cytometry described <br /> prior S phase essential subunit Cdc7 protein   previously (27). <br /> kinase (24). Indeed, Dbf4 target Cdc7 origins of <br /> replication binds subunit ORC protein complex <br /> that, turn, binds origins (25). The mutant, dbf3,       Cloning disruption DBF3/PRP8 gene <br /> originally characterised having defect initiation DNA    Seven plasmids complemented temperature sensitive <br /> synthesis suggesting Dbf3 protein function          growth dbf3-1 strain L149-7B isolated clone <br /> control Gl/S phase transition (18,19). Consistent        bank yeast chromosomal DNA vector YEp24. Six these <br /> S phase role Dbf3, alleles mutant            plasmids rescued dbj3-1 temperature-sensitivity efficiently and <br /> isolated independent screen mutants defective DNA          restriction analysis confirmed contained DNA the <br /> synthesis (26).                                                         genomic region. The seventh plasmid rescued dbJ3-1 <br />    Here confm DBF3 role cell cycle             defect weakly contain SSBJ gene (see <br /> the timing function coincides S phase. At restrictive   Results). For deletion chromosomal DBF3 gene, YEp24 <br /> temperature 37&#176;C dbJ3-1 cells arrest approximately           containing 9.8 kbp region DBF3 locus digested with <br /> 2C content DNA undivided nucleus. DNA                    BamHI remove 6.0 kb DBF3 ORF then <br /> replication probably narrowly fails completed majority     religated. The truncated DBF3 gene removed the <br /> of dbJ3-1 cells restrictive temperature. Sequencing showed       YEp24 vector digestion Sail ClaI inserted <br /> DBF3 identical PRP8 predicted protein 2413            Sal ClaI sites located polylinker a <br /> amino acid residues. This 58% homologous           modified pBluescriptKS vector (Stratagene) form plasmid pKS- <br /> 2300 amino acid residues predicted protein                dbJ3A. pKS-dbj3A digested BamHI URA3 gene <br /> nematode worm <span id='am-40' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span>Caenorhabditis</span> elegans. Additionally, parts            inserted form plasmid pKS-dbJ3A::URA3. This plasmid <br /> a human DBF3/PRP8 homologue 61 %                  digested EcoRI resulting fragment <br /> identical yeast sequence 87% identical worm           containing URA3 flanked DBF3 sequences used to <br /> sequence. Thus DBF3/PRP8 unique gene encoding large              transform diploid strain CG378/CG379. Integration the <br /> protein highly conserved wide range eukaryotes.         fragment disruption chromosomal copy the <br /> From study gene yeast appears role           DBF3 gene confirmed Southern hybridisation analysis. <br /> in cell cycle previously characterised role in <br /> pre-mRNA splicing.                                                      a-Factor synchronisation <br />                                                                         Cultures synchronised a-factor grown mid- <br /> MATERIALS AND METHODS                                                   logarithmic phase YPD (5 x 106 1 x 107 cells/ml) at- <br /> Yeast strains                                                           factor added final concentration 2.5 /g/ml. Incubation <br /> L149-7B      MATa   dbJ3-1 trp-289 ura3-52                              continued percentage budded cells declined <br /> J219-3D      MATa   arg4 his7-2 leu2-3 spol2D::TRPI trpl-289            5%. The a-factor removed filtration and <br /> JL130        MATa   adel ade2 dna39-1 gall his7-2 lys2 tyrl ural        washing. The cells resuspended appropriate density <br /> JL2U         MATa   adel ade2 dna39-2 gall his7-2 lys2 tyrl ural        fresh medium, prewarmed 25 37&#176;C required. <br />                                                                               Nucleic Acids Research, 1994, Vol. 22, No. 25 5557 <br />  <br /> Transposon sequencing mutagenesis                           viable spore. All viable spores Ura-. Some non-viable <br /> Generation TnlOOO transposon insertions target plasmids        spores germinated produce single dumbbell-shaped cell, <br /> and use sequencing strategy essentially described (30).    formed microcolony cells after <br /> An extension method used locate dbf3-1 rescuing     7 days incubation. Microdissection cells showed a <br /> activity SSBJ gene seventh plasmid (31).             small proportion capable cell division required <br />                                                                        48 h so. The daughter cells produced these <br />                                                                        divisions small rarely divided themselves. Thus <br /> RESULTS                                                                DBF3 functionally required mitotic growth. <br /> Cloning genetic mapping DBF3/PRP8 gene                      The dbJ3-1, dna39 prp8 mutations allelic <br /> The wild type DBF3 gene cloned complementation               We surprised discovery DBF3 PRP8 were <br /> the dbJ3-1 ts mutation localised 9.4 kb region contained      gene previously dbf3-1 allele <br /> on 9.8 kb insert (Fig. 1). Northern hybridisation analysis           complemented rna8-1/prp8-1 ts defect dbJ3-1 <br /> revealed single transcript 7.8 kb (results shown)             mutant appeared effect RNA synthesis 37&#176;C <br /> confirming large size gene. The identity cloned      (18,19). We repeated complementation tests by <br /> DNA authentic DBF3 locus confirmed integrative           mating dbJ3-1 strain different strains containing the <br /> mapping (32). DBF3 located chromosome VIII                   prp8-1 allele prp8 alleles. All dbJ3-J/prp8-J <br /> hybridisation yeast pulsed-field chromosome blot (33). This       diploids capable growth 37&#176;C confirming earlier <br /> was confirmed genetic mapping, showed DBF3               result. However, crosses dbJ3-1 prp8 <br /> is linked SP012 (16 parental ditypes, 0 non-parental ditypes        alleles resulted diploids unable grow 37&#176;C confirming <br /> and 4 tetratypes). Use standard equations places DBF3 10            allelism (Table 1). One prp8-1/dbJ3-1 diploids that <br /> centimorgans SP012 right arm chromosome VIII.           grew 37&#176;C sporulated tetrads dissected. Amongst <br /> DBF3 showed linkage ARG4, unclear          39 spores recombinant capable growth 37&#176;C, <br /> side SP012 gene DBF3 gene located. None           consistent mutations gene. The <br /> mutations previously mapped region phenotypes <br /> similar dbf3. Thus, unlikely gene. <br /> Determination nucleotide sequence (see below) showed                                                                                            B BC <br />                                                                                                                      Ss H      Bg <br /> that DBF3 reading frame starts 221 nucleotides downstream          A                                        Bg <br /> of CDC23, genes transcribed direction.                                              BE   SIP        P            E |IB          P// <br />                                                                                                    EH~o     II&apos;                                 Ik] <br /> The sequence DBF3 shows identical PRP8 <br /> Approximately 7845 bp DNA containing dbf3-1                                                                                                              1 kb <br />                                                                                                                                                                LJ-- <br /> complementing activity sequenced strands. One large <br /> open reading frame (ORF) 7239 nucleotides found <br /> (GenBank accession number L29421), capable encoding               B <br /> protein 2413 amino acids calculated molecular weight <br /> of 268 kda (Fig. 2). The 3&apos; end CDC23 gene (34)                                                                                             C <br /> found 221 nucleotides upstream DBF3/PRP8. The predicted <br /> Dbf3/Prp8 protein highly hydrophilic estimated pI <br /> of 7.85. Interestingly, 78 residues N-terminal                                                                        B&apos;H <br /> region runs containing 5, 8, 7 7 proline <br /> residues. There potential nuclear localisation signal (35)                                                                    URA3 <br /> located downstream runs proline residues positions <br /> 109-124. <br />   After completed sequencing DBF3 gene, PRP8            C <br /> sequence (36) available databases,                              M <br /> that DBF3 PRP8 (RNA8) gene.                                                                          Bg <br />                                                                                                S                       C            E        E          H <br /> Although predicted proteins Dbf3 Prp8 identical                H     P               E <br />                                                                                                                                     I <br /> numbers amino acid residues, number minor                                                                              8L...0...8l <br /> differences. There frame shift causes difference between <br /> the proteins residues 386-419. This genuine <br /> polymorphism reading amino acids                                                          Tn5 Tn6 Tn2 Tnl <br />                                                                                                                                         3 <br />                                                                                                                                                               500bp <br /> region equally homologous corresponding region                                                                         4 <br /> worm gene (see below). Additionally, single <br /> amino acid substitutions positions 1132 1566 protein. <br />                                                                        Figure 1. Restriction map DBF3 SSB1, disruption strategy for <br /> The DBF3/PRP8 gene essential mitotic growth                     DBF3. (A) Partial restriction map DBF3. Abbreviations restriction sites: <br /> Isolation ts mutation does necessarily prove gene    B, BamHI; Bg, BgEl; C, ClaI; E, EcoRI; H, HindHI; P, PstI; S, Sall; Ss, SstI. <br /> is essential growth. We disrupted chromosomal        The direction transcription indicated arrow. (B) Disruption DBF3 <br />                                                                        URA3 described Materials Methods. (C) Partial restriction map of <br /> copy DBF3 gene diploid strain using URA3 (Fig. 1B)         SSBJ gene flanking sequences. The insertion sites TnlOOO <br /> and following sporulation asci dissected. Of tetrads,         <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="transposons" datatype="xsd:string"></span>transposons</span> SSB1 gene destroyed suppressor activity indicated <br /> eight contained viable spores contained          arrows. <br />  5558 Nucleic Acids Research, 1994, Vol. 22, No. 25 <br />                                                                            858 KN8QERQRQIQKNGP8ITP8RATrIFSVMVEWLESRSFSPIPFPPLTYKN 1 907 <br />    1 NSGLPPPPPGF88DS 15 <br />   16   DLALPPPPPPPPGYEIBELDNPMVPSSVN8DTFLPPPPPPPSNFEINA88          65      776 KS8Q8RQHNYLKDGPYISAEEAVAIYTTTVHWLESRRFSPIPFPPLSYKH 1 825 <br />   66   IVDFTLPPPPPPPGLDE..LETKAEKKVELHGKRKLDIGKDTFVTRKSRK          113     908 DTKILVLALEDLKDVYASKVRLNASEREELALI88AYDNPHDTLNRI KKY 957 <br />    1   MANYGGHPQTEP38AIPDSILEEKSRKWKQLQGKR ....      ...........   34      826   DTKLLILALERLKESYSVKNRLNQSQR8ELALIEQAYDNPHEALSRI KRH    875 <br />  114   RA1CMTKKAK8SNLYTPKAEMPPKHLRKIINTHSD88ASJQYSTDJKAFLK         163     958 LLTQRVFKPVDITMMSNYQNISPVYSVDPLEKITDAYLDQYLWYKADQRK 1007 <br />   35   .. .. YSEKKKFGMSDTQKE8MPPEHVRKVIRDHGDNTSRKYRHDKRVYLG        80      876   MLTQRAFKEV9I8F8MDLYTHLIPVYDIEPLEKVTIAYLDQYLWYEADKRR    925     1706   VHEW8VSKPSLLHETNDSFKGLITNKMWFDVQLRYGDYDSHDISRYVRAK 1755 <br />  164   ALKYLPHAILKLLSNMPHPWEQAKBVKVLYHTSGAITFVNETPRVIEPVY          213    1008 LFPNWIKPSDSEIPPLLVYKWTQGINNLSE IWDVSRGQSAVLLETTwGEM 1057 <br />                                                                                                                                                 1626   QYKWNVSRPSLMADSKDVMDNTTTQKYWLDVQLRWGDYDSHDVERYARAK          1675 <br />   81   ALKYMPHAVLKLLSNMPMPWEQIRDVKVLYHITGAITFVNDIPRVIEPVY 130              926 LFPAWVKPGDT8PPPLLTYKWCQGLNNLQDVWETSEGECNVIMETKLEKI 975           1756   FLDYTTDNVSMYPSPTGVMIGIDLAYNMYDAYGNWFNGLKPLIQNSMRTI 1805 <br /> 214    TAQWSATWIAMRR8KRDRTHFKRMRFPPFDDDEPPLSY8QHIENIE8PLDP         263    1058 ABKIDFTLLNRLLRLIVDPNIADYITAKNNVVINFKDMSHVNKYGLIRGL 1107 <br />                                                                                                                                                 1676   FLDYTTDNMSIYPSPTGVLIAIDLAYNLYSAYGNWFPGMKPLIRQANAKI          1725 <br /> 131    MAQWGTMWIMMRR8KRDRRHFKRNRPPPFDD88PPLDYADNILDVEPLEP          180     976 AEKMDLTLLNRLLRLIVDHNIADYMTS0KNNVLINYKDMHTNS FGIIRGL 1025         1806   MKANPALYVLRERIRKGLQIYQSSVQEPFLNSSNYA8LFMNNDIKLFVDDT         1855 <br /> 264    INLPLDSQDDBYVKDWLYDSRPLE8DSKKVNGTSYKICVSFDLPEMSNLYR         313    1108 KFASFIFQYYGLVIDLLLLGQERASDLAGPANNPNEFMQFKSKEVEKAHP 1157 <br />                                                                                                                                                 1726   IKANPAFYVLR8RIRKGLQLYSSEPTEPYLTSQNYGELFSNQIIWFVDDT          1775 <br /> 181    IQM8LDP88DGAVA8WFYDHKPL.ATTRFVNGPTYRKWAFSSIPQMSTLYR         2 29   1026 QFASFIVQFYGLVLDLLVLGLRRASEIAGPPQCPNEFLQFQDVATEIGHP 1075          1856   NVYRVTVHKTFEGNVATKAINGCIFTLNPKTGHLFLKIIHTSVWAGQKRL 1905 <br /> 314 LSTPLRD8VTDKNYYYLFDKKSFFNGKALNNAIPGGPKFEPLYPREEEED 363                1158 IRLYTRYLDRIYMLFHFEEDEGEELTDEYLAENPDPNFENSIGYNNRKCW 1207          1776   NVYRVTIHKTFEGNLTTKPINGAIFIPFNPRTGQLFLKIIHTSVWAGQKRL 1825 <br /> 2 30 LANQLLTDLVDDNYFYLFDMKSFFTAKALNVAI PGGPKFEPLVKDLHDED 2 79             1076 IRLYCRYIDRVWIMFRFSAD8ARDLIQRYLT8HPDPNNENIVGYNNKKCW 1125          1906   SQLAKWKTAEEVSALVRSLPK8EQPKQIIVTRKAMDPLEVHMLDFPNIA 1955 <br /> 364 YNWFNSIDRVIFRVPIRSEYKVAFLI .... YIIPGPVQCAYHGIIIQCRV 409              1208 PKDSRMRLIRQDVNLGRAVFWEIQSRVPTSLTSIKWENAFVSVYSKNNPN 1257          1826   SQLAKWKTAEEVAALIRSLPVEEQPRQIIVTRKAMLDPLEVHLLDFPNIV 1875 <br /> 280 WNEFNDINKVIIRAPIRTEYRIAFPFMYNNLISSLPVQVSWYH. .TPSVV 327               1126   PRDAR8RDUHDVNLGRAVFWDIKNRLPRSoITTVEW8NSFVSVYSKDNPN 1175        1956   IRPT8LRLPFSAAMSIDKLSDVVMKATEPQMVLFNIYDDWLDRISSYTAF 2005 <br /> 410 LSRTMRSTTRLALFFDPSLNPIPHFIDNNSSLNVSNTKENGDFTLPEDFA 459                1258 LLFSMCGFEVRILPRQR.MEEVVSNDEGVWDLVDERTKQRTAKAYLKVSE 1306 <br />                                                                                                                                                          1   11 11              111111111 11111 1111111 <br />                                                                                                                                                 1876   IKGSELMLPFQAIMKVEKFGDLI&apos;LKATEPQMVLFNLYDDWLKTISSYTAF 1925 <br /> 328 FIKTEDPDLP .AFYYDPLINPIVLSNLKAT8ENLP8G8EEDEWELPEDVR 376               1176 MLFDMSGFECRILPKCRTANEESFVHRDGVWNLQNEVTKERTAQCFLKVDE 1225         2006   SRLTLLLRALKTNEESAIILLSDPTITIKSYHLWPSFTDEQWITIESQM 2055 <br /> 460 PLLAEE8ELILPNTKDAMSLYHSPFPFNRTKGKMVRAQDVALAKKWFLQH 509                1307 EEIKKFDSRIRGILMASGSTFTKVAAKWNTSLISLFTYFREAIVATEPL 1356           1926   SRVVLIMRGMHINPDKTKVILKPDKTTIT8PHHIWPTLSDDDWIKVELAL 1975 <br /> 377 <br />         1    1   1 11         .1               11 <br />        PIF. EDVPLYTDNTANGLALLWAPRPFNLRSGRTRRAVDVPLVKSWYREH         425    1226 ESLSKFHNRIRQILMSSGSTTFTKIVNKWNTALIGLMTYFREAVVWNTQEL 1275         2056   RDLILTEYGRKYNVNISALTQTEIKDIILGQNIKAPSVKRQKMA8L8AAR 2105 <br /> 510    PDEEYPVKVKVSYQKLLKNYVLNELHPTLPTNHNKTKLLKSLKNrKYFQQ 559             1357 LDILVKGETRIQNRVKLGLNSKMPTRFPPAVFYTPKELGGLGMISASHIL 1406                  1 11.11111  1II <br />                                                                                                                                                                  1 II              11111I 1.111 .11.1 <br />                                                                                                                                                 1976 KDMILADYGKKNNVNVASLTQSEVRDIILBMEISAPSQQRQQIADI ....           2021 <br /> 426 CPAGMPVKVRVSYQKLLKVFVLNALKHRPPKPQKRRYLFRSFKATKFFQT 475                1276 LDLLVKCENKIQTRIKIGLNSKMPSRFPPVVFYTPKEIGGLGMLSMGHVL 1325          2106   SEKQNDEEAAGASTVMKTKTINAQGEEIVVVASADYESQTFSSKNEWRKS 2155 <br /> 560 TrIDWVEAGLQLCRQGHNMLNLLIHRKGLTYLHLDYNFNLKPTKTLTTKE 609                1407 IPASDLSWSKQTD.TGITHFRAGMTHEDEKLIPTIFRYITrTENEFLDSQ 1455          2022   .EKQTKEQSQVTAT. .TrRTVNKHGDEIITATTSNYSTASFASRTEWRVR         2068 <br /> 476    TTLDWVEAGLQVLRQGYNMLNLLIHRKNLNYLHLDYNFNLKPVKTLTTKE          52z5   1326 IPQSDLRWMSQTEAGGVTHFRSGMSHDEDQLIPNLYRYIQPWEAEFVDSV 1375          2156   AIANTLLYLRLKNIYVSADDFVEEQNVYVLPKNLLKKFI8ISDVKIQVAA 2205 <br /> 610 RKKSRLGNSFHLMRELLKKMXLIVDTHVQFRLGNVDAFQLADGIHYILNH 659                1456 RVWAEYATKRQEAIQQNRRLAFEELEGSWDRGIPRISTLFQRDRHTLAYD 1505          2069 <br />                                                                                                                                                         11   1.111-111.11       :..*111           11-11     11 <br />                                                                                                                                                        AISSTNLHLRTQHIYVNSDDVKDTGYTYILPKNILKKFITISDLRTQIAG 2118 <br /> 526    RKKSRFGNAFHLCRBILRLTKLVVDAHVQYRLNNVDAYQLADGLQYIFAH          575 <br />                                                                                  1111111 111H11i1111111l1111 <br />                                                                                                111 <br />                                                                           1376 RVWAEYALKRQEANAQNRRLTLEDLDDSWDRGIPRINTLFQKDRHTLAYD 1425          2206 FIYGMSAKDHPKVKEIKTWVLVPQLGHVGSVQISNIIPDIGDLPDTE ... .G 2252 <br /> 660 IGQLTGIYRYKYKVMHQIRACKDLKHIIYYKFNKN .LGKGPGCGFWQPAW 708               1506   RGHRIRREFKQYSLERNSOPFWWTNSHHDGKLWNLNAYRTDVIQALGGIET 1555       2119 FMYGVSPPDNPQVKEIRCIVLVPQ ... .TGSHQQVNLP. TQLPDHELLRD 2163 <br /> 576 VGQLTG YRYKYKLMRQVRMCKDLKHLIYYRFNTGPVGKGPGCGFWAPGW 625                1426 KGWRVRTEFKAYQILKQNPFWWTHQRHDGKLWNLNNYRTDMIQALGGVEG 1475          2253 LELLGWIHTQTEELKFMAASEVATHSKLFADKIR....DCIDISIFSTPG 2298 <br /> 709 RVWLNFLRGTIPLLERYIGNLITRQFEGR.SNEIVKTTTKQRLDAYYDLE 757                1556 ILEHTLFKGTGFNSWEGLFCEKASGFEDSMQFKKLTHAQRTGLSQIPNRR 1605 <br />                                                                                                                                                          111 <br />                                                                                                                                                            11        1          1.11.11.1         . I..      I11 <br />                                                                                                                                                 2164 FEPLGWMHTQPNELPQLSPQDVTTHAKLLTDNISWDGEKTVMITCSFTPG 2213 <br /> 626 RVWLFFLRGITPLL8RWLGNLLSRQFEGRHSKGVAKTVTKQRVESHFDLE 675                1476 <br />                                                                                    111111I 11111 Illll ll IIII <br />                                                                                  ILEHTLFRGTYFPTWEGLFWERASGFEESMKFKKLTNAQRSGLNQI PNRR    1525    2299 SVSLSAYNLTDEGYQWGR88NKIMNVLSEGFEPTFSTHAQLLLSDRITGN 2348 <br /> 758 LRNSVMDDILEMMEP8SIRQKKARTILQHLS8AWRCWKANIPWDVPGMPAP 807 <br />                                                                                                                                                             11 <br />                                                                                                                                                          111111 11 11                    1   11   .111111 <br />                                                                           1606   FTLWWSPTINRANVYVGFLVQLDLTGIIFLHGKIPTLKISLIQIFRAHLWQ 1655       2214 SVSLTAYKLTPSGY8WGKANlDKGNN .PKGYKPTHYEKVQMLLSDRFLGY 2262 <br />                                                                                  1111111111111 111i <br />                                                                                                  I III I111111 111111111111111111 <br /> 676    LRAAVMHDILDMMPDGIKQNKARVILQHLSSAWRCWKANIPWKVPGLPTP          725    1526   FTLWWSPTINRANVYVGFQVQLDLTGIFFMHGKIPTLKISLIQIFRAHLWQ    1575    2349 FIIPSGNVWNYTFMGTAFNQEGDYNFKYGIPLBFYNl8HRPVHFLQFSEL 2398 <br /> 808                                                                       1656 KIHESIVFDICQILDGELDVLQIESVTKETVHPRKSYKMNSSAADITMES 1705          2263 <br />                                                                                                                                                              .11111                 .:              111111 *1- <br />                                                                                                                                                        FMVPSNGVWNYNFQGQRWSPA8EKFDVCLSNPJONYYHEDHRPVHFHNFK          2310 <br />         :.  :I    111   ...   lll           111111 <br /> 726 VENMI LRYVKAKADWWTNSAHYNRERVRRGATVDKTVCKKNLGRLTRLYL <br />                                                                                  11111   1I <br />                                                                                          11 111 11111111111111 11 <br />                                                                           1576 K I8HE SWMDLCQVFDQE LDALE IQTVQKETI8HPRKS0YKMN8SSCADVLLFA 1625   2399   AGDEELKA8QIDVFS   2413 <br />  <br />  <br />  <br />  <br /> Figure 2. Alignment predicted yeast Dbf3 protein sequence homologous predicted protein C. elegans. The upper line shows sequence of <br /> the yeast protein lower line shows sequence worm protein. Vertical lines indicate amino acid residues identical sequences, <br /> and colons indicate conserved residues. The DBF3 nucleotide sequence GenBank accession number L29421. <br />  <br />  <br /> ability dbf3-1 allele complement ts defect                                                    Similarly, expressed sequence tag (EST) database (National <br /> prp8-1 allele indicates intra-allelic complementation                                             Center Biotechnology Information), contains mostly <br /> suggests like calmodulin Dbf3/Prp8 protein contain                                                 human transcribed sequence fragments unknown function, was <br /> independently functional domains (37).                                                                          searched human protein sequences significant <br />    The dna39 mutations isolated separate screen                                                 homology Dbf3 (39). These EST sequences <br /> for ts mutants defective DNA synthesis (26). Transformation                                                  correspond residues 1445-1556 1989-2106 the <br /> of dbJ3-1 dna39 strains cloned DBF3/PRP8 gene                                                  Dbf3/Prp8 protein, probable are <br /> rescued ts defect mutants equally                                                      different parts human gene. The alignment all <br /> complementation tests strains confirmed                                                       genes, human, worm yeast (Fig. 3), showed the <br /> allelism.                                                                                                       19 amino acid residues partial human sequence (Fig. <br />                                                                                                                 3A) homologous worm yeast, therefore <br /> DBF3/PRP8     highly conserved eukaryotic gene                                                             certain belong here. Thus, ignoring 19 <br /> Searches GenBank EMBL databases predicted                                                   amino acids, sequences contain 211 amino acids that <br /> amino acid sequence Dbf3/Prp8 protein revealed significant                                               compared organisms; yeast worm <br /> homology predicted protein Caenorhabditis elegans                                                     58% identical, yeast human 61% identical, whilst <br /> (worm) derived DNA sequence contained base                                                         human worm 87% identical regions. Since <br /> pairs 6247 14 007 cosmid C50C3, accession number                                                         identity yeast worm sequences these <br /> L14433 (38). Eight predicted exons region                                                 regions representative overall identity yeast <br /> 7074 14 007 corresponding putative protein labelled                                                  worm sequences, probable 58 87% good <br /> C50C3.6, unknown function. In addition, basis                                                estimates overall level identity human <br /> of homology Dbf3, appears ninth exon                                                    Dbf3/Prp8 protein. <br /> worm sequence positions 6190 7002. The exons <br /> together predict protein 58% sequence identity                                                  The dbJ3-1 mutation suppressed high copy <br /> Dbf3 2300 amino acids (Fig. 2). The homology extends                                                       expression SSB1 <br /> from position 83 Dbf3/Prp8 sequence end                                                   As described Materials Methods, seven library <br /> genes, does include proline rich domains                                                   plasmids obtained search DBF3 gene only <br /> the N-terminal region Dbf3.                                                                                  weakly complemented dbf3-1 defect 37&#176;C contained <br />                                                                                          Nucleic Acids Research, 1994, Vol. 22, No. 25 5559 <br />  <br />                              Table 1. Complementation dbJ3-1 alleles prp8 <br />  <br />                                 Strain       Genotype         Growth Diploid           Genotype          Strain <br />                                                               370C <br />                                 D273         dbf3-1                 +                      p8-              SPJ8-31 <br />                                 D273         dbf3-1                 +                      prp8-1           SPJ8-2 <br />                                 D273         dbf3-1                 +                      prp8-1           J 16a <br />                                 L149a        dbf3-1                                       prp8-2            JDY8-22 <br />                                L149a         dbf3-1                                       8prp-3            JDY8-31 <br />                                L149a         dbf3-1                                       prp8-7            DJY76 <br />                                L149a         dbf3-1                                       prp8-8            JDY8-81 <br />                                L149-7B       db13--                                        dna39-1          JL130 <br />                                L149-7B       dbf3-1                                        dna39-2          JL253 <br />                                L149-7B       dbf3-1 -                                      dna39-3          JL342 <br />                              The dbJ3-1 mutation genetic backgrounds shown left hand column crossed with <br />                              prp8 dna39 alleles shown right. Growth resulting diploids 37&#176;C shown <br />                              centre column. <br />  <br />  <br /> a different genomic insert plasmids containing authentic               A <br /> DBF3/PRP8 gene. Using combined transposon mutagenesis                       human (1) LKXGGCSXPR LHHCTPARE| E <br />                                                                                 worm 1365 QPDAEVE V31.WU I I <br /> sequencing (31) suppressor activity plasmid                     yeast 1445 TTNW1E Q           JI I <br /> mapped region insert contained SSBI gene                                                                               E    (112) <br /> (Fig. 1), member hsp70 heat shock protein family (40).                                                                             El   1476 <br /> Suppression dbJ3 SSBJ conditions specific                                                                              To   1556 <br /> for dbf3-1 allele. A number prp8 dna39 alleles                    B <br /> tested rescued SSBJ temperature                   human (1l)   I         IIXEIKE IK_ERf N UE l <br />                                                                                                                Vw3IIII POO <br />                                                                                                                                        IENIEWI <br /> 35&#176;C. This suggest specific interaction Ssbl                  worm 1909                                             IEEES3PDj <br />                                                                                 yeast 1989    EIlJER I _Z TIEU(TI EESEUDS FPITIKSYM LUSFEI3Qj <br /> the defective protein encoded dbJ3-1 allele, remains                    FEWVI X _WMP%fllg                 IO118) <br /> uncertain normally interact cell                           NAJ                 lgWCO VlX_1        _    _       2026 <br /> division wild-type yeast. A link heat shock proteins                       llTIISDIRS MrEER4YI MISA3I IE3ND KWVKMWM IDJAARS 2106 <br /> and pre-mRNA splicing previously described (41,42), <br /> however instance suppression ts defect                  Figure 3. Alignment partial Dbf3 sequences human, worm yeast. <br /> of snRNP protein over-expression heat shock protein.                    Protein sequences deduced human nucleotide sequences (A) EST05386 and <br />                                                                                   (B) EST05073 aligned corresponding regions yeast worm <br /> The dbJ3-1 mutation confers cell cycle arrest phenotype                         protein sequences. <br /> Our previous description dbJ3-1 mutant focused largely <br /> on effect DNA synthesis (18,19). Since connection <br /> between pre-mRNA splicing cell cycle, suggested the <br /> allellism PRP8 DBF3, entirely unexpected,                      apparent 2C DNA content, arrested cell cycle stage <br /> further characterised dbJ3-1 phenotype establish clearly                   metaphase anaphase transition. <br /> that role cell cycle. This important                    In prp8-1 culture, cell division ceased rapidly there <br /> none prp8 alleles examined showed uniform                        evidence cell cycle stage specific arrest divsion. <br /> cellular morphology 37&#176;C like dbJ3 (a &apos;dumbbell&apos;,                 This resulted arrested cells nuclear morphology <br /> an enlarged bud attached mother cell, Fig. 4D).                        DNA content remained original state time of <br />    To examine cell cycle arrest dbJ3-1, log phase cultures                 temperature shift. Thus prp8-1 results rapid cessation <br /> of dbJ3-1, prp8-1 wild type strain grown                        cell cycle 37&#176;C, independent cell cycle stage in <br /> permissive temperature shifted 37&#176;C 4 h. Samples                      progress. <br /> were taken hourly intervals determine cell density,                           In contrast dbj3-1 culture, cell numbers prp8-1 <br /> proportion budded cells proportion cells divided               culture increased 11 % shifting 37&#176;C. This raised <br /> chromatin (Fig. 4).                                                               possibility 46% increase cell numbers in <br />    In dbJ3-1 culture, 46 % cells completed cell division               dbJ3-1 culture 37&#176;C resulted leakiness <br /> after shifting restrictive temperature, expected cell             dbJ3-1 allele. We extended FACS analysis <br /> cycle mutant (data shown). After 4 h 37&#176;C, 75% cells                    dna39-3, allele tight <br /> had arrested buds (Fig. 4C) 75 % cells contained                 temperature sensitive mutant (Fig. 4E). This resulted higher <br /> a 2C content DNA (Fig. 4C A). However, essentially                         proportion population accumulating 1C DNA than <br /> none cells divided chromatin (Fig. 4B D). They                     occurred dbJ3-1, consistent function early cell <br /> had proceeded nuclear division, spite                  cycle (see below). Despite apparent cell cycle specific block <br />                                                                                                                    7~ <br /> 5560 Nucleic Acids Research, 1994, Vol. 22, No. 25 <br />  <br />                      A                           dbf3-1       pIp8-1                                          Wild Type <br />  <br />  <br />                                   Ohr <br />  <br />  <br />  <br />                                   1hr <br />  <br />  <br />  <br />                              C9   2hr <br />                              cu <br />  <br />                              E <br />  <br />  <br />                                   3hr <br />  <br />  <br />  <br />                                   4hr <br />  <br />  <br />  <br />         B       30                                            C                  100- <br />  <br />                                                                                  80 -                                                    -0 <br />             I <br />                 20 <br />             c <br />                                                                       ;          60 <br />                                                                       0 <br />             I                                                         0 <br />                                                                       00 <br />             a <br />             =   10 <br />                                                                                   20 <br />             0 <br />                  0                                                                 0 <br />                                                           S                               0               1          2               3       4        5 <br />                                         Time Hours                                                              Time Hours <br />  <br />  <br />  <br />                          D                                        E <br />                                              a <br />                                          : <br />                                                                                                    jdbf       1 <br />                                                                             0 <br />                                   9a                                       - <br />  <br />                                                                                               I1 <br />                                                                           a)r_    50 <br />                                                                            c <br />                                                                            0C <br />                                                                           C) <br />                                                                           0 <br />                                                                           z           0   - <br />                                                                           C                         dna   79-3 <br />  <br />                                                                                   50      - <br />                                                                            3.1 <br />                                                                           -c <br />                                                                           C) <br />                                                                                       0-                                                         In <br />                                                                                                                                          4       5 <br />                                                                                                                          Ho L   -s <br />                                                                                                    Nucleic Acids Research, 1994, Vol. 22, No. 25 5561 <br />  <br /> in dna39-3, cells did assume expected terminal                                  The dbf3 mutant specific defect early cell cycle <br /> phenotype &apos;dumb-bells&apos; mixed morphology even <br /> after protracted incubation 37&#176;C.                                                      With ts mutants having cell cycle defect, possible to <br />    Wild-type cells continued dividing temperature shift                           determine time cell cycle gene product <br /> but transient delay budding occurs presumably result                              functions. This time, known &apos;execution point&apos; is <br /> of heat shock effect known specifically delay                             characteristic particular cell cycle mutants confirms a <br /> cell cycle START (43). This transient effect observed                           cell cycle role gene product question. In practice one <br /> in dbJ3-1 culture.                                                                      normally determines final time cell cycle the <br />    Thus, dbJ3-J allele clearly cell cycle arrest phenotype                        particular protein carries essential function. To this <br /> but phenotype prp8-1 quite different.                                             synchronous cultures dbJ3-1 dna39-3 mutants were <br />  <br />  <br />  <br />                    A.            30- <br />                                                                                              C.          30- <br />                          E <br />                          U) <br />                                  25-                                                              (A <br />                                                                                                          25- <br />                          00                                                                        0 <br />                          0       20-                                                              7,)&apos; 20- <br />                           x                                                                       10)x <br />                           Un <br />                          L-      15-                                                               n     15- <br />                          .0 <br />                          E 10 <br />                          m <br />                                                                                                    E     10- <br />                          z                                                                        z <br />                                   5-                                                               a)     5- <br />                          0 <br />                                   0                                                                       0-                                             l <br />                                         l   I               I           I           &apos;                           I            I   I   I   I   I       l <br />                                         0       1       2       3           4   5       6                       0   1    2       3       4       5       6 <br />                                                         Time Hours                                                    Time Hours <br />  <br />                    B.           100 <br />                                                                                              D.          100- <br />  <br />                                  80-                                                                     80- <br />  <br />                                 ! 60-                                                               U1) 60- <br />                           C&apos;) <br />  <br />  <br />                                  40-                                                               0D 40- <br />                           a) <br />  <br />                          m       20-                                                              m      20- <br />                           0 <br />  <br />  <br />  <br />                                   0-                                                                      0- I~~~~~~~~~~~~~~~ <br />                                                     I           I   I <br />                                         0       1       2       3                                            0     1    2     3 <br />                                             Time Hours                                                           Time Hours <br />  <br /> Figure 5. Execution point analysis dbf3-1 dna39-3 Alleles. Mid-log phase cultures dbJ3-1 (strain L149a, panels A B) dna39-3 (strain <br /> JL342, panels C D) grown 25&#176;C YPD synchronised using ca-factor. The progress synchronous cell cycles following removal a-factor <br /> were monitored determining percentage budded cells 25&#176;C (panels B D). After release ca-factor, cell numbers measured (panels A and <br /> C) incubation 25&#176;C (open circles) 37&#176;C (closed circles). Additionally, samples transfered 15 min intervals 25&#176;C culture <br /> to 37&#176;C cell numbers determined 4 h incubation (squares) determine execution point. <br />  <br />  <br />  <br />  <br /> Figure 4. DNA synthesis asynchronous dbJ3-1, prp8-1 wild type cultures permissive restrictive temperatures. The dbf3-1 strain (L149-7B), prp8-1 <br /> strain (SPJ8-31) wildtype strain (CG378) grown 25&#176;C approximately 10 generations mid-log phase minimal medium. The cultures then <br /> shifted 37&#176;C samples taken hourly intervals determination DNA content cell flow cytometry (panel A). The percentage cells divided <br /> chromatin strain determined microscopically using propidium iodide stained cells shown panel B (filled circles, dbf3-1 culture; open circles, <br /> prp8-1 culture; squares, wild type culture). The percentage budded cells strain calculated shown panel C (figure legend panel <br /> B). dbJ3-1 dumbbell-shaped cells formed 4 h holding 37&#176;C contain single undivided nucleus (panel D). The cells stained propidium iodide as <br /> for flow cytometry photographed light UV illumination. Panel E shows proportion IC cells accumulate 37&#176;C culture dbf3-1 <br /> compared dna39-3. The IC cells calculated FACS analysis like panel A. Note dbJ3-1 data derived panel A independent <br /> experiment. <br />  5562 Nucleic Acids Research, 1994, Vol. 22, No. 25 <br />  <br />           A                                                                            prepared permissive temperature using a-factor method. <br />                                                                                        After synchronisation samples transferred 37&#176;C regular <br />                                                                                        timepoints ensuing cell cycle cell numbers <br />                                                                                        measured determine proportion cells complete <br />            i.a5                                                                        division cycle restrictive temperature. Cells have <br />           to0                                                                          passed execution point prior transfer 37&#176;C able to <br />            x                                                                           divide, cells passed execution point not <br />            0                                                                           divide. The percentage budded cells main cultures at <br />            E                                                                           25&#176;C determined, measure synchrony and <br />            z                                                                           indicate DNA synthesis commenced bud emergence <br />                                                                                        roughly coincident initiation S phase (44). Thus the <br />                                                                                      time cells acquired ability divide 37&#176;C could <br />                                                                                        compared timing S phase comparison with <br />                                       Time Hours                                    bud emergence. <br />           B                                                                               Importantly, samples cultures showed clear, <br />                   30                                                                   step-wise doubling cell numbers transferred 37&#176;C, <br />                                                                                        indicative specific cell cycle defect (Fig. 5). With dna39-3, <br />                                                                                        45 min incubation 25&#176;C post ca-factor release, cells <br />                   20 <br />                                                                                        acquired ability divide 37&#176;C (Fig. SC). At time <br />            &apos;E0                                                                         bud emergence culture 60% (Fig. 5D). This <br />            AxS                                                                         consistent early execution point time <br />                                                                                        initiation S phase. In contrast, dbJ3-1 cells able <br />                   10                                                                   divide 37&#176;C 1 h 15 min prior incubation 250C, at <br />            z                                                                           time budding plateau 90% (Fig. 5A and <br />                                                                                      B). This indicated execution point close end S phase. <br />            U                                                                           Thus, execution point estimates support role for <br />                     0 <br />                                          4        6                  10 <br />                                                                                        Dbf3 protein early cell cycle. <br />                                       Time Hours                                       A simple explanation difference alleles <br />                                                                                        mutant proteins differing stabilities 370C. <br />                                                                                        Alternatively, Dbf3 protein functions, of <br />           c                                                                            required beginning near end S <br />                   20                                                                   phase, different alleles affect function <br />                                                                                        severely. To help determine relationship DBF3 <br />                                                                                        function DNA synthesis, reciprocal shift experiments were <br />                                                                                        performed hydroxyurea (HU), inhibitor DNA <br />                                                                                        synthesis, restrictive temperature. Inititally dbf3-1 and <br />              x <br />              E <br />              0 <br />                                                                                        dna39-3 cultures blocked HU generation time <br />              (A                                                                        250C immediately transferred 37&#176;C removal <br />                                                                                        HU. If hydroxyurea-sensitive step (DNA synthesis) <br />                                                                                        precedes Dbf3 function cells unable divide, <br />                                                                                        Dbf3 function cells undergo cell <br />                                                                                        division arresting 37&#176;C. In experiment, strains <br />                         0         2           4           6           8                showed doubling cell numbers transfer 370C. In the <br />                                       Time Hours                                    case dna39-3 exact doubling (Fig. 6B), <br />                                                                                        dbj3-1 cell numbers doubled <br />                                                                                        suggesting allele somewhat leaky (Fig. 6A). Thus, <br />                                                                                        function Dbf3 precedes occurrence complete DNA <br /> Figure 6. Reciprocal shift experiments hydroxyurea 37&#176;C            replication. <br /> dbJ3-1 dna39-3 Alleles. (A) A mid-log phase dbf3-1 culture (strain L149-7B)           In second experiment dbJ3-1 culture held <br /> grown YPD 25&#176;C divided two; hydroxyurea added                370C 2.5 h (over generation time), addition <br /> of cultures concentration 300 mM (filled squares), culture <br /> was incubated absence hydroxyurea (open squares). After 3 <br />                                                                                        HU transfer 25&#176;C. The majority cells (60%) did <br /> h incubation hydroxyurea removed filtration, cells washed          divide HU (even 100% recover absence <br /> with saline culture split two. Half cells resuspended       HU), indicating cells HU-sensitive step is <br /> in prewarmed YPD 37&#176;C (filled circles), half resuspended         dependent prior Dbf3 function (Fig. 6C), consistent the <br /> in YPD 25&#176;C (open circles). The cultures continued incubation         experiment. Presumably 370C DNA must <br /> 5.5 h cell numbers determined 30 min intervals. (B) The same <br /> experiment carried dna39-3 culture (strain JL342),             remain unreplicated cells apparent 2C DNA <br /> difference cell numbers monitored 4.5 h removal           content judged FACS analysis (Fig. 4A). However, <br /> hydroxyurea. (C) As second stage experiment (A), mid-log          40% cells did divide HU. Possibly dbJ3-1 allele <br /> phase dbJ3-1 culture (strain L149-7B) grown YPD 25&#176;C transferred             leaky allows proportion cells fully replicate their <br /> to 37&#176;C generation time (filled circles). The culture divided <br /> into two; hydroxyurea added portion 300 mM (filled squares), whilst <br />                                                                                        DNA 370C. These cells resistant HU and <br /> the portion received hydroxyurea (open squares). Both cultures           divide 25&#176;C resulting 40% increase cell <br /> transferred 25&#176;C cell numbers determined.                                   numbers. This result confirmed dna39-3 <br />                                                                               Nucleic Acids Research, 1994, Vol. 22, No. 25 5563 <br />  <br /> strain, incubation mutant 37&#176;C rapidly lethal       protein. Thus polyproline sequences yeast <br /> and cells divide transfer 25&#176;C absence       specific involved S phase function Dbf3/Prp8 <br /> of HU. This suggests defect dna39-3 mutant               protein. Certainly, yeast U5 snRNP larger its <br /> irreversible.                                                           mammalian homologue additional functions <br />                                                                         (1). <br />                                                                            Apart proline sequences, Prp8/Dbf3 highly <br /> DISCUSSION                                                              conserved eukaryotic protein. It 58% identity a <br /> We cell cycle mutation dbJ3-1 allele          C.elegans putative protein, spread entire 2300 amino <br /> of PRP8, U5 snRNP. This result quite unexpected               acids protein, 61 % identity partial deduced <br /> led analyse dbJ3-1 phenotype confirm          sequence human protein. Furthermore, C. elegans and <br /> it does display specific cell cycle defect. The dbJ3-1         human proteins 87% identical. This level homology is <br /> mutant previously shown arrest 37&#176;C cellular           comparable cyclin dependent kinase p34CDC28 has <br /> morphology typical cell cycle defect (18,19). Here         60.5% identity yeast human, includes the <br /> that restrictive temperature 75% cells accumulate    catalytic domain common nearly protein kinases. This <br /> with approximately 2C content DNA cells arrest            similarity protein sequence confirms previous <br /> with single undivided nucleus. This important result            immunological data showed Prp8 homologue is <br /> showing cells progress mitosis, consistent        present higher eukaryotes. <br /> with specific block cell cycle. The remaining cells arrested       The discovery DBF3 PRP8 gene raises <br /> with 1C content DNA probably START. Presumably,                 possibility pre-mRNA splicing plays role cell <br /> in cells defect Dbf3/Prp8 protein prevents entry       cycle progression, snRNA complexes <br /> into cell cycle reflect differing levels functional      dual roles cell cycle progression splicing. <br /> Dbf3/Prp8 protein mutant cells.                                 A role pre-mRNA splicing cell cycle progression would <br />    A specific cell cycle defect dbf3-1 cells indicated    imply existence intron-containing gene encoding rate- <br /> the execution point analysis. At 370C dbJ3-1 dna39-3           limiting cell cycle protein. To explain cycle arrest at <br /> have clear execution points defective gene products        37&#176;C dbJ3 mutants protein unstable and <br /> are carrying essential functions particular defined         synthesised novo early cell cycle START and <br /> points cell cycle. Both execution points early        end S phase. Only limited number known cell cycle <br /> cell cycle, time S phase, but, unexpectedly,        proteins fulfil requirements, have <br /> the dna39-3 execution occurred earlier dbJ3.               shown contain intron. However, studies using <br /> Consistent earlier execution point dna39-3, higher         cycloheximide, protein synthesis inhibitor, shown cell <br /> proportion cells accumulate wih IC DNA content 370C              cycle progression requires protein synthesis G1 (51) also <br /> than dbf3-1. This reflect difference leakiness        distinct intervals GI (52). If unknown <br /> between alleles or, conceivably, Dbf3/Prp8 protein          proteins encoded gene containing introns could <br /> might function different times cell cycle.                 account cell cycle block dbJ3. A prediction of <br />    The dbJ3 mutant originally identified group          hypothesis splicing cell cycle mutants might <br /> mutants S phase defects (18,19) dna39 alleles         allelic. So far, does appear case, the <br />  identified screen mutants defective DNA               possible exception described below. <br />  replication (26). Our data DNA synthesis inhibitor                The alternative hypothesis mentioned certain <br />  hydroxyurea consistent role Dbf3/Prp8          snRNA complexes role cell cycle progression is <br />  near S phase. In spite 75 % cells arresting          independent splicing itself. Unlike prp8 (rna8) mutation, <br />  apparent 2C DNA content, 60% cells remain sensitive           inhibits RNA synthesis (2,6,7), dbJ3-1 allele no <br /> to HU. Thus, DNA cells apparent 2C DNA                  detectable effect gross RNA synthesis (18,19) nonetheless <br />  content remain unreplicated. This consistent          results cell cycle stage-specific arrest restrictive <br />  known behaviour certain mutants defective initiation S        temperature. Therefore cell cycle phenotype dbf-J cannot <br /> phase (45). The 40% cells completed cell cycle       easily explained secondary result primary defect <br />  presence HU indicate degree leakiness          RNA synthesis processing. One dbf mutant, dbfS, <br /> dbJ3-1, allows cells complete DNA synthesis             allelic prp mutant, prp3 (rna3) (18). <br />  370C.                                                                  Interestingly, U4/U5 snRNP (2) DBF3/PRP8 is <br />    In summary, dbJ3 clearly cell cycle phenotype         necessary assembly U4/U6.U5 triple snRNP <br />  defect occurring early cell cycle. The screens used          complex pathway splicosome assembly (9). The <br />  identify dbJ3 dna39, HU reciprocal shift         coincidence cell cycle phenotypes caused mutations the <br />  experiments, suggest role S phase, possibly initiation       genes encoding snRNP&apos;s DBF5/PRP3 DBF3/PRP8 points <br />  DNA synthesis.                                                         U4/U6.U5 triple snRNP complex potential <br />     The sequence DBF3/PRP8 gene shows tracts              role regulation S phase. Conceivably, formation of <br />  proline residues 78 residues N-terminus. Polyproline     complex necessary early cell cycle co- <br />  sequences number transcriptional                ordinated S phase dual function Dbf3/Prp8 and <br />  activators (46), suggesting Dbf3/Prp8 affect             Dbf5/Prp3. Dbf3/Prp8 certainly large protein ample <br />  transcription genes required S phase expressed      structural potential multifunctional. Consistent the <br />  cell cycle control late G1, S phase specific       dbJ3-1 allele complements ts defect prp8-1 allele <br />  cyclins CLB5 CLB6 (47-50). Importantly, proline               suggesting defects lie different functional domains. <br />  residues present worm sequence indicate           Lastly, worth drawing attention apparent link <br />  unnecessary pre-mRNA splicing function            carbon catabolite metabolism pre-mRNA splicing <br />  5564 Nucleic Acids Research, 1994, Vol. 22, No. 25 <br /> (see Introduction). Taken connection                    38. Sulston, J., Thomas, K., Wilson, R., Hillier, L., Staden, R., Halloran, N., <br /> splicing cell cycle, suggest RNA                               Green, P., Thierry-Meig, J., Qui, L., Dear, S., Coulson, A., Craxton, M., <br /> metabolism co-ordinated events                         Durbin, R., Berks, M., Metzstein, M., Hawkins, T., Ainscough, R. and <br />                                                                                       Waterson, R. (1992) Nature 356, 37-41. <br /> cell. If so, links splicing cell biological             39. Adams, M.D., Kerlavage, A.R., Fields, C. Venter, J.C. (1993) Nature <br /> pathways found.                                                               Genetics 4, 256-257. <br />                                                                                   40. Slater, M.L. Craig, E.A. (1989) Nuc. Acids Res. 17, 4891-4895. <br />                                                                                   41. Yost, H.J. Lindquist, S. (1986) Cell 45, 185-193. <br /> ACKNOWLEDGEMENTS                                                                  42. Yost, H.J. Lindquist, S. (1991) Mol. Cell. Bio. 11, 1062-1068. <br />                                                                                   43. Rowley, A., Johnston, G.C., Butler, B., Wemer-Washbume, M. Singer, <br /> We thank Dr R.Singer, Halifax, informing dbJ3                         R.A. (1993) Mol. Cell. Biol. 13, 1034-1041 <br /> dna39 mutations allelic, Chris Atkins, NIMR, help                     44. Williamson, D.H. (1965) J. Cell Biol. 25, 517-528. <br /> with flow cytometry, Jean  <br /> </body></html>