<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 1989, p. 3919-3930                                                                      Vol. 9, No. 9 <br /> 0270-7306/89/093919-12$02.00/0 <br /> Copyright &#169;D 1989, American Society Microbiology <br />  <br />  <br />           hsp82 Is Essential Protein That Is Required Higher <br />          Concentrations Growth Cells Higher Temperatures <br />  KATHERINE A. BORKOVICH,1t FRANCES W. FARRELLY,2 DAVID B. FINKELSTEIN,2t JOHN TAULIEN,1 <br />                                            AND SUSAN LINDQUIST&apos;* <br /> Howard Hughes Medical Institute, The University Chicago, Chicago, Illinois 60637,1 Department Biochemistry, <br />                      University Texas Southwestern Medical Center, Dallas, Texas 752352 <br />                                              Received 20 April 1989/Accepted 8 June 1989 <br />  <br />            hsp82 highly conserved abundantly synthesized heat shock proteins eucaryotic cells. <br />          The yeast <span id='am-36' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> contains closely related genes HSP82 gene <span id='am-60' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>. HSC82 was <br />          expressed constitutively high level moderately induced high temperatures. HSP82 was <br />          expressed constitutively lower level strongly induced heat. Site-directed disruption <br />          mutations produced genes. Cells homozygous mutations did grow temperature. <br />          Cells carrying combinations HSP82 HSC82 mutations grew 25&#176;C, ability to <br />          grow higher temperatures varied gene copy number. Thus, HSP82 HSC82 constitute essential <br />          gene family yeast cells. Although proteins different patterns expression, appeared have <br />          equivalent functions; growth higher temperatures required higher concentrations protein. <br />          Biochemical analysis hsp82 vertebrate cells suggests protein binds variety cellular <br />          proteins, keeping inactive reached proper intracellular location received the <br />          proper activation signal. We speculate reason cells require higher concentrations hsp82 hsc82 for <br />          growth higher temperatures maintain proper levels complex formation proteins. <br />  <br />    When exposed elevated temperatures forms                  rate wild-type cells exposed extreme tempera- <br /> stress, cells tissues wide variety organisms                  tures. Cells carrying mutations SSAI SSA2, two <br /> synthesize proteins known heat shock proteins                        members HSP70 gene family, temperature sensi- <br /> (HSPs). Coincident induction HSPs, cells                    tive growth rapidly killed extreme <br /> organisms tolerant exposure extreme                       temperatures wild-type cells (12). (The constitutive <br /> temperatures. <span id='am-80' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-81'  ></span><span id='am-82' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> observation led general as-                   overproduction HSPs, including members of <br /> sumption HSPs provide protection toxic effects                HSP70 gene family, mutants explain this <br /> of heat. The assumption reinforced lines                   result.) Experiments type indicate different <br /> of evidence. For example, virtually stress treat-                 proteins responsible different temperature-re- <br /> ments induce HSPs, exposure ethanol,                    lated phenomena; proteins allow cells grow at <br /> <span id='am-15' about='obo:NCBITaxon_1207117' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_7055 obo:NCBITaxon_7056 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_75546 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_41086 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Anoxia" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>anoxia</span>, heavy-metal ions, induce thermotolerance.                  upper end normal temperature range, may <br /> Moreover, cases kinetics HSP synthesis                help cells survive chronic exposure supraoptimal temper- <br /> degradation shown parallel kinetics of <br /> thermotolerance induction decay (reviewed refer-                     atures, some, <span id='am-152' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-153' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-168'  ></span>unidentified</span>, increase survival <br /> ences 26 27).                                                           exposure extreme temperatures. <br />    The results experiments, however, conflict                    While attempts define roles HSPs high <br /> the hypothesis. For example, cells, cycloheximide                   temperature continued, apparent that <br /> blocks induction HSPs blocking induction                 proteins, close relatives, synthesized <br /> of thermotolerance (25, 48, 49). Genetic investigations                normal growth temperatures induced specific <br /> also produced seemingly contradictory results. On                   times development. Thus, addition putative <br /> hand, <span id='am-93' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-94' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> mutations block expression                   protective functions exposure high temperature <br /> cr32 (the transcriptional regulator heat shock genes)                forms stress, proteins provide essential <br /> block induction HSPs induction                       basal developmental functions. This clearly <br /> thermotolerance (20, 32). On hand, artificial induc-              demonstrated case hsp70 family proteins in <br /> tion (r32 normal temperatures leads induction               yeast cells (12, 13). Deletions certain individual genes and <br /> HSPs does induce thermotolerance (47).                              certain gene combinations gene family result in <br />    Mutations individual HSPs provided important                     lethal phenotypes. <br /> new insights. Cells yeast <span id='am-53' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae                      Here effects mutations yeast <br /> that carry mutations heat-inducible ubiquitin gene               HSP82 genes. All eucaryotes produce heat- <br /> killed rapidly wild-type cells long-term expo-                 inducible proteins molecular sizes range 80 to <br /> sure temperatures just maximum growth tem-                     90 kilodaltons (kDa). Proteins family highly <br /> perature (19). However, mutants killed                  conserved, eucaryotes distantly related <br />                                                                             yeasts, fruit flies, mammals, trypanosomes sharing at <br />   * <br />     Corresponding author.                                                   50% amino acid sequence identity (3, 17, 18, 30). <br />   t Present address: Division Biology, <span id='am-20' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span> Institute           Moreover, eucaryotic proteins share approximately 40% <br /> Technology, Pasadena, CA 91125.                                             amino acid sequence identity known member <br />   t Present address: Panlabs Incorporated, Bothell, WA 98011.               family E. coli, htpG C62.5 (3). These proteins <br />                                                                     3919. <br />  3920     BORKOVICH ET AL.                                                                                                                    MOL. CELL. BIOL. <br />  <br />                           A                                     B <br />                           pUTX202          L                     I <br />                                                                                       B <br />  <br />                                                                                           I <br />                                                                                                               Pf,E       p   2000bp <br />  <br />  <br />                                                         B E              E   E                      B <br />                                                                                                                PI    1 OOObp <br />                          pUTX203       I                |       HSC82&apos;*                             L                    i1 <br />  <br />  <br />                                      ..       .           4- <br />                                            4- 4-4-4- 4-                                                       4- <br />                               -W             4- 4- 4.. 4- *-4                                            -                   4- <br />                      B                     H K H Hp                                                Hc <br />                          E HR X   BBc SS|                                                                                       S <br />                                                    I IKI E                                                   Xm <br />                                                                          RI RHC               HPI EIIl          HCI HL               200bp <br />                            I\kS\\        1                  I <br />                                                                 Yn <br />                                                                      I      I I <br />                                                                                       R <br />                                                                                           I                   I <br />                                                                                                                          N\\x        I <br />  <br />  <br />  <br />  <br />                                    -*          -~ -*                                                     * <br />  <br />   FIG. 1. Subclones sequencing strategy HSC82 gene S. cere0isiae. (A) Restriction maps <span id='am-147' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> constructed during <br /> subcloning HSC82 gene. Symbols:     LI, BamHI fragment containing gene flanking regulatory sequences; _, flanking <br /> yeast sequences;     ,  YEp13 (pUTX202)- pBR322 (pUTX203)-derived sequences. (B) Detailed restriction map portion HSC82 <br /> that sequenced, 5&apos; end left. Symbols:    ,coding region; M adjoining yeast DNA. Arrows indicate positions lengths <br />                                                                                   , <br />  <br />  <br />  <br />  <br /> of sequences determined messenger-identical strand; arrows indicate positions lengths sequences determined the <br /> messenger-encoding strand. Abbreviations: B, BamHI; Bc, BcIl; E, EcoRI; H, HindIII; Hc, HinclI; Hp, HpallI; K, KpnI; P, P&apos;uII Ps, PstI; <br /> R, RsaI; S, Sau3A; X, XbaI; Xm, XmnI; bp, base pairs. <br />  <br />  <br /> have variously named literature, according                         fragment clone, containing HSC82 gene, was <br /> their apparent molecular weights sodium dodecyl sulfate                        subcloned pBR322 pUTX203 (Fig. 1) se- <br /> (SDS)-gels. Since nucleic acid sequence analysis                           quenced Maxam-Gilbert protocol previously de- <br /> genes encoding proteins reveals virtually                      scribed (18). Amino acid sequences aligned using <br /> proteins family predicted molecular sizes 80                    program GAP (15), utilizes algorithm of <br /> 84 kDa, refer hsp82 family. (Two                      Needleman Wunsch. The percent identity number <br /> major exceptions exist: E. coli protein predicted                       identical amino acids bases sequences after <br /> molecular weight 62.5 kDa, vertebrate cells contain                        alignment divided number units se- <br /> additional genes encoding proteins signal sequences                      quence mentioned comparison. The GenBank acces- <br /> entry endoplasmic reticulum predicted                          sion number HSC82 M26044. <br /> molecular weights 90 94 kDa.)                                                 Plasmid constructions yeast transformations. Plasmid <br />    The yeast S. cerevisiae produces proteins                          YEp13 (7) cut Bglll, fragment containing <br /> family molecular weights 80,885 (hsc82; paper)                       yeast LEU2 gene purified. This fragment was <br /> and 81,419 (hsp82; 18). Like HSP82 genes higher                            inserted BglII site pUTX17 (HSP82) (17), creating <br /> eucaryotes, yeast genes complexly regulated. They                         plasmid pUTX123. Plasmid pUTX212 constructed by <br /> are induced response high temperatures                         replacing BclI site EcoRI site site pUTX203 <br /> also approach stationary phase                           (HSC82) 5.1-kilobase BamHI-EcoRI fragment of <br /> sporulation (24; K. A. Borkovich S. Lindquist, manu-                          YIp5 (46). Vector pKAT7 digesting pUTX203 <br /> script preparation). We examined effects                           completion KpnI, treating T4 DNA polymerase, <br /> mutations yeast HSC82 HSP82 genes growth                         religating presence BglII linkers. Plasmid <br /> various temperatures, survival extreme temperatures,                        pKAT12 constructed ligating BglII LEU2 frag- <br /> and sporulation.                                                               ment pUTX123 BglII site pKAT7. <br />                                                                                      One pair isogenic haploid strains S. cerevi- <br />                                                                                                                                 a. <br />  <br />  <br />                 MATERIALS AND METHODS                                             siae recipients DNA transformations: <br />                                                                                   W3031A (a ade2-1 canl-100 his3-12,16 leu2-3,112 trpl-l <br />   Screening genomic library, DNA sequencing,                           ura3-1) W3031B (ot ade2-1 can1-100 his3-12,16 leu2- <br /> sequence  comparison. The HSC82 gene isolated                        3,112 trpl-l ura3-J). Diploids created mating <span id='am-127' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-131'  ></span>iso</span>- <br /> Nasmyth-Tatchell YEp13 library (31) hybridization                         genic haploids independent transformations. Cells were <br /> the HindIII-EcoRI fragment HSP82 clone pUTX1                            transformed linear DNA (21, 41) plated expres- <br /> previously described (18) formamide concen-                       sion appropriate auxotrophic marker follows: for <br /> tration 30% temperature 42&#176;C. One positive                        W303leu, BglII LEU2 gene fragment pUTX123; for <br /> clone, pUTX202, having insert approximately 10                              PLD82, XbaI fragment pUTX123 containing LEU2 <br /> kilobases, chosen study (Fig. 1). The BamHI                       gene; strain CLD82, BamHI fragment pKAT12 <br />  VOL. 9, 1989                                                                        CHARACTERIZATION OF hsp82              3921 <br />  <br />                                                 TABLE 1. Yeast strains used <br />           Strain                                            Relevant genotype                                         Source <br /> HSC82 HSP82 wild type <br />   W3031A                                         leu2 his3 ura3 HSP82 HSC82 MATa                                   R. Rothstein <br />   W3031B                                         Ieu2 his3 ura3 HSP82 HSC82 MATa                                   R. Rothstein <br />   LP112                                          leu2 his3 ura3 HSP82 HSC82                                        L. Petko <br />                                                  Ieu2 his3 ura3 HSP82 HSC82 <br />   W303leu                                        LEU2 his3 ura3 HSP82 HSC82                                        This study <br />                                                  LEU2 his3 ura3 HSP82 HSC82 <br />   W303ura                                        leu2 his3 ura3 HSP82 HSC82 CEN URA3                               This study <br />                                                  Ieu2 his3 ura3 HSP82 HSC82 CEN URA3 <br /> hsc82 hsp82 mutant <br />   CLD82                                          leu2 his3 ura3 HSP82 hsc82::LEU2 <br />                                                  leu2 his3 ura3 HSP82 hsc82::LEU2                                  This study <br />   CUD82                                          Ieu2 his3 ura3 HSP82 hsc82::URA3                                  This study <br />                                                  leu2 his3 ura3 HSP82 hsc82::URA3 <br />   PLD82                                          leu2 his3 ura3 hsp82::LEU2 HSC82                                  This study <br />                                                  leu2 his3 ura3 hsp82::LEU2 HSC82 <br />   SCC82                                          leu2 his3 ura3 hsp82::LEU2 HSC82 <br />                                                  Ieu2 his3 ura3 hsp82::LEU2 hsc82::URA3                            This study <br />   SCP82                                          leu2 his3 ura3    HSP82 hsc82:: URA3                              This study <br />                                                  Ieu2 his3 ura3 hsp82::LEU2 hsc82::URA3 <br />   PCD82                                          leu2 his3 ura3 HSP82 hsc82:: URA3                                 This study <br />                                                  Ieu2 his3 ura3 hsp82::LEU2 HSC82 <br />  <br />  <br /> containing LEU2 gene; strain CUD82,              incubated 25&#176;C (control) 39&#176;C (heat shock) 1 h. Cells <br /> HindIII fragment pUTX212 containing URA3 gene.            harvested centrifugation, denatured extracts <br /> The diploid wild-type strain LP112 created mating      lysis glass beads absolute ethanol <br /> W3031A W3031B (36). Wild-type strain W303ura             containing 3 mM phenylmethylsulfonyl fluoride. Total cellu- <br /> constructed transformation W3031A W3031B          lar proteins 106 cells separated electrophoresis <br /> the YCp5O vector, followed mating resultant haploid       7% polyacrylamide-SDS gels transferred electro- <br /> transformants. Strain PCD82 constructed mating            phoretically nitrocellulose nylon membrane (36). To <br /> CUD82 PLD82 haploids. Strain SCC82               increase resolution hsp82 hsc82, electro- <br /> transformation strain PLD82 HindIII fragment      phoresis continued proteins range 80 kDa <br /> plasmid pUTX212, strain SCP82 constructed             run two-thirds way gel. The blot was <br /> transformation CUD82 XbaI fragment plasmid        preincubated 5% milk phosphate-buffered saline, fol- <br /> pUTX123.                                                         lowed incubation solution containing 1:500 <br />   The diploid strains examined greatest      dilution polyclonal antibody specific hsp82 hsc82 <br /> W303leu (the wild-type strain transformed LEU2          yeast cells (Borkovich Lindquist, preparation). The <br /> auxotrophic marker), PLD82 (two wild-type HSC82 genes            sheet washed phosphate-buffered saline, immu- <br /> and mutant hsp82 genes LEU disruptions), CLD82          noreactive proteins visualized using 1251I-protein A <br /> (two wild-type HSP82 genes mutant hsc82 genes            (ICN Pharmaceuticals Inc.) previously described (6). <br /> with LEU disruptions), SCP82 (a single copy wild-type        In vivo protein labeling. Strains cultured SD <br /> HSP82 gene, mutant hsp82 gene, mutant hsc82          medium density 5 x 106 cells ml 25&#176;C. Aliquots <br /> genes), SCC82 (a single copy wild-type HSC82          (1 ml) incubated 25 39&#176;C 30 min, after <br /> gene, mutant hsc82 gene, mutant hsp82 genes).        20 ,uCi [3H]isoleucine (99 Ci/mmol; Amersham) was <br /> The relevant genotypes strains summarized          added incubation continued 30 min. Cells were <br /> Table 1.                                                         pelleted proteins extracted described above. <br />    Analysis yeast genomic DNAs. Yeast genomic DNAs            Samples containing 10,000 cpm base-stable, trichloroace- <br /> were purified stationary-phase YPDA cultures pre-        tic acid-precipitable counts separated SDS-poly- <br /> viously described (14). DNAs cut indicated         acrylamide gel electrophoresis prepared fluorog- <br /> restriction enzyme, separated agarose gel electrophoresis     raphy incubating gel 20% 2,5-diphenyloxazole <br /> (29), blotted Hybond-N (Amersham Corp.) accord-         (PPO) glacial acetic acid drying. <br /> ing recommendations manufacturer. Prehybrid-          Vegetative growth 25 37&#176;C. Strains grown in <br /> ization, hybridization, washing blots        liquid YPDA (1% yeast extract, 2% Bacto-Peptone, 2% <br /> recommended membrane omission         glucose, 10 jxg adenine sulfate) medium vigorous <br /> dextran sulfate. A 32P-labeled probe prepared nick        aeration 25&#176;C mid-log phase diluted 3 x <br /> translation (29) HindIII-EcoRI fragment pUTX1          104 cells ml (25&#176;C cultures) 104 cells ml (37&#176;C <br /> (HSP82), hybridizes equally HSP82           cultures) YPDA medium. Samples taken various <br /> and HSC82 genes (F. W. Farrelly D. B. Finkelstein,           times counted hemacytometer. Genotypes were <br /> unpublished <span id='am-137' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-141'  ></span>data</span>).                                               verified immunological analysis proteins extracted <br />    Heat shock control conditions immunological de-       cells various times described above. <br /> tection hsp82 hsc82. Yeast cells grown minimal       Sporulation efficiency spore viability. Yeast strains <br /> dextrose medium (SD) mid-log phase (5 x 106 cells         cultured YPAc (1% yeast extract, 2% Bacto-Pep- <br /> ml). Portions 1 ml transferred glass tubes        tone, 2% potassium acetate) medium 3 x 107 5 x 107 <br />  3922    BORKOVICH ET AL.                                                                                       MOL. CELL. BIOL. <br />  <br /> cells ml 25&#176;C. The cultures centrifuged pellet     transcript mapped -41 relative ATG <span id='am-85' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> by <br /> the cells, washed 1% potassium acetate,          Si analysis (data shown). The largest open reading frame <br /> resuspended cell density 1% potassium              extended +1 +2118 predicted acidic protein <br /> acetate. Equal portions withdrawn different times         molecular weight 80,885 (Fig. 2). This protein was <br /> after transfer potassium acetate monitored     97% identical yeast hsp82 protein amino acid <br /> presence dyads, triads, tetrads, using hemacytom-        level. There region 71% base identity 5&apos; <br /> eter. Percent sporulation taken total number        upstream regions genes corresponding posi- <br /> dyads, triads, tetrads divided total number         tions -233 -196 HSC82 gene (data shown). <br /> cells field. To assess spore viability, asci 4-day    This region significant homology original <br /> sporulation cultures dissected YPDA plates.             purported heat shock consensus sequence CTGGAATNT <br /> Plates incubated 30&#176;C 2 3 days scored     TCTAGA (34), segment -192 -179 in <br /> for presence colonies formed germinated spores.         HSC gene contained 9-of-15-nucleotide match. It has <br /> To recover haploid spores containing functional copies      suggested, however, heat shock consensus <br /> HSP82 HSC82, strain PCD83 dissected YPDA                sequence actually composed multiple GAA segments <br /> plates incubated 30&#176;C 3 days 17&#176;C 3      arranged alternating orientations separated 2- <br /> weeks scoring. Colonies patched tester           nucleotide spacers 5-base-pair core motif (2). A <br /> plates check genotypes.                                        minimum 5-base-pair cores, core preceded <br />    Thermotolerance vegetative growth 25&#176;C. (i)          nucleotides TA, TC, GA, required for <br /> YPDA medium. Diploid strains W303leu, PLD82,                  optimal induction. An extra core, containing GAA or <br /> CLD82 inoculated 2 x 104 cells ml YPDA           TTC block, separate functional cores long the <br /> medium, cultures grown 25&#176;C overnight. At             proper spacing maintained. This theory consistent with <br /> various times day, equal portions withdrawn         results obtained (35, 45, 50). Inspection the <br /> and placed ice. The samples sonicated briefly          yeast HSC82 HSP82 genes revealed region of <br /> disaggregate cells, 3- 300-pI portions         upstream homology (-233 -196) contained an <br /> sample transferred glass Corning tubes (16 80          element: TTCTA GAANG NNNNN GAANA, N des- <br /> mm) ice. Two tubes (control pretreatment)        ignating nucleotides differ genes. Since <br /> were placed 25&#176;C; tube incubated 37&#176;C         HSP82 HSC82 basic <br /> (pretreatment) 30 min. At time, control tube     element, different patterns expression suggest the <br /> put ice incubated 50&#176;C 8         participation elements regulation. <br /> min. After heating, tubes placed ice.      The yeast hsp82 hsc82 proteins shared 62, 62, 63% <br /> Chilled cultures diluted ice-cold YPDA medium,          amino acid identity <span id='am-73' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span>Drosophila melanogaster</span> hsp82 <br /> and 100-,u portions spread YPDA plates. The             (5), human hsp90 (37), <span id='am-63' about='obo:NCBITaxon_5693' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_47570 obo:NCBITaxon_5653'><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Trypanosoma cruzi" datatype="xsd:string"></span><span id='am-67'  ></span>Trypanosoma cruzi</span> hsp90 (17) <br /> plates incubated 30&#176;C 2 days, colonies        proteins, respectively. All eucaryotic members hsp82 <br /> were counted.                                                     family sequenced date contain highly-charged central <br />    (ii) YPAc medium. The protocol described YPDA-             region repeated acidic basic residues. This region is <br /> grown cells used original cell densities      absent E. coli protein htpG, accounting, part, its <br />  varied 2 x 105 1 x 108 cells ml, time         lower molecular weight (3). For yeast hsc82 protein, it <br />  incubation 50&#176;C varied 3 30 min, cultures    extends residues 221 259 contains four <br />  diluted ice-cold YPAc medium. In experiments,          repeats pentapeptide sequence (acidic amino <br />  cells plated dilution.                          acid)3(lysine)2, beginning residues 221, 226, 231, 246 <br />                                                                   (Fig. 2). The yeast hsp82 protein shows similar motif (18), <br />                            RESULTS                                higher molecular weight insertion of <br />                                                                   extra EEKKD sequence. The yeast E. coli proteins <br />   Characterization HSC82 HSP82 genes. Cloning          lack cysteine residues, Drosophila, <span id='am-68' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span>Trypanosoma</span>, <br /> and initial characterization yeast HSP82 gene         human proteins contain five, six, six, respectively, <br /> described previously (18). Hybridization gene          concentrated carboxy termini. Finally, <span id='am-121' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br /> restricted, electrophoretically separated genomic DNAs            boxy-terminal amino acids yeast hsc82 hsp82 <br /> (Southern blot analysis) revealed second, closely related       proteins EEVD, carboxy termini virtually <br /> gene haploid yeast genome. Clones related         eucaryotic members hsp82 apparently <br /> gene obtained screening plasmid library        unrelated hsp70 protein families. (The major exceptions <br /> HSP82 gene. Expression genes analyzed              members families enter the <br /> during growth normal temperatures heat shock        endoplasmic reticulum.) <br /> by hybridizing gene-specific probes total cellular RNAs          Construction disruption mutations HSP82 HSC82. <br /> (data shown). RNAs produced constitutively          Mutations created HSP82 HSC82 genes by <br /> both genes 25&#176;C, HSP82-related gene        inserting restriction fragments containing auxotrophic mark- <br /> being abundant. The gene named            ers protein-coding sequence (Fig. 3A). Linear DNAs <br /> HSC82 denote relationship HSP82 higher          containing disrupted genes transformed yeast <br /> constitutive level expression. RNAs HSC82 gene        cells, gene conversion events replacing wild-type <br /> increased modestly concentration heat shock,             genes mutations obtaineu selecting for <br /> whereas RNAs HSP82 gene increased manyfold.              expression auxotrophic marker (41). Both LEU2 and <br /> Complementary results obtained quantifying expres-        URA3 auxotrophic genes used mark disrupted <br /> sion proteins specific antiserum (see Fig.      alleles. For purposes comparison, original strains <br /> 3C).                                                              transformed auxotrophic markers. This <br />   The entire coding region HSC82 adjacent se-              essential correct identification phenotypes, the <br /> quences sufficient regulated expression contained     auxotrophic markers influenced certain traits we <br /> the 5-kilobase insert pUTX203 (Fig. 1). The 5&apos; end      examined (see below). Since transformation itself be <br />  VOL. 9, 1989                                                                                       CHARACTERIZATION OF hsp82                 3923 <br />  <br />          &apos;112   RGCTTTTRRC   CGTRCTRGRT RGTTTRTARC CCRTTRCGCR TTTGRTTRTR RTTTGCTTCT TRGGCRRART TRRTATTTRC GTTCTTTTRT RTTCTTTCTT TTTGT <br />          -307   RTTCRTRGAR   CRGCRGCCRT TRCCRRTRGR RRGTTRRRRT RGCCGCCGRT GCRTTTTATT RCCC6CCTTT CYGTTTTCTG GGCRCTTTTC TTTCTRGRRG GTGRR <br />          -212   RGRACRRTTT   TTCTCGTTTT CTCGRRCTTC CRCCRRGCGT TGGGTRRTGR GGGR66R6RT TTGTOTRRRR RGROTGGCRT OT6RRCTGCC TRCCGTRRGT GRCRT <br />           -97   GRRCRCRtGC   RTTRTRTTTT TTGTGRTRTR TTCTTTCTCT TGTTTTCTTT TTCTTGRRRC GCTACRGRRC CRATRORRAR TRGRRTCRTT CTGRRRT RTG GCT <br />             1                                                                                                                 Rla <br />            7 GGT GRR RCT TTT GRA TTT CAR GCT GRR RTC RCT CRG TTG RTG RGT TTO RTC RTC ARC RCT GTC TAT TCT RRC RAG GRR RTT TTC TTO <br />            3 Glu Glu Thr Phe Glu Ph. Gln Rla Glu I I Thr Gin Lou let Ser Lou I1 I 1II Ron Thr UV I TVr Ser Ron L_ GI u I I. Phe Lou <br />           94 RGR GRR CTG RTC TCT RRC GCC TCC GRT GCT TTR GRC RAR RTT RGR TAC CRR OCT TT6 TCT ORT CCR RRG CRR TTG GRR RCC GRR CCR <br />           32 Rrg Glu Lou I I Ser Rsn Rl Ser Rep Rla Lou Rep Lys IIe Rrg Tur G I n RI Lou Se Rep Pro Lyo Gin Lou Glu Thr Glu Pro <br />          181 GRT TTG TTC RTT RGR RTC RCC CCR RRR CCR GAR GRR RRR GTT TTG GRR RTC RGR OAT TCT GOT RTT GOT RtG RCC RRG         GCT GRR TTG <br />           61 Rep Lou Ph. Ii. Rrg I <span id='am-26' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span>Ia</span> Thr Pro LVs Pro Glu Glu Le UVal Lou Glu I I Rrg Rep Ser Gly lie Gl_ net Thr Lye        RAl Glu Lou <br />          268 RTT RRC RRT TTG GGT RCC RTT GCT RAG TCT GGT RCT RRR GCT TTC RTG GAR 6CT CTR TCT OCT GGT 0CC GRT GTR TCC         RTG RTT GGT <br />           90 I Is Ron Rsn Lou GIy Thr II1 RAla Lys Ser GIV Thr LVe RAI Phe net Olu Rla Lou Ser Rla Gly RAl Rep U I Ser       II* GIy <br />          355 CRR TTC GGT GTT GGT TTT TRC TCT TTR TTC TTR GTC 0CC GRC RGR OTT CRR OTT RTT TCC RRG RRC ART GRG GRC GAR CRR TRT RTT <br />          119 Gin Phe Gly UaI Gly Phe Tyr Ser Lou Ph. Lou Ual <span id='am-116' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-120'  ></span>Ala</span> Rsp Rrg Ual Gin Uai I II Ser Lye RAn Ron Glu Rep Glu Gin Tyr Ile <br />          142 TGG GAR TCT RAT GCC GGT GOT TCT TTC RCC GTT ACT TTG GRC GRR GTT ARC GRR RAR RTT GOT AGR GOT RCC GTC TTG RGR TTR TTC <br />          116 Trp Olu Ser Ren RAl GIl G1y Ser Ph. Thr UVl Thr Lou Rep Glu UVl Ron GIu Rrg Ie O ly_ Rrg G61 Thr Ua. Leu Rrg Lou Ph. <br />          529 TTG RRR GRT GRC CRR TTG GRG TRC TTG GAR ORR RAG RGR RTT RAR GRR OTC RTC RRG RGR CRT TCT GRR TTC GTT GCT TRC CCT RTC <br />          177 Lou Lye Rep Rep Gin Lou Olu Tyr Lou Glu Glu Lys Rrg I I Lye Glu Ual I1I Lye Rrg His Ser Olu Ph. Ua I RAl Tyr Pro I I . <br />          616 CRR CTT CTR GTC RCC RRG GRR GTC GRR RRG ORR OTT CCR RTT CCR ORR GRR GRR RAG RRR GRC 6RG GRR RAG RRG GRT ORR GAT GRC <br />          206 Gin Lou Lou Ual Thr Luy Iu Ual GIu Lus Glu Ual Pro I I Pro Glu Glu Glu Lye LVe Rep Olu Glu Lye Lys Rep Glu Rep Rep <br />          703 RRG ARR CCR RRR TTG ORR GRR GTC GRT GAR GRR ORR GRR GRR RRG RRG CCR RRR RCC RRR RRR GTT RRR GAR GRG GTT CRR ORR TTR <br />          235 Lye Lye Pro Luy Lou GIu GIu UVI Rsp Glu Glu Glu Glu Glu Lye Lye Pro Lye Thr Lye Lye Ual Lye Glu Glu UVi Gin Glu Lou <br />          790 GRR GRG TTG RRC RRG RCT RRG CCR TTR TOG ACT RGR RRC CCR TCT GRT RTC RCT CAR GAG ORR TRC RRT GCT TTC TRT RRG TCT RTT <br />          264 GIu GIu Lou Ren Lye Thr Lye Pro Lou Tep Thr Rrg Ron Pro Ser Rep lIe Thr Gin Glu Glu Tyr Ron RAl Phe Tyr Lye Ser I I <br />          077 TCT RRC GRC TGG GRR GRC CCR TTG TRC OTT RRG CRT TTC TCT GTT GRR GOT CRR TTG GRR TTT RGR OCT RTC TTG TTC RTT CCR RAG <br />          293 Ser Rsn Rep Trp GIu Rep Pro Lou Tyr Ual Lye His Ph. Ser UVI Glu Gly Gin Lou Olu Phe Rrg Rla I I . Lou Phe I I . Pro Lys <br />          964 RGR GCR CCR TTC GRC TTR TTT GRG RGT RAG RRG RRG RAG RRC RRT RTC RRG TTG TRC OTT CGT COT GTC TTC RTC ACT OAT RAR OCT <br />          322 Rrg RIa Pro Phe Rsp Lou Phe GIu Ser Lye Lys Lye Lys Ron Ron I I * Lys Lou Tyr UV I Rrg Rrg Ua I Ph. lIe The Rep Glu Rla <br />          1051 GRR GRC TTG RTT CCR GRG TGG TTR TCT TTC GTC RAG GOT OTT OTT GRC TCT GRR GRT TTR CCR TTG RART TTG TCC RGR ORR RTO TTR <br />           351 Glu Rap Lou Ilie Pro Glu Trp Lou Ser Phe Ual Lye Gly Ual Ual Rep Ser Glu Rep Lou Pro Lou Rsn Lou Ser Rrg GIu Net Lou <br />          1138 CRR CRR RRT RRG RTT RTG RRG OTT RTT RGR RAG RAT RTT GTC RRG RRR TTO RTT GRR GCC TTC RRC ORR RTC OCT ORR GRC TCC ORG <br />           380 Gln Gln Ron Lys lie n.t Lys Ual li. Rrg Lye Ron I1 VUal Lye Lye Lou I I G I u RAl Phe Ron Glu IIe Rlo Glu Rep Ser Glu <br />          1225 CRR TTT GRC RRR TTT TRC TCT 0CC TTC OCT RAG RRC RTT RRG CTG GOT OTR CRT ORG GRC RCT CRR RRC RGR OCT OCT TTR GCT RAG <br />           109 Gin Ph. Rep Lys Phe Tyr Ser RIa Phe Rla Lye Ron Ii. Lye Lou Gly Ual His Glu Rep Thr Gln Ron Rrg Rla RAl Lou Rla Lye <br />          1312 TTG CTR CGT TRC RRT TCT RCT RRR TCT GTC GRT GRR TTO RCT TCC TTG RCT GRT TRC OTT RCT RGR RTG CCIORR CRC CRR RRG ARC <br />           138 Lou Lou Rrg Tyr Rsn Ser Thr Lye Ser UVl Rep GIu Lou Thr Ser Lou Thr Rep Tyr Ual Thr Rrg Net Pro Glu His Gln Lye Ron <br />          1399 RTC TRT TRC RTC RCC GGT ORR TCT CTR RRG OCR GTC GRR RAG TCT CCR TTC TTG GRC GCC TTG RRG OCT RRG RRC TTT ORR OTT TTG <br />           467 Ile Tyr Tgr Ilie Thr GIy GIu Ser Lou Lys Rl UVal GIu Lye Ser Pro Ph. Lou Rep RAl Lou Lye RA Lys Ron Phe GIu UVI Lou <br />          1186 TTC TTG RCC GRC CCR RTT ORT ORR TRC OCT TTC RCT CRR TTG RAG ORR TTC GRG GGT RAR RCT TTG GTT GRC RTT RCT        RRR ORT TTC <br />           496 Phe Lou Thr Rep Pro lie Rep Glu Tyr Ala Phe Thr Gin Lou Lye Glu Ph. Glu Gly Lye Thr Lou Ual Rep l Ie Thr       Lye Rep Ph. <br />          1573 GRR TTG GRR GRR RCR GRC ORR GRR RRR GCT GRR RGR ORG RRG GRG RTC RRR ORR TRC GRR CCR TTG RCC RRG 0CC TTG        RRG ORT RTC <br />           525 Glu Lou Glu Glu Thr Rep Glu Glu Lye Rla Glu Rrg Glu Lye Glu I I Lye Glu Tyr Glu Pro Lou Thr Lye RAl Lou        Lye Rep I I e <br />         1660 TTG GGT GRC CRR GTG GRG RRG GTT GTT GTT TCT TRC RRR TTG CTR GRT GCT CCR GCT GCC RTC RGR RCT GOT CRR TTC GGC TOG TCT <br />          551 Lou Gly Rep GIn Ual GIu Lys Ual Ual Ual Ser Tur Lye Lou Lou Rep Ala Pro RAl RAl l Ie Rrg Thr Gly GIn Phe GIV Trp Ser <br />         1747 GCT RRC RTG GRR RGR RTC RtG RRG GCT CRR GCC TTG RAR GRC TCT TCC RTG TCC TCC TRC RTG TCT TCC IRRG RRG RCT TTC ORR RTT <br />          583 Rla Ron n.t Glu Rrg lie net Les Rla GIn RAl Lou Rrg Rep Ser Ser Net Ser Ser Tgr Ser Ser Lye Lye Thr Ph. GIu I I a <br />         1834 TCT CCR RRR TCT CCR RTT RTT RTT ORR RCG RRR RAG RGR OTT GRT GRG GOT GOT GCR CRR ORT RRG RCC IGTC RRR OAT TTG RCT RRC <br />          612 Ser Pro Lye Ser Pro lie lie lie GIu Thr Lye Lys Rrg U I Rep Glu Gly Gly Rla Gin Rep Ly TThr UVl Lye Rep Lou Thr Ron <br />         1921 TTR TTR TTC GRG RCC GCT TTG TTG RCT TCT GOT TTC ROT TTG GRR GAR CCR RCT TCT TTT GCR TCR RGR RTR RRT RGR TTG RTT TCT <br />          611 Lou Lou Phe GIu Thr RAl Lou Lou Thr Ser GyI Ph. Ser Lou GIu GIu Pro Thr Ser Phe RAl Ser Rrg I Ii. Ron Rrg Lou I I* Ser <br />         2008 TTR GOT TT6 RRC RTT GRT GRG GRT GRR ORR RCR GRR RCC GCT CCR 6AR GCT TCT RCC GRR GCT CCR GTT GRR GORG GTT CCR OCT GRC <br />          670 Lou Gly Lou Ron lie Rep GIu Rep GIu Glu Thr Glu Thr Rio Pro 61u Ala Sep Thr Glu Rla Pro Ual Glu Glu UVl Pro RAl Rep <br />         2095 RCC GRG RTG GAR GRAR GTT GRT TGR TCTCTT      TTTTCGCCTT CRTGTTTTAT RTATTATRTA RRTTTGTTTT   CTTATTTTTR CTRTTTGTRA TRRTGRTTCC <br />          699 Thr G I u Net G I u G I u U I Rep End <br />         2195 TGCTTTRCGC GCCTTTRRRRRG <br />   FIG. 2. Nucleotide deduced amino acid sequences HSC82 S. cerev&apos;isiae. The sequence presented messenger-identical <br /> or plus strand. The nucleotide position (+1) corresponds translation origin HSC82. Numbering nucleotides amino acids <br /> is shown left. <br />  <br /> mutagenic, case independent haploid trans-                   results, et haploid transformants mated pro- <br /> formants selected compared respect growth                   duce diploids. Any random recessive mutations one <br /> at 25 37&#176;C. In case, haploids given               haploid expected covered wild-type <br /> transformation reaction displayed characteristics.                  allele other. The use diploid cells allowed to <br /> The exceptional strain discarded. Finally, minimize                   construct strains broader variation HSP82 and <br /> the chance extraneous mutations influence                     HSC82 gene copy number. <br />  3924     BORKOVICH ET AL. <br />  <br />                        A <br />  <br />                            w:,       ;; <br />                                           J <br />  <br />                                           ,; <br />                                                            _, . : . <br />                                                            __ <br />  <br />                                                            ,,, <br />                                                                         A; + t- <br />  <br />  <br />  <br />  <br />                                                                             ,,               }_. _ <br />                                                                                                            B <br />  <br />                                                                                                                      -a-8-   4 <br />                                                                                                                                  0 <br />                                                                                                                                      So.&apos;*- <br />                                                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                              .-            0. <br />  <br />  <br />  <br />  <br />                                                  -A <br />                            :7    , <br />                                                ---] <br />                                                       ..   -_.    .:.             41- <br />  <br />  <br />  <br />  <br />                                                            C <br />  <br />                                                                                 Hsp82- <br />                                                                                 Hsc82-   -                      a3 <br />  <br />  <br />                                                                                 Hsp82- <br />                                                                                 Hsc82-~                ~        - <br />  <br />  <br />                                                                                                  2C   _n   r           ;1    J <br />  <br />  <br />  <br />  <br />   FIG. 3. Construction characterization mutations HSC82 HSP82. (A) Plasmid constructions. A URA3 LEU2 marker was <br /> inserted HSC82 HSP82 shown. (Adjoining pBR322 sequences shown.) (B) Southern blot hybridization portion HSP82 <br /> to yeast genomic DNA wild-type mutant cells. The HSP82 fragment labeled nick translation hybridized to <br /> electrophoretically separated yeast DNAs cleaved indicated enzyme. Positions fragments containing wild-type HSC82 and <br /> HSP82 genes marked. (C) Expression hsc82 hsp82 mutant wild-type cells incubation 25 39&#176;C. Total cellular proteins <br /> from indicated strains electrophoretically separated. transferred nylon membrane, reacted polyclonal antiserum <br /> specific hsp82 hsc82, followed 1251-protein A. Two different autoradiographic exposures blot shown. <br />  <br />   That transformations resulted disruption                                                   SCC82) produced roughly half protein cells <br /> targeted genes relatives confirmed                                                        carrying wild-type alleles genes. Therefore, <br /> Southern blot analysis genomic DNAs (Fig. 3B). To                                                           genes  limited capacity dosage compen- <br /> confirm expression appropriate protein                                                         sation. <br /> eliminated mutations examine effects                                                    To examine inducibility proteins high temper- <br /> mutations expression related gene, total cellular                                                    atures, matched samples cells shifted 25 to <br /> proteins various diploid strains electrophoreti-                                                     39&#176;C, temperature gives maximal induction hsp82, <br /> cally separated, transferred nylon membrane,                                                          90 min, total cellular proteins prepared and <br /> reacted antiserum specific hsp82 hsc82                                                     processed described above. Wild-type cells showed a <br /> proteins. Since wide variations expression observed                                                    strong induction hsp82 moderate induction hsc82 <br /> in experiments, different exposures 80-kDa                                                    (Fig. 3C). (The induction hsc82 readily apparent <br /> region blot displayed Fig. 3C. Wild-type strains                                                 shorter exposure.) In mutant strains, gene <br /> produced proteins 25&#176;C, hsc82 produced                                                      appeared induced independently. That is, cells carrying <br /> at higher level hsp82 (approximately 10 times                                                      single copy HSP82 gene produced half as <br /> greater densitometry; data shown). Cells homozygous                                                     hsp82 protein cells carrying copies gene. <br /> for hsc82 mutations (strain CLD82) produced hsc82                                                       Moreover, produced quantities hsp82 <br /> protein showed slight induction hsp82 relative                                                   regardless produced hsc82. Thus, as <br /> to wild-type cells. Cells homozygous hsp82 mutations                                                   case 25&#176;C, genes little, any, capacity <br /> (strain PLD82) produced hsp82 showed change                                                       dosage compensation high temperatures. <br /> the level hsc82 relative wild-type cells. Finally, cells                                                    Comparison total cellular proteins experi- <br /> carrying single wild-type HSP82 HSC82 gene (SCP82                                                         ments stained gels demonstrated hsc82 of <br />  VOL. 9, 1989                                                                                              CHARACTERIZATION OF hsp82                                         3925 <br />  <br />  <br /> the abundant proteins cell temperatures,     A_ <br /> accounting intense Coomassie-stainable                                 cn       c,,                                                          U,    cnO <br /> bands (data shown). hsp82 produced lightly <br /> stained band 25&#176;C but, consistent results of <br /> Western blot (immunoblot) analysis, accumulated approx- <br /> imately level hsc82 90 min 39&#176;C. In <br /> separate experiments, hsp82 accumulated approximately              1              + |            +                      1-(1 )(+)D                   1   +    -     + <br /> the level hsc82 4 h 37&#176;C.                            2 - (+)(+) D  2   (+)(+) D   2                                                        +          - <br />    Expression member HSP82 gene family              1             ~~~~~2        3 <br /> essential growth. Wild-type haploids readily trans-          3 + _-        3 +  _  _      3                                                            D(+)(+)D <br /> formed constructs designed produce disruption mu-             4              +     +                  -            4   +       --              4+_ -           - <br /> tations HSP82 HSC82 gene. However,                      1             (&apos; &apos;)(1 D                       1 +         -               1         (+)(+)      D <br /> haploids carried disruption muta- <br /> tions transformed constructs designed            2              + | _                                 2   -(+)(+)D                 2+       _      _ <br />                                                                  4                                                    5                            6 <br /> disrupt gene despite repeated attempts. If            3              +     --                              3   +       -+              3    +-- <br />  <br /> single-disruption haploids transformed             4              -   (+)(+) D                          4   +       +       -       4    _ (+)(+) D <br /> extrachromosomal plasmid containing HSP82 HSC82                 1              +     + -                             1   +       -       -       1        D(+)(+)D <br /> gene, readily transformed constructs <br /> designed disrupt chromosomal gene.                  7    2              +     -+                         8    2   +               _   9   2    +     +     _ <br />    These results suggested proteins produced          3              +              +                      3   -(+)(+)D                3    +     - <br /> two genes serve identical nearly identical functions           4              +     +                               4   -(+)(+)D                4    +    _     _ <br /> that functions essential. The results several <br /> other experiments confirmed case. First,             1              -I(+)T(+)            D                1   +   __     1                 +         __ <br /> diploid cells heterozygous mutations HSP82             2              +     +        -                      2   +       +-2                  -+() <br /> HSC82 genes sporulated 30&#176;C dissected. Figure       10                                                    11                           12- <br />                                                                       3              +     -+                              3   +       --3++- <br /> 4A shows results obtained strain PCD82, carrying a <br />  URA3-marked mutation HSC82 gene LEU2-                          4+            -- <br />                                                                                                                            4-+(+)IDI                   4 <br />                                                                                                                                                             +I <br /> marked mutation HSP82 gene. No colonies were <br /> obtained Leu+ Ura+ spores, spores            B <br /> produced colonies expected frequencies. Microscopic <br /> examination spores did form colonies re-                                                     m C <br /> vealed did germinate germinated                          0 <br />                                                                                      &gt;     c, <br />                                                                                                 C     | <br />                                                                                                     &gt;&gt;&gt; <br />                                                                                                         mM <br /> divided times. To determine                                      m        F   m  &apos;mm <br /> spores lacking HSP82 HSC82 genes grow <br /> and germinate lower temperatures, experiment                      1 -(+)(+D                                     1   +       +       -1+                -+ <br /> repeated 17&#176;C. The results obtained 30&#176;C           2              -(+)(+)D                              2   +       -       + 2 +              - + <br /> (Fig. 4B). Additionally, dissection 58 tetrads produced        1 <br />                                                                       3 <br />                                                                              T+T-T-1 <br />                                                                               +-- <br />                                                                                                                       2 <br />                                                                                                                            3   +       +- <br />                                                                                                                                                    1 3T+l+l-l I <br />                                                                                                                                                         3   +     +- <br /> evidence linkage HSP82 HSC82 (data shown). <br />    These analyses demonstrate cells produce             4           +          1                 1 <br />                                                                                                                            4   +   | <br />                                                                                                                                                +        4       D(+)(+)D <br /> the hsp82 hsc82 protein spore outgrowth. To confirm                           +                                     1   + <br />                                                                                                                                  -+    1                    +    - - <br /> that proteins required vegetative growth, hetero-            2           +      - +                            2   +   _ -   2                            I(+) <br /> zygous diploids constructed mutations      4                                                    5        6_ <br /> the HSP82 HSC82 genes marked LEU2. The                     3       |         (+)(+)         D                3   + _ +   3                 I-(+)+) D <br /> strain transformed 2,um plasmid containing               4           +      +  -                           4   +           -           4 + - - <br /> the wild-type HSC82 gene URA3 gene, sporulated, <br /> and dissected. In tetrads showing 2:2 segregation          FIG. 4. Analysis viability Hsc- Hsp- spores. SD-uracil <br /> LEU2 gene, Leu+ cells chosen analysis.         SD-leucine indicate synthetic dextrose medium minus uracil and <br /> These cells carried mutations chromosomal           leucine, respectively. Growth scores: +. growth: -, growth. <br /> HSP82 chromosomal HSC82 genes carried             Scores parentheses expected dead spores based on <br />                                                                 scores spores tetrad. A D <span id='am-126' about='rdfs:comment' typeof='owl:Thing'>comment</span> marks <br /> wild-type HSC82 gene 2,um plasmid (confirmed          double-disruption (or his(c82 hisp82) mutant spores.           0 <br /> Southern blots). They grew normal rates 25&#176;C YPDA <br />  medium. To select loss URA-containing plasmid, <br /> the cells plated medium containing 5-fluoro-          density (Fig. SA). When cells diluted trans- <br /> orotate. No colonies recovered 104 cells plated.      ferred log phase growth 25 37.5&#176;C, cells <br />  Many colonies obtained 5-fluoro-orotate cells       homozygous hsp82 lIsc82 mutation grew more <br /> carrying URA HSC82 plasmid carried chromo-           slowly wild-type cells (Fig. SB). The slow-growth <br />  somal HSC82 gene, chromosomal HSP82 gene,              phenotype proportional gene dosage. Cells carrying <br /> extrachromosomal HSC82 gene plasmid marked            single copy HSP82 HSC82 genes virtually <br /> HIS.                                                            stopped growing. When mutants transformed with <br />    Individual mutations HSP82 HSC82 affect growth      extrachromosomal centromeric plasmid carrying the <br /> high temperatures. The effects hIsp82 hs1C82         HSC82 gene, growth rates restored expected <br /> mutations vegetative growth examined         values (data shown). Thus, defect growth high <br /> different media different temperatures. The strains     temperatures mutants reduced synthesis <br />  shown Fig. 3C examined greatest detail. All grew    hsc82 hsp82 proteins. <br /> at rate 25&#176;C rich dextrose medium,          In rich dextrose medium (YPDA), difference growth <br /> doubling time 1.8 h, achieved final cell    rates observed mutant wild-type cells <br />  3926 <br />  <br />  <br />  <br />        *09 <br />        w <br />        w0 <br />        x <br />  <br />  <br />  <br />  <br />        w <br />        0 <br />        x <br />        E <br />         (0 <br />        .1 <br />        C) <br />              BORKOVICH ET AL. <br />  <br />              10000 <br />  <br />  <br />               1000 <br />  <br />  <br />                100 <br />  <br />  <br />                 10 <br />  <br />  <br />  <br />  <br />              10000 <br />  <br />  <br />               1000 <br />  <br />  <br />                100 <br />  <br />                 10 <br />                      0 <br />                          A <br />  <br />  <br />  <br />  <br />                          [ <br />                          l <br />  <br />                          I <br />                              , ~ ~CLD82 <br />                              /W <br />                                &#167; <br />  <br />  <br />                               10 <br />                                    3    0 <br />  <br />  <br />  <br />  <br />                                        20 <br />                                             3 LE--&gt;-W33LU <br />  <br />  <br />  <br />  <br />                                         Hours <br />                                                  A <br />                                                  * <br />                                                  * <br />  <br />                                                 30 <br />                                                       PLD82 <br />                                                      ~~~~SCC82 <br />                                                       SCP82 <br />  <br />                                                       40         50 <br />                                                                               g 50 <br />  <br />                                                                               &#176;. 30 <br />                                                                              CO <br />                                                                                  , <br />                                                                                      70 - <br />                                                                                      60 <br />  <br />                                                                                      40 <br />  <br />  <br />                                                                                      20 <br />                                                                                      10 <br />  <br />  <br />  <br />  <br />                                                                       indicated times. <br />                                                                                       0- <br />                                                                                       20           40 <br />                                                                                                          Hours <br />                                                                                                                      60 <br />                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                                                                 80 <br />  <br />                                                                         FIG. 6. Demonstration mutations HSC82 HSP82 do <br />                                                                       influence sporulation efficiency. Cells indicated strains <br />                                                                       inoculated sporulation medium 30&#176;C. Percent asci the <br />                                                                       culture (calculated sum two-, three-, four-spored asci) <br />                                                                       determined microscopic inspection cultures the <br />  <br />  <br />        C.) <br />        0)                                                             Figure 6 displays time course sporulation strains <br />                                                                       W303leu, PLD82, CLD82, LP112, W303ura, CUD82 at <br />                                                                       25&#176;C liquid medium. The significant difference in <br />                                        20                             sporulation efficiency strains tempera- <br />                                         Hours                         ture difference nutritional markers; LEU2 <br />   FIG. 5. Demonstration mutations HSC82 HSP82 affect       ura3 strains sporulated faster higher final efficiency <br /> growth 37.5&#176;C 25&#176;C. (A) Growth 25&#176;C. Cells         leu2 URA3 strains. In separate experiments comparing <br /> inoculated YPDA 25&#176;C counted indicated times    strains depicted Fig. 3 5 (W303leu, CLD82, <br /> a hemacytometer. (B) Growth 37.5&#176;C. Cells inoculated     PLD82, SCC82, SCP82, marked LEU2) 18, <br /> YPDA 37.5&#176;C counted panel A. Key symbols      25, 30, 34&#176;C liquid medium plates, consis- <br /> same panel A.                                                   tent, substantial differences sporulation efficiencies were <br />                                                                       observed (data shown). Finally, spore viability was <br />                                                                       examined dissected asci cells sporulated 25&#176;C <br /> at temperatures 36&#176;C. Between 36 38&#176;C, differ-              (Table 2). For strains examined, spore viabilities <br /> ences mutant wild-type cells pro-               approximately same, LEU2 spores having a <br /> nounced higher temperature. (As typical S.              slight advantage leu2 spores. <br /> cerevisiae, wild-type strain does grow 39&#176;C.)              Thermotolerance. The effects mutations HSP82 <br /> Curiously, negative effects mutations growth         HSC82 genes thermotolerance (that is, ability <br /> high temperatures somewhat dependent cell densi-              cells survive brief exposures extreme temperatures) <br /> ties. That is, effect pronounced log-               examined dextrose medium, supports growth <br /> phase cells diluted 104 105/ml 106/ml.         fermentation, acetate medium, supports <br /> Cultures mutant cells grew slowly high                growth respiration. The results typical thermotoler- <br /> temperatures stationary-phase cells used                ance assay, obtained exposing cells growing rich <br /> inoculation (data shown).                                         dextrose medium 25 50&#176;C 10 min, presented in <br />    Microscopic analysis strains examined Fig. 3         Fig. 7A. Survival rates strains increased cells went <br /> 5 13 h high temperature revealed large number           early log phase stationary phase growth <br /> double cells cell clusters. The cells bigger            (Fig. 7B). Survival rates increased 2 orders of <br /> than wild-type cells, associations dis- <br /> rupted sonication. Occasionally, large cells were <br /> observed lyse, leaving ghosted cell walls. Surpris-                                TABLE 2. Spore viability <br /> ingly, substantial number cells retained               Strain                   Relevant genotype                   viability <br /> colony-forming capacity; 25 h 37.5&#176;C, 10% the <br /> cells strain SCC82 1% cells strain SCP82             W303leu             LEU2 ura3 HSP90 HSC82                        100 <br /> produced colonies plated 25&#176;C YPDA medium                                      LEU2 ura3 HSP90 HSC82 <br /> (data shown).                                                     CLD82               leu2 ura3 HSP82 hsc82::LEU2                   95 <br />    Sporulation efficiencies spore viability. As discussed                             leu2 ura3 HSP82 hsc82::LEU2 <br /> above, hsp82 protein induced stationary phase          PLD82               Ieu2 ura3 hsp82::LEU2 HSC82                  100 <br /> and sporulation (Borkovich Lindquist, prepa-                                Ieu2 ura3 hsp82::LEU2 HSC82 <br /> ration). Sporulation efficiencies compared          W303ura             Ieu2 ura3 HSP82 HSC82 CEN URA3                 83 <br /> various strains 18, 25, 30, 34WC liquid medium                              leu2 ura3 HSP82 HSC82 CEN URA3 <br /> on plates. (As commonly observed yeast strains,                 CUD82               Ieu2 ura3 HSP82 hsc82:: URA3                   80 <br /> sporulation efficiencies strongly adversely affected                             leu2 ura3 HSP82 hsc82::URA3 <br />                                                                       LP112               Ieu2 ura3 HSP82 HSC82                          90 <br /> in parental stain temperatures 35&#176;C; sporula-                                Ieu2 ura3 HSP82 HSC82 <br /> tion examined mutants 34&#176;C.) <br />  VOL. 9, 1989                                                                                       CHARACTERIZATION OF hsp82                   3927 <br />  <br />            1000o <br />                          A                                                         4 <br />                                                                                200-4&apos; <br />  <br />                          v2&apos; <br />             100- <br />  <br />      &apos;a        10I                                                                    97- <br />                                                                                         .t~~tI   ~  - hsp82 <br />                 1-           ,X-0-- W303LEU+PT <br />       e-                       -0-- CLD82+PT                                    68-       _jSql_&quot;|g_ hsc82 <br />                 .1            6-tx- PLD82+PT <br />                              _@- W303LEU-PT <br />                .01 <br />  <br />             .001 <br />                  I           - <br />                                -U-- CLD82-PT <br />                                - - PLD82-PT <br />                                                                                 434J                       t:hw- <br />                      0           10       20         30 <br />                                       Hours                                        26  - <br />  <br />  <br />            109       I                                     .~~~~~~~~~~~~~~- <br />  <br />  <br />  <br />                          B <br />                                                                                 18- <br />            108 <br />                                                                                                            C H ,C H,CHM <br />       E                                                                                              r, <br />                                                                                                              -o      co <br />                                                                                              CA) <br />                                                                                                              c-      o <br />                                                                                                                      CD       C: <br />                                             ^~~~~ <br />                                                  PLD82                                       CA) <br />                                                                                               D      O       00               oo <br />                                                                                                     N        Na      C        ro <br />                                                                                              CD <br />             16 <br />                      0           10        20        30 <br />                                                                                 FIG. 8. Demonstration mutations HSC82           HSP82 do <br />                                       Hours                                   affect expression cellular proteins 25 39&#176;C. <br />   FIG. 7. Demonstration hsp82 hsc82 mutants                   Cells grown 25&#176;C incubated 30 min 25 39&#176;C. <br /> thermotolerant wild-type cells YPDA. (A) Thermotolerance             [3Hlisoleucine added, incubation continued 30 min <br /> YPDA. Cells designated strain inoculated YPDA               more. Total cellular proteins separated     SDS-polyacryl- <br /> low density cultured 25&#176;C. Equal portions withdrawn            amide gel visualized fluorography. <br /> various times, incubated 50&#176;C 8 min, plated YPDA <br /> medium determine fraction surviving cells described           strains disappeared. Apparently, mutations make the <br /> Materials Methods. +PT, Cells pretreated 37&#176;C 30 <br /> min 50&#176;C incubation; -PT, cells did receive                  cells sensitive manipulation heat treatment. <br /> pretreatment. (B) Growth curve YPDA. Cell densities                 Depending protocol used, not <br /> cultures analyzed panel A determined indicated times           manifest itself difference thermotolerance. <br /> by counting hemacytometer.                                                Finally, examined patterns protein synthesis in <br />                                                                               wild-type mutant cells normal temperatures and <br /> magnitude cells pretreated 37&#176;C 30 min                       heat shock determine reduced synthesis of <br /> before 10-min treatment 50&#176;C. As observed previ-                       hsp82 hsc82 mutant cells effect the <br /> ously, pretreatments lesser effects thermotolerance                    expression proteins (Fig. 8). No differences be <br /> as cells entered stationary phase, late-stationary-phase                 control heat-shocked cells wild type or <br /> cells showing effect all. At equivalent points                   mutants visual inspection fluorograph a <br /> growth curve, mutant wild-type strains showed similar                     one-dimensional gel. <br /> rates survival pretreatment similar in- <br /> creases survival pretreatment.                                                               DISCUSSION <br />    Thermotolerance examined cells grown                          We shown haploid yeast genome contains <br /> acetate, forces respiratory metabolism. As                    closely related genes HSP82 gene family, HSC82 <br /> was case growth glucose, increase                 HSP82, overall amino acid identity 97%. The <br /> thermotolerance strains cells transited early              product HSC82 gene, hsc82, expressed very <br /> log stationary phase growth. In initial experiments,                high level 25&#176;C abundant proteins in <br /> mutant wild-type cells showed differences thermo-                   cell. It induced 1.5- 2-fold 39&#176;C. The product of <br /> tolerance point growth curve shifted                       HSP82 gene, hsp82, expressed lower level at <br /> directly 25 50&#176;C. However, cells                        25&#176;C. It exhibits approximately 20-fold increase expres- <br /> given conditioning treatment 37&#176;C shift                    sion 39&#176;C accumulates nearly level the <br /> 50&#176;C, marked difference temperature                        hsc82 protein. Haploid spores contain func- <br /> sensitivity strains. Mutants readily killed                  tional HSC82 HSP82 gene germinate germinate <br /> than wild-type cells virtually point growth                   divide times. Such cells rescued an <br /> curve, greatest differences strains (on                  extrachromosomal plasmid carrying wild-type HSC82 <br /> <span id='am-90' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> 10-fold) observed early-log-phase late-sta-                   gene. Vegetatively growing cells lose plasmid die <br /> tionary-phase cells. To investigate difference thermo-                rescued transformation an <br /> tolerance detail, variety modifications                     HSP82 gene. Thus, patterns expression <br /> made basic protocol. In initial experiments, cells                vary, proteins serve identical nearly identical <br /> diluted ice-cold medium plating. When cells                   functions, expression essential <br /> were plated directly, dilution, differences                   growth normal temperatures. <br />   3928     BORKOVICH ET AL.                                                                                          MOL. CELL. BIOI. <br />  <br />     Diploid cells carrying single copy HSP82           temperature, find. Our results also <br />  gene grow wild-type cells 25&#176;C. Since genes       indicate cells need higher concentrations hsp82 or <br />  have, best, limited capacity dosage compensa-         hsc8&apos; protein growth higher temperatures. This result <br />  tion, means cells express order 1/10      keeping model. Protein-protein interac- <br />  th normal level protein exhibit phenotype             tions profoundly affected increases temperature, <br />  tefperature. In   separate experiments (unpublished data).          association process involves large changes the <br />   wi haploid cells carrying extra copy         entropy proteins solvent, water (summarized <br />  HSC82 gene CEN plasmid produce nearly twice            reference 10). We suggest stability interactions <br />  normal level protein, apparent ill effects. Cells        hsp82 hsc82 proteins decreases the <br />  carrying extra copy HSC82 gene 2pm plasmid           temperature increases. Thus, higher protein concentrations <br />  contain, average, roughly times wild-type level      required maintain proper levels complex <br />  protein. These cells grow wild type       formation. In case. efficient and <br />  contain relatively high concentration breakdown prod-          economical induce hsp82 hsc82. protein <br />  ucts protein. Thus, cells tolerate con-        shared variety complexes cells able <br />  siderable variation concentration hsp82 hsc82       tolerate considerable excess protein ill <br />  low temperature, does appear upper limit.            effect. <br />     At temperatures 36.5&#176;C. rate cell growth                Comparison results HSP82 gene family in <br />  varies copy number HSC82 HSP82                  S.(cerevisiae results similar experiments E. oli <br />  genes. At   37.5&#176;C.    diploid cells homozygous        reveals important similarities differences. Deletion mu- <br />  mutations HSP82 HSC82 genes grow slowly              tants litpG gene (which encodes homologous C62.5 <br />  wild-type. When homozygous mutants               protein E.(oli) display extremely subtle disad- <br />  heterozygous mutations gene,             vantage growth normal temperatures, disad- <br />  virtually grow all. At temperatures,             vantage pronounced extreme temperatures <br />  wild-type cells hsp82 hsc82 proteins expressed       (4). This phenotype similar phenotypes individual <br />  nearly level abundant               mutations yeast HSP82 HSC82 genes suggests <br />  proteins cell. Thus, cells require high concentra-      similarity function. However, inc(oli <br />                                                                                                              E.       appears to <br />  tions proteins growth high temperatures,         gene family. deletion viable. It <br />  order 10 20 times protein require         doubtful closely related gene covers essential <br />  growth at25&#176;C.       In light results, different   function htpjG cross-reacting material de- <br />  patterns expression observed functionally          tected polyclonal antibodies cross-hybridizing <br />  equivalent proteins make great deal sense. One gene          material detected low-stringency Southern blot <br />  expressed constitutively high level (probably          analysis. It appears, then, protein essential in <br />  close maximal level yeast gene). This provides        yeast cells serves auxiliary, growth-enhancing <br />  protein act buffer broad range           role E.(o/i. Given extraordinary degree conserva- <br />  temperature fluctuations. Toward upper extreme           tion protein (40% amino acid identity the <br />  natural growth range, however, gene supply             yeast E. o/i proteins [3]), function of <br />  sufficient quantity protein itself another. heat-          protein the organisms this <br />  inducible gene employed.                                          function simply essential E. o/i. If, example, the <br />      Although little known biochemical associa-          function particular target proteins inactive during <br />  tions hsp82 yeast cells, proteins avian             intracellular transport, smaller size bacterial cells and <br />  mammalian cells, usually referred hsp90.             lack internal membranes make valuable but <br />  reported associate different proteins,          nonessential function. Alternatively, protein have <br />  small percentage total protein pool           acquired novel functions eucaryotic evolution (perhaps <br />  complex protein. For example. hsp90,           related appearance highly charged domain that <br />  cellular phosphoprotein 50 kDa (pp5O),           eucaryotic proteins), these <br />  shown associate newly synthesized               novel functions essential. Extending biochemical <br />  transforming kinases virus-infected cells. These kinases          methods analysis used protein family verte- <br />   little activity cytosolic complex. When      brate cells organisms amenable genetic analysis E. <br />  complex reaches plasma membrane, hsp90 pp5O                  coli S. cerevisiae provide answer these <br />  dissociate, kinase phosphorylated ty-             questions. <br />   rosines acquires activity (1. 8, 9. 11, 28. 51). hsp90 <br />   binds untransformed steroid hormone receptors.                                 ACKNOWLEDGMENTS <br />   Transformation receptor DNA-binding state, <br />   addition hormone. coincides dissociation         This work supported Public Health Service grants <br /> hsp90     receptor (16, 22. 38, 39, 42. 43). hsp90 also <br />   shows affinity tubulin (44), F-actin calmodulin (23. <br />                                                                      GM3582 GM25829 National Institutes Health. K. A. <br />                                                                      Borkovich recipient Public Health Service National <br />                                                                      Research Fellowship award GM11223 National Institutes <br />   33), protein kinase C (F. 0. Fields J. Thorner. personal       Health. <br />   communication), eIF2o kinase (40). spe- <br />   cific nature associations clear.                                        LITERATURE CITED <br />      On surface, associations appear       1. Adkins, B., T. Hunter, B. M. Sefton. 1982. The transforming <br />   different nature, does underlying           proteins PRCII virus <span id='am-9' about='obo:NCBITaxon_11886' typeof='owl:Thing obo:NCBITaxon_11632 obo:NCBITaxon_1 obo:NCBITaxon_35268 obo:NCBITaxon_10239 obo:NCBITaxon_327045 obo:NCBITaxon_153057'><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Rous sarcoma virus RSV" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Rous sarcoma virus" datatype="xsd:string"></span>Rous sarcoma virus</span> form complex <br />   theme. The general function hsp82 bind a <br />   wide variety proteins, keeping quiescent. untIl           cellular phosphoproteins. J. Virol. 43: <br />                                                                          448-45-5. <br />   arrived proper intracellular location         2. Amin, J., J. Ananthan, R. Voellmy. 1988. Key features of <br />   received proper signal activation. One expect            heat shock regulatory elements. Mol. Cell. Biol. 8:3761-3769. <br />   protein essential growth            3. Bardwell. J. C. A., E. A. Craig. 1987. Eukaryotic <br />                                                                                                                     M,           83 .000 <br />  VOL. 9, 1989                                                                                   CHARACTERIZATION OF hsp82                  3929 <br />  <br />       heat shock protein homologue <span id='am-109' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> co/i. Proc.          ences 43&#176;C 45&#176;C. J. Cell. Physiol. 136:991. <br />       Nati. Acad. Sci. USA 84:5177-5181.                                 26. Li, G. C., A. Laszlo. 1985. Thermotolerance mammalian <br />  4.   Bardwell, J. C. A., E. A. Craig. 1988. Ancient heat shock          cells: possible role heat shock proteins, p. 349-371. In <br />       gene dispensible. J. Bacteriol. 7:2977-2983.                        B. G. Atkinson S. B. Walden (ed.), Changes gene <br />  5.   Blackman, R. K., M. Meselson. 1986. Interspecific nucleo-          expression response environmental stress. Academic <br />       tide sequence comparisons used identify regulatory              Press, Inc. (London), Ltd., London. <br />       structural features Drosophlila HSP82 gene. J. Mol. Biol.   27. Lindquist, S., E. A. Craig. 1988. The heat shock proteins. <br />       188:499-515.                                                           Annu. Rev. Genet. 22:631-677. <br />  6.   Borkovich, K. A., R. L. Weiss. 1987. Purification          28. Lipsich, L. A., J. R. Cutt, J. S. Brugge. 1982. Association of <br />       characterization arginase Neuirospora <span id='am-31' about='obo:NCBITaxon_688987' typeof='owl:Thing obo:NCBITaxon_37581 obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_116123 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_57992 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="rdfs:label" content="Crassa" datatype="xsd:string"></span>crassa</span>. J. Biol.         transforming proteins Rous, Fujinami, Y73 avian <br />       Chem. 262:7081-7086.                                                   sarcoma <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span>viruses</span> cellular proteins. Mol. Cell. <br />  7.   Broach, J. R., J. N. Strathern, J. B. Hicks. 1979. Transfor-       Biol. 2:875-880. <br />       mation yeast: development <span id='am-142' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-143' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-145'  ></span><span id='am-146' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> cloning vector        29. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />       isolation CANI gene. Gene 8:121-133.                            cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />  8.   Brugge, J. S., E. Erikson, R. L. Erikson. 1981. The specific       Cold Spring Harbor, N.Y. <br />       interaction Rous sarcoma virus transforming protein,        30. Mazzarella, R. A., M. Green. 1987. ERp99, abundant, <br />       pp6Osrc, cellular proteins. Cell 25:363-372.                  conserved glycoprotein endoplasmic reticulum, homol- <br />  9.   Brugge, J. S., W. Yuonemoto, D. Darrow. 1983. Interaction          ogous 90-kDa heat shock protein (hsp90) 94-kDa <br />       Rous sarcoma virus transforming protein            glucose regulated protein (GRP94). J. Biol. Chem. 262:8875- <br />       cellular phosphoproteins: analysis turnover distribu-       8883. <br />       tion complex. Mol. Cell. Biol. 4:2697-2704.                31. Nasmyth, K. A., K. Tatchell. 1980. The structure trans- <br /> 10.   Clothia, C., J. Janin. 1975. Principles protein-protein         posable yeast mating type loci. Cell 19:753-764. <br />       recognition. Nature (London) 256:705-708.                          32. Neidhardt, F., R. VanBogelen, E. Lau. 1983. Molecular <br /> 11.   Courtneidge, S. A., J. M. Bishop. 1982. Transit pp60&apos;,-rc       cloning expression gene controls high temper- <br />       plasma membrane. Proc. Natl. Acad. Sci. USA 79:                 ature regulon Escherichia coli. J. Bacteriol. 153:597-603. <br />       7117-7121.                                                         33. Nishida, E., S. Koyasu, H. Sakai, I. Yahara. 1986. Calmod- <br /> 12.   Craig, E. A., K. Jacobsen. 1984. Mutations heat-            ulin-regulated binding 90-kDa heat shock protein actin <br />       inducible 70 kilodalton genes yeast confer temperature sen-         filaments. J. Biol. Chem. 261:16033-16036. <br />       sitive growth. Cell 38:841-849.                                    34. Pelham, H. R. B. 1982. A regulatory upstream promoter element <br /> 13.   Craig, E. A., J. Kramer, J. Kosic-Smithers. 1987. SSCI,          Drosophila Hsp7O heat-shock gene. Cell 30:517-528. <br />       member 70-kDa heat shock protein multigene family        35. Pelham, H. R. B. 1985. Activation heat-shock genes in <br />       Saccharomyces cerev&apos;isiae, essential growth. Proc. NatI.        eukaryotes. Trends Genet. 1:31-35. <br />       Acad. Sci. USA 84:4156-4160.                                       36. Petko, L., S. Lindquist. 1986. Hsp26 required for <br /> 14.   Davis, R. W., M. Thomas, J. Cameron, T. P. St. John, S.                growth high temperatures, thermotolerance, spore <br />       Scherer, R. A. Padgett. 1980. Rapid DNA isolations             development, germination. Cell 45:885-894. <br />       enzymatic hybridization analysis. Methods Enzymol. 65:         37. Rebbe, N. F., J. Ware, R. M. Bertina, P. Modrich, D. W. <br />       404 411.                                                               Stafford. 1987. Nucleotide sequence cDNA member of <br /> 15.   Devereux, J., P. Haeberli, 0. Smithies. 1984. A comprehen-         human 90-kDa heat-shock protein family. Gene 53:235-245. <br />       sive set sequence analysis programs VAX. Nucleic        38. Redeuilh, G., B. Moncharmont, C. Secco, E.-E. Baulieu. <br />       Acids Res. 12:387-395.                                                 1987. Subunit composition molybdate-stabilized &quot;8-9 S&quot; <br /> 16.   Dougherty, J. J., R. K. Puri, D. 0. Toft. 1984. Polypeptide        nontransformed estradiol receptor purified calf uterus. J. <br />       components 8 S forms chicken oviduct progesterone            Biol. Chem. 262:6969-6975. <br />       receptor. J. Biol. Chem. 259:8004-8009.                            39. Renoir, J.-M., T. Buchou, E.-E. Baulieu. 1986. Involvement <br /> 17.   Dragon, E. A., S. R. Sias, E. A. Kato, J. D. Gabe. 1987. The       non-hormone-binding 90-kilodalton protein nontrans- <br />       genome Trypanosotna cruzi contains constitutively ex-             formed 8S form rabbit uterus progesterone receptor. <br />       pressed, tandemly arranged multicopy gene homologous              Biochemistry 25:6405-6413. <br />       major heat shock protein. Mol. Cell. Biol. 7:1271-1275.            40. Rose, D. W., R. E. H. Wettenhall, W. Kudlicki, G. Kramer, and <br /> 18.   Farrelly, F. W., D. B. Finkelstein. 1984. Complete sequence        B. Hardesty. 1987. The 90-kilodalton peptide heme- <br />       heat shock-inducible HSP90 gene Saccharomyces                regulated eIF-2a kinase sequence similarity 90- <br />       cerev&apos;isiae. J. Biol. Chem. 259:5745-5751.                             kilodalton heat shock protein. Biochemistry 26:6583-6587. <br /> 19.   Finley, D., E. Ozkaynak, A. Varshavsky. 1987. The yeast        41. Rothstein, R. J. 1983. One-step gene disruption yeast. Meth- <br />       polyubiquitin gene essential resistance high tempera-        ods Enzymol. 101:202-211. <br />       tures, starvation stresses. Cell 48:1035-1046.           42. Sanchez, E. R., P. R. Housley, W. B. Pratt. 1986. The <br /> 20.   Grossman, A. D., J. W. Erickson, C. A. Gross. 1984. The            molybdate-stabilized glucocorticoid binding complex L-cells <br />       htpR gene Escherichia coli sigma factor heat-shock         contains 98-100 Kdalton steroid binding phosphoprotein a <br />       promoters. Cell 28:282-290.                                            90 Kdalton nonsteroid-binding phosphoprotein the <br /> 21.   Ito, H., Y. Fukada, K. Murata, A. Kimura. 1983. Transfor-          murine heat-shock complex. J. Steroid Biochem. 24:9-18. <br />       mation intact yeast cells treated alkali cations. J.       43. Sanchez, E. R., S. Meshinchi, W. Tienrungroj, M. J. Schlesinger, <br />       Bacteriol. 153:163-168.                                                D. 0. Toft, W. B. Pratt. 1987. Relationship 90-kDa <br /> 22.   Joab, I., C. Radanyi, M. Renoir, T. Buchou, M.-G. Catelli, N.          murine heat shock protein untransformed trans- <br />       Binart, J. Mester, E.-E. Baulieu. 1984. Common non-                formed states L cell glucocorticoid receptor. J. Biol. <br />       hormone binding component non-transformed chick oviduct             Chem. 262:6986-6991. <br />       receptors steroid hormones. Nature (London) 308:           44. Sanchez, E. R., T. Redmond, L. C. Scherrer, E. H. Bresnik, <br />       850-853.                                                               M. J. Welsh, W. B. Pratt. 1988. Evidence 90- <br /> 23. Koyasu, S., E. Nishida, T. Kadowaki, F. Matsuzaki, K. lida, F.           kilodalton heat shock protein associated tubulin-con- <br />     Harada, M. Kasuga, H. Sakai, I. Yahara. 1986. Two                    taining complexes L cell cytosol intact PtK cells. Mol. <br />     mammalian heat shock proteins, HSP90 HSP100 actin-               Endocrinol. 2:756-760. <br />     binding proteins. Proc. NatI. Acad. Sci. USA 83:8054-8058.           45. Slater, M. R., E. A. Craig. 1987. Transcriptional regulation <br /> 24. Kurtz, S., S. L. Lindquist. 1984. The changing pattern            hsp70 heat shock gene yeast Saccharomyces <br />     gene expression sporulating yeast. Proc. Natl. Acad. Sci.             cerei&apos;isiae. Mol. Cell. Biol. 7:1906-1916. <br />     USA 81:7323-7327.                                                    46. Struhl, K., D. T. Stinchcomb, S. Scherer, R. W. Davis. 1979. <br /> 25. Lee, Y. J., W. C. Dewey. 1988. Thermotolerance induced            High frequency transformation yeast: autonomous replication <br />     heat, sodium arsenite puromycin: inhibition differ-           hybrid DNA molecules. Proc. Natl. Acad. Sci. USA 76: <br />  3930     BORKOVICH ET AL.                                                                                            MOL. CELL. BIOL. <br />  <br />     1035-1039.                                                           protein synthesis, heat shock protein mRNA accumulation <br /> 47. VanBogelen, R. A., M. A. Acton, F. C. Neidhardt. 1987.           rat fibroblasts. Mol. Cell. Biol. 6:1088-1094. <br />     Induction heat shock regulon does produce thermo-     50. Xiao, H., J. T. Lis. 1988. Germline transformation used to <br />     tolerance Escherichia coli. Genes Dev. 1:525-531.                 define key features heat-shock response elements. Science <br /> 48. Watson, K., G. Dunlop, R. Cavicchioli. 1984. Mitochondrial       239:1139-1142. <br />     cytoplasmic protein synthesis required heat shock    51. Ziemiecki, A. 1986. Characterization monomeric and <br />     acquisition ethanol- thermotolerance yeast. FEBS           complex-associated forms gaig-onc fusion proteins of <br />     Lett. 169:267-273.                                                   isolates feline sar <br /> </body></html>