<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Science. Author manuscript; available PMC 2013 December 10. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Science. 2012 February 24; 335(6071): . doi:10.1126/science.1215156. <br />  <br />  <br />  <br />                            Control non-apoptotic developmental cell death C. elegans <br />                            polyglutamine-repeat protein <br />                            Elyse S. Blum, Mary C. Abraham, Satoshi Yoshimura, Yun Lu, Shai Shaham* <br />                            Laboratory Developmental Genetics, The Rockefeller University, 1230 York Avenue, New <br />                            York, NY 10065 USA <br />  <br />                            Abstract <br />                                    Death vital developmental cell fate. In <span id='am-11' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span>Caenorhabditis elegans</span>, programmed death the <br />                                    linker cell, leads gonadal elongation, proceeds independently caspases apoptotic <br />                                    effectors. To identify genes promoting linker-cell death, performed genome-wide RNA <br />                                    interference screen. We linker-cell death requires gene pqn-41, encoding an <br />                                    endogenous polyglutamine-repeat protein. pqn-41 functions cell autonomously, expressed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                    onset linker-cell death. pqn-41 expression controlled MAP kinase kinase SEK-1, <br />                                    functions parallel Zn-finger protein LIN-29 promote cellular demise. Linker-cell <br />                                    death morphologically similar cell death associated normal vertebrate development and <br />                                    polyglutamine-induced neurodegeneration. Our results may, therefore, provide molecular in-roads <br />                                    understanding non-apoptotic cell death metazoan development disease. <br />  <br />                                                Programmed cell death essential metazoan development required sculpt <br />                                                organs, eliminate harmful cells, counter cell division (1, 2). Apoptosis, extensively <br />                                                studied cell death process, requires caspase activation, accompanied stereotypical <br />                                                morphological signature including chromatin compaction, cytoplasmic shrinkage, no <br />                                                gross disruption organelles (3). Mice lacking key apoptotic effectors caspase-3, <br />                                                caspase-9, Apaf-1, Bax Bak, mild defects, survive adulthood (4). <br />                                                Given prevalence cell death murine development (5), observations raise <br />                                                possibility non-apoptotic cell death pathway plays key roles animal <br />                                                development. Although genes promoting necrotic cell death identified (6), they <br />                                                required normal development (7). Indeed, genes dedicated non-apoptotic <br />                                                developmental cell death described. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                We previously described programmed death C. elegans linker cell male <br />                                                reproductive development (8). The linker cell leads migration male gonad <br />                                                dies fourth larval stage (L4) adulthood (9). Linker cell demise is <br />                                                orchestrated cell-autonomous process independent C. elegans caspases other <br />                                                known cell death genes (8, 10). <span id='am-34' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Electron</span> microscopy (EM) dying linker cell reveals <br />                                                non-apoptotic features, including crenellation (indentation) nuclear envelope, <br />                                                uncondensed chromatin, organelle swelling (8) (Fig. 1C). However, linker cell death <br />                                                does require unfolded protein response stress responses (tables S1 S2). <br />                                                Similar morphological features seen normal developmental death neurons in <br />                                                vertebrate spinal cord ciliary ganglion (11&#8211;13), suggesting linker cell-type death <br />                                                morphologically conserved. <br />  <br />                                                To uncover molecular mechanisms leading linker cell death, performed genome- <br />                                                wide RNA interference (RNAi) screen identify genes inactivation prevents linker <br />  <br />                            * <br />                                To correspondence addressed. Tel (212) 327-7126, Fax (212) 327-7129, shaham@rockefeller.edu. <br />                             Blum et al.                                                                                           Page 2 <br />  <br />  <br />                                          cell death. 18,132 bacterial clones, expressing double-stranded RNA (dsRNA) <br />                                          corresponding 89% predicted protein-coding C. elegans genes, fed individually to <br />                                          RNAi-sensitized males expressing lag-2 promoter::GFP linker cell reporter (14). The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          screen strategy validated recovery lin-29 gene, previously identified as <br />                                          required linker cell death (8). We recovered additional RNAi clones leading to <br />                                          linker cell survival. Three clones affect linker cell migration, affect <br />                                          multiple aspects linker cell fate. One remaining clones, derived gene <br />                                          pqn-41 (Fig. 2A), caused linker cell survival 20% animals (Fig. 1A) obvious <br />                                          pleiotropic effects (fig. S1), studied. <br />  <br />                                          To confirm clone identified inactivates pqn-41, generated deletion allele, <br />                                          ns294, gene. pqn-41(ns294) animals lack 337 nucleotides, removing portions intron <br />                                          17 exon 18 predicted genomic structure (Fig. 2A). Consistent RNAi <br />                                          results, pqn-41(ns294) adult males possess inappropriately surviving linker cell (Fig. <br />                                          1A,B). Furthermore, pqn-41(ns294) mutants, linker cell persists 24 hours after <br />                                          L4-adult transition (19% survival, n=79), suggesting reducing pqn-41 function not <br />                                          delays, block linker cell death. RNAi pqn-41 pqn-41(ns294) animals <br />                                          does increase linker cell survival (16% survival, n=108), suggesting ns294 a <br />                                          strong loss-of-function allele. To test pqn-41(ns294) mutants defects in <br />                                          developmental apoptosis, scored surviving pharyngeal cells normally die <br />                                          apoptotically (15), observed (n=10). Thus, non-apoptotic nature linker cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          death reflect distinct molecular machineries. <br />  <br />                                          None pqn-41(ns294) surviving cells exhibit nuclear crenellation (n=19), observed <br />                                          light microscopy. To examine surviving linker cell morphology higher resolution, we <br />                                          performed serial-section EM. We cell forms adherens junctions to <br />                                          surrounding epithelia (8), nuclear envelope crenellation mild (Fig. 1D; n=2). In <br />                                          animals, however, organelle swelling evident (Fig. 1D; fig. S2A,B), suggesting <br />                                          pqn-41 required nuclear crenellation organelle swelling. <br />  <br />                                          To determine pqn-41 functions cell-autonomously, examined rde-1(ne219) RNAi <br />                                          deficient mutants (16), containing mig-24 promoter::rde-1 cDNA transgene restoring <br />                                          RNAi linker cell (17). Feeding pqn-41 RNAi bacteria mutants resulted in <br />                                          linker cell death defects similar systemic RNAi (Fig. 1A), suggesting pqn-41 <br />                                          functions cell-autonomously. To examine issue, assessed expression <br />                                          pqn-41 linker cell restore linker cell death pqn-41(ns294) mutants. The <br />                                          pqn-41 locus encodes alternate transcripts designated pqn-41A, B, C <br />                                          (Fig. 2). pqn-41A/B span locus, differing alternative in-frame exon b. pqn-41C <br />                                          mRNA initiates immediately upstream exon b (Fig. 2B). We expression of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          pqn-41C sufficient promote linker cell demise pqn-41(ns294) mutants (Fig. 2D), <br />                                          strongly suggesting pqn-41 functions cell autonomously promote linker cell death. <br />  <br />                                          pqn-41C encodes protein composed runs glutamine residues eight <br />                                          residues run (Fig. 2C; fig. S3A). 151 427 pqn-41C codons encode glutamine. The <br />                                          average number non-glutamine amino acids interrupting adjacent glutamine residues in <br />                                          PQN-41C 1.8 second smallest C. elegans proteome (fig. S4). Glutamine- <br />                                          rich domains hallmarks neurodegenerative disease (ND) proteins Q/N- <br />                                          rich prions (18). Both domain classes adopt coiled-coil structures (18). Similarly, <br />                                          PQN-41C predicted contain coiled-coil motifs demarcated flanking prolines <br />                                          (CC1-6; fig. S5). Three sequences outside coiled-coil motifs conserved among <br />                                          <span id='am-28' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-29' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span>nematodes</span> (CD1-3; fig. S3B), overall proportion glutamines (38%, 37%, and <br />                                          37% C. remanei, C. brenneri C. briggsae, respectively). <br />  <br />  <br />  <br />                                               Science. Author manuscript; available PMC 2013 December 10. <br />                             Blum et al.                                                                                            Page 3 <br />  <br />  <br />                                          ND Q/N-rich polyglutamine proteins tend aggregate cells (18). Similarly, found <br />                                          PQN-41C::GFP protein forms cytoplasmic aggregates linker cell (Fig. 3G), <br />                                          suggesting PQN-41C shares structural features proteins. To understand the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          importance coiled-coil conserved regions PQN-41C function, examined the <br />                                          effects protein truncations ability PQN-41C rescue pqn-41(ns294) mutants <br />                                          (fig. S6A). Truncation consecutive pairs coiled-coil domains abolishes PQN-41C <br />                                          rescuing activity. Coil-breaking prolines coiled-coil domains 2, 4, 6, reduce <br />                                          rescue efficiency. Thus, sequences encoding coiled-coil regions important for <br />                                          PQN-41C function. Truncation conserved domain CD3 abolishes rescue, whereas <br />                                          deletion CD1/2 domains modest effects. <br />  <br />                                          Expression polyglutamine repeats toxic. However, PQN-41C does exhibit <br />                                          non-specific cellular toxicity. Expression PQN-41C linker cell pqn-41(ns294) <br />                                          males starting L2 larvae did cause precocious cell death (n=227). Rather, cell death <br />                                          initiated appropriately L4-adult transition. Likewise, expression PQN-41C in <br />                                          hermaphrodite distal tip cells anchor cell did kill cells (n&gt;38). Thus, <br />                                          PQN-41C requires appropriate cellular context promote death. <br />  <br />                                          To investigate functions pqn-41A/B transcripts, tested could <br />                                          restore linker cell death pqn-41(ns294) mutants. We transcripts enhance <br />                                          linker cell survival pqn-41(ns294) (Fig. 2D) wild-type animals (fig. S6B). While <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          polyglutamine proteins protect cells polyglutamine toxicity (19), is <br />                                          puzzling PQN-41A/B protect linker cell, given proteins contain the <br />                                          glutamine-rich sequences PQN-41C. It possible N-terminus PQN-41A/B <br />                                          overrides cell death-promoting activity glutamine-rich domain. Supporting <span id='am-23' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-24' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>this</span> <br />                                          <span id='am-1' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>, N-terminus sufficient block linker cell death (fig. S6B). Furthermore, <br />                                          PQN-41A/B::GFP proteins expressed linker cell fail aggregate (Fig. 3E, F). <br />  <br />                                          ND Q/N-rich proteins contain coiled-coil motifs outside glutamine-rich <br />                                          domain, associate proteins containing domains (18). Similarly, PQN-41A B <br />                                          N-termini predicted contain coiled-coil motifs (fig. S7). We found, <br />                                          however, deletion sequences encoding different coiled-coil domains did abolish <br />                                          ectopic survival induced pqn-41B (fig. S6B). <br />  <br />                                          We sought characterize pqn-41 expression. A 13 kb DNA fragment derived the <br />                                          pqn-41 locus fused gfp sequences broadly expressed transgenic animals most <br />                                          cells starting embryo (Figs. 2A, 3). However, expression switched the <br />                                          linker cell cell begins die (Fig. 3A&#8211;D). A 2.5 kb DNA sub-fragment derived from <br />                                          reporter promotes gfp expression nearly exclusively linker cell, cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          death initiation (Fig. 2A; fig. S8). The sequences driving linker cell expression pqn-41 <br />                                          overlap deleted pqn-41(ns294) mutants, lie upstream pqn-41C start <br />                                          (Fig. 2A), suggesting control pqn-41C expression. <br />  <br />                                          We previously demonstrated role LIN-29 Zn-finger transcription factor linker <br />                                          cell death (8)(fig. S9B). lin-29 expressed linker cell cell begins die, <br />                                          suggesting lin-29 control pqn-41 expression. However, pqn-41 expression only <br />                                          modestly disturbed lin-29(RNAi) males (Fig. 4B; fig. S10A). To identify strong <br />                                          regulators pqn-41 expression, tested clones identified RNAi screen. A clone <br />                                          directed gene tir-1, encoding p38 MAP kinase scaffolding protein important for <br />                                          C. elegans innate immunity, neuronal differentiation stress responses (20, 21), blocks <br />                                          linker cell death, does tir-1(qd4) genetic lesion (fig. S9B). We tested other <br />                                          genes involved innate immunity stress affect linker cell death. Two independent <br />                                          alleles p38 cascade MAPKK gene sek-1 strongly block linker cell death (Fig. 4A; fig. <br />  <br />  <br />                                               Science. Author manuscript; available PMC 2013 December 10. <br />                             Blum et al.                                                                                                          Page 4 <br />  <br />  <br />                                          S9; table S2); however, genes tested effect (table S2). Thus, <br />                                          sek-1 promotes linker cell death independently innate immunity, stress response, and <br />                                          neuronal differentiation pathways. Unlike lin-29(RNAi) animals, pqn-41 expression is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          detected sek-1 mutants (Fig. 4B), suggesting sek-1 functions upstream of <br />                                          pqn-41. Consistent idea, double mutants carrying strong loss-of-function lesion in <br />                                          sek-1 pqn-41(ns294) allele survival defect similar sek-1 mutants <br />                                          (fig. S9B). Furthermore, EM surviving linker cells sek-1(ag1) mutants reveals no <br />                                          nuclear crenellation, ER swelling (n=2; Fig. 4C), consistent regulation of <br />                                          pqn-41 sek-1. The differential effects lin-29 sek-1 pqn-41 expression suggest <br />                                          genes function independently. Indeed, sek-1 expression does require <br />                                          lin-29 function vice versa (fig. S10B,C). Furthermore, lin-29 mutants exhibit the <br />                                          extent ER swelling sek-1 mutants (fig. S11; n=3). Finally, strong loss-of-function <br />                                          mutations lin-29 sek-1 interact additively (fig. S9B), suggesting genes may <br />                                          function parallel. <br />  <br />                                          To determine sek-1 functions, examined expression using sek-1 genomic <br />                                          region::GFP reporter. This transgene expressed linker cell cell&#8217;s <br />                                          development (Fig. 4D). Expression sek-1 cDNA using mig-24 linker cell-specific <br />                                          promoter restored cell death sek-1 mutants greater extent expression sek-1 <br />                                          using lin-48 promoter, active surrounding cells (22) (Fig. 4A). These results support a <br />                                          cell-autonomous role sek-1 linker cell death. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The studies described here, morphological similarities linker cell <br />                                          death vertebrate developmental cell death, raise possibility PQN-41-like <br />                                          proteins mediate non-apoptotic developmental cell death vertebrates. The <br />                                          vertebrate proteins MED12 p400 good candidates proteins. They are <br />                                          similar primary sequence structure PQN-41, contain glutamine-rich C-termini <br />                                          (fig. S12), nuclearly localized, important tumor formation (23, 24). <br />                                          Intriguingly, EM studies reveal nuclear envelope crenellation strongly associated with <br />                                          polyQ expansion diseases (fig. S13). Our studies, thus, raise possibility these <br />                                          disease proteins promote neurodegeneration inappropriately activating linker cell <br />                                          death-type process. <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          We thank J. Darnell Shaham lab members discussions, N. Tishbi technical assistance, D. Kim, C. <br />                                          Bargmann, M. Kinet, M. Kato, P. Sternberg, K. Nishiwaki, K. Matsumoto reagents. Some nematode strains <br />                                          used work provided <span id='am-18' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22'  ></span>Caenorhabditis</span> Genetics Center, funded National Center for <br />                                          Research Resources (NCRR). ESB supported Rockefeller Women &amp; Science Fellowship Program <br />                                          NIH training grant CA09673. S.S. supported NIH grant R01HD042680. <br />  <br />  <br />                               References <br />                                          1. Abraham MC, Shaham S. Trends Cell Biol. 2004; 14:184. [PubMed: 15066636] <br />                                          2. Fuchs Y, Steller H. Cell. 2011; 147:742. [PubMed: 22078876] <br />                                          3. Kerr JF, Wyllie AH, Currie AR. Br J <span id='am-6' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-7' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span>. 1972; 26:239. [PubMed: 4561027] <br />                                          4. Honarpour N, et al. Dev Biol. 2000; 218:248. [PubMed: 10656767] <br />                                          5. Coucouvanis E, Martin GR. Cell. 1995; 83:279. [PubMed: 7585945] <br />                                          6. Declercq W, Vanden Berghe T, Vandenabeele P. Cell. 2009; 138:229. [PubMed: 19632174] <br />                                          7. Newton K, Sun X, Dixit VM. Mol Cell Biol. 2004; 24:1464. [PubMed: 14749364] <br />  <br />  <br />                                                Science. Author manuscript; available PMC 2013 December 10. <br />                             Blum et al.                                                                                             Page 5 <br />  <br />  <br />                                          8. Abraham MC, Lu Y, Shaham S. Dev Cell. 2007; 12:73. [PubMed: 17199042] <br />                                          9. Sulston JE, Albertson DG, Thomson JN. Dev Biol. 1980; 78:542. [PubMed: 7409314] <br />                                          10. Ellis HM, Horvitz HR. Cell. 1986; 44:817. [PubMed: 3955651] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          11. Pilar G, Landmesser L. J Cell Biol. 1976; 68:339. [PubMed: 942724] <br />                                          12. Oppenheim RW, et al. J Neurosci. 2001; 21:4752. [PubMed: 11425902] <br />                                          13. Borsello T, Mottier V, Castagne V, Clarke PG. J Comp Neurol. 2002; 453:361. [PubMed: <br />                                              12389208] <br />                                          14. Simmer F, et al. Curr Biol. 2002; 12:1317. [PubMed: 12176360] <br />                                          15. Ellis RE, Horvitz HR. Development. 1991; 112:591. [PubMed: 1794327] <br />                                          16. Tabara H, et al. Cell. 1999; 99:123. [PubMed: 10535731] <br />                                          17. Tamai KK, Nishiwaki K. Dev Biol. 2007; 308:562. [PubMed: 17588558] <br />                                          18. Fiumara F, Fioriti L, Kandel ER, Hendrickson WA. Cell. 2010; 143:1121. [PubMed: 21183075] <br />                                          19. Faber PW, Voisine C, King DC, Bates EA, Hart AC. Proc Natl Acad Sci U S A. 2002; 99:17131. <br />                                              [PubMed: 12486229] <br />                                          20. Liberati NT, et al. Proc Natl Acad Sci U S A. 2004; 101:6593. [PubMed: 15123841] <br />                                          21. Chuang CF, Bargmann CI. Genes Dev. 2005; 19:270. [PubMed: 15625192] <br />                                          22. Johnson AD, Fitzsimmons D, Hagman J, Chamberlin HM. Development. 2001; 128:2857. <br />                                              [PubMed: 11532910] <br />                                          23. Fuchs M, et al. Cell. 2001; 106:297. [PubMed: 11509179] <br />                                          24. Makinen N, et al. Science. 2011; 334:252. [PubMed: 21868628] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Science. Author manuscr <br /> </body></html>