<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Dec. 1994, p. 8037-8050                                                                        Vol. 14, No. 12 <br /> 0270-7306/94/$04.00+0 <br /> Copyright &#169; 1994, American Society Microbiology <br />  <br />  <br />                 Mutations <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> CDC1 Gene <br />                         Affect Double-Strand-Break-Induced <br />                          Intrachromosomal Recombination <br />                                          JAMES HALBROOK* AND MERL F. HOEKSTRA <br />                                              ICOS Corporation, Bothell, Washington 98021 <br />                          Received 8 June 1994/Returned modification 19 July 1994/Accepted 21 September 1994 <br />  <br />              To isolate <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29'  ></span>Saccharomyces</span> cerevisiae mutants defective recombinational DNA repair, constructed strain <br />           contains duplicated ura3 alleles flank LEU2 ADES genes ura3 locus chromosome V. <br />           When HO endonuclease cleavage site located ura3 alleles, Ura+ recombination is <br />           increased 100-fold wild-type strains following HO induction GAL1,10 promoter. <span id='am-52' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> strain <br />           used screen mutants exhibited reduced levels HO-induced intrachromosomal recombination <br />           significantly affecting spontaneous frequency Ura+ recombination. One mutations isolated <br />           screen affect essential gene CDC]. This mutation, cdcl-100, completely <br />           eliminated HO-induced Ura+ recombination maintained spontaneous preinduced recombination <br />           levels cell viability. cdcl-100 mutants moderately sensitive killing methyl methanesulfonate and <br />           -y irradiation. The elect cdcl-100 mutation recombinational double-strand break repair indicates that <br />           recombinationally silent mechanism sister chromatid exchange responsible efficient <br />           repair DNA double-strand breaks. <br />  <br />    Genomic integrity maintained generations                 cycle progression arrested hydroxyurea (10) cyclohex- <br /> coordinated control DNA replication chromosome                         imide (53) wild-type cells. These <span id='am-105' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-109'  ></span>data</span> suggest abnor- <br /> segregation cell growth division. An important aspect                mally processed DSBs responsible cell death rad52 <br /> of process recognition repair DNA lesions                  mutant cells. <br /> that arise course normal cellular metabolism                    Experiments HO-induced DSBs introduced <br /> from exposure genotoxic agents environment. In                  ribosomal DNA cluster 18-fold repeated CUP1 <br /> yeast Saccharomyces cerevisiae, toxicity DNA double-strand                 gene locus demonstrate efficient repair occur a <br /> breaks (DSBs) dependent factors, including                    RAD52-independent mechanism (43). When CUPJ repeats <br /> ploidy (9, 15, 48), cell cycle position (6, 39, 49), genetic          reduced 18 3, however, RAD52 dependency <br /> constitution cell (15, 36). Prior DNA replication, <br /> haploid cells G1 phase particularly sensitive killing           DSB repair restored. RAD52-independent DSB repair <br />                                                                               tandemly repeated sequences thought result from <br /> by ionizing radiation, indicating DSBs repaired a <br /> DNA homology-dependent mechanism. Mutant strains defec-                       extensive single-stranded exonucleolytic DNA diges- <br /> tive recombinational DNA repair processes, notably                    tion likely causes lethal lesions DSBs introduced <br /> those mediated RAD52 epistasis group, reduced                   unique DNA sequences. Single-stranded DNA degra- <br /> threshold level DSB tolerance (26, 49). Mutations                 dation adjacent repeats provide complementary <br /> genes confer sensitivity killing physical chemical                  single-strand DNA chains heteroduplex formation and <br /> agents generate DSBs, ionizing radiation                     restoration chromosome continuity (14, 43, 56). Since rad52 <br /> methyl methanesulfonate (MMS). That collection genes                  mutant cells proficient repair damage among <br /> also affect genetic recombination interpreted                     multiply repeated elements, likely DSBs not <br /> indicate role mitotic recombination repair DSBs               intrinsically responsible cell death, DNA breaks <br /> (15, 23, 36, 48).                                                             lethally processed absence redundant intrachro- <br />    DSBs inhibit cell division cycle unless repaired                   mosomal homology. <br /> represent lethal lesions. A single DSB generated enzymatically                   These observations led consider repair processes <br /> by HO endonuclease MA4T (40, 59) HO cut site                   identified mutations generate lethal <br /> introduced genome (42, 53) causes death                   lesions following introduction genomic DSBs. Without <br /> rad52 cells. Estimates physical analysis genomic DNA                  regard particular recombinational DNA repair model, we <br /> from X-irradiated yeast cells indicated approximately                interested isolating mutants affected HO-induced <br /> DSB responsible lethality radS2 mutants (27, 49).                  DSB recombination directly repeated genes without <br /> Unrepaired DSBs lethal strains                  grossly altering level spontaneous recombination. Al- <br /> lethal trans, zygotes formed X-irradiated and <br /> untreated rad52 rad54 mutant strains inviable (27).                    spontaneous mitotic recombination thought result <br /> Extensive exonucleolytic DNA degradation occurs                      repair DNA lesions incurred vegetative <br /> site HO cut rad52 mutant strains (61). In contrast,                   growth (44), repair necessarily proceed or <br /> HO-generated DSBs stable hours cell                      result DSB. We reasoned requiring basal-level <br />                                                                               background recombination proficiency mutants <br />                                                                               uncover set reactions unique processing and <br />   * <br />     Corresponding author. Mailing address: ICOS Corporation, 22021            repair DSBs. We report DSB-stimulated <br /> 20th Ave. SE, Bothell, WA 98021.                                              recombination mutants capacity repair a <br />                                                                        8037 <br />  8038      HALBROOK AND HOEKSTRA                                                                                      MOL. CELL. BIOL. <br />  <br /> HO-induced DSB duplicated ura3 gene                    Strains. K1864 (AL4 Ta his4-260 ade2-1 ade5 ura3-52 leu2- <br /> stimulating frequency Ura&apos; recombination.                     3,112 trp1-HIII canl 1ys2-ABX::CAN1 LYS2) gift R. <br />                                                                      Keil (38). The ura3-52 allele replaced (52) ura3 gene <br />                METHODS AND MATERIALS                                 containing NotI recognition site StuI (ura3-SNot). This <br />                                                                      strain transformed uracil prototrophy linearized <br />    <span id='am-115' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-116' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span>. The pREGIA-1 disruption plasmid donated             hisG-URA3-hisG (2) disrupted allele REG1 gene. Ura+ <br /> by N. Dhillon. Briefly, REG1 gene (provided M.                transformant colonies streaked media containing <br /> Johnston) inserted ClaI site pBluescript II        5-fluoro-orotic acid (5); 5-fluoro-orotic acid-resistant papillae <br /> (Stratagene, La Jolla, Calif.) derivative lacked BamHI        tested uracil auxotrophy, deletion REG1 <br /> site. Digestion BamHI released entire REG1 coding           gene confirmed DNA hybridization. Strain 543 (Table <br /> sequence, replaced BglII-BamHI fragment             1) constructed targeted integration pALUM the <br /> from pNKY51 (2) containing URA3 gene flanked                  ura3-SNot allele. This yielded ura3 duplication, ura3-SNot:: <br /> <span id='am-83' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span>Salmonella</span> hisG DNA.                                                 ADE5 LEU2 ura3-HOcs (Fig. 1). <br />    The ura3-HOcs plasmid, pALUM, constructed insert-             Strain 543 transformed histidine prototrophy by <br /> ing ura3-CIR HindIII fragment YIplacl28 (17). This ura3       targeted integration (52) pHGH12 generate partial <br /> allele ClaI site changed EcoRI site (42). An           duplication his4 locus, his4-260A::GAL1,10-HO HIS4. <br /> approximately 5-kb ADE5 fragment pL1067 (provided            The structure locus strain 619 confirmed DNA <br /> R. Keil) introduced polylinker              hybridization. <br /> YIplacl28 backbone create pALU. A 117-bp HO site                Targeted integration YIpCDCl strains 622 and <br /> MATa (34) flanked EcoRI linkers inserted yield             601-7D (Table 1) carried transforming plasmid <br /> pALUM. Integration pALUM targeted ura3                 linearized unique XhoI site (52). The resulting trypto- <br /> locus digestion NsiI.                                        phan prototrophic strains, 619-1 (CDCl::CDCl TRP1) and <br />    Plasmid pRSgalHO (TRP1) contains 3.7-kb EcoRI-SalI              601-7D (cdcl-100::CDC1 TRPJ), confirmed Southern <br /> GAL1,10 promoter-HO endonuclease DNA fragment                   analysis. Each strain showed identical shift 7 kb of <br /> pGALHO (31) pRS314 (54).                                          unique CDC1 DNA band analyzed BglII or <br />    The GAL1,10:HO integration plasmids, pHGH12                   ClaI restriction enzyme digestion genomic DNA. <br /> pHGH15, contain 1.6-kb Sall fragment 470 bp 5&apos;            Mutant genes introduced strain 619 genetic <br /> HIS4 coding sequence pR82 (gift R. Keil) SalI           background backcrossing mutant segregants 619 its <br /> and XhoI sites pBluescript 11-SK (Stratagene). The resulting      derivatives times. The initial cdcl-100 mutation <br /> plasmid digested EcoRI EcoRV ligated           backcrossed (seven times) strain 619. The <br /> the 2.5-kb EcoRI-EcoRV GAL1,10:HO fragment                      rad52A::TRPJ mutation strain W838-1OB gift R. <br /> pGALHO (31). Plasmid pHGH12 GAL1,10:HO his4A                 Rothstein. Strains 369 (cdcl-1) 342 (cdcl-2) (Table 1) <br /> oriented transcribed opposite directions,        provided L. Hartwell. <br /> whereas oriented transcribed           Yeast culture conditions. Yeast methods media were <br /> same direction pHGH15. Targeted integration of          essentially described Rose et al. (51). YEPD 1% <br /> these plasmids strain 543 following linearization      (wt/vol) yeast extract, 2% (wt/vol) Bacto Peptone, 2% <br /> XhoI resulted truncated 5&apos; his4 allele contained      (wt/vol) glucose. YEPD(0.5) contained 0.5% (wt/vol) glucose. <br /> the his4-260 mutation HIS4 gene flanking                   YEP lacks carbon source. YEPGal supplemented with <br /> GAL1,10:HO gene. HO expression affected               2% galactose instead glucose. YEPDGal contained both <br /> relative orientation HIS4 GAL1,10:HO genes.                   glucose galactose 1% (wt/vol) each. SC synthetic <br />    Plasmid YIpCDC1 inserting 3.8-kb BglII              complete medium (2% glucose), SC-H lacked histidine, <br /> fragment containing CDC1 BglII site YIplac2O4            SC-LH lacked histidine leucine, SC-U lacked uracil, SC-T <br /> (17).                                                                lacked tryptophan, SC-LT lacked leucine tryptophan. <br />  <br />                                            TABLE 1. S. cerevisiae strains used study <br />  Strain                                                                                        Genotype <br /> 543                        ..M.                        LYS2::CAN1 lys2-ABX reglA <br />                                                          ATa ho ura3-SNot:ADE5 LEU2 ura3-HOcs <br />            his4-260 ade2-1 ade5 can] leu2-3,112 trpl-HIII <br /> 619   ...   4Tat ho ura3-SNot::ADE5 LEU2 ura3-HOcs LYS2::CAN1 lys2-ABX reglA <br />            his4A-260::GALJ 10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII <br /> 622 ...M4Ta ho ura3-SNot::ADE5 LEU2 ura3-HOcsslys2-ABX reglA his4A-260::GALl <br />            10-HO ade2-1 adeS canl leu2-3,112 <br /> 601-1B    ATa ho ura3-SNot::ADE5 LEU2 ura3-HOcs LYS2::CANJ lys2-ABX reglA <br />            his4A-260::GAL1 10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII cdcl-100 <br /> 601-7D    MAT <br />           ....      ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA his4A-260::GALJ <br />                                                                    o <br />  <br />            10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-100 <br /> 295-4A     4Ta ho ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA his4A-260::GALJ <br />                                                        A. <br />  <br />  <br />  <br />            10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII radS2::TRPJ <br /> 648-lA                                                  MATa ho ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA      his4A-260::GALl <br />                                                           10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-1 <br /> 631-4A    ...                                                           his4A-260::GALl <br />                                                         MATa ho ura3-SNot::ADE5 LEU2 ura3-HOcs lys2-ABX regl A <br />              10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-2 <br /> 369 ....MATa adel ade2 ural his7 tyrl lys2 gall cdcl-l <br /> 342        MATa adel ade2        his7 tyrl lys2 gall cdc]-2              ural <br />  <br /> 488   .....4..a <br />             o .       -H <br />               leu2-3.112   t          rl                  II I rA canl ura3-SNot ade2-1 ade5 <br />            MAToa leu2-3,112 his4-260 lys2-ABX ho trpl-HIII reglA canl ura3-SNot ade2-1 adeS <br />  VOL. 14, 1994                                                               CDCI GENE IN S. CEREVISIAE   8039 <br />  <br />  <br />  <br />  <br />                                          X <br />                            CS <br />                 QO <br />                  CD <br />                 c. <br />                        CD            - <br />  <br />                                                    D <br />                            0.   n                                                        Do <br />                                 CN <br />  <br />  <br />  <br />  <br />                                               Lo <br />  <br />                                               0 <br />  <br />  <br />  <br />  <br />                        CD       f-~~~~D <br />  <br />                                                   0.                                     It <br />                                               mA                   0 <br />  <br />  <br />                                               DI                                   CD <br />                                                                                     y <br />  <br />                                                                X <br />                                                    CoI <br />                                                    CD                                    Xf, <br />  <br />                                                          N              0 <br />  <br />  <br />  <br />  <br />                                                                    I- <br />                                                                    SI                         0 <br />  <br />  <br />                                                                    0u <br />  <br />                                                          oft <br />                                                          ro <br />                                          c <br />  <br />  <br />  <br />                                                                Ut1 <br />                                                                0b <br />                                          &apos;a <br />                                                                                      4., o <br />                                                                aD <br />  <br />  <br />  <br />  <br />                                                                                          Qj <br />                                 N <br />  <br />  <br />  <br />  <br />                       C: <br />  <br />  <br />  <br />  <br />                                                                                      N <br />  <br />  <br />  <br />  <br />                        w <br />  <br />  <br />  <br />                            0a <br />  8040    HALBROOK AND HOEKSTRA                                                                                    MOL. CELL. BiOL. <br />  <br /> The suffix DG indicates glucose galactose (1% each)      19% efficiency produced 5% four-spored asci. The <br /> were supplied (e.g., SC-HDG).                                     132 dissected tetrads incubated 240C 3 days. Plates <br />    Sporulation medium 1% potassium acetate, 0.1% Bacto        replicated SC medium (incubated 24 370C) and <br /> Yeast Extract, 0.05% glucose, 1.5% agar complete       SC-T medium (incubated 240C). Viability dissected <br /> supplement amino acids (51).                                   tetrads 88%. All 119 viable four-spored tetrads exhibited <br />    Solid medium containing methyl methanesulfonate (MMS;          parental ditype segregation pattern, confirming identity <br /> Kodak Chemical Co., Rochester, N.Y.) prepared         cloned (cdcl-100-suppressing) gene CDC1. <br /> than 24 h advance use. MMS (100%) added           Recombination assays. SC-LH medium containing 0.5% <br /> YEPD medium allowed cool prior pouring             glucose (5 ml) inoculated single colony incu- <br /> plates.                                                           bated overnight vigorous aeration. Cells diluted into <br />    All incubations carried 30&apos;C noted.    5 ml YEPD(0.5) 106 cells ml incubated for <br /> cdcl-1 cdcl-2 mutant cells incubated 240C (per-       2 4 h. HO endonuclease induced addition 20% <br /> missive temperature).                                             galactose 2% final concentration. This yields final ratio <br />    Mutant isolation. A single colony strain 543 (pRSGAL-       galactose glucose 4 1. The degree induction from <br /> HO) inoculated 5 ml SC-LT medium grown            GAL1,10 promoter regl strain dependent both <br /> overnight 30&apos;C. This culture diluted 10 ml fresh   absolute galactose concentration galactose-to- <br /> SC-LT medium grown hours concentration        glucose ratio (18, 29). Aliquots removed after <br /> 1 X 106 2 X 106 cells ml addition 200 ,ul     galactose addition plated YEPD SC medium to <br /> ethyl methanesulfonate (100%; Kodak). The culture di-         determine viable cell concentration SC-U medium to <br /> vided equal portions incubated additional    select Ura&apos; recombinants. Recombination frequencies were <br /> 90 min 30&apos;C. Sodium thiosulfate (50%) added 5%        calculated ratio Ura&apos; colonies viable cells plated. <br /> final concentration, cells pool plated    Red, white, half-sectored red/white colonies scored <br /> SC-LT medium density 1,000 3,000 cells plate   SC (unselected) SC-U (selected) media. <br /> incubated 30&apos;C. Under conditions, viability typi-       Recombination measured serially replica plating <br /> cally reduced approximately 10%, resulting 100 300       patches master plate ultimately SC-U medium <br /> colonies plate.                                               growth SC-H SC-HDG media (for strains derived <br />    Colonies mutagenized cells replicated HO-         619). The number Ura&apos; papillae formed uracil <br /> repressing medium (SC-T) HO-inducing medium (SC-           omission medium pregrown glucose plus galactose relative <br /> TDG), incubated overnight 30&apos;C, replicated SC-U.        pregrown glucose indication the <br /> With dissection microscope, differences frequency     level HO-stimulated recombination. <br /> Ura&apos; recombination estimated comparing level          Mating-type switching assayed filtering 106 log-phase <br /> Ura&apos; papillae derived colonies previously grown SC-T      cells (prepared single colonies described above) onto <br /> and SC-TDG. Potential isolates, demonstrating HO-stimu-        nitrocellulose discs. Filters washed YEP me- <br /> lated Ura&apos; recombination, rescreened eliminate false-     dium, placed YEPD YPGal medium, incu- <br /> positive, petite, galactose utilization-defective strains.    bated 300C. Filters removed plates the <br />    CDC1 gene cloning molecular analysis. As recom-        indicated times, suspended YEP medium, serially diluted, <br /> bination defect cdcl-100 mutation recessive,        plated SC medium 30&#176;C. After 2 3 days, colonies <br /> sought identify molecular DNA clone genetic comple-       replica plated mating-type tester lawns scored <br /> mentation (50). Strain 601-7D (cdcl-100) transformed          2 days. For ALTa strains, mating-type interconversion <br /> (13) yeast genomic DNA centromere library (ATCC            expressed (number AL4To colonies + number of <br /> 77164 [provided P. Hieter]). Tryptophan prototrophs       nonmating colonies)/total number colonies. The back- <br /> isolated screened HO-stimulated recombination pro-        ground frequency ALT switching cells approxi- <br /> ficiency. From 900 colonies screened, recombination-        mately 1 2%. This high level presumably leaky <br /> proficient transformants obtained. Structural analysis    expression HO endonuclease. <br /> plasmids isolated strains indicated repre-      DNA damage induction. Single colonies used inoc- <br /> sented genomic segment. TnJOLUK mutagenesis              ulate 5 10 ml SC-LH medium grown overnight. <br /> (30) yielded transposon insertions near pair closely     Cultures diluted 1 X 106 2 X 106 cells ml into <br /> juxtaposed BamHI sites eliminated HO-induced recombi-        YEPD(0.5) grown 2 3 h. For radiation-induced <br /> nation. This region complementing 3.5-kb BglII frag-     damage, cells counted serially diluted, 5 ,ul was <br /> ment subcloned pRS200 (ATCC 77165 [54]) revealed           placed YEPD medium. Aliquots 10-fold serial <br /> single open reading frame 1,473 nucleotides (491 amino         dilutions contained average 3 X 104 30 cells per <br /> acids). This plasmid designated p220. Deletion 390-bp    spot. Plates immediately irradiated -y UV <br /> BamHI fragment resulted in-frame deletion 130 amino      light, using Mark I-68A 137Cs _y irradiator (J. L. <br /> acid residues (p221). Insertion 3.8-kb BglII-BamHI frag-     Shepard &amp; Associates, San Fernando, Calif.) used 414 <br /> ment containing URA3 gene flanked <span id='am-60' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span>Escherichia coli</span>         krads/min 15-W germicidal lamp (General Electric) a <br /> hisG repeats pNKY51 (2) remaining BamHI site        flux 3.3 J/m2/s. Irradiated cells incubated 30&#176;C and <br />  p221 yielded deletion/insertion mutation CDC1            scored 16 48 h. <br />  (cdc1::URA3) p222.                                               Treatment MMS carried adding drug <br />    The cdcl::URA3 allele used replace       directly log-phase cultures prepared irradiation <br /> wild-type copies CDC1 diploid strain 488 transforming    experiments. Cultures grown aeration 30&#176;C 1 <br /> with p222 linearized digestion BglII NotI. A          h. Sodium thiosulfate (50%) added 5% final concen- <br />  candidate uracil prototroph sporulated, tetrads     tration, cells serially diluted placed YEPD <br />  dissected.                                                       medium described above. Cell growth scored 16 to <br />     To demonstrate cloned gene fact CDC1,         48 h. <br />  strain 619-1 mated strain 648-1A, diploids          Cell viability continuous exposure MMS was <br />  sporulated 5 days 30&#176;C. The diploid strain sporulated     determined transferring serially diluted cells log-phase <br />  VOL. 14, 1994                                                                               CDCJ GENE IN S. CEREVISAE             8041 <br />  <br /> cultures medium containing MMS. Cells incubated               measure recombination. Viability determined micro- <br /> 30&apos;C scored 16 48 h.                                    scopic analysis plated cells. Percent viability number <br />    Cell viability function level HO expression     microcolonies divided total number cells plated. <br /> assayed transferring serially diluted log-phase cells         Ura&apos; recombination frequency number uracil <br /> YEPD, YEPDG, YEPGal media. Cells incubated 30&apos;C                prototrophs divided total number cells plated. This <br /> were scored 16 48 h.                                         experiment carried times. <br />    Because cdcl-100 strains grow slowly solid media,              Nucleotide sequence accession number. The nucleotide se- <br /> examined cells microscopically scored viable cell growth       quence data reported paper appear EMBL, <br /> terms microcolony formation. Viable microcolonies             GenBank, DDBJ nucleotide sequence databases the <br /> usually characterized uniform radial growth producing              accession number X81813. <br /> round colonies composed normally <br /> shaped (round) cells. Inviable terminal microcolonies                                      RESULTS <br /> characterized jagged unevenly edged colonies composed of <br /> 2 20 aberrantly shaped, enlarged elongated cells.          A DSB-stimulated genetic recombination assay. In design- <br /> These colonies, composed cells capable          ing genetic recombination assay, goal create a <br /> limited number cell divisions, did support continued           allow identify characterize mitotic <br /> growth did increase size time. Viable cell            recombination-defective mutants. Two primary elements of <br /> frequencies calculated number viable microcolo-        assay (i) regulated initiation recombination <br /> nies divided total number cells microcolonies           events introduction specific DSB (ii) a <br /> counted.                                                              duplicated chromosomal gene DSB targeted. <br />    This method reproducible cdcl-100             A nontandem duplication ura3 locus chromosome V <br /> cells examined media containing MMS galactose           generated integration plasmid containing LEU2, <br /> the sole carbon source. Colony growth rate                ADES, ura3 gene resides 117-bp HO <br /> conditions qualitatively reduced relative growth            endonuclease recognition sequence (Fig. 1). Recombination <br /> YEPD YEPDGal media. This combina-                 events occurred duplicated ura3 genes were <br /> tion effects including expanded cell division cycle time        inferred colonies papillae formed uracil <br /> and/or increased rate killing. cdel-100 cells grown       omission medium. The parental yeast strain (Table 1) ade2 <br /> these conditions scored 2 days 1 day.          adeS mutations formed white colonies. With ADES <br /> This perturbation cell growth partly reflected          provided ura3 duplication, adenine biosynthetic <br /> large standard deviation independent determina-           pathway blocked ade2 step, cells accumulate red <br /> tions. Contributing variation microcolony formation         metabolic intermediate. Colony color used distinguish <br /> on YEPGal MMS media difficult                   recombination events resulted deletion of <br /> distinction viable nonviable microcolonies. Be-           DNA flanked ura3 alleles (Fig. 1) gene conversion <br /> cause colonies combinations cells normal            events associated deletions (Fig. 1). The former <br /> and abnormal morphology, subjective decision            yielded white Ura- (Leu-) Ura&apos; (Leu-) recombinants, <br /> the basis relative number cell types             yielded red Ura&apos; colonies. Ura&apos; recombi- <br /> microcolony. Despite obstacle describing cell growth       nants arise unequal sister chromatid exchange <br /> these media, clear cdcl-100 cells capable         (SCE) events. <br /> completing repeated cell division cycles conditions          Several potential recombinant products obtained by <br /> completely inhibited radS2A cell growth (see Tables 3 4).         repair reactions sister chromatids. Two examples are <br />    Recombination rate determinations. Mitotic recombination           represented Fig. 1: unaltered ura3 locus generated an <br /> frequencies determined fluctuation analysis. YEPD             equal SCE event triplicated ura3 gene resulting from <br /> medium (3 ml) inoculated approximately 50 cells,             gene conversion associated crossover mis- <br /> which grown 30&apos;C late log phase (about 45 h). Cells        aligned sister chromatids. Because S(NotI) the <br /> were plated SC medium determine viable cell number,             RV(HO) mutations converted unequal SCE <br /> on SC-U medium determine uracil prototrophy,                event generate functional URA3 gene (with ura3 DNA <br /> canavanine medium measure deletion formation lys2           configuration illustrated Fig. 1), expected <span id='am-49' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of <br /> locus. Colonies scored 2 days strain 622 4         recombinants small. Alternatively, allele need <br /> days strain 601-1B. The method median (37)             converted unequal SCE reaction associated with <br /> used analyze data seven independent cultures         reciprocal exchange generate URA3 gene. <br /> determination. Recombination rates calculated divid-             Intrachromosomal recombination repeated genomic <br /> ing frequency ln n - ln (62).                               DNA sequences occurs rates <span id='am-57' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> 10&apos; 10-6 <br />    DSB induction a-factor-arrested cells. To log-phase             (1, 56, 62). Introduction HO cut site gene <br /> cultures (2 ml 1 X 106 2 x 106 cells ml) grown           duplication stimulates recombination hundred-fold <br /> YEPD(0.5) added a-factor (Sigma Chemical Co., St.                 HO endonuclease expressed (42). To effec- <br /> Louis, Mo.) 5 ,uM final concentration incubated           tively use GALl,10 promoter-regulated HO means for <br /> 30&apos;C. After 90 min, 10 [LI 1 mM stock solution        stimulating double-strand DNA cleavage, mutation the <br /> a-factor added incubation continued. After 3 h             galactose regulatory network utilized (29, 41). Deletion of <br /> medium containing a-factor, 95 90% wild-             REG1 gene mitigated effects catabolite repression, <br /> type cdcl-100 cells, respectively, unbudded. Cultures        allowing galactose serve gratuitous inducer HO while <br /> were divided half, culture strain         strains remained growing glucose. This strategy provided a <br /> added galactose (20%) 2% final concentration (galactose          rapid efficient HO induction shifting carbon <br /> to glucose ratio, 4 1) final hour a-factor treatment.   sources. Furthermore, previous mutant screens designed to <br /> Cells incubated galactose washed             identify mutations viable faced with <br /> remove a-factor galactose serially diluted, 5-,ul         persistent expression GALJ,10::HO (28), that <br /> aliquots plated SC score viability SC-U             predominant class mutants Gal-. The reglA <br />  8042           HALBROOK AND HOEKSTRA                                                                                           MOL. CELL. BIOL. <br />  <br />        TABLE 2. Spontaneous DSB-induced recombination                       bination wild-type cells differences between <br />            frequencies wild-type cdcl-100 strains                       mutant wild-type colonies distinguished. <br />                                    Recombinants/viable cell (10-4)a                Ethyl methanesulfonate-mutagenized strains 543 619 <br />       Strain     Galac-                                                         screened colonies exhibited reduced levels of <br />                   tose      Ura+          Ura+         Ura+ white      Ura-     Ura&apos; papillae overnight growth media containing <br />                                            red                         white <br />                                                                                 glucose galactose. A detailed analysis approach and <br /> 622                -       7.1 &#177; 2.8    3.8 &#177; 1.8      3.2 &#177; 0.9       &lt;2.5&quot;    mutants isolated described elsewhere <br />   (wild type)                                                                   (18). Briefly, anticipated broad classes mutations <br />                    +      1,200 &#177; 280 360 &#177; 180        930 &#177; 200 250 &#177; 210      screen: strains (i) showed reduced viability on <br />                                                                                 galactose presumably affect reactions essential the <br /> 601-1B             -        38 &#177; 11     33 &#177; 13        4.8 &#177; 1.3       &lt;3.8b    repair DNA DSBs, (ii) exhibited reduced spontaneous and <br />   (cdcl-100)                                                                    DSB-stimulated recombination represent mutations <br />                    +        56&#177; 13      49&#177; 16         6.6&#177;3.9        18&#177;8.7    affect wide range DNA metabolic reactions that <br />   Data averages &#177; standard deviations independent experiments.   influence recombination DSB repair and, (iii) demon- <br />   b No white colonies observed 4,000 (&lt;2.5) 2,600 (&lt;3.8)         strated spontaneous Ura&apos; recombination proficiency were <br /> unselected colonies.                                                            defective DSB-stimulated recombination. Because the <br />                                                                                 rad52A mutant showed spontaneous HO-stimulated <br />                                                                                 Ura+ recombination reduced viability, mutants might <br /> mutation reduced number Gal- false-positive results                      appear and/or second categories. We specifically <br /> from analysis.                                                             focused mutations class potentially <br />    Addition galactose media containing glucose stimu-                     represent mutations play role strand break repair that <br /> lated frequency Ura+ recombination 100-fold                         reported previously S. cerevisiae. <br /> (Table 2). The frequency white colonies increased                        Examination 90,000 colonies formed mutagenized cells <br /> 500-fold following HO induction. Among spontaneous Ura+                         yielded 300 mutants. Preliminary analysis collec- <br /> recombinant colonies, 50% white,                       tion suggests complementation groups exist (18). <br /> 75% HO-induced Ura+ recombinants white. These                           Partial characterization mutant strains indicated that <br /> values agree reported Nickoloff et al.,                       mutant clearly satisfied criteria imparting a <br /> reported use similar ura3 alleles flanked HIS3                    DSB recombination defect studied further. As shown <br /> marker (42). The stimulation Ura+ recombination fre-                         below, mutation identified allele the <br /> quency reglA strains directly proportional galac-                 previously described CDC1 gene, refer mutant <br /> tose concentration ratio galactose glucose (data                  allele cdcl-100. <br /> not shown) (29, 41).                                                               The cdcl mutant defective recombinational repair <br />    Identification recombination-defective mutants. Because                   DSBs. The recombination defect conferred cdcl-100 <br /> DSBs recombinogenic lethal, depending                      mutation backcrossed wild-type strains 619 622 <br /> repair capacity particular strain examined,                     reexamined using regimen used initial <br /> were uncertain extent DSB recombination-defective                       identification (Fig. 2A). Strain 601-7D (cdcl-100) exhibited no <br /> mutants sensitive HO-induced killing. To deter-                     increase Ura&apos; papillae following galactose treatment, <br /> mine differential levels recombination proficiency                   wild-type strain 622 showed increased frequency of <br /> inhibiting cell division potentially broad spectrum DSB                 Ura&apos; papillae identical conditions. Both spontane- <br /> repair-defective mutants, simultaneously minimized                       ous galactose-induced recombination frequencies strain <br /> killing impact DSBs maximized recombinogenic                       601-7D appear decreased; however, result a <br /> potential attenuated expression HO galactose                     reduced growth rate strain solid media (Fig. 2B). <br /> promoter. Since DSBs lethal rad52-1 mutants (15, 40,                        The cdcl-100 mutation initially isolated strain 543, <br /> 59), examined plating efficiency GAL::HO radS2A                     harbored GALJ,10::HO gene pRS314. The pos- <br /> (Table 3) mutant strain media containing galactose. As                       sibility mutations HO endonuclease responsi- <br /> expected, radS2A strain inviable grown                          ble galactose-uninducible recombination phenotype <br /> YEPGal medium arrested large-budded cells (18)                      eliminated ways. First, mutant derivative of <br /> result inability repair HO-induced DSBs ura3                       strain 543 cured pRS314 (GALl,10::HO), retrans- <br /> and/or MA4T (40, 59). On YEPDGal, rad52A mutant strain                      formed, retain cdcl-100 phenotype. Second, <br /> exhibited 70% viability (see Table 3) produced                    cdcl-100 mutation, introduced genetic crosses into <br /> no Ura+ recombinants. In contrast, wild-type strains 543                    strain 619 possessed integrated copy GALJ,10::HO, <br /> 619 showed 100% viability increased Ura+ recombina-                      retained DSB-stimulated recombination-defective <br /> tion frequency incubated YEPDGal (see below).                           phenotype. Finally, HO endonuclease activity assayed by <br /> Therefore, rad52A mutation served control viability                measuring DSB formation duplicated ura3 locus. Mo- <br /> when challenged HO-induced DSBs assay system.                       lecular analysis genomic DNA isolated cdcl-100 <br /> The plating efficiency rad52A mutant strain YEPD-                     mutant cells revealed galactose-dependent DSB was <br /> Gal suggests 30% cells experienced DSB                         ura3 duplication (18). <br /> completing cell division. This finding consistent                    To determine frequency Ura+ recombinants <br /> reduced growth rate rad52A colonies YEPDGal (data                         arise cdcl-100 strain, cells grown liquid culture were <br /> not shown). These results suggest GALJ,10 promoter                     assayed. In absence galactose, number Ura+ <br /> is partially active reglA cells grown                      recombinants viable cell cdcl-100 cultures about <br /> presence glucose galactose fraction                    fivefold greater wild-type cultures (Table 2). However, <br /> cells population experience HO-induced DSB. Since                    wild-type cells exhibited 170-fold increase Ura+ recombi- <br /> continual generation DSBs likely growth inhibitory                  nants following galactose treatment. In contrast, HO-stimu- <br /> to recombinational repair mutants, chose investigate                      lated Ura+ recombination eliminated cdcl-100 <br /> attenuated GALJ,10 promoter activity, stimulated recom-                   mutation, Ura+ recombination frequency following <br />  VOL. 14, 1994                                                                                      CDC1 GENE IN S. CEREVISIAE           8043 <br />  <br />                   A.                                                    B. <br />  <br />                                   Spontaneous Induced                                       24tC       37tC <br />  <br />                            622    -                                              622- <br />                                                         p200                                                   p200 <br />                        cdcl-100                         p220(CDCl)           cdcl-100                          p220(CDC1) <br />                                                         p221 (Cdc1A)                                           p221 (cdclA) <br />  <br />                            622    -                                              622    - <br />                                                          p200                                                  p200 <br />                          cdcl-1 [                        p220(CDCI)            cdcl-1   [                      p220(CDCl) <br />                                                          p221(cdclA)                                           p221 (cdc 1A) <br />  <br />  <br />                             622   -                                              622    - <br />                                                          p200                                                  p200 <br />                          cdcl-2 [                        p220(CDC1)           cdc 1-2 [                        p220(CDC1) <br />                                                          p221 (cdclA)                                          p221 (cdclf) <br />  <br />    FIG. 2. (A) HO-induced Ura+ recombination assayed solid media. Cells patched SC-LH medium replica plated SC-H <br /> (2% glucose) SC-HDG (1% glucose galactose) media grown overnight. Cells replicated uracil omission medium, <br /> and Ura+ papillae scored photographed following overnight growth. Cells panels (cdcl-100) incubated 2 days <br /> to better demonstrate partial complementation p221 cdcl-100 recombination defect. (B) Temperature-dependent cell growth. Cells <br /> patched SC-LH medium replica plated SC medium incubated 24&#176;C (permissive cdcl) 37&#176;C (nonpermissive for <br /> cdcl). Strains: 622, wild type; 601-7D, cdcl-100; 648-1A, cdcl-1; 631-4A, cdcl-2. Plasmids: p200, control plasmid, TRPI; p220, BglII fragment <br /> containing CDC1; p221, BglII fragment BamHI fragment deleted, removing 130 internal amino acids Cdcl. <br />  <br />  <br /> galactose treatment (5.6 X 10-&apos;) significantly different         measured. Wild-type cells exhibited hyperbolic kinetics, <br /> from uninduced value.                                                reaching maximum 50% mating-type interconversion <br />    The spontaneous Ura+ recombination rates wild-type                1 h galactose treatment (Fig. 3B). In contrast, the <br /> and cdcl-100 strains roughly equivalent,             cdcl-100 mutation adversely affected rate mating-type <br /> 5.8 X 10-6 1.5 X 10-5 Ura+ cells viable cell                 switching. The number switched cells increased linearly <br /> generation, respectively. This represents twofold increase             5 h continuous incubation galactose reached <br /> relative wild-type value. The spontaneous rate                 level 15%. Although mating-type interconversion was <br /> recombination nontandem repeated alleles LYS2                  inhibited cdcl-100 mutation, eliminated, as <br /> gene (lacking HO cut site) flanking CANM inferred               switching increased approximately sevenfold. That MAT <br /> identification canavanine-resistant colonies (38). The rates          switching, involves gene conversion associated with <br /> of canavanine resistance viable cell generation              deletion formation (25), permissive cdcl-100 mutant <br /> wild-type cdcl-100 strains 6.8 X 10-6 2.1 X 10-6,           background consistent data presented Table 2 <br /> respectively. This represents threefold decrease relative           showing gene conversion events ura3 locus occurred <br /> the wild-type value. These data indicate cdcl-100               frequently deletion reactions. <br /> mutation did adversely affect spontaneous recombination                 Recombination measured following treatment with <br /> between repeated genes, contrast dramatic affect            MMS determine nonenzymatically generated DSBs <br /> DSB-initiated recombination.                                             induced recombination differentially CDC1 cdcl-100 <br />    To examine cdcl-100 mutation affected              cells. The frequency Ura+ recombination wild-type <br /> the rate extent Ura+ recombination, kinetics            cultures induced approximately 20-fold following 40-min <br /> HO-induced recombination measured (Fig. 3). Wild-type               treatment 0.01% MMS. Similar treatment cdcl-100 <br /> cultures displayed rapid, hyperbolic response galac-           cells failed induce Ura+ recombination (data shown). <br /> tose treatment, reaching nearly 70% potential maximum            This finding indicates recombinational repair defect of <br /> within 15 min. Since HO-generated DSB repair occurs 1              cdcl-100 mutation specific HO-generated DSBs <br /> to 2 h (10, 42, 53, 61), recombinogenic DSBs           extends notion cdcl-100 primarily affects non- <br /> brief galactose treatment likely repaired plating.           spontaneous recombination. <br /> cdcl-100 cultures showed negligible increase Ura+ cells               The cdcl-100 mutation affected quality HO- <br /> after 2 h. The cdcl-100 mutation, therefore, defective           stimulated recombinants cleavage site recombination <br /> somehow blocked events normally occur re-               substrate. Among wild-type cells grown absence of <br /> combinational repair HO-induced DSBs              galactose, roughly equal numbers Ura+ recombinants were <br /> rate extent Ura+ recombination affected.                white red. The level white Ura+ recombinants increased <br />   To determine CDC1 function specific HO-gener-               times level red Ura+ recombinants when <br /> ated breaks ura3, HO-induced mating-type interconversion             cells grown liquid medium containing glucose and <br />  8044      HALBROOK AND HOEKSTRA                                                                                            MOL. CELL. BIOL. <br />  <br />  A                                                                         repaired SCE relative homologous recombination. Chro- <br />                                                                            mosomal repair equal SCE mechanism one <br />                                                                            chromatids received DSB recombina- <br />                                                                            tionally silent assay (Fig. 1). However, gene conversion <br />                                                                            associated reciprocal exchange misaligned <br />                                                                            sister chromatids result triplication ura3 locus <br />          0 <br />  <br />  <br />          CD                                                                URA3 gene restored (Fig. 1). Analysis the <br />                                                                            URA3 locus Southern blotting 20 selected 20 <br />                                                                            unselected red Ura&apos; CDC1 cdcl-100 recombinants, both <br />               D <br />                                                                            HO induced uninduced, failed identify triplication of <br />                      20 <br />                                                                            ura3 gene indicative unequal SCE (data shown). <br />                                                                            Therefore, SCE occurred 5% duplication. <br />                                                                               Evidence unequal SCE sought scoring <br />                                                                            red/white half-sectored colonies. Half-sectored red/white col- <br />                                         Time (min) <br />                                                                            onies generated G2 unequal SCE through <br />                                                                            intrachromosomal recombination reactions occur inde- <br />                                                                            pendently homolog. The frequencies half-sectored <br />                                                                            colonies wild-type cdcl-100 strains following galac- <br />                                                                            tose treatment approximately 2 0.5%, <br />                                                                            respectively. These data indicate unequal SCE ac- <br /> B                                                                          count minority recombinants, assay <br />                     100                                                    conditions, DNA exchange properly aligned sister <br />                                                                            chromatids responsible efficient recombinationally <br />                                                                            silent repair. <br />                                                                               To address issue equal SCE, examined the <br />               C) <br />                                                                            behavior (x-factor-arrested cdcl-100 mutant cells when <br />               C) <br />                                                                            challenged galactose. In experiments, wild-type <br />               CD <br />                                                                            ao-factor-arrested cells maintained greater 95% viability in <br />               01)                                                          absence galactose. In presence galactose during <br />               0-                                                           (x-factor-induced arrest, viability varied 64 90%. cdcl- <br />                                                                            100 cells somewhat sensitive a(-factor arrest, exhibiting <br />                                                                            57 88% viability. However, presence at-factor <br />                                                                            galactose, viability remained high (66 88%) cdcl-100 <br />                           0   1     2        3       4      5      6       cultures. Under conditions, wild-type Ura&apos; recombina- <br />                                         Time (hrs)                         tion induced average 75-fold. Recombination the <br />                                                                            cdcl-100 strain uninduced galactose treatment and <br />    FIG. 3. Kinetics HO-induced recombinant formation wild-           remained threefold lower wild-type cells, at <br /> type cdcl-100 strains. Galactose (2% final concentration) was <br /> added log-phase cultures strains 622 601-1B (cdcl-100)           10-3 Ura&apos; recombinants cell. Therefore, ax-factor-arrested <br /> grown YEPD(0.5). Aliquots removed indicated times, <br />                                                                            cdcl-100 cells able survive 1-h incubation galac- <br /> diluted, plated uracil omission medium score Ura&apos; colonies       tose, indicating equal SCE occurring significant <br /> and SC medium determine Ura&apos; recombination frequency (A).            extent DSB repair cdcl-100 wild-type cells the <br /> Colonies developed SC medium tested mating           ura3 duplication locus. This consistent observation <br /> type (B). The fraction cells switched represented number   log-phase cdcl-100 cultures maintain high viability when <br /> ax colonies plus number nonmating colonies divided total     treated galactose, G1 cells killed SCE <br /> number colonies (ax + + nonmating). Open circles, strain 622;         responsible high-viability DSB repair. <br /> closed circles, strain 601-1B (cdcl-100).                                     Efficient repair recombinogenic lesions cdcl-100 mu- <br />                                                                            tants. We identified level HO endonuclease expression <br />                                                                            threshold efficient killing known <br />                                                                            DSB repair-deficient strain, radS2A (see below), still <br /> galactose (Table 2). These data similar Nick-              capable providing recombinogenic lesions a <br /> oloff et al. (42) differ results Ray et al. (45),          wild-type population enable visual detection large <br /> who conversions associated deletions occurred              stimulation recombination (Fig. 2A). Because cdcl-100 <br /> infrequently. In contrast, ratio red white Ura&apos; cells            mutation differentially affected DSB recombination repair, <br /> was independent galactose treatment remained con-                   cell survival following treatment galactose (to induce full <br /> stant 7 1 strain 601-1A (cdcl-100) cultures. Hence,           expression HO) DSB-inducing agents exam- <br /> the ratio Ura+ gene conversion deletion events affected           ined. <br /> by cdcl-100 mutation altered following HO-                     Plating efficiencies determined wild-type, cdcl-100, <br /> induced DSB repair.                                                        rad52A strains grown media containing glucose, <br />   Are SCE events responsible efficient dsb repair                   galactose, (Table 3). Because rad52 mutants unable <br /> cdcl-100 mutants? SCE reactions damaged intact                 repair DSB, included rad52A strain analysis as <br /> chromatids repair mechanism account               control DSB induction. Wild-type cells grew readily on <br /> high viability absence increased Ura&apos; recombination              medium containing glucose galactose (YEPDGal). <br /> frequency observed cdcl-100 mutant. Repair ionizing              Viability YEPGal 92%. Viability cdcl-100 cells <br /> DNA damage G2 diploid cells efficient (6, 7). Kadyk             plated YEPDG medium comparable wild- <br /> and Hartwell (33) demonstrated recombination-                    type cells (97%). cdcl-100 rad52A mutant cells were <br /> inducing lesions incurred G2 diploids preferentially                capable colony formation YEPGal medium (61 versus <br />  VOL. 14, 1994                                                                                           CDC1 GENE IN S. CEREVISIE              8045 <br />  <br />        TABLE 3. Viability solid media containing glucose                        observed DSBs induced galactose, cdcl-100 <br />                         and/or galactose                                           mutant cells grew slowly increased frequency of <br />                                                                                    aberrant cellular morphology grown continuously on <br />        Strain             Carbon source         nb       Avg colony formation      medium containing MMS; however, sensitivity HO and <br />                                                                &#177;SD   (%)C <br />                                                                                    MMS approached rad52A level. <br /> 622 (wild type)          Glc                    920               100                 The repair capacity cdcl-100 mutants examined <br />                          Glc + Gal              904             98 &#177; 2             following single exposures DNA-damaging agents (Fig. 4). <br />                          Gal                    999             92 &#177; 7             Survival following y irradiation marginally for <br /> 601-1B (cdcl-100)         Glc                  1,331              100              cdcl-100 wild-type cells. Roughly 30% cdcl-100 <br />                           Glc + Gal            1,136            97 &#177; 2             cells resistant doses greater 20 kilorads. Wild-type <br />                           Gal                  1,040            61 &#177; 22            cultures exhibited 40% survival 20 kilorads. The plateau in <br />                                                                                    resistance ionizing radiation results G2-phase cells in <br /> 295-4A (radS2A)           Glc                  1,026              100              culture time irradiation (15, 16). Because wild- <br />                           Glc + Gal            1,109            70 &#177; 15            type cdcl-100 survival curves similar, slightly <br />                           Gal                  1,093            &lt;0.004d            increased sensitivity cdcl-100 cells represent lower <br />   Media: Glc, YEPD; GIc + Gal, YEPDG; Gal, YEPGal.                               level G2 cells time irradiation. <br />   b Total number cells examined experiments.                             The cdcl-100 mutant showed intermediate sensitivity to <br />   c Normalized growth YEPD medium. Average plating efficiencies         killing single exposures MMS relative wild-type and <br /> four experiments 99% &#177; 1% strain 622, 89% &#177; 3% strain 601-1B;         rad52A strains. Although cdcl-100 mutation less <br /> and 85% &#177; 4% strain 295-4A. <br />   d No colonies approximately 250 cells examined exper-        sensitive MMS rad52A, similar dose <br /> iment. Examination larger numbers cells spotted plate revealed   response rad52A exponential killing occurred with <br /> no colony formation approximately 2.5 x 104 cells spotted            elimination low-dose-resistant shoulder observed in <br /> experiment.                                                                        wild-type strains. Irradiation UV light slight <br />                                                                                    effect cdcl-100. As noted others, radS2A mutant strains <br />                                                                                    marginally sensitive killing UV irradiation <br /> 0%). radS2A mutant cells completed                         (15, 23, 47). <br /> cell division cycles galactose media usually arrested                       Molecular cloning characterization CDC] gene. <br /> very large budded cells result inability repair                   The wild-type CDC1 gene isolated complementation of <br /> DSBs generated ura3 AMT loci. These results                             mutant recombination defect. DNA sequencing revealed a <br /> indicate cdcl-100 mutant capable efficient DSB                      single open reading frame encoding predicted polypeptide <br /> repair conditions induce complete inviability                    product 491 amino acids (Fig. 5). No significant similarities <br /> of DSB repair-defective rad52A mutant.                                         existing proteins SwissProt, PIR, Gen- <br />    Cells similarly analyzed colony formation me-                       Pept, GPUpdate databases, using BLAST search <br /> dium containing MMS (Table 4). Wild-type cells exhibited                           program (3). We noted Cdcl primary sequence did <br /> high-level plating efficiency concentration                         terminally located regions (of unknown function) <br /> range tested (0.005 0.02%). As expected, rad52A strain                      rich serine basic amino acid residues (mostly lysine; Fig. <br /> experienced extensive killing, lowest drug dose. On                    5). This open reading frame examined M. Goebl to <br /> plates 0.005% MMS, cdcl-100 cells formed colonies                        determine showed similarities proteins contained <br /> nearly wild-type efficiency (95%). At intermediate drug                        database. His analysis revealed gene was <br /> concentration (0.01%), colony formation reduced                          identical open reading frame identified M. Paidhungat <br /> 50% decreased 5% 0.02% MMS. As                               S. Garrett complement temperature-sensitive <br />                                                                                    mutation obtained cdcl strain. Using genomic grid <br />                                                                                    plate provided <span id='am-100' about='obo:NCBITaxon_1191644' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_294789 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="rdfs:label" content="Linda" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Linda</span> Riles, CDC1 gene localized to <br />          TABLE 4. Viability solid media containing MMS                          chromosome IV near RAD9. Northern blot (RNA) analysis <br />                                                                                    identified RNA transcript approximately 1.4 kb, large <br />          Strain <br />        Strain              Conen (%) <br />                            Concn              ~~a <br />                                                n         Avg colony formation <br />                                                                 SD (%)b            encode putative protein (data shown). <br />                                                                                       To determine CDC1 mutants showed pheno- <br /> 622 (wild type)                 0            1,000                100              typic similarity cdcl-100, examined recombination <br />                                 0.005          928             100 &#177;0              phenotype mutant alleles CDC1 (21). Strains 369 (cdcl-l) <br />                                 0.01           964              99 &#177; 0             342 (cdcl-2) backcrossed strain 622 create <br />                                 0.02           876              98 &#177; 1             strains 648-1A (cdcl-l) 631-4A (cdcl-2). Under con- <br /> 601-1B (cdcl-100)               0            1,080                100              ditions assay, cdcl-2 allele cdcl-1 allele <br />                                 0.005        1,152              95 &#177; 5             exhibited galactose-inducible recombination (Fig. 2A). cdel-] <br />                                 0.01         1,040              56 &#177; 20            exhibited 5-fold increase HO-induced Ura+ recom- <br />                                 0.02         1,441               5&#177;2               bination frequency (after 2 h), strain cdcl-2 showed a <br />                                                                                    50-fold increase (18). Although cdcl-2 mutation was <br /> 295-4A (rad52A)                 0            1,046                100              proficient stimulation recombination following a <br />                                 0.005        1,400            0.02 &#177; 0.02          DSB, extent fold increase obtained <br />                                 0.01         1,400              &lt;0.005c            wild-type cells (Fig. 2A). <br />                                 0.02         1,400              &lt;0.005c               The cloned CDC1 gene capable restoring HO- <br />   a <br />     Total number cells examined experiments.                          induced Ura+ recombination proficiency cdcl-100 and <br />   b <br />     Normalized no-drug control. The average plating efficiencies      cdcl-1 strains (Fig. 2A). The plasmid copy CDC1 gene <br /> experiments 100% &#177; 0% strain 622, 87% &#177; 3% strain 601-1B,         reversed slow-growth defect cdcl-100 and <br /> 84% &#177; 2% strain 295-4A. <br />   c In example, 1,400 cells counted, positive.        cdcl-1 cells grown 24&#176;C solid media temperature- <br /> However, examination larger numbers cells spotted plate          sensitive growth (37&#176;C) phenotype cdcl-1 cdcl-2 (Fig. <br /> revealed colony formation average frequency approximately 1 2 X 104.   2B). Surprisingly, in-frame deletion allele CDC1 (initially <br />  8046       HALBROOK AND HOEKSTRA                                                                                                 MOL. CELL. BIOL. <br />  <br />                              y - irradiation                                  MMS                                      UV - irradiation <br />           100 <br />  <br />  <br />  <br />  <br />            101 <br />     9 <br />  <br />  <br />    U)2. <br />    ux <br />    (L <br />    0. <br />  <br />  <br />            1 <br />  <br />  <br />  <br />  <br />                                  Krad                                     percent (v / v)                                   J / m2 <br />   FIG. 4. DNA damage dose-response curves. Aliquots log-phase cultures irradiated -y radiation UV light plating <br /> on solid medium treated directly MMS liquid culture 1 h plating drug-free medium. In case, percent survival <br /> was calculated following microscopic determination viable nonviable cells. Circles, strain 622; squares, strain 601-1B (cdcl-100); triangles, <br /> strain 295-4A (rad52::TRPI). <br />  <br />  <br /> obtained intermediate molecule construction               100 cdcl-100/cdcl-1 (or cdcl-2) diploids exhibited poor <br /> deletion/insertion mutation; Materials Methods) par-                sporulation frequencies spore viabilities 3% <br /> tially restored HO-stimulated recombination cdcl-100                 allele combination. The cdcl-100 mutation identified <br /> cdcl-1 strains (despite missing 130 internal amino acid resi-               impact mitotic DSB recombinational DNA repair also <br /> dues) capable complementing temperature                  adversely affects meiosis, phenotype dominant to <br /> sensitivity cdcl-1 cdcl-2 strains. This finding                cdcl-1 cdcl-2 mutations. <br /> suggests amino and/or carboxy termini Cdcl                    To investigate properties severe mutations CDC1, <br /> form domain influences DSB-induced recombination                     inserted URA3 gene flanked E. coli hisG DNA (2) <br /> and temperature-sensitive mutations reside                  130-amino-acid deletion allele used replace <br /> internal domain.                                                            copy CDC1 diploid strain 488. Tetrad analysis of <br />    To demonstrate cloned gene CDC1, mated                   Ura+ transformants showed 2+:2- segregation viability. In <br /> a cdcl-l mutant strain tagged CDC1 strain constructed                41 44 tetrads, exhibited viable Ura+ segregants (82 of <br /> by integration plasmid containing cloned CDC1 gene                 82 Ura-), demonstrating CDC1 essential gene (of <br /> and TRP1 gene. Tetrad analysis sporulated                        aberrant tetrads, showed viable spores, <br /> CDC1::CDC1 TRPJ/cdcl-1 diploid revealed 119                     Ura- Ura+). <br /> viable four-spored asci (out 132 total tetrads dissected)                Microscopic examination dissected spores 3 days <br /> demonstrated parental ditype segregation pattern. That is,              showed unbudded small-budded single cells. <br /> there complete coincidence 2+:2- growth 37&#176;C <br /> and 2+:2- tryptophan prototrophy 24&#176;C, indicating that <br /> the cloned gene CDC1 tightly linked. Unfor-                                             DISCUSSION <br /> tunately, poor sporulation heteroallelic diploid combi- <br /> nations cdcl-100 mutation impossible                       Recombinational DSB repair defect cdcl-100 muta- <br /> observe segregation cdcl-100 cdcl-l cdcl-2               tion. An intramolecular recombination assay used to <br /> alleles. Although cloned gene integrated                    isolate DSB-stimulated recombination-defective mutants S. <br /> cdcl-100 strain complemented mutant defect, strain                 cerevisiae. One these, cdcl-100, characterized found <br /> (cdcl-100::CDCJ TRP1) sporulated poorly mated                     deficient DSB-stimulated recombination between <br /> with cdcl mutant strains, suggesting dosage effect             duplicated ura3 alleles. This defect associated with <br /> of CDC1 sporulation.                                                     increased cell lethality, indicating DSBs effectively <br />    Sporulation spore viability affected ho-                repaired. The absence DSB-induced killing cdcl-1 <br /> mozygous cdcl-100 diploids heteroallelic cdcl-1 (or cdcl-               cdcl-100 mutations suggests manner which <br /> 2)/cdcl-100 diploids (Table 5). Whereas cdcl-l/cdcl-l ho-                   recombinational DNA DSB repair normally carried can <br /> mozygous diploids sporulated high efficiency (55)                  altered eliminating repair occurs RADS2A <br />  exhibited spore viability greater 75%, cdcl-100/cdcl-              mutants. This level repair proficiency suggests that <br />  VOL. 14, 1994                                                                                                                      CDCI GENE IN S. CEREVISME                                 8047 <br />  <br />  <br /> A                                                                                 =   AC-A w -GC TCAAAACTCGCGGG&apos; AGA TT <br /> -24 0                                                   GGr-cnTi-AATo-TGCTAC7M- <br />          TIAAAAATACCCA TTG--AT;AACTl^CTCATATScTTTkAArcTAGTAUC;A~ <br /> -1 20     TT=A7-rAAAATAcTAcAcAcGATT7,GAACAACkAATAAC CAAAGkGAAACAG-TACCGCTCTACTA ATIC7&apos;lr;ATA CATTAGAC AATAAGCI1CTAGZACGT <br />     1    K        V   Y   R   U   R   SL&amp;     S   V   L   S   T   H   S    K       K   S   D    D   K    A    H   Y    K   S   A    S     K       K   K    S   K    S   R   S       K    R    L <br />     1     Am;GTATAMGCAACACAAAGAGCGTTATCTACACACAGCAAGAAGAGCGACTATA.AGGCCCACTATAAA                                                   &apos;ACG       A           TCAAAAAGCAGGACkAGAAGATA <br />   41      R  I Y W R Y I S I V W I L W L 0 L I S Y Y E S V V V K R A M K K C 0 W S T W E D <br />  12 1     AGAATCTAC7GAGATACA TATCTATTSTATGA7ATITAGGATTAATAT&apos;-ATrA7&apos;AATC AG1TTGTlrAGAG AGCAAr-AAGAA-v,CCAAI&apos; H,;WC ACTAGGArT <br />   81      W       P   E   0   A   K   S   H   R   V   0   L   F   A   D    P       Q   I   N    D   E    Y    S    Y P G R P Q I V H Y F T R V r v D <br />  24 1     TGCCTAAGGGTAAAGMATAGGroG-ATGC^AGACC CAC AAATAAIACGAATAC                                                 ATA TCC7GAGGCCTCAAATAGTTAArTAr=lAC AAGAGTIArT7,AC <br />  <br />  121      H       Y   H   A   R   U   W   K   Y   V   Q   Y   Y   L   D    P       D   S   W    F   F    L    C    D   L   F   D    0     0       R   U    W   D    D   K   Q   W   I    K    E <br />  3 61     CATTAMACAGAGAM7ATAMTCCAGTATTAC~rGATCCT                                       ATAGCAIrTCCTGGGA TCTATr.A~rAGAAAGAC                                         TA TAA GCAAlr,,ATNAGAC <br />  161      Y       T   R   F   S   Q     F   P   K   K   P   L   R   R    T       V   X   8    L   P    0    S   H    D   I   0    F     0       D   T    V   V    E   8   8   L   Q    R    F <br />  4 81     TACACACGA~rAACCAAATATCCCCACAGCCACNrAGAAGAACGG rA&apos;LALN-L -ll-ACCA0G TAAT1 ACGATA7_Ii&apos;-GAGACACAGIrTCrAGAGT-ACAGAGATT <br />  201      8 8 Y F 0 E T S 8 S L D A 0 W H T F V L L D S I 8 L S D K S U P U V S R V P R Q <br />  6 01     TCTAGTTAC &apos;GAGAAACCMCTCCTCArrAGACGCTC-AATCACTACCTTMTTCAGATACAAT-rC&apos;MTGATAAAAC AAAWCGAATIGTCTCGAGGGICCCTAGACAG <br />  241      r  L D X F A K 0 S H P L P R I L L T H V P L W R D P K Q Q T C 0 Q L R E S K E P <br />  &apos;7 21    IiGrATAAlolCAAAl oGTACATCCGC7CCTAGAACTTTC&apos;&apos;&apos;-AACTCACGnCC CAIrATGGAGG;GATlCCAGAACAG&apos;CAAACGlIGCGG TCAGCTACGGATIC CAAGAC CT <br />  281      T       P   I   Q   K   C   H   Q   Y   Q   T   V   I   E   Y    D       I   S    Q   E    I   L    T    K   I   0   P    K     I       L   r    S   0    D   D   H   D   H    C    Q <br />  84 1     lTMCTATTAGAAAGGGCACCAGTACCAGACAGTTATAGAAACGATATATCACAAGAAATnTAACGAAAATTCAACCAGAAATATTATICGGGGATGATCAMACCAWGCAG <br />  321      I S H 8 Y P F Q 0 K T K                             U   A   Q    K       I   T    V   K    S   C    A    M   S   K   0    I     S       R   P    A   I    0   L   L   S   L    Y    H <br />  961l     ATA7CACACTATACGCrC GGGAACTAAAA                      rCCAAGAAATTACAGTAAAGTCAM7TGAATr.ACA7WCTCAGTACGCCIGTATrkAlrCTCTICCTTATATAA T <br />  361          P   8   D   L   T   U   V   U   A   0   0       Y   A   S    K       T   Y    Q   T    E   L     C Y     K   P   D    P     Y       K   A    I   R    X   Y   L   W   0    L    L <br /> 10 81     CCTTCGA7rACAAGTITAAGCCGGCGGAGAGTATACTT                          -AAAACCAAACAGAGCTAlE;              CTATA     TGCCAGATICCCTATAAGGCAATCAGAATIGTACCTAlE;GGGATTATTA <br />  401          F S A A F I A Y U H F F            P K S F S Y R V A T I K U R V F T R P D 0 U T S D L A L <br /> 12 01         7 XIX:CLGC                          CCAAAAlI~lWAATkATICGCGTAGCAACCATAA~tAACAGAGTATTTAC ACGACC TGA~r.;GTAA CACC^C AGA TIrGCATTA <br />                          CATCGCATACAlrf;CA71r&apos;l-ss- s <br />  441          P T 8 I BK S K S K BJL       K     L T H S K Y A V U D T R s I iK Q r L V N A I V L F G S V <br /> 1 3 21        CCTACAAGTACCAAATCGAAATCCAAAAAATCA7&apos;1GACACATAGTA.AATACGCM7GTAAlfATACGCGTTCGATCAAACAA TTETAGTTAA l;C AAT=TCC7&apos;r-r7GlMC m;G <br />  481          U   P   I   F   I   Y   F   Y   T   V   V   - <br /> 14 41         ASGCCCATMrrATTbA~mAATAC7&apos;GTAGTICTAGACAGAATTACTS;CTAT7TCTC7TCCAGGATCAA.AAT&lt;;CTIGrAAGGGAAACG&apos;CT&apos;iC;TATTAATATATIGCGAATACACCTTAT <br /> 1 5 61        ATACAGACAAsxACAAGAGATATAAATTATACCGOGG7GGT~ATTACTATGCTICACCAAACAGI&apos;rrl.CATAAC C&apos;-TACTGGArTC ATATAGl TllE;GTCCkAGC TAAAAC 1GC A <br /> l168B1        GGACIX;GCGTAACACCAATTACT~rCACTATACoGGCGACAAAAAAGAACACCTAGGTAAGlEIlwloGTTCTTTAG -MrrllATA7TCAAATTTAGCA.AGTCATIGACGGCAAAlC7TAST <br />  <br />  <br />  <br />  <br />     B <br />                                                                                                                                                                    N-   _               N- <br />          (.                                                                            t3                                                                               N-O r <br />                   LnCo <br />                                                                                                              , <br />  <br />  <br />  <br />                                                                                        __ <br />  <br />                                                                                I <br />  <br />  <br />  <br />                                                                             B <br />  <br />  <br />  <br />                                               hisG                        URA3                                hisG <br />    FIG. 5. (A) Nucleotide inferred amino acid sequences CDC1 gene. Primary sequences rich serine, lysine, arginine are <br /> underlined. (B) CDCI gene structure. The drawing represents wild-type gene subcloned p220. The middle drawing shows in-frame <br /> deletion constructed removing 390-nucleotide internal BamHI fragment (p221). The drawing shows cdcl::URA3 null allele used <br /> to replace CDCI gene (p222). Triangles, sites TnlOLUK insertions used locate CDC1 open reading frame; dashed lines, DNA <br /> deletions; unbroken lines, DNA insertions. Restriction enzymes: B, BamHI; E, EcoRV; M, MluI; P, PstI; R, EcoRI; S, Sail; X, XbaI; Z, XhoI. <br />  <br />  <br /> the recombinational repair defect cdel mutants does                                        mutations DSB repair partial loss function and <br /> not lead formation unrepairable reaction intermedi-                                       differs significantly dramatic affect rad52 null <br /> ates products likely occurs radS2 cells (56, 61). Since                                    mutation. However, finding cdcl-100 mutation <br /> CDCJ essential gene, likely affect cdcl                                     alters spectrum recombinants affecting cell <br />  8048       HALBROOK AND HOEKSTRA                                                                                                 MOL. CELL. BIOL. <br />  <br />            TABLE 5. Sporulation defect cdcl mutations                      A rad52-like sensitivity cdcl-100 cells DSBs the <br />                                                                               absence homologous sequences provide additional <br />   Relevant genotype                                                           evidence support homology-dependent recombinational <br />    parental strain        Four-spored           Total            Spore     repair DNA DSBs. <br />                                  <span id='am-110' about='obo:NCBITaxon_320285' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_42449 obo:NCBITaxon_7114 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_152601 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-111' property="rdfs:label" content="Ascia" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ascia</span>          sporulation&apos;      viability      Repair damage induced MMS -y irradiation. The <br /> CDC1 x CDC1                       45                63               96       cdcl-100 mutation somewhat unusual DNA repair <br /> CDC1 X cdcl-100                   49                55               90       characteristics, particularly sensitive large single- <br /> CDC1 x cdcl-1                     22                53               80       dose exposures -y irradiation, continuous expression of <br /> CDC1 X cdcl-2                     26                51               83       HO endonuclease, sensitive MMS treatment. <br /> cdcl-100 X cdcl-100                8                18                2       However, phenotype unprecedented, several <br /> cdcl-100 X cdcl-1                  0                 2              NDc       genes mediate repair alkylation damage S. <br /> cdcl-100 X cdcl-2                  3                17                3       cerevisiae (32, 58) DSBs <span id='am-30' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span>Schizosaccharomyces</span> pombe <br /> cdcl-1 X cdcl-1                   30                52               75       (12) MMS sensitivity refractory killing by <br /> cdcl-1 X cdcl-2                   53                60               98 <br /> cdcl-2 X cdcl-2                   64                79               88       ionizing radiation. The differential -y-irradiation MMS <br />                                                                               sensitivity nature DSBs resulting from <br />   &quot; Calculated following examination 400 cells. Sporulation   treatments. DSBs induced MMS thought an <br /> carried 24&#176;C.                                                          indirect consequence incomplete excision repair alkylated <br />   &quot; Calculated analysis 22 dissected tetrads. <br />   c ND, determined.                                                       nucleotide base residues close proximity opposite <br />                                                                               DNA strands (9). DNA breaks caused ionizing radiation <br />                                                                               generate variety DNA double-strand termini (24) which <br />                                                                               result nucleophilic attack activated oxygen radi- <br /> viability indicates mutation profound effect               cals DNA direct excitation ionization DNA <br /> the manner DSBs repaired.                                        components (46). Therefore, repair MMS-generated DSBs <br />    A number possibilities exist explain apparent                    efficiently repaired y-ray-induced breaks in <br /> dissociation DSB repair measurable recombina-                     cdcl-100 strain. Alternatively, cdcl-100 mutant may <br /> tion observed cdcl-100 mutant strain. One possibility                  suffer defect alkylation excision repair. The <br /> that ruled SCE events responsible                 absence MMS-resistant shoulder mutant strain <br /> high-viability DSB repair strains. Although SCE              suggests regardless lesion responsible inhibition <br /> efficient DNA repair pathway (33), appears play                   cell growth, repair capacity saturated doses do <br /> major role repair DSBs ura3 gene duplication                 affect wild-type cells. <br /> substrate wild-type cdcl-100 mutant cells. Instead,                 At intermediate levels HO expression continuous <br /> the cdcl-100 mutation slow inhibit recombinational                 exposure low levels MMS (&lt;0.01%), DSBs probably <br /> repair process reactions normally occur               generated rate repair capacity cdcl-100 cells <br /> at low rates divert substrates and/or reaction interme-               repeated cell division cycles completed. <br /> diates competing repair pathways increased effi-                    Protracted exposure intermediate concentrations MMS <br /> ciency. Similarly, rad52 mutations slow recombi-                 (&gt;0.01%) maximal HO endonuclease expression (on YEP- <br /> nation rate step reactions deleterious               Gal) result DNA damage rate approaches the <br /> effects (e.g., extensive exonucleolytic DNA degradation)                  saturation level repair. High-level HO expression growth <br /> occur. Alternatively, exist point recombi-                 0.01% MMS reduces cdcl-100 plating efficiency nearly <br /> national DNA repair process blocked,                         50%; however, treatment result re- <br /> preventing completion recombination inhib-                 peated cutting sites: AM T ura3 (when repair <br /> iting, stimulating, reverse reactions ligation            restores HO cut site). Presumably, growth MMS <br /> of initial partially processed DNA DSB. If cdcl-100                medium continually creates DNA lesions, sites distrib- <br /> mutation causes inhibition entire DSB recombinational                   uted genome. Interestingly, 30% log-phase <br /> repair pathway, cells utilize alternative, hitherto               cdcl-100 mutant culture (compared 40% wild-type cell <br /> <span id='am-120' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span>unidentified</span> pathway. A related possibility major                   cultures) withstand 40-krad exposure y rays. From the <br /> repair pathway blocked simultaneously creating                 data Resnick Martin (49), estimate expo- <br /> an alternative repair pathway peculiar cdcl mutants               sure induce 6 12 DSBs haploid genome, <br /> not normally observed wild-type cells. Finally,                  indicating radiation-induced lesions single treat- <br /> exist nonrecombinational mechanisms normally compete                     ment repaired efficiently cdcl-100 mutant. Hence, <br /> with recombinational repair pathways DNA DSB sub-                         repeated generation DSB AVT (and possibly ura3) <br /> strates.                                                                      represents special situation severely affects cell <br />    Substantial evidence suggests DSB repair occurs                  growth cdcl-100 growth wild-type strains. <br /> recombinational mechanism (15, 36). A large measure                           The cell division cycle recombinational DNA repair. <br /> support comes observation homologous DNA                     When introduced recombination assay strain, the <br /> required efficient DSB repair (9, 15, 39, 48). The observa-               cdcl-1 allele exhibited slow-growth HO-stimulated re- <br /> tion mutations affect induced mitotic recombina-                 combination defect (each permissive temperature) sim- <br /> tion cell survival following DNA damage enhances                  ilar cdcl-100 allele. As cdcl-100 mutation, these <br /> this notion (15, 36). Although properties cdcl-100                 defects recessive suppressed plasmid-borne <br /> mutation imply existence nonrecombinational DNA                      copy cloned CDC1 gene. In contrast, cdcl-2 allele is <br /> repair process (as increased recovery Ura&apos; recombinants                 HO-stimulated recombination proficient. Therefore, but <br /> was associated DSB repair), tested                   mutations CDC1 gene disrupt DSB recombina- <br /> this mutation repair competency absence DNA                     tional DNA repair. <br /> homology. Recently, Kramer et al. (35) demonstrated                         cdcl mutants arrest nonpermissive temperature a <br /> that HO-induced DSBs repaired R4D52-                     heterogeneous population respect cell spindle <br /> DNA homology-independent manner, albeit low efficiency.                    pole body morphology (8, 19, 21). cdcl mutant cells incubated <br />  VOL. 14, 1994                                                                                       CDC1 GENE IN S. CEREVISIAE                   8049 <br />  <br /> at nonpermissive temperature exhibit spectrum cell                 3. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. <br /> morphologies ranging cells single duplicated                      1990. Basic local alignment search tool. J. Mol. Biol. 215:403-410. <br /> spindle pole bodies partially elongated nuclear               4. Atkinson, D. E. 1977. Cellular energy metabolism regula- <br /> spindles. Although DNA replication affected, nuclear                     tion. Academic Press, New York. <br />                                                                             5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive <br /> division inhibited. Bud development inhibited                     selection mutants lacking orotidine-5&apos;-phosphate decarboxyl- <br /> cultures arrest unbudded small-budded cells. The                   ase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> participation CDC1 gene product discontinuous                     Genet. 197:345-346. <br /> events cell cycle questioned, cdcl-1               6. Brunborg, G., M. A. Resnick, D. H. Williamson. 1980. <br /> allele suffers temperature-sensitive defect RNA                  Cell-cycle-specific repair DNA double-strand breaks Saccha- <br /> synthesis (19, 20).                                                             romyces cerevisiae. Radiat. Res. 82:547-548. <br />    The CDC1 mutations, cdcl-J, cdcl-2, cdcl-100,                   7. Brunborg, G., D. H. Williamson. 1978. The relevance the <br /> studied unique respect combination                    nuclear division cycle radiosensitivity yeast. Mol. Gen. Genet. <br /> of mutant defects exhibit. Whereas cdcl-1 cdcl-2                  162:277-286. <br /> exhibit temperature-sensitive cell cycle arrest (cdcl-100 does               8. Byers, B., L. Goetsch. 1974. Duplication spindle plaques <br />                                                                                 integration yeast cell cycle. Cold Spring Harbor Symp. <br /> not), cdcl-J cdcl-100 recombinational repair                       Quant. Biol. 38:123-131. <br /> defective. A deletion allele CDC1, lacking 130 internal                  9. Chlebowicz, E., W. J. Jachymczyk. 1979. Repair MMS- <br /> amino acids, partially suppress recombination defect                 induced DNA double-strand breaks haploid cells Saccharo- <br /> the cdcl-100 cdcl-J mutations temperature-                      myces cerevisiae, requires presence duplicate <br /> sensitive growth arrest phenotype cdcl-J cdcl-2 muta-                    genome. Mol. Gen. Genet. 167:279-286. <br /> tions. This finding suggests elements amino and/or             10. Connolly, B., C. I. White, J. E. Haber. 1988. Physical moni- <br /> carboxy termini responsible recombinational DSB                     toring mating type switching Saccharomyces cerevisiae. Mol. <br /> repair property CDC1 gene product cdcl-1                    Cell. Biol. 8:2342-2349. <br /> cdcl-100 mutations reside regions. The terminal               11. Davies, P. J., W. E. Evans, J. M. Parry. 1975. Mitotic <br />                                                                                 recombination induced chemical physical agents the <br /> amino acid sequences rich basic residues                       yeast Saccharomyces cerevisiae. Mutat. Res. 29:301-334. <br /> support interactions nucleic acids. Since cdcl-1 mu-              12. Dhillon, N., M. F. Hoekstra. 1994. Characterization two <br /> tation temperature-sensitive cell cycle defect                 protein kinases <span id='am-35' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-36' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> involved the <br /> recombinational DNA repair defect, analysis additional                       regulation DNA repair. EMBO J. 13:2777-2788. <br /> cdel mutants required conclusions                    13. Elble, R. 1992. A simple efficient procedure transformation <br /> reached regarding possible roles CDC1 functional                         yeasts. BioTechniques 13:18-20. <br /> domains.                                                                   14. Fishman-Lobell, J., N. Rudin, J. E. Haber. 1992. Two <br />    Initial characterization cdcl mutants did indicate                  alternative pathways double-strand break repair kinet- <br /> role CDC1 gene events associated DNA                            ically separable independently modulated. Mol. Cell. Biol. <br />                                                                                 12:1292-1303. <br /> recombination. However, behavior cdcl-1 allele               15. Game, J. C. 1983. Radiation-sensitive mutants repair yeast, <br /> our recombination assay strain indicates CDC1 partici-                     p. 109-137. In J. F. T. Spencer et al. (ed.), Yeast genetics: <br /> pates aspect DNA metabolism, recombina-                      fundamental applied aspects. Springer-Verlag, Berlin. <br /> tional repair DSBs. Byers Goetsch (8) observed                 16. Game, J. C., R. K. Mortimer. 1974. A genetic study X-ray <br /> cdcl mutants exhibit heterogeneous arrest respect                     sensitive mutants yeast. Mutat. Res. 24:281-292. <br /> spindle pole body spindle morphology. If CDC1 partici-                 17. Gietz, R. D., A. Sugino. 1988. New yeast-Eschenichia coli <br /> pates aspect microtubule metabolism nuclear                       shuttle <span id='am-96' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> constructed vitro mutagenized yeast genes <br /> division, influence DSB-induced recombination                    lacking six-base pair restriction sites. Gene 74:527-534. <br /> repeated sequences direct indirect conse-               18. Halbrook, J., M. Hoekstra. Unpublished data. <br />                                                                            19. Hartwell, L. H. 1971. Genetic control cell division cycle in <br /> quence disrupting DNA metabolism. Further study                     yeast. IV. Genes controlling bud emergence cytokinesis. Exp. <br /> mutations addition cdcl (20) DSB-stimulated                     Cell Res. 69:265-276. <br /> recombination-defective mutations isolated (18)               20. Hartwell, L. H., J. Culotti, J. R. Pringle, B. J. Reid. 1974. <br /> should provide better understanding recom-                     Genetic control cell division cycle yeast. Science 183:46- <br /> binational DNA repair processes integrated                         51. <br /> execution cell division cycle.                                      21. Hartwell, L. H., J. Culotti, B. Reid. 1970. Genetic control of <br />                                                                                 cell-division cycle yeast. I. Detection mutants. Proc. Natl. <br />                       ACKNOWLEDGMENTS                                           Acad. Sci. USA 66:352-359. <br />                                                                            22. Hartwell, L. H., R. K Mortimer, J. Culotti, M. Culotti. 1973. <br />    We appreciate critical discussions Namrita Dhillon              Genetic control cell division cycle yeast. V. Genetic <br /> help constructing plasmids used study. We appreciate            analysis cdc mutants. Genetics 74:267-286. <br /> helpful comments provided Jac Nickoloff preparation       23. Haynes, R. H., B. A. Kunz. 1981. DNA repair mutagenesis <br /> manuscript. We thank Christi Wood providing DNA sequenc-                yeast, p. 371-414. In J. N. Strathern, E. W. Jones, J. R. <br /> ing acumen Mark Goebl analyzing cloned DNA sequence                 Broach (ed.), The molecular biology yeast Saccharomyces: <br /> against database. We appreciate S. Garrett sharing             life cycle inheritance. Cold Spring Harbor Laboratory, Cold <br /> results.                                                                        Spring Harbor, N.Y. <br />    This work partly supported <span id='am-1' about='obo:NCBITaxon_317678' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_317663 obo:NCBITaxon_6843 obo:NCBITaxon_317662 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_61977 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Damon" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Damon</span> Runyon-Walter Winchell          24. Henner, W. D., S. M. Grunberger, W. A. Haseltine. 1982. Sites <br /> <span id='am-44' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-45' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Research Fund fellowship DRG-1077 J.H.                                structure y radiation-induced DNA strand breaks. J. Biol. <br />                                                                                 Chem. 257:11750-15754. <br />                              REFERENCES                                    25. Herskowitz, I., J. Rine, J. N. Strathern. 1992. Mating-type <br />  1. Aguilera, A., H. L. Klein. 1988. Genetic control intrachro-          determination mating-type interconversion Saccharomyces <br />     mosomal recombination Saccharomyces cerevisiae. I. Isolation             cerevisiae, p. 583-656. In E. W. Jones, J. R. Pringle, J. R. <br />     genetic characterization hyper-recombination mutations.              Broach (ed.), The molecular cellular biology yeast <br />     Genetics 119:779-790.                                                       Saccharomyces. Cold Spring Harbor Laboratory Press, Cold <br />  2. Alani, E., L Cao, N. Kleckner. 1987. A method gene                  Spring Harbor, N.Y. <br />     disruption allows repeated use URA3 selection con-      26. Ho, K. S. Y. 1975. Induction DNA double-strand breaks by <br />     struction multiply disrupted yeast strains. Genetics 116:541-545.        X-rays radiosensitive strain yeast Saccharomyces <br />  8050      HALBROOK AND HOEKSTRA                                                                                                     MOL. CELL. BIOL. <br />  <br />     cerevisiae. Mutat. Res. 33:157-164.                                           nation yeast, p. 407-522. In J. R. Broach, E. W. Jones, J. R. <br /> 27. Ho, K. S. Y., R. K. Mortimer. 1973. Induction dominant                 Pringle (ed.), The molecular cellular biology yeast <br />     lethality X-rays radiosensitive strain yeast. Mutat. Res.          Saccharomyces. Cold Spring Harbor Laboratory Press, Cold <br />     20:45-51.                                                                     Spring Harbor, N.Y. <br /> 28. Hoekstra, M. Unpublished data.                                          45.   Ray, A., I. Siddiqi, A. L. Kolodkin, F. W. Stahl. 1988. <br /> 29. Hovland, P., J. Flick, M. Johnston, R. A. Sclafani. 1989.                 Intra-chromosomal gene conversion induced DNA double- <br />     Galactose gratuitous inducer GAL gene expression                   strand break Saccharomyces cerevisiae. J. Mol. Biol. 201:247- <br />     yeasts growing glucose. Gene 83:57-64.                                     260. <br /> 30. Huisman, O., W. Raymond, K. U. Froehlich, P. Errada, N.                 46.   Repine, J. E., 0. W. Pfenninger, D. W. Talmage, E. M. Berger, and <br />     Kleckner, D. Botstein, M. A. Hoyt. 1987. A TnlO-lacZ-                     D. E. Pettijohn. 1981. Dimethyl sulfoxide prevents DNA nicking <br />     kanR-URA3 gene fusion transposon insertion mutagenesis                    mediated ionizing radiation iron/hydrogen peroxide-gener- <br />     fusion analysis yeas <br /> </body></html>