<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.14 no. 10 pp.2239-2249, 1995 <br />  <br />  <br /> Mutations GCD11, structural gene eIF-2y in <br /> yeast, alter translational regulation GCN4 the <br /> selection start site protein synthesis <br />  <br /> David R.Dorris, FLes Erickson                                      GTP-bound state. Thus eIF-2-GDP produced the <br /> Ernest M.Hannig1                                                       initiation step exchange GDP GTP <span id='am-72' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to <br />                                                                        participate subsequent round protein synthesis <br /> Department Molecular Cell Biology, The University Texas   initiation. The GDP/GTP exchange reaction eIF-2 is <br /> Dallas, PO Box 830688, Richardson, TX 75083-0688, USA                  catalyzed eIF-2B, factor composed five <br /> &apos;Corresponding author                                                  non-identical subunits humans yeast (Cigan <br /> Communicated G.Thireos                                              et al., 1993). The structural genes encoding subunits <br />                                                                        yeast eIF-2B, denoted GCDJ, GCD2, GCD6, GCD7 <br /> Translation initiation factor 2 (eIF-2) eukaryotic                  GCN3, isolated sequenced, with <br /> organisms composed non-identical subunits,                 human cDNAs encoding homologs yeast GCD2 <br /> a, ,B y. In previous report, identified GCDI1                 (eIF-2B6) GCD6 (eIF-2BE) genes (Hannig and <br /> as essential gene encoding y subunit eIF-2                Hinnebusch, 1988; Hill Struhl, 1988; Paddon et al., <br /> the yeast <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. The predicted                      1989; Bushman et al., 1993; Price et al., 1994). The <br /> amino acid sequence yeast eIF-2y displays remark-                   exchange reaction catalyzed eIF-2B regulated <br /> able similarity bacterial elongation factor lTu, includ-            control rate protein synthesis initiation. One <br /> ing presence sequence elements conserved                 well-characterized mechanism involves phosphorylation of <br /> known guanine nucleotide binding proteins. We                     SerS 1 eIF-2ax specific protein kinases. In mammalian <br /> identified molecular defects present seven unique               cells, phosphorylation event increases affinity <br /> alleles GCDII characterized partial loss                    eIF-2 eIF-2B reduces eIF-2B function by <br /> function. Three mutations result amino acid                sequestering limiting amounts exchange factor, <br /> substitutions putative GTP binding domain                   reducing level ternary complex cell <br /> of eIF-2y. We gcdll mutations specifically               (Siekierka et al., 1984; Merrick, 1992). Mammalian eIF- <br /> alter regulation GCN4 expression translational               2a kinases activated conditions lead cellular <br /> level, altering scanning mechanism                     stress, including heat shock, heme deprivation, amino acid <br /> protein synthesis initiation. Six mutant alleles                starvation presence low amounts double- <br /> presumably alter function eIF-2y,                   stranded RNA occur certain viral infections <br /> its abundance. A single allele, gcdll-R5J0H, suppresses                (Merrick, 1992). GCN2, eIF-2a kinase Saccharo- <br /> a mutant his4 allele lacks functional AUG start                 myces cerevisiae, activated response amino acid <br /> <span id='am-64' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-65' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68'  ></span>codon</span>. The result indicates y subunit               starvation increased levels uncharged tRNAs that <br /> eIF-2 participates recognition start site                recognized domain GCN2 similar <br /> protein synthesis, role previously demonstrated                   histidyl-tRNA synthetase (Krupitza Thireos, 1990; <br /> yeast eIF-2a eIF-2I.                                           Dever et al., 1992; Hinnebusch, 1993). <br /> Key words: eIF-2y/general control/protein synthesis                       GCN2 originally identified positive effector <br />                                                                        general control pathway acts increase <br />                                                                        transcription 30 unlinked amino acid biosyn- <br />                                                                        thetic genes, multiple pathways, response starvation <br /> Introduction                                                           single amino acid (Wolfner et al., 1975; Hinnebusch <br /> Initiation factor 2 (eIF-2) plays critical role               Liebman, 1991; Hinnebusch, 1993). Transcriptional <br /> initiation protein synthesis eukaryotic organisms.               activation amino acid biosynthetic genes requires <br /> eIF-2 forms ternary complex GTP amino-                      transcriptional activator protein GCN4 (Hinnebusch <br /> acylated tRNA1Met delivers Met-tRNAiMet 40S                Liebman, 1991; Hinnebusch, 1993). GCN4 itself <br /> ribosomal subunit (reviewed Hinnebusch Liebman,                 regulated translational level <span id='am-98' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102'  ></span>cis</span>-acting elements <br /> 1991; Merrick, 1992). Based available <span id='am-113' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117'  ></span>data</span>,                trans-acting effectors. The cis-acting elements include <br /> is believed 43S pre-initiation complex consisting            short upstream open reading frames (uORFs) present <br /> the 40S subunit, ternary complex additional initiation             5&apos;-leader region GCN4 mRNA. Trans-acting <br /> factors binds near 5&apos;-capped end mRNA                    effectors products GCD (negative effectors) and <br /> scans 5&apos; -&gt;3&apos; direction start codon recognized         additional GCN (positive effectors) genes. In unstarved <br /> (Kozak, 1989). The GTP bound eIF-2 hydrolyzed               cells, uORFs present GCN4 mRNA leader <br /> in reaction requires eIF-5, eIF-2-GDP                    inhibit synthesis GCN4 protein. Initiation and <br /> binary complex released. A 60S ribosomal subunit               subsequent termination uORF believed to <br /> attaches 40S- mRNA-Met-tRNAiMet complex                      generate fraction 40S subunits resume scanning <br /> form 80S complex enters elongation phase              absence eIF-2. Dever et al. (1992) proposed that <br /> of protein synthesis.                                                  unstarved cells contain high level ternary complex <br />    In order bind Met-tRNAiMet, eIF-2                 allows 40S subunits re-acquire ternary <br />  <br /> K Oxford University Press                                                                                                     2239 <br />  D.R.Dorris, F.L.Erickson E.M.Hannig <br />  <br />  <br /> Table I. <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-34'  ></span>Saccharomyces</span> cerevisiae strains used study <br /> Strain               Genotype                                                                          Source reference <br /> EY159                MATa leu2-3 leu2-112 ura3-52 gcn3::LEU2                                           Hannig Hinnebusch, 1988 <br /> EY250                MATa gcdll-508 gcn2-101 gcn3-101 ura3-52                                          Hannig et al., 1993 <br /> EY305                MATax gedll-505 leu2-3 leu2-112 ura3-52 gcn3::LEU2                                <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> study <br /> EY555                MATa gcdll-507 gcn2-101 gcn3-101 hisl-29 ura3-52 (HIS4::lacZ ura3-52)             This study <br /> EY558                MATa gcdll-506 gcn2-101 gcn3-101 hisl-29 ura3-52 (HIS4::lacZ ura3-52)             This study <br /> EY582a               MATa gcdll::hisG leu2-3 leu2-112 ura3-52 &lt;Ep293(GCDII)&gt;                           This study <br /> EY585                MATas gcdll::hisG leu2-3 leu2-112 ura3-52 &lt;Ep293(GCDII)&gt;                          This study <br /> EY606a               MATa gcdll::hisG leu2-3 leu2-112 ura3-52 gcn3::hisG &lt;Ep293(GCDII)&gt;                This study <br /> EY636a               MATa gcdll::hisG leu2-3 leu2-112 ura3-52 &lt;Ep552(GCDII)&gt;                           This study <br /> EY640d               MATa his4-303(AUU) inol-13 ura3-52                                                This study <br /> EY641d               MATa his4-303(AUU) inol-13 ura3-52 SUI3-2                                         This study <br /> EY647                MATa gcdll::hisG his4-303(AUU) inol-13 leu2-3 leu2-112 ura3-52 &lt;Ep293(GCDII)&gt;     This study <br /> F35                  MATa inol can] ura3-52 (HIS4::lacZ URA3+)                                         Lucchini et al., 1984 <br /> H4                   MATa leu2-3 leu2-112 ura3-52                                                      Harashima Hinnebusch, 1986 <br /> H57b                 MATa GCDII-501 gcn2-101 gcn3-101 his1-29 ura3-52 (HIS4::IacZ ura3-52)             Harashima Hinnebusch, 1986 <br /> H59b                 MATa GCD11-502 gcn2-101 gcn3-101 hisl-29 ura3-52 (HIS4::lacZ ura3-52)             Harashima Hinnebusch, 1986 <br /> H6lb                 MATa GCDJJ-503 gcn2-101 gcn3-101 hisl-29 ura3-52 (HIS4::lacZ ura3-52)             Harashima Hinnebusch, 1986 <br /> H79b                 MATa GCDJJ-504 gcn2-101 gcn3-101 his1-29 ura3-52 (HIS4::1acZ ura3-52)             Harashima Hinnebusch, 1986 <br /> H96b                 MATa gcn2-101 gcn3-101 his1-29 ura3-52 (HJS4::lacZ ura3-52)                       Harashima Hinnebusch, 1986 <br /> HI 17c               MATa gcn2-101 gcn3-101 his] -29 inol ura3-52 (HIS4::lacZ URA3+)                   Harashima Hinnebusch, 1986 <br /> H236c                MATa gcdll-505 gcn2-101 gcn3-101 hisl -29 inol ura3-52 (HIS4::lacZ URA3+)         Harashima Hinnebusch, 1986 <br /> H245C                MATa gcdll-506 gcn2-101 gcn3-101 hisl-29 inol ura3-52 (HIS4::lacZ URA3+)          Harashima Hinnebusch, 1986 <br /> H271c                MATa gcdll-507 gcn2-101 gcn3-101 hisl-29 inol ura3-52 (HIS4::lacZ URA3+)          Harashima Hinnebusch, 1986 <br /> H272c                MATa gcdll-508 gcn2-101 gcn3-101 hisl-29 inol ura3-52 (HIS4::lacZ URA3+)          Harashima Hinnebusch, 1986 <br /> H1402                MATax leu2-3 leu2-112 inol ura3-52 (HIS4::lacZ ura3-52)                           Hannig et al., 1990 <br /> TD1 17-IAd           MATa his4-303(AUU) inol-13 ura3-52 (his4(AUU)::lacZ URA3+)                        Castilho-Valavicius et al., 1990 <br /> TD1 17-1 AR7d        MATa his4-303(AUU) inol-13 ura3-52 (his4(AUU)::lacZ URA3+) SUI3-2                 Castilho-Valavicius et al., 1990 <br /> alsogenic strains. <br /> blsogenic strains. <br /> Clsogenic strains. <br /> dlsogenic strains. <br />  <br /> complex re-initiate uORFs 2-4. Re-initiation                  alleles GCDJJ originally described partially <br /> termination events uORFs 2-4 preclude scanning                    characterized (Harashima Hinnebusch, 1986; <br /> of sufficient numbers 40S ribosomal subunits                   Harashima et al., 1987). Four alleles dominant <br /> authentic GCN4 start codon (Miller Hinnebusch, 1989;                 (GCDII-50J, GCD11-502, GCDJJ-503 GCDJJ-504) <br /> Grant Hinnebusch, 1994). However, activation                  recessive (gcdll-505, gcdll-506, gcdll- <br /> GCN2 kinase amino acid starvation reduces level                   507 gcdll-508) suppressor phenotype. The <br /> of ternary complex, allowing fraction scanning 40S                  predicted 527 amino acid sequence yeast eIF-2y is <br /> ribosomes initiate terminate uORFl scan                   remarkably similar bacterial elongation factor EF-Tu <br /> through uORFs 2-4 prior re-acquiring ternary                      (Hannig et al., 1993), extensively studied GTP binding <br /> complex. 40S ribosomes re-acquire ternary com-                  protein forms ternary complex aminoacylated <br /> plex interval uORF4 GCN4 start                    elongator tRNAs. Yeast eIF-2y contains complete set of <br /> codon initiate protein synthesis GCN4 start codon,                consensus sequence elements required guanine <br /> leading 5- 10-fold increase GCN4 protein                   nucleotide binding EF-Tu GTP binding <br /> starved cells (Grant et al., 1994). The model predicts              proteins (Bourne et al., 1991). In addition, residues C- <br /> that partial loss-of-function mutations genes encoding                terminal GTP binding domain (G domain) eIF- <br /> subunits eIF-2 eIF-2B reduce level                    2y correspond residues domains II III the <br /> ternary complex mimic starvation conditions                   EF-Tu crystal structure similar protein <span id='am-69' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> <br /> lead constitutive derepression GCN4 translation.                   compatible similar domain structure the <br />    In yeast Saccharomyces cerevisiae essential                   yeast protein (Hannig et al., 1993; Gaspar et al., 1994). <br /> genes SUI2, SUI3 GCDJJ encode (37 kDa),                         In report identified molecular defect <br /> (36 kDa) y (58 kDa) subunits eIF-2 respectively                   present original gcdll alleles. Seven <br /> (Donahue et al., 1988; Cigan et al., 1989; Hannig et al.,                alleles contain single amino acid substitution (six <br /> 1993). Mutant alleles SUI2 SUI3 originally                   unique) single allele (gcdll-508) contains a <br /> isolated genetic selection suppressors                    transposable (Ty) element inserted near C-terminus <br /> initiation codon mutation yeast HIS4 gene (Donahue                extends GCDJJ ORF 30 amino acids. Strains <br /> et al., 1988; Cigan et al., 1989) subsequently                  harboring mutant alleles GCDJJ exhibit characteristics <br /> shown confer constitutive derepression GCN4 expres-                consistent global defects protein synthesis initiation <br /> sion (Willliams et al., 1989). Mutations GCDJI                   constitutively increase translation GCN4 <br /> isolated suppressors inability derepress                mRNA. A single allele, gcdl]-R5JOH, alters start site <br /> yeast general control pathway gcn2 gcn3 double                      protein synthesis tested using mutant his4 <br /> mutant (Harashima Hinnebusch, 1986). Eight mutant                    allele lacks functional AUG codon. R5 10 maps to <br />  <br /> 2240 <br />                                                                                            Mutations yeast gene encoding eIF-2y <br />  <br /> domain III modeled eIF-2y structure, suggesting          S.c. <br />                                                                 S .p <br />                                                                          1   msdlqdqeps iiingnlepv gepdiveete vvaqetqetq dadkpkkkva ftgleedget <br /> that region eIF-2y function monitoring           H.s.     1 <br />                                                                                                    .......... <br />                                                                                                    . <br />  <br />                                                                                                                                                         m <br />  <br /> interactions ribosomal P (peptidyl) site                                                               (GCD11-502,504) N <br /> initiation process.                                             S.c.    61    eeekrkrefe eggglpeqpl NpDfSKLnPL SaEIInRQAT INIGTIGHVA HGKSTVVrAI <br />                                                                 S.p.     1   ..........         mae NLDiSeLsPi hpaIISRQAT INIGTIGHVA HGKSTVVKAI <br />                                                                 H.s.     2    aggeagvtlg qphlsrqdlt tLDvtKLtPL ShEvISRQAT INIGTIGHVA HGKSTVVKAI <br /> Results                                                         Consensus                        ---   --   D-- -L-P   ---- I-RQAT INIGTIGHVA HGKSTVV-AI <br />  <br />                                                                        (GCD11-501) L      (gcdll-506) H <br /> Identification molecular defects gcdll                S.c.   121    SGVqTVRFKd ELERNITIKL GYANAKIYKC qePtCPePdC YRSfkSdKEi sPkCqrPGcP <br /> alleles                                                         S.p. <br />                                                                 H.s. <br />                                                                         44 <br />                                                                         62 <br />                                                                               SGVHTVRFKN ELERNITIKL GYANAKIYKC sneeCPRPgC YRSysSnKED hPPCeIcnsP <br />                                                                               SGVHTVRFKN ELERNITIKL GYANAKIYK1 ddPsCPRPeC YRScgSstpD efPtdIPGtk <br /> Eight mutant alleles GCDJJ originally isolated          Consensus    SGV-TVREK- ELERNITIL GYANAKIYX- ----CP-P-C YRS--S----             ---------- <br />  <br />  <br /> as suppressors 3-AT-sensitive phenotype conferred <br />                                                                 S.c.   181   GrykLVRHVS FVDCPGHDIL MsTMLsGAAV MDAALLLIAG NESCPQPQTS EHLAAIEIMK <br /> by mutations positive regulators general control      S.p. <br />                                                                 H.s. <br />                                                                        104 <br />                                                                        122 <br />                                                                              ..mnLVRHVS FVDCPGHDIL MATMLNGAAV MDAALLLIAG NESCPQPQTS EHLAAIEIMq <br />                                                                              GnfrLVRHVS FVDCPGHDIL MATMLNGAAV MDAALLLIAG NESCPQPQTS EHLAAIEIMK <br /> encoded GCN2 GCN3 (Table I; Harashima                Consensus     ----LVRBVS IVDCPGKEIL K-THL-GAAV 3DAALLLIAG NESCPQPQTS EHLAAIEIM- <br /> Hinnebusch, 1986; Harashima et al., 1987). The gcdll <br /> mutations original isolates conferred varying degrees    S.c. <br />                                                                 S.p. <br />                                                                        241 <br />                                                                        162 <br />                                                                              LKHvIILQNK VDLmREesAl EHqkSILKFI rGTiAdGAPI VPISAQLKYN IDAVnEfIVK <br />                                                                              LKHIIILQNK VDLiRESaAe EHYqSILKFI kGTVAEnsPI VPISAQLKYN IDAulEYIVK <br /> of slow growth (slg-) constitutive derepression        H.s.   182   LKHI1ILQNK iDLvkESqAk EqYeqILaFv qGTVAEGAPI iPISAQLKYN IevVcEYIVK <br />                                                                 Consensus    LH--ILQIMK-DL--E--A- E----IL-F- -GT A---PI -PISAQLKYN I----E-IVK <br /> of general control-regulated genes, suggesting multiple <br /> alleles represented. We cloned allele two-       S.c. <br />                                                                 S.p. <br />                                                                        301 <br />                                                                        222 <br />                                                                              tIPVPPRDFm isPRLIVIRS FDVNKPGAEi eDLKGGVAGG SILnGVfKLG DEIEIRPGIV <br />                                                                              KIPiPvRDFT taPRLIVIRS FDVNKPGAEV DDLKGGVAGG SILtGVLrLn DEIEIRPGIV <br /> step integration/excision method (Rothstein, 1991)          H.s.   242   KIPVPPRDFT sePRLIVIRS FDVNKPGcEV DDLKGGVAGG SILkGVLKvG qEIEvRPGIV <br />  <br /> determined molecular defect allele DNA           Consensus    -IP-P-RDF- --PRLIVIRS FDVNKP-E- -DLKGGVAGG SIL-GV---- -EIE-RPGIV <br />  <br /> sequence analysis 2.1 kb HindIll-SnaBI restriction                                    (gcdll-505) K       (gcdll-507) A <br />  <br /> fragment includes complete GCDJJ open reading          S.c. <br />                                                                 S.p. <br />                                                                        361 <br />                                                                        282 <br />                                                                              TKDDkGKIqC KPIFSnIVSL FAEqNDLKfA VPGGLIGVGT KVDPTLCRAD RLVGQVvGAK <br />                                                                              TKDDdGrIrC qPIFSrIiSL FAEHNDLKiA VPGGLIGVGT tVDPTLCRAD RLVGQVLGsK <br /> frame. The mutations corresponding allele assign-     H.s.   302   sKDseGKlmC KPIFSkIVSL FAEHNDLqyA aPGGLIGVGT KiDPTLCRAD RmVGQVLGAv <br />  <br /> ments follows: GCDII-501 (R127L, CGU-CUU);               Consensus    -KD--G---C -PIFS-I-SL FAE-NDL--A -PGGLIGVGT --DPTLCRAD R-VGQV-G-- <br />  <br /> GCDJJ-502 -504 (H108N, CAU-&gt;AAU); GCDJJ-                    S.c.   421   GhLPnIYTdi EINYFLLRRL LGVKTdGqK. qAKVrKLepN EVLMVNIGSt aTGaRVvAVK <br />                                                                 S.p.   342   GnLPEvYTEL EINYFLLRRL LGVKs .GDKn ttKVQKLaKN EVLMVNIGS. STGGRVmmVK <br /> 503 (R5lOH, CGU-4CAU); gcdll-505 (E383K, GAA-o                  H.s.   362   GaLPEIfTEL EIsYFLLRRL LGVrTeGDKKEaAKVQKLsKN EVLMVNIGS1 STGGRVsAVK <br /> AAA); gcdll-506 (Y142H, UAC-4CAC); gcdll-507                    Consensus    G-LP---T-- EI-YFLLRRL LGV---G-K- --ICV-IL--N EVIMVNIGS- -TG-RV--Vy <br />  <br /> (G397A, GGU-&gt;GCU). gcdll-508 contains Ty element                                                     (GCD11-503) H             (gcdll-508) (insertion) <br /> (Clare Farabaugh, 1985) inserted codon 525 (out          S.c <br />                                                                 s.p. <br />                                                                        480 <br />                                                                       401 <br />                                                                              ADMArlqLTs PACTEInEKI ALSRRIEKHW <br />                                                                              ADMAKIlLTa PACTEIGEKv ALSRRIEKHW <br />                                                                                                                        RLIGWAtIkk GtTLePia.. <br />                                                                                                                        RLIGWAkvve GkTLKv.... <br /> of 527). The insertion alters encoded amino acid            H.s.  422    ADlgKIvLTn PvCTEvGEKI ALSRRvEKHW          RLIGWgqIrr GvTiKPtvdd d <br /> sequence, beginning codon 526, VGIEINYHLL-              Consensus    AD-----LT- P-CTE--EE- ALSRR-EEW           RLIGW----- G T--- <br />  <br /> TSIYITSILSYTVLEDDANDEKstop. The missense                    Fig. 1. Location mutations eIF-2y. The amino acid changes in <br /> mutations alter residues conserved eIF-2y     seven missense alleles site insertion Ty element in <br /> homologs S.cerevisiae, <span id='am-35' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-36' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39'  ></span>Schizosaccharomyces</span> pombe           gcdll-508 shown amino acid sequence eIF-2y from <br /> and humans (Figure 1).                                          S.cerevisiae (S.c.; Hannig et al., 1993). The deduced amino acid <br />                                                                 sequences eIF-2y <span id='am-40' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> (S.p.; Erickson <br /> Effects mutations steady-state levels                  Hannig, preparation) humans (H.s.; Gaspar et al., 1994) are <br /> elF-2y                                                          shown S.cerevisiae sequence. A consensus sequence of <br />                                                                 residues conserved homologs shown bold. Three <br /> We determined gcdll alleles altered steady-    sequence elements conserved GTP binding proteins underlined. <br /> state levels eIF-2y, reduced levels eIF-2 would <br /> provide simple explanation general control <br /> and growth phenotypes mutant strains. We             presence gcn2 gcn3 <br /> measured steady-state levels eIF-2,y immunoblot       mutations present original isolates. Toward this <br /> analysis protein extracts prepared exponentially        goal, constructed isogenic set GCDII + and <br /> growing cultures original gcn2 gcn3 gcdll strains        gcdll strains GCN+ strain EY582 using plasmid <br /> and isogenic GCDII + parents (Figure 2A). Quantita-       shuffle technique. In strains, sole copy GCDJJ <br /> tion immunoblot <span id='am-108' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-109' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>laser</span> densitometry direct         harbored centromere-containing, low copy <br /> counting revealed level eIF-2y       plasmid pSB32. To insure plasmid-bome alleles <br /> six strains harboring point mutation GCDIJ differed        did alter relative steady-state levels eIF-2y in <br /> reproducibly &lt;1.5-fold isogenic wild-type           strains, measured steady-state level eIF- <br /> strains. This suggests phenotypes mutant      2yin strain immunoblot analysis. Results presented <br /> strains functional defect eIF-2y protein,   Figure 2B indicate levels eIF-2y equivalent <br /> rather defect expression GCDII. The level      GCDII+ gcdll plasmid shuffle strains (&lt;1.5- <br /> of eIF-2y reduced -2-fold gcdll-E383K strain.        fold difference) gcdll-508 strain, which <br /> In contrast, level elongated form eIF-2y       level eIF-2y reduced one-third the <br /> the gcdll-508 strain reduced -10-fold compared           GCDJJ+ strain. <br /> with isogenic wild-type strain. Thus phenotypes            During course experiments, were <br /> exhibited gcdll-508 strains result reduced          consistently unable construct strains GCN3+ <br /> levels eIF-2, intrinsic biochemical defect    EY582 background harboring pSB32-borne versions of <br /> in encoded eIF-2,y protein.                                 gcdll-R127L gcdll-E383K sole copy of <br />                                                                 GCDII. Phenotypes conferred certain alleles GCDJJ <br /> Construction isogenic gcdll GCN3 strains                     known exacerbated presence wild-type <br /> We wished examine phenotypes conferred gcdll              GCN3 (Harashima et al., 1987), encodes non- <br /> mutations GCN+ strains order rule effects     essential subunit eIF-2B (Hannig Hinnebusch, <br />  <br />                                                                                                                                                    2241 <br />  D.R.Dorris, F.L.Erickson E.M.Hannig <br />  <br />    Ait <br />  <br />  <br />  <br />  <br />   ,;7i <br />  <br />  <br />  <br />  <br />   ::-1 <br />           i- <br />           C) <br />  <br />  <br />           C <br />               11   -.- <br />  <br />  <br />                    i. <br />                            I., <br />  <br />  <br />  <br />                            -]&apos;.   ;&gt;1.   :&gt; <br />                                               ..,1- <br />                                                LLJ      I,)-, <br />                                                                 CC. <br />                                                                 C, <br />                                                                   8 <br />                                                                                        B <br />  <br />  <br />                                                                                          ! <br />                                                                                          . .: <br />                                                                                         .1:    tz:.~ <br />                                                                                               .- <br />  <br />  <br />                                                                                            45..... <br />                                                                                                     -CID <br />  <br />  <br />  <br />  <br />                                                                                               ::..:1,. <br />                                                                                                          ,,~ <br />                                                                                                           a? <br />                                                                                                                   /Nv <br />  <br />  <br />  <br />  <br />                                                                                                                         : <br />                                                                                                                                 ---, <br />                                                                                                                                 .5&gt;)a <br />                                                                                                                                 , <br />  <br />  <br />  <br />                                                                                                                                 :. <br />  <br />                                                                                                                               eIF-2-. <br />                                                                                               *WwI.. - ..:.::... .....:e::. :.. &quot;e <br />                                                                                                                                        a. <br />                                                                                                                                                     eIF-2   -.508 <br />  <br />  <br />  <br />  <br />                ~ <br />                ;~~~~~~~ <br />                 :i -&apos;13 ~ ~~~~~~~~~~~~~~~~~~~ <br />                               4     6  7 <br />  <br /> Fig. 2. Phosphorimage immunoblots showing steady-state levels eIF-2y exponentially growing GCDII gedil strains. (A) Whole cell <br /> extracts  prepared exponentially growing cultures original gcdll isolates wild-type parents. Equal amounts protein from <br /> each preparation analyzed immunoblot analysis using polyclonal antisera specific eIF-2y, detected using [125I]protein A. The extracts <br /> were prepared following strains: lane 1, H96 + Ep362 (Ep362 contains GCDII high copy plasmid YEp24); lane 2, H96; lane 3, <br /> H117; lane 4, H59; lane 5, H57; lane 6, H245; lane 7, H236; lane 8, H271; lane 9, H61; lane 10, H272. The strains used lanes 1, 2, 4, 5 9 <br /> are isogenic, strains used lanes 3, 6, 7, 8 10. (B) Whole cell extracts prepared isogenic strains, constructed EY582 <br /> by plasmid shuffling, examined (A). The strain used lane 1 harbors wild-type GCDII high copy 2g plasmid YEpl3, those <br /> used lanes 2-7 harbor low copy <span id='am-118' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> indicated GCDII alleles. The mobilities molecular weight markers (in kDa) listed in <br /> both panels left. <br />  <br />  <br /> 1988; Cigan et al., 1993). To confirm inability                    Table II. Relative growth rates isogenic GCN3 Agcn3 strains <br /> construct certain gcdll GCN strains                             harboring different gedll alleles <br /> presence  GCN3 result undefined <br /> element genetic background, used plasmid                         GCDJJ allele              Relative growth rate single colonies 30&#176;C <br /> shuffle technique construct GCDII + gcdll strains                                                 EY582a        EY606a          EY606 + Ep69b <br /> in EY606, Agcn3 strain isogenic EY582. As                                                                 (Agcn3)         (GCN3) <br /> expected, gcdll-508 allele conferred reduced growth <br /> rate EY606 relative EY582 (Hannig et al., 1993).                         GCDII                     +++++              +++++           +++++ <br /> As shown Table II, mutant alleles functioned                     High copy GCDJJ           +++++              +++++           n.d.c <br />                                                                                H108N                     +                  +               + <br /> as sole copy GCDJJ strain EY606. Introduction                        R127L                     -                  ++              +/- <br /> of low copy plasmid containing wild-type GCN3                           Y142H                     +                  ++              + <br /> the plasmid shuffle strains constructed EY606 exacer-                       E383K                     -                                  + <br /> bated slow growth defect gcdll-R127L, -Y142H                        G397A                     ++++               ++++            n.d. <br />                                                                                R510H                     +++                +++             n.d. <br /> and -E383K strains, completely alleviated                       508 (Ty insertion)        ++++               ++              ++++ <br /> growth defect gcdll-508 derivative. Our results <br /> extend studies Harashima et al. (1987) suggest                      alsogenic strains. <br /> further inability construct certain gcdll GCN3                     bEp69 low copy, autonomously replicating plasmid contains <br /> strains plasmid shuffle technique result                       wild-type GCN3 gene. <br /> antagonism encoded eIF-2y protein GCN3.                              CNot determined. <br />  <br /> Altered monosome:polysome ratios gcdll                                      The polysome profiles depicted Figure 3 increased <br /> strains                                                                        MNP ratios gcdll strains relative an <br /> Mutations structural gene eIF-2y                           isogenic GCDJJ+ strain. The data Figure 3 show <br /> expected confer global defects translation initiation.                   correlation magnitude increase the <br /> Strains defective translation initiation exhibit                      M/P ratio severity growth defect each <br /> altered polysome profiles obtained following centrifuga-                       mutant strain. However, increased M/P ratios gcdll <br /> tion extracts prepared cycloheximide-treated cells                     strains solely decreased growth rates of <br /> (to inhibit translation elongation) sucrose                            strains, relative M/P ratio isogenic <br /> gradients. In particular, strains increased                      petite GCDJJ+ strain nearly identical its <br /> monosome:polysome (M/P) ratios accumulation                         grande GCDII+ parent (1.2 versus 1.0), the <br /> of 40S 60S ribosomal subunits interact                         doubling time petite strain 2-fold greater <br /> absence mRNA form 80S &apos;couples&apos; (Cigan et al.,                           (3.2 versus 1.8 h). In addition, doubling time the <br /> 1991). Under conditions used, 80S couples co-                        gcdll-R5JOH strain equivalent petite <br /> fractionate monosomes bound mRNA.                                      GCDJJ + strain, shows 2.3-fold increase <br />    We asked isogenic gcdll GCN3 strains                            M/P ratio relative petite strain. The polysome <br /> we constructed showed increased M/P ratios                           profiles strains harboring missense mutations in <br /> consistent global defect translation initiation.                     GCDJJ consistent global defect in <br /> 2242 <br />                                                                                                Mutations yeast gene encoding eIF-2y <br />  <br />            2.0                                                                                f-Galactosidase activity (U) GCN4-lacZ fusions <br />                                                 GCDII <br />                                                 M/P=1.0 <br />          1.0                 60S    SOS       T=1.8 hr <br />       0                                                                                              p180                p226                p227 <br />                                                                               Plasmid-borne <br />                                                                               GCD11 allele      R           DR     R            DR     R            DR <br />                                       80S <br />            2.0                                                                H108N            921          1012   104          121   2335      2326 <br />                                       + <br />                                                gcdll-HI08N                    Y142H            804          788    84           90    2465      1652 <br />                                                M/P=5.3 <br />       0 <br />       0    1.0                      60S       ~~~T=6.4 <br />                                                     hr                        G397A            273          424     11          37    1321      1357 <br />                                                                               R510H            277          518    25           67    1518      1437 <br />                                                                               -508             116          316    20           38    1387      1470 <br />                                       80S                                     wt               23           415    10           47    936       1316 <br />            2.0 <br />                                                gcdll-Y142H                    high copy wt      52          249    13           46    888       1110 <br />                                                M/P=5.4 <br />       O <br />       0 <br />            1.0                     aos        ~~~T=5.7 hr                 Fig. 4. ,B-Galactosidase activities gcdll strains containing <br />                         40S    4 <br />                                                                           GCN4-lacZ fusions. Plasmids p180, p226 p227, contain a <br />                                                                           GCN4-lacZ fusion upstream open reading frames <br />            2.0                                                            (uORFs), uORF4 uORFs respectively represented by <br />                                                gcdll-G397A                diagrams. A box represents open reading frame X <br />                  z80S     M/P=1.5 <br />       0    1.0   \60S +T=2.3 hr                                           represents point mutation removes start codon the <br />       0 <br />                         40S                                               corresponding open reading frame. These plasmids introduced <br />                                                                           strains harboring different GCDII alleles (shown first <br />                                                                           column) low copy plasmid. Cultures 3-5 independent <br />                                                                           transformants grown repressing (R) derepressing (DR) <br />            2.0                                                            conditions analyzed described Materials methods. Units <br />                  l80S                          gcdll-R510H <br />                  l +                           M/P=2.8                    f-galactosidase activity given nmol o-nitrophenyl-f-D- <br />       o    1.0                 60S             T=3.3 hr                   galactopyranoside (ONPG) cleaved/min/mg total protein. The activities <br />       0                 40S                                               given average activities 3-5 transformants, which <br />                                      )                                    differed &lt;30% average. <br />  <br />            2.0 <br />                                                gcdll-508                  GCN4 mRNA (Mueller et al., 1987). We extended <br />                                      805 <br />                                                M/P=1.3                    analyses mutations GCDJJ specific- <br />                                       ,, <br />       o    1.0                60S <br />                               4 <br />                                                T=1.9 hr                   ally alter GCN4 expression translational level in <br />       0                40S <br />                                                                           GCN+ strains altered expression mutant <br />                                                                           strains requires short ORFs present GCN4 <br />                                                                           mRNA leader. For experiments used isogenic <br />            2.0                                 GCDII pet <br />                                                M/P=1.2 <br />                                                                           GCDJJ + gcdll strains constructed plasmid <br />        S                       0S 80S          T=3.2 hr                   shuffle technique GCN+ strain EY582. We intro- <br />       o    1.0 <br />       0                40S    4      *                                    duced strains low copy plasmid containing <br />                                                                           GCN4-lacZ alleles differ the <br />                                                                           region encoding GCN4 mRNA leader. These alleles <br /> Fig. 3. Polysome profile analysis GCDIJ gedli strains. Cell        contained uORFs (p180), uORF4 only <br /> extracts prepared cycloheximide-treated (50 gg/ml)              (p226) uORFs (p227). In case, missing <br /> exponentially growing cultures isogenic plasmid shuffle strains        ORFs (indicated X Figure 4) removed by <br /> harboring listed GCDII allele sole copy. Sixteen OD260 units <br /> from extract layered 7-47% sucrose gradients,                introducing point mutations AUG codon the <br /> centrifuged 2 h 200 000 g fractions collected        uORF (Mueller Hinnebusch, 1986). <br /> constant monitoring OD254. The monosome:polysome ratio (M/P)              Figure 4 shows results assays P-galactosidase <br /> was calculated dividing area 80S peak area        activity used model GCN4 expression both <br /> under 2mer-Smer peaks normalized ratio              repressing (R) amino acid starvation (DR, derepress- <br /> GCDII+ strain, set 1. The peaks corresponding the <br /> 40S, 60S 80S labeled accordingly. OD254 optical density    ing) conditions. P-Galactosidase activity p180 trans- <br /> at 254 nm. The doubling time (X) strain determined        formants GCDII + strain increased -20-fold under <br /> rich medium monitoring optical density 600 nm.                  amino acid starvation conditions. P-Galactosidase activity <br />                                                                           expressed GCN4-lacZ allele lacks four <br /> protein synthesis initiation. The gcdll-508 GCN3+ strain                  uORFs (p227), used model changes level the <br /> shows M/P ratio similar wild-type control               GCN4-lacZ transcript (Mueller Hinnebusch, 1986), <br /> (1.2 versus 1.0), isogenic gcdll-508 Agcn3 strain                  increased 1.4-fold starved cells. Taken together, <br /> has M/P ratio 2.6 (data shown). The difference                   data indicate increased f-galactosidase <br /> in polysome profiles growth rates                     activity expressed p180 starved cells due <br /> two gcdll-508 strains suggests GCN3 contribute                   primarily increase translation GCN4-lacZ <br /> to function eIF-2y gcdll-508 strains.                           mRNA. The constitutively low levels P-galactosidase <br />                                                                           activity p226 (uORF4 only) transformants suggests <br /> Mutations GCDII alter translational                                scanning mechanism translation initiation is <br /> regulation GCN4 expression                                             altered starvation conditions. These results are <br /> Harashima Hinnebusch (1986) demonstrated previously                   agreement previous results using related wild-type <br /> that GCN4 expression increased constitutively                         strains (Mueller Hinnebusch, 1986). <br /> GCDII-501 gcn2 gcn3 yeast strain. This increased expres-                      p180 transformants gedli strains constitu- <br /> sion later shown to increased translation                 tively increased GCN4-lacZ activity. In cases, <br />                                                                                                                                                          2243 <br />  D.R.Dorris, F.L.Erickson E.M.Hannig <br />  <br />  <br />                                                                                                                          His +/ - in <br />                                           HIS4 -IacZ           AUG         his4    -G_IacZ      UUG       UUG ratio I      his4Au <br />                      Relevant allele      p-gal activity (U)   ratio     P-gal activity (U)     ratio       AUG ratio       strain <br />  <br />                  gcdl 1-H108N                 5030            5.6               449           18.7           3.3             - <br />  <br />                  gcd11Y142H                   4990            5.5               508           21.1           3.8             - <br />  <br />                  gcdl 1-E383K                  ndc             nd                nd            nd            nd              - <br />  <br />                  gcdI1-G397A                  2220            2.5               129            5.3          2.2              - <br />  <br />                  gcdl 1-R510H                 2607            2.9               724           30.1          10.4             + <br />                  gcd11-508                    1915            2.1               59             2.4           1.1 <br />                  uR                           901             1.0                24            1.0           1.0 <br />                  high COpy GCDI <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ia</span>              1110            1.2                39            1.6           1.3 <br />                  SUI3 b                         528             1.0                7             1.0           1.0 <br />                  SUI3-2 b                       1346            2.5               637           89.9          35.3            + <br />  <br />                  a <br />                  b <br />                    Isogenic strains. <br />                    Isogenic strains. <br />                  c nd - determined. <br />  <br /> Fig. 5. f3-Galactosidase activities gcdll strains harboring low copy plasmids containing HIS4AUG-IacZ his4UUG_1acZ. Isogenic slasmid <br /> shuffle strains harboring various alleles GCDII transformed low copy plasmids containing HIS4AUG-IacZ his4Uu -lacZ. <br /> The constructs identical single base change his4UUG_1acZ fusion alters start codon AUG UUG. <br />  3-Galactosidase activities determined 3-5 independent transformants described legend Figure 4. The AUG ratio the <br /> P-galactosidase activity strains harboring HIS4AUG-lacZ plasmid divided activity isogenic wild-type strain. Likewise, UUG <br /> ratio ratio P-galactosidase activity strain harboring his4UUG-lacZ plasmid activity wild-type strain harboring same <br /> plasmid. The UUG ratio/AUG ratio UUG ratio divided AUG ratio. The column shows ability allele suppress a <br /> his4AUU allele allow growth media lacking histidine. Plasmids containing gcdll alleles listed column the <br /> sole copy GCDJJ EY647 (his4AUU) plasmid shuffling. The rows right column results following introduction a <br /> low copy plasmid containing SUI3 SUI3-2 GCDII+ EY647 derivative. <br />  <br />  expression p180 increases 5- 40-fold non-                             trans alter selection start site protein <br />  starved cells relative GCDII+ strain. Conversely,                          synthesis (Donahue et al., 1988; Cigan et al., 1989; <br /> expression GCN4-lacZ p227 gcdll strains                             Castilho-Valavicius et al., 1990, 1992). Using alleles <br />  shows small increase 1.5- 2.6-fold. Thus                              genetic selection His&apos; revertants, identified <br /> constitutive expression GCN4 gcdll strains                                mutations genes encoding (SUI2) 3 (SUI3) <br /> primarily constitutive increase translation                         subunits eIF-2 shown allow translation <br />  GCN4 mRNA. Qualitatively, degree derepression                              initiation in-frame UUG codon (Donahue et al., <br /> in unstarved cells (Figure 4, p180, R) correlates                        1988; Cigan et al., 1989; Yoon Donahue, 1992). These <br /> the growth rate defect increased M/P ratios                            alleles constitutively derepress GCN4 expression <br /> mutant strains. Note smallest increase                                (Williams et al., 1989). We asked mutations GCDII, <br /> GCN4-lacZ expression occurs gcdll-508 strain,                              isolated based general control phenotype, could <br /> yet strain exhibits greatest change level                      alter start site selection allow expression P- <br /> eIF-2y protein (a reduction one-third wild-type;                            galactosidase his4UUG-lacZ allele the <br /> Figure 2B). This supports notion                             normal HIS4 AUG codon replaced UUG codon. <br /> defects seen gcdll missense strains result                            In his4UuG-lacZ construct, initiation downstream <br /> functional defects encoded eIF-2y proteins. p226                           AUG codons rise protein product with <br /> transformants gedl] strains greatly reduced <br /> levels GCN4-lacZ expression, suggesting                               ,3-galactosidase activity (Donahue Cigan, 1988). As <br /> scanning mechanism remains intact gcdll strains.                               control, measured expression HIS4AUG-lacZ. <br /> Expression GCN4-lacZ p226 greatest                              This necessary HIS4 - lacZ transla- <br /> gcdll-H108N gcdll-YJ42H strains,                               tional fusions expressed yeast HIS4 promoter <br /> the highly derepressing gcdll alleles examined.                       subject transcriptional regulation by <br /> Elevated expression p226 suggests defective                         general control pathway. <br /> eIF-2 complexes strains prone                                 Figure 5 shows results 3-galactosidase assays <br /> to increased &apos;leaky scanning&apos; allows scanning                            using extracts prepared isogenic GCDJJ + and <br /> ribosomes bypass highly inhibitory uORF4                              gcdll strains harboring low copy plasmid containing <br /> frequently strains.                                                 HIS4AUG-lacZ his4UUG-lacZ. All gcdll <br />                                                                                   strains showed increased 0-galactosidase activity the <br /> Initiation non-AUG codons                                                      HIS4AUG-lacZ transformants compared the <br /> Donahue colleagues used his4 alleles lacking func-                          GCDII+ strains. The increased activity reflection of <br /> tional AUG codon isolate mutations genes act                           increase level HIS4-lacZ transcript due <br />  <br /> 2244 <br />                                                                                    Mutations yeast gene encoding eIF-2y <br />  <br /> to constitutive derepression general control       derepression GCN4 expression sufficient alter <br /> gcdll strains. When normalized activity wild-type      recognition start site protein synthesis. <br /> strain (Figure 5, AUG ratio), derepression response is <br /> seen vary 2- 5.6-fold unstarved cells,          Discussion <br /> agreement previous studies (Harashima and <br /> Hinnebusch, 1986). In his4UUG- lacZ transformants,           We identified molecular defects present eight <br /> the increased P-galactosidase activity gcdll strains   mutant alleles GCDJJ (Harashima Hinnebusch, <br /> presumably combined effect transcriptional activation        1986; Harashima et al., 1987). The defects include six <br /> from HIS4 promoter increased initiation non-        unique point mutations single Ty element insertion. <br /> AUG start codon. Initiation non-AUG codons             With exception gcdll-508, gcdll alleles <br /> estimated ratio normalized ,-galactosidase         leads significant reduction steady-state level of <br /> expression his4UUG-lacZ divided                eIF-2,y, suggesting phenotypes latter <br /> HIS4AUG-lacZ (Figure 5, UUG ratio/AUG ratio). As               strains simply overall reduction the <br /> control, performed similar analysis using isogenic          level eIF-2. Collectively, gcdll alleles confer an <br /> SUI3 SUI3-2 strains (Donahue et al., 1988; Castilho-         unconditional slow growth phenotype constitutively <br /> Valavicius et al., 1990). The SUI3-2 strain showed 35-         derepress general control pathway. The gcdll strains <br /> fold increase (UUG ratio/AUG ratio) expression           examined exhibit increased monosome:polysome <br /> his4UUG-lacZ, represents initiation non-AUG             ratios consistent general defect protein synthesis <br /> codons.                                                          initiation specifically alter translational regulation <br />    Among gcdll strains, gcdll-RSOH strain                GCN4 expression. <br /> showed largest increase (10.4-fold) expression            The analysis mutations GCDJJ facilitated by <br /> P-galactosidase activity his4UUG-lacZ relative           similarities predicted amino acid sequences <br /> HIS4AUG_lacZ. Five additional gcdll alleles showed             eIF-2y bacterial elongation factor EF-Tu. Structures <br /> &lt;4-fold increase expression his4UuG-lacZ            EF-Tu p21 ras GDP- GTP-bound <br /> may accounted increased level               states determined based X-ray crystallo- <br /> his4-lacZ transcript (note that, gcdll-R5JOH,         graphic studies (de Vos et al., 1988; Pai et al., 1989; Tong <br /> the highly derepressing gcdll alleles higher       et al., 1989; Kjeldgaard Nyborg, 1992; Berchtold <br /> UUG ratio/AUG ratio values). Northern blot analysis (data        et al., 1993; Kjeldgaard et al., 1993). Although G <br /> not shown) revealed individual gcdll allele,         domains EF-Tu p21ras share 16% amino <br /> the steady-state level his4-lacZ mRNA equivalent          acid sequence identity, x-carbon backbones the <br /> in his4UUG- lacZ HIS4AUG-lacZtransformants.          respective GTP binding domains nearly superimpos- <br /> As expected, lacZ transcript levels increased        able (<span id='am-103' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-104' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107'  ></span>Valencia</span> et al., 199 1 ). eIF-2y exhibits higher <br /> the gcdll mutants compared GCDII + strain. In           degree sequence similarity EF-Tu G domain <br /> addition, level his4UUG-lacZ transcript -2-       (34% identical, 49% similar) considerable similarity <br /> fold higher gcdll-H108N strain relative            domains II III (19% identical, 33% similar) that <br /> gcdll-RSJOH strain, his4UUG-lacZ activity            C-terminal G domain (domain I) proteins <br /> reduced 40% strain. This supports           (Hannig et al., 1993; Gaspar et al., 1994). We used <br /> hypothesis increase lacZ expression         structure EF-Tu-GDP (Kjeldgaard Nyborg, <br /> his4UUG-lacZ fusion gcdll-RSJOH strain          1992) model illustrate location residues in <br /> increased translation initiation non-AUG codons.              eIF-2y altered gcdll alleles described in <br />    To examine physiological significance increased        report (Figure 6). The gcdll mutations map regions <br /> expression his4UUG-lacZ gcdll strains,              corresponding domain I (gcdll-H108N, gcdll-R127L <br /> constructed set isogenic GCDII+ gcdll strains           gcdll-Y142H), domain II (gcdll-E383K, gcdll- <br /> by plasmid shuffle technique His- yeast strain        G397A) domain III (gcdll-R5JOH gcdll-508) of <br /> EY647. This strain harbors his4AUu allele HIS4        EF-Tu structure. None alleles alter residues <br /> locus. In background, GCDII+ strains histidine          88 amino acid N-terminal extension yeast eIF- <br /> auxotrophs. As control, dominant SUI3-2 allele,            2y absent EF-Tu. This N-terminal extension is <br /> when introduced low copy plasmid (p364) yeast          conserved human S.pombe eIF-2y homologs <br /> strain EY647 (GCDJJ+), suppressed his4AUu mutation           (Figure 1) serve species-specific <br /> and allowed growth absence histidine (Figure 5,        function. <br /> right column). As expected, introduction wild-type               His 108 yeast eIF-2y located first <br /> SUI3 low copy plasmid failed suppress his4AUu        conserved motif (G-X4-G-K-S/T) GTP binding <br /> mutation. Strains harboring gcdll-H108N, gcdll-              proteins conserved second position this <br /> Y142H, gcdll-E383K, gcdll-G397A gcdll-508 alleles             motif GTP binding translation factors (Bourne et al., <br /> were histidine auxotrophs. However, gcdll-RSJOH              1991). This sequence element located loop region <br /> allele suppressed his4AUu mutation allowed growth        chains residues loop contact P- <br /> on media lacking histidine. This strain His- prior           phosphate guanine nucleotide EF-Tu-GDP. The <br /> to plasmid shuffling (to remove GCDII + allele),             analogous residue EF-Tu H19, interacts with <br /> suggesting suppressor phenotype recessive.           Ql 15 (conserved Q228 eIF-2y) a3-<span id='am-54' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-55' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span>helix</span> of <br /> These results indicate eIF-2y participates recogni-      domain I EF-Tu-GDP structure (Kjeldgaard and <br /> tion start codon eIF-2 initiation protein    Nyborg, 1992). Mutations alter amino acids sur- <br /> synthesis. These results demonstrate           rounding H 19 EF-Tu analogous residue p21ras <br /> partial loss-of-function y subunit eIF-2 strong    (Al l) reduce ability proteins hydrolyze GTP <br />  <br />                                                                                                                         2245 <br />  D.R.Dorris, F.L.Erickson E.M.Hannig <br />  <br />  <br />  <br />  <br /> Fig. 6. Ribbon model EF-Tu-GDP showing relative locations amino acid residues eIF-2y altered gcdll strains. The three <br /> structural domains EF-Tu bound GDP molecule shown. The location N-terminal insertion eIF-2y (relative EF-Tu) is <br /> indicated. The positions amino acid substitutions eIF-2y shown black. The mutations domain II shown, greater <br /> degree uncertainty aligning residues region. Note absence L2 loop, coordinates structure obtained from <br /> trypsin-treated EF-Tu (Kjeldgaard Nyborg, 1992). <br />  <br />  <br /> (Gibbs et al., 1984; Jacquet Parmeggiani, 1988). In                        alters residue [(124)QTVRFKDELERNITI(138)] a <br /> the initiation reaction, GTP hydrolysis eIF-2 required                   region near junction C-terminal end al- <br /> for formation 80S complexes (Merrick, 1992). In                        helix G domain loop L2 (the putative effector <br /> respect, increased expression GCN4-lacZ                            loop) EF-Tu structure (Gaspar et al., 1994; our <br /> both p180 (all uORFs) p226 (uORF4 only)                            unpublished observations). The eIF-2y sequence 13 <br /> gcdll-HJ08N strains explained leaky scanning                       amino acids shorter EF-Tu sequence region <br /> at uORFs inefficient initiation pre-                             (Kjeldgaard Nyborg, 1992) assignment <br /> initiation complexes containing eIF-2 [eIF-2y(H108N)].                         R127 al-helix tentative. It possible this <br /> Alternatively, accumulation ribosomes stalling                       helix shorter eIF-2y compared EF-Tu, the <br /> AUG codons reduce overall level ternary                           corresponding a-helix p2lras (Pai et al., 1989; Valencia <br /> complex strains lead increased                         et al., 1991). In case, R127 form portion of <br /> GCN4 translation.                                                              loop L2. It note sequences the <br />    The gcdll-YJ42H mutation alters tyrosine residue                       putative al-helix human eIF-2y G domain be <br /> the 52-strand immediately follows putative effector                   cross-linked initiator Met-tRNAiMet (Gaspar et al., <br /> loop [loop L2, (136)ITIKLGYAN(144)]. The correspond-                           1994). gcdll-R5JOH, discussed below, <br /> ing amino acid residue EF-Tu His66 [(60)ITINT-                           alters residue [(5 1O)RLIGWATIKK(519), EF-Tu residues <br /> SHVE(68)], located the cleft                         (381 )RTVGAGVVAK(390)] maps near loop con- <br /> between domains I III EF-Tu-GTP crystal                             necting 35- P6-strands domain III C- <br /> structure (Berchtold et al., 1993; Kjeldgaard et al., 1993)                    terminus EF-Tu. As shown Figure 6, have <br /> and cross-linked tRNA (Duffy et al., 1981;                           modeled loop eIF-2y Y142. <br /> Metz-Boutigue et al., 1989). This histidine residue                     The corresponding face EF-Tu EF-la (toward the <br /> conserved EF-la, functional homolog EF-Tu                         viewer Figure 6) implicated tRNA binding <br /> eukaryotes, possibility remains                    using cross-linking chemical protection studies (Kinzy <br /> residue close proximity tRNA                       et al., 1992). In eIF-2y residues (Y142 R510) <br /> direct contact (Kinzy et al., 1992). We observed                        involved tRNA binding or, perhaps, interactions <br /> slight reduction relative eIF-2y(Y142H)                       proteins eIF-2p, be <br /> that ribosome bound eIF-2 complex (data                           cross-linked GTP Met-tRNA,Met (Anthony et al., <br /> shown), consistent possibility eIF-2y(Y142H)                     1990). We attempting purify eIF-2 containing altered <br /> might affect binding Met-tRNAiMet. gcdll-R127L                          y subunits biochemical analyses provide further <br />  <br /> 2246 <br />                                                                                       Mutations yeast gene encoding           eIF-2y <br /> insight importance residues eIF-2          2B subunits yeast provide information critical for <br /> function.                                                        understanding molecular basis essential and <br />    Previous studies provided genetic evidence          regulatory interactions eIF-2 eIF-2B. <br /> the (SUI2) 3 (SUI3) subunits eIF-2 (Donahue <br /> et al., 1988; Cigan et al., 1989; Castilho-Valavicius et al., <br /> 1990, 1992), initiation factor SUI 1 (Yoon Donahue,      Materials methods <br /> 1992), tRNA1Met (Cigan et al., 1988) function             Strains genetic techniques <br /> recognition start site protein synthesis. Our         <span id='am-75' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-76' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> strain DH5a used propagate plasmid DNAs that <br /> results indicate eIF-2y involved selection       introduced standard methods (Sambrook et al., 1989). Plasmid <br /> of start site eIF-2. A single gcdll allele (gcdll-        DNA introduced yeast strains alkali cation method (Ito <br />                                                                  et al., 1983). 5-Fluoroorotic acid (5-<span id='am-6' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>) purchased PCR <br /> RSJOH) alters recognition start site protein          Incorporated. 3-Amino-1,2,4-triazole (3-AT) purchased Sigma. <br /> synthesis. Additional alleles confer similar phenotypes     Additional media used growth yeast strains prepared as <br /> have recently isolated (S.Rippel E.M.Hannig,            described (Sherman et al., 1979). <br /> unpublished data). The mutant suil SUI3 alleles            Yeast strains used experiments listed Table I. All are <br /> been shown allow initiation UUG codons (Yoon           derived strain S288C using standard genetic techniques (Sherman <br />                                                                  et al., 1979). EY305 derived H236 crossing EY159. <br /> Donahue, 1992). The mechanism gcdll-RSJOH               EY555 EY558 obtained ascospores crosses H96 <br /> suppresses his4 alleles lack functional AUG codon         H271 H245 respectively. EY582 used construct isogenic <br /> is currently study. Relative gcdll alleles    gcdli strains plasmid shuffle technique obtained as <br /> examined, gcdll-R5JOH moderately derepresses                follows. A 6.4 kb EcoRI-SalI restriction fragment containing the <br /> GCN4 translation (Figure 4) ability         gcdil::hisG-URA3-hisG (AI-527) allele purified plasmid <br />                                                                  Ep523 (see below) introduced Ura- diploid strain formed by <br /> initiate non-AUG codons simply related          mating strains H4 H1402. DNA purified Ura+ yeast <br /> degree partial loss function (as judged growth     transformants, digested EcoRI analyzed Southern blot <br /> rates monosome:polysome ratios).                             analysis using radiolabeled 4.3 kb BamHI-BglIl restriction fragment <br />    Our analyses gcdll alleles isogenic GCN3            Ep264, containing GCDII gene, probe (Feinberg and <br />                                                                  Vogelstein, 1983; Hannig et al., 1993). Diploid strains containing a <br /> Agcn3 strains (Table II) confirm extend previous             single copy disrupted allele chromosomal <br /> observations GCN3 (encoding non-essential subunit         GCDJJ loci plated media containing 5-FOA select for <br /> of eIF-2B) exacerbates slow growth phenotype              Ura- strains lost URA3 gene result recombination <br /> gcdll-R127L, gcdll-Y142H gcdll-E383K strains.                hisG direct repeats (5-FOA toxic URA3+ cells; Boeke <br />                                                                 et al., 1987). The recombination event confirmed Southern <br /> Conversely, introduction GCN3 Agcn3 gcdll-        blot analysis Ura- derivatives (data shown). The YCp5O/ <br /> 508 strain ameliorates slow growth phenotype        GCDJJ(URA3+) plasmid Ep293 (Hannig et al., 1993) intro- <br /> strain, stabilizing gcdl 1-508 protein (but      duced Ura- diploid strain Ura+ transformants selected <br /> not wild-type levels; Figure 2). In contrast,        sporulated. EY582 EY585 obtained Ura+ ascospores <br /> presence GCN3 observable effect growth         inviable tested 5-FOA medium, indicating the <br /> of gcdll-H108N, gcdll-G397A gcdll-R5JOH strains.            plasmid Ep293 harbored sole copy GCDJ1 strain. EY636 <br />                                                                 harbors wild-type GCDII allele pSB32 (Rose Broach, 1991) <br /> Various explanations account genetic            constructed EY582 using plasmid shuffle technique <br /> interactions. gcdll alleles affected presence/absence    (see below). <br /> of GCN3 directly affect interactions eIF-            EY606 lacks entire GCN3 open reading frame constructed <br /> 2y eIF-2B or, alternatively, alter quaternary     introducing 7 kb EcoRI-PvuII fragment containing gcn3::hisG- <br />                                                                 URA3-hisG (AJ-305) allele Ep545 EY636. Chromosomal <br /> structure eIF-2 indirectly affect interactions   DNA prepared Ura+ transformants EY636 digested with <br /> of eIF-2 exchange factor. Hinnebusch col-          EcoRI examined Southern blot analysis, using radiolabeled <br /> leagues proposed presence GCN3 increases       4 kb GCN3 fragment Epl 86, identify strains GCN3 <br /> the stability eIF-2(a-PO4)-eIF-2B complexes (Vasquez         locus disrupted. Ura+ transformants patched onto <br /> de Aldana et al., 1993). In similar fashion,             5-FOA media Ura- derivative selected analyzed by <br />                                                                 Southern blot analysis above. Plasmid Ep293 introduced this <br /> altered eIF-2y proteins act stabilize eIF-2-eIF-       strain Ura+ transformants selected. Following growth media <br /> 2B complexes (mimicking eIF-2a phosphorylation),          lacking uracil, EY606 obtained Ura+ Leu- colony had <br /> others hinder formation active eIF-2-eIF-2B           lost resident pSB32(LEU2+)/GCDl 1 plasmid Ep552. EY606 is <br />                                                                 sensitive 30 mM 3-AT (a Gcn- phenotype), parent strain <br /> complex. In scenario, GCN3 enhance             EY636 resistant 3-AT. <br /> defect associated eIF-2y(R127L), eIF-2y(Y142H)            EY640 EY641 Ura- derivatives isogenic strains TDl 17- <br /> eIF-2y(E383K).                                                   1A TD1 17-lAR7 respectively obtained Ura- papillae <br />    Studies translational regulation GCN4 expres-      5-FOA medium. EY647 obtained ascospore cross <br /> sion revealed inhibition GCN4 expression        EY640 EY585. <br /> the uORFs 5&apos;-leader sequences mRNA can <br /> be overcome controlling level ternary complex         Plasmids <br />                                                                 Ep264 low copy YCp5O-based plasmid contains 4.5 kb <br /> in cell. A number eukaryotic mRNAs yeast            genomic fragment harboring GCDJJ gene (Hannig et al., 1993). A <br /> to mammals contain uORFs 5&apos;-leader sequences             4.3 kb BamHI-BgllI fragment Ep264 subcloned the <br /> that subject regulation GCN4-like           unique BamHI site yeast integration vector YIp5 (Struhl et al., <br /> translational control mechanism (Kozak, 1989; Geballe           1979) create Ep487. A 1.5 kb HpaI restriction fragment encoding <br />                                                                 GCD II amino acid residues 1-487 (out 527 residues) removed <br /> and Morris, 1994). In support <span id='am-49' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-50' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span>, yeast      Ep487 plasmid resealed create Ep488. A 4.5 kb <br /> mammals possess eIF-2ax kinases that, activated,           Sall-EcoRI fragment Ep264 subcloned SalI/EcoRI <br /> are thought decrease level ternary complex         sites YIp5 create Ep467. Ep467 digested Hindlll and <br /> stabilizing eIF-2-eIF-2B complex (Koromilas et al.,         SnaBI, blunt-ended Klenow enzyme recircularized following <br />                                                                 addition BamHI linkers create Ep520. Ep520 lacks 2.1 kb <br /> 1992; Barber et al., 1993). Further genetic biochemical     HindIII-SnaBI fragment contains entire GCDJJ open reading <br /> analysis structural genes encoding eIF-2 eIF-        frame, 250 235 bp 5&apos;- 3&apos;-flanking sequences <br />  <br />                                                                                                                                   2247 <br />  D.R.Dorris, F.L.Erickson E.M.Hannig <br />  <br /> respectively. The 3.8 kb BamHI-BgllI fragment pNKY51 (Alani            equal amounts total protein, determined preliminary experiments, <br /> et al., 1987), containing yeast URA3 gene flanked direct repeats     fractionated denaturing 10% SDS - polyacrylamide gels <br /> of Ecoli hisG gene, inserted BamHI site EpS20        (Laemmli, 1970), transferred nitrocellulose paper stained with <br /> create Ep523. EpS52 low copy pSB32-based plasmid contains         Ponceau S ensure equal transfer. Immunoblot analysis performed <br /> the GCDJJ+ allele isolated yeast strain H96.                           using polyclonal rabbit antisera specific yeast eIF-2y described <br />    Ep543 created Ep3O8, harbors gcn3::LEU2 (Al-          (Hannig et al., 1993). Bound antibodies detected using [125 ]protein <br /> 305) allele (Hannig Hinnebusch, 1988), digestion BamHI,         A (ICN; &gt;30 mCi/mg) 0.3 ,uCi/ml. Blots analyzed autoradio- <br /> followed flush-ending Klenow enzyme recircularization        graphy densitometric scanning multiple lightly exposed autoradio- <br /> remove unique BamHI site present Ep308. Ep543 digested      graphs direct counting using phosphorimager (Molecular <br /> with HindlIl remove LEU2 gene plasmid recircularized     Dynamics). Phosphorimages labeled using CorelDraw version 3.0. <br /> following addition BamHI linkers. A 3.8 kb BamHI-BgllI <br /> fragment pNKY51 containing hisG-URA3-hisG sequences            Analysis polysome profiles <br /> ligated new BamHI site create EpS45.                            Cultures grown YEPD media 30&#176;C harvested OD6W <br />    p180, p226 p227, kindly provided A.G.Hinnebusch, low          1.0. Cycloheximide added cultures 50 gg/ml immediately <br /> copy plasmids harboring GCN4-lacZ alleles contain             prior harvesting cell extracts prepared presence of <br /> upstream open reading frames (uORFs), uORF4 uORFs               50 ,ug/ml cycloheximide described (Cigan et al., 1991). The breaking <br /> respectively. uORFs removed introducing point mutations        buffer supplemented 1 ,ug/ml aprotinin, 0.5 gg/ml leupeptin, <br /> the AUG codon respective reading frames (Mueller                 0.7 1g/ml pepstatin A I mM phenylmethylsulfonyl fluoride. Sixteen <br /> Hinnebusch, 1986). p367 p391 (gifts T.F.Donahue) low           A260 units 11 000 g supernatant layered 11.6 ml 7-47% <br /> copy plasmids containing HIS4AUG-1acZ his4UUG_lacZ alleles          linear sucrose gradients centrifuged 39 000 r.p.m. Beckman <br /> respectively (Donahue Cigan, 1988). p364 YCpS0 plasmid             SW41 rotor 4&#176;C 2 h. Fractions collected using ISCO <br /> containing dominant SUI3-2 allele (Donahue et al., 1988).               Model 640 gradient fractionator monitored continuously 254 nm. <br /> Recovery chromosomal alleles <br /> Ep488 linearized SnaBI introduced Ura- strains            Acknowledgements <br /> harboring wild-type (H96) mutant (EY250, EY305, EY555, EY558,            We thank Alan Hinnebusch Dennis Thiele critical reading the <br /> H57, H59, H61 H79) GCDJJ alleles. The GCDII sequences                manuscript thoughtful suggestions. We grateful Tom <br /> Ep488 lack 92% GCDJJ open reading frame, reducing            Donahue Alan Hinnebusch gifts plasmids yeast strains. <br /> the probability gene conversion events. Integration single copy     We thank Morten Kjeldgaard Jens Nyborg providing atomic <br /> of linearized Ep488 GCDJJ locus confirmed Southern            coordinates EF-Tu-GDP. We thank Barbara Freeman for <br /> blot analysis HindIII-cleaved genomic DNA purified Ura+             excellent technical assistance. This work supported grant from <br /> transformants (data shown). An 8.7 kb genomic HindIIl fragment       National Science Foundation (DMB-9105892) E.M.H. <br /> the transformants contains entire GCDII ORF flanking sequences, <br /> along YIp5 vector sequences necessary selection propagation <br /> in Ecoli. This 8.7 kb fragment circularized using DNA ligase and <br /> recovered following introduction CaCl2-treated Ecoli DHSa cells <br />                                                                             References <br /> (Sambrook et al., 1989). For allele, 2.1 kb HindIII-SnaBI          Alani,E., Cao,L. Kleckner,N. (1987) Genetics, 116, 541-545. <br /> GCDJJ fragment sequenced using Sequenase version 2.0 kit (US          Anthony,D.D.,Jr, Kinzy,T.G. Merrick,W.C. (1990) Arch. Biochem. <br /> Biochemical) synthetic oligonucleotide primers. The sequence           Biophysics, 281, 157-162. <br /> the region containing mutation verified DNA sequence            Barber,G.N., Wambach,M., Wong,M.-L, Dever,T.E., Hinnebusch,A.G. <br /> analysis plasmids recovered additional independent yeast           Katze,M.G. (1993) Proc. Natl Acad. Sci. USA, 90, 4621-4625. <br /> transformants.                                                              Berchtold,H.L. Reshetnikova,C.O. Reiser,A, Schirmer,N.K., Sprinzl,M. <br />                                                                                Hilgenfeld,R. (1993) Nature, 265, 126-132. <br /> Construction isogenic strains plasmid shuffling                       Boeke,J.D., Trueheart,J., Natsoulis,G. Fink,G.R. (1987) Methods <br /> A 3.7 kb HindIII-BamHI restriction fragment isolated              Enzymol., 154, 164-175. <br /> YIp plasmids recovered wild-type mutant strains          Bourne,H.R., Sanders,D.A. McCormick,F. (1991) Nature, 349, <br /> and cloned low copy LEU2 plasmid pSB32 (Rose Broach,             117-127. <br /> 1991). This genomic fragment contains entire GCDII open reading         Bradford,M.M. (1976) Anal. Biochem., 72, 248-254. <br /> frame 0.25 1.6 kb 5&apos;- 3-flanking sequences            Bushman,J.L., Asuru,A.I., Matts,R.L. Hinnebusch,A.G. (1993) Mol. <br /> respectively. The pSB32-borne alleles introduced Leu-            Cell. Biol., 13, 1920-1932. <br /> yeast strains EY582, EY606 EY647 Leu+ prototrophs               Castilho-Valavicius,B., Yoon,H. Donahue,T.F. (1990) Genetics, 124, <br /> selected. These strains harbor GCDJJ+ URA3+ plasmid                483-495. <br /> Ep293. The doubly-transformed strains patched solid media         Castilho-Valavicius,B., Thompson,G.M. Donahue,T.F. (1992) Gene <br /> containing 5-FOA obtain derivatives lost Ep293      Expression, 2, 297-309. <br /> contain pSB32-borne allele sole copy GCDIJI. Petite           Cigan,A.M., Feng,L. and. Donahue,T.F. (1988) Science, 242, 93-97. <br /> derivatives GCDJJ + strain isolated following growth     Cigan,A.M., Pabich,E.K., Feng,L. Donahue,T.F. (1989) Proc. Natl <br /> presence ethidium bromide described (Sherman et al., 1979).              Acad. Sci. USA, 86, 2784-2788. <br />                                                                             Cigan,A.M., Foiani,M., Hannig,E.M. Hinnebusch,A.G. (1991) Mol. <br /> /3-Galactosidase assays                                                        Cell. Biol., 11, 3217-3228. <br /> Cultures f-galactosidase assays grown liquid SD medium          Cigan,A.M., Bushman,J.L., Boal,T.R. Hinnebusch,A.G. (1993) Proc. <br /> supplemented appropriately maintain selection plasmids. A               Natl Acad. Sci. USA, 90, 5350-5354. <br /> saturated overnight culture grown 23&#176;C diluted fresh medium     Clare,J. Farabaugh,P. (1985) Proc. Natl Acad. Sci. USA, 82, <br /> and grown 8 h 300C aeration [repressing (R) conditions].          2829-2833. <br /> For derepressing (DR) conditions, 3-AT added final concentration   Dever,T.E., Feng,L., Wek,R.C., Cigan,A.M., Donahue, <br /> </body></html>