<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                    INVESTIGATION <br />  <br />  <br />  <br />  <br />                                                   Probing Boundaries Orthology: The <br />                                                            Unanticipated Rapid Evolution of <br />                                                                     Drosophila centrosomin <br />                                                                                                         Robert C. Eisman Thomas C. Kaufman1 <br />                                                                                        Department Biology, Indiana University, Bloomington, Indiana 47405 <br />  <br />  <br />  <br />  <br /> ABSTRACT The rapid evolution essential developmental genes protein products intriguing problematic. The <br /> rapid evolution gene products simple protein folds lack well-characterized functional domains typically result low <br /> discovery rate orthologous genes. Additionally, absence orthologs dif&#65533;?cult study processes mechanisms <br /> underlying rapid evolution. In study, investigated rapid evolution centrosomin (cnn), essential gene encoding <br /> centrosomal protein isoforms required syncytial development <span id='am-198' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-199' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-203'  ></span><span id='am-204' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span>. Until recently rapid divergence of <br /> cnn identi&#65533;?cation orthologs dif&#65533;?cult questionable Cnn violates assumptions underlying models for <br /> protein evolution. To overcome limitations, identi&#65533;?ed group insect orthologs present conserved features likely <br /> to required functions attributed cnn D. <span id='am-327' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-328' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-329'  ></span><span id='am-330' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>melanogaster</span>. We rapid divergence Cnn isoforms is <br /> apparently frequent coding sequence indels accelerated rate intronic additions eliminations. These changes <br /> appear buffered multi-exon multi-reading frame maximum potential ORFs, simple protein folds, splicing <br /> machinery. These buffering features occur genes Drosophila help prevent potentially deleterious mutations <br /> due indels genes large coding exons exon-dense regions separated small introns. <span id='am-239' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-240'  ></span><span id='am-241' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> work promises useful for <br /> future investigations cnn potentially rapidly evolving genes proteins. <br />  <br />  <br />  <br />  <br /> A     S genomic era advances, molecular biology has <br />       shifted single-gene studies global analyses of <br /> genomes broad phylogenetic range organisms. For <br />                                                                                        known genes model organisms, essential or- <br />                                                                                        thology assignments accurate (Hulsen et al. 2006). <br />                                                                                        The importance orthology evident the <br /> single-gene global studies, critical &#65533;?rst step molec-                        number databases reporting orthologous gene sets by <br /> ular biologists accurate identi&#65533;?cation orthologous                           number terms introduced relational sub- <br /> genes. The concept orthology introduced Fitch                             categories time Fitch initially introduced the <br /> differentiate orthologous genes, one-                             concept orthologous paralogous gene relationships. <br /> to-one relationship <span id='am-261' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-262' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, paralogous genes,                                 Since orthologous genes related descent and <br /> which result duplication event original ortholog                         orthology based sequence homology phylogeny, <br /> within species (Fitch 1970, 2000). Because orthologs                             process identi&#65533;?cation relatively simple. This <br /> often closely related paralogs, generally as-                          certainly true genes, signi&#65533;?cant subset <br /> sumed gene function likely conserved. Since                          genes problems confound process. One <br /> functional annotation uncharacterized genomes gener-                             method &#65533;?nding orthologs use orthologous gene sets <br /> ally computationally based overall sequence identity                           public databases, gene sets variable <br />                                                                                        depending methods used determine orthology and <br />                                                                                        number genomes used create database <br /> Copyright &#169; 2013 Genetics Society America                                    (Altenhoff Dessimoz 2009). It shown that <br /> doi: 10.1534/genetics.113.152546 <br /> Manuscript received April 25, 2013; accepted publication May 28, 2013              gene prediction errors (Nagy et al. 2011b) confusing <br /> Supporting information available online http://www.genetics.org/lookup/suppl/    orthologs, paralogs, epaktologs, genes acquire <br /> doi:10.1534/genetics.113.152546/-/DC1. <br /> 1 <br />  Corresponding author: Indiana University, 1001 E. Third St., Bloomington, IN 47405. <br />                                                                                        conserved domain independently (Nagy et al. 2011a), all <br />  E-mail: kaufman@indiana.edu                                                           confound orthologous gene identi&#65533;?cation. In Nagy <br />  <br />  <br />  <br />                                                                                                          Genetics, Vol. 194, 903&#8211;926 August 2013        903 <br />  et al. studies errors common         characterized genes <span id='am-290' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-291' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> pro- <br /> enough result erroneous predictions global      teins. Several experimental studies genes do- <br /> analyses protein-domain architecture evolution. At        main <span id='am-44' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>fungi</span> vertebrates gene <br /> single-gene level, problems com-        function diverges rapidly lineages, as <br /> plicated rapid evolution absence known cu-         paralogous vertebrate genes (see Discussion). <br /> rated functional domains. Although generally accepted      Additionally, D. melanogaster shown this <br /> that multi-domain proteins encoded orthologous genes          domain constitutive cnn exon long- and <br /> should share multiple domains domain architecture,           short-form families Cnn proteins (Eisman et al. 2009). <br /> and low-complexity regions used de-        However, localization short forms mitotic centro- <br /> termine orthology (Galperin Koonin 1998; Mushegian           somes mutants lacking long forms fails rescue mi- <br /> et al. 1998), criteria easily ignored molecular   totic defects (Eisman et al. 2009). This shows single <br /> <span id='am-332' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-333' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-335' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-336'  ></span>data</span> limited.                                                domain explain Cnn function domain <br />     One example issues complicate orthologous       determine orthology, but, mentioned <br /> gene identi&#65533;?cation provided Drosophila melanogaster        above, cases has. <br /> centrosomin (cnn), gene function essential            While data suggest known functions D. <br /> formation functional centrosomes, organization         melanogaster cnn applicable insect <br /> actin microtubule cytoskeleton, chromosome segre-        species, claims orthology cnn vertebrate <br /> gation rapid cleavage divisions syncytial em-      genes CDK5RAP2 mmg infer function becoming <br /> bryos (Megraw et al. 1999; Vaizel-Ohayon Schejter            common literature. Additionally, Bloomington <br /> 1999; Eisman et al. 2009). Simple searches orthologous        Stock Center lists cnn ortholog CDK5RAP2 <br /> gene sets cnn public databases variable. Based      useful studying, Drosophila, human disease auto- <br /> on results OrthoDB EggNOG cnn orthologs        somal recessive primary microcephaly. Since cnn and <br /> limited insects, Homologene predicts insects     CDK5RAP2 centrosomal proteins necessary local- <br /> and vertebrate gene Cyclin Dependent Kinase5 Regula-         ization g-tubulin (Megraw et al. 2001; Fong et al. 2008), <br /> tory Activating Protein2 (CDK5RAP2), paralo-         possible experimental studies CDK5RAP2 could <br /> gous vertebrate gene myomegalin (mmg), Inparanoid            provide new insights directions future molecular <br /> has complex somewhat inconsistent mix putative          studies cnn. Alternatively, phenotypic similarities between <br /> orthologs. Sequence alignments various gene sets        mutational studies cnn CDK5RAP2 to <br /> suggest OrthoDB EggNOG best set          direct indirect effects associated loss essential <br /> functional inference D. melanogaster experimental           centrosome function. <br /> data, correct assumption?                              To differentiate possibilities, impor- <br />     There reasons explain variability   tant establish reliable cnn orthologous gene set likely to <br /> in cnn orthologous gene sets. Comparisons Cnn proteins        similar functions. As mentioned above, cnn transcrip- <br /> within <span id='am-287' about='obo:NCBITaxon_genus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-288' property="rdfs:label" content="genus" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>genus</span> Drosophila indicate gene           tionally complex mixed pool Cnn-encoded splice <br /> encoded proteins evolving rapidly approxi-          variants present embryonic centrosomes (Eisman et al. <br /> mately half length Cnn proteins form simple           2009). The identi&#65533;?cation splice variants dif&#65533;?cult at <br /> coiled-coil domains, reliable determining      best gene models species predict single protein <br /> orthology (Tatusov et al. 1997). Additionally, D. mela-          rapid evolution cnn make ab initio exon and <br /> nogaster cnn transcriptionally complex, encoding     splicing predictions unreliable. However, D. mela- <br /> a dozen splice variants form protein families (Eis-     nogaster Cnn-PA isoform spans entire gene, <span id='am-322' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-323' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-324'  ></span><span id='am-325' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>- <br /> man et al. 2009). Possibly complexity    form suf&#65533;?cient rescue mutant phenotype provided <br /> rapidly evolving sequence, cnn gene models       short isoforms present (Eisman et al. 2009). Moreover, <br /> insects variable, containing possible exon   orthologous genes encoding Cnn-PA-like isoforms have <br /> the gene lacking exons. Finally,         sequence information future transcriptional studies and <br /> exception small domain (Flory et al. 2002; Sawin          identify majority protein domains necessary for <br /> et al. 2004; Zhang Megraw 2007; Fong et al. 2008),           cnn function. <br /> cnn encodes identi&#65533;?ed functional protein domains.           One possible explanation fact gene model <br />     The conserved domain cnn used infer orthology      predictions cnn insects incorrect may <br /> across broad phylogenetic distances protein        nature sequence spanning coding exons <br /> superfamily predicted associated micro-        intervening introns. The general assumption the <br /> tubules (Flory et al. 2002; Venkatram et al. 2004). This         reading frame coding exon encounter stop <span id='am-256' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-257' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-259'  ></span><span id='am-260' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> <br /> domain single gene fungal insect species    adjacent intron. A recent study fre- <br /> and genes vertebrate species. Although conser-     quency in-frame stop codons &#65533;?rst 40&#8211;70 bases of <br /> vation domain predicts genes superfamily      adjacent downstream intron 40&#8211;70%, respectively <br /> are likely functional similarities, does     (Zhang et al. 2003). However, study insect cnn <br /> necessarily itself support strong functional inference   orthologs, reading frame exon <br />  <br />  <br />  <br /> 904    R. C. Eisman T. C. Kaufman <br />  frequently extends adjacent introns,           framework functional annotation orthologs, <br /> some cases single reading frame include multiple            paralogs, epaktologs share degree homol- <br /> coding exons introns. For discussion purposes, pres-       ogy Cnn. <br /> ent concept maximum potential open reading frames <br /> (MPOs) associated coding exons. While MPO            Materials Methods <br /> most coding exons likely similar actual spliced <br /> exon, subset genes Drosophila          Identi&#65533;?cation centrosomin orthologs <br /> contain longer MPOs multi-exon MPOs. These               The genus Drosophila orthologs identi&#65533;?ed GBrowse <br /> extensive MPOs associated exon-dense regions con-        BLAST downloaded FlyBase (http://&#64258;ybase. <br /> taining small introns.                                            org/) taken frozen CAF1 assemblies and <br />     We type MPO associated         <span id='am-163' about='obo:NCBITaxon_7237' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32355 obo:NCBITaxon_32358 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-164' property="rdfs:label" content="Drosophila pseudoobscura" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-167'  ></span>Drosophila pseudoobscura</span> Release 2.0. We retrieved <br /> frequent nucleotide changes rapid accumulation         genomic sequences RNA-Seq data additional <br /> random small insertions deletions (indels). Since indels      sequenced Drosophila genomes Baylor sequenc- <br /> in protein-coding sequences considered rare             ing center modENCODE project (http://www.hgsc. <br /> genomic changes (Rokas Holland 2000) indels               bcm.tmc.edu/collaborations/insects/dros_modencode/ <br /> ranging single amino acid 20 amino acids           RNAseq_data/). These manually annotated by <br /> been used determine phylogenies (Gupta 1998; Keeling           exon-to-exon comparisons translation based on <br /> et al. 2000), apparently common random           known D. melanogaster complementary DNA (cDNA) <br /> in Cnn? The underlying assumption indels        clones available FlyBase. All non-Drosophila sequences <br /> due codon loss, rapid nucleotide sequence diver-       identi&#65533;?ed orthology links FlyBase) and <br /> gence adjacent Cnn indels suggests            downloaded ENSEMBLE <span id='am-91' about='obo:NCBITaxon_7091' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_7090 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_37569 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_119258 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7089 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_475327 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-92' property="oboInOwl:hasExactSynonym" content="silk moth" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="domestic silkworm" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="silkworm" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bombyx mori Linnaeus, 1758" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Bombyx mori" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Bombyx mori</span>, which <br /> nontriplet indels change reading frame. In          based full-length cDNA obtained silk <br /> study, present evidence nontriplet indels          moth genome project. Sequences annotated and <br /> buffered extended MPOs, conjunction            mapped using MacVector 12.5 (Accelrys Inc.). Electronic <br /> an unknown relaxed splicing mechanism simple protein          versions annotated sequences available on <br /> folds encoded exons. These features appear        request. <br /> make possible retain protein function normally <br />                                                                   Drosophila stocks <br /> deleterious mutations occur. <br />     This work presents detailed analysis cnn ortholo-      All &#64258;ies used study grown standard <br /> gous gene set insects. In addition domain         cornmeal&#8211;agar medium 25&#176;. To express D. mela- <br /> mentioned above, identify additional highly con-         nogaster GFP::Cnn-PA fusion transgene embryogen- <br /> served motifs unique insects overall         esis, UASpBacNPF::GFP::Cnn-PA homozygous transgenic <br /> structure Cnn-PA conserved insect            males D. melanogaster, <span id='am-187' about='obo:NCBITaxon_7240' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-188' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-189'  ></span><span id='am-190' property="rdfs:label" content="Drosophila simulans" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Drosophila simulans</span>, Drosophila <br /> orders. While protein motifs structure conserved,         yakuba, D. pseudoobscura crossed appropriate <br /> the gene protein diverge considerably species         females homozygous pBac(3xP3-EGFPafm)::nos- <br /> and orders. The intron&#8211;exon structure cnn similar           Gal4::(pW8-) transgene previously described (Holtzman <br /> within <span id='am-264' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-265' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-266' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> signi&#65533;?cantly different orders       et al. 2010). Embryos collected pBac(3xP3-EGFPafm):: <br /> investigated, changes correlated           nos-Gal4::(pW8-); w; P{w+mC=pUASP- GFP::Cnn-PA} moth- <br /> rapid evolution Cnn proteins. In addition gene archi-       ers stained Cnn, a-tubulin, DNA determine <br /> tecture changes, nucleotide substitution         effects D. melanogaster Cnn-PA chimeric protein on <br /> frequent indels associated rapid evolution        native Cnn function divergent species. <br /> Cnn-PA changes appear buffered by <br />                                                                   Cloning transformation <br /> the inherent nature sequence, simple protein folds, <br /> and potentially relaxed splicing mechanisms. While          To construct GFP::Cnn-PA fusion protein, PCR- <br /> changes buffered species,           ampli&#65533;?ed EGFP (Living Colors Invitrogen) the <br /> divergence Cnn-PA dominant-negative affect            following primers: SpeGfp-59 (ACT AGT ATG GTG AGC <br /> protein function species. Finally, present         AAG GGC GAG) GfpMlu-39 (ACG CGT CTT GTA CAG <br /> concept MPOs associated coding exons. The ex-             CTC GTC CAT G). We ampli&#65533;?ed Cnn-PA transcript from <br /> tended MPOs present cnn appear common rapidly         previously described Cnn-PA cDNA (Heuer et al. 1995) <br /> evolving genes Drosophila buffer poten-        added MluI EcoRI sites oligos using standard <br /> tially deleterious genomic changes. Additionally,         PCR techniques. PCR products cloned Topo-TA <br /> that comparative studies changes homologous            pCR2.1 vector (Invitrogen) manufacturer&#8217;s pro- <br /> MPOs provide useful tool identi&#65533;?cation relatively    tocol veri&#65533;?ed sequencing. All cloning done <br /> small signi&#65533;?cant changes coding regions broad     using in-gel ligation techniques (Kalvakolanu Livingston <br /> phylogenetic range. It hope provides com-      1991). PCR products digested appropriate <br /> prehensive set orthologs molecular studies cnn      enzymes subcloned pBlueskript vector. Following <br />  <br />  <br />  <br />                                                                                                    Rapid Evolution Cnn   905 <br />  digestion NotI, GFP::Cnn-PA fusion shuttled       formation parthenogenetic development in <br /> into UASpBacNPF vector, plasmid transformation         <span id='am-158' about='obo:NCBITaxon_7253' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_32281 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32323 obo:NCBITaxon_33208 obo:NCBITaxon_32321 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-159'  ></span><span id='am-160' property="rdfs:label" content="Drosophila mercatorum" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila mercatorum</span> (Eisman Kaufman 2007). Addi- <br /> was described (Holtzman et al. 2010).                  tionally, shown long- short-splice <br />                                                                variants isoforms present development and <br /> Fixation immunostaining <br />                                                                protein families differ signi&#65533;?cantly pro- <br /> Embryos collected &#65533;?xed 50% heptane:50%             tein sequence level (Eisman et al. 2009). However, two <br /> Methanol/EGTA previously described (Eisman et al.           pieces evidence suggest gene protein prod- <br /> 2006). Embryos immunostained previously de-            ucts evolving rapid rate genus. First, <br /> scribed (Gorman et al. 1991). Specimens mounted           hybridization ef&#65533;?ciency Cnn-PA coding sequence to <br /> on glass slides 90% glycerol 10% PBS 0.2 mM        genomic DNA Southern blots decreases rapidly within <br /> n-propyl gallate (Sigma). The following antibodies        <span id='am-319' about='obo:NCBITaxon_subgenus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-320' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-321' property="rdfs:label" content="subgenus" datatype="xsd:string"></span>subgenus</span> <span id='am-376' about='obo:NCBITaxon_32341' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_46877 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-377' property="rdfs:label" content="Sophophora" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-379'  ></span><span id='am-380' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Sophophora</span> detectable outside the <br /> used: guinea pig (whole sera) anti-Cnn.Ex1a2 1:500          subgenus Drosophila (Figure 1, top). Second, reactivity <br /> (Megraw et al. 1999) anti-a-tubulin 84B (Matthews          original Cnn antibody, covers 67% Cnn- <br /> et al. 1989). DNA stained TOTO3 (Molecular            PA protein (Heuer et al. 1995), decreases signi&#65533;?cantly <br /> Probes) &#65533;?nal concentration 1:1000. All &#64258;uorescent      genus (Figure 1, bottom). <br /> secondary antibodies used 1:200 (Jackson Immu-            One possible explanation single relatively small <br /> noResearch Labs).                                              motif (discussed below) protein responsible <br /> Microscopy imaging                                         Cnn function remainder protein is <br />                                                                important diverge. Alternatively, Cnn <br /> All images captured Leica TCS confocal micro-        apparently essential syncytial development <br /> scope 63X HCX Plan Apo oil immersion objective,         spermatogenesis, role gene environ- <br /> using TCSNT software. Images Z-series range       ment relatively novel, functionally orthologous <br /> from 2.5 6 mm thick, composed 0.5- 0.9-mm-thick       genes present limited phylogenetic <br /> sections. Projected images processed assembled        range. To differentiate possibilities, we <br /> into &#65533;?gures Adobe Photoshop version 7.0 software.         searched analyzed putative orthologs from <br /> MPO search, genomic sequences, gene annotation             Drosophila species insects suf&#65533;?- <br />                                                                cient genomic sequence. We &#65533;?rst report results our <br /> To search D. melanogaster genome MPOs, origi-       analysis genus Drosophila. <br /> nally determined far reading frame annotated       The D. melanogaster cnn-RA transcript used de- <br /> exons extended designated 59 39 ends using      termine orthology transcript spans entire gene, <br /> Release 4.2. These annotations updated con-      contains constitutive D. melanogaster cnn exons, and <br /> &#65533;?rmed using recent FlyBase release (5.4). From          rescues cnn mutant phenotype species provided <br /> these results selected genes MPO          Cnn short isoforms present (Eisman et al. 2009). This <br /> extending .3000 nucleotides annotated end.           suggests cnn-RA splice variant contains critical <br /> From set selected seven genes manually anno-       domain domains necessary Cnn long-isoform function. <br /> tated sequence gene based protein isoforms      Since transcript spans entire gene, provides <br /> available FlyBase. The algorithm available request.   landmarks investigate conservation, lack thereof, of <br /> Genomic sequences FlyBase frozen CAF1        complex splicing previously described D. mela- <br /> assemblies D. pseudoobscura Release 2.0. Sequences     nogaster cnn gene (Eisman et al. 2009). Based D. <br /> were manually annotated described above. The cnn            melanogaster Cnn-PA protein identi&#65533;?ed putative <br /> sequences North Carolina population D. mela-         cnn orthologs 13 Drosophila species. <br /> nogaster (Mackay et al. 2012) recovered http:// <br /> dgrp.gnets.ncsu.edu/. These annotated described     Conserved motifs identify cnn orthologs <br /> the results. Electronic versions annotated sequences    Prior identi&#65533;?cation cnn orthologs reported here, <br /> are available request.                                      knew D. melanogaster Cnn-PA putative <br />                                                                leucine zippers coiled-coil regions (Heuer et al. <br /> Results                                                        1995) protein phosphorylated syn- <br />                                                                cytial development (Li Kaufman 1996). To in- <br /> The coding regions centrosomin diverge rapidly <br />                                                                vestigate rapid divergence Cnn-PA, &#65533;?rst wanted to <br /> Genetic molecular analyses cnn D. melanogaster       identify conserved features protein. This done <br /> suggest multiple motifs domains con-        using identi&#65533;?ed putative orthologs predicted <br /> served orthologs. Cnn proteins required     encoded proteins, compared conserved <br /> initiation PeriCentriolar Matrix (PCM) biosynthesis D.   sequences identi&#65533;?ed. It noted used only <br /> melanogaster (Dobbelaere et al. 2008; Conduit Raff         orthologs high-quality genomic sequence lacked <br /> 2010), function appears conserved        large stretches ambiguous base calls, poly-&#8220;N&#65533;? regions <br /> the genus Drosophila Cnn required novo centro-    cnn gene, unfortunately removed closely <br />  <br />  <br />  <br /> 906    R. C. Eisman T. C. Kaufman <br />                                                                                               Figure 1 The rapid divergence cnn in <br />                                                                                              Drosophila. (Top) The coding sequence <br />                                                                                              centrosomin gene evolves rapidly <br />                                                                                              genus Drosophila shown <br />                                                                                              weak hybridization labeled <br />                                                                                              cnn-RA transcripts genomic DNA <br />                                                                                              Southern blots. The &#65533;?rst seven <br />                                                                                              lanes species subgenus Soph- <br />                                                                                              ophora, strongest signals are <br />                                                                                              members <span id='am-381' about='obo:NCBITaxon_32346' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-382'  ></span><span id='am-383' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-384' property="rdfs:label" content="melanogaster group" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>melanogaster group</span>. <br />                                                                                              The lanes species sub- <br />                                                                                              genus Drosophila, representing ex- <br />                                                                                              tent phylogenetic range <br />                                                                                              producing detectable signal. (Bottom) <br />                                                                                              The Cnn-PA protein evolves rapidly, <br />                                                                                              evidenced decreased reactivity of <br />                                                                                              anti-Cnn antibody immunostained <br />                                                                                              embryos. Cnn staining (green) strong <br />                                                                                              Dmel, Dsim, Dere, Dyak, mem- <br />                                                                                              bers <span id='am-366' about='obo:NCBITaxon_32351' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-367' property="rdfs:label" content="melanogaster subgroup" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-369'  ></span><span id='am-370' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>melanogaster subgroup</span>, but <br />                                                                                              weak Dpse undetect- <br />                                                                                              able Dvir using antibody <br />                                                                                              concentrations. The staining both <br />                                                                                              Dpse Dvir improved in- <br />                                                                                              creasing concentration Cnn <br />                                                                                              antibody. Microtubules shown in <br />                                                                                              red DNA blue. Dmel: D. mela- <br />                                                                                              nogaster; Dsim: D. simulans; Dere: D. <br />                                                                                              erecta; Damb: D. ambigua; Dwil: D. wil- <br />                                                                                              listoni; Dsal: D. saltans; Dana: D. <span id='am-38' about='obo:NCBITaxon_4614' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_4613 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-39'  ></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Ananas" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Ananas Mill." datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ananas</span>- <br />                                                                                              sae; Dmer: D. mercatorum; Dvir: D. <br />                                                                                              virilis; Dyak: D. yakuba; Dpse: D. <br />                                                                                              pseudoobscura. <br />  <br />  <br />  <br />  <br /> related D. simulans <span id='am-168' about='obo:NCBITaxon_7238' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Drosophila sechellia Tsacas and Bachli, 1981" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-172'  ></span><span id='am-173' property="rdfs:label" content="Drosophila sechellia" datatype="xsd:string"></span>Drosophila sechellia</span> species    inclusion &#8220;Centrosomin&#65533;? previous infers <br /> analyses. Any putative protein orthologs possess          Cnn function described D. melanogaster be <br /> a similar structural architecture shared motifs. While       retained proteins motif. Second, is <br /> we identi&#65533;?ed features promise inter-        sequence based implies general function the <br /> esting candidates future molecular experiments,      motif consistent experimental studies. <br /> study present features likely conserved          The conserved motifs present Drosoph- <br /> in cnn orthologs.                                            ila Cnn potential orthologs. The &#65533;?rst centrally located and, <br />    The &#65533;?rst conserved Cnn motif D. mela-         based sequence, named SESAW motif <br /> nogaster cnn splice variants highly conserved      (Figure 2). This motif known phosphorylated at <br /> Drosophila species investigated (Figure 2). This motif   protein kinase A site (http://www.phosphopep.org/) <br /> a conserved domain superfamily proteins        and, based silico analyses, casein kinase <br /> from fungi animals amino terminus           II glycogen synthase site shaggy kinase site with <br /> these proteins. Experimental evidence shows         high probability scores phosphorylated. The sec- <br /> of proteins associated microtubule cyto-      ond Drosophila-conserved sequence named SPD <br /> skeletal and/or endomembrane system, overall       motif (Figure 3). It located near carboxy terminus of <br /> functions considerably different (see Discussion). The       protein shown phosphorylated at <br /> motif previously named Centrosomin Motif 1              putative protein kinase C site. Both motifs fall in <br /> (Zhang Megraw 2007), prefer Ken-             regions predicted helical character capable <br /> Flee Cytoskeleton motif (KFC) motif reasons. First,      forming coiled-coil. Nonetheless, coiled-coil domains <br />  <br />  <br />  <br />                                                                                                      Rapid Evolution Cnn        907 <br />  Figure 2 Conserved motifs structure Cnn-PA Drosophila. Protein alignments KFC, SESAW, SPD motifs carboxy terminus are <br /> nearly invariant genus Drosophila present long-form splice variants Cnn. Their position Cnn-PA shown aligned the <br /> secondary structure protein. Coiled-coil domains represented spirals noncoiled a-helical regions solid. Dbip: D. biarmipes; Dtak: D. <br /> takahashii; D&#65533;?c: D. &#65533;?cusphila; Dele: D. elegans; Dbip: D. bipectinata; Dmoj: D. mojavensis; Dgri: D. grimshawi. <br />  <br />  <br /> bound both. Based BLAST analyses motifs pres-                      tions attributable Cnn, certain will <br /> ent arthropods. The actual function                   functional orthologs. <br /> motifs subject future studies, seems <br />                                                                                Rapid changes intron&#8211;exon structure cnn <br /> likely involved regulating Cnn function dur- <br />                                                                                orthologs MPOs <br /> ing cell cycle. <br />    The fourth conserved Cnn motif carboxy                            While conservation motifs invaluable the <br /> terminus (Figure 2) nearly invariant genus                       identi&#65533;?cation potential cnn orthologs, motifs provide <br /> Drosophila. We previously shown deletion                     insights mechanism(s) underlying rapid evo- <br /> COOH motif SPD D. melanogaster results mis-                      lution gene. Since non-Drosophila proteins <br /> localization truncated Cnn region chromo-                     (discussed below) based ab initio gene models, we <br /> somes dividing nuclei centrosomes                           wanted know apparent changes gene <br /> (Eisman et al. 2009). Thus potential function                     architecture exon&#8211;intron structure relative D. mel- <br /> both motifs restrict Cnn centrosome.                     anogaster cDNA-based model transcripts. <br />    In addition conserved motifs, Drosophila                     When initiated work Drosophila genomes <br /> Cnn-PA proteins  50% coiled-coil, arrangement                      assembled, GLEAN consensus mod- <br /> of coiled-coil regions similar D. melanogaster (Fig-                 els available individual models highly <br /> ure 2). The D. melanogaster Cnn-PA leucine zippers                       variable, manually annotated genes. To identify <br /> are conserved genus Drosophila, &#65533;?rst                    individual exons, &#65533;?rst identi&#65533;?ed short coding &#8220;word&#65533;? <br /> third zippers absent proteins described                     based conserved motifs presumed exon <br /> here. The data predict true cnn orthologs                    sequences extended reading frame containing <br /> encode coiled-coil protein containing KFC, SESAW,                        word upstream downstream &#65533;?rst stop codon. <br /> and SPD motifs, carboxy terminus similar                       We designate extended reading frames MPO of <br /> the Drosophila tail. It likely proteins                  coding exon. Each MPO aligned D. <br /> an important component centrosome, predic-                     melanogaster cnn-RA coding exons determine best esti- <br /> tion shown experimentally outside Drosophila.                    mate splicing. This method necessary since, based <br /> Proteins subset motifs share func-                    silico analysis, cnn paucity strong canonical <br />  <br />  <br />  <br /> 908     R. C. Eisman T. C. Kaufman <br />  Figure 3 Drosophila MPOs confound gene-modeling programs. Splicing predictions cnn transcripts dif&#65533;?cult lack strong splicing <br /> signals, multi-exon MPOs, MPOs extend ends known splice sites. A comparison MPOs Drosophila species this <br /> study (boxed area) showing MPOs cnn reveals MPOs variable species. At MPOs D. melanogaster cnn; <br /> the multi-colored boxes exons encoding Cnn-PA (below grey boxes), gray black boxes coding exons present cnn <br /> splice variants. The MPO maps transcriptional complexity D. melanogaster cnn probably conserved genus. <br />  <br />  <br /> splicing sequences, partially explain variabil-                  To produce reasonable gene models rely <br /> ity seen individual ab initio models, appeared                 solely ab initio predictions, necessary some <br /> since started analysis. This fact combined                 independent assessment actual splicing pattern used <br /> observation MPO domains D. melanogaster                     different species. The robust model seen D. mel- <br /> all members genus examined over-                     anogaster historically relied collection cDNAs, <br /> lapping cases contain multiple exons, extend-                   recently largely substantiated by <br /> ing regions known intronic D. melanogaster,                RNA-Seq data (McQuilton et al. 2012). Unfortunately, with <br /> made construction cogent gene model spe-                   exception single EST D. pseudoobscura, no <br /> cies dif&#65533;?cult. A summary positions extent                 cDNA evidence available 13 species. Fortu- <br /> MPOs D. melanogaster 13 species shown                   nately, however, RNA-Seq data 5 13 <br /> in Figure 3.                                                                species (https://www.hgsc.bcm.edu/content/drosophila- <br />  <br />  <br />  <br />                                                                                                                     Rapid Evolution Cnn      909 <br />  Figure 4 Intronic indels common random Drosophila cnn. Cnn-PA coding exons (color coded Fig. 3) span entire cnn gene. When <br /> maps scale aligned conserved carboxy terminus, intronic indels obvious. Because frequent random indels, exon positions <br /> are variable size cnn genus. Gene sizes shown left. <br />  <br />  <br />  <br /> modencode-project). Using data, possible                     splicing cnn transcripts, available RNA-Seq data <br /> construct substantiated gene models exons encod-                     indicate species splice small intron out <br /> ing ORF corresponding Cnn-PA. With models                       mature transcript. <br /> in hand, possible infer precisely mod- <br />                                                                              Annotation cnn insects <br /> els orthologs Cnn-PA 8 species. <br />    The results analysis, shown Figure 4, reveal              We identi&#65533;?ed potential cnn orthologs other <br /> within genus Drosophila overall gene architecture                 insects initially based BLAST analyses available data- <br /> apparently conserved putative coding exons                      bases. Our initial probes genomes derived <br /> similar size, introns exons accumulated                      conserved domains genus Drosophila. <br /> numerous indels resulting gene size variability (Figure                   We able unambiguously identify KFC, SESAW, and <br /> 4). Unlike RNA-Seq-substantiated deduced coding                      SPD mosquitos, hymenopterans, lepidop- <br /> exons, MPOs variable size, generally larger                  terans, Tribolium (Figure 5). <br /> the spliced exons, observed changes correlate                    It possible identify COOH terminal <br /> with phylogenetic order, suggesting sequence                        conserved domain mosquitos, <span id='am-143' about='obo:NCBITaxon_7399' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-144' property="oboInOwl:hasRelatedSynonym" content="hymenopterans" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27527" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Hymenoptera" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27531" datatype="xsd:string"></span><span id='am-149'  ></span><span id='am-150' property="oboInOwl:hasExactSynonym" content="hymenopterans" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27522" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Hymenoptera</span>, and <br /> changes relatively common undirected. Much                        <span id='am-83' about='obo:NCBITaxon_7088' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_85604 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="moths" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Lepidoptera" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="moths" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="butterflies and moths" datatype="xsd:string"></span>Lepidoptera</span> but, domain conserved within <br /> the variability MPOs different species                     orders, peptides divergent orders. <br /> accounted small internal indels con-                      Although terminus diverges, recog- <br /> served motifs, changes outside coding                  nized BLAST Clustal analyses always <br /> regions, indels account protein size variability               contained coding exon annotated genes. <br /> (Figure 4). Additionally, exons 6 7 D. mel-                     Interestingly, COILS (Lupas et al. 1991) analyses show <br /> anogaster model reside single MPO seven                       Cnn-PA orthologs investigated similar ar- <br /> Drosophila species investigated. Thus potential exists                   rangement coiled-coil domains. Additionally, Cnn- <br /> to alternatively splice exons larger single exon              PA orthologs vary  720 amino acids Lepidoptera <br /> retaining intron exons 6 7 (Figure 3                     1400 amino acids Hymenoptera, spacing of <br /> Figure 4). However, extensive cDNA data D. melanogaster                   protein motifs proportionally constant species. The <br /> show evidence 647 intron retained                    second centrally located zipper present insect species <br />  <br />  <br />  <br /> 910     R. C. Eisman T. C. Kaufman <br />  Figure 5 Cnn-PA motifs structure conserved insects. Similar Figure 2, conserved motifs carboxy terminus aligned to <br /> the structure Cnn-PA A. mellifera. Because databases literature assert vertebrate genes CDK5RAP2 mmg are <br /> orthologous cnn, included human co-orthologs alignments. The human genes signi&#65533;?cantly different structure (not shown), the <br /> carboxy terminus signi&#65533;?cant homology insects, terminus divergent human paralogs insects <br /> (bottom). The SESAW SPD motifs detectable vertebrate gene. The Cnn motifs diverged orders highly conserved <br /> within orders. The carboxy terminus (CTERM) divergent conserved motifs shown alignments Hymenoptera (top) and <br /> D. melanogaster (top, line), highly conserved Hymenoptera Lepidoptera (bottom). Agam: A. gambiae; Aegy: A. egypti; Amel: <br /> A. mellifera; Bter: B. terrestris; Bimp: B. <span id='am-356' about='obo:NCBITaxon_35939' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_41945 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_25692 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-357' property="rdfs:label" content="Impatiens" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-360'  ></span>impatiens</span>; Nvit: N. vitripennis; C&#64258;o: C. &#64258;oridanus; Aech: A. echinatior; Sinv: S. invicta; Hsal: H. salator; Bmor: B. <br /> mori; Dple: D. <span id='am-293' about='obo:NCBITaxon_243410' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_94020 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_243884 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_243362 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-294' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-295' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-296' property="rdfs:label" content="Plexippus" datatype="xsd:string"></span><span id='am-297'  ></span>plexippus</span>; Hsap; <span id='am-244' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-245'  ></span><span id='am-246' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span>Homo sapiens</span>. <br />  <br />  <br />  <br /> examined honeybee <span id='am-122' about='obo:NCBITaxon_7459' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_7458 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_70987 obo:NCBITaxon_34735 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_83321 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Apis" datatype="xsd:string"></span>Apis</span> mellifera. Thus cen-                           With exception mosquitos, Cnn-PA appar- <br /> trally located leucine zipper functionally important                    ently encoded exons relative genus Drosoph- <br /> some species, good predictor orthology.                        ila insects investigated. Although fragmented, <br />    Using conserved motifs, knowledge gene                        hymenopteran coding exons islands length <br /> structure &#64258;ies, cDNA B. mori, available gene                        gene, intronic deletions result Dro- <br /> models species annotated cnn gene                         sophila-like maps. In general, overall gene architecture is <br /> these 13 additional insects. For ant species <span id='am-267' about='obo:NCBITaxon_604375' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_43085 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_141711 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-268' property="rdfs:label" content="Harpegnathos" datatype="xsd:string"></span><span id='am-269'  ></span><span id='am-270' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Harpegnathos</span>                     conserved orders considerably different among <br /> salator <span id='am-9' about='obo:NCBITaxon_13686' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_34695 obo:NCBITaxon_33208 obo:NCBITaxon_13685 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_144017 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-10' property="oboInOwl:hasExactSynonym" content="imported red fire ant" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="red fire ant" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="red imported fire ant" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Solenopsis invicta" datatype="xsd:string"></span>Solenopsis invicta</span> altered predicted gene                      orders. The exception pattern <span id='am-232' about='obo:NCBITaxon_7159' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_41827 obo:NCBITaxon_131567 obo:NCBITaxon_1056966 obo:NCBITaxon_7147 obo:NCBITaxon_33154 obo:NCBITaxon_7148 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_53541 obo:NCBITaxon_7157 obo:NCBITaxon_7158 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_43786 obo:NCBITaxon_43817 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-233' property="oboInOwl:hasRelatedSynonym" content="Stegomyia aegypti" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasExactSynonym" content="yellow fever mosquito" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="Aedes aegypti" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-238'  ></span>Aedes aegypti</span>, which <br /> models models missing &#65533;?rst cod-                         additional intron relative <span id='am-223' about='obo:NCBITaxon_7165' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_44534 obo:NCBITaxon_33213 obo:NCBITaxon_41827 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7164 obo:NCBITaxon_7147 obo:NCBITaxon_44537 obo:NCBITaxon_33154 obo:NCBITaxon_7148 obo:NCBITaxon_44542 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7157 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_43816 obo:NCBITaxon_43786 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-224' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasRelatedSynonym" content="Anopheles gambiae Giles, 1902" datatype="xsd:string"></span><span id='am-226'  ></span><span id='am-227' property="rdfs:label" content="Anopheles gambiae" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasExactSynonym" content="African malaria mosquito" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasRelatedSynonym" content="Anopheles gambia" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasExactSynonym" content="Anopheles gambiae sensu stricto" datatype="xsd:string"></span>Anopheles gambiae</span> and <br /> ing exons present Hymenoptera models                             Drosophila species (Figure 6). Of particular are <br /> these exons clearly present genomic scaffold                          coding sequences encompassing D. melanogaster exons 6 <br /> used generate original models. In lepidopter-                      7, region contains orthologous motifs <br /> ans B. mori <span id='am-17' about='obo:NCBITaxon_13037' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_151542 obo:NCBITaxon_33208 obo:NCBITaxon_13036 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_344705'><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="Danaus plexippus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="monarch butterfly" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Danaus (Danaus) plexippus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="American monarch" datatype="xsd:string"></span>Danaus plexippus</span>, region similar D.                          described exon 6 largest cnn exon D. <br /> melanogaster exon 4 apparently lost. In beetle                       melanogaster. In non-dipteran species examined, exon 7 <br /> <span id='am-76' about='obo:NCBITaxon_7070' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_1304792 obo:NCBITaxon_7065 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7069 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="red flour beetle" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="rust-red flour beetle" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Tribolium castaneum" datatype="xsd:string"></span>Tribolium castaneum</span>, exons span                          single terminal exon cases, homologous <br /> region correlated D. melanogaster exons 4 5,                         exon 6 region appears split three, four, &#65533;?ve <br /> the peptide sequences exons similar                      introns, depending species (Figure 6). Unlike the <br /> of species study. The results analysis                  extensive MPOs Drosophila (Figure 2) as <br /> are shown Figure 6.                                                            mosquito A. egypti (data shown), MPOs the <br />  <br />  <br />  <br />                                                                                                                               Rapid Evolution Cnn        911 <br />  Figure 6 cnn intron&#8211;exon structure changes signi&#65533;?cantly insect orders. A comparison Cnn-PA coding exons relative D. melanogaster in <br /> the <span id='am-216' about='obo:NCBITaxon_7147' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="flies" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-219'  ></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="flies" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="Diptera" datatype="xsd:string"></span>Diptera</span> (A), Hymenoptera (B), Lepidoptera coleopteran (C) shows multiple exon fusions splitting events occurred cnn <br /> orthologs. Coding exons color-coded Figure 3 indicate sequence homology. The gray boxes T. castaneum homology other <br /> Cnn proteins, motifs structure conserved, lepidopterans exons homologous D. melanogaster exon 4. Although <br /> homologous exons split hymenopterans D. plexippus, overall arrangement coding exons similar Drosophila. As Drosophila, <br /> gene size (left) variable. <br />  <br />  <br /> species shown Figure 6, exception Hyme-                  Sequence divergence Cnn genus Drosophila <br /> noptera (discussed below), approximately co-extensive                   insects <br /> with coding exons. The presence multi-exon MPOs                   As noted above, highly conserved domains <br /> Drosophila sp. A. gambiae, species                  amino acid sequence Cnn-PA genus <br /> larger genomes, suggests multi-exon MPOs as-                    Drosophila. This stands contrast fact pro- <br /> sociated reduced genome size.                                          tein evolving rapidly. To determine nature <br />    Although ancestral intron&#8211;exon structure cnn               rapid change, compared sequences Cnn-PA <br /> known, likely multiple exon                   dipterans represented sample (Figure 7). As <br /> fusions dipteran lineages investigated. This consis-              seen, percentage identity amino acid se- <br /> tent &#65533;?nding reduction genome size                   quence diverges sharply genus Drosophila and <br /> Drosophila (Petrov Hartl 1997, 1998; Petrov et al. 2000;                dissimilar comparing Drosophila spe- <br /> Petrov 2002) A. gambiae (Nene et al. 2007)                    cies mosquitos. Members melanogaster <br /> deletions intronic intergenic regions. Since imme-               subgroup (<span id='am-205' about='obo:NCBITaxon_7220' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-206' property="rdfs:label" content="Drosophila erecta" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-208'  ></span><span id='am-209' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Drosophila erecta</span> D. yakuba) 92% iden- <br /> diate gain complete intron likely deleterious               tity D. melanogaster 95% other. Addi- <br /> effects protein function introduce premature stop                   tional evidence similarity gapping produce <br /> codon, multi-exon MPOs represent intermediate                    alignment minimal. As moves D. mel- <br /> step process exon fusion.                                     anogaster (D. biarmipis, D. takahashii, D. &#65533;?cusphila, and <br />  <br />  <br />  <br /> 912     R. C. Eisman T. C. Kaufman <br />  D. elegans), identity D. melanogaster drops aver-      mellifera group ants, gaps needed to <br /> age 85% concomitant increase gapping              achieve alignment moderately high. The Hy- <br /> achieve alignment. The members ananassae               menoptera <span id='am-138' about='obo:NCBITaxon_7424' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7423 obo:NCBITaxon_272242 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140'  ></span><span id='am-141' property="rdfs:label" content="Nasonia" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Nasonia</span> viripenis, parasitic wasp. It the <br /> group (<span id='am-192' about='obo:NCBITaxon_7217' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_545632 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_480118 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32347 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-193'  ></span><span id='am-194' property="rdfs:label" content="Drosophila ananassae" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasRelatedSynonym" content="Drosophila annanassae" datatype="xsd:string"></span>Drosophila ananassae</span> <span id='am-298' about='obo:NCBITaxon_42026' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_186282 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_480118 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32347 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-299' property="rdfs:label" content="Drosophila bipectinata" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-302'  ></span>Drosophila bipectinata</span>)           divergent group 50&#8211;60% sequence identity <br /> are diverged, showing 78% 79% identity           compared bees ants. Gapping mod- <br /> to D. melanogaster increase gapping. This trend         erately high. The sequence divergence seen Hy- <br /> continues <span id='am-371' about='obo:NCBITaxon_32355' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-372'  ></span><span id='am-373' property="rdfs:label" content="obscura group" datatype="xsd:string"></span><span id='am-374' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>obscura group</span> (D. pseudoobscura) and,      menoptera magnitude seen the <br /> <span id='am-153' about='obo:NCBITaxon_7260' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_480118 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_32367 obo:NCBITaxon_32365 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-155'  ></span><span id='am-156' property="rdfs:label" content="Drosophila willistoni" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila willistoni</span>, 67% 65% identity D. mela-           comparison Drosophila species belonging two <br /> nogaster, respectively, additional increment gap-      subgenera (Drosophila Sophophora). The lepidop- <br /> ping. It interesting note comparisons      teran species (B. mori D. plexippus) diverged at <br /> and aforementioned groups          similar level, showing 67% identity. When are <br /> subgenus Sophophora similar levels divergence          compared hymenopteran species, percentage of <br /> observed comparison D. melanogaster. The        identity 11% signi&#65533;?cant number gaps needed <br /> members subgenus Drosophila (<span id='am-174' about='obo:NCBITaxon_7230' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_198037 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_480118 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_32281 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32324 obo:NCBITaxon_33208 obo:NCBITaxon_32321 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasRelatedSynonym" content="Drosophila mohavensis" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Drosophila mojavensis" datatype="xsd:string"></span>Drosophila mojavensis</span>,        achieve alignment signi&#65533;?cant difference in <br /> <span id='am-180' about='obo:NCBITaxon_7244' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_32335 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_32281 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-181' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasRelatedSynonym" content="Drosophila irilis" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasRelatedSynonym" content="Drosophila virilis Sturtevant, 1916" datatype="xsd:string"></span><span id='am-184'  ></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="Drosophila virilis" datatype="xsd:string"></span>Drosophila virilis</span>, <span id='am-210' about='obo:NCBITaxon_7222' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_32378 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_48384 obo:NCBITaxon_48301 obo:NCBITaxon_480118 obo:NCBITaxon_48302 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_46877 obo:NCBITaxon_504493 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-211' property="rdfs:label" content="Drosophila grimshawi" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasRelatedSynonym" content="Idiomyia grimshawi" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-214'  ></span><span id='am-215' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Drosophila grimshawi</span>) roughly     protein size. This level divergence seen compar- <br /> equivalently diverged D. melanogaster          ison single coleopteran (T. castaneum) D. mela- <br /> other range 54&#8211;68% identity signi&#65533;?cant level      nogaster. This level divergence compares species <br /> of gapping achieve alignment. Finally, comparison     different insect orders points reason dif&#65533;?culty <br /> two mosquitos (A. gambiae A. aegypti) shows amino        discovering orthologs cnn reason molecular <br /> acid identity drops 25&#8211;30% range large num-         probes designed D. melanogaster sequence rapidly loose <br /> ber gaps required &#65533;?nd level alignment.         potency phylogenetic distance. Despite rapid <br /> Interestingly, neighbor-joining tree derived solely    divergence Cnn protein sequence, neighbor-joining <br /> Cnn sequence precisely tracks consensus phylogeny         tree derived sequences, case the <br /> the genus Drosophila relationship mosquitos.       Diptera, closely tracks consensus phylogeny species <br />    Since Cnn seen diverge rapidly           used analysis (Figure 9). <br /> within genus Drosophila highly            Pair-wise comparisons structure Cnn protein <br /> conserved motifs, wanted positions           models hymenopterans lepidopterans, similar to <br /> domains conservation relative diverg-       dipterans, performed, selected results <br /> ing. As noted above, Cnn-PA orthologs similar coiled-      shown Figure 10. As dipterans, coiled-coil <br /> coil domain arrangement similar spacing KFC,           structure relative positioning KFC, SESAW, SPD, <br /> SESAW, SPD, C-terminal motifs allowing pairwise           C-terminal motifs conserved A. mellifera and <br /> comparisons D. melanogaster/D. virilis, A. gambiae/       N. vitripenis lepidopterans, B. mori D. plex- <br /> A. aegypti, D. melanogaster/A. gambiae (Figure 8). The        ippus. This conservation structure independent overall <br /> KFC motif near amino terminus              protein size lepidopteran proteins half <br /> a coiled-coil domain SESAW SPD motifs           length dipteran hymenopteran examples. <br /> in helical domains &#64258;ank largely coiled-coil domain.           As Diptera, KFC motif near amino <br /> The C-terminal motif separated SPD motif           terminus coiled-coil domain, SESAW SPD are <br /> a short domain mixed coiled-coil helical character.      located COOH terminus helical domains that <br /> The regions conserved motifs            &#64258;ank coiled-coil region, C terminus separated <br /> conserved, percentage identity mirrors seen     SPD short span combined coiled-coil/helical <br /> in comparison Cnn-PA proteins. It case     character. The regions conserved motifs less <br /> that essentially gaps needed alignment      conserved level divergence and <br /> found regions conserved motifs.                gapping seen entire protein. These results clearly <br />    A similar comparison insects        essentially divergence occurs regions of <br /> the results comparison shown Figure 9. Since 8     simple coiled-coil domains helical protein folds. Finally, <br /> of 11 species considered hymenopterans, based         comparison D. melanogaster A. mellifera B. <br /> our comparisons honeybee A. mellifera. The           mori demonstrates that, despite signi&#65533;?cant differences in <br /> species bee (A. mellifera, <span id='am-337' about='obo:NCBITaxon_30195' typeof='owl:Thing obo:NCBITaxon_83311 obo:NCBITaxon_33340 obo:NCBITaxon_70988 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_28641 obo:NCBITaxon_33154 obo:NCBITaxon_7458 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_34735 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_144708 obo:NCBITaxon_50557'><span id='am-338' property="oboInOwl:hasExactSynonym" content="large earth bumblebee" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasExactSynonym" content="buff-tailed bumblebee" datatype="xsd:string"></span><span id='am-340'  ></span><span id='am-341' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasRelatedSynonym" content="Bombus terrestri" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="Bombus terrestris" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Bombus terrestris</span>, Bombus       gene structure (exon&#8211;intron positions), protein size, and <br /> impatiens) relatively high levels sequence identity    different interspersion helical coiled-coil domains, <br /> A. mellifera 85%. The alignment does require modest          relative disposition conserved motifs the <br /> number gaps, however. The bees genus Bombus         structure protein maintained. It shown <br /> show high level sequence identity (98%)        domain architecture conserved orthologs <br /> and require gaps achieve alignment. A comparison            high degree conservation likely be <br /> with ant species (<span id='am-25' about='obo:NCBITaxon_13390' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_7479 obo:NCBITaxon_72773 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-26' property="oboInOwl:hasExactSynonym" content="carpenter ants" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Camponotus" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:84542" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Camponotus</span> &#64258;oridanus, <span id='am-314' about='obo:NCBITaxon_64782' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_143999 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_34695 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-315' property="rdfs:label" content="Acromyrmex" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-318'  ></span>Acromyrmex</span>           necessary functional conservation (Forslund et al. <br /> echinatior, S. invicta, H. salator) yields sequence identi-   2011). Although motifs used study are <br /> ties 60&#8211;80% range comparison A.            curated domains, results suggest that <br />  <br />  <br />  <br />                                                                                                       Rapid Evolution Cnn   913 <br />  Figure 7 Sequence divergence frequent indels associated rapid evolution Cnn-PA Diptera. Pair-wise comparisons Cnn-PA <br /> proteins Drosophila mosquitoes (gray boxes) percentage identity protein decreases (section diagonal solid boxes) <br /> and indels accumulate (section diagonal solid boxes) relatively short periods time. A neighbor-joining tree Cnn-PA proteins <br /> showing distance based number differences gaps distributed proportionally shows molecular phylogeny cnn consistent <br /> with accepted organismal phylogeny species. Cnn-PA protein size shown species (top row) haploid genome sizes in <br /> parentheses (left column). <br />  <br />  <br /> the domain architecture cnn orthologs similar                 species. Based pairwise comparisons closely <br /> that motif required functions at-                  related species study, likely would <br /> tributed Cnn D. melanogaster.                                          minimal allelic variation cnn species. To assess <br />                                                                              evolution Cnn species, used available <br /> Sequence divergence Cnn population of <br />                                                                              sequences 162 lines single North Carolina pop- <br /> D. melanogaster <br />                                                                              ulation D. melanogaster (http://www.dpgp.org/; http:// <br /> Clearly, rapid divergence Cnn characterized                     service004.hpc.ncsu.edu/mackay/Good_Mackay_site/DBRP. <br /> sequence changes multiple insertions deletions                       html; data shown). These lines representative the <br />  <br />  <br />  <br /> 914     R. C. Eisman T. C. Kaufman <br />  Figure 8 The divergence Cnn- <br /> PA coiled-coil domains dipter- <br /> ans. A schematic representation <br /> of divergence dipteran <br /> Cnn-PA conserved motifs (black <br /> bars) coiled-coil domains <br /> (shaded bars) shows average <br /> percentage identity the <br /> number gaps needed for <br /> alignment region.       39 <br /> The least-conserved motif the <br /> carboxy terminus. The divergence <br /> graph structural models <br /> for (top pair) D. melanogaster <br /> and D. virilis, (middle pair) A. <br /> gambiae A. aegypti, D. <br /> melanogaster A. gambiae. <br /> Arrowheads indicate exon splice <br /> sites species. <br />  <br />  <br />  <br />  <br />  Rapid Evolution Cnn      915 <br />  Figure 9 A lower limit divergence number indels Cnn-PA orthologs. A table similar Figure 7 showing divergence Cnn- <br /> PA insects study reveals similar trend dipterans. Comparisons orders suggest lower limit sequence <br /> divergence upper limit number indels tolerated Cnn-PA. <br />  <br />  <br /> haploid genomes present single population                     future experimental studies required that <br /> time point (Mackay et al. 2012). Sixty lines                       true. The amino acid changes single- <br /> removed analysis ambiguous base calls                      nucleotide substitutions explain rapid <br /> or heterogeneity individual bases coding                  sequence divergence exons, as <br /> sequence Cnn-PA.                                                            Cnn-PA exon 4 region. This divergent exon <br />    We compared transcript protein sequences                     (s) orthologs; lost lepidopterans and <br /> reference, wild-type Cnn-PA sequence, FlyBase                      produces multiple gaps alignments species in <br /> found 83 variable nucleotide positions including 23 non-                       study. To investigate effect potential deletions <br /> synonymous substitutions. These changes translate 29                      North Carolina population, took sequences <br /> protein variants plus wild-type protein                     single doublet ambiguous base calls (&#8220;n&#65533;?), removed <br /> four amino acid changes Cnn-PA (Table 1). Approxi-                          sequence, translated sequence to <br /> mately two-thirds substitutions coiled-coil                      analyze resultant peptides. Surprisingly, the <br /> regions, substitutions nonsimilar amino acid                      translated peptides approximately length as <br /> residues (Figure 11). While variants pres-                    D. melanogaster reference sequence diverged <br /> ent just genomes, frequent,                    nearly 100% point deletion. We pre- <br /> and variant changes frequent                      viously reported use overlapping reading <br /> designated wild-type sequence (14 lines each).                                 frames used produce unique splice variants same <br />    These results signi&#65533;?cant protein                   genomic sequence (Eisman et al. 2009). The result the <br /> variation Cnn-PA population, assume                         North Carolina sequences demonstrates exons in <br /> that alleles functionally compatible,                       cnn multiple reading frames spanning large portions of <br />  <br />  <br />  <br /> 916     R. C. Eisman T. C. Kaufman <br />  Figure 10 The divergence Cnn-PA consistent insects. A schematic representation divergence Cnn-PA, similar Figure 8, <br /> showing comparisons (top left) A. mellifera (bottom) N. vitripennis (bottom); (top right) B. mori (top) D. plexippus (bottom); (bottom <br /> left) D. melanogaster (top) A. mellifera (bottom); (bottom right) D. melanogaster (top) B. mori (bottom). The rate change the <br /> Hymenoptera Lepidoptera similar rate Diptera. The comparisons D. melanogaster lower limit divergence Cnn-PA <br /> across phylogenetic range. Arrowheads indicate exon splice sites species. <br />  <br />  <br /> the coding exon, potentially buffering nontriplet                    loss Drosophila precise imprecise genomic deletion <br /> insertions deletions coding sequence. Obviously, pro-               nonhomologous end joining (NHEJ) DNA break <br /> duce full-length transcript, splicing re-                 repair (Llopart et al. 2002; Yenerall et al. 2011). A third <br /> laxed accommodate nontriplet insertion deletion                   mechanism intron loss D. melanogaster genomic dele- <br /> an internal coding exon. The lack strong canonical splic-                 tions remove 59 intronic splice sites shorten in- <br /> ing signals cnn allow happen, ex-                    tron size capable spliced primary <br /> tensive sequencing transcripts lines                   transcript, resulting retention intron (Talerico <br /> required &#65533;?rm conclusions.                                                Berget 1994). To differentiate mecha- <br />                                                                              nisms, analyzed exon 6/7 boundary Dro- <br /> A potential buffering function MPOs in <br />                                                                              sophila species known splice sites based cDNA or <br /> Drosophila cnn <br />                                                                              RNA-Seq data. <br /> The analyses consistent conclusion                      The exon 6/7 splice boundary present species in <br /> cnn diverging rapidly nucleotide substitution                  study, suggesting intron conserved, but <br /> frequent insertions deletions coding sequence. Addi-                  changes MPOs genus Drosophila (Figure 3) <br /> tionally, intron&#8211;exon structure gene changed                  sequence spanning area divergent. <br /> considerably insect orders intron gain loss                 Consistent sequence divergence, protein sequences <br /> events. The divergent regions Cnn-PA orthologs                       &#64258;anking splice junction divergent Drosophila spe- <br /> have multiple exon fusion intron loss events Dro-                  cies. To investigate changes intron &#64258;anking cod- <br /> sophila relative orders (arrowheads, Figure 10).                    ing sequences, aligned genomic sequences the <br /> Protein alignments species study                     upstream SPD motif start downstream con- <br /> suggest short sequences &#64258;anking fusion points                   served COOH tail D. melanogaster, Drosophila biar- <br /> divergent orders continue evolve rapidly                      mipes, D. pseudoobscura. The splicing based on <br /> after fusion event orders. Global analyses Dro-                cDNA, RNA-Seq, EST data, respectively. <br /> sophila shown introns lost reverse                      The intron 61 nucleotides D. melanogaster D. <br /> transcriptase mechanism, resulting perfect excision                pseudoobscura 57 nucleotides D. biarmipes. Relative <br /> the intron (Coulombe-Huntington Majewski 2007;                           D. melanogaster, combination triplet nontriplet <br /> Yenerall et al. 2011). A rare second mechanism intron                     deletions introduce in-frame stop codon D. biarmipes, <br />  <br />  <br />  <br />                                                                                                                      Rapid Evolution Cnn       917 <br />           Table 1 Cnn-PA allelic variation population <br />  <br />          North Carolina lines             Variable nucleotide positions                Nonsynonymous positions                 Alleles protein variants <br />                    105                                     83                                        23                              29 + wild type <br />  <br />                                                   No. amino acid changes (corresponding no. alleles) <br />                  1 <span id='am-308' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-309'  ></span><span id='am-310' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-313' property="rdfs:label" content="Aa" datatype="xsd:string"></span>aa</span> (9)                              2 aa (6)                          3 aa (11)                                       4 aa (3) <br />          The characterization allelic variation 105 unique strains single population (top row) shows total number nucleotide <br />          substitutions, number nonsynonymous substitutions, number protein variants. The number amino acid differences (one four) <br />          number allelic variants carrying differences shown the table. The difference nonsynonymous <br />          substitutions protein variants unique combinations nonsynonymous substitutions encoding total 29 protein variants. <br />  <br />  <br />  <br /> changing MPO structure (Figure 12A). Surprisingly,                               deletion (Figure 12B) MPOs long <br /> while D. melanogaster D. pseudoobscura introns                               secondary reading frames overlapping actual coding se- <br /> are equal lengths, offset 22 nucleotides                            quence. While somewhat surprising, previously <br /> respect coding sequence spliced different phases                           shown overlapping reading frames differen- <br /> (Figure 12A). Protein alignments reveal rapid divergence                             tially used generate Cnn splice variants (Eisman et al. <br /> and frequent indels &#64258;anking known splice junction be-                            2009). These results nontriplet insertion de- <br /> tween species (arrowheads, Figure 12A). The                              letion coding sequence near splice junction result <br /> rapid divergence accelerated rate non-                           rapid codon change buffered cou- <br /> synonymous substitutions sequence &#64258;anking intron,                             pled relaxed splicing mechanisms. <br /> nontriplet insertions deletions change multiple                                 The type MPO discussed appears have <br /> codons single event, combination mech-                           limited phylogenetic range restricted the <br /> anisms. While intron appears conserved                             genus Drosophila. However, did identify single multi- <br /> a broad phylogenetic range, sequence splice sites                            exon MPO parasitic wasp N. vitripennis absent <br /> evolve rapidly.                                                                      Hymenoptera. The multi-exon MPO spans the <br />    The results splicing Drosophila                            exons homologous exon 4 D. mela- <br /> evolve rapidly and, cases, relaxed compared                           nogaster Cnn-PA intron exon 4 and <br /> to canonical splicing mechanisms. Additionally,                             exon 5 (Figure 12C). In Hymenoptera, &#65533;?rst two <br /> parsimonious explanation rapid protein divergence                             exons homologous exon 4 homologous exon 5 are <br /> that nontriplet deletions caused frameshifts                            conserved. The fragment exon 4 homology, exon <br /> divergent peptides actually encoded unique sets                            4c, length A. mellifera N. vitripennis, yet <br /> codons. Typically, nontriplet insertions deletion coding                       encoded peptide diverged signi&#65533;?cantly (bottom, Fig- <br /> sequence predicted generate deleterious nonsense                              ure 12C). The coding sequence nontriplet inser- <br /> mutations. One possibility extended reading                              tions deletions gaps allowed (data shown) <br /> frames associated Drosophila MPOs buffer                          or, exons length, accelerated <br /> events. We analyzed Drosophila MPOs                              substitution rate (boxed, Figure 12C). Similar above <br /> that paucity stop codon dinucleotides                          example Drosophila, parsimonious explana- <br /> exonic intronic regions. A paucity stop codon dinu-                           tion rate change nontriplet insertions or <br /> cleotides small introns insect                     deletions create multiple new codons. In addition coding <br /> species. Perhaps feature relaxed splicing mecha-                            changes exon 4c, N. vitripennis intron exons <br /> nisms act buffers premature stop codons                            4c 5 large deletion, including 59 splice site <br /> intron size reduction prior fusion exons.                              (Figure 12C), suggesting splicing evolve rapidly as <br />    To investigate potential buffering capacity                        does Drosophila. <br /> MPOs, looked exon 6/7 intron North                                      Taken together, data suggest multi-exon <br /> Carolina D. melanogaster population sequences. The major-                            MPOs, conjunction relaxed splicing mechanisms, <br /> ity lines wild-type sequence, &#65533;?ve lines                           buffer small potentially deleterious insertions and <br /> a 2-nt deletion, &#65533;?ve lines deletion single-                           deletions, especially genes regions genes small <br /> nucleotide deletion, line deletions                             introns separating coding exons. These results show <br /> an additional single-nucleotide deletion. These genomic                              comparisons MPOs species power- <br /> sequences assembled reference D. melanogaster                            ful tool detection small, signi&#65533;?cant, changes <br /> genomic sequence deletions represented &#8220;n&#65533;?                          genes. <br /> assemblies. While sequencing required to <br />                                                                                      The dominant-negative effect divergent Cnn-PA <br /> verify real deletions, fact occur <br />                                                                                      species <br /> in multiple lines suggests random sequenc- <br /> ing errors. In lines nontriplet deletions,                          The data identify Cnn-PA diverges rapidly and <br /> new reading frames extend point                               provide possible explanation processes that <br />  <br />  <br />  <br /> 918    R. C. Eisman T. C. Kaufman <br />  Figure 11 Cnn-PA allelic variation D. melanogaster population. The 23 amino acid substitutions present single population D. <br /> melanogaster mapped structural model Cnn-PA, showing amino acid substitution position model. Ten 23 <br /> amino acid substitutions chemically similar residues. In addition wild-type protein, unique combinations these <br /> substitutions produce 29 allelic variants Cnn-PA population. Brackets model positions conserved Cnn motifs, <br /> and positions exon splice junctions indicated solid triangles model. <br />  <br />  <br /> underlie changes, does account possible              function cleavage. The obvious experiments to <br /> effects protein function. The &#65533;?nding rapid diver-                 conduct express cnn orthologs D. melanogaster and <br /> gence associated coiled-coil regions suggests                test rescue ability divergent alleles D. mel- <br /> these protein folds buffer changes coiled-              anogaster native protein absent, scope <br /> coil protein folds composed simple heptad repeats              study. <br /> have minimal structural constraints (Subbiah 1989). How- <br />                                                                           Multi-exon MPOs correlated rapid divergence <br /> ever, if, example, protein forms homodimers, sequence <br />                                                                           alternative splicing <br /> divergence small indels negative effect on <br /> protein function. Divergent proteins repel other,                Although coding exon MPO, prediction is <br /> preventing dimer formation, numerous indels                     MPOs co-extensive actual coding <br /> shift functional motifs register other,             sequence. Since clearly case cnn Dro- <br /> making coexpression heteromorphic peptides                     sophila, wanted know unique phenome- <br /> possibly antagonistic, resulting mutant phenotype.                   non genes D. melanogaster contained long <br />    To test possibility, expressed D. mela-                 MPOs relative known coding sequences. To identify can- <br /> nogaster GFP::Cnn-PA fusion protein syncytial devel-               didate genes, queried annotated exon asked <br /> opment D. melanogaster, D. simulans, D. yakuba, D.                 longer MPO extended 59 39 <br /> pseudoobscura. In live embryos, GFP localizes centro-              ends annotated exon. We MPOs D. <br /> some normally D. melanogaster D. simulans,                    melanogaster varied essentially size <br /> accumulates relatively high levels D. yakuba                 annotated exon extending 14,000 nucleotides <br /> an abnormal pattern, low variable accumula-                   59 39 direction annotated end exon. <br /> tion levels D. pseudoobscura (Figure 13, A-D).                         We identi&#65533;?ed 398 genes containing MPO <br />    In &#65533;?xed embryos immunostained detect Cnn                   extending 1000 nucleotides end of <br /> fusion protein native Cnn, Cnn staining similar            annotated exon, representing  3% genes D. <br /> live images prophase (Figure 13, &#8220;D. yakuba&#65533;? col-                  melanogaster. Since wanted manually annotate several <br /> umn). During metaphase, defects associated accu-                 genes range Drosophila species identify simi- <br /> mulation fusion protein apparent. In D.                     larities evolution cnn, restricted list to <br /> simulans D. yakuba, centrosomes detach spin-                 genes MPO extending .3000 nucleotides beyond <br /> dle and, D. yakuba, centrosome replication precocious               annotated exon. There 39 genes category <br /> at spindles (Figure 13, Q-T). Metaphase spindles D.               chose 7 traditional Drosophila names, <br /> pseudoobscura defective, having multiple poles               genes likely essential developmental <br /> and fusion spindles (Figure 13, T), similar                genes characterized genetic molecular level. <br /> cnn mutant phenotypes described D. melanogaster                        The extent analysis changes our <br /> (Megraw et al. 1999; Eisman et al. 2009). Consistent                 candidate genes, study report high- <br /> this phenotype, development fails embryos.               lights related target MPOs identi&#65533;?ed screen <br /> The control embryos experiments normal                       (Table 2; Supporting Information: Figure S1, Figure S2, Fig- <br /> throughout cell cycle (Figure 13, E-H M-P) de-                ure S3, Figure S4, Figure S5. <br /> velopment proceeds normally.                                                 Of seven genes investigated, reported be <br />    These data demonstrate coexpression divergent                  rapidly evolving Drosophila. Only shortstop (shot) is <br /> alleles cnn dominant-negative effect protein                  conserved, target MPO, evolving <br />  <br />  <br />  <br />                                                                                                                  Rapid Evolution Cnn     919 <br />  Figure 12 Splicing changes nontriplet indels create new codons Cnn-PA orthologs. (A) Rapid changes splicing intron exon 6 <br /> (green) exon 7 (red) D. melanogaster, D. biarmipes, D. pseudoobscura indels shortened intron D. biarmipes moved the <br /> intron D. pseudoobscura. The exon MPO (light red) maps (top) aligned SPD motif (left) start carboxy terminus (right) <br /> showing effect changes exon position. A single substitution D. biarmipes introduced stop codon (boxed, middle), changing the <br /> MPO. The aligned protein sequences (bottom) showing splice sites (arrowheads) rapid divergence associated changes, <br /> which typical existing splice sites orthologs sites exon fusion. The evidence suggests changes nontriplet indels <br /> and new codon usage coding sequence relaxed splicing. (B) An alignment region North Carolina lines shows multiple <br /> lines begun accumulate nontriplet indels (top). These changes change splicing, reveal buffering capacity MPOs (light <br />  <br />  <br />  <br />  <br /> 920     R. C. Eisman T. C. Kaufman <br />  rapidly, demonstrating divergent MPOs exist other-               &#65533;?cation additional orthologs, accurate functional annota- <br /> wise conserved genes. Target MPOs dumpy (dp), <span id='am-350' about='obo:NCBITaxon_82038' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_44985 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_703547 obo:NCBITaxon_1068921 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-351' property="oboInOwl:hasRelatedSynonym" content="Prospero Salisb." datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-354' property="rdfs:label" content="Prospero" datatype="xsd:string"></span><span id='am-355'  ></span>prospero</span>                    tion, future investigations processes and <br /> (pros), short shot, mushroom body defective (mud)                    compensatory mechanisms potentially underlie the <br /> known alternatively spliced intron retention/exclu-              rapid evolution cnn. <br /> sion (IRE) mechanism uncharacterized splice                    Based analyses, cnn orthologs appear re- <br /> variants dp. The target MPOs rhinocerous (rno), lava lamp              stricted insects, agreement OrthoDB Egg- <br /> (lva), Futsch, mud associated signi&#65533;?cant vari-              NOG orthologous gene sets, additional genomes <br /> ation gene models genus Drosophila,                   expand range groups. Orthologous <br /> manual annotation MPOs &#65533;?nds reason                      Cnn proteins highly conserved KFC, SESAW, <br /> considerable variation (Figure S1, Figure S2, Figure S3, Figure              SPD motifs, divergent carboxy ter- <br /> S4, Figure S5, File S1, Figures 2&#8211;5). However, &#65533;?nd                 minus. The arrangement motifs coiled-coil <br /> strong evidence genes studied loss              domains conserved proteins vary size from <br /> gain introns outside target MPOs, require               707 amino acids D. plexippus 1424 amino acids T. <br /> future sequencing verify changes. Finally, target               castaneum. This conservation structure suggests func- <br /> MPOs encode coiled-coil simple repetitive protein              tion conserved (Forslund et al. 2011) provides <br /> domains, similar MPOs cnn.                                         schematic Cnn domain architecture direct future <br />    In general, regions high conservation separated                     studies. <br /> regions high divergence characterize peptides                         The somewhat limited phylogenetic range Cnn ortho- <br /> encoded target MPOs discussed above. The divergent                    logs provides new insight functional evolution <br /> regions introduce multiple small gaps aligned sequences                   Cnn. In D. melanogaster, Cnn required males during <br /> and result signi&#65533;?cant variation protein length                   spermatogenesis (Li et al. 1998) maternal-effect <br /> species similar cnn orthologs. Our target                embryonic lethal gene females, required centro- <br /> MPOs appear confound gene model algorithms                        syncytial development cellularization <br /> be associated changes intron&#8211;exon structure                   (Megraw et al. 1999; Vaizel-Ohayon Schejter 1999; <br /> genes. While mechanisms clear, presence                   Eisman et al. 2009). Other stages develop- <br /> reading frames extending known end                        ment, maternal supply Cnn+ oocytes suf&#65533;?cient for <br /> coding exons apparently indicative rapid evolution.                    development morphologically normal sterile adult <br />                                                                              &#64258;ies. Since type development common species <br />                                                                              study, Cnn novel centrosomal protein that <br /> Discussion <br />                                                                              evolved organize actin microtubule cytoskeleton in <br /> The rapid divergence cnn problematic                    absence cell membranes. The ancestral gene genes <br /> intriguing lab &#65533;?rst described gene                      cnn derived remains unknown time. <br /> (Heuer et al. 1995). Surprisingly, rate divergence                        One possibility cnn chimera derived mul- <br /> per million years cnn rapid rate reported               tiple genes, suggested superfamily proteins with <br /> for vertebrate albumin genes (Doolittle 1995)                   KFC motif. Experimental data superfamily exist <br /> rapid rate 20&#8211;30% cytoskeletal proteins pre-                 fungal genes Anucleate primary sterigmataB (AspB) in <br /> dicted report. The rapid divergence                  <span id='am-70' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-71'  ></span><span id='am-72' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span>Aspergillus</span> nidulans (Westfall Momany 2002), microtu- <br /> dif&#65533;?cult identify domains motifs necessary Cnn                    bule organizer 1 (Mto1p) <span id='am-53' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-54' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span>Schizosaccharomyces</span> pombe <br /> function, essential directed molecular dissec-                (Sawin et al. 2004), vertebrate genes mmg <span id='am-361' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-362' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-364'  ></span><span id='am-365' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>- <br /> tion protein. Moreover, rapid evolution                 phodiesterase 4d interacting protein (Pde4dip) (Verde et al. <br /> publicly available orthologous gene sets makes impossible                 2001) CDK5RAP2, cnn, D. melanogaster. <br /> to completely rule orthologous relationships in-                   The KFC motif Mto1p (Sawin et al. 2004), CDK5RAP2 <br /> sect, vertebrate, fungal genes encoding KFC motif.                   (Fong et al. 2008), Cnn (Megraw et al. 2001; Zhang <br /> Although rapid evolution problematic,                       Megraw 2007) nucleates g-tubulin microtubule- <br /> always intrigued processes drive diver-                     organizing centers function mechanistically <br /> gence compensatory mechanisms exist Drosoph-                     different fungi, vertebrates, &#64258;ies (Fong et al. 2008). <br /> ila offset rapid changes essential developmental                       Unlike proteins, AspB involved nuclear migra- <br /> proteins. For reasons, characterized cnn                     tion spore formation colocalizes F-actin <br /> orthologous gene set insects, useful identi-                (Westfall Momany 2002), Mmg interacts with <br />  <br />  <br />  <br /> red boxes, bottom) comparisons MPOs useful detection small signi&#65533;?cant changes sequence. (C) A <br /> schematic (top) comparison MPOs (light red) exon 4c (blue) exon 5 (brown) A. mellifera N. vitripennis suggest change <br /> has occurred. Comparisons aligned introns (top) large deletion, different nontriplet indels N. vitripennis signi&#65533;?cantly <br /> changed splicing. Alignments coding sequence (middle, boxed) translated peptide encoded exon 4c nontriplet indels <br /> have generated new codons or, unknown reason, nucleotide substitution rate accelerated exon. <br />  <br />  <br />  <br />                                                                                                                       Rapid Evolution Cnn       921 <br />                                                                                                                      Figure 13 Minimal divergence <br />                                                                                                                     proteins perturbs Cnn- <br />                                                                                                                     PA function development. <br />                                                                                                                     The ectopic expression D. <br />                                                                                                                     melanogaster EGFP::Cnn-PA fu- <br />                                                                                                                     sion protein D. melanogaster, <br />                                                                                                                     D. simulans, D. yakuba, D. <br />                                                                                                                     pseudoobscura syncytial <br />                                                                                                                     development shows minimal di- <br />                                                                                                                     vergence            proteins <br />                                                                                                                     expressed heterospeci&#65533;?c <br />                                                                                                                     embryo disrupts normal cleav- <br />                                                                                                                     age. Live imaging EGFP (A&#8211;D) <br />                                                                                                                     shows tagged protein <br />                                                                                                                     localizes centrosome, but <br />                                                                                                                     D. yakuba protein less <br />                                                                                                                     punctate diffuse and <br />                                                                                                                     D. pseudoobscura amount <br />                                                                                                                     tagged protein centrosomes <br />                                                                                                                     reduced variable. Immu- <br />                                                                                                                     nostaining &#65533;?xed embryos rec- <br />                                                                                                                     ognizes tagged and <br />                                                                                                                     native Cnn protein similar <br />                                                                                                                     observed live ani- <br />                                                                                                                     mals prophase (compare <br />                                                                                                                     I&#8211;L A&#8211;D). Although centro- <br />                                                                                                                     somes obvious D. <br />                                                                                                                     pseudoobscura &#65533;?xed material, <br />                                                                                                                     signi&#65533;?cant variability <br />                                                                                                                     nuclei (L). During <span id='am-282' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-283'  ></span><span id='am-284' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>meta</span>- <br />                                                                                                                     phase (Q&#8211;T) defects clear in <br />                                                                                                                     D. simulans (Q) D. yakuba <br />                                                                                                                     (S) indicated free centro- <br />                                                                                                                     somes (white arrowheads). Either <br />                                                                                                                     centrosome replication preco- <br />                                                                                                                     cious centrosomes split at <br />                                                                                                                     poles D. yakuba. The <br />                                                                                                                     phenotype seen D. pseudoobs- <br />                                                                                                                     <span id='am-303' about='obo:NCBITaxon_69520' typeof='obo:NCBITaxon_6157 owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_31262 obo:NCBITaxon_147100 obo:NCBITaxon_33154 obo:NCBITaxon_166126 obo:NCBITaxon_1292243 obo:NCBITaxon_1292253 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6159'><span id='am-304' property="rdfs:label" content="Cura" datatype="xsd:string"></span><span id='am-305' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-306' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-307'  ></span>cura</span> interesting, spindles <br />                                                                                                                     similar spindles cnn loss- <br />                                                                                                                     of-function mutant embryos. <br />                                                                                                                     However, D. pseudoobscura <br />                                                                                                                     (T), Cnn centrosomes, <br />                                                                                                                     spindle poles multiple cen- <br />                                                                                                                     trosomes spindles are <br />                                                                                                                     multipolar. These results show <br />                                                                                                                     ,10% divergence sin- <br />                                                                                                                     gle indel coex- <br />                                                                                                                     pressed       Cnn     proteins     is <br />                                                                                                                     suf&#65533;?cient signi&#65533;?cantly perturb <br /> normal function. Control animals expressing EGFP-tagged Cnn appear normal cell cycle (prophase: E&#8211;H; metaphase: M&#8211; <br /> P). Fixed embryos (E&#8211;T) stained anti-Cnn (green) anti-b-tubulin (red) DNA stained TOTO3 (blue). <br />  <br />  <br /> cyclic nucleotide phosphodiesterase (Verde et al. 2001)                      Functional data SESAW SPD motifs the <br /> A-kinase (Uys et al. 2011) vertebrate muscles. Addition-                   carboxy terminus limited. Global phosphorylation <br /> ally, proteins share signi&#65533;?cant homology outside                  studies D. melanogaster shown SESAW and <br /> the KFC motif &#64258;anking domain, unique motifs                      SPD motifs phosphorylated (http://www.phosphopep. <br /> and domains present proteins, removal                    org/), suggesting potentially important regu- <br /> of KFC motif BLAST searches returns completely                      latory motifs used cell cycle. The function the <br /> different set &#8220;best hits.&#65533;? Taken together, diverse func-               carboxy terminus unknown, have <br /> tions limited homology suggest superfamily                      shown loss SPD motif carboxy terminus <br /> may epaktologs inherited KFC domain in-                      results low levels truncated Cnn centrosome <br /> dependently different lineages (Nagy et al. 2011a),                 high levels metaphase plate (Eisman et al. <br /> than orthologs related descent.                                            2009). A recent study reported motif Cnn <br />  <br />  <br />  <br /> 922     R. C. Eisman T. C. Kaufman <br />  Table 2 Divergence genes multi-exon MPOs Drosophila <br />  <br /> Gene/protein ID              Gene size range (kb)                Protein size range (aa)               Alternative splicing mechanism                   Exons annotated <br /> Shot-PH                                                                  3070&#8211;3498                                   Yes (IRE)                             Exon 14 only <br /> Dp                            Incomplete sequences                                                                Yes (Unknown) <br /> Pros-PK                            10.8&#8211;20.0                             1403&#8211;519                                    Yes (IRE)                             Entire   gene <br /> Mud-PL                             10.3&#8211;16.5                             1641&#8211;2756                                   Yes (IRE)                             Exons    2&#8211;11 <br /> Rno-PB                             11.0&#8211;14.8                             3201&#8211;4386                                      No                                 Entire   gene <br /> Lva-PC                              7.9&#8211;9.5                              2406&#8211;2837                                      No                                 Exons    2&#8211;8 <br /> Futsch-PC                          16.7&#8211;21.1                             4922&#8211;6507                                      No                                 Exons    3&#8211;8 <br /> A summary comparative analyses seven genes Drosophila species based manual annotations. These genes &#65533;?rst identi&#65533;?ed D. melanogaster <br /> due presence extremely long MPO compared annotated coding exons. Only target exon SHOT-PH protein evolving rapidly, analysis <br /> was exon. Dp sequences incomplete species, hindering complete analysis gene. For additional details, File S1 Figure S1, <br /> Figure S2, Figure S3, Figure S4, Figure S5. <br />  <br />  <br />  <br />  <br /> carboxy terminus conserved mouse CDK5RAP2 (Wang                                    mulation indels occurs speciation. However, pre- <br /> et al. 2010), PsiBLAST D. melanogaster motif                              sented Results, evidence nontriplet <br /> found minimal homology, E-value 7.3 &#64258;y                                indels exon 4, suggesting types changes <br /> and mouse. Clearly, future studies Cnn required                                exist single population. Future RNA-seq anal- <br /> de&#65533;?ne molecular basis Cnn function.                                               yses lines North Carolina population be <br />     The conservation Cnn domain architecture                                    required verify validity cause apparent <br /> striking contrast divergence orthologous gene                                  rapid changes coding sequence. <br /> architecture, gene size, coding sequences. In general,                                   A key question regarding protein insertions deletions <br /> the intron&#8211;exon structure cnn conserved insect                              underlying nucleotide indels triplet or <br /> orders strikingly different orders. Relative                           nontriplet changes. The presumably removes single <br /> the dipterans, Cnn encoded exons species                               codon causes frameshift. The assump- <br /> investigated trend exon fusion                                 tion global analyses protein indels result of <br /> dipterans. The common mechanism intron loss                                  perfect insertion deletion codons, complete <br /> mediated reverse transcriptase (Coulombe-Huntington                                   divergence multiple amino acids surrounding indels in <br /> and Majewski 2007; Yenerall et al. 2011), results                                Cnn-PA alignments suggests true. The <br /> suggest cnn introns lost genomic deletions                                   parsimonious explanation divergence coding <br /> (Llopart et al. 2002; Yenerall et al. 2011). Our comparison                              regions &#64258;anking exon 6/7 splice site Drosophila and <br /> of exon structure cnn orthologs shows potential in-                             exon 4c/5 splice site hymenopterans nontriplet <br /> termediate step process, especially hyme-                             indels. In examples, upstream downstream con- <br /> nopterans. Initially, intronic deletions bring multiple exons                            served regions &#64258;ank divergent regions indels <br /> into close proximity other, forming exon islands.                              eventually total triplet combinations splicing <br /> These data resolve multiple exons fuse,                                   machinery restore downstream reading frame. Al- <br /> imprecise NHEJ repair (Llopart et al. 2002; Yenerall et al.                              data required address possibilities, <br /> 2011) intronic deletions resulting intron retention                                probable mechanisms resolve nontriplet <br /> (Talerico Berget 1994) plausible mechanisms. These                               indels coding sequences. <br /> two mechanisms, loss coding exons                                   The types changes driving rapid evolution Cnn- <br /> lepidopterans incorporation nonhomologous cod-                                PA suggest buffering mechanisms the <br /> ing sequence T. castaneum, account                              sequence protein level. In study, presented <br /> divergence Cnn-PA proteins.                                                           MPOs possible buffering mechanism sequence <br />     The divergence Cnn-PA proteins appears to                               level. The buffering capacity MPOs appears inherent <br /> several types changes. Imprecise exon fusion events,                                  sequence span, coding small <br /> nonsynonymous nucleotide substitution, frequent small                                    introns paucity dinucleotides needed introduce <br /> indels coding exons, and, based analyses,                                  stop codons. This results multiple overlapping reading <br /> nontriplet indels resulting frameshifts generate new                             frames genomic region, demonstrated the <br /> codons play role divergence Cnn-PA. Within                               intron exons 6 7 North Carolina lines. In <br /> orders, Cnn-PA divergence rapid uniform                                example, splicing changed, but, similar changes <br /> phylogeny, comparisons orders                                  occur coding sequence, as-yet-unknown apparently <br /> Cnn-PA minimal homology, represent                                         present splicing mechanism retain protein function. <br /> the minimal conservation necessary retain Cnn-PA                                      Perhaps sequence level evolution selects paucity <br /> function. Analysis D. melanogaster North Carolina                                 stop codon dinucleotides buffer inevitable <br /> lines suggests nucleotide substitution responsible                               deleterious deletions occur genome size is <br /> for allelic variation population accu-                             signi&#65533;?cantly reduced. <br />  <br />  <br />  <br />                                                                                                                                         Rapid Evolution Cnn             923 <br />      Although buffering sequence level makes              subset features shared <br /> possible transcribe similar primary sequence,             functions, unwise transfer function cnn <br /> resultant protein fold properly retain function.           based mutational analyses D. melanogaster genes <br /> Based Cnn-PA orthologs proteins MPO              limited homology. This orthologous gene set provides <br /> screen, rapid divergence primary sequence                valuable information future molecular transcrip- <br /> restricted structural proteins. The frequent motif         tional studies. We realize divergence cnn <br /> found study coiled-coil domain, a-helical        does &#65533;?t neatly current models, in <br /> fold composed simple heptad repeats common com-           silico global analyses multiple genomes &#8220;see&#65533;? the <br /> ponent structural proteins. Since residues sub-        small-yet-signi&#65533;?cant changes described. One way detect <br /> stitute heptad positions, protein motif   changes incorporate MPOs computer <br /> surprising fold diverge rapidly retain func-     programs comparisons MPO changes species <br /> tion. The rapid divergence Futsch suggests second        means detect single-nucleotide deletion. It is <br /> fold buffering capacity high-copy-number simple            hope work provides basis continued inves- <br /> repeats. Although did analyze evolution Dp,          tigations rare genomic changes presumably <br /> it shown repetitive PIGFEAST region           deleterious mutations drive rapid evolution essen- <br /> undergoing concerted evolution Drosophila un-            tial genes. <br /> equal crossing (Carmon et al. 2007, 2010), similar what <br /> we Futsch. While protein folds, disor- <br />                                                                    Acknowledgments <br /> dered regions, likely buffering capacity, the <br /> rapid divergence associated Cnn-PA orthologs likely        The authors thank <span id='am-345' about='obo:NCBITaxon_39170' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216706 obo:NCBITaxon_216718 obo:NCBITaxon_3398 obo:NCBITaxon_4136 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-346'  ></span><span id='am-347' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-348' property="rdfs:label" content="Melissa" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Melissa</span> Phelps Stacy Holtzman for <br /> to restricted subset protein types. An additional       outstanding technical assistance enduring moral sup- <br /> and somewhat surprising buffering capacity inherent         port; FlyBase easy access genetic genomic <br /> in insects tolerate signi&#65533;?cant differences    information; Bloomington Drosophila Stock Center for <br /> in protein size closely related species. This suggests     &#64258;y stocks; Center Genomics Bioinformatics for <br /> that molecular machinery animals relatively        help algorithm search MPOs; Baylor <br /> plastic wide range size tolerances.                      Sequencing Center Fringy Richards access the <br />     While rapid divergence buffered levels,      recently sequenced modENCODE Drosophila genomes and <br /> it reasonable expect constraints protein          associated RNAseq data; Josh Goodman help in <br /> function, especially coiled-coil domains interact form       retrieving relevant sequences Mackay Raleigh <br /> oligomers. Although know oligomerization             population sequences; and, &#65533;?nally, Matt Hahn his <br /> state Cnn-PA, simplest state Cnn-PA forms parallel     evisceration previous incarnation article. His <br /> homodimer, divergent Cnn-PA orthologs interfere          comments helpful, insightful, remaining <br /> with protein folding domain alignment. To test            de&#65533;?ciencies pigheaded refusal his <br /> hypothesis, ectopically expressed D. melanogaster             well-proffered advice. Supported Indiana Genomics <br /> GFP::Cnn-PA fusion protein D. simulans, D. yakuba,          Initiative. <br /> D. pseudoobscura syncytial development. These <br /> experiments showed fusion protein interferes with <br /> native Cnn-PA function species inten-        Literature Cited <br /> sity dominant-negative effect increases diver- <br />                                                                    Altenhoff, A. M., C. Dessimoz, 2009 Phylogenetic func- <br /> gence native protein increases. Since GFP::Cnn-PA          tional assessment orthologs inference projects methods. <br /> fusion protein rescues cnnmfs mutant phenotype D.              PLOS Comput. Biol. 5: e1000262. <br /> melanogaster, D. simulans D. yakuba Cnn-PA             Carmon, A., M. Wilkin, J. Hassan, M. Baron, R. MacIntyre, <br /> proteins similar, unlikely results          2007 Concerted evolution Drosophila dumpy gene. <br /> are GFP tag functional changes               Genetics 176: 309&#8211;325. <br /> species. This suggests that, functional homodimer         Carmon, A., M. J. Guertin, O. Grushko, B. Marshall, R. MacIn- <br />                                                                       tyre, 2010 A molecular analysis mutations complex <br /> form, degree protein divergence tolerated low and <br />                                                                       dumpy locus Drosophila melanogaster. PLoS ONE 5: e12319. <br /> indels rare species. However,               Conduit, P. T., J. W. Raff, 2010 Cnn dynamics drive cen- <br /> effects D. yakuba Cnn-PA orthologs            trosome size asymmetry ensure daughter centriole retention <br /> type divergence taking place essential devel-          Drosophila neuroblasts. Curr. Biol. 20: 2187&#8211;2192. <br /> opmental proteins, tempting propose com-         Coulombe-Huntington, J., J. Majewski, 2007 Intron loss and <br /> bined effect associated speciation, speci&#65533;?cally           gain Drosophila. Mol. Biol. Evol. 24: 2842&#8211;2850. <br />                                                                    Dobbelaere, J., F. Josue, S. Suijkerbuijk, B. Baum, N. Tapon et al., <br /> in light data presented postfertilization <br />                                                                       2008 A genome-wide RNAi screen dissect centriole dupli- <br /> isolation. <br />                                                                       cation centrosome maturation Drosophila. PLoS Biol. 6: <br />     We presented comprehensive set cnn orthologs            e224. <br /> and features likely necessary Cnn       Doolittle, R. F., 1995 The multiplicity domains proteins. <br /> function associated D. melanogaster. While proteins              Annu. Rev. Biochem. 64: 287&#8211;314. <br />  <br />  <br />  <br /> 924    R. C. Eisman T. C. Kaufman <br />  Eisman, R., T. C. Kaufman, 2007 Cytological investigation         Lupas, A., M. Van Dyke, J. Stock, 1991 Predicting coiled coils <br />     mechanism parthenogenesis Drosophila mercatorum.              protein sequences. Science 252: 1162&#8211;1164. <br />     Fly (Austin) 1: 317&#8211;329.                                             Mackay, T. F., S. Richards, E. A. Stone, A. Barbadilla, J. F. Ayroles <br /> Eisman, R. C., N. Stewart, D. Miller, T. C. Kaufman,                    et al., 2012 The Drosophila melanogaster Genetic Reference <br />     2006 centrosomin&#8217;s beautiful sister (cbs) encodes GRIP-do-            Panel. Nature 482: 173&#8211;178. <br />     main protein marks Golgi inheritance functions       Matthews, K. A., D. F. Miller, T. C. Kaufman, <br />     centrosome cycle Drosophila. J. Cell Sci. 119: 3399&#8211;3412.            1989 Developmental distribution RNA protein products <br /> Eisman, R. C., M. A. Phelps, T. C. Kaufman,                             Drosophila alpha-tubulin gene <span id='am-67' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-68' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. Dev. Biol. 132: 45&#8211; <br />     2009 Centrosomin: complex mix long short isoforms              61. <br />     required centrosome function early development      McQuilton, P., S. E. St Pierre, J. Thurmond; FlyBase Consor- <br />     Drosophila melanogaster. Genetics 182: 979&#8211;997.                         tium, 2012 FlyBase 101: basics navigating FlyBase. Nu- <br /> Fitch, W. M., 1970 Distinguishing homologous analogous                 cleic Acids Res. 40: D706&#8211;D714. <br />     proteins. Syst. Zool. 19: 99&#8211;113.                                    Megraw, T. L., K. Li, L. R. Kao, T. C. Kaufman, 1999 The <br /> Fitch, W. M., 2000 Homology personal view                  centrosomin protein required centrosome assembly and <br />     problems. Trends Genet. 16: 227&#8211;231.                                    function cleavage Drosophila. Development 126: <br /> Flory, M. R., M. Morphew, J. D. Joseph, A. R. Means, T. N.              2829&#8211;2839. <br />     Davis, 2002 Pcp1p, Spc110p-related calmodulin target           Megraw, T. L., L. R. Kao, T. C. Kaufman, 2001 Zygotic de- <br />     centrosome &#65533;?ssion yeast Schizosaccharomyces                  velopment functional mitotic centrosomes. Curr. Biol. <br />     pombe. Cell Growth Differ. 13: 47&#8211;58.                                   11: 116&#8211;120. <br /> Fong, K. W., Y. K. Choi, J. B. Rattner, R. Z. Qi,                    Mushegian, A. R., J. R. Garey, J. Martin, L. X. Liu, 1998 Large- <br />     2008 CDK5RAP2 pericentriolar protein functions             scale taxonomic pro&#65533;?ling eukaryotic model organisms: com- <br />     centrosomal attachment gamma-tubulin ring complex.               parison orthologous proteins encoded human, &#64258;y, nem- <br />     Mol. Biol. Cell 19: 115&#8211;125.                                            atode, yeast genomes. Genome Res. 8: 590&#8211;598. <br /> Forslund, K., I. Pekkari, E. L. Sonnhammer, 2011 Domain              Nagy, A., L. B&#225;nyai, L. Patthy, 2011a Reassessing domain ar- <br />     architecture conservation orthologs. BMC Bioinformatics 12:          chitecture evolution metazoan proteins: major impact errors <br />     326.                                                                    caused confusing paralogs epaktologs. Genes 2: 516&#8211;561. <br /> Galperin, M. Y., E. V. Koonin, 1998 Sources systematic            Nagy, A., G. Szl&#225;ma, E. Szarka, M. Trexler, L. B&#225;nyai et al., <br />     error functional annotation genomes: domain rearrange-            2011b Correction: Nagy, et al. Reassessing domain architec- <br />                                                                             ture evolution metazoan proteins: major impact gene pre- <br />     ment, non-orthologous gene displacement operon disrup- <br />                                                                             diction errors. Genes 2011, 2, 449&#8211;501. Genes 2: 599&#8211;607. <br />     tion. In Silico Biol. 1: 55&#8211;67. <br />                                                                          Nene, V., J. R. Wortman, D. Lawson, B. Haas, C. Kodira et al., <br /> Gorman, O. T., W. J. Bean, Y. Kawaoka, I. Donatelli, Y. J. Guo et al., <br />                                                                             2007 Genome sequence Aedes aegypti, major arbovirus <br />     1991 Evolution in&#64258;uenza A virus nucleoprotein genes: im- <br />                                                                             vector. Science 316: 1718&#8211;1723. <br />     plications origins H1N1 human classical swine <br />                                                                          Petrov, D. A., 2002 DNA loss evolution genome size in <br />     <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>viruses</span>. J. Virol. 65: 3704&#8211;3714. <br />                                                                             Drosophila. Genetica 115: 81&#8211;91. <br /> Gupta, R. S., 1998 Protein phylogenies signature sequences: <br />                                                                          Petrov, D. A., D. L. Hartl, 1997 Trash DNA gets <br />     reappraisal evolutionary relationships archaebacte- <br />                                                                             thrown away: high rate DNA loss Drosophila. Gene 205: <br />     ria, eubacteria, eukaryotes. Microbiol. Mol. Biol. Rev. 62: <br />                                                                             279&#8211;289. <br />     1435&#8211;1491. <br />                                                                          Petrov, D. A., D. L. Hartl, 1998 High rate DNA loss the <br /> Heuer, J. G., K. Li, T. C. Kaufman, 1995 The Drosophila ho- <br />                                                                             Drosophila melanogaster Drosophila virilis species groups. <br />     meotic target gene centrosomin (cnn) encodes novel centro-            Mol. Biol. Evol. 15: 293&#8211;302. <br />     somal protein leucine zippers maps genomic             Petrov, D. A., T. A. Sangster, J. S. Johnston, D. L. Hartl, K. L. <br />     region required midgut morphogenesis. Development 121:              Shaw, 2000 Evidence DNA loss determinant ge- <br />     3861&#8211;3876.                                                              nome size. Science 287: 1060&#8211;1062. <br /> Holtzman, S., D. Miller, R. Eisman, H. Kuwayama, T. Niimi et al.,        Rokas, A., P. W. Holland, 2000 Rare genomic changes as <br />     2010 Transgenic tools members genus Drosophila               tool phylogenetics. Trends Ecol. Evol. 15: 454&#8211;459. <br />     sequenced genomes. Fly (Austin) 4: 349&#8211;362.                     Sawin, K. E., P. C. Lourenco, H. A. Snaith, 2004 Microtubule <br /> Hulsen, T., M. A. Huynen, J. Vlieg, P. M. Groenen,                   nucleation non-spindle pole body microtubule-organizing <br />     2006 Benchmarking ortholog identi&#65533;?cation methods using                  centers requires &#65533;?ssion yeast centrosomin-related protein <br />     functional genomics data. Genome Biol. 7: R31.                          mod20p. Curr. Biol. 14: 763&#8211;775. <br /> Kalvakolanu, D. V., W. H. Livingston, III, 1991 Rapid            Subbiah, S., 1989 Are coiled-coil proteins evolutionarily related? <br />     inexpensive protocol generating greater 95% recombi-           J. Mol. Biol. 206: 689&#8211;692. <br />     nants subcloning experiments. Biotechniques 10: 176&#8211;177.          Talerico, M., S. M. Berget, 1994 Intron de&#65533;?nition splicing <br /> Keeling, P. J., M. A. Luker, J. D. Palmer, 2000 Evidence           small Drosophila introns. Mol. Cell. Biol. 14: 3434&#8211;3445. <br />     <span id='am-32' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Beta" datatype="xsd:string"></span>beta</span>-tubulin phylogeny <span id='am-105' about='obo:NCBITaxon_6029' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-106' property="oboInOwl:hasExactSynonym" content="microsporidians" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Microsporidia" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Microsporida" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="PMID:10563021" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasAlternativeId" content="NCBITaxon:6031" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:17051209" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:12142484" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:6030" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Microsporea" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Microspora" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:9892676" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="microsporidians" datatype="xsd:string"></span><span id='am-121'  ></span>microsporidia</span> evolved        Tatusov, R. L., E. V. Koonin, D. J. Lipman, 1997 A genomic <br />     fungi. Mol. Biol. Evol. 17: 23&#8211;31.                                  perspective protein families. Science 278: 631&#8211;637. <br /> Li, K., T. C. Kaufman, 1996 The homeotic target gene cen-            Uys, G. M., A. Ramburan, B. Loos, C. J. Kinnear, L. J. Korkie et al., <br />     trosomin encodes essential centrosomal component. Cell 85:           2011 Myomegalin novel A-kinase anchoring protein in- <br />     585&#8211;596.                                                                volved phosphorylation cardiac myosin binding protein <br /> Li, K., E. Y. Xu, J. K. Cecil, F. R. Turner, T. L. Megraw et al.,           C. BMC Cell Biol. 12: 18. <br />     1998 Drosophila centrosomin protein required male mei-        Vaizel-Ohayon, D., E. D. Schejter, 1999 Mutations centro- <br />     osis assembly &#64258;agellar axoneme. J. Cell Biol. 141:           somin reveal requirements centrosomal function during <br />     455&#8211;467.                                                                early Drosophila embryogenesis. Curr. Biol. 9: 889&#8211;898. <br /> Llopart, A., J. M. Comeron, F. G. Brunet, D. Lachaise, M. Long,      Venkatram, S., J. J. Tasto, A. Feoktistova, J. L. Jennings, A. J. Link <br />     2002 Intron presence-absence polymorphism Drosophila                 et al., 2004 Identi&#65533;?cation characterization novel <br />     driven positive Darwinian selection. Proc. Natl. Acad. Sci.          proteins affecting &#65533;?ssion yeast gamma-tubulin complex func- <br />     USA 99: 8121&#8211;8126.                                                      tion. Mol. Biol. Cell 15: 2287&#8211;2301. <br />  <br />  <br />  <br />                                                                                                                 Rapid Evolution Cnn     925 <br />  Verde, I., G. Pahlke, M. Salanova, G. Zhang, S. Wang et al.,             Yenerall, P., B. Krupa, L. Zhou, 2011 Mechanisms intron <br />    2001 Myomegalin novel protein Golgi/centrosome               gain loss Drosophila. BMC Evol. Biol. 11: 364. <br />    interacts cyclic nucleotide phosphodiesterase. J. Biol.   Zhang, J., T. L. Megraw, 2007 Proper recruitment gamma- <br />    Chem. 276: 11189&#8211;11198.                                                  tubulin D-TACC/Msps embryonic Drosophila centro- <br /> Wang, Z., T. Wu, L. Shi, L. Zhang, W. Zheng et al., 2010 Conserved          somes requires Centrosomin Motif 1. Mol. Biol. Cell 18: 4037&#8211; <br />    motif CDK5RAP2 mediates localization centrosomes           4049. <br />    Golgi complex. J. Biol. Chem. 285: 22658&#8211;22665.                   Zhang, X., J. Lee, L. A. Chasin, 2003 The effect nonsense <br /> Westfall, P. J., M. Momany, 2002 <span id='am-272' about='obo:NCBITaxon_162425' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-273' property="oboInOwl:hasAlternativeId" content="NCBITaxon:5072" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasRelatedSynonym" content="Emericella nidulans" datatype="xsd:string"></span><span id='am-275'  ></span><span id='am-276' property="oboInOwl:hasRelatedSynonym" content="Aspergillus nidulellus" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasRelatedSynonym" content="Aspergilus nidulans" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasRelatedSynonym" content="A. nidulans" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-281' property="rdfs:label" content="Aspergillus nidulans" datatype="xsd:string"></span>Aspergillus nidulans</span> septin            codons splicing: genomic analysis. RNA 9: 637&#8211;639. <br />    AspB plays pre- postmitotic roles septum, branch, and <br />    conidiophore development. Mol. Biol. Cell 13: 110&#8211;118.                                              Communicating editor: N. Perrimon <br />  <br />  <br />  <br />  <br /> 926     R. C. Eisman T. C. Kaufman <br />       GENETICS <br />                                                        Supporting Information <br />     http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.113.152546/-/DC1 <br />  <br />  <br />  <br />  <br /> Probing Boundaries Orthology: The <br />         Unanticipated Rapid Evolution of <br />                  Drosophila centrosomin <br />                                      Robert C. Eisman Thomas C. Kaufman <br />  <br />  <br />  <br />  <br />                                         Copyright &#169; 2013 Genetics Society America <br />                                                         DOI: 10.1534/genetics.113.152546 <br />                                                                                                                       &#160; <br /> &#160; <br /> Figure&#160;S1&#160;&#160;&#160;Three&#160;large&#160;multi&#65533;?exon&#160;MPOs&#160;in&#160;D&#160;melanogaster&#160;dumpy.&#160;The&#160;D.&#160;melanogaster&#160;dp&#160;gene&#160;(top)&#160;is&#160;shown&#160;with&#160;the&#160;MPOs&#160; <br /> (orange&#160;boxes)&#160;shown&#160;above&#160;the&#160;known&#160;coding&#160;exons&#160;(blue&#160;arrows).&#160;The&#160;PIGFEAST&#160;repeat&#160;region&#160;and&#160;the&#160;exon&#160;37&#160;to&#160;70&#160;region&#160; <br /> are&#160;indicated&#160;by&#160;bars.&#160;&#160;An&#160;expanded&#160;view&#160;of&#160;the&#160;exon&#160;37&#160;to&#160;70&#160;region&#160;(bottom),&#160;which&#160;contains&#160;34&#160;coding&#160;exons&#160;in&#160;13&#160;MPOs&#160;or&#160; <br /> reading&#160;frames.&#160;&#160;Incomplete&#160;sequences&#160;in&#160;the&#160;other&#160;species&#160;made&#160;a&#160;complete&#160;analysis&#160;impossible&#160;but&#160;a&#160;similar&#160;ratio&#160;of&#160;exons&#160;to&#160; <br /> MPOs&#160;was&#160;found&#160;in&#160;D.&#160;virilis,&#160;although&#160;the&#160;arrangement&#160;of&#160;corresponding&#160;exons&#160;is&#160;different&#160;between&#160;species.&#160;&#160; <br /> &#160;                                      &#160; <br />  <br />  <br />  <br />  <br /> 2&#160;SI                                            R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;&#160; <br /> &#160; <br />                                                                                                                                            &#160; <br /> &#160; <br />     Figure&#160;S2&#160;&#160;&#160;Annotation&#160;of&#160;Drosophila&#160;MUD&#65533;?PL.&#160;The&#160;manual&#160;annotation&#160;of&#160;the&#160;MUD&#65533;?PL&#160;splice&#160;variant&#160;from&#160;exon&#160;2&#160;to&#160;11&#160;is&#160;shown&#160; <br />     for&#160;eight&#160;species&#160;with&#160;the&#160;MPOs&#160;(orange&#160;boxes)&#160;above&#160;the&#160;coding&#160;exons&#160;(blue&#160;arrows)&#160;below.&#160;&#160;The&#160;first&#160;coding&#160;exon&#160;could&#160;not&#160;be&#160; <br />     assigned&#160;with&#160;high&#160;confidence&#160;in&#160;at&#160;least&#160;one&#160;species.&#160;&#160;The&#160;numbers&#160;on&#160;the&#160;right&#160;indicate&#160;the&#160;gene&#8217;s&#160;genomic&#160;extent&#160;(top)&#160;and&#160; <br />     the&#160;translated&#160;protein&#160;size&#160;(bottom)&#160;for&#160;each&#160;model,&#160;the&#160;species&#160;is&#160;on&#160;the&#160;left.&#160;&#160;The&#160;gene&#160;models&#160;are&#160;to&#160;scale&#160;and&#160;are&#160;aligned&#160;on&#160; <br />     the&#160;left&#160;at&#160;the&#160;3&#8242;&#160;terminal&#160;stop&#160;codon.&#160;&#160;The&#160;target&#160;MPO&#160;identified&#160;in&#160;our&#160;screen&#160;for&#160;long&#160;MPOs&#160;in&#160;D.&#160;melanogaster&#160;is&#160;shaded&#160;in&#160; <br />     gray.&#160;The&#160;asterisk&#160;downstream&#160;of&#160;the&#160;target&#160;MPO&#160;is&#160;a&#160;clear&#160;intron&#160;loss&#160;in&#160;D.&#160;melanogaster&#160;and&#160;D.&#160;erecta.&#160;Interestingly&#160;this&#160;exon&#160; <br />     in&#160;D.&#160;melanogaster&#160;encodes&#160;a&#160;conserved&#160;functional&#160;motif&#160;in&#160;the&#160;protein.&#160; <br />     Abbreviations:&#160;(Dmel:&#160;D.&#160;melanogaster;&#160;Dere:&#160;D.&#160;erecta;&#160;Dyak:&#160;D.&#160;yakuba;&#160;Dana:&#160;D.&#160;ananassae;&#160;Dpse:&#160;D.&#160;psuedoobscura;&#160;Dwil:&#160;D.&#160; <br />     willistoni;&#160;Dmoj:&#160;D.&#160;mojavensis;&#160;Dvir:&#160;D.&#160;virilis;&#160;Dgri:&#160;D.&#160;grimshawi)&#160; <br />  <br />     &#160;                                      &#160; <br />  <br />  <br />  <br />  <br />                                                      R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;                                              3&#160;SI&#160; <br />     &#160; <br />                                                                                                                                        &#160; <br />     &#160; <br />     Figure&#160;S3&#160;&#160;&#160;Annotation&#160;of&#160;Drosophila&#160;Rno&#65533;?PB.&#160;The&#160;manual&#160;annotation&#160;of&#160;the&#160;entire&#160;RNO&#65533;?PB&#160;splice&#160;variant&#160;is&#160;shown,&#160;labeled&#160;as&#160;in&#160; <br />     Supplemental&#160;Figure&#160;2.&#160;&#160;All&#160;the&#160;variation&#160;in&#160;the&#160;protein&#160;size&#160;between&#160;species&#160;is&#160;due&#160;to&#160;many&#160;indels&#160;within&#160;the&#160;large&#160;terminal&#160; <br />     coding&#160;exon&#160;identified&#160;in&#160;our&#160;screen.&#160; <br /> &#160; <br />     &#160;                                      &#160; <br />  <br />  <br />  <br />  <br />     4&#160;SI                                            R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;&#160; <br />     &#160; <br />                                                                                                                                         &#160; <br /> &#160; <br /> Figure&#160;S4&#160;&#160;&#160;Annotation&#160;of&#160;Drosophila&#160;Lva&#65533;?PC.&#160;The&#160;manual&#160;annotation&#160;of&#160;exons&#160;2&#160;to&#160;8&#160;of&#160;the&#160;LVA&#65533;?PC&#160;splice&#160;variant&#160;is&#160;shown,&#160; <br /> labeled&#160;as&#160;in&#160;Supplemental&#160;Figure&#160;2.&#160;&#160;The&#160;first&#160;coding&#160;exon&#160;could&#160;not&#160;be&#160;assigned&#160;with&#160;high&#160;confidence&#160;in&#160;at&#160;least&#160;one&#160;species.&#160;&#160;In&#160; <br /> D.&#160;psuedoobscura&#160;the&#160;two&#160;coding&#160;exons&#160;5&#8242;&#160;of&#160;the&#160;terminal&#160;exon&#160;are&#160;interrupted&#160;by&#160;a&#160;single&#160;stop&#160;codon&#160;created&#160;by&#160;a&#160;non&#65533;?triplet&#160; <br /> indel,&#160;which&#160;may&#160;be&#160;due&#160;to&#160;a&#160;sequencing&#160;error.&#160;&#160;All&#160;other&#160;splice&#160;sites&#160;were&#160;determined&#160;by&#160;their&#160;alignment&#160;with&#160;the&#160;known&#160;D.&#160; <br /> melanogaster&#160;cDNA&#160;and&#160;will&#160;require&#160;future&#160;RNA&#160;sequencing&#160;to&#160;determine&#160;the&#160;actual&#160;transcripts&#160;present&#160;in&#160;each&#160;species.&#160; <br /> &#160;                                       &#160; <br />  <br />  <br />  <br />  <br />                                                   R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;                                              5&#160;SI&#160; <br /> &#160; <br />                                                                                                                                       &#160; <br /> &#160; <br /> Figure&#160;S5&#160;&#160;&#160;Annotation&#160;of&#160;Drosophila&#160;Futsch&#65533;?PC.&#160;The&#160;manual&#160;annotation&#160;of&#160;exons&#160;3&#160;to&#160;8&#160;of&#160;the&#160;Futsch&#65533;?PC&#160;splice&#160;variant&#160;is&#160;shown,&#160; <br /> labeled&#160;as&#160;in&#160;Supplemental&#160;Figure&#160;2.&#160;&#160;The&#160;first&#160;two&#160;coding&#160;exons&#160;could&#160;not&#160;be&#160;assigned&#160;with&#160;high&#160;confidence&#160;in&#160;at&#160;least&#160;one&#160; <br /> species.&#160;As&#160;in&#160;RNO&#65533;?PB,&#160;the&#160;protein&#160;size&#160;variation&#160;is&#160;due&#160;to&#160;frequent&#160;indels&#160;in&#160;the&#160;exons&#160;corresponding&#160;to&#160;the&#160;single&#160;large&#160;coding&#160; <br /> exon&#160;associated&#160;with&#160;our&#160;target&#160;MPO&#160;in&#160;D.&#160;melanogaster.&#160;&#160;The&#160;single&#160;breaks&#160;in&#160;the&#160;large&#160;exons&#160;in&#160;D.&#160;mojavensis&#160;and&#160;D.&#160; <br /> grimshawi&#160;are&#160;due&#160;to&#160;a&#160;single&#160;base&#160;substitution&#160;that&#160;introduces&#160;an&#160;in&#65533;?frame&#160;stop&#160;codon,&#160;probably&#160;due&#160;to&#160;sequencing&#160;errors.&#160;The&#160; <br /> next&#160;to&#160;last&#160;exon&#160;in&#160;species&#160;below&#160;D.&#160;ananassae&#160;are&#160;present&#160;but&#160;we&#160;could&#160;not&#160;predict&#160;where&#160;the&#160;potential&#160;5&#8242;&#160;intron&#160;would&#160;be&#160;in&#160; <br /> actual&#160;transcripts.&#160; <br />  <br /> &#160;                                      &#160; <br />  <br />  <br />  <br />  <br /> 6&#160;SI                                             R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;&#160; <br /> &#160; <br />                                                            File&#160;S1&#160; <br />                                                               &#160; <br />                                                    Supplemental&#160;Text&#160; <br />                                                               &#160; <br /> Our&#160;screen&#160;for&#160;extensive&#160;MPOs&#160;as&#160;compared&#160;to&#160;coding&#160;exon&#160;length&#160;identified&#160;several&#160;genes&#160;in&#160;D.&#160;melanogaster.&#160;&#160;Although&#160;we&#160; <br /> limited&#160;our&#160;analyses&#160;to&#160;genes&#160;with&#160;MPOs&#160;&gt;3000&#160;nt&#160;longer&#160;than&#160;an&#160;associated&#160;coding&#160;exon,&#160;it&#160;is&#160;likely&#160;the&#160;evolution&#160;of&#160;genes&#160;with&#160; <br /> less&#160;extensive&#160;MPOs&#160;will&#160;be&#160;interesting&#160;as&#160;well.&#160;&#160;For&#160;example,&#160;the&#160;largest&#160;MPO&#160;in&#160;cnn&#160;is&#160;only&#160;1000&#160;nt&#160;longer&#160;than&#160;one&#160;of&#160;the&#160; <br /> associated&#160;coding&#160;exons.&#160;&#160;Additionally,&#160;genes&#160;with&#160;significant&#160;variation&#160;in&#160;their&#160;MPO&#160;maps&#160;across&#160;some&#160;phylogenetic&#160;range&#160; <br /> should&#160;be&#160;good&#160;candidates&#160;for&#160;genomic&#160;and&#160;gene/protein&#160;evolutionary&#160;studies.&#160;&#160;Below&#160;is&#160;a&#160;very&#160;brief&#160;description&#160;of&#160;the&#160;seven&#160; <br /> genes&#160;we&#160;have&#160;manually&#160;annotated&#160;in&#160;nine&#160;Drosophila&#160;species.&#160; <br /> &#160;         The&#160;dumpy&#160;(dp)&#160;gene&#160;makes&#160;maximum&#160;use&#160;of&#160;multi&#65533;?exon&#160;MPOs&#160;and&#160;contained&#160;the&#160;longest&#160;excess&#160;beyond&#160;an&#160; <br /> annotated&#160;exon&#160;end&#160;(14,000&#160;nt).&#160;&#160;The&#160;dp&#160;locus&#160;spans&#160;over&#160;100&#160;kb,&#160;encoding&#160;a&#160;2.5&#160;MDa&#160;modular&#160;extracellular&#160;matrix&#160;protein&#160;with&#160; <br /> 308&#160;EGF&#160;modules&#160;and&#160;185&#160;DPY&#160;modules&#160;required&#160;for&#160;the&#160;anchoring&#160;of&#160;epidermal&#160;tissue&#160;to&#160;the&#160;fly&#160;cuticle&#160;(WILKIN&#160;et&#160;al.&#160;2000).&#160;The&#160; <br /> 79&#160;dp&#160;coding&#160;exons&#160;are&#160;in&#160;53&#160;MPOs,&#160;and&#160;the&#160;region&#160;from&#160;exon&#160;37&#160;to&#160;70&#160;has&#160;34&#160;coding&#160;exons&#160;in&#160;13&#160;MPOs&#160;(Supplemental&#160;Figure&#160; <br /> 1).&#160;Due&#160;to&#160;extensive&#160;repeats&#160;within&#160;the&#160;gene&#160;the&#160;exact&#160;changes&#160;in&#160;coding&#160;exon&#160;to&#160;MPO&#160;organization&#160;within&#160;the&#160;genus&#160;Drosophila&#160; <br /> are&#160;difficult&#160;to&#160;determine&#160;precisely,&#160;all&#160;species&#160;have&#160;a&#160;similar&#160;multi&#65533;?exon&#160;MPO&#160;arrangement.&#160;A&#160;recent&#160;study&#160;of&#160;dp&#160;found&#160;the&#160;area&#160; <br /> containing&#160;the&#160;greatest&#160;concentration&#160;of&#160;multi&#65533;?exon&#160;MPOs&#160;is&#160;alternatively&#160;spliced,&#160;although&#160;the&#160;actual&#160;splicing&#160;is&#160;unknown&#160; <br /> (CARMON&#160;et&#160;al.&#160;2010).&#160;Because&#160;many&#160;dp&#160;genes&#160;in&#160;the&#160;genus&#160;Drosophila&#160;appear&#160;to&#160;have&#160;assembly&#160;errors&#160;creating&#160;large&#160;deletions,&#160; <br /> possibly&#160;due&#160;to&#160;the&#160;highly&#160;repetitive&#160;nature&#160;of&#160;dp&#160;protein&#160;modules,&#160;we&#160;were&#160;not&#160;able&#160;to&#160;do&#160;a&#160;complete&#160;analysis&#160;of&#160;the&#160; <br /> divergence&#160;of&#160;target&#160;MPOs.&#160; <br /> &#160;         The&#160;target&#160;MPOs&#160;in&#160;four&#160;of&#160;the&#160;genes&#160;identified,&#160;mushroom&#160;body&#160;defective&#160;(mud)&#160;(Supplemental&#160;Figure&#160;2),&#160;rhinocerous&#160; <br /> (rno)&#160;(Supplemental&#160;Figure&#160;3),&#160;lava&#160;lamp&#160;(lva)&#160;(Supplemental&#160;Figure&#160;4),&#160;and&#160;Futsch&#160;(Supplemental&#160;Figure&#160;5),&#160;span&#160;a&#160;small&#160;and&#160; <br /> large&#160;coding&#160;exon&#160;in&#160;D.&#160;melanogaster.&#160;Interestingly,&#160;the&#160;ab&#160;initio&#160;models&#160;for&#160;these&#160;four&#160;genes&#160;are&#160;variable&#160;in&#160;the&#160;genus&#160; <br /> Drosophila,&#160;including&#160;a&#160;split&#160;of&#160;the&#160;gene&#160;into&#160;two&#160;separate&#160;transcription&#160;units&#160;and&#160;the&#160;elimination&#160;of&#160;relatively&#160;large&#160;regions&#160;of&#160; <br /> coding&#160;sequence.&#160;Most&#160;of&#160;the&#160;variability&#160;in&#160;these&#160;models&#160;is&#160;associated&#160;with&#160;the&#160;region&#160;covered&#160;by&#160;our&#160;target&#160;MPOs,&#160;suggesting&#160; <br /> these&#160;coding&#160;regions&#160;confound&#160;modeling&#160;algorithms,&#160;possibly&#160;due&#160;to&#160;a&#160;lack&#160;of&#160;strong&#160;splicing&#160;signals.&#160;However,&#160;the&#160;manual&#160; <br /> annotation&#160;of&#160;these&#160;genes&#160;finds&#160;little&#160;evidence&#160;to&#160;support&#160;significant&#160;changes&#160;relative&#160;to&#160;known&#160;transcripts&#160;from&#160;D.&#160; <br /> melanogaster.&#160; <br /> &#160;&#160;        The&#160;rno&#160;locus&#160;encodes&#160;a&#160;transcription&#160;factor&#160;predicted&#160;to&#160;be&#160;a&#160;chromatin&#65533;?remodeling&#160;protein&#160;required&#160;to&#160;restrict&#160;the&#160; <br /> Ras&#160;pathway&#160;during&#160;eye&#160;development&#160;in&#160;D.&#160;melanogaster&#160;(Voas&#160;&amp;&#160;Rebay,&#160;2003).&#160;&#160;The&#160;target&#160;MPO&#160;encodes&#160;a&#160;long&#160;(3929&#160;aa)&#160; <br /> coiled&#65533;?coil&#160;peptide&#160;and&#160;is&#160;present&#160;in&#160;all&#160;Drosophila&#160;species&#160;used&#160;in&#160;this&#160;study,&#160;as&#160;are&#160;the&#160;five&#160;amino&#160;terminal&#160;coding&#160;exons.&#160;Based&#160; <br /> on&#160;these&#160;results&#160;there&#160;is&#160;no&#160;clear&#160;explanation&#160;for&#160;the&#160;significant&#160;variations&#160;in&#160;gene&#160;models.&#160; <br /> &#160;         The&#160;lva&#160;gene&#160;encodes&#160;a&#160;rapidly&#160;evolving&#160;coiled&#65533;?coil&#160;golgin&#160;protein&#160;required&#160;for&#160;Golgi&#160;vesicle&#160;transport&#160;and&#160; <br /> cellularization&#160;during&#160;embryogenesis&#160;in&#160;D.&#160;melanogaster&#160;(SISSON&#160;et&#160;al.&#160;2000).&#160;The&#160;lva&#160;target&#160;MPO&#160;spans&#160;exon&#160;4&#160;and&#160;exon&#160;5&#160;in&#160;the&#160; <br /> Lva&#65533;?PC&#160;isoform&#160;but&#160;the&#160;intron&#160;between&#160;these&#160;exons&#160;is&#160;retained&#160;in&#160;an&#160;alternatively&#160;spliced&#160;variant&#160;(Sisson&#160;et&#160;al,&#160;2000).&#160;The&#160;only&#160; <br /> evidence&#160;we&#160;see&#160;for&#160;model&#160;changes&#160;in&#160;the&#160;genus&#160;involve&#160;fusions&#160;with&#160;the&#160;downstream&#160;adjacent&#160;exon&#160;and&#160;the&#160;possible&#160;loss&#160;of&#160;an&#160; <br /> intron.&#160;&#160;However,&#160;based&#160;on&#160;the&#160;exon6/7&#160;splicing&#160;in&#160;cnn&#160;(described&#160;in&#160;the&#160;main&#160;text)&#160;accurate&#160;models&#160;will&#160;require&#160;sequencing&#160; <br /> transcripts&#160;or&#160;an&#160;analysis&#160;of&#160;RNA&#65533;?Seq&#160;data.&#160;&#160;Nevertheless&#160;even&#160;in&#160;the&#160;absence&#160;of&#160;cDNA&#160;or&#160;RNA&#65533;?Seq&#160;data&#160;all&#160;Drosophila&#160;lva&#160;gene&#160; <br /> models&#160;appear&#160;to&#160;be&#160;similar&#160;to&#160;D.&#160;melanogaster.&#160; <br />  <br />  <br />  <br />  <br />                                                    R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;                                               7&#160;SI&#160; <br /> &#160; <br />  &#160;         The&#160;futsch&#160;locus&#160;encodes&#160;a&#160;cytoskeletal&#160;protein&#160;required&#160;for&#160;the&#160;development&#160;of&#160;axons&#160;and&#160;dendrites&#160;in&#160;D.&#160; <br /> melanogaster&#160;(HUMMEL&#160;et&#160;al.&#160;2000),&#160;and&#160;organizes&#160;the&#160;microtubule&#160;cytoskeleton&#160;during&#160;synaptic&#160;growth&#160;(ROOS&#160;et&#160;al.&#160;2000).&#160;The&#160; <br /> Futsch&#160;protein&#160;appears&#160;to&#160;be&#160;a&#160;chimera&#160;of&#160;vertebrate&#160;genes,&#160;comprised&#160;of&#160;vertebrate&#160;MAP1B&#65533;?like&#160;amino&#65533;?&#160;and&#160;carboxy&#65533;?termini&#160; <br /> separated&#160;by&#160;a&#160;repetitive&#160;central&#160;region&#160;similar&#160;to&#160;vertebrate&#160;neurofilament&#160;proteins&#160;(HUMMEL&#160;et&#160;al.&#160;2000).&#160;The&#160;central&#160;region&#160; <br /> contains&#160;60&#160;repeats&#160;of&#160;approximately&#160;37&#160;amino&#160;acids&#160;each&#160;that&#160;may&#160;be&#160;phosphorylation&#160;targets&#160;(HUMMEL&#160;et&#160;al.&#160;2000)&#160;and&#160;are&#160; <br /> encoded&#160;by&#160;Futsch&#65533;?PB&#160;exon&#160;6&#160;which&#160;is&#160;in&#160;our&#160;target&#160;MPO.&#160;Exon&#160;6&#160;is&#160;a&#160;continuous&#160;exon&#160;in&#160;all&#160;species&#160;except&#160;D.&#160;mojavensis&#160;and&#160;D.&#160; <br /> grimshawi,&#160;which&#160;both&#160;have&#160;single&#160;base&#160;changes&#160;creating&#160;a&#160;stop&#160;codon,&#160;and&#160;D.&#160;yakuba&#160;appears&#160;to&#160;have&#160;a&#160;small&#160;insertion&#160;and&#160; <br /> stop&#160;codon.&#160;While&#160;these&#160;changes&#160;may&#160;be&#160;real,&#160;since&#160;they&#160;occur&#160;in&#160;the&#160;highly&#160;repetitive&#160;sequence&#160;it&#160;is&#160;also&#160;possible&#160;they&#160;are&#160; <br /> sequencing&#160;or&#160;assembly&#160;errors.&#160;Surprisingly,&#160;even&#160;though&#160;this&#160;exon&#160;is&#160;similar&#160;in&#160;size&#160;to&#160;D.&#160;melanogaster&#160;in&#160;the&#160;other&#160;species,&#160;the&#160; <br /> gene&#160;models&#160;for&#160;several&#160;species&#160;remove&#160;approximately&#160;half&#160;of&#160;the&#160;exon&#160;from&#160;the&#160;5&#8242;&#160;end.&#160;&#160; <br /> &#160;         Of&#160;the&#160;four&#160;genes&#160;analyzed&#160;with&#160;multi&#65533;?exon&#160;MPOs,&#160;mud&#160;has&#160;the&#160;most&#160;complex&#160;and&#160;potentially&#160;exhibits&#160;real&#160;changes&#160;in&#160; <br /> splicing.&#160;The&#160;mud&#160;gene&#160;encodes&#160;a&#160;microtuble&#160;binding&#160;protein&#160;required&#160;at&#160;the&#160;meiosis&#160;II&#160;spindle&#160;in&#160;Drosophila&#160;embryos&#160;(YU&#160;et&#160;al.&#160; <br /> 2006),&#160;and&#160;is&#160;required&#160;for&#160;the&#160;proliferation&#160;(GUAN&#160;et&#160;al.&#160;2000)&#160;and&#160;asymmetric&#160;division&#160;of&#160;neuroblast&#160;cells&#160;(IZUMI&#160;et&#160;al.&#160;2006;&#160; <br /> SILLER&#160;et&#160;al.&#160;2006).&#160;Mud&#160;protein&#160;is&#160;predicted&#160;to&#160;be&#160;functional&#160;ortholog&#160;of&#160;mammalian&#160;NuMA&#160;based&#160;on&#160;conserved&#160;domains&#160;in&#160;the&#160; <br /> carboxy&#65533;?terminus&#160;of&#160;the&#160;D.&#160;melanogaster&#160;protein&#160;(SILLER&#160;et&#160;al.&#160;2006).&#160;&#160;Our&#160;target&#160;MPO&#160;spans&#160;the&#160;Mud&#65533;?PB&#160;large&#160;exons&#160;6&#160;and&#160;7,&#160; <br /> encoding&#160;a&#160;coiled&#65533;?coil&#160;domain&#160;(GUAN&#160;et&#160;al.&#160;2000).&#160;Based&#160;on&#160;protein&#160;alignments&#160;it&#160;appears&#160;the&#160;coding&#160;portions&#160;of&#160;the&#160;exons&#160;are&#160; <br /> fused&#160;in&#160;D.&#160;yakuba,&#160;D.&#160;erecta,&#160;D.&#160;pseudoobscura&#160;and&#160;D.&#160;grimshawi&#160;as&#160;removal&#160;of&#160;an&#160;intron&#160;would&#160;result&#160;in&#160;a&#160;significantly&#160;smaller&#160; <br /> protein&#160;compared&#160;to&#160;other&#160;species&#160;in&#160;this&#160;analysis.&#160;Additionally,&#160;these&#160;proteins&#160;are&#160;several&#160;hundred&#160;amino&#160;acids&#160;shorter&#160;than&#160; <br /> species&#160;with&#160;the&#160;intron.&#160;In&#160;D.&#160;mojavensis&#160;there&#160;are&#160;two&#160;exons&#160;but&#160;the&#160;3&#8242;&#160;end&#160;of&#160;exon&#160;6&#160;is&#160;fused&#160;to&#160;exon&#160;7,&#160;producing&#160;a&#160;much&#160; <br /> smaller&#160;peptide.&#160;Finally,&#160;in&#160;D.&#160;virilis&#160;there&#160;is&#160;a&#160;short&#160;fused&#160;single&#160;coding&#160;exon&#160;and&#160;an&#160;upstream&#160;insertion&#160;of&#160;approximately&#160;5&#160;kb,&#160; <br /> bearing&#160;no&#160;sequence&#160;similarity&#160;to&#160;any&#160;of&#160;the&#160;mud&#160;genes&#160;in&#160;this&#160;study.&#160;Our&#160;mud&#160;target&#160;MPO&#160;identified&#160;the&#160;region&#160;of&#160;Mud&#160;protein&#160; <br /> that&#160;led&#160;others&#160;to&#160;identify&#160;D.&#160;melanogaster&#160;as&#160;a&#160;functional&#160;ortholog&#160;of&#160;NuMA&#160;rather&#160;than&#160;an&#160;orthologous&#160;gene.&#160;In&#160;their&#160;study&#160; <br /> they&#160;showed&#160;conserved&#160;domains&#160;in&#160;Mud&#160;retained&#160;NuMA&#160;function&#160;but&#160;the&#160;coiled&#65533;?coil&#160;region&#160;had&#160;no&#160;similarity&#160;to&#160;vertebrate&#160; <br /> NuMA&#160;(GUAN&#160;et&#160;al.&#160;2000),&#160;which&#160;are&#160;80%&#160;identical&#160;between&#160;human&#160;and&#160;mouse&#160;(WHITE&#160;and&#160;ERICKSON&#160;2006).&#160;While&#160;our&#160;results&#160; <br /> neither&#160;strengthen&#160;nor&#160;weaken&#160;orthology&#160;arguments&#160;they&#160;do&#160;suggest&#160;the&#160;evolution&#160;of&#160;coiled&#65533;?coil&#160;domains&#160;may&#160;be&#160;more&#160;plastic&#160; <br /> in&#160;Drosophila&#160;compared&#160;to&#160;vertebrates.&#160; <br />           The&#160;last&#160;two&#160;genes&#160;in&#160;our&#160;screen&#160;were&#160;identified&#160;because&#160;the&#160;target&#160;MPOs&#160;spanned&#160;exons&#160;known&#160;to&#160;be&#160;alternatively&#160; <br /> spliced&#160;by&#160;an&#160;intron&#160;retention/exclusion&#160;mechanism.&#160;The&#160;prospero&#160;(pros)&#160;gene&#160;encodes&#160;a&#160;transcription&#160;factor&#160;required&#160;for&#160;cell&#160; <br /> fate&#160;specification&#160;during&#160;central&#160;nervous&#160;system&#160;development&#160;in&#160;D.&#160;melanogaster&#160;(CHU&#65533;?LAGRAFF&#160;et&#160;al.&#160;1991).&#160;Orthologous&#160;genes&#160; <br /> of&#160;the&#160;Pros/Prox1&#160;family&#160;all&#160;contain&#160;a&#160;conserved&#160;COOH&#160;terminus&#160;which&#160;contains&#160;the&#160;structurally&#65533;?unique&#160;Homeo&#65533;?Prospero&#160; <br /> domain&#160;(RYTER&#160;et&#160;al.&#160;2002),&#160;and&#160;motifs&#160;required&#160;for&#160;the&#160;complex&#160;regulation&#160;of&#160;the&#160;protein&#160;(BI&#160;et&#160;al.&#160;2003).&#160;&#160;The&#160;amino&#160;half&#160;of&#160;D.&#160; <br /> melanogaster&#160;Pros&#160;protein&#160;contains&#160;two&#160;small&#160;conserved&#160;motifs&#160;present&#160;in&#160;chicken&#160;Prox1,&#160;but&#160;shares&#160;no&#160;other&#160;similarities&#160; <br /> upstream&#160;of&#160;the&#160;homeodomain&#160;(TOMAREV&#160;et&#160;al.&#160;1996).&#160;The&#160;pros&#160;target&#160;MPO&#160;spans&#160;the&#160;exon&#160;encoding&#160;the&#160;amino&#160;half&#160;of&#160;Pros&#160;and&#160; <br /> accounts&#160;for&#160;the&#160;majority&#160;of&#160;the&#160;changes&#160;between&#160;Drosophila&#160;species.&#160;Protein&#160;alignments&#160;between&#160;Drosophila&#160;species&#160;show&#160;the&#160; <br /> encoded&#160;peptide&#160;is&#160;a&#160;mix&#160;of&#160;high&#160;conservation&#160;interspersed&#160;with&#160;regions&#160;of&#160;nearly&#160;100%&#160;divergence&#160;and&#160;gaps&#160;accumulate&#160;with&#160; <br /> phylogenetic&#160;distance,&#160;similar&#160;to&#160;all&#160;MPOs&#160;identified&#160;in&#160;the&#160;screen.&#160; <br /> &#160;         The&#160;last&#160;gene&#160;in&#160;this&#160;study&#160;is&#160;shortstop&#160;(shot),&#160;a&#160;transcriptionally&#160;complex&#160;gene,&#160;which&#160;is&#160;a&#160;member&#160;of&#160;the&#160;Plakin&#160; <br /> protein&#160;family.&#160;Shot&#160;proteins&#160;link&#160;the&#160;actin&#160;and&#160;microtubule&#160;cytoskeleton&#160;to&#160;a&#160;variety&#160;of&#160;junctional&#160;complexes&#160;in&#160;different&#160;cell&#160; <br /> types&#160;(SONNENBERG&#160;and&#160;LIEM&#160;2007)&#160;required&#160;for&#160;cytoskeleton&#160;organization&#160;during&#160;neurogenesis&#160;and&#160;in&#160;muscles&#160;and&#160;tendon&#160;cells&#160; <br />  <br />  <br /> 8&#160;SI                                              R.&#160;C.&#160;Eisman&#160;and&#160;T.&#160;C.&#160;Kaufman&#160;&#160; <br /> &#160; <br />  (SUBRAMANIAN&#160;et&#160;al.&#160;2003).&#160;The&#160;central&#160;region&#160;of&#160;Shot&#160;long&#160;isoforms&#160;form&#160;a&#160;long&#160;coiled&#160;coil&#160;(LEE&#160;et&#160;al.&#160;2000),&#160;a&#160;region&#160;partially&#160; <br /> covered&#160;by&#160;the&#160;shot&#160;MPO.&#160;Alternative&#160;3&#8242;&#160;splicing&#160;of&#160;our&#160;target&#160;MPO&#160;in&#160;shot&#160;produces&#160;eighteen&#160;mRNAs&#160;that&#160;include&#160;the&#160;5&#8242;&#160; <br /> terminus&#160;of&#160;the&#160;exon&#160;encoding&#160;a&#160;conserved&#160;peptide&#160;(154&#160;aa),&#160;whereas&#160;the&#160;Shot&#65533;?RH&#160;mRNA&#160;includes&#160;the&#160;entire&#160;exon&#160;encoding&#160;a&#160; <br /> large&#160;(3499&#160;aa&#160;in&#160;D.&#160;melanogaster)&#160;divergent&#160;peptide.&#160;Three&#160;Shot&#160;splice&#160;variants&#160;skip&#160;this&#160;exon&#160;entirely.&#160;Interestingly,&#160;27&#160;exons&#160; <br /> common&#160;to&#160;all&#160;is <br /> </body></html>