<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                                   JCB: Report <br />  <br />  <br />  <br />  <br />                               Ubiquitylation Ltn1 E3 ligase protects 60S <br />                               ribosomes starvation-induced selective autophagy <br />                               Batool Ossareh-Nazari,1 Carlos A. Ni&#241;o,1 Mario H. Bengtson,2 Joong-Won Lee,2 Claudio A.P. Joazeiro,2 <br />                               Catherine Dargemont1 <br />                               1 <br />                               Institut Jacques Monod, University <span id='am-54' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Paris</span> Diderot, Sorbonne Paris Cit&#233;, Centre National la Recherche Scientifique Unit&#233; Mixte Recherche 7592, 75205 Paris, <br />                               Cedex 13, France <br />                               2 <br />                               Department Cell Molecular Biology, The Scripps Research Institute, La Jolla, CA 92037 <br />  <br />  <br />  <br />  <br />                               A <br />                                        utophagy, process proteins or-                              E3 implicated translational surveillance, acts an <br />                                        ganelles engulfed autophagosomes                                 inhibitor 60S ribosomal subunit ribophagy is <br />                                        delivered vacuolar/lysosomal degradation,                               antagonized Ubp3. The ribosomal protein Rpl25 a <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               induced ensure survival starvation                             relevant target. Its ubiquitylation Ltn1 dependent and <br />                               stresses. A selective autophagic pathway 60S ribo-                               Ubp3 reversed, mutation ubiquitylation site <br />                               somal subunits elicited nitrogen starvation yeast&#8212;                             rendered ribophagy dependent Ubp3. Consis- <br />                               ribophagy&#8212;was recently described requires                                    tently, expression Ltn1&#8212;but Ubp3&#8212;rapidly <br />                               Ubp3-Bre5 deubiquitylating enzyme. <span id='am-39' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43'  ></span>This</span> discovery im-                                decreased starvation, presumably allow riboph- <br />                               plied E3 ligases act upstream, inhibiting                            agy proceed. Thus, Ltn1 Ubp3-Bre5 likely contrib- <br />                               process providing initial required signal. In                         <span id='am-76' about='obo:NCBITaxon_365374' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_86011 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_45124 obo:NCBITaxon_131567 obo:NCBITaxon_73655 obo:NCBITaxon_27929 obo:NCBITaxon_2759'><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Ute" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ute</span> adapt ribophagy activity nutrient supply <br />                               paper, Ltn1/Rkr1, 60S ribosome-associated                             protein translation. <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               Rapid degradation mature ribosomes occur nutri-                         Ubp3-Bre5, role ubiquitylation/deubiquitylation the <br />                               ent starvation conditions selective autophagic pathway,                        process, E3 ligases acting upstream Ubp3-Bre5 have <br />                               called ribophagy, presumably contributing maintenance                         remained elucidated. <br />                               intracellular amino acid pool (Kraft et al., 2008; Kristensen                       Two simple models potential role of <br />                               et al., 2008). Ribophagy promotes relocalization mature ribo-                     ubiquitin ribophagy. Ubiquitylation prevent ribo- <br />                               somes cytoplasm vacuole, resulting deg-                     somes degraded required ribophagy pro- <br />                               radation. Importantly, 60S 40S ribosomal subunits appear                      viding initial degradation signal, deubiquitylation being <br />                               independently targeted degradation; fol-                  required complete process. Insights mechanisms <br />                               lows pathway remains elucidated, selective degradation                  greatly benefit identification E3 ligases in- <br />                               60S ribosomal subunits specifically requires Ubp3-Bre5                        volved. In recent work, Rsp5 E3 ligase proposed to <br />                               deubiquitinating enzyme complex (Kraft et al., 2008; Ossareh-                        cooperate Ubp3-Bre5 ribophagy providing &#8220;engulf <br />                               Nazari et al., 2010; Dargemont Ossareh-Nazari, 2012).                            me&#65533;? signal second model (Kraft Peter, 2008). How- <br />                               Ubp3 Bre5 form heterotetrameric complex able cleave                         ever, Rsp5 mutation displays synthetic ribophagy defect with <br />                               ubiquitin diverse specific substrates (Cohen et al., 2003;                  loss Ubp3, suggesting E3 likely acts distinct <br />                               Li et al., 2005, 2007). In addition, 60S ribophagy requires                      molecular step upstream Ubp3-Bre5. <br />                               AAA ATPase Cdc48/p97 cofactor, ubiquitin binding                               Among E3 ligases characterized, Ltn1/ <br />                               protein Ufd3, play key role deciphering                          Rkr1 emerged likely 60S ribophagy regulator candidate <br />                               fates ubiquitylated proteins (Ossareh-Nazari et al., 2010;                        virtue association large ribosomal subunit <br />                               Dargemont Ossareh-Nazari, 2012). The relevant substrates <br />                                                                                                                    &#169; 2014 Ossareh-Nazari et al.&#8195; This article distributed terms Attribution&#8211; <br />                                                                                                                    Noncommercial&#8211;Share Alike&#8211;No Mirror Sites license months pub- <br />                                                                                                                    lication date (see http://www.rupress.org/terms). After months available a <br />                                                                                                                    Creative Commons License (Attribution&#8211;Noncommercial&#8211;Share Alike 3.0 Unported license, <br />                               Correspondence Catherine Dargemont: dargemont@gmail.com                           described http://creativecommons.org/licenses/by-nc-sa/3.0/). <br />  <br />  <br />  <br />  <br />                               The Rockefeller University Press&#8195;&#8202;$30.00 <br />                               J. Cell Biol. Vol. 204 No. 6&#8195; 909&#8211;917 <br />                               www.jcb.org/cgi/doi/10.1083/jcb.201308139                                                                                                                                    JCB    909 <br />            (Bengtson Joazeiro, 2010). Ltn1 previously shown         cytoplasmic + vacuole). Ltn1 belongs RING (really <br />           function context mRNA surveillance ribosome-         interesting new gene) domain <span id='am-31' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> E3 ligases, inter- <br />           associated quality control (Bengtson Joazeiro, 2010; Brandman     action domain ubiquitin-charged E2-conjugating <br />           et al., 2012; Defenouill&#232;re et al., 2013; Shao et al., 2013). Cyto-   enzymes underlies E3 activity (Deshaies Joazeiro, 2009). <br />           plasmic mRNA quality control mechanisms depend transla-            The ribophagy process examined strain whose <br />           tion and, addition mediating decay aberrant messages,        chromosomal copy LTN1 harbored deletion C-terminal <br />           associated degradation encoded nascent protein        RING domain. Deletion RING domain able rescue <br />           recycling stalled ribosomes. Ltn1 shown       ribophagy defect UBP3-null cells (Fig. S1 B). <br />           function specifically targeting products mRNA devoid               These results Ubp3-Bre5 deubiquitylation ac- <br />           stop codons degradation (Bengtson Joazeiro, 2010).         tivity critical ribophagy absence Ltn1 its <br />           Consistent Ltn1&#8217;s role protein quality control, mutation      catalytic activity, raising possibility antagonistic ac- <br />           mouse homologue, Listerin, causes neurodegeneration         tion enzymes process suggesting that <br />           mice (Chu et al., 2009).                                              Ltn1 ubiquitylate target Ubp3-Bre5&#8211;dependent <br />                  Here, Ltn1 protects ribosomes degrada-       deubiquitylation required ribophagy occur. Two con- <br />           tion Ubp3-antagonistic manner. We identify ribosomal        served proteins, Tae2 Rqc1, recently reported to <br />           protein Rpl25 substrate Ltn1 Ubp3,     form ribosome-bound quality control complex with <br />           Rpl25 lysine residue involved ubiquitin-mediated inhibi-       Ltn1 (Brandman et al., 2012; Defenouill&#232;re et al., 2013). How- <br />           tion 60S ribophagy. Finally, consistent critical role   ever, deletion TAE2 did impact ribophagy process in <br />           Ltn1 inhibiting ribophagy, expression E3      wild-type ubp3&#65533;?&#65533; cells, indicating Tae2 does repre- <br />           severely reduced nutrient starvation.                         sent constitutive Ltn1 cofactor (Fig. S1 C). <br />  <br />                                                                                 The 60S ribosomal protein Rpl25 a <br />           Results discussion <br />                                                                                 substrate Ltn1 Ubp3 <br />           LTN1 deletion rescues ribophagy                                   To investigate Ltn1 Ubp3 regulate ribophagy, con- <br />           defect Ubp3-deficient yeast                                        sidered possibility mediated ubiquity- <br />           The Ltn1 E3, previously implicated cotranslational       lation 60S ribosomal proteins. It previously shown <br />           quality control, predominantly associated ribosomes,          ubiquitylation ribosomal subunits and/or ribosome- <br />           60S possibly 80S particles (Bengtson         associated proteins specifically enriched Ubp3-deficient <br />           Joazeiro, 2010; Brandman et al., 2012; Defenouill&#232;re et al., 2013;    cells nitrogen starvation 3 h, suggesting direct mech- <br />           Shao et al., 2013). Because ribophagy large ribosomal sub-     anism (Kraft et al., 2008). We set examine the <br />           unit involves ubiquitylation&#8211;deubiquitylation, reasonable      ubiquitylation ribosomal proteins directly. In light of <br />           test Ltn1 implicated process.             Ltn1&#8217;s known function protein quality control, reason- <br />                   To obtain evidence role Ltn1 ribophagy, ribo-     able hypothesize RING domain prop- <br />           degradation monitored vivo using GFP-tagged               erly positioned target nascent polypeptides binding the <br />           Rpl5 Rpl25 ribosomal proteins markers, previously            vicinity opening ribosomal exit tunnel. Therefore, <br />           described (Kraft et al., 2008). These tagged proteins compe-      Ltn1 mediates ubiquitylation ribosomal proteins, that <br />           tent efficiently incorporated ribosomes there-     surface exposed surroundings tunnel opening <br />           fore located cytoplasm wild-type          represent good candidates initial analysis, namely <br />           mutant cells starvation (Fig. 1 B depicted).       Rpl17, Rpl19, Rpl25, Rpl26, Rpl31, Rpl35 (Jenner et al., <br />           Nitrogen starvation induces relocalization GFP sig-     2012; Klinge et al., 2012). <br />           nal cytoplasm vacuole, leading degradation             We decided begin analysis Rpl25 this <br />           Rpl moiety vacuolar accumulation free GFP,           protein encoded single gene, amenable tagging, and <br />           relatively resistant vacuolar proteases (Kraft et al.,     known modified ubiquitin; appeared to <br />           2008). Vacuolar targeting cleavage GFP report-        place single lysine residue based mass spectrom- <br />           ers easily followed direct GFP fluorescence       etry study, identification additional ubiquitylation <br />           Western blotting total cell extracts. In agreement     sites Rpl25 hampered detection limit <br />           previous study, deletion UBP3 led delayed GFP cleavage        analysis (Starita et al., 2012). A cupper-inducible His-tagged <br />           vacuolar accumulation starvation, indicating       ubiquitin coexpressed GFP-tagged Rpl25 followed by <br />           enzyme required optimal rates ribophagy (Fig. 1; Kraft      purification ubiquitylated proteins nickel column and <br />           et al., 2008). Absence Ltn1 E3 ligase did signifi-   anti-GFP blotting. This assay showed Rpl25-GFP mono&#173; <br />           cantly affect ribophagy kinetics compared wild-       ubiquitylated, GFP (Fig. 2 A). Impor- <br />           type cells (Fig. 1) general autophagy, measured GFP-Atg8     tantly, deletion LTN1 RING domain severely impaired <br />           degradation (Fig. S1 A). Strikingly, combined, deletion       ubiquitylation Rpl25-GFP consistent requirement <br />           LTN1 UBP3 led rescue cleavage defect            Ltn1 E3 enzymatic activity modification (Fig. 2 B). <br />           Rpl5- Rpl25-GFP reporters compared ubp3&#65533;?&#65533; cells               In agreement results, analysis Rpl25-GFP modifi- <br />           (Fig. 1, A C), consistent increased accumulation       cation endogenous untagged ubiquitin total cell extracts <br />           GFP fluorescence vacuole (Fig. 1, B D, vacuole         confirmed Ltn1 RING domain dependency (Fig. 2 C, t = 0). <br />  <br />  <br />     910   JCB &#8226; VOLUME 204 &#8226; NUMBER 6 &#8226; 2014 <br />  Figure 1.&#8195; LTN1 deletion rescues ribophagy defect UBP3-null cells. (A&#8211;D) Cells expressing Rpl5-GFP (A B) Rpl25-GFP (C D) grown <br /> in rich medium (before starvation) starved SD-N indicated periods (starvation). (A C, left) Degradation GFP-tagged proteins was <br /> analyzed anti-GFP blot cell extracts. (right) The ratio cleaved GFP full-length protein quantified time point four <br /> independent experiments. a.u., arbitrary unit. (B D) Cells starvation starved 24 h (starvation) examined fluorescence mi- <br /> croscopy differential interferential contrast (DIC). Note 100% wild-type (WT) mutant cells expressed GFP-tagged proteins cytoplasm <br /> before starvation. C, cytoplasmic localization; V, vacuolar accumulation; C + V, localization cytoplasm vacuole. Bars, 5 &#181;m. The significance <br /> of differences observed ribophagy efficiency evaluated using Student&#8217;s t test. *, P = 0.01&#8211;0.05; **, P = 0.001&#8211;0.01. The errors bars cor- <br /> respond standard deviations. <br />  <br />  <br />                                                                    Ubiquitylation ribosome inhibits ribophagy &#8226; Ossareh-Nazari et al.                  911 <br />            Figure 2.&#8195; The 60S ribosomal protein Rpl25 substrate Ltn1 Ubp3. (A) GFP Rpl25-GFP expressed cells transformed (+) not <br />           transformed (&#57394;) plasmid encoding 6His-ubiquitin. (top) Expression levels GFP Rpl25-GFP cell lysates verified anti-GFP <br />           blotting. (middle) Purified 6His-ubiquitin conjugates examined anti-GFP blotting. (bottom) 6His-ubiquitin expression efficiency purification <br />           controlled using anti-6His antibody. White lines indicate intervening lanes spliced out. (B) Analysis ubiquitylated forms Rpl25-GFP <br />           wild-type indicated mutant cells A. (C) Expression Rpl25-GFP modified form analyzed anti-GFP blot extracts from <br />           wild-type mutant cells starved indicated periods. Ub, ubiquitin; WT, wild type. <br />  <br />  <br />           The specificity results underscored observa-                   deletion UBP3, suggesting enzymes catalyze con- <br />           tion ubiquitylation different 60S subunit protein, Rpl5,                jugation deconjugation lysine residues Rpl25 <br />           Ltn1 dependent (Fig. S2). Next, examined                       (Fig. 2 B). Together, results raised possibility the <br />           Rpl25 ubiquitylation regulated Ubp3                    ubiquitin modification Rpl25 provide critical signal in <br />           steady-state ubiquitylation Rpl25-GFP, Rpl5-                    regulation ribophagy. <br />           GFP, increased ubp3&#65533;?&#65533; cells (Fig. 2, B C, t = 0;                       Interestingly, steady-state levels ubiquitylated Rpl25 <br />           Fig. S2). Therefore, Rpl25 appears specific target                altered nutrient starvation, rapid decay the <br />           E3 ligase activity Ltn1 deubiquitylating                      modified compared unmodified form wild-type cells. <br />           activity Ubp3. Importantly, low level ubiquitylated                     In contrast, ubp3&#65533;?&#65533; cells, ubiquitylated Rpl25-GFP levels re- <br />           Rpl25-GFP remaining LTN1 deletion increased                          mained readily detectable 6 h starvation (Fig. 2 C). <br />  <br />  <br />     912   JCB &#8226; VOLUME 204 &#8226; NUMBER 6 &#8226; 2014 <br />  Figure 3.&#8195; Ubiquitylation Rpl25 K74 regulates 60S ribophagy. (A) Indicated strains transformed <span id='am-71' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-72' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> encoding Rpl25-GFP <br /> (Rpl25) rpl25 K74,75R-GFP (rpl25KR) deletion genomic copy RPL25. (top) Comparable expression levels Rpl25-GFP rpl25 <br /> K74,75R-GFP cell lysates confirmed anti-GFP blotting. Purified 6His-ubiquitin&#8211;conjugated forms Rpl25-GFP rpl25 K74,75R-GFP <br /> (middle) 6His-ubiquitin expression (bottom) analyzed Fig. 2. Ub, ubiquitin. (B) Wild-type ubp3&#65533;?&#65533; cells expressing Rpl25-GFP rpl25 <br /> K74,75R-GFP starved indicated period time. (C) Degradation GFP-tagged proteins analyzed anti-GFP blot cell ex- <br /> tracts, ratio cleaved GFP full-length protein quantified time points independent experiments. a.u., arbitrary units. <br /> (D) Localization GFP-tagged proteins. C, cytoplasmic localization; V, vacuolar accumulation; C + V, localization cytoplasm vacuole (E) Deg- <br /> radation Rpl3 analyzed 24 h starvation Western blotting cell extracts using anti-Rpl3 (gift V. Albanese, Institut Jacques <br /> Monod, Paris, France) anti-Mex67 (as control) antibodies. Rpl3 Mex67 expression quantified normalized expression level at <br /> time 0. Significance differences observed ribophagy efficiency evaluated using Student&#8217;s t test. *, P = 0.01&#8211;0.05; **, P = 0.001&#8211;0.01; <br /> ***, P &lt; 0.001. White lanes indicate intervening lanes spliced out. The errors bars correspond standard deviations. <br />  <br />  <br />                                                                    Ubiquitylation ribosome inhibits ribophagy &#8226; Ossareh-Nazari et al.                   913 <br />            These results suggest starvation decreases Ltn1- <br />           dependent ubiquitylation Rpl25 and/or favors deconjuga- <br />           tion Ubp3 ribophagy. <br />  <br />           Ubiquitylation Rpl25 regulates <br />           60S ribophagy <br />           To examine Rpl25 ubiquitylation deu- <br />           biquitylation Ltn1 Ubp3-Bre5, respectively, critical <br />           ribophagy, set generate putative ubiquitylation- <br />           resistant Rpl25 mutant. Rpl25 lysine residue 74 (K74) been <br />           identified ubiquitylation site recent large-scale proteomic <br />           study (Starita et al., 2012). Because Rpl25 K74&#8217;s neighboring <br />           residue lysine (K75), residues sub- <br />           stituted arginine attempt generate ubiquitylation- <br />           resistant mutant. Plasmids encoding wild-type K74/75R mutant <br />           Rpl25-GFP (rpl25KR) introduced different strains fol- <br />           lowed deletion endogenous RPL25 chromosomal copy. <br />           As shown Fig. 3 A, mutation K74 75 arginine did not <br />           affect Rpl25 expression levels did reduce ubiquity- <br />           lation wild-type ubp3&#65533;?&#65533; cells (Fig. 3 A, compare sec- <br />           ond fourth lanes sixth eighth lanes), indicating that <br />           K74/75 constitutes major ubiquitylation site Rpl25. <br />           Residual monoubiquitylation rpl25KR detected might <br />           result modification alternative site site ar- <br />           tificially unmasked mutation (Hayakawa et al., 2012). <br />           Deleting LTN1 led severe decrease Rpl25 ubiquitylation, <br />           including residual ubiquitylation observed rpl25KR mu- <br />           tant protein, indicating Ltn1 represents unique ubiq- <br />           uitin ligase Rpl25 (Fig. 3 A, 10th 12th lanes). Finally, <br />           combining deletion UBP3 deletion LTN1 did re- <br />           store ubiquitylation wild-type mutant Rpl25, consistent <br />           model enzymes catalyze conjugation de- <br />           conjugation lysine residues Rpl25 (Fig. 3 A, 14th <br />           16th lanes). <br />                  Our model predicted Rpl25 K74 critical target <br />           Ltn1 inhibition ribophagy, rpl25KR mutant <br />           phenocopy LTN1 deletion functional assays. We <br />                                                                                   Figure 4.&#8195; Ltn1 rapidly degraded RING domain&#8211; proteasome- <br />           examined ribophagy phenotype strains expressing             dependent manner. (A) Wild-type cells expressing Bre5-GFP, Ltn1-Flag, <br />           rpl25KR mutant proteins source Rpl25. The              ltn1-&#65533;?&#65533;Ring-Flag starved indicated periods time. (B) Cells <br />           results Fig. 3 (B&#8211;E) starvation-induced degrada-           treated 30 min 100 &#181;M cycloheximide 100 &#181;M MG132. <br />                                                                                   Expression tagged proteins analyzed anti-GFP anti-Flag <br />           tion Rpl25-GFP endogenous untagged                 blotting cell extracts. Five times extracts analyzed for <br />           Rpl3 60S component similar strains expressing wild-              ltn1-&#65533;?&#65533;Ring-Flag cells. A nonspecific band served loading control. <br />           type rpl25KR mutant protein. In contrast, expression          White lanes indicate intervening lanes spliced out. WT, <br />                                                                                   wild type. <br />           ubiquitylation-resistant Rpl25 mutant (but wild-type <br />           Rpl25) partially significantly restored ribophagy defect <br />           observed ubp3&#65533;?&#65533; background (Fig. 3, B&#8211;D). This rescue             Ltn1 levels markedly down-regulated on <br />           observed monitoring degradation             nitrogen starvation <br />           Rpl3 60S component (Fig. 3 E). Thus, ubiquitylation Rpl25            The aforementioned <span id='am-66' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> led question ubiqui- <br />           does protect Rpl25 starvation-induced degradation         tylation/deubiquitylation balance regulated starvation <br />           generally protects 60S subunit. Importantly,               conditions. In course examining Ltn1 Ubp3- <br />           combining LTN1 deletion rpl25KR mutation did               Bre5 remain associated ribosomes the <br />           lead additional rescue ubp3&#65533;?&#65533; effect (Fig. S3), indi-          autophagic process, discovered nitrogen starvation in- <br />           cating Rpl25 K74/75 represents critical target         duced rapid marked decay Ltn1-Flag Ltn1-GFP <br />           role Ltn1 ribophagy. In conclusion, Ubp3-mediated de&#173;             levels quasicomplete disappearance 3 h; contrast, <br />           ubiquitylation critical efficient 60S ribophagy        Bre5-GFP degradation modest condi- <br />           Rpl25 ubiquitylation impaired LTN1 deletion,            tions significant degradation observed 24-h <br />           K74/75 mutation, both.                                               starvation (Fig. 4 A Fig. S3 B). The UBP3 deletion did not <br />  <br />  <br />     914   JCB &#8226; VOLUME 204 &#8226; NUMBER 6 &#8226; 2014 <br />                                                                                                       Figure 5.&#8195; Model role Ltn1 and <br />                                                                                                      Ubp3-Bre5 starvation-induced autophagy of <br />                                                                                                      60S ribosome. R yet-<span id='am-81' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-97'  ></span>unidentified</span> spe- <br />                                                                                                      cific autophagic receptor ribosome. Ub, <br />                                                                                                      ubiquitin; WT, wild type. <br />  <br />  <br />  <br />  <br /> result Ltn1 stabilization, arguing role Ubp3-        Rpl25 accelerate targeting 60S ribosomes to <br /> Bre5 upstream Ltn1 (Fig. S3 B). In contrast, RING do-              autophagosomes. In absence Ubp3, ubiquitylated Rpl25 <br /> main required starvation-induced degradation               accumulates, resulting defective degradation 60S ri- <br /> Ltn1, indicating catalytic activity Ltn1 required      bosomes. Cargoes selective autophagy pathways recognized <br /> its decay (Fig. 4 A). This rapid turnover Ltn1 oc-            specific receptors simultaneously bind autophagosome- <br /> curred starvation rich medium shown            associated proteins, ubiquitin-like protein Atg8 (LC3 <br /> disappearance 30-min treatment cycloheximide. In                mammalian cells; Shintani et al., 2002; Novak et al., 2010; <br /> rich medium, efficient degradation Ltn1 required          Suzuki et al., 2010; Motley et al., 2012). Although cargo adap- <br /> RING domain inhibited proteasome inhibitor                 <span id='am-61' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> identified ribophagy, speculate Ltn1- <br /> MG132 (Fig. 4 B).                                                         mediated ubiquitylation Rpl25 prevent interaction <br />       Together, results indicate efficient degrada-        receptor 60S ribosomes (Fig. 5). Alterna- <br /> tion Ltn1 mediated autoubiquitylation (RING domain)              tively, ubiquitin moiety preclude engulfment com- <br /> and degradation proteasome occurs rich me-             pletion autophagosome mechanism <br /> dium starvation. The general starvation-induced inhi-            (Shaid et al., 2013). <br /> bition protein synthesis likely results drastic change               The involvement Ltn1 ribophagy potentially <br /> the equilibrium translation degradation Ltn1,              provide cross talk mechanism protein quality control <br /> leading severe reduction Ltn1 expression consequent           ribosomal disposal, ribosomes stalled <br /> Ltn1-dependent ubiquitylation ribosome.                            translation aberrant proteins targeted for <br />       The ubiquitin protease activity Ubp3-Bre5 pre-          ribophagic degradation. However, degradation 60S ribosomes <br /> viously identified critical regulator selective autophagy      affected, measurable extent, strain har- <br /> 60S ribosomal subunits. The results presented indicate          boring mutation increases nonstop protein production <br /> deficiency 60S ribosome-associated Ltn1 E3 ligase rescued          (unpublished data; Bradley et al., 2003; Bengtson Joazeiro, <br /> the ribophagy defect ubp3&#65533;?&#65533; cells, indicating Ltn1 acts         2010). Moreover Tae2, partner Ltn1 ribosome quality <br /> a ribophagy inhibitor upstream Ubp3. We func-          control complex (Brandman et al., 2012; Defenouill&#232;re et al., <br /> tion Ltn1 mediated ubiquitylation lysine 74        2013) did regulate ribophagy. Nonetheless, cotranslational <br /> Rpl25, protein large ribosomal subunit, identified          quality control ribosome degradation pathways clearly share <br /> presumably direct target Ltn1 Ubp3. However, defi-           common factor, Cdc48, ubiquitin-dependent AAA <br /> ciency Rpl25 ubiquitylation did fully rescue ribophagy         ATPase. Cdc48 linked ribophagy nonstop <br /> defect ubp3&#65533;?&#65533; cells, suggesting additional Ubp3-dependent          protein degradation distinct ribosomal degrada- <br /> deubiquitylation events required ribophagy proceed             tion pathway eliminates 60S subunits containing nonfunctional <br /> optimally. As 60S ribosome known extensively ubiqui-         25S ribosomal RNA (Ossareh-Nazari et al., 2010; Brandman et al., <br /> tylated (Starita et al., 2012), interesting identify   2012; Fujii et al., 2012). <br /> additional modification sites implicated regulating riboph- <br /> agy efficiency. <br />                                                                           Materials methods <br />       Our data consistent model (Fig. 5) which <br /> Ltn1 leave ubiquitin marks Rpl25 prevent selec-           Yeast strains, plasmids, media <br /> tive degradation 60S subunits constitutively cleaved           The <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> strains plasmids used study are <br />                                                                           listed Table 1 Table 2. The plasmid encoding rpl25 K74,75R-GFP <br /> off Ubp3 wild-type cells. Upon starvation, rapid decay          constructed pRS315-RPL25-EGFP plasmid indicated Table 2 <br /> of Ltn1 combined Ubp3-mediated removal ubiquitin              (Gadal et al., 2001) using site-directed mutagenesis kit (QuikChange; <br />  <br />  <br />                                                                Ubiquitylation ribosome inhibits ribophagy &#8226; Ossareh-Nazari et al.              915 <br />            Table 1.&#8195; Strains used study <br />           Strains                                                                 Genotype                                                        Reference <br />  <br />           WT (BY4741)                          Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0                                                          EUROSCARF <br />           ubp3&#65533;?&#65533;                                Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0 UBP3::HIS                                                 This study <br />           ltn1&#65533;?&#65533;                                Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::KanMX6                                    Bengtson Joazeiro, 2010 <br />           &#65533;?&#65533;ring                                Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, ltn1&#65533;?&#65533;4500-3&#215;FLAG::KanMX6                        Bengtson Joazeiro, 2010 <br />           ltn1&#65533;?&#65533; ubp3&#65533;?&#65533;                          Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::KanMX6, UBP3::HIS                                  This study <br />           tae2&#65533;?&#65533;                                Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, TAE2::KanMX6                                            EUROSCARF <br />           ubp3&#65533;?&#65533; tae2&#65533;?&#65533;                          Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, TAE2::KanMX6 UBP3::HIS                                   This study <br />           Ltn1-Flag                            Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1-3&#215;FLAG-KanMX6                              Bengtson Joazeiro, 2010 <br />           Ltn1-GFP                             Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1-GFP-HIS                                             This study <br />           Ubp3&#65533;?&#65533; Ltn1-GFP                       Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, UBP3::KanMX6 LTN1-GFP-HIS                                This study <br />           Bre5-GFP                             Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, BRE5-GFP-HIS                                             This study <br />           rpl25&#65533;?&#65533; Rpl25                         Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, RLP25::HIS, pRS315-RPL25-EGFP                            This study <br />           rpl25&#65533;?&#65533; rpl25KR                       Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, RPL25::HIS, pRS315-RPL25KR-EGFP                          This study <br />           ubp3&#65533;?&#65533; rpl25&#65533;?&#65533; Rpl25                   Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, UBP3::KanMX6, RPL25::HIS,                                This study <br />                                                 pRS315-RPL25-EGFP <br />           ubp3&#65533;?&#65533; rpl25&#65533;?&#65533; rpl25KR                 Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, UBP3::KanMX6, RPL25::HIS,                                  This study <br />                                                 pRS315-RPL25KR-EGFP <br />           ltn1&#65533;?&#65533; rpl25&#65533;?&#65533; Rpl25                   Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::HPH RLP25::HIS,                                      This study <br />                                                 pRS315-RPL25-EGFP <br />           ltn1&#65533;?&#65533; rpl25&#65533;?&#65533; rpl25KR                 Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::HPH RPL25::HIS,                                      This study <br />                                                 pRS315-RPL25KR-EGFP <br />           ltn1&#65533;?&#65533; ubp3&#65533;?&#65533; rpl25&#65533;?&#65533; Rpl25             Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::HPH UBP3::KanMX6,                                    This study <br />                                                 RPL25::HIS, pRS315-RPL25-EGFP <br />           ltn1&#65533;?&#65533; ubp3&#65533;?&#65533; rpl25&#65533;?&#65533; rpl25KR           Mat a, his3&#65533;?&#65533;1, leu2&#65533;?&#65533;0, met15&#65533;?&#65533;0, ura3&#65533;?&#65533;0, LTN1::HPH UBP3::KanMX6,                                    This study <br />                                                 RPL25::HIS, pRS315-RPL25KR-EGFP <br />  <br />           EUROSCARF, European <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae archive functional analysis; WT, wild type. <br />  <br />  <br />           Agilent Technologies). Yeast cultures grown 30&#176;C rich me-     Transformer; Physik Instrumente) mounted underneath objective <span id='am-1' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Lens" datatype="xsd:string"></span>lens</span>. <br />           dium (YPD [yeast, peptone, dextrose]) synthetic medium (SD) contain-      Maximum intensity projections performed using MetaMorph software <br />           ing 0.67% yeast nitrogen base ammonium sulfate 2% dextrose             (Molecular Devices). The localization GFP-tagged proteins analy&#173; <br />           supplemented appropriate nutrients. Cells starved starva-      zed &#8805;200 cells condition experiment, results ob- <br />           tion medium (SD-N) containing 0.17% yeast nitrogen base amino           tained independent experiments. <br />           acids 2% dextrose (Kraft et al., 2008). <br />                                                                                           Purification ubiquitylated proteins <br />           Preparation yeast total extracts                                             Cells transformed plasmid encoding 6His-ubiquitin CUP1 <br />           Cells grown YPD synthetic medium collected expo-          promoter grown selective media stimulated overnight with <br />           nential growth phase (OD600 1.5 0.8, respectively). Total protein ex-     0.1 mM CuSO4. Cells lysed 6 M guanidinium-HCl, 0.1 M <br />           tracts prepared NaOH-TCA lysis method analyzed               Na2HPO4/NaH2PO4, 0.01 M Tris-HCl, pH 8.0, 0.1% Triton X-100 plus <br />           Western blotting using anti-GFP antibodies (Roche). Signals corresponding       5 mM imidazole, 10 mM &#65533;?&#65533;-mercaptoethanol, protease inhibitors. <br />           GFP fusion proteins GFP quantified using ImageJ software        Purifications His6-ubiquitin&#8211;modified proteins performed on <br />           (National Institutes Health), ratio cleaved noncleaved GFP     Ni2+-nitrilotriacetic acid&#8211;agarose beads eluted (QIAGEN). The beads <br />           protein used measure ribophagy process (Kraft et al., 2008;          washed 8 M urea, 0.1 M Na2HPO4/NaH2PO4, 0.01 M Tris-HCl, <br />           Ossareh-Nazari et al., 2010).                                                   pH 6.3, 10 mM &#65533;?&#65533;-mercaptoethanol, 0.2% Triton X-100 elution <br />                                                                                           Western blot analysis using anti-GFP (Roche) anti-6His (Covalab) <br />           Fluorescence microscopy                                                         antibodies (Vitaliano-Prunier et al., 2008; Hayakawa et al., 2012). <br />           Wide-field fluorescence images living cells acquired using micro- <br />           scope (DMR; Leica) 100&#215; Plan Apochromat HCX oil immersion                Statistical analyses <br />           objective high-sensitive cooled interlined charge-coupled device          Significance differences observed ribophagy efficiency (Fig. 1, <br />           <span id='am-44' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span>camera</span> (CoolSNAP HQ2; Photometrics). Rapid precise z positioning            A&#8211;D; Fig. 3, C&#8211;E) evaluated using Student&#8217;s t test. No asterisk cor- <br />           accomplished piezoelectric motor (Linear Variable Differential         responds P &gt; 0.05, asterisk corresponds P = 0.01&#8211;0.05, two <br />                                                                                           asterisks correspond P = 0.001&#8211;0.01. <br />  <br />           Table 2.&#8195; Plasmids used study                                           Online supplemental material <br />                                                                                           Fig. S1 shows Ltn1 does affect general autophagy its <br />           Plasmid                        Description                Reference             RING domain, cofactor Tae2, required function ri- <br />                                                                                           bophagy. Fig. S2 shows ribosomal protein Rpl5 ubiquitylated but <br />           YEp352-6HisUb            2&#65533;?&#65533; URA3 CUP-6His-Ub <br />                                                    Vitaliano-Prunier et al., <br />                                                                                           Ltn1- Ubp3-independent manner. Fig. S3 demonstrates com- <br />                                                              2008 <br />                                                                                           bining LTN1 deletion K74/75R mutation Rpl25 synergistic <br />           pRS315-RPL25-EGFP       CEN LEU2           Gadal et al., 2001                   additive effect. Online supplemental material available http:// <br />           pRS315-RPL25KR-   CEN LEU2, RPL25K74,75R         This study                     www.jcb.org/cgi/content/full/jcb.201308139/DC1. <br />             EGFP <br />                                                                                           We grateful E. Hurt, V. Panse, V. Albanese gift reagents. <br />           pRS315-RPL5-EGFP        CEN LEU2           Gift V. Pansea <br />                                                                                           We thank A. Babour J. Weitzman critically reading manuscript. <br />           Plasmid pRS315-RPL25-EGFP gift E. Hurt (Heidelberg University,               This study funded grants Association Research <br />           Heidelberg, Germany).                                                           <span id='am-34' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> C. Dargemont, Fondation pour la Recherche Medi- <br />           a <br />            Eidgen&#246;ssische Technische Hochschule Z&#252;rich, Z&#252;rich, Switzerland.              cale (to C.A. Ni&#241;o), National Institute Neurological Disorders and <br />  <br />  <br />     916   JCB &#8226; VOLUME 204 &#8226; NUMBER 6 &#8226; 2014 <br />  Stroke National Institutes Health (grant R01 NS075719),               Li, K., B. Ossareh-Nazari, X. Liu, C. Dargemont, R. Marmorstein. 2007. <br /> a Research Scholar Grant (RSG-08-298-01-TBE) American Cancer                          Molecular basis bre5 cofactor recognition ubp3 deubiquitylat- <br /> Society C.A.P. Joazeiro.                                                                    ing enzyme. J. Mol. Biol. 372:194&#8211;204. http://dx.doi.org/10.1016/j.jmb <br />                                                                                                .2007.06.052 <br />        The authors declare competing financial interests. <br />                                                                                        Motley, A.M., J.M. Nuttall, E.H. Hettema. 2012. Pex3-anchored Atg36 tags <br /> Submitted: 23 August 2013                                                                      peroxisomes degradation Saccharomyces cerevisiae. EMBO J. <br />                                                                                                31:2852&#8211;2868. http://dx.doi.org/10.1038/emboj.2012.151 <br /> Accepted: 5 February 2014 <br />                                                                                        Novak, I., V. Kirkin, D.G. McEwan, J. Zhang, P. Wild, A. Rozenknop, V. <br />                                                                                                Rogov, F. L&#246;hr, D. Popovic, A. Occhipinti, et al. 2010. Nix selective <br />                                                                                                autophagy receptor mitochondrial clearance. EMBO Rep. 11:45&#8211;51. <br /> References                                                                                     http://dx.doi.org/10.1038/embor.2009.256 <br />                                                                                        Ossareh-Nazari, B., M. Bonizec, M. Cohen, S. Dokudovskaya, F. Delalande, C. <br /> Bengtson, M.H., C.A. Joazeiro. 2010. Role ribosome-associated E3 <br />                                                                                                Schaeffer, A. Van Dorsselaer, C. Dargemont. 2010. Cdc48 Ufd3, <br />         ubiquitin ligase protein quality control. Nature. 467:470&#8211;473. http:// <br />                                                                                                new partners ubiquitin protease Ubp3, required ribophagy. <br />         dx.doi.org/10.1038/nature09371 <br />                                                                                                EMBO Rep. 11:548&#8211;554. http://dx.doi.org/10.1038/embor.2010.74 <br /> Bradley, M.E., S. Bagriantsev, N. Vishveshwara, S.W. Liebman. 2003. <br />                                                                                        Shaid, S., C.H. Brandts, H. Serve, I. Dikic. 2013. Ubiquitination selec- <br />         Guanidine reduces stop <span id='am-49' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> read-through caused missense muta- <br />                                                                                                tive autophagy. Cell Death Differ. 20:21&#8211;30. http://dx.doi.org/10.1038/ <br />         tions SUP35 SUP45. Yeast. 20:625&#8211;632. http://dx.doi.org/10.1002/ <br />                                                                                                cdd.2012.72 <br />         yea.985 <br />                                                                                        Shao, S., K. von der Malsburg, R.S. Hegde. 2013. Listerin-dependent na- <br /> Brandman, O., J. Stewart-Ornstein, D. Wong, A. Larson, C.C. Williams, G.W. Li, <br />                                                            <br /> </body></html>