<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Identification M. tuberculosis Rv3441c M. <br /> smegmatis MSMEG_1556 Essentiality M. <br /> smegmatis MSMEG_1556 <br /> Shuang Li1., Jian Kang1., Wendan Yu1, Yan Zhou1, Wenli Zhang1, Yi Xin2,3, Yufang Ma1,3* <br /> 1 Department Biochemistry Molecular Biology, Dalian Medical Universtiy, Dalian, Liaoning, China, 2 Department Biotechnology, Dalian Medical University, Dalian, <br /> Liaoning, China, 3 Liaoning Provincial Core Lab Glycobiology Glycoengineering, Dalian, Liaoning, China <br />  <br />  <br />  <br />      Abstract <br />      The normal growth mycobacteria attributes integrity cell wall core consists peptidoglycan (PG), <br />      arabinogalactan (AG) mycolic acids. N-acetyl glucosamine (GlcNAc) essential component PG AG of <br />      mycobacterial cell wall. The biosynthetic pathway UDP-N-acetylglucosamine (UDP-GlcNAc), sugar donor GlcNAc, <br />      different prokaryotes eukaryotes. The conversion glucosamine-6-phosphate glucosamine-1-phosphate, which <br />      catalyzed phosphoglucosamine mutase (GlmM), unique prokaryotes. Bioinformatic analysis showed Msm <br />      MSMEG_1556 Mtb Rv3441c homologous Ec GlmM. In study, soluble Msm MSMEG_1556 protein Mtb <br />      Rv3441c protein expressed E. coli BL21(DE3) phosphoglucosamine mutase activity detected. In <span id='am-81' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> <br />      investigate essentiality MSMEG_1556 growth M. smegmatis, generated conditional <br />      MSMEG_1556 knockout mutant, harbored thermo-sensitive rescue plasmid carrying Mtb Rv3441c. As rescue <br />      plasmid unable complement MSMEG_1556 deficiency 42uC, MSMEG_1556 knockout mutant did grow. The <br />      dramatic morphological changes MSMEG_1556 knockout mutant temperature shift 30uC 42uC been <br />      observed scanning <span id='am-84' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> microscope. These results demonstrated MSMEG_1556 essential growth M. <br />      smegmatis. <span id='am-68' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span>This</span> study provided evidence GlmM enzyme potential target developing anti-tuberculosis <br />      drugs. <br />  <br />   Citation: Li S, Kang J, Yu W, Zhou Y, Zhang W, et al. (2012) Identification M. tuberculosis Rv3441c M. smegmatis MSMEG_1556 Essentiality M. <br />   smegmatis MSMEG_1556. PLoS ONE 7(8): e42769. doi:10.1371/journal.pone.0042769 <br />   Editor: Riccardo Manganelli, University Padova, Italy <br />   Received April 17, 2012; Accepted July 10, 2012; Published August 8, 2012 <br />   Copyright: &#223; 2012 Li et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Basic Research Program China (2012CB518803; http://www.973.gov.cn/English/Index.aspx), National <br />   Natural Science Foundation China (30970067; http://www.nsfc.gov.cn/Portal0/default166.htm), National Science Technology Major Project <br />   (2012ZX10003003-006; http://www.nmp.gov.cn/). The funders role study design, <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: yufang_ma@hotmail.com <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                           cursor synthesis PG layer, direct glycosyl <br />                                                                                        donor disaccharide linker, therefore, plays vital <br />    Nowadays, millions deaths year (2% increased                               role mycobacterial growth [12,13,14]. Three enzymes <span id='am-89' about='obo:NCBITaxon_173684' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_41937 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_4011 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Gluta" datatype="xsd:string"></span>gluta</span>- <br /> incidence) caused <span id='am-34' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span>Mycobacterium</span> tuberculosis considered                    fructose-6-phosphate transferase (GlmS), phosphoglucosa- <br /> to major public health threat [1]. However, vaccine failed                     mutase (GlmM), glucosamine-1-phosphate acetyl transfer- <br /> provide protective immunity efforts control tuberculosis                    ase/N-Acetylglucosamine-1-phosphate         urididyl     transferase <br /> were compromised evolved stronger, drug-                             (GlmU) involve metabolic pathway UDP-GlcNAc E. <br /> resistant forms [2,3,4,5]. As know, cell wall                            coli [13,15,16,17,18]. It noteworthy reactions catalyzed by <br /> Mycobacterium <span id='am-78' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> waxy, hydrophobic, thicker                           GlmM GlmU unique prokaryotes (Fig. 1). The function <br /> than bacteria, low permeability resistance cell wall                  Msm GlmU Mtb GlmU identified GlmU <br /> substantially contributes defense adverse factors [6,7].                     confirmed essential growth M. smegmatis and <br /> Thereby, enzymes involved metabolic pathways                         M. tuberculosis [19,20]. Therefore, GlmM essential for <br /> cell wall potential excellent targets new anti-tuberculosis                    mycobacteria used potential target anti- <br /> drugs [8,9].                                                                           tuberculosis drug. <br />    The mycobacterial cell wall consists mycolate                               The bioinformatics analysis data showed Msm <br /> peptidoglycan (PG) layer held arabinogalactan (AG)                         MSMEG_1556 Mtb Rv3441c homologous Ec GlmM. <br /> layer [10,11]. AG attached muramic acid residue                       We proposed Msm MSMEG_1556 protein Mtb Rv3441c <br /> PG disaccharide linker (a-L-rhamnosyl-a-D-N-acetyl-                          protein catalyzing reaction biosynthetic pathway of <br /> glucosaminosyl-1-phosphate), glycan PG disaccha-                       UDP-GlcNAc E. coli. In study, identified both <br /> ride repeat unit (N-acetylmuramic acid-N-acetyl glucosamine).                          Msm MSMEG_1556 Mtb Rv3441c protein the <br /> UDP-N-acetylglucosamine (UDP-GlcNAc) important pre-                              phosphoglucosamine mutase activity. The essentiality of <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         1                                August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                                  Mycobacterial GlmM <br />  <br />  <br />                                                                                  constructing conditional M. smegmatis MSMEG_1556 gene <br />                                                                                  knockout strain allelic exchange recombinant experiments. <br />                                                                                  Sucrose added LB agar final concentration 10% <br />                                                                                  required. Antibiotics added following final <br />                                                                                  concentrations: ampicillin (Amp), 100 mg/ml; gentamicin (Gen), <br />                                                                                  5 mg/ml E. coli; kanamycin (Kan), 50 mg/ml E. coli and <br />                                                                                  25 mg/ml M. smegmatis; streptomycin (Str), 25 mg/ml E. coli <br />                                                                                  12.5 mg/ml M. smegmatis. <br />  <br />                                                                                  Sequence alignment Ec GlmM, Msm MSMEG_1556 and <br />                                                                                  Mtb Rv3441c <br />                                                                                    The amino acid sequences Ec GlmM protein, Msm <br />                                                                                  MSMEG_1556 protein Mtb Rv3441c protein aligned <br />                                                                                  using Multalin online tool (http://multalin.toulouse.inra.fr/ <br />                                                                                  multalin/) [22]. <br />  <br />                                                                                  Expression purification Ec GlmM protein, Msm <br />                                                                                  MSMEG_1556 protein, Mtb Rv3441c protein, Mtb <br />                                                                                  GlmU protein <br /> Figure 1. Biosynthetic pathway UDP-GlcNAc prokaryotes                         The genomic DNA E. coli BL21(DE3) prepared as <br /> and eukaryotes. The biosynthetic pathway UDP-GlcNAc                        described previously [23], modification. Ec glmM gene was <br /> prokaryotes eukaryotes different. In prokaryotes, UDP-GlcNAc            amplified E. coli BL21(DE3) genomic DNA using Ec <br /> was biosynthesized reactions (1), (2), (3) (6); In eukaryotes,       glmM1 Ec glmM2 primers (Table 1) cloned into <br /> UDP-GlcNAc formed reactions (1), (4), (5) (6). The               pJET1.2/blunt vector generate plasmid pKJ3 (pJET-Ec glmM) <br /> reactions catalyzed phosphoglucosamine mutase (GlmM) and <br /> glucosamine-1-phosphate acetyl transferase (GlmU) unique to <br />                                                                                  (Table S1). The genomic DNA M. semgmatis mc2155 was <br /> prokaryotes.                                                                     prepared described 20 ml culture [11]. The Msm <br /> doi:10.1371/journal.pone.0042769.g001                                            MSMEG_1556 amplified M. smegmatis mc2155 genomic <br />                                                                                  DNA PCR using Msm glmM1 Msm glmM2 primers <br /> MSMEG_1556 mycobacteria determined observing                          (Table 1) cloned pJET1.2/blunt vector generate <br /> growth curve, colony forming unit (CFU) morphology                        pKJ1 (pJET-Msm MSMEG_1556) plasmid (Table S1). The Mtb <br /> MSMEG_1556 knockout mutant. In addition, dinitrothiocyano-                       Rv3441c gene amplified H37Rv genomic DNA <br /> benzene (DNTB) chromogenic method [20,21] microtiter plate                    (supplied Colorado State University NIH contract) by <br /> for measurement phosphoglucosamine mutases activity                   using Mtb glmM1 Mtb glmM2 primers (Table 1), was <br /> described.                                                                       cloned pMD18-T vector generate pLS1 (pMD18-Mtb <br />                                                                                  Rv3441c) plasmid (Table S1). After confirmation DNA <br /> Materials Methods                                                            sequencing, Ec glmM ligated NcoI BamHI <br />                                                                                  sites pET16b, resulting pKJ4 (pET16b-Ec glmM) (Table S1). <br /> Bacterial strains, <span id='am-99' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>, culture conditions                              The Msm MSMEG_1556 ligated NdeI EcoRI <br />    The bacterial strains plasmids used study listed             sites pCold II, resulting pKJ2 (pCold II -MSMEG_1556) <br /> in Table S1. E.coli DH5a, NovaBlue BL21(DE3) cells                      (Table S1), Mtb Rv3441c ligated NdeI and <br /> grown <span id='am-58' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span>-Bertani (LB) medium 37uC routinely. M.                         BamHI sites pCold II yielding pLS2 (pCold II-Mtb Rv3441c) <br /> smegmatis mc2155 strain grown LB broth containing 0.05%                   plasmid (Table S1). pKJ4, pKJ2 pLS2 transformed E. <br /> Tween 80 LB agar 37uC routinely. M. smegmatis mc2155                    coli BL21(DE3) overexpression Ec GlmM protein, Msm <br /> was used cloning MSMEG_1556 gene upstream region                    MSMEG_1556 protein Mtb Rv3441c protein, respectively. <br />  <br />  Table 1. Primers used study. <br />  <br />  <br />  Primers          Primer sequences (59R39) <br />  <br />  Ec glmM1         GCCCATGGTGATGAGTAATCGTAAATATTTCG (underlined NcoI site) <br />  Ec glmM2         TTGGATCCTTAATGATGATGATGATGATGAACGGCTTTTACTGCATCGGCG (underlined BamHI site) <br />  Msm glmM1        TTCATATGGCTCGACTGTTCGGCAC (underlined NdeI site) <br />  Msm glmM2        CCGAATTCTATCCCTGCAGACTCAC (underlined EcoRI site) <br />  Mtb glmM1        GGCGCATATGGGTCGACTGTTTGGCAC (underlined NdeI site) <br />  Mtb glmM2        TAATGGATCCTCAGCGCGCGGTGCTCACCG (underlined BamHI site) <br />  Msm glmM3        GACTAGTGGTGTCCTCGAAGACGTGCCCATCGG (underlined SpeI site) <br />  Msm glmM4        TGCGGCCGCCTATCCCTGCAGACTCACGGATTC (underlined NotI site) <br />  Msm glmM5        ACTGGGCACCGCAGGTGTC <br />  Msm glmM6        GTTCGGCGACTATCCCTGC <br />  <br />  doi:10.1371/journal.pone.0042769.t001 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   2                          August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                             Mycobacterial GlmM <br />  <br />  <br />                                                                           correct trial errors reaction GlmU protein with <br />                                                                           GlcNH2-6-P. The reaction incubated 37uC 30 min and <br />                                                                           terminated adding 50 ml stop solution containing 50 mM <br />                                                                           Tris-HCl, pH 7.5, 6 M guanidine hydrochloride then <br />                                                                           incubated 10 min addition 50 ml Ellman&#8217;s reagent <br />                                                                           solution containing 0.2 mM DTNB buffer 50 mM Tris- <br />                                                                           HCl, pH 7.5, 1 mM EDTA. TNB, product generated <br /> Figure 2. Assay phosphoglucosamine mutase. Phosphogluco-               reaction CoA-SH DTNB, monitored at <br /> samine mutase catalyzes conversion GlcNH2-6-P GlcNH2-1-P.       405 nm Benchmark Plus plate reader (Thermo). To confirm <br /> GlcNH2-1-P acetyl CoA catalyzed glucosamine-1-<span id='am-104' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>-           product CoA-SH reaction mixture applied Nova-Pak <br /> phate acetyltransferase activity GlmU produce GlcNAc-1-P        C18 (3.96150 mm, 4 mm) CoA-SH separated flow <br /> CoA-SH detected DTNB. TNB, product generated        rate 1 ml/min phosphate buffer (pH 6.5)-methanol (95:5) <br /> the reaction CoA-SH DTNB, monitored 405 nm. <br /> doi:10.1371/journal.pone.0042769.g002 <br />                                                                           monitored 259 nm. One unit specific enzyme activity <br />                                                                           defined nmol CoA-SH produced mg protein per <br />                                                                           minute specific condition. <br />    E. coli BL21(DE3) strains carrying different expression plasmids <br /> were grown 37uC 300 ml LB broth containing Amp. When <br /> OD600 culture reached 0.5, BL21(DE3)/pKJ4 culture           Construction conditional M. smegmatis MSMEG_1556 <br /> induced 0.4 mM isopropyl-D-thiogalactopyranoside (IPTG)             gene knockout strain <br /> 37uC 3 hours; BL21(DE3)/pKJ2 BL21(DE3)/pLS2                      Msm MSMEG_1556 gene (1362 bp) upstream <br /> transferred 15uC-incubator induced 1 mM IPTG              sequence (500 bp) amplified mc2155 genomic DNA by <br /> 24 hours. The cells harvested resuspended 8 ml           using Msm glmM3 Msm glmM4 primers (Table 1). The PCR <br /> lysis buffer (20 mM Tris-HCl, pH 8.0, 500 mM NaCl 20%                 product (1862 bp) purified cloned pMD18-T vector <br /> glycerol) 1 mM phenylmethyl-sulphonyl fluoride (PMSF)                yield pLS3 (pMD18-Msm MSMEG_1556) plasmid (Table S1). <br /> followed sonication. After centrifugation 20,0006g              After confirmation DNA sequencing, kanR cassette from <br /> 30 minutes, resulting supernatant loaded 1 ml Ni-            pUC4K inserted XhoI site pLS3, resulting pLS4 <br /> NTA column (Qiagen) previously equilibrated equilibrium              (pMD18-Msm MSMEG_1556::kanR) plasmid (Table S1). pLS4 <br /> buffer. The column washed 20 ml wash buffer (20 mM               digested NotI SpeI Msm MSMEG_1556::kanR <br /> Tris-HCl, pH 8.0, 500 mM NaCl, 20% glycerol 45 mM                     fragment cloned NotI SpeI sites pPR27-xylE to <br /> imidazole). The purified protein eluted 15 ml elute buffer       generate conditional replication plasmid pLS5 (pPR27-Msm <br /> (20 mM Tris-HCl, pH 8.0, 500 mM NaCl, 20% glycerol                    MSMEG_1556::kanR) (Table S1). pLS5 harbors counter- <br /> 200 mM imidazole) 1 mM PMSF, 7 ml                  selectable marker sacB <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span>Bacillus subtilis</span> use selecting the <br /> collected analyses SDS-PAGE Western blot            double crossover event presence sucrose. <br /> detection phosphoglucosamine mutase activity.                             Since Mtb Rv3441c gene Msm MSMEG_1556 gene <br />    The purified Ec GlmM protein, Msm MSMEG_1556 protein                   identified glmM gene encode phosphoglucosamine <br /> and Mtb Rv3441c protein run 12% SDS-PAGE                      mutase, Mtb Rv3441c gene used construction rescue <br /> transferred nitrocellulose membrane (Pall Corp) blotting          plasmid. pLS2 digested NdeI BamHI Mtb <br /> buffer (20 mM Tris-base, 150 mM <span id='am-63' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> 20% methanol).                Rv3441c gene ligated NdeI BamHI sites of <br /> The membrane blocked 1% BSA TBST buffer                       pET23b-Phsp60, yielding pLS6 (pET23b-Phsp60-Mtb Rv3441c) <br /> (10 mM Tris&#8211;HCl, pH 8.0, 150 mM NaCl, 0.05% Tween 20)                 plasmid. pLS6 digested XbaI BamHI Phsp60-Mtb <br /> incubated (anti)-polyhistidine clone His-1 antibody (Sigma)       Rv3441c fragment cloned XbaI BamHI sites of <br /> 1:5000 dilution followed washing TBST. The membrane               pCG76 generate rescue plasmid pLS7 (pCG76-Phsp60- Mtb <br /> was incubated antimouse-IgG-AP conjugate protein             Rv3441c) [25]. <br /> band visualized BCIP/NBT solution.                                    M. smegmatis mc2155 electro-competent cells prepared as <br />    The Mtb GlmU protein purified described previously              described [26]. The conditional replication plasmid pLS5 was <br /> [19].                                                                     electroporated mc2155 competent cells Electroporator <br />                                                                           2510 (Eppendorf). The transformants grown LB agar <br />                                                                           plates containing Kan Gen 30uC. One colony was <br /> Assaying phosphoglucosamine mutase Ec GlmM                             incubated LB broth containing 0.05% Tween 80, Kan and <br /> protein, Msm MSMEG_1556 protein Mtb Rv3441c                           Gen 30uC culture plated LB agar <br /> protein                                                                   plate containing Kan Gen followed incubation 42uC for <br />    The phosphoglucosamine mutase activity Ec GlmM protein,             6 days. Since plasmid pLS5 able replicate 42uC, the <br /> Msm MSMEG_1556 protein Mtb Rv3441c protein                       MSMEG_1556::kanR pLS5 single crossover upstream <br /> determined coupled assay, GlcNH2-1-P converted            downstream MSMEG_1556 locus M. smegmatis <br /> from GlcNH2-6-P mutase quantitatively converted           genome colonies grown 42uC. M. smegmatis LS1 mutants (M. <br /> UDP-GlcNAc presence purified GlmU enzyme [24]. The              smegmatis mc2155 pLS5 plasmid integrated the <br /> by-product CoA-SH reaction catalyzed GlmU                     MSMEG_1556 gene locus) (Table S1) homolo- <br /> acetyltransferase activity measured colorimetric            gous recombination selected Southern blot [11]. <br /> assay coupled 5, 59-dithio-bis-(2-nitrobenzoic acid) (DTNB)             The rescue plasmid pLS7 electroporated M. smegmatis <br /> (Fig. 2). The reaction mixture 50 ml contained 50 mM Tris-             LS1 electro-competent cells transformants grown on <br /> HCl, pH8.0, 2.5 mM MgCl2, 1 mM GlcNH2-6-P, 0.6 mM acetyl                  LB agar plate containing Kan, Gen Str grown 30uC. <br /> CoA, 0.7 mM Glc-1, 6-diP, purified Mtb GlmU protein (1 mg)            One colony incubated LB broth containing 0.05% Tween <br /> purified Ec GlmM protein (0.625 mg), Msm MSMEG_1556                       80, Kan, Gen Str grown 30uC. The culture then <br /> protein (0.625 mg) Mtb Rv3441c protein (0.625 mg) 96-             plated LB agar plate containing Kan, Str 10% <br /> well microtiter plate. A control GlmM protein used         sucrose. M. smegmatis LS2 (mcD1556::pLS7), MSMEG_1556 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            3                           August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                       Mycobacterial GlmM <br />  <br />  <br />  <br />  <br /> Figure 3. Sequence alignment Ec GlmM, Msm MSMEG_1556 Mtb Rv3441c. Msm MSMEG_1556 protein 38% identical Ec <br /> GlmM Mtb Rv3441c protein 41% identical Ec GlmM. Mtb Rv3441c protein Msm MSMEG_1556 protein 79% identities. * indicates <br /> the homology organisms, # indicates homology M. tuberculosis M. smegmatis, ! indicates homology <br /> between E. coli M. smegmatis E. coli M. tuberculosis. <br /> doi:10.1371/journal.pone.0042769.g003 <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                         4                          August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                                Mycobacterial GlmM <br />  <br />  <br />                                                                               enzyme. The resulting DNA fragments separated 0.8% <br />                                                                               agarose gel. The DNA transferred Hybond-N+ <br />                                                                               membrane (GE Healthcare). DNA hybridization detection of <br />                                                                               probe performed recommended Roche. <br />  <br />                                                                               Growth M. smegmatis LS2 strain <br />                                                                                  The growth M. smegmatis LS2 determined observing <br />                                                                               growth curve CFU, respectively. To measure growth curve, <br />                                                                               M. smegmatis LS2 inoculated 5 ml LB broth containing <br />                                                                               0.05% Tween 80 appropriate antibiotics, cells were <br />                                                                               incubated 30uC 42uC respectively. M. smegmatis mc2155 <br />                                                                               carrying rescue plasmid pLS7 wild type M. smegmatis <br />                                                                               mc2155 used controls. The OD600 cultures was <br />                                                                               monitored interval 24 h growth curves 30uC <br />                                                                               42uC obtained. <br />                                                                                  To measure CFU, M. smegmatis LS2 grown LB medium <br />                                                                               containing 0.05% Tween 80 Kan 30uC OD600 <br />                                                                               reached 0.10, culture transferred 42uC incubator. <br />                                                                               The culture grown 30uC control. The OD600 was <br />                                                                               determined interval 24 hours. For CFU determinations <br />                                                                               (taken time points), dilutions spread LB agar <br />                                                                               plates containing Str Kan. The plates incubated 30uC <br />                                                                               CFU counted. <br /> Figure 4. Analysis purified Ec GlmM protein, Msm <br /> MSMEG_1556 protein Mtb Rv3441c protein SDS-PAGE <br /> (A) Western blot (B). A. SDS-PAGE analysis. M. PageRuler                  Expression MSMEG_1556 protein M. smegmatis LS2 <br /> prestained protein ladder (Fermentas); lane 1. purified Ec GlmM           strain <br /> protein expected molecular weight 48.99 kDa; lane 2.              Western blot used analyze expression levels GlmM <br /> purified Msm MSMEG_1556 protein expected molecular weight <br />                                                                               protein M. smegmatis LS2. M. smegmatis LS2 grown 500 ml <br /> of 47.99 kDa; lane 3. purified Mtb Rv3441c protein expected <br /> molecular weight 47.31 kDa. B. Western blot analysis. M. PageRuler         LB broth 30uC, switched 42uC OD600 the <br /> prestained protein ladder (Fermentas); lane 1. purified Ec GlmM           culture 0.01. M. smegmatis LS2 kept growing 30uC as <br /> protein; lane 2. purified Msm MSMEG_1556 protein; lane 3.             control. The culture temperature shifting 72 h was <br /> purified Mtb Rv3441c protein.                                                 harvested cells resuspended 4 ml lysis buffer with <br /> doi:10.1371/journal.pone.0042769.g004                                         1 mM PMSF followed sonication. After centrifugation at <br />                                                                               20,0006g 30 minutes, resulting supernatant collected <br /> gene knockout strain (Table S1) Msm MSMEG_1556 gene                     Western blot analyses. The protocol followed previously <br /> was knocked second homologues recombination                   described (anti)-MSMEG_1556 antibody (prepared lab, <br /> was selected using Southern blot. To confirm M. smegmatis LS2,             unpublished data) used instead (anti)-polyhistidine <br /> the DNA fragment containing Msm MSMEG_1556::kanR                      clone His-1 antibody. <br /> flanking sequences upstream downstream Msm <br /> MSMEG_1556 amplified M. smegmatis LS2 genome                      Morphology M. smegmatis LS2 strain shifting <br /> using Msm glmM5 Msm glmM6 primers (Table 1). The <br />                                                                               temperature 30uC 42uC <br /> amplified PCR products Msm MSMEG_1556::kanR M. <br />                                                                                  M. smegmatis LS2 grown 20 ml LB broth 30uC, and <br /> smegmatis LS2 Msm MSMEG_1556 wild type M. <br />                                                                               switched 42uC OD600 culture 0.01. M. <br /> smegmatis mc2155 distinguished 1% agrose gel. <br />                                                                               smegmatis LS2 kept growing 30uC control. The cells were <br />                                                                               harvested shifting temperature 72 h, 120 h 192 h, <br /> Southern blot analysis                                                        respectively. Scanning electron microscopy (SEM) samples were <br />    The MSMEG_1556 DNA probe prepared using DIG                         prepared previously described [19]. Briefly, cell pellet was <br /> High Prime Labeling Detection Starter Kit I (Roche). The                  washed times 0.1 M Phosphate buffer (pH 7.4) and <br /> pLS3 digested SpeI NotI Msm MSMEG_1556                         fixed 2.5% glutaraldehyde followed fixation 1% <br /> was purified. One microliter purified Msm MSMEG_1556                   OsO4. The cells dehydrated graded series of <br /> was used template generate probe. The genomic DNA                     ethanol (20, 40, 60, 70, 80, 90, 100%). After critical point dry <br /> isolated different colonies digested overnight SmaI               cells applied silicon wafer slide coated gold. <br />  <br />  Table 2. Specific phosphoglucosamine mutase activity GlmM proteins. <br />  <br />  <br />  Proteins                                                             Specific enzyme activity (nmol?min21?mg21) <br />  <br />  Purified Ec GlmM                                                     156 <br />  Purified Msm MSMEG_1556                                              145 <br />  Purified Mtb Rv3441c                                                 151 <br />  <br />  doi:10.1371/journal.pone.0042769.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                5                          August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                               Mycobacterial GlmM <br />  <br />  <br />  <br />  <br /> Figure 5. Southern blot PCR analyses M. smegmatis LS2 strains. The genomic DNA digested overnight SmaI enzyme. The <br /> resulting DNA fragments separated 0.8% agarose gel. The DNA transferred Hybond-N+ membrane hybridized by <br /> MSMEG_1556 probe. A. The expected DNA fragments hybridized MSMEG_1556 probe 2.12 kb 2.55 kb. The expected size PCR product <br /> from M. smegmatis LS2 strain 3.15 kb. B. The expected DNA fragments hybridized MSMEG_1556 probe 3.37 kb expected size of <br /> PCR product M. smegmatis mc2155 strain 1.88 kb. C. Confirmation M. smegmatis LS2 strains Southern blot. Lanes 1&#8211;5. M. smegmatis <br /> LS2 strains hybridized DNA bands 2.12 kb 2.55 kb; lane 6. wild type M. smegmatis mc2155 hybridized DNA band 3.37 kb. D. <br /> Confirmation M. smegmatis LS2 strains PCR. Lanes 1&#8211;5. PCR product (3.15 kb) M. smegmatis LS2 strains; lane 6. PCR product <br /> (1.88 kb) wild type M. smegmatis mc2155. <br /> doi:10.1371/journal.pone.0042769.g005 <br />  <br /> The cells observed using JSM-6360 15 kV                Nova-Pak C18 retention time 9.1 min, is <br /> accelerating voltage.                                                       consistent CoA-SH standard (data shown). <br />  <br /> Results                                                                     Conditional M. smegmatis MSMEG_1556 gene knockout <br />                                                                             strain LS2 constructed <br /> Msm MSMEG_1556 Mtb Rv3441c homologous                                 Once conditional replication plasmid pLS5 electroporated <br /> to Ec GlmM                                                                  M. smegmatis mc2155, MSMEG_1556::kanR pLS5 made <br />    The bioinformatics analysis data showed Msm                         single crossover upstream downstream MSMEG_1556 <br /> MSMEG_1556 protein 38% identical Ec GlmM                         locus M. smegmatis genome colonies grown 42uC, resulting <br /> Mtb Rv3441c protein 41% identical Ec GlmM. Msm                       M. smegmatis LS1 mutants integrated the <br /> MSMEG_1556 protein Mtb Rv3441c protein 79%                          MSMEG_1556::kanR MSMEG_1556 locus the <br /> identities (Fig. 3). The results demostrated Rv3441c            homologous recombination. M. smegmatis LS1 mutants were <br /> ortholog GlmM M. tuberculosis MSMEG_1556                   selected Southern blot (data shown). The results suggested <br /> ortholog GlmM M. smegmatis.                                           MSMEG_1556 essential gene. When rescue plasmid <br />                                                                             pLS7 electroporated M. smegmatis LS1 mutant, colonies <br /> Soluble Ec GlmM protein, Msm MSMEG_1556 protein                         grown LB agar plate containing Kan, Str Sucrose 30uC <br /> Mtb Rv3441c protein overexpressed                                      undergone second homologous recombination, resulting <br />    Soluble Ec GlmM protein, Msm MSMEG_1556 protein                      MSMEG_1556 gene knockout. Five M. smegmatis LS2 <br /> Mtb Rv3441c protein overexpressed BL21(DE3) respec-                 (MSMEG_1556 gene knockout) strains selected Southern <br /> tively purified Ni-NTA affinity chromatography. The                  blot. All M. smegmatis LS2 strains showed expected DNA <br /> purified proteins analyzed SDS-PAGE Western blot.               fragments 2.12 kb 2.55 kb (Fig. 5A, 5C), wild type <br /> The results showed purified Ec GlmM protein, Msm                       M. smegmatis mc2155 showed expected DNA fragment of <br /> MSMEG_1556 protein Mtb Rv3441c protein                          3.38 kb (Fig. 5B, 5C). The PCR result indicated M. <br /> expected molecular weight 48.99 kD, 47.99 kD,                        smegmatis LS2 strains expected PCR product 3.15 kb, <br /> 47.31 kD, respectively (Fig. 4).                                            wild type M. smegmatis mc2155 showed expected PCR <br />                                                                             product 1.88 kb (Fig. 5D). <br /> Msm MSMEG_1556 protein Mtb Rv3441c protein <br />                                                                             MSMEG_1556 gene essential mycobacterial <br /> had phosphoglucosamine mutase activity <br />    The specific enzyme activity Ec GlmM,             growth <br /> positive control, 156 nmol?min21?mg21. The specific enzyme                Growth curves M. smegmatis LS2 M. smegmatis mc2155 <br /> activity Msm MSMEG_1556 protein Mtb Rv3441c protein                  carrying pCG76 30uC 42uC obtained determine <br />                                                                             essentiality MSMEG_1556 growth mycobacteria. <br /> were 145 151 nmol?min21?mg21, respectively. The results <br />                                                                             The results showed M. smegmatis LS2 able grow at <br /> demonstrated Msm MSMEG_1556 protein Mtb <br />                                                                             30uC 42uC, M. smegmatis mc2155 carrying <br /> Rv3441c protein phosphoglucosamine mutase activity same <br />                                                                             pCG76 able grow 30uC 42uC (Fig. 6A), which <br /> as Ec GlmM protein (Table 2). In reaction catalyzed Mtb <br />                                                                             consistent growth curve wild type M. smegmatis <br /> Rv3441c protein, product CoA-SH detected by <br />                                                                             mc2155 (data shown). To measure CFU, M. smegmatis LS2 was <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              6                           August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                                 Mycobacterial GlmM <br />  <br />  <br />  <br />                                                                               bars M. smegmatis LS2 30uC; White bars M. smegmatis LS2 <br />                                                                               grown 30uC 24 h grown 42uC. <br />                                                                               doi:10.1371/journal.pone.0042769.g006 <br />  <br />                                                                               grown 30uC 0.10 OD600, culture switched <br />                                                                               incubator 42uC. The OD600 CFU time were <br />                                                                               obtained shown (Fig. 6B, 6C). Since Rv3441c protein was <br />                                                                               produced M. smegmatis LS2 cells 30uC, cells multiplied for <br />                                                                               24 hours temperature shift. Then dramatically, <br />                                                                               increased incubation time 42uC, CFUs dropped the <br />                                                                               Rv3441c protein decreased. These results demonstrat- <br />                                                                               ed MSMEG_1556 gene essential M. smegmatis. <br />  <br />                                                                               Rv3441c protein detected M. smegmatis LS2 <br />                                                                               strain <br />                                                                                  Western blot used analyze expression levels of <br />                                                                               Rv3441c protein. (Anti)-MSMEG_1556 antibody performed a <br />                                                                               cross reaction Rv3441c protein (Fig. 7A), demonstrated that <br />                                                                               (anti)-MSMEG_1556 antibody used detect the <br />                                                                               Rv3441c protein M. smegmatis LS2 strain. Western blot results <br />                                                                               showed M. smegmatis LS2 grown 42uC, the <br />                                                                               expression Rv3441c protein detected compared the <br />                                                                               Rv3441c protein M. smegmatis LS2 grown 30uC (Fig. 7B). The <br />                                                                               MSMEG_1556 wild type M. smegmatis detected. The <br />                                                                               data indicated lacking Rv3441c protein M. smegmatis LS2 <br />                                                                               effect bacterial growth 42uC. <br />  <br />                                                                               M. smegmatis LS2 grown 42uC morphological <br />                                                                               change <br />                                                                                  The morphology M. smegmatis mc2155, M. smegmatis LS2 <br />                                                                               grown 30uC M. smegmatis LS2 grown 42uC observed <br />                                                                               using SEM. The results showed cellular shape size <br />                                                                               M. smegmatis LS2 grown 42uC (switched 30uC) changed <br />                                                                               compared kept growing 30uC wild type M. <br />                                                                               smegmatis mc2155. As incubation time extended, M. smegmatis <br />                                                                               LS2 strain 42uC longer shapes, &#8216;&#8216;bulb&#8217;&#8217; heads, rougher <br />                                                                               cell surface lysis (Fig. 8D, 8E, 8F). Whereas, shape <br />                                                                               size M. smegmatis LS2 strain kept growing 30uC (Fig. 8A, <br />                                                                               8B, 8C) similar wild type M. smegmatis mc2155 <br />                                                                               (Fig. 8G). <br />  <br />                                                                               Discussion <br />                                                                                  UDP-GlcNAc essential common precursor bacterial cell <br />                                                                               wall PG outer membrane lipopolysaccharide, as <br />                                                                               important synthesis enterobacterial common antigen <br />                                                                               [13,14]. The pathway UDP-GlcNAc synthesis prokaryotes is <br /> Figure 6. Growth curves CFU M. smegmatis LS2 strain. A.                somewhat different eukaryotes [17]. Three enzymes, <br /> Growth curves M. smegmatis LS2 M. smegmatis mc2155 carrying            glutamine fructose-6-phosphate transferase (GlmS), phosphoglu- <br /> pCG76 plasmid 30uC 42uC. The cultures grown LB                 cosamine mutase (GlmM) glucosamine-1-phosphate acetyl <br /> medium 30uC 42uC, growth monitored measuring                transferase/N-Acetylglucosamine-1-phosphate urididyl transferase <br /> the absorbance cultures 600 nm. M. smegmatis mc2155 strains         (GlmU), catalyzed formation UDP-GlcNAc E. coli have <br /> carrying pCG76 plasmid similar growth patterns 30uC 42uC. <br /> M. smegmatis LS2 able grow 30uC 42uC. (e) M.        identified [13,15,16,17,18]. GlmM catalyzes conversion <br /> smegmatis LS2 30uC; (X) M. smegmatis LS2 42uC; (#) M.                   GlcNH2-6-P GlcNH2-1-P. The homologs GlmM also <br />                                              N <br /> smegmatis mc2155 carrying pCG76 30uC; ( ) M. smegmatis mc2155 <br /> carrying pCG76 42uC. B. Growth curves M. smegmatis LS2 after <br />                                                                               Gram-negative bacteria e.g. H. influenzae, H. pylori, <br />                                                                               P. aeruginosa [27,28,29] Gram-positive bacteria e.g. S. gordonii <br /> shifting temperature 30uC 42uC. M. smegmatis LS2 grown         S. aureus [30,31]. The glmM gene essential cell growth <br /> 30uC 0.10 OD600, transferred 42uC incubator. The             Gram-negative bacteria (e.g. E. coli). The inactivation glmM <br /> culture grown 30uC control. The OD600 determined the <br /> interval 24 hours. X-<span id='am-73' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-74' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77'  ></span>axis</span> showed time point temperature          gene E. coli followed various alterations cell shape and <br /> transferred. (&amp;) M. smegmatis LS2 grown 30uC 24 h,        finally cells lysed [13]. This probably the <br /> grown 42uC; (%) M. smegmatis LS2 kept 30uC control. C. CFU        progressive depletion precursors PG lipopolysaccharide <br /> of M. smegmatis LS2 30uC 42uC. CFU determinations taken           synthesis. However, mutation glmM gene reduced the <br /> at time points (B). Dilutions spread LB agar plates       growth rate increase cell autolysis Gram-postive bacteria <br /> and plates incubated 30uC CFU counted. Black <br />                                                                               [30,32]. Furthermore, glmM mutation reduced the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                7                           August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                                     Mycobacterial GlmM <br />  <br />  <br />                                                                                mutase catalyze second step synthesis E. coli <br />                                                                                UDP-GlcNAc [13], Msm MSMEG_1556 protein Mtb <br />                                                                                Rv3441c protein significant homology Ec GlmM. <br />                                                                                    To detect phosphoglucosamine mutase activity Msm <br />                                                                                MSMEG_1556 protein Mtb Rv3441c protein, required to <br />                                                                                acquire soluble protein. Mtb Rv3441c protein produced by <br />                                                                                using pET16b vector, unfortunately, Mtb Rv3441c protein was <br />                                                                                insoluble (data shown). To avoid formation inclusion <br />                                                                                bodies, cold-shock expression vector pCold II Rv3441c <br />                                                                                gene constructed. This vector designed perform <br /> Figure 7. Western blot analysis proteins M. smegmatis                  efficient protein expression utilizing promoter derived cspA <br /> mc2155, M. smegmatis LS2 306C M. smegmatis LS2                       gene, cold-shock gene. When incubation <br /> 426C. A. The purified Mtb Rv3441c protein detected using                temperature E. coli host cells reduced sufficiently, the <br /> (anti)-MSMEG_1556 antibody. The result showed (anti)-MSMEG_1556                growth temporarily halted protein expression <br /> antibody cross reaction Rv3441c. M. PageRuler prestained <br /> protein ladde; lane 1. purified Mtb Rv3441c protein. B. Rv3441c            decrease, expression group proteins called &#8216;&#8216;cold- <br /> protein M. smegmatis LS2 grown 42uC 30uC MSMEG_1556              shock proteins&#8217;&#8217; specifically induced. A significant overpro- <br /> protein M. smegmatis mc2155 strain analyzed using (anti)-           duction soluble Mtb Rv3441c protein E. coli BL21(DE3) was <br /> MSMEG_1556 antibody. M. PageRuler prestained protein ladder; lane 1.           observed. The Msm MSMEG_1556 protein produced by <br /> the Rv3441c protein M. smegmatis LS2 grown 42uC                  using protocol. <br /> detectable; lane 2. Rv3441c protein M. smegmatis LS2 grown at <br /> 30uC detectable; lane 3. MSMEG_1556 protein M. smegmatis                In study, set enzyme assay detection of <br /> mc2155 strain detectable.                                                  phosphoglucosamine mutase activity. DNTB chromogenic meth- <br /> doi:10.1371/journal.pone.0042769.g007                                          od convenient time saving, facilitates high- <br />                                                                                throughput inhibitor screening, compared previous methods <br /> formation bacterial biofilm increased sensitivity                    autoradiography HPLC [13,17]. <br /> penicillins [30,32].                                                               The activity Msm MSMEG_1556 protein Mtb <br />    In mycobacteria, UDP-GlcNAc important sugar donor                 Rv3441c protein dependent presence Mg2+. Glc-1,6- <br /> both formation dissarchride linker biosynthesis                      diP required activity. The purified proteins exhibited <br /> peptidoglycan mycobacterial cell wall, therefore, lacking                   little activity Glc-1,6-diP, however, phosphoglucosamine <br /> UDP-GlcNAc effect structural integration                  mutase activity remarkably enhanced presence this <br /> mycobacterial cell wall cell morphology. Ec               compound. The phosphorylated phosphoglucosamine mutase was <br /> GlmM characterized phosphoglucosamine                         assumed active [13]. It unclear Glc-1,6-diP a <br />  <br />  <br />  <br />  <br /> Figure 8. Scanning electron micrographs M. smegmatis LS2 strain shifting temperature 306C 426C. M. smegmatis LS2 <br /> grown 30uC switched 42uC OD600 reached 0.01, M. smegmatis LS2 strain kept growing 30uC control. The cells were <br /> harvested temperature switched 72 h, 120 h 192 h, respectively. The micrographs showed shape size M. smegmatis LS2 <br /> strain kept growing 30uC (A, B, C) similar wild type strains, M. smegmatis LS2 strain 42uC longer shapes, <br /> &#8216;&#8216;bulb&#8217;&#8217; heads, rougher cell surface lysis (D, E, F). A, B, C. M. smegmatis LS2 grown 30uC 72, 120, 192 h (100006); D, E, F. M. smegmatis <br /> LS2 grown 42uC 72, 120 192 h (100006). H, I. M. smegmatis LS2 grown 42uC 120 192 h (200006); G. Wild type M. smegmatis <br /> mc2155 (100006). <br /> doi:10.1371/journal.pone.0042769.g008 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 8                             August 2012 | Volume 7 | Issue 8 | e42769 <br />                                                                                                                                                           Mycobacterial GlmM <br />  <br />  <br /> phosphorylating agent activator. We attempt co-crystallize                              structure cell wall changed, resulting cell death. Therefore, <br /> Glc-1,6-diP Mtb Rv3441c protein reveal                                      GlmM potential target development anti-tuberculosis <br /> mechanism follow-up research.                                                              drugs. <br />    Mtb Rv3441c gene (annotated mrsA TubercuList Server) <br /> has proved essential growth cells using                              Supporting Information <br /> Himar1-based transposon site hybridization (TraSH) methodology <br /> [33]. To assess effect mutated glmM gene cell growth,                               Table S1 Bacterial strains plasmids used study. <br /> morphology, cell wall structure, etc., used model mycobac-                               (DOC) <br /> terial strain, M. smegmatis, construct conditional MSMEG_1556 <br /> gene knockout strain LS2 inserting kanR cassette. M. smegmatis                             Acknowledgments <br /> LS2 unable grow 42uC (non-permissive temperature) <br />                                                                                               We thank Mycobacteria Research Laboratories, Department Microbi- <br /> since rescue plasmid carrying Mtb Rv3441c gene                                  ology, Immunology, Pathology, Colorado State University, for <br /> replicate. It demonstrated MSMEG_1556 gene essential                              providing M. tuberculosis H37Rv genomic DNA. We thank <br /> for growth M. smegmatis. Furthermore, GlmM                               reviewers suggesting experiments express E. coli GlmM a <br /> protein expressed M. smegmatis LS2 strain. To observe                              control, measure CFU detect M. tuberculosis GlmM LS2 strain. <br /> morphological change LS2 strain, LS2 cells harvested <br /> after shifting temperature 30uC (permissive temperature)                              Author Contributions <br /> 42uC. The morphological analysis LS2 SEM revealed that <br /> the LS2 cells longer rougher surface compared                                 Conceived designed experiments: YM SL JK. Performed the <br />                                                                                               experiments: SL JK WY YZ WZ YX. Analyzed data: SL JK YZ YX <br /> the wild type cells. With increased incubation time 42uC, <br />                                                                                               YM. Contributed reagents/materials/analysis tools: YX YM. Wrote the <br /> many cells fused lysed eventually. These results suggested                           paper: YM SL JK. <br /> lacking GlmM block PG synthesis make the <br />  <br /> References <br />  1. Published World Health Organization (WHO) 2006 Tuberculosis Facts <br /> </body></html>