<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Feb. 1991, p. 620-631                                                                   Vol. 11, No. 2 <br /> 0270-7306/91/020620-12$02.00/0 <br /> Copyright &#169; 1991, American Society Microbiology <br />  <br />  <br />    Cloning Characterization ERG8, Essential Gene of <br /> <span id='am-16' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-32'  ></span>Saccharomyces cerevisiae</span> That Encodes Phosphomevalonate Kinase <br />                                    YIM H. TSAY AND GORDON W. ROBINSON* <br />  Department Cellular Biology, Bristol-Myers Squibb Pharmaceutical Research Institute, Princeton, New Jersey 08543 <br />                                             Received 8 August 1990/Accepted 30 October 1990 <br />  <br />              <span id='am-33' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae strains contain erg8-1 mutation temperature sensitive growth due <br />            defect phosphomevalonate kinase, enzyme isoprene ergosterol biosynthesis. A plasmid bearing <br />            yeast ERG8 gene isolated YCp5O genomic library functional complementation erg8-1 <br />            mutant strain. Genetic analysis demonstrated integrated copies ERG8 plasmid mapped erg8 <br />            locus, confirming identity clone. Southern analysis showed ERG8 single-copy gene. <br />            Subcloning DNA sequencing defined functional ERG8 regulon 850-bp upstream region an <br />            adjacent 1,272-bp open reading frame. The deduced 424-amino-acid ERG8 protein showed homology to <br />            known proteins putative ATP-binding domain present kinases. Disruption the <br />            chromosomal ERG8 coding region integration URA3 HIS3 marker fragments lethal haploid <br />            cells, indicating gene essential. Expression ERG8 gene S. cerevisiae galactose- <br />            inducible galactokinase (GALI) promoter resulted 1,000-fold-elevated levels phosphomevalonate kinase <br />            enzyme activity. Overproduction soluble protein predicted 48-kDa size phosphomevalonate <br />            kinase observed yeast cells. <br />  <br />   Phosphomevalonate kinase [ATP:(R)-5-phosphomeval-                     erg8 strains cosegregated, suggesting traits were <br /> onate phosphotransferase; EC 2.7.4.2] catalyzes pyro-               result single genetic defect. erg8 strains were <br /> phosphorylation mevalonate second steps              recently used explore regulation ergosterol metab- <br /> leading formation isopentenyl diphosphate: (3R)-              olism S. cerevisiae (16, 47), detailed characteriza- <br /> phosphomevalonate        +    ATP   -- <br />                                          (3R)-pyrophosphomeval-         tion yeast gene phosphomevalonate kinase has <br /> onate + ADP. Although identified cytoplasmic                 reported. <br /> enzyme    Saccharomyces cerevisiae, phosphomevalonate                   In mammalian cells, synthesis sterols isoprenoid <br /> kinase isolated variety plant animal             compounds regulated primarily control key <br /> sources (10, 24, 33, 50). <span id='am-48' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> enzyme functions                  enzymes mevalonate formation, HMG coenzyme A (CoA) <br /> elaboration isoprene subunits, used                synthase HMG CoA reductase (15). The analogous yeast <br /> synthesis variety essential compounds including                 biosynthetic pathway regulated steps <br /> sterols, dolichols, ubiquinones. Isoprene-derived mole-             (12, 47, 53). Some evidence suggests yeast and <br /> cules used <span id='am-64' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> covalent modifi-            mammalian cells, significant pathway regulation be <br /> cation tRNAs (17) specific proteins, including ras           mediated additional enzymes sterol biosynthesis, <br /> a-factor S. cerevisiae (1, 27, 46). Phosphomevalonate                including mevalonate kinase squalene synthetase (18, <br /> kinase pig liver purified homogeneity              20, 52). To investigate potential secondary <br /> shown monomeric protein molecular mass                control points pathway, initiated molecular <br /> 22 kDa (6, 32). It exhibits strong nucleotide specificity           analysis S. cerevisiae enzymes early <br /> ATP utilizes Mg2` preferred cofactor. Chemical                 sterol biosynthesis. In report cloning of <br /> modification studies suggested enzyme re-                yeast ERG8 gene phosphomevalonate kinase using <br /> quires cysteine lysine residues activity (7,          erg8-1 mutant complementation approach char- <br /> 54), little known structure enzyme           acterization gene DNA sequencing other <br /> its active site.                                                        analyses. We ERG8 single-copy essential <br />    A large number temperature-conditional S. cerevisiae              yeast gene encodes protein twice size the <br /> mutants defective ergosterol biosynthesis isolated           previously reported mammalian enzyme. <br /> Karst et. al (29, 48). These erg mutants required ergosterol <br /> supplementation growth 25&#176;C inviable 36&#176;C <br /> even supplementation. Mutant strains designated erg8-                            MATERIALS AND METHODS <br /> 1, -2, -3 defective phosphomevalo-                 Materials. D-Sorbitol, P-glucuronidase, D-galactose, <br /> nate kinase labeling experiments (29). Whole cells failed            Tween 80, polyethylene glycol 4000 obtained from <br /> to synthesize ergosterol acetate, extracts erg8 <br /> cells unable convert mevalonate mevalonate-                  Sigma Chemical Co. Glass milk (GeneClean) from <br /> pyrophosphate vitro. Genetic analysis showed erg8               BiolOl, LaJolla, Calif., BioTrans membranes from <br /> mutations recessive wild-type ERG8 allele                ICN Biomedicals, Inc., Irvine, Calif. Sequenase (modified <br /> heterozygous diploids. In addition, experiments dem-              T7 DNA polymerase) U.S. Biochemical Corp., <br /> onstrated thermosensitive biochemical defects              Cleveland, Ohio. [a-32P]dCTP, [a-35S]dATP, 14C- <br />                                                                         labeled sterol precursors Amersham Corp. 5-Flu- <br />                                                                         oroorotic acid PCR, Inc., Gainsville, Fla., and <br />                                                                         silica gel LK6D thin-layer chromatography plates from <br />   * <br />       Corresponding author.                                             Whatman. 5-Bromo-4-chloro-3-indolyl-,-D-galactopyrano- <br />                                                                   620 <br />  VOL. 11, 1991                                     CLONING OF S. CEREVISIAE PHOSPHOMEVALONATE KINASE                             621 <br />  <br />                                                 TABLE 1. S. cerevisiae strains <br />   Strain                                               Genotype                                                        Source <br /> SPi                       MATa erg8-1                                                                              F. Karst <br /> AG8-6B                    MATa erg8-1 his3-11,15 leu2-3,112 trpl-l                                                 This study <br /> AG8-2A                    MATa erg8-1 his3-11,15 ura3-1 leu2-3,112 ade2-1                                          This study <br /> ET7-6B                    MATa erg8-1 upcl his3-11,15 leu2-3,112 ural                                              This study <br /> ET15-6D                   MATa erg8-1 upcl his3-11,15 leu2-3,112 trpl-l ura3                                       This study <br /> ET20-2A                   MATa erg8-1 ura3-52 lys2-801                                                             This study <br /> W303-1A                   MATa ERG8 ura3-1 his3-11,15 leu2-3,112 trpl-l ade2-1 can1-100                            R. Rothstein <br /> W303-1B                   MATa ERG8 ura3-1 his3-11,15 leu2-3,112 trpl-l ade2-1 canl-100                            R. Rothstein <br /> JT35-1A                   MATa ERG8 upcl ura3-1 his3-11,15 trpl-J ade2-1                                           J. Tkacz <br /> JT35-6B                   MATa ERG8 upcl ural his3 leu2-3,112 trpl-J                                               J. Tkacz <br /> JT42-5B                   MATa ERG8 ura3-1 LYS2                                                                    J. Tkacz <br /> Y294                      MATa ERG8 his3d ura3-52 leu2-3,112 trpl-J GAL                                            M. Haffey <br /> YM197                     MATa/MATa ERG81ERG8 ura3-521ura3-52 Iys2-801/lys2-801 ade2-101/ade2                      J. Rine <br /> ET16                      MATa/MATa ERG81erg8-1 ura3-l/ura3-1 his3-J1,lS/his3-11,15 ade2-llade2-1                  This study <br />                             leu2-3,1121leu2-3,112 TRPl/trpl-l <br />  <br />  <br />  <br /> side Bethesda Research Labs, Inc., Gaithersburg,        yeast transformants rapid glass bead disruption and <br /> Md. Reagents spectrophotometric assay <span id='am-151' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-152' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154'  ></span><span id='am-155' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>-           phenol extraction (39). Yeast chromosomal DNA pre- <br /> phomevalonate kinase obtained Boehringer           pared method Struhl et al. (51). Large-scale <br /> Mannheim Biochemicals, Indianapolis, Ind., R,S-       preparations minipreparations plasmid DNA were <br /> 5-phosphomevalonate, kindly provided L.             prepared E. coli boiling method (26). Large <br /> Parker, Bristol-Myers Squibb Pharmaceutical Research In-         preparations purified centrifugation ce- <br /> stitute, Princeton, N.J.                                         <span id='am-110' about='obo:NCBITaxon_48052' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_241787 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Sium" datatype="xsd:string"></span><span id='am-114'  ></span>sium</span> chloride-ethidium bromide gradients. <br />    Strains genomic libraries. The S. cerevisiae strains         DNA manipulations. Restriction enzymes DNA <br /> used study described Table 1         modification enzymes purchased Bethesda Re- <br /> derivatives strain S288C. The <span id='am-67' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> strains        search Labs, Inc., New England BioLabs, Inc., and <br /> used cloning plasmid propagation RR1 [F- pro        used according specifications manufacturers. <br /> leu thi lacY Strf hsdR (rK- MK-) hsdM endol], DH5a [F-           DNA restriction fragments cloning purified from <br /> supE44 thi-J recAl gyrA96 relAl hsdR17 (rK- MK+) endAl           agarose gels glass milk-sodium iodide procedure (55). <br /> j80dlacZAM15 A(lacZYA-argF)U169, JM101 [supE thi             Radiolabeled ERG8 probes Southern analysis pre- <br /> recA+ rK+ A (lac-proA,B)/F&apos; traD36 proA,B+ laclqZAM15].          pared random oligomer priming (19) purified restriction <br /> The libraries genomic DNA used work          fragments nick translation kit Boehringer Mann- <br /> the Nasmyth Tatchell library (38), containing yeast          heim. Unincorporated [32P]dCTP removed the <br /> DNA inserts high-copy-number vector YEp13 (ob-            labeled probe spin columns (Boehringer Mannheim). <br /> tained ATCC), Siede Eckardt-Schupp             Plasmid constructions. <span id='am-146' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span> used localization of <br /> library (49), containing yeast DNA inserts low-copy-      ERG8 gene constructed pET45 insertion of <br /> number vector YCp50 (provided W. Siede).                      restriction fragments (see Fig. 3) compatible sites the <br />    Genetic methods media. Yeast cells grown at <br /> 28&#176;C unless indicated. Standard procedures        tetracycline resistance region YCp5O. Overhanging 5&apos; and <br /> used mating, sporulation, tetrad analysis (37). Auxo-    3&apos; ends restriction fragments converted to <br /> trophic markers introduced erg8-1 mutant           blunt ends using respective polymerase 3&apos;-exonu- <br /> strain (SP1) cross strain W303-1A. Yeast complete      clease functions T4 DNA polymerase. Plasmid pET58 <br /> medium (YC) yeast minimal medium (YM)                contained SalI-BamHI fragment pET53 (see Fig. <br /> previously described (40). YM supplemented nu-          3), cloned multicopy vector <br /> trients appropriate selective growth strains. YME      YEp352. Plasmid pET48 cloning 4.5-kb <br /> medium prepared supplementation YM 20 ,ug         BamHI fragment pET45 pBR322. A 2.1-kb XhoI- <br /> of ergosterol ml fresh ergosterol stock solution      BamHI fragment derived pET48 used ERG8 <br /> prepared Tween 80-ethanol (1:1, vol/vol); YCE medium          probe Southern analysis. <br /> was YC comparably supplemented. Where indicated, 2%                 Plasmids pET61 pET62, used production of <br /> galactose substituted 2% glucose carbon           respective URA+ HIS&apos; disruptions chromosomal <br /> source YM. Growth S. cerevisiae induction       ERG8 gene, constructed pET48 replacement <br /> GAL) promoter described previously (57). E. coli          1.0-kb XhoI-EcoRV internal fragment ERG8 gene <br /> strains grown 37&#176;C LB medium (35) supple-             appropriate prototrophic marker fragments (see Fig. <br /> mented 50 ,ug ampicillin ml select plasmid    6A). Plasmid pET61 constructed three-piece ligation <br /> transformants.                                                   2.0-kb SphI-SmaI URA3 fragment Ylp5 (51), a <br />   Transformations. All yeast strains transformed         3.7-kb SphI-PstI fragment pET48 contained the <br /> the spheroplast method (25). erg8 erg8 upcl mutant           ERG8 5&apos; end, 4.2-kb PstI-EcoRV fragment from <br /> strains grown overnight YCE 25&#176;C sphero-       pET48 contained ERG8 3&apos; end. Plasmid pET62 was <br /> plasting, transformants harvested YME          constructed analogous three-piece ligation 2.1-kb <br /> regeneration agar 6 days incubation 25&#176;C. E. coli    SphI-EcoRV HIS3 fragment YEp6 (13) two <br /> strains transformed CaCl2 method (41).               pET48 fragments described above. Linear, disrupted ERG8 <br />   Preparation DNA. Crude plasmid DNA prepared             fragments transformation diploid yeast cells were <br />  622    TSAY AND ROBINSON                                                                                     MOL. CELL. BIOL. <br />  <br /> obtained plasmids pET61 pET62 digestion        counted determine extent 14C incorporation into <br /> PvuII XbaIL                                                 organic solvent-soluble products. Thin-layer chromatogra- <br />    Plasmid pET77, used overproduction phospho-       phy used examine various products vitro <br /> mevalonate kinase, constructed insertion 2.4-kb     reactions. After incubation S4K fractions 14C- <br /> XhoI-SalI ERG8 fragment pET48 SalI site        labeled substrates described above, aqueous reaction mix- <br /> pMH101 (21). The ERG8 coding region properly ori-           tures fractionated silica gel LK6D thin-layer chro- <br /> ented respect GAL) promoter plasmid (see    matography plates n-propanol-NH40H-water (6:3:1, <br /> Fig. 7A).                                                       vol/vol/vol) solvent system. The dried plates were <br />    DNA sequence analysis. The BgIII-XbaI fragment          examined autoradiography. <br /> pET45 sequenced dideoxy-chain termination              Overexpression phosphomevalonate kinase esti- <br /> method Sanger et al. (45) (see Fig. 3B). DNA restriction     mated using enzyme assays. The sensitive was <br /> fragments subcloned M13mpl8 M13mpl9                vitro complementation assay, utilized [14C] <br /> primed chain extension 17-bp universal primer,     phosphomevalonate measured conversion this <br /> and [35S]dATP incorporated Sequenase recom-         substrate squalene described above. Small amounts <br /> mended manufacturer. Sequences compiled         (ranging 0.001 0.01 ml) S4K fraction putative <br /> analyzed Intelligenetics software programs. The        overproducing cells added reaction mixtures that <br /> EMBL/GenBank PIR <span id='am-131' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-135'  ></span>data</span> bases searched               contained large (0.15 ml) S4K fraction erg8 <br /> homologies DNA protein sequences phospho-         mutant cells. Reactions terminated various times, <br /> mevalonate kinase 5&apos; open reading frame (ORF)           conversion rates determined; S4K fraction from <br /> peptide.                                                        untransformed wild-type (ERG8) yeast strain used to <br />    Southern analysis. Southern blot hybridization analysis      calibrate assay plasmid-transformed strains. A spec- <br /> was performed yeast chromosomal DNA described             trophotometric assay phosphomevalonate kinase was <br /> previously (40). Genomic DNA digested completion         used measure enzyme overproduction cells bearing <br /> with EcoRI, restriction fragments separated       pET77 pMH101 plasmids. Conditions assay were <br /> 0.8% agarose gel. The gel blotted BioTrans mem-        previously described (42), 9 mM R,S-phosphome- <br /> brane, hybridized 32P-labeled ERG8 probe       valonate used substrate. The generation ADP <br /> and autoradiographed.                                           ATP phosphomevalonate kinase reaction was <br />    Preparation crude extracts. Enzyme extracts pre-     coupled reactions catalyzed pyruvate kinase and <br /> pared late-logarithmic-phase yeast cells centrifuga-    lactate dehydrogenase, reaction rate monitored <br /> tion suspension cells pellet weight (gram)        340 nm double-beam spectrophotometer. <br /> equivalent 4&#176;C breakage buffer (100 mM Tris hydrochlo-          Determination crude ergosterol content. Nonsaponifiable <br /> ride [pH 7.5]-10 mM P-mercaptoethanol). Two pellet weight       lipids prepared, crude ergosterol content of <br /> equivalents acid-washed glass beads (diameter, 0.45 ,um)     yeast cells determined method Karst et. al (47). <br /> were added, cells disrupted vigorous mixing        Gel electrophoresis yeast proteins. Cultures yeast <br /> on Vortex Genie (three 45-s pulses). The disrupted cell       strain Y294 contained ERG8 overexpression plasmid <br /> suspension centrifuged 2,400 x g 10 min,       pET77 control plasmid pMH101 (21) grown in <br /> which supernatant removed centrifuged 4,000      YM-glucose YM-galactose 18 h. Cells were <br />  x g 15 min. The 4,000 x g supernatant (S4K) fraction   harvested, S100K fractions prepared described <br /> used directly 14C-labeled substrate conversion assay     above. Aliquots contained 20 jig protein mixed <br /> or centrifuged 100,000 x g 60 min obtain        Laemmli sample buffer (30), boiled 3 min, electro- <br /> soluble S100K fraction, used spectrophoto-     phoresed 12% sodium dodecyl sulfate-polyacrylamide <br /> metric assay phosphomevalonate kinase. Protein concen-      gels (Integrated Separation Systems) described Laem- <br /> trations S4K Sl10K fractions determined      mli (30), stained Coomassie blue stain. Band <br /> method Bradford (14). Enzyme extracts frozen        intensities stained gels quantified densitometric <br /> stored -70&#176;C assay negligible losses          scanning. <br /> phosphomevalonate kinase activity. <br />    Enzyme assays phosphomevalonate kinase. Comple-                                    RESULTS <br /> mentation erg8 mutant strain assessed biochemi- <br /> cally measuring ability S4K fractions convert         Biochemical genetic analyses erg8-1 mutants. Two <br />  [14Ciphosphomevalonate sterol precursors, predomi-        temperature-sensitive haploid strains, AG8-2A AG8-6B <br /> nantly squalene. This assay functions uncomple-         (Table 1), derived Karst erg8-1 mutant strain SP1 <br /> mented erg8 mutants contain enzymes necessary        characterized biochemically attempts clone <br /> for conversion, phosphomevalonate kinase. Re-       ERG8 gene initiated. Cell extracts prepared from <br /> action mixtures contained following total volume    erg8 strains possessed detectable phosphomeval- <br /> 0.2 ml: 5 mM MgCl2, 4 mM ATP, 1 mM NADPH, 10 mM                 onate kinase activity unable convert [&quot;&apos;C] <br />  KF, 100 mM Tris hydrochloride (pH 7.5), 0.1 ,uCi [5-14C]    phosphomevalonate, substrate phosphomevalonate ki- <br /> R-phosphomevalonate (58 mCi/mmol), 0.15 ml           nase, product sterol pathway (Fig. 1 and <br /> S4K fraction. In experiments, equivalent amounts        2). In contrast, extracts ERG8 wild-type yeast strains <br />  [2-14C]R-mevalonate (57 mCi/mmol), [14Cipyrophosphome-         capable efficient vitro conversion substrate <br /> valonate (58 mCi/mmol), [1-14C]isopentenyl disphosphate      products, chiefly squalene (Fig. 1), showing that <br />  (56 mCi/mmol) substituted labeled phosphomeval-       contained sterol enzymes necessary this <br /> onate mixture. Reactions incubated 37&#176;C      conversion. ERG8 erg8 extracts equally capable of <br />  20 min terminated addition 0.8 ml CHCl3-      converting [&quot;&apos;C]pyrophosphomevalonate, product phos- <br />  methanol (2:1, vol/vol). The phases vortexed briefly      phomevalonate kinase, squalene. Similar experiments <br />  separated low-speed centrifugation. The organic         [&quot;&apos;C]mevalonate (Fig. 1) [&quot;&apos;C]isopentenyl diphos- <br />  phase removed, washed twice methanol,             phate (data shown) substrates confirmed the <br />  VOL . 1 l, 1991 <br />  <br />  <br />  <br />  <br />     MEV. <br />  <br />  <br />  <br />  <br />    a- <br />      (A) <br />  <br />  <br />  <br />  <br /> enzyme <br />                erg <br />  <br />  <br />  <br />  <br /> squalene synthesis. <br />  <br />  <br />  <br />  <br />         4) <br />                      + <br />  <br />                      A <br />                       1 2 <br />  <br />  <br />  <br />  <br />                       * <br />  <br />  <br />  <br />  <br />                      12 <br />                            + + <br />  <br />                            B <br />  <br />  <br />                           erg8 <br />  <br />                       YC:p5O <br />                               +} <br />                                     3 4 5 6 7 8 9 <br />                                    00 V <br />  <br />  <br />  <br />  <br />                                    C A <br />                                      I1 <br />  <br />  <br />  <br />  <br />                                    MEV-5-P <br />                                      (B) <br />                                           + <br />  <br />  <br />  <br />                                           .-- <br />  <br />  <br />  <br />  <br />   FIG. 1. Complementation biochemical <br />                                                   B <br />                                                    + <br />  <br />  <br />  <br />  <br />                                                   erg8 <br />                                                    + <br />  <br />                                                   pET45 <br />                                                        + <br />  <br />                                                        C <br />  <br />  <br />  <br />  <br />                                                        erg <br />  <br />  <br />  <br />  <br /> mutant strain. Cell extracts prepared ERG8 (wild-type) <br /> yeast strain YCp5O- pET45-transfo rmed erg8-1 mutant <br /> strain incubated indicated &apos;4C-labe led sterol precursor <br /> (A, B, C) 20 min 37&#176;C. Equal amoun Its total reaction <br /> mixture applied thin-layer chromatogr; <br /> were developed described Materials M <br /> tion standard compounds (A, B, C, E )) plates is <br /> indicated arrows. Abbreviations: MEV., mev; alonate; MEV.-5-P, <br /> phosphomevalonate; MEV.-5-PP, pyrophosphormevalonate. <br />  <br />  <br />  <br />           defect erg8 strains tstricted phos- <br /> phomevalonate kinase affected later steps of <br />   Initial attempts transform erg8 str ains showed that <br />                                                              XI <br />  <br />  <br />  <br />  <br />                                                              8 <br />                                                                   # <br />  <br />  <br />  <br />  <br />                                                                    + <br />  <br />                                                                   A B C <br />                                                                        + <br />  <br />  <br />  <br />  <br />                                                                     ERG8 <br />                                                                            + <br />  <br />  <br />  <br />                                                                            ._ <br />  <br />  <br />  <br />  <br />                                                                        MEV-5-PP- <br />                                                                           (C) <br />                                                                                       ~ <br />                                                                                  -Squaalene <br />  <br />                                                                                 &apos;-MEMf. (A) <br />  <br />  <br />  <br />  <br />                                                                                  MEV. -5-P <br />  <br />                                                                                 -MEV.1-5-PP <br />                                                                                               (D) <br />  <br />  <br />  <br />  <br />                                                                                               (B) <br />                                                                                                (C) <br />                                                                                                     CLONING OF S. CEREVISIAE PHOSPHOMEVALONATE KINASE <br />  <br />  <br />  <br />  <br />                                                                                                     Squaene <br />                                                                                                       (D) <br />                                                                                        defect erg8-1 <br />  <br />  <br />  <br />                                                                                       aepthydslatehse mich <br />                                                                                                               growth YCE poor recipients for <br />                                                                                                               plasmid DNA, yielding 1 2 transformants ,ug of <br />                                                                                                               DNA. Plating erg8 strains YME medium at <br />                                                                                                               temperatures 25&#176;C reduced colony-forming abil- <br />                                                                                                               ity, expected, additionally produced colony size <br />                                                                                                               heterogeneity complicate screening yeast <br />                                                                                                               libraries plasmids capable reversing erg8 temper- <br />                                                                                                               ature-sensitive phenotype. It appeared likely of <br />                                                                                                               strain characteristics function both <br />                                                                                                               impaired synthesis ergosterol inefficient uptake of <br />                                                                                                               ergosterol, upcl mutation (34), known to <br />                                                                                                               enhance uptake ergosterol aerobically grown yeast <br />                                                                                                               cells, crossed erg8 strains. The resulting erg8 <br />                                                                                                               upcl strains, ET7-6B ET15-6D (Table 1), grew <br />                                                                                                               better YME 25&#176;C parent erg8 strains also <br />                                                                                                               formed colonies uniform size temperatures. <br />                                                                                                               Like parent strains, erg8 upcl strains formed fewer <br />                                                                                                               colonies YME elevated temperatures 34&#176;C. <br />                                                                                                               The erg8 upcl strains better recipients for <br />                                                                                                               plasmid DNA erg8 strains, giving 20 50 transfor- <br />                                                                                                               mants ,ug DNA. <br />                                                                                                                  Cloning ERG8 gene functional complementation. <br />                                                                                                               Yeast recombinant DNA libraries screened isolate <br />                                                                                                               candidate ERG8 plasmid clones. The erg8 strain ET7-6B was <br />                                                                                                               transformed leucine prototrophy high-copy- <br />                                                                                                               number Nasmyth Tatchell library (38), strain <br />                                                                                                               ET15-6D transformed uracil prototrophy the <br />                                                                                                               low-copy-number Siede Eckhardt-Schupp library (49). <br />                                                                                                               Approximately 35,000 LEU+ transformants the <br />                                                                                                               Nasmyth Tatchell library equal number URA+ <br />                                                                                                               transformants Siede Eckhardt-Schupp library <br />                                                                                                               separately pooled screening. <br />                                                                                                                  Transformant pools screened ERG8 gene by <br />                                                                                                               spreading cells YME selecting cells capable of <br />                                                                                                               colony formation 2 days 34&#176;C. With LEU+ <br />                                                                                                               transformant pool, 2 5 cells formed colonies every <br />                                                                                                               10,000 cells plated. Extracts prepared colonies <br />                                                                                                               assayed biochemically ability convert <br />                                                                                                               [&apos;4C]mevalonate organic solvent-soluble products (Ma- <br />                                                                                                               terials Methods). None LEU+ candidate cells was <br />                                                                                                               efficient conversion untransformed erg8 <br />                                                                                                               upcl cells, colonies characterized. <br />                                                                                                               With URA+ transformant pool, 5 10 cells formed <br />                                                                                                               colonies 34&#176;C 10,000 cells plated. Cell extracts <br />                                                                                                               URA+ candidates tested biochemically, and <br />                                                                                                               ability convert [14C]mevalonate squalene. <br />                                                                                                                                                                          623 <br />  <br />  <br />  <br />  <br />                                                                                                               efficient wild-type cells <br />                                                                                                                  Plasmid DNA extracted 22 ERG8 candidate colonies <br />    er <br />     C <br />    CO <br />          41) <br />          &gt;                                                                                                    amplified E. coli examined restriction enzyme <br />       OC <br />      0&gt; <br />     Xxo <br />        c <br />                                                                                                               analysis. Three unrelated plasmids identified. Cells that <br />        (,)                                                                                                    carried plasmids partially complemented <br />    a- <br />                                                                                                               erg8 biochemical defect analyzed <br />                                                                                                               further. Cells carried plasmid completely <br />                                                                                                               complemented. This plasmid, designated pET45, con- <br />                                                                                        ,________ <br />                                                                                                ^              tained 4.5-kb insert restriction sites shown Fig. <br />                           2                   5                      10                15      20             3A. To verify contained ERG8 gene, pET45 was <br />                                                                   rime (min.)                                 transformed erg8 strain lacked upcl mutation <br />   FIG. 2. Effect ERG8 plasmid copy number phosphomeval-                                                 strain ET15-6D. The pET45 transformants strain <br /> onate kinase activity. Equivalent, small amounts ,of cell extract                                        AG8-2A viable YM 37&#176;C, extracts these <br /> either untransformed ERG8 strain (-) fr(Dm ERG8 strain                                               cells showed &apos;4C-labeled substrate conversion ability <br /> transformed multicopy ERG8 plasmi4 d pET49 (0) were <br /> mixed excess erg8-1 mutant extract measure enzymatic                                            characteristic wild-type cells (Fig. 1). To test the <br /> complementation. Mixed extracts incubati ed 14C-labeled                                             premise pET45 contained ERG8 structural gene, the <br /> phosphomevalonate various times 37&#176;C, md conversion                                             entire 4.5-kb insert subcloned YEp24 generate <br /> products measured described Mateirials Methods.                                                 high-copy-number plasmid pET49. This plasmid was <br /> Unsupplemented erg8-1 mutant extracts (A) verted &lt;2,000 cpm                                               transformed wild-type strain W303-1A, extracts <br /> into products times incubation.                                                                     prepared pET49 transformants untrans- <br />  624     TSAY AND ROBINSON                                                                                                               MOL. CELL. BIOL. <br />  <br />                     A. <br />                           PLASMID                                                                                      ERG 8 FUNCTION <br />                                       B R          LA   Pa   Bg            A BgXS   <span id='am-125' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Aa</span> SsHXbE              Xb   R Ba <br />                            pET45    S~ I&apos;~ ~   I             A--               _____ <br />                                                                                aIII     I if <br />                                                                                                                 &apos; <br />                              I1 <br />                              pET                                                                            I <br />                                                                                                             0 .SKb&apos; <br />                            pET 51 <br />                            pET 53 <br />                            pET 74                                                                                            + <br />                            pET 52 <br />                            pET 63                                                                                            + <br />                            pET 54 <br />                            pET 64 <br />                      B.                                                           -----1   ----a   ----- <br />  <br />  <br />  <br />  <br />                                                                   W-   ~    -0-             .-~ <br />                                                                                             ~ <br />  <br />  <br />  <br />  <br />    FIG. 3. Restriction map pET45 various deletion plasmids sequencing strategy ERG8 gene. (A) Location restriction <br /> sites 4.5-kb insert pET45. The arrow indicates location direction transcription ERG8 gene. The location of <br /> an incomplete ORF indicated bracket. The phenotypic biochemical complementation ability (ERG8+) various YCp5O-based <br /> deletion plasmids extent ERG8 region DNA present shown. (B) Diagrammatic representation strategy used for <br /> sequencing ERG8 gene flanking regions pET45. Restriction enzyme sites: A, AccI; B, BamHI; Bg, BglII; E, EcoRV; H, HindIII; <br /> L, Sall; P, PvuI; S, SphI; Ss, SspI; R, EcoRI; X, XhoI; Xb, XbaI. <br />  <br /> formed cells. As shown earlier, erg8 extracts contain                      end ERG8 gene situated two <br /> the enzymes squalene synthesis phosphomeval-                            BglII sites pET45. Although pET52 failed complement, <br /> onate kinase, possible complement                        odds finding pET63 fully comple- <br /> deficiency vitro addition small quantities                        mented erg8 mutant pET63 contained a <br /> extract ERG8 cells. Using approach, relative                        smaller pET45 fragment. A possible explanation this <br /> amount phosphomevalonate kinase extracts esti-                          anomaly provided nucleotide sequence the <br /> mated measuring increased rate [14C]phosphome-                          ERG8 gene discussed below. Plasmids pET54 and <br /> valonate conversion (Fig. 2). From mixing assays,                         pET64, terminated XbaI EcoRV sites, respec- <br /> was determined pET49 resulted approximately                            tively, failed complement erg8 mutation, indicating <br /> eightfold overproduction phosphomevalonate kinase.                             end ERG8 gene situated down- <br /> Analogous experiments described sterol                        stream EcoRV site pET45. <br /> biosynthetic genes S. cerevisiae similar gene-dosage                          Nucleotide predicted ERG8 polypeptide sequences. The <br /> effects reported (2, 43).                                               strategy used determination ERG8 DNA <br />    Genetic verification ERG8 clone. To establish                       sequence shown Fig. 3B, complete nucleotide <br /> identity putative ERG8 clone, ability pET45                         sequence shown Fig. 4. The 2,399-bp region sequenced <br /> DNA target integration plasmid homologous                             spanned distal BglII XbaI sites pET45 (Fig. <br /> recombination yeast erg8 locus examined. For                           3A) included 846 bp 5&apos;-flanking sequences 278 bp <br /> this purpose, SalI-BamHI fragment pET45 sub-                             3&apos;-flanking sequences. The presumptive ERG8 coding <br /> cloned YIp5 (Materials Methods), vector                           region starts 3 bp XhoI site continues 1,272 <br /> contains yeast URA3 gene lacks yeast replication                        bp, terminating TAA <span id='am-59' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. The ERG8 ORF capable <br /> origin (51). The resultant plasmid, pET46, cut                         encoding 424-amino-acid polypeptide predicted <br /> ERG8 sequences unique SphI site transformed                         molecular weight 47,328. No TACTAAC-like intron con- <br /> the erg8 strain ET20-2A. Stable URA+ transformants se- <br /> lected 25&#176;C grew 37&#176;C biochemically <br /> ERG8. One URA+ integrant crossed wild-                                 TABLE 2. Genetic analysis ERG8 plasmid integrantsa <br /> type strain JT42-5B diploids, sporulated <br /> and subjected tetrad analysis. In tetrads exam-                          Segregation ERG                   Segregation L YS and <br /> ined, 4:0 segregation ERG81erg8 seen bio-                                  URA phenotypes                           URA phenotypes <br /> chemical level, indicating close linkage ERG8                             Spore           No. spores            Spore           No. spores <br /> gene integrated plasmid erg8 chromosomal                             phenotype        phenotype          phenotype        phenotype <br /> allele (Table 2). These data provided genetic evidence that <br /> pET45 contained ERG8 gene.                                                    ERG+     URA+                 44        LYS+   URA+               25 <br />    Localization ERG8 gene pET45. To localize                        ERG+     URA-                 44        LYS+   URA-               19 <br />                                                                                   ERG-     URA+                  0        LYS-   URA+               19 <br /> erg8-1 complementing region pET45, subclones                              ERG-     URA-                  0        LYS-   URA-               25 <br /> constructed YCp5O using restriction fragments <br /> shown Fig. 3A. These subclones tested                             Plasmid pET46 (ERG8 URA3) integrated erg8 strain ET20-2A <br /> ability complement temperature-sensitive phenotype                         obtain URA + transformant crossed ERG8 strain <br />                                                                                   JT42-5B. The resulting diploids sporulated dissected. Nutritional <br /> and biochemical defect strain AG8-2A. Data obtained                            markers segregated 2+:2- four-spored tetrads, ERG8 (or erg8) <br /> from plasmids pET51, pET52, pET53, pET74 indicated                            status spores determined biochemical analysis. <br />  VOL . 1 l, 1991                                                 CLONING OF S. CEREVISIAE PHOSPHOMEVALONATE KINASE <br />                                                                                                                                                       625 <br />                                                                 854)            .8411               -830          8420            <span id='am-136' about='obo:NCBITaxon_1432123' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_93875 obo:NCBITaxon_33154 obo:NCBITaxon_41033 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_177681 obo:NCBITaxon_177682 obo:NCBITaxon_30263 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-137' property="rdfs:label" content="Silo" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-139'  ></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>silo</span> <br />                                                               ACArt; r ( A!&apos;I&apos;(:&apos;1;ATr              C(X:(&apos;C:(;ls&apos; ATAA(;(;AA(;GA AG(,S;AC;ACAT <br />                          790             -7m0            770) I1ill .711)                           -750              -740            -3 <br />         7120 <br />                                           (;BAArA&apos;1T1&apos; ACCG(C;.AAA ACAA CT CCUCC <br />                         A&apos;MT(&apos;AAAT&apos;rrl&apos; CA(&apos;                                                        AACAIX:C(A(C      (aAA(;AA;TTAIT; AA&apos;AAI(^En <br />                          710             -70x <br />        AGAVACGCCA AATAGAIXCTC AT&apos;TATC&apos;&amp;(aCAAAT TA       -690             .680                        670              .66Mb             -6150 <br />                                                             ATA&apos;<span id='am-120' about='obo:NCBITaxon_634723' typeof='owl:Thing obo:NCBITaxon_6475 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_75116 obo:NCBITaxon_54976 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-121' property="rdfs:label" content="Taia" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>TAIA</span> ATTC(CCCCCA(A                  C,C-ACCCTI&apos;CMA GAAACACGAT AAAA((:AAT; <br />        -640              63(1            -620            4610            *6o0)                      -590            -58(         -570 <br />        TAAt&apos;A&apos;A1V&apos;tI&apos;;r TIrCl&apos;A&apos;T&apos;t;AT   ACA&apos;TVA GA A C A&apos;mrA&apos;rAl&apos;TA(&apos;TA C,CAC(ATG&apos;I &apos;              C(A&apos;ATAATGAGC(, C^^ACACGATAA CffA(ATAA(:C <br />                         -5150             540           -W5( <br />        ATCAAAGTAA ATC.AAC(;(T(AA cGAcCCCC:GAA CAI AA CaCAC&apos; TCCATATA(C&apos;( -520                       -510              -500              .490 <br />                                                                                                     TGVTTAAAGTA GAAGTT&apos;AAAG             AAGAACACGA <br />        -480             -470             .460           -460 <br />        GGAAATt;C(&quot;CT TCAAAAT(IV TACTTGAAGA ACAGAAA(C(CA -440             AGT&apos;ATGGAAC <br />                                                                                                                 -420 <br />                                                                                                     TAATCAATGA TAAAACTAG,T <br />                                                                                                                                         .410 <br />                                                                                                                                        CCAGAAAATA <br />        -400             -390)            -380)          -370             -360                       -360        -340                    -330 <br />        ATAATCATCA GGAAACAAAC CCGCAAAAGc ACAAAACTAA                             ACCAAAAAAG           AAAACVC(CTA AAACAAACTA              CAACAG:ACA <br />        -320        -310         .300        .290                               -280                 -270          .260                 -~250 <br />        AGACCGGAAAA GAAAAATCAC AGAAGGCAGT TCT(CCGCAT                            CAAATACTAA           AAGGCCTACG GGCCACGACC              CAAAAAGCCG <br />        -240        -230         -220        -210                               -200 Acc1            -190          .18Q                 -~170 <br />        TGACCACGAAT .150 <br />                    ACGATCCAA CTTGTCl=ACI&apos;AA ATAATCAAAT <br />                           B III -140        -130 <br />                                                                                TGTATACAGC <br />                                                                                .120 <br />                                                                                                     AATATCTACA TAGTAATAAT              TATrCCGATC <br />        C160                             11                                                          -110        .100                   -90 <br />         CGrMWITTTA <br />        .__ <br />                    TACGAGCATT AAAAGUAAAA AAGTI&apos;CGCTT <br />                 _ -7(0          -60         -54) <br />                                                                                rrAAGGCCGGA <br />                                                                                -40 <br />                                                                                                     GTTAACGTCA TGTCGAATCG <br />                                                                                                     -30        -20        -10 CboI <br />                                                                                                                                        AAACAAACAT <br />         ITAATXMAATCsGAAAAAC AAAAAAGCCTC AAAAAACAW                              V&apos;CCTGAAGGCC         TCATAATAAA TCAATCGGCT GCCTCGAAA <br />        1                       15                              30                             45                              60 <br />        ATG TCA GAG TIC AGA GCC rrC ACT CC&apos; CCA CGC AAA CCC TTIA CTA                                 GCT GCT GGA TAT VTA GTT VTA CAT ACA <br />        Met Ser Clu lou Arg <span id='am-115' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-119'  ></span>Ala</span> Phe Ser Ala P&apos;ro Cly L.ys Ala Leou Lou                               Ala Cly Cly Tyr Lou Val Lou Asp Thr <br />        75                   90                   105          SphI <br />        AAA TAT CAA GCA VIff GTA CTC GA VrA TrCG UCA AGc ATCATT GCT <br />                                                                                                     120                 135 <br />        Lys Tyr Glu Ala Phe Val Val Cly l.eu Ser Ala Arg Met Ills Ala                                GTA CCC CAT CCT TAC GGT TCA TIC CAA <br />                                                                                                     Val Ala His Pro Tyr            Cly Ser Lou CIn <br />             150                 165                  180                                                195                            2tO0 <br />         GCC TCT CAT AAC TTT GAA CTG CUT CTG AAA ACT AAA CAA VIT AAA                                 CAT CCC GAG TGG CTG            TAC CAT ATA ACT <br />        8ly  Ser Asp Lys Phe Glu Val Arg Vail Lys Ser Lys Clii Phe Lys                               Asp Gly Glu Trp Lou            Tyr liis lie Scr <br />                 225                 240                  255                                                270                            285 <br />        CCT AAA ACT CCC TTC ATT CCT CGT TCC ATA CQ( GGA TCT AAG AAC                                  CCT TTC ATT GAA AAA            GTT ATC OCT AAC <br />        Pro Lys Ser Cly Phe lie Pro Val Ser Ile CIy Gly Ser Lys Asi                                  Pro Ph. lie GU Lys             Val 11. Ala Asn <br />                     300                 315                                             330            toS A11&apos; GAT A1T TTC                     32A <br />        GTA TTT AGC TAC Trr AAA CCT AAC( ATG CAC&apos;                       GAC TAC TGC AAT ACA AAC TTC TTCCTT <br />        Val Phe Ser Tyr Phe Lys Pro Asn Metl Asp                        Asp Tyr Cyi Asn Arg Asn Lou Phe Val 1l1 Asp Ila Ph. 5er <br />                         375                 390                                               405                             420 <br />        CAT CAT CCC TAC CAT TCT CAC GAC CAT ACC                         CrV ACC GAA CAT CUT          CGC AAC ACA ACA           TTC ACT TITT CAT TCG <br />        Asp Asp Ala Tyr Hia Ser Cln Glu Asp Ser                         Val Thr Clu liis Arg         Cly Aan Arg Arg           Lou 5cr Phe llia Scr <br />        435                  450                                        465                          480                         495 <br />        CAC ACA ATT SAA GAA CTT CCC AAA ACA CCC                         CTU CCC TCC TCC GCA          COT TTA CTC ACA        CTT TTA ACT ACA CCT <br />        Hias Arg lb GIU Gu Val Pro Lys Thr Cly                          Leu Uly Sor Ser Ala          Gly Lou Vail Thr       Vail Lou Thr hMr Ala <br />                                 525                                        540    AGAC AAA                 6M         C57A0&apos;CTAT I           V <br />        TTC WC TCC TTT TTT <span id='am-100' about='obo:NCBITaxon_43651' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_42425 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8511 obo:NCBITaxon_7776 obo:NCBITaxon_8515 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-101' property="rdfs:label" content="Uta" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>UTA</span> TCC CAC (TCG CAA                         AAT AAT                      TAT         CAA             CCA <br />        Lou   AlaSer      Ph. <br />                         Phe     Ser Asp <br />                                       Val   Clu          Leu           Aso Asn Val Asp Lys          Tyr Arg Clu       Val Ile His Ann Lou Ala <br />                                                    600                         615                          630                         645 <br />        CAA CTT OCT CAT TUT CAA CCT VAC OCT AAA ATT                         CCA ACC (;(; TTT         CAT CTA CcG        CCC CCA CCA TAT UCA TCT <br />        Gln Val Ala His Cys Gin Ala ln Cly Lys lie                          Cly Ser Cly Phe          Asp Val Ala       Ala Ala Arg Tyr Gly Ser <br />                    660                  675                                        690                      705                       720 <br />        ATC ACA TAT ACA AGA TTC CCA CCC GCA TTA ATC                         TCT AAT TTC CCA   CAT ATT GCA ACT CUT ACT TAC CCC ACT <br />        1le Arg Tyr Arg Arg Ph. Pro Pro Ala Leu 11                          5cr Aan Lou Pro   Asp IIs Cly Ser Ala Thr Tyr Cly Ser <br />                        735                  750                                         766                       780 <br />        AAA CTC CCC CAT TT CGTT CAT CAA GAA CAC TCG                         AAT ATT ACC ATT   AAA ACT AAC CAT VTA CCT T(CU GGA TTA <br />        Lys Lou Ala His Lou Val Asp GCU GCU Asp Trp                         Aso Ile Thr i0    Lya Ser Asn Ills Lou Pro 5cr CIy Lou <br />        795                  810                 825                                          840                       855 <br />        ACT TVA TUG ATG CCCG CAT ATT AAC AAT sGT TCA                     CAA ACA CTA AAA CTG CTC CAC AAC GTA AAA AAT TUG TAT <br />        Thr Lou Trp Met Cly Asp II   Lys Aan Cly Sor                     Glu Thr Val Lys Leu Val Cin Lys Val Lys Aan Trp Tyr <br />              870 <br />        CAT TCG CAT ATU CCA CAA ACC TTC <br />                                              885                        900                       9IS         T.hal. <br />                                                          AAA ATA TAT ACA CAGA CTC CAT CAT GCA AAT TUT AGA TT ATU CAT GGA <br />                                                                                                                             930 <br />        Asp Ser His Met Pro Clu Ser Leu                   lyas lie Tyr Thr Clu Lou Asp His Ala Asn Ser Arg Ph. Met Asp Cly <br />                945                 960                                         975                     990                       1005 <br />        CTA TCT AAA CTA CAT CGC TTA CAC                   GAG ACr CAT CAC CAT TAC ACC CAT CAC ATA TVT GAC TCT CTT SAC AUG <br />        Lou Ser Lys Lou Asp Arg Lou His                   Ulu Thr His Asp Asp Tyr Ser Asp Cln Ile Ph. Clu Ser Lou lu Arg <br />                    1020 TOT AA AAG TAT <br />        AAT GAC TGT Thr <br />                    ACC <br />                                                          1035 <br />                                                          CCIT CAA ATC ACA CAA COTT 1050 ACA CAT CCA CTT G(&apos;c106 <br />                                                                                                                   ACA ATT AGA CUT 1rC <br />                                                                                                                                    A <br />                                                                                                                                       1080 <br />        Amn Asp Cys      Cys ln Lys Tyr                   Pro Glu lie Thr Clu Val Arg Asp Ala Val Ala Thr lie Arg Arg aer <br />                                1095                            1110 ECORV               1125                      1140 <br />       TTT ACA AAA ATA ACT AAA                 &quot;AA TCT    GCT CCC CAT ATC CAA CCr CCC CTA CAA ACT A(sC TTA TTC C(AT CAT TC&apos; <br />       Ph. Arg l.ys IIe Thr l1ys              Glu 8r      Gly Ala sp iie Clu Pro Pro Val Cin Thr Ser lou lou Asp Asp Cys <br />       IlI                   1170                                   1185                      1200                      1215 <br />       CAG AC-C&apos; TTA AAA GGA CTT              CTT &apos;ACT    TGC( TTA ATA C&apos;C&apos;T CCT GC&apos;T CCT QGT TAT GAC&apos; ;CCC ATT (XIA GTG AVI&apos; A(&apos;T <br />       Gln Thr Lou Lysa Cly Val               Leu Thri    Cys Lou lIe 1ro Clgy Al Clgy ly Tyr Asp AJla-A                Ala Val lie Thr <br />              1230                       1245                            1260                      1275                 1285           1295 <br />        AAC   C`AA CAT   (;I&apos;T (CAT CVIT ACC <br />        Lys GIn Asp Val Asip Lou Arg Ala <br />                                              CGIC        CAA CCC (TA ATG ACA AAA CAT TTC TAA <br />                                                          Uin Pro LAu Met Thr l.ys As Phe c <br />                                                                                                              C(C`TTA(AATGC  (CITCA1TCTAA. <br />                  130X(          131(3                    1:320          1:130i          1340           1360            1360           1370 <br />        T&apos;A(XX,TAC T;(Z;G;TTA GGAAAGAAAA AGA&apos;CC(;GAA A(CTA1XTM&apos; ATAAATAAC&apos;T TAACGTACAT AATAC;T:Tt&apos; <br />                  1310           1390            14(W)         1410           1420(       1430          144()         14501 <br />        C:AT(;T(;GA(AT C&quot;ITTATAAAT (<span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>IA</span> AGM(;7&apos;rCs A ACAA(Wr(a (;(CTGCAACATC TAA(&apos;A(&apos;An&apos;A TCVI&apos;&apos;CCATA CTA I&apos;AAA <br />                  1460           1470)           14W0          1490          I.150        1510          1524)         1530 <br />         &apos;WVII:X&quot;VIl(- CO(OC(XXCAT(:&apos;e&apos; AT&apos;AAAAVI&apos;(&apos;T rTfTAATIX;GCT AC(AAC&apos;CTrrTA C(ATA(CTACA TI;(ACCATAICsT At&quot;IIA&apos;OAAAC( <br />                  1540 Xhal      &amp;411 <br />       ATIAAATAAAC(: VrTll&apos;A(A <br />   FIG. 4. Nucleotide predicted amino acid sequence ERG8 gene. Nucleotides DNA sequence numbered starting the <br /> ERG8 translational start site nucleotides 5&apos;-untranslated region designated negative numbers. The predicted amino acid <br /> sequence phosphomevalonate kinase shown underneath. A consensus TATA box enclosed box, possible polyadenylation <br /> signals near 3&apos; end underlined. Restriction sites labeled underlined overlined. <br />  626     TSAY AND ROBINSON                                                                                                     MOL. CELL. BIOL. <br />  <br /> Concensus                       - - -   -G   G        G     X(12-25)..     K      (9). A codon <span id='am-6' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>bias</span> index 0.31 calculated the <br />                                                                                   predicted ERG8 protein, suggesting low moderate expres- <br /> ERG8           ISOV                                                               sion gene S. cerevisiae. <br />                       P -     K T G L G S         S A G      ...X(25)...   K183 <br />                                                                                      Disruption ERG8 gene chromosomal locus. To <br /> HSKI           90M    P   I   G S       G L G S   S   A C    ...X(16)...   K116   determine ERG8 gene essential isoprene <br />                                                                                   sterol biosynthesis S. cerevisiae, chromosomal dis- <br /> RAR1           140L   P   I   G A G L G S S A S              ...X(24)... K174     ruptions gene constructed method of <br /> GALl           162    P   T G           G L SS S S    A A         X(2)... K19S <br />                                                                                   Rothstein (44). A large internal region ERG8 coding <br />                                                                                   region replaced DNA fragments containing select- <br />                                                             ... <br />  <br />  <br />  <br />  <br />   FIG. 5. Peptide homologies phosphomevalonate kinase                     able markers (Fig. 6A). Constructs contained ERG8 <br /> and proteins. The consensus sequence ATP-binding site                gene disrupted URA3 gene HIS3 gene <br /> present protein kinases aligned deduced amino                 transformed linear DNA fragments diploid <br /> acid sequence yeast phosphomevalonate kinase (ERG8), E. coli                   strain ET16, heterozygous erg8 mutation <br /> homoserine kinase (HSKI), yeast RARI protein, yeast galac-                    (Table 1). Stable uracil histidine prototrophs se- <br /> tokinase (GAL]). Positions homologous peptides the <br /> respective proteins indicated superscript numbers.                         lected; approximately half type be <br />                                                                                   temperature sensitive growth 37&#176;C, suggesting that <br />                                                                                   contained disruption ERG8 allele the <br />                                                                                   chromosome. To confirm temperature-sensitive <br /> sensus sequence element (31) region,                        URA+ HIS&apos; diploid cells contained appropriate null <br /> indicating ERG8 mRNA spliced. An incom-                           alleles ERG8 gene, Southern analysis per- <br /> plete ORF 637 bp (Fig. 3A; Fig. 4, positions -846                           formed. Genomic DNA disrupted undis- <br /> -200) begins 5&apos; BglII site terminates near                     rupted strains examined (Fig. 6B). DNA three <br /> AccI site detected 207 bp upstream ERG8                           undisrupted strains yielded single 3.6-kb band, demonstrat- <br /> ORF. No function incomplete ORF                         ing erg8-1 ERG8 alleles present diploid <br /> study.                                                                            strain similar size. Chromosomal DNA the <br />    Searches EMBL/GenBank PIR data bases                           ERG8-disrupted diploid strains produced second band of <br /> the ERG8 DNA predicted protein sequence failed                              4.6 4.8 kb. The additional bands size <br /> identify closely related genes proteins. However, lim-                       ERG8:: URA3 ERG8::HIS3 disruption fragments <br /> ited region ERG8 protein beginning amino acid 150                       plasmids pET61 pET62 (Fig. 6B, lanes 7 8), <br /> was share homologous sequences                              proving represented new alleles, erg8-AJ and <br /> other proteins (Fig. 5). As indicated, proteins                         erg8-A2, ERG8 gene. <br /> contain motif Gly-X-Gly-XX-Ala-X(13-26)-Lys. A simi-                             To examine phenotype ERG8 disrupted cells, tem- <br /> lar motif contains <span id='am-11' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span> instead alanine                    perature-sensitive URA+ HIS&apos; prototrophs strain <br /> residue described (23, 36). Considerable                       ET16 sporulated dissected. For 10 complete tet- <br /> evidence suggests structure forms ATP-binding                        rads ERG8:: URA3 diploid 22 complete tetrads <br /> pocket kinases (23, 36). Phosphomevalonate kinase                         ERG8::HIS3 diploid, spore viability segregated 2 <br /> and proteins identified Fig. 5 use modified                          live:2 dead YCE (Table 3). The viable spores tetrads <br /> version structure serve comparable function.                         ura his, respectively, diploids, <br /> Searches data bases incomplete 5&apos;-ORF DNA                         indicating inviable spores contained deletion <br /> or predicted protein sequence turned homologous                             markers. These results obtained tetrads <br /> sequences.                                                                        derived ERG8 homozygous diploid strain YM197 <br />   Many yeast genes contain TATA box-related sequences                             ERG8 alleles disrupted HIS3 <br /> upstream coding regions serve basal promoter                     marker (Table 3). Together, results confirmed data <br /> elements. They typically 60 120 bp upstream                       Southern blot analysis showed ERG8 was <br /> the transcription initiation sites (22). A strong match                    single-copy essential gene haploid yeast cells. <br /> consensus yeast TATA sequence present position -80                             Transformation strain ET16 ERG8::HIS3 dis- <br /> (TATATAAA) ERG8 nucleotide sequence (Fig. 4).                              ruption gave rise equal numbers temperature-sensitive <br /> Weak matches consensus sequence                         non-temperature-sensitive diploids, suggesting that <br /> positions -120 (TCTAAA) -198 (TATACA). No signif-                             equal likelihood disrupting ERG8 <br /> icant inverted repeat sequences found, 8-bp perfect                      erg8-1 allele strain. To examine premise, one <br /> repeat sequences (at positions -447 -455 -483                         temperature-sensitive non-temperature-sensitive <br /> through -491) 20-bp imperfect repeat sequences (at                            ERG8::HIS3 diploid transformed multicopy <br /> positions -556 -585 -644 -663)                           plasmid pET58, contained ERG8 URA3 <br /> detected upstream presumptive ERG8 TATA boxes.                             genes, tetrad dissection. Spore viability the <br /> The significance sequences transcription                       URA+ marker segregated 4+:0- tetrads derived from <br /> the ERG8 gene remains tested.                                               diploids (Table 3). Furthermore, URA+ <br />    Sequences required efficient termination transcrip-                      spores temperature sensitive. These results demon- <br /> tion RNA polyadenylation described                             strated ERG8 gene rescued inviability the <br /> number yeast genes. No tripartite termination elements                       erg8-A2 disruption complemented erg8-1 mutation. <br /> the type proposed Zaret Sherman (58)                         To verify findings, isolated spores tetrads <br /> the 3&apos; end ERG8 gene. However, sequences similar                         streaked YME plates contained 5-flu- <br /> those putative consensus polyadenylation signal,                            oroorotic acid counterselect cells carried URA+ <br /> AATAAA, detected 72 113 bp downstream                              plasmid (11). Among tetrads examined, the <br /> ERG8 TAA stop codon (Fig. 4). The frequency codon use                           spores tetrad viable 5-fluorootic <br /> in ERG8 coding region showed striking bias                           acid plates. The viable spores consistently either <br /> codons utilized highly expressed yeast genes                         temperature sensitive not, depending they <br />  VOL. 11, 1991                                                            CLONING OF S. CEREVISIAE PHOSPHOMEVALONATE KINASE                                                      627 <br />  <br />                                A. <br />                                                    R       P            Bg         A     BgXS               A SsHXb E            Xb    R <br />                                                                         I          I     I    IiI.1.         i_    It        I <br />                                      ERG 8          I-                                                                                             3.6 Kb <br />  <br />  <br />                                                                                                                         URA3 <br />  <br />                                      erg 8-1 F-                                                                                                    4.6 Kb <br />                                                                                                             HIS3 <br />                                                                                                                                  -14 <br />                                      erg 8-A2 -                                                                                                    4.8 Kb <br />                                                                                              x                                                B <br />                                      probe                                                                                                   --i <br />  <br />                                 B.                              Kb 1 2 3 4 5 6 7 8   Kb <br />                                                                  5-                _48Kb <br />                                                                      4- <br />                                                                                                                   4.6Kb <br />                                                                                              _~4. <br />                                                                   4- <br />  <br />                                                                                                              -    3.6Kb <br />                                                                 3- <br />  <br />  <br />  <br />                                                                  2- <br />  <br />                                                                1.6-&apos; <br />   FIG. 6. Southern blot analysis yeast genomic DNA. (A) Diagrammatic representation ERG8 fragments pBR322-based <br /> plasmids pET61 (ERG8:: URA3) pET62 (ERG8::HIS3) used create ERG8 chromosomal disruptions, erg8-AI] erg8-A2. <br /> The restriction map symbols described legend Fig. 3B. Fragment sizes listed right indicate ERG8 band sizes <br /> expected EcoRI digestion plasmid genomic DNA containing disruptions. (B) Southern analysis EcoRI-digested yeast genomic <br /> DNA (lanes 1 5) plasmid DNA (lanes 6 8). Genomic DNA ERG8 haploids (lanes 1 2), ERG81erg8-1 diploid <br /> ET16 (lane 3), erg8-A/1erg8-1 diploid (lane 4), erg8-A2/erg8-1 diploid (lane 5), plasmid DNA pET45, pET61, and <br /> pET62 (lanes 6 8, respectively). All lanes probed radiolabeled XhoI-BamHI ERG8 fragment shown Fig. 6B. <br />  <br />  <br /> came       temperature-sensitive               non-temperature-sen-                       experiments indicated alternate ERG8 alleles had <br /> sitive diploid originally. Thus, erg8-A2 allele rescue                                     disrupted heterozygous diploid strain. <br /> erg8-1 allele complementation dependent                                                 Expression ERG8 gene GALl promoter. As <br /> continuous presence ERG8 plasmid. The results                                            described above, levels phosphomevalonate kinase were <br /> were obtained analysis low-                                              increased severalfold wild-type cells bearing ERG8 <br /> copy-number ERG8 plasmid pET63 (Table 3). Together                                                  gene multicopy plasmid. However, level expres- <br />  <br />  <br />  <br />                                                     TABLE 3. Genetic analysis ERG8 gene disruptions <br />           Diploid strain                                                     Spore viability                                                         Spore segregationa <br />    Disruption&apos;             Plasmid&apos;          4 :O               3+:1&#176;            2+:2&#176;              1+:3&#176;           0+:4&#176;         ERG8:erg8                 URA3:ura3     HIS3:his3 <br /> YM197 <br />   ERG8:: URA3              None                0                 0                 13                 0                  0             2:0                     0:2          NA <br /> ET16 <br />   ERG8::URA3               None                0                 0                 10                 0                  0             0:2                    0:2           NA <br />   ERG8::HIS3               None                0                 0                 22                 0                  0             0:2                    NA            0:2 <br />   ERG8::HIS3               pET58               7                 0                  0                 0                  0             4:0                    4:0           2:2 <br />   ERG8::HIS3               pET63               5                 0                  0                 0                  0             4:0                    4:0           2:2 <br />   erg8::HIS3               None                0                 0                 19                 0                  0             2:0                    NA            0:2 <br />   erg8::HIS3               pET58              13                 0                  0                 0                  0             4:0                    4:0           2:2 <br />   aERG8 (or erg8) status spores determined biochemical analysis. NA, Not applicable. <br />   bThe wild-type diploid strain YM197 (ERG8/ERG8) heterozygous diploid strain ET16 (ERG81erg8) disrupted ERG8 (or erg8) allele by <br /> integration linear ERG8:: URA3 ERG8: :HIS3 disruption fragment. <br />   c Some ERG8: :HIS3 diploids secondarily transformed pET63 (YCp5O) pET58 (YEp352) plasmids contained URA3 ERG8 <br /> structural genes sporulation tetrad dissection. <br />  628     TSAY AND ROBINSON                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                        A.                                                        B.      pMH101      pET 77 <br />                                                                                          GLU GAL GLU GAL M            MW <br />  <br />  <br />  <br />                                                                                                                    -200 <br />  <br />  <br />  <br />  <br />                                                                                                 .                  -97 <br />  <br />                                                                                                                 &quot;u-68 <br />  <br />                                                                                                                     -13 <br />  <br />  <br />  <br />                                                                                                 _               _~~-26 <br />  <br />                                                                                                                     -18 <br />                                                                                            1    2    3     4 <br />   FIG. 7. Overexpression ERG8 gene GAL] promoter. (A) Structure multicopy plasmid pET77 used galactose <br /> inducible expression ERG8 gene S. cerevisiae. For details plasmid construction Materials Methods. (B) Yeast cells that <br /> contained pET77 parent plasmid pMH101 grown overnight glucose medium (GLU) galactose medium (GAL). Equal amounts <br /> of cell protein electrophoresed 12% sodium dodecyl sulfate-polyacrylamide gel stained Coomassie blue. An arrow <br /> indicates position overexpressed 48-kDa ERG8 protein lane 4. M, Molecular size markers. <br />  <br />  <br /> sion low permit ERG8 gene product                  findings utilized erg8-1 mutant isolate yeast <br /> identified sodium dodecyl sulfate-polyacrylamide gels                library plasmid complemented temperature- <br /> (data shown). To obtain higher levels expression,            sensitive biochemical defects mutant strain. This <br /> ERG8 coding region placed regulatory control              plasmid shown genetic analysis contain yeast <br /> of yeast galactokinase (GALl) promoter plasmid                   ERG8 gene. Southern analysis, gene disruption, plasmid <br /> pET77 (Fig. 7A). The parent vector pMH101 (21) pET77                rescue experiments confirmed cloned gene con- <br /> were transformed GAL&apos; strain, strain                 tained sequences corresponded ERG8 chromo- <br /> grown YM-glucose transferred YM-galactose                somal locus. Consistent known gene dosage effects, <br /> to induce expression GAL] promoter. Protein                    ERG8 gene present cells high-copy- <br /> extracts glucose- galactose-grown cells run               number plasmid, corresponding eightfold overproduction <br /> on sodium dodecyl sulfate-polyacrylamide gels (Fig. 7B).                phosphomevalonate kinase observed. <br /> Extracts pET77-transformed cells grown                   Subcloning nucleotide sequence analyses showed that <br /> galactose contained strong band approximately 48 kDa.              yeast ERG8 gene encoded unexpectedly large protein <br /> An analogous band seen extracts pET77-                  approximately 47 kDa. Previous characterizations the <br /> transformed cells grown glucose cells              porcine phosphomevalonate kinase shown 22 <br /> bearing pMH101 grown carbon source. The size                  kDa (6, 32). This difference protein size surprising, <br /> and galactose-regulated expression protein strongly             date comparisons yeast mammalian <br /> implied predicted (47.3-kDa) ERG8 gene                  clones enzymes sterol biosynthesis, including <br /> product. To confirm this, extracts analyzed                   HMG CoA synthase (27a), HMG CoA reductase (5), meval- <br /> standard spectrophotometric assay phosphomevalonate                 onate kinase (51a), farnesyl diphosphate synthetase (3), and <br /> kinase activity. Extracts pET77-transformed cells                  lanosterol demethylase (28), revealed strong conserva- <br /> grown galactose nearly 1,000 times enzyme <br /> activity did extracts cells grown glucose (Table <br /> 4). Simple chromatographic fractionation extracts con-                 TABLE 4. Phosphomevalonate kinase activity ergosterol <br /> taining overexpressed protein demonstrated phos-                                content yeast strainsa <br /> phomevalonate kinase activity copurified 48-kDa <br /> protein (data shown). No excess ergosterol accumulated,             Strain        Plasmid        Medium           Ergosterol        Sp act (nmol/ <br />                                                                                                                      content ()            min/mg) <br /> and deleterious effects enzyme overexpression were <br /> observed, prolonged growth (72 h) cells                Y294          pMH101        Glucose               0.76                &lt;15 <br /> YM-galactose.                                                           Y294          pMH101        Galactose             0.80                &lt;15 <br />                                                                         Y294          pET77         Glucose               0.75                &lt;15 <br />                                                                         Y294          pET77         Galactose             0.82             13,200 <br />                             DISCUSSION                                    a <br />                                                                              Yeast cells contained pMH101 (without ERG8) pET77 (with <br />   Studies Karst Lacroute suggested yeast strains            ERG8) grown overnight minimal medium indicated carbon <br /> that contained erg8 mutations blocked isoprene              source. Phosphomevalonate kinase activity crude ergosterol content of <br />                                                                         cells (percentage dry cell weight) determined described Materials <br /> ergosterol biosynthesis presence thermolabile           Methods. A specific activity 15 corresponds minimal of <br /> phosphomevalonate kinase (29). Our investigation confirmed              enzyme detectable spectrophotometric assay used. <br />  VOL . 1 l, 1991                                      CLONING OF S. CEREVISIAE PHOSPHOMEVALONATE KINASE                             629 <br />  <br /> tion protein size protein sequence. Molecular         formed inviable colonies medium contained ergoste- <br /> cloning cDNA encoding mammalian phosphomeval-             rol, tetrad dissection anaerobic <br /> onate kinase necessary determine regions       conditions known enhance ergosterol uptake S. cere- <br /> the yeast polypeptide conserved smaller           visiae (4). Repeated attempts rescue inviability erg8 <br /> mammalian protein. Potentially, additional sequences           null mutants addition dolichol ubiquinone to <br /> present yeast phosphomevalonate kinase confer           medium ergosterol unsuccessful. <br /> on novel properties associated mammalian           Such findings suggest requirement additional isoprene- <br /> enzyme. This possibility explored future            derived products farnesol support growth erg8 <br /> characterization yeast enzyme.                              null mutants. In contrast erg8 results, null mutants <br />    Another feature phosphomevalonate kinase deduced             lanosterol demethylase (28) squalene monooxygenase <br /> from ERG8 nucleotide sequence location            (52a) viable grown medium supplemented <br /> small peptide represent portion enzyme           ergosterol. However, enzymes utilized <br /> active site. Its importance suggested finding         exclusively ergosterol biosynthesis. It prove <br /> that homologous peptides present similar positions         interesting examine effect erg8 mutations other <br /> in known potential (RARI) metabolic ki-           processes rely isoprene-derived products, as <br /> nases. Further support <span id='am-43' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> came recogni-        protein farnesylation. <br /> tion related peptide motif identified protein      A curious finding study phosphomeval- <br /> kinases (23). In tyrosine serine threonine protein          onate kinase accumulate S. cerevisiae high <br /> kinases lysine residue consensus motif, GXiX            levels toxicity. Densitometric scanning protein <br /> X... .K, thought serve contact point,           gels indicated phosphomevalonate kinase constituted <br /> glycine repeat thought serve pocket ATP             approximately 20%o soluble protein growth cells in <br /> binding (23, 36). Phosphomevalonate kinase proteins        galactose medium. Despite enormous elevation en- <br /> shown Fig. 5 contain GAGL-iSSA... K motif          zyme levels, cells appeared normal visible and <br /> unlikely possess protein kinase activity lack      phase-contrast microscopy. Likewise, viability nor <br /> the additional peptide motifs conserved protein    cell doubling time appreciably altered cells. <br /> kinases. However, motif ERG8 protein             Moreover, ergosterol accumulation cells no <br /> involved ATP binding, alanine substitutions       greater control cultures, indicating the <br /> third glycine residue described nucleo-         overexpression phosphomevalonate kinase little ef- <br /> tide binding proteins (36). Modification motif           fect synthesis isoprene-derived product. Sim- <br /> phosphomevalonate kinase mutagenesis permit              ilar findings previously reported studies the <br /> the significance motif determined.                   activity acetoacetyl CoA thiolase (16) HMG CoA <br />    Although ERG8 promoter analysis conducted,           reductase (56) increased tenfold yeast strains bearing <br /> it truncation ERG8 upstream se-              corresponding structural gene multicopy plasmid. <br /> quences position -140 rendered gene unable               From studies own, yeast sterol <br /> complement erg8 mutant. This result indicate          production regulated primarily rate- <br /> sequences upstream ERG8 consensus TATA              limiting enzymes sterol biosynthesis HMG CoA <br /> box required optimal expression gene. A             synthetase squalene synthetase. Alternatively, vivo <br /> puzzling finding complete ERG8 coding region          mechanisms exist down-regulating enzyme <br /> carried pET63, absence upstream sequences,            activity increases measured vitro assays. We have <br /> could complement erg8 defect. It likely        begun testing various sterol isoprenoid compounds to <br /> reflected joining ERG8 coding region spurious      act feedback inhibitors phos- <br /> promoter vector sequences. A precedent vector          phomevalonate kinase manner consistent vivo <br /> artifacts established studies (8). We        regulation enzyme. <br /> currently examining <span id='am-95' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-96' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99'  ></span>cis</span>-acting elements ERG8 <br /> promoter greater detail.                                                              ACKNOWLEDGMENTS <br />    Karst Lacroute (29) postulated exogenous ergo-            We thank Wolfram Siede YCp5O genomic library. We <br /> sterol improved growth erg8-1 mutants            appreciate critical reading manuscript Tobe Fisch, <br /> allowed divert limited isoprene supply      Jessica Gorman, Richard Gregg, William Scott. We grateful <br /> synthesis essential products ergosterol. In      colleagues Jan Tkacz, Richard Tanaka, Don Kirsch, <span id='am-90' about='obo:NCBITaxon_483412' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6800 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6796 obo:NCBITaxon_6072'><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Myra" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Myra</span> <br /> course cloning work, compared growth              Kurtz, Rose Kelly helpful discussions. We thank Ellen <br /> ergosterol medium erg8 mutants           Tornquist Jose Alcantara contributions prepara- <br /> erg8 upcl mutants healthier parent erg8              tion manuscript illustrations, respectively. <br />                                                                       We acknowledge Bristol-Myers Squibb Pharmaceutical Research <br /> strains, judged number criteria. Unlike erg8            Institute supporting basic aspects research sterol metabo- <br /> strains, erg8 upcl strains formed uniformly sized         lism. <br /> colonies permissive temperatures. We believe these <br /> differences reflect consistent uptake ergosterol by <br /> the erg8 upcl cells. The upcl mutation reported                               ADDENDUM <br /> similar effects growth yeast mutants defective              The ERG8 protein sequence reported contains four <br /> oxidosqualene cyclase (34), enzyme ergosterol           polypeptide domains highly conserved <br /> biosynthesis. Our observations growth erg8 upcl          rat mevalonate kinase (52) yeast meval- <br /> mutant cells consistent Karst Lac-          onate kinase, recently cloned laboratory (42a). <br /> route hypothesis.                                                                         REFERENCES <br />    Even presence ergosterol, growth erg8 erg8       1. Anderegg, R. J., R. Betz, S. A. Carr, J. W. Crabb, W. <br /> upcl cells blocked restrictive temperatures, indicating          Duntze. 1988. Structure Saccharomyces cerevisiae mating <br /> that phosphomevalonate kinase essential               hormone a-factor: identification S-farnesyl cysteine a <br /> just ergosterol synthesis. Haploid erg8 null strains likewise          structural component. J. Biol. Chem. 263:18236-18240. <br />  630     TSAY AND ROBINSON                                                                                                  MOL. CELL. BIOL. <br />  <br />  2. Anderson, M. S., M. Muehibacher, I. P. Street, J. Proffitt,           ase Saccharomyces cerevisiae detection virus-specific <br />     C. D. Poulter. 1989. Isopentenyl diphosphate: dimethylallyl               enzyme activity cell-free lysates. J. Virol. 62:4493-4498. <br />     diphosphate isomerase; improved purification enzyme         22. Hahn, S., E. T. Hoar, L. Guarente. 1985. Each three <br />     isolation gene Saccharomyces cerevisiae. J.               &quot;TATA elements&quot; specifies subset transcription initi- <br />     Biol. Chem. 264:19169-19175.                                              ation sites CYCI promoter Saccharomyces cerevisiae. <br />  3. Anderson, M. S., J. G. Yarger, C. L. Burck, C. D. Poulter.            Proc. Natl. Acad. Sci. USA 82:8562-8566. <br />     1989. Farnesyl diphosphate synthetase: molecular cloning, se-         23. Hanks, S. K., A. M. Quinn, T. Hunter. 1988. The protein <br />     quence expression essential gene S. cerevisiae.            kinase <span id='am-40' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>: conserved features deduced phylogeny the <br />     J. Biol. Chem. 264:19176-19184.                                           catalytic domains. Science 241:42-52. <br />  4. Andreasen, A. A., T. J. B. Stier. 1953. Anaerobic nutrition       24. Heilig, H., G. Popjak. 1961. Studies biosynthesis of <br />     Saccharomyces cerevisiae. I. Ergosterol requirement                cholesterol. XIII. Phosphomevalonic kinase liver. J. Lipid <br />     growth defined medium. J. Cell. Comp. Physiol. 41:23-35.             Res. 2:235-243. <br />  5. Basson, M. E., M. Thorsness, J. Finer-Moore, R. M. Stroud,        25. Hinnen, A., J. B. Hicks, G. R. Fink. 1978. Transformation of <br />     J. Rine. 1988. Structural functional conservation             yeast. Proc. Natl. Acad. Sci. USA 75:1929-1933. <br />     yeast human HMG-coenzyme A reductases, rate-                  26. Holmes, D. S., M. Quigley. 1981. A rapid boiling method for <br />     limiting enzyme sterol biosynthesis. Mol. Cell. Biol. 8:3797-          preparation bacterial plasmids. Anal. Biochem. 114:193- <br />     3808.                                                                     197. <br />  6. Bazaes, S., E. Beytia, A. M. Jabalquinto, F. S. Ovando, I.         27. Ishibashi, Y., Y. Sakagemi, A. Isogai, A. Suzuki. 1984. <br />     Gomez, J. Eyzaguirre. 1980. Pig liver phosphomevalonate               Structures tremerogens A-9291-I 9291-VIII: petidyl sex <br />     kinase. I. Purification properties. Biochemistry 19:2300-             hormones <span id='am-105' about='obo:NCBITaxon_29896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_5215 obo:NCBITaxon_5234 obo:NCBITaxon_2759 obo:NCBITaxon_105767 obo:NCBITaxon_155616'><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Tremella brasiliensis" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Tremella brasiliensis</span>. Biochemistry 23:1399-1404. <br />     2304.                                                                 27a.Jarman, K., T. Mozer. Personal communication. <br />  7. Bazaes, S., E. Beytia, A. M. Jabalquinto, F. S. Ovando, I.         28. Kalb, V. F., C. W. Woods, C. R. Dey, T. R. Sutter, T. G. Turi, <br />     Gomez, J. Eyzaguirre. 1980. Pig liver phosphomevalonate               J. C. Loper. 1987. Primary structure P450 lanosterol <br />     kinase. II. Participation cysteinyl lysyl groups cataly-        demethylase gene Saccharomyces cerevisiae. DNA 6:529- <br />     sis. Biochemistry 19:2305-2310.                                           537. <br />  8. Beier, D. R., A. Sledziewski, E. T. Young. 1985. Deletion         29. Karst, F., F. Lacroute. 1977. Ergosterol biosynthesis in <br />     analysis identifies region, upstream ADH2 gene                Saccharomyces cerevisiae. Mol. Gen. Genet. 154:269-277. <br />     Saccharomyces cerevisiae, required ADRJ-medi-            30. Laemmli, U. K. 1970. Cleavage structural proteins the <br />     ated derepression. Mol. Cell. Biol. 5:1743-1749.                          assembly head bacteriophage T4. Nature (London) <br />  9. Bennetzen, J. L., B. D. Hall. 1982. Codon selection yeast.         227:680-685. <br />     J. Biol. Chem. 257:3026-3031.                                         31. Langford, C. J., F. J. Klinz, C. Donath, D. Gallwitz. 1984. <br /> 10. Bloch, K., S. Chaykin, A. H. Phillips, A. Waard. 1959.             Point mutations identify conserved, intron-contained <br />     Mevalonic acid pyrophosphate isopentenylpyrophosphate.                TAACTAAC box essential splicing signal sequence in <br />     J. Biol. Chem. 234:2595-2604.                                             yeast. Cell 36:645-653. <br /> 11. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. A positive           32. Lee, C. S., W. J. O&apos;Sullivan. 1985. Improved procedures for <br />     selection mutants lacking orotidine-5&apos;-phosphate decarbox-            synthesis phosphomevalonate assay and <br />     ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen.       purification pig liver phosphomevalonate kinase. Biochim. <br />     Genet. 197:345-346.                                                       Biophys. Acta 839:83-89. <br /> 12. Boll, M., M. Lowel, J. Still, J. Berndt. 1975. Sterol biosyn-     33. Levy, H. R., G. Popjak. 1960. Studies biosynthesis of <br />     thesis yeast. 3-Hydroxy-3-methylglutaryl CoA reductase            cholesterol X. Mevalonic kinase phosphomevalonic kinase <br />     regulatory enzyme. Eur. J. Biochem. 54:435 444.                           liver. Biochem. J. 75:417-428. <br /> 13. Botstein, D., S. C. <span id='am-53' about='obo:NCBITaxon_8952' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8949 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8948 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="falcons" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Falco" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Falco</span>, S. E. Stewart, M. Brennan, S. Scherer,     34. Lewis, T. A., F. R. Taylor, L. W. Parks. 1985. Involvement <br />     D. T. Stinchcomb, K. Struhl, R. W. Davis. 1979. Sterile host          heme biosynthesis control sterol uptake S. cerevi- <br />     yeasts (SHY): eukaryotic biological containment recombi-            siae. J. Bacteriol. 163:199-207. <br />     nant DNA experiments. Gene 8:17-24.                                   35. Miller, J. H. 1972. Experiments molecular genetics, p. <br /> 14. Bradford, M. M. 1976. A rapid sensitive method                431-435. Cold Spring Harbor Laboratory, Cold Spring Harbor, <br />     quantitation microgram quantities protein utilizing             N.Y. <br />     principle protein-dye binding. Anal. Biochem. 72:248-254.          36. Moller, W., R. Amons. 1985. Phosphate-binding sequences <br /> 15. Brown, M., J. Goldstein. 1980. Multivalent feedback regu-             nucleotide-binding proteins. FEBS Lett. 186:1-7. <br />     lation HMG CoA reductase, control mechanism coordinat-           37. Mortimer, R. K., D. C. Hawthorne. 1969. Yeast genetics, p. <br />     ing isoprenoid synthesis cell growth. J. Lipid Res. 21:505-           385-460. In A. H. Rose J. S. Harrison (ed.), The yeasts, vol. <br />     517.                                                                      1. Academic Press, Inc., New York. <br /> 16. Dequin, S., F. Boutelet, M. Servouse F. Karst. 1988. Effect       38. Nasmyth, K. A., K. Tatchell. 1980. The structure trans- <br />     acetoacetyl CoA thiolase amplification sterol synthesis          posable yeast mating type loci. Cell 19:753-764. <br />     yeasts S. cerevisiae S. uvarum. Biotechnol. Lett.             39. Naumovski, L., E. C. Friedberg. 1982. Molecular cloning of <br />      10:457-462.                                                              eucaryotic genes required excision repair UV-irradiated <br /> 17. DiHanich, M. E., D. Najarian, R. Clark, E. C. Gillman, N. C.              DNA: isolation partial characterization RAD3 gene of <br />     Martin, A. Hopper. 1987. Isolation characterization            Saccharomyces cerevisiae. J. Bacteriol. 152:323-331. <br />     MOD5, gene required isopentenylation cytoplasmic         40. Naumovski, L., E. C. Friedberg. 1984. Saccharomyces <br />     mitochondrial tRNAs Saccharomyces cerevisiae. Mol. Cell.               cerevisiae RAD2 gene: isolation, subcloning, characteriza- <br />     Biol. 7:177-184.                                                          tion. Mol. Cell. Biol. 4:290-295. <br /> 18. Faust, J. R., J. L. Goldstein, M. S. Brown. 1979. Synthesis       41. Okayama, H., P. Berg. 1982. High efficiency cloning of <br />     ubiquinone cholesterol human fibroblasts: regulation            full-length cDNA. Mol. Cell. Biol. 2:161-170. <br />     branched pathway. Arch. Biochem. Biophys. 192:86-99.             42. Popjak, G. 1969. Enzymes sterol biosynthesis <br /> </body></html>