<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Ericson et al. BMC Biology 2014, 12:14 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> On extent role small proteome in <br /> the parasitic eukaryote <span id='am-64' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-65' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Trypanosoma</span> brucei <br /> Megan Ericson1, Michael A Janes1,4, Falk Butter2,5, Matthias Mann2, Elisabetta Ullu3 Christian Tschudi1* <br />  <br />  <br />   Abstract <br />   Background: Although technical advances genomics proteomics research yielded better <br />   understanding coding capacity genome, major challenge remaining identification all <br />   expressed proteins, especially 100 amino acids length. Such information particularly <br />   relevant human pathogens, <span id='am-69' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span>Trypanosoma brucei</span>, causative agent African trypanosomiasis, it <br />   provide insight parasite biology life cycle. <br />   Results: Starting 993 T. brucei transcripts, previously shown RNA-Sequencing coincide with <br />   annotated coding sequences (CDS), homology searches revealed 173 predicted short open reading frames in <br />   transcripts conserved kinetoplastids 13 conserved representative eukaryotes. Mining <br />   mass spectrometry <span id='am-163' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-167'  ></span>data</span> sets revealed 42 transcripts encoding matching peptide. RNAi-induced <br />   down-regulation 42 transcripts revealed seven essential insect-form trypanosomes also <br />   required bloodstream life cycle stage. To validate specificity RNAi results, lethal phenotype <br />   rescued co-expressing RNAi-resistant construct corresponding CDS. These previously <br />   non-annotated essential small proteins localized variety cell compartments, including cell surface, <br />   mitochondria, nucleus cytoplasm, inferring diverse biological roles likely play T. brucei. We <br />   provide evidence small proteins required replicating kinetoplast (mitochondrial) DNA. <br />   Conclusions: Our studies highlight presence significance small proteins protist expose potential <br />   new targets block survival trypanosomes insect vector and/or mammalian host. <br />   Keywords: Genomics, Proteomics, Mass spectrometry data, Non-coding RNA, Mitochondria <br />  <br />  <br /> Background                                                                            (sORFs) remains largely unexplored territory many <br /> Recent advances high-throughput sequencing tech-                                   organisms. Nevertheless, copious reports clearly indi- <br /> nologies led discovery large number                               cate play crucial biological roles, including <br /> transcripts originating regions genome pre-                               protection pathogens [6,7], signal transduction <br /> viously thought silent [1]. One major challenge                                 [8], serving molecular chaperones [9], developmental <br /> arising observations determine                               regulation [10-13] calcium transport <span id='am-150' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-151' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-152'  ></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br /> these transcripts code protein classi-                             diac muscle contraction[14]. <br /> fied non-coding RNAs. <span id='am-93' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-94' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> task overwhelm-                                 Several proteins encoded sORFs identified <br /> ing, majority transcripts                              serendipitously biochemical methods com- <br /> potential encode small proteins, generally                               plex product processed precursor protein. One <br /> 100 amino acids (<span id='am-144' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-145'  ></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span>) [2,3]. Historically, arbitrary cut-                           example Drosophila tarsal-less (tal) gene, originally <br /> off open reading frames 100 aa applied                                  annotated non-coding, later shown encode three <br /> genome annotation projects [4,5] extent                                  small proteins crucial role fly development [13]. <br /> and functional significance small open reading frames                              Several studies used genome-wide approaches to <br />                                                                                       gauge prevalence sORFs. When examining potential <br /> * Correspondence: christian.tschudi@yale.edu                                          small proteins <span id='am-86' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-87' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span>, Ladoukakis et al. <br /> 1 <br />  Department Epidemiology Microbial Diseases, School Public Health,           identified 4,561 sORFs conserved closely re- <br /> Yale University, New Haven, CT, USA <br /> Full list author information available end article                lated <span id='am-120' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, <span id='am-81' about='obo:NCBITaxon_7237' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32355 obo:NCBITaxon_32358 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-82'  ></span><span id='am-83' property="rdfs:label" content="Drosophila pseudoobscura" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila pseudoobscura</span> [15]. Synteny, <br />  <br />                                         &#169; 2014 Ericson et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                         unless stated. <br />  Ericson et al. BMC Biology 2014, 12:14                                                                            Page 2 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> evidence transcription nucleotide substitution, nar-     Illumina high-throughput cDNA sequencing [28-31]. In <br /> rowed 4,561 conservative estimate 401          particular, provided evidence coding poten- <br /> sORFs. A study <span id='am-21' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-22' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-27'  ></span>Arabidopsis</span> small proteome                tial T. brucei genome larger originally <br /> assessed evolutionary conservation examined evi-            anticipated identifying 1,114 transcripts mapping to <br /> dence transcription predict expression          regions genome annotated ORFs [28]. A <br /> many 3,241 sORFs [16]. A report mammalian             total 993 transcripts potential to <br /> small proteome Frith et al. used FANTOM cDNA data            contain coding sequence (CDS) 25 amino <br /> to identify potential 1,240 sORFs using CRITICA             acids remaining 121 transcripts no <br /> gene-detection program [17]. Additionally, 25 sORFs        coding potential ORF larger 75 nucleo- <br /> GFP-tagged and, following transfection cells, 14        tides. However, remains established whether <br /> the fusion proteins detected, providing evidence        transcripts encode functional proteins. <br /> translation [17]. More recently, using novel combination         Founded set transcripts identified tran- <br /> of peptidomics RNA-Sequencing (RNA-Seq), Slavoff            scriptome analysis [28], applied bioinformatics ap- <br /> et al. identified 86 novel small proteins humans         proaches identify small proteins conserved across <br /> two tagged shown localize mitochon-          kinetoplastid species representative eukaryotes. Com- <br /> dria cytoplasm [18]. Nevertheless, date func-        bined mass spectrometry data, pinpointed 42 <br /> tional studies proteins encoded sORFs           high-confidence small proteins ranging size 49 to <br /> performed. In yeast, 140 small proteins tested          219 amino acids. RNAi-knockdown revealed seven essen- <br /> generating gene deletions 22 effect Saccha-       tial proteins insect-stage life cycle their <br /> romyces cerevisiae growth various conditions [19],        diverse subcellular localizations suggested involvement in <br /> whereas overexpression 473 small proteins Arabidop-       aspects T. brucei biology. <br /> sis resulted 49 recognizable phenotypes [20]. <br />    Mass spectrometry, powerful technique proteo-           Results <br /> mics validate existence putative protein candi-       T. brucei transcripts encoding evolutionarily conserved <br /> dates, applied studies [18,21-25].          potential small proteins <br /> High-resolution mass spectrometry provides accur-          We previously published single-nucleotide resolution <br /> ate precursor ion masses combined stringent            genomic map T. brucei transcriptome, in- <br /> statistical methods enhances certainty peptide           cluded 1,114 transcripts originating annotated <br /> identification [26]. This key issue validation   CDS ([28]; original RNA-Seq data submitted to <br /> newly identified sORFs. In general, protein database          National Center Biotechnology Information (NCBI) <br /> derived genome used shotgun proteomics           Sequence Read Archive - SRA [32] - accession no. <br /> to identify peptides proteins mass spectromet-         SRA012290 1,114 transcripts accessible through <br /> ric raw data, frame translation genome        community file, Tbrucei_novel_transcripts.fasta, Tri- <br /> also frequently employed [24,25]. In case, cer-      TrypDB [33]). After reexamination data set <br /> tainty existence protein increased         using latest T. brucei genome annotation (GeneDB ver- <br /> by observed corresponding RNA transcript. Recently,          sion 5, [34]), excluded 39 10 transcripts coding for <br /> we used combination stringent methods, is,            snoRNAs annotated proteins larger 300 amino <br /> ribosome footprinting, generation sequencing           acids, respectively, added novel transcripts coding <br /> advanced mass spectrometric technology, discover           proteins identified mass spectrometry (MS) data <br /> plethora novel sORFs <span id='am-11' about='obo:NCBITaxon_10358' typeof='owl:Thing obo:NCBITaxon_10357 obo:NCBITaxon_1 obo:NCBITaxon_548681 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_10292'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="Cytomegalovirus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cytomegalovirus</span>,             (Figure 1). Setting lower limit 25 aa, 987 the <br /> which determined exist protein level [23].         remaining transcripts contain (112 transcripts) <br />    The question functional small proteins            31 (1 transcript) ORFs total 4,699 ORFs [see <br /> exist particularly relevant organisms tightly      Additional file 1]. Eighty transcripts classified non- <br /> organized genome, parasitic protozoan Try-          coding RNAs, predicted ORFs 75 <br /> panosoma brucei. Protein-coding genes arranged           nucleotides. However, exclude possibility <br /> long unidirectional clusters intergenic regions       category coding potential using alter- <br /> a nucleotides length, leaving little        native initiation codons encoding proteins smaller than <br /> space sORFs non-coding RNAs. The initial se-             25 aa. <br /> quencing annotation 11 megabase-sized chro-             The selected 4,699 ORFs highly enriched short <br /> mosomes, published 2005, predicted 9,068 protein-            ORFs (sORFs), is, 100 amino acids, with <br /> coding genes [27]. As November 2013, number             4,499 ORFs (96%) falling category [see Additional <br /> has increased 10,574 (TriTrypDB); however, major           file 2: Figure S1]. Since proteins encoded sORFs largely <br /> challenge remains identify expressed proteins. This      escape standard genome annotations, examined evolu- <br /> quest addressed RNA-Seq studies using            tionary conservation combination computational <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                       Page 3 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                                                             1,114 T. brucei transcripts           Remove 39 transcripts <br />                                                                 coinciding               encoding snoRNAs and <br />                                                            annotated CDS identified             10 encoding proteins <br />                                                                   Kolev et al. [22]                 longer 300 aa <br />                               Add 2 transcripts with <br />                                matching MS data <br />  <br />  <br />                                                                 1,067 transcripts <br />  <br />  <br />                                                           987 transcripts encode           Remove 80 non-coding <br />                                                              4,699 ORFs (25-300 aa)               transcripts (&lt;25 aa) <br />  <br />  <br />  <br />  <br />                                                     58 predicted          173 ORFs conserved across <br />                                                    small proteins          kinetoplastids, including <br />                                                    supported           13 ORFs conserved across <br />                                                       MS data                    eukaryotes <br />  <br />  <br />                                 Add 5 T. brucei- <br />                               specific proteins with <br />                                matching MS data                                            Remove 21 proteins (predicted <br />                                                                63 small proteins          ribosomal annotated proteins, <br />                                                                                                 multiple copies) <br />  <br />  <br />                                                                42 small proteins <br />  <br />  Figure 1 Flowchart strategy used analyze T. brucei transcripts coinciding annotated coding sequences (CDS). <br />  <br />  <br />  <br /> approaches screen ORFs conserved kinetoplasti-                    domains (five hits) RNA recognition motif (RRM) <br /> dae representative eukaryotes benchmark pro-                   <span id='am-128' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-129' property="rdfs:label" content="superfamily" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>superfamily</span> (three hits) overrepresented. Finally, our <br /> tein expression. Kinetoplastid protists belong                       analysis SignalP [37] TMHMM [38] predictions re- <br /> <span id='am-38' about='obo:NCBITaxon_phylum' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="phylum" datatype="xsd:string"></span>phylum</span> <span id='am-185' about='obo:NCBITaxon_33682' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-186' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasRelatedSynonym" content="Euglenozoans" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="Euglenozoa" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Euglenozoa</span> include significant number                       vealed 5 173 potential small proteins a <br /> disease-causing parasites, T. brucei T. cruzi,                  predicted signal peptide considerable number <br /> the causative agent African trypanosomiasis Chagas                   (43 25%) predicted trans-membrane domain <br /> disease, respectively, Old New World Leish-                     seven having predicted domain. <br /> mania parasites, cause various forms leishmania- <br /> sis worldwide. First, conducted Basic Local Alignment                    Identification small protein candidates mass <br /> Search Tool (BLAST) analyses [35] kinetoplastid ge-                      spectrometry data <br /> nomes annotated proteins, excluding T. brucei                       For alternative approach based peptide evidence <br /> <span id='am-136' about='obo:NCBITaxon_subspecies' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-137' property="rdfs:label" content="subspecies" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subspecies</span> (see Methods details). Of 987 tran-                      recognize transcripts coding small proteins, we <br /> scripts, 157 encoded ORF conserved                       surveyed MS data ([21,22] study) peptides <br /> least kinetoplastid organism transcripts (Tb4.                 matching 4,699 selected ORFs described above. As <br /> NT.51, Tb5.NT.84, Tb6.NT.58 Tb8.NT.142) encoded                         reported previously [28], searching proteome data of <br /> between conserved ORFs total 173                    Panigrahi et al. [21] provided evidence expres- <br /> conserved ORFs [see Additional file 3]. Second, com-                     sion 16 small proteins, 16 the <br /> pared selected 4,699 ORFs annotated proteins                     173 small protein candidates identified bioinformatically. <br /> from representative eukaryotes, S. cerevisiae, Cae-                  In addition, MS data Butter et al. [22] this <br /> norhabditis elegans, <span id='am-28' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span>Arabidopsis thaliana</span>, D. melanoga-                     study revealed 63 hits. As providing validation <br /> ster, <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span>Mus musculus</span> <span id='am-98' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-104'  ></span>Homo sapiens</span>. We 13                       58 173 small protein candidates, data also <br /> ORFs significant alignments BLAST bit scores                       predicted small proteins specific T. brucei with <br /> ranging 34 227, 6 coding ribosomal pro-                    recognizable homologues kinetoplastids. We <br /> teins [see Additional file 4]. It worth noting                performed search hexatranslations the <br /> 13 ORFs set conserved kinetoplastids.                   trypanosome genome, revealed set of <br /> We surveyed 173 conserved ORFs known                           newly identified proteins (data shown). Taken to- <br /> protein domain(s) using CD-Search Tool (cdsearch/                       gether, able provide supporting MS data for <br /> cdd v3.10 [36]) detected domains 61 ORFs covering                    63 predicted small proteins 22 represented in <br /> a broad spectrum [see Additional file 5]. However,                      MS data set, and, T. brucei- <br /> ribosomal protein superfamily (six hits), various Zn finger                 specific hits, matches the <br />  Ericson et al. BMC Biology 2014, 12:14                                                                            Page 4 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> evolutionarily conserved 173 small protein candidates           Tb3.NT.18, Tb10.NT.86, Tb10.NT.87, Tb10.NT.90, Tb11. <br /> [see Additional file 3].                                        NT.28, Tb11.NT.29 Tb11.NT.108 essential genes <br />    The 63 small protein candidates supporting MS           (Table 1; Additional file 2: Figure S3). For subsequent <br /> data filtered removing predicted ribosomal      analyses focused seven essential predicted <br /> proteins annotated proteins predicted function        small proteins RNAi knockdowns revealing a <br /> and CDS multiple copies genome, leaving          change doubling time pursued further. <br /> with 42 small proteins analysis [see Additional        To confirm observed essentiality seven <br /> file 3]. This final group selected proteins ranges size   small proteins specific RNAi knockdown the <br /> from 49 219 amino acids 35 qualify small pro-         predicted transcript, performed following experi- <br /> teins. Transcript lengths vary 333 4,100 nucleo-        ments. First, verified transcript length expected by <br /> tides average 5&#8217; UTR length 119 nucleotides          RNA-Seq data [28] using Northern blot analysis. In six <br /> with median 110 nucleotides. This similar        seven cases single predominant hybridizing band <br /> global analysis transcriptome [28,30], me-       detected observed size matched predicted <br /> dian length 128 130 nucleotides reported. On          size limits resolution Northern blotting <br /> the hand, 3&#8217; UTR average 390 nucleotides        [see Additional file 2: Figure S4]. The seventh transcript, <br /> long median 237 nucleotides, be-      Tb3.NT.18, bands detected Northern blot. <br /> ing notably smaller medians reported            One band corresponded size predicted novel <br /> aforementioned transcriptome studies, 400 nucle-         transcript 709 nucleotides. Further interrogation by <br /> otides [30] 388 nucleotides [28].                           Northern blotting RT-PCR probes specific for <br />    Noteworthy characteristics collection 42          upstream (Tb927.3.1080) downstream annotated <br /> small proteins follows: putative <span id='am-105' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>-        gene (Tb927.3.1090) led conclude longer <br /> logues representative eukaryotes (Tb7.NT.49, Tb11.           RNA contained Tb3.NT.18 downstream <br /> NT.47 Tb11.NT.220); predicted domains include               transcript encoding component T. brucei U1 <br /> two RRMs Zn-finger domains; sixteen              small nuclear ribonucleoprotein (snRNP). This finding was <br /> predicted trans-membrane domain; pre-             reminiscent presence upstream open reading <br /> dicted signal peptide [see Additional file 3].                  frame (uORF) described organisms <span id='am-41' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>fungi</span> to <br />                                                                 humans [42,43]. uORFs defined predominantly <br /> RNAi screen 42 small proteins revealed 7           short ORFs 5&#8217; UTR previously annotated <br /> essential insect life-cycle stage                        gene experiments ongoing investigate whether <br /> RNAi knock-down strategies revolutionized              Tb3.NT.18 qualifies uORF. <br /> functional analysis genes T. brucei [39]. mRNA               Second, semi-quantitative RT-PCR verified the <br /> degradation triggered efficiently double-            knockdown seven essential transcripts effi- <br /> stranded RNA (dsRNA) produced vivo hairpin              cient [see Additional file 2: Figure S5]. Third, confirm <br /> RNA transcribed tetracycline-inducible promoter.         specificity RNAi knockdown, set to <br /> Thus, generated hairpin construct          rescue lethal phenotype expression an <br /> 42 ORFs using pTrypRNAiGate plasmid [40]. Each              RNAi-resistant construct. To this, CDS targeted <br /> construct stably integrated non-transcribed          RNAi assembled synthetic sequence bearing <br /> rRNA spacer region special procyclic-form recipient        silent mutation 12 contiguous base-pairs <br /> strain, named 29.13.6, expressing tet repressor         [44] flanked heterologous UTR sequences (see <br /> T7 RNA polymerase [41], clonal cell lines              Methods). In addition, HA-TEV-FLAG epitope tag or <br /> established. Upon RNAi induction tetracycline, 12          GFP tag added C-terminus [see Additional <br /> had growth phenotype differed un-induced            file 2: Figure S6]. Upon co-expression hairpin tar- <br /> control cells. Three knockdowns resulted            geting endogenous transcript corresponding <br /> slow-growth phenotype. For example, RNAi Tb5.                modified CDS stable cell line, endogenous tran- <br /> NT.58 resulted cell division time 16 hours           script destroyed, shown RT-PCR, all <br /> compared 8.5 hours un-induced cells             seven cases cells survived RNAi-resistant <br /> phenotype accompanied noticeable changes             transcript encoding small protein [see Additional <br /> in cell morphology [see Additional file 2: Figure S2].          file 2: Figure S3]. These results led conclude the <br /> In addition, knockdown small proteins (Tb11.             essential phenotype direct consequence the <br /> NT.222 Tb11.NT.66) resulted faster growth           knockdown targeted CDS. <br /> no obvious changes cell morphology [see Additional <br /> file 2: Figure S2]. Monitoring cell growth RNAi in-       Initial characterization essential small proteins <br /> duction remaining seven revealed cell lines     The RNAi rescue experiments described established <br /> stopped dividing eventually died demonstrating         epitope-tagged small proteins functional <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                  Page 5 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> Table 1 Characteristics seven essential proteins procyclics <br /> ORF ID                   Transcript              AA/MW                pI                 Predicted                    Predicted         Localization <br />                          size (nt)                                                       signal peptide               TM domain <br /> Tb10.NT.87               478                     64/8.2               11.46              No                           No                Mitochondria <br /> Tb11.NT.28               1,285                   56/6.3               9.3                No                           Yes               Mitochondria <br /> Tb11.NT.29               796                     62/7.6               6.96               No                           Yes               Surface <br /> Tb3.NT.18                709                     85/9.3               7.91               No                           Yes               Cytoplasm <br /> Tb10.NT.86               605                     93/10.5              9.67               No                           Yes               Cytoplasm <br /> Tb11.NT.108              461                     96/10.5              6.49               No                           No                Cytoplasm <br /> Tb10.NT.90               529                     67/7.3               4.49               No                           Yes               Nucleus <br /> AA, number amino acids; MW, molecular weight; ORF, open reading frame; pI, isoelectric point; TM, transmembrane. <br />  <br /> and, thus, used biochemical cell bio-                           immunofluorescence, proteins (Tb3.NT.18, Tb10. <br /> logical experiments. Using fluorescence microscopy                            NT.86 Tb11.NT.108) shown enriched in <br /> detected expression seven small proteins (Figure 2)                       cytoplasm, Tb11.NT.108 distributed through- <br /> and Western blot analysis confirmed proteins                            compartment, Tb3.NT.18 Tb10. <br /> had predicted relative molecular mass [see Additional                        NT.86 appeared somewhat concentrated the <br /> file 2: Figure S7]. Tb11.NT.28 Tb10.NT.87 revealed                           nucleus. Finally, Tb10.NT.90 distinct localization <br /> a fluorescence pattern typical procyclic trypano-                         nucleus, possibly indicative nucleolus. <br /> some branched tubular mitochondrion (Figure 2). Tb11.                              Since T. brucei undergoes extensive morphological and <br /> NT.29 appeared surface protein, observation                           metabolic changes life cycle alternating between <br /> supported subsequent experiments (see below). By                              mammalian (bloodstream) insect (procyclic) hosts, <br />  <br />                                       Small protein              Hoechst                      DIC <br />  <br />  <br />                                                                                                              Tb10.NT.87 <br />  <br />  <br />  <br />  <br />                                                                                                              Tb11.NT.28 <br />  <br />  <br />  <br />  <br />                                                                                                              Tb11.NT.29 <br />  <br />  <br />  <br />  <br />                                                                                                              Tb3.NT.18 <br />  <br />  <br />  <br />                                                                                                              Tb10.NT.86 <br />  <br />  <br />  <br />                                                                                                              Tb11.NT.108 <br />  <br />  <br />  <br />                                                                                                              Tb10.NT.90 <br />  <br />  <br />  Figure 2 Localization seven small proteins essential procyclic life cycle stage. Anti-HA anti-GFP antibodies used to <br />  detect C-terminal HA- (Tb11.NT.28 Tb11.NT.29) GFP- (Tb10.NT.87, Tb10.NT.90, Tb10.NT.86, Tb3.NT.18, Tb11.NT.108) tagged versions the <br />  proteins fluorescence microscopy. DNA stained Hoechst (blue) DIC images shown right panels. DIC, differential <br />  interference contrast. <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                                                   Page 6 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> it gauge essentiality seven                                               detected BiP, signal Tb11.NT.29 <br /> small proteins bloodstream forms. Thus, hairpin                                                  visible (Figure 4B). This behavior similar <br /> constructs transfected bloodstream form cell                                                procyclin (Figure 4C), well-characterized T. brucei <br /> line competent RNAi and, following induction, Tb11.                                                 surface protein specific procyclic life-cycle stage <br /> NT.29, potential surface protein, Tb10.NT.87,                                                   [45]. Since epitope tag C-terminus the <br /> probable mitochondrial protein, shown essen-                                                 Tb11.NT.29, result suggested portion <br /> tial stage life cycle (Figure 3A B). Growth                                          protein exposed surface. To corrobor- <br /> curves nonessential proteins                                               ate localization, performed cell fractionation <br /> Figure 3C Additional file 2: Figure S8. Based                                                experiments Western blot analysis Tb11.NT.29 <br /> above results, selected Tb11.NT.29, Tb11.NT.28                                                   enriched membrane fraction, similar pro- <br /> Tb10.NT.87 analysis.                                                                        cyclin, HSP70 was, expected, enriched in <br />                                                                                                         cytoplasmic fraction (Figure 4D). <br /> Tb11.NT.29: putative surface protein                                                                    To begin probe potential role Tb11.NT.29, <br /> The 796 nucleotide-long Tb11.NT.29 transcript encodes                                                 monitored effect RNAi knockdown cell <br /> 62 aa protein, highly conserved kinetoplastids                                              cycle progression cell morphology. In procyclic cells <br /> (84% identity T. brucei L. major)                                                 RNAi resulted slowdown growth days <br /> predicted trans-membrane domain (Figure 4A). Initial im-                                                followed cell death day post- <br /> munofluorescence suggested Tb11.NT.29                                                     induction (Figure 5A), RNAi effect was <br /> localized surface (Figure 2). To address possi-                                             pronounced bloodstream form cells cell <br /> bility, compared signal cells expressing epitope-                                            death occurring day (Figure 3A). <br /> tagged Tb11.NT.29 permeabilized                                                     For cell cycle analysis procyclics, parasites were <br /> detergent prior antibody exposure remained non-                                                   stained Hoechst various time points induc- <br /> permeabilized. Under permeabilized conditions de-                                                    tion number position nuclei kineto- <br /> tected Tb11.NT.29 endoplasmic reticulum                                                    plasts (mitochondrial kDNA) cell recorded. <br /> (ER) protein BiP (Figure 4B). However, omitted                                                  Cells kinetoplast nucleus (1K1N) are <br /> the permeabilization step, limiting access anti-                                                G1 cell cycle, cells kinetoplasts and <br /> bodies potential surface molecules, signal                                                    nucleus (2K1N) segregated kinetoplast and <br />  <br />  <br />                              A                                                                          B <br />                                                            8                                                            10 <br />                                                                            - tet                                                             - tet <br />                                   log cells (cumulative) <br />  <br />  <br />  <br />  <br />                                                                            + tet                                                             + tet <br />                                                                                                         log cells (cumulative) <br />  <br />  <br />  <br />  <br />                                                                                                                                  9 <br />                                                            7 <br />                                                                                                                                  8 <br />  <br />                                                                                                                                  7 <br />                                                            6 <br />                                                                                                                                  6 <br />                                                                                    Tb11.NT.29/BS                                                      Tb10.NT.87/BS <br />                                                            5                                                                     5 <br />                                                                0       1       2      3     4       5                                0   1     2   3    4    5   6   7 <br />                                                                              Days RNAi                                                       Days RNAi <br />                              C <br />                                                        10 <br />                                                                            - tet <br />                                                                            + tet <br />                               Log cells (cumulative) <br />  <br />  <br />  <br />  <br />                                                            9 <br />  <br />                                                            8 <br />  <br />                                                            7 <br />  <br />                                                            6 <br />                                                                                    Tb11.NT.28/BS <br />                                                            5 <br />                                                                0   1        2  3    4   5       6   7 <br />                                                                             Days RNAi <br />  Figure 3 Bloodstream-form growth following RNAi. (A) Tb11.NT.29 RNAi bloodstream-form cells (BS). (B) Tb10.NT.87 RNAi BS cells. <br />  (C) Tb11.NT.28 RNAi BS cells. Growth un-induced (&#8722;tet) induced cells (+ tet) shown log scale. The data based three <br />  independent experiments, average values standard deviations presented. <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                 Page 7 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                              A                                           TM <span id='am-76' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-77' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80'  ></span>Helix</span>                              [62 aa] <br />                               Tbr MLLGGFVPRRFSQFNRDPCWMFFIFSVGFWLGEYPAMMIKYNARDLVYDPHRYVWSHHDDHH <br />                               Tcr MFLGGFVPRRFSQFNRDPCWMFFIFSIGFWWGEYPAMQIKYNARDLVFDPHRYVWSHHDDHH <br />                               Lm MFLGGFVPRRFAQFNRDPWWMFFIFAIGAWAGEYPAMQIKYNARDLVLDPHRYVWSHLDDHH <br />  <br />                              B             Permeabilized                            Non-permeabilized <br />  <br />  <br />  <br />  <br />                                Tb11.NT.29      Hoechst       BiP          Tb11.NT.29      Hoechst       BiP <br />                              C <br />  <br />  <br />  <br />  <br />                                Procyclin       Hoechst       BiP            Procyclin   Hoechst         BiP <br />  <br />  <br />  <br />                              D    T        C     M       N   CSK <br />                                                                    Tb11.NT.29 <br />                                                                    GFP-tagged <br />  <br />  <br />                                                                    Procyclin <br />  <br />  <br />  <br />                                                                    BiP <br />  <br />                                                                    HSP 70 <br />  <br />  Figure 4 Characterization Tb11.NT.29. (A) Sequence conservation Tb11.NT.29 kinetoplastids. Amino acid sequences T. brucei <br />  (Tbr), T. cruzi (Tcr) L. major (Lm) aligned ClustalW conserved residues shaded black boxes similar residues are <br />  shaded grey. The predicted transmembrane domain (TM helix) indicated. (B) Immunofluorescence analysis cells expressing C-terminal <br />  HA-tagged version Tb11.NT.29. Permeabilized cells (left panels) non-permeabilized cells (right panels) probed HA <br />  (green) BiP (red) DNA stained Hoechst (blue). (C) Immunofluorescence analysis procyclin described panel (B). <br />  (D) Cell fractionation parasites expressing GFP-tagged version Tb11.NT.29. Western blot analysis performed GFP-tagged Tb11. <br />  NT.29 (top panel), procyclin (second top), BiP (third panel top) HSP 70 (bottom panel) total (T), cytoplasmic (C), membrane (M), <br />  nuclear (N) cytoskeleton (CSK) fractions. <br />  <br />  <br />  <br /> are end S phase, cells kinetoplasts                       knockdown, including flagellar detachment as <br /> and nuclei (2K2N) completed mitosis                             specific areas separation cell membrane <br /> poised cytokinesis [46]. Any arrangement aber-                      flagellum, complete separation only <br /> rant point defects cell cycle progression.                       visible contact point cell body fla- <br /> In wild-type cells, expected asynchronously                            gellum (Figures 5C D). We noted change in <br /> growing cell population, majority parasites                       shape cell body appeared specific to <br /> 1K1N 10% cells having 2K1N                             2K1N 2K2N cells. After kinetoplast replication and <br /> 2K2N configuration.                                                              separation, narrowing cell body evident be- <br />   RNAi-induced down-regulation Tb11.NT.29 pro-                             tween daughter kinetoplasts (Figure 5D), which <br /> cyclics resulted accumulation cells containing                         indicate defect cytokinesis. <br /> either 1K2N 1K0N, referred zoids <br /> (Figure 5B). Zoids 1K2N cells increased nearly                               Tb11.NT.28: mitochondrial inner membrane protein <br /> equally number days induction                         The 56-amino acid Tb11.NT.28 protein 45% iden- <br /> comprised 7.9% 6.9% cell population, respect-                         tical T. brucei L. major contains a <br /> ively. Several morphological changes observed                         predicted trans-membrane domain (Figure 6A). Initial <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                                           Page 8 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                     A                                                                        B       Tb11.NT.29 Cell Cycle Progression <br />                                    10                                                            100% <br />                                                              - tet <br />                                                              + tet <br />                         log cells (cumulative) <br />                                                                                                   80%                         &gt;2N <br />                                                  9 <br />                                                                                                                               1K2N <br />                                                                                                   60%                         1K0N <br />                                                  8 <br />                                                                                                                               2K2N <br />                                                                                                   40% <br />                                                                                                                               2K1N <br />                                                  7                                                                            1K1N <br />                                                                                                   20% <br />                                                                  Tb11.NT.29/PC <br />                                                  6                                                 0% <br />                                                      0   1   2 3 4 5            6    7                     0 1 2 3 4 <br />                                                              Days RNAi                                  Days RNAi <br />  <br />                     C                            Tb11.NT.29 Morphology                   D <br />                       100% <br />                                                                      Full flagellar <br />                         80%                                          detachment                                                                        DIC <br />                                                                      Partial flagellar <br />                         60%                                          detachment <br />                                                                      Defective <br />                         40%                                          cytokinesis <br />                                                                      Normal <br />                                                                                                                                                        Hoechst <br />                         20% <br />  <br />                            0%                                                                 Defective        Partial        Full       Normal Cell <br />                                                        0 4                                   cytokinesis      flagellar    flagellar <br />                                                      Days RNAi                                            detachment   detachment <br />  <br />  Figure 5 Growth cell cycle analysis following RNAi Tb11.NT.29. (A) Tb11.NT.29 RNAi procyclics (PC). Growth un-induced (&#8722;tet) <br />  induced cells (+tet) shown log scale. The data based independent experiments, average values standard deviations <br />  presented. (B) Analysis K/N configurations induction Tb11.NT.29 RNAi procyclic cells. Cells scored having one <br />  kinetoplast nucleus (1K1N), kinetoplasts nucleus (2K1N), kinetoplasts nuclei (2K2N), kinetoplast no <br />  nucleus (1K0N), kinetoplast nuclei (1K2N) nuclei (&gt;2 N). The data based independent experiments <br />  time point 1,000 cells counted. (C) Accumulation cells altered morphology induction Tb11.NT.29 RNAi in <br />  procyclic cells. (D) Example images normal cell cells altered morphology. Images DNA content (Hoechst) DIC displayed. <br />  At time point 500 cells counted. DIC, differential interference contrast. <br />  <br />  <br />  <br /> immunofluorescence staining revealed similar localization                                                 known inner membrane protein T. brucei mitochondria <br /> pattern Tb11.NT.28 fluorescent dye Mito-                                                    [50], occurred digitonin concentration, <br /> Tracker Red, cell-permeable mitochondrion selective                                                       portion mitochondrial HSP70, matrix pro- <br /> probe (Figure 6B). Cell fractionation experiments corrobo-                                                  tein, released little 0.015% digitonin. <br /> rated potential mitochondrial localization Tb11.                                                  RNAi-induced knockdown Tb11.NT.28 resulted in <br /> NT.28 enriched mitochondrial fraction similar                                                    cell death procyclic forms (Figure 7A), did not <br /> to RNA-editing associated protein (REAP), known mito-                                                     affect growth bloodstream forms (Figure 3C). An ana- <br /> chondrial marker [47], cytoplasmic HSP70                                                    lysis cell cycle progression procyclic cells fol- <br /> excluded fraction (Figure 6C). To determine                                                    lowing RNAi did result accumulation cells <br /> which mitochondrial compartment Tb11.NT.28                                                         containing aberrant DNA amounts. However, steady <br /> localized, exposed cells increasing concentra-                                                  increase number 2K1N cells observed with <br /> tions digitonin, detergent preferentially solubilizes                                             cell type constituting cells days <br /> the plasma membrane outer membrane                                                           post-induction (Figure 7B). In category, 50% cells <br /> mitochondria. As digitonin concentration increased,                                                  duplicated kinetoplast, daughter kine- <br /> specific mitochondrial compartments shown                                                      toplast remained linked dumbbell-shaped body, a <br /> be solubilized 0.015% digitonin releasing proteins                                                     larger number seen wild-type cells [46]. This in- <br /> from inter-membrane space, 0.025% digitonin solubil-                                                    dicated cells entered S phase not <br /> izing matrix proteins, 0.04% digitonin resulting release                                                 completed. <br /> of outer membrane proteins 0.1% digitonin solubiliz- <br /> ing inner membrane proteins [48,49]. As Tb11.NT.28                                                      Tb10.NT.87: mitochondrial matrix protein <br /> only released exposure 0.1% digitonin, likely                                                   Tb10.NT.87, basic protein 64 aa, highly <br /> localization inner membrane (Figure 6D),                                                       conserved kinetoplastids 10 aa <br /> solubilization trypanosome alternative oxidase (TAO),                                                  (Figure 8A). Performing fluorescence microscopy on <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                        Page 9 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                       A                                                         TM Helix                        [56 aa] <br />                        Tbr MFPGMQQPKALPMGEAHTQFWTNFNRFLTTSALSITVSLFIRYCEGRTSAVLLNGI <br />                        Tcr MFPGAQGPKALPDGESHKQFWSNFHRFLAVSAVTISFSLFVRHCERRTSTLLFAGV <br />                        Lma MFPGMHGPQALPDQEAYPMFWRNLNRFIAITTATVAVAVAVRYCEQRTHSMLVSSL <br />  <br />                       B <br />  <br />  <br />  <br />  <br />                       Tb11.NT.28       Mito-        Merge       Hoechst          DIC <br />                                       Tracker <br />  <br />                       C                                                      D <br />                             T    C      N      ER Mito                           0 0.015 0.025 0.04 0.05 0.1 % Digitonin <br />                                                            Tb11.NT.28                                                 Tb11.NT.28 <br />  <br />                                                             REAP                                                       TAO <br />  <br />                                                             HSP70                                                      mtHSP70 <br />  <br />  Figure 6 Characterization Tb11.NT.28. (A) Sequence conservation Tb11.NT.28 kinetoplastids. Amino acid sequences T. brucei <br />  (Tbr), T. cruzi (Tcr) L. major (Lma) aligned ClustalW conserved residues shaded black boxes similar residues are <br />  shaded grey. The predicted transmembrane domain (TM helix) indicated. (B) Immunofluorescence analysis cells expressing C-terminal <br />  HA-tagged Tb.11.NT.28. Cells stained anti-HA antibody (green) MitoTracker (red) merge signal HA MitoTracker <br />  shown yellow. (C) Cell fractionation cells expressing C-terminal HA-tagged Tb11.NT.28. Western blot analysis performed against <br />  C-terminal HA-tagged Tb11.NT.28 (top panel), RNA-editing associated protein (REAP, middle panel) cytoplasmic HSP70 (bottom panel) total <br />  (T), cytoplasmic (C), nuclear (N), endoplasmic reticulum (ER) mitochondrial (Mito) fractions. (D) Digitonin extraction analysis cells expressing <br />  C-terminal HA-tagged Tb11.NT.28. Parasites exposed increasing concentrations detergent digitonin indicated panels and <br />  solubilized proteins analyzed Western blot HA-tagged Tb11.NT.28 (top panel), trypanosome alternative oxidase (TAO, middle panel) and <br />  mitochondrial HSP 70 (mtHSP70, panel). <br />  <br />  <br /> live cells expressing GFP-tagged RNAi-resistant                            division mitochondrial DNA. The enlarged cell body <br /> construct revealed similar localization patterns                       remained unit single flagellum expected <br /> Tb10.NT.87 MitoTracker, indicative mitochon-                          cell kinetoplast. A substantial number of <br /> drial localization (Figure 8B). Cell fractionation experi-                     zoids (13.7%) 1K2N cells (16%) accumulated by <br /> ments consistent immunofluorescence                              time point. In accordance presence a <br /> data (Figure 8D) digitonin solubilization assays,                          single kinetoplast cells containing multiple nu- <br /> as described above, suggested Tb10.NT.87                            clei, potential defect kDNA replication further <br /> matrix protein, solubilization properties                       manifested appearance cells small kDNA <br /> similar mitochondrial HSP70, established marker                          detectable kDNA (Figures 9D E). For example, <br /> for matrix (Figure 8E). To pinpoint cellu-                     days Tb10.NT.87 RNAi, 32% cells had <br /> lar localization, used immunogold <span id='am-131' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-132' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-135'  ></span>electron</span> micros-                          normal-sized kDNA, 65% cells were <br /> copy cells expressing HA-TEV-FLAG tagged Tb10.                              scored having small kDNA. <br /> NT.87. Micrographs sections showed the <br /> protein mitochondrial matrix distri-                        Discussion <br /> bution appeared uniform (Figure 8C).                                     Although genome sequencing projects provided a <br />    Tb10.NT.87 shown essential procyc-                        wealth information genome structure and <br /> lic bloodstream forms (Figures 9A 3B)                            organization, encountered challenge cata- <br /> significant increase cells nuclei                        logue protein-coding genes. Since gene annotation pro- <br /> and grossly enlarged cell bodies noted knock-                        grams perform predicting small proteins, <br /> down Tb10.NT.87 procyclics (Figures 9B C).                           is, 100 aa, common practice to <br /> After days induction, 12.5% cells                       set artificial length cutoff 100 aa avoid flawed pre- <br /> three nuclei. Most cells contained                        dictions. However, recent computational functional <br /> single kinetoplast, indicating mitosis                       studies highlighted existence importance small <br /> occurring corresponding replication                        proteins numerous organisms [16-20,51]. Our approach <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                       Page 10 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                      A                                                                   B            Tb11.NT.28/PC <br />                                                                                          100% <br />                                                                 - tet                                                            1K0N <br />                                                10               + tet <br />                       Log cells (cumulative)                                                                                     &gt;2N <br />                                                                                          80% <br />                                                                                                                                  1K2N <br />                                                9                                                                                 2K2N <br />                                                                                          60% <br />                                                                                                                                  2K1N <br />                                                8                                                                                 1K1N <br />                                                                                          40% <br />  <br />                                                7                                         20% <br />  <br />                                                                         Tb11.NT.28/PC <br />                                                6                                          0% <br />                                                     0   1   2     3 4 5 6        7   8           0     1    2     3      4 <br />                                                                 Days RNAi                            Days RNAi <br />  Figure 7 Analysis growth cell cycle progression following RNAi Tb11.NT.28. (A) Tb11.NT.28 RNAi procyclics (PC). Growth for <br />  uninduced (&#8722;tet) induced (+tet) cells shown log scale. (B) Analysis K/N configurations induction Tb11.NT.28 RNAi. Cells were <br />  scored 24 hours having normal DNA (one kinetoplast nucleus (1K1N), kinetoplasts nucleus (2K1N) two <br />  kinetoplasts nuclei (2K2N)) abnormal DNA content (one kinetoplast nucleus (1K0N), kinetoplast nuclei (1K2N) <br />  nuclei (&gt;2 N). Cell counts described legend Figure 5. <br />  <br />  <br /> to gauge extent small proteome T. brucei in-                                   substantial proportion small proteome. The cur- <br /> tegrated experimental data (transcriptome MS data)                                       rently available MS data support view, pep- <br /> and evolutionary conservation. Starting 1,117 tran-                                     tide hits largely confined set 173 proteins. <br /> scribed sequences mapping annotated CDS [28],                                      The exception identification T. brucei- <br /> 987 potential code proteins                                   specific proteins. Thus, fully expose catalogue of <br /> at 25 amino acids. This data set examined                                     small proteins, future approaches concentrate <br /> for conserved proteins related kinetoplastids repre-                                  identification T. brucei-specific ORFs through <br /> sentative eukaryotes. A similar strategy applied                                      MS analysis ribosome profiling [52,53]. <br /> yeast, 60% 299 small proteins                                           Examining functional importance small proteins <br /> identified significant similarities annotated pro-                                  yeast, revealed 22 (15.5%) 140 sORF knockout <br /> teins eukaryotes, including humans [19]. Likewise,                                  cell lines essential phenotype specific growth <br /> an analysis M. <span id='am-123' about='obo:NCBITaxon_112137' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_301955 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126'  ></span><span id='am-127' property="rdfs:label" content="Musculus" datatype="xsd:string"></span>musculus</span> small proteome revealed                                       conditions [19] Arabidopsis, overexpression of <br /> that two-thirds potential sORFs identified con-                                  10% handpicked set 500 small proteins <br /> served rat 50% conserved human [17]. In                                       resulted abnormal phenotype [20]. Our results <br /> contrast, 1.3% (13) predicted ORFs T. brucei                                  showed 16.7% ORFs tested essential in <br /> had potential homologues representative eukaryotes,                                       procyclic cells, additional 12% altered normal <br /> with half coding ribosomal proteins. One pos-                                    growth patterns. For example, Tb11.NT.28 essential <br /> sible explanation stark difference evolutionary                                  procyclics bloodstream-form parasites. We <br /> conservation early divergence T. brucei                                      provided evidence small protein 56 <br /> from eukaryotes. We observed comparable lack                                      amino acids likely localized inner membrane of <br /> conservation searching potential homologues                                      mitochondria. In T. brucei, size activity <br /> related kinetoplastid species: 16.3% (161) 987 tran-                                  mitochondria vary dramatically life cycle <br /> scripts predicted ORFs significant homology                                      stages. In bloodstream form, mitochondrial respira- <br /> to annotated proteins six-frame translation                                      tory activity repressed single tubule or- <br /> available genomes. This result somewhat unexpected,                                      ganelle maintained. On hand, procyclic <br /> since half small proteins identified                                     cells mitochondrion forms extensive, branching <br /> Arabidopsis homologues closely related plants                                    network active respiration. Although ex- <br /> [16] 3,000 sORFs con-                                         periments required elucidate function of <br /> served D. <span id='am-158' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-160' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-161'  ></span><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>melanogaster</span> closely related spe-                                    Tb11.NT.28, examination cell cycle progression fol- <br /> cies, D. pseudoobscura [15]. On hand,                                     lowing RNAi induction revealed decreased ability of <br /> has argued conservation closely related spe-                                    cells divide replicated kinetoplasts. In contrast, <br /> cies evidence translation, 173 small proteins                                     Tb10.NT.87, mitochondrial matrix protein 64 amino <br /> identified evolutionary conservation represent                                    acids, essential procyclic- bloodstream- <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                               Page 11 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                      A                                                                                           [64aa] <br />                      Tbr    MFSLTQTWLI-AHWYCGHKFRHRFMRDKRFHPSLQASHDARNRFSKRRHFKTNRWNYQQAYRDMP <br />                      Tcr    MLRLSQPVLI-AHWFCGHKFRHRFMRDKRFHPSFQAAHDARNRFSRRRHFKTNRWNYQQAYRDMP <br />                      Lma    MIFRPTSVSFVAHWFCGHKFRHRFMRDKRFHPSHQAACDARNRFSKRRHFKTNRWNYTQAYKDMP <br />  <br />  <br />                      B                                                      D <br />                                                                                 T   C   N   ER W1 W2 Mito <br />                                                                                                               Tb10.NT.87-GFP <br />  <br />                                                                                                               &#945;-63/P1H3 <br />  <br />                                                                                                               HSP70 <br />  <br />  <br />  <br />                      Tb10.NT.87    Mito      Merge    Hoechst     DIC <br />                                   Tracker <br />  <br />                      C                                                      E <br />                                                                                 0 0.015 0.025 0.04 0.05 0.1 % Digitonin <br />                                                                                                               Tb10.NT.87-GFP <br />                                                       K <br />  <br />                                                                                                               mtHSP70 <br />  <br />  <br />                                                           M                                                   TAO <br />                                   M <br />  <br />  <br />  <br />  Figure 8 Characterization Tb10.NT.87. (A) Sequence conservation Tb10.NT.87 kinetoplastids. Amino acid sequences T. brucei <br />  (Tbr), T. cruzi (Tcr) L. major (Lma) aligned ClustalW conserved residues shaded black boxes similar residues are <br />  shaded grey. (B) Live fluorescence microscopy cells expressing C-terminal GFP-tagged Tb10.NT.87. GFP-signal (green), MitoTracker (red), <br />  GFP MitoTracker merge (yellow), Hoechst (blue) DIC images shown. (C) Immunogold electron microscopy C-terminal HA-tagged <br />  Tb10.NT.87. Mitochondrial tubules indicated letter M kinetoplast (mitochondrial DNA) indicated arrow K. (D) Cellular <br />  fractionation cells expressing C-terminal GFP-tagged Tb10.NT.87. Western blot C-terminal GFP-tagged Tb10.NT.87 (top panel), editing <br />  protein (&#945;-63/P1H3, middle panel) cytoplasmic HSP70 (bottom panel) total (T), cytoplasmic (C), nuclear (N), endoplasmic reticulum (ER) <br />  mitochondria (Mito) fractions. (E) Digitonin extraction analysis cells expressing C-terminal GFP-tagged Tb10.NT.87. Parasites exposed <br />  increasing concentrations detergent digitonin indicated solubilized proteins analyzed Western blot GFP-tagged Tb10. <br />  NT.87 (top panel), mitochondrial HSP 70 (mtHSP70, middle panel), trypanosome alternative oxidase (TAO, panel). DIC, differential <br />  interference contrast. <br />  <br />  <br /> form parasites. Ablation protein procyclics re-                 At time, somewhat puzzling unex- <br /> sulted accumulation cells containing multiple nu-                pectedly low number transcripts matching anno- <br /> clei single kinetoplast, cells small              tated proteins having conservation closely related <br /> no kinetoplast, suggesting kinetoplast replication im-             species identified containing functional ORFs. <br /> paired. A similar scenario observed bloodstream-                    Even small proteins expressed low <br /> form cells RNAi Tb10.NT.87 (data shown).                       levels detectable MS be <br /> The basic nature Tb10.NT.87 (pI 11.5) intriguing                  expressed specific stages life cycle, tempting <br /> and need investigation. Nevertheless,                     speculate T. brucei transcriptome includes a <br /> current data consistent Tb10.NT.87 playing role                 substantial number non-coding RNAs. As small <br /> in kinetoplast replication, essential process               proteins identified essential unique kinetoplas- <br /> both life cycle stages [54]. Finally, Tb11.NT.29, indispens-               tids, new targets block survival of <br /> able procyclic bloodstream stages,                     trypanosomes insect vector and/or mammalian <br /> likely localized cell surface. Two days post-induction              host. The important question tackled the <br /> cells appeared asymmetrical hourglass shape                  mechanism action small proteins. <br /> kinetoplast sequestered smaller half cell, while <br /> the remaining portion contains 1K1N 1K2N. The pro- <br /> portional accumulation zoids 1K2N cells began                  Methods <br /> after cells arose suggested aberrant cytokin-                Standard methods <br /> esis asymmetrical hourglass cells occurred.                       Western blots [55], transfection procyclic cells [56] and <br />                                                                            RNA isolation [56] performed using previously pub- <br /> Conclusions                                                                lished protocols. Oligonucleotides used prepare clones <br /> Our study provides evidence existence import-                  probes Northern blots listed Additional <br /> ance small proteins human pathogen T. brucei.                    file 2: Figure S9. <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                                                   Page 12 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br />                              A                                                      B               Tb10.NT.87                   C <br />                                                                                               Cell Cycle Progression <br />                                        10                                           100% <br />                                                              - tet <br />                      log cells (cumulative)                  + tet                                                                                       Enlarged Cell <br />                                               9                                         80%                              &gt;2N                             1K1N <br />                                                                                                                          1K2N <br />                                               8                                         60% <br />                                                                                                                          1KON <br />                                               7                                         40%                              2K2N                            Enlarged Cell <br />                                                                                                                                                          1K&gt;2N <br />                                                              Tb10.NT.87/PC                                               2K1N <br />                                               6                                         20% <br />                                                   0    1     2 3 4 5           6                                         1K1N           DIC    Hoechst <br />                                                            Days RNAi <br />                                                                                         0% <br />                                                                                                0   1     2 3 4 5 <br />                                                                                                        Days RNAi <br />  <br />  <br />                                                   D                                                        E <br />                                                            kDNA size Tb10.NT.87 knockdown <br />                                                   100% <br />  <br />                                                   80% <br />                                                                                               Absent <br />                                                   60% <br />                                                                                               Asymmetrical <br />                                                   40%                                         Small <br />                                                                                               Normal <br />                                                   20% <br />  <br />                                                       0% <br />                                                              0       1    2    3    4 <br />                                                                      Days RNAi <br />  <br />                                                                                                                 Normal          Small      Small     Absent <br />                                                                                                                  kDNA           kDNA      daughter   kDNA <br />                                                                                                                                            kDNA <br />  <br />  Figure 9 Analysis growth cell cycle progression following RNAi Tb10.NT.87. (A) Tb10.NT.87 RNAi procyclics (PC). Growth for <br />  uninduced (&#8722;tet) induced (+tet) cells shown log scale. (B) Cell cycle progression procyclic cells RNAi Tb10.NT.87. Cells were <br />  scored 24 hours having normal DNA (one kinetoplast nucleus (1K1N), kinetoplasts nucleus (2K1N) two <br />  kinetoplasts nuclei (2K2N)) abnormal DNA content (one kinetoplast nucleus (1K0N), kinetoplast nuclei (1K2N) <br />  nuclei (&gt;2 N)). (C) Example images abnormal morphology noted RNAi Tb10.NT.87. Hoechst staining DIC are <br />  shown. (D) Examination kDNA size Tb10.NT.87 RNAi procyclic cells. Cells scored having normal kDNA size, small kDNA, absent <br />  kDNA asymmetrical kDNA, cell daughter kDNA different sizes, visually comparing kDNA size the <br />  uninduced control. (E) Examples cells normal kDNA, small kDNA (both G1 dividing) absent kDNA. Hoechst DIC images are <br />  displayed. Cell counts described legend Figure 5. DIC, differential interference contrast. <br />  <br />  <br />  <br /> Bioinformatics                                                                                                     The predicted T. brucei proteins scanned do- <br /> The 987 transcripts translated using getorf pro-                                                        mains using NCBI CD-Search Tool (cdsearch/cdd <br /> gram European Molecular Biology Open Software                                                             v3.10; [36]). Transmembrane helices proteins were <br /> Suite setting lower limit 25 aa including                                                          predicted using TMHMM Server v. 2.0 [38,58] and <br /> ORFs contained start stop <span id='am-115' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-118'  ></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> [57].                                                                 presence location signal peptide cleavage sites <br />   We used NCBI BLAST suite (BLAST 2.2.28, [35])                                                              scanned SignalP 4.1 Server [37,59]; both <br /> for protein searches. The predicted 4,699 T. brucei                                                          servers Technical University Denmark. <br /> ORFs used queries blastp search non- <br /> redundant kinetoplastid protein sequences (taxid: 5653),                                                         Mass spectrometry analysis <br /> using e-value cutoff 0.1. T. brucei (taxid: 5691).                                                         Procyclic-form trypanosomes (MiTat 1.4) lysed in <br /> T. b. gambiense (taxid: 31285) T. b. brucei (taxid: 5702)                                                    50 mM Tris (pH 7.3) 4% SDS. To decrease sample <br /> sequences excluded search. Similarly,                                                              complexity, samples filtered directly using differ- <br /> tblastn used search kinetoplastid translated                                                          ent (3 kDa, 10 kDa, 30 kDa 50 kDa molecular weight <br /> nucleotide database e-value cutoff 0.1. The                                                           cut (MWCO)) Amicon Ultracel centrifugal filter units <br /> annotated proteins S. cerevisiae (taxid: 4932), C. ele-                                                       (Millipore, Billerica, MA, USA) strong de- <br /> gans (taxid: 6239), A. thaliana (taxid: 3702), D. melano-                                                        naturing conditions 8 M urea. The filtrate pre- <br /> gaster (taxid: 7227), M. musculus (taxid: 10090) H.                                                          cipitated using 4 volumes ethanol subsequently <br /> sapiens (taxid: 9606) queried strat-                                                          resuspended 20 &#956;l 8 M urea. The samples reduced <br /> egy. All alignments manually inspected verified                                                         1 mM dithiothreitol (DTT) alkylated using <br /> to exclude false positives relaxed threshold.                                                         5 mM iodoacetamide, prior digestion 0.2 &#956;g Lys-C <br />  Ericson et al. BMC Biology 2014, 12:14                                                                          Page 13 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> (Wako, Richmond, VA, USA) hours followed          diluted six, five-fold dilutions. Each dilution <br /> digestion 0.2 &#956;g trypsin (Promega, Madison, WI,           plated 96-well plate clones selected from <br /> USA) overnight. The peptides desalted using             dilutions fewer 30% wells growth. T. <br /> StageTip [60]. For MS analysis, peptides separated     brucei SM Lister 427 bloodstream-form cells [41] were <br /> a nanoflow liquid chromatography EASY-nLC          maintained 37 &#176;C 5% CO2 HMI-9 media supple- <br /> capillary packed Reprosil-C18 (Dr. Maisch)        mented 10% Tet-system approved heat inactivated <br /> acetonitrile gradient 2% 60% flow rate         FBS (Clontech), 100 units/ml penicillin, 100 &#956;g/ml strepto- <br /> 250 nl/minute 230 minutes. The Orbitrap XL mass            mycin, 50 &#956;g/ml gentamicin, 2.5 &#956;g/ml G418. For <br /> spectrometer operated data-dependent acquisi-         bloodstream form transfections, 3 &#215; 107 cells centri- <br /> tion mode performing Top10 MS/MS cycle. Data          fuged, washed quickly Tb-BSF buffer [63], resuspended <br /> analysis MaxQuant version 1.2.0.18 [26]          100 &#956;l Tb-BSF buffer containing 10 &#956;g plasmid DNA <br /> using concatenated database TREU 927 v.2.3 (10,533        transfected protocol X-01 AMAXA Nucleo- <br /> entries, tritrypDB.org) hits generated RNA-     fector&#174;. Transfected cells placed 30 ml pre-warmed <br /> Seq analysis (987 entries, Additional file 1). Enzyme search   HMI-9 media 10-fold serial dilutions plated <br /> specificity set tryptic peptides. Up misclea-   24-well plate. Six hours transfection, pre-warmed <br /> vages peptide allowed. Carbamidomethylation      medium supplemented appropriate selectable drugs <br /> on cysteines set fixed modification, methionine   added. The final concentration selectable markers <br /> oxidation protein N-acetylation considered         2.5 &#956;g/ml phleomycin 5 &#956;g/ml blasticidin. For in- <br /> variable modifications. The search performed       duction hairpin RNAi-resistant construct expression, <br /> initial mass tolerance 6 ppm mass accuracy pre-     10 &#956;g/ml 1 &#956;g/ml doxycycline added to <br /> cursor ion 0.5 Da. False discovery rate fixed   procyclic- bloodstream-form cells, respectively. <br /> percent peptide protein level. For second data <br /> set [22], reanalyzed previously generated MS data using     Plasmid constructions <br /> MaxQuant standard settings described            Constructs encoding hairpin RNAs <br /> concatenated database. The MS proteomics data        We followed Gateway&#174;-adapted cloning scheme de- <br /> deposited ProteomeXchange Consortium [61]       veloped Margaret Phillips colleagues [40]. Briefly, <br /> PRIDE partner repository [62] dataset identifier      300 400 base pair region PCR amplified and <br /> PXD000711 [22] PXD000712 (this study).                     TA-cloned (deoxythymidine, T, deoxyadenosine, A) into <br />                                                                plasmid pCR/8GW/TOPO (Invitrogen, Grand Island, <br /> Cell culture                                                   NY, USA) generate entry clone. The entry clone <br /> T. brucei 29.13.6 Lister 427 procyclic cells [41] cul-    recombined destination vector <br /> tured 28&#176;C 5% CO2 Cunningham&#8217;s media sup-           pTrypRNAiGate. Final constructs verified re- <br /> plemented 10% Tet-system approved heat inactivated        striction enzyme digestions DNA sequencing. <br /> fetal bovine serum (FBS, Clontech, Mountain View, CA, <br /> USA), 2 mM L-glutamine, 100 units/ml penicillin, 100 &#956;g/ml     RNAi-resistant constructs <br /> streptomycin, 50 &#956;g/ml gentamicin, 15 &#956;g/mL G418               To rescue lethal RNAi phenotype, RNAi-resistant <br /> and 50 &#956;g/ml hygromycin B. A total 1 &#215; 107 29.13.6 cells    versions CDS synthesized GeneWiz, Inc, <br /> were used procyclic-form transfection. Cells     Cambridge, MA, USA. A silent mutation introduced <br /> spun down, washed Cytomix (20 mM KCl, 0.15 mM               twelfth nucleotide [44], C-terminal HA-TEV- <br /> CaCl2, 10 mM K2HPO4, 25 mM 4-(2-hydroxyethyl)pipera-           FLAG tag added CDS flanked 5&#8217; <br /> zine-1-ethanesulfonic acid (Hepes), 2 mM ethylenedi-           3&#8217; end Hind III Bam HI restriction site, re- <br /> aminetetraacetic acid (EDTA) 5 mM MgCl2, pH7.6),           spectively. The RNAi-resistant construct cloned into <br /> then resuspended 500 &#956;l Cytomix. Then, 25 &#956;g linear-     inducible pLew100v5 BSR plasmid flanked the <br /> ized plasmid DNA added solution cells      GPEET procyclin 5&#8217; UTR aldolase 3&#8217;UTR. In <br /> pulsed twice 1,600 V time constant 0.6 ms    addition, inducible GFP-tagged RNAi-resistant constructs <br /> GenePulser Xcell (BioRad, Hercules, CA, USA). Cells       generated pLew100v5 BSR. The <span id='am-180' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-181' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> contain- <br /> allowed rescue 24 hours selective drug       ing RNAi-resistant constructs digested Not <br /> added. Phleomycin blasticidin added final        I allow recombination T. brucei ribosomal <br /> concentration 2.5 &#956;g/ml 10 &#956;g/ml, respectively.         RNA spacer region following transfection. For each <br /> Transfected cells cloned 24 hours trans-   knock-down, clones tested phenotype. <br /> fection. Serial dilutions cells in <br /> media 20% serum presence 3 &#215; 107 29.13.6       Northern blotting <br /> cells transfected. A total 200 &#956;l      Total RNA separated 1.5% agarose gels the <br /> transfected cells added 1.8 ml cloning media    presence 6.3% formaldehyde 40 mM 3- <br />  Ericson et al. BMC Biology 2014, 12:14                                                                         Page 14 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> morpholinopropane-1-sulfonic acid (<span id='am-168' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-169' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-171'  ></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>MOPS</span>) 2 mM             rabbit (Invitrogen)) used 1:1,000 dilution. <br /> EDTA. RNA ladder 0.5 10 kb (Invitrogen) used        Samples washed times 10 minutes total. <br /> marker. The RNA transferred overnight Hybond-N       Wash solution removed FluorSave (Calbiochem, <br /> nylon membrane (GE Healthcare, Little Chalfont, United        Darmstadt, Germany) added. Next, coverslip was <br /> <span id='am-155' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-156' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>) capillary transfer 10&#215; SSC (0.15 M so-       placed slide, FluorSave reagent dried two <br /> dium citrate, 1.5 M sodium chloride), UV cross-linked      hours overnight cells imaged Zeiss <br /> the membrane stained methylene blue. The mem-        (Jena, Germany) Axioplan 2 fluorescence microscope. For <br /> brane pre-hybridized hour 5&#215; SET (0.75 M       kDNA size estimation, cells fixed stained with <br /> sodium chloride, 5 mm EDTA, 0.15 M Tris-HCl, pH 7.4),         Hoechst, following protocol outlined above. Cells were <br /> 10&#215; Denhardt&#8217;s solution, 1% SDS 100 &#956;g/ml yeast           imaged kDNA size compared visually between <br /> RNA, hybridized overnight solution.      un-induced cells induced cells. <br /> DNA probes internally labeled synthesis with <br /> specific dsDNA templates, sense antisense primers     MitoTracker <br /> Pfx DNA polymerase (Invitrogen) presence            A total 1 &#215; 107 cells spun down, washed in <br /> [&#945;-32P]dCTP. The membrane washed             cell wash (20 mM Tris&#8211;HCl (pH 7.5), 100 mM NaCl <br /> times (10 minutes each) 2&#215; SSC, 0.1% SDS             3 mM MgCl2) resuspended Cunningham&#8217;s <br /> hybridization signals detected PhosphorImager.        media added serum. MitoTracker Red CM- <br />                                                               H2xRos (Invitrogen) added final concentration <br /> Semi-quantitative reverse transcriptase (RT)-PCR              1 &#956;M. Cells incubated 28&#176;C 5% CO2 for <br /> For sample, 5 &#956;g total RNA treated 2 units   10 15 minutes, centrifuged, washed placed in <br /> of DNase RQ (Promega), phenol extracted ethanol pre-      fresh media serum. Cells rescued media <br /> cipitated. DNase-treated RNA reverse transcribed          MitoTracker 25 minutes, Hoechst <br /> using random primers (Promega) Superscript II en-         (5 &#956;g/ml) added; cells spun finally <br /> zyme (Invitrogen) according manufacturer&#8217;s protocol.   resuspended 20 50 &#956;l PBSG. <br /> Twenty-two cycles PCR performed using <br /> Platinum Pfx (Invitrogen), according manufacturer&#8217;s    Live cell imaging <br /> instructions, knockdown histone 4 used     A total 5 8 &#215; 106 cells collected, Hoechst <br /> control. An annealing temperature 50&#176;C used    (5 &#956;g/ml) added cells incubated the <br /> oligonucleotides extension time 30 seconds.        dark minutes. Cells centrifuged, washed <br />                                                               phosphate-buffered saline glucose (PBSG), <br /> Immunofluorescence                                            resuspended 20 50 &#956;l PBSG, imaged a <br /> A total 5 8 &#215; 106 cells spun washed       Zeiss Axioplan 2 fluorescence microscope. <br /> twice cell wash (20 mM Tris&#8211;HCl (pH 7.5), <br /> 100 mM NaCl 3 mM MgCl2) placed               Cell fractionations <br /> on slides coated poly-L-lysine settling     Cell compartment qproteome kit (Qiagen) <br /> minutes. Then, 4% paraformaldehyde (PFA) added            Each fractionation used 1 &#215; 109 cells expressing GFP- <br /> and cells fixed 30 minutes 4&#176;C. Cells    tagged Tb11.NT.29 manufacturer&#8217;s (Qiagen, <br /> washed twice exposed 0.1% NP-40 detergent         Venlo, Limburg, The Netherlands) instructions were <br /> in solution 2% goat serum. Slides washed      followed. The anti-HSP70 anti-BiP antibodies were <br /> and blocked 10% goat serum 10 minutes. Upon          generously provided Jay Bangs. <br /> removal blocking solution, primary antibody, diluted <br /> in 2% goat serum, administered hour. Anti-        Mitochondrial isolation qproteome kit (Qiagen) <br /> GFP (Roche, Basel, Switzerland), anti-HA (Covance,            Each fractionation used 1 &#215; 109 cells expressing GFP- <br /> Princeton, NJ, USA), &#945;-GPEET procyclin (Cedarlane         tagged Tb10.NT.87 Tb11.NT.28 manufacturer&#8217;s <br /> labs, Burlington, Ontario, Canada) antibodies ob-        (Qiagen) instructions followed. The antibodies to <br /> tained commercially. The antibodies BiP gener-        REAP TbMP63 generously provided Steve <br /> ously provided Jay Bangs. Cells washed           Hajduk Ken Stuart, respectively. <br /> times, addition secondary antibody and <br /> 5 &#956;g/ml Hoechst (Cell Signaling Technology, Inc,              Digitonin solubilization assay <br /> Beverly, MA, USA). Cells exposed secondary            Cells (1 &#215; 108) expressing GFP-tagged Tb10.NT.87 or <br /> antibody, diluted 2% goat serum, hour. All         HA-tagged Tb11.NT.28 spun assay, <br /> secondary antibodies (<span id='am-16' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Alexa</span> Fluor 488-conjugated goat         washed twice 20 mM sodium phosphate (pH 7.9), <br /> anti-mouse, 594-conjugated goat anti-mouse, 488-              20 mM glucose 0.15 M NaCl, resuspended <br /> conjugated goat anti-rabbit 594-conjugated goat anti-     500 &#956;l SoTE buffer (20 mM Tris&#8211;HCl (pH 7.5), 0.6 M <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                              Page 15 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> sorbitol, 2 mM EDTA). Next, 500 &#956;l SoTE buffer                              essential small protein procyclic cells un-induced (&#8722;) induced (+) for <br /> varying digitonin amounts added sample                               RNAi. Histone 4 (H4) used control. The small protein transcript and <br /> a final concentration detergent 0.015%, 0.025%,                               H4 indicated right panel. Figure S6. Listing seven <br />                                                                                     essential proteins procyclics C-terminal GFP HA tag. All <br /> 0.04%, 0.05% 0.1% [48,49]. Samples incubated                                constructs rescued lethal RNAi phenotype. Figure S7. Western blot <br /> for minutes 4&#176;C followed centrifugation                              analysis GFP-tagged proteins encoded RNAi-resistant construct. <br /> three minutes 5,000 g 4&#176;C. SDS-sample buffer                              Size markers (kDa) shown left. Figure S8. Growth analysis of <br />                                                                                     bloodstream-form cells following RNAi small proteins. Growth <br /> added supernatants samples analyzed                              un-induced (tet-) induced (tet+) cells shown log scale. The <br /> SDS-PAGE Western blotting. The antibodies                                    data based independent experiments. Figure S9. Listing of <br /> TAO mtHSP70 generously provided Minu                                    oligonucleotides used study. <br /> Chaudhuri Jay Bangs, respectively.                                              Additional file 3: Listing 178 transcripts (sheet 178 ORFs) <br />                                                                                     42 transcripts (sheet 42 ORFs) encoding potential ORFs with <br />                                                                                     matching peptides hits Kinetoplastids. Panigrahi et al.: A <br /> Electron microscopy                                                                 comprehensive analysis Trypanosoma brucei mitochondrial proteome. <br /> Samples fixed 4% PFA/0.1% gluteraldehyde PBS                             Proteomics 2009, 9:434&#8211;450 [21]. Butter et al.: Comparative proteomics of <br />                                                                                     life cycle stages stable isotope-labeled Trypanosoma brucei reveals <br /> for 30 minutes followed fixation 4% PFA                           novel components parasite&apos;s host adaptation machinery. Mol Cell <br /> one hour, rinsed PBS, scraped re-suspended 10%                            Proteomics 2013, 12:172&#8211;179 [22]. viv, T. vivax; cru, T. cruzi; Leish, <br /> gelatin. Chilled blocks trimmed, placed 2.3 M su-                           Leishmania; con, T. congolense; evansi, T. evansi. PTM Abbreviations: P, <br />                                                                                     phosphorylation; S, sumoylation; PM, palmitoylation; G, glycosylation. <br /> crose overnight rotor 4&#176;C, transferred aluminum                          Phosphorylation prediction: NetPhos http://www.cbs.dtu.dk/services/ <br /> pins frozen rapidly liquid nitrogen. The frozen                              NetPhos/; Sumoylation prediction: http://sumosp.biocuckoo.org/; <br /> blocks cut Leica Cryo-EMUC6 UltraCut                                  Palmitoylation Prediction: http://csspalm.biocuckoo.org/; Glycosylation <br />                                                                                     Prediction: http://www.cbs.dtu.dk/services/YinOYang/. <br /> 65 nm sections collected using Tokoyasu <br />                                                                                     Additional file 4: T. brucei predicted ORFs conserved representative <br /> method [64] placed carbon/formvar coated grids                               eukaryotes. BLAST bit scores T. brucei predicted ORFs potential <br /> and floated dish PBS immunolabeling. Grids                              homologs representative eukaryotes. Bit scores shown the <br /> were placed section drops 0.1 M ammo-                               accession number represent normalized version raw <br />                                                                                     BLAST alignment score. <br /> nium chloride quench untreated aldehyde groups, then <br />                                                                                     Additional file 5: Survey conserved domains 173 T. brucei <br /> blocked nonspecific binding 1% fish skin gelatin                          predicted ORFs. <br /> PBS. Single labeled grids incubated primary anti- <br /> body mouse anti-HA (Covance) 1:50 dilution, re- <br /> quired rabbit anti-mouse bridge (JacksonImmuno, West                             Abbreviations <br />                                                                                    aa: amino acids; BLAST: Basic Local Alignment Search Tool; CDS: coding <br /> Grove, PA, USA). The secondary antibody 10 nm Pro-                             sequences; dsRNA: double-stranded RNA; EDTA: ethylenediaminetetraacetic <br /> tein A gold (Utrecht Medical Center). All grids rinsed                        acid; ER: endoplastic reticulum; GFP: green fluorescent protein; MS: mass <br /> in PBS, fixed using 1% gluteraldehyde minutes,                            spectrometry; PBS: phosphate-buffered saline; PFA: paraformaldehyde; <br />                                                                                    REAP: RNA-editing associated protein; RNA-Seq: RNA sequencing; RRM: RNA <br /> rinsed transferred UA/methylcellulose drop                          recognition motif; RT-PCR: reverse transcriptase-polymerase chain reaction; <br /> before collected dried. Samples viewed                              snoRNA: small nucleolar RNA; snRNA: small nuclear RNA; sORF: short open <br /> using FEI Tencai Biotwin TEM 80 Kv. Images                               reading frame; TAO: trypanosome alternative oxidase; uORF: upstream open <br />                                                                                    reading frame; UTR: untranslated region. <br /> taken using Morada CCD iTEM (Olympus) software. <br />  <br />                                                                                    Competing interests <br /> Additional files                                                                   The authors declare competing interests. <br />  <br />  Additional file 1: Coding potential 987 transcripts described <br />  Figure 1. Setting lower limit 25 aa, potential ORFs           Authors&#8217; contributions <br />  listed. For example Tb1.NT.1_1 refers novel transcript Tb1.NT.1            ME, MAJ, FB, MM, EU CT participated study design. ME, MAJ and <br />  according nomenclature Kolev et al. [28] _1 indicates ORF           FB performed experiments. CT FB performed bioinformatics <br />  #1. The numbers parenthesis specify CDS.                                   analyses. MM contributed novel instrumentation. ME, MAJ, FB, EU CT <br />  Additional file 2: Figure S1. Length distribution predicted 4,699          contributed data analysis. ME CT wrote manuscript. All <br />  ORFs. Figure S2. Growth analysis small proteins faster        authors read approved final manuscript. <br />  slower growth procyclics following induction RNAi. Figure S3. <br />  Growth analysis seven small proteins essential procyclics <br />  following induction RNAi expression                     Acknowledgements <br />                                                                                    This work supported Public Health Service grants AI28798 and <br />  RNAi-resistant construct. The data based independent <br />  experiments. Figure S4. Northern blot analysis essential ORFs. Northern        AI56333 EU AI043594 CT. ME supported grant the <br />  blots (Tb11.NT.108, Tb10.NT.90, Tb10.NT.86, Tb10.NT.87, Tb11.NT.28    National Institutes Health (T32 AI007404 CT). Work Mann <br />  Tb3.NT.18) seven essential transcripts. Marker sizes kb indicated   laboratory supported Max Planck Society. We thank J. Bangs for <br />                                                                                    providing BiP Hsp70 antibodies, M. Chaudhuri antibodies against <br />  left. The predicted transcript size indicated panel. <br />  Northern blot analysis Tb11.NT.29 Kolev NG, Franklin JB,      alternative oxidase, S. Hajduk REAP antibodies K. Stuart antibodies <br />  Carmi S, Shi H, Michaeli S, Tschudi C: The transcriptome human             TbMP63. We especially thank M. Phillips providing Gateway&#174; <br />  pathogen Trypanosoma brucei single-nucleotide resolution. PLoS Pathog          compatible vector advice guidance. The authors acknowledge <br />                                                                                    Center Cellular Molecular Imaging Yale contribution, P. De <br />  2010, 6:e1001090 [28]. Figure S5. Semi-quantitative RT-PCR analysis RNAi <br />  knockdown. RT-PCR performed transcripts encoding                  Camilli use AMAXA Nucleofector&#174;, advice comments <br />                                                                                    N. Kolev. <br />  Ericson et al. BMC Biology 2014, 12:14                                                                                                                      Page 16 17 <br /> http://www.biomedcentral.com/1741-7007/12/14 <br />  <br />  <br />  <br />  <br /> Author details                                                                          19. Kastenmayer JP, Ni L, Chu A, Kitchen LE, Au WC, Yang H, Carter CD, Wheeler <br /> 1 <br />  Department Epidemiology Microbial Diseases, School Public Health,                 D, Davis RW, Boeke JD, Snyder MA, Basrai MA: Functional genomics of <br /> Yale University, New Haven, CT, USA. 2Department Proteomics Signal                   genes small open reading frames (sORFs) S. cerevisiae. Genome <br /> Transduction, Max Planck Institute Biochemistry, Martinsried, Germany.                   Res 2006, 16:365&#8211;373. <br /> 3 <br />  Department Cell Biology Internal Medicine, School Medicine, Yale             20. Hanada K, Higuchi-Takeuchi M, Okamoto M, Yoshizumi T, Shimizu M, <br /> University, New Haven, CT, USA. 4Current address: San Francisco General                     Nakaminami K, Nishi R, Ohashi C, Iida K, Tanaka M, Horii Y, Kawashima M, <br /> Hospital, Pulmonary &amp; Critical Care, San Francisco, CA, USA. 5Current address:              Matsui K, Toyoda T, Shinozaki K, Seki M, Matsui M: Small open reading <br /> Institute Molecular Biology gGmbH, Mainz, Germany.                                       frames associated morphogenesis hidden plant genomes. <br />                                                                                             Proc Natl Acad Sci U S A 2013, 110:2395&#8211;2400. <br /> Received: 18 December 2013 Accepted: 6 February 2014                                    21. Panigrahi AK, Ogata Y, Zikova A, Anupama A, Dalley RA, Acestor N, Myler PJ, <br /> Published: 19 February 2014                                                                 Stuart KD: A comprehensive analysis Trypanosoma brucei <br />                                                                                             mitochondrial proteome. Proteomics 2009, 9:434&#8211;450. <br />                                                                                         22. Butter F, Bucerius F, Michel M, Cicova Z, Mann M, Janzen CJ: Comparative <br /> References                                                                                  proteomics life cycle stages stable isotope-labeled <br /> 1. Jacquier A: The complex eukaryotic transcriptome: unexpected pervasive                   Trypanosoma brucei reveals novel components parasite&apos;s host <br />     transcription novel small RNAs. Nat Rev Genet 2009, 10:833&#8211;844.                     adaptation machinery. Mol Cell Proteomics 2013, 12:172&#8211;179. <br /> 2. Kageyama Y, Kondo T, Hashimoto Y: Coding vs non-coding: translatability              23. Stern-Ginossar N, Weisburd B, Michalski A, Le VT, Hein MY, Huang SX, Ma M, <br />     short ORFs putative non-coding transcripts. Biochimie                       Shen B, Qian SB, Hengel H, Mann M, Ingolia NT, Weissman JS: Decoding <br />     1981&#8211;1986, 2011:93.                                                                     human cytomegalovirus. Science 2012, 338:1088&#8211;1093. <br /> 3. Dinger ME, Pang KC, Mercer TR, Mattick JS: Differentiating protein-coding            24. Bitton DA, Wood V, Scutt PJ, Grallert A, Yates T, Smith DL, Hagan IM, Miller <br />     noncoding RNA: challenges ambiguities. PLoS Comput Biol 2008,                   CJ: Augmented annotation <span id='am-50' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe genome <br />     4:e1000176.                                                                             reveals additional genes required growth viability. Genetics 2011, <br /> 4. Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B, Feldmann H, Galibert F,              187:1207&#8211;1217. <br />     Hoheisel JD, Jacq C, Johnston M, Louis EJ, Mewes HW, Murakami Y, Philippsen         25. Bitton DA, Smith DL, Connolly Y, Scutt PJ, Miller CJ: An integrated mass- <br />     P, Tettelin H, Oliver SG: Life 6000 genes. Science 1996, 274:547&#8211;563.              spectrometry pipeline identifies novel protein coding-regions the <br /> 5. Carninci P, Kasukawa T, Katayama S, Gough J, Frith MC, Maeda N, Oyama R,                 human genome. PLoS One 2010, 5:e8949. <br />     Ravasi T, Lenhard B, Wells C, Kodzius R, Shimokawa K, Bajic VB, Brenner SE,         26. Cox J, Mann M: MaxQuant enables high peptide identification rates, <br />     Batalov S, Forrest AR, Zavolan M, Davis MJ, Wilming LG, Aidinis V, Allen JE,            individualized p.p.b.-range mass accuracies proteome-wide protein <br />     Ambesi-Impiombato A, Apweiler R, Aturaliya RN, Bailey TL, Bansal M, Baxter              quantification. Nat Biotechnol 2008, 26:1367&#8211;1372. <br />     L, Beisel KW, Bersano T, Bono H, et al: The transcriptional landscape        27. Berriman M, Ghedin E, Hertz-Fowler C, Blandin G, Renauld H, Bartholomeu <br />     mammalian genome. Science 2005, 309:1559&#8211;1563.                                          DC, Lennard NJ, Caler E, Hamlin NE, Haas B, B&#246;hme U, Hannick L, Aslett MA, <br /> 6. Jung HW, Tschaplinski TJ, Wang L, Glazebrook J, Greenberg JT: Priming                 Shallom J, Marcello L, Hou L, Wickstead B, Alsmark UC, Arrowsmith C, Atkin <br />     systemic plant immunity. Science 2009, 324:89&#8211;91.                                       RJ, Barron AJ, Bringaud F, Brooks K, Carrington M, Cherevach I, Chillingworth <br /> 7. Anginot A, Espeli M, Chasson L, Mancini SJ, Schiff C: Galectin 1 modulates               TJ, Churcher C, Clark LN, Corton CH, Cronin A, et al: The genome the <br />     plasma cell homeostasis regulates humoral immune response.                      African trypanosome Trypanosoma brucei. Science 2005, 309:416&#8211;422. <br />     J Immunol 2013, 190:5526&#8211;5533.                                                      28. Kolev NG, Franklin JB, Carmi S, Shi H, Michaeli S, Tschudi C: The <br /> 8. Coso OA, Chiariello M, Yu JC, Teramoto H, Crespo P, Xu N, Miki T, Gutkind                transcriptome human pathogen Trypanosoma brucei at <br />     JS: The small GTP-binding proteins Rac1 Cdc42 regulate activity                 single-nucleotide resolution. PLoS Pathog 2010, 6:e1001090. <br />     JNK/SAPK signaling pathway. Cell 1995, 81:1137&#8211;1146.                         29. Nilsson D, Gunasekera K, Mani J, Osteras M, Farinelli L, Baerlocher L, Roditi I, <br /> 9. Haslbeck M, Franzmann T, Weinfurtner D, Buchner J: Some like hot:                 Ochsenreiter T: Spliced leader trapping reveals widespread alternative <br />     structure function small heat-shock proteins. Nat Struct Mol Biol                splicing patterns highly dynamic transcriptome Trypanosoma <br />     2005, 12:842&#8211;846.                                                                       brucei. PLoS Pathog 2010, 6:e1001037. <br /> 10. Chilley PM, Casson SA, Tarkowski P, Hawkins N, Wang KL, Hussey PJ, Beale            30. Siegel TN, Hekstra DR, Wang X, Dewell S, Cross GA: Genome-wide analysis <br />     M, Ecker JR, Sandberg GK, Lindsey K: The POLARIS peptide Arabidopsis                 mRNA abundance life-cycle stages Trypanosoma brucei and <br />     regulates auxin transport <span id='am-110' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>root</span> growth effects ethylene                       identification splicing polyadenylation sites. Nucleic Acids Res 2010, <br />     signaling. Plant Cell 2006, 18:3058&#8211;3072.                                               38:4946&#8211;4957. <br /> 11. Pi H, Huang YC, Chen IC, Lin CD, Yeh HF, Pai LM: Identification                  31. Veitch NJ, Johnson PC, Trivedi U, Terry S, Wildridge D, MacLeod A: Digital <br />     11-amino acid peptides disrupt Notch-mediated processes                         gene expression analysis life cycle stages human-infective <br />     Drosophila. J Biomed Sci 2011, 18:42.                                                   parasite, <span id='am-173' about='obo:NCBITaxon_31285' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5691 obo:NCBITaxon_5653'><span id='am-174'  ></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma gambiense" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei gambiense" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Trypanosoma brucei gambiense" datatype="xsd:string"></span>Trypanosoma brucei gambiense</span> reveals differentially expressed <br /> 12. Pueyo JI, Couso JP: The 11-aminoacid long Tarsal-less peptides trigger                clusters co-regulated genes. BMC Genomics 2010, 11:124. <br />     cell signal Drosophila leg development. Dev Biol 2008, 324:192&#8211;201.              32. National Center Biotechnology Information. Sequence Read Archive. <br /> 13. Kondo T, Plaza S, Zanet J, Benrabah E, Valenti P, Hashimoto Y, Kobayashi S,             http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi <br />     Payre F, Kageyama Y: Small peptides switch transcriptional activity             33. The EuPathDB Project Team. TriTrypDB: The Kinetoplastid Genomics <br />     Shavenbaby Drosophila embryogenesis. Science 2010,                            Resource. www.tritrypdb.org. <br />     329:336&#8211;339.                                                                        34. Logan-Klumpler FJ, De Silva N, Boehme U, Rogers MB, Velarde G, McQuillan JA, <br /> 14. Magny EG, Pueyo JI, Pearl FM, Cespedes MA, Niven JE, Bishop SA, Couso JP:               Carver T, Aslett M, Olsen C, Subramanian S, Phan I, Farris C, <span id='am-139' about='obo:NCBITaxon_69650' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57612 obo:NCBITaxon_69649 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142'  ></span><span id='am-143' property="rdfs:label" content="Mitra" datatype="xsd:string"></span>Mitra</span> S, Ramasamy G, <br />     Conserved regulation cardiac calcium uptake peptides encoded                   Wang H, Tivey A, Jackson A, Houston R, Parkhill J, Holden M, Harb OS, Brunk BP, <br />     small open reading frames. Science 2013, 341:1116&#8211;1120.                                 Myler PJ, Roos D, Carrington M, Smith DF, Hertz-Fowler C, Berriman M: GeneDB&#8211; <br /> 15. Ladoukakis E, Pereira V, Magny EG, Eyre-Walker A, Couso JP: Hundreds                 annotation database pathogens. Nucleic Acids Res 2012, 40:D98&#8211;D108. <br />     putatively functional small open reading frames Drosophila.                      35. Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ: <br />     Genome Biol 2011, 12:R118.                                                              Gapped BLAST PSI-BLAST: new generation protein database <br /> 16. Hanada K, Zhang X, Borevitz JO, Li WH, Shiu SH: A large number novel                 search programs. Nucleic Acids Res 1997, 25:3389&#8211;3402. <br />     coding small open reading frames intergenic regions                   36. Marchler-Bauer A, Lu S, Anderson JB, Chitsaz F, Derbyshire MK, DeWeese-Scott <br />     Arabidopsis thaliana genome transcribed and/or purifying                       <br /> </body></html>