<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mos1-Mediated Transgenesis Probe Consequences of <br /> Single Gene Mutations Variation-Rich Isolates of <br /> <span id='am-10' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-11' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> <br /> <span id='am-38' about='obo:NCBITaxon_99390' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_6804 obo:NCBITaxon_116706 obo:NCBITaxon_6805 obo:NCBITaxon_197562 obo:NCBITaxon_116704 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-39' property="rdfs:label" content="Maja" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Maia" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Maja</span> Tarailo-Graovac*, Nansheng Chen <br /> Department Molecular Biology Biochemistry, Simon Fraser University, Burnaby, British Columbia, Canada <br />  <br />  <br />  <br />      Abstract <br />      <span id='am-17' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans, especially N2 isolate, invaluable biological model system. Numerous additional natural C. <br />      elegans isolates shown unexpected genotypic phenotypic variations encouraged <br />      researchers use generation sequencing methodology develop complete picture genotypic variations <br />      isolates. To understand phenotypic effects genomic variation (GV) single gene, variation-rich <br />      genetic background, analyze particular GV understood genetic background. In C. elegans, the <br />      analysis usually N2, requires extensive crossing bring GV. <span id='am-22' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26'  ></span>This</span> time consuming <br />      procedure important establish fast efficient approach test effect GVs different isolates N2. <br />      Here use Mos1-mediated single-copy insertion (MosSCI) method phenotypic assessments GVs variation- <br />      rich Hawaiian strain CB4856 N2. Specifically, investigate effects variations identified CB4856 strain tac-1 <br />      essential gene necessary mitotic spindle elongation pronuclear migration. We the <br />      usefulness MosSCI method using EU1004 tac-1(or402) control. or402 temperature sensitive lethal allele <br />      well-conserved TACC domain (transforming acidic coiled-coil) results leucine phenylalanine change at <br />      amino acid 229. CB4856 contains variation affects second exon tac-1 causing cysteine tryptophan change <br />      amino acid 94 TACC domain. Using MosSCI method, analyze tac-1 CB4856 N2 <br />      background demonstrate C94W change, albeit significant, does cause obvious decrease viability. <br />      This MosSCI method proven rapid efficient way analyze GVs. <br />  <br />   Citation: Tarailo-Graovac M, Chen N (2012) Mos1-Mediated Transgenesis Probe Consequences Single Gene Mutations Variation-Rich Isolates of <br />   Caenorhabditis elegans. PLoS ONE 7(11): e48762. doi:10.1371/journal.pone.0048762 <br />   Editor: Denis Dupuy, Inserm U869, France <br />   Received June 18, 2012; Accepted October 3, 2012; Published November 14, 2012 <br />   Copyright: &#223; 2012 Tarailo-Graovac, Chen. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research funded Canadian Institutes Health Research (CIHR) Fanconi <span id='am-1' about='obo:NCBITaxon_12939' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3290 obo:NCBITaxon_33090 obo:NCBITaxon_693762 obo:NCBITaxon_131221 obo:NCBITaxon_693766 obo:NCBITaxon_58023 obo:NCBITaxon_241806 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Anemia Sw." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Anemia" datatype="xsd:string"></span>Anemia</span> Fellowship Maja Tarailo-Graovac, the <br />   Discovery Grant Natural Science Engineering Research Council (NSERC) Canada Nansheng Chen. The funders role study design, <span id='am-44' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br />   collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: mta57@sfu.ca <br />  <br />  <br />  <br /> Introduction                                                                         extensively compared N2 genetically phenotypically. <br />                                                                                      In addition large number genome variations (small and <br />    C. elegans central biomedical, molecular, cell de-                      large changes DNA sequence) N2 CB4856 <br /> velopmental biology research, best genetically                      [20,21,22,23,24], number phenotypic differences the <br /> and molecularly characterized understood model organisms.                        strains described. For example, CB4856 contains <br /> The studied best understood C. elegans strain N2,                    multiple variations PAZ/PIWI domain-containing protein <br /> which obtained mushroom compost Bristol, England                         (ppw-1) renders Hawaiian strain resistant effecting <br /> [1]. The genome N2 multi-cellular animal                   germline-expressed genes feeding dsRNA directed against <br /> that fully sequenced published [2]. While N2                        genes [25]. Recently CB4856 resistant to <br /> been widely used research model organism past 40                     avermectins four-amino-acid deletion ligand- <br /> years [3] C. elegans wild-strains isolated globally                  binding domain GLC-1, alpha-subunit glutamate-gated <br /> from human-associated habitats rotting fruits compost                    chloride channel [26]. Identification naturally occurring <br /> heaps [3,4]. With goal reaching better understanding                     four-amino-acid deletion GLC-1 Hawaiian strain the <br /> genotypic phenotypic differences strains,                  genetic evidence mechanism nematode resistance to <br /> as studying relationships genetic interactions, wild                 anthelmintics, type resistance, nematode <span id='am-32' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, <br /> isolates subjected [5] partial genome                   represents major global health agricultural problem [26]. <br /> sequencing [4]. Genetic studies different C. elegans wild-strains                 Genome variations (GVs) phenotypic differences <br /> [4,6,7,8,9] revealed little genetic diversity compared                       strains identified traits <br /> closely related species [4,7,8,9,10,11,12,13,14,15,16,17], com-                  including egg-laying behaviour vulva development not <br /> parable genetic diversity human populations [18].                           identified genetic bases. To identify variation responsible <br />    CB4856, isolated 1972 pineapple field                      particular phenotype variation-rich natural isolate like <br /> Hawaii [19], wild-isolate strain                           CB4856, typically introduce GVs N2 genetic <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                          November 2012 | Volume 7 | Issue 11 | e48762 <br />                                                                                                             MosSCI Study Single Gene Mutations <br />  <br />  <br /> background follow observable trait <span id='am-35' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> map               GATTTTAATAG. Amplicons cloned pCFJ178 <br /> a chromosomal region. Alternatively, genome sequencing                  vector, described previously [27]. Similar previous <br /> allows candidate gene approach candidate                     findings using essential cell cycle gene cyb-3 [28,29], have <br /> variation particular trait introduced                  tac-1, injected high concentrations, results <br /> understood N2 background extensive outcrossing scored               toxic effect. Thus obtain stable single-copy insertions, we <br /> for visible phenotype. However, outcrossing procedure               co-injected 5 ng/ml tac-1-targetting constructs 50 ng/mL of <br /> be extremely time consuming particularly difficult               pJL43.1, 5 ng/mL pGH8, 5 ng/mL pCFJ104 2.5 ng/mL <br /> phenotypes subtle GVs followed using                pCFJ90 gonad 32 (tac-1CB4856) 38 (tac-1EU1004) <br /> PCR PCR followed restriction enzyme digests. Overcoming                 young adult P0 EG6250 hermaphrodites. The plates that <br /> these difficulties establishing fast efficient approaches         contained wild-type looking mCherry expressing worms were <br /> analyzing phenotypes candidate GVs variation-rich                 starved 25uC screened stable integrants as <br /> strains N2 background essential analysis phenotypic          previously described [27]. Within weeks multiple stable lines <br /> differences strains.                                              obtained construct. One confirmed to <br />    Mos1-mediated single-copy insertion (MosSCI) recently                 contain single, truncation-free, integration cxTi10882 site. <br /> developed method C. elegans allows integration transgenes          These analyzed JNC150 (tac-1CB4856) JNC152. <br /> as single copies defined genomic site [27]. MosSCI eliminates            (tac-1EU1004). Neither strains obvious increase in <br /> problems associated common methods generating                        lethality developmental delay observed 14uC, 20uC <br /> transgenes C. elegans, including concatenation injected                 25uC (Table 1 data shown). <br /> DNA, formation multicopy arrays over- <br /> expressed somatic cells silenced germline [27].                 Analysis ok3305 knockout allele <br /> Furthermore genetically neutral Mos1 insertion alleles exist                 ok3305 812bp deletion removes majority tac-1 <br /> allow expression transgenes endogenous levels [27,28].                  gene. Previously, loss TAC-1 mainly studied using <br /> Recently demonstrated powerful aspect MosSCI                      RNAi deplete tac-1 product [30,31,32]. In absence of <br /> method allows increase gene copy number controlled            TAC-1 progeny arrest embryos defective microtubule <br /> fashion analyze consequence doubling [28] tripling              formation [30,31,32]. The knockout allele, tac-1(ok3305), also <br /> [29] gene dosage animal development different genetic                   results lethality kept balanced heterozygote <br /> backgrounds. In study examined genome variations                  translocation (mT1). However, stage tac- <br /> affecting essential gene CB4856 behave N2 using                1(ok3305) homozygotes arrest determined previously. <br /> MosSCI method. We examined GVs affecting essential gene                    In study, analyzed VC2580 determine ok3305&#8217;s <br /> tac-1 [30,31,32,33]. We demonstrated power method               phenotype. We VC2580 segregates approximately <br /> phenocopying lethal phenotype tac-1(or402) EU1004                 63% arrested embryos mT1 translocation aneuploidies, <br /> strain [33]. Importantly, showed non-synonymous                   ,6% mT1 homozygotes Dpy sterile, ,6% wild- <br /> radical change essential TACC domain does cause                type looking tac-1(ok3305) progeny 25% tac-1(ok3305)/ <br /> an apparent decrease viability N2 background. The                   mT1 II; +/mT1[dpy-10(e128)] III heterozygotes. Analysis the <br /> protocol fast efficient analysis single-       ,6% tac-1(ok3305) homozygotes, segregated tac- <br /> gene variations variation-rich C. elegans strains the <br />                                                                               1(ok3305)/mT1 II; +/mT1[dpy-10(e128)] III heterozygous her- <br /> extensively studied N2 background. Furthermore, protocol is <br />                                                                               maphrodites, revealed tac-1(ok3305) homozygotes produce <br /> effective ruling lack readily detectable phenotypes to <br />                                                                               progeny 100% arrest embryos (Table 1). This is <br /> a presence putative modifiers. <br />                                                                               common phenotype maternal effect genes. Namely, F1 tac- <br />                                                                               1(ok3305) homozygotes likely receive TAC-1 protein tac- <br /> Materials Methods                                                         1(ok3305)/mT1 II; +/mT1[dpy-10(e128)] heterozygous hermaph- <br />                                                                               rodites allows develop adult animals. However, <br /> Strains Culturing Conditions <br />                                                                               F2 generation tac-1(ok3305) homozygotes does any <br />    The following mutant alleles used work: unc- <br />                                                                               wild-type TAC-1, leads 100% embryonic arrest. This <br /> 119(ed3), cxTi10882, or368/or402, ok3305, dpy-10(e128), mT1. <br />                                                                               phenotype similar phenotype observed RNAi is <br /> The following strains used work: N2 (Bristol strain as <br />                                                                               used deplete TAC-1 [30,31,32]. Thus, conclude TAC-1 <br /> a wild-type), EU1004 [tac-1(or402) II], CB4856, EG6250 [unc- <br />                                                                               likely maternally supplied loss TAC-1 results in <br /> 119(ed3) III; cxTi10882 IV] VC2580 [tac-1(ok3305)/mT1 II; +/ <br /> mT1[dpy-10(e128)] III ]. The alleles dotSi120 dotSi121,               maternal effect embryonic lethality. <br /> JNC150 [dotSi120 IV [Y54E2A.3CB4856 + unc-119(+)]], JNC151 <br /> [(tac-1(ok3305) II; dotSi120 IV [Y54E2A.3CB4856 + unc-119(+)]],               Use ok3305 knockout allele phenotypic analysis <br /> JNC152 [dotSi121 IV [Y54E2A.3EU1004 + unc-119(+)]] JNC153                 tac-1 GVs <br /> [tac-1(ok3305) II; dotSi121 IV [Y54E2A.3EU1004 + unc-119(+)]]                   First, generated JNC150 JNC152 males heat shock. <br /> strains generated study. All strains maintained             These males mated tac-1(ok3305) homozygotes to <br /> using standard protocol nematode growth media (NGM) plates                 generate JNC151 [(tac-1(ok3305) II; dotSi120 IV [Y54E2A.3CB4856 + <br /> seeded OP50 bacteria [34]. Strains maintained 20uC               unc-119(+)]] JNC153 [tac-1(ok3305) II; dotSi121 IV <br /> while phenotypic analyses performed 14uC 25uC                [Y54E2A.3EU1004 + unc-119(+)]]. Unlike tac-1(ok3305) homozy- <br /> as noted manuscript.                                                   gotes, arrest embryos, JNC151 homozygotes are <br />                                                                               indistinguishable N2 (Table 1), JNC153 homozygotes <br /> Mos1-mediated transgenesis                                                    indistinguishable EU1004 (Table 1). <br />    The tac-1 locus amplified using Phusion (NEB), high-fidelity <br /> DNA polymerase CB4856 EU1004 single worm                       Phenotypic analysis <br /> lysates. The following primers used: FORWARD- AAACTAT-                    For analysis, L4 hermaphrodites grown fresh <br /> TACCTTCGCCTTCGC REVERSE-CTGGAAAATTGCAA-                                   OP50 plates 14uC 25uC. The hermaphrodites were <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                2                      November 2012 | Volume 7 | Issue 11 | e48762 <br />                                                                                                            MosSCI Study Single Gene Mutations <br />  <br />  <br />  <br />  Table 1. Phenotypes tac-1 alleles. <br />  <br />  <br />  Genotypes                                                              Developmental arrests 14uC (%)            Developmental arrests 25uC (%) <br />  <br />  N2 (reference)                                                         0.8 (n = 1441)                            1.4 (n = 1987) <br />  dotSi120 IV [Y54E2A.3CB4856 + unc-119(+)]                              0.6 (n = 1596)                            1.2 (n = 1902) <br />  dotSi121 IV [Y54E2A.3EU1004 + unc-119(+)]                              0.4 (n = 1844)                            1.5 (n = 1157) <br />  EU1004 [tac-1(or402) II]                                               64.9 (n = 336)                            94.9 (n = 295) <br />  tac-1(ok3305) II (F2)                                                  100 (n = 445)                             100 (n = 394) <br />  tac-1(ok3305) II; dotSi120 IV [Y54E2A.3CB4856 + unc-119(+)]            0.6 (n = 1647)                            1.2 (n = 1714) <br />  tac-1(ok3305) II; dotSi121 IV [Y54E2A.3EU1004 + unc-119(+)]            61.2 (n = 276)                            94.4 (n = 697) <br />  <br />  doi:10.1371/journal.pone.0048762.t001 <br />  <br />  <br /> transferred fresh plates 12 hours. Eggs did hatch         CB4856 tac-1 N2 background using MosSCI <br /> were scored embryonic arrest, eggs hatched did            method (Figures 1 2). <br /> not grow adulthood scored larval arrest. Together,                  The MosSCI method relies presence Mos1 insertion <br /> embryonic larval arrest reported developmental arrest            specific locus C. elegans genome [27]. A large collection of <br /> in Table 1.                                                                 mapped Mos1 insertion alleles generated the <br />                                                                             European NEMAGENETAG consortium [35]. Many the <br /> Results Discussion                                                      Mos1 insertions identified &#8216;&#8216;genetically neutral&#8217;&#8217; (inserted <br />                                                                             genomic regions 39 coding genes) shown have <br />    Recently used whole-genome sequencing identify GVs                 robust germline expression [27,36]. Previously, successfully <br /> that disrupt protein-coding genes CB4856 wild-                used ttTi5605 [28] cxTi10882 [29] Mos1 insertions, which <br /> isolate strain C. elegans Hawaii (Vergara, Tarailo-Graovac,         located center chromosomes II IV respectively, to <br /> et al. preparation). We particularly interested variations <br />                                                                             study effects cyb-3 dosage C. elegans development and <br /> that expected cause significant disruption essential <br />                                                                             anaphase onset [28,29]. Neither Mos1 insertions <br /> protein-coding genes. One variation identified tac- <br />                                                                             interfered proper function inserted cyb-3 gene <br /> 1 wanted test phenotype variation N2 <br />                                                                             [28,29]. For analysis GVs affecting tac-1, selected the <br /> background. tac-1 essential gene member the <br />                                                                             cxTi10882 Mos1 insertion located different <br /> transforming acidic coiled-coil (TACC) protein <span id='am-7' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> C. elegans <br />                                                                             chromosome tac-1 [27,36]. Since cxTi10882 located the <br /> whose function crucial pronuclear migration mitotic <br />                                                                             centre chromosome IV natural tac-1 position distal <br /> spindle elongation [30,31,32]. In CB4856 identified number <br />                                                                             end chromosome II, inserting tac-1 cxTi10882 Mos1 site <br /> of single nucleotide changes affect tac-1. In particular, one <br />                                                                             place different genomic environment [37]. Namely, <br /> variation affected second exon tac-1 causing C94W change <br /> in amino acid sequence (Figure 1a). To date, point                individual autosomes chromosome arms centers <br /> mutant alleles tac-1 isolated genetic screens           differ important properties including content highly <br /> temperature-sensitive mutants using EMS mutagenesis [33]                 expressed genes, repetitive elements, chromatin composition <br /> addition knockout allele. Both tac-1(or369) tac-1(or402)         [37]. To test cxTi10882 Mos1 insertion site method we <br /> have mutation TACC domain results                   asked temperature sensitive phenotype tac-1(or402) <br /> a L229F amino acid change [33]. tac-1(or455) M58I change              allele phenocopied using approach. We amplified tac- <br /> in TACC domain [33]. Both amino acid changes occur               1 including 59 sequence immediately upstream predicted <br /> within residues highly conserved, impact            ATG initiator site 39 sequence immediately downstream of <br /> these mutations significant [33]. Both tac-1(or369/or402) tac-       predicted stop <span id='am-27' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span>, EU1004 (Figure 2a). Then, we <br /> 1(or455) temperature sensitive alleles result .95%               cloned tac-1 amplicon pCFJ178 vector inserted <br /> embryonic arrest restrictive temperature [33]. Similar        transgene cxTi10882 Mos1 integration site (Figure 2a). <br /> point mutations isolated tac-1, C94W change occurs            The strain generated contains N2 copy tac-1 located <br /> essential TACC domain (Figure 1a) does affect                  endogenous position chromosome II, the <br /> a highly conserved residue (Figure 1b). Since C94W variation            chromosome IV integrated copy EU1004 tac-1 (dotSi121) that <br /> results cysteine, polar amino acid, replaced           encodes L229F change (Figure 2a). To uncover effects of <br /> the non-polar amino acid tryptophan, C94W change               L229F change, tac-1 (dotSi121) analyzed absence <br /> considered radical L229F M58I                   endogenous tac-1 gene product. <br /> changes, CB4856 animals temperature sensitive                tac-1(ok3305) 812 bp deletion removes majority of <br /> phenotype EMS-derived point mutants do. Thus,                tac-1 (part exon one, exons three) likely be <br /> interested lack phenotype               null mutation. To analyze ok3305 performed detailed <br /> a presence modifying mutations CB4856 simply               phenotypic analysis (see MATERIALS AND METHODS). Our <br /> the C94W change does affect TAC-1 function                    analysis revealed tac-1(ok3305) results maternal effect <br /> L229F M58I changes do. Inspired enormous potential               embryonic arrest. Namely, F1 tac-1(ok3305) homozygotes likely <br /> of recently developed MosSCI method [27], allowed              receive wild-type TAC-1 protein tac-1 heterozygous P0s that <br /> to doubling dosage Cyclin B3 C. elegans             allows develop adult animals. However, F2 <br /> bypasses need functional spindle-assembly checkpoint            homozygotes wild-type TAC-1, leads to <br /> component MDF-1/Mad1 survival genera-                  100% embryonic arrests (Table 1). The embryonic arrest <br /> tion [28,29], decided investigate effect GVs detected        phenotype putatively identical phenotype observed when <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              3                       November 2012 | Volume 7 | Issue 11 | e48762 <br />                                                                                                             MosSCI Study Single Gene Mutations <br />  <br />  <br />  <br />  <br /> Figure 1. The C94W change TACC domain. (a) Schematic representation TAC-1 (1 260 amino-acid sequence). The majority of <br /> the protein composed TACC domain depicted cyan box. Location nature of point mutants identified date are <br /> shown well. The ok3305 knockout allele, removes majority tac-1 depicted using pink box. (b) The multiple sequence alignment of <br /> TAC-1 adopted Bellanger et al. 2007 [33]. The positions previously isolated point mutants tac-1(or455) tac-1(or369/402) are <br /> depicted using red stars, C94W change identified CB4856 depicted using red arrow. The cyan box highlights presence the <br /> TACC domain. All known point mutations occur TACC domain, affect conserved amino acids. <br /> doi:10.1371/journal.pone.0048762.g001 <br />  <br />  <br />  <br />  <br /> Figure 2. A single-copy transgene insertion used investigate consequences single gene mutations variation-rich isolates of <br /> C. elegans. (a) Analysis tac-1(or402) using Mos1-mediated transgenesis. tac-1, including 59 39 regulatory sequences, amplified using <br /> high-fidelity DNA polymerase EU1004 genomic DNA cloned pCFJ178 vector. The red dotted line located exon tac-1 <br /> depicts or369/402 A G change results L229F amino acid change. Once cloned pCFJ178 vector, transgene inserted <br /> into cxTi10882 Mos1 (depicted orange) integration site chromosome IV (depicted purple). The resulting JNC152 strain contains the <br /> endogenous copy tac-1 located chromosome II (depicted blue), tac-1 isolated EU1004 inserted chromosome IV, dotSi121. To <br /> uncover effect tac-1(or402), dotSi121 examined absence endogenous TAC-1 using ok3305. Thus, constructed JNC153. (b) <br /> Schematic representation method used investigate consequences tac-1 variations detected CB4856 tac-1, including 59 39 <br /> regulatory sequences, amplified using high-fidelity DNA polymerase CB4856 genomic DNA cloned pCFJ178 vector. The red <br /> dotted lines represent single nucleotide changes detected CB4856 tac-1. Then, transgene inserted cxTi10882 Mos1 (depicted in <br /> orange) integration site chromosome IV (depicted purple). JNC150 contains endogenous copy tac-1 located chromosome II <br /> (depicted blue) tac-1 isolated CB4856 dotSi120 inserted chromosome IV. Then, dotSi120 analyzed absence endogenous <br /> tac-1(ok3305). (c) PCR bands expected size (6kb) stably integrated single copy insertions tac-1, dotSi121 dotSi120. <br /> doi:10.1371/journal.pone.0048762.g002 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                       November 2012 | Volume 7 | Issue 11 | e48762 <br />                                                                                                                          MosSCI Study Single Gene Mutations <br />  <br />  <br /> RNAi used deplete TAC-1 [30,31,32]. Thus, conclude                         phenotypic variations individuals, especially disease <br /> that ok3305 presumably null allele tac-1.                                   susceptibility treatment. To analyze impact GVs on <br />    Next, analyzed tac-1 (dotSi121) absence                         single gene test presence putative modifiers due <br /> endogenous tac-1 product using ok3305 knockout allele                        natural variations, needs analyze causative variations in <br /> (Figure 2a). Our analysis revealed original strain EU1004                  understood established genetic backgrounds. Here, we <br /> tac-1(or368/or402) II strain created using                      fast efficient approach analyzing GVs from <br /> MosSCI method JNC153 tac-1(ok3305) II; dotSi121 IV                                  variation-rich strains well-understood C. elegans N2 <br /> [Y54E2A.3EU1004 + unc-119(+)] indistinguishable (Table 1).                      background. Using approach, based MosSCI method, <br /> Phenocopying lethal phenotype L229F change strongly                      desired strains usually generated weeks. <br /> supports use outlined method analysis variations                  Alternative approaches relay extensive outcrossing usually <br /> affecting single gene.                                                            months. In addition time-consuming nature the <br />    Most heritable traits, including different susceptibility disease             alternative approaches, analysis complicated by <br /> and different responses drug treatments, genetically                         involvement loci variation rich-strains. For <br /> complex, resulting contributions mutations                          example, analysis CX11307 JU751 wild-isolate strains <br /> different genes [38]. Using model organisms, shown                      abamectin resistance, suggested presence putative <br /> that majority phenotypes [39] genes [40]                         dominant resistance factor addition glc-1 variation [26]. <br /> phenotypic consequence identical GVs affected                             Also, analysis resistance dsRNA directed against <br /> modifiers, variations present loci genome                    germline-expressed genes [25] CB4856 suggested the <br /> organism [41]. To determine lack phenotype                       presence modifier allele CB4856 addition <br /> be presence modifiers CB4856 simply                   ppw-1 mutation [25]. Furthermore, case tac-1, an <br /> cysteine 94 residue essential TAC-1 function,                         alternative analysis relies extensive outcrossing place <br /> analyzed CB4856 GVs affecting tac-1 N2 background. In                        mutations N2 difficult lack of <br /> addition variation results C94W amino acid <br />                                                                                     obvious phenotypes fact C94W point mutation <br /> change (Figure 1), tac-1 mutations located <br />                                                                                     followed easily using PCR PCR followed by <br /> upstream located downstream gene (Figure 2b). To <br />                                                                                     restriction enzyme digest. Using MosSCI method, were <br /> rule possibility mutations affect the <br />                                                                                     able show, time-effective manner cysteine to <br /> expression CB4856 tac-1, investigated data recent <br />                                                                                     tryptophan change amino acid 94 tac-1 albeit significant, does <br /> studies analyzed differences gene expression N2 <br />                                                                                     cause decrease viability N2 background. This <br /> and CB4856 [42,43]. Based data, tac-1 does appear to <br />                                                                                     result suggests lack phenotype CB4856 the <br /> be differentially expressed CB4865. Thus, determine the <br /> effect C94W change N2 background, decided                       presence putative modifiers. Instead, cysteine amino acid 94 <br /> amplify CB4856 tac-1, including 59 39 sequences. We                         play important role function essential <br /> cloned amplicon pCFJ178 vector inserted                        TACC domain TAC-1. The usefulness approach that <br /> transgene cxTi10882 Mos1 integration site (Figure 2b).                     manuscript lies ability applied to <br /> Then, analyzed tac-1 (dotSi120) encodes C94W change                     engineer amino acid changes already <br /> in absence endogenous tac-1 gene product using                           existing variations background another. In way, <br /> knockout allele (ok3305) (Figure 2b). Unlike L229F, C94W does                       analyze effect specific amino acid the <br /> not obvious effect viability 99.4% 98.8%                    development viability C. elegans altering particular <br /> of embryos analyzed 14uC 25uC respectively develop                       amino acid. Furthermore, using approach, genes from <br /> into adults, similar observed N2                  different Caenorhabditis species analyzed without <br /> (Table 1). Thus, able show, time-effective                     having deal over-expression issues. <br /> manner, variations affecting CB4856 tac-1 result an <br /> obvious phenotype expected radical change,                     Acknowledgments <br /> such C94W, affecting essential gene tac-1. The method <br />                                                                                     We like thank Robert Johnsen critical review the <br /> outlined allowed efficiently rule                       manuscript. We like thank C. elegans Gene Knockout <br /> hypothesis lack phenotypic consequence presence                    Consortium generating deletion mutants, Caenorhabditis Genetics <br /> of C94W putative GVs CB4856 genome                       Center (CGC) providing strains Erik M. Jorgensen <span id='am-49' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> <br /> that modify effects C94W variation. In contrast,                   strains. <br /> data suggest L229 M58 residues TACC domain <br /> are sensitive change C94 residue.                                  Author Contributions <br />                                                                                     Conceived designed experiments: MTG NC. Performed the <br /> Conclusion                                                                          experiments: MTG. Analyzed data: MTG NC. Contributed reagents/ <br />   Understanding phenotypic consequence GVs different                      materials/analysis tools: MTG NC. Wrote paper: MTG NC. <br /> genetic backgrounds great importance understanding <br />  <br /> References <br />  1. Riddle DL, Blumenthal T, Meyer BJ, Priess JR (1997) Introduction C.           5. Hillier LW, Marth GT, Quinlan AR, Dooling D, Fewell G, et al. (2008) Whole- <br />     elegans.                                                                            genome sequencing variant discovery C. elegans. Nat Methods 5: 183&#8211; <br />  2. C. elegans Sequencing Consortium (1998) Genome sequence nematode             188. <br />     C. elegans: platform investigating biology. Science 282: 2012&#8211;2018.        6. Sivasundar A, Hey J (2003) Population genetics Caenorhabditis elegans: the <br />  3. Kiontke K, Sudhaus W (2006) Ecology Caenorhabditis species. WormBook:            paradox low polymorphism widespread species. Genetics 163: 147&#8211;157. <br />     1&#8211;14.                                                                            7. Barriere A, Felix MA (2005) High local genetic diversity low outcrossing rate <br />  4. Andersen EC, Gerke JP, Shapiro JA, Crissman JR, Ghosh R, et al. (2012)              Caenorhabditis elegans natural populations. Curr Biol 15: 1176&#8211;1184. <br />     Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic           8. Haber M, Schungel M, Putz A, Muller S, Hasert B, et al. (2005) Evolutionary <br />     diversity. Nat Genet 44: 285&#8211;290.                                                   history Caenorhabditis elegans inferred microsatellites: evidence for <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                      5                           November 2012 | Volume 7 | Issue 11 | e48762 <br />                                                                                                                                    MosSCI Study Single Gene Mutations <br />  <br />  <br />       spatial temporal genetic differentiation occurrence outbreeding.         26. Ghosh R, Andersen EC, Shapiro JA, Gerke JP, Kruglyak L (2012) Natural <br />       Mol Biol Evol 22: 160&#8211;173.                                                                  variation chloride channel subunit confers avermectin resistance C. <br />  9.   Sivasundar A, Hey J (2005) Sampling natural populations RNAI                      elegans. Science 335: 574&#8211;578. <br />       reveals high outcrossing population structure Caenorhabditis elega <br /> </body></html>