<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Amyloid-Mediated Sequestration of Essential Proteins <br /> Contributes to Mutant Huntingtin Toxicity in Yeast <br />  <br />  <br />      Abstract <br />      Background: Polyglutamine expansion is responsible for several neurodegenerative disorders, among which Huntington <br />      disease is the most well-known. Studies in the yeast model demonstrated that both aggregation and toxicity of a huntingtin <br />      (htt) protein with an expanded polyglutamine region strictly depend on the presence of the prion form of Rnq1 protein <br />      ([PIN+]), which has a glutamine/asparagine-rich domain. <br />  <br />      Principal Findings: Here, we showed that aggregation and toxicity of mutant htt depended on [PIN+] only quantitatively: <br />      the presence of [PIN+] elevated the toxicity and the levels of htt detergent-insoluble polymers. In cells lacking [PIN+], toxicity <br />      of mutant htt was due to the polymerization and inactivation of the essential glutamine/asparagine-rich Sup35 protein and <br />      related inactivation of another essential protein, Sup45, most probably via its sequestration into Sup35 aggregates. <br />      However, inhibition of growth of [PIN+] cells depended on Sup35/Sup45 depletion only partially, suggesting that there are <br />      other sources of mutant htt toxicity in yeast. <br />  <br />      Conclusions: The obtained <span id='am-26' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-27' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest that induced polymerization of essential glutamine/asparagine-rich proteins and <br />      related sequestration of other proteins which interact with these polymers represent an essential source of htt toxicity. <br />  <br />  <br /> Introduction                                                                            transcription of a subset of genes and decreases cell viability [3]. <br />                                                                                         Besides glutamine, prion domains of yeast proteins are also rich in <br />    Expansion of polyglutamine (polyQ) stretches in nine otherwise                       asparagine residues (N). Importantly, expanded polyN stretches <br /> unrelated human proteins causes neurodegenerative diseases                              are similar to polyQ in their propensity to form aggregates in yeast <br /> accompanied by deposition of amyloid protein aggregates formed                          and were also used to model polyQ disorders since nuclear <br /> by these proteins. One of the most common polyQ disorders,                              localization of such proteins also causes transcriptional abnormal- <br /> Huntington disease, is caused by exon I IT15 mutations that                             ities and cell death [4]. Cytoplasmically expressed htt with an <br /> increase the number of CAG triplets, coding for Q in the                                expanded polyQ region was also shown to be toxic, although its <br /> huntingtin (htt) protein and develops at a probability proportional                     toxicity and aggregation depended on the presence of [PIN+], the <br /> to the number of these repeats. Htt with expanded N-terminal                            prion form of the QN-rich protein Rnq1, known to facilitate the de <br /> polyQ can aggregate and form insoluble intracellular inclusions                         novo appearance of other yeast prions [5,6]. Besides, toxicity of <br /> which appear mostly in the nucleus and, to a lesser extent, in the                      expanded htt in yeast cytoplasm is modulated by the sequences <br /> cytoplasm [1]. Despite extensive studies, the molecular bases of                        flanking the polyQ stretch [7]. Use of the yeast model revealed <br /> polyQ diseases are still unclear, though it was shown that the toxic                    that polyQ aggregation results in endocytosis impairment, <br /> effect of expanded polyQ proteins is related to interference with                       suggesting that <span id='am-8' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-9' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> is one of the causes of cellular toxicity [8]. <br /> the normal function of cellular proteins thus affecting various                         Another reason for toxicity of proteins with expanded polyQ can <br /> cellular processes. Indeed, pathological htt impairs gene transcrip-                    be interactions with certain genome-encoded Q/N-rich proteins <br /> tion, ubiquitin-proteasome system, causes mitochondrial dysfunc-                        whose lack or overproduction can cause or abolish polyQ toxicity <br /> tion, dysregulation of Ca2+ homeostasis, impairment of axonal                           [6]. <br /> transport and genotoxic stress (see [2] for a review).                                     Previously we have shown that aggregation of proteins with <br />    PolyQ disorders and Huntington disease in particular are                             expanded polyQ, including mutant htt, in the cytoplasm of yeast <br /> especially attractive for modeling in yeast, because, similar to                        cells caused polymerization of chromosomally-encoded Q/N-rich <br /> them, yeast prions rely on domains enriched with Q. As in                               proteins [9]. Since there are a large number of proteins with long <br /> humans, in yeast aggregation and toxicity of htt increase with                          Q/N-rich stretches in both humans and yeast [10,11], it is likely <br /> polyQ length and targeting of mutant htt into the nucleus alters                        that at least some of them would efficiently polymerize in response <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   1                               January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                              Huntingtin Toxicity in Yeast <br />  <br />  <br /> to the accumulation of polyQ amyloids, which may cause                         though approximately 3-fold less efficiently than in [PIN+] cells <br /> depletion of their functional soluble form and, as a result, cell              (Fig. 1). <br /> death, if these proteins are essential. Here, we used the yeast model             The difference in the 103Q-GFP polymer abundance in [PIN+] <br /> of Huntington disease to confirm this suggestion for the essential             and [pin2] cells could be due to the influence of [PIN+] on the <br /> Q/N-rich protein, translation termination factor eRF3, which is                levels of this protein (Fig. 2). Most likely this was not due to <br /> usually designated in yeast as Sup35.                                          increased expression of 103Q-GFP, since [PIN+] did not affect the <br />                                                                                levels of 25Q-GFP, which did not form SDS-insoluble polymers in <br /> Results                                                                        either [PIN+] or [pin2] cells (data not shown), despite the 25Q- <br />                                                                                GFP-encoding gene being under the control of the same promoter <br /> [PIN+]-independent polymerization of 103Q-GFP <br />                                                                                as that of 103Q-GFP. Therefore, one can suggest that higher levels <br />    In yeast, aggregation and toxicity of overproduced htt with <br />                                                                                of 103Q-GFP in [PIN+] cells compared with [pin2] were due to <br /> expanded polyQ requires the presence of either [PIN+] or [PSI+] <br />                                                                                accelerated accumulation of its polymers (Fig. 1) which are less <br /> prions [5,6]. This was shown for htt with a polyQ stretch <br />                                                                                susceptible to proteolysis than the soluble form of this protein. <br /> consisting of 103 Q (103Q) fused to the green fluorescent protein <br />                                                                                Interestingly, 25Q-GFP levels in both [PIN+] and [pin2] cells were <br /> (GFP), which allows monitoring of 103Q-GFP aggregation <br />                                                                                higher than those of 103Q-GFP in [PIN+], which could reflect a <br /> microscopically, as distinct fluorescent foci. However, as we have <br />                                                                                lesser stability of 103Q-GFP. <br /> shown previously, polyQ proteins could also form SDS-insoluble <br /> polymers in cells lacking these prions ([psi2] [pin2] cells) [9,12]. <br /> This prompted us to study the dependence of 103Q-GFP                           Effect of [PIN+] on 103Q-GFP-induced polymerization of <br /> polymerization on the [PIN+] prion. In agreement with published                Rnq1 and Sup35 <br /> data, the ability of 103Q-GFP to form fluorescent foci depended                   Previously we have demonstrated that SDS-insoluble polymers <br /> on [PIN+]: in the absence of this prion 103Q-GFP aggregates                    of 103Q-GFP can seed polymerization of Sup35 in [PIN+] cells <br /> were rarely found, while the same protein but with a stretch of 25             [9]. Here we showed that polymers of 103Q-GFP also seeded <br /> Q, 25Q-GFP, did not form aggregates neither in [PIN+] nor                      Sup35 (Fig. 3A) and Rnq1 polymerization (data not shown) in <br /> [pin2] (data not shown). However, SDD-AGE analysis showed                      [pin2] cells. The amount of Sup35 polymers correlated with the <br /> that overproduced 103Q-GFP formed polymers in [pin2] cells,                    levels of 103Q-GFP polymers and was approximately 3-fold <br />  <br />  <br />  <br />  <br /> Figure 1. Time-dependent appearance of the 103Q-GFP SDS-insoluble polymers. (A) Cells of the [psi2] [PIN+] or [psi2] [pin2] transformants <br /> of the strain 74-D694 with the multicopy p103Q-GFP plasmid were grown in liquid SC-Ura glucose-containing medium (Glc), then in the same <br /> selective raffinose-containing (Raf) and galactose-containing (Gal) media, as described in Materials and Methods, and incubated in the latter medium <br /> for 1, 2, 4, 6 and 9 h. (B) 1:3, 1:9, 1:27 and 1:81, dilutions of the sample taken after 9 h incubation. Polymers of 103Q-GFP were visualized by SDD-AGE. <br /> Blots were stained with the monoclonal anti-GFP antibody. <br /> doi:10.1371/journal.pone.0029832.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                            January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                              Huntingtin Toxicity in Yeast <br />  <br />  <br />                                                                                 than in [PIN+] cells (Fig. 3B), which caused a detectable increase of <br />                                                                                 nonsense <span id='am-13' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> readthrough (Fig S1). In contrast to 103Q-GFP, <br />                                                                                 overproduction of non-polymerizing 25Q-GFP neither inhibited <br />                                                                                 growth of [PIN+] and [pin2] strains (Fig. 4) nor caused appearance <br />                                                                                 of Sup35 polymers in [PIN+] and [pin2] cells (Fig. 3A and data not <br />                                                                                 shown). <br />  <br />                                                                                 103Q-GFP toxicity partially depends on depletion of <br />                                                                                 Sup35 and Sup45 <br />                                                                                    Unlike Rnq1, Sup35 is an essential protein and therefore <br />                                                                                 depletion of its functional, soluble form should be deleterious for <br />                                                                                 the cell. In agreement with this, though overproduction of 103Q- <br />                                                                                 GFP caused growth inhibition of both [PIN+] and [pin2] strains, <br />                                                                                 this effect was more pronounced in a [PIN+] background (Fig. 4) in <br />                                                                                 which both 103Q-GFP and Sup35 polymerized most efficiently <br />                                                                                 (Fig. 1 and 3). Expression of the non-aggregating form of Sup35, <br />                                                                                 which lacked the NM region (Sup35C), instead of the full-length <br /> Figure 2. Semi-quantitative dot-blot analysis of 103Q-GFP and                   protein, alleviated the toxic effect of 103Q-GFP in [PIN+] cells and <br /> 25Q-GFP. Transformants of 74-D694 [psi2] [PIN+] or [psi2] [pin2] with <br />                                                                                 completely abolished it in [pin2] cells (Fig. 4). This suggested that <br /> the multicopy p103Q-GFP or p25Q-GFP <span id='am-31' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> were grown as <br /> described in Materials and Methods with incubation in SC-Ura Gal                Q103-GFP toxicity was due to depletion of soluble Sup35. <br /> medium for indicated time. Equal amounts of total protein (confirmed            Alternatively, it was possible that this effect was due to the <br /> by staining the same membranes by Ponceau S, a non-specific protein             influence of Sup35C on the levels of 103Q-GFP and Rnq1 <br /> stain) from each lysate were serially diluted in four-fold increments and       polymers or due to effects of Sup35C on cell growth irrespective of <br /> applied to a nitrocellulose membrane. Blots were probed with the                the presence of these polymers. However since neither the levels <br /> monoclonal anti-GFP antibody. <br /> doi:10.1371/journal.pone.0029832.g002                                           nor size distributions of Rnq1 and 103Q-GFP polymers did not <br />                                                                                 depend on the presence of the NM region in Sup35 (Fig. S2), and <br /> higher in [PIN+] as compared to [pin2] cells. Correspondingly, the              Sup35C did not affect growth of cells not expressing 103Q-GFP <br /> amount of soluble Sup35 was inversely proportional to the levels of             (Fig. S3), these possibilities seemed unlikely. Importantly, there <br /> its polymerized form being approximately 3-fold higher in [pin2]                were also other causes for toxicity of 103Q-GFP in [PIN+] cells, <br />  <br />  <br />  <br />  <br /> Figure 3. 103Q-GFP-dependent polymerization of Sup35. (A) Polymers of Sup35 visualized by SDD-AGE. 74-D694 [psi2] [PIN+] and [psi2] <br /> [pin2] transformants with the p103Q-GFP plasmid were grown as described in Materials and Methods. After incubation in SC-Ura Gal medium for 9 h, <br /> cells were harvested and their lysates were used to analyze the amount of SDS-insoluble polymers of Sup35. (B) The levels of Sup35 monomer, SDS- <br /> PAGE analysis. Lysates were obtained from cells grown as described above. The samples were not boiled before loading onto the gel which only <br /> allowed SDS-soluble Sup35 to enter the gel [34]. Lower bands represent a Sup35 degradation product, characteristic of the non-prion/amyloid form <br /> of Sup35 [36]. Total lysates (T) and their serial dilutions are indicated. 74-D694 [psi2] [pin2] expressing 25Q-GFP and 74-D694 [PSI+], respectively, are <br /> shown for comparison. Blots were stained with anti-Sup35NM antibody. Quantification of Sup35 by densitometric analysis demonstrated that upon <br /> expression of 103Q-GFP the levels of soluble Sup35 were approximately 3-fold lower in [PIN+] than in [pin2] cells. <br /> doi:10.1371/journal.pone.0029832.g003 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           3                           January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                            Huntingtin Toxicity in Yeast <br />  <br />  <br />  <br />  <br /> Figure 4. PolyQ toxicity depends on [PIN+] and Sup35. The [psi2] [PIN+] and [psi2] [pin2] transformants of the 74-D694 strain (SUP35) or its 74- <br /> D694 DS35 derivative disrupted for chromosomal SUP35 and carrying SUP35-C on a centromeric plasmid (SUP35-C), both expressing either 103Q-GFP <br /> or 25Q-GFP, were grown at 30uC in liquid SC-Ura medium with glucose resuspended in the same medium but with raffinose instead of glucose and <br /> after 12 h incubation, cell suspensions were diluted to an OD600 of 1.0, and then spotted onto SC-Ura plates with galactose as a sole carbon source <br /> (Gal) and incubated for 4 days. Equal spotting was controlled by spotting the cells onto SC-Ura plates containing glucose as carbon source (Glc) in <br /> parallel. Five serial 5-fold dilutions of cell suspensions are shown. <br /> doi:10.1371/journal.pone.0029832.g004 <br />  <br /> since a [PIN+] strain expressing the non-polymerizing Sup35C                     aggregates in a large proportion of [PIN+] cells. However, SDD- <br /> protein still manifested a slight growth defect.                                 AGE polymer analysis led us to another conclusion. While this <br />    It is known that Sup35 prion aggregates may include different                 analysis confirmed the inability of overproduced 25Q-GFP to <br /> proteins able to interact with Sup35 [13,14] in an SDS-sensitive                 polymerize in both [pin2] and [PIN+] cells, 103Q-GFP was shown to <br /> manner. Sequestration of Sup45 (translation termination factor                   form polymers irrespective of the [PIN+] state, albeit more polymers <br /> eRF1) into Sup35 prion aggregates can cause growth inhibition,                   were observed in [PIN+] than in [pin2] cells. In agreement with this, <br /> and this defect can be alleviated by increasing Sup45 levels                     the GFPN104 protein (N-terminal fusion of GFP to the sequence of <br /> [15,16]. Similarly to prion polymers of Sup35, non-prion polymers                104 N) formed SDS-insoluble aggregates in the cytoplasm of [pin2] <br /> of this protein also contained Sup45, since centrifugation of lysates            cells, which could not be visualized microscopically [4]. Thus, <br /> of [PIN+] cells overproducing 103Q-GFP revealed appearance of                    despite the convenience of the GFP test for aggregate detection, to <br /> Sup45 in the pellet fraction and a related 2-fold decrease of the                avoid misinterpretations, it should be supported by alternative <br /> levels of its soluble form, while in cells expressing non-aggregating            approaches. <br /> 25Q-GFP Sup45 was primarily soluble (Fig. 5). Introduction of a                     Contrary to earlier studies, we observed that overproduced <br /> centromeric plasmid with SUP45 into [PIN+] and [pin2] cells                      103Q-GFP inhibited growth of both [PIN+] and [pin2] cells, <br /> expressing 103Q-GFP partially alleviated the toxic effects of                    though this effect was more pronounced in [PIN+] cells, in which <br /> 103Q-GFP (Fig. 6), which could be related to increased levels of                 103Q-GFP polymerized most efficiently. Importantly, the toxic <br /> soluble Sup45 (Fig. 5).                                                          effect of overproduced 103Q-GFP was related to Sup35 <br />                                                                                  polymerization, since expression of a non-aggregating Sup35C <br />                                                                                  variant of this protein abolished 103Q-GFP toxicity in [pin2] cells <br /> Discussion <br />                                                                                  and reduced it in isogenic [PIN+] cells. Finally, the toxicity of <br />    Earlier it was reported that in contrast to [PIN+] cells, cells lacking       103Q-GFP also depended on sequestration of Sup45 into Sup35 <br /> this prion rarely possess fluorescently detectable aggregates of                 aggregates. The critical role of Sup35 in 103Q-GFP toxicity may <br /> overproduced 103Q-GFP and exhibit no decreased viability, while                  seem surprising, since it is just one of more than a hundred of yeast <br /> protein with a shorter polyQ stretch, 25Q-GFP, does not form                     Q/N-rich proteins, many of which are essential [10,11]. However, <br /> aggregates and manifest toxicity in ether [PIN+] or [pin2] cells [7,8].          it is necessary to stress that among essential Q/N-rich proteins, <br /> This indicated that the aggregation and toxicity of htt in yeast                 Sup35 has one of the highest expression levels [17] and that its <br /> depends both on the polyQ expansion and the presence of the                      polymerization caused aggregation and inactivation of another <br /> [PIN+] prion. We also observed that 103Q-GFP in [pin2] cells and                 essential protein, Sup45. It is also necessary to take into account <br /> 25Q-GFP in both [pin2] and [PIN+] cells formed bright fluorescent                that 103Q-GFP may seed polymerization of different cellular Q/ <br /> foci only occasionally, while 103Q-GFP formed numerous visible                   N-rich proteins with varying efficiency and Sup35 may be among <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            4                          January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                             Huntingtin Toxicity in Yeast <br />  <br />  <br />  <br />  <br /> Figure 5. Centrifugation analysis of aggregation of Sup35 and Sup45 in the presence of 103Q-GFP or 25Q-GFP. Transformants of the <br /> strain 74-D694 [psi2] [PIN+] with plasmids expressing 103Q-GFP or 25Q-GFP were grown as described in Materials and Methods. After incubation in <br /> SC-Ura Gal medium for 9 h, cells were harvested. Cell lysates were fractionated by centrifugation and fractions were analyzed by Western blotting: <br /> staining with either anti-Sup35NM (Sup35) or anti-Sup45 (Sup45) polyclonal antibodies. The lower bands in the Sup35 image represent a degradation <br /> product (see legend to Fig. 3); the lower bands in the Sup45 image are nonspecific protein staining [20]. T, total protein; S, soluble fraction (1:2 and <br /> 1:4, dilutions); P, pellet. CEN-SUP45, the centromeric plasmid pRS315-SUP45; empty vector, pRS315. Quantitative data on Sup45 levels obtained by <br /> densitometric analysis demonstrated that expression of Q103-GFP caused an approximately 2-fold depletion of the soluble fraction of Sup45, with <br /> the remaining portion of the protein being relocated into the pellet fraction. Presence of pRS315-SUP45 caused an approximately 4-fold increase in <br /> the level of soluble Sup45 in Q103-GFP expressing cells as compared with the same cells with an empty vector. <br /> doi:10.1371/journal.pone.0029832.g005 <br />  <br /> those whose polymerization is seeded most efficiently. At last,                factors could not completely account for 103Q-GFP toxicity, <br /> proteins enriched in Q and N may differ from each other by their               because overproduction of the 103Q-GFP protein also manifested <br /> intrinsic propensity to polymerize and Sup35 may be among those                toxicity in the [PIN+] strain expressing Sup35C which is unable to <br /> which are most prone to polymerization. The contribution of                    form SDS-insoluble polymers. <br /> Sup45 depletion to 103Q-GFP toxicity is not surprising, because                   At present it is believed that microscopically visible protein <br /> sequestration of this protein into [PSI+] aggregates, but not of               aggregates, which are frequently associated with neurodegenera- <br /> other essential proteins able to interact with Sup35, was                      tive diseases, in fact play a cytoprotective role, while toxicity of <br /> responsible for growth inhibition of [PSI+] cells overproducing                amyloidogenic proteins is related to accumulation of their <br /> Sup35 [14]. Notably sequestration of the translation termination               oligomeric <span id='am-18' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> [18,19]. In this work we observed that in <br />  <br />  <br />  <br />  <br /> Figure 6. PolyQ toxicity is modulated by the levels of Sup45. Growth of the [psi2] [PIN+] and [psi2] [pin2] transformants of the 74-D694 strain <br /> carrying either a centromeric plasmid with SUP45 or an empty vector as a control and expressing either 103Q-GFP or 25Q-GFP, was analyzed as <br /> described in the legend to Fig. 4. <br /> doi:10.1371/journal.pone.0029832.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          5                            January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                      Huntingtin Toxicity in Yeast <br />  <br />  <br />  <br />  Table 1. Plasmids used in this study.                                     grown for the indicated time. The final concentration of galactose <br />                                                                            in the medium was 2%. <br />  <br />  Plasmid             Characteristics                      Reference        Preparation of yeast cell lysates <br />                                                                               Yeast cultures grown in liquid selective media were harvested, <br />  pRS315              Centromeric LEU2 vector              [37] <br />                                                                            washed in water and lyzed by beating with glass beads (Bullet <br />  pRS315-SUP45        Same as pRS315, but with SUP45       [21]             Blender, Next Advance) in buffer A: 30 mM Tris-HCl, pH 7.4, <br />  pRS315-SUP35C       Same as pRS315, but with SUP35-C     [22]             150 mM NaCl, 1 mM dithiothreitol and 1% Triton X-100. To <br />  p103Q-GFP           Multicopy URA3 pYES2 plasmid,        [5]              prevent proteolytic degradation, 10 mM phenylmethylsulfonyl <br />                      encoding fusion of 103Q with GFP                      fluoride and CompleteTM protease inhibitor cocktail (Roche <br />                      under the control of GAL1 promoter <br />                                                                            Applied Science) were added. Cell debris was removed by <br />  p25Q-GFP            Multicopy URA3 pYES2 plasmid,        [5]              centrifugation at 1500 g for 4 min. <br />                      encoding fusion of 25Q with GFP <br />                      under the control of GAL1 promoter <br />                                                                            Centrifugation <br />  doi:10.1371/journal.pone.0029832.t001                                        To separate Sup35 and Sup45 polymer and monomer fractions, <br />                                                                            500 ml of yeast cell lysates were centrifuged at 100,000 g <br /> [pin2] cells, lacking microscopically visible 103Q-GFP aggregates,         (48,000 rpm in a Ti75 rotor, Beckman Optima TL ultracentrifuge) <br /> toxicity was related to SDS-insoluble aggregates of this protein,          for 1 h at 4uC. <br /> which may correspond to toxic oligomers in mammals. At the <br /> same time, overproduced 103Q-GFP was more toxic in [PIN+]                  Electrophoresis and blotting <br /> than [pin2] cells, which correlated with increased formation of its           These were performed as described previously [32&#8211;34]. Protein <br /> fluorescently detectable aggregates. However, [PIN+] cells also            loads were equalized for each gel. For analysis of amyloid <br /> contain more SDS-insoluble polymers not detectable microscop-              polymers we used horizontal 1.8% agarose gels in the Tris- <br /> ically on their own, which did not allow us to make conclusions            Acetate-EDTA (TAE) buffer with 0.1% SDS. Lysates were <br /> about the role of microscopically detectable aggregates of this            incubated in the sample buffer (0.56 TAE, 2% SDS, 5% glycerol <br /> protein in toxicity.                                                       and 0.05% Bromophenol Blue) for 5 min at 37uC. After the <br />    Notably, depletion of Sup35 and Sup45 induced by 103Q-GFP               electrophoresis, proteins were transferred from gels to nitrocellu- <br /> may impair not only translation termination, but also other                lose membrane sheets (ThermoScientific, USA) by vacuum- <br /> processes, since both termination factors have essential non-              assisted capillary blotting for 5 h (agarose gels), or electrophoret- <br /> translational functions, being important for cytoskeleton organi-          ically (polyacrylamide gels). Bound antibody was detected using <br /> zation, cytokinesis, cell cycle regulation [20,21] and probably for        the ECL West <span id='am-21' about='obo:NCBITaxon_1369122' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_27548 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Dura" datatype="xsd:string"></span>Dura</span> system (Thermo Scientific). It should be noted <br /> other processes [22,23], such as coupling termination and                  that detergents (SDS or sarcosyl) in non-boiled samples increase <br /> initiation steps of translation [24,25] and regulation of mRNA             degradation of Sup35 monomers. This can result in the absence of <br /> deadenylation and decay [26], which are mediated by interaction            Sup35 monomer bands in SDD-AGE gels. Rabbit polyclonal <br /> of the N-terminal region of Sup35 with poly(A)+ binding protein            antibodies against Sup35 and Sup45 were used. Anti-GFP <br /> PABP. Modeling of htt toxicity in yeast showed that polymeriza-            monoclonal antibody was obtained from Rusbiolink (Russia). <br /> tion of expanded polyQ caused polymerization of cellular Q/N-              Estimation of relative amount of 103Q-GFP or 25Q-GFP in <br /> rich proteins and related sequestration of other proteins able to          lysates was performed as described in [35], with minor <br /> interact with these Q/N-rich proteins. If such proteins are                modifications. Densitometry measurements were performed using <br /> essential, depletion of their soluble and functional form may be           ImageJ software. <br /> deleterious to the cell. Thus, though there may be different sources <br /> of polyQ toxicity in eukaryotes, this mechanism can explain why a          Supporting Information <br /> wide variety of different processes, including translation, were           Figure S1 Polymerization of 103Q-GFP causes an in- <br /> found to be altered in the yeast model of Huntington disease [27].         crease in nonsense codon readthrough. The strain 74-D694 <br /> It is also likely that a similar mechanism underlies depletion of          [psi2][PIN+] carrying the URA3 p25Q-GFP (25Q) or p103Q-GFP <br /> transcription factors possessing polyQ repeats in mammalian cells          (103Q) plasmids was transformed with either the LEU2 plasmid <br /> expressing proteins with expanded polyQ [28&#8211;30].                           pUKC815-L (encodes a PGK1-lacZ gene fusion) or pUKC817-L <br />                                                                            (encodes the same gene fusion but with in frame UAA at the <br /> Materials and Methods                                                      junction of the PGK1 and lacZ genes) (Stansfield I, Jones KM, <br />                                                                            Kushnirov VV, Dagkesamanskaya AR, Poznyakovski AI, Paush- <br /> Plasmids, strains and growth conditions                                    kin SV, Nierras CP, Cox BS, Ter-Avanesyan MD, Tuite MF <br />    The strain 74-D694 [psi2] [pin2] and its [psi2] [PIN+] derivative       (1995) The products of the SUP45 (eRF1) and SUP35 genes <br /> [31], as well as 74-D694 DS35 [psi2] [PIN+] and [psi2] [pin2] with         interact to mediate translation termination in <span id='am-1' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-7'  ></span>Saccharomyces</span> <br /> SUP35 disrupted by the insertion of TRP1 [12] were used. The               cerevisiae. EMBO J 14: 4365&#8211;4373). Transformants were grown <br /> plasmids used in this study are described in Table 1. Yeast were           consecutively in liquid glucose-, raffinose- and galactose-contain- <br /> grown at 30uC in rich (YPD, 1% yeast extract, 2% peptone, 2%               ing media selective for the plasmids, and after a 9 h incubation in <br /> glucose) or synthetic (SC, 0.67% yeast nitrogen base, 2% glucose           SC-Ura -Leu Gal medium appropriate aliquots of yeast culture <br /> supplemented with the required amino acids) media. To induce               were taken and b-galactosidase activity was assayed. All data <br /> the synthesis of 103Q-GFP and 25Q-GFP chimeric proteins, cell              represent an average of at least three independent experiments. <br /> cultures with corresponding plasmids were transferred to liquid            The nonsense readthrough levels were determined as ratio of b- <br /> selective media with 2% raffinose as a sole carbon source and              galactosidase activities in the cells transformed with the plasmid <br /> grown until mid-log phase. Then cultures were transferred to of            pUKC817-L to that of the transformant with pUKC815-L. <br /> the same medium with galactose instead of raffinose and cells were         (TIF) <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       6                          January 2012 | Volume 7 | Issue 1 | e29832 <br />                                                                                                                                                 Huntingtin Toxicity in Yeast <br />  <br />  <br /> Figure S2 The levels of Rnq1 and 103Q-GFP polymers                                         centromeric pRS315-SUP35C plasmid (SUP35-C), were grown as <br /> do not depend on the presence of the Sup35 NM region.                                      described in the legend to Fig. 4. Cell suspensions were diluted to <br /> The 74-D694 [psi2] [PIN+] strain (SUP35) or its 74-D694 DS35                               an OD600 of 1.0, spotted onto Gal and Glc plates and incubated <br /> derivative disrupted for the chromosomal SUP35 gene and                                    for 4 days. Five serial 5-fold dilutions of cell suspensions are shown. <br /> carrying SUP35-C on a centromeric plasmid (SUP35-C), both                                  (TIF) <br /> expressing 103Q-GFP, were grown as described in Materials and <br /> Methods. After incubation in SC-Ura Gal medium for 9 h cells                                <br /> were harvested and their lysates were used to estimate the amount <br /> of Q103-GFP and Rnq1 SDS-insoluble polymers by SDD-AGE                                    <br /> analysis. Blots were stained with anti-Rnq1 polyclonal antibody (A)                        <br /> or anti-GFP monoclonal antibody (B). <br /> (TIF) <br />                                                                                            Author Contributions <br /> Figure S3 Sup35C does not affect growth of cells not                                       Conceived and designed the experiments: NVK-P AIA MDT-A. <br /> expressing 103Q-GFP. The [psi2] [PIN+] and [psi2] [pin2]                                   Performed the experiments: NVK-P AIA. Analyzed the data: NVK-P <br /> transformants of the 74-D694 strain (SUP35) or its 74-D694 DS35                            AIA MDT-A. Contributed reagents/materials/analysis tools: NVK-P AIA <br /> derivative with disruption of chromosomal SUP35 carrying the                               MDT-A. Wrote the paper: MDT-A. <br />  <br /> </body></html>