<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br /> Chromosomal Deletion Formation System Based <br /> on Tn5 Double Transposition: Use For Making <br /> Minimal Genomes and Essential Gene Analysis <br />  <br />  <br />  <br />        In <span id='am-48' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> communication, we describe the use of specialized <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span>transposons</span> (Tn5 derivatives) to create deletions in the <br />        <span id='am-64' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-79'  ></span>Escherichia coli</span> K-12 chromosome. These transposons are essentially rearranged composite transposons that have <br />        been assembled to promote the use of the internal transposon ends, resulting in intramolecular transposition <br />        events. Two similar transposons were developed. The first deletion transposon was utilized to create a <br />        consecutive set of deletions in the E. coli chromosome. The deletion procedure has been repeated 20 serial times <br />        to reduce the genome an average of 200 kb (averaging 10 kb per deletion). The second deletion transposon <br />        contains a conditional origin of replication that allows deleted chromosomal DNA to be captured as a <br />        complementary plasmid. By plating cells on media that do not support plasmid replication, the deleted <br />        chromosomal material is lost and if it is essential, the cells do not survive. This methodology was used to analyze <br />        15 chromosomal regions and more than 100 open reading frames (ORFs). This provides a robust technology for <br />        identifying essential and dispensable genes. <br />        [Supplemental material is available online at www.genome.org and is supplied as an extended table enumerating <br />        genes lost in two multiple round deletion strains (&#9004;20-1 and &#9004;20-4). These <span id='am-90' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> are summarized in Table 1.] <br />  <br />  <br /> Transposons are powerful tools for performing genomic struc-               chromosomal transposon insertions, followed by sequencing <br /> ture/function studies. They have long been used in the gen-                or PCR analysis of inserts in surviving cells, it can be assumed <br /> eration of knockout mutations. With the advent of in vitro                 that any gene that is not found to contain a transposon in- <br /> transposition systems (Devine and Boeke 1994; Gwinn et al.                 sertion is essential. In essence, these methods catalog ob- <br /> 1997; Akerley et al. 1998; Goryshin and Reznikoff 1998; Grif-              served nonessential genes and assume that other genes must <br /> fin IV et al. 1999; Haapa et al. 1999), the use of transposons in          therefore be essential. An important exception is the devel- <br /> genome analysis has been greatly expanded to include, for                  opment of a method using a transposon with regulated out- <br /> instance, their being applied as mobile primer binding sites in            ward-facing promoters. Insertion of this transposon into the <br /> high-throughput sequencing efforts (Butterfield et al. 2002;               promoter region of an essential gene can be recovered under <br /> Shevchenko et al. 2002). In this communication, we will de-                conditions in which the transposon promoters are turned on <br /> scribe another powerful application of DNA transposition                   to support expression of the essential gene. Insertions can <br /> that combines in vitro and in vivo Tn5-based technologies to               then be screened on medium that does not turn on the trans- <br /> generate random deletions in the <span id='am-80' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli K12 genome.              poson promoter, resulting in loss of viability of those cells <br /> As will be described, the Tn5 deletion formation system can                that depend on the conditional transposon promoter for <br /> be used in essentially any bacterial <span id='am-53' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> to define essential           growth. This results in the ability to positively identify essen- <br /> genes, to generate minimal essential genomes, and to clone                 tial genes and has been used for this purpose (Judson and <br /> genes.                                                                     Mekalanos 2000b). However, the technique is hindered by the <br />      The entire DNA sequences of many bacterial genomes                    fact that few of the transposon inserts within a population <br /> have been determined. However, the experimentalist is still                will be inserted into gene promoter regions. <br /> faced with the challenge of determining the role of various                      In addition to insertions, transposons are also capable of <br /> putative genes. Transposition-based strategies have been de-               promoting other types of DNA rearrangements. Deletion (and <br /> veloped recently for identifying essential genes (for review,              inversion) formation is a natural feature of composite trans- <br /> see Judson and Mekalanos 2000a; Hamer et al. 2001; Gerdes et               posons. Composite transposons can create deletions or inver- <br /> al. 2002). Conceptually, these methods are based on the fact               sions by an intramolecular transposition mechanism (Fig. 1). <br /> that transposon insertion into a gene causes loss of gene func-            Internal transposon ends are oriented in such a way that <br /> tion (gene knockout), and insertion into an essential gene is              DNA between the ends can be considered as the donor DNA, <br /> lethal to the organism and cannot be observed (Akerley et al.              which is released by transposase-catalyzed cleavage. The rest <br /> 1998; Hare et al. 2001). By generating large libraries with                of the DNA (a plasmid or a chromosome) is recognized <br />                                                                            as a transposon that can undergo self-integration, leading <br />                                                                            to a deletion or inversion event. During this process, two de- <br /> 1 <br />  These authors contributed equally to this work.                           letions are formed, removal of the internal part of the trans- <br /> 2 <br />  Corresponding author. <br /> E-MAIL Reznikoff@biochem.wisc.edu; FAX (608) 265-2603. <br />                                                                            poson by double-ended cleavage at the transposon ends and <br /> Article and publication are at http://www.genome.org/cgi/doi/10.1101/      deletion of a portion of the chromosome by the integration <br /> gr.611403. Article published online before print in March 2003.            event. <br />  <br />  <br />  <br /> 644     Genome Research                        13:644&#8211;653 &#169;2003 by Cold Spring Harbor Laboratory Press ISSN 1088-9051/03 $5.00; www.genome.org <br />            www.genome.org <br />                                                                           Tn5 Double Transposition and Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 1 Strategy for recursive deletion and coupled deletion/plasmid formation systems. The strategy for deletion formation can be used after <br /> integration of the transposon into the host&#8217;s genome. The internal transposon ends (MEs) are used in the second transposition event. Two deletions <br /> result from this transposition event in vivo, the first leading to the removal of the internal part of the transposon, and the second resulting in the <br /> deletion of a portion of the chromosome. TnpEK/LP binds to the MEs, resulting in blunt-end cleavage and loss of the donor DNA. Tnp-EK/LP then <br /> facilitates intra molecular strand transfer into the chromosome. (A) Not all events during strand transfer will result in deletion. Inversions may also <br /> result in this mechanism. Deletions may happen to the left or to the right, defining the loss of the corresponding part of the transposon. Deletions <br /> to the right will result in loss of all transposon DNA, with the exception of a neutral linker. This event can be detected by replica plating (under <br /> conditions of high transposition frequency). Traditional transposition by use of the external pair of transposon ends (IEs) does not occur because <br /> the expressed Tnp does not interact with the external ends. (B) The addition of a conditional origin of replication allows for the capture of the <br /> deleted chromosomal material into a complementary self-replicating plasmid. The presence of IPTG in the medium results in the capture of these <br /> circular DNAs as <span id='am-105' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> that are complementary to the chromosome. <br />  <br />  <br />  <br />  <br />      The system that we will describe makes use of both types                  RESULTS <br /> of deletions associated with intramolecular transposition (Fig. <br /> 1A). Double-ended cleavage eliminates the transposase gene                     Deletion Modules: Composition and Strategy <br /> and the selectable marker used for the prior transposon inser-                 The transposons we used exploit the observation that com- <br /> tion selection. The self-integration reaction results in <span id='am-87' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-88' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>-                posite transposons make deletions by use of internal transpo- <br /> tion of the second deletion that starts at the point of trans-                 son ends. The structures of transposons used in this study are <br /> poson location and extends to the point on the chromosome                      shown in Figure 2. The transposon Tn5Del7 is designed for <br /> or plasmid defined by the second transposition event. This                     immediate deletion of chromosomal DNA and does not con- <br /> deletion is the deletion of interest. An important feature of                  tain an origin of replication. The Tn5Del8 transposon con- <br /> this scenario is that the protocol eliminates all selectable                   tains a conditional origin of replication that allows the cap- <br /> markers and the transposase gene and, thus, can be performed                   ture of deleted chromosomal DNA in the cell as a comple- <br /> repetitively for an accumulation of deletions.                                 mentary plasmid. Elimination of this plasmid is triggered by <br />      In this communication, we will describe the use of the                    the removal of IPTG from the medium. <br /> transposon-based deletion strategy as a tool for both defining                      The particular features of the transposons used in these <br /> essential genes and for removing nonessential genes from the                   experiments are as follows. Both transposons are flanked by <br /> chromosome.                                                                    19-bp inside ends (IEs) and can be separated from the donor <br />  <br />  <br />  <br />                                                                                                                     Genome Research              645 <br />                                                                                                                        www.genome.org <br />  Goryshin et al. <br />  <br />  <br />  <br />  <br /> Figure 2 Structure and features of transposons used in this work. The structure of Tn5Del7 and Tn5Del8 is conceptually the same. Both <br /> transposons are defined by IE sequences CTGTCTCTTGATCAGATCT (open triangle indicates IE sequence). Two ME sequences CTGTCTCTTATA- <br /> CACATCT (filled triangle indicates ME sequence) are faced toward each other, defining donor DNA for transposition using these ends. Distance <br /> between the tips of the IE and ME ends on the left is 64 bp. This is the size of the linker that remains in the chromosome after deletion. Donor DNA <br /> encodes the Tnp gene for ME end-mediated transposition under the control of an arabinose inducible promoter and also a KmR gene (Kan). <br /> Between the right pair of transposon ends, both transposons have a selectable marker (Cam). The only difference is that Tn5Del8 has a conditional <br /> origin of replication. Lac repressor encoded by the lacI gene controls the origin. Moderate plasmid (after its formation) copy number is ensured <br /> by Rop function. <br />  <br />  <br /> DNA component of the vector by digestion with PshAI. The                     located at 362,522 bp on the E. coli chromosome within the <br /> Tn5 IE ends are capable of participating in synaptic complex                 lacZ gene. <br /> formation in the presence of mutant Tnp protein Tnp sC7v2.0                       Transposition events were stimulated by inducing Tnp <br /> (Naumann and Reznikoff 2002). The synaptic complexes are                     EK/LP synthesis. The resulting colonies were tested for trans- <br /> delivered into cells by electroporation (Goryshin et. al. 2000).             position-associated events by replica plating as described <br /> The second set of 19-bp end sequences in the deletion mod-                   above and in the Methods section. As was typical of experi- <br /> ules are artificially constructed mosaic ends (MEs) (Zhou et. al.            ments using Tn5Del7, transposition events had occurred in <br /> 1998). We use MEs in combination with Tnp EK/LP for the                      &#8764;50% of the surviving cells with 1/4 of these events being <br /> second transposition step, as this was the most efficient Tn-                deletions in the desired direction, as indicated by loss of re- <br /> p:DNA end combination available when these experiments                       sistance to chloramphenicol. The large fraction of cells that <br /> were performed. A high efficiency of in vivo transposition                   had undergone transposition-associated events is presumably <br /> events is absolutely critical for their identification by screen-            due to the hyperactive nature of the transposase end- <br /> ing.                                                                         sequence combination that was used. In addition, Tnp expres- <br />      The deletion formation protocol is as follows (also see                 sion is known to be relatively toxic (Weinreich et al. 1994), <br /> Fig. 1). Cells are electroporated with preformed transposome                 and, therefore, cells that undergo transposition and loose the <br /> complexes, and transposon inserts are selected for by plating                Tnp gene would have a selective advantage. <br /> on LB medium containing both kanamycin and chloram-                               A total of nine independent deletions were selected and <br /> phenicol. Then, either a single colony or a collection of colo-              analyzed by DNA sequencing either directly from chromo- <br /> nies is used to start a liquid culture. At early exponential                 somal DNA or by the use of inverse PCR. Two deletions ended <br /> phase, arabinose is added to induce Tnp EK/LP synthesis. In                  within the Cam gene. The other seven deletions are described <br /> the case of Tn5Del8, IPTG and chloramphenicol are added to                   in Figure 3. The deletion sizes varied from 4&#8211;23 kb, with most <br /> activate the transposon origin of replication and to select for              deletion sizes being around 20 kb. This large deletion size <br /> the presence of the excised plasmid, respectively. After a few               makes it feasible to create a minimal genome in a reasonable <br /> hours, cells are subcultured in the same medium with a 500-                  amount of time by using our method recursively. <br /> fold dilution and grown overnight with shaking. Cells are <br /> then diluted and plated on agar medium containing the same                   Recursive Deletion Formation <br /> components to obtain single colonies for replica plating. In                 An important feature of this transposition/deletion system is <br /> the case of Tn5Del7, colonies that are sensitive to both kana-               that after the deletion is generated, all components of the <br /> mycin and chloramphenicol are picked as deletions. In the                    transposon are lost except for a short linker (64 bp). The loss <br /> case of Tn5Del8, cells are checked for sensitivity to kanamycin              of the transposase gene ensures the stability of the chromo- <br /> on plates that contain IPTG. Plasmid DNA is then isolated for                some (in terms of transposition). The loss of all selectable <br /> further analysis.                                                            markers provides an opportunity for recursive deletion forma- <br />                                                                              tion using transposon insertion and transposase-mediated de- <br /> Deletions in the Lactose Operon Region                                       letion formation, repetitively. <br /> The maximum size of chromosomal DNA that can be deleted                           We performed 20 rounds of recursive deletion formation <br /> in a single transposition-mediated event can be limited by the               in strain MG1655. Rather than focusing on one deletion at a <br /> relative location of essential DNA. To analyze the distribution              time, we isolated and mixed at least 10 colonies following <br /> of deletion sizes created by Tnp without limitations imposed                 each round. This was then used as the starting material for the <br /> by the presence of essential genes, we chose to isolate a                    subsequent rounds. It was assumed that strains with debili- <br /> Tn5Del7 insert in the lactose operon, as it is known that de-                tating deletions would be lost because other deletion strains <br /> letions exceeding 100 kb can be isolated in this region with no              with unimpaired growth would out compete them during <br /> negative impact on cell growth in rich medium (Bachmann                      growth of the <span id='am-59' about='obo:NCBITaxon_1306155' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="mixed culture" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mixed culture</span>. We obtained 10 final strains <br /> 1996). After electroporation of the deletion module into                     after 20 rounds that had a growth rate equal to that of the <br /> MG1655 and plating on Lactose-MacConkey agar with kana-                      parental strain (data not shown). <br /> mycin and chloramphenicol, we selected a few white colonies                       Pulsed-field gel electrophoresis analysis of NotI-digested <br /> among thousands of red colonies. One colony was isolated,                    chromosomal DNA was used to analyze the diversity among <br /> and the transposon insert site was sequenced. The insert was                 the 10 deletion strains and to estimate the average deletion <br />  <br />  <br />  <br /> 646     Genome Research <br />            www.genome.org <br />                                                                         Tn5 Double Transposition and Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 3 Deletions characterized for the lactose operon region. Genes deleted in E. coli MG1655 with the deletion transposon Tn5Del7 are <br /> shown by a gray line for each of the seven deletions in the lac operon. The insertion site was located 362,522 bp in the E. coli chromosome within <br /> the lacZ gene. <br />  <br />  <br /> size that occurred. An example of such a gel is shown in Figure             the genome or genes that have known paralogs in the ge- <br /> 4. The differences and similarities between samples are evi-                nome. Another surprising result is that there are fewer appar- <br /> dent by inspection of the gel. In some cases, bands disappear               ent deletions in each of the strains than might be expected <br /> from their original position (in the MG1655 DNA digestion)                  from 20 cycles (9 and 11 deletions were found in the two <br /> and are substituted by shorter fragments. In other cases,                   strains). It is possible that some deletions may have ended <br /> longer fragments appear, presumably due to the loss of NotI                 within the transposon as found in our lac insert-deletion <br /> site(s), which results in the formation of a large band coupled             analysis and, thus, they would not be detected. Alternatively, <br /> with the loss of two or more smaller bands. We calculated the               some deletions may be too small to have been picked up by <br /> total amount of DNA deleted for 4 strains that had undergone                the microarray hybridization technique. In some cases, what <br /> 20 rounds of deletion by estimating the size of each chromo-                looks like one deletion by the array data may really represent <br /> somal band compared with marker DNAs. From several gels                     two or more adjacent deletions. Finally, both of the strains <br /> that were run under various conditions to resolve different                 each have one extremely large deletion (&#8764;125,000 and <br /> regions in the DNA pattern, we calculated that the four strains             146,000-bp long) in NotI fragment G, near the chromosome <br /> whose DNAs were analyzed in Figure 4 contain deletions of                   replication terminus. These two deletions have one end in <br /> 250, 262, 100, and 247 kb. This indicates that the average                  common (near the NotI I/G fragment boundary), which sug- <br /> deletion size per round is 11 kb, which is in reasonable agree-             gests that they arose from the same insertion event, but had <br /> ment with the results obtained for the lactose operon region                separate deletion events. Thus, it is likely that the two strains <br /> in the previous experiment.                                                 diverged with the formation of these deletions. The observa- <br />                                                                             tion that these deletions were both relatively large indicates <br /> Microarray Mapping of Deletions                                             that the transposition/deletion system is capable of generat- <br /> The locations of deletions found in strains &#9004;20-1 (as judged by             ing extremely large deletions, that there are no essential genes <br /> the pulsed-field gel electrophoresis to be missing &#8764;250 kb) and             in this region, and that the chromosome condensation of this <br /> &#9004;20-4 (missing &#8764;262 kb) were mapped by microarray <span id='am-100' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104'  ></span>hybrid</span>-                   region of the chromosome may favor large deletion forma- <br /> ization experiments versus the progenitor strain. This tech-                tion. <br /> nique provided an excellent means to determine which genes <br /> were removed by the transposition/deletion procedure. Figure <br /> 5 presents the results schematically and Table 1 describes the              Coupled Chromosomal Deletion/Plasmid <br /> results in more detail. The two strains are apparently each                 Formation System <br /> lacking four common sets of genes. &#9004;20-1 has seven unique                   The obvious limitation of the deletion formation system de- <br /> deletions and &#9004;20-4 has five unique deletions. The data sug-                scribed above is the inability to introduce deletions involving <br /> gests that the two strains diverged after the fourth common                 essential genes. To address this, we developed a technique to <br /> deletion was generated. The two strains contain a wide range                conditionally save deleted DNA in the same cell and to at- <br /> of deletion sizes. The deletion sizes presumably reflect the                tempt its elimination later. In this manner, the essentiality of <br /> locations of the essential genes nearest the different transpo-             various sections of the chromosome can be directly tested by <br /> son inserts and/or differences in localized DNA condensation                removal of the deleted DNA from the cell and testing for <br /> near various inserts (see below). The mapped deletions are                  viability. Tn5Del7 was modified by inserting a conditional <br /> consistent with both the pulsed-field gel electrophoresis pat-              origin of replication to generate Tn5Del8 (Fig. 2). When a <br /> tern and the total amount of DNA missing, as determined by                  deletion is formed, the deleted material is excised as a circle <br /> the pulsed-field gel electrophoresis patterns.                              (Fig. 1B). Because the circle contains an origin of replication, <br />      There are a few surprises. First, we found positive hybrid-            the deletion has formed a replicating plasmid. <br /> ization results interrupting what would otherwise appear to                       The protocol used for the Tn5Del8 experiments is de- <br /> be a contiguous set of deleted genes. This observation is likely            scribed above, in the Methods section, and in Figure 1B. The <br /> due to a phenomenon observed by Richmond et al. (1999)                      critical feature is that IPTG is present during all steps, com- <br /> that cross hybridization occurs between IS or paralog-                      mencing with the induction of transposase in <span id='am-56' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-57' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to sup- <br /> containing sequences. The majority of these positive hybrid-                port replication of plasmids formed during deletion forma- <br /> ization signals do represent IS sequences found elsewhere in                tion. In addition to supporting replication of the newly <br />  <br />  <br />  <br />                                                                                                                 Genome Research             647 <br />                                                                                                                     www.genome.org <br />  Goryshin et al. <br />  <br />  <br />                                                                            depresses cell growth and creates selection against cells that <br />                                                                            had not undergone excision of the transposon-encoded origin <br />                                                                            of replication, as in these experiments, &gt;95% of the resulting <br />                                                                            colonies contained the desired <span id='am-45' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of deletions. <br />                                                                                  We used a coupled deletion/plasmid system for the <br />                                                                            search of essential genes. A total of 15 insertion events were <br />                                                                            treated independently as above to isolate deletion events. Fi- <br />                                                                            nal induced cultures for each were diluted and plated on agar <br />                                                                            medium containing chloramphenicol and IPTG. For each in- <br />                                                                            sert, a total of 20 colonies were analyzed by isolation of plas- <br />                                                                            mid DNA, followed by an estimation of plasmid size by <br />                                                                            agarose gel electrophoresis. In each case, the largest plasmid <br />                                                                            isolated was analyzed by DNA sequencing of the two trans- <br />                                                                            poson/chromosomal DNA junctions to determine the portion <br />                                                                            of the chromosome that was deleted. Cells from the colony <br />                                                                            were then streaked onto plates containing only LB (no IPTG) <br />                                                                            to determine whether cells could survive without the deleted <br />                                                                            chromosomal material. The colonies were also streaked on LB <br />                                                                            plates with chloramphenicol and no IPTG to confirm loss of <br />                                                                            plasmid without integration into the chromosome. Growth of <br />                                                                            cells on this medium was not detected in any case, indicating <br />                                                                            that reintegration of the plasmid did not occur. <br />                                                                                  Results of this analysis are shown in Table 2. Of 15 in- <br />                                                                            sertion strains, 11 yielded viable cells following plasmid loss <br />                                                                            from the largest isolated deletion. This indicated that these <br />                                                                            regions of DNA are dispensable. In the other four cases, cells <br />                                                                            were unable to grow without the deleted plasmid DNA, indi- <br />                                                                            cating the presence of at least one essential gene in the deleted <br />                                                                            material. An example of this is deletion 4.9. In this case, a <br />                                                                            smaller deletion, 4.6, was analyzed and found to survive in <br />                                                                            the absence of the plasmid (Fig. 6). In this case, the result <br />                                                                            indicates that genes glyS and/or glyQ are essential. <br />                                                                                  In the case of insert 8, only a short chromosomal DNA <br />                                                                            deletion was isolated. This transposon insert resides between <br />                                                                            open reading frames (ORFs), and the deletion does not con- <br />                                                                            tain any predicted ORFs. The inability to isolate larger dele- <br />                                                                            tions from this insert could be due to the fact that the area <br />                                                                            contains a large region of essential DNA that cannot be by- <br />                                                                            passed by typical deletion lengths. Alternatively, the localized <br />                                                                            condensation of the DNA may not be amenable to the forma- <br />                                                                            tion of large deletions. A final possibility is that the capture of <br />                                                                            DNA immediately next to the insert, when transferred to the <br />                                                                            moderate copy number plasmid, may be lethal to the cell. We <br />                                                                            cannot distinguish between these options, but this area con- <br />                                                                            tains a stretch of essential genes, including gyrB and dnaA. <br />                                                                                  The data shown in Table 2 were compared with known <br /> Figure 4 Pulsed-field gel electrophoretic analysis of MG1655 dele-         information about the essentiality of E. coli ORFs by using the <br /> tion derivatives after 20 rounds of transposition and deletion. Trans- <br />                                                                            PEC database (www.shigen.nig.ac.jp/ecoli/pec/). Of the four <br /> positional deletion of chromosomal DNA was performed recursively <br /> for a total of 20 cycles. After these 20 cycles, chromosomal DNA was       deletions that we determined to contain essential DNA, all <br /> prepared from 4 deletion strains as well as the initial strain MG1655.     but one contained an ORF(s) that had been reported previ- <br /> The chromosomal DNA was digested to completion with NotI restric-          ously to be essential. In the remaining case, the deletion oc- <br /> tion endonuclease and analyzed by pulsed-field gel electrophoresis.        curs in a single ORF, dnaK. Mutations in dnaK give rise to a <br /> Visualization by ethidium bromide staining and UV irradiation re-          conditionally essential phenotype at 37&#176;C (Bukau and Walker <br /> vealed the pattern of DNA bands. The pattern for MG1655 matches <br /> that reported previously (Heath et al. 1992), and the identification <br />                                                                            1989; Wild et al. 1992). Of the nonessential cases, all ORFs <br /> letter for each fragment is shown at right. For each of the deletion       have been reported as either nonessential or unknown. The <br /> strains, the pattern is altered and indicates a loss of DNA. The size of   deletions that fall under this category indicate that 40 ORFs of <br /> each new band in the four deletion strains was estimated by com-           unknown essentiality can now be considered nonessential. <br /> parison with DNA markers. The estimation for total amount of deleted <br /> DNA in each of the strains is shown at bottom. <br />                                                                            DISCUSSION <br />                                                                            In this communication, we describe a simple and reliable sys- <br /> formed plasmids, addition of IPTG activates origins of repli-              tem for making chromosomal deletions in E. coli K-12. This <br /> cation that remain on the chromosome in cells that have not                system should also be of use in any other bacterial species for <br /> undergone deletion formation. This activation presumably                   which the Tn5 transposome/electroporation strategy can be <br />  <br />  <br />  <br /> 648      Genome Research <br />            www.genome.org <br />                                                                         Tn5 Double Transposition and Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 5 Microarray deletion map of deletions found in strains &#9004;20-1 and &#9004;20-4. Each dot represents the log ratio of normalized signal intensities <br /> for a particular ORF from the deletion strain to its control strain counterpart. Deletion regions were determined by consecutive signal ratios of E. <br /> coli ORFs below 1 and P-values &lt;0.05. These hybridization results were plotted against base pair location for comparison with a NotI restriction map <br /> (Heath et al. 1992). <br />  <br />  <br /> utilized. We used transposition driven by Tn5 transposase de-                considered nonessential, and genes that do not contain <br /> rivatives in such a way that we can separate two transposition               knockouts are labeled as essential. It is then assumed that a <br /> events, each using different Tnp:transposon end combina-                     chromosome containing only the essential genes would be <br /> tions (Naumann and Reznikoff 2002). The first transposition                  sufficient for survival. However, this analysis assumes that <br /> event delivers components for the second transposition event                 genes that are not found to contain transposon knockouts are <br /> into the chromosome via transposase&#8211;DNA complex codeliv-                     essential when they may, instead, be poor targets for the <br /> ery (Goryshin et al. 2000). The second transposition event                   transposon and, more importantly, that removal of one gene <br /> generates a deletion or inversion event. By simple screening                 has no effect on the essentiality of remaining genes. This sec- <br /> for loss of antibiotic resistance, deletions with only a small               ond assumption likely results in an artificially low estimate of <br /> portion of the transposon left in the chromosome can be cho-                 the number of genes that are needed to form a viable organ- <br /> sen. As a variation of this technique, deleted material can be               ism. In any event, the construction of a living cell with a <br /> saved in the same cell as a conditional plasmid.                             minimal genome is desirable. <br />       Determining the minimal genome content is a topic of                        E. coli K-12 is an ideal subject for the minimal genome <br /> interest for many laboratories. One approach used for deter-                 construction. It has a short generation time, extensive genetic <br /> mining the minimal genome is to assemble a theoretical                       tools, and a wealth of knowledge of its genome and physiol- <br /> minimal genome in silico by the comparison of a variety of                   ogy. It therefore seems attractive to try to generate a minimal <br /> different microbial genomes. This method assumes that all                    or significantly reduced E. coli K-12 genome. In addition to <br /> essential DNA exists in homologous form in all genomes, and                  the knowledge gained by the creation of such a strain, the <br /> that all nonessential DNA will be absent in one or more cases.               strain itself could be useful. Two examples of the usefulness of <br /> Alternatively, the smallest genome among existing genomes                    such a strain are (1) overproduction of recombinant proteins <br /> (<span id='am-11' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span>mycoplasma</span>) has been analyzed by transposon knockout                        with fewer E. coli proteins that need to be removed, and (2) as <br /> mutagenesis and large-scale sequencing (Hutchison III et al.                 a simplified metabolic model that could be used for math- <br /> 1999). Any genes that are found as knockout mutants are                      ematical modeling of whole cell metabolism. We are currently <br />  <br />  <br />  <br />                                                                                                                   Genome Research             649 <br />                                                                                                                      www.genome.org <br />  Goryshin et al. <br />  <br />  <br />                                                                            chromosomal DNA is not a random coil, but rather is a com- <br />  Table 1. Genes Deleted in Strains &#9004;20-1 and &#9004;20-4 <br />                                                                            pact nucleoid body with a supercoiled domain structure that <br />          Deletion                                                          brings distant points into close proximity (Staczek and Hig- <br />                                              Map          Estimated        gins 1998). The isolation of two very large deletions (&gt;100 kb) <br />  &#9004;20-1        &#9004;20-4      B-Numbera         Positionb      Sizec (bp)       near the replicon terminus suggests that this region of the <br />                                                                            chromosome was unusually condensed. <br />  1.1                       b0116              2.8            1,424               Recently, two strategies for repetitive reduction of the <br />  1.2            4.1     b0288&#8211;b0315         6.5&#8211;7.2         29,606 <br />                                                                            E. coli chromosome have been described (Kolisnychenko et al. <br />  1.3                    b1188&#8211;b1197        26.6&#8211;26.9        11,667 <br />  1.4                    b1335&#8211;b1452        30.1&#8211;32.8       124,647         2002; Yu et al. 2002). In the first communication, 12 planned <br />                 4.2     b1336&#8211;b1471        30.1&#8211;33.3       145,467         deletions were introduced by the use of the efficient &#9261; red <br />  1.5            4.3     b1878&#8211;b1892        42.3&#8211;42.6        15,625         recombination system. This approach is different from ours in <br />                 4.4        b2379             53.8            1,238         several fundamental ways. First, the approach described by <br />  1.6                    b2649&#8211;b2655          60.0            3,604         Kolisnychenko et al. (2002) generates planned deletions and <br />                 4.5     b2648&#8211;b2655          60.0            4,117 <br />                 4.6     b2853&#8211;b2861        64.5&#8211;64.6         4,293 <br />                                                                            thus uses the prior knowledge of which genes are dispensable, <br />                 4.7        b3027             68.3              332         and requires knowledge of the genome sequence. Our system <br />  1.7                       b3647             82.3            1,688         utilizes a random approach and can be used without prior <br />  1.8            4.8     b3708&#8211;b3715        83.8&#8211;84.0         8,724         knowledge of which genes are dispensable or prior sequence <br />  1.9                       b4216             95.6              554         information. Secondly, the Kolisnychenko et al. (2002) ap- <br />  1.10           4.9     b4308&#8211;b4315        97.7&#8211;97.9         9,252         proach requires the use of the &#9261; red recombination system. <br />  1.11                   b4349&#8211;b4354        98.7&#8211;98.9         2,165 <br />                                                                            We do not know whether this system has a limited host range. <br />  a                                                                         The strategy described in this communication uses the Tn5 <br />    Deletion span based on b-number (Blattner et al. 1997). <br />  b <br />    Map position of genes lost within deletion span.                        transposase, which has been found to be active in all tested <br />  c <br />   Size is estimated from left end of first deleted ORF to right end of     bacterial species (Goryshin at al. 2000). Third, the strategy <br />  last detected ORF.                                                        described in this communication allows one to impose a se- <br />                                                                            lection for fast growing cells among several possible deletion <br />                                                                            strains. Finally, with our system, the deleted material can be <br /> going beyond the 20 rounds presented here in an effort to                  saved as a conditional plasmid. Yu and colleges used the Cre/ <br /> create such a strain. The average size of deletions observed in            loxP system delivered by Tn5 derivatives and P1 transduction <br /> our experiments for the lac operon area and random deletions               to locate two Tn5 inserts in the same cell (Yu et al. 2002). <br /> gives us hope that we will be able to reduce the size of the E.            Importantly, this system depends on E. coli genetic tools (P1 <br /> coli chromosome significantly in a reasonable time frame.                  transduction) and involves multiple steps for the formation of <br />       The large average deletion size is striking, because if the          each deletion. Furthermore, it fails to save deleted material as <br /> genomic DNA were a random coil, the intramolecular trans-                  a complementary plasmid. <br /> position site selection would be strongly biased toward dis-                     In this communication, we started a list of essential/ <br /> tances within a few hundred base pairs of the transposon end               nonessential genes for E. coli by using the Tn5Del8 system. In <br /> sequences. We observed this <span id='am-35' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-36' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>bias</span> in vitro, in which case, the              the future, it is technically possible to create a representative <br /> DNA was free from packing (York at al. 1998). One explana-                 library of deletions with complementary plasmids that would <br /> tion for the observed large in vivo deletion size is that the              cover the entire E. coli genome multiple times. Systematic <br />  <br />  <br />  <br />  Table 2. The Summary of Analyzed Deletion/Plasmid Combinations for Different Insertions <br />  <br />  Insert                Insert                                                                                                          Grow w/o <br />  No.         &#9004; &#9013;&#9265;.      End        &#9004; End        Size      Partial ORFs                       Complete ORFs                               IPTG <br />  <br />      1         1.3    1073109     1094823      21714     b1028; b1012    b1013; putA; putP; b1016&#8211;1018; ycdB; phoH; <br />                                                                             b1021&#8211;1025; tra5&#9004;3; b1027                                    Y <br />      2         2.8    2776930     2770104       6826     b2638; b2647    b2639&#8211;b2646                                                     Y <br />      3         3.1    4407731     4403605       4126     yjfl            yjeB; vacB; yjfH                                                Y <br />      4         4.2    3735047     3725227       9821     yiaB            xylB; xylA; xylF; xylG; xylH; xylR; b3570                       Y <br />      4         4.6    3735047     3723521      11526     none            yiaH; yiaA; yiaB; xylB; xylA; xylF; xylG; xylH; xylR; b3570     Y <br />      4         4.9    3735047     3719490      15557     yi5b            glyS; glyQ; yiaH; yiaA; yiaB; xylB; xylA; xylF; xylG;xylH; <br />                                                                             xylR; b3570                                                  N <br />   5           5.1     1224154     1210042      14112     mcrA; minD      b1160-b1173; minE                                               Y <br />   6           6.2     3045069     3042244       2825     bglA; gcvP      ygfF                                                            Y <br />   7           7.1     2779269     2767042      12227     b2633; b2647    b2634&#8211;B2646                                                     Y <br />   8           8.4     3882592     3882500         92     none            none                                                            Y <br />   9           9.7     4589240     4595093       5853     mdoB; tsr       yjjN; yjjM; b4356                                               Y <br />  10          10.3     4141018     4117696      23322     menA; frwC      hsIU; hsIV; ftsN; cytR; <span id='am-40' about='obo:NCBITaxon_558101' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_577241 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_116151 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Pria" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>priA</span>; rpmE; yiiX; metJ; metB; metL; <br />                                                                             metF; katG; yijE; yijF; gldA; talC; ptsA; yijl               N <br />  11          11.3     2793099     2795902       2803     gabP; b2668     ygaE; b2665&#8211;b2667                                               Y <br />  12          12.17      14166       12088       2078     none            dnaK                                                            N <br />  13          13.19    4542104     4546611       4507     fimC; fimH      fimD; fimF; fimG                                                Y <br />  14          14.17    3821524     3818998       2526     spoT            rpoZ; gmk                                                       N <br />  15          15.17    1570686     1544492      26194     yddC; yddG      fdnG; fdnH; fdnl; b1477; b1478; sfcA; rpsV; b1481; <br />                                                                             osmC; b1483&#8211;1491; xasA; gadB                                 Y <br />  <br />  <br />  <br />  <br /> 650        Genome Research <br />              www.genome.org <br />                                                                     Tn5 Double Transposition and Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 6 An example of plasmid dependent strains. (A) The gel showing two plasmids generated from the same insertion of deletion module <br /> Tn5Del8. (B) Strains 4.6 and 4.9 are streaked on TYE medium with and without IPTG. The difference between plasmids 4.6 and 4.9 lies in the <br /> absence of glyS and glyQ in plasmid 4.6. The elimination of both plasmids may indicate that either glyS or glysQ, or both, are essential. <br />  <br />  <br /> sequencing of deletion borders coupled with survival tests              EcoRI site of pGT6. The resulting plasmid contains the <br /> could be used to determine the essentiality of all of the genes         Tn5Del7 transposon with the structure shown in Figure 2. <br /> in the entire chromosome. This type of analysis could also be                 Plasmid pGT8 was the source of transposon Tn5Del8 <br /> adapted, through the use of an appropriate origin of replica-           (Fig. 2). The two KpnI&#8211;BssSI fragments of pGT7 were ligated <br />                                                                         with an AatII&#8211;BamHI fragment of pAM34 (ATCC 77185, Hare <br /> tion, for analyzing the chromosomes of other bacteria. <br />                                                                         et al. 2001). Tn5Del8 is similar in structure to Tn5Del7, except <br />                                                                         that it contains a regulated origin of replication. <br /> METHODS                                                                 Transposase Purification <br /> Medium                                                                  Transposase protein TnpsC7v2.0 was purified using IMPACT <br />                                                                         T7 system (NEB) as described previously (Naumann and <br /> For all experiments, we used <span id='am-6' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span>-Bertanii (LB) broth liquid           Reznikoff 2000). <br /> or agar medium or Lactose MacConkey agar medium (Sam- <br /> brook et. al. 1989) modified to contain the following antibi-           Transposome Complex Preparation <br /> otics when indicated: chloramphenicol (20 mg/L), kanamycin <br /> (40 mg/L), ampicillin (100mg/L).                                        Transposons Tn5Del7 and Tn5Del8 were cut out of the donor <br />                                                                         plasmid DNA by digestion with PshAI, followed by purifica- <br /> Bacterial Strains                                                       tion from an agarose gel using the QIAquick Gel Extraction <br />                                                                         Kit (QIAGEN). <br /> For making deletions, we used E. coli K12 strain MG1655                      Transposome complexes were assembled by incubation <br /> (Blattner et al. 1997). For DNA manipulations, we used E. coli          of precut transposon with TnpsC7v2.0 in 20 mM Tris Acetate <br /> K12 strain DH5&#9251; (Sambrook et al. 1989).                                 (pH 7.5), 100 mM K Glutamate for 1 h at 37&#176;C. The DNA:pro- <br />                                                                         tein molar ratio was 1:5, with a DNA concentration of <br /> Plasmids                                                                0.1&#181;g/&#181;l. <br /> Plasmid pGT7 was the source of transposon Tn5Del7 (Fig. 2). <br /> Plasmid pGT7 was constructed by inserting different compo-              Selection for Initial Transposition Events <br /> nents into pGT4 (Nauman and Reznikoff 2002). In many                    Electroporation of complexes (Goryshin et al. 2000) was done <br /> cloning steps, blunt ends were generated using T4 DNA poly-             using standard recommended conditions (2.5 Kv, 5mS). <br /> merase (Promega). Two ME sequences were taken from                           After electroporation, cells were recovered by incubation <br /> pPDM-2 (Epicentre) along with the KmR gene on a HindIII&#8211;                for 1 h in LB broth, and then plated on LB agar containing <br /> EaeI fragment that was ligated to the large HindIII&#8211;SmaI frag-          kanamycin and chloramphenicol. <br /> ment of pGT4, giving rise to pGT5. The Tnp EK/LP gene and <br /> the AraC gene with a portion of the KmR gene were taken                 Induction of Deletions <br /> from pGRARAK (I. Goryshin, unpubl.) on the BclI&#8211;ClaI frag- <br /> ment and ligated to the ClaI&#8211;NaeI large fragment of pGT5 to             (Secondary Transposition Events) <br /> give pGT6. Finally, the CmR gene from pACYC184 was taken                Individual or pooled colonies from this initial selection were <br /> by isolating a BsaAI&#8211;PshAI fragment and ligating it into the            inoculated into LB medium with arabinose at a concentration <br />  <br />  <br />  <br /> of 0.4%, and grown for 8 h at 37&#176;C. In the case of Tn5Del8,         <br /> IPTG was also included at a concentration of 1 mM. Cells were <br />                                                                    <br /> subcultured with a 500-fold dilution in LB medium (plus <br />                                                                     <br /> chloramphenicol and IPTG for Tn5Del8), grown overnight, <br />                                                                     <br /> and plated with an appropriate dilution on LB agar (plus <br />                                                                   <br /> chloramphenicol and IPTG for Tn5Del8) to obtain individual <br />                                                                    <br /> colonies for analysis. <br />                                                                    <br />      Screening for deletions with Tn5Del7 was done by rep- <br />                                                                    <br /> lica plating colonies from LB agar onto agar containing LB, <br />                                                                     <br /> LB-kanamycin, and LB-chloramphenicol. Cells sensitive to <br />                                                                     <br /> kanamycin were considered to have undergone transposition, <br />                                                                  <br /> and cells sensitive to both drugs were considered to have a <br />                                                                       <br /> deletion of adjacent DNA and the correct portion of the trans- <br />                                                                    <br /> poson with only a small transposon linker being left on the <br />                                                                    <br /> chromosome. In the case of Tn5Del8, the cells of interest were <br />                                                                    <br /> kanamycin sensitive and chloramphenicol resistant, as the <br /> replication of excised DNA circles was supported by the pres- <br /> ence of 1 mM IPTG in the medium. <br />                                                                     <br /> Microarray Analysis                                    <br /> We used spotted DNA microarrays containing 95% of the               <br /> 4290 ORFs from E. coli K-12 genome. These spotted microar-           <br /> rays were supplied by the Gene Expression Center, University   <br /> of Wisconsin. Genomic DNAs from E. coli strain MG1655,          <br /> &#9004;20-1 and &#9004;20-4 were prepared using a Master Pure DNA Pu-      <br /> rification Kit (Epicentre) digested with AluI (Promega) and         <br /> further purified by phenol/chloroform extraction and ethanol       <br /> precipitation. Genomic DNA labeling with Cy3 and Cy5                 <br /> dUTP (NEN, Life Science) and hybridizations were performed        <br /> according to protocols provided by the Gene Expression Cen-             <br /> ter, University of Wisconsin (http://www.gcow.wisc.edu/Gec/         <br /> index.htm). Microarray images were scanned using Packard          <br /> BioChip SA5000 and quantitated using Scanalyze 2.1. Nor-    <br /> malization of microarray signal intensities involved median       <br /> background subtraction and calculation of percent signal in-    <br /> tensities for each spot (Richmond et al. 1999). The percent       <br /> signal intensities were used to determine the ratio of deletion   <br /> strain to the control E. coli MG1655 signals. Values for percent  <br /> signal intensity were log transformed in order to combine          <br /> data for paired slide (dye swap) experiments. The ratios of       <br /> deletion strain to control values were used for subsequent       <br /> Z-score calculations to determine deletion sites. We estab-            <br /> lished the null hypothesis for the presence of DNA, which was           <br /> indicated by ratios with values close to 1 and that rejected the    <br /> null for ratios with P values &lt;0.05, which indicated the de-       <br /> leted material.             <br /> DNA Sequence Analysis                                         <br /> DNA sequencing was performed using an ABI PRISM model             <br /> 377, according to the standard Big Dye protocol. For the lacZ       <br /> insertion, we performed direct chromosomal DNA sequenc-            <br /> ing using primer FWD2, 5&#11032;-CAGATCTCATGCAAGCTTGA           <br /> GCTC-3&#11032;, which is complementary to the transposon linker.     <br /> For sequencing of deletions produced from this insertion, we        <br /> generated DNA using inverse PCR (Ochman et al. 1990) after            <br /> digestion of chromosomal DNA with FspI followed by ligation  <br /> and 30 cycles of PCR (30 sec at 94&#176;C, 1 min at 60&#176;C, 1 min at          <br /> 72&#176;C) with primers 5&#11032;-GGTCTGCTTTCTGACAAACTCGGGC-                   <br /> 3&#11032; and 5&#11032;-ACGCGAAATACGGGCAGACATGGCC-3&#11032; comple-                    <br /> mentary to transposon ends. PCR products were purified from            <br /> an agarose gel and sequenced with the standard Big Dye pro-            <br /> tocol.                                              <br /> Pulsed-Field Gel Electrophoresis                            <br /> Pulsed-field gel electrophoresis was performed using the              <br /> CHEF-DR II BioRad system according to the protocol found in         <br /> Heath et al. (1992), with minor technical modifications. Run-      <br /> ning time was 40 h, voltage = 180 v, ramping &#8211; 5 to 80 sec.            <br /> </body></html>