<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                      <span id='am-133' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-134' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-136'  ></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Meta</span> Gene 2 (2014) 819&#8211;830 <br />  <br />  <br />                                              Contents lists available ScienceDirect <br />  <br />                                                           Meta Gene <br />  <br />                                                          journal homepage: <br />  <br />  <br />  <br />  <br /> let-65 cytoplasmic methionyl tRNA synthetase <br /> in C. elegans <br /> Maha Z. Alriyami a,c,&#65533;?&#65533;, Martin R. Jones b, Robert C. Johnsen a, <br /> Yajnavalka Banerjee c, David L. Baillie a <br /> a <br />     Department Molecular Biology Biochemistry, Simon Fraser University Burnaby, 8888 University Drive, Burnaby, BC V5A 1S6 Canada <br /> b <br />     Genome Sciences Centre, British Columbia <span id='am-55' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-56' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Research Centre, 600 West 10th Avenue, Vancouver, BC V5Z 4E6, Canada <br /> c <br />     Department Biochemistry, College Medicine Health Sciences, Sultan Qaboos University, 35, Al-Khod 123, Oman <br />  <br />  <br />  <br />  <br /> a r t c l e            n f o                         b s t r c t <br />  <br /> Article history:                                         Cytoplasmic methionyl tRNA synthetase (MetRS) 20 <br /> Received 13 September 2013                               cytoplasmic aminoacyl tRNA synthetase enzymes (ARS). <span id='am-82' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> <span id='am-52' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> of <br /> 8 August 2014                                            enzymes catalyzes process fundamental protein translation. <br /> Accepted 11 August 2014 <br />                                                          Using combination genetic mapping, oligonucleotide array compar- <br /> Available online xxxx <br />                                                          ative genomic hybridization, phenotypic correlation, that <br />                                                          mutations essential gene, let-65, reside predicted <br /> Keywords: <br />                                                          <span id='am-60' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> homologue MetRS, named <br /> <span id='am-67' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span>Caenorhabditis</span> elegans <br /> let-65 <br />                                                          mars-1. We demonstrate lethality associated alleles of <br /> Aminoacyl tRNA synthetase, methionyl-tRNA                let-65 fully rescued transgenic array spans mars-1 geno- <br /> synthetase                                               mic region. Furthermore, sequence analysis reveals let-65 alleles <br /> Oligonucleotide array comparative genomic                lead alteration highly conserved amino acids. <br /> hybridization                                              &#169; 2014 The Authors. Published Elsevier B.V. This open access <br /> Essential genes                                                                              article CC BY-NC-ND license <br />                                                                              (http://creativecommons.org/licenses/by-nc-nd/4.0/). <br />  <br />  <br /> Introduction <br />  <br />    Analysis mutations affecting genes essential survival model organisms, the <br /> nematode Caenorhabditis elegans, instrumental developing understanding fundamental <br /> biological processes, example cell division morphology (Green et al., 2011). Many of <br /> these processes direct relevance understanding human diseases metabolic disorders <br /> and obesity (Hashmi et al., 2013). The generation mutations essential genes correlation a <br /> speci&#65533;?c genetic locus is, therefore, &#65533;?rst step understanding nature gene&apos;s function promoting <br /> survival. To easily identify genes essential survival model organism C. elegans, we <br /> created physical de&#65533;?ciency map encompasses portion Chromosome IV, allowing correlation <br />  <br />  &#65533;?&#65533; Corresponding author at: Department Biochemistry, College Medicine Health Sciences, Sultan Qaboos University, Oman, 35, <br /> Al-Khod 123, Oman. Tel.: +968 24141152; fax: +968 2441 3880. <br />     E-mail address: mahazr@squ.edu.om (M.Z. Alriyami) <br />  <br />  <br /> http://dx.doi.org/10.1016/j.mgene.2014.08.006 <br /> 2214-5400 &#169; 2014 The Authors. Published Elsevier B.V. This open access article CC BY-NC-ND license <br /> (http://creativecommons.org/licenses/by-nc-nd/4.0/). <br />  820                                  M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830 <br />  <br />  <br /> of genetically de&#65533;?ned mutations physically de&#65533;?ned candidate gene lists. In study, the <br /> use physical de&#65533;?ciency map identify essential gene (let-65), previously isolated ethyl <br /> methanesulfonate (EMS) genetic screens (Clark Baillie, 1992; Clark et al., 1988; Moerman Baillie, <br /> 1981; Rogalski Baillie, 1985; Rogalski et al., 1982). From analysis determined mutations in <br /> let-65 correspond methionyl tRNA synthetase gene, mars-1. <br />     Organisms utilize key components translate genetic information proteins: ribosomes, messenger <br /> RNAs (mRNA), transfer RNAs (tRNA), aminoacyl-tRNA synthetases (ARS), reviewed Brown et al. (2010). <br /> Ribosomes two-subunit organelles direct synthesis proteins translating nucleic acid triplets, carried <br /> by mRNAs, amino acids (Crick et al., 1961). mRNAs, generated DNA transcription, act trans- <br /> lation templates, tRNAs act interfaces mRNA strands amino acids added poly- <br /> peptide chain transferring required amino acids growing chain (Crick et al., 1961). <br />     The ARS family proteins encompasses 20 multi-domain enzymes. Each proteins <br /> speci&#65533;?cally recognizes amino acid corresponding subset cognate tRNA molecules catalyze <br /> the aminoacylation two-step reaction. In &#65533;?rst step speci&#65533;?c amino acid <br /> is activated ATP produce aminoacyl adenylate; second step amino acid transferred onto <br /> the 3&#8242;-end tRNA (Jakubowski, 2001). ARS categorized classes (I II) based their <br /> distinct core domains (Wolf et al., 1999). The catalytic domain <span id='am-72' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Class</span> I synthetases exhibit typical Rossman <br /> fold, three-layer topology (&#945;/&#946;/&#945;) inner core &#65533;?ve parallel <span id='am-22' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-27'  ></span>beta</span> strands. However, Class <br /> II synthetases generally consist &#946; strands (Berg et al., 2001). Class I II synthetases recognize <br /> different faces tRNA molecules (Berg et al., 2001). The tRNA CCA terminus (containing amino acid <br /> attachment site) adopts different conformations tRNA-synthetase complexes class. The CCA termini <br /> of Class I enzymes hairpin conformations, whereas, class II enzymes helical con&#65533;?rmations <br /> observed free tRNAs (Berg et al., 2001). An intriguing enzyme belongs family Class I <br /> synthetase, methionyl tRNA synthetase (MetRS), recognizes initiator tRNA the <br /> methionine carrying tRNA functions elongation peptide chains (Deniziak Barciszewski, <br /> 2001). <br />     In study, use oligonucleotide array Comparative Genomic Hybridization (aCGH) de&#65533;?ne phys- <br /> ical de&#65533;?ciency map region Chromosome IV (LGIV) nematode C. elegans. We use map to <br /> assign precisely de&#65533;?ned lists candidate genes previously generated genetically mapped mutations, <br /> which confer lethal phenotype organism. Using approach, demonstrate lethality <br /> associated alleles let-65 fully rescued transgenic array spans mars-1 genomic region. <br />  <br /> Materials methods <br />  <br /> Growth handling C. elegans <br />  <br />    Nematode strains cultured previously described (Brenner, 1974). Strains used study: N2 <br /> Bristol (BC00049), dpy-5(e907) (CB00907), let-65(s254); nT1(IV) (BC00962), let-65(s694); nT1(IV) <br /> (BC1121), let-65(s1154); nT1(IV) (BC2028), let-65(s1222); nT1(IV) (BC2116), let-65(s1083); nT1(IV) <br /> (BC1909), let-65(s1084); nT1(IV) (BC1910), let-65(s1730); nT1(IV) (BC3258), let-65(s1777); nT1(IV) <br /> (BC3305), let-65(s1083), unc-22(s7)(IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8656), let-65(s1222), <br /> unc-22(s7) (IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8696). <br />  <br /> Oligonucleotide array comparative genomic hybridization (aCGH) <span id='am-169' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-170'  ></span><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-173' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> analysis <br />  <br />       aCGH experiments analysis performed previously described (Jones et al., 2007). <br />  <br /> DNA sequence analysis <br />  <br />    Genomic DNA extracted let-65 homozygous animals alleles (s1730, s1083, s694, s1084, <br /> s254, s1154, s1777 s1222). PCR fragments spanning mars-1 ORF ampli&#65533;?ed genomic DNA <br /> templates primers designed using Primer 3 (Rozen Skaletsky, 2000) (primer sequences available <br /> upon request). The ampli&#65533;?ed templates subsequently sequenced (Macrogen, Korea) appropriate <br /> sequencing primers (Supplementary Table 1). <br />                                            M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830                                  821 <br />  <br />  <br /> Transgenic rescue <br />  <br />    The fosmid WRM0615dH10, spans mars-1 genomic region (11,631,182 11,665,626 bp on <br /> LGIV), injected syncytial gonads dpy-5 (e907) animals &#65533;?nal concentration 3 ng/&#956;l, <br /> along pharyngeal targeted green &#64258;uorescent protein (GFP) reporter myo-2::GFP 10 ng/&#956;l and <br /> the plasmid pCeh361,which contains wild-type dpy-5 rescues dpy-5 phenotype, 100 ng/&#956;l <br /> (Thacker et al., 2006). Injected worms plated (&#65533;?ve P0s plate) phenotypically wild-type animals <br /> from F1s generation expressed pharyngeal GFP reporter individually isolated propagated. <br /> GFP positive wild-type F2 animals transmitting transgenic array selected PCR speci&#65533;?c the <br /> fosmid backbone performed con&#65533;?rm presence fosmid (Supplementary Table 1). A single, stably <br /> transmitting, animal selected create transgenic lines. To conduct transgenic rescue experiments indi- <br /> vidual hermaphrodites carrying WRM0615dH10 + myo-2::GFP + pCeh361 crossed +/+ N2 males to <br /> obtain dpy-5 (e907)/+[WRM0615dH10 + myo-2::GFP + pCeh361] males. These GFP-expressing males were <br /> then crossed let-65 (s1222); unc-22 (s7) let-65 (s1803); unc-22 (s7) heterozygously maintained <br /> hermaphrodites. F1 progeny screened presence males ensure cross successful <br /> and F1 wild-type hermaphrodites plated individually. F2 animals individually plated screened for <br /> unconditional twitchers, indicative viable homozygous animals genotype let-65 (s1083 or <br /> s1222) unc-22 (s7)/let-65 (s1083 s1222) unc-22 (s7)(IV)[WRM0615dH10 + myo2::GFP + pCeh361]. <br /> Unconditional twitchers individually plated incubated 20 &#176;C assess rescue. <br />  <br /> Microscopy <br />  <br />    Screening transgenic animals performed using Zeiss Axioscope (Quorum Technologies) set a <br /> QImaging <span id='am-87' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Camera" datatype="xsd:string"></span>Camera</span> appropriate &#65533;?lter sets used GFP expression analysis. Animals were <br /> immobilized 100 mM sodium azide (in water) immediately prior imaging. <br />  <br /> Bioinformatics studies <br />  <br />    All reference DNA sequences derived WormBase release WS208. Analysis sequence data and <br /> image processing performed using BioEdit (http://www.mbio.ncsu.edu/BioEdit/BioEdit.html), <br /> WolFPSORT (Horton et al., 2007) (http://wolfpsort.org/), BLASTP. <br />  <br /> Results <br />  <br /> aCGH mapped de&#65533;?ciencies physically de&#65533;?ne seven zones unc-22 region LGIV <br />  <br />     In effort characterize genes essential survival C. elegans previously generated and <br /> mapped mutations confer lethal phenotype unc-22 region C. elegans genome (Rogalski <br /> et al., 1982). The unc-22 region (from unc-43 unc-31) represents approximately map units (mu) on <br /> chromosome IV, spanned 35 well-de&#65533;?ned genetic de&#65533;?ciencies (Clark Baillie, 1992; Clark <br />  <br /> Table 1 <br /> aCGH de&#65533;?ned de&#65533;?ciency breakpoints gene complement. <br />  <br />   Zone        Left breakpoint                      Right breakpoint                     Genes zones         Size kb <br />  <br />               Left (bp)         Right (bp)         Left (bp)          Right (bp)        Leftmost       Rightmost <br />  <br />   1           10,942,926        10,947,094         11,070,201         11,076,154        Y69E1A.1       Y5F2A.3     123&#8211;134 <br />   2           11,070,201        11,076,154         11,629,649         11,629,799        Y5F2A.4        lys-6       553&#8211;560 <br />   3           11,629,649        11,629,799         11,639,511         11,639,570        F58B3.4        F58B3.7a     10&#8211;11 <br />   4           11,639,511        11,639,570         11,983,621         11,983,655        F58B3.7a       unc-22a     343&#8211;344 <br />   5           11,983,621        11,983,655         12,243,159         12,251,414        unc-22a        sre-16      252&#8211;253 <br />   6           12,243,159        12,251,414         12,551,447         12,553,668        sre-19         ZK795.1a    315&#8211;317 <br />   7           12,551,447        12,553,668         12,757,572         12,776,111        ZK795.1a       inx-6       196&#8211;204 <br />  <br /> Regions annotated based probe sequence falling adjacent breakpoint position. <br />  a <br />     Breakpoints falling single gene. <br />  822                                  M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830 <br />  <br />  <br /> et al., 1988; Marra Baillie, 1994; Moerman Baillie, 1981; Rogalski Baillie, 1985; Rogalski et al., <br /> 1982; Schein et al., 1993). Previously, genetic analysis used identify position 31 lethal gene loci <br /> within genetic de&#65533;?ciency map (Clark et al., 1988). However, precise molecular extent of <br /> the de&#65533;?ciencies known construction de&#65533;?nitive candidate gene lists corresponded to <br /> each mutated loci possible. <br />     To physically de&#65533;?ne de&#65533;?ciency map unc-22 region analyzed informative <br /> de&#65533;?ciencies using array comparative genomic hybridization (aCGH). aCGH method used compare <br /> the DNA ratio individual samples organism <span id='am-100' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> determine copy number var- <br /> iations chromosomal genome-wide scale (Dhami et al., 2005). This method used determine <br /> the precise physical extent genetic de&#65533;?ciencies C. elegans (Jones et al., 2007). In cases, array data <br /> was suf&#65533;?cient position de&#65533;?ciency deletion breakpoints single-gene resolution approximately <br /> 6 kb (Table 1). Deletion breakpoints fell single genes genes. Using data a <br /> seven-zone physical de&#65533;?ciency map constructed spanning approximately 800 kb unc-22 region <br /> (Fig. 1A Table 1). Furthermore, incorporating previously generated genetic mapping data mo- <br /> lecularly unidenti&#65533;?ed lethal loci known map region, able assign locus precisely <br /> de&#65533;?ned list candidate genes. The largest number lethal loci map zone de&#65533;?ciency map <br /> (nine genes). Zone contains seven lethal loci, zones seven contain lethal loci each. <br /> Zones &#65533;?ve contain lethal loci each. Finally, single lethal locus positioned zone <br /> three (Fig. 1A). <br />  <br /> Sequence analysis let-65 <br />  <br />     let-65 single lethal loci mapped zone de&#65533;?ciency map (Fig. 1A). Based the <br /> physical extent zone annotated genes, F58B3.4, F58B3.5, F58B3.6 F58B3.7 candi- <br /> dates let-65. let-65 represented alleles, isolated EMS screens larval lethal- <br /> ity (Clark Baillie, 1992; Clark et al., 1988; Rogalski Baillie, 1985; Rogalski et al., 1982). Since mutations <br /> in let-65 confer lethal phenotype correlated available RNAi data order rank candidate genes <br /> based severity reported RNAi phenotype (Kamath et al., 2003; Maeda et al., 2001; Piano <br /> et al., 2002; Simmer et al., 2003; Sonnichsen et al., 2005) (Fig. 1B). Using approach identi&#65533;?ed <br /> F58B3.5 F58B3.4 strongest candidate genes. To identify mutations F58B3.5 F58B3.4 <br /> their genomic regions ampli&#65533;?ed PCR let-65 (s254) homozygous animals sequenced. A <br /> comparison sequence data reference C. elegans genome (WormBase release WS208) revealed no <br /> mutations associated F58B3.4. However, point mutation identi&#65533;?ed coding region <br /> of F58B3.5. This mutation resulted glutamic acid lysine substitution (Glu364Lys) highly conserved <br /> amino acid (Fig. 2 Table 2). To provide additional con&#65533;?rmation mutation F58B3.5 associated <br /> with let-65 sequenced F58B3.5 ORF remaining let-65 alleles, detecting presence additional <br /> mutations seven alleles tested. All mutations identi&#65533;?ed G &#8594; C A &#8594; T transitions, con- <br /> sistent known mutational spectrum ethane methylsulfonate (EMS) mutagenesis (Fig. 2, Table 2) <br /> (Maple Moller, 2007). In case, exception s1777 s1222, identi&#65533;?ed mutations were <br /> substitutions highly conserved amino acids predicted MetRS domain F58B3.5 (Figs. 1C 2). The <br /> lesion s1777 resulted 508 amino acid truncation, s1222 contained mutation &#65533;?rst base the <br /> &#65533;?rst intron. s1222 potentially prevents splicing &#65533;?rst intron, result premature trunca- <br /> tion protein amino acid 28 (Fig. 1C). Together data demonstrate let-65 F58B3.5 repre- <br /> sent gene. F58B3.5 encodes cytoplasmic Class I synthetase, methionyl tRNA synthetase (MetRS) <br /> that evolutionary conserved, 58% identity 71% similarity human homologue (BlastP <br /> alignment Altschul et al., 1997, 2005) (Fig. 2). We renamed F58B3.5 mars-1 (Methionyl Amino-acyl <br /> tRNA Synthetase-1). <br />  <br /> Rescue let-65 fosmid containing mars-1 <br />  <br />    We shown mutations mars-1 present let-65 animals. To provide evidence that <br /> these mutations caused lethality associated let-65, performed transgenic rescue experiments. <br /> The fosmid clone WRM0615dH10 spans entire mars-1 genomic region, excludes F58B3.4 ORF <br /> (Fig. 1B). We transgenic animals carrying WRM0615dH10 extra-chromosomal array <br />                                              M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830                                         823 <br />  <br />  <br /> A <br />  <br />  <br />  <br />  <br /> B <br />  <br />  <br />  <br />  <br /> C <br />  <br />  <br />  <br />  <br /> Fig. 1. Physical map unc-22 region LGIV including aCGH de&#65533;?ciency mapping data. A. Schematic LGIV showing regions <br /> uncovered de&#65533;?ciencies analyzed study shown &#65533;?gure (dark gray). aCGH data region four <br /> analyzed de&#65533;?ciencies shown. Commonly used genetic markers physical positions de&#65533;?ciency breakpoints (estimated <br /> to 0.5 kb) shown array data. A schematic describing seven zones de&#65533;?ned breakpoints de&#65533;?ciencies is <br /> shown array data. let-56 zone &#65533;?ve zone six. B. An expansion zone showing candidate genes for <br /> let-65. The region covered fosmid used complementation tests depicted gene models. C. Schematic showing protein <br /> structures human MetRS C. elegans MARS-1. Conserved domains locations indicated. The positions identi&#65533;?ed <br /> mars-1 mutations shown. Figure scale. <br />  824                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830 <br />  <br />  <br />  <br />  <br /> Fig. 2. C. elegans MARS-1 alignment human (GenBank accession no. NP_004981), D. <span id='am-164' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-165' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167'  ></span><span id='am-168' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span> (GenBank accession no. <br /> NP_611382), S. cerevisiae (GenBank accession no. CAA97293) orthologs showing conserved amino acids identi&#65533;?ed let-65 <br /> mutations. MARS-1 evolutionarily conserved 58% identity 71% similarity human ortholog. Red lines indicate the <br /> positions amino acid changes let-65 alleles. s1222 prevents splicing &#65533;?rst intron. s1777 induces stop <span id='am-92' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> prema- <br /> turely truncates protein. <br />                                             M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830                        825 <br />  <br />  <br /> Table 2 <br /> Mutations identi&#65533;?ed let-65 sequencing BioEdit analysis. <br />  <br />   Allele                       Nucleotide change                         Codon mutation            Amino acid change <br />  <br />   s174                         ND                                        ND                        NDa <br />   s254                         G1188A                                    gag &#8594; aag                 Glu-364 &#8594; Lys <br />   s694                         G655A                                     gga &#8594; gaa                 Gly-186 &#8594; Glu <br />   s1083                        G429A                                     gga &#8594; gaa                 Gly-126 &#8594; Glu <br />   s1084                        G921A                                     gga &#8594; aga                 Gly-275 &#8594; Arg <br />   s1154                        C1297T                                    ccc &#8594; ctc                 Pro-400 &#8594; Leu <br />   s1222                        G81A                                      (&#8722;) &#8594; aga                 (&#8722;)27 &#8594; Arg <br />   s1730                        C174T                                     gct &#8594; gtt                 <span id='am-149' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-153'  ></span>Ala</span>-41 &#8594; Val <br />   s1777                        G1324A                                    tgg &#8594; tag                 Trp-409 &#8594; Stop <br />  a <br />      Not determined &#8212; potentially complex lesion. <br />  <br />  <br /> successfully speci&#65533;?cally rescued lethality associated alleles let-65 tested (s1083 s1222). <br /> Furthermore transgenic animals super&#65533;?cially wild type appearance twitching pheno- <br /> type presence unc-22 (s7) homozygous mutation, used visible phenotypic marker. <br /> Individual viable animals isolated start strains BC8656 BC8696 alleles s1083 s1222 respec- <br /> tively, cultured multiple generations con&#65533;?rm continued rescue lethal phenotype. Together the <br /> sequence rescue data con&#65533;?rm mutation mars-1 gives rise lethal phenotype let-65 animals. <br />  <br /> Mitochondrial methionyl tRNA synthetase candidate <br />  <br />     MARS-1 predicted encode cytoplasmic form MetRS (Havrylenko et al., 2010). Computa- <br /> tional analysis predicted MARS-1 protein (WormBase WS209) demonstrated mars-1 encodes a <br /> protein localizes subcellularly cytoplasm. BLASTP analysis Y105E8A.20 (Altschul et al., 1997), <br /> a second gene predicted encode methionyl tRNA synthetase C. elegans (WormBase WS211), implies <br /> mitochondrial localization encoded protein. <br />  <br /> Discussion <br />  <br />     In study used aCGH de&#65533;?ne physical de&#65533;?ciency map 2 Mb portion LGIV C. elegans. <br /> Using de&#65533;?ciency map correlated physical candidate gene lists mutations number previously <br /> identi&#65533;?ed essential genes (Clark, 1990; Clark Lyckegaard, 1988; Moerman, 1980; Rogalski Baillie, <br /> 1985; Rogalski et al., 1982). From analysis able determine genetically identi&#65533;?ed <br /> locus let-65 corresponds mars-1, encodes C. elegans homologue MetRS. let-65 number <br /> of lethal genes previously identi&#65533;?ed EMS screens essential genes right end LGIV. These screens <br /> generated seven alleles let-65 resulted lethality (s253, s1777, s1083, s1154, s1222 s1730), <br /> and additional allele causing sterility (s694). Previously reported inactivation MARS-1 by <br /> RNAi leads embryonic lethality (Kamath et al., 2003; Sonnichsen et al., 2005). Moreover, loss MARS-1 <br /> function lead sterility (Havrylenko et al., 2011). Furthermore, correlation available phenotypic in- <br /> formation ARS family members deleted targeted mutations demonstrates complete correlation <br /> of lethal sterile phenotypes, highlighting essential nature family enzymes (Table 3). Together <br /> these data indicate mars-1 essential processes require large scale protein biosynthesis, as <br /> embryogenesis (Havrylenko et al., 2011). <br />     In C. elegans cytoplasmic MetRS (MARS-1) 917 amino acid protein multiple predicted func- <br /> tional domains. The minimal core enzyme, bacteria <span id='am-1' about='obo:NCBITaxon_2713' typeof='owl:Thing obo:NCBITaxon_187857 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_64898 obo:NCBITaxon_200783 obo:NCBITaxon_32069'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Aquifex" datatype="xsd:string"></span><span id='am-5'  ></span>Aquifex</span> aeolicus, consists core catalytic <br /> domain (CAT), distinguished Rossmann fold characteristic domain Class I aminoacyl <br /> tRNA synthetase enzymes (catalyzes aminoacylation reaction), anticodon-binding domain (ABD) <br /> (promotes tRNA&#8211;protein association) (Nakanishi et al., 2005). Additional domains, eukaryotes, are <br /> appended core domain enzyme believed increase stability of <br /> tRNA&#8211;protein complexes (Havrylenko et al., 2010). In humans, MARS carries C-terminal domain causes <br /> a slow release aminoacyl-tRNA provides aminoacylation reaction limiting step (Havrylenko <br /> et al., 2010). In C. elegans, MARS-1 protein model contains t-RNA binding domain (tRBD) 170 <br />  826                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830 <br />  <br />  <br /> Table 3 <br /> Correlation known viability phenotypes ARS-family genes. <br /> Data source: WormBase, http://www.wormbase.org, release WS243, May 2014. <br />  <br />   Gene                                   RNAi phenotype                                    Mutation phenotype [allele] <br />  <br />   aars-1                                      ste <br />   aars-2                                      emb; lvl; ste <br />   cars-1                                      emb; let; lvl; ste <br />   cars-2                                      ste <br />   dars-1                                      emb; lvl; ste <br />   dars-2                                      emb; lvl; ste                                     let; ste [tm3799] <br />   ears-1                                      emb; lvl <br />   ears-2                                      emb; lvl; ste <br />   fars-1                                      emb; lvl; ste <br />   fars-2                                      ND <br />   fars-3                                      emb; lvl; ste <br />   gars-1                                      emb; lvl; ste <br />   hars-1                                      emb; let; lvl; ste                                let; ste [tm4074] (Pierce et al., 2011) <br />   iars-1                                      emb; ste <br />   iars-1                                      emb; let <br />   kars-1                                      emb; lvl; ste <br />   lars-1                                      emb; let; lvl; ste <br />   lars-2                                      emb                                               let; ste [tm5774] <br />   mars-1                                      emb; lvl <br />   nars-1                                      emb; let; lvl; ste                                let; ste [tm524] <br />   nars-2                                      ND <br />   pars-1                                      emb; let; lvl; ste <br />   pars-2                                      ND <br />   qars-1                                      emb; let; lvl; ste <br />   rars-1                                      emb; lvl; ste                                     emb; lvl; ste [gc47] <br />   rars-2                                      emb; ste <br />   sars-1                                      emb; lvl; ste <br />   sars-2                                      emb <br />   tars-1                                      emb; let; lvl; ste <br />   vars-1                                      emb; lvl; ste <br />   vars-2                                      emb; let; lvl; ste                                let; ste [tm3947] <br />   wars-1                                      emb; lvl; ste <br />   yars-1                                      emb; lvl; ste <br />  <br /> emb = embryonic lethality, lvl = larval lethality, let = lethal, ste = sterile, ND = determined. <br />  <br />  <br /> residues appended <span id='am-144' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-145'  ></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span> C-terminus enzyme, homologous tRBD domain appended <br /> at C-terminus MetRS Oriza sativa (rice) (Havrylenko et al., 2010). However, C. elegans, domain <br /> is separated ABD domain putative protein-binding domain approximately 150 residues <br /> (Havrylenko et al., 2010). This domain similar human P43 protein, non-synthetase compo- <br /> nent aminoacyl tRNA synthetase complex required assembly protein complex <br /> (Havrylenko et al., 2011). The predicted C. elegans MARS-1 catalytic core domain comprises amino acids 36 to <br /> 407 (Marchler-Bauer et al., 2009). In let-65 mutations reside domain (s1730, s1083, <br /> s694, s1084, s254 s1154) suggesting disrupt function catalytic core. The EMS induced <br /> mutation let-65(s1777) creates truncated protein lacks anticodon-binding domain (AA416 to <br /> AA545) (Marchler-Bauer et al., 2009) tRNA binding domain (AA755 AA857) (Marchler-Bauer <br /> et al., 2009). let-65(s1222) contains mutation &#65533;?rst base &#65533;?rst intron putatively preventing splic- <br /> ing &#65533;?rst intron resulting putative premature truncation amino acid 28. <br />     mars-1 highly conserved encodes protein unique signi&#65533;?cance translational control. Struc- <br /> tures MARS-1 <span id='am-110' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-119'  ></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> (Mechulam et al., 1999) <span id='am-103' about='obo:NCBITaxon_270' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_1297'><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Thermus" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:8240955" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:5781580" datatype="xsd:string"></span>Thermus</span> thermophiles (<span id='am-159' about='obo:NCBITaxon_940433' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_6073 obo:NCBITaxon_6074 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6097 obo:NCBITaxon_37516 obo:NCBITaxon_940447'><span id='am-160' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162'  ></span><span id='am-163' property="rdfs:label" content="Sugiura" datatype="xsd:string"></span>Sugiura</span> et al., 2000) <br /> have reported. MARS-1 E. coli greater identity T. thermophiles protein C. elegans. <br /> Amino acids altered C. elegans missense mutations located conserved regions both <br /> E. coli T. thermophiles: let-65(s1083): &#946;C; let-65(s694): turn &#946;2 &#946;3 (zinc-binding domain); <br /> let-65(s254): turn &#946;7 &#945;6; let-65(s1154): turn &#945;7 &#946;8. let-65(s1730) turn <br /> between &#946;A &#945;A let-65(s1084) turn &#946;5 &#946;6. All identi&#65533;?ed let-65 changes located <br /> in E. coli Rossman-like fold. <br />                                      M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830                         827 <br />  <br />  <br />     let-65(s1777) contains nonsense mutation tryptophan codon amino acid 408 an <br /> amber stop codon. let-65(s1222) mutation putatively prevents splicing &#65533;?rst intron resulting in <br /> premature truncation, truncated protein lacks majority synthetase including catalytic <br /> domain, binding domains, anticodon-binding domain, tRNA binding domain. s1777 and <br /> s1222 are, therefore, putatively null alleles. The phenotypes (mid-larval arrest) (Clark Baillie, 1992; <br /> Clark et al., 1988) mutants indistinguishable let-65 alleles. Based obser- <br /> vation predict alleles null MARS-1 function, reduce MARS-1 activity threshold <br /> required survival. Moreover, survival <span id='am-138' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-139' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-142'  ></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span>nematodes</span>, containing alleles mid-larva, suggests that <br /> for early developmental stages, mother supplies MARS-1 eggs oogenesis. <br />     let-65(s694) let-65(s1083) contain mutations change <span id='am-10' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-11' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span> glutamic acid amino acids 186 <br /> and 126 respectively. These amino acids sequence blocks conserved human yeast. <br /> let-65(s1084) contains mutation changes glycine arginine amino acid 275. Generally, change <br /> from small non-charged non-hydrophilic glycine large positively charged hydrophilic arginine could <br /> disrupt structure protein render non-functional, case let-65 would <br /> cause lethality. let-65 (s1730) contains mutation changes alanine valine position 41. In humans, <br /> <span id='am-75' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span>Drosophila melanogaster</span> <span id='am-28' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, serine codon located position within <br /> a highly conserved sequence block. Because change polar amino acid (serine) non-polar <br /> amino acid (valine) disrupt protein&apos;s structure replacing alanine valine analyzed <br /> the available sequences con&#65533;?rm C. elegans wild type reference sequence position 41. After analyzing <br /> the sequence stretch let-65 alleles concluded de&#65533;?nitively alanine present at <br /> position 41. The mutation let-65(s254) replaces conserved acidic amino acid (glutamic acid) a <br /> basic amino acid (lysine) position 364. This position catalytic domain MARS-1. let- <br /> 65(s1154) polar amino acid (proline) instead non-polar amino acid (leucine) position 400 <br /> that catalytic domain. Additionally, alterations proline &#64258;anking regions active <br /> sites known change activity protein (Proline bracket hypothesis) (Kini et al., 1998), this <br /> mutation exhibit drastic effects MARS-1 activity. <br />     Typically, eukaryotic cells forms ARS: cytoplasmic mitochondrial (Walker et al., 1983). <br /> MARS-1 localizes cytoplasm (Havrylenko et al., 2010). Therefore, attempted &#65533;?nd candidate <br /> gene coding mitochondrial form enzyme. The C. elegans genome contains second ORF, <br /> Y105E8A.20 predicted encode methionyl tRNA synthetase (WormBase WS211). Y105E8A.20 a <br /> 1221 bp gene encodes 406 amino acid protein. BlastP used search Swiss-Prot database <br /> with MARS-1 amino acid sequences queries (Altschul et al., 1997, 2005). In BlastP searches <br /> S. cerevisiae used canonical genome identify protein localizations C. elegans. S. cerevisiae <br /> was used it&apos;s simplest regard identifying open reading frames (ORFs), the <br /> most highly characterized eukaryotic genome (Fisk et al., 2006). BlastP localization, based S. cerevisiae <br /> matches, predicted mitochondrial localization protein encoded Y105E8A.20. <br />     The C. elegans million mutations project (MMP) comprises library 2007 mutagenized ho- <br /> mozygous C. elegans strains sequenced identify resulting mutations. The aim this <br /> project generate mutant alleles gene C. elegans genome. However, limitation <br /> of approach severely detrimental mutations genes required viability select- <br /> ed strains viable fertile. Of 14 MMP mutations recorded for <br /> mars-1, missense mutations single exception mutation predicted affect <br /> splicing intron C-terminus protein. This intron conserved <span id='am-97' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> <br /> (Table 4). Out remaining 13 alleles 10 affect residues conserved. Interestingly <br /> three mutations reside catalytic core enzyme. Since strains viable is, how- <br /> ever, unlikely lesions severely affect function protein. Alternatively secondary <br /> mutations strains present compensate detrimental effect on <br /> enzyme function. Further studies required ascertain case. <br />     In study isolated identi&#65533;?ed mars-1 alleles lead loss-of-function pheno- <br /> type. Furthermore, number alleles isolated mars-1 greater average number alleles <br /> isolated genes screens type (Berg et al., 2001; Brenner, 1974; Deniziak Barciszewski, <br /> 2001; Horton et al., 2007; Jones et al., 2007; Rozen Skaletsky, 2000; Thacker et al., 2006). This indicates <br /> that let-65 (mars-1) large mutagenic target EMS. The availability loss function alleles allows <br /> for exploration gene&apos;s structure function tractable model system. Moreover, will <br />  828                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830 <br />  <br />  <br /> Table 4 <br /> mars-1 MMP mutations. <br />  <br />   Allele            Mutation           Effect                  Domain affected            Type                    Conserved residue <br />  <br />   gk213073          C&#8594;T                A887T                   None                       Missense                No <br />   gk410589          G&#8594;A                P865S                   None                       Missense                No <br />   gk445673          C&#8594;T                Affects splicing        None                       Intron, splicing        No <br />   gk485164          C&#8594;T                A453T                   Anticodon_1                Missense                No <br />   gk509072          C&#8594;T                D190N                   tRNA-synt_1g               Missense                Yes <br />   gk515000          C&#8594;T                G668E                   coiled_coil_region         Missense                No <br />   gk541772          G&#8594;A                P265L                   tRNA-synt_1g               Missense                Yes <br />   gk550543          C&#8594;T                A613T                   Anticodon_1                Missense                No <br />   gk636296          G&#8594;A                S837L                   tRNA_bind                  Missense                No <br />   gk656240          T&#8594;C                M844V                   tRNA_bind                  Missense                No <br />   gk719138          C&#8594;T                A431T                   tRNA-synt_1g               Missense                Yes <br />   gk780068          C&#8594;T                D752N                   None                       Missense                No <br />   gk793812          A&#8594;T                F412L                   tRNA-synt_1g               Missense                No <br />   gk889082          G&#8594;A                A520V                   Anticodon_1                Missense                No <br />  <br />  <br /> facilitate better understanding ARS, lead deeper understanding complete protein <br /> synthesis machinery organisms including humans. <br />     Supplementary data article online http://dx.doi.org/10.1016/j.mgene.2014.08.006. <br />  <br />  <br /> Abbreviations <br /> mRNA      messenger RNAs <br /> tRNA      transfer RNAs <br /> ARS       aminoacyl-tRNA synthetases <br /> METRS methionyl tRNA synthetase <br /> aCGH      oligonucleotide array comparative genomic hybridization <br /> EMS       ethyl methanesulfonate <br /> mars-1 C. elegans methionyl tRNA synthetase encoding gene <br /> GFP       green &#64258;uorescent protein <br /> NLS       nuclear localization sequence <br /> MARS-1 C. elegans methionyl tRNA synthetase <br />  <br />  <br />  <br /> Funding <br />  <br />     This project funded Natural Sciences Engineering Research Council Canada (NSERC) <br /> [RGPIN9975] Canada. Maha Z. AlRiyami supported fellowship Sultan Qaboos University, Sultanate <br /> of Oman. <br />  <br />  <br /> Acknowledgments <br />  <br />    We thank Domena Tu help injecting constructs study. We thank Shu Yi <br /> Chua designing fosmid PCR primers. <br />  <br />  <br /> References <br />  <br /> Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J., 1997. Gapped BLAST PSI-BLAST: new <br />     generation protein database search programs. Nucleic Acids Res. 25, 3389&#8211;3402. <br /> Altschul, S.F., Wootton, J.C., Gertz, E.M., Agarwala, R., Morgulis, A., Schaffer, A.A., Yu, Y.K., 2005. Protein database searches using <br />     compositionally adjusted substitution matrices. FEBS J. 272, 5101&#8211;5109. <br /> Berg, J.M., Tymoczko, J.L., Stryer, L., 2001. Biochemistry. Biochemistry. W.H. Freeman Company, New York, p. 822. <br /> Brenner, S., 1974. The genetics Caenorhabditis elegans. Genetics 77, 71&#8211;94. <br /> Brown, M.V., Reader, J.S., Tzima, E., 2010. Mammalian aminoacyl-tRNA synthetases: cell signaling functions protein translation <br />     machinery. Vasc. Pharmacol. 52, 21&#8211;26. <br />                                                   M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#8211;830                                                    829 <br />  <br /> Clark, D.V., 1990. The unc-22 (IV) region Caenorhabditis elegans: genetic analysis molecular mapping(PhD Thesis) Simon Fraser <br />     University. <br /> Clark, D.V., Baillie, D.L., 1992. Genetic analysis complementation germ-line transformation lethal mutations unc-22 IV <br />     region Caenorhabditis elegans. Mol. Gen. Genet. 232, 97&#8211;105. <br /> Clark, A.G., Lyckegaard, E.M., 1988. Natural selection nuclear cytoplasmic transmission. III. Joint analysis segregation and <br />     mtDNA Drosophila melanogaster. Genetics 118, 471&#8211;481. <br /> Clark, D.V., Rogalski, T.M., Donati, L.M., Baillie, D.L., 1988. The unc-22(IV) region Caenorhabditis elegans: genetic analysis lethal <br />     mutations. Genetics 119, 345&#8211;353. <br /> Crick, F.H., Barnett, L., Brenner, S., Watts-Tobin, R.J., 1961. General nature genetic code proteins. Nature 192, 1227&#8211;1232. <br /> Deniziak, M.A., Barciszewski, J., 2001. Methionyl-tRNA synthetase. Acta Biochim. Pol. 48, 337&#8211;350. <br /> Dhami, P., Coffey, A.J., Abbs, S., Vermeesch, J.R., Dumanski, J.P., Woodward, K.J., Andrews, R.M., Langford, C., Vetrie, D., 2005. Exon array <br />     CGH: detection copy-number changes resolution individual exons human genome. Am. J. Hum. Genet. 76, <br />     750&#8211;762. <br /> Fisk, D.G., Ball, C.A., Dolinski, K., Engel, S.R., Hong, E.L., Issel-Tarver, L., Schwartz, K., Sethuraman, A., Botstein, D., Cherry, J.M., 2006. <br />     <span id='am-6' about='obo:NCBITaxon_559292' typeof='owl:Thing obo:NCBITaxon_4932 obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Saccharomyces cerevisiae S288c" datatype="xsd:string"></span>Saccharomyces cerevisiae S288C</span> genome annotation: working hypothesis. Yeast 23, 857&#8211;865. <br /> Green, R.A., Kao, H.L., Audhya, A., Arur, S., Mayers, J.R., Fridolfsson, H.N., Schulman, M., Schloissnig, S., Niessen, S., Laband, K., Wang, S., <br />     Starr, D.A., Hyman, A.A., Schedl, T., Desai, A., Piano, F., Gunsalus, K.C., Oegema, K., 2011. A high-resolution C. elegans essential gene <br />     network based phenotypic pro&#65533;?ling complex tissue. Cell 145, 470&#8211;482. <br /> Hashmi, S., Wang, Y., Parhar, R.S., Collison, K.S., Conca, W., Al-Mohanna, F., Gaugler, R., 2013. A C. elegans model study human metabolic <br />     regulation. Nutr. Metab. 10, 31. <br /> Havrylenko, S., Legouis, R., Negrutskii, B., Mirande, M., 2010. Methionyl-tRNA synthetase Caenorhabditis elegans: speci&#65533;?c <br />     multidomain organization convergent functional evolution. Protein Sci. 19, 2475&#8211;2484. <br /> Havrylenko, S., Legouis, R., Negrutskii, B., Mirande, M., 2011. Caenorhabditis elegans evolves new architecture multi-aminoacyl- <br />     tRNA synthetase complex. J. Biol. Chem. 286, 28476&#8211;28487. <br /> Horton, P., Park, K.J., Obayashi, T., Fujita, N., Harada, H., Adams-Collier, C.J., Nakai, K., 2007. WoLF PSORT: protein localization predictor. <br />     Nucleic Acids Res. 35, W585&#8211;W587. <br /> Jakubowski, H., 2001. Translational accuracy aminoacyl-tRNA synthetases: implications atherosclerosis. J. Nutr. 131, 2983S&#8211;2987S. <br /> Jones, M.R., Maydan, J.S., Flibotte, S., Moerman, D.G., Baillie, D.L., 2007. Oligonucleotide array comparative genomic hybridization <br />     (oaCGH) based characterization genetic de&#65533;?ciencies aid gene mapping Caenorhabditis elegans. BMC Genomics 8, 402. <br /> Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M., Welchman, D.P., <br />     Zipperlen, P., Ahringer, J., 2003. Systematic functional analysis Caenorhabditis elegans genome using RNAi. Nature 421, <br />     231&#8211;237. <br /> Kini, R.M., Caldwell, R.A., Wu, Q.Y., Baumgarten, C.M., Feher, J.J., Evans, H.J., 1998. Flanking proline residues identify L-type Ca2+ <br />     channel binding site calciseptine FS2. Biochemistry 37, 9058&#8211;9063. <br /> Maeda, I., Kohara, Y., Yamamoto, M., Sugimoto, A., 2001. Large-scale analysis gene function Caenorhabditis elegans high- <br />     throughput RNAi. Curr. Biol. 11, 171&#8211;176. <br /> Maple, J., Moller, S.G., 2007. Mutagenesis <span id='am-15' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-16' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span>Arabidopsis</span>. Methods Mol. Biol. (Clifton, N.J.) 362, 197&#8211;206. <br /> Marchler-Bauer, A., Anderson, J.B., Chitsaz, F., Derbyshire, M.K., DeWeese-Scott, C., Fong, J.H., Geer, L.Y., Geer, R.C., Gonzales, N.R., Gwadz, <br />     M., He, S., Hurwitz, D.I., Jackson, J.D., Ke, Z., Lanczycki, C.J., Liebert, C.A., Liu, C., Lu, F., Lu, S., Marchler, G.H., Mullokandov, M., Song, J.S., <br />     Tasneem, A., Thanki, N., Yamashita, R.A., Zhang, D., Zhang, N., Bryant, S.H., 2009. CDD: speci&#65533;?c functional annotation the <br />     Conserved Domain Database. Nucleic Acids Res. 37, D205&#8211;D210. <br /> Marra, M.A., Baillie, D.L., 1994. Recovery duplications drug resistance selection Caenorhabditis elegans. Genome 37, 701&#8211;705. <br /> Mechulam, Y., Schmitt, E., Maveyraud, L., Zelwer, C., Nureki, O., Yokoyama, S., Konno, M., Blanquet, S., 1999. Crystal structure of <br />     <span id='am-126' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-131'  ></span><span id='am-132' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli methionyl-tRNA synthetase highlights species-speci&#65533;?c features. J. Mol. Biol. 294, 1287&#8211;1297. <br /> Moerman, D.G., 1980. A genetic analysis unc-22 region Caenorhabditis elegans(PhD Thesis) Simon Fraser University. <br /> Moerman, D.G., Baillie, D.L., 1981. Formaldehyde mutagenesis nematode Caenorhabditis elegans. Mutat. Res. 80, 273&#8211;279. <br /> Nakanishi, K., Ogiso, Y., Nakama, T., Fukai, S., Nureki, O., 2005. Structural basis anticodon recognition methionyl-tRNA synthetase. <br />     Nat. Struct. Mol. Biol. 12, 931&#8211;932. <br /> Piano, F., Schetter, A.J., Morton, D.G., Gunsalus, K.C., Reinke, V., Kim, S.K., Kemphues, K.J., 2002. Gene clustering based RNAi phenotypes <br />     ovary-enriched genes C. elegans. Curr. Biol. 12, 1959&#8211;1964. <br /> Pierce, S.B., Chisholm, K.M., Lynch, E.D., Lee, M.K., Walsh, T., Opitz, J.M., Li, W., Klevit, R.E., King, M.C., 2011. Mutations mitochondrial <br />     histidyl tRNA synthetase HARS2 cause ovarian dysgenesis sensorineural hearing loss Perrault syndrome. Proc. Natl. Acad. <br />     Sci. U. S. A. 108, 6543&#8211;6548. <br /> Rogalski, T.M., Baillie, D.L., 1985. Genetic organization unc-22 IV gene adjacent region Caenorhabditis elegans. Mol. Gen. <br />     Genet. 201, 409&#8211;414. <br /> Rogalski, T.M., Moerman, D.G., Baillie, D.L., 1982. Essential genes de&#65533;?ciencies unc-22 IV region Caenorhabditis elegans. <br />     Genetics 102, 725&#8211;736. <br /> Rozen, S., Skaletsky, H.J., 2000. Primer3 WWW general users biologist programmers. In: Krawetz, S., M.S.e (Eds.), <br />     Methods Molecular Biology. Humana Press, Totowa, NJ, pp. 365&#8211;386. <br /> Schein, J.E., Marra, M.A., Benian, G.M., Fields, C., Baillie, D.L., 1993. The use de&#65533;?ciencies determine essential gene content the <br />     let-56-unc-22 region Caenorhabditis elegans. Genome 36, 1148&#8211;1156. <br /> Simmer, F., Moorman, C., van der Linden, A.M., Kuijk, E., van den Berghe, P.V. <br /> </body></html>