<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Basic Research <br />        Basic Research Paper <br />                       Paper <br />  <br />                                                                                                       Autophagy 10:4, 572&#8211;587; April 2014; &#169; 2014 Landes Bioscience <br />  <br />  <br />  <br />  Autophagy essential cardiac morphogenesis <br />         vertebrate development <br />          Eunmyong Lee,1 Yeon Koo,2 Aylwin Ng,3,4,5 Yongjie Wei,1,6,7 Kate Luby-Phelps,8 Amy Juraszek,9 Ramnik J Xavier,3,4,5 <br />                                Ondine Cleaver,2 Beth Levine,1,6,7,10,* James F Amatruda1,2,9,* <br />   Department Internal Medicine; University Texas Southwestern Medical Center; Dallas, TX USA; 2Department Molecular Biology; University Texas Southwestern <br />   1 <br />  <br />  <br />   Medical Center; Dallas, TX USA; 3Center Computational &amp; Integrative Biology; Massachusetts General Hospital; Boston, MA USA; 4Gastrointestinal Unit; Massachusetts <br />   General Hospital; Boston, MA USA; 5Broad Institute Harvard MIT; Cambridge, MA USA; 6Center Autophagy Research; University Texas Southwestern Medical <br />  Center; Dallas, TX USA; 7Howard Hughes Medical Institute; University Texas Southwestern Medical Center; Dallas, TX USA; 8Department Cell Biology; UT Southwestern <br /> Medical Center; Dallas, TX USA; 9Department Pediatrics; University Texas Southwestern Medical Center; Dallas, TX USA; 10Department Microbiology; University Texas <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 &#169;2014 Landes Bioscience. Do distribute. <br />                                                                 Southwestern Medical Center; Dallas, TX USA <br />  <br />  <br />                                      Keywords: autophagy, zebrafish, heart development, tbx2, atg5, atg7, becn1 <br />       Abbreviations: AV, atrioventricular; AVC, atrioventricular canal; dpf, days post-fertilization; hpf, hours post-fertilization; <br />                                                  MO, morpholino oligonucleotide <br />  <br />  <br />  <br />               Genetic analyses indicate autophagy, evolutionarily conserved lysosomal degradation pathway, essen- <br />           tial eukaryotic differentiation development. However, little known autophagy contributes <br />           morphogenesis embryogenesis. To address question, examined role autophagy early <br />           development zebrafish, model organism studying vertebrate tissue organ morphogenesis. Using zebrafish <br />           transgenically express fluorescent autophagy reporter protein, GFP-LC3, autophagy active in <br />           multiple tissues, including heart, embryonic period. Inhibition autophagy morpholino knockdown <br />           essential autophagy genes (including atg5, atg7, becn1) resulted defects morphogenesis, increased num- <br />           bers dead cells, abnormal heart structure, reduced organismal survival. Further analyses cardiac development <br />           autophagy-deficient zebrafish revealed defects cardiac looping, abnormal chamber morphology, aberrant valve <br />           development, ectopic expression critical transcription factors including foxn4, tbx5, tbx2. Consistent with <br />           results, Atg5-deficient mice displayed abnormal Tbx2 expression defects valve development chamber <br />           septation. Thus, autophagy plays essential, conserved role cardiac morphogenesis vertebrate development. <br />  <br />  <br />  <br />                                Introduction                                             extension; protection <span id='am-35' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cancer</span>, neurodegenerative dis- <br />                                                                                         orders, infection, diseases. <br />    Autophagy evolutionarily conserved process                                Another key function autophagy role differen- <br /> cytoplasmic constituents including macromolecules (such pro-                         tiation development.4-7 From evolutionary perspective, <br /> teins, glycogens, lipids, nucleic acids) organelles (such                       autophagy arisen mechanism protect unicel- <br /> as mitochondria, peroxisomes, endoplasmic reticulum)                            lular organisms starvation forms environ- <br /> degraded lysosome.1 The autophagy pathway plays                               mental stress (such overcrowding extreme temperatures). <br /> essential role enabling eukaryotic organisms adapt nutri-                      However, stimulus degrade organelles led to <br /> ent deprivation forms environmental stress; gener-                      evolutionary advantages, including ability undergo <br /> ates building blocks necessary macromolecular synthesis,                        differentiation development. Both differentiation and <br /> energy production, cell survival settings. By serving                       development require cells undergo major morphological <br /> as cellular &#8220;disposal&#65533;? mechanism removes misfolded                            changes thus, require mechanisms degradation and <br /> unfolded proteins damaged organelles, autophagy path-                           recycling obsolete cellular components. Therefore, not <br /> way maintains organelle integrity protein quality con-                         coincidence certain differentiation developmental <br /> trol. The nutrient recycling organelle/protein quality control                      events, especially lower eukaryotic organisms, triggered <br /> functions autophagy likely contribute important                          influenced autophagy-inducing environmental stressors; <br /> roles physiology protection diseases.2,3 Indeed,                         autophagy mechanistically involved processes. <br /> autophagy genes important metazoan survival star-                         In support concept, autophagy genes essential for <br /> vation; protein, carbohydrate, lipid metabolism; life-span                          starvation-induced differentiation developmental <br />  <br />  <br />  *Correspondence to: James F Amatruda; Email: james.amatruda@utsouthwestern.edu; Beth Levine; Email: beth.levine@utsouthwestern.edu <br />  Submitted: 07/23/2013; Revised: 12/18/2013; Accepted: 12/23/2013 <br />  http://dx.doi.org/10.4161/auto.27649 <br />  <br />  <br /> 572	Autophagy	                                                                                                                                           Volume 10 Issue 4 <br />  processes, including sporulation yeast, dauer development         patterning tissue morphogenesis early vertebrate <br /> C. elegans, fruiting body formation <span id='am-30' about='obo:NCBITaxon_5782' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_142796 obo:NCBITaxon_33083 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Dictyostelium" datatype="xsd:string"></span>Dictyostelium</span> discoideum,        embryonic development. <br /> and developmental transitions protozoan parasites (reviewed             To address question, used morpholino oligonucle- <br /> in refs. 4&#8211;7).                                                          otides (MOs) knock expression 3 different autophagy <br />     The role autophagy development extends        genes, becn1, atg5, atg7 developing zebrafish (<span id='am-43' about='obo:NCBITaxon_7955' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7954 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio frankei" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="zebrafish" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="leopard danio" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="zebra danio" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="zebra fish" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio Hamilton, 1822" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Danio rerio frankei" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27702" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Danio frankei" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Danio rerio" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37966" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Brachidanio rerio" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Danio rerio (Hamilton, 1822)" datatype="xsd:string"></span>Danio rerio</span>) <br /> other (environmental stress-independent) processes occur           embryos. Becn1 (also known Beclin 1) functions autophagy <br /> in multicellular organisms. For example, abnormalities lar-          <span id='am-40' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> III phosphatidylinositol 3-kinase complex <br /> val development, synapse formation, eye development             involved autophagic vesicle nucleation, Atg5 Atg7 are <br /> observed autophagy gene mutant Drosophila.8-12 In C. elegans,        ubiquitin-like protein conjugation systems func- <br /> autophagy genes play role apoptotic corpse clearance            tion autophagosome membrane expansion completion.21 <br /> embryonic development,13 L1 larval stage,14 adult           The zebrafish presents unique advantages vertebrates for <br /> gonad,15 degrading germline P granules somatic cells,16       study developmental patterning tissue morphogenesis <br />  <br />  <br />  <br />  <br />                                                                                                                                                &#169;2014 Landes Bioscience. Do distribute. <br /> in selective elimination paternal mitochondria.17,18 In mice,    (reviewed refs. 22 23). Unlike mammalian embryo, <br /> autophagy genes play essential role preimplantation devel-        zebrafish does require functional cardiovascular system <br /> opment, survival neonatal starvation period,          embryogenesis oxygen diffusion surround- <br /> the differentiation specific cell lineages, including adi-   ing medium sufficient survival.24 Therefore, organs <br /> pocytes, erythrocytes, lymphocytes (T cells B-1a cells)         mutant zebrafish embryos, especially heart, more <br /> (reviewed ref. 7).                                                   readily observed mouse embryos. <br />     One surprising finding mice lacking different          We MO knockdown different autophagy genes <br /> essential autophagy genes, including Atg3, Atg5, Atg7, Atg9,        led developmental defects, including structural cardiac defects. <br /> Atg16L1, reported lack apparent anatomical abnormalities         Autophagy-deficient zebrafish displayed defective cardiac loop- <br /> (reviewed ref. 6). The obvious interpretation find-     ing, mispatterning genes normally expressed atrio- <br /> ing autophagy machinery required tissue          ventricular (AV) valve, upregulation genes encoding <br /> morphogenesis embryonic development. However,               heart transcription factors, including Foxn4 Tbx5, in <br /> finding puzzling predict autophagy, path-        microarray analyses cardiac-enriched tissue. Consistent with <br /> way eliminate pre-existing structures provide         findings, autophagy morphants exhibited increased, ecto- <br /> materials building new structures, ideal candidate        pic expression tbx2, downstream target foxn4 tbx5. <br /> involvement developmental processes require cellular        Similarly, cardiac atrioventricular canal (AVC) defects Tbx2 <br /> tissue remodeling. Thus, question arises          misexpression observed autophagy-deficient atg5 &#8722;/&#8722; <br /> absence apparent anatomical abnormalities auto-           mutant mice. These <span id='am-133' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-134'  ></span><span id='am-135' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> demonstrate unequivocal role for <br /> phagy gene knockout mouse strains truly reflects lack          autophagy genes tissue patterning morphogenesis during <br /> role autophagy morphogenesis factors,           vertebrate development. <br /> such gene redundancy, activation compensatory pathways, <br /> or difficulties detection phenotypic abnormalities                                         Results <br /> be confounding variables. Of note, zebrafish, Hu et&#160;al. <br /> reported knockdown atg5 leads abnormal morphogen-               Autophagy active early embryonic development in <br /> esis brain regionalization body plan.19                          zebrafish <br />     In contrast mice lacking Atg3, Atg5, Atg7, Atg9,                To study role autophagy early embryonic develop- <br /> Atg6l1, mice lacking 3 genes involved autophagy, Becn1,        ment zebrafish, temporal pattern autophagy activa- <br /> Ambra1, Rb1cc1/Fip200, die embryogenesis (reviewed           tion zebrafish development monitored. Unlike in <br /> in refs. 5 7). becn1&#8722;/&#8722; embryos die embryonic d 7.5,         mouse embryos, autophagy induction observed at <br /> with massive cell death failure close proamniotic cav-       1-cell stage stage,25 reported autophagy is <br /> ity. Mice lacking Ambra1 embryonic lethal d E10&#8211;14           induced 32 h post-fertilization (hpf) zebrafish of <br /> show defective neural tube development hyperproliferation           delayed expression proteins essential autophagy.26 To con- <br /> of neural tissues. (In accord result, Benato cowork-      firm temporal expression pattern autophagy transcripts, <br /> ers reported knockdown zebrafish Ambra1 orthologs           prepared RNA different stages embryonic develop- <br /> causes defects body morphogenesis brain development20).          ment used RT-PCR detect expression atg5, becn1, <br /> rb1cc1&#8722;/&#8722; mice embryonic lethal d E13.5&#8211;16.5, display        atg7, ulk1b (Fig.&#160;1A). All 4 transcripts detected 1-cell <br /> defects heart liver development. The discrepancy         stage-embryos, indicating maternally deposited, <br /> the embryonic lethal phenotypes becn1&#8722;/&#8722;, ambra1&#8722;/&#8722;,             sphere (3 hpf), germ ring (6 hpf) 5-somite <br /> rb1cc1&#8722;/&#8722; mice vs. embryonic viability atg3 &#8722;/&#8722;, atg5&#8722;/&#8722;,    (12 hpf) stages 24 48 hpf. atg5 becn1 transcripts <br /> atg7&#8722;/&#8722;, atg9&#8722;/&#8722;, atg16l1&#8722;/&#8722; mice generally attributed        appeared decrease early gastrulation (germ ring) stage, <br /> autophagy-independent functions group genes.           similar reported atg5,19 ambra1a, ambra1b.20 <br /> However, explanations possible, additional studies        He coworkers previously reported transcripts <br /> in tractable model systems needed better determine          including ulk1b expressed 23 hpf.26 The inconsis- <br /> whether autophagy pathway functions developmental                tency previous report different RT-PCR <br />  <br />  <br /> www.landesbioscience.com	Autophagy	573 <br />  conditions. He et&#160;al. amplified full-length RNA auto-                  similarly observed atg7 morphants, suggesting efficient <br /> phagy gene expression detection, here, amplified shorter                knockdown Atg7 protein expression atg7 MO injection. <br /> 2-exon fragments transcript. It possible amplifying            Embryos injected becn1 MOs did complete <br /> shorter segments transcripts results sensitive detection            block formation Atg12&#8211;Atg5 complex, expected <br /> than amplification full-length transcripts. Hu cowork-              result Becn1 required Atg12&#8211;Atg5 complex for- <br /> ers identified atg5 transcripts early developmental time               mation; however, did observe decrease total amount <br /> points, including 0.2, 5, 10, 18, 24 hpf.19                                conjugate. Compared control morphants, becn1 <br />     To confirm occurrence autophagy early embryonic                  morphants decreased Becn1 protein expression (Fig.&#160;1D, <br /> development zebrafish, 10-somite stage embryos moni-                   lower panel). Thus, atg5, atg7, becn1 MOs successfully <br /> tored autophagosome structures <span id='am-105' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-106'  ></span><span id='am-107' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>electron</span> microscopy.                     inhibit expression target autophagy protein zebrafish <br /> We detected early autophagosomes autolysosomes                     embryos. <br /> somite tissue (Fig.&#160;1B), indicating presence autophagy                      Next, evaluated autophagy MOs inhibit auto- <br />  <br />  <br />  <br />  <br />                                                                                                                                                               &#169;2014 Landes Bioscience. Do distribute. <br /> at stage. To evaluate autophagy occurs                 phagy vivo, using fluorescent autophagy reporter strain of <br /> vivo, used fluorescent autophagy reporter transgenic fish               fish, Tg(cmv:GFP-lc3).26 To efficiently quantify effects of <br /> line Tg(cmv:GFP-lc3).26 During autophagy induction, LC3-I                      autophagy MOs zebrafish autophagy, prepared primary <br /> (which diffusely distributed cytoplasm)                      cells blastula-stage Tg(cmv:GFP-lc3) embryos injected with <br /> conjugated phosphatidylethanolamine form LC3-II,                       control autophagy MOs counted number GFP-Lc3 <br /> which stably associates autophagosomal membrane.                      puncta cell. Primary cells derived blastula-stage auto- <br /> GFP-tagged LC3-II accumulates punctate structures (auto-                    phagy morphants displayed reduced numbers GFP-Lc3 <br /> phagosomes), serving useful vivo marker auto-                  puncta, indicating knockdown atg5, becn1, atg7 inhib- <br /> phagy.21 Using confocal microscopy, noted accumulation                  ited autophagosome accumulation vivo (Fig.&#160;1E F). As <br /> of GFP-Lc3 puncta zebrafish 15-somite stage indicat-                 evidence autophagy-directed MOs inhibit auto- <br /> ing similar organisms, autophagy active dur-                  phagy, performed immunoblots protein lysates pri- <br /> ing early development (Fig.&#160;1C). Some caution warranted                  mary cell cultures MO-injected embryos, using antibody <br /> interpreting result, owing possibility expression             specific Lc3 (Fig.&#160;1G H). Autophagy MOs efficiently <br /> of GFP-Lc3 CMV promoter lead enhanced                        blocked Lc3-II conversion, indicating inhibition autophagy <br /> puncta formation high expression levels. However, taken                 vivo. <br /> together data including electron microscopy,                    After demonstrating autophagy-specific MOs inhibit <br /> results likely indicate presence autophagy early zebraf-             autophagy vivo, assessed effects autophagy inhibi- <br /> ish embryos.                                                                   tion early zebrafish development. Autophagy morphants <br />     Autophagy gene knockdown results developmental                          displayed reproducible abnormal developmental phenotypes <br /> defects                                                                        including small heads (arrows) eyes (asterisks), twisted body <br />     To investigate role autophagy embryo morphogen-                  shapes, pericardial edema (arrowheads) 2 d post fertiliza- <br /> esis, used antisense morpholino oligonucleotides (MOs)                   tion (dpf) (Fig.&#160;2A B). In general, morphological defects <br /> inhibit translation zebrafish mRNAs encoding 3 key com-                     similar comparing atg5, becn1, atg7 knockdown, <br /> ponents autophagy machinery, atg5, atg7, becn1.                     differences penetrance, becn1 <br /> We confirmed protein knockdown immunoblot analysis                       morphants severely affected overall, body morphol- <br /> whole embryos 48 hpf (Fig.&#160;1D). Atg5 normally conjugated                 ogy defects prominent atg7 (as becn1) morphants. Most <br /> to Atg12 conjugation reaction requires action                strikingly, 62% atg5 morphants, 80% becn1 morphants, <br /> the E1-like enzyme Atg7. MO knockdown Atg5 led loss                   31% atg7 morphants exhibited cardiac defects as <br /> free Atg5 expected loss Atg12&#8211;Atg5 complex for-                  pericardial edema, defective blood flow heart, defec- <br /> mation (Fig.&#160;1D, panel). Loss Atg12&#8211;Atg5 complex                    tive heart looping, enlarged atria, linearized hearts (Fig.&#160;S1). <br />  <br />  Figure&#160;1 (See opposite page). Autophagy present early zebrafish development efficiently inhibited autophagy gene knockdown. <br />  (A) Autophagy transcripts expressed early development zebrafish. RT-PCR performed using RNA isolated 1-cell, sphere, germ <br />  ring, 5-somite, 24 hpf, 2 dpf stage wild-type embryos using gene-specific primers. nRT, no-RT negative control. (B) Early autophagosomes (left) and <br />  autolysosomes (right) detected 10-somite stage embryos electron microscopy. Scale bars, left: 200 nm, right: 500 nm. (C) GFP-Lc3 puncta were <br />  visualized confocal microscopy 15-somite stage embryos Tg(cmv::GFP-lc3) fish. Left, z-stack images somites. Right, GFP-Lc3 puncta somites <br />  merged bright-field image. Scale bars, left: 30 &#956;m, right: 30 &#956;m. (D) Expression autophagy proteins autophagy morpholino (MO) injection. <br />  Embryos injected 1-cell stage control autophagy-specific MOs, lysates prepared immunoblot analysis 48 hpf. Top panel, <br />  immunoblot anti-Atg5 antibody. Bottom panel, immunoblot anti-Becn1 antibody. Actin shown loading control. (E F) Autophagy <br />  gene knockdown impairs autophagy zebrafish. Primary cells prepared Tg(cmv:GFP-lc3) MO injection GFP-Lc3 puncta visu- <br />  alized confocal microscopy. (E) Representative images GFP-Lc3 puncta embryos injected autophagy gene-specific control MOs. <br />  (F) Quantification data shown (E). Representative results 3 separate experiments, expressed mean &#177; SEM 50 100 cells each <br />  group. ***P &lt; 0.001, **P &lt; 0.01, *P &lt; 0.05 autophagy morphants vs. control morphants; one-tailed t test. (G) Representative immunoblots detect <br />  Lc3 levels control- autophagy MO-injected fish. Lysates prepared primary cells immunoblot MO injection. Actin shown as <br />  loading control. Similar results observed 3 independent experiments. (H) Quantification ratio Lc3-II/Actin 3 independent experi- <br />  ments, including representative results shown (G). Bars represent mean &#177; SEM. **P &lt; 0.01, *P &lt; 0.05 autophagy morphants vs. control morphants; <br />  2-tailed t test. <br />  <br />  <br /> 574	Autophagy	                                                                                                                            Volume 10 Issue 4 <br />                                               &#169;2014 Landes Bioscience. Do distribute. <br />  <br />  <br />  <br />  <br /> Figure&#160;1. For figure legend, page 574. <br />  <br />  <br />  <br />  <br /> www.landesbioscience.com	Autophagy	575 <br />                                                                                                                                                           &#169;2014 Landes Bioscience. Do distribute. <br />  Figure&#160;2. Autophagy gene knockdown zebrafish results developmental defects increased numbers dead cells. (A B) Morphology of <br />  autophagy morphants 2 dpf. (A) Representative images autophagy morphants, displayed small heads (arrows), abnormal eyes (asterisks), <br />  twisted body shapes cardiac defects (arrowheads) 2 dpf. (B) Quantification phenotypes depicted (A). Results represent mean &#177; SEM of <br />  3 independent experiments. ***P &lt; 0.001, **P &lt; 0.01, *P &lt; 0.05 autophagy morphants vs. control morphants; one-tailed t test. (C D) Acridine <br />  orange (AO) staining control autophagy gene morphants 1 dpf. AO-positive cells counted area spanning end yolk <br />  extension end tail. (D) Quantification data shown (C). Results represent mean &#177; SEM 40 embryos group. ***P &lt; <br />  0.001 autophagy morphants vs. control morphants; one-tailed t test. <br />  <br />  <br /> In comparison, 95% fish normal control               mutant (tp53M214K ) fish28 (Fig.&#160;S3A S3B). More 80% <br /> MO injection.                                                                control morphants normal 2 dpf. Except slight <br />    We took complementary approaches ensure               decrease percentage atg7 knockdown embryos exhibit- <br /> results reflected on-target effects MOs. First, addition            ing small head size, autophagy morphants continued exhibit <br /> the translation-blocking MOs described above, injected               developmental defects. 100% atg5 morphants, than <br /> splice blocking-MOs designed inhibit splicing specific exon-           80% becn1 morphants slightly 30% atg7 <br /> intron junctions atg5, atg7, becn1 primary mRNA tran-              morphants showed cardiac defects. Overall, cardiac defects <br /> script. Similar abnormal phenotypes observed embryos                 autophagy MOs significantly affected loss of <br /> injected splice blocking-MOs translation-block-             Tp53 function, indicating cardiac defects observed in <br /> ing MO-injected autophagy morphants, incidence                    wild-type background zebrafish caused Tp53- <br /> abnormal phenotypes increased relationship injected                dependent off-target effects. The finding similar phenotypes <br /> MO concentration (Fig.&#160;S2A S2B). Splicing inhibition                  using 2 independent MOs wild-type tp53-deficient <br /> MO injection confirmed RT-PCR (Fig.&#160;S2C).                             backgrounds strongly suggests observed phenotypes <br />    MO injection induce abnormal developmental phenotypes                 autophagy gene knockdown. <br /> due off-target effects attributed Tp53-mediated cel-             Autophagy gene knockdown results increased numbers <br /> lular toxicity.27 To exclude possibility phenotypes             dead cells <br /> of autophagy morphants off-target effects,                    Autophagy genes required clearance dead <br /> we injected control autophagy MOs homozygous tp53                   cells embryonic development mice,29 chick,30 and <br />  <br />  <br /> 576	Autophagy	                                                                                                                       Volume 10 Issue 4 <br />  <span id='am-89' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span>nematodes</span>,13 mice null mutation autophagy          autophagy morphants, atria enlarged cardiac loop- <br /> gene Ambra1 increased cell proliferation cell death      ing defective, resulting atria ventricles appearing in <br /> the central nervous system.31 To determine autophagy         histological plane. Additionally, 3 autophagy mor- <br /> genes play similar role regulation cell death dead    phants exhibited accumulation blood cells atrium com- <br /> cell clearance zebrafish embryonic development,            pared ventricle, suggesting autophagy inhibition <br /> stained embryos 1 dpf acridine orange (AO), quanti-      cause defects cardiac valve development circulatory <br /> fied number dead cells unit area defined region      defects. <br /> of embryo. Compared control morphants, autophagy                Cardiac morphogenesis influenced <span id='am-110' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-111' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-114'  ></span>car</span>- <br /> morphants showed greater 2-fold increase numbers        diac-extrinsic developmental factors blood circulation <br /> dead cells (Fig.&#160;2C D). A similar magnitude increase       vascular development. Vascular defects cause second- <br /> numbers dead cells seen injection autophagy MOs       ary cardiac defects increasing cardiac afterload. To exclude <br /> into tp53 mutant (tp53M214K ) embryos (Fig.&#160;S3C S3D). These      vascular defects possible source cardiac phenotype of <br />  <br />  <br />  <br />  <br />                                                                                                                                            &#169;2014 Landes Bioscience. Do distribute. <br /> results indicate autophagy play important role        autophagy morphants, monitored vascular development of <br /> either preventing cell death and/or removal dead cells dur-   morphants using Tg(fli1:EGFP) line, expresses GFP in <br /> ing zebrafish development.                                           endothelial cells including vasculature, blood cells, endocar- <br />     Interestingly, followed control autophagy mor-        dial cells.34 Autophagy morphants did vasculature <br /> phants long-term survival, saw striking effects auto-      defects 3 dpf (Fig.&#160;S6), suggesting cardiac defects in <br /> phagy gene knockdown. More 60% control morphants             autophagy morphants indirect consequence vascu- <br /> survived 4 wk. In contrast, survival poor      lar abnormalities. <br /> autophagy morphants, approximately 40% atg5 mor-                 As autophagy morphants showed incorrect looping and <br /> phants 10% becn1 atg7 morphants surviving        potential cardiac valve defects histological analysis (Fig.&#160;3C), <br /> more 10 d (Fig.&#160;S4). As translation-blocking effects     sought confirm defects using situ <span id='am-138' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-141'  ></span><span id='am-142' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>- <br /> MOs generally wear 4&#8211;5 d injection,32 data    ization cardiac-specific AV valve-specific markers. To <br /> suggest autophagy 4&#8211;5 d development      assess cardiac looping, performed situ hybridization to <br /> essential long-term fish viability.                              detect gene expression myl7/cmlc2 (myosin, light polypeptide <br />     Autophagy occurs cardiac development                      7, regulatory) 48 hpf. Control morphants displayed normal <br />     The striking cardiac phenotypes observed autophagy mor-       looping pattern, resulting ventricle placed slightly <br /> phants prompted investigate role autophagy      rostral right atrium (Fig.&#160;4). In contrast, auto- <br /> during cardiac morphogenesis. To determine auto-             phagy morphant hearts showed defects including failure to <br /> phagy occurs normal cardiac development zebrafish,         develop distinct heart tube cases, failure looping <br /> embryos Tg(cmv:GFP-lc3) autophagy reporter strain           resulting co-linear orientation 2 chambers cases <br /> were monitored GFP-Lc3 puncta formation. In control mor-         (Fig.&#160;4A C). <br /> phant hearts, autophagosomes cardiomyocytes,               To analyze cardiac valve development autophagy <br /> endocardial cells, AVC, ventricular outflow tract,       morphants, performed situ hybridization detect expres- <br /> indicating autophagy occurs cardiac development          sion 3 genes, vcan, bmp4, notch1b, shown <br /> (Fig.&#160;3A B). At 3 dpf, atg5 atg7 morphants showed            important cardiac valve development (Fig.&#160;4B D). <br /> significantly reduced numbers autophagosomes           Atrioventricular valve development highly regulated process, <br /> the entire heart (Fig.&#160;3B). becn1 morphants showed trend      involving regulation multiple signaling networks car- <br /> toward reduced numbers autophagosomes heart           diomyocytes endocardial cells, resulting endothelial <br /> did reach statistical significance. Imaging embryos 2      cells undergoing epithelial-mesenchymal transition populate <br /> dpf showed presence autophagosomes heart,         endocardial cushions, rise valves.35-37 In <br /> which appeared decreased autophagy morphant             control MO-injected embryos, expression vcan, bmp4, and <br /> embryos (Fig.&#160;S5).                                                   notch1b confined region AV valve. In contrast, <br />     Inhibition autophagy induces cardiac defects                  genes misexpressed knockdown autophagy <br />     To better characterize cardiac defects autophagy mor-     genes. vcan ectopically expressed ventricle in <br /> phants, histological analysis performed control auto-    atg5 atg7 morphants, expression increased the <br /> phagy MO-injected embryos 3 dpf. During zebrafish heart           AV valve parts atrium ventricle becn1 mor- <br /> development, cardiac progenitor cells migrate lateral       phants. The cardiomyocyte marker bmp4 endocardial <br /> mesoderm form linear heart tube 24 hpf. The heart       marker notch1b ectopically expressed, strikingly the <br /> tube subsequently tilts slightly left undergoes looping   ventricle, 50% autophagy morphants. In auto- <br /> around 2 dpf, representing structural change linearized     phagy morphants, expression bmp4 notch1b mark- <br /> tube 3-dimensional chamber (reviewed ref.33). Serial sec-    edly decreased absent (Fig.&#160;4B D). Thus, autophagy gene <br /> tions hearts autophagy morphants showed enlarged atria       knockdown leads defective cardiac looping, defective valve <br /> compared control morphants, incorrect loop-        formation ectopic expression valve markers, suggesting <br /> ing 3 dpf (Fig.&#160;3C). In control morphants, atria prop-       inhibition autophagy disrupts gene regulatory networks <br /> erly placed caudally left ventricles. However,     critical proper cardiac morphogenesis. <br />  <br />  <br /> www.landesbioscience.com	Autophagy	577 <br />                                                                                                                                                                &#169;2014 Landes Bioscience. Do distribute. <br /> Figure&#160;3. Autophagy gene knockdown results abnormal cardiac development. (A B) Autophagy active normal cardiac development <br /> and impaired autophagy MOs. (A) Tg(cmv:GFP-lc3) embryos injected control autophagy MOs imaged 3 dpf using confocal <br /> microscopy. Representative images showing GFP-Lc3 puncta (white arrows) atrium (A), ventricle (V) AV canal (AVC). The areas delineated the <br /> white dashed lines shown higher magnification panel. (B) Quantification data shown (A). Results represent mean &#177; SEM of <br /> more 3 embryos group. *P &lt; 0.05 vs. control morphants; one-tailed t test. (C) Hematoxylin eosin staining serial sagittal sections of <br /> hearts representative control autophagy morphants 3 dpf. The head positioned right. In control morphant heart, atrium, AV <br /> canal (arrow), ventricle, ventricular outflow tract (arrow head) (from left right) visualized. All autophagy morphant hearts display pericardial <br /> edema. The atg5 becn1 morphant hearts shown enlarged atria compared control morphant. The becn1 morphant heart <br /> displays accumulation blood cells atrium. All autophagy morphant hearts shown contain atrium ventricle plane indicat- <br /> ing incorrect cardiac looping. VOT, ventricular outflow tract. <br />  <br />  <br />  <br />  <br /> 578	Autophagy	                                                                                                                            Volume 10 Issue 4 <br />                                                                                                                                                          &#169;2014 Landes Bioscience. Do distribute. <br />  <br />  <br />  <br /> Figure&#160;4. Autophagy gene knockdown results abnormal cardiac looping valve development. (A C) In situ hybridization myl7 probe <br /> of morphant hearts 48 hpf. (A) Representative images control morphant showing correct looping atrium placed left caudal, and <br /> the ventricle right rostral, autophagy morphants, showing linearized hearts looping. (C) Quantification data shown (A). More <br /> than 30 embryos analyzed group. (B D) In situ hybridization morphants probes detect cardiac valve markers 2 dpf. <br /> (B) Representative images vcan, bmp4, notch1b expression hearts control autophagy mutants. (D) Quantification data shown in <br /> (B). More 30 embryos group analyzed. R, right; L, left. <br />  <br />  <br />  <br /> www.landesbioscience.com	Autophagy	579 <br />      Autophagy knockdown alters gene expression pattern                      Atg5 knockout mice exhibit cardiac defects <br /> the developing heart                                                               Our data indicate autophagy knockdown induces <br />     To examine effects autophagy inhibition car-             developmental cardiac defects zebrafish. We asked <span id='am-71' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-72' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span>this</span> <br /> diac gene expression, purified RNA hearts control               role autophagy conserved mammals, especially light of <br /> and autophagy morphants 3 dpf profiled gene expression                  striking conservation autophagy pathway func- <br /> by microarray analyses. We identified 109 genes (Fig.&#160;5A)                 tions diverse <span id='am-81' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. To investigate autophagy <br /> were differentially expressed (P &lt; 0.05 log2fold-change &gt; 2&#215;)              plays role mammalian cardiac development, imaged <br /> between control MO- autophagy MO-injected hearts,                      hearts E9.5 embryos GFP-LC3 transgenic mice39 and <br /> also informative predictive altered autophagy state,                 observed GFP puncta, indicating presence autophagy dur- <br /> as evaluated achieving area receiver-operating                 ing cardiac development (Fig.&#160;S8). Next, examined postna- <br /> characteristic curve (AUROC) &gt; 0.75, indicates favor-                 tal d 0 (P0) Atg5-deficient mice. atg5 &#8722;/&#8722; mice previously <br /> able true-positive rates false-positives. Transcripts           shown die hours birth, defec- <br />  <br />  <br />  <br />  <br />                                                                                                                                                              &#169;2014 Landes Bioscience. Do distribute. <br /> differentially regulated autophagy morphants included large               tive nutrient energy homeostasis.40 atg5&#8722;/&#8722; mice derived from <br /> number genes important cardiac development                 crosses Atg5+/&#8722; heterozygotes significantly smaller than <br /> function, including slc12a9, nup155, lrpprc, snrpb, crsp7/MED26,               Atg5 +/+ Atg5 +/&#8722; mice P0 (Fig.&#160;6A B). atg5&#8722;/&#8722; pups fre- <br /> and spred1. The differentially expressed genes enriched               quently small heads morphologically abnormal eyes, <br /> fundamental biological processes including regionalization, cell               reminiscent defects seen autophagy morphant zebraf- <br /> cycle, hematopoiesis, tRNA metabolism, noncoding RNA                       ish. Upon macroscopic analysis P0 hearts, noted enlarged <br /> metabolic processes. Most strikingly, observed dif-                   right atria (relative size total heart mass) Atg5&#8722;/&#8722; <br /> ferentially expressed genes morphants overrepresentation                 mice examined (Fig.&#160;6A, middle). Upon histological analysis, <br /> (hypergeometric P = 0.015) transcriptional factors roles               4 9 atg5 &#8722;/&#8722; mice AV canal defects <br /> in development, including foxn4, dbx1b, eed, eng1a, hoxb2a,                    including abnormal valve morphology membranous ven- <br /> hoxc10a, hoxc1a, msxc, mynn, vox, ved, npas4 (Fig.&#160;5B). The                tricular septal defect (VSD) P0 (Fig.&#160;6A C). Six the <br /> magnitude effects greatest atg7 morphants,                9 atg5&#8722;/&#8722; mice exhibited abnormal, thickened valves (Fig.&#160;6A). <br /> similar changes observed atg5 becn1 morphants.                Atg5 +/+ mice Atg5+/&#8722; mice did cardiac devel- <br /> We confirmed differential expression dbx1b, eed, hoxb2a,                opmental abnormalities (n = 11 group), 1 out <br /> her15.1, mynn, msxc, znfl2a quantitative RT-PCR RNA                  11 Atg5 +/+ Atg5+/&#8722; groups exhibited small <br /> from purified hearts (Fig.&#160;S7). These data demonstrate                    muscular VSD. <br /> developmental cardiac gene transcription programs aberrant                     To determine impairment autophagy murine <br /> in autophagy morphant zebrafish.                                               heart leads similar gene expression abnormalities we <br />     We performed additional studies validate abnormal                   observed zebrafish autophagy morphants, isolated E9.5 <br /> expression key gene important cardiac development,                    embryos crosses Atg5+/&#8722; heterozygotes performed in <br /> foxn4, previously shown critical proper AVC formation                situ hybridization Tbx2 (Fig.&#160;6D). At stage, Tbx2 expres- <br /> in zebrafish.38 To validate differential expression foxn4            sion normally restricted myocardium AVC, the <br /> autophagy morphant fish, performed quantitative RT-PCR                   inner curvature, outflow tract.41,42 Consistent the <br /> purified heart mRNA (Fig.&#160;5C). In agreement microar-                  results zebrafish, atg5&#8722;/&#8722; mice displayed abnormal Tbx2 <br /> ray results, foxn4 expression upregulated 3 autophagy               expression; mutant exhibited stronger expression, three <br /> morphants (Fig.&#160;5C). tbx5 normally cooperates foxn4                       displayed expanded, ectopic Tbx2 expression. In contrast, <br /> to regulate expression tbx2b formation AVC.                   4/4 Atg5 +/&#8722; 4/5 Atg5 +/+ littermates displayed normal Tbx2 <br /> Misexpression tbx2b causes defective AV canal development                   expression. These results indicate requirement auto- <br /> and impaired cardiac function.38 In situ hybridization showed                  phagy developmental cardiac patterning conserved from <br /> that autophagy gene knockdown caused increased expression                      fish mammals. <br /> and mislocalization tbx5 developing heart (Fig.&#160;5D and <br /> E). Consistent result, autophagy morphants exhibited                                               Discussion <br /> expanded expression tbx2b atrium ventri- <br /> cle AVC (Fig.&#160;5D E). Taken together,                  Here demonstrated autophagy plays essential <br /> data indicate critical role autophagy regulating cardiac              role cardiac morphogenesis vertebrate development. <br /> development proper patterning AV canal.                             Autophagy induced period rapid early embryonic <br />  <br />  <br />  Figure&#160;5 (See opposite page). Autophagy gene knockdown results upregulation transcription factors involved development. (A) Heat <br />  map altered transcripts hearts autophagy morphants 3 dpf. 109 genes identified informative predictive various <br />  altered autophagy states. (B) Enrichment analysis biological processes overrepresented genes differentially expressed informative of <br />  altered autophagy state. Of set transcription factors involved development, including foxn4 (arrow). (C) Real-time PCR <br />  measurement foxn4 mRNA levels purified hearts autophagy morphants 3 dpf. Results represent mean &#177; SEM triplicate samples. *P &lt; <br />  0.05 vs. control morphants; one-tailed t test. (D E) In situ hybridization detect tbx5 tbx2b expression morphant hearts. (D) Representative <br />  images control autophagy morphants, autophagy morphants showing stronger expression tbx5 tbx2b heart <br />  (arrows). (E) Quantification data shown (D). More 18 embryos group analyzed. <br />  <br />  <br /> 580	Autophagy	                                                                                                                           Volume 10 Issue 4 <br />                                               &#169;2014 Landes Bioscience. Do distribute. <br />  <br />  <br />  <br /> Figure&#160;5. For figure legend, page 580. <br />  <br />  <br />  <br />  <br /> www.landesbioscience.com	Autophagy	581 <br />                                                                                                                                                               &#169;2014 Landes Bioscience. Do distribute. <br />  Figure&#160;6. atg5&#8722;/&#8722; mice exhibit defective cardiac development. (A) Representative images atg5&#8722;/&#8722; Atg5+/+ mice (at d P0). Top row, photomicro- <br />  graphs heads eyes; second row top, photomicrographs hearts; fourth rows, hematoxylin eosin sections hearts. Brackets <br />  row denote enlarged right atrium; arrow row denotes ventricular septal defect. Arrows fourth row denote AV valve thickening. <br />  (B) Body weights atg5&#8722;/&#8722;, Atg5+/&#8722; Atg5+/+ mice. Results represent mean &#177; SEM ***P &lt; 0.005 indicated comparison; one-tailed t test. (C) Summary <br />  prevalence membranous ventricular septal defect mice indicated genotype d P0. (D) In situ hybridization detect Tbx2 expression <br />  atg5&#8722;/&#8722; Atg5+/+ mice (at d E9.5). Top panels, photomicrographs Tbx2 expression embryo; lower panels, photomicrographs Tbx2 <br />  expression hearts. Brackets lower panel denote expanded Tbx2 expression atrioventricular cushion (AVC). Results shown representative of <br />  results 4 embryos genotype. <br />  <br />  <br />  <br /> growth organogenesis, knockdown autophagy genes                     vertebrate development. We mice defi- <br /> causes severe developmental defects. The autophagy pathway                     cient Atg5 exhibit defective development AV canal and <br /> is active developing heart, inhibition autophagy                 membranous VSDs. These findings consistent defects <br /> disrupts gene expression program responsible cardiac                   endocardial cushion development. Consistent role for <br /> morphogenesis, leading misregulation key cardiac-specific                autophagy early developmental morphogenesis heart, <br /> transcription factors ectopic expression cardiac pattern-               showed autophagy-deficient atg5&#8722;/&#8722; mice display aberrant <br /> ing genes. The result aberrant development cardiac                   expression cardiac patterning gene Tbx2. These observa- <br /> chambers atrioventricular canal, increased                  tions strongly resemble cardiac phenotype zebrafish defi- <br /> embryonic lethality. While previous studies indicated                cient autophagy, indicating essential role ATG genes <br /> specific autophagy factors required development                 developmental morphogenic patterning conserved through- <br /> Drosophila eye,12,43 zebrafish brain,19,20 mouse devel-            vertebrate evolution. <br /> opment (reviewed ref. 7), results provide evidence               In mouse embryos, autophagy activated the <br /> that autophagy contributes cardiac morphogenic program                  oocyte-to-embryo transition occurs fertilization in <br />  <br />  <br /> 582	Autophagy	                                                                                                                           Volume 10 Issue 4 <br />  preimplantation embryos.25 The viable phenotype mouse           atg7 morphants different efficiencies knock- <br /> autophagy gene mutants attributed maternally sup-           down, based immunoblot detection protein levels, <br /> plied autophagy factors.7 In zebrafish, yolk serves source     MOs appeared equally effective decreasing autophagy <br /> of nutrients early development. Using biochemical            gene expression. Alternatively, half-life mater- <br /> conversion Lc3-I Lc3-II marker active autophagy, He       nally expressed autophagy genes differ genes <br /> and coworkers reported autophagy detectable beginning          studied. It possible individual autophagy genes are <br /> at 32 hpf.26 We detected autophagy somite-stage (approxi-            expressed different levels and/or required different degrees <br /> mately 15 hpf) embryos, evidenced presence GFP-Lc3         specific tissues organs. In mice, different autophagy mutants <br /> puncta, mark autophagosomes CMV:GFP-Lc3                    exhibit range developmental defects, early embryonic <br /> transgenic reporter line. The earlier appearance autophagy        lethality viability (reviewed ref. 7). Interestingly, despite <br /> our experiments increased sensitivity        overall similar phenotypes atg5, becn1, atg7 knockdown, <br /> confocal microscopy detect occurrence autophagy            autophagy MOs distinct effects long-term survival <br />  <br />  <br />  <br />  <br />                                                                                                                                                    &#169;2014 Landes Bioscience. Do distribute. <br /> individual cells, opposed biochemical assay used He        zebrafish. Even 5 d, time MOs generally no <br /> et&#160;al. In fact, detected autophagosome formation primary          longer provide effective knockdown,47 morphants continued <br /> cultures cells prepared shield-stage embryos (6 hpf).           impaired survival. Poor survival did directly correlate <br /> While possible autophagy setting cellular       severity embryonic phenotype, atg7 morphants <br /> response vitro culture absence yolk, results   exhibited rapid fall viability despite having some- <br /> indicated embryonic cells competent autophagy         milder apparent developmental phenotype. Taken together, <br /> an early time development. Taken detection         despite variation penetrance abnormal phenotypes in <br /> of autophagy gene transcripts, maternally supplied          different autophagy morphants, results indicate role for <br /> several stages 24 h development, dem-    autophagy embryonic morphogenesis physiological pro- <br /> onstration robust autophagy occurring intact, somite-stage        cesses essential long-term viability. <br /> embryos, results indicated autophagy early          In zebrafish, cardiac development begins 12 hpf, <br /> developmental program zebrafish embryos, period           specification bilateral stripes cardiac-progenitor <br /> of active tissue morphogenesis organogenesis.                       cells lateral mesoderm.48,49 The progenitors migrate medi- <br />     It somewhat surprising extensive autophagy          ally fuse form linear heart tube, undergoes <br /> occurs early fish development, given amniotes,       looping folding movements, resulting formation 2 <br /> yolk serves ready supply nutrients. In setting, auto-      distinct chambers. During process, approximately 37 <br /> phagy serve support high metabolic demands tis-           hpf 72 hpf, set endocardial cells atrioventricular <br /> sues undergoing large-scale morphogenesis cell movement,            canal (AVC) region undergo morphological changes delami- <br /> which fulfill energetic needs supported           nation which, course days, results in <br /> relatively slow diffusion nutrients yolk. Alternatively,    formation AV valve leaflets.50,51 Molecular markers this <br /> autophagy developing zebrafish tissues reflect intrinsic      change include restriction bmp4 vcan AVC myo- <br /> role pathway cellular differentiation, similar        <span id='am-100' about='obo:NCBITaxon_268263' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_105710 obo:NCBITaxon_6580 obo:NCBITaxon_55708 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6599 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Cardium" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cardium</span>, notch1b AVC endocardium.35 <br /> has described yeast sporulation, Dictyostelium fruiting             Having noted striking cardiac phenotype resulting from <br /> body formation, erythrocyte maturation, differentiation         autophagy knockdown, examined genes differ- <br /> of trypanosomes.7                                                       entially expressed knockdown embryos, enrich- <br />     Whether serves support metabolic demands tissue        ment number fundamental cellular pathways, including <br /> undergoing morphogenesis, cellular differentiation program,        cell cycle, metabolism, M-phase. Most significantly, saw <br /> both processes, autophagy necessary normal development           altered expression large number genes encoding transcrip- <br /> in zebrafish, evidenced pleiotropic effects autophagy      tional regulators. In group genes, foxn4 particu- <br /> gene knockdown. Knockdown atg5, becn1, atg7 led            lar regulates development AVC, cooperating <br /> similar defects embryos, including microcephaly, microph-        tbx5 regulate expression tbx2b.38 Rather nor- <br /> thalmia, abnormal body shape. Autophagy gene knockdown              mally restricted expression AVC, tbx5 tbx2b were <br /> in tp53-deficient strains produced nearly identical results,    ectopically expressed heart tube autophagy <br /> ruling off-target effects.                                          knockdown. Thus, proper functioning autophagy pathway <br />     Many autophagy genes essential functioning           required normal morphological gene expression <br /> pathway, shown phenotypes loss-of-function muta-           changes driving AVC formation. <br /> tions individual autophagy genes yeast, nematodes,                We identified autophagy multiple regions developing <br /> mammalian cells.7,44-46 Reflecting nonredundant functions        hearts. The sensitivity AVC development autophagy inhi- <br /> autophagy genes, knockdown atg5, becn1, atg7         bition reflect need autophagy contribute the <br /> were sufficient inhibit autophagy induce developmental           energy metabolism cells undergoing dramatic morphologi- <br /> defects. We did observe subtle differences knockdown             cal changes movements. Alternatively, autophagy be <br /> phenotypes different morphants; example atg7 mor-                required regulate signaling pathways necessary to <br /> phants exhibited lower percentage embryos defective           restrict expression valve development program specific <br /> cardiac looping. These differences atg5, becn1,           set cells. In absence autophagy, misexpression key <br />  <br />  <br /> www.landesbioscience.com	Autophagy	583 <br />  AVC transcription factors leads continued ectopic expression        shield stage sterilized 70% ethanol, dissoci- <br /> of downstream targets notch1b, bmp4a, vcan outside         ated trypsin/EDTA solution (GIBCO, 15400). Primary cells <br /> the AVC, results impaired cardiac development.                  plated Lab-tek chamber slide (Nunc, 177437) LDF <br />     To understand findings specific zebrafish     (Leibovitz 50%, DMEM 35%, F-12 15%) medium containing <br /> or represent generally conserved role autophagy car-     antibiotics incubated 25 &#176;C. <br /> diac development, carefully examined phenotype Atg5-             Zebrafish RNA preparation RT-PCR <br /> deficient mice. Atg5 knockout mice born normal ratios,              RNA 1-cell, sphere, germ ring, 5-somite, 24 hpf <br /> but die soon birth suckling defects impaired         2 dpf stages zebrafish embryos purified Trizol <br /> starvation response.40 The cardiac phenotype mice         (Invitrogen, 15596026) according manufacturer&#8217;s instruc- <br /> not studied detail. Compared wild-type Atg5-           tions. cDNA synthesized using RT2 HT strand kit <br /> heterozygous siblings, atg5&#8722;/&#8722; mice weighed smaller       (Qiagen, 330411). To detect different autophagy transcripts, <br /> heads smaller eyes, strongly reminiscent developmen-        PCR reactions performed. The primer sequences used are <br />  <br />  <br />  <br />  <br />                                                                                                                                            &#169;2014 Landes Bioscience. Do distribute. <br /> tal phenotypes autophagy-deficient zebrafish. Most signifi-         listed Supplementary Methods. <br /> cantly, P0 nearly half atg5&#8722;/&#8722; mice exhibited severe AVC         Morpholino injections <br /> developmental defects, including abnormal valve development                Translation-blocking splice-blocking MOs directed <br /> and membranous VSD. At earlier stages development, atg5&#8722;/&#8722;          atg5, becn1, atg7 MOs obtained Gene- <br /> mice abnormal expression transcription factor, Tbx2.       Tools. Control injections standard control MO <br /> Therefore, impaired autophagy leads defects transcrip-       Gene-Tools. Embryos injected 1-cell stage with <br /> tional patterning morphogenesis AVC, effect        0.3&#8211;1 mM MOs. The MO sequences used listed the <br /> is conserved vertebrates.                                       Supplementary Methods. <br />     There implications work. First,            Western blot analyses <br /> revealed unanticipated role autophagy cardiac <span id='am-115' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Morpho" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119'  ></span>morpho</span>-            Embryos 2 dpf stage anesthetized tricaine <br /> genesis development. Autophagy present developing         incubated 30 min ice lysis buffer containing <br /> tissues, knockdown autophagy genes causes profound              150 mM NaCl, 10mM Tris pH7.4, 0.2% Triton X-100, 0.3% <br /> defects development tissues eye, brain, heart,   NP-40, 0.2 mM Na3VO4 protease inhibitors (Roche, <br /> which undergo major morphological changes devel-           11697498001). After centrifugation 13,000 &#215; g, supernatant <br /> opment. Second, effect deficient autophagy         fractions used immunoblotting. Primary cells were <br /> on developmental patterning AVC conserved fish          prepared autophagy control MO injection used for <br /> to mammals. These results indicate autophagy genes         immunoblotting. Antibodies Atg5 (Novus, NB110-53818), <br /> be examined candidates unexplained cases            Becn1 (Beclin 1) (Santa Cruz Biotechnology, sc-11427), LC3 <br /> human congenital cardiac defects. Furthermore, autophagy            (Novus, NB100-2220) Actb/&#946;-actin antibody (Millipore, <br /> has previously invoked having adaptive mal-           MAB1501R) used immunoblotting. <br /> adaptive effects hearts hemodynamic stress,52-54       Acridine orange staining <br /> of study autophagic control developmental           1-dpf embryos washed PBS incubated PBS <br /> gene expression, described here, play role    containing 2 &#956;g/ml acridine orange (Fisher Biotech, BP116-10) <br /> response cardiac stress injury.                                 30 min.56 After staining, embryos washed 5 times with <br />                                                                        E3 buffer (5 mM NaCl, 0.17 mM KCl, 0.33 mM CaCl2, 0.33 <br />                   Materials Methods                                mM MgOS4). Mounted embryos 3% methylcellulose (Sigma, <br />                                                                        M0387) visualized using Leica MZ16 FA fluorescence ste- <br />    Zebrafish strains lines                                         reo dissecting microscope (Leica). <br />    Embryos wild-type AB strain used MO injec-                Microscopy imaging <br /> tion. Fish raised maintained standard condi-                Images whole-mount zebrafish embryos taken a <br /> tions.55 Autophagy reporter fish Tg(cmv:GFP-lc3) (gift Dr           Nikon Coolpix 4500 mounted Leica MZ125 stereo dissect- <br /> Daniel J Klionsky, University Michigan) previously             ing microscope. Images GFP-Lc3 primary cells zebraf- <br /> described.26 Tg(fli1:GFP) 34 gift Dr Nathan Lawson           ish embryos acquired Zeiss LSM 510 <span id='am-84' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-85' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88'  ></span>META</span> confocal <br /> (University Massachusetts) tp53M214K mutants28 gift      microscope. The embryos treated 0.003% 1-phenyl- <br /> of Dr Tom Look, Dana-Farber Cancer Institute.                          2-thiourea reduce pigment 24 hpf. For confocal imaging, <br />    Mouse strains                                                       embryos maintained 0.002% tricaine mounted in <br />    Autophagy reporter GFP-LC3 mice39 Atg5 knockout                 3% low-melt agarose. Confocal z-stack images used and <br /> mice40 previously described. atg5&#8722;/&#8722; mice gener-        analyzed IMARIS. <br /> ated obtaining offspring Atg5+/&#8722; crosses. All animal pro-            Transmission electron microscopy <br /> tocols approved UT Southwestern Medical Center                 Ten somite stage-embryos fixed 2.5% glutar- <br /> Institutional Animal Care Use Committee.                           aldehyde 0.1 M cacodylate buffer pH 7.4 3 h. Sections <br />    Zebrafish embryonic cell cultures                                   imaged using JEOL 1200 EX electron microscope 120 <br />    Approximately 50 autophagy MO- control MO-injected               KV equipped Sis Morada 11 megapixel mount CCD <br /> embryos used primary cell culture. Injected embryos        <span id='am-76' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-77' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>camera</span>. <br />  <br />  <br /> 584	Autophagy	                                                                                                         Volume 10 Issue 4 <br />     Survival assay                                                                           computed respective morphant compared with <br />    More 100 embryos injected autophagy con-                               control group. The expression SNR profiles transcripts <br /> trol MOs. The number surviving injected fish counted                                 identified differentially expressed (P &lt; 0.05 fold-change <br /> daily month.                                                                          &gt; 2&#215;) subjected feature (attribute) selection61,62 <br />    Histological analysis                                                                    using Information Gain (Kullback-Leibler divergence relative <br />    Injected zebrafish embryos fixed 3 dpf 4% parafor-                            entropy) identify transcripts informative the <br /> maldehyde (PFA) 48 h sagittal sections prepared.                               respective altered autophagy state. The accuracy class prediction <br /> Postnatal pups sacrificed heart dissected, fixed                           using Java implementation na&#239;ve Bayes classifier62,63 incor- <br /> in 4% PFA, dehydrated. Transverse sections stained                                 porating features evaluated using 10-fold cross-valida- <br /> with hematoxylin eosin tissue morphology.                                           tion. Enrichment analysis biological processes performed <br />    In situ hybridization                                                                    computing hypergeometric test previously described.64 <br />    Whole mount situ hybridizations performed using                                  Biological process annotations assignments obtained <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          &#169;2014 Landes Bioscience. Do distribute. <br /> standard protocols zebrafish embryos 2 dpf detect                                Gene Ontology (GO). Microarray data deposited <br /> myl7, vcan, bmp4, notch1b, tbx2b 3 dpf detect                                 www.ncbi.nlm.nih.gov/geo, accession number GSE51541 <br /> tbx2b tbx5. Primers used riboprobe synthesis listed                                Real-time PCR analyses <br /> in Supplementary Methods. T7 primer (TAATACGACT                                            Total RNA prepared heart used quantitative <br /> CACTATAGGG AGA) added reverse primers facilitate                                  RT-PCR. cDNA synthesized using RT2 HT strand kit <br /> antisense riboprobe synthesis. Riboprobe vcan kindly                            (Qiagen, 330411). Quantitative PCR reaction foxn4 per- <br /> provided Dr Didier Stainier (UCSF). Mouse embryos E9.5                                formed QuantiFast SYBR Green PCR kit (Qiagen, 204054) <br /> were used situ hybridization detect Tbx2 using previ-                           using Applied Biosystems 7500 Real-time thermocycler. All <br /> ously described Riboprobe.57                                                                signals analyzed normalized actb/&#946;-actin mRNA. <br />    Microarray analyses                                                                      Primers foxn4 obtained Qiagen primers for <br />    Zebrafish hearts purified control MO- auto-                                 actb described previously.65 <br /> phagy MO-injected embryos described.58 RNA each <br /> group injected embryos purified 3 dpf Trizol,                                                   Disclosure Potential Conflicts Interest <br /> digested DNase, purified using RNeasy columns                                  No potential conflicts disclosed. <br /> (Qiagen, 74104). RNA analyses performed microar- <br /> ray core facility Harvard Partners Healthcare Center                                                              Acknowledgments <br /> for Genetics Genomics. The quantity, purity, integrity                                 We thank Ashley Ratley zebrafish care. This work was <br /> of RNA evaluated UV spectrophotometry RNA-                                      supported grant 5R01CA135731 grant Amon <br /> nano Bioanalyzer (Agilent). Sample processing hybridization                             G Carter Foundation JFA; R01CA109618 CPRIT <br /> on Zebrafish Genome GeneChip microarrays (Affymetrix)                                  RP120718-P1 BL. We thank Daniel Klionsky, Noboru <br /> performed according manufacturer&#8217;s instructions. Probeset-                           Mizushima, Tom Look providing critical reagents and <br /> level normalization achieved using MAS5.0 algorithm.59                              Haley Harrington providing assistance manuscript <br /> Data analysis linear modeling performed R pro-                              preparation. <br /> gramming language, utilizing functions Linear Models for <br /> Microarray Data.60 For probe, moderated t-statistic (with                                                           Supplemental Materials <br /> standard errors moderated genes) computed using                                  Supplemental materials here: <br /> Bayesian model. For transcript, signal-to-noise ratios (SNR)                           www.landesbioscience.com/journals/autophagy/article/27649 <br />                                                             6.	 Mizushima N, Komatsu M. Autophagy: <span id='am-123' about='obo:NCBITaxon_61833' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_45443 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_28771 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Renova" datatype="xsd:string"></span><span id='am-127'  ></span>renova</span>-                  11.	 Shen W, Ganetzky B. Autophagy promotes synapse <br />                      References <br />                                                                  tion cells tissues. Cell 2011; 147:728-41;               development Drosophila. J Cell Biol 2009; 187:71- <br /> 1.	 Mizushima N. Autophagy protein organ-                 PMID:22078875;             http://dx.doi.org/10.1016/j.         9; PMID:19786572; http://dx.doi.org/10.1083/ <br />     elle turnover. Cold Spring Harb Symp Quant Biol              cell.2011.10.026                                                jcb.200907109 <br />     2011; 76:397-402; PMID:21813637; http://dx.doi.         7.	 Mizushima N, Levine B. Autophagy mamma-                  12.	 Chen SF, Kang ML, Chen YC, Tang HW, Huang <br />     org/10.1101/sqb.2011.76.011023                               lian development differentiation. Nat Cell Biol             CW, Li WH, Lin CP, Wang CY, Wang PY, Chen GC, <br /> 2.	 Levine B, Kroemer G. Autophagy pathogenesis           2010; 12:823-30; PMID:20811354; http://dx.doi.                  et&#160;al. Autophagy-related gene 7 downstream heat <br />     disease. Cell 2008; 132:27-42; PMID:18191218;             org/10.1038/ncb0910-823                                         shock protein 27 regulation eye morphology, <br />     http://dx.doi.org/10.1016/j.cell.2007.12.018            8.	 Juh&#225;sz G, Csik&#243;s G, Sinka R, Erd&#233;lyi M, Sass M. The              polyglutamine toxicity, lifespan Drosophila. J <br /> 3.	 Mizushima N, Levine B, Cuervo AM, Klionsky DJ.               Drosophila homolog Aut1 essential auto-               Biomed Sci 2012; 19:52; PMID:22621211; http:// <br />     Autophagy fights disease cellular self-diges-        phagy development. FEBS Lett 2003; 543:154-                 dx.doi.org/10.1186/1423-0127-19-52 <br />     tion. Nature 2008; 451:1069-75; PMID:18305538;               8; PMID:12753924; http://dx.doi.org/10.1016/               13.	 Huang S, Jia K, Wang Y, Zhou Z, Levine B. <br />     http://dx.doi.org/10.1038/nature06639                        S0014-5793(03)00431-9                                           Autophagy genes function apoptotic cell corpse <br /> 4.	 Levine B, Klionsky DJ. Development self-             9.	 Berry DL, Baehrecke EH. Growth arrest auto-                  clearance C. elegans embryonic development. <br />     digestion: molecular mechanisms bio-                     phagy required salivary gland cell degra-               Autophagy 2013; 9:138-49; PMID:23108454; <br />     logical functions autophagy. Dev Cell 2004;               dation Drosophila. Cell 2007; 131:1137-48;                   http://dx.doi.org/10.4161/auto.22352 <br />     6:463-77;       PMID:15068787;         http://dx.doi.        PMID:18083103;             http://dx.doi.org/10.1016/j.    14.	 Li W, Zou W, Yang Y, Chai Y, Chen B, Cheng S, <br />     org/10.1016/S1534-5807(04)00099-1                            cell.2007.10.048                                                Tian D, Wang X, Vale RD, Ou G. Autophagy genes <br /> 5.	 Cecconi F, Levine B. The role autophagy mam-      10.	 Denton D, Shravage B, Simin R, Mills K, Berry DL,               function sequentially promote apoptotic cell <br />     malian development: cell makeover cell           Baehrecke EH, Kumar S. Autophagy, apopto-                   corpse degradation engulfing cell. J Cell Biol <br />     death. Dev Cell 2008; 15:344-57; PMID:18804433;              sis, essential midgut cell death Drosophila.          2012; 197:27-35; PMID:22451698; http://dx.doi. <br />     http://dx.doi.org/10.1016/j.devcel.2008.08.012               Curr Biol 2009; 19:1741-6; PMID:19818615; http://               org/10.1083/jcb.201111053 <br />                                                                  dx.doi.org/10.1016/j.cub.2009.08.042 <br />  <br />  <br />  <br /> www.landesbioscience.com	Autophagy	585 <br />  15.	 Ruck A, Attonito J, Garces KT, N&#250;nez L, Palmisano       29.	 Qu X, Zou Z, Sun Q, Luby-Phelps K, Cheng P, Hogan        43.	 Zirin J, Perrimon N. Drosophila model sys- <br />      NJ, Rubel Z, Bai Z, Nguyen KC, Sun L, Grant BD,              RN, Gilpin C, Levine B. Autophagy gene-dependent              tem study autophagy. Semin Immunopathol <br />      et&#160;al. The Atg6/Vps30/Beclin 1 ortholog BEC-1                clearance apoptotic cells embryonic devel-          2010; 32:363-72; PMID:20798940; http://dx.doi. <br />      mediates endocytic retrograde transport addition          opment. Cell 2007; 128:931-46; PMID:17350577;                 org/10.1007/s00281-010-0223-y <br />      autophagy C. elegans. Autophagy 2011; 7:386-           http://dx.doi.org/10.1016/j.cell.2006.12.044             44.	 Mel&#233;ndez A, Tall&#243;czy Z, Seaman M, Eskelinen EL, <br />      400; PMID:21183797; http://dx.doi.org/10.4161/          30.	 Mell&#233;n MA, la <span id='am-11' about='obo:NCBITaxon_3764' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3745 obo:NCBITaxon_3398 obo:NCBITaxon_3744 obo:NCBITaxon_171638 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_1176516 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="rdfs:label" content="Rosa" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Rosa L." datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Rosa</span> EJ, Boya P. The autophagic              Hall DH, Levine B. Autophagy genes essential for <br />      auto.7.4.14391                                               machinery necessary removal cell corpses            dauer development life-span extension C. ele- <br /> 16.	 Zhang Y, Yan L, Zhou Z, Yang P, Tian E, Zhang K,             developing retinal neuroepithelium. Cell             gans. Science 2003; 301:1387-91; PMID:12958363; <br />      Zhao Y, Li Z, Song B, Han J, et&#160;al. SEPA-1 medi-             Death Differ 2008; 15:1279-90; PMID:18369370;                 http://dx.doi.org/10.1126/science.1087782 <br />      ates specific recognition degradation P           http://dx.doi.org/10.1038/cdd.2008.40                    45.	 Shintani T, Mizushima N, Ogawa Y, Matsuura A, <br />      granule components autophagy C. elegans. Cell     31.	 Fimia GM, Stoykova A, Romagnoli A, Giunta L, Di               Noda T, Ohsumi Y. Apg10p, novel protein-con- <br />      2009; 136:308-21; PMID:19167332; http://dx.doi.              Bartolomeo S, Nardacci R, Corazzari M, Fuoco C,               jugating enzyme essential autophagy yeast. <br />      org/10.1016/j.cell.2008.12.022                               Ucar A, Schwartz P, et&#160;al. Ambra1 regulates auto-             EMBO J 1999; 18:5234-41; PMID:10508157; <br /> 17.	 Al Rawi S, Louvet-Vall&#233;e S, Djeddi A, Sachse M,              phagy development nervous system.                  http://dx.doi.org/10.1093/emboj/18.19.5234 <br />      Culetto E, Hajjar C, Boyd L, Legouis R, Galy V.              Nature 2007; 447:1121-5; PMID:17589504                   46.	 Kametaka S, Okano T, Ohsumi M, Ohsumi Y. <br />      Postfertilization autophagy sperm organelles pre-    32.	 Bill BR, Petzold AM, Clark KJ, Schimmenti LA,                 Apg14p Apg6/Vps30p form protein complex <br />      vents paternal mitochondrial DNA transmission.               Ekker SC. A primer morpholino use zebrafish.           essential autophagy yeast, <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                         &#169;2014 Landes Bioscience. Do distribute. <br />      Science 2011; 334:1144-7; PMID:22033522; http://             Zebrafish 2009; 6:69-77; PMID:19374550; http://               cerevisiae. J Biol Chem 1998; 273:22284-91; <br />      dx.doi.org/10.1126/science.1211878                           dx.doi.org/10.1089/zeb.2008.0555                              PMID:9712845;              http://dx.doi.org/10.1074/ <br /> 18.	 Sato M, Sato K. Degradation paternal mito-           33.	Stainier DY. Zebrafish genetics verte-                     jbc.273.35.22284 <br />      chondria fertilization-triggered autophagy             brate heart formation. Nat Rev Genet 2001;               47.	 Nasevicius A, Ekker SC. The zebrafish novel sys- <br />      C. elegans embryos. Science 2011; 334:1141-4;                2:39-48;        PMID:11253067;          http://dx.doi.        tem functional genomics therapeutic devel- <br />      PMID:21998252;            http://dx.doi.org/10.1126/         org/10.1038/35047564                                          opment applications. Curr Opin Mol Ther 2001; <br />      science.1210333                                         34.	 Lawson ND, Weinstein BM. In vivo imaging                   3:224-8; PMID:11497344 <br /> 19.	 Hu Z, Zhang J, Zhang Q. Expression pattern               embryonic vascular development using trans-              48.	 Serbedzija GN, Chen JN, Fishman MC. Regulation <br />      functions autophagy-related gene atg5 zebraf-          genic zebrafish. Dev Biol 2002; 248:307-18;                   heart field zebrafish. Development 1998; <br />      ish organogenesis. Autophagy 2011; 7:1514-27;                PMID:12167406;             http://dx.doi.org/10.1006/         125:1095-101; PMID:9463356 <br />      PMID:22082871;             http://dx.doi.org/10.4161/        dbio.2002.0711                                           49.	 Lee RK, Stainier DY, Weinstein BM, Fishman <br />      auto.7.12.18040                                         35.	 Beis D, Bartman T, Jin SW, Scott IC, D&#8217;Amico LA,              MC. Cardiovascular development zebrafish. <br /> 20.	 Benato F, Skobo T, Gioacchini G, Moro I, Ciccosanti          Ober EA, Verkade H, Frantsve J, Field HA, Wehman              II. Endocardial progenitors sequestered within <br />      F, Piacentini M, Fimia GM, Carnevali O, <span id='am-1' about='obo:NCBITaxon_1105815' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_218718'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Dalla" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Dalla</span> Valle          A, et&#160;al. Genetic cellular analyses zebraf-            heart field. Development 1994; 120:3361-6; <br />      L. Ambra1 knockdown zebrafish leads incom-             ish atrioventricular cushion valve development.           PMID:7821208 <br />      plete development severe defects organogen-        Development 2005; 132:4193-204; PMID:16107477;           50.	 Stainier DY, Beis D, Jungblut B, Bartman T. <br />      esis. Autophagy 2013; 9:476-95; PMID:23348054;               http://dx.doi.org/10.1242/dev.01970                           Endocardial cushion formation zebrafish. Cold <br />      http://dx.doi.org/10.4161/auto.23278                    36.	 Chang CP, Neilson JR, Bayle JH, Gestwicki JE, Kuo             Spring Harb Symp Quant Biol 2002; 67:49- <br /> 21.	 Mizushima N, Yoshimori T, Levine B. Methods               A, Stankunas K, Graef <span id='am-6' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10'  ></span>IA</span>, Crabtree GR. A field             56;        http://dx.doi.org/10.1101/sqb.2002.67.49; <br />      mammalian autophagy research. Cell 2010; 140:313-            myocardial-endocardial NFAT signaling underlies               PMID:12858523 <br />      26; PMID:20144757; http://dx.doi.org/10.1016/j.              heart valve morphogenesis. Cell 2004; 118:649-           51.	 Scherz PJ, Huisken J, Sahai-Hernandez P, Stainier <br />      cell.2010.01.028                                             63; PMID:15339668; http://dx.doi.org/10.1016/j.               DY. High-speed imaging developing heart valves <br /> 22.	 Dodd A, Curtis PM, Williams LC, Love DR.                     cell.2004.08.010                                              reveals interplay morphogenesis function. <br />      Zebrafish: bridging gap develop-            37.	 Hurlstone AF, Haramis AP, Wienholds E, Begthel                Development 2008; 135:1179-87; PMID:18272595; <br />      ment disease. Hum Mol Genet 2000; 9:2443-                H, Korving J, Van Eeden F, Cuppen E, Zivkovic                 http://dx.doi.org/10.1242/dev.010694 <br />      9; PMID:11005800; http://dx.doi.org/10.1093/                 D, Plasterk RH, Clevers H. The Wnt/<span id='am-17' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span>beta</span>-catenin          52.	 Nakai A, Yamaguchi O, Takeda T, Higuchi Y, <br />      hmg/9.16.2443                                                pathway regulates cardiac valve formation. Nature             Hikoso S, Taniike M, Omiya S, Mizote I, Matsumura <br /> 23.	 Liu J, Stainier DY. Zebrafish study early          2003; 425:633-7; PMID:14534590; http://dx.doi.                Y, Asahi M, et&#160;al. The role autophagy cardio- <br />      cardiac development. Circ Res 2012; 110:870-                 org/10.1038/nature02028                                       myocytes basal state response hemo- <br />      4; PMID:22427324; http://dx.doi.org/10.1161/            38.	 Chi NC, Shaw RM, De Val S, Kang G, Jan LY,                    dynamic stress. Nat Med 2007; 13:619-24; http:// <br />      CIRCRESAHA.111.246504                                        Black BL, Stainier DY. Foxn4 directly regulates               dx.doi.org/10.1038/nm1574; PMID:17450150 <br /> 24.	 Pelster B, Burggren WW. Disruption hemoglobin             tbx2b expression atrioventricular canal <span id='am-120' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-121' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>-       53.	 Zhu H, Tannous P, Johnstone JL, Kong Y, Shelton <br />      oxygen transport does impact oxygen-dependent            tion. Genes Dev 2008; 22:734-9; PMID:18347092;                JM, Richardson JA, Le V, Levine B, Rothermel BA, <br />      physiological processes developing embryos             http://dx.doi.org/10.1101/gad.1629408                         Hill JA. Cardiac autophagy maladaptive response <br />      zebra fish (<span id='am-62' about='obo:NCBITaxon_7954' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Celestichthys" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:46777" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:487617" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Danio" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Brachydanio" datatype="xsd:string"></span>Danio</span> rerio). Circ Res 1996; 79:358-        39.	 Mizushima N, Yamamoto A, Matsui M, Yoshimori T,               hemodynamic stress. J Clin Invest 2007; 117:1782- <br />      62; PMID:8756015; http://dx.doi.org/10.1161/01.              Ohsumi Y. In vivo analysis autophagy response           93; PMID:17607355; http://dx.doi.org/10.1172/ <br />      RES.79.2.358                                                 nutrient starvation using transgenic mice express-         JCI27523 <br /> 25.	 Tsukamoto S, <span id='am-128' about='obo:NCBITaxon_996740' typeof='obo:NCBITaxon_33511 owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_33208 obo:NCBITaxon_1312402 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_147099 obo:NCBITaxon_131567 obo:NCBITaxon_31265 obo:NCBITaxon_309553 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-129' property="rdfs:label" content="Kuma" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-132'  ></span>Kuma</span> A, Murakami M, Kishi C,                    ing fluorescent autophagosome marker. Mol Biol         54.	 Wang ZV, Ferdous A, Hill JA. Cardiomyocyte auto- <br />      Yamamoto A, Mizushima N. Autophagy essential              Cell 2004; 15:1101-11; PMID:14699058; http://                 phagy: metabolic profit loss. Heart Fail Rev <br />      preimplantation development mouse embryos.            dx.doi.org/10.1091/mbc.E03-09-0704                            2013; 18:585-94; PMID:23054219; http://dx.doi. <br />      Science 2008; 321:117-20; PMID:18599786; http://        40.	 Kuma A, Hatano M, Matsui M, Yamamoto A,                       org/10.1007/s10741-012-9350-y <br />      dx.doi.org/10.1126/scienc <br /> </body></html>