<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           55.2, Phage T4 ORFan Gene, Encodes an <br />                                                           Inhibitor <span id='am-61' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> Topoisomerase I <br />                                                           Increases Phage Fitness <br />                                                           Yves Mattenberger*, Filo Silva, Dominique Belin <br />                                                           Department Pathology Immunology, University Geneva, Geneva, Switzerland <br />  <br />                                                           Academic Editor Eric Cascales <br />                                                           Centre National la Recherche Scientifique, <span id='am-48' about='obo:NCBITaxon_8831' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8830 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8826 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-49' property="rdfs:label" content="Aix" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Aix</span>-Marseille Universit&#233;, FRANCE <br />  <br />                                                           * yves.mattenberger@unige.ch <br />  <br />  <br />  <br />  <br />                                                           Abstract <br />                                                           Topoisomerases enzymes alter topological properties DNA. Phage T4 en- <br />                                                           codes topoisomerase utilize host-encoded topoisomerases. Here we <br />                                                           characterized 55.2, phage T4 predicted ORF unknown function. High levels expres- <br />     OPEN ACCESS <br />                                                           sion cloned 55.2 gene toxic E. coli. <span id='am-43' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-44' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> toxicity suppressed in- <br />                                                           creased topoisomerase I expression partial inactivation ATPase subunit the <br /> Citation: Mattenberger Y, Silva F, Belin D (2015) <br /> 55.2, Phage T4 ORFan Gene, Encodes Inhibitor         DNA gyrase. Interestingly, low-level expression 55.2, non-lethal wild type <br /> of <span id='am-77' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli Topoisomerase I Increases         E. coli, prevents growth deletion mutant topoisomerase I (topA) gene. In vitro, <br /> Phage Fitness. PLoS ONE 10(4): e0124309.                  gp55.2 binds DNA blocks specifically relaxation negatively supercoiled DNA by <br /> doi:10.1371/journal.pone.0124309 <br />                                                           topoisomerase I. In vivo, expression gp55.2 low non-toxic levels alters steady state <br /> Academic Editor: Eric Cascales, Centre National        DNA supercoiling reporter plasmid. Although 55.2 essential gene, competition <br /> la Recherche Scientifique, Aix-Marseille Universit&#233;, <br />                                                           experiments indicate required optimal phage growth. We propose role of <br /> FRANCE <br />                                                           gp55.2 subtly modulate host topoisomerase I activity infection insure optimal <br /> Received: October 3, 2014 <br />                                                           T4 phage yield. <br /> Accepted: March 11, 2015 <br />  <br /> Published: April 14, 2015 <br />  <br /> Copyright: &#169; 2015 Mattenberger et al. This an <br /> open access article distributed terms the <br /> Creative Commons Attribution License, permits <br /> unrestricted use, distribution, reproduction any <br /> medium, provided original author source       Introduction <br /> credited.                                                 The various DNA manipulations occur normal life cycle cell result in <br /> <span id='am-144' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147'  ></span><span id='am-148' property="rdfs:label" content="Data" datatype="xsd:string"></span>Data</span> Availability Statement: All relevant data        topologically altered forms DNA: knots, catenanes, supercoils [1]. Supercoils de- <br /> within paper Supporting Information files.    scribe DNA state number times strands cross differs that <br /> Funding: Fondation Schmidheiny DB Geneva <br />                                                           unconstrained DNA molecule. If number lower higher expected, DNA is <br /> Department Education The funders role        said negatively positively supercoiled. Processes require unwinding or <br /> study design, data collection analysis, decision   rotation topologically constrained DNA double <span id='am-33' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-34' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span>, replication transcrip- <br /> publish, preparation manuscript                 tion, lead accumulation compensatory supercoils hinder polymerase move- <br /> Competing Interests: The authors declared            ment [2,3]. Although torsional stress problem, supercoiling important <br /> that competing interests exist.                        physiological functions. Bacteria, normally maintain chromosome <span id='am-149' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-150' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> a <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                           1 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           negatively supercoiled state important factor chromosome compaction <br />                                           transcriptional regulation [4,5]. <br />                                               DNA&#8217;s diverse topological problems resolved topoisomerases, <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> enzymes <br />                                           allow DNA strands DNA double helices pass other. Type I topoisomer- <br />                                           ases transiently cut DNA strand, releasing superhelical tension, type II enzymes <br />                                           cut DNA strands allowing ATP-dependent transport DNA double helix <br />                                           [6,7]. Given universality DNA topological constraints, surpris- <br />                                           ing topoisomerases domains life. In addition, number <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>viruses</span>, <br />                                           eukaryotic bacterial, encode topoisomerases, [8,9]. Because critical <br />                                           role determining DNA topology, topoisomerases target choice de- <br />                                           velopment antibiotics anticancer drugs. Consequently, type II topoisomerase in- <br />                                           hibitors employed clinically potent antibacterial antitumor agents [10,11]. <br />                                               Escherichia coli type I topoisomerases (Topo I III), type II topoisomer- <br />                                           ases (DNA gyrase Topo IV). The negative supercoiling bacterial chromosome and <br />                                           plasmids homeostatically regulated counterbalancing activities Topo I Topo <br />                                           IV, remove negative supercoils, DNA gyrase, adds negative super- <br />                                           coils [12]. Topo III Topo IV primarily responsible unlinking sister chromo- <br />                                           somes replication [13]. Phage T4, infects E. coli, expresses own <br />                                           topoisomerase subunits encoded genes 39, 52, 60 [14,15]. This type II <br />                                           topoisomerase absolutely essential T4 growth, topoisomerase mutants display the <br />                                           so-called DNA-delay phenotype [16&#8211;18]. Indeed, phage topoisomerase absolutely re- <br />                                           quired early mode phage DNA replication. In topoisomerase mutant phages, DNA <br />                                           replication depends instead late, recombination-dependent, join-cut-copy replication <br />                                           pathway (reviewed [19&#8211;21]). In absence topoisomerase, T4 growth is <br />                                           completely dependent host DNA gyrase [22]. <br />                                               Through isolation large collections nonsense temperature-sensitive mutants, <br />                                           phage T4 organism nearly essential genes identified [16,23]. <br />                                           In addition 62 essential genes, genome known encode auxiliary proteins, <br />                                           increasing total number characterized genes 156. However, T4 genome pre- <br />                                           dicted 289 protein-coding genes (open reading frames ORF); nearly addi- <br />                                           tional 126 genes known function [24]. Since new unknown T4 genes <br />                                           detectable homology known proteins, considered databases orphans <br />                                           (ORFans). Nevertheless, present genomes related T4-like <br />                                           phages [25]. Since lytic phages abundant organisms planet [26,27], <br />                                           numerous diverse ORFans represent enormous reservoir unchartered genetic <br />                                           information, so-called &#8220;dark matter biosphere&#65533;? [26,28]. <br />                                               We previously identified T4 ORFans ectopic expression significantly inhib- <br />                                           E. coli growth [29]. As described above, DNA transactions require resolution of <br />                                           DNA topological issues. Thus, considered possibility T4 ORFans <br />                                           involved control phage DNA topology possibly conjunction the <br />                                           host topoisomerases. We report characterization gene 55.2, toxic T4 ORFan <br />                                           affects supercoiling regulation E. coli inactivation reduces phage T4 proge- <br />                                           ny production. <br />  <br />  <br />                                           Materials Methods <br />                                           E. coli phage strains, plasmids growth conditions <br />                                           The bacterial strains phage strains used study listed Table 1; plasmids are <br />                                           listed S1 File. The construction strains, plasmids genomic DNA library are <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                               2 / 23 <br />                                                                                       A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           Table 1. E. colia T4 strains. <br />  <br />                                           Strain          Genotype                                                                 Reference <br />                                           AS1047          MG1655 &#916; lacIZYA::frt pAST111                                            [31] <br />                                           AS1050          MG1655 &#916; lacIZYA::frt &#916; topA::apra pAST111                               [31] <br />                                           BE              Prototrophic                                                             [32] <br />                                           BL21(DE3)       F- ompT gal dcm lon hsdSB &#955;(DE3)                                         [32] <br />                                           BP199           W3110 gyrB221(CouR) gyrB203(ts)                                          [13] <br />                                           BW25113         F- &#916;(araD-araB)567 &#916;lacZ4787(::rrnB-3) rph-1 &#916;(rhaD-rhaB)568             CGSC <br />                                                           hsdR514 <br />                                           CL001           W3110 parE10(ts)                                                         [13] <br />                                           CR63            F+, supD, lamB63                                                         lab collection <br />                                           DHB3            MC1000 malF&#916; 3 phoA&#916; (PvuII) phoR                                        [33] <br />                                           DB503           MC4100 malE16-1 &#916; ara714                                                 lab collection <br />                                           DB870           MG1655 &#916; ara714, ydeA::kan                                               study <br />                                           DH5&#945;            F- &#916;(argF-lac)169, &#966;80dlacZ58(&#916;M15), &#916;phoA8, glnV44, deoR481,            CGSC <br />                                                           rfbC1, gyrA96(NalR), recA1, endA1, thiE1, hsdR17 <br />                                           JW0204-2        BW25113 &#916;rnhA733::kan                                                    [34] <br />                                           MC1000          F araD139 &#916; (araA-leu)7697 &#916; (lac)X74 rpsL150 galE15 galK16 relA1 <br />                                                            - <br />                                                                                                                                    [35] <br />                                                           thi <br />                                           MC4100          F- araD139 &#916; (argF-lac)U169 &#64258;hD5301 fruA25 relA1 rpsL150 rbsR22 &#916;        [36] <br />                                                           (&#65533;?mB-&#65533;?mE)632 deoC1 thi <br />                                           MG1655          F- rph-1                                                                 CGSC <br />                                           W3110           F- IN(rrnD-rrnE)1 rph-1                                                  CGSC <br />                                           YM63            BW25113 PlacZ-topA76(ts) zci-2234::cat                                   study <br />                                           YM64            BW25113 PlacZ-topA76(ts) zci-2234::cat &#916;topB761::kan                     study <br />                                           T4+             T4D                                                                      [16] <br />                                           T4 K10          38amB262 51amS29 nd28 (denA) rIIPT8 (&#916;denB-rII)                          [37] <br />                                           T4 K10          K10 55.2 (ATG-&gt; ACA)                                                     study <br />                                           55.2 <br />                                           T4 39           T4D 39amEA29                                                             R.H. Epstein&#8217;s <br />                                                                                                                                    collection <br />                                           T4 55.2         T4D 55.2 (ATG-&gt; ACA)                                                     study <br />                                           T4 39 55.2      T4D 39amEA29 55.2 (ATG-&gt; ACA)                                            study <br />                                           a <br />                                            All strains E. coli K-12 derivatives BE BL21(DE3), E. coli B strains. <br />  <br />                                           doi:10.1371/journal.pone.0124309.t001 <br />  <br />  <br />  <br />                                           described S1 File. Unless stated, E. coli strains cultivated 37&#176;C LB, M9 <br />                                           minimal (M9), M63 minimal (M63) medium [30] supplemented amino acids <span id='am-139' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-140' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-141'  ></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br />                                           bon source indicated. For growth solid medium, 1.5% bacteriological agar included. <br />                                           Antibiotics used following concentrations: ampicillin (Ap), 200 &#956;g ml-1; chloram- <br />                                           phenicol (Cm), 30 &#956;g ml-1; kanamycin (Kn), 40 &#956;g ml-1; spectinomycin (Sp), 50 &#956;g ml-1; tet- <br />                                           racycline (Tc), 7.5 &#956;g ml-1. <br />  <br />  <br />  <br />                                           Plasmid based lethality assay <br />                                           Plasmid based lethality assays performed previously described [31]. Briefly, bacteria <br />                                           cultivated overnight LB Kn Ap maintain 55.2 topA plas- <br />                                           mids. The morning, saturated cultures diluted 1/80 M63 medium supplemented <br />                                           0.2% glucose Kn Ap grown A600 nm = 0.4, spreading <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                          3 / 23 <br />                                                                                  A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           dilutions M63 agar plates supplemented 0.2% glucose, Kn, 600 &#956;M IPTG, 80 &#956;g <br />                                           ml-1 X-Gal. Colonies counted photographed 36 h 37&#176;C. <br />  <br />  <br />                                           Purification gp55.2-His6 <br />                                           The protein expressed pMCN1 BL21(DE3) cells. Exponentially growing bacterial <br />                                           cultures induced 0.2% arabinose 2 hours 37&#176;C. The protein purified from <br />                                           frozen cell pellets Ni-NTA agarose (#30210, Quiagen) according manufacturer instruc- <br />                                           tions. Eluted fractions containing His&#8211;tagged protein desalted PD-10 column <br />                                           (GE Healthcare) equilibrated TKDG buffer (50mM Tris pH 7.4, 100mM KCl, 1mM DTT, <br />                                           10% glycerol) concentrated centricon microconcentrator (3 kDa molecular weight <br />                                           cut-off, #4202, Amicon). Protein concentration determined Bradford assay (Bio-Rad <br />                                           Protein Assay) purity (&gt;99%) assessed Coomassie brilliant blue staining proteins <br />                                           separated SDS-polyacrylamide gel electrophoresis. <br />  <br />  <br />                                           Electrophoretic Mobility Shift Assay (EMSA) <br />                                           EMSA performed form I form I&#8217; DNA plasmid pDB29 prepared described <br />                                           below; linear DNA obtained restriction digestion EcoRV. Plasmid DNA (0.3 &#956;g, 79 <br />                                           fmol) incubated indicated amounts gp55.2-His6 EMSA Buffer (50 mM Tris, <br />                                           100 mM KCl, 1 mM &#946;-mercaptoethanol, 1 mg ml-1 Bovine serum albumin (BSA), 10% glycerol, <br />                                           pH 7.5 25&#176;C) 30 min 37&#176;C. Samples electrophoresed 3V cm-1 through <br />                                           0.8% agarose gels 1x TBE (90 mM Tris, 90 mM boric acid, 2.5 mM EDTA) 20 h 4&#176;C <br />                                           constant buffer recirculation. Gels stained ethidium bromide photographed <br />                                           UV light using E-BOX VX5 (Vilber Lourmat, France) Chemi Doc MP <br />                                           (Bio-Rad Laboratories) <br />  <br />  <br />                                           DNA relaxation supercoiling assays <br />                                           Covalently closed, negatively supercoiled circular DNA (form I) plasmid pDB29 [38] was <br />                                           obtained standard alkaline lysis procedure, followed cesium chloride density gradient <br />                                           centrifugation [39]. Covalently closed, relaxed circular DNA (form I&#8217;) prepared incubat- <br />                                           ing form I DNA wheat germ topoisomerase I (Promega, #M285) according manufac- <br />                                           turer instruction. All plasmid DNAs extracted phenol/CHCl3 ethanol <br />                                           precipitated used substrates supercoiling reactions. For relaxation assays, E. <br />                                           coli Topo I purchased New England Biolabs (#M0301) reactions performed <br />                                           1x CutSmart buffer 0.6 &#956;g form I pDB29 DNA (158 fmol). The indicated amounts of <br />                                           gp55.2-His6 incubated DNA 15 min 37&#176;C, indicated amounts of <br />                                           Topo I added samples incubated 37&#176;C 15 min. Reactions terminated <br />                                           described [40]. Briefly, EDTA added 25mM samples incubated 2 min at <br />                                           37&#176;C; then, SDS proteinase K added 1% 100 &#956;g/ml, respectively, sam- <br />                                           ples incubated additional 15 min 37&#176;C. DNA products extracted phe- <br />                                           nol/CHCl3 electrophoresed 2V cm-1 0.8% agarose gels 1x TBE buffer 48 h <br />                                           4&#176;C constant buffer recirculation. Gels stained photographed explained <br />                                           above. For supercoiling assays, E. coli DNA gyrase purchased New England Biolabs <br />                                           (#M0306) reactions performed 1x Gyrase buffer 0.6 &#956;g form I&#8217; pDB29 <br />                                           (158 fmol). The indicated amounts gp55.2-His6 incubated DNA 15 min <br />                                           37&#176;C, indicated amounts DNA gyrase added reactions further <br />                                           incubated 37&#176;C 30 min. Reactions terminated analyzed described the <br />                                           relaxation assays. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             4 / 23 <br />                                                                                  A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           In vivo analysis plasmid topoisomers <br />                                           To obtain monomeric molecules, plasmids linearized self-circularized large liga- <br />                                           tion volumes. The ligations transformed appropriate strains independent <br />                                           clones purified used subsequent analyses. Equal amounts (A600nm) exponential- <br />                                           ly growing bacterial cultures quickly chilled mixing equal volumes ice- <br />                                           cold growth medium transferred ice. DNA extracted standard alkaline lysis pro- <br />                                           cedure, followed phenol/CHCl3 extraction ethanol precipitation. The nucleic acid pel- <br />                                           lets dissolved TE buffer 2 &#956;g ml-1 RNAse A analyzed 0.8% agarose/ TBE <br />                                           gels 1 1.5 &#956;g ml-1 chloroquine (CLQ) described previous section. Loaded vol- <br />                                           umes adjusted account difference plasmid copy number 55.2-ex- <br />                                           pressing control strains. After electrophoresis, gels washed thrice 15 min with <br />                                           0.5x TBE, stained ethidium bromide (1 &#956;g ml-1) H2O 1 hour, destained 20 min with <br />                                           1mM MgSO4, photographed described above. The migration plasmid DNA <br />                                           samples gels different CLQ concentrations allowed determine bands <br />                                           correspond negatively supercoiled plasmids [41]. Densitometric analyses per- <br />                                           formed unsaturated images using MultiGauge (v.3) software (Fujifilm LifeScience). The <br />                                           data normalized total DNA lane. Two-dimensional gel (2D) gel <br />                                           analyses performed previously described [42] using 0.8% agarose/ TBE gels 1.5 &#956;g <br />                                           ml-1 25 &#956;g ml-1 CLQ second dimension, respectively. <br />  <br />                                           Plasmid copy number quantification <br />                                           Plasmid DNA extracted equal amounts bacteria (A600nm) standard alkaline lysis <br />                                           procedure resuspended TE buffer. Equal volumes linearized appropriate <br />                                           restriction enzyme, ran standard agarose TAE gels, visualized ethidium bromide. <br />                                           Unsaturated images analyzed using MultiGauge (v.3) software (Fujifilm LifeScience) <br />                                           signals calibrated using standard curve diluted High Mass DNA ladder (Invitro- <br />                                           gen) loaded gel. <br />  <br />                                           Intracellular phage growth phage competition assays <br />                                           Intracellular phage growth assays performed previously described [29] follow- <br />                                           ing modifications. CR63 bacteria grown M9 medium supplemented 50 &#956;g ml-1 <br />                                           tryptophan 0.4% glucose (M9) 1% casmino acids (M9S) concentration <br />                                           1 x 108 cfu ml-1 placed ice. Before infection, bacteria prewarmed 10 min and <br />                                           infection (t = 0) started mixing bacteria equal volume phages diluted the <br />                                           corresponding growth media. Phage growth carried indicated temperature vig- <br />                                           orous agitation. Competition assay performed CR63 cells grown M9S. For each <br />                                           growth cycle, aliquot bacteria prewarmed 10 min 37&#176;C, mixed equal vol- <br />                                           ume phage diluted M9S (1 x 106 pfu ml-1, multiplicity infection (moi) = 0.01), and <br />                                           growth carried 37&#176;C vigorous agitation. After 45 min, cultures lyzed with <br />                                           drop CHCl3 placed ice 10 min, transferred tube, kept 4&#176;C. Be- <br />                                           cause average burst size conditions 130, lysates diluted 65 times <br />                                           M9S directly used carry growth cycle. The titer lysate determined <br />                                           ensure moi growth cycle exceeded 0.1. <br />  <br />                                           Determination ratio 55.2+ total phage <br />                                           Aliquots phage lysates ( 1 x 105 pfu) used template amplification with <br />                                           primers 55.3up (5&#8217;-GGAAATTTAAATGAATCCTGAATC) 55.1dn (5&#8217;-AGACCTATCTTAAAGCATAGAG) <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             5 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           using Taq DNA polymerase (Invitrogen) following manufacturer&#8217;s instruction. Aliquots of <br />                                           PCR digested BsrGI, cuts mutated 55.2 initiation <span id='am-53' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> the <br />                                           amplified fragment. Restriction digests run 1% agarose/TAE gels, stained ethid- <br />                                           ium bromide, imaged UV light, intensity values DNA bands deter- <br />                                           mined unsaturated images using Multi Gauge 3.0 software (FujiFilm). The K10/ (K10- <br />                                           55.2 + K10) ratios calculated dividing intensity value undigested band the <br />                                           cumulated intensity values undigested band bands resulting BsrGI di- <br />                                           gestion. We confirmed accuracy PCR-restriction determination testing standard <br />                                           curve prepared known amounts K10 K10-55.2 phage. In addition, ratio se- <br />                                           lected growth cycle lysates confirmed determination genotype individual <br />                                           plaques (n = 20). <br />  <br />  <br />                                           Results <br />                                           Ectopic expression cloned phage T4 gene 55.2 inhibits irreversibly E. <br />                                           coli growth <br />                                           Using <span id='am-105' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> expressing T4 genes control inducible arabinose PBAD promoter, <br />                                           previously demonstrated ectopic expression 55.2 ORFan deleterious for <br />                                           E. coli growth [29]. Such plasmids confer arabinose sensitive phenotype (AraS) carri- <br />                                           er host strain. In experiments, performed medium copy number plasmid derived <br />                                           ColEI, residual expression 55.2 PBAD promoter absence arabinose <br />                                           induction sufficient affect bacterial growth. Initial growth dilution stationary <br />                                           phase cultures delayed compared bacteria harboring vector control <br />                                           (S1A Fig). Furthermore, colony forming units (cfu) saturated cultures 10-fold <br />                                           lower controls (S1B Fig). To minimize effects residual expression re- <br />                                           pressed PBAD promoter, 55.2 ORFan sequence cloned pBAD101, low-copy in- <br />                                           ducible vector pSC101 derived replicon [43]. This essentially abolished phenotypes <br />                                           observed original construct absence arabinose. However, arabinose-in- <br />                                           duced conditions, 55.2 expression prevented bacterial colony formation. Growth curve as- <br />                                           says showed addition arabinose led slow growth arrest complete only <br />                                           3 h (Fig 1A). However, reversibility assay demonstrated hour of <br />                                           55.2 induction, ~100 fold decrease capacity induced cells form colonies <br />                                           absence arabinose (Fig 1B). Thus 55.2 irreversible bactericidal action. <br />  <br />  <br />                                           Overexpression topA, gene encoding E. coli topoisomerase I, <br />                                           suppresses 55.2 toxicity <br />                                           We searched E. coli genes that, present multiple copies, suppressed AraS pheno- <br />                                           type induced 55.2 expression (multicopy suppressors). Initial attempts using standard E. coli <br />                                           genomic libraries identified non-specific suppressors interfere PBAD induction. <br />                                           These non-specific suppressors included genes araBAD operon, encodes enzymes <br />                                           degrade arabinose, ydeA, encodes transporter facilitates arabinose export <br />                                           [44]. Thus, prepared new library strain harbors araBAD ydeA dele- <br />                                           tions. The plasmid library introduced electroporation cells carrying pBAD101 <br />                                           plasmid 55.2 insert transformants directly selected LB plates containing <br />                                           0.2% arabinose. Plasmids suppressor strains purified suppressor activi- <br />                                           ty library plasmids verified. Non-specific suppressors identified eliminated as <br />                                           previously described [29]. After testing equivalent   180 E. coli genomes, isolated <br />                                           plasmid conferred specific, albeit partial, resistance 55.2 expression. The 4259 bp <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                              6 / 23 <br />                                                                                            A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />  <br /> Fig 1. The toxicity 55.2 E. coli suppressed increase copy number topA. (A) Liquid growth assay. Overnight cultures DB503 <br /> cells harboring pBAD101 (ctr) pDB2114-101 (55.2) plasmids diluted fresh LB medium optical density (A600nm) measured indicated <br /> times. At A600nm = 0.3, cultures split half induced 0.2% arabinose (vertical arrow). When A600nm reached   1, cultures were <br /> diluted 10-fold prewarmed medium plus minus arabinose. The graph depicts data representative experiment. (B) Reversibility assay. During a <br /> growth curve assay, aliquots arabinose-induced DB503 cultures, harboring pBAD101 (55.2 &#8211;) pDB2114-101 (55.2 +) plasmids, withdrawn the <br /> indicated times, washed cold media arabinose, adjusted A600nm. Serial 10-fold dilutions spotted LB plates without <br /> arabinose. (C) DB503 cells transformed pBAD101 (vector) pDB2114-101 (55.2) compatible plasmids, pDB868-2 (vector, 1&#8211;2), pDB34- <br /> 8-4 (topA, 3&#8211;4), pDB34-8 (topA, 5&#8211;6) streaked LB plates 0.2% arabinose. (D) Plasmid based lethality assay. Overnight cultures of <br /> AS1047 (topA +) AS1050 (topA-) transformed pBAD33-K (55.2 &#8211;) pDB2114-33-K (55.2 +) diluted outgrown indicated Materials <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                            7 / 23 <br />                                                                                              A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br /> and Methods section. Aliquots diluted, plated M63 plates supplemented glucose X-gal. The number blue white colonies was <br /> scored 36h 37&#176;C. Representative photographs shown upper panel (the position rare white colony inset IV indicated arrow). <br /> The lower panel depicts percentage white colonies; average standard deviation independent experiments. <br /> doi:10.1371/journal.pone.0124309.g001 <br />  <br />                                                 insert plasmid contained yciN, topA cysB. Subsequent subcloning revealed <br />                                                 region containing entire topA coding sequence 272 bp sequence upstream <br />                                                 initiation codon sufficed suppression (Fig 1C, streak 4). The product topA the <br />                                                 DNA topoisomerase I (Topo I), type <span id='am-9' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-10' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span>IA</span> topoisomerase removes excess negative super- <br />                                                 coils introduced DNA gyrase E. coli chromosome plasmids [45,46]. <br />  <br />                                                 Incompatibility low-level expression 55.2 reduced expression <br />                                                 Topo I Topo III <br />                                                 Since high-level expression topA partially suppressed 55.2-induced toxicity, examined <br />                                                 effect E. coli varying levels expression 55.2 topA. The viability topA <br />                                                 null mutants absence compensatory DNA gyrase mutations long debated <br />                                                 [47,48]. Recently, shown Rudolph coworkers loss topA results se- <br />                                                 vere growth defect [31]. In assay, copy topA gene cloned unstable mini-F <br />                                                 lac+ plasmid complemented chromosomal topA deletion. In lac&#8212;strain, loss plasmid <br />                                                 followed colony color X-gal plates. Using system, determined effect <br />                                                 low-level expression 55.2 topA loss. TopA+ &#916;topA bacteria harboring topA lac+ <br />                                                 mini-F plasmid transformed pBAD vector 55.2 insert. After out- <br />                                                 growth absence antibiotic selection topA mini-F plasmid, bacteria plated <br />                                                 minimal X-gal glucose plates selection pBAD plasmid (Fig 1D). In presence <br />                                                 topA+ chromosomal copy (I), colonies white indicating loss the <br />                                                 topA+ mini-F plasmid effect growth. In absence topA chromosomal copy <br />                                                 (III), 65% colonies retained topA+ mini-F plasmid; importantly, white colo- <br />                                                 nies notably smaller blue. These results confirm topA mutants severe <br />                                                 growth defect. When 55.2-coding plasmid present, white colonies observed <br />                                                 (5.5%) size significantly smaller white colonies obtained absence <br />                                                 55.2-coding plasmid (compare insets IV III). Thus, topA function apparently nec- <br />                                                 essary tolerate low-level expression 55.2 non-toxic wild type cells. These re- <br />                                                 sults obtained using 55.2 expressing plasmid  30 copies cell the <br />                                                 presence glucose, represses expression PBAD promoter [49]. Consequently, <br />                                                 level 55.2 expression manifests incompatibility topA deletion strain low. <br />                                                 Significantly, presence 55.2-coding plasmid reduced frequency topA+ mini- <br />                                                 F plasmid loss topA+ genetic background (II). This consistent low levels of <br />                                                 55.2 expression having effect growth wild-type bacteria (S1A S1B Fig) and <br />                                                 suggests mere duplication topA locus suffices partially overcome effect. <br />                                                     As mentioned introduction, E. coli possesses type IA topoisomerase, Topo III <br />                                                 (topB). We asked whether, context low-level expression 55.2, Topo III partial- <br />                                                 ly compensate decrease Topo I activity. A medium copy pBAD plasmid expressing 55.2 <br />                                                 transformed E. coli strains harboring temperature sensitive (ts) allele topA driven <br />                                                 lacZ promoter wild-type disrupted topB gene. We assessed via- <br />                                                 bility strains presence various concentration IPTG 37&#176;C, temperature <br />                                                 Topo I produced topAts allele active wild-type enzyme [50]. <br />                                                 The results, presented S1C Fig, confirm incompatibility reduced Topo I activity with <br />                                                 residual expression 55.2 PBAD promoter (upper panel). The incapacity PlacZ- <br />                                                 topA &#916;topB bacteria grow absence IPTG repeated known non-viability topA <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                                8 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           topB double mutants [48,51]. Importantly, reduced viability phenotype caused low-level <br />                                           expression 55.2 stronger absence functional Topo III (lower panel). This fur- <br />                                           ther decrease viability caused 55.2 &#916;topB strain evident presence <br />                                           1mM IPTG, concentration topB+ &#916;topB strains harboring control vec- <br />                                           <span id='am-134' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-138'  ></span>tor</span> difference viabilty. These results indicate E. coli requires minimal type IA <br />                                           topoisomerase activity tolerate low-level expression 55.2. <br />  <br />                                           Inhibiting GyrB ATPase subunit gyrase reduces 55.2-induced <br />                                           toxicity <br />                                           The supercoiling level E. coli chromosome plasmids primarily determined by <br />                                           opposing activities Topo I, removes negative supercoils DNA gyrase (gyrAB), <br />                                           adds negative supercoils (Fig 2A). Thus, inhibition DNA gyrase expected to <br />                                           similar effect increasing Topo I activity. Consequently, likely an <br />                                           inhibition counteract 55.2-induced toxicity. We tested sensitivity 55.2-express- <br />                                           ing E. coli cells novobiocin, aminocoumarin type antibiotic inhibits gyrase&#8217;s DNA su- <br />                                           percoiling competing ATP binding GyrB subunit [52&#8211;54]. As before, <br />                                           arabinose inhibited growth bacteria harboring 55.2 coding pBAD101 plasmid. Howev- <br />                                           er, low concentrations novobiocin (between 16 64 &#956;g ml-1), effect the <br />                                           growth bacteria carrying control plasmid, partially reduced toxicity associated with <br />                                           55.2 expression (Fig 2B). Furthermore, 55.2 expression permitted bacterial growth bacteria <br />                                           128 &#956;g ml-1 novobiocin, concentration inhibited growth absence arabinose <br />                                           harbored pBAD101 plasmid (see arrowhead Fig 2B). <br />                                               This suggested limiting DNA gyrase function alleviated, partially, growth in- <br />                                           hibition mediated 55.2. However, novobiocin inhibit topoisomerase IV (Topo <br />                                           IV) E. coli binding ParE subunit, homologous GyrB [55]. Thus, asked <br />                                           partial inactivation DNA gyrase Topo IV sufficed suppress 55.2 toxic- <br />                                           ity. We transformed gyrB parE ts mutant strains 55.2 coding plasmid, tested <br />                                           sensitivity arabinose different temperatures. While 55.2 expression blocked bacterial <br />                                           growth strains permissive temperature (34&#176;C), intermediate temperature <br />                                           (37&#176;C), permitted growth bacteria carrying ts alleles, led <br />                                           partial suppression 55.2-induced toxicity gyrBts parEts mutant strain <br />                                           (Fig 2C). Complete inhibition DNA gyrase 42&#176;C abolished growth independently 55.2 <br />                                           expression. Thus, conclude partial inhibition GyrB sufficient offset toxicity <br />                                           high-level expression 55.2. <br />  <br />                                           Gp55.2 binds DNA vitro <br />                                           Next, asked gp55.2 bind DNA. The vitro binding gp55.2 DNA was <br />                                           demonstrated electrophoretic mobility shift assays (EMSA) employing purified gp552-His6 <br />                                           protein plasmid DNA pDB29 (a pBR322 derivative containing mouse urokinase <br />                                           cDNA insert; [38]). As shown Fig 3A, gp55.2 caused linear pDB29 DNA (upper panel) to <br />                                           migrate slowly. This shift easily detectable ratio gp55.2 molecule 16.4 <br />                                           bp DNA (380 ng gp55.2 300 ng DNA). At protein DNA ratio, gp55.2 re- <br />                                           tarded migration covalently closed, relaxed circular DNA (form I&#8217;, middle panel), co- <br />                                           valently closed, negatively supercoiled circular DNA (form I, lower panel), small <br />                                           nicked circular DNA (form II, lower panel, arrow) contaminated form I <br />                                           DNA. Additional EMSA using M13 phage DNA showed gp55.2 retarded migration of <br />                                           double-stranded replicative form circular single-stranded virion DNA (S2A <br />                                           Fig). Thus, gp55.2 appears bind negatively supercoiled, relaxed, single-stranded DNA. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                              9 / 23 <br />                                                                                           A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />  <br /> Fig 2. Partial inhibition DNA gyrase ATPase activity protects 55.2-induced toxicity. (A) Outline regulation steady-state level DNA <br /> supercoiling opposite actions DNA gyrase Topo I. (B) Novobiocin sensitivity assay. Overnight cultures DB503 bacteria transformed with <br /> pBAD101 (55.2 &#8211;) pDB2114-101 (55.2 +) spotted 24 wells LB agar plates 0.2% arabinose containing indicated amounts of <br /> novobiocin, incubated overnight 37&#176;C. (C) W3110 (wt), CL001 (parEts), BP199 (gyrBts) transformed pBAD101 (55.2 &#8211;) pDB2114-101 <br /> (55.2 +). Dilution overnight cultures spotted LB agar plates 0.2% arabinose incubated overnight the <br /> indicated temperatures. <br /> doi:10.1371/journal.pone.0124309.g002 <br />  <br />                                                In vitro gp55.2 inhibits Topo I does stimulate gyrase activity <br />                                                The genetic evidence combined capacity gp55.2 bind DNA suggests this <br />                                                protein inhibit Topo I activity stimulate DNA gyrase. We directly <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                          10 / 23 <br />                                                                                             A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />  <br /> Fig 3. Gp55.2 binds DNA, inhibits Topo I relaxation activity does affect DNA gyrase supercoiling activity. (A) Electrophoretic mobility shift <br /> assays (EMSA). Mixtures containing 300 ng (= 79 fmol) linear, form I&#8217;, form I pDB29 DNA indicated gp55.2-His6 (95 ng = 6.9 pmol, <br /> namely gp55.2 molecule 65.6 bp DNA used assay) incubated analyzed described Materials Methods. An <br /> arrow indicates migration position form II DNA contaminating form I DNA. (B) Relaxation assays mixtures containing 600 ng (= 158 fmol) form I <br /> pDB29 DNA indicated units Topo I (0.47 U = 415 fmol) incubated presence (+) absence (&#8211;) 855 ng gp55.2-His6 (= 63 pmol, one <br /> gp55.2 molecule 14.5 bp DNA), DNA products analyzed described Materials Methods. The migration positions form I, form I&#8217; <br /> and form II DNA indicated left; linear DNA migration position indicated arrowhead. Results representative independent experiments <br /> are shown. (C) Supercoiling assays mixtures containing 158 fmol form I&#8217; pDB29 DNA indicated units DNA gyrase (0.42 U = 196 fmol) were <br /> incubated presence (+) absence (&#8211;) 855 ng gp55.2-His6 DNA products analyzed described Materials Methods <br /> section. Results representative independent experiments shown. (D) Representation form I, form I&#8217; form II plasmid DNA. Note the <br /> treatment form I plasmid DNA eukaryotic Topo I obtain form I&#8217; DNA results, equilibrium, distribution relaxed DNA topoisomers (A, middle <br /> panel) maximum corresponds fully relaxed form I&#8217; plasmid illustrated D. <br /> doi:10.1371/journal.pone.0124309.g003 <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                            11 / 23 <br />                                                                                  A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           examined vitro effect gp55.2 E. coli Topo I DNA gyrase activities. The relaxing <br />                                           activity Topo I measured presence absence purified gp552-His6 concen- <br />                                           tration slightly higher lowest concentration band shift observed the <br />                                           EMSA assays. As shown Fig 3B, gp55.2 strongly inhibited relaxation activity Topo I, <br />                                           lowest molar ratio gp55.2 Topo I tested (150:1; lane 10). Based amount <br />                                           enzyme required obtain similar relaxation levels absence presence gp55.2, <br />                                           estimated relaxation activity inhibited 10 30 fold (compare lanes 2 3 <br />                                           lane 10). Further experiments, Topo I fixed gp55.2 <br />                                           varied, indicated ratio gp55.2 molecule 7 bp DNA required com- <br />                                           plete inhibition (S3A Fig, lane 8). However, lower gp55.2 DNA ratio, super- <br />                                           coiled relaxed topoisomers observed presence absence of <br />                                           gp55.2 (S3A Fig, lane 4 7), suggesting gp55.2 stimulate Topo I activity low <br />                                           protein DNA ratio. In stark contrast, gp55.2 effect negative supercoiling activ- <br />                                           ity DNA gyrase (Fig 3C). We tested gp55.2 inhibit relaxation su- <br />                                           percoiled DNA wheat germ Topo I, type 1B topoisomerase structure and <br />                                           mechanism relaxation unrelated E. coli Topo I [6]. The enzymatic activity of <br />                                           wheat germ Topo I largely insensitive gp55.2; furthermore, observed limited inhibi- <br />                                           tion relaxation (  20%) independent concentration gp55.2 used, suggesting <br />                                           reached saturation (S3B Fig). Altogether, results gp55.2 inhibits specifi- <br />                                           cally E. coli Topo I vitro. <br />  <br />  <br />                                           55.2 affects DNA topology control vivo <br />                                           We investigated effect vivo low, non-toxic levels 55.2 expression supercoiling <br />                                           plasmid DNA. To this, used pDB2114, pBAD-55.2 plasmid intermediate <br />                                           copy number, source 55.2 expression, residual expression PBAD ab- <br />                                           sence arabinose. The plasmid served reporter DNA supercoiling. As a <br />                                           control, used closely related plasmid 55.2 coding sequence dis- <br />                                           rupted 5bp insertion ATG initiation codon producing frame-shifted 18 amino <br />                                           acid peptide instead gp55.2. This frame shift abolished gp55.2 phenotypes parent <br />                                           plasmid presence absence arabinose. Plasmid DNA extracted bac- <br />                                           teria growing exponentially absence arabinose analyzed dimension agarose <br />                                           gel electrophoresis presence chloroquine, intercalating ligand adds positive su- <br />                                           percoils plasmid DNA allows separation supercoiled topoisomers [56]. As seen in <br />                                           Fig 4A, distribution topoisomers slightly different plasmids carrying in- <br />                                           tact mutated copy 55.2. Changes distribution slow migrating, partially re- <br />                                           laxed topoisomers evident gel containing 1 &#956;g ml-1 CLQ, fast <br />                                           migrating, negatively supercoiled topoisomers better resolved gel containing 1.5 &#956;g <br />                                           ml-1 CLQ. The densitometry profiles right panel summarize analysis indepen- <br />                                           dent DNA samples plasmid (see S4A Fig). Three principal differences ob- <br />                                           served. The slight shift profiles 5 bp difference size the <br />                                           plasmids. It theoretically predicted, experimentally demonstrated, small <br />                                           changes size (&lt;10bp), affect perceptibly mobility linear form II <br />                                           (nicked circle) DNA, cause size-dependent decrease gel mobility topoisomers <br />                                           linking number. The half turn shift observed perfectly accordance with <br />                                           published results [57]. Aside size-dependent change, relevant differences <br />                                           observed. First, 55.2-coding plasmid broader distribution topoisomers com- <br />                                           pared control plasmid. In addition, peak topoisomer distribution 55.2- <br />                                           coding plasmid shifted negative supercoiling half helical turn. To <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                            12 / 23 <br />                                                                                        A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />  <br /> Fig 4. 55.2 expression affects control DNA topology plasmid copy number E. coli. (A) Plasmid topoisomers analysis. Left panel: plasmid <br /> DNA extracted exponentially growing DH5&#945; harboring pYM58 (55.2IS) pDB2114 (55.2) plasmids. Plasmid topoisomers resolved TBE <br /> agarose gels containing indicated CLQ. The position migration relaxed and/or nicked circular DNA indicated (R). Right panel: <br /> Densitometry analysis topoisomer distribution 1.5 &#956;g ml-1 CLQ gel independent samples pYM58 pDB2114 plasmid DNA (gel images <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                     13 / 23 <br />                                                                                               A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br /> are depicted S4A Fig). Plotted average (lines) standard deviation (shaded area) relative plasmid density function negative supercoiling. <br /> (B) 2D electrophoretic separation plasmid topoisomers. Left panel: schematic representation 2D gel. The migration positions negatively supercoiled <br /> (&#8211;), positively supercoiled (+), form II topoisomers (N) indicated. Right panel: gel images 2D topoisomer distribution plasmid DNA <br /> samples prepared A. Chloroquine concentration 1.5 &#956;g ml-1 25 &#956;g ml-1 second dimension, respectively. (C) Plasmid copy <br /> number analysis. Linearized plasmid DNA samples experiment shown A quantified normalized (A600nm) bacteria used <br /> to extract plasmids. The data represents means standard error independent cultures. (D) Plasmid DNA extracted overnight cultures <br /> of DHB3, DB503, DH5&#945; transformed pBAD101 (55.2 &#8211;) pDB2114-101 (55.2 +), reporter plasmid (pDB868-2). Linearized plasmids were <br /> analyzed agarose gel electrophoresis. <br /> doi:10.1371/journal.pone.0124309.g004 <br />  <br />                                                 obtain better resolution relaxed topoisomers, supercoiling state plasmid <br />                                                 analyzed using two-dimensional chloroquine gels. The results presented Fig 4B <br />                                                 clearly low-level expression 55.2 caused appearance relaxed and <br />                                                 negatively supercoiled topoisomers. The early stop codon 55.2 mutant plasmid <br />                                                 used control cause premature Rho-dependent transcription termination [58]. Be- <br />                                                 cause transcription elongation affects vivo supercoiling level plasmids [59], per- <br />                                                 formed control experiment determine potential effect transcriptional polarity. <br />                                                 We compared pBAD plasmids containing wild type 55.1 gene, T4 ORFan adja- <br />                                                 cent functionally unrelated 55.2, mutated 55.1 version ATG start <br />                                                 codon changed AGG. Both plasmid DNAs displayed identical topoisomer distri- <br />                                                 butions; difference slight increase the negatively super- <br />                                                 coiled topoisomers plasmid wild-type 55.1 sequence (S4B S4C Fig). <br />                                                 However, increase marked case wild-type 55.2 plasmid. <br />                                                 Thus, conclude low-level non-toxic expression 55.2 effect regulation of <br />                                                 plasmid DNA supercoiling E. coli. <br />                                                    To determine chromosomal DNA supercoiling affected 55.2 expression, we <br />                                                 measured level topA gyrA mRNA transcription genes deter- <br />                                                 mined, part, supercoiling promoter regions. The transcription rate topA in- <br />                                                 creases increased negative supercoiling gyrA increases DNA <br />                                                 relaxation [60&#8211;62]. Total RNA extracted bacteria harboring 55.2-coding <br />                                                 pBAD101 plasmid, vector, cultures growing exponentially ab- <br />                                                 sence arabinose. RNAse protection assays performed using topA gyrA probes (S4D <br />                                                 Fig), relative ratio topA gyrA-protected fragments determined densi- <br />                                                 tometric analysis. As shown S4E Fig, topA gyrA ratio slightly lower bacteria <br />                                                 harboring 55.2 plasmid indicating low-level expression 55.2 did increase super- <br />                                                 coiling promoters somewhat reduced it. <br />  <br />  <br />                                                 Low non-toxic level 55.2 increases copy number plasmids <br />                                                 replication controlled antisense RNA <br />                                                 In analyzing effects DNA supercoiling mediated gp55.2, noticed amount <br />                                                 DNA gp55.2 encoding plasmid systematically higher present vari- <br />                                                 ous control plasmids. This pBAD vector origin replication derived ColEI, whose <br />                                                 replication requires formation R-loop plasmid origin comple- <br />                                                 mentary RNAII serves primer DNA replication [63]. The formation R-loops is <br />                                                 increased negative supercoiling inhibited action RNAse H1 Topo I [59]. <br />                                                 To quantify effect described above, aliquots plasmid DNA prepared supercoil- <br />                                                 ing experiment described Fig 4A linearized restriction digestion, quantified aga- <br />                                                 rose gel comparison DNA standard, normalized total bacteria <br />                                                 used extract plasmid. The results 55.2-coding plasmid was <br />                                                 seven times greater 55.2-mutant plasmid (Fig 4C). Similarly, overnight <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                                14 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           cultures DH5&#945; DB503 (S4F Fig), copy number 55.2-coding plasmid in- <br />                                           creased fold compared control plasmid. We determined the <br />                                           presence gp55.2 coding sequence operated <span id='am-97' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span> gp55.2 acted in <br />                                           trans. Bacteria transformed 55.2-coding pBAD101 plasmid reporter plasmid <br />                                           p15A-derived replication origin, related ColEI replicon [64]. Plasmid <br />                                           DNA extracted equal amounts overnight cultures linearized restriction di- <br />                                           gestion. Compared control, 55.2-coding plasmid increased quantity the <br />                                           reporter plasmid strains tested (Fig 4D). This demonstrates very <br />                                           low levels gp55.2 expression (no arabinose present cultures 55.2 gene was <br />                                           low copy plasmid) affect plasmid copy number regulation trans. Interestingly, 55.2 <br />                                           did noticeably affect copy number pBAD101 (upper band Fig 4D), replica- <br />                                           tion RepA dependent [65,66]. These results confirm interesting unexpected observa- <br />                                           tion gp55.2 significantly increases copy number plasmids replication is <br />                                           dependent R-loop formation. This effect entirely compatible observed increased <br />                                           plasmid supercoiling. <br />  <br />  <br />                                           A null mutation gene 55.2 decreases fitness phage T4 <br />                                           Most T4 ORFans probably &#8220;non-essential genes&#65533;? lie genomic regions that <br />                                           deleted noticeably affecting phage yield [24]. However, excluded <br />                                           absence gp55.2 function influence production T4 progeny stan- <br />                                           dard laboratory conditions. We used T4 I/S [37] replace ATG initiation <br />                                           codon 55.2 ACA (threonine) codon genome K10 T4 strain; analo- <br />                                           gous substitution plasmid completely abolished 55.2-induced AraS phenotypes (data <br />                                           shown). The plaque morphology resulting 55.2 mutant indistinguishable from <br />                                           K10 parent strain (data shown). However, 55.2 mutant phage exhibited a <br />                                           small reproducible minutes delay accumulation intracellular viral particles in <br />                                           minimal media. Nonetheless, eventually achieved burst size similar parental <br />                                           strain (Fig 5A). A comparable delay observed complete media 37&#176;C (S5A Fig). In <br />                                           <span id='am-58' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> determine small difference change overall fitness 55.2 <br />                                           mutant strain, compared growth parental strain successive growth <br />                                           cycles. In experiments, mixture control mutant phages grown E. coli bac- <br />                                           teria; progeny initial growth cycle used perform second growth cycle so <br />                                           forth. Importantly, infections performed low moi (&lt; 0.1) minimize co-infection <br />                                           complementation wild type phage. The initial phage mix contained 90% K10-55.2 <br />                                           mutant 10% K10 wild-type phages decreased fitness 55.2 mutant an- <br />                                           ticipated. After 19 growth cycles, 55.2 mutant phage represented 30% popu- <br />                                           lation (Fig 5B). A simulation showed results obtained mutant phage <br />                                           16% growth disadvantage growth cycle (dotted line Fig 5B). Hence, conclude <br />                                           that, 55.2 &#8220;non-essential gene&#65533;?, confers non-trivial growth advan- <br />                                           tage T4 standard laboratory conditions. <br />  <br />  <br />                                           Discussion <br />                                           We previously identified 55.2 T4 ORFan ectopic expression toxic unin- <br />                                           fected E. coli [29]. When cloned medium copy pBAD plasmid (ColEI replicon), low- <br />                                           level &#8220;leaky&#65533;? expression 55.2 PBAD promoter absence arabinose caused an <br />                                           increased lag phase growth diluted liquid cultures decreased bacterial titer sat- <br />                                           uration. These effects detected lower copy plasmid (pSC101 replicon) the <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             15 / 23 <br />                                                                                       A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />  <br />                                           Fig 5. Loss gene 55.2 function reduces T4 phage fitness. (A) CR63 cells grown M9 medium were <br />                                           infected T4 K10 (wt) T4 K10-55.2 (55.2) moi 6 30&#176;C. Intracellular phage accumulation was <br />                                           followed indicated time points; free phages 25 min represented &lt; 12% total infective centers. <br />                                           Data represents mean ranges (wt) (55.2) independent experiments. (B) Competition <br />                                           experiment. A mix T4 K10 T4 K10-55.2 initial ratio 1:9 grown E. coli CR63 M9S <br />                                           medium 37&#176;C low moi (&lt; 0.1) successive growth cycles. The percentage 55.2+ phages was <br />                                           determined PCR digestion described Materials Methods. Data represent mean and <br />                                           standard deviation independent experiments; dotted line represents result simulation were <br />                                           55.2 mutant 16% disadvantage growth cycle. The intracellular phage accumulation a <br />                                           single growth cycle conditions shown S5A Fig. <br />                                           doi:10.1371/journal.pone.0124309.g005 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                       16 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           absence arabinose. Induced expression gp55.2 presence arabinose led irrevers- <br />                                           ible growth arrest plasmid backgrounds. <br />                                               Further analysis identified ways partially suppress 55.2-mediated bacterial kill- <br />                                           ing: multicopy expression Topo I partial inhibition GyrB activity. These results sug- <br />                                           gested gp55.2 acting inhibiting Topo I activity increasing DNA gyrase <br />                                           activity. A plasmid-based lethality assay clearly demonstrated low-level, normally non- <br />                                           toxic, expression gp55.2 compatible reduced Topo I activity. This incompatibility <br />                                           independently confirmed using strain topA expression regulated by <br />                                           IPTG. Interestingly, absence Topo III, second type IA topoisomerase E. coli, <br />                                           bacteria expressing low levels gp55.2 required higher expression Topo I grow. Taken to- <br />                                           gether, results support notion 55.2 inhibits DNA relaxation and/or stimulates <br />                                           DNA supercoiling. <br />                                               In vitro experiments demonstrated gp55.2 inhibits relaxing activity E. coli Topo I <br />                                           does affect introduction negative supercoils E. coli gyrase. The inhibi- <br />                                           tion Topo I activity gp55.2 result direct protein-protein interaction. Alterna- <br />                                           tively, gp55.2 binding DNA alter double helix conformation and/or occlude the <br />                                           sites Topo I binds. Increased Topo I expression shown suppress toxici- <br />                                           ty associated Tn5 transposase (Tnp) overproduction [67,68]. A direct interaction between <br />                                           proteins responsible vitro inhibition Topo I activity in <br />                                           vivo suppression Tnp toxicity. We failed detect interaction gp55.2 Topo I <br />                                           using vivo pull-down assays (data shown). Although opti- <br />                                           mal condition detect interaction, nonetheless arguments disfa- <br />                                           vor hypothesis. Firstly, alteration DNA topology required suppression <br />                                           Tnp toxicity overexpression partially defective mutant Topo I suppress Tnp <br />                                           toxicity. Conversely, showed partial inhibition gyrase partly suppressed gp55.2 le- <br />                                           thality, reduce topA expression [60,62]. Second, molar ratio pro- <br />                                           tein Topo I required observe vitro inhibition far lower Tnp gp55.2 (1&#8211; <br />                                           10:1 vs. &gt;150:1). The high concentration gp55.2 required Topo I inhibition supports the <br />                                           alternative site occlusion/DNA conformation hypothesis. Such mechanism explains inhi- <br />                                           bition Topo I nucleoid-associated protein (NAP) HU high protein DNA ratio <br />                                           [69]. EMSA showed gp55.2 Topo I, 35:1 molar ratio, bind simultaneously to <br />                                           plasmid DNA (S2B Fig). Nevertheless, occlusion place higher gp55.2 Topo <br />                                           I ratio required detect inhibition (  150:1). We tested gp55.2 alter DNA <br />                                           topology own. The incubation gp55.2 relaxed DNA (form I&#8217;) presence of <br />                                           wheat germ Topo I showed that, studied conditions, phage protein constrain <br />                                           negative positive supercoils (S3C Fig). Further studies needed determine <br />                                           precise mechanism inhibition. One possibility gp55.2 binds specific topo- <br />                                           logical features favored negative supercoiling, like single-stranded DNA regions are <br />                                           required DNA supercoil relaxation Topo I [70]. This hypothesis, inferred ca- <br />                                           pacity gp55.2 bind single-stranded M13 DNA, supported observations. <br />                                           Firstly, gp55.2 marginally affected activity wheat germ Topo I does require <br />                                           single-stranded DNA substrate [6]. Secondly, low protein DNA ratio, gp55.2 to <br />                                           stimulate inhibit Topo I activity. An ambivalent effect Topo I activity previ- <br />                                           ously reported E. coli single-stranded DNA binding protein (SSB) stimulates non- <br />                                           covalent interaction Topo I DNA stabilizing single-stranded region low protein to <br />                                           DNA ratio inhibits DNA relaxation higher ratio [71]. <br />                                               The vivo distribution plasmid topoisomers indicates low-level expression 55.2 <br />                                           caused appearance negatively supercoiled relaxed topoisomers, re- <br />                                           sulting broader distribution absence gp55.2. The measurement the <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             17 / 23 <br />                                                                                   A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           relative expression topA gyrA compatible unchanged slightly reduced chro- <br />                                           mosome superhelical density bacteria expressing gp55.2. These results appear counterintui- <br />                                           tive complete defect Topo I results increased negative supercoiling both <br />                                           plasmids bacterial chromosome [45]. However, limited gp55.2 produced <br />                                           uninduced PBAD promoter unlikely completely inhibit Topo I activity. Actually, <br />                                           low-level 55.2 expression did block rapid relaxation induced inhibition DNA gyr- <br />                                           ase norfloxacin (S4G Fig), process entirely dependent Topo I activity <br />                                           [72]. Furthermore, partial inhibition Topo I compensated homeostatic <br />                                           regulation bacterial DNA supercoiling takes place enzyme activity gene <br />                                           expression levels [73]. We note vitro Topo I activity inhibited <br />                                           stimulated low gp55.2 DNA ratio resulting broader topoisomer distribution than <br />                                           absence gp55.2. The vivo observations reconciled genetic in <br />                                           vitro data inhibition Topo I gp55.2 stronger DNA substrates on <br />                                           others. DNA containing R-loops potential candidate. Indeed, low-level expression 55.2 <br />                                           caused strong increase copy number plasmids use R-loop replication, R- <br />                                           looped supercoiled plasmids better substrate Topo I supercoiled plasmids without <br />                                           R-loops [74]. Topo I prevents excessive R-loop formation topA null alleles incompati- <br />                                           ble deletions rnhA, encodes RNAse HI, enzyme degrades RNA strand <br />                                           R-loops [47,59,75]. Thus, severe phenotype induced low levels gp55.2 the <br />                                           absence RNAse H1 (S1C Fig) consistent partial inhibition Topo I reflect <br />                                           preferential inhibition Topo I activity supercoiled DNA containing R-loops. <br />                                               The vitro experiments suggest high levels gp55.2 completely inhibit Topo I <br />                                           vivo. This supported suppression 55.2 lethality topA overexpression partial <br />                                           inactivation DNA gyrase. Indeed, thermosensitive mutation gyrB (gyrB203) used in <br />                                           study suppress growth defect topA null mutation 37&#176;C [47]. Yet, Topo I inhibi- <br />                                           tion entirely explain gp55.2 killing. Indeed, consensus fact that <br />                                           Topo I absolutely required growth [31,48]. Furthermore, plasmid-based le- <br />                                           thality assay experiments tunable topA allele showed low-level expression <br />                                           55.2 diminished growth capacity bacteria devoid Topo I. An interesting pos- <br />                                           sibility gp55.2 inhibits Topo I Topo III. As mentioned above, topA topB <br />                                           double mutants non-viable. Although topoisomerase different physiological <br />                                           role, Topo III structurally functionally related Topo I relax negatively super- <br />                                           coiled DNA vitro [76]. Indeed, showed bacteria required Topo III tolerate low <br />                                           level gp55.2 Topo I activity reduced. <br />                                               Altogether, simplest hypothesis explain data presented study gp55.2 <br />                                           inhibits Type IA topoisomerases. Nevertheless, rule reduction of <br />                                           DNA negative supercoiling required accommodate suppress consequence a <br />                                           gp55.2 interaction DNA, conjunction NAP. In case, vitro in- <br />                                           hibition Topo I gp55.2 incidental consequence gp55.2 binding inter- <br />                                           mediate DNA structure required supercoil relaxation Type IA topoisomerase. <br />                                               Disruption 55.2 T4 phage showed gene bona fide non-essential gene, since <br />                                           intracellular accumulation virion particles slightly retarded null mutant. <br />                                           However, gp55.2 conserved T4-related phages infect different bacterial genera <br />                                           [25]. Competition assays showed that, standard laboratory conditions, 55.2 provided <br />                                           easily detectable growth advantage. Thus, 55.2 pathways re- <br />                                           quired optimal viral replication cycle. The toxicity 55.2 characterization of <br />                                           mRNA early transcript [77] suggests role transition host phage metab- <br />                                           olism phage DNA replication. The gp55.2 protein play role regulation su- <br />                                           percoiling phage. Although physically linear, phage genome shows transient <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             18 / 23 <br />                                                                                  A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           acquisition negative superhelicity precedes T4 DNA synthesis requires host gyr- <br />                                           ase [78]. The function DNA supercoiling T4 infection clear: hand, <br />                                           gyrase activity required T4 DNA synthesis [22,79], hand, burst <br />                                           size T4 phage reduced gyrB mutant host [22] release superhelical tension <br />                                           slightly reduces rate T4 DNA synthesis [72]. Further investigation required de- <br />                                           termine fitness provided gp55.2 directly linked modulation host Topo I ac- <br />                                           tivity infection. We examined importance gp55.2 function absence of <br />                                           phage topoisomerase. The intracellular growth curve 39am, 55.2 double mutant was <br />                                           similar 39 single mutant, showing extended eclipse period reduced burst size <br />                                           characteristic DNA delay mutants (S5B Fig). However, mature phage particle accumulation <br />                                           slightly delayed double mutant, observed comparison 55.2 single mu- <br />                                           tant T4+. This additive effect suggests gp55.2 T4 topoisomerase act indepen- <br />                                           dent biochemical pathways. <br />                                               Despite T4 studied phages, genome codes &gt;100 ORFans whose <br />                                           biological functions, any, largely unknown [24]. These genes generally thought to <br />                                           non-essential largely regrouped so-called hyperplastic regions (HPR) of <br />                                           T4 genome. Several hypotheses proposed explain maintenance through <br />                                           evolution. Some genes contribute &#8220;molecular arms race&#65533;? resistance <br />                                           mechanisms new ways circumvent constantly evolving bacteria phages <br />                                           [80,81]. The existence numerous ORFans far unique branch T4-like <br />                                           phage tree [25,82] suggests HPR serve breeding ground novo phage gene <br />                                           birth [26,83]. The present work, recent characterization ORFans 55.1 [29] and <br />                                           39.2 [84], fit hypothesis: T4 ORFans modify host protein functions to <br />                                           fine tune bacterial metabolism permit optimal virion production (see &#8220;molecular <br />                                           splint&#65533;? hypothesis [26]). Gp39.2 shifts open/close equilibrium host GroEL chaper- <br />                                           one. Gp55.1 impacts host folate metabolism blocks nucleotide excision repair inter- <br />                                           acting FolD UvrA respectively, gp55.2 affects host control DNA <br />                                           supercoiling. Although deletion mutants genes perfectly viable E. <br />                                           coli K12 strains, Gp39.2 permits T4 growth E. coli groEL mutants other- <br />                                           wise yield productive infections. Furthermore, demonstrated 55.2 55.1 mutants <br />                                           (Fig 5B data shown) detectably reduced fitness. We conclude T4 ORFans <br />                                           major evolutionary role providing small evolutionary significant growth <br />                                           advantages non-optimal hosts environmental conditions. <br />  <br />  <br />  <br />  <br />                                           Supporting Information <br />                                           S1 Fig. Supplementary growth curves viability assays. <br />                                           (TIFF) <br />                                           S2 Fig. Supplementary EMSA. <br />                                           (TIFF) <br />                                           S3 Fig. Supplementary topoisomerase assays. <br />                                           (TIFF) <br />                                           S4 Fig. Supplementary analyses vivo DNA supercoiling plasmid copy number. <br />                                           (TIFF) <br />                                           S5 Fig. Supplementary analyses intracellular phage accumulation. <br />                                           (TIFF) <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                            19 / 23 <br />                                                                                       A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           S1 File. Supporting Materials Methods. Construction E. coli strains, T4 phage strains, <br />                                           plasmids, genomic library; methods used supporting figures; list plasmids <br />                                           used study. <br />                                           (DOCX) <br />  <br />  <br />                                           Acknowledgments <br />                                           We thank Drs. C. Georgopoulos G. Panis, members laboratory, anonymous re- <br />                                           viewers helpful discussions suggestions; Dr. H. Krisch critical readings manu- <br />                                           script Dr. J. Curran English editing; Marie Claire Teuscher L&#233;o Zangelmi for <br />                                           plasmid T4 strain construction, respectively; Dr A. Stasiak help set and <br />                                           analysis chloroquine gel electrophoresis; Dr K. Kreuzer sharing knowledge the <br />                                           phage T4 topoisomerase. We grateful Dr C.J. Rudolph Dr K.J. Marians plas- <br />                                           mids strains. <br />  <br />                                           Author Contributions <br />                                           Conceived designed experiments: YM DB. Performed experiments: YM FS. Ana- <br />                                           lyzed data: YM FS DB. Wrote paper: YM DB. <br />  <br />                                           References <br />                                            1.   Schvartzman JB, Stasiak A (2004) A topological view replicon. EMBO Rep 5: 256&#8211;261. PMID: <br />                                                 14993926 <br />                                            2.   Liu LF, Wang JC (1987) Supercoiling DNA template transcription. Proc Natl Acad Sci U S <br />                                                 A 84: 7024&#8211;7027. PMID: 2823250 <br />                                            3.   Higgins NP, Deng S, Pang Z, Stein R, Champion K, Manna D (2005) Domain behavior supercoil <br />                                                 dynamics bacterial chromosomes In: Higgins NP, editor. The Bacterial Chromosome. Washington, <br />                                                 DC: American Society Microbiology Press. pp. 133&#8211;153. <br />                                            4.   Travers A, Muskhelishvili G (2005) DNA supercoiling&#8212;a global transcriptional regulator entero- <br />                                                 bacterial growth? Nat Rev Microbiol 3: 157&#8211;169. PMID: 15685225 <br />                                            5.   Stavans J, Oppenheim A (2006) DNA-protein interactions bacterial chromosome architecture. <br />                                                 Phys Biol 3: R1&#8211;10. PMID: 17200598 <br />                                            6.   Champoux JJ (2001) DNA topoisomerases: structure, function, mechanism. Annu Rev Biochem <br />                                                 70: 369&#8211;413. PMID: 11395412 <br />                                            7.   Wang JC (2002) Cellular roles DNA topoisomerases: molecular perspective. Nat Rev Mol Cell Biol <br />                                                 3: 430&#8211;440. PMID: 12042765 <br />                                            8.   Forterre P, Gadelle D (2009) Phylogenomics DNA topoisomerases: origin putative roles in <br />                                                 emergence modern organisms. Nucleic Acids Res 37: 679&#8211;692. doi: 10.1093/nar/gkp032 PMID: <br />                                                 19208647 <br />                                            9.   Forterre P, Gribaldo S, Gadelle D, Serre MC (2007) Origin evolution DNA topoisomerases. Bio- <br />                                                 chimie 89: 427&#8211;446. PMID: 17293019 <br />                                           10.   Kreuzer KN (1998) Bacteriophage T4, model understanding mechanism type II topo- <br />                                                 isomerase inhibitors. Biochim Biophys Acta 1400: 339&#8211;347. PMID: 9748648 <br />                                           11.   Drlica K (1999) Mechanism fluoroquinolone action. Curr Opin Microbiol 2: 504&#8211;508. PMID: <br />                                                 10508721 <br />                                           12.   Zechiedrich EL, Khodursky AB, Bachellier S, Schneider R, Chen D, Lilley DM, et al. (2000) Roles of <br />                                                 topoisomerases maintaining steady-state DNA supercoiling Escherichia coli. J Biol Chem 275: <br />                                                 8103&#8211;8113. PMID: 10713132 <br />                                           13.   Perez-Cheeks BA, Lee C, Hayama R, Marians KJ (2012) A role topoisomerase III Escherichia coli <br />                                                 chromosome segregation. Mol Microbiol 86: 1007&#8211;1022. doi: 10.1111/mmi.12039 PMID: 23066834 <br />                                           14.   Liu LF, Liu CC, Alberts BM (1979) T4 DNA topoisomerase: new ATP-dependent enzyme essential for <br />                                                 initiation T4 bacteriophage DNA replication. Nature 281: 456&#8211;461. PMID: 226889 <br />                                           15.   Stetler GL, King GJ, Huang WM (1979) T4 DNA-delay proteins, required specific DNA replication, <br />                                                 form complex ATP-dependent DNA topoisomerase activity. Proc Natl Acad Sci U S A 76: <br />                                                 3737&#8211;3741. PMID: 226976 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                      20 / 23 <br />                                                                                        A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           16.   Epstein RH, Bolle A, Steinberg CM, Kellenberger E, Boy la Tour E, Edgar RS, et al. (1964) Physio- <br />                                                 logical studies conditional lethal mutants bacteriophage T4D. Cold Spring Harbor Symp Quant <br />                                                 Biol 28: 375&#8211;394. <br />                                           17.   Mufti S, Bernstein H (1974) The DNA-delay mutants bacteriophage T4. J Virol 14: 860&#8211;871. PMID: <br />                                                 4609406 <br />                                           18.   McCarthy D, Minner C, Bernstein H, Bernstein C (1976) DNA elongation rates growing point distri- <br />                                                 butions wild-type phage T4 DNA-delay amber mutant. J Mol Biol 106: 963&#8211;981. PMID: <br />                                                 789903 <br />                                           19.   Mosig G (1994) Homologous recombination. In: Karam JD, editor. Molecular biology bacteriophage <br />                                                 T4. Washington, D. C.: ASM press. pp. 54&#8211;82. <br />                                           20.   Mosig G, Gewin J, Luder A, Colowick N, Vo D (2001) Two recombination-dependent DNA replication <br />                                                 pathways bacteriophage T4, roles mutagenesis horizontal gene transfer. Proc Natl <br />                                                 Acad Sci U S A 98: 8306&#8211;8311. PMID: 11459968 <br />                                           21.   Kreuzer KN, Brister JR (2010) Initiation bacteriophage T4 DNA replication replication fork dy- <br />                                                 namics: review Virology Journal series bacteriophage T4 relatives. Virol J 7: 358. <br />                                                 doi: 10.1186/1743-422X-7-358 PMID: 21129203 <br />                                           22.   McCarthy D (1979) Gyrase-dependent initiation bacteriophage T4 DNA replication: interactions of <br />                                                 Escherichia coli gyrase novobiocin, coumermycin phage DNA-delay gene products. J Mol Biol <br />                                                 127: 265&#8211;283. PMID: 372540 <br />                                           23.   Epstein RH, Bolle A, Steinberg CM (2012) Amber mutants bacteriophage T4D: isolation ge- <br />                                                 netic characterization. Genetics 190: 833&#8211;840. doi: 10.1534/genetics.112.138438 PMID: 22419076 <br />                                           24.   Miller ES, Kutter E, Mosig G, Arisaka F, Kunisawa T, Ruger W (2003) Bacteriophage T4 genome. <br />                                                 Microbiol Mol Biol Rev 67: 86&#8211;156, table contents. PMID: 12626685 <br />                                           25.   Petrov VM, Ratnayaka S, Nolan JM, Miller ES, Karam JD (2010) Genomes T4-related bacterio- <br />                                                 phages windows microbial genome evolution. Virol J 7: 292. doi: 10.1186/1743-422X-7-292 <br />                                                 PMID: 21029436 <br />                                           26.   Comeau AM, Hatfull GF, Krisch HM, Lindell D, Mann NH, Prangishvili D (2008) Exploring prokaryot- <br />                                                 ic virosphere. Res Microbiol 159: 306&#8211;313. doi: 10.1016/j.resmic.2008.05.001 PMID: 18639443 <br />                                           27.   Brussow H, Hendrix RW (2002) Phage genomics: small beautiful. Cell 108: 13&#8211;16. PMID: 11792317 <br />                                           28.   Comeau AM, Tremblay D, Moineau S, Rattei T, Kushkina AI, Tovkach FI, et al. (2012) Phage morpholo- <br />                                                 gy recapitulates phylogeny: comparative genomics new group myoviruses. PLoS One 7: <br />                                                 e40102. doi: 10.1371/journal.pone.0040102 PMID: 22792219 <br />                                           29.   Mattenberger Y, Mattson S, Metrailler J, Silva F, Belin D (2011) 55.1, gene unknown function of <br />                                                 phage T4, impacts Escherichia coli folate metabolism blocks DNA repair NER. Mol Micro- <br />                                                 biol 82: 1406&#8211;1421. doi: 10.1111/j.1365-2958.2011.07897.x PMID: 22029793 <br />                                           30.   Miller JH (1992) A short course bacterial genetics. Cold Spring Harbor, NY: Cold Spring Harbor Lab- <br />                                                 oratory Press. <br />                                           31.   Stockum A, Lloyd RG, Rudolph CJ (2012) On viability Escherichia coli cells lacking DNA topo- <br />                                                 isomerase I. BMC Microbiol 12: 26. doi: 10.1186/1471-2180-12-26 PMID: 22373098 <br />                                           32.   Daegelen P, Studier FW, Lenski RE, Cure S, Kim JF (2009) Tracing ancestors relatives Escheri- <br />                                                 chia coli B, derivation B strains REL606 BL21(DE3). J Mol Biol 394: 634&#8211;643. doi: 10. <br />                                                 1016/j.jmb.2009.09.022 PMID: 19765591 <br />                                           33.   Boyd D, Manoil C, Beckwith J (1987) Determinants membrane protein topology. Proc Natl Acad Sci <br />                                                 USA 84: 8525&#8211;8529. PMID: 3317413 <br />                                           34.   Baba T, <span id='am-38' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-39' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, Baba M, et al. (2006) Construction Escherichia <br />                                                 coli K-12 in-frame, single-gene knockout mutants: Keio collection. Mol Syst Biol 2: 2006 0008. <br />                                           35.   Casadaban MJ, Cohen SN (1980) Analysis gene control signals DNA fusion cloning in <br />                                                 Escherichia coli. J Mol Biol 138: 179&#8211;207. PMID: 6997493 <br />                                           36.   Casadaban MJ (1976) Transposition fusion lac genes selected promoters Escherichia <br />                                                 coli using bacteriophage lambda Mu. J Mol Biol 104: 541&#8211;555. PMID: 781293 <br />                                           37.   Selick HE, Kreuzer KN, Alberts BM (1988) The bacteriophage T4 insertion/substitution vector system. <br />                                                 A method introducing site-specific mutations virus chromosome. J Biol Chem 263: 11336&#8211; <br />                                                 11347. PMID: 3403529 <br />                                           38.   Belin D, Vassalli J-D, Combepine C, Godeau F, Nagamine Y, Reich E, et al. (1985) Cloning, nucleotide <br />                                                 sequencing expression cDNAs encoding mouse urokinase-type plasminogen activator. Eur- <br />                                                 JBiochem 148: 225&#8211;232. <br />                                           39.   Sambrook J, Fritsch EF, Maniatis T (1990) Molecular cloning. A laboratory manual.: Cold Spring Har- <br />                                                 bor Laboratory Press. <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                        21 / 23 <br />                                                                                        A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           40.   Reckinger AR, Jeong KS, Khodursky AB, Hiasa H (2007) RecA stimulate relaxation activity of <br />                                                 topoisomerase I: Molecular basis topoisomerase-mediated genome-wide transcriptional responses <br />                                                 Escherichia coli. Nucleic Acids Res 35: 79&#8211;86. PMID: 17151069 <br />                                           41.   Clark DJ, Leblanc B (2009) Analysis DNA supercoiling induced DNA-protein Interaction. In: Moss <br />                                                 T, Leblanc B, editors. Methods Molecular Biology: Humana Press. pp. 523&#8211;536. doi: 10.1007/978-1- <br />                                                 60327-015-1_30 PMID: 19378184 <br />                                           42.   Cameron AD, Stoebel DM, Dorman CJ (2011) DNA supercoiling differentially regulated environ- <br />                                                 mental factors FIS Escherichia coli <span id='am-84' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span>Salmonella</span> enterica. Mol Microbiol 80: 85&#8211;101. doi: 10. <br />                                                 1111/j.1365-2958.2011.07560.x PMID: 21276095 <br />                                           43.   Bieler S, Silva F, Soto C, Belin D (2006) Bactericidal activity secreted nonsecreted microcin <br />                                                 E492 requires mannose permease. J Bacteriol 188: 7049&#8211;7061. PMID: 17015644 <br />                                           44.   Bost S, Silva F, Belin D (1999) Transcriptional activation ydeA, encodes member the <br />                                                 major facilitator <span id='am-102' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-103' property="rdfs:label" content="superfamily" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>superfamily</span>, interferes arabinose accumulation induction Escherichia <br />                                                 coli arabinose PBAD promoter. J Bacteriol 181: 2185&#8211;2191. PMID: 10094697 <br />                                           45.   Pruss GJ, Manes SH, Drlica K (1982) Escherichia coli DNA topoisomerase I mutants: increased super- <br />                                                 coiling corrected mutations near gyrase genes. Cell 31: 35&#8211;42. PMID: 6297751 <br />                                           46.   DiNardo S, Voelkel KA, Sternglanz R, Reynolds AE, Wright A (1982) Escherichia coli DNA topoisomer- <br />                                                 ase I mutants compensatory mutations DNA gyrase genes. Cell 31: 43&#8211;51. PMID: 6297752 <br />                                           47.   Drolet M, <span id='am-24' about='obo:NCBITaxon_4719' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4710 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_169748 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_169700 obo:NCBITaxon_40551 obo:NCBITaxon_3193'><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Phoenix L." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Phoenix" datatype="xsd:string"></span>Phoenix</span> P, Menzel R, Masse E, Liu LF, Crouch RJ (1995) Overexpression RNase H partial- <br />                                                 ly complements growth defect Escherichia coli &#916;topA mutant: R-loop formation major <br />                                                 problem absence DNA topoisomerase I. Proc Natl Acad Sci U S A 92: 3526&#8211;3530. PMID: <br />                                                 7536935 <br />                                           48.   Stupina VA, Wang JC (2005) Viability Escherichia coli topA mutants lacking DNA topoisomerase I. J <br />                                                 Biol Chem 280: 355&#8211;360. PMID: 15522872 <br />                                           49.   Guzman LM, Belin D, Carson MJ, Beckwith J (1995) Tight regulation, modulation, high-level ex- <br />                                                 pression vectors containing arabinose PBAD promoter. J Bacteriol 177: 4121&#8211;4130. PMID: <br />                                                 7608087 <br />                                           50.   Zhu CX, Qi HY, Tse-Dinh YC (1995) Mutation Cys662 Escherichia coli DNA topoisomerase I con- <br />                                                 fers temperature sensitivity change DNA cleavage selectivity. J Mol Biol 250: 609&#8211;616. PMID: <br />                                                 7623379 <br />                                           51.   Zhu Q, Pongpech P, DiGate RJ (2001) Type I topoisomerase activity required proper chromosom- <br />                                                 al segregation Escherichia coli. Proc Natl Acad Sci U S A 98: 9766&#8211;9771. PMID: 11493711 <br />                                           52.   Gellert M, O&apos;Dea MH, Itoh T, Tomizawa J (1976) Novobiocin coumermycin inhibit DNA supercoiling <br />                                                 catalyzed DNA gyrase. Proc Natl Acad Sci U S A 73: 4474&#8211;4478. PMID: 794878 <br />                                           53.   Drlica K, Snyder M (1978) Superhelical Escherichia coli DNA: relaxation coumermycin. J Mol Biol <br />                                                 120: 145&#8211;154. PMID: 347091 <br />                                           54.   Gormley NA, Orphanides G, Meyer A, Cullis PM, Maxwell A (1996) The interaction coumarin antibiot- <br />                                                 ics fragments DNA gyrase B protein. Biochemistry 35: 5083&#8211;5092. PMID: 8664301 <br />                                           55.   Hardy CD, Cozzarelli NR (2003) Alteration Escherichia coli topoisomerase IV novobiocin resis- <br />                                                 tance. Antimicrob Agents Chemother 47: 941&#8211;947. PMID: 12604525 <br />                                           56.   Shure M, Pulleyblank DE, Vinograd J (1977) The problems eukaryotic prokaryotic DNA packag- <br />                                                 ing vivo conformation posed superhelix density heterogeneity. Nucleic Acids Res 4: 1183&#8211; <br />                                                 1205. PMID: 197488 <br />                                           57.   Vetcher <span id='am-128' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-129'  ></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>, McEwen AE, Abujarour R, Hanke A, Levene SD (2010) Gel mobilities linking-number <br />                                                 topoisomers dependence DNA helical repeat elasticity. Biophys Chem 148: 104&#8211;111. <br />                                                 doi: 10.1016/j.bpc.2010.02.016 PMID: 20346570 <br />                                           58.   Richardson JP (1991) Preventing synthesis unused transcripts Rho factor. Cell 64: 1047&#8211; <br />                                                 1049. PMID: 2004415 <br />                                           59.   Drolet M (2006) Growth inhibition mediated excess negative supercoiling: interplay between <br />                                                 transcription elongation, R-loop formation DNA topology. Mol Microbiol 59: 723&#8211;730. PMID: <br />                                                 16420346 <br />                                           60.   Tse-Dinh YC, Beran RK (1988) Multiple promoters transcription Escherichia coli DNA topo- <br />                                                 isomerase I gene regulation DNA supercoiling. J Mol Biol 202: 735&#8211;742. PMID: 2845101 <br />                                           61.   Menzel R, Gellert M (1983) Regulation genes E. coli DNA gyrase: homeostatic control DNA <br />                                                 supercoiling. Cell 34: 105&#8211;113. PMID: 6309403 <br />                                           62.   Peter BJ, Arsuaga J, Breier AM, Khodursky AB, Brown PO, Cozzarelli NR (2004) Genomic transcrip- <br />                                                 tional response loss chromosomal supercoiling Escherichia coli. Genome Biol 5: R87. PMID: <br />                                                 15535863 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                        22 / 23 <br />                                                                                       A Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase I <br />  <br />  <br />  <br />                                           63.   del Solar G, Giraldo R, Ruiz-Echevarria MJ, <span id='am-123' about='obo:NCBITaxon_1118665' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_272259 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_7423 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126'  ></span><span id='am-127' property="rdfs:label" content="Espinosa" datatype="xsd:string"></span>Espinosa</span> M, Diaz-Orejas R (1998) Replication control <br />                                                 circular bacterial plasmids. Microbiol Mol Biol Rev 62: 434&#8211;464. PMID: 9618448 <br />                                           64.   Selzer G, Som T, Itoh T, Tomizawa J (1983) The origin replication <span id='am-14' about='obo:NCBITaxon_2470' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="rdfs:label" content="Plasmid P15a" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmid p15A</span> comparative <br />                                                 studies nucleotide sequences origin related plasmids. Cell 32: 119&#8211;129. PMID: <br />                                                 6186390 <br />                                           65.   Armstrong KA, Acosta R, Ledner E, Machida Y, Pancotto M, McCormick M, et al. (1984) A 37 X 10(3) <br />                                                 molecular weight plasmid-encoded protein required replication copy number control the <br />                                                 <span id='am-19' about='obo:NCBITaxon_2625' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="rdfs:label" content="Plasmid pSC101" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmid pSC101</span> temperature-sensitive derivative pHS1. J Mol Biol 175: 331&#8211;348. PMID: <br />                                                 6327996 <br />                                           66.   Churchward G, Linder P, Caro L (1983) The nucleotide sequence replication maintenance func- <br />                                                 tions encoded plasmid pSC101. Nucleic Acids Res 11: 5645&#8211;5659. PMID: 6310509 <br />                                           67.   Yigit H, Reznikoff WS (1998) Escherichia coli DNA topoisomerase I suppression killing Tn5 <br />                                                 transposase overproduction: topoisomerase I modulates Tn5 transposition. J Bacteriol 180: 5866&#8211; <br />                                                 5874. PMID: 9811643 <br />                                           68.   Yigit H, Reznikoff WS (1999) Escherichia coli DNA topoisomerase I copurifies Tn5 transposase, <br />                                                 Tn5 transposase inhibits topoisomerase I. J Bacteriol 181: 3185&#8211;3192. PMID: 10322021 <br />                                           69.   Ghosh S, Mallick B, Nagaraja V (2014) Direct regulation topoisomerase activity nucleoid-associ- <br />                                                 ated protein. Nucleic Acids Res 42: 11156&#8211;11165. doi: 10.1093/nar/gku804 PMID: 25200077 <br />                                           70.   Kirkegaard K, Wang JC (1985) Bacterial DNA topoisomerase I relax positively supercoiled DNA <br />                               <br /> </body></html>