<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genome-Scale Metabolic Network Validation of <br /> <span id='am-33' about='obo:NCBITaxon_70863' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis Venkateswaran et al. 1999" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:10319494" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Shewanella oneidensis" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:321336" datatype="xsd:string"></span>Shewanella oneidensis</span> Using Transposon Insertion <br /> Frequency Analysis <br /> Hong Yang1,2, Elias W. Krumholz1, Evan D. Brutinel2,3, Nagendra P. Palani1, Michael J. Sadowsky2,4, <br /> Andrew M. Odlyzko5, Jeffrey A. Gralnick2,3, Igor G. L. Libourel1,2* <br /> 1 Department Plant Biology, University Minnesota, St. Paul, Minnesota, United States America, 2 BioTechnology Institute, University Minnesota, St. Paul, <br /> Minnesota, United States America, 3 Department Microbiology, University Minnesota, Minneapolis, Minnesota, United States America, 4 Department Soil, <br /> Water, Climate, University Minnesota, St. Paul, Minnesota, United States America, 5 School Mathematics, University Minnesota, Minneapolis, Minnesota, <br /> United States America <br />  <br />  <br />  <br />      Abstract <br />      Transposon mutagenesis, combination parallel sequencing, powerful tool en-masse mutant <br />      analysis. A probability generating function used explain observed miniHimar transposon insertion patterns, gene <br />      essentiality calls transposon insertion frequency analysis (TIFA). TIFA incorporated observed genome and <br />      sequence motif <span id='am-6' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span> miniHimar transposon. The gene essentiality calls compared to: 1) previous genome-wide <br />      direct gene-essentiality assignments; and, 2) flux balance analysis (FBA) predictions existing genome-scale <br />      metabolic model <span id='am-16' about='obo:NCBITaxon_211586' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_70863 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis ATCC 700550" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:60479" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Shewanella sp. MR-1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Shewanella oneidensis MR-1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis str. MR-1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis strain MR-1" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Shewanella oneidensis MR-1</span>. A three-way comparison FBA, TIFA, direct essentiality calls <br />      validate TIFA approach. The refinement interpretation observed transposon insertions <br />      demonstrated genes insertions necessarily essential, genes contain insertions not <br />      nonessential. The TIFA calls reasonable agreement direct essentiality calls S. oneidensis, agreed <br />      closely E. coli essentiality calls orthologs. The TIFA gene essentiality calls good agreement the <br />      MR-1 FBA essentiality predictions, agreement TIFA FBA predictions substantially better than <br />      FBA direct gene essentiality predictions. <br />  <br />   Citation: Yang H, Krumholz EW, Brutinel ED, Palani NP, Sadowsky MJ, et al. (2014) Genome-Scale Metabolic Network Validation <span id='am-114' about='obo:NCBITaxon_22' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Shewanella" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Shewanella MacDonell and Colwell 1986" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Shewanella</span> oneidensis Using <br />   Transposon Insertion Frequency Analysis. PLoS Comput Biol 10(9): e1003848. doi:10.1371/journal.pcbi.1003848 <br />   Editor: Costas D. Maranas, The Pennsylvania State University, United States America <br />   Received April 4, 2014; Accepted August 7, 2014; Published September 18, 2014 <br />   Copyright: &#223; 2014 Yang et al. <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   <span id='am-132' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-133' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Data</span> Availability: The authors confirm data underlying findings fully available restriction. All relevant data paper its <br />   Supporting Information files. <br />   Funding: This work funded Office Naval Research, awards N141310552 N141210309 National Science Foundation award NSF/MCB- <br />   1042335. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * Email: libourel@umn.edu <br />  <br />  <br />  <br /> Introduction                                                                            application Tn-seq infectious agents <span id='am-25' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> identify <br />                                                                                         essential genes serve targets therapy [8&#8211;12]. <br />    Transposon mutant analysis extensively used                              However, fitness disruption coding sequences the <br /> generate genome-wide mutant libraries define gene                                type data obtained method. When <br /> essentiality [1,2]. With introduction parallel sequencing,                       transposon mutant libraries generated genome-saturating <br /> transposon-based methods developed phenotype infor-                           conditions, essentiality related disruption non-coding <br /> mation gathering tools, forward genetic screens                        regions identified [13,14], facilitating identification non- <br /> the aim isolate individual mutant strains. Tn-seq, closely                       coding regulatory elements. In S. oneidensis, himar Tn5 <br /> related methods Bar-seq DNA shearing [3], investigate                        <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span>transposons</span> used identify number mutants and <br /> mutant fitness genomic scale counting abundance                        elucidate cellular physiology [15&#8211;17]. Barcoded genome-wide <br /> mutant-specific DNA sequence short compet-                           mutants S. oneidensis created himar <br /> itive growth period [4,5]. In Bar-seq unique piece DNA                        transposon individual fitness evaluated large number <br /> located known flanking sequences sequenced                           growth conditions using microarrays [18]. The creation of <br /> directly [6]. In Tn-seq, type IIS restriction enzyme cuts                        tagged transposon mutant libraries enabled systems-level <br /> outside recognition sequence used extract transposons                    analyses S. oneidensis, mass-spectrometry based <br /> the mutant genomes, including flanking sequence (17 bp                            metabolite profiling mutants [19] computational inference <br /> miniHimar) used map location transposon                           gene regulatory networks based fitness data [20]. <br /> insertion.                                                                                 Transposon mutagenesis-based gene essentiality measurements <br />    Transposon insertion sequencing used identify                            exceptionally informative validation genome-wide <br /> essential genes increasing number microorganisms                        modeling techniques. The genome-wide scale low-cost nature <br /> wide range ecological niches [7]. Of particular significance                  disposable single gene knockout libraries provide powerful <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                   1                       September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                                  Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />   Author Summary                                                                 recognized others, recent estimations gene <br />                                                                                  essentiality using hidden Markov models (HMM) [26,27]. TIFA <br />   Metabolic modeling techniques play central role                           distinguishes itself HMM approach considering <br />   rational design industrial strains, personalized medicine,                  insertional biases sequence preference genomic location <br />   automated network reconstruction. However,                          insertion. The insertional biases miniHimar <br />   large size models, comprehen-                        transposon S. oneidensis investigated using existing <br />   sively tested using single gene knockout mutants                     dataset [28]. TIFA determines likelihood number of <br />   gene model. Such genetic test evaluate                          experimentally observed insertions gene utilizes a <br />   genes given condition predicted                      probability generating function accounts observed <br />   essential model, essential reality (and                  insertional biases miniHimar transposon. The thus <br />   vice versa). We developed new probability-based                              determined gene essentiality growth Shewanella Basal <br />   technology identifies essentiality genes                      Medium (SBM) aerobic conditions used investigate <br />   observed transposon insertion data. This data was <br />                                                                                  stoichiometric thermodynamic constraints existing <br />   acquired pooling tens thousands transposon <br />                                                                                  MR-1 metabolic model [29], simulating aerobic growth SBM. <br />   mutants, localizing insertion locations once <br />   using massive parallel sequencing. We utilized gene <br />   essentiality data genome-scale genetic validation                   Results <br />   metabolic model. For instance: work identified <br />   nonessential genes predicted essential                     miniHimar transposon insertions frequencies location <br />   growth existing metabolic model Shewanella                            sequence biased <br />   oneidensis, highlighting incomplete areas                             The miniHimar transposon strong preference insert <br />   metabolic model.                                                               inside TA sequence [30]. The flanking sequences the <br />                                                                                  chromosomal transposon insertion library confirmed prefer- <br />                                                                                  ence, just 95% mutations located inside TA <br /> datasets evaluate performance genome-scale network                     sequence. We discarded remaining ,5% insertions and <br /> reconstructions [21], enrich genomic information                   investigated TA inserted transposons (Fig. 1). A <br /> currently used automated network reconstructions [22&#8211;24].                    potential bias chromosomal insertion position investi- <br />    The presented transposon insertion frequency analysis (TIFA)                  gated plotting insertion frequency function of <br /> improves direct interpretation presence                  chromosomal location (Fig. 1A). Because exact insertion <br /> an insertion gene core interpreted sufficient evidence           occurred independent colonies, number of <br /> nonessentiality, absence insertions interpreted                     insertions estimated number TA locations had <br /> essentiality [18,25]. The need sophisticated approach                 inserted (Material Methods). The locational bias was <br />  <br />  <br />  <br />  <br /> Figure 1. Evaluation transposon insertion bias. (A) Scatter plot genome location dependent insertion probability. Each data point <br /> represents genomic window 20,000 nucleotides, shifted 2,000 nucleotides consecutive evaluation. The insertion <br /> probability gradually decreased origin replication. The solid line best fit ax2+b|x|+c, a, b c equal 0.0032, 0.0081 and <br /> 0.1615, respectively, x location genome. (B) Boxplot flanking sequence dependent insertion probability distribution showing that <br /> several motifs contained significantly insertions others. (C) Relative occurrence nucleotides flanking positions inserted TA <br /> sites. <br /> doi:10.1371/journal.pcbi.1003848.g001 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                            2                     September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                            Transposon Insertion Frequency Analysis <br />  <br />  <br /> quantified fitting absolute value second order polynomial             probabilities weighted genome locational specific <br /> through observed frequency data (Fig. 1A). The insertion                bias. Using probability model, number expected <br /> frequency showed approximate 25% symmetrical bias                insertions calculated gene compared the <br /> the origin replication compared midpoint                   observed number insertions (Fig. 2). Genes called essential <br /> chromosome. Presumably, bias result presence             combined probability finding as, fewer insertions <br /> of multiple partial copies actively replicating circular              observed, number genes the <br /> chromosome [31] near replication fork, leading higher              dataset i.e. accepted false positive essential gene <br /> physical copy number genes closer origin replication.          selection. To establish exact cutoff value, marginal <br />    The effect flanking nucleotides         probabilities nonessential gene essential by <br /> target dinucleotide investigated using genes             chance given cutoff value summed, cutoff value <br /> contained fewer insertions (p.0.1) expected                adjusted marginal probabilities summed exactly <br /> binomial distribution prediction (Fig. 1B). The dataset                 one. Monte Carlo sampling used confirm result, and <br /> subdivided 136 sections, section corresponding               cutoff value retrieved (Text S1). The transposon <br /> unique combination flanking nucleotides prior            insertion distribution visualized histogram the <br /> two nucleotides following TA location. The complementary              difference expected observed number of <br /> strands genome result possible sequence orientations,         insertions, scaled standard deviation calculated the <br /> with sequence plus strand matching complementary              probability generating function. Monte Carlo sampling generated <br /> sequence minus strand. Sixteen sequences palindromic,            similar distribution (Fig. 2). The right hand this <br /> resulting total (256-16)/2+16 = 136 unique sequences. For           distribution particular interest, elongated tail the <br /> each 136 unique sequence combinations, occurrences               observations suggest genes significantly insertions <br /> and insertion events determined before. The insertion               expected. Only genes observed (SO3264, <br /> probability sequence dependent, 3% probabilities            SO4100, SO4785), indicating TIFA formed good descrip- <br /> differing significantly mean probability (Fig. 1B).                tion observed transposon insertion behavior. The poorer fit <br />    To determine flanking nucleotides affected insertion          left hand distribution result of <br /> probability independently, assumed number                    mutants lower wild type fitness. Clones reduced <br /> insertions flanking sequence product the <br />                                                                             fitness likely escape detection lower <br /> independent contributions nucleotides. Thus, the <br />                                                                4 <br />                                                                             absolute abundance, yielding fewer observed insertions nones- <br /> 136 determined insertion frequencies given pseq&#240; j&#222; ~ P pij ,        sential genes reduced fitness deletion. Some genes <br />                                                               i~1 <br />                                                                             slow growing mutant phenotypes called <br /> where pseq&#240; j&#222; observed insertion probability given <br />                                                                             essential result. For 50 273 identified essential genes a <br /> flanking nucleotide combination j, pij contributions         fitness value larger zero observed. To investigate if <br /> the individual nucleotides combination. Using weights           mutants insertions genes called essential grew <br /> were inversely proportional variance sequence <br />                                                                             slowly clones, nonessentiality probability was <br /> insertion probability pseq&#240; j&#222; , nonlinear x2-fitting used to <br />                                                                             plotted fitness (Fig. 3A). Although average fitness of <br /> calculate parameter pij. Assuming multinomial variance,             clones insertions essential genes lower fitness of <br /> observed probabilities differed significantly linear model (x2       clones insertions nonessential genes, spread very <br /> test, p,0.01), indicating contribution flanking             large, indicating slow growth insufficient explanation <br /> nucleotides insertion probability independent. <br />                                                                             detected insertions essential genes. The insertion locations <br /> Fig. 1C shows linear approximation nucleotide <br />                                                                             plotted positions conserved domains and, <br /> contributions, linear approximation indicated <br />                                                                             investigate potential reinitiation transcription and <br /> that TATATG sequence highest insertion probability, <br />                                                                             translation downstream transposon, estimates strength <br /> the independent contributions underestimated insertion <br /> probability sequence 30%. In addition, were <br /> three sequences higher experimental insertion <br /> probability experimental value TATATG was <br /> found. Visual inspection experimental sequence probabil- <br /> ities suggested preferred consensus sequence was <br /> TATAxA. Just TA enrichment sufficient, exempli- <br /> fied barely average insertion probability associated with <br /> AATATT. The highest experimental insertion probability was <br /> associated TATATA (p = 0.53), smallest probability <br /> was associated GTTAAC (p = 0.057), indicating approx- <br /> imate tenfold spread insertion preference. <br />  <br /> Essential gene calls <br />    Transposon mutagenesis results random disruption of <br /> genes, reducing eliminating gene function. In principle, <br /> transposon mutagenesis reports gene essentiality. The <br /> essentiality gene investigated comparing number <br /> of observed mutations gene number expected                   Figure 2. Observed Monte Carlo simulated transposon <br />                                                                             insertions. The normalized deviation expectation insertions are <br /> insertions. A transposon probability model formulated              shown TIFA essential (red), nonessential (green) unknown (blue) <br /> accounted observed sequence location specific biases.           genes. The solid line represents average outcome 1000 Monte <br /> For gene, expected number insertions estimated              Carlo simulations, flanked standard deviation (dotted lines). <br /> using probability generating function sequence specific          doi:10.1371/journal.pcbi.1003848.g002 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                       3                    September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                               Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />  <br /> Figure 3. (A) 50 273 essential genes associated fitness value. The circumference circles represents logarithmic <br /> read counts insertion, color circles represents gene location insertion: 10% (magenta), 10% (cyan), and <br /> middle 80% (black). 22 essential genes contained $2 insertions middle. The spread fitness read counts large, suggesting <br /> different causes existence insertions essential genes. (B) Transposon insertion locations, fitnesses, projected ribosome binding site <br /> (RBS) strengths associated intra-gene start codons. Black dots TA locations, row diamonds observed insertions with <br /> color coded fitness. The diamonds represent alternative start codons gene. The color scales logarithmically associated RBS <br /> strengths. Each couple rows represents RBS strengths intra-gene start codons possible insertion orientations mutant, <br /> with row representing insertion. <br /> doi:10.1371/journal.pcbi.1003848.g003 <br />  <br />  <br /> of ribosomal binding sites associated alternative start codons           genes significantly fewer expected for <br /> inside genes (Figure S1).                                                 nonessential genes, significantly expected <br />                                                                               essential genes. For example, closer inspection SO3872 <br /> Nonessential gene calls                                                       revealed insertions concentrated second quartile <br />    Following identification essential genes nonessential            showed reduced fitness (Fig. 3B). Two additional <br /> gene insertion model, nonessential genes identified             insertions half gene showed high fitness, no <br /> an essential gene insertion model. The insertion frequency                 insertions present. There good sequence <br /> essential genes approximated multiplying TA location-              support gene assignment based sequence alignment <br /> specific insertion probabilities ratio observed               arylsulfate sulfo-transferase pfam PF05935 [32]. Con- <br /> insertions essential genes expected number insertions            ceivably, beginning second half gene were <br /> for nonessential genes. The expected number insertions            essential production functional protein, start <br /> gene calculated using essential gene model, gene            codons midpoint gene associated ribosomal <br /> that contained significantly insertions expected,               binding sites moderate projected strength [33], suggested that <br /> called nonessential. Following earlier logic, cutoff value          translation reinitiated gene resulting separate <br /> used allowed single false-positive nonessential gene               expression second half gene (Fig. 3B). More generally, <br /> identification. This method identified 2,216 genes nonessential.           essential genes nonessential regions a <br /> No essentiality 1,722 genes, genes           regulatory site highly inserted, adjacent to <br /> (SO2148, SO3175 SO3872) identified essential                 uninserted regions necessary essential gene role, which <br /> and nonessential. Hence, number insertions              cause dual essential-nonessential identification. Gene <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         4                     September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                                                                                                                                                  Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         The essentiality classes (columns) previously defined [18] direct essentiality calls (DECs), cross compared TIFA essentiality classes (rows). Columns (1,2) represent DEC essential gene calls; <br /> essentiality calls performed previous work bases <br /> of absence presence insertions [18] compared TIFA <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            273 (120) <br /> calls (Table 1 Dataset S1). Seventy essential <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Total <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                        2216 <br />                                                                                                                                                                                                                       1605 <br />                                                                                                                                                                                                                                 4214 <br /> gene calls 1,958 nonessential gene calls agreement. <br /> Eighty genes previously identified nonessential <br /> (including orthologs essential genes E. coli) were <br /> identified essential TIFA, 36 contained no <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Unknown <br /> insertions dataset. Fifty genes previously <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            114 (10) <br /> identified essential identified nonessential TIFA, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                        207 <br />                                                                                                                                                                                                                       501 <br />                                                                                                                                                                                                                                 832 <br /> and contained average 10 insertions gene. <br />  <br /> Essential genes inserted frequently very <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Dispensable unclear E.coli <br /> end gene <br />    The number TA sequences number detected <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         columns (3,4,5) DEC nonessential gene calls. The excluded TIFA essential gene calls explained alternatively operon polar effects listed brackets. <br /> insertions tallied gene percentile investigate the <br /> positions insertions inside (essential) genes biased (Fig. 4). <br /> No percentile showed difference relative insertion frequency <br /> prior identification essential genes (Fig. 4A). Analysis just <br /> the nonessential essential genes revealed substantial increase <br /> of insertions 2&#8211;3% essential genes. Essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            0 (0) <br /> were twice likely contain insertions 2% <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                        57 <br />                                                                                                                                                                                                                       31 <br />                                                                                                                                                                                                                                 88 <br />                                                                                                                                                     5 <br /> nucleotides, insertion frequencies half the <br /> frequencies observed nonessential genes (Fig. 4B). Nonetheless, <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Surprise dispensable <br /> the increased insertion frequency end essential genes <br /> provides quantitative support omission insertion data <br /> from 2% genes. Previously, larger areas genes <br /> were excluded insertion analysis, arguing insertions in <br /> the distal parts genes effective eliminating gene <br /> function [4]. No relative increase insertion frequency was <br />  <br />                                                                                                                                                                                            2 (0) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                 14 <br /> observed beginning genes. <br />  <br />                                                                                                                                                                                                        6 <br />                                                                                                                                                                                                                       6 <br />                                                                                                                                                     4 <br />  <br />  <br />  <br />  <br /> Validation genome scale MR-1 model essentiality <br /> predictions <br />    The ability identify essential nonessential genes with <br /> TIFA utilized validate essentiality predictions an <br />                                                                                                                                                     Dispensable <br />  <br />  <br />  <br />  <br /> existing genome-scale metabolic model S. oneidensis MR-1 <br /> that manually curated previously [29]. The MR-1 <br />                                                                              Table 1. Comparison gene essentiality calls TIFA DEC. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            79 (23) <br />  <br />  <br />  <br />  <br /> model includes 774 reactions, 783 gene 634 unique <br />                                                                                                                                                                                                        1895 <br />  <br />  <br />                                                                                                                                                                                                                                 2878 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                          essential gene calls conflicting absent corroborating evidence orthologs, <br />                                                                                                                                                                                                                       881 <br />  <br />  <br />  <br />  <br /> metabolites. Flux balance analysis (FBA) used infer gene <br />  <br />                                                                                                                                                                                                                                         A detailed description DEC categories given Text S1 reference [18]. <br />                                                                                                                                                     3 <br />  <br />  <br />  <br />  <br /> essentiality genes present MR-1. The <br />                                                                                                                                                                                                                                          nonessential gene calls corroborating evidence E. coli orthologs. <br /> transposon mutants selected aerobic conditions SB <br />                                                                                                                                                     New essential <br />  <br />  <br />  <br />  <br /> media, FBA predicted 209 essential genes 574 <br /> nonessential genes MR-1 model conditions. <br /> The annotations thirteen genes MR-1 model were <br />                                                                                                                                                                                                                                          nonessential gene calls conflicting evidence orthologs. <br />  <br />  <br />  <br />  <br /> obsolete, reduced comparison FBA TIFA <br />                                                                                                                                                                                            12 (6) <br />                                                                                                                                                                                                        17 <br />                                                                                                                                                                                                                       31 <br />                                                                                                                                                                                                                                 66 <br />  <br />  <br />  <br />  <br /> predictions 770 genes. TIFA able determine essentiality <br />                                                                                                                                                     2 <br />  <br />  <br />  <br />  <br /> for 481 genes (62%) set (Table 2) providing compre- <br />                                                                                                                                                     Expected essential <br />  <br />  <br />  <br />  <br /> hensive evaluation network essentiality predictions. <br />    Of 273 identified essential genes identified TIFA, 75 were <br /> present MR-1 model. Fifty seven included 75 genes <br /> were correctly predicted essential, 374 406 FBA- <br /> nonessential genes present MR-1 model were <br />                                                                                                                                                                                            66 (81) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         doi:10.1371/journal.pcbi.1003848.t001 <br />  <br />  <br />  <br />  <br /> identified nonessential TIFA (Fig. 5). FBA essentiality <br />                                                                                                                                                                                                                       155 <br />                                                                                                                                                                                                                                 336 <br />                                                                                                                                                                                                        34 <br />                                                                                                                                                     1 <br />  <br />  <br />  <br />  <br /> predictions insensitive 1% biomass production cutoff, <br /> with 1% growth resulting knockout predictions no <br /> growth (Text S1). The TIFA essentiality calls fairly sensitive <br />                                                                                                                                                                                                                                          nonessential orthologs; <br />  <br />  <br />  <br />  <br /> to essentiality cutoff. If instance lower 2.5% likelihood <br />                                                                                                                                                                                                                                          Essential orthologs; <br />  <br />  <br />  <br />  <br /> of gene nonessentiality used essential gene cutoff <br /> instead (Fig. 5a), twice genes (623) been <br />                                                                                                                                                                                                        Nonessential <br />  <br />  <br />  <br />  <br /> identified essential suggesting half essential <br />                                                                                                                                                                                                                       Unknown <br />                                                                                                                                                                                            Essential <br />  <br />  <br />  <br />  <br /> genes called. However, number false positive would <br />                                                                                                                                                                                     TIFA <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                 Total <br />                                                                                                                                                     DEC <br />  <br />  <br />  <br />  <br /> have larger (54 genes, calculated marginal <br />                                                                                                                                                                                                                                         1 <br />                                                                                                                                                                                                                                         2 <br />                                                                                                                                                                                                                                         3 <br />                                                                                                                                                                                                                                         4 <br />                                                                                                                                                                                                                                         5 <br />  <br />  <br />  <br />  <br /> probabilities), agreement FBA TIFA <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                        5                                                                                                            September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                                        Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />  <br /> Figure 4. Insertion frequency genes. Genes evaluated 0.2% gene increments. No insertion preference observed the <br /> complete gene population (A). The analysis performed (1% increments) essential (red) nonessential (green) gene identification <br /> (B). The ,2% 39 end essential genes inserted frequently. <br /> doi:10.1371/journal.pcbi.1003848.g004 <br />  <br /> prediction substantially better using stringent                   TIFA essential genes associated blocked reactions <br /> cutoff (Fig. 5a).                                                                     (Text S1). These reactions evidently required for <br />                                                                                       biomass formation model, indicating biomass <br /> Genes falsely predicted nonessential                                                  equation sufficiently inclusive test essential genes, or <br />    False nonessential FBA predictions caused                           reality unused reactions provided alternative route to <br /> array pleiotropic effects, built-up toxic                         essential biomass. In addition, reactions were <br /> metabolite resulting removal downstream reaction                        blocked, TIFA suggested MR-1 model requires <br /> in pathway. More directly, false nonessential predictions                    modifications unblock TIFA essential reactions. In the <br /> caused combination incorrect gene-reaction association,                       case aconitase (E.C. 4.2.1.3), discrepancy MR- <br /> lax thermodynamic constraints, over-inclusion metabolic                        1 model TIFA essentiality data highlighted need a <br /> reactions. Such overly inclusive metabolic network                      comprehensive gene expression protein activity regulation <br /> arise result absence of, incomplete gene                            simulation metabolic models. In MR-1 model, acnB <br /> regulatory layer. Blocked reactions, carry flux                          (SO0432) acnD (SO0343) independently assigned the <br /> under circumstances, point easily interpretable                       aconitase reaction (OR relationship), resulting nonessential <br /> shortcomings network. Eleven 18 FBA-nonessential,                       prediction acnB. However, acnB deletion strain was <br />  <br />  <br />  Table 2. Comparison TIFA gene essentiality FBA predictions. <br />  <br />  <br />                                                            TIFA essential                                      TIFA nonessential <br />  <br />                                                            FBA-E             FBA-N             % True          FBA-N             FBA-E             % True <br />  <br />  Original MR-1 model                                       57                18                76              374               32                92 <br />  All metabolites leave model                           56                19                75              374               32                92 <br />  No thermodynamic constraints                              33                42                44              384               22                95 <br />  Metabolites leave, thermodynamic constrains        33                42                44              384               22                95 <br />  <br />  TIFA essentiality calls compared FBA predictions MR-1 model: using original model (row 1); removal stoichiometric constraints endpoint <br />  metabolites (row 2); removal thermodynamic constraints (row 3); following removal stoichiometric thermodynamic constraints (row 4). <br />  FBA-E: FBA essential gene predictions; FBA-N: FBA nonessential gene predictions. <br />  doi:10.1371/journal.pcbi.1003848.t002 <br />  <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                 6                     September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                               Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />  <br /> Figure 5. Comparison gene essentiality TIFA calls FBA predictions. Genes grouped 40 bins based their <br /> cumulative probability values. The blue bars represent genes present MR-1 model. The red green bars represent FBA essential <br /> and FBA nonessential genes. Genes lower cumulative probability value likely essential, genes higher cumulative <br /> probability value likely nonessential. The leftmost bar (A) shows number genes cumulative probability value less <br /> than cut-off value (TIFA essential, allowing 1 false positive). The rightmost bar (B) shows number genes cumulative probability <br /> of nonessential cut-off value (TIFA nonessential, allowing 1 false positive). <br /> doi:10.1371/journal.pcbi.1003848.g005 <br />  <br /> unable grow presence oxygen [28],                       central metabolism S. oneidensis MR-1 [36] was <br /> consistent transposon insertion data.                                     formulated elementary mode analysis, larger scale <br />                                                                                MR-1 model improved gene essentiality predictions substantially. <br /> Genes falsely predicted essential                                              Only gene (SO3547) previously genes falsely <br />    Genes falsely predicted essential FBA                 predicted essential (SO0424, SO0323, SO0538, SO1926, <br /> interpreted resulting missing reaction network,                 SO2629, SO3547, SO0274, SO3517) predicted <br /> insufficiently permissive thermodynamic constraints, errors                 incorrectly, demonstrating previously observed enhanced <br /> gene-to-reaction relationships, overly inclusive (essential)                predictive capabilities complete networks [37] <br /> biomass equation, incorrect regulatory layer. However, <br /> pleiotropic effect explain discrepancies. Of 2216             TIFA essentiality FBA predictions relative close <br /> TIFA identified nonessential genes, 374 genes correctly                   agreement <br /> predicted nonessential 32 incorrectly                         A direct comparison TIFA previously made <br /> predicted essential FBA (Fig. 5). Of incorrectly essential              direct essentiality calls (DECs) based presence/ <br /> predictions, SO1498 SO3745 correctly                       absence insertions 80% core sequence genes [18] is <br /> predicted nonessential biomass components lipopoly-                 shown Table 1. Note FBA gene essentiality predictions for <br /> saccharide (LPS) glycogen included biomass                 DEC dataset computed LB medium instead of <br /> equation. LPS required growth [34]. The                  SBM. Due richer composition LB, 18 genes fewer were <br /> transporter <span id='am-120' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-121' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ammonia</span>, amtB (SO0760)                         FBA essential, explained presence <br /> essential low ammonia concentrations [35]. A detailed                  tryptophan pyrimidine LB (Dataset S1). <br /> comparison TIFA FBA essentiality shown Text                     One expect FBA prediction agree the <br /> S1.                                                                            TIFA essentiality calls original MR-1 essentiality <br />                                                                                calls, 1) TIFA identifications improvement the <br /> Network constraints scale                                                  original direct method, 2) FBA predicts gene essentiality <br />    The observed discrepancies TIFA model                       correctly not. A three-way comparison between <br /> predictions investigated by: 1) allowing metabolites           TIFA essentiality calls, DECs, FBA predictions used <br /> freely leave network, 2) removing thermodynamic                 investigate relative agreement FBA predictions <br /> constrains model. Lifting stoichiometric constraints            essential gene identification methods. A gene-by-gene <br /> endpoint metabolites resulted marginal deterioration                   comparison included supplementary data (Dataset S1). <br /> essential gene predictions, change nonessential gene                 Because TIFA predictions alternatively explained <br /> predictions (Table 2), suggesting blockage reactions            polar operon effects removed, number of <br /> stoichiometric constraints endpoint metabolites                     comparisons TIFA FBA substantially fewer <br /> important FBA gene essentiality predictions MR-1                    DEC essentiality calls FBA (Fig. 6). The <br /> model. Removal thermodynamic constraints resulted                  overall performance TIFA, expressed percent correct <br /> substantial deterioration essential gene predictions             predictions (combined true essential true nonessential <br /> marginal improvement nonessential gene predictions (Table 2),               predictions divided essentiality predictions), much <br /> confirming importance correct thermodynamic constraints                 higher DEC: 90% vs 79% (Fig. 6), indicating TIFA <br /> in gene essentiality predictions. In comparison model                  calls better. To investigate potential influence of <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          7                    September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                             Transposon Insertion Frequency Analysis <br />  <br />  <br />  <br />  <br /> Figure 6. Three-way comparison FBA predictions TIFA DEC essentiality calls. (A) Comparison FBA TIFA. <br /> (B) Comparison FBA direct gene essentiality calls (DEC). The sectors comparison matrix intersections FBA essential <br /> genes (E), FBA nonessential genes (N) genes MR-1 model (NA) TIFA DEC calls essential genes (E), nonessential genes (N) <br /> and uncalled genes (U). The TIFA essential gene calls alternatively explained polar operon effects shown brackets. <br /> doi:10.1371/journal.pcbi.1003848.g006 <br />  <br /> polar effects essential gene predictions, TIFA calls          essentiality 1725 (41%) genes called. The <br /> had previously removed dataset                    observed transposon insertion pattern close agreement with <br /> essentiality explained alternatively polar effects,          Monte Carlo insertion simulations utilized location specific <br /> compared essential gene TIFA calls. Note polar effects            insertion probabilities. The absence insertional &#8216;&#8216;hotspots&#8217;&#8217; in <br /> are problem essential, nonessential gene               comparison Monte Carlo simulations interpreted as <br /> calls. The correct prediction percentage discarded essential           validation essential gene assignments. The data used for <br /> gene predictions 69% (Fig. 6, 43/(43+19)),                 study generated transposon transcrip- <br /> slightly lower 76% (Fig. 6, 57/(57+18)) retained             tionally terminated. As consequence, essentiality be <br /> TIFA comparisons. This suggested polar effects                   determined subset genes identified as <br /> result false essential gene assignments assignments          essential TIFA (,70%, Table 1, 273/(273+120)). A substantial <br /> had used. This surprising given polar effect          number genes (120) contained sufficiently insertions be <br /> to occur, downstream neighboring gene operon                identified essential (Table 1), lack viability could <br /> been identified essential.                                                 alternatively explained presence adjacent <br />    Genes exclusively identified essential DEC            downstream essential gene operon. Note that <br /> wrong (40% correct essential gene                         experimental operon predictions RNA-seq data [38] could <br /> predictions, (Dataset S1). Genes exclusively identified essential          improve used computational operon projections. Alter- <br /> using TIFA, correct prediction percentage 68%,                 natively, utilization unterminated read-through transposon <br /> was slightly lower 76% TIFA essential gene              eliminate polar gene essentiality experimentally. The TIFA <br /> predictions (Dataset S1). Genes exclusively identified nonessen-           essential gene predictions agreed fairly direct essentiality <br /> tial TIFA equally good agreement FBA                          calls MR-1[18], closer agreement essentiality <br /> predictions entire TIFA nonessential gene calls (92%                   expectations E. coli orthologs (Dataset S1) FBA <br /> compared 91% agreement). DEC nonessential gene assign-                     predictions MR-1 model. In addition, genes that <br /> ments 87% agreement FBA predictions,                         previously identified essential based absence of <br /> exclusive DEC nonessential gene assignments 74%                  insertions contained insertions dataset. TIFA <br /> agreement FBA predictions. In summary, TIFA gene                     able provide transposon insertion-based essentiality calls for <br /> essentiality calls (both essential nonessential gene calls)       481 770 (62%) non-obsolete genes MR-1 model, and <br /> much closer agreement FBA model predictions                 able perform comprehensive validation the <br /> DEC essentiality calls, providing strong support TIFA                MR1 model. For example: 32 genes incorrectly predicted <br /> better method gene essentiality calls (Dataset S1).                       essential FBA, suggested current MR-1 model was <br />                                                                               incomplete. And, 11 TIFA essential genes associated with <br /> Discussion                                                                    blocked reactions suggested essential reactions the <br />                                                                               MR-1 model used, indicating current <br />    Tn-seq powerful readily available technology              MR-1 network incomplete. In addition, FIFA data demon- <br /> genome-wide evaluation metabolic networks. The observed                    strated thermodynamic constraints reaction direction- <br /> transposon insertion pattern suggested 28.6% essential                alities greatly improved FBA gene essentiality predictions. <br /> genes occasionally inserted, insertions essential <br /> genes limited periphery genes previously            Materials Methods <br /> presumed [18]. Conversely, genes did contain <br /> insertions identified essential. Both TIFA,               Transposon insertion data <br /> previously developed HMM models, able identify essential                   A detailed description transposon experiment that <br /> genes containing insertions, unlike current HMM models,               generated data used work published previously <br /> TIFA explicitly corrects observed transposon insertional              [28]. Briefly, single-mutant library S. oneidensis generated <br /> biases. Note confidence associated essentiality         miniHimar transposon kanamycin selection on <br /> was depended number TA locations gene. For                 Schewanella Basal Medium (SBM, well-defined rich <br /> genes contained handful TA sites, essentiality           medium), plates aerobic conditions [39]. The fitnesses the <br /> not established, insertions found. Hence,               pooled clones evaluated aerobic conditions using SBM <br /> establish essentiality short GC rich genes, large           previously described [4]. Samples pooled library were <br /> mutant library required. Conversely, genes TA                    collected short growth period. The raw <br /> locations called essential, contained                 sequence data sample mapped genome, was <br /> significant number insertions. With current library size,              filtered retain sequences occurred times, <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         8                    September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                            Transposon Insertion Frequency Analysis <br />  <br />  <br /> and mapped uniquely. For TIFA, data             resulted ,1% biomass production relative wild type [40] <br /> samples combined assess viability insertional                  Using zero biomass production alternative cutoff [41,42] <br /> mutants condition used generate                    resulted identical predictions, suggesting prediction were <br /> library. The fitness values used manuscript calculated            insensitive cutoff value (Text S1 details), was <br /> from aerobic treatment data only. Gene fitness calculated             consistent previous observations E. coli networks [40,42]. <br /> as median fitness clones inserted          Computationally zero growth assessed biomass produc- <br /> gene (Text S1).                                                               tion ,1e-6 eliminate influence computational noise. <br />  <br /> Estimation miniHimar insertion probabilities                               Databases software <br />    The probability insertion occurring r times given                  The S. oneidensis MR-1 metabolic model previously <br />                                                                 lr            reconstructed original genome annotation [29], which <br /> location approximated Poisson distribution: p~e{l ,                used study comparison (NCBI, NC_004347.1). <br />                                                                 r! <br /> which reduces p~e{l r = 0, l insertion                    Genes removed later annotations (NC 004347.2) were <br /> probability m/n m number colonies, n number               used. In addition, TA loci areas genes <br /> of TA sites genome. This yielded probability finding            overlapped excluded dataset. The essentiality of <br />                                                          m=                   4,214 genes S. oneidensis genome investigated TIFA. <br /> at insertion given location: l~1{e n , the <br />                                                    m                          The S. oneidensis essentiality fitness predictions were <br /> total number observed insertions insobs ~n 1{e n . The <br />                                                               = <br />  <br />                                                                               performed MATLAB (MathWorks, Natick MA), using the <br /> number colonies library estimated substituting           COBRA toolbox [43] combination linear optimization <br /> number unique sequences library insobs , number              routine (simplex algorithm) CPLEX software suite (IBM, <br /> TA sites (n) genome, mean insertion probability p.             Armonk NY). Operon calls S. oneidensis taken from <br /> The approach followed determining sequence-                  ProOpDB [44] terminal genes operons used <br /> specific insertion probabilities, limiting analysis fraction       fitness analysis. Essentiality calls terminal <br /> of TA locations given flanking motif (Text S1).                        genes. In addition, upstream genes operon used they <br />                                                                               evaluated nonessential using TIFA, directly <br /> Identification essential genes                                             downstream gene called nonessential TIFA. TIFA, genome <br />                                                                               Monte Carlo analyses performed custom MATLAB <br />    Assuming equal insertion probability TA site, the <br />                                                                               <span id='am-137' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-138' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141'  ></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Python</span> (http://www.python.org/) scripts. Unless otherwise <br /> probability observing number experimentally <br />                                                                               indicated, significance evaluated p,0.05. <br /> inserted locations gene given cumulative probability <br /> of binomial    distribution, b(s, pe) equates to:                   Supporting Information <br />              Pt <br />                    S <br /> Pr &#240;X &#402;t&#222;~             &#240;pe &#222;i &#240;1{pe &#222;s{i , pe insertion <br />               0                                                             Dataset S1 Gene gene comparison essentiality, <br /> probability TA site, s total number TA sites             fitness operon calls. <br /> gene, t number observed mutations. A probability                   (XLSX) <br /> generating function used equal insertion probability could <br />                                                                               Figure S1 Properties essential genes inserted the <br /> not assumed. The general form G(x) gene was <br />                        s                                                      gene core 80%. Of 273 TIFA identified essential genes, 50 <br /> written as: G&#240;x&#222;~ P &#240;pi xz&#240;1{pi &#222;&#222;~P&#240;X ~0&#222;zP&#240;X ~1&#222;xz                          genes contained insertions associated fitness values. 28 the <br />                        i <br /> P&#240;X ~2&#222;x2 z       zP&#240;X ~s&#222;xs , s number TA                    50 contained insertions beginning (10%) and/or the <br /> locations gene, pi specific probability insertion        end (10%), and/or single insertion middle gene. Of <br /> location. In power series expansion G(x), coefficient xt        remaining 22 genes insertion location associated <br /> is probability P(X = t). The cumulative probability observing          fitness (top diamonds), TA sequences (black diamonds), conserved <br /> up t insertions s possible TA locations expressed                protein domains (red line segments) intra gene start codons <br />          s                                                                    associated ribosomal binding site (RBS) strengths (bottom <br /> G &#240;x&#222;~ P &#240;pi xz&#240;1{pi &#222;&#222;~P&#240;X ~0&#222;zP&#240;X ~1&#222;xz P&#240;X ~ 2&#222;x2                          diamonds) shown. Fitness values outside visualized range <br />          i <br /> z       zP&#240;X ~s&#222;xs . The transposon insertion expectation               shown extreme ends scale. RBS strength was <br />                                Ps                                             shown logarithmically 20.03 4.3, lower values were <br /> gene calculated E &#240;s&#222;~     rP&#240;x~r&#222; variance                  omitted. For gene, row underneath line <br />                                   r~1 <br />              P <br />              s                                                                represents RBS strengths associated intra gene start <br />                    2          2 <br /> Var&#240;s&#222;~     r P&#240;x~r&#222;{E . The normalized deviation expec-                   codons uninserted gene. Each successive couple rows <br />              r~1 <br />                                                p&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;                      represents RBS strengths downstream start codons for <br /> tation (NDE) given by: NDE~&#240;Obs{Expected &#222;= Var.                           insertion. Two rows shown mutant the <br />                                                                               orientation insertions unknown. All downstream RBS <br /> FBA essentiality calls                                                        strengths shown transposon insertion alter <br />    Transposon gene deletion simulations performed                RBS strength downstream intra gene start codons. <br /> different media conditions: LB dataset Deutschbauer              Mutations genes (SO4432, SO3185 SO4669) caused <br /> [18], SBM dataset Brutinel [28]. Because                substantially slower growth rates. Insertions genes <br /> mutant libraries generated aerobic conditions, FBA                 (SO4391, SO0148, SO3873, SO4068 SO3084) not <br /> predictions aerobic conditions. To prevent artifacts            explained presented data. For rest genes, <br /> resulting unrealisticly large redox exchanges media,            insertions outside conserved protein domains, and <br /> nutrient uptake rates limited concentrations               insertions conserved domains corresponded slow <br /> media (Text S1 details). The low concentration metals               growth (SO2545). Insertions genes (SO3178, SO1441, <br /> both media unable sustain dissimilatory metal-              SO3993, SO3874, SO4283, SO2133, SO1442, SO4359, <br /> reducing growth. Genes designated FBA essential removal               SO0730, SO2544) resulted functional protein <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         9                   September 2014 | Volume 10 | Issue 9 | e1003848 <br />                                                                                                                                      Transposon Insertion Frequency Analysis <br />  <br />  <br /> by reinitiating transcription translation transposon                                  Author Contributions <br /> insertion. <br />                                                                                                 Conceived designed experiments: HY AMO JAG IGLL. <br /> (PDF)                                                                                           Performed experiments: HY EWK EDB NPP JAG IGLL. Analyzed <br /> Text S1 Additional methods results. Supplemental file                                       data: HY EWK EDB NPP JAG IGLL. Contributed writing of <br /> contains detailed description TIFA method, Monte                                   manuscript: HY NPP MJS JAG AMO IGLL. <br /> Carlo analysis used validation TIFA, calculation <br /> of fitness values. The supplement contains additional <br /> results. <br /> (PDF) <br />  <br /> References <br />  1. Cameron DE, Urbach JM, Mekalanos JJ (2008) A defined transposon mutant                      22. Henry CS, DeJongh M, Best <span id='am-125' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>, Frybarger PM, Linsay B, et al. (2010) High- <br />     library use identifying motility genes <span id='am-80' about='obo:NCBITaxon_666' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_662 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae-asiaticae" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholera" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Liquidivibrio cholerae" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:9272984" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Vibrio comma" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Pacinia cholerae-asiaticae" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-asiaticae" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Vibrio cholerae" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Bacillo virgola del Koch" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Vibrio choleae" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Kommabacillus" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholerae-asiaticae" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Microspira comma" datatype="xsd:string"></span><span id='am-98'  ></span>Vibrio cholerae</span>. Proceedings            throughput generation, optimization analysis genome-scale metabolic <br />     National Academy Sciences United States America 105: 8736&#8211;                     models. Nat Biotechnol 28: 977&#8211;982. <br />     8741.                                                                                       23. Kim TY, Sohn SB, Kim YB, Kim WJ, Lee SY (2012) Recent advances in <br />  2. Jacobs MA, Alwood A, Thaipisuttikul I, Spencer D, Haugen E, et al. (2003)                       reconstruction applications genome-scale metabolic models. Curr Opin <br />     Comprehensive transposon mutant library <span id='am-41' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa.                              Biotechnol 23: 617&#8211;623. <br />     Proceedings National Academy Sciences United States                     24. Krumholz EW, Yang H, Weisenhorn P, Henry CS, Libourel IG (2012) <br />     America 100: 14339&#8211;14344.                                                                       Genome-wide metabolic network reconstruction picoalga <span id='am-28' about='obo:NCBITaxon_70447' typeof='owl:Thing obo:NCBITaxon_13792 obo:NCBITaxon_33090 obo:NCBITaxon_1 obo:NCBITaxon_1035538 obo:NCBITaxon_131567 obo:NCBITaxon_3041 obo:NCBITaxon_2759'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Ostreococcus" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ostreococcus</span>. <br />  3. van Opijnen T, Camilli A (2013) Transposon insertion sequencing: new tool                     J Exp Bot 63: 2353&#8211;2362. <br />     systems-level analysis microorganisms. Nat Rev Microbiol 11: 435&#8211;442.                25. Zomer A, Burghout P, Bootsma HJ, Hermans PW, van Hijum SA (2012) <br />  4. van Opijnen T, Bodi KL, Camilli A (2009) Tn-seq: high-throughput parallel                       ESSENTIALS: software rapid analysis high throughput transposon <br />     sequencing fitness genetic interaction studies microorganisms. Nat                   insertion sequencing data. PLoS One 7: e43012. <br />     Methods 6: 767&#8211;772.                                                                         26. Dejesus MA, Ioerger TR (2013) A Hidden Markov Model identifying <br />  5. Smith AM, Heisler LE, Mellor J, Kaper F, Thompson MJ, et al. (2009)                             essential growth-defect regions bacterial genomes transposon <br />     Quantitative phenotyping deep barcode sequencing. Genome research 19: <br />                                                                                                     insertion sequencing data. BMC Bioinformatics 14: 303. <br />     1836&#8211;1842. <br />                                                                                                 27. Chao MC, Pritchard JR, Zhang YJ, Rubin EJ, Livny J, et al. (2013) High- <br />  6. Smith AM, Heisler LE, St Onge RP, Farias-Hesson E, Wallace IM, et al. (2010) <br />                                                                                                     resolution <span id='am-131' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> <span id='am-99' about='obo:NCBITaxon_662' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-100' property="rdfs:label" content="Vibrio" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:7520733" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Microspira" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="PMID:17978204" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Pacinia" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:4935323" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Listonella" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:21296930" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasDbXref" content="PMID:1371064" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasAlternativeId" content="NCBITaxon:705" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="PMID:8590667" datatype="xsd:string"></span>Vibrio</span> cholerae essential gene set hidden Markov <br />     Highly-multiplexed barcode sequencing: efficient method parallel analysis <br />                                                                                                    <br /> </body></html>