<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.12 no.13 pp.5267-5275, 1993 <br />  <br />  <br /> Genetic analysis Cln/Cdc28 regulation cell <br /> morphogenesis budding yeast <br />  <br />  <br /> Benjamin K.Benton, Arthur H.Tinkelenberg,                          protein kinase activity constitutive cell <br /> Dickson Jean1, Suzanne Driscoll Plump2                         cycle (Wittenberg et al., 1990; Tyers et al., 1993). <br /> Frederick R.Cross                                                     When yeast cells traverse START, bud formation is <br />                                                                    initiated (Pringle Hartwell, 1981). At sets <br /> The Rockefeller University, 1230 York Avenue, New York,            genes involved bud formation. One <span id='am-33' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> genes <br /> NY 10021, USA                                                      involved bud site selection. Haploid yeast cells usually <br /> 1Present address: Bard College, Annondale-on-the-Hudson, NY, USA   bud axially (i.e. subsequent budding occurs the <br /> 2Present address: Department Developmental Biology, Stanford    previous bud site), diploid cells bud opposite poles <br /> University School Medicine, Stanford, CA 94305, USA             (Chant Herskowitz, 1991). Mutations RSRJ/BUD1, <br /> Communicated K.Nasmyth                                          BUD2 BUD5 cause haploids diploids bud <br />                                                                    randomly, mutations BUD3 BUD4 cause bipolar <br /> The CLN1, CLN2 CLN3 gene <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> G1-acting                   budding haploids (Chant Herskowtiz, 1991). Bud <br /> cyclin homologs <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> functionally        site selection non-essential process, absence <br /> redundant: Cln proteins sufficient         cells bud randomly obvious ill effects (Chant and <br /> for activation Cdc28p protein kinase activity cell          Herskowitz, 1991). Another class genes required for <br /> cycle START. The START event leads multiple                     budding site, independent bud site selection. These <br /> processes (including DNA replication bud emergence);           include CDC24, CDC42, BEMI BEM2 (Sloat et al., <br /> how Cln/Cdc28 activity activates processes remains           1981; Adams et al., 1990; Bender Pringle, 1991). <br /> unclear. CLN3 substantially different structure          Mutations genes lead unbudded arrest, the <br /> regulation CLNJ CLN2, functional                   cells continue grow depolarized fashion, replicate their <br /> redundancy CLNJ CLN2 poorly                       DNA undergo nuclear division. The class the <br /> understood. We isolated mutations alter             cytoskeletal components required budding, include <br /> redundancy, making CLN3 insufficient cell viability            actin cables bud neck filament organizers CDC3, <br /> in absence CLNI CLN2 expression. Mutations              CDCJO, CDCJJ CDC12 (Pringle Hartwell, 1981; <br /> causing phenotypes specific cell division cycle       Drubin, 1991; Ford Pringle, 1991; Kim et al., 1991). <br /> analyzed detail. Mutations gene result                   Bud site selection assembly proteins require small <br /> complete failure bud formation, leading depolarized          GTPases. Rsrl/Budlp Ras-like GTPase Cdc42p is <br /> cell growth. <span id='am-36' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39'  ></span><span id='am-40' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> gene identified BUD2, previously          rho-class GTPase (Bender Pringle, 1989; Johnson and <br /> described non-essential gene required proper bud          Pringle, 1990). The GTP-bound form GTPase is <br /> site selection required budding viability.         generally considered active form (Bourne et al., <br /> Bud2p probably GTPase-activating protein                 1991). GTPase action generates GDP-bound form, which <br /> Rsrlp/Budlp [Park,H., Chant,I. Herskowitz,I.                   probably inactive. In <span id='am-41' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> reactivate protein, <br /> (1993) Nature, 365, 269-274]; Rsrlp                GDP exchanged GTP. The guanine nucleotide <br /> required bud2 lethal phenotype. Mutations           exchange factor Budlp Bud5p (Chant et al., 1991; <br /> other genes (ERCIO ERCJ9) result different                Powers et al., 1991). Intrinsic GTPase activity most <br /> morphogenetic defect: failure cytokinesis resulting          GTP-binding proteins quite low, GTPase- <br /> the formation long multinucleate tubes. These results           activating proteins (GAPs) required stimulate their <br /> suggest direct regulation diverse aspects bud                activity (Boume et al., 1991). Recently, Bud2p been <br /> morphogenesis Cln/Cdc28p activity.                              identified GAP Budlp (Park et al., 1993). <br /> Key words: BUD2/erc/GAPIRSRllsmall GTPases                            Any single CLNis sufficient activate START (Richardson <br />                                                                    et al., 1989). This functional redundancy surprising because <br />                                                                    Clnlp Cln2p structurally homologous their <br /> Introduction                                                       expression cell cycle-regulated, Cln3p distantly <br />                                                                    related Clnlp Cln2p expression constitutive <br /> In budding yeast <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span> cerevisiae, Cdc28p          cell cycle (Hadwiger et al., 1989; above). <br /> protein kinase controls cell cycle progression GI/S             We devised screen order identify potential <br /> and G2/M transitions (Nasmyth, 1993). Traversal                regulators downstream targets specific Cln proteins. <br /> through Cdc28p-controlled point G1 (called START)           We screened mutants longer survive in <br /> commits cells DNA replication bud emergence (Pringle        presence single CLN gene elevated <br /> and Hartwell, 1981). This transition controlled              requirement CLNs (erc). Here report isolation <br /> complexes Cdc28p GI-acting cyclin homologs,           erc mutants clnl cln2 CLN3 background. Of these, <br /> Clnlp, Cln2p Cln3p (Richardson et al., 1989; Wittenberg        distinct defects bud morphogenesis, not <br /> et al., 1990; Tyers et al., 1993). Transcription Cdc28p-       block DNA replication. The ERC25 gene be <br /> associated kinase activity CLNJ CLN2 cell cycle-         allelic BUD2 (Chant Herskowitz, 1991; Park et al., <br /> regulated, peaking START, CLN3 transcription           1993), allelic CLA2 gene, isolated in <br />                                                                                                                             5267 <br />  B.K.Benton et al. <br />  <br /> a conceptually similar screen [see accompanying manuscript               discarded. We required mutants exhibit a <br /> (Cvr&apos;ckova Nasmyth, 1993)].                                          differential plating efficiency restrictive versus <br />                                                                          permissive conditions &lt; 1 104. We backcrossed at <br /> Results                                                                  member complementation group wild- <br />                                                                          type, kept mutations yielding 2:2 segregation of <br /> Isolation characterization erc mutants                            conditional inviability, indicative mutation single <br />                                                                          gene. Seventy-six mutants 24 complementation groups <br /> Mutant isolation. clnl cln2 CLN3 strains viable                      identified screen (Table I). Unexpectedly, <br /> (Richardson et al., 1989), 38&#176;C [exhibiting high <br /> plating efficiency despite morphological abnormalities <br /> (Hadwiger et al., 1989; Figures 1, 3 4)]. A clnl cln2                                      ..   &apos;11mck.ise: <br /> CLN3 strain GALI ::CLN1 CEN plasmid was <br /> mutagenized ethylmethylsulfonate (EMS) (see Materials <br /> and methods). Mutagenized cells screened 38&#176;C for <br /> their ability grow galactose, induces over- <br /> expression CLNJ, glucose, represses <br /> CLNJ expression. Mutants viable galactose but <br /> not glucose loss GALI:: CLNI plasmid                 ff              il <br />  <br />  <br />  <br />  <br /> Table I. erc mutants recovered <br />                       Number            cdc arrest glucoseb <br />                       allelesa <br /> cln3                   7   (7)             unbudded, START-C <br /> swi4                   3   (0)             NCd <br /> swi6                   1   (1)             NCd <br /> cdc28k                 1   (0)             unbudded, START-c                ,r,14 <br /> ercl                   6   (5)             _e <br /> ercS                   3   (1) <br /> erc7                   1   (0) <br /> erc8                   4   (0) <br /> erclO                  4   (3)             tubular, multinucleate <br /> ercil                  2   (0)                                             rc      _ in <br />  <br />  <br /> ercl2                  6   (0) <br /> ercl4                  3   (0)             small unbudded,   GIf <br /> ercl6                  2   (0) <br /> ercl 7                 6   (0) <br /> ercl9                  2   (0)             tubular, multinucleateg <br /> erc2O                  3   (0) <br /> erc2l                  1   (0) <br /> erc22                  1   (0)             unbudded,   GIh <br /> erc23                  1   (0) <br /> erc24                  1   (0)                                           S,,r .1   fJ1 <br /> erc25 (bud2)           9   (0)             unbudded, multinucleate&apos; <br /> erc26                  2   (0) <br /> erc27                  3   (0) <br /> erc28                  2   (0) <br /> UnsortedJ             29   (NCd)           NCd <br /> Total number mutagenized cells screened    =   30 000 <br />  <br /> aThe number alleles shown; number tight lethal <br /> phenotype 30&#176;C 38&#176;C shown parentheses. <br />  cdc arrest defined cycle arrest uniform terminal <br /> morphology mutants shifted glucose medium <br /> (YEPD). Cellular morphology DNA content (determined by <br /> FACS analysis; <span id='am-44' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown) listed. <br /> CSTART arrest, cells continue growing elongated shmoos. <br /> dNC, characterized. <br /> eNon-cdc arrest. <br /> fStrains arrest 65-86% unbudded cells. <br /> gCells tubular galactose phenotype severe          Fig. 1. Cell morphology mutants. Strains genotype <br /> plucose.  Cell morphology makes determination cdc arrest unclear.     clnl cln2 CLN3 GALI::CLNJ. Cultures grown 38&#176;C in <br />  Strain arrests 80-90% unbudded cells. Mutant efficiently           YEPGal, switched pre-warmed YEPD 8 h staining <br /> rescued SIT4 CEN plasmid.                                        DNA DAPI performing fluorescent photomicroscopy. Left <br /> &apos;Strains arrest 80-90% unbudded cells 45% cells          panels: YEPGal; right panels: YEPD. Top: ERC+; second row: <br /> binucleate 9 h shift glucose.                                   bud2::URA3 (erc25 null allele; text); row: erclO-l; bottom <br /> iMutants complement complementation groups listed           row: ercl4-1. The erclO-l mutant incubated 30&#176;C, (but <br /> above.                                                                   mutants) exhibited strong mutant phenotype both <br /> kSee Cross Blake (1993).                                             300C 380C. <br /> 5268 <br />                                                                                                                   CLNs yeast morphogenesis <br />  <br /> many mutants viable glucose 30&#176;C,                       CLN gene rescue erc mutants. We examined ability <br /> despite tight inviability 380C. The effect temperature                different CLN genes <span id='am-54' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> rescue erc <br /> in mutant hunt understood. It relevant                   mutants. While CLNI, CLN2 CLN3-2 [a hyperactive <br /> in connection Tyers et al. (1992) showed                     allele CLN3 (Cross, 1988)] low copy number CEN <br /> intrinsic temperature sensitivity Cln3p-associated                   plasmids rescued alleles erc25 (erc25-1 erc25-9), <br /> kinase activity.                                                            CLN3 rescue activity (data shown). erclO showed <br />    Mutations genes expected based previous               spectrum CLN gene rescue similar erc25. <br /> work: mutations cln3 (Richardson et al., 1989), cdc28                    Although erclO erc25 mutants rescued by <br /> (Hadwiger et al., 1989), swi4 swi6 (Nasmyth Dirick,                 hyperactive CLN3-2 level viability, rescued <br /> 1991; Ogas et al., 1991) expected recovered                   strains maintained highly defective cell morphology (data <br /> based complementation analysis (Table I). sit4 mutations                 shown). In contrast, CLNJ CLN2 rescued erc25 and <br /> increase CLN requirement (Fernandez-Sarabia et al.,                     erclO strains essentially normal cell morphology. Thus <br /> 1992); recovered mutation (erc22) efficiently               rescue erc mutations specific CLNJ and <br /> rescued SIT4 gene low copy number plasmid.                      CLN2. The ercl4 mutant rescued CLN <br />                                                                             genes CEN plasmids. <br /> A subset erc mutations exhibit cell division cycle (cdc) <br /> mutant phenotype. While START CLN-dependent                            Characterization ERC25 <br /> commitment cell cycle progression,                       An erc25 mutant strain transformed YCp5O <br /> processes required successfully complete DNA                         genomic library (Cross, 1988) complementing plasmids <br /> synthesis cellular division. These downstream events                    rescued. One plasmid (pBB01) containing CLNJ, <br /> could genetic targets CLN action. Thus,                     CLN2 CLN3-2 subcloned URA3 integrating <br /> plausible mutations affected ability CLN3               vector transformed ERC25-targeted integration into <br /> carry specific cell cycle processes                 diploid heterozygous erc25-9 allele. In tetrad <br /> be isolated. Such mutations Cdc- phenotype:                    analysis diploid, Ura+ phenotype cosegregated <br /> arrest uniform terminal morphology, implying                    Erc+ phenotype 12 tetrads. Tetrad analysis of <br /> the mutated gene required specific point                crosses marked URA3: :ERC25 other <br /> a particular process cell division cycle (Pringle                erc25 point mutant alleles showed recombinant spores <br /> Hartwell, 1981). Excluding mutations previously                      12 tetrads analyzed cross. Partial sequence analysis <br /> characterized cln3, cdc28, swi4, swi6 sit4 genes (see                   ERC25 showed significant homology with <br /> above), erc mutations exhibited Cdc-                          GAPs Ras-like GTP-binding proteins (data shown). <br /> phenotype (Figure 1). Mutations erclO cause first-cycle                The recently cloned BUD2 gene shown encode GAP <br /> arrest multinucleate tubular phenotype shift                 Rsrlp/Budlp (Park et al., 1993). Comparison the <br /> glucose. Mutation CDC3, CDCIO, CDCJJ CDC12                           sequences (H.O.Park, personal communication) revealed <br /> in CLNI CLN2 CLN3 background results similar                         identity BUD2 ERC25. This surprising <br /> phenotype (see Introduction). The erc25 mutation results                 result, CLNJ CLN2 CLN3 background, bud2 <br /> first-cycle arrest large, spherical, unbudded cells,               mutants lethal, randomize bud site selection <br /> become multinucleate prolonged arrest. This phenotype                  (Chant Herskowitz, 1991). <br /> was similar observed mutants bud site                      F.Cvrckova K.Nasmyth (see accompanying manu- <br /> assembly genes CDC24, CDC42, CDC43 BEMI                            script) isolated bud2 alleles (under &apos;cla2&apos;) <br /> CLNI CLN2 CLN3 background (see Introduction). The ercl4                     screen clnl cln2 synthetic lethals, confirming the <br /> mutation causes first-cycle arrest small, unbudded              requirement BUD2 clnl cln2 strains. However, we <br /> cells (65-85% unbudded).                                                    Cvrckova Nasmyth observed efficient com- <br />  <br />                                   PX              B          H                       BI       HI            xS <br />  <br />                                                                                  GAP region                         erc25 <br />                                                                                                                     Rescue <br />  <br />                        pBB03                                                                                           + <br />  <br />  <br />  <br />                        pBB17.1 <br />  <br />                        pBB18 <br />                                                             (Sm)        URA3-kan r            (Sm) <br />                                                                                                      1 kb <br />  <br /> Fig. 2. Mapping ERC25 BUD2. The upper line shows map subcloned insert used integration meiotic mapping ERC25. <br /> pBB03 minimum subclone able rescue erc25-1 erc25-9 mutants glucose 38&#176;C. Deletion right hand HpaI site (pBB17.1) <br /> inactivated rescue activity, did substitution URA31kanr cassette internal HpaI fragment (pBB18). Sequencing revealed indicated <br /> open reading frame, GAP homology center. Comparison sequence recently isolated BUD2 sequence (Park et al., 1993; <br /> H.O.Park, personal communication) showed identity ERC25-rescuing activity BUD2. P, PvuH; X, XhoI; B, BamHI; H, HpaI; S, Sall; Sm, <br /> SnaI Oost subcloning). <br />                                                                                                                                            5269 <br />  B.K.Benton et a/. <br />  <br />                                                                              ( , a11 (i to&lt;&gt; iilsv;                          W1Ac(IOL: <br />                                                                                                                               :I <br />                                                                            (Z. <br />                                                                              t. / .( &apos;Z, &apos; ai               }1               WI:(&apos; l\&apos;NI <br />                                                                                                                              f                 t <br />  <br />                                                                       30 <br />  <br />                         CLNI cln2 chn3 <br />                          cliI (ILX2          cn-3 <br />                                                                                                                                                    BU&apos;1)2 <br />  <br />  <br />  <br />  <br />                          c-IZl <br />                             (iii zn2      (L   XiV3 <br />                          clri I cln2 cin-3 <br />                                                            LEV0 <br />                         (&apos;XNI clrn2         cn3.3                                           r&quot; <br />  <br />  <br />                          clIn I CLN2 X          n3 <br />                                                                                                                                                    bud2 <br />                          clItzi   (lnz2 (CL3 <br />                          clnIi cIn2 chln3 <br />  <br />  <br />                            All strains artv <br />                            (,ALI::(CIN1 <br />  <br /> Fig. 3. Specificity CLN rescue bud2:: URA3 strains. Strains indicated genotype constructed tetrad analysis (see Materials and <br /> methods). All strains contained GALI :C:LNI cassette (Richardson et al., 1989) integrated leu2. bud2: strains contained bud2:: URA3 allele <br /> (see Figure 2 text). Serial dilutions YEPGal cultures strains indicated genotypes spotted YEPD (glucose) YEPGal <br /> (galactose) plates incubated 3 days indicated temperature. All strains isogenic typical large number similar strains <br /> generated tetrad analysis. <br />  <br />                                                                                         (    ,    c 1 v       sc <br />                                                                                                                                   (   .   1cos e <br />                                                                                 (&quot; {Z.,1i.&apos;. &apos;{         .         iJ;   {1       off./, <br />                                                                                                                                     ..\ <br />  <br />  <br />  <br />  <br />                                   t-   iil c1n CLN3r( <br />                                                   - <br />                                        (1 1f 1. ( /               I <br />                                                     1)2r <br />  <br />  <br />  <br />  <br />                                        ( -I I., 1:. (   Is        I <br />                                               bud--)&apos; <br />  <br />  <br />  <br />  <br /> Fig. 4. Staining chitin bud scars. Strains indicated genotype grown YEPGal 38&#176;C, switched YEPD 3 h. Chitin <br /> and bud scars stained Calcofluor described Pringle (1991). BUD2 + strains exhibit clustering bud sites characteristic axial <br /> budding (Chant Herskowitz, 1991). clnl cln2 CLN3 bud2 ::URA3 GAL] ::aNI strains bud randomly galactose medium (&apos;GAL ::CLNI on&apos;) <br /> and deposition chitin entire cell surface associated depolarized growth failure budding glucose medium <br /> (&apos;GALI ::CLNI off). <br />  <br /> 5270 <br />                                                                                                                                            CLNs yeast morphogenesis <br />  <br /> plementation cla2 erc25 mutations. In                                                    viable, sick glucose 30&#176;C inviable glucose <br /> addition, detailed complementation analysis,                                                38&#176;C (Figure 3). While plating galactose (&apos;GAL] ::CLNJ <br /> the erc25 alleles showed weak complementation diploids                                                on&apos;) rescued lethality 380C, did rescue random <br /> (data shown). This interallelic complementation                                               budding phenotype 300C 38&#176;C (Figure 4, Table II). <br /> explainable examining structure Bud2p. The central                                                The bud2 null allele crossed various CLN <br /> core Bud2p contains GAP catalytic region (Figure 2).                                              backgrounds. In crosses, conditional GAL]: CLNJ <br /> This region flanked oa-helical regions coiled-coil                                            gene used recover potentially lethal combinations; <br /> domains weak homology myosin (data                                                      gene inactivated glucose medium the <br /> shown). The carboxyl-terminal flanking region essential                                               rescue test. Strains harboring bud2 deletion containing <br /> for function (Figure 2). If Bud2p uses putative myosin-                                            CLNJ CLN2 sole CLN genes grew well <br /> like domains multimerization, envision                                                isogenic BUD2 strains, bud2 strains containing <br /> mutations block multimerization homozygous                                                   CLN3 sole CLN gene inviable as <br /> state, interallelic complementation.                                                      strains lacking CLN gene, 38&#176;C (Figure 3). This <br />                                                                                                          demonstrates CLN], CLN2 gene pair is <br /> The bud2 null phenotype different CLN backgrounds. A                                                  necessary sufficient bud2 rescue. Transcriptional <br /> deletion allele bud2 (Figure 2) constructed intro-                                            control CLN] CLN2 (see Introduction) probably <br /> duced diploid clnl cln2 CLN3 ERC+ GAL] ::CLN]                                                     relevant bud2 rescue, <span id='am-49' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-50' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> gene containing <br /> strain (see Materials methods). Tetrad analysis revealed                                             CLN3 promoter driving CLN2 coding sequence <br /> that clnl cln2 CLN3 bud2:: URA3 GAL]:: CLNI cells                                                    (Valdivieso et al., 1993) efficiently rescued viability bud2 <br />                                                                                                          null strains 38&#176;C present sole CLN gene (data <br />                                                                                                          shown). Not extra copies CLN3 a <br /> Table II. Suppression erc25 rsrl:: URA3                                                            plasmid unable rescue bud2 lethality 380C, but <br /> Cross Genotype                                Viability glucose medium                                spores expressing GALI ::CLN3 bud2 null <br /> No                                            38&#176;C tetrads                                         background unable form colonies 30&#176;C (data <br />                                               4:0         3:la       2:2a                                shown). Thus, CLN rescue appeared specific for <br />                                                                                                          CLN] CLN2. <br /> 2322 clnl cln2 CLN3 GALJ::CLNJ 0                          0                 23 <br />      RSRJ erc25-91ERC25 (BUD2)                                                                            7he relationship Erc- phenotype bud2 random <br /> 1486 clnl cln2 CLN3 GALI ::CLNJ 5                        11                   2 <br />         rsrl:: URA31+ erc25-91ERC25 <br />                                                                                                          budding RSR]/BUDJ. Since bud2 mutants CLN] <br /> 1485    clnl cln2 CLN3 GALI::CLNJ 5                      11                   3                          CLN2 CLN3 strains bud randomly (Chant Herskowitz, <br />         rsrl:: URA31+ erc25-1IERC25                                                                      1991), possible clnl cln2 CLN3 strains might <br />                                                                                                          require bud site targeting efficient buding, unlike CLNI] + <br /> aln 3:1 2:2 tetrads, glucose-inviable segregants                                         CLN2 + strains. To test this, deleted RSRI/BUDI <br /> RSR1 + (i.e. Ura-).                                                                                      gene clnl cln2 CLN3 GAL] ::CLNI background. <br /> 2:2 segregation rsrl ::URA3 observed, control crosses                                      RSRI/BUDI Ras-like protein required targeted <br /> (like x2322) 2:2 segregation erc25 (i.e. glucose-inviable) was <br /> observed. The 4:0 tetrads (non-parental ditype)                                            budding (Bender Pringle, 1989; Chant Herskowitz, <br /> interpreted containing rsrl::URA3 erc25 segregants,                                         1991). Deletion RSRI/BUDI did result Erc- <br /> are suppressed erc25 lethality, RSRJ + ERC25+                                                phenotype background, budding random <br /> (wild-type) segregants. The 2:2 tetrads (parental ditype)                                            (data shown). Thus inactivation bud site selection <br /> interpreted containing rsrl ::URA3 ERC25+ segregants,                                         pathway sufficient cause Erc- phenotype. <br /> two RSRI + erc25 (Erc- glucose-inviable) segregants. The 3:1 <br /> tetrads (tetratype) interpreted containing strains                                          BUD2 proposed encode GAP for <br /> different genotypes: ERC25 + rsrl :: URA3 (glucose-viable), ERC25 +                                      Rsrlp/Budlp (Park et al., 1993; Bender, 1993). Thus the <br /> RSRI + (glucose-viable), erc25 rsrl ::URA3 (glucose-viable)                                      Erc- phenotype bud2 strains the <br /> erc25 RSRI + (glucose-inviable). Only tetrads viable spores                                    accumulation Rsrlp/Budlp GTP-bound form, due <br /> and 2:2 segregation AL4T, LEU2 rsrl ::URA3 tabulated. <br /> These constituted three-quarters tetrads dissected                                      lack GAP activity. The RSR]val12 mutant allele of <br /> cross. Most excluded tetrads viable spores                                     RSRI inhibits GTPase activity Rsrlp therefore <br /> not analyzed further.                                                                                    &apos;locks&apos; Rsrlp GTPase GTP-bound form (Bourne <br />  <br />                                                                          ( FALKIII&apos;llu: <br />                                                                  t ,    I I I : : ( I. % I          II                    GI   ( I 1( <br />                                                                                                                                    , <br />  <br />  <br />                                                               ;ii&apos;l (                        IN x   41              &apos; <br />                                                                                                                         ,of.   C   3I(I <br />  <br />  <br />                               /,R.SR1-vi12/l <br />                               c/ni&apos; iM2 (1 \*. <br />                                 rsrl::tI R <br />                                   II   &apos;(,( t rlor <br />  <br />                                    (,-I I::(I \ <br />  <br /> Fig. 5. Erc- phenotype RSR Vall2. A clnl cln2 CLN3 GAL4I: :CLNJ rsrl:: URA3 strain transformed pRS314 vector, vector <br /> containing RSRJ Va112 gene. Two transformants DNA tested viability indicated conditions Figure 3, that <br /> Sc-trp medium requiring retention plasmids used throughout. <br />                                                                                                                                                                   5271 <br />  B.K.Benton et al. <br />  <br />  <br />  <br />  <br /> et <br />                             .frvv,1*l <br />  <br />  <br />  <br />  <br />                            }? &#162; <br />                                   r <br />                                       .;r&gt;,Z/-e1~. <br />                                    R. R I - <br />  <br />  <br />  <br />                                   R X f r1T <br />                                               R <br />  <br />  <br />  <br />  <br />      al., 1991; Ruggieri et al., 1992). We introduced <br />                                                      3 <br />                                                          s, s <br />  <br />  <br />  <br />                                                           .: <br />                                                           _, <br />                                                                 7 <br />  <br />  <br />  <br />  <br />                                                                     .... <br />                                                                               Imte <br />                                                                            .- .&apos;. <br />                                                                                       )&quot;c <br />                                                                                      ;.   \s /   I I <br />  <br />  <br />  <br />                                                                                                  1) fi <br />  <br />  <br />  <br />  <br /> each genotype tested. Two dilutions 10-fold dilution series shown strain. <br />                                                                                                          ( <br />                                                                                                                                            (;I [IC( we: <br />                                                                                                                                   (G1. I 1. 1:. ( T. 1 I oit&apos; <br />  <br />                                                                                                                            3s1t 1} ( <br />  <br />  <br />  <br />  <br /> Fig. 6. Suppression erc25 rsrl ::URA3. Segregants x1485 x1486 (Table III) analyzed Figure 3. Strains genotyped as <br /> follows: rsrl ::URA3 indicated Ura+ phenotype. erc25 rsrl ::URA3 strains ERC+ RSRJ + strains selected 4:0 tetrads (see <br /> Table II). erc25 RSRI + ERC+ rsrl ::URA3 strains selected 2:2 tetrads (see Table II). From x1485 x1486, strains of <br />                                                                                                                                                          z3sO   (, <br />  <br />  <br />  <br />  <br />                                                                                                              Table III. Budding patterns <br /> RSRlval12 low copy number CEN plasmid clnl <br /> cln2 CLN3 rsrl::URA3 GALI ::CLNI strain. RSRIvaIl2                                                           Genotype                                           Medium        % Axiala <br /> generated significant Erc- phenotype 38&#176;C (Figure 5).                                                   CLNJ CLN2 CLN3 BUD2                                YEPD 300C     788 6      (3) <br /> This phenotype required deletion wild-type RSRJ gene                                                  CLNI CLN2 CLN3 bud2                                YEPD 300C     18 X 1     (2) <br /> (data shown). This result consistent <span id='am-28' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-29' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span>idea</span>                                               CLNJ CLN2 cln3 BUD2                                YEPD 300C     71 + 1     (2) <br /> accumulation Rsrlp-GTP account bud2                                                         clnl cln2 CLN3 BUD2                                YEPD 300C     72 + 10    (4) <br /> Erc- phenotype.                                                                                              clnl cln2 CLN3 BUD2 GALI::CLNJ                     YEPGal 380C   73 (1) <br />   This hypothesis predicts deletion RSRJ                                                      clnl cln2 CLN3 bud2 GALI ::CLNJ                    YEPGal 380C   17 (1) <br /> suppress Erc- phenotype erc25. clnl cln2 CLN3                                                         clnl cln2 cln3 BUD2 GALI ::CLNJ                    YEPGal 380C   73 &#177; 4     (2) <br /> GAL] ::CLNJ strains erc25-1 erc25-9                                                      clnl cln2 cln3 BUD2 GALI ::CLN2                    YEPGal 380C   35 + 9     (3) <br /> were crossed clnl cln2 CLN3 GAL]:: CLNJ rsrl:: URA3                                                     clnl cln2 cln3 BUD2 GALI::CLN3                     YEPGal 380C   71 (1) <br /> strain, tetrads dissected. In control cross                                                    Cultures pregrown indicated medium (liquid) log <br /> rsrl:: URA3 segregating, Erc- phenotype (inviability                                                  phase, sonicated briefly plated medium (solid). <br /> glucose 38&#176;C) erc25-9 segregated 2:2 expected                                                   1.5-2.5 h plating, plates examined microscopically. <br /> (Table II). In crosses rsrl ::URA3 erc25-1                                                   4-cell/bud microcolonies (200 strain) analyzed budding <br /> or erc25-9 segregating, Erc- phenotype failed                                                         pattern essentially described Chant Herskowitz (1991). <br />                                                                                                              aPercentage microcolonies mother daughter cells <br /> segregate 2:2; instead, segregation pattern observed                                                   produced buds hemisphere nearest cell. The <br /> indicating efficient suppression Erc- phenotype                                                    mean, standard deviation number strains tested are <br /> erc25 rsrl ::URA3 (Table II; Figure 6).                                                                   presented. <br />    We conclude bud2 mutants exhibit Erc- <br /> phenotype accumulation Rsrlp/Budlp                                                         bud2 disruption essentially eliminated pattern <br /> bound GTP inability hydrolyze GTP GDP.                                                  (Table HI; Figure 4). Axial budding clnl cln2 strains and <br /> The results support proposal (Park et al., 1993)                                                 cln3 strains confirmed Calcofluor staining bud <br /> that Bud2p specific GAP Rsrlp/Budlp.                                                                scars (Chant Herskowitz, 1991; data shown). Thus, <br />    These results suggest crossover bud-site selection                                                     bud site selection probably control a <br /> functions bud-site assembly functions (see Introduction).                                               specific CLN gene. However, overexpressing CLN2 the <br /> Consistent idea, observed partial rescue                                                   GALI promoter galactose induced significant degree of <br /> of erc25 mutants high copy number plasmid containing                                                    random budding (Table II), contexts CLN function <br /> CDC42, low copy number CEN plasmid containing                                                           role regulation bud site selection. <br /> BEM] (data shown). <br /> Bud site selection CLN genes. The identification bud2 <br />                                                                                                              Discussion <br /> as erc screen suggest bud site selection                                               Specificity erc screen <br /> is CLN control. Strains various combinations                                                   Looking mutants elevated CLN requirements, <br /> cln mutations tested budding pattern (Chant                                                     identified mutations 24 genes cause <br /> Herskowitz, 1991). clnl cln2 CLN3, CLNJ CLN2 cln3                                                        inviability CLN3 CLN present the <br /> wild-type strains, clnl cln2 cln3 strains overexpressing                                                  genome, rescued overexpression CLNI. <br /> CLNJ CLN3 maintained axial budding phenotype,                                                     Excluding mutations genes previously described as <br /> 5272 <br />                                                                                                          CLNs yeast morphogenesis <br />  <br /> synthetically lethal clnl cln2 (cln3, cdc28, swi4, swi6,         A. <br /> sit4), mutations Cdc- phenotype. <br /> Specificity genetic screen best evaluated saturation,                                                             Bud <br /> that is, point having multiple mutant alleles          Cln/Cdc28 -*              START                    * morphogenesis <br /> each gene mutate given phenotype. We have <br /> probably saturated Erc- depolarized growth phenotype, <br /> with mutations falling complementation <br /> group (bud2) (Table I). This specificity confirmed <br /> by fact conceptually similar technically different <br /> search mutants carried different strain back-            B. <br /> ground F.Cvrfckova K.Nasmyth (see accompanying                                      Polarization cell <br /> manuscript) identified bud2 mutation giving                               growth (bud site sel&apos;n?) <br /> Cdc- depolarized growth phenotype. <br />    Although permissive conditions used screen <br /> allowed gross overexpression CLNJ GAL]                  Cln/Cd1c28                                                  Bud <br />                                                                                                                            morphogenesis <br /> promoter (Richardson et al., 1989), overexpression was <br /> not required, case bud2. The bud2 mutant                                   Bud constriction; <br /> is rescued single wild-type copy CLN]                                       neck formation <br /> CLN2, absence CLNs, not <br /> rescued extra copies CLN3 GAL]:: CLN3,           Fig. 7. Two models relationship Cln/Cdc28p activity and <br /> only poorly rescued CLN3-2 (see above). However,            bud morphogenesis. (A) In model, Cln/Cdc28p activity is <br /> these observations leave possibility open CLN3          considered activate event called &apos;START&apos;. The program bud <br />                                                                     emergence occurs downstream of, dependent on, event. <br /> simply active CLN gene CLN] CLN2;               (B) In model, Cln/Cdc28p activity independently activates multiple <br /> thus, difference rescue activity bud2 mutant       pathways leading bud morphogenesis. This model appears better <br /> might simply reflect overall &apos;CLN activity&apos;             able explain finding Cln-specific mutations altering specific <br /> intrinsic CLN specificity. This difficult possibility       aspects bud morphogenesis. <br /> evaluate different methods comparing activity of <br /> different CLN genes yield different answers. <br />    Tyers et al. (1993) showed using histone HI              previously. Certain cdc24 mutations permissive temperature <br /> a substrate, Clnlp- Cln2p-associated kinases                induce random budding, does overexpression CDC42 <br />  - <br />    200-fold active Cln3p-associated kinase. The               (Sloat et al., 1981; Johnson Pringle, 1990). RSRI/BUDJ <br /> GALI ::CLN3 construct strongly increases Cln3p levels               originally isolated multicopy suppressor cdc24 <br /> (Tyers et al., 1992; Cross Blake, 1993), does         mutant (Bender Pringle, 1989). beml, mutation a <br /> not rescue bud2 null strains (see above). However, low          gene required bud site assembly, identified as <br /> activity Cln3p-associated kinase consistent      synthetic lethal bud5 (Chant et al., 1991) suggesting <br /> quantitative model explaining differential CLN gene             interaction gene products. Similarly, <br /> rescue bud2 strains.                                             extra copies CDC42 BEMI partially rescued the <br />    Instead histone HI kinase activity assay relative   lethality bud2 point mutants erc25-1 erc25-9 <br /> Cln function use ability various CLNs drive       (data shown). <br /> the START transition measured decreases cell                  The Erc- phenotype bud2 (i) mimicked BUD2 <br /> volume length GI. In absence            strains introduction mutant Rsrlp/Budlp unable to <br /> CLN gene, GAL]:: CLN3 CLN3-2 significantly              hydrolyze GTP (ii) suppressed deletion RSR]/BUDI <br /> potent CLNJ CLN2 (Cross, 1988, 1990; Nash et al.,           (see above). Park et al. (1993) proposed model Bud2p <br /> 1988; Hadwiger et al., 1989; Lew Reed, 1992; F.Cross,           function bud site selection Rsrlp binds an <br /> unpublished data). This comparison suggest high             essential component bud site assembly (e.g. Bemlp, <br /> degree intrinsic CLN specificity rescue bud2          Cdc24p Cdc42p; Introduction), brings this <br /> lethality. Possible reasons discrepancy         essential component appropriate bud site; there, Bud2p <br /> low vitro kinase activity CLN3 high biological        stimulates GTP hydrolysis Rsrlp-GTP, causing localized <br /> activity discussed Tyers et al. (1993).                      release essential component leading targeted <br />                                                                     budding. In absence Bud2p, essential components must <br />                                                                     escape binding Rsrlp-GTP arrive &apos;random&apos; site <br /> CLNs bud site assembly                                          resulting randomized budding. According model, <br /> The bud mutants originally isolated CLNJ CLN2             results imply release essential component from <br /> CLN3 background mutants longer budded axially            Rsrlp-GTP requires CLN] CLN2 action. Two predictions <br /> (i.e. budding adjacent previous bud site) haploids        follow model: first, overexpression of <br /> (Chant Herskowitz, 1991), viable.            essential component(s) suppress Erc- <br /> However, work reported uncovers potential          phenotype bud2, second, overexpression CLN] <br /> role BUD2. clnl cln2 bud2 mutants arrested large             CLN2 randomize bud site selection driving <br /> unbudded cells delocalized cell growth multiple            release essential component(s) Rsrlp-GTP at <br /> nuclei, similar phenotype mutations genes              non-targeted sites. Both predictions partially <br /> required bud site assembly CDC42 CDC24              satisfied results presented above: overexpression of <br /> (Pringle Hartwell, 1981).                                       Cdc42p Bemlp suppress Erc- phenotype bud2 <br />    Cross-over function bud site selection genes      (data shown), overexpression CLN2 (but not <br /> and genes required bud assembly observed              CLNI) does partially randomize bud site selection (Table <br />                                                                                                                                       5273 <br />  B.K.Benton et al. <br />  <br /> III). Further work identifying genetic targets mediate                    Plasmid constructions <br /> CLNJ CLN2 rescue bud2 phenotype identify                        pBB03 5.5 kb XhoI fragment containing BUD2 pBBO1 subcloned <br />                                                                                XhoI site pRS316 (Sikorski Hieter, 1989). pBB06 the <br /> CLN-regulated genes/proteins required bud site assembly.                   6.0 kb SaII-Pvull fragment BUD2 subcloned integrating vector <br />    The effects CLN genotype budding proficiency                      pRS306 (Sikorski Hieter, 1989). pBB18 constructed digesting <br /> demonstrated bud2 Erc - phenotype consistent                      pBB03 HpaI substituting SnaI-released URA3-kanr cassette <br /> with data suggesting CLNJ CLN2 drive                            pJA53 (kindly provided S.Elledge) 2.2 kb internal HpaI <br /> hyperpolarized cell growth resulting elongated buds                    fragment BUD2. pBB36 1.6 kb EcoRI-SalI fragment from <br />                                                                                YCp5O-(RSRJ Val 12) (Ruggieri et al., 1992), subcloned EcoRI - <br /> overexpressed (Lew Reed, 1993). CLNJ CLN2,                             SalI sites pRS314 (Sikorski Hieter, 1989). <br /> but CLN3, polarize cortical actin pre-bud site. <br /> This consistent model presented                        Sequencing <br /> where CLNJ CLN2 activate function needed bud                         Sequencing carried plasmid fl phage DNA, using <br /> site assembly Bud2p substitutes function                       Sequenase version 2.0 (US Biochemical) according manufacturer&apos;s <br />                                                                                instructions. <br /> clnl cln2 CLN3 cells. <br />                                                                                Gene disruption <br /> Multiple roles Cln/Cdc28p START                                         The 5.5 kb XhoI fragment pBB18 containing UR43-kanr <br /> In Figure 7, models effect Cln/Cdc28p activity                  substitution BUD2 isolated low melting point agarose and <br />                                                                                transformed diploid 2330d3.1 (Erc+ diploid). Gene replacement was <br /> on bud morphogenesis presented. In model A, Cln/                           verified Southern hybridization. Diploids successful replacements <br /> Cdc28p activates START activates global program                        sporulated tetrads dissected above. <br /> for bud morphogenesis. In model, bud morphogenesis <br /> is &apos;post-START&apos; direct input regulation                          Calcofluor DAPI staining <br />                                                                                Cells fixed stained Calcofluor (Pringle, 1991) DAPI <br /> Cln/Cdc28p. In model B, Cln/Cdc28p directly activates                       (Sherman et al., 1986) described. <br /> least processes leading bud morphogenesis. The data <br /> presented support second model. First, bud2                       Analysis budding pattern <br /> mutants suggest role Cln/Cdc28p polarization cell                  Cells grown mid-log phase appropriate medium and <br /> growth bud site assembly. Secondly, erclO mutants                      temperature. Cells plated YEPGal YEPD plates <br />                                                                                pre-warmed assay temperature. Cells incubated allowed <br /> suggest bud site constriction, required proper                        undergo rounds budding. The budding pattern scored essentially <br /> cytokinesis, controlled Cln/Cdc28p independently                     according criteria used Chant Herskowitz (1991). <br /> of control polarized growth. Cln/Cdc28p activate the <br /> processes spindle pole body duplication DNA replica-                    Plating efficiency assays <br /> tion (Pringle Hartwell, 1981). Since processes                       Stationary phase cells resuspended medium carbon <br />                                                                                source 10-fold serial dilutions performed. Five microliters from <br /> continue absence bud morphogenesis (Pringle                      dilutions plated appropriate plates incubated <br /> Hartwell, 1981; Figure 1), additional targets Cln/Cdc28p                   2 days. <br /> activity required account initiation processes <br /> at START. <br />                                                                                Acknowledgements <br /> Materials methods                                                          Special thanks A.Abdul help initial isolation mutants, and <br />                                                                                L.Riles providing lambda phage grid genomic mapping and <br /> Cell culture genetics                                                      analysis mapping data. We thank A.Bender, J.Chant, S.Elledge, <br /> Media genetic methods described Ausubel et al. (1987). All      A.Sutton C.Wittenberg plasmids. We thank A.Bender, F.Cvfckova, <br /> yeast strains isogenic BF264-15D. Mutant clnl, cln2 cln3         K.Nasmyth H.O.Park communicating results publication. <br /> alleles, GALI: CLN1 overexpression cassette, described       We thank F.Cvrckova, K.Nasmyth, E.Vallen, S.DiNardo J.Pringle for <br /> previously (Richardson et al., 1989; Cross Tinkelenberg, 1991).            useful discussions. F.R.C. Scholar Lucille P.Markey Charitable <br />                                                                                Trust. A.H.T. supported NSF Fellowship. This work was <br /> EMS mutagenesis screening erc mutants                                  supported Markey Trust, A. M.Beckman Foundation, <br /> Strain 1242-8B (clnl cln2 CLN3 pGALI ::CLN3/URA3) mutagenized              Rockefeller University PHS grant GM47238. <br /> with EMS (Sherman et al., 1986) approximate mutagenesis frequency <br /> of 1/1000 locus. Mutagenized cells plated YEPGal 38&#176;C, <br /> and colonies replica-plated YEPD plates 38&#176;C. Colonies inviable     References <br /> on YEPD characterized. The rapidity arrest Cdc <br /> specificity arrest determined growing backcrossed mutant       Adams,A.E.M., Johnson,D.I., Longnecker,R.M., Sloat,B.F. and <br /> strains mid-log 38&#176;C YEPGal  <br /> </body></html>