<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 92, pp. 3127-3131, April 1995 <br /> Genetics <br />  <br />  <br /> Identification gene (SSU71/TFGJ) encoding largest <br /> subunit transcription factor TFIIF suppressor a <br /> TFIIB mutation <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />      (SSU71/transcription start-site selection) <br /> <span id='am-1' about='obo:NCBITaxon_143324' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_81386 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_81393 obo:NCBITaxon_7776 obo:NCBITaxon_123357 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Zu" datatype="xsd:string"></span>ZU</span>-WEN SUN AND MICHAEL HAMPSEY <br /> Department Biochemistry Molecular Biology, Louisiana State University Medical Center, Shreveport, LA 71130-3932 <br /> Communicated Phillip A. Sharp, Massachusetts Institute Technology, Cambridge, MA, December 15, 1994 <br />  <br /> ABSTRACT          Mutations <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae                         potential amphipathic <span id='am-33' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> flanked repeats (10, 13). <br /> SSU71 gene isolated suppressors transcription                          These elements organized structurally distinct do- <br /> factor TFIIB defect confers cold-sensitive growth                       mains defined compact, protease-resistant C-terminal <br /> defect downstream shift transcription start-site                           core, protease-susceptible N-terminal region (14, 15). <br /> selection cycl locus. The ssu71-1 suppressor                        Mutational analysis demonstrated C-terminal core is <br /> suppresses conditional phenotype restores                          necessary sufficient interaction TBP- <br /> normal pattern transcription initiation cycl. In addition,                    promoter complex N-terminal region interacts <br /> the ssu71-1 suppressor confers heat-sensitive phenotype                      pol II-TFIIF essential basal transcription <br /> is dependent presence defective form TFIIB.                      initiation (14-18). The N-terminal region includes the <br /> Molecular genetic analysis cloned SSU71 gene                             phylogenetically conserved region TFIIB. <span id='am-41' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> se- <br /> demonstrated SSU71 single-copy essential gene                             quence lies immediately distal zinc finger required <br /> encoding highly charged protein molecular mass                          accurate start-site selection vivo (19). <br /> 82,194 daltons. Comparison deduced Ssu71 amino acid                             TFIIF identified human [RAP30/RAP74 <br /> sequence protein <span id='am-87' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91'  ></span>data</span> banks revealed significant                           (20)], rat [3/,y (21)], Drosophila [factor 5 (22)], S. cerevisiae <br /> similarity RAP74, larger subunit human general                        [factor g (23)]. Human TFIIF composed subunits with <br /> transcription factor TFIIF. Moreover, Ssu71 identical                         apparent molecular masses 30 kDa 74 kDa (24), <br /> p15, component yeast TFIIF. Taken together, data                         factor g composed subunits 105 kDa, 54 <br /> demonstrate functional interaction TFIIB                          kDa, 30 kDa (23). A number biochemical properties <br />                                                                                     assigned TFIIF, including high affinity pol II <br /> large subunit TFIIF interaction affect                         (20, 22, 23, 25), stabilization preinitiation complex <br /> start-site selection vivo.                                                       (26-29), accurate start site recognition (24), ability to <br />                                                                                     stimulate rate elongation (22). The individual contri- <br /> Accurate transcription initiation RNA polymerase II (pol                         butions RAP30 RAP74 subunits TFIIF functions <br /> II) requires auxiliary factors include TATA-binding                        understood. Although RAP30 implicated <br /> protein (TBP) transcription factors TFIIA, TFIIB, TFIIE,                        functions attributed TFIIF (5, 27,30-32), the <br /> TFIIF, TFIIH (1-3). These factors assemble defined                         role RAP74 remained obscure. However, recent evi- <br /> <span id='am-56' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> DNA template generate functional transcrip-                         dence indicates subunits required most <br /> tion preinitiation complex vitro (4). A minimal transcription                    functions attributed TFIIF (29, 33). <br /> system composed pol II, TBP, TFIIB, TFIIF neces-                             We using genetic approach identify components of <br /> sary sufficient accurate basal transcription (5, 6),                        transcription preinitiation complex, emphasizing identifi- <br /> although TFIIF dispensable transcription super-                       cation factors functionally interact TFIIB. We <br /> coiled template (7). In yeast Saccharomyces cerevisiae,                         recently reported replacement phylogenetically <br /> several factors exist holoenzyme complex pol                     invariant Glu-62 lysine TFIIB dramatic effect <br /> II, consisting 12 pol II subunits Srb proteins,                      transcription start-site selection (19). In addition, the <br /> along TFIIB, TFIIF, TFIIH (8) TFIIF, Sugl,                       Glu-62 -&gt; Lys replacement confers pronounced cold- <br /> and Galil (9).                                                                      sensitive (Csm-) growth defect, phenotype associated <br />    The functional role basal transcription factors                   defects assembly multisubunit complexes (34). We <br /> mechanism transcription initiation subjects                           taken advantage Csm- phenotype isolate <br /> intense investigation. Mutations yeast genes encoding                        extragenic suppressors TFIIB defect. Here report <br /> TFIIB (SUA7) largest subunit pol II (RPB1/SUA8)                          mutations gene designated SSU71 * compensate for <br /> can dramatically affect transcription start-site selection,                         Csm- phenotype altered initiation pattern <br /> thereby establishing role components                             conferred Glu-62 -- Lys replacement. SSU71 was <br /> defining start sites vivo (10, 11). Moreover, TFIIB pol                      identified gene encoding p105 subunit factor g. <br /> II appear sole determinants accurate start-site                        This information shows functional interaction between <br />                                                                                     TFIIB large subunit TFIIF shows this <br /> selection vitro, replacement TFIIB pol II                      interaction affect start site selection vivo. <br /> from S. cerevisiae counterparts Schizosaccharo- <br /> myces pombe sufficient switch start-site selection the <br /> pattern characteristic S. pombe (12).                                                          MATERIALS AND METHODS <br />    Sequence analysis TFIIB revealed structural fea-                          Yeast Strains, Genetic Methods, Nomenclature. The <br /> tures: zinc-finger motif near N terminus, imperfect                       yeast strains used study listed Table 1. Standard <br /> repeats C-terminal two-thirds protein, a <br />                                                                                     Abbreviations: pol II, RNA polymerase II; TBP, TATA-binding protein; <br /> The publication costs article defrayed page charge          Csm-, cold-sensitive; Tsm-, heat-sensitive. <br /> payment. This articie marked &quot;advertisement&quot;            *The sequence SSU71 reported paper deposited in <br /> accordance 18 U.S.C. &#167;1734 solely indicate fact.                        GenBank data base (accession no. U14527). <br />                                                                              3127 <br />  3128        Genetics: Sun Hampsey                                                                   Proc. Natl. Acad Sci. USA 92      (1995) <br />                        Table 1. Yeast strains, relevant genotypes, phenotypes <br />                                                                                                          Phenotypet <br />                              Strain*                      Relevant genotype                   Cyt c, %          Csm         Tsm <br />                        T16                         cycl-5000                                     &lt;2              +           + <br />                        YDW546                      cycl-5000 sua7-1                               30             -           + <br />                        YMH71-9C                    cycl-5000 sua7-1                               30             -           + <br />                        YZS14                       cycl-5000 sua7-1 ssu7l-1                      &lt;2              + <br />                        YZS45                       cycl-5000 sua7-1 ssu7l-2                      &lt;2              + <br />                        YZS14-2B                    cycl-5000 sua7-1 ssu71-1                      &lt;2              + <br />                        YZS14-3B                    cycl-5000 sua7-1 ssu71-1                      &lt;2              + <br />                        YZS14 x YDW546              cycl-5000/cycl-5000                            30             -           + <br />                                                      sua7-1/sua7-J SSU71/ssu71-1 <br />                        *Strains T16, YDW546, YMH71-9C described previously (10, 11); described in <br />                         study. <br />                        tPhenotypes defined follows: Cyt c corresponds <span id='am-82' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-83' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86'  ></span>iso</span>-1-cytochrome c content determined <br />                         low-temperature, whole-cell spectroscopy (35) reported percentage iso-1-cytochrome <br />                         c content CYC1+ strain; Csm Tsm refer growth rich (yeast extract/peptone/dextrose) <br />                         medium 16&#176;C 37&#176;C, respectively. <br /> procedures used crossing strains, selecting diploids,                         restriction analysis 3.2-kb Kpn I-Sph I DNA fragment (Fig. <br /> inducing sporulation, dissecting tetrads (36). Standard                            1). Southern blot analysis confirmed 3.2-kb Kpn I-Sph <br /> yeast nomenclature used designate                                I DNA fragment isolated intact yeast genome and <br /> genotypes phenotypes. For example, ssu71-1 denotes                               SSU71 single-copy gene (data shown). <br /> recessive allele wild-type SSU71 gene. The symbols                                 DNA Sequence Analysis. The sequences strands of <br /> Csm- (cold-sensitive) Tsm- (heat-sensitive) refer dis-                          SSU71 determined dideoxy-terminator method <br /> tinctly impaired growth (or growth) rich medium (yeast                           using M13 universal primer SSU71-specific primers and <br /> extract/peptone/dextrose) 16&#176;C 37&#176;C, respectively. Cyt                          single-stranded DNA derived exodeoxyribonuclease III <br /> c Table 1 corresponds iso-1-cytochrome c content                          deletions SSU71. <br /> determined low-temperature, whole-cell spectroscopy (35)                               Sequence Comparisons Homologies. DNA protein <br /> and reported percentage iso-1-cytochrome c                              sequence comparisons performed using BLAST <br /> content in.a CYCl+ strain.                                                             algorithm (40), accessed National Center Biotech- <br />    Isolation SSU71 Gene. The SSU71 gene isolated                            nology Information E-mail service. <br /> from YCp5O genomic library (37) complementation                               Determination Transcript Start Sites. Primer extension <br /> Tsm- phenotype conferred ssu71-1 allele using strain                            performed described previously, using total RNA and <br /> YZS14 host. The relationship complement-                            cycl-specific primer oIP-13 (10). Primer extension prod- <br /> ing DNA genetically defined ssu7l locus estab-                             ucts resolved 8% polyacrylamide DNA sequencing <br /> lished integration cloned DNA genome                             gel visualized autoradiography. <br /> strain YDW546 (sua7 SSU71 + leu2) homologous recombi- <br /> nation (38). This tags chromosomal locus cloned                                                        RESULTS <br /> DNA LEU2 marker. The resulting strain crossed <br /> with strain YZS14-2B (sua7 ssu7l leu2), sporulated,                                  Isolation SSU71 Gene. The sua7-1 allele strain <br /> dissected. The LEU2 marker segregated opposite ssu7l                                YMH71-9C encodes TFIIB Glu-62 -&gt; Lys replacement that <br /> (Tsm-) progeny. The SSU71 gene delimited                              confers altered start-site selection severe Csm- <br />  <br />  I          I                                     I           I            I       I          I <br />                    1                    2                     3                    4                    5 <br />        E <br />            as~~~~~~~~~~~~~~~~~~~~~~c <br />  <br />  n. cT <br />     0  0,c &apos;2CO                                                                , <br /> C2                                                             CIOU71 <br />  <br />                    TyrRSSS7 <br />  <br />  <br />                                                                                                               FIG. 1. Restriction map SSU71 re- <br />                                                                                                             gion. A partial restriction map SSU71 <br />                                                                                                             locus shown. DNA fragments comple- <br />                                                           I                                         I       ment Tsm- Csm+ phenotypes <br />                                                                                                             associated ssu71 suppressor sua7-1 <br />                                                                                                             background indicated black bars, <br />                                                                                                             open bars denote failure comple- <br />                                                                                                             ment. The complementing 3.2-kb Kpn I-Sph I <br />                                                                                                             fragment vector YCplac33 designated <br />                                                                                                             pM435. The TyrRS gene, encodes ty- <br />                                                                                                             rosine tRNA synthetase (39), lies immediately <br />                                  I                                             I                            adjacent SSU71. <br />           Genetics: Sun   Hampsey                                                Proc. Natl. Acad Sci. USA 92 (1995)          3129 <br /> phenotype (19). Spontaneous Csm+ revertants strain                           MSRRNPPGSRNGGGPTNASPFIKRDRMRRN       30 <br />                                                                                 FLRMRMGQNGSNSSSPGVPNGDNSRGSLVK       60 <br /> YMH71-9C selected rich medium 16&#176;C. When                             KDDPEYAEEREKMLIQIGVEADAGRSNVKV       90 <br /> scored potential pleiotropic phenotypes, 3 120 Csm+                      KDEDPNEYNEFPLRAIPKEDLENMRTHLLK      120 <br />                                                                                 FQSKKKINPVTDFHLPVRLHRKDTRNLQFQ      150 <br /> revertants Tsm-, failing grow 37&#176;C (Fig.                 LTRAEIVQRQKEISEYKKKAEQERSTPNSG      180 <br /> 2). All strains reacquired Csm- Tsm+ pheno-                       GMNKSGTVSLNNTVKDGSQTPTVDSVTKDN <br />                                                                                 TANGVNSSIPTVTGSSVPPASPTTVSAVES <br />                                                                                                                     210 <br />                                                                                                                     240 <br /> types crossed sua7-1 mutant, indicating                   NGLSNGSTSAANGLDGNASTANLANGRPLV      270 <br /> phenotypes result recessive mutations (Fig. 2). A                    TKLEDAGPAEDPTKVGMVKYDGKEVTNEPE      300 <br />                                                                                 FEEGTMDPLADVAPDGGGRAKRGNLRRXTR      330 <br /> diploid strain derived revertants (YZS14;                     QLKVLDENAKKLRFEEFYPWVMEDFDGYNT      360 <br /> Table 1) sporulated dissected. The Csm+ Tsm-                        WVGSYEAGNSDSYVLLSVEDDGSFTMIPAD      390 <br />                                                                                 KVYKFTARNKYATLTIDEAEKRMDKKSGEV      420 <br /> phenotypes cosegregated, defining Tsm- marker                      PRWLMKHLDNIGTTTTRYDRTRRKLKAVAD      450 <br /> for suppressor sua7-1.                                                   QQAMDEDDRDDNSEVELDYDEEFADDEEAP      480 <br />                                                                                 IIDGNEQENKESEQRIKKEMLQANAMGLRD      510 <br />    The Tsm- phenotype exploited clone suppressor                     EEAPSENEEDELFGEKKIDEDGERIKKALQ      540 <br /> gene. A yeast genomic library introduced strain                        KTELAALYSSDENEINPYLSESDIENKENE      570 <br />                                                                                 SPVKKEEDSDTLSKSKRSSPKKQQKKATNA      600 <br /> YZS14, transformants selected 37&#176;C. Two classes                     HVHKEPTLRVKSIKNCVIILKGDKKILKSF      630 <br /> of transformants identified. Consistent antici-                   PEGEWNPQTTKAVDSSNNASNTVPSPIKQE      660 <br />                                                                                 EGLNSTVAEREETPAPTITEKDIIEAIGDG      690 <br /> pated phenotypes wild-type suppressor gene, <span id='am-38' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-39' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span>                   KVNIKEFGKFIRRKYPGAENKKLMFAIVKK <br />                                                                                 LCRKVGNDHMELKKE* <br />                                                                                                                     720 <br />                                                                                                                     735 <br /> of Tsm+ transformants reacquired Csm- phenotype <br /> associated sua7-1. When cured plasmid DNA,             FIG. 3. The Ssu71 deduced amino acid sequence (single letter <br /> strains reacquired Tsm- Csm+ phenotypes, indicating        code). The asterisk signifies C terminus. <br /> that conditional phenotypes transformants a <br /> consequence plasmid DNA, strain reversion.          structural characteristics. Most strikingly, Ssu71 <br /> Plasmid DNA recovered transformants,            RAP74 highly charged, consisting 34% 38% D, <br /> subsequent genetic analysis established wild-type         E, R, K residues, respectively. Neither protein contains <br /> suppressor gene cloned (see Materials          structural features commonly associated transcrip- <br /> Methods). Plasmid pM435 restored Tsm+ Csm- phe-            tion factors, including leucine-zipper zinc-finger motifs [the <br /> notypes YZS14 YZS45, indicating         proteins include (RAP74) (Ssu7l) cys- <br /> strains compensate sua7-1 defect consequence       teine residues]. Both proteins contain multiple potential <span id='am-92' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>- <br /> mutations gene. This gene designated SSU71         phorylation sites cAMP-dependent protein kinases and <br /> (suppressor sua7, gene 1), suppressor alleles      protein kinase C, sites concentrated the <br /> designated ssu71-1 (YZS14) ssu71-2 (YZS45) (Table 1).          C-terminal half protein (41). This information suggests <br />    The second class Tsm+ transformants remained Csm+.           functional relationship Ssu71 RAP74. <br /> Analysis plasmid DNA recovered transformants            SSU71 Encodes p105 Subunit Yeast TFIIF. Factor g <br /> revealed restriction pattern characteristic SUA 7 gene.   reported yeast counterpart human TFIIF is <br /> This suggested Tsm- phenotype conferred ssu71       composed polypeptides apparent molecular <br /> dependent sua7-1 mutation. This confirmed             masses 30, 54, 105 kDa (23). The genes encoding these <br /> genetically. Whereas Tsm-:Tsm+ segregated 2:2            subunits recently isolated sequenced. The <br /> meiotic progeny cross YZS14 sua7-1 mutant <br /> (sua7/sua7 SSU71 /ssu 71), Tsm -:Tsm+ segregated over-       deduced N termini (14 residues) TFG1-encoded p105 <br /> all ratio 1:3 cross YZS14 wild-type strain       subunit Ssu71 protein compared; se- <br /> (SUA7/sua7 SSU71/ssu7l). The dependency Tsm-                quences identical (42). Subsequent compar- <br /> phenotype associated ssu71 sua7-1 allele implies     ison entire sequences confirmed Ssu71 p105 are <br /> functional relationship TFIIB product           identical (42). Thus, SSU71 identical TFG1 and <br /> SSU71.                                                             encodes largest subunit yeast TFIIF. The discrepancy <br />    The SSU71 Sequence. The DNA sequence SSU71 iden-             deduced (82 kDa) apparent (105 kDa) <br /> tified open reading frame encoding 735 amino acids,             molecular masses Ssu71/p105 protein presumably due <br /> corresponding protein molecular mass 82,194         anomalous migration SDS/PAGE. Highly charged pro- <br /> daltons (Fig. 3). Consistent size coding region,   teins known exhibit anomalously slow migration in <br /> RNA (Northern) blot analysis detected single SSU71 tran-         SDS/PAGE (43), RAP74 protein, a <br /> script 2.3 kb (data shown). The Ssu71 amino acid            calculated molecular mass 58 kDa apparent molec- <br /> sequence compared protein sequence data banks.          ular mass 74 kDa (41, 44). Although slower mobility of <br /> The significant similarities RAP74 subunit     RAP74 consequence posttranslational modifi- <br /> human, Drosophila, Xenopus TFIIF. The similarity            cation, recombinant RAP74 synthesized <span id='am-59' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> Ssu71 RAP74 evident considering              migrates 74 kDa (44). <br />  <br />                                                                                           FIG. 2. Phenotypes associated sua7 <br />                                                                                        ssu7l mutants. Growth strains <br />                                                                                        YMH71-9C (sua7-1 SSU71), YZS14 (sua7-1 <br />                                                                                        ssu71-1), diploid strain (2N) derived from <br />                                                                                        cross YZS14 YDW546 (sua 7-1 SSU71) <br />                                                                                        shown yeast extract/peptone/dextrose <br />                                                                                        (YPD) reduced (16&#176;C), normal (30&#176;C), and <br />                                                                                        elevated (37&#176;C) temperatures. YZS14 se- <br />                                                                                        lected spontaneous revertant of <br />                                                                                        YMH71-9C YPD medium 16&#176;C. Whereas <br />                                                                                        YMH71-9C phenotypically Csm- Tsm+, <br />                                                                                        YZS14 Csm+ Tsm-. Both Csm+ and <br />                                                                                        Tsm- result recessive mutation(s), <br />                                                                                        Csm- Tsm+ phenotypes are <br />                                                                              37 0      restored YZS14 backcrossed strain <br />                                                                                        YDW546. <br />  3130           Genetics: Sun Hampsey                                                                    Proc. Natl. Acad ScL USA 92           (1995) <br />    SSU71 Essential Cell Viability. The SSU71 open                                                            DISCUSSION <br /> reading frame disrupted insertion LEU2 gene <br /> into BamHI site (Fig. 1), resulting construct                             The data presented paper establish functional inter- <br /> used disrupt allele SSU71 homozygous leu2-                                 action TFIIB largest subunit TFIIF in <br /> diploid strain. Upon sporulation dissection, viable-to-                               yeast. First, SSU71 gene, encodes largest subunit <br /> inviable spores segregated ratio 2:2 (14 tetrads),                        TFIIF, uncovered suppressor Glu-62 -&gt; Lys <br /> viable spores phenotypically Leu- (data shown).                                  replacement (sua7-1) TFIIB. Second, sua7 ssu7l double <br /> Four-spore viability restored diploid strain                             mutants exhibit synthetic phenotype: double sua7 ssu7l <br /> transformed plasmid pM435 (SSU71) prior dissection.                               mutants heat-sensitive, sua7 SSU71+ nor <br /> These experiments define SSU71 essential gene.                                      SUA7+ ssu7l single mutants exhibit phenotype 37&#176;C. <br />    The ssu71 Suppressor Restores Normal Start-Site Selection                              Finally, distinct downstream shift transcription start-site <br /> in sua7 Background. The sua7-1 mutation confers marked                               selection cycl locus associated sua7-1 allele is <br /> effect transcription start-site selection. Since Csm-                              fully suppressed ssu71-1 allele, resulting normal <br /> phenotype start site effects consequence                            pattern initiation cycl. <br /> mutation (10, 19), asked ssu71-1 suppressor                                   Glu-62 invariant residue phylogeneti- <br /> also compensates start-site shift. Restoration                             cally conserved region TFIIB. This region adjacent <br /> normal initiation pattern ssu7l strains anticipated                            zinc-finger motif included proteolytically <span id='am-51' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>- <br /> based iso-1-cytochrome c phenotypes. These strains                               ceptible domain TFIIB required recruitment of <br /> carry cycl-5000 allele, result aberrant                            pol II-TFIIF promoter-TBP-TFIIB complex for <br /> ATG translation start <span id='am-46' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> [upstream ATG (uATG), posi-                                   transcription initiation (14-18). TFIIF thought mediate <br /> tion -20] transcribed leader region cycl-SOOC (10).                             interaction pol II promoter-TBP-TFIIB <br /> This mutation diminishes iso-1-cytochrome c synthesis &lt;2%                              complex function attributed specifically to <br /> of normal. The sua7-1 mutation partially suppresses defect                           RAP30 subunit (27). Consistent premise, TFIIB <br /> by shifting transcription initiation downstream uATG,                              binds RAP30, interaction mapped the <br /> enhancing iso-1-cytochrome c level &quot;30% normal                                  N-terminal domain TFIIB (18). However, suppression the <br /> (10). However, iso-1-cytochrome c level restored                               Glu-62 -&gt; Lys replacement yeast TFIIB mutation the <br /> &lt;2% normal ssu7l mutants (Table 1), suggesting                             gene encoding homolog RAP74 suggests interaction <br /> initiation shifted upstream uATG.                                         TFIIB TFIIF involves subunits TFIIF. <br />    An effect ssu7l start-site selection confirmed                            However, currently evidence direct interaction <br /> mapping transcription start sites cycl gene primer                              TFIIB RAP74 Ssu71/p105; efforts to <br /> extension analysis. Consistent previous results, sua7-1                              characterize interaction TFIIB Ssu71/p105 <br /> enhanced..initiation downstream sites, including discrete                              hampered toxicity associated expression <br /> sites positions -28 -14 (Fig. 4, lanes 1 vs. 2). The                          SSU71 derivatives E. coli (unpublished observation). <br /> normal initiation pattern restored ssu71-1 mutant                                 Activities attributed TFIIF reminiscent those <br /> (Fig. 4, lane 3), effect cosegregated ssu71-1 (Fig.                         assigned ur70 <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> bacterial transcription factors, <br /> 4, lane 4). Thus, ssu71-1 does simply compensate Csm-                             including tight binding pol II, suppression nonspecific <br /> but influences start-site selection, presumably through <br /> interaction, direct indirect, TFIIB.                                       binding pol II free DNA, stabilization the <br />                                                                                           preinitiation complex (45). Furthermore, limited sequence <br />                                       1 2 3 4 A C G T                                     similarities a.70 noted RAP30 and <br />                                                                                           RAP74 (46-48). Comparison Ssu71/p105 RAP74 <br />                                                                                           sequences E. coli a70 revealed regions sequence <br />                                                                                           similarity corresponding o,70 RpoD box, straddles <br />                              -464o <br />                                                                                           homology regions 2.3 2.4, homology region 4.2 (Fig. <br />                                                                                           5). This observation particularly noteworthy regions <br />                                                                                           2.4 4.2 involved recognition -10 -35 <br />                                                                                           regions, respectively, bacterial promoter (49). The <br />                                    0. <br />                                  -28 <br />                                                                                           question functional significance these <br />                                                                                           sequence similarities readily investigated mu- <br />                                                                                           tational analysis cloned SSU71 gene. <br />                                                                                                                   RpoD box <br />                                                                                                 421     EYRRGYKFST---YATWWIRQAITR           a70 <br />                                                                                                           :: III::   III I::I <br />                              -14      40      - <br />                                                                                                 388     PADKVYKFTARNKYATLTIDEAEKR           Ssu7l/p105 <br />                                                                                                         1::: 1:11: :P111::111:: <br />                                                                                                  13 9   PVHNWYNFTPLARHRTLTAEEAEEE           RAP7 4 <br />  <br />                                                                                                                region 4. 2 <br />                                                                                                 572     TLEEVGKQFDVTRERIRQIEAKALRK a70 <br />                                                                                                          I : <br />                                                                                                         11:              :   1: @ <br />   FiG.   4.    Primer extensionanalysis cycl transcription start sites.                       607    TLRVKSIKNCVIILKGDKKILKSFPE Ssu7l/p105 <br /> Lanes: 1, T16  (SUA7SU7J); 2, YMH71-9C (sua7-l SSU71); 3, YZS14                                         III:         I: 1:: <br /> (sua7-J ssu7l-1); 4, YZS14-3B (sua7-l ssu7l-1). Strain YZS14-3B                            400     TLRAAASK-----LEQGKRVSEM-PA RAP74 <br /> segregant backcross YZS14 YDW546 (Table 1). Tran- <br /> scription start sites numbered relative ATG translation start                     FIG. 5. Alignment E. coli oJ70, yeast Ssu71/p1O5, human <br /> codon. The sequence ladder (A,C,G,T) corresponds SUA7 DNA                              RAP74. Sequence identity similarity indicated vertical lines <br /> and       used          molecular   size   marker.   Whereas   initiation     colons, respectively. Similarity rules PrA==S, G=A=S, <br /> enhanced downstream                sites sua7 mutant relative            T=A=S, D-E==N=Q=K=R=H, A=-V=I=L, F=Y=W. <br /> wild-type       (compare lanes 1 2), normal initiation pattern <br />               strain                                                                      The E. coli RpoD box underlined encompasses portions Co70 <br /> is restored     isogenie sua7 ssu7l strain (lane 3); effect <br />                                                                                       regions 2.3 2.4. The entire region 4.2 a70 underlined. The <br /> cosegregates ssu7l marker (lane 4).                                              similarity co70 RAP74 noted previously (48). <br />            Genetics: Sun    Hampsey                                                   Proc. Natl. Acad Sci. USA 92 (1995)              3131 <br />  <br />    The mechanism transcription start-site selection           17. Hisatake, K., Roeder, R. G. &amp; Horikoshi, M. (199 <br /> </body></html>