<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Chu et al. BMC Genomics 2014, 15:361 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> High-throughput capturing characterization <br /> of mutations essential genes <span id='am-27' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-28' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> <br /> elegans <br /> Jeffrey Shih-Chieh Chu1*, Shu-Yi Chua2, Kathy Wong1, Ann Marie Davison3, Robert Johnsen2, David L Baillie2 <br /> and Ann M Rose1 <br />  <br />  <br />   Abstract <br />   Background: Essential genes critical development organisms associated many <br />   human diseases. These genes difficult category study prior availability balanced lethal <br />   strains. Despite power targeted mutagenesis, limitations identifying mutations essential genes. <br />   In paper, identification coding regions essential genes mutated using forward genetic <br />   screens <span id='am-20' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span>. The lethal mutations described isolated maintained wild-type <br />   allele rescuing duplication. <br />   Results: We applied genome sequencing identify causative molecular lesion resulting lethality in <br />   existing C. elegans mutant strains. These strains balanced easily maintained subsequent <br />   characterization. Our method effectively used analyze mutations large number essential genes. <br />   We identification 64 essential genes region chromosome I covered duplication <br />   sDp2. Of these, 42 nonsense mutations, splice signal mutations, deletion, 15 non-synonymous <br />   mutations. Many essential genes region function cell cycle, transcriptional regulation, RNA processing. <br />   Conclusions: The essential genes identified represented mutant strains, than <br />   mutant allele. The genetic resource utilized understanding essential gene function will <br />   applicable study C. elegans development, conserved cellular function, ultimately lead improved <br />   human health. <br />   Keywords: Whole genome sequencing, EMS, Mutagenesis, Essential genes, Balanced mutation, Lethal mutation, <br />   C. elegans, Cell cycle <br />  <br />  <br /> Background                                                                           genes, generating lethal alleles model systems is <br /> The proper development viability organism                               required. In nematode Caenorhabditis elegans, the <br /> dependent group genes called essential genes. In                             essential gene set largest set genes esti- <br /> humans, gene essentiality long associated                              mated contain 25% genes [6-9]. Using RNAi, <br /> many diseases miscarriages [1,2], heritable dis-                             3500 genes annotated essential (<span id='am-96' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> <br /> eases, <span id='am-15' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-16' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cancer</span> [3]. Recent studies shown                                collected WormBase [10,11]). Inparanoid, se- <br /> over-expression essential genes promotes cell                                quence based orthology inference tool, detects about <br /> proliferation cancer [4]. Due importance sur-                          40% C. elegans genes orthologous hu- <br /> vival, essential genes targets new therapeu-                           man genes. But approximately 60% essential genes <br /> tics antimicrobials [5]. To effectively study essential                           clear human orthologs, showing high conservation <br />                                                                                      essential genes, makes C. elegans excellent <br />                                                                                      platform examination essential gene functions that <br /> * Correspondence: jeff.sc.chu@gmail.com <br /> 1 <br />  Department Medical Genetics, University British Columbia, Vancouver, <br />                                                                                      relevant human health. Many important genes, <br /> Canada                                                                               let-60/Ras [12] let-740/dcr-1 [13,14], were <br /> Full list author information available end article <br />  <br />                                         &#169; 2014 Chu et al.; licensee BioMed Central Ltd. <span id='am-37' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                         unless stated. <br />  Chu et al. BMC Genomics 2014, 15:361                                                                               Page 2 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> first discovered C. elegans genetics. However,       established mutagenesis technique latest sequen- <br /> genetic resource studying genes severely lack-      cing technology <span id='am-67' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> close gap essential <br /> ing. Even concerted community effort        gene knock-out collection. <br /> C. elegans Deletion Mutant Consortium [15], mutations in <br /> many essential genes lacking knock-out col-     Results discussion <br /> lection. The consortium generated close 6000             Chromosome I left high percentage essential genes <br /> knock-out strains 1998, 1436 essential            The leftmost 7.3 Mbp chromosome I highest <br /> genes current collection [16,17]. In addition      percentage mapped essential genes closest satur- <br /> the considerable time effort required generate          ation 237 essential genes isolated mapped [19]. <br /> single knock-out allele, outstanding disadvantage      The mutant strains derived mutagenizing KR235 <br /> targeted deletion approach extra effort needed     [dpy-5 (e61), +, unc-13 (e450)/dpy-5(e61), unc-15(e73), +; <br /> balance lethal mutation [18]. Recently, Consortium       sDp2] low dose EMS isolating let-x dpy-5 <br /> has adopted procedure random mutagenesis followed           unc-13 homozygotes rescued wild-type allele <br /> by genome sequencing (WGS) generate                 dpy-5 let-x balanced free duplication sDp2 [35] <br /> identify large number mutations [15]. Although         (see schematic Additional file 1). In order position <br /> project generate mutations shorter time,       genes, mutations mapped 60 zones using a <br /> method does capture mutations exhibit lethal            combination three-factor mapping complementa- <br /> phenotypes, thus, essential genes selected against.      tion series duplications deficiencies [19]. <br /> This outcome indicates thousands essential genes           Within zones, lethal mutants inter-complementation <br /> not knockout alleles.                                       tested. The earliest developmental arrest stages de- <br />    To complement effort C. elegans community,         termined complementation group [19]. The can- <br /> we took advantage balancer system,              didate lesions present copies rescued a <br /> developed 30 years ago capturing maintaining le-         wild-type allele sDp2. Thus, high throughput <br /> thal mutations, next-generation DNA sequencing          identification method focused finding heterozygous <br /> technologies. Almost 70% C. elegans genome           mutations exhibit allelic ratio range of <br /> been successfully balanced large genomic rearrange-           40% 90% [30]. In order assess accuracy re- <br /> ments [18]. By mutagenizing pre-balanced strain removes        cently developed high throughput method [30], se- <br /> the need perform additional genetic crosses balance      lected 81 genes set criteria they <br /> lethal mutation. The balancer system, designed specifically      formed complementation group having one <br /> to capture maintain lethal mutations,       allele (Additional file 2). The extra alleles provide added <br /> choice generating mutations essential genes. Such         resource validation. We sequenced 10 indexed gen- <br /> screens carried regions chromosome I        omic DNA samples Illumina HiSeq lane obtained <br /> [19,20], II [21], III [22], IV [23-26], V [27], X [28,29].   total 385 Gbp sequence. The sequencing reads <br /> In laboratories, generated 1350 lethal          aligned using BWA [36] WS200 C. elegans ref- <br /> mutations fall 486 complementation groups.             erence sequence. We achieved 30X coverage average <br />    The hurdle analysis essential genes     genome average 35X coverage <br /> molecular identification genomic lesion,         coding elements. In case strains, 6X <br /> date involved enormous effort. Traditional methods        coverage obtained: let-369(h125) let-594(h407). <br /> of gene cloning rely candidate identification       Genomes strains removed sub- <br /> mapped mutations months years. This gene-            sequent analysis. <br /> by-gene approach able characterize 30 essen- <br /> tial genes library date. This problem       The mutational landscape provided quality check <br /> difficult solve recent advances sequencing       Our analytical step, quality check, con- <br /> technology. To address problem coding region              firm presence dpy-5 (e61) unc-13 (e450) <br /> identification, recently developed fast scal-      mutations genome. For unc-13, expected <br /> able pipeline takes advantage genome se-           variant ratio 100% duplication <br /> quencing bioinformatics analysis identify causal      does extend far provide additional <br /> mutation responsible lethal phenotype [30]. Recent       wild-type allele. For dpy-5 however, wild-type <br /> studies, including initial analysis let-504 [30],    allele sDp2, expect 66% <br /> shown genome sequencing efficient           variant ratio. We expected ratios 76 the <br /> cost-effective approach identifying encoded gene          79 genomes. Three genomes deviated norm: <br /> product especially additional alleles        let-516(h144) missing e450 e61 (all reads <br /> can sequenced provide confirmation [30-34]. In             supported reference sequence); let-388(h88) miss- <br /> this paper, approach combining             ing e61; let-393(h225) e61, 33% ratio <br />  Chu et al. BMC Genomics 2014, 15:361                                                                            Page 3 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> rather expected 66%. We examined                For remaining 28 genomes, high confidence <br /> strains presence duplication sDp2. When         identity 22 genes based map position. <br /> the duplication present, read depth 33% greater      Thus, including previously described let-504, now <br /> in 7Mbp chromosome I rest         coding region assignments 64 let- genes the <br /> chromosome. Our analysis showed             sDp2 region. Because genes study have <br /> three genomes showed depth difference (Additional          multiple alleles, inference, confidently <br /> file 3). It likely strains carry sDp2. Al-     identified coding regions affected total 259 <br /> though sDp2 does crossover normal <span id='am-42' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="Homo" datatype="xsd:string"></span>homo</span>-           mutant strains (Additional file 2). <br /> logs readily detectable frequency, rare exchange            Seven genes molecularly identified <br /> events occur resulting subsequent loss du-       phenotypically described. let-603, aurora kinase <br /> plicated fragment [37]. These strains ana-      [46], let-605, cyclin E, severe gonadal defects <br /> lyzed further.                                                 [56]. let-355, DEAD box helicase, let-384, inte- <br />                                                                grator subunit, failed develop gametes [56]. let-370, let- <br /> Coding sequence correlated high confidence                599, let-604 produced malformed embryos were <br /> We analyzed parental strain KR235 identified           laid hatched [56]. let-370 encodes hexaprenyl <br /> 571 SNVs 167 small indels &gt;40% read              pyrophosphate synthetase associated Parkin- <br /> support Chromosome I compared C. ele-           son&#8217;s disease [51]. let-599 encodes N-acetyl transferase <br /> gans WS200 reference using VarScan (see Methods).              nath-10. let-604 encodes mdt-18, mediator subunit. A <br /> These mutations represent background mutations          comprehensive summary let- encoded products is <br /> which lethal mutations maintained. For            given Table 1. <br /> remaining 76 genomes, filtered background <br /> mutations average 44 SNVs &gt;40%          Novel knock-out alleles provide new genetic resources <br /> allelic ratio sDp2 region. Most SNVs         We generated new alleles 13 genes currently <br /> G &gt; A C &gt; T changes expected previously              knock-out alleles available: let-595 (imb-1), let- <br /> observed EMS treatment [30,38]. We            362 (Y71G12B.8), rnp-6 (let-147), aars-2 (let-366), let-598 <br /> an average 7 small indels 1&#8211;2 bps. We categorized        (F27C1.6), let-355 (T05E8.3), let-384 (C06A5.1), fars-1 <br /> each mutation nonsense, missense, <span id='am-47' about='ncbitaxon:synonym' typeof='owl:Thing'>synonym</span>-           (let-396), let-611 (C48E7.2), mdt-18 (let-604), acdh-5 (let- <br /> ous, splice signal disruption, frame shifting indel, frame     383), rpb-5 (let-397), let-630 (Y110A7A.19). Eight of <br /> preserving indel, noncoding mutation. Noncoding             genes predicted roles essential basic <br /> mutations defined mutation located outside         functions transcription translation. This not <br /> of coding regions. A list SNVs indels,    surprising, expect genes function basic <br /> strain, available server http://lethal.mbb.sfu.   cellular processes essential best captured <br /> ca/jschu/essential_genes.                                      using balancer systems. Besides novel alleles, we <br />   We identified candidate mutations 76 genomes         provided additional loss function alleles many <br /> using bioinformatics pipeline developed pre-       characterized genes (Table 1). Additional alleles affect- <br /> viously [30] (see Methods) validated subset      ing different parts gene disrupt different <br /> our candidates sequencing second allele com-        domains providing allelic series correlating dif- <br /> plementation testing (Table 1). Nine candidate le-      ferent phenotypes. <br /> sions genes previously identified           Genetic strains carrying heritable mutational changes <br /> and published. In cases, candidates expected       provide lasting resource used variety <br /> in separate genes located coding re-          experimental conditions compared information <br /> gion. These observations confirmed gen-        gained RNAi knock-down experiments. We cross- <br /> etic complementation tests (Additional file 4). Previously     checked high confidence list RNAi data an- <br /> identified let-631 let-103 allelic     notated WormBase lethal phenotype was <br /> let-363. As result, let-363 gains new sDp2-            observed RNAi experiments. Although <br /> balanced alleles (h216, h451, h502) addition         part, RNAi data agrees mutational <br /> nine existing ones. let-519 let-104 allelic let-    data, gene supported RNAi. We found <br /> 526 let-526 gains new alleles: h799, h373,       genes showing lethal phenotype RNAi and <br /> h405 h526. let-630 fails complement let-596         genes showing lethal phenotype variable <br /> now alleles: h355, h702, h432, h782, h258.        penetrance (Table 1). Of genes no <br /> Thirty-five candidates tested sequencing sec-        RNAi lethal phenotype, (inx-12, coq-1, lim-7, tag- <br /> ond allele using previously published complementation          146, let-381, let-503) additional knock-out <br /> data [19]. Of these, confirmed 29 identities. All        alleles lethal, suggesting RNAi did reveal the <br /> all, tested 48 candidates confirmed 42 (87.5%).         null phenotype genes. The additional information <br />                                                                                                                                                                                        http://www.biomedcentral.com/1471-2164/15/361 <br />                                                                                                                                                                                       Chu et al. BMC Genomics 2014, 15:361 <br /> Table 1 Coding DNA Sequence (CDS) identifications let- genes <br /> Gene              Allele Allele mutation Molecular identity                       Support   Confirmation status   Human ortholog Associated human conditions             References <br /> lin-6/mcm-4       h92    C&gt;*              Mini chromosome maintenance             RNAi      Confirmed1            MCM4            Natural killer cell glucocorticoid [39] <br />                                                                                                                                   deficiency DNA repair defect <br /> let-354/dhc-1     h79    Q&gt;*              Dynein heavy chain                      Both      Confirmed1            DYNC1H1         Charcot-Marie-Tooth disease, Mental [40-42] <br />                                                                                                                                   retardation, Spinal muscular atrophy <br /> let-502/rock      h392   Q&gt;*              Rho associated kinase                   RNAi      Confirmed1            ROCK1                                                  [43] <br />                                                                                                      1 <br /> let-363/<span id='am-86' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90'  ></span>tor</span>       h98    Splice variant   Tor kinase                              Both      Confirmed             MTOR            pancreatic neuroendocrine tumors       [44,45] <br />                   h420a Q &gt; *                                                               Confirmed3 <br />                   h502a Splice variant                                                      Confirmed3 <br /> let-603/air-2     h289   W&gt;*              Aurora-related serine/threonine         Both      Confirmed1            AURKA           Susceptibility colon cancer         [46] <br />                                           kinase <br /> let-512/vps-34    h797   P&gt;S              phosphoinositide 3-kinase                         Confirmed1            PIK3C3                                                 [47] <br />                                                                                                      1 <br /> let-381/foxf      h107   splice variant   Forkhead transcription factor F         K.O.      Confirmed             FOXF2                                                  [48] <br /> let-607/bZip      h402   Q&gt;*              Leucine zipper transcription factor     Both      Confirmed1            CREB3L3                                                [49] <br /> let-504/E01A2.4   h448   M&gt;I              NFkB activating protein                 Both      Confirmed1            NKAP <br /> let-152/ccb-1     h685   W&gt;*              Calcium channel subunit                           Confirmed2            CACNB2          Brugada syndrome 4 <br />                                                                                                      2 <br /> let-355/hel/      h81    Y&gt;*              DEAD/H helicase                         RNAi      Confirmed             DHX33 <br /> T05E8.3 <br /> let-362/rhel/     h86    R&gt;*              DEAD/H RNA helicase                     RNAi      Confirmed2            DDX27 <br /> Y71G12B.8 <br /> let-366/aars-2    h112   Q&gt;*              Alanine tRNA synthetase                 RNAi      Confirmed2            AARS            Charcot-Marie-Tooth disease            [50] <br />                                                                                                      2 <br /> let-368/inx-12    h121   W&gt;*              Innexin gap junction                    K.O.      Confirmed <br /> let-370/coq-1     h128   G&gt;E              hexaprenyl pyrophosphate                K.O.      Confirmed2            PDSS1           Coenzyme Q10 deficiency, Parkinson&#8217;s [51] <br />                                           synthetase                                                                              disease <br /> let-389/nars-1    h680   G&gt;E              Asparagine tRNA synthetase              Both      Confirmed2            NARS <br /> let-396/fars-1    h217   Q&gt;*              Phenylalanine tRNA synthetase           RNAi      Confirmed2            FARSA <br />                                                                                                      2 <br /> let-522/hlh-2     h735   W&gt;*              <span id='am-32' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-33'  ></span><span id='am-34' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Helix</span> loop helix transcription factor   Both      Confirmed             TCF3            Acute lymphoblastic leukemia <br /> let-529/asd-2     h238   Q&gt;*              KH domain containing RNA                RNAi      Confirmed2            QKI             Mental retardation <br />                                           binding protein <br /> let-575/ptr-2     h345   W&gt;*              Sterol sensing domain protein           RNAi      Confirmed2            PTCHD1          Autism spectrum disorders              [52-54] <br />                                                                                                      2 <br /> let-585/inx-13    h784   W&gt;*              Innexin gap junction                    RNAi      Confirmed <br /> let-595/imb-1     h353   R&gt;*              Importin                                RNAi      Confirmed2            KPNB1 <br />                                                                                                      2 <br /> let-598/F27C1.6   h213   Q&gt;*              U3 small nucleolar                      RNAi      Confirmed             UTP14C <br />                                           ribonucleoprotein <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                Page 4 13 <br /> let-599/nath-10   h290   L&gt;F              N-acetyl transferase                    Both      Confirmed2            NAT10 <br />  Table 1 Coding DNA Sequence (CDS) identifications let- genes (Continued) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      http://www.biomedcentral.com/1471-2164/15/361 <br />                                                                                                                                                                      Chu et al. BMC Genomics 2014, 15:361 <br /> let-608/ncbp-1    h706       Q&gt;*              Nuclear cap binding protein           RNAi    Confirmed2       NCBP1 <br />                                                                                                          2 <br /> let-611/C48E7.2   h756       Q&gt;*              RNA polymerase III subunit            RNAi    Confirmed        POLR3C <br /> let-612/apm-1     h466       splice variant   Adaptin subunit                       RNAi    Confirmed2       AP1M1 <br />                                                                                                          2 <br /> let-365/sep-1     h108       W&gt;*              Separase                              Both    Confirmed        ESPL1          Breast cancer oncogene <br /> let-364/mat-1     h104       S&gt;F              Anaphase promoting complex            RNAi    Confirmed2       CDC27 <br />                                               subunit <br /> let-101/npp-6     h242       W&gt;*              Nuclear pore complex protein          Both    Confirmed2       NUP160 <br />                                                                                                          2 <br /> let-106/hcp-6     h787       C&gt;Y              Condensin subunit                     Both    Confirmed        NCAPD3 <br /> let-379/tag-345   h127       W&gt;*              Nucleolar protein complex             RNAi    Confirmed2       WDR12 <br />                                               member <br /> let-503/R12E2.2   h313       Q&gt;*              Protein unknown function           K.O.    Confirmed2       SUCO <br /> let-517/spg-7     h264       G&gt;E              Metalloprotease                       Both    Confirmed2       AFG3L2         Spastic ataxia, Spinocerebellar ataxia <br />                                                                                                          2 <br /> let-597/hcp-4     h349       E&gt;*              Holocentromeric protein               RNAi    Confirmed        CENPC <br /> let-630/          h355b R &gt; *                 Pentatricopeptide repeat              RNAi    Confirmed2       PTCD3 <br /> Y110A7A.19                                    containing protein <br />                   h782b W &gt; *                                                               Confirmed2 <br /> let-646/pat-10    h233       G&gt;E              Troponin C                            RNAi    Confirmed2       TNNC1          Cardiomyopathy <br />                          c <br /> let-526           h799       Q&gt;*              SWI/SNF complex subunit               Both    Confirmed3       ARID1A         Mental retardation <br />                          c                                                                               3 <br />                   h405       W&gt;*                                                            Confirmed <br /> let-129/zfh-2     h379       Q&gt;*              zinc finger homeobox protein          Both    Prediction       ZFHX3, ZFHX4   Susceptibility prostate cancer, <br />                                                                                                                             Ptosis <br /> let-147/rnp-6     h463       G&gt;E              RNA splicing factor                   RNAi    Prediction       PUF60          Verheij syndrome <br />                                                                                     range <br /> let-373/unc-73    h234       Del              Guanine nucleotide exchange factor    Both    Prediction       TRIO <br /> let-377/lim-7     h110       W&gt;*              LIM homeodomain protein               K.O.    Prediction       ISL2 <br /> let-378/dnj-21    h124       G&gt;E              DnaJ domain containing protein        RNAi    Prediction       DNAJC15 <br /> let-380/knl-2     h80        W&gt;*              Centromeric protein                   Both    Prediction <br /> let-382/nuo-2     h82        Q&gt;*              Mitochondria complex I subunit        Both    Prediction       NDUFS3         Leigh syndrome, Mitochondrial <br />                                                                                                                             complex I deficiency <br /> let-383/T21G5.6   h115       W&gt;*              Protein unknown function                   Prediction <br /> let-384/C06A5.1   h84        Q&gt;*              Integrator subunit                    RNAi    Prediction       INTS1 <br /> let-385/teg-4     h85        splice variant   splicing factor                       RNAi    Prediction       SF3B3 <br /> let-386/dbr-1     h117       G&gt;E              RNA lariat-debranching enzyme         RNAi    Prediction       DBR1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                               Page 5 13 <br />                                                                                     range <br /> let-391/tag-146   h91        Q&gt;*              Uncharacterized zinc finger protein   K.O.    Prediction <br />  Table 1 Coding DNA Sequence (CDS) identifications let- genes (Continued) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                           http://www.biomedcentral.com/1471-2164/15/361 <br />                                                                                                                                                                                                                           Chu et al. BMC Genomics 2014, 15:361 <br /> let-397/rpb-5        h228    Q&gt;*                 RNA polymerase II subunit               RNAi        Prediction                             POLR2E <br /> let-400/prpf-4       h269    D&gt;G                 Pre-mRNA processing factor              RNAi        Prediction                             PRPF4B                                                        [55] <br /> let-509/unc-73       h142    W&gt;*                 Guanine nucleotide exchange             Both        Prediction                             TRIO <br />                                                  factor <br /> let-527/nhr-23       h207    R&gt;Q                 Nuclear hormone receptor                Both        Prediction                             RORC <br /> let-534/ahcy-1       h260    Q&gt;*                 S-adenosylhomocysteine hydrolase Both               Prediction                             AHCY                 Hypermethioninemia <br /> let-581/unc-11       h725    A&gt;V                 clathrin adaptor protein                RNAi;       Prediction                             PICALM               Acute lymphoblastic leukemia, Acute <br />                                                                                          Range                                                                   T-cell lymphoblastic leukemia <br /> let-601/cuti-1       h281    Q&gt;*                 Cuticle regulatory protein              Both        Prediction <br /> let-602/T09B4.9      h283    W&gt;*                 translocase                             RNAi        Prediction                             TIMM44 <br /> let-604/mdt-18       h293    splice variant      Mediator subunit                        RNAi        Prediction                             MED18 <br /> let-605/cye-1        h312    W&gt;*                 E-type cyclin                           Both        Prediction                             CCNE1 <br /> let-614              h138                                                                            Tested F27C1.3 did <br />                                                                                                      confirm <br /> let-376              h130                                                                            Tested F55F8.3 did <br />                                                                                                      confirm <br /> let-375              h241                                                                            Tested imb-1 did not <br />                                                                                                      confirm <br /> let-387              h87                                                                             Tested pnk-1 did not <br />                                                                                                      confirm <br /> let-515              h730                                                                            Tested rpl-13 did not <br />                                                                                                      confirm <br /> let-501              h714                                                                            Tested rpl-4 did not <br />                                                                                                      confirm <br /> let-361              h97                                                                             candidate <br /> let-531              h733                                                                            candidate <br /> let-576              h816                                                                            candidate <br /> let-518              h316                                                                            candidate <br /> let-523              h751                                                                            candidate <br /> let-525              h874                                                                            candidate <br /> let-584              h743                                                                            candidate <br /> The asterisk (*) signify stop <span id='am-54' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. Support column describes CDS lethal treated RNAi knock-out (K.O.) allele, both. RNAi Range signifies RNAi lethal phenotype varying degree <br /> of penetrance. Confirmation status notes: 1Confirmed previous publication. 2Confirmed sequencing 2nd allele. 3Confirmed complementation testing. Annotation human orthologs associated human <br /> conditions literature public databases WormBase OMIM. The genes sorted confirmation status genomic coordinates. <br /> a <br />   let-103 (h420) let-631 (h502) collapsed let-363. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                    Page 6 13 <br /> b <br />   let-596 (h782) let-630 (h355) confirmed sequencing second allele failed complement other. Thus, collapsed let-630. <br /> c <br />  let-104 (h799) let-519 (h405) collapsed let-526. <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                             Page 7 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> provided genetic mutation highlights importance               metaphase anaphase [63]. lin-6 (mcm-4) required for <br /> our collection.                                                         DNA replication activates checkpoint entering <br />                                                                         M phase [39]. let-599 (nath-10) let-354 (dhc-1) <br /> Essential genes sDp2 function cell cycle                      crucial cytokinesis cell division [64,65]. let- <br /> cytokinesis, transcriptional regulation, RNA processing             385 (teg-4) component splicing complex A that <br /> To identify processes essential, investigated           functions meiosis entry decision [66,67]. Our data <br /> the function high confidence gene set                 indicate disrupting phase cell cycle process <br /> their orthologs D. <span id='am-91' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-95'  ></span>melanogaster</span> (fly), S. cerevisiae (yeast),        lead lethality. <br /> and H. sapiens (humans). Essential genes con-                   Are functions essential genes identified this <br /> served important biological roles. Fifty-four              study representative essential genes? Random <br /> of identified essential genes readily identifiable             sampling simulation 3500 essential genes indi- <br /> orthologs humans [57] (Table 1). We catego-                  cated RNAi shows different GO term distribu- <br /> rized gene functional                   tion (Figure 1). In larger set samples, observed <br /> groups based GO annotations (Figure 1). To                     cell cycle cytokinesis (p = 1.02e&#8722;22, &#967;2 test), re- <br /> have better picture roles different essential               gulation transcription (p = 2.48e&#8722;20, &#967;2 test), RNA <br /> genes, multi-functional genes categorized                processing (p = 5.43e&#8722;10, &#967;2 test) under-represented <br /> than functional group. The cell cycle &amp; cytokinesis,                compared sequenced set. Although acknow- <br /> transcriptional regulation, transport, RNA processing,              ledge comparing lethal mutants RNAi phenocopies <br /> transcription categories contained genes did                  fully equivalent, present time a <br /> the groups representing translation, signal transduction,               large mutant essential gene collection this <br /> and groups includes metabolic struc-                 comparison. It intriquing raise ques- <br /> tural processes.                                                        tion regional differences essential gene functions and <br />   Of functional groups, groups               look forward having complete dataset that <br /> that significantly enriched sDp2 region                used address issue. <br /> compared non-essential genes sDp2: cell cycle &amp; <br /> cytokinesis (p = 3.61e&#8722;9, &#967;2 test), regulation transcription         Essential gene transcripts supplied maternally <br /> (p = 6.21e&#8722;8, &#967;2 test), RNA processing (p = 6.35e&#8722;12, &#967;2            From set 59 essential genes, 34 arrest devel- <br /> test). Our analysis indicates members pro-                opment embryos early larvae, indicating they <br /> cesses enriched essential genes. We previously              important early development. To test hypoth- <br /> shown components spindle assembly check-                    esis, analyzed temporal expression genes <br /> point essential survival [58]. Here showed              using RNA-seq divided 23 separate 30-minute embry- <br /> genes sDp2 region function various phases              onic stages, 4 larval stages, pre-gravid young adult stage, <br /> cell cycle. For instance, let-380 (knl-2) critical load-         young adult stage. The normalized RNA-seq data <br /> ing hcp-3 (CENP-A) chromatin forming kineto-                 obtained modENCODE project [68,69]. <br /> chore [59]. let-603 (air-2), let-597 (hcp-4), let-106                 Seven distinct patterns seen heatmap <br /> (hcp-6) remove cohesions proper resolution cen-                  (Figure 2). Five genes (colored red) express highly during <br /> tromeric connections segregation homologous                      mid-embryonic stage (300 min &#8211; 600 min), genes (col- <br /> chromosomes meiosis [60-62]. let-365 (sep-1) es-              ored blue) express highly late-embryonic stage <br /> sential chromatid separation proper anaphase. In                (600 min &#8211; hatch), seven genes (colored green) ex- <br /> addition, let-364 (mat-1), member anaphase pro-                press highly mid-embryonic late-embryonic <br /> moting complex (APC), crucial transition                stages. Eighteen genes (colored purple) elevated <br />  <br />  <br />  <br />  <br />  Figure 1 Functional categorization essential genes identified study using GO terms. The Y-<span id='am-59' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-60' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span> indicates GO term categories. <br />  The X-axis represents number genes category. Random sampling 1000 iterations selecting equal number genes <br />  sDp2 genes set essential genes identified RNAi. Error bars represent standard error. <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                                  Page 8 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br />  Figure 2 This figure represents normalized transcript level (read number coding length million reads) gene across <br />  developmental stages including 23 embryo stages separated 30 minute interval, larval stages (L1-L4), pre-gravid young <br />  adult, gravid young adult. For comparing germline expression, we&#8217;ve included transcript level JK1107 carrying mutation in <br />  glp-1, essential mitotic germ cell proliferation [71]. The heatmap represents normalized transcript level high (yellow) low <br />  (blue). Seven distinct clusters based expression pattern shown colored branches. Purple: early-embryonic; Brown: <br />  early- mid-embryonic; Red: mid-embryonic; Blue: late-embryonic; Green: mid- late-embryonic; Orange: gastrulation; Black: larval. <br />  <br />  <br /> expression early embryonic development (0 min &#8211;                      From RNAseq data, observed 18 genes ex- <br /> 300 min). Most genes, however, dramatic                     pression patterns indicated maternal contribution <br /> drop expression level 150 min, gas-                    early embryogenesis. This ratio significantly <br /> trulation occurs [70]. Observing genes                  different set essential genes. However, when <br /> also strong expression young adults lar-                compared set non-essential genes, essential <br /> val stages suggests messages highly tran-                   gene list significantly enriched genes strong ma- <br /> scribed germline likely maternally derived                  ternal contribution (1.24e&#8722;5, &#967;2 test). These data indicate <br /> in embryo. On hand, genes (colored                      essential genes important early embryonic <br /> brown) early embryonic expression                       development maternal contribution. <br /> their strongest expression mid-embryonic stages. A <br /> group genes (colored orange) specific expres-                 Conclusions <br /> sion gastrulation. Lastly, genes (colored black)              The function essential genes poorly understood. <br /> have elevated expression specific larval stages.                    Having combination genetic strains the <br />  Chu et al. BMC Genomics 2014, 15:361                                                                               Page 9 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> molecular identity known provide powerful re-         analyzed group better understand functions <br /> source study. However, model        living multi-cellular animal needs survival. Under- <br /> C. elegans, 25% essential genes         standing function individual essential genes ap- <br /> knockout alleles. RNAi used identify essen-     plications medicine. Essential genes bacteria have <br /> tial genes [72,73]. Despite success studies,   exploited develop new antimicrobials [5]. An un- <br /> a small subset (~800 genes) profiled phenotypic-       derstanding essential genes exploited new <br /> ally [72]. We large collection mutant strains,     medical uses. For example, human ortholog let-400/ <br /> only technically feasible easily identify     prpf-4, induce G1/S arrest may <br /> their corresponding coding regions. Our library currently        function cancer suppressor [55]. Therefore, resource <br /> consists 1350 lethal mutations maintained balancers        described identifying studying essen- <br /> in chromosomes I, III, IV, V, chromosome I          tial genes model organisms direct benefit. <br /> is closest saturation [19]. Recent genome              We shown essential genes left half of <br /> screening experiments using CRISPR/Cas9          chromosome I C. elegans function cell cycle control, <br /> opened possibility identifying essential genes         transcriptional regulation, RNA processing. Previous <br /> using targeted approach. However, targeted approaches       reports studying genomic regions shown differ- <br /> directed identifying essential genes intact        ent gene classes regulated GATA tran- <br /> multicellular organism limited terms recovery    scription factor [74] sex-regulated genes [75] are <br /> and maintenance lethal mutations impractical          non-randomly distributed genome. Thus, believe <br /> large scale screens. The relative ease capturing          organization genes genome also <br /> maintaining lethal mutations makes balancer systems          non-random. With method, possible gener- <br /> method choice essential gene studies. However, using      ate genetic resources capture majority essen- <br /> random mutagenesis possible achieve 100% sa-           tial genes. The study provide global <br /> turation (finding essential genes). Small targets     picture minimum set genes pathways is <br /> smaller chance mutated likely missed         needed survival multi-cellular organism, and <br /> mutagenesis experiments. Also, finding new essential genes       organization genome. An increased under- <br /> in subsequent screenings more difficult         standing nature essential genes relevant only <br /> because screens follow (approximately) Poisson distri-     knowledge biological survival organism <br /> bution giving diminishing returns. Thus, combination        potential better medical procedures. <br /> targeted forward mutational approaches best. <br />    We previously developed pipeline applied          Methods <br /> the identification let-504 [30]. In analysis pre-         Strains <br /> sented here, applied pipeline analyze          The strains used study listed Table 1. We <br /> 76 essential genes Chromosome I produced high             listed available alleles let- <br /> confidence identification 64 genes. Some con-         gene Additional file 2. The strains grown and <br /> firmed candidates outside mapped re-              maintained nematode growth medium streaked with <br /> gion suggesting boundaries genetically           E. coli OP50 [76]. The strains used study were <br /> identified zones refined. We shown           generated mutagenizing KR235 [dpy-5 (e61), +, unc- <br /> that approach efficient cost-               13 (e450)/dpy-5(e61), unc-15(e73), +; sDp2] 12 mM <br /> effective traditional method. Assessments          EMS [35]. Briefly, treated gravid wildtypes indi- <br /> this study help improve identification pipe-         vidually plated 5 cm plates wildtype gravid F1s <br /> line confidence apply technique          individually plated 5 days later. Their progeny <br /> to rest collection essential genes.                (F2s) screened absence Dpy-5 Unc-13 in- <br />    Our results provide additional alleles known          dividuals (Additional file 1). A single Unc-13 animal was <br /> genes provide new alleles. The added alleles          transferred confirm existence lethal mutation. <br /> will valuable establishing allelic series        A balanced lethal exhibit Unc-13 develop- <br /> exhibit different phenotypes. For instance let-147/rnp-6         mentally arrested Dpy-5 Unc-13 [35]. All strains <br /> has 4 alleles showing different arrest stage [19],        maintained 20&#176;C selecting Unc animals. <br /> suggesting different protein domains disrupted.        Each strain grown hermaphrodite ex- <br /> More importantly, results provided 13 new alleles         panded 20 2-inch plates. The worms collected <br /> essential genes alleles existed. The genetic re-        rinsing plates M9 (6 g Na2HPO4, 3 g <br /> sources provided method beneficial           KH2PO4, 5 g NaCl, 0.2 g MgSO4 1 L H20). The <br /> the field essential gene research.                            worms washed 12 ml M9 times and <br />    We demonstrated Let mutants             incubated room temperature 2 hours. The final <br /> used, individually study gene&#8217;s function,        pellet frozen &#8722;80&#176;C. <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                         Page 10 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> Genomic DNA extraction sequencing                          predicted candidate coding region con- <br /> Genomic DNA extracted phenol/chloroform              firmed did form single complementation group <br /> described previously [30]. Briefly, worm pellet        shown Additional file 3. <br /> lysed 0.5% SDS 100ug Proteinase K 50&#176;C           Strains carrying lethal mutation selected com- <br /> two hours. DNA extracted phenol/chloroform            plementation testing lethal-carrying strains <br /> three times precipitated 100% ethanol. 20 ug       based identification candidate mutations the <br /> RNase A added eluted sample remove               gene. In order determine allelism, let-x dpy-5 unc- <br /> RNA contaminants followed           13/let-x dpy-5 unc-13; sDp2 hermaphrodites mated <br /> rounds phenol/chloroform extraction ethanol             wild-type males. F1 males (let-x dpy-5 unc-13/ + + +) <br /> precipitation. 10 ug purified genomic DNA se-           crossed hermaphrodites carrying second lethal <br /> quenced BC Cancer Agency Genome Sciences                (let-y dpy-5 unc-13/let-y dpy-5 unc-13; sDp2). The diag- <br /> Centre using Illumina PET HiSeq technology.                    nostic phenotype indicating complementation the <br />                                                                progeny cross Dpy Unc males fertile her- <br /> Mutation identification procedure                              maphrodites (let-x dpy-5 unc-13/let-y dpy-5 unc-13). A <br /> Sequencing reads aligned WS200 C. elegans          minimum wild-type males plate con- <br /> genome using BWA [36] default settings. Dupli-           sidered sufficient conclude absence Dpy <br /> cated reads filtered GATK [77]. Further realign-     Unc animals poor mating. <br /> ment indels GATK. The BAM <br /> files analyzed SNV small indels using Varscan     Gene ontology analysis <br /> [78]. The SNVs indels returned Varscan filtered     Orthologs predicted set programs consisting <br /> by 1) mutations parental strain KR235 mutation,         Inparanoid [83], OrthoMCL [84], Ensembl-Compara <br /> 2) variant ratio (90% &gt; x &gt; 40%), 3) genomic location      [85] methods previously described [57]. The protein <br /> (in coding sequences only). Allelic ratio calculated    sets used were: C. elegans (WS230), S. cerevisiae (64-1-1), <br /> the ratio mutant allele:reference allele. The effect    D. melanogaster (r5.46), H. sapiens (GRCh37.66). GO <br /> each CDS accumulate effect mutations        annotation using Blast2GO [86]. GO profile com- <br /> the genome analyzed using Coovar [79]. Mutational          parison using genes sDp2 all <br /> landscape analysis using SNVs exhibiting G &gt; A        essential genes identified RNAi collected from <br /> or C &gt; T transitions described previously [30]. Each        WormBase WS230. <br /> genes sDp2 carrying non-synonymous mutation <br /> was considered ranked according severity         RNA-seq expression analysis <br /> the mutation. Mapping information [19] used        Normalized RNA-seq data downloaded the <br /> a guide likely mutation. The mutations        modEncode website (www.modencode.org). The average <br /> for strain downloaded http://lethal.mbb.      normalized read count CDS calculated as <br /> sfu.ca/jschu/essential_genes.                                  total normalized read count coding base-pairs <br />    Sequencing second allele Sanger          divided length CDS. The expression profile <br /> sequencing WGS. PCR primers designed using             clustering using agnes clustering R. <br /> Primer3 [80,81] spaced 250 bp apart staggered <br /> orientation. This allowed sufficient overlap each <br />                                                                Additional files <br /> position covered twice. The Sanger reads <br /> were aligned wildtype transcript sequence using          Additional file 1: This figure describes lethal mutations are <br /> Clustal [82]. The alignments Sanger read         balanced sDp2 [35]. KR235 mutagenized 12 mM EMS. <br /> merged analyzed Bioedit. A mutation con-           The treated gravid wildtypes individually plated 5 cm plates and <br />                                                                 wildtype gravid F1s individually plated 5 days later. Their <br /> firmed supported Sanger reads          progeny (F2s) screened absence Dpy-5 Unc-13 individuals. <br /> the sequencing traces clear double peak. A pre-          A single Unc-13 animal transferred confirm existence a <br /> diction confirmed WGS second allele          lethal mutation. A balanced lethal exhibit Unc-13 developmentally <br />                                                                 arrested Dpy-5 Unc-13. The asterisk (*) denotes EMS mutation. In the <br /> has different mutation gene.                      F1 generation, mutation homolog both. <br />                                                                 Additional file 2: List genes studied associated alleles. <br /> Confirmation complementation testing                         The alleles used WGS listed 2nd column. The alleles used for <br /> Allelic combinations established previously com-        confirmation noted asterisk (*). <br /> plementation testing described [19] following    Additional file 3: Comparison genomes missing dpy-5 and/or <br />                                                                 unc-13 markers. The average read depth 10Kbp coding element <br /> exceptions. In cases, candidate SNVs       plotted length chromosome I. The x-axis shows the <br /> mutations, previously described mapping        coordinate 10 K units. The y-axis shows number reads. <br /> separate zones, single coding region. In cases       The control genome 33% reads 7 Mbp the <br />                                                                 genome missing markers shows flat distribution. <br /> complementation testing mutations <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                                                    Page 11 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br />  Additional file 4: Complementation table let-363 (h98), let-130                       R, Spieth J, Sternberg PW, et al: WormBase: comprehensive data <br />  (h216), let-130 (h451), let-631 (h502), let-630 (h355), let-596 (h782),                   resource Caenorhabditis biology genomics. Nucleic Acids Res <br />  let-526 (h185), let-104 (h799), let-519 (h405). (&#8722;) indicates                     2005, 33:D383&#8211;D389. <br />  mutations fail complement (+) indicates mutations                        12.   Han M, Sternberg PW: let-60, gene specifies cell fates C. <br />  complement other. N.D. indicates particular combination                      elegans vulval induction, encodes ras protein. Cell 1990, 63:921&#8211;931. <br />  done.                                                                           13.   Grishok A, Pasquinelli AE, Conte D, Li N, Parrish S, Ha I, Baillie DL, Fire A, <br />                                                                                            Ruvkun G, Mello CC: Genes mechanisms related RNA interference <br />                                                                                            regulate expression small temporal RNAs control C. elegans <br /> Abbreviations                                                                              developmental timing. Cell 2001, 106:23&#8211;34. <br /> WGS: Whole genome sequencing; SNV: Single nucleotide variation;                      14.   Hill DA, Ivanovich J, Priest JR, Gurnett CA, Dehner LP, Desruisseau D, <br /> EMS: Ethyl methanesulfonate; GO: Gene ontology.                                            Jarzembowski JA, Wikenheiser-Brokamp KA, Suarez BK, Whelan AJ, Williams <br />                                                                                            G, Bracamontes D, Messinger Y, Goodfellow PJ: DICER1 mutations in <br /> Competing interests                                                                        familial pleuropulmonary blastoma. Science 2009, 325:965. <br /> The authors declare competing interests.                           15.   Thompson O, Edgley ML, Strasbourger P, Flibotte S, Ewing B, Adair R, Au V, <br />                                                                                            Chaudhry I, Fernando L, Hutter H, Kieffer A, Lau J, Lee N, Miller A, Raymant <br /> Authors&#8217; contributions                                                                     G, Shen B, Shendure J, Taylor J, Turner EH, Hillier LW, Moerman DG, <br /> AMR DLB conceived project. JSCC, SYC, KW, AMD, AMR                             Waterston RH: The million mutation project: new approach genetics <br /> performed experiments. JSCC, RJ, DLB, AMR analyzed data.                       Caenorhabditis elegans. Genome Res 2013, 23:1749&#8211;1762. <br /> JSCC, DLB, AMR wrote paper. All authors read approved                16.   C. elegans Deletion Mutant Consortium T: Large-scale screening for <br /> final manuscript.                                                                          targeted knockouts Caenorhabditis elegans genome. G3 (Bethesda) <br />                                                                                            2012, 2:1415&#8211;1425. <br /> Acknowledgements                                                                     17.   Moerman DG, Barstead RJ: Towards mutation gene in <br /> We thank Shir Hazir technical support. We thank Dr. Nansheng                  Caenorhabditis elegans. Brief Funct Genomic Proteomic 2008, 7:195&#8211;204. <br /> Chen members Rose lab comments editing               18.   Edgley ML, Baillie DL, Riddle DL, Rose AM: Genetic balancers. 2006, http:// <br /> manuscript. JSCC supported CIHR Fanconi <span id='am-1' about='obo:NCBITaxon_12939' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3290 obo:NCBITaxon_33090 obo:NCBITaxon_693762 obo:NCBITaxon_131221 obo:NCBITaxon_693766 obo:NCBITaxon_58023 obo:NCBITaxon_241806 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Anemia" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Anemia Sw." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6'  ></span>Anemia</span>. AR DB                        www.wormbook.org/chapters/www_geneticbalancers/geneticbalancers.html. <br /> supported CIHR NSERC.                                                         19.   Johnsen RC, Jones SJ, Rose AM: Mutational accessibility essential genes <br />                                                                                            chromosome I(left) Caenorhabditis elegans. Mol Gen Genet 2000, <br /> Author details                                                                             263:239&#8211;252. <br /> 1 <br />  Department Medical Genetics, University British Columbia, Vancouver,          20.   Rose AM, Baillie DL: Genetic organization region UNC-15 (I), a <br /> Canada. 2Department Molecular Biology Biochemistry, Simon Fraser                    gene affecting paramyosin Caenorhabditis elegans. Genetics 1980, <br /> University, Burnaby, Canada. 3Department Biology, Kwantlen Polytechnic                  96:639&#8211;648. <br /> University, Surrey, Canada.                                                          21.   Sigurdson DC, Spanier GJ, Herman RK: Caenorhabditis elegans deficiency <br />                                                                                            mapping. Genetics 1984, 108:331&#8211;345. <br /> Received: 5 December 2013 Accepted: 6 May 2014                                       22.   Stewart HI, O&apos;Neil NJ, Janke DL, Franz NW, Chamberlin HM, Howell AM, <br /> Published: 12 May 2014                                                                     Gilchrist EJ, Ha TT, Kuervers LM, Vatcher GP, Danielson JL, Baillie DL: Lethal <br />                                                                                            mutations defining 112 complementation groups 4.5 Mb sequenced <br /> References                                                                                 region Caenorhabditis elegans chromosome III. Mol Gen Genet 1998, <br /> 1. Goh KI, Cusick ME, Valle D, Childs B, Vidal M, Barabasi AL: The human                   260:280&#8211;288. <br />     disease network. Proc Natl Acad Sci U S A 2007, 104:8685&#8211;8690.                   23.   Clark DV, Rogalski TM, Donati LM, Baillie DL: The unc-22(IV) region of <br /> 2. Park D, Park J, Park SG, Park T, Choi SS: Analysis human disease genes            Caenorhabditis elegans: genetic analysis lethal mutations. Genetics <br />     context gene essentiality. Genomics 2008, 92:414&#8211;418.                           1988, 119:345&#8211;353. <br /> 3. Dickerson JE, Zhu A, Robertson DL, Hentges KE: Defining role               24.   Rogalski TM, Baillie DL: Genetic organization unc-22 IV gene and <br />     essential genes human disease. PLoS One 2011, 6:e27368.                             adjacent region Caenorhabditis elegans. Mol Gen Genet 1985, <br /> 4. Solimini NL, Xu Q, Mermel CH, Liang AC, Schlabach MR, Luo J, Burrows AE,                201:409&#8211;414. <br />     Anselmo AN, Bredemeyer AL, Li MZ, Beroukhim R, Meyerson M, Elledge SJ:           25.   Rogalski TM, Moerman DG, Baillie DL: Essential genes deficiencies the <br />     Recurrent hemizygous deletions cancers optimize proliferative                   unc-22 IV region Caenorhabditis elegans. Genetics 1982, 102:725&#8211;736. <br />     potential. Science 2012, 337:104&#8211;109.                                            26.   Clark DV, Baillie DL: Genetic analysis complementation germ-line <br /> 5. Juhas M, Eberl L, Church GM: Essential genes antimicrobial targets               transformation lethal mutations unc-22 IV region Caenorhabditis <br />     cornerstones synthetic biology. Trends Biotechnol 2012, 30:601&#8211;607.                 elegans. Mol Gen Genet 1992, 232:97&#8211;105. <br /> 6. Johnsen RC, Baillie DL: Mutation. In C Elegans II. 2nd edition. Edited         27.   Johnsen RC, Baillie DL: Genetic analysis major segment [LGV(left)] of <br />     Riddle DL, Blumenthal T, Meyer BJ, Priess JR. NY: Cold Spring Harbor Press;            genome Caenorhabditis elegans. Genetics 1991, 129:735&#8211;752. <br />     1997.                                                                            28.   Meneely PM, Herman RK: Suppression function X-linked lethal and <br /> 7. Ramani AK, Chuluunbaatar T, Verster AJ, Na H, Vu V, Pelte N, Wannissorn N,              sterile mutations Caenorhabditis elegans. Genetics 1981, 97:65&#8211;84. <br />     Jiao A, Fraser AG: The majority animal genes required wild-type       29.   Meneely PM, Herman RK: Lethals, steriles deficiencies region of <br />     fitness. Cell 2012, 148:792&#8211;802.                                                       X chromosome Caenorhabditis elegans. Genetics 1979, 92:99&#8211;115. <br /> 8. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kanapin A, Le          30.   Chu JS, Johnsen RC, Chua SY, Tu D, Dennison M, Marra M, Jones SJ, Baillie <br />     Bot N, Moreno S, Sohrmann M, Welchman DP, Zipperlen P, Ahringer J:                     DL, Rose AM: Allelic ratios mutational landscape reveal <br />     Systematic functional analysis Caenorhabditis elegans genome                    biologically significant heterozygous SNVs. Genetics 2012, 190:1225&#8211;1233. <br />     using RNAi. Nature 2003, 421:231&#8211;237.                                            31.   Sarin S, Prabhu S, O&#8217;<span id='am-49' about='obo:NCBITaxon_79794' typeof='obo:NCBITaxon_33511 owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_33208 obo:NCBITaxon_1312402 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_52051 obo:NCBITaxon_52050 obo:NCBITaxon_147099 obo:NCBITaxon_131567 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Meara" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Meara</span> MM, Pe&#8217;er I, Hobert O: Caenorhabditis elegans <br /> 9. Kemphues K: Essential Genes. [http://www.wormbook.org/chapters/                         mutant allele identification whole-genome sequencing. Nat Methods <br />     www_essentialgenes/essentialgenes.html]                                                2008, 5:865&#8211;867. <br /> 10. Yook K, Harris TW, Bieri T, Cabunoc A, Chan J, Chen WJ, Davis P, la Cruz N,   32.   Rose AM, O&#8217;Neil NJ, Bilenky M, Butterfield YS, Malhis N, Flibotte S, Jones MR, <br />     Duong A, Fang R, Ganesan U, Grove C, Howe K, Kadam S, Kishore R, Lee R, Li Y,          Marra M, Baillie DL, Jones SJ: Genomic sequence mutant strain of <br />     Muller HM, Nakamura C, Nash B, Ozersky P, Paulini M, Raciti D, Rangarajan A,           Caenorhabditis elegans altered recombination pattern. <br />     Schindelman G, Shi X, Schwarz EM, Ann Tuli M, Van Auken K, Wang D, et al:              BMC Genomics 2010, 11:131. <br />     WormBase 2012: genomes, data, new website. Nucleic Acids Res           33.   Shen Y, Sarin S, Liu Y, Hobert O, Pe&#8217;er I: Comparing platforms C. <br />     2012, 40:D735&#8211;D741.                                                                    elegans mutant identification using high-throughput whole-genome <br /> 11. Chen N, Harris TW, Antoshechkin I, Bastiani C, Bieri T, Blasiar D, Bradnam K,          sequencing. PLoS One 2008, 3:e4012. <br />     Canaran P, Chan J, Chen CK, Chen WJ, Cunningham F, Davis P, Kenny E,             34.   Zuryn S, Le Gras S, Jamet K, Jarriault S: A strategy direct mapping and <br />     Kishore R, Lawson D, Lee R, Muller HM, Nakamura C, Pai S, Ozersky P,                   identification mutations whole-genome sequencing. Genetics 2010, <br />     Petcherski A, Rogers A, Sabo A, Schwarz EM, Van Auken K, Wang Q, Durbin                186:427&#8211;430. <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                                                      Page 12 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br /> 35. Howell AM, Gilmour SG, Mancebo RA, Rose AM: Genetic analysis large                  Fiebig A, Franke A, Schreiber S, Stewart AF, Roberts R, McPherson R, Guter <br />     autosomal region Caenorhabditis elegans use free                          SJ, Cook EH Jr, Dawson G, Schellenberg GD, et al: Disruption the <br />     duplication. Genet Res 1987, 49:207&#8211;213.                                                 PTCHD1 locus Xp22.11 autism spectrum disorder intellectual <br /> 36. Li H, Durbin R: Fast accurate short read alignment Burrows-                     disability. Sci Transl Med 2010, 2:49ra68. <br />     Wheeler transform. Bioinformatics 2009, 25:1754&#8211;1760.                              55.   Liu RY, Diao CF, Zhang Y, Wu N, Wan HY, Nong XY, Liu M, Tang H: miR-371- <br /> 37. Rose AM, Baillie DL, Curran J: Meiotic pairing behavior free                      5p down-regulates pre mRNA processing factor 4 homolog B (PRPF4B) <br />     duplications linkage group I Caenorhabditis elegans. Mol Gen Genet                 facilitates G1/S transition human hepatocellular carcinoma <br />     1984, 195:52&#8211;56.                                                                         cells. Cancer Lett 2013, 335:351&#8211;360. <br /> 38. Flibotte S, Edgley ML, Chaudhry I, Taylor J, Neil SE, Rogula A, Zapf R, Hirst M,   56.   McDowall JS, Rose AM: Genetic analysis sterile mutants dpy-5 <br />     Butterfield Y, Jones SJ, Marra MA, Barstead RJ, Moerman DG: Whole-                       unc-13 (I) genomic region Caenorhabditis elegans. Mol Gen Genet <br />     genome profiling mutagenesis Caenorhabditis elegans. Genetics                      1997, 255:60&#8211;77. <br />     2010, 185:431&#8211;441.                                                                 57.   Shaye DD, Greenwald I: OrthoList: compendium C. elegans genes <br /> 39. Korzelius J, The I, Ruijtenberg S, Portegijs V, Xu H, Horvitz HR, van den                human orthologs. PLoS One 2011, 6:e20085. <br />     Heuvel S: C. elegans MCM-4 general DNA replication checkpoint             58.   Kitagawa R, Rose AM: Components spindle-assembly checkpoint <br />     component epidermis-specific requirement growth                          essential Caenorhabditis elegans. Nat Cell Biol 1999, 1:514&#8211;521. <br />     viability. Dev Biol 2011, 350:358&#8211;369.                                             59.   Maddox PS, Hyndman F, Monen J, Oegema K, Desai A: Functional <br /> 40. Lye RJ, Wilson RK, Waterston RH: Genomic structure cytoplasmic                      genomics identifies Myb domain-containing protein <span id='am-12' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-13' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> required <br />     dynein heavy chain gene nematode Caenorhabditis elegans.                        assembly CENP-A chromatin. J Cell Biol 2007, 176:757&#8211;763. <br />     Cell Motil Cytoskeleton 1995, 32:26&#8211;36.                                            60.   Kaitna S, Pasierbek P, Jantsch M, Loidl J, Glotzer M: The aurora B kinase <br /> 41. O&#8217;Rourke SM, Dorfman MD, Carter JC, Bowerman B: Dynein modifiers C.                   AIR-2 regulates kinetochores mitosis required separation <br />     elegans: light chains suppress conditional heavy chain mutants.                          homologous Chromosomes meiosis. Curr Biol 2002, 12:798&#8211;812. <br />     PLoS Genet 2007, 3:e128.                                                           61.   Rogers E, Bishop JD, Waddle JA, Schumacher JM, Lin R: The aurora kinase <br /> 42. Gil-Krzewska AJ, Farber E, Buttner EA, Hunter CP: Regulators actin                AIR-2 functions release chromosome cohesion Caenorhabditis <br />     cytoskeleton mediate lethality Caenorhabditis elegans dhc-1                         elegans meiosis. J Cell Biol 2002, 157:219&#8211;229. <br />     mutant. Mol Biol Cell 2010, 21:2707&#8211;2720.                                          62.   Moore LL, Stanvitch G, Roth MB, Rosen D: HCP-4/CENP-C promotes the <br /> 43. Wissmann A, Ingles J, McGhee JD, Mains PE: Caenorhabditis elegans                        prophase timing centromere resolution enabling centromere <br />     LET-502 related Rho-binding kinases human myotonic                             association HCP-6 Caenorhabditis elegans. Mol Cell Biol 2005, <br />     dystrophy kinase interacts genetically homolog                         25:2583&#8211;2592. <br />     regulatory subunit smooth muscle myosin phosphatase affect cell              63.   Shakes DC, Sadler PL, Schumacher JM, Abdolrasulnia M, Golden A: <br />     shape. Genes Dev 1997, 11:409&#8211;422.                                                       Developmental defects observed hypomorphic anaphase-promoting <br /> 44. Long X, Spycher C, Han ZS, Rose AM, Muller F, Avruch J: TOR deficiency                complex mutants linked cell cycle abnormalities. Development <br />     C. elegans causes developmental arrest intestinal atrophy                         2003, 130:1605&#8211;1620. <br />     inhibition mRNA translation. Curr Biol 2002, 12:1448&#8211;1461.                      64.   Shen Q, Zheng X, McNutt MA, Guang L, Sun Y, Wang J, Gong Y, Hou L, <br /> 45. Holzenberger M, Dupont J, Ducos B, Leneuve P, Geloen A, Even PC, <span id='am-75' about='obo:NCBITaxon_1162614' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_6073 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6132 obo:NCBITaxon_723795 obo:NCBITaxon_6101 obo:NCBITaxon_40677 obo:NCBITaxon_723794 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-76' property="rdfs:label" content="Cervera" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79'  ></span>Cervera</span>                 Zhang B: NAT10, nucleolar protein, localizes midbody and <br />     P, Le Bouc Y: IGF-1 receptor regulates lifespan resistance oxidative              regulates cytokinesis acetylation microtubules. Exp Cell Res 2009, <br />     stress mice. Nature 2003, 421:182&#8211;187.                                                315:1653&#8211;1667. <br /> 46. Woollard A, Hodgkin J: Stu-7/air-2 C. elegans aurora homologue                65.   Schmidt DJ, Rose DJ, Saxton WM, Strome S: Functional analysis of <br />     essential chromosome segregation embryonic                                cytoplasmic dynein heavy chain Caenorhabditis elegans fast-acting <br />     post-embryonic development. Mech Dev 1999, 82:95&#8211;108.                                    temperature-sensitive mutations. Mol Biol Cell 2005, 16:1200&#8211;1212. <br /> 47. Roggo L, Bernard V, Kovacs AL, Rose AM, Savoy F, Zetka M, Wymann MP,               66.   Bruder MB, Mogro-Wilson C, Kerins GJ: A survey assessing presence of <br />     Muller F: Membrane transport Caenorhabditis elegans: essential                     &#8220;medical home&#65533;? adults disabilities Connecticut. Conn Med <br />     role VPS34 nuclear membrane. EMBO J 2002, 21:1673&#8211;1683.                       2010, 74:341&#8211;348. <br /> 48. Amin NM, Shi H, Liu J: The FoxF/FoxC factor LET-381 directly regulates             67.   Mantina P, MacDonald L, Kulaga A, Zhao L, Hansen D: A mutation teg-4, <br />     cell fate specification cell differentiation C. elegans                      encodes protein homologous SAP130 pre-mRNA splicing <br />     mesoderm development. Development 2010, 137:1451&#8211;1460.                                   factor, disrupts balance proliferation differentiation in <br /> 49. McDowall JS, Rose A: Alignment genetic physical maps                   C. elegans germ line. Mech Dev 2009, 126:417&#8211;429. <br />     dpy-5 bli-4 (I) region C. elegans serial cosmid rescue lethal         68.   Contrino S, Smith RN, Butano D, Carr A, Hu F, Lyne R, Rutherford K, <br />     mutations. Mol Gen Genet 1997, 255:78&#8211;95.                                                Kalderimis A, Sullivan J, Carbon S, Kephart ET, Lloyd P, Stinson EO, <br /> 50. Zhao Z, Hashiguchi A, Hu J, Sakiyama Y, Okamoto Y, Tokunaga S, Zhu L, Shen               Washington NL, Perry MD, Ruzanov P, Zha Z, Lewis SE, Stein LD, Micklem G: <br />     H, Takashima H: Alanyl-tRNA synthetase mutation family dominant                modMine: flexible access modENCODE data. Nucleic Acids Res 2012, <br />     distal hereditary motor neuropathy. Neurology 2012, 78:1644&#8211;1649.                        40:D1082&#8211;D1088. <br /> 51. Schapira AH, Mann VM, Cooper JM, Dexter D, Daniel SE, Jenner P, Clark JB,          69.   Gerstein MB, Lu ZJ, Van Nostrand EL, Cheng C, Arshinoff BI, Liu T, Yip KY, <br />     Marsden CD: Anatomic disease specificity NADH CoQ1 reductase                      Robilotto R, Rechtsteiner A, Ikegami K, Alves P, Chateigner A, Perry M, Morris <br />     (complex I) deficiency Parkinson&#8217;s disease. J Neurochem 1990, 55:2142&#8211;2145.           M, Auerbach RK, Feng X, Leng J, Vielle A, Niu W, Rhrissorrakrai K, Agarwal A, <br /> 52. Pinto D, Pagnamenta AT, Klei L, Anney R, Merico D, Regan R, Conroy J,                    Alexander RP, Barber G, Brdlik CM, Brennan J, Brouillet JJ, Carr A, Cheung <br />     Magalhaes TR, Correia C, Abrahams BS, <span id='am-70' about='obo:NCBITaxon_706296' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33355 obo:NCBITaxon_33354 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7524 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33351 obo:NCBITaxon_528045 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_33349 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_82738 obo:NCBITaxon_33345 obo:NCBITaxon_33347 obo:NCBITaxon_33342 obo:NCBITaxon_50557 obo:NCBITaxon_33343'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="rdfs:label" content="Almeida" datatype="xsd:string"></span>Almeida</span> J, Bacchelli E, Bader GD,                  MS, Clawson H, Contrino S, et al: Integrative analysis the <br />     Bailey AJ, Baird G, Battaglia A, Berney T, Bolshakova N, Bolte S, Bolton PF,             Caenorhabditis elegans genome modENCODE project. Science <br />     Bourgeron T, Brennan S, Brian J, Bryson SE, Carson AR, Casallo G, Casey J,               2010, 330:1775&#8211;1787. <br />     Chung BH, Cochrane L, Corsello C, et al: Functional impact global rare          70.   Sulston JE, Schierenberg E, White JG, Thomson JN: The embryonic cell <br />     copy number variation autism spectrum disorders. Nature 2010,                         lineage nematode Caenorhabditis elegans. Dev Biol 1983, <br />     466:368&#8211;372.                                                                             100:64&#8211;119. <br /> 53. Marshall CR, Noor A, Vincent JB, Lionel AC, Feuk L, Skaug J, Shago M,              71.   Austin J, Kimble J: glp-1 required germ line regulation the <br />     Moessner R, Pinto D, Ren Y, Thiruvahindrapduram B, Fiebig A, Schreiber S,                decision mitosis meiosis C. elegans. Cell 1987, <br />     Friedman J, Ketelaars CE, Vos YJ, Ficicioglu C, Kirkpatrick S, Nicolson R,               51:589&#8211;599. <br />     Sloman L, Summers A, Gibbons CA, Teebi A, Chitayat D, Weksberg R,                  72.   Green RA, Kao HL, Audhya A, Arur S, Mayers JR, Fridolfsson HN, Schulman <br />     Thompson A, Vardy C, Crosbie V, Luscombe S, Baatjes R, et al: Structural                 M, Schloissnig S, Niessen S, Laband K, Wang S, Starr DA, Hyman <span id='am-80' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Aa" datatype="xsd:string"></span>AA</span>, Schedl <br />     variation chromosomes autism spectrum disorder. Am J Hum Genet                     T, Desai A, Piano F, Gunsalus KC, Oegema K: A high-resolution C. elegans <br />     2008, 82:477&#8211;488.                                                                        essential gene network based phenotypic profiling complex <br /> 54. Noor A, Whibley A, Marshall CR, Gianakopoulos PJ, Piton A, Carson AR,                    tissue. Cell 2011, 145:470&#8211;482. <br />     Orlic-Milacic M, Lionel AC, Sato D, Pinto D, Drmic I, Noakes C, Senman L,          73.   Sonnichsen B, Koski LB, Walsh A, Marschall P, Neumann B, Brehm M, <br />     Zhang X, Mo R, Gauthier J, Crosbie J, Pagnamenta AT, Munson J, Estes AM,                 Alleaume AM, Artelt J, Bettencourt P, Cassin E, Hewitson M, Holz C, Khan M, <br />  Chu et al. BMC Genomics 2014, 15:361                                                                                                              Page 13 13 <br /> http://www.biomedcentral.com/1471-2164/15/361 <br />  <br />  <br />  <br />  <br />       Lazik S, Martin C, Nitzsche B, Ruer M, Stamford J, Winzi M, Heinkel R, Roder <br />       M, Finell J, Hantsch H, Jones SJ, Jones M, Piano F, Gunsalus KC, Oegema K, <br />       Gonczy P, Coulson A, et al: Full-genome RNAi profiling early <br />       embryogenesis Caenorhabditis elegans. Nature 2005, 434:462&#8211;469. <br /> 74.   Pauli F, Liu Y, Kim YA, Chen PJ, Kim SK: Chromosomal clustering GATA <br />       transcriptional regulation intestine-expressed genes C. elegans. <br />       Development 2006, 133:287&#8211;295. <br /> 75.   Reinke V, Gil IS, Ward S, Kazmer K: Genome-wide germline-enriched and <br />       sex-biased expression profiles Caenorhabditis elegans. Development <br />       2004, 131:311&#8211;323. <br /> 76.   Brenner S: The genetics Caenorhabditis elegans. Genetics 1974, 77:71&#8211;94. <br /> 77.   McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, <br />       Garimella K, Altshuler D, Gabriel S, Daly M, DePristo MA: The Genome <br />       Analysis Toolkit: MapReduce framework analyzing next-generation <br />       DNA sequencing data. Genome Res 2010, 20:1297&#8211;1303. <br /> 78.   Koboldt DC, Zhang Q, Larson DE, Shen D, McLellan MD, Lin L, Miller CA, <br />       Mardis ER, Ding L, Wilson RK: VarScan 2: somatic mutation copy <br />       number alteration discovery cancer exome sequencing. Genome <br />       Res 2012, 22:568&#8211;576. <br /> 79.   Vergara <span id='am-7' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-8' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>IA</span>, Frech C, Chen N: CooVar: Co-occurring variant analyzer. <br />       BMC Res Notes 2012, 5:615. <br /> 80.   Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG: <br />       Primer3&#8211;new capabilities interfaces. Nucleic Acids Res 2012, 40:e115. <br /> 81.   Koressaar T, Remm M: Enhancements modifications primer design <br />       program Primer3. Bioinformatics 2007, 23:1289&#8211;1291. <br /> 82.   Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam <br />       H, Valentin F, Wallace IM, Wilm A, Lopez R, Thompson JD, Gibson TJ, <br />       Higgins DG: Clustal W Clustal X version 2.0. Bioinformatics 2007, <br />       23:2947&#8211;2948. <br /> 83.   Ostlund G, Schmitt T, Forslund K, Kostler T, Messina DN, Roopra S, Frings O, <br />       Sonnhammer EL: InParanoid 7: new algorithms tools eukaryotic <br />       orthology analysis. Nucleic Acids Res 2010, 38:D196&#8211;D203. <br /> 84.   Chen F, Mackey AJ, Stoeckert CJ Jr, Roos DS: OrthoMCL-DB: querying a <br />       comprehensive multi-<span id='am-64' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-65' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> collection ortho <br /> </body></html>