<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Weaver et al. BMC Systems Biology 2014, 8:79 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                            Open Access <br />  <br /> A genome-scale metabolic flux model of <br /> <span id='am-155' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-163'  ></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli</span> K&#8211;12 derived EcoCyc <br /> database <br /> Daniel S Weaver1* , Ingrid M Keseler1 , Amanda Mackie2 , Ian T Paulsen2 Peter D Karp1 <br />  <br />  <br />   Abstract <br />   Background: Constraint-based models <span id='am-171' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-173'  ></span><span id='am-174' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli metabolic flux played key role computational <br />   studies cellular metabolism genome scale. We sought develop next-generation constraint-based E. coli <br />   model achieved improved phenotypic prediction accuracy frequently updated easy use. We <br />   sought compare model predictions experimental <span id='am-249' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-250' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-251'  ></span><span id='am-252' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> highlight open questions E. coli biology. <br />   Results: We present EcoCyc&#8211;18.0&#8211;GEM, genome-scale model E. coli K&#8211;12 MG1655 metabolic network. The <br />   model automatically generated current state EcoCyc using MetaFlux software, enabling release of <br />   multiple model updates year. EcoCyc&#8211;18.0&#8211;GEM encompasses 1445 genes, 2286 unique metabolic reactions, and <br />   1453 unique metabolites. We demonstrate three-part validation model breaks new ground breadth and <br />   accuracy: (i) Comparison simulated growth aerobic anaerobic glucose culture experimental results from <br />   chemostat culture simulation results E. coli modeling literature. (ii) Essentiality prediction 1445 <br />   genes represented model, EcoCyc&#8211;18.0&#8211;GEM achieves improved accuracy 95.2% predicting <br />   growth phenotype experimental gene knockouts. (iii) Nutrient utilization predictions 431 different media <br />   conditions, model achieves overall accuracy 80.7%. The model&#8217;s derivation EcoCyc enables <br />   query visualization EcoCyc website, facilitating model reuse validation inspection. We present an <br />   extensive investigation disagreements EcoCyc&#8211;18.0&#8211;GEM predictions experimental data highlight <br />   areas E. coli modelers experimentalists, including 70 incorrect predictions gene essentiality on <br />   glucose, 80 incorrect predictions gene essentiality glycerol, 83 incorrect predictions nutrient utilization. <br />   Conclusion: Significant advantages derived combination model organism databases flux <br />   balance modeling represented MetaFlux. Interpretation EcoCyc database flux balance model results a <br />   highly accurate metabolic model provides rigorous consistency check information stored database. <br />   Keywords: Escherichia coli, Flux balance analysis, Constraint-based modeling, Metabolic network reconstruction, <br />   Metabolic modeling, Genome-scale model, Gene essentiality, Systems biology, EcoCyc, Pathway Tools <br />  <br />  <br /> Background                                                                           Escherichia coli K&#8211;12 MG1655 metabolic network. A <br /> Constraint-based modeling techniques flux bal-                               series E. coli constraint-based models pub- <br /> ance analysis (FBA) central systems                                   lished group B. Palsson [3-6], extending work on <br /> biology [1,2], enabling wealth informative simula-                              stoichiometric constraint-based modeling E. coli dating <br /> tions cellular metabolism. Many constraint-based mod-                             years [7-10]. These models consti- <br /> eling techniques demonstrated                                tute gold standard E. coli modeling, seen <br />                                                                                      range applications [11-13] including metabolic engi- <br />                                                                                      neering, model-driven discovery, cellular-phenotype pre- <br /> *Correspondence: weaver@ai.sri.com                                                   diction, analysis metabolic network properties, studies <br /> 1 Bioinformatics Research Group, SRI International, 333 Ravenswood Ave., <br />                                                                                      evolutionary processes, modeling interspecies <br /> 94025 Menlo Park, CA, USA <br /> Full list author information available end article               interactions. <br />  <br />  <br />                                          &#169; 2014 Weaver et al.; licensee BioMed Central Ltd. <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60'  ></span>This</span> Open Access article distributed terms Creative <br />                                          Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                          reproduction medium, provided original work properly credited. The Creative Commons Public Domain Dedication <br />                                          waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, unless otherwise <br />                                          stated. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                   Page 2 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />    Motivated widespread use E. coli metabolic        Table 1 Survey recent E. coli genome-scale model <br /> models, aimed illustrate benefits integrat-       statistics <br /> ing metabolic modeling model organism databases        Statistics                  Feist et al.   Orth et al.   EcoCyc&#8211;18.0&#8211;GEM <br /> developing E. coli model derived directly Eco-                                   2007 [5]      2011 [6] <br /> Cyc bioinformatics database [14]. First, aimed use       # Genes                           1260          1366                    1445 <br /> the extensive biochemical literature referenced EcoCyc      # Unique reactions                1721          1863                    2286 <br /> to develop model improved accuracy pheno-           # Unique metabolites              1039          1136                    1453 <br /> typic prediction, specifically predicting phenotypes   Gene knockout accuracy           91.4%         91.3%                   95.2% <br /> of gene knock-outs, predicting growth lack          # PM growth conditions             170             &#8211;                     431 <br /> thereof different nutrient conditions.                   PM growth condition              75.9%             &#8211;                   80.7% <br />    Second, sought make model easy under-          accuracy <br /> stand operate. Our goal high level model          # biomass metabolites                65            72                    108 <br /> accessibility readability (a) tight web-based      Gene knockout prediction accuracy represents simulated growth glucose <br />                                                                minimal media aerobic conditions. Reaction metabolite counts <br /> integration model extensive model query        represent reactions cytosol periplasm, EcoCyc&#8211;18.0&#8211;GEM <br /> visualization tools, (b) model representation       does cover porin-mediated diffusion metabolites periplasmic <br />                                                                space. <br /> captures extensive information enriches model <br /> and aids understanding, metabolic pathways, <br /> chemical structures, genetic regulatory information. <br />                                                                  The MetaFlux component Pathway Tools trans- <br /> Metabolic models just mathematical entities       <span id='am-43' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-44' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>lates</span> Pathway/Genome Database (PGDB) reactions and <br /> output predictions; artifacts scientists <br />                                                                compounds constraint-based metabolic models. Our <br /> interact multiple ways. If model quickly <br />                                                                methodology fusing systems-biology models bioin- <br /> and easily understood, scientists likely trust     formatics databases advantages of <br /> its predictions, model easier reuse, modify <br />                                                                strong synergies approaches. Databases <br /> and extend, learn from, validate inspec- <br />                                                                models require extensive literature-based <span id='am-228' about='obo:NCBITaxon_69520' typeof='obo:NCBITaxon_6157 owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_31262 obo:NCBITaxon_147100 obo:NCBITaxon_33154 obo:NCBITaxon_166126 obo:NCBITaxon_1292243 obo:NCBITaxon_1292253 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6159'><span id='am-229' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-231'  ></span><span id='am-232' property="rdfs:label" content="Cura" datatype="xsd:string"></span>cura</span>- <br /> tion. These aspects metabolic model depend strongly <br />                                                                tion refinement. It efficient perform that <br /> on model represented, software tools <br />                                                                curation manner benefits database and <br /> available interactively inspect model,       model, duplicate curation efforts database <br /> tightly integrated model software tools. <br />                                                                project modeling project. Furthermore, model- <br />    Third, sought produce model frequently <br />                                                                ing process identifies errors, omissions, inconsisten- <br /> updated integrate new knowledge E. coli metabolism.      cies description metabolic model, therefore <br />    Fourth, sought use EcoCyc-derived E. coli <br />                                                                drives correction curation database if <br /> metabolic model identify errors EcoCyc, open <br />                                                                efforts coupled. We 80 Eco- <br /> problems E. coli biology, performing in-depth inves-     Cyc updates result comparing model predictions <br /> tigations disagreements phenotypic <br />                                                                experimental data literature work. In <br /> predictions model experimental results. <br />                                                                addition, bioinformatics database curation methods such <br />    We present EcoCyc&#8211;18.0&#8211;GEM, constraint-based <br />                                                                use evidence codes citations provide <br /> genome-scale metabolic model E. coli K&#8211;12 MG1655 <br />                                                                data provenance, incorporation mini-review <br /> that directly derived EcoCyc model organism        summaries enzymes pathways, benefit <br /> database (http://EcoCyc.org) built genome <br />                                                                systems-biology models, typically lack data prove- <br /> sequence E. coli K&#8211;12 MG1655. The model imple- <br />                                                                nance explanations. <br /> mented using MetaFlux [15] component Path- <br /> way Tools software [16]. <br />                                                                Advances model size. Compared iJO1366, <br /> Results discussion                                         EcoCyc&#8211;18.0&#8211;GEM represents 6% increase num- <br /> The EcoCyc&#8211;18.0&#8211;GEM generated EcoCyc 18.0                 ber genes, 23% increase number unique <br /> encompasses 1445 genes, 2286 unique cytosolic              cytosolic periplasmic reactions, 28% increase in <br /> periplasmic reactions, 1453 unique metabolites.            number unique metabolites. The size EcoCyc&#8211; <br /> Table 1 compares statistics EcoCyc&#8211;18.0&#8211;GEM             18.0&#8211;GEM currently exceeded mathe- <br /> with previous E. coli metabolic models. EcoCyc&#8211;18.0&#8211;           matically complex ME-model O&#8217;Brien et al. [17], which <br /> GEM advance previous stoichiometric models          includes simulation gene expression, transcriptional <br /> of E. coli metabolism respects: size;    regulation, protein synthesis. <br /> accuracy; form, readability accessibility; in <br /> its update frequency. Here summarize results;         Advances model accuracy. We conducted three- <br /> these points expanded subsequent subsections.       phase validation EcoCyc&#8211;18.0&#8211;GEM assess its <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                   Page 3 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> accuracy (see Table 1). In phase I compared simulated       sets, detailed length Additional file 1 <br /> EcoCyc&#8211;18.0&#8211;GEM rates nutrient uptake prod-             Additional file 2: Table S1. The biomass metabolite <br /> uct secretion aerobic anaerobic glucose culture         set establishes requirements growth determines <br /> with experimental rates derived chemostat culture;        growth rate simulation. The biomass metabo- <br /> the performance EcoCyc&#8211;18.0&#8211;GEM equivalent              lite set EcoCyc&#8211;18.0&#8211;GEM based iJO1366 <br /> to previous models. In phase II compared essential-         wild-type core biomass reaction sets published by <br /> ity prediction 1445 genes involved model        Orth et al., revisions stemming differ- <br /> with experimental gene essentiality datasets; error rate   ences content functionality EcoCyc and <br /> in predicting gene-knockout phenotypes decreased            iJO1366 model. Gene essentiality constraint-based <br /> 46% best previous model. In phase III com-         models principally determined biomass demands <br /> pared nutrient utilization predictions EcoCyc&#8211;18.0&#8211;         cell. Inclusion metabolite biomass <br /> GEM 431 experimental nutrient utilization tests;      metabolite set forces genes required manufacture <br /> accuracy predicting growth respiration dif-       metabolite essential simulation. <br /> ferent nutrient conditions increased 4.8% previous        A wild-type biomass metabolite set, derived <br /> models number nutrient conditions expanded           measurement biomolecular composition of <br /> 2.5-fold. We investigated conflicts experimental       healthy, growing cells, representative min- <br /> results predictions EcoCyc&#8211;18.0&#8211;GEM detail,          imal set biomass metabolites required cell sur- <br /> and provide extensive discussion conflicts         vival. Because biomass metabolites truly required for <br /> within context EcoCyc literature.               cell survival generate false simulation predictions of <br />   Subjects particular include alternative cat-     essentiality biosynthetic pathways, concept of <br /> alytic routes capable replacing genes thought         core biomass metabolite set developed Feist et al. <br /> essential; compounds unclear routes catabolism         The core biomass metabolite set biomass metabolite <br /> which capable supporting growth and/or cellular         set defined aim maintaining quanti- <br /> respiration; regulatory environmental perturbations        tative accuracy regards cell performance while <br /> of stoichiometric network model; investigations     predicting observed experimental essentiality data as <br /> what, exactly, constitutes gene essentiality.                  accurately possible. Because work focuses <br />                                                                testing minimum requirements cell growth, we <br /> Advances model form, readability, accessibility.        frequently employed core biomass metabolite set in <br /> Another benefit coupling systems-biology models        simulations. We use term &#8220;expanded biomass set&#65533;? <br /> databases, corresponding advance model,        refer version wild-type biomass metabo- <br /> that generating constraint-based model database       lite set described Orth et al., wish <br /> that associated web-based visualization tools leads     imply simulated cells represented the <br /> a literate model (by analogy Knuth&#8217;s notion literate     wild-type state. <br /> programming [18]). A literate model easy read,       The biomass metabolite sets described underwent <br /> highly accessible understandable scientists.         revisions reflecting differences scope between <br />                                                                EcoCyc&#8211;18.0&#8211;GEM iJO1366. Whereas iJO1366 a <br /> Advances model update frequency. Because                purpose-built model developed using COBRA Tool- <br /> MetaFlux component Pathway Tools generates                  box input KEGG, EcoCyc, databases, <br /> constraint-based models directly EcoCyc               EcoCyc database schema entries <br /> PGDBs, database refined new cura-            programmatically transformed FBA model. <br /> tion, refinements automatically incorporated         The specific metabolites present iJO1366 therefore <br /> into newly generated versions model. We release         matched specific metabolites <br /> new versions EcoCyc-based model times             generated EcoCyc MetaFlux one-to-one <br /> per year; previous models updated years        basis. Several biomass metabolites represented dis- <br /> [4-6]. Although reasons limit frequency       tinct iJO1366, phosphatidylethanolamines <br /> of releases <span id='am-64' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> tie well-defined version     different chain lengths saturations, summed <br /> of database throughly test accuracy new         heading single representative metabolite <br /> versions, believe frequent model updates          EcoCyc&#8211;18.0&#8211;GEM biomass set. Additionally, not <br /> are useful organism important E. coli.            processes covered EcoCyc&#8211;18.0&#8211;GEM covered <br />                                                                iJO1366, reverse true. As result, the <br /> Validation biomass metabolites, nutrients,              biomass metabolite sets differ slightly. Additional file 2: <br /> secretions                                                     Table S2 contains complete side-by-side comparison of <br /> Refinement EcoCyc&#8211;18.0&#8211;GEM began vali-             EcoCyc&#8211;18.0&#8211;GEM iJO1366 biomass metabolite <br /> dation biomass, nutrient, secretion metabolite      sets, lists differences them. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                    Page 4 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />    We constructed standard nutrient sets based culture       Table 2 Flux comparison EcoCyc&#8211;18.0&#8211;GEM and <br /> conditions reflecting experiments glucose glycerol        iJO1366: ATP maximization objective aerobic <br /> minimal media model&#8217;s capability use sub-         conditions <br /> strates. Those substrates include glucose glycerol        EcoCyc                                             iJO1366 <br /> appropriate, O2 , NH+  4 , phosphate, sulfate, ferrous iron,    Reaction                               Flux        Reaction             Flux <br /> water, CO2 , minerals appropriate biomass objec-         ATP maximization objective              216        ATPM                  235 <br /> tive function, <span id='am-254' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-255' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-258'  ></span>MOPS</span> buffer (usable sulfur source) <br />                                                                 Glucose uptake                           10        EX_glc(e)              10 <br /> where appropriate. Because passive diffusion high <br />                                                                 O2 uptake                                60        EX_o2(e)               60 <br /> concentration ammonium used experimental cul- <br /> ture, NH+  4 supplied directly cytosol (avoiding <br />                                                                 H2 O production                          60        EX_h2o(e)              60 <br /> false negative essentiality predictions high-affinity   CO2 production                           60        EX_co2(e)              60 <br /> nitrogen transporter amtB), nutrients         ATP synthase*                           176        ATPS4rpp              195 <br /> are supplied periplasmic space.                          NADH-DEHYDROG-A-RXN                     100        NADH16pp              100 <br />    Finally, developed large set secreted com- <br />                                                                 Cytochrome bo oxidase*                   60        CYTBO3_4pp            120 <br /> pounds supplied growth condi- <br />                                                                 PYRNUTRANSHYDROGEN-RXN                   20        NADTRHD                20 <br /> tions explored model. It contains plausible <br /> products E. coli metabolism dead-end metabo-             GAPOXNPHOSPHN-RXN                        20        GAPD                   20 <br /> lites [19] model. The presence absence         ISOCITDEH-RXN                            20        ICDHyr                 20 <br /> metabolites set construed indicate     3PGAREARR-RXN                            20        ENO                    20 <br /> their presence absence E. coli culture media.             ACONITATEDEHYDR-RXN                      20        ACONTa/b               20 <br />    We verified metabolic reachability compo- <br />                                                                 PHOSGLYPHOS-RXN                          20        PGK                    20 <br /> nent EcoCyc biomass metabolite set sup- <br />                                                                 CITSYN-RXN                               20        CS                     20 <br /> plying nutrients representing aerobic glucose minimal <br /> medium setting production individual            2OXOGLUTARATEDEH-RXN                     20        AKGDH                  20 <br /> metabolite turn optimization goal MetaFlux,        2PGADEHYDRAT-RXN                         20        ENO                    20 <br /> and repaired gaps means literature-based manual           Succinate dehydrogenase*                 20        SUCDi                  20 <br /> curation EcoCyc expansion relevant metabo-        MALATE-DEH-RXN                           20        MDH                    20 <br /> lite sets. <br />                                                                 FUMHYDR-RXN                              20        FUM                    20 <br />                                                                 PYRUVDEH-RXN                             20        PDH                    20 <br /> ATP maximization validation <br /> We confirmed simulations aerobic growth         SUCCOASYN-RXN                            20        SUCOAS                 20 <br /> glucose run maximization ATP production        PGLUCISOM-RXN                            10        PGI                    10 <br /> objective appropriate use glycolytic TCA        Glucose PTS uptake*                      10        GLCptspp               10 <br /> cycle pathways agreed previous work E. coli         TRIOSEPISOMERIZATION-RXN                 10        TPI                    10 <br /> FBA. <br />                                                                 RXN0-313                                 10        F6PA                   10 <br />   The maximization ATP production aerobic <br />                                                                 2.7.1.121-RXN                            10        DHAPT                  10 <br /> conditions studied setting ATP consump- <br /> tion reaction ATP + H2 O &#8594; ADP + Pi + H+                 Glucose uptake set 10 mmol/gCDW/hr. Other nutrient secretion fluxes <br />                                                                 unbounded. Flux rates reported absolute values clarity. True <br /> objective function maximized MetaFlux. The             iJO1366 fluxes EX_glc(e), EX_o2(e), PGK, PGM, SUCOAS negative <br /> fluxes resulting maximization ATP produc-           reaction directionality convention. Most EcoCyc reactions are <br />                                                                 identifed EcoCyc frame IDs. The remainder marked asterisks, <br /> tion glucose aerobic conditions compared          frame IDs follows: ATP synthase EcoCyc frame ID: <br /> with fluxes COBRA Toolbox [20] simulations              TRANS-RXN-249; cytochrome bo oxidase EcoCyc frame ID: RXN0-5268; succinate <br /> iJO1366 conditions             dehydrogenase EcoCyc frame ID: SUCCINATE-DEHYDROGENASE-UBIQUINONE- <br />                                                                 RXN; glucose PTS uptake EcoCyc frame ID: TRANS-RXN-157/RXN0-6717/RXN0- <br /> be largely identical (Table 2). Differences arose vari-    6718; fructose 6-phosphate aldolase EcoCyc frame ID: RXN0-313; <br /> ances proton translocation stoichiometries           dihydroxyacetone kinase EcoCyc frame ID: 2.7.1.121-RXN. <br /> the EcoCyc&#8211;18.0&#8211;GEM version NADH:ubiquinone <br /> oxidoreductase I (NADH-DEHYDROG-A-RXN) (4 H+ <br /> translocated 2 e&#8722; , proposed Treberg et al. [21])     EcoCyc summary enzyme. If consensus develops <br /> and iJO1366 version NADH:ubiquinone oxi-             3 H+ 2 e&#8722; view translocation stoichiom- <br /> doreductase (NADH16pp) (3 H+ translocated 2 e&#8722; ,            etry, future versions EcoCyc changed reflect <br /> as proposed Wikstrom Hummer [22]). The exact             fact. <br /> number protons translocated NADH:ubiquinone             Further numerical differences technical <br /> oxidoreductase issue open discussion sci-       consideration: EcoCyc cytochrome bo oxidase reaction <br /> entific literature, uncertainty described    stoichiometry written terms molecules of <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                      Page 5 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> oxygen, iJO1366 CYTBO3_4pp written terms                          Table 4 Comparison experimental anaerobic glucose- <br /> of half-molecules (1 O2 consumed vs. 0.5 O2 ).                                limited chemostat growth data EcoCyc&#8211;18.0&#8211;GEM <br />                                                                               iJO1366 constraint-based model predictions <br /> Comparison iJO1366 simulation chemostat data                         Anaerobic fluxes                        Experimental        EcoCyc       iJO1366 <br /> After completing basic validation biomass pro-                         Specific growth rate &#181; (hr&#8722;1 )                      0.30        0.24          0.24 <br /> duction energy generation, maximized rate                          Glucose uptake (mmol/gCDW /hr)                      10.0       10.00         10.00 <br /> of EcoCyc&#8211;18.0&#8211;GEM biomass metabolite set production                          O2 uptake                                           0.00        0.00          0.00 <br /> under minimal media conditions ensured                       NH4 uptake                                                      2.52          0.82 <br /> we obtained results comparable iJO1366 results for <br />                                                                               Sulfate uptake                                                  0.06          0.06 <br /> the conditions obtained using COBRA Toolbox. <br />                                                                               Phosphate uptake                                                0.23          0.22 <br /> Divergences addressed literature-based manual <br />                                                                               CO2 production                                                  0.04         -0.08 <br /> curation EcoCyc modification MetaFlux reac- <br /> tion sets. We compared extracellular flux                         H2 O production                                                 -1.92        -1.84 <br /> distributions resulting simulations                       H+ production                                       27.8        27.5 <br /> experimental data obtained carbon-limited chemo-                           Acetate production                                    7.5       8.29          8.23 <br /> stat environments aerobic anaerobic                            Formate production                                  11.3       17.37         17.25 <br /> conditions.                                                                   Succinate production                                  1.2       0.00          0.08 <br />   Tables 3 4 compare extracellular metabolite flux                        Ethanol production                                    8.7       8.13          8.08 <br /> results derived EcoCyc&#8211;18.0&#8211;GEM simulation,                              Metabolite uptake production rates units mmol/gCDW/hr. Growth <br /> iJO1366 simulation, experimental data [23,24]                         units hr&#8722;1 . Experimental data [24] [25]. Formate-hydrogen lyase <br />                                                                               (FHLMULTI-RXN) left inactive purposes comparison. <br /> the canonical cases aerobic anaerobic growth on <br /> glucose-limited chemostat culture. In simulations, the <br /> experimental rate glucose supply fixed <br /> constraint; nutrients secretions left                      optimal. During transition purely anaerobic <br /> unconstrained.                                                                aerobic domains, competing demands energy <br />   The behavior MetaFlux/EcoCyc&#8211;18.0&#8211;GEM simula-                            metabolism redox elimination cause characteristic <br /> tions similar regards behavior                        pattern mixed acid fermentation described Varma <br /> of COBRA/iJO1366 simulations. Respiration fermen-                         et al., ethanol, formate, finally acetate <br /> tation rates scaled nutrient uptake comparable                        production eliminated cell&#8217;s oxygen supply <br /> rates. The generally higher rates O2 uptake observed                       completely sufficient support aerobic respira- <br /> experimentally lend support lower practical efficiency                   tion. Figures 1 2 use Cellular Overview Omics <br /> of proton translation stoichiometry vivo, aug-                     Popup visualization functionalities Pathway Tools to <br /> mented respiratory inefficiencies futile cycling                   illustrate behavior EcoCyc&#8211;18.0&#8211;GEM a <br /> and generation reactive oxygen <span id='am-61' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. Both mod-                          transition anaerobicity aerobicity. <br /> els secrete expected 1:2:1 mix acetate, formate,                          Comparisons FBA-predicted extracellular <br /> and ethanol anaerobic growth glucose                           fluxes experimental fluxes EcoCyc&#8211;18.0&#8211; <br /> Varma et al. [25] originally identified stoichiometrically                 GEM iJO1366 FBA predictions agree closely <br />                                                                               with experimental flux results, <br /> Table 3 Comparison experimental aerobic glucose-                           correspondence simulation and <br /> limited chemostat growth data EcoCyc&#8211;18.0&#8211;GEM                            experiment quite close experimental fluxes <br /> and iJO1366 constraint-based model predictions                                consideration. This result expected given the <br /> Aerobic fluxes                       Experimental      EcoCyc     iJO1366 <br />                                                                               adaptation iJO1366 biomass function use in <br />                                                                               EcoCyc&#8211;18.0&#8211;GEM, use iJO1366 preceding <br /> Specific growth rate (&#181;) (1/hr)               0.300       0.272       0.288 <br />                                                                               reconstructions benchmarks development of <br /> Glucose uptake (mmol/gCDW /hr)                3.008       3.008       3.008 <br />                                                                               EcoCyc&#8211;18.0&#8211;GEM, use EcoCyc refer- <br /> O2 uptake                                     7.413       6.158       5.703   ence construction iJO1366 predecessors. <br /> NH4 uptake                                    2.367       2.899       3.021   The experimental measurements generally demonstrate <br /> Sulfate uptake                                            0.067       0.073   higher fluxes respiratory gases O2 CO2 than <br /> Phosphate uptake                                          0.260       0.267   simulated fluxes, suggesting degree respiratory <br /> CO2 production                                  7.38      6.850       6.288   inefficiency properly modeled FBA. Similarly, <br /> H2 O production                                         13.926       13.704   small quantities succinate lactate produced <br /> H+ production                                             2.454       2.552   experimental fermentation, indicating degree of <br /> Metabolite uptake production rates units mmol/gCDW/hr. Growth   divergence metabolic optimality vivo. Broader <br /> is units hr&#8722;1 . Experimental data Kayser et al. [23].              cellular constraints regulation, protein crowding, <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                           Page 6 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />  Figure 1 Pathway Tools visualization EcoCyc&#8211;18.0&#8211;GEM flux aerobic transition. Example visualization EcoCyc&#8211;18.0&#8211;GEM flux <br />  transition anaerobic aerobic growth, created interactive Cellular Overview diagram Pathway Tools. The upper bound <br />  glucose uptake set 10 mmol/gCDW/hr, upper bound oxygen uptake increased 0 20 mmol/gCDW/hr 2.5 <br />  mmol/gCDW/hr steps. Omics Popups used illustrate flux acetaldehyde dehydrogenase, pyruvate-formate lyase, phosphoglucose <br />  isomerase, glyceraldehyde 3-phosphate dehydrogenase, <span id='am-208' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-209' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-211'  ></span><span id='am-212' property="rdfs:label" content="Cis" datatype="xsd:string"></span>cis</span>-aconitate hydratase, valine biosynthesis. <br />  <br />  <br />  <br /> pathway enzyme synthesis requirements, pathway-                        Gene essentiality analysis <br /> throughput limits underlie differences [17,26,27].                   One exciting aspects genome-scale flux <br /> Successive generations evolution constant                         modeling ability rapidly test computational gene <br /> growth conditions bring experimental result                      knockouts (KOs) effects metabolic function. <br /> closer theory, described Ibarra et al. [28].                      Gene KO simulation useful prediction and <br />  <br />  <br />  <br />  <br />  Figure 2 Pathway Tools visualization mixed-acid fermentation flux aerobic transition. Visualization EcoCyc&#8211;18.0&#8211;GEM flux in <br />  mixed-acid fermentation transition anaerobic aerobic growth, created EcoCyc mixed-acid fermentation pathway page <br />  Pathway Tools. The upper bound glucose uptake set 10 mmol/gCDW/hr, upper bound oxygen uptake increased 0 to <br />  20 mmol/gCDW/hr 2.5 mmol/gCDW/hr steps. Omics Popups used illustrate changes flux mixed-acid fermentation products <br />  formate, acetate, ethanol cellular energy redox balance evolves aerobic transition. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                           Page 7 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> for validation: silico FBA screens gene KOs         criteria originally defined relative terms Joyce <br /> been applied variety metabolic engineering efforts     et al., slowest-growing ninth Keio collection <br /> [29-31], E. coli KO library collections well-         deletion mutants. <br /> characterized growth behavior provide important tool          In absolute terms, approach treats essential <br /> for flux model validation.                                     deletion mutants measured Baba et al. have <br />    FBA gene KO essentiality prediction depends          OD &#8804; 0.091 24 hr growth glucose minimal media, <br /> types database associations genes chem-         indicates impaired growth shorter period. <br /> ical reactions: genes products catalyze reactions,       The practical difference perspectives <br /> and genes products reaction substrates (e.g.,        illustrated Figure 3, displays distribution <br /> acyl-carrier protein). Simulation gene KOs carried         OD600 data rich media-viable Keio collection <br /> out identifying reactions involving gene,       mutants 24 hr growth MOPS media contain- <br /> then identifying genes capable catalyzing         ing 0.4% glucose, originally published Supplementary <br /> the reactions supplying substrates identi-         Table Baba et al. As figure illustrates, the <br /> fied. Reactions isozymes alternative sub-      broad essentiality criteria include population cells <br /> strate supplies removed FBA             severe growth defects contained the <br /> stoichiometric network. An FBA solution calcu-         narrow essentiality data. The comparison narrow <br /> lated new model. If simulated gene KO          broad essentiality criteria expanded glycerol <br /> caused deletion reactions required          minimal media comparing narrow glycerol essential- <br /> for synthesis biomass metabolite, generation       ity criteria observed growth rich media the <br /> the biomass metabolite set blocked        glycerol essentiality criteria Orth et al., derived <br /> FBA simulation returns no-growth result. Such result       criteria Joyce et al. involving successive divi- <br /> represents prediction gene essentiality. If com-      sion thirds. <br /> plete biomass metabolite set produced            In order examine criteria experimental gene <br /> spite simulated gene knockout, FBA simulation       essentiality deeply illustrate effect of <br /> returns growth result, indicating prediction gene       defining core biomass metabolite set, conducted <br /> non-essentiality.                                              essentiality testing using expanded core <br />    The experimental essentiality data used tests con-   biomass metabolite sets proposed Orth et al. Differ- <br /> sisted major datasets. The first, used study gene    ences essentiality predictions data <br /> essentiality rich glucose minimal media, the <br /> deletion study Baba et al. [32] updated Yamamoto <br /> et al. [33], tested Keio collection library of <br /> 4288 E. coli gene deletion strains growth LB rich <br /> media MOPS minimal media 0.4% glucose. We <br /> conducted glycerol minimal media tests using the <br /> gene knockout essentiality data Joyce et al. [34], an <br /> expansion study Keio collection essential- <br /> ity include growth M9 minimal medium 1% <br /> glycerol. <br />    Several E. coli gene deletions strongly affect growth <br /> on various types minimal media, nonessen- <br /> tial growth rich media. Because FBA simulation <br /> result treated binary test (growth no-growth), <br /> gene deletions strongly affect growth minimal <br /> media producing completely lethal phenotype <br /> must defined experimentally essential as <br /> experimentally nonessential. <br />    Two representative perspectives <span id='am-248' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> are <br /> the narrow essentiality criteria observable growth in <br /> minimal media broad essentiality criteria used by <br /> Orth et al. The narrow glucose essentiality criteria treat      Figure 3 Essentiality criteria basis high-throughput KO data. <br /> as essential Baba et al. Yamamoto et al. gene         Histogram OD600 measurements rich media-viable Baba <br /> deletion mutants OD600 &#8804; 0.005 24 48 hr          et al. deletion mutants 24 hr growth MOPS media <br />                                                                 containing 0.4% glucose. Data Supplementary Table of <br /> growth glucose minimal media. This requires per- <br />                                                                 Baba et al. <br /> ceptible growth long period. The broad essentiality <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                        Page 8 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> sets illustrated differences standard cell                              Table 6 Comparison experimental gene essentiality <br /> composition nominal conditions minimal                                results computational EcoCyc&#8211;18.0&#8211;GEM results for <br /> composition required cell growth.                                               aerobic growth MOPS medium 1% glycerol <br />    We simulated single gene KOs glucose glycerol                             KO growth glycerol (sim/exp)                BC         BE     NC      NE <br /> minimal media 1445 genes EcoCyc&#8211;18.0&#8211;GEM                                 True positive (growth/growth)                1165     1124      1195   1154 <br /> to test resulting EcoCyc&#8211;18.0&#8211;GEM gene                                  False positive (growth/no growth)              58          63     28     35 <br /> deletion mutants capable generating core and <br />                                                                                     False negative (no growth/growth)              22          63    102    143 <br /> expanded biomass metabolite sets sets nutrients <br /> based experimental culture media Baba et al.                              True negative (no growth/no growth)           200         195    120    113 <br /> and Joyce et al. Gene KO simulations capable generat-                            See Table 5 caption description column headings. <br /> ing growth scored nonessential, whereas <br /> gene KOs blocking generation biomass metabolite                              situations considerable development <br /> set scored essential.                                                       EcoCyc reference, examined greater <br />    We compared results simulated essentiality                           case growth glucose, reference to <br /> screen experimental gene essentiality results based                            E. coli literature. Our examination covered types <br /> on narrow broad gene essentiality criteria. Incor-                         incorrect gene deletion growth predictions. The first <br /> rect essentiality predictions addressed literature-                         type false positive growth prediction. These genes, <br /> based manual curation EcoCyc modification                                 experimentally essential conditions <br /> MetaFlux metabolite sets. Final essentiality prediction                             tested Baba et al., predicted nonessential by <br /> results curation summarized Tables 5 glu-                          EcoCyc&#8211;18.0&#8211;GEM. The second type false negative <br /> cose 6 glycerol. The overall accuracy prediction                         growth prediction. These genes, experi- <br /> for growth glucose core biomass metabo-                                 mentally essential conditions tested Baba <br /> lite set broad essentiality criteria 1375/1445                              et al., predicted essential EcoCyc&#8211;18.0&#8211; <br /> (95.2% accuracy, 99.0% sensitivity, 77.5% specificity). For                         GEM. <br /> prediction growth glycerol simu-                                  Tables 7, 8, 9, 10, 11 12 present broad categories <br /> lation conditions, overall accuracy prediction                           incorrect gene deletion predictions EcoCyc&#8211;18.0&#8211; <br /> 1365/1445 (94.5% accuracy, 98.1% sensitivity, 77.5% speci-                          GEM. Table 7 cover false predictions involving open ques- <br /> ficity). Sensitivity refers percentage gene                          tions E. coli biology, false predictions resulting from <br /> deletions resulting growth correctly identified                         interesting facets experimental simulation meth- <br /> by simulation, specificity refers percentage                        ods, situations special relevance. Table 8 <br /> gene deletions resulting growth correctly                            covers false predictions core glycolytic, pentose <span id='am-270' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-271' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-272'  ></span><span id='am-273' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>phos</span>- <br /> identified simulation.                                                           phate, Entner-Doudoroff, TCA cycle metabolism. <br />    Tables 5 6 illustrate gene essentiality pre-                        This highly interconnected region E. coli metabolism <br /> dictions EcoCyc&#8211;18.0&#8211;GEM differed number                                 contains isozymes opportunities reversibil- <br /> cases gene essentiality conclusions generated                              ity, presents challenge FBA essentiality pre- <br /> by high-throughput gene KO screening. Because                                 dictions absence complete regulatory model- <br />                                                                                     ing. Table 9 cover false predictions result <br />                                                                                     unmodeled regulation gene expression enzyme <br /> Table 5 Comparison experimental gene essentiality                                activity. Genes repressed Baba et al. experimen- <br /> results computational EcoCyc&#8211;18.0&#8211;GEM results                              tal growth conditions, insufficiently expressed isozymes, <br /> aerobic growth MOPS medium 0.4% glucose <br />                                                                                     cases enzyme inhibition fall category. <br /> KO growth glucose (sim/exp)                  BC        BE        NC        NE    Table 10 covers situations essentiality conclu- <br /> True positive (growth/growth)                 1175      1136      1204      1164    sions high-throughput essentiality screen differed <br /> False positive (growth/no growth)                58        64        29        36   significantly essentiality conclusions the <br /> False negative (no growth/growth)                12        51        78      118    E. coli K&#8211;12 literature. Table 11 covers false gene essen- <br />                                                                                     tiality predictions relating systems scope of <br /> True negative (no growth/no growth)            200       194       134       127 <br />                                                                                     EcoCyc&#8211;18.0&#8211;GEM&#8217;s biomass objective function. Finally, <br /> Four comparisons provided &#8212; gene essentiality evaluated regard to <br /> (B)road (N)arrow experimental essentiality criteria, regard a <br />                                                                                     Table 12 covers false gene essentiality predictions made <br /> (C)ore biomass metabolite set maximizes accuracy essentiality           result MetaFlux EcoCyc technical problems <br /> predictions (E)xpanded biomass metabolite set reflecting              discovered course study. <br /> experimental measurements healthy cells. The results comparisons <br /> are arranged based correspondence simulation experiment:                Several false gene essentiality predictions <br /> (1) true positive (simulation predicts growth, experiment shows growth),            described tables discussed work <br /> (2) false positive (simulation predicts growth, experiment shows respiration), <br /> (3) false negative (no simulated growth, experiment shows respiration), and <br />                                                                                     Kim Copley, examined essentiality con- <br /> (4) true negative (no simulated growth, experiment shows respiration).           clusions Baba et al. E. coli core metabolism with <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                              Page 9 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 7 False gene essentiality predictions resulting open questions E. coli biology gene essentiality <br /> Gene       HT      Sim      Conv       Citations           Comments <br /> argD       +        &#8211;         +        [35,36]             EcoCyc lists argD enzyme capable carrying N-succinyldiaminopimelate <br />                                                            aminotransferase reaction lysine biosynthesis. Cox Wang [35] demonstrate second, <br />                                                            far <span id='am-275' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-276' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-280'  ></span><span id='am-281' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-288' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>unidentified</span> enzyme ArgD catalyze DapC reaction. argD discussed [36]. <br /> cysC        &#8211;       +         &#8211;        [5,32,34,37,38]     Gene essentiality sulfate utilization pathway bypassed cysIJ experiments [32], <br /> cysH        &#8211;       +         &#8211;                            conducted MOPS buffer capable used sulfur source ssuEADCB <br /> cysN        &#8211;       +         +                            alkanesulfonate desulfonation sulfite. Joyce et al. [34] use MOPS-free M9 buffer and <br />                                                            consequently shows cysADQUW essentiality sulfate uptake. The MOPS desulfonation <br />                                                            explanation data [32] complicated fact sulfite MOPS bypass <br />                                                            cysNDCH assimilatory sulfate reduction pathway. Instead, deletions fall or <br />                                                            near broad essentiality criteria [5]; cysN barely broad essentiality criteria (OD600 <br />                                                            0.088 24 hr) cysD just (OD600 0.104 24 hr) cysCH deep cutoff. <br />                                                            We propose explanation based inactivation ssu pathway transcriptional regulator <br />                                                            Cbl binding adenosine 5&#8217;-phosphosulfate (APS), product CysND sulfate <br />                                                            adenylyltransferase [37,38]. cysND mutants express ssu pathway able better <br />                                                            support growth MOPS catabolism. cysCH mutants produce APS extracellular sulfate, <br />                                                            leading Cbl inactivation APS binding, repression ssu, weaker growth. <br /> dapF       +        &#8211;         +        [36,39,40]          EcoCyc lists dapF enzyme capable converting LL-diaminopimelate to <br />                                                            meso-diaminopimelate lysine biosynthesis, [39] demonstrates dapF null mutants grow <br />                                                            unsupplemented glucose minimal media. dapF described [36]. <br /> dut         &#8211;       +         &#8211;        [41,42]             NudI MazG substitute Dut activity EcoCyc&#8211;18.0&#8211;GEM. The NudI Km the <br />                                                            dUTP-consuming reaction shared Dut mM range. MazG activity 70% inhibited by <br />                                                            MazEF toxin-antitoxin system; [42]. el-Hajj et al. [41] discuss Dut greater length. <br /> folB       +        &#8211;        N/A       [32,34,43]          folB reported nonessential [32], essential [34], despite obvious reason for <br />                                                            differential essentiality glucose glycerol. Haussmann et al. [43] studied enzyme did <br />                                                            construct deletion mutant test glucose minimal media; attempt made <br />                                                            unknown. folB upstream genes reported essential [32] folate biosynthesis <br />                                                            pathway, including folK, folC, folA, possibly folP (see below). <br /> folP        &#8211;       &#8211;        N/A       [32,33,44,45]       To knowledge, clear folP hemE null mutant test glucose minimal <br /> hemE        &#8211;       &#8211;        N/A                           media. A folP deletion mutant grows poorly rich media according [45]. folP hemE gene <br />                                                            duplications preventing observation null phenotype [32] identified [33,36] and <br />                                                            genes described uncertain essentiality [33]. Genes uncertain essentiality [33] <br />                                                            partial duplication isolates considered nonessential [44], which <br />                                                            tested culture growth rich Antibiotic Medium 3 medium containing beef extract opposed to <br />                                                            LB agar [33]. Under broad essentiality criteria [6], genes uncertain essentiality in <br />                                                            [33] considered nonessential. EcoCyc&#8211;18.0&#8211;GEM supports conclusion essentiality for <br />                                                            folP hemE. <br /> ftsW        &#8211;       +         &#8211;        [46-50]             It clear FtsW, MurJ, carry lipid II flippase activity E. coli. See the <br /> murJ        &#8211;       +         &#8211;                            listed references additional information topic. <br /> kdsC        &#8211;       +         +        [51-53]             kdsC upstream essential genes kdsB waaA CMP-KDO biosynthesis pathway. <br />                                                            Sperandeo et al. [51] suggeststhat isozymes KdsC&#8217;s 3-deoxy-D-manno-octulosonate <br />                                                            8-phosphate phosphatase activity exist. Nonspecific phosphatase activity carry <br />                                                            reaction. See [52,53] subject CMP-KDO requirements E. coli temperature <br />                                                            sensitivity. <br /> pabC       +        &#8211;         &#8211;        [32,34,36,54]       Green et al. [54] report PABA, product pabC, required growth minimal media <br />                                                            (although carbon source used media described) establishes one <br />                                                            copy gene exists E. coli. PabC upstream enzymes reported essential [32] its <br />                                                            pathway. Kim Copley [36] hypothesize nutrient carryover rich culture lack of <br />                                                            essentiality [32]. pabC nonessential M9 glycerol medium [34]. <br /> Cases EcoCyc&#8211;18.0&#8211;GEM essentiality predictions differed experimental gene-essentiality results aerobic growth MOPS medium 0.4% glucose, <br /> and posed open biological questions highlight metabolic network interactions particular EcoCyc&#8211;18.0&#8211;GEM. Certain genes deserve further <br /> investigation experimental community; highlight interesting aspects essentiality testing. Kim Copley [36] discussed genes, <br /> which remain open issues literature. See text additional details. Column headings follows: HT: High-throughput experiment (loose essentiality criteria <br /> of [6]). Sim: Simulation. Conv: Conventional experiment. Column entries follows: &#8211;: In experiments, deletion mutant recovered testing, or <br /> was tested did grow 0.091 OD600 24 hr, isolates duplications. In simulations, FBA biomass flux zero. <br /> +: In high-throughput experiments, deletion mutant tested showed growth greater 0.091 OD600 24 hr. In conventional experiments, <br /> growth observed. In simulations, FBA biomass flux positive. N/A: Information available deletion mutant glucose minimal media. For example, argD <br /> is essential data [32] according broad essentiality criteria [6], EcoCyc&#8211;18.0&#8211;GEM predicts nonessential. <br />  <br />  <br />  <br /> reference then-current state EcoCyc. Constraint-                            et al. examined COBRA <span id='am-40' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> of <br /> based model improvement gap-filling based gene                                 constraint-based models E. coli metabolism Reed <br /> essentiality predictions derived work Baba                                et al. [140], Kumar et al. [141], Kumar Maranas <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                            Page 10 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 8 False gene essentiality predictions glycolytic TCA cycle metabolism <br /> Gene       HT      Sim       Conv       Citations          Comments <br /> aceE        +       +          &#8211;        [32,36,55-57]      Baba et al. [32] observe aceE null mutant growth OD600 0.353 (nonessential), aceF null mutant <br /> aceF        &#8211;       +          &#8211;                           growth OD600 0.091 (the line loose essentiality), lpd null mutant growth OD600 0.061 <br /> lpd         &#8211;       +          &#8211;                           (loose essential). Langley Guest [55] observe pyruvate dehydrogenase complex essentiality and <br />                                                            [56,57] discuss context poxAB; [36] suggest mutants overexpressing pyruvate oxidase <br />                                                            allow growth glucose. <br />  <br /> eno         &#8211;       +          &#8211;        [36,58,59]         Hillman Fraenkel [58] gapA essentiality [59] describes eno/gapA/pgk <br /> gapA        &#8211;       +          &#8211;                           essentiality. Null mutants enzymes suffer glucose toxicity glucose <br /> pgk         &#8211;       +          &#8211;                           catabolite repression carbon utilization pathways. See [36]. <br />  <br /> icd         &#8211;       +          &#8211;        [32,36,60-67]      EcoCyc&#8211;18.0&#8211;GEM icd deletion mutants grow condensation propionyl-CoA glyoxylate <br />                                                            2-hydroxyglutarate [60,61], oxidation 2-hydroxyglutarate 2-oxoglutarate LhgO [66]. <br />                                                            icd broadly essential minimal glucose media [32]. icd deletion mutants require glutamate <br />                                                            [62,63] grow LB [64,65], [67] reports growth M9 minimal media. <br />                                                            See [36]. <br />  <br /> fbaA        &#8211;       +          &#8211;        [36,68-71]         The <span id='am-48' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> I fructose bisphosphate aldolase fbaB listed isozyme fbaA EcoCyc, is <br />                                                            expressed gluconeogenic conditions [68,69]. Use fsa/dhaK pathway [70,71] to <br />                                                            substitute fbaA appears blocked results [32] lowered dhaK expression E. coli <br />                                                            operational phosphotransferase systems. Other effects fbaA deletion discussed [36]. <br />  <br /> gltA        &#8211;       +          &#8211;        [36,72,73]         PrpC, 2-methylcitrate synthase propionate utilization pathway, isozyme GltA <br />                                                            citrate synthase. prpC conditionally expressed presence propionate. Discussed [36,73]. <br />  <br /> pfkA        &#8211;       +          &#8211;        [32,36,74,75]      pfkA pfkB listed isozymes 6-phosphofructokinase reaction, PfkB activity is <br />                                                            insufficient allow strong growth vivo absence pfkA according [74]. pfkA loosely <br />                                                            essential (OD600 0.087) [32]. See [36] [75], indicate pfkA pfkB be <br />                                                            deleted block growth. <br />  <br /> ppc         &#8211;       +          +        [32,76,77]         Peng et al. [76] demonstrate growth ppc deletion mutants detectable Ppc activity M9 <br />                                                            glucose minimal media. ppc grows rich media narrowly essential minimal glucose <br />                                                            conditions high-throughput assay [32]. EcoCyc&#8211;18.0&#8211;GEM ppc deletion mutants grow via <br />                                                            activation glyoxylate shunt, agreement observations [76]. Patrick et al. [77] <br />                                                            identified overexpression osmoregulatory regulator EcfM uncharacterized <br />                                                            protein YccT capable rescuing ppc deletion mutants. <br />  <br /> tpiA        +       +          &#8211;        [32,36,78,79]      Kim Copley [36] tpiA essentiality based methylglyoxal formation tpiA null <br />                                                            mutants, tpiA nonessentiality result [32] based methylglyoxal pathway-expressing <br />                                                            mutants [78,79]. <br /> The available pathway options navigating route glucose TCA cycle make superpathway glycolysis, pentose phosphate pathway <br /> and Entner-Doudoroff pathway. Without modeling regulation, product inhibition, metabolite toxicity, multiple re-entry points superpathway and <br /> the reversible nature pentose phosphate pathway allow carbon flux route deletions weak growth. EcoCyc&#8211;18.0&#8211;GEM consequently produces <br /> several false positive results pathways. This set genes previously discussed length [36]. The aceE tpiA genes identified essential in <br /> Figure [36] judged nonessential [32] consequently [6], predicted nonessential EcoCyc&#8211;18.0&#8211;GEM, nonessentiality <br /> conclusions appear incorrect based conventional experiments described literature, included table. The converse true ppc, <br /> which considered essential high-throughput experiments, essential conventional experiment. See Table 7 caption description of <br /> column headings. <br />  <br />  <br />  <br />  <br /> [142], Barua et al. [143], Orth Palsson [73,144],                             Nutrient utilization analysis <br /> Tervo Reed [145]. Our revisions EcoCyc&#8211;18.0&#8211;                                   Observation culture growth various nutrient sources <br /> GEM included manual application subset Grow-                                  foundation microbiology [146]. EcoCyc 18.0 con- <br /> Match [142] gap-filling methods, specifically resolution                              tains information E. coli respiration 428 types of <br /> of false positive gene essentiality predictions associated                            media, including 22 conventional types minimal growth <br /> with blocked genes false negative results associated                              media 383 Biolog Phenotype Microarray (PM) wells. <br /> with secretion metabolites. The essentiality prediction                            The 383 Biolog PM media conditions represent high- <br /> accuracy resulting manual curation process                                throughput method evaluating metabolic phenotypes <br /> similar accuracy resulting applying                              culture based tetrazolium dye assay cellular res- <br /> GrowMatch algorithm iAF1260 model.                                             piration. Each Biolog 96-well PM plate contains <br />   Additional file 2: Table S3 provides detailed listings                              standard minimal media composition plus nutrient <br /> of essentiality status model predictions, including                             source varied PM plate, element <br /> breakdown gene essentiality prediction status crite-                            supplied varying nutrient source dependent the <br /> ria used.                                                                             type Biolog PM plate use [147-149]. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                            Page 11 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 9 False gene essentiality predictions resulting isozymes pathways operational experimental <br /> conditions Baba et al. <br /> Gene     HT    Sim     Conv     Citations      Comments <br /> aroE     &#8211;      +        &#8211;      [80,81]        ydiB encodes isozyme AroE shikimate dehydrogenase EcoCyc. Johansson and <br />                                                Liden [81] suggest NAD+ /NADP+ specificity YdiB high intracellular NAD+ <br />                                                concentrations lead operate &#8220;reversed&#65533;? shikimate dehydrogenase direction, opposed <br />                                                biosynthetic direction chorismate. <br />  <br /> can      &#8211;      +        &#8211;      [82]           cynT, carbonic anhydrase cyn cyanate degradation operon, encodes isozyme Can <br />                                                carbonic anhydrase reaction EcoCyc. cynT conditionally expressed presence cyanate, <br />                                                absent [32] minimal media conditions. <br />  <br /> folA     &#8211;      +        &#8211;      [83,84]        folM, 7,8-dihydromonapterin reductase tetrahydromonapterin biosynthesis pathway a <br />                                                weak 7,8-dihydrofolate reductase activity, encodes isozyme FolA dihydrofolate reductase <br />                                                reaction EcoCyc. folM insufficiently expressed vivo supply E. coli growth requirements for <br />                                                tetrahydrofolate. <br />  <br /> folD     &#8211;      +        &#8211;      [85]           10-formyl-tetrahydrafolate formation folD shortcircuited EcoCyc&#8211;18.0&#8211;GEM purN&#8217;s <br />                                                reversible phosphoribosylglycinamide formyltransferase activity. PurN&#8217;s kcat substantially higher <br />                                                forward direction reverse direction vitro, vivo reversibility the <br />                                                enzyme uncertain. <br />  <br /> glyA     &#8211;      +        &#8211;      [77,86,87]     The threonine dehydrogenase Tdh 2-amino-3-ketobutyrate CoA ligase Kbl provide an <br /> serA     &#8211;      +        &#8211;                     alternate, threonine-based route <span id='am-29' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> synthesis serine GlyA serine <br />                                                hydroxymethyltransferase. tdh kbl control Lrp leucine repressor system, <br />                                                conditionally expressed presence leucine, absent [32] minimal media <br />                                                conditions. Patrick et al. [77] determined overexpression Tdh, LtaE low-specificity <br />                                                threonine aldolase, YneH glutaminase, Rsd anti-sigma factor led rescue glyA deletion <br />                                                mutants, YneH capable rescuing serA null mutants. <br /> guaB     &#8211;      +        &#8211;      [88]           guaB essential growth glucose [32], [88] indicates guaB mutants require <br />                                                guanine growth. In EcoCyc, nucleotide salvage pathways allow degradation IMP to <br />                                                inosine, subsequent conversion inosine xanthine XapA/DeoD XdhA, finally <br />                                                conversion xanthine XMP Gpt. <br />  <br /> ilvA     &#8211;      +        &#8211;      [77,89]        The threonine dehydratase TdcB acts isozyme IlvA threonine deaminase EcoCyc. <br />                                                tdcB expressed anaerobic conditions. Patrick et al. [77] identified TdcB EmrD <br />                                                multidrug efflux transporter overexpression capable rescuing ilvA deletion mutants. <br />  <br /> lipB     &#8211;      +        &#8211;      [90]           The lipoyl-carrier protein N6 -octanoyl-L-lysine intermediate lipoate synthesis produced <br />                                                LipB LplA EcoCyc. LplA primarily involved assimilation extracellularly <br />                                                sourced lipoate. See [90] details. <br /> metC     &#8211;      +        &#8211;      [77,91-93]     MalY&#8217;s &#946;-cystathionine lyase activity listed isozyme MetC EcoCyc. malY involved <br />                                                complex regulatory interactions, expression repressed malI WT strains. Without <br />                                                appropriate signaling, malY expressed. Patrick et al. [77] rescued metC deletion mutants via <br />                                                MalY, Alr alanine racemase, FimE phase-variation switch regulator overexpression. <br />  <br /> metL     &#8211;      +        &#8211;      [36,94]        E. coli&#8217;s aspartokinases, ThrA, MetL, LysC, isozymes aspartokinase reaction in <br /> thrA     &#8211;      +        &#8211;                     EcoCyc. The end-product inhibition of enzymes prevent aspartokinase gene <br />                                                KOs rescued isozymes, adequate amino acid pools pathways of <br />                                                remaining isozymes inhibit activity. Kim Copley [36] suggest metL not <br />                                                expressed glucose. <br /> nrdA     &#8211;      +        &#8211;      [95]           The NrdDE ribonucleoside diphosphate reductase acts isozyme NrdAB ribonucleoside <br /> nrdB     &#8211;      +        &#8211;                     diphosphate reductase EcoCyc. nrdDE expressed anaerobic conditions. <br /> pdxB     &#8211;      +        &#8211;      [77,96,97]     EcoCyc&#8211;18.0&#8211;GEM overcome deletion serC pdxB using thrB/ltaE route from <br /> serC     &#8211;      +        &#8211;                     glycolaldehyde (produced FolB formed spontaneously 3-hydroxypyruvate supplied <br />                                                YeaB) production 4-phospho-hydroxy-threonine subsequently pyridoxal-5&#8217;- <br />                                                phosphate [96,97]. Replacement serC pdxB pathways requires overexpression of <br />                                                thrB yeaB/nudL, growth overexpression reported solid media [97], whereas <br />                                                assays [32] conducted liquid media. Patrick et al. [77] rescued pdxB deletion mutants <br />                                                Tdh threonine dehydrogenase PurF amidophosphoribosyl transferase overexpression, and <br />                                                serC deletion mutants YneH glutaminase overexpression. <br />  <br /> prs      &#8211;      +        &#8211;      [32,98-100]    The PRPP biosynthesis II pathway substitute prs deletion EcoCyc&#8211;18.0&#8211;GEM. This pathway <br />                                                based connection ribose 5-phosphate DeoB phosphomutase PhnN <br />                                                ribose 1,5 bisphosphokinase activity putative ribose 1-phosphokinase activity [98,99]. PhnN is <br />                                                phn operon, [99,101] expression repressed 2 mM phosphate glucose <br />                                                minimal media conditions used [100]. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                           Page 12 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 9 False gene essentiality predictions resulting isozymes pathways operational experimental <br /> conditions Baba et al. (Continued) <br /> serB        &#8211;         +         &#8211;        [77,102]           EcoCyc&#8211;18.0&#8211;GEM overcome deletion serB synthesizing serine threonine Tdh <br />                                                             reversible action GlyA. Ravnikar Somerville [102] isolated pseudorevertants containing <br />                                                             elevated levels Tdh ser deletions following growth media supplemented threonine, <br />                                                             leucine, arginine, lysine, methionine followed growth minimal media. Patrick et al. [77] <br />                                                             additionally identified overexpression Gph, HisB, YtjC phosphatases capable rescuing <br />                                                             serB deletion mutants. <br /> False KO predictions caused presence isozymes able catalyze reaction model, down-regulated media <br /> conditions [32] reasons unlikely substitute knocked-out enzyme vivo. Because MetaFlux does model regulation, assumes that <br /> these enzymes active. See Table 7 caption description column headings. <br />  <br />  <br />   We evaluated performance EcoCyc&#8211;18.0&#8211;GEM                                      Conventional media compositions growth results <br /> in predicting growth following available datasets:                           drawn literature. Aerobic Biolog PM nutri- <br /> (1) aerobic E. coli growth 22 common conven-                                  ent utilization assay results compiled four <br /> tional minimal growth media; (2) consensus estimates                                 different datasets captured EcoCyc: (1) own <br /> of respiration based different experimentalists&#8217;                             experiments; (2) dataset obtained B. Bochner; <br />                                                                                      recent publications (3) AbuOun et al. <br /> measurements aerobic Biolog 96-well plates PM1&#8211;4, <br />                                                                                      [150] (4) Yoon et al. [151]. Anaerobic Biolog PM <br /> representing 313 conflict-free growth observations; and <br />                                                                                      nutrient utilization assay results obtained B. <br /> (3) anaerobic Biolog PM1 plate assay surveying carbon                             Bochner. We did include data Baumler et al. <br /> source utilization absence oxygen, representing                            analysis Biolog PM results varia- <br /> 96 anaerobic growth observations. Biolog PM data stored                              tion culture conditions high degree conflict <br /> in EcoCyc measures utilization nutrients sources                               datasets aerobic anaerobic <br /> of carbon (PM1&#8211;2), nitrogen (PM3), sulfur (PM4),                                 conditions [152]. See Methods section additional <br /> phosphorus (PM4).                                                                    details. <br />  <br />  <br /> Table 10 Genes EcoCyc&#8211;18.0&#8211;GEM predictions identified likely errors high-throughput essentiality <br /> screening, EcoCyc&#8211;18.0&#8211;GEM predictions confirmed conventional essentiality experiments <br /> Gene       HT       Sim       Conv       Citations              Comments <br /> alsK        &#8211;        +         N/A       [32,103,104]           The circumstances leading gene&#8217;s essentiality [32] uncertain. Kim et al. 1997 <br />                                                                 [103] Poulsen et al. 1999 [104] demonstrate alsK required growth allose, <br />                                                                 demonstrate growth glycerol minimal media. Poulsen et al. [104] constructed alsK <br />                                                                 null mutants transposon insertions, demonstrating alsK required allose <br />                                                                 catabolism, renamed gene yjcT. alsK/yjcT listed essential rich media (LB) [32]. <br />  <br /> aroD        +        &#8211;          &#8211;        [32,36,105,106]        Baba et al. [32] suggest aroD null mutant grow glucose minimal media, [105] <br />                                                                 established E. coli K&#8211;12 aroD mutants require aromatic amino acids growth. <br />                                                                 See discussion [36] reference [106]. <br />  <br /> atpB        &#8211;        +          +        [73,107,108]           von Meyenburg et al. [107] demonstrate growth strains lacking intact ATP synthase on <br /> atpC        &#8211;        +          +                               glucose fermentable carbon source minimal media, reduced growth rates. <br /> atpE        &#8211;        +          +                               Growth atp null mutants discussed [73,108]. <br /> cydA        &#8211;        +          +        [73,109-114]           Green Gennis [109] demonstrate aerobic growth cyd mutants glucose minimal <br /> cydC        &#8211;        +          +                               media use cytochrome bo. See references discussion redundancy in <br />                                                                 E. coli cytochromes operation cytochrome bd-I. Discussed [73]. <br /> ptsH        &#8211;        +          +        [77,115-121]           Steinsiek Bettenbrock [120] Escalante et al. [121] review glucose uptake E. coli <br /> ptsI        &#8211;        +          +                               mutant strains glucose PTS defects. See references additional details. Patrick et al. <br />                                                                 [77] identified overexpression FucP fucose transporter, XylE xylose transporter, GalE <br />                                                                 UDP-glucose 4-epimerase capable rescuing ptsI deletion mutants. <br /> spoT        &#8211;        +          +        [73,122-126]           spoT &#8722; mutants grow slowly glucose minimal media [122,123]. Absence SpoT ppGpp <br />                                                                 hydrolase activity leads high levels ppGpp, inversely correlated growth <br />                                                                 rate [126]. See references additional details. Discussed [73]. <br />  <br /> ubiA        &#8211;        +          +        [127,128]              Cox et al. [127] constructed ubiquinone-free mutants K&#8211;12 capable growth on <br /> ubiD        &#8211;        +          +                               fermentable substrates including glucose. Wu et al. [128] constructed ubiA null mutants <br /> ubiE        &#8211;        +          +                               capable growing minimal media containing fermentable carbon sources. <br /> waaU        &#8211;        +          +        [129,130]              Essentiality glucose minimal media clearly determined. Klena et al. [129] <br />                                                                 constructed waaU null mutants demonstrated viability rich media, contradiction <br />                                                                 determination essentiality rich media [32]. <br /> See Table 7 caption description column headings. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                        Page 13 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 11 False gene essentiality predictions genes representing systems scope EcoCyc&#8211;18.0&#8211;GEM <br /> biomass function <br /> Gene       HT         Sim     Conv        Citations        Comments <br /> der         &#8211;         +         &#8211;         [131-133]        Der essential maintenance 50S ribosomal subunit stability. Der&#8217;s GTPase activity regulates <br />                                                            specificity interactions ribosomal subunit. Regulatory GTPase activities, as <br />                                                            Der, current scope EcoCyc&#8211;18.0&#8211;GEM. <br />  <br /> mrdA        &#8211;         +         &#8211;         [134,135]        MrdA FtsI, referred penicillin-binding proteins 2 3, essential cell division <br /> ftsI        &#8211;         +         &#8211;                          maintenance cell shape. Ogura et al. [135] determined mrdA essential K&#8211;12, while <br />                                                            [134] determined ftsI essential. Modeling unique roles mrdA ftsI the <br />                                                            current scope EcoCyc&#8211;18.0&#8211;GEM. <br />  <br /> suhB        &#8211;         +         &#8211;         [136]            Wang et al. [136] studied suhB null mutants, exhibit cold-sensitive phenotype, reported <br />                                                            unable grow 30&#176;C, grew 42&#176;C. Growth studied 37&#176;C, not <br />                                                            described. Heat cold sensitivity current scope EcoCyc&#8211;18.0&#8211;GEM. <br /> Examples systems include cold response cell envelope maintenance. The EcoCyc GEM biomass function expanded incorporate operation of <br /> these systems future versions. See Table 7 caption description column headings. <br />  <br />  <br />  <br />  <br /> Table 12 False gene essentiality predictions caused technical issues MetaFlux EcoCyc <br /> Gene             HT             Sim               Comments <br /> bioB              &#8211;               +               Lipoate biosynthesis final step biotin biosynthesis operational EcoCyc&#8211;18.0&#8211;GEM due <br /> birA              &#8211;               +               reaction mass imbalances EcoCyc 18.0. This leads presence dethiobiotin instead biotin in <br /> lipA              &#8211;               +               objective function, prevents biotinylation biotin carboxyl carrier protein. <br />  <br /> entD              &#8211;               +               EntDEFG activity active EcoCyc&#8211;18.0&#8211;GEM EntD presence individual EntE, <br />                                                   EntF, EntG activity EcoCyc addition activity describing overall EntDEFG complex, which <br />                                                   leads entD knockouts ineffective. <br />  <br /> entD              &#8211;               +               EntDEFG activity active EcoCyc&#8211;18.0&#8211;GEM EntD presence individual EntE, <br />                                                   EntF, EntG activity EcoCyc addition activity describing overall EntDEFG complex, which <br />                                                   leads entD knockouts ineffective. <br />  <br /> fdx               &#8211;               +               The IspG reaction EcoCyc MEP pathway incorrectly requires ferredoxin (encoded fdx) instead of <br />                                                   flavodoxin I [137]. <br />  <br /> iscS              &#8211;               +               The IscS cysteine desulfurase lacks chemical structure EcoCyc, preventing participation thiazole <br /> thiL              &#8211;               +               biosynthesis iron-sulfur cluster synthesis. This affects thiL essentiality blocking thiazole synthesis and <br />                                                   rendering EcoCyc&#8211;18.0&#8211;GEM unable synthesize thiamin. Due exogenous thiamin contamination the <br />                                                   experiments [32] (see [73]; note thiL apparently mislabeled thiI therein) omission <br />                                                   manifests itself incorrect prediction thiL nonessentiality, endogenous exogenous <br />                                                   thiamin pathways substitute knockouts final thiamin monophosphate kinase <br />                                                   activity ThiL. <br />  <br /> metE              &#8211;               +               MetaFlux does correctly model polymerization reactions, prevents synthesis of <br />                                                   5-methyltetrahydropteroyl tri-L-glutamate folate polyglutamylation. This turn prevents MetE&#8217;s <br />                                                   cob(I)alamin-independent methionine synthase reaction operating properly. <br />  <br /> metH              +               &#8211;               MetH&#8217;s cob(I)alamin-dependent methionine synthase activity does currently require cob(I)alamin <br />                                                   cofactor EcoCyc&#8211;18.0&#8211;GEM cofactor requirements accounted enzymatic reactions. <br />                                                   Because MetE inoperational result lack folate polyglutamylation (see above), incorrectly <br />                                                   operational MetH methionine synthase reaction essential. This ambiguity remedied in <br />                                                   future versions MetaFlux. <br />  <br /> ndk               +               &#8211;               ndk falsely predicted essential UDP kinase dTDP kinase activities provide routes <br />                                                   EcoCyc UTP dTTP, respectively. In vivo, ndk null mutants rescued broad substrate specificity <br />                                                   adk [36,138]. <br />  <br /> pyrI              +               &#8211;               The catalytic subunit PyrB aspartate transcarbamylase PyrBI active itself vitro [139]. The PyrBI <br />                                                   complex catalyzes physiologically regulated reaction, reaction assigned PyrBI <br />                                                   complex EcoCyc. This causes pyrI gene KO simulation block PyrBI aspartate transcarbamylation <br />                                                   activity entirely. <br />  <br /> trxB              +               &#8211;               The activity glutaredoxin pathway substitute thioredoxin pathway E. coli, vice versa. <br />                                                   This ability properly modeled EcoCyc&#8211;18.0&#8211;GEM result pathways&#8217; structure EcoCyc. <br /> These technical issues addressed future versions MetaFlux EcoCyc. See Table 7 caption description column headings. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                   Page 14 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />    Growth given type media tested con-                           structures substrates reaction X, X chem- <br /> structing simulated MetaFlux nutrient sets corresponding                        ically balanced?&#65533;? &#8220;What metabolic pathway(s) reaction <br /> to contents media question comparing                          X member of, adjacent reactions?&#65533;? <br /> EcoCyc&#8211;18.0&#8211;GEM growth predictions experimen-                              &#8220;Which E. coli enzymes inhibited ADP?&#65533;? &#8220;What <br /> tal growth results. Due absence enterobactin                          transcriptional regulators affect expression the <br /> iron uptake modeling EcoCyc, Fe3+ medium                          enzymes reaction X?&#65533;? Their ability answer these <br /> replaced Fe2+ . Anaerobic simulations prepared                        questions rapidly accurately strongly dependent on <br /> identically performed aerobic growth,                       model representation, software tools available for <br /> for removal oxygen nutrient set, inclusion                      querying visualizing representation, tightness <br /> of formate-hydrogen lyase reaction, removal                         tools integrated model, and <br /> of protoheme pyridoxal 5&#8217;-phosphate synthesis                           presence additional enriching information the <br /> requirement biomass.                                                   model. <br />    Literature-based EcoCyc curation appropriate mod-                           Existing E. coli models represented spreadsheet <br /> ifications MetaFlux metabolite sets used                             files SBML files, making tedious impossible <br /> address incorrect nutrient utilization predictions. The                         non-programmers answer preceding questions <br /> final results PM array validation curation                        directly files. Although SBML files be <br /> listed Table 13. Overall accuracy growth predic-                          imported software tools RAVEN Tool- <br /> tion aerobic Biolog PM assays 252/313 (80.5%),                          box [181] rbionet [182], practice approach <br /> with 70 assays evaluated experimental con-                       limited variations SBML encodings, the <br /> flicts (see Methods section). Anaerobic Biolog PM                           effort required install integrate multiple software <br /> assay predictions overall accuracy 74/96 (77.1%).                     tools disparate capabilities, limited visual- <br /> Aerobic growth tests conventional minimal media                              ization capabilities tools. More fundamentally, <br /> contained EcoCyc overall accuracy 22/22                            previous E. coli models capture (nor SBML <br /> (100.0%).                                                                       capture) additional enriching information that, not <br />    The overall accuracy nutrient utilization prediction                      required mathematical operation model, <br /> across aerobic anaerobic PM conventional                            greatly enhances ability validate understand a <br /> growth media 348/431 (80.7%). Tables 14, 15, 16                          model, answer preceding questions. Examples <br /> 17 provide detailed discussions false negatives                          enriching information present EcoCyc&#8211;18.0&#8211; <br /> false positives aerobic PM assays. Tables results                    GEM metabolite chemical structures, arrangements of <br /> anaerobic PM assays conventional growth media                           reactions metabolic pathways, gene regulatory <br /> available Additional file 2: Tables S8 S9, respec-                       information. Note introducing ad-hoc definitions of <br /> tively.                                                                         data (e.g., pathways) SBML &#8220;Notes&#65533;? field, or <br />                                                                                 introducing SBML links external databases, be <br /> Model readability accessibility                                             considered bounds: pathways cap- <br /> Scientists naturally need ask questions                              tured formally SBML specification today, is <br /> a metabolic model, &#8220;What chemical                               guarantee regarding interoperability software tools <br />                                                                                 ad-hoc data. <br /> Table 13 EcoCyc&#8211;18.0&#8211;GEM nutrient utilization prediction                           EcoCyc&#8211;18.0&#8211;GEM highly understandable because <br /> results                                                                         interactively queried visualized through <br /> Nutrient utilization       Aerobic PM      Anaerobic PM       Conventional <br />                                                                                 EcoCyc web site desktop Pathway Tools soft- <br /> (sim/exp)                                                                       ware, supports visualization metabolic path- <br />                                                                                 ways reaction diagrams; metabolite pages depict <br /> True positive                       137                 35                17 <br /> (growth/growth)                                                                 metabolite structures reactions metabolite is <br />                                                                                 involved in; depiction gene/reaction connections and <br /> False positive                        15                14                  0 <br /> (growth/no growth)                                                              genome organization genome browser; naviga- <br />                                                                                 tion E. coli gene regulatory network; con- <br /> False negative                        46                  8                 0 <br /> (no growth/growth)                                                              structing structured queries as: reactions <br />                                                                                 given metabolite; enzymes utilizing given <br /> True negative                       115                 39                  5 <br /> (no growth/no growth)                                                           cofactor; presentation text summaries cita- <br />                                                                                 tions explain support aspects model. <br /> Aerobic PM: 313 absolute consensus results Biolog Phenotype Microarray <br /> carbon, nitrogen, sulfur, phosphorus source tests (Biolog plates PM1&#8211;4)     In general, tools metabolic model visualiza- <br /> conducted aerobic conditions. Anaerobic PM: 96 Biolog Phenotype           tion tend comprehensive, less <br /> Microarray carbon source tests (Biolog plates PM1&#8211;4) conducted under <br /> anaerobic conditions. Conventional: 22 E. coli minimal growth media described <br />                                                                                 closely coupled model; [183-186] recent <br /> in EcoCyc.                                                                      reviews. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                           Page 15 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 14 Conflicts EcoCyc&#8211;18.0&#8211;GEM growth predictions experimental carbon source utilization data for <br /> aerobic growth Biolog PM plates 37&#176;C <br /> Carbon source                          HT      Sim   Comments <br /> Dextrins                               +        &#8211;    General dextrin uptake catabolism glg described EcoCyc, not <br />                                                      applied dextrins EcoCyc&#8211;18.0&#8211;GEM MetaFlux&#8217;s current inability model <br />                                                      polymerization reactions. <br />  <br /> Lactulose                              +        &#8211;    Lactulose taken MelB melibiose transporter [153], route uptake <br />                                                      present EcoCyc. Lactulose capable inhibiting LacY transport o-nitrophenyl-&#946;- <br />                                                      D-galactopyranoside [154], anaerobically fermented E. coli [155], route <br />                                                      catabolism unknown. <br />  <br /> Methyl-&#945;-D-galactopyranoside           +        &#8211;    Methyl-&#945;-D-galactopyranoside taken MelB melibiose transporter [153], is <br />                                                      capable inhibiting LacY transport o-nitrophenyl-&#946;-D-galactopyranoside [154]. The route <br />                                                      catabolism unknown. <br />  <br /> Methyl-&#946;-D-galactoside                 +        &#8211;    Methyl-&#946;-D-galactoside taken MglABC transporter MelB transporter, the <br />                                                      catabolic pathway unknown. It capable inhibiting LacY transport o-nitrophenyl- <br />                                                      &#946;-D-galactopyranoside [154], reported substrate LacZ [156]. Inside cell, <br />                                                      methyl-&#946;-D-galactoside acetylated LacA galactoside acetyltransferase [157], which <br />                                                      fate unclear. <br />  <br /> Methyl pyruvate                        +        &#8211;    Methyl pyruvate competitive inhibitor active pyruvate transport [158]. No <br />                                                      route uptake methyl pyruvate present EcoCyc, route catabolism is <br />                                                      unknown. <br />  <br /> Melibionate                            +        &#8211;    These compounds&#8217; route uptake unknown. <br /> 1-O-methyl-&#946;-D-glucuronate             +        &#8211; <br /> 3-O-&#946;-D-galactopyranosyl-D-arabinose   +        &#8211; <br /> Methyl D-lactate                       +        &#8211; <br /> Mono-methyl hydrogen succinate         +        &#8211; <br /> L-galactono-1,4-lactone                +        &#8211;    The route uptake unknown; catabolized ring opening L-galactonate, <br />                                                      D-galactono-1,4-lactone. <br />  <br /> Meso-tartrate                          +        &#8211;    PM experiments indicate meso-tartrate used carbon source E. coli, in <br />                                                      contradiction reports [159] [160]. meso-tartrate associated with <br />                                                      L/D-tartrate uptake processes EcoCyc. <br /> Bromosuccinate                         +        &#8211;    Bromosuccinate described literature irreversible inhibitor aspartate <br />                                                      transcarbamylase [161] taken pathways aspartate. No route <br />                                                      uptake present EcoCyc. <br />  <br /> 2-hydroxybutyrate                      +        &#8211;    The route uptake unknown. <br /> Citrate                                &#8211;       +     Most strains E. coli use citrate carbon source aerobic conditions because <br />                                                      lack transporter expression [162]; citrate/succinate antiporter CitT expressed under <br />                                                      anaerobic conditions, cosubstrate required generate reducing power to <br />                                                      form succinate [163]. MetaFlux does currently model gene regulation. <br />  <br /> Putrescine                             &#8211;       +     These nitrogenous compounds used carbon sources high-nitrogen <br /> 4-aminobutyrate                        &#8211;       +     conditions Biolog PM carbon source assay, given lack Ntr-mediated expression <br /> Ornithine                              &#8211;       +     catabolic pathways [164]. MetaFlux does currently model gene regulation. <br /> L-arginine                             &#8211;       +     Arginine used carbon source E. coli K&#8211;12 absence of <br />                                                      induction transport [165,166]. MetaFlux does currently model gene regulation. <br />  <br /> Cellobiose                             &#8211;       +     Cellobiose used carbon source E. coli K&#8211;12 inability abolish <br />                                                      repression ChbABC chitobiose/cellobiose PTS permease NagC [167,168]. <br />                                                      MetaFlux does currently model gene regulation. <br />  <br /> Glycine                                &#8211;       +     Biolog PM experiments employing glycine carbon source return consensus no-growth <br />                                                      result, EcoCyc&#8211;18.0&#8211;GEM predicts glycine used carbon source via <br />                                                      assimilation 5,10-methyltetrahydrofolate glycine cleavage system. This a <br />                                                      wasteful pathway, producing CO2 molecule 5,10-THF glycine molecule <br />                                                      taken up. We information conventional growth experiments assaying ability <br />                                                      E. coli K&#8211;12 use glycine carbon source. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                    Page 16 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 14 Conflicts EcoCyc&#8211;18.0&#8211;GEM growth predictions experimental carbon source utilization data for <br /> aerobic growth Biolog PM plates 37&#176;C (Continued) <br /> D-tartrate                    &#8211;                +               D-tartrate does support growth aerobic conditions experiments [160]. It <br />                                                                uses anaerobic TtdT transporter EcoCyc&#8211;18.0&#8211;GEM; DcuB transporter the <br />                                                                correct route entry D-tartrate anaerobic conditions [159]. <br />  <br /> Ethanolamine                  &#8211;                +               E. coli requires source cob(I)alamin catabolism ethanolamine the <br />                                                                adenosylcobalamin-dependent ethanolamine <span id='am-213' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-214' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-216'  ></span><span id='am-217' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span>ammonia</span>-lyase [169-172]. MetaFlux does not <br />                                                                currently model enzyme cofactor requirements. <br /> Column headings follows: &lt;Element&gt; Source: Nutrient source test. HT: High-throughput experiment. Sim: Simulation. Column entries follows: <br /> +: Nutrient support growth. &#8211;: Nutrient support growth. For example, D-fructose supports growth according consensus experimental aerobic <br /> Biolog PM assays recorded EcoCyc, EcoCyc&#8211;18.0&#8211;GEM predicts does support growth. <br />  <br />  <br />  <br />  <br />   The reaction fluxes computed EcoCyc&#8211;18.0&#8211;GEM                               databases. It provides comprehensive genome-scale cov- <br /> are understandable previous E.                               erage E. coli metabolic network, represent- <br /> coli models EcoCyc&#8211;18.0&#8211;GEM fluxes                                 ing gene function unprecedented degree of <br /> immediately painted EcoCyc Cellular Overview,                            accuracy. <br /> a zoomable diagram complete metabolic map E.                              Integration EcoCyc&#8211;18.0&#8211;GEM EcoCyc <br /> coli allows immediate visual inspection flux pat-                         database gives investigators working model <br /> terns. Although software tools exist visualizing                        access Pathway Tools bioinformatics data <br /> flux patterns metabolic networks, e.g., RAVEN                              visualization suite. This allows construction complex <br /> Toolbox, unlikely easily usable previ-                        database queries involving range biochemical <br /> ous E. coli models. For example, RAVEN Toolbox requires                           entities E. coli, visualization pathways and <br /> that user manually construct metabolic network                            reactions model change the <br /> diagram, days weeks effort. In con-                        course construction. As EcoCyc, EcoCyc&#8211;18.0&#8211; <br /> trast, Pathway Tools generates metabolic map diagrams                             GEM receive frequent updates remain abreast of <br /> algorithmically PGDB.                                                      recent research developments. <br />                                                                                     The process EcoCyc&#8211;18.0&#8211;GEM construction and <br />                                                                                   validation resulted 80 updates EcoCyc. <br /> Conclusions                                                                       These included expansion revision periplasmic <br /> EcoCyc&#8211;18.0&#8211;GEM demonstrates advantages                                    phosphatase activities updates sugar transport <br /> literate modeling based comprehensive organism                                 phosphotransferase modeling; correction of <br />  <br />  <br />  <br /> Table 15 Conflicts EcoCyc&#8211;18.0&#8211;GEM growth predictions experimental nitrogen source utilization data for <br /> aerobic growth Biolog PM plates 37&#176;C <br /> Nitrogen source          HT       Sim    Comments <br /> Guanine                   +        &#8211;     Guanine used nitrogen source E. coli [173,174] vivo, degradation does proceed past <br />                                          allantoin. EcoCyc&#8211;18.0&#8211;GEM able use guanine source ammonia means glucose deamination and <br />                                          immediate excretion xanthine urate. <br />  <br /> 5-aminopentanoate         +        &#8211;     Routes uptake catabolism unknown 5-aminopentanoate glucuronamide. <br /> Glucuronamide             +        &#8211; <br />  <br /> Ethanolamine              &#8211;       +      E. coli requires source cob(I)alamin catabolism ethanolamine adenosylcobalamin-dependent <br />                                          ethanolamine ammonia-lyase [169-172]. MetaFlux does currently model enzyme cofactor requirements. <br />  <br /> Allantoin                 &#8211;       +      Anaerobic conditions required E. coli use allantoin nitrogen source [175]. MetaFlux does currently <br />                                          model gene regulation. <br />  <br /> Nitrate                   &#8211;       +      Nitrate nitrite pass nitrate nitrite reductase pathways, operate anaerobic <br /> Nitrite                   &#8211;       +      conditions function assimilatory fashion E. coli [176-178]. <br />  <br /> L-tyrosine                &#8211;       +      E. coli lacks catabolic aromatic amino acid transaminase, preventing utilization L-tyrosine nitrogen <br />                                          source; path utilization EcoCyc&#8211;18.0&#8211;GEM involves tyrosine lyase used thiazole biosynthesis <br />                                          followed spontaneous dissociation 2-iminoacetate glyoxylate ammonium, biologically <br />                                          realistic production large quantities dead-end metabolites 5&#8217;-deoxyadenosine p-cresol. <br /> See Table 14 caption description column headings. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                 Page 17 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> Table 16 Conflicts EcoCyc&#8211;18.0&#8211;GEM growth predictions experimental sulfur source utilization data for <br /> aerobic growth Biolog PM plates 37&#176;C <br /> Phosphorus source                       HT        Sim      Comments <br /> dTMP                                     +         &#8211;       dTMP, GMP, thymidine 3&#8217;-monophosphate, thiophosphate substrates periplasmic <br /> Thiophosphate                            +         &#8211;       nonspecific phosphatases PhoA <span id='am-243' about='obo:NCBITaxon_475258' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_37569 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_119258 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_119271 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_513153 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-244' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-245'  ></span><span id='am-246' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Apha" datatype="xsd:string"></span>AphA</span>, enzymatic reactions covering metabolites <br />                                                            added course EcoCyc development. Other phosphorylated metabolites have <br />                                                            associated individual phosphatases, pathways utilization subject <br />                                                            future research. <br />  <br /> 3-phospho-D-glycerate                    +         &#8211;       The route phosphorus uptake sources unknown. Although catabolic pathways <br /> 2-phosphoglycolate                       +         &#8211;       phosphorylated metabolites exist EcoCyc, route phosphorus uptake may <br /> &#945;-D-mannose 1-phosphate                  +         &#8211;       involve nonspecific dephosphorylation. <br />  <br /> Glucosamine-6-phosphate                  +         &#8211; <br /> 6-phospho-D-gluconate                    +         &#8211; <br /> O-phospho-D-tyrosine                     +         &#8211; <br /> O-phospho-D-serine                       +         &#8211; <br /> N-phospho-L-arginine                     +         &#8211; <br /> Trimetaphosphate                         +         &#8211; <br /> 2-deoxy-D-glucose 6-phosphate            +         &#8211; <br /> Creatine-phosphate                       +         &#8211; <br /> Dithiophosphate                          +         &#8211; <br /> See Table 14 caption description column headings. <br />  <br />  <br />  <br /> incorrect compartment assignments; fixes L-lactate                           removal incorrect reactions gene-protein <br /> dehydrogenase action; revisions glutathione hydrol-                           relationships; numerous fixes reaction reversibility <br /> ysis; new transport reactions compounds identified                           directionality; compound class reassign- <br /> as nutrient sources Biolog PM testing; addition                           ments correct issues reaction instantiation; mass <br /> of MOPS catabolism alkanesulfonate pathway;                              rebalancing reactions; revisions ATP <br />  <br />  <br /> Table 17 Conflicts EcoCyc&#8211;18.0&#8211;GEM growth predictions experimental sulfur source utilization data for <br /> aerobic growth Biolog PM plates 37&#176;C <br /> Sulfur source                  HT      Sim       Comments <br /> L-methionine                    +        &#8211;       Methionine related compounds enable respiration Biolog PM assays supplied sulfur <br /> D-methionine                    +        &#8211;       source. A route catabolism present EcoCyc, methionine considered support sulfur <br /> N-acetyl-DL-methionine          +        &#8211;       requirements E. coli [179]. This result suggests investigation. <br /> Gly-Met                         +        &#8211; <br /> L-cystathionine                 +        &#8211; <br /> L-methionine S-oxide            +        &#8211; <br />  <br /> L-cysteine                      +        &#8211;       L/D-cysteine lack clear pathways uptake EcoCyc&#8211;18.0&#8211;GEM. This problem subject future <br /> D-cysteine                      +        &#8211;       EcoCyc development. <br />  <br /> Thiophosphate                   +        &#8211;       PhoA activity thiophosphate added future versions EcoCyc. <br />  <br /> Djenkolate                      +        &#8211;       The route uptake unknown. Catabolism proceed MetC [180]. <br /> Lanthionine                     +        &#8211; <br />  <br /> 3-sulfinoalanine                +        &#8211;       The route uptake unknown. SufS CsdA convert 3-sulfinoalanine alanine sulfite. <br />  <br /> Cysteamine                      +        &#8211;       Routes uptake catabolism cysteamine, dithiophosphate, hypotaurine, tetrathionate, and <br /> Dithiophosphate                 +        &#8211;       thiourea unknown. <br /> Hypotaurine                     +        &#8211; <br /> Tetrathionate                   +        &#8211; <br /> Thiourea                        +        &#8211; <br /> See Table 14 caption description column headings. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                  Page 18 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> synthase proton stoichiometry. These updates out-         sri.com/ptools/. The simulation tests constructed <br /> lined Additional file 2: Table S11.                        using Lisp scripting Pathway Tools Lisp API, <br />    The MetaFlux software improved          documented http://brg.ai.sri.com/ptools/api/. Further <br /> result FBA validation EcoCyc. These improve-        details construction EcoCyc&#8211;18.0&#8211;GEM be <br /> ments include upgrades compartmentalization han-           Additional file 1. <br /> dling, gene deletion code, <span id='am-218' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-219'  ></span><span id='am-220' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> transfer reac-          EcoCyc&#8211;18.0&#8211;GEM development employed the <br /> tion handling. MetaFlux solution log files      MetaFlux component Pathway Tools, documented in <br /> updated contain additional statistical information     Pathway Tools User&#8217;s Guide [15]. All simula- <br /> provide detailed explanation metabolic net-     tions run solving mode 2.7 GHz i7 MacBook <br /> work construction process user. Numerous updates       Pro 16 GB RAM. The &#8220;minimize-fluxes: yes&#65533;? option <br /> and revisions MetaFlux model E. coli      used taxicab norm minimization fluxes. For <br /> introduced effort. Biomass metabolite sets    additional information, Pathway Tools User&#8217;s <br /> have revised reflect work Orth et al.,     Guide. A MetaFlux .fba file demonstrating simulation of <br /> additional updates basis val-    aerobic growth E. coli BW25113 glucose included <br /> idation process order create accurate final    Additional file 3. <br /> product possible.                                               We note choice stoichiometric representa- <br />    Many questions E. coli modelers         tion equations affect flux balance solutions when <br /> experimentalists raised course EcoCyc&#8211;         minimization summed flux used objec- <br /> 18.0&#8211;GEM development. By highlighting questions         tive function; [187] details. Stoichiometric <br /> and presenting context EcoCyc         coefficients EcoCyc&#8211;18.0&#8211;GEM scaled to <br /> reference database, address interests general   provide minimum whole-integer stoichiometry. Applica- <br /> metabolic modeling audience E. coli experimen-         tion different scaling flux network lead <br /> talists interested using models explore results   altered flux solutions. <br /> and generate new leads research. We summarize         Briefly, MetaFlux creates stoichiometric metabolic <br /> questions here.                                               flux network run time metabolites and <br />    Experimental measurements respiratory fluxes         reactions contained Pathway Tools PGDB. During <br /> glucose-fed aerobic chemostat culture higher         network construction, MetaFlux removes PGDB reac- <br /> those predicted simulation, small quantities suc-   tions are: ambiguously instantiated (see below); <br /> cinate lactate generated experimental anaerobic    unbalanced having undetermined balance state; <br /> fermentations, suggesting interesting vivo deviations      disconnected network; marked physiologi- <br /> from theoretical silico optimality.                        cally irrelevant; involved polymerization; involved in <br />    A small number metabolic byproducts             polymer segment protein modification; lacking sub- <br /> removed directly cytosol secretion set        strates side; containing substrate entities are <br /> because lack known salvage excretion pathways.     described strings; possessed variable stoichiom- <br /> The fate metabolites interest.                 etry; 10,000 permutations must <br />    Several incorrect essentiality predictions associ-     checked instantiation. MetaFlux instanti- <br /> ated unclear cellular biomass requirements, path-        ates reactions containing compound classes, replacing the <br /> ways potential alternate routes catalysis, uncer-     class reactions mass-balanced reactions containing <br /> tain determinations essentiality glucose minimal        instances relevant compound classes. The resulting <br /> media, ambiguous missing gene function. Sim-           set metabolites reactions constitutes metabolic <br /> ilarly, nutrient utilization predictions identified    flux network operated MetaFlux. Enzymatic reac- <br /> number compounds lack clear pathways entry         tion gene-protein relationship data encoded the <br /> into metabolism capable supporting respiration     PGDB used associate reactions network <br /> and/or growth. Resolution uncertainties        enzymes genes appropriate. <br /> improve understanding E. coli function varying        COBRA [20,188] simulations iJO1366 E. coli <br /> environments.                                                 genome-scale reconstruction employed order <br />                                                               validate EcoCyc FBA simulations pro- <br /> Methods                                                       vide point comparison existing reference mod- <br /> The data used project obtained Eco-        els. The iJO1366 simulations performed using the <br /> Cyc version 18.0, bioinformatics flux analysis    COBRA Toolbox 5.0.0 MATLAB R2010a and <br /> procedures documented performed           cobrapy 0.2 <span id='am-264' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-265' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-269'  ></span>Python</span> 2.7.6. All iJO1366 simu- <br /> the Web desktop environment Pathway Tools           lations used taxicab norm minimization fluxes as <br /> 18.0 software. Pathway Tools downloaded,         described documentation optimizeCbModel <br /> with documentation example files, http://brg.ai.       function. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                     Page 19 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br />    Biomass metabolite sets constructed using          simulation. Because simulations nutrient utilization <br /> wild-type core biomass sets Orth et al. modi-       scored according growth growth, exper- <br /> fied according research findings experimental       imental Biolog PM results indicating &#8216;normal&#8217; &#8216;low&#8217; <br /> data gene essentiality. The ATP turnover require-           respiration combined &#8216;positive&#8217; result. The <br /> ment growth-associated maintenance costs (GAM)             majority Biolog PM tests (313/383) displayed con- <br /> was set 53.95 mmol ATP/gCDW, ATP                  sensus respiration respiration all <br /> turnover requirement non-growth-associated mainte-         experimental datasets used. In 70 383 cases, no <br /> nance costs (NGAM) set 3.15 mmol ATP/gCDW,              clear consensus reached. These divergent cases <br /> per Orth et al.                                                omitted nutrient-utilization assay valida- <br />    All coefficients biomass metabolite set represent    tion reliable conclusion reached <br /> millimoles (mmol) metabolite required gram         regarding results. See Additional file 2: Table S10 a <br /> of cell dry weight (gCDW). Coefficients nutrient        list omitted PM data. <br /> and metabolite sets represent specific uptake output <br /> fluxes, millimoles metabolite supplied gram     Additional files <br /> of cell dry weight hour (mmol/gCDW/hr). An overall <br /> biomass flux 1.0 represents specific growth rate      Additional file 1: Supplementary information text table <br /> &#956; 1.0 hr&#8722;1 (one new gram cell dry weight gram         descriptions. <br /> of cell dry weight hour).                                   Additional file 2: Supplementary tables (11 tables). <br />    Aerobic glucose chemostat data experimental com-         Additional file 3: MetaFlux .fba file demonstrating simulation of <br /> parisons obtained [23]. Anaerobic glucose             aerobic growth E. coli BW25113 glucose. <br /> chemostat data obtained [24] [25]. <br />    The protocols followed PM experiments            Competing interests <br />                                                                DSW, IMK, PDK receive fees SRI International commercial licensing of <br /> as follows: PM plates 1&#8211;4 containing 190 sole carbon           Pathway Tools. <br /> sources, 95 sole nitrogen sources, 59 sole phosphate <br /> sources 35 sole sulfur sources used anal-     Authors&#8217; contributions <br />                                                                PDK DSW conceived designed experiments. DSW constructed the <br /> ysis. E. coli MG1655 obtained Yale Coli           model conducted simulations. DSW analyzed data. IMK, AM, and <br /> Genetic Stock Center, pre-grown nutrient agar           DSW curated database. DSW PDK wrote manuscript. AM ITP <br /> used inoculate plates following Biolog instruc-         performed phenotype microarray experiments. All authors read approved <br />                                                                final manuscript. <br /> tions. The data collected analyzed using the <br /> OmniLogH PM system, records color change             Acknowledgements <br /> every 15 min 96 assay plates. All    The authors like thank Mario Latendresse Markus Krummenacker <br />                                                                MetaFlux development support, Larry Reitzer calling our <br /> incubations performed 37&#176;C 48 hr. For         attention problem dapC essentiality. This work supported by <br /> complete details PM assay conditions, refer   award number U24GM077678 National Institute General Medical <br /> the original publications.                                     Sciences. The content article solely responsibility authors <br />                                                                does necessarily represent official views National Institute of <br />    The PM nutrient-source assay assay respiration     General Medical Sciences National Institutes Health. <br /> based generation NADH carbon metabolism <br /> and subsequent reduction tetrazolium redox dye        Author details <br />                                                                1 Bioinformatics Research Group, SRI International, 333 Ravenswood Ave., <br /> by NADH. As such, does directly measure          94025 Menlo Park, CA, USA. 2 Department Chemistry Biomolecular <br /> the cell growth simulated FBA biomass objectives         Science, Macquarie University, Balaclava Rd, North Ryde NSW 2109, Australia. <br /> the uptake noncarbon sources. However, checkpoint <br />                                                                Received: 22 March 2014 Accepted: 19 June 2014 <br /> linkage carbon-source catabolism nitrogen, phos-         Published: 30 June 2014 <br /> phorus, sulfur source catabolism enables <span id='am-233' about='obo:NCBITaxon_1310384' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_83136 obo:NCBITaxon_1 obo:NCBITaxon_83139 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_426515 obo:NCBITaxon_6843 obo:NCBITaxon_426501 obo:NCBITaxon_33154 obo:NCBITaxon_6933 obo:NCBITaxon_6656 obo:NCBITaxon_70332 obo:NCBITaxon_6946 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_70320 obo:NCBITaxon_6947 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-234' property="rdfs:label" content="Tetra" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-237'  ></span>tetra</span>- <br /> zolium redox dye assay probe metabolism non-         References <br />                                                                1.   Orth JD, Thiele I, Palsson B&#216;: What flux balance analysis? Nat <br /> carbon sources [189]. Bochner describes phenomenon              Biotechnol 2010, 28(3):245&#8211;248. <br /> of checkpoint linkage starvation elemental nutri-       2.   Thiele I, Palsson B&#216;: A protocol generating high-quality <br /> ent source leading arrest cellular respiration,           genome-scale metabolic reconstruction. Nat Protoc 2010, <br />                                                                     5(1):93&#8211;121. <br /> to redox imbalance alarmone synthesis. We there-            3.   Edwards J, Palsson B&#216;: The Escherichia coli MG1655 silico <br /> fore compared PM respiration results directly FBA              metabolic genotype: definition, characteristics, capabilities. <br /> growth simulation.                                                  Proc Natl Acad Sci U S A 2000, 97:5528&#8211;5533. <br />                                                                4.   Reed JL, Vo TD, Schilling CH, Palsson B&#216;: An expanded genome-scale <br />    Biolog PM results stored EcoCyc database          model <span id='am-259' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-260' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-261' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli K-12</span> (iJR904 GSM/GPR). Genome Biol 2003, <br /> and accessible Pathway Tools API EcoCyc             4(9):54. <br /> website. Individual Biolog PM assay scores       5.   Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br />                                                                     Broadbelt LJ, Hatzimanikatis V, Palsson B&#216;: A genome-scale metabolic <br /> were compared experimental datasets estab-                reconstruction Escherichia coli K&#8211;12 MG1655 accounts for <br /> lish consensus comparison EcoCyc&#8211;18.0&#8211;GEM                1260 ORFs thermodynamic information. Mol Syst Biol 2007, 3:121. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                      Page 20 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> 6.    Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist AM, Palsson B&#216;: A           29.   Milne CB, Kim PJ, Eddy JA, Price ND: Accomplishments in <br />       comprehensive genome-scale reconstruction Escherichia coli                       genome-scale silico modeling industrial medical <br />       metabolism &#8212; 2011. Mol Syst Biol 2011, 7:535.                                       biotechnology. Biotechnol J 2009, 4(12):1653&#8211;16570. <br /> 7.    Majewski RA, Domach MM: Simple constrained-optimization view               30.   Gianchandani EP, Chavali AK, Papin JA: The application flux <br />       acetate overflow E. coli. Biotechnol Bioeng 1990, 35(7):732&#8211;738.                 balance analysis systems biology. WIREs Syst Biol Med 2010, <br /> 8.    Varma A, Boesch BW, Palsson B&#216;: Biochemical production capabilities                 2(3):372&#8211;382. <br />       Escherichia coli. Biotechnol Bioeng 1993, 42(1):59&#8211;73.                     31.   Lee JW, Na D, Park JM, Lee J, Choi S, Lee SY: Systems metabolic <br /> 9.    Varma A, Palsson B&#216;: Metabolic capabilities Escherichia coli: I.                 engineering microorganisms natural non-natural <br />       synthesis biosynthetic precursors cofactors. J Theor Biol 1993,              chemicals. Nat Chem Biol 2012, 8:536&#8211;546. <br />       165(4):477&#8211;502.                                                               32.   Baba T, <span id='am-51' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br /> 10.   Pramanik J, Keasling JD: Stoichiometric model Escherichia coli                   Tomita M, Wanner BL, Mori H: Construction Escherichia coli K&#8211;12 <br />       metabolism: incorporation growth-rate dependent biomass                          in-frame, single-gene knockout mutants: The Keio collection. Mol <br />       composition mechanistic energy requirements. Biotechnol                         Syst Biol 2006, 2(2006):0008. <br />       Bioeng 1997, 56(4):398&#8211;421.                                                   33.   Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, Touda Y, <br /> 11.   Oberhardt MA, Palsson B&#216;, Papin JA: Applications genome-scale                    Furubayashi A, Kinjyo S, Dose H, Hasegawa M, Datsenko KA, Nakayashiki <br />       metabolic reconstructions. Mol Syst Biol 2009, 5:320.                               T, Tomita M, Wanner BL, Mori H: Update collection of <br /> 12.   Lewis NE, Nagarajan H, Palsson B&#216;: Constraining metabolic                       Escherichia coli single-gene deletion mutants. Mol Syst Biol 2009, <br />       genotype-phenotype relationship using phylogeny silico                      5:335. <br />       methods. Nat Rev Microbiol 2012, 10(4):291&#8211;305.                               34.   Joyce AR, Reed JL, White A, Edwards R, Osterman A, Baba T, Mori H, <br /> 13.   McCloskey D, Palsson B&#216;, Feist AM: Basic applied uses                        Lesely SA, Palsson B&#216;, Agarwalla S: Experimental computational <br />       genome-scale metabolic network reconstructions Escherichia                       assessment conditionally essential genes Escherichia coli. <br />       coli. Mol Syst Biol 2013, 9:661.                                                    J Bacteriol 2006, 188(23):8259&#8211;8271. <br /> 14.   Keseler IM, Mackie A, Peralta-Gil M, Santos-Zavaleta A, Gama-Castro S,        35.   Cox RJ, Wang PSH: Is N-acetylornithine aminotransferase real <br />       Bonavides-Martinez C, Fulcher C, Huerta AM, Kothari A, Krummenacker                 N-succinyl-LL-diaminopimelate aminotransferase Escherichia <br />       M, Latendresse M, Muniz-Rascado L, Ong Q, Paley S, Schroder I, Shearer              coli <span id='am-1' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15'  ></span>Mycobacterium</span> smegmatis? J Chem Soc, Perkin Trans 2001, <br />       AG, Subhraveti P, Travers M, Weerasinghe D, Weiss V, Collado-Vides J,               1:2006&#8211;2008. <br />       Gunsalus RP, Paulsen I, Karp PD: EcoCyc: fusing model organism                36.   Kim J, Copley S: Why metabolic enzymes essential or <br />       databases systems biology. Nucleic Acids Res 2013,                             nonessential growth Escherichia coli K&#8211;12 glucose. <br />       41(Database issue):605&#8211;612.                                                         Biochemistry 2007, 46(44):12501&#8211;12511. <br /> 15.   Latendresse M, Krummenacker M, Trupp M, Karp PD: Construction             37.   van der Ploeg JR, Eichhorn E, Leisinger T: Sulfonate-sulfur metabolism <br />       completion flux balance models pathway databases.                           regulation Escherichia coli. Arch Microbiol 2001, 176:1&#8211;8. <br />       Bioinformatics 2012, 28:388&#8211;396.                                              38.   Bykowski T, van der Ploeg JR, Iwanicka-Nowicka R, Hryniewicz MM: The <br /> 16.   Karp PD, Paley SM, Krummenacker M, Latendresse M, Dale JM, Lee T,                   switch inorganic organic sulphur assimilation Escherichia <br />       Kaipa P, Gilham F, Spaulding A, Popescu L, Altman T, Paulsen I, Keseler             coli: adenosine 5&#8217;-phosphosulphate (APS) signalling molecule <br />       IM, Caspi R: Pathway Tools version 13.0: integrated software                    sulphate excess. Mol Microbiol 2002, 43(5):1347&#8211;1358. <br />       pathway/genome informatics systems biology. Brief Bioinform               39.   Richaud C, Higgins W, Mengin-Lecruelx D, Stragier P: Molecular <br />       2010, 11:40&#8211;79 doi:10.1093/bib/bbp043.                                              cloning, characterization, chromosomal localization dapF, <br /> 17.   O&#8217;Brien EJ, Lerman JA, Chang RL, Hyduke DR, Palsson B&#216;: Genome-scale                Escherichia coli gene diaminopimelate epimerase. J Bacteriol <br />       models metabolism gene expression extend refine                          1987, 169(4):1454&#8211;1459. <br />       growth phenotype prediction. Mol Syst Biol 2013, 9:693.                       40.   Mengin-Lecruelx D, Michaud C, Richaud C, Blanot D, van Heijenoort J: <br /> 18.   Knuth DE: Literate programming. Comput J 1984, 27(2):97&#8211;111.                        Incorporation LL-diaminopimelic acid peptidoglycan of <br /> 19.   Mackie A, Keseler IM, Nolan L, Karp PD, Paulsen IT: Dead end                        Escherichia coli mutants lacking diaminopimelate epimerase <br />       metabolites &#8212; defining known unknowns E. coli                            encoded dapF. J Bacteriol 1988, 170(5):2031&#8211;2039. <br />       metabolic network. PLoS One 2013, 8(9):75210.                                 41.   el-Hajj HH, Zhang H, Weiss B: Lethality dut (deoxyuridine <br /> 20.   Schellenberger J, Que R, Fleming RM, Thiele I, Orth JD, Feist AM, Zielinski         triphosphatase) mutation Escherichia coli. J Bacteriol 1988, <br />       DC, Bordbar A, Lewis NE, Rahmanian S, Kang J, Hyduke DR, Palsson B&#216;:                170(3):1069&#8211;1075. <br />       Quantitative prediction cellular metabolism                           42.   Gross M, Marianovsky I, Glaser G: MazG - regulator programmed <br />       constraint-based models: COBRA toolbox v2.0. Nat Protoc 2011,                   cell death Escherichia coli. Mol Microbiol 2006, 59(2):590&#8211;601. <br />       6:1290&#8211;1307.                                                                  43.   Haussmann C, Rohdich F, Schmidt E, Bacher A, Richter G: Biosynthesis <br /> 21.   Treberg JR, Brand MD: A model proton translocation                           pteridines Escherichia coli. Structural mechanistic <br />       mechanism complex I. JBC 2011, 286(20):17579&#8211;17584.                              similarity dihydroneopterin-triphosphate epimerase and <br /> 22.   Wikstrom M, Hummer G: Stoichiometry proton translocation                      dihydroneopterin aldolase. J Biol Chem 1998, 273(28):17418&#8211;17424. <br />       respiratory complex I mechanistic implications. PNAS 2012,            44.   Kato J, Hashimoto M: Construction consecutive deletions the <br />       109(12):4431&#8211;4436.                                                                  Escherichia coli chromosome. Mol Syst Biol 2007, 3:132. <br /> 23.   Kayser A, Weber J, Hecht V, Rinas U: Metabolic flux analysis               45.   Fermer C, Swedberg G: Adaptation sulfonamide resistance in <br />       Escherichia coli glucose-limited continuous culture. i.                          <span id='am-67' about='obo:NCBITaxon_482' typeof='owl:Thing obo:NCBITaxon_481 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_206351'><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="&quot;Merismopedia&quot; Zopf 1885" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Gonococcus" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Neisseria Trevisan 1885" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="&quot;Gonococcus&quot; Lindau 1898" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Nesseira" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:7520730" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Neisseria" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Neisseria</span> meningitidis required compensatory changes <br />       growth-rate-dependent metabolic efficiency steady state.                         retain enzyme function: kinetic analysis dihydropteroate <br />       Microbiology 2005, 151:693&#8211;706.                                                     synthases N. meningitidis expressed knockout mutant of <br /> 24.   Belaich A, Belaich JP: Microcalorimetric study anaerobic                     Escherichia coli. J Bacteriol 1997, 179(3):831&#8211;837. <br />       growth Escherichia coli: growth thermograms synthetic                 46.   Inoue A, Murata Y, Takahashi H, Tsuji N, Fujisaki S, Kato J: Involvement <br />       medium. J Bacteriol 1976, 72:497&#8211;499.                                               essential gene, mviN, murein synthesis Escherichia coli. <br /> 25.   Varma A, Boesch BW, Palsson B&#216;: Stoichiometric interpretation                    J Bacteriol 2008, 190(21):7298&#8211;7301. <br />       Escherichia coli glucose catabolism various oxygenation                 47.   Ruiz N: Bioinformatics identification MurJ (MviN) the <br />       rates. Appl Environ Microbiol 1993, 59(8):2465&#8211;2473.                                peptidoglycan lipid II flippase Escherichia coli. PNAS 2008, <br /> 26.   Zhuang K, Vemuri GN, Mahadevan R: Economics membrane                             105(21):15553&#8211;15557. <br />       occupancy respiro-fermentation. Mol Syst Biol 2011, 7:500.                48.   Mohammadi T, van Dam V, Sijbrandi R, Vernet T, Zapun A, Bouhss A, <br /> 27.   van Hoek MJA, Merks RMH: Redox balance key explaining vs.                MD-d Bruin, Nguyen-Disteche M, Kruijff B, Breukink E: Identification <br />       partial switching low-yield metabolism. BMC Syst Biol 2012, 6:22.                FtsW transporter lipid-linked cell wall precursors across <br /> 28.   Ibarra RU, Edwards JS, Palsson B&#216;: Escherichia coli K-12 undergoes                  membrane. EMBO J 2011, 30(8):1425&#8211;1432. <br />       adaptive evolution achieve silico predicted optimal growth.             49.   Jermy A: Bacterial physiology: flipping lipids. Nat Rev Microbiol 2011, <br />       Nature 2002, 420:186&#8211;189.                                                           9(5):314. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                    Page 21 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> 50.   Butler EK, Davis RM, Bari V, Nicholson PA, Ruiz N: Structure-function       72.   Gerike U, Hough DW, Russell NJ, Dyall-Smith ML, Danson MJ: Citrate <br />       analysis MurJ reveals solvent-exposed cavity containing                      synthase 2-methylcitrate synthase: structural, functional and <br />       residues essential peptidoglycan biogenesis Escherichia coli.              evolutionary relationships. Microbiology 1998, 144(4):929&#8211;935. <br />       J Bacteriol 2013, 195(20):4639&#8211;4649.                                        73.   Orth JD, Palsson B: Gap-filling analysis iJO1366 Escherichia coli <br /> 51.   Sperandeo P, Pozzi C, Deho G, Polissi A: Non-essential kdo                        metabolic network reconstruction discovery metabolic <br />       biosynthesis new essential cell envelope biogenesis genes                  functions. BMC Syst Biol 2012, 6:30. <br />       Escherichia coli yrbG-yhbG locus. Res Microbiol 2006,                   74.   Kotlarz D, Garreau H, Buc H: Regulation the <br />       157(6):547&#8211;558.                                                                   activity phosphofructokinases pyruvate kinases in <br /> 52.   Mamat U, Meredith TC, Aggarwal P, Kuhl A, Kirchoff P, Lindner B,                  Escherichia coli. Biochim Biophys Acta 1975, 381(2):257&#8211;268. <br />       Hanuszkiewicz A, Sun J, Holst O, Woodard RW: Single amino acid              75.   Lovingshimer MR, Siegele D, Reinhart GD: Construction an <br />       substitutions YhjD MsbA confer viability                          inducible, pfkA pfkB deficient strain Escherichia coli the <br />       3-deoxy-d-manno-oct-2-ulosonic acid-depleted Escherichia coli.                    expression purification phosphofructokinase bacterial <br />       Mol Microbiol 2008, 67(3):633&#8211;648.                                                sources. Protein Expr Purif 2006, 46:475&#8211;482. <br /> 53.   Klein G, Lindner B, Brabetz W, Brade H, Raina S: Escherichia coli, K&#8211;12     76.   Peng L, Arauzo-Bravo MJ, Shimizu K: Metabolic flux analysis ppc <br />       suppressor-free mutants lacking early glycosyltransferases                    mutant Escherichia coli based 13C-labelling experiments <br />       late acyltransferases: minimal lipopolysaccharide structure                   enzyme activity assays intracellular metabolite <br />       induction envelope stress response. J Biol Chem 2008,                          measurements. FEMS Microbiol Lett 2004, 235(1):17&#8211;23. <br />       284(23):15369&#8211;151389.                                                       77.   Patrick WM, Quandt EM, Swartzlander DB, Matsumura I: Multicopy <br /> 54.   Green JM, Merkel WK, Nichols BP: Characterization sequence                 suppression underpins metabolic evolvability. Mol Biol Evol 2007, <br />       Escherichia coli pabC, gene encoding aminodeoxychorismate                     24(12):2716&#8211;2722. <br />       lyase, pyridoxal phosphate-containing enzyme. J Bacteriol 1992,           78.   Ferguson GP, Totemeyer S, MacLean MJ, Booth IR: Methylglyoxal <br />       174(16):5317&#8211;5323.                                                                production bacteria: suicide survival? Arch Microbiol 1998, <br /> 55.   Langley D, Guest JR: Biochemical genetics &#945;-keto acid                      170(4):209&#8211;219. <br />       dehydrogenase complexes Escherichia coli K&#8211;12: Isolation             79.   Fong SS, Nanchen A, Palsson B&#216;, Sauer U: Latent pathway activation <br />       biochemical properties deletion mutants. J Gen Microbiol 1977,                 increased pathway capacity enable Escherichia coli adaptation <br />       99:263&#8211;276.                                                                       loss key metabolic enzymes. J Biol Chem 2006, <br /> 56.   Chang YY, Cronan JE: Mapping nonselectable genes Escherichia                   281(12):8024&#8211;8033. <br />       coli using <span id='am-24' about='obo:NCBITaxon_2389' typeof='owl:Thing obo:NCBITaxon_2387 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-25' property="rdfs:label" content="Transposon Tn10" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>transposon Tn10</span>: location gene affecting <br />                                                                                   80.   Michel G, Roszak AW, Sauve V, Maclean J, Matte A, Coggins JR, Cygler M, <br />       pyruvate oxidase. J Bacteriol 1982, 151(3):1279&#8211;1289. <br />                                                                                         Lapthorn AJ: Structures shikimate dehydrogenase AroE its <br /> 57.   Abdel-Hamid AM, Attwood MM, Guest JR: Pyruvate oxidase <br />                                                                                         paralog YdiB. common structural framework different <br />       contributes aerobic growth efficiency Escherichia coli. <br />                                                                                         activities. J Biol Chem 2003, 278(21):19463&#8211;19472. <br />       Microbiology 2001, 147:1483&#8211;1498. <br />                                                                                   81.   Johansson L, Liden G: Transcriptome analysis shikimic acid <br /> 58.   Hillman JD, Fraenkel DG: Glyceraldehyde 3-phosphate <br />                                                                                         producing strain Escherichia coli W3110 grown carbon- <br />       dehydrogenase mutants Escherichia coli. J Bacteriol 1975, <br />                                                                                         phosphate-limited conditions. J Biotechnol 2006, 126(4):528&#8211;545. <br />       122(3):1175&#8211;1179. <br /> 59.   Irani M, Maitra PK: Properties Escherichia coli mutants deficient     82.   Guilloton MB, Lamblin AF, Kozliak EI, Gerami-Nejad M, Tu C, Silverman D, <br />       enzymes glycolysis. J Bacteriol 1977, 132(2):398&#8211;410.                          Anderson PM, Fuchs JA: A physiological role cyanate-induced <br /> 60.   Reeves HC, Ajl SJ: Alpha-hydroxyglutaric acid synthetase. J Bacteriol             carbonic anhydrase Escherichia coli. J Bacteriol 1993, <br />       1962, 84:186&#8211;187.                                                                 175(5):1443&#8211;1451. <br /> 61.   Wegener WS, Reeves HC, Ajl SJ: Heterogeneity                         83.   Howell EE, Foster PG, Foster LM: Construction dihydrofolate <br />       glyoxylate-condensing enzymes. J Bacteriol 1965, 90(3):594&#8211;598.                   reductase-deficient mutant Escherichia coli gene <br /> 62.   Helling RB, Kukora JS: Nalidixic acd-resistant mutants Escherichia             replacement. J Bacteriol 1988, 170(7):3040&#8211;3045. <br />       coli deficient isocitrate dehydrogenase. J Bacteriol 1971,               84.   Pribat A, Blaby IK, Lara-N&#250;&#241;ez A, Gregory JF3rd, Cr&#233;cy-Lagard V, <br />       105(3):1224&#8211;1226.                                                                 Hanson AD: FolX FolM essential tetrahydromonapterin <br /> 63.   Lakshmi TM, Helling RB: Selection citrate synthase deficiency              synthesis Escherichia coli <span id='am-99' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-108'  ></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa. <br />       icd mutants Escherichia coli. J Bacteriol 1976, 127(1):76&#8211;83.                  J Bacteriol 2010, 192(2):475&#8211;482. <br /> 64.   Kabir MM, Shimizu K: Metabolic regulation analysis icd-gene              85.   Shim JH, Benkovic SJ: Evaluation kinetic mechanism of <br />       knockout Escherichia coli based 2d electrophoresis                        Escherichia coli glycinamide ribonucleotide transformylase. <br />       maldi-tof mass spectrometry enzyme activity measurements.                     Biochemistry 1998, 37(24):8776&#8211;8782. <br />       Appl Microbiol Biotechnol 2004, 65(1):84&#8211;96.                                86.   Newman EB, Kapoor V, Potter R: Role L-threonine dehydrogenase <br /> 65.   Lin H, Bennett GN, San KY: Genetic reconstruction aerobic                  catabolism threonine synthesis glycine by <br />       central metabolism Escherichia coli absolute aerobic                   Escherichia coli. J Bacteriol 1976, 126(3):1245&#8211;1249. <br />       production succinate. Biotechnol Bioeng 2005, 89(2):148&#8211;156.             87.   Liu JQ, Dairi T, Itoh N, Kataoka M, Shimizu S, Yamada H: Gene cloning, <br /> 66.   Kalliri E, Mulrooney SB, Hausinger RP: Identification Escherichia coli         biochemical characterization physiological role a <br />       ygaF L-2-hydroxyglutarate oxidase. J Bacteriol 2008,                        thermostable low-specificity L-threonine aldolase Escherichia <br />       190(11):3793&#8211;3798.                                                                coli. Eur J Biochem 1998, 255(1):220&#8211;226. <br /> 67.   Reinoso CA, Appanna VD, Vasquez CC: &#945;-ketoglutarate accumulation            88.   Vales LD, Chase JW, Murphy JB: Orientation guanine operon of <br />       dependent isocitrate dehydrogenase activity                      Escherichia coli K&#8211;12 utilizing strains containing guaB-xse and <br />       tellurite detoxification Escherichia coli. Biomed Res Int 2013,                guaB-upp deletions. J Bacteriol 1979, 139(1):320&#8211;322. <br />       2013:784190.                                                                89.   Umbarger HE, Brown B: Threonine deamination Escherichia coli. II. <br /> 68.   Stribling D, Perham RN: Purification characterization                  Evidence L-threonine deaminases. J Bacteriol 1957, <br />       fructose diphosphate aldolases Escherichia coli (Crookes&#8217;                    73(1):105&#8211;112. <br />       strain). Biochem J 1973, 131(4):833&#8211;841.                                    90.   Zhao X, Miller JR, Jiang Y, Marletta MA, Cronan JE: Assembly the <br /> 69.   Scamuffa MD, Caprioli RM: Comparison mechanisms                     covalent linkage lipoic acid cognate enzymes. <br />       distinct aldolases Escherichia coli grown gluconeogenic                   Chem Biol 2003, 10(12):1293&#8211;1302. <br />       substrates. Biochim Biophys Acta 1980, 614(2):583&#8211;590.                      91.   Reidl J, Boos W: The malX malY operon Escherichia coli encodes a <br /> 70.   Schurmann M, Sprenger GA: Fructose-6-phosphate aldolase                      novel enzyme II phosphotransferase recognizing <br />       novel class I aldolase Escherichia coli related novel            glucose maltose enzyme abolishing endogenous <br />       group bacterial transaldolases. JBC 2001, 276:11055&#8211;11061.                     induction maltose system. J Bacteriol 1991, 173(15):4862&#8211;4876. <br /> 71.   Baechler C, Schneider P, Baehler P, Lustig A, Erni B: Escherichia coli      92.   Zdych E, Peist R, Reidl J, Boos W: MalY Escherichia coli enzyme <br />       dihydroxyacetone kinase controls gene expression binding                    activity <span id='am-34' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-35' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span>beta</span> C-S lyase (cystathionase). J Bacteriol 1995, <br />       transcription factor dhaR. EMBO J 2005, 24(2):283&#8211;293.                            177(17):5035&#8211;5039. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                          Page 22 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> 93.    Boos W, Shuman H: Maltose/maltodextrin Escherichia coli:            116. Flores N, Xiao J, Berry A, <span id='am-223' about='obo:NCBITaxon_1463643' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_7401 obo:NCBITaxon_7402 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_65231 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_7399 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-224'  ></span><span id='am-225' property="rdfs:label" content="Bolivar" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Bolivar</span> F, Valle F: Pathway engineering the <br />        transport, metabolism, regulation. Microbiol Mol Biol Rev 1998,                production aromatic compounds Escherichia coli. Nat <br />        62(1):204&#8211;229.                                                                     Biotechnol 1996, 14(5):620&#8211;623. <br /> 94.    Cohen GN, Stanier RY, Bras GL: Regulation biosynthesis              117. Chen R, Yap WM, Postma PW, Bailey JE: Comparative studies of <br />        amino acids aspartate family coliform bacteria                       Escherichia coli strains using different glucose uptake systems: <br />        pseudomonads. J Bacteriol 1969, 99(3):791&#8211;801.                                     metabolism energetics. Biotechnol Bioeng 1997, 56(5):583&#8211;590. <br /> 95.    Garriga X, Eliasson R, Torrents E, Jordan A, Barbe J, Gibert I, Reichard P:   118. Zeppenfeld T, Larisch C, Lengeler JW, Jahreis K: Glucose transporter <br />        nrdD nrdG genes essential strict anaerobic growth                   mutants Escherichia coli K-12 changes substrate <br />        Escherichia coli. Biochem Biophys Res Commun 1996, 229(1):189&#8211;192.                 recognition IICB(Glc) induction behavior ptsG gene. <br /> 96.    Lam HM, Winkler ME: Metabolic relationships pyridoxine                     J Bacteriol 2000, 182(16):4443&#8211;4452. <br />        (vitamin B6) serine biosynthesis Escherichia coli K&#8211;12.                119. Flores N, Flores S, Escalante A, Anda R, Leal L, Malpica R, Georgellis D, <br />        J Bacteriol 1990, 172(11):6518&#8211;6528.                                               Gosset G, Bolivar F: Adaptation fast growth glucose by <br /> 97.    Kim J, Kershner JP, Novikov Y, Shoemaker RK, Copley SD: Three                      differential expression central carbon metabolism gal <br />        serendipitous pathways E. coli bypass block                            regulon genes Escherichia coli strain lacking the <br />        pyridoxal-5&#8217;-phosphate synthesis. Mol Syst Biol 2010, 6:436.                       phosphoenolpyruvate:carbohydrate phosphotransferase system. <br /> 98.    Hove-Jensen B, Rosenkrantz TJ, Zechel DL, Willemoes M: Accumulation                Metab Eng 2005, 7(2):70&#8211;87. <br />        intermediates carbon-phosphorus lyase pathway                   120. Steinsiek S, Bettenbrock K: Glucose transport Escherichia coli <br />        phosphonate degradation phn mutants Escherichia coli.                        mutant strains defects sugar transport systems. J Bacteriol <br />        J Bacteriol 2010, 192(1):370&#8211;374.                                                  2012, 194(21):5897&#8211;5908. <br /> 99.    Zhang Q, van der Donk WA: Answers carbon-phosphorus                    121. Escalante A, Cervantes AS, Gosset G, Bolivar F: Current knowledge of <br />        lyase conundrum. Chembiochem 2012, 13(5):627&#8211;629.                                  Escherichia coli phosphoenolpyruvate-carbohydrate <br /> 100.   Wanner BL: Gene regulation phosphate enteric bacteria. J Cell                phosphotransferase system: peculiarities regulation impact <br />        Biochem 1993, 51(1):47&#8211;54.                                                         growth product formation. Appl Microbiol Biotechnol 2012, <br /> 101.   Wanner BL, Boline JA: Mapping molecular cloning phn                     94(6):1483&#8211;1494. <br />        (psiD) locus phosphonate utilization Escherichia coli.                 122. Sarubbi E, Rudd KE, Cashel M: Basal ppgpp level adjustment shown <br />        J Bacteriol 1990, 172(3):1186&#8211;1196.                                                new spoT mutants affect steady state growth rates rrnA <br /> 102.   Ravnikar PD, Somerville RL: Genetic characterization highly                   ribosomal promoter regulation Escherichia coli. Mol Gen Genet <br />        efficient alternate pathway serine biosynthesis Escherichia                  1988, 213(2&#8211;3):214&#8211;222. <br />        coli. J Bacteriol 1987, 169(6):2611&#8211;2617.                                     123. Sarubbi E, Rudd KE, Xiao H, Ikehara K, Kalman M, Cashel M: <br /> 103.   Kim C, Song S, Park C: The D-allose operon Escherichia coli K&#8211;12.               Characterization spoT gene Escherichia coli. J Biol Chem <br />        J Bacteriol 1997, 179(24):7631&#8211;7637.                                               1989, 264(25):15074&#8211;15082. <br /> 104.   Poulsen TS, Chang YY, Hove-Jensen B: D-allose catabolism                   124. Xiao H, Kalman M, Ikehara K, Zemel S, Glaser G, Cashel M: Residual <br />        Escherichia coli: involvement alsI regulation als regulon                guanosine 3&#8217;,5&#8217;-bispyrophosphate synthetic activity relA null <br />        expression allose ribose. J Bacteriol 1999, 181(22):7126&#8211;7130.              mutants eliminated spoT null mutations. J Biol Chem 1991, <br /> 105.   Pittard J, Wallace BJ: Distribution function genes concerned                266(9):5980&#8211;5990. <br />        aromatic biosynthesis Escherichia coli. J Bacteriol 1966,             125. Gentry DR, Cashel M: Mutational analysis Escherichia coli spoT <br />        91(4):1494&#8211;1508.                                                                   gene identifies distinct overlapping regions involved ppgpp <br /> 106.   Bottomley JR, Clayton CL, Chalk PA, Kleanthous C: Cloning,                         synthesis degradation. Mol Microbiol 1996, 19(6):1373&#8211;1384. <br />        sequencing, expression, purification, preliminary                         126. Jin DJ, Cagliero C, Zhou YN: Growth rate regulation Escherichia <br />        characterization type II dehydroquinase <span id='am-89' about='obo:NCBITaxon_209' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="PMID:17329766" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Helicobacter" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:1704793" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28203" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:11156001" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:15143020" datatype="xsd:string"></span>Helicobacter</span>                     coli. FEMS Microbiol Rev 2012, 36(2):269&#8211;287. <br />        pylori. Biochem J 1996, 319(2):559&#8211;565.                                       127. Cox GB, Gibson F, Pittard J: Mutant strains Escherichia coli K&#8211;12 <br /> 107.   von Meyenburg K, Jorgensen BB, Nielsen J, Hansen FG: Promoters                  unable form ubiquinone. J Bacteriol 1968, 95(5):1591&#8211;1598. <br />        atp operon coding membrane-bound ATP synthase                  128. Wu G, Williams HD, Gibson F, Poole RK: Mutants Escherichia coli <br />        Escherichia coli mapped Tn10 insertion mutations. Mol Gen Genet                 affected respiration: cloning nucleotide sequence of <br />        1982, 188(2):240&#8211;248.                                                              ubiA, encoding membrane-bound <br /> 108.   Jensen PR, Michelsen O: Carbon energy metabolism atp                        p-hydroxybenzoate:octaprenyltransferase. J Gen Microbiol 1993, <br />        mutants Escherichia coli. J Bacteriol 1992, 174(23):7635&#8211;7641.                  139(8):1795&#8211;1805. <br /> 109.   Green GN, Gennis RB: Isolation characterization                     129. Klena JD, Ashford RS, Schnaitman CA: Role Escherichia coli K-12 rfa <br />        Escherichia coli mutant lacking cytochrome d terminal oxidase.                     genes rfp gene <span id='am-138' about='obo:NCBITaxon_620' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-139'  ></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="Shigella" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Shigella</span> dysenteriae 1 generation of <br />        J Bacteriol 1983, 154(3):1269&#8211;1275.                                                lipopolysaccharide core heterogeneity attachment o <br /> 110.   Georgiou CD, Fang H, Gennis RB: Identification cydC locus                   antigen. J Bacteriol 1992, 174(22):7297&#8211;7307. <br />        required expression functional form cytochrome d            130. Roncero C, Casadaban MJ: Genetic analysis genes involved in <br />        terminal oxidase complex Escherichia coli. J Bacteriol 1987,                    synthesis lipopolysaccharide core Escherichia coli K&#8211;12: <br />        169(5):2107&#8211;2112.                                                                  operons rfa locus. J Bacteriol 1992, 174(10):3250&#8211;3260. <br /> 111.   Puustinen A, Finel M, Haltia T, Gennis RB, Wikstrom M: Properties      131. Hwang J, Inouye M: The tandem GTPase, der, essential the <br />        terminal oxidases Escherichia coli. Biochemistry 1991,                      biogenesis 50S ribosomal subunits Escherichia coli. Mol <br />        30(16):3936&#8211;3942.                                                                  Microbiol 2006, 61(6):1660&#8211;1672. <br /> 112.   Pittman MS, Robinson HC, Poole RK: A bacterial glutathione                    132. Bharat A, Jiang M, Sullivan SM, Maddock JR, Brown ED: Cooperative and <br />        transporter (Escherichia coli CydDC) exports reductant                      critical roles G domains GTPase activity cellular <br />        periplasm. J Biol Chem 2005, 280(37):32254&#8211;32261.                                  function ribosome-associated Escherichia coli EngA. J Bacteriol <br /> 113.   Portnoy VA, Herrgard MJ, Palsson B&#216;: Aerobic fermentation                       2006, 188(22):7992&#8211;7996. <br />        d-glucose evolved cytochrome oxidase-deficient Escherichia              133. Tomar SK, Dhimole N, Chatterjee M, Prakash B: Distinct GDP/GTP <br />        coli strain. Appl Environ Microbiol 2008, 74(24):7561&#8211;7569.                        bound states tandem G-domains EngA regulate ribosome <br /> 114.   Borisov VB, Gennis RB, Hemp J, Verkhovsky MI: The cytochrome bd                    binding. Nucleic Acids Res 2009, 37(7):2359&#8211;2370. <br />        respiratory oxygen reductases. Biochim Biophys Acta 2011,                     134. Spratt BG: Distinct penicillin binding proteins involved the <br />        1807(11):1398&#8211;1413.                                                                division, elongation, shape Escherichia coli K&#8211;12. PNAS 2009, <br /> 115.   Chou CH, Bennett GN, San KY: Effect modified glucose uptake                     72(8):2999&#8211;3003. <br />        using genetic engineering techniques high-level recombinant                135. Ogura T, Bouloc P, Niki H, D&#8217;Ari R, Hiraga S, Jaffe A: Penicillin-binding <br />        protein production Escherichia coli dense cultures. Biotechnol                  protein 2 essential wild-type Escherichia coli lov or <br />        Bioeng 1994, 44(8):952&#8211;960.                                                        cya mutants. J Bacteriol 1989, 171(6):3025&#8211;3030. <br />  Weaver et al. BMC Systems Biology 2014, 8:79                                                                                                     Page 23 24 <br /> http://www.biomedcentral.com/1752-0509/8/79 <br />  <br />  <br />  <br />  <br /> 136. Wang Y, Stieglitz KA, Bubunenko M, Court DL, Stec B, Roberts MF: The         160. Lukas H, Reimann J, Kim OB, Grimpo J, Unden G: Regulation of <br />      structure R184A mutant inositol monophosphatase                     aerobic anaerobic d-malate metabolism Escherichia coli by <br />      encoded suhB implications functional interactions               LysR-type regulator DmlR (YeaT). J Bacteriol 2010, <br />      Escherichia coli. J Biol Chem 2007, 282(37):26989&#8211;26996.                          192(10):2503&#8211;2511. <br /> 137. <span id='am-238' about='obo:NCBITaxon_1498367' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_75054 obo:NCBITaxon_1 obo:NCBITaxon_74970 obo:NCBITaxon_131567 obo:NCBITaxon_1498386 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-239' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-240'  ></span><span id='am-241' property="rdfs:label" content="Puan" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Puan</span> KJ, Wang H, Dairi T, Kuzuyama T, Morita CT: fldA essential        161. Lauritzen AM, Lipscomb WN: Modification active site lysine <br />      gene required 2-C-methyl-D-erythritol 4-phosphate pathway                  residues catalytic subunit aspartate transcarbamylase by <br />      isoprenoid biosynthesis. FEBS Lett 2005, 579(17):3802&#8211;3806.                   D- L-bromosuccinate. J Biol Chem 1982, 257:1312&#8211;1319. <br /> 138. Lu Q, Inouye M: Adenylate kinase complements nucleoside                      162. Hall BG: Chromosomal mutation citrate utilization by <br />      diphosphate kinase deficiency nucleotide metabolism. PNAS                      Escherichia coli K&#8211;12. J Bacteriol 1982, 151(1):269&#8211;273. <br />      1996, 93(12):5720&#8211;5725.                                                      163. Lutgens M, Gottschalk G: Why co-substrate required for <br /> 139. Gerhart JC, Schachman HK: Distinct subunits regulation                anaerobic growth Escherichia coli citrate. J Gen Microbiol 1980, <br />      catalytic activity aspartate transcarbamylase. Biochemistry 1965,              119(1):63&#8211;70. <br />      4(6):1054&#8211;1062.                                                              164. Schneider BL, Ruback S, Kiupakis AK, Kasbarian H, Pybus C, Reitzer L: The <br /> 140. Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, Herring CD, Bui OT,            Escherichia coli gabDTPC operon: specific gamma-aminobutyrate <br />      Knight EM, Fong SS, Palsson B&#216;: Systems approach refining                      catabolism nonspecific induction. J Bacteriol 2002, <br />      genome annotation. Proc Natl Acad Sci U S A 2006,                                 184(24):6976&#8211;6986. <br />      103(46):17480&#8211;17484.                                                         165. Cunin R, Glansdorff N, Pierard A, Stalon V: Biosynthesis and <br /> 141. Kumar VS, Dasika MS, Maranas CD: Optimization based automated                     metabolism arginine bacteria. Microbiol Rev 1986, 50(3):314&#8211;352. <br />      curation metabolic reconstructions. BMC Bioinformatics 2007,              166. Schneider BL, Kiupakis AK, Reitzer LJ: Arginine catabolism the <br />      8:212.                                                                            arginine succinyltransferase pathway Escherichia coli. J Bacteriol <br /> 142. Kumar VS, Maranas CD: GrowMatch: An automated method                          1998, 180(16):4278&#8211;4286. <br />      reconciling silico/in vivo growth predictions. PLoS Comput Biol           167. Plumbridge J, Pellegrini O: Expression chitobiose operon of <br />      2009, 5(3):1000308.                                                               Escherichia coli regulated transcription factors: NagC, <br /> 143. Barua D, Kim J, Reed JL: An automated phenotype-driven approach                   ChbR CAP. Mol Microbiol 2004, 52(2):437&#8211;449. <br />      (geneforce) refining metabolic regulatory models. PLoS               168. Kachroo AH, Kancherla AK, Singh NS, Varshney U, Mahadevan S: <br />      Comput Biol 2010, 6(10):1000970.                                                  Mutations alter regulation chb operon of <br /> 144. Orth JD, Palsson B&#216;: Systematizing generation missing                      Escherichia coli allow utilization cellobiose. Mol Microbiol 2007, <br />      metabolic knowledge. Biotechnol Bioeng 2010, 107(3):403&#8211;412.                      66(2):1382&#8211;1395. <br /> 145. Tervo CJ, Reed JL: BioMog: computational framework              169. Chang GW, Chang JT: Evidence B12-dependent enzyme <br />      novo generation modification essential biomass                              ethanolamine deaminase <span id='am-191' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-192' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-196'  ></span><span id='am-197' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-200' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Salmonella</span>. Nature 1975, <br />      components. PLoS One 2013, 8(12):81322.                                           254(5496):150&#8211;151. <br /> 146. Monod J: The growth bacterial cultures. Annu Rev Microbiol 1949,          170. Jones PW, Turner JM: Interrelationships enzymes of <br />      3:371&#8211;394.                                                                        ethanolamine metabolism Escherichia coli. J Gen Microbiol 1984, <br /> 147. Bochner BR, Gadzinski P, Panomitros E: Phenotype microarrays                  130(2):299&#8211;308. <br />      high-throughput phenotypic testing assay gene function.               171. Blackwell CM, Turner JM: Microbial metabolism amino alcohols. <br />      Genome Res 2001, 11(7):1246&#8211;1255.                                                 formation coenzyme B12-dependent ethanolamine <br /> 148. Shea A, Wolcott M, Daefler S, Rozak DA: Biolog phenotype                          ammonia-lyase concerted induction Escherichia coli. <br />      microarrays. In Microbial Systems Biology: Methods Protocols. New             Biochem J 1978, 176(3):751&#8211;757. <br />      York: Humana Press; 2012:331&#8211;73. Chap. 12.                                   172. Akita K, Hieda N, Baba N, Kawaguchi S, Sakamoto H, Nakanishi Y, <br /> 149. Mackie AM, Hassan KA, Paulsen IT, Tetu SG: Biolog phenotype                       Yamanishi M, Mori K, Toraya T: Purification properties of <br />      microarrays phenotypic characterization microbial cells.                   wild-type N-terminal-truncated ethanolamine ammonia-lyase <br />      Methods Mol Biol 1096, 2014:123&#8211;301.                                              Escherichia coli. J Biochem 2010, 147(1):83&#8211;93. <br /> 150. Oun MA, Suthers PF, Jones GI, Carter BR, Saunders MP, Maranas CD,            173. Xi H, Schneider BL, Reitzer L: Purine catabolism Escherichia coli and <br />      Woodward MJ, Anjum MF: Genome scale reconstruction                           function xanthine dehydrogenase purine salvage. J Bacteriol <br />      salmonella metabolic model: comparison similarity                          2000, 182(19):5332&#8211;5341. <br />      differences commensal Escherichia coli strain. J Biol Chem            174. Reitzer L: Nitrogen assimilation global regulation Escherichia <br />      2009, 284(43):29480&#8211;29488.                                                        coli. Annu Rev Microbiol 2003, 57:155&#8211;176. <br /> 151. Yoon SH, Han MJ, Jeong H, Lee CH, Xia XX, Lee DH, Shim JH, Lee SY, Oh        175. Cusa E, Obradors N, Baldoma L, Badia J, Aguilar J: Genetic analysis a <br />      TK, Kim JF: Comparative multi-omics systems analysis Escherichia               chromosomal region containing genes required assimilation of <br />      coli strains B K&#8211;12. Genome Biol 2012, 13(5):37.                              allantoin nitrogen linked glyoxylate metabolism Escherichia <br /> 152. Mackie A, Paley S, Keseler IM, Shearer A, Paulsen IT, Karp PD: Addition        coli. J Bacteriol 1999, 181(24):7479&#8211;7484. <br />      Escherichia coli K&#8211;12 growth-observation gene essentiality               176. Harborne NR, Griffiths L, Busby SJ, Cole JA: Transcriptional control, <br />      data EcoCyc database. J Bacteriol 2014, 196(5):982&#8211;988.                    translation function products open reading <br /> 153. Wilson DM, Wilson TH: Cation specificity sugar substrates              frames Escherichia coli nir operon. Mol Microbiol 1992, <br />      melibiose carrier Escherichia coli. Biochim Biophys Acta 1987,                 6(19):2805&#8211;2813. <br />      904(2):191&#8211;200.                                                              177. Wang H, Tseng CP, Gunsalus RP: The napF narG nitrate reductase <br /> 154. Sandermann H Jr: &#946;-D-galactoside transport Escherichia coli:                   operons Escherichia coli differentially expressed response <br />      substrate recognition. Eur J Biochem 1977, 80(2):507&#8211;515.                         submicromolar concentrations nitrate nitrite. <br /> 155. Sahota SS, Bramley PM, Menzies IS: The fermentation lactulose               J Bacteriol 1999, 181(17):5303&#8211;5308. <br />      colonic bacteria. Microbiology 1981, 128(2):319&#8211;325.                         178. Bertero MG, Rothery RA, Palak M, Hou C, Lim D, Blasco F, Weiner JH, <br /> 156. Wallenfels K, Weil R: &#946;-galactosidase. In The Enzymes, 3rd Ed. New York:          Strynadka NC: Insights respiratory electron transfer <br />      Academic Press; 1972:617&#8211;663.                                                     pathway structure nitrate reductase A. Nat Struct Biol <br /> 157. Roderick SL: The lac operon galactoside acetyltransferase. CR                     2003, 10(9):681&#8211;687. <br />      Biologies 2005, 328:568&#8211;575.                                                 179. Seiflein TA, Lawrence JG: Methionine-to-cysteine recycling in <br /> 158. Lang VJ, Leystra-Lantz C, Cook RA: Characterization specific               <span id='am-178' about='obo:NCBITaxon_570' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-179' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:11411716" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasDbXref" content="PMID:12635932" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="Klebsiella" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasDbXref" content="PMID:10555350" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasRelatedSynonym" content="Hyalococcus" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Drancourt et al. 2001" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Carter et al. 1999" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39823" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasRelatedSynonym" content="&quot;Hyalococcus&quot; Schroeter 1886" datatype="xsd:string"></span><span id='am-190'  ></span>Klebsiella</span> aerogenes. J Bacteriol 2001, 183(1):336&#8211;346. <br />      pyruvate tr <br /> </body></html>