<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Kim et al. BMC Biotechnology 2014, 14:84 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br />  METHODOLOGY ARTICLE                                                                                                                           Open Access <br />  <br /> A versatile highly efficient method for <br /> scarless genome editing <span id='am-42' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span> and <br /> <span id='am-65' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span>Salmonella</span> enterica <br /> Juhan Kim1,2, Anthony M Webb1, Jamie P Kershner1,2, Stephen Blaskowski1 Shelley D Copley1,2* <br />  <br />  <br />   Abstract <br />   Background: Recently developed methods genome editing bacteria advantage introduction of <br />   double-strand breaks I-SceI mutation cassette select cells homologous recombination has <br />   healed break introduced desired mutation. <span id='am-27' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-28' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> elegantly designed method did work our <br />   hands genes. <br />   Results: We corrected mutation gene encoding I-SceI compromised function previously used <br />   Red helper plasmid. Further, transcription extending mutation cassette interferes cleavage <br />   I-SceI. Addition transcription terminators upstream cleavage site dramatically increases efficiency <br />   genome editing. We developed improved method modification essential genes. Inclusion <br />   segment essential gene consisting synonymous codons restores open reading frame the <br />   mutation cassette integrated genome decreases frequency recombination events fail to <br />   incorporate desired mutation. The optimized protocol takes 5 days 100% successful over <br />   100 genomic modifications hands. <br />   Conclusions: The method reliable versatile, enabling various types genome editing in <br />   <span id='am-58' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli <span id='am-78' about='obo:NCBITaxon_28901' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_590 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Salmonella enterica ser. choleraesuis" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-suis" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Salmonella enterica" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Salmonella cholerae-suis" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Salmonella choleraesuis" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:591" datatype="xsd:string"></span>Salmonella enterica</span> straightforward modifications mutation cassette. We provide <br />   detailed descriptions methods designs insertions, deletions, introduction point mutations. <br />   Keywords: Genome editing, I-SceI, Escherichia coli, Salmonella enterica <br />  <br />  <br /> Background                                                                           &#955; recombination promote gene replacement [4] <br /> A facile completely general method modification                              strains low intrinsic recombination rate. For this <br /> of genomic sequence bacteria                             purpose, Datsenko Wanner [1] developed Red <br /> of enormous value wide range applications,                                  helper plasmid, pKD46, Blattner lab developed <br /> including construction strains metabolic engineering,                         derivatives, pACBSR [3] pK-HT [2]. Some varia- <br /> studies effects mutations acquired adaptive                         tions commonly used methods use &#955; Red <br /> evolution, construction strains molecular biology                         recombinase encoded genomic DNA [5-7] rather <br /> studies require chromosomal modification genes                               Red helper plasmid. <br /> such allele replacement epitope tagging.                                         The two-step recombination method uses selection <br />    The frequently used techniques genetic modi-                             marker identify cells mutation cassette <br /> fication Escherichia coli Salmonella enterica                             composed homologous recombination fragments <br /> the two-step recombination method [1,2] &#8216;gene                                surrounding selection marker integrated into <br /> gorging&#8217; method [3]. Both methods use bacteriophage                              target site. Subsequently, selection marker be <br />                                                                                      removed FLP recombinase [1] cleavage the <br /> * Correspondence: shelley.copley@colorado.edu                                        I-SceI meganuclease followed homologous recombin- <br /> 1 <br />  Department Molecular, Cellular, Developmental Biology, University         ation using host&#8217;s RecA [2]. Use FLP <br /> Colorado Boulder, Boulder, CO 80309, USA <br /> 2 <br />  Cooperative Institute Research Environmental Sciences, University of <br />                                                                                      recombinase leaves short FRT sequence scar. Use <br /> Colorado Boulder, Boulder, CO 80309, USA                                             host&#8217;s RecA double-strand breakage <br />                                         &#169; 2014 Kim et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies <span id='am-92' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> available article, <br />                                         unless stated. <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                         Page 2 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> by I-SceI leaves scar end. In contrast, gene     HR2) target cassette desired chromosomal <br /> gorging method relies screening identify colonies      location. HR3 provides region homologous <br /> in desired genetic change introduced          recombination occur I-SceI cleavage gen- <br /> by recombination mutation cassette genome.       omic DNA. Two HR3 sequences flank sequence encod- <br /> It leaves scar, inefficient     ing double transcriptional terminator (depicted the <br /> use special recombination strain [8] specially       double hairpin, &#8216;TT&#8217;, Figure 1) I-SceI rec- <br /> designed synthetic DNA chemically modified bases          ognition site selection marker. The step the <br /> [9]. These techniques used extensively           procedure introduction helper plasmid pSLTS <br /> modified enable deletions, insertions, epitope          target strain. Overexpression &#955; Red re- <br /> tagging [5,10-18].                                             combinase induced L-arabinose (Figure 1, Step <br />    In hands, genome editing two-step           1). The mutation cassette introduced cells <br /> recombination method gene gorging highly inef-         carrying helper plasmid electroporation (Figure 1, <br /> ficient cases. We focused attention         Step 2). Integration mutation cassette tar- <br /> two-step recombination method advantage         location accomplished &#955; Red recombinase <br /> of employing selection identify cells muta-    (Figure 1, Step 3). Cells mutation cassette <br /> tion cassette integrated genome. We          integrated genome selected on <br /> identified corrected mutations limit        plates containing antibiotic selection <br /> efficacy previously used helper plasmid stream-       cassette confers resistance. Subsequently, cells are <br /> lined procedure carried         spread plates containing ampicillin (to maintain the <br /> higher efficiency fewer experimental manipulations.        helper plasmid) anhydrotetracycline (to induce ex- <br /> However, observed wide range efficiencies        pression I-SceI) (Figure 1, Step 4). Most cells do <br /> depending identity target gene.                  survive double-strand break introduced I-SceI. <br />    We discovered intransigence certain genes       However, minority cells, homologous recombin- <br /> was active transcription continued        ation HR3 sequences enables survival and <br /> inserted mutation cassette prevented cleavage       accomplishes desired chromosomal modification <br /> I-SceI site. Difficulties modifying actively transcribed    (Figure 1, Step 5). Here development and <br /> genes present important limitation use            validation method applications E. coli <br /> genome editing procedures. Inclusion successive         Salmonella. <br /> terminator sequences I-SceI cleavage site <br /> obviated problem resulted highly reliable          Results discussion <br /> chromosomal manipulation.                                      Construction optimized helper plasmid <br />    Modification essential genes problematic using   The ideal helper plasmid two-step scarless chromo- <br /> the two-step procedure step results       somal modification characteristics: 1) <br /> interruption target gene mutation cassette.      genes encoding &#955;-Red recombinase increase the <br /> A solution dilemma include mutation       frequency successful integration mutation cas- <br /> cassette extra fragment target gene restores    sette recombination; 2) gene encoding I-SceI; 3) <br /> an open reading frame mutation cassette inte-      temperature-sensitive origin replication facilitate <br /> grated genome. However, homologous              removal helper plasmid recombination the <br /> recombination occur extended          mutation cassette genome. Further, &#955;-Red <br /> extra fragment, recombination events fail         recombinase genes gene encoding I-SceI should <br /> integrate small region encoding desired mutation       inducible different conditions. Three <span id='am-97' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-98' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> <br /> into genome. We addressed problem using          characteristics, pKDTS [2], pWRG99 [14] and <br /> fragment target gene composed synonymous             pRedI [18], reported. <br /> codons prevent homologous recombination            We used pKDTS helper plasmid introduce a <br /> extra fragment.                                                sequence encoding 3 &#215; FLAG tag 3&#8217;-end rpoD <br />    Our optimized procedure summarized Figure 1.          E. coli. We re-constructed plasmid previously <br /> The new helper plasmid pSLTS encodes I-SceI              described inserting 1407 bp Nco1 fragment en- <br /> the control anhydrotetracycline-inducible tetA          coding Ptet-I-sceI pK-HT [2] NcoI site of <br /> promoter Lambda Red recombinase genes            pKD46 [2]. We pKDTS effectively promoted <br /> the control arabinose-inducible araB promoter,          recombination mutation cassette genome, <br /> and carries Temperature-Sensitive origin replication.     judged growth colonies medium containing <br /> (Descriptions plasmids used work pro-       antibiotic resistance gene carried on <br /> vided Table 1). The mutation cassette consists         mutation cassette. However, cleavage I-SceI <br /> fragment containing 100 bp homology regions (HR1           site mutation cassette I-SceI extremely <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                         Page 3 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br />  Figure 1 The general strategy scarless genome editing. HR, homology region. HR1 HR2 identical target site the <br />  genome. The purple bar wedge HR3 indicate mutation introduced. <br />  <br />  <br />  <br />  <br /> inefficient. After induction I-SceI expression anhy-                 We discovered mutations region pKDTS <br /> drotetracycline, small proportion cells                 derived pK-HT. One these, results re- <br /> survive; cells healed double strand break                 placement 5 amino acid residues N-terminus of <br /> introduced I-SceI homologous recombination                       I-SceI single amino acid (MHMKNIK&#10140;MHQ), <br /> that removes selection cassette. In experiment,                 detected previously [2]. Two mutations <br /> when 36 colonies spread individually plates                 resulted missense changes Asn11 Ser Lys33 <br /> the absence presence anhydrotetracycline, dif-                 Arg TetR (AAC&#10140;AGC AAG&#10140;AGG, respect- <br /> ference number colonies observed (see                      ively). All mutations present pK-HT <br /> Figure 2), suggesting intended cleavage                  plasmid fragment containing genes I- <br /> genome I-SceI ineffective.                                        SceI TetR taken. Any mutations <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                                       Page 4 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> Table 1 Plasmids used work <br /> Plasmids Characteristics                                                                                                                         Reference <br /> pSLTS*     ori SC101(Ts) Ampr; ParaB &#955;-Red; PtetR I-SceI                                                                                 This work <br /> pHA1887    1887 bp fragment pUC19 contains replication origin bla gene conferring ampicillin resistance <br /> pT2SC*     Ampr Cmr; template plasmid amplification selection cassette includes transcription terminator elements,        This work <br />            I-SceI site chloramphenicol resistance gene ligated pHA1887. <br /> pT2SK*     Ampr Kanr; template plasmid amplification selection cassette includes transcription terminator elements,          This work <br />            I-SceI site kanamycin resistance gene ligated pHA1887. <br /> pT2ST*     Ampr Trimr; template plasmid amplification selection cassette includes transcription terminator elements,         This work <br />            I-SceI site trimethoprim resistance gene ligated pHA1887. <br /> pT2SCb*    Ampr Cmr; template plasmid amplification selection cassette includes transcription terminator elements,        This work <br />            I-SceI site modified chloramphenicol resistance gene provides bacterial consensus ribosomal binding site the <br />            3&#8217;-end ligated pHA1887. <br /> pASC       Ampr Cmr; template plasmid amplification selection cassette includes I-SceI site chloramphenicol             This work <br />            resistance gene ligated pHA1887. <br /> pASK       Ampr Kanr; template plasmid amplification selection cassette includes I-SceI site kanamycin resistance       This work <br />            gene ligated pHA1887. <br /> pAST       Ampr Trimr; template plasmid amplification selection cassette includes I-SceI site trimethoprim              This work <br />            resistance ligated pHA1887. <br /> pTSC       Ampr Cmr; template plasmid amplification selection cassette includes transcription terminator, I-SceI site       This work <br />            chloramphenicol resistance gene ligated pHA1887. <br /> pTSK       Ampr Kanr; template plasmid amplification selection cassette includes transcription terminator, I-SceI site      This work <br />            kanamycin resistance gene ligated pHA1887. <br /> pTST       Ampr Trimr; template plasmid amplification selection cassette includes transcription terminator, I-SceI site     This work <br />            trimethoprim resistance gene ligated pHA1887. <br /> pKD46      ori SC101(Ts) Ampr; ParaB &#955;-Red                                                                                                       [1] <br /> pK-HT      ori SC101(Ts&#8722;) Ampr; Prha &#955;-Red; PtetR I-SceI                                                                                     [2] <br /> *These plasmids deposited addgene repository freely available academic use. <br />  <br />  <br /> might responsible failure I-SceI                        lost selection cassette induction I-SceI ex- <br /> cleavage step. After correction mutations,                        pression, correct insertion (Table 2). pKDTS2, <br /> the new helper plasmid, pSLTS, successfully used                          helper plasmid mutation I-SceI <br /> introduce sequence encoding 3 &#215; FLAG tag 3&#8217;-                          corrected, successfully promoted recom- <br /> end rpoD E. coli (see Table 2 Figure 2). The                         bination mutation cassette genome and <br /> frequency viable colonies induction I-SceI ex-                       showed expected diminution number col- <br /> pression 3.6 (&#177;1.5) &#215; 10&#8722;5, attesting efficiency                      onies induction I-SceI expression (Figure 2). <br /> of I-SceI cleavage step. Among colonies                           However, colonies lost the <br />  <br />  <br />  <br />  <br />  Figure 2 Comparison performance helper plasmids enabling I-SceI cleavage target site. Individual colonies E. coli K-12 <br />  MG1655 cells containing helper plasmid mutation cassette designed introduce C-terminal 3xFLAG tag RpoD had <br />  integrated spread agar plates containing LB Miller medium 100 &#956;g/mL ampicillin. Plates right half set contained <br />  100 ng/ml anhydrotetracycline induce expression I-SceI. Successful cleavage mutation cassette results diminution number <br />  viable colonies. <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                      Page 5 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> Table 2 Comparison performance helper                            Manipulation highly transcribed genes requires an <br /> plasmids enabling introduction sequence                            additional element selection cassette <br /> encoding C-terminal 3XFLAG tag RpoD E. coli                        The two-step recombination method using pSLTS <br /> K-12 MG1655                                                                worked cases, including insertion a <br /> Helper plasmid                                            pKDTS2 pSLTS     sequence encoding 3 &#215; FLAG tag 3&#8217;-end rpoD <br /> Fraction colonies lost selection marker        14/20   17/20   discussed above. However, consistently failed gen- <br /> after double-strand cleavage mutation cassette                      erate successful mutants attempted introduce <br /> by I-SceI <br />                                                                            point mutations rpe aceE E. coli, delete an <br /> Fraction colonies lost selection marker     4/8     8/8 <br /> for fragment correct size was <br />                                                                            intergenic region thrL thrA small re- <br /> amplified colony PCR using primers flanking                         gion pta E. coli, introduce sequence encod- <br /> gene                                                           ing 3XFLAG-tag C-terminus GapA E. coli <br /> Fraction colonies correct PCR              4/4     8/8    C-termini GapA FbaA S. enterica. In <br /> product obtained contained desired                            case, mutation cassette successfully inte- <br /> mutation <br />                                                                            grated genome step. However, spread- <br />                                                                            ing cells plates containing anhydrotetracycline to <br /> antibiotic resistance marker correct insertion                     induce expression I-SceI did result expected <br /> (Table 2). The remaining colonies wild type.                     decrease number colonies introduction <br />   Scarless deletion thrB E. coli K-12 MG1655                     double-strand break (Figure 3). This result suggested <br /> argC S. enterica SL1344 easily accomplished                         I-SceI failed cleave target site inserted <br /> using pSLTS (Additional file 1: Table S1). Again,                   mutation cassette. We hypothesized active transcrip- <br /> the colonies lost selection cassette in-                tion initiated promoter upstream target gene <br /> duction I-SceI expression intended deletion.                    extend mutation cassette interfere <br /> The general design constructs insertion                    binding I-SceI target site. To test hy- <br /> deletion sequences diagrammed Additional file 1:                  pothesis, introduced terminator sequences <br /> Figure S1.                                                                 upstream I-SceI site. We used most- <br />                                                                            efficient artificial terminators iGEM parts collec- <br /> Colony purification introduction mutation                     tion (http://parts.igem.org/Terminators/Catalog). When <br /> cassette improves efficiency recovery mutant                     terminator element (BBa_B1006, 99% efficiency) <br /> strains                                                                    present cleavage site, slight de- <br /> The recovery colonies containing desired muta-                      crease number colonies observed in- <br /> tion improved re-streaking colonies obtained                     duction I-SceI, suggesting cleavage I-SceI <br /> after introduction mutation cassette sec-                    site inefficient. Insertion terminator <br /> ond set plates containing LB ampicillin (LBA)                       elements (BBa_B1006 BBa_B1002, 99 98% <br /> and antibiotic mutation cassette con-                    efficiencies, respectively) resulted dramatic increase <br /> fers resistance. Without colony purification step,                 cleavage efficiency, evidenced substantial de- <br /> routinely wild type colonies                        crease number colonies obtained cells <br /> that grew agar plates containing LB, ampicillin                     spread plates containing anhydrotetracycline <br /> anhydrotetracycline (LBAaTc). For example, ex-                      (Figure 3). In E. coli S. enterica, possible <br /> periment introduced sequence encoding                      identify correct mutant screening fewer a <br /> C-terminal 3 &#215; FLAG tag RpoD E. coli K-12                            dozen colonies selection cassette contained a <br /> MG1655 using pSLTS helper plasmid, 50%                         double terminator element, necessary to <br /> colonies obtained LBAaTc agar plates wild type                     screen colonies correct mutant the <br /> when colony purification performed inte-                     selection cassette contained single terminator element <br /> gration mutation cassette. In contrast, wild                     (Table 3). <br /> type cells obtained colony purification was <br /> performed (Table 2). When attempted gen-                       Introduction mutations essential genes <br /> etic modification S. enterica, 100 colonies                  Introduction mutations essential genes difficult <br /> tested induction I-SceI wild type col-                  typical strategy initial integration a <br /> ony purification done, 8 colonies                 mutation cassette genome disrupts target <br /> tested desired modification colony purifi-                    gene. We designed modification strategy <br /> cation performed. Apparently wild type cells                      enables efficient manipulation essential genes. <br /> were able survive vicinity colonies                    frr ppa essential growth E. coli K-12 <br /> integrated mutation cassette able detox-                   BW25113 rich media LB [19]. frr encodes a <br /> ify antibiotic immediate environment.                         ribosome recycling factor ppa encodes inorganic <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                              Page 6 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br />   Figure 3 Terminator elements up-stream I-SceI cleavage site enhance efficiency I-SceI cleavage. One terminator <br />   elements introduced ahead I-SceI cleavage site mutation cassettes designed introduce sequence encoding C-terminal <br />   3 &#215; FLAG tag GapA E. coli K-12 MG1655 S. enterica Typhimurium SL1344. The efficiency double-strand break formation I-SceI <br />   determined spreading cells agar plates containing LB Miller medium 100 &#956;g/mL ampicillin presence absence of <br />   100 ng/ml anhydrotetracycline. <br />  <br />  <br />  <br /> pyrophosphatase. These genes previously used                          additional fragment replaced synonymous <br /> demonstrate method introduction amber stop                        <span id='am-34' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> produce sequence encodes same <br /> codon subsequently used conjunction                          protein recombine target gene the <br /> with suppressor tRNA enable condition-specific dis-                      genome. An alternative design (Figure 5 dis- <br /> ruption translation [2]. We targeted genes fa-                    cussion below) restores complete open reading frame <br /> cilitate comparison approach previously                        wild type sequence essential gene, re- <br /> published approach.                                                           solves modified gene double-strand cleavage and <br />   We designed mutation cassette HR3 con-                           homologous recombination. In addition maintaining <br /> tains silent mutation codon serine (AGC&#10132;TCT                        function target gene procedure, <br /> or TCT&#10132;AGC) 5&#8217;-half 3&#8217;- half frr                      strategy helps confine sequences capable re- <br /> and ppa. To maintain function encoded pro-                         combination small window, decreasing the <br /> teins procedure, added additional frag-                      frequency recombination events regenerate the <br /> ment encodes target gene                          wild type sequence (Additional file 1: Figure S2). Such re- <br /> intact open reading frame reconstituted                      combination events account variable fre- <br /> control promoter (Figure 4a) control                      quency integration amber stop codon target <br /> of constitutive promoter upstream selection                        genes previous report [2]. Because unproductive re- <br /> marker (cat) preceding target gene fragment                               combination events increase screening re- <br /> (Figure 4b) integration mutation cassette                   quired identify colonies desired mutation, the <br /> the genome. (The strategy optimal                         synonymous codon strategy significantly improves effi- <br /> editing gene operon                       ciency procedure, particular importance <br /> avoid polar effects downstream genes). Each codon                       endeavors requiring large number modifications. <br />  <br />  <br /> Table 3 Effect terminator elements ahead I-SceI cleavage site introduction sequence encoding a <br /> C-terminal 3XFLAG tag GapA E. coli K-12 MG1655 S. enterica Typhimurium SL1344 <br />                                                                                        E. coli K-12 MG1655     S. enterica Typhimurium SL1344 <br /> Number terminators I-SceI recognition site                           0       1       2          0            1             2 <br /> Fraction colonies lost selection marker double-strand cleavage    0/100   2/40   5/11       0/100         1/40           4/7 <br /> the mutation cassette I-SceI <br /> Fraction colonies lost selection marker fragment        NA*     1/2     1/2       NA*           1/1            2/2 <br /> correct size amplified colony PCR using primers flanking gene interest <br /> Fraction colonies correct PCR product obtained contained     NA*     1/1     1/1       NA*           1/1            2/2 <br /> the desired mutation <br /> *NA, applicable. <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                              Page 7 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br />  Figure 4 A gene fragment synonymous codons maintains function essential gene editing. An intact open reading <br />  frame reconstituted control promoter (a) control constitutive promoter upstream selection <br />  marker (cat) preceding target gene fragment (b). <br />  <br />  <br />                                                                               We able introduce silent mutations both <br />                                                                            5&#8217; 3&#8217; halves frr 3&#8217; half ppa using this <br />                                                                            approach (Table 4). The mutation cassette success- <br />                                                                            fully integrated 100% colonies obtained se- <br />                                                                            lection agar plates containing antibiotic which <br />                                                                            selection marker conferred resistance. We typically <br />                                                                            selected colonies plates spread them <br />                                                                            plates containing ampicillin anhy- <br />                                                                            drotetracycline induce expression I-SceI. Effective <br />                                                                            cleavage I-SceI evidenced dramatic reduction <br />                                                                            number colonies plates containing anhy- <br />                                                                            drotetracycline. Subsequently, patched col- <br />                                                                            onies plate containing anhydrotetracycline <br />                                                                            plates absence presence antibiotic <br />                                                                            mutation cassette confers resistance. All of <br />                                                                            colonies lost antibiotic resistance marker <br />                                                                            contained desired mutation (Table 4). <br />                                                                               We initially unable introduce silent mutation <br />                                                                            5&#8217; half ppa failure integrate mu- <br />                                                                            tation cassette, suggesting construct did en- <br />                                                                            able production sufficient Ppa support growth. <br />                                                                            This failure difficulties tran- <br />                                                                            scription dependence ppa transcription on <br />                                                                            constitutive promoter upstream cat, this <br />                                                                            issue comparable experiment frr. <br />                                                                            A likely explanation translation syn- <br />                                                                            onymously coded fragment sup-optimal. To explore <br />                                                                            possibility, introduced silent mutations cat <br />                                                                            gene upstream synonymous coding region ppa <br />                                                                            mutation cassette generate ribosome binding <br />  Figure 5 Introduction multiple point mutations recA            site closely resembling bacterial consensus <br />  E. coli. The hatched area indicates region replaced the <br />                                                                            ribosome binding site [20]. This strategy successful <br />  pink fragment mutation cassette. <br />                                                                            (see Figure 6), suggesting strong ribosome binding <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                                   Page 8 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> Table 4 Efficiency introduction silent mutations essential genes E. coli K-12 BW25113 using pSLTS a <br /> helper plasmid <br /> Target gene                                                                                         frr                              ppa <br /> length gene (bp) affected amino acid residue                                                  558 Ser33                           531 <br /> Position mutation (bp)                                                                97-99      445-447 Ser149     109-111 Ser37   343-345 Ser115 <br /> Mutation                                                                               TCT&#10140;AGC            AGC&#10140;TCT       AGC&#10140;TCT            AGC&#10140;TCT <br /> Efficiency integration mutation cassette                                     100% (4/4)         100% (4/4)     67% (2/3)         100% (4/4) <br /> Fraction colonies lost selection marker double-strand cleavage          2/7                3/6           4/10               9/20 <br /> of mutation cassette I-SceI <br /> Fraction colonies lost selection marker fragment           2/2                3/3           2/2                4/4 <br /> correct size amplified colony PCR using primers flanking gene interest <br /> Fraction colonies correct PCR product obtained contained        2/2                3/3           2/2                4/4 <br /> the desired mutation <br />  <br />  <br />  <br />  <br />  Figure 6 Introduction ribosome binding site improves production protein encoded downstream gene. (a) Four silent <br />  mutations 3&#8217;-end cat introduced generate effective ribosomal binding site upstream ppa. Sequences junction <br />  cat ppa shown original (1) modified (2) constructs. (b) Growth cells plates containing ampicillin (to force <br />  retention helper plasmid) chloramphenicol (to confirm integration mutation cassette genome) introduction the <br />  mutation cassette electroporation. The lack colonies left indicates construct 1 does enable production essential <br />  protein Ppa. However, construct 2 allows growth, indicating sufficient Ppa produced. <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                          Page 9 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> site upstream essential gene includes <span id='am-32' about='ncbitaxon:synonym' typeof='owl:Thing'>synonym</span>-       codons restore coding sequence gene after <br /> ous codons required produce sufficient protein        integration genome double benefit of <br /> for cell survival cases.                                maintaining function limiting region within <br />                                                                 homologous recombination place, thereby <br /> &#955;-Red recombinase used increase efficiency        increasing proportion crossovers incorporate <br /> of double-strand break repair                                   desired change genome. <br /> The synonymous codon strategy described              The method described enabled highly efficient <br /> used regenerate open reading frame         modification gene targeted, including a <br /> mutant target gene (Figure 4) wild type target gene      number previously published strategies have <br /> (Figure 5) integration mutation cassette      unsuccessful. The streamlined approach have <br /> the genome. The strategy useful desir-     developed substantially decreases time effort <br /> able maintain wild type function gene       required generate genetically modified strains; rou- <br /> the final step. This case endeavored         tinely generate desired mutants week. In <br /> introduce mutations recA2278-5 allele        addition, approach highly versatile, allowing <br /> [21] recA E. coli active RecA needed      introduction point mutations, insertions <br /> repair double-strand break introduced I-SceI [2]         deletions difficult targets, using common <br /> in second step genome editing procedure. Even        set materials procedures. <br /> though wild type RecA available,              The method described paper compares favor- <br /> found frequency loss selection marker        ably recently developed methods genome <br /> after induction I-SceI low; 25           editing E. coli S. enterica. The CRISPR/Cas sys- <br /> colonies patched LBA agar plates lost resistance      tem developed genome editing eukaryotes re- <br /> antibiotic (Additional file 1: Table S2). The reconstitution    cently applied bacteria. This method less <br /> of recA gene using synonymous codons               convenient method described it <br /> resulted lower level RecA. To counter diffi-       requires construction editing fragment spe- <br /> culty, over-expressed Red recombinase             cific targeting RNA mutation introduced, <br /> inducing expression I-SceI. It previously re-       two-step recombination method requires <br /> ported Exo <span id='am-16' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-21'  ></span>Beta</span> Red recombinase increase        mutation cassette. Only examples successful <br /> the efficiency double-strand break repair 100-fold     gene editing reported E. coli [7], general <br /> cells containing wild type RecA [22]. Indeed, overproduc-       applicability bacteria proven. <br /> tion Red recombinase dramatically improved effi-           A distinctly different method accomplishing inser- <br /> ciency mutagenesis; 25 colonies     tions, deletions inversions bacterial genomes using <br /> tested I-SceI induction step lost selection   mobile group II introns Cre-lox been <br /> marker (Additional file 1: Table S2), indicating hom-      recently described [23]. This method useful for <br /> ologous recombination occurred.                             large-scale genomic changes, smaller-scale <br />                                                                 changes, introduction point mutation a <br /> Conclusions                                                     sequence encoding tag. Furthermore, method re- <br /> Although impressive progress               quires initial disruption target gene, be <br /> past decade developing methods introduc-             used present form essential genes. <br /> tion specific genetic changes genomes E.           Finally, new two-plasmid method scarless genome <br /> coli S. enterica, ease performance           editing E. coli reported Yang et al. [24]. <br /> efficiencies best methods vary             This method combines gene gorging use se- <br /> widely. We identified major impediments         lection marker loss indication introduction <br /> success strategies require creation double-          desired mutation genome. The plas- <br /> strand breaks I-SceI followed homologous recom-           mid carries genes Red recombinase I- <br /> bination accomplish genetic changes. In addition,         SceI endonuclease, selection marker flanked <br /> found transcription initiated upstream muta-        I-SceI sites homology regions target particu- <br /> tion cassette apparently interferes ability I-      lar genomic site. After integration selection <br /> SceI cleave recognition site. Introduction     marker, second plasmid carrying mutation cassette, <br /> terminators upstream I-SceI site alleviates         flanked I-SceI sites, introduced. After induc- <br /> problem.                                                        tion I-SceI, cleavage recognition sites plas- <br />   Genetic modification essential genes particularly       mid genome generates fragments can <br /> challenging function gene            recombine integrate mutation fragment the <br /> maintained procedure. We shown              genome. This procedure shown efficient <br /> that use mutation cassette synonymous             cases. However, suited editing of <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                       Page 10 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> essential genes. The method used edit essential      pKDTS digesting plasmid AfeI (Eco47III) and <br /> gene (metK, essential biosynthesis S-          PvuII. The remaining 7187 bp fragment purified gel <br /> adenosylmethionine). However, procedure required         extraction. The 5&#8217;-phosphate removed using Antarctic <br /> clever trick; initial replacement metK gene encod-   phosphatase (New England BioLabs). A 610 bp synthetic <br /> ing transporter allowed uptake S-adenosylmethionine       DNA fragment (gBlock1) (Integrated DNA Technologies, <br /> and rendered MetK activity dispensable. This approach     Coralville, <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>IA</span>) used replace removed fragment. <br /> not applicable cases. Further, Yang et al.      gBlock1 contains 405 bp 5&#8217;-end tetR fragment with <br /> method requires time experimental manipula-           mutations corrected based sequence the <br /> tion method described here.                           tetR gene <span id='am-6' about='obo:NCBITaxon_2389' typeof='owl:Thing obo:NCBITaxon_2387 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Transposon Tn10" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>transposon Tn10</span> (GenBank Accession num- <br />                                                                ber, J01830.1), 81 bp tetA promoter, 81 bp the <br /> Methods                                                        corrected 5&#8217;-end gene encoding I-SceI, 21 <br /> Materials                                                      22 bp complementary 7187 bp pKDTS fragment <br /> E. coli K-12 BW25113 K-12 MG1655 obtained             5&#8217;- 3&#8217;-ends, respectively. gBlock1 assembled <br /> from Nara Institute Science Technology          7187 bp pKDTS fragment make pSLTS the <br /> the American Type Culture Collection (ATCC), respect-          one-step enzymatic DNA assembly method [25] using the <br /> ively. S. enterica Typhimurium SL1344 obtained             Gibson Assembly Master Mix (New England BioLabs). <br /> from Detweiler laboratory University Color-      The procedure used make pKDTS2 the <br /> ado Boulder. Chemically competent E. coli Mach1TM              7187 bp fragment pKDTS gBlock2, encodes <br /> cells (Life Technologies, Inc., Grand Island, NY)         intact I-SceI contains mutations tetR. Cor- <br /> used cloning procedures. Bacteria grown        rect construction pSLTS, pKDTS2, pKDTS was <br /> Miller&#8217;s modified <span id='am-11' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span> Broth (LB) Miller&#8217;s Luria        confirmed sequencing modified parts plas- <br /> agar medium (Research Products International Corp.)            mids using sequencing primers pKDTS-F pKDTS-R. <br /> unless specified. The following antibiotics were <br /> added media required selection: ampicillin,          Construction selection cassette template plasmids <br /> 100 &#956;g/mL; streptomycin, 50 &#956;g/mL; chloramphenicol,            We constructed set template plasmids which <br /> 20 &#956;g/mL; kanamycin, 50 &#956;g/mL kan resistance          selection cassettes containing I-SceI site positive <br /> gene present high-copy-number plasmid             selection marker amplified subsequent incorp- <br /> for selection integration mutation cassette       oration mutation cassettes. These plasmids were <br /> into target DNA, 20 &#956;g/mL subsequent               constructed ligating selection cassette containing an <br /> experiments kan resistance gene inte-       I-SceI recognition site followed antibiotic resistance <br /> grated genomic DNA; trimethoprim, 50 &#956;g/mL            gene pHA1887 (see below). The resulting plas- <br /> when resistance gene present high copy-           mids, pASC, pASK pAST, contain selection cassettes <br /> number plasmid 25 &#956;g/mL resistance gene            confer resistance chloramphenicol, kanamycin and <br /> was integrated genomic DNA. Q5 High-Fidelity          trimethoprim, respectively. A second generation tem- <br /> DNA polymerase (New England Biolabs) Phusion               plate plasmids constructed adding sequences <br /> High-Fidelity DNA polymerase (New England Biolabs)             encoding artificial transcription terminators <br /> were used polymerase chain reactions. Primer se-           upstream I-SceI site. These plasmids assem- <br /> quences descriptions synthetic DNA fragments            bled pHA1877, sequence containing two <br /> are provided Additional file 1.                         transcriptional terminators (STS STTS), selec- <br />                                                                tion cassette (amplified pAS series plasmid using <br /> Construction Red helper plasmids                            primers MarkerF MarkerR), using one-step en- <br /> pKD46 pK-HT obtained Blattner lab            zymatic DNA assembly method [25]. The names these <br /> (University Wisconsin-Madison). pKD46 encodes           plasmids indicate number terminators the <br /> &#955; Red recombinase genes control araB pro-         antibiotic confer resistance. pTS series <br /> moter temperature-sensitive origin replication.       plasmids contain terminator element, I-SceI rec- <br /> We re-constructed previously described pKDTS            ognition site gene conferring resistance chloram- <br /> inserting 1407 bp Nco1 fragment pK-HT,          phenicol (pTSC), kanamycin (pTSK), trimethoprim <br /> encodes I-SceI control anhydrotetracycline-       (pTST). pT2S series plasmids contain terminator ele- <br /> inducible tetA promoter, NcoI site pKD46 [2].      ments, I-SceI recognition site gene conferring <br />   We constructed pSLTS correcting mutations           resistance chloramphenicol (pT2SC pT2SCb), <br /> found pKDTS. A 555 bp fragment containing               kanamycin (pT2SK) trimethoprim (pT2ST). Correct <br /> three mutations encompassing 405 bp 5&#8217;-end          insertion selection cassettes pHA1887 con- <br /> of tetR gene, 81 bp tetA promoter, 69 bp        firmed case sequencing inserted elements <br /> of 5&#8217;-end I-SceI gene removed              using sequencing primers pHA.seq.F pHA.seq.R. <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                    Page 11 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br />    Selection cassettes used construction mu-      spacer elements, patched spa- <br /> tation cassettes amplified selection cassette        cer BBa_K259002, terminator BBa_B1006, spacer <br /> template plasmids using primers MF MR pTS              BBa_B0040 parts. The sequence STTS, con- <br /> and pT2S series plasmids (except pT2SCb,         tains terminator elements flanked spacer <br /> MR2 used instead MR) primers MarkerF            elements, designed minimize secondary structure <br /> MR pAS series plasmids. After treatment amp-        spacer regions using UNAFold (Integrated DNA <br /> lification mixture DpnI remove template            Technologies) patched of <br /> plasmid, amplified selection cassettes gel-purified.      spacer BBa_K259002, spacer BBa_B0040, termin- <br />    The following paragraphs origin as-        ator BBa_B1002, spacer BBa_B0040, terminator <br /> sembly various pieces selection cassette tem-    BBa_B1006, spacer BBa_K259002. STS and <br /> plate fragments.                                               STTS prepared minigenes (Integrated DNA <br />    pHA1887 1887 bp fragment amplified pUC19         Technologies) amplified PCR using primers <br /> using primers pHAFor pHARev. pHA1887 contains              STSF STSR. The amplification mixture treated <br /> the replication origin confers high copy number       DpnI remove template plasmid STS <br /> the bla gene confers ampicillin resistance. PCR-          STTS fragments gel-purified. <br /> amplified pHA1887 treated DpnI remove the <br /> template plasmid gel-purified.                             Construction mutation cassette template plasmids <br />    Genes encoding antibiotic resistance amplified         We constructed mutation cassette template plasmids to <br /> using primers introduced extra sequences use       facilitate subsequent production mutation cassette <br /> either PCR amplification selection cassettes      PCR amplification (see Additional file 1: Figure S3). <br /> genome editing procedure itself. The cat gene ampli-       Mutation cassettes contain selection marker preceded <br /> fied pACYC184 using primers ISceIcatF catR.           transcriptional terminators flanked se- <br /> ISceIcatF contains I-SceI site recognition site followed   quences homologous genomic target (see Figure 1). <br /> by 18 bp 5&#8217; end cat. catR complementary        While 50 bp homology regions (HR1 HR2) each <br /> the 3&#8217; end cat gene. The kan gene amplified         selection marker generally sufficient for <br /> from pACYC177 primers ISceIkanF kancat16R.            homologous recombination genome, typically <br /> ISceIkanF contains I-SceI recognition site followed     obtained colonies using 100 bp homology regions. <br /> 18 bp 5&#8217;-end kan. kancat16R contains        Because efficiency recombination varies among <br /> 16 bp cat followed 3&#8217;-end kan. (The 16 bp cat     genomic targets, routinely use longer 100 bp hom- <br /> sequence included common primer (MR)             ology regions ensure successful integration. The muta- <br /> could used subsequent amplification selection        tion cassette contains identical regions (HR3) that <br /> cassettes containing kan cat). Amplified             span desired genomic modification. The length of <br /> fragments containing cat kan treated       HR3 30&#8211;50 bp, sufficient RecA-mediated recom- <br /> DpnI remove template plasmid, gel-purified          bination (see Step 5 Figure 1), short to <br /> ligated pHA1887 fragment using Rapid DNA          minimize frequency recombination events fail <br /> ligation kit (Thermo Fisher Scientific, Inc.) make pASC     incorporate desired genetic change (see Additional <br /> and pASK, respectively.                                        file 1: Figure S2a). <br />    gISceIdfrA synthetic DNA fragment designed             Mutation template plasmids assembled mix- <br /> contain I-SceI site followed 630 bp dfrA se-        ing 5&#8217;- 3&#8217;- mutation fragments (125 fmol each) <br /> quence (from EZ-Tn5&#8482; &lt; DHFR-1 &gt; (Epicentre))           pHA1877, linear fragment pUC19 obtained by <br /> the 16 bp common primer binding sequence. Twenty bp            PCR (see Additional file 1: Information) (25 fmol), and <br /> sequences complementary pHA1887 included            selection cassette encoding antibiotic resistance <br /> each side. gISceIdfrA assembled pHA1887            gene (75 fmol). The volume mixture ad- <br /> make pAST using one-step enzymatic DNA assem-              justed 10 &#956;L DNase-free water 10 &#956;L <br /> bly method [25].                                               Gibson Assembly Master Mix (New England <br />    gISceIcat2 synthetic DNA fragment designed          BioLabs) added. After incubation 1 hour 50&#176; <br /> contain I-SceI recognition site modified cat         C, 2 &#956;L reaction mixture introduced into <br /> gene (cat2) 3&#8217;-end sequence modified          chemically competent E. coli Mach1TM cells following <br /> to generate sequence resembling consensus bacter-        manufacturer&#8217;s protocol. The transformed cells <br /> ial ribosomal binding site [20].                               spread LB ampicillin antibiotic to <br />    The sequences terminator fragments STS           selection cassette conferred resistance. Typ- <br /> STTS (see Additional file 1: list synthetic DNA frag-       ically, colonies selected confirmation of <br /> ments) designed based iGEM parts. The se-            correct assembly mutation fragments and <br /> quence STS, contains terminator flanked           correct sequence mutation cassette using <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                       Page 12 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> primers pHA.seq.F pHA.seq.R (see Additional                Genome editing procedure <br /> file 1: Table S3).                                             A mutation cassette (50 100 ng) mixed 50 &#956;L <br />    Most mutation fragments used experiments             electrocompetent cells carrying Red helper plasmid <br /> described paper obtained synthetic             ice. After electroporation, 450 &#956;L SOC [27] added <br /> double-stranded DNAs (gBlocks) Integrated                 cells mixture transferred 15 mL <br /> DNA Technologies, Inc., mutation fragments             culture tube. Cells incubated 3 hours shak- <br /> also amplified plasmid genomic DNA. Gener-          ing incubator 30&#176;C spread plates con- <br /> ally, 5&#8217;- mutation fragment contained approximately        taining LB plus ampicillin (LBA) E. coli LB plus <br /> 200 bp target gene flanked sequence           ampicillin streptocymcin (LBAS) S. enterica <br /> overlaps 3&#8217;-end pHA1887 sequence over-       Typhimurium SL1344 additional antibiotic (chlor- <br /> laps 5&#8217;-end selection cassette. The 3&#8217;-mutation       amphenicol (20 &#956;g/mL), kanamycin (50 &#956;g/mL), or <br /> fragment contains approximately 200 bp tar-        trimethoprim (50 &#956;g/mL)) needed select cells in <br /> get gene flanked sequence overlaps 3&#8217;-end        mutation cassette integrated the <br /> of selection cassette sequence overlaps     genome. After overnight incubation 30&#176;C, col- <br /> 5&#8217;-end pHA1887. Both fragments contain 30&#8211;50 bp           onies picked streaked fresh selection <br /> region designated HR3 3&#8217;-end 5&#8217;-mutation         plates. After incubation overnight 30&#176;C, colony <br /> fragment 5&#8217;-end 3&#8217;-mutation fragment.        plate suspended sterile phosphate- <br /> The desired mutation included HR3 (Figure 1          buffered saline (PBS [27]). Because frequency of <br /> and Figure 4), immediately following HR3 (Figure 5).        double strand break repair 1 105, ne- <br />    Mutation cassettes amplified mutation tem-        cessary plate small proportion cells colony <br /> plate plasmids PCR using appropriate primers (see           <span id='am-39' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> obtain colonies subsequent <br /> Additional file 1: Figure S3). The template plasmid re-    step. Aliquots cell suspensions spread on <br /> moved digestion DpnI mutation cassette         plates containing LBA agar LBA agar plus anhydro- <br /> was purified electrophoresis 1% TAE-agarose gel.       tetracycline (100 ng/mL) (LBAaTc). For S. enterica, the <br />                                                                plates contained streptomycin (50 &#956;g/mL). In gen- <br /> Preparation electrocompetent cells carrying Red           eral, plating 50 100 &#956;L cell suspension a <br /> helper plasmid                                                 single colony resuspended 500 &#956;L PBS re- <br /> A Red helper plasmid introduced E. coli K-12          sulted tens colonies plates containing <br /> BW25113 E. coli K-12 MG1655 using method             anhydrotetracycline. To confirm double strand break <br /> Chung et al. [26]. The plasmid (100 ng) mixed         repair resulted excision selection cassette, <br /> competent cells prepared 1 mL LB culture. A Red         colonies LBAaTc plate patched <br /> helper plasmid introduced electrocompetent S.         LB agar LB agar supplemented the <br /> enterica Typhimurium SL1344 cells electroporation. (S.      antibiotic mutation cassette provided re- <br /> enterica Typhimurium SL1344 cells rendered elec-          sistance. (Streptomycin added S. enterica <br /> trocompetent described Molecular Cloning [27]).          Typhimurium SL1344). Ampicillin included in <br /> The transformants spread LB Miller agar medium         plates force retention helper plasmid if <br /> containing ampicillin (100 &#956;g/mL). After overnight growth      rounds mutations planned. Colonies that <br /> at 30&#176;C, single colony used inoculate 5 mL LB      grew LB agar plates plates supple- <br /> Miller medium containing ampicillin (100 &#956;g/mL) (LBA)          mented antibiotic streaked fresh LB <br /> for E. coli K-12 strains ampicillin (100 &#956;g/mL)         agar plates. After overnight incubation, individual <br /> streptomycin (50 &#956;g/mL) (LBAS) S. enterica Typhi-          colonies used prepare freezer stocks. The intro- <br /> murium SL1344. After overnight incubation shaking         duction desired mutation verified amplify- <br /> at 30&#176;C, 1 mL culture inoculated 100 mL        ing target region PCR sequencing. <br /> of LBA LBAS 500 mL Erlenmyer flask. The culture           When necessary increase efficiency double- <br /> was incubated shaking hour 30&#176;C. L-             strand break repair, expression Red recombinase <br /> Arabinose added final concentration 1 mM       induced expression I-SceI. A single colony <br /> E. coli K-12 BW25113 2 mM E. coli K-12 MG1655           mutation cassette integrated was <br /> and S. enterica Typhimurium SL1344 induce expression        resuspended 50 &#956;L sterile DNase-free water. An ali- <br /> of &#955;-Red recombinase. Incubation 30&#176;C shak-        quot (20 &#956;L) boiled 100&#176;C prior amplification of <br /> ing continued 2&#8211;3 hours. When OD600         mutation cassette PCR confirm correct integra- <br /> culture reached 0.7-0.9, cells harvested centri-   tion mutation cassette genome. The <br /> fugation 4500 &#215; g washed twice ice-cold 10%        remaining aliquot (30 &#956;L) mixed 120 &#956;L ster- <br /> glycerol. The cells resuspended 10% glycerol       ile PBS. Fifty &#956;L suspended cells spread on <br /> stored &#8722;70&#176;C use.                                    plates containing LBA LBAaTc. The remaining 50 &#956;L <br />  Kim et al. BMC Biotechnology 2014, 14:84                                                                                                             Page 13 13 <br /> http://www.biomedcentral.com/1472-6750/14/84 <br />  <br />  <br />  <br />  <br /> were inoculated 1 mL LBA 15 mL culture                              7.      Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA: RNA-guided editing of <br /> tube. The culture incubated shaking incubator                                   bacterial genomes using CRISPR-Cas systems. Nat Biotechnol 2013, <br />                                                                                          31(3):233&#8211;239. <br /> for hour 30&#176;C. L-Arabinose added final                            8.      Mosberg JA, Gregg CJ, Lajoie MJ, Wang HH, Church GM: Improving lambda <br /> concentration 1 mM E. coli K-12 BW25113                                        red genome engineering Escherichia coli rational removal of <br /> 2 mM E. coli K-12 MG1655 S. enterica induce                                   endogenous nucleases. PLoS One 2012, 7(9):e44638. <br />                                                                                  9.      Wang HH, Xu G, Vonner AJ, Church G: Modified bases enable high- <br /> production Red recombinase. After 2 hours fur-                                 efficiency oligonucleotide-mediated allelic replacement mismatch <br /> ther incubation 30&#176;C shaking incubator, 50 &#956;L ali-                               repair evasion. Nucleic Acids Res 2011, 39(16):7336&#8211;7347. <br /> quots spread plates containing LBA LBAaTc.                            10.     Wendland J: PCR-based methods facilitate targeted gene manipulations <br />                                                                                          cloning procedures. Curr Genet 2003, 44(3):115&#8211;123. <br />                                                                                  11.     Cho BK, Knight EM, Palsson BO: PCR-based tandem epitope tagging <br /> Additional file                                                                          Escherichia coli genome engineering. Biotechniques 2006, <br />                                                                                          40(1):67&#8211;72. <br />                                                                                  12.     Cox MM, Layton SL, Jiang T, Cole K, Hargis BM, Berghman LR, Bottje WG, <br />  Additional file 1: Figure S1. Diagrams design mutation cassettes                 Kwon YM: Scarless site-directed mutagenesis Salmonella enteritidis <br />  insertion (a) deletion (b). Figure S2. Synonymous codon                         chromosome. BMC Biotechnol 2007, 7:59. <br />  fragment increases efficiency genome editing essential            13.     Stringer AM, Singh N, Yermakova A, Petrone BL, Amarasinghe JJ, Reyes-Diaz <br />  target. (a) Homologous recombination mutation cassette                  L, <span id='am-22' about='obo:NCBITaxon_7506' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_267069 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_7505 obo:NCBITaxon_2759 obo:NCBITaxon_7504 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_627860 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6970 obo:NCBITaxon_33341 obo:NCBITaxon_50557'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Mantis" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26'  ></span>Mantis</span> NJ, Wade JT: FRUIT, scar-free targeted chromosomal <br />  target gene step 3 fail introduce mutations HR3 (purple band               mutagenesis, epitope tagging, promoter replacement Escherichia <br />  wedge) 3&#8217;-end mutation fragment second                       coli Salmonella enterica. PLoS One 2012, 7(9):e44841. <br />  cross-over occurs green red region HR3. The             14.     Blank K, Hensel M, Gerlach RG: Rapid highly efficient method for <br />  consequence regeneration wild type sequence                        scarless mutagenesis Salmonella enterica chromosome. PLoS <br />  second homologous recombination step (step 4). (b) The use                         One 2011, 6(1):e15763. <br />  synonymous codo <br /> </body></html>