<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Ecosal Plus. 2014 ; 2014: . doi:10.1128/ecosalplus.ESP-0009-2013. <br />  <br />  <br />  <br />                            The EcoCyc Database <br />                            Peter D. Karp1, Daniel Weaver1, Suzanne Paley1, Carol Fulcher1, Aya Kubo1, <span id='am-72' about='obo:NCBITaxon_188061' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5338 obo:NCBITaxon_34450 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_451864 obo:NCBITaxon_5204 obo:NCBITaxon_2759 obo:NCBITaxon_155619'><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Anamika" datatype="xsd:string"></span>Anamika</span> <br />                            Kothari1, Markus Krummenacker1, Pallavi Subhraveti1, Deepika Weerasinghe1, Socorro <br />                            Gama-Castro2, Araceli M. Huerta2, Luis Mu&#241;iz-Rascado2, C&#233;sar Bonavides-Martinez2, <br />                            Verena Weiss2, Martin Peralta-Gil2, Alberto Santos-Zavaleta2, Imke Schr&#246;der3,4, Amanda <br />                            Mackie5, Robert Gunsalus3,4, Julio Collado-Vides2, Ingrid M. Keseler1, Ian Paulsen5 <br />                            1Bioinformatics     Research Group, SRI International, Menlo Park, USA <br />                            2Programade Gen&#243;mica Computacional, Centro Ciencias Gen&#243;micas, Universidad Nacional <br />                            Aut&#243;noma M&#233;xico, A.P. 565-A, Cuernavaca, Morelos 62100, M&#233;xico <br />                            3Department Microbiology, Immunology, Molecular Genetics, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span>, Los <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Angeles, CA 90095, USA <br />                            4UCLA     Institute Genomics Proteomics, University California, Los Angeles, CA 90095, <br />                            USA <br />                            5Department Chemistry Biomolecular Sciences, Macquarie University, Sydney, New South <br />                            Wales 2109, Australia <br />  <br />                            Abstract <br />                                 EcoCyc bioinformatics database available EcoCyc.org describes genome the <br />                                 biochemical machinery <span id='am-90' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> K-12 MG1655. The long-term goal project to <br />                                 complete molecular catalog E. coli cell, functions of its <br />                                 molecular parts, facilitate system-level understanding E. coli. EcoCyc electronic <br />                                 reference source E. coli biologists, biologists work related microorganisms. <br />                                 The database includes information pages E. coli gene, metabolite, reaction, operon, and <br />                                 metabolic pathway. The database includes information E. coli gene essentiality, on <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 nutrient conditions support growth E. coli. The web site downloadable <br />                                 software contain tools analysis high-throughput datasets. In addition, steady-state <br />                                 metabolic flux model generated new version EcoCyc. The model predict <br />                                 metabolic flux rates, nutrient uptake rates, growth rates different gene knockouts and <br />                                 nutrient conditions. <span id='am-60' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-61'  ></span><span id='am-62' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> chapter provides detailed description <span id='am-136' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-140'  ></span>data</span> content EcoCyc, <br />                                 procedures content generated. <br />  <br />  <br />                            1 EcoCyc Overview <br />                                              EcoCyc1 bioinformatics database describes genome biochemical <br />                                              machinery E. coli K-12 MG1655. The project&#8217;s long-term goal describing complete <br />                                              molecular catalog E. coli cell, functions of molecular parts, <br />  <br />  <br />  <br />                            1&#8220;EcoCyc&#65533;? pronounced &#8220;eeko-sike.&#65533;? It sounds like &#8220;ecology&#65533;? like &#8220;encyclopedia&#65533;?. <br />                             Karp et al.                                                                                             Page 2 <br />  <br />  <br />                                          facilitate system-level understanding E. coli. EcoCyc electronic reference source <br />                                          E. coli biologists researchers work E. coli related <br />                                          microorganisms. In addition database, steady-state metabolic flux model is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          available, generated new version EcoCyc. <br />  <br />                                          This chapter provides overview EcoCyc&#8217;s data content procedures which <br />                                          data enter EcoCyc. <br />  <br />                                          EcoCyc accelerates science. EcoCyc designed different modes interactive <br />                                          use EcoCyc.org web site conjunction downloadable Pathway <br />                                          Tools [1] software (Section 13 lists resources available assist users learning web <br />                                          site software)): <br />  <br />                                              &#8226;    EcoCyc encyclopedic reference providing information biological <br />                                                   roles E. coli genes, metabolites, pathways. Visualization tools, a <br />                                                   genome browser, metabolic map display, regulatory network diagram, aid in <br />                                                   comprehension complex data. <br />  <br />                                              &#8226;    EcoCyc facilitates analysis high-throughput data gene-expression and <br />                                                   metabolomics data tools enrichment analysis, visualizing omics data <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   metabolic map diagram, complete genome diagram, regulatory network <br />                                                   diagram. <br />  <br />                                              &#8226;    The EcoCyc metabolic flux model predict growth no-growth wildtype and <br />                                                   knock-out E. coli strains different nutrient conditions. <br />  <br />                                          Users EcoCyc fall different groups. Experimental biologists use EcoCyc as <br />                                          encyclopedic reference genes, pathways, regulation, use omics-data <br />                                          analysis tools analyze gene-expression metabolomics data. Examples papers citing <br />                                          EcoCyc analysis functional genomics data include: [2, 3, 4, 5, 6]. <br />  <br />                                          Because EcoCyc data structured sophisticated ontology amenable to <br />                                          computational analyses, EcoCyc enables scientists ask computational questions spanning <br />                                          entire genome E. coli, known metabolic network E. coli, known transport <br />                                          complement E. coli, known genetic regulatory network E. coli, combinations <br />                                          thereof. Past work includes use EcoCyc develop methods studying path lengths <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          metabolic networks [7, 8, 9]; studies relating protein structure metabolic <br />                                          network [10, 11]; analysis E. coli regulatory network [12, 13]. <br />  <br />                                          The development new bioinformatics methods requires high-quality, gold-standard <br />                                          datasets training validation methods. EcoCyc used gold- <br />                                          standard dataset development genome-context methods predicting gene <br />                                          function [14, 15], operon-prediction methods [16, 17], prediction promoters and <br />                                          transcription start sites [18, 19], regulatory network reconstruction [20], prediction <br />                                          functional direct protein-protein interactions [21, 22, 23]. The EcoCyc metabolic data <br />                                          used studies concerning predicted metabolic networks growth prediction <br />                                          [24, 25], model checking symbiotic bacteria&#8217;s metabolic network [26]. <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 3 <br />  <br />  <br />                                          Metabolic engineers alter microbes produce biofuels, industrial chemicals, and <br />                                          pharmaceuticals; de-grade toxic pollutants; sequester carbon [27, 28, 29]. <br />                                          Metabolic engineers use E. coli host organism consult EcoCyc aid in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          optimizing production end product better under-standing the <br />                                          metabolic network regulation, predict undesirable effects metabolic <br />                                          alteration. Metabolic engineering studies using EcoCyc include [30, 31, 32]. <br />  <br />                                          According Thomson Reuters Web Knowledge citation index, August 2013, <br />                                          23 EcoCyc RegulonDB papers authored 1997 cited 2,395 publications <br />                                          1997&#8211;2013. According Google Analytics, approximately 100,000 visitors query the <br />                                          EcoCyc website year, generating 177,000 object page views month average in <br />                                          2012. <br />  <br />                                          EcoCyc data available download multiple file formats (see http://biocyc.org/ <br />                                          download.shtml) queried programmatically web services (see http:// <br />                                          biocyc.org/web-services.shtml). <br />  <br />                                          The Pathway Tools software underlies EcoCyc [1] specific E. coli, rather <br />                                          applied manage genomic biochemical data thousands organisms. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2 Overview EcoCyc Data Content <br />                                          EcoCyc covers broad array data types. Key understanding EcoCyc data its <br />                                          presentation EcoCyc website Pathway Tools notion database <span id='am-52' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span>, <br />                                          describes specific type data. For example, class Genes provides database <br />                                          <span id='am-135' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> gene, including attributes (e.g., starting nucleotide position the <br />                                          genome) relationships (e.g., linkage gene gene product) class. <br />                                          Each specific gene EcoCyc stored single database object, frame, an <br />                                          instance class Genes. <br />  <br />                                          No one-to-one mapping exists EcoCyc classes data pages the <br />                                          EcoCyc website, data page typically integrates information multiple <br />                                          classes. For example, pathway data page integrates information objects the <br />                                          classes Pathways, Reactions, Genes, Proteins, Chemicals. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Genome <br />                                          EcoCyc contains complete genome sequence E. coli describes nucleotide <br />                                          position function known protein-coding RNA-coding E. coli genes. Genome- <br />                                          related classes populated EcoCyc include Genes, Pseudo-Genes, Promoters, <br />                                          DNA-Binding-Sites, REP-Elements. Gene Ontology (GO) terms assigned genes <br />                                          EcoCyc curators import GO terms UniProt [33]. EcoCyc data the <br />                                          essentiality E. coli genes described Section 6. <br />  <br />                               Proteome <br />                                          EcoCyc describes known monomers multimeric protein complexes E. coli. <br />                                          EcoCyc contains extensive annotation features E. coli proteins, as <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 4 <br />  <br />  <br />                                            phosphorylation sites, metal-ion binding sites, enzyme active sites, assigned EcoCyc <br />                                            curators imported UniProt. Relevant classes EcoCyc include Polypeptides <br />                                            Protein-Complexes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               RNAome <br />                                            EcoCyc describes known RNAs protein-RNA complexes E. coli. Relevant classes <br />                                            EcoCyc include RNAs, rRNAs, Regulatory-RNAs. Note EcoCyc does not <br />                                            explicitly represent messenger RNAs. <br />  <br />                               Regulation <br />                                            EcoCyc contains complete description regulatory network organism. <br />                                            It covers E. coli operons, promoters, transcription factors, transcription-factor binding sites, <br />                                            attenuators, small-RNA regulators, substrate-level regulation E. coli <br />                                            enzymes. Each molecular regulatory interaction described instance class <br />                                            Regulation, subclasses different types regulation. <br />  <br />                               Metabolism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            EcoCyc describes known metabolic signal-transduction pathways E. coli. It <br />                                            describes metabolic enzyme E. coli including cofactors, activators, inhibitors, and <br />                                            subunit structure. <br />  <br />                               Membrane Transporters <br />                                            EcoCyc annotates E. coli transport proteins associated transport reactions they <br />                                            mediate. <br />  <br />                               Growth Observations <br />                                            EcoCyc integrates data growth E. coli different growth conditions, as <br />                                            described Section 5. <br />  <br />                               Database Links <br />                                            EcoCyc linked biological databases containing protein nucleic acid sequence <br />                                            data, bibliographic data, protein structures, descriptions different E. coli strains. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3 Literature-Based Curation <br />                                            Curation process manually refining updating bioinformatics database. The <br />                                            EcoCyc project uses literature-based curation approach database updates are <br />                                            based evidence experimental literature. EcoCyc largely date respect to <br />                                            curation activities. As October 2013, EcoCyc encodes information than <br />                                            25,000 publications. A staff full-time curators updates annotation E. coli <br />                                            genome ongoing basis. <br />  <br />                                            The transcriptional regulatory information EcoCyc RegulonDB curated the <br />                                            group Dr. Julio Collado-Vides UNAM; therefore, databases include same <br />  <br />  <br />  <br />                                                Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                            Page 5 <br />  <br />  <br />                                          data content transcriptional regulation gene expression. The actual data curation occurs <br />                                          EcoCyc, information periodically propagated RegulonDB. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Curators collect gene, protein, pathway, compound names synonyms. They classify <br />                                          genes gene products using Gene Ontology [34] MultiFun [35] ontologies, and <br />                                          classify pathways Pathway Tools pathway ontology. Protein complex <br />                                          components stoichiometry subunits captured; cellular localization of <br />                                          polypeptides protein complexes entered, experimentally determined protein <br />                                          molecular weights; enzyme activities enzyme prosthetic groups, cofactors, <br />                                          activators, inhibitors captured. Operon structure gene regulation information are <br />                                          encoded. <br />  <br />                                          Curators author textual summaries extensive citations. Within summaries for <br />                                          proteins, RNAs, pathways, operons, curators capture additional information not <br />                                          captured highly structured database fields EcoCyc. For example, curators <br />                                          use free-text summary sections overall function gene product, the <br />                                          phenotypes caused mutation, depletion, overproduction gene product; any <br />                                          known genetic interactions; protein domain architecture structural studies; similarity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          proteins; functional complementation experiments described. <br />                                          Summaries used note cases published reports present <br />                                          contradictory results. In cases, viewpoints presented proper <br />                                          attribution. This approach strives ensure information lost. <br />  <br />                                          EcoCyc entries generally updated new literature available. Regular <br />                                          PubMed searches used generate lists potentially curatable publications, are <br />                                          evaluated prioritized curation. Papers containing newly identified functions of <br />                                          gene products, substantial advances understanding functions known gene <br />                                          products, given highest priority curation. Because Pathway Tools software <br />                                          continues evolve enable addition new data types, older entries being <br />                                          updated systematic fashion (e.g., enzyme metabolic pathway) time allows. <br />  <br />                               4 Statistics EcoCyc Content <br />                                          Tables 1, 2, 3, 4 present statistics EcoCyc content. The listed numbers current as <br />                                          version 17.5, released October 2013. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               5 Conditions E. coli Growth Non-Growth <br />                                          As 2011, EcoCyc incorporates media shown experimentally support or <br />                                          support growth wildtype knock-out strains E. coli K-12. This work has <br />                                          goals. First assemble comprehensive encyclopedia E. coli growth conditions <br />                                          experimentalists. The spectrum environmental conditions supporting growth a <br />                                          bacterium important phenotypic traits. We expect understand the <br />                                          functions genes organism unless understand range environments in <br />                                          cell grow. Second, comprehensive collection E. coli growth media will <br />                                          drive accurate systems biology modeling E. coli. The larger set growth media <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 6 <br />  <br />  <br />                                          computational models validated, accurate and <br />                                          comprehensive models be. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EcoCyc captures approximately 20 media commonly used E. coli laboratories; <br />                                          growth data provided media. EcoCyc records results high- <br />                                          throughput experiments using Biolog Phenotype Microarrays (PMs), measure cell <br />                                          respiration sensitive <span id='am-12' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="rdfs:label" content="Indicator" datatype="xsd:string"></span>indicator</span> microbial growth [36]. The commercially available PM <br />                                          microorganisms provides comprehensive set phenotype tests including <br />                                          information ability metabolize 190 carbon (C) compounds, 95 nitrogen (N) <br />                                          compounds, 59 phosphorus (P) compounds, 35 sulfur (S) compounds. EcoCyc currently <br />                                          documents sets PM data following sources: <br />  <br />                                              &#8226;    B. Bochner X. Lei, personal communication, 2012. <br />  <br />                                                   Strain: E. coli K&#8211;12 BW30270 (rph+ (RNase PH) derivative MG1655; the <br />                                                   strains PyrE deficiency. Found fnr+ well, according to <br />                                                   Datsenko Wanner, unpublished results.) <br />  <br />                                                   This dataset includes aerobic growth observations complement C, N, <br />                                                   P, S compounds included PM plus growth observations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   95 C sources anaerobic conditions. <br />  <br />                                              &#8226;    &#8220;Genome Scale Reconstruction <span id='am-113' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-120'  ></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span>Salmonella</span> Metabolic Model&#65533;? <br />  <br />                                                   AbuOun et al. 2009 [37] <br />  <br />                                                   Strain: E. coli K&#8211;12 MG1655 (American Type Culture Collection 700926) <br />  <br />                                                   This dataset includes growth observations complement C, N, P, S <br />                                                   compounds aerobic conditions. Bacteria pre-grown LB agar prior to <br />                                                   inoculation Biolog plates incubation 37&#176;C 26 hours. The Omnilog <br />                                                   instrument (a specialized incubator plus reader) used data collection and <br />                                                   analysis. <br />  <br />                                              &#8226;    &#8220;The Evolution Metabolic Networks E. coli&#65533;? <br />  <br />                                                   Baumler et al. 2011 [38] <br />  <br />                                                   Strain: E. coli K&#8211;12 MG1655 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   This dataset consists growth observations 95 C compounds aerobic and <br />                                                   anaerobic conditions. Bacteria pre-grown Biolog Universal Growth Agar <br />                                                   plus sheep blood (BUG-S), prior inoculation Biolog plates incubation at <br />                                                   37&#176;C. Growth monitored measuring optical density 600nm readings <br />                                                   taken 3, 6, 12, 24, 48 hours (D. Baumler, personal communication). <br />  <br />                                              &#8226;    Mackie et al. 2013 [39] <br />  <br />                                                   Strain: E. coli K&#8211;12 MG1655 (Coli Genetic Stock Center 7740). <br />  <br />                                                   This data set consists growth observations complement C, N, P <br />                                                   S compounds aerobic conditions. Bacteria pre-grown LB <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 7 <br />  <br />  <br />                                                   R2A agar prior inoculation Biolog plates incubation 37&#176;C 48 <br />                                                   hours. The Omnilog instrument used data collection analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              &#8226;    &#8220;Comparative Multi-Omics Systems Analysis <span id='am-106' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-112'  ></span>Escherichia</span> coli strains B and <br />                                                   K-12&#65533;? <br />  <br />                                                   Yoon et al. 2012 [40] <br />  <br />                                                   Strain: E. coli K&#8211;12 MG1655 <br />  <br />                                                   This data set consists growth observations complement C, N, P <br />                                                   S compounds aerobic conditions. Bacteria pregrown BUG-S <br />                                                   agar prior inoculation Biolog plates incubation 37&#176;C 48 hours. The <br />                                                   Omnilog instrument used data collection analysis. <br />  <br />                                          Data growth conditions accessed EcoCyc website invoking menu <br />                                          command Search &#8594; Growth Media clicking button &#8220;All Growth Media for <br />                                          Organism.&#65533;? Individual media shown initial table; PM data shown the <br />                                          following tables. The coloring box indicates degree growth observed under <br />                                          condition. Three levels growth recorded: growth, low growth, growth (see <br />                                          legend indicates colors associated level growth). Click growth <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          medium request page describing composition, genes essential or <br />                                          essential growth condition. <br />  <br />                               6 Essential Gene Information <br />                                          As 2011, EcoCyc incorporates large-scale datasets gene essentiality E. coli. <br />                                          Gene essentiality information useful for <br />  <br />                                              &#8226;    Predicting antibiotic targets pathogenic bacteria. <br />  <br />                                              &#8226;    Guiding design minimal genomes. <br />  <br />                                              &#8226;    Validating genome-scale metabolic flux models. Model predictions be <br />                                                   compared experimental data recorded EcoCyc assess model accuracy. <br />  <br />                                              &#8226;    Providing clues regarding functions genes unknown function, when <br />                                                   essentiality varies depending conditions growth. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EcoCyc incorporates data essentiality following publications: <br />  <br />                                              &#8226;    &#8220;Experimental Determination System Level Analysis Essential Genes in <br />                                                   Escherichia coli MG1655&#65533;? <br />  <br />                                                   Gerdes et al. [41] <br />  <br />                                                   Strain: E. coli K&#8211;12 MG1655 (F - &#955; - ilvG rfb-50 rph-1) <br />  <br />                                                   This study used genetic footprinting technique Tn5 based transposome <br />                                                   reported unambiguous assessment approximately 87% E. coli <br />                                                   ORFs essentiality. 626 genes identified essential aerobic growth in <br />                                                   rich media 3126 genes dispensable. Note inability obtain an <br />                                                   insertion mutant using cases reflection non- <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 8 <br />  <br />  <br />                                                targeted nature transposon insertion reflection gene essentiality. <br />                                                For technical reasons 327 genes classified study as <br />                                                ambiguous regard essentiality. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          &#8226;     Construction <span id='am-146' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span>Escherichia coli K-12</span> In-Frame, Single-Gene Knockout Mutants: <br />                                                The Keio Collection Baba et al. [42] corrections [43] <br />  <br />                                                Strain: E. coli K&#8211;12 BW25113 (rpoS(Am) rph-1 &#955; - rrnB3 &#916;lacZ4787 hsdR514 <br />                                                &#916;(araBAD)567 &#916;(rhaBAD)568 rph-1) <br />  <br />                                                This study created 3985 in-frame, single gene deletion mutants using lambda <br />                                                RED recombinase system. 303 genes unable disrupted were <br />                                                predicted essential growth rich media 37&#176;C. Note cases <br />                                                secondary impacts single-gene deletions, compensating <br />                                                suppressor mutations. There errors mutants described in <br />                                                paper, later corrected [43]. This study profiled growth of <br />                                                mutants minimal glucose <span id='am-141' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-142' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-144'  ></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>MOPS</span> media identify genes are <br />                                                conditionally essential conditions. <br />  <br />                                          &#8226;     &#8220;Experimental Computational Assessment Conditionally Essential Genes in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Escherichia coli&#65533;? Joyce et al. [44] <br />  <br />                                                Strain: E. coli K&#8211;12 BW25113 (rpoS(Am) rph-1 &#955; - rrnB3 &#916;lacZ4787 hsdR514 <br />                                                &#916;(araBAD)567 &#916;(rhaBAD)568 rph-1) (the [42]) <br />  <br />                                                This study used Keio collection single gene knockout mutants profiled <br />                                                growth glycerol supplemented minimal medium. 119 genes were <br />                                                identified essential growth glycerol. They combined these <br />                                                observations Baba et al [42] regarding conditional <br />                                                essentiality mutants grown glucose supplemented minimal media <br />                                                able identify conserved conditionally essential core 94 genes <br />                                                required E. coli K&#8211;12 grow minimal nutritional <br />                                                supplementation essential growth rich conditions. <br />  <br />                                          &#8226;     &#8220;A Genome-Scale Metabolic Reconstruction Escherichia coli K-12 MG1655 <br />                                                Accounts 1260 ORFs Thermodynamic Information&#65533;? <br />  <br />                                                Feist et al. [45] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                This publication used experimental data regarding conditional gene essentiality <br />                                                Joyce et al. [44] Baba et al. [42] compared the <br />                                                computationally predicted essential genes genome scale metabolic <br />                                                reconstruction E. coli. This dataset included EcoCyc facilitate <br />                                                benchmarking computational predictions essentiality EcoCyc model <br />                                                computations model Feist et al. Multicopy suppression underpins <br />                                                metabolic evolvability. <br />  <br />                                          &#8226;     &#8220;Multicopy suppression Underpins Metabolic Evolvability&#65533;? <br />  <br />                                                Patrick et al. 2007 [46] <br />  <br />  <br />  <br />  <br />                                          Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 9 <br />  <br />  <br />                                                   Strain: E. coli BW25113 (rpoS(Am) rph-1 &#955; - rrnB3 &#916;lacZ4787 hsdR514 <br />                                                   &#916;(araBAD)567 &#916;(rhaBAD)568 rph-1) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   This study used conditionally essential gene sets identified Baba et al and <br />                                                   Joyce et al tested ability form colonies glucose M9 agar. <br />                                                   They identified 107 genes conditionally essential conditions. <br />  <br />                                          When essentiality data available given gene, EcoCyc gene page includes table <br />                                          conditions gene essential essential <br />                                          growth. Clicking condition navigate growth-medium page lists all <br />                                          essentiality information growth condition. <br />  <br />                               7 EcoCyc Metabolic Flux Model <br />                                          A quantitative steady-state metabolic flux model derived EcoCyc using flux <br />                                          balance analysis (FBA) [47, 48]. By running model different parameters, scientists <br />                                          model growth E. coli different nutrient conditions different gene <br />                                          knock-outs. Every time model executed, freshly generated EcoCyc, meaning <br />                                          reactions EcoCyc updated curation, model automatically reflects <br />                                          changes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The EcoCyc FBA model distinct E. coli FBA models derived Palsson <br />                                          group [49, 45, 50], models common EcoCyc the <br />                                          iAF1260 model partially unified 2007 [45], groups consult other&#8217;s <br />                                          work updating models. <br />  <br />                                          The Supplementary Information, provided accompanying paper-suppl-info.xlsx file, <br />                                          details E. coli biomass metabolite set used model biomass production metabolite <br />                                          requirements EcoCyc FBA. This metabolite set derived iJO1366 model WT <br />                                          biomass reaction Orth et al. [50]. The Supplementary Information contains a <br />                                          description nutrient secretion metabolite sets supply inputs outputs the <br />                                          FBA model, description differences EcoCyc FBA biomass <br />                                          metabolite set iJO1366 WT biomass reaction. <br />  <br />                                          To run EcoCyc FBA model, download install Pathway Tools software <br />                                          configuration includes EcoCyc, invoke MetaFlux modeling component of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Pathway Tools (see Chapter 8 Pathway Tools User&#8217;s Guide). <br />  <br />                                          EcoCyc provides example files describing invocations FBA model under <br />                                          different nutrient conditions. Those files installed Pathway Tools <br />                                          directory tree pathway-tools/aic-export/pgdbs/biocyc/ecocyc/VERSION/data/fba/. Output <br />                                          files produced result successful FBA runs supplied. fba files included. <br />                                          The supplied input files are: <br />  <br />                                              1.   GlucoseAer.fba: 10 mmol/gCDW/hr glucose uptake, minimal media, aerobic <br />                                                   conditions <br />  <br />                                              2.   GlucoseAnaer.fba: 10 mmol/gCDW/hr glucose uptake, minimal media, anaerobic <br />                                                   conditions <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                            Page 10 <br />  <br />  <br />                                               3.   GlycerolAer.fba: 10 mmol/gCDW/hr glycerol uptake, minimal media, aerobic <br />                                                    conditions <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               4.   GlycerolAnaer.fba: 10 mmol/gCDW/hr glycerol uptake, minimal media, anaerobic <br />                                                    conditions <br />  <br />                               7.1 External Flux Predictions <br />                                           MetaFlux metabolic flux predictions EcoCyc version 17.5 aerobic growth on <br />                                           glucose glycerol given Tables 5 6. Model predictions anaerobic growth on <br />                                           glucose glycerol given Tables 7 8. In cases, uptake rate carbon <br />                                           source set upper bound reflecting experimental uptake rates mmol/gCDW/hr. O2 <br />                                           uptake rates set upper bound 0.00 mmol/gCDW/hr anaerobic conditions. <br />                                           All nutrient sources left free. <br />  <br />                               7.2 Improvement Metabolic Model <br />                                           With EcoCyc release plan include improved version EcoCyc metabolic <br />                                           flux model reflects recent improvements knowledge E. coli metabolic <br />                                           network. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Model predictions differ experimental measurements number reasons <br />                                           including operation additional, unmodeled reactions metabolites; existing <br />                                           reactions operating different fashion model (e.g. model contains &#8220;perfect&#65533;? <br />                                           respiratory <span id='am-126' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span>-transfer chain possibility reactive oxygen-<span id='am-77' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-78' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> <br />                                           generation); presence regulation product inhibition deactivates reactions or <br />                                           limits throughput; differences optimization objective functions depending the <br />                                           specified feed source. <br />  <br />                               8 Update Frequency <br />                                           The EcoCyc.org BioCyc.org websites downloadable files updated 3&#8211;4 times per <br />                                           year. A faster, powerful version EcoCyc install locally your <br />                                           (Macintosh, PC/Windows, PC/Linux) released semiannually. <br />  <br />                               9 Data Sources Incorporated EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           EcoCyc includes data imported following bioinformatics databases. In cases, <br />                                           data re-imported twice year. We note literature references <br />                                           EcoCyc obtained PubMed. <br />  <br />                               9.1 UniProt Features <br />                                           UniProt protein features (the UniProt KB term sequence annotations) complete <br />                                           proteome E. coli K-12 MG1655 SwissProt imported EcoCyc EcoCyc <br />                                           release. We import protein features experimental non-experimental evidence <br />                                           qualifiers following types: turn, <span id='am-47' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Helix" datatype="xsd:string"></span>helix</span>, <span id='am-17' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span> strand, coiled-coil. The chain <br />                                           type imported does span entire length protein. Examples of <br />                                           imported feature types include catalytic domains, phosphorylation sites, metal ion <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 11 <br />  <br />  <br />                                          binding sites. We import citations associated UniProt protein features include an <br />                                          associated PubMed ID. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The import protein features EcoCyc UniProt Feature Importer tool <br />                                          Pathway Tools software. <br />  <br />                               9.2 Gene Ontology <br />                                          For years, EcoCyc EcoliWiki/PortEco collaborating improving <br />                                          maintaining GO annotations E. coli. GO applications described in <br />                                          [56]. Since summer 2008, periodically generating file <br />                                          containing E. coli K-12 GO term annotations, called gene_association.ecocyc, may <br />                                          obtained Gene Ontology Consortium. <br />  <br />                                          GO annotation standard EcoCyc&#8217;s manual literature-based curation process. The <br />                                          GO annotations added database objects represent functional gene products <br />                                          multimers, directly gene objects. This approach models biology more <br />                                          accurately indicates exactly form gene product specified GO <br />                                          function. In parallel, manual annotation E. coli genes GO ongoing EcoliWiki. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          On regular basis, GO annotations merged. The latest UniProt EcoliWiki <br />                                          annotations imported EcoCyc. Because GO consortium does accept <br />                                          electronic annotations gene association file annotations than <br />                                          year old, UniProt annotations reimported EcoCyc regular basis. <br />  <br />                                          EcoCyc incorporates electronic experimental GO term annotations E. coli K-12 <br />                                          gene products obtained &#8220;UniProt [multispecies] GO Annotations @ EBI&#65533;? file <br />                                          downloaded Gene Ontology Consortium. When import performed in <br />                                          2007, approximately 30,000 new IEA (&#8220;Inferred Electronic Annotation&#65533;?) GO term <br />                                          assignments added EcoCyc, approximately 1,000 assignments with <br />                                          experimental evidence codes including assignments high-throughput protein- <br />                                          interaction studies. During import GO terms UniProt EcoCyc, filtering <br />                                          operation applied prune GO term annotations based solely computational (IEA) <br />                                          evidence EcoCyc gene product specific GO annotations (in other <br />                                          words, GO terms children GO term imported) experimental <br />                                          evidence available. For example, gene product contained experimental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          annotation term &#8220;galactose kinase,&#65533;? software add computational <br />                                          annotation &#8220;carbohydrate kinase.&#65533;? This filtering leads removal approximately 1,000 <br />                                          specific redundant annotations. <br />  <br />                                          A gene association file generated quarterly EcoCyc releases. This file sent to <br />                                          EcoliWiki team Texas A&amp;M processing. At EcoliWiki, annotations made <br />                                          wiki-based community annotation EcoCyc update added to <br />                                          file, annotations containing qualifiers (mainly contributes_to) yet <br />                                          supported EcoCyc. Only annotations complete GO consortium standards <br />                                          extracted EcoliWiki; incomplete annotations left hope community <br />                                          members eventually complete them. EcoliWiki runs GO consortium validation <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 12 <br />  <br />  <br />                                             scripts deposits file GO consortium Concurrent Versioning <br />                                             System. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               9.3 Genbank <br />                                             The Genbank record U00096, produced Blattner laboratory October 1997, the <br />                                             source original E. coli MG1655 genome sequence annotation incorporated by <br />                                             EcoCyc. A corrected nucleotide sequence deposited GenBank U00096.2 2004, <br />                                             revised sequence incorporated EcoCyc version 8.6 (November 2004). <br />                                             The revised genome annotation published [57] incorporated EcoCyc version <br />                                             10.0 (March 2006). <br />  <br />                               9.4 RefSeq Collaboration <br />                                             EcoCyc involved collaboration update genome annotation GenBank <br />                                             (U00096.2) RefSeq (NC 000913.2) entries E. coli K-12 MG1655 ongoing <br />                                             basis. The primary collaborators include EcoCyc, EcoGene, UniProtKB/Swiss-Prot, and <br />                                             NCBI. The collaborators routinely share data resolve data conflicts. The updates of <br />                                             gene names, gene positions, gene product names shared partners. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               9.5 MetaCyc <br />                                             The EcoCyc MetaCyc databases exchange data release processes both <br />                                             databases. The updates occurred enzymes, genes, pathways, reactions, and <br />                                             metabolites exchanged databases based automated comparisons of <br />                                             update dates ensure latest information corrections propagated the <br />                                             databases. <br />  <br />                               10 EcoCyc Accession Numbers <br />                               10.1 Gene Accession Numbers <br />                                             Three systems accession numbers typically available genes EcoCyc. Any of <br />                                             accession numbers used querying EcoCyc genes &#8220;by name,&#65533;? the <br />                                             website Quick Search. <br />  <br />                                                 &#8226;    EcoCyc ID: The EcoCyc project assigns unique identifiers gene for <br />                                                      historical reasons variable syntax, form &#8220;Gnnnn,&#65533;? &#8220;EGnnnnn,&#65533;? <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      &#8220;G0-nnnnn.&#65533;? EcoCyc IDs stored frame id EcoCyc gene object. <br />  <br />                                                 &#8226;    B-numbers: Originally assigned Blattner laboratory E. coli <br />                                                      genome project, b-number identifiers form &#8220;bnnnn.&#65533;? B-numbers were <br />                                                      originally assigned sequentially genome. When gene object removed <br />                                                      genome decision insufficient evidence existence <br />                                                      gene available, b-number retired reused. When new <br />                                                      genes added genome, assigned highest available b- <br />                                                      number. Thus, b-numbers longer purely sequential genome. B- <br />                                                      numbers stored EcoCyc slot Accession-1. <br />  <br />  <br />  <br />  <br />                                                Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 13 <br />  <br />  <br />                                              &#8226;    ECK numbers: ECK numbers assigned E. coli K-12 MG1655 and <br />                                                   W3110 genomes 2005 attempt provide shared accession numbers for <br />                                                   genes common genomes [57]. ECK numbers stored EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   slot Accession-2. For 18 genes E. coli K-12 MG1655 <br />                                                   genome b-number ECK number number; subsequent genes <br />                                                   numbers diverged. <br />  <br />                               11 Other E. coli <span id='am-85' about='obo:NCBITaxon_620' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-86' property="rdfs:label" content="Shigella" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Shigella</span> PGDBs BioCyc <br />                                          EcoCyc larger BioCyc collection Pathway/Genome Databases (PGDBs). <br />                                          BioCyc version 17.5 (2013) includes 160 E. coli Shigella PGDBs. Most PGDBs <br />                                          generated computationally lack extensive manual literature-based curation of <br />                                          EcoCyc K-12 database. The E. coli genomes BioCyc focused complete <br />                                          genomes, include draft genomes. <br />  <br />                                          Two PGDBs undergone additional curation: BioCyc PGDBs strains E. <br />                                          coli W3110 E. coli B str. REL606. Both strains underwent computational <br />                                          annotation-normalization procedure gene names, product names, heteromultimeric <br />                                          protein complexes, Gene Ontology terms propagated EcoCyc their <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          orthologous genes strains (the orthologs computed SRI as <br />                                          bidirectional best-BLAST hits additional manual review curation). This procedure <br />                                          performed assumption genome-annotation pipelines typically introduce <br />                                          syntactically large semantically insignificant variation naming genes gene <br />                                          products. In addition, E. coli B str. REL606 underwent literature-based curation SRI to <br />                                          incorporate experimental information regarding genes pathways present strain <br />                                          EcoCyc strain MG1655. This curation supported PortEco project. <br />  <br />                                          To select given genome querying BioCyc website, click words &#8220;change <br />                                          organism database&#65533;? Quick Search Gene Search buttons upper right <br />                                          corner EcoCyc webpages. <br />  <br />                               12 We Encourage Your Feedback <br />                                          Feedback scientific community proved invaluable improving EcoCyc during <br />                                          years development. We strongly encourage comments suggestions for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          improvements areas, including: <br />  <br />                                              &#8226;    The database content EcoCyc <br />  <br />                                              &#8226;    The presentation information EcoCyc website <br />  <br />                                              &#8226;    The analysis tools provided conjunction EcoCyc <br />  <br />                                              &#8226;    The performance EcoCyc website <br />  <br />                                          If error omission EcoCyc, report using &#8220;Report Errors or <br />                                          Provide Feedback&#65533;? link data page. Please email suggestions or <br />                                          questions biocyc-support ai dot sri dot com. <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                               Page 14 <br />  <br />  <br />                                          During EcoCyc release email summary new developments biocyc-users <br />                                          mailing list. To subscribe mailing list, http://biocyc.org/subscribe.shtml. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               13 How Learn More <br />                                               &#8226;     How use Pathway Tools website EcoCyc <br />  <br />                                               &#8226;     Downloadable instructional videos use EcoCyc <br />  <br />                                               &#8226;     Pathway/Genome Database Concepts Guide <br />  <br />                                               &#8226;     Publications EcoCyc: [58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70] <br />  <br />                                               &#8226;     BioCyc User&#8217;s Guide <br />  <br />                                               &#8226;     MetaCyc User&#8217;s Guide <br />  <br />                                               &#8226;     Guide Pathway Tools Schema <br />  <br />                                               &#8226;     How download Pathway Tools organism flat-file databases <br />  <br />                               14 How Cite EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Please cite EcoCyc publications benefit use EcoCyc database or <br />                                          website. Please cite EcoCyc recent Nucleic Acids Research Database issue <br />                                          article, currently: <br />  <br />                                          Keseler et al., Nuc Acids Res, 41:D605&#8211;12 2013. <br />  <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                          Monica Riley led curation EcoCyc years, inception. Her efforts created content the <br />                                          organism-scale metabolic database. John Ingraham valued advisor EcoCyc years. We thank <br />                                          scientists contributed corrections suggestions EcoCyc years, thank the <br />                                          scientists served EcoCyc Steering Committee. Many contributors EcoCyc listed the <br />                                          EcoCyc credits page. <br />  <br />                                          The development EcoCyc funded NIH grants GM77678 GM71962 NIH National Institute of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          General Medical Sciences. <br />  <br />  <br />                               References <br />                                          1. Karp PD, Paley SM, Krummenacker M, Latendresse M, Dale JM, Lee T, et al. Pathway Tools <br />                                             version 13.0: Integrated Software Pathway/Genome Informatics Systems Biology. Brief <br />                                             Bioinform. 2010; 11:40&#8211;79. Available from: http://bib.oxfordjournals.org/cgi/content/abstract/ <br />                                             bbp043. 10.1093/bib/bbp043 [PubMed: 19955237] <br />                                          2. Kim KS, Lee S, Ryu CM. Interspecific bacterial sensing airborne signals modulates <br />                                             locomotion drug resistance. Nat Commun. 2013; 4:1809. [PubMed: 23651997] <br />                                          3. Bower JM, Gordon-Raagas HB, Mulvey MA. Conditioning uropathogenic Escherichia coli for <br />                                             enhanced colonization host. Infect Immun. 2009 May; 77(5):2104&#8211;2112. [PubMed: 19255192] <br />                                          4. Rhodius V, Dyk TKV, Gross C, LaRossa RA. Impact genomic technologies studies of <br />                                             bacterial gene expression. Annu Rev Microbiol. 2002; 56:599&#8211;624. [PubMed: 12142487] <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                  Page 15 <br />  <br />  <br />                                          5. Gonzalez R, Tao H, Purvis JE, York SW, Shanmugam KT, Ingram LO. Gene array-based <br />                                             identification changes contribute ethanol tolerance ethanologenic Escherichia coli: <br />                                             Comparison KO11 (parent) LY01 (resistant mutant). Biotechnol Prog. 2003 Mar; 19(2):612&#8211; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             623. [PubMed: 12675606] <br />                                          6. Taoka M, Yamauchi Y, Shinkawa T, Kaji H, Motohashi W, Nakayama H, et al. Only small subset <br />                                             horizontally transferred chromosomal genes Escherichia coli translated proteins. <br />                                             Mol Cell Proteomics. 2004 Aug; 3(8):780&#8211;787. [PubMed: 15115803] <br />                                          7. Ravasz E, Somera AL, Mongru DA, Oltvai ZN, Barabasi AL. Hierarchical organization of <br />                                             modularity metabolic networks. Science. 2002 Aug; 297(5586):1551&#8211;1555. [PubMed: 12202830] <br />                                          8. Simeonidis E, Rison SC, Thornton JM, Bogle ID, Papageorgiou LG. Analysis metabolic <br />                                             networks using pathway distance metric linear programming. Metab Eng. 2003 Jul; 5(3): <br />                                             211&#8211;219. [PubMed: 12948755] <br />                                          9. <span id='am-80' about='obo:NCBITaxon_705219' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Arita" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Arita</span> M. The metabolic world Escherichia coli small. Proc Natl Acad Sci USA. 2004 Feb; <br />                                             101(6):1543&#8211;1547. [PubMed: 14757824] <br />                                          10. Jardine O, Gough J, Chothia C, Teichmann SA. Comparison small molecule metabolic <br />                                               enzymes Escherichia coli <span id='am-23' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Genome Res. 2002 Jun; 12(6):916&#8211; <br />                                               929. [PubMed: 12045145] <br />                                          11. Rison SC, Thornton JM. Pathway evolution, structurally speaking. Curr Opin Struct Biol. 2002 <br />                                               Jun; 12(3):374&#8211;382. [PubMed: 12127458] <br />                                          12. Ma HW, Kumar B, Ditges U, Gunzer F, Buer J, Zeng AP. An extended transcriptional regulatory <br />                                               network Escherichia coli analysis hierarchical structure network motifs. Nuc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Acids Res. 2004; 32(22):6643&#8211;6649. <br />                                          13. Shen-Orr SS, Milo R, Mangan S, Alon U. Network motifs transcriptional regulation network <br />                                               Escherichia coli. Nat Genet. 2002 May; 31(1):64&#8211;68. [PubMed: 11967538] <br />                                          14. Karimpour-Fard A, Leach SM, Gill RT, Hunter LE. Predicting protein linkages bacteria: Which <br />                                               method best depends task. BMC Bioinformatics. 2008; 9:397. [PubMed: 18816389] <br />                                          15. Bowers PM, Pellegrini M, Thompson MJ, Fierro J, Yeates TO, Eisenberg D. Prolinks: A database <br />                                               protein functional linkages derived coevolution. Genome Biol. 2004; 5(5):R35. [PubMed: <br />                                               15128449] <br />                                          16. Price MN, Huang KH, Alm EJ, Arkin AP. A novel method accurate operon predictions all <br />                                               sequenced prokaryotes. Nuc Acids Res. 2005; 33(3):880&#8211;892. <br />                                          17. Steinhauser D, Junker BH, Luedemann A, Selbig J, Kopka J. Hypothesis-driven approach to <br />                                               predict transcriptional units gene expression data. Bioinformatics. 2004 Aug; 20(12):1928&#8211; <br />                                               1939. [PubMed: 15044239] <br />                                          18. Burden S, Lin YX, Zhang R. Improving promoter prediction NNPP2.2 algorithm: A case <br />                                               study using Escherichia coli DNA sequences. Bioinformatics. 2005 Mar; 21(5):601&#8211;607. <br />                                               [PubMed: 15454410] <br />                                          19. Gordon L, Chervonenkis AY, Gammerman AJ, Shahmuradov <span id='am-7' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span>IA</span>, Solovyev VV. Sequence <br />                                               alignment kernel recognition promoter regions. Bioinformatics. 2003 Oct; 19(15):1964&#8211; <br />                                               1971. [PubMed: 14555630] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          20. Fu Y, Jarboe LR, Dickerson JA. Reconstructing genome-wide regulatory network E. coli using <br />                                               transcriptome data predicted transcription factor activities. BMC Bioinformatics. 2011; <br />                                               12:233. [PubMed: 21668997] <br />                                          21. Watanabe RL, Morett E, Vallejo EE. Inferring modules functionally interacting proteins using <br />                                               Bond Energy Algorithm. BMC Bioinformatics. 2008; 9:285. [PubMed: 18559112] <br />                                          22. Muley VY, Ranjan A. Effect reference genome selection performance computational <br />                                               methods genome-wide protein-protein interaction prediction. PLoS One. 2012; 7(7):e42057. <br />                                               [PubMed: 22844541] <br />                                          23. Moreno-Hagelsieb G, Jokic P. The evolutionary dynamics functional modules the <br />                                               extraordinary plasticity regulons: Escherichia coli perspective. Nucleic Acids Res. 2012; <br />                                               40(15):7104&#8211;12. [PubMed: 22618875] <br />                                          24. Kastenmuller G, Schenk ME, Gasteiger J, Mewes HW. Uncovering metabolic pathways relevant to <br />                                               phenotypic traits microbial genomes. Genome Biol. 2009; 10(3):R28. [PubMed: 19284550] <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                 Page 16 <br />  <br />  <br />                                          25. Kumar VS, Maranas CD. GrowMatch: An automated method reconciling silico/in vivo <br />                                              growth predictions. PLoS Comput Biol. 2009 Mar.5(3):e1000308. [PubMed: 19282964] <br />                                          26. Thomas GH, Zucker J, Macdonald SJ, Sorokin A, Goryanin I, Douglas AE. A fragile metabolic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              network adapted cooperation symbiotic bacterium <span id='am-65' about='obo:NCBITaxon_9' typeof='obo:NCBITaxon_32199 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-66' property="rdfs:label" content="Buchnera aphidicola" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28241" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Buchnera aphidicola Munson et al. 1991" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71'  ></span>Buchnera aphidicola</span>. BMC Syst Biol. <br />                                              2009; 3:24. [PubMed: 19232131] <br />                                          27. Frazier ME, Johnson GM, Thomassen DG, Oliver CE, Patrinos A. Realizing potential the <br />                                              genome revolution: The Genomes Life program. Science. 2003 Apr; 300(5617):290&#8211;293. <br />                                              [PubMed: 12690188] <br />                                          28. Bailey JE. Toward Science Metabolic Engineering. Science. 1991 Jun.252:1668&#8211;1675. <br />                                              [PubMed: 2047876] <br />                                          29. Stephanopoulos G, Vallino JJ. Network rigidity metabolic engineering metabolite <br />                                              overproduction. Science. 1991 Jun.252:1675&#8211;1681. [PubMed: 1904627] <br />                                          30. Arense P, Bernal V, Charlier D, Iborra JL, Foulquie-Moreno MR, Canovas M. Metabolic <br />                                              engineering high yielding L(&#8722;)-carnitine production Escherichia coli. Microb Cell Fact. <br />                                              2013; 12(1):56. (ENG). [PubMed: 23718679] <br />                                          31. Jantama K, Zhang X, Moore JC, Shanmugam KT, Svoronos SA, Ingram LO. Eliminating side <br />                                              products increasing succinate yields engineered strains <span id='am-131' about='obo:NCBITaxon_498388' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Escherichia coli C" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli C</span>. Biotechnol <br />                                              Bioeng. 2008 Dec; 101(5):881&#8211;893. [PubMed: 18781696] <br />                                          32. Weber J, Hoffmann F, Rinas U. Metabolic adaptation Escherichia coli temperature- <br />                                              induced recombinant protein production: 2. Redirection metabolic fluxes. Biotechnol Bioeng. <br />                                              2002 Nov; 80(3):320&#8211;330. [PubMed: 12226865] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          33. Update activities Universal Protein Resource (UniProt) 2013. Nuc Acids Res. 2013; <br />                                              41(Database issue):D43&#8211;7. <br />                                          34. Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al. Gene Ontology: Tool for <br />                                              unification biology. Nature Genetics. 2000; 25:25&#8211;29. [PubMed: 10802651] <br />                                          35. Serres MH, Riley M. MultiFun, multifunctional classification scheme Escherichia coli K&#8211;12 <br />                                              gene products. Genome Biol. 2000; 5(4):205&#8211;222. <br />                                          36. Bochner BR, Gadzinski P, Panomitros E. Phenotype Microarrays High-Throughput Phenotypic <br />                                              Testing Assay Gene Function. Genome Res. 2001 Jul; 11(7):1246&#8211;1255. [PubMed: <br />                                              11435407] <br />                                          37. Oun MA, Suthers PF, Jones GI, Carter BR, Saunders MP, Maranas CD, et al. Genome scale <br />                                              reconstruction Salmonella metabolic model: comparison similarity differences a <br />                                              commensal Escherichia coli strain. J Biol Chem. 2009; 284(43):29480&#8211;8. [PubMed: 19690172] <br />                                          38. Baumler DJ, Peplinski RG, Reed JL, Glasner JD, <span id='am-151' about='obo:NCBITaxon_38948' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_301959 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-152' property="rdfs:label" content="Perna" datatype="xsd:string"></span><span id='am-153'  ></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Perna</span> NT. The evolution metabolic networks <br />                                              E. coli. BMC Syst Biol. 2011; 5:182. [PubMed: 22044664] <br />                                          39. Mackie A, Paley S, Keseler IM, Shearer A, Paulsen IT, Karp PD. Addition Escherichia coli K&#8211; <br />                                              12 Growth-Observation Gene Essentiality Data EcoCyc database. J Bacteriol. 2013 <br />                                          40. Yoon SH, Han MJ, Jeong H, Lee CH, Xia XX, Lee DH, et al. Comparative multi-omics systems <br />                                              analysis Escherichia coli strains B K&#8211;12. Genome Biol. 2012; 13(5):R37. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              22632713] <br />                                          41. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD, et al. Experimental <br />                                              determination level analysis essential genes Escherichia coli MG1655. J <br />                                              Bacteriol. 2003 Oct; 185(19):5673&#8211;5684. [PubMed: 13129938] <br />                                          42. Baba T, <span id='am-55' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, Baba M, et al. Construction Escherichia <br />                                              coli K&#8211;12 in-frame, single-gene knockout mutants: The Keio collection. Mol Syst Biol. 2006; <br />                                              2:2006.0008. <br />                                          43. Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, Touda Y, et al. Update the <br />                                              collection Escherichia coli single-gene deletion mutants. Mol Syst Biol. 2009; 5:335. [PubMed: <br />                                              20029369] <br />                                          44. Joyce AR, Reed JL, White A, Edwards R, Osterman A, Baba T, et al. Experimental and <br />                                              computational assessment conditionally essential genes Escherichia coli. J Bacteriol. 2006 <br />                                              Dec; 188(23):8259&#8211;8271. [PubMed: 17012394] <br />                                          45. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, et al. A genome-scale <br />                                              metabolic reconstruction Escherichia coli K&#8211;12 MG1655 accounts 1260 ORFs and <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 17 <br />  <br />  <br />                                              thermodynamic information. Mol Syst Biol. 2007; 3:121&#8211;38. Available from: http:// <br />                                              www.nature.com/doifinder/10.1038/msb4100155. [PubMed: 17593909] <br />                                          46. Patrick WM, Quandt EM, Swartzlander DB, Matsumura I. Multicopy suppression underpins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              metabolic evolvability. Mol Biol Evol. 2007; 24(12):2716&#8211;22. [PubMed: 17884825] <br />                                          47. Orth JD, Thiele I, Palsson BO. What flux balance analysis? Nat Biotechnol. 2010; 28(3):245&#8211;8. <br />                                              [PubMed: 20212490] <br />                                          48. Thiele I, Palsson BO. A protocol generating high-quality genome-scale metabolic <br />                                              reconstruction. Nat Protoc. 2010; 5(1):93&#8211;121. [PubMed: 20057383] <br />                                          49. Reed JL, Vo TD, Schilling CH, Palsson BO. An expanded genome-scale model Escherichia coli <br />                                              K-12 (iJR904 GSM/GPR). Genome Biol. 2003; 4(9):R54. [PubMed: 12952533] <br />                                          50. Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist AM, et al. A comprehensive genome-scale <br />                                              reconstruction Escherichia coli metabolism &#8212; 2011. Mol Syst Biol. 2011; 7:535. [PubMed: <br />                                              21988831] <br />                                          51. Kayser A, Weber J, Hecht V, Rinas U. Metabolic flux analysis Escherichia coli glucose- <br />                                              limited continuous culture. I. Growth-rate-dependent metabolic efficiency steady state. <br />                                              Microbiology. 2005; 151:693&#8211;706. [PubMed: 15758216] <br />                                          52. Ibarra RU, Edwards JS, Palsson BO. Escherichia coli K-12 undergoes adaptive evolution to <br />                                              achieve silico predicted optimal growth. Nature. 2002; 420:186&#8211;189. [PubMed: 12432395] <br />                                          53. Belaich A, Belaich JP. Microcalorimetric study anaerobic growth Escherichia coli: <br />                                              growth thermograms synthetic medium. J Bacteriol. 1976; 72:497&#8211;9. <br />                                          54. Varma A, Boesch BW, Palsson BO. Stoichiometric interpretation Escherichia coli glucose <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              catabolism various oxygenation rates. Applied Environmental Microbiology. 1993; <br />                                              59(8):2465&#8211;73. [PubMed: 8368835] <br />                                          55. Dharmadi Y, Murarka A, Gonzalez R. Anaerobic fermentation glycerol Escherichia coli: <br />                                              anew platform metabolic engineering. Biotechnology Bioengineering. 2006; 94(5):821&#8211;9. <br />                                              [PubMed: 16715533] <br />                                          56. Hu JC, Karp PD, Keseler IM, Krummenacker M, Siegele DA. What learn about <br />                                              Escherichia coli application Gene Ontology. Trends Microbiol. 2009 Jul; 17(7):269&#8211; <br />                                              278. [PubMed: 19576778] <br />                                          57. Riley M, Abe T, Arnaud MB, Berlyn MK, Blattner FR, Chaudhuri RR, et al. Escherichia coli <br />                                              K-12: A cooperatively developed annotation snapshot&#8211;2005. Nuc Acids Res. 2006; 34(1):1&#8211;9. <br />                                          58. Keseler IM, Collado-Vides J, Santos-Zavaleta A, Peralta-Gil M, Gama-Castro S, Muniz-Rascado <br />                                              L, et al. EcoCyc: A Comprehensive Database Escherichia coli Biology. Nuc Acids Res. 2011; <br />                                              39:D583&#8211;90. <br />                                          59. Keseler IM, Bonavides-Martinez C, Collado-Vides J, Gama-Castro S, Gunsalus RP, Johnson DA, <br />                                              et al. EcoCyc: A comprehensive view E. coli biology. Nuc Acids Res. 2009; 37:D464&#8211;70. <br />                                              Available from: http://nar.oxfordjournals.org/cgi/reprint/gkn751? <br />                                              ijkey=7epgizfnGFYQHCe&amp;keytype=ref. <br />                                          60. Karp PD, Keseler IM, Shearer A, Latendresse M, Krummenacker M, Paley SM, et al. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Multidimensional annotation Escherichia coli K-12 genome. Nuc Acids Res. 2007; <br />                                              35:7577&#8211;90. Available from: http://nar.oxfordjournals.org/cgi/content/full/35/22/7577. <br />                                          61. Keseler IM, Collado-Vides J, Gama-Castro S, Ingraham J, Paley S, Paulsen IT, et al. EcoCyc: A <br />                                              comprehensive database resource E. coli. Nuc Acids Res. 2005; 33:D334&#8211;7. Available from: <br />                                              http://nar.oupjournals.org/cgi/content/full/33/suppl_1/D334? <br />                                              ijkey=80p4BbGpEFjLQ&amp;keytype=ref. <br />                                          62. Karp PD, Arnaud M, Collado-Vides J, Ingraham J, Paulsen IT, Saier MHJ. The E. coli EcoCyc <br />                                              Database: No Longer Just Metabolic Pathway Database. ASM News. 2004; 70(1):25&#8211;30. <br />                                          63. Karp PD, Riley M, Saier M, Paulsen IT, Paley S, Pellegrini-Toole A. The EcoCyc Database. Nuc <br />                                              Acids Res. 2002; 30(1):56&#8211;8. <br />                                          64. Karp PD, Riley M, Saier M, Paulsen IT, Paley S, Pellegrini-Toole A. The EcoCyc MetaCyc <br />                                              Databases. Nuc Acids Res. 2000; 28(1):56&#8211;59. <br />                                          65. Karp, PD. Nucleic Acid Protein Databases How To Use Them. London: Academic Press; <br />                                              1999. Using EcoCyc Database; p. 269-280. <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 18 <br />  <br />  <br />                                          66. Karp, PD.; Riley, M. Bioinformatics Databases Systems. Norwell, MA: Kluwer Academic <br />                                              Publishers; 1999. EcoCyc: The resource lessons learned; p. 47-62. <br />                                          67. Karp P, Riley M, Paley S, Pellegrini-Toole A, Krummenacker M. EcoCyc: Electronic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1999; 27(1):55&#8211;58. <br />                                          68. Karp P, Riley M, Paley S, Pellegrini-Toole A, Krummenacker M. EcoCyc: Electronic <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1998; 26(1):50&#8211;53. <br />                                          69. Karp P, Riley M, Paley S, Pellegrini-Toole A, Krummenacker M. EcoCyc: Electronic <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1997; 25(1):43&#8211;50. <br />                                          70. Karp P, Riley M, Paley S, Pellegrini-Toole A. EcoCyc: Electronic Encyclopedia E. coli Genes <br />                                              Metabolism. Nuc Acids Res. 1996; 24(1):32&#8211;40. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 19 <br />  <br />  <br />  <br />  <br />                                                                                         Table 1 <br />  <br />                            Genes gene products EcoCyc. Protein features annotations protein sites regions as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            enzyme active sites, metal ion binding sites, transmembrane domains. A small number IS elements are <br />                            included count Genes included sub-categories genes. <br />  <br />                             Genes/Proteins <br />  <br />                             Genes                                    4501 <br />                                Protein-Coding Genes                  4284 <br />                                tRNA Genes                               86 <br />                                rRNA Genes                               22 <br />                                Regulatory RNA Genes                     41 <br />                                Other RNA Genes                          56 <br />                             Pseudogenes                                133 <br />                             Polypeptides                             4393 <br />                             Protein Complexes                          995 <br />                                Heteromultimeric Protein Complexes      290 <br />                                Homomultimeric Protein Complexes        705 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Protein Features                        23,114 <br />                             Enzymes (excluding transporters)         1245 <br />                             Transporters                               267 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                         Page 20 <br />  <br />  <br />  <br />  <br />                                                                                         Table 2 <br />  <br />                            Gene annotation status EcoCyc. Genes known molecular function experimental evidence their <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            assigned function, genes predicted molecular function function predicted <br />                            computationally. <br />  <br />                             Gene Annotation Status <br />  <br />                             Genes known predicted molecular function   3127 <br />                                Genes known molecular function             2710 <br />                                Genes predicted molecular function          417 <br />                             Genes unknown molecular function              1374 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                        Page 21 <br />  <br />  <br />  <br />  <br />                                                                                                Table 3 <br />  <br />                            Reactions, compounds pathways EcoCyc. Superpathways connected sets base metabolic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            pathways (connected shared substrates.) <br />  <br />                             Reactions/Metabolites/Pathways <br />  <br />                             Metabolic Reactions                                                1443 <br />                             Transport (including Electron Transfer) Reactions                   379 <br />                             Pathways                                                            401 <br />                                Small-molecule metabolism base pathways                          291 <br />                                Signaling pathways                                                29 <br />                                Superpathways                                                     81 <br />                             Metabolites                                                        2466 <br />                                Metabolites substrates enzyme-catalyzed reactions   1331 <br />                                Metabolites physiological enzyme regulators             121 <br />                                Metabolites cofactors prosthetic groups               56 <br />                                Transported metabolites                                          274 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                          Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 22 <br />  <br />  <br />  <br />  <br />                                                                                             Table 4 <br />  <br />                            Regulation-related objects interactions EcoCyc. Each member &#8220;Instances Regulation of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Transcription Initiation&#65533;? describes single regulatory interaction transcription factor binding <br />                            site. <br />  <br />                             Transcriptional/Translational Regulation <br />  <br />                             Transcription Units                                      4510 <br />                             Promoters                                                3777 <br />                             Terminators                                               259 <br />                             Transcription Factors                                     194 <br />                             Transcription Factor Binding Sites                       2773 <br />                             Instances Regulation Transcription Initiation      3293 <br />                             Instances Regulation Transcriptional Attenuation     20 <br />                             Instances Regulation Translation                    146 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 23 <br />  <br />  <br />  <br />  <br />                                                                                      Table 5 <br />  <br />                            Comparison experimental aerobic glucose-limited chemostat growth data EcoCyc iJO1366 FBA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            model predictions (389 reactions active EcoCyc). Metabolite uptake production rates units mmol/ <br />                            gCDW/hr. Growth units hr&#8722;1. Experimental data Kayser et al. [51]. <br />  <br />                                                Experimental   EcoCyc <br />                             Glucose uptake        3.008       3.008 <br />  <br />                             Growth rate           0.300       0.276 <br />  <br />                             O2 uptake             7.413       4.472 <br />  <br />                             NH4 uptake            2.367       3.026 <br />  <br />                             Sulfate uptake                    0.068 <br />  <br />                             Phosphate uptake                  0.288 <br />  <br />                             CO2 production         7.38       5.480 <br />  <br />                             H2O production                    13.026 <br />  <br />                             H+ production                     2.582 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 24 <br />  <br />  <br />  <br />  <br />                                                                                      Table 6 <br />  <br />                            EcoCyc FBA model performance aerobic glycerol-limited growth (385 reactions active Eco-Cyc). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Uptake production rates units mmol/gCDW/hr. Growth units hr&#8722;1. Experimental OUR/G1-UR <br />                            estimated graphically Ibarra et al. [52]. <br />  <br />                                                Experimental   EcoCyc <br />                             Glycerol uptake        10         10.00 <br />  <br />                             Growth rate                        0.53 <br />  <br />                             O2 uptake              11          8.96 <br />  <br />                             NH4 uptake                         5.80 <br />  <br />                             Sulfate uptake                     0.13 <br />  <br />                             Phosphate uptake                   0.55 <br />  <br />                             CO2 production                     5.74 <br />  <br />                             H2O production                    30.36 <br />  <br />                             H+ production                      4.95 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                         Page 25 <br />  <br />  <br />  <br />  <br />                                                                                       Table 7 <br />  <br />                            EcoCyc FBA model performance anaerobic glucose-limited growth (383 reactions active EcoCyc). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Uptake production rates units mmol/gCDW/hr. Growth units hr&#8722;1. Experimental data from <br />                            Belaich Belaich [53] Varma et al. [54]. FHL set inactive purposes comparison. <br />  <br />                                                    Experimental   EcoCyc <br />                             Glucose uptake             10.0       10.00 <br />  <br />                             Growth rate                0.30        0.25 <br />  <br />                             O2 uptake                  0.00        0.00 <br />  <br />                             NH4 uptake                             2.76 <br />  <br />                             Sulfate uptake                         0.06 <br />  <br />                             Phosphate uptake                       0.26 <br />  <br />                             CO2 production                         0.27 <br />  <br />                             H2O production                         0.57 <br />  <br />                             H+ production                         27.14 <br />  <br />                             Acetate production         7.5         8.03 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Formate production         11.3       16.76 <br />  <br />                             Succinate production       1.2         0.00 <br />  <br />                             Ethanol production         8.7      <br /> </body></html>