<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Lee et al. Microbial Cell Factories 2014, 13:61 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                     Open Access <br />  <br /> Genome-scale metabolic network reconstruction <br /> and silico flux analysis thermophilic <br /> bacterium <span id='am-307' about='obo:NCBITaxon_274' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-308' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-309' property="rdfs:label" content="Thermus thermophilus" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasDbXref" content="PMID:8590676" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasAlternativeId" content="NCBITaxon:273" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasRelatedSynonym" content="Thermua flavus" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus subsp. thermophilus" datatype="xsd:string"></span><span id='am-315'  ></span><span id='am-316' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. FLAVUS)" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus flavus" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:hasRelatedSynonym" content="Thermus flavus" datatype="xsd:string"></span><span id='am-319' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. THERMOPHILUS)" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasDbXref" content="PMID:4334342" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus thermophilus" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasRelatedSynonym" content="Thermus themophilus" datatype="xsd:string"></span>Thermus thermophilus</span> HB27 <br /> Na-Rae Lee1&#8224;, Meiyappan Lakshmanan2&#8224;, Shilpi Aggarwal2, Ji-Won Song1, Iftekhar A Karimi2, Dong-Yup Lee2,3* <br /> and Jin-Byung Park1,4* <br />  <br />  <br />   Abstract <br />   Background: <span id='am-324' about='obo:NCBITaxon_270' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_1297'><span id='am-325' property="oboInOwl:hasDbXref" content="PMID:8240955" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasDbXref" content="PMID:5781580" datatype="xsd:string"></span><span id='am-328'  ></span><span id='am-329' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-330' property="rdfs:label" content="Thermus" datatype="xsd:string"></span>Thermus</span> thermophilus, extremely thermophilic bacterium, widely recognized model <br />   organism studying microbes survive adapt high temperature environment. However, the <br />   thermotolerant mechanisms cellular metabolism remains unravelled. Thus, highly required to <br />   consider systems biological approaches T. thermophilus metabolic network model employed together <br />   high throughput experimental <span id='am-465' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-466'  ></span><span id='am-467' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-468' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> elucidating physiological characteristics harsh <br />   conditions. <br />   Results: We reconstructed genome-scale metabolic model T. thermophilus, iTT548, large-scale <br />   network thermophilic bacterium, accounting 548 unique genes, 796 reactions 635 unique metabolites. <br />   Our initial comparative analysis model <span id='am-370' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-371' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-372'  ></span><span id='am-373' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-374' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-377' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-381' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-382' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-383' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-384' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> revealed distinctive metabolic reactions, <br />   mainly amino acid metabolism carotenoid biosynthesis, producing relevant compounds retain cellular <br />   membrane withstanding high temperature. Constraints-based flux analysis was, then, applied simulate the <br />   metabolic state glucose minimal amino acid rich media. Remarkably, resulting growth predictions were <br />   highly consistent experimental observations. The subsequent comparative flux analysis different <br />   environmental conditions highlighted cells consumed branched chain amino acids preferably utilized <br />   directly relevant anabolic pathways fatty acid synthesis. Finally, gene essentiality study also <br />   conducted single gene deletion analysis, identify conditional essential genes glucose minimal and <br />   complex media. <br />   Conclusions: The reconstructed genome-scale metabolic model elucidates phenotypes T. thermophilus, thus <br />   allowing gain valuable insights cellular metabolism silico simulations. The information obtained <br />   analysis shed light understanding physiology thermophiles helps to <br />   devise metabolic engineering strategies develop T. thermophilus thermostable microbial cell factory. <br />   Keywords: Thermus thermophilus, Thermophile, Genome-scale metabolic model, Constraints-based flux analysis, Ethanol <br />  <br />  <br />  <br />  <br /> * Correspondence: cheld@nus.edu.sg; jbpark06@ewha.ac.kr <br /> &#8224; <br />  Equal contributors <br /> 2 <br />  Department Chemical Biomolecular Engineering, National University <br /> of Singapore, 4 Engineering Drive 4, Singapore 117585, Singapore <br /> 1 <br />  Department Food Science &amp; Engineering, Ewha Womans University, 11-1 <br /> Daehyun-dong, Seodaemun-gu, Seoul 120-750, Korea <br /> Full list author information available end article <br />  <br />                                        &#169; 2014 Lee et al.; licensee BioMed Central Ltd. <span id='am-277' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-278' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-279'  ></span><span id='am-280' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                        Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                        unless stated. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                  Page 2 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Background                                                      utilized postulate various strain improvement strategies <br /> Thermus thermophilus gram-negative, obligate aerobic       [17-19,24,25]. Thus, development T. thermophilus <br /> bacterium, representing best-studied ther-           GSMM based currently available biochemical and <br /> mophiles. It usually colonizes terrestrial volcanic hot     genomic information subsequent silico analysis <br /> springs (grows optimally 65 72&#176;C)           enables elucidate unique metabolic behaviour. <br /> originally isolated Japanese thermal spa [1]. In           In thermophilic microbes regard, a <br /> addition ability surviving high tempera-      initial attempts model cellular metabolisms. <br /> tures, T. thermophilus resistant stress     First, silico model <span id='am-174' about='obo:NCBITaxon_2335' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_200918 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_188708 obo:NCBITaxon_188709 obo:NCBITaxon_2419'><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-178' property="rdfs:label" content="Thermotoga" datatype="xsd:string"></span>Thermotoga</span> maritima pre- <br /> harsh chemical conditions [2]. These properties motivated       sented, covering central metabolism 3D <br /> researchers extract isolate numerous proteins        structures enzymes accounted network <br /> T. thermophilus, making model organism struc-        [26]. Recently, genome-scale metabolic model of <br /> tural genomics significant industrial potential [3-6].     thermophilic archeon, <span id='am-185' about='obo:NCBITaxon_2284' typeof='owl:Thing obo:NCBITaxon_118883 obo:NCBITaxon_1 obo:NCBITaxon_28889 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183924 obo:NCBITaxon_2281'><span id='am-186' property="rdfs:label" content="Sulfolobus" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:4559703" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-190'  ></span><span id='am-191' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Sulfolobus</span> solfataricus, also <br /> For example, thermostable proteins used     developed, used autotrophic growth <br /> in commercial processes, including DNA polymerase        bicarbonate hydroxypropionate-hydroxybutyrate <br /> PCR techniques, &#945;-amylases glucose isomerases            cycle aerobic conditions [27]. However, models <br /> starch processing, esterases, lipases proteases          mature explain molecular mecha- <br /> organic synthesis, xylanases paper pulp              nisms high temperature adaptations con- <br /> manufacturing [7,8]. Moreover, T. thermophilus         sider detailed biosynthetic machinery biomolecules <br /> recognized potential microbial cell factory low    help retain integrity cell wall <br /> cost ethanol fermentation lignocellulosic waste ma-        membranes. Therefore, work, reconstructed the <br /> terials grow utilizing C5/C6        genome-scale metabolic model T. thermophilus based <br /> carbon sources relatively high temperatures, i.e. 70&#8211;80&#176;     genome annotation HB27 wild-type strain [28] <br /> C, reducing energy costs: cooling step re-       investigating unique characteristics thermophilic <br /> quired following enzymatic hydrolysis, rendering easier      microbes. Additionally, model functionally char- <br /> to distil subsequent fermentations [9].                         acterized gene essentiality studies identify essential <br />    Despite enormous potentials biotechnological ap-         genes cellular growth growing glucose <br /> plications, current knowledge unique cellular        minimal amino acid supplemented complex media. <br /> physiology T. thermophilus limited; date, <br /> the production distinctive carotenoid molecules [10]         Results <br /> and use adaptive protein synthesis strategies [11]       Reconstruction T. thermophilus genome-scale <br /> are notable traits unravelled molecular         metabolic network <br /> level. Such limited studies technical     The genome-scale metabolic network T. thermophilus <br /> difficulties cultivating analysing thermophilic mi-      HB27 reconstructed step procedure: <br /> crobes; cell culture experiments require high         (1) construction draft network compilation genes, <br /> energy maintain optimal growth conditions. Hence,        reactions pathway information biochemical data- <br /> it required develop systematic approaches     bases based genome annotation T. thermophilus <br /> for improving understanding T. thermophilus cellu-       HB27, (2) manual curation draft model verifying <br /> lar behaviour. In regard, constraints-based silico      elemental balances reactions assigning proper <br /> metabolic modeling analysis considered        gene-reaction relationships, (3) gap filling using <br /> of promising techniques characterize physio-         organism specific knowledge (see Methods). During the <br /> logical behaviour metabolic states organism           reconstruction process, significant efforts highly re- <br /> upon various environmental/genetic changes sys-         quired identify resolve network gaps across <br /> tematically capture genotype-phenotype relationships        various metabolic pathways. Such gaps exist the <br /> from entire genome information [12,13]. As result,        incomplete genome annotations result missing <br /> several genome-scale metabolic models (GSMMs)           biochemical reactions dead ends. These gaps be <br /> available describing metabolic organization vari-    appropriately filled addition new reactions based <br /> ous organisms including <span id='am-386' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-387' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-388' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-389'  ></span><span id='am-390' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli [14], Bacillus         information obtained literature inferred by <br /> subtilis [15], <span id='am-250' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-251' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-261'  ></span><span id='am-262' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> [16], <span id='am-243' about='obo:NCBITaxon_4919' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_1156497 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-244' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasRelatedSynonym" content="Hansenula" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-247'  ></span><span id='am-248' property="oboInOwl:hasAlternativeId" content="NCBITaxon:34359" datatype="xsd:string"></span><span id='am-249' property="rdfs:label" content="Pichia" datatype="xsd:string"></span>Pichia</span> pastoris   genome annotation organisms. For example, <br /> [17], <span id='am-118' about='obo:NCBITaxon_1718' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1716 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="&apos;Corynebacterium lactofermentum&apos;" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium lactofermentum" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lilium" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="PMID:1713055" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Corymebacterium glutamicum" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium seonmiso" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="PMID:11988527" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium taipei" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153277" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153278" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium saccharolyticum" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Micrococcus maripuniceus" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153271" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium chang-fua" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="&apos;Brevibacterium lactofermentum&apos;" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium glutamicum (Kinoshita et al. 1958) Abe et al. 1967" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153272" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lilium Lee and Good 1963 (Approved Lists 1980)" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium glutamigenes" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium divaricatum" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37715" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:10220171" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium thiogenitalis" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153274" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153234" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="&quot;Micrococcus glutamicus&quot; Kinoshita et al. 1958" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lactofermentum" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="Corynebacterium glutamicum" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasRelatedSynonym" content="Cornyebacterium lactofermentum" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153273" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="Micrococcus glutamicus" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153276" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasAlternativeId" content="NCBITaxon:92707" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium divaricatum Su and Yamada 1960 (Approved Lists 1980)" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasAlternativeId" content="NCBITaxon:70020" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153275" datatype="xsd:string"></span>Corynebacterium glutamicum</span> [18], <span id='am-406' about='obo:NCBITaxon_48736' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_119060 obo:NCBITaxon_2'><span id='am-407' property="oboInOwl:hasRelatedSynonym" content="Ralstonia Yabuuchi et al. 1996" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasDbXref" content="PMID:8657018" datatype="xsd:string"></span><span id='am-411'  ></span><span id='am-412' property="rdfs:label" content="Ralstonia" datatype="xsd:string"></span>Ralstonia</span> eutro-         initial model contained metabolic gaps the <br /> pha [19], <span id='am-331' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-332' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-340'  ></span><span id='am-341' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa [20],             synthetic pathway thermozeaxanthin thermobis- <br /> multicellular eukaryotes <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-10'  ></span>Mus musculus</span> [21]          zeaxanthin, unique type carotenoids found <br /> <span id='am-282' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-283'  ></span><span id='am-284' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Homo sapiens</span> [22]. Moreover, availability           thermophiles, enabling cellular membrane to <br /> several conveniently accessible constraints-based model-        retain fluidity high temperatures [29-31]. <br /> ing software tools [23], models largely         Therefore, <span id='am-294' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-295' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-296' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> gaps, added reactions <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                        Page 3 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> corresponding glycosyltransferase acyltransferase en- <br /> zymes <span id='am-29' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Staphylococcus</span> aureus subsp. aureus 11819&#8211;97 <br /> and <span id='am-111' about='obo:NCBITaxon_1570' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_45667'><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Sporosarcina halophila" datatype="xsd:string"></span><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:519939" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Halobacillus halophilus" datatype="xsd:string"></span>Halobacillus halophilus</span> DSM 2266 sequential <br /> glycosylation esterification zeaxanthin glucose <br /> and branched-chain fatty acids, producing thermozeax- <br /> anthin. Similarly, draft model gaps in <br /> the carbon assimilatory pathways, unable to <br /> consume carbon sources, namely, trehalose, palatinose, <br /> isomaltose, cellobiose, glutamate mannose. However, <br /> earlier studies reported T. thermophilus grow <br /> on carbon sources [32,33]. Such discrepancies <br /> were resolved adding new reactions corresponding <br /> to &#945;-glucosidase mannokinase enzymes based <br /> on information available KEGG MetaCyc <br /> databases. Overall, included 74 new reactions 63 <br /> enzymes, improving network connectivity <br /> (see Additional file 1 complete list reactions added). <br /> The gap filling draft model followed identifi-      Figure 1 Distribution reactions genes various <br /> cation genetic evidence newly added enzymes            metabolic subsystems iTT548. <br /> via sequence-based homology searches. For purpose, <br /> a BLASTp search performed NCBI database for <br /> the enzymes resolve network gaps using            comparison, 363 enzymes/genes common <br /> their amino acid sequences collected various          E. coli T. thermophilus, belonging to <br /> organisms non-redundant protein sequences            central metabolic pathways glycolysis, pentose <br /> of T. thermophilus HB27 genome. In way, 63         phosphate pathway TCA cycle. However, certain <br /> enzymes added, assign putative locus 10           notable differences observed amino acid <br /> of them, providing possible new annotations (see            synthetic pathways T. thermophilus. For instance, lysine <br /> Additional file 1 list new annotations). The          synthesized alpha-aminoadipate pathway instead <br /> final genome-scale metabolic network T. thermophilus,         diaminopimelate pathway. Similarly, upstream of <br /> iTT548, contains 548 unique genes (ORF coverage &#8211;                methionine synthetic pathway conserved E. <br /> 24%), 796 reactions 635 unique metabolites. In               coli: precursor molecule, homocysteine, produced <br /> iTT548, 796 reactions classified seven         O-acetyl-L-homoserine hydrogen sulphide via <br /> major metabolic subsystems: carbohydrates, amino acid,           O-acetyl-L-(<span id='am-289' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-290' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-292'  ></span><span id='am-293' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>) serine sulfhydrylase T. thermophilus <br /> energy cofactors, lipids, nucleotides, carotenoids       O-succinyl-L-homoserine cysteine through <br /> transport. Among them, amino acid metabolism largest         O-succinyl-homoserine lyase E. coli. The comparison <br /> number reactions genes, followed carbohydrates         transport reactions revealed T. <br /> and energy cofactors metabolism (Figure 1). The de-          thermophilus lacks phosphoenolpyruvate-dependent <br /> tailed list completely curated T. thermophilus HB27           phosphotransferase (PTS), typical bacterial trans- <br /> metabolic network containing various genes, reactions,       port system, consumes carbohy- <br /> and metabolites obtained Additional file 1,          drates including glucose ATP-binding cassette (ABC) <br /> and available Systems Biology Markup Language            transporters. <br /> (SBML) file (level 2, version 1, http://sbml.org/) (Additional      Interestingly, unique feature T. thermophilus, <br /> file 2).                                                         iTT548 contains necessary biosynthetic machinery <br />                                                                  synthesizing molecules help in <br /> Network characteristics iTT548 comparison        habituating high temperatures. Unlike gram- <br /> E. coli GSMM                                                     negative bacteria, T. thermophilus does contain lipo- <br /> Figure 2A presents overall network features iTT548        polysaccharides cell outer membrane [34]. Instead, it <br /> and comparison E. coli using metabolic model        embeds complex carotenoid glucoside esters various <br /> (iAF1260) [14] terms EC numbers. Here,              branched chain fatty acids, known thermozeaxanthins <br /> should noted unable fairly compare        thermobiszeaxanthins, lipid bilayers. Such an <br /> metabolic characteristics T. thermophilus          arrangement offers multiple advantages including the <br /> thermophiles T. maritima model limited         retention membrane fluidity high temperatures <br /> to central metabolism S. solfataricus GSMM             reduction oxygen diffusion membrane <br /> was accessible link provided. From            preventing oxidation damage [10,35]. Furthermore, <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                       Page 4 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  Figure 2 Metabolic organization biomass composition T. thermophilus E. coli. (A) General features iTT548 comparison <br />  E. coli iAF1260 GSMM (Feist et al. 2007), (B) Central metabolic network T. thermophilus, (C) amino acid composition (mol%) (D) fatty <br />  acid composition (mol%). The numbers Venn diagram represents enzymes organism. The common unique pathways T. <br />  thermophilus highlighted blue red backgrounds, respectively. The number unique common enzymes identified using the <br />  EC numbers. The biomass data E. coli obtained iAF1260 GSMM. See supplementary 1 metabolite enzyme abbreviations used <br />  network diagram. <br />  <br />  <br /> T. thermophilus synthesizes unique polyamines                      branched chain fatty acids amino acids as <br /> such thermine, spermine, thermospermine caldo-                      valine, leucine isoleucine ketoisovalerate oxi- <br /> pentamine using distinct pathway L-arginine                     doreductase (vorA) gram-positive bacteria as <br /> aminopropyl agmatine [36]. These polyamines essen-                     Bacillus. Table 1 list unique carotenoid poly- <br /> tial high temperature protein synthesis ensuring                amine molecules, relevant genes corresponding <br /> proper structure formation initiation complex                       synthetic pathways accounted iTT548; Figure 2 illus- <br /> among 30S ribosomal subunit, messenger,                        trates central metabolic network T. thermophilus <br /> initial aminoacyl-tRNA. As notable exception gram-                    branched chain fatty acids carotenoid syn- <br /> negative bacterium, T. thermophilus synthesizes                       thetic pathways highlighted. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                Page 5 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Table 1 Biosynthetic machinery unique molecules T. thermophilus <br /> Molecules                                                 No.    Genes involved                              Major metabolic <br />                                                           reactions                                             precursors <br /> Branched fatty acids                                      19       ilvE (TTC1870), vorA (TTC1756) vorB      Leucine, valine, isoleucine <br />                                                                    (TTC1757), FabF (TTC0049 TTC0045), FabG   acetyl-coA <br /> (i-15:0, ai-15:0, i-16:0, i-17:0, ai-17:1)                         (TTC0047 TTC0394), FabZ (TTC1463), FabI <br />                                                                    (TTC0343) <br /> Carotenoids                                               18       CrtE (TTC1986), CrtB (TT_P0057), CrtI         Isopentenyl diphosphate <br />                                                                    (TT_P0066), CrtY (TT_P0060), CrtZ (TT_P0059), (IPP), UDP-glucose and <br /> (thermocryptoxanthin, thermozeaxanthin and <br />                                                                    CruC (TT_P0062), CruD (TT_P0061)              branched fatty acids <br /> thermobiszeaxanthin) <br /> Polyamines                                                16       <span id='am-475' about='obo:NCBITaxon_30316' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_7711 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_8431 obo:NCBITaxon_8432 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_41666 obo:NCBITaxon_8292 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-476'  ></span><span id='am-477' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-478' property="rdfs:label" content="Spea" datatype="xsd:string"></span><span id='am-479' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>SpeA</span> (TTC1277), SpeB (TTC0764 TTC1132), Putrescine, spermidine and <br />                                                                    SpeD1 (TTC0473), SpeD2 (TTC1093), SpeE     spermine <br /> (1,3-diaminopropane, norspermidine, sym-homospermidine,            (TTC0472) <br /> thermine, thermospermine, homospermine, caldopentamine, <br /> thermopentamine, homocaldopentamine, caldohexamine, <br /> homocaldohexamine, tris(3-aminopropyl)amine, <br /> tetrakis(3-aminopropyl)ammonium) <br />  <br />  <br />   In iTT548, included biomass equation               70&#176;C. It noted medium supple- <br /> based amino acid compositional analysis              mented 20 amino acids. The nutrient consump- <br /> data obtained literature. Importantly, biomass            tion profiles indicated glucose consumed first. <br /> composition carefully formulated avoid               Subsequently, carbohydrates trehalose and <br /> erroneous conclusions flux balance analysis                amino acids assimilated (Figure 3B). Notably, cells <br /> [37]. It noted earlier thermophile               did consume amino acid supplemented the <br /> models, T. maritima S. solfataricus, adopted bio-           medium preferred branched chain amino acids ahead <br /> mass equation E. coli M. bakeri, respectively,             amino acids, possibly synthesize branched <br /> both grow 37&#176;C optimal grow-             chain fatty acids (Figures 3C D). Furthermore, the <br /> ing temperature range thermophiles (50 ~ 80&#176;C). In               complex medium did appreciable lag phase <br /> this regard, compositions amino acids, val-             cells grew twice fast minimal <br /> ine, lysine, threonine, lysine, glutamine isoleucine,           media. <br /> are distinctive T. thermophilus E. coli              To simulate cellular growth minimal medium, <br /> (Figure 2C). Similarly, fatty acid compositions T.           biomass equation maximized flux analysis <br /> thermophilus different E. coli; linear           simulations simultaneously constraining glucose <br /> chain fatty acids composition negligible              uptake rates measured exponential phase based <br /> T. thermophilus branched-chain fatty acids                assumption wild-type organisms typically <br /> such <span id='am-452' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-453' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-454' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-455'  ></span><span id='am-456' property="rdfs:label" content="Iso" datatype="xsd:string"></span>iso</span>-C17:0 iso-C15:0, present              evolve maximization cellular growth during <br /> in E. coli, contribute bulk total lipid compositions         exponential phase [38]. The exchange fluxes NH3, <br /> (Figure 2D). Collectively, results highlight need         phosphate, sulphite, H2O, Fe2+, Mg H+ left <br /> for careful estimation biomass equation mod-           unconstrained provide basic nutrients minerals for <br /> eling thermophiles.                                                 cell growth. The oxygen uptake rate constrained at <br />                                                                     average specific uptake rate 10 mmol g&#8722;1 DCW <br /> Model validation using minimal complex media                    hr&#8722;1 based previous publication [39]. Additionally, <br /> during batch cultures                                               lactate exchange flux constrained the <br /> We validated iTT548 using data batch cultures              measured uptake/secretion rates phase. A growth <br /> of T. thermophilus growing glucose minimal                   associated maintenance (GAM) value 58.34 mmol g&#8722;1 <br /> complex media. In case glucose minimal medium,                   DCW h&#8722;1, E. coli GSMM, NGAM requirement <br /> cells cultured DMM containing 0.6% (w/v) glu-             14 mmol g&#8722;1 DCW h&#8722;1, calculated based established <br /> cose 70&#176;C. The residual concentration glucose             methods [14], used simulations (see <br /> the cell density monitored (Figure 3A). Initially,           Additional file 3 detailed calculation NGAM calcu- <br /> prolonged lag phase 2 d observed, followed             lations). Here, noted comparing <br /> exponential growth phase 8 h stationary               NGAM requirement T. thermophilus E. coli <br /> phase reached. The cell cultures indicated             [14], possess high value (14 compared of <br /> presence acetate, lactate ethanol trace amounts           8 mmol g&#8722;1 DCW h&#8722;1) possibly differences in <br /> during lag exponential growth phases. In order               H+ permeability cytoplasmic membrane optimal <br /> analyse growth behaviour T. thermophilus rich           growth temperature; T. thermophilus cytoplasmic mem- <br /> medium, cells grown complex TM medium                 brane highly permeable H+, thus, leaks protons <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                    Page 6 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  Figure 3 Batch fermentation profile optical density various nutrients glucose minimal complex medium. (A) Profiles of <br />  optical density residual concentrations glucose, acetate lactate glucose minimal medium, (B) optical density residual <br />  concentrations glucose, trehalose, lactate acetate complex medium, (C) concentrations amino acids rapidly consumed in <br />  complex media (D) amino acids completely consumed. Highlighted regions correspond exponential growth phases of <br />  cultures corresponding nutrient consumption/secretion profiles used silico simulations. <br />  <br />  <br />  <br /> without ATP synthesis ATPase [40]. Notably, simula- <br /> tion results highly consistent observed growth <br /> rates (Figure 4). For complex media, the <br /> biomass equation maximized simultaneously <br /> constraining uptake/secretion rates nutrients <br /> (glucose, trehalose, lactate, acetate amino acids) <br /> during exponential growth phase. The silico pre- <br /> dicted cell growth 0.66 h&#8722;1 close (within the <br /> acceptable error range 10%) experimentally <br /> observed specific growth rate 0.64 h&#8722;1, clearly <br /> indicating high predictive ability iTT548 in <br /> complex media. <br />  <br /> In silico comparative metabolic flux analysis minimal <br /> and complex media <br /> When comparing cellular growth rates T. thermophi-                  Figure 4 Comparison silico growth rate experimentally <br />                                                                            observed growth rate exponential phase cell culture <br /> lus minimal complex media, surprisingly, was <br />                                                                            glucose minimal medium. <br /> significantly higher owing availability of <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                    Page 7 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> rich carbon nitrogen sources form various           media, (ii) conditionally essential &#8211; required one <br /> amino acids. Since microbe            media (iii) non-essential &#8211; dispensable both <br /> amino acids directly medium, possible        media. The gene essentiality study revealed 23.5% <br /> that protein biosynthetic demand partially ful-        19.5% total 548 genes iTT548 essential <br /> filled. In order confirm hypothesis, compared          cell growth minimal complex media, respect- <br /> the simulated metabolic fluxes minimal (phase P1)         ively (see Additional file 4 complete list essential <br /> and complex media, observed lower fluxes                 genes). A total 107 genes essential condi- <br /> through relevant biosynthetic reactions amino acids        tions additional 21 genes essential in <br /> in complex medium (Table 2). Herein,             minimal media. In order understand knock- <br /> noted conducted flux variability analysis        functional category genes crucial <br /> [41] re-assure confidence simulated metabolic       cell viability condition, identified the <br /> states environmental condition.                           distribution essential genes various metabolic <br />    Interestingly, 20 amino acids supple-        processes (Figure 5). Interestingly, carotenoids metab- <br /> mented complex medium, cells consumed             olism contained lethal genes (82%), suggesting <br /> a preferably. In order understand cel-        unique pathway synthesizing thermozeax- <br /> lular behavior, compared uptake rate individual         anthin thermobiszeaxanthin does al- <br /> amino acids actual biosynthetic demands.               ternative routes quite rigid T. thermophilus. <br /> Methionine cysteine consumed medium             Following carotenoids metabolism, nucleotides and <br /> according biosynthetic demands alanine,            lipids metabolism second highest number of <br /> valine, leucine, isoleucine glutamate consumed           completely essential genes (32.7% 24.6%, respectively). <br /> in excess individual biosynthetic requirements           On hand, examination conditionally essential <br /> (Table 3). Further analysis revealed surplus valine/     genes, i.e. genes essential minimal <br /> leucine/isoleucine alanine/glutamate utilized         media, revealed amino acid metabolism contains <br /> synthesize branched chain fatty acids amino         genes (Figure 5). Since complex media <br /> acids, respectively. Notably, leucine consumed           supplemented amino acids, them <br /> more biosynthetic demand (almost 7 times                 directly consumed media utilizing <br /> higher) mainly synthesize iso-15 iso-17 fatty acids,       biosynthetic pathways, classifying genes <br /> which constitute major composition T. thermophi-           pathways non-essential condi- <br /> lus. Remarkably, amino acid utilization pattern           tions. However, time, distribution certain <br /> completely different E. coli grown complex media,         completely essential genes amino acid metabolism also <br /> which consumed serine, aspartate, <span id='am-238' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-239' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-240' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-242'  ></span>glycine</span> threonine           indicates biosynthetic pathways tryptophan, proline <br /> ahead amino acids [42]. The subsequent silico         tyrosine crucial cell growth they <br /> analysis E. coli revealed rapidly depleted serine     synthesized cell albeit availabil- <br /> was primarily converted pyruvate serine deaminase,       ity complex medium. Interestingly, gene dele- <br /> and acetate producing ATP similarly           tion analyses showed high number completely <br /> observed microbes C. glutamicum          essential genes oxidative phosphorylation and <br /> and <span id='am-63' about='obo:NCBITaxon_1358' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1357 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Streptococcus lactis" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Lactococcus lactis (Lohnis) Schleifer et al. 1986" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacterium lacti" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Lactococcus lactis" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Lactococcus lactis</span>. In regard, interesting      TCA cycle. Oxygen key <span id='am-413' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-414' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-415'  ></span><span id='am-416' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-417' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> acceptor cellular <br /> note T. thermophilus possess unique nutrient con-            metabolism accept electrons redox <br /> sumption pattern, preferably synthesizing branched chain          cofactors involved TCA cycle generates energy <br /> fatty acids improving energy production further.      oxidative phosphorylation. If oxygen devoid or <br /> Overall, silico analysis highlights consumed   oxidative phosphorylation TCA cycle reac- <br /> amino acids contribute directly indirectly      tions perturbed, generally, bacteria regen- <br /> biomass synthesis, strategies increase uptake non-      erate redox cofactors switching fermentative <br /> consumed amino acids tyrosine, lysine, tryptophan         growth help substrate level phosphorylation. <br /> and histidine postulated enhance cellular           In regard, T. thermophilus lacks PTS and <br /> growth complex medium.                                         utilizes ABC transporters glucose uptake, could <br />                                                                   possible microbe switch fermenta- <br /> Analysis essential genes T. thermophilus                       tive metabolism completely thus, requires oxygen to <br /> We analysed essentiality individual genes T.        generate sufficient energy cell growth TCA cycle <br /> thermophilus glucose minimal complex TT                 oxidative phosphorylation. <br /> medium conditions using iTT548 model deleting every <br /> gene time (see Methods). The genes             Discussion <br /> categorized classes: (i) completely essential &#8211;        Thermophilic microbes represent unique <span id='am-274' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-275' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> organ- <br /> genes required cellular growth          isms distinct cell wall assembly enables them <br />                                                                                                                                                                                                                           http://www.microbialcellfactories.com/content/13/1/61 <br />                                                                                                                                                                                                                          Lee et al. Microbial Cell Factories 2014, 13:61 <br /> Table 2 Comparison metabolic reaction fluxes amino acids biosynthetic reactions minimal complex media <br /> Reaction ID Enzyme (Gene)                                              Reaction details                                                              Complex media                  Minimal media <br />                                                                                                                                                           (LB &lt; L &lt; UB)                  (LB &lt; L &lt; UB) <br /> R192           Alanine dehydrogenase (ald)                                  H[c] + NADH[c] + PYR[c] + NH4[c] &lt; = &gt; L_ALA[c] + NAD[c] + H2O[c] &#8722;0.031 &lt; &#8722;0.031 &lt; &#8722;0.031 0.234 &lt; 0.234 &lt; 2.416 <br /> R521           Asparagine synthase (asnB)                                   L_ASP[c] + L_GLN[c] + ATP[c] + H2O[c] &lt; = &gt; L_GLU[c] + L_ASN[c] +             &#8722;1.808 &lt; &#8722;1.808 &lt; &#8722;1.783       &#8722;0.537 &lt; &#8722;0.537 &lt; &#8722;0.522 <br />                                                                             PPi[c] + H[c] + AMP[c] <br /> R198           Aspartate aminotransferase (aspC)                            L_ASP[c] + AKG[c] &lt; = &gt; L_GLU[c] + OAC[c]                                     &#8722;1.782 &lt; &#8722;1.782 &lt; &#8722;1.782       &#8722;0.416 &lt; &#8722;0.416 &lt; &#8722;0.416 <br /> R205           Glutamine synthetase (gln)                                   L_GLU[c] + ATP[c] + NH4[c] - &gt; L_GLN[c] + ADP[c] + Pi[c] + H[c]               &#8722;1.793 &lt; &#8722;1.793 &lt; &#8722;1.793       &#8722;0.474 &lt; &#8722;0.474 &lt; &#8722;0.474 <br /> R202           Glutamate dehydrogenase (gdh)                                L_GLU[c] + NADP[c] + H2O[c] &lt; = &gt; H[c] + NADPH[c] + AKG[c] + NH4[c]           &#8722;9.830 &lt; &#8722;9.145 &lt; 4.713        &#8722;9.288 &lt; &#8722;8.813 &lt; 2.977 <br /> R227           Cysteine synthase (cysM)                                     OAC_SER[c] + H2S[c] - &gt; L_CYS[c] + H[c] + AC[c]                               0&lt;0&lt;0                          0 &lt; 0.005 &lt; 0.005 <br /> R235           Methionine synthase (metH)                                   L_HCYS[c] + 5MTHF[c] - &gt; L_MET[c] + H[c] + THF[c]                             0&lt;0&lt;0                          0 &lt; 0.009 &lt; 0.009 <br /> R272           N2-acetyl-L-lysine deacetylase (lysK)                        H2O[c] + N2AC_L_LYS[c] - &gt; L_LYS[c] + AC[c]                                   0.229 &lt; 0.229 &lt; 0.229          0.05 &lt; 0.05 &lt; 0.05 <br /> R651           3-phosphoserine phosphatase (serB)                           H2O[c] + PPTD_SER[c] - &gt; L_SER[c] + Pi[c]                                     0&lt;0&lt;0                          0 &lt; 1.072 &lt; 2.545 <br /> R655           Serine hydroxymethyltransferase (glyA)                       L_SER[c] + THF[c] - &gt; L_GLY[c] + H2O[c] + METHF[c]                            0&lt;0&lt;0                          0 &lt; 0.512 &lt; 6.42 <br /> R213           Serine hydroxymethyltransferase (glyA)                       L_GLY[c] + H2O[c] + 5_10_MNTHF[c] &lt; = &gt; L_SER[c] + THF[c]                     0.135 &lt; 0.135 &lt; 0.135          &#8722;1.787 &lt; &#8722;0.442 &lt; 3.993 <br /> R212           Tryptophan synthase (trpA)                                   L_SER[c] + IND_GLYP[c] &lt; = &gt; L_TRP[c] + H2O[c] + GAP[c]                       0.054 &lt; 0.054 &lt; 0.054          0.026 &lt; 0.026 &lt; 0.026 <br /> R221           Threonine synthase (thrC)                                    H2O[c] + OP_HSER[c] - &gt; L_THR[c] + Pi[c]                                      0.0714 &lt; 0.0714 &lt; 0.0714       0.063 &lt; 0.063 &lt; 0.063 <br /> R245           Branched-chain amino acid aminotransferase (ilvE) L_VAL[c] + AKG[c] &lt; = &gt; L_GLU[c] + 2K3M_BUT[c]                                           0.113 &lt; 0.113 &lt; 0.113          &#8722;0.068 &lt; &#8722;0.068 &lt; &#8722;0.068 <br /> R244           Branched-chain amino acid aminotransferase (ilvE) L_GLU[c] + 4M_2OPENTN[c] &lt; = &gt; L_LEU[c] + AKG[c]                                         &#8722;1.179 &lt; &#8722;1.179 &lt; &#8722;1.179 0.154 &lt; 0.154 &lt; 0.154 <br /> R246           Branched-chain amino acid aminotransferase (ilvE) L_ILE[c] + AKG[c] &lt; = &gt; L_GLU[c] + 3M_2OPENTN[c]                                         0.478 &lt; 0.478 &lt; 0.478          &#8722;0.024 &lt; &#8722;0.024 &lt; &#8722;0.024 <br /> R341           Pyrroline-5-carboxylate reductase (ProC)                     L_PRO[c] + NADP[c] &lt; = &gt; 2 H[c] + NADPH[c] + PYRR5CARB[c]                     &#8722;0.432 &lt; &#8722;0.432 &lt; &#8722;0.432       &#8722;0.119 &lt; &#8722;0.119 &lt; &#8722;0.119 <br /> R195           Argininosuccinate lyase (argH)                               L_ARG_SUCC[c] - &gt; L_ARG[c] + FUM[c]                                           1.144 &lt; 1.144 &lt; 1.144          0.132 &lt; 0.132 &lt; 0.132 <br /> R297           Aspartate aminotransferase (aat)                             L_PHE[c] + AKG[c] &lt; = &gt; L_GLU[c] + PPYR[c]                                    0&lt;0&lt;0                          &#8722;0.054 &lt; &#8722;0.054 &lt; &#8722;0.054 <br /> R284           Histidinol dehydrogenase (hisD)                              2 NAD[c] + H2O[c] + L_HISN[c] - &gt; L_HIS[c] + 3 H[c] + 2 NADH[c]               0.025 &lt; 0.025 &lt; 0.025          0.015 &lt; 0.015 &lt; 0.015 <br /> The reactions highlighted bold possess flux complex media. FVA conducted re-assure simulated fluxes results provided attainable lower upper bounds identified <br /> from analysis. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        Page 8 14 <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                          Page 9 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Table 3 Consumption production pattern amino acids complex media <br /> Amino acids     Supply    Biosynthetic    Contribution <br />                 medium         demand <br />                                                To     From     Remarks <br /> Alanine         0.7399         0.355           0.3853                        Surplus alanine contributes glycine synthesis <br /> Glycine         0.116          0.308                         0.1918          Some produced alanine <br /> Valine          0.224          0.109           0.1151                        Excess valine utilized synthesis anteiso-17:1 anteiso-15:0 <br />                                                                              fatty acids <br /> Leucine         1.427          0.244           1.1827                        Excess leucine utilized iso-15:0 iso-17:0 fatty acids synthesis <br /> Isoleucine      0.5181         0.039           0.4794                        Excess isoleucine utilized synthesis anteiso fatty acid synthesis <br /> Proline         &#8722;0.24          0.189                         0.4291          Synthesized glutamate <br /> Methionine      0.015          0.015                                         Utilized media biosynthetic demand <br /> Serine          0.057          0.135                         0.0785          Synthesized glycine <br /> Threonine       0.03           0.100                         0.0699          Synthesized aspartate <br /> Phenylalanine   0.086          0.086                                         Utilized media biosynthetic demand <br /> Aspartate       &#8722;0.12          0.103                         0.2232          Synthesized oxaloacetate glutamate <br /> Cysteine        0.0087         0.009                                         Utilized media biosynthetic demand <br /> Glutamate       1.48           0.176           1.3041                        Excess glutamate utilized amino acid synthesis <br /> Arginine        &#8722;0.93          0.211                         1.1409          Synthesized aspartate <br /> Asparagine      &#8722;0.023         0.103                         0.1262          Synthesized aspartate <br /> Lysine          &#8722;0.141         0.087                         0.2283          Synthesized oxoglutarate <br /> Glutamine       0              0.177                         0.1771          Synthesized glutamate <br /> Histidine       0              0.024                         0.0245          Synthetsized PRPP <br /> Tyrosine        &#8722;0.03204       0.069                         0.1007          Synthesized PEP, E4P <br /> Tryptophan      &#8722;0.01116       0.043                         0.0538          Synthesized serine <br />  <br />  <br />  <br /> to maintain cellular membrane integrity                 temperatures [10]. In regard, iTT548 completely cap- <br /> high temperatures. It reported thermophilic                 tures biosynthetic pathways thermozeaxanthins, <br /> organisms, including T. maritima S. solfataricus,                    addition metabolic routes biomass <br /> synthesize ether lipids long chain dicarboxylic                     precursors amino acids, nucleotides lipids. <br /> fatty acids fatty alcohols [43]. However, Thermus                    Similarly, iTT548 contains unique biosynthetic <br /> sp. synthesize ether lipids produce unique                    pathways unusual polyamines help in <br /> carotenoid molecules thermozeaxanthin                        stabilizing nucleotide strands proteins synthesis <br /> thermobizeaxanthin, embed lipid bi-layer                 high temperatures. It earlier reported that <br /> to attain required cellular membrane fluidity high                T. thermophilus unique polyamine synthesis: even <br />  <br />  <br />  <br />  <br />  Figure 5 Distribution essential genes T. thermophilus metabolic subsystems. Black, grey white colors indicate completely-, <br />  conditionally- non-essential genes, respectively. The numbers parenthesis represent number genes subsystem. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                           Page 10 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> the extreme thermophiles S. solfataricus pro-                low temperature range, hypothesize that <br /> duces relatively shorter polyamines [36]. Collectively,              thermophiles likely adjust biomass <br /> these results clearly detailed metabolic cover-             composition selectively better adapt growth <br /> age iTT548 thermophiles comparing                environment. Therefore, use appropriate biomass <br /> preceding GSMMs.                                                     equations simulating cellular growth correspond- <br />   Furthermore, work includes prudently drafted                ing temperature ranges crucial reliable prediction. <br /> biomass equation specific thermophiles, espe-                The gene deletion analyses iTT548 revealed <br /> cially Thermus sp. As mentioned earlier, comparative             interesting traits function deleted <br /> analysis T. thermophilus E. coli biomass composi-             genes respect overall cellular metabolism T. <br /> tions highlighted significant differences               thermophilus. Among them, notable rela- <br /> amino acid fatty acid compositions. Noticeably,              tively high percentage essential genes compared <br /> T. thermophilus biomass analysis revealed con-              E. coli (23% 13%), possibly smaller <br /> centration thermolabile amino acids              Open Reading Frames (ORF) content (only 2,263 com- <br /> as threonine histidine substantially lesser E.          pared 4,623) despite possessing necessary modules <br /> coli proline concentration higher. It            cell viable high temperatures. Furthermore, <br /> should highlighted observations                 observation highlights fact T. ther- <br /> good agreement earlier reports suggested              mophilus thrives higher temperatures other <br /> selective usage amino acid residues key             microbes, fitness relatively competitive <br /> adaptive strategy employed thermophiles [44,45].                  rigid network organization. Interestingly, the <br /> Arguably, cellular compositions thermophilic                  gene essentiality analyses indicated carotenoids <br /> microbes change depending growth temperature;                 metabolism functionally quite fragile of <br /> the current biomass equation derived based com-               genes essential cellular growth. However, it <br /> positional analysis T. thermophilus grown 70&#176;C. In             noted gene deletion analysis results are <br /> order clarify temperature dependent compositional             sensitive parameters silico medium <br /> change biomass, measured amino acid compositions               setup biomass composition. In regard, current <br /> in T. thermophilus 45&#176;C. Their comparison com-               biomass composition obtain T. thermophilus at <br /> positional data 70&#176;C clearly indicated           optimum growth temperatures, i.e. 70&#176;C, gene <br /> significant difference individual overall amino          deletion results current study applicable to <br /> acid concentrations (Figure 6A). Similarly, com-             condition. <br /> pared fatty acid compositions 40&#176;C 70&#176;C <br /> using data literature [30]. Very interestingly,             Conclusions <br /> unlike amino acid comparison, fatty acid compositions,               We presented genome-scale metabolic network T. <br /> both overall individual lower 40&#176;C                  thermophilus, iTT548, representing ther- <br /> (Figure 6B). Although able make complete            mophiles, containing 548 unique genes, 796 reactions <br /> comparison low high growth temperatures                  635 unique metabolites. As unique feature of <br /> since data available cellular constituents           T. thermophilus, iTT548 contains necessary metabolic <br /> such peptidoglycans thermotolerant carotenoids             pathways synthesizing unique carotenoids <br />  <br />  <br />  <br />  <br />  Figure 6 Influence temperature T. thermophilus biomass composition. (A) amino acid composition (mol%) 70&#176;C 45&#176;C and <br />  (B) fatty acid composition (mol%) 70&#176;C 40&#176;C. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                               Page 11 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> and polyamines help habituating high            butyldimethylsilyl-trifluoro-acetamide (MBDSTFA) in <br /> temperatures. The reconstructed metabolic model           DMF 80&#176;C 30 min. After centrifugation 14,000 g <br /> subsequently validated batch culture experiments     5 min, supernatant injected GC/MS split <br /> on glucose minimal complex medium            injection mode (1:10 split ratio). <br /> silico growth predictions iTT548 good <br /> agreement observed experimental results. The         Metabolic network reconstruction <br /> comparative flux analysis minimal complex         The genome-scale metabolic network T. thermophilus <br /> media highlighted consumption utilization     HB27 reconstructed using published genome <br /> branched chain amino acids directly relevant fatty     annotation [28] information collected vari- <br /> acids anabolic pathways, resulting higher growth      ous biological genomic databases basis the <br /> rates rich medium. A gene essentiality study       established procedure [48]. First, initial draft model <br /> also conducted silico simulation studies   constructed compiling annotated metabolic <br /> minimal complex media, highlighting high           genes corresponding biochemical reactions <br /> percentage lethal genes comparison E. coli,        KEGG [49] MetaCyc [50]. Then, reactions <br /> suggesting metabolic backbone T. thermo-          corrected elemental imbalances mapped <br /> <span id='am-442' about='obo:NCBITaxon_583468' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_584324 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-443' property="rdfs:label" content="Philus" datatype="xsd:string"></span><span id='am-444'  ></span><span id='am-445' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>philus</span> quite rigid. Overall, metabolic        appropriate genes devise proper gene-protein- <br /> network presented current study expected         reaction (GPR) relationships. Additionally, spontan- <br /> be significant contribution systems analysis     eous non-gene-associated reactions including <br /> thermophiles metabolic model utilized        metabolite transport incorporated the <br /> along high throughput datasets better un-        model based physiological evidence literature <br /> derstanding organism.                                      databases. The connectivity draft network was <br />                                                               checked using GapFind algorithm the <br /> Methods                                                       gaps [51]. The identified missing links filled either <br /> Microorganism culture conditions                          introduction sink reactions allow material <br /> T. thermophilus HB27 strain used model organ-        exchange cell surrounding environ- <br /> ism. For fermentation complex medium, single colony      ment adding reactions similar microbes <br /> was cultivated overnight 70&#176;C 150 rpm 5 mL      close knowledge gaps. <br /> the TM medium [46], culture transferred a <br /> 500 mL baffled-flask containing 100 mL TM broth. In <br /> case cultivation defined glucose minimal medium,        Biomass composition <br /> the overnight seed grown 70&#176;C 150 rpm 5 mL         Cellular biomass composition important pre- <br /> of TM medium transferred 500 mL             requisite silico flux analysis, especially during <br /> baffled-flask containing 100 mL defined minimal            exponential growth phase, primary cellular <br /> medium (DMM) 0.6% (w/v) glucose cultivated           objective maximize growth. Amino acid composition <br /> for 24 hours. Then, 5 mL flask culture DMM in-      T. thermophilus HB27 estimated hydrolyzing <br /> oculated 100 mL fresh DMM. During fermentations,        cell pellets 6 N HCl 24 h 130&#176;C, subse- <br /> cell growth monitored measuring optical density    quently analysing hydrolysates using HPLC equipped <br /> at 600 nm. The dry cell weight (DCW) estimated       UV-detector C18 column. Cell wall lipid <br /> by predetermined conversion factor 0.34.                 compositions obtained previous publications <br />                                                               Thermus sp. [30,31,35,52]. The overall DNA RNA <br /> Analytical methods                                            composition assumed E. coli [14] since <br /> Concentrations glucose, organic acids ethanol       data available Thermus sp. The individual <br /> the culture broth measured high performance           weights nucleotides DNA RNA calcu- <br /> liquid chromatography (HPLC) (Waters, Milford, MA)            lated based reported G + C content 69.4% [28]. <br /> equipped HPX-87H column (Bio-Rad, Hercules,           Detailed information biomass composition calculations <br /> CA), dual &#955; absorbance detector. The collected samples      Additional file 3. <br /> were centrifuged 14,000 g 4&#176;C 5 min the <br /> supernatant analyzed column using 5 mM           Constraints-based flux analysis <br /> sulfuric acid mobile phase 0.6 mL&#8722;1 min. Concen-      We implemented constraints-based flux analysis to <br /> trations amino acids determined gas chromato-      simulate T. thermophilus metabolism varying <br /> graph/mass spectrophotometer (GC/MS) (Agilent, Santa          environmental conditions. The biomass reaction was <br /> Ciara, CA) equipped HP-5MS column (Agilent),          maximized simulate exponential growth phase <br /> as previously reported [47]. In brief, samples       described [53-55]. Mathematically, op- <br /> centrifuged, dried derivatized methyl-N-t-           timization problem, i.e. maximization biomass subjected <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                  Page 12 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> to stoichiometric capacity constraints, formu-           supplied carbon source based complex <br /> lated follows:                                                   media based nutrient consumption profile. The simu- <br />                X                                                    lation results subsequently analyzed identify the <br />     max Z &#188;       cj vj                                             essential genes gene classified essential <br />                        j <br />                                                                     resulting cell growth prediction correspond- <br />            X                                                        ing mutant equal 5% wild-type. Note <br />     s:t:          S ij vj &#188; 0 &#8704; metabolite i <br />             j <br />                                                                     gene deletion analysis study per- <br />                                                                     formed using COBRA toolbox. <br />     vmin <br />      j   &#8804; vj &#8804; vmax <br />                  j               &#8704; reaction j <br />                                                                     Additional files <br /> where Sij refers stoichiometric coefficient me- <br /> tabolite involved reaction j, vj denotes flux            Additional file 1: Details iTT548 containing genes, reactions, <br /> or specific rate metabolic reaction j, vmin j   vmax <br />                                                           j <br />                                                                      metabolites. A list reactions added gap-filling possible <br />                                                                      new annotations identified study provided. <br /> represent lower upper limits flux of <br />                                                                      Additional file 2: SBML file iTT548. <br /> reaction j, respectively; Z corresponds cellular <br />                                                                      Additional file 3: Biomass composition T. thermophilus HB27 and <br /> objective linear function metabolic reactions        NGAM calculations. <br /> where relative weights determined coeffi-             Additional file 4: List essential genes glucose minimal and <br /> cient cj. In study, constraints-based flux analysis         complex media. <br /> problems solved using COBRA toolbox [56]. <br />                                                                     Competing interests <br /> Flux variability analysis                                           The authors declare competing interests. <br /> As constraints-based flux analysis optimization <br />                                                                     Authors&#8217; contributions <br /> based technique, possible multiple              NRL, ML, JBP DYL conceived designed study. NRL JWS <br /> flux distributions attaining physiological state.          performed batch culture experiments. NRL SA created draft <br /> Therefore, order confirm plausibility internal         model. ML refined model performed simulations. NRL, ML, IAK, <br />                                                                     JBP DYL wrote manuscript. JBP DYL coordinated and <br /> metabolic fluxes simulated minimal complex media             directed project. All authors read approved final <br /> by flux analysis, performed flux variability analysis        manuscript. <br /> (FVA) identify possible range fluxes while <br />                                                                     Acknowledgements <br /> simulating particular phenotypic state. Mathematically,           This work supported National University Singapore, Biomedical <br /> the optimization problem specific FVA repre-              Research Council A*STAR (Agency Science, Technology Research), <br /> sented follows:                                                  Singapore, Korea Research Foundation (KRF- 2010&#8211;0009169), Republic of <br />                                                                     Korea, grant Next-Generation BioGreen 21 Program (SSAC, <br />      max= min              vj                                       No. PJ009520), Rural Development Administration, Republic Korea. <br />                            X <br />                 s:t:              S ij vj &#188; 0                       Author details <br />                                                                     1 <br />                                                                      Department Food Science &amp; Engineering, Ewha Womans University, 11-1 <br />                              j <br />                            X                                        Daehyun-dong, Seodaemun-gu, Seoul 120-750, Korea. 2Department of <br />                                   cj vj &#188; Z obj                     Chemical Biomolecular Engineering, National University Singapore, 4 <br />                              j                                      Engineering Drive 4, Singapore 117585, Singapore. 3Bioprocessing <br />                                                                     Technology Institute, Agency Science, Technology Research <br />                            vmin <br />                             j   &#8804; vj &#8804; vmax <br />                                         j         j &#188; 1; &#8230;; n   (A*STAR), 20 Biopolis Way, #06-01, Centros, Singapore 138668, Singapore. <br />                                                                     4 <br />                                                                      Global Top5 Research Program, Ewha Womans University, 11-1 <br />                                                                     Daehyeon-dong, Seodaemun-gu, Seoul 120750, Korea. <br /> where Zobj denotes value objective calculated by <br /> flux analysis n number fluxes. The upper              Received: 14 February 2014 Accepted: 17 April 2014 <br /> range fluxes identified maximizing objective           Published: 28 April 2014 <br /> whereas lower range obtained minimizing the <br />                                                                     References <br /> same. In study, FVA implemented using                  1. Oshima T, Imahori K: Description <span id='am-447' about='obo:NCBITaxon_433555' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_433502 obo:NCBITaxon_33154 obo:NCBITaxon_41097 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-448' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-449' property="rdfs:label" content="Themus" datatype="xsd:string"></span><span id='am-450'  ></span><span id='am-451' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Themus</span> thermophilus (Yoshida and <br /> COBRA toolbox.                                                          Oshima) comb. nov., nonsporulation thermophilic bacterium a <br />                                                                         Japanese thermal spa. Int J Syst Bacteriol 1974, 24:102&#8211;112. <br />                                                                     2. Cava F, Hidalgo A, Berenguer J: Thermus thermophilus biological model. <br /> Gene deletion analysis                                                  Extremophiles 2009, 13:213&#8211;231. <br /> Gene deletion simulations performed maximiz-                3. Wimberly BT, Brodersen DE, Clemons WM Jr, Morgan-Warren RJ, Carter AP, <br /> ing cellular biomass constraining flux                Vonrhein C, Hartsch T, Ramakrishnan V: Structure 30S ribosomal <br />                                                                         subunit. Nature 2000, 407:327&#8211;339. <br /> the corresponding reaction(s) zero GPR re-            4. Sazanov LA, Hinchliffe P: Structure hydrophilic domain of <br /> lationships defined nutrient uptake rates. In case                respiratory complex I Thermus thermophilus. Science 2006, <br /> of glucose minimal medium, glucose fueled                   311:1430&#8211;1436. <br />                                                                     5. Selmer M, Dunham CM, Murphy FV, Weixlbaumer A, Petry S, Kelley AC, Weir <br /> carbon source. On hand, glucose, trehalose                    JR, Ramakrishnan V: Structure 70S ribosome complexed with <br /> and amino acids valine, leucine isoleucine                  mRNA tRNA. Science 2006, 313:1935&#8211;1942. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                                      Page 13 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> 6.    Yokoyama K, Ohkuma S, Taguchi H, Yasunaga T, Wakabayashi T, Yoshida M:         28. Henne A, Bruggemann H, Raasch C, Wiezer A, Hartsch T, Liesegang H, <br />       V-type H + &#8722;ATPase/synthase thermophilic <span id='am-168' about='obo:NCBITaxon_1730' typeof='owl:Thing obo:NCBITaxon_186806 obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-169' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Eubacterium" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="PMID:8863439" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-173'  ></span>eubacterium</span>, Thermus               Johann A, Lienard T, Gohl O, Martinez-Arias R, Jacobi C, Starkuviene V, <br />       thermophilus - Subunit structure operon. J Biol Chem 2000,                     Schlenczeck S, Dencker S, Huber R, Klenk HP, Kramer W, Merkl R, Gottschalk <br />       275:13955&#8211;13961.                                                                   G, Fritz HJ: The genome sequence extreme thermophile Thermus <br /> 7.    Pantazaki <span id='am-424' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-425'  ></span><span id='am-426' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-428' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-429' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>, Pritsa AA, Kyriakidis DA: Biotechnologically relevant enzymes        thermophilus. Nat Biotechnol 2004, 22:547&#8211;553. <br />       Thermus thermophilus. Appl Microbiol Biotechnol 2002, 58:1&#8211;12.            29. Kaneda T: Iso- anteiso-fatty acids bacteria: biosynthesis, function, <br /> 8.    Niehaus F, Bertoldo C, Kahler M, Antranikian G: Extremophiles source          taxonomic significance. Microbiological Rev 1991, 55:288&#8211;302. <br />       novel enzymes industrial application. Appl Microbiol Biotechnol 1999,   30. Nordstrom KM, Laakso SV: Effect growth temperature fatty acid <br />       51:711&#8211;729.                                                                        composition Thermus strains. Appl Environ Microbiol 1992, <br /> 9.    Riyanti EI: Genetic manipulation thermophiles ethanol production.           58:1656&#8211;1660. <br />       In PhD Thesis. The University New South Wales: School biotechnology      31. Pask-Hughes RA, Shaw N: Glycolipids extreme thermophilic <br />       biomolecular sciences; 2007.                                                   bacteria belonging geus Thermus. J Bacteriol 1982, 149:54&#8211;58. <br /> 10.   Tian B, Hua Y: Carotenoid biosynthesis extremophilic <span id='am-56' about='obo:NCBITaxon_1298' typeof='owl:Thing obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_118964 obo:NCBITaxon_183710 obo:NCBITaxon_1297'><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:9103641" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Deinococcus" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Deinobacter" datatype="xsd:string"></span>Deinococcus</span>&#8211;Thermus    32. Alarico S, da Costa MS, Empadinhas N: Molecular physiological role of <br />       bacteria. Trends Microbiol 2010, 18:512&#8211;520.                                       trehalose-hydrolyzing alpha-glucosidase Thermus thermophilus <br /> 11.   Shigi N, Suzuki T, Tamakoshi M, Oshima T, Watanabe K: Conserved bases           HB27. J Bacteriol 2008, 190:2298&#8211;2305. <br />       TPsi C loop tRNA determinants thermophile-specific              33. Lengsfeld C, Schonert S, Dippel R, Boos W: Glucose- glucokinase- <br />       2-thiouridylation position 54. J Biol Chem 2002, 277:39128&#8211;39135.               controlled mal gene expression Escherichia coli. J Bacteriol 2009, <br /> 12.   Lewis NE, Nagarajan H, Palsson BO: Constraining metabolic                      191:701&#8211;712. <br />       genotype-phenotype relationship using phylogeny silico                 34. Holst O: Structure Lipopolysaccharide Core Region. In Bacterial <br />       methods. Nat Rev Microbiol 2012, 10:291&#8211;305.                                       Lipopolysaccharides. Edited Knirel YA, Valvano MA. Wien: Springer-Verlag; <br /> 13.   Liu L, Agren R, Bordel S, Nielsen J: Use genome-scale metabolic models          2011:21&#8211;39. <br />       understanding microbial physiology. FEBS Letters 2010, 584:2556&#8211;2564.      35. Mandelli F, Yamashita F, Pereira JL, Mercadante AZ: Evaluation biomass <br /> 14.   Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD,                    production, carotenoid level antioxidant capacity produced by <br />       Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale metabolic               <span id='am-302' about='obo:NCBITaxon_276' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-303' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-304'  ></span><span id='am-305' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-306' property="rdfs:label" content="Thermus filiformis" datatype="xsd:string"></span>Thermus filiformis</span> using fractional factorial design. Braz J Microbiol 2012, <br />       reconstruction <span id='am-470' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-471' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-474' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> MG1655 accounts 1260             43:126&#8211;134. <br />       ORFs thermodynamic information. Mol Syst Biol 2007, 3:121.                 36. Oshima T: Unique polyamines produced extreme thermophile, <br /> 15.   Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome-scale                Thermus thermophilus. Amino Acids 2007, 33:367&#8211;372. <br />       reconstruction metabolic network <span id='am-78' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-79' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span>Bacillus subtilis</span> based              37. Raghunathan AU, Perez-<span id='am-430' about='obo:NCBITaxon_67919' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_41937 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_23513 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-431'  ></span><span id='am-432' property="rdfs:label" content="Correa" datatype="xsd:string"></span><span id='am-433' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Correa</span> JR, Bieger LT: Data reconciliation and <br />       high-throughput phenotyping gene essentiality data. J Biol Chem                parameter estimation flux-balance analysis. Biotechnol Bioeng 2003, <br />       2007, 282:28791&#8211;28799.                                                             84:700&#8211;709. <br /> 16.   Mo ML, Palsson BO, Herrgard MJ: Connecting extracellular metabolomic           38. Schuster S, Pfeiffer T, Fell DA: Is maximization molar yield metabolic <br />       measurements intracellular flux states yeast. BMC Syst Biol 2009,            networks favoured evolution? J Theor Biol 2008, 252:497&#8211;504. <br />       3:37.                                                                          39. Demirtas MU, Kolhatkar A, Kilbane JJ 2nd: Effect aeration agitation <br /> 17.   Chung BK, Selvarasu S, Andrea C, Ryu J, Lee H, Ahn J, Lee DY:                      growth rate Thermus thermophilus batch mode. J Biosci Bioeng <br />       Genome-scale metabolic reconstruction silico analysis                    2003, 95:113&#8211;117. <br />       methylotrophic yeast Pichia pastoris strain improvement. Microb            40. Mckay A, Quilter J, Jones CW: Energy conservation extreme <br />       Cell Fact 2010, 9:50.                                                              thermophile <span id='am-399' about='obo:NCBITaxon_300852' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_274 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-400' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-401' property="rdfs:label" content="Thermus thermophilus HB8" datatype="xsd:string"></span><span id='am-402' property="oboInOwl:hasRelatedSynonym" content="Thermus thermophilus ATCC 27634" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasExactSynonym" content="Thermus thermophilus strain HB8" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasExactSynonym" content="Thermus thermophilus str. HB8" datatype="xsd:string"></span>Thermus Thermophilus HB8</span>. Arch Microbiol 1982, 131:43&#8211;50. <br /> 18.   Kjeldsen KR, Nielsen J: In silico genome-scale reconstruction              41. Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br />       validation <span id='am-158' about='obo:NCBITaxon_1716' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-159'  ></span><span id='am-160' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="PMID:7547291" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="Corynebacterium" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasDbXref" content="PMID:10425754" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasDbXref" content="PMID:19661509" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="PMID:7547293" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span>Corynebacterium</span> glutamicum metabolic network.                    constraint-based genome-scale metabolic model. Metab Eng 2003, <br />       Biotechnol Bioeng 2009, 102:583&#8211;597.                                               5:264&#8211;276. <br /> 19.   Park JM, Kim TY, Lee SY: Genome-scale reconstruction silico             42. Selvarasu S, Ow DS-W, Lee SY, Lee MM, Oh SK-W, Karimi <span id='am-24' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Ia" datatype="xsd:string"></span>IA</span>, Lee D-Y: <br />       analysis <span id='am-11' about='obo:NCBITaxon_381666' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_119060 obo:NCBITaxon_2 obo:NCBITaxon_106590 obo:NCBITaxon_106589'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Wautersia eutropha H16" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Ralstonia eutropha strain H16" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Ralstonia eutropha H16" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Ralstonia eutropha str. H16" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Ralstonia eutropha ATCC 17699" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:16964242" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Alcaligenes eutropha H16" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Cupriavidus necator ATCC 17699" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:349973" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Cupriavidus necator H16" datatype="xsd:string"></span>Ralstonia eutropha H16</span> polyhydroxyalkanoate                    Characterizing Escherichia coli DH5&#945; growth metabolism a <br />       synthesis, lithoautotrophic growth, 2-methyl citric acid production.           complex medium using genome-scale flux analysis. Biotechnol Bioeng <br />       BMC Syst Biol 2011, 5:101.                                                         2009, 102:923&#8211;934. <br /> 20.   Oberhardt MA, Puchalka J, Fryer KE, Martins dosSantos VA, Papin JA:            43. Koga Y: Thermal adaptation archaeal bacterial lipid <br />       Genome-scale metabolic network analysis opportunistic pathogen              membranes. <span id='am-199' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-200' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-208'  ></span><span id='am-209' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span>Archaea</span> 2012, 2012:789652. <br />       <span id='am-347' about='obo:NCBITaxon_287' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_136841 obo:NCBITaxon_135621'><span id='am-348' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas pyocyanea" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931957" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931956" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437770" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasRelatedSynonym" content="Peudomonas aeruginosa" datatype="xsd:string"></span><span id='am-355'  ></span><span id='am-356' property="oboInOwl:hasRelatedSynonym" content="Bacillus aeruginosus" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437769" datatype="xsd:string"></span><span id='am-358' property="rdfs:label" content="Pseudomonas aeruginosa" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437768" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasAlternativeId" content="NCBITaxon:932477" datatype="xsd:string"></span><span id='am-361' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931958" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931955" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyocyaneus" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224290" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasRelatedSynonym" content="Bacillus pyocyaneus" datatype="xsd:string"></span><span id='am-366' property="oboInOwl:hasRelatedSynonym" content="Bacterium pyocyaneum" datatype="xsd:string"></span><span id='am-367' property="oboInOwl:hasRelatedSynonym" content="Bacterium aeruginosum" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas polycolor" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasRelatedSynonym" content="probable synonym or variety: &quot;Pseudomonas polycolor&quot; Clara 1930" datatype="xsd:string"></span>Pseudomonas aeruginosa</span> PAO1. J Bacteriol 2008, 190:2790&#8211;2803.                  44. Kumar S, Tsai CJ, Nussinov R: Factors enhancing protein thermostability. <br /> 21.   Selvarasu S, Karimi IA, Ghim GH, Lee DY: Genome-scale modeling              Protein Eng 2000, 13:179&#8211;191. <br />       silico analysis mouse cell metabolic network. Mol bioSyst 2010,             45. Mallik S, Kundu S: A comparison structural evolutionary attributes <br />       6:152&#8211;161.                                                                         Escherichia coli Thermus thermophilus small ribosomal subunits: <br /> 22.   Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, Palsson        signatures thermal adaptation. PloS One 2013, 8:e69898. <br />       BO: Global reconstruction human metabolic network based              46. Koyama Y, Hoshino T, Tomizuka N, Furukawa K: Genetic transformation of <br />       genomic bibliomic data. Proc Natl Acad Sci U S A 2007,                         extreme thermophile Thermus thermophilus Thermus <br />       104:1777&#8211;1782.                                                                     spp. J Bacteriol 1986, 166:338&#8211;340. <br /> 23.   Lakshmanan M, Koh G, Chung BK, Lee DY: Software applications flux          47. Yang KM, Lee NR, Woo JM, Choi W, Zimmermann M, Blank LM, Park JB: <br />       balance analysis. Brief Bioinform 2014, 15:108&#8211;122.                                Ethanol reduces mitochondrial membrane integrity impacts <br /> 24.   Lee SJ, Lee DY, Kim TY, Kim BH, Lee J, Lee SY: Metabolic engineering            carbon metabolism <span id='am-267' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-268' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-270'  ></span><span id='am-271' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-273' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae. FEMS Yeast Res 2012, <br />       Escherichia coli enhanced production succinic acid, based                12:675&#8211;684. <br />       genome comparison silico gene knockout simulation. Appl Environ         48. Thiele I, Palsson BO: A protocol generating high-quality genome-scale <br />       Microbiol 2005, 71:7880&#8211;7887.                                                      metabolic reconstruction. Nat Protocol 2010, 5:93&#8211;121. <br /> 25.   Matsuda F, Furusawa C, Kondo T, Ishii J, Shimizu H, Kondo A: Engineering       49. Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, Kawashima S, <br />       strategy yeast metabolism higher alcohol production. Microb Cell            Katayama T, Araki M, Hirakawa M: From genomics chemical genomics: <br />       Fact 2011, 10:70.                                                                  new developments KEGG. Nucleic Acids Res 2006, 34:D354&#8211;D357. <br /> 26.   Zhang Y, Thiele I, Weekes D, Li Z, Jaroszewski L, Ginalski K, Deacon AM,       50. Caspi R, Altman T, Dreher K, Fulcher CA, Subhraveti P, Keseler IM, Kothari <br />       Wooley J, Lesley SA, Wilson IA, Palsson BO, Osterman A, Godzik A:                  A, Krummenacker M, Latendresse M, Mueller LA, Ong Q, Paley S, Pujar A, <br />       Three-dimensional structure view central metabolic network               Shearer AG, Travers M, Weerasinghe D, Zhang P, Karp PD: The MetaCyc <br />       <span id='am-179' about='obo:NCBITaxon_2336' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2335 obo:NCBITaxon_200918 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_188708 obo:NCBITaxon_188709 obo:NCBITaxon_2419'><span id='am-180' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Thermotoga maritima" datatype="xsd:string"></span><span id='am-182'  ></span><span id='am-183' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:11542087" datatype="xsd:string"></span>Thermotoga maritima</span>. Science 2009, 325:1544&#8211;1549.                                  database metabolic pathways enzymes BioCyc <br /> 27.   Ulas T, Riemer SA, Zaparty M, Siebers B, Schomburg D: Genome-scale                 collection pathway/genome databases. Nucleic Acids Res 2012, <br />       reconstruction analysis metabolic network                        40:D742&#8211;D753. <br />       hyperthermophilic archaeon <span id='am-192' about='obo:NCBITaxon_2287' typeof='owl:Thing obo:NCBITaxon_118883 obo:NCBITaxon_1 obo:NCBITaxon_28889 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183924 obo:NCBITaxon_2284 obo:NCBITaxon_2281'><span id='am-193'  ></span><span id='am-194' property="oboInOwl:hasRelatedSynonym" content="Sulfolobus fataricus" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasAlternativeId" content="NCBITaxon:44290" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="Sulfolobus solfataricus" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Sulfolobus solfataricus</span>. PloS One 2012,             51. Satish Kumar V, Dasika MS, Maranas CD: Optimization based automated <br />       7:e43401.                                                                          curation metabolic reconstructions. BMC Bioinform 2007, 8:212. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                              Page 14 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> 52. Ray PH, White DC, Brock TD: Effect growth temper <br /> </body></html>