<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Biotechnol. 2014 November ; 32(11): 1141&#8211;1145. doi:10.1038/nbt.3011. <br />  <br />  <br />  <br />                            Sequence-specific antimicrobials using efficiently delivered <br />                            RNA-guided nucleases <br />                            Robert J. Citorik1,2,7, Mark Mimee1,2,7, Timothy K. Lu1,2,3,4,5,6 <br />                            1MIT   Microbiology Program, Massachusetts Institute Technology, Cambridge, Massachusetts, <br />                            USA <br />                            2MITSynthetic Biology Center, Massachusetts Institute Technology, Cambridge, <br />                            Massachusetts, USA <br />                            3Departmentof Biological Engineering, Massachusetts Institute Technology, Cambridge, <br />                            Massachusetts, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Department Electrical Engineering Computer Science, Massachusetts Institute of <br />                            Technology, Cambridge, Massachusetts, USA <br />                            5Harvard     Biophysics Program, Harvard University, Boston, Massachusetts, USA <br />                            6Broad    Institute MIT Harvard, Cambridge, Massachusetts, USA <br />  <br />                            Abstract <br />                                 Current antibiotics tend broad spectrum, leading indiscriminate killing commensal <br />                                 bacteria accelerated evolution drug resistance. Here, use CRISPR-Cas technology to <br />                                 create antimicrobials spectrum activity chosen design. RNA-guided nucleases <br />                                 (RGNs) targeting specific DNA sequences delivered efficiently microbial populations using <br />                                 bacteriophage bacteria carrying <span id='am-283' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-284' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-285' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> transmissible conjugation. The DNA targets of <br />                                 RGNs undesirable genes polymorphisms, including antibiotic resistance virulence <br />                                 determinants carbapenem-resistant <span id='am-191' about='obo:NCBITaxon_543' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-192' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae Rahn 1937" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="Enterobacteriaceae" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="PMID:10555334" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae (ex Lapage 1979) Lapage 1982, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasRelatedSynonym" content="gamma-3 proteobacteria" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:10555323" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae (ex Rahn 1937) Ewing et al. 1980, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-202'  ></span><span id='am-203' property="oboInOwl:hasDbXref" content="PMID:16166704" datatype="xsd:string"></span>Enterobacteriaceae</span> enterohemorrhagic <span id='am-204' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-205' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-210'  ></span><span id='am-211' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-219' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span>Escherichia coli</span>. <br />                                 Delivery RGNs significantly improves survival <span id='am-127' about='obo:NCBITaxon_7136' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_40082 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_37573 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_7135 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="Galleria" datatype="xsd:string"></span><span id='am-130'  ></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Galleria</span> mellonella infection model. We <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 RGNs enable modulation complex bacterial populations selective knockdown <br />                                 targeted strains based genetic signatures. RGNs constitute <span id='am-124' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-125' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> highly discriminatory, <br />                                 customizable antimicrobials enact selective pressure DNA level reduce the <br />                                 prevalence undesired genes, minimize off-target effects enable programmable remodeling <br />                                 <span id='am-9' about='obo:NCBITaxon_13613' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1446379 obo:NCBITaxon_1 obo:NCBITaxon_3313 obo:NCBITaxon_3367 obo:NCBITaxon_131567 obo:NCBITaxon_33090 obo:NCBITaxon_1437180 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Microbiota" datatype="xsd:string"></span>microbiota</span>. <br />  <br />  <br />  <br />                            &#169; 2014 Nature America, Inc. All rights reserved. <br />                            Correspondence addressed T.K.L. (timlu@mit.edu). <br />                            7These authors contributed equally work. <br />                            Note: Any Supplementary Information Source <span id='am-278' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-279' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-280'  ></span><span id='am-281' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-282' property="rdfs:label" content="Data" datatype="xsd:string"></span>Data</span> files available online version paper. <br />                            AUTHOR CONTRIBUTIONS <br />                            R.J.C. M.M. designed performed experiments. R.J.C., M.M. T.K.L. conceived study, analyzed data, discussed <br />                            results wrote manuscript. <br />                            COMPETING FINANCIAL INTERESTS <br />                            The authors declare competing financial interests: details available online version paper. <br />                            Reprints permissions information available online http://www.nature.com/reprints/index.html. <br />                             Citorik et al.                                                                                         Page 2 <br />  <br />  <br />                                             There mounting concern emergence proliferation multidrug-resistant <br />                                             bacterial pathogens dwindling treatment options organisms1. Recently, <br />                                             carbapenem-resistant Enterobacteriaceae, group intestinal Gram-negative bacteria <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             known cause life-threatening opportunistic infections, highlighted US Centers <br />                                             Disease Control Prevention urgent threats antibiotic- <br />                                             resistant bacteria1. Carbapenems traditionally reserved resort treatment <br />                                             Gram-negative infections, spread extended-spectrum &#946;-lactamases has <br />                                             necessitated increased usage carbapenems favored emergence carbapenem- <br />                                             resistant strains refractory current treatment options. The responsible <br />                                             enzymes, including New-Delhi metallo-&#946;-lactamase 1 (NDM-1), confer <span id='am-141' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-142'  ></span><span id='am-143' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="Pan" datatype="xsd:string"></span>pan</span>-resistance <br />                                             &#946;-lactam antibiotics frequently co-harbored additional resistance <br />                                             determinants mobile plasmids facilitate rapid dissemination beyond <br />                                             Enterobacteriaceae2. The diversity multidrug-resistant bacteria compounds difficulty <br />                                             developing treatments target pathogens commensal reservoirs avoid <br />                                             nonspecific broad-spectrum activity. <br />  <br />                                             Here, introduce alternative antimicrobial approach imposes direct evolutionary <br />                                             pressure gene level using efficiently delivered, programmable RGNs. We <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             engineered clustered, regularly interspaced, short palindromic repeats (CRISPR)&#8211; <br />                                             CRISPR-associated (Cas) system, naturally employed bacteria defense strategy <br />                                             mobile elements3,4, effect cell death plasmid loss detection genetic <br />                                             signatures associated virulence antibiotic resistance. The Type II CRISPR-Cas <br />                                             <span id='am-14' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span>Streptococcus</span> pyogenes effective, programmable tool genome editing <br />                                             gene expression wide variety organisms5. The specificity CRISPR-Cas is <br />                                             dictated short, spacer sequences flanked direct repeats encoded CRISPR locus, <br />                                             transcribed processed CRISPR RNAs (crRNA)6. With aid trans- <br />                                             activating small RNA (tracrRNA), crRNAs enable Cas9 endonuclease introduce <br />                                             double-stranded breaks target DNA sequences6,7. Through simple modifications of <br />                                             spacers CRISPR locus, RGN direct cleavage DNA sequence, <br />                                             design restriction requisite NGG motif immediately 3&#8242; target <br />                                             sequence7. By packaging RGNs bacteriophage particles harnessing mobilizable <br />                                             plasmids, implemented conditional-lethality devices high specificity, modularity <br />                                             multiplexability undesired DNA sequences (Fig. 1a Supplementary Fig. 1). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             To establish RGN functionality mediating sequence-specific cytotoxicity, designed <br />                                             RGNs induce double-stranded breaks blaSHV-18 blaNDM-1, encode extended- <br />                                             spectrum pan-resistance &#946;-lactam antibiotics, respectively (Supplementary Table <br />                                             1)8,9. Transformation E. coli plasmid-borne RGNs (pRGNs) containing a <br />                                             chromosomal copy target genes resulted nearly 1,000-fold decrease in <br />                                             transformation efficiency compared wild-type cells lacking target (Supplementary <br />                                             Fig. 2a). These results corroborate mutual exclusivity functional crRNA a <br />                                             cognate target locus10,11. Sequence analysis 30 escape mutants (cells receive and <br />                                             maintain RGN plasmid despite presence target sequence) revealed tolerance <br />                                             exclusively defective construct frequently arose spacer deletion <br />                                             CRISPR locus (Supplementary Fig. 3). Furthermore, deletion tracrRNA as <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                             Page 3 <br />  <br />  <br />                                             inactivation RuvC-like nuclease domain Cas9 (D10A)7 abrogated loss <br />                                             transformation efficiency cells harbored target sequence. Thus, catalytically <br />                                             active endonuclease, tracrRNA crRNA necessary sufficient mediate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             sequence-specific cytotoxicity E. coli (Fig. 1a). <br />  <br />                                             Antibiotic resistance genes present large, multicopy plasmids capable of <br />                                             autonomous transfer microbial populations, resulting horizontal dissemination drug <br />                                             resistance2. RGN activity high-copy plasmids verified using GFP-expressing, <br />                                             ColE1-derived vector containing standard &#946;-lactamase selectable marker (pZE-blaz-gfp)12 <br />                                             blaNDM-1 (pZE-blaNDM-1-gfp). <span id='am-258' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-259' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-260' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Vectors</span> bearing ColE1 origin reported be <br />                                             present copy numbers 50&#8211;70 cell12. Transformation cells containing pZE- <br />                                             blaNDM-1- gfp pRGNndm-1, plasmid-borne RGN targeting blaNDM-1, led three- <br />                                             log10 reduction transformants retaining resistance &#946;-lactam antibiotic carbenicillin, <br />                                             transformation cells containing target-free pZE-blaz-gfp pRGNndm-1 did <br />                                             lead reduction resistant transformants (Supplementary Fig. 2b). The activity of <br />                                             RGNs sufficient exclude high-copy antibiotic resistance plasmids from <br />                                             cells resensitize resistant population antibiotics. Similarly, transformation of <br />                                             cells possessing pZE-blaNDM-1-gfp pRGNndm-1 led ~1,000-fold decrease in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             GFP-expressing cells, measured flow cytometry, decrease with <br />                                             transformation cells possessing pZE-blaz-gfp pRGNndm-1 (Supplementary Fig. 4). <br />  <br />                                             The usefulness RGNs antimicrobial therapy hinges high-efficiency delivery of <br />                                             genetic constructs bacterial cells. We explored mechanisms horizontal gene <br />                                             transfer used bacteria acquire foreign genetic elements: plasmid conjugation <br />                                             viral transduction. Although constrained requirements cell-cell contact, <br />                                             conjugative plasmids wide host ranges recipient factors necessary for <br />                                             DNA uptake identified13. Efficient transfer RGNs achieved using the <br />                                             broad-host-range <span id='am-35' about='obo:NCBITaxon_2478' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Plasmid R1162" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmid R1162</span> mobilized E. coli S17-1, contains conjugative <br />                                             machinery <span id='am-40' about='obo:NCBITaxon_2503' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Plasmid RP4" datatype="xsd:string"></span><span id='am-44'  ></span>plasmid RP4</span> integrated chromosome. In filter mating experiments, <br />                                             conjugative transfer RGNs resulted 40- 60-fold reduction target carbenicillin- <br />                                             resistant recipient cells (Supplementary Fig. 1b). Under selection transconjugants, <br />                                             transfer RGNs recipients yielded 2- 3-log10 reduction target cells compared <br />                                             controls, suggesting conjugation efficiency, opposed RGN activity, limited <br />                                             RGN efficacy context (Supplementary Fig. 1c). Future work necessary to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             optimize efficiency conjugation-based delivery vehicles antimicrobials <br />                                             based mobilizable RGNs. <br />  <br />                                             Bacteriophages natural predators bacteria highly adept injecting DNA into <br />                                             host cells. To adapt phage RGN delivery, engineered phagemid vectors pairing <br />                                             RGN constructs targeting blaNDM-1 blaSHV-18 f1 origin facilitate packaging <br />                                             M13 particles. Phage-packaged RGNndm-1 (&#934;RGNndm-1) capable of <br />                                             comprehensively transducing population E. coli EMG2 (Supplementary Fig. 5). To test <br />                                             &#934;RGNs, conjugated native plasmids containing blaNDM-1 (pNDM-1) blaSHV-18 <br />                                             (pSHV-18) clinical isolates EMG2. Treatment EMG2 pNDM-1 EMG2 <br />                                             pSHV-18 strains cognate &#934;RGNs resulted 2- 3-log10 reductions viable cells <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                              Page 4 <br />  <br />  <br />                                             absence selection (Fig. 1b). Furthermore, &#934;RGNs engendered toxicity <br />                                             wild-type EMG2 EMG2 containing noncognate plasmids (Fig. 1b). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             In naturally occurring Type II CRISPR-Cas systems, CRISPR locus contain <br />                                             multiple spacers, processed independent crRNA molecule that, with <br />                                             tracrRNA, enables Cas9 cleave cognate DNA sequences6. To explore utility a <br />                                             single &#934;RGN exhibiting activity genetic signature, engineered a <br />                                             construct containing spacers encoding different crRNAs targeting blaNDM-1 <br />                                             blaSHV-18 resistance genes (&#934;RGNndm-1/shv-18). &#934;RGNndm-1/shv-18 generated 2- to <br />                                             3-log10 reductions viable cells EMG2 pNDM-1 EMG2 pSHV-18, wild- <br />                                             type EMG2 (Fig. 1b). Thus, RGNs multiplexed different genetic signatures, <br />                                             enabling simultaneous targeting variety virulence factors resistance genes that <br />                                             exist microbial populations. <br />  <br />                                             In addition antibiotic-modifying enzymes, &#946;-lactamases, alterations host <br />                                             proteins constitute major antibiotic resistance mechanism bacteria14. Owing the <br />                                             specificity CRISPR-Cas system, suspected RGNs discriminate between <br />                                             susceptible resistant strains differ single-nucleotide mutation DNA gyrase <br />                                             (gyrA), confers resistance quinolone antibiotics (Supplementary Table 1)14. Indeed, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             &#934;RGNgyrAD87G specifically cytotoxic quinolone-resistant E. coli harboring the <br />                                             chromosomal gyrAD87G mutation isogenic strains wild-type <br />                                             gyrA gene (Fig. 1c). <br />  <br />                                             Killing curves revealed &#934;RGNs mediated rapid killing target cells, viable cell <br />                                             counts decayed exponentially (t1/2 ~ 13 min) maximal bactericidal effect was <br />                                             achieved 2&#8211;4 h (Fig. 2a). Moreover, &#934;RGN antimicrobial activity increased with <br />                                             phagemid particle concentration (Fig. 2b Supplementary Fig. 6). To characterize <br />                                             cellular response RGN-mediated targeting, assessed treatment cells that <br />                                             harbored GFP reporter SOS regulation. E. coli bacteria respond to <br />                                             chromosomal double-stranded breaks, including artificially generated the <br />                                             meganuclease I-SceI, inducing DNA repair activation SOS response15. <br />                                             We observed 2.6- 4.0-fold increase fluorescence cells containing reporter <br />                                             plasmid plasmid-borne (blaNDM-1) chromosomal (gyrAD87G) target site, <br />                                             respectively, treated cognate versus noncognate &#934;RGNs (Supplementary Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             7). These results confirm RGNs induce DNA damage target cells and <br />                                             demonstrate coupled SOS-based reporters detect specific genes or <br />                                             sequences, single-nucleotide level. <br />  <br />                                             We intrigued observe targeted cleavage blaNDM-1 &#934;RGNndm-1 the <br />                                             context native plasmid lethal host cells, targeted cleavage same <br />                                             gene standard cloning vector (&#8216;pNDM-1&#8217; versus &#8216;pZA-ndm1-gfp&#8217;, respectively, <br />                                             Fig. 2c). Therefore, hypothesized &#934;RGN-induced plasmid loss itself does not <br />                                             elicit lethality, results cytotoxicity means co-harbored plasmid- <br />                                             borne functions. Toxin-antitoxin systems components natural plasmids ensure <br />                                             persistence bacterial populations inhibiting growth daughter cells do <br />                                             inherit plasmid. Toxin-antitoxin systems traditionally consist labile antitoxin that <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                            Page 5 <br />  <br />  <br />                                             quenches activity stable toxin. Owing differential stability two <br />                                             components, cessation gene expression plasmid loss leads depletion the <br />                                             antitoxin pool faster toxin pool, resulting de-repression toxin activity and, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             ultimately, stasis programmed cell death16. Analysis sequenced pSHV-18 plasmid <br />                                             revealed presence single toxin-antitoxin module, pemIK, commonly found <br />                                             isolates harboring extended-spectrum &#946;-lactamases17. When complemented the <br />                                             PemI antitoxin expressed constitutively trans (pZA31-pemI), &#934;RGNshv-18 treatment of <br />                                             EMG2 pSHV-18 abrogated cytotoxicity instead resulted resensitization <span id='am-149' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-150' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152'  ></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> <br />                                             multidrug-resistant strain carbenicillin (Fig. 2d). We attributed effect PemI <br />                                             inactivation PemK toxin, enabling loss pSHV-18 plasmid without <br />                                             concomitant bacterial killing. Toxin-antitoxin systems dictate outcome of <br />                                             &#934;RGN activity episomal targets, presence leads cytotoxicity absence <br />                                             neutralization plasmid loss. <br />  <br />                                             To demonstrate versatility RGNs specifically combating pathogens, we <br />                                             designed &#934;RGN target intimin, chromosomally encoded virulence factor of <br />                                             enterohemorrhagic E. coli O157:H7 (EHEC) necessary intestinal colonization and <br />                                             pathology. Encoded eae gene, intimin cell-surface adhesin mediates intimate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             attachment host epithelium, permitting subsequent disruption intestinal tight <br />                                             junctions effacement microvilli18. Treatment EHEC &#934;RGNeae resulted a <br />                                             20-fold reduction viable cell counts; cytotoxicity increased additional 100- <br />                                             fold kanamycin selection &#934;RGNeae transductants (Fig. 3a). The increase in <br />                                             cytotoxicity selection cells receiving construct suggests efficacy of <br />                                             &#934;RGN treatment limited delivery strain. Furthermore, &#934;RGN treatment was <br />                                             assessed G. mellonella larvae, infection model yields virulence data often <br />                                             predictive higher-<span id='am-154' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-155' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> mammals19. This model used evaluate the <br />                                             efficacy antimicrobials phage therapy various Gram-negative, Gram-positive <br />                                             fungal pathogens19. Administration &#934;RGNeae EHEC-infected G. mellonella <br />                                             larvae significantly improved survival treatment off-target &#934;RGN control (log- <br />                                             rank test, P &lt; 0.001) (Fig. 3b). Moreover, &#934;RGNeae significantly effective than <br />                                             chloramphenicol treatment, EHEC strain resistant (log-rank test, P &lt; 0.05) <br />                                             (Supplementary Fig. 8 Supplementary Table 1). Although &#934;RGNeae treatment was <br />                                             inferior carbenicillin, bacteria susceptible (Supplementary Fig. 8 and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Supplementary Table 1), data support RGNs viable alternatives cases where <br />                                             bacteria highly resistant existing antibiotics. Improvements delivery efficiency with <br />                                             &#934;RGNeae expected improve treatment efficacy outcome. <br />  <br />                                             In addition implementing targeted antimicrobial therapies, RGNs used modulate <br />                                             composition complex bacterial populations (Fig. 4). Current therapies use a <br />                                             prebiotic, probiotic drug modify human microbiota demonstrated potential for <br />                                             alleviating various disease states, therapies remain poorly characterized terms of <br />                                             off-target effects specific mechanisms act20. In concert host <br />                                             range delivery vehicle, RGN activity selectively remove bacteria specific <br />                                             genomic content. This reduce prevalence unwanted genes, including antibiotic <br />  <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                            Page 6 <br />  <br />  <br />                                             resistance virulence loci, metabolic pathways bacterial communities without <br />                                             affecting bystanders. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             As proof-of-principle experiment &#8216;bacterial knockdowns&#8217; using RGNs, constructed <br />                                             synthetic consortium comprising phage-susceptible E. coli strains differential <br />                                             antibiotic resistance profiles. We used &#946;-lactam&#8211;resistant E. coli EMG2 pNDM-1, <br />                                             quinolone-resistant RFS289 (gyrAD87G) chloramphenicol- resistant CJ236. Application <br />                                             &#934;RGNndm-1 elicited &gt;400-fold greater killing EMG2 pNDM-1, compared control, <br />                                             leaving RFS289 CJ236 cell populations intact. Treatment &#934;RGNgyrAD87G <br />                                             resulted &gt;20,000-fold greater killing RFS289, compared control, a <br />                                             concomitant reduction EMG2 pNDM-1 CJ236 (Fig. 4). These results demonstrate that <br />                                             RGNs selectively knockdown bacteria contain target DNA sequences, thereby <br />                                             allowing remaining nontarget bacteria dominate population. Adapting this <br />                                             approach tuning endogenous microbiota accomplished delivering RGNs in <br />                                             vivo means broad-host-range phages phage cocktails, conjugative plasmids. <br />                                             An appropriately targeted bacterial knockdown approach used functional studies <br />                                             complex microbiota complement additive therapies, probiotics, for <br />                                             microbiome-associated diseases clearing specific niches removing defined genes from <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             bacterial populations. <br />  <br />                                             In light rising tide antibiotic resistance, engineered cellular viral <br />                                             therapeutics potential biological solutions infectious disease resurged. By <br />                                             repurposing parts developed nature, synthetic biologists designed artificial gene <br />                                             circuits antimalarial production21 engineered probiotics22 phage therapeutics to <br />                                             eradicate biofilms23 potentiate antibiotic activity24,25. Here, demonstrate that <br />                                             transmissible CRISPR-Cas systems act platform programmable antimicrobials <br />                                             harness site-specific cleavage induce cytotoxicity, activate toxin-antitoxin systems, <br />                                             resensitize bacterial populations antibiotics modulate bacterial consortia. This work <br />                                             complements recent finding <span id='am-157' about='obo:NCBITaxon_666' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_662 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="Vibrio choleae" datatype="xsd:string"></span><span id='am-161'  ></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="Liquidivibrio cholerae" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholerae-asiaticae" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="Vibrio comma" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="Bacillo virgola del Koch" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-asiaticae" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Microspira comma" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasRelatedSynonym" content="Pacinia cholerae-asiaticae" datatype="xsd:string"></span><span id='am-169' property="rdfs:label" content="Vibrio cholerae" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae-asiaticae" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasRelatedSynonym" content="Kommabacillus" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholera" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="PMID:9272984" datatype="xsd:string"></span>Vibrio cholerae</span> phage, ICP1, encodes own <br />                                             CRISPR-Cas counteract host-encoded phage defense locus26, CRISPR- <br />                                             Cas constructs transformed electrocompetent cell populations incompatible with <br />                                             cells contain cognate target sequences11,27,28. In contrast studies, we <br />                                             CRISPR-Cas technology applied situ removal undesired <br />                                             genes microbial populations vivo treat infection absence artificial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             selection. Moreover, demonstrate RGNs used activate plasmid-borne <br />                                             toxin-antitoxin systems recently attractive antimicrobial strategy29. In <br />                                             addition validating antimicrobial activity, demonstrate potential applications of <br />                                             RGNs deletion plasmids cells detection DNA elements to <br />                                             single-nucleotide resolution using DNA-damage-responsive reporter. <br />  <br />                                             Because CRISPR-Cas systems widely conserved bacteria <span id='am-50' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span>archaea</span>, isolation, <br />                                             development optimization delivery vehicles required creation RGNs <br />                                             capable targeting additional strains, including multidrug-resistant pathogens as <br />                                             key members endogenous microbiota. These next-generation phage systems be <br />                                             designed goal avoiding anticipated shortcomings, including host range <br />                                             limitations, phage resistance concerns immune reactions (Supplementary <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                          Page 7 <br />  <br />  <br />                                             Discussion). Delivery systems function higher organisms enable RGNs to <br />                                             modulate prevalence specific genes wild-type populations30. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Owing modularity simplicity CRISPR-Cas engineering, libraries of <br />                                             multiplexed &#934;RGNs rapidly constructed simultaneously target plethora of <br />                                             antibiotic resistance virulence determinants modulate composition complex <br />                                             microbial communities. The addition facile, sequence-informed, rational design field <br />                                             dominated time- cost-intensive screening broad-spectrum, small- <br />                                             molecule antibiotics potential reinvigorate pipeline new antimicrobials. <br />  <br />                                ONLINE METHODS <br />                                Strains culture conditions <br />                                             Unless noted, bacterial cultures grown 37 &#176;C <span id='am-45' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span>-Bertani (LB) <br />                                             medium (BD Difco). Bacterial strains used study listed Supplementary Table 2. <br />                                             Where indicated, antibiotics added growth medium following final <br />                                             concentrations: 100 &#956;g/ml carbenicillin (Cb), 30 &#956;g/ml kanamycin (Km), 25 &#956;g/ml <br />                                             chloramphenicol (Cm), 100 &#956;g/ml streptomycin (Sm) 150 ng/ml ofloxacin (Ofx). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Strain construction <br />                                             E. coli EMG2 SmR generated plating overnight culture E. coli EMG2 LB <br />                                             +Sm. Spontaneous resistant mutants re-streaked LB+Sm isolated colony <br />                                             picked used recipient conjugation multidrug resistance plasmids. <br />                                             Overnight cultures EMG2 SmR (recipient), E. coli CDC1001728 (donor pNDM-1) <br />                                             K. pneumoniae K6 (donor pSHV-18) washed sterile PBS 100 &#956;l of <br />                                             donor recipient spotted LB agar plates incubated 37 &#176;C overnight. <br />                                             Transconjugants harvested scraping cells 1 ml sterile PBS plating onto <br />                                             LB+Sm+Cb. <br />  <br />                                             The chromosomal integrations blaNDM-1 blaSHV-18 &#946;-lactamase genes and <br />                                             generation EMG2 gyrAD87G performed &#955;-Red recombineering using pSIM9 <br />                                             system31. Templates integration nonessential lacZYA locus generated by <br />                                             amplifying blaNDM-1 blaSHV-18 genes lysates CDC1001728 K6 using <br />                                             primers rcD77/78 rcD73/74, respectively. Templates construction EMG2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             gyrAD87G obtained amplifying gyrA RFS289 using primers mmD155/161. <br />  <br />                                Plasmid construction <br />                                             To generate RGN plasmids (Supplementary Fig. 2), created intermediate vector <br />                                             pZA-RGN&#216;, lacks CRISPR locus. The tracrRNA PL(tetO-1) promoter were <br />                                             synthesized (Genewiz) amplified using primers mmD98/99, cas9 amplified from <br />                                             pMJ806 (ref. 7) using primers mmD74/75, vector backbone amplified from <br />                                             pZA11G using primers mmD82/83. Each PCR product purified ligated Gibson <br />                                             assembly32. To create final backbone vector RGN plasmids, pBBR1 origin, <br />                                             chloramphenicol resistance marker, tL17 terminator, CRISPR locus cloning site were <br />                                             amplified intermediate vector pBBR1-MCS1-tL17 using mmD151/154, digested <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                            Page 8 <br />  <br />  <br />                                             NheI SacI-HF, ligated pZA-RGN&#216; digested SacI-HF AvrII to <br />                                             create pZB-RGN&#216;. Digestion vector PstI-HF XbaI allowed insertion <br />                                             assembled CRISPR loci. The tracrRNA pRGNndm-1 plasmid created by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             amplification pRGNndm-1 mmD162/163, ClaI digestion, self-ligation. The <br />                                             Cas9D10A mutant plasmid constructed site-directed mutagenesis of <br />                                             pRGNndm-1 primers mmD108/109 KAPA HiFi PCR kit (KAPA Biosystems). <br />  <br />                                             The CRISPR loci constructed isothermal annealing ligation short, <br />                                             single-stranded oligonucleotides (Integrated DNA Technologies). Each spacer repeat <br />                                             piece built corresponding oligo duplex connected adjacent pieces 6-bp <br />                                             overhangs. In addition, terminal repeats designed contain 17-bp extension <br />                                             comprised BsaI restriction site generate overhang allowed insertion the <br />                                             pUC57-Km-crRNA&#216; backbone vector synthesized Genewiz. The oligos used build <br />                                             RGN listed Supplementary Tables 3 4. <br />  <br />                                             To assemble CRISPR loci, 500 pmol sense antisense oligos given duplex <br />                                             annealed boiling 10 min 99 &#176;C cooled room temperature. 300 pmol of <br />                                             annealed duplex combined 15 U T4 polynucleotide kinase (Affymetrix), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             400 U T4 DNA ligase (NEB), T4 ligase buffer (NEB) ddH2O volume 20 &#956;l. <br />                                             Following incubation 25 &#176;C 1 h, reaction products purified using Qiagen <br />                                             QIAquick PCR Purification Kit. Purified products digested 3 h BsaI-HF and <br />                                             re-purified using QIAquick. To prepare crRNA backbone vector, pUC57-Km-crRNA&#216; <br />                                             amplified using primers mmD104/105, subsequently digested BbsI generate <br />                                             compatible overhangs, ligated assembled CRISPR loci. Positive clones the <br />                                             CRISPR loci digested entry vector using PstI-HF XbaI ligated into <br />                                             pZB-RGN&#216; digested enzymes create final RGN plasmids. <br />  <br />                                             Phagemid vector pZEf-gfp created previously adding f1 origin amplified from <br />                                             yeast shuttle pRS series33 pZE22-gfp12. The RGN constructs consisting genes <br />                                             encoding tracrRNA, Cas9 sequence-targeting crRNA amplified single <br />                                             product respective pRGN vectors using KAPA HiFi polymerase (Kapa <br />                                             Biosystems) primers rcD169/183 digested AvrII XmaI (New England <br />                                             Biolabs). These inserts ligated backbone derived amplifying the <br />                                             kanamycin resistance cassette, ColE1 replication origin f1 origin required for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             packaging M13 particles pZEf-gfp primers rcD184/185 digesting with <br />                                             enzymes. E. coli DH5&#945;Pro transformed ligated plasmids sequence <br />                                             verification plasmid purification. <br />  <br />                                             The pZE-blaNDM-1-gfp pZA-blaNDM-1-gfp vectors constructed swapping the <br />                                             antibiotic resistance cassette Lutz-Bujard vectors pZE12G pZA12G12. The <br />                                             blaNDM-1 gene amplified lysate CDC1001728 using primers mmD8/9, the <br />                                             PCR product digested SacI-HF XhoI. The digested product ligated into <br />                                             Lutz-Bujard vectors digested enzymes. <br />  <br />                                             The PemI antitoxin complementation plasmid pZA31-pemI created amplifying <br />                                             pemI coding sequence pSHV-18 mmD253/254. The PCR product was <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                            Page 9 <br />  <br />  <br />                                             digested BamHI KpnI ligated large fragment pZA31G digest with <br />                                             enzymes. The SOS-responsive pZA3LG reporter plasmid derived from <br />                                             pZE1LG34 swapping origin replication antibiotic resistance marker with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             pZA31G using AatII AvrII restriction enzymes. <br />  <br />                                             Mobilizable RGNs created amplifying R1162 replication origin oriT <br />                                             using mmD266/267. The chloramphenicol selection marker RGN locus amplified <br />                                             pRGNndm-1 pRGNshv-18 mmD247/248. PCR products digested with <br />                                             SpeI XmaI, ligated used transform E. coli S17-1 &#955;pir create donor cells <br />                                             used matings. <br />  <br />                                Minimum inhibitory concentration (MIC) determination <br />                                             MICs determined broth microdilution using LB broth according CLSI <br />                                             guidelines35. <br />  <br />                                Transformation assays <br />                                             Overnight cultures diluted 1:100 fresh LB grown optical density (OD600) <br />                                             ~0.3&#8211;0.5. Following 15 min incubation ice, cultured cells centrifuged at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             3,200g, pellets resuspended tenth volume TSS buffer (LB, 10% <br />                                             polyethylene glycol, 5% dimethyl sulfoxide, 50 mM Mg2+ pH 6.5)36. A 100 &#956;l aliquot of <br />                                             cells incubated 10 ng RGN plasmid DNA. Plasmids purified the <br />                                             DH5&#945;Pro cloning host using Qiagen QIAprep Spin Miniprep Kit concentration <br />                                             determined using Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen). Following 30 <br />                                             min incubation ice, cells heat shocked 42 &#176;C 30 s, returned ice 2 min <br />                                             recovered 1.5 h 37 &#176;C 1 ml SOC broth (HiMedia). For chromosomal <br />                                             target assay, serial dilutions cells plated LB+Cm select transformants. <br />                                             Plates incubated overnight 37 &#176;C, number colony-forming units (CFU) <br />                                             enumerated following day. Transformation efficiency used assess whether <br />                                             given RGN plasmid toxic cells calculated CFU/ml &#956;g DNA <br />                                             transformed (Supplementary Fig. 2a). <br />  <br />                                             For episomal target assay (Supplementary Fig. 2b), following recovery, cultures were <br />                                             washed fresh LB, diluted 1:100 LB supplemented chloramphenicol select for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             transformants incubated 16 h 37 &#176;C. Samples washed sterile PBS, serially <br />                                             diluted plated LB+Cm LB+Cm+Cb analyzed flow cytometry <br />                                             (Supplementary Fig. 4). Colonies enumerated following day plasmid loss was <br />                                             inferred calculating ratio CbR+CmR CFUs CmR CFUs. <br />  <br />                                             Overnight cultures RGN transformants diluted 1:100 sterile PBS, aliquoted <br />                                             duplicates 96-well plate immediately assayed using BD LSRFortessa cell <br />                                             analyzer. Cells consistently gated scatter forward scatter across <br />                                             independent biological replicates. Fluorescence measurements performed using 488- <br />                                             nm <span id='am-253' about='obo:NCBITaxon_354775' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-254' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-255'  ></span><span id='am-256' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-257' property="rdfs:label" content="Argon" datatype="xsd:string"></span>argon</span> excitation <span id='am-273' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-274' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-276' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-277'  ></span>laser</span>. The GFP gate laser voltages initially determined using <br />                                             untreated pZE-blaZ-gfp EMG2 cells positive negative controls, respectively, and <br />  <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                         Page 10 <br />  <br />  <br />                                             implemented biological replicates. BD FACSDIVA software used data <br />                                             acquisition analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Sequence analysis <br />                                             Escape mutants transformation assays re-isolated passaging surviving colonies <br />                                             LB+Cm+Cb. DNA isolation escape sequencing analysis either <br />                                             extracting plasmid DNA isolated escape mutants using Qiagen QIAprep Spin <br />                                             Miniprep Kit amplifying integrated target locus using primers mmD9/234 or <br />                                             mmD3/4 blaNDM-1 blaSHV-18, respectively. Sequencing performed Genewiz <br />                                             using primers mmD112&#8211;115/153 rcD11 analysis RGN plasmids and <br />                                             mmD3 mmD234 examination integrated resistance genes. <br />  <br />                                Phagemid purification <br />                                             Phagemids encoding RGNs purified using Qiagen QIAprep Spin Miniprep Kit <br />                                             (Qiagen) used transform E. coli DH5&#945;Pro m13cp helper plasmid for <br />                                             generation phagemid-loaded M13 particles37. Strains inoculated grown <br />                                             overnight 250 ml LB+Cm+Km maintain M13cp phagemid, respectively. Cells <br />                                             pelleted supernatant fluid containing phagemid particles passed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             0.2-&#956;m filter. For purifications &#934;RGNgyrAD87G purification the <br />                                             dose-response curve, M13 phagemid particles precipitated addition 5% <br />                                             polyethylene glycol (PEG-6000) 0.5M NaCl incubation overnight 4 &#176;C (ref. 38) <br />                                             pelleted 12,000g 1 h. Purified phagemid pellets resuspended gently in <br />                                             1/100th volume SM buffer (50 mM Tris-HCl [pH 7.5]), 100 mM NaCl, 10 mM MgSO4) <br />                                             stored 4 &#176;C. For &#934;RGNgyrAD87G purification dose-response curve (Fig. <br />                                             2b), M13 phagemid particles precipitated39 addition concentrated HCl <br />                                             pH 4.2 subsequently pelleted 13,000g 15 min. The phagemid pellet was <br />                                             resuspended 1/100th volume water concentrated NaOH added pH 7.0 to <br />                                             solubilize phagemid particles. Tris-HCl [pH 7.5], NaCl MgSO4 added to <br />                                             reconstitute composition SM buffer. <br />  <br />                                             Titers measured incubating sample dilutions E. coli EMG2 30 min and <br />                                             enumerating transductants plating LB LB+Km. Titers defined TFU100/ml, <br />                                             concentration phagemid ~100% recipient population an <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             equivalent cell concentration transduced. <br />  <br />                                Phagemid kill assays <br />                                             Cultures inoculated grown overnight LB appropriate antibiotics 37 &#176;C <br />                                             shaking. The following day, overnights subcultured 1:100 3 ml LB (no <br />                                             antibiotics) grown 37 &#176;C shaking OD600 reached ~0.8. Cultures were <br />                                             diluted LB ~108 CFU/ml pNDM-1 pSHV-18 assays (Fig. 1b, 2a) ~106 <br />                                             CFU/ml gyrAD87G (Fig. 1c) EHEC assays (Fig. 3a) 245 &#956;l suspension was <br />                                             added 5 &#956;l purified phagemid stock 96-well plate incubated static 37 &#176;C. <br />                                             The number viable cells samples interval time-course 2 h for <br />                                             endpoint assays determined serial dilution spot plating LB, LB+Cb, LB <br />                                             +Km analyze cytotoxicity, plasmid loss phagemid delivery, respectively. Initial <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                          Page 11 <br />  <br />  <br />                                             suspensions diluted plated LB quantify initial bacterial inocula. <br />                                             Colonies enumerated 8&#8211;9 h incubation 37 &#176;C calculate cell viability <br />                                             (CFU/ml) averaged independent experiments. Nonlinear curve fitting the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             time-course exponential decay curve performed using GraphPad Prism. <br />  <br />                                G. mellonella model <br />                                             Larvae model organism G. mellonella19 purchased Vanderhorst Wholesale, <br />                                             Inc. (St. Marys, OH, USA) received final larval instar survival assays. Larvae <br />                                             removed food source, allowed acclimate 24 h room temperature <br />                                             dark, used 4 d receipt. For injections, KDS100 (KD Scientific) or <br />                                             Pump 11 Elite (Harvard Apparatus) automated syringe pump set dispense 10 &#956;l <br />                                             volume flow rate ~1 &#956;l/s 1-ml syringe (BD) 26 G needle (BD). To <br />                                             prepare bacteria injection, overnight culture E. coli O157:H7 43888 F&#8242; was <br />                                             subcultured Dulbecco&#8217;s Modified Eagle Medium (Gibco) 4 h 37 &#176;C shaking <br />                                             OD600 ~0.6. Cultures washed twice PBS diluted concentration ~4 &#215; <br />                                             105 CFU/ml. In accordance studies40, 20 larvae treatment group were <br />                                             randomly selected based size (150&#8211;250 mg) excluded based poor health as <br />                                             evidenced limited activity, dark coloration reduced turgor experiments. Larvae <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             delivered injections blinding PBS bacteria final left <br />                                             proleg. Approximately hour injection, SM buffer, antibiotic &#934;RGN <br />                                             treatment administered final right proleg (Fig. 3c, Supplementary Fig. 8). <br />                                             Larvae incubated 37 &#176;C survival monitored 12 h intervals 72 h, with <br />                                             death indicated lack movement unresponsiveness touch19. Kaplan-Meier <br />                                             survival curves generated analyzed log-rank test using GraphPad Prism. <br />  <br />                                LexA reporter assay <br />                                             Overnight cultures EMG2 WT, EMG2 pNDM-1 EMG2 gyrAD87G containing the <br />                                             SOS-responsive reporter plasmid pZA3LG diluted 1:50 LB incubated with <br />                                             SM buffer, &#934;RGNndm-1 &#934;RGNgyrAD87G MOI ~5 2 h 37 &#176;C <br />                                             (Supplementary Fig. 7). Cultures diluted 1:5 250 &#956;l sterile PBS analyzed <br />                                             using BD LSR Fortessa cell analyzer, above. BD FACSDIVA software used for <br />                                             data acquisition analysis performed using FlowJo software. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Bacterial conjugation <br />                                             Donor recipient strains grown overnight LB appropriate antibiotics diluted <br />                                             1:100 fresh media grown OD600 ~1. Cells pelleted resuspended in <br />                                             sterile PBS, mating pairs mixed donor recipient ratio 340 &#177; 66:1. Mating <br />                                             mixtures pelleted, resus-pended 20 &#956;l PBS spotted nitrocellulose filters <br />                                             placed LB agar plates. Initial bacterial suspensions serially diluted plated LB <br />                                             agar plates quantify initial inocula. Matings proceeded 37 &#176;C 3 h single <br />                                             mixing step. At 90 min, mating mixtures collected vigorously vortexing filters <br />                                             1 ml sterile PBS. Cells pelleted, resuspended 20 &#956;l PBS re-seeded filters <br />                                             incubated remaining 90 min. At end 3 h mating, cells were <br />                                             recovered vigorously vortexing filters 1 ml sterile PBS. Mating mixtures <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                                            Page 12 <br />  <br />  <br />                                             serially diluted PBS plated LB+Cb select total number Cb- <br />                                             resistant recipient cells LB+Cb+Cm select transconjugants. Colonies were <br />                                             enumerated following overnight incubation 37 &#176;C determine viable cell counts and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             averaged independent biological replicates (Supplementary Fig. 1). <br />  <br />                                Synthetic consortia remodeling <br />                                             E. coli CJ236, EMG2 pNDM-1 RFS289 strains grown overnight LB appropriate <br />                                             antibiotics diluted 1:100 fresh LB (no antibiotics) grown OD600 ~0.8. <br />                                             Cultures seeded fresh LB initial mixture contained ~1 &#215; 106 CFU/ml <br />                                             strain 245 &#956;l suspension added 5 &#956;l SM buffer purified <br />                                             &#934;RGNndm-1 &#934;RGNgyrAD87G triplicate 96-well plate spotted LB +Cm, <br />                                             +Sm +Ofx quantify initial concentration CJ236, EMG2 pNDM-1 RFS289, <br />                                             respectively. Samples incubated, plated enumerated phagemid kill <br />                                             assays. The composition synthetic ecosystem treatment condition was <br />                                             determined counting viable colonies plates selective strain data <br />                                             calculated viable cell concentration (CFU/ml) averaged biological <br />                                             replicates (Fig. 4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Data analysis statistics <br />                                             All data analyzed using GraphPad Prism version 6.0 (GraphPad Software, San Diego, <br />                                             CA, USA, http://www.graphpad.com/). <br />  <br />  <br />                                Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                             We like thank R. Meyer (Institute Cell Molecular Biology, University Texas Austin) R1162, <br />                                             D.L. Court (Center <span id='am-119' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-120'  ></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Research, National Cancer Institute Frederick) pSIM9, A.R.M. Bradbury <br />                                             (Los Alamos National Laboratory) M13cp. The authors thank J. Rubens assistance flow cytometry <br />                                             experiments H. Gancz D. Zurawski assistance Galleria infection model. T.K.L. acknowledges <br />                                             support US National Institutes Health (NIH) New Innovator Award (1DP2OD008435), NIH <br />                                             National Centers Systems Biology grant (1P50GM098792), Defense Threat Reduction Agency <br />                                             (HDTRA1-14-1-0007), US Army Research Laboratory US Army Research Office Institute <br />                                             Soldier Nanotechnologies (W911NF13D0001) Henry L. Grace Doherty Professorship Ocean <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Utilization. R.J.C. supported funding NIH/National Institute General Medical Sciences <br />                                             Interdepartmental Biotechnology Training Program (5T32 GM008334), M.M. Howard Hughes Medical <br />                                             Institute International Student Research fellow recipient Fonds recherche Sant&#233; Qu&#233;bec Master&#8217;s <br />                                             Training Award. <br />  <br />  <br />                                References <br />                                             1. Centers Disease Control Prevention. Antibiotic Resistance Threats United States. <br />                                                2013 <br />                                             2. Nordmann P, Dortet L, Poirel L. Carbapenem resistance Enterobacteriaceae: storm! <br />                                                Trends Mol Med. 2012; 18:263&#8211;272. [PubMed: 22480775] <br />                                             3. Barrangou R, et al. CRISPR provides acquired resistance <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-8'  ></span>viruses</span> prokaryotes. Science. <br />                                                2007; 315:1709&#8211;1712. [PubMed: 17379808] <br />                                             4. Garneau JE, et al. The CRISPR/Cas bacterial immune cleaves bacteriophage plasmid <br />                                                DNA. Nature. 2010; 468:67&#8211;71. [PubMed: 21048762] <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                                  Page 13 <br />  <br />  <br />                                             5. Mali P, Esvelt KM, Church GM. Cas9 versatile tool engineering biology. Nat Methods. <br />                                                2013; 10:957&#8211;963. [PubMed: 24076990] <br />                                             6. Deltcheva E, et al. CRISPR RNA maturation trans-encoded small RNA host factor RNase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                III. Nature. 2011; 471:602&#8211;607. [PubMed: 21455174] <br />                                             7. Jinek M, et al. A programmable dual-RNA-guided DNA endonuclease adaptive bacterial <br />                                                immunity. Science. 2012; 337:816&#8211;821. [PubMed: 22745249] <br />                                             8. Rasheed JK, et al. Characterization extended-spectrum <span id='am-89' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-90' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span>-lactamase reference strain, <br />                                                <span id='am-227' about='obo:NCBITaxon_570' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-228'  ></span><span id='am-229' property="oboInOwl:hasDbXref" content="PMID:11411716" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Carter et al. 1999" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasDbXref" content="PMID:10555350" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasRelatedSynonym" content="Hyalococcus" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39823" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasRelatedSynonym" content="&quot;Hyalococcus&quot; Schroeter 1886" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="Klebsiella" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Drancourt et al. 2001" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasDbXref" content="PMID:12635932" datatype="xsd:string"></span>Klebsiella</span> pneumoniae K6 (ATCC 700603), produces novel enzyme SHV-18. <br />                                                Antimicrob Agents Chemother. 2000; 44:2382&#8211;2388. [PubMed: 10952583] <br />                                             9. Rasheed JK, et al. New Delhi metallo-&#946;-lactamase-producing Enterobacteriaceae, United States. <br />                                                Emerg Infect Dis. 2013; 19:870&#8211;878. [PubMed: 23731823] <br />                                             10. Bikard D, Hatoum-Aslan A, Mucida D, Marraffini LA. CRISPR interference prevent natural <br />                                                  transformation virulence acquisition vivo bacterial infection. Cell Host Microbe. <br />                                                  2012; 12:177&#8211;186. [PubMed: 22901538] <br />                                             11. Gomaa <span id='am-262' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-263' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-264' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-265'  ></span><span id='am-266' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-267' property="rdfs:label" content="Aa" datatype="xsd:string"></span>AA</span>, et al. Programmable removal bacterial strains use genome-targeting CRISPR- <br />                                                  Cas systems. MBio. 2014; 5:e00928&#8211;13. [PubMed: 24473129] <br />                                             12. Lutz R, Bujard H. Independent tight regulation transcriptional units <span id='am-220' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-221' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-224'  ></span><span id='am-225' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-226' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli via <br />                                                  LacR/O, TetR/O AraC/I1&#8211;I2 regulatory elements. Nucleic Acids Res. 1997; 25:1203&#8211; <br />                                                  1210. [PubMed: 9092630] <br />                                             13. P&#233;rez-<span id='am-268' about='obo:NCBITaxon_1112527' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_243882 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_94020 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_156536 obo:NCBITaxon_6072'><span id='am-269' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-271'  ></span><span id='am-272' property="rdfs:label" content="Mendoza" datatype="xsd:string"></span>Mendoza</span> D, la Cruz F. Escherichia coli genes affecting recipient ability plasmid <br />                                                  conjugation: any? BMC Genomics. 2009; 10:71. [PubMed: 19203375] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             14. Jacoby GA. Mechanisms resistance quinolones. Clin Infect Dis. 2005; 41(suppl 2):S120&#8211; <br />                                                  S126. [PubMed: 15942878] <br />                                             15. Pennington JM, Rosenberg SM. Spontaneous DNA breakage single living Escherichia coli cells. <br />                                                  Nat Genet. 2007; 39:797&#8211;802. [PubMed: 17529976] <br />                                             16. Hayes F. Toxins-antitoxins: plasmid maintenance, programmed cell death, cell cycle arrest. <br />                                                  Science. 2003; 301:1496&#8211;1499. [PubMed: 12970556] <br />                                             17. Mnif B, et al. Molecular characterization addiction systems plasmids encoding extended- <br />                                                  spectrum beta-lactamases Escherichia coli. J Antimicrob Chemother. 2010; 65:1599&#8211;1603. <br />                                                  [PubMed: 20507859] <br />                                             18. Kaper JB, Nataro JP, Mobley HL. Pathogenic Escherichia coli. Nat Rev Microbiol. 2004; 2:123&#8211; <br />                                                  140. [PubMed: 15040260] <br />                                             19. Desbois AP, Coote PJ. Utility greater wax moth larva (<span id='am-132' about='obo:NCBITaxon_7137' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_40082 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_37573 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_7135 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7136 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-133' property="rdfs:label" content="Galleria mellonella" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasExactSynonym" content="greater wax moth" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="Galleria mellonela" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasRelatedSynonym" content="Galleria melonella" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasAlternativeId" content="NCBITaxon:185836" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-140'  ></span>Galleria mellonella</span>) evaluating the <br />                                                  toxicity efficacy new antimicrobial agents. Adv Appl Microbiol. 2012; 78:25&#8211;53. [PubMed: <br />                                                  22305092] <br />                                             20. Sonnenburg JL, Fischbach MA. Community health care: therapeutic opportunities human <br />                                                  microbiome. Sci Transl Med. 2011; 3:78ps12. <br />                                             21. Paddon CJ, et al. High-level semi-synthetic production potent antimalarial artemisinin. <br />                                                  Nature. 2013; 496:528&#8211;532. [PubMed: 23575629] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             22. Duan F, March JC. Engineered bacterial communication prevents <span id='am-176' about='obo:NCBITaxon_662' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-177' property="oboInOwl:hasDbXref" content="PMID:17978204" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="PMID:1371064" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="Listonella" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:8590667" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasRelatedSynonym" content="Pacinia" datatype="xsd:string"></span><span id='am-182'  ></span><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="Vibrio" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="PMID:4935323" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:21296930" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasAlternativeId" content="NCBITaxon:705" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasRelatedSynonym" content="Microspira" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:7520733" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Vibrio</span> cholerae virulence an <br />                                                  infant mouse model. Proc Natl Acad Sci USA. 2010; 107:11260&#8211;11264. [PubMed: 20534565] <br />                                             23. Lu TK, Collins JJ. Dispersing biofilms engineered enzymatic bacteriophage. Proc Natl Acad <br />                                                  Sci USA. 2007; 104:11197&#8211;11202. [PubMed: 17592147] <br />                                             24. Lu TK, Collins JJ. Engineered bacteriophage targeting gene networks adjuvants antibiotic <br />                                                  therapy. Proc Natl Acad Sci USA. 2009; 106:4629&#8211;4634. [PubMed: 19255432] <br />                                             25. Edgar R, Friedman N, Molshanski-Mor S, Qimron U. Reversing bacterial resistance antibiotics <br />                                                  phage-mediated delivery dominant sensitive genes. Appl Environ Microbiol. 2012; 78:744&#8211; <br />                                                  751. [PubMed: 22113912] <br />                                             26. Seed KD, Lazinski DW, Calderwood SB, Camilli A. A bacteriophage encodes own <br />                                                  CRISPR/Cas adaptive response evade host innate immunity. Nature. 2013; 494:489&#8211;491. <br />                                                  [PubMed: 23446421] <br />                                             27. Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA. RNA-guided editing bacterial genomes <br />                                                  using CRISPR-Cas systems. Nat Biotechnol. 2013; 31:233&#8211;239. [PubMed: 23360965] <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                                    Page 14 <br />  <br />  <br />                                             28. Vercoe RB, et al. Cytotoxic chromosomal targeting CRISPR/Cas systems reshape bacterial <br />                                                 genomes expel remodel pathogenicity islands. PLoS Genet. 2013; 9:e1003454. [PubMed: <br />                                                 23637624] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             29. Williams JJ, Hergenrother PJ. Artificial activation toxin-antitoxin systems antibacterial <br />                                                 strategy. Trends Microbiol. 2012; 20:291&#8211;298. [PubMed: 22445361] <br />                                             30. Esvelt KM, Smidler AL, Catteruccia F, Church GM. Concerning RNA-guided gene drives the <br />                                                 alteration wild populations. Elife. 2014:e03401. [PubMed: 25035423] <br />                                             31. Datta S, Costantino N, Court DL. A set recombineering plasmids Gram-negative bacteria. <br />                                                 Gene. 2006; 379:109&#8211;115. [PubMed: 16750601] <br />                                             32. Gibson DG, et al. Enzymatic assembly DNA molecules kilobases. Nat <br />                                                 Methods. 2009; 6:343&#8211;345. [PubMed: 19363495] <br />                                             33. Sikorski RS, Hieter P. A shuttle vectors yeast host strains designed efficient <br />                                                 manipulation DNA <span id='am-95' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-96' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Genetics. 1989; 122:19&#8211;27. [PubMed: <br />                                                 2659436] <br />                                             34. Dwyer DJ, Kohanski MA, Hayete B, Collins JJ. Gyrase inhibitors induce oxidative damage <br />                                                 cellular death pathway Escherichia coli. Mol Syst Biol. 2007; 3:91. [PubMed: 17353933] <br />                                             35. Clinical Laboratory Standards Institute. Methods Dilution Antimicrobial Susceptibility <br />                                                 Tests Bacteria Grow Aerobically; Approved Standard. 7. Clinical Laboratory <br />                                                 Standards Institute; Wayne, Pennsylvania, USA: 2006. <br />                                             36. Chung CT, Niemela SL, Miller RH. One-step preparation competent Escherichia coli: <br />                                                 transformation storage bacterial cells solution. Proc Natl Acad Sci USA. 1989; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 86:2172&#8211;2175. [PubMed: 2648393] <br />                                             37. Chasteen L, Ayriss J, Pavlik P, Bradbury ARM. Eliminating helper phage phage display. <br />                                                 Nucleic Acids Res. 2006; 34:e145. [PubMed: 17088290] <br />                                             38. Westwater C, et al. Use genetically engineered phage deliver antimicrobial agents bacteria: <br />                                                 alternative therapy treatment bacterial infections. Antimicrob Agents Chemother. 2003; <br />                                                 47:1301&#8211;1307. [PubMed: 12654662] <br />                                             39. Dong D, Sutaria S, Hwangbo JY, Chen P. A simple rapid method isolate purer M13 phage <br />                                                 isoelectric precipitation. Appl Microbiol Biotechnol. 2013; 97:8023&#8211;8029. [PubMed: <br />                                                 23807666] <br />                                             40. Ramarao N, Nielsen-Leroux C, Lereclus D. The insect Galleria mellonella powerful infection <br />                                                 model investigate bacterial pathogenesis. J Vis Exp. 2012; 439210.3791/4392 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. <br />                                             RGN constructs delivered bacteriophage particles (&#934;RGN) exhibit efficient specific <br />                                             antimicrobial effects strains harboring plasmid chromosomal target sequences. (a) <br />                                             Bacteriophage-delivered RGN constructs differentially affect host cell physiology a <br />                                             sequence-dependent manner. If target sequence is: (i) absent, RGN exerts effect; <br />                                             (ii) chromosomal, RGN activity cytotoxic; (iii) episomal, RGN leads (iiia) <br />                                             cell death (iiib) plasmid loss, depending presence absence toxin-antitoxin <br />                                             (TA) systems, respectively. (b) Treatment EMG2 wild-type (WT) EMG2 containing <br />                                             native resistance plasmids, pNDM-1 (encoding blaNDM-1) pSHV-18 (encoding <br />                                             blaSHV-18), SM buffer, &#934;RGNndm-1, &#934;RGNshv-18, multiplexed &#934;RGNndm-1/ <br />                                             shv-18 multiplicity infection (MOI) ~20 showed sequence-dependent cytotoxicity as <br />                                             evidenced strain-specific reduction viable cell counts (n = 3). CFU, colony-forming <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             units. (c) E. coli EMG2 WT EMG2 gyrAD87G populations treated SM buffer, <br />                                             &#934;RGNndm-1 &#934;RGNgyrAD87G MOI ~20, viable cells determined plating <br />                                             Luria-Bertani agar (n = 3). <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. <br />                                             Characterization &#934;RGN-mediated killing antibiotic-resistant bacteria. (a) Time-course <br />                                             treatment EMG2 WT EMG2 pNDM-1 SM buffer, &#934;RGNndm-1 &#934;RGNshv-18 <br />                                             multiplicity infection (MOI) ~20. Data represent fold change viable colonies at <br />                                             indicated time points relative time 0 h. (b) Dose-response curve EMG2 WT EMG2 <br />                                             gyrAD87G treated various concentrations &#934;RGNgyrAD87G 2 h. Data represent <br />                                             fold change viable colonies relative samples treated SM buffer. Error bars (a, b), <br />                                             s.e.m. independent biological replicates (n = 3). (c) EMG2 E. coli containing the <br />                                             natural pNDM-1 plasmid blaNDM-1 gene synthetic expression vector (pZA-ndm1- <br />                                             gfp) treated &#934;RGNndm-1 &#934;RGNshv-18 MOI ~ 20 plated onto <br />                                             nonselective LB LB + carbenicillin (Cb) select blaNDM-1-containing cells. <br />                                             &#934;RGNndm-1 treatment cells harboring pNDM-1 resulted reduction viability the <br />                                             absence selection, &#934;RGNndm-1 treatment cells pZA-ndm1-gfp <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             demonstrated similar cytotoxicity selective pressure maintenance pZA- <br />                                             ndm1-gfp plasmid. (d) EMG2 pSHV-18 complemented cognate antitoxin (pZA31- <br />                                             pemI) PemK toxin control vector (pZA31-gfp) treated SM buffer, <br />                                             &#934;RGNndm-1 &#934;RGNshv-18. Cultures plated LB LB + Cb colonies were <br />                                             enumerated assess cytotoxicity plasmid loss. <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 May 01. <br />                             Citorik et al.                                                                                       Page 17 <br /> NIH-PA Author  <br /> </body></html>