<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Nati. Acad. Sci. USA <br /> Vol. 89, pp. 11416-11420, December 1992 <br /> Biochemistry <br />  <br />  <br /> RAD25 (SSL2), yeast homolog human xeroderma <br /> pigmentosum group B DNA repair gene, essential viability <br />      (excision repir/DNA Icse/pferentlal repir/Cocayne syndrome) <br /> EUN PARK*, SAMI N. GUZDERt, MARCEL H. M. KOKEN*, IRIS JASPERS-DEKKERt, GEERT WEEDAt, <br /> JAN H. J. HOEUMAKERSt, SATYA PRAKASHt, AND LOUISE PRAKASH* <br /> *Department Biophysics, University Rochester School Medicine, Rochester, NY 14642; tDepartment Biology, University Rochester, Rochester, <br /> NY 14627; tDepartment Cell Biology Genetics, Medical Genetics Center, Erasmus University, P.O. Box 1738, 3000 DR Rotterdam, The <br /> Netherlands <br /> Communicated Allan M. Campbell, September 4, 1992 <br />  <br /> ABSTRACT           Xeroderma pi         um (XP) patients                      excision repair (15). Many yeast excision repair genes <br />        y sensitive ultrav t (UV) light suffer                       multifunctional, play diverse roles other <br /> high Inidence skin cancers, defect nuleotde                        cellular processes. For example, RAD3 essential via- <br /> excson repair. The diseas Is gecay h,                                         bility (17, 18) RAD) RADIO finction mitotic <br /> seven cm entat             grops, A-G, identified.                      recombination (19, 20). The RAD3 RAD10 proteins have <br /> Homologs human es repair genes ERCCI, XPDC/                                    purified characterized laboratory. RAD3 a <br /> ERCC2, XPAC Idenfied In yeast Saccharo-                         single-stranded DNA-dependent ATPase, possesses <br /> myces cerieviae. Since homolog human XPBC/ERCC3                             DNA DNARNA helicase activities (21-23). Mutational <br /> exited known yeast genes, cloned yeast                           inactivation RAD3 ATPase/helicase activities renders <br />    log using XPBC cDNA hybridization probe. The                           cells UV-sensitive defective excision repair no <br /> yeast h og, RAD25 (SSL2), encodes protein 843 amino                          effect viability (24). The RAD10 protein binds preferen- <br /> acidds (Mr 95,356). The RAD25 (SSL2)- XPBC4ecoded                             tially single-stranded DNA promotes renatura- <br />           share 55% Idential 72% conserved amino acid                         tion complementary strands DNA (25). <br /> rI~neg, <span id='am-58' about='obo:NCBITaxon_73106' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_9108 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_54375 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Otis" datatype="xsd:string"></span>otis</span> rm contain-                                  The yeast RAD3, RADIO, RADi4 genes homologs <br /> in conserved DNA heicase sequence motif. A n e                                human XPDC/ERCC2, ERCCJ, XPAC genes, respec- <br /> mutation <span id='am-53' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> 799 deletes 45 C-terminal amino acid                   tively (6-9, 16). Since known yeast genes showed <br /> residues RAD25 (SSL2) confers UV senitivity. <span id='am-48' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> mutation                     homology human XPBC/ERCC3 gene (desa <br /> shows episi genes exciin repair group, Wh                             XPBC paper), cloned yeast homolog using <br /> as        ic se UV seit               occurs com-                 XPBC cDNA hybridization probe. The yeast homolog has <br /> bined mutation ge DNA repair pathways,                           cloned independently suppressor blocked trans- <br /> indcating RAD25 (SSL2) f               excision repair                lation initiation presence stem-loop structure <br /> not repai pathways. We RAD25 (SSL2)                       5&apos; untranslated region HIS4 mRNA, those <br /> is essential gene. A mutation Lysm2 residue      e                 workers named gene SSL2 (26). A mutation the <br /> in conserved Walker type A nu               big motif                      C-terminal region protein encoded yeast gene <br /> lethal, suggesting esential role putative RAD2S (SSL2)                  renders cells UV-sensitive, mutation shows epistasis <br /> ATPase/lONA helicase activity viability.                                       genes excision repair group. Because the <br />                                                                                   requirement yeast homolog DNA repair, follow- <br /> In human, xeroderma pigmentosum (XP) autosomal                              ing convention nomenclature yeast DNA repair <br /> recessive disorder, cells XP individuals defec-                      genes, named gene RAD25. Our studies a <br /> tive incision step excision repair. Cell fusion studies                 genomic deletion mutation indicate RAD25 essential <br /> have indicated existence 7 excision-deficient XP com-                      gene. The RAD25-encoded protein shows homology DNA <br /> plementation groups, A-G (1-3); addition, 10                          helicases, change residue Lys3 arginine the <br /> complementation groups identified UV-                             Walker type A nucleotide-binding sequence motif results in <br /> sensitive excision-defective rodent cell lines (4-6). Five                         lethality, suggesting putative RAD25 ATPase/DNA <br /> human genes-ERCCI, ERCC2, ERCC3, ERCC5,                                        helicase activity indispensable viability.&#167; <br /> ERCC6-have cloned cross-complementation UV <br /> sensitivity rodent cell lines (7-11). In addition, human <br /> XPAC gene isolated complementation excision                                        MATERIALS AND METHODS <br /> defect XP-A cell lines (12). ERCCI does complement                               Closing Nudeotide Sequence Deterninatlon S. <br /> any XP cell lines (13), ERCC2 ERCC3                                cereWiae RAD25 Gene. Southern blot hybridization S. <br /> restore normal excision repair XP-D XP-B mutant cell                           cerevisiae genomic DNA low-stringency salt condi- <br /> lines (9, 14), respectively, indicating degree overlap                        tions 5&apos; 3&apos; XPBC cDNA probe spanning first <br /> between XP ERCC genes.                                                        two-thirds XPBC open reading frame, <br />    At 10 genes involved excision repair yeast                     respectively, indicated extensive sequence homology of <br /> <span id='am-16' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Mutations 6 genes-                              XPBC cDNA yeast DNA. Several clones yeast <br /> RADI, RAD2, RAD3, RAD4, RADIO, RAD14-cause                                        XPBC homolog identified low-stringency hybridiza- <br /> extreme UV sensitivity defect incision step                           tion genomic S. cerevisiae A EMBL3 library these <br /> excision repair (15, 16). The genes-RAD7,                                  probes (27). <br /> RAD16, RAD23, MMSJ9-affect proficiency of <br />                                                                                       Abbreviations: XP, xeroderma pigmentosum; CS, Cockayne syn- <br />  The publication costs article defrayed page charge           drome. <br />  payment. This article marked &quot;advertisement&quot;                &#167;The sequence reported paper deposited the <br />  accordance 18 U.S.C. &#167;1734 solely indicate fact.                      GenBank <span id='am-68' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72'  ></span>data</span> base (accession no. L01414). <br />                                                                               11416 <br />             Biochemistry: Park et aL                                                 Proc. Natl. Acad. Sci. USA 89 (1992)    11417 <br />      For nucleotide sequence determination, DNA fiagments             mutation, generating rad25 Arg392 CEN URA3 plasmid <br />    cloned M13 derivative phages sequence         pPM147. <br />    strands RAD25 determined dideoxy <br />   chain-termination method (28, 29).                                                           RESULTS <br />      Yeast Strains Media. The S. cerevisiae strains used in <br />   study given Table 1. Media conditions              Nudeotide Sequence RAD2S. Nucleotide sequence anal- <br />   determining UV survival previously described (27).           ysis indicated RAD25 encoded protein 843 amino <br />      Planids Generation Yeast rad25 Mutant Strains. The        acids Mr 95,356 (Fig. 1). While paper prepa- <br />   RAD25 gene inserted HindIll-BamHI fragment            ration, nucleotide sequence yeast homolog of <br />   CEN URA3 low-copy plasmid YCp5O YCp5O                XPBC published (26). Our data agree reported <br />   derivative TRPJ gene replaced URA3 gene,           sequence, differences amino acid positions 9 <br />   generating <span id='am-73' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> pEP23 pEP39, respectively. pEP22            48, results indicate seine leucine, <br />   contains rad257gam mutant allele cloned URA3           respectively, instead proline serine. In addition, we <br />   integrating vector YIp5 Sph I site          noted nucleotide differences two <br />   removed. The rad257&quot;am mutation generated inser-             sequences. <br />   tion T residue codons 797 798 site-                Homology Between Yeast RAD25 Human XPBC- <br />   directed oligonucleotide mutagenesis 1.46-kilobase           Encoded Proteins. The protein encoded yeast RAD25 <br />   (kb) Sph I-BamHI DNA fragment encoding terminal 249             gene contains N-terminal extension 50 amino acids not <br />   amino acids 843-residue RAD25 protein. Codons                present XPBC protein; otherwise, proteins <br />   797-799 wild type RAD25 gene GCC-GTT-AGA,                share extensive homology, 55% identical residues, and <br />   rad257gam mutant gene, corresponding             72% similar residues identical conserved changes <br />   codons GCC-TGT-TAG-A, T inserted res-              grouped (Fig. 1). RAD25 resembles XPBC in <br />  idue. The rad25gam mutation changes codon 798 GTT               containing seven conserved sequence motifs are <br />  (Val) TGT (Cys), followed amber nonsense         proteins bind hydrolyze nucleotides and <br />  codon, TAG. Thus, addition change amino acid 798,        possess DNA RNA helicase activities (30). The homology <br />  mutant protein differs wild-type protein            proteins particularly striking these <br />  lacking 45 C-terminal amino acid residues. The rad25S99am       domains (Fig. 1). Domain I contains conserved Walker <br />  mutation used replace wild-type RAD25 allele          type A sequence Gly-Xaa-Gly-Lys-Thr/Ser [GXGK(T/S)J <br />  strain DBY747 transformation Ura+ linearized             (31). At residues 389-393, RAD25 contains sequence <br />  plasmid pEP22 followed screening Ura- 5-fluoro-           GAGKT, residues 343-347 XPBC, correspond- <br />  orotic acid (5-<span id='am-11' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-15'  ></span>FOA</span>) resistance UV sensitivity, yield-       ing sequence GAGKS. The conserved lysine (K) the <br />  ing haploid strain, EPY88. Genomic deletion mutations        GK(T/S) sequence involved interactions a <br />  RADI, RAD6, RAD52 genes rad2S7&quot;a,,, strain           phosphoryl group bound nucleotide (32). Domain II <br />  EPY88 described (16), confirmed allelism            contains Walker type B sequence motif, to <br />                                                                      conservative hydrophobic amino acids followed by <br />  tests known rad mutations, designated EPY90,               aspartate residue (31). This sequence occurs residues <br />  EPY91, EPY92, respectively (Table 1).                           479-493 RAD25, residues 432-446 XPBC. <br />     To generate genomic deletion mutation RAD25,             Aspartate sequence proposed interact with <br /> used pEP24, pUC19-based plasmid contains                 Mg2+ MgATP, hydrophobic residues serve to <br /> flanking RAD25 sequences 2.2-kb Nco I-Stu           exclude water ATP reaction center (32). The role of <br /> I DNA fragment located entirely RAD25 open               conserved sequence motifs helicases known. <br /> reading frame replaced yeast LEU2 gene               RAD25 Contains Sequence Motifs Characteristic DNA <br /> a 2.1-kb DNA fragment. Transformation Leu- yeast                 Helicas Not RNA Helicases. Both DNA RNA <br /> strains Leu+ HindIII-Bgl II fiagment,               helicases sequences motif I differ <br /> contained 726 nucleotides 5&apos; flanking sequence 59            sequences motifs II VI. Fig. 2 shows com- <br /> N-terminal codons RAD25, yeast LEU2 gene followed            parison helicase sequence motifs I, II, VI among <br /> by 46 C-terminal RAD25 codons, 590 nucleotides 3&apos;            RAD25, RAD3, eIF4A. The S. cerevisiae RAD3 protein <br /> flanking sequence RAD25, produced genomic rad25lA              single-stranded DNA-dependent ATPase, pos- <br /> mutation. This rad25 mutation lacked 738 843 codons          sesses DNA DNARNA helicase activities (21-23). <br /> present RAD25. The rad25 <span id='am-63' about='obo:NCBITaxon_95269' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_85772 obo:NCBITaxon_6656 obo:NCBITaxon_85773 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7399 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-64' property="rdfs:label" content="Arge" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Arge</span> mutation, results            RAD3 does hydrolyze ATP presence RNA <br /> in change Lys392 Arg Gly-Lys-Thr nucleotide-         cofactors RNA helicase activity (23). eIF4A is <br /> binding motif, obtained site-directed oligonucleotide        eukaryotic translation initiation factor possesses an <br /> mutagenesis 1.94-kb Bal I-Sph I fiagment              RNA helicase activity capable unwinding mRNAs <br /> RAD2S gene replacement corresponding wild-type           (33). eIF4A belongs <span id='am-40' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> proteins, which <br /> RAD25 DNA segment fiagment carrying Arg392               shown possess RNA-dependent ATPase and <br />                  Table 1. S. cerevisiae strains <br />                    Strain                                            Genotype <br />                  EPY59           MATa ade2-1 arg4-8 cyhiOR his4-280 leu2-27 trplA ura3-1 <br />                  EPY60           MATa ade2-1 his4-260 leu2-3,112 trplA ura3-1 <br />                  EPY61           EPY59 x EPY60 (RAD25/RAD25) <br />                  EPY62           EPY61 carrying rad25SA mutation homologs (RAD25/rad25A::LEU2) <br />                  DBY747          MATa his3-AJ leu2-3,112 trpl-289 ura3-52 <br />                  EPY88           DBY747 rad25Sam mutation replacing genomic RAD25 gene <br />                  EPY90           EPY88radJA:: URA3 <br />                  EPY91           EPY88rad6A:: URA3 <br />                  EPY92           EPY88rad52A:: URA3 <br />                     The gene symbol following double colon (::) indicates yeast selectable marker used in <br />                  generating radA mutation. <br />  11418     Biochemistry: Park et al.                                                                                                 Proc. Natd. Acad. Sci. USA 89 (1992) <br />             XP-B                                                                                                  MEKUUR A U ......... D3GK    HR EVEE LD D E EEP[.S <br />                                                                                                                                                          I       hGN FEE AVV                A <br />             RAD25     M       Y k SKGKIFPDML&apos; E[SFfSSDEDA PAISSUDEN YDDNRECLEG <br />                           DVLUIQS                                                                                 RIECUT G A 1    LKKPWKW3T MSRH7AAWS S M NQM K DK A LJOD T h S <br />  <br />              PF-B,    AGKQMDE5GI KVDEYGAKWY                 RLQM PEKD  T_!VAli.                MH FMA              VVYKY   BkM V Cfj TVLE                                      IQSTEY1 <br />             R A D2    PADFPDV S .GMFRSHWF                   SY L    RJR <br />                                                                     PM A IS [S                 ERIIES             [L A E Q 5T= I S SCMIM <br />                                                                                                                                                                             *ElDjI S VL <br />  <br />             XP-B      RKTfTGPD GBfMQ                  C1    V                    _    S    C 11 PeIU    H         &amp;MR E C RY <br />                                                                                                                           NSEGLA EL&gt; &apos;SK KME <br />                                                                                                                                           DEPQ <br />                                                                                                                                              SSGGP Rfi <br />             RAD25     DRV      PEAE       SHI NM&amp; A1                      IITT <br />                                                                            1   QAL91 LIMEN MS. GPM5 IDSDHQVQPFP EtL <br />                                                                                                                  Dv*.QG iWG K PA                                           N       %NCVEa <br />  <br />             XP-B      KSDBPMDLFD          Fr   EQMDKDW      EEEETQTV            V KQ IM IE L    QINHLE            LIRA                 IAVLfl <br />                                                                                                                                  SVINTVN                             R         t       V <br />             RAD25     SAVED ..                 GADNEREM DDDIDAVHM IIAN               SV VS             EID        V E          IHR WLD <br />                                                                                                                                  HR    S I*   5   TTQ <br />                            I          --              r                   la <br />             XP B      _SW             3VlV                R C   LMGCTNJAS      JEMKAMK M ESIJDDSQIC RlMIAUD K . PITCSVAIflULGHfT ClDWEAERMEW EKT QUML <br />               RAD25~~T=                        ~~I        S VI                  MMRQMLQ CMLQPENCA V~NMEMF QTESGLVVM 3VANDRNW HDSDQKMDF LITGRMFID <br />            XP-B       IKR~L <br />                       EIT        T IV QMC <br />                        V VW I|IUV ASA I VMA                            P <br />                                                                                  &apos; <br />                                                                                      Vl <br />                                                                               S [DWER jE SLTAM RVAKTR <br />                                                                                                                               NNIEP~            NSP                                 MM <br />            RA D 2 5                                                               Q f TH&apos;I <br />                                                                                   GK                 Q  F     F <br />                                                        r-     -- ---mVI <br />                                                                      I <br />                                           FCFEXNU A=F5JK <br />                                                          Z5LKMGEF MPUSItGW <br />                                           YIGUES!YJA 1lIRLNEY <br />             XP-B      =IR        C                                         IOIMlwH P     I I FNV          G  E3 <br />             RAD 25    ORS                                          IPlQjW WONLIMYW DO      L mfIIhh CTC <br />                                                                                                      5    Y IFI <br />  <br />             XP-R <br />              AD25G <br />             RAD2 5 <br />                      MVAVEK <br />                    M3IMR        - <br />                                    E _QR <br />                         R R N D13G F  1 Y H;A 1W <br />                                                  *W                                                                   M K3AflE. E FSTFS <br />                                                                                                                        HMHMN I P NIMYA S P R   RREEILQKA AKDLDEEGI vGDDGEFDGSSG <br />                                                                                                                                                R REMEMZL K N E E Aj]G ILMS G D DC3D N S V/ GS <br />  <br />             XP-B~ <br />             RAD25 <br />                       Sr A S RM       TTM <br />                                      ..   SMSrBDj <br />                                            .....T   MS1HSSRSiA <br />                       G NGHKJS KAVRGEESLI GLAGE3MAAGSTNKNfE <br />                                                                                                PS   .FRFRp <br />                                                                                                LKEM. IR <br />                                                                                                              . <br />  <br />  <br />                                                                                                                   KMYYBNLKI <br />  <br />   FIG. 1. Homology human XPBC (9) S. cerevisiae RAD25 proteins. The amino acid sequences aligned using <br /> the BESTFIT program Genetics Computer Group, Madison, WI. Identical residues indicated white lettering black background. <br /> The helicase consensus sequences (30) denoted horizontal lines Roman numerals (I, <span id='am-6' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-7' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ia</span>, II-VI) sequences. <br /> RNA helicase activities, share sequence                                                         splice junction intron/exon, produces <br /> motif DEAD domain II HRIGR domain VI (34, 35).                                                         frameshift resulting protein alteration 43 <br /> These motifs absent known DNA helicases,                                                            C-terminal residues (9). To examine role RAD25 in <br /> including RAD3. RAD25 lacks DEAD HRIGR                                                              DNA repair yeast, constructed mutation, rad2Sm,,.,, <br /> sequence motifs, suggesting RAD25 possesses DNA-                                                        inserting T residue codons 797 798. This <br /> dependent ATPase DNA helicase activities                                                         frameshift mutation changes amino acid residue 798 from <br /> the ability hydrolyze ATP nucleotides                                                         valine cysteine, amber nonsense codon <br /> presence RNA RNA helicase activity.                                                                    position 799, encoded protein predicted lack the <br />    RAD25 Ib Esenti Gene. Mutations human XPBC                                                          45 C-terminal amino acids. As expected, rad2S799,,,. <br /> gene equivalent gene rodent cell lines cause                                                      mutation conferred UV sensitivity yeast, indicating an <br /> defect excision repair UV-damaged DNA. However,                                                            involvement RAD25 DNA repair. In S. cerevisiae, the <br /> since genomic deletion mutations yeast DNA                                                            UV DNA repair genes belong epistasis <br /> repair genes uncovered additional functions iden-                                                       groups. The RAD3 group consists excision repair genes; <br /> tified previous studies missense mutations,                                                       genes RAD6 epistasis group function postreplication <br /> made genomic deletion mutation RAD25 examined                                                           repair UV-damaged DNA damage-induced muta- <br /> its effect cell viability. The RAD+/RAD+ diploid strain                                                       genesis, genes RADS2 group ore involved in <br /> EPY61 transformed linear DNA fragment                                                            double-strand break repair recombination. Since RAD25 <br /> contained 5&apos; 3&apos; flanking sequences RAD25                                                     essential cell viability, role DNA repair be <br /> a small portion open reading frame                                                       general just involvement excision repair. To <br /> internal 2.2-kb segment RAD25 open reading frame                                                          examine possibility RAD2S functions than <br /> was replaced yeast LEU2 gene. The generation                                                         pathway DNA repair, constructed double mutant <br /> deletion mutation RAD25 gene homologs                                                       combinations rad2S7wmam radA mutation each <br /> and replacement LEU2 confirmed Southern                                                           epistasis groups compared UV sensitivity of <br /> blot analysis (data shown). Genetic analysis tetrads                                                      single double mutants. In combination excision- <br /> following sporulation RAD25/rad25A::LEU2 diploid                                                          defective radlAo mutation, observed epistatic relation- <br /> strain EPY62 revealed asci viable                                                          ship, UV survival rad257m,,m radlA double <br /> spores. Of 91 tetrads examined, 70 produced viable                                                           mutant the radl A Mutant (Fig. 3A). <br /> spores 21 produced viable spore. In addition,                                                        In contrast, synergistic decline UV survival occurred <br /> spores recovered EPY62 UV-resistant                                                           rad25mam mutation combined the <br /> were Leu+, indicating rad2SA::LEU2 spores invia-                                                        rad6A (Fig. 3B) rad52A (Fig. 3C) &apos;mutation. These <br /> ble. In contrast, RAD+/RAD+ strain EPY61 produced                                                            results indicate involvement RAD25 excision repair <br /> tetrads containing viable spores. Thus, RAD25                                                      repair processes. The lower level UV <br /> is essential gene.                                                                                            sensitivity rad25m. mutation totally <br />    Asment RAD25 Excision Repair Epis                                                                   excision defective radlA mutation reflect fact that <br /> Group. The human XPIJBE mutation, causes defect                                                          RAD25 absolutely required excision repair. Alter- <br /> in excision repair, insertion 4 base pairs (bp)                                                 natively, rad2Sm7&quot;am mutation leaky of <br />                                  DOMAIN                                                                          II                                     VI <br />                                  RAD3 <br />                                  RAD25 <br />                                                       fI LEMP S <br />                                                           IVLP      K <br />                                                                           F    VLL ---------VSKDSIVI F-AHNI                   G <br />                                                                                                              ---------AMRHAAQC* <br />                                                                                 VGI---------GREWGFIILDEIVHVV---------SRRQEAQRIGR <br />                                                                                                                                                              LgKD <br />                                                                                                                                                              LRIRKR <br />                                  BIF-4A           I       JAQAQ5SfI~K3A TFA---------PKYIKMFV                             DE   ---------NRENYIHRTIZG*R          FGR <br />  <br />  <br />   FIG. 2. Comparison RAD25 RAD3, DNA helicase, eukaryotic translation initiation factor eIF4A, RNA helicase, in <br /> domains I, II, VI seven conserved domains superfamilies helicases. Identical residues shared proteins <br /> are boxed. The GKT box present proteins, DEAD box HRIGR sequence motifs present eIF4A RAD3 <br /> or RAD25 denoted stars. <br />              Biochemistry: Park et aL                                                        Proc. Natl. Acad. Sci. USA 89 (1992)        11419 <br />  <br />                                   A                               B                           C <br />                                100                          100 <br />  <br />                                 10 <br />                                                             10 <br />  <br />                           F       I <br />                                                               1 <br />                           2O     .1 <br />  <br />                                                              .1 <br />                                 .01 <br />  <br />                                .001                         .01 <br />                                       0   2 4   6 8 10 12         0   2 4     6 8 10 12       0   2   4   6 8 10 12 <br />  <br />                                                                   UV irradiation   (Jim2) <br />     FIG. 3. Survival UV irradiation RAD+ strain DBY747 isogenic derivatives bearing deletions various RAD genes. (A) *, <br />   RAD+; o, rad25,9am; *, radlA; o, rad25S9qamradIA. (B) *, RAD+; o, rad2579Sam; *, rad6A; o, rad2579gam rad6A. (C) *, RAD+; o, rad2579am; <br />   *, rad52A; o, rad25Samrad52A. Survival curves represent average experiments strain. <br />   low-level frameshifting resulting small amounts normal                 able like XPBC, RAD25 required incision. <br />   protein.                                                                     Thus, products seven genes-RADJ, RAD2, <br />      Lethality rad2SArg392 Mutation. Mutation Lys48               RAD3, RAD4, RADIO, RAD14, RAD25-may indis- <br />   arginine GKT Walker type A sequence motif RAD3                     pensable incision UV-damaged DNA yeast. <br />   results loss ATPase, DNA helicase, DNA-RNA                            In addition requirement excision repair, RAD25 is <br />   helicase activities ability bind ATP                    essential cell viability. In respect, RAD25 resembles <br />   single-stranded DNA (23, 24). This mutation does affect                  RAD3, required excision repair cell <br />   cell viability function RAD3 renders cells UV-                    viability. The DNA repair viability functions RAD3 are <br />   sensitive defective excision repair (24). To examine              mutationally separable, suggesting different segments of <br />   biological role putative ATPase/DNA helicase activity                 RAD3 interact different protein components carrying <br />  RAD25, mutated Lys392 GKT sequence                            functions. Since rad257wam mutation affects <br />  arginine. Since unable obtain genomic replace-                  DNA repair viability function, RAD25 also, <br />  ment wild-type RAD25 allele rad25 Arg392                     domains affecting functions reside different <br />  mutant allele, considered possibility mutation              portions protein. The C terminus RAD25, of <br />  lethal. To test this, RAD25/rad25A::LEU2                        XPBC, specifically involved interactions the <br />  diploid strain EPY62 transformed RAD25                   protein components incision complex. <br />  gene rad25 Arg392 mutation carried CEN URA3                       Both XPBC RAD25 contain conserved domains <br />  plasmids pEP23 pPM147, respectively. As expected,                        proteins bind hydrolyze nucleotides. A <br />  genetic analysis following sporulation EPY62 transformed                  comparison domains RAD25 present in <br />  pEP23 (RAD25) revealed tetrads                       RAD3, possesses DNA DNARNA helicase ac- <br />  viable spores, 35% (41/116) spores                     tivities, eIF-4A, RNA helicase, suggests that <br />  Leu+, carried RAD25 plasmid pEP23, indi-                    RAD25 (and XPBC) possesses DNA helicase activity. <br />  cating restoration viability rad25A::LEU2 spores               In aspect too, RAD25 appears resemble RAD3. <br /> by RAD25 plasmid. In contrast, 60 tetrads analyzed                 However, studies indicate RAD3 puta- <br /> from EPY62 carrying rad25Arg392 plasmid pPM147,                     tive RAD25 DNA helicase activities differ regard their <br /> contained viable spores,                       roles DNA repair viability. A mutation Lys4 to <br /> spores Leu+, indicating rad25 Arg392 mutant                    arginine Walker type A consensus sequence RAD3 <br /> gene does complement inviability defect                       results protein devoid ATPase, DNA helicase, or <br /> rad25A::LEU2 spores. Sixty-three percent (60/96)                      DNA-RNA helicase activity (23, 24). The rad3 Arg48 mutation <br /> RAD25 spores contained rad25 Arg392 plasmid pPM147,                      effect viability confers extreme sensitivity <br /> and spores grew normally, indicating rad25                    UV light defect excision repair (24). In contrast, <br /> Arg392 mutant gene effect viability wild-type                   shown study, similar conservative change the <br /> RAD25 cells. The rad25 Arg392 mutation, therefore, reces-                 Lys392 residue arginine Walker type A nucleotide- <br /> sive lethal. In summary, results suggest vital role            binding motif RAD25 lethal, implicating indispensable <br /> putative ATPase/DNA helicase activity RAD25.                              role RAD25 ATPase/DNA helicase activities viability. <br />                                                                                 In addition suffering characteristic symptoms <br />                         DISCUSSION                                          XP, XP-B XP-D patients exhibit clinical features <br />                                                                             DNA repair disorder, Cockayne syndrome (CS) <br /> The yeast RAD25 gene shows high degree homology                     (3, 9). Since XP CS disorders extremely rare, <br /> the human XPBC gene, genes resemble                    simultaneous occurrence patients likely <br /> in role DNA repair. A frameshift mutation XPBC                  result mutation XPBC XPDC gene. CS <br /> that alters 43 C-terminal amino acids renders cells UV-                 occur accompaniment XP. CS pa- <br /> sensitive defective incision step excision repair             tients suffer severe dwarfism mental retardation, <br /> (9). The epistatic relationship rad2S799am mutation                  premature aging. CS cells sensitive UV <br /> encoding protein lacking 45 C-terminal amino acids               light defective preferential repair actively <br /> genes excision repair group genes                  transcribed genes (36). The association XP-B XP-D <br /> function repair pathways consistent                    CS suggests role XPBC XPDC proteins and <br /> involvement RAD25 excision repair. It highly prob-                 yeast homologs providing link transcrip- <br />  11420    Biochemistry: Park et al.                                                   Proc. Nadl. Acad. Sci. USA 89 (1992) <br /> tion excision repair. In eukaryotes prokaryotes,               Vermeulen, W., Weeda, G., Klein, B., Hoebmakers, J. H. J., <br /> the transcribed strand repaired efficiently                Bootsma, D. &amp; Westerveld, A. (1989) Mutat. Res. 217, 83-92. <br /> nontranscribed strand (37, 38), UV damage              14.   Lehmann, A. R., Hoejmakers, J. H. J., van Zeeland, A. A., <br /> inhibitory transcription stalling RNA poly-                Backendorf, C. M. P., Bridges, B. A., Collins, A., Fuchs, <br /> merase complex damage site (39, 40). A mechanism                     R. P. D., Margison, G. P., Montesano, R., Moustacchi, E., <br /> that couples displacement stalled RNA polymerase                     Natarajan, A. T., Radman, M., Sarasin, A., Seeberg, E., <br /> with assembly repair complex damaged                      Smith, C. A., Stefanini, M., Thompson, L. H., van der Schans, <br />                                                                             G. P., Weber, C. A. &amp; Zdzienicka, M. Z. (1992) Mutat. Res. <br /> transcribed strand enable cells repair transcribed             273, 1-28. <br /> strand efficiently. The putative RAD25 DNA helicase              15.   Prakash, S., Sung, P. &amp; Prakash, L. (1990) The Eukaryotic <br /> activity collaborate RAD3 helicase role.                   Nucleus, eds. Strauss, P. R. &amp; Wilson, S. H. (Telford, Cald- <br />    Since yeast homolog XPBC isolated                well, NJ), pp. 275-292. <br /> its ability restore His+ phenotype His4- cells carrying       16.   Bankmann, M., Prakash, L. &amp; Prakash, S. (1992) Nature <br /> an artificial stem-loop structure HIS4 mRNA, role              (London) 355, 555-558. <br /> this gene translation initiation proposed (26).           17.   Higgins, D. R., Prakash, S., Reynolds, P., Polakowska, R., <br /> Such role would, however, require RAD25 contain                   Weber, S. &amp; Prakash, L. (1983) Proc. Natl. Acad. Sci. USA 80, <br /> activity capable unwinding stem-loop structures                     5680-5684. <br /> mRNAs. It remains seen RAD25 contains              18.   Naumovski, L. &amp; Friedberg, E. C. (1983) Proc. Natl. Acad. <br /> an RNA helicase activity, absence characteristic                 Sci. USA 80, 4818-4821. <br />                                                                       19.   Schiestl, R. H. &amp; Prakash, S. (1988) Mol. Cell. Biol. 8, 3619- <br /> RNA helicase sequence motifs RAD25 argues                   3626. <br /> a possibility.                                                        20.   Schiestl, R. H. &amp; Prakash, S. (1990) Mol. Cell. Biol. 10, <br />                                                                             2485-2491. <br />   We thank E. Miller M. Sweet excellent technical assis-      21.   Sung, P., Prakash, L., Weber, S. &amp; Prakash, S. (1987) Proc. <br /> tance, P. Sung reading manuscript, R. Johnson               Natl. Acad. Sci. USA 84, 6045-6049. <br /> sequence alignment. This work supported Grants DE-FG02-        22.   Sung, P., Prakash, L., Matson, S. W. &amp; Prakash, S. (1987) <br /> 88ER60621 Department Energy CA41261                     Proc. Natd. Acad. Sci. USA 84, 8951-8955. <br /> CA35035 National Institutes Health Dutch       23.   Bailly, V., Sung, P., Prakash, L. &amp; Prakash, S. (1991) Proc. <br /> <span id='am-43' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Society (Project IKR 88-02 90-20), European Com-             Natl. Acad. Sci. USA 88, 9712-9716. <br /> munity (Contract B16-141-NL), Council Medical Re-         24.   Sung, P., Higgins, D., Prakash, L. &amp; Prakash, S. (1988) EMBO <br /> search Netherlands Organization Scientific Research              J. 7, 3263-3269. <br /> (Contract 900-501-091).                                               25.   Sung, P., Prakash, L. &amp; Prakash, S. (1992) Nature (London) <br />                                                                             355, 743-745. <br />  1. Cleaver, J. E. &amp; Kraemer, K. H. (1989) The Metabolic Basis     26.   Gulyas, K. D. &amp; Donahue, T. F. (1992) Cell 69, 1031-1042. <br />     Inherited Disease, eds. Scriver, C. R., Beaudet, A. L., Sly,   27.   Reynolds, P., Koken, M. H. M., Hoeomakers, J. H. J., <br />     W. S. &amp; Valle, D. (McGraw-Hill, New York), Vol. 2, pp.                  Prakash, S. &amp; Prakash, L. (1990) EMBO J. 9, 1423-1430. <br />     2949-2971.                                                        28.   Sanger, F., Nicklen, S. &amp; Coulson, A. R. (1977) Proc. Natd. <br />  2. De Weerd-Kastelein, E. A., Keijzer, W. &amp; Bootsma, D. (1972)             Acad. Sci. USA 74, 5463-5467. <br />     Nature (London) (New Biol.) 238, 80-83.                           29.   Bigin, M. D., Gibson, T. J. &amp; Hong, G. F. (1983) J. Biol. <br />  3. Vermeulen, W., Stefanini, M., Giliani, S., Hoeomakers,                  Chem. 134, 315-319. <br />     J. H. J. &amp; Bootsma, D. (1991) Mutat. Res. 255, 201-208.           30.   Gorbalenya, A. E., Koonin, E. V., Donchenko, A. P. &amp; Bli- <br />  4. Thompson, L. H., Shiomi, T., Salazar, E. P. &amp; Stewart, S. A.            nov, V. L. (1989) Nucleic Acids Res. 17, 4713-4730. <br />     (1988) Somatic Cell Mol. Genet. 14, 605-612.                      31.   Walker, J. E., Saraste, M., Runswick, M. J. &amp; Gay, N. J. <br />  5. Stefanini, M., Collins, A. R., Riboni, R., Klaude, M., Botta,           (1982) EMBO J. 1, 945-951. <br />     E., Mitchell, D. L. &amp; Nuzzo, F. (1991) Cancer Res. 51,            32.   Fry, D. C., Kuby, S. A. &amp; Mildvan, A. S. (1986) Proc. Natd. <br />     3%5-3971.                                                               Acad. Sci. USA 83, 907-911. <br />  6. Hata, H., <span id='am-1' about='obo:NCBITaxon_871458' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_33361 obo:NCBITaxon_33362 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33317 obo:NCBITaxon_130551 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_36151 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Numata" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Numata</span>, M., Tohda, H., Yasui, A. &amp; Oikawa, A.           33.   Ray, B. K., Laeson, T. G., Kramer, J. C., Cladaras, M. H., <br />     (1991) Cancer Res. 51, 195-198.                                         Grifo, J. A., Abramson, R. D., Merrick, W. C. &amp; Thach, R. T. <br />  7. Van Duin, M., Wit, J., Odijk, H., Westerveld, A., Yasui, A.,         (1985) J. Biol. Chem. 260, 7651-7658. <br />     Koken, M. H. M., Hoeomakers, J. H. J. &amp; Bootsma, D. (1986)        34.   Linder, P., Lasko, P. F., Ashburner, M., Leroy, P., Nielson, <br />     Cell 44, 913-923.                                                       P. J., Nishi, K., Schnier, J. &amp; Slonimski, P. 0. (1989) Nature <br />  8. Weber, C. A., Salazar, E. P., Stewart, S. A. &amp; Thompson,                (London) 337, 121-122. <br />     L. H. (1990) EMBO J. 9, 1437-1447.                                35.   Kim, S.-H., Smith, J., Claude, A. &amp; Lin, R.-J. (1992) EMBO J. <br />  9. Weeda, G., Van Hamm, R. C. A., Vermeulen, W., Bootsma,                  11, 2319-2326. <br />     D., van der Eb, A. J. &amp; Hoeijmakers, J. H. J. (1990) Cell 62,     36.   Venema, J., Mullenders, L. H. F., Natarajan, A. T., Van <br />      777-791.                                                               Zeeland, A. A. &amp; Mayne, L. V. (1990) Proc. Natl. Acad. Sci. <br /> 10. Mudgett, J. S. &amp; MacInnes, M. A. (1990) Genomics 8, 623-                 USA 87, 4707-4711. <br />     633.                                                              37.   Mellon, I., Spivak, G. &amp; Hanawalt, P. C. (1987) Cell 51, <br /> 11. Troelstra, C., Odijk, H., Wit, J., Westerveld, A., Thompson,         241-249. <br />     L. H., Bootsma, D. &amp; Hoeijmakers, J. H. J. (1990) Mol. Cell.      38.   Mellon, I. &amp; Hanawalt, P. C. (1989) Nature (London) 342, <br />     Biol. 10, 5806-5813.                                                    95-98. <br /> 12. Tanaka, K., Miura, N., Satokata, I., Miyamoto, I., Yoshida,       39.    Sauerbier, W. &amp; Hercules, K. (1978) Annu. Rev. Genet. 12, <br />     M. C., Satoh, Y., Kondo, S., Yasui, A., Okayama, H. &amp;                 329-363. <br />     Okada, Y. (1990) Nature (London) 348, 73-76.                      40. Selby, C. P. &amp; Sancar, A. (1990) J. Biol. Chem. 265, 21330- <br /> 13. Van Duin, M., Vredeveldt, G., Mayne, L. V., Odijk, H.,                2133 <br /> </body></html>