<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Proc. Natl. Acad. Sci. USA <br />    Vol. 89, pp. 10355-10359, November 1992 <br />    Genetics <br />  <br />  <br />    NIPI, gene required nuclear transport yeast <br />          (<span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>/nudear import/mitochondrial inport/cytochrome c) <br />    ZHENYU Gu*, RICHARD P.                 MOERSCHELLtt,         FRED SHERMANt&#167;, AND DAVID S. GOLDFARB*1 <br />    *Department Biology, University Rochester, Rochester, NY 14627; Departments tBiophysics &#167;Biochemistry, University Rochester Medical <br />    School, Rochester, NY 14642 <br />    Contributed Fred Sherman, April 23, 1992 <br />  <br />    ABSTRACT            Cytochrome c nuclear localization                        transport, essential cellular process. Mutants se- <br />    signal added N terminus mistargeted nucleus,                      lected increased growth 30&apos;C lactate medium and <br />    resulting yeast strain deficient mitochondrial cy-                          subsequently screened temperature-conditional growth at <br />    tochrome c. Reversion strain allowed isolation                       370C glucose medium. Among mutants <br />    temperature-conditional mutants defective nuclear trans-                         defects nuclear targeting translocation. Here, we <br />    port, demonstrated mutants, nip)-),                       isolation characterization nip1-] <br />    shown defective nuclear accumulation LacZ                         mutant cloning sequence NIPJ+ gene. 11 <br />   protein containing nuclear localization signal yeast <br />   ribosomal protein L29. The NIP1P gene cloned shown                                                     METHODS <br />   encode 93,143-Da protein. Furthermore, epitope- <br />   labeled NIPi protein migrated SDS/polyacrylamide gels                                 Nomenclature. CYCI+ CYCI denote, respectively, the <br />   mass -100,000 Da shown immunofluo-                               wild-type allele chromosomal locus encoding yeast <br />   rescence localize mainly cytoplasm. NIP1P shown                         <span id='am-85' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>-1-cyto c. The cycl-31 allele causes complete deficiency <br />   essential gene gene disruption experiments. In-                           nonsense/frameshift mutation corresponding to <br />   triguingly, NIPi seine-rich acidic N-terminal region                      amino acid position 4 (7). The cycl-1004 allele, denoted <br />   similar regard N-terminal region previously                   cycl-NLS paper, contains &lt;5% normal holo- <br />   described nuclear localization signal-binding protein, NSR1.                         iso-1-cyto c N-terminal region contains NLS <br />                                                                                        SV40 large T antigen; protein encoded cycl- <br />    Scores gene products likely involved                            NLS denoted NLS-cyto c. Holo-NLS-cyto c denotes the <br />   nucleocytoplasmic trafficking macromolecules. The 125-                            form NLS-cyto c contains heme group required for <br />   MDa nuclear pore complex (NPC) contain &gt;50                                 function spectral properties; apo-NLS- <br />   distinct polypeptides (ref. 1; G. Blobel S. Rout, personal                       cyto c denotes form NLS-cyto c lacks heme <br />   communication). Although significant progress                               group. NIPJP nipi-] denote, respectively, wild-type <br />   higher eukaryotes cell biology, biochem-                            allele mutant allele; NIP]-HA denotes the <br />   istry, structure transport apparatus (2), use                     functional allele NIP] contains epitope the <br />   genetics elucidate process nuclear transport                     influenza hemagglutinin protein (HA). NLS-/3-gal denotes <br />   infancy (3).                                                                 <span id='am-48' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span>Escherichia coli</span> LacZ protein [8-galactosidase (J3-gal)] <br />      We genetic selection yeast Sac-                         NLS yeast ribosomal protein L29 (6). <br />   charomyces cerevisiae isolation mutants                             Yeast Strains <span id='am-95' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span>. The major yeast strains used in <br />   defective nuclear import proteins. The selection                       study B-7528 (MATa cycl-31 cyc7-67 ura3-52 <br />   based fact yeast cells utilize lactate sole                 lysS-10), B-8106 (MATa cycl-NLS cyc7-67 ura3-52 lys5-10), <br />  carbon source mitochondrial cytochrome c (cyto c) levels                          B-8302 (MATa cycl-NLS nip1-1 cyc7-67 ura3-52 lys5-10), <br />  fall 10%o normal level. Also,                               B-8303 [MATa cycl-NLS nip1-] cyc7-67 ura3-52 lys5-10 <br />  N-terminal region yeast iso-l-cyto c required                          (pNLS-lacZ)], B-8305 [MATa cycl-NLS cyc7-67 ura3-52 <br />  mitochondrial import (4), alterations                       lysS-10 (pNIP1-HA)]. These strains completely lack iso-2- <br />  region significantly affecting import function                     cytochrome c cyc7-67 deletion. <br />  protein. We altered N-terminal region corre-                            The major plasmids used YCp5O, centromere- <br />  spond nuclear localization signal (NLS) simian                        carrying (CEN) shuttle vector (8); pNLS-lacZ, denoted <br />  virus 40 (SV40) large T antigen. The T-antigen NLS                         pNLS-ElZ Underwood Fried (6), containing the <br />  shown target proteins efficiently yeast nuclei (refs. 5                   NLS-lacZ gene; pNIP1, constructed insertion 3.6- <br />  6; Z.G., unpublished results). Because N-terminal NLS                           kilobase (kb) HindI1 fragment, encompassing NIP)+ <br />  apparently functionally dominant native mito-                           gene, HindI11 site YCp5O; pNIPl-HA, derived <br />  chondrial import signal, located internally                         pNIP1 containing NIPJ-HA gene. <br /> protein, NLS-cyto c misdirected nucleus, causi,                              Cyto c Levels. Relative amounts cyto c determined <br />  &gt;20-fold reduction mitochondrial cyto c conseque, ly                         low-temperature (-1960C) spectroscopic examination of <br /> diminished growth lactate medium.                                                 intact yeast (9) grown derepressed conditions (10), and <br />     Our strategy isolate temperature-conditional r. u-                        comparisons strains having known amounts iso-1- <br /> tants defects nuclear transport resulted hip ier                     cyto c. <br /> mitochondrial levels NLS-cyto c. These strains shotuld <br /> show elevated holo-cyto c levels improved growth                            Abbreviations: NPC, nuclear pore complex; NLS, nuclear localiza- <br /> on lactate medium, grow restric-                         tion signal; cyto c, cytochrome c; HA, hemagglutinin; SV40, simian <br /> tive temperatures conditional nuclear                           virus 40; a-gal, B-galactosidase. <br />                                                                                      tPresent address: Institute Protein Research, Osaka University, <br />                                                                                       3-2 Yamadaoka, Suita, Osaka 565, Japan. <br /> The publication costs article defrayed page charge           ITo reprint requests addressed. <br /> payment. <span id='am-38' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> article marked &quot;advertisement&quot;                IThe sequence reported paper deposited the <br /> in accordance 18 U.S.C. &#167;1734 solely indicate fact.                      GenBank <span id='am-90' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94'  ></span>data</span> base (accession no. L02899). <br />  <br />                                                                              10355 <br />  10356    Genetics: Gu et al.                                                   Proc. Natl. Acad. Sci. USA 89 (1992) <br />   Construction Altered Genes. The cycl-NLS mutation            KG1&apos; P-&apos;KKKR.KVEDPKG1&apos; direct oligonucleotide <br /> was constructed transforming strain B-7528 directly       transformation yeast strain B-7528 incorporate the <br /> oligonucleotide 0L89.204 (5&apos;-TTCACACACTAAATTAAL                   NLS SV40 large T antigen (underlined) (21). The N-ter- <br /> CLCCGAAGAAAAAAAGGAAGGTTGAAGATC-                                   minal proline assigned position -1 fusion <br /> CGAAAGGTGCTAC-3&apos;), using method Moerschell et              protein residue longer wild type. The resulting <br /> al. (7). This method allows recovery cycl alleles     cycl-NLS strain B-8106 contained &lt;5% normal level of <br /> produce 1% normal level activity      holo-cyto c (Table 1). Because N-terminal region cyto <br /> iso-1-cyto c. The sequence verified PCR amplification      c dispensable function mitochondrial import (4), the <br /> and DNA sequencing. pNIP1-HA constructed ligating          simplest explanation extreme deficiency cycl- <br /> 0L91.201 (5&apos; -TGTACCCATACGACGTCCCAGAC-                            NLS strain apo-NLS-cyto c misdirected the <br /> TACGCTG-3&apos;) complementary oligonucleotide                 nucleus heme attachment, occurs the <br /> OL91.202 pNIP1 cleaved unique Pvu II site,            mitochondria (22). Cyto c covalently crosslinked syn- <br /> resulting insertion 9-amino acid epitope (YPYD-       thetic peptide NLSs imported tissue culture nuclei by <br /> VPDYA; ref. 11) amino acid position 598.                       receptor-mediated mechanism (23), and, yeast, NLS- <br />    Genetic Analysis Molecular Techniques. Previously          cyto cl gene fusion product imported nuclei (24). <br /> described methods media used testing                Isolation nip Mutations. The impaired growth lactate <br /> genetic analysis general yeast mutations (12, 13) cyc   medium ofthe cycl-NLS strain (Table 1) permits isolation <br /> mutations specific iso-1-cyto c (10, 14).          revertants restored levels holo-cyto c. The resto- <br /> Standard YPD medium, denoted glucose medium               ration occur types distinct mutations: <span id='am-71' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Cis" datatype="xsd:string"></span>cis</span> <br /> paper, synthetic media used growing testing yeast     mutations CYCI locus destroy NLS the <br /> strains described (13). Oligonucleotides           cycl-NLS allele, trans mutations somehow <br /> prepared described (7).                                        impair nuclear localization enhance mitochondrial import. <br />    Cloning Sequencing NIP]+ Gene. NIP]I+           Further, trans mutations expected exhibit <br /> cloned complementing ura3-52 marker strain        temperature-sensitive growth glucose medium. <br /> B-8302 YCp5O single-copy genomic bank (8). The Ura+           The cycl-NLS strain B-8106 treated UV light and <br /> transformants suspended sterile water streaked        plated lactate medium 30TC. Revertant colonies were <br /> on glucose plates, incubated 37TC. Plasmids         subcloned glucose medium subsequently tested for <br /> complementing nip)-) presumptive NIP)+ transform-          growth glucose medium 30&#176;C 37&#176;C. Subclones <br /> ants transferred yeast E. coli analyzed     exhibiting temperature-conditional growth exam- <br /> restriction endonucleases. After number plasmids        ined holo-cyto c content permissive restrictive <br /> various deletions examined, 3.6-kb HindIII restriction     temperatures. Desired mutants, illustrated cycl- <br /> fragment shown complement nip)-) defect.               NLS nip)-) strain Table 1, enhanced holo-cyto c levels <br />    NIP) + sequenced inserting 1.6-kb HindIll-    conditional growth glucose medium higher temper- <br />  Sal I fragment 2.0-kb HindIII-Sal I fragment        atures. A total 6 candidates having desired character- <br />  M13 mpl8 mpl9 <span id='am-76' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> (15). A series unidirectional       istics uncovered examination 200 revertants. <br />  nested deletions exonuclease III (16). DNA         Only these, B-8302, chosen analysis. <br />  sequencing performed described (17). Sequences       The analysis meiotic progeny heterozygous <br />  analyzed University Wisconsin Genetics Com-          B-8302 crosses revealed single-gene mutation, nip)-), that <br />  puter Group program (18).                                        caused temperature-sensitive growth glucose me- <br />    Immunofluorescence Microscopy. Cells culture medium          dium enhanced holo-cyto c levels (Table 1). Genetic <br />  fixed 4% formaldehyde room temperature 2         analysis revealed degree nip)-) defect was <br />  hr. Spheroplasts prepared zymolyase glusu-          modified single gene heterozygous crosses and <br />  lase immobilized polylysine-coated slides (6). Anti-       nip)-) unlinked cycl-NLS (data shown). <br />  body incubations performed room temperature          Characteristics nip)-). cycl-NLS cells containing the <br />  2 hr. For NLS-,3-gal localization, cells containing         nip)-) allele grew better host strain lactate <br />  pNLS-lacZ incubated 370C 2 hr induction        medium 30&#176;C holo-cyto c levels elevated <br />  370C medium containing 2% galactose. 3-gal in-          (Table 1). Because holo-NLS-cyto c levels increased in <br />  duced approximately level normal (1-hr             nip)-) cells 33&#176;C 350C, temperature-conditional <br />  induction) nip)-) mutant (2-hr induction) cells deter-     growth glucose medium probably directly a <br />  mined enzyme activity (19). Mouse anti-f3-gal (Promega),       defect nuclear transport allowed apo-NLS-cyto <br />  used 1:200, stained rhodamine-conjugated goat         c bypass nucleus enter mitochondria (see below). <br />  anti-mouse IgG dilution 1:400. NIP1-HA labeled           Cellular Localization Molecular Mass NIPI <br />  mouse anti-HA monoclonal antibody 12CA5 (11)              Polypeptide. The cloned NIP] + gene (see below) epitope- <br />  1:100 localized rhodamine-conjugated goat anti-          labeled unique Pvu II site inserting synthetic <br />  mouse IgG 1:200. Nuclei stained 4&apos;,6-diamidino-      oligonucleotide encoding 9-amino acid HA sequence recog- <br />  2-phenylindole (1 pg/ml). <br />     Immunoblot Analysis. Total yeast protein (20) electro-    Table 1. Phenotypes strains cycl-NLS and <br />  phoresed SDS/11o polyacrylamide gels transferred          nipl-l mutations <br />  electrophoretically nitrocellulose filters,                        Growth <br />  blocked PBS (225 mM NaCl/10 mM Na2HPO4, pH 7.5) <br />  containing 2% Carnation nonfat dry milk. After 3-hr incu-       Pertinent    Lactate       Glucose        Holo-cyto c level, % <br />  bation room temperature incubation primary antibody       genotype      (30&#176;C)    300C     37C     300C     330C     35&#176;(C <br />   12CA5, filters washed PBS/0.05% Tween 20                                                     100      100     100 <br />   developed alkaline phosphatase-conjugated anti-         CYCI+           +         +       + <br />                                                                    cycl-31         0         +       +         0        0        0 <br />   mouse IgG (Bio-Rad).                                                             0                         &lt;5       &lt;5       &lt;5 <br />                                                                    cycl-NLS                  +       + <br />                                                                    cycl-NLS        +         +          0     15       20      25 <br />                          RESULTS                                     nipi- <br />    Expression NLS-cyto c Protein Yeast. The N-terminal         The cycl-31 strain completely lacks iso-1-cyto c. All these <br />  region iso-1-cyto c changed T&apos;EFKAGSAK-               strains lack iso-2-cyto c cyc7-67 deletion. <br />           Genetics: Gu et al.                                                        Proc. Natl. Acad. Sci. USA 89 (1992)             10357 <br />                                      HA <br />                                NIPI NIPI <br />                                                                         Defective Nuclear Import nipl-) Cels. We examined the <br />                                                                      effect nip1-1 lesion nuclear import an <br />                                                                      inducible, nonessential karyophile contained native <br />                                                                      yeast NLS instead mammalian NLS. The E. coli lacZ <br />                                                                      gene fused NLS ribosomal protein L29 produces a <br />                                            4                         protein (NLS-.8-gal) accumulated nuclei NIP] yeast <br />                       97   P                                         (6) (Fig. 3A). In contrast, import NLS-.8-gal was <br />                                                                      defective cycl-NLS nip)-) (Fig. 3B). Depending the <br />                                                                      focal plane, mutant cells displayed perinuclear <br />                                                                      NLS-,3-gal staining. Rarely, wild-type cell displayed (3-gal <br />                       66   o <br />                                                                      staining phenotype interpreted partially <br />                                                                      perinuclear. To achieve similar levels NLS-(3-gal activity <br />                                                                      wild-type mutant cells, necessary induce <br />                       450                                            expression 1 hr wild type 2 hr mutant. We <br />                                                                      determine experiment NLS-(3-gal <br />                                                                      binding simply concentrating vicinity the <br />                                                                      nuclear envelope. After longer induction 3rC (up 6 hr), <br />                                                                      NLS-(3-gal stained cell, including entire <br />                                                                      nucleus (data presented). Relating this, growth of <br />  <br />                                 1    2                               A <br />   FIG. 1. Immunoblot analysis. Lane 1, NIP) strain; lane 2, <br /> NIPJ-HA strain. Markers left kilodaltons. The NIPl-HA <br /> protein (arrowhead) migrated mass 100 kDa. <br />  <br /> nized mouse monoclonal antibody 12CA5 (11). The <br /> epitope-labeled NIPi protein able fully complement the <br /> nip)-) lesion, indicating functional. On immuno- <br /> blots, 12CA5 recognized -100-kDa polypeptide was <br /> absent cells lacking HA sequence (Fig. 1). By <br /> immunofluorescence, epitope-labeled NIP1 localized pre- <br /> dominantly cytoplasm (Fig. 2), possible <br /> that minor fraction associated nucleus. <br />  <br />  <br />  <br />  <br />                                                                       B <br />  <br />  <br />  <br />  <br />   FIG. 2. Localization NIP1-HA epitope fusion cytoplasm. <br /> NIP) strain (a b) NIPI-HA strain (c d) incubated <br /> either mouse anti-HA epitope antibody (12CA5) followed by <br /> rhodamine-conjugated goat anti-mouse IgG, localize NIPl-HA          FIG. 3. Immunofluorescence localization L29 NLS--gal in <br /> epitope fusion (b d), 4&apos;,6-diamidino-2-phenylindole   NIP1+ cells induced 1 hr (A) nip)-) cells induced 2 hr (B). <br /> localize nuclei staining DNA (a c). (x4300.)                  (x3200.) <br />  10358     Genetics: Gu et al.                                                                                      Proc. Nadl. Acad. Sci. USA 89 (1992) <br />            &apos;A <br />         -190   GCTTTTTTTTTCGCATTTTTTTCGCCGTTGAAAAATTTTTCAATGAGAAAAAGAGCTTTGAAAGGAATGAAAAATTGACTACAGGTTCGCAGTTGCACTTGTTAGTAGAGTAGCATAGAA <br />          -70   GGAAAAAAAGGAGAACAAGGCACATATTTGCAATCTTCACACAATAGCCTGCCAAGCCCGAGATCTACGAAAATGTCCCGTTTCTTTTCGTCTAATTACGAATACGATGTAGCCAGTTCT <br />            1                                                                                                   M. S    R   F   F   S   S   N   Y   E   Y   D   V   A   S   S <br />           50 TCATCCGAAGAAGATCTTTTATCTTCGTCTGAAGAAGATTTGTTAAGCTCTTCCTCCTCTGAGTCTGAATTGGACCAAGAATCTGACGACTCCTTTTTCAATGAAAGTGAAAGTGAAAGT <br />           17 s s E E D L L S                   S   S   E   E   D   L   L   S   S   S   S   S   E   S   E   L   D   O E     S   D   D   S   F   F   N   E   S   E   S   E   S <br />          170   GAAGCTGATGTAGACTCCGATGATTCTGATGCAAAGCCTTATGGTCCTGACTGGTTCAAGAAATCTGAGTTCAGAAAACAAGGTGGAGGTTCAAATAAATTTTTGAAAAGCTCTAACTAT <br />           57 E A D V D S D D S D A K P Y G P D W F K K S E F R K 0 G G G S N K F L K S S N Y <br />          290 GATTCCAGTGATGAAGAATCCGATGAAGAAGATGGCAAGAAGGATGTCAAGTCTGCCAAAGAAAAACTATTGGATGAAATGCAAGACGTTTATAATAAGATCTCTCAAGCTGAGAACTCA <br />           97 D S S D E E S D E E D G K K D V K S A K E K L L D E M 0 D V Y N K I S Q A E N S <br />          410 GATGACTGGTTGACTATTTCTAATGAGTTTGATTTGATCTCGCGTCTCTTAGTTAGGGCTCAACAACAAAACTGGGGGACTCCAAATATTTTCATCAAGGTTGTTGCCCAAGTGGAGGAC <br />          137 D D W L T I S N E F D L I S R L L V R A 0 0 0 N W G T P N I F I K V V A 0 V E D <br />          530   GCTGTGAATAATACACAACAAGCTGATTTGAAGAATAAAGCTGTTGCAAGAGCTTATAACACTACAAAGCAAAGAGTCAAGAAAGTTTCTAGAGAAAATGAAGACTCAATGGCTAAATTC <br />          177 A    V N      N   T   0   0   A   D   L   K   N   K   A   V   A   R   A   Y   N   T   T   K   0   R   V   K   K   V   S   R   E   N   E   D   S   M   A   K   F <br />          650 AGAAATGATCCTGAATCATTTGATAAGGAACCAACCGCAGATTTGGATATTTCTGCTAATGGATTCACAATTTCTTCGTCTCAAGGCAATGACCAAGCGGTACAAGAAGATTTCTTCACT <br />          217 R     N   D   P   E   S   F   D   K   E   P   T   A   D   L   D   I   S   A   N   G   F   T   I   S   S   S   Q   G   N   D O     A   V   0   E   D   F   F   T <br />          770 AGATTACAAACAATAATTGACTCAAGAGGTAAGAAGACTGTCAATCAACAATCCTTGATTTCTACTTTGGAGGAGTTATTAACTGTAGCTGAAAAACCATATGAATTCATAATGGCTTAT <br />          257 R L      T I <br />                        Q       D   I       S   R   G   K   K   T   V   N   Q Q     S   L   I   S   T   L   E   E   L   L   T   V   A   E   K   P   Y   E   F   I   M   A   Y <br />          890   TTGACTTTGATTCCATCAAGATTCGATGCCTCAGCTAACCTATCTTACCAACCAATTGATCAATGGAAATCTTCATTCAACGATATTAGTAAATTATTGTCTATTTTAGACCAGACAATT <br />          297 L T L I P S R F D A S A N L S Y Q P I D Q W K S S F N D I S K L L S I L D O T I <br />         1010 GACACCTACCAAGTTAATGAATTTGCTGATCCAATCGATTTCATTGAAGATGAACCTAAAGAAGATTCTGATGGTGTCAAGAGGATTCTGGGTTCCATTTTCTCATTTGTTGAAAGATTA <br />          337 D T Y Q V N E F A D P I D F I E D E P K E D S D G V K R I L G S I F S F V E R L <br />         1130 GATGACGAATTCATGAAATCCCTGTTAAACATCGATCCTCATTCCAGTGATTATTTGATCCGTTTAAGGGATGAACAATCAATCTATAATTTGATCCTAAGAACTCAATTGTACTTTGAA <br />          377 D D E F M K S L L N I D P H S S D Y L I R L R D E Q S I Y N L I L R T 0 L Y F E <br />         1250 GCGACTTTGAAAGATGAACACGACCTAGAAAGAGCATTGACACGTCCATTCGTCAAGAGATTGGATCATATCTACTATAAATCCGAAAATTTGATAAAAATTATGGAAACTGCTGCTTGG <br />          417 A T L K D E H D L E R A L T R P F V K R L D H I Y Y K S E N L I K I M E T A A W <br />         1370 AATATCATACCTGCTCAATTCAAATCTAAATTTACTTCAAAAGACCAGCTCGATTCTGCTGATTATGTCGACAATTTAATAGACGGATTATCGACAATCTTATCCAAGCAAAACAACATT <br />          457 N I I P A Q F K S K F T S K D Q L D S A D Y V D N L I D G L S T I L S K Q N N I <br />         1490 GCTGTTCAAAAACGTGCTATTTTATACAACATTTACTACACTGCATTAAACAAAGATTTCCAAACTGCTAAAGATATGTTACTAACTTCCCAAGTTCAAACAAATATCAACCAATTCGAT <br />          497 A V O K R A I L Y N I Y Y T A L N K D F Q T A K D M L L T S Q V Q T N I N Q F D <br />         1610 TCATCCCTACAAATTTTATTCAACAGGGTTGTTGTTCAATTGGGTCTATCCGCCTTTAAATTATGTTTGATTGAAGAATGTCATCAAATTTTGAATGATCTTCTGTCAAGTTCTCACTTA <br />          537 s S L Q I L F N R V V V O L G L S A F K L C L I E E C H Q I L N D L L S S S H L <br />         1730 AGAGAAATTTTGGGCCAACATTCCCTACACAGAATATCTCTCAATTCTAGTAACAATGCTTCAGCTGATGAGCGTGCTAGACAATGTTTGCCATATCACCAACACATCAATCTCGATTTA <br />          577 R E I L G Q H S L H R I S L N S S N N A S A D E R A R 0 C L P Y H Q H I N L D L <br />         1850 ATCGATGTCGTCTTCTTAACATGTTCCTTATTGATCGAAATTCCAAGAATGACTGCCTTCTATTCCGGTATTAACGTCAACAGAATTCCTTACTCTCCAAAATCCATTCGTCGTTCCTTA <br />          617 I D V V F L T C S L L I E I P R M T A F Y S G I N V N R I P Y S P K S I R R S L <br />         1970 GAACATTACGACAAGTTAAGTTTCCAAGGTCCACCAGAAACTTTAAGAGATTATGTCTTGTTTGCTGCCAAATCAATGCAAAAAGGTAACTGGAGAGACTCTGTTAAATACTTAAGAGAA <br />          657 E H Y D K L S F Q G P P E T L R D Y V L F A A K S M O K G N W R D S V K Y L R E <br />         2090 ATAAAATCTTGGGCTTTATTACCAAACATGGAAACGGTGTTGAATAGTTTAACGGAAAGAGTACAAGTTGAATCTTTGAAGACTTATTTCTTTTCTTTCAAGAGGTTCTATTCAAGTTTT <br />          697 I K S W A L L P N M E T V L N S L T E R V 0 V E S L K T Y F F S F K R F Y S S F <br />         2210 TCTGTTGCTAAACTAGCCGAATTATTTGATCTTCCAGAAAATAAGGTGGTTGAAGTTTTGCAATCTGTTATCGCAGAATTGGAAATCCCAGCCAAATTAAACGACGAGAAGACCATCTTT <br />          737 s V A K L A E L F D L P E N K V V E V L Q S V I A E L E I P A K L N D E K T I F <br />         2330 GTTGTCGAAAAGGGTGATGAAATTACTAAATTGGAAGAAGCAATGGTAAAATTGAACAAAGAATATAAAATCGCTAAAGAACGTCTTAACCCACCATCAAATCGTCGTTGATCAATAAAA <br />          777 V V E K G D E I T K L E E A M V K L N K E Y K I A K E R L N P P S N R R End <br />         2450 TCAAAAACTACTTCGAAAGCATTCTTAACAAAAAATCATTGTAGATAGAAGAATAACATCAGCAACCAAATATAGAAGGAAACTAAGCAGGAGAGTATAAACCATATTCACTTTTGC <br />         2570 TTTTTATTCTTGATATTGCTCATCTTTTTTTTTTTCGATTCTTTTAGCGATCTCTTTATTTTTAGTTTTAGGGTTTGAATCTAATCGTCTTCACTGTACATTACACATAAGCAAATATAT <br />         2690 ATATAGAAAAAAAACTTACCGTAAACACTCTTTATAATATAATACAAACTATTAAACTTTAAGAAAAGTATGATAT <br />  <br />  <br />            B <br />                M   A   K   T T     K V     K   G   N K     K E V K A S K Q                 A S K O A K E E                 K A     K A     V   S   S S     S   S   E   S   S <br />                S   S   S   S S     S S     ES      ES      ES ES ES ES                     SSSSSS SD                       SE      S S     S   S   S S     S   D   S   E   S <br />                E   A   E   T K     K E     E S     K D     S S S S S S D S                 S S D E E E E E                 E K     E E     T   K   K E     E   S   K   E   S <br />                S   S   S   D S     S S     S S     S S     D S E S E K E E                 S N D K K R K S                 E D     A E     E   E   E D     E   D   E   E   S <br />                S   N   K   K O     K N     E   E   T E     E P A- - - <br />  <br />  <br />   FIG. 4. (A) Nucleotide sequence NIPI+ gene deduced amino acid sequence NIPI protein. Nucleotide amino acid <br /> numbers, left margin, start with, respectively, A ATG translation initiator <span id='am-43' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>, initiator methionine. Serine, glutamic, <br /> and aspartic residues N-terminal region underlined. The putative 3&apos;-end-forming signal TATATAT (25), doubly underlined. (B) The <br /> N-terminal region NSR1 protein (26, 27), serine, glutamic, aspartic residues underlined. <br /> nip)-) cells glucose medium stopped 2 hr                                     ura3-52/ura3-52 diploid strain ZY10 transformed with <br /> temperature shift 300C 370C (data presented). The                                  DNA fragment containing construct, denoted <br /> doubling time 300C 5 hr. Thus, block nuclear                                     nip)::URA3. Stable Ura+ transformants selected and <br /> transport absolute nonpermissive growth                                        shown growth defect. Nine tetrads sporu- <br /> temperatures.                                                                                  lated heterozygous strain analyzed. In cases, only <br />    The NIPI Gene. NIP) cloned complementation                                        spores tetrad viable, ofthem were <br /> a nip)-) strain 370C isolated 3.6-kb HindIII                                   Ura-. Microscopic examination revealed nonviable <br /> fragment, contained 813-amino acid open reading                                       spores germinated divided twice produce no <br /> frame encoded 93,143-Da protein (Fig. 4A). Curiously,                                   cells. <br /> NIP1 contains acidic N terminus rich serine, feature <br /> that strongly represented NSR1, yeast protein origi- <br /> nally identified basis binding synthetic NLS                                                         DISCUSSION <br /> peptides (26, 27) (Fig. 4B).                                                                   We described genetic selection, characterization, <br />    NIP1P Is Essential Gene. An 1175-base-pair Xba I-Pvu                                     molecular cloning gene, NIP), required for <br /> II fragment comprising nucleotides 619-1793 NIP)                                        efficient nuclear transport yeast. For selection proce- <br /> deleted replaced selectable marker URA3. The                                      dure, assumed SV40 T-antigen NLS-cyto c fusion <br />               Genetics: Gu et al.                                                         Proc. Natl. Acad. Sci. USA 89 (1992)            10359 <br />  <br />     protein targeted nucleus            scheme used NLS-cyto cl fusion (24). The N-terminal <br />     apo form cyto c unstable cytoplasm (20).              region cyto c modified various sequences <br />     Consistent assumption, strain partially defi-        significantly reducing function (4), cyto <br />     cient mitochondrial holo-cyto c (about 5% wild-type             cl precursor contains typical matrix-targeting signal the <br />     level) exhibits diminished growth lactate medium. The           type known function exposed N <br />     nip)-) mutation, causes increase mitochondrial            terminus. Thus, N-terminal NLS-cyto cl fusion protein <br />     holo-cyto c levels, selected growth lactate medium         probably imported mitochondria the <br />     pursued temperature-sensitive growth           NLS proteolytically removed. In addition, rapid and <br />     glucose medium.                                                    efficient oligonucleotide transformation method been <br />       The localization NLS-P-gal nip)-) cells revealed            developed allows alteration N-terminal region of <br />    defect nuclear transport. The defect, expressed            cyto c, example, inserting various NLSs. This has <br />    best 370C, characterized transient perinuclear            allowed level holo-NLS-cyto c yeast <br />    localization NLS-,8-gal (Fig. 3). Nuclear import           mitochondria inversely proportional relative strength <br />    divided stages: (i) initial NLS-mediated targeting         NLS fused cyto c, providing estimate NLS <br />    nuclear envelope/NPC (ii) ATP-dependent translo-               strength yeast (Z.G., F.S., D.S.G., unpublished re- <br />    cation nuclear envelope (28, 29). When translo-             sults). <br />    cation prevented ATP depletion, chilling, by <br />    inhibitors NPC function, karyophiles accumulate               We acknowledge technical assistance <span id='am-80' about='obo:NCBITaxon_1191644' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_294789 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-81' property="rdfs:label" content="Linda" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83'  ></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Linda</span> Comfort, <br />    nuclear envelope (23, 28, 29). Perinuclear localization is,           Lynne Spitz, Aravinda deSilva help immunofluorescence, <br />    therefore, expected natural intermediate            Xiuwen Liu assistance sequence analysis. This <br />    import pathway. This fact explain wild-type             investigation supported American <span id='am-30' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-31' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Society <br />   cells displayed vague NLS-fi-gal perinuclear staining early         Grant BE-64666 U.S. Public Health Service Research Grant ROI <br />   times following induction. An interpretation nip)-)             GM12702 National Institutes Health. <br />   phenotype cells efficiently target nuclear <br />   proteins nuclear envelope partially defective            1. Reichelt, R., Holzenburg, A., Buhle, E. L., Jarnik, M., Engel, <br />   subsequent step. However, determined                         A. &amp; Aebi, U. (1990) J. Cell Biol. 110, 883-894. <br />   nip)-) allele encodes defective component                 2. <span id='am-1' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Garcia" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>Garcia</span>-Bustos, J., Heitman, J. &amp; Hall, M. N. (1991) Biochim. <br />   nuclear transport apparatus or, alternatively, encodes                    Biophys. Acta 1071, 83-101. <br />   gene controls processes, nuclear envelope               3. Silver, P. A. (1991) Cell 64, 489-497. <br />                                                                             4. Baim, S. &amp; Sherman, F. (1988) Mol. Cell. Biol. 8, 1591-1601. <br />   assembly (24), indirectly affect protein import.               5. Nelson, M. &amp; Silver, P. (1989) Mol. Cell. Biol. 9, 384-389. <br />   Another <span id='am-35' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> mutations produce conditional                 6. Underwood, M. R. &amp; Fried, H. M. (1990) EMBO J. 9, 91-99. <br />   increases mitochondrial cyto c increase             7. Moerschell, R. P., Tsunasawa, S. &amp; Sherman, F. (1988) Proc. <br />    mitochondrial import NLS-cyto c. Although mu-                        Nati. Acad. Sci. USA 85, 524-528. <br />    tation similar selection (3), rule        8. Rose, M. D., Novick, P., Thomas, J. H., Botstein, D. &amp; Fink, <br />    possibility nip)-) strain nuclear                G. R. (1987) Gene 60, 237-243. <br />                                                                             9. Sherman, F. &amp; Slonimski, P. P. (1964) Biochim. Biophys. Acta <br />    transport NLS-,O-gal protein clearly defective. <br />    If nip)-) defect mitochondrial import,                   90, 1-15. <br />   nuclear import NLS-fi-gal unaffected.                       10. Sherman, F., Stewart, J. W., Jackson, M., Gilmore, R. A. &amp; <br />       The 813-amino acid sequence NIP1 reveals obvious                     Parker, J. H. (1974) Genetics 77, 255-284. <br />                                                                           11. Wilson, I., Niman, H., Houghten, R., Cherenson, A., Con- <br />   membrane-spanning domains apparent endoplasmic retic-                       nolly, M. &amp; Lerner, R. (1984) Cell 37, 767-778. <br />   ulum targeting signals, likely NIPi              12. Sherman, F., Fink, G. R. &amp; Hicks, J. B. (1987) Methods in <br />   membrane protein. By immunofluorescence, NIP1-HA                           Yeast Genetics (Cold Spring Harbor Lab., Cold Spring Harbor, <br />   protein localized predominantly cytoplasm. We                    NY). <br />   know NIPi protein associated            13. Sherman, F. (1991) Methods Enzymol. 194, 3-21. <br />   nuclear envelope, role nuclear transport              14. Sherman, F., Jackson, M., Liebman, S. W., Schweingruber, <br />   easily understood. Also, rule remote                         A. M. &amp; Stewart, J. W. (1975) Genetics 81, 51-73. <br />  possibility insertion  <br /> </body></html>