<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />   Disruption of the <span id='am-39' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-40' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span>Candida</span> albicans CYB5 Gene Results in Increased <br />                           Azole Sensitivity&#8224; <br />  <br />              Sterol synthesis in <span id='am-30' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-31'  ></span><span id='am-32' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span>fungi</span> is an aerobic process requiring molecular oxygen and, for several cytochrome- <br />           mediated reactions, aerobically synthesized heme. Cytochrome b5 is required for sterol C5-6 desaturation and <br />           the encoding gene, CYB5, is nonessential in <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Cyb5p and Ncp1p (cytochrome P-450 <br />           reductase) appear to have overlapping functions in <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> organism, with disruptions of each alone being viable. <br />           The cytochrome P-450 reductase phenotype has also been shown to demonstrate increased sensitivity to azole <br />           antifungals. Based on this phenotype, the CYB5 gene in the human pathogen <span id='am-45' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-46' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span>Candida albicans</span> was investigated <br />           to determine whether the cyb5 genotype was viable and would also demonstrate azole sensitivity. Sequential <br />           disruption of the CYB5 alleles by direct transformation resulted in viability, presumably conferred by the <br />           presence of a third copy of the CYB5 gene. Subsequent disruption procedures with a pMAL2-CYB5 rescue <br />           cassette and a CYB5-URA3 blaster cassette resulted in viable cyb5 strains with no third copy. The C. albicans <br />           CYB5 gene is concluded to be nonessential. Thus, the essentiality of this gene and whether we observed two or <br />           three alleles was dependent upon the gene disruption protocol. The C. albicans cyb5 strains produced a sterol <br />           profile containing low ergosterol levels and sterol intermediates similar to that reported for the S. cerevisiae <br />           cyb5. The C. albicans cyb5 shows increased sensitivity to azoles and terbinafine, an inhibitor of squalene <br />           epoxidase, and, unexpectedly, increased resistance to morpholines, which inhibit the ERG2 and ERG24 gene <br />           products. These results indicate that an inhibitor of Cyb5p would not be lethal but would make the cell <br />           significantly more sensitive to azole treatment. <br />  <br />  <br />   The fungal end product sterol, ergosterol, and its biosyn-                   coded by CYP51, also referred to as ERG11 and ERG16), the <br /> thetic pathway are the targets of the majority of the antifungal               cytochrome P-450-dependent desaturation step at C-22 (en- <br /> compounds currently in use for human infections and agricul-                   coded by CYP61, also referred to as ERG5), and the cyto- <br /> tural applications. The primary <span id='am-53' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of compounds used in                     chrome b5 requiring C-5 desaturation (encoded by ERG3). In <br /> human infections is the azoles, drugs that inhibit the C-14                    the case of the cytochrome P-450 steps, two distinct enzymes <br /> demethylation of the pathway intermediate lanosterol. Over-                    are involved (11, 12), but each requires the same <span id='am-110' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> <br /> use of the azoles, especially in immunocompromised patients,                   donor, the NADPH cytochrome P-450 oxidoreductase (en- <br /> has led to increases in the incidence of antifungal resistance                 coded by NCP1). Ncp1p also donates electrons in the pre- <br /> (5). As resistance levels to these compounds rise, the need to                 sterol, oxygen-requiring reaction catalyzed by squalene epoxi- <br /> identify new targets for antifungal therapy expands. Several                   dase (32). The cytochrome b5 step in the pathway utilizes <br /> reactions in the ergosterol biosynthetic pathway of the patho-                 NADH-cytochrome b5 reductase or NADPH-cytochrome c re- <br /> genic fungus Candida albicans have been identified (1, 18, 26)                 ductase as electron carriers in the creation of the double bond <br /> as required for viability and, thus, might be subject to explo-                at C5-6 (25). <br /> ration as new antifungal sites. Inhibition of these steps could                   Sutter and Loper (27) reported that disruption of the Sac- <br /> prove to be effective antifungal interventions because this or-                charomyces cerevisiae NCP1 gene is not lethal despite the fact <br /> ganism cannot import exogenous sterol (19, 29), making sterol                  that two steps in the pathway utilize this electron donor and <br /> biosynthesis obligatory.                                                       that one of the steps, the C-14 demethylation of lanosterol, has <br />   Sterol biosynthesis in fungi is an exclusively aerobic process               proven to be essential based on the fact that disruption of the <br /> because several steps in the ergosterol biosynthetic pathway                   cytochrome P-450 demethylase gene (CYP51) results in non- <br /> require molecular oxygen or heme, which are also synthesized                   viability (17). The presence of an alternative electron carrier, <br /> only under aerobic conditions and used primarily in cyto-                      perhaps cytochrome b5, was postulated to explain the viability <br /> chrome-mediated reactions. Among the latter are the cyto-                      of the ncp1 mutant (13). This substitution of function by cyto- <br /> chrome P-450-dependent step in lanosterol demethylation (en-                   chrome b5 was confirmed in a follow-up study (20). Similarly, <br />                                                                                the gene encoding cytochrome b5 in S. cerevisiae was found to <br />                                                                                be nonessential (28). Disruption of CYB5 generates no growth <br />   * Corresponding author. Mailing address: Department of Biology,              phenotype in a wild-type background but results in lethality <br /> Indiana University Purdue University Indianapolis, West Michigan St.,          when present in a ncp1 background (28). This suggests that <br /> Indianapolis, IN 46202-5132. Phone: (723) 274-0588. Fax: (317) 274- <br /> 2846. E-mail: nlees@iupui.edu. <br />                                                                                NCP1 might provide a reciprocal function for the missing cy- <br />   &#8224; M.B. and N.D.L. dedicate this manuscript to the memory of their            tochrome b5 protein. <br /> mothers, Tillie Bard and Adeline Lees.                                            Ncp1p and Cyb5p appear to have overlapping functions in S. <br />  <br />                                                                         3425 <br />  3426       ROGERS ET AL.                                                                                                   ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                        TABLE 1. C. albicans strains derived from BWP17 generated in this study <br />  Strain                              Disruption protocol                                                               Genotype <br />  <br /> KRC2                              Direct transformation                            CYB5/cyb5::ARG4 <br /> KRC3                              Direct transformation                            cyb5::URA3/cyb5::ARG4/CYB5 <br /> KRC6                              pMAL-CYB5 cassette                               CYB5/cyb5::ARG4 ADE2/ade2::pMAL-CYB5 <br /> KRC7                              pMAL-CYB5 cassette                               cyb5::URA3/cyb5::ARG4 ADE2/ade2::pMAL-CYB5 <br /> KRC9                              URA3-CYB5 blaster                                CYB5/cyb5::HIS1 <br /> KRC10                             URA3-CYB5 blaster                                CYB5/cyb5::HIS1 ERG25/erg25::hisG-URA3-CYB5-hisG <br /> KRC11                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG-URA3-CYB5-hisG <br /> KRC12                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG <br /> KRC13                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG ADE2/ade2::pMAL-CYB5 <br />  <br />  <br />  <br />  <br /> cerevisiae. One of the phenotypes of ncp1 is an increased sen-                        Transformations. E. coli DH5&#9251; transformations were accomplished by stan- <br /> sitivity to azole antifungals (27). This is postulated to be due to                dard protocols (2). C. albicans transformations were carried out as previously <br />                                                                                    described (31). <br /> the increased sensitivity of the substitute electron carrier in the                   DNA sequencing. DNA sequencing was performed in the Biochemistry Bio- <br /> C-14 demethylation reaction and suggests a mechanism of ac-                        technology Facility at the Indiana University School of Medicine. Initial se- <br /> tion for azoles beyond interacting with Cyp51p. A more recent                      quence <span id='am-120' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-121' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> obtained with the T3 and T7 primers was used to construct new <br /> study (28) has confirmed the azole sensitivity of the ncp1 phe-                    primers. Primers were obtained from Sigma-Genosys, Ltd. (The Woodlands, <br /> notype and has demonstrated that ncp1 cells still produce                          Tex.). <br />                                                                                       PCR. The standard PCR was composed of template DNA (5 to 10 ng of <br /> about 25% of the ergosterol produced by NCP1 strains. In                           plasmid DNA or 10 to 50 ng of genomic DNA), sequence-specific oligonucleo- <br /> addition, elevated levels of lanosterol were not detected, indi-                   tide primers (10 to 60 pmol of each primer), 0.2 mM deoxynucleoside triphos- <br /> cating that the cells were able to complete the C-14 demeth-                       phates (Sigma, St. Louis, Mo.), appropriate buffer, sterile water, and a thermo- <br /> ylation reaction as well as the other reactions where Ncp1p                        stable DNA polymerase. <br />                                                                                       The two kinds of thermostable DNA polymerases used in this study were Taq <br /> normally functions. Based on an in vitro assay (13) and on <br />                                                                                    (Promega) and Expand High Fidelity Taq polymerase (EHF Taq) (Roche, In- <br /> suppression of the ncp1 phenotype by CYB5 (28), Cyb5p could                        dianapolis, Ind.). Taq polymerase was largely used for screening genetic con- <br /> be the replacement electron donor.                                                 structions and genetic insertions or other PCRs where high DNA fidelity was not <br />    Based on the increased azole sensitivity of ncp1 strains cou-                   critical. The reaction buffer contained 10&#11003; buffer without MgCl2 (50 mM KCl, 10 <br /> pled with the fact that required reactions in sterol biosynthesis                  mM Tris-HCl [pH 9.0], 1% Triton X-100), 1.5 to 2.0 mM MgCl2 and 1.25 U of <br />                                                                                    enzyme. The other polymerase used in this study was EHF Taq (Roche). This <br /> in S. cerevisiae and C. albicans can differ (16), it is worthwhile <br />                                                                                    polymerase was used for PCRs when DNA accuracy was more crucial. The <br /> to investigate the characteristics of the cyb5 and the ncp1                        reaction buffer contained 10&#11003; buffer with 15 mM MgCl2 (20 mM Tris-HCl [pH <br /> phenotypes in this pathogenic fungus. This report investigated                     7.5], 100 mM KCl, 1 mM dithiothreitol, 0.1 mM EDTA, 0.5% [vol/vol] Tween 20, <br /> whether CYB5 is an essential gene and whether Cyb5p may be                         0.5% [vol/vol] Nonidet P40, 50% [vol/vol] glycerol), and 2.6 U of polymerase was <br /> a potential drug target. This report also employed various                         used per reaction. <br />                                                                                       PCRs were carried out on a Perkin Elmer GeneAmp2400 (Perkin Elmer, Inc., <br /> methods of gene disruption due to difficulties in isolating ho-                    Foster City, Calif.) or the PTC-0200 DNA Engine Thermal Cycler Chassis (M.J. <br /> mozygous mutants for the determination of gene essentiality.                       Research, Waltham, Mass.) with standard methods as described in Jia et al. (16). <br />                                                                                    The PCR primers used in this study are listed in Table 2. <br />                                                                                       Sterol analysis. Sterols were isolated as previously described (22) and analyzed <br />                        MATERIALS AND METHODS                                       by gas chromatography. An HP5890 Series II equipped with a Hewlett-Packard <br />    Strains and <span id='am-125' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-126' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>. <span id='am-69' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-84'  ></span>Escherichia coli</span> strain DH5-&#9251; [&#9278;80dlacZ&#9004;M15 recA1         Chemstation software package was used to analyze the sterols. The capillary <br /> endA1 gyrA96 thi-1 hsdR17 supE44 relA1 deoR &#9004;(lacZYA-argF)U169] was used           column (DB-5) was 15 m by 0.25 mm by 0.25 &#9262;m (J&amp;W Scientific, Folsom, <br /> for routine bacterial transformations. C. albicans strain BWP17 (ura3&#9004;::           Calif.). The gas chromatograph was programmed from 195&#176;C (1 min at 195&#176;C, <br /> &#9261;imm-434/ura3&#9004;::&#9261;imm-434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG) was ob-      then an increase at 20&#176;C/min until 240&#176;C and from 240 to 280&#176;C at 2&#176;C/min). The <br /> tained from A. Mitchell (31). C. albicans strain CAI-4 (ura3&#9004;::&#9261;imm-434/           linear velocity was 30 cm/s, with nitrogen as the carrier gas. All injections were <br /> ura3&#9004;::&#9261;imm-434) was obtained from W. Fonzi (9). S. cerevisiae strain Wb5&#9004; was     run in the splitless mode. <br /> obtained from G. Truan (28). The C. albicans strains generated in this study are      Gas chromatography/mass spectrometry analyses of sterols were done with a <br /> listed in Table 1. Plasmids pGEM-URA3, pGEM-HIS1, and pRS-ARG4 were                Thermoquest Trace 2000 gas chromatograph interfaced to a Thermoquest Voy- <br /> obtained from A. Mitchell (31). Plasmid pDBI52 was obtained from C. Kum-           ager mass spectrometer. The gas chromatography separations were done on a <br /> amoto (6).                                                                         fused silica column, DB-5MS, 20 m by 0.18 mm by 0.18 &#9262;m film thickness (J&amp;W <br />    Media and growth. E. coli strains were grown at 37&#176;C in <span id='am-1' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-2' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>Luria</span> broth (LB)        Scientific, Folsom, Calif.). The injector temperature was 190&#176;C. The oven tem- <br /> (Bio101) containing 10 g of tryptone, 5 g of yeast extract, and 10 g of NaCl per   perature was programmed to remain at 100&#176;C for 1 min, followed by a temper- <br /> liter. For solid media, 20 g of agar (Difco) per liter was added. Ampicillin       ature ramp of 6.0&#176;C/min to a final temperature of 300&#176;C. The final temperature <br /> (Sigma) was added after autoclaving at a concentration of 60 &#9262;g/ml for plasmid     was held for 25 min. Helium was the carrier gas with a linear velocity of 50 cm/s <br /> selection.                                                                         in the splitless mode. The mass spectrometer was in the electron impact ioniza- <br />    S. cerevisiae and C. albicans strains were grown on complete synthetic medium   tion mode at an electron energy of 70 eV, an ion source temperature of 150&#176;C, <br /> (CSM) or YPD medium. CSM medium contained 0.17% yeast nitrogen base                and scanning from 40 to 850 atomic mass units at 0.6-s intervals. <br /> without amino acids (Difco), 0.5% ammonium sulfate, 2% D-glucose, and a               Southern analysis. The probes used were amplified from a C. albicans wild- <br /> mixture of amino acids (Bio 101). CSM lacking specific nutrients was used in       type strain (BWP17) by PCR. Probe 1 was used for cytb5 fragments disrupted by <br /> selection. YPD was comprised of 1% Bacto yeast extract, 2% Bacto peptone, and      selectable markers. The primers used, b5-5probeI and b5-3probeI (Table 2), were <br /> 2% D-glucose. Adenine was added at 360 mg/liter to C. albicans cultures to         located 1,539 bp and 862 bp, respectively, upstream of the start <span id='am-61' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> and <br /> ensure optimal growth (YPAD). Uridine was added at 80 mg/liter to media used       amplified a 677-bp DNA fragment. Probe 2 was designed to hybridize to the <br /> to cultivate C. albicans ura3 strains. Maltose was used at 2% to replace glucose   CYTB5 open reading frame. The primers used, b5-5ProbeX and b5-3ProbeX <br /> in some media (YPAM). S. cerevisiae cultures were grown at 30&#176;C, while C.          (Table 2), were located 46 bp downstream of the start codon and 129 bp up- <br /> albicans cultures were grown at 30&#176;C and 37&#176;C.                                     stream of the stop codon, respectively, and amplified a 200-bp DNA fragment. <br />  <br />  <br /> The probes were designed from sequences not present in cyb5 remnants of the         membrane (Schleicher and Schuell, Keene, N.H.) according to the manufactur- <br /> disrupted alleles. Probes were labeled with the Rediprime II random prime           er&#8217;s protocol. <br /> labeling system (Amersham, Piscataway, N.J.) according to the manufacturer&#8217;s          Growth and drug susceptibility testing. Cells were grown overnight at 30&#176;C in <br /> protocol. Blotting was accomplished with a Nytran Supercharge nylon transfer        50-ml cultures. Cell concentration was determined by hemacytometer and di- <br />  3428       ROGERS ET AL.                                                                                          ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 1. Disruption methods used in this study. (A) Direct transformation disruption method. (B) Disruption method with an inducible rescue <br /> cassette. (C) Disruption method with the URA3 blaster system. <br />  <br />  <br />  <br />  <br /> luted to yield concentrations of 2 &#11003; 107, 2 &#11003; 106, 2 &#11003; 105, and 2 &#11003; 104 cells/ml;   disruption was carried out assuming that this gene was also <br /> 5 &#9262;l of each dilution was spotted on the medium to be tested. Cerulenin,            nonessential. <br /> clotrimazole, cycloheximide, and nystatin were obtained from Sigma. Flucon- <br /> azole and ketoconazole were obtained from ICN Pharmaceuticals Inc. (<span id='am-105' about='obo:NCBITaxon_192393' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_182732 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6552 obo:NCBITaxon_192392 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-106' property="rdfs:label" content="Cosa" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109'  ></span>Cosa</span>               Method I, disruption of CYB5 by direct transformation; <br /> Mesa, Calif.), and tridemorph and fenpropimoph were obtained from Sigma             disruption of the first allele. Method I is diagrammatically <br /> Aldrich. Terbinafine was a gift from I. Hapala (Slovak Academy of Science), and     represented in Fig. 1A. PCR primers b5-5drI and b5-3drI (Ta- <br /> itraconazole was a gift from Janssen Pharmaceuticals. Stock solutions of ceru-      ble 2) containing 60 bp of the CYB5 gene and 20 bp of plasmid <br /> lenin, terbinafine, and tridemorph were made in 95% ethanol, and stock solu- <br /> tions of cycloheximide and fluconazole were made in distilled H2O. Dimethyl         sequence flanking a selectable marker were used to amplify the <br /> sulfoxide was used as the solvent for clotrimazole, itraconazole, and ketocon-      ARG4 gene on plasmid pRS-ARG4. The resulting 2.1-kb DNA <br /> azole. All stocks were stored at &#11002;20&#176;C.                                             fragment was used to transform C. albicans strain BWP17, and <br />                                                                                     transformants were selected on CSM-Arg&#11001;Ade medium. PCR <br />                                                                                     was used to verify that the disruption was correct. Checking <br />                                  RESULTS <br />                                                                                     primers b5-5chI and b5-3chI (Table 2) were designed to amplify <br />    The C. albicans CYTB5 gene was identified from the C.                            a 0.5-kb fragment in the CYB5 allele and a 2.4-kb fragment in <br /> albicans genomic DNA sequence (19X contig) obtained from                            the ARG4-disrupted allele. Figure 2, lanes A and B, show the <br /> the Stanford University DNA Sequencing Project. The CYB5                            predicted bands of the wild type and the heterozygote KRC2, <br /> gene is a 381-bp open reading frame encoding a 126-amino-                           respectively. Sterol analysis (not shown) for KRC2 showed a <br /> acid protein. C. albicans Cyb5p shows 43% identity at the                           wild-type profile. <br /> amino acid level with S. cerevisiae Cyb5p.                                             Disruption of the second allele. PCR primers b5-5drIII and <br />    In <span id='am-66' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to determine whether Cyb5p is essential for growth                      b5-3drIII (Table 2) containing 60 bp of CYB5 sequence not <br /> in C. albicans, both copies of the CYB5 gene would have to be                       represented in the ARG4-disrupted CYB5 allele and 20 bp of <br /> sequentially disrupted. Since this organism does not import                         plasmid sequence were used to amplify the URA3 gene on <br /> exogenous sterol (19, 29), the second disruption would have to                      pGEM-URA3. The PCR products were use to transform <br /> take place in the presence of a third copy of the gene if Cyb5p                     KRC2, and transformants were selected on CSM-Ura- <br /> is essential. Since the CYB5 gene from S. cerevisiae was shown                      Arg&#11001;Ade. Confirmation of disruption was accomplished with <br /> to be nonessential (28), the first attempt at C. albicans CYB5                      primers b5-5chI and b5-3chI (Table 2), which would amplify a <br />  VOL. 48, 2004                                                                             STEROL SYNTHESIS IN C. ALBICANS                 3429 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 3. PCR confirmation of disruption of the C. albicans CYB5 <br />                                                                           gene with an inducible rescue cassette. Lane A shows the 3.4-kb frag- <br />                                                                           ment representing the integration of pKR02 containing the pMAL- <br />                                                                           CYB5 cassette at the ADE2 locus of KCR6. Lane B shows the 2.4-kb <br />                                                                           fragment of the ARG4-disrupted cyb5 allele and the 1.7-kb fragment of <br />   FIG. 2. PCR confirmation of disruption of the C. albicans CYB5          the HIS1-disrupted cyb5 allele of KCR7. Lane C confirms the presence <br /> gene by sequential transformation. Lane A shows the 0.5-kb fragment       (3.4-kb fragment) of the pMAL-CYB5 cassette in KCR7. <br /> of the CYB5 wild-type allele; lane B shows the 0.5-kb CYB5 allele and <br /> the 2.4-kb fragment of the ARG4-disruptped cyb5 allele in the hetero- <br /> zygote KCR2. Lane C shows the 1.7-kb fragment of the HIS1-disrupted       strain CAI-4. HIS1 was ligated to replace URA3 to generate <br /> cyb5 allele, the 2.4-kb fragment of the ARG4-disrupted cyb5 allele, and <br /> the 0.5-kb fragment of the third copy CYB5 wild-type allele of KCR3.      pKR02. This plasmid was linearized within the ade2 sequence <br />                                                                           with BamHI and used to transform the CYB5 heterozygote <br />                                                                           KRC2. Transformants were selected on CSM-Arg-His&#11001;Ade. <br />                                                                           Randomly selected colonies were checked by PCR for the <br /> 1.7-kb fragment in the URA3-disrupted allele. Figure 2, lane C,           proper integration at the ADE2 locus. Primers pMAL-5 and <br /> shows the results of the confirmation with one isolate (KRC3)             ADE-3ck2 were designed to amplify a 3.4-kb fragment repre- <br /> of the cyb5 homozygote. The 1.7-kb fragment representing the              senting the proper integration. Figure 3, lane A, shows the <br /> URA3-disrupted allele and the 2.4-kb fragment representing                proper integration for one isolate, KRC6. <br /> the ARG4-disrupted allele are present. However, a 0.5-kb frag-               Disruption of the second CYB5 allele. The remaining wild- <br /> ment reflecting a wild-type allele was also present. Sequencing           type CYB5 allele in KRC6 was disrupted by transformation <br /> of the 0.5-kb fragment isolated from the gel indicated 100%               again with the 1.7-kb URA3 fragment. Selection was on CSM- <br /> identity with the CYB5 sequence, indicating that it was an                Arg-His-Ura with maltose as the carbon and energy source. <br /> authentic copy of the wild-type allele. The same results were             Several colonies were analyzed by PCR to determine the na- <br /> obtained with other isolates of KRC3. Sterol analysis (not                ture of the disruptions and integration. Primers b5-5chI and <br /> shown) of strain KRC3 indicated a wild-type sterol profile,               b5-3chI (Table 2) were used to check for the ARG4 and URA3 <br /> indicating that the third allele was functional. Despite success-         disruptions of the CYB5 alleles. Figure 3, lane B, shows the <br /> ful disruption of both CYB5 alleles, the presence of the third            2.4-kb fragment representing the ARG4-disrupted allele and <br /> wild-type allele may imply that this gene is essential. To con-           the 1.7-kb fragment representing the URA3-disrupted allele in <br /> firm the essentiality of this gene, we employed an alternative            one isolate (KRC7). With primers pMAL-5 and ADE-3ck2 <br /> method of disruption.                                                     (Table 2), the plasmid integration at the ADE2 locus is shown <br />    Method II, disruption of CYB5 with an inducible rescue                 to be correct by virtue of the presence of the 3.4-kb fragment <br /> cassette; preparation of the rescue cassette. Method II is                (Fig. 3, lane C). Identical results were obtained with four <br /> shown in Fig. 1B. The vector pKR01, containing a copy of the              additional isolates of KRC7. <br /> CYB5 gene under the control of the pMAL2 promoter, was                       The cyb5 homozygote (KRC7) was spot plated on maltose <br /> constructed by inserting a CYB5 open reading frame into                   and glucose media to determine whether the CYB5 gene was <br /> pDB152 (6). Primers b5-5EcoRV and b5-3XbaI (Table 2) were                 essential. The results shown in Fig. 4 indicate that growth was <br /> used to amplify the CYB5 open reading frame in strain                     equivalent on all media. If the CYB5 gene is essential, normal <br /> BWP17, creating a 5&#11032; EcoRV site and a 3&#11032; XbaI site on the                 growth in maltose would be expected, but growth on glucose <br /> 0.6-kb fragment. The plasmid was confirmed to have the cor-               would be minimal due to lack of induction of the wild-type <br /> rect insertion and generated a 6.4-kb fragment when linear-               CYB5 allele under the control of pMAL2. While some leakage <br /> ized.                                                                     through the pMAL2 promoter would be expected (1, 3), the <br />    Since the disruption system in place used ARG4 and URA3                amount of product formed would not allow normal growth if <br /> as selectable markers for the CYB5 disruptions, the URA3                  the gene is essential. Since growth was normal on glucose, the <br /> marker in pKR01 was replaced with HIS1. Primers HIS1-5 and                CYB5 gene is concluded to be nonessential. <br /> HIS1-3 (Table 2) were designed with a 5&#11032; BamHI site and a 3&#11032;                 Sterol accumulation in strain KRC7 grown on maltose and <br /> XbaI site to amplify a 1.6-kb HIS1 fragment from C. albicans              glucose was determined. Figure 5 shows the sterol profiles <br />  3430     ROGERS ET AL.                                                                                  ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />          FIG. 4. Growth of C. albicans cyb5 with the pMAL-CYB5 cassette (KRC7) in YPA and CSM media with glucose and maltose. <br />  <br />  <br />  <br /> along with those of BWP17 and Wb5&#9004; (an S. cerevisiae cyb5                CYB5 was essential and the rescue cassette method indicated <br /> mutant). The sterol profile of maltose-grown KRC7 resembles              that the gene was not essential, a third method (shown in Fig. <br /> that of its wild-type strain, with ergosterol constituting over          1C) was employed for final confirmation. In this method, the <br /> 85% of total sterol. When grown on glucose, however, KRC7                CYB5 gene was integrated adjacent to the URA3 gene and <br /> produced a profile similar to that of Wb5&#9004;, with decreased               flanked by <span id='am-92' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-93' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span>Salmonella</span> hisG repeats (URA3 blaster cassette) <br /> ergosterol and accumulation of the pathway intermediates er-             (9). This third copy would serve as a rescue copy until plating <br /> gosta-7,22-dien-ol, episterol, and fecosterol. The accumulation          on 5-fluoroorotic acid, which selects against URA3 and for cells <br /> of substantial amounts of ergosta-7,22-dien-ol was expected              in which hisG recombination excises the CYB5-URA3 frag- <br /> because C-5 desaturation is a Cyb5p-dependent reaction.                  ment. <br />   Method III, disruption of CYB5 with the URA3 blaster sys-                 HIS1 disruption of the first CYB5 allele. Primers b5-5drII <br /> tem. Since the direct transformation method suggested that               and b5-3drII (Table 2) were designed to contain 60 bp of the <br />                                                                          CYB5-flanking sequence and located 769 bp upstream of the <br />                                                                          start codon and 289 bp downstream of the stop codon, respec- <br />                                                                          tively, and 20 bp of plasmid sequence. They were used to <br />                                                                          amplify the HIS1 marker on pGEM-HIS1. The resulting 2.7-kb <br />                                                                          fragment containing the HIS1 gene flanked by adjacent CYB5 <br />                                                                          sequence was used to transform strain BWP17, and transfor- <br />                                                                          mants were selected on CSM-His&#11001;Ade. Ten of the resulting <br />                                                                          93 transformants were tested by PCR for the proper configu- <br />                                                                          ration. Checking primers b5-5chII and b5-3chII (Table 2) were <br />                                                                          designed to amplify a 1.9-kb fragment representing the wild- <br />                                                                          type CYB5 allele and a 3.1-kb fragment representing the HIS1- <br />                                                                          disrupted allele. Figure 6, lane A, shows the presence of both <br />                                                                          fragments in the heterozygote KRC9. <br />                                                                             Construction and integration of the CYB5-URA3 blaster cas- <br />                                                                          sette. The vector pIU1300 was used as the basic structure for <br />                                                                          the URA3 blaster cassette. pIU1300 is pBluescript into which <br />                                                                          the C. albicans ERG25 gene (encoding the sterol C-4 methyl- <br />                                                                          oxidase) has been integrated. The URA3 blaster cassette has <br />                                                                          been integrated within the ERG25 sequence. <br />                                                                             With primers b5-5promoter and b5-3XbaI (Table 2), the <br />                                                                          CYB5 sequence including 654 bp of promoter and 187 bp of <br />                                                                          terminal sequence was amplified from BWP17. The resulting <br />                                                                          1.3-kb fragment contained XbaI sites at both ends. After re- <br />                                                                          moval of one of the XbaI sites in pIU1300, both the resulting <br />                                                                          plasmid, pKR04, and the CYB5 fragment were digested with <br />                                                                          XbaI, ligated, and used to transform E. coli DH5&#9251;. Forty <br />                                                                          resulting colonies out of 1,450 were screened for the appropri- <br />                                                                          ate integration of the CYB5 sequence within the URA3 blaster <br />   FIG. 5. Sterol profiles of the C. albicans wild-type (BWP17), the S.   adjacent to the URA3 gene. Transformants that yielded the <br /> cerevisiae cyb5 mutant (Wb5&#9004;), the C. albicans cyb5 mutant (KCR7) <br /> grown on maltose and glucose, and the C. albicans cyb5 mutant <br />                                                                          proper orientation were designated pKR05. <br /> (KCR12). Peaks: A, ergosterol; B, fecosterol; C, epsiterol; D, lanos-       XhoI and ClaI were used to extract a 9.1-kb fragment from <br /> terol; E, ergosta-7,22-dien-ol. RT, retention time.                      pKR05 containing the erg25-hisG-CYB5-URA3-hisG-erg25 <br />  VOL. 48, 2004                                                                              STEROL SYNTHESIS IN C. ALBICANS                 3431 <br />  <br />  <br />  <br />  <br />    FIG. 6. PCR confirmation of disruption of the C. albicans CYB5 gene with the URA3 blaster cassette. Lane A shows the 3.1-kb fragment of the <br /> HIS1-disrupted cyb5 allele and the 1.9-kb fragment of the wild-type CYB5 allele of KCR9. Lane B shows the 1.7-kb fragment indicating integration <br /> of the URA3-CYB5 cassette at the ERG25 locus in KCR10. Lane C confirms the CYB5 heterozygosity of KCR10. Lane D shows the 3.7-kb fragment <br /> of the ARG4-disrupted cyb5 allele and the 3.1-kb fragment of the HIS1-disrupted cyb5 allele of KCR11. Lane E confirms (1.7-kb fragment) the <br /> continued presence of the URA3-CYB5 cassette at the ERG25 locus in KCR11. Lane F shows the presence of the ARG4- and HIS1-disrupted cyb5 <br /> alleles in KCR12. Lane G shows the confirmation of hisG recombination eliminating the URA3-CYB5 construct (2.9-kb fragment) and the wild-type <br /> ERG25 allele on the second chromosome. <br />  <br />  <br /> construct. This fragment was used to transform KRC9, and                  KRC12 isolates yielded 1.9-kb and 2.9-kb fragments, indicating <br /> transformants (KRC10) were selected on CSM-His-Ura&#11001;Ade.                   the presence of a wild-type ERG25 allele and the hisG recom- <br /> To confirm integration at the ERG25 chromosomal locus,                    bination at the other ERG25 locus, respectively (Fig. 6, lane <br /> primers hisG-3 and E25-b5-3chI (Table 2) were designed to                 G). Sterol analysis of KCR12 indicated that the predominant <br /> amplify a sequence within the hisG repeat to a chromosomal                sterol was ergosta-7,22-dien-ol, consistent with the sterol pro- <br /> region flanking ERG25, respectively. The expected fragment                file in the authentic S. cerevisiae cyb5 mutant Wb5&#9004; (26). Other <br /> would be 1.7 kb. Figure 6, lane B, shows the presence of the              sterols found in lesser amounts, such as fecosterol, ergosterol, <br /> 1.7-kb fragment in KRC10. KRC10 was also rechecked for the                episterol, and lanosterol, were also found in both C. albicans <br /> continued presence of the chromosomal heterozygosity (Fig. 6,             and S. cerevisiae cyb5 mutants (Fig. 5). Cyb5p is nonessential by <br /> lane C) of CYB5.                                                          this procedure. <br />    ARG4 disruption of the second CYB5 allele. Primers b5-                    Confirmation of disruptions by Southern analysis. Southern <br /> 5drIII and b5-3drIII (Table 2) were used to amplify the ARG4              analysis was done to confirm the results generated by PCR for <br /> gene on plasmid pRS-ARG4, as described in the disruption of               the CYB5 heterozygotes and cyb5 nulls created in this study. <br /> the first CYB5 allele by direct transformation above. The re-             Two probes were employed. The first was located upstream of <br /> sulting 2.1-kb fragment was used to transform strain KRC10,               the CYB5 chromosomal alleles and designed to detect wild- <br /> and transformants were selected on CSM-Arg-His-Ura.                       type and disrupted versions of the CYB5 gene (primers b5- <br /> Twelve of 240 transformants were analyzed by PCR for the                  5probeI and b5-3probeI in Table 2). The second probe was <br /> proper configuration. Primers b5-5chII and b5-3chII (Table 2)             designed to detect the CYB5 gene integrated at any position <br /> were used to check the disrupted CYB5 alleles, and primers                within the C. albicans genome. (primers b5-5probeX and b5- <br /> hisG-3 and E25-b5-3chI (Table 2) were used to confirm the                 3probeX in Table 2). DNA was prepared for both probes with <br /> CYB5-URA3 blaster cassette integration at the ERG25 locus.                EcoRI. <br /> Figure 6, lane D, shows the 3.1-kb HIS1-disrupted allele and                 Figure 7A shows the results with probe 1. The wild-type <br /> the 3.7-kb ARG4-disrupted allele fragments, and lane E shows              (BWP17) shows the predicted 5.6-kb fragment representing <br /> the 1.7-kb fragment indicating cassette integration at ERG25              the wild-type CYB5 allele. Strain KRC3 containing the ARG4- <br /> for KRC11.                                                                and URA3-disrupted alleles shows a heavy band at approxi- <br />    Recombination of hisG fragments and the cyb5 null. KCR11               mately 4.5 kb and the presence of the wild-type allele (5.6 kb). <br /> was freshly grown and plated on CSM containing 5-fluoroo-                 Further experiments with the URA3 and ARG4 probes (prim- <br /> rotic acid. After 3 days of incubation, a total of 40 colonies            ers in Table 2) confirmed that both the predicted 4.5-kb ARG4- <br /> emerged. The colonies were patched onto CSM-Ura to con-                   and 4.3-kb URA3-disrupted bands are present in the heavy <br /> firm the loss of the URA3 gene. Twenty of the colonies were               band shown, confirming the presence of the third copy of the <br /> confirmed as Ura&#11002;. Five isolates were chosen for confirmation             CYB5 gene in KRC3. KRC7 shows the same doublet at 4.5 kb, <br /> by PCR for continuing CYB5 disruptions, and hisG recombi-                 and again both the ARG4- and URA3-disrupted alleles were <br /> nation expected in the resulting strain, KRC12. The ARG4-                 found to be present. KRC9, a heterozygote, shows the pres- <br /> and HIS1-disrupted alleles were confirmed as still present (Fig.          ence of the 5.6-kb wild-type allele and the 6.9-kb fragment <br /> 6, lane F) in four of the five KRC12 isolates chosen. The fifth           representing the HIS1-disrupted allele. KRC12 (and KRC11- <br /> showed only the HIS1-disrupted allele and a wild-type allele              not shown) shows the presence of the HIS1-disrupted allele <br /> and was set aside. Primers E25-b5-3chI and e25-5chII (Table 2)            (6.9 kb) and the ARG4-disrupted allele (4.5 kb). The results <br /> were used to amplify a 1.9-kb fragment representing the wild-             confirm those obtained with PCR. <br /> type ERG25 allele and a 2.9-kb fragment representing the hisG                Probe 2 results are shown in Fig. 7B. The chromosomal <br /> recombination product in the second allele into which the                 wild-type CYB5 allele is displayed as a 5.6-kb band in BWP17. <br /> CYB5-URA3 blaster cassette was inserted. All four of the                  KRC12, as expected, produced no bands, indicating that no <br />  3432     ROGERS ET AL.                                                                                           ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                                                            TABLE 3. Effects of antifungal drugs and cellular inhibitors on <br />                                                                               wild-type (CYB5) and cyb5 null mutants of C. albicans <br />                                                                                                                         Inhibitory concna (&#9262;g/ml) <br />                                                                               Inhibitor <br />                                                                                                                     CYB5                            cyb5 <br />  <br />                                                                          Ketoconazole                              &#11022;1.5                             0.5 <br />                                                                          Fluconazole                              &#11022;10                               0.5 <br />                                                                          Itraconazole                               1.5                            &#11021;0.5 <br />                                                                          Tridemorph                                 0.025                          &#11022;5 <br />                                                                          Fenpropimorph                              0.025                           0.1 <br />                                                                          Terbinafine                                3                               1 <br />                                                                          Cycloheximide                           &#11022;140                            &#11022;140 <br />                                                                          Cerulenin                                 &#11022;1.5                             1.5 <br />                                                                          Fluphenazine                             &#11022;10                             &#11022;10 <br />                                                                           a <br />                                                                               Concentration at which there was no visible growth after 72 h of incubation. <br />  <br />  <br />  <br />                                                                          glucose resulted in increased azole sensitivity, as expected with <br />                                                                          no functional Cyb5p present. Induction of the CYB5 gene on <br />                                                                          maltose resulted in azole sensitivity approaching the levels <br />    FIG. 7. Confirmation of the CYB5 disruption strategies by South-      seen with the wild type (Fig. 8). <br /> ern analysis. The upper panel shows the results with probe 1 to detect <br /> wild-type and disrupted versions of CYB5. The wild-type band of 5.6 kb <br /> is shown in the wild-type strain (BWP17) and as a third copy in KCR3.                                   DISCUSSION <br /> KCR3 and KCR7 show the presence of a thick band of approximately <br /> 4.4 kb, comprising the 4.3-kb URA3-disrupted and 4.5-kb ARG4-dis-           The CYB5 gene of C. albicans was identified from a database <br /> rupted cyb5 alleles. The 6.9-kb band representing the HIS1-disrupted <br />                                                                          based on homology to the CYB5 gene from S. cerevisiae. In <br /> cyb5 allele is shown in KCR9 and KCR12. The lower panel shows the <br /> results with probe 2. BWP17 shows the wild-type allele (5.6-kb band),    order to determine whether Cyb5p was essential for growth in <br /> while KCR12 shows no bands confirming that the CYB5 gene is not          C. albicans, both copies of the gene were sequentially disrupted <br /> found at any location in the genome.                                     with CYB5 sequences flanking selectable markers. The direct <br />                                                                          transformation disruption protocol resulted in the successful <br />                                                                          disruption of both alleles in the presence of a third copy of the <br /> copies of the CYB5 gene remain after hisG recombination                  wild-type CYB5 allele. The third-copy phenomenon has been <br /> eliminated the cassette copy of CYB5.                                    previously reported in C. albicans and was thought to occur <br />    Drug susceptibilities of the C. albicans cyb5 mutant KRC12.           only with essential genes (8). Based on this assumption, we <br /> The viable C. albicans cyb5 mutant KCR12 generates a novel               would conclude that the CYB5 gene is essential. However, <br /> sterol profile, and it was of interest to see if this array of sterol    since the gene is not essential in S. cerevisiae, confirmation <br /> alters the cell&#8217;s sensitivity to a variety of drugs. Of particular       through other approaches was sought. <br /> interest would be the sensitivities to commonly used antifungal             With a second protocol, a third copy of the CYB5 gene under <br /> agents. Figure 8 shows the growth patterns of the C. albicans            the control of pMAL2, the maltose-inducible promoter, was <br /> wild-type, BWP17 (CYB5), and KRC12, the cyb5 null, on three              inserted at the ADE2 locus in a CYB5 heterozygote. The sec- <br /> azoles (ketoconazole, fluconazole, and itraconazole), inhibi-            ond allele of the CYB5 gene was then disrupted. This was done <br /> tors of Erg11p (30), two morpholines (tridemorph and fen-                in the presence of maltose to rescue cyb5 homozygotes should <br /> propimorph), inhibitors of Erg24p and Erg2p (4), and terbin-             the gene be essential. When transferred to glucose, the cyb5 <br /> afine, an inhibitor of squalene epoxidase (Erg1p) (14). In               null (KRC7) grew normally, indicating that the gene is nones- <br /> addition, cycloheximide, a protein synthesis inhibitor, cerule-          sential in C. albicans. Low-level leakage through the pMAL2 <br /> nin, a fatty acid synthase inhibitor (23), and fluphenazine, a           promoter in the absence of inducer might be anticipated, but <br /> calmodulin antagonist (10), were tested.                                 previous studies (1, 3) with this promoter show that very slow <br />    KCR12 was shown to be significantly more sensitive to the             growth results when the gene in question is essential. Sterol <br /> azoles than the wild type. Terbinafine sensitivity also increased,       analysis of KRC7 indicated a profile that included low levels of <br /> but the cyb5 strain was considerably more resistant to <span id='am-115' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-116'  ></span><span id='am-117' property="rdfs:label" content="Morpho" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>morpho</span>-           ergosterol and an array of intermediates that closely resembled <br /> lines than the CYB5 strain. There were no changes in sensitiv-           the sterol accumulation of the cyb5 (Wb5&#9004;) mutant of S. cer- <br /> ities to general metabolic inhibitors. These patterns are shown          evisiae (28). This approach did not reveal a third copy of the <br /> in Table 3 in terms of the drug concentration at which no                CYB5 gene and indicated that the gene is not essential. <br /> growth was evident after 72 h of incubation.                                A final disruption procedure was used to confirm the results <br />    In order to confirm that the KRC12 azole-sensitive pheno-             with the pMAL2 inducible rescue cassette. This approach used <br /> type was due to the loss of Cyb5p, a copy of the CYB5 gene was           the URA3 blaster cassette into which a third copy of the CYB5 <br /> reintroduced into KCR12. Plasmid pKR02 was linearized                    gene had been integrated adjacent to the URA3 marker. The <br /> within the ade2 sequence with AvaI and used to transform                 cassette was integrated into an unrelated chromosomal site in <br /> KRC12. Transformants (KRC13) were selected on CSM-Arg-                   the genome of a CYB5 heterozygote. Following disruption of <br /> His-Ura. Figure 9 shows the growth of KRC13 on glucose and               the second allele, the resulting chromosomal cyb5 homozygote <br /> maltose in the presence of the three tested azoles. Growth on            was plated on 5-fluoroorotic acid to eliminate the URA3-CYB5 <br />  VOL. 48, 2004                                                                           STEROL SYNTHESIS IN C. ALBICANS                 3433 <br />  <br />  <br />  <br />  <br />      FIG. 8. Sensitivity of C. albicans BWP17 (CYB5) and the cyb5 null (KRC12) to a variety of antifungal agents and cellular inhibitors. <br />  <br />  <br />  <br /> fragment via hisG recombination. The resulting strain                   YAL36, CPP1, CPH1, and FKS1 are found to be triplicated but <br /> (KRC12) was viable, indicating, once again, that the CYB5               nonessential. However, our results suggest that we see trisomic <br /> gene is not essential. This procedure has the advantage of              CYB5 only when performing a transformation of the CYB5/ <br /> having no remaining copies of the CYB5 gene, thus eliminating           cyb5 heterozygote and not when we introduced a rescue cas- <br /> the possibility of recombination of the CYB5 allele, as is pos-         sette before disruption of the second allele and not when we <br /> sible in the pMAL2 system (1, 3), to other sites not under the          selected for deletion of the CYB5 allele with the URA3 blaster <br /> control of pMAL2.                                                       approach. Thus, direct transformation to determine essential- <br />    The issue of gene disruption is complicated due to the third-        ity was less reliable than rescue or deletion approaches. Im- <br /> allele phenomenon which occurs in C. albicans and also due to           portantly, the pMAL2 inducible CYB5 cassette and the URA3 <br /> possible trisomies that occur in this organism. Whereas se-             blaster-CYB5 deletion cassettes were not associated with the <br /> quential gene disruption of essential genes results in two dis-         CYB5 locus to avoid allelic and chromosomal ploidy interfering <br /> rupted alleles, a third allele is observed. The origin of these         with the final test for gene essentiality. <br /> trisomies is unclear because various C. albicans strains may               Sterol analysis of KRC12 indicated the synthesis of some <br /> have trisomic regions or entire chromosomes. A recent report            ergosterol and sterol intermediate accumulation similar to that <br /> by Chen et al. (7) finds that in some isolates of CAI-4, chro-          of S. cerevisiae Wb5&#9004;. This indicates that another electron <br /> mosome 1 is trisomic, explaining why genes such as GLY1,                donor is able to partially substitute for Cyb5p and allows the <br />  <br />  <br />  <br />  <br />   FIG. 9. Azole sensitivity of C. albicans KCR13, a cyb5 null containing a CYB5 gene under pMAL2 control, grown on glucose and maltose. <br />  <br />  <br />  <br /> synthesis of some ergosterol. Based on these data, accompa-                                          <br /> nied by PCR and Southern analysis confirmations, we conclude        <br /> that the CYB5 gene is not essential in C. albicans.          <br />    The results obtained here raise a question regarding the       <br /> third gene copy phenomenon. Chromosome duplication and               <br /> duplication of smaller genomic elements (8) have been sug- <br />                                                                     <br /> gested as mechanisms by which third copies of some genes         <br /> arise. To this point, third copies have been shown to occur only   <br /> with essential genes, and recently the use of the UAU1 dis-       <br /> ruption procedure was suggested as a rapid test for determin-       <br /> ing whether a given gene was essential (8). Unfortunately, this     <br /> procedure could not be successfully implemented in this case,       <br /> possibly because the CYB5 gene is so small. Since there was no      <br /> evidence of a third copy in the latter two disruption protocols,     <br /> the results also indicate the possibility that the third-copy phe-    <br /> nomenon may occur as a spontaneous event in certain cases        <br /> when the chromosomal alleles are being inactivated. Alterna- <br /> tively, the successful direct disruption procedure where the       <br /> third copy was found may have occurred only in a small sub-   <br /> population in which the CYB5 allele existed in triplicate.           <br />    Since inhibition of Cyb5p would result in a viable cell with    <br /> altered sterol composition, it was of interest to determine        <br /> whether such a cell would show altered responses to common           <br /> inhibitors and antifungal drugs. Surprisingly, we found a sig-          <br /> nificant increase in azole sensitivity for KCR12. In order to    <br /> confirm the linkage of azole sensitivity and the lack of Cyb5p,   <br /> the azole experiment was repeated with KRC13, a derivative of       <br /> KCR12. KCR13 is KRC12 into which the pMAL2-CYB5 cas-                  <br /> sette has been integrated at the ADE2 locus. When grown on        <br /> glucose, KRC13 shows a pattern of increased inhibition by           <br /> azoles similar to that seen with KCR12. The increased sensi-   <br /> tivity is not quite as great, probably due to low-level leakage   <br /> through the pMAL2 promoter (1, 3). Nearly normal azole          <br /> sensitivity is restored when KCR13 is grown on maltose, indi-       <br /> cating that the loss of Cyb5p is responsible for the increased     <br /> azole sensitivity. Other C. albicans sterol mutants that also       <br /> produced altered sterol profiles showed no such increases in            <br /> azole sensitivity (15, 16). Coupled with no changes in inhibition      <br /> levels with cellular inhibitors, this indicates that increased       <br /> azole sensitivity is not likely due to increased permeability to        <br /> these compounds. The cyb5 null was also more sensitive to         <br /> terbinafine, an inhibitor of squalene epoxidase.                        <br />    These results with the azoles are consistent with these drugs&#8217;     <br /> having effects on substitute electron donors in reactions within    <br /> the sterol biosynthetic pathway. Both the squalene epoxidase           <br /> and the C-14 demethylase steps are dependent upon NCP1 as        <br /> an electron carrier. The ncp1 genotype has also been shown to         <br /> be more sensitive to azoles, indicating the possibility that        <br /> Cyb5p and Cprp do have significant functional overlap and       <br /> that both may be sensitive to azole inhibition. Further investi-       <br /> gations are needed to understand why cyb5 mutants are more                <br /> resistant to the morpholine class of sterol biosynthesis inhibi-     <br /> tors. However, S. cerevisiae elo2 and elo3 sphingolipid mutants <br />                                                                      <br /> (24) are commonly found to be morpholine resistant, and these            <br /> mutations are known to be suppressors of erg2 and erg24 (21).          <br />                                                                          <br /> </body></html>