<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natd. Acad. Sci. USA <br /> Vol. 88, pp. 11373-11377, December 1991 <br /> Biochemistry <br />  <br />  <br /> RAM2, essential gene yeast, RAMI encode two <br /> polypeptide components farnesyltransferase that <br /> prenylates a-factor Ras proteins <br />      (CAAX motifs/membrane targeting/post-                tioal     ion/extr                c suppressors) <br /> BIN HE*, PENG CHENt, SEI-YU CHEN*, KATHLEEN L. VANCURAt, SUSAN MICHAELISt, AND SCOTT POWERS*t <br /> *Department Biochemistry, Robert Wood Johnson Medical School, University Medicine Dentistry New Jersey, Piscataway, NJ 08854; and <br /> tDepartment Cell Biology Anatomy, The Johns Hopkins University School Medicine, Baltimore, MD 21205 <br /> Communicated Leland Hartwell, September 23, 1991 <br />  <br /> ABSTRACT           In yeast Saccharomyccs cerevwise, qiuta-                      CAAX sequence noted (22, 23). Geranylgeranyltrans- <br /> tions unlinked genes, RAM)-1or RAM2, abolish                        ferase type I (GGTase I), modifies proteins that <br /> the farnesyltrnsferae activity responsible prenylation                        terminate second type CAAX sequence (23-25), <br /> Ras proteins a-factor mating pheromone. Here                              heterodimeric enzyme appears share a <br /> report thefunction RAM) antd RAM2 genme required                          common subunit p21&quot; protein farnesyltransferase <br /> for membrane laliation ofRas proteins a-factor. The                          (21, 23). A enzyme identified, GGTase II, <br /> RAM2 gene sequenced encode 38-kDa protein.                             geranylgeranylates Yptl/Sec4 <span id='am-38' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> proteins <br /> We examined tie functional interaction RAM2 andRAMf                            (21). <br /> expressing genes Escherchia colt. Extracts derived                          Genetic studies yeast identified genes, each <br /> an E. coil strain coexpressed RAM] RAM2 efficiently                         uniquely affects type prenyltransferase <br /> farnesylated a-factor peptide Ras protein substrates. In                         reaction. One genes, RAM) (also called DPRI), was <br /> contrast, extracts derived E. colN strains expressed                       identified suppressor RAs2ra19, mutationally acti- <br /> either RAM gene devoid activity; however,                         vated RAS allele (26, 27). <span id='am-41' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> gene identified <br /> the extracts mixed, protein farnesyltransferaSe                          based involvement a-factor production (26, 28, 29) <br /> activity reconstituted. These results indicate yeast                    suppressor G protein function (30). ram) extracts <br /> farnesyl-protein transferase comprised Raml Ram2                           defective a-factor Ras protein farnesyltransferase <br /> polypeptides. Although Raml component enzyme,                            activity prenyltransferase activities (21, 23, <br /> disruption RAM) gene yeast lethal indicating                       31, 32). Furthermore, Raml protein produced <span id='am-59' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65'  ></span>Escherichia</span> <br /> that Raml-Ram2 farnesyltserase essential                              coli, lacking enzyme activity own, biochem- <br /> viability. In contrast, disruption RAM2 lethal, suggest-                      ically complements ram] extracts a-factor peptide farne- <br /> ing Ram2 essential function addition role                      syltransferase activity (31). This result suggests the <br /> with Raml protein farnesylation.                                                  RAM] gene product necessary insufficient compo- <br />                                                                                      nent protein farnesyltransferase. Mutants harboring <br /> The post-translational modification proteins cova-                         lesions CDC43/CALI, gene shows homology to <br /> lent attachment isoprenoid groups plays important role                         RAM) (33), specifically defective GGTase I activity <br /> in membrane targeting various proteins (1-5). Three                           (21, 23, 34). Another yeast gene homologous RAM), <br /> classes prenylated proteins eukaryotic cells                         called BET2, affects membrane localization Yptl and <br /> described. The class, represented certain fungal                            Sec4 proteins (35) specifically affects yeast GGTase II <br /> mating pheromones, Ras proteins, nuclear lamins,                              activity (21). <br /> initially synthesized C-terminal CAAX sequence,                                  In report characterize yeast gene, RAM2, <br /> where X = Ser, Cys, Met, <span id='am-71' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Ala" datatype="xsd:string"></span>Ala</span>, modified thioether                        identified extragenic suppressor RAS2val&apos;9. <br /> linkage C15 farnesyl group cysteine residue (6-9).                       Previous reports shown ram2 mutants, like ram] <br /> The second class prenylated proteins, represented                          mutants, defective Ras protein farnesyltransferase <br /> y subunits certain heterotrimeric G proteins various                          activity (23, 32). Here report DNA sequence RAM2 <br /> Ras-like proteins, initially synthesized different C-ter-                    present direct evidence RAM] RAM2 encode <br /> minal CAAX sequences, X = Leu Phe,                                essential subunits yeast farnesyl-protein transfer- <br /> modified attachment C20 geranylgeranyl group                          ase. <br /> the cysteine residue (10-12). Members of these <br /> classes prenylated proteins subsequently processed                                      MATERIALS AND METHODS <br /> proteolytic removal terminal amino acids                               Yeast Strains Materials. The ram2-1 mutants used in <br /> methylation newly exposed carboxyl group                               study derived original mutant STS39 by <br /> prenyl-cysteine (6, 10-15). The class prenylated                            backcrosses RS16-4C (26). Other yeast strains been <br /> proteins, represented Yptl/Sec4 (Rab) <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>                               described (26, 29, 36). Recombinant p21&quot;5 purified as <br /> Ras-like GTPases terminate sequence CC                              described (37). Biotinylated KTSCVIM peptide (38) was <br /> CXC, geranylgeranylated C-terminal cysteine                             obtained Multiple Peptide Systems (San Diego). The <br /> residue(s) (16-18).                                                                  a-factor peptides YIIKGVFWDPAC YIIKGVFWD- <br />   The mammalian p21&quot;5 protein farnesyltransferase                               PACVIA synthesized P. Shenbagamurthi (Johns <br /> heterodimeric enzyme (19-21) appears responsible <br /> for modification proteins type of <br />                                                                                      Abbreviations: CAAX, protein sequence C = cysteine, A = <br />                                                                                      usually aliphatic amino acid, X = amino acid; FPP, <br /> The publication costs article defrayed page charge           farnesyl pyrophosphate; GGTase I, geranylgeranyltransferase type- <br /> payment. This article marked &quot;advertisement&quot;                I; GGTase II, GGTase type II. <br /> in accordance 18 U.S.C. &#167;1734 solely indicate fact.                     ITo reprint requests addressed. <br />                                                                              11373 <br />  11374    Biochemistry: He et al.                                                   Proc. Natl. Acad. Sci. USA 88 (1991) <br /> Hopkins). [3H]Farnesyl pyrophosphate ([3H]FPP; 20 Ci/             ysis 10 complete tetrads, phenotypes resulted <br /> mmol; 1 Ci = 37 GBq) purchased NEN.                      single mutation (ram2-1). ram2-1 previously <br />    Plasmid Constructions Gene Disruptions. The integra-       demonstrated unlinked ram) (32). <br /> tive mapping plasmid pBH43 contains the1l.85-kilobase-pair           ru2 Mutants Are Defettive Farnesylatlon <span id='am-51' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-52' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Mea</span>. <br /> (kbp) Sac I-Pst I RAM2 fragment inserted pRS305 (39).        brane Targeting Both -Ras Proteins a-Factor Ma <br /> pram2::LEU2 constructed insertion ofthe 1.5-kbp Pst        Pheromone. It shown extracts prepared from <br /> I-Bgl II RAM2 fragment pBH43 gene replacements           ram2-1 mutants fail farnesylate bacterially produced Ras <br /> were generated digestion Pst I transformation         proteins vitro (23, 32). We determined ram2-) extracts, <br /> into diploid strain TTSD1 (40). pram):: URA3 (pSM353)         like ram] extracts, defective farnesylating a-factor <br /> contains 4.6-kbp BamHI-Bgl II fragment RAM)            vitro (<span id='am-76' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown). To determine lack of <br /> locus inserted vector pUC-4k, URA3              farnesylation measured vitro reflected vivo a <br /> HindIII fragment place RAM) HindIII fragment,           defect membrane targeting, radiolabeled wild-type and <br /> such codons 1-322 deleted (41). Gene replacements        mutant yeast cells examined soluble particulate <br /> were generated digestion EcoRI transformation         fractions immunoprecipitation using antibodies rec- <br /> into diploid strain SM1060 (36). To express RAM2 E.        ognized Ras proteins a-factor (Fig. 1). In wild-type <br /> coli, RAM2 coding sequence cassette generated            strain, Ras2 localized particulate fraction an <br /> PCR utilizing oligonucleotide primers overlapped         apparent 40-kDa <span id='am-56' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. In contrast, ram2-1 the <br /> either initiation <span id='am-46' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span>codon</span> termination codon shown     ram) mutants, Ras2 localized soluble fraction and <br /> Fig. 3. The relevant sequence 5&apos; portion cassette   migrated slowly (Fig. lA). Fig. 1B shows a-factor <br /> is CTCGAGGAAAAM, underlined ATG corre-                  detected particulate fraction wild-type cells as <br /> sponds initiation codon RAM2. The 3&apos; portion     precursor species (P1 P2), a-factor synthe- <br /> cassette contained artificial BamHI site following         sized ram) ram2-1 mutants predominantly localized <br /> termination codon. This cassette inserted chlor-     soluble fraction slower migrating form (PO). Thus, <br /> amphenicol-resistance plasmid pBC-KS+ (Stratagene)           ram2-1, like mutations RAM), blocks membrane <br /> had cleaved Xho I BamHI, generate pBH57,         targeting Ras proteins a-factor. <br /> which utilizes lac promoter direct synthesis        Cloing Sequence Analysis RAM2. We utilized the <br /> N-terminal lacZ-RAM2 fusion protein. To express RAM)           Ts- phenotype ram2-1 mutants clone wild-type <br /> E. coli, plasmid pBH63 constructed inserting Sal     RAM2 gene complementation. A ram2-1 mutant strain <br /> I-Kpn I RAM] coding sequence cassette ampicillin-        transformed yeast genonuc library two <br /> resistance plasmid pUC119. The RAM) cassette gener-           transformants exhibited plasmid-dependent <br /> ated PCR utilizing oligonucleotide primers over-      Ts&apos; growth. <span id='am-81' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-82' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span> recovered transform- <br /> lapped initiation codon termination codon       ants restored normal growth mating properties ram2 <br /> reported DPRI (41). The relevant sequence 5&apos;           mutant strains. Restriction enzyme mapping revealed the <br /> portion cassette GTCGACTAIG, under-           plasmids contained overlapping inserts common <br /> lined ATG indicates initiation codon RAM].                 region DNA &gt;11 kbp (Fig. 2). To determine the <br />    Preparation E. cofl Extracts Farnesyltrnsferase As-     gene cloned corresponded locus identified the <br /> says. E. coli extracts prepared culturing JM110           ram2-1 mutation, integrated LEU2 &apos;-marked plasmid <br /> double transformants mid-logarithmic phase followed         (pBH43) containing fragment cloned DNA a <br /> induction 0.5 mM isopropyl fB-D-thiogalactoside          RAM2 + strain. Integration targeted chromosomal <br /> incubation 1 hr. Cells collected, washed extrac-      locus corresponding cloned fragment digestion with <br /> tion buffer (50 mM sodium Mes, pH 6.5/0.1 mM EGTA/0.1             Sph I (see Fig. 2), correctly targeted integration was <br /> mM MgC12/5% glycerol/2 mM phenylmethylsulfonyl fluo- <br /> ride), transferred 0.5-ml tubes, pelleted, lysed 200               A                                  2 <br /> ,l extraction buffer glass beads Vortex mixing.                                      i---       S       P       S   -I P <br /> Soluble fractions prepared centrifugation 50,000 x                                                                      P~~~~~~~~~~~~~~~~~~~ <br />                                                                                    41    _ <br /> g 1 hr.                                                                        40&gt; <br />    Other Methods. Fractionation yeast detection of <br /> a-factor Ras2 proteins immunoprecipitation were <br /> performed described (26, 36). Yeast. extracts pre- <br /> pared described (32). Farnesyltransferase assays using <br /> recombinant p21lS substrate performed analyzed                   B                   &apos; <br />                                                                                                  1              2                3 <br /> as described (32). Assays using synthetic a-factor peptides                                  7   D      S       P       3        P   S <br /> were carried 250 pmol peptide essentially                       p <br /> described (31).                                                                pp2.,Et   46 a* <br />                                                                                                             ,       -w * <br />  <br />               EXPERIMENTAL RESULTS <br />   Identification Characterization ram2 Mutation.         FIG. 1. Localization Ras proteins a-factor ram mutants. <br /> Several yeast mutations suppress RAS2vall9, including ram),       (A) SDS/PAGE analysis immunoprecipitated Ras2 proteins from <br /> cyr), capl/srv2 (26, 27, 42, 43). Among set            soluble (S) particulate (P) fractions extracts prepared from <br /> RAS2v8119 extragenic suppressor mutants derived heat-          [35S]methionine-labeled wild-type mutant yeast strains, of <br /> shock selection MATa rasf RAS2va&quot;l9 strain PT1-6         contained high-copy RAS2 plasmid (26). Lanes: 1, wild-type <br /> (26), recessive mutations fall       (SP1); 2, ram2-1 (STS39); 3, ram)-) (STS11). The soluble precursor <br /> known complementation groups. One suppressors,           form Ras2 (41 kDa) particulate mature form Ras2 (40 <br /> STS39, displayed nonselected phenotypes: inability      kDa) marked arrows. (B) SDS/PAGE analysis immuno- <br /> grow high temperatures (370C) significant mating    precipitated a-factor total extracts (T) particulate (P) and <br />                                                                   soluble (S) fractions total extracts prepared [35Slcysteine- <br /> defect. The mating defect determined MATa-              labeled wild-type mutant yeast strains, contained <br /> specific correlated defective a-factor production.       high-copy MFal plasmid (29). Lanes: 1, wild-type (SM1229); 2, <br /> Based cosegregation phenotypes           ram)::URA3 (SM1866); 3, ram2-) (SM1863). Three intracel- <br /> following cross wild-type strain subsequent anal-      lular forms a-factor marked arrows (PO, P1, P2). <br />             Biochemistry: He et al.                                                                Proc. Natl. Acad. Sci. USA 88 (1991)                               11375 <br />                                                                               -444    TCTAGAGTATAATTATTTTGAATGAGTTTCGTTGACATTTTCAGTTCATGCGATCATCTT <br />                                                                               -384    ACTTTCATGGTTGCTCCTTGCCGTTAAAATTGAAACGAAAAAGCGGGAATACAACAATG <br />                                                                               -324    TCACCTTACACGAGCAGAGAAAAGGGCAGCAATAGCGACAATAACCGATAATGCCAGGCC <br />                                                                               -264    TTCCAGAGCCCTATGCTGACCTATATTCATCATGACGCTATCCATATGCAACTAGAGACC <br />                                                                               -204    TGCTGGGAAGCGCTCTCTCATTTTTGAAGGTTGCAGCAGCTTGCGCGCCGGTTTTCGATT <br />                                                                               -144    GGGCCTCGTGATAAAACACATCAGCAAAGTTTCGCTTTTGACATTCTGTTAAATTGGTCC <br />                                                                                -84    TTAACTGTTGGTGCAGACCTACTGC.AAGACTGTTAATTATGGTATATTATATATCAAGTG <br />                                                                                -24    CTGATACAACGTACACCATCGAAAATGGAGGAGTACGATTATTCAGACGTTAAACCTTTG <br />                                                                                                                    1       M   E   E    Y   D   Y   S   D    V   K   P   L <br />                                                                                 36    CCCATTGAGACAGACTTGCAGGATGAACTGTGCAGGATTATGTATACCGAGGATTATAAG <br />                                                                                 12    P   I   E   T   D   L    Q       D   E   L   C    R   I   M   Y   T    E   D   Y   K <br />                                                                                 96    CGGTTGATGGGACTCGCAAGGGCTCTGATCAGCCTTAACGAACTGTCACCCAGGGCACTA <br />                                        clo   CL                                 32    R   L   M   G   L   A    R       A   L   I   S    L   N   E   L   S    P   R   A   L <br />            enuJL           xen x       = = <br />    E                            ~ ~ gU CO (1          G <br />                                                                        I <br />                                                                        Eco     156    CAGCTAACAGCCGAAATTATCGACGTGGCGCCAGCCTTCTACACCATATGGAACTACCGA <br />                                                               0. <br />                                                                                 52    Q   L   T   A   E   I    I   D       V   A   P    A   F   Y   T   I    W   N   Y   R <br />                                                       I <br />           L-       I   I       I   -   .     L                          I.. <br />                                                                         s5     216    TTCAATATCGTCAGGCACATGATGAGTGAATCCGAAGACACTGTCTTGTACCTGAACAAG <br />    0           1           2       3              4       5        6            72    F   N   I   V   R   H    M   M       S   E   S    E   D   T   V   L    Y   L   N   K <br />                                    RAM2                                        276 <br />                                                                                 92 <br />                                                                                       GAATTAGACTGGCTAGATGAAGTTACGCTGAATAATCCAAAGAACTATCAGATCTGGTCC <br />                                                                                       E L D       W   L   D    E   V       T   L   N    N   P   K   N   Y    Q   I   W   S <br />    FIG. 2. Restriction map RAM2 locus ram2::LEU2                336 <br />                                                                                112 <br />                                                                                       TATAGACAGTCTCTTTTGAAGCTACATCCGTCTCCTTCCTTCAAAAGAGAGCTGCCTATC <br /> gene disruption construct. The restriction map RAM2 locus                   Y   R   Q S     L   L    K       L   H   P   S    P   S   F   K   R    E   L   P   I <br /> shown. Both original library plasmid isolates contained                 396    TTAAAACTGATGATTGATGATGATTCCAAGAATTATCACGTTTGGTCGTACAGAAAGTGG <br />                                                                                132    L       L   M   I   D    D   D       S            Y <br /> additional regions DNA (-6 kbp) shown map.                        K                                    K   N        H   V   W   S    Y   R   K   W <br /> Shown map -y-disruption vector pram2::LEU2,                   456    TGCTGTTTGTTCTTCAGTGACTTTCAACATGAGCTCGCCTACGCCAGCGACCTCATCGAG <br />                                                                                152    C   C   L   F   F   S    D   F       Q <br /> which contains noncontiguous fragments RAM2 region                                                                H   E    L   A   Y   A   S    D   L   I   E <br /> inserted head-to-tail orientation yeast integration vector       516 <br />                                                                                172    T   D   I   Y   N   N    S   A       W       H <br /> pRS305 (39). Integration plasmid RAM2 locus results                                                            T        R   M   F   Y   W    V   N   A   K <br /> in deletion RAM2 coding region resides             576    GATGTCATTTCAAAAGTGGAATTGGCCGACGAGCTCCAGTTCATTATGGACAAGATTCAA <br />                                                                                192    D V     I   S   K   V    E   L                        Q <br /> indicated Bgl II Sac I restriction sites.                                                                              A   D   E    L       F   I   M    D   K   I   Q <br />                                                                               636     TTGGTTCCGCAGAACATCAGTCCGTGGACCTACCTCCGTGGTTTCCAAGAGCTATTCCAT <br />                                                                               212     L   V   P   Q   N   I    S <br /> confirmed DNA-blot hybridization transformant                                                           P       W   T   Y    L   R   G   F   Q    E   L   F   H <br /> BHYDC1. In cross ram2-1 ieu2 strain BHYDC1,                        696 <br />                                                                               232 <br />                                                                                       GATAGGCTACAGTGGGATAGCAAAGTAGTCGACTTCGCCACAACCTTCATCGGTGACGTA <br />                                                                                       D R     L   Q   W   D    S   K       V   V   D    F   A   T   T   F    I   G   D   V <br /> the segregation pattern LEU2 + ram2-1 indicated <br />                                                                               756     TTGTCACTTCCAATTGGCTCACCAGAGGATTTGCCCGAGATCGAGTCCTCATATGCCCTG <br /> complete linkage (no recombinants 25 tetrads),                      252     L   S   L   P   I   G    S   P       E   D   L    P   E   I   E   S    S   Y   A   L <br /> therefore conclude cloned gene RAM2.                       816     GAATTCCTGGCATATCACTGGGGGGCAGACCCTTGTACCCGAGACAACGCTGTTAAGGCC <br />    Subclones original plasmids constructed                    272     E   F   L   A   Y   H    W   G       A   D   P    C   T   R   D   N    A   V   K   A <br /> and based ability complement ram2 mutation                    876     TATAGCTTGCTAGCAATCAAATACGATCCTATTAGAAAAAACTTGTGGCACCACAAAATA <br /> we determined entire RAM2 gene (or                        292     Y   S   L   L   A   I    K   Y       D   P   I    R   K   N   L   W    H   H   K   I <br /> functional segment) resided 1.4-kbp Xba I-Sph I                    936 <br />                                                                               312 <br />                                                                                       AATAATCTGAACTGGAATATATGTAAGCACTCTATTTGTACGAATATATTTACGTGTGTAC <br />                                                                                       N   N   L   N   * <br /> fragment shown Fig. 2. The DNA sequence region <br />                                                                               996     GTGCATGC <br /> was determined strands contains open <br /> reading frame encode protein 316 amino acids if <br /> the ATG used initiate translation (Fig. 3). The                     FIG. 3. Nucleotide sequence RAM2 region predicted <br /> predicted molecular mass Ram2 protein 38 kDa.                       amino acid sequence Ram2 protein. <br />    Reconstitution Farnesyltransferase Activity. To investi-                  We tested farnesyltransferase activity be <br /> gate possibility Raml Ram2 proteins                   reconstituted mixing E. coli extracts containing Raml <br /> structural subunits yeast farnesyl-protein transferase,                extracts containing Ram2 alone. The results <br /> we expressed RAM] RAM2 genes E. coli and <br /> assayed extracts farnesyltransferase activity. Using                    shown Table 1 indicate extracts, mixed, do <br /> substrate peptide corresponding C-terminal sequence                    exhibit farnesyltransferase activity. The activity pro- <br /> of K-rasB, control extracts extracts prepared                        duced mixed extract -25-fold lower that <br /> bacterial strains expressed RAM] RAM2 gene                     observed E. coli extracts cells coexpressed <br /> alone completely inactive (Table 1). In contrast, ex-                    Table 1. Farnesyltransferase activity extracts E. coli <br /> tracts prepared strain coexpressed RAM) and <br /> RAM2 exhibited high levels famesyltransferase activity <br />                                                                                     E. coli                   RAM gene expressed                            [3H]Farnesyl <br /> (Table 1). We determined specific activity                                                                                                  transferred, <br /> such E. coli extracts 100-fold higher                transformant                    RAM]                     RAM2                 pmol tube <br /> similarly prepared wild-type yeast extracts (data shown).                 BHE-1                             -                        - <br />                                                                                                                                          &lt;0.006 <br />    Extracts prepared E. coli cells coexpressed                      BHE-2                     +                                - <br />                                                                                                                                          &lt;0.004 <br /> RAM) RAM2 efficiently farnesylated synthetic                         BHE-3                     -              +                 &lt;0.002 <br /> a-factor substrate. We observed SDS/PAGE analysis                     BHE-4                     +              +              5.5 &#177; 0.1 <br /> control extracts derived E. coli carrying plasmids with-                 Mixed extract             +              +              0.19 &#177; 0.02 <br /> out RAM inserts produced <span id='am-86' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span>unidentified</span> product                           The E. coli host JM110 transformed sequentially two <br /> [3H]FPP (Fig. 4A, lane 3) related a-factor               different plasmids generate strains used experiment. <br /> it observed reactions peptide                    Each assay (25 jl) performed duplicate contained 50 mM <br /> absent (Fig. 4A, lane 9). In contrast, using extracts cells              potassium phosphate (pH 7.4), 10 mM MgCl2, 5 mM dithiothreitol, <br /> that coexpressed RAM] RAM2, reaction product                        0.2% Tween 20, 50 ,ug soluble extract indicated E. coil <br /> observed (Fig. 4A, lane 1) reactions                    transformant, 90 pmol biotinylated KTSCVIM peptide, 25 <br /> utilizing extracts prepared E. coli expressed                pmol [3H]FPP. After incubation 30 min 300C, of <br /> the RAM) RAM2 gene (Fig. 4A, lanes 4 5). This                    [3H]farnesyl transferred peptide measured described (38). <br /> product comigrated authentic mature a-factor produced                    For mixed extract, 50 zg BHE-2 BHE-3 E. coli extracts <br />                                                                               mixed microcentrifuge tube left ice 30 min before <br /> in vivo (Fig. 4B) detected control reactions                   performing assay. In separate experiment, BHE-4 extract <br /> containing synthetic a-factor peptide ter-                   transferred 1.6 &#177; 0.1 pmol [3H]farnesyl bacterially produced <br /> minal-AAX residues (of CAAX sequence) missing                        p211, control extracts listed transferred &lt;6 fmol Ras <br /> (Fig. 4A, lane 8).                                                            substrate. <br />  11376         Biochemistry: He et al.                                                        Proc. -Natl. Acad. Sci. USA 88 (1991) <br />  <br />           A                                     B                              significantly defective growth low temperatures <br />           1     2   3   4   5   6   7   8   9     1   2                        completely unable grow 37TC (Fig. 5). <br />    2.3- <br />                                                                                   The viability ram) null mutants marked contrast to <br />                                                     z-.,   --ii---a-factor   lethal effects disrupting RAM2 gene. This due <br />                                                                                genetic variations strains employed disruption <br />                                                                                RAM] lethal strains used test effects <br />                                                                                RAM2 disruption (data shown). To explore pos- <br />    FIG. 4. SDS/PAGE analysis a-factor famesyltransferase ac-                sibility residual protein farnesyltransferase activity in <br /> tivity E. coli extracts. (A) Farnesylation assays a-factor         ram) null mutants, prepared soluble extract ram) <br /> quindecapeptide substrate performed (see text) analyzed               null mutant assayed farnesyltransferase activity. No <br /> by SDS/PAGE fluorography. Unless stated, reac-               reaction product observed using a-factor quinde- <br /> tion mixtures contained buffer, [3H]FPP, a-factor 15-mer sub-              capeptide substrate, long exposures fluo- <br /> strate, E. coli extract. Mock reactions shown          rogram (data shown). The [3H]farnesyl trans- <br /> E. coli extract peptide substrate (lane 1) E. coli extract         ferred K-rasB peptide using ram] null mutant <br /> (lane 2) absent. Lanes 3-7 display complete reactions containing <br /> extract control E. coli strain BHE-1 (lane 3), extract           extract significantly higher blank values (&lt;2 <br /> BHE-2, expresses RAM] (lane 4), extract BHE-3,                fmol), parallel reactions extract prepared from <br /> which expresses RAM2 (lane 5), mixture extracts                wild-type strain SP1 transferred 284 + 50 fmol. Since we <br /> BHE-2 BHE-3 (lane 6), extract BHE-4, coex-                  did detect residual protein farnesyltransferase activity in <br /> presses RAM] RAM2 (lane 7). Lanes 8 9 represent control                ram] null mutants, conclude activity not <br /> reactions identical shown lane 7 substitution         required yeast viability. This raises paradox, RAS <br /> the 12-mer a-factor substrate (lane 8) peptide substrate (lane 9).       function essential yeast. Possible resolution the <br /> (B) Comigration vitro farnesylated a-factor authentic                paradox, explanations disparate effects dis- <br /> a-factor. Lane 1, reaction mixture lane 7 A. Lane 2,             rupting RAM) RAM2, discussed below. <br /> a-factor immunoprecipitated culture supernatant of <br /> a MATa strain radiolabeled [35S]cysteine (36, 44). <br /> RAM) RAM2 genes (Table 1). The lower activity of <br />                                                                                                     DISCUSSION <br /> mixed extracts observed experiments utilizing                      We characterized newly identified yeast gene, RAM2, <br /> Ras proteins substrate (data shown) reactions                    shown expression E. coli RAM) and <br /> utilizing quindecapeptide a-factor substrate (Fig. 4A,                  RAM2 encode required components yeast <br /> lane 6).                                                                       enzyme responsible farnesylation a-factor Ras <br />    Effect Disruption RAM) RAM2 Genes Yeast. To                    proteins. The finding Raml Ram2 proteins are <br /> examine Raml-Ram2 farnesyl-protein transfer-                       required reconstitution protein farnesyltransferase <br /> ase essential yeast viability, plasmids                   activity parallels finding mammalian Ras protein <br /> contained deletion alleles RAM) RAM2. By DNA                            farnesyltransferase comprised different polypeptide <br />                                                                                subunits (19-21). It noteworthy components <br /> transformation, constructed diploids heterozy-                    mammalian enzyme, f3 subunit subunit, <br /> gous disrupted RAM alleles. Analysis oftetrads                  structurally homologous Raml Ram2, respectively <br /> from ram2/RAM2+ diploid indicated ram2 null mu-                         (21, 45, 46). <br /> tants nonviable; 2:2 segregation viability ob-                       RAS function essential yeast, mutant Ras pro- <br /> served 30 tetrads, 60 viable colonies Leu-,                    teins farnesylated supply essential RAS <br /> indicating contained wild-type RAM2 gene. By                       function unless significantly overexpressed (47-49). It is <br /> microscopic examination, observed ram2- spores                         somewhat surprising RAM), abso- <br /> occasionally germinated divided times                  lutely required farnesyl-protein transferase activity, not <br /> ceased growing.                                                                essential gene. One possible explanation, based the <br />    To test effects disrupting RAM) gene,                     ability CAAX peptides efficient substrates the <br /> analyzed tetrads derived ram) -/RAM) + diploid. We                      farnesyl-protein transferase serve efficient sub- <br /> observed 300C ram) spores germinated slowly                        strates GGTase I, ram) - mutant cells small but <br /> formed colonies, indicating ram) null mutants viable                  sufficient Ras proteins geranylgeranylated by <br /> (Fig. 5). Further analysis indicated ram) null mutants                    yeast GGTase I (22, 23, 25). Accordingly, propose that <br />  <br />  <br />  <br />  <br />    FIG. 5. Tetrad analysis raml-/RAM)Idiploids growth phenotype segregants. (A) Tetrads sporulation dissection <br /> of diploid SM1589 (ram:: URA3/RAMI), derived SM1060 (36), grown YEPD plates 6 days 30&apos;C. Nine 10 tetrads <br /> displayed 2:2 segregation slow growth. All slow-growing colonies determined Ura&apos;, indicating contained ram):: URA3 <br /> allele. (B) All colonies single tetrad streaked YEPD plate incubated 300C 3 days. The ramP:: URA3 <br /> segregants grew slower wild-type segregants. (C) All colonies single tetrad streaked YEPD plate <br /> and incubated 370C 3 days. The ramP:: URA3 segregants did grow all. <br />            Biochemistry: He et al.                                                      Proc. Natl. Acad. Sci. USA 88 (1991)            11377 <br /> ram2- null mutants nonviable result                 18. Kinsella, B. T. &amp; Maltese, W. A. (1991) J. Biol. Chem. 266, <br /> deficient farnesyltransferase activity deficient                     8540-8544. <br /> GGTase I activity. This proposal based                 19. Reiss, Y., Goldstein, J. L., Seabra, M. C., Casey, P. J. &amp; <br /> finding mammalian subunit farnesyl-protein                  Brown, M. S. (1990) Cell 62, 81-88. <br />                                                                          20. Reiss, Y., Seabra, M. C., Armstrong, S. A., Slaughter, C. A., <br /> transferase, homologous Ram2, appears                  Goldstein, J. L. &amp; Brown, M. S. (1991) J. Biol. Chem. 266, <br /> a structural subunit GGTase I (21, 24). In addition, ram2-1                10672-10677. <br /> mutants shown somewhat defective                      21. Kohl, N. E., Diehl, R. E., Schaber, M. D., Rands, E., Soder- <br /> GGTase I activity (23). Further analysis required                  man, D. D., He, B., Moores, S. L., Pompliano, D. L., Ferro- <br /> fully understand role Ram2 prenyltransferase                  Novick, S., Powers, S., Thomas, K. A. &amp; Gibbs, J. B. (1991) <br /> reactions.                                                                   J. Biol. Chem., press. <br />    Two yeast enzymes postulated involved              22. Reiss, Y., Stradley, S. J., Gierasch, L. M., Brown, M. S. &amp; <br /> in post-translational modification Ras proteins                    Goldstein, J. L. (1991) Proc. Nail. Acad. Sci. USA 88, 732-736. <br />                                                                          23. Moores, S. L., Schaber, M. D., Mosser, S. D., Rands, E., <br /> a-factor definitively assigned particular                    O&apos;<span id='am-1' about='obo:NCBITaxon_385297' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_186628 obo:NCBITaxon_1 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7995 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489793 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_385271 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Hara" datatype="xsd:string"></span>Hara</span>, M. B., Garsky, V. M., Marshall, M. S., Pompliano, <br /> genes <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> reconsti-                     D. L. &amp; Gibbs, J. B. (1991) J. Biol. Chem. 266, 18884-18888. <br /> tuted expression E. coli. In addition Raml-Ram2             24. Seabra, M. C., Reiss, Y., Casey, P. J., Brown, M. S. &amp; <br /> farnesyl-protein transferase, shown STEJ4                    Goldstein, J. L. (1991) Cell 65, 429-434. <br /> encodes methyltransferase modifies a-factor Ras             25. Yokoyama, K., Goodwin, G. W., Ghomshchi, F., Glomset, <br />                                                                              J. A. &amp; Gelb, M. H. (1991) Proc. Nail. Acad. Sci. USA 88, <br /> proteins (36). In absence STEJ4 function, prior steps                  5302-5306. <br /> in Ras protein processing, presumably prenylation prote-              26. Powers, S., Michaelis, S., Broek, D., Santa <span id='am-66' about='obo:NCBITaxon_48819' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_214597 obo:NCBITaxon_131567 obo:NCBITaxon_214607 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1477611 obo:NCBITaxon_26122 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Anna" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Anna</span>-A., S., Field, <br /> olytic removal C-terminal amino acids,                     J., Herskowitz, I. &amp; Wigler, M. (1986) Cell 47, 413-422. <br /> significantly delayed. This finding suggests different              27. Fujiyama, A., Matsumoto, K. &amp; Tamanoi, F. (1987) EMBO J. <br /> steps modification Ras proteins, biochem-                 6, 223-228. <br /> ically separable, coupled vivo.                        28. Wilson, K. L. &amp; Herskowitz, I. (1987) Genetics 155, 441-449. <br />                                                                         29. Michaelis, S. &amp; Herskowitz, I. (1988) Mol. Cell. Biol. 8, <br />   S.P. thanks Michael Wigler, laboratory STS39 strain            1309-1318. <br /> was isolated, continuing encouragement. We grateful N.   30. Nakayama, N., Arai, K.-I. &amp; Matsumoto, K. (1988) Mol. Cell. <br /> Kohl J. Gibbs communicating results publication, M.           Biol. 8, 5410-5416. <br /> Kozak critical reading manuscript, S. Ferro-Novick   31. Schafer, W. R., Trueblood, C. E., Yang, C., Mayer, M. P., <br /> discussions. This work supported National Institutes               Rosenberg, S., Poulter, C. D., Kim, S.-H. &amp; Rine, J. (1990) <br /> Health Grants GM41258 (S.P.) GM41223 (S.M.), American                    Science 249, 1133-1139. <br /> <span id='am-33' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Society Grant MG-1 (S.P.), American Heart Asso-           32. Goodman, L. E., Judd, S. R., Farnsworth, C. C., Powers, S., <br /> ciation-Maryland Affiliate Grant-in-Aid (S.M.).                              Gelb, M. H., Glomset, J. A. &amp; Tamanoi, F. (1990) Proc. Nail. <br />                                                                              Acad. Sci. USA 87, 9665-9669. <br />                                                                         33. Ohya, Y., Goebl, M., Goodman, L. E., Petersen-Bjorn, S., <br />   1. Gibbs, J. G. (1991) Cell 65, 1-4.                                       Friesen, J. D., Tamanoi, F. &amp; Ankura, Y. (1991) J. Biol. Chem. <br />   2. Glomset, J. A., Gelb, M. H. &amp; Farnsworth, C. C. (1990)                  266, 12356-12360. <br />       Trends Biochem. Sci. 15, 139-142.                                 34. Finegold, A. A., Johnson, D. I., Farnsworth, C. C., Gelb, <br />   3. Rine, J. &amp; Kim, S. (1990) New Biol. 2, 219-226.                         M. H., Glomset, J. A. &amp; Tamanoi, F. (1991) Proc. Natl. Acad. <br />   4. Goldstein, J. L. &amp; Brown, M. S. (1990) Nature (London) 343,             Sci. USA 88, 4448-4452. <br />      425-430.                                                           35. Rossi, G., Jiang, Y., Newman, A. P. &amp; Ferro-Novick, S. (1991) <br />   5. Der, C. J. &amp; Cox, A. D. (1991) Cancer Cells 3, 331-340.                 Nature (London) 351, 159-161. <br />   6. Anderegg, R. J., Betz, R., Carr, S. A., Crabb, J. W. &amp; Duntze,     36. Hrycyna, C. A., Sapperstein, S. K., Clarke, S. &amp; Michaelis, S. <br />      W. (1988) J. Biol. Chem. 263, 18236-18240.                              (1991) EMBO J. 10, 1699-1709. <br />  7. Casey, P. J., Solski, P. A., Der, C. J. &amp; Buss, J. E. (1989)        37. Broek, D., Samiy, N., Fasano, O., Fujiyama, A., Tamanoi, F., <br />      Proc. Natl. Acad. Sci. USA 86, 8323-8327.                               Northup, J. &amp; Wigler, M. (1985) Cell 41, 763-769. <br />  8. Ishibashi, Y., Sakagami, Y., Isogai, A. &amp; Suzuki, A. (1984)         38. Reiss, Y., Seabra, M. C., Goldstein, J. L. &amp; Brown, M. S. <br />      Biochemistry 23, 1399-1404.                                             (1990) Meth. Compan. Meth. Enzymol. 1, 241-245. <br />  9. Farnsworth, C. C., Wolda, S. L., Gelb, M. H. &amp; Glomset, J.          39. Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19-27. <br />      (1989) J. Biol. Chem. 264, 20422-20429.                            40. Broek, D., Toda, T., Michaeli, T., Levin, L., Birchmeier, C., <br /> 10. Yamane, H. K., Farnsworth, C. C., Xie, H., Howald, W.,                   Zoller, M., Powers, S. &amp; Wigler, M. (1987) Cell 48, 789-799. <br />      Fung, B. K.-K., Clarke, S., Gelb, M. &amp; Glomset, J. A. (1990)       41. Goodman, L. E., Perou, C. M., Fujiyama, A. &amp; Tamanoi, F. <br />      Proc. Natl. Acad. Sci. USA 87, 5868-5872.                               (1988) Yeast 4, 271-281. <br /> 11. Yamane, H. K., Farnsworth, C. C., Xie, H., Evans, T., Gelb,         42. Fields, J., Vojtek, A., Ballester, R., Bolger, G., Collicelli, J., <br />      M., Glomset, J. A., Clarke, S. &amp; Fung, B. K.-K. (1991) Proc.            Ferguson, K., Gerst, J., Kataoka, T., Michaeli, T., Powers, S., <br />      Natl. Acad. Sci. USA 88, 286-290.                                       Riggs, M., Rodgers, L., Weiland, I., Wheland, B. &amp; Wigler, M. <br /> 12. Kawata, M., Farnsworth, C. C., Yoshida, Y., Gelb, M. H.,                 (1990) Cell 61, 319-327. <br />      Glomset, J. A. &amp; Takai, Y. (1990) Proc. Natl. Acad. Sci. USA       43. Fedor-Chaiken, M., Deschennes, R. J. &amp; Broach, J. R. (1990) <br />      87, 8960-8964.                                                          Cell 61, 329-340. <br /> 13. Clarke, S., Vogel, J. P., Deschennes, R. J. &amp; Stock, J. (1988)      44. Berkower, C. &amp; Michaelis, S. (1991) EMBO J., press. <br />      Proc. Natl. Acad. Sci. USA 85, 4643-4647.                   <br /> </body></html>