<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proteomic analysis identifies a new complex <br /> required for nuclear pre-mRNA retention <br /> and splicing <br />  <br />  <br />                                                                  some onto the pre-mRNA. Interestingly, two different but <br />                                                   evolutionarily related spliceosomes co-exist in most eukar- <br />                                                    yotic cells. Formation of major spliceosomes involves the U1, <br />                                                 U2, U4, U5 and U6 snRNPs, while that of the minor spliceo- <br />                                                             somes requires U11, U12, U4ATAC, U5 and U6ATAC snRNPs. For <br />  <br />                                                                                         major spliceosomes, U1 snRNP binding to the intron 50 splice <br />             site allows the formation of a commitment complex for yeast <br />  <br />  <br />                                                            or complex E in mammals (Se&#180;raphin and Rosbash, 1989a; <br />                                                                                         Michaud and Reed, 1991). <span id='am-144' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-147'  ></span><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> step is followed by pre- <br /> Using the proteomic tandem affinity purification (TAP)                                  spliceosome formation through U2 snRNP addition. Finally, <br /> method, we have purified the <span id='am-50' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisie U2                                 the U4/U6.U5 triple snRNP joining generates the complete <br /> snRNP-associated splicing factors SF3a and SF3b. While                                  spliceosome (Pikielny et al, 1986; Konarska and Sharp, 1987). <br /> SF3a purification revealed only the expected subunits                                   After remodelling steps, during which the U1 and U4 snRNPs <br /> Prp9p, Prp11p and Prp21p, yeast SF3b was found to con-                                  dissociate, the first splicing reaction occurs. A second remo- <br /> tain only six subunits, including previously known com-                                 delling step precedes the last transesterification reaction. <br /> ponents (Rse1p, Hsh155p, Cus1p, Hsh49p), the recently                                   Recently, an alternative scenario, suggesting that a pre-as- <br /> identified Rds3p factor and a new small essential protein                               sembled complex of all snRNPs binds the pre-mRNA as a <br /> (Ysf3p) encoded by an unpredicted split ORF in the yeast                                single unit, was proposed (Stevens et al, 2002). In this model, <br /> genome. Surprisingly, Snu17p, the proposed yeast ortho-                                 the spliceosome also must undergo remodelling before each <br /> logue of the seventh human SF3b subunit, p14, was not                                   catalytic step. In the cell, molecular mechanism must act in <br /> found in the yeast complex. TAP purification revealed that                              parallel to splicing to ensure that pre-mRNAs are not ex- <br /> Snu17p, together with Bud13p and a newly identified                                     ported to the cytoplasm before their complete processing. <br /> factor, Pml1p/Ylr016c, form a novel trimeric complex.                                   This process is not well understood. Many factors involved in <br /> Subunits of this complex were not essential for viability.                              spliceosome assembly have associated pre-mRNA leakage <br /> However, they are required for efficient splicing in vitro                              and splicing inhibition phenotypes (Legrain and Rosbash, <br /> and in vivo. Furthermore, inactivation of this complex                                  1989). Pre-mRNA leakage is then likely to result indirectly <br /> causes pre-mRNA leakage from the nucleus. The corre-                                    from pre-mRNA accumulation, even though this is not always <br /> sponding complex was named pre-mRNA REtention and                                       the case (Rutz and Se&#180;raphin, 2000). More recently, the <br /> Splicing (RES). The presence of RES subunit homologues                                  new yeast factor, Mlp1, was directly implicated in nuclear <br /> in numerous eukaryotes suggests that its function is                                    pre-mRNA retention without affecting splicing itself (Galy <br /> evolutionarily conserved.                                                               et al, 2004). <br /> The EMBO Journal (2004) 23, 4847&#8211;4856. doi:10.1038/                                        U2 and U6 snRNPs have been implicated in catalysis of <br /> sj.emboj.7600482; Published online 25 November 2004                                     splicing in the active spliceosome (Valadkhan and Manley, <br /> Subject Categories: RNA; genomic &amp; computational biology                                2001), while U5 snRNP is known to interact with exons <br /> Keywords: RES; SF3a; SF3b; tandem affinity purification                                 (Newman and Norman, 1992; Wyatt et al, 1992). <br /> (TAP); U2 snRNP                                                                         Spliceosome remodelling allows the U2 and U6 snRNAs to <br />                                                                                         form a base pair (Brow, 2002). In addition, these snRNAs <br />                                                                                         interact with the pre-mRNA: U2 snRNA is involved in <br />                                                                                         branchpoint adenosine <span id='am-191' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> (Parker et al, 1987), while <br /> Introduction                                                                            U6 snRNA selects the 50 splice site (Kandels-Lewis and <br /> Pre-mRNA splicing occurs by two sequential transesterifica-                             Se&#180;raphin, 1993; Lesser and Guthrie, 1993). The U2 snRNP <br /> tion reactions that are catalysed by a large and dynamic                                is a large structure containing the U2 snRNA, seven Sm <br /> ribonucleoprotein complex: the spliceosome. Every splicing                              proteins and two U2-specific proteins: U2A0 and U2B00 <br /> reaction necessitates the stepwise assembly of a full spliceo-                          (Lea1p and Msl1p in <span id='am-33' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>) (Lu&#168;hrmann <br />                                                                                         et al, 1990; Tang et al, 1996; Caspary and Se&#180;raphin, 1998). In <br /> *Corresponding author. Centre de Ge&#180;ne&#180;tique Mole&#180;culaire, CNRS                         addition, two multisubunit complexes associate with U2 <br /> UPR2167, Avenue de la Terrasse, 91198 Gif sur Yvette, France.                           snRNP: SF3a and SF3b (Brosi et al, 1993). SF3a is composed <br /> Tel.: &#254; 33 1 69 82 38 84; Fax: &#254; 33 1 69 82 38 77; <br /> E-mail: seraphin@cgm.cnrs-gif.fr                                                        of three polypeptides (SF3a120, SF3a66, SAP130, in human; <br /> 4 <br />   Present address: EPO, Landsbergerstr. 30, 80339 Mu&#168;nchen, Germany                     Prp9p, Prp11p and Prp21p in yeast; Kra&#168;mer, 1996), while <br /> 5 <br />   Present address: MediGene, Lochhamerstr. 11, 82152 Martinsried,                       human SF3b was reported to contain seven proteins (SAP130, <br /> Germany <br />                                                                                         SAP155, SAP145, SF3b49, SF3b14b, p14 and SF3b10)(Gozani <br /> Received: 29 June 2004; accepted: 21 October 2004; published online:                    et al, 1996; Das et al, 1999; Will et al, 2002). Yeast <span id='am-156' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-159'  ></span><span id='am-160' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>- <br /> 25 November 2004                                                                        logues of five of these factors have been characterized (Wells <br />  <br /> &amp; 2004 European Molecular Biology Organization                                                                       The EMBO Journal   VOL 23 | NO 24 | 2004 4847 <br />        Yeast SF3a, SF3b and RES splicing factors <br />       A Dziembowski et al <br />  <br />  <br />  <br />       et al, 1996; Igel et al, 1998; Caspary et al, 1999; Wang and       A             SF3a                    B             SF3b <br />  <br />  <br />  <br />  <br />                                                                                                                           TAP Rse1p <br />  <br />                                                                                                                                       Ysf3p TAP <br />       Rymond, 2003). Interestingly, human SF3a acts exclusively <br />       during the splicing of major introns (Will et al, 1999), while                  Prp9p TAP <br />  <br />       SF3b is required for splicing of both types of introns by <br />       associating with U2 and U12 snRNPs, respectively (Das                                                    212 <br />       et al, 1999; Will et al, 1999). Thus, both factors are essential                                                                            CBP Rse1p <br />                                                                                 212                            158                                Rse1p <br />       for splicing. Furthermore, many SF3b subunits can be cross- <br />       linked to the branchpoint region of the pre-mRNA (Staknis                 158                            116                                Hsh155p <br />       and Reed, 1994; Gozani et al, 1996; Query et al, 1996;                                                     97 <br />       McPheeters and Muhlenkamp, 2003). Interestingly, the                      116 <br />                                                                                                                  66 <br />       human SF3b p14 subunit that crosslinks directly to the                     97                                                               Cus1p <br />                                                                                                                  56 <br />       branchpoint lies in the centre of a shell made by the other <br />                                                                                                                  43 <br />       SF3b subunits in a structural model of the complex obtained                66               Prp9p CBP <br />  <br />       by cryo-<span id='am-176' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-180'  ></span>electron</span> microscopy (Golas et al, 2003).                           56 <br />                                                                                                                  36 <br />          We have used the TAP proteomic approach developed                       43                              27 <br />       in our laboratory (Rigaut et al, 1999; Puig et al, 2001), to                                                                                Hsh49p <br />                                                                                  36               Prp21p         20 <br />       characterize the yeast U2 snRNP-associated complexes.                                       Prp11p <br />                                                                                                                                                   Ysf3p CBP <br />                                                                                  27 <br />       Identification of the SF3a subunits by mass spectrometry                                                   14                               Rds3p <br />                                                                                                                                                   Ysf3p <br />       revealed the presence of the three previously known factors:               20                               7 <br />                                                                                  14 <br />       Prp9p, Prp11p and Prp21p, but no additional component. In- <br />       depth analysis of SF3b revealed six subunits, four of which                                                              1      2 <br />       had been identified previously, namely Rse1p, Hsh155p, <br />                                                                              Yeast Ysf3p    MAEKQRQLKLQKIYKQKYIGLGDESTTREQWQRNVRNDTLNTLQGHSASLEYVSLSRGDL <br />       Cus1p and Hsh49p (Wells et al, 1996; Igel et al, 1998;             C   Human SF3b10   MTDRYTIHSQLEHLQSKYIGTGHADTTKWEWLVNQHRDSYCSYMGHFDLLNYFAIAENES <br />                                                                                             *:::    . : :.**** *. .**: :* * :.*: : **        *:*.:::..: <br />       Caspary et al, 1999; Puig et al, 2001). We also found Rds3p <br />                                                                              Yeast Ysf3p    SIRDTRIHLLKSMS-P-GYKAYLREER <br />       that was independently identified while this work was in               Human SF3b10   KAR-VRFNLMEKMLQPCGPPADKPEEN <br />                                                                                             . * .*::*::.* * * *     **. <br />       progress (Wang and Rymond, 2003). Interestingly, we identi- <br />       fied a novel, unpredicted small yeast protein, as a new            Figure 1 Characterization of yeast SF3a and SF3b complexes. <br />       subunit of this complex. This subunit, encoded by a small          (A) TAP purification of yeast SF3a using a PRP9 TAP fusion <br />                                                                          identifies three subunits. The TAP purified material was fractio- <br />       split yeast ORF, is essential for yeast growth. Surprisingly,      nated on a 5&#8211;20% gradient SDS&#8211;PAGE gel and stained with <br />       however, Snu17p, which was suggested to represent the yeast        Coomassie Blue. Proteins were identified by MALDI. (B) TAP <br />       orthologue of the human p14 branchpoint-binding protein            purification of yeast SF3b complex using TAP Rse1 or Ysf3 TAP <br />       (Gottschalk et al, 2001; Wang and Rymond, 2003), was               fusions identifies six subunits. The purified material was fractio- <br />                                                                          nated on a 5&#8211;20% gradient SDS&#8211;PAGE gel, which was then stained <br />       consistently absent from our purifications. TAP purification       with silver. Proteins were identified by MALDI. (C) Sequence <br />       revealed that Snu17p is present in an independent trimeric         alignment of human SF3b10 protein and its yeast homologue <br />       complex. Subunits of this new complex, called pre-mRNA             Ysf3p. Symbols indicate identity (*), high (:) or low (.) similarity. <br />       REtention and Splicing (RES), are dispensable for yeast cell <br />       viability. These proteins are, however, conserved in eukar- <br />       yotes and required for efficient pre-mRNA splicing in vivo and     Prp21p. We conclude that yeast SF3a contains only three <br />       in vitro. More importantly, inactivation of RES subunits           subunits. <br />       induces nuclear pre-mRNA leakage. Overall, our proteomic <br />       analysis of the yeast SF3a and SF3b splicing factors estab-        Characterization of yeast SF3b reveals new subunits <br />       lished definitively their composition, revealing altogether the    TAP purification of the yeast U2 snRNP led us previously to <br />       presence of unsuspected subunits. It also led to the identifica-   identify and characterize Rse1p, a yeast homologue to human <br />       tion and functional characterization of the new RES complex.       SF3b130 (Caspary et al, 1999). Hsh155p was also identified in <br />       These findings have implication for our understanding of the       the purified material (<span id='am-192' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-193' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-196'  ></span>data</span> not shown). We used a N-terminal <br />       composition and organization of the mammalian splicing             TAP-tagged Rse1p to characterize yeast SF3b (Puig et al, <br />       machinery.                                                         2001). Mass spectrometry analysis identified previously <br />                                                                          known components of yeast SF3b (Rse1p, Hsh155p, Cus1p <br />                                                                          and Hsh49p; Puig et al, 2001) as well as a new 14 kDa protein: <br />       Results                                                            Rds3p (data not shown). While this work was in progress, the <br />       Yeast SF3a contains three subunits                                 presence of Rds3p in yeast SF3b was independently reported <br />       The yeast SF3a subunits, Prp9p, Prp11p and Prp21p, were            (Wang and Rymond, 2003). Additional smaller products were <br />       identified genetically and, based on their homology to human       consistently seen in SF3b purifications (e.g., Figure 1B, lane <br />       subunits and in vitro studies, suggested to be SF3a compo-         1), but we were unable to identify them in the yeast protein <br />       nents (reviewed in Kra&#168;mer, 1996). However, as yeast SF3a          database through mass spectrometry (data not shown). The <br />       was not purified, it remained possible that more subunits          recent purification of human SF3b revealed the presence of <br />       were present and/or that these three proteins did not form the     two new small subunits of 14 and 10 kDa, namely SF3b14 and <br />       predicted complex. Thus, we purified SF3a using a Prp9p TAP        SF3b10 (Will et al, 2002). While the similarity between <br />       fusion. Purified products were fractionated by SDS&#8211;PAGE.           SF3b14 and Rds3p was noted, SF3b10 was reported to have <br />       This consistently revealed three proteins present in apparent      no homologues in yeast. BLAST searches against the yeast <br />       relative stoichiometric amount (Figure 1A). These factors          genome database revealed, however, a sequence that, when <br />       were identified by mass spectrometry as Prp9p, Prp11p and          translated, showed significant similarity to human SF3b10 <br />  <br /> 4848 The EMBO Journal VOL 23 | NO 24 | 2004                                                       &amp; 2004 European Molecular Biology Organization <br />                                                                                                                      Yeast SF3a, SF3b and RES splicing factors <br />                                                                                                                                          A Dziembowski et al <br />  <br />  <br />  <br /> (Figure 1C). The corresponding ORF had not been predicted              with our findings, the protein profile was different from the <br /> because a putative intron disconnects the initiating ATG from          pattern observed in SF3b purifications (compare Figures 1B <br /> the downstream coding sequence. To confirm that this new               and 2A). Three proteins were present in apparent relative <br /> gene, yeast splicing factor 3b subunit (YSF3), was active, we          stoichiometric amounts. This new complex was named RES <br /> performed a 50 RACE analysis. Sequencing of the resulting              (see below). Mass spectrometry identified its subunits as <br /> product confirmed that this region was transcribed and that            Snu17p, Bud13p and the product of the YLR016c ORF (pre- <br /> the putative intron was removed, thus generating the pre-              mRNA leakage 1 (Pml1p), see below), as well as low levels of <br /> dicted coding sequence (data not shown). To confirm that the           contaminating ribosomal and heat shock proteins that were <br /> cognate polypeptide was made and that it was a subunit of              not reproducibly found (data not shown). None of the RES <br /> yeast SF3b, we fused the TAP tag to the C-terminus of Ysf3p.           subunits were detected or identified in the six TAP purifica- <br /> Western blot analysis confirmed that Ysf3p was expressed               tions performed with tagged SF3b subunits (see above). <br /> and the profile of the six subunits recovered after TAP                   Biocomputing analyses revealed that RES subunits are <br /> purification, gel fractionation and silver staining was identi-        conserved and that they contain various domains (Table I). <br /> cal to the one observed following purification of TAP Rse1p            Like human p14, Snu17p possesses an RNA-binding domain <br /> (Figure 1B, compare lanes 1 and 2; note the expected mobility          (RBD). Interestingly, BLAST searches revealed that Snu17p is <br /> change of Rse1p and Ysf3p due to the tag). Mass spectro-               more related to the human CGI-79 protein than to p14. <br /> metry identified unequivocally the six subunits consistently           Phylogenetic analysis of proteins similar to these factors, <br /> recovered in such purifications in apparent relative stoichio-         rooted using the RBD domain of Hsh49p, indicates that p14 <br /> metric amounts as Rse1p, Hsh155p, Cus1p, Hsh49p, Rds3p                 and Snu17p belong to two separate lineages (Figure 2B). In <br /> and Ysf3p. To confirm this finding and to rule out the                 each <span id='am-161' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, besides yeast, two factors are present, one from <br /> presence of an undetected co-migrating subunit, we TAP                 each <span id='am-107' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span>. However, no clear p14 orthologue is found in <br /> tagged the remaining subunits of the yeast SF3b complex,               the yeast genome. This result is consistent with our finding <br /> namely Hsh155p, Cus1p, Hsh49p and Rds3p. Parallel TAP                  that human and yeast SF3b have different compositions and <br /> purification with the six SF3b subunits revealed that the              that Snu17p belongs to a new complex. Bud13p contains a <br /> major proteins recovered co-migrated (except for the tagged            unique, phylogenetically conserved, C-terminal region of <br /> subunit that was enlarged by the expected mass) and that no            unknown function, while the C-terminus of Pml1p contains <br /> additional subunit could be detected (data not shown).                 a conserved FHA domain, implicated in phosphothreonine <br /> Overall, these results demonstrated that yeast SF3b contains <br /> only six subunits, including the newly identified Rds3p and <br /> Ysf3p factors. <br />                                                                            RES <br />                                                                        A Snu17                    B <br />                                                                                TAP <br /> Snu17p/Ist3 belongs to a new evolutionally conserved                   212   d                      P14 orthologues                       Snu17 orthologues <br />                                                                        158 <br /> trimeric complex: RES                                                  116 <br />                                                                         97 <br />  <br />  <br />  <br />  <br />                                                                                                                                                  Snu17 S.c <br /> While seven proteins have been reported to be present in <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 E.g <br />                                                                                                            P14 <br />  <br />  <br />  <br />  <br />                                                                                                                                                    Sn <br />                                                                         66                           20P14 S.j <br />                                                                                                     P14 A.t P.f <br />  <br />  <br />  <br />  <br />                                                                                                                                                      u1 <br />  <br />  <br />  <br />                                                                                                                                                               17 <br />                                                                                                                                           Sn <br /> human SF3b (Will et al, 2002), our analysis failed to detect                                                                                                              .c <br />  <br />  <br />  <br />  <br />                                                                                                                                                        7S <br />                                                                         56                         P14 O.s                             Snu1 u17 A                     7N <br />  <br />  <br />  <br />  <br />                                                                                                                                                             Snu <br />                                                                                                   P14 D.m                                   7 H.s .t <br />  <br />  <br />  <br />  <br />                                                                                                                                                          .p <br />                                                                                                                                                                    u1 <br />                                                                                                   P14 C.e                             (CGI-                     Sn <br /> the yeast homologue to the human p14 subunit (Table I).                 43                                P14 H.s <br />                                                                                                                                            79) <br />                                                                                                                                                                Sn <br />                                                                                                                                                               Sn u17 X.l <br />                                                                                                                                                                 u1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        Snu17 O.s <br />                                                                                                                                                        Snu <br /> More importantly, we could not detect the Snu17p factor,                36           Bud13p                                                                        7D <br />                                                                                                                                                                        .m <br />  <br />  <br />  <br />  <br />                                                                                                                                                            17 C <br />                                                                         27 <br /> which was suggested to represent the yeast p14 homologue                             Pml1p <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 .e <br />                                                                         20           Snu17p CBP <br /> (Gottschalk et al, 2001; Will et al, 2001; Wang and Rymond, <br /> 2003). To ascertain that Snu17p was not a yeast SF3b subunit,           14 <br />  <br /> we TAP purified Snu17p and associated factors. Consistent <br />                                                                         7                                                            HSH49 S.c <br />  <br />  <br />  <br /> Table I Comparison of yeast and human SF3a, SF3b and RES <br /> subunits                                                               Figure 2 Identification of the new RES complex associated with <br /> Yeast                  Human                     Domains               Snu17p. (A) Proteins associated with Snu17p were purified by the <br />                                                                        TAP method. Purified material was fractioned on a 10% tricine <br /> SF3a                                                                   PAGE gel and stained with Coomassie Blue. Proteins were identified <br />   Prp9p                SF3a60                    2   Zn finger         by MALDI. In addition to Snu17p, Bud13p and Pml1p, we identified <br />   Prp11p               SF3a66                    Zn finger             Ssa2p heat shock protein and some ribosomal proteins. These <br />   Prp21p               SF3a120                   Swap domain           common contaminants were not reproducibly seen in other purifi- <br />                                                                        cations (not shown). (B) Snu17 is not a P14 branchpoint-binding <br /> SF3b                                                                   protein orthologue. A phylogenetic tree of Snu17 and human P14 <br />   Rse1p                SAP130                    b-Propeller repeats   homologue RNA-binding domains was built and rooted with the <br />   Hsh155p              SAP155                    HEAT repeats          RNA-binding domain of Hsh49. Accession numbers of the se- <br />   Cus1p                SAP145                                          quences are as follows: P14 <span id='am-95' about='obo:NCBITaxon_5820' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Plasmodium" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmodium</span> falciparum (Q8I5G8); <br />   Hsh49p               SF3b49                    2   RRM               P14 <span id='am-16' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span>Arabidopsis</span> thaliana (Q9FMP4); P14 <span id='am-57' about='obo:NCBITaxon_4530' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_147367 obo:NCBITaxon_359160 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_147380 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_4527 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-58' property="oboInOwl:hasExactSynonym" content="red rice" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Oryza sativa" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="PMID:17617555" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="rice" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:19706531" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:389215" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:17907810" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:15654106" datatype="xsd:string"></span>Oryza sativa</span> (Q7XZG6); <br />   Rds3p                SF3b14b                   Zn finger             P14 <span id='am-115' about='obo:NCBITaxon_6181' typeof='obo:NCBITaxon_6157 owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6180 obo:NCBITaxon_31244 obo:NCBITaxon_6178 obo:NCBITaxon_31245 obo:NCBITaxon_6179 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Schistosoma" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span>Schistosoma</span> japonicum (Q86EF3); P14 <span id='am-110' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-111' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114'  ></span>Caenorhabditis</span> <br />   &#8212;                    p14                       RRM                   elegans (Q8ITY4); P14 <span id='am-137' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-138' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143'  ></span>Drosophila melanogaster</span> (Q9VRV7) P14 <br />   Ysf3p                SF3b10                                          <span id='am-149' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-150' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-153'  ></span><span id='am-154' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span>Homo sapiens</span> (Q9Y3B4); Snu17 C. elegans (Q18318); Snu17 <br />                                                                        D. <span id='am-186' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-187'  ></span><span id='am-188' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-189' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>melanogaster</span> (Q9VIS0); Snu17 <span id='am-125' about='obo:NCBITaxon_8355' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8360 obo:NCBITaxon_2759 obo:NCBITaxon_262014 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8353 obo:NCBITaxon_7776 obo:NCBITaxon_8352 obo:NCBITaxon_30319 obo:NCBITaxon_41666 obo:NCBITaxon_8292'><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Xenopus laevis (Daudin, 1802)" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasExactSynonym" content="common platanna" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="clawed frog" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasExactSynonym" content="African clawed frog" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Xenopus leavis" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Bufo laevis" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Xenopus laevis" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasExactSynonym" content="platanna" datatype="xsd:string"></span>Xenopus laevis</span> (AAH56844) <br /> RES                                                                    Snu17 <span id='am-76' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe (O94290); Snu17 O. sativa <br />   Snu17p               CGI-79                    RRM                   (Q94GF0); Snu17 <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span>Mus musculus</span> (Q8R0F5); Snu17 <span id='am-90' about='obo:NCBITaxon_5140' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-91' property="rdfs:label" content="Neurospora" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Neurospora</span> <br />   Bud13p               MGC13125                                        <span id='am-11' about='obo:NCBITaxon_688987' typeof='owl:Thing obo:NCBITaxon_37581 obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_116123 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_57992 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="rdfs:label" content="Crassa" datatype="xsd:string"></span>crassa</span> (CAE76416); Snu17/CGI-79 H. sapiens (Q9Y3I8); <br />   Pml1p                Snip                      FHA domain            Snu17 <span id='am-207' about='obo:NCBITaxon_33170' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-208' property="oboInOwl:hasAlternativeId" content="NCBITaxon:33168" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasAlternativeId" content="NCBITaxon:46675" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-211'  ></span><span id='am-212' property="oboInOwl:hasAlternativeId" content="NCBITaxon:244059" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasAlternativeId" content="NCBITaxon:244058" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasRelatedSynonym" content="Ashbya" datatype="xsd:string"></span><span id='am-215' property="rdfs:label" content="Eremothecium" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasAlternativeId" content="NCBITaxon:33172" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasAlternativeId" content="NCBITaxon:217889" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasRelatedSynonym" content="Holleya" datatype="xsd:string"></span>Eremothecium</span> gossypii (AAS51937); Snu17 S. cerevisiae <br />                                                                        (YIR005W); Hsh49 S. cerevisiae (YOR319W). <br />  <br /> &amp; 2004 European Molecular Biology Organization                                                        The EMBO Journal            VOL 23 | NO 24 | 2004 4849 <br />        Yeast SF3a, SF3b and RES splicing factors <br />       A Dziembowski et al <br />  <br />  <br />  <br />       binding (Hammet et al, 2000). Interestingly, Pml1p was                                <br />       identified in complex containing all five snRNPs, supporting <br />  <br />  <br />       a role in splicing (Stevens et al, 2002). <br />  <br />       SF3b subunits are essential for yeast growth, while <br />       inactivation of RES causes slow growth phenotypes <br />                                                                         U4 snRNA <br />       To understand the role of the newly identified factors, we <br />       disrupted the corresponding genes in yeast. Inactivation of <br />       RDS3 and YSF3 in a diploid strain followed by tetrad analysis <br />       revealed that they are, like all other SF3b subunits, essential <br />                                                                         U2 snRNA <br />       for vegetative growth (data not shown). In contrast, all RES <br />       subunits were dispensable but their inactivation generated <br />       a slow growth phenotype that was exacerbated at 371C <br />       (Figure 3). Interestingly, this phenotype was stronger for <br />       BUD13 and SNU17 inactivation than for PML1. Overall, the <br />       genetic analysis was consistent with the biochemical data <br />       indicating the presence of two functionally different com-        U6 snRNA <br />       plexes. <br />  <br />       RES is not associated with snRNAs <br />       SF3b subunits have been shown to co-precipitate the U2 <br />       snRNA in yeast and human cells, while Snu17p was also <br />       proposed to associate with this species (Gottschalk et al, <br />       2001).We performed coimmunoprecipitation experiments to           U5 snRNA <br />       assess whether RES subunits were associated with the U2 <br />       snRNP. Extracts from strains expressing either Snu17p, <br />       Bud13p, Pml1p, or as positive controls the U2 snRNP protein <br />       Lea1p (U2A0 ), or SF3b subunits Rse1p and Ysf3p fused to the      U1 snRNA <br />       TAP tag were precipitated with IgG-coated beads and the <br />       presence of snRNAs in the pellets was analysed by primer <br />                                                                                       1 2 3 4 5 6 7          8 9 10 11 12 13 14 15 <br />       extension. A wild-type extract served as a negative control. <br />       Surprisingly, for tagged Snu17p, Bud13p and Pml1p, like for       Figure 4 In contrast to SF3b, the RES complex is not stably <br />                                                                         associated with snRNAs. Extracts from a control wild-type yeast <br />       the wild-type extract, we did not detect snRNA co-precipita- <br />                                                                         strain (WT) and strains harbouring TAP-tagged Lea1p, Rse1p, <br />       tion (Figure 4). In contrast, under these conditions, Lea1p,      Ysf3p, Snu17p, Bud13p or Pml1p were immunoprecipitated with <br />       Rse1p and Ysf3p co-precipitated efficiently the U2 snRNA as       IgG-coated beads. RNA extracted from input and pellet was ana- <br />       well as small amounts of U5 and U6 snRNAs. Western                lysed by primer extension with primers specific for the U1, U2, U4, <br />                                                                         U5 and U6 snRNAs. The positions of the corresponding signals are <br />       blotting revealed that more than 50% of all tagged proteins <br />                                                                         shown on the left. RNA from a four times more extract was used for <br />       were immunoprecipitated (data not shown). Thus, lack of           the pellets relative to the input fractions. <br />       snRNA co-precipitation by RES subunits does not result from <br />       inefficient pull-down. These data confirm that Snu17 is not <br />       an SF3b subunit, and indicates further that RES is not stably <br />       associated with an snRNP.                                         reactions and assaying its association with pre-mRNA, spli- <br />                                                                         cing intermediates and/or products. Extracts from a wild-type <br />       RES associates with pre-mRNA in vitro                             strain and strains expressing TAP-tagged Snu17p, Bud13p, <br />       In the absence of snRNA pull-down, we tested for a link           Pml1p and Lea1p, Rse1p, Ysf3p were incubated either with <br />       between RES and splicing by performing in vitro splicing          radioactively labelled wild-type pre-mRNA or mutant pre- <br />                                                                         mRNA containing a branchpoint deletion as a negative con- <br />                                                                         trol. After precipitation with IgG-coated beads, RNAs were <br />                                                                         extracted from pellets and detected by autoradiography fol- <br />                           30&#176;C                             37&#176;C <br />                                                                         lowing denaturing acrylamide gels electrophoresis. <br />           WT                                        WT                  Supernatants and one input were analysed in parallel to <br />                                                                         control for precipitation efficiency and absence of RNA <br />       &#8710;snu17                                 &#8710; snu17                    degradation. As expected (Figure 5), Lea1p, Rse1p and Ysf3 <br />                                                                         precipitated significant amount of pre-mRNA over the back- <br />       &#8710; bud13                                &#8710; bud13                    ground observed with wild-type extract or with the mutant <br />                                                                         pre-mRNA lacking a branchpoint (Se&#180;raphin and Rosbash, <br />        &#8710; pml1                                     &#8710; pml1                1991). In addition, Rse1p and Ysf3p co-precipitated low but <br />                                                                         significant amount of lariat intermediate, while Lea1p effi- <br />       Figure 3 RES subunits are required for wild-type growth. Serial   ciently pulled down splicing intermediates and the lariat <br />       dilutions of strains harbouring deletion of the RES subunits <br />                                                                         intron. The pre-mRNA was also co-precipitated with the <br />       (Dsnu17, Dbud13, or Dpml1) and an isogenic wild-type strain <br />       control were placed on YPDA plates and incubated at 30 and 371C   tagged RES, even though to a lower extent than with the <br />       for 2 and 3 days, respectively.                                   other tagged factors. The signal was significant and specific, <br />  <br /> 4850 The EMBO Journal VOL 23 | NO 24 | 2004                                                 &amp; 2004 European Molecular Biology Organization <br />                                                                                                                                                                                                               Yeast SF3a, SF3b and RES splicing factors <br />                                                                                                                                                                                                                                   A Dziembowski et al <br />  <br />  <br />  <br />                                                                        Immunoprecipitations                                                                                         Extract            WT                         &#8710;bud13 <br />                                           Supernatants                                                                             Pellets                                            RES         &#8722;             +             &#8722;              + <br />  <br />  <br />  <br />  <br />                                                                                  Bud13 TAP <br />  <br />  <br />  <br />  <br />                                                                                                                                                            Bud13 TAP <br />                                                                      Snu17 TAP <br />  <br />  <br />  <br />  <br />                                                                                                                                                Snu17 TAP <br />                                                           Pml1 TAP <br />  <br />  <br />  <br />  <br />                                                                                                                                     Pml1 TAP <br />                                     TAP Rse1 <br />  <br />  <br />  <br />  <br />                                                                                                              TAP Rse1 <br />                          Lea1 TAP <br />  <br />  <br />  <br />  <br />                                                                                                   Lea1 TAP <br />                                                Ysf3 TAP <br />  <br />  <br />  <br />  <br />                                                                                                                         Ysf3 TAP <br />               WT input <br />                                                                                                                                                                                 Time (min)    0 15 30 45    0 15 30 45    0 15 30 45    0 15 30 45 <br />    Extract <br />  <br />  <br />  <br />  <br />                                                                                              WT <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        WT <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />  <br />              &#8710;bp <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br />  <br />              WT <br /> Pre-mRNA <br />  <br />  <br />  <br />  <br />              1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 <br />  <br /> Figure 5 SF3b and RES associate with the pre-mRNA. Splicing <br /> reaction were assembled with a wild-type pre-mRNA and, as a <br /> negative control, a branchpoint deletion pre-mRNA mutant, using <br /> extracts prepared from an untagged wild-type strain or isogenic <br /> strains harbouring the TAP tag, fused either to Lea1p, Rse1p, Ysf3p, <br /> Snu17p, Bud13p or Pml1p. RNA immunoprecipitated with IgG                                                                                                                                      1 2 3 4       5 6 7 8       9 10 11 12   13 14 15 16 <br /> sepharose (pellet) and RNA remaining in the supernant (super- <br /> natant) were extracted together with one input fraction, fractionated                                                                                                       Figure 6 Extracts lacking Bud13p are defective in splicing in vitro <br /> by denaturing acrylamide gel electrophoresis and detected by                                                                                                                and inhibition of splicing can be relieved by addition of purified RES <br /> phosphorimaging. Structures of the various species are indicated                                                                                                            complex. Splicing reactions were performed using extracts prepared <br /> on the left.                                                                                                                                                                from wild-type strain or isogenic Dbud13 strains. Products of 0, 15, <br />                                                                                                                                                                             30 and 45 min reactions were fractionated by denaturing gels <br />                                                                                                                                                                             electrophoresis and detected by phosphorimaging. To reconstitute <br />                                                                                                                                                                             activity, TAP-purified RES complex was added to the reaction <br /> however, as it was consistently above background detected                                                                                                                   mixtures prior to addition of the pre-mRNA. <br /> with the wild-type extract or the mutant pre-mRNA. Pre- <br /> mRNA co-precipitation indicated that RES associates with <br /> the spliceosome before step 1; the weaker signal relative to <br /> Lea1p, Rse1p and Ysf3p suggests that RES interaction is                                                                                                                     spliceosomes (data not shown). This result is consistent with <br /> weaker, more transient and/or that the RES-tagged subunit                                                                                                                   a role of SF3b in the commitment complex&#8211;pre-spliceosome <br /> is less accessible when incorporated into spliceosome.                                                                                                                      transition. <br />  <br /> Snu17p and Bud13p are required for efficient splicing                                                                                                                       RES is required for efficient splicing in vivo <br /> in vitro                                                                                                                                                                    Taking advantage of the viability and thermosensibility of the <br /> To gain further evidence for the function of the newly                                                                                                                      Dbud13, Dsnu17 and Dpml1 mutant strains, we tested the <br /> discovered proteins in splicing, we analysed in vitro splicing                                                                                                              effect of RES subunit inactivation on pre-mRNA splicing <br /> and spliceosome formation, by incubation of radiolabelled                                                                                                                   in vivo. Reporter <span id='am-202' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-203' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-206' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> containing the RP51A intron or <br /> pre-mRNA in the extract lacking RES subunits. Interestingly,                                                                                                                mutants thereof inserted within the lacZ reading frame were <br /> spliceosome formed in extracts lacking Bud13p or Snu17p                                                                                                                     introduced in these strains and splicing of these construct <br /> migrated faster than complexes formed in wild-type extracts,                                                                                                                assayed by b-galactosidase assays and by primer extension <br /> while commitment complex formation and mobility were                                                                                                                        analysis (Jacquier et al, 1985). <br /> unaffected (data not shown). In contrast, removal of Pml1p                                                                                                                     Splicing of the wild-type reporter, assayed by the produc- <br /> did not alter spliceosome migration. Analysis of the reaction                                                                                                               tion of b-galactosidase, was essentially normal in the three <br /> products revealed that in vitro splicing is significantly inhib-                                                                                                            mutant strains (at most two-fold reduction compared to an <br /> ited before the first catalytic step in the absence of Bud13p                                                                                                               isogenic wild-type strain, Figure 7A). We obtained similar <br /> (Figure 6) or Snu17p (data not shown). Again, removal of                                                                                                                    results for a branchpoint mutant (data not shown). In con- <br /> Pml1p had no significant effect. To control for the specificity                                                                                                             trast, the reporter mRNA carrying a poor 50 splice site (50 II: <br /> of this effect, we added TAP-purified RES complex to the                                                                                                                    GUAUaU) was poorly spliced in strains lacking Snu17p or <br /> reaction, which partially restored the in vitro splicing defi-                                                                                                              Bud13p grown at 251C (100-fold reduction of b-galactosidase <br /> ciency of Dbud13 extracts (Figure 6) and Dsnu17 (data not                                                                                                                   levels). This effect was strongly exacerbated at 371C (500-fold <br /> shown). In contrast, purified RES addition to a wild-type                                                                                                                   reduction compared to wild type). Inactivation of Pml1p <br /> extract did not affect splicing. These results demonstrate a                                                                                                                resulted in a milder phenotype as a 15-fold reduction of <br /> direct role for RES in splicing in vitro and, together with the                                                                                                             b-galactosidase could only be detected by combining the <br /> co-precipitation of pre-mRNA, they indicate that it acts before                                                                                                             50 splice site mutation and the nonpermissive temperature. <br /> the first splicing step.                                                                                                                                                       To ascertain that splicing was affected by RES inactivation <br />    Analysis of the new SF3b subunit also provided evidence                                                                                                                  and to determine which step was inhibited, we assessed <br /> for their implication in splicing in vitro (data not shown).                                                                                                                mRNA, pre-mRNA and lariat intermediate levels by a more <br /> Strikingly, extracts containing an Rds3p TAP fusion formed                                                                                                                  direct and sensitive primer extension analysis. For the wild- <br /> commitment complexes, but were unable to assemble pre-                                                                                                                      type reporter, we observed an accumulation of pre-mRNA in <br />  <br /> &amp; 2004 European Molecular Biology Organization                                                                                                                                                        The EMBO Journal     VOL 23 | NO 24 | 2004 4851 <br />        Yeast SF3a, SF3b and RES splicing factors <br />       A Dziembowski et al <br />  <br />  <br />  <br />       A      Reporter                         WT           5&apos; II              WT            5&apos; II            mutant reporter, mRNA levels were significantly reduced in <br />           Temperature                         25&#730;C         25&#730;C                37&#730;C            37&#730;C          the Dsnu17 and Dbud13 backgrounds. In the Dpml1 strain, <br />                     6                                                                                        pre-mRNA accumulation was significantly lower, and a com- <br />                   Log(&#946;-gal activity)   5                                                                    bination of high temperature and 50 splice site mutation was <br />                                         4                                                                    required to detect some mRNA decrease. In all cases, the <br />                                         3 <br />                                                                                                              latter splicing block occurs concomitantly with a reduction of <br />                                                                                                              the lariat intermediate level indicative of a first step block. <br />                                         2 <br />                                                                                                              Overall, these data demonstrate that RES is required for <br />                                         1                                                                    efficient splicing. Furthermore, RES appears to act before <br />                                         0                                                                    the first splicing step and to be more critical for introns <br />                                                                                                              with weak 50 splice sites. <br />   <br />  <br />       B     Reporter                        WT         5&apos; II         WT             5&apos; II                    RES is required for nuclear pre-mRNA retention in vivo <br />         Temperature                         25&#176;C       25&#176;C           37&#176;C         37&#176;C <br />                                                                                No reporter                   To check whether, in addition to splicing process itself, RES <br />                                                                                                              complex is also involved in pre-mRNA retention in the <br />  <br />               Strain                                                                                         nucleus, we used a previously described reporter system <br />  <br />                                                                                                              composed of two related plasmids with either the pre- <br />                                                                                                              mRNA (pre-mRNA in-frame) or the mRNA (mRNA in- <br />                                                                                                              frame) encoding b-galactosidase (Rain and Legrain, 1997). <br />                                                                                                              Henceforth, b-galactosidase activity generated by the pre- <br />                                                                                                              mRNA in-frame construct allows for the estimation of pre- <br />                                                                                                              mRNA leakage from the nucleus, while enzymatic activity <br />                                                                                                              originating from the mRNA in-frame construct reports spli- <br />                                                                                                              cing. As a positive control, we have used isogenic Mlp1 <br />                                                                                                              knockout strain, which is known to have pre-mRNA leakage <br />                                                                                                              phenotype without associated splicing defect (Galy et al, <br />                                                                                                              2004). For all the three RES knockouts, we observed a <br />                                                                                                              significant pre-mRNA leakage at 251C, evidenced by in- <br />                                                                                                              creased b-galactosidase activity with the pre-mRNA in- <br />                                                                                                              frame construct (Figure 8A). This effect was not exacerbated <br />                                                                                                              at 371C (data not shown). Splicing (mRNA in-frame con- <br />                                                                                                              struct) was not more affected in the Dpml1 background <br />                                                                                                              than in the Dmlp1 strain in contrast to the significant reduc- <br />                                                                                                              tion observed for Dsnu17 and Dbud13. Thus, while mRNA <br />                                                                                                              leakage observed in the absence of Snu17 or Bud13 may <br />                                                                                                              result from poor splicing, the effect observed in Dpml1 is, like <br />                                                                                                              for Dmlp1 (Galy et al, 2004), likely to be direct. This conclu- <br />                                                                                                              sion is further supported by the observation that splicing <br />                                                                                                 Endogenous <br />                                                                                                              defects are weak and thermosensitive with this mutant <br />                                                                                                              (see above), while pre-mRNA leakage is not. Primer exten- <br />                                                                                                              sion analyses confirmed that, for Dpml1 as for Dmlp1, there is <br />                                         1 2 3 4      5 6 7 8       9 10 11 12 13 14 15 16 17 <br />                                                                                                              no significant increase in pre-mRNA level in comparison <br />                                                                                                              to wild type (Figure 8B and C), while splicing of this <br />       Figure 7 RES subunits are involved in splicing in vivo. The splicing                                   reporter RNAs was defective for Dbud13 and Dsnu17, with <br />       efficiency of reporter constructs was analysed at two different <br />       temperatures (25 and 371C) in a wild-type strain or mutant lacking                                     a three-fold increase in pre-mRNA to mRNA ratio. These <br />       SNU17, BUD13 or PML1. Splicing reporters contained either the                                          results indicate that the major function of Pml1 is nuclear <br />       wild-type RP51A intron (WT) or a 50 splice site mutant derivative                                      pre-mRNA retention. Furthermore, the RES complex, at the <br />       (50 II), interrupting the reading frame of the lacZ gene. (A)                                          edge between splicing and pre-mRNA retention, may <br />       b-Galactosidase activity of the wild-type and 50 II reporters. Given <br />       the wide variation, data are presented on logarithmic scale. In each                                   connect these processes particularly for introns with weak <br />       strain, activities are normalized to the activity of a construct                                       50 splice sites. <br />       containing no intron in the lacZ gene. (B) Primer extension analysis <br />       of RNA present in the various strains. As a control, primer extension <br />       using RNA isolated from wild-type strain without reporters is <br />       depicted. Owing to multiple transcription initiation sites of the <br />       inducible GAL promoter, mRNAs appear as multiple bands.                                                Discussion <br />       Positions of reporter-derived pre-mRNA, mRNA, lariat intermediate <br />       and endogenous RP51A mRNA are indicated.                                                               Using a proteomic strategy, we have characterized the yeast <br />                                                                                                              SF3a and SF3b splicing factors and identified a new complex, <br />                                                                                                              RES, required for efficient intron removal and nuclear pre- <br />       the Dsnu17 and Dbud13 strains, both at 25 and 371C                                                     mRNA retention. These data reveal interesting similarities <br />       (Figure 7B), with no concomitant significant effect on the                                             and unexpected differences between the yeast and mamma- <br />       lariat intermediate or mRNA levels. For the 50 splice site                                             lian splicing systems. <br />  <br /> 4852 The EMBO Journal VOL 23 | NO 24 | 2004                                                                                     &amp; 2004 European Molecular Biology Organization <br />                                                                                                                             Yeast SF3a, SF3b and RES splicing factors <br />                                                                                                                                                 A Dziembowski et al <br />  <br />  <br />  <br /> A       Reporter : Pre-mRNA in-frame                             mRNA in-frame                human SF3b contains an additional protein, p14, which was <br />                       (leakage)                                    (splicing) <br />                25                                                                             shown to crosslink to the branchpoint adenosine (Query et al, <br />                                    20                                                         1997; Will et al, 2001). It was previously suggested that <br />            &#946;-gal activity                                                                     Snu17p is the yeast orthologue of p14 and a subunit of <br />                                    15 <br />                                                                                               SF3b (Gottschalk et al, 2001; Wang and Rymond, 2003). <br />                                    10 <br />                                                                                               Our data strongly argue against this conclusion, for several <br />                                      5 <br />                                                                                               reasons: <br />                                      0 <br />                                                                                               (1) TAP purification of SF3b, using any of the six tagged <br />                                                                                                    subunits, failed to reveal a protein migrating at the <br />                                          <br />  <br />  <br />  <br />  <br />                                                                 &#8710; <br />                                                        Strain                                     position expected for Snu17. Furthermore, mass spectro- <br />                                                                                                   metry analyses failed to identify this factor. <br /> B   Reporter : Pre-mRNA in-frame                                mRNA in-frame <br />                                                                                               (2) TAP purification of Snu17p reveals that it belongs to a <br />  <br />                                                                                                   SF3b subunits copurified with it. The presence of an <br />                                                                                                   unaccessible fraction of Snu17 in SF3b is unlikely be- <br />                                                                                                   cause more than 80% of Snu17 TAP can be retained on <br />                                                                                                   an IgG column (data not shown). Furthermore, quanti- <br />                                                                                                   tative Western blotting indicates that SF3b and RES <br />                                                                                                   subunits are present in roughly similar quantities <br />                                                                                                   (Ghaemmaghami et al, 2003). <br />                                                                                               (3) Tagged Snu17p was unable to precipitate U2 snRNA, a <br />                                                                                                   characteristic feature of SF3b subunits. <br />                                                                                               (4) SNU17 deletion indicates that, like other RES subunits <br />                                    1 2 3 4 5   6 7 8 9 10   11 12 13 14 15   16 17 18 19 20       but contrasting with SF3b subunits, it is not an essential <br />                                                                                                   gene. <br /> C      Reporter : Pre-mRNA in-frame                             mRNA in-frame <br />                                                                                               (5) SNU17 inactivation did not lead to commitment complex <br />               10 <br />              Pre-mRNA/mRNA ratio <br />  <br />  <br />  <br />  <br />                9                                                                                  accumulation, as for SF3b mutants, but to a later-stage <br />                8 <br />                7                                                                                  defect in the splicing process. <br />                6                                                                              (6) Snu17p is more similar to the human CGI-79 protein than <br />                5 <br />                4                                                                                  to p14. Furthermore, a true p14 orthologue is not found <br />                3 <br />                2                                                                                  in S. cerevisiae. <br />                1 <br />                0                                                                                 Given this compelling evidence, why was Snu17 suggested <br />     to be a yeast SF3b subunit? First, Northern blot analysis of <br />                                  RNAs co-precipitating with Snu17 suggested a weak specific <br />                                                                                               association with U2 snRNA (Gottschalk et al, 2001). However, <br /> Figure 8 RES subunits are involved in nuclear pre-mRNA retention <br /> in vivo. Phenotypes of Dsnu17, Dbud13 or Dpml1 were analysed <br />                                                                                               the large size of U2 snRNA and differences in probe specific <br /> using reporters containing a synthetic intron reporting pre-mRNA                              activity were not taken into account to correct the strong U2 <br /> leakage or splicing (&#8216;pre-mRNA in-frame&#8217; and &#8216;mRNA in-frame&#8217;,                                 signal before concluding that U2 snRNA was more specifi- <br /> respectively). Isogenic wild-type and DMlp1 strains were used as                              cally precipitated than other snRNAs (see Figure 2 in <br /> controls. (A) b-Galactosidase activity of pre-mRNA retention repor- <br />                                                                                               Gottschalk et al, 2001). It is conceivable that spliceosomes, <br /> ters. Activity is expressed as a % of the activity of a construct <br /> containing no intron. (B) Primer extension analysis of the RNA                                rather than free snRNPs, were detected. Second, based on this <br /> produced by the reporters containing the pre-mRNA and mRNA in-                                result, Wang and Rymond (2003) detected a protein band <br /> frame. The experiment was made in duplicate using two indepen-                                corresponding in size to Snu17p in a purified SF3b fraction. <br /> dent transformants. Owing to multiple transcription initiation sites <br />                                                                                               However, this protein was not identified by mass spectro- <br /> of the inducible GAL promoter, mRNAs appear as multiple bands. <br /> Positions of pre-mRNA and mRNA are indicated. (C) Quantification                              metry. It may thus simply correspond to a contaminant or a <br /> of the pre-mRNA/mRNA ratio from the primer extension. Error bars                              degradation product. <br /> are calculated from the analysis of independent transformants.                                   Taken together, the available data are consistent with the <br />                                                                                               fact that yeast SF3b contains only six subunits. All available <br />                                                                                               evidence, including our results on Rds3p and Ysf3p, indicate <br /> Composition, organization and function                                                        that SF3b is required for the commitment complex&#8211;pre- <br /> of SF3 complexes                                                                              spliceosome transition as part of the U2 snRNP. Thus, bio- <br /> TAP purification and mass spectrometry analyses were used                                     chemical characterization of Rds3p revealed that it was <br /> to determine the composition of the SF3a and SF3b com-                                        required for pre-spliceosome formation as part of the yeast <br /> plexes. Our data indicate that yeast SF3a contains only three                                 SF3b complex (data not shown). It is unclear whether SF3b <br /> subunits: Prp9p, Prp11p and Prp21p, with no trace of other                                    remains associated with U2 snRNA in the spliceosome <br /> partners. Thus, this complex appears to be homologous to its                                  throughout the splicing process. For each of the SF3b sub- <br /> human counterparts (Table I).                                                                 units that we tested, precipitation of the pre-mRNA was <br />    Our results revealed the presence of two new subunits in                                   efficient but splicing intermediates were only recovered in <br /> yeast SF3b; it is thus composed of six proteins: Rse1p,                                       small amounts and intron lariat was not detectable. This <br /> Hsh155p, Cus1p, Hsh49p, Rds3p and Ysf3p. All of them are                                      contrasted with core U2 snRNP protein such as Lea1 <br /> homologous to human SF3b subunits (Table I). However,                                         (Figure 5). This could suggest that SF3b subunits dissociate <br />  <br /> &amp; 2004 European Molecular Biology Organization                                                                      The EMBO Journal     VOL 23 | NO 24 | 2004 4853 <br />        Yeast SF3a, SF3b and RES splicing factors <br />       A Dziembowski et al <br />  <br />  <br />  <br />       from the spliceosome shortly after the first splicing step,       Interestingly, pre-mRNA precipitation is weak. Furthermore, <br />       during rearrangements leading to the positioning of the           intermediates and splicing products are not precipitated, <br />       30 splice site in the catalytic site. However, we cannot          suggesting that RES interaction with the spliceosome is <br />       formerly rule out that all SF3b subunits are not accessible       transient. As RES is not essential, this interaction is not <br />       after the first splicing reaction, even though this seems         obligatory either. Thus, RES can be seen as a factor enhancing <br />       unlikely. We note that while SF3b subunits were detected in       splicing. This function becomes particularly important for <br />       partially purified human spliceosomes blocked after the first     weak introns carrying a poor 50 splice site, but not for those <br />       catalytic step. However, as this preparation was not homo-        carrying a weakened branchpoint. Given the similarity be- <br />       genous, SF3b may have been associated with the unspliced          tween Snu17p and p14, it will be of interest to find out which <br />       pre-mRNA still present (Jurica et al, 2002). Crosslinking         pre-mRNA and/or snRNA sequence(s) are recognized by the <br />       analyses have shown that, in the spliceosome, SF3b subunits       Snu17p RNA-binding domain. <br />       bind to the region surrounding the branchpoint, while p14            A very interesting characteristic of the RES complex is the <br />       has been shown to bind to this sequence directly (Staknis and     pre-mRNA leakage to cytoplasm that occurs in its absence. <br />       Reed, 1994; Gozani et al, 1996; Query et al, 1996; McPheeters     Moreover, for the Pml1p pre-mRNA leakage, phenotype <br />       and Muhlenkamp, 2003). The structural model of human              occurs in the absence of significant splicing defect (at <br />       SF3b obtained by cryo-electron microscopy indicates the           251C). By all criteria, Pml1p does not differ from Mlp1p, <br />       presence of p14 in the centre of a globular cage-like structure   which has been demonstrated to specifically implicated in <br />       made by the other SF3b subunits (Golas et al, 2003). It is thus   nuclear pre-mRNA retention (Galy et al, 2004). We conclude <br />       tempting to speculate that pre-mRNA threads through SF3b,         that the main function of Pml1p is also nuclear pre-mRNA <br />       with the branchpoint contacting its central component.            retention. Interestingly, disruption of MLP1 similarly to PML1 <br />       Rearrangement in the association of p14/SF3b with the pre-        generates very mild growth defect. Pml1p contains an FHA <br />       mRNA, allowing the attack of the 50 splice site by the            domain implicated in binding to phosphothreonine residues <br />       branchpoint adenosine, is thus also likely to occur before        (Hammet et al, 2000). This raises the interesting possibility <br />       the first transesterification reaction. The absence of a p14      that Pml1p could be involved in phophorylation-dependent <br />       homologue in yeast SF3b can explain some differences be-          retention of pre-mRNA in the nucleus. It could thereby play a <br />       tween in vitro spliceosome formation in the yeast and human       regulatory role in alternative splicing, particularly for intron <br />       systems. Thus, yeast pre-mRNA lacking a functional 50 splice      carrying a weak 50 splice site. Further analyses should reveal <br />       site is unable to assemble pre-spliceosomes (Se&#180;raphin and        whether this is indeed the case and whether such feature <br />       Rosbash, 1989b) although this occurs in human splicing            extends to its mammalian orthologue. <br />       extracts (Query et al, 1997). Thus, due to the presence of <br />       p14, mammalian intron recognition may occur through an <br />       initial recognition of the branchpoint or the 50 splice site;     Materials and methods <br />       however, only the latter pathway would be available to yeast.     Yeast strain construction <br />                                                                         Gene disruption and tagging on the chromosome were performed <br />       A new trimeric complex, RES, is required for efficient            using PCR fragments following a published strategy (Puig et al, <br />       splicing and nuclear pre-mRNA retention                           1998) SNU17, BUD13 and PML1 genes were disrupted with the S. <br />                                                                         pombe HIS3 marker from pFA-HIS3MX6 (Wach et al, 1997) in the <br />       Purification of Snu17p-associated proteins identified a new       haploid strain BMA64-1a (MATa, ura3-1, trp1- 2, leu2-3,112, his3-11, <br />       protein complex, RES, containing, in addition to Snu17p,          ade2-1, can1-101). RDS3 and YSF3 genes were disrupted by <br />       Bud13p and Pml1p proteins. Interaction between Snu17p             integrating K. tactis TRP1 marker from pBS1479 in the diploid <br />       and Bud13p proteins is also supported by global two-<span id='am-197' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-198' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-199'  ></span><span id='am-200' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span>        strain BSY320 (ade2, arg4, leu2-3 112, trp1-289, ura3-52). <br />                                                                            Ysf3 TAP, Snu17 TAP, Bud13 TAP and Pml1 TAP strains were <br />       studies (Uetz et al, 2000; Ito et al, 2001). Furthermore,         constructed as described previously (Rigaut et al, 1999). The TAP <br />       disruption of the RES subunit genes generates highly related      Rse1 fusion strain was described previously (Puig et al, 2001).The <br />       phenotypes including slow growth and thermosensitivity.           Dmlp1 strain was described (Galy et al, 2004). <br />       This supports their association in a complex. Pre-mRNA co- <br />       precipitation, together with in vitro and in vivo splicing        TAP purification <br />                                                                         All TAP purifications were performed as described previously from <br />       assays, demonstrates that RES is required for efficient spli- <br />                                                                         2 l of culture (Rigaut et al, 1999). Purified proteins were <br />       cing. However, inactivation of PML1 had a weaker effect than      concentrated by lyophilization, separated by SDS&#8211;PAGE and stained <br />       SNU17 or BUD13 in all the assays that we performed. In this       with Coomassie Blue or silver. <br />       vein, it is noteworthy that inactivation of either SNU17 or <br />       BUD13 also alters the yeast budding pattern (Ni and Snyder,       Mass spectrometry analysis <br />       2001), possibly by reducing the splicing of transcript(s)         Proteins were identified following &#8216;in gel&#8217; digestion (Pandey and <br />                                                                         Mann, 2000; Godovac-Zimmermann and Brown, 2001; Rappsilber <br />       encoding key factor(s) implicated in this process. All RES        and Mann, 2002). Coomassie-stained gels were treated directly, <br />       subunits have orthologues in higher eukaryotes (Table I). In      while silver-stained bands were rapidly destained using the Silver <br />       addition, the Snu17p and Bud13p human orthologues, CGI-79         Quest decoloration kit (Sigma Aldrich). Digestion was carried out <br />       and MGC13125, respectively, were identified in a purified         overnight with the addition of 50 ng of trypsin. A MALDI-Tof mass <br />                                                                         spectrometer (Voyager DE STR, Applied Biosystem) fitted with a <br />       spliceosome fraction (Rappsilber et al, 2002), supporting         pulsed nitrogen <span id='am-181' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-182'  ></span><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-185' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span> (337 nm) was used. Mass spectra were <br />       their involvement in splicing.                                    acquired in the reflectron mode. The total acceleration voltage <br />          What is the exact function of RES in splicing? Snu17p and      was 20 kV, with a grid voltage of 68% and a delay extraction of <br />       RES are clearly not stably associated with the U2 snRNA as        240 ns. Close external calibration was realized using a standard <br />                                                                         peptide mix solution ranging from 573 to 3496 Da (LaserBio Labs, <br />       proposed previously (Gottschalk et al, 2001). In vivo and         SophiaAntipolis). Samples were prepared in the CHCA matrix at a <br />       in vitro results, as well as the co-precipitation of pre-mRNA,    final concentration of 10 mg/ml in acetonitrile/trifluoroacetic acid <br />       indicate that it acts before the first catalytic step.            (70/0.1%) solution. In all, 1 ml of this sample solution corresponding <br />  <br /> 4854 The EMBO Journal VOL 23 | NO 24 | 2004                                                  &amp; 2004 European Molecular Biology Organization <br />                                                                                                           Yeast SF3a, SF3b and RES splicing factors <br />                                                                                                                               A Dziembowski et al <br />  <br />  <br />  <br /> to a dilution of 1:2 was then deposited on the MALDI target and          In vivo splicing assays <br /> dried.                                                                   The Dsnu17, Dbud13, Dpml1 strains and an isogenic control were <br />    Database scans were performed by using MS Fit and Profound            transformed with reporters: RP51A wild-type intron (HZ18), 50 II <br /> search engines. Protein identifications were obtained with a             (HZ12) (Jacquier et al, 1985), pre-mRNA in-frame (pLG-Nde1Acc1), <br /> sequence coverage of 55&#8211;86% in average, and mass accuracies of           mRNA in-frame or no intron (pLG-SD5) (Jacquier et al, 1985; Rain <br /> about 15&#8211;60 ppm.                                                         and Legrain, 1997). Reporters were assayed for b-galactosidase <br />                                                                          activity at two temperatures, 25 and 371C. Strains were grown <br /> In vitro splicing analysis                                               overnight at 251C in a synthetic medium without uracil containing <br /> Splicing reactions were as described before, except that the             2% lactate (pH 5.5), 2% glycerol and 0.05% glucose to an OD600 of <br /> incubation was performed for 30 min at 251C (Se&#180;raphin et al,            0.5&#8211;0.8. Cultures were maintained at 251C or shifted to 371C for 1 h <br /> 1988). Pre-mRNA was generated by in vitro transcription of plasmid       before a 2 h induction of b-galactosidase. b-Galactosidase activity <br /> pBS195 (wild type) or pBS199 (DUACUAAC) digested with DdeI.              was tested as described previously (Rutz and Se&#180;raphin, 2000). All <br /> Reactions were stopped by addition of 200 ml of PK buffer (0.1 M         the experiments were performed in duplicate using two indepen- <br /> Tris&#8211;HCl (pH 7.5), 12.5 mM EDTA (pH 8.0), 150 mM NaCl, 1% SDS)           dent transformants. Error bars present standard deviation. Primer <br /> containing 80 mg of proteinase K (Sigma) and 10 mg of <span id='am-169' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-170' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-171'  ></span><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span>        extensions to analyse splicing and retention reporters were <br /> coli tRNA. After incubation for 20 min at 371C, RNA was extracted        performed as described previously (Jacquier et al, 1985). <br /> and analysed in a 15% polyacrylamide&#8211;7 M urea gel. <br />    For complementation of in vitro splicing reaction with purified       Biocomputing <br /> RES complex, it was TAP purified and dialysed against buffer D           BLAST was used for database searches. Multiple sequence align- <br /> (20 mM HEPES-KOH (pH 7.9), 150 mM KCl, 8% glycerol, 0.5 mM               ments were done with ClustalW. The neighbour-joining tree was <br /> phenylmethylsulphonyl fluoride and 0.5 mM dithiothreitol. A              computed, excluding positions with gaps and correcting for <br /> volume of 1 ml of purified material (or as control buffer D) was         multiple substitutions using Clustal W (Thompson et al, 1994). <br /> added to 10 ml splicing reactions. <br />  <br /> Immunoprecipitation and primer extension <br />                                                                          <br /> Immunoprecipitation and primer extension were as described             <br /> previously (Se&#180;raphin, 1995). Immunoprecipitation of pre-mRNA            <br /> was performed similarly: Briefly, 50 ml splicing reactions were         <br /> preformed. In all, 10 ml of the reaction was extracted and kept as an    <br /> input, while the remaining 40 ml was diluted in 500 ml of IPP150        <br /> buffer (10 mM Tris (pH 8), 150 mM NaCl, 0.1% NP40) and used for           <br /> immunoprecipitation.                                                     <br />  <br />  <br /> </body></html>