<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Nad. Acad. Sci. USA <br /> Vol. 88, pp. 6986-6990, August 1991 <br /> Biochemistry <br />  <br />  <br /> Expression RNase P RNA <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> is <br /> controlled unusual RNA polymerase III promoter <br />      (RPRI gene/transcripfion factor TFUIC/internal promoters) <br /> JAE-YONG LEE, CLAIRE F. EVANSt, AND DAVID R. ENGELKEt <br /> Department Biological Chemistry, The University Michigan, Ann Arbor, MI 48109-0606 <br /> Communicated Minor J. Coon, May 20, 1991 (received review March 20, 1991) <br />  <br /> ABSTRACT           The RNA subunit <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevi-                            In report pot III promoter strategy proposed that <br /> siae nuclear RNase P encoded single-copy, essential                         circumvents need promoters located RNA <br /> gene, RPRI. The 369-nucleotide mature form RNA                           coding region. The RPRI gene Saccharomyces cerevi- <br /> an apparent precursor 84-nucleotlde 5&apos; leader --33                      siae encodes highly structured 369-nucleotide RNA sub- <br /> nucleotides additional 3&apos; sequence. Analysis RPRI tran-                       unit nuclear RNase P (14). <span id='am-28' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> RNA appears be <br /> scription strain temperature-sensitive lesion RNA                    precursor 84-nucleotide 5&apos; leader &apos;z30 <br /> polymerase m shows gene transcribed vivo                          extra 3&apos; nucleotides, ending U4-6 sequence common to <br /> RNA polymerase m. Examination potential promoter re-                             pot III terminators. It previously demonstrated the <br /> gions using progressive upstream deletions point                           human RNase P RNA (H1 RNA) gene transcribed <br /> mutations indicates sequences contained                    pol III vitro (15). Here temperature-sensitive lesion in <br /> the 5&apos; leader region essential expression vivo,                    pol III used RPRI RNA synthesized pot <br /> sequences farther upstream influence efficiency. The required                       III vivo. Further, combination upstream deletions and <br /> leader elements resemble tRNA gene-like A-box B-box                             point mutations suggests DNA upstream 5&apos; <br /> internal promoters sequence spacing. As tRNA                          leader influences efficiency expression, essential <br /> genes, transcription factor TFIIIC binds region vitro                    portions promoter A box nonconsensus B <br /> and binding severely reduced A-box B-box point                      box located leader region. Thus &quot;internal&quot; <br /> mutations Impair expression vivo. It appears                      promoter placed upstream structural RNA to <br /> the yeast RNase P RNA gene adopted promoter strategy                          drive expression. <br /> that places RNA polymerase m &quot;internal&quot; promoter <br /> upstream mature structural domain help drive                                           MATERIALS AND METHODS <br /> transcription. <br />                                                                                       Strains Genetic Methods. S. cerevisiae strain W3031A <br /> A variety eukaryotic genes encoding small RNAs                               (MATa ade2-1 his3-11,15 leu2-3,112 trpl-1 ura3-1 can1-100) <br /> transcribed RNA polymerase (pot) III. Characterization                        isogenic strain W3031B (MATa ade2-1 his3-11,15 <br /> the promoter elements genes identified                             leu2-3,112 trpl-1 ura3-1 canl-100) obtained R. <br /> divergent strategies placement transcriptional regula-                       Rothstein (Columbia University). A temperature-sensitive <br /> tory elements DNA. Early work genes 5S rRNA                           pol III mutant, D27-7C (MATa pep4::HIS+MAT ura3-52 <br /> and tRNAs demonstrated expression con-                               his3-200 trpl-901 lys2-801 ade2-101 rpcI60-41) obtained <br /> trolled largely DNA elements RNA coding                               Pierre Thuriaux Andre Sentenac (16) its <br /> regions, cases levels activity                     parental strain, YNN281. Yeast transformation per- <br /> modulated upstream flanking regions (reviewed refs. 1                         formed lithium acetate procedure (17). Sporulation and <br /> and 2). 7SL RNA genes, small viral RNA genes,                           tetrad analysis performed using standard genetic meth- <br /> pol III transcription units highly repetitive                        ods (18). Cells grown selection medium (0.67% <br /> DNA elements fall promoter <span id='am-25' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> (1, 2). All                         yeast nitrogen base amino acids, 2% dextrose, and <br />                                                                                     appropriate amino acid supplements) YPD medium (2% <br /> members class 5S related bipartite inter-                       dextrose, 2% peptone, 1% yeast extract). <br /> nal promoters consisting &quot;A box&quot; &quot;B box&quot;                                    Preparation RPRI Gene Variants. An RPRI gene sub- <br /> separated 30-90 base pairs (bp), A box 8-25 bp                          clone&apos;containing 1.3-kilobase (kb) Bgi Il-HindIII frag- <br /> downstream ofthe transcription start site. The A B boxes                        ment the&apos;BamHI-HindIII polylinker sites pUC19 (14) <br /> are initially bound complex transcription factor, TFIIIC,                      used making series deletions 5&apos; flanking <br /> and transcription complex subsequently assembled                          region RPRJ gene. The plasmid cut one <br /> by addition factor (TFIIIB) pot                       following restriction endonucleases: EcoRI -407, <br /> III (3-5). More recently established                       Hpa I -182, Hae I -130, Stu I -69, Rsa I -51 <br /> class genes, including 7SK (6, 7) U6 plants (8)                     (see Fig. 1). EcoRI octamer linkers added the <br /> vertebrates (9-12), regulated primarily upstream se-                          resulting ends gene fragment new EcoRI site <br /> quences similar pol II promoters. In                     downstream HindIII site cloned the <br /> one reported case appears combination                        EcoRI HindIII sites YCp5O. To delete entire <br /> two strategies. The yeast U6 gene upstream promoter                             upstream region RPRI, region downstream + 1 was <br /> elements similar vertebrate U6 gene                       amplified polymerase chain reaction using oligonucle- <br /> also normally positioned A box U6 RNA                              otide introduced EcoRI site + 1 (14) an <br /> coding sequence functional B box downstream the <br /> coding region (13).                                                                 Abbreviation: pol, RNA polymerase. <br />                                                                                     tPresent address: Department Neuropharmacology, Scripps <br /> The publication costs article defrayed page charge           Clinic Research Foundation, 10666 North Torrey Pines Road, <br /> payment. This article marked &quot;advertisement&quot;                La Jolla, CA 92037. <br /> in accordance 18 U.S.C. &#167;1734 solely indicate fact.                    iTo reprint requests addressed. <br />                                                                              6986 <br />                 Biochemistry: Lee et al.                                                           Proc. Natl. Acad. Sci. USA 88 (1991)          6987 <br />  <br />    oligonucleotide site downstream HindIII site.                        (19). Membranes parallel blot transfers hybridized <br />   Point mutations created site-directed mutagenesis                           antisense RPRJ tRNA3LeU radioactive probes, <br />   (19) using 1.3-kb Bgl I-HindIII fragment phage vector                          washed, subjected autoradiography (14, 18). As a <br />   M13mpl8. Mutations introduced A- B-box promoter                            control RNA synthesis pol II, duplicate blots were <br />   candidates shown Fig. 1. To specifically detect RPRJ                         probed 5&apos;_32p oligonucleotide specifically detected <br />   RNA expression plasmid-borne RPRI genes, 7-bp                                LEU2 mRNA, remained stable increased slightly at <br />   segment positions 69-75 loop region RPRJ RNA                         nonpermissive temperature pol III mutant (<span id='am-42' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-46'  ></span>data</span> not <br />   (ref. 20 unpublished observations) replaced                         shown). To uniquely detect levels RPRI RNA from <br />   constructs (see Fig. 3) confirmed gene                       plasmid genes, neutral 7-bp substitution the <br />   7-bp replacement complement RPRI                                coding region (positions 69-75) addition upstream <br />   disruption. The 7-bp substitution combined up-                            internal promoter candidate mutations indicated Fig. <br />   stream deletions candidate internal promoter mutations                         3. Total RNA prepared above, growth was <br />   ligating appropriate EcoRI-BamHI-HindIII fragments                          30&apos;C synthetic medium lacking uracil. Ten micrograms <br />   EcoRI-Sty I-HindIII fragments M13mpl8 replicative                            RNA electroblotted denaturing 8% <br />   form EcoRI-HindIII sites YCp5O. Identities                          polyacrylamide gel probed 5&apos;-32P antisense <br />   mutated recombined genes confirmed DNA                                 28-mer oligonucleotide used create 7-bp <br />   sequencing.                                                                        substitution. Radioactivity combined RPRI RNA <br />     Complementation RPRI Gene Variants. Diploid strain                            bands determined using Betascope 603 blot analyzer <br />   W303 copy RPRJ disrupted HIS3 (14)                              (Betagen, Waltham, MA). <br />  transformed single-copy yeast <span id='am-33' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> containing RPRJ                             DNA Binding Assays TFIIIC. A SUP53 tRNA gene <br />  gene variants URA3 gene selection marker                        fragment prepared end-filling vector EcoRI site of <br />  grown selection plates lacking uracil. Transformants                       pUC53 (70 bp upstream tRNA gene; ref. 22) with <br />  sporulated, tetrads dissected, spores grown                        [a-32P]dATP [a-32P]dTTP (3000 cpm/mmol), followed by <br />  YPD plates test viability.                                                      cleavage downstream gene HindIII release a <br />     RNA Blot Analysis. For analysis RNAs pol III                        652-bp fragment. End-labeled fragments RPR1 gene <br />  temperature-sensitive mutant parental wild-type                            mutated variants used mobility shifts footprinting <br />  strain, inocula fresh cultures grown 250C 370C                     sense strand prepared cleavage Eae I site <br />  -20 hr YPD medium reaching OD650 1.0.                           position -136, labeling polynucleotide kinase (23), <br />  Total RNA isolated described (21). The                            cleavage HindIII site created position +472 give <br />  RNA giving intensity rRNA bands (about 10 gg)                          608-bp fragment. For footprinting antisense strand, an <br />  determined ethidium bromide-stained agarose gel                          RPRJ fragment extending Taq I site position -90 <br />  electrophoresed 2 M formaldehyde/0.8% agarose gels                          HindIII site inserted Acc I-HindIII sites <br />                                                                                     plasmid pGEM-Blue (Promega). End labeling an- <br />  -407                                                                              tisense strand vector EcoRI site for <br />     agatctgcc aattgaacat aacatggtag ttacatatac tagtaatatg gttcggcaca <br />                                                                                    SUP53, EcoRI-HindIII fragment isolated. Pu- <br />  -348 <br />    cattaaaagt ataaaaacta tctgaattac gaattacata tattggtcat aaaaatcaat <br />                                                                                    rification TFIIIC extracts yeast strain PP1002 <br />                                                                                    chromatography Bio-Rex 70 (Bio-Rad), DEAE- <br />  -288 <br />    caatcatcgt gtgttttata tgtctcttat ctaagtataa gaatatccat agttaatatt               Sephadex A-25 (Pharmacia), B-box DNA affinity col- <br />  -228                                              -182 <br />                                                                                    umns described (22, 24). The factor <br />    cacttacgct accttttaac ctgtaatcat tgtcaacagg tatgttaacg acccacattg               required mobility shifts Fig. 4 determined <br />  -168                                     -130                                     empirically approximately wild-type <br />    ataaacgcta gtatttcttt ttcctcttct tattggccgg ctgtctctat actcccctat               tRNA RPRI genes. In mobility-shift experiments end- <br />  -108                                       -69 <br />    agtctgtttc ttttcgtttc gattGTTTTA CGTTTGAGGC CTC         CGC <br />                                                                ABx        -51 <br />                                                                    ACA.GEACG <br />                                                                                    labeled DNA fragments incubated 1.5 <br />                                                                          cc        Al affinity-purified TFIIIC 250C 12 min final <br />  -48 <br />    CTGTGGTGCT CGCGGC <br />                             B* Box <br />                            TG <br />                                                                1 <br />                                       C TGGGAGCTGC GATTGGCAGT GGAACAGTGG           volume 10 Al containing 20 mM Hepes (pH 7.9), 70-85 mM <br />  13 <br />                               G C <br />                                                                    replacemnt <br />                                                                                    ammonium sulfate, 4 mM MgCl2, 5.5% (vol/vol) glycerol, <br />   TAATTCCTAC   GATTAAGAA    CCTGTTTACA GAAGATCCC CACCTATGGG CGCTTATCA              0.1% (wt/vol) bovine serum albumin, 100 ng competitor <br />  73 <br />                                                                   GTCC            pUC9 DNA (cleaved Rsa 1), 200 ng additional <br />   GATATTATCA anTGGGAAAT TCGGTGGAAC ACAGTGGAGC CTTGTCCTCC GGGTTAATGT <br />   TGC <br />                                                                                   plasmid DNA competitor indicated. Samples run <br />  133                                                                              300-400 V nondenaturing 4% polyacrylamide gels in <br />   CGCTTTTGGC   ATTGGCCCCT   GCTCCTGAGA GAAGAAATAT ACTGGGGAAC CATCTTTAC            0.25x TBE buffer (lx TBE: 89 mM Tris borate/2 mM <br />  193 <br />   CGACCGTTGT TATCAGAAAT TCACGGAGTT      CGGCCTAZ;T B1CGACTCCA <br />                                                      Box <br />                                                                    TGGAACGGC <br />                                                                                   EDTA, pH 8.3) plus 5% glycerol 40C. For DNase I <br />                                                  C         T                      footprinting, 2 ng end-labeled DNA preincubated with <br />  253 <br />   AACGGTTGTT CCGTTTGACT TGTCMCCCGC TACGGCGTGA GCGTCAAGGT CTGTTGAGTG <br />                                                                                   4,l TFIIIC (-4-fold excess) conditions as <br />                                                                                   shifts, glycerol concentrations held 0-3%. <br />  313                                                           B2zBoX 369         DNase I digestions analysis cleavage sites were <br />   CAATCGTACG   ACGTCATTAG            CC GATACCGATT <br />  373 <br />                                                                      T        G   performed described (25). Products dideoxynucleotide <br />   TATCCAACTT CCAATTTAAT CTTTCTTTTT Taattttcac ttatttgcga tacagaaaga               sequencing reactions (data shown) used size <br />                                                                                   markers positions DNase I cleavages. <br />    FIG. 1. Features RPRI gene sequence positions of <br /> mutations. Positions numbered relative nucleotide in <br /> the mature RNase P RNA (14, 20). The 5&apos; termini upstream                                                RESULTS <br /> deletions indicated bold numbers (-407, -182, -130, -69,                 The predominant RNA associated purified nuclear <br /> -51, 1). The internal promoter homologies contained                    RNase P S. cerevisiae 369 nucleotides long, a <br /> boxes, nucleotides shown line indicate replace-                 defective form gene encoding RNA, RPRI, is <br /> ment nucleotides point mutations used test candidate internal <br /> promoters. A neutral 7-bp substitution positions 69-75, used                defective 5&apos; end processing tRNA precursors (14, 20). A <br /> generate unique oligonucleotide hybridization site, shown.              longer, abundant form RNA present in <br /> The underlined sequences represent mature RNA coding region,                  extracts highly purified enzyme 84-nucleotide <br /> and uppercase letters extent larger, putative                 5&apos; leader 27-33 nucleotides extra 3&apos; sequence. Al- <br /> precursor RNA.                                                                    precursor-produCt processing relationship not <br />   6988      Biochemistry: Lee et al.                                                                                Proc. Natl. Acad Sci. USA 88 (1991) <br />  <br />                                                                                                                                                                   Comple-   % Steady state <br />                                                                                                a&apos;A40 <br />                                                                                                 ~mentation                                                                   level RNA <br />                                                                                              5&apos;A0407 <br />                                                                                                    D4                                                                +           100 <br />                                                                                              5&apos;A-182               -182         3                                    +           41 <br />                    r-. <br />             IF.          If,        T-- <br />                                                                                              5&apos;A-130 <br />                                                                                                                           -69 <br />                                                                                                                                     00                              ~~+          26 <br />                                                                                              5&apos;A-69                                                                  At           11 <br />            l <br />            lL                             .I                   R A <br />                                                                                              5&apos;A-51                             C51                                                0 <br />                                                                                              VAI                                     1                               -            0 <br />             -     ~~mmpUEP                    p atrursor <br />                                           ~*- i-fl(Il                _                      rprI   A     -407            *       3           --                                4 <br />                                                                                             rprl noB*                               U           _                                 0 <br />                                                                                             RPRI noBl                       0               _                        +           94 <br />                                                                                             RPRI noB2                      0A_ <br />                                                                                                                              --                                      +           88 <br />  <br />                                                                                                FIG. 3. Complementation RPRI RNA expression using <br />   FIG. 2. RPRJ RNA tRNA~&quot; levels po11II mutant                                  variant RPRI genes. Expression indicated RPRI variants from <br /> nonpermissive temperature. Total RNA isolated wild-type cells                          single-copy <span id='am-47' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> tested diploid strain carrying one <br /> or derivative pol III temperature-sensitive (ts) mutant grown                          normal disrupted chromosomal copy essential RPRI <br /> 250C 370C subjected Northern blot analysis using antisense                        gene (14) complementation RNA blot analysis. The <br /> RPRI RNA (Upper) tRNApU (Lower) probes. The identities                               effects progressive 5&apos; deletions point mutations suspected <br /> the various RNAs given right confirmed (14, 26).                               A-box B-box promoter elements complementation levels <br />                                                                                             RPRI RNA shown. The extent larger RPRI RNA is <br /> been rigorously established forms,                                    indicated arrow top. The coding region smaller <br /> coordinate response RNAs pol III promoter                                 RNA, major form associated RNase P activity (20), shown <br />                                                                                             dark bar, A-box B-box promoter homologies (see <br /> mutations (see below) consistent assignment. The                               Figs. 1 6) shown open boxes. &quot;X&quot; boxes indicates point <br /> extent RNAs RPRJ gene region                                       mutations; t indicates slightly reduced growth rate. Complemen- <br /> shown Fig. 1, positions numbered according                                   tation analyses quantitation RNA plasmid-borne <br /> smaller, mature form RNA. The positions deletions,                                RPRI variants RNA blots described Materials Meth- <br /> pol III internal promoter homologies, nucleotides sub-                                  ods. <br /> stituted directed mutations used work                                   (Fig. 2). The level forms RPRJ RNA severely <br /> indicated.                                                                                  reduced mutant grown nonpermissive temperature <br />   To establish RPRJ transcribed pol III, level                               (370C) relative growth permissive temperature <br /> of RPRI determined S. cerevisiae mutant harboring                                 parental strain grown temperature. The <br /> a temperature-sensitive lesion large subunit pol III                              degree reduced expression comparable of <br />                                                                                x <br />                                                                           .0 <br />                                                                          o     .o <br />                                                                           0 <br />                                                                           0    0 <br />                                                                                0 <br />         RPfllvariant                                   r-C <br />                                           c.(L    C. <br />                                                   CL <br />                                                        cc ccQ Q                C                                                                     n     . .. <br />                                                                                                                                                            0.o; <br />                                                                                                                                                                             x <br />                                                                                                                                                                                  0x <br />                                                                                                                                                                                   0 <br />                                                                                                                                                      o &lt;   m <br />                                                                                                                           0 <br />                                                                                                                           0                                                 &lt;     m <br />                                                        n        LO                                                                                   Dr <br />      Competitor DNA                                                                                                                                        LI. <br />                                                                                                                                                                             _     :: <br />                                                        C <br />                                                                                        Competitor DNA                o <br />                                                                                                                                                                      a: <br />                                                                                                                                                                      CL <br />                                                                                                                                                                             _ <br />                                                                                                                                                                             a-    I. <br />  <br />  <br />                   TFIIIC                  -      +     + <br />                                                                                                                      C- <br />                                                                                                                      C      cI) <br />                                                                                                                             = <br />                                                                                                                             0 M~ En &gt;   &gt;                   J <br />                                                                                                                                                                             !a    C <br />  <br />                                                                 +        +     + <br />                                                                                                    TFIIIC                   +       +           +    +               +t     +     + <br />  <br />  <br />  <br />                                                        _ <br />                                                  * _. <br />                                                  __ll1                                                                          s.                   &apos;s                          u <br />                                                                                     unbound SUP53 DNA <br />  unbound RPRI      DNA         w- <br />                                                  4_1       4             41    4_                           i- <br />                                                                                                                                     a** <br />   FIG. 4. Binding TFIIIC SUP53 tRNA <br /> RPRJ gene variant DNA fragments tested Opne <br />                                                      RPRI gene variants. Electrophoretic mobility shift SUP53 tRNA gene or <br />                                                 binding affinity-purified transcription factor TFIIIC. (Left) Radiolabeled DNA fragment <br /> probes contained wild-type RPRJ gene (RPRI lanes) variants mutated A B* internal promoters [rprl (no A box) rprl <br /> (no B* box) lanes]. Binding reactions additional unlabeled competitor plasmid (0.2 p&amp;g) (vector lane) (SUP53 lane) SUP53 <br /> tRNA gene insert indicated. (Right) The radiolabeled DNA probe lanes contained SUP53 tRNA gene. The indicated DNA <br /> competitors contained unlabeled plasmid DNA (0.2 jug). SUP53 RPRI denote wild-type genes; supS3 rprl denote gene variants <br /> with mutated internal promoters. The upper shifted bands correspond TFIIIC complexes. Less intense bands migrating the <br /> unbound probe complex correspond shifts observed partially proteolyzed TFIIIC (unpublished observations). <br />                Biochemistry: Lee et A                                                            Proc. Natl. Acad. Sci. USA 88 (1991)          6989 <br />                        Sense Strand                   Antisense Strand                     a <br />                           -   +         ~~ <br />                                              TFIIIC        -     +                             A box consensus   :       ZRGCNNLGYN X <br />                                                                                                    A at-64               TGGCGCACATGG <br />                                                                                                B box consensus       :   GfT=GANT.C <br />                                                                                                    B* -30             GGAACGAAACT <br />  <br />                                                                                            b <br />                                                                                                             Sequence             Position <br />                                                                                                U6:        TTTTTTTCG             -18 <br />                               -04                                                              RPRI:      TTTCTTTCG             -102 (-18) <br />                                                                                                U6:        CTATA                -31 <br />      Mature RPR1 RNA <br />      5&apos; End                  .; 4 <br />                           __ADZ                                                                RPRI:      CTATA                -112 (-28) <br />                                 -10                        _l <br />                                                            .,.   fl <br />                                                                       .-5 <br />                                                                        --2                                CTATA                -122 (-38) <br />                 By box                                                                         U6:        TCCAP TAT            -48 <br />                                                                                                RPRI:      TCCCCTAT             -116 (-32) <br />                                                                                                U6:        AGTATTTCGT           -57   &apos; <br />                                                                                                RPRI:      AGTATTTCTT           -159 (-75) <br />                                                                                                U6:        TAAAAGTAT            -61 <br />                  A box                                                                         RPRI:      TAAAAGTAT            -345 (-261) <br />                                                                                                U6:        TCAACA               -87 <br />                                                       A                                        RPRI:      TCAACA               -196 (-112) <br />                                   --74                                            FIG. 6. Sequence comparisons internal promoters and <br />                                                                                RPRI/U6 upstream regions. (a) Invariant positions tRNA <br />                                                                                internal promoter consensus (ref. 1) underlined. (b) Sequence <br />                                   -84 <br />                                                                         -75 <br />                                                                                similarities shown upstream region RPRI the <br />                                                                                S. cerevisiae U6 gene transcription start (+1) the <br />                                                                         -79    beginning solo <span id='am-37' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-38'  ></span><span id='am-39' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>delta</span> repetitive element (-90). Position num- <br />                                                                                bers indicate nucleotide sequence. In case of <br />                                                                                RPRI sequences, position number corresponds the <br />                                                                       -84      numbering scheme relative nucleotide smaller, <br />                                                                                mature RNA Fig. 1. The numbers parentheses refer the <br />                                                                                position relative 5&apos; end leader RNA. <br />                                                                                 candidate internal promoter element ensure inactivation. <br />     FIG. 5. Binding TFIIIC RPRI leader region. Excess                 The variant genes tested complementation an <br />  affinity-purified S. cerevisiae TFIIIC bound RPRI gene                  RPRI gene disruption levels RNA production in <br /> fragments labeled upstream coding region sense             vivo (Fig. 3). <br /> or antisense strand. DNase I footprinting performed as <br /> described text. A DNA fragment digested DNase I                Deletions upstream region far -130 no <br /> absence TFIIIC run comparison, DNA sequencing                    effect cell growth variant RPRI genes used <br /> lanes (not shown) served position markers. Indicated left          complement RPRJ disruption haploid strain. A <br /> each autoradiograph positions corresponding RPRI mature              deletion -69, leader removing A <br /> domain (thick solid bar, + 1 above) leader region (thin solid           box, gave slightly slow growth phenotype. However, a <br /> bar, -84 -1). The positions A box B* box            deletion (to -51) removes A box lethal. <br /> indicated. Cleavages intensified presence TFIIIC                 Double point mutations A box B* box also <br /> denoted arrows right autoradiograph,                lethal, mutations B1 B2 gave normal growth <br /> minimum extent footprint defined protected cleavage <br /> sites denoted open bars.                                                    phenotypes. Analysis RNA synthesized RPRJ <br />                                                                                variants gave parallel results, upstream region <br />  tRNA, pol III transcript known diminished                    substantially influence efficiency expres- <br />  mutant (16).                                                          sion. Successive deletions 5&apos; flanking region the <br />     Identification DNA elements required pol III expres-                putative A box reduced RNA expression as <br />  sion vivo accomplished constructing types                    10-fold. However, deletions removed A box, <br />  RPRI mutations. Progressive upstream deletions                      point mutations B* box, eliminated detectable <br />  sequence comparisons yeast U6 gene sug-                      expression, double point mutation A box reduced <br />  gested numerous homologies RPRI (see Discussion). In                     expression 20-fold. In contrast, mutations either <br />                                                                               B1 B2 box coding region little effect <br />  addition, analysis revealed tRNA-like A-box                       RPRI RNA levels, small reductions have <br /> promoter properly positioned 5&apos; leader                     partial destabilization mature RNase P <br /> consensus B boxes (B1 B2) correct orientations                     holoenzyme. Northern blot analysis RPRI RNA expressed <br /> within mature RNA coding domain (see Fig. 1). Although                    variants (data shown) indicated larger <br /> the B1 B2 elements farther downstream                       smaller (mature form) RNAs reduced eliminated <br /> the RNA 5&apos; ends optimum tRNA gene promoters,                      coordinately mutations, consistent de- <br /> a B box similarly far downstream yeast U6 gene                     rived single transcript. <br /> been shown regulate transcription (13). Another B box                         These observations indicated A B* boxes the <br /> (B*) relatively poor match consensus                        5&apos; leader region essential elements RPRI pro- <br /> identified leader region TFIIIC                    moter. To investigate sequences be- <br /> to bind exclusively leader region (see Fig. 5). Double                 haved like bipartite internal promoters tRNA genes, <br /> point mutations invariant positions                      binding transcription complex assembly factor, TFIIIC, <br />  6990     Biochemistry: Lee et al.                                                  Proc. Natl. Acad. Sci. USA 88 (1991) <br /> to RPRJ genes containing normal mutated A B* boxes           RPRI, - 31 CTATA sequence U6 present direct <br /> was examined. DNA mobility-shift experiments (Fig. 4)               repeat - 28 - 38 relative start ofthe RPRI leader. <br /> showed TFIIIC bound RPRI gene              The -59 AAAGTTATTCG proximal sequence element com- <br /> SUP53 tRNA gene binding gene            mon mammalian U6 genes overlaps strong <br /> competitively inhibited other. As tRNA gene (1,      U6/RPRI similarities -75 -261 relative RPRI <br /> 27), mutations A B* elements severely reduced      leader. The thymine-rich element -18 suggested <br /> or eliminated TFIIIC binding. Mutations RPRI B1 B2        potential U6 regulatory site (27), sequence <br /> elements effect binding (data shown).                 similarities -48 -87 (U6 positions) previ- <br />    The region RPRI TFIIIC bound deter-              ously noted comparison mammalian <br /> mined DNase footprinting sense               genes investigated functional significance. Further <br /> antisense strand (Fig. 5). Stable TFIIIC footprints        investigation required elucidate control mech- <br /> observed upstream mature RNA coding regions               anisms operate upstream <br /> (data shown), binding TFIIIC protected       sequences cooperate required internal promoters to <br /> leader region, -74 -4 (9 bp upstream        increase expression RPRI. <br /> A box 15 bp downstream B* box). Both extent <br /> of protection (open bars Fig. 5) existence               We thank J. Warner, P. Thuriaux, A. Sentenac, E. Esposito <br /> intensified cleavages near ends protected region         cloned DNA yeast strains. We thank D. Kindelberger, C. <br />                                                                     Rohlman, M. Hull, G. Butler critical reading manu- <br /> (denoted arrows) comparable patterns seen            script. This work supported National Institutes Health <br /> with tRNA gene internal promoters.                                  Grant GM34869 National Science Foundation Grant DMB- <br />                                                                     8901559 D.R.E. <br />                          DISCUSSION                                  1. Geiduschek, E. P. &amp; Tocchini-Valentini, G. P. (1988) Annu. Rev. <br /> Efficiency RPRI transcription strongly influenced              Biochem. 57, 873-914. <br /> upstream DNA sequences, essential pol III promoter           2. Murphy, S., Moorefield, B. &amp; Pieler, T. (1989) Trends Genet. 5, <br /> elements correspond A box/B box combination                122-126. <br />                                                                      3. Huibregtse, J. M. &amp; Engelke, D. R. (1989) Mol. Cell. Biol. 9, <br /> 5&apos; leader region binds transcription assembly factor               3244-3252. <br /> TFIIIC. This arrangement advantage use           4. Kassavetis, G. A., Riggs, D. L., Negri, R., Nguyen, L. H. &amp; <br /> an efficient pol III internal promoter constraining         Geiduschek, E. P. (1989) Mol. Cell. Biol. 9, 2551-2566. <br /> functional RPRI RNA sequence requiring pro-              5. Kassavetis, G. A., Braun, B. R., Nguyen, L. H. &amp; Geiduschek, <br /> moter retained mature form. The RPRI 5&apos; leader                E. P. (1990) Cell 60, 235-245. <br />                                                                      6. Murphy, S., DiLiegro, C. &amp; Melli, M. (1987) Cell 51, 81-87. <br /> region does appear bona fide tRNA gene,                  7. Murphy, S., Tripodi, M. &amp; Melli, M. (1986) Nucleic Acids Res. 14, <br /> however, sequence folded clover-             9243-9261. <br /> leaf structure does closely correspond se-            8. Waibel, F. &amp; Filipowicz, W. (1990) Nature (London) 346, 199-202. <br /> quence known yeast tRNA (unpublished observa-                 9. Krol, A., Carbon, P., Ebel, J.-P. &amp; Appel, B. (1987) Nucleic Acids <br /> tions). It known leader RNA           Res. 15, 2463-2478. <br /> distinct function cell related unchar <br /> </body></html>