<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Dec. 1999, p. 8042&#8211;8051                                                                                    Vol. 19, No. 12 <br /> 0270-7306/99/$04.00&#11001;0 <br /> Copyright &#169; 1999, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />            A Subunit Yeast TFIIIC Participates Recruitment <br />                           TATA-Binding Protein <br />                ERIC DEPREZ,&#8224; ROSALI&#180;A ARREBOLA, CHRISTINE CONESA,                                    AND       &#180; SENTENAC* <br />                                                                                                            ANDRE <br />                Service Biochimie et Ge&#180;ne&#180;tique Mole&#180;culaire, <span id='am-1' about='obo:NCBITaxon_1107390' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_272274 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7423 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Cea" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>CEA</span>/Saclay, F-91191 Gif-sur-Yvette Cedex, France <br />                        Received 29 June 1999/Returned modification 5 August 1999/Accepted 7 September 1999 <br />  <br />            TFIIIC plays key role nucleating assembly initiation factor TFIIIB <span id='am-91' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-92' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> III genes. We have <br />          characterized essential gene, TFC8, encoding 60-kDa polypeptide, &#65533;?&#65533;60, present affinity-purified <br />          TFIIIC. Hemagglutinin-tagged variants &#65533;?&#65533;60 TFIIIC-tDNA complexes reside <br />          downstream DNA-binding domain &#65533;?&#65533;B. Unexpectedly, thermosensitive phenotype of <br />          N-terminally tagged &#65533;?&#65533;60 suppressed overexpression &#65533;?&#65533;95, belongs &#65533;?&#65533;A domain, two <br />          TFIIIB components, TATA-binding protein (TBP) B&#11542;/TFIIIB90 (but TFIIIB70). Mutant TFIIIC was <br />          deficient activation certain tRNA genes vitro, transcription defect selectively alleviated <br />          increasing TBP concentration. Coimmunoprecipitation experiments support direct interaction between <br />          TBP &#65533;?&#65533;60. It suggested &#65533;?&#65533;60 links &#65533;?&#65533;A &#65533;?&#65533;B domains participates TFIIIB assembly its <br />          interaction TBP. <br />  <br />  <br />    The primary step tRNA gene activation binding               gel retardation DNA photo-cross-linking (31). TBP stabi- <br /> TFIIIC A B blocks intragenic promoter. Its                lizes weak interaction TFIIIB70 TFIIIC- <br /> main function assemble initiation factor TFIIIB              DNA complex complete upstream footprint the <br /> upstream transcription start site (30). Yeast TFIIIC (&#65533;?&#65533;              characteristic stability TFIIIB-DNA complex requires <br /> factor) multisubunit protein ca. 600 kDa organized               recruitment B&#11033;/TFIIIB90 (29, 31). A cascade confor- <br /> two large globular domains &#65533;?&#65533;A &#65533;?&#65533;B similar size mass               mational rearrangements protein DNA levels are <br /> (10-nm diameter ca. 300 kDa), interacting                 accompanying assembly steps, evidenced successive <br /> promoter element, visualized <span id='am-149' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-150'  ></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>electron</span> microscopy (17,                 changes accessibility TFIIIB70, TBP, &#65533;?&#65533;131 to <br /> 53). Binding &#65533;?&#65533;B B block predominant favors             site-specific DNA cross-linking (31), DNA bending in- <br /> binding &#65533;?&#65533;A A block (4). TFIIIC-DNA interaction                    duced TFIIIB binding (11, 38, 46), presence of <br /> displays remarkable adaptability variable A-B dis-                 cryptic DNA binding domain TFIIIB70 (24). &#65533;?&#65533;131 appears <br /> tances different tRNA genes (3). Affinity-purified                 play major role positioning TFIIIB only <br /> <span id='am-23' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> TFIIIC comprises polypeptides               TFIIIC subunit accessible DNA cross-linking upstream of <br /> 138, 131, 95, 91, 60, 55 kDa (6, 19, 47, 59). Gene cloning,             start site (5, 7) interact TFIIIB70 (14, 33) <br /> protein-DNA cross-linking, mutagenesis, protein-protein                 TFIIIB90 (51). TFIIIB effect assembly onto <br /> interaction studies provided global view location              TATA-containing SNR6 gene interaction of <br /> role TFIIIC subunits TFIIIC-DNA com-                  TBP strong TATA box (27, 43, 45). Interestingly, <br /> plex. &#65533;?&#65533;138 &#65533;?&#65533;91 subunits reside &#65533;?&#65533;B domain co-                 Whitehall et al. (60) TBP discern the <br /> operate downstream DNA binding (1, 10, 36, 37); &#65533;?&#65533;95                  polarity TATA element directed TFIIIB assembly in <br /> &#65533;?&#65533;55 interact physically, belong &#65533;?&#65533;A domain,                   orientations. However, contrast TATA-depen- <br /> thought participate A block binding (7, 17, 40, 48, 59).              dent assembly, TFIIIC placed TFIIIB correct orienta- <br /> &#65533;?&#65533;131 (42) stands TFIIIC subunit responsible TFIIIB               tion. Since TFIIIC component known interact with <br /> assembly based genetic evidence (49, 61), upstream lo-               TBP, presumed unidirectional binding TBP <br /> cation (7), interaction TFIIIB components (14,             TATA box dictated oriented interaction of <br /> 33, 51).                                                                    TFIIIB70 &#65533;?&#65533;131 (60). <br />    TFIIIB stable molecular entity like TFIIIC. It              In present work completed characterization <br /> resolved chromatographically fractions named B&#11032;                TFIIIC components cloning yeast gene, named TFC8, <br /> B&#11033; (29). B&#11032; comprises TATA-binding protein (TBP)                    encodes 60-kDa polypeptide. We present genetic and <br /> TFIIB-related factor TFIIIB70/Brf1 (12, 16, 31, 39), B&#11033;               biochemical evidence component, named &#65533;?&#65533;60, resides <br /> contains TFIIIB90 (32, 50, 51). The TFIIIC-dependent TFIIIB                 &#65533;?&#65533;B domain participates TFIIIB <br /> assembly TATA-less class III genes multistep pathway                recruitment TBP binding. <br /> that decomposed vitro (29, 31) reconstituted <br /> with recombinant TFIIIB components (32, 51). The <span id='am-119' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-120' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>                                           MATERIALS AND METHODS <br /> interaction TFIIIB70, TBP, B&#11033;, shown                    Yeast strains, <span id='am-165' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-166' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-169' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>, genetic techniques. YNN281&#11003;YNN282 (21) was <br />                                                                             used gene disruption. Cells grown standard rich medium (YPD) or <br />                                                                             minimal medium (Casamino acid medium). The plasmids used suppression <br />   * Corresponding author. Mailing address: Service Biochimie et          studies described Lefebvre et al. (37). Preparation media, tetrad dis- <br /> de Ge&#180;ne&#180;tique Mole&#180;culaire, CEA/Saclay, F-91191 Gif-sur-Yvette,            section, transformation lithium acetate-treated cells, plasmid shuffling by <br />                                                                             using 5-fluoro-orotic acid performed standard techniques (2). <br /> France. Phone: 33-1-69-08-22-36. Fax: 33-1-69-08-47-12. E-mail: <br />                                                                                Amino acid sequence determination. The polypeptide components purified <br /> sentenac@dsvidf.cea.fr.                                                     TFIIIC preparation separated preparative electrophoresis 6% <br />   &#8224; Present address: Laboratoire Physicochimie et Pharmacologie          polyacrylamide&#8211;sodium dodecyl sulfate (SDS) gel (36). The gel lightly <br /> des Macromole&#180;cules Biologiques (UMR8532), ENS Cachan,                   stained Coomassie blue. A gel slice containing 60-kDa polypeptide was <br /> F-94235 Cachan Cedex, France.                                               excised, crushed, subjected trypsin digestion. Preparation analysis of <br />  <br />                                                                      8042 <br />  VOL. 19, 1999                                                                                 &#65533;?&#65533;60 PARTICIPATES IN TFIIIB RECRUITMENT                              8043 <br />  <br />  <br /> tryptic peptides previously described (36, 59). One peptide sequence        Amersham ECL Kit. The polypeptides revealed antibodies directed TBP or <br /> was partially determined (??TLYLTT[T/F]PT).                                           &#65533;?&#65533;60 identified independent experiments. <br />    Cloning, construction plasmids, disruption TFC8. Two 32-mer                 DNA binding vitro transcription assays. TFIIIC-tDNA &#65533;?&#65533;B-tDNA <br /> oligonucleotides used amplify open reading (ORF) frame TFC8         interaction monitored gel retardation analysis described previously (25, <br /> PCR yeast genomic DNA. The resulting DNA fragment labeled            37). A 32P-labeled DNA fragment (3 10 fmol; 4,000 10,000 cpm) carrying <br /> [&#65533;?&#65533;-32P]dCTP used screen FL100 library (57) containing yeast genomic        tRNA3Glu (198 bp) SUP4tRNATyr (375 bp) genes incubated 10 <br /> DNA fragments inserted pFL44L (2&#65533;?&#65533;, URA3) plasmid (9). The pFL44L             min 25&#176;C 15-&#65533;?&#65533;l reaction mixture containing 10 mM Tris-HCl (pH 8.0), <br /> plasmid isolated hybridizing clones contain large      10% glycerol, bovine serum albumin, DNA competitor, TFIIIC (MonoQ <br /> DNA insert comprising TFC8 gene named pCC12 (pFL44L-TFC8).                fraction). The final ammonium sulfate concentration 75 mM. &#65533;?&#65533;B-tDNA <br /> The 2.3-kb SalI-FspI DNA fragment pCC12 harboring TFC8 gene              complexes obtained 10 min incubation 25&#176;C TFIIIC- <br /> cloned pUN45 creating pYED1 plasmid. A 69-mer oligonucleotide            tDNA complexes 10 ng &#65533;?&#65533;-chymotrypsin (Sigma). Digestion stopped <br /> used introduce NdeI restriction site initiation <span id='am-111' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> TFC8, followed   addition 1 ng aprotinin (Sigma). Complexes analyzed nonde- <br /> by sequence encoding hemagglutinin (HA) epitope (YPYDVPDYA)                   naturing gel electrophoresis (5% polyacrylamide). <br /> derived influenza virus HA protein (62), oligonucleotide-mediated            Transcription mixtures (40 &#65533;?&#65533;l) contained 20 mM HEPES (pH 8.0); 5 mM <br /> mutagenesis single-stranded pYED1 DNA using Muta-Gene kit (Bio-               MgCl2; 1 mM DTT; 0.1 mM EDTA; 5% glycerol; 8 U RNasin (Amersham); <br /> Rad), yielding pYED2 plasmid (encoding HA Nter-&#65533;?&#65533;60). The sequence                 0.6 mM ATP, GTP, CTP; 0.03 mM UTP 10 &#65533;?&#65533;Ci [32P]UTP; <br /> encoding HA epitope introduced stop codon TFC8          TFIIIB (370 ng Cibacron-blue fraction) rTBP (250 ng); rTFIIIB70 (1.2 &#65533;?&#65533;g) <br /> PCR-mediated mutagenesis oligonucleotides. One contained SalI              partially purified B&#11033; fraction (1.8 &#65533;?&#65533;g); RNA polymerase III (Pol III; 50 ng); <br /> restriction site nucleotides complementary upstream region          TFIIIC (MonoQ fraction) indicated. The final KCl concentration 110 <br /> TFC8 promoter, harbored sequence encoding HA epitope,           mM. After 10 min preincubation 25&#176;C, transcription reaction initiated <br /> a NotI restriction site, nucleotides complementary stop codon region       addition 130 ng plasmid DNA harboring different tRNA genes and <br /> of TFC8. After PCR amplification, SalI-NotI DNA fragment inserted             allowed proceed 45 min 25&#176;C. Transcripts analyzed electro- <br /> into corresponding sites pYED1 pYED3 (encoding HA Cter-            phoresis 8 M urea gel (6% polyacrylamide). <br /> &#65533;?&#65533;60). The NdeI-BamHI DNA fragment pYED2 inserted                       In vivo labeling RNAs Northern blot analysis. RNA labeling done <br /> corresponding sites pET28c vector (Novagen), yielding pET60.                       [3H]uracil strains ura3 mutation complemented the <br />    Disruption TFC8 gene performed previously described (8, 40).         URA3 plasmid pFL44L. Cells exponentially grown uracil-free Casamino <br />                                                                                       acid medium supplemented adenine (Casa&#11001;Ade) optical density of <br /> Two 55-mer oligonucleotides harboring sequences complementary TFC8 and <br />                                                                                       0.4 600 nm. Then, 150 &#65533;?&#65533;Ci [3H]uracil added 10 ml culture 15 <br /> to yeast HIS3 selectable marker used amplify PCR &#11011;1.1-kb <br />                                                                                       min. The cells harvested chilled 10 ml ice-cold sterile <br /> DNA fragment containing HIS3 gene flanked TFC8 promoter termi- <br />                                                                                       water. RNAs extracted previously described (22, 52). Small RNA <span id='am-116' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-117' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> <br /> nator sequences. The PCR-amplified DNA fragment used transform the <br />                                                                                       analyzed loading separating equal amounts RNA (6 &#65533;?&#65533;g lane) <br /> strain YNN281&#11003;YNN282. The structure His&#11001; diploids verified by <br />                                                                                       7 M urea gel electrophoresis (6% polyacrylamide). <br /> PCR analysis. To determine TFC8 essential growth, sporulation <br />                                                                                          RNA extraction gel electrophoresis Northern blot analysis per- <br /> and dissection analysis performed. The diploid His&#11001; strain trans- <br />                                                                                       formed described previous section. Electrophoretic transfer nylon <br /> formed pCC12 plasmid (pFL44L-TFC8) sporulated. A His&#11001; spore              membrane (Bio-Rad apparatus), UV cross-linking DNA (Stratalinker appara- <br /> containing pCC12 plasmid chosen strain YCC8 used plasmid          tus), hybridization DNA probe sodium phosphate buffer carried <br /> shuffling.                                                                            previously described (15). The DNA probe used Northern blot <br />    Purification TFIIIC. TFIIIC purified starting &#11011;12 g S. cerevisiae   experiment shown Fig. 2B 327-bp 32P-labeled PCR fragment encom- <br /> cells, using fast-protein liquid chromatography-grade resins. The preparation      passing yeast tRNA3Leu gene amplified pGE2 plasmid. <br /> the cell extract described Huet et al. (25). Crude extract first <br /> diluted 0.25 M ammonium sulfate (AS) buffer I (20 mM Tris-HCl, pH 8.0; <br /> 0.5 mM EDTA; 10 mM &#65533;?&#65533;-mercaptoethanol; 10% [vol/vol] glycerol)                                                  RESULTS <br /> loaded 2.5 ml/min 25-ml heparin Hyper D (BioSepra) column previously <br /> equilibrated buffer I (0.25 M AS). The resin washed 5 ml/min            TFC8 encodes &#65533;?&#65533;60, 60-kDa subunit yeast TFIIIC. <br /> with 250 ml buffer I (0.35 M AS). A linear gradient AS 0.35 0.70        TFIIIC S. cerevisiae comprises polypeptides 138, <br /> M 180 ml buffer I applied 2.5 ml/min. Fractions (2 ml)         131, 95, 91, 60, 55 kDa. In order clone gene encod- <br /> collected assayed TFIIIC-DNA binding activity. TFIIIC-containing frac- <br /> tions (0.45 0.55 M AS) pooled dialyzed buffer I (0.07 M AS).      ing hypothetical &#65533;?&#65533;60 subunit, polypeptide purified <br /> Proteins loaded 0.5 ml/min 1-ml MonoQ column (Pharmacia,            SDS-PAGE subjected tryptic digestion. The se- <br /> Piscataway, N.J.) previously equilibrated buffer I (0.07 M AS). The column       quence peptide determined (??TLYLTT <br /> was washed 0.5 ml/min 20 ml buffer I (0.07 M AS). A linear gradient        [T/F]PT). When compared DNA sequences databases by <br /> of AS 0.07 0.4 M 15 ml buffer I applied 0.5 ml/min. <br /> Fractions (200 &#65533;?&#65533;l) containing TFIIIC-DNA binding activity eluted between <br />                                                                                       using BLAST program, peptide sequence found <br /> 0.24 0.30 M AS. Based Western blot experiments anti-&#65533;?&#65533;55 anti-&#65533;?&#65533;60       slight variation (DGTLYLTTFPD) unique yeast <br /> antibodies, TFIIIC preparation Ntag-&#65533;?&#65533;60 mutant cells            hypothetical protein 588 residues, theoretical molec- <br /> contain half factor wild type.                                         ular weight 67,640 isoelectric point 5.87. The gene <br />    Expression purification &#65533;?&#65533;60 <span id='am-122' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-130'  ></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli</span>. Recombinant Tfc8p          encoding protein unique, maps chromosome XVI <br /> tagged N-terminal end histidines HA epitope (HA- <br /> r&#65533;?&#65533;60) expressed plasmid pET60 E. coli BL21 (pLysS). Cell culture,         (13), shows similarity sequences EMBL/ <br /> protein induction, extract preparation performed described earlier        GenBank <span id='am-155' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-156' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-159'  ></span>data</span> bank (S. cerevisiae, <span id='am-74' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span>Caenorhabditis elegans</span>, and <br /> (1). Crude cell extract loaded 1 ml/min 5 ml Ni2&#11001;-charged HiTrap       current versions <span id='am-6' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-12'  ></span>Arabidopsis</span> thaliana, <span id='am-99' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo sapiens</span>, and <br /> chelating column (Pharmacia) previously equilibrated 20 mM HEPES (pH             <span id='am-47' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-48' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51'  ></span>Schizosaccharomyces</span> pombe genomic sequences). The gene, <br /> 7.8), 300 mM NaCl, 10% glycerol containing 10 mM imidazole. Proteins <br /> were eluted linear gradient 60 ml buffer containing 20 270 <br />                                                                                       previously designated YPL007C, named TFC8. <br /> mM imidazole 1 ml/min. Fractions (1 ml) assayed HA-r&#65533;?&#65533;60                   To investigate TFC8 essential growth, one <br /> Western blotting anti-&#65533;?&#65533;60 antibodies. Fractions containing HA-r&#65533;?&#65533;60          chromosomal copy TFC8 gene deleted diploid <br /> eluted concentrations 160 190 mM imidazole.                            strain YNN281&#11003;YNN282 replacing ORF yeast <br />    Anti-&#65533;?&#65533;60 polyclonal antibodies. A 14-mer peptide (N-12-M) encompassing the <br /> C-terminal amino acid residues &#65533;?&#65533;60 synthesized (Neosystem, Strasbourg, <br />                                                                                       HIS3 selectable marker. Sporulation tetrad analysis re- <br /> France) conjugated maleimide-activated keyhole limpet hemocyanin               vealed viable His&#11002; spores nonviable spores, sug- <br /> (KLH). N-12-M&#8211;KLH conjugate injected rabbits antibody               gesting TFC8 essential gene. To confirm con- <br /> production injections week intervals.                         clusion, diploid His&#11001; strain (which contains copy the <br />    Immunoprecipitation experiments. rTBP (200 ng) rTBP (200             TFC8 gene) transformed 2&#65533;?&#65533; plasmid, pCC12, har- <br /> ng) HA-r&#65533;?&#65533;60 (650 ng) preincubated 90 min 25&#176;C 40 &#65533;?&#65533;l buffer <br /> A containing 20 mM HEPES (pH 8.0), 5 mM MgCl2, 1 mM dithiothreitol (DTT),             boring TFC8 gene, sporulated. Only resulting His&#11001; <br /> 0.1 mM EDTA, 5% glycerol, 0.05% NP-40, 110 mM KCl. Magnetic beads                 haploid strains containing pCC12 plasmid TFC8 <br /> (Dynabeads/Dynal) coated 12CA5 antibody added mixtures.              gene able grow. These results demonstrated that, like <br /> The beads incubated gentle shaking 3 h 10&#176;C washed          previously characterized genes encoding other <br /> three times buffer A. Bound proteins eluted boiling Laemmli <br /> sample buffer, separated SDS-polyacrylamide gel electrophoresis (PAGE) <br />                                                                                       TFIIIC subunits (1, 36, 40, 42, 48, 59), TFC8 required cell <br /> and analyzed Western blotting using mixture polyclonal antibodies          viability. <br /> directed TBP &#65533;?&#65533;60. Bound antibodies revealed using                  To demonstrate presence TFC8 gene product in <br />  8044     DEPREZ ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> TFIIIC factor, sequence encoding HA epitope was <br /> fused 5&#11032; 3&#11032; end TFC8. A haploid strain which <br /> lacked chromosomal TFC8 gene expressed HA <br /> C-terminally tagged &#65533;?&#65533;60 (Ctag-&#65533;?&#65533;60) grew normally. When the <br /> sequence encoding HA epitope fused just the <br /> initiation codon, growth haploid strain expressing <br /> the N-terminally tagged &#65533;?&#65533;60 (Ntag-&#65533;?&#65533;60) slightly affected at <br /> 30&#176;C (the mutant cells having doubling time 130 min in <br /> liquid medium, instead 110 min wild-type strain). <br /> Furthermore, Ntag-&#65533;?&#65533;60 strain temperature sensitive. <br /> At 37&#176;C, cells grew apparent doubling time 190 <br /> min approximately 16 18 h cell death occurred. <br />    TFIIIC factor purified strains, from <br /> a strain expressing HA-tagged version &#65533;?&#65533;138 (36). Pre- <br /> formed TFIIIC-tDNA3Glu complexes incubated 30 <br /> min 25&#176;C increasing concentrations anti-HA mono- <br /> clonal antibody analyzed gel retardation experiments <br /> (Fig. 1A). The anti-HA antibody clearly altered migration <br /> rate Ctag-&#65533;?&#65533;60 TFIIIC-tDNA3Glu complexes (compare <br /> lanes 6 7) extent HA-&#65533;?&#65533;138-containing com- <br /> plexes (lanes 11 12). On hand, migration rate <br /> of Ntag-&#65533;?&#65533;60 TFIIIC-tDNA3Glu complexes altered <br /> (lanes 9 10), presence 1 &#65533;?&#65533;g of <br /> anti-HA antibody (data shown), HA epitope was <br /> buried inaccessible antibody. Note that, in <br /> contrast previous results TFIIIC containing HA-tagged <br /> &#65533;?&#65533;131 (42) &#65533;?&#65533;138 subunits (see lanes 11 12 reference <br /> 36), binding anti-HA antibody Ctag-&#65533;?&#65533;60 sub- <br /> unit appeared cause dissociation TFIIIC- <br /> tDNA3Glu complexes. <br />    To gain insight localization &#65533;?&#65533;60 within <br /> TFIIIC, performed limited proteolysis TFIIIC-tDNA <br /> complexes &#65533;?&#65533;-chymotrypsin. Limited proteolysis yeast <br /> TFIIIC causes separation transcription factor          FIG. 1. TFC8 encodes 60-kDa subunit yeast TFIIIC. (A) TFIIIC was <br /> two domains ca. 300 kDa each, called &#65533;?&#65533;A &#65533;?&#65533;B (44). The         purified haploid strains expressing HA-tagged versions TFC3 (HA-&#65533;?&#65533;138) <br /> &#65533;?&#65533;B domain generated proteolysis forms stable complex           TFC8 (Ctag-&#65533;?&#65533;60 Ntag-&#65533;?&#65533;60, tagged C-terminal N-terminal end of <br /> with B block visualized gel shift assays     Tfc8p, respectively). TFIIIC preincubated 32P-labeled DNA fragment <br />                                                                     containing tRNA3Glu gene 10 min 25&#176;C incubated <br /> supershifted addition anti-&#65533;?&#65533;138 polyclonal antibodies     various amounts 12CA5 monoclonal antibody directed HA epitope <br /> (19). A similar result obtained present study         30 min 25&#176;C. Protein-tDNA complexes analyzed gel retardation <br /> anti-HA antibodies neatly supershifted tagged-&#65533;?&#65533;138 &#65533;?&#65533;B-tDNA          assay revealed autoradiography. Lanes 1 8, TFIIIC, antibody; <br /> complexes (Fig. 1B, lanes 7 8). When &#65533;?&#65533;B-tDNA complexes          lanes 2, 7, 9, 11, antibody; lanes 3 6, 10, 12, addition 12CA5 <br />                                                                     monoclonal antibody, indicated. (B) &#65533;?&#65533;B, protease-resistant domain of <br /> obtained limited proteolysis Ctag-&#65533;?&#65533;60 TFIIIC-tDNA          TFIIIC, HA-tagged &#65533;?&#65533;138 &#65533;?&#65533;60, obtained digestion TFIIIC- <br /> complexes incubated increasing amounts anti-HA         tDNA complex 10 ng &#65533;?&#65533;-chymotrypsin 10 min 25&#176;C. Digestion was <br /> antibodies, defined supershifted band complex ob-         stopped 1 ng aprotinin addition 12CA5 antibody. Left <br /> served, antibodies reduced migration rate &#65533;?&#65533;B-        panel, C-terminally HA-tagged TFC8 gene (Ctag-&#65533;?&#65533;60); right panel, HA-tagged <br />                                                                     TFC3 gene (HA-&#65533;?&#65533;138) N-terminally HA-tagged TFC8 gene (Ntag-&#65533;?&#65533;60). Lanes <br /> tDNA complexes, causing marked trailing band (Fig.         1 9, control TFIIIC-tDNA complex; lanes 2, 7, 10, &#65533;?&#65533;B generated by <br /> 1B, lanes 2 6). A weak accessibility partial proteolytic    proteolysis; lanes 3 6, 8, 11, incubation &#65533;?&#65533;B-tDNA complex various <br /> degradation HA epitope possibly caused dissoci-         amounts 12CA5 monoclonal antibody, indicated. &#65533;?&#65533;, TFIIIC-tDNA complex; <br /> ation immune complex electrophoresis. Such          &#65533;?&#65533;-IgG, immunoglobulin G (IgG)-TFIIIC-tDNA ternary complex; &#65533;?&#65533;B, &#65533;?&#65533;B-tDNA <br />                                                                     complex; &#65533;?&#65533;B-IgG, IgG-&#65533;?&#65533;B-tDNA ternary complex. <br /> phenomenon observed Ntag-&#65533;?&#65533;60 &#65533;?&#65533;B-tDNA com- <br /> plexes (lanes 10 11), untagged &#65533;?&#65533;B-tDNA com- <br /> plexes (data shown), yield migration rate were <br /> unaffected monoclonal antibody. Altogether, gel        served RNA labeling pattern wild-type mu- <br /> shift experiments indicated polypeptide encoded         tant cells grown 30&#176;C (lanes 1 2). In contrast, when <br /> TFC8 TFIIIC C-terminal end      shifted 6 12 h 37&#176;C, mutation altered labeling <br /> located &#65533;?&#65533;B domain.                                           pattern tRNAs. A smear radioactivity detected just <br />    In vivo characterization Ntag-&#65533;?&#65533;60 mutant. We took         labeled tRNA bands (lanes 4 5), could <br /> advantage thermosensitive phenotype Ntag-&#65533;?&#65533;60          correspond partially matured tRNAs, 12 h the <br /> mutant explore role TFC8 gene product              labeling tRNA bands selectively reduced (lane 5). <br /> TFIIIC. The effect mutation Pol III transcripts vivo   Clearly, fusion HA epitope &#65533;?&#65533;60 N-terminal <br /> is shown Fig. 2A. The wild-type mutant cells grown      extremity disturbed Pol III transcription system. Like in <br /> at 30&#176;C shifted nonpermissive temperature        previous studies different Pol III mutants, syn- <br /> (37&#176;C) 6 12 h. The cells grown 30 37&#176;C            thesis 5S RNA did decrease significantly (20, 41). <br /> incubated temperature tritiated uracil 15        A similar tRNA maturation defect previously ob- <br /> min; RNAs extracted, separated 7 M urea          served thermosensitive mutants TFIIIC subunits, <br /> gel, analyzed autoradiography. No difference ob-         &#65533;?&#65533;138 (37), &#65533;?&#65533;131 (56) or, recently, &#65533;?&#65533;91 (1a). To <br />  VOL. 19, 1999                                                                              &#65533;?&#65533;60 PARTICIPATES IN TFIIIB RECRUITMENT                             8045 <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 3. Suppressibility Ntag-&#65533;?&#65533;60 mutation overexpression Pol III- <br />                                                                                    related genes. Stationary-phase cultures mutant cells harboring different mul- <br />                                                                                    ticopy plasmids diluted 10-, 102-, 103-fold water spotted (5 &#65533;?&#65533;l) <br />                                                                                    solid minimum medium (Casa&#11001;Ade plates). The plates incubated for <br />                                                                                    4 days 37&#176;C. The different genes harbored high-copy-number vector <br />                                                                                    pFL44L indicated. &#11002;, plasmid insert. RPR1 encodes RNA co- <br />                                                                                    factor RNase P. <br />  <br />  <br />  <br />  <br />                                                                                    explore possible relationship maturation defect, <br />                                                                                    TFIIIC function cell lethality phenotype, per- <br />                                                                                    formed Northern blot analysis various tRNA precursors. <br />                                                                                    Total RNA extracted wild-type mutant strains <br />                                                                                    grown 8 h 37&#176;C, separated electrophoresis under <br />                                                                                    denaturing conditions, transferred membrane, and <br />                                                                                    probed various labeled tRNA genes (Fig. 2B). The wild- <br />                                                                                    type mutant strain contained multicopy plasmid <br />                                                                                    pFL44L, mutant strains contained pFL44L <br />                                                                                    harboring RPR1 SPT15 (TBP). The precursor pattern <br />                                                                                    tRNA3Leu mutant characterized presence <br />                                                                                    extra RNA band absent wild-type extract (compare <br />                                                                                    lanes 1 2). A similar result obtained labeled- <br />                                                                                    SUP4tDNATyr tDNA3Glu (data shown). <span id='am-94' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-98'  ></span>This</span> addi- <br />                                                                                    tional RNA species previously observed RNase P <br />                                                                                    mutants shown correspond unmatured spliced <br />                                                                                    tRNA (18, 35, 58). RNase P endonuclease cleaves <br />                                                                                    pre-tRNA substrates yield mature 5&#11032; extremity. When the <br />                                                                                    RNA component RNaseP, RPR1, overexpressed the <br />                                                                                    mutant strain, anomalous RNA species disappeared (lane <br />                                                                                    3). However, mutant cells overexpressing RPR1 retained <br />                                                                                    thermosensitive lethal phenotype (see Fig. 3). In contrast <br />                                                                                    RPR1, overexpression TBP did change tRNA <br />                                                                                    maturation pattern mutant (Fig. 2, lane 4) sup- <br />                                                                                    pressed lethality (see below). A direct relationship between <br />                                                                                    maturation defect thermosensitive phenotype of <br />                                                                                    Ntag-&#65533;?&#65533;60 mutant excluded. The tRNA <br />                                                                                    maturation problem likely consequence reduced <br />    FIG. 2. Comparison vivo tRNA synthesis wild-type mutant            synthesis class III RPR1 RNA, class III <br /> (Ntag-&#65533;?&#65533;60) yeast strains. (A) Autoradiogram vivo pulse-labeling exper- <br /> iments. [3H]uracil incorporation wild-type mutant strains    transcripts (34). <br /> 15 min 30&#176;C (lane 1, wild-type; lane 2, mutant) 37&#176;C 6 h (lane 4,      We looked multicopy suppressors compensate <br /> mutant) 12 h (lane 3, wild-type; lane 5, mutant) growth 37&#176;C. RNA         interaction defects TFIIIC preinitiation com- <br /> species analyzed electrophoresis 7 M urea gel (6% polyacrylamide)     plex. The Ntag-&#65533;?&#65533;60 mutant cells transformed high- <br /> by using equal amounts RNA (6 &#65533;?&#65533;g lane). The asterisk indicates the <br /> trailing radioactive material tRNA species, suggesting defect    copy-number plasmids harboring different genes Pol III <br /> tRNA maturation. (B) Northern blot analysis tRNA precursors wild-type      transcription system. A series cell dilutions plated on <br /> and mutant cells labeled tDNA3Leu probe. RNA extraction      minimal medium incubated permissive nonpermis- <br /> 8 h growth 37&#176;C. Gel electrophoresis, electrophoretic transfer nylon      sive temperature. Among TFIIIC genes, TFC8, <br /> membrane, hybridization DNA probe described Mate- <br /> rials Methods. The positions primary transcript tRNA3Leu, 5&#11032;-    TFC1 (encoding &#65533;?&#65533;95) acted multicopy suppressor <br /> and 3&#11032;-processed nonspliced precursor, spliced 5&#11032;- 3&#11032;-unproc-      (Fig. 3 Table 1). None suppressors tsv115 muta- <br /> essed precursor, mature tRNA3Leu indicated right. Lanes 1       tion TFC3, RPR1, NOP1, FHL1 (37), involved in <br /> and 2, wild-type mutant cells, respectively, containing high-copy-number   maturation stable RNAs, able suppress the <br /> plasmid pFL44L; lane 3, mutant cells containing high-copy-number plasmid <br /> pFL44L harboring RPR1 gene; lane 4, mutant cells containing pFL44L harboring       thermosensitive defect mutant cells (Table 1). Figure 3 <br /> SPT15 gene (TBP).                                                                  shows high dosage TFIIIB genes, TFC5 <br />                                                                                    (TFIIIB90), SPT15 (TBP), BRF1 (TFIIIB70), sup- <br />  8046        DEPREZ ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />        TABLE 1. Suppression &#65533;?&#65533;60 mutation gene dosage <br />                                                  Suppression on: <br /> Multicopy <br />                       Gene product            Minimal       Rich <br />  gene <br />                                               medium       medium <br />  <br />  TFC8         &#65533;?&#65533;60                             &#11001;&#11001;&#11001;           &#11001;&#11001;&#11001; <br />  RPR1         RNA component RNase P         &#11002;             &#11002; <br />                 (tRNA maturation) <br />  NOP1         Fibrillarin homolog (rRNA         &#11002;             &#11002; <br />                 maturation) <br />  FHL1         Forkhead-like protein (rRNA       &#11002;             &#11002; <br />                 maturation) <br />  TFC3         &#65533;?&#65533;138, subunit TFIIIC          &#11002;              &#11002; <br />  TFC4         &#65533;?&#65533;131, subunit TFIIIC          &#11002;              &#11002; <br />  TFC1         &#65533;?&#65533;95, subunit TFIIIC           &#11001;&#11001;             &#11002; <br />  TFC6         &#65533;?&#65533;91, subunit TFIIIC           &#11002;              &#11002; <br />  TFC7         &#65533;?&#65533;55, subunit TFIIIC           &#11002;              &#11002; <br />  BRF1         Brf1/TFIIIB70, subunit        &#11002;              &#11002; <br />                 TFIIIB <br />  SPT15        TBP (TATA-binding                &#11001;&#11001;             &#11001; <br />                 protein), subunit TFIIIB <br />  TFC5         B&#11033;/TFIIIB90, subunit          &#11001;&#11001;             &#11002; <br />                 TFIIIB <br />  <br />  <br />  <br />  <br /> pressed lethal phenotype Ntag-&#65533;?&#65533;60 mutant cells min- <br /> imal medium 37&#176;C. Remarkably, overproduction of <br /> SPT15 (TBP) restore cell growth rich YPD medium <br /> at nonpermissive temperature (the suppression level was <br /> weaker conditions observed min-            FIG. 4. DNA-binding properties wild-type mutant TFIIIC. (A) <br /> imal medium, Table 1).                                          TFIIIC-tDNA complex formation tRNA3Glu SUP4tRNATyr genes. <br />    In vitro characterization Ntag-&#65533;?&#65533;60 mutant. TFIIIC         TFIIIC-DNA complexes formed Ntag-&#65533;?&#65533;60 wild-type TFIIIC prep- <br />                                                                     arations (MonoQ fraction) analyzed electrophoresis described in <br /> factor partially purified wild-type strain      Materials Methods. To add similar amounts wild-type mutant TFIIIC <br /> the strain expressing HA N-terminal version &#65533;?&#65533;60           (based Western blot experiments), twice mutant protein fraction was <br /> compare DNA binding properties. TFIIIC fractions pu-          added follows: 25 ng (lanes 2 7) 100 ng (lanes 3 8) wild-type <br /> rified mutant strain reproducibly West-      TFIIIC 50 ng (lanes 4 9) 200 ng (lanes 5 10) mutant TFIIIC <br />                                                                     fraction. Lanes 1 5, tRNA3Glu gene; lanes 6 10, SUP4tRNATyr gene. (B) <br /> ern blotting anti-&#65533;?&#65533;60 anti-&#65533;?&#65533;55 polyclonal antibodies       Effect temperature &#65533;?&#65533;B-tDNA complex stability. Preformed TFIIIC- <br /> (40) contain half TFIIIC wild-type fractions.         tDNA3Glu complexes digested &#65533;?&#65533;-chymotrypsin generate &#65533;?&#65533;B-DNA <br /> Therefore, care taken use wild-type       complexes, proteolytic digestion stopped addition aprotinin, the <br /> or mutant TFIIIC following gel retardation experiments.      mixtures incubated 10 min various temperatures as <br />                                                                     indicated. Lanes 1 8, wild-type mutant TFIIIC-tDNA complexes, re- <br /> Two labeled probes, tDNA3Glu SUP4tDNATyr, incu-             spectively; lanes 2 7, wild-type &#65533;?&#65533;B; lanes 9 14, mutant &#65533;?&#65533;B. &#65533;?&#65533;, TFIIIC-tDNA <br /> bated limiting amounts wild-type mutant TFIIIC.          complex; &#65533;?&#65533;B, &#65533;?&#65533;B-tDNA complex. <br /> TFIIIC-tDNA complexes analyzed electrophoresis un- <br /> der nondenaturing conditions revealed autoradiogra- <br /> phy. Figure 4A shows yields TFIIIC-DNA com- <br /> plexes formed wild-type mutant TFIIIC factor                Wild-type mutant TFIIIC compared their <br /> tDNA3Glu SUP4tDNATyr similar. In addition,             ability direct specific transcription various tRNA genes in <br /> wild-type mutant TFIIIC-tDNA complex stabilities var-        vitro. Identical amounts TFIIIC preparations, cali- <br /> ious temperatures ionic strengths indistinguish-       brated Western blot analysis anti-&#65533;?&#65533;60 anti-&#65533;?&#65533;55 <br /> able (data shown). In contrast, TFIIIC-tDNA3Glu            polyclonal antibodies gel retardation analysis with <br /> complexes subjected limited proteolysis, significantly      tDNA3Glu SUP4tDNATyr probes, used reconstituted <br /> fewer mutant &#65533;?&#65533;B-tDNA complexes obtained compared            transcription assays presence B&#11033; fraction, recombinant <br /> the wild type (Fig. 4B, compare lanes 2 9). Moreover,           TBP (rTBP), TFIIIB70 (rTFIIIB70), purified RNA Pol <br /> mutant &#65533;?&#65533;B-tDNA complexes stable wild type         III. As shown Fig. 5, transcriptional activity mutant <br /> increasing temperatures. The mutant &#65533;?&#65533;B-tDNA               TFIIIC similar wild-type factor with <br /> complexes decreased drastically 10 min 55&#176;C 5%       tDNA3Glu, tDNA2Lys, tDNA1Ile templates (lanes 1 2, <br /> the initial control level 25&#176;C, 60% wild-type &#65533;?&#65533;B-       7 8, 11 12). Surprisingly, transcriptional ac- <br /> tDNA complexes resisted conditions (lanes 6          tivity mutant factor impaired tem- <br /> and 13), determined using PhosphorImager Image-        plates, SUP4tDNATyr, tDNA1Phe, tDNA1Pro (lanes <br /> Quant software (Molecular Dynamics). Whatever struc-            3 6 9 10). However, shown previously, mutant <br /> tural defect causing increased thermolability (either       wild-type factor preparations bound similarly tDNA3Glu <br /> presence HA epitope subtle difference pro-        SUP4tDNATyr probes (see above). Since difference in <br /> teolysis indirectly caused presence epitope),   DNA-binding properties account differ- <br /> observations clearly revealed defect &#65533;?&#65533;B level, enforcing   ence transcription efficiency, explored ability mu- <br /> the <span id='am-64' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> N-terminal &#65533;?&#65533;60 belongs B           tant TFIIIC recruit TFIIIB. We used tDNA3Leu, was, <br /> block-binding domain.                                               like SUP4tDNATyr, poor template, mutant TFIIIC. <br />  VOL. 19, 1999                                                                             &#65533;?&#65533;60 PARTICIPATES IN TFIIIB RECRUITMENT                  8047 <br />  <br />  <br />                                                                                    tDNA3Glu template (Fig. 6E). Therefore, inferred the <br />                                                                                    TBP concentration standard transcription assay was <br />                                                                                    optimal tDNA3Glu, tDNA2Lys, tDNA1Ile limiting <br />                                                                                    SUP4tDNATyr, tDNA1Phe, tDNA1Pro, tDNA3Leu. The <br />                                                                                    analysis promoter region genes did disclose <br />                                                                                    feature TFIIIB binding region account <br />                                                                                    differential TBP requirement. <br />                                                                                       &#65533;?&#65533;60 interacts TBP. Two lines evidence indicated that <br />                                                                                    mutant TFIIIC defective TBP recruitment, vivo <br />                                                                                    (the multicopy suppressor studies) vitro (the transcrip- <br />                                                                                    tion experiments). We explored possibility that <br />                                                                                    &#65533;?&#65533;60 subunit directly participate TBP binding. The <br />                                                                                    physical interaction &#65533;?&#65533;60 TBP investigated by <br />                                                                                    coimmunoprecipitation experiments recombinant pro- <br />                                                                                    teins. rTBP (200 ng) preincubated 90 min 25&#176;C with <br />    FIG. 5. Specific transcription various tRNA genes presence         HA-r&#65533;?&#65533;60 (650 ng) transcription buffer. The protein samples <br /> wild-type mutant TFIIIC. In vitro transcription performed described in <br /> Materials Methods using 100 200 ng wild-type Ntag-&#65533;?&#65533;60            added magnetic beads precoated anti-HA <br /> MonoQ-TFIIIC fraction (respectively), rTBP, rTFIIIB70, partially purified B&#11033;       antibodies, beads washed buffer, and <br /> fraction, RNA Pol III. The different plasmid templates indicated. Odd      bound proteins eluted SDS competition with <br /> lanes, wild-type TFIIIC. Even lanes, mutant TFIIIC.                                HA peptide. The input eluted proteins subse- <br />                                                                                    quently analyzed SDS-PAGE immunoblotting an- <br />                                                                                    ti-TBP anti-&#65533;?&#65533;60 antibodies (Fig. 7). Compared low <br /> Wild-type mutant TFIIIC similar DNA-binding                            background level TBP retained beads absence <br /> properties tDNA3Leu (data shown). Transcription                         &#65533;?&#65533;60, sevenfold-higher TBP bound upon <br /> performed presence increasing amounts TFIIIB                          preincubation &#65533;?&#65533;60, quantified using ImageQuant <br /> with wild-type mutant TFIIIC. Remark-                        software. A similar result obtained bound proteins <br /> ably, shown Fig. 6A, high doses TFIIIB fraction                     eluted competition HA peptide the <br /> compensate transcription defect mutant TFIIIC. The                      beads washed extensively transcription buffer <br /> relative transcriptional activity mutant TFIIIC reached 92%                     (data shown). The background-corrected TBP/Tfc8 ratio in <br /> of wild-type level 19 ng TFIIIB &#65533;?&#65533;l, concentration                  eluted fraction quantified corresponded to <br /> at TFIIIB longer limiting wild-type TFIIIC                       0.56 (compared 0.9 input), showing mole- <br /> (the maximum tRNA synthesis presence wild-type                        cule bound &#65533;?&#65533;60 specifically associated 0.5 molecule of <br /> TFIIIC reached ca. 15 ng TFIIIB &#65533;?&#65533;l).                                 TBP. From result, tentatively estimate Kd TBP- <br />    To explore TFIIIB component critical TFIIIB                       &#65533;?&#65533;60 interaction ca. 100 nM, indicative a <br /> assembly mutant TFIIIC, performed multiple-round                             reasonably strong interaction. The Kd reported for <br /> transcriptions tDNA3Leu template various                              TFIIIB70-TBP interaction Sethy-Coraci et al. (55). These <br /> amounts rTFIIIB70 (Fig. 6B), B&#11033; (Fig. 6C), rTBP (Fig.                        observations strongly suggested direct interaction between <br /> 6D). Increasing concentration rTFIIIB70 did correct                     &#65533;?&#65533;60 subunit TBP. In similar experiment, physical <br /> the transcription defect mutant TFIIIC (Fig. 6B). In 12-                   interaction detected r&#65533;?&#65533;60 rTFIIIB70: <br /> 48-ng/&#65533;?&#65533;l concentration range rTFIIIB70 factor, tRNA syn-                        Western blot analysis revealed significant difference the <br /> thesis increased similarly wild-type mutant                           amounts rTFIIIB70 bound beads presence or <br /> TFIIIC, relative transcriptional efficiency mutant                      absence HA-r&#65533;?&#65533;60 (data shown). <br /> TFIIIC remained low constant (8 16%). At concen- <br /> tration 48 ng/&#65533;?&#65533;l, rTFIIIB70 appeared titrate a <br /> component transcription system, causing dramatic                                                    DISCUSSION <br /> decrease tRNA synthesis directed wild-type and <br /> mutant TFIIIC. As shown Fig. 6C, increasing concen-                            We report characterization 60-kDa polypep- <br /> tration B&#11033; 11- 45-ng/&#65533;?&#65533;l concentration range did                    tide present affinity-purified TFIIIC. This polypeptide, <br /> not correct transcription B&#11033; 11- 45-ng/&#65533;?&#65533;l concen-                   named &#65533;?&#65533;60, shown essential subunit TFIIIC. The <br /> tration range did correct transcription defect                     yeast TFIIIC factor, therefore, comprises subunits, &#65533;?&#65533;138, <br /> mutant TFIIIC. At concentration B&#11033; &#11022;45 ng/&#65533;?&#65533;l,                          &#65533;?&#65533;131, &#65533;?&#65533;95, &#65533;?&#65533;91, &#65533;?&#65533;60, &#65533;?&#65533;55, essential cell <br /> transcription efficiency wild-type mutant TFIIIC                       viability (Table 2). &#65533;?&#65533;60 appears reside, part, &#65533;?&#65533;B, <br /> decreased similarly (Fig. 6C). The dosage-dependent effect                      downstream DNA-binding domain. Paradoxically, how- <br /> rTBP transcriptional efficiency totally different (Fig.                     ever, analysis &#65533;?&#65533;60 mutant indicated &#65533;?&#65533;60 plays role in <br /> 6D). In concentration range 2.5 8 ng rTBP factor                    TFIIIB assembly interaction TBP. The results sug- <br /> &#65533;?&#65533;l, transcriptional efficiency optimal wild-type                       gest network interactions linking TFIIIC TFIIIB com- <br /> TFIIIC. In contrast, mutant TFIIIC, tRNA synthesis                        ponents preinitiation complex formation. <br /> rate increased rTBP concentration reach 97%                            The demonstration &#65533;?&#65533;60 integral subunit TFIIIC <br /> wild-type level. This result indicated rTBP limiting                  rests biochemical genetic evidence. First, antibodies <br /> component TFIIIB assembly directed mutant TFIIIC,                        directed C-terminally HA-tagged &#65533;?&#65533;60 altered migration <br /> least tDNA3Leu.                                                                 tagged TFIIIC-tDNA &#65533;?&#65533;B-tDNA complexes. Second, a <br />    The observation tRNA genes normally                           thermosensitive mutant TFIIIC (harboring N-terminally HA- <br /> transcribed presence mutant TFIIIC (Fig. 5) sug-                         tagged &#65533;?&#65533;60) affected tRNA gene transcription vivo <br /> gested templates different TBP require-                      vitro. Third, vivo phenotype suppressed by <br /> ments. Indeed, decreasing TBP concentra-                         overexpression components class III transcription <br /> tion revealed defect mutant TFIIIC transcription                    machinery (a TFIIIC subunit, &#65533;?&#65533;95, components the <br />  8048       DEPREZ ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Effects TFIIIB, TFIIIB70, B&#11033;, TBP concentration transcription tRNA3Leu tRNA3Glu genes presence wild-type mutant TFIIIC. <br /> (A) Effect TFIIIB. Transcription mixtures (40 &#65533;?&#65533;l) contained 100 ng wild-type 200 ng Ntag-&#65533;?&#65533;60 TFIIIC, 220 ng plasmid DNA harboring tRNA3Leu <br /> gene, RNA polymerase, various concentrations TFIIIB (Cibacron blue fraction) indicated. (B) Effect TFIIIB70. Transcription mixtures contained wild-type <br /> or mutant TFIIIC, plasmid DNA, RNA polymerase panel A, 2.5 ng rTBP &#65533;?&#65533;l, 45 ng partially purified B&#11033; fraction &#65533;?&#65533;l, various concentrations <br /> of rTFIIIB70 indicated. (C) Effect B&#11033;. Transcription mixtures contained wild-type mutant TFIIIC, plasmid DNA, RNA polymerase panel A, 4 ng of <br /> rTBP &#65533;?&#65533;l, 35 ng rTFIIIB70 &#65533;?&#65533;l, various concentrations partially purified B&#11033; fraction indicated. (D) Effect TBP. Lanes panels, with <br /> 35 ng rTFIIIB70 &#65533;?&#65533;l, 45 ng partially purified B&#11033; fraction &#65533;?&#65533;l, various concentrations rTBP indicated. (E) Effect TBP. Lanes described <br /> for panel D, plasmid DNA harboring tRNA3Glu gene instead tRNA3Leu gene used template. Transcripts analyzed by <br /> electrophoresis autoradiography (left panels). Odd lanes, wild-type TFIIIC; lanes, mutant TFIIIC. Transcripts quantified (right panels) using a <br /> PhosphorImager ImageQuant software (Molecular Dynamics). a.u., arbitrary units. Gray bars, wild-type TFIIIC; black bars, mutant TFIIIC. The relative <br /> transcription efficiencies mutant versus wild-type TFIIIC indicated bar graphs. ND, determined. Two independent experiments <br /> were compiled quantification. <br />  <br />  <br />  <br /> TFIIIB factor, TFIIIB90/B&#11033; TBP). Fourth, vitro                              coding &#65533;?&#65533;60 essential cell viability. Altogether, re- <br /> transcription defect strongly alleviated increasing                         sults indicate &#65533;?&#65533;60 plays important role TFIIIC. <br /> concentration TFIIIB TBP. Finally, like sub-                             The location TFIIIC subunits site-specific DNA cross- <br /> units TFIIIC previously characterized, TFC8 gene en-                            linking provided broad view possible functions of <br />  VOL. 19, 1999                                                                              &#65533;?&#65533;60 PARTICIPATES IN TFIIIB RECRUITMENT                8049 <br />  <br />  <br />                                                                                  weakened stability &#65533;?&#65533;B-DNA complex. However, mutant <br />                                                                                  TFIIIC-DNA complex formation notably affected <br />                                                                                  compared wild-type factor, unfavorable con- <br />                                                                                  ditions, high ionic strengths, high temperatures. There- <br />                                                                                  fore, role &#65533;?&#65533;60 DNA binding excluded not <br />                                                                                  strongly supported present data, keeping lack <br />                                                                                  detectable &#65533;?&#65533;60-DNA cross-linking (28). Schematic models of <br />                                                                                  TFIIIC-DNA complexes based site-specific cross-linking <br />                                                                                  studies tentatively visualized 60-kDa polypeptide being <br />                                                                                  unbound DNA, connecting &#65533;?&#65533;95 &#65533;?&#65533;138 subunits, and <br />                                                                                  overlapping A block-B block interval (10). This proposal <br />                                                                                  fits remarkably present observations, &#65533;?&#65533;60 <br />                                                                                  appears reside partly &#65533;?&#65533;B possibly contact <br />                                                                                  &#65533;?&#65533;95, suggested vivo suppression &#65533;?&#65533;60 mutant <br />                                                                                  lethality overexpression &#65533;?&#65533;95, two-<span id='am-160' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-161'  ></span><span id='am-162' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> exper- <br />                                                                                  iments did confirm interaction (data shown). <br />     FIG. 7. Coimmunoprecipitation experiment. HA-tagged r&#65533;?&#65533;60 (650 ng)           In view position &#65533;?&#65533;60 TFIIIC-DNA complex, <br /> preincubated rTBP (200 ng) 1.5 h 25&#176;C purified magnetic       effect &#65533;?&#65533;60 mutation level TFIIIB recruitment <br /> beads coated anti-HA antibodies. Bound proteins eluted analyzed <br /> by Western blotting anti-&#65533;?&#65533;60 anti-TBP polyclonal antibodies. The po- <br />                                                                                  unexpected. Since &#65533;?&#65533;131 subunit of <br /> sitions rTBP native HA-r&#65533;?&#65533;60 shown. The intermediate bands      TFIIIC extending upstream transcriptional start site (7), <br /> specifically revealed anti-&#65533;?&#65533;60 antibodies. Lane 1, rTBP&#11001;HA-r&#65533;?&#65533;60; lane 2,      assumed entirely responsible TFIIIB <br /> rTBP alone; lane 3, 25% input rTBP&#11001;HA-r&#65533;?&#65533;60 proteins.                          assembly. Indeed, &#65533;?&#65533;131 TFIIIC subunit found <br />                                                                                  interact TFIIIB70/Brf1 TFIIIB90/B&#11033; (14, 33, 51). <br />                                                                                  Several lines evidence, however, indicate &#65533;?&#65533;60 likely <br /> the different polypeptides TFIIIC-DNA complex                         participates TFIIIB recruitment: (i) lethality the <br /> (Table 2), good agreement                         Ntag-&#65533;?&#65533;60 variant nonpermissive temperature sup- <br /> biochemical genetic data (28). Unfortunately, &#65533;?&#65533;60                     pressed overexpression TFIIIB components, <br /> only TFIIIC polypeptide cross-linked                       TFIIIB90/B&#11033; TBP (not TFIIIB70 though); (ii) in <br /> located elegant powerful approach (its subunit                       vitro transcription defect mutant TFIIIC strongly <br /> status questioned reason). Two inde-                    alleviated increasing TFIIIB concentration spe- <br /> pendent experiments suggested &#65533;?&#65533;60 resides &#65533;?&#65533;B do-                     cifically increasing TBP concentration (again TFIIIB70 <br /> main: &#65533;?&#65533;B-DNA complexes obtained proteolysis TFIIIC                         B&#11033;); (iii) &#65533;?&#65533;60 interact directly TBP. <br /> harboring HA-tagged &#65533;?&#65533;60 thermosen-                         The possibility indirect role &#65533;?&#65533;60 TFIIIB recruitment <br /> sitive untagged complexes recognized antibod-                    envisioned view possible interaction &#65533;?&#65533;95 <br /> ies directed HA epitope. Since, experiments,                 belongs &#65533;?&#65533;A domain. The &#65533;?&#65533;60 mutation perturb <br /> HA tag placed N-terminal C-termi-                    &#65533;?&#65533;95-A block binding affect indirectly, &#65533;?&#65533;95, TFIIIB assem- <br /> nal end polypeptide, inferred extremities                 bly &#65533;?&#65533;131. If case, expect suppres- <br /> &#65533;?&#65533;60 resided &#65533;?&#65533;B, &#65533;?&#65533;138 (19) &#65533;?&#65533;91 (1). The                      sion vivo vitro defects increased TFIIIB70 <br /> total mass polypeptides, 275 kDa, corre-                    dosage. Indeed, mutations TFIIIC (37), TBP (12, 16), or <br /> spond reasonably mass &#65533;?&#65533;B (&#11011;300 kDa) visualized                    promoter elements decreasing factor-DNA interaction (39) <br /> by scanning transmission electron microscopy B block                    best suppressed vivo overexpres- <br /> (53). Note <span id='am-154' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> &#65533;?&#65533;B electron microscopy                  sion TFIIIB70. This makes sense TFIIIB70 interacts <br /> by limited proteolysis necessarily equivalent. The pro-                  TFIIIC (14, 33), initiates TFIIIB assembly (31), ap- <br /> tease-resistant form &#65533;?&#65533;B observed gel retardation assay                     pears limiting transcription whole-cell extracts <br /> may contain pieces TFIIIC unrelated B-block-binding                    vivo (39, 54, 55). Since increasing TFIIIB70 con- <br /> function. The participation &#65533;?&#65533;60 &#65533;?&#65533;B (as defined limited                  centration did correct TFIIIC mutant phenotype in <br /> proteolysis gel retardation assay) points role                  vivo vitro, rate-limiting step caused &#65533;?&#65533;60 <br /> subunit DNA binding. Indeed, presence HA                           mutation expected follow TFIIIB70 recruitment step, <br /> epitope N-terminal end &#65533;?&#65533;60 decreased yield                     i.e., TBP recruitment step (31). Since appears unlikely <br />  <br />  <br />                                                            TABLE 2. Subunits yeast TFIIIC <br />             Calculated                    Null             Location proposed role                                                  Source(s) or <br /> Subunit                      Gene                                                                              Features <br />             mass (kDa)                  phenotype             TFIIIC-DNA complex                                                           reference <br />  <br />  &#65533;?&#65533;138           132         TFC3         Lethal        In &#65533;?&#65533;B; B block binding                  Two HMG domains <br />  &#65533;?&#65533;131           120         TFC4         Lethal        Most 5&#11032;; TFIIIB assembly;               11 TPR motifs; phosphorylated;               23, 41a <br />                                                          interacts TFIIIB70             ortholog H. sapiens, C. elegans, <br />                                                          TFIIIB90                                K. lactis <br />  &#65533;?&#65533;95             74         TFC1         Lethal        In &#65533;?&#65533;A; A block binding; interacts       Ortholog H. sapiens, D.m., C. elegans     23, 27a <br />                                                          &#65533;?&#65533;55                                S. pombe <br />  &#65533;?&#65533;91             75         TFC6         Lethal        Most 3&#11032;; cooperates &#65533;?&#65533;138        (Cys&#11001;His)-rich domain; ortholog H.        1a <br />                                                          DNA binding                             sapiens S. pombe <br />  &#65533;?&#65533;60             68         TFC8          Lethal       In &#65533;?&#65533;B; TFIIIB assembly; interacts <br />                                                          TBP <br />  &#65533;?&#65533;55             49         TFC7          Lethal       In &#65533;?&#65533;A; interacts &#65533;?&#65533;95               Chimeric protein <br />  8050       DEPREZ ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> that &#65533;?&#65533;60 mutation indirect effect                                 Araujo, A. Aparicio, B. Barrell, K. Badcock, V. Benes, D. Botstein, S. Bow- <br /> recruitment TBP TFIIIB70, favor idea                              man, M. Bruckner, J. Carpenter, J. M. Cherry, E. Chung, C. Churcher, F. <br />                                                                                            Coster, K. Davis, R. W. Davis, F. S. Dietrich, H. Delius, T. DiPaolo, J. Hani, <br /> &#65533;?&#65533;60 directly involved TFIIIB recruitment contacting                               et al. 1997. The nucleotide sequence <span id='am-40' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae chromo- <br /> TBP. The coimmunoprecipitation TBP &#65533;?&#65533;60 comes                                     XVI. Nature 387:103&#8211;105. <br /> support proposal. The interaction TBP TFIIIC                           14. Chaussivert, N., C. Conesa, S. Shaaban, A. Sentenac. 1995. Complex <br /> subunits previously reported. However, Huet et                                interactions yeast TFIIIB TFIIIC. J. Biol. Chem. 270:15353&#8211; <br />                                                                                            15358. <br /> al. (26) noted TBP slightly retarded migration                        15. Church, G. M., W. Gilbert. 1984. Genomic sequencing. Proc. Natl. Acad. <br /> TFIIIC-DNA complexes TATA-less gene. Since TFIIIC                                     Sci. USA 81:1991&#8211;1995. <br /> specifies orientation TFIIIB DNA (and                            16. Colbert, T., S. Hahn. 1992. A yeast TFIIB-related factor involved in <br /> the direction transcription) (60), direct interaction                            RNA polymerase III transcription. Genes Dev. 6:1940&#8211;1949. <br />                                                                                        17. Conesa, C., R. N. Swanson, P. Schultz, P. Oudet, A. Sentenac. 1993. On <br /> TFIIIC TBP, &#65533;?&#65533;60, contribute fix                                 subunit composition, stoichiometry, phosphorylation yeast <br /> correct orientation DNA. Alternatively, &#65533;?&#65533;60-TBP inter-                              transcription factor TFIIIC/&#65533;?&#65533;. J. Biol. Chem. 268:18047&#8211;18052. <br /> action involved positioning TBP DNA                           18. Dichtl, B., D. Tollervey. 1997. Pop3p essential activity the <br /> only represent transient step facilitating delivery TBP                           RNase MRP RNase P ribonucleoproteins vivo. EMBO J. 16:417&#8211;429. <br />                                                                                        19. Gabrielsen, O. S., N. Marzouki, A. Ruet, A. Sentenac, P. Fromageot. <br /> to TFIIIB70-TFIIIC-DNA complex.                                                        1989. Two polypeptide chains yeast transcription factor &#65533;?&#65533; interact with <br />                                                                                            DNA. J. Biol. Chem. 264:7505&#8211;7511. <br />                           ACKNOWLEDGMENTS                                              20. Gudenus, R., S. Mariotte, A. Moenne, A. Ruet, S. Me&#180;met, J.-M. Buhler, A. <br />                                                                                            Sentenac, P. Thuriaux. 1988. Conditional mutants RPC160, gene <br />    We grateful Christophe Carles, Michel Riva, Olivier Lefebvre,                    encoding largest subunit RNA polymerase C Saccharomyces cer- <br /> and Franc&#184;oise Bouet help peptide microsequence deter-                        evisiae. Genetics 119:517&#8211;526. <br /> mination Jacques Grassi Christophe Creminon                       21. Heinemeyer, W., A. Gruhler, V. Mohrle, Y. Mahe, D. H. Wolf. 1993. <br /> help raising polyclonal antibodies. We thank Emmanuel Favry                         PRE2, highly homologous human major histocompatibility complex- <br />                                                                                            linked RING10 gene, codes yeast proteasome subunit necessary for <br /> protein preparations Olivier Lefebvre advice TFIIIC puri-                       chymotryptic activity degradation ubiquinated proteins. J. Biol. <br /> fication immunoprecipitation experiments. We thank Chris-                         Chem. 268:5115&#8211;5120. <br /> tian Marck He&#180;le`ne Dumay providing plasmids harboring dif-                    22. Hermann-Le Denmat, S., M. Werner, A. Sentenac, P. Thuriaux. 1994. <br /> ferent tRNA genes helpful discussions Olivier Lefebvre,                        Suppression yeast RNA polymerase III mutations FHL1, gene coding <br /> Michel Riva, Michel Werner improving manuscript.                               fork head protein involved rRNA processing. Mol. Cell. Biol. 14: <br />    E.D. supported postdoctoral fellowships Associa-                        2905&#8211;2913. <br /> tion pour la Recherche contre le <span id='am-69' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-70' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73'  ></span>Cancer</span>, Ligue Nationale contre le                 23. Hsieh, Y.-J., Z. Wang, R. Kovelman, R. G. Roeder. 1999. Cloning and <br /> Cancer, Fondation pour la Recherche Me&#180;dicale, R.A.                        characterization evolutionarily conserved subunits (TFIIIC102 and <br />                                                                                            TFIIIC63) human TFIIIC involvement functional interactions <br /> supported postdoctoral fellowship European Union <br />                                                                                            TFIIIB RNA polymerase III. Mol. Cell. Biol. 19:4944&#8211;4952. <br /> (Marie Curie Training Mobility Researchers).                                    24. Huet, J., C. Conesa, C. Carles, A. Sentenac. 1997. A cryptic DNA <br />                                                                                            binding domain COOH terminus TFIIIB70 affects formation, sta- <br />                                  REFERENCES                                                bility, function preinitiation complexes. J. Biol. Chem. 272:18341&#8211; <br />  1. Arrebola, R., N. Manaud, S. Rozenfeld, M. C. Marsolier, O. Lefebvre, C.                18349. <br />     Carles, P. Thuriaux, C. Conesa, A. Sentenac. 1998. &#65533;?&#65533;91, essential           25. Huet, J., N. Manaud, G. Dieci, G. Peyroche, C. Conesa, O. Lefebvre, A. Ruet, <br />     subunit yeast TFIIIC, cooperates &#65533;?&#65533;138 DNA binding. Mol. Cell.               M. Riva, A. Sentenac. 1996. RNA polymerase III class III transcrip- <br />     Biol. 18:1&#8211;9.                                                                          tion factors Saccharomyces cerevisiae. Methods Enzymol. 273:249&#8211;267. <br />  1a.Arrebola, R. Unpublished data.                                                     26. Huet, J., A. Sentenac. 1992. The TATA-binding protein participates in <br />  2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A.            TFIIIB assembly tRNA genes. Nucleic Acids Res. 20:6451&#8211;6454. <br />     Smith, K. Struhl. 1994. Current protocols molecular biology. John           27. Joazeiro, C. A. P., G. A. Kassavetis, E. P. Geiduschek. 1994. Identical <br />     Wiley &amp; Sons, Inc., New York, N.Y.                                                     components yeast transcription factor IIIB required sufficient for <br />  3. Baker, R. E., S. Camier, A. Sentenac, B. D. Hall. 1987. Gene size                  transcription TATA box-containing TATA-less genes. Mol. Cell. Biol. <br />     differentially affects binding yeast transcription factor &#65533;?&#65533; intra-       14:2798&#8211;2808. <br />     genic regions. Proc. Natl. Acad. Sci. USA 84:8768&#8211;8772.                            27a.Jourdain, S. Personal communication. <br />  4. Baker, R. E., O. S. Gabrielsen, B. D. Hall. 1986. Effects tRNATyr           28. Kassavetis, G. A., C. Bardeleben, B. Bartholomew, B. R. Braun, C. A. P. <br />     point mutations binding yeast RNA polymerase III transcription               Joazeiro, M. Pisano, E. P. Geiduschek. 1994. Transcription RNA <br />     factor C. J. Biol. Chem. 261:5275&#8211;5282.                                                polymerase III, p. 107&#8211;126. In R. C. Conaway J. W. Conaway (ed.), <br />  5. Bartholomew, B., B. R. Braun, G. A. Kassavetis, E. P. Geiduschek. 1994.            Transcription: mechanisms regulation. Raven Press, Ltd., New York, <br />     Probing close DNA contacts RNA polymerase III transcription complexes               N.Y. <br />     photoactive nucleoside 4-thiodeoxythymidine. J. Biol. Chem. 269:          29. Kassavetis, G. A., B. Bartholomew, J. A. Blanco, T. E. Johnson, E. P. <br />     18090&#8211;18095.                                                                           Geiduschek. 1991. Two essential components Saccharomyces cerevisiae <br />  6. Bartholomew, B., G. A. Kassavetis, B. R. Braun, E. P. Geiduschek. 1990.            transcription factor TFIIIB: transcription DNA-binding properties. <br />     The subunit structure Saccharomyces cerevisiae transcription factor IIIC            Proc. Natl. Acad. Sci. USA 88:7308&#8211;7312. <br />     probed novel photocrosslinking reagent. EMBO J. 9:2197&#8211;2205.                30. Kassavetis, G. A., B. R. Braun, L. H. Nguyen, E. P. Geiduschek. 1990. S. <br />  7. Bartholomew, B., G. A. Kassavetis, E. P. Geiduschek. 1991. Two com-                cerevisiae TFIIIB transcription initiation factor proper RNA poly- <br />     ponents Saccharomyces cerevisiae transcription factor IIIB (TFIIIB)             merase III, TFIIIA TFIIIC assembly factors. Cell 60:235&#8211;245. <br />     stereospecifically located upstream tRNA gene interact           31. Kassavetis, G. A., C. A. P. Joazeiro, M. Pisano, E. P. Geiduschek, T. Colbert, <br />     second-largest subunit TFIIIC. Mol. Cell. Biol. 11:5181&#8211;5189.                       S. Hahn, J. A. Blanco. 1992. The role TATA-binding protein in <br />  8. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul-              assembly function multisubunit yeast RNA polymerase III <br />     lin. 1993. A simple efficient method direct gene deletion Saccha-           transcription factor, TFIIIB. Cell 71:1055&#8211;1064. <br />     romyces cerevisiae. Nucleic Acids Res. 21:3329&#8211;3330.                               32. Kassavetis, G. A., S. T. Nguyen, R. Kobayashi, A. Kumar, E. P. Geiduschek, <br />  9. Bonneaud, N., O. Ozier-Kalogeropoulos, G. Li, M. Labouesse, L. Minvielle-              M. Pisano. 1995. Cloning, expression, function TFC5, gene <br />     Sebastia, F. Lacroute. 1991. A <span id='am-61' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> low high copy replicative,            encoding B&#11033; component Saccharomyces cerevisiae RNA polymer- <br />     integrative single-stranded S. cerevisiae/E. coli shuttle <span id='am-145' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span>. Yeast           ase III transcription factor TFIIIB. Proc. Natl. Acad. Sci. USA 92:9786&#8211;9790. <br />     7:609&#8211;615.                                                                         33. Khoo, B., B. Brophy, S. P. Jackson. 1994. Conserved functional domains <br /> 10. Braun, B. R., B. Bartholomew, G. A. Kassavetis, E. P. Geiduschek. 1992.            RNA polymerase III general transcription factor BRF. Genes Dev. <br />     Topography transcription factor complexes Saccharomyces cerevi-              8:2879&#8211;2890. <br />     siae 5 S RNA gene. J. Mol. Biol. 228:1063&#8211;1077.                                    34. Lee, J. Y., C. F. Evans, D. R. Engelke. 1991. Expression RNase P RNA <br /> 11. Braun, B. R., G. A. Kassavetis, E. P. Geiduschek. 1992. Bending             Saccharomyces cerevisiae controlled unusual RNA polymerase III <br />     Saccharomyces cerevisiae 5S rRNA gene transcription factor complexes.               promoter. Proc. Natl. Acad. Sci. USA 88:6986&#8211;6990. <br />     J. Biol. Chem. 267:22562&#8211;22569.                                                    35. Lee, J. Y., C. E. Rohlman, L. A. Molony, D. R. Engelke. 1991. Charac- <br /> 12. Buratowski, S., H. Zhou. 1992. A suppressor TBP mutations encodes               terization RPR1, essential gene encoding RNA component of <br />     RNA polymerase III transcription factor homology TFIIB. Cell                Saccharomyces cerevisiae nuclear RNase P. Mol. Cell. Biol. 11:721&#8211;730. <br />     71:221&#8211;230.                                                                        36. Lefebvre, O., C. Carles, C. Conesa, R. N. Swanson, F. Bouet, M. Riva, and <br /> 13. Bussey, H., R. K. Storms, A. Ahmed, K. Albermann, E. Allen, W. Ansorge, R.             A. Sentenac. 1992. TFC3: Gene encoding B-block binding subunit the <br />  VOL. 19, 1999                                                                                &#65533;?&#65533;60 PARTICIPATES IN TFIIIB RECRUITMENT                                 8051 <br />  <br />  <br />     yeast transcription factor TFIIIC. Proc. Natl. Acad. Sci. USA 89:10512&#8211;          50. Roberts, S., S. J. Miller, W. S. Lane, S. Lee, S. Hahn. 1996. Cloning and <br />     10516.                                                                               functional characterization gene encoding TFIIIB90 subunit of <br /> 37. Lefebvre, O., J. Ru  &#168;th, A. Sentenac. 1994. A mutation largest           RNA polymerase III transcription factor TFIIIB. J. Biol. Chem. 271:14903&#8211; <br />     subunit yeast TFIIIC affects tRNA 5 S RNA synthesis. Identification           14909. <br />     classes suppressors. J. Biol. Chem. 269:23374&#8211;23381.                   51. Ru&#168;th, J., C. Conesa, G. Dieci, O. Lefebvre, A. Du  &#168;sterho&#168;ft, S. Ottonello, and <br /> 38. Le&#180;veillard, T., G. A. Kassavetis, E. P. Geiduschek. 1991. Saccharomyces         A. Sentenac. 1996. A suppressor mutations class III transcription <br />     cerevisiae transcription factors IIIB IIIC bend DNA tRNAGln             encodes component yeast TFIIIB. EMBO J. 15:1941&#8211;1949. <br />     gene. J. Biol. Chem. 266:5162&#8211;5168.                                              52. Schmitt, M. E., T. A. Brown, B. L. Trumpower. 1990. A rapid simple <br /> 39. Lo&#180;pez-De-Leo&#180;n, A., M. Librizzi, K. Puglia, I. Willis. 1992. PCF4 encodes       method preparation RNA Saccharomyces cerevisiae. Nucleic <br />     RNA polymerase III transcription factor homology TFIIB. Cell              Acids Res. 18:3091&#8211;3092. <br />     71:211&#8211;220.                                                                      53. Schultz, P., N. Marzouki, C. Marck, A. Ruet, P. Oudet, A. Sentenac. <br /> 40. Manaud, N., R. Arrebola, B. Buffin-Meyer, O. Lefebvre, H. Voss, M. Riva, C.          1989. The DNA-binding domains yeast transcription factor &#65533;?&#65533; as <br />     Conesa, A. Sentenac. 1998. A chimeric subunit TFIIIC forms                  observed scanning transmission electron microscopy. EMBO J. 8:3815&#8211; <br />     subcomplex &#65533;?&#65533;95. Mol. Cell. Biol. 18:3191&#8211;3200.                                  3824. <br /> 41. Mann, C., J. Y. Micouin, N. Chiannilkulchai, I. Treich, J.-M. Buhler, A.     54. Sethy, I., I. M. Willis. 1995. Recessive mutations second largest <br />     Sentenac. 1992. RPC53 encodes subunit Saccharomyces cerevisiae RNA              subunit TFIIIC suggest new step RNA polymerase III transcription. <br />     polymerase C (III) inactivation leads predominantly G1 arrest.            Gene Expr. 5:35&#8211;47. <br />     Mol. Cell. Biol. 12:4314&#8211;4326.                                                   55. Sethy-Coraci, I., R. D. Moir, A. Lo   &#180;pez-De-Leo&#180;n, I. M. Willis. 1998. A <br /> 41a.Marck, C. Personal communication.                                                    differential response wild type mutant promoters TFIIIB70 over- <br /> 42. Marck, C., O. Lefebvre, C. Carles, M. Riva, N. Chaussivert, A. Ruet, A. <br />                                                                                          expression vivo vitro. Nucleic Acids Res. 26:2344&#8211;52. <br />     Sentenac. 1993. The TFIIIB-assembling subunit yeast transcription factor <br />                                                                                      56. Simos, G., H. Tekotte, H. Grosjean, A. Segref, K. Sharma, D. Tollervey, and <br />     TFIIIC tetratricopeptide repeat basic-<span id='am-86' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-87' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span>-loop-helix motifs. <br />                                                                                          E. C. Hurt. 1996. Nuclear pore proteins involved biogenesis of <br />     Proc. Natl. Acad. Sci. USA 90:4027&#8211;4031. <br /> 43. Margottin, F., G. Dujardin, M. Ge&#180;rard, J. M. Egly, J. Huet, A. Sentenac.        functional tRNA. EMBO J. 15:2270&#8211;2284. <br />     1991. Participation TATA factor transcription yeast U6 gene     57. Stettler, S., N. Chiannilkulchai, S. Hermann-Le Denmat, D. Lalo, F. La- <br />     RNA polymerase C. Science 251:424&#8211;426.                                            croute, A. Sentenac, P. Thuriaux. 1993. A general suppressor RNA <br /> 44. Marzouki, N., S. Camier, A. Ruet, A. Moenne, A. Sentenac. 1986.                  polymerase I, II III mutations Saccharomyces cerevisiae. Mol. Gen. <br />     Selective proteolysis defines DNA binding domains yeast transcription         Genet. 239:169&#8211;176. <br />     factor &#65533;?&#65533;. Nature 323:176&#8211;178.                                                    58. Stolc, V., S. Altman. 1997. Rpp1, essential protein subunit nuclear <br /> 45. Moenne, A., S. Camier, G. Anderson, F. Margottin, J. Beggs, A. Sente-            RNase P required processing precursor tRNA 35S precursor <br />     nac. 1990. The U6 gene Saccharomyces cerevisiae transcribed RNA             rRNA Saccharomyces cerevisiae. Genes Dev. 11:2414&#8211;2425. (Corrected <br />     polymerase C (III) vivo vitro. EMBO J. 9:271&#8211;277.                          republished, 11:2926&#8211;2937.) <br /> 46. Pardee, T. S., C. S. Bangur, A. S. Ponticelli. 1998. The N-terminal region   59. Swanson, R. N., C. Conesa, O. Lefebvre, C. Carles, A. Ruet, E. Quemeneur, <br />     yeast TFIIB contains adjacent functional domains involved stable           J. Gagnon, A. Sentenac. 1991. Isolation TFC1, gene encoding one <br />     RNA polymerase II binding transcription start site selection. J. Biol.           DNA-binding subunits yeast transcription factor &#65533;?&#65533; (TFIIIC). Proc. <br />     Chem. 273:17859&#8211;64.                                                                  Natl. Acad. Sci. USA 88:4887&#8211;4891. <br /> 47. Parsons, M. C., P. A. Weil. 1990. Purification characterization       60. Whitehall, S. K., G. A. Kassavetis, E. P. Geiduschek. 1995. The sym- <br />     Saccharomyces cerevisiae transcription factor TFIIIC. Polypeptide composi-           metry yeast U6 RNA gene&#8217;s TATA box orientation the <br />     tion defined polyclonal antibodies. J. Biol. Chem. 265:5095&#8211;5103.               TATA-binding protein yeast TFIIIB. Genes Dev. 9:2974&#8211;2985. <br /> 48. Parsons, M. C., P. A. Weil. 1992. Cloning TFC1, Saccharomyces         61. Willis, I., P. Schmidt, D. So&#168;ll. 1989. A selection mutants RNA <br />     cerevisiae gene encoding 95-kDa subunit transcription factor                  polymerase III transcription apparatus: PCF1 stimulates transcription of <br />     TFIIIC. J. Biol. Chem. 267:2894&#8211;2901.                                                tRNA 5S RNA genes. EMBO J. 8:4281&#8211;4288. <br /> 49. Rameau, R., K. Puglia, A. Crowe, I. Sethy, I. M. Willis. 1994. A mutation    62. Wilson, I. A., H. L. Niman, R. A. <br /> </body></html>