<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 2001, p. 1453&#8211;1462                                                                           Vol. 75, No. 5 <br /> 0270-7306/01/$04.00&#11001;0 DOI: 10.1128/MCB.21.5.1453&#8211;1462.2001 <br /> Copyright &#169; 2001, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         The <span id='am-17' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> TIF6 Gene Encoding Translation <br />              Initiation Factor 6 Is Required 60S Ribosomal <br />                              Subunit Biogenesis <br />                   UTTIYA BASU,1 KAUSIK SI,1&#8224; JONATHAN R. WARNER,2                             AND   UMADAS MAITRA1* <br />                        Department Developmental Molecular Biology1 Department Cell Biology,2 <br />                                    Albert Einstein College Medicine, Bronx, New York 10461 <br />                      Received 6 October 2000/Returned modification 14 November 2000/Accepted 1 December 2000 <br />  <br />              Eukaryotic translation initiation factor 6 (eIF6), monomeric protein 26 kDa, bind 60S <br />           ribosomal subunit prevent association 40S ribosomal subunit. In <span id='am-34' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, eIF6 <br />           encoded single-copy essential gene. To understand function eIF6 yeast cells, constructed a <br />           conditional mutant haploid yeast strain functional rapidly degradable form eIF6 fusion <br />           protein synthesized repressible GAL10 promoter. Depletion eIF6 yeast cells resulted a <br />           selective reduction level 60S ribosomal subunits, causing stoichiometric imbalance 60S-to-40S <br />           subunit ratio inhibition rate vivo protein synthesis. Further analysis indicated eIF6 not <br />           required stability 60S ribosomal subunits. Rather, eIF6-depleted cells showed defective pre-rRNA <br />           processing, resulting accumulation 35S pre-rRNA precursor, formation 23S aberrant pre-rRNA, de- <br />           creased 20S pre-rRNA levels, accumulation 27SB pre-rRNA. The defect processing 27S pre- <br />           rRNA resulted reduced formation mature 25S 5.8S rRNAs relative 18S rRNA, ac- <br />           count selective deficit 60S ribosomal subunits cells. Cell fractionation indirect <br />           immunofluorescence studies showed c-Myc hemagglutinin epitope-tagged eIF6 distributed through- <br />           cytoplasm nuclei yeast cells. <br />  <br />  <br />    Eukaryotic translation initiation factor 6 (eIF6), mono-                observed depletion eIF6 yeast strain resulted <br /> meric protein 26 kDa, originally isolated                   inhibition cell growth rate vivo protein <br /> characterized wheat germ (22, 23) mammalian                    synthesis (29). However, analysis polysome profiles <br /> cell extracts (17, 34) based vitro assay measured              eIF6-depleted cells showed reduction the <br /> the ability protein bind specifically 60S ribo-             amounts polysomes amounts 80S <br /> somal subunit prevent association 40S ribo-              ribosomes free 60S ribosomal subunits accumula- <br /> somal subunit form 80S ribosomes. Because ribo-                   tion half-mer polysomes. Finally, analysis total ribo- <br /> somal subunit antiassociation property, eIF6 thought                  somal subunit content eIF6-depleted cells showed that <br /> play direct role provision free ribosomal subunits               selective reduction total 60S respect to <br /> required initiation protein synthesis. The protein                total 40S ribosomal subunits, causing stoichiometric im- <br /> therefore classified eukaryotic translation initiation factor           balance 60S/40S subunit ratio resulting <span id='am-84' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="forma" datatype="xsd:string"></span>forma</span>- <br /> (13), role translation mRNAs defined              tion half-mer polysomes. Similar observations also <br /> in original studies. More recently, facilitate              reported Sanvito et al. (26), identified eIF6 from <br /> characterization eIF6 understand function              mammalian cells &#65533;?&#65533;4 integrin-interacting protein p27 (3) <br /> protein translation, cloned expressed Esch-                designated yeast homologue p27BBP/eIF6 (26). <br /> erichia coli human cDNA (28) yeast Saccharo-                     The polysome-ribosome profiles observed eIF6-depleted <br /> myces cerevisiae gene (29) encoding functionally active eIF6,                cells characteristic cells containing inactive trans- <br /> each 245 amino acids (calculated Mr, 26,558 human eIF6                lation initiation factor. If eIF6 plays essential role the <br /> and 25,550 yeast eIF6). The proteins 72% identical.              initiation phase protein synthesis, depletion yeast cells <br /> Detailed characterization yeast gene encoding eIF6,                   caused reduction polysome content <br /> designated TIF6, showed TIF6 single-copy gene                 simultaneous increase (not decrease) pool of <br /> maps chromosome XVI (as YPR016C) essential                     free 40S, 60S, 80S ribosomes. These results our <br /> cell growth viability. These properties TIF6 used             observation lysates yeast cells lacking eIF6 remained <br /> construct conditional null allele placing expression                active translation mRNAs vitro (29) led conclude <br /> under control regulated GAL10 promoter (29). We                   eIF6 does function translation initiation factor for <br />                                                                              global protein synthesis. Rather, inhibition translation <br />   * Corresponding author. Mailing address: Department Develop-            observed eIF6-depleted cells selective reduction of <br /> mental Molecular Biology, Albert Einstein College Medicine,           60S ribosomal subunits cells. Thus, eIF6 a <br /> 1300 Morris Park Ave., Bronx, NY 10461. Phone: (718) 430-3505. Fax:          translation initiation factor, essential cellular function <br /> (718) 430-8567. E-mail: maitra@aecom.yu.edu. <br />   &#8224; Present address: Howard Hughes Medical Institute Center for <br />                                                                              does protein perform? More specifically, does eIF6 <br /> Neurobiology, Columbia University College Physicians Sur-             maintain steady-state level 60S ribosomal subunits in <br /> geons, New York, NY 10032.                                                   yeast cells? <br />  <br />                                                                       1453 <br />  1454       BASU ET AL.                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />                       TABLE 1. Yeast strains used                                   eIF6 control natural promoter present inserted <br />                                                                                     1.752-kb fragment. <span id='am-51' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> plasmid used transform diploid strain <br />                                                                       Reference     KSY601 (MATa/MAT&#65533;?&#65533; tif6::HIS3/TIF6), transformants sporulated at <br />   Strain                          Genotype <br />                                                                       source     30&#176;C; resulting tetrads dissected, spores germinated YPD <br /> W303a/&#65533;?&#65533;      MATa/MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1           21             plates followed selection SD&#11002;His&#11002;Ura plates. Only spores containing the <br />               ura3-1 can1-100                                                       tif6::HIS3 gene harboring plasmid pRS316-TIF6 germinate yield His&#11001; <br /> W303&#65533;?&#65533;        MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1 ura3-1         21             Ura&#11001; colonies. This haploid yeast strain named KSY606 (Table 1). Yeast <br />               can1-100                                                              strains grown 30&#176;C rich YPD medium (1% [wt/vol] yeast extract, <br /> KSY601       MATa/MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1           29             2% [wt/vol] Bacto Peptone, 2% [wt/vol] dextrose) YPGal medium, where <br />               ura3-1 can1-100 tif6::HIS3/TIF6                                       2% (wt/vol) galactose replaced dextrose carbon source. Where indicated, <br /> KSY603       MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1 ura3-1         29             haploid yeast cells grown synthetic complete medium (0.67% Bacto <br />               can1-100 tif6::HIS3 p[URA3 GAL10::Ub-TIF6]                            Yeast Nitrogen Base amino acids, 0.2% amino acid mixture supple- <br /> KSY605       MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1 ura3-1         This work      mented nutrients required auxotrophic deficiencies) containing either <br />               can1-100 tif6::HIS3 p[TRP1 TIF6-myc]                                  2% galactose (SGal) 2% dextrose (SD) carbon source. For vivo <br /> KSY606       MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1 ura3-1         This work      [35S]methionine incorporation, 0.2% amino acid mixture SGal <br />               can1-100 tif6::HIS3 p[URA3 TIF6]                                      SD medium did contain methionine. The methionine-lacking media <br /> KSY607       MAT&#65533;?&#65533; leu2-3,112 his3-11,15 ade2-1 trp1-1 ura3-1         This work <br />                                                                                     designated SGal&#11002;Met SD&#11002;Met, respectively. Yeast genetic methods <br />               can1-100 tif6::HIS3 p[LEU2 TIF6-HA] <br />                                                                                     performed following standard protocols (19). <br />                                                                                        Construction haploid yeast strain expression C-terminally Myc- <br />                                                                                     tagged hemagglutinin (HA)-tagged eIF6 fusion protein. To express C- <br />                                                                                     terminally Myc-tagged eIF6 fusion protein cognate promoter yeast <br />    In recent years, number yeast genes mutations                         cells, primers designed: 5&#11032;-end primer Yf6G5 (5&#11032;CGGGATCCAAGG <br /> cause selective reduction 60S subunits                        TGCAAGATCAGAC3&#11032;), complementary sequence 500 nucleo- <br /> respect 40S subunits concomitant accumulation half-                       tides upstream TIF6 ORF (BamHI site underlined); 3&#11032;-end primer <br /> mer polysomes identified. These genes encode sev-                         Yf6Myc (5&#11032;GGAATTCATGCATCTAGAGGTCTTCTTCGGAAATCAACTTC <br />                                                                                     TGTTCTGAGTAGGTTTCAA3&#11032;), containing Myc tag (in italics; EcoRI <br /> eral ribosomal proteins (5, 7, 14, 15, 20) nonribosomal <br />                                                                                     site underlined). A 1.2-kb S. cerevisiae genomic fragment containing entire <br /> proteins like Nip7p (39), Dbp6p (11), Nmd3p (9), Sqt1p                          TIF6 gene amplified PCR using <span id='am-6' about='obo:NCBITaxon_2260' typeof='owl:Thing obo:NCBITaxon_183968 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2259 obo:NCBITaxon_2258 obo:NCBITaxon_28890'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Pyrococcus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Pyrococcus</span> DNA polymerase the <br /> (6). Removal proteins yeast cells shown                      end primers. The PCR product digested BamHI EcoRI <br /> to impair processing rRNA precursors (pre-rRNAs)                          cloned sites TRP1-based yeast CEN plasmid pRS314 <br /> mature 25S 5.8S rRNAs, constituents                        (30) yield pRS314-TIF6-Myc. This plasmid used transform the <br />                                                                                     haploid yeast strain KSY603 (MAT&#65533;?&#65533; tif6::HIS3 p[URA3 GAL10::Ub-TIF6]), and <br /> 60S ribosomal subunit. Failure produce mature 25S 5.8S                       Trp&#11001; Ura&#11001; transformants selected SGal plates. The transformants were <br /> rRNAs account inhibition synthesis 60S                            replica plated SGal plate contained uracil 5-fluo- <br /> ribosomal subunits. In light observations, possibil-                   roorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>) promote loss URA3 plasmid. The yeast strain <br /> ity exists eIF6 itself involved processing                    recovered 5-FOA plate, designated KSY605, disrupted chro- <br />                                                                                     mosomal copy TIF6 harbored pRS314-TIF6-Myc complementing <br /> pre-rRNAs mature 25S 5.8S rRNAs consequently <br />                                                                                     plasmid. These cells expressed, natural promoter, functional eIF6 with <br /> in biosynthesis 60S ribosomal subunits.                                      Myc tag C-terminal end protein, approximately wild-type levels. <br />    In report, functional analysis                            To express C-terminally HA-tagged eIF6 yeast cells, 3&#11032;-end primer yf63GF <br /> S. cerevisiae TIF6 gene. We product eIF6 local-                       (5&#11032;CGAATTCGCGGCCGCCGCCTGAGTAGGTTTCAATCAA3&#11032;) (NotI site <br /> izes nucleus cytoplasm. Depletion                         underlined) 5&#11032;-end primer yf6G5 (see above) used amplify yeast <br />                                                                                     genomic DNA PCR, PCR product digested HindIII and <br /> eIF6 yeast cells, results selective reduction                    NotI. The resulting HindIII/NotI fragment used three-fragment ligation <br /> 60S ribosomal subunits, does affect stability preex-                     NotI/BamHI fragment derived plasmid pGEXYP1 LEU2- <br /> isting 60S ribosomal particles. Rather, vivo depletion eIF6                   based CEN plasmid pRS315 digested HindIII BamHI yield <br /> causes impaired processing 35S 27S pre-rRNA precur-                          new recombinant plasmid pRS315-TIF6-HA. This plasmid used to <br />                                                                                     transform KSY606 (MAT&#65533;?&#65533; tif6::HIS3 p[URA3-TIF6]) cells, Leu&#11001; Ura&#11001; <br /> sors, resulting decreased formation mature 25S 5.8S <br />                                                                                     transformants selected appropriate SD plates. The transformants were <br /> rRNAs constituents 60S ribosomal subunits.                          replica plated SD&#11002;Leu plates containing uracil 5-FOA pro- <br /> These observations consistent role eIF6 60S                       mote loss URA3 plasmid. The yeast strain recovered 5-FOA <br /> ribosomal subunit biogenesis.                                                       plate, designated KSY607 (Table 1), disrupted chromosomal copy of <br />                                                                                     TIF6 harbored pRS315-TIF6-HA complementing plasmid. These cells <br />                                                                                     expressed functional HA-tagged eIF6 natural promoter approximately <br />                         MATERIALS AND METHODS <br />                                                                                     wild-type levels. <br />    Strains, media, genetic methods. The S. cerevisiae haploid strains used      Subcellular fractionation yeast cells Western blotting. Exponentially <br /> this work derived diploid strain W303a/&#65533;?&#65533;. The genotypes       growing KSY605 yeast cells expressing c-Myc-tagged eIF6 fractionated into <br /> strain yeast strains used study listed Table 1. Strain     nuclear cytosolic fractions adaptation procedure Aris and <br /> KSY603 (MAT&#65533;?&#65533;tif6::HIS3 p[URA3 GAL10::Ub-TIF6]), constructed described            Blobel (1). Proteins subcellular fractions precipitated 10% <br /> previously (29), contains conditional eIF6 expression N-ter-      trichloroacetic acid analyzed sodium dodecyl sulfate (SDS)-polyacryl- <br /> minally fused ubiquitinated eIF6 fusion protein expressed transcrip-     amide gel electrophoresis followed immunoblotting using anti-c-Myc antibod- <br /> tion unit consisting ubiquitin gene cassette (Ub-R-lacI-TIF6, R   ies probes detect c-Myc-tagged-eIF6. Rabbit anti-Nop1p (a kind gift from <br /> <span id='am-61' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-65'  ></span>codon</span> arginine) (16) acts protein-destabilizing genetic element      T. Meier institution) anti-yeast eIF5 (4) antibodies used to <br /> fused NH2 terminus open reading frame (ORF) TIF6 gene          detect nuclear cytoplasmic protein markers, respectively. <br /> under transcriptional control GAL10 promoter. The haploid yeast             Indirect immunofluorescence. Cells (&#11011;3 A600 units) exponentially <br /> strain KSY606, containing chromosomal copy TIF6 gene inactivated      growing culture strain KSY607 (tif6::HIS3 p[LEU2 TIF6-HA]) YPD me- <br /> insertion HIS3 marker gene harboring centromeric (CEN) expression        dium harvested centrifugation. The harvested cells prepared for <br /> plasmid expresses yeast eIF6 natural promoter, constructed    immunofluorescence described Wang Chang (37). DAPI (4&#11032;,6-di- <br /> as follows. A 1.752-kb yeast genomic fragment containing entire TIF6 gene       amidino-2-phenylindole dihydrochloride; Fluka) used stain chromosomal <br /> was amplified yeast genomic DNA PCR using 5&#11032;-end primer, Yf6G5            DNA. To detect eIF6-HA permeabilized spheroplasts, cells incubated <br /> (29), 3&#11032;-end primer, Yf6G3 (29). The PCR product digested            overnight monoclonal mouse anti-HA (BAbCo, Berkeley, Calif.) antibody <br /> BamHI EcoRI cloned sites CEN plasmid pRS316                primary dilution 1:1,000, goat anti-mouse Cy3-conjugated antibody <br /> (URA3 based) (30) yield plasmid pRS316-TIF6. In plasmid, expression     (Jackson ImmunoResearch Laboratories) used secondary antibody at <br />  VOL. 75, 2001                                                            eIF6 IS INVOLVED IN 60S RIBOSOMAL SUBUNIT BIOGENESIS                                     1455 <br />  <br />  <br /> a dilution 1:1,000. Fluorescence-labeled cells examined Olympus          TTCATC3&#11032; (probe 6), 5&#11032;GGCCAGCAATTTCAAGTTA3&#11032; (probe 7), 5&#11032;C <br /> high-resolution IX70 microscope attached cooled charge-coupled device             TCCGCTTATTGATATGC3&#11032; (probe 9). Approximately 100 ng oligo- <br /> <span id='am-56' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60'  ></span>camera</span> Analytical Imaging Facility institution. The figures        nucleotide probe end labeled 32P using 50 &#65533;?&#65533;Ci [&#65533;?&#65533;-32P]ATP (6,000 <br /> arranged using Adobe Photoshop.                                                        Ci/mmol) T4 polynucleotide kinase. Exponentially growing cultures of <br />    Pulse-chase labeling ribosomal subunits analysis stability. The     W303&#65533;?&#65533;[TIF6] KSY603[GAL10::Ub-TIF6] transferred YPD medium. <br /> haploid yeast strains W303&#65533;?&#65533;[TIF6] KSY603[GAL10::Ub-TIF6]                 At various times following shift YPD, cells (10 A600 units) withdrawn <br /> grown 60 ml SGal&#11002;Met medium early logarithmic phase. Each culture             total RNA isolated. Approximately 5 &#65533;?&#65533;g RNA resolved 1.2% <br /> was treated 200 &#65533;?&#65533;Ci [35S]methionine shaken 30 min 30&#176;C         agarose&#8211;6% formaldehyde gels 7% polyacrylamide&#8211;8 M urea gels, transferred <br /> to <span id='am-83' about='rdfs:label' typeof='owl:Thing'>label</span> ribosomal proteins. The cells culture quickly har-         Hybond-N&#11001; nylon membranes (Amersham) described Sambrook et al. <br /> vested, resuspended 60 ml prewarmed SD-Met medium containing unla-               (25), cross-linked paper Stratagene UV cross-linker accord- <br /> beled methionine (500 &#65533;?&#65533;g/ml), grown 30&#176;C. At indicated time, 15 ml          ing manufacturer&#8217;s specifications. The membrane blots prehybridized <br /> of culture harvested, cells washed cold water               buffer containing 1% bovine serum albumin, 1 mM EDTA, 0.5 M NaPO4 <br /> quick-frozen. Labeled ribosomal subunits isolated frozen cells            (pH 7.2), 7% SDS 65&#176;C 1 h hybridized 12 h same <br /> follows. Cells thawed washed 1 ml cold buffer R (20 mM                buffer containing approximately 106 cpm 32P-labeled oligonucleotide probes. <br /> Tris-HCl [pH 7.5], 10 mM MgCl2, 50 mM KCl, 1 mM dithiothreitol, 0.2 mM                 The membrane blots washed times 2&#11003; SSC (1&#11003; SSC 0.15 M <br /> EDTA, 1 mM phenylmethylsulfonyl fluoride, 0.1% Triton X-100). The washed               NaCl 0.015 M sodium citrate) containing 0.5% SDS followed 0.1&#11003; <br /> cells resuspended 200 &#65533;?&#65533;l buffer R lysed vortexing       SSC&#8211;0.5% SDS room temperature. The dried blots analyzed auto- <br /> equal volume glass beads. The cell lysates clarified centrifugation   radiography. <br /> at 12,000 &#11003; g 15 min 4&#176;C, followed recentrifugation supernatant          Other methods. The procedure used preparation yeast cell lysates for <br /> at 17,000 &#11003; g 10 min. The clear supernatant layered 1.6 ml        immunoblot analysis adapted described Sachs Davis (24). <br /> of 10% sucrose buffer R centrifuged Beckman TL100.3 rotor 55,000        The rate protein synthesis measured pulse-labeling 1.0 A600 unit of <br /> rpm 2.5 h. Each ribosomal pellet suspended 100 &#65533;?&#65533;l ice-cold buffer        yeast cells [35S]methionine 5 min measuring incorporation of <br /> R gentle shaking 2 h treated 2 mM puromycin 30&#176;C           radioactivity total cellular proteins described previously (29). <br /> 15 min. The suspension treated 100 &#65533;?&#65533;l 2&#11003; buffer S (1&#11003; buffer <br /> S 50 mM Tris-HCl [pH 7.5], 800 mM KCl, 10 mM MgCl2, 10 mM dithiothreitol <br /> and 1 mM phenylmethylsulfonyl fluoride) rotated slowly 4&#176;C                                             RESULTS <br /> 2 h dissociate ribosomes 40S 60S ribosomal subunits. Equivalent <br /> amounts A254-absorbing materials (approximately 4.0 A254 units 200 &#65533;?&#65533;l)              Kinetic analysis eIF6 disappearance, 60S ribosomal sub- <br /> were layered 11 ml 15 35% (wt/vol) linear sucrose gradients buffer S       unit reduction, inhibition protein synthesis yeast <br /> and centrifuged 40,000 rpm 8 h 4&#176;C Beckman SW41 rotor. The              cells. We previously constructed used (29) hap- <br /> gradients fractionated ISCO density gradient fractionator, the <br /> absorbance profile 254 nm analyzed ISCO UA-5 absorbance mon- <br />                                                                                        loid yeast strain KSY603, chromosomal copy of <br /> itor. An aliquot (100 &#65533;?&#65533;l) fraction counted Aquasol 35S              TIF6 gene inactivated insertion HIS3 marker <br /> radioactivity liquid scintillation spectrometer. The stability labeled     gene essential eIF6 function provided mainte- <br /> ribosomal subunits synthesized growth cells SGal medium prior          nance CEN plasmid harboring conditional eIF6 expres- <br /> transfer SD medium calculated total counts minute 35S <br />                                                                                        sion system. In expression system, eIF6 expressed from <br /> radioactivity recovered 40S 60S ribosomal subunit fractions. <br />    Pulse-chase labeling pre-rRNA analysis gel electrophoresis. rRNAs         transcription unit containing protein-destabilizing ubiquitin <br /> were labeled [methyl-3H]methionine adaptation procedure              gene cassette consisting GAL-UAS-UBI4-R-lacI-HA (where <br /> of Kressler et al. (11). Briefly, haploid strains W303&#65533;?&#65533;[TIF6] KSY603           R codon arginine) (16) fused NH2 terminus of <br /> [GAL10::Ub-TIF6] grown SGal&#11002;Met medium A600 0.5.                TIF6 ORF transcriptional control the <br /> Cells harvested centrifugation resuspended SD-Met medium <br /> prewarmed 30&#176;C A600 0.2. After 1.5 h growth at <br />                                                                                        GAL10 promoter. When exponentially growing culture of <br /> 30&#176;C eIF6 depletion KSY603 cells, cells (&#11011;40 A600 units) <span id='am-66' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-              KSY603 SGal medium transferred SD medium to <br /> pended 750 &#65533;?&#65533;l SD&#11002;Met medium, prewarmed 30&#176;C labeled                  repress synthesis eIF6, preexisting eIF6 protein was <br /> 250 &#65533;?&#65533;Ci [methyl-3H]methionine (70 85 Ci/mmol; Amersham) 2 min                rapidly removed yeast cells (29). Analysis various pa- <br /> at 30&#176;C. The chase initiated adding 7 ml SD medium containing <br />                                                                                        rameters protein synthesis eIF6-depleted cells showed <br /> unlabeled methionine (500 &#65533;?&#65533;g/ml). At 0, 1.5, 3.0, 6.0 min, 2-ml samples <br /> were withdrawn centrifuged 10 s, cells quickly frozen            selective reduction level 60S ribo- <br /> dry-ice&#8211;ethanol bath used RNA isolation. For labeling RNA               somal subunits cells, resulting inhibition protein <br /> [3H]uracil, exponentially growing yeast cells SGal medium harvested,           synthesis eventually cell growth (29). <br /> resuspended SD medium supplemented uracil (20 &#65533;?&#65533;g/ml), grown                   In work, carried detailed kinetic <br /> for 1.5 h 30&#176;C. Cells harvested centrifugation, resuspended fresh <br />                                                                                        analysis decline level 60S ribosomal subunits in <br /> SD medium lacking uracil, pulse-labeled 3 min 30&#176;C 100 &#65533;?&#65533;Ci <br /> of [5, 6-3H]uracil. The chase initiated addition unlabeled uracil        eIF6-depleted cells. For purpose, transferred exponen- <br /> to final concentration 300 &#65533;?&#65533;g/ml. At various times, samples taken            tially growing cultures wild-type W303&#65533;?&#65533; KSY603 from <br /> and quickly frozen dry-ice&#8211;ethanol bath. RNA isolated               galactose- glucose-containing medium monitored <br /> [methyl-3H]methionine- [3H]uracil-labeled cells hot-phenol                  rates disappearance eIF6 level 60S ribosomal <br /> method described (25). <br />    The 3H-labeled small RNAs analyzed electrophoresis 7% poly- <br />                                                                                        subunits measuring (i) levels eIF6 cell lysates by <br /> acrylamide gels containing 8 M urea, transferred nylon membrane, incu-            immunoblot analysis, (ii) relative amounts total 40S and <br /> bated En3Hance (Dupont-NEN), subjected autoradiography. Large                60S ribosomal subunits, (iii) relative rates protein <br /> RNA molecules resolved 1.2% agarose&#8211;6% formaldehyde gels                   synthesis KSY603 cell lysates (Fig. 1). As shown Fig. 1A, <br /> transferred Hybond-N&#11001; nylon membranes (Amersham); membranes were <br />                                                                                        rapid decline eIF6 fusion protein KSY603 <br /> baked 2 h 80&#176;C, sprayed En3Hance (Dupont-NEN), dried, and <br /> subjected autoradiography &#11002;70&#176;C intensifying screen.                     cells transfer glucose medium. Within hour, the <br />    Northern analysis. Steady-state levels pre-rRNA mature rRNA             eIF6 fusion protein barely detectable cell lysates. <br /> determined Northern blot analysis using 32P-labeled deoxyoligonucleotide            In contrast, level eIF6 KSY603 cell lysates continuing <br /> probes (numbered 1 9 Fig. 3). The sequences probes,           grow SGal medium remained fairly constant 6-h <br /> complementary various regions 35S pre-rRNA, 5&#11032;GGTCTCTCTGC <br /> TGCCGG3&#11032; (probe 1), 5&#11032;CATGGCTTAATCTTTGAGAC3&#11032; (probe 2), 5&#11032;CG <br />                                                                                        period (only <span id='am-87' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-91'  ></span>data</span> 1.5-h time point shown for <br /> GTTTTAATTGTCCTA3&#11032; (probe 3), 5&#11032;TGTTACCTCTGGGCCC3&#11032; (probe 4),                           comparison). However, significant reduction molar ratio <br /> 5&#11032;AATTTCCAGTTACGAAAATTCTTG3&#11032; (probe 5), 5&#11032;TTTCGCTGCGTTC                                60S 40S ribosomal subunits occurred near- <br />  1456      BASU ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                                                                            noted eIF6 undetectable about <br />                                                                            1.5 h following transfer KSY603 cells galactose- to <br />                                                                            glucose-containing medium, cell growth rate did begin <br />                                                                            decrease 3 h. (There 10% <br />                                                                            decrease growth rate 4 h.) From time point, was <br />                                                                            progressive increase doubling time 15 h, <br />                                                                            cell growth ceased (data shown) (29). <br />                                                                               eIF6 required stability 60S ribosomal sub- <br />                                                                            units cytoplasm. The selective reduction steady- <br />                                                                            state level 60S ribosomal subunits removal eIF6 <br />                                                                            yeast cells suggested eIF6 required for <br />                                                                            stability 60S particles cytoplasm bio- <br />                                                                            genesis 60S ribosomal subunits. <br />                                                                               The effect eIF6 depletion stability 60S ribosomal <br />                                                                            subunits yeast cells investigated. An exponentially <br />                                                                            growing culture KSY603[GAL10::Ub-TIF6] the <br />                                                                            wild-type W303&#65533;?&#65533;[TIF6] cells labeled [35S]methi- <br />                                                                            onine 30 min. Cells transferred glucose-con- <br />                                                                            taining medium containing excess unlabeled methionine, <br />                                                                            stability [35S]methionine-labeled 60S ribosomal sub- <br />                                                                            units synthesized growth galactose medium mon- <br />                                                                            itored growth period glucose medium rapid <br />                                                                            eIF6 depletion occurred. We isolated 40S 60S ribo- <br />                                                                            somal subunits cell lysate sucrose gradient cen- <br />                                                                            trifugation determined total 35S radioactivity each <br />    FIG. 1. Kinetics disappearance 60S ribosomal subunits         ribosomal subunit pool. If eIF6 required stability of <br /> inhibition protein synthesis eIF6-depleted cells. (A) Exponen- <br /> tially growing cultures KSY603[GAL10::Ub-HA-TIF6] cells              60S ribosomal subunits, removal eIF6 yeast cells <br /> SGal&#11002;Met medium transferred SD&#11002;Met medium allowed              result progressive decline 35S <br /> to grow. At indicated times, cell lysates prepared 1 A600    radioactivity 60S ribosomal subunit pool. Figure 2B <br /> unit cells, approximately 100 &#65533;?&#65533;g protein cell lysate   shows wild-type W303&#65533;?&#65533;[TIF6] KSY603 cells, <br /> was analyzed Western blotting using monoclonal anti-HA antibod- <br /> ies probes. (B) Total ribosomes isolated KSY603 (&#18567;) and <br />                                                                            35S radioactivity incorporated 60S 40S <br /> control wild-type (&#9632;) cells indicated times following shift   subunits growth SGal medium recovered follow- <br /> culture SGal-Met SD-Met medium. Ribosomes dissoci-            ing growth SD medium. These results suggest eIF6 <br /> ated subunits sedimented 15 40% sucrose gradi-         depletion did significantly reduce stability preexist- <br /> ents described Materials Methods reference 29. (C)        ing 60S ribosomal subunits. However, observed the <br /> Rates protein synthesis. At indicated times following shift of <br /> cells SGal-Met SD-Met medium, 1 A600 unit cells       specific radioactivity 60S subunits decreased wild-type <br /> culture harvested suspended 300 &#65533;?&#65533;l SD-Met medium             W303&#65533;?&#65533;[TIF6] cell lysates exponentially time during <br /> containing 0.5 &#65533;?&#65533;Ci [35S]methionine (1,175 Ci/mmol), cells      growth SD medium. This expected synthe- <br /> shaken 5 min 30&#176;C. The pulse terminated             sis new unlabeled ribosomal subunits growth <br /> addition stop buffer containing unlabeled methionine (1.8 mg/ml) <br /> and cycloheximide (50 &#65533;?&#65533;g/ml). The rate protein synthesis each <br />                                                                            period SD medium. The specific radioactivities 60S sub- <br /> time point calculated described previously (29). The data pre-      units decreased nearly 50 25% growth W303&#65533;?&#65533; in <br /> sented panels B C averages independent experi-           SD medium 2 4 h, respectively. These results correlate <br /> ments.                                                                     doubling time 2 h W303&#65533;?&#65533; cells SD <br />                                                                            medium. In contrast, specific radioactivity 60S ribosomal <br />                                                                            subunits KSY603 cells did decrease significantly during <br /> complete disappearance eIF6 yeast cells, re-              growth SD medium eIF6 depletion (Fig. 2A). The <br /> duction occurred gradually period hours (Fig.            small (approximately 15%) decrease specific radioactivity of <br /> 1B). The kinetics inhibition rate protein synthesis           60S subunits KSY603 cells 2 h following transfer yeast <br /> was similar observed reduction 60S/40S              cells SGal SD medium explained obser- <br /> molar ratio (Fig. 1C), indicating inhibition protein               vation eIF6 does disappear completely immediately <br /> synthesis eIF6-depleted cells probably deficit            following transfer KSY603 cells galactose- glu- <br /> 60S ribosomal subunits cells. As expected, control             cose-containing medium. Rather, progressive de- <br /> wild-type W303&#65533;?&#65533;[TIF6] cells grown SD medium KSY603                   cline eIF6 levels hour growth SD <br /> cells grown galactose-containing medium, molar                 medium (Fig. 1A). Presumably, period new 60S <br /> ratio 60S 40S subunits rate protein                subunits synthesized reduced level KSY603 cells, <br /> synthesis remained unchanged (in Fig. 1, data W303&#65533;?&#65533;               decreasing slightly specific radioactivity 60S sub- <br /> are shown). These results clearly indicate reduction           units. At rate, removal eIF6 yeast cells did <br /> steady-state levels 60S ribosomal subunits begins            significantly reduce stability preexisting 60S ribo- <br /> near-complete disappearance eIF6 yeast cells,             somal subunits (Fig. 2B), result suggests eIF6 deple- <br /> reduction occurs gradually period hours. It              tion inhibits synthesis new 60S ribosomal subunits. Oth- <br />  VOL. 75, 2001                                                  eIF6 IS INVOLVED IN 60S RIBOSOMAL SUBUNIT BIOGENESIS                        1457 <br />  <br />  <br />                                                                            undergoes sequence ordered endo- exonucleo- <br />                                                                            lytic cleavage trimming reactions requiring participa- <br />                                                                            tion small nucleolar RNAs trans-acting protein <br />                                                                            factors generate mature 18S, 25S, 5.8S rRNAs (Fig. 3) <br />                                                                            (for reviews, references 12, 32, 33, 35). <br />                                                                               Decreases levels ribosomal subunits muta- <br />                                                                            tions depletion nonribosomal protein usually the <br />                                                                            result defects pre-rRNA processing and/or assembly of <br />                                                                            ribosomal subunits. Mutation depletion number of <br />                                                                            yeast proteins causes defect pre-rRNA processing (see <br />                                                                            references 12 35 reviews). In cases, stability <br />                                                                            mature rRNAs affected (9). Presumably, pro- <br />                                                                            teins associate pre-rRNA steps the <br />                                                                            processing reactions. <br />                                                                               To study role eIF6 60S subunit biogenesis, we <br />                                                                            examined effects eIF6 depletion synthesis and <br />                                                                            processing pre-rRNA measured [methyl-3H]methi- <br />                                                                            onine pulse-chase labeling experiments. Since 25S 18S <br />                                                                            rRNAs highly specifically methylated process- <br />                                                                            ing, processing pre-rRNA easily monitored labeling <br />    FIG. 2. Synthesis stability 60S ribosomes affected by <br /> eIF6 depletion. Exponentially growing cultures W303&#65533;?&#65533;[TIF6] or <br /> KSY603[GAL10::Ub-TIF6] SGal&#11002;Met medium (approximately 50 <br /> A600 units cells) labeled 200 &#65533;?&#65533;Ci [35S]methionine 30 <br /> min 30&#176;C. Cells transferred 100 ml SD-Met medium <br /> containing unlabeled methionine (500 &#65533;?&#65533;g/ml). At indicated times, <br /> cells (10 A600 units) harvested, washed, lysed described in <br /> Materials Methods. Each lysate subjected 15 40% (wt/ <br /> vol) sucrose gradient centrifugation separate ribosomal subunits <br /> and fractionated ISCO density gradient fractionator attached to <br /> a UA-5 absorbance monitor. The amounts 60S subunits quan- <br /> tified absorbance 254 nm, 35S radioactivity content of <br /> each fraction determined counting Aquasol liquid scin- <br /> tillation spectrometer. The total 35S radioactivity present in <br /> either 40S 60S ribosomal subunit pool condition <br /> following transfer SD medium calculated. These values are <br /> shown panel B percentage total counts minute origi- <br /> nally incorporated 60S 40S ribosomal subunits prior transfer <br /> to SD medium. The specific radioactivity 60S ribosomal subunits <br /> during growth SD medium calculated counts minute of <br /> 35 <br />    S radioactivity 1 A254 unit 60S ribosomal subunits. <br />  <br />  <br /> erwise, specific radioactivity 60S ribosomes have <br /> decreased exponentially observed wild-type W303 <br /> cells (Fig. 2A). It noted signif- <br /> icant reduction growth rate KSY603 cells SD me- <br /> dium 2 h growth. After 4 h, only <br /> about 10% reduction growth rate (data shown). <br />    eIF6 required pre-rRNA processing consequently <br /> for biogenesis 60S ribosomes. Experiments presented <br /> above clearly suggested eIF6 depletion affected bio- <br /> genesis 60S ribosomal subunits. <br />    Ribosome biogenesis eukaryotic cells complex process <br /> that occurs nucleolus, rRNAs                     FIG. 3. Schematic diagram pre-rRNA processing yeast <br /> formed, modified, processed assembly 78 (in                S. cerevisiae. (A) Organization processing sites 35S pre-rRNA. <br /> yeast) 79 (in mammals) ribosomal proteins mature                  The 35S precursor contains sequences mature 18S, 25S, and <br />                                                                            5.8S rRNAs, separated internal transcribed spacers <br /> 40S 60S ribosomal subunits. In yeast S. cerevisiae           ITS1 ITS2 external transcribed spacers, 5&#11032;ETS and <br /> the process best characterized biochemically             3&#11032;ETS, 5&#11032; 3&#11032; ends, respectively. The oligonucleotides cor- <br /> genetically, 18S rRNA 40S subunits 25S                  responding different regions 35S rRNA used probes <br /> 5.8S rRNAs 60S subunits transcribed RNA polymer-                 Northern analysis indicated numbers 1 9. (B) Pre- <br />                                                                            rRNA processing steps different processing intermediates. The <br /> ase I single 35S pre-rRNA, fourth rRNA, 5S,               major cleavage sites, endonucleolytic exonucleolytic processing <br /> a constituent 60S subunit, transcribed independently             steps, processing intermediates, pathways leading mature rRNA <br /> from 5S DNA RNA polymerase III. The 35S pre-rRNA                        indicated (adapted reference 11). <br />  1458     BASU ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                                                                          appearance weakly labeled unusual 23S <span id='am-71' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> was <br />                                                                          observed wild-type cells. Taken together, results <br />                                                                          suggest primary defect eIF6 depletion the <br />                                                                          formation 60S-specific 25S rRNA 27S pre-rRNA. The <br />                                                                          processing 35S pre-RNA slightly affected, resulting <br />                                                                          reduced formation precursor mature rRNAs. <br />                                                                             To rule possibility altered defective methyl- <br />                                                                          ation responsible impaired formation 25S rRNA, as <br />                                                                          monitor processing formation low-molec- <br />                                                                          ular-weight rRNAs, similar experiment carried with <br />                                                                          [3H]uracil. A pulse-label [5,6-3H]uracil 3 min was <br />                                                                          chased 5, 20, 60 min excess unlabeled uracil. <br />                                                                          (It noted shorter periods labeling and <br />                                                                          chase time used experiments [3H]methionine than <br />                                                                          [3H]uracil. This endogenous pool <br />                                                                          S-adenosylmethionine known rapidly saturated and <br />                                                                          chased [36]). In contrast, [3H]uracil utilized slow- <br />                                                                          er kinetics.) Analysis total [3H]RNA samples agarose- <br />                                                                          formaldehyde gel electrophoresis conditions similar to <br />                                                                          used Fig. 4A yielded results similar obtained <br />                                                                          [methyl-3H]labeled RNA (data shown). More impor- <br />                                                                          tant, analysis low-molecular-weight (5.8S 5S) rRNAs <br />    FIG. 4. Depletion eIF6 leads reduced synthesis 25S 5.8S     [3H]uracil labeling experiments polyacrylamide- <br /> rRNAs. (A) Exponentially growing cultures wild-type strain W303&#65533;?&#65533;      urea gel electrophoresis showed 5.8S rRNA was <br /> [TIF6] strain KSY603[GAL10::Ub-TIF6] SGal&#11002;Met medium              formed eIF6-depleted cells (Fig. 4B). The lack formation <br /> were shifted SD&#11002;Met medium grown 90 min remove             5.8S rRNA consistent lack formation 25S rRNA <br /> eIF6 KSY603 cells. Cells (40 A600 units) pulse-labeled with <br /> 250 &#65533;?&#65533;Ci [methyl-3H]methionine 2 min 30&#176;C chased       mature rRNAs formed processing 27S <br /> with excess unlabeled methionine (500 &#65533;?&#65533;g/ml) indicated     pre-rRNA. The formation 5S RNA 4S tRNA was, <br /> times. Total RNA isolated batch cells; time    however, readily observed, eIF6-depleted KSY603 <br /> point, RNA sample containing 20,000 cpm 3H radioactivity     cells showed lower overall incorporation [3H]uracil 5S <br /> was analyzed 1.2% formaldehyde-agarose gel subjected to <br /> fluorography described Materials Methods. The positions of <br />                                                                          RNA eIF6-containing wild-type cells (Fig. 4B), possibly <br /> mature 18S 25S rRNAs pre-rRNAs indicated. (B) Expo-          turnover 5S RNA incorporated 60S <br /> nentially growing cultures KSY603[GAL10::Ub-TIF6] control         particles. <br /> KSY606[TIF6][URA3] cells SGal-Ura medium (A600 &#11005; 0.5)               To characterize steps pre-RNA processing pathway <br /> transferred SD-Ura medium allowed grow 90 min, pulsed      defective absence eIF6, steady-state levels <br /> with 200 &#65533;?&#65533;Ci [5,6-3H]uracil 3 min, chased ex- <br /> cess unlabeled uracil (1 mg/ml) indicated times. Total cel-   pre-rRNAs mature rRNAs determined North- <br /> lular RNA sample isolated, approximately 60,000        ern blot analysis total RNA isolated eIF6-depleted and <br /> cpm [3H]RNA sample subjected 7% polyacrylamide&#8211;8          control wild-type cells, using deoxyoligonucleotide probes <br /> M urea gel electrophoresis, transferred Hybond-N&#11001; membrane,         complementary defined regions 35S pre-rRNA shown <br /> and analyzed fluorography described Materials Methods. <br /> The positions 5.8S 5S RNAs 4S tRNA indicated. <br />                                                                          Fig. 3. RNA isolated wild-type W303&#65533;?&#65533;[TIF6] and <br />                                                                          KSY603[GAL10::Ub-TIF6] cells initially grown in <br />                                                                          YPGal medium shifted YPD medium allowed <br /> yeast RNA [methyl-3H]methionine (33, 36). For pur-             grow various times. Figure 5 shows eIF6 depletion <br /> pose, exponentially growing cultures yeast strain KSY603              caused marked decrease 25S rRNA slight de- <br /> [GAL10::Ub-TIF6] wild-type strain W303&#65533;?&#65533;[TIF6]                 crease 18S rRNA steady-state levels (Fig. 5A B). Hy- <br /> SGal medium transferred SD&#11002;Met medium, cells                 bridization oligonucleotide probe 1, comple- <br /> were allowed grow 90 min remove eIF6 KSY603               mentary sequences 5&#11032; site A0 (Fig. 3) detects the <br /> cells. Cells pulsed-labeled [methyl-3H]methi-             precursor 35S rRNA, showed eIF6-depleted KSY603 cells <br /> onine 2 min chased 1.5, 3.0, 6.0 min             accumulated 35S pre-rRNA, indicating processing defect at <br /> excess unlabeled methionine.                                          site A0 (Fig. 5C). In addition, aberrant intermediate pro- <br />    In wild-type W303&#65533;?&#65533; cells, 35S pre-rRNA 27S 20S                cessing product 23S accumulated cells (Fig. 5C). <br /> processing intermediates rapidly chased mature 25S             This 23S pre-rRNA detected oligonucleotide <br /> and 18S rRNAs, expected (Fig. 4A). In contrast, eIF6-              probe 3 (Fig. 5D) probe 4 (not shown) probe <br /> depleted KSY603 cells, processing of pre-rRNA                 5 (Fig. 5E), indicating 23S RNA correspond to <br /> intermediates slightly slower. This true 35S 27S           region 35S pre-rRNA extends 5&#11032; <br /> and 20S species, processing 20S 18S mature rRNA,            externally transcribed spacer (5&#11032;ETS) site A3. Thus, the <br /> and disappearance 27S pre-rRNA. The striking aspect            aberrant 23S pre-rRNA probably originated premature <br /> is eIF6-depleted cells, processing 27S pre-rRNA           cleavage 35S pre-rRNA site A3. It noted that <br /> leads complete loss RNA production              investigators observed accumulation 23S rRNA <br /> the 25S mature species. In addition, transient               intermediate yeast cells defective 35S pre- <br />  VOL. 75, 2001                                              eIF6 IS INVOLVED IN 60S RIBOSOMAL SUBUNIT BIOGENESIS                         1459 <br />  <br />  <br />  <br />  <br />    FIG. 5. Steady-state levels mature rRNAs rRNA precursors eIF6-depleted cells. Haploid yeast strains W303&#65533;?&#65533;[TIF6] KSY603 <br /> [GAL10::Ub-TIF6] grown YPGal medium shifted YPD medium. At indicated times, cells harvested, total RNA was <br /> extracted subjected Northern analysis using 32P-labeled oligonucleotide probes complementary different regions 35S pre-rRNA as <br /> described Materials Methods Fig. 3. (A) Probe 9, sequences mature 25S rRNA; (B) probe 2, sequences mature <br /> 18S rRNA; (C) probe 1, sequences upstream site A0 5&#11032;ETS; (D) probe 3, sequences ITS1 sites D A2; (E) probe <br /> 5, sequences ITS1 sites A3 B1L; (F) probe 6, sequences mature 5.8S rRNA; (G) probe 7, sequences ITS2 E and <br /> C2; (H) probe 6, detection small 5.8S rRNA. The positions mature 25S 18S rRNAs different precursor rRNAs indicated. The <br /> blots analyzed using probe corresponding U1 small nuclear RNA, served internal loading control (not shown). <br />  <br />  <br />  <br /> rRNA processing pathway (11, 39; references 12              hybridization data suggest removal eIF6 yeast cells <br /> 35). Hybridization oligonucleotide probe 3 (Fig. 5D)           results inhibition A0-A2 cleavage, leading in- <br /> showed concomitant eIF6 depletion,               creased accumulation 35S pre-rRNA, appearance a <br /> 20S pre-rRNA diminished significantly. The decrease               23S aberrant rRNA premature cleavage 35S pre-rRNA at <br /> level 20S pre-rRNA observed eIF6-depleted cells                site A3, reduced steady-state levels 20S, 27SA2, 27SA3 <br /> probably defects processing 35S pre-rRNA            pre-rRNAs, increased accumulation 27SB pre-rRNA, <br /> accumulation aberrant 23S pre-rRNA. In view                processing blocked 27SB pre-rRNA. Inhibition <br /> observation synthesis 18S rRNA weakly im-               processing 27SB pre-rRNA results decreased <br /> paired eIF6-depleted cells (Fig. 5B), mature 18S          formation mature 25S rRNA 7S pre-rRNA. <br /> rRNA derived processing 23S                      eIF6 present nucleus cytoplasm. eIF6 was <br /> rRNA. When oligonucleotide 5 used hybridization                 originally isolated cytoplasm cell extracts higher <br /> probe (Fig. 5E), observed levels 27SA2 27SA3              eukaryotes (17, 22, 23, 34), including extracts rabbit reticu- <br /> pre-rRNAs lower eIF6-depleted cells                 locytes, lack nucleus (17, 28). In yeast strain KSY603, <br /> wild-type cells. However, probing oligonucleotide 6 (Fig.           HA-eIF6 expressed GAL10 promoter be <br /> 5F), detect 27SA2, 27SA3 pre-RNAs                  associated free cytoplasmic 60S ribosomal subunits (29). <br /> as 27SB pre-RNA, observed eIF6 depletion                 However, observation work eIF6 required for <br /> caused initial accumulation 27S pre-rRNA. Similar results             processing pre-rRNAs mature rRNAs, process is <br /> were obtained probe 7 (data shown). Based                known occur nucleolus, suggests eIF6 also <br /> results Fig. 5E F, conclude depletion           present nuclei yeast cells. We investigated the <br /> of eIF6 yeast cells caused accumulation 27SB pre-rRNA,             subcellular localization eIF6 constructing yeast strain <br /> indicating block processing 27SB pre-rRNA 25S             (KSY605) functional eIF6 expressed own <br /> rRNA 7S pre-rRNA. This confirmed determining                  natural promoter C-terminally c-Myc-tagged fusion pro- <br /> the steady-state levels 7S pre-rRNA 5.8S mature rRNA              tein. Yeast cells expressing c-Myc-tagged eIF6 fusion protein <br /> in eIF6-depleted KSY603 cells relative wild-type W303&#65533;?&#65533;                fractionated nuclear cytoplasmic fractions, <br /> cells using oligonucleotide probes 7 (Fig. 5G) 6 (Fig. 5H),          nuclei purified described Materials <br /> respectively. Concomitant gradual depletion eIF6            Methods. The proteins nuclear cytoplasmic <br /> KSY603 cells, steady-state levels 7S pre-rRNA ma-             fractions analyzed Western blotting using anti-c-Myc <br /> ture 5.8S rRNA decreased. Taken together, Northern              antibodies. Figure 6A shows eIF6-Myc present both <br />  1460      BASU ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                            nuclear postnuclear (cytoplasmic) fractions. The validity of <br />                                                                            fractionation verified using antibodies eIF5 (a <br />                                                                            marker cytoplasmic proteins) Nop1p (a marker for <br />                                                                            nucleolar proteins). To visualize directly presence eIF6 <br />                                                                            subcellular fractions, constructed haploid yeast strain <br />                                                                            KSY607, eIF6 expressed C-terminally-tagged <br />                                                                            HA fusion protein cognate promoter. This eIF6-HA <br />                                                                            fusion protein functionally complement disruption in <br />                                                                            genomic copy TIF6, indicating presence HA <br />                                                                            C terminus eIF6 does alter function yeast <br />                                                                            cells (data shown). Exponentially growing cells KSY607 <br />                                                                            expressing eIF6-HA examined indirect immunofluo- <br />                                                                            rescence using anti-HA antibodies followed decoration with <br />                                                                            goat anti-mouse Cy3-conjugated antibodies. The nuclei were <br />                                                                            stained DAPI. The fluorescence photographs showed that <br />                                                                            HA-eIF6 distributed evenly nucleus cyto- <br />                                                                            plasm fully apparent bud lacking nucleus (Fig. <br />                                                                            6B). No signal obtained combination anti-HA <br />                                                                            goat anti-mouse Cy3-conjugated antibodies cells of <br />                                                                            KSY606 expressing untagged eIF6 analyzed indirect <br />                                                                            immunofluorescence (Fig. 6B, Control). It noted <br />                                                                            calculation relative distribution total eIF6 in <br />                                                                            nuclear cytosolic fractions experiment presented in <br />                                                                            Fig. 6A showed &#11022;90% total cellular eIF6 present <br />                                                                            cytosolic fraction, 5 8% present in <br />                                                                            nuclear fraction (data shown). <br />  <br />                                                                                                      DISCUSSION <br />  <br />                                                                               A molecular genetic approach using yeast S. cerevisiae <br />                                                                            previously used laboratory (29) investigate wheth- <br />                                                                            er eIF6, originally isolated characterized basis its <br />                                                                            vitro ability bind 60S ribosomal subunits (17, 22, 23, 34) <br />                                                                            prevent association 40S ribosomal subunits, <br />                                                                            functions translation initiation factor vivo in <br />                                                                            yeast cells. Using yeast strain KSY603, contains con- <br />                                                                            ditional eIF6 expression system, able remove eIF6 <br />                                                                            rapidly yeast cells measure various parameters of <br />                                                                            protein synthesis vivo vitro (29). We observed that <br />                                                                            removal eIF6 yeast cells caused progressive inhibi- <br />                                                                            tion rate protein synthesis yeast cells. However, this <br />                                                                            inhibition protein synthesis defect trans- <br />                                                                            lation initiation selective reduction level of <br />                                                                            60S ribosomal subunits yeast cells, causing stoichiometric <br />                                                                            imbalance 40S 60S ribosomal subunits con- <br />                                                                            sequent formation half-mer polysomes (29). Similar obser- <br />                                                                            vations reported Sanvito et al. (26), identified <br />                                                                            eIF6 mammalian cells &#65533;?&#65533;4 integrin-interacting protein <br />                                                                            (3). The question, however, remained: does eIF6 function <br />    FIG. 6. Subcellular distribution eIF6. (A) Cell fractionation. Ly-   maintain steady-state level 60S ribosomal subunits in <br /> sates haploid yeast strain KSY605 expressing c-Myc-tagged         yeast cells? <br /> eIF6 natural promoter fractionated postnuclear             Experiments presented stability ma- <br /> supernatant (cytosolic fraction) nuclear pellet described in <br /> Materials Methods. Approximately 25 &#65533;?&#65533;g protein each <br />                                                                            ture 60S particles synthesized KSY603 cells presence <br /> fraction subjected Western blot analysis using rabbit polyclonal    eIF6 significantly affected following removal eIF6 <br /> anti-c-Myc, anti-yeast eIF5, anti-Nop1p antibodies probes. (B) <br /> Localization eIF6-HA fusion protein. Indirect immunofluorescence <br /> staining performed KSY607 cells expressing eIF6-HA from <br /> the TIF6 promoter. Chromatin DNA yeast cells stained           panels merged. Note merged photograph, <br /> DAPI. eIF6-HA detected immunofluorescence staining             nuclear region assumes magenta color. In control panel, <br /> monoclonal mouse anti-HA primary antibody Cy3-conjugated sec-          indirect immunofluorescence performed untagged eIF6 ex- <br /> ondary antibody (second panel). In panel, photographs     pressed TIF6 promoter KSY606 cells. <br />  VOL. 75, 2001                                           eIF6 IS INVOLVED IN 60S RIBOSOMAL SUBUNIT BIOGENESIS                     1461 <br />  <br />  <br /> from cells. Rather, evidence clearly indicates   processing modification pre-rRNA. Mutation deple- <br /> eIF6-depleted cells, reduction 60S subunit levels    tion proteins involved ribosome biogenesis usually leads <br /> to primary defect pre-rRNA processing. A pulse-label          defects multiple steps pathway synthesis of <br /> shortly eIF6 depletion led relatively normal incorpo-     ribosomes (12, 35). Thus, experiments rule the <br /> ration rRNA molecules expected, slightly de-       possibility eIF6, addition required efficient <br /> layed, processing 27S 20S pre-rRNA species. However,        processing pre-rRNA, required synthesis ribo- <br /> while 20S precursor RNA processed 18S rRNA              somal proteins and/or assembly mature ribosomes. It <br /> efficiently, 27S pre-rRNA species degraded         noted measurement rate transcription <br /> without forming 25S 5.8S mature rRNAs.                         rRNA genes using nuclear run-on assays described <br />    These results confirmed Northern blot analysis. In      Elion Warner (8) did significant difference <br /> eIF6-depleted cells, increase steady-state      synthesis 35S rRNA wild-type eIF6- <br /> level 35S 27SB precursor rRNAs marked decrease        depleted cell extracts (data shown). <br /> in level mature 25S 5.8S rRNAs 7S pre- <br />                                                                       In work, studied subcellular localiza- <br /> rRNA. This result reflects defect processing 27SB <br />                                                                    tion eIF6 yeast cells. Although eIF6 associated free <br /> species mature 25S rRNA 7S pre-rRNA. Whereas de- <br />                                                                    cytoplasmic 60S subunits mammalian cells (17, 22, 23, <br /> pletion eIF6 leads marked reduction steady-state <br />                                                                    28, 34) yeast S. cerevisiae (29), indirect immunoflu- <br /> level 25S rRNA, presumably dilution, only <br />                                                                    orescence studies using eIF6-HA fusion protein cell <br /> a slight decrease 18S rRNA time <br /> period. This observation points specific imme-     fractionation studies clearly protein localized <br /> diate block synthesis 60S-specific rRNAs eIF6-        cytoplasm nucleus. These localization <br /> depleted cells. Taken together, results suggest eIF6    data consistent role eIF6 pre-rRNA process- <br /> is necessary formation 60S subunits       ing, mainly nucleolar event. However, predominant cy- <br /> necessary formation 25S 5.8S rRNAs. Perhaps      toplasmic localization eIF6 suggests eIF6 a <br /> absence eIF6 induces processing machinery make        function cytoplasm. It tempting speculate eIF6 <br /> error results immediate degradation 27S RNA.        exported nucleus association 60S ribosomal <br /> Alternatively, eIF6 needed signal degradation     subunits released 60S subunit it <br /> machinery 60S particle functional.                     binds 40S initiation complex form 80S initiation <br />    We know accumulation 35S 23S pre-         complex. This accord observation HA- <br /> rRNAs eIF6-depleted cells results directly eIF6 deple-     tagged eIF6 expressed yeast cells associates free cyto- <br /> tion consequence defective 27SB pre-rRNA            solic 60S subunits 80S monosomes polyri- <br /> processing causes form feedback inhibition        bosomes (29). Association eIF6 free cytoplasmic 60S <br /> delayed processing 35S pre-rRNA. Inhibition processing       subunits suggests eIF6 act chaperone trans- <br /> of 35S pre-rRNA, accumulation aberrant 23S pre-         porting 60S subunits nucleus cytoplasm. The <br /> rRNA cleavage 35S RNA site A3, reduced steady-        possibility exists eIF6 does directly func- <br /> state levels 20S pre-rRNA reported       tion general translation initiation factor, binding 60S <br /> mutants affect biogenesis 60S ribosomal subunits (2, 10,   ribosomal subunits serve checkpoint subunit <br /> 12, 18, 31, 38, 39; reference 35). These observations     joining step initiation protein synthesis. We found <br /> suggest delayed processing sites A0 A2         time ago (34) 60S ribosomal subunits containing <br /> general feature mutations inhibit formation ma-     bound mammalian eIF6 incapable joining 40S ini- <br /> ture 25S 5.8S rRNAs. It noted pre-     tiation complex form 80S initiation complex. These <br /> rRNA processing defects observed eIF6-depleted cells            observations suggest mechanism exist re- <br /> closely resemble previously reported yeast cells lack-   lease eIF6 60S subunit prior concom- <br /> ing Nip7p, Nop2p, Dbp3p (10, 18, 35, 39). Depletion of <br />                                                                    itant joining 60S subunit 40S initiation <br /> each Nip7p Nop2p yeast cells caused block the <br />                                                                    complex. Interestingly, Nip7p (39) Nmd3p (9) also <br /> processing 27SB pre-rRNA 25S rRNA 7S pre-rRNA, <br />                                                                    identified proteins required 60S biogenesis that <br /> while cells lacking Dbp3p shown defective the <br />                                                                    associate specifically free 60S subunits cyto- <br /> processing 27SA2 27SA3 pre-rRNA, resulting in- <br />                                                                    plasm yeast cells. It remains seen pro- <br /> hibition synthesis mature 25S 5.8S rRNAs; cells <br /> also accumulated 35S 23S pre-rRNAs reduced levels          teins possess ribosomal subunit antiassociation activity sim- <br /> of 20S pre-rRNA. Reduction level 20S pre-rRNA,        ilar observed eIF6 (29). On hand, the <br /> turn, causes inhibition formation 18S rRNA. It      cytoplasmic localization association eIF6 60S ribo- <br /> has postulated (35) requirement 25S 5.8S     somal subunits functional consequence might <br /> processing proteins 18S rRNA synthesis indirect. It      involved unknown function eIF6. <br /> likely pre-rRNA processing machinery single large       eIF6 evolutionarily conserved protein. Proteins struc- <br /> complex formed specific steps 25S 5.8S      turally homologous eIF6 phyloge- <br /> rRNA formation. Thus, defects processing protein            netically distant species including archeaons plants not <br /> might inhibit early steps including eventually 18S rRNA syn-       eubacteria (3, 28). The high degree conservation amino <br /> thesis.                                                            acid sequence phylogenetically distant species indicates <br />    It established synthesis ribosomal      eIF6 plays important role essential process the <br /> proteins assembly ribosomes tightly coupled          cell. <br />  1462        BASU ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />                            ACKNOWLEDGMENTS                                              18. Ripmaster, T. L., G. P. Vaughn, J. L. Woolford, Jr. 1992. A putative <br />                                                                                             ATP-dependent RNA helicase involved Saccharomyces cerevisiae ribo- <br />   U. Basu K. Si contributed equally work.                                       assembly. Proc. Natl. Acad. Sci. USA 89:11131&#8211;11135. <br />   This work supported grants GM15399 U.M. GM25532                         19. Rose, M. D., F. Winston, P. Hieter. 1990. Methods yeast genetics, a <br /> to J.R.W. National Institutes Health <span id='am-41' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45'  ></span>Cancer</span> Core                         laboratory course manual. Cold Spring Harbor Laboratory Press, Cold <br /> Support Grant P30CA13330 National Cancer Institute.                                Spring Harbor, N.Y. <br />   We indebted Amy Chang institution considerable                     20. Rotenberg, M. O., M. Moritz, J. L. Woolford, Jr. 1988. Depletion of <br /> help studies indirect immunofluorescence eIF6.                                     Saccharomyces cerevisiae ribosomal protein L16 causes decrease 60S <br />                                                                                             ribosomal subunits formation half-mer polyribosomes. Genes Dev. 2: <br />                                                                                             160&#8211;172. <br />                                   REFERENCES                                            21. Rothstein, R. J. 1983. One-step gene disruption yeast. Methods Enzymol. <br />  1. Aris, J. P., G. Blobel. 1991. Isolation yeast nuclei. Methods Enzymol.           101:202&#8211;211. <br />     194:735&#8211;749.                                                                        22. Russell, D. W., L. L. Spremulli. 1979. Purification characterization <br />  2. Berges, T., E. Petfalski, D. Tollervey, E. C. Hurt. 1994. Synthetic lethality       ribosome dissociation factor (eukaryotic initiation factor 6) wheat <br />     fibrillarin identifies NOP77p, nucleolar protein required pre rRNA           germ. J. Biol. Chem. 254:8796&#8211;8800. <br />     processing modification. EMBO J. 13:3136&#8211;3148.                                  23. Russell, D. W., L. L. Spremulli. 1980. Mechanism action wheat <br />  3. Biffo, S., F. Sanvito, S. Costa, L. Preve, R. Pignatelli, L. Spinardi, P. C.        germ ribosome dissociation factor: interaction 60 S subunit. Arch. <br />     Marchisio. 1997. Isolation novel <span id='am-11' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-12' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16'  ></span>beta</span> 4 integrin-binding protein (p27BBP)          Biochem. Biophys. 201:518&#8211;526. <br />     highly expressed epithelial cells. J. Biol. Chem. 272:30314&#8211;30321.               24. Sachs, A. B., R. W. Davis. 1989. The poly(A) binding protein required <br />  4. Chakravarti, D., T. Maiti, U. Maitra. 1993. Isolation immunochemi-              poly(A) shortening 60S ribosomal subunit-dependent translation <br />     cal characterization eukaryotic translation initiation factor 5 Sac-            initiation. Cell 58:857&#8211;867. <br />     charomyces cerevisiae. J. Biol. Chem. 268:5754&#8211;5762.                                25. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />  5. Deshmukh, M., Y.-F. Tsay, A. G. Paulovich, J. L. Woolford, Jr. 1993.                laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring <br />     Yeast ribosomal protein L1 required stability newly synthesized           Harbor, N.Y. <br />     5S rRNA assembly 60S ribosomal subunits. Mol. Cell. Biol. 13:            26. Sanvito, F., S. Piatti, A. <span id='am-74' about='obo:NCBITaxon_117093' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_117077 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_50704 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_50674 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Villa" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Villa</span>, M. Bossi, G. Lucchini, P. C. Marchisio, S. <br />     2835&#8211;2845.                                                                              Biffo. 1999. The &#65533;?&#65533;4 integrin interactor p27 (BBP/eIF6) essential nuclear <br />  6. Eisinger, D. P., F. A. Dick, E. Denke, B. L. Trumpower. 1997. SQT1,                 matrix protein involved 60S ribosomal subunit assembly. J. Cell Biol. 144: <br />     encodes essential WD domain protein Saccharomyces cerevisiae,               823&#8211;837. <br />     suppresses dominant-negative mutations ribosomal protein gene                27. Schmitt, M. E., T. A. Brown, B. L. Trumpower. 1990. A rapid simple <br />     QSR1. Mol. Cell. Biol. 17:5146&#8211;5155.                                                    method preparation RNA Saccharomyces cerevisiae. Nucleic <br />  7. Eisinger, D. P., F. A. Dick, B. L. Trumpower. 1997. Qsr1p, 60S                    Acids Res. 18:3091&#8211;3092. <br />     ribosomal subunit protein, required joining 40S 60S subunits.         28. Si, K., J. Chaudhuri, J. Chevesich, U. Maitra. 1997. Molecular cloning <br />     Mol. Cell. Biol. 17:5136&#8211;5145.                                                          functional expression human cDNA encoding translation initiation <br />  8. Elion, E. A., J. R. Warner. 1984. The major promoter element rRNA                factor 6. Proc. Natl. Acad. Sci. USA 94:14285&#8211;14290. <br />     transcription yeast lies 2 kb upstream. Cell 39:663&#8211;673.                         29. Si, K., U. Maitra. 1999. The Saccharomyces cerevisiae homologue of <br />  9. Ho, J. H.-N., A. W. Johnson. 1999. NMD3 encodes essential cyto-                  mammalian translation initiation factor 6 does function translation <br />     plasmic protein required stable 60S ribosomal subunits Saccharomyces             initiation factor. Mol. Cell. Biol. 19:1416&#8211;1426. <br />     cerevisiae. Mol. Cell. Biol. 19:2389&#8211;2399.                                          30. Sikorski, R. S., P. Hieter. 1989. A shuttle <span id='am-79' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> yeast <br /> 10. Hong, B., J. S. Brockenbrough, P. Wu, J. P. Aris. 1997. Nop2p                    host strains designed efficient manipulation DNA Saccharomyces <br />     required pre-rRNA processing 60S ribosome subunit synthesis in <br />                                                                                             cerevisiae. Genetics 122:19&#8211;27. <br />     yeast. Mol. Cell. Biol. 17:378&#8211;388. <br />                                                                                         31. Sun, C., J. L. Woolford, Jr. 1994. The yeast NOP4 gene essential <br /> 11. Kressler, D., J. D.-L. Cruz, M. Rojo, P. Linder. 1998. Dbp6p an <br />                                                                                             nucleolar protein required pre-rRNA processing accumulation of <br />     essential putative ATP-dependent RNA helicase required 60S ribosomal- <br />                                                                                             60S ribosomal subunits. EMBO J. 13:3127&#8211;3135. <br />     subunit assembly Saccharomyces cerevisiae. Mol. Cell. Biol. 18:1855&#8211;1865. <br />                                                                                         32. Tollervey, D. 1996. Trans-acting factors ribosome synthesis. Exp. Cell Res. <br /> 12. Kressler, D., P. Linder, J. D.-L. Cruz. 1999. Protein trans-acting factors <br />     involved ribosome biogenesis Saccharomyces cerevisiae. Mol. Cell. Biol.           229:226&#8211;232. <br />     19:7897&#8211;7912.                                                                       33. Udem, S., J. R. Warner. 1972. Ribosomal RNA synthesis Saccharo- <br /> 13. Merrick, W. C., J. W. B. Hershey. 1996. The pathway mechanism                myces cerevisiae. J. Mol. Biol. 65:227&#8211;242. <br />     eukaryotic protein synthesis, p. 31&#8211;69. In J. W. B. Hershey, M. B. Matthews,        34. <span id='am-46' about='obo:NCBITaxon_186129' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_38127 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_239228 obo:NCBITaxon_298052 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_159975 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_30259 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Valenzuela" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Valenzuela</span>, D. M., A. Chaudhuri, U. Maitra. 1982. Eukaryotic ribosomal <br />     N. Sonenberg (ed.), Translational control. Cold Spring Harbor Labora-               subunit anti-association activity calf liver contained single polypep- <br />     tory Press, Cold Spring Harbor, N.Y.                                                    tide chain protein Mr &#11005; 25,500 (eukaryotic initiation factor 6). J. Biol. <br /> 14. Moritz, M., B. A. Pulaski, J. L. Woolford, Jr. 1991 Assembly 60S                 Chem. 257:7712&#8211;7719. <br />     ribosomal subunits perturbed temperature-sensitive yeast mutants de-          35. Venema, J., D. Tollervey. 1999. Ribosome synthesis Saccharomyces <br />     fective ribosomal protein L16. Mol. Cell. Biol. 11:5681&#8211;5692.                        cerevisiae. Annu. Rev. Genet. 33:261&#8211;311. <br /> 15. Moritz, M., A. G. Paulovich, Y. F. Tsay, J. L. Woolford, Jr. 1990.              36. Warner, J. R. 1991. Labeling RNA phosphoproteins Saccharomyces <br />     Depletion yeast ribosomal proteins L16 rp59 disrupts ribosome assem-              cerevisiae. Methods Enzymol. 194:423&#8211;427. <br />     bly. J. Cell Biol. 111:2261&#8211;2274.                                                   37. Wang, Q., A. Chang. 1999. Eps1, novel PDI-related protein involved in <br /> 16. Park, E.-C., D. Finley, J. W. Szostak. 1992. A strategy generation          ER quality control yeast. EMBO J. 18:5972&#8211;5982. <br />     conditional mutations protein destabilization. Proc. Natl. Acad. Sci.         38. Weaver, P. L., C. Sun, T.-H. Chang. 1997. Dbp3p, putative RNA <br />     USA 89:1249&#8211;1252.                                                                       helicase Saccharomyces cerevisiae, required efficient pre-rRNA pro- <br /> 17. Raych <br /> </body></html>