<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Oct. 2001, p. 6668&#8211;6680                                                                            Vol. 21, No. 19 <br /> 0270-7306/01/$04.00&#11001;0 DOI: 10.1128/MCB.21.19.6668&#8211;6680.2001 <br /> Copyright &#169; 2001, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />      Molecular Genetic Dissection TAF25, Essential Yeast Gene <br />             Encoding Subunit Shared TFIID <span id='am-132' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="rdfs:label" content="Saga" datatype="xsd:string"></span>SAGA</span> <br />                   Multiprotein Transcription Factors <br />               JAY KIRCHNER, STEVEN L. SANDERS, EDWARD KLEBANOW,&#8224;                                     AND   P. ANTHONY WEIL* <br />                                Department Molecular Physiology Biophysics, Vanderbilt University <br />                                        School Medicine, Nashville, Tennessee 37232-0615 <br />                            Received 30 April 2001/Returned modification 31 May 2001/Accepted 27 June 2001 <br />  <br />              We performed systematic structure-function analysis <span id='am-33' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-34' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> TAF25, evolu- <br />           tionarily conserved, single-copy essential gene encodes 206-amino-acid TAF25p protein. TAF25p is <br />           integral subunit 15-subunit general transcription factor TFIID multisubunit, chromatin- <br />           acetylating transcriptional coactivator SAGA. We used hydroxylamine mutagenesis, targeted deletion, alanine- <br />           scanning mutagenesis, high-copy suppression methods, two-<span id='am-192' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-193' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-196'  ></span>hybrid</span> screening dissect TAF25. Temper- <br />           ature-sensitive mutant strains generated used coimmunoprecipitation transcription analyses to <br />           define vivo functions TAF25p. The results analyses TAF25p comprised multiple <br />           mutable elements contribute importantly RNA polymerase II-mediated mRNA gene transcription. <br />  <br />  <br />    mRNA gene transcription mediated RNA polymerase                       TAF130p (also known TAF145p) (58), metazoan <br /> II working concert multiple general transcription fac-                 counterparts (human Drosophila TAF250p) contain intrin- <br /> tors (GTFs). The basal mRNA gene transcription machinery,                      sic enzymatic activities contribute transcription (15, 16, <br /> as originally defined vitro, comprised GTFs TFIIA,                47, 49, 53, 54, 56). <br /> TFIIB, TFIID, TFIIE, TFIIF, TFIIH, function                        Genetic biochemical experiments indicated di- <br /> RNA polymerase II promote preinitiation complex (PIC)                       rect interactions activation domains (AD) tran- <br /> formation vitro vivo (see references 29 55                   scriptional activators subunits TFIID play key roles <br /> reviews). GTFs contribute PIC formation multiple ways,                   transactivation (11, 22, 23, 38, 40, 45, 62, 63, 70, 73, 74, 77). <br /> either acting sequentially stepwise fashion (7) form                 <span id='am-119' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-120' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> coactivator function manifested molecular <br /> PIC acting single unit comprising RNA polymerase                   level DNA-bound activators stabilizing (recruiting) <br /> II holoenzyme (35, 39). Although composition basal                  TFIID TATA box-core promoter (TATA-INR-DPE) <br /> transcription machinery possible modes action                  (8, 42, 64, 65) <span id='am-160' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-163'  ></span><span id='am-164' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span>-linked genes or, perhaps, (also) acti- <br /> been fairly characterized, mechanisms transcrip-                   vating latent enzymatic activities residing subunits of <br /> tional activation poorly understood (44). Transactiva-               TFIID itself. Regardless exact mechanisms which <br /> <span id='am-179' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-180'  ></span><span id='am-181' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> proteins shown interact variety                     transactivation occurs, systematic molecular genetic <br /> targets, including protein components RNA poly-                     dissection components comprising GTFs, particularly <br /> merase II transcription machinery chromatin con-                    TFIID, complex process fully understood. <br /> stituents activities modify chromatin. Among                      The yeast SAGA complex contains 14 subunits with <br /> putative transactivator targets GTF TFIID                      molecular masses ranging 430 17 kDa. The Gcn5p <br /> SAGA (Spt-<span id='am-88' about='obo:NCBITaxon_125078' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158332 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_158418 obo:NCBITaxon_2759 obo:NCBITaxon_158394 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Ada Lindl." datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Ada" datatype="xsd:string"></span><span id='am-93'  ></span>Ada</span>-Gcn5 acetylase) histone acetyltransferase <br />                                                                                subunit SAGA carries catalytic activity capable acety- <br /> complex. Both multisubunit complexes ex- <br />                                                                                lating nucleosomal histones (24). Interestingly, sub- <br /> tensively studied yeasts metazoans (see references 1, 4, <br />                                                                                units TFIID, TAF90p, TAF61(68)p, TAF60p, TAF25p, and <br /> 18, 66 recent reviews). <br />                                                                                TAF17p, shared SAGA (25). Except TRA1, which <br />    Yeast TFIID composed 14 TATA box DNA binding <br />                                                                                encodes largest subunit, known, non- <br /> protein-associated factors (TAFs) exhibiting molecular masses <br />                                                                                TAF SAGA subunits encoded essential genes. Presum- <br /> ranging 150 17 kDa (61). Although identifies of <br />                                                                                ably, genetic nonessentiality reflects redundant nature <br /> TFIID subunits known, exact stoichiometry these <br />                                                                                function(s) SAGA chromatin-modifying <br /> multiple subunits complex not. With exception <br /> of TAF30p (30), TFIID subunits encoded single-copy                  complexes (see references 36, 66, 75 recent reviews). <br /> essential genes, display high degree sequence                     Like TFIID, SAGA shown play key role gene <br /> conservation eukaryotes. One subunits, yeast                    induction transcriptional regulation. Recent vivo (13, <br />                                                                                26, 41, 69) vitro (46, 52, 72) studies demonstrated <br />                                                                                SAGA specifically associates target genes early the <br />   * Corresponding author. Mailing address: Department Molecular             transcriptional activation process. The association these <br /> Physiology Biophysics, Vanderbilt University School Medicine,           chromatin-modifying complexes thought direct effect <br /> Nashville, TN 37232-0615. Phone: (615) 322-7007. Fax: (615) 322-7236. <br />                                                                                specific, high-affinity AD-SAGA interactions. Complex for- <br /> E-mail: tony.weil@mcmail.vanderbilt.edu. <br />   &#8224; Present address: Argus Research Corporation, New York, NY                  mation AD SAGA mutationally sensitive and <br /> 10006.                                                                         readily observed vitro (33, 72). Clearly, thorough <br />  <br />                                                                         6668 <br />  VOL. 21, 2001                                                                                  TAF25p STRUCTURE-FUNCTION ANALYSIS                             6669 <br />  <br />  <br /> and detailed understanding mechanisms gene activa-                        aration, antibody preparation, immunoblotting, immunoprecipitation were <br /> tion require extensive dissection components                       performed detailed previously (61). Immunoblots quantitated using a <br />                                                                                     Fluor-S MultiImager (Bio-Rad). Yeast two-hybrid screening performed us- <br /> comprising SAGA.                                                                    ing Clontech yeast strain L40 (according manufacturer protocols) and <br />    Certain TAFp-TAFp interactions understood mo-                         yeast strain PJ69-4A detailed previously (34). TAF25 &#8220;bait&#65533;? molecules used in <br /> lecular level. The cocrystal structures TAFp-TAFp                            screening fused TAF25p-encoding open reading frame (ORF) either <br /> (core) complexes solved X-ray crystallography (5,                      LexA Gal4 DNA binding domains (DBD). <br /> 76). From work quite clear highly <br /> conserved protein-protein interaction motif, histone fold                                                        RESULTS <br /> (HF), interacting proteins (68), TAFps <br /> are exception. Indeed, Burley Roeder proposed key                             Inactivation TAF25pG101E temperature shift dramat- <br /> role histone-like TAFps mediating TFIID functions                       ically reduces polymerase II-mediated gene transcription in <br /> years ago (9). The HF motif mediates protein interactions                      vivo complete disruption TFIID SAGA. We per- <br /> within TFIID (i.e., TAF61p-TAF48p, TAF60p-TAF17p,                           formed hydroxylamine mutagenesis TAF25 goal of <br /> TAF25p-TAF47p, TAF25p-TAF65p) SAGA,                                  generating temperature-sensitive mutant alleles gene <br /> where HF-mediated interactions TAF25p Spt7p                             prove useful characterization TAF25p <br /> as TAF68p Ada1p identified char-                          functions. We successful endeavor obtained <br /> acterized (18, 19, 20). Dissecting molecular rules defining                     mutant alleles, clustered TAF25 <br /> these protein-protein interactions prove crucial under-                     sequences encoding amino acids 101 111 (Fig. 1). In an <br /> standing roles multisubunit transcription factors                    earlier work, Sanders et al. (60) used particular temperature- <br /> play regulated mRNA gene transcription.                                          conditional mutant collection, strain YEK25.75, <br />    TAF25p, HF-containing protein, provides unique in-                          expressed form TAF25p bearing single mutation <br /> sight study mechanisms transcriptional reg-                      resulting G3E amino acid substitution position 101 <br /> ulation TAF integral subunit TFIID                       (TAF25pG101E). When YEK25.75 cells shifted a <br /> and SAGA. We originally cloned TAF25 characterized                          permissive (22&#176;C) nonpermissive (37&#176;C) temperature, <br /> encoded protein, TAF25p, presence TFIID                           high-level RNA polymerase II-mediated mRNA gene tran- <br /> preparations (37, 57). It shown TAF25p plays                        scription reduced &#11350;60% 30 min (60). One can <br /> key role mediating transcription vitro (37)                       readily envision limit cases explain transcriptional <br /> vivo (43, 60). However, TAF25p resident                            phenotype. On hand, TFIID and/or SAGA complexes <br /> TFIID SAGA, possible unambiguously de-                            substantially completely disrupted tem- <br /> termine TAF25p-containing complex responsible                             perature shift. Indeed, situation HF motif- <br /> for observed transcription effects aforementioned                        containing TAFs described literature (2, 25, 48, 50, 51, <br /> studies. In <span id='am-129' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-130' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> address gaps under-                      52, 59, 60). Alternatively, second possibility TAF25p is <br /> standing TAF25p function, initiated systematic analysis                     inactivated situ TFIID and/or SAGA complexes, and <br /> of protein, including detailed analysis structure-                    inactivation disrupts critical, positively acting protein-pro- <br /> function relationships TAF25. In report,                    tein contacts TAF25p yet-to-be-de- <br /> efforts dissect TAF25p functional domains                           fined component(s) RNA polymerase II transcription <br /> genetic biochemical experimentation.                                            machinery. The second possibilities mechanis- <br />                                                                                     tically interesting experimentally useful. Prior <br />                         MATERIALS AND METHODS                                       embarking detailed analysis TAF25, wanted to <br />    Yeast bacterial strains. The parental yeast strains YEK16 (MATa leu2- <br />                                                                                     convince simply studying effects <br /> 3,112 ura3-52 his3-&#9004;200 trp1-&#9004;901 suc2-&#9004;9 ade2-101 lys2-801 taf25&#9004;1::TRP1           TFIID and/or SAGA complex dissociation. We therefore <br /> [pRS416-HA3-TAF25 WT]) yJK800 (MAT&#65533;?&#65533; leu2&#9004;0 ura3&#9004;0 his3&#9004;1 taf25D::               conducted experiment allow distinguish <br /> KAN [pRS416-HA3-TAF25]) transformed HIS3-marked CEN-ARS                   limit cases. <br /> <span id='am-197' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-198' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-201' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> (pRS413) carrying TAF25 genes various mutations (Fig. 1). <br />                                                                                        To probe TFIID SAGA complex integrity, per- <br /> The plasmids exchanged plating resulting pseudodiploid strains <br /> on 5-fluoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>) select lost URA3-marked       formed coimmunoprecipitation analysis TAF25p-contain- <br /> plasmids (pRS416-HA3-TAF25 WT) (6). Other strains used study include        ing complexes. A 22&#176;C-grown culture YEK25.75 cells ex- <br /> YEK25.75 (60), YEK20 (60), YSLS74 (MAT&#65533;?&#65533;/MATa leu2&#9004;0/leu2&#9004;0 ura3&#9004;0/              pressing three-hemagglutinin (HA3)-tagged TAF25p mutant <br /> ura3&#9004;0 his3&#9004;1/his3&#9004;1 lys2&#9004;0/LYS2 met16&#9004;0/MET15 taf25&#9004;::KAN),             protein split; left half 22&#176;C shifted half <br /> yJK800 derived sporulation dissection. Additional information on <br /> strains plasmids available request. <span id='am-137' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-148'  ></span><span id='am-149' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> cells grown <br />                                                                                     37&#176;C. After 30 min, WCEs generated, immunoprecipi- <br /> in <span id='am-11' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-15'  ></span>Luria</span>-Bertani media supplemented ampicillin. Yeast cells cultured      tates prepared (antibody excess) using control anti- <br /> in liquid solid defined media (minimal defined [SD] complete defined       FLAG monoclonal antibody (MAb) anti-HA MAb, the <br /> [SC]) rich media (yeast extract-peptone-dextrose [YPD] supplemented         precipitated proteins analyzed immunoblotting. All <br /> adenine needed [YPAD]) formulated described previously (27). <br />                                                                                     components analyzed present 22&#176;C 37&#176;C <br />    Plasmids. TAF25 plasmids constructed standard techniques. In all <br /> cases, plasmid-based TAF25p expression driven normal TAF25 reg-          WCEs (Fig. 2, In). Consistent results previous <br /> ulatory sequences. The 2&#65533;?&#65533;m-based plasmids (pRS426) carrying ADA1, TAF47,            study (60), of subunits analyzed was <br /> and TAF65 constructed standard techniques. The 2&#65533;?&#65533;m <span id='am-165' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> contain-         slightly reduced 37&#176;C WCE (10 25%), were <br /> ing SPT7, TAF25, TAF40 kind gifts Steve Buratowski Fred             large amounts intact TFIID SAGA cells <br /> Winston. <br />    Molecular biological methods. DNA manipulation, purification, analysis, <br />                                                                                     (Fig. 2, Pellet, HA). Thus, primary cause transcrip- <br /> RNA purification hybridization, yeast transformations per-         tion decrement mutant cell strain complete com- <br /> formed described previously (3, 12, 37, 60). Whole-cell extract (WCE) prep-      plex disruption TFIID SAGA. The transcription defect <br />                                                                                                                                                                                                       6670 <br />                                                                                                                                                                                                      KIRCHNER ET AL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Alignment TAF25p orthologs various eukaryotes. Sequences aligned S. cerevisiae (Sc) TAF25p; numbering shown aligned sequences refers the <br /> amino acid sequence protein. <span id='am-57' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe (Sp), C. albicans (Ca), <span id='am-16' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span>Arabidopsis</span> thaliana (At), <span id='am-99' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> (Ce), N-terminally truncated human mouse <br /> (del N-m del N-h) TAF25p orthologs used Clustal W alignment MacVector software (version 6.5.3). These sequences numbered left; translational start site of <br /> the Ca ortholog determined. The sequence Ca TAF25p obtained Stanford DNA Sequencing Technology Center website http://www-sequence.stanford <br /> .edu/group/<span id='am-74' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-79'  ></span>candida</span>; sequences obtained GenBank. Asterisks indicate arginine glutamine residues (R108, R111, Q15, Q18, Q152, Q157, Q158, Q159, Q163, Q164, Q167, <br /> Q168, Q169, Q171, Q172) which, cognate encoding codons mutated hydroxylamine termination codons, produced C-terminally truncated TAF25 proteins conferring <br /> lethality. The locations Q residues (Q48, Q118, Q147, Q155) affected hydroxylamine indicated filled inverted triangles. Residues individually mutated alanine <br /> in background mini-TAF25 indicated circles labeled A (Sc TAF25p residues P86A, I88A, P89A, D90A, L97A, G101A, D106A, R108A, Q118A, K119A, D123A, D127A, L183A, <br /> T184A, L188A, E194A, Y195A, G196A, P202A). Shown sequence alignments putative HF consensus structure alpha helices separated linker regions (alpha <br />                                                                                                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br /> N, LN, alpha 1, L1, alpha 2, L2, alpha 3), proposed Gangloff et al. (18). <br />  VOL. 21, 2001                                                                   TAF25p STRUCTURE-FUNCTION ANALYSIS                 6671 <br />  <br />  <br />                                                                        plasmid shuffling described above. Twenty-five continued to <br />                                                                        confer lethality, DNA sequences TAF25 genes <br />                                                                        determined. Twenty-three carried nonsense mutations <br />                                                                        TAF25p glutamine residues (mutating CAA CAG <br />                                                                        UAA UAG; 21 25) arginine residues (CGA to <br />                                                                        UGA; 2 25). Of 17 possible Q residue targets, 4 <br />                                                                        identified screen, CGA- <br />                                                                        encoded R residues mutated (Fig. 1). The two <br />                                                                        mutations collection mutated initiator methi- <br />                                                                        onine residue. The Q157, Q159, Q171, R111 mutants were <br />                                                                        isolated five, three, two, times, respectively,while all <br />                                                                        Q R mutants isolated once, suggesting the <br />                                                                        screen relatively complete. These results indicated that <br />                                                                        sequences length 618-bp TAF25 ORF <br />                                                                        important function and, particularly, that <br />                                                                        residues C terminal Q172 deleted without <br />                                                                        compromising TAF25p function. This portion TAF25p (res- <br />                                                                        idues 172 206) defines region molecule is <br />                                                                        highly conserved. <br />                                                                           Core residues TAF25p comprising evolutionarily con- <br />                                                                        served sequences sufficient support growth. Alignment <br />                                                                        TAF25p orthologs yeasts, plants, worms, and <br />                                                                        mammals <span id='am-50' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae TAF25p showed, as <br />                                                                        noted previously, portion protein con- <br />                                                                        served organisms (5, 21, 37, 51). Notably, segments <br />                                                                        S. cerevisiae TAF25p (amino acids 2 73 amino acids <br />    FIG. 2. Neither TFIID SAGA complexes significantly dis- <br /> rupted YEK25.75 cells shifted permissive nonpermissive      142 179) conserved (Fig. 1). In order examine <br /> growth conditions. Logarithmically growing YEK25.75 cells expressing   nonconserved N-terminal C-terminal domains of <br /> HA3-tagged TAF25p cultured 22&#176;C YPAD media. Cells           TAF25p critical function, generated deletion de- <br /> were collected centrifugation equal portions, pelleted   rivatives TAF25 removed N-terminal portion <br /> cells resuspended equivalent volume YPAD media either <br /> at 22&#176;C prewarmed 37&#176;C. Both 22&#176;C 37&#176;C cultures <br />                                                                        (amino acids 2 74; termed TAF25&#9004;N), C-terminal por- <br /> were shaken temperatures 30 min. WCEs prepared,      tion (amino acids 140 179; termed TAF25&#9004;C), both <br /> and immunoprecipitations (IP) performed anti-         nonconserved portions (TAF25&#9004;N&#11001;&#9004;C). Vector alone, the <br /> FLAG anti-HA MAb. Fractions Input (In, 2%)           wild-type gene, deletion forms TAF25 were <br /> precipitate (Pellet, 4%) analyzed immunoblotting      introduced yeast, plasmid shuffling used to <br /> indicated antibody. Twenty-five percent 22&#176;C precipitate <br /> was analyzed adjust &#11011;25% decrease          test forms TAF25 supported growth. None the <br /> HA3-TAF25p 37&#176;C WCE. &#11002;, IP.                                  deletion mutant alleles TAF25 exhibited strong dominant- <br />                                                                        negative effects growth (Fig. 3A, &#11002;FOA). After shuffling, <br />                                                                        double-deletion form TAF25 (TAF25&#9004;N&#11001;&#9004;C), <br /> could caused inactivation TAF25p situ            referred core TAF25 mini-TAF25, supported <br /> within complexes slight reduction TFIID          wild-type levels growth. Growth curves various <br /> SAGA levels. Regardless, encouraged <span id='am-187' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-188' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-190' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-191'  ></span>data</span>,               strains liquid media 30&#176;C indistinguishable (Fig. 3B). <br /> clearly demonstrated TAF25p mRNA gene transcrip-              Clearly, conditions (and SD media; data not <br /> tion vivo inactivated induction com-        shown), nonconserved portions TAF25p dispens- <br /> plete disruption TFIID SAGA, embarked                 able normal growth properties. <br /> detailed genetic dissection TAF25p structure-function rela-            Residues conserved core TAF25p important for <br /> tionships.                                                             TAF25p function vivo. Although experiments above <br />    Amino acid sequences C terminal glutamine 172 re-            proved useful, decided systematic muta- <br /> quired produce functional TAF25p. Although useful,              tional approach elucidating structure-function <br /> collection TAF25 mutants described gave little             relationships TAF25 reasons. First, wanted to <br /> insight overall structure-function relationships       assess amino acids conserved 96-ami- <br /> molecule. Indeed, unlikely, plausible, explanation         no-acid core (i.e., mini-TAF25p) really important for <br /> not finding mutations TAF25p              TAF25p function. If so, sequences mu- <br /> residues protein centered amino acids 101 111         tationally sensitive. Second, importantly, wanted <br /> contributed function. To test <span id='am-94' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95'  ></span><span id='am-96' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span>, returned using       prepare panel TAF25 genetic reagents use <br /> hydroxylamine mutagenesis screened mutations              attempt define domains protein uniquely <br /> were lethal. Of &#11011;30,000 colonies screened, 30 failed grow        contribute TFIID-specific SAGA-specific functional- <br /> 5-FOA-containing plates. The relevant HIS3-marked TAF25-               ities. To accomplish goals, performed alanine-scan- <br /> containing mutagenized plasmids separately recovered              ning mutagenesis mini-TAF25. The triplets encoding 19 <br /> from strains, passaged E. coli, rescreened        highly conserved amino acids core domain of <br />  6672     KIRCHNER ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Nonconserved TAF25p sequences dispensable yeast cell growth. (A) Shown photograph plates the <br /> indicated yeast strains inoculated. The plate right contained 5-FOA, selects cells lost URA3-marked <br /> pRS416-TAF25WT plasmid, consequently revealing chromosomal taf25&#9004;1::TRP1 allele testing ability HIS3-marked pRS413 <br /> vector (pRS413) wild-type TAF25 (pRS413-WT) deletion alleles TAF25 (pRS413-TAF25&#9004;N, pRS413-TAF25&#9004;C, pRS413- <br /> TAF25&#9004;N&#11001;&#9004;C) support growth; constructs labeled VECTOR, TAF25, TAF25&#9004;N, TAF25&#9004;C, TAF25&#9004;N&#11001;&#9004;C, respectively. Plates <br /> were incubated 30&#176;C 3 days (left) 6 days (right) imaged. (B) Growth curves various TAF25 deletion alleles 30&#176;C. <br /> Overnight cultures indicated deletion alleles TAF25 diluted appropriate liquid media equal concentration, determined <br /> by cell counts using hemocytometer. The cultures grown 30&#176;C. At various time points, cell counts performed. WT, wild type. <br />  <br />  <br />  <br />  <br /> TAF25p separately mutated encode alanine (Fig. 1,                 TAF25 mutationally sensitive important for <br /> residues marked circles labeled &#8220;A&#65533;?). All               function(s) molecule. <br /> mutants constructed double-deletion mini-                      In order gain insight TAF25p function, the <br /> TAF25 gene backbone cloned pRS413 HA3-tagged                    mutant collection subjected battery growth tests in <br /> proteins. Each construct sequenced, including &#11011;200 bp              attempt sort members <span id='am-71' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> functional <br /> 5&#11032; 3&#11032; sequences. The <span id='am-174' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ala</span> mutants introduced                 subsets. We tested Ala mutant family growth various <br /> yeast strain YEK16 generate family TAF25 pseudodip-               media lacking certain nutrients (methionine inositol) <br /> loid strains; mutants recessive (data shown).                containing alternate carbon sources (glycerol, galactose, <br />    Following plasmid shuffling, ability Ala mutants         maltose, sucrose), nonspecific growth inhibitors (<span id='am-184' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-185' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>- <br /> support growth assessed. With exception mutant                 mide, sorbitol, caffeine, ethanol), inhibitors DNA or <br /> L183A, lethal, mutants supported growth              RNA synthesis (hydroxyurea, thiolutin, azauracil). The use <br /> room temperature (&#11011;22&#176;C), various extents. With               selective media growth tests identifies defects in <br /> the exception P86A, mutants grow                  components transcriptional apparatus (28, 29). More- <br /> 39&#176;C, control strain carrying mini-TAF25p                 over, mutations certain genes encoding SAGA subunits have <br /> (Fig. 4A). Wild-type, TAF25&#9004;N, TAF25&#9004;C, TAF25&#9004;N&#11001;&#9004;C,                       reported result differential growth properties in <br /> and G101E strains included controls. The lethality             tests (67). The results screens are <br /> strain L183A inactivity mutant             summarized Table 1. The different members Ala <br /> form TAF25p protein instability; mutant protein             mutant family behave distinct fashion disparate <br /> was undetectable Western blots (data shown). Strains               growth tests. Mutants I88A, L188A, G101A the <br /> expressing I88A L188A mutant forms TAF25p grew                 non-Ala mutant G101E showed enhanced growth defects on <br /> particularly slowly extreme temperatures (12, 22, 37,              media containing nonspecific growth inhibitors (caffeine, for- <br /> 39&#176;C) (Fig. 4A). Control strain G101E displayed low-             mamide, ethanol). I88A exhibited little growth on <br /> high-temperature-sensitive growth (Fig. 4A). Residue I88 (po-             media lacking inositol containing glycerol maltose a <br /> tential loop region) residue L188 (alpha <span id='am-111' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-114'  ></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span> three)             carbon source. Other strains (Q118A, K119A, D90A) <br /> 100% conserved TAF25p othologs map                   showed slight growth defects ethanol formamide <br /> putative HF region TAF25p (Fig. 1) (18). In addition            (Table 1). All residues located different portions <br /> marked growth deficiencies noted above, subtle defects               canonical HF motif. In aggregate, data suggested <br /> were apparent. Mutants D90A, G101A, D106A, Q118A,                    functionally distinct domains TAF25p exist that <br /> and K119A exhibited mild cold-sensitive growth pheno-               domains contribute differentially specific transcrip- <br /> type; patterns conditional growth observed               tional functions molecule. <br /> two genetic backgrounds (S288C W303) (data shown).                   Two hybrid screens identify TFIID subunit, SAGA sub- <br /> Protein stability does appear explain variable growth          unit, transacting factors TAF25p-interacting partners. <br /> properties, steady-state levels Ala mutant proteins         To identify potential TAF25p-interacting proteins, per- <br /> approximately (Fig. 4B). Collectively, results             formed sets two-hybrid screens (17) using Gal4 <br /> indicated residues encoded conserved core              DBD-TAF25p LexA DBD-TAF25p bait molecules two <br />  VOL. 21, 2001                                                                            TAF25p STRUCTURE-FUNCTION ANALYSIS                         6673 <br />  <br />  <br />  <br />  <br />    FIG. 4. Phenotypes yeast strains expressing wild-type TAF25 mutant alleles TAF25. (A) TAF25 haploid yeast strains carrying the <br /> indicated wild-type (WT) mutant TAF25 alleles plated YPAD media order test growth different temperatures. Serial fivefold <br /> dilutions cells plated. The 12, 22, 25, 30, 37, 39&#176;C plates photographed 330, 66, 66, 46, 66, 66 h incubation, respectively. <br /> (B) Steady-state levels TAF25p produced various alleles TAF25. TAF25 haploid shuffled yeast cells carrying indicated HA3-tagged <br /> mutant forms mini-TAF25 grown selective media mid-log phase, equivalent amounts cells (optical density 600 nm, &#11011;5) were <br /> harvested centrifugation frozen &#11002;80oC. Protein extracts prepared frozen cells fractionated sodium dodecyl <br /> sulfate-polyacrylamide gel electrophoresis, HA3-TAF25p content measured immunoblotting using peroxidase-conjugated anti-HA <br /> MAb (Roche). <br />  <br />  <br />  <br /> different yeast strains different yeast genomic                   proteins identified (PRP22, CLP1, PRP40). The <br /> DNA Gal4p-AD fusion libraries; tested total &#11011;1.12 &#11003;                    specificity physiological significance classes <br /> 107 yeast colonies. The DNA sequences 209 clones                      genes remain authenticated. <br /> met appropriate selective criteria determined. Several                    Overexpression selected TFIID SAGA subunits <br /> the hits assay notable. First, TFIID (TAF47)                  weakly suppresses temperature-sensitive growth distinct <br /> and SAGA (SPT7) subunit-encoding genes identified.                        TAF25 Ala mutants. We tested effects overexpression of <br /> This result expected, TAF25p integral                     TFIID SAGA subunits interact TAF25p <br /> subunit complexes. Interestingly, TAF25p-TAF47p                        (TAF47p, TAF65p, Spt7p) (19) ability the <br /> and TAF25p-Spt7p protein-protein interactions ob-                    TAF25 Ala mutant family grow nonpermissive tempera- <br /> served recently Gangloff et al. (19), used directed                     tures. In addition, tested overexpression TAF40p, it <br /> two-hybrid screens DBD-TAF25p TAF47p-AD                           identified putative TAF25p interactor (71). <br /> Spt7p-AD fusions demonstrate interactions                     As expected, TAF25 vector suppressed temper- <br /> HF-containing proteins. These results corroborate nondi-                   ature-sensitive growth temperature-sensitive strains (Fig. <br /> rected two-hybrid interactions reported here. A second feature                 5). We used ADA1 specificity control, encodes an <br /> of analyses genes encoding known                      HF motif SAGA-specific subunit does physically inter- <br /> putative transcriptional regulators (ASK10, UME6, YPR115W,                     act TAF25p (19). Overexpression TAF25 effect <br /> TAO3, RIM15, RLM1, ECM22, RAP1, YAP6) iden-                           growth of strains. Indeed, situ- <br /> tified. Finally, genes encoding components RNA processing                   ation genes examined, exception SPT7. <br />  6674       KIRCHNER ET AL.                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />                                                 TABLE 1. Phenotypic analyses TAF25 mutant familya <br />                                                                                            Growth on: <br />        Strain <br />                          MAL        GAL       GLYC         SUC       SORB       &#11002;MET        &#11002;INO        AZA-U       CAFF       THIO       FORM         ETH       HYD-U <br />  <br /> Wild type               &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> G101E                   &#11001;&#11001;         &#11001;&#11001;         &#11001;&#11001;          &#11001;&#11001;         &#11001;&#11001;         &#11001;&#11001;          &#11001;&#11001;          &#11001;&#11001;          &#11001;          &#11001;          &#11001;           &#11001;           &#11001;&#11001; <br /> TAF25&#9004;N                 &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> TAF25&#9004;C                 &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> TAF25&#9004;N &#11001; &#9004;C            &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> P86A                    &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> I88A                    &#11006;          &#11001;          &#11006;           &#11001;&#11001;         &#11001;          &#11001;           &#11006;           &#11001;           &#11006;          &#11001;          &#11001;           &#11006;           &#11001; <br /> P89A                    &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> D90A                    &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;          &#11001;&#11001;          &#11001;&#11001;&#11001; <br /> L97A                    &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> G101A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;         &#11001;&#11001;&#11001;        &#11006;           &#11006;           &#11001;&#11001;&#11001; <br /> D106A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> R108A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> Q118A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;          &#11001;&#11001;          &#11001;&#11001;&#11001; <br /> K119A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;          &#11001;&#11001;          &#11001;&#11001;&#11001; <br /> D123A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> D127A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> T184A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> L188A                   &#11001;&#11001;         &#11001;&#11001;         &#11001;&#11001;          &#11001;&#11001;         &#11001;&#11001;         &#11001;&#11001;          &#11001;&#11001;          &#11001;&#11001;          &#11001;          &#11001;&#11001;         &#11002;           &#11001;           &#11001;&#11001; <br /> E194A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> Y195A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> G196A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br /> P202A                   &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;        &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001;         &#11001;&#11001;&#11001; <br />   Haploid yeast strains carrying indicated TAF25 alleles plated various solid media supplemented different compounds, deficient certain growth <br /> component, containing inhibitors order test growth. Serial fivefold dilutions cells plated. Growth assessed qualitatively compared the <br /> the isogenic wild-type strain 90 h (in presence maltose [MAL], galactose [GAL], glycerol [GLYC], sucrose [SUC], sorbitol [SORB]; absence of <br /> methionine [&#11002;MET] inositol [&#11002;INO]; presence azauracil [AZA-U], caffeine [CAFF], thiolutin [THIO]) 100 h (in presence formamide <br /> [FORM], ethanol [ETH], hydroxyurea [HYD-U]) inoculation. Growth ranged little growth (&#11006;) wild-type growth (&#11001;&#11001;&#11001;). &#11002;, growth. <br />  <br />  <br />  <br />  <br /> Overexpression gene reproducibly induced slow-                                analyses performed using independent sets yeast cul- <br /> growth phenotype strains, permissive growth                           tures subjected temperature shift regimen outlined <br /> conditions (Fig. 5 data shown). Cooverexpression                             above. Values deemed different (decreased &#11011;2-fold in- <br /> TAF25 reversed, dose-dependent fashion, dominant-                              creased &#11011;50%) control strain TAF25&#9004;N&#11001; <br /> negative growth effect SPT7 overexpression (data shown).                         &#9004;C indicated bold type Table 2. From data it <br /> The molecular mechanism phenomenon unknown                                seen total poly(A)&#11001; mRNA synthesis dramat- <br /> present. Unfortunately, however, dominant-negative effect                          ically reduced (down 78%) control strain G101E was <br /> of SPT7 prevented using overexpression-suppres-                             shifted nonpermissive temperature, consistent pre- <br /> sion approach effect map TFIID-specific SAGA-                            vious results (60). Total poly(A)&#11001; mRNA synthesis also <br /> specific domains TAF25p. Overexpression TAF47,                                decreased substantially Ala mutant family <br /> TAF40, TAF65 weakly suppressed temperature-sensi-                               members, particularly mutant I88A and, lesser extent, <br /> tive phenotype particular members (P89A, D90A, L97A,                                 mutants G101A, D123A, L188A, temperature shift. <br /> K119A, D123A, T184A, Y195A, G196A) TAF25                                     These data consistent idea TAF25p function <br /> Ala mutant family. Although suppression cases                               integral high-level mRNA gene transcription (43, 60). <br /> weak, reproducibly suppression levels con-                                To rigorously test hypothesis TAF25p func- <br /> ferred ADA1 control.                                                             tion required mRNA synthesis, examined specific <br />    TAF25 Ala mutant family members display differential ef-                             mRNA gene transcription mutant family RNA blot- <br /> fects mRNA gene transcription vivo. The two-hybrid                              ting. Three classes genes chosen hybridization <br /> data, data Table 1 Fig. 5, strongly sug-                          probes based recently published gene array transcrip- <br /> gested TAF25p contributed substantially RNA poly-                               tion data Lee colleagues (see Table 3 reference 43 <br /> merase II-catalyzed mRNA gene transcription. Such results                               website web.wi.mit.edu/young/pub/expressionanalysis <br /> are consistent previously published observations (37, 43,                          .html). The comprised genes transcription re- <br /> 60). Given results, believed important examine                           ported uniquely generally dependent TFIID <br /> mRNA gene transcription Ala mutant collection                                (PMT4, WSC2, TRX1). The second composed genes <br /> therefore tested total gene-specific mRNA syntheses.                           transcription reported uniquely depend upon <br />    The results transcription analyses presented                            SAGA (YHB1, PHO84, YDL124W, YJL012C). The third <br /> in Table 2. The numbers refer ratio number                             represented genes transcription reduced un- <br /> transcripts detected cultures grown restrictive temper-                         defined temperature-sensitive taf25 strain nonpermissive <br /> ature 39&#176;C 1 h value control culture grown                        temperature 45 min (GIC2, YOR248W, FAS2). An <br /> 22&#176;C. These data represent averages independent                            example results RNA blotting analyses pre- <br />  VOL. 21, 2001                                                                        TAF25p STRUCTURE-FUNCTION ANALYSIS                       6675 <br />  <br />  <br />  <br />  <br />    FIG. 5. Suppression temperature-sensitive growth patterns yeast TAF25 mutants overexpression genes encoding TFIID SAGA <br /> subunits. (A) Yeast strains bearing alleles TAF25 transformed 2&#65533;?&#65533;m URA3-marked plasmids carrying indicated TFIID SAGA <br /> subunit-encoding genes. Serial fivefold dilutions cells plated SC media incubated permissive (30&#176;C) restrictive (37 <br /> to 39&#176;C) temperature. Growth restrictive temperature (37, 38, 39&#176;C, appropriate) scored qualitatively comparison the <br /> pRS426-TAF25 positive control represented &#11001; &#11001;&#11001;&#11001;&#11001; (strains carrying 2&#65533;?&#65533;m TAF25); &#11002;, growth. Note strains carrying 2&#65533;?&#65533;m <br /> SPT7 displayed slow-growth phenotype temperatures examined compared strains carrying 2&#65533;?&#65533;m vector. (B) <br /> Suppression analysis. Strain 97A transformed indicated plasmids, serial fivefold dilutions plated selective SC media at <br /> either 30 37&#176;C 90 h imaged. <br />  <br />  <br />  <br /> sented Fig. 6, YOR248W mRNA levels exam-                      shared TFIID SAGA. We number of <br /> ined; overall data summarized Table 2. These analyses                important observations regarding TAF25p function. <br /> were performed triplicate, values shown Table 2                  Inactivation TAF25pG101E dramatically reduces polymer- <br /> represent averages independent experiments.                    ase II-mediated gene transcription vivo complete <br /> Analysis data indicates TAF25 mutant family                 disruption TFIID SAGA. It hypothesized that <br /> displays variety gene-specific defects transcription          basically types TAFps regard tran- <br /> often vary dramatically allele allele transcript             scriptional dependencies. One <span id='am-116' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-117' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> TAFps induce a <br /> transcript.                                                                 relatively large drop total poly(A)&#11001; mRNA synthesis when <br />                                                                             functionally inactivated. This group includes HF motif- <br />                            DISCUSSION                                       containing TAFps: TAF17p, TAF25p, TAF40p, TAF60p, and <br />                                                                             TAF61p. At TAF17p, large drop high-level <br />   In report efforts systematically define           poly(A)&#11001; mRNA gene transcription translates corre- <br /> the structure-function relationships TAF25, single-copy                spondingly large drop gene-specific mRNA gene transcrip- <br /> essential gene encodes HF motif-containing subunit                 tion (2). The group TAFps comprised that <br />  6676        KIRCHNER ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />                                               TABLE 2. Transcription analyses TAF25 mutant familya <br />  <br />                        Total poly (A)&#11001;                             SAGA                                         TFIID                               TAF25p <br />        Strain <br />                            mRNA              YHB1        PHO84        YJL012C        YDL124W         PMT4       WSC2        TRX1        GIC2       YOR248W         FAS2 <br />  <br /> Wild type                    124              123          394          101             233           82          90          44         92            52           96 <br /> G101E                         22               22          240           87             160           37          36          54         86            26           37 <br /> TAF25&#9004;N                       87               26          334           47             168           40          64          45         82            34           31 <br /> TAF25&#9004;C                       98               42          422           64             136           55          72          45         63            63           37 <br /> TAF25&#9004;N&#11001;&#9004;C                    80               62          324           61             235           85          77          56         59           107           57 <br /> 86A                           77               37          228           71             153           50          43          36         76            26           55 <br /> 88A                           35               53           89           51             285           41          30          50         75            33           56 <br /> 89A                           79               45          166           59             102           62          38          42         53            38           40 <br /> 90A                           78               28          315           60             265           64          38          48         78            37           37 <br /> 97A                           70               52          358           70             277           56          32          31         54            57           64 <br /> 101A                          52               50          139           56             331           26           2          48         52            24           32 <br /> 106A                          71               36          166           71              97           36          31          46         46            37           29 <br /> 108A                          81               44          270           59             224           46          60          47         47            45           28 <br /> 118A                          78               39          205           64             121           52          60          35         55            46           35 <br /> 119A                          63               43          220           50             267           45          25          40         35            32           32 <br /> 123A                          52               32          286           56             135           41          26          47         17            38           37 <br /> 127A                          62               59          363           66             197           48          74          41         63            35           26 <br /> 184A                          83               56          236           68             161           62          43          40         59            41           32 <br /> 188A                          54               21          342           67              60           48           3          22         24             9           12 <br /> 194A                          81               56          124           68             275           65          53          46         52            34           36 <br /> 195A                          59               21          435           51              97           24          27          27         48            30           27 <br /> 196A                          72               30          281           77             145           34          34          32         63            57           49 <br /> 202A                          83               43          235           61             124           38          67          41         55            43           34 <br />    a <br />      All analyses performed times using independent cultures indicated yeast strains. RNA directly blotted probed total <br /> poly(A)&#11001; mRNA hybridization 32P-labeled oligo(dT)20 fractionated denaturing agarose gel electrophoresis, blotted, detected indicated <br /> 32 <br />    P-labeled, PCR-generated gene-specific hybridization probes detailed Materials Methods. TAF25&#9004;N&#11001;&#9004;C relevant control strain. Values were <br /> at 50% increased 2-fold decreased relative control strain bold type; text details. The values shown represent means for <br /> three independent analyses; standard errors &#65533;?5% indicated values total poly(A)&#11001; mRNA analyses &#65533;?20% indicated values RNA blotting <br /> experiments. The classes genes used transcription analyses chosen based data Lee et al. (43). For TFIID-dependent genes PMT4, <br /> WSC2, TRX1, transcription reduced temperative-sensitive taf145(130) strain 45 min nonpermissive temperature 3.2-, 4.8-, 5.3- fold (mRNA <br /> half-lives 10, 11, 18 min), respectively. For SAGA-dependent genes YHB1, PHO84, YDL124W, YJL012C, transcription reduced gcn5 null strain <br /> 4.1-, 20.1-, 3.5-, 3.0- fold (half-lives 6, 7, 14, 14 min), respectively. For TAF25-dependent genes GIC2, YOR248W, FAS2 transcription reduced <br /> 4.2-, 4.1-, 3.1- fold (half-lives 36, 13 13). <br />  <br />  <br />  <br />  <br /> do contain HF motifs. When TAFps inactivated,                            motif-containing TAFps inactivated, TFIID (and/or <br /> they fail induce large decrease high-level total poly(A)&#11001;                      SAGA) complex rapidly totally disrupted, resulting in <br /> mRNA synthesis; include TAF150p, TAF130/145p, TAF90p,                             TAFp degradation (2, 25, 48, 50, 51, 52, 59, 60). This degra- <br /> and TAF67p. One possible explanation differences                            dation fact HF motif-containing <br /> transcriptional output inactivation HF                           TAFps fulfill key structural roles TFIID SAGA. <br />  <br />  <br />  <br />  <br />   FIG. 6. Transcriptional profiles yeast strains expressing wild-type mutant alleles TAF25. Specific mRNA gene transcription was <br /> measured using denaturing agarose gel electrophoresis, blotting, hybridization 32P-labeled gene-specific probe derived YOR248W. <br /> Prior hybridization, blot stained 0.01% (wt/vol) methylene blue detect rRNAs (the loading control) 4.40-, 2.37-, 1.35-kb <br /> RNA size markers (bottom panel). YOR248W mRNA content detected autoradiography phosphorimaging (see Table 2). RNA samples <br /> were loaded pairs&#8212;RNA cells grown permissive (P) temperature 22&#176;C restrictive (R) temperature 39&#176;C (left right lanes, <br /> respectively, bar)&#8212;for strains shown. Phosphorimaging counts samples 22 39&#176;C, respectively, follows: <br /> &#9004;N (164,949 56,082), &#9004;C (151,639 103,114), &#9004;N&#11001;&#9004;C (123,804 115,840), wild type (WT) (48,114 21,494), temperature-sensitive <br /> G101E (G101Ets) (146,270 31,543), 86A (126,730 38,422), 88A (148,997 42,889), 89A (178,012 80,655), 90A (98,835 36,172), <br /> 97A (123,489 69,778), 101A (55,756 15,984), 106A (130,987 50,679), 108A (157,686 71,411), 118A (91,983 50,313), 119A <br /> (106,343 40,123), 123A (108,767 45,670), 127A (177,102 67,832), 184A (114,814 44,283), 188A (68,941 6,099), 194A (304,320 <br /> and 120,734), 195A (290,872 106,468), 196A (134,893 72,202), 202A (218,445 84,486). <br />  VOL. 21, 2001                                                                 TAF25p STRUCTURE-FUNCTION ANALYSIS                   6677 <br />  <br />  <br /> Although ability induce rapid loss TFIID useful     conserved N-terminal portion TAF25p play role <br /> phenomenon studying TFIID biology, mutants in-             function TFIID SAGA both. <br /> duce rapid complex dissociation useful biochemical         TAF25p interacts transcription factors with <br /> and genetic experimentation aimed elucidating molec-          TFIID SAGA constituents. One popular models for <br /> ular mechanisms action multisubunit complexes.           TAF function, originally formulated Chen colleagues <br /> Therefore, excited molecularly dissecting       (10) Goodrich colleagues (23), posits TAFps <br /> TAF25 mechanism mutation inacti-          function coactivators receptors specifically and <br /> vate vivo function(s) TFIID and/or SAGA disrup-          positively interact activation domains DNA-bound <br /> tion. We consequently carefully analyzed effects tem-       transactivator proteins. Such interactions thought stabi- <br /> perature shift complex integrity temperature-            lize (recruit) TFIID cis-linked promoter, stim- <br /> sensitive allele TAF25 (60).                                      ulating PIC formation transcription. This model is <br />    We used cells carrying taf25G101E allele, mutation          supported extensive body experimental data derived <br /> which causes temperature-sensitive growth dra-         vitro vivo studies performed metazoans. <br /> matically reduces RNA polymerase II-mediated mRNA tran-              Whether model applicable S. cerevisiae unclear. <br /> scription vivo (60), SAGA TFIID coimmunoprecipi-             In attempt address question TAF25p <br /> tation studies. It appears primary defect causing       interact directly transactivator proteins, per- <br /> precipitous drop transcription complete complex dis-       formed two-hybrid screens using LexA-TAF25p or <br /> sociation se (Fig. 2) (60), transcription decreased   Gal4p-TAF25p DBD baits. A number putative TAF25p- <br /> greater extent complex dissociation 30 min postshift.        interacting proteins identified, were <br /> This observation argues TAF25pG101E inactivated           TAF47p Spt7p, proteins integral subunits of <br /> situ TFIID and/or SAGA inactivation          TFIID SAGA, respectively. We identified genes en- <br /> responsible primary vivo decrement transcription.      coding number known transcription factors&#8212;ASK10, <br /> If correct, data suggest TAF25p makes          UME6, YPR115W, TAO3, RIM5, RLM1, ECM22, RAP1, and <br /> multiple, positively acting protein-protein interactions      YAP6&#8212;suggesting transcription factors oper- <br />                                                                      ate AD-TAFp interaction-coactivator mechanism. An <br /> or multisubunit assemblies contribute crit- <br />                                                                      intriguing additional class hits observed screens was <br /> ically ongoing high-level RNA polymerase II transcription. <br />                                                                      group genes (proteins) involved RNA processing <br /> Indeed, two-hybrid screens recent work of <br />                                                                      (PRP22, CLP1, PRP40). RNA processing coupled <br /> Gangloff et al. (19) support hypothesis. TAF25p interacts <br />                                                                      transcription TFIID C-terminal do- <br /> specifically, HF motif, complementary HF <br />                                                                      main RNA polymerase II (14, 31). It interesting if <br /> domains TAF47p, TAF65p, Spt7p, subunits either <br />                                                                      TAF25p TFIID (or SAGA?) contributed specific con- <br /> TFIID SAGA. These observations add under- <br />                                                                      tact surfaces RNA processing enzymes proteins <br /> standing TAF25p. More importantly, though, data <br />                                                                      help localize proteins active genes. It will <br /> encouraged analyze structure-function relationships <br />                                                                      important follow putative TAF25p-interact- <br /> of molecule detail. <br />                                                                      ing proteins determine physiologically relevant <br />    Less half TAF25 ORF required produce          and, so, determine make direct versus indirect <br /> mini-TAF25p capable supporting yeast cell growth. To gain         contacts TAF25p. It informative elucidate <br /> greater insight TAF25p, performed additional anal-        context, TFIID, SAGA, both, contacts are <br /> ysis hydroxylamine-generated mutant family          made. <br /> produced yeast strain YEK25.75 (60). We searched hydrox-            Distinct TAF25p residues contribute differentially gene- <br /> ylamine-induced mutations conferring lethality. Such muta-           specific transcription. We created family alanine-scanning <br /> tions readily identified mapped sequences distrib-       mutants TAF25 generate reagents use <br /> uted TAF25 ORF. These data indicated             assess contributions individual amino acid residues <br /> nearly entire length TAF25 mutable (Fig. 1); hence,       TAF25p function molecule. Of 19 ami- <br /> sequences TAF25p contribute func-            acids targeted, L183A mutant was <br /> tion.                                                                lethal. The 19 amino acids subjected mutation represent 18 <br />    The idea entire length TAF25p contributed      amino acids 100% conserved (excluding <br /> function tested directly complementary ways: first,       A128) seven orthologs aligned (Fig. 1). Moreover, <br /> by sequence alignments coupled deletion mutagenesis,            19 amino acids represent 54% invariant amino <br /> and second, site-directed alanine-scanning mutagenesis.           acid residues conserved seven orthologs compared. <br /> Our deletion mutagenesis studies shown 96         Some targeted residues map structured <br /> amino acids comprise evolutionarily conserved portions         regions putative histone fold domain TAF25p (Fig. 1). <br /> of TAF25p required support yeast cell growth. Indeed,         The residues chosen mutagenesis, span length of <br /> this conserved region supports wild-type levels growth (Fig.      mini-TAF25p, comprise representative sampling potential <br /> 3B). However, deletion nonconserved residues, amino           protein-protein interaction surfaces molecule. <br /> acids 2 73 amino acids 142 179, effect.        Members family display distinct phenotypes cold <br /> Strains expressing &#9004;N &#9004;N&#11001;&#9004;C deletion forms                sensitivity, temperature sensitivity, differential susceptibil- <br /> TAF25p display different total poly(A)&#11001; mRNA levels              ity inhibitors. We reasoned these <br /> exhibit different patterns specific transcript levels compared    phenotypes caused defects transcription. We there- <br /> to wild type (Table 2). These results suggest non-      fore performed series direct tests transcription the <br />  6678     KIRCHNER ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> mutant cells. We analyzed total poly(A)&#11001; mRNA synthe-             plex-specific functions TFIID SAGA. The puta- <br /> sis gene-specific mRNA production cells grown             tive TAF25p-dependent genes, GIC2, YOR248W, FAS2, <br /> permissive conditions 60 min shift nonper-          chosen serve controls, transcription these <br /> missive temperature. Others carried gene array stud-          genes reported depend directly TAF25p function, <br /> ies assessed effect yeast TAF protein inactivation        displaying 4.2-, 4.1-, 3.1-fold decreases (with excellent con- <br /> upon mRNA gene transcription (32, 43). In studies,               fidence) (43) TAF25p inactivated temperature <br /> single temperature-sensitive alleles genes question          shift. Analysis genes provided crucial test <br /> were used, stated instances, an-          assumption gene array analyses one <br /> other temperature-sensitive allele scored similarly assays      temperature-sensitive TAF allele equals transcriptional <br /> (43). The actual nucleotide amino acid lesions cog-          functions. <br /> nate genes proteins reported.                                 With proviso limited sampling genes been <br />    We goals generating family TAF25 Ala              analyzed, data argue problematic use single <br /> mutants. The prepare group mutants             temperature-sensitive TAF mutant define totality tran- <br /> might allow identify domains TAF25p primarily           scriptional functions molecules, TAF25. Of <br /> contributed TFIID function, SAGA function, func-           putative TAF25p-dependent genes an- <br /> tions multisubunit complexes. The second use            alyzed detail, behaved predicted a <br /> this panel reagents determine single temperature-         bona fide TAFp-dependent gene. YOR248W transcription in <br /> sensitive mutant allele TAF25 entirety        cells expressing L97A G196A TAF25 alleles was <br /> the transcriptional functions TAF25p, implied Lee         reduced nonpermissive temperature (range, 2.3- 11.9- <br /> colleagues (43). Our success achieving goals          fold). In contrast, examined GIC2 FAS2 expres- <br /> mixed, finite number transcripts         sion Ala mutant family, mutant alleles <br /> could queried using RNA blotting methods                did decrease transcription temperature shift, <br /> gene array technology. However, number conclusions            transcription essentially unaffected alleles. <br /> be drawn transcription experiments (Table 2). First,          For GIC2, 3 18 mutants displayed decreased transcription at <br /> analysis total poly(A)&#11001; mRNA synthesis clearly indicated            nonpermissive temperature, FAS2, 5 18 <br /> that conditional cell growth does relate readily overall        mutants showed decrease transcription. The data obtained <br /> RNA polymerase II transcription. Second, depending                YOR248W rule possibility al- <br /> which strain chooses baseline (wild type, TAF25&#9004;N,            leles compromised sufficiently display decrease spe- <br /> TAF25&#9004;C, TAF25&#9004;N&#11001;&#9004;C), actual percent decrease                cific gene transcription temperature shift. <br /> mRNA synthesis temperature shift vary. Since               The situation clouded the <br /> mutants constructed TAF25&#9004;N&#11001;&#9004;C background,                 SAGA-dependent TFIID-dependent genes consid- <br /> any values considered different (decreased &#11011;2-fold in-           ered. Transcription TRX1 YJL012C genes essen- <br /> creased &#11011;50%) value control strain TAF25&#9004;N&#11001;            tially unaffected TAF25 mutation, transcription the <br /> &#9004;C shown bold type (note mutant G101E                  genes tested displays unique patterns TAF25 Ala <br /> present full-length TAF25 ORF background).                      mutant dependencies. It clear consideration of <br />    Our TAF25 mutant collection exhibited broad range          data TAF25p does contribute importantly to <br /> of total mRNA synthesis phenotypes. Mutants I88A, G101A,               high-level, ongoing mRNA gene transcription. However, at <br /> D123A, L188A displayed decreased total mRNA syn-               juncture, feasible ascribe complex-specific <br /> thesis levels (as did G101E control) (60); mutant I88A dis-        functions discrete domains residues protein. Such <br /> playing largest decrease total mRNA synthesis. The val-         mechanistic details require intensive investigation. <br /> ues mutants near TAF25&#9004;N&#11001;&#9004;C.             Conclusions. The studies described report argue that <br /> The fact high-level total poly(A)&#11001; mRNA values             TAF25p contributes importantly gene-specific transcription. <br /> reveal picture underscored transcription         Despite fact totally analyze TAF25p, <br /> of distinct mRNAs mutants analyzed. There clear          experiments dissected critical, shared TFIID and <br /> and large allele-specific variations gene-specific mRNA tran-       SAGA subunit. We shown TAF25p, <br /> scription members collection. For 10              observe large decrease transcription without <br /> genes studied, transcription YJL012C appears to-           corresponding loss TFIID SAGA complex integrity. <br /> tally unaffected mutation TAF25, genes       Finally, shown distinct temperature-sensitive al- <br /> at different alleles affect transcription. In aggregate,   leles TAF25 display differential unique patterns gene- <br /> these assays clearly argue individual TAF25          specific transcription vivo. Our future studies aimed <br /> mutant allele totality transcriptional         characterizing novel TAF25p protein-protein interac- <br /> repertoire protein.                                             tions described utilizing collection mu- <br />    The genes chose RNA blotting experiments            tants additional detailed functional tests the <br /> were selected implicated (43)           molecular mechanisms TAF25p functions RNA <br /> dependent TFIID (PMT4, WSC2, TRX1) SAGA                    polymerase II-catalyzed mRNA gene transcription the <br /> (YHB1, PHO84, YJL012C, YDL124W) uniquely depen-                 context TFIID SAGA. <br /> dent TAF25p function (GIC2, YOR248W, FAS2). <br /> We reasoned TFIID SAGA dependencies                                        ACKNOWLEDGMENTS <br /> were correct, panel genes allow assign         First, thank laboratory colleagues freely sharing <br /> residues TAF25p contributing shared and/or com-           reagents, strains, constructive criticism through- <br />  VOL. 21, 2001                                                                                   TAF25p STRUCTURE-FUNCTION ANALYSIS                              6679 <br />  <br />  <br /> out course work. Their input better                 22. Gill, G., E. Pascal, Z. H. Tseng, R. Tjian. 1994. A glutamine-rich <br /> study. Second, acknowledge Erin Baldwin&#8217;s contributions                        hydrophobic patch transcription factor Sp1 contacts dTAF110 com- <br /> overexpression complementation studies presented here. Third,                         ponent Drosophila TFIID complex mediates transcriptional ac- <br /> thank Steve Buratowski Fred Winston kindly supplying yeast                       tivation. Proc. Natl. Acad. Sci. USA 91:192&#8211;196. <br />                                                                                      23. Goodrich, J. A., T. Hoey, C. J. Thut, A. Admon, R. Tjian. 1993. Dro- <br /> expression plasmids experiments. Finally, thank Laszlo <br />                                                                                          sophila TAF40 interacts VP16 activation domain basal <br /> Tora, <span id='am-6' about='obo:NCBITaxon_146270' typeof='owl:Thing obo:NCBITaxon_6514 obo:NCBITaxon_33208 obo:NCBITaxon_216331 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_146272 obo:NCBITaxon_33154 obo:NCBITaxon_680346 obo:NCBITaxon_1206795 obo:NCBITaxon_70849 obo:NCBITaxon_216305 obo:NCBITaxon_216307 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-7' property="rdfs:label" content="Doris" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>Doris</span> Kirschner, Gael Gangloff, Irwin Davidson freely                      transcription factor TFIIB. Cell 75:519&#8211;530. <br /> sharing ideas information regarding TAF25 histone folds                24. Grant, P. A., L. Duggan, J. Cote, S. M. Roberts, J. E. Brownell, R. Candau, <br /> with us.                                                                                 R. Ohba, T. Owen-Hughes, C. D. Allis, F. Winston, S. L. Berger, J. L. <br />    This work supported NIH grant GM52461. J.K. received                           Workman. 1997. Yeast Gcn5 functions multisubunit complexes to <br /> partial support NIH training grant T32 CA09385. Sequencing                       acetylate nucleosomal histones: characterization Ada complex the <br /> <span id='am-80' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span><span id='am-87'  ></span>Candida albicans</span> accomplished support NIDR                       SAGA (Spt/Ada) complex. Genes Dev. 11:1640&#8211;1650. <br /> the Burroughs Wellcome Fund.                                                         25. Grant, P. A., D. Schieltz, M. G. Pray-Grant, D. J. Steger, J. C. Reese, J. R. <br />                                                                                          Yates, J. L. Workman. 1998. A subset TAF(II)s integral compo- <br />                                                                                          nents SAGA complex required nucleosome acetylation tran- <br />                                  REFERENCES                                              scriptional stimulation. Cell 94:45&#8211;53. <br />  1. Albright, S. R., R. Tjian. 2000. TAFs revisited: data reveal new        26. Gregory, P. D., A. Schmid, M. Zavari, M. Munsterkotter, W. Horz. 1999. <br />     twists confirm old ideas. Gene 242:1&#8211;13.                                         Chromatin remodelling PHO8 promoter requires SWI-SNF and <br />  2. Apone, L. M., C. A. Virbasius, F. C. Holstege, J. Wang, R. A. Young,             SAGA step subsequent activator binding. EMBO J. 18:6407&#8211;6414. <br />     M. R. Green. 1998. Broad, universal, transcriptional requirement     27. Guthrie, C., G. R. Fink (ed.). 1991. Guide yeast genetics molec- <br />     yTAF17, histone H3-like TAF present TFIID SAGA. Mol. Cell 2:                ular biology. Academic Press, Inc., New York, N.Y. <br />     653&#8211;661.                                                                         28. Hampsey, M. 1997. A review phenotypes Saccharomyces cerevisiae. <br />  3. Bai, Y., G. M. Perez, J. M. Beechem, P. A. Weil. 1997. Structure-function        Yeast 13:1099&#8211;1133. <br />     analysis TAF130: identification characterization high-affinity       29. Hampsey, M. 1998. Molecular genetics RNA polymerase II general <br />     TATA-binding protein interaction domain N terminus yeast                   transcriptional machinery. Microbiol. Mol. Biol. Rev. 62:465&#8211;503. <br />     TAF(II)130. Mol. Cell. Biol. 17:3081&#8211;3093.                                       30. Henry, N. L., A. M. Campbell, W. J. Feaver, D. Poon, P. A. Weil, R. D. <br />  4. Bell, B., L. Tora. 1999. Regulation gene expression multiple forms         Kornberg. 1994. TFIIF-TAF-RNA polymerase II connection. Genes Dev. 8: <br />     TFIID novel TAFII-containing complexes. Exp. Cell Res. 246:             2868&#8211;2878. <br />     11&#8211;19.                                                                           31. Hirose, Y. J. L. Manley. 2000. RNA polymerase II integration <br />  5. Birck, C., O. Poch, C. Romier, M. Ruff, G. Mengus, A. C. Lavigne, I.                 nuclear events. Genes Dev. 14:1415&#8211;1429. <br />     Davidson, D. Moras. 1998. Human TAF(II)28 TAF(II)18 interact             32. Holstege, F. C., E. G. Jennings, J. J. Wyrick, T. I. Lee, C. J. Hengartner, <br />     histone fold encoded atypical evolutionary conserved motifs             M. R. Green, T. R. Golub, E. S. Lander, R. A. Young. 1998. Dissecting <br />     SPT3 family. Cell 94:239&#8211;249.                                      regulatory circuitry eukaryotic genome. Cell 95:717&#8211;728. <br />  6. Boeke, J. D., J. Trueheart, G. Natsoulis, G. R. Fink. 1987. 5-Fluoroorotic   33. <span id='am-169' about='obo:NCBITaxon_1269045' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_6427 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41799 obo:NCBITaxon_41798 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6341 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6340 obo:NCBITaxon_6072'><span id='am-170'  ></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="Ikeda" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ikeda</span>, K., D. J. Steger, A. Eberharter, J. L. Workman. 1999. Activation <br />     acid selective agent yeast molecular genetics. Methods Enzymol.              domain-specific general transcription stimulation native histone <br />     154:164&#8211;175.                                                                         acetyltransferase complexes. Mol. Cell. Biol. 19:855&#8211;863. <br />  7. Buratowski, S., S. Hahn, L. Guarente, P. A. Sharp. 1989. Five interme-       34. James, P., J. Halladay, E. A. Craig. 1996. Genomic libraries host <br />     diate complexes transcription initiation RNA polymerase II. Cell 56:           strain designed highly efficient two-hybrid selection yeast. Genetics <br />     549&#8211;561.                                                                             144:1425&#8211;1436. <br />  8. Burke, T. W., J. T. Kadonaga. 1996. Drosophila TFIID binds              35. Kim, Y. J., S. Bjorklund, Y. Li, M. H. Sayre, R. D. Kornberg. 1994. A <br />     conserved downstream basal promoter element present                  multiprotein mediator transcriptional activation interaction with <br />     TATA-box-deficient promoters. Genes Dev. 10:711&#8211;724.                                 C-terminal repeat domain RNA polymerase II. Cell 77:599&#8211;608. <br />  9. Burley, S. K. Roeder, R. G. 1996. Biochemistry structural biology     36. Kingston, R. E., G. J. Narlikar. 1999. ATP-dependent remodeling and <br />     transcription factor IID (TFIID). Annu. Rev. Biochem. 65:769&#8211;799.                    acetylation regulators chromatin fluidity. Genes Dev. 13:2339&#8211;2352. <br /> 10. Chen, J. L., L. D. Attardi, C. P. Verrijzer, K. Yokomori, R. Tjian. 1994.    37. Klebanow, E. R., D. Poon, S. Zhou, P. A. Weil. 1996. Isolation and <br />     Assembly recombinant TFIID reveals differential coactivator require-              characterization TAF25, essential yeast gene encodes RNA <br />     ments distinct transcriptional activators. Cell 79:93&#8211;105.                       polymerase II-specific TATA-binding protein-associated factor. J. Biol. <br /> 11. Chiang, C. M., R. G. Roeder. 1995. Cloning intrinsic human TFIID           Chem. 271:13706&#8211;13715. <br />     subunit interacts multiple transcriptional activators. Science 267:    38. Klemm, R. D., J. A. Goodrich, S. Zhou, R. Tjian. 1995. Molecular <br />     531&#8211;536.                                                                             cloning expression 32-kDa subunit human TFIID reveals <br /> 12. Choder, M. 1991. A general topoisomerase I-dependent transcriptional re-             interactions VP16 TFIIB mediate transcriptional activation. <br />     pression stationary phase yeast. Genes Dev. 5:2315&#8211;2326.                   Proc. Natl. Acad. Sci. USA 92:5788&#8211;5792. <br /> 13. Cosma, M. P., T. Tanaka, K. Nasmyth. 1999. Ordered recruitment            39. Koleske, A. J., R. A. Young. 1994. An RNA polymerase II holoenzyme <br />     transcription chromatin remodeling factors cell cycle- develop-         responsive activators. Nature 368:466&#8211;469. <br />     mentally-regulated promoter. Cell 97:299&#8211;311.                                    40. Kotani, T., K. Banno, M. Ikura, A. G. Hinnebusch, Y. Nakatani, M. Kawai- <br /> 14. Dantonel, J. C., K. G. Murthy, J. L. Manley, L. Tora. 1997. Transcription        chi, T. Kokubo. 2000. A role transcriptional activators antirepres- <br />     factor TFIID recruits factor CPSF formation 3&#11032; end mRNA.               sors autoinhibitory activity TATA box binding transcription <br />     Nature 389:399&#8211;402.                                                                  factor IID. Proc. Natl. Acad. Sci. USA 97:7178&#8211;7183. <br /> 15. Dikstein, R, S. Ruppert, R. Tjian. 1996. TAF250 bipartite protein       41. Krebs, J. E., M. H. Kuo, C. D. Allis, C. L. Peterson. 1999. Cell cycle- <br />     kinase phosphorylates basal transcription factor RAP74. Cell 84:            regulated histone acetylation required expression yeast HO gene. <br />     781&#8211;790.                                                                             Genes Dev. 13:1412&#8211;1421. <br /> 16. Dunphy, E. L., T. Johnson, S. S. Auerbach, E. H. Wang. 2000. Require-        42. Kutach, A. K., J. T. Kadonaga. 2000. The downstream promoter ele- <br />     ment TAF(II)250 acetyltransferase activity cell cycle progression. Mol.       ment DPE appears widely used TATA box Drosophila core <br />     Cell. Biol. 20:1134&#8211;1139.                                                            promoters. Mol. Cell. Biol. 20:4754&#8211;4764. <br /> 17. Fields, S., O. Song. 1989. A novel genetic detect protein-         43. Lee, T. I., H. C. Causton, F. C. Holstege, W. C. Shen, N. Hannett, E. G. <br />     protein interactions. Nature 340:245&#8211;246.                                            Jennings, F. Winston, M. R. Green, R. A. Young. 2000. Redundant roles <br /> 18. Gangloff, Y. G., C. Romier, S. Thualt, S. Werten, I. Davidson. 2001. The         TFIID SAGA complexes global transcription. Nature 405: <br />     histone fold key structural motif transcription factor TFIID. Trends         701&#8211;704. <br />     Biochem. Sci. 26:250&#8211;257.                                                        44. Lemon, B., R. Tjian. 2000. Orchestrated response: symphony tran- <br /> 19. Gangloff, Y. G., S. L. Sanders, C. Romier, D. Kirschner, P. A. Weil, L. Tora,        scription factors gene control. Genes Dev. 14:2551&#8211;2569. <br />     I. Davidson. 2001. Histone folds mediate selective heterodimerization     45. Macpherson, N., V. Measday, L. Moore, B. Andrews. 2000. A yeast taf17 <br />     yeast TAF(II)25 TFIID components yTAF(II)47 yTAF(II)65                  mutant requires Swi6 transcriptional activator viability shows <br />     SAGA component ySPT7. Mol. Cell. Biol. 21:1841&#8211;1853.                            defects cell cycle-regulated transcription. Genetics 154:1561&#8211;1576. <br /> 20. Gangloff, Y. G., S. Werten, C. Romier, L. Carre, O. Poch, D. Moras, I.       46. Massari, M. E, P. A. Grant, M. G. Pray-Grant, S. L. Berger, J. L. Workman, <br />     Davidson. 2000. The human TFIID components TAF(II)135 TAF(II)20                  C. Murre. 1999. A conserved motif present class helix-loop-helix <br />     yeast SAGA components Ada1 TAF(II)68 heterodimerize                   proteins activates transcription direct recruitment SAGA complex. <br />     form histone-like pairs. Mol. Cell. Biol. 20:340&#8211;351.                                Mol. Cell 4:63&#8211;73. <br /> 21. Georgieva, S., D. B. Kirschner, T. Jagla, E. Nabirochkina, S. Hanke, H.          47. Matangkasombut, O., R. M. Buratowski, N. W. Swilling, S. Buratowski. <br />     Schenkel, C. Lorenzo, P. Sinha, K. Jagla, B. Mechler, L. Tora. 2000.          2000. Bromodomain factor 1 corresponds missing piece yeast TFIID. <br />     Two novel Drosophila TAF(II)s homology human TAF(II)30                 Genes Dev. 14:951&#8211;962. <br />     differentially regulated development. Mol. Cell. Biol. 20:1639&#8211;       48. Michel, B., P. Kamarnitsky, S. Buratowski. 1998. Histone-like TAFs are <br />     1648.                                                                                essential transcription vivo. Mol. Cell 2:663&#8211;673. <br />  6680       KIRCHNER ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br /> 49. Mizzen, C. A., X. J. Yang, T. Kokubo, J. E. Brownell, A. J. Bannister, T.        64. Shen, W. C., M. R. Green. 1997. Yeast TAF(II)145 functions core <br />     Owen-Hughes, J. Workman, L. Wang, S. L. Berger, T. Kouzarides, Y. Na-                promoter selectivity factor, general coactivator. Cell 90:615&#8211;624. <br />     katani, C. D. Allis. 1996. The TAF(II)250 subunit TFIID histone       65. Smale, S. T., D. Baltimore. 1989. The &#8220;initiator&#65533;?as transcription con- <br />     acetyltransferase activity. Cell 87:1261&#8211;1270.                                       trol element. Cell 57:103&#8211;113. <br /> 50. Moqtaderi, Z., M. Keaveney, K. Struhl. 1998. The histone H3-like TAF         66. Sterner, D. E., S. L. Berger. 2000. Acetylation histones transcrip- <br />     broadly required transcription yeast. Mol. Cell 2:675&#8211;682.                 tion-related factors. Microbiol. Mol. Biol. Rev. 64:435&#8211;459. <br /> 51. Moqtaderi, Z., J. D. Yale, K. Struhl, S. Buratowski. 1996. Yeast <span id='am-124' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-125'  ></span><span id='am-126' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>homo</span>-       67. Sterner, D. E., P. A.Grant, S. M. Roberts, L. J. Duggan, R. Belotserkovskaya, <br />     logues higher eukaryotic TFIID subunits. Proc. Natl. Acad. Sci. USA 93:           L. A. Pacella, F. Winston, J. L. Workman, S. L. Berger. 1999. Functional <br />     14654&#8211;14658.                                                                         organization yeast SAGA complex: distinct components involved in <br /> 52. Natarajan, K., B. M. Jackson, E. Rhee, A. G. Hinnebusch. 1998. yTAF61            structural integrity, nucleosome acetylation, TATA-binding protein in- <br />     general role RNA polymerase II transcription required             teraction. Mol. Cell. Biol. 19:86&#8211;98. <br />     Gcn4p recruit SAGA coactivator complex. Mol. Cell 2:683&#8211;692.              68. Sullivan, S. A., L. Aravind, I. Makalowska, A. D. Baxevanis, D. Lands- <br /> 53. O&#8217;Brien, T., R. Tjian. 1998. Functional analysis human TAF250             man. 2000. The histone database: comprehensive WWW resource for <br />     N-terminal kinase domain. Mol. Cell 1:905&#8211;911.                                       histones histone fold-containing proteins. Nucleic Acids Res. 28:320&#8211; <br /> 54. O&#8217;Brien, T., R. Tjian. 2000. Different functional domains TAF250              322. <br />     modulate expression distinct subsets mammalian genes. Proc. Natl.          69. Syntichaki, P., I. Topalidou, G. Thireos. 2000. The Gcn5 bromodomain <br />     Acad. Sci. USA 97:2456&#8211;2461. <br />                                                                                          coordinates nucleosome remodelling. Nature 404:414&#8211;417. <br /> 55. Orphanides, G., T. Lagrange, D. Reinberg. 1996. The general transcrip- <br />                                                                                      70. Thut, C. J., J. L. Chen, R. Klemm, R. Tjian. 1995. p53 transcriptional <br />     tion factors RNA polymerase II. Genes Dev. 10:2657&#8211;2683. <br />                                                                                          activation mediated coactivators TAF40 TAF60. Science 267:100&#8211; <br /> 56. Pham, A. D., F. Sauer. 2000. Ubiquitin-activating/conjugating activity of <br />                                                                                          104. <br />     TAF250, mediator activation gene expression Drosophila. Science <br />                                                                                      71. Uetz, P., L. Biot, G. Cagney, T. A. Mansfield, R. S. Judson, J. R. Knight, D. <br />     289:2357&#8211;2360. <br />                                                                                          Lockshon, V. Narayan, M. Strinivasan, P. Pachart, A. Qureshi-Emill, Y. Li, <br /> 57. Poon, D., Y. Bai, A. M. Campbell, S. Bjorklund, Y. J. Kim, S. Zhou, R.  <br /> </body></html>