<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>PCI proteins eIF3e eIF3m define distinct translation initiation <br /> factor 3 complexes <br />  <br />                  Background: PCI/MPN domain protein complexes comprise 19S proteasome lid, COP9 <br />                  signalosome (CSN), eukaryotic translation initiation factor 3 (eIF3). eIF3 complex                  thought composed essential core subunits required global protein synthesis non- <br />                  essential subunits modulate mRNA specificity. Interactions unclear significance                  reported eIF3 subunits PCI proteins contained CSN. <br />                  Results:  report unexpected finding fission yeast distinct eIF3 complexes <br />                  sharing common core subunits, distinguished PCI proteins eIF3e novel eIF3m, <br />                  previously annotated putative CSN subunit. eIF3e eIF3m <br />                  contribute non-essential activities CSN cullin-RING ubiquitin ligase control, eif3m, <br />                  unlike eif3e, essential gene required global cellular protein synthesis polysome <br />                  formation. Using ribonomic approach, phenotypic distinction correlated different <br />                  set mRNAs associated eIF3e eIF3m complexes. eIF3m complex <br />                  appears associate bulk cellular mRNAs, eIF3e complex associates far                  restricted set. microarray findings independently corroborated random set 14 <br />                  mRNAs RT-PCR analysis. <br />                  Conclusion: propose PCI proteins eIF3e eIF3m define distinct eIF3 complexes <br />                  assist translation different sets mRNAs. <br />  <br />  <br />  <br />  <br /> Background                                                                    CSN/eIF3) domains [1]. proteasome 20S catalytic protein complexes conserved yeast                      particle 19S regulatory subunit cooperate humans, 19S proteasome lid, CSN, eIF3,                        degrading polyubiquitylated proteins (reviewed [2]). <br /> tain subunits characterized protein motifs:                        19S proteasome separated base com- <br /> MPN (Mpr1/ Pad1 N-terminal) PCI (proteasome/                          plex, binds unfolds substrates [3],  <br />  <br />                                                                                                                                        Page 1 16 <br />                                                                                                                  (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  subunit lid complex, cleaves ubiquitin        Results <br /> substrates, apparently facilitating entry sub-     csn6 csn7b essential genes <br /> strates catalytic proteasome barrel [4,5].            CSN complexes higher eukaryotes typically contain <br />                                                                distinct subunits  MPN PCI proteins). higher eukaryotes, subunits 19S lid          fission yeast subunits known <br /> pair-wise similarity subunits CSN [6-9].   (Csn1p, 2p, 3p, 4p, 5p, 7Ap; Ref. [13]), vivo, CSN promotes activity cullin-RING ubiqui-      essential viability [40-42]. noticed genes tin ligases [10-16], multiprotein complexes containing         Schizosaccharomyces pombe genome database, originally <br /> cullins, RING protein RBX1, hun-        annotated csn6            (SPBC4C3.07) csn7b <br /> dred substrate-specific adaptors [17-22]. MPN              (SPAC1751.03), encode MPN PCI domain <br /> domain containing CSN subunit 5 harbors protease             containing proteins considerable similarity meta- <br /> motif [23] cleaves ubiquitin-related <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span>          zoan CSN6 CSN7B, respectively (data shown). NEDD8 cullins [24,25]. activity, acting       order determine genes function cert CSN-associated deubiquitylation enzyme           known biochemical pathways regulated CSN, cul- <br /> Ubp12, proposed promote cullin function facil-       lin deneddylation Ubp12p-mediated deubiquityla- <br /> itating recruitment labile substrate adaptors           tion [24,26], deleted csn6 csn7b wild-type <br /> [11,16,26,27].                                                 diploid cells. sporulation tetrad analysis,                                                                viable spores recovered case, PCI/MPN complex, eIF3, distantly             <span id='am-30' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> auxotroph (Fig. 1A, data related CSN 19S lid (reviewed [28]).             shown).  unlike genes encoding known human eIF3 consists 13 subunits,          subunits fission yeast CSN [41,42], csn6 csn7b <br /> secutively named eIF3a &#226;&#8364;&#8220; l GA17 [29,30], budding           essential. <br /> yeast contains subunits (depending purification conditions). subunits           Csn6p Csn7Bp associate eIF3 components <br /> orthologs human eIF3a, b, c, g, eIF3i [31,32]       elucidate essential functions Csn6p Csn7Bp, <br /> appear constitute conserved core complex [31,33].         sought identify interacting proteins. Fission yeast contains core subunits, addi-   end, attempted modify csn6 csn7b tion non-core subunits eIF3d/Moe1p, eIF3e/Int6p,        endogenous genomic loci consecutive protein eIF3h [34-36]. putative eIF3f ortholog          epitope tags (proA). C-terminal tag contained identified, biochemical evidence confirming       cleavage site Tobacco Etch Virus (TEV) protease <br /> authentic eIF3 subunit functioning protein synthesis     upstream protein moieties. Diploid analysis indi- outstanding [34,35,37,38].                               cated C-terminal tagging destroyed essential func- <br />                                                                tion csn6 (data shown). contrast, csn7b-proA <br /> eIF3 complex translation initiation factor     haploid strain viable did display obvious <br /> plays important roles protein synthesis.       phenotypes (data shown). indicated  eIF3 binds 40S ribosome facilitates load-    tagged allele functional. Total protein lysate ing Met-tRNA/eIF2&#194;&#183; <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span>GTP</span> ternary complex form          csn7b-proA haploid strain absorbed Immunoglobu- 43S preinitiation complex. Subsequently, eIF3 appar-       lin G (IgG) resin specifically retained proteins ently assists eIF4 recruiting mRNAs 43S complex.     eluted sodium dodecyl sulfate (SDS) cleavage critical vivo function eIF3 core subunits     TEV protease. eluates resolved gel <br /> processes indicated lethality respective     electrophoresis Csn7Bp-proA interacting proteins <br /> budding yeast deletion strains [31]. contrast, fission   identified peptide mass fingerprinting using <br /> yeast, non-core subunits eIF3d eIF3e           matrix-assisted laser desorption/ionization-time flight <br /> essential viability global protein synthesis            (MALDI-TOF) mass spectrometry. <br /> [34,36,38,39]. proposed distinct sub- <br /> classes eIF3 complexes, containing different combina-       eluates contained identical set proteins. tions core non-core subunits, regulate specific     addition Csn6p, Csn7Bp complex contained subsets mRNAs fission yeast [34,36,38,39].           eIF3 core subunits eIF3a, b, c,  non- <br /> study provides experimental evidence substantiat-    core subunit eIF3h (Fig. 1B, Table 1). eIF3g core sub- <br /> ing hypothesis demonstrating biochemically        unit detected using MALDI-TOF purifica- <br /> distinct eIF3 complexes defined PCI domain pro-         tion, detected independent purification teins eIF3e eIF3m  novel eIF3 protein) associate         Csn7Bp complex using sensitive method different sets mRNAs.                                  nano-liquid chromatography tandem mass spectrom- <br />                                                                etry (LC-MS/MS, Fig. 5A).  eIF3e eIF3d, <br />                                                                previously described components S. pombe eIF3 <br />  <br />  <br />                                                                                                                   Page 2 16 <br />                                                                                             (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                             http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br />  <br />                                                      TEV                                            SDS                                       B <br />                                                       csn7b-                                         csn7b- <br />                                                    wt proA                                        wt proA <br />          &#226;&#710;&#8224;csn6      &#226;&#710;&#8224;csn7b <br />                                            kDa                                           kDa <br />                                                                      eIF3c               121                         eIF3c <br />  1                                         121 <br />                                                                      eIF3a                                           eIF3a <br />                                                                      eIF3b                                           eIF3b <br />  2 <br />                                             75                                            75                         Csn7Bp-ProA <br />                                                                      eIF3c* <br />  3                                                                                                                   eIF3c* <br />                                                                      IgH <br />  4                                          50                       Csn7Bp <br />                                                                                           50                         IgH <br />                                             42                       eIF3h <br />                                                                                           42                         eIF3h <br />                                                                      eIF3i <br />                                                                      Csn6p                                           eIF3i <br />                                                                                                                      Csn6p <br />                                                                                           27 <br />                                             27                       TEV, IgL <br />                                                                      ProA                                            IgL, ProA <br />  <br />  <br />                                                  10%SDS-PAGE                                     Trycine Gel <br />  <br /> Figure <br /> Analysis 1of csn6 csn7b genes proteins <br /> Analysis csn6 csn7b genes proteins.   csn6 csn7b genes disrupted diploid S. pombe cells inserting ura4 marker. Diploids sporulated spore viability examined tetrad analysis. spores viable, indicating csn6 csn7b essential. (B) Cell lysate strain carrying csn7b modified C-terminal <br /> pro tags TEV cleavage site endogenous genomic locus absorbed IgG resin, followed elution bound <br /> proteins TEV protease (left panel) SDS (right panel). Gels stained Coomassie Brilliant Blue, proteins identified MALDI-TOF mass spectrometry. asterisks denote degradation products elF3c. <br />  <br />  <br />  <br />  <br /> Table 1: Summary eIF3 subunits various eukaryotes <br />  <br />  Unified           Domain             Human               S. cerevisiae       S. pombe Csn7Bp   S. pombe Int6p       S. pombe eIF3b <br />  Nomenclature                                             (Core subunits)     complex           complex              complex <br />  <br />  eIF3a             PCI                p170                Tif32p              p107              p107                 p107 <br />  eIF3b             RRM                p116                Prt1p               p84               p84                  p84 <br />  eIF3c             PCI                p110                Nip1p               p104              p104                 p104 <br />  eIF3d             -                  p66                 -                   -                 Moe1p                Moe1p <br />  eIF3e             PCI                p48                 -                   -                 Int6p                Int6p <br />  eIF3f             MPN                p47                 -                   Csn6p             Csn6p                Csn6p <br />  eIF3g             RRM/Zn finger      p44                 Tif35p              Tif35p            Tif35p               Tif35p <br />  eIF3h             MPN                p40                 -                   p40               -                    p40 <br />  eIF3i             WD repeat          p36                 Tif34p              Sum1p             Sum1p                Sum1p <br />  eIF3j             -                  p35                 -                   -                 -                    - <br />  eIF3k             PCI                p28                 -                   -                 -                    - <br />  eIF3l             PCI                p67                 -                   -                 -                    - <br />  eIF3m             PCI                GA17 (?)            -                   Csn7Bp            -                    Csn7Bp <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 3 16 <br />                                                                                                     (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                       http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> [34,36,38,39] consistently absent Csn7Bp         strong reduction protein levels 20 32 hours <br /> complex analyzed MALDI-TOF LC-            (Fig. 2B). Coincident shut eIF3f eIF3m <br /> MS/MS (Fig. 1B 5A).                                        expression, global cellular protein synthesis dimin- <br />                                                                ished ~80%, determined metabolic labeling Notably, authentic CSN subunits           35S-methionine (Fig. 2C). <br />  <br /> Csn7Bp complex, consistent previous demon- <br /> stration S. pombe CSN contains non-essential     biochemical analysis sucrose gradient velocity <br /> subunits [13]. Structural analysis indicated Csn6p     centrifugation revealed strong reduction forma- related human CSN6, shows slightly           tion polysomes eif3f eif3m mutants following <br /> greater overall similarity human eIF3f (data            promoter shut (Fig. 2D). contrast, described pre- <br /> shown). Based functional characterization               viously [34,36,38,39], eif3e mutants showed minor <br /> described (Fig. 2), fission yeast Csn6p        reduction total protein synthesis (data shown) appears ortholog human eIF3f.                     essentially normal polysomes (Fig. 2D). <br />  Csn7Bp ortholog known human eIF3,              conserved cullin deneddylation function CSN recently identified subunit GA17 [30] shows       impaired eif3f eif3m mutants eif3e <br /> considerable similarity Csn7Bp (K. Hofmann, personal        mutants, cullin 1 (Cul1p) did accumulate exclu- <br /> communication). addition, interactions metazoan          sively neddylated form does csn5 deletion <br /> CSN7 eIF3 subunits previously observed          strains (Fig. 2E). Similarly, unlike csn5 mutants (Fig. 2E; <br /> [43,44]. regards unified nomenclature eIF3      [26]), CSN/Ubp12p-mediated inhibition Cul1p subunits [29], propose eIF3m appropriate          vitro ubiquitin ligase activity impaired shut- novel fission yeast eIF3 subunit encoded       eif3f eif3m expression. Cul1p neddylation <br /> csn7b  Table 1). manuscript,       activity unaffected eif3e mutants (Fig. <br /> use proposed new names eif3f eif3m genes       2E). data suggested eIF3f eIF3m bona products. original annotations recently     fide subunits fission yeast eIF3 CSN, perform- revised curator S. pombe genome data-       ing essential functions protein synthesis similar eIF3 <br /> base, order reflect findings presented          core subunits. <br />  eIF3f eIF3m proteins primarily cytoplasmic         eIF3m eIF3e proteins define distinct eIF3 evidence presented suggested eIF3f          complexes <br /> eIF3m essential subunits S. pombe eIF3. substan-     purification eIF3m complex robustly <br /> tiate possibility, determined subcellular local-   retrieved roughly stoichiometric amounts known eIF3 <br /> ization eIF3f eIF3m. proteins             subunits, subunits, eIF3d eIF3e, conspicu- <br /> authentic eIF3 subunits involved protein synthesis,         ously absent (Fig. 1B, Table 1). exclude possibility assume subcellular localization similar       overlooked subunits known eIF3 subunits. prepared strains       obscured IgG bands, affinity-purified eIF3e- green fluorescent protein (GFP) tagged alleles         associated proteins eif3e-proA strain. retrieved <br /> eIF3b, eIF3e, eIF3m integrated respective       proteins identified using MALDI-TOF mass spec- <br /> genomic loci. addition, C-terminal GFP-tagging        trometry  <br /> destroyed essential function small eIF3f protein, prepared strain mildly overexpressing N-terminally        Like eIF3m complex, eIF3e complex contained <br /> GFP-tagged eIF3f pREP81 plasmid. Fluorescence           roughly stoichiometric amounts eIF3 core subunits <br /> microscopy live cells revealed proteins        b, c, g, (Fig. 3A, Table 1). addition, eIF3f primarily localized cytoplasm (Fig. 2A),     present  unexpectedly, eIF3m eIF3h miss- consistent function protein transla-        ing. did detect proteasome lid subunit Rpn5p, <br /> tion. contrast, CSN subunits largely nuclear fis-    previously shown associate eIF3e <br /> sion yeast ([41], unpublished observation).        eIF3d [45]. observation indicated                                                                described eIF3d/eIF3e/Rpn5p interaction unsta- <br /> eIF3f eIF3m required global protein synthesis      ble purification conditions contains directly assess potential role eIF3f eIF3m    minor fraction total eIF3e engaged protein <br /> protein synthesis, prepared haploid eif3f eif3m         interactions. <br /> deletion strains maintained viable plasmids driving regulated expression eif3f eif3m, respectively.         results raised intriguing possibility fission <br /> Expression plasmids turned      yeast contains distinct eIF3 complexes comprise <br /> addition thiamin growth media, resulting        overlapping set core subunits, distinguished <br />  <br />  <br />                                                                                                                    Page 4 16 <br />                                                                                              (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                                                              http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br />  <br />                                          eIF3b.GFP         eIF3e.GFP             B         Strain:              + Thiamin (h)      0       20 24    28    32 <br />                                                                                                                                                                   kDa <br />                                                                                               &#226;&#710;&#8224;eif3f                                                              40 <br />                                                                                          pRep81.myc-eif3f         Myc-eIF3f <br />  <br />                                                                                               &#226;&#710;&#8224;eif3m                                                               50 <br />                                                                                          pRep81.myc-eif3m        Myc-eIF3m <br />  <br />  <br />                                                                                                                               60S Polysomes <br />                                                                                                  D <br />  <br />                                                                                                                   WT                                          &#226;&#710;&#8224;eif3e <br />                                            GFP.eIF3f         eIF3m.GFP <br />                                                                                                                            40S     80S <br />  <br />        C    Background                                  wt       &#226;&#710;&#8224;eif3f   &#226;&#710;&#8224;eif3m <br />                                                                                                                  eif3f                                        eif3f <br />                                                               pRep81. pRep81.                           &#226;&#710;&#8224;eif3f <br />                                            Plasmid     pRep81 myc-eif3f myc-eif3m                                                                            <br />                                            Thiamin <br />  <br />  <br />                                                                                                        &#226;&#710;&#8224;eif3m    eif3m                                        eif3m <br />                                                                                                                                                                    Autoradiography <br />  <br />                                                                                                                                      n5 3f     3m 3e <br />                                                                                                  E                                 cs eif &#226;&#710;&#8224;eif &#226;&#710;&#8224;eif <br />                                                                                                                    Background: wt &#226;&#710;&#8224; &#226;&#710;&#8224; <br />                                                                                                                   pRep81.myc-      eif3f eif3m <br />                                                                                                                       Thiamin <br />              Coomassie <br />               staining <br />  <br />                                                                                                                  Blot:     kDa <br />               Relative 35S incorporation <br />  <br />  <br />  <br />  <br />                                            1.2                                                                                                            Ub(n) <br />                                             1                                                  IP:            &#206;&#177;-Ub     180 <br />                                                                                                    &#206;&#177;-Cul1p                 120 <br />                                            0.8 <br />                                            0.6 <br />                                                                                                                                                              Cul1p-Ned8p <br />                                                                                                                  &#206;&#177;-Cul1p <br />                                            0.4                                                                                75                             Cul1p <br />                                            0.2                                                                                           * <br />                                                                                                    Total                      50                            Myc-eIF3m <br />                                             0                                                                    &#206;&#177;-Myc <br />                                                  Vector         eif3f     eif3m                    lysate                     40                            Myc-eIF3f <br />  <br />  <br />  <br /> Roles <br /> Figureof2eif3e eif3min protein synthesis <br /> Roles eif3e eif3min protein synthesis.   Subcellular localization. Live cells expressing GFP-tagged alleles eif3b, <br /> eif3e, eif3m endogenous genomic loci examined fluorescence microscopy. N-terminally GFP-tagged eif3f expressed low levels pREP81 plasmid. (B) Shut strains. Diploid heterozygous eif3f eif3m deletion <br /> strains transformed pREP81 plasmids driving thiamin-repressible expression Myc-tagged eIF3f eIF3m, <br /> respectively. Diploids sporulated homozygous disruptants carrying eif3f eif3m plasmids selected. Cells grown liquid medium OD595 0.3 absence thiamin, followed promoter shut addition thiamin. Samples taken indicated time points promoter shut analyzed expression plasmid <br /> borne eIF3f eIF3m immunoblotting anti-Myc antibodies. (C) Effect total protein synthesis. eif3f eif3m <br /> shut strains maintained absence presence thiamin indicated. Strains transformed pREP81 <br /> plasmid included control. Cells metabolically labeled 35S-methionine, aliquots total cellular proteins separated SDS-PAGE analyzed autoradiography. Coomassie Blue stained gel shown document equal <br /> protein loading. Data quantified PhosphorImager analysis results displayed bar graph. (D) Effect polys- <br /> omes. Polysome profiles determined indicated strains described Methods section. eif3e deletion <br /> strain shown reference. (E) Effect CSN-mediated regulation cullin-RING ubiquitin ligases. Cul1p complexes immunopurified indicated strains employed substrate-independent vitro ubiquitylation reactions puri- <br /> fied E1, E2 Cdc34p, ubiquitin, <span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span>ATP</span> [26]. Polyubiquitin chains formed reaction indicated  panel).  <br /> trast csn5 mutants, derepression Cul1p activity observed cells lacking eif3f, eif3m, eif3e. neddylation state Cul1p determined immunoblotting (middle panel). Hyperneddylated Cul1p accumulates csn5 mutants. Shut- eif3f eif3m expression verified immunoblotting anti-Myc antibodies (lower panel). <br />  <br />  <br />  <br />                                                                                                                                                                       Page 5 16 <br />  <br />  <br /> Figure 3eIF3 complexes <br /> Distinct <br /> Distinct eIF3 complexes.   eIF3 complexes associated eIF3e eIF3m affinity-purified absorption respective pro tagged proteins IgG beads. Bound proteins eluted cleavage SDS (eIF3e) TEV protease <br /> (eIF3m). (B) eluates described   analyzed immunoblotting eIF3d antibodies. asterisk refers cross- <br /> reactivity secondary antibody Ig light chains. (C) Lysates cells expressing Myc-tagged Csn5p, eIF3e, eIF3m endogenous genomic loci immunoprecipitated anti-eIF3d antisera. Coprecipitated proteins identi- <br /> fied immunoblotting indicated. Total cell lysates shown document expression levels endogenously tagged <br /> proteins. (D) Protein lysate cells expressing protein tagged eIF3b endogenous promoter absorbed IgG <br /> beads, specifically retained proteins identified LC-MS/MS. <br />  <br />  <br />  <br />                                                                                                                      Page 6 16 <br />                                                                                                (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                       http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  presence essential PCI protein eIF3m       analysis revealed 106 mRNAs uniquely enriched MPN domain protein eIF3h non-essential           eIF3e complex, 2050 transcripts uniquely <br /> PCI protein eIF3e binding partner eIF3d.             enriched eIF3m complex (Fig. 4A,C). <br /> exclude possibility mass spectrometry analy- <br /> sis missed substoichiometric amounts eIF3d            ranked list 106 mRNAs uniquely <br /> eIF3m complex, analyzed eIF3m complexes immu-             enriched fold eIF3e complex noblotting eIF3d antisera. eIF3d readily       assembled ( additional file 2]). Approximately 75% detectable eIF3e complex, undetectable     mRNAs fell categories: mRNAs <br /> eIF3m complex (Fig. 3B). reciprocal experiment,          encoding proteins involved intermediary metabolism <br /> eIF3d antibodies immunoprecipitated eIF3e-13Myc,             (25.5%), encoding proteins involved protein eIF3m-13Myc Csn5p-13Myc (Fig. 3C).                   metabolism (14.2%), encoding proteins                                                                 unknown functions (35.8%; Fig. 4B). remainder exclude possibility cell lysis affinity      distributed roughly equally categories: <br /> purification conditions led uncontrolled release eIF3     mRNAs encoding proteins involved nucleic acid metab- <br /> subunits, mimicking existence distinct sub-         olism, encoding proteins involved transcription, <br /> complexes, performed purification using protein        encoding transporter proteins, encoding tagged eIF3b core subunit bait. determined LC-       unclassified proteins. <br /> MS/MS performed excised bands detected com- <br /> plex PAGE separation, eIF3b complex contained          corresponding list 106 highly enriched known eIF3 core components eIF3f             mRNAs exclusively present eIF3m complex  eIF3m (Fig. 3D). eIF3e eIF3d proteins         total 2050 enriched factor greater  <br /> present, albeit clearly substoichiometric amounts.           revealed ~58% fell categories: mRNAs findings indicate purification eIF3b resulted     encoding proteins involved nucleic acid metabolism copurification distinct eIF3m- eIF3e      (9.4%), encoding transporter proteins (18.9%), taining complexes. Taken  data strongly          encoding proteins unknown function (20.8%). <br /> suggested existence distinct eIF3 complexes          remainder roughly equally distributed defined PCI proteins eIF3m eIF3e.                    categories: mRNAs encoding proteins involved                                                                 intermediary metabolism, mRNAs encoding pro- eIF3m eIF3e proteins associate distinct sets       teins involved protein metabolism, encoding mRNAs                                                        proteins involved transcription, encoding finding eif3m essential, eif3e dispen-   unclassified proteins (Fig. 4B,  additional file 2]). <br /> sable, suggested possibility different eIF3 <br /> complexes define regulate different subsets             exclude possibility affinity purification <br /> mRNAs. approach  sought identify mRNAs            protocol unspecifically enriched highly expressed mRNAs, <br /> specifically associated eIF3m eIF3e com-           averaged data competively hybrid- <br /> plexes. complexes affinity-purified described       ized samples total S. pombe cDNA determine rel- presence RNAse inhibitors, associ-      ative expression rank mRNA (rank 1 = ated RNA extracted, amplified, converted Cy3-        abundant mRNA, rank 5407 = abundant mRNA; labeled cDNA described Methods section.           legend Fig. 4C . plot expression rank vs. <br /> Cy3-labeled cDNA hybridized competitively              factor enrichment 106 mRNAs enriched Cy5-labeled cDNA prepared total RNA micro-            eIF3e eIF3m complexes revealed eIF3m <br /> arrays representing 4988 predicted S. pombe open read-      complex contained mRNAs distributed equally <br /> ing frames (ORFs) RNAs. Based background             entire range expression levels. contrast, subtracted hybridization signals, eIF3-associated mRNAs         eIF3e complex enriched rare mRNAs (~65 % ranked according factor enrichment      eIF3e-associated mRNAs expression rank &gt; 4000; <br /> eIF3m eIF3e complexes mock purified sample           Fig. 4C). lack linear correlation ( additional file 1]).                                      mRNA expression level enrichment eIF3e                                                                 eIF3m complexes argues unspecific copurification microarray analysis suggested global role eIF3m       abundant mRNAs. restricted function eIF3e translation. <br /> Using arbitrary cut value fold enrichment       Specific association mRNAs eIF3m eIF3e mock, eIF3m associated 2464 different mRNAs,          complexes eIF3e associated 520 distinct species         validate results microarray hybridizations, <br /> (Fig. 4A). observed 414 mRNAs enriched             chose mRNAs random list 106 fold samples. (Fig. 4A). addition,         used RT-PCR confirm interactions  <br />  <br /> Figure 4 mRNAs eIF3 complexes <br /> Association <br /> Association mRNAs eIF3 complexes.   Messenger RNAs associated eIF3e eIF3m complexes identified microarray hybridization described Methods section. graph indicates numbers mRNAs <br /> enriched fold mock complexes. (B) Classification functional groups 106 mRNAs highly uniquely enriched complexes eIF3e eIF3m. (C) relative expression ranks 106 highly uniquely enriched mRNAs eIF3e eIF3m complexeswere determined using relative hybridization signals obtained total S. pombe cDNA ( additional file 1]). expression ranks blotted factor enrichment mRNA eIF3 complexes. correlation expression rank enrichment apparent, indicating highly <br /> expressed mRNAs unspecifically enriched eIF3 complexes. 5407 individual features microarray slides <br /> represent 4988 ORFs transcripts predicted Sanger Centre S. pombe genome database various controls. simplify analysis, expression ranks assigned control spots, resulting expression ranks ORFs higher theoretically possible number 4988. <br />  <br />  <br />  <br /> Figure association <br /> Specific 5           mRNAs eIF3e eIF3m complexes <br /> Specific association mRNAs eIF3e eIF3m complexes.   eIF3 complexes shown affinity- <br /> purified described Methods section, separated SDS-PAGE followed staining Coomassie Brilliant Blue. labeled bands identified LC-MS/MS. eIF3f* refers band eIF3m complex identified mass <br /> spectrometry experiment, assigned based comigration eIF3f identified eIF3e complex <br /> (right lane). eIF3f positively identified MALDI-TOF mass spectrometry subunit eIF3m complex Fig. 1B. (B) indicated eIF3e- eIF3m-associated mRNAs extracted purified complexes employed RT-PCR <br /> reactions using primers specific mRNA  Table 3). PCR products obtained total S. pombe RNA shown reference left panel. factor enrichment mRNA respective eIF3 complex enrichment rank <br />  106 mRNAs enriched complex) indicated gels. <br />  <br />  <br />  <br />  <br /> respective eIF3 complexes. eIF3e-proA- eIF3m-proA-                     enriched eIF3e complex microarray <br /> associated complexes affinity-purified                  analysis preferentially associated separated PAGE. complexes showed dif-                     complex RT-PCR (Fig. 5B). Based result, ferential subunit composition described (Fig.                     calculate false positive rate 0.11 95% <br /> 5A). associated RNA purified, split equal                     confidence interval 0.00 &#226;&#8364;&#8220; 0.31. mRNAs portions, employed RT-PCR reactions gene-                      high enrichment factor (&gt;10,  additional file 2]) <br /> specific primer pairs.                                                     appeared exclusively associated eIF3e-proA <br />                                                                             limit detection), lower enrich- RT-PCR analysis total fourteen mRNAs                           ment factor (&lt;10) displayed low associ- <br /> included mRNAs identified eIF3e complex,                      ation eIF3m complex (Fig. 5B). Conversely, mRNAs identified eIF3m complex,                        mRNAs highly enriched eIF3m complex mRNA associated complexes                       determined microarray analysis ( additional file 2]) <br /> similar enrichment factors. mRNAs varied widely                   exclusively bound complex ana- factors enrichment enrichment ranks                  lyzed RT-PCR (Fig. 5B). <br /> indicated Fig. 5B. exception, mRNAs <br />  <br />  <br />                                                                                                                                     Page 9 16 <br />                                                                                                               (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Notably, mRNA encoding proteasome subunit             viability. competing models proposed Rpn5p, complexes                   explain pleiotropic phenotype eif3e deletion <br /> microarray analysis similar enrichment factors           mutants, includes slow growth minimal media <br /> (eIF3e = 5.1; eIF3m = 8.1) confirmed        meiotic defects [34,36,38,39,50,51]. associated eIF3 complexes RT-PCR (Fig. 5B).      model, eIF3e regulates translation transcripts, results strongly suggest eIF3m eIF3e       rare mRNAs affected fractional <br /> complexes specifically associate distinct sets        reduction translation efficiency rise mRNAs.                                                        distinct phenotypes. second model, eIF3e regulates <br />                                                               translation specific subset mRNAs encoding <br /> Discussion                                                    proteins depletion leads distinct phenotypes PCI proteins define distinct eIF3 complexes                   lethality. findings presented strongly favor results Csn6p Csn7Bp subunits        second model. <br /> fission yeast eIF3. Csn6p shown eIF3f subu- <br /> nit conserved eukaryotes budding yeast.     microarray analysis revealed 106 mRNAs uniquely <br /> sistent finding, eIF3f previously    enriched eIF3e complex, rare, associated eIF3g, role eIF3f    2050 transcripts uniquely enriched eIF3m <br /> protein synthesis examined [34]. study       complex. finding consistent severe identified distinct eIF3 complexes defined     protein synthesis defect lethality eif3m deletion PCI proteins eIF3e eIF3m (Fig. 3A). direct         mutants. mRNA lists based equivalent eIF3m apparent eIF3 preparations     arbitrary cut value fold enrichment eukaryotes. Considering essential function       mock, criterion proved stringent confirm <br /> eIF3m protein synthesis fission yeast, possible   specific association respective eIF3 complexes PCI proteins substitute eIF3m         14 mRNAs retested RT-PCR.  calcu- <br /> organisms.                                                    late false positive rate 0.11 95% <br />                                                               confidence interval 0.00 &#226;&#8364;&#8220; 0.31, maximum number Consistent idea finding PCI         mRNAs listed potentially false positively enriched domain containing GA17, recently identified protein         eIF3e complex 33 106  95% <br /> copurifying human eIF3 [30], shows similarity       confidence). eIF3m PCI protein (K. Hofmann,  <br /> sonal communication). addition, PCI domain          consider unlikely mRNAs taining CSN subunits 1, 3, 7, 8 interact    enriched eIF3e complex exclusive translational eIF3 subunits Arabidopsis thaliana human       targets complex, cellular phenotypes cells [43,44,46]. addition, Pci8p/Csn11p, non-essen-     eif3e mutants discrete. tial PCI subunit budding yeast CSN [47,48], copu-      eIF3e-associated mRNAs encode essential proteins, viabil- <br /> rifies essential eIF3 core subunits [49].   ity, global protein synthesis, polysome formation functional significance interactions        affected mutants grown rich <br /> unclear, findings raise intriguing possibility   media (Fig. 2D, data shown). likely define multiple subclasses distinct eIF3 complexes.   eIF3m PCI domain proteins deputize fission yeast appears rely essential eIF3    eIF3e cases. fact, microarray study revealed <br /> complex classified eIF3m non-essential complex       substantial portion eIF3e-associated mRNAs specified eIF3e, higher eukaryotes divide task     recovered eIF3m complex (Fig. 4A). Nonethe- mRNA translation multitude eIF3 com-             strongly suspect inefficient translation plexes. individual PCI proteins defining      limited set eIF3e-associated mRNAs contributes dis- <br /> subcomplexes expected present substoichio-       tinct cellular phenotypes eif3e mutants  metric amounts relative core components, eIF3e    . S. pombe core complex, escaped detec- <br /> tion bulk eIF3 preparations higher eukaryotes.        eIF3e protein translational control stress <br />                                                               response <br /> Distinct eIF3 complexes translational specificity         eif3e mutants display marked sensitivity wide finding distinct eIF3 complexes asso-    variety cellular stresses, osmotic stress, nutrient <br /> ciate different sets mRNAs surprising, circum-    starvation, low temperature [34,38,39,50,52] addi- <br /> stantial evidence pointing existence provided    tion, eif3e mutants synthetic lethality proteas- previous studies. example, fission yeast eIF3e copu-   ome mutants, presumed accumulate large <br /> rifies eIF3 core subunits,  unlike core subu-    load misfolded proteins [45]. Conversely, overexpres- <br /> nits, eIF3e required global protein synthesis   sion eif3e confers resistance broad spectrum  <br />  <br />                                                                                                                  Page 10 16 <br />                                                                                             (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                          http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> unrelated drugs, commonality appears              far unsuccessful identifying induce cellular stress [38]. Overexpression           common sequence elements upstream regions <br /> eIF3e activates genes involved stress defense,        mRNAs enriched eIF3e complex (data thioredoxin [38]. Interestingly, mRNA encoding              shown), analysis severely hampered fact <br /> thioredoxin identified putative translational        5&apos;UTRs poorly defined S. pombe. target eIF3e ( additional file 2]). Finally, eIF3e          possible critical features, including secondary <br /> undergoes relocalization cytoplasmic foci                  structure determinants, lie 5&apos;UTR response heat osmotic stress, implicating        regions eIF3e-associated mRNAs. Substantial addi- <br /> stress regulation [35].                                            tional work required delineate exact mecha- <br />                                                                    nism eIF3e contribute translation Global protein synthesis actively switched               specific mRNAs potential impact mecha- <br /> response cellular stress nutrient deprivation               nism human cancers defective INT6/eIF3e [58,59]. <br /> (reviewed Refs. [53,54]). shut protects cells proteotoxicity relieving chaperones load          Conclusion unfolded client proteins, conserving amino acids            provide biochemical evidence existence essential functions, attenuating metabolic           distinct eIF3 complexes fission yeast. complexes <br /> consequences protein synthesis.  long-term              contain overlapping set subunits, distin- <br /> adaptation stress conditions, repair recovery               guished PCI proteins eIF3e eIF3m. Based require synthesis new stress-induced proteins.           finding distinct eIF3 complexes associate stress response, mRNAs translated              different mRNAs, propose different presence repressed general translation machinery,         translational specificities. <br /> suggesting select eIFs escape global repres- <br /> sion. studies raise intriguing possibility        Methods <br /> eIF3e complex specialized function               Fission yeast techniques <br /> stress response. Lack eif3e prevent stress-induced          Preparation fission yeast cultures, cDNA cloning, yeast <br /> synthesis critical translational targets, leading   transformation, PCR-based genomic epitope tagging known stress sensitivity mutant. Substantial            gene deletion, mating, tetrad analysis carried additional work required address                   using standard methods described [26]. strains <br /> possibility.                                                       used study listed Table 2. <br />  <br /> Potential mechanisms underlying mRNA discrimination             Affinity purification mass spectrometry <br /> eIF3 complexes                                                     Epitope-tagged strains (eif3m-proA eif3e-proA strains) lines evidence begun implicate            grown 12 L yeast extract supplements (YES) <br /> general translation factors, including eIF2 eIF4,           medium 1.5 optical density (OD) 595nm <br /> translational specificity [55]. Importantly, factors         wavelength harvested centrifugation. Cell lysates downstream important signaling cascades, includ-            prepared bead lysis lysis buffer containing 50 <br /> ing nutrient-sensing TOR pathway variety              mM Tris-HCl pH 8.0, 150 mM NaCl, 0.5% Triton X-100, <br /> stress-induced kinases (reviewed Refs. [55,56]).                1 mM dithiothreitol, supplemented protease inhibi- details regulatory mechanisms            tors (10 ug/ml leupeptin, 10 ug/ml pepstatin, 5 ug/ml <br /> beginning emerge, unclear exactly eIF2          aprotinin 1mM phenylmethylsulfonylfluoride). eIF4 contribute mRNA specificity.                           centrifugation 17,000 rpm 50 min 4&#194;&#176;C, <br />                                                                    300 mg total protein volume 50 ml absorbed eIF3 cooperates cap-binding eIF4 complex            100 ul Dyna beads (Dynal Biotech, Oslo, Norway) cou- directing 5&apos; end mRNAs 40S ribosome,          pled rabbit IgG (Jackson Immunochemicals, eIF3 subunits contain RNA binding motifs,              West Grove, Pennsylvania, USA). Mock purifications data consistent model eIF3 complexes            performed parallel absorbing lysate cells lack- <br /> defined distinct PCI proteins recognize determining             ing tagged proteins IgG beads. Beads collected <br /> features mRNAs presented 40S subunit concert          magnetic device washed times 5 ml lysis eIF4. agreement idea recent finding       buffer. TEV protease (Invitrogen, Carlsbad, California, human PCI protein eIF3a specifically facilitates          USA) cleavage elution 5% SDS solution used translation mRNA encoding ribonucleotide                collect bound proteins. Eluted proteins separated reductase M2 [57]. addition,  thaliana eIF3h, non-          trycine gels conventional SDS-polyacrylamide gel <br /> core subunit, recently shown specifically                electrophoresis (SDS-PAGE), visualized Coomas- <br /> required efficient translation transcription factor     sie G250 staining. Protein bands excised gels, <br /> ATB2 [46].                                                         digested trypsin (Promega Corporation, Madison, <br />  <br />  <br />                                                                                                                      Page 11 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                          http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Table 2: Strains used study <br />  <br />                      Genotype                                                                                      Source <br />  <br />  DS448/1                  leu1-32 ura4-d18 ade6-704           h+                                                        Lab stock <br />  DS448/2                  leu1-32 ura4-d18 ade6-704           h-                                                        Lab stock <br />  C399/3                   leu1-32 ura4-d18 ade6-704           csn5.13myc kan h-                                         Lab stock <br />  C485/3                   leu1-32 ura4-d18 ade6-704           eif3m.13myc kan h-                                        Lab stock <br />  C617/1                   leu1-32 ura4-d18 ade6-704           eif3m.CBP.tev.5proA kan h-                                Lab stock <br />  C642/1                   leu1-32 ura4-d18                    diploid, eif3f/eif3f::ura4                                study <br />  C642/2                   leu1-32 ura4-d18                    diploid, eif3f/eif3f::ura4                                study <br />  C648                     leu1-32 ura4-d18 ade6-704           eif3e.cbp.tev.5proA kan h+                                study <br />  C650/1                   leu1-32 ura4-d18 ade6-704           eif3f::ura4 pRep81.6xhis.myc.eif3f h-                     study <br />  C650/2                   leu1-32 ura4-d18 ade6-704           eif3m::ura4 pRep81.6xhis.myc.eif3m h-                     study <br />  C652/1                   leu1-32 ura4-d18 ade6-704           eif3e::ura4 h+                                            study <br />  C663/1                   leu1-32 ura4-d18 ade6-704           eif3m.gfp kan h-                                          study <br />  C663/2                   leu1-32 ura4-d18 ade6-704           eif3e.gfp kan h-                                          study <br />  C665                     leu1-32 ura4-d18 ade6-704           eif3e.13myc kan h-                                        study <br />  C701                     leu1-32 ura4-d18 ade6-704           eif3b.gfp kan, h-                                         study <br />  C702                     leu1-32 ura4-d18 ade6-704           eif3b.cbp.tev.5proA kan h-                                study <br />  <br />  <br />  <br />  <br /> Wisconsin, USA), subjected mass spectrometry.            RT-PCR eIF3 complexes analyzed MALDI-TOF,           RNAs associated eIF3e eIF3m complexes <br /> LC-MS/MS Harvard NIEHS Center Proteomics Facil-          purified described  Primer design RT- <br /> ity, according standard procedures.                          PCR conditions according manual supplied <br />                                                                 Platinum Quantitative RT-PCR Thermoscript kit <br /> Protein synthesis polysome analysis                         (Invitrogen). Primers used RT-PCR listed Table eif3f eif3m conditional shut strains inoc-     3. primer concentration actin amplification ulated 10 ml Edinburgh Minimal Media (EMM)           0.25 uM. primers used 1 uM. Total RNA thiamin cultured 30&#194;&#176;C 30 hrs. 100 uCi        obtained described [19]. <br /> 35S-methionine added cells,  <br /> labeled 30 min harvested centrifugation.        Immunological techniques <br /> Cell lysates prepared bead disruption equal         Immunoprecipitation, immunoblotting, vitro <br /> amounts lysates analyzed SDS-PAGE. 35S-          ubiquitination assays carried exactly described <br /> methionine incorporation determined autoradiog-          previously [26]. Antibodies protein ubiqui- <br /> raphy quantified PhosphorImager analysis (Bio-           tin purchased Sigma Genzyme (Cam- <br /> Rad, Hercules, California, USA).                                bridge, Massachusetts, USA), respectively. Anti-eIF3d/ <br />                                                                 Moe1p polyclonal antibody provided E. Chang <br /> Polysome analysis performed exactly described         (Baylor College Medicine, Houston, Texas, USA) <br /> Ref. [60]. Briefly, cells grown EMM with- thiamin 30 hrs. Cycloheximide (100 ug/ml)           Microarray analysis <br /> added culture 20 min. Cells harvested          eIF3 complexes affinity-purified described lysed breaking buffer (20 mM Tris/HCl pH 7.4, 50          50 U/ml SuperRNasein (Ambion, Aus- <br /> mM NaCl, 30 mM MgCl2, 1 mM DTT, 50 ug/ml cyclohex-              tin, Texas, USA) 5 mM ribonucleoside vanadyl com- <br /> imide, 0.2 mg/heparin, 50 U/ml SuperRNasein, pro-           plex (New England Biolab, Beverly, Massachusetts, USA) <br /> tease inhibitors), followed centrifugation 13,000 &#195;&#8212; g     added lysis buffer. mock purification 5 min. 20 A260 nm units supernatant vol-       performed identical conditions lysate ume 500 ul fractionated 5-45 % sucrose gradi-        cells lacking tagged proteins. methods extrac- <br /> ents 3 hrs 30,000 rpm using SW40-Ti rotor         tion RNA, cDNA synthesis, RNA amplification, Beckman L70 centrifuge. Polysome profiles obtained         aminoallyl labeling adopted previous publica- monitoring absorbance 254 nm gradi-         tions [61,62]. Briefly, binding washing, beads <br /> ent using Bio-Rad fractionator, output            treated 3 U/ul DNase  followed elution <br /> recorded using Bio-Rad UV detector.                           5 % SDS, extraction phenol/chloroform                                                                 ethanol precipitation. associated RNA  <br />  <br />                                                                                                                            Page 12 16 <br />                                                                                                       (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                      http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> Table 3: RT-PCR primers                                        Control Cy5-labeled aminoallyl-cDNA oligo <br />                                                                dT primed reverse transcription total S. pombe RNA. <br /> Gene         Sequence                       Product <br />                                                  Size (bp) <br />                                                                Labeled cDNAs hybridized glass slide microar- <br /> actinF            GAGCTTCCTGATGGTCAAGT              200        rays. Microarrays spotting purified ds-PCR <br /> actinR            GGATACATAGTGGTACCACC                         products 500 1200 bp length glass slides coated <br /> cdc16F            AAGTTAGCGCCCAAATCACC              150        aminopropylsilane (Erie Scientific, Portsmouth, <br /> cdc16R            TTTTGAATGCCCCCCCACGG                         New Hampshire, USA). PCR products used repre- <br /> gar2F             GGTGCCATTGAGAAACCTTC              150 <br />                                                                sented 4988 predicted ORFs transcripts annotated <br /> gar2R             CCGAACCTTCAAAGAAATCA <br /> pof6F             TCTGATCGGCCTAAGCTGTC              150 <br />                                                                Sanger Centre [63] located 3&apos; end pof6R             CAATCTTGCAAATTCAACTC                         predicted genes maximize sensitivity detecting oligo- <br /> rpn5F             TGAGAAGCAAGTTCGTCAGG              160        dT primed reverse transcription products. Descriptions rpn5R             TGAAACAATCGATGACAAGT                         primer pairs features microarrays avail- <br /> rps8-1F           AAGCGTATTCACGAGGTCCG              150        able Longhorn Array Database [64]. <br /> rps8-1R           CAACTCGTTGTTAGAAGGGT <br /> scd1F             TCAGAGTTGGCTGCTTTCTT              150 <br />                                                                Median pixel intensities spot obtained scd1R             ATCCATTGTGTGCCCTGTTC <br /> SPAC1348.05F      CTTAGTGAACAGTTTGGAAG              150        hybridizations eIF3-associated RNA ( additional <br /> SPAC1348.05R      TGATAAACCAACGGATCCGA                         file 1, sheet 1]) corrected local background sub- <br /> SPAC21E11.04F     GAGACATCACCTGCTCCAGA              150        traction divided corresponding values <br /> SPAC21E11.04R     TTTGCTCCGGTGACTAGGTG                         obtained mock purification. resulted SPBC12C2.06F      TCTGTTCCCAAACCTCAAGC              150        factor enrichment mRNAs bound eIF3 complexes <br /> SPBC12C2.06R      ATCGATTTTTGCACCTTTAG                         mock. mRNAs ranked according SPBC1683.01F      GGCCGTAAATTTGTCTACGG              150 <br />                                                                factor enrichment ( additional file 1, sheet 2]). SPBC1683.01R      ACCACCAATACCAACACCAA <br /> SPBC36B7.03F      CTCTCAATTAAATTTCACCC              150        enrichment factor chosen arbitrary <br /> SPBC36B7.03R      AGGAGTACCGTATAAAGCAT                         threshold. 2464 mRNAs enriched greater <br /> SPCC70.05cF       GTACCTGGAAATAACTCTCC              150        fold mock eIF3m complex, SPCC70.05cR       CATAGCCTTTTCTAAGAGAT                         observed 520 mRNAs enriched eIF3e complex <br /> tf2-12F           AAGCATGTACCAGAGATAGG              150        ( additional file 1, sheet 3]). Comparison data <br /> tf2-12R           TGAATCACCTAGAAGAATTA                         sets revealed 414 mRNAs occurred com- <br /> ung1F             ACTTTGGAGAGTTCTTGGTT              150 <br />                                                                plexes, 2050 mRNAs uniquely enriched eIF3m <br /> ung1R             TGGAGTATGATGTGACCATG <br />                                                                complex 106 mRNAs exclusively enriched <br /> F = Forward; R = Reverse                                       eIF3e complex ( additional file 1, sheet 4; addi- <br />                                                                tional file 2]). corresponding list 106 mRNAs <br />                                                                highly uniquely enriched eIF3m complex <br />                                                                assembled ( additional file 2]) used select <br />                                                                specific mRNAs RT-PCR analysis. <br /> reversely transcribed using Superscript II reverse tran- <br /> scriptase (Gibco-BRL Division Invitrogen) 1 uM         competitive Cy5-labeled total cDNA provided clear <br /> oligo-dT(15)-T7 primer (5&apos;-AAA CGA CGG CCA GTG AAT             microarray feature identification guide relative <br /> TGT AAT ACG ACT CAC TAT AGG CGC-3&apos;) 1 uM tem-              sequence abundance pre-purification sample plate switch primer (5&apos;-AAG CAG TGG TAT CAA CGC                relative expression rank mRNA AGA GTA CGC GGG-3&apos;). length double stranded               determined ( additional file 1, sheet 2]). study, <br /> (ds)-cDNA synthesized using Advantage Polymerase           data adjusted ratio normalization total <br /> (Clontech, BD Biosciences, Mountain View, California,          RNA signals local area normalization functions. USA). Newly synthesized ds-cDNA passed             microarray data deposited public ArrayEx- <br /> Bio-6 columns (Bio-Rad) dried. RNA amplification           press (E-MEXP-208) [65] GEO [66] databases. performed T7 megascript kit (Ambion). <br /> Amplified RNA purified using RNeasy kit (Qia-          List abbreviations <br /> gen, Hilden, Germany) reversely transcribed           CSN COP9 signalosome <br /> Superscript II reverse transcriptase presence ami- <br /> noallyl-<span id='am-25' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="dUTP" datatype="xsd:string"></span>dUTP</span> reactions primed oligo-dT (Invitro-       Cul Cullin <br /> gen). RNAse H digestion purification QIAquick columns (Qiagen), aminoallyl-cDNA             eIF Eukaryotic initiation factor <br /> labeled Cy3, purified QIAquick columns. <br />                                                                EMM Edinburgh Minimal Media <br />  <br />  <br />                                                                                                                   Page 13 16 <br />                                                                                              (page number citation purposes) <br />  BMC Biology 2005, 3:14                                                                        http://www.biomedcentral.com/1741-7007/3/14 <br />  <br />  <br />  <br /> GFP Green fluorescent protein                                                 Houseman assistance microarray data analysis. study                                                                              funded National Institute Health (NIH) grant GM59780 DAW, NIH <br /> IgG Immunoglobulin G                                                         grant P40 RR16320 JL, NIEHS Center Grant ES-00002,                                                                              NIEHS Training Grant ES07155. <br />  <br /> LC MS/MS Liquid chromatography tandem mass <br />                                                                           <br /> spectrometry                                                   <br /> MALDI-TOF Matrix-assisted laser desorption ionization     <br /> time flight                                         <br /> MPN Mpr1/Pad1 N-terminal                            <br /> OD Optical density                                   <br /> PAGE Polyacrylamide gel electrophoresis         <br /> PCI Proteasome, COP9 Signalosome, eIF3               <br /> ProA Protein                                               <br /> RT-PCR Reverse transcription-polymerase chain reaction     <br /> TEV Tobacco etch virus                               <br /> YES Yeast extract supplements (growth media)                     <br /> Authors&apos; contributions                                                  <br /> CZ performed bulk experiments shown                     <br /> study. FA SW prepared various strains used                <br /> study. SK ARI performed mass spectrometry. AO                       <br /> formed microarray hybridizations, JL participated                  design microarray studies data analy-                    <br /> sis. DAW contributed study design drafting                       <br /> manuscript.                                                                    <br />   <br /> </body></html>