<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Apr. 1997, p. 2267&#226;&#8364;&#8220;2273                                                                                               Vol. 179,  7 <br /> 0021-9193/97/$04.0010 <br /> Copyright q 1997, American Society Microbiology <br />  <br />  <br />  <br />    aarC, Essential Gene Involved Density-Dependent Regulation <br />            29-N-Acetyltransferase Providencia stuartii <br />                        PHILIP N. 1,2* KIMBERLY  SOLINSKY,2 MICHAEL R. PARADISE,2 <br />                                                                      1 <br />                                               MILICA M. PAROJCIC <br />  <br />           Research Service, Veterans Affairs Medical Center,1 Departments Medicine Molecular Biology                    Microbiology, Case Western Reserve University School Medicine,2 Cleveland, Ohio 44106 <br />                                              Received 30 August 1996/Accepted 3 December 1996 <br />  <br />              2*-N-acetyltransferase [AAC(2*)-Ia] Providencia stuartii dual function involved           acetylation <span id='am-25' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-26' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span>peptidoglycan</span> certain aminoglycosides. search negative regulators aac(2*)-Ia <br />           gene resulted identification aarC. missense allele (aarC1) resulted 8.9-fold increase           b-galactosidase accumulation aac(2*)-lacZ transcriptional fusion. Northern blot analysis demon- <br />           strated increase aac(2*)-Ia mRNA accumulation specific cells high density. addition,           aarC1 allele resulted substantial increase expression aarP, transcriptional activator           aac(2*)-Ia gene. wild-type aarC gene isolated complementation encodes predicted protein           365 amino acids molecular mass 39,815 Da. predicted AarC protein exhibited 88% <span id='am-18' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> <br />           homology previously identified GcpE protein Escherichia coli 86% homology gene product <br />           Haemophilus influenzae. E. coli gcpE gene able functionally complement aarC1 allele           P. stuartii. aarC1 allele identified T G transversion resulted valine <span id='am-36' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> substi- <br />           tution position 136 AarC protein. aarC gene appears essential cell viability construc- <br />           tion disrupted copy (aarC::lacZ) possible cells carried episomal copy aarC gcpE. <br />  <br />  <br />    Providencia stuartii gram-negative enteric bacterium                 high density;  aarC act pathway sponsible variety human infections, especially                P. stuartii responds density-dependent signal. urinary tract (11, 16, 25). family Enterobacte- <br /> riaceae, P. stuartii members tribe Proteeae                                            MATERIALS METHODS <br /> (Proteus Morganella spp.) contain unique modification                    Bacterial strains, plasmids, growth conditions. P. stuartii PR50 wild- peptidoglycan corresponding O acetylation C-6 hy-                  type strain described previously (20) parent strain droxyl muramic acid (4). modification proposed                    isolation mutants (Table 1). E. coli XL1-Blue (Stratagene) DH5a <br /> control activity autolysins, serving sub-                 (Gibco/BRL) used hosts propagation plasmids. addition,                                                                                 propagate R6K pir mutant derivatives, strains DH5a lpir (14) SM10 lpir <br /> strate blocking activity (4&#226;&#8364;&#8220;6, 19). Recently, chro-             (18) used. Plasmids pACYC184 (2), pBluescript SK(2) (Stratagene), <br /> mosomally encoded aminoglycoside 29-N-acetyltransferase                     pBCKS (2) (Stratagene), pKNG101 (13) used cloning vectors. shown role O acetylation peptidogly-                   bacteria grown Luria-Bertani (LB) medium supplemented 15 g P. stuartii (19). enzyme, encoded aac(29)-Ia                 agar/liter needed. Antibiotics used following concentrations                                                                                 growth E. coli: ampicillin, 150 mg/ml; chloramphenicol, 25 mg/ml; streptomy- <br /> gene, universal P. stuartii initially identified           cin, 25 mg/ml; tetracycline, 15 mg/ml; kanamycin, 20 mg/ml. growth P. <br /> role aminoglycoside resistance (3, 20, 26). Presumably, ami-                 stuartii, antibiotics used following concentrations: ampicillin, 150 <br /> noglycosides true substrate AAC(29)-Ia                   mg/ml; chloramphenicol, 100 mg/ml; streptomycin 75, mg/ml; tetracycline, 30 <br /> acetylated structural similarity peptidoglycan.               mg/ml. Selection sucrose resistance P. stuartii 308C LB agar <br />                                                                                 plates NaCl containing 5% sucrose. <br />    Regulation aac(29)-Ia expression complex. Recessive                       Isolation subcloning aarC. Sau3AI recombinant library P. stuartii <br /> mutations loci, aarA, aarB, aarD, aarG,               chromosomal DNA pACYC184 described previously (14). Plasmid <br /> increase aac(29)-Ia mRNA accumulation identified                      pACYC184.aarC contains approximately 1.7-kb Sau3A partial fragment (15, 20, 23). addition, transcriptional activator, aarP,               pACYC184. Digestion plasmid SalI released 2.0-kb fragment <br />                                                                                 containing P. stuartii insert DNA 276 bp pACYC184 identified (14). additional level regulation includes                DNA BamHI cloning site SalI site. 2.0-kb SalI fragment repression mediated diffusible extracellular factor, AR-                   cloned SalI site pBluescript II SK(2), resulting pSK-2.0. Digestion <br /> factor, acts unknown mechanism decrease                          pSK-2.0 BamHI released 600-bp fragment, subsequent religation aac(29)-Ia mRNA accumulation cells approach high density                     vector-containing fragment resulted pSK.aarC1.4, contained 1.4- <br />                                                                                 kb insert entirely composed P. stuartii DNA. 1.4-kb insert cloned <br /> (22).                                                                           pBCKS (2) 1.4-kb BamHI-SalI fragment, resulting pBC.aarC1.4. <br />    study reports identification characterization                Plasmid pBC.aarC1.4 subcloned digestion SspI BamHI, aarC gene P. stuartii. deduced AarC protein dis-                     released approximately 1.0-kb fragment subcloned played extensive identity GcpE Escherichia coli, gene                   EcoRV- BamHI-digested pBCKS (2) pBluescript II SK(2).  <br />                                                                                 sulted pBC.aarC1.0 pSK.aarC1.0, respectively. isolate DNA frag- <br /> product function unknown                         ment containing length version aarC gene, chromosomal DNA ported essential (1, 7). P. stuartii, role aarC             PR50 digested SalI fragments 4 6 kb ligated pBlue- <br /> gene product regulation aac(29)-Ia specific cells                script II SK(2). Previously, Southern blot analysis demonstrated aarC                                                                                 approximately 5-kb SalI fragment. Colony hybridization 309-bp <br />                                                                                 EcoRV fragment pBC.aarC1.4 used identify recombinant plasmids <br />                                                                                 containing 5-kb SalI fragment length aarC gene. plasmid <br />                                                                                 designated pSK.aarC5.0. Plasmid pPR25 contains approximately 0.7-kb <br />    * Corresponding author. Mailing address: Infectious Diseases Sec-            NruI fragment pSK.aarC5.0 cloned EcoRV site pBCKS (2). <br /> tion 1110W, Wade Park Medical Center, 10701 E. Boulevard, Cleve-                Plasmid pPR26 contains 3.9-kb NruI-SalI fragment pSK.aarC5.0 cloned <br /> land, OH 44106. Phone: (216) 791-3800, ext. 4674. Fax: (216) 229-8509.          EcoRV SalI sites pBCKS (2). <br /> E-mail: pxr17@po.cwru.edu.                                                         isolate aarC1 mutation, chromosomal DNA PR60 digested <br />  <br />                                                                          2267 <br />  2268       ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />                                                             TABLE 1. Bacterial strains plasmids <br />   Strain plasmid                                              Genotype relevant markers                                                       Source <br />  <br /> Strains <br />   E. coli <br />      XL1-Blue                  recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 Dlac-pro [F9 proAB lacIq lacZDM15 Tn10]                      Stratagene <br />      DH5a                      recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 DlacZYA-argF U169 f80dlacZDM15                               Gibco/BRL <br />      DH5a lpir                 DH5a lysogenized lpir                                                                                Macinga et al. (14) <br />      SM10 lpir                 thi thr leu tonA lacY supE recA RP4-2-Tc::Mu Kmr lpir                                                     Miller Mekalanos (18) <br />   P. stuartii <br />      PR50                      Wild type                                                                                                 et al. (20) <br />      PR500                     PR50 containing pR401 pACYC184.aarG                                                                   study <br />      PR60                      PR50 aarC1                                                                                                study <br /> Plasmids <br />   pACYC184                     Medium-copy-number vector; Cmr Tcr                                                                        Chang Cohen (2) <br />   pKNG101                      R6K-derived suicide plasmid containing Strr sacB                                                      Kaniga et al. (13) <br />   pBluescript SK(2)            High-copy-number vector; Apr                                                                              Stratagene <br />   pBCKS (2)                    High-copy-number vector; Cmr                                                                              Stratagene <br />   pR401                        aac(29)-lacZ transcriptional fusion                                                                       Orosz (21) <br />   pACYC184.aarC                pACYC184 containing 1.7-kb P. stuartii chromosomal fragment                                               study <br />   pSK-2.0                      pBluescript SK(2)::2.0-kb SalI fragment pACYC184.aarC                                                study <br />   pSK.aarC1.4                  pBluescript SK(2)::1.4-kb SalI-BamHI fragment pSK-2.0                                                study <br />   pBC.aarC1.4                  pBCKS (2)::1.4-kb SalI-BamHI fragment pSK-2.0                                                        study <br />   pSK.aarC1.0                  pBluescript SK(2)::1.0-kb SspI-BamHI fragment pSK.aarC1.4                                            study <br />   pBC.aarC1.0                  pBCKS (2)::1.0-kb SspI-BamHI fragment pSK.aarC1.4                                                    study <br />   pBC.aarC::lacZ               pBC.aarC1.0::lacZ inserted unique NruI site                                                            study <br />   pKNG101.aarC::lacZ           pKNG101::4.5-kb SalI-BamHI fragment pBC.aarC::lacZ                                                   study <br />   pSK.aarC5.0                  pBluescript SK(2)::5.0-kb SalI chromosomal fragment PR50                                             study <br />   pPR25                        pBluescript SK(2)::0.7-kb NruI fragment pSK.aarC5.0                                                  study <br />   pPR26                        pBluescript SK(2)::3.9-kb NruI-SalI fragment pSK.aarC5.0                                             study <br />   pSK.aarC1.4mut               pBluescript SK(2)::1.4-kb SalI-BamHI fragment containing aarC1 allele                                     study <br />   pSK.aarC1.0mut               pSK.aarC1.0 substituted 320-bp NruI-BclI fragment pSK.aarC1.4mut                                study <br />  <br />  <br />  <br />  BamHI SalI, fragments range 1 2 kb ligated         cloned EcoRV site pBluescript II SK(2) used <br /> pBluescript II SK(2) digested BamHI SalI. Colony hybridization           sequencing template. DNA sequence analysis insert described used identify plasmids 1.4-kb fragment            pBC.aarC1.4 indicated presence partial open reading frame 738 bp <br /> containing mutant aarC gene. plasmid designated pSK.aarCmut.1.4.        capable encoding protein 246 amino acids truncated carboxy <br />    Construction aarC::lacZ disruption. Plasmid pBC.aarC1.0 containing        terminus. sequence remainder aarC coding region, cloned 1.0-kb portion aarC coding region linearized digestion NruI,      0.5-kb fragment pACYC184.aarC containing additional 39 coding region. cuts position 454 aarC coding region. promoterless lacZ        resulted additional 266 nucleotides downstream previous se- <br /> cassette obtained pQF50 (8) digestion SmaI ScaI             quence;  stop codon open reading frame ligated correct orientation pBC.aarC, creating aarC::lacZ transcrip-    reached. complete sequence, plasmid pSK.aarC5.0, containing entire <br /> tional fusion. DNA fragment containing fusion excised digestion        aarC gene 5.0-kb SalI fragment, isolated. Custom primers BamHI SalI cloned suicide vector pKNG101 (13), creating        used finish DNA sequence analysis. Protein homology searches pKNG101.aarC::lacZ. plasmid contains pir-dependent origin replica-         performed using blastp program National Center Biotechnology <br /> tion, counterselectable sacB gene, streptomycin resistance gene.          Information. <br /> introduce pKNG101.aarC::lacZ P. stuartii chromosome, E. coli SM10              b-Galactosidase assays. Plasmid pR401 containing aac(29)-lacZ transcrip- <br /> lpir (18) containing pKNG101.aarC::lacZ mated P. stuartii strains          tional fusion described previously (21). b-Galactosidase activity described previously (21). Mating mixtures plated LB plates         reported Miller units (17). Reported values represent averages triplicate <br /> taining streptomycin (75 mg/ml), tetracycline (30 mg/ml), X-Gal (5-bromo-        samples independent experiments. <br /> 4-chloro-3-indolyl-b-D-galactopyranoside). case P. stuartii strains       <span id='am-10' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. sequence aarC sub- <br /> taining plasmid DNA, ampicillin added concentration 300 mg/ml.      mitted GenBank assigned accession number U67933. <br /> Selection loss integrated pKNG101.aarC::lacZ achieved growth 308C LB plates NaCl containing 5% sucrose. Southern blot <br /> analysis performed strains digestion chromosomal DNA SalI                                            RESULTS <br /> followed transfer nylon membrane. aarC probe contained 1.4-kb <br /> BamHI-SalI fragment. probe specific lacZ 3.2-kb PstI fragment pMC1871 (Pharmacia). <br />                                                                                         Isolation aarC1 allele. chromosomal 29-N-acetyl- <br />    Northern blot analysis. RNA prepared Northern blot analysis using      transferase [AAC(29)-Ia] role O acetylation TRIzol reagent (Gibco/BRL) fractionated 1.2% agarose gel            peptidoglycan P. stuartii (19) acetylates ami- <br /> containing 2.2 M <span id='am-60' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> transferred nylon membrane capillary       noglycosides (3, 20, 26). result, P. stuartii cells  <br /> transfer. ensure equal loading transfer RNA nylon mem- <br /> brane, RNAs directly photographed nylon filter UV illumination. <br />                                                                                      creased aac(29)-Ia expression resistant higher levels <br /> Filters probed digoxigenin-labeled 602-bp TaqI-SspI fragment        aminoglycosides (14, 15, 20, 21). identify genes involved <br /> containing aac(29)-Ia coding sequence. Filters developed Lumi-         regulating aac(29)-Ia expression, strategy gener- <br /> Phos 530 (Boehringer Mannheim) exposed X-ray film.                            ate mutations simultaneously increased expression    DNA sequencing. Fluorescein-labeled universal reverse primers used sequence double-stranded DNA using AutoRead sequencing kit (Phar- <br />                                                                                      chromosomal copy aac(29)-Ia aac(29)-lacZ fusion <br /> macia). Internal primers used needed gaps contiguous          present plasmid pR401 (21). addition, wanted sequence. <span id='am-65' about='rdfs:label' typeof='owl:Thing'>Label</span> incorporated sequencing reactions using flu-       prevent reisolation aarG mutations alter reg- <br /> orescein-labeled <span id='am-5' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="dATP" datatype="xsd:string"></span>dATP</span>. sequencing reactions analyzed L.F.           ulation aac(29)-Ia (23).  used strain PR500, <br /> automated sequencer (Pharmacia). entire nucleotide sequence strands aarC gene flanking DNA determined manner. <br />                                                                                      PR50 derivative containing pR401 [aac(29)-lacZ] <br /> Specific templates sequencing generated follows. Digestion            pACYC184.aarG, compatible pACYC184 derivative <br /> pBC.aarC1.4 EcoRV released fragments 335, 420, 267, 309, 80 bp,          containing aarG gene. PR500 forms white colonies  VOL. 179, 1997                                                        DENSITY-DEPENDENT REGULATOR aac(29)-Ia                           2269 <br />  <br />  <br /> plates containing X-Gal aminoglycoside resistance 4 <br /> mg gentamicin/ml. Selection mutants achieved plating approximately 108 cells PR500 LB plates contain- <br /> ing 15 mg gentamicin/ml, X-Gal  select identify reg- <br /> ulatory mutants), ampicillin (150 mg/ml; maintain pR401), chloramphenicol (100 mg/ml; maintain pACYC184.aarG). observed previously, colonies arose frequency 1026 1027 (20). Colonies grew conditions exhibited blue color analyzed  colonies <br /> presumably resulted mutations trans-acting regulator aac(29)-Ia expression chromosomal copy aac(29)-Ia plasmid-encoded aac(29)-lacZ fusion increased. class mutants formed small blue <br /> colonies identified, representative isolate, PR60/ <br /> pR401/pACYC184.aarG, characterized. strain cured pACYC184.aarG, resulting PR60/pR401, displayed phenotype identical PR500, indicating presence aarG, multicopy did alter effects aarC1 mutation. Introduction previously cloned <br /> aarA, aarB, aarD genes (15, 20, 21) PR60/pR401 did result complementation (data shown), indicating PR60 contained new regulatory mutation, designated aarC1. <br />    aarC1 increases aac(2*)-Ia high cell density. accumu- <br /> lation b-galactosidase wild-type PR50/pR401 PR60             FIG. 1. aarC density-dependent regulator. Total RNA (20 mg) pre- <br /> aarC1/pR401 measured cells mid-log phase,          pared PR50 (wild-type) PR60 (aarC1) cells early log phase <br /> 8.9-fold increase b-galactosidase accumulation observed      (OD600 5 0.2) early stationary phase (OD600 5 1.2). experiment, <br />                                                                     lane 1 represents RNA PR50 lane 2 represents RNA PR60 aarC1. PR60 (32.1 6 2.4 Miller units) relative PR50 (3.6 6 0.2         RNA probed 602-bp TaqI-SspI fragment specific aac(29)-Ia. <br /> Miller units). experiment repeated times           (B) RNA probed digoxigenin-labeled 0.4-kb PCR-generated fragment <br /> gave increases 8.8- 9.2-fold, respectively, PR60.         specific aarP. <br /> Northern blot analysis used directly examine effects aarC1 allele aac(29)-Ia mRNA accumulation. <br /> RNA prepared wild-type PR50 PR60 <br /> (aarC1) cured plasmid pR401 log phase (optical density     introduced electroporation PR60/pR401 [aac(29)- <br /> 600 nm [OD600] 5 0.2) examined Northern blot anal-           lacZ] colonies normal growth phenotype iso- <br /> ysis. Unexpectedly, levels aac(29)-Ia mRNA accumula-         lated. resulted isolation plasmids different <br /> tion PR60 (aarC1) significantly different wild     inserts, containing common region DNA based type instances appeared decrease. Based      restriction mapping. Retransformation representative <br /> result, effects aarC1 allele aac(29)-Ia expression    plasmid, pACYC184.aarC, demonstrated insert P. stu- examined various stages growth. RNA prepared         artii DNA responsible correcting slow growth phe- cells log phase (OD600 5 0.2) stationary phase          notype. addition, resulting transformants formed <br /> (OD600 5 1.2), aac(29)-Ia mRNA accumulation ex-             white colonies X-Gal plates. slow growth <br /> amined Northern blot analysis (Fig. 1A). levels           phenotype increased aac(29)-Ia expression corrected <br /> aac(29)-Ia mRNA cells PR50 (wild type) PR60               wild type introduction plasmids, reason- <br /> (aarC1) approximately equal log phase (Fig. 1).         able recombinant plasmids contained wild-type  late log/early stationary phase, PR60 demonstrated       aarC gene. <br /> considerable increase accumulation aac(29)-Ia mRNA           sequence aarC coding region determined relative wild-type PR50.                                         described Materials Methods. deduced AarC pro- <br />    Increased expression transcriptional activator aarP    tein contained 365 amino acids exhibited 88% identity aarC1 background. Previously, identified tran-          stretch 362 amino acids GcpE protein E. coli <br /> scriptional activator, AarP, activates aac(29)-Ia gene    (1) (Fig. 2). function GcpE unknown;  (14). increased aac(29)-Ia expression observed      proposed essential (1). Introduction E. coli aarC1 background resulted increased aarP          gcpE gene plasmid pSE401 (7) PR60 aarC1 restored <br /> expression. accumulation aarP mRNA examined           wild-type growth levels, indicating gcpE functionally <br /> Northern blot analysis shown Fig. 1B. low cell density,    substitute aarC. deduced AarC protein displayed aarC1 allele did significantly affect accumulation   86% homology 361 amino acids open reading frame <br /> aarP mRNA PR60 relative wild-type PR50.            identified Haemophilus influenzae genome (accession <br /> cells high density, accumulation aarP mRNA            number H64063) (9) 51% identity 333 amino acids significantly increased aarC1 background.                    YqfY Bacillus subtilis (accession number D84432). <br />    Cloning analysis aarC. PR60 aarC1 formed colonies            Identification aarC1 allele. Plasmid pSK.5.0 contain- considerably smaller wild-type PR50 24 h       ing entire aarC gene flanking DNA 5.0-kb SalI growth. aarC1 mutation selected spontane-          fragment capable complementing aarC1 mutation. <br /> ously, reasoned single mutation responsible        Subclones fragment generated pBluescript SK slow growth phenotype increased aac(29)-       (2) contained approximately 700-bp NruI fragment <br /> Ia expression.  isolate wild-type aarC gene,     (pPR25) 3.9-kb NruI-SalI fragment (pPR26) (Fig. 3). <br /> recombinant library P. stuartii DNA pACYC184 (2, 14)          plasmid capable complementing aarC1 mu- <br />  2270       ET AL.                                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />     FIG. 2. Comparison AarC GcpE. deduced amino acid sequences AarC GcpE aligned. Vertical lines indicate identical amino acids, <br /> colons indicate conserved amino acids, dots indicate amino acids low similarity. position valine <span id='am-48' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-49' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span> substitution AarC mutant protein indicated position 136. <br />  <br />  <br />  <br /> tation introduced PR60;  PR60 cells                             aarC::lacZ, integrated chromosome taining pPR26 unstable colonies rapidly                          PR50 conjugal mating described Materials Meth- <br /> papillated fast-growing sectors. contrast, cells contain-                     ods. integration resulted streptomycin-resistant (Smr) <br /> ing pPR25 remained small. fast-growing cells result                         sucrose-sensitive (Sucs) colonies blue correction aarC1 allele high frequency                           presence X-Gal. Integration pKNG101.aarC::lacZ combination pPR26 chromosome. case,                           occur crossovers (Fig. 4) result aarC1 mutation predicted downstream NruI site.                        strain diploid aarC. Crossover 1 yield functional <br /> Furthermore, plasmid pSK.aarC1.0 containing 1.0-kb SspI-                            copy aarC;  crossover 2 result non- <br /> BamHI fragment shown partially complement aarC1 <br /> allele. results indicated aarC1 mutation likely reside NruI BamHI interval. <br />    chromosomal DNA fragment containing aarC1 allele cloned PR60 1.4-kb SalI-BamHI fragment pBluescript SK(2), resulting plasmid pSK.aarC1.4mut. plasmid failed complement aarC1 allele intro- <br /> duced PR60. presence pSK.aarC1.4mut eith- <br /> er wild-type P. stuartii PR50 E. coli XL1 substantially <br /> reduced growth rate cells, indicating mutant <br /> AarC gene product dominant-negative effect. localize aarC1 mutation 1.4-kb fragment, <br /> restriction fragment exchanges performed pSK.aarC1.0 pSK.aarC1.4mut. Substitution wild- <br /> type 320-bp NruI-BclI fragment pSK.aarC1.0 mu- <br /> tant fragment pSK.aarC1.4mut resulted E. coli trans- <br /> formants slow growing, resulting hybrid <br /> plasmid, pSK.aarC1.0mut, failed complement PR60. indicated aarC1 mutation 320-bp NruI- <br /> BclI interval. nucleotide sequence fragment  <br /> vealed single T G transversion resulting valine glycine substitution position 136, position conserved E. coli H. influenzae homologs (Fig. 2). <br />    aarC essential gene. previous report indicated                           FIG. 3. Mapping aarC1 allele. Recombinant plasmids containing gcpE essential gene E. coli (1). assess role                      aarC region shown. shaded area represents portion aarC gene <br /> aarC P. stuartii, inserted promoterless lacZ cassette                    plasmid. ability plasmid complement marker rescue <br />                                                                                       aarC1 mutation shown. 1 (marker rescue), extensive formation papillae unique NruI site approximately aarC gene.                        colonies; 11 (complementation), restoration wild-type growth rate aarC::lacZ insertion transferred sacB                           aarC1 mutant; 1 (complementation), partial complementation; 2, comple- <br /> taining suicide plasmid pKNG101 (13), resulting pKNG101.                           mentation. <br />  VOL. 179, 1997                                                        DENSITY-DEPENDENT REGULATOR aac(29)-Ia                                   2271 <br />  <br />  <br />  <br />  <br />                                                                        FIG. 4. Southern blot analysis aarC::lacZ disruption.   structure <br />                                                                     plasmid pKNG101.aarC::lacZ depicted  5.0-kb SalI <br />                                                                     chromosomal region PR50 containing aarC shown immediately  <br />                                                                     Integration pKNG101.aarC::lacZ chromosome occur                                                                     552-bp interval 59 inserted lacZ cassette depicted crossover 1  1) <br />                                                                     579-bp interval 39 inserted lacZ cassette depicted                                                                     crossover 2  2). predicted structure chromosomal region                                                                     strain containing aarC::lacZ disruption shown 3. probe used                                                                     Southern blot analysis 1.4-kb SalI-BamHI fragment depicted                                                                     PR50 chromosomal region. (B C) Southern blot verification                                                                     aarC::lacZ disruption. DNAs digested SalI. Panel B represents <br />                                                                     DNA probed aarC. Lane 1, purified pSE401 plasmid DNA; lane 2, PR50/ <br />                                                                     pSE401; lanes 3 7, putative PR50 aarC::lacZ/pSE401 mutants. Panel C rep- <br />                                                                     resents DNA probed lacZ gene. DNAs SalI digested. Lane 1, <br />                                                                     PR50/pSE401; lanes 2 6, putative PR50 aarC::lacZ/pSE401 mutants. <br />  <br />  <br />  <br />  <br /> functional copies aarC, truncation        6). hybridization detected DNA PR50/ <br /> taining lacZ disruption. Southern blot analysis 28 Smr       pSE401 (Fig. 4C, lane 1). <br /> Sucs blue colonies demonstrated contained pKNG101.            determine viability strain aarC::lacZ <br /> aarC::lacZ integrated crossover 1 (data shown)      disruption dependent presence pSE401, retained functional copy aarC. unexpected,         grew PR50 (aarC::lacZ)/pSE401 absence ampicillin homology crossover 2 (579 bp versus 552 bp        approximately 12 generations plated single crossover 1). Excision integrated plasmid selection   colonies LB plates. resulting colonies (474 474 tested) 5% sucrose failed yield blue colonies corresponding        remained ampicillin resistant, indicating retained pSE401. aarC::lacZ disruption. results obtained       contrast, strain PR50/pKNG101.aarC::lacZ/pSE401, independent experiments, suggesting colonies containing        diploid aarC (Fig. 4), cured pSE401 frequencies aarC::lacZ disruption viable.                          50% conditions.  presence    test possibility, attempted resolve    gcpE pSE401 essential construction integrated copy pKNG101.aarC-lacZ PR50 transformed            aarC::lacZ disruption chromosome, resulting plasmid pSE401 containing E. coli gcpE gene (7).       strain requires pSE401 viability. <br /> gcpE gene chosen previously shown complement growth defect PR60 aarC1, use                                          DISCUSSION gene limit potential problem recombina- <br /> tional rescue aarC::lacZ disruption. Selection su-          aarC gene identified negative regulator crose resistance strain resulted approximately       aac(29)-Ia P. stuartii. missense allele (aarC1) resulted equal numbers blue white colonies. Southern blot anal-       increased accumulation aac(29)-Ia mRNA cells <br /> ysis randomly chosen blue colonies confirmed pres-      high density. aarC1 allele increased ac- <br /> ence aarC::lacZ disruption, demonstrated Fig. 4B       cumulation mRNA specific aarP, gene encoding C. probe specific aarC used, SalI fragment      transcriptional activator aac(29)-Ia (14). increase 5 kb corresponds aarC locus present           aarP expression specific cells high density (Fig. <br /> PR50/pSE401 (Fig. 4B, lane 2). addition, cross-hybridiza-        1). possible model consistent observations tion seen fragment pSE401 (Fig. 4B, lanes 1       aac(29)-Ia overexpression aarC1 background simply <br /> 7). putative aarC::lacZ mutants 5.0-kb SalI         overexpression aarP. test possibility, fragment longer present replaced fragment       attempted introduce aarP::Cm disruption aarC1 approximately 8.5 kb size (lanes 3 7).         background;  attempts construct double mu- <br /> expected size, lacZ cassette inserted aarC approx-   tant failed repeatedly. <br /> imately 3.5 kb. confirm  probe specific lacZ          aarC1 allele recessive wild type, suggesting used putative aarC::lacZ mutants demonstrated          regulatory consequences aarC1 result loss hybridization 8.5-kb fragment (Fig. 4C, lanes 2         AarC function. Interestingly, presence aarC1 allele <br />  2272     ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />  high-copy-number plasmid detrimental growth          probably essential function early cell growth. P. stuartii E. coli. interpretation result   essential nature aarC remarkable conservation AarC functions multimer overexpression       gene product different gram-negative mutant subunit interferes function wild-type         gram-positive bacteria make attractive target  <br /> subunits. Alternatively, AarC interact complex         velopment new antimicrobials. proteins. Analysis AarC protein did reveal obvious functional motifs,  mechanism                                     ACKNOWLEDGMENTS regulates mRNA accumulation remains deter-              grateful J. Parker plasmid pSE401 comments mined. addition, hydropathy analysis AarC did reveal       manuscript. addition, thank D. Macinga review obvious signal sequence hydrophobic regions, suggest-         manuscript. <br /> ing AarC cytoplasmic. AarC shared striking homology            work supported grant MCB9405882 National GcpE E. coli (1) H. influenzae (9) YqfY         Science Foundation Merit Review Award Depart- <br /> product B. subtilis. high degree conservation      ment Veterans Affairs. proteins suggests involved important <br />                                                                                                         REFERENCES <br /> housekeeping function. Attempts disrupt gcpE E. coli <br />                                                                       1. Baker, J., D. B. Franklin, J. Parker. 1992. Sequence characteriza- possible cells contained episomal copy           tion gcpE gene E. coli. FEMS Microbiol. Lett. 92:175&#226;&#8364;&#8220;180. <br /> gcpE, indicating gcpE essential (1). study dem-      2. Chang,  C. Y., S. N. Cohen. 1978. Construction characterization onstrated aarC required viability P.             amplifiable DNA cloning vehicles derived p15A cryptic miniplas- <br /> stuartii. addition, AarC GcpE functionally equiva-            mid. J. Bacteriol. 134:1141&#226;&#8364;&#8220;1156. <br />                                                                       3. Chevereau, M., P. Daniels, J. Davies, F. LeGoffic. 1974. Aminoglycoside <br /> lent, introduction pSE401 containing E. coli gcpE              resistance bacteria mediated gentamicin acetyltransferase II,  <br /> gene PR60 resulted complementation aarC1                  zyme modifying 29-amino group aminoglycoside antibiotics. Biochem- <br /> allele allowed construction chromosomal                istry 13:598&#226;&#8364;&#8220;603. <br /> aarC::lacZ disruption. presence gene E. coli       4. Clarke,  J. 1993. Extent peptidoglycan O acetylation tribe Pro- <br />                                                                          teeae. J. Bacteriol. 175:4550&#226;&#8364;&#8220;4553. <br /> H. influenzae, contains aac(29)-Ia, suggests         5. Dupont, C.,  J. Clarke. 1991. Evidence N3O acetyl migration additional functions carried AarC. Consistent            mechanism O acetylation peptidoglycan Proteus mirabilis. J.  aarC1 allele resulted slow growth phe-          Bacteriol. 173:4318&#226;&#8364;&#8220;4324. <br /> notype altered cell morphology formation         6. Dupont, C.,  J. Clarke. 1991. Dependence lysozyme catalysed <br />                                                                          solubilization Proteus mirabilis peptidoglycan extent O-acetyla- <br /> short rods, spherical.                                tion. Eur. J. Biochem. 195:763&#226;&#8364;&#8220;769. <br />    aarC gene added list genes,         7. Eisenbeis, S. J., J. Parker. 1981. Strains Escherichia coli containing <br /> includes aarA, aarB, aarD, aarG, serve negative              structural gene histidyl-tRNA synthetase high copy plasmid. <br /> regulators aac(29)-Ia (15, 20, 21, 23). genes             Mol. Gen. Genet. 183:115&#226;&#8364;&#8220;122. <br />                                                                       8. Farinha, M.,  Kropinski. 1990. Construction broad-host-range aarG locus, encodes putative response regu-             plasmid vectors easy visible selection analysis promoters. J. Bac- <br /> lator sensor kinase, exhibits clear similarity transcrip-       teriol. 172:3496&#226;&#8364;&#8220;3499. <br /> tional regulators (23). aarA gene product appears        9. Fleishman, R. D., M. D. Adams, O. White, R.  Clayton, E. F. Kirkness, <br /> integral membrane protein, aarD gene encodes CydD,                R. Kerlavage, C. J. Bult, J. F. Tomb, B.  Dougherty, J. M. Merrick, K. <br />                                                                          McKenney, G. Sutton, W. FithHugh, C.  Fields, J. D. Gocayne, J. D. Scott, component ABC transporter complex involved                R. Shirley, L.  Liu,  Glodeck, J. M. Kelley, J. F. Weidman, C.  Phillips, <br /> formation functional cytochrome bd oxidase (15).                T. Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T. <br /> product aarB presently unknown. lack similarity             Nguyen, D. M. Saudeck, R. C. Brandon, L. D. Fine, J. L. Fritchman, J.  aar genes transcriptional regulators suggests             Furmann, N. S. Geoghagen, C. L. Gnehm, L.  McDonald, K. V. Small, <br />                                                                          C. M. Fraser, H. O. Smith, J. C. Venter. 1995. genome random act indirectly affect common cellular process          sequencing assembly Haemophilus influenzae Rd. Science 269:496&#226;&#8364;&#8220; aac(29)-Ia aarP regulation coupled. obvious           512. <br /> candidate process given identification aarD          10. Fuqua, W. C., S. C. Winans, E. Peter Greenberg. 1994. Quorum sensing <br /> (cydD) electron transport.  purely specula-           bacteria: LuxR-LuxI family cell density-responsive transcriptional <br />                                                                          regulators. J. Bacteriol. 176:269&#226;&#8364;&#8220;275. <br /> tive time.                                                   11. Hawkey, P. 1984. Providencia stuartii: review multiply antibiotic resis- <br />    shown accumulation aac(29)-Ia mRNA                tant organism. J. Antimicrob. Chemother. 13:209&#226;&#8364;&#8220;226. decreased high cell density accumulation      12. Kaiser, D., R. Losick. 1993. bacteria talk  <br /> extracellular signaling molecule termed AR-factor (22).             Cell 73:873&#226;&#8364;&#8220;885. <br />                                                                      13. Kaniga, K.,  Delor, G. Cornelis. 1991. wide-host-range suicide vector <br /> type regulation termed quorum sensing mediated              improving reverse genetics gram-negative bacteria: inactivation accumulation diverse <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> signals, including               blaA gene Yersinia enterocolitica. Gene 109:137&#226;&#8364;&#8220;141. <br /> amino acids, peptides, acyl derivatives homoserine lac-       14. Macinga, D. R., M. M. Parojcic, P. N.  1995. Identification tone (10, 12, 24). requirement high cell density              analysis aarP, transcriptional activator 29-N-acetyltransferase                                                                          Providencia stuartii. J. Bacteriol. 177:3407&#226;&#8364;&#8220;3413. <br /> observe aarC mutant phenotype raises possibility        15. Macinga, D. R., P. N.  1996. aarD, Providencia stuartii homolog <br /> AarC involved extracellular signaling process.          cydD: role 29-N-acetyltransferase expression, cell morphology possibilities exist including requirement aarC AR-              growth presence extracellular factor. Mol. Microbiol. 19:511&#226;&#8364;&#8220; <br /> factor production ability respond AR-factor,         520. <br />                                                                      16. McHale, P., F. Walker, B. Scully, L. English, C. T. Keane. 1981. Provi- <br /> experiments progress examine possibilities. Al-          dencia stuartii infections: review 117 cases year period. J. <br /> ternatively, aarC regulate gene expression response            Hosp. Infect. 2:155&#226;&#8364;&#8220;165. density-dependent signals oxygen nutrient       17. Miller, J. H. 1972. Experiments molecular genetics. Cold Spring Harbor <br /> depletion growth rate. physiological significance         Laboratory, Cold Spring Harbor, N.Y. <br />                                                                      18. Miller, V. H., J. J. Mekalanos. 1988. novel suicide vector use <br /> regulation related role aac(29)-Ia O                construction insertion mutations: osmoregulation outer membrane <br /> acetylation peptidoglycan, modification affects deg-          proteins virulence determinants Vibrio cholerae requires toxR. J. <br /> radation muramidases control autolytic activ-            Bacteriol. 170:2575&#226;&#8364;&#8220;2583. <br /> ity (4&#226;&#8364;&#8220;6, 19). decrease aac(29)-Ia high density        19. Payie, K. G., P. N. Rathe <br /> </body></html>