<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Activity Reaction Core Plasticity Metabolic Networks <br /> Eivind Almaas1,2, Zolta&#194;&#180;n N. Oltvai3*, Albert-La&#194;&#180;szlo&#194;&#180; Baraba&#194;&#180;si2,4 <br /> 1 Microbial Systems Division, Biosciences Directorate, Lawrence Livermore National Laboratory, Livermore, California, United States America, 2 Center Complex <br /> Network Research Department Physics, University Notre Dame, Notre Dame, Indiana, United States America, 3 Department Pathology, University Pittsburgh <br /> School Medicine, Pittsburgh, Pennsylvania, United States America, 4 Center Cancer Systems Biology, Dana-Farber Cancer Institute, Harvard University, Boston, <br /> Massachusetts, United States America <br />  <br />  <br /> Understanding level adaptive changes taking place organism response variations environment key issue contemporary biology. Current modeling approaches, constraint-based flux- <br /> balance analysis, proved highly successful analyzing capabilities cellular metabolism, including capacity predict deletion phenotypes, ability calculate relative flux values metabolic reactions, capability identify properties optimal growth states.  use flux-balance analysis thoroughly assess activity Escherichia coli, Helicobacter pylori, Saccharomyces cerevisiae metabolism 30,000 diverse simulated <br /> environments. identify set metabolic reactions forming connected metabolic core carry non-zero fluxes growth conditions, flux variations highly correlated. Furthermore, enzymes <br /> catalyzing core reactions display considerably higher fraction phenotypic essentiality evolutionary <br /> conservation catalyzing noncore reactions. Cellular metabolism characterized large number species-specific conditionally active reactions organized evolutionary conserved, active, <br /> metabolic core. Finally, current antibiotics interfering bacterial metabolism target core <br /> enzymes, indicating findings important implications antimicrobial drug-target discovery. <br /> Citation: Almaas E, Oltvai ZN, Baraba&#194;&#180;si (2005) activity reaction core plasticity metabolic networks. PLoS Comput Biol 1(7): e68. <br />  <br />  <br />                                                                                           changes induce structural plasticity, resulting Introduction <br />                                                                                           changes metabolism&#226;&#8364;&#8482;s active wiring diagram, turning <br />   Constraint-based modeling approaches, &#239;&#172;&#8218;ux-bal-                                  previously zero-&#239;&#172;&#8218;ux reactions inhibiting previously <br /> ance analysis (FBA) [1,2], proved highly successful                               active pathways. example, shifting E. coli cells analyzing capabilities cellular metabolism, including                          glucose- succinate-rich media, 11 previously active <br /> capacity predict deletion phenotypes, ability                                   reactions turned completely, previously <br /> calculate relative &#239;&#172;&#8218;ux values metabolic reactions,                             inactive reactions turned  capability identify properties alternate optimal                                   types response mechanisms described &#239;&#172;&#8218;ux <br /> growth states wide range simulated single-carbon-                                 structural plasticity imply possible existence source environmental conditions [3&#226;&#8364;&#8220;5]. Recent analyses                               group reactions subject structural plasticity, <br /> indicate deletion phenotype substantial subset                           active environmental conditions.  <br /> metabolic enzymes growth-condition-dependent [6], argu-                                metabolic reactions carry non-zero &#239;&#172;&#8218;uxes <br /> ing selective use metabolic reactions distinct                                S. cerevisiae different growth conditions [6]. environments.                                                                             itself,  &#239;&#172;?nding compatible scenario                                                                                           active reactions randomly distributed: Results                                                                                   typically q fraction metabolic reactions active <br />                                                                                           speci&#239;&#172;?c growth condition, n distinct conditions <br />    thoroughly examine utilization relative &#239;&#172;&#8218;ux                                 predict non-zero overlap encompassing qn <br /> rates metabolic reaction wide range simulated                             fraction reactions.  &#239;&#172;?nd increase <br /> environmental conditions, sampled 30,000 randomly                                  number inspected conditions, n, number uniformly chosen optimal growth conditions, single-carbon-source minimal medium conditions suf&#239;&#172;?cient <br />                                                                                           Received June 27, 2005; Accepted November 2, 2005; Published December 16, growth, using FBA [1,2] reconstructed metabolic                                2005 <br /> networks Helicobacter pylori, Escherichia coli, Saccharomyces                      DOI: 10.1371/journal.pcbi.0010068 <br /> cerevisiae [7&#226;&#8364;&#8220;9]  Materials Methods).                               Copyright: &#195;&#8220; 2005 Almaas et al. open-access article distributed assuming optimal growth microorganisms,                                 terms Creative Commons Attribution License, permits unrestricted <br />                                                                                           use, distribution, reproduction medium, provided original author metabolism adapts environmental changes                                  source credited. distinct mechanisms. common mechanism                                                                                           Abbreviation: FBA, flux-balance analysis <br /> &#239;&#172;&#8218;ux plasticity, involving changes &#239;&#172;&#8218;uxes active reactions organism shifted                                    Editor: Daniel Segre, Boston University, United States America <br /> growth condition  example, changing                                   * correspondence addressed. E-mail: oltvai@pitt.edu <br /> glucose- succinate-rich media alters &#239;&#172;&#8218;ux 264 E. coli                            previous version article appeared Early Online Release November <br /> reactions 20%. commonly, environmental                                  2, 2005 (DOI: 10.1371/journal.pcbi.0010068.eor). <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                          0557                                     December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                      Metabolic Network Core <br />  <br />  <br /> Synopsis                                                                               &#239;&#172;&#8218;ux-based plasticity, rest reactions                                                                                        conditionally active, turned speci&#239;&#172;?c cellular metabolism investigated                            growth conditions subject structural- <br /> cellular functions, understood changes                       &#239;&#172;&#8218;ux-based plasticity. <br /> adapts systems level response environmental variations.                     Figure 2 displays metabolic reactions E. coli study, authors advantage reconstructed                            remain active 30,000 simulated growth conditions. metabolic networks Helicobacter pylori, Escherichia coli,                       striking feature diagram fact reactions <br /> Saccharomyces cerevisiae computational method flux- <br />                                                                                        form single connected cluster, encompassing 90 <br /> balance analysis investigate functional plasticity metabolic networks large number simulated growth <br />                                                                                        reactions. unique feature E. coli; H. pylori, conditions.                                                                            138 reactions form single cluster, S. cerevisiae 33 <br />                                                                                        reactions connected. Given relatively low number approach, authors identify metabolic cores                      reactions active, likelihood form organisms. metabolic cores represent connected sets                     single large cluster chance negligible, p , <br /> reactions used tested environments. cross-                        1e 6 H. pylori E. coli, p ; 2e 6 S. cerevisiae. Given <br /> correlating predicted cores experimental data, authors                    compact clustered nature group reactions cores display significantly higher fraction,                                                                                        active, refer collectively essential evolutionary conserved enzymes, noncore <br /> counterparts. Additionally, extended mRNA half-lives core <br />                                                                                        metabolic core. <br /> enzymes support notion core reactions                            metabolic core contains types reactions: represent main integrators metabolic activity.                                      &#239;&#172;?rst type consists essential biomass <br />                                                                                        formation environmental conditions (81 90 metabolic core reactions indispensable                   reactions E. coli), second type reaction growth microorganism, existing antibiotics target                   required assure optimal metabolic performance <br /> select bacterial enzymes core, authors suggest                     (Table S1). case inactivation second type, <br /> metabolic core important applications antimicrobial <br />                                                                                        alternative sub-optimal pathways used ensure <br /> drug-target discovery. <br />                                                                                        cellular survival. Interestingly, compact core S. <br />                                                                                        cerevisiae, 33 reactions predicted indispensable <br />                                                                                        biomass formation growth conditions.  <br /> reactions carrying non-zero &#239;&#172;&#8218;ux condition decreases                             assuming 10% reduction growth rate <br /> rapidly.  eventually saturates constant value                          (compared optimal growth), size identity E. <br /> (Figure 1A&#226;&#8364;&#8220;C), identifying group reactions non-                         coli metabolic core remains largely unchanged, retaining 83 zero &#239;&#172;&#8218;ux 30,000 simulated growth conditions.                                 original 90 reactions, nonessential. Speci&#239;&#172;?cally, &#239;&#172;?nd 138 381 H. pylori (36.2%), 90                           indicates concept metabolic core valid 758 E. coli (11.9%), 33 1,172 S. cerevisiae (2.8%),                          optimal suboptimal growth, metabolic reactions active (Figure 1D 1E).                              difference identity individual reactions. note,                                                                                        metabolic core represents subset minimal reaction reactions respond environmental changes                                                                                        sets [10,11] overlap alternative  e., degenerate) <br />                                                                                        minimal reaction sets [4,5]. minimal reaction set [11] <br />                                                                                        contains metabolic core addition,                                                                                         reactions necessary sustained growth chosen <br />                                                                                        substrate, minimal reaction set [5] consists                                                                                        201 reactions active E. coli 136 <br />                                                                                        aerobic anaerobic single-carbon-source minimal envi- <br />                                                                                        ronments capable sustaining optimal growth.                                                                                        &#239;&#172;?nding excellent agreement Figure 1B (upper <br />                                                                                        curve), demonstrates analysis low number                                                                                        growth conditions signi&#239;&#172;?cantly overestimate set                                                                                        reactions active. <br />                                                                                           identify factors determine size                                                                                        metabolic core, note number core <br />                                                                                        metabolic reactions systematically decreases                                                                                        H. pylori S. cerevisiae (Figures 3A 1E), relative size                                                                                        core decreasing 36.2% reactions H. pylori Figure 1. Emergence Metabolic Core <br />                                                                                        11.9% E. coli 2.8% S. cerevisiae. trend  C) average relative size number reactions active function number sampled conditions (black                 explained collective network effect: relatively small <br /> line)   H. pylori, (B) E. coli, (C) S. cerevisiae. number          size H. pylori metabolic network leaves little &#239;&#172;&#8218;exibility <br /> conditions increases, curve converges constant denoted                 biomass production, requiring continuous activity dashed line, identifying metabolic core organism. red line <br /> denotes number reactions active activity                                                                                        high fraction available metabolic reactions.  <br /> randomly distributed metabolic network. fact                        &#239;&#172;?nd average approximately 61% H. pylori <br /> converges zero indicates real core represents collective                 reactions active given environmental condition. network effect, forcing group reactions active conditions.           larger number metabolic reactions present E. coli offers <br /> (D E) number metabolic reactions (D) number metabolic core reactions (E) studied organisms.                           higher degree metabolic &#239;&#172;&#8218;exibility, allowing DOI: 10.1371/journal.pcbi.0010068.g001                                                 signi&#239;&#172; fraction biomass produced  <br />        PLoS Computational Biology | www.ploscompbiol.org                        0558                               December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                    Metabolic Network Core <br />  <br />  <br />  <br />  <br /> Figure 2. Metabolic Core E. coli reactions active 30,000 investigated external conditions shown. Metabolites contribute directly biomass formation [8] colored blue, core reactions (links) catalyzed essential  nonessential) enzymes [13] colored red  green). <br /> (Black-colored links denote enzymes unknown deletion phenotype.) blue dashed lines indicate multiple appearances metabolite, links arrows denote unidirectional reactions. Note 20 51 metabolites necessary biomass synthesis present core, <br /> indicating produced  consumed) growth-condition-specific manner. Protocol S1 Table S1 abbreviations metabolites list core reactions E. coli, H. pylori, S. cerevisiae. folate <span id='am-25' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-35' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span>peptidoglycan</span> biosynthesis pathways indicated blue brown shading, respectively, white numbered arrows denote current antibiotic targets inhibited  (1) sulfonamides, (2) trimethoprim, (3) <br /> cycloserine, (4) fosfomycin. Note reactions appear disconnected omitted drawing cofactors. <br /> DOI: 10.1371/journal.pcbi.0010068.g002 <br />  <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                   0559                                December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                Metabolic Network Core <br />  <br />  <br />                                                                                    alternative pathways.  average utilization E. <br />                                                                                    coli metabolic network given growth condition                                                                                    35.3%. S. cerevisiae, metabolic network size <br />                                                                                    signi&#239;&#172;?cantly exceeds H. pylori E. coli,                                                                                    higher metabolic &#239;&#172;&#8218;exibility, activity                                                                                    19.7% reactions required typical environment. <br />                                                                                       fact core reactions active                                                                                    investigated environmental conditions suggests                                                                                    play key role maintaining metabolism&#226;&#8364;&#8482;s overall <br />                                                                                    functional integrity.  absence individual <br />                                                                                    reactions metabolic core lead                                                                                    signi&#239;&#172; metabolic disruptions.  using genome-scale <br />                                                                                    deletion-phenotype data obtained rich growth media <br />                                                                                    [12,13], &#239;&#172;?nd 74.7% E. coli enzymes                                                                                    catalyze core metabolic reactions  e., core enzymes)                                                                                    essential, compared 19.6% lethality fraction character- <br />                                                                                    izing noncore enzymes. gap core                                                                                    noncore enzymes signi&#239;&#172; S. cerevisiae,                                                                                    essential enzymes catalyze 84% core reactions,                                                                                    average essentiality conditionally active enzymes                                                                                    15.6% (Figure 3B). Note likelihood random <br />                                                                                    concentration essential enzymes core                                                                                    extremely small, p-values 3.3e 23 9.0e 13 E. coli <br />                                                                                    yeast, respectively, presume presence                                                                                    errors deletion-phenotype data (Protocol S1). Taken <br />                                                                                     results indicate organism&#226;&#8364;&#8482;s ability                                                                                    adapt changing environmental conditions rests, large <br />                                                                                    extent, continuous activity metabolic core, <br />                                                                                    regardless environmental conditions. <br />                                                                                       Intuitively, assume core represents                                                                                    subset high-&#239;&#172;&#8218;ux reactions characterizing activity                                                                                    metabolic networks [3].  measurements indicate <br />                                                                                     average, &#239;&#172;&#8218;uxes core noncore reactions <br />                                                                                    highly comparable (Figure 3C). Alternatively,                                                                                    assume main biological role metabolic <br />                                                                                    core ensure continuous production biomass                                                                                    growth conditions. contrast, &#239;&#172;?nd 20                                                                                    51 metabolites (39%) considered necessary biomass <br />                                                                                    production E. coli [8] produced core <br />                                                                                    reactions; instead growth-condition-dependent fash- <br />                                                                                    <span id='am-7' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> produced various alternative metabolic <br />                                                                                    pathways. core,  contains large number                                                                                    reactions selected anabolic pathways, including                                                                                    membrane <span id='am-1' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span>-, cell envelope-, peptidoglycan-biosyn- <br />                                                                                    thesis pathways (Table S1). core reactions represent <br />                                                                                    network bottlenecks, paths synthesis                                                                                    certain biomass components.  appears                                                                                    composition metabolic core determined                                                                                    factors.  metabolic reactions directly contrib- <br />                                                                                    ute biomass production tend metabolic <br />                                                                                    core organism. Second,  tendency offset <br />                                                                                    network-induced redundancy: reactions pathways <br />                                                                                    end-products synthesized                                                                                    alternative pathways environmental redundancy Figure 3. Characterizing Metabolic Cores                                       structural plasticity, eliminated core. <br />   number overlapping metabolic reactions metabolic core             reactions metabolic network possesses H. pylori, E. coli, S. cerevisiae.                                           Figure 1D), stronger network-induced <br /> (B) fraction metabolic reactions catalyzed essential enzymes          redundancy, smaller core (Figure 1E). cores (black) outside core E. coli S. cerevisiae. <br /> (C) distribution average metabolic fluxes core                                                                                       Given important functional role played noncore reactions E. coli.                                                      metabolic core given organism, expect <br /> DOI: 10.1371/journal.pcbi.0010068.g003                                             signi&#239;&#172; parts core conserved different <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                    0560                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                      Metabolic Network Core <br />  <br />  <br /> organisms.  E. coli H. pylori cores 63            redundant core, regulatory mechanisms shifted reactions common, 18 33 core reactions S.            mRNA decay rate nature regulatory <br /> cerevisiae present E. coli H. pylori          interactions, offering higher regulatory stability metabolic cores  Figure 3A).  considering               core enzymes. <br /> enzyme orthologs 32 divergent bacteria [13], &#239;&#172;?nd                &#239;&#172;?nding core reactions form single cluster metabolic core enzymes E. coli display high             suggest activity participating reactions <br /> degree evolutionary conservation, average core enzyme          highly synchronized, changes &#239;&#172;&#8218;ux having orthologs 71.7% reference bacteria (p ,              reaction affecting &#239;&#172;&#8218;ux reactions   1e 6). contrast, conditionally active noncore enzymes          able discover core inspecting evolutionary retention 47.7% [13]. Given          correlations activities metabolic reactions. <br /> observed correlation evolutionary retention           test hypothesis, calculated &#239;&#172;&#8218;ux correlation <br /> essentiality gene products [13], difference          coef&#239;&#172;?cient, Cij, reaction pair E. coli metabolism <br /> simple consequence high-lethality fraction core         [17] inspecting &#239;&#172;&#8218;ux values reactions enzymes.  analysis indicates         30,000 simulated growth conditions. Using Cij case: random selection 90 enzymes 74.7%                 matrix metric hierarchical clustering algorithm <br /> lethality ratio average evolutionary retention         [18], observed emergence group reactions <br /> 63.4%. Taken  results indicate major              &#239;&#172;&#8218;uxes change simultaneously environmental <br /> portions metabolic core conserved,                   shifts. Interestingly, highly correlated reactions signi&#239;&#172;?- <br /> displaying higher evolutionary retention individual        cantly overlap metabolic core (Figure 4A), indicating <br /> essentiality participating enzymes indicate,             environmental conditions altered, suggesting maintaining core&#226;&#8364;&#8482;s integrity collective      &#239;&#172;&#8218;uxes core reactions increased decreased need organism.                                                 synchronized fashion. <br />    requirement continuous activity core               synchrony observed inspect exper- <br /> reactions impact regulation catalytic        imental mRNA data core reactions changes <br /> enzymes. expect activity core                    environmental conditions. Calculating correlations <br /> reactions &#239;&#172;&#8218;ux magnitude needs                core noncore enzymes using correspond- <br /> modulated display higher degree stability           ing mRNA copy number data  Materials Methods) different regulatory control reactions outside        41 experiments [19], &#239;&#172;?nd correlations core. Evidence regulatory effects provided         mRNA copy numbers core enzymes systematically <br /> measurements.  inspected experimentally deter-           higher observed correlations noncore <br /> mined mRNA half-lives E. coli metabolic enzymes           counterparts (Figure 4B), average correlation ,C. <br /> cells grown Luria-Bertani medium [14].               &#194;&#188; 0.23 core enzymes ,C. &#194;&#188; 0.07 noncore enzymes distribution mRNA half-life times core            (p , 1e 4). Notably, &#239;&#172;?nding different average <br /> enzymes displays shift higher values, average           correlation mRNA expression 66 &#226;&#8364;&#732;&#226;&#8364;&#732;correlated <br /> half-life time mRNAs 14.0 min compared            reaction sets&#226;&#8364;&#8482;&#226;&#8364;&#8482; (groups reactions 10.5 min noncore mRNAs (p &#194;&#188; 0.016). Furthermore,              turned , frequency 2% core enzymes corresponding mRNA half-             correlated reaction set used does affect life times shorter 5 min, compared 23% ones           mRNA correlation [5]. <br /> catalyzing conditionally active noncore reactions  Materi- <br /> als Methods).  continuous metabolic need                Discussion reactions apparently affected mechanisms <br /> responsible mRNA decay catalytic enzymes,                       Previous studies &#239;&#172;?rmly established environmental <br /> providing higher dynamic inertia environmental                    changes induce &#239;&#172;&#8218;ux plasticity, altering &#239;&#172;&#8218;ux levels changes.                                                                  individual reactions (reviewed [20]). Similarly, fact    Additionally, enzyme-encoding operon,                      metabolism displays structural plasticity, turning counted number activating repressive regulatory                reactions growth conditions altered, links E. coli transcriptional regulatory network [15,16].          observed [3,4,6].  demonstration regulatory interactions currently known 13             group reactions predicted active environmental core enzyme-encoding operons, considered                       conditions forming connected metabolic core extended core, representing set 234 reactions                 substantially improve understanding organization active 90% 30,000 simulated growth                utilization metabolic networks. emergence conditions. results indicate fraction repressive          core represents collective network effect, channeling regulatory links extended core 52.3%,                 production indispensable biomass components fraction activating interactions 35.7%,                key reactions replaced alternative <br /> remaining 12% represents regulatory links                 pathways environmental conditions. collective <br /> activate suppress enzyme&#226;&#8364;&#8482;s mRNA synthesis rate.                 origin core supported observed changes contrast, noncore enzyme-encoding operons,                core size. number biomass components difference fraction activating repressing              metabolism needs produce comparable links, representing 45% regulatory interactions.              organisms (49 H. pylori, 51 E. coli, 44 S. results offer evidence evolution core             cerevisiae), number metabolic reactions contributing metabolic network corresponding regulatory network:               differs. larger size metabolic network <br /> given requirement continuously active non-                    signi&#239;&#172;?cantly increases capacity redundancy, decreas- <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org            0561                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                          Metabolic Network Core <br />  <br />  <br />                                                                                       bacterial yeast gene products essential,                                                                                       high fraction essential speci&#239;&#172;?c environ- <br />                                                                                       ments. example, recent measurements indicate 76% <br />                                                                                       S. cerevisiae genes inactive nutrient-rich <br />                                                                                       conditions fact active, essential                                                                                       growth conditions [6].  effective <br />                                                                                       antimicrobial drug needs able kill target organism <br />                                                                                       physiological conditions exist.                                                                                       pharmacological interventions targeting speci&#239;&#172;?c pathway <br />                                                                                       effective environments pathway                                                                                       needed corresponding enzyme turned  Instead, <br />                                                                                       effective antimicrobials target activity                                                                                       core reactions, disruption impact                                                                                       microorganism&#226;&#8364;&#8482;s ability function environ- <br />                                                                                       mental conditions.  currently used anti- <br />                                                                                       biotics, fosfomycin cycloserine act inhibiting cell-wall <br />                                                                                       peptidoglycan, sulfonamides trimethoprim inhibit <br />                                                                                       tetrahydrofolate biosynthesis, pathways present                                                                                       H. pylori E. coli core  Figure 2). addition, <br />                                                                                       core pathways involved synthesis &#239;&#172;&#8218;avin <span id='am-20' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> <br />                                                                                       dinucleotide, 2-dehydro-3-deoxy-D-octonate, lipopolysac- <br />                                                                                       charide potential antibiotic targets [21]. <br />                                                                                          indicates core enzymes essential biomass <br />                                                                                       formation, optimal suboptimal growth (Table <br />                                                                                       S1), prove effective antibiotic targets given cell&#226;&#8364;&#8482;s <br />                                                                                       need maintain activity enzymes                                                                                       conditions. fact core reactions                                                                                       shared bacteria offers possibility identify <br />                                                                                       bacterium-speci&#239;&#172;?c drug targets. Finally, results pertain- <br />                                                                                       ing existence core characteristics                                                                                       means limited studied organisms,                                                                                       analysis carried organisms high- <br />                                                                                       quality metabolic reconstructions available. Given                                                                                       large number sequenced bacterial genomes, studies <br />                                                                                       open new avenues rapid silico antimicrobial <br />                                                                                       drug-target identi&#239;&#172;?cation. <br />  <br />                                                                                       Materials Methods <br /> Figure 4. Correlations E. coli Metabolic Reactions                                FBA. Starting published stoichiometric matrices   calculated Pearson correlation using flux values 30,000               reconstructed E. coli MG1655, H. pylori, S. cerevisiae metabolic <br /> conditions reaction pair grouping reactions                       networks [7&#226;&#8364;&#8220;9], steady-state concentrations internal <br /> according hierarchical average-linkage clustering algorithm.                 metabolites organism satisfy <br /> values flux-correlation matrix range  1 (red) 0                                           d         X <br /> (white) unity (blue). horizontal color bar denotes reaction                                    &#194;&#189;Ai   &#194;&#188;    Sij mj &#194;&#188; 0                  &#195;&#176;1&#195;&#382; <br />                                                                                                              dt          j <br /> member core (green), vertical color bar denotes enzymes catalyzing reaction essential (red).                              Sij stoichiometric coef&#239;&#172;?cient metabolite Ai reaction <br /> (B) Distribution Pearson correlation mRNA copy numbers 41                  j, mj &#239;&#172;&#8218;ux reaction j. use convention experiments [14]. correlations core reactions clearly shifted          metabolite Ai substrate  product) reaction j, Sij , 0 (Sij . 0). higher values, average correlation coefficient ,C. &#194;&#188;               vector &#239;&#172;&#8218;uxes fmjg satis&#239;&#172;?es equation 1 corresponds 0.23 compared average noncore coefficient ,C. &#194;&#188; 0.07                      state metabolic network,  potential state DOI: 10.1371/journal.pcbi.0010068.g004                                                operation cell. Using linear programming, choose                                                                                       solution maximizes biomass production respective <br />                                                                                       organisms. Reactions active, likely re&#239;&#172;&#8218;ecting annota- <br />                                                                                       tion errors incomplete data, ignored &#239;&#172;&#8218;ux analysis. <br /> ing size metabolic core.  metabolic                             Metabolic core identi&#239;&#172;?cation. model possible cellular <br /> core contains reactions necessary optimal cellular                       environments investigated FBA models, described performance regardless environmental conditions,                               [3]. Brie&#239;&#172;&#8218;y, metabolic-uptake reaction, &#239;&#172;?x uptake rate <br />                                                                                       randomly chosen value 0 20 mmol/g/h (dry conditionally active metabolic reactions represent                          weight) calculating optimal &#239;&#172;&#8218;uxes con&#239;&#172;?guration different ways cell capable optimally                          using linear programming. reaction subsequently tagged utilizing substrates environment.                                            active (non-zero &#239;&#172;&#8218;ux) inactive (zero &#239;&#172;&#8218;ux).                                                                                       large number possible combinations selected uptake <br />   identi&#239;&#172;?cation metabolic core important                          rates, repeat process 30,000 times. addition, calculate <br /> practical implications: given continuous activity high                        optimal &#239;&#172;&#8218;uxes single-carbon-source con&#239;&#172;?gurations                                                                                       minimal-uptake medium consisting unlimited ammonia, sulfate, <br /> degree essentiality core enzymes, represent                            phosphate, carbon dioxide, potassium, restricted oxygen E. <br /> potential targets antimicrobial intervention. Speci&#239;&#172;?cally,                        coli, H. pylori, S. cerevisiae, plus unlimited ergosterol zymosterol <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                       0562                                  December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                               Metabolic Network Core <br />  S. cerevisiae. reaction, assign value, qi, zero             database (http://www.shigen.nig.ac.jp/ecoli/pec) given Table S2 unity, representing relative number conditions                      Gerdes et al. [13], resulting total 62 essential enzymes reaction active. preliminary metabolic core de&#239;&#172;?ned                    seven enzymes determine deletion <br /> set reactions active sampled conditions (qi &#194;&#188; 1).                  phenotype. deletion phenotype yeast enzymes taken <br /> determine &#239;&#172;?nal metabolic core removing preliminary                       CYGD database [12]. <br /> core reactions  &#239;&#172;&#8218;ux rate constrained zero total set initial conditions, leave growth rate <br /> unchanged.                                                                               Supporting Information <br />    Correlations clustering. evaluate FBA &#239;&#172;&#8218;ux correlations calculating Pearson coef&#239;&#172;?cient possible reaction pair                     Protocol S1. Supplementary Analyses List Metabolite <br /> combinations 30,000 different simulated growth conditions.                        Abbreviations subsequently group reactions employing correlation <br /> values metric standard hierarchical average-linkage                          DOI: 10.1371/journal.pcbi.0010068.sd001 (628 KB DOC). <br /> clustering algorithm [22].                                                               Table S1. Metabolic Core Reactions <br />    Data analysis. annotated metabolic FBA models specify enzymes genes catalyzing various metabolic reactions.                         DOI: 10.1371/journal.pcbi.0010068.st001 (57 KB XLS). <br /> represent mRNA activity level half-life time metabolic <br /> enzyme averaging available experimental values, Ek, catalyzing gene products,                                                             Acknowledgments <br />                                     1 X <br />                                ri &#194;&#188;           Ek                         &#195;&#176;2&#195;&#382;             thank S. Mobashery S. Vakulenko discussions. Work                                     Ni k&#194;&#188;1;Ni                                            University Notre Dame University Pittsburgh                                                                                          supported Department Energy, National Institutes Ni number genes. assess essentiality                         Health, National Science Foundation. <br /> reaction considering essential catalyzing <br /> enzymes  gene products) essential survival                             Competing interests. authors declared competing <br /> organism  Protocol S1 error analysis).                                           interests exist. <br />   Core noncore essentiality. deletion phenotype E. coli                          Author contributions. EA, ZNO, ALB conceived study. EA <br /> enzymes taken Gerdes et al. [13]. nonlethal core                            ALB designed study. EA performed computational <br /> reactions, cross-checked deletion phenotype PEC                          work. EA, ZNO ALB wrote paper.                             <span id='am-13' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; <br />  <br />  <br />  <br />  <br /> References                                                                               12. Gu&#194;&#168;ldener U, Mu&#194;&#168;nsterko&#194;&#168;tter M, Kastenmu&#194;&#168;ller G, Strack N, van Helden J, et <br /> 1. Edwards JS, Palsson BO (2000) Escherichia coli MG1655 silico                       al. (2005) CYGD: Comprehensive Yeast Genome Database. Nucleic <br />     metabolic genotype: de&#239;&#172;?nition, characteristics, capabilities. Proc               Acids Res 33: D364&#226;&#8364;&#8220;D368. <br />     Natl Acad Sci U S 97: 5528&#226;&#8364;&#8220;5533.                                                   13. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al. (2003) <br /> 2. Schilling CH, Letscher D, Palsson BO (2000) Theory systemic                       Experimental determination level analysis essential genes     de&#239;&#172;?nition metabolic pathways use interpreting metabolic                  Escherichia coli MG1655. J Bacteriol 185: 5673&#226;&#8364;&#8220;5684. <br />     function pathway-oriented perspective. J Theor Biol 203: 229&#226;&#8364;&#8220;248.             14. Selinger DW, Saxena RM, Cheung KJ, Church GM, Rosenow C (2003) Global <br /> 3. Almaas E, Kova&#194;&#180;cs B, Vicsek T, Oltvai ZN, Baraba&#194;&#180;si AL (2004) Global                      RNA half-life analysis Escherichia coli reveals positional patterns     organization metabolic &#239;&#172;&#8218;uxes bacterium Escherichia coli. Nature                transcript degradation. Genome Res 13: 216&#226;&#8364;&#8220;223. <br />     427: 839&#226;&#8364;&#8220;843.                                                                        15. Shen-Orr S, Milo R, Mangan S, Alon U (2002) Network motifs 4. Burgard AP, Nikolaev EV, Schilling CH, Maranas CD (2004) Flux coupling                    transcriptional regulation network Escherichia coli. Nat Genet 31: 64&#226;&#8364;&#8220;68. <br />     analysis genome-scale metabolic reconstructions. Genome Res 14: 301&#226;&#8364;&#8220;              16. Salgado H, Gama-Castro S, Martinez-Antonio  Diaz-Peredo E, Sanchez- <br />     312.                                                                                     Solano F, et al. (2004) RegulonDB (version 4.0): Transcriptional regulation, <br /> 5. Reed JL, Palsson BO (2004) Genome-scale silico models E. coli                  operon organization growth conditions Escherichia coli K-12. Nucleic <br />     multiple equivalent phenotypic states: Assessment correlated reaction                 Acids Res 32: D303&#226;&#8364;&#8220;D306. <br />     subsets comprise network states. Genome Res 14: 1797&#226;&#8364;&#8220;1805.                      17. Wiback SJ, Famili  Greenberg HJ, Palsson BO (2004) Monte Carlo <br /> 6. Papp B, Pal C, Hurst LD (2004) Metabolic network analysis causes <br />                                                                                              sampling used determine size shape steady-state <br />     evolution enzyme dispensability yeast. Nature 429: 661&#226;&#8364;&#8220;664. <br />                                                                                              &#239;&#172;&#8218;ux space. J Theor Biol 228: 437&#226;&#8364;&#8220;447. <br /> 7. Schilling CH, Covert MW, Famili  Church GM, Edwards JS, et al. (2002) <br />                                                                                          18. Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis     Genome-scale metabolic model Helicobacter pylori 26695. J Bacteriol 184: <br />                                                                                              display genome-wide expression patterns. Proc Natl Acad Sci U S 95: <br />     4582&#226;&#8364;&#8220;4593. <br /> 8. Reed JL, Vo TD, Schilling CH, Palsson BO (2003) expanded genome-                       14863&#226;&#8364;&#8220;14868. <br />     scale model Escherichia coli K-12 (iJR904 GSM/GPR). Genome Biol 4:                19. Allen TE, Herrgard MJ, Liu MZ, Qiu Y, Glasner JD, et al. (2003) Genome- <br />     R54.1&#226;&#8364;&#8220;R54.12.                                                                            scale analysis uses Escherichia coli genome: Model-driven <br /> 9. Duarte NC, Herrgard MJ, Palsson BO (2004) Reconstruction validation                   analysis heterogeneous data sets. J Bacteriol 185: 6392&#226;&#8364;&#8220;6399. <br />     Saccharomyces cerevisiae iND750, fully compartmentalized genome-scale           20. Price ND, Reed JL, Palsson BO (2004) Genome-scale models microbial <br />     metabolic model. Genome Res 14: 1298&#226;&#8364;&#8220;1309.                                               cells: Evaluating consequences constraints. Nat Rev Microbiol 2: 886&#226;&#8364;&#8220; <br /> 10. Burgard AP, Maranas CD (2001) Probing performance limits                      897. <br />     Escherichia coli metabolic network subject gene additions deletions.           21. Gerdes SY, Scholle MD, D&#226;&#8364;&#8482;Souza M, Bernal  Baev MV, et al. (2002)     Biotechnol Bioeng 74: 364&#226;&#8364;&#8220;375.                                                           genetic footprinting antimicrobial drug targets: Examples cofactor <br /> 11. Burgard AP, Vaidyraman S, Maranas CD (2001) Minimal reaction sets                    biosynthetic pathways. J Bacteriol 184: 4555&#226;&#8364;&#8220;4572. <br />     Escherichia coli metabolism different growth requirements uptake           22. Dunn G, Everitt BS (1982) introduction mathematical taxonomy. <br />     environments. Biotechnol Prog 17: 791&#226;&#8364;&#8220;797.                                               Cambridge: Cambridge University Press. 152 p. <br />  <br />  <br />  <br />  <br />        PLoS Computational Bio <br /> </body></html>