<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />           Coordination Fission Yeast Glucan Formation Growth <br />                           Requires Sphingolipase Activity <br />  <br />                                                                  ABSTRACT <br />                 css1 mutants display novel defect Schizosaccharomyces pombe cell wall formation. mutant cells              temperature-sensitive accumulate large deposits material stain calcofluor aniline blue <br />              periplasmic space. Biochemical analyses material indicate consists &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucans <br />              ratio cell walls wild-type S. pombe. Strikingly, glucan deposits css1 mutant <br />              cells affect overall morphology. cells remain rod shaped, thickness walls <br />              unaltered. Css1p essential protein related mammalian neutral sphingomyelinase responsible <br />              inositolphosphosphingolipid-phospholipase C activity observed S. pombe membranes.  <br />               expression css1&#226;&#171;&#185; compensate loss ISC1, enzyme responsible activity              Saccharomyces cerevisiae membranes. Css1p localizes entire plasma membrane secretory pathway; <br />              C-terminal fragment Css1p, predicted encode single membrane-spanning segment, sufficient <br />              direct membrane localization heterologous protein, GFP. results predict existence              enzyme(s) process(es) essential coordination S. pombe cell wall formation division <br />               turn, regulated <span id='am-11' about='obo:IMR_0001359' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="sphingolipid" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:26739" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001359" datatype="xsd:string"></span>sphingolipid</span> metabolite. <br />  <br />  <br />                                                                           tomannans; alkali-soluble &#226;&#65533;&#163;-(1,3)-, &#226;&#65533;&#164;-(1,6)-, &#226;&#65533;&#164;-(1,3)- <br /> S    CHIZOSACCHAROMYCES pombe cells, like      yeast, surrounded rigid cell wall pro- <br /> vides mechanical strength protection environ- <br />                                                                           linked glucans; alkali-insoluble &#226;&#65533;&#164;-(1,3)-linked <br />                                                                           glucan &#226;&#65533;&#164;-(1,3)-linked glucans forming ma- <br /> mental stresses (Cabib et al. 1997). wall required                 jority cell wall <span id='am-1' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>polysaccharide</span> (Bush et al. 1974; Man- formation maintenance cell shape                       ners Meyer 1977). components arranged remodeled continuously cell cycle allow                  distinct layers visualized electron mi- <br /> cell expansion. Cell wall remodeling occurs                    croscopy (Kopecka et al. 1995; Osumi et al. 1998). sponse extracellular cues trigger shape changes,                  outer shell comprised glycoproteins, middle nutrient deprivation exposure mating                       layer amorphous glucan material, inner layer <br /> factors. vegetative growth, S. pombe cells rod                 densely woven glucan fibrils. S. pombe galacto- <br /> shaped grow tip elongation, old end                   syltransferases, Golgi enzymes required produc- ends. Subsequently, cell wall deposi-                    tion cell wall glycoproteins, identified tion occurs septation new ends                     characterized (Chappell Warren 1989; Chappell sealed (Mitchison Nurse 1985). Patches                       et al. 1994; Yoko-o et al. 1998). enzymes F-actin concentrated areas growth,                     identified required synthesis widely held view direct wall formation              &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucans. &#226;&#65533;&#163;-glucan sites (Marks et al. 1987; Nurse 1994). Electron                  synthases  termed Mok1p/Ags1p, micrographic studies regenerating S. pombe proto-                      characterized (Hochstenbach et al. 1998; Katayama <br /> plasts substantiated view demonstrating                   et al. 1999). family &#226;&#65533;&#164;-glucan synthases <br /> intimate association F-actin structures sites                 S. pombe.  Cps1p, critical forma- new cell wall formation (Osumi et al. 1998).                           tion septum (Ishiguro et al. 1997; Le Goff et al. <br />    <span id='am-41' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-42' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>carbohydrate</span> composition S. pombe cell                      1999; Liu et al. 1999).  Bgs2p, essential wall known time consist &#226;&#65533;&#163;-galac-                  maturation spore wall (Liu et al. 2000; Martin <br />                                                                           et al. 2000). Mok1p/Ags1p Cps1p essential                                                                           cell wall integrity viability vegetatively growing <br />   Corresponding author: Kathleen L. Gould, HHMI Department            cells. absence, cell wall Cell Biology, B2309 MCN, Vanderbilt University School Medi- <br /> cine, 1161 21st Ave. S., Nashville, TN 37232.                             organized, thicker, sensitive degradation, <br /> E-mail: kathy.gould@mcmail.vanderbilt.edu                                 cells rounded, swollen, eventually <br /> Genetics 158: 1397&#226;&#8364;&#8220;1411 (August 2001) <br />  1398                                                    Feoktistova et al. <br />  <br /> lyse unless provided osmostabilizing medium.               dominance, css1-3 strain crossed mei1-102 strain <br /> activities glucan synthases regulated com-          (mei1-102 mutants mate h&#226;&#171;&#186; partners, resulting <br />                                                                     diploids sporulate) stable diploids obtained <br /> plex manner activities Rho family GTPases                 using complementation nutritional markers. protein kinase C&#226;&#8364;&#8482;s (reviewed Arellano et al.                    parental S. cerevisiae strain diploid JK9-3d (homozy- <br /> 1999).  overproduction regulators                 gous leu2-3, 112 ura3-52 rme1 trp1 his4) MATa/MAT&#226;&#65533;&#163; <br /> dramatically affects S. pombe morphology cell wall              (Heitman et al. 1991). RCD319 MATa haploid derivative <br /> integrity.  studies, considerable progress           JK9-3d carrying yer019-&#226;&#338;&#172;1::Kan allele, bases <br />                                                                     &#226;&#171;&#186;4 &#226;&#171;&#185;967 replaced lox-kanamycin-lox cassette (Guldener identifying understanding signal-               et al. 1996). YPD (Sherman 1991) S-leu (Skrzypek et al. <br /> ing pathways governing synthesis cell wall compo-            1998) media used culture S. cerevisiae cells nents. contrast, virtually understood S.           membranes  S. cerevisiae cells grown PYED <br /> pombe various components wall                  medium (Skrzypek et al. 1998) radiolabel sphingolipids. organized functional unit subsequent               Plasmids molecular biological techniques: plasmid <br />                                                                     manipulations bacterial transformations standard <br /> synthesis.                                                          techniques (Sambrook et al. 1989). Essential features plas- <br />     present isolation characterization              mid construction described. sequencing plasmid css1 mutants display novel defect cell wall            DNA performed using Thermo Sequenase (Amersham <br /> formation. mutants, large excess &#226;&#65533;&#163;-               Life Sciences, Cleveland, OH) according manufacturer&#226;&#8364;&#8482;s <br />                                                                     instructions. Yeast genomic DNA isolated described <br /> &#226;&#65533;&#164;-glucans accumulates periplasmic space              (Moreno et al. 1991; Hoffman 1993). incorporated cell wall affect-              expression css1&#226;&#171;&#185; control nmt1 promoter, <br /> ing rod shape cells. present evidence            NdeI site introduced initiating methionine codon <br /> Css1p, related human neutral sphingomyelinase,                PCR amplification open reading frame (ORF) essential membrane protein functions ino-             Pfu polymerase. BamHI site introduced just 3&#226;&#172;&#732;                                                                     termination codon. PCR fragment subcloned sitolphosphosphingolipid-phospholipase C S. pombe.               pZERO (Invitrogen, San Diego). css1&#226;&#171;&#185; ORF Css1p substitute loss enzyme activity          removed XhoI BamHI subcloned pREPX Saccharomyces cerevisiae encoded ISC1 (Wells et al.           series fission yeast expression plasmids (Basi et al. 1993; <br /> 1998; Sawai et al. 2000). agonists stresses ac-            Forsburg 1993; Maundrell 1993). expression S. cere- <br /> tivate mammalian sphingomyelinases subsequent                  visiae, fragment subcloned expression <br />                                                                     vector pRS425GAL1 (Mumberg et al. 1994). dropping hydrolysis <span id='am-16' about='obo:IMR_0001361' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0001383 obo:IMR_0000001 obo:IMR_0001362'><span id='am-17' property="rdfs:label" content="sphingomyelin" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001361" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00550" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:17636" datatype="xsd:string"></span>sphingomyelin</span> plasma membrane               internal NdeI fragment pKG1560 (pREP1GFPcss1), generate phosphocholine <span id='am-118' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-119' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span>ceramide</span>, intracellu-                plasmid pKG1845 created fused green fluorescent <br /> lar signaling molecule (Kolesnick Kronke 1998;                  protein (GFP) Css1p residues 332&#226;&#8364;&#8220;424. GFP fusions Kronke 1999; Levade Jaffrezou 1999). agonists               amino acids 332&#226;&#8364;&#8220;389 332&#226;&#8364;&#8220;663 Css1p                                                                     amplification css1&#226;&#171;&#185; sequences Pfu polymerase using stresses include cytokines, interleukin 1&#226;&#65533;&#164; Fas <br />                                                                     oligonucleotides 5&#226;&#172;&#732;-CGTCTTCATATGAGACTTCG-3&#226;&#172;&#732; ei- <br /> ligands, chemotherapeutic drugs, environmental stresses,            ther 5&#226;&#172;&#732;-CTAGGATCCTTATCTACCAAATAGCAATCC-3&#226;&#172;&#732; injury infections. Ceramide thought regulate           5&#226;&#172;&#732;-CTAGGATCCTTACTTTAACCAAGCGGG-3&#226;&#172;&#732;, respectively. <br /> signaling pathways control growth arrest, apoptosis,           PCR products digested NdeI BamHI senescence, immune responses mammals. Pro-                   subcloned pREP1GFP (pKG1538) generate pKG1528 <br />                                                                     pKG1974. GFP fusion protein Css1p residues 1&#226;&#8364;&#8220;331 <br /> cesses regulated sphingolipases identi-            created subcloning css1&#226;&#171;&#185; NdeI fragment fied fungi (Dickson 1998). results suggest              pKG1538. GFP fusion protein Css1p residues 1&#226;&#8364;&#8220;389 <br /> Css1p hydrolyzes complex sphingolipids yield cera-               created inserting css1&#226;&#171;&#185; NdeI fragment (encoding <br /> mide, regulates pathway process essential                residues 1&#226;&#8364;&#8220;331) unique NdeI site pKG1528 create coordination cell wall formation                pKG2138. <br />                                                                        Genomic cloning DNA sequencing: css1-3 strain <br /> growth S. pombe. example sphingol-        transformed S. pombe genomic library constructed <br /> ipase essential fungal cell growth.                     plasmid vector pUR19 (Barbet et al. 1992),                                                                     Ura&#226;&#171;&#185; colonies grow 36&#226;&#172;&#353; isolated. Plasmid <br />                                                                     DNAs recovered colonies comparison               MATERIALS METHODS                                 genomic DNA inserts restriction mapping Southern <br />                                                                     blotting revealed contained overlapping DNA se- <br />    Yeast strains, methods, media: S. pombe strains used      quences. region clones responsible comple- study listed Table 1. grown yeast          mentation css1-3 narrowed constructing subclones <br /> extract medium (YE) minimal medium appropriate              testing ability rescue. DNA se- <br /> supplements (Moreno et al. 1991). Crosses performed            quences ends smallest rescuing fragment (MluI- glutamate medium (minimal medium lacking ammonium                MluI) determined compared sequences <br /> chloride containing 0.01 m glutamate, pH 5.6). Random           databases. end, DNA sequence matched <br /> spore tetrad analyses performed described               terminal segment cosmid c32F12 S. pombe genome <br /> (Moreno et al. 1991). Transformations performed             sequencing project database. end did match electroporation (Prentice 1992). regulated expression           sequence database did match region adjacent css1&#226;&#171;&#185; nmt1 promoter (Maundrell 1993), cells          orp2&#226;&#171;&#185; gene (Leatherwood et al. 1996).  grown minimal media lacking thiamine allow             &#226;&#181;&#8216;800-bp gap ends DNA fragments. <br /> expression addition 5 &#226;&#65533;&#174;m thiamine repress         DNA sequence gap region determined <br /> expression. Double mutant strains constructed iden-        strands using custom-synthesized oligonucleotides <br /> tified tetrad analysis random spore analysis. test      allowed identification single open reading frame. <br />                                                    Essential S. pombe Sphingolipase                                              1399 <br />  <br />                                                                TABLE 1 <br />                                                  S. pombe strains used study <br />  <br /> Strain                                                        Genotype                                                      Source <br /> KGY28            h&#226;&#171;&#186;    972 wild type                                                                                       stock <br /> KGY246           h&#226;&#171;&#186;    leu1-32 ura4-D18 ade6-M210                                                                          stock <br /> KGY1145          h&#226;&#171;&#186;    css1-3 leu1-32 ura4-D18 ade6-M210                                                                   study <br /> KGY1205          h&#226;&#171;&#186;    css1-5 ura4-D18 leu1-32 ade6-M210                                                                   study <br /> KGY2037                mei1-102/mei1&#226;&#171;&#185; css1-3/css1&#226;&#171;&#185; lys1-3/lys1&#226;&#171;&#185; leu1-32/leu1&#226;&#171;&#185; ura4-D18/ura4&#226;&#171;&#185; ade6-M210/ade6&#226;&#171;&#185;               study <br /> KGY2351          h&#226;&#171;&#186;    css1::css1-myc/Kanr ura4-D18 leu1-32 ade6-M210                                                      study <br /> KGY2378          h&#226;&#171;&#186;    css1-2 ura4-D18 leu1-32 ade6-M210                                                                   study <br /> KGY2380          h&#226;&#171;&#186;    css1-4 ura4-D18 leu1-32 ade6-M210                                                                   study <br /> KGY2482          h&#226;&#171;&#186;    css1-2/css1::ura4&#226;&#171;&#185; leu1-32/leu1-32 ura4-D18/ura4-D18 ade6-M210/ade6-M216                            study <br /> KGY2550          h&#226;&#171;&#186;    css1-2                                                                                              study <br />  <br />  <br />  <br />    integration mapping, genomic fragment shown            &#226;&#65533;&#164;-(1,3)-linked glucan (Smith McCully 1978). immu- <br /> Figure 4A subcloned pJK210, integrating vector           nostaining, cells fixed mixture <span id='am-96' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-97' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> contains ura4&#226;&#171;&#185; selectable marker (Keeney Boeke          glutaraldehyde (Balasubramanian et al. 1997) 1994). construct linearized NdeI integrated        stained 9E10 monoclonal antibody followed Alexa- genome css1-3 ura4-D18 strain homologous          conjugated goat anti-mouse IgG. GFP-Css1p fusion proteins <br /> recombination. outcrosses integrant strain wild         visualized live cells. <br /> type, ura4&#226;&#171;&#185; marker segregated css1&#226;&#171;&#185; phenotype             Analysis cell wall polysaccharides: slightly different <br /> 14 14 tetrads analyzed.                                       procedures used analyze cell wall polysaccharide <br />    sequencing css1 mutant alleles, genomic DNA            composition. based fact metabolic <br /> prepared relevant strains css1 coding region        conversion uniformly labeled [14C]glucose labeled man- amplified PCR following oligonucleotides:            nose galactose occurs rapidly, leading homogeneous <br /> css1-F6 (5&#226;&#172;&#732;-TTGACGGCAATAACTCCG-3&#226;&#172;&#732;) css1-R6 (5&#226;&#172;&#732;-TA                labeling cell wall component. procedure, <br /> AATTTATGATTTAGC-3&#226;&#172;&#732;). PCR products sequenced                 labeling fractionation cell wall polysaccharides directly using thermosequenase cycle sequencing kit              described (Arellano et al. 1997) presented (Amersham) customized oligonucleotides.                          Figure 3A. Briefly, exponentially growing cultures S. pombe, <br />    Gene deletion: delete css1&#226;&#171;&#185; S. pombe genome,        incubated 25&#226;&#172;&#353;, supplemented [U-14C]glucose <br /> HindIII fragment containing majority open reading         (1 &#226;&#65533;&#174;Ci/ml) incubated additional 4 hr different <br /> frame replaced ura4&#226;&#171;&#185; gene  Figure 4).             temperatures (25&#226;&#172;&#353;, 28&#226;&#172;&#353;, 32&#226;&#172;&#353;, 37&#226;&#172;&#353;). Cells harvested <br /> linear fragment construct including css1&#226;&#171;&#185; 5&#226;&#172;&#732; 3&#226;&#172;&#732;        total glucose incorporation monitored measuring <br /> flanking sequences transformed diploid strain        radioactivity TCA insoluble material. Mechanical relevant genotype css1&#226;&#171;&#185;/css1-2. Stable Ura&#226;&#171;&#185; trans-       breakage cells using prechilled glass beads added <br /> formants selected 25&#226;&#172;&#353; screened tempera-         cells, lysis achieved Fast-Prep FP120 <br /> ture-sensitive growth. Temperature-sensitive diploids           (BIO 101, Vista, CA; Savant, Farmingdale, NY), using presumably css1::ura4&#226;&#171;&#185;/css1-2 screened PCR          15-sec intervals 5.5 speed. Cell walls pelleted 1000 &#226;&#171;&#187; <br /> Southern blotting appropriate gene replacement.              g 5 min washed 5% NaCl times    Epitope tagging css1&#226;&#180;&#8482;: genomic version css1 encoding      1 mm EDTA times. Aliquots (100 &#226;&#65533;&#174;l) total wall <br /> 13 copies myc epitope fused C terminus              incubated 100 units Zymolyase 100T (Seikagaku <br /> Css1p generated method described Bahler et             America, Ijamsville, MD) Quantazyme (Quantum Biotech- <br /> al. (1998). Oligonucleotide primers used amplify         nologies, Blaine, WA) 36 hr 30&#226;&#172;&#353;. Aliquots enzyme <br /> myc13-kan cassette (Bahler et al. 1998), PCR frag-           included control. samples centrifuged ment transformed diploid cells (KGY1218). cells         supernatant washed pellet counted separately. allowed recover 12 hr YE plates replica      supernatant Zymolyase 100T reaction  <br /> plated YE plates containing 100 &#226;&#65533;&#174;g/ml G418 (Geneticin;          sidered &#226;&#65533;&#164;-glucan plus galactomannan pellet GIBCO BRL, Grand Island, NY). G418-resistant colonies           &#226;&#65533;&#163;-glucan. supernatant Quantazyme reactions <br /> screened homologous recombinants PCR; presence            considered &#226;&#65533;&#164;-glucan pellet &#226;&#65533;&#163;-glucan epitope confirmed immunoblotting              plus galactomannan. <br /> 9E10 antibody. Sporulation tetrad analysis showed 2:2               second procedure (Figure 3B Table 2), wild- <br /> segregation G418 resistance, confirming hap-         type css1-2 cells grown 23&#226;&#172;&#353; 75 ml YE 1% <br /> loid strain, css1::css1-myc, viable.                             glucose, 1 &#226;&#65533;&#174;Ci/ml d-[14C(U)]glucose (300Ci/mmol; Ameri- <br />    Fluorescence microscopy: fluorescence microscopy          Radiochemical Corporation, St. Louis) OD600 performed using Zeiss axioscope appropriate set         0.3. Cultures harvested shifted 36&#226;&#172;&#353; 12 hr <br /> filters. Calcofluor used detect cell wall material live    harvesting. Cells washed PBS-10 mm sodium formaldehyde-fixed cells, DNA visualized using 4&#226;&#172;&#732;,6-          azide, resuspended PBS containing 1 mm phenylmethylsul- <br /> diamidino-2-phenylindole (DAPI), actin visualized            fonyl fluoride (PMSF), subjected mechanical breakage <br /> using rhodamine-conjugated phalloidin described (Bala-            glass beads &#226;&#172;&#381;95% cells lysed (microscopic <br /> subramanian et al. 1997). cell wall visualized       observation). glass beads allowed settle staining formaldehyde-fixed cells 0.5 mg/ml methyl blue         recovered homogenate, washes, centri- <br /> (M-6900; Sigma, St. Louis) PBS (Kippert Lloyd 1995).          fuged 1600 &#226;&#171;&#187; g 10 min pellet cell wall. Methyl blue contains aniline blue stains specifically           supernatant representing total intracellular material  1400                                                       Feoktistova et al. <br />  separated 100,000-g pellet (intracellular organ-           tube (Kimble  73785, 15 ml, screw thread, Teflon-lined <br /> elles) cytosol. total radioactivity incorporated          cap) stream N2. dried lipids suspended fraction determined total acid hydrolysis           270 &#226;&#65533;&#174;l buffer [final concentrations: 250 mm 2-(N-Mor- <br /> liquid scintillation counting. wild-type cells tempera-     pholino)ethanesulfonic acid monohydrate (pH 6.0), 0.6% <br /> tures, css1-2 mutant 23&#226;&#172;&#353;, 15&#226;&#8364;&#8220;20% total        n-octyl-&#226;&#65533;&#164;-glucoside, 5 mm MgCl2] treated 10 min radioactivity incorporated cell wall fraction.        sonic water bath. Membrane protein (30 &#226;&#65533;&#174;l, 75&#226;&#8364;&#8220;300 &#226;&#65533;&#174;g) css1-2 mutant grown 36&#226;&#172;&#353;, 50&#226;&#8364;&#8220;55% total radioac-        added reaction incubated intermittent shak- <br /> tivity incorporated cell wall fraction.           ing 2 hr 30&#226;&#172;&#353;. reaction stopped adding <br /> wall fractions boiled 5 min, washed 10 times             0.9 ml 2.27% sodium dodecylsulfate extracted twice <br /> PBS/1 mm PMSF, frozen &#226;&#171;&#186;70&#226;&#172;&#353;.                                     equal volume methyl-tert-butylether. upper <br />    Cell walls fractionated according Manners               layer transferred glass tube extracted 1 ml <br /> Meyer (1977) minor modifications. Briefly, walls              water. upper organic layer containing [3H]cera- extracted times 3% NaOH 60 min 75&#226;&#172;&#353;              mide transferred glass scintillation vial, taken dry- obtain, centrifugation, alkali-soluble alkali-      ness stream N2, resuspended 4.5 ml Ultima- <br /> insoluble fraction. alkali-insoluble fraction contained            Gold scintillation fluid (Packard, Meriden, CT), sonicated half &#226;&#65533;&#164;-1,3 glucan, rest            water bath 10 min, counted liquid scintillation <br /> alkali-soluble fraction. quantify &#226;&#65533;&#164;-1,3 glucan, frac-      counter. <br /> tions independently treated &#226;&#65533;&#164;-1,3 glucanase (Zymo-              Hydrolysis radiolabeled S. pombe mannose-inositol-phos- <br /> lyase 100T; Seikagaku America). enzyme recombi-            phoceramide (MIPC) determined drying 1.5 &#226;&#171;&#187; 104 cpm <br /> nant determined completely devoid &#226;&#65533;&#164;-1,6          [3H]inositol-MIPC 4000 cpm [3H]dihydrosphingosine- <br /> glucanase activity inability hydrolyze purified &#226;&#65533;&#164;-1,6        MIPC glass screw-capped tube stream N2, <br /> glucan (pustulan). digestion products separated            followed suspension reaction buffer (final concentra- <br /> P4 (Bio-Rad, Richmond, CA) chromatography character-               tions: 25 mm potassium phosphate, pH 7.0, 2.5 mm dithiothrei- <br /> ized layer chromatography Silica Gel G (solvent             tol, 5 mm MgCl2, 0.6% n-octyl-&#226;&#65533;&#164;-glucoside). sample  butanol/ethanol/<span id='am-48' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span>; 5:3:2. laminari-            sonicated described warmed 30&#226;&#172;&#353;, oligosaccharides (&#226;&#65533;&#164;-1,3-linked glucose oligosaccharides)            reaction started adding 150 &#226;&#65533;&#174;g membranes (total <br /> tected served quantify &#226;&#65533;&#164;-1,3 glucan. &#226;&#65533;&#163;-1,3 glucan           volume 150 &#226;&#65533;&#174;l). Reactions incubated 2 hr obtained void volume P4 chromatography               stopped addition 15 &#226;&#65533;&#174;l 0.5 m disodium ethylenedi- &#226;&#65533;&#164;-1,3 glucanase-treated alkalki-soluble fraction. galacto-      amine acetate, pH 7.1. Samples containing [3H]inositol-MIPC <br /> mannan obtained alkali-soluble fraction selec-         heated boiling water bath 2 min followed tive precipitation (Fehling&#226;&#8364;&#8482;s reagent). <span id='am-102' about='obo:IMR_0002125' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-103' property="oboInOwl:id" content="IMR:0002125" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="MeSH:D009005" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="monosaccharide" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="CHEBI:35381" datatype="xsd:string"></span>Monosaccharide</span> com-            2-min treatment sonic water bath. Reactions containing <br /> position indicated presence mannose, galactose,             [3H]dihydrosphingosine-MIPC vortexed organic sol- <br /> &#226;&#172;&#65533;5% glucose.                                                           vent (5 volumes sample:16 volumes chloroform:16 volumes <br />    Cell fractionation immunoblotting: Protein lysates         methanol) centrifuged pellet denatured proteins. prepared glass bead disruption S. pombe cells          cases 5&#226;&#8364;&#8220;50 &#226;&#65533;&#174;l chromatographed 15 &#226;&#171;&#187; 15-cm buffers: TEG (Katayama et al. 1999), NP-40, SDS-              piece silica gel-impregnated Whatman SG-81 paper treated <br /> lysis (Gould et al. 1991). buffer, protein lysates       ethylenediamine tetraacetate (Steiner Lester heated immediately following lysis 96&#226;&#172;&#353; diluted             1972). chromatogram developed chloroform/ NP-40 buffer. Following removal filtration glass            methanol/4.2 m ammonia (9:7:2, v/v/v), dried, lane <br /> beads, soluble insoluble fractions lysate sepa-       cut 2-cm wide 1-cm tall pieces. pieces rated microcentrifugation 14,000 rpm 20 min 4&#226;&#172;&#353;.           placed 4.5 ml UltimaGold scintillation fluid soni- supernatant fraction added 1&#226;&#65533;&#8222;5 volume 5&#226;&#171;&#187; SDS            cated 3 min counting liquid scintillation <br /> gel sample buffer. pellet resuspended volume             counter. background control sample lacked membranes. 1&#226;&#171;&#187; SDS gel sample buffer equivalent total               Membranes prepared 50&#226;&#8364;&#8220;100 A600 units S. <br /> supernatant fractions. fractions heated 96&#226;&#172;&#353;          cerevisiae cells grown A600 3&#226;&#8364;&#8220;6 YPD medium 30&#226;&#172;&#353;. 5 min. Equal volumes soluble insoluble fractions            cells centrifuged room temperature, washed twice resolved 6&#226;&#8364;&#8220;20% SDS-PAGE, transferred poly-             distilled water, resuspended 1 ml ice-cold extrac- <br /> vinylidene fluoride membrane, Css1p-myc visualized             tion buffer (50 mm potassium phosphate, pH 7.0, 5 mm dithi- enhanced chemiluminescence (ECL) subsequent incu-                othreitol, 1 mm phenylmethylsulfonyl fluoride). half vol- <br /> bation 9E10 antibody (1 &#226;&#65533;&#174;g/ml) peroxidase-                ume glass beads (0.5-mm diameter) added, conjugated goat anti-mouse IgG (1:50,000; Jackson Labora-              sample chilled ice vortexed times tories, West Grove, PA).                                               30 sec 30-sec chilling period vortexing.    Electron microscopy: electron microscopy, 106 css1-2            sample centrifuged 10 min 4000 &#226;&#171;&#187; g Sorvall <br /> cells fixed 6-hr shift 36&#226;&#172;&#353; 1.5% potassium           SS-34 rotor 4&#226;&#172;&#353;. resulting supernatant fluid centri- <br /> permanganate. treated 2% glutaralde-               fuged 1 hr 100,000 &#226;&#171;&#187; g Beckman TLA100 rotor 4&#226;&#172;&#353;. <br /> hyde osmium tetroxide, block stained uranyl acetate,          resulting membrane pellet suspended extraction dehydrated ethanol series. Cells embedded               buffer 20 strokes glass tissue homoge- <br /> spurr BEEM capsules, polymerized vac-              nizer. centrifugation suspension steps repeated <br /> uum 65&#226;&#172;&#353;. Sections cut 75 &#226;&#65533;&#174;m Leica Ultracut,          final membrane pellet suspended extraction <br /> picked grids, stained uranyl acetate lead citrate,       buffer containing 30% glycerol. Aliquots 50 &#226;&#65533;&#174;l frozen imaged Philips 300 electron microscope 60 kV.              stored &#226;&#171;&#186;80&#226;&#172;&#353;. enzyme assay, frozen sample    Phospholipase C assays: phospholipase assays           thawed ice used immediately. Membranes pre- <br /> formed described (Wells et al. 1998) minor modifica-           pared way S. pombe cells grown 25&#226;&#172;&#353; tions. reaction 4 &#226;&#171;&#187; 104 cpm [3H]dihydrosphingo-            yeast extract medium A600 0.5  temperature <br /> sine-labeled inositolphosphoceramide (IPC-3),              shift) switched 36&#226;&#172;&#353; 2 hr incubation. Protein <br /> ceramide composed phytosphingosine &#226;&#65533;&#163;-OH-C26 fatty               concentration determined using Bio-Rad DC pro- <br /> acid (Lester Dickson 1993), 10 nmol nonradioac-             tein assay kit bovine serum albumin standard. <br /> tive IPC-3  S. cerevisiae) dried conical glass        Radiolabeled sphingolipids: S. cerevisiae sphingolipids                                                   Essential S. pombe Sphingolipase                                            1401 <br />  <br /> labeled [3H]inositol purified using published proce-     time nonpermissive temperature increasing <br /> dure (Wells et al. 1998). S. pombe sphingolipids labeled       amounts &#226;&#8364;&#339;cell wall&#226;&#8364;&#65533; material (Figure 1, D&#226;&#8364;&#8220;G). [3H]inositol [3H]dihydrosphingosine similar man- <br /> ner cells grown yeast extract medium.       relative percentages cells containing material [3H]inositol-labeled S. pombe MIPC purified acid         end cell, ends cells, middle <br /> precipitation. Following extraction lipids cells using      cells quantified alleles (Figure 1, H <br /> solvent B (Wells et al. 1998), 37.5 &#226;&#65533;&#174;l Glacial <span id='am-113' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span>       . localization material suggested added milliliter sample stored overnight 4&#226;&#172;&#353;.      onset phenotype independent cell sample centrifuged 10,000 rpm Sorvall SS-34 <br /> rotor 15 min 4&#226;&#172;&#353;. resulting precipitate resus-        cycle position. <br /> pended 0.5 ml solvent C (chloroform/methanol/water               Physiological analysis css1 mutants: determine <br /> 16:16:5, v/v/v) warming 60&#226;&#172;&#353; treating 5 min         long took css1 mutant cells arrest division, sonic water bath. sample dried stream             cell numbers wild-type css1-2 mutant cells N2, resuspended 0.5 ml monomethylamine reagent             quantified shift nonpermissive temperature. <br /> (Clarke Dawson 1981), incubated 30 min 50&#226;&#172;&#353; deacylate glycerolipids. evaporation dryness        wild-type cells continued grow exponen- <br /> resuspension solvent C, radiopurity MIPC          tially, css1-2 cells divided (Figure 2A). Dur- <br /> verified chromatography Whatman HP-K layer             ing experiment, portion css1-2 wild- <br /> plate (200 &#226;&#65533;&#174;m, 10 &#226;&#171;&#187; 20 cm) using chloroform/methanol/               type cells released 36&#226;&#172;&#353; 25&#226;&#172;&#353; determine <br /> 4.2 m ammonia (9:7:2, v/v/v) solvent. Radioactivity          division arrest css1-2 cells reversible. <br /> located using BioScan apparatus. [3H]inositol-sphin- <br /> golipids &#226;&#181;&#8216;90% counts MIPC, 5% IPC,           Unlike wild-type cells, css1-2 cells lost viability rapidly &#226;&#172;&#65533;5% small broad bands unknown composi-           incubation 36&#226;&#172;&#353;, indicating arrest tion near origin. procedures used purify       terminal (Figure 2B). <br /> [3H]dihydrosphingosine-labeled MIPC,               initial analysis asynchronous cul- <br /> purification achieved chromatography Whatman            tures indicated specific time HP-K TLC plate. band MIPC TLC located <br /> (BioScan apparatus), scraped plate, loaded          cell cycle cell wall material deposited css1 <br /> glass column (6 ml  504394; Supelco, Bellefonte, PA),        mutants, wanted determine material <br /> eluted solvent C. Radioactive fractions (0.5 ml)          deposited preferentially site cells <br /> pooled, dried, resuspended 0.5 ml solvent C.              synchronized prior inactivation Css1-2p.                                                                     examine  css1-2 cells G2 phase cell <br />                                                                     cycle selected elutriation shifted                           RESULTS <br />                                                                     36&#226;&#172;&#353;. observed septation index began <br />    Phenotype css1 mutants: visual screen tem-           rise, 50&#226;&#8364;&#8220;60% cells failed proceed perature-sensitive lethal mutants (Balasubramanian et               point (Figure 2C) accumulated calcofluor-stainable <br /> al. 1998), identified mutants accu-            material medial region (Figure 2D). remain- <br /> mulation nonvital dye, phloxin B, uneven                 der cells underwent septation cell division cells. phloxin B-containing plates, por-              normally accumulated material old end <br /> tions mutant cells remained white              cell growth resumed (Figure 2D). <br /> parts pink. Pairwise crosses mutants             observations indicate material likely <br /> indicated contained mutations              deposited site active growth Css1- closely linked, gene, termed css1&#226;&#171;&#185;                  2p inactivated. . css1 mutations recessive; diploids               css1 mutants accumulate &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucans: effort relevant genotype css1-3/css1&#226;&#171;&#185; css1-2/css1&#226;&#171;&#185; grew            determine identity accumulated material, 36&#226;&#172;&#353; (data shown). chose mutants,                 stained css1 mutant cells aniline blue. fluo- <br /> css1-2 css1-3, phenotypic analyses.                 rochrome reported bind preferentially &#226;&#65533;&#164;-(1,3) <br />    address mutant cells stained unevenly            glucans (Smith McCully 1978) stain spe- <br /> phloxin B, grown liquid culture, shifted            cifically cell wall septum S. pombe (Kippert <br /> 36&#226;&#172;&#353; various periods time, stained cal-              Lloyd 1995). material css1 mutants stained <br /> cofluor visualize cell wall material (Figure 1, B            aniline blue (Figure 2E), indicating C). 6 hr, clear mutant cells                portion comprised &#226;&#65533;&#164;-glucan. Treatment accumulating material stained intensely cal-              cells isolated walls recombinant <br /> cofluor 24 hr nonpermissive temperature,              &#226;&#65533;&#164;-1,3 glucanase (Quantazyme) completely inhibited ani- <br /> cells nearly filled material (Figure 1C).            line blue binding (data shown). cell wall, particularly S. pombe           identify material precisely, cell wall  <br /> septum, normally structure stain intensely           stituents isolated characterized growing calcofluor (Figure 1A), mutant designated              cells presence [U-14C]glucose. Incorpora- <br /> css1&#226;&#171;&#185; t stop synthesizing cell wall. cells             tion radioactive glucose cell wall css1-2 stained phalloidin visualize F-actin               cells similar wild type 25&#226;&#172;&#353;, slightly higher <br /> DAPI visualize DNA, clear cyto-               28&#226;&#172;&#353;, considerably higher wild type plasm css1 mutant cells compressed                32&#226;&#172;&#353;  32 45% total glucose incorporated)  1402                                               Feoktistova et al. <br />  <br />  <br />  <br />  <br />                                                                                             Figure 1.&#226;&#8364;&#8221;Phenotype                                                                                          css1 mutants. css1-2 css1-3 cells <br />                                                                                          grown 25&#226;&#172;&#353; midlog phase <br />                                                                                          YE medium, shifted 36&#226;&#172;&#353;                                                                                          indicated number hours, <br />                                                                                          fixed formaldehyde. <br />                                                                                          Wild type grown 25&#226;&#172;&#353;                                                                                            css1-2 cells (B C)                                                                                          stained calcofluor, css1-3 <br />                                                                                          cells permeabilized                                                                                          stained phalloidin (D F) <br />                                                                                          DAPI (E G). Arrows indi- <br />                                                                                          cate regions accumulating                                                                                          calcofluor-staining material.                                                                                          percentage css1-3 (H) css1-2 <br />                                                                                            cells accumulating calcofluor- <br />                                                                                          stainable material end, <br />                                                                                          ends, middle,                                                                                          tips middle                                                                                          quantified. <br />  <br />  <br />  <br />  <br /> 36&#226;&#172;&#353;  33 53%; Figure 3A). dramatic             sured vitro, increased (data shown), sevenfold increase &#226;&#65533;&#163;-1,3 &#226;&#65533;&#164;-1,3            likely feedback regulation prevent cell wall <br /> glucan detected css1-2 mutants grown 12 hr          formation absence division lost 36&#226;&#172;&#353; (Figure 3B Table 2). ratio &#226;&#65533;&#164;/&#226;&#65533;&#163;-         css1 mutants. scenario, glucan synthases glucan wild-type S. pombe        continue produce cell wall material needed. <br /> cell walls, indicating simultaneous increase            Gene cloning: explore molecular basis glucan polymers (Figure 3 Table 2).           css1&#226;&#171;&#186; phenotype, determine phenotype <br /> hand galactomannan signifi-              css1 null allele, cloned css1&#226;&#171;&#185; gene comple- <br /> cantly affected, leading drastic alteration        mentation css1-3 mutant phenotype. Sequence <br /> glucan galactomannan ratio. 23&#226;&#172;&#353; wildtype            analysis smallest rescuing fragment cloned css1-2 mutants exhibited 2&#226;&#8364;&#8220;3 times glucan             DNAs (MluI-MluI Figure 4A) revealed presence galactomannan. 36&#226;&#172;&#353; detected 10 times         single ORF encoded predicted protein prod- glucan galactomannan css1-2 mutant            uct 424 amino acids 48 kD (Figure 4B). data- <br /> (Table 2). changes solely css1            base search predicted <span id='am-53' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-54' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-56' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence mutation temperature shift se did          Css1p revealed hypothetical protein S. cerevisiae <br /> cause major alterations cell wall composition        encoded ORF YER019w  termed ISC1), wild-type S. pombe (Figure 3 Table 2). results       37% identical Css1p (Figure 4B). yeast <br /> suggest css1 mutants, grown restrictive       proteins shared significant sequence similarity, espe- <br /> temperature, lost ability properly coordinate      cially N-terminal 160 amino acids, mam- <br /> glucan polymer synthesis cell growth.         malian neutral sphingomyelinases (Tomiuk et al. 1998). <br /> specific activity (1,3)&#226;&#65533;&#164;-d-glucan synthase, mea-        Like mammalian enzymes, Css1p Isc1p lack                                               Essential S. pombe Sphingolipase                                            1403 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 2.&#226;&#8364;&#8221;Physiological analysis                                                                                     css1 mutant cells. Wild-type css1-2 <br />                                                                                     cells grown 25&#226;&#172;&#353; shifted <br />                                                                                     36&#226;&#172;&#353; indicated number hours. <br />                                                                                       Cell number determined trip- <br />                                                                                     licate. (B) determine percentage <br />                                                                                     survival, samples taken                                                                                     time point shift 36&#226;&#172;&#353;, cell number <br />                                                                                     determined, number <br />                                                                                     cells plated triplicate. num- <br />                                                                                     ber colonies form 25&#226;&#172;&#353; 5 <br />                                                                                     days quantified. (C D) <br />                                                                                     css1-2 cells synchronized early <br />                                                                                     G2 phase centrifugal elutriation                                                                                     shifted 36&#226;&#172;&#353; indicated number <br />                                                                                     hours. septation index deter- <br />                                                                                     mined (C) positions cal- <br />                                                                                     cofluor-stainable material deposited <br />                                                                                     accumulated determined <br />                                                                                     (D). Abnormal deposits material                                                                                     observed middle cells                                                                                     old tips. (E) calcofluor-stain- <br />                                                                                     able material binds &#226;&#65533;&#164;-glucan- <br />                                                                                     specific dye aniline blue. Wild-type cells <br />                                                                                     grown 25&#226;&#172;&#353; css1-2 cells                                                                                     grown 25&#226;&#172;&#353; midlog phase YE  <br />                                                                                     dium, shifted 36&#226;&#172;&#353; 12 hr,                                                                                     stained aniline blue. <br />  <br />  <br />  <br />  <br /> signal sequence N terminus, hydropathy           copy suppressor confirmed integration mapping. <br /> plot suggests presence adjacent membrane-            addition, css1 mutant alleles sequenced spanning domains Css1p C terminus (Figure 4C).           contain single point mutations resulted mammalian enzymes similar               single amino acid substitutions. css1-2 allele, large family Mg2&#226;&#171;&#185;-dependent phosphodiesterases             <span id='am-72' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Gly" datatype="xsd:string"></span>glycine</span> 18 altered arginine; css1-3 allele, <br /> (Tomiuk et al. 1998). fact, subsequent mutagenesis           alanine 291 altered valine; css1-4 al- certain conserved residues predicted involved          lele, arginine 255 altered histidine; css1-5 catalysis abrogates function mammalian             allele, <span id='am-84' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-85' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> 15 altered <span id='am-60' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-61' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span>glutamic acid</span> (Figure <br /> zyme vitro (Rodrigues-Lima et al. 2000; Tomiuk et al.        4B).  concluded cloned DNA repre- <br /> 2000). Css1p contains glutamic acid histidine         sented css1&#226;&#171;&#185; gene. <br /> residues, predicted involved substrate          determine css1&#226;&#171;&#185; essential gene, <br /> binding catalysis, respectively, family       copy gene replaced ura4&#226;&#171;&#185; gene <br /> zymes (Figure 4B).                                              diploid relevant genotype css1&#226;&#171;&#185; ura4-D18/ <br />    rescuing gene encoded css1&#226;&#171;&#185; high          css1-2 ura4-D18 step gene deletion method. <br />  1404                                                    Feoktistova et al. <br />  <br />                                                                     ization Css1p, chromosomal css1&#226;&#171;&#185; locus modi- <br />                                                                     fied encode protein 13 copies myc <br />                                                                     epitope C terminus. haploid strain pro- <br />                                                                     ducing Css1p-myc grew morphology kinetics <br />                                                                     indistinguishable wild-type strain, tagged ver- <br />                                                                     sion Css1p functional. Css1p predicted <br />                                                                     contain transmembrane domains, tested <br />                                                                     Css1p membrane-associated protein. <br />                                                                     Css1p-myc wild-type cells lysed buffers  <br />                                                                     taining detergent, 1% NP-40, 1% SDS, soluble <br />                                                                     insoluble fractions separated centrifuga- <br />                                                                     tion. Immunoblot analysis fractions indicated                                                                     detergent required release significant amounts                                                                     Css1p soluble pool (Figure 4D). indicated <br />                                                                     Css1p released supernatant readily <br />                                                                     degraded.  NP-40 buffer, soluble  <br />                                                                     length protein detected (Figure 4D, lane 3). <br />                                                                        determine intracellular localization Css1p, <br />                                                                     asynchronously growing css1myc cells stained                                                                     antibodies myc epitope (9E10; Figure 5,                                                                     B). Specific staining outlined cell nuclear pe- <br />                                                                     ripheries. Css1p-myc detected speckles <br />                                                                     cytoplasm excluded nu- <br />                                                                     clei. staining patterns consistent lo- <br />                                                                     calization Css1p membrane structures, especially <br />                                                                     plasma membrane, secretory pathway. <br />                                                                     confirm localization pattern, css1&#226;&#171;&#185; ORF                                                                     appended frame 3&#226;&#172;&#732; end gene encoding <br />                                                                     GFP expressed wild-type cells control                                                                     thiamine-repressible nmt1 promoter. Like Css1p- <br />                                                                     myc, GFP-Css1p detected primarily  <br />                                                                     tire cell periphery structures resembling vesicles <br />                                                                     cytoplasm (Figure 5C). Weak staining <br />                                                                     nuclei observed cells. <br />    Figure 3.&#226;&#8364;&#8221;Composition cell wall css1 mutant.          determine predicted transmembrane <br />   relative levels [14C]glucose radioactivity incorpo- <br /> rated cell wall polysaccharide 4-hr labeling     domains required observed localization <br /> shown wild-type (972h&#226;&#171;&#186;) css1-2 (KGY2550) strains        pattern, removed GFP fusion C-terminal <br /> grown different temperatures. Values means       sequences encoding 92 amino acids independent experiments duplicated samples. Standard           predicted contain transmembrane domains. deviations total carbohydrate values shown. (B)         resultant protein, GFP-Css1&#226;&#338;&#172;Cp, present  levels radioactivity [14C]glucose incorporated cell wall polysaccharides wild-type css1-2 mutants         cytoplasm excluded nucleus.  shown. Values milligram wet weight means             like length protein, detected cell duplicate cultures. Measurements individual cell wall         periphery, division plane, nucleus <br /> fractions triplicate culture. Cells      (Figure 5D). Overproduced GFP-Css1&#226;&#338;&#172;Cp ap- <br /> grown 10 hr 23&#226;&#172;&#353; YE, 1% glucose (1 &#226;&#65533;&#174;Ci [U-14C]glu-         peared form aggregates cells. determine <br /> cose/ml), harvested shifted 36&#226;&#172;&#353; 12 hr harvesting. Cell walls isolated fractionated         92 amino acids (332&#226;&#8364;&#8220;424) Css1p <br /> scribed materials methods. Exact values stan-            sufficient membrane localization, dard deviations shown Table 2.                               fused GFP. GFP-Css1p332-424 localized                                                                     cell periphery speckles; staining                                                                     nuclear periphery intense Ura4&#226;&#171;&#185; temperature-sensitive diploids cor-              observed length protein (Figure 5E). rect gene replacement, determined PCR,                   determine putative transmembrane  <br /> sporulated. tetrad, Ura&#226;&#171;&#186; temperature-              mains Css1p sufficient membrane localiza- <br /> sensitive colonies formed (data shown), indicating              tion, Css1p residues 332&#226;&#8364;&#8220;389 fused GFP.  css1&#226;&#171;&#185; essential gene. contrast, ISC1          observed localization GFP fusion protein essential S. cerevisiae viability.                       nuclear cell peripheries (Figure 5F). Finally, <br />    Intracellular localization Css1p: study local-         fused amino acids 332&#226;&#8364;&#8220;363 encoding just                                                      Essential S. pombe Sphingolipase                                           1405 <br />  <br />                                                                  TABLE 2 <br />                                                14 <br />          Incorporation radioactivity [ C]glucose cell wall polysaccharides counts minute wet weight <br />  <br />                                                                                                                    &#226;&#65533;&#164;-/      Glucan <br /> Strain      Temp.    Galactomannan        36&#226;&#172;&#353;/23&#226;&#172;&#353;    &#226;&#65533;&#163;-1,3-Glucan      36&#226;&#172;&#353;/23&#226;&#172;&#353;    &#226;&#65533;&#164;-1,3-Glucan        36&#226;&#172;&#353;/23&#226;&#172;&#353;   &#226;&#65533;&#163;-Glucan   g-mannan <br /> css1-2       23&#226;&#172;&#353;      6,814   &#226;&#171;&#190;   698      1         2,761   &#226;&#171;&#190;   486     1       12,891   &#226;&#171;&#190;   1,160    1          4.7        2.3 <br /> css1-2       36&#226;&#172;&#353;     10,509   &#226;&#171;&#190;   1,001    1.5      20,717   &#226;&#171;&#190;   574     7.5     79,923   &#226;&#171;&#190;   3,048    6.2        3.8        9.6 <br /> WT           23&#226;&#172;&#353;      6,412   &#226;&#171;&#190;   668      1         3,641   &#226;&#171;&#190;   473     1       15,618   &#226;&#171;&#190;   1,975    1          4.2        3.0 <br /> WT           36&#226;&#172;&#353;      6,263   &#226;&#171;&#190;   579      0.98      4,892   &#226;&#171;&#190;   181     1.3     15,461   &#226;&#171;&#190;   1,056    0.99       3.1        3.2 <br />   numbers represent average &#226;&#171;&#190; standard deviation duplicate cultures. Measurements individual cell wall fractions triplicate culture. combined recovery radioactivity different fractions 80% total <span id='am-101' about='rdfs:label' typeof='owl:Thing'>label</span> incorporated cell wall. results representative obtained independent experiments. <br />  <br />  <br />  <br />  <br /> transmembrane domain GFP                         form inositolphosphosphingolipids (Dickson fusion protein localized plasma membrane                     Lester 1999). previously determined S. <br /> apparently membranes surrounding nucleus (Fig-                   pombe contains primarily MIPC small ure 5G). conclude data                     IPC (G. B. Wells R. L. Lester, unpublished  <br /> transmembrane domain Css1p necessary suffi-                   sults). hypothesis explain css1-2 <br /> cient membrane localization. Consistent                  cells stop growing 36&#226;&#172;&#353; abnormal <br /> putative function membrane, length GFP-                     sphingolipids loss Css1p activity. <br /> Css1p GFP-Css1&#226;&#338;&#172;Cp able rescue growth                     examined hypothesis measuring accumu- css1-2 (data shown). GFP-Css1p1-389, lacks                 lation radioactive sphingolipids following shift sequences C-terminal transmembrane                       cells restrictive temperature. time shift- <br /> mains, able localize correctly membranes (Fig-                ing log phase cells 25&#226;&#172;&#353; 36&#226;&#172;&#353;, [3H]inositol ure 5H) unable rescue growth css1-2 (data                 added culture, incubated 4 hr. shown). observation suggests C-termi-                Lipids extracted analyzed. nal extension Css1p provides essential function                   wild-type mutant css1-2 cells contained unrelated membrane targeting.                                        radiolabeled MIPC A600 unit cells <br />    css1 mutant cells maintained normal rod shape                   (data shown). data indicate Css1p spite accumulation glucans. dissimilar                regulating sphingolipid synthesis accumulation. situation loss function overproduction                     determined Css1p hydrolyze IPC. genes involved cell wall biosynthesis causes                 assay activity, css1&#226;&#171;&#185; gene overex- <br /> changes cell cell wall morphology (reviewed                   pressed S. cerevisiae strain RCD318 using galactose- <br /> Arellano et al. 1999).  wanted determine                     inducible GAL1 promoter. S. cerevisiae homolog excess glucans deposited                      css1, ISC1, deleted strain RCD318 strain <br /> periplasmic space cytoplasm. purpose,                lacks phospholipase C activity present wild-type utilized GFP-Css1p332-424 construct visualize                 S. cerevisiae cells hydrolyzes inositolphosphosphin- position plasma membrane relative                     golipids (Wells et al. 1998). example, membranes <br /> deposits. GFP-Css1p332-424 expressed css1-2                 prepared strain RCD318 transformed cells nonpermissive temperature, plasma mem-                 vector (RCD318-GAL1) hydrolyze [3H]IPC-3 <br /> brane clearly longer adjacent cell wall               membranes prepared wild-type JK9-3d S. <br /> separated accumulated material (Figure                   cerevisiae cells (Figure 7A). mem- <br /> 5I). situation observed electron micros-               branes Css1p overexpressing strain (RCD- <br /> copy css1-2 mutant cells (Figure 6). glucans accu-               318GAL1css1) hydrolyzed IPC-3 wild-type <br /> mulated cell wall plasma mem-                       S. cerevisiae membranes (Figure 7A). data strongly <br /> brane did affect integrity appearance                    suggest Css1p able hydrolyze inositolphosphos- cell wall. results consistent                 phingolipids. <br /> observation css1-2 cells resistant                     results shown Figure 7A predict mem- <br /> sensitive cell wall degrading enzymes wild-type                 branes prepared wild-type S. pombe cells cells (data shown).                                                 contain activity hydrolyzes IPC-3. prediction <br />    Css1p sphingolipase: Css1p shows greatest se-                   verified using membranes prepared cells <br /> quence similarity mammalian neutral sphingomy-                       grown 25&#226;&#172;&#353; cells shifted 36&#226;&#172;&#353; 2 hr prior <br /> elinases (Rodrigues-Lima et al. 2000). Fungi                     membrane isolation (Figure 7B). expected contain sphingomyelin; contain sphin-                 membranes prepared css1-2 cells grown  <br /> golipids <span id='am-5' about='obo:IMR_0001385' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="phosphoinositol" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="inositol phosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="phosphoinositol" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001385" datatype="xsd:string"></span>inositol phosphate</span> linked ceramide                strictive temperature (2 hr 36&#226;&#172;&#353;) lack  <br />  1406                                                 Feoktistova et al. <br />  <br />  <br />  <br />  <br />                                                               Figure 4.&#226;&#8364;&#8221;Cloning css1&#226;&#171;&#185;.   Genomic organization css1&#226;&#171;&#185;, <br />                                                            constructs tested css1-3 rescue, deletion strategy. ORF <br />                                                            indicated solid box direction transcription <br />                                                            indicated arrow. X, XbaI; M, MluI; H, HindIII;  Acc 651. (B) <br />                                                            Amino acid sequence alignment Css1p S. cerevisiae Isc1p. <br />                                                            Identical amino acids indicated vertical lines. glutamic <br />                                                            acid histidine residues predicted involved substrate <br />                                                            binding catalysis, respectively, family enzymes                                                            indicated arrowhead box. residues mutated                                                            css1 mutant strains indicated asterisks. (C) hydropathy <br />                                                            plot Css1p potential transmembrane domains near <br />                                                            C terminus. (D) Css1p membrane-associated protein. css1myc <br />                                                            (lanes 1, 3, 5, 7, 9, 11) wild-type (lanes 2, 4, 6, 8, 10,                                                            12) cells lysed detergent TEG buffer (lanes 1, 2, 7, <br />                                                            8), 1% NP-40 NP-40 buffer (lanes 3, 4, 9, 10), 1% <br />                                                            SDS lysis buffer, boiled immediately (lanes 5, 6, 11, 12). <br />                                                            lysates fractionated soluble (lanes 1&#226;&#8364;&#8220;6) insoluble <br />                                                            (lanes 7&#226;&#8364;&#8220;12) fractions. Css1p-myc detected immunoblotting <br />                                                            9E10. <br />  <br />  <br />  <br /> zyme activity (Figure 7B).             ity S. pombe cells behaves like phospholipase type <br /> Unexpectedly, membranes prepared css1-2 grown               C enzyme. Cleavage MIPC phospholipase C-like permissive temperature (25&#226;&#172;&#353;) lacked enzyme           enzyme yield ceramide MIP reaction <br /> activity (Figure 7B). Lack activity cells grown         products. experiments used MIPC pre- permissive temperature probably reflects variant           pared wild-type cells radiolabeled [3H]inositol <br /> protein inactivated membrane prepara-             label polar head group radiolabeled tion enzyme assay, contains low            [3H]dihydrosphingosine label ceramide. radiola- <br /> concentration detergent. reason,           beled product reactions analyzed paper <br /> clear data css1-2 cells lack IPC-3           chromatography using solvent MIP <br /> hydrolase activity present wild-type cells. data        remained origin ceramide moved establish css1 encodes sphingolipid           solvent (Wells et al. 1998). Cleavage [3H-inosi- <br /> hydrolytic activity present S. pombe cells          tol]MIPC gave radiolabeled MIP product (Figure <br /> assay conditions.                                                7C) cleavage [3H-dihydrosphingosine]MIPC gave <br />    verified sphingolipid hydrolytic activ-      radiolabeled ceramide product (Figure 7D). <br />                                              Essential S. pombe Sphingolipase                                          1407 <br />  <br />  <br />  <br />  <br />                                                                                                Figure 5.&#226;&#8364;&#8221;Css1p local- <br />                                                                                             ized membranes. css1myc <br />                                                                                             cells grown 32&#226;&#172;&#353;                                                                                             stained   9E10 (B) <br />                                                                                             DAPI. Wild-type cells express- <br />                                                                                             ing (C) GFP-Css1p, (D) GFP- <br />                                                                                             Css1&#226;&#338;&#172;Cp, (E) GFP-Css1p332- <br />                                                                                             424, (F) GFP-Css1p332-389, (G) <br />                                                                                             GFP-Css1p332-363, (H) <br />                                                                                             GFP-Css1p1-389 grown                                                                                             midlog phase absence                                                                                             thiamine 18 hr. Images                                                                                             live cells captured.   <br />                                                                                             css1-2 cells expressing GFP- <br />                                                                                             Css1p332-424 grown                                                                                             midlog phase 25&#226;&#172;&#353; ab- <br />                                                                                             sence thiamine 14 hr                                                                                             shifted 36&#226;&#172;&#353; 6 hr. Images <br />                                                                                             live cells captured. <br />  <br />  <br />  <br />   sphingolipid hydrolase wild-type S. pombe         turnover activity enzymes important coor- <br /> cells behaves like phospholipase C.                          dinating glucan synthesis cell growth. <br />                                                                   Using different polysaccharide labeling analy- <br />                                                                sis strategies, conclusion reached regard- <br />                      DISCUSSION                                ing increased levels &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucans,                                                                galactomannans, css1 mutant cells. result  <br />    study, presented characterization         sistent accumulated material staining css1 mutants display novel defect cell wall       calcofluor aniline blue, dyes known bind glucan <br /> tabolism. css1 mutants arrest division quickly         polymers. genes encoding <br /> tinue accumulate large amounts &#226;&#65533;&#163;- &#226;&#65533;&#164;-glu-        &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucan synthases S. pombe cells, cans periplasmic space. material      possible prevent glucan deposition making appro- incorporated cell wall, does affect        priate mutant strains. example, css1mok1 rod shape does cause compaction cell&#226;&#8364;&#8482;s        css1cps1 double mutant strains accumulated large <br /> cytoplasm. growth arrest unusual glucan depo-          amounts calcofluor-staining material (data sition observed css1 mutants appear irreversible      shown). surprising given absence <br /> lethal events. css1 mutants defective sphingophos-      single glucan synthase gene, pre- <br /> pholipid-phospholipase C activity   phe-        sumably functional. Typically, strict coordi- <br /> notype points connection sphingolipid             nation synthesis cell wall components <br />  1408                                                       Feoktistova et al. <br />  <br />                                                                        dramatic polymer accumulation, indicating                                                                        synchrony &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucan synthases al- <br />                                                                        tered css1 mutant cells. polysaccharide <br />                                                                        component cell wall, galactomannan, did                                                                        accumulate css1 mutant, appears                                                                        specific defect feedback control &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucan <br />                                                                        synthases. activity does appear                                                                        increased css1 mutants, determined measuring <br />                                                                        total &#226;&#65533;&#164;-glucan synthase activity vitro (data shown), <br />                                                                        appears normal feedback control                                                                        enzymes operative css1 mutants.                                                                        possible discover biochemical nature                                                                        control mechanisms analysis css1-inter- <br />                                                                        acting genes. <br />    Figure 6.&#226;&#8364;&#8221;css1 mutants accumulate glucan polymers               Css1p similar mammalian neutral sphingo- <br /> periplasmic space. Electron micrograph representative             myelinases sequence. enzymes play roles css1-2 mutant cell 6-hr shift 36&#226;&#172;&#353;. Note unperturbed <br /> appearance cell wall.                                           apoptosis, senescence, cell cycle arrest, differentia- <br />                                                                        tion. exact roles remain determined, does <br />                                                                        way activity regulated (Kolesnick growth cell wall, known               Kronke 1998; Kronke 1999; Levade Jaffrezou <br /> synthesis different polymers synchronized.               1999). mammalian enzymes N-terminal <br /> ratio &#226;&#65533;&#163;- &#226;&#65533;&#164;-glucan maintained spite                catalytic domain C-terminal extension relative  <br />  <br />                                                                                                     Figure 7.&#226;&#8364;&#8221;css1 inositol- <br />                                                                                                  phosphosphingolipase-phos- <br />                                                                                                  pholipase C activity.   Hy- <br />                                                                                                  drolysis IPC-3 Css1p <br />                                                                                                  produced S. cerevisiae. Mem- <br />                                                                                                  branes prepared S. cerevis- <br />                                                                                                  iae parental strain JK9-3d,                                                                                                  isc1&#226;&#338;&#172; deletion derivative trans- <br />                                                                                                  formed galactose-induc- <br />                                                                                                  ible css1&#226;&#171;&#185; gene (RCD318- <br />                                                                                                  GAL1css1), deletion mu- <br />                                                                                                  tant transformed vector <br />                                                                                                  (RCD318-GAL1) assayed <br />                                                                                                  hydrolysis S. cerevisiae <br />                                                                                                  [3H]DHS-IPC-3, using pro- <br />                                                                                                  cedure described materials <br />                                                                                                  methods. Cells grown <br />                                                                                                  overnight 30&#226;&#172;&#353; stationary <br />                                                                                                  phase S-leu medium  <br />                                                                                                  taining 2% galactose car- <br />                                                                                                  bon source. Values av- <br />                                                                                                  erage assays &#226;&#171;&#190; standard <br />                                                                                                  deviation. (B) S. pombe css1-2 <br />                                                                                                  cells lack phospholipase activ- <br />                                                                                                  ity hydrolyzes IPC-3. Mem- <br />                                                                                                  branes prepared log <br />                                                                                                  phase (A600 0.5&#226;&#8364;&#8220;0.6) wild- <br />                                                                                                  type 972h&#226;&#171;&#186; css1-2 mutant <br />                                                                                                  cells grown permissive <br />                                                                                                  temperature (25&#226;&#172;&#353;) cells <br />                                                                                                  shifted restrictive tem- <br />                                                                                                  perature (36&#226;&#172;&#353;) grown                                                                                                  2 hr. S. cerevisiae [3H]DHS- <br /> IPC-3 used substrate product ceramide values average membranes prepared cultures &#226;&#171;&#190; <br /> standard deviation. (C D) Phospholipase C activity S. pombe wild-type cells. Membranes prepared log phase <br /> (A600 0.5&#226;&#8364;&#8220;0.6) wild-type 972h&#226;&#171;&#186; css1-2 mutant cells grown permissive temperature (25&#226;&#172;&#353;) cells shifted restrictive temperature (36&#226;&#172;&#353;) grown 2 hr. substrate [3H]inositol-labeled MIPC (C) [3H]dihydrosphingosine- <br /> labeled MIPC (D) isolated strain 972h&#226;&#171;&#186;. Data presented percentage substrate converted radioactive product <br /> [MIP (C) ceramide (D)]. ratio substrate product determined chromatography SG-81 paper described materials methods. <br />                                                  Essential S. pombe Sphingolipase                                         1409 <br />  <br />                                                                    secretory pathway (reviewed Rapoport <br />                                                                    et al. 1996). results suggest  directed                                                                    ER transmembrane domain, portions <br />                                                                    Css1p facilitate transport secretory <br />                                                                    pathway GFP fusions C-terminal domain                                                                    Css1p concentrated juxta-nuclear <br />                                                                    regions length GFP-Css1p. <br />                                                                       S. pombe cells make IPC MIPC (Dickson                                                                    Lester 1999) presence sphingolipids, <br />                                                                    sphingomyelin, reported.                                                                    showed Css1p function clearly necessary    Figure 8.&#226;&#8364;&#8221;Models Css1p function. types sub-         yeast cells&#226;&#8364;&#8482; ability hydrolyze IPC MIPC  strates Css1p act products     MIPC, ceramide generated vitro  Figure <br /> reactions shown. data support model            8 model Css1p function). Css1p <br /> substrates IPC MIPC, exclude models        S. cerevisiae homolog, Isc1p, responsible B C. Cellular processes enzymes targets hydrolysis products discussed text.             type enzymatic activity yeast membrane prepa- <br />                                                                    rations. <br />                                                                       mammalian homolog, nSMaseI, able bacterial sphingomyelinases contains putative             function major sphingolipase membrane prepa- <br /> transmembrane domains (Rodrigues-Lima et al. 2000).                rations vitro (Fensome et al. 2000; Tomiuk et al. 2000). <br /> Css1p organized similarly. mutations            Unlike situation yeast membranes,  css1 mutant alleles isolated map sequences              appear additional isozymes immunodeple- <br /> encoding putative catalytic domain.                 tion nSMaseI does remove neutral <br /> showed region C-terminal transmem-                 sphingomyelinase activity membrane prepa- <br /> brane domains transmembrane domains                    rations. Interestingly, nSMaseI produced bacteria essential function. sequences C-terminal          insect cells possesses sphingolipase activity comparable transmembrane domains essential                  activity membrane preparations, arguing known. showed required proper             class enzyme does require subunits Css1p localization; influence activity           activity (Fensome et al. 2000). S. cerevisiae ap- catalytic domain. Given Css1p behaves               pear require sphingolipid hydrolysis integral membrane protein localized vesicles             tolerate deletion css1 homolog ISC1, S. pombe plasma membrane, surprising             cells survive css1. did detect transmembrane domains essential function.              significant increase S. pombe sphingoli- likely Css1p anchored membranes                 pids css1 mutant cells (data shown) function correctly. closest known relative Css1p            reduced synthesis sphingolipids mammalian neutral sphingomyelinases,                     response drop hydrolysis. <br /> nSMase1, thought localized predominantly                  detectable buildup sphingolipids, <br /> endoplasmic reticulum (ER) membranes (Rodrigues-                   phenotype caused loss css1 function Lima et al. 2000; Tomiuk et al. 2000) sphingo-            likely decrease product sphingoli- <br /> myelinase activity abundant brain plasma mem-           pid hydrolysis, e., ceramide MIP IP (Figure 8, <br /> brane preparations (Tomiuk et al. 2000).              model . role ceramide mammalian cells Css1p plays role ER secretory apparatus            great level increases response plasma membrane remains deter-                extracellular ligands stresses, thought <br /> mined.                                                             act <span id='am-108' about='obo:IMR_0100022' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-109' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:id" content="IMR:0100022" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="second messenger" datatype="xsd:string"></span><span id='am-112' property="obo:IAO_0000115" content="Small molecule that is formed in or released into the cytosol in response to an extracellular signal and helps to relay the signal to the interior of the cell. Examples include cAMP, IP3, and Ca++." datatype="xsd:string"></span>second messenger</span> (reviewed Dickson <br />    unusual feature Css1p membrane-targeting           1998). S. cerevisiae, limited evidence domain. predicted contain membrane-                sphingolipids metabolites serve second mes- <br /> spanning domains likely form hairpin loop             sengers stress responses (reviewed membrane, membrane-spanning domain                   Dickson Lester 1999). data presented Css1p sufficient direct GFP S. pombe mem-             suggest Css1p hydrolyzes sphingolipids generate <br /> branes. Interestingly, GFP fusion transmem-         ceramide, acts signal regulate cell wall <br /> brane domain mammalian nSNase1 resulted                 formation (Figure 8, model . targeting GFP ER mammalian cells                     signaling, functions ceramides (Rodrigues-Lima et al. 2000). unusual,                identified S. cerevisiae. compelling evi- <br /> membrane insertion strategy unique. pro-            dence ceramide enhances transport glycosyl- <br /> teins, including synaptobrevin, known insert                <span id='am-28' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-29' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span>phosphatidylinositol</span> (GPI)-anchored proteins ER membranes C-terminal                    secretory apparatus (Horvath et al. 1994; Skrzypek <br /> membrane-spanning domains transported                    et al. 1997) <span id='am-22' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-26' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipid</span> moiety lipid-linked <br />  1410                                                              Feoktistova et al. <br />  <br /> proteins major constituents cell wall                <br />                                                                              <br /> (Conzelmann et al. 1992; reviewed Dickson 1998).                             possible,  loss Css1p function causes                      <br /> reduction unknown ceramide-anchored cell wall                  <br /> protein(s) function coordinate glucan syn-                       <br /> thesis incorporation wall cell growth                      <br /> (Figure 8, model B). Finally, data exclude                      <br /> possibility Css1p acts lipid phosphodiesters                 <br /> lacking ceramide (Figure 8, model C). S. pombe                   <br /> Css1p essential function, analysis css1-                    <br />                                                                         <br /> interacting genes provide unique opportunity                        <br /> probe role ceramides sphingolipid                         <br /> metabolites yeast cell function.                                           <br /> </body></html>