<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, July 1995, p. 4194&#226;&#8364;&#8220;4197                                                                              Vol. 177,  14 <br /> 0021-9193/95/$04.0010 <br /> Copyright 1995, American Society Microbiology <br />  <br />  <br />  <br />                  MurA (MurZ), Enzyme Catalyzes                   Committed Step <span id='am-1' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-2' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span>Peptidoglycan</span> Biosynthesis, <br />                          Essential Escherichia coli <br />           ERIC D. BROWN,1 EUGENIO  VIVAS,2 CHRISTOPHER T. WALSH,1                                 ROBERTO KOLTER2* <br />                                                                                          1 <br />                 Department Biological Chemistry Molecular Pharmacology Department Microbiology <br />                         Molecular Genetics,2 Harvard Medical School, Boston, Massachusetts 02115 <br />                                             Received 1 February 1995/Accepted 4 1995 <br />  <br />             Escherichia coli gene murZ recently shown encode <span id='am-12' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="UDP" datatype="xsd:string"></span>UDP</span>&#226;&#8364;&#8220;N-acetylglucosamine enolpyruvyl trans- <br />          ferase, catalyzes committed step peptidoglycan biosynthesis (J. L. Marquardt, D.  Siegele, <br />          R. Kolter, C. T. Walsh, J. Bacteriol. 174:5748&#226;&#8364;&#8220;5752, 1992). map position murZ (69.3 min) differed <br />          determined murA (90 min), gene previously proposed encode          activity (P. S. Venkateswaran H. C. Wu, J. Bacteriol. 110:935&#226;&#8364;&#8220;944, 1972). construction <br />          chromosomal deletion murZ plasmid containing murZ arabinose control. Growth cells <br />          containing murZ deletion dependent expression murZ plasmid. conclude          murZ essential gene encodes sole UDP&#226;&#8364;&#8220;N-acetylglucosamine enolpyruvyl transferase E. coli.          simplify nomenclature, recommend murA used designate gene 69.3 min encodes <br />          activity designation murZ abandoned. <br />  <br />  <br />    bacterial cell wall peptidoglycan essential cellular          tated chromosomal sequences cloned component involved maintenance shape protec-                 vector. plasmid carries temperature-sensitive origin tion osmotic shock lysis. Escherichia coli peptidogly-            replication (pSC101-ts) chloramphenicol resistance assembled basic building block composed                   nonpermissive temperature (428C) presence <br /> N-acetylglucosamine (GlcNAc) N-acetylmuramic acid                      antibiotic, chromosomal integrants selected. attached pentapeptide (10). committed step            results generation tandem duplication wild- <br /> peptidoglycan biosynthesis catalyzed UDP-GlcNAc enol-                type mutant alleles cloned gene plasmid <br /> pyruvyl transferase. enzyme stems            sequence copies. subsequent shift unusual reaction mechanism (2) involving transfer enol-             permissive temperature (308C) selects excisants.  <br /> pyruvate phosphoenolpyruvate 3-OH UDP-                      pending flanking sequence used recombina- <br /> GlcNAc status target antibiotic                 tion event, excision results restoration wild-type <br /> fosfomycin (6).                                                            gene creation mutant allele chromosome. Cells <br />    1972, Venkateswaran Wu reported isolation               lost plasmid selected temperature-sensitive fosfomycin-resistant mutant E.                presence pKO3 Bacillus subtilis sacB gene, coli (12, 14). UDP-GlcNAc enolpyruvyl transferase activ-               expression presence sucrose detrimental E. coli <br /> ity mutant resistant fosfomycin vitro.               (1). Figure 1A depicts pKO3-DZ, contains kanamycin <br /> antibiotic-resistant temperature-sensitive allele, desig-              resistance cassette murZ flanking sequences <br /> nated murA, mapped conjugation near                       (denoted L R). kanamycin resistance gene makes <br /> argECBH cluster 90 min (12, 14). recently cloned                  possible selection excision event generating mu- <br /> E. coli gene encoding UDP-GlcNAc enolpyruvyl trans- <br />                                                                            tant allele following shift permissive temperature. ferase screening fosfomycin resistance resulting                                                                            result loss murZ recombination overexpression gene multicopy plasmid (9). <br />                                                                            cointegrant yield DmurZ::kan deletion-insertion muta- map position gene determined 69.3 <br /> min. map position differed murA,                  tion (Fig. 1C). designated gene murZ. Taken  studies                    attempts delete murZ, wild-type E. coli ZK126 <br /> suggested E. coli contain genes encoding                    transformed pKO3-DZ, transformants UDP-GlcNAc enolpyruvyl transferase activities. test                grown overnight 308C Luria-Bertani (LB) broth contain- <br /> hypothesis, set construct null allele murZ.              ing kanamycin chloramphenicol  Table 1 strains <br /> reasoned second gene encoded redundant trans-                  used study). estimated 10,000 transformants ferase, strain containing deletion murZ                   plated prewarmed LB agar plates containing chlor- <br /> viable.                                                                    amphenicol kanamycin incubated 428C    strategy constructing precise deletion murZ                select cointegrants. Typically, 100 colonies ob- <br /> employed gene replacement protocol originally developed               tained. Fifty colonies streaked LB agar <br /> Hamilton et al. (5) refined recently Link et al. (7)           containing kanamycin 5% (wt/vol) sucrose incu- development knockout plasmid pKO3. Plasmid                 bated 308C 24 h select cells plasmid <br /> pKO3 designed replace genomic DNA mu-                     excision resulted replacement murZ mutant <br />                                                                            allele loss plasmid.  isolates <br />                                                                            obtained way retained kanamycin chloramphen- <br />   * Corresponding author. Phone: (617) 432-1776. Fax: (617) 738-           icol resistance, indicating retention plasmid sequences. 7664. Electronic mail address: kolter@mbcrr.harvard.edu.                   retention chloramphenicol resistance growth su- <br />  <br />                                                                     4194 <br />  VOL. 177, 1995                                                                                                                                NOTES          4195 <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 2. Verification gene replacement mutant ZK1745. PCR  <br />                                                                                     plifications purified plasmid DNA single-colony DNA primer                                                                                     (59-TAACCGCATTCATGCTGTGT-39), primer b (59-CACCCCTTGTATTA <br />                                                                                     CTGTTTATGT-39), primer c (59-ACAGAACGCAGTTGATGCGTAG <br />    FIG. 1. Strategy deletion chromosomal copy murZ.               CT-39) shown. Primers c  c) primers b c (b-c) used pKO3-DZ   constructed pKO3 (7), derived gene              amplifications templates: plasmid pJLM2 (containing replacement vectors developed originally Hamilton et al. (5). pKO3-DZ        murZ gene flanking DNA), colony ZK126 (wild-type E. coli), plasmid <br /> tains kanamycin resistance cassette flanked sequence identical   pKO3-DZ (knockout vector containing kanamycin cassette murZ flank- 542 bp E. coli chromosome upstream adjacent start codon     ing sequences; legend Fig. 1), colony ZK1745 (E. coli mutant murZ (flank L) sequence identical 873 bp           ZK126 DmurZ::kn/pBAD30 . <br /> immediately downstream termination codon murZ (flank R). flanking sequences amplified PCR Vent DNA polymerase template plasmid pJLM2 (8) following primers: upstream flank, 59- <br /> CCTTAAGTAAACGACAAAGTGTA-39 (primer 1) 59-CGAGATCTCGC                            mid pBAD30 derived vector pBAD30, series <br /> CCCCTTAGTTTGTTCTCAGT-39 (primer 2), downstream flank, 59-                       pBAD expression vectors recently  <br /> GGGGCGAGATCTCGTAATCGTCTGAGAGCT-39 (primer 3) 59-CCT <br /> TAAGAACAGTTAGGTAAAAA-39 (primer 4) (BglII sites underlined,                 structed aid studying phenotypes null mutations complementarity primers 2 3 shown italic). resulting PCR-      essential genes (4). Gene replacement experiments amplified flanks L R ligated complementary se-          ZK126/pBAD30 pKO3-DZ performed identically <br /> quences   annealing (2 min 988C, 10 min 658C, 10 min 378C),       described  integration 428C LB (ii) extension Klenow polymerase 15 min 228C followed denaturation 10 min 758C, finally (iii) PCR amplification Vent      agar containing ampicillin, chloramphenicol, kanamycin, DNA polymerase primers 1 4. resulting L-R fragment (1,443 bp)           0.2% arabinose. Colonies 428C plates subse- blunt ligated EcoRV-digested pBluescript II SK create plasmid      quently streaked LB agar containing ampicillin, kanamycin, <br /> pBlue-L R. kanamycin resistance cassette 1,706-bp BamHI fragment          5% sucrose, 0.2% arabinose incubated 308C. from mini-Tn10kan (13) ligated unique BglII site vector <br /> pBlue-L R create pBlue-L Kn R. kanamycin resistance cassette             sucrose-containing plates, number isolates flanks digested plasmid pBlue-L Kn R (3,188-bp SalI-BamHI            chloramphenicol sensitive growth LB <br /> fragment) ligated SalI-BamHI-digested plasmid pKO3 produce              agar containing kanamycin ampicillin dependent pKO3-DZ. Plasmid pBAD30 (B) contained murZ gene control             presence medium 0.2% arabinose col- arabinose promoter Pbad. murZ gene (sequence, 219 stop) amplified PCR Vent DNA polymerase (primers 59-GCGGCTAGCTA                   lected. Subsequent P1 transduction kanamycin resistance <br /> ACTGAGAACAAACTAA-39 59-GCGCAAGCTTCGATTATTCGCCTT                                 mutants ZK126/pBAD30 TCAC-39) blunt ligated EcoRV-digested pBluescript SK generate           Ara mutant bearing plasmid (ZK1488/pBAD30  <br /> plasmid pBlue  Subsequently, murZ gene digested pBlue              successful transductants plated  <br /> (1,385-bp XbaI-KpnI fragment) ligated XbaI-KpnI-digested pBAD30 <br /> (C) (4).                                                                            dium containing 0.2% arabinose. Subsequent restreaking                                                                                     transductants LB minimal glycerol plates <br />                                                                                     yielded isolates presence 0.2% arabinose. <br />                                                                                      viability DmurZ::Kanr insertion-deletion mu- <br /> crose easily explained spontaneous mutations                         tant maintained murZ provided sacB. results consistent hypothesis                     trans. murZ essential.                                                                Figure 2 illustrates use PCR confirm replace- <br />    test possibility murZ essential,                      ment murZ kanamycin resistance cassette experiment carried strain ZK126, car-                         mutant (ZK1745). Primers c complementary ried pBAD30 (Fig. 1B), plasmid containing murZ                            sequences flank murZ wild type (ZK126) control arabinose-inducible Pbad promoter. plas-                     location kanamycin cassette mutant ZK1745. <br />                                                                                     Amplification murZ-containing templates, plasmid <br />                                                                                     pJLM2 (containing cloned murZ flanking sequences)                         TABLE 1. Bacterial strains                                  ZK126 chromosome, produced fragments 1,488 bp,                                                                      Source                                                                                     expected intact murZ. Amplification  Strain                         Description                                         primers pKO3-DZ ZK1745 chromo- <br />                                                                      reference <br />                                                                                     failed generate amplified product  theoretical <br /> ZK126             W3110 tna2 DlacU169                              3                product 1,848 bp). suspected difficulty ZK1488            ZK126 Dara                                       Lab stocks       amplifying product large inverted <br /> ZK1745            ZK126 DmurZ::kan(pBAD30                        work <br /> ZK1746            ZK1488 DmurZ::kan(pBAD30                       work <br />                                                                                     repeat 363 bp present 59 39 <br />                                                                                     ends kanamycin resistance cassette. overcome  4196       NOTES                                                                                                                                   J. BACTERIOL. <br />  <br />  <br />                                                                                    growth wild-type ZK126 LB broth inoculation                                                                                    0.75 ml overnight culture diluted                                                                                    OD600 equivalent ZK1745 inoculum (OD600 5 <br />                                                                                    0.05). lag phases growth wild type (2 h) <br />                                                                                    ZK1745 grown presence arabinose (4 h) <br />                                                                                    differed slightly, growth rates final optical densities                                                                                    dissimilar, indicating expression murZ Pbad <br />                                                                                    resulted slight growth differences compared                                                                                    growth wild-type control.  differences                                                                                    lag differences                                                                                    amounts viabilities inocula mutant                                                                                    wild-type cells. <br />    FIG. 3. Growth murZ deletion (ZK1745) presence absence <br />                                                                                        results provide strong evidence support arabinose. growth curves shown 100-ml shaking cultures ZK126     hypothesis murZ, located 69.3 min, essential gene LB (E), ZK1745 LB kanamycin, ampicillin, 0.2% arabinose (h),        question existence second ZK1745 LB kanamycin ampicillin arabinose ({).          UDP-GlcNAc enolpyruvyl transferase coded gene <br /> Prior experiment, 2 ml LB kanamycin ampicillin arabinose inoculated cells single colony ZK1745  LB&#226;&#8364;&#8220; <br />                                                                                    located near 90 min (murA) (12, 14). Consistent kanamycin&#226;&#8364;&#8220;ampicillin&#226;&#8364;&#8220;0.2% arabinose agarose plates), cells grew       conclusion inability candidate open reading <br /> night OD600 0.05. growth curves ZK1745           frames second UDP-GlcNAc enolpyruvyl transferase arabinose (h {), 0.75 ml overnight culture inoculum.   using MurZ sequence probe search available case ZK126, overnight culture diluted OD600 0.05 0.75 ml diluted culture served inoculum. <br />                                                                                    translated sequences E. coli, include complete 80- <br />                     <br /> </body></html>