<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Aug. 1997, p. 4664&#226;&#8364;&#8220;4670                                                                                     Vol. 179,  15 <br /> 0021-9193/97/$04.0010 <br /> Copyright &#194;&#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />  Saccharomyces cerevisiae Mevalonate Diphosphate Decarboxylase <br />     Essential Viability, Single Leu Pro Mutation           Conserved Sequence Leads Thermosensitivity <br />                                         ` S,* DANIEL GUYONNET, <br />                             THIERRY BERGE                                                 FRANCIS KARST <br />            Faculte&#194;&#180; des Sciences, Laboratoire Ge&#194;&#180;ne&#194;&#180;tique Physiologique et Mole&#194;&#180;culaire, Institut Biologie Mole&#194;&#180;culaire <br />                           et d&#226;&#8364;&#8482;Inge&#194;&#180;nierie Ge&#194;&#180;ne&#194;&#180;tique, Universite&#194;&#180; Poitiers, F-86022 Poitiers Cedex, France <br />                                               Received 6 March 1997/Accepted 2 June 1997 <br />  <br />             mevalonate diphosphate decarboxylase enzyme converts mevalonate diphosphate iso- <br />          pentenyl diphosphate, building block isoprenoids. used Saccharomyces cerevisiae temperature- <br />          sensitive mutant defective mevalonate diphosphate decarboxylase previously described (C. Chambon, V. <br />          Ladeve`ve, M. Servouse, L. Blanchard, C. Javelot, B. Vladescu, F. Karst, Lipids 26:633&#226;&#8364;&#8220;636, 1991) charac- <br />          terize mutated allele. showed single change conserved <span id='am-63' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-64' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-67' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> accounts tempera- <br />          ture-sensitive phenotype mutant. Complementation experiments erg19-mutated <br />          background strain ERG19 gene, shown essential gene yeast,          disrupted. Epitope tagging wild-type mevalonate diphosphate decarboxylase allowed isolate          enzyme active form versatile step immunoprecipitation procedure. Furthermore,          course study, observed high level expression wild-type ERG19 gene led lower sterol <br />          steady-state accumulation compared wild-type strain, suggesting enzyme key <br />          enzyme mevalonate pathway regulation. <br />  <br />  <br />    mevalonate (MVA) pathway provides cell nu-                  cumulation fetal brain development phenylketonuria <br /> merous fundamental molecules called isoprenoids, including                  results severe brain damage leading microcephaly sterols, chain <span id='am-37' about='obo:IMR_0200483' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200483" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00399" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Ubiquinone" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ubiquinone</span>, dolichol, prenyl groups           mental retardation (24). MVA-PP decarboxylase prenylated proteins ras, plant hormones, aromatic               early enzyme MVA pathway, phenylacetate effects compounds, (11, 14). reason, groups                  lack fundamental intermediary end prod- <br /> focused pathway, potential target inhibi-             ucts MVA pathway. observations led trials <br /> tors acting antifungal compounds agriculture               using phenylacetate antiproliferative agent malig- <br /> medicine cellular antiproliferative agents means             nant gliomas, form primary brain tumors (23). Recently lowering serum <span id='am-70' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cholesterol</span> levels  review, refer-             Prasanna et al. (22) shown specific inhibition ence 20). instance, simvastatin, inhibitor 3-hy-              MVA-PP decarboxylase phenylacetic acid sodium salt <br /> droxy-3-methylglutaryl&#226;&#8364;&#8220;coenzyme (HMG-<span id='am-58' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span>) reductase                    (NaPA), used lovastatin, specific inhibi- <br /> used hypocholesterolemic agent terbinafin, inhibitor               tor HMG-CoA reductase, reverse malignant phe- fungal <span id='am-32' about='obo:IMR_0200481' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="oboInOwl:id" content="IMR:0200481" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Squalene" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00751" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>squalene</span> epoxidase, ketoconazole, inhibitor               notype human glioblastoma cells. fungal <span id='am-17' about='obo:IMR_0200096' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200096" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C01724" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Lanosterol" datatype="xsd:string"></span>lanosterol</span>-14-demethylase, known anti-                  temperature-sensitive yeast strain deficient fungal agents commonly used human disease therapy.                   MVA-PP decarboxylase activity isolated characterized <br />    structural genes coding enzymes involved             years ago (10), far cloning functional complemen- sterol biosynthesis cloned characterized              tation enzyme&#226;&#8364;&#8482;s structural gene failed, mainly  organisms, regulation pathway higher                    cause relatively high reversion rate MN19-34 <br /> eucaryotes documented (14). substantial                 mutant deficient MVA-PP decarboxylase. <br /> tribution cloning genes come                      Recently Toth Huwyler (26) reported isolation isolation collection ergosterol auxotrophic mutants gen-            cDNAs encoding rat, human, yeast MVA-PP decarbox- <br /> erated yeast Saccharomyces cerevisiae (17). sterol           ylase  called MPD) help <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> se- <br /> biosynthesis structural genes cDNAs cloned                  quence data obtained enzyme purified rat liver <br /> functional complementation mutants (13, 18, 21).                   (27). <br />    enzymes MVA pathway, MVA diphosphate                          paper report cloning gene encoding (MVA-PP) decarboxylase known.                  yeast MVA-PP decarboxylase functional complementation <br /> enzyme converts MVA-PP <span id='am-27' about='obo:IMR_0200287' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="rdfs:label" content="Isopentenyl diphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00129" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200287" datatype="xsd:string"></span>isopentenyl diphosphate</span> (IPP),                  yeast mutant isolated laboratory building block isoprenoids, requires <span id='am-47' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-48' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> Mg21                gene essential yeast cell viability. report activity (8).                                                       cloning sequencing mutated recessive allele.    MVA-PP decarboxylase shown competitively                  localized mutation conserved amino acid. inhibited phenylalanine derivatives, phenylpyruvate,             single change leucine proline suffi- <br /> phenylacetate, phenylalanine itself (9). Phenylacetate ac-              cient confer temperature-sensitive phenotype strain <br />                                                                             harboring allele. <br />  <br />   * Corresponding author. Mailing address: IBMIG, Laboratoire Ge&#194;&#180;ne&#194;&#180;tique Physiologique et Mole&#194;&#180;culaire, 40 avenue du Recteur                                   MATERIALS METHODS <br /> Pineau, F-86022 Poitiers Cedex, France. Phone: (33) 0549-45-40-29.            Cloning characterization ERG19 gene. yeast genomic library <br /> Fax: (33) 0549-45-35-03. E-mail: berges@hermes.univ-poitiers.fr.            generous gift F. Lacroute. library, yeast genomic DNA <br />  <br />                                                                      4664 <br />  VOL. 179, 1997                                                          EFFECTS MVA-PP DECARBOXYLASE GENE VIABILITY                                          4665 <br />  <br />  <br /> fragments partially digested Sau3A ligated high-copy-number plas-       (3,000 3 g, 2 min), supernatant kept (SUP fraction) pellet mid pFL44L (7). gene marker used select transformants URA3              washed times potassium phosphate buffer. pellet eventually <br /> gene. library used transform temperature-sensitive strain MN19-34         resuspended sodium dodecyl sulfate-polyacrylamide gel electrophoresis load- <br /> (MATa ura3 trp1), deficient MVA-PP decarboxylase activity             ing buffer. derived original mutant strain MNerg19 (10). Approximately 105            MVA-PP decarboxylase activity determination. assay enzyme activity, independent Ura1 transformants pooled plated selective medium          crude protein extract centrifuged 12,000 3 g prepared described  nonpermissive temperature (36&#194;&#176;C). strains growing 36&#194;&#176;C showed         200 mg protein mixture 10 mM MgCl2, 20 mM ATP, 0.5 mM <span id='am-42' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>, <br /> plasmid-dependent growth kept analysis. strain har-          1 mM glucose-6-phosphate, 2 U glucose-6-phosphate dehydrogenase, 23.5 <br /> bored plasmid containing genomic insert size estimated            mM [2-14C]MVA (53 mCi/mmol) potassium phosphate buffer (0.1 M, pH 7.4) 6.5 kb. unique SalI site unique BglII site located approximately     added, total volume 200 ml. middle 6.5-kb genomic insert used, addition restriction sites      Following 40-min incubation 37&#194;&#176;C, 500 mg cold carrier ergosterol, high-copy-number plasmid pFL44L, subclone original insert.              lanosterol, squalene, squalene oxide added. mixture gently <br /> subclones corresponding SmaI-BglII restriction fragment ;3 kb           saponified 30 min 42&#194;&#176;C presence methanolic KOH (10% final <br /> SalI-SalI restriction fragment ;3 kb subcloned plasmid pFL44S (7)       concentration) extracted heptane. heptane phase concentrated <br /> generate, respectively, pTB1 pTB2. plasmids subsequently           vacuum, radioactivity aliquot measured liquid scin- <br /> transformed temperature-sensitive strain MN19-34. pTB2 reversed         tillation. main fraction separated Silica Gel G developed phenotype mutated strain, e., allowing growth 36&#194;&#176;C. insert        dichloromethane. distribution radioactivity determined tained pTB2 partly deleted using hand XbaI site           Berthold 2382 layer chromatography (TLC) scanner. assay activity located insert multiple cloning site generate pTB3      immunoprecipitated protein, experiment performed hand PstI site generate pTB4. complementing activity        presence immobilized fusion protein immunoprecipitated  <br /> restricted 1.94-kb PstI-SalI restriction fragment pTB4,      ditions described (200 mg protein). <br /> fully sequenced.                                                                         Sterol extraction analysis. Sterol extraction essentially  <br />    Disruption ERG19 gene. order inactivate ERG19 gene,           scribed previously (16). Sterols quantified UV absorption (% 5,7 5 <br /> BglII restriction sites generated PCR. sequences oligo-        OD281.5 3 volume heptanolic phase milliliters 3 396/11,500 3 dry <br /> nucleotides 59TTTAGATCTGGTGCTCACTGTCTGG39 (195BG),                         weight milligrams, %5,7 percentage dry weight D5,7 designed create BglII site position 538  PstI site), 7     sterols [sum dry weight percentages ergosterol ergosta-5,7-dienol]), <br /> nucleotides upstream ATG start codon, 59ATTAGATCTTGAGT                   various species analyzed <span id='am-6' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span>gas</span>-liquid chromatography. gas <br /> TGCAAAAGG39 (193BG), designed create second BglII site              chromatograph (GC 6000; Carlo-Erba) supplied flame-ionization <br /> position 1620, 112 nucleotides upstream TAA stop codon. pUC18-           detector column injector. column RSL 150 (Altech) <br /> 295 construct used template PCRs. PCR product 1 (PCR1)            capillary. Cholesterol (Sigma) used internal standard determine 540 bp corresponding promoter ERG19 gene                 relative retention times  minutes), follows: squalene, 0.72; <br /> obtained universal M13 195BG primers. PCR product 2 (PCR2)                cholesta-8,24-dienol (zymosterol), 1.06; ergosta-5,7,22-trienol (ergosterol), 1.09; 320 bp corresponding 39 ERG19 gene obtained         ergosta-8,24(28)-dienol (fecosterol) ergosta-8,14-dienol (ignosterol), 1.13; reverse 193BG primers. PCR1 digested PstI BglII,         ergosta-8-enol, 1.14; ergosta-7,24(28)-dienol (episterol) ergosta-5,7-dienol, <br /> PCR2 digested BglII SalI. fragments ligated           1.16; 4,4,14-trimethylcholesta-8,24-dienol (lanosterol), 1.24. <br /> pUC18 vector digested PstI SalI, generating pUC18-D19 interme- <br /> diary construct, coding sequence ERG19 gene deleted.                                                                                                                RESULTS <br />    TRP1 gene prepared YDp-W plasmid 0.8-kb restriction <br /> fragment ligated pUC18-D19 cut BglII dephosphorylated,                 Cloning characterization ERG19 gene. generating pUC18-D19W plasmid. pUC18-D19W plasmid cut                isolated genomic DNA fragment able reverse temper- PstI SalI order purify 1.66-kb restriction fragment harboring      ature-sensitive phenotype MN19-34 strain deficient erg19::TRP1 allele, subsequently transformed 5302 (trp1/       MVA-PP decarboxylase activity. <br /> trp1 ura3/ura3) diploid strain (19). Trp1 transformants sporulated, tetrads dissected.                                                                  insert size plasmid library (pDG295) <br />    Epitope tagging MVA-PP decarboxylase. domains corresponding           carrying complementing activity estimated 6.5 kb. immunoglobulin G (IgG) binding domains protein Staphylo-             limited restriction map developed permit choice <br /> coccus aureus fused frame N terminus MVA-PP decarbox-           convenient restriction sites subcloning high-copy- <br /> ylase. 1.1-kb BamHI-SacI restriction fragment corresponding NOP1 <br /> promoter domains protein prepared                  number plasmid pFL44S (7). <br /> pProtA-NOP1 plasmid (5). order fuse frame coding sequence            subcloning steps (Fig. 1; Materials ERG19 gene, designed PA19 oligonucleotide contains SacI                Methods) complementing activity restricted restriction site just downstream ATG codon (59TTTGAGCTCGACC                  1.94-kb restriction fragment sequence determined. <br /> GTTTACACAGCATC39). ERG19 gene amplified PCR PA19 reverse primers, pUC18-295 construct template.           head head open reading frames (ORFs) 1.4-kb PCR product digested SacI SalI ligated          insert: incomplete ORF encoding 102 amino acids 1.1-kb BamHI-SacI fragment (NOP1 promoter plus domains)             identified sequence comparison portion yeast <br /> pFL44L plasmid cut BamHI SalI generate pPA295-1 plasmid.              COQ2 gene coding N-terminal para-hydroxy- primers procedure used clone erg19 mutated <br /> allele temperature-sensitive MN19-34 strain (giving pPA295ts-1           benzoate polyprenyltransferase (3). second ORF com- <br /> construct).                                                                           plete corresponds MVA-PP decarboxylase coding <br />    tagged gene prepared 2.5-kb BamHI-SalI restriction fragment          sequence reported course study cloned pRS316 single-copy plasmid giving pPA295-2 (wild-type         (MVD1 gene) (26). <br /> ERG19) pPA295ts-2 (erg19 allele temperature-sensitive strain) <br /> constructs.                                                                               confirmed gene able complement <br />    Immunoprecipitation tagged protein. Immunoprecipitation pro-         MN19-34 mutant corresponds structural gene coding <br /> tein fusion proteins described Berge`s et al. (5)          MVA-PP decarboxylase. <br /> modifications. Cells grown saturation selective medium (100 ml),              ERG19 gene essential viability. order inac- <br /> washed potassium phosphate buffer (0.1 M, pH 7.4), eventually resus- <br /> pended potassium phosphate buffer (0.1 M, pH 7.4) containing 0.5 mM EDTA <br />                                                                                       tivate ERG19 gene, replaced coding se- 10 mM dithiothreitol. cells broken glass beads, debris      quence TRP1 gene having generated convenient eliminated short centrifugation 3,000 3 g. homogenate       restriction sites PCR. construct, deletion includes <br /> centrifuged 30 min 12,000 3 g 4&#194;&#176;C, protein         ATG start codon ERG19 37 codons crude extract  micrograms milliliter) estimated performing following calculation: OD228.5 2 OD234.5 3 0.317 3 dilution factor (TOT frac- <br />                                                                                       C-terminal remain coding sequence. tion), OD228.5 optical density 228.5 nm.                              deleted erg19::TRP1 gene inserted homologous <br />    700 ml potassium phosphate buffer (0.1 M, pH 7.4) 4 mg protein            recombination 5302 (trp1/trp1 ura3/ura3) diploid strain Sepharose (Sigma) 10 mg rabbit IgG  molecule; Sigma)              (19). Tetrad dissection resulted 2:2 segregation added, mixture incubated 30 min. Unbound IgG eliminated washing potassium phosphate buffer. pellet 700 ml potassium <br />                                                                                       viability (Fig. 1), viable spores auxotrophs phosphate buffer 300 500 mg protein crude extract added,        tryptophan (data shown).  pTB4 plasmid mixture incubated 4&#194;&#176;C 1.5 h. short centrifugation          transformed ERG19/erg19::TRP1 trp1/trp1 ura3/ura3 <br />  4666           ` S ET AL. <br />            BERGE                                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                                                    fusion protein efficiently detected Western <br />                                                                                    blot PAP antibodies (peroxidase-antiperoxidase rabbit <br />                                                                                    IgG). observed molecular mass fusion protein <br />                                                                                    (;60,000 Da) consistent predicted molecular mass <br />                                                                                    chimeric ProtA-ERG19 protein deduced se- <br />                                                                                    quence gene (44,000 1 15,000 Da) (Fig. 2). <br />                                                                                       pPA295-1 construct transformed diploid <br />                                                                                    strain copy ERG19 gene disrupted <br />                                                                                    (TB1D strain; ura3/ura3 trp1/trp1 ERG19/erg19::TRP1).                                                                                    haploid Ura1 Trp1 progeny selected sporulation;                                                                                    carried erg19::TRP1 allele plasmid pPA295-1. <br />                                                                                       chimeric gene expressing ProtA-ERG19 fusion pro- <br />                                                                                    tein cloned single-copy vector pRS316, giving <br />                                                                                    plasmid pPA295-2. construct transformed                                                                                    MN19-34 temperature-sensitive strain. transformed strain <br />                                                                                    grew normally 36&#194;&#176;C, showing fusion protein pro- <br />                                                                                    duced single-copy plasmid fully complements mu- <br />                                                                                    tation (data shown). construct transformed <br />                                                                                    TB1D (ERG19/erg19::TRP1) diploid strain obtain                                                                                    strain MVA-PP decarboxylase tagged <br />                                                                                    protein. sporulation, viable Ura1 Trp1 progeny                                                                                    obtained, confirming fusion protein functional <br />                                                                                    (Fig. 4B, lane 2). Western blot analysis showed level <br />                                                                                    production fusion protein single-copy vector <br />                                                                                    severely reduced (data shown). <br />    FIG. 1. Subcloning steps locate ERG19 gene   perform gene <br /> disruption (B). pDG295 plasmid represents original genomic clone <br /> isolated library. various restriction fragments subcloned high-copy-number plasmid pFL44S  Materials Methods),  <br /> struct assayed complementation MN19-34 strain. Tetrad dissection carried YPD. viable spore shown prototrophic tryptophan. panel B, arrows indicate transcription orientation. WT, <br /> wild type. <br />  <br />  <br />  <br />  <br /> diploid strain (TB1D), sporulation induced, hap- <br /> loid progeny analyzed. Haploid Trp1 Ura1 strains obtained did survive 5-FOA-containing medium <br /> (data shown), confirming ERG19 gene dispensable viability yeast. <br />    Epitope tagging protein encoded ERG19. order immunoprecipitate protein encoded ERG19 gene assay MVA-PP decarboxylase activity vitro, chose tag protein IgG binding domain pro- <br /> tein S. aureus. strategy used times allow immunoprecipitation affinity <br /> purification immunolocalization protein compensate lack specific antibodies directed protein (15, 25, 29). epitope domain used consists repetitive IgG binding domains  Z) 15,000 Da fused N terminus amino acid sequence deduced ERG19 gene. construct, transcription chimeric gene driven yeast NOP1 promoter. NOP1 housekeeping gene encoding yeast nucleolar protein fibrillarin, abundant protein <br /> involved ribosome biogenesis highly conserved  evolution. promoter strong                       FIG. 2. Expression ERG19 tagged IgG binding domains protein known promoters like PGK ADH1 promoters,                         yeast.   Schematic drawing ProtA-ERG19 construct. IgG bind- <br /> recognized constitutive relatively strong promoter.                       ing domains   fused N terminus ERG19 coding sequence <br />                                                                                    cloned pFL44S plasmid  Materials Methods). prom, pro- chimeric gene inserted high-copy-number plas-                       moter. (B) pPA295-1 construct fusion protein (3) transformed <br /> mid (pFL44L). resulting plasmid, pPA295-1, trans-                          MN19-34 strain deficient MVA-PP decarboxylase, growth formed MN19-34 strain, replica plating                         assayed YPD 28&#194;&#176;C nonpermissive temperature (36&#194;&#176;C). Controls <br /> Ura1 transformants nonpermissive temperature showed                           untransformed MN19-34 strain (1) MN19-34 strain transformed <br />                                                                                    untagged Erg19p protein pTB4 (2). (C) Immunoblot analysis. fusion protein fully complements temperature-                         crude protein extract prepared MN19-34 strain transformed sensitive phenotype (Fig. 2).                                                      pPA295-1 construct analyzed Western blotting. fusion pro- <br />    crude protein extract prepared strain,                      tein (arrow) detected PAP IgG. <br />  VOL. 179, 1997                                                      EFFECTS MVA-PP DECARBOXYLASE GENE VIABILITY                                    4667 <br />  <br />  <br />                                                                                    TABLE 1. Conversion radiolabeled MVA nonsaponifiable <br />                                                                                                  lipids crude protein extracts <br />                                                                                                     Amt                  % Reaction productb <br />                                                                                         Strain      MVAa <br />                                                                                                     (nmol)   Squalene   Squalene oxide    Lanosterol   Baseline <br />  <br />                                                                                   FL100              0.83       93             2             2            3 <br />                                                                                   erg19::TRP1c       0.42       74             2             5           18 <br />                                                                                   MN19-34            0.07       67             9            NDd          17 <br />                                                                                   MN19-34 1 IPe      0.52       85             5             4            6 <br />                                                                                                                                                                           [2-14C]MVA converted nonsaponifiable lipids. <br />                                                                                     b <br />                                                                                       Percent total nonsaponifiable lipids loaded. Separation TLC,                                                                                   radioactivity measurements performed Berthold 2382 TLC analyzer. <br />                                                                                     c <br />                                                                                       Strain carrying gene erg19::TRP1 (pPA295-1; multicopy, wild-type strain). <br />                                                                                     d <br />                                                                                       ND, detected. <br />                                                                                     e <br />                                                                                       IP, immunoprecipitated protein. <br />  <br />  <br />  <br />    FIG. 3. Immunoprecipitation tagged Erg19p. crude protein extract      mid (pTB4), strain transformed prepared haploid strain carrying erg19::TRP1 disrupted allele trans- <br /> formed pPA295-1 plasmid expressing ProtA-Erg19p fusion protein. <br />                                                                                   pPA295-2 plasmid (tagged ERG19, monocopy). sterols proteins (500 mg) incubated IgG coupled ProtA-Sepharose      mentioned strains analyzed UV absorption <br /> 1 h 4&#194;&#176;C. precipitation, samples corresponding supernatant (SUP    gas chromatography (GC)  Materials Methods). <br /> fraction) (2) pellet (3) loaded sodium dodecyl sulfate-      results obtained independent experiments polyacrylamide gel electrophoresis gel analyzed Coomassie staining   Western blotting (B) using PAP antibodies detect fusion protein <br />                                                                                   shown Table 2. <br /> (arrow). control lane (TOT fraction) (1) corresponds sample          mean value percentage dry weight crude extract immunoprecipitation. Approximately        D5,7 sterols (sum ergosterol ergosta-5,7-dienol) protein loaded lane. Standards kilodaltons.                    wild-type strain 0.80%, MN19-34 strain produced <br />                                                                                   significantly smaller sterols (0.25%). GC <br />                                                                                   analysis revealed mutant strain, ergosterol    cell extract (centrifuged 12,000 3 g) pre-                      significantly detected. contrast wild-type strain, <br /> pared nondenaturing conditions strain                         sterol represents 56%.  producing ProtA-ERG19 fusion protein (pPA295-1 plas-                          appears MN19-34 mutant strain produces 6-fold- <br /> mid). Nonspecific rabbit IgGs preincubated protein                           smaller sterol corresponding wild-type iso- Sepharose added protein extract incubated                       genic strain. noted conditions 90 min 4&#194;&#176;C. sedimentation resin, pro-                     MN19-34 strain sterol auxotrophic 28&#194;&#176;C. <br /> teins extract, supernatant,                           tagged version ERG19 control NOP1 <br /> immunoprecipitation pellet analyzed Coomassie                        promoter, carried single-copy vector (pPA295-2), led staining Western blot analysis using PAP antibodies.                       wild-type-like ergosterol production level 0.86% result experiment shows 90%                      strain carrying erg19::TRP1. contrast, wild-type <br /> fusion protein crude protein extract retained                     ERG19 gene carried multicopy vector (pTB4) pellet (Fig. 3).  fusion protein signifi-                    strain, ergosterol level remains low (0.38%, e., cantly abundant pellet visible Coomassie                   half wild type). GC analysis showed staining, heavy chains IgGs,                    sterol biosynthesis intermediates lower levels protein detected immunoprecipitated pellet.                           associated wild-type strain. <br />    check enzyme prepared condi-                           result suggests high level MVA-PP decar- <br /> tions active, showed crude protein extract                    boxylase gene expression basis small <br /> (centrifuged 12,000 3 g) prepared MN19-34 mu-                         ergosterol produced strain carrying erg19::TRP1 <br /> tant incubated radiolabeled MVA produced                            (pTB4). <br /> small amounts sterols (1.6% MVA converted) compared                            Cloning erg19 allele temperature-sensitive produced extract wild-type strain (17%                         strain. PCR product corresponding erg19 allele coding <br /> MVA converted) produced extract strain                      sequence obtained MN19-34 temperature-sensi- <br /> carrying erg19::TRP1 complemented pPA295-2 (wild-type                        tive strain using oligonucleotides designed ERG19; monocopy) (9% MVA converted), confirming                            allow epitope tagging allele. chimeric ProtA- <br /> leaky phenotype mutant (Table 1). immuno-                        erg19ts allele construction high-copy-number <br /> precipitated fusion protein added MVA-PP                           plasmid pFL44L single-copy plasmid pRS316, generat- <br /> decarboxylase-deficient mixture, significant sterols                  ing, respectively, pPA295ts-1 (multicopy) pPA295ts-2 produced (11% MVA converted), sep-                          (monocopy) plasmids. independent PCR products arated TLC identified (Table 1). results showed                      cloned parallel pUC18 sequencing. addition, MVA pathway restored addition                     erg19 coding sequence cloned expression vector <br /> immunoprecipitated MVA-PP decarboxylase.                                          pPA295ts-2 sequence determined. <br />    Overexpression ERG19 gene impairs MVA path-                            cases, sequencing erg19 allele coding sequence <br /> way. Total sterols extracted control wild-type                        allowed single (T3C) point mutation position <br /> strain (FL100) grown complete medium yeast-peptone-dex-                        779  PstI site), resulting replacement leucine <br /> trose (YPD), MN19-34 strain grown permissive                        79 proline (L79P). <br /> temperature YPD addition exogenous ergosterol,                         pPA295ts-1 pPA295ts-2 plasmids trans- strain carrying disrupted erg19::TRP1 gene com-                      formed MN19-34 strain, Ura1 transformants <br /> plemented ERG19 gene high-copy-number plas-                         restreaked complete medium 36&#194;&#176;C. transfor- <br />  4668           ` S ET AL. <br />            BERGE                                                                                                                                        J. BACTERIOL. <br />  <br />  <br />                                                TABLE 2. Comparative sterol compositions various strains <br />                                                                                                            % Sterol 6 SD <br />            Strain                         % 5,7a <br />                                                                          b <br />                                                               Zymosterol            Ergosterol           Ergost-8-enol           Ergosta-5,7-dienol          Lanosterol <br />  <br /> FL100                                  0.80 6 0.06             16 6 2                 56 6 6                10 6 5                    12 6 1                    664 <br /> MN19-34                                0.25 6 0.02               NDc                  99 6 1                 ND                        ND                       161 <br /> erg19::TRP1 (pTB4)d                    0.38 6 0.02            12.0 6 0.6              75 6 2                 ND                        862                      561 <br /> erg19::TRP1 (pPA295-2)e                0.86 6 0.09              22 6 3                58 6 3                 261                      12 6 2                    461 <br />         % 5,7 percentage dry weight D5,7 sterols (sum dry weight percentages ergosterol ergosta-5,7-dienol). equation % 5,7 given Materials Methods. <br />    b <br />      Values determined GC. <br />    c <br />      ND, detected. <br />    d <br />      Multicopy. <br />    e <br />      Monocopy. <br />  <br />  <br />  <br /> mants obtained pPA295ts-1 grow 36&#194;&#176;C                               long 39-untranslated region (UTR) mRNA  600 extent wild-type strain, transformants ob-                           nucleotides) reported Toth Huwyler (26). result, <br /> tained single-copy plasmid pPA295-2 comple-                             obtained cDNA isolated authors, consis- <br /> ment temperature-sensitive phenotype (data shown).                             tent yeast genome sequencing data.  aver- transformed plasmids TB1D strain. Ura1 transformants sporulated, viable haploid Ura1 <br /> Trp1 strains recovered. strain carrying erg19:: <br /> TRP1 gene produced wild-type ERG19 (pPA295-2; mono- <br /> copy) carrying erg19::TRP1 gene produced mutated erg19 (pPA295ts-2; monocopy) restreaked 36&#194;&#176;C (Fig. 4B). strain wild-type allele grow 36&#194;&#176;C, showing erg19 allele mutant <br /> strain responsible temperature-sensitive phe- <br /> notype MN19-34 strain. <br />    Western blot analysis confirmed fusion protein expressed, apparent molecular mass identical protein produced wild-type ERG19 (Fig. 4C), doubling time permissive temperature slightly enhanced  factor 1.2), suggesting  <br /> zyme MN19-34 mutant, permissive temper- <br /> ature, slightly active wild-type counterpart. proteins prepared 6-h growth 36&#194;&#176;C, difference detected (data shown). concluded defect mutant strain higher turnover messenger RNA ERG19 protein point mutation affect directly catalytic site interaction domain necessary ho- <br /> modimerization. important point Leu-79 <br /> residue residues conserved yeast, <br /> human (26), Arabidopsis thaliana MVA-PP decarboxylases <br /> (11a). <br />  <br />                                DISCUSSION <br />  <br />    paper reports cloning functional complementa- <br /> tion yeast gene encoding MVA-PP decarboxylase. Evi- <br /> dence given mutant strain MN19-34 (10) bears defect MVA-PP decarboxylase structural gene.                                     FIG. 4. mutated erg19 allele corresponds point mutation leading    genomic sequence ERG19 identical                           thermosensitivity.   Sequence comparison erg19 allele cloned PCR <br />                                                                                        temperature-sensitive MN19-34 strain wild-type allele showing <br /> cDNA, reported Toth Huwyler (26). reported                             T3C point mutation changes leucine 79 proline. (B) Dilution authors, ERG19 COQ2 (encoding para-hydroxyben-                               series showing growth strain carrying erg19::TRP1, contains <br /> zoate polyprenyl transferase) transcribed divergent                           monocopy plasmid pRS316 harboring wild-type ERG19 allele tagged promoters, ORFs separated 250 bp.                           protein (pPA295-2 plasmid) (lane 2) harboring mutated erg19 allele <br />                                                                                        tagged protein (pPA295ts-2 plasmid) (lane 3). Growth assayed 28 significantly average length data                      36&#194;&#176;C YPD. MN19-34 thermosensitive strain shown control <br /> yeast genome sequence indicate divergent ORFs                              (lane 1). (C) Immunoblot analysis. Crude protein extracts prepared average 600 nucleotides apart (12). possible                             strain carrying erg19::TRP1 gene produced wild-type ERG19 <br /> physiological significance peculiarity stud-                      (pPA295-2; monocopy) (lane 2) carrying erg19::TRP1 gene <br />                                                                                        produced erg19 mutated allele (pPA295ts-2; monocopy) (lane 3). ied, tempting speculate sharing short                         fusion protein (arrow) detected PAP IgG. Coomassie-stained gel run <br /> promoter area way coregulate related pathways.                           parallel shown control total protein loaded. Protein noticeable feature ERG19 gene unusually                          standards (lane 1) kilodaltons. <br />  VOL. 179, 1997                                          EFFECTS MVA-PP DECARBOXYLASE GENE VIABILITY                                           4669 <br />  <br />  <br /> age 39 UTRs yeast potential genes 160      decarboxylase indirectly, e., checking ability restore <br /> nucleotides (12). expression vectors,     functional MVA pathway leading sterols (squalene, squa- <br /> original genomic clone (pDG295, Fig. 1), half     lene oxide, lanosterol) radiolabeled MVA incu- <br /> region truncated (;150 nucleotides original 39 UTR        bated nonfunctional, MVA-PP decarboxyl- left pPA295 constructs), effect            ase-deficient protein extract. obvious complementation. significance long 39 UTR        assay used enzyme biochemical parameter  <br /> MVA-PP decarboxylase mRNA determined.                  termination, experiment showed tagged <br />    results presented MVA-PP decarboxyl-          MVA-PP decarboxylase isolated active form <br /> ase gene disruption lethal yeast. result expected,   immunoprecipitation procedure.   ergosterol unique end product MVA           zyme preparation procedure allow researchers  <br /> pathway yeast cells impermeable organic py-        say MVA-PP decarboxylase activity vitro various  <br /> rophosphates supplemented IPP,         ditions enzyme prepared native conditions product MVA-PP decarboxylase. similar result           yeast test activity mutant counterpart. described MVA kinase (21), phosphomevalonate ki-           study help design new inhibitors key <br /> nase (28), IPP isomerase (1), <span id='am-22' about='obo:IMR_0200097' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200097" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00448" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Farnesyl diphosphate" datatype="xsd:string"></span>farnesyl diphosphate</span> (FPP)        enzyme study structural functional rela- <br /> synthase (2, 6) gene disruptions, showing essential   tionships vitro-mutagenized ERG19. <br /> genes.  study, exclude presence second gene MVA-PP decarboxylase,                                      ACKNOWLEDGMENTS case HMG-CoA reductase (4).  mu-               thank Johann Joets helpful advice GC analysis tant (MNerg19) defective MVA-PP decarboxylase           various members laboratory helpful discussions described yeast. mutant exhibits temperature-sensi-      course work. grateful Randy Worobo Mat- <br /> tive phenotype, speculated phenotype      thieu gnacq critically reading manuscript. Ed Hurt (Univer- consequence gene inactivation leaky     sity Heidelberg) acknowledged gift pRS316 plasmid. <br /> mutation.                                                              work supported grants GREG/INRA   9/95) <br />    Surprisingly, overexpression wild-type ERG19          French Ministry National Education Research. <br /> gene leads low sterol content yeast disrupted <br />                                                                                                        REFERENCES <br /> ERG19 gene. way account result                                                                      1. Anderson, M. S., M. Muehlbacher,  P. Street, J. Proffit, C. D. Poulter. <br /> argue MVA-PP decarboxylase limiting enzyme             1989. Isopentenyl diphosphate:<span id='am-12' about='obo:IMR_0200094' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00235" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200094" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Dimethylallyl diphosphate" datatype="xsd:string"></span>dimethylallyl diphosphate</span> isomerase. im- <br /> sterol pathway yeast overexpression results           proved purification enzyme isolation gene Saccharo- <br /> accumulation IPP, <span id='am-75' about='obo:IMR_0200399' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-76' property="rdfs:label" content="Geranyl diphosphate" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0200399" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00341" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>geranyl diphosphate</span>, FPP             myces cerevisiae. J. Biol. Chem. 264:19169&#226;&#8364;&#8220;19175. <br /> feed MVA kinase HMG-CoA reduc-                2. Anderson, M. S., J. G. Yarger, C. L. Burck, C. D. Poulter. 1989. Farnesyl <br />                                                                         diphosphate synthetase. Molecular cloning, sequence expression tase. strain able grow temper-          essential gene Saccharomyces cerevisiae. J. Biol. Chem. 264:19176&#226;&#8364;&#8220; <br /> ature, colonies heterogeneous size, suggesting          19184. growth impaired.                                     3. Ashby, M. N., S. Y. Kutsunai, S. Ackerman,  Tzagoloff, P.  Edwards. <br />    mutated gene presents leucine pro-             1992. COQ2 candidate structural gene encoding para-hydroxy- <br />                                                                         benzoate:polyprenyltransferase. J. Biol. Chem. 267:4128&#226;&#8364;&#8220;413 <br /> </body></html>