<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Nov. 2000, p. 8343&#226;&#8364;&#8220;8351                                                                         Vol. 20,  22 <br /> 0270-7306/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />        Functional Interaction Ssu72 Rpb2 Subunit               RNA Polymerase II Saccharomyces cerevisiae <br />                                      DONALD L. PAPPAS, JR.,                MICHAEL HAMPSEY* <br />             Department Biochemistry, Division Nucleic Acids Enzymology, Robert Wood Johnson Medical School, <br />                                                   Piscataway, New Jersey 08854 <br />                          Received 18 July 2000/Returned modification 9 August 2000/Accepted 15 August 2000 <br />  <br />              SSU72 essential gene encoding phylogenetically conserved protein unknown function interacts <br />           general transcription factor TFIIB. recessive ssu72-1 allele identified synthetic enhancer <br />           TFIIB (sua7-1) defect, resulting heat-sensitive (Ts&#226;&#180;&#353;) phenotype dramatic downstream shift           transcription start site selection. new allele, ssu72-2, confers Ts&#226;&#180;&#353; phenotype           SUA7 wild-type background. effort define Ssu72, isolated suppressors ssu72-2 <br />           mutation. suppressor allelic RPB2, gene encoding second-largest subunit RNA polymerase <br />           II (RNAP II). Sequence analysis rpb2-100 suppressor defined cysteine replacement phyloge- <br />           netically invariant arginine residue position 512 (R512C), located homology block D Rpb2.           ssu72-2 rpb2-100 mutations adversely affected noninduced gene expression, apparent effects           activated transcription vivo. isolated suppressor ssu72-2 Ts&#226;&#180;&#353; defect, rpb2-100 enhanced <br />           transcriptional defects associated ssu72-2. Ssu72 protein interacts directly purified RNAP II <br />           coimmunoprecipitation assay, suggesting genetic interactions ssu72-2 rpb2-100           consequence physical interactions. results define Ssu72 highly conserved factor physically <br />           functionally interacts RNAP II core machinery transcription initiation. <br />  <br />  <br />    Eukaryotic RNA polymerase II (RNAP II) multisubunit                  Rpb1 subunit RNAP II, promoter melting promoter <br /> enzyme responsible transcription protein-                 clearance (25). <br /> encoding genes. Saccharomyces cerevisiae RNAP II 12-                       GTFs discovered based requirement subunit complex encoded genes RPB1 RPB12                        accurate initiation vitro (29, 52) highly conserved <br /> viewed references 2, 53, 55). largest subunits,               eukaryotic organisms (reviewed references 17, 30, 33, <br /> Rpb1 Rpb2, homologous &#226;?&#164;&#226;&#172;&#732; &#226;?&#164; subunits                    38). Furthermore, TBP TFIIB requirements bacterial RNA polymerase (RNAP), respectively (23, 49).                  transcription predate divergence Eukarya Archaea <br /> yeast counterpart bacterial &#226;?&#163; subunit appears                   (reviewed reference 5). Nonetheless, RNAP II require- <br /> shared Rpb3 Rpb11 subunits (26, 57). Ac-                     ment GTFs universal instead dictated cordingly, Rpb1, Rpb2, Rpb3, Rpb11 comprise func-                    promoter sequence architecture (16, 35, 51).  ge- <br /> tional equivalent bacterial &#226;?&#163;2&#226;?&#164;&#226;?&#164;&#226;&#172;&#732; core RNAP. func-                nome-wide expression analysis revealed components <br /> tions remaining RNAP II subunits clear,                      RNAP II transcriptional machinery dispensable subunits, Rpb5, Rpb6, Rpb8, Rpb10, Rpb12,                  expression subsets genes (21). Conversely, shared forms RNAP                       limited number promoters analyzed vitro, function transcription genes. exception                conceivable additional GTFs identified based Rpb4 Rpb9, subunits essential cell viability.                requirement accurate initiation specific promot- <br /> Rpb4 forms subcomplex Rpb7 required pro-                  ers. factors identified based ability moter-specific initiation dispensable elongation (13).            stimulate activator-independent transcription. example, <br />    RNAP II unable initiate promoter-specific transcription             yeast Tsp1 (Sub1) protein identified positive  Promoter recognition requires general tran-                  effector vitro transcription absence sequence- <br /> scription factors (GTFs), include TATA binding                     specific activator (20). Tsp1 interacts physically genetically <br /> protein (TBP), TFIIB, TFIIE, TFIIF, TFIIH (reviewed                   TFIIB homologous human transcriptional <br /> references 17 33). TBP nucleates assembly transcrip-                cofactor PC4 (20, 27). Additional basal factors likely tion preinitiation complex binding TATA element                   identified, requirement promoter-specific <br /> inducing sharp bend DNA template. TFIIB binds                   initiation, ability stimulate activator-independent <br /> TATA-TBP complex responsible defining polar-                  transcription, genetic interactions GTFs RNAP II <br /> ity transcription binding asymmetrically BRE ele-               subunits. <br /> ment, located immediately upstream TATA box (28, 50).                    yeast gene encoding TFIIB (SUA7) identified RNAP II, association TFIIF, binds TATA-TBP-                      effector transcriptional accuracy (36). Replacements ei- <br /> TFIIB ternary complex, followed TFIIE TFIIH. TFIIH                    ther phylogenetically invariant residues, glutamate-62 multisubunit complex catalytic activities responsible              (E62) arginine-78 (R78), caused marked start site shift phosphorylation carboxy-terminal repeat domain                 downstream normal site CYC1 ADH1 pro- <br />                                                                              moters (37). Nearly identical effects transcriptional accu- <br />    * Corresponding author. Mailing address: Department Biochem-           racy conferred sua8 alleles RPB1, suggesting istry, Robert Wood Johnson Medical School, 675 Hoes Lane, Piscat-            functional interaction TFIIB Rpb1 start <br /> away, NJ 08854. Phone: (732) 235-5888. Fax: (732) 235-5889. E-mail:          site selection (6). effort identify factors hampsemi@umdnj.edu.                                                          affect initiation, isolated suppressors sua7-1 (E62K) <br />  <br />                                                                       8343 <br />  8344       PAPPAS HAMPSEY                                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> mutation. ssu71 ssu73 suppressors encode altered                             glucose mid-log phase (optical density 600 nm [OD600] &#226;&#171;&#189; 0.6). Cells forms largest subunit TFIIF (Tfg1) Rpb9                            collected, washed, used inoculate 50-ml cultures synthetic complete <br />                                                                                      medium containing 2% glucose 3% glycerol plus 2% ethanol sole <br /> subunit RNAPII, respectively (46, 47). contrast, ssu72                     carbon sources. Cultures incubated 30&#194;&#176;C 8 h, cells har- <br /> identified enhancer TFIIB E62K defect:                              vested. Total RNA isolated S1 nuclease protection assays  <br /> ssu72-1 allele confers heat-sensitive (Ts&#226;&#171;&#186;) growth defect                      formed described previously (22), using synthetic oligonucleotides 5&#226;&#172;&#732;-GTGT dramatic downstream start site shift, effects                      AGCACCTTTCTTAGCAGAACCGGCCTTGAATTCAGTCGGACGG                                                                                      5&#226;&#172;&#732;-GGAATTTCCAAGATTTAATTGGAGTCGAAAGCTCGCCTTA CYC1 <br /> dependent sua7-1 allele (48).                                               tRNAW, respectively. <br />    SSU72 gene essential cell viability encodes                         INO1 transcript levels analyzed Northern blotting. Cells grown novel protein undefined function (48). charac-                         30&#194;&#176;C synthetic complete medium mid-log phase, collected centrifugation, <br /> terize Ssu72, generated new ssu72 allele confers                         washed, used inoculate 50-ml cultures synthetic complete medium <br />                                                                                      lacking containing inositol (55 &#226;?&#174;M). Cultures incubated 30&#194;&#176;C <br /> tight Ts&#226;&#171;&#186; phenotype, independent TFIIB defect.                          8 h, cells harvested. Total RNA extracted analyzed taken advantage ssu72 Ts&#226;&#171;&#186; phenotype isolate new                          described previously (36). INO1 probe generated nick translation rpb2 mutation encoding altered Rpb2 subunit                        1.0-kb INO1 EcoRI-HindIII fragment derived plasmid pJM310 RNAP II. results presented demonstrate Ssu72                           presence [&#226;?&#163;-32P]<span id='am-24' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span>dCTP</span>. U6 control probe nick-translated 450 bp <br />                                                                                      fragment encompassing SNR6 gene. CYC1 INO1 transcript levels transcription factor interacts core RNAP II                       quantified using National Institutes Health Image program (http://rsb  <br /> machinery vivo vitro.                                                 fo.nih.gov/nih-image/) normalized tRNAW U6 RNA loading <br />                                                                                      control signals, respectively. <br />                                                                                         &#226;?&#164;-Galactosidase assays. Strains harboring CYC1-lacZ (pLG265-UP1 [15]), <br />                         MATERIALS METHODS <br />                                                                                      INO1-lacZ (pJS325 [42]), PGK1-lacZ (pN1086 [7]) reporter plasmids    Yeast strains. yeast strains used study derived LRB535      grown 30&#194;&#176;C &#226;&#171;&#186;Ura medium mid-log phase (OD600 &#226;&#171;&#189; 0.6). Cells (MATa his3&#226;&#338;&#172;200 leu2-3,112 ura3-52). YZS84 isogenic LRB535,          harvested resuspended &#226;&#171;&#186;Ura medium containing   2% glucose ssu72-2 allele, generated site-directed mutagenesis subse-      3% glycerol plus 2% ethanol CYC1-lacZ strains, (ii) synthetic complete <br /> quent allele replacement, described  YDP19 (ssu72-2 rpb2-100 shs2-1)        medium containing lacking 55 &#226;?&#174;M inositol INO1-lacZ strains, (iii) spontaneous Ts&#226;&#171;&#185; revertant YZS84; YDP135 (ssu72-2 rpb2-100)              addition PGK1-lacZ strains. Cells incubated 8 h, harvested segregant derived cross YDP19 YDP87 (ssu72-2 RPB2 SHS2).              centrifugation, resuspended 500 &#226;?&#174;l breaking buffer (100 mM Tris-HCl plasmid-shuffle strain YZS89A (ssu72::LEU2 [SSU72-URA3]) described           [pH 8], 1 mM dithiothreitol, 20% glycerol; 2 mM phenylmethylsulfonyl fluoride). <br /> previously (48).                                                                     Cell extracts prepared vortexing 0.5-mm-diameter glass beads    Growth media, genetic methods, phenotypes. growth media              times 15-s bursts. &#226;?&#164;-Galactosidase assays described previously <br /> prepared according standard recipes (43). &#226;&#171;&#186;Ino medium synthetic complete       (24). Activities determined duplicate assays independent trans- <br /> medium lacking inositol; &#226;&#171;&#185;Ino control medium contains 55 &#226;?&#174;M inositol. Stan-          formants expressed according following formula: (1.7 ml &#226;&#171;&#187; OD420 <br /> dard yeast genetic methods used making crosses, selecting diploids,         units)/(0.0045 &#226;&#171;&#187; cell extract volume [milliliters] &#226;&#171;&#187; reaction time [minutes] &#226;&#171;&#187; <br /> inducing sporulation, dissecting tetrads (44). following designations    protein concentration [milligrams milliliter]). <br /> used phenotypes: Ts&#226;&#171;&#186;, temperature sensitivity, defined impaired growth           vitro transcription translation reactions. Radiolabeled TBP, TFIIB, YPD medium 37&#194;&#176;C; Cs&#226;&#171;&#186;, cold sensitivity, defined impaired growth          Ssu72 generated using TNT kit (Promega) presence YPD medium 16&#194;&#176;C; Ino&#226;&#171;&#186;, impaired growth &#226;&#171;&#186;Ino medium 30&#194;&#176;C                 [35S]methionine according vendor specifications. Template DNAs puri- <br /> relative growth &#226;&#171;&#185;Ino control medium. Plasmids carrying URA3 marker         fied using Mini-Prep kit (Qiagen). Plasmids pT7-IID (9) pET-SUA7 were counterselected synthetic medium containing 5-fluoro-<span id='am-1' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span>orotic acid</span> (8).        used templates synthesis TBP TFIIB, respectively. SSU72 ssu72 <br />    Construction ssu72-2 mutant. ssu72-2 allele constructed         allelic DNAs amplified PCR using oligonucleotides 5&#226;&#172;&#732;-GGAATTCCA <br /> site-directed mutagenesis described previously (48) using synthetic oligonu-      TATGCCTAGTCATCGC 5&#226;&#172;&#732;-CGCGGATCCTTAGTGATGGTGATGGT <br /> cleotide oZS-200 (5&#226;&#172;&#732;-GAAGATTTGATGAATGCAGGTGGGAAATTAAAC-                              GATGTTTGTAATATGAAGGAGCG. amplifies SSU72 ORF 3&#226;&#172;&#732;). underlined GC bases GCA triplet encode alanine <span id='am-17' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>         sequences coding hexahistidine tag C terminus flanking NdeI position 129. EcoRI-KpnI DNA fragment encompassing ssu72-2 open              BamHI sites (underlined). PCR products digested ligated reading frame (ORF) cloned corresponding sites plasmid pRS314        NdeI BamHI sites pCITE-4a(&#226;&#171;&#185;) (Novagen), generating ssu72 tem- <br /> (45), creating ssu72-2 TRP1 CEN plasmid pM721. phenotype ssu72-2          plates pM1540 (SSU72), pM1541 (ssu72-1), pM1542 (ssu72-2), pM1543 initially determined plasmid shuffle using host strain YZS89A.                (ssu72-4). <br />    Allele replacement normal SSU72 chromosomal locus                 RNAP II immunoprecipitation. Immunoprecipitation assays performed <br /> integration-excision (40). EcoRI-KpnI fragment encompassing ssu72-2          described previously (34), minor modifications. Highly purified yeast <br /> cloned corresponding sites plasmid pRS306 (45), yielding ssu72-      RNAP II (425 ng) vitro-transcribed translated input proteins (10 &#226;?&#174;l) <br /> 2-integrating plasmid pM728. pM728 linearized unique ClaI site            incubated 250 &#226;?&#174;l P100 buffer (50 mm Tris-acetate [pH 7.9], 10% <br /> located SSU72 coding region introduced strain LRB535. Ura&#226;&#171;&#185;         glycerol, 100 mM potassium acetate, 14 mM magnesium acetate, 4 mM dithio- <br /> transformants isolated subsequently counterselected 5-fluoro-orotic      threitol, 1 mM EDTA, 50 &#226;?&#174;g bovine serum albumin ml, protease <br /> acid medium. Strains retaining ssu72-2 allele identified based       inhibitors [0.5 &#226;?&#174;g pepstatin ml, 0.5 &#226;?&#174;g leupeptin ml, 2 &#226;?&#174;g Ts&#226;&#171;&#186; phenotype associated allele   confirmed           chymostatin ml, 2.5 &#226;?&#174;g antipain ml, 1 mM phenylmethylsulfonyl <br /> ability Ts&#226;&#171;&#186; phenotype complemented plasmid-borne SSU72.              fluoride, 2 mM benzamidine]) 4&#194;&#176;C 18 h. Pansorbin (Calbiochem) Southern blot analysis confirmed integration excision SSU72 locus         harvested centrifugation resuspended blocking buffer (50 mM Tris- <br /> (D. L. Pappas, Jr., unpublished results).                                            acetate [pH 7.9], 100 mM potassium acetate, 25 mg PVP-40 [Sigma] ml <br />    Isolation sequence analysis rpb2-100 allele RPB2. rpb2-100      30 mg bovine serum albumin ml [pH 7.9]). Following incubation allele RPB2 recovered gap repair (40). Plasmid RY2119 (RPB2 URA3           20 min room temperature, Pansorbin centrifuged resuspended CEN) digested completion SnaBI, removes RPB2           original volume P100 buffer. microliters preblocked Pansorbin <br /> ORF. Vector DNA flanked RPB2 sequences isolated agarose gel                added reaction mixture. Samples incubated 4&#194;&#176;C 30 min <br /> electrophoresis, purified, introduced strain YDP135 (rpb2-100). Ura&#226;&#171;&#185;        centrifuged 15,000 rpm 2 min. Supernatants transferred fresh <br /> colonies selected subsequently screened retention Cs&#226;&#171;&#186;        tubes containing 5 &#226;?&#174;g anti-RNAP II monoclonal antibody (8WG16; Covance, <br /> Ino&#226;&#171;&#186; phenotypes conferred rpb2-100 allele RPB2. Plasmid DNA            ) incubated 90 min 4&#194;&#176;C. Preblocked Pansorbin (25 &#226;?&#174;l) added, <br /> recovered, amplified Escherichia coli, analyzed restriction digestion   reaction mixtures incubated 60 min 4&#194;&#176;C. samples confirm gap repair rpb2-100 locus. resulting plasmid, pDP73, failed      centrifuged 16,000 &#226;&#171;&#187; g 2 min. pellets washed times 250 rescue Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes introduced strains YDP19         &#226;?&#174;l P100 buffer containing 0.1% NP-40 250 &#226;?&#174;l P100 buffer. <br /> YDP135, confirming recovery rpb2-100. facilitate sequence analysis,        Pellets resuspended 40 &#226;?&#174;l sample buffer, 20 &#226;?&#174;l 1.88-kb KpnI-EcoRI, 1.05-kb EcoRI-BglII, 2.5-kb PstI-SalI rpb2-100 frag-         resolved sodium dodecyl sulfate&#226;&#8364;&#8220;12% polyacrylamide gel. Following elec- <br /> ments pDP73 cloned KpnI-EcoRI sites pRS316, EcoRI-             trophoresis, gel treated En3Hance (Dupont NEN), dried, BamHI sites pRS316, PstI-SalI sites pRS424, respectively. Single-          exposed phosphor screen. Products visualized using PhosphorImager <br /> stranded DNA generated using VCS-M13 helper phage presence             ImageQuant software (Molecular Dynamics). <br /> kanamycin described previously (41). DNA sequence entire RPB2 <br /> ORF determined using ABI Prism automated DNA sequencer. DNA <br /> sequence comparisons performed using BLAST algorithm accessed <br />                                                                                                                        RESULTS Saccharomyces Genome Database (http://genome-www.stanford.edu <br /> /Saccharomyces/). <br />                                                                                        Ssu72 protein phylogenetically conserved. Ssu72    RNA analyses. CYC1 transcript levels analyzed S1 nuclease protec-         reported novel protein apparent homologs tion. Strains grown 30&#194;&#176;C synthetic complete medium containing 2%          databases (48). Genome sequencing projects  VOL. 20, 2000                                                                                                     Ssu72-RNAP II INTERACTION                  8345 <br />  <br />  <br />  <br />  <br />   FIG. 1. Sequence similarities eukaryotic Ssu72 proteins. Ssu72 proteins S. cerevisiae (SC), S. pombe (SP), human (HS1 HS2), fruit fly (DM),  thaliana   identified using BLAST algorithm (1) aligned using University Wisconsin Genetics Group programs PILEUP BOXSHADE. Identical similar amino acid residues highlighted black gray, respectively. overbar encompassing residues 122 150 S. cerevisiae <br /> Ssu72 indicates region sequence similarity <span id='am-6' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>-dependent RNA helicases noted previously (48). asterisks indicate residues region phylogenetically invariant helicases. ssu72-2 allele encodes alanine replacement invariant arginine position 129 (R129A). <br />  <br />  <br />  <br />  <br /> identified homologs Ssu72 Schizosaccharomyces pombe,                          Ssu72 (Fig. 1). SSU72 essential cell viability, human, Drosophila melanogaster, Arabidopsis thaliana,                           reasoned replacement conserved amino acid Caenorhabditis elegans cells.  distinct homologs                      region confer growth phenotype. Accordingly, Ssu72 human cells. yeast, human, fly,                        constructed allele, ssu72-2, encodes alanine replace- plant proteins approximately 200 residues length                     ment invariant arginine position 129 (R129A). Using exhibit 46 54% sequence identity (BLAST values                            plasmid-shuffle assay, ssu72-2 mutant &#226;&#172;?e&#226;&#171;&#186;39) S. cerevisiae Ssu72 (Fig. 1). C. elegans protein                   viable failed grow 37&#194;&#176;C. plasmid-borne ssu72-2 larger, comprising 1,357 residues, region                      allele used replace chromosomal copy sequence similarity confined N terminus, showing 43%                      SSU72, yielding strain YZS84. Consistent phenotypes <br /> identity S. cerevisiae Ssu72 (data shown). proteins                       associated plasmid-borne ssu72-2 mutation, YZS84 structural similarity Ssu72 bac-                       viable distinctly Ts&#226;&#171;&#186; (Fig. 2). <br /> terial archaeal protein sequence databases, suggesting                         Genetic suppression ssu72-2. identify factors  <br /> Ssu72 function common eukaryotes conserved                         teract Ssu72, isolated suppressors ssu72-2 Ts&#226;&#171;&#186; eubacteria archaea.                                                    defect. spontaneous Ts&#226;&#171;&#185; revertants strain YZS84    Construction ssu72-2 mutant. original ssu72 allele                    isolated 37&#194;&#176;C. scored pleiotropic phenotypes, (ssu72-1) enhances sua7-1 defect conferring Ts&#226;&#171;&#186;                            revertants failed grow absence inositol grew <br /> growth defect dramatically shifting transcription initia-                    normally synthetic complete medium. phenotype tion downstream normal site ADH1 gene (48).                           denoted Ino&#226;&#171;&#186; associated defects compo- pronounced effects dependent                             nents general transcriptional machinery (18). sua7-1 mutation.  ssu72-1 mutation confers ap-                        Ino&#226;&#171;&#186; revertants cold sensitive (Cs&#226;&#171;&#186;), growing poorly <br /> parent phenotype SUA7 wild-type background.                                    YPD medium 16&#194;&#176;C. strains, YDP19, exhib- <br />    efforts characterize SSU72, sought                       ited distinct Ts&#226;&#171;&#185; Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes (Fig. 2) chosen <br /> ssu72 mutant exhibits phenotypes independent sua7-1.                        characterization. Ssu72 amino acid sequence includes region, encompass-                            Strain YDP19 (Ts&#226;&#171;&#185; Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;) backcrossed strain <br /> ing residues 122 150, highly conserved ATP-                        YDP87 (Ts&#226;&#171;&#186; Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;), resulting diploid strain dependent RNA helicases (Fig. 1). sequences                          phenotypically Ts&#226;&#171;&#186; Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, indicating YDP19 <br /> common class helicases, including DEAD box                           phenotypes result recessive mutation(s). HRIGRXXR motif, Ssu72,                              strain sporulated dissected (11 spore tetrads), invariant residues conserved region present                       Ts&#226;&#171;&#185;:Ts&#226;&#171;&#186; phenotypes segregated 0:4, 1:3, 2:2,  8346       PAPPAS HAMPSEY                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Growth defects associated ssu72, rpb2-100, shs2 mutations. Relevant genotypes indicated wild-type strain LRB535 (SSU72 SHS1 SHS2), <br /> primary mutant YZS84 (ssu72-2 RPB2 SHS2), revertant YDP19 (ssu72-2 rpb2-100 shs2-1), YDP19 derivative carrying plasmid-borne copy RPB2 (ssu72-2 RPB2 <br /> shs2-1), YDP135 segregant (ssu72-2 rpb2-100 SHS2). complete genotypes strains growth media defined Materials Methods. <br />  <br />  <br />  <br /> Cs&#226;&#171;&#185;:Cs&#226;&#171;&#186; Ino&#226;&#171;&#185;:Ino&#226;&#171;&#186; phenotypes segregated 2:2.                           YDP135. resulting transformants phenotypically  Ts&#226;&#171;&#185; segregants Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;. data                          Ts&#226;&#171;&#186;, Cs&#226;&#171;&#185;, Ino&#226;&#171;&#185;, confirming RPB2, dicate suppression ssu72-2 Ts&#226;&#171;&#186; phenotype                        flanking gene, complements suppressor mutation.  <br /> result mutations unlinked genes mutation                     firm suppressor allelic RPB2, URA3 gene genes confers Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes.                         integrated homologous recombination adjacent RPB2 <br /> tentatively designated suppressor genes shs1-1 (rpb2-                    locus strain YDP87. resulting strain (ssu72-2 ura3 <br /> 100  ) shs2-1. shs1-1 allele responsible                  RPB2-URA3) crossed YDP19 (ssu72-2 ura3 shs1-1 Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes, shs2-1 confers pro-                        shs2-1), diploid strain sporulated dissected. nounced phenotype absence shs1-1.                                        total 44 segregants obtained scored. Ura&#226;&#171;&#185; <br />    Identification shs1-1 suppressor. exploited                       progeny phenotypically Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, Ura&#226;&#171;&#186; prog- <br /> Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes shs1-1 clone wild-type gene                      eny Cs&#226;&#171;&#186; Ino&#226;&#171;&#186;.  suppressor segregates op- YCp50 genomic library (39). approximately                              posite RPB2, confirming shs1-1 allelic 90,000 Ura&#226;&#171;&#185; transformants YDP135 (ssu72-2 shs1-1), 25 <br />                                                                                    RPB2. Accordingly, renamed shs1-1 suppressor <br /> Cs&#226;&#171;&#185; strains isolated, 4 restored Ino&#226;&#171;&#185;. <br />                                                                                    rbp2-100. <br /> Plasmid DNAs isolated transfor- <br />                                                                                       rpb2-100 mutation. rpb2-100 allele cloned mants include overlapping DNA inserts YCp50 vector. plasmid, pDP99, reintro-                          gap repair, entire ORF sequenced using collec- <br /> duced YDP135, scored transformants                        tion primers span RPB2 ORF. single-base-pair <br /> Cs&#226;&#171;&#185; Ino&#226;&#171;&#185;, indicating complementation Cs&#226;&#171;&#186;                       substitution position 1534 (C1534T) identified, encod- <br /> Ino&#226;&#171;&#186; phenotypes consequence plasmid DNA                            ing replacement arginine-512 cysteine (R512C). strain reversion.                                                             Rpb2 subunit RNAP II structurally conserved    Restriction analysis plasmid pDP99 defined insert                      eukaryotic, archaeal, bacterial RNAPs, including eukary- <br /> approximately 8.0 kb YCp50 vector. Partial DNA se-                          otic RNAPs  II, III. Sequence comparisons identified <br /> quence analysis identified segment yeast genome                      blocks, designated  highly conserved <br /> chromosome XV encompassing RPB2, gene encoding                             RNAPs (49). R512 amino acid second-largest subunit RNAP II. define                        homology block D phylogenetically invariant (Fig. 3). <br /> lationship shs1-1 RPB2, plasmid RY2119,                          results presented indicate R512, despite phy- <br /> includes entire RPB2 coding region                          logenetic invariance, essential cell viability. flanking ORFs original clone, introduced                           rpb2-100 allele normal (SSU72&#226;&#171;&#185;) genetic  <br />  VOL. 20, 2000                                                                                                        Ssu72-RNAP II INTERACTION                    8347 <br />  <br />  <br />                                                                                       wild-type strain (compare lanes 1 2). Induced levels <br />                                                                                       diminished approximately 70% normal                                                                                       ssu72-2 mutant (lane 4) reduced 43%                                                                                       normal ssu72-2 rpb2-100 shs2-1 suppressor strain (lane <br />                                                                                       6). Interestingly, rpb2-100 dramatic effect                                                                                       absence shs2-1, nearly eliminating INO1 expression (lane 8). <br />                                                                                       results consistent effects ssu72-2                                                                                       rpb2-100 INO1-lacZ reporter gene expression  <br />                                                                                       pressing conditions (Fig. 5B). Northern analysis <br />                                                                                       does distinguish effects noninduced versus <br />  <br />  <br />  <br />  <br />    FIG. 3. rpb2-100 mutation. rpb2-100 allele RPB2 encodes R512C replacement phylogenetically invariant position homology <br /> block D Rpb2. alignment homology block D (30 residues) shown S. cerevisiae (Sc), S. pombe (Sp),  thaliana  , Plasmodium falciparum (Pf), C. <br /> elegans (Ce), Homo sapiens (Hs), Drosophila melanogaster (Dm), Pyrococcus <br /> horikoshii (Ph), second-largest subunits (Rpa135 Rpc128) RNAP RNAP III S. cerevisiae [Sc   Sc (III), respectively], &#226;?&#164; <br /> subunits E. coli (Ec) T. aquaticus (Ta) RNAPs. Amino acids indicated single-residue code; numbers right indicate position C-terminal residue block D. <br />  <br />  <br />  <br />  <br /> ground displays weak growth impairment 30&#194;&#176;C severely Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; (data shown). <br />    ssu72-2 rpb2-100 alleles affect noninduced tran- <br /> scription vivo. effects ssu72-2 rpb2-100 mutations transcription specific promoters determined vivo using wild-type strain (LRB535), ssu72-2 primary <br /> mutant (YZS84), ssu72-2 rpb2-100 shs2-1 suppressor strain <br /> (YDP19), ssu72-2 rpb2-100 segregant (YDP135). Ef- <br /> fects CYC1 expression presented Fig. 4A. repressing conditions (2% glucose), CYC1 transcript level reduced ssu72-2 mutant 42% normal (compare <br /> lanes 1 3). Expression diminished ssu72-2 rpb2-100 shs2-1 revertant 17% normal (lane 5), effect attributed rpb2-100 shs2-1, level expression observed ssu72-2 <br /> rpb2-100 mutant (lane 7). effects occur level noninduced transcription, strains retained ability respond derepressing conditions. <br />    effects CYC1 transcription confirmed using CYC1-lacZ reporter. wild-type strain 1,300 U activity measured repressing conditions, activity enhanced 3.5-fold derepressing conditions (Fig. 4B). <br /> Noninduced expression diminished 9-fold ssu72-2 <br />                                                                                          FIG. 4. Effects ssu72 shs mutations CYC1 expression.   primary mutant (140 U) reduced additional 30-fold                          panel indicates CYC1 transcript levels, determined S1 nuclease protection, ssu72-2 rpb2-100 shs2-1 suppressor strain (4.8 U).                         strains grown repressing (2% glucose) (lanes R) derepressing (3% <br /> shs2-1 allele did contribute effect, activity                   glycerol 2% ethanol) (lanes D) conditions. Strains LRB535 (wild type <br /> equally low ssu72-2 rpb2-100 strain (3.3 U). mutant                        [WT]), YZS84 (ssu72-2), YPD19 (ssu72-2 rpb2-100 shs2-1), YDP135 <br />                                                                                       (ssu72-2 rpb2-100). Transcript levels repressing conditions presented <br /> responded derepressing conditions, absolute                           percentages, normalized level wild-type strain. panel <br /> activities diminished parallel effects basal ac-                      loading control indicates tRNAW levels. (B) &#226;?&#164;-Galactosidase activities <br /> tivity. results consistent RNA analyses (Fig.                      expressed CYC1-lacZ plasmid pLG265-UP1. Activity units  <br /> 4A), confirming ssu72-2 rpb2-100 diminish nonin-                             pressing derepressing conditions, respectively, 1,300 &#226;&#171;&#190; 66 4,900 &#226;&#171;&#190; <br />                                                                                       1030 (WT), 140 &#226;&#171;&#190; 13 310 &#226;&#171;&#190; 40 (YZS84), 4.8 &#226;&#171;&#190; 1.2 19 &#226;&#171;&#190; 3.5 (YDP19), <br /> duced transcription CYC1 promoter.                                           3.3 &#226;&#171;&#190; 0.54 97 &#226;&#171;&#190; 14 (YDP135). indicated units &#226;?&#164;-galactosidase <br />    INO1 expression determined Northern blot analysis                           activity represent means independent transformants assayed (Fig. 5A). Strong induction INO1 mRNA observed                              duplicate; standard deviations indicated error bars. <br />  8348       PAPPAS HAMPSEY                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     activated expression, measurable levels &#226;?&#164;-galactosidase <br />                                                                                     activity INO1-lacZ reporter repressing condi- <br />                                                                                     tions demonstrate effects ssu72-2 rpb2-100                                                                                     INO1 expression accounted effects nonin- <br />                                                                                     duced transcription. <br />                                                                                        INO1-lacZ reporter assays repeated using strain <br />                                                                                     YDP135 complemented plasmid-borne <br />                                                                                     copies SSU72 RPB2, allowing effects ssu72-2 <br />                                                                                     rpb2-100 assessed isogenic background inde- <br />                                                                                     pendent shs2-1. Consistent previous results, ssu72-2 <br />                                                                                     minor effect INO1-lacZ expression, <br />                                                                                     effects rpb2-100 single ssu72-2 rpb2-100 <br />                                                                                     double mutations dramatic, yielding 10% <br />                                                                                     normal activity (Fig. 5C).  ssu72-2 exerts adverse ef- <br />                                                                                     fects gene expression markedly enhanced                                                                                     rpb2-100 allele, effects manifest level                                                                                     noninduced transcription. <br />                                                                                        effects ssu72-2 rpb2-100 assessed using <br />                                                                                     PGK1 promoter, expressed high constitutive <br />                                                                                     level. PGK1-lacZ reporter plasmid introduced                                                                                     strain, &#226;?&#164;-galactosidase activities measured (Fig. 6). <br />                                                                                     Expression PGK1 promoter diminished 60% <br />                                                                                     normal ssu72-2 mutant 40% normal                                                                                     ssu72-2 rpb2-100 shs2-1 triple mutant. Expression mark- <br />                                                                                     edly diminished ssu72-2 rpb2-100 double mutant, 3% <br />                                                                                     normal. experiments distinguish non- <br />                                                                                     induced activated expression, dramatic effect                                                                                     double mutant indicates ssu72-2 rpb2-100 exert <br />                                                                                     synergistic effects gene expression. adverse effect <br />                                                                                     rpb2-100 SHS2&#226;&#171;&#185; background consistent                                                                                     effect INO1 expression (Fig. 5). results demonstrate <br />                                                                                     shs2-1, required addition rpb2-100 sup- <br />                                                                                     pression ssu72-2 Ts&#226;&#171;&#186; phenotype (Fig. 2), suppresses <br />                                                                                     adverse effects rpb2-100 noninduced transcription. <br />                                                                                        Physical interaction Ssu72 RNAP II. ge- <br />                                                                                     netic interactions ssu72, sua7, rpb2 suggest                                                                                     Ssu72 interacts directly core transcriptional machin- <br />                                                                                     ery.  Ssu72 directly binds TFIIB, consistent allele- <br />                                                                                     specific interactions ssu72 sua7 (54). Physical <br />                                                                                     interaction Ssu72 RNAP II investigated                                                                                     coimmunoprecipitation assay using purified yeast RNAP II <br />                                                                                     monoclonal antibody directed Rpb1 subunit <br />                                                                                     RNAP II. Input proteins 35S-labeled Ssu72, la- <br />                                                                                     beled TBP TFIIB controls. RNAP II known bind <br />                                                                                     TFIIB, TBP, interaction RNAP II requires <br />                                                                                     TFIIB bridging factor. Ssu72 input included                                                                                     forms protein, corresponding length Ssu72                                                                                     N-terminally truncated Ssu72 resulting initiation AUG <br />                                                                                     codon 23. results demonstrate RNAP II binds stably <br />                                                                                     TFIIB Ssu72 TBP control (Fig. 7A).  <br />                                                                                     genetic interaction ssu72-2 rpb2-100 likely <br />                                                                                     reflect physical interaction Ssu72 RNAP II. <br />                                                                                        N terminus Ssu72 essential Ssu72 function                                                                                     cell viability, RNAP II binds smaller form Ssu72, <br />                                                                                     lacking N-terminal 22 residues. investi- <br />  <br />  <br />  <br />                                                                                     (&#226;&#171;&#186;Ino medium) (lanes D) conditions. Strains identical Fig. 4.                                                                                     panel loading control showing U6 RNA levels. (B) &#226;?&#164;-Galactosidase <br />                                                                                     activities expressed INO1-lacZ plasmid pJS325 repressing  <br />                                                                                     ditions. Activity units 310 &#226;&#171;&#190; 39 (wild type [WT]), 210 &#226;&#171;&#190; 36 (YZS84), 86 &#226;&#171;&#190; <br />                                                                                     12 (YDP19), 17 &#226;&#171;&#190; 4.1 (YDP135). Error bars indicate standard deviations. <br />                                                                                     (C) panel B, host strain case YDP135 (ssu72-2 <br />                                                                                     rpb2-100) carrying INO1-lacZ reporter plasmid-borne copies wild-type <br />    FIG. 5. Effects ssu72 shs mutations INO1 expression.         SSU72, RPB2, vectors  indicated genotypes. Activity units panel indicates INO1 transcript levels, determined Northern blot analysis,   56 &#226;&#171;&#190; 2.8 (WT), 47 &#226;&#171;&#190; 4.7 (ssu72-2), 1.9 &#226;&#171;&#190; 0.07 (rpb2-100), 5.3 &#226;&#171;&#190; 0.63 (ssu72-2 <br /> strains grown repressing (&#226;&#171;&#185;Ino medium) (lanes R) derepressing             rpb2-100). <br />  VOL. 20, 2000                                                                                                         Ssu72-RNAP II INTERACTION                      8349 <br />  <br />  <br />  <br />  <br />    FIG. 6. &#226;?&#164;-Galactosidase activities expressed PGK1-lacZ plasmid. <br /> Strains identical Fig. 4 Fig. 5A B, transformed PGK1-lacZ reporter plasmid pN1086. Activity units 2,500 &#226;&#171;&#190; 88 (wild type [WT]), 1,500 &#226;&#171;&#190; 160 (YZS84), 1,000 &#226;&#171;&#190; 52 (YDP19), 86 &#226;&#171;&#190; 3.6 (YDP135). <br />  <br />  <br />  <br /> gated asking specific mutations Ssu72 N <br /> terminus affect RNAP II binding. ssu72-1 allele,                               FIG. 7. Physical interaction Ssu72 RNAP II.   Equal <br /> enhances sua7-1 mutation, encodes 10-amino-acid dupli-                          amounts highly purified yeast RNAP II incubated 35S-labeled mock, <br /> cation N terminus Ssu72 (residues 9 18),                             TBP, TFIIB, Ssu72 input proteins. Samples immunoprecipitated (IP) <br /> ssu72-4 encodes serine replacement cysteine 15 abol-                        using monoclonal antibody directed C-terminal repeat domain                                                                                       Rpb1. Following centrifugation, samples washed times buffer <br /> ishes cell viability (48). interaction Ssu72                          containing 100 mM potassium acetate plus 0.1% NP-40 buffer <br /> RNAP II appeared unaffected ssu72-2 (R129A)                              containing 100 mM potassium acetate resuspended sample buffer. ssu72-4 (C15S) mutations (Fig. 7B, lanes 5 8).                        Input proteins (5% total; lanes 1, 3, 5, 7, respectively) ingly,  10-amino-acid duplication resulted pref-                       precipitates (50% total; lanes 2, 4, 6, 8, respectively) fractionated <br />                                                                                       sodium dodecyl sulfate-polyacrylamide gel electrophoresis analyzed erential binding RNAP II longer form Ssu72 (Fig.                         autoradiography. WT, wild type. (B) panel  different forms <br /> 7B, lanes 1 4). results confirm Ssu72 binds di-                         Ssu72 used input proteins. ssu72-1 allele encodes duplication rectly RNAP II imply Ssu72 N terminus affects                         amino acid residues 9 18, ssu72-2 encodes R129A replacement, interaction.                                                                     ssu72-4 encodes inviable C15S replacement. upper lower bands                                                                                       Ssu72 input proteins correspond, respectively, length protein                                                                                       truncated protein N terminus corresponds methionine position 23.                                DISCUSSION                                             case Ssu72-1, RNAP II interacts length truncated <br />                                                                                       forms protein. Control reactions lacking RNAP II failed coimmuno- <br />    results paper define genetic physical inter-                       precipitate Ssu72 (data shown). <br /> actions Ssu72 RNAP II support role Ssu72 basal (noninduced) transcription RNAP II. Ssu72 initially identified based genetic interaction                          tags added N terminus appear expose cryptic pro- <br /> TFIIB. ssu72-1 allele confers synthetic Ts&#226;&#171;&#186; growth                           teolytic site just Ssu72 N terminus, removing <br /> fect dependent sua7-1 mutation (48).                             tag, sequences added C terminus render <br /> interaction ssu72-1 sua7 specific, resulting syn-                       Ssu72 nonfunctional vivo, conserved, <br /> thetic phenotypes combination sua7 alleles                          tandem aromatic residues C terminus (Fig. 1).  <br /> affect transcription start site selection (54). Ssu72 ge-                     ingly, apparent Ssu72 homologs bacterial netically linked Sub1 (Tsp1), homolog human gen-                          archaeal genomes, implying Ssu72 function specific eral transcriptional cofactor PC4, sub1 deletion                       eukaryotes. yeast, depletion Ssu72 causes growth arrest <br /> enhances sua7 mutations does allele-specific man-                        diminishes total poly  RNA levels 20 40% <br /> ner identical specificity ssu72-1 sua7 inter-                   effect RNAP RNAP III transcription <br /> actions (54). Consistent genetic data, TFIIB directly                        (Pappas, unpublished results). results suggest Ssu72 <br /> interacts Ssu72 Sub1 proteins (27, 54).                          critical transcription   RNAP II genes <br /> demonstrate Ssu72 binds stably RNAP II                           plays role RNAP RNAP III transcription. <br /> ssu72-2 rpb2-100 alleles affect basal transcription. Taken                           evidence points role Ssu72 core promoter  data demonstrate functional interactions                           function. original ssu72-1 allele, encodes 10-ami- <br /> Ssu72 TFIIB, Sub1, Rpb2 provide strong support                           acid duplication near N terminus, affects transcription Ssu72 RNAP II transcription factor.                                         start site selection (48). effect dramatic, depen- <br />    know specific function Ssu72. efforts                         dent sua7 allele (TFIIB) affects initiation. characterize Ssu72 hampered sequence                             demonstrate ssu72-2 allele adversely affects <br />  8350     PAPPAS HAMPSEY                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> noninduced transcription. rpb2-100 suppressor ssu72-2          R512C replacement context RNAP II diminishes noninduced transcription, effect         dimensional structure.  crystallographic synergistic ssu72-2. magnitude ssu72-2 effect      structure RNAP E. coli determined 12-&#195;&#8230; transcription appears promoter specific:             resolution (11, 12),  recently, RNAP T. <br /> CYC1 transcription diminished nearly 10-fold,            aquaticus determined 3.3-&#195;&#8230; resolution (56). R512 <br /> 2-fold effects observed INO1 PGK1 expression.         highly conserved, corresponding R548 R428 &#226;?&#164; <br /> promoter elements responsible effects unknown           subunits RNAPs E. coli T. aquaticus, respectively readily addressed genome microarray               (Fig. 3). sequence conservation forms RNAP <br /> analysis using wild-type ssu72-2 strains.                         nearly superimposable structures E. coli T. <br />    noteworthy rpb2-100 isolated suppressor     aquaticus RNAPs (32) allow R512C replacement ssu72-2 Ts&#226;&#171;&#186; growth defect, rpb2-100 ssu72-2 exert         interpreted context T. aquaticus crystal structure. <br /> similar effects transcription. anomaly explained        comprehensive description site-specific inter- results genetic analysis YDP19 revertant,          actions catalytically competent RNAP promoter <br /> revealing mutations genes, rpb2-100 shs2-1,       open DNA recently defined protein-DNA photo-cross- <br /> required suppression ssu72-2 Ts&#226;&#171;&#186; phenotype.             linking (31). Accordingly, R428 forms binding site <br /> deed, effect allele absence   double-stranded DNA downstream position &#226;&#171;&#185;1 enhance ssu72-2 phenotype, essentially eliminating growth         positioned make direct contact phosphate 5&#226;&#172;&#732; 37&#194;&#176;C (Fig. 2). result accounts similar       position &#226;&#171;&#185;6 nontemplate strand. position effects ssu72-2 rpb2-100 transcription          residue suggests possible roles R428 stabilizing RNAP- <br /> consistent synergistic effects transcription       DNA interaction, facilitating clamping RNAP duplex PGK1 promoter (Fig. 6). isolated suppres-           DNA, facilitating RNAP translocation, defining <br /> sor, rpb2-100 allele enhancer ssu72-2 absence     downstream boundary transcription bubble.  shs2-1.                                                             catabolite gene activator protein effect    rpb2-100 allele encodes single amino acid replace-           cross-linking RNAP core promoter region <br /> ment, R512C, located position homology            (31). results readily account effects rpb2-100 <br /> block D Rpb2 (Fig. 3). viability R512C mutant           noninduced transcription underscore premise indicates R512, despite phylogenetic invariance,      Ssu72 core RNAP II transcription factor. <br /> functionally invariant. does rpb2-100 allele  combination shs2-1) suppress ssu72-2 Ts&#226;&#171;&#186; phenotype,                            ssu72-2 rpb2-100 shs2-1 strain (YDP19) exhibits                                                                  <br /> marginally impaired growth rich (YPD) medium 30&#194;&#176;C.            viability rpb2-100 mutants attributed          <br /> suppressive effect ssu72-2 rpb2-100, rpb2-100         <br /> mutant viable SSU72&#226;&#171;&#185; background, growth          <br /> rate comparable YDP19 (Pappas, unpublished              <br /> sults). Furthermore, rpb2-100 SSU72&#226;&#171;&#185; strain exhibits       severe Cs&#226;&#171;&#186; Ino&#226;&#171;&#186; phenotypes YDP19,                  <br /> confirming phenotypes R512C  <br />                                                       <br /> placement. <br />                                                        <br />    rpb2 alleles isolated genetic selections            . mutations affect core promoter function (3, 4, 14, 19).       spt alleles RPB2 identified suppressors &#226;?&#166;-el-           <br /> ement insertions HIS4 LYS2 promoters (19).          <br /> alleles encode amino acid replacements region              <br /> homology blocks B C Rpb2 underscore role            Rpb2 subunit core promoter recognition. sit1          <br /> sit2 alleles RPB1 RPB2, respectively, isolated            <br /> enhancers HIS4 transcription absence trans               <br /> activators Gcn4, Bas1, Bas2 (4). mutations com-          <br /> parable rpb2-100 affect noninduced tran-             <br /> scription, albeit opposite effects. Furthermore, sit1         sit2 mutants, like rpb2-100 mutant, inositol auxo-      <br /> trophs. sit1 alleles encode Rpb1 amino acid replacements               region D region F, near        active center RNAP (3). Thermus aquaticus               <br /> RNAP structure, sit1 region F replacements adjacent           <br /> R428, counterpart R512 Rpb2. opposite effects        sit1 rpb2-100 mutations transcription ac-           <br /> counted proximity secondary channel,          distinction stimulatory inhibitory effects          accounted structural changes affect sub-           <br /> strate access active site  consequently, efficiency         promoter clearance.                                         <br />    crystal structure yeast RNAP II recently solved           resolution 3 &#195;&#8230; (10). Unfortunately, R512 located           approximately 13-residue gap precludes interpretation          <br /> </body></html>