<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Apr. 1998, p. 2079&#226;&#8364;&#8220;2086                                                                                           Vol. 180,  8 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology <br />  <br />  <br />  <br />         Isolation Characterization EPD1, Essential Gene         Pseudohyphal Growth Dimorphic Yeast, Candida maltosa <br />           TAKANOBU NAKAZAWA, HIROYUKI HORIUCHI, AKINORI OHTA,                                              MASAMICHI TAKAGI* <br />                      Department Biotechnology, University Tokyo, Yayoi, Bunkyo-ku, Tokyo 113, Japan <br />                                             Received 21 August 1997/Accepted 17 February 1998 <br />  <br />             Additional copies centromeric DNA (CEN) region induce pseudohyphal growth dimorphic yeast, <br />           Candida maltosa (T. Nakazawa, T. Motoyama, H. Horiuchi,  Ohta, M. Takagi, J. Bacteriol. 179:5030&#226;&#8364;&#8220; <br />           5036, 1997). understand mechanism transition, screened gene library C. maltosa           sequences suppress morphological change. result, isolated 5* end new gene, <br />           EPD1  essential pseudohyphal development), cloned entire gene. predicted <span id='am-40' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-41' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-43' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> <br />           sequence Epd1p highly homologous Ggp1/Gas1/Cwh52p, glycosylphosphatidylinositol- <br />           anchored protein Saccharomyces cerevisiae, Phr1p Phr2p Candida albicans. expression           EPD1 moderately regulated environmental pH. homozygous EPD1 null mutant showed mor- <br />           phological defects reduction growth rate reduced levels alkali-soluble alkali-insoluble <br />           b-glucans.  mutant undergo transition yeast form pseudohyphal form <br />           induced additional copies CEN sequence pH 4 n-hexadecane pH 4 pH 7, suggesting           EPD1 essential yeast form growth essential transition pseudohyphal form.  <br />           expression amino-terminal Epd1p control GAL promoter suppressed           pseudohyphal development induced additional copies CEN sequence, overexpression           length EPD1 did  result initial isolation 5* end EPD1 suppressor           pseudohyphal growth induced CEN sequence suggest amino-terminal Epd1p           dominant-negative effect functions Epd1p pseudohyphal growth induced CEN <br />           sequence. <br />  <br />  <br />    Dimorphic fungi exhibit yeast-like form                      analysis closely related C. albicans (34). recombi- <br /> (pseudo)hyphal form mainly response environmental                       nant DNA technology, host-vector systems  <br /> conditions, switch yeast-like form filamen-               structed laboratory (13, 16, 35&#226;&#8364;&#8220;37, 51). Recently, tous form correlates pathogenicities. Candida                centromeric DNA (CEN) region, albicans typical dimorphic fungus cause life-                  present plasmid, induces pseudohyphal growth threatening infections immunocompromised patients.                 yeast. suggested trans-acting factors  <br /> organism, dimorphic transition thought critical              teract sequence essential centromeric activity, GGT <br /> pathogenesis, elongated hyphal form facilitates tissue                  AGCG, involved regulation transition penetration (47&#226;&#8364;&#8220;49). Environmental conditions dimor-                    yeast form pseudohyphal form (31). help  <br /> phic transition studied related species                 stand mechanism transition, screened gene <br /> Candida tropicalis (52). Saccharomyces cerevisiae shows                library C. maltosa DNA sequences suppress <br /> dimorphic transition (12, 20, 23, 43, 57), induced                   pseudohyphal growth induced additional copies <br /> mainly nitrogen starvation (12). number genes                    CEN sequence. shown participate dimorphism C. albicans S. cerevisiae. S. cerevisiae, reported mito-                                   MATERIALS METHODS <br /> gen-activated protein kinase (MAPK) involved pseudohy-                     Strains media. C. maltosa IAM12247  wild-type strain) deriv- <br /> phal development nitrogen starvation activated                   atives, CHA1 (his5 ade1) (16) CHAU1 (his5 ade1 ura3) (37), used. Ras proteins stimulate pseudohyphal growth                       media C. maltosa YPD (1% yeast extract, 2% Bacto Peptone, 2% <br />                                                                               glucose), SD (pH 4) [0.17% yeast nitogen base amino acid ammonium <br /> MAPK-dependent MAPK-independent pathways (7, 23,                          sulfate (Difco), 0.5% (NH4)2SO4, 2% glucose, appropriate nutrients], SG (pH 4) <br /> 25, 29). addition, genes reported                  (SD glucose replaced 2% galactose), hexadecane <br /> regulation influences pseudohyphal growth (2, 3, 9, 11, 56).                  medium (pH 4)  SD SG, n-hexadecane, Furthermore, variety genes, PHR1, HYR1, CPH1                     supplied vapor, sole carbon source). SD (pH 7), SG (pH 7),                                                                               hexadecane (pH 7) media buffered 150 mM HEPES adjusted (ACPR), CST20, HST7, EFG1, RBF1, TUP1,                          pH 7.0. necessary, agar added concentration 2%. <br /> shown affect dimorphism C. albicans (1, 4, 15, 18, 21, 22,                 Escherichia coli MV1190 [D(srl-recA)306::Tn10(Tetr) D(lac-pro) thi supE (F9 <br /> 24, 26, 30, 45, 50). spite investigations, mecha-             proAB lacIq lacZDM15 traD36)], HB101 [hsdS20(r2 m2) recA13 ara-14 proA2 <br /> nisms dimorphism clear.                                      lacY1 galK2 rpsL20(Smr) xyl-5 mtl-1 l2 mcrA1 mcrB], DH5 (supE44 hsdR1 <br />                                                                               recA1 endA1 gyrA96 thi-1 relA1) grown Luria-Bertani broth used    Candida maltosa dimorphic fungus diploid ge-                   hosts propagation plasmids construction genomic sub- <br /> nome (14, 19, 31), shown phylogenetic                      genomic libraries C. maltosa. <br />                                                                                  Plasmid construction transformation. construction pUAH2A                                                                               pUAHHH1 described previously (31). Ligation DNA ends                                                                               cohesive blunt ending T4 DNA <br />   * Corresponding author. Mailing address: Department Biotech-             polymerase. DraI-EcoT22I fragment URA3 C. maltosa, AatII-ClaI <br /> nology, University Tokyo, Yayoi, Bunkyo-ku, Tokyo 113, Japan.          fragment containing ARS (13, 51), 208-bp HincII-HindIII fragment Phone: 81-3-3812-2111, ext. 5169. Fax: 81-3-3812-9246. E-mail:                CEN sequence ligated NaeI site, NspI site, EcoRI <br /> amtakag@hongo.ecc.u-tokyo.ac.jp.                                              site, respectively, pBluescript II KS(1) yield plasmid pBLU1.  <br />                                                                        2079 <br />  2080       NAKAZAWA ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />                                                                                         Construction genomic subgenomic libraries C. maltosa. Total DNA <br />                                                                                      prepared C. maltosa IAM12247 described previously (36).                                                                                      partial digestion Sau3AI, total DNA size fractionated sucrose <br />                                                                                      density gradient centrifugation. DNA fragments 6 8 kb ligated                                                                                      BamHI-digested phosphatase-treated pUAA10 transformed E. coli <br />                                                                                      DH5. subgenomic library pUC119 constructed follows. Total DNA <br />                                                                                      C. maltosa IAM12247 digested EcoRI electrophoresed                                                                                      0.6% agarose gel. piece gel containing EcoRI fragments 4.5 <br />                                                                                      kb cut  DNA contained eluted, purified ligated                                                                                      EcoRI-digested phosphatase-treated pUC119. E. coli MV1190 trans- <br />                                                                                      formed ligated DNA, approximately 5,000 ampicillin-resistant col- <br />                                                                                      onies selected used screening colony hybridization. <br />                                                                                         Deletion EPD1. pEPD1::HIS5 digested ApaLI used trans- <br />                                                                                      form C. maltosa CHAU1. Stable His1 transformants selected obtain                                                                                      strain UEP11. pEPD1::ADE1 digested ApaLI used transform <br />                                                                                      strain UEP11. Stable His1 Ade1 transformants selected obtain                                                                                      strain UEP21 (epd1/epd1). Disruption EPD1 gene strains                                                                                      confirmed Southern hybridization PCR. <br />                                                                                         Glucan analysis. alkali-insoluble -soluble 1,3- 1,6-b-D-glucans <br />                                                                                      isolated quantified described Popolo et al. (40). <br />                                                                                         Southern blot analysis. Total DNA C. maltosa isolated 10-ml <br />                                                                                      culture grown 15 h selective medium described previously (36). South- <br />                                                                                      ern blot analysis performed using Amersham ECL direct nucleic acid <br />                                                                                      labelling detection according instructions supplier.                                                                                      548-bp EcoRV-PvuII fragment EPD1 used probe. Colony hybridiza- <br />                                                                                      tion carried Hybond-N1 membrane (Amersham) follows.                                                                                      EcoRI-XbaI fragment insert pPHS1SX labelled  32P]<span id='am-47' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="dCTP" datatype="xsd:string"></span>dCTP</span> <br />                                                                                      (Amersham) use random primer labelling kit (Takara) used                                                                                      probe. Hybridization membrane washing carried method <br />                                                                                      suggested membrane supplier. <br />                                                                                         Northern blot analysis. Total yeast RNA various culture conditions                                                                                      extracted method Schmitt et al. (46), separated agarose gel electro- <br />                                                                                      phoresis, blotted Hybond-N membrane (Amersham), analyzed. North- <br />                                                                                      ern hybridization carried described previously (38). <br />                                                                                         DNA sequence analysis EPD1. Appropriate restriction fragments    FIG. 1. Colony morphology C. maltosa CHA1 containing plasmids                  EPD1 gene subcloned pUC series plasmids, DNA sequencing <br /> pUAHHH1 pUAA10 (vector   , pUAHHH1 pPHS1 (suppres-                   carried automated DNA sequencer (LI-COR model 4000L). <br /> sor) (B), pUAHHH1 pPHS1SX (subclone insert pPHS1) (C).                <span id='am-32' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. nucleotide sequence data  cells grown SG agar plates (pH 4) 3 days.                            ported paper submitted DDBJ, EMBL, GenBank <br />                                                                                      nucleotide sequence databases accession  AB005130. <br />  <br />  <br /> AatII-ClaI fragment ARS DraIII-SspI fragment ADE1 (16) C. <br /> maltosa ligated EcoO109I-SspI site SalI site, respectively,                                     RESULTS pUC119 yield plasmid pUAA10. XbaI-XbaI fragment containing <br /> EPD1  Fig. 4) ligated XbaI site pUC119, DraIII-SspI fragment ADE1 (16) inserted PvuII site EPD1           Isolation multicopy suppressor pseudohyphal growth <br /> yield plasmid pEPD1::ADE1. XbaI-XbaI fragment containing                induced additional copies CEN sequence. Previously, <br /> EPD1 ligated XbaI site pUC119, SalI fragment            CEN region plasmid induce <br /> containing HIS5 (13) inserted PvuII site EPD1 yield plasmid pEPD1::HIS5. EcoRI-BstPI fragment containing length <br />                                                                                      pseudohyphal growth introduced C. maltosa <br /> EPD1  Fig. 4) XbaI-XbaI fragment GAL promoter (39)           (Fig. 1A) (31). analyze mechanism induction, ligated SmaI site XbaI site pUAA10, respectively, yield   carried screening genes suppress pseudohyphal <br /> plasmid pUAAEB5. EcoRI-BglII fragment amino-terminal              growth conditions.  confirmed Epd1p  Fig. 4) XbaI-XbaI fragment GAL promoter (39) ligated SmaI site pUAA10 yield plasmid pUAAEBG3.                                                                                      host strain does pseudohyphal growth SG agar <br /> EcoRI-SalI, XbaI-EcoRV, SalI-XbaI fragments pPHS1  Fig. 2)           plates pH 4 pH 7 host strain containing <br /> ligated SmaI site pUAA10 construct plasmids pPHS1ES,                  pUAHHH1 carrying truncated CEN sequence does pPHS1XE, pPHS1SX, respectively.                                                  SG agar plates pHs, pseudohyphal <br />    Transformation C. maltosa performed electroporation described <br /> previously (27). DNA manipulations E. coli transformation                                                                                      morphology evident pH 4. strain described previously (44). DNA enzymes purchased Takara Shuzo           transformed C. maltosa genomic library constructed (Ohtu, Japan) used according manufacturer&#226;&#8364;&#8482;s instructions.                 high-copy-number vector pUAA10. 3 days,  <br />  <br />  <br />  <br />   FIG. 2. Delimitation inserted C. maltosa DNA pPHS1. Colony morphology observed 3 days SG agar plates (pH 4). 1, suppression pseudohyphal growth induced additional copies CEN sequence; 2, suppression pseudohyphal growth. <br />  VOL. 180, 1998                                    EPD1 ESSENTIAL PSEUDOHYPHAL GROWTH C. MALTOSA                                        2081 <br />  <br />  <br /> 35,000 transformants obtained SG (pH 4) agar plates examined microscopically. colonies showed pseudo- <br /> hyphal growth, showed reduced pseudohyphal growth. <br /> Plasmids isolated colonies reintro- <br /> duced C. maltosa CHA1 containing pUAHHH1, suppression pseudohyphal growth plasmids confirmed. plasmids designated pPHS1 (Fig. <br /> 1B). Restriction enzyme analyses subcloning insert pPHS1 performed shown Fig. 2. plasmid <br /> pPHS1SX, carried smallest fragment, suppressed <br /> pseudohyphal growth level pPHS1 (Fig. 1C). open reading frame (ORF) possible <br /> promoter region inserted DNA pPHS1SX. deduced amino acid sequence ORF homologous N-terminal parts S. cerevisiae protein <br /> Ggp1/Gas1/Cwh52p C. albicans proteins Phr1p Phr2p. isolated plasmids, HIS5 gene cloned. transformants YRp-type plasmids car- <br /> rying ADE1 HIS5 lost pUAHHH1 carrying HIS5 truncated CEN sequence,  did pseudohyphal growth. investigated newly <br /> isolated plasmid (pPHS1) suppress pseudohyphal <br /> transition induced n-hexadecane, strong inducer transition  Fig. 6 7). did observe suppression plasmid conditions <br /> (data shown). <br />    Isolation entire EPD1 gene. clone length <br /> ORF, EcoRI-XbaI fragment insert pPHS1SX used hybridization probe. Southern blot analysis EcoRI digests C. maltosa IAM12247 genomic DNA carried conditions high stringency, pos- <br /> itive band 4.5 kb detected (data shown).  C. maltosa subgenomic DNA library contained DNA <br /> fragment 4.5 kb constructed pUC119 screened colony hybridization probe. positive clone obtained, isolated plasmid des- <br /> ignated pPCO10. Subcloning sequencing inserted <br /> DNA plasmid revealed single, uninterrupted ORF 1,650 nucleotides, encode 549-residue protein. designated gene EPD1. plasmid pPHS1SX  <br /> tained promoter region region encoding amino <br /> acids Met-1 Asn-126 Epd1p. deduced amino acid <br /> sequence entire Epd1p shown Fig. 3. <br />    proteins amino acid sequences similar length Epd1p identified databases       FIG. 3. Comparison predicted amino acid sequences Epd1p, Ggp1/ <br /> FASTA program. products S. cerevisiae GGP1/           Gas1/Cwh52p, Phr1p, Phr2p. Identical similar residues indicated                                                                    asterisks dots, respectively. hydrophobic amino carboxy termini GAS1/CWH52 C. albicans PHR1 PHR2. Epd1                 boxed. proposed GPI attachment sites Epd1p Ggp1/Gas1/Cwh52p <br /> protein 58.4, 56.0, 74.0% identical proteins        indicated bold letters, serine-rich regions underlined. Amino- <br /> Ggp1/Gas1/Cwh52p, Phr1p, Phr2p, respectively. GGP1/            acid N-126, EPD1 gene plasmid pPHS1SX truncated                                                                     Results), boxed inversely. <br /> GAS1/CWH52 codes GPI-anchored plasma membrane <br /> glycoprotein (6, 32, 33, 40&#226;&#8364;&#8220;42, 53&#226;&#8364;&#8220;55), PHR1 encodes putative GPI-anchored cell surface glycoprotein, production differentially regulated response pH     respectively (10, 32). serine-rich region located near growth medium (45). PHR2 encodes functional homolog           carboxy terminus conserved. <br /> PHR1, expression regulated pH, way        Disruption EPD1. epd1 null mutant constructed exactly inverse PHR1 (30). Mutants         analyze role EPD1 morphogenesis. EPD1 alleles <br /> genes showed defects general morphogenesis (30, 40, 41,         C. maltosa CHAU1 inactivated successive gene <br /> 45). homologous proteins showed similarity          replacements insertion alleles epd1::ADE1 entire sequences. regions function-     epd1::HIS5 (Fig. 4A). Single double EPD1 mutants ally significant conserved (Fig. 3).          generated described Materials Methods desig- <br /> Epd1p hydrophobic amino terminus, characteristic          nated UEP11 (EPD1/epd1) UEP21 (epd1/epd1), respec- <br /> secretory signal sequences, hydrophobic carboxy termi-       tively. Replacement EPD1 gene homologous recom- <br /> nus, characteristic GPI-linked proteins (8, 28). Ser-523,   bination confirmed hybridization genomic <br /> Ala-524, Ala-525 residues Epd1p correspond v,        Southern blot (Fig. 4B) PCR (data shown). null <br /> v11, v12 sites consensus GPI attachment site,           mutants viable, indicating EPD1 essential  2082      NAKAZAWA ET AL.                                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Disruption EPD1.   Construction disrupted genes epd1::ADE1 epd1::HIS5. detailed methods disruption described Materials Methods. (B) Southern blot analysis EPD1 mutants. Genomic DNAs indicated strains digested XbaI hybridized labelled EPD1 DNA region indicated panel probe. electrophoretic positions DNA size markers indicated left. <br />  <br />  <br />  <br /> cell viability. duplication times strains CHAU1                       indicate EPD1 involved maintaining normal yeast <br /> UEP21 exponential growth phase SD (pH 4)                           form growth SD pH 4. morphology yeast form 80 120 min, respectively.                                               cells UEP21 (epd1/epd1) different    yeast form cells strains grown SD (pH 4)                      wild-type cells SD pH 7 (data shown). examined microscopically. UEP11 (EPD1/epd1) cells                              examined effect EPD1 disruption somewhat larger wild-type cells slightly                       pseudohyphal growth. wild-type strain CHAU1 (Fig. 6A) <br /> round. early time incubation, UEP21 (epd1/epd1)                        UEP11 (data shown), contained pBLU1 <br /> cells larger wild-type cells round,                   (containing CEN sequence), showed pseudohy- <br /> frequently buds instead  phenotype similar observed ggp1 mutant S. cerevisiae C. albicans phr1/phr1 phr2/phr2 mutants (30, 40, 45). prolonged incubation, morphological characteris- <br /> tics mutants evident (Fig. 5). results <br />  <br />  <br />  <br />  <br />                                                                                     FIG. 6. Effect disruption EPD1 colony morphology pH 4.                                                                                   B) CHAU1 bearing pBLU1 UEP21 containing pBLU1, respectively,   FIG. 5. Effect disruption EPD1 yeast form growth C. maltosa        grown SG agar plates (pH 4) 1 day. (C D) CHAU1 UEP21, <br /> CHA1 (EPD1/EPD1)  , UEP11 (EPD1/epd1) (B), UEP21 (epd1/epd1)               respectively, grown plates containing n-hexadecane sole carbon source <br /> (C). Cells grown SD (pH 4) early stationary phase.             (pH 4) 3 days. <br />  VOL. 180, 1998                                             EPD1 ESSENTIAL PSEUDOHYPHAL GROWTH C. MALTOSA                                            2083 <br />  <br />  <br />                                                                                         TABLE 1. b-Glucan contents epd1D mutant cellsa <br />                                                                                                                                                           Alkali <br />                                                                                                                            Alkali insoluble <br />                                                                                                                                                          soluble <br />                                                                                          Strain                  1,3- 1                                  1,3- 1 <br />                                                                                                                                 1,6-b-D- <br />                                                                                                                 1,6-b-D-                      ZUPb      1,6-b-D- <br />                                                                                                                                  glucan <br />                                                                                                                  glucan                                  glucan <br />  <br />                                                                               CHAU1 (EPD1/EPD1)                    94             72           28         100 <br />                                                                               UEP11 (EPD1/epd1)                    96             75           30         107 <br />                                                                               UEP21 (epd1/epd1)                    61             43           62          87 <br />                                                                                                                                                                   fractions expressed micrograms milligram (dry weight) cells. <br />                                                                                b <br />                                                                                    ZUP, Zymolyase-undigestible pellet. <br />  <br />  <br />  <br />  <br />                                                                               tionary phase SD (pH 4) (Table 1). strain UEP21,                                                                               total alkali-insoluble fraction decreased                                                                               alkali-insoluble 1,6-b-D-glucan 50%                                                                               wild-type strain. <span id='am-25' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>carbohydrate</span> content                                                                               alkali-insoluble material obtained Zymolyase diges- <br />                                                                               tion (Table 1). fraction increased strain UEP21                                                                               comparison wild-type strain. carbohydrate <br />                                                                               contents fractions strain UEP11 similar                                                                               wild-type strain. carbohydrate content alkali- <br />                                                                               soluble fraction strain UEP21 lower                                                                               wild-type strain; different case S. <br />                                                                               cerevisiae ggp1 mutant, 150%                                                                               wild-type strain (40). <br />                                                                                  Transcriptional regulation EPD1. examined                                                                               transcriptional regulation EPD1 Northern blot analysis. <br />                                                                               Total RNA isolated CHA1 containing plasmid <br />    FIG. 7. Effect disruption EPD1 colony morphology pH 7. CHAU1   pUAH2A, does induce pseudohyphal growth, carrying pBLU1   UEP21 bearing pBLU1 (B) grown SG (pH 7) <br /> plates 1 day. CHAU1 (C) UEP21 (D) grown hexadecane            centromere plasmid pUAHHH1, induces pseudohyphal <br /> medium (pH 7) 2 3 days, respectively.                                 growth various growth phases (early log phase mid-log <br />                                                                               phase) SG (pH 4). transcriptional level influ- <br />                                                                               enced growth phase pseudohyphal morphogen- <br />                                                                               esis, transcript detected stationary-phase <br /> phal growth SG agar plates (pH 4) 1 day incubation,               cells (data shown). examined effect pH homozygous mutant, UEP21, containing pBLU1                        transcriptional regulation EPD1. Total RNA iso- <br /> did (Fig. 6B). addition, strain CHAU1 growing n-                    lated early log phase (optical densities 600 nm [OD600], <br /> hexadecane sole carbon source (pH 4) showed pro-                         0.5 2) SD (pH 4) SD (pH 7). transcripts nounced pseudohyphal growth 3 days (Fig. 6C), strain                 detected samples, level EPD1 mRNA UEP21 undergo transition yeast form                   reduced cells grown pH 7 (Fig. 8). result similar pseudohyphal form (Fig. 6D). Strain UEP11 showed                     PHR2 (30).  expression slight transition pseudohyphal form hexadecane                    EPD1 strictly regulated environmental pH <br /> plates (pH 4) (data shown). contrast C. albicans                signal PHR2. <br /> phr2 null mutant, failed grow pH 4 (30), epd1                    amino-terminal Epd1p dominant-negative <br /> null mutant grow SG (pH 4) hexadecane medium                     effect pseudohyphal growth induced CEN se- <br /> (pH 4) SD (pH 4), duplication time                 quence. 59 end EPD1 initially isolated sup- <br /> UEP21 (120 min) 1.5 times wild-type strain (80                pressor pseudohyphal growth induced additional <br /> min).  incapability strain UEP21 demonstrate                   copies CEN sequence. confirm phenotype <br /> pseudohyphal growth attributed growth defect.                truncated EPD1 gene, did following examined morphological characteristics                       experiments. Plasmids pUAAEB5 pUAAEBG3, ex- <br /> strains SG agar plates (pH 7). Strain CHAU1 containing                     press length truncated EPD1, respectively, pBLU1 showed pseudohyphal growth (Fig. 7A),                      control GAL promoter (39), constructed trans- <br /> frequency transition high seen pH                formed wild-type strain CHA1 containing pUAHHH1. <br /> 4. surprise, strain UEP21 containing pBLU1 showed                      3 days incubation SG (pH 4), transformants pseudohyphal growth equal parent strain (Fig.                  examined microscope. CHA1 containing 7B). hand, UEP21 undergo transition                  pUAHHH1 pUAAEB5 showed pseudohyphal growth, yeast form pseudohyphal form hexadecane plates                     CHA1 containing pUAHHH1 pUAAEBG3 <br /> (pH 7) (Fig. 7D), contrast pseudohyphal growth               did (Fig. 9). result initial isolation 59 <br /> parent strain (Fig. 7C).                                                      end EPD1 suggest overexpression ami- <br />    ggp1 mutant S. cerevisiae, content alkali-                terminal region Epd1p multicopy plasmid insoluble 1,6-b-D-glucan shown 50%                 dominant-negative effect pseudohyphal growth induced wild-type strain (40).  examined                    extra CEN sequence. hand, overexpression glucan epd1 mutants entry sta-                 length EPD1 using GAL promoter CHA1 did <br />  2084       NAKAZAWA ET AL.                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                                                    form, comparable pseudohypha- <br />                                                                                    forming wild-type strain hexadecane medium pH 4,                                                                                    contrast failure growth disruptant PHR2,                                                                                    homolog EPD1. incapability strain UEP21                                                                                    pseudohyphal growth,  attributed                                                                                    growth defect. UEP21 cells round larger wild- <br />                                                                                    type cells SD (pH 4), observed ggp1, phr1,                                                                                    phr2 mutants (30, 40, 45), lack cell <br />                                                                                    wall integrity caused defects b-glucan assembly.                                                                                    results suggest function Epd1p identical, <br />                                                                                    quite similar, GGP1/GAS1/CWH52, <br />                                                                                    PHR1, PHR2 products. <br />                                                                                       conclude increased expression EPD1                                                                                    required pseudohyphal growth,  <br />                                                                                    expression length EPD1 did affect transition <br />                                                                                    Northern blot analysis showed ex- <br />    FIG. 8. Northern blot analysis EPD1 transcript. C. maltosa CHA1      pression level EPD1 affected pseudohyphal <br /> incubated SD (pH 4) SD (pH 7) media. RNA isolated, Northern         morphogenesis. EPD1 expression constitutive blot hybridization performed described Materials Methods. Lane       early mid-log phases, contrast expression 1, OD600 0.5 SD (pH 4); lane 2, OD600 0.5 SD (pH 7); lane 3, OD600 2 SD (pH 4); lane 4, OD600 2 SD (pH 7). upper panel displays   HYR1, C. albicans gene encoding GPI-anchored surface <br /> results hybridization EPD1, lower panel displays results    protein activated response hyphal development <br /> ethidium bromide (EtBr) staining rRNA. positions 25S 18S rRNA        (1). EPD1 gene product general function shown right.                                                            transition yeast form pseudohyphal form.                                                                                    examined effect pH EPD1 expression induce pseudohyphal growth SG agar plates (pH 4) <br /> (data shown). <br />  <br />                               DISCUSSION <br />  <br />    Cells yeasts fungi surrounded walls responsible mechanical strength cell (17) essential maintenance cell shape (58). Fungal cell <br /> walls composed primarily classes polysaccha- <br /> rides: mannoproteins, b-glucans, chitins (5). b-glucans <br /> constitute 60% cell wall carbohydrate believed important elements determining cell <br /> morphology S. cerevisiae (58) probably C. albicans C. maltosa. deduced protein product EPD1 isolated C. maltosa present work contains conserved <br /> structural features GPI-anchored proteins, hydrophobic signal sequence N terminus, C-terminal <br /> hydrophobic domain, GPI attachment site (8, 10, 28, 32). structural similarities suggest Epd1p modified addition GPI homolog S. cerevisiae GGP1/GAS1/CWH52 gene product C. <br /> albicans PHR1 PHR2 gene products. primary <br /> function family genes currently unknown (30, 40, <br /> 45), essential assembly b-glucan, lack EPD1 C. maltosa GGP1 S. cerevisiae (40) <br /> reduces level b-glucans (Table 1). <br />    reported C. albicans phr1/phr1 mutants display <br /> defects formation germ tubes apical growth hyphal forms restrictive pH. Prolonged incubation wild-type C. albicans higher pH resulted mixture yeast form pseudohyphal-form cells, incubation phr1/phr1 null mutant did result pseudohyphal-form <br /> cells (45). effect EPD1 disruption similar results <br /> obtained Saporito-Irwin et al. (45), function EPD1 transition yeast form pseudohyphal form similar PHR1 morphological transition C. albicans.  case induction pseudohy- <br /> phae n-hexadecane, disruption EPD1 affected <br />                                                                                      FIG. 9. Effect GAL promoter-induced expression EPD1 colony <br /> pseudohyphal morphology pH 4 7.                           morphology. CHA1 containing pUAHHH1 pUAAEB5 length <br /> growth rate homozygous null mutant, UEP21, lower                        EPD1   pUAHHH1 pUAAEBG3 amino-terminal wild-type strain SD pH 4 yeast                       EPD1 (B) grown SG agar plates (pH 4) 3 days. <br />  VOL. 180, 1998                                                  EPD1 ESSENTIAL PSEUDOHYPHAL GROWTH C. MALTOSA                                               2085 <br />  <br />  expression EPD1 regulated pH, similar                        7. Cook, J. G., L. Bardwell, J. Thorner. 1997. Inhibitory activating strictly PHR2 (30). C. maltosa, pseudohyphal                                functions MAPK Kss1 S. cerevisiae filamentous-growth signalling <br />                                                                                          pathway. Nature 390:85&#226;&#8364;&#8220;88. <br /> growth pronounced pH 7 pH 4.                      8. Ferguson, M.  J. 1988. Cell-surface anchoring proteins glycosylphos- explained lower level expression EPD1 pH 7.                           phatidylinositol structures. Annu. Rev. Biochem. 57:285&#226;&#8364;&#8220;320. results suggest EPD1 works mainly pH 4,                             9. Gavrias, V.,  Andriannopoulos, C. J. Gimeno, W. E. Timberlake. 1996. <br /> partially pH 7. addition, hexadecane plates, UEP11                              Saccharomyces cerevisiae TEC1 required pseudohyphal growth. Mol. <br />                                                                                          Microbiol. 19:1255&#226;&#8364;&#8220;1263. <br /> (EPD1/epd1) showed reduced pseudohyphal growth                                   10. Gerber, L. D., K. Kodukula, S. Udenfriend. 1992. <span id='am-12' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-13' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span>Phosphatidylinositol</span> <br /> UEP21 (epd1/epd1) did pseudohyphal growth                               glycan (PI-G) anchored membrane proteins. J. Biol. Chem. 267:12168&#226;&#8364;&#8220; <br /> pH 7. Pseudohyphal growth hexadecane require                                      12173. <br /> greater amounts Epd1p does SG agar plates.                             11. Gimeno, C. J., G. R. Fink. 1994. Induction pseudohyphal growth                                                                                          overexpression PHD1, Saccharomyces cerevisiae gene related tran- <br />   interesting finding work amino-                          scriptional regulators fungal development. Mol. Cell. Biol. 14:2100&#226;&#8364;&#8220;2112. <br /> terminal Epd1p dominant-negative effect                         12. Gimeno, C. J., P. O. Ljungdahl, C.  Styles, G. R. Fink. 1992. Unipolar <br /> pseudohyphal growth induced CEN sequence.                                  cell divisions yeast S. cerevisiae lead filamentous growth: regulation <br /> inhibit functions Epd1p pseudohyphal growth                                starvation RAS. Cell 68:1077&#226;&#8364;&#8220;1090. <br />                                                                                      13. Hikiji, T., M. Ohkuma, M. Takagi, K. Yano. 1989. improved host- <br /> induced CEN sequence cells. total glucan                            vector Candida maltosa using gene isolated genome content wild-type strain CHA1 contains pUAAEBG3                              complements his5 mutation Saccharomyces cerevisiae. Curr. Genet. <br /> bearing truncated EPD1 downstream GAL promoter                                16:261&#226;&#8364;&#8220;266. identical strain CHA1 containing vector pUAA10                        14. Hofmann, K. H., E. Polnisch. 1990. Characterization mutant                                                                                          yeast Candida maltosa defective catabolite inactivation gluconeogenic <br /> (data shown), expression amino-terminal                           enzymes. Arch. Microbiol. 154:514&#226;&#8364;&#8220;517. <br /> Epd1p cause abnormal architecture, ab-                           15. Ishii, N., M. Yamamoto, F. Yoshihara, M. Arisawa, Y. Aoki. 1997. <br /> normal composition, cell wall.  expression                            Biochemical genetic characterization Rbf1p, putative transcription amino terminus Epd1p promoter                                factor Candida albicans. Microbiology 143:429&#226;&#8364;&#8220;435. <br />                                                                                      16. Kawai, S., T. Hikiji, S. Murao, M. Takagi, K. Yano. 1991. Isolation suppressive effect pseudohyphal growth induced n-                              sequencing gene, C-ADE1, use host-vector hexadecane sole carbon source. differential pheno-                            Candida maltosa genetic markers. Agric. Biol. Chem. 55:59&#226;&#8364;&#8220;65. <br /> types differences cell wall                          17. Klis, F. M. 1994. Cell wall assembly yeast. Yeast 10:851&#226;&#8364;&#8220;869. <br /> architectures pseudohypha-forming cells induced n-                         18. Ko&#194;&#168;hler, R. J., G. R. Fink. 1996. Candida albicans strains heterozygous <br />                                                                                          mutations mitogen-activated protein kinase signaling components hexadecane induced CEN sequence.                                        defects hyphal development. Proc. Natl. Acad. Sci. USA 93:13223&#226;&#8364;&#8220;13228. <br />   examples suggesting dose                           19. Komagata, K., T. Nakase, N. Katsuya. 1964. Assimilation hydrocar- <br /> gene affects filamentous growth yeast. example,                               bons yeasts. II. Determination hydrocarbon-assimilating yeast. J. Gen. <br /> hyphal formation sensitive dose MAPK cascade                            Appl. Microbiol. 4:323&#226;&#8364;&#8220;331. <br />                                                                                      20. Kron, S. J., C.  Styles, G. R. Fink. 1994. Symmetric cell division components C. albicans (18). previous data indicated                              pseudohyphae yeast Saccharomyces cerevisiae. Mol. Biol. Cell 5:1003&#226;&#8364;&#8220; copies YCp-type plasmid induce marked                                 1022. <br /> pseudohyphal growth single copy C. maltosa (31),                         21. Lambrechts, M. G., F. F. Bauer, J. Marmur,  S. Pretorius. 1996. Muc1, present results indicate heterozygous disruptant                          mucin-like protein regulated Mss10, critical pseudohyphal EPD1 shows slight transition yeast form                            differentiation yeast. Proc. Natl. Acad. Sci. USA 93:8419&#226;&#8364;&#8220;8424. <br />                                                                                      22. Leberer, E., D. Harcus,  D. Broadbent, K. L. Clark, D. Dignard, K. <br /> pseudohyphal form induced n-hexadecane. results                                 Ziegelbauer,  Schmidt, N.  R. Gow,  J. P. Brown, D. Y. Thomas. <br /> taken suggest copy numbers cellular                            1996. Signal transduction homologs Ste20p protein kinase components important induction pseudohyphal                               trigger hyphal formation pathogenic fungus Candida albicans. Proc. <br /> growth.                                                                                  Natl. Acad. Sci. USA 93:13217&#226;&#8364;&#8220;13222. <br />                                                                                      23. Liu, H., C.  Styles, G. R. Fink. 1993. Elements yeast pheromone <br />   EPD1 essential growth, essential                          response pathway required filamentous growth diploid. Science 262: <br /> pseudohyphal transition. Defining precise role Epd1p                              1741&#226;&#8364;&#8220;1744. helpful understanding mechanism dimor-                        24. Liu, H., J. Ko&#194;&#168;hler, G. R. Fink. 1994. Suppression hyphal formation phism fungi.                                                                          Candida albicans mutation STE12 homolog. Science 266:1723&#226;&#8364;&#8220;1725. <br />                                                                                      25. Madhani, H. D., G. R. Fink. 1997. Combinational control required                                                                                          specificity yeast MAPK signaling. Science 275:1314&#226;&#8364;&#8220;1317. <br />                          ACKNOWLEDGMENTS                                             26. Malathi, K., K. Ganesan,  Datta. 1994. Identification putative <br />                                                                                          transcription factor Candida albicans complement mating <br />   work performed using facilities Biotechnology                  defect Saccharomyces cerevisiae ste12 mutants. J. Biol. Chem. 269:22945&#226;&#8364;&#8220; <br /> Research Center University Tokyo. T.N. thanks members                      22951. Takagi Laboratory helpful discussions support.                  27. Mauersberger, S., M. Ohkuma, W.-H. Schunck, M. Takagi. 1995.  <br />                                                                                          dida maltosa, p. 411&#226;&#8364;&#8220;580. K. Wolf (ed.), Nonconventional yeasts bio- <br />                                                                                          technology. Springer, Heidelberg, Germany. <br />                                 REFERENCES                                           28. McConville, M. J., M.  J. Ferguson. 1993. structure, biosynthesis <br />  1. Bailey, D. , P. J. F. Feldmann, M. Bovey, N.  R. Gow,  J. P. Brown.        function glycosylated phosphatidylinositols parasitic protozoa <br />     1996. Candida albicans HYR1 gene, activated response              higher eukaryotes. Biochem. J. 294:305&#226;&#8364;&#8220;324. <br />     hyphal development, belongs gene family encoding yeast cell wall pro-       29. Mo &#194;&#168;sch, H.-U., R. L. Roberts, G. R. Fink. 1996. Ras2 signals     teins. J. Bacteriol. 178:5353&#226;&#8364;&#8220;5360.                                                  Cdc42/Ste20/mitogen-activated protein kinase module induce filamentous <br />  2. Blacketer, M. J., C. M. Koehler, S. G. Coats,  M. Myers, P. Madaule.           growth Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA 93:5352&#226;&#8364;&#8220; <br />     1993. Regulation dimorphism Saccharomyces cerevisiae: involvement           5356. <br />     novel protein kinase homolog Elm1p protein phosphatase 2A. Mol.          30. Mu &#194;&#168;hlschlegel, F. , W.  Fonzi. 1997. PHR2 Candida albicans <br />     Cell. Biol. 13:5567&#226;&#8364;&#8220;5581.                                                            encodes functional homolog pH-regulated gene PHR1  3. Blacketer, M. J., P. Madaule,  M. Myers. 1995. Mutational analysis          inverted pattern pH-dependent expression. Mol. Cell. Biol. 17:5960&#226;&#8364;&#8220;5967. <br />     morphologic differentiation Saccharomyces cerevisiae. Genetics 140:1259&#226;&#8364;&#8220;      31. Nakazawa, T., T. Motoyama, H. Horiuchi,  Ohta, M. Takagi. 1997. <br />     1275.                                                                                Evidence centromeric DNA region induces pseudohyphal <br />  4. Braun, B. R.,  D. Johnson. 1997. Control filament formation              growth dimorphic yeast, Candida maltosa. J. Bacteriol. 179:5030&#226;&#8364;&#8220;5036. <br />     Candida albicans transcriptional repressor TUP1. Science 277:105&#226;&#8364;&#8220;109.         32. Nuoffer, C.,  Horvath, H. Riezman. 1993. Analysis sequence <br />  5. Cabib, E., R. Roberts,  Sburlati, J. Silverman. 1982. Synthesis yeast       requirements glycosylphosphatidylinositol anchoring Saccharomyces <br />     cell wall regulation. Annu. Rev. Biochem. 51:763&#226;&#8364;&#8220;793.                        cerevisiae Gas1 protein. J. Biol. Chem. 268:10558&#226;&#8364;&#8220;10563. <br />  6. Conzelmann, , H. Riezman, C. Desponds, C. Bron. 1988. major              33. Nuoffer, C., P. Jeno&#194;&#168;,  Conzelmann, H. Riezman. 1991. Determinants <br />     125-kd membrane glycoprotein Saccharomyces cerevisiae attached          glycophospholipid anchoring Saccharomyces cerevisiae GAS1 pro- <br />     <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> bilayer inositol-containing <span id='am-1' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span>phospholipid</span>. EMBO J. 7:2233&#226;&#8364;&#8220;           tein plasma membrane. Mol. Cell. Biol. 11:27&#226;&#8364;&#8220;37. <br />     2240.                                                                            34. Ohkuma, M., C. W. Hwang, Y. Masuda, H. Nishida, J. Sugiyama,  Ohta, <br />  2086         NAKAZAWA ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />       M. Takagi. 1993. Evolutionary position n-alkane-assimilating yeast         46. Schmitt, M., T.  Brown, B. L. Trumpower. 1990. rapid simple <br />       Candida maltosa shown nucleotide sequence small-subunit ribosomal               method preparation RNA Saccharomyces cerevisiae. Nucleic <br />       RNA gene. Biosci. Biotechnol. Biochem. 57:1793&#226;&#8364;&#8220;1794.                                  Acids Res. 18:3091. <br /> 35.   Ohkuma, M., K. Kobayashi, S. Kawai, C. W. Hwang,  Ohta, M.                  47. Soll, D. R. 1996. emerging molecular biology switching Candida <br />       Takagi. 1995. Identification centromeric activity autonomously            albicans. ASM News 8:415&#226;&#8364;&#8220;420. <br />       replicating TRA region allows improvement host-vector           48. Soll, D. R., B. Morrow, T. Srikantha. 1993. High-frequency phenotypic <br />       Candida maltosa. Mol. Gen. Genet. 249:447&#226;&#8364;&#8220;455.                                        switching Candida albicans. Trends Genet. 9:61&#226;&#8364;&#8220;65. <br /> 36.   Ohkuma, M., S. Muraoka, C. W. Hwang,  Ohta, M. Takagi. 1993.                49. Stewart, S., P. T. Magee. 1990. Genetics Candida albicans. Microbiol. <br />       Cloning C-URA3 gene construction triple auxotroph (his5,              Rev. 54:226&#226;&#8364;&#8220;241. <br />       ade1, ura3) useful host genetic engineering Candida maltosa.      50. Stoldt, V. R.,  Sonneborn, C. E. Leuker, J. F. Ernst. 1997. Efg1p,       Curr. Genet. 23:205&#226;&#8364;&#8220;210.                                                              essential regulator morphogenesis human pathogen Candida albi- <br /> 37.   Ohkuma, M., S. Muraoka, T. Tanimoto, M. Fujii,  <br /> </body></html>