<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> GCD14p, Repressor GCN4 Translation, Cooperates Gcd10p <br />     Lhp1p Maturation Initiator Methionyl-tRNA <br />                    Saccharomyces cerevisiae&#226;&#8364;&#160; <br />  <br />             Gcd10p Gcd14p identified genetically repressors GCN4 mRNA translation Saccha- <br />           romyces cerevisiae. Recent findings indicate Gcd10p Gcd14p reside nuclear complex required           presence 1-methyladenosine tRNAs. Gcd14p essential protein predicted <br />           binding motifs S-adenosylmethionine, consistent direct function tRNA methylation. different <br />           gcd14 mutants exhibit defects cell growth accumulate high levels initiator methionyl-tRNA (tRNAMet <br />                                                                                                                       ) <br />           precursors containing 5* 3* extensions, suggesting defect processing primary transcript. Dosage <br />           suppressors gcd10 mutations, encoding tRNAMet   (hcIMT1 hcIMT4; hc indicates gene carried <br />           high-copy-number plasmid) homologue human La protein implicated tRNA 3*-end formation <br />           (hcLHP1), suppressed gcd14 mutations. fact, lethality GCD14 deletion suppressed           hcIMT4, indicating essential function Gcd14p required biogenesis tRNAMet <br />                                                                                                         . mutation           GCD10 deletion LHP1 exacerbated defects cell growth expression mature tRNAMet    gcd14 <br />           mutants, consistent functional interactions Gcd14p, Gcd10p, Lhp1p vivo. Surprisingly,           amounts NME1 RPR1, RNA components RNases P MRP, substantially lower gcd14 <br />           lhp1::LEU2 double mutants corresponding single mutants, 5S rRNA present           wild-type levels. findings suggest Gcd14p Lhp1p cooperate maturation subset RNA <br />           polymerase III transcripts. <br />  <br />  <br />    Eukaryotic cells respond different conditions starvation            short upstream open reading frames (uORFs) present stress regulating rates protein biosynthesis.             GCN4 mRNA leader sequence prevent ribosomes initi- aspect response mammalian cells involves phos-                ating translation GCN4 start codon conditions phorylation serine 51 alpha subunit translation                 <span id='am-39' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> sufficiency. Amino acid starvation triggers activa- <br /> initiation factor 2 (eIF2a) (53). phosphorylation event                 tion protein kinase Gcn2p, phosphorylates eIF2a <br /> reduces activity eIF2B, <span id='am-70' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>guanine</span> <span id='am-31' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> exchange               inhibits eIF2-<span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span>-tRNAMet     ternary complex  <br /> factor eIF2 recycles eIF2-<span id='am-9' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="GDP" datatype="xsd:string"></span>GDP</span> active form                    mation. situation specifically favors GCN4 translation  <br /> eIF2-GTP (20, 40, 46, 59). eIF2-GTP bind initiator                  cause, translating uORF1, ribosomes ac- <br /> methionyl-tRNA (tRNAMet      ) produce eIF2-GTP-                    quire ternary complex time recognize start codons <br /> tRNAMeti   ternary complex delivers initiator tRNA               uORF2, -3, -4 continue scanning downstream. <br /> 43S preinitiation complex;  phosphorylation eIF2                  majority ribosomes rebind ternary complex hibits general protein synthesis initiation. budding yeast            time reach GCN4 AUG codon reinitiate Saccharomyces cerevisiae, eIF2a phosphorylated                       instead (19, 20). cells deprived amino acid purine, leading                  Recessive mutations genes encoding depletion ternary complex levels, just occurs mamma-                essential subunits eIF2B, GCD1, GCD2, GCD6, GCD7, <br /> lian cells (19, 31). Interestingly, specifically stimulates          mimic effects eIF2 phosphorylation derepress <br /> translation distinctive mRNA encoding Gcn4p, tran-                  GCN4 mRNA translation absence Gcn2p amino <br /> scriptional activator 40 genes involved amino                acid starvation (29, 30). gcd mutations lead  <br /> acid biosynthesis (29), alleviating nutrient limita-             conditional slow growth temperature sensitivity nutrient- <br /> tion conditions trigger phosphorylation eIF2a yeast.              rich medium, phenotypes result defects general <br />    molecular mechanism couples GCN4 translation                  translation initiation (9, 16, 22, 27). thought gcd1, <br /> amino acid availability explored great               gcd2, gcd6, gcd7 mutations impair ability eIF2B past years (reviewed references 31 32).                                                                              recycle eIF2-GDP eIF2-GTP decrease rate <br />                                                                              ternary complex formation vivo (31, 46). causes <br />                                                                              lower rates translation initiation mRNAs leads <br />   * Corresponding author. Mailing address: Instituto Microbio- <br />                                                                              increased translation GCN4 mRNA, inversely <br /> log&#196;&#177; Bioqu&#196;&#177;&#194;&#180;mica del CSIC/Universidad Salamanca, Edificio  <br /> partamental Biolog&#196;&#177;  Campus Miguel Unamuno, 37007                    coupled availability ternary complexes cell (20). <br /> Salamanca, Spain. Phone: 34-923-121673. Fax: 34-923-224876. E-mail:             Recessive mutations GCD10 lead temperature- <br /> tamame@gugu.usal.es.                                                         sensitive growth constitutive derepression GCN4 trans- <br />   &#226;&#8364;&#160; M.T. dedicates paper memory Toma&#194;&#180;s Santos.                lation absence Gcn2p (26, 42). GCD10 essential <br />  <br />                                                                       4167 <br />  4168       CALVO ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> gene shown encode 62-kDa protein copuri- <br /> fied coimmunoprecipitated subunits eIF3 (23). <br /> eIF3 stimulates steps initiation pathway mam- <br /> malian cells   binding ternary complexes 40S ribosomes [47]). Accordingly, proposed gcd10 mu- <br /> tations derepress GCN4 translation decreasing ability eIF3 promote rebinding ternary complexes 40S subunits translated uORF1 continued scan- <br /> ning downstream GCN4 mRNA leader. allow subunits skip uORF4 reinitiate translation GCN4 AUG reduction level ter- <br /> nary complex formation (23). recently, gcd10 mutations reduce expression mature tRNAMet         posttranscriptional step phenotypes gcd10 <br /> mutants suppressed multiple copies genes IMT1 IMT4, encoding tRNAMet       (3). Interestingly, gcd10 mutants <br /> lack 1-methyladenosine (m1A) position 58 tRNAMet        tRNAs (17  containing modified base. absence m1A impair specifically expression mature tRNAMet     degradation unprocessed <br /> precursors containing 59 39 extensions (3). reduction mature tRNAMet     levels gcd10 mutants diminish <br /> ternary complex formation, decreasing rate general <br /> translation initiation producing constitutive derepression GCN4 translation (20).  impaired maturation tRNAMeti    explain known phenotypes gcd10 <br /> mutants (3). <br />    previously identified GCD14 GCD15 novel genes                             FIG. 1. Analysis GCD14 coding region. partial restriction map S. cerevisiae required translational repression                  GCD14 region shown center, indicating positions ORFs, <br />                                                                                     GCD14 SPT10 (44), identified plasmid isolated genomic <br /> GCN4 mRNA amino acid-replete conditions (17).                            library bearing GCD14 (pRC5). direction transcription indicated report molecular cloning GCD14                              arrows. line depicts genomic DNA insert pRC5, encodes essential protein containing binding motifs                       depicted fragments present subclones constructed pRC5 S-adenosylmethionine (S-AdoMet). gcd14 mu-                            low-copy-number vector pRS316 (pRC55 pRC56). ability sub- <br />                                                                                     clone complement phenotypes gcd14-1 gcd14-2 mutants shown tants reduced levels mature tRNAMet       accumulate                  right. map GCD14 region shown enlargements tRNAMeti   precursors containing 59 39 extensions. Similar                   2.6-kb SpeI-ClaI region plasmid pRC56 3.2-kb SpeI-ClaI region case gcd10 mutants, presence IMT1 IMT4                         pRC560 bearing gcd14::URA3 allele, 1.1-kb URA3 fragment <br /> high copy number suppresses phenotypes gcd14-1                           replaces ;730 bp GCD14 coding sequence. Restriction enzyme sites: B, <br />                                                                                     BamHI; Bg, BglII; C, ClaI; H, HindIII; Hp, HpaI; S, Sau3A; X, XbaI; Sp, SpeI. <br /> gcd14-2 mutants, overexpression IMT4 overcomes lethality gcd14::URA3 deletion. Gcd14p physically asso- <br /> ciated Gcd10p cell extracts (3), observed addi- <br /> tive effects gcd14 gcd10 mutations cell growth                        containing NotI site (59CGATCCACGGAAAAACGCGGCCGCTAATTAAA <br /> expression mature tRNAMet      . Interestingly,                  TGATTAAC39; NotI site underlined, GCD14 stop codon gcd14 mutations lead reduced expression                            boldface). resulting plasmid named pRC61.  avoid methylation                                                                                     ClaI site located downstream GCD14 gene plasmid pRC61, tRNAs RNA components RNases P (38) MRP                               amplified dam mutant Escherichia coli strain GM119.) (iii) 2.4-kb <br /> (54) combined deletion LHP1, encoding yeast                        ClaI fragment containing GCD14 coding sequences subcloned homologue human autoantigen La. findings                           YCp50 high-copy-number YEp24 vector (50), generating plasmids (3) suggest Gcd10p Gcd14p functionally coop-                        pRC62 pRC63, respectively. (iv) 115-bp NotI fragment, encoding                                                                                     tandem copies hemagglutinin (HA) epitope, inserted NotI <br /> erate Lhp1p promote maturation subset                           site pRC62 pRC63, create plasmids pRC64 pRC65, respectively. <br /> RNA polymerase III transcripts.                                                     Sequence analysis confirmed frame insertion HA-coding sequence <br />                                                                                     just upstream GCD14 stop codon. <br />                         MATERIALS METHODS                                          670-bp HpaI-EcoRI fragment 39 coding region GCD14 gene <br />                                                                                     fused frame trpE coding sequence pATH2 (21) generate <br />    Plasmids. Plasmids constructed work represented Fig. 1. Plasmid   pTRPC14. Construction correct frame fusion confirmed DNA <br /> pRC5 isolated yeast genomic library low-copy-number vector        sequencing. <br /> YCp50 (51) selecting complementation 3-amino-1,2,4-triazol resis-        Plasmid pJA128 constructed inserting ca. 3.5-kb XbaI-XhoI frag- <br /> tance (3ATr) Slg2 phenotypes strain Hm316 (gcd14-2). Plasmids pRC55          ment bearing GCD10 pMG107 (23) YCplac111 (24). Plasmid pJA103, pRC56 contain, respectively, 2.3-kb XbaI 2.8-kb SpeI-ClaI           containing lhp1::hisG::URA3::hisG allele, constructed PCR amplifi- <br /> fragments pRC5 subcloned corresponding sites pRS316 (56).          cation 200- 350-bp fragments containing, respectively, 59 39 <br />    Plasmid pRC56 digested HpaI BglII delete internal 730-bp      flanking sequences LHP1 using appropriate primers. 59 PCR <br /> fragment GCD14 coding sequence, BglII end filled         product digested EcoRI BglII cloned EcoRI- BglII- <br /> Klenow fragment. resulting linearized plasmid ligated blunt-ended      digested pNKY51 (1) create pJA1039. 39 PCR product digested 1.1-kb HindIII-HindIII fragment containing URA3 gene, generating pRC560.        BamHI SphI cloned BamHI- SphI-digested pJA1039 create <br />    Plasmid pRC57 generated subcloning pRS306 ;2.8-kb HpaI-            pJA103. <br /> BamHI fragment pRC5, extending 1 kb upstream GCD14 ATG                plasmids used work previously described: pE107 ;30 nucleotides 59 stop codon. Plasmids pRC64 pRC65              (GCD10) (23); p1775 (IMT4) (20); p2632 (IMT1), p2633 (IMT2), p2634 (IMT3), <br /> constructed follows.   2.8-kb SpeI-ClaI fragment pRC56            p2635 (IMT4), p2626(LHP1) (3); pNK985 (1); pRS315 (56); pRS426 (13); <br /> cloned corresponding sites SK1 polylinker Bluescript M131        pKOIII (60). <br /> (Stratagene), generating plasmid pRC6. (ii) NotI site introduced upstream        Yeast strain constructions. genotypes origins yeast strains used GCD14 stop codon vitro mutagenesis MUTA-GENE vitro           study summarized Table 1. Strains Hm295 (gcd14-1), Hm296 <br /> mutagenesis kit (Bio-Rad), using plasmid pRC6 oligonucleotide (RC 22)        (gcd14-2), Hm316 (gcd14-2) selected 3ATr Slg2 ascospores  VOL. 19, 1999                                                                  Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                                4169 <br />  <br />  <br />                                                                        TABLE 1. Yeast strains <br />   Strain                                                                   Genotype                                                                Source reference <br />  <br /> H117              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ URA3)                                                                    26 <br /> H160              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ URA3)                                                            17 <br /> H168              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ URA3)                                                            17 <br /> H752              MATa gcn2::LEU2 leu2-3 leu2-112 ura3-52                                                                                          26 <br /> H753              MATa gcn2::LEU2 leu2-3 leu2-112 ura3-52                                                                                          26 <br /> H1515             MATa leu2,3-112 trp1D63 ura3-52                                                                                                   G. Hinnebusch <br /> Hm295             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ ura3-52)                                                         17 <br /> Hm296             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ ura3-52)                                                         17 <br /> Hm296g            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (gcd14 URA3)                                                                 study <br /> Hm296G            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (GCD14 URA3)                                                                 study <br /> Hm298             MATa gcn2-101 gcn3-101 his1-29 ura3-52 gcd10-505 (HIS4::lacZ ura3-52)                                                            23 <br /> Hm316             MATa gcd2-101 his1-29 ino1 leu2-3 leu2-112 ura3-52 gcd14-2                                                                       17 <br /> Hm397             MATa gcn2-101 gcn3-101 his1-29 ino1 leu2::hisG ura3-52 gcd10-505 gcd14-2                                                         study <br /> Hm406             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                               study <br /> Hm407             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                       study <br /> Hm408             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                       study <br /> F104              MATa ura3-52                                                                                                                     G. Fink <br /> F105              MATa ura3-52                                                                                                                     G. Fink <br /> F294              MATa prt1-1 ura3-52 ade2-1 leu2-3 leu2-112                                                                                       G. Johnston <br /> JAy113            MATa leu2,3-112 trp1D63 ura3-52 lhp1::hisG::URA3::hisG (p2705::GCD10-HA)                                                         study <br /> JAy206            MATa gcd10D1 ura3-52 trp1 leu2D 1 his3D200 pep4::HIS4 prb1D6 can1 (pJA128::GCD10) <br /> JAy142            MATa gcd10D1 ura3-52 trp1 leu2D1 his3D200 pep4::HIS4 prb1D6 can1(GCD10-HA URA3)                                                  3 <br /> JAy143            MATa gcd10D1 ura3-52 trp1 leu2D1 his3D200 pep4::HIS4 prb1D6 can1 (GCD10 URA3)                                                    3 <br /> W303 1A           MATa ura3-52 leu2-3 leu2-112 trp1-289 his3-11 ade2-1 can1-100                                                                    F. del Rey <br /> W303 1B           MATa ura3-52 leu2-3 leu2-112 trp1-289 his3-11 ade2-1 can1-100                                                                    F. del Rey <br /> YNG1              MATa ura3-52 /ura3-52 leu2-3 leu2-112/leu2-3 leu2-112 trp1-289 /trp1-289 his3-11/his3-11 ade2-1/ade2-1                         study <br />                    can1-100/can1-100 <br /> YNG174            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52)                                                                   study <br /> YNG175            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) gcd14-1                                                           study <br /> YNG176            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) gcd14-2                                                           study <br /> YRC1              MATa gcn2-101/gcn2::LEU2 ura3-52/ura3-52 leu2-3 leu2-112/leu2-3 leu2-112 his1-29/HIS1 ino1/INO1                                  study <br />                    gcd14-2/GCD14 <br /> YRC2              MATa ura3-52/ura3-52                                                                                                           study <br />  <br />  <br />  <br /> genetic crosses previously described (17). Isogenic Hm296G Hm296g strains         coding sequence completely replaced LEU2 (60) follows. Ura2 obtained transforming Hm296 Ura1 integrating plasmid              derivative strain H117 selected 5-FOA medium, strain, named <br /> pRC57, containing incomplete copy GCD14. Depending location           YNG174 (GCD14), Hm295 (gcd14-1) Hm296 (gcd14-2), crossover, integration pRC57 yielded nontandem duplication wild-          transformed 6.5-kb BglII fragment pNK985 (1) described type GCD14 truncated gcd14 allele, gcd14-2 truncated gcd14         disrupt LEU2 gene strain hisG::URA3::hisG sequences. Leu2 <br /> alleles, separated plasmid sequences.  Hm296G) led              Ura1 transformants selected transformed 4.6-kb SalI-XbaI <br /> 3AT-sensitive phenotype,  Hm296g) conferred 3AT resis-          fragment pKOIII (60) containing null allele lhp1::LEU2. Total RNA <br /> tance.                                                                                isolated Leu1 transformants analyzed Northern blotting    3.2-kb SpeI-ClaI fragment pRC560 containing gcd14::URA3 used            verify absence LHP1 mRNA isogenic strains Hm406 (GCD14 step gene disruptions (52) GCD14 alleles diploid         lhp1::LEU2), Hm407 (gcd14-1 lhp1::LEU2), Hm408 (gcd14-2 lhp1::LEU2). <br /> strain YRC1 YRC2, constructed crossing Hm316 H753 F104               Strain JAy113 constructed transforming H1515 (MATa ura3-52 leu2- <br /> F105, respectively (Table 1). Total DNA isolated Ura1 transformants          3,112 trp1D63) Ura1 4.3-kb SphI/EcoRI fragment isolated pJA103 analyzed Southern blotting verify replacement null allele       containing lhp1::hisG::URA3::hisG. Disruption LHP1 confirmed PCR successful. Heterozygous GCD14/gcd14::URA3 gcd14/gcd14::URA3 dip-              amplification fragments diagnostic lhp1::hisG::URA3::hisG JAy113 <br /> loids sporulated, tetrad analysis conducted described Results.     genomic DNA. Strain JAy206 constructed transforming JAy143 (3)    diploid strain YNG1 constructed crossing W3031A (ura3-52 leu2-          Leu1 pJA128, bearing single-copy GCD10 LEU2, stable Ura2 <br /> 3,112) W3031B (ura3-52 leu2-3,112) (Table 1). allele GCD14             derivative exhibiting wild-type growth isolated medium containing 1 mg <br /> YNG1 replaced gcd14::URA3  described , Ura1 Leu2                5-FOA ml (6). <br /> heterozygous GCD14/ gcd14::URA3 diploid transformed high-copy-              Media genetic techniques. Sensitivity 3AT (Sigma; catalog  A8056) <br /> number LEU2 plasmid p1775 containing IMT4 (20). Leu1 diploid transfor-              tested previously described (28). Transformation yeast strains mant sporulated, tetrad analysis conducted following             carried described Ito et al. (34). Standard genetic techniques media <br /> results: 4:0 segregation viability observed ascospores 7 tetrads   previously described (55). <br /> 12 analyzed, showing p1775 (IMT4) rescued lethality gcd14::URA3            Chromosomal mapping GCD14. GCD14 physically mapped chromo- <br /> deletion.                                                                             X using Chromo-Blot (Clontech), yeast chromosomes    construct strain Hm397, gcd10-505 gcd14-2 ascospore clone isolated        fractionated clamped homogeneous electrical field electrophoresis blot- tetratype tetrad obtained cross Hm296 Hm298 (23).           ted nylon membrane. individual blot lanes probed, hybridizing genotypes ascopore clones tetrad transformed      radiolabeled 2.8-kb HpaI-BamHI fragment pRC5 contains vectors low-copy-number plasmids containing GCD10 GCD14, fol-             GCD14 coding sequence. probe radiolabeled 2.4-kb XbaI frag- <br /> lowed phenotype testing transformants. LEU2 gene disrupted          ment pRC55 used hybridize set filters containing ordered gcd10-504 gcd14-2 strain transforming 6.5-kb BglII fragment       lambda library yeast genomic DNA (49). probe hybridized clone pNK985 (1), containing 3.8-kb hisG::URA3::hisG cassette inserted          70692, second hybridized clone clone 70091, EcoRI site LEU2 gene, Leu2 Ura1 transformants selected.           bearing inserts corresponding containing region encompassing TIF2 transformants plated SD medium containing 1 mg 5-fluoro-orotic         URA2 sequences left arm chromosome X. map GCD14 <br /> acid (5-FOA) ml (6) select Ura2 derivative Hm397  Table 1          genetically, crossed H168 (gcn2-101 gcn3-101 gcd14-2 ino1) H753 <br /> genotypes). Hm397 transformed low-copy-number plasmids containing            (gcn2::LEU2 GCD14 INO1) analyzed cosegregation 3ATr, Slg2, GCD10 (pJA128), GCD14 (pRC56), vectors (pRS315 pRS316)                 Ino2 phenotypes 45 tetrads. observed 32 parental ditype, 0 nonparental <br /> generate isogenic transformants genotypes GCD10 gcd14-2                  ditype, 13 tetratype asci, locating GCD14 14.4 centimorgans (Hm420), gcd10-505 GCD14 (Hm421), GCD10 GCD14 (Hm422), gcd10-505                  INO1. <br /> gcd14-2 (Hm423).                                                                         DNA sequence analysis cloning gcd14 alleles. nucleotide sequence <br />    LHP1 replaced GCD14 gcd14 strains null allele       2.8-kb SpeI-ClaI fragment pRC56 obtained strands using <br />  4170        CALVO ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br />  Sequenase version 2.0 kit (U.S. Biochemical Corp). Reverse primer, M13-40 <br /> primer, oligonucleotides derived previously determined <br /> sequences used primers, pRC56 used double-stranded DNA <br /> template. DNA sequence analyzed using DNAsis sequence <br /> analysis program (Hitachi software), comparisons GCD14 sequence BLAST programs (2). <br />    gcd14-1 gcd14-2 alleles cloned PCR genomic DNAs strains H160 H168, respectively, templates Expand High Fidelity <br /> PCR (Boehringer Mannheim). primers consisted oligonucleotides <br /> having 20 bases corresponding sequences beginning (200 bp upstream ATG) end GCD14 coding sequence. Amplification products <br /> corresponding gcd14-1 gcd14-2 isolated independent PCRs case, subcloned pGEM vector (Promega), sequenced strands using appropriate oligodeoxynucleotides derived previously determined GCD14 sequence primers. <br />    Production Gcd14p-specific antiserum. TrpE-Gcd14p fusion protein <br /> encoded plasmid pTRPC14 overexpressed E. coli HB101 partially <br /> purified insoluble fraction cell extracts sodium dodecyl <br /> sulfate (SDS)-polyacrylamide gel electrophoresis described (21). Gel <br /> slices containing ;200 mg fusion protein used inoculate rabbits. <br /> Immunization rabbits collection immune antisera  <br /> ducted Hazelton Laboratories (Vienna, Va.). <br />    RNA isolation Northern blot analysis. Total RNA isolated previ- <br /> ously described (36). Samples dried, resuspended loading buffer (90% <br /> formamide, 1 mM EDTA [pH 8.0], 0.1% bromophenol blue, 0.1% xylene cya- <br /> nol), heated .90&#194;&#176;C 10 min separation 6 8% polyacryl- <br /> amide-bisacrylamide (19:1)&#226;&#8364;&#8220;8.3 M <span id='am-75' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-76' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> gels electrophoresis 250 V approximately 3.5 h. Gels run 250 V 20 min prior separation                 FIG. 2. Deduced amino acid sequence Gcd14p. predicted amino acid <br /> RNAs. Following electrophoresis, gels soaked 0.53 Tris-borate-EDTA                 sequence S. cerevisiae Gcd14p given single-letter code. regions 20 min, RNA transferred positively charged nylon membranes                  sequence similarity observed S-AdoMet-dependent methyltransferases <br /> (Boehringer Mannheim) buffer electrotransfer (Trans-Blot Cell;              boxed indicated motif motif II (35). conserved G residue Bio-Rad) 3.5 h 19 V. RNA immobilized membranes UV                         position 5 motif G118 Gcd14p. glutamate residue commonly located <br /> cross-linking UV-Stratalinker 2400 (Stratagene) according manu-              17 19 residues C terminal motif 17 residues C terminal motif <br /> facturer&#226;&#8364;&#8482;s instructions. tRNAs membranes detected hybridization             Gcd14p, position 139 (*), cluster hydrophobic residues, hhXh radiolabeled oligonucleotides hybridization buffer (0.25 M Na2HPO4 [pH             (D/E), positions 135 138 (underlined, LFSF) precedes E element. 7.5], 7% SDS, 1% bovine serum albumin, 1 mM EDTA) 50 52&#194;&#176;C 12                  central invariant aspartate motif II conserved position 209 Gcd14p. <br /> 20 h. Oligonucleotides radiolabeled 59 ends [g-32P]<span id='am-20' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span> (6,000             Motifs II separated 83 residues Gcd14p. putative nuclear <br /> Ci/mmol) T4 polynucleotide kinase (Pharmacia). Following hybridization,                localization signal (7) (shaded box) located residues K282 R294 membranes washed 23 SSC (13 SSC 0.15 M NaCl plus 0.015                   Gcd14p sequence. <br /> M sodium citrate)&#226;&#8364;&#8220;0.1% SDS 30 min room temperature 13 <br /> SSC&#226;&#8364;&#8220;0.1% SDS 60&#194;&#176;C 20 min prior detection autoradiography. Direct <br /> quantitation hybridization signals conducted phosphorimager anal- <br /> ysis BAS-1500 PhosphorImager MacBAS Ver.2.x software (Fuji <br /> Film). reprobing, membranes stripped radioactive probes incu-               cloned authentic GCD14, mapped chromosomal <br /> bation 100&#194;&#176;C 1.0% SDS cooled room temperature.                                location genomic insert pRC5 chromosome X S. <br />    oligonucleotides used probes Northern blots corresponding             cerevisiae. radiolabeled 2.8-kb HpaI-BamHI fragment ob- <br /> transcript species specifically hybridize (RNA) follows:           tained pRC5 hybridized exclusively chromosome X 59-TCGTTTCGATCCCGAGGACATCAGGGTTATGA-39 (tRNAMet                              ), (ii) 59- <br /> TCGGTTTCGATCCCGAGGACATCAGGGTTATGA-39 (tRNAMet <br />                                                                                                                                                     ), (iii) 59-   yeast Chromo-Blot  Materials Methods). cloned <br />                                                                        e <br /> TGCTCGAGGTGGGGWTTGAACCCACGACGG-39 (tRNAUAU                         Ile <br />                                                                         ), (iv) 59-CA      sequence mapped left arm chromosome X, GTTGATCGGACGGGAAAC-39 (5S rRNA), (v) 59-TGCGTTCTTCATCGA                                    region close SPT10/PBS2/TRK1, hybridizing TGCGAGAACC-39 (5.8S rRNA), (vi) 59-GACGTCCTACGATTGCACTC-39                                 HpaI-BamHI fragment filters containing ordered lambda <br /> (RPR1 RNA), (vii) 59-TCCCCCGGGTGAATCCATGGACCAAGA-39 (NME1 <br /> RNA), (viii) 59-GGTTCATCCTTATGCAGGG-39 (U6 RNA), (ix) 59-AGCCG                             library yeast genomic clones  Materials Methods). <br /> AACTTTTTATTCCATTCG-39 (pre-tRNACGA             Ser <br />                                                    ) (61), (x) 59-AGCCCAAG             tetrad analysis estimated gcd14-2 14.4 <br /> AGATTTCGAGTCTCTCG-39 (tRNACGA            Ser <br />                                               ) (61).                                      centimorgans INO1  Materials Methods).    Nucleotide sequence accession number. EMBL accession number                 fact INO1 located 50 kb cloned sequence GCD14 nucleotide sequence Z54149. <br />                                                                                            left arm chromosome X verifies insert pRC5 <br />                                                                                            contains wild-type allele GCD14. <br />                                     RESULTS                                                   define boundaries GCD14, subclones                                                                                            genomic insert pRC5 constructed low-copy-number <br />    Isolation characterization wild-type GCD14 gene                              plasmids tested complementation gcd14-1 gcd14-1 gcd14-2 mutant alleles. Mutations                                gcd14-2 mutations (Fig. 1). results analysis localized <br /> GCD14 isolated suppressors 3AT phenotype                                 GCD14 2.8-kb SpeI-ClaI fragment pRC56 (Fig. 1 gcn2-101 gcn3-101 double mutant (17). gcn mutations                                  data shown), deletions remove sequences confer sensitivity 3AT impair derepression                              59 39 end (pRC55) fragment completely <br /> GCN4, histidine biosynthetic genes regulated Gcn4p,                              abolished complementation activity (Fig. 1). SpeI-ClaI response histidine starvation imposed 3AT (28). gcd14                             fragment sequenced strands contain <br /> mutations restore derepression GCN4 translation                                single ORF 1,149 bp predicted encode protein <br /> confer 3AT resistance gcn2 gcn3 background. addition,                              383 amino acids molecular mass 43,917 Da. (Fig. <br /> recessive gcd14 mutations lead slow-growth phenotype                                  2). Comparison deduced amino acid sequence (Slg2) rich media 28&#194;&#176;C exacerbated                                  Gcd14p protein sequences GenBank, EMBL, lower higher temperatures (18 37&#194;&#176;C) (Ts2 phenotype)                                SWIS-PROT databases revealed gene 60% simi- <br /> (data shown). cloned wild-type allele GCD14                                  larity 47.5% identity Schizosaccharomyces pombe yeast genomic library plasmid pRC5 (Fig. 1)                                   CPD1, gene proposed encode 42-kDa subunit DNA <br /> complementing 3ATr Slg2 phenotypes conferred                                    polymerase d (62). <br /> gcd14-2 gcn2-101 gcd14-2 strain Hm316. prove                                   predicted polypeptide sequence Gcd14p contains  VOL. 19, 1999                                                Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                                   4171 <br />  <br />  <br /> regions sequence similarity described S-AdoMet- <br /> dependent methyltransferases (35). motifs occur order separated intervals similar lengths, suggested contribute binding substrate <br /> S-AdoMet product S-adenosylhomocysteine S-Ado <br /> Met-dependent methyltransferases. Motif (VIEAGTGSG), <br /> located residues 114 122 Gcd14p (Fig. 2), similar conserved regions described DNA <span id='am-15' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-16' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> cytosine methyltransferases (33) 69 84 <br /> non-DNA methyltransferases (35). Gcd14p contains conserved glutamate (E) residue located 17 residues C termi- <br /> nal motif cluster hydrophobic residues, LFSF, positions 135 138 preceding E element, matching established consensus hhXh (D/E). Motif II (APWDAI <br />                                                                        FIG. 3. Polysome profiles isogenic gcd14-2 GCD14 strains. Isogenic <br /> PH), located amino acid residues 206 213             strains Hm296g Hm296G cultured overnight yeast extract-peptone- <br /> Gcd14p (Fig. 2), present 46 84 enzyme sequences        dextrose 23&#194;&#176;C used inoculate flasks containing 300 ml yeast <br /> analyzed (35), including bacterial tRNA methyltransferase       extract-peptone-dextrose optical density 600 nm ;0.1. Cultures (33). Generally, motifs II separated 57 6 13 amino     incubated room temperature rotary shaker 250 rpm. optical <br />                                                                     density 600 nm ;4.0, cells collected resuspended appro- <br /> acids, Gcd14p separation 82 residues;         priate volume yeast extract-peptone-dextrose prewarmed 39&#194;&#176;C grown  RNA methyltransferases porphyrin precursor            optical density 600 nm ;2.0. flasks, 150 ml collected <br /> methyltransferases known exceptions rule.               immediately 39&#194;&#176;C, t 5 0 samples, 150 ml fresh prewarmed yeast <br />    identified mutations gcd14-1 gcd14-2 PCR        extract-peptone-dextrose added flask. Samples 150 ml                                                                     collected flasks 1.5 h processed polysome analysis amplification alleles genomic DNAs H160         velocity sedimentation cell extracts sucrose gradients (22). Gradients <br /> H168, respectively, followed automatic DNA sequencing            fractionated scanned 254 nm, resulting absorbance <br />  Materials Methods). gcd14-1 allele contains          profiles shown, tops gradients left. positions 40S, <br /> single point mutation, consisting transversion G C     60S, 80S ribosomal species indicated arrows. strain,                                                                     gradients correspond samples incubated 39&#194;&#176;C 0 h (t 5 0) nucleotide 265, replacing histidine codon (GAC)        gradients contain samples obtained 1.5 h incubation 39&#194;&#176;C. <br /> aspartate codon (CAC) amino acid residue 89. muta-          Areas delimited inside profiles correspond fractions total <br /> tion falls region ca. 23 amino acid residues   absorbance profiles represented 2-mer 3-mer polysomes. gcd14-2 conserved CPD1 S. pombe GCD14;              strain, 2-mers 3-mers represented 26% polysome mass                                                                     permissive temperature comprised 32% polysomes 1.5 h 39&#194;&#176;C.  does alter predicted S-AdoMet binding motifs       contrast, 2-mers 3-mers represented 24 25% polysome mass Gcd14p. gcd14-2 allele contains single G T trans-       23&#194;&#176;C 1.5 h 39&#194;&#176;C, respectively, wild-type strain Hm296G. version nucleotide 3 GCD14 coding sequence, replac-       P/M ratios calculated t 5 0 t 5 1.5 h independent <br /> ing GCD14 methionine start codon ATG (Met) ATT             experiments, following mean values standard errors  parentheses) <br />                                                                     obtained: t 5 0, P/M 5 4.2 (0.83) GCD14 2.8 (0.44) gcd14-2; <br /> (Ile).                                                              t 5 1.5 h, P/M 5 5.2 (0.35) GCD14 3.5 (0.72) gcd14-2. <br />    Immunoblot analysis cell extracts antibodies Gcd14p  Materials Methods) showed gcd14-1 product expressed level greater equal wild type, gcd14-2 product           formed URA3 low-copy-number plasmid containing substantially diminished abundance (data shown). Pre-        GCD14 coding region (pRC56 [Fig. 1]), induced sumably, low level gcd14-2 protein produced          sporulate. Tetrads contained viable ascospores inefficient translation initiation ATT start codon.          dissected, showing GCD14 complements lethal phe- <br />    GCD14 essential gene. slow-growth phenotype         notype haploid spores bearing gcd14::URA3 mutation. gcd14-1 gcd14-2 mutations suggests Gcd14p          gcd14-2 cells exhibit modest defect general translation <br /> essential function role GCN4-specific transla-        initiation. essential nature GCD14 gene tional control. test possibility, constructed dele-    role repressor GCN4 mRNA translation (17) <br /> tion-insertion null allele, gcd14::URA3, plasmid pRC560          suggested Gcd14p required general <br /> (Fig. 1) used replace allele GCD14 ura3-        initiation translation. investigate possibility,  <br /> 52/ura3-52 diploid strains YRC1 (gcn2/gcn2 GCD14/gcd14-2            alyzed total polysome profiles wild-type mutant <br /> ura3-52/ura3-52) YRC2 (GCN2/GCN2 GCD14/GCD14                    gcd14-2 strains fractionating cell extracts sucrose <br /> ura3-52/ura3-52)  Materials Methods). verified        gradients velocity sedimentation. gcd14-2 mutants Southern blot analysis gcd14::URA3 allele replaced         form colonies 37&#194;&#176;C;  exhibited little copies GCD14 Ura1 transformants         growth defect liquid medium hours 37&#194;&#176;C <br /> YRC1 YRC2 obtained (data shown). Tetrad           (data shown). Quantitation total polysome profiles analysis transformants revealed 24 asci       gcd14-2 GCD14 strains grown 23&#194;&#176;C mid-logarithmic diploid, spores formed colonies        phase incubated 4 h 37&#194;&#176;C revealed signifi- <br /> rich medium 4 days 28&#194;&#176;C (data shown). viable       differences polysome/monosome (P/M) ratio. Ac- <br /> spores Ura2 (ura3-52)  contain GCD14),            cordingly, examined polysome profiles isogenic YRC1 transformant 3ATs (ura3-52            strains Hm296g (gcd14-2) Hm296G (GCD14) grown gcn2 GCD14), indicating gcd14-2 allele            39&#194;&#176;C, gcd14-2 mutant exhibits detectable <br /> replaced gcd14::URA3. results demonstrate             growth defect liquid medium. shown Fig. 3, P/M <br /> GCD14 essential growth.                                      ratio considerably lower gcd14-2 mutant (P/M ratio, <br />    confirm GCD14 essential gene          2.4) isogenic GCD14 parent strain (P/M ratio, 5.7) <br /> function disrupted, Ura2 derivative het-          1.5 h 39&#194;&#176;C. small reduction erozygous gcd14::URA3/GCD14 transformant diploid strain          average size polysomes gcd14-2 mutant 39&#194;&#176;C <br /> YRC2 isolated growth 5-FOA medium (6), trans-              Fig. 3 legend). differences polysome <br />  4172       CALVO ET AL.                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     gcd10 alleles (data shown). Furthermore, suppression                                                                                     hcIMT hcLHP1 specific gcd10 gcd14, <br />                                                                                     temperature-sensitive mutations genes encoding subunits <br />                                                                                     translation initiation factors eIF2 eIF3 (sui2-1                                                                                     prt1-1, respectively) suppressed dosage sup- <br />                                                                                     pressors (3). <br />                                                                                        gcd14 mutants defective processing primary pre- <br />                                                                                     cursors initiator tRNAMet. High-copy-number IMT1, -2, <br />                                                                                     -3, -4 LHP1 suppress phenotypes gcd10 mutants <br />                                                                                     mutants contain reduced amounts mature <br />                                                                                     tRNAMeti    , dosage suppressors compensate                                                                                     defect increasing mature tRNAMet                nearly wild-type <br />                                                                                     (hcLHP1) higher wild-type (hcIMT) levels (3). Ac- <br />                                                                                     cordingly, used Northern analysis investigate                                                                                     steady-state levels mature tRNAMet            reduced                                                                                     gcd14 mutants H160 (gcd14-1) H168 (gcd14-2) compared <br />                                                                                     parental wild-type strain (H117). probe                                                                                     tRNAMeti    , containing sequences complementary nucleotides <br />                                                                                     33 64 tRNAMet        , hybridizes mature tRNAMet                                                                                         different precursor tRNAMet          species bearing unique 59-39 ex- <br />                                                                                     tensions encoded different IMT genes (3)  Materials <br />                                                                                     Methods). Northern data Fig. 5 quantified <br />                                                                                     phosphorimager, results given Table 2. <br />                                                                                     Relative level elongator tRNAMet (tRNAMet              e   ),                                                                                     calculated amounts mature tRNAMet             mutant <br />                                                                                     strains ca. half wild-type strain 28&#194;&#176;C <br />                                                                                     1.5 h 37&#194;&#176;C (Fig. 5A; Table 2). addition,    FIG. 4. Phenotypes gcd14 dosage suppressors. independent trans- <br /> formants strains Hm295 (gcd14-1) Hm296 (gcd14-2) carrying vector       levels tRNAMet        precursors 2.3- 3.2-fold higher pRS426 (gcd14), low-copy-number plasmid bearing GCD14 (pRC56), high-           28 37&#194;&#176;C, respectively, leading fivefold increase copy-number plasmids bearing IMT1 (p2632), IMT2 (p2633), IMT3 (p2634),              ratio precursor mature tRNAMet             gcd14 mutants <br /> IMT4 (p2635), GCD10 (pE107), LHP1 (p2636) streaked single               versus wild type (Fig. 5A; Table 2). data strongly <br /> colonies minimally supplemented SD plates incubated 28&#194;&#176;C  plates) 37&#194;&#176;C  plates) 2 days. locations transformants   suggest Gcd14p required efficient processing plates indicated inside central schematic.                                  nascent tRNAMet         transcripts. noteworthy tRNAMet                                                                                        species migrating faster wild-type mature form                                                                                     detected gcd14 mutants conditions (species f <br />                                                                                     Fig. 5A), suggesting fully 59-39-processed tRNAMet              size content dramatic, consistently ob-                       unstable incorrectly processed mutants. Sim- <br /> served independent experiments  Fig. 3 legend).                        ilar results obtained gcd14-2 mutant Anderson et data suggest gcd14-2 mutants exhibit modest                        al. (3), accumulation pre-tRNAMet           37&#194;&#176;C defect general translation initiation. magnitude                     extensive observed  <br /> defect observed gcd1 gcd2 mutants,                             described gcd10-504 (3), gcd14-1 gcd14-2 <br /> consistent lesser derepression GCN4 translation                         mutations little effect size steady-state <br /> associated gcd14 versus gcd1 gcd2 mutations (17, 26).                       elongator tRNAMet (Fig. 5B; Table 2). case <br />    Genes encoding initiator tRNAMet Lhp1p dosage sup-                        tRNAIle UAU, gcd14-1 gcd14-2 mutations led small <br /> pressors Gcd2 Slg2 phenotypes gcd14 mutants.                          increases amounts primary transcript containing <br /> gcd10 gcd14 mutations lead constitutive derepression                      59 39 extensions plus intron (slower-migrating <br /> GCN4 translation absence Gcn2p (Gcd2 phenotype),                          species pretRNAIle       UAU) possibly small reduction conferring resistance 3AT gcn2 strain background (17,                       levels mature tRNAIle         UAU mutant strains (Fig. 5C; <br /> 26). recently (3) 3ATr Ts2 phenotypes                         Table 2). Consequently, precursor tRNA/mature tRNA <br /> conferred gcd10 mutations gcn2 background fully                      ratios tRNA slightly elevated mutant versus <br /> suppressed presence high copy number of                        wild-type strain (Table 2), albeit extent observed genes, IMT1 IMT4, encoding tRNAMet   (11, 14)                     tRNAMet    . similar result obtained tRNATyr          GTA partially suppressed high-copy-number LHP1 (39, 60).                       (data shown). results raise possibility view physical interaction Gcd10p                           Gcd14p required primarily efficient removal 59 39 <br /> Gcd14p (3), asked dosage suppressors gcd10                        extensions initiator tRNAMet make small  <br /> mutations suppress 3ATr Slg2 phenotypes                          tribution processing certain tRNAs conferred gcd14 mutations. IMT genes                         initiator. <br /> high-copy-number plasmids (hcIMT) suppressed 3ATr                                  confirmed Northern analysis presence (data shown) Slg2 (Fig. 4) phenotypes gcd14-1                            hcIMT hcLHP1 plasmids gcd14 mutants led  <br /> gcn2-101 gcd14-2 gcn2-101 mutants                             creased levels mature tRNAMet           , accounting sup- <br /> extent did low-copy-number plasmid containing GCD14,                           pressor phenotypes. Transformants gcd14-1 strain  high-copy-number LHP1 (hcLHP1) partially sup-                          taining hcIMT1, hcIMT2, hcIMT3, hcIMT4 levels pressed mutant Slg2 phenotype (Fig. 4 data                              mature tRNAMet         3- 7-fold higher ob- <br /> shown). hcGCD10 suppressor activity gcd14 mu-                         served vector transformant 1.5- 3.5-fold higher tants (Fig. 4), single-copy GCD14 hcGCD14                           seen wild-type GCD14 transformant (data suppressed phenotypes mutants containing different                           shown). High-copy LHP1 led 1.5-fold increase  VOL. 19, 1999                                                                    Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                                    4173 <br />  <br />  <br />                                                                                          wild-type strain (Fig. 6B; Table 3), similar results  <br />                                                                                          scribed  Compared gcd14-2 mutant, gcd10-505 <br />                                                                                          single mutant 37&#194;&#176;C showed similar accumulation                                                                                          precursors somewhat greater reduction mature tRNAMet                                                                                             expression 37&#194;&#176;C (Fig. 6B, lanes 6 8). mutants showed <br />                                                                                          7- 10-fold increases precursor tRNA/mature tRNA <br />                                                                                          ratios tRNAMeti    (Table 3). findings  <br />                                                                                          ing suggest Gcd10p required                                                                                          efficient end processing tRNAMet     . gcd10-505 gcd14-2 <br />                                                                                          double mutant 37&#194;&#176;C showed greater reduction mature <br />                                                                                          tRNAMeti   levels did single mutant, exhibiting levels <br />                                                                                          30% seen wild-type transformant (Fig. 6B, <br />                                                                                          lanes 5 8; Table 3). additivity defects mature <br />                                                                                          tRNAMeti   expression suggests Gcd10p Gcd14p partici- <br />                                                                                          pate process involved pre-tRNAMet      processing. <br />                                                                                             Following temperature shift, double-mutant culture <br />                                                                                          increased mass 30% (data shown), <br />                                                                                          level mature tRNAMet      reduced ca. 69% <br />                                                                                          (Table 3).  appears reduction mature <br />                                                                                          tRNAMeti    levels accounted dilution                                                                                          preexisting mature molecules cell growth following                                                                                          temperature shift. implies substantial fraction                                                                                          mature tRNAMet     degraded 37&#194;&#176;C double mutant. <br />                                                                                          noteworthy level tRNAMet      precursors                                                                                          double mutant 37&#194;&#176;C, higher wild-type <br />                                                                                          strain, lower seen single mutants.                                                                                          indicate unprocessed tRNAMet        precursors ac- <br />                                                                                          cumulate double mutant degraded rapidly <br />                                                                                          37&#194;&#176;C single mutants. Finally, interesting                                                                                          single double mutants showed ca. 10-fold-higher levels <br />    FIG. 5. gcd14 mutants defective processing primary precursors           tRNAMeti     precursors 28&#194;&#176;C compared wild-type <br /> initiator tRNAMet tRNAUAU    Ile <br />                                       . Northern blot analysis total RNA (10 mg) <br /> isolated strains H117 (GCD14), H160 (gcd14-1), H168 (gcd14-2) grown <br />                                                                                          strain, little reduction mature yeast extract-peptone-dextrose medium 28&#194;&#176;C mid-exponential phase (0 h           tRNAMeti   expression conditions (Table 3). 37&#194;&#176;C) shifted 37&#194;&#176;C 1.5 h shown. blot probed              processing tRNAMet    slower mutants temper- <br /> radiolabeled oligonucleotide specifically hybridized tRNAMet             atures leads deficit mature tRNAMet          levels <br />                                                                                                                                                    stripped reprobed radiolabeled oligonucleotides specific        Met                Ile <br /> tRNAe (B) tRNAUAU (C)  Materials Methods). positions pre-tRNAMeti   species, mature tRNAMet    , tRNAMet <br />                                                     e  , primary (upper band) <br />                                                                              Ile 59- 39-end-processed intron-containing (lower band) pre-tRNAUAU                          Ile <br /> mature tRNAUAU indicated left. Indicated right                          Met <br /> positions pre-tRNAi containing 59 39 extensions encoded IMT2                     TABLE 2. Comparison tRNAMet <br />                                                                                                                             tRNAUAU  Ile <br />                                                                                                                                                levels <br /> IMT3 (b), pre-tRNAMet      containing 59 39 extensions encoded IMT1                      GCD14 gcd14 isogenic strainsa <br /> IMT4 (c), mature tRNAMet      (e), aberrantly processed tRNAMet    species (f) <br />  text details).                                                                                                                     Value  <br />  <br />                                                                                                                              GCD14              gcd14-1            gcd14-2 <br />                                                                                                Parameter                                                                                                                               strain             strain             strain <br />                                                                                                    tRNA <br /> level mature tRNAMet    gcd14-1 mutant Hm295                                                             28&#194;&#176;C <br />                                                                                                                                   37&#194;&#176;C, <br />                                                                                                                                           28&#194;&#176;C <br />                                                                                                                                                      37&#194;&#176;C, <br />                                                                                                                                                              28&#194;&#176;C <br />                                                                                                                                                                         37&#194;&#176;C, <br /> 37&#194;&#176;C, reaching level slightly observed                                                                        1.5 h              1.5 h              1.5 h <br /> GCD14 strain (data shown). finding accor-                            Relative level (% dance fact hcLHP1 relatively weak suppressor                              wild type 28&#194;&#176;C) Gcd2 (data shown) Slg2 (Fig. 4) phenotypes                              pre-tRNAMet                100       126      237       320      234       324 <br /> gcd14-1.                                                                                  tRNAMeti                     100        95       48        46       51        46 <br />                                                                                                  Met <br />                                                                                           tRNAe                        100       102      100        82      101       165 <br />    Additive effects gcd10-504 gcd14-2 mutations pro-                            Primary pre-tRNAUAU Ile <br />                                                                                                                        100       172      172       192      167       172 <br /> cessing accumulation tRNAMet     . view similar                        Intron-containing            100        80      113        76      139        94 <br /> phenotypes gcd14 gcd10 mutations, investigated pos-                                pre-tRNAUAU <br />                                                                                                  Ile <br />                                                                                                         Ile <br />  <br />  <br /> sible genetic interactions  gcd10-505                                    tRNAUAU                      100       115      75         79      101        87 <br /> gcd14-2 double mutant Hm397 transformed low-copy- <br />                                                                                          Ratio primary precur- <br /> number plasmids containing GCD10, GCD14, vectors                                    sors mature tRNA generate isogenic transformants following ge-                            tRNAMeti                        0.03     0.04     0.16      0.23    0.15       0.23 <br /> notypes:   gcd10-505 gcd14-2, (ii) GCD10 gcd14-2, (iii) gcd10-                          tRNAUAUIle <br />                                                                                                                           0.05     0.08     0.12      0.13    0.10       0.11 <br /> 505 GCD14, (iv) GCD10 GCD14. shown Fig. 6A,                                                                                                                               relative intensities hybridization signals autoradiograms <br /> gcd10-505 gcd14-2 double mutant grew slowly 28                               shown Fig. 5 quantified phosphorimaging analysis, data 34&#194;&#176;C did single mutant, indicating additivity                         normalized tRNAMet e   signals. amounts tRNA species (pre- <br /> growth defects conferred mutations. Northern analy-                             cursor mature forms) measured strain 28&#194;&#176;C 1.5 h 37&#194;&#176;C <br />                                                                                          expressed relative corresponding values wild-type strain sis transformants grown 37&#194;&#176;C showed gcd14-2                           28&#194;&#176;C, set 100%. ratios pre-tRNAMet      mature tRNAMet    , <br /> single mutant contained high levels tRNAMet    precursors                                               Ile <br />                                                                                          primary pre-tRNAUAU     mature tRNAUAUIle <br />                                                                                                                                                                                                                                                                                                                   listed lower half modest reduction mature tRNAMet       levels relative                      table. <br />  4174       CALVO ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Exacerbation gcd14-2 phenotypes gcd10-505 mutation.   double mutant Hm423 (gcd10-505 gcd14-2) isogenic GCD10 gcd14-2 (Hm420), <br /> gcd10-505 GCD14 (Hm421), GCD10 GCD14 (Hm422) strains streaked single colonies minimally supplemented SD plates incubated 28, 34, 37&#194;&#176;C 2 days. genotypes transformants indicated adjacent appropriate sectors 28&#194;&#176;C plate. (B) Northern blot analysis total RNA (10 <br /> mg) isolated strains analyzed panel  conducted described Fig. 5 cells grown minimally supplemented SD 28&#194;&#176;C (t 5 0) <br /> (lanes 1 4) shifted 37&#194;&#176;C 4 h (lanes 5 8). blot probed 5S rRNA, tRNAMet                                                                                                                         Ile <br />                                                                                                              , tRNAUAU   described Fig. 5. <br />  <br />  <br />  <br /> relative stable RNAs high growth rates                         analyzed Fig. 6B;  did observe additive <br /> occurring elevated temperatures.                                                 reductions levels mature tRNAIle <br />                                                                                                                             UAU combining <br />   Similar results shown gcd14 single mu-                       gcd14-2 gcd10-505 mutations (Table 3). tants (Fig. 5), observed modest (15 35%) reductions                        small increases amounts tRNAIle <br />                                                                                                                                UAU primary tran- <br /> mature tRNAIle UAU expression single double mutants                      script single double mutants 28&#194;&#176;C  <br />  <br />                    TABLE 3. Effects combining gcd10-505 gcd14-2 mutations expression tRNAMet <br />                                                                                                                                                                                                                         Ile <br />                                                                                                           tRNAUAU  <br />  <br />                                                                                                         Value  <br />  <br />            Parameter tRNA                         gcd10 gcd14 strain                gcd14 strain                    gcd10 strain         GCD10 GCD14 strain <br />  <br />                                                      28&#194;&#176;C         37&#194;&#176;C, 4 h        28&#194;&#176;C         37&#194;&#176;C, 4 h            28&#194;&#176;C       37&#194;&#176;C, 4 h     28&#194;&#176;C        37&#194;&#176;C, 4 h <br />  <br /> Relative level (% wild type 28&#194;&#176;C) <br />  Pre-tRNAMet                                     1,031           535            661           1,236            639             930        100            198 <br />  tRNAMeti                                            145            31            101              60             95              45        100             92 <br />                      Ile <br />  Primary pre-tRNAUAU                                 182            82            164             260            195             216        100            160 <br />                                Ile <br />  Intron-containing pre-tRNAUAU                       104            62            103              98            100              71        100             69 <br />         Ile <br />  tRNAUAU                                              76            85             80              77             82              65        100             95 <br />  <br /> Ratio primary precursor mature <br />  tRNA <br />  tRNAMeti                                              0.37           0.88           0.34              1.06           0.35         1.06        0.05           0.11 <br />         Ile <br />  tRNAUAU                                               0.35           0.14           0.30              0.50           0.35         0.50        0.15           0.25 <br />       relative intensities hybridization signals autoradiograms shown Fig. 6B quantified phosphorimaging analysis, data normalized 5S rRNA signals. amounts tRNA species (precursor mature forms) measured strain 28&#194;&#176;C 4 h 37&#194;&#176;C expressed <br /> relative corresponding values wild-type strain 28&#194;&#176;C, set 100%. ratios pre-tRNAMet <br />                                                                                                                       mature tRNAMet <br />                                                                                                                                                                                                                                                                                                       Ile <br />                                                                                                                                           primary pre-tRNAUAU <br />                   Ile mature tRNAUAU      listed lower half table. <br />  VOL. 19, 1999                                                             Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                               4175 <br />  <br />  <br />  <br />  <br />    FIG. 7. GCD14 essential presence hcIMT4.   Ascospore clones spored tetrads (designated 5 7) obtained heterozygous GCD14/gcd14::URA3 diploid (YNG1) containing high-copy-number plasmid p1775 (hcIMT4) (20). strains tetrad (5A 5D 7A 7D), containing hcIMT4, streaked single colonies minimally supplemented SD medium incubated 28 37&#194;&#176;C 3 days. position spore clone GCD14 genotype indicated adjacent appropriate plate sector. 1, wild-type GCD14; D, gcd14::URA3. (B) Northern blot analysis total <br /> RNA (10 mg) isolated spore clones tetrad 5 (strains 5A 5D) bearing hcIMT4 (described panel . strains grown mid-logarithmic phase minimally supplemented SD medium 28&#194;&#176;C (t 5 0 37&#194;&#176;C) shifted 37&#194;&#176;C 4 h. blot probed tRNAMet  , tRNAMet <br />                                                                                                                                     e <br />                                                                                                                                                 Ile <br />                                                                                                                                        , tRNAUAU   described Fig. 5. <br />  <br />  <br />  <br />  <br /> single mutants 37&#194;&#176;C (Fig. 6B), suggesting processing                    tetrad grew similarly 28 37&#194;&#176;C (Fig. 7A, spores precursor occurred slowly mutants.                       7A 7D). second tetrad,  gcd14::URA3 <br /> ingly, primary tRNAIle UAU precursor did accumulate                        hcIMT4 strains grew slowly did GCD14 hcIMT4 double mutant 37&#194;&#176;C, indicating                        strains, particularly 37&#194;&#176;C (Fig. 7A, spores 5A 5D).  susceptible degradation, suggested pre-                       certain genetic backgrounds (e.g., spores 7C tRNAMet .  concluded  gcd10 gcd14                          7D), overexpression tRNAMet      IMT4 makes Gcd14p <br /> mutations considerably smaller effects maturation                      dispensable growth 28 37&#194;&#176;C. pre-tRNAIle UAU versus pre-tRNAi <br />                                         Met <br />                                             , particularly                     conducted Northern blot analysis strains <br /> gcd10-505 gcd14-2 double mutant (Table 3). fact, shown                       tetrads just described, containing hcIMT4.  maturation tRNAMet   appears essential                   shown Fig. 7B, gcd14::URA3 hcIMT4 strains accumu- <br /> function Gcd14p.                                                                lated 3- 10-fold-higher levels pre-tRNAMet      species, <br />                                                                                                                                             GCD14 dispensable yeast strains overexpressing initi-                     derive hcIMT4 (3), did congenic <br /> ator tRNAMet. Having shown GCD14 essential                        GCD14 hcIMT4 strains.  appears processing gcd14 mutations impair maturation pre-tRNAMet        ,                 overproduced precursor impaired strains lacking GCD14. <br /> asked overexpression IMT gene allow <br />                                                                                    mature tRNAMet     reduced ca. half 37&#194;&#176;C cells survive absence Gcd14p. test possi- <br />                                                                                    gcd14::URA3 hcIMT4 strains (spore clones 5D <br /> bility, replaced GCD14 alleles wild- <br />                                                                                    7D) versus GCD14 hcIMT4 strains (Fig. 7B type diploid strain YNG1 gcd14::URA3 deletion  <br /> sertion allele verified gene replacement Southern                       data shown).  certain genetic backgrounds (spore <br /> blot analysis (data shown). sporulation resulting                 clones 5B 7C), overproduction pre-tRNAMet           GCD14/gcd14::URA3 diploid, tetrad analysis revealed ex-                        hcIMT4 completely overcame requirement Gcd14p pected 21:22 segregation cell viability, viable spores                 express high levels mature tRNAMet    .  species mi- Ura2 (bearing GCD14). GCD14/gcd14::URA3                              grating faster mature tRNAMet      visible diploid, homozygous leu2, transformed                       deletion strains tetrad 5 (Fig. 7B) tetrad 7 (data high-copy-number LEU2 plasmid p1775 bearing IMT4                            shown), suggesting fraction mature fully pro- <br /> (20) subjected tetrad analysis, 20 32 tetrads                       cessed tRNAMet   present gcd14::URA3 hcIMT4 strains <br /> showed 41:02 segregation cell viability 21:22 segre-                     improperly processed partially degraded gation Ura phenotype. Importantly, Ura1                         nonfunctional translation. gcd14::URA3 hcIMT4 strains <br /> spores (bearing gcd14::URA3) Leu1 (bearing hcIMT4                          showed little defect expression elongator tRNAMet <br /> p1775) (data shown). results showed p1775                           mature form tRNAIle     UAU (Fig. 7B);  (hcIMT4) overcame lethality gcd14::URA3 deletion.                       modest accumulation primary (slower-migrating) gcd14::URA3 hcIMT4 GCD14 hcIMT4 strains derived                            precursor tRNAIle  UAU, particularly 37&#194;&#176;C, reaching lev- <br />  4176     CALVO ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                        FIG. 8. Synthetic interactions gcd14 lhp1::LEU2 mutations.   <br />                                                                     Strains YNG174 (GCD14), YNG175 (gcd14-1), YNG176 (gcd14-2)                                                                     isogenic lhp1::LEU2-containing derivatives Hm406 (GCD14 lhp1::LEU2), <br />                                                                     Hm407 (gcd14-1 lhp1::LEU2), Hm408 (gcd14-2 lhp1::LEU2) streaked <br />                                                                     single colonies minimally supplemented SD plates incubated 28                                                                     37&#194;&#176;C 2 days. locations strains indicated relevant <br />                                                                     genotypes schematic  (B) Northern blot analysis total RNA <br />                                                                     (20 mg) isolated strains described panel separated elec- <br />                                                                     trophoresis 6% polyacrylamide&#226;&#8364;&#8220;8.3 M urea gel. Strains grown                                                                     minimally supplemented SD 28&#194;&#176;C. blot probed tRNAMet              , <br />                                                                            Ile <br />                                                                     tRNAUAU     , tRNAMet <br />                                                                                         e <br />                                                                                                        Ser <br />                                                                                            , tRNACGA    using appropriate oligonucleotides <br />                                                                      Materials Methods). Indicated right panel                                                                     positions various precursor processed forms tRNAMet  , described                                                                                                                                           Ser <br />                                                                     legend Fig. 5. positions precursor mature tRNACGA        species <br />                                                                     indicated right panel: primary precursor containing 59 <br />                                                                     39 extensions (g), processing intermediate containing 39 extension <br />                                                                     (h), 59- 39-end-processed intron-containing precursor  , mature <br />                                                                            Ser <br />                                                                     tRNACGA      (j). (C) Northern blot panel B probed RPR1 RNA, <br />                                                                     NME1 RNA, 5S rRNA, U6 RNA using appropriate oligonucleotides <br />                                                                      Materials Methods). positions precursor mature RPR1 <br />                                                                     species indicated left. <br /> els ca. 1.5- 4-fold higher seen GCD14 hcIMT4 <br /> strains (Fig. 7B). <br />    Deletion LHP1 exacerbates phenotypes gcd14 mu- <br /> tations. investigate role Lhp1p processing          tions 77, 86, 68% GCD14, gcd14-1, gcd14-2 <br /> tRNAMeti    precursors gcd14 mutants, disrupted LHP1          strains, respectively (Fig. 8B; Table 4). Lhp1p impli- LEU2 gene wild-type GCD14 gcd14 mutant              cated stabilizing precursors promoting correct 39 end <br /> strains  Materials Methods). lhp1::LEU2 mutation        processing certain tRNAs (61).  account did affect growth rate wild-type strain H117,     reduced amounts mature tRNAMet       seen double mu- <br /> agreement previous results (60);  greatly exac-     tants, suggest deletion LHP1 leads degradation erbated Slg2 phenotypes gcd14-1 gcd14-2 mu-          substantial fraction tRNAMeti    precursors tants (Fig. 8A). addition, observed additive effects   degradation extensive gcd14 mutants lhp1::LEU2 gcd14 mutations levels mature              GCD14 strain. <br /> tRNAMeti   . lhp1::LEU2 allele reduced mature              contrast findings tRNAMet <br />                                                                                                               , relatively <br /> tRNAMeti    ca. 27% GCD14 cells, caused reductions    small effects deletion LHP1 levels precursor ca. 60% gcd14-1 gcd14-2 cells (Fig. 8B; Table 4).     mature forms tRNAIle   UAU strains, levels tRNAMeti    precursors greatly diminished    little indication lhp1::LEU2 gcd14 mutations lhp1::LEU2 mutation backgrounds, reduc-       additive effects expression fully processed <br />  VOL. 19, 1999                                                               Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                                  4177 <br />  <br />  <br />          TABLE 4. Effects combining gcd14 lhp1::LEU2 mutations expression tRNAMet <br />                                                                                              , tRNAMet <br />                                                                                                      e <br />                                                                                                                Ile <br />                                                                                                          , tRNAUAU , tRNASer <br />                                                                                                                          CGA, <br />                                                    RPR1, NME1, U6 RNAsa <br />                                                                                                          Value  <br />               Parameter tRNA                        GCD14        GCD14 lhp1::LEU2         gcd14-1      gcd14-1 lhp1::LEU2        gcd14-2      gcd14-2 lhp1::LEU2 <br />                                                          strain           strain               strain             strain              strain             strain <br />  <br /> Relative level (% wild type 28&#194;&#176;C) <br />  Pre-tRNAMet                                          100                23                 126                 18                 236                 74 <br />  tRNAMeti                                               100                73                  37                 15                  52                 20 <br />         Met <br />  tRNAe                                                  100                96                 112                114                 120                 92 <br />                      Ile <br />  Primary pre-tRNAUAU                                    100                62                 118                 79                 116                 79 <br />                                Ile <br />  Intron-containing pre-tRNAUAU                          100                96                  58                 61                  91                 71 <br />         Ile <br />  tRNAUAU                                                100               104                  65                 66                  97                 59 <br />  Primary pre-tRNASer CGA                                100                38                  57                 39                  60                 26 <br />  Intermediate precursor                                 100                 &#226;&#8364;&#8221;b                 59                  &#226;&#8364;&#8221;                  56                  &#226;&#8364;&#8221; <br />  Intron-containing pre-tRNASer CGA                      100                61                  49                 24                  30                 33 <br />  tRNASerCGA                                             100                94                  94                 32                  75                 50 <br />  Pre-RPR1                                               100                51                  49                 15                  82                 46 <br />  RPR1                                                   100                73                  68                 24                  60                 52 <br />  NME1                                                   100                61                 100                 28                  98                 28 <br />  U6                                                     100                96                  85                 73                  80                 55 <br />  <br /> Ratio primary precursor mature RNA <br />  tRNAMeti                                                 0.08               0.025               0.30                 0.08             0.35               0.31 <br />         Ile <br />  tRNAUAU                                                  0.04               0.03                0.08                 0.05             0.05               0.06 <br />  tRNASerCGA                                               0.28               0.11                0.17                 0.34             0.22               0.15 <br />  RPR1                                                     0.51               0.36                0.36                 0.33             0.70               0.45 <br />         relative intensities hybridization signals autoradiograms shown Fig. 8B C quantified phosphorimaging analysis, data normalized 5S rRNA signals. amounts tRNA RNA species (precursors mature forms) measured strain 28&#194;&#176;C expressed relative corresponding values wild-type strain 28&#194;&#176;C, set 100%. ratios pre-tRNAMet <br />                                                                                                              mature tRNAMet <br />                                                                                                                                                                                                                                                                                        Ile <br />                                                                                                                                    , primary pre-tRNAUAU  mature <br />        Ile                          Ser                     Ser <br /> tRNAUAU    , primary pre-tRNACGA      mature tRNACGA      , pre-RPR1 mature RPR1 listed lower half table. <br />    b <br />      &#226;&#8364;&#8221;, undetectable. <br />  <br />  <br />  <br />  <br /> form tRNA (Fig. 8B; Table 4).  similar                           effect especially marked gcd14-1 background. findings gcd10 gcd14 double mutants (Fig. 6), expression                         suggested tRNAMeti  , Lhp1p increase stabil- tRNAIle UAU relatively insensitive mutations                         ity pre-RPR1 RNA, function critically <br /> showed strong additive effects production mature                           required production mature RPR1 gcd14-1 mutants. <br /> tRNAMeti  . defects whatsoever expression elongator                         Finally, U6 RNA showed modest reductions gcd14 <br /> tRNAMet single double mutants relative                      lhp1::LEU2 double mutants compared corre- <br /> wild type observed (Fig. 8B; Table 4).  ob-                          sponding single mutants, similar magnitude  <br /> served substantial reduction (70%) level mature                          scribed tRNAIle UAU (Fig. 8B; Table 4). Taken  <br /> tRNASerCGA gcd14-1 lhp1::LEU2 double mutant compared                          gether, results Fig. 8B C strongly suggest corresponding gcd14-1 single mutant additive                           Gcd14p Lhp1p cooperate maturation accumulation <br /> effect lesser degree tRNA gcd14-2                               subset RNA polymerase III transcripts. case lhp1::LEU2 double mutant (Fig. 8B; Table 4). Unlike sit-                         tRNAMeti  , requirement Gcd14p readily observed <br /> uation tRNAMet    , amounts tRNASer   CGA precursors                    mutants defective protein, decreased, increased, gcd14 single                           tRNASerCGA, NME1, RPR1, strong dependence Gcd14p <br /> mutations;  tRNAs, deletion LHP1 greatly                         revealed absence Lhp1p. <br /> reduced amounts precursors GCD14 gcd14                               showed previously Gcd14p copurified poly- <br /> backgrounds.  Gcd14p Lhp1p function                             histidine-tagged form Gcd10p nickel affinity resin stabilizing tRNASer  CGA precursors. unknown                       protein stably associated PRT1  <br /> tRNASer                   1 <br />        CGA contains m position 58 (57).                                         coded subunit eIF3 (3).  Gcd10p Gcd14p <br />    considered possibility expression                       detected highly purified eIF3 complex (48), RNA polymerase III transcripts reduced gcd14                         proteins showed prominent nuclear localization (3).  <br /> lhp1::LEU2 double mutants. shown Fig. 8C, levels                        appears Gcd10p Gcd14p reside nuclear protein <br /> 5S rRNA nearly indistinguishable gcd14                            complex distinct eIF3. view genetic interactions <br /> lhp1::LEU2 single double mutants. contrast, ob-                            gcd14 mutations lhp1::LEU2 allele, served strong additive effects gcd14-1 lhp1::LEU2                          determine Lhp1p tightly associated expression RPR1 NME1 transcripts.                          Gcd10p-Gcd14p complex. answer question, RNA components RNase P RNase MRP,                               investigated Lhp1p coimmunoprecipitated <br /> involved processing tRNAs rRNA, respectively                           cell extracts HA-tagged form Gcd10p. (38, 54). observed similar interaction gcd14-2                          shown previously GCD10-HA allele employed lhp1::LEU2 NME1 RNA, lesser effect RPR1                            study wild-type GCD10 indistinguishable abil- <br /> RNA, double mutant versus gcd14-1 lhp1::LEU2                             ity complement lethal phenotype gcd10::URA3 mu- <br /> strain (Table 4). interesting deletion LHP1 reduced                    tation (3). cell extracts prepared isogenic RPR1 precursor strains,                          GCD10-HA GCD10 strains containing high-copy- <br />  4178       CALVO ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 9. Gcd14p, Lhp1p, tightly associated Gcd10p cell extracts.   cell extracts prepared described previously (47) strains <br /> H1515 (LHP1) YJA113 (lhp1) transformants YJA142 (GCD10HA) YJA143 (GCD10) bearing high-copy-number plasmid p2626 containing LHP1 vector YEp24. Aliquots containing 200 mg total cell protein mixed 2 4 mg anti-HA monoclonal antibody HA.11 (Babco), prebound protein Sepharose 2 h ice, mixture incubated 1 h 4&#194;&#176;C. washes lysis buffer (20 mM Tris-HCl [pH 7.4], 100 mM KCl, 1.0 <br /> mM Mg acetate, 0.1% [vol/vol] Triton X-100) containing complete protease inhibitor tablet (Boehringer Mannheim) 25 ml, immunoprecipitates collected centrifugation proteins eluted boiling Laemmli buffer (37). total proteins immunoprecipitated 200 mg (pellet [P]; lanes 4, 6, 8) 50 <br /> mg starting cell extracts (input  ; lanes 3, 5, 7) separated SDS-polyacrylamide gel electrophoresis (36) transferred nitrocellulose <br /> membrane (Millipore) 25 mM Tris&#226;&#8364;&#8220;192 mM <span id='am-46' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-47' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span>&#226;&#8364;&#8220;0.1% SDS containing 20% (vol/vol) methanol. Lanes 1 2 contain 50 mg starting cell extracts strains H1515 YJA113, respectively, included establish identity Lhp1p. membrane blocked overnight 4&#194;&#176;C BLOTTO (5% <br /> [wt/vol] nonfat dry milk, 10 mM Tris-HCl [pH 7.4], 150 mM NaCl, 0.05% [vol/vol] Tween 20). single immunoblot probed 3 mg rabbit anti-HA antibody <br /> HA.11 (Babco) ml, stripped, reprobed 1:1,000 dilution anti-Gcd14p antibodies  Materials Methods), reprobed 1:1,000 dilution anti-Lhp1p antibodies (60). Immune complexes detected horseradish peroxidase-conjugated sheep antimouse (Amersham) donkey antirabbit (Amersham) <br /> secondary antibodies enhanced chemiluminescence kit (ECL; Amersham). (B) longer exposure lanes 5 6 blot panel probed anti-Lhp1p <br /> antibodies, included detect small amounts Lhp1p immunoprecipitates. <br />  <br />  <br />  <br /> number plasmid bearing LHP1 vector                               phenotypes nonlethal gcd14 mutations partially <br /> immunoprecipitated monoclonal anti-HA antibodies.                             suppressed hcLHP1, increased level mature <br /> expected, comparable proportions Gcd14p HA-Gcd10p                              tRNAMeti    just wild-type levels.   immunoprecipitated anti-HA antibodies                              ings indicate essential function Gcd14p required <br /> extracts prepared strains containing GCD10-HA, nei-                          efficient accurate production mature tRNAMet    . <br /> ther protein immunoprecipitated GCD10 extract                               fact gcd14 mutants exhibit tRNAMet         precursor <br />                                                                                                                                          (Fig. 9A, lanes 4, 6, 8). detectable Lhp1p coim-                           tRNA/mature tRNA ratios 5- 10-fold higher wild type <br /> munoprecipitated HA-Gcd10p Gcd14p,                                 (Fig. 5; Table 2) easily explained defect pre- <br /> Lhp1p greatly overexpressed (Fig. 9A, lanes 4 6,                          tRNAMet     processing. precursor molecules accumu- <br />                                                                                              B, lane 6). accordance finding, Lhp1p                                                                                       late gcd14 mutants contain 59 39 extensions, indi- <br /> detectable preparation Gcd10p-Gcd14p complex <br />                                                                                       cating cleavage 59 extension RNase P (18) <br /> purified homogeneity (48a). conclude Lhp1p                                                                                       removal 39 extension endo- exonucleases occur <br /> tightly associated Gcd10p-Gcd14p complex cell <br /> extracts.                                                                             efficiently gcd14 mutants. accumulation mole- <br />                                                                                       cules shorter wild-type mature tRNAMet   gcd14 mutants <br />                                                                                       indicate portion 59- 39-processed <br />                               DISCUSSION                                              tRNAMet     lacks CCA triplet added posttranscriptionally                                                                                                 Evidence Gcd14p required efficient processing                       39 ends tRNAs (18). Alternatively, shorter <br /> tRNAMet <br />        . Recessive gcd14 mutations confer constitutive dere-                        molecules degradation intermediates. <br /> pression GCN4 mRNA translation absence eIF2a                                Evidence Gcd14p Gcd10p function pro- <br /> phosphorylation (17), phenotype generally indicating                             mote maturation tRNAMet     vivo. Nonlethal gcd10 muta- <br /> duced formation eIF2-GTP-tRNAMet        ternary complex                      tions lead reduced expression mature tRNAMet     (20). gcd14 mutants Slg2 phenotype,                          suppressed hcIMT hcLHP1 (3), just shown general translation initiation slightly impaired                     gcd14 mutations (Fig. 4).  lethal effect mutants, elevated temperatures (Fig. 3). addition,                     deleting GCD10 suppressed overexpression deletion GCD14 lethal. findings suggest                              tRNAMet     high-copy-number IMT4 (3).  essen- <br />                                                                                              Gcd14p essential function required ternary complex <br />                                                                                       tial functions proteins, low growth temper- <br /> formation vivo. agreement conclusion,                                                                                       atures, required expression mature tRNAMet     . gcd14 mutants contain approximately 50% lower steady-                                                                                        <br /> state levels mature tRNAMet     present isogenic                       addition genetic similarities, Gcd10p <br />                                wild-type strains (Tables 2 4). contain processed                        Gcd14p coimmunoprecipitated (Fig. 9) copu- <br /> tRNAMet    molecules appear smaller wild-type                         rified (3) cell extracts. physical association <br />        mature tRNAMet      (Fig. 5 7, species f)                          proteins confirmed using                 defective aminoacylation ternary complex formation.                         polyhistidine-tagged form Gcd10p isolate Gcd10p- <br /> Gcd2 Slg2 phenotypes gcd14 mutants fully                              Gcd14p complex cell extracts affinity chroma- <br /> suppressed introducing multiple copies IMT genes                            tography (3) yeast hybrid analysis (4). addition, <br /> encoding tRNAMet    , restored expression mature                           epitope-tagged forms proteins Gcd10p Gcd14p <br /> tRNAMeti   greater wild-type levels.  pres-                      shown immunofluorescence exhibit prominent nu- <br /> ence hcIMT4 overcame lethality gcd14 deletion.                        clear localization yeast cells (3). findings strongly <br />  VOL. 19, 1999                                                         Gcd14p, Gcd10p, Lhp1p, tRNA PROCESSING YEAST                    4179 <br />  <br />  <br /> suggest Gcd10p Gcd14p reside nuclear complex                   Gcd10p copurified eIF3 activity (23), stabilizing functions directly maturation tRNAMet         .             ternary complex function ascribed eIF3 (47). <br />    provided additional vivo evidence                    Evidence Lhp1p required protect tRNAMet         clusion showing gcd14-2 gcd10-505 mutations                  degradation gcd14 mutants. wild-type cells, <br /> additive effects cell growth expression mature                     deletion LHP1 led substantially decreased levels pre- <br /> tRNAMeti  . double mutant 28&#194;&#176;C accumulated higher                     cursor tRNAMet     small reduction mature <br /> levels pre-tRNAMet   did single mutant, suggest-            tRNAMeti    levels (Fig. 8B; Table 4). explained ing compounded defects processing.  gcd10-                   postulating role Lhp1p transcription IMT genes. <br /> 505 single mutant accumulated tRNAMet           precursors concom-          view allele-specific interactions itant diminished amounts mature tRNAMet             (Fig. 6B),     gcd14 mutations lhp1::LEU2 deletion (Fig. 8B), plus <br /> implicating Gcd10p pre-tRNAMet        processing. previous         evidence Gcd14p functions methylation pro- <br /> study showed expression mature tRNAMet                   cessing pre-tRNAMet    , favor idea Lhp1p affects <br /> duced ca. fivefold hours 37&#194;&#176;C gcd10-504             tRNAMeti     expression posttranscriptionally. suggested mutant;  observed accumulation pre-                         yeast tRNAs (61), binding Lhp1p poly(U) <br /> tRNAMeti   restrictive temperature strain (3).           stretch 39 end pre-tRNAMet       stabilize  explained proposing gcd10-504 gcd10-                formation needed precise endonucleolytic 39 cleavage 505 (studied  mutations lead defects processing              block access 39-59 exonucleases. absence Lhp1p, pre-tRNAMet      unprocessed molecules              significant fraction pre-tRNAMet      incorrectly <br />               susceptible degradation gcd10-504 mutant               trimmed 39 end subsequently degraded, reducing <br /> gcd10-505 gcd14 strains analyzed  accordance                 yield mature tRNAMet       . explanation, obtained evidence precursor                  Deletion LHP1 gcd14 mutants led larger reduc- <br /> mature tRNAMet       highly susceptible degradation           tions precursor mature tRNAMet         GCD14 <br />                 gcd10-505 gcd14-1 double mutant 37&#194;&#176;C (Fig. 6B), similar                cells (Fig. 8B), suggesting greater need putative chap- previous findings gcd10-504 single mutant (3).                   erone function Lhp1p gcd14 versus GCD14 cells.    far, observed defect pre-tRNAMet                syn-   shown previously mutation yeast tRNASer      CGA makes                                                                  thesis processing cell extracts prepared gcd10                 processing completely dependent Lhp1p, process- <br /> gcd14 mutants (3a).  Gcd10p Gcd14p                      ing intermediates appear rapidly degraded Lhp1p <br /> appear essential components processing machin-                  depleted cells (61). proposed Lhp1p ery. discovery Gcd14p contains sequence motifs                  essential stabilize conformation mutant tRNASer    CGA <br />                                                                              permit correct endonucleolytic cleavage removal 39 <br /> conserved S-AdoMet-dependent methyltransferases, <br />                                                                              trailer. Similarly, absence m1A position 58 including tRNA methyltransferase (35), raised                                                                              perturb structure initiator tRNAMet gcd14 mu- <br /> possibility nucleotides pre-tRNAMet                                                                                                                                                                  tants create stronger requirement binding Lhp1p <br /> undermethylated gcd14 mutants. alter  <br />                                                                              39 trailer achieve precise endonucleolytic 39 end <br /> formation pre-tRNAMet       way impairs removal                                                                            processing. Lhp1p overexpressed gcd14-1 cells, 59 39 extensions addition CCA triplet 39 end. <br />                                                                              led increased steady-state levels precursor ma- <br /> Consistent possibility, recently m1A              ture length tRNAMet         intermediate precursor <br />                                                                                                        lacking total tRNA isolated gcd10D cells (3). base              transcribed IMT2 IMT3 37&#194;&#176;C containing 39 <br /> modification occurs position 58 initiator elongator            extension (data shown). increased binding forms tRNAMet, tRNATyr     GTA  examined ,            39 trailers precursors, overproduced Lhp1p 15 tRNAs positions yeast              promote correct endonucleolytic cleavage, suggested  <br /> tRNAs (57). initiator, m1A58 residue involved               accounting ability hcLHP1 increase expression unique tertiary substructure observed elongator                 mature tRNAMet       gcd14 mutants elevated temperatures. <br />                                                                                               tRNAs, involving residues unique eukaryotic initia-                 type functional interaction Gcd14p Lhp1p <br /> tors, A60, A54, A20 (5).  lack methylation                maturation tRNAMet          consistent lack                                                                                                                position 58 perturb structure initiator tRNAMet,                strong physical interaction proteins (Fig. 9). impair processing stability, similarly affect-              Combining deletion LHP1 gcd14-1 mutation <br /> ing elongator tRNAMet tRNAs bearing m1A58.                    led unexpectedly large reductions expression unknown tRNAIle                       Ser <br />                             UAU tRNACGA, processing                NME1 RNA, RPR1 RNA, tRNASer                 CGA, addition accumulation slightly impaired gcd14 mutants,                    tRNAMeti   . gcd14-2 lhp1::LEU2 mutant contained sub- <br /> contain m1A58.                                                               stantially reduced amounts NME1 RNA. double mu- <br />    remains determined Gcd14p-Gcd10p                     tants exhibited small reductions levels tRNAIle   UAU U6 <br /> complex directly responsible m1A58 formation yeast                 RNA compared corresponding single mutants tRNA Gcd14p methylase activity, pre-                      wild-type levels elongator tRNAMet 5S rRNA (Fig. 8B). <br /> dicted presence S-AdoMet binding motifs.                      appears Gcd14p Lhp1p cooperate sidering Gcd10p RNA binding activity (23),                 maturation subset RNA polymerase III transcripts. chaperone function facilitate methylation               transcripts analyzed far, tRNAMet     shows strong <br /> pre-tRNAMeti     Gcd14p wild-type cells. results Fig.           requirement Gcd14p cells containing Lhp1p.  <br /> 6B indicate gcd10-505 mutation leads deg-                moval Lhp1p reveals additional strong requirements radation unprocessed precursors accumulate                    Gcd14p production mature RPR1 RNA, NME1 <br /> GCD10 gcd14-2 cells.  Gcd10p required                    RNA, tRNASer    CGA lesser requirement U6 expres- <br /> protect hypomethylated pre-tRNAMet            exonucleolytic          sion. Similarly tRNAs, RPR1 RNA transcribed degradation. Gcd10p performs chaperone function,                     369-nucleotide precursor RNA 84 nucleotides 59 leader conceivably interact mature tRNAMet          cyto-       30 nucleotides 39 trailing sequences (38). plasm promote formation ternary complex                      intriguing possibility Gcd14p required methylation <br /> eIF2 GTP. hypothetical function explain                   pre-RPR1 RNA hypomethylated RPR1 RNA <br />  <br />  <br />  <br /> present gcd14-2 mutants highly susceptible                        <br /> degradation cells lacking Lhp1p, postulated                           <br />                                                                                  <br /> tRNAMeti   . remains determined RPR1 RNA                          <br /> methylated. Obviously, rule possibility                          <br /> combining gcd14 lhp1::LEU2 mutations                             <br /> strains leads unforeseen indirect effect transcription                         <br />                                                                                        maturation NME1 RNA, RPR1 RNA, tRNASer            CGA.                      <br />    fact gcd14 mutations lead 30 40%                                 <br /> reductions levels mature RPR1 RNA (Fig. 8B; Table                          <br /> 4) raises possibility reduction RNase P activity                        <br />                                                                                      <br /> contributes defect processing tRNAMet      gcd14                       <br /> single mutants. absence m1A58, atten-                           <br /> dant defects tertiary structure, renders tRNAMet   sus-                        <br /> ceptible tRNAs reductions RNase P activity.                             <br />                                                                                     token, proposed dramatic                                <br /> reductions RPR1 RNA levels observed gcd14-1                                    <br /> lhp1::LEU2 double mutant largely account addi-                         <br /> tive effects mutations reducing mature tRNAMet                                                                                             <br />                                                                                     <br /> levels.  shown previously depletion                          <br /> Rpp1p, protein subunit RNase P (12, 58), mutation                            POP1-encoded subunit RNase P (41) leads accumu-                      <br /> lation unprocessed tRNA precursors concomitant                                <br />                                                                                           <br /> duced amounts mature tRNAs. fact pre-                           <br /> cursors mature tRNAMet       greatly diminished                      <br /> gcd14-1 lhp1::LEU2 double mutant (Fig. 8B) suggests                        <br /> unprocessed tRNA precursors, accumulate                            <br />                                                                                         <br /> response reduced RNase P levels, rapidly degraded                       absence Lhp1p wild-type Gcd14p.                                     <br />    interesting identify complete role                          <br /> Gcd10p-Gcd14p complex modification processing                                 <br />                                                                                   <br /> different RNA polymerase III transcripts investigate                             functional molecular interactions complex                        Lhp1p. recently deletion CPD1,                              <br /> GCD14 homologue fission yeast, lethal leads                           <br /> accumulation tRNAMet         precursors (11a).  appears                       <br />                                                                                    function Gcd14p tRNAMet      maturation evolu-                       <br /> tionarily conserved.                                                                     <br />   <br /> </body></html>