<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A conserved role GATA transcription factor regulating epithelial innate immune responses <br /> Michael Shapira*, Brigham J. Hamlin*, Jiming Rong*, Karen Chen*, Michal Ronen*, Man-Wah Tan*&#226;&#8364;&#160;&#226;&#8364;&#161; <br /> Departments *Genetics &#226;&#8364;&#160;Microbiology Immunology, Stanford University School Medicine, Stanford, CA 94305 <br />  <br /> Edited Frederick M. Ausubel, Harvard Medical School, Boston, MA, approved July 10, 2006 (received review April 26, 2006) <br />  <br /> Innate immunity ancient conserved defense mechanism.                 adulthood (17). GATA transcription factors ex- host responses various pathogens                     pressed adulthood mature tissues, animals delineated, responses orchestrated                     little known postdevelopmental roles. <br /> animal understood. unbiased genome-wide                        C. elegans human cells, study performed Caenorhabditis elegans, identified                 endodermal GATA transcription factor postdevelopmental <br /> served function endodermal GATA transcription factors                   role local defense human Gram-negative bacterial <br /> regulating local epithelial innate immune responses. Gene expres-              pathogen Pseudomonas aeruginosa. Using genome gene ex- <br /> sion functional RNAi-based analyses identified tissue-                 pression functional analyses C. elegans, identified specific GATA transcription factor ELT-2 major regulator            GATA transcription factor ELT-2 major regulator early <br /> early intestinal protective response infection human               protective response P. aeruginosa. human <br /> bacterial pathogen Pseudomonas aeruginosa. adult worm,                  homolog, GATA6, protects lung epithelial cells similar <br /> ELT-2 required specifically infection responses survival         infection, describing conserved role GATA transcription <br /> pathogen makes significant contribution gene expression              factors mediating local epithelial responses infection. <br /> associated intestinal maintenance resistance cad- <br /> mium, heat, oxidative stress. demonstrate             Results <br /> function conserved, human endodermal transcrip-                 Conserved C. elegans-Specific Gene Expression Responses P. <br /> tion factor GATA6 protective function lung epithelial cells           aeruginosa Infection. better understand host responses infec- <br /> exposed P. aeruginosa. findings expand repertoire              tion context organism, characterized gene <br /> innate immunity mechanisms illuminate unknown func-                  expression responses young adult wild-type C. elegans infection <br /> tion endodermal GATA proteins.                                              PA14, clinical isolate P. aeruginosa (18). Using GFP- <br />                                                                                expressing PA14 derivative (PA14-GFP), followed bacterial <br /> gene expression &#229;&#8230;&#169; innate immunity &#229;&#8230;&#169; infection &#229;&#8230;&#169; ELT-2 &#229;&#8230;&#169; Pseudomonas            accumulation worm intestine (Fig. 1A). Time points chosen <br /> aeruginosa                                                                     gene expression analyses 4, 12, 24 h exposure,                                                                                bacterial colonization progressed gradually primarily <br />  <br /> T    innate immune evolutionarily conserved  <br />      fense mechanism, pathogen-associated molecular pat- <br /> terns (PAMPs) infection products, recognized recep- <br />                                                                                pharyngeal localization (Fig. 1 data shown)                                                                                densely filling intestinal lumen, concomitant distension                                                                                intestinal lumen (Fig. 1 . time points precede <br /> tors leading regulated expression immune modulators                  deaths (18). times, infection restricted antimicrobial molecules (1&#226;&#8364;&#8220;3). central mechanism stud-               alimentary tract. compared mRNA abundance &#226;&#172;&#381;90% C. <br /> ied organisms involves axis formed PAMP-recognition                     elegans genes, using spotted PCR microarrays, animals <br /> Toll-like receptors NF-&#226;?&#172;B transcription factors (4, 5).            exposed PA14 nonpathogenic Escherichia coli <br /> Caenorhabditis elegans lacks Toll-NF-&#226;?&#172;B axis (6) effec-            strain OP50. <br /> tively protects itself diverse pathogens, underscoring                   identified 232 genes expression changed response <br /> importance mechanisms innate immunity. C. elegans              infection, using procedure based way ANOVA. shown protect itself localized bacterial infections, rely         197 induced 35 repressed (Fig. 1B Table 2, virulence factors necessary infection vertebrates           published supporting information PNAS web <br /> (7), function integration conserved signal          site). Quantitative RT-PCR (qRT-PCR) measurements confirmed <br /> transduction pathways. include Sma&#229;&#8230;&#190;TGF-&#226;?&#164;, insulin,               microarray results 49 52 responding genes tested (Table 3, <br /> p38 mitogen-activated protein kinase pathways (8&#226;&#8364;&#8220;11).               published supporting information PNAS web <br /> <span id='am-6' about='oboInOwl:date' typeof='owl:Thing'>date</span>, transcription factor directly shown mediate           site). examined tissue distribution responding genes <br /> function pathways regulating transcriptional                using lists muscle-, germline-, intestine- enriched transcripts <br /> responses infection.                                                        (19&#226;&#8364;&#8220;22). Fifty-seven percent gut-enriched, 8% <br />    GATA zinc-finger transcription factors play critical roles development differentiation vertebrates inverte- <br /> brates. family members exist vertebrates, GATA1-3              Author contributions: M.S. M.-W.T. designed research; M.S., B.J.H., J.R., K.C.,                                                                                M.-W.T. performed research; M.S., B.J.H., K.C., M.R., M.-W.T. contributed new  crucial hematopoiesis (12), GATA4-6, mesoen- <br />                                                                                agents&#229;&#8230;&#190;analytic tools; M.S., K.C., M.R., M.-W.T. analyzed data; M.S. M.-W.T. <br /> dodermal development (13). additional role identified                wrote paper. <br /> Drosophila larvae, early meso- endodermal develop-                authors declare conflict  <br /> ment GATA factor, Serpent (14), regulates systemic induction                                                                                paper submitted directly (Track II) PNAS office. <br /> antimicrobial peptides response systemic infection (15). Sim- <br />                                                                                Abbreviations: DUF, domain uncharacterized function; Esp, enhanced susceptibility ilar vertebrates Drosophila, C. elegans GATA proteins               pathogens; qRT-PCR, quantitative RT-PCR; CUB, complement subcomponents C1s&#229;&#8230;&#190;C1r, <br /> essential endodermal development (12). Acting sequentially              Uegf, Bmp1. E cell lineage regulatory cascade, GATA transcription                 Data deposition: microarray data deposited Stanford Microarray <br /> factors direct formation worm intestine (16).           Database (http:&#229;&#8230;&#190;&#229;&#8230;&#190;smd.stanford.edu) Gene Expression Omnibus database (acces- cascade elt-2 -7, restricted expression         sion  GSE5584). intestine responsible terminal differentiation              correspondence addressed. E-mail: mwtan@stanford.edu. <br /> intestinal cells. pair, elt-2 continues expressed         &#194;&#169; 2006 National Academy Sciences USA <br />  <br />  <br />  <br /> 14086 &#226;&#8364;&#8220;14091 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; September 19, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169;  38                                                      www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0603424103 <br />                                                                                      packaging, requires equal ratios core histone <br />                                                                                     subunits. possible  context infection, induction                                                                                     specific isoforms serves provide antimicrobial <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> <br />                                                                                     precursors, reported various organisms (24, 25).                                                                                     repressed genes, group, annotated Gene Ontology <br />                                                                                     annotation &#226;&#8364;&#732;&#226;&#8364;&#732;<span id='am-7' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> metabolism,&#226;&#8364;&#8482;&#226;&#8364;&#8482; enriched (Tables 1 2). <br />                                                                                        individually assessed functional significance 190                                                                                     232 responding genes knocking expression adult <br />                                                                                     animals measuring sensitivity PA14,                                                                                     lifespan OP50 (Materials Methods Supporting Text, <br />                                                                                     published supporting information PNAS web <br />                                                                                     site). approach allowed determine gene contribution                                                                                     immunity independently possible functions important devel- <br />                                                                                     opment normal lifespan. Knocking expression 21 <br />                                                                                     genes resulted enhanced susceptibility pathogen (Esp), Fig. 1. P. aeruginosa accumulation worm gut accompanied                                                                                     did affect lifespan, suggesting immunity genes  esp <br /> robust gene expression changes host.   Representative images (&#226;&#171;&#187;400)        genes; Table 4, published supporting information worms (10 &#226;&#8364;&#8220;20 group) exposed PA14-GFP 4, 12, 24 h. Asterisks      PNAS web site). comparison, similar screen targeted mark posterior pharyngeal bulb; wedges mark intestinal lumen                majority (2,251) chromosome genes RNAi egg <br /> boundaries. Yellow signal autofluorescence intestinal granules; addi-      stage yielded esp genes (B.J.H. M.-W.T., unpublished <br /> tion, yellow pharyngeal signal unknown source appeared consistently         results), indicating esp genes highly enriched 12-h time point. (B) Genes differentially expressed P. aeruginosa     genes expression modulated infection. esp <br /> infection. Hierarchically clustered expression profiles (rows) 248 PCR prod-    genes included members multigene families, CUB- <br /> ucts, corresponding 232 genes, differentially expressed                                                                                     containing genes F08G5.6 F20G2.5. included exposure PA14 compared OP50. Data independent exper- <br /> iments (columns) shown time point, separated dotted lines. <br />                                                                                     histone genes 10 -16, supporting notion histone <br /> Vertical bars mark clusters genes repressed   induced  .           gene induction provide immune-specific function, presum- <br />                                                                                     ably precursors antimicrobial peptides. Interestingly,                                                                                     esp genes repressed infection, including putative fatty genes enriched body muscle, tissue relative mass                   acid synthase gene, thn-1, encodes homolog comparable intestine. fraction transcripts                    thaumatin family plant antifungal proteins. Overall, func- <br /> enriched tissues negligible. preferential response                 tional analyses demonstrate identified transcriptional  intestinal genes infection suggests local infection                  sponse represents functional immune response. <br /> caused P. aeruginosa elicits response local. <br />    major functional domains observed response                        Infection Responses Distinct General Stress Responses. summarized Table 1. Induced genes showed significant enrich-                   possible gene expression changes observed ment members conserved gene families: lysozymes,                     infection indirect effect associated cellular damage lectins, histones, genes encoding proteins GST domain.                   represented general stress response. examine similar similar enrichment DUF274 genes, representing                     infection response general stress responses, compared <br /> family unique Caenorhabditis (DUF stands domain                        list infection-response genes &#226;&#172;&#8225;1,000 genes previously <br /> known function) members gene family encode                        affected cadmium, wide-range toxic metal (26). proteins CUB-like domain  known DUF141),                       overlap 82 genes, including GST, DUF274, variant domain diverse proteins, including C1s                CUB-containing genes (Table 2).  greater C1r complement peptidases (CUB stands Complement                         infection response distinct caused cadmium subcomponents C1r&#229;&#8230;&#190;C1s, Uegf, Bmp1; ref. 23). Induction                       differed functional composition. Notably, lysozyme, lectin, CUB-containing genes describes expansion                   heat-shock protein induced infection, compared 14 previous report studying responses Gram-negative                    strongly induced cadmium, histone gene induction, bacteria (9). induction histone genes preferentially                  prominent feature infection response, altogether missing <br /> histone H2A H2B, suggesting functions DNA                            cadmium response. grossly overlapping portion  <br />  <br />                         Table 1. Major functional groups represented infection response <br />                                                                                       infection response <br />                                                              analyzed dataset         list  197 induced,          P value                         Functional groups&#226;&#8364;&#160;          genome          7,308 genes)           35 repressed genes)&#226;&#8364;&#161;           enrichment&#194;&#167; <br />  <br />                         Lysozymes                      10                   9                            2                       0.02 <br />                         Lectins                       285                  64                           11                     8 &#226;&#171;&#187; 10&#226;&#171;&#186;7 <br />                         CUB-like                       52                  29                           17                      &#226;&#172;?10&#226;&#171;&#186;8 <br />                         DUF274                         19                  10                            9                      &#226;&#172;?10&#226;&#171;&#186;8 <br />                         Histones                       76                  48                            7                     2 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br />                         GST                            70                  38                           11                      &#226;&#172;?10&#226;&#171;&#186;8 <br />                         Lipid metabolism              214                 119                            3                       0.016 <br />                                                                                                                                                                IMMUNOLOGY <br />  <br />  <br />  <br />  <br />                         &#226;&#8364;&#160;Shown   major functional groups. Genes lacking shared and&#229;&#8230;&#190;or clear functional affiliation                          considered  members group retrieved WormBase using gene names  e., *, <br />                          lys-*) protein motif match. Lectins include proteins C-type lectin, selectin, galectin domains. <br />                         &#226;&#8364;&#161;Number genes induced  categories) repressed  lipid metabolism category); specific <br />  <br />                          genes, Table 2. <br />                         &#194;&#167;P values enrichment calculated based hypergeometric distribution. <br />  <br />  <br />  <br />  <br /> Shapira et al.                                                                                       PNAS &#229;&#8230;&#169; September 19, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169;  38 &#229;&#8230;&#169; 14087 <br />  infection cadmium responses, subtler differences apparent: seven overlapping CUB-containing genes, (F55G11.2) induced infection, repressed cadmium exposure, (F08G5.6 F55G11.5) responded strongly infection cadmium. comparison <br /> demonstrates  despite common features, infection <br /> response largely distinct general stress response. <br />    prominent stress mechanism C. elegans forkhead <br /> transcription factor DAF-16, necessary lifespan <br /> extension (27) increased resistance infection (11) daf-2 <br /> insulin pathway mutants.  wondered DAF-16 regulator infection response. compared infection-response gene list previously reported list DAF-16 <br /> targets (28)  overlap existed lists (41 genes), trends opposite. Lipid metabolism thaumatin genes, induced DAF-16, repressed infection. Similarly, infection-induced genes, lectin, <br /> DUF274, CUB-containing genes, repressed DAF-16. <br /> DUF-16 function requires nuclear translocation (27). Using strain expresses functional DAF-16::GFP protein fusion, observed  unlike oxidative, starvation heat stresses, nuclear <br /> localization detected PA14-infected animals (data shown).  results imply DAF-16 active regulator observed infection response, regulatory mechanism involved. <br />  GATA Motif Enriched Promoters Infection-Response Genes.        Fig. 2. Functional consequences elt-2 expression knockdown.   Gene <br />                                                                           expression changes elt-2 C18G1.2 infection. Means &#226;&#171;&#190; SEM identify mechanisms directly regulate observed gene <br />                                                                           microarray measurements shown. (B) Survival assays expression changes, searched proximal promoter regions             spe-26 mutants fed E. coli expressing elt-2 RNAi (open circles; n &#226;&#171;&#189; <br /> (500 bp) infection-response genes shared DNA motifs,         102) containing control RNAi vector (filled circles; n &#226;&#171;&#189; 99) mediate binding transcriptional regulators (Supporting           subsequently transferred  time 0) PA14 lawns. Shown curve Text). exclusively identified GATA motif (TGATAAGA;                means &#226;&#171;&#190; SD fraction live animals plates. (C) Lifespan <br /> Table 5, published supporting information PNAS         assays experimental groups [n &#226;&#171;&#189; 85 elt-2(RNAi) <br /> web site) 35% induced genes 25% repressed            animals; n &#226;&#171;&#189; 92 controls]. Worms exposed, RNAi treatment, genes, compared 10% prevalence proximal promoter regions          kanamycin-killed OP50-1. (D E) Faster intestinal accumulation P. aerugi- 4,000 random genes (P &#226;&#172;? 10&#226;&#171;&#186;8, hypergeometric distribution).            nosa. Representative images (&#226;&#171;&#187;200) glp-4;rrf-3 mutants grown control <br />                                                                           (D) elt-2 -RNAi-expressing bacteria (E). Asterisks mark pharynx. <br /> GATA motifs preferentially localized 250-bp proximal region. <br />    enrichment GATA motifs suggested functional role         initial exposure, 2.6% 5.9% control animals motif regulating infection responses. Consistent    independent experiments showed intestine completely filled <br /> genes encoding GATA-binding transcription factors induced            bacteria (Fig. 2D). contrast, 19% 57% elt-2(RNAi) infection: elt-2 C18G1.2. induction observed       animals, respectively, showed filled intestines (Fig. 2E; P &#226;&#171;&#189; 0.02, &#226;?&#185;2 <br /> microarray results (Fig. 2A) verified qRT-PCR <br />                                                                           test, 40 100 animals group, respectively). suggests (Table 3). hypothesized transcription <br />                                                                           ELT-2 functions slow bacterial accumulation worm <br /> factors GATA-binding protein modulated                                                                           intestine. <br /> expression infection-responsive genes. <br />                                                                           elt-2 Knockdown Impairs Infection-Specific Gene Induction. fur- <br /> ELT-2 Functions Postdevelopmentally Protect Worms Infec- <br />                                                                           ther investigate role elt-2 infection, asked tion. ELT-2 essential normal intestinal development embryonic larval stages (17). test hypothesis      susceptibility elt-2(RNAi) worms correlated changes postdevelopmental role innate immunity, determined            expression infection-response genes. used qRT-PCR effect elt-2 knockdown adulthood susceptibility           determine effect elt-2 knockdown expression worms infection. exposure RNAi, elt-2 expression              infection-induced genes GATA motifs promoters: <br /> remained suppressed &#226;&#172;&#381;6 days (Fig. 6, published            lys-2, member lysozyme family bacterial-envelope <br /> supporting information PNAS web site). Reducing elt-2              hydrolyzing enzymes, CUB-containing genes, F55G11.2 <br /> expression 10% levels control-treated animals,            F08G5.6.  F08G5.6 itself showed substantial <br /> assessed qRT-PCR, resulted significant decrease survival       contribution infection resistance (Table 4). Knocking elt-2 PA14 (P &#226;&#172;? 0.0001, log-rank test; Fig. 2B). represented         expression adult animals significantly reduced basal expression increased sensitivity infection general shortening        genes (Fig. 3A). additionally abolished induction <br /> lifespan (Fig. 2C). Comparable results diverse genetic      24-h exposure PA14  F08G5.6 F55G11.2) backgrounds, sterile fertile wild-type animals (results       changed induction repression  lys-2; Fig. 3C). shown), demonstrating elt-2 specifically required         results suggest genes targets ELT-2 antibacterial defense worm. Conversely, detect        provides link ELT-2-dependent gene expression significant effect C18G1.2 knockdown disruption (results       functional consequences inactivation. shown).                                                                  contrast infection-response genes, expression <br />    increased sensitivity elt-2(RNAi) worms infection        general intestinal genes, including GATA-regulated genes <br /> accompanied faster accumulation pathogen worm            (ges-1, ifb-2; ref. 17) genes contribute intestine structure <br /> intestine, seen animals grown PA14-GFP. Fourteen hours           development maintenance (let-413, eps-8, gob-1; refs. <br />  <br /> 14088 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0603424103                                                                                         Shapira et al. <br />  Fig. 3.   elt-2 knockdown selectively affects expression infection- <br /> response genes. RNA levels measured using qRT-PCR. Excluding D,            Fig. 4. Consequences elt-2 knockdown protection infection vs. <br /> columns represent means &#226;&#171;&#190; SD separate experiments.  B)             general stress conditions. Survival assays cdc-25.1-RNAi-sterilized wild- <br /> Effects constitutive expression. Shown normalized fractions specific     type worms, fed elt-2-RNAi-expressing bacteria (filled) control bacte- <br /> RNA levels elt-2(RNAi) animals relative control-treated animals. Double      ria (open) subsequently subjected P. aeruginosa infection  , oxidative <br /> asterisks mark significant decreases (P &#226;&#172;? 0.001; t test).   Infection-response   stress (B), cadmium stress (C), heat shock (D).   Killing P. aeruginosa. <br /> genes. (B) Intestine functional structural markers. (C D) Effects       Means &#226;&#171;&#190; SD fraction live animals plates. Control animals <br /> induced expression. (C) Infection responses. Normalized fold changes ani-       (n &#226;&#171;&#189; 90); elt-2(RNAi) animals (n &#226;&#171;&#189; 110). (B) Sensitivity oxidative stress. <br /> mals exposed 24 h PA14 compared exposed OP50-1. Open          Fraction survival function paraquat concentration (6-h exposure) columns represent control-treated animals; filled columns represent elt-           control elt-2(RNAi) animals. Shown means &#226;&#171;&#190; SD experiments <br /> 2(RNAi) animals. Asterisks mark significant changes (P &#226;&#172;? 0.02; t test). (D)        performed duplicate (18 &#226;&#8364;&#8220; 45 animals point). (C) Sensitivity cadmium. <br /> Cadmium responses. Normalized fold changes animals exposed 100 &#226;?&#174;M            Means &#226;&#171;&#190; SD fraction live animals plates [control, n &#226;&#171;&#189; <br /> cadmium 24 h relative cadmium controls. Columns described         75; elt-2(RNAi), n &#226;&#171;&#189; 85] shown. Cadmium concentration 100 &#226;?&#174;M,  Shown results experiment (means &#226;&#171;&#190; SEM). results              RNAi directed cdr-1 resulted measurable increase sensi- <br /> presented obtained spe-26 worms. Wild-type worms showed similar            tivity (results shown). (D) Sensitivity heat stress. Means &#226;&#171;&#190; SD trends.                                                                            fraction live animals plates subjected 37&#194;&#176;C 17 h <br />                                                                                    [control, n &#226;&#171;&#189; 69; elt-2(RNAi), n &#226;&#171;&#189; 66] shown. <br />  <br /> 29&#226;&#8364;&#8220;31), largely affected elt-2 knockdown (Fig. 3B). <br /> Furthermore, knocking expression genes                       Consistent results described  elt-2 adulthood  did test ges-1) did affect susceptibility              expression necessary protecting adult animals infection <br /> PA14 (Fig. 7, published supporting information                  general stress. assayed wild-type worms <br /> PNAS web site).  results indicate elt-2 knock-                 sterile cdc-25.1 silencing, observed greatest adult animals increases susceptibility infection                    differences control elt-2(RNAi) animals, examining impairment general intestinal function                  resistance variety stress conditions causing detrimental specific role regulating immunity genes.                                    effects wide range time. elt-2(RNAi) animals <br />    elt-2 did contribute infection-associated gene repression,            significantly sensitive PA14 infection controls <br /> observed thn-2, thaumatin gene contains putative                      (P &#226;&#172;? 0.0005; log-rank test), sensitive oxidative <br /> GATA factor-binding site promoter repressed <br />                                                                                    stress caused paraquat, 37&#194;&#176;C heat stress, exposure cadmium infection. elt-2 knockdown did decrease thn-2 basal <br />                                                                                    (Fig. 4). propose adult animal, ELT-2 <br /> expression (Fig. 3A) abolish repression infection (Fig. <br />                                                                                    functions mainly regulating protective antimicrobial responses. <br /> 3C); exacerbated repression, suggesting ELT-2 <br /> positively regulates expression thn-2 infection,                                                                                    Human GATA6 Important Protecting Lung Epithelial Cells factors responsible repression. Finally,                                                                                    Infection. identification ELT-2 local intestinal regulator <br /> expression ELT-2-dependent genes remained unaltered infection. mtl-1, GATA-containing metallothionein gene                    immune responses C. elegans prompted ask regulated ELT-2 cadmium exposure larvae (32),                  endodermal GATA transcription factors perform similar roles did respond infection affected elt-2 knock-                  humans. GATA zinc finger domain ELT-2 resembles (Fig. 3C).                                                                    human GATA4&#226;&#8364;&#8220;6 (33),  like ELT-2, central <br />                                                                                    factors endoderm development (13, 34). adults, GATA4 ELT-2 Functions Adult Worms Immune-Specific Regulatory                    GATA6 expressed endodermally derived tissues,  <br /> Mechanism. test specificity elt-2 inducing                    cluding small intestine lung (35, 36). <br /> infection responses, analyzed gene expression confirmed                    investigate possible involvement GATA4 GATA6 <br /> ELT-2 targets described Fig. 3C exposure cadmium (Fig.                 local infection responses human cells, used A549 lung <br /> 3D). exception lys-2, similarly induced                   epithelial cells infected PA14. qRT-PCR                                                                                                                                                                       IMMUNOLOGY <br />  <br />  <br />  <br />  <br /> cadmium infection ELT-2-dependent manner,                          GATA6 induced &#226;&#172;&#381;4-fold infection (Fig. 5A). genes presented qualitatively different responses cadmium                contrast, GATA4 expression did change (data shown). infection (Fig. 3 C D). Similarly, contribution elt-2          used small inhibitory RNA knock expression expression different conditions, demon-                  GATA6 cells (Fig. 5A) measured ability strating infection-specific regulatory function.                                withstand infection. GATA6 knockdown resulted significantly <br />  <br /> Shapira et al.                                                                                      PNAS &#229;&#8230;&#169; September 19, 2006 &#229;&#8230;&#169; vol. 103 &#229;&#8230;&#169;  38 &#229;&#8230;&#169; 14089 <br />                                                                                          induction specific isoforms functional analyses, serve <br />                                                                                         precursors antimicrobial peptides; unchar- <br />                                                                                         acterized gene families, encode proteins DUF274                                                                                         CUB-like domains. difficult demonstrate functional signif- <br />                                                                                         icance responding genes single gene knockdown                                                                                         redundancy multigene families. example, failed                                                                                         observe significant effect sensitivity infection knock- <br />                                                                                         ing expression tested lysozymes,                                                                                         confer resistance overexpressed (9).  <br />                                                                                          knockdown CUB-containing genes did result Fig. 5. GATA6 important protecting human epithelial cells P.                marked sensitivity infection, suggesting greater importance aeruginosa infection.   GATA6 induced infection A549 cells. Fold           genes protecting worms infection  alternatively, changes GATA6 RNA levels A549 cells, treated GATA6 RNAi (filled              functional redundancy despite presumed structural similarity. <br /> bars), transfection reagent (open bars), exposure PA14 <br />                                                                                         existence CUB domain variant raises intriguing PBS  Fold changes control-treated noninfected cells (set 1). RNA levels measured qRT-PCR. Shown means &#226;&#171;&#190; SD                                                                                         possibility family functions way similar com- <br /> experiments. (B) GATA6 knockdown increases cells&#226;&#8364;&#8482; susceptibility infection.          plement-associated proteases, remains tested. <br /> Percentage damaged membrane-permeabilized cells, marked Trypan                       Overall, considerable subset observed responses blue accumulation, total number cells. Counts performed               distinct general stress responses, cadmium exposure <br /> randomly captured images (109 &#226;&#8364;&#8220;187 cells image), different           (26), expression patterns associated general stress plate. Shown means &#226;&#171;&#190; SD wells,        mechanisms, attributed DAF-16 (28). experiments similar results. Statistically significant values (t test; P &#226;&#172;? 0.01)   DAF-16 required daf-2&#226;&#8364;&#8482;s resistance bacterial infection (11), marked asterisks.                                                              disassociation observed infection response suggests                                                                                         contributes resistance parallel route, probably <br />                                                                                         affecting basal state infected worms induced <br /> increased PA14-induced cytotoxicity 1 h exposure                                                                                         responses, affecting constitutive expression antimicro- <br /> diluted log-phase bacteria (Fig. 5B). cytotoxicity infection- <br />                                                                                         bial genes  specifically, affecting general  <br /> specific, GATA6 knockdown cytotoxic itself <br />                                                                                            RNA level measurements small set ELT-2-dependent <br /> (data shown). larger increase cytotoxicity                           infection response genes highlighted differences  <br /> observed infected cells treated siRNA directed                      tween responses infection cadmium distinct <br /> NF-&#226;?&#172;B subunit gene p105, serving positive control. Cytotoxicity                    contribution elt-2 responses. knocking elt-2 probably attributed secreted P. aeruginosa toxins,                  expression did impair general intestinal expression did secreted stationary phase. short exposure time                            affect animal survival panel stress assays supports notion <br /> sufficient cause cytotoxicity excludes substantial accu-                     adult animals, ELT-2 contributes survival immune <br /> mulation toxins medium, suggesting GATA6                            regulatory mechanism.  considering  <br /> provides protection pathogen itself.                                           sponding genes lack promoter GATA motif, ELT-2    ELT-2, expect GATA6 contribute protecting                           probably sole regulator worm infection responses. <br /> cells gene regulation. Mining published data genes                           similarities differences role identified responding infection lung epithelial cells (37, 38) revealed                    ELT-2 regulating innate immune responses previously <br /> high prevalence GATA motif (&#226;&#172;&#381;75%) strongly induced                            described Drosophila GATA transcription factor, Serpent. <br /> genes, suggesting GATA-dependent regulation common.                             Like ELT-2, Serpent essential development gut.  results provide evidence GATA6 important                         Serpent contributes early endodermal development protecting human lung cells infection.                                       (14), ELT-2 responsible terminal differentiation                                                                                         intestinal cells. Serpent additionally required formation Discussion <br />                                                                                         mesodermal fat body hematopoiesis. regard identification ELT-2 key regulator robust infection                    regulating innate immune responses, Serpent functions hemo- <br /> transcriptional response describes representative regulatory                     cytes larval fat body induce expression antimicrobial <br /> level missing worm, connecting                           peptides response systemic infection (15). Serpent <br /> protective signal transduction pathways immune gene induc-                          controls epithelial responses larvae adults known, <br /> tion. Importantly, experiments human lung epithelial cells                       reported dispensable antimicrobial peptide induc- <br /> suggest main function identified ELT-2 C. elegans                    tion epithelial cells embryonic epidermis (42). contrast evolutionarily conserved. findings consistent role                    Serpent&#226;&#8364;&#8482;s role systemic responses, ELT-2 responds local GATA6 protective local epithelial responses lung                      infection regulates essentially local response epithelial <br /> potentially endodermal tissues intestine.                     cells. comparison suggests genes orthologs. different previously described functions GATA proteins                       Instead, propose Drosophila endoderm-specific dGATAe hematopoietic differentiation (39) systemic anti-infection                    transcription factor, required terminal differentiation responses Drosophila&#226;&#8364;&#8482;s serpent (15).                                                 fruitfly intestinal cells greater sequence similarity ELT-2 <br />    P. aeruginosa infection worm localized alimentary                   (33), likely elt-2 ortholog. <br /> tract, similar human bacterial pathogens infect C.                           human immune responses, GATA transcription factors <br /> elegans (40). transcriptional response delineated                   (GATA1-3) function hematopoietic cell-terminal differentiation <br /> infection apparent 4 h exposure pathogen,                       (39). identification ELT-2 regulator local innate <br /> 57% responding genes primarily expressed                              immune responses, sequence homology tissue speci- <br /> intestine, suggesting major response associated                 ficity, pointed branch family, endodermal pathogen entry primary local intestinal response.                     GATA transcription factors GATA4-6, possible functional ho- <br />    responding genes members gene families                              mologs. experiments human lung epithelial cells suggest known suspected various levels innate immune                      function identified ELT-2 C. elegans conserved, <br /> functions: lectins known serve pathogen recognition                           consistent role GATA6 protective local epithelial <br /> proteins organisms (41); lysozymes known                         responses lung. unknown genes regulated <br /> bacterial hydrolyzing activity; histones, based preferred                        GATA6 infection. target previously reported  <br /> 14090 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0603424103                                                                                               Shapira et al. <br />  surfactant protein (43). high prevalence GATA motifs                             hatch grow presence cdc-25.1 RNAi, developing <br /> promoters epithelially expressed genes responding infection                            sterile adult animals. <br /> suggests additional targets exist. results encourage                           Larval stage L4 (spe-26 glp-4;rrf-3 mutants) young adult (N2 <br /> exploring function GATA transcription factors lungs                            cdc-25.1-sterilized) worms fed 48 h double-strand cystic fibrosis patients, affected P. aeruginosa                          (ds)RNA-expressing E. coli (Supporting Text) transferred <br /> infections.                                                                                  preseeded PA14 plates. Bacteria containing RNAi <br />                                                                                              expression vector served control. Results survival assays Materials Methods                                                                        analyzed using Kaplan&#226;&#8364;&#8220;Meier analysis, scoring significance Infection. Worms infected feeding bacteria grown                               log-rank test. <br /> modified nematode growth (NG) plates 25&#194;&#176;C (18).                                              Lifespan, heat-stress, cadmium-survival assays  <br />                                                                                              formed kanamycin- UV-killed OP50&#226;&#8364;&#8220;1 bacteria (Supporting <br /> Growing Worms Microarray Experiments. Synchronized C. elegans                            Text). <br /> cultures prepared using standard techniques (Supporting <br /> Text). Young adult animals split modified NG plates                             qRT-PCR. qRT-PCR performed using standard techniques <br /> preseeded OP50 PA14 incubated 4, 12, 24 h                                 (Supporting Text). amplification reaction performed harvesting. experiment repeated times                               duplicate triplicate, fold changes normalized changes <br /> different occasions.                                                                         actin RNA levels. t tests performed using normalized <br />                                                                                              threshold-cycle values. Quantification GATA6 RNA levels Microarray Analyses. cDNA probes prepared poly  &#226;&#171;&#185;                                A549 cells performed similarly, using RNA levels human <br /> RNA hybridized spotted microarrays (Supporting Text).                                 actin gene ACTB normalization. <br /> Log-transformed (base 2) ratios used statistical <br /> analyses.                                                                                    Cell Culture, siRNA Transfection, RNA Extraction, Cytotoxicity <br />                                                                                              Assays. A549 human lung epithelial cells (American Type Culture <br /> Identifying Genes Respond Infection. data set passed                        Collection, Manassas, VA) grown standard conditions <br /> quality filters contained 7,972 PCR products representing 7,308                              confluent, transferred plates, transfected genes (Dataset 1, published supporting information                            GATA6 p105 siRNA SMARTpools using DharmaFECT1 PNAS web site). identified infection-response genes                            reagent manufacturer&#226;&#8364;&#8482;s instructions (Dharmacon, Lafay- <br /> procedure based way ANOVA, allows estimation                                 ette, . hours siRNA treatment, cells false discovery rates (FDR; Supporting Text). procedure                             washed, incubated 1 h serum-free medium, infected implemented MATLAB available request.                                                                                              log-phase PBS washed PA14 multiplicity infection 100:1. <br /> maximize number true positives, selected FDR 20% <br />                                                                                              hours 1 h infection, respectively, cells (Fig. 8, published supporting information PNAS <br />                                                                                              harvested RNA extraction (RNeasy; Qiagen, Valencia, CA) web site). <br />                                                                                              assayed cytotoxic effects using Trypan blue (Sigma&#226;&#8364;&#8220;Aldrich, St. <br />                                                                                              Louis, MO). <br /> RNAi Knockdown Survival Assays. avoid confounding effects progeny survival assays, used sterile worm strains <br />                                                                                              thank Florencia Pauli, Min Jiang, Stuart Kim assistance spe-26(it112) glp-4(bn2);rrf-3(pk1426). Additionally, used                            microarray analysis David Schneider, Rosie Alegado, Jennifer <br /> wild-type animals N2 strain, sterile exposure                              Berman, Trupti Kawli comments manuscript. work <br /> RNAi directed cdc-25.1, gene important germline                                supported National Institutes Health Grant GM66269. M.S. <br /> proliferation (44): gravid worms laid cdc-25.1 RNAi plates                           supported Life Science Research Foundation Fellowship 4 h transferred similar plates additional 4 h                         sponsored O&#226;&#8364;&#8482;Donnell Foundation. M.-W.T. Basil O&#226;&#8364;&#8482;Connor egg laying.  gravids removed, eggs left                          V Foundation Scholar. <br />  <br />  <br />  1.   Janeway CA, Jr, Medzhitov R (2002) Annu Rev Immunol 20:197&#226;&#8364;&#8220;216.                        23. Bork P, Beckmann G (1993) J Mol Biol 231:539&#226;&#8364;&#8220;545. <br />  2.   Dangl JL, Jones JD (2001) Nature 411: 826&#226;&#8364;&#8220;833.                                         24. Park CB, Yi KS, Matsuzaki K, Kim MS, Kim SC (2000) Proc Natl Acad Sci USA <br />  3.   Matzinger P (2002) Science 296:301&#226;&#8364;&#8220;305.                                                    97:8245&#226;&#8364;&#8220;8250. <br />  4.   Lemaitre B, Nicolas E, Michaut L, Reichhart, JM, Hoffmann JA (1996) Cell 86:973&#226;&#8364;&#8220;983.   25. Brinkmann V, Reichard U, Goosmann C, Fauler B, Uhlemann Y, Weiss DS, Weinrauch <br />  5.   Medzhitov R, Preston-Hurlburt P, Janeway CA, Jr (1997) Nature 388:394&#226;&#8364;&#8220;397.                 Y, Zychlinsky (2004) Science 303:1532&#226;&#8364;&#8220;1535. <br />  6.   Pujol N, Link EM, Liu LX, Kurz CL, Alloing G, Tan MW, Ray KP, Solari R, Johnson        26. Huffman DL, Abrami L, Sasik R, Corbeil J, van der Goot FG, Aroian RV (2004) Proc <br />       CD, Ewbank JJ (2001) Curr Biol 11:809&#226;&#8364;&#8220;821.                                                 Natl Acad Sci USA 101:10995&#226;&#8364;&#8220;11000. <br />  7.   Alegado RA, Campbell MC, Chen WC, Slutz SS, Tan MW (2003) Cell Microbiol               27. Lin K, Hsin H, Libina N, Kenyon C (2001) Nat Genet 28:139&#226;&#8364;&#8220;145. <br />       5:435&#226;&#8364;&#8220;444.                                                                             28. Murphy CT, McCarroll SA, Bargmann CI, Fraser  Kamath RS, Ahringer J, Li H, <br />  8.   Tan M-W (2001) Pediatric Pulmonol 32:96&#226;&#8364;&#8220;97.                                                Kenyon C (2003) Nature 424:277&#226;&#8364;&#8220;283. <br />  9.   Mallo GV, Kurz CL, Couillault C, Pujol N, Granjeaud S, Kohara Y, Ewbank JJ (2002)      29. Bossinger O, Fukushige T, Claeys M, Borgonie G, McGhee JD (2004) Dev Biol <br />       Curr Biol 12:1209&#226;&#8364;&#8220;1214.                                                                    268:448&#226;&#8364;&#8220;456. <br /> 10.   Garsin DA, Villanueva JM, Begun J, Kim DH, Sifri CD, Calderwood SB, Ruvkun G,          30. Croce  Cassata G, Disanza  Gagliani MC, Tacchetti C, Malabarba MG, Carlier MF, <br />       Ausubel FM (2003) Science 300:1921.                                                        Scita G, Baumeister R, Di Fiore PP (2004) Nat Cell Biol 6:1173&#226;&#8364;&#8220;1179. <br /> 11.   Kim DH, Feinbaum R, Alloing G, Emerson FE, Garsin DA, Inoue H, Tanaka-Hino M,          31. Kormish JD, McGhee JD (2005) Dev Biol 287:35&#226;&#8364;&#8220;47. <br />       Hisamoto N, Matsumoto K, Tan MW, Ausubel FM (2002) Science 297:623&#226;&#8364;&#8220;626.                32. Moilanen LH, Fukushige T, Freedman JH (1999) J Biol Chem 274:29655&#226;&#8364;&#8220;29665. <br /> 12.   Patient RK, McGhee JD (2002) Curr Opin Genet Dev 12:416&#226;&#8364;&#8220;422.                           33. Okumura T, Matsumoto  Tanimura T, Murakami R (2005) Dev Biol 278:576&#226;&#8364;&#8220;586. <br /> 13.   Molkentin JD (2000) J Biol Chem 275:38949&#226;&#8364;&#8220;38952.                                       34. Yang H, Lu MM, Zhang L, Whitsett JA, Morrisey EE (2002) Development (Cambridge, <br /> 14.   Rehorn KP, Thelen H, Michelson  Reuter R (1996) Development (Cambridge, UK)             UK) 129:2233&#226;&#8364;&#8220;2246. <br />       122:4023&#226;&#8364;&#8220;4031.                                                                         35. Dusing MR, Wiginton DA (2005) J Mol Histol 36:15&#226;&#8364;&#8220;24. <br /> 15.   Petersen UM, Kadalayil L, Rehorn KP, Hoshizaki DK, Reuter R, Engstrom Y (1999)         36. Caramori G, Lim S, Ito K, Tomita K, Oates T, Jazrawi E, Chung KF, Barnes PJ, Adcock <br />       EMBO J 18:4013&#226;&#8364;&#8220;4022.                                                                       IM (2001) Eur Respir J 18:466&#226;&#8364;&#8220;473. <br /> 16.   Maduro MF, Rothman JH (2002) Dev Biol 246:68&#226;&#8364;&#8220;85.                                       37. Ichikawa JK, Norris  Bangera MG, Geiss GK, van &#226;&#8364;&#8482;t Wout AB, Bumgarner  Lory <br /> 17.   Fukushige T, Hawkins MG, McGhee JD (1998) Dev Biol 198:286&#226;&#8364;&#8220;302.                            S (2000) Proc Natl Acad Sci USA 97:9659&#226;&#8364;&#8220;9664. <br /> 18.   Tan MW, Mahajan-Miklos S, Ausubel FM (1999) Proc Natl Acad Sci USA 96:715&#226;&#8364;&#8220;720 <br /> </body></html>