<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Why Hubs Tend Essential Protein <br /> Networks? <br /> Xionglei  Jianzhi Zhang* <br /> Department Ecology Evolutionary Biology, University Michigan, Ann Arbor, Michigan, United States America <br />  <br />  protein&#226;&#8364;&#8220;protein interaction (PPI) network small number highly connected protein nodes (known hubs) poorly connected nodes. Genome-wide studies deletion hub protein likely lethal deletion non-hub protein, phenomenon known centrality-lethality rule. rule widely believed reflect special importance hubs organizing network, turn suggests biological significance network architectures, key notion systems biology. Despite popularity explanation, underlying <br /> cause centrality-lethality rule critically examined. propose concept essential PPIs, PPIs indispensable survival reproduction organism. network analysis suggests centrality-lethality rule unrelated network architecture, explained simple fact hubs large numbers PPIs, high probabilities engaging essential PPIs. estimate ; 3% PPIs essential yeast, accounting ; 43% essential genes. expected, essential PPIs evolutionarily conserved nonessential PPIs. Considering role essential PPIs determining gene essentiality, yeast PPI network functionally robust random networks, far robust potential optimum. findings provide new perspectives biological relevance network structure robustness. <br /> Citation: X, Zhang J (2006) hubs tend essential protein networks? PLoS Genet 2(6): e88. DOI: 10.1371/journal.pgen.0020088 <br />  <br />  <br />                                                                                        functional importance node thought arise <br /> Introduction <br />                                                                                       structural importance network [6,7,9,10].    network composed multiple nodes connected                               true, interpretation suggests biological signi&#239;&#172;?cance edges. complex networks scale-free, power-                            network structures fundamental systems <br /> law distribution number edges node, node                             biology. challenge view proposing connectivity [1,2].  scale-free network contains                          alternative explanation centrality-lethality rule small number highly connected nodes (hubs) large                             does invoke network architecture. evaluate <br /> number poorly connected nodes (non-hubs). relative                             new explanation empirical data demonstrate <br /> importance node network measured                            prevailing interpretation centrality-lethality <br /> magnitude changes network structure caused                               rule unlikely correct. <br /> removal node. accurately, measure termed structural importance node. instance, <br /> computational analysis shows removing hubs increases                             Results/Discussion proportion unreachable pairs nodes mean                             Alternative Explanation Centrality-Lethality Rule <br /> shortest path length pairs reachable nodes                          Based Essential PPIs network  e., network diameter) removing                                  current analysis PPI networks treats edges <br /> non-hubs [3].  hubs important non-hubs                            equally. reality, PPIs important maintenance global network structure.                                 consideration particularly meaningful <br /> biomolecular networks, genes proteins nodes                              PPIs essential (indispensable) molecular interactions edges, importance                             survival reproduction organism. essential <br /> node measured magnitude changes                              interaction proteins makes proteins <br /> network function organismal &#239;&#172;?tness caused removal                           essential, removal protein causes node. measure called functional <br /> importance node. example, genome-wide gene <br /> deletion studies small faction genes                              Editor: Takashi Gojobori, National Institute Genetics, Japan <br /> genome indispensable survival reproduction                           Received January 24, 2006; Accepted April 26, 2006; Published June 2, 2006 organism [4,5]; genes referred essential                           previous version article appeared Early Online Release April 26, <br /> genes. scale-free protein&#226;&#8364;&#8220;protein                            2006 (DOI: 10.1371/journal.pgen.0020088.eor). <br /> interaction (PPI) network [6&#226;&#8364;&#8220;8], hubs tend essential [6].                       DOI: 10.1371/journal.pgen.0020088 phenomenon observed yeast, nematode, <br />                                                                                       Copyright: &#195;&#8220; 2006 Zhang. open-access article distributed &#239;&#172;&#8218;y [9&#226;&#8364;&#8220;11] commonly referred centrality-                          terms Creative Commons Attribution License, permits unrestricted <br /> lethality rule [6]. Using terms described                               use, distribution, reproduction medium, provided original author <br /> centrality-lethality rule indicates correlation                           source credited. <br /> node&#226;&#8364;&#8482;s structural importance PPI network                               Abbreviations: ER, Erdo&#194;&#168;s-Re&#194;&#180;nyi; IBEP, interaction essential proteins; PPI, <br /> functional importance. critical analysis, corre-                         protein&#226;&#8364;&#8220;protein interaction <br /> lation widely interpreted causal relationship.                          * correspondence addressed. E-mail: jianzhi@umich.edu <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                    0826                                            June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                               Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  <br /> Synopsis                                                                      IBEPs randomly rewired networks (Figure 1B).                                                                               mean m 592.6. 10,000 m values greater Proteins interactions form protein&#226;&#8364;&#8220;protein interaction            number IBEPs real network, strongly suggesting <br /> network, proteins nodes interactions            excess IBEPs real network (p , 0.0001). edges. Genomic studies deleting highly connected              excess evident different datasets yeast PPIs protein node (hub) likely lethal organism            nematode PPIs [14,15]  Figures S1 S2). <br /> deleting lowly connected node (non-hub), phenomenon known                 assumption excess IBEPs entirely centrality-lethality rule. hubs important <br />                                                                               caused essential PPIs, estimate &#194;&#188; (807&#226;&#8364;&#8220;592.6)/7356 non-hubs organizing global network structure,                                                                               &#194;&#188; 2.92% interactions yeast PPI network essential. <br /> centrality-lethality rule widely believed reflect significance network architecture determining network function, key                standard error 0.23%. used random <br /> notion systems biology. work, authors proposed               rewiring estimate easy ways small fraction randomly distributed essential interactions,        calculate analytically unless self-interactions allowed. lethal organism disrupted. scenario,              network, 836 proteins, 20.3% nodes, hub likely essential non-hub simply            essential. addition essential PPIs, factors hub interactions higher chance engage              (e.g., protein-DNA interaction) render protein essential interaction.  centrality-lethality rule          essential. Let b probability node explained involvement network architecture. Using              essential factors. estimate b, &#239;&#172;?rst remove <br /> yeast data, authors provided empirical evidence supporting                                                                               information gene essentiality yeast PPI network. <br /> hypothesis. proposal results challenge prevailing view systems biology provide new perspective role                                                                               randomly assign 807-m essential edges network structures biology.                                                network, m randomly drawn distribution <br />                                                                               Figure 1B. Note 807-m estimated number                                                                               essential edges. Nodes having essential edges marked <br />                                                                               essential.  mimic in&#239;&#172;&#8218;uence factors <br /> lethality infertility disruption interaction. <br />                                                                               cause gene essentiality randomly marking nodes Empirical data indicate existence essential PPIs.                                                                               essential, total number essential nodes example, yeast proteins SPT16 POB3 essential <br />                                                                               network 836. Repeating procedure 10,000 times, form heterodimers function DNA replica- <br />                                                                               estimate essential PPIs render 8.7% nodes <br /> tion; genetic studies showed interaction critical <br />                                                                               essential, factors render b &#194;&#188; 12.64% nodes function [12]. Essential PPIs potentially explain <br />                                                                               essential. standard error b 0.63%. Note centrality-lethality rule, proteins PPIs <br />                                                                               nodes (1.1%) affected essential PPIs greater probability engage essential <br />                                                                               factors. Approximately 43% (8.7%/20.3%) protein essen- <br /> PPI, having higher chance essential. Note               tiality attributable essential interactions PPI <br /> network architecture invoked explanation.                      network. <br /> Evaluation Number Essential PPIs                             estimates b biased factors. <br />                                                                                features yeast PPI network Contribution Gene Essentiality <br />                                                                               distorted random rewiring, affecting estimation <br />   dif&#239;&#172;?cult identify essential PPIs experimentally                                                                                example, known links highly <br /> genomic scale, identi&#239;&#172;?cation requires                                                                               connected nodes suppressed PPI networks [16,17]. <br /> demonstration disrupting interaction                                                                               highly connected nodes tend essential [6,9&#226;&#8364;&#8220;11], <br /> essential proteins affecting aspects                                                                               suppression reduces number IBEPs.  protein functions causes lethality infertility. use              suppression accounted rewiring, computational approach evaluate prevalence                          overestimated m, consequently underestimated  <br /> essential PPIs contribution essential PPIs gene                  bias probably small, suppression <br /> essentiality genomic level. analysis focuses                appears largely limited nonessential proteins [14]. <br /> yeast Saccharomyces cerevisiae PPI gene                  Second, quality PPI data affect reliability essentiality data complete species.                          estimates. particular, transforming protein complex <br />   built yeast PPI network, 4,126 protein                      information binary PPI data using &#226;&#8364;&#732;&#226;&#8364;&#732;spoke&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> nodes linked 7,356 edges. PPI data used                    model  bait predicted interact members compiled manually Comprehensive Yeast Genome                           complex) &#226;&#8364;&#732;&#226;&#8364;&#732;matrix&#226;&#8364;&#8482;&#226;&#8364;&#8482; model  members complex <br /> Database [13] literature published large-scale                   predicted interact members experiments. mentioned, proteins forming                            complex) [18] tends generate extra IBEPs large <br /> essential PPI essential (Figure 1A). contrary,                 complexes, lead overestimation  <br /> interactions essential proteins (IBEPs)                     data include protein essential, essentiality protein                complex information recently produced high-throughput factors essential PPIs (Figure 1A).                    methods [19,20], largely immune feature allows estimate number essential PPIs               problem.  assumed excess observed IBEPs <br /> network, number IBEPs increases number                     real network entirely essential PPIs, essential PPIs. 807 IBEPs network.                     caused nonrandom features real <br /> generated control network randomly rewiring edges                    network [14,15]. Finally, estimation b based real network keeping node connectivity (k)                   assumption factors causing protein essentiality <br /> unchanged node. repeating procedure                         affect nodes PPI network equally random <br /> 10,000 times, obtained distribution number (m)                  manner. <br />  <br />       PLoS Genetics | www.plosgenetics.org                                 0827                              June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                      Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  <br />                                                                                     yeast PPI network largely captured step <br />                                                                                     procedure, involves essential PPIs randomly <br />                                                                                     distributed edges essentiality-determining <br />                                                                                     factors randomly distributed nodes. &#239;&#172;?nal <br />                                                                                     critical evaluation estimates b                                                                                     test protein essentiality predicted using                                                                                     estimates. protein nonessential, conditions <br />                                                                                     satis&#239;&#172;?ed.  protein essential PPI. <br />                                                                                     Second, protein affected factors                                                                                     cause essentiality.  probability (PE) protein <br />                                                                                     k PPIs essential  <br />  <br />                                                                                                         PE &#194;&#188; 1   &#195;&#176;1   b&#195;&#382;&#195;&#176;1   a&#195;&#382;k ;                    &#195;&#176;1&#195;&#382; <br />                                                                                     b estimated earlier.  PE values <br />                                                                                     predicted k using equation.                                                                                     observed PE yeast PPI network matched                                                                                     predicted PE (Figure 2A). did compare PE values k <br />                                                                                     . 10, nodes k value k . <br />                                                                                     10. Equation 1 rewritten natural logarithm  <br />  <br />                                                                                                  ln&#195;&#176;1   PE &#195;&#382; &#194;&#188; k ln&#195;&#176;1   a&#195;&#382; &#195;&#190; ln&#195;&#176;1   b&#195;&#382;:               &#195;&#176;2&#195;&#382; <br />  <br />                                                                                       Equation 2 predicts ln(1-PE) changes linearly k. <br />                                                                                     linear relationship con&#239;&#172;?rmed yeast PPI <br />                                                                                     network (correlation coef&#239;&#172;?cient &#194;&#188; 0.927, p &#194;&#188; 0.0001; Figure <br />                                                                                     2B). estimate &#194;&#188; 3.29% b &#194;&#188; 12.8% slope <br />                                                                                     Y-intersect linear regression, respectively (Figure <br />                                                                                     2B). estimates signi&#239;&#172;?cantly different                                                                                     earlier estimates based simulated networks (p . 0.5). <br />                                                                                     Taken  tests con&#239;&#172;?rm estimates                                                                                     b reasonably good. <br /> Figure 1. Essential Edges (Interactions) PPI Networks <br />   hypothetical PPI network 12 proteins. Black white nodes <br />                                                                                     Essential PPIs Evolutionarily Conserved refer essential nonessential proteins, respectively.          Nonessential PPIs <br /> edges depict essential nonessential interactions, respectively.                    interesting predict PPIs essential. <br /> Proteins linked essential interaction essential, interaction essential proteins (IBEP)                                                                                     prediction naturally dif&#239;&#172;?cult estimating <br /> essential.                                                                          percentage PPIs essential, (B) IBEPs yeast PPI network randomly rewired                    scarcity information individual PPIs. Nonetheless, networks. &#226;&#8364;&#732;&#226;&#8364;&#732;Observed&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates observed number (807) IBEPs              clear IBEPs essential. probability real network. gray bars distribution number (m) IBEPs 10,000 randomly rewired networks.                                       IBEP essential (807&#226;&#8364;&#8220;592.6)/807 &#194;&#188; 0.27. 807 total <br /> DOI: 10.1371/journal.pgen.0020088.g001                                              number IBEPs (807&#226;&#8364;&#8220;592.6) estimated number                                                                                     essential interactions. interacting essential proteins   examine potential biases evaluate                             interact essential proteins (observation O), reliability estimates, conducted tests.                       posterior probability interaction essential (event <br /> according analysis, factors essential PPIs                        E) derived Bayes theorem  <br /> render b &#194;&#188; 12.6% proteins essential. 1,952 yeast <br /> proteins PPIs included                                           P&#195;&#176;OjE&#195;&#382;P&#195;&#176;E&#195;&#382;       13a <br />                                                                                           P&#195;&#176;EjO&#195;&#382; &#194;&#188;               &#194;&#188; <br /> PPI network. Interestingly, 11.9% 6 0.8 % (233/1,952)                                                 P&#195;&#176;O&#195;&#382;      &#195;&#190; &#195;&#176;1   a&#195;&#382;b2 proteins essential, number statistically indistin-                                                   0:029 <br /> guishable b (p . 0.4, chi-squares test). congruence                                     &#194;&#188;                                &#194;&#188; 0:65:            &#195;&#176;3&#195;&#382; <br />                                                                                                       0:029 &#195;&#190; &#195;&#176;1   0:029&#195;&#382; 3 0:1262 <br /> suggests estimate b reliable assumption stochastically equal in&#239;&#172;&#8218;uences factors                       yeast PPI network contains 38 &#226;&#8364;&#732;&#226;&#8364;&#732;probably <br /> nodes acceptable. Second, estimation relied                       essential&#226;&#8364;&#8482;&#226;&#8364;&#8482; PPIs  Table S1 gene names functions). <br /> simulated networks, compared network features                            Compared nonessential PPIs, essential PPIs expected simulated real networks. particular, node essentiality                       conserved evolution importance randomly reassigned estimation b,                        organismal survival reproduction. test network structure unaltered.                                  hypothesis, assembled PPI network fruit &#239;&#172;&#8218;y <br /> frequency distribution node connectivity similar                              Drosophila melanogaster. 1,066 PPIs yeast reassigned networks real network                            proteins orthologs fruit &#239;&#172;&#8218;y, 4.3% essential nonessential nodes (Figure S3). result                      PPIs conserved species (Table 1 suggests determination node essentiality                         Table S2). comparison, 7.6% IBEPs 26.3%  <br />        PLoS Genetics | www.plosgenetics.org                                  0828                                   June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                                              Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  <br />                                                                                                    probably essential PPIs conserved species, <br />                                                                                                    con&#239;&#172;?rming prediction essential PPIs evolutio- <br />                                                                                                    narily conserved nonessential PPIs (Table 1                                                                                                    Table S2). phylogenetic conservation, 38 <br />                                                                                                    probably essential interactions special <br />                                                                                                    features. apparently enriched functional <br />                                                                                                    categories, biological processes, stable protein complexes. <br />                                                                                                    example, 45% 38 probably essential interactions <br />                                                                                                    involve proteins appear protein <br />                                                                                                    complexes, compared 47% 748 IBEPs (p . <br />                                                                                                    0.5, v2 test). possible certain enrichment does exist, <br />                                                                                                    dif&#239;&#172;?cult discern small sample size. <br />  <br />                                                                                                    Essential PPIs Explain Centrality-Lethality Rule <br />                                                                                                       analysis yeast PPI network suggests                                                                                                    centrality-lethality rule simple fact highly <br />                                                                                                    connected nodes involved PPIs poorly <br />                                                                                                    connected nodes, having greater probabilities engag- <br />                                                                                                    ing essential PPIs. Equation 1 PE                                                                                                    determined factors. protein <br />                                                                                                    connectivity, arising solely essential PPIs,                                                                                                    factor independent protein connectivity.                                                                                                    success equation describing empirical observa- <br />                                                                                                    tions (Figure 2) congruence estimates                                                                                                    b obtained different approaches suggest factors <br />                                                                                                    dependent protein interactions, unrelated essential <br />                                                                                                    PPIs, trivial, implying gene essentiality unlikely                                                                                                    cumulative pleiotropic effects PPI level. <br />                                                                                                    Furthermore, suggest structural features <br />                                                                                                    PPI network, protein connectivity sole <br />                                                                                                    determinant protein essentiality, determi- <br />                                                                                                    nation essential PPIs. results argue                                                                                                    hypothesis centrality-lethality rule attributable                                                                                                    relative importance hub proteins maintenance <br />                                                                                                    network architecture [6,7,9,10]. support                                                                                                    hypothesis, node centrality, measured betweenness                                                                                                    closeness, higher essential nodes                                                                                                    nonessential nodes yeast PPI network, control <br /> Figure 2. Relationship Probability Protein                                                                                                    node connectivity (Tables 2 3).  betweenness Essential (PE) Connectivity (k) Protein <br />                                                                                                    node proportion shortest paths pairs   Observed predicted PE values. observed values estimated yeast PPI network predicted values                                 reachable nodes node, closeness <br /> computed using Equation 1 parameters &#194;&#188; 2.92% b &#194;&#188; 12.6%.                                 node mean shortest path length node <br /> Error bars standard (sampling) error observed values.                              reachable nodes network. betweenness <br /> (B) Linear regression ln(1-PE) k. Using Equation 2, estimated regression &#194;&#188; 3.29% b &#194;&#188; 12.8%. 95% <br />                                                                                                    closeness measure centrality node global <br /> confidence interval 2.23%&#226;&#8364;&#8220;4.35%. 95% confidence                               network structure. support hypothesis comes <br /> interval b 6.7%&#226;&#8364;&#8220;18.6%. Proteins k . 10 (; 5%                            recent analysis yeast PPI network, hubs <br /> proteins) considered small sample sizes.                                        classi&#239;&#172;?ed types according coexpression <br /> DOI: 10.1371/journal.pgen.0020088.g002 <br />  <br />  <br /> Table 1. Conservation Essential PPIs Yeast Fruit Fly <br />  <br /> Types Interactions           Number Yeast Protein Interactions                   Number Conserved Interactions                  Proportion Conserved        p-Valuec <br />                                 Conserved Proteinsa                              Speciesb                          Interactions <br />  <br /> PPIs                            1,066                                                  46                                                0.043 <br /> IBEPsd                            340                                                  26                                                0.076                          0.0006 <br /> Probably essential PPIse           19                                                   5                                                0.263                          0.0023 <br />  <br />   identified 1,764 orthologous gene pairs yeast fruit fly genes form 1,066 PPIs yeast. <br /> b <br />   proteins forming yeast PPI orthologs fruit fly, fruit fly proteins form PPI. <br /> c <br />  Fisher&#226;&#8364;&#8482;s exact test hypothesis proportion conserved interactions identical row. <br /> d <br />   Interactions essential proteins. 27% IBEPs essential. comparison, 3% PPIs essential. <br /> e <br />   IBEPs. addition, proteins forming interaction interacting proteins essential. 65% PPIs essential. <br /> DOI: 10.1371/journal.pgen.0020088.t001 <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                                0829                                            June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                                               Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  <br />  <br /> Table 2. Normalized Betweenness Essential Nonessential Proteins Yeast PPI Network <br />  <br /> Connectivity (k)               Essential Proteins                                                      Nonessential Proteins <br />                                Number Proteins                  Mean Betweenness                    Number Proteins                   Mean Betweenness                    p-Valuea <br />  <br />  1                             243                                 0.000000                            1,383                                0.000000                            1.0000 <br />  2                             170                                 0.000210                              784                                0.000264                            0.0080 <br />  3                             119                                 0.000513                              401                                0.000545                            0.3120 <br />  4                              76                                 0.000762                              238                                0.000810                            0.4725 <br />  5                              51                                 0.000914                              118                                0.000996                            0.1016 <br />  6                              29                                 0.001316                               87                                0.001204                            0.6997 <br />  7                              24                                 0.001345                               54                                0.001346                            0.7867 <br />  8                              18                                 0.001380                               46                                0.002016                            0.1393 <br />  9                              16                                 0.001287                               25                                0.002157                            0.0283 <br /> 10                              18                                 0.001329                               29                                0.002540                            0.0073 <br /> . 10                            72                                 0.009365                              125                                0.009740                            0.9772 <br /> Total                          836                                 0.001214                            3,290                                0.000715 <br />  <br />   p-value computed tailed Mann-Whitney test. proteins considered, partial rank correlation protein essentiality betweenness control connectivity  0.089 (p , 10 7). result indicates centrality measured betweenness higher nonessential proteins essential proteins control connectivity. <br /> DOI: 10.1371/journal.pgen.0020088.t002 <br />  <br />  <br /> patterns interacting proteins [21].                                    loss edge.  evidence essential removing type hub increases network                                          PPIs important nonessential PPIs maintain- <br /> diameter removing type, types                                        ing network architecture. similar essentiality [21,22]. <br />    argue essentiality PPI                                  Yeast PPI Network Functionally Robust special location network removing                                         Random Networks <br /> essential PPI disturb network architecture                                        said scale-free networks robust removing nonessential PPI. Unfortunately, unknown                                        random removals nodes, majority nodes certainty PPIs essential yeast network.                                    poorly connected, play relatively unimportant roles IBEPs essential, removing IBEPs                                           organizing global network structure [3]. PPI <br /> expected increase network diameter                                              networks factor determining protein essentiality removing non-IBEPs, essential PPIs important                                         essential PPIs, possible examine PPI network nonessential PPIs maintaining network architec-                                      structured particularly robust fashion. Based ture.  trend (Figure 3A).                                      estimates network rewiring linear <br /> removing IBEPs generates fewer unreachable pairs nodes                                        regression, assume 220 edges (3% edges) removing non-IBEPs (Figure 3B). probably                                            yeast PPI network essential. randomly assign 220 IBEPs tend occur highly connected                                             essential edges yeast PPI network, average 368 nodes <br /> nodes, affected lowly connected nodes                                        essential (Figure 4A). connectivity distribution <br />  <br />  <br /> Table 3. Normalized Closeness Essential Nonessential Proteins Yeast PPI Network <br />  <br /> Connectivity (k)                 Essential Proteins                                                     Nonessential Proteins <br />                                  Number Proteins                    Mean Closeness                   Number Proteins                     Mean Closeness                   p-Valuea <br />  <br />  1                               243                                   0.145943                         1,383                                  0.157117                          0.0260 <br />  2                               170                                   0.180588                           784                                  0.192916                         ,0.0001 <br />  3                               119                                   0.192545                           401                                  0.202836                          0.0025 <br />  4                                76                                   0.200897                           238                                  0.207470                          0.0093 <br />  5                                51                                   0.208483                           118                                  0.212829                          0.1196 <br />  6                                29                                   0.218147                            87                                  0.215018                          0.2344 <br />  7                                24                                   0.211802                            54                                  0.216865                          0.2358 <br />  8                                18                                   0.211058                            46                                  0.224958                          0.0006 <br />  9                                16                                   0.216784                            25                                  0.228474                          0.1147 <br /> 10                                18                                   0.211258                            29                                  0.227951                          0.0019 <br /> .10                               72                                   0.236238                           125                                  0.236972                          0.8113 <br /> Total                            836                                   0.184768                         3,290                                  0.184522 <br />  <br />   p-value computed tailed Mann-Whitney test. proteins considered, partial rank correlation protein essentiality closeness control connectivity  0.093 (p , 10 8). result indicates centrality measured closeness higher nonessential proteins essential proteins control connectivity. <br /> DOI: 10.1371/journal.pgen.0020088.t003 <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                              0830                                               June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                      Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  <br />  <br />  <br />                                                                                  Figure 4. Robustness PPI Networks <br /> Figure 3. Effects Random Removal Edges Global Structure            Numbers essential nodes generated 220 essential edges Yeast PPI Network                                                            various networks. Black gray bars depict distribution                                                                                  number essential nodes 10,000 replications random assign- <br />   Effects network diameter, mean shortest path length          ments 220 essential edges real yeast PPI network simulated reachable pairs nodes network.                               ER networks, respectively. ER network number nodes <br /> (B) Effects proportion unreachable pairs nodes               edges real network, distribution node <br /> network. Note total number IBEPs 807 network.              connectivity follows Poisson distribution. shown minimal <br /> DOI: 10.1371/journal.pgen.0020088.g003                                           maximal numbers essential nodes produced 220 essential <br />                                                                                  edges possible network numbers nodes                                                                                  edges yeast PPI network. minimum 22, number <br /> does follow power-law scale-free networks,                     edges 22 nodes high 21 3 22/2 &#194;&#188; 231 . 220. follows Poisson distribution Erdo&#194;&#168;s-Re&#194;&#180;nyi (ER)                        maximum 220 3 2 &#194;&#188; 440. <br />                                                                                  (B) Proportions essential nodes generated given numbers random networks [23], average 417 essential nodes                       essential edges scale-free (power-law) ER networks. networks <br /> result 220 essential edges (Figure 4A). fact,                        contain 4,000 nodes 4,352 edges. scale-free network expected number essential nodes generated given                          node connectivity following power-law distribution P(k) } k  c,                                                                                  P(k) probability node k edges. used c &#194;&#188; 2.29, number essential edges lower scale-free                          real yeast PPI network  Figure S4). ER network networks ER networks (Figure 4B). suggest                       connectivity distribution following Poisson distribution mean scale-free network robust ER                           connectivity node 2.176. result essential nodes <br />                                                                                  produced ER networks scale-free networks given <br /> network, consider underlying mechanism                       number essential edges applies c values  Figure S5). <br /> node essentiality. Note interpretation                         DOI: 10.1371/journal.pgen.0020088.g004 <br /> network robustness different previous analyses. previous investigations, robustness measured terms network structure [3], measured network                        ness [6]. likely, robustness emerged byproduct function. caution higher robustness scale-                    evolutionary processes contingencies. Furthermore, <br /> free yeast PPI network ER networks does imply                      interesting note yeast PPI network far robustness originated natural selection robust-                     robust network possible. instance, design <br />  <br />       PLoS Genetics | www.plosgenetics.org                                0831                                      June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br />  network 220 essential edges link 22 essential nodes              average 1.01 essential partners, signi&#239;&#172;?cantly (Figure 4A). Obviously, evolution did work way.                 expected number (0.94) estimated randomly rewired <br />                                                                             networks (p &#194;&#188; 0.002). analysis suggests essential PPIs <br /> Caveats                                                                     potentially contribute higher essentiality singletons <br />    analysis based PPI data Comprehensive               duplicates, supporting view singleton genes Yeast Genome Database [13]. examine results                  intrinsically important duplicate genes [29]. similar different yeast PPI datasets used, tried datasets, nodes                      Implications <br /> edges [24] fewer nodes edges                       biological networks networks, <br /> [21]. using simulated networks using linear               different edges different levels importance. <br /> regression gave similar estimates b given                    Treating edges quantitatively qualitatively <br /> dataset, different datasets provided different esti-               different way reveal previously unknown patterns mates (Figures S1 S2). results unexpected,                provide new insights. work, propose concept given datasets used vary greatly                   essential protein interactions demonstrate computa- <br /> numbers nodes edges, mean connectivity,                          tional network analysis large faction gene <br /> proportion essential nodes. variations re&#239;&#172;&#8218;ect                      essentiality essential PPIs. important stress <br /> different numbers false-negative false-positive data                 using essential PPIs explain gene essentiality protein essentiality PPI different datasets.                tautological, explanation provides molecular noise incompleteness data potentially                  understanding certain genes essential offers undermine ability predict PE.  long                    conceptual framework future experimental proofs. <br /> essential PPIs randomly distributed edges                 Logically, question essential PPIs essential. essentiality-causing factors affect nodes equally            essential PPIs likely occupy <br /> random fashion, Equation 1 work. fact,                    central locations PPI network nonessential PPIs. <br /> congruence estimates b simulated                   essentiality PPI does determined <br /> networks regression analysis datasets              network structures particular functions <br /> strongly suggest explanation cause protein               interaction. Alternatively, in&#239;&#172;&#8218;uence network <br /> essentiality largely correct. assumption                  architecture subtle require false-negative false-positive PPIs randomly distrib-                scrutiny larger accurate PPI data. Similarly, uted network, false-negative PPIs affect                    results suggest simpler explanation centrality- essential nonessential PPIs affected                 lethality rule does invoke role protein hubs extent. contrary, false-positive PPIs lead                organizing global network structure. Furthermore, underestimation  number essential PPIs               hypothesis quantitatively explains centrality-lethality affected, total number PPIs in&#239;&#172;&#8218;ated.              rule, network architecture hypothesis lacks predictions con&#239;&#172;?rmed simulation 50%                   quantitative model. &#239;&#172;?nding appears argue yeast PPIs randomly removed added.                         biological signi&#239;&#172;?cance PPI network architecture. <br /> &#239;&#172;?ndings suggest estimated dataset                       pointed gene <br /> minimal false-positive PPIs [21] accurate.                      essentiality important phenomenon deter- <br /> Nonetheless, dataset contains fewer nodes                mines organismal survival reproduction, signi&#239;&#172;?cance datasets estimated                         network architecture lie aspects applicable subset nodes. recent study                  cellular life explored. Furthermore, pure high-throughput yeast hybrid data PPIs showed                 analysis focuses PPI networks, unclear weaker centrality-lethality relationship previously              results extend biomolecular networks.  better corroborated data [25]. result expected                 fore, role network architecture biology pure high-throughput yeast hybrid data                      dismissed time.  studies contain high proportions false-positive PPIs, resulting             needed nascent &#239;&#172;?eld systems biology address lower (e.g, 1.2% Ito et al.&#226;&#8364;&#8482;s data [26]) consequently           important questions biological meaning evolu- <br /> weaker in&#239;&#172;&#8218;uence k PE  Equation 1).                                tionary origin architecture robustness biological <br />    known singleton genes likely              networks [7,30&#226;&#8364;&#8220;32]. <br /> essential duplicate genes [4,27,28]. interesting ask singletons likely duplicates engage                Materials Methods essential interactions.  singletons                                                                                yeast PPI data downloaded ftp://ftpmips.gsf yeast/ <br /> duplicates form separate PPI networks,                     PPI. self- interactions contain important biological <br /> impossible estimate separate values                   information, considered analysis, mainly  potential functional compensations duplicates                 approach using IBEPs infer essential interactions                                                                             work self-interactions. centrality-lethality rule mask true essentiality duplicate gene.               observed self-interactions excluded, analysis nonessential duplicate genes actually essential               biologically meaningful. excluded analysis 43 <br /> PPIs. avoid problems, classify genes single-               interactions involving Ty elements involving mitochondrial <br /> tons duplicates examine interaction partners,                 genes, resulting 7,356 non-redundant PPIs linking 4,126 yeast <br />                                                                             nuclear genes, 836 genes essential. mean ignoring essentiality genes                connectivity protein 3.57. Yeast genes subject yeast duplicate genes average 0.89                       single-gene deletion studies listed  http://www-deletion. <br /> essential partners, signi&#239;&#172;?cantly fewer expected                    stanford.edu/YDPM. Essential genes listed  http://www. <br />                                                                             sequence.stanford.edu/group/yeast_deletion_project/ <br /> number (0.94) estimated 5,000 randomly rewired                         Essential_ORFs.txt. 162 genes protein network <br /> networks (p &#194;&#188; 0.004). contrary, yeast singletons                lacked essentiality information treated  <br />       PLoS Genetics | www.plosgenetics.org                           0832                                     June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                              Essential Protein&#226;&#8364;&#8220;Protein Interactions <br />  <br /> nonessential analyses. strategy rendered ;                      Figure S2. Relationship Probability Protein 0.8% genes network misclassi&#239;&#172;?ed terms gene                         Essential (PE) Connectivity (k) Protein Yeast PPI <br /> essentiality. results virtually identical 162               Network <br /> genes excluded protein network. Essential genes                                                                                        yeast PPI information compiled Han colleagues [21] indispensable growth yeasts YPD-rich media. <br />                                                                                        downloaded  http://www.nature.com/nature/journal/v430/n6995/ set genes apparently fundamental cellular processes yeast, additional genes indispensable                    suppinfo/nature02555.html. 2.493 interactions adverse conditions [33]. Yeast stable protein complex dataset                      1,379 genes (including 530 essential genes).   Observed downloaded Saccharomyces Genome Database (ftp://genome-                           predicted PE values. observed values estimated ftp.stanford.edu/pub/yeast/data_download/literature_curation/                          yeast PPI network. Error bars standard (sampling) error go_protein_complex_slim.tab), contained 188 complexes                            observed values. predicted values computed using <br /> comprising 1,226 genes.                                                                Equation 1 parameters &#194;&#188; 7.4% 6 0.5 % b &#194;&#188; 21.8% 6 1.4 <br />    Singleton genes duplicate genes de&#239;&#172;?ned all                  %, estimated using rewired essentiality-reassigned <br /> BLASTP searches yeast proteins, following [34]. Speci&#239;&#172;?cally, gene                 networks described main text (10,000 replications). (B) considered singleton non-self hits E-value &#194;&#188;                 Linear regression ln(1-PE) k. estimated 0.1. gene considered duplicate non-self                 regression Equation 2 parameters &#194;&#188; 7.3% b &#194;&#188; 24.9%. <br /> hit E-value &#194;&#188; 10 20.                                                                paucity proteins high connectivity,    fruit &#239;&#172;&#8218;y PPI network [35] included 4,579 proteins connected                  seven edges considered counted 6.5 <br /> 4,663 non-self high-con&#239;&#172;?dence interactions. conducted genome-                     edges, edges considered wide all BLASTP search (E-value cutoff &#194;&#188; 10 10)                    8.5 edges. Proteins   10 edges (; 8% proteins) 5,773 yeast 13,434 fruit &#239;&#172;&#8218;y proteins, downloaded                   considered paucity data k. <br /> Saccharomyces Genome Database (http://www.yeastgenome.org)                         DOI: 10.1371/journal.pgen.0020088.sg002 (11 KB PDF). <br /> ENSEMBL (http://www.ensembl.org), respectively. 1,764 reciprocal <br /> best hits  considered orthologous proteins                 Figure S3. Similarity Node Connectivity Yeast PPI species. control fact essential genes                  Network Simulated Networks   Essential (B) Non- <br /> tend evolutionarily conserved, examined yeast PPIs                 essential Nodes partners orthologs fruit &#239;&#172;&#8218;y. 1,764                construct simulated networks, removed node <br /> proteins form 1,066 PPIs yeast 156 PPIs fruit &#239;&#172;&#8218;y.                    essentiality information real network reassigned <br />    Network parameters diameter, closeness, betwe-                      node essentiality step random fashion  main text). enness calculated using software Pajek,                         mean frequencies shown 10,000 simulated networks. <br /> loaded  http://vlado.fmf.uni-lj.si/pub/networks/pajek. node <br /> connectivity yeast PPI network approximated                         DOI: 10.1371/journal.pgen.0020088.sg003 (10 KB PDF). <br /> power-law distribution parameter c &#194;&#188; 2.29 (Figure S4). simulate scale-free (power-law) network parameter c                         Figure S4. Frequency Distribution Connectivity (k) Protein Figure 4B), &#239;&#172;?rst computed P(k), expected frequency nodes                     Yeast PPI Network Follows Power-Law P(k) } k  2.29 k edges (k &#194;&#188; 1, 2, 3, . . .), using P(k) &#194;&#188; ak  c, constant           network contains 4,126 nodes connected 7,356 edges. <br /> determined                                                                          DOI: 10.1371/journal.pgen.0020088.sg004 (41 KB PDF). <br />                                        X&#226;&#8364;&#732; <br />                                &#194;&#188; 1=      P&#195;&#176;k&#195;&#382;:                         &#195;&#176;4&#195;&#382;           Figure S5. Proportions Essential Nodes Generated Given <br />                                      k&#194;&#188;1                                               Numbers Essential Edges Scale-Free ER Networks decided connectivity 4,000 nodes                     scale-free network node connectivity following network following P(k) distribution randomly paired                      power-law distribution, ER network node connectiv- nodes considering connectivity. generating                         ity following Poisson distribution.   Comparison corresponding ER network, randomly paired 4,000 nodes                     power-law network c &#194;&#188; 2 ER network. networks total number edges reached corresponding scale-                     contain 4,000 nodes 5,995 edges randomly generated <br /> free network.                                                                          following respective connectivity distributions. (B) Comparison <br />                                                                                        power-law network c &#194;&#188; 2.5 ER network. <br />                                                                                        networks 4,000 nodes 3,620 edges. <br /> Supporting Information                                                                 DOI: 10.1371/journal.pgen.0020088.sg005 (130 KB PDF). <br /> Figure S1. Relationship Probability Protein Essential (PE) Connectivity (k) Protein Yeast PPI                Table S1. Probably Essential PPIs Yeast <br /> Network                                                                                DOI: 10.1371/journal.pgen.0020088.st001 (53 KB PDF). yeast PPI information downloaded GRID (General                            Table S2. Conserved PPIs Yeast Fruit Fly <br /> Repository Interaction Datasets) [24] http://biodata.mshri ca/ <br /> yeast_grid/&#239;&#172;?les/Full_Data_Files/interactions.txt. excluding                      DOI: 10.1371/journal.pgen.0020088.st002 (70 KB PDF). <br /> self-interactions interactions involving Ty elements mito- <br /> chondrial genes, total 13,189 physical PPIs connecting 4,674 <br /> genes (including 972 essential genes) obtained.   Observed                  Acknowledgments <br /> predicted PE values. observed values estimated yeast PPI network. Error bars standard (sampling) error                    thank members Zhang lab discussions Wendy Grus, observed values. predic <br /> </body></html>