<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOBNAME: RNA 12#7 2006 PAGE: 1 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />     Use targetrons disrupt essential nonessential <br />     genes Staphylococcus aureus reveals temperature <br />     sensitivity Ll.LtrB group II intron splicing <br />  <br />  <br />     JUN YAO,1 JIN ZHONG,1 YUAN FANG,2 EDWARD GEISINGER,2 RICHARD P. NOVICK,2 <br />     ALAN M. LAMBOWITZ1 <br />     1 <br />      Institute Cellular Molecular Biology, Department Chemistry Biochemistry, Section Molecular Genetics <br />     Microbiology, School Biological Sciences, University Texas Austin, Austin, Texas 78712, USA <br />     2 <br />      Skirball Institute Biomolecular Medicine, Program Molecular Pathogenesis Departments Microbiology Medicine, <br />     New York University Medical Center, New York, New York 10016, USA <br />  <br />  <br />  <br />  <br />     ABSTRACT <br />     targetron based Lactococcus lactis Ll.LtrB group II intron used efficient chromosomal gene <br />     disruption human pathogen Staphylococcus aureus. Targetrons expressed derivatives vector pCN37, uses <br />     cadmium-inducible promoter, pCN39, derivative pCN37 temperature-sensitive replicon, gave site-specific <br />     disruptants hsa seb genes 37%&#226;&#8364;&#8220;100% plated colonies selection. disrupt hsa, essential gene,     used group II intron integrates sense orientation relative target gene transcription removed     RNA splicing, enabling production functional HSa protein. splicing Ll.LtrB intron     intron-encoded protein temperature-sensitive, method yields conditional hsa disruptant grows 32&#194;&#176;C     43&#194;&#176;C. temperature sensitivity splicing reaction suggests general means obtaining step conditional disruptions <br />     organism. nature, temperature sensitivity group II intron splicing limit temperature range organism <br />     containing group II intron inserted essential gene. <br />     Keywords: functional genomics; gene targeting; retrotransposon; reverse transcriptase; ribozyme <br />  <br />  <br />  <br />  <br />     INTRODUCTION                                                                      tion. Unfortunately, gene targeting S. aureus                                                                                       hampered low efficiency homologous <br />     Staphylococcus aureus facultative pathogen, responsible <br />                                                                                       recombination. traditional strategies,     infections suffered hospital patients                                                                                       <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> transposon mutagenesis, recently developed <br />     increasingly implicated community infections <br />                                                                                       methods inactivating genes S. aureus use antisense <br />     (Fridkin et al. 2005). danger pathogen reflects                                                                                       RNAs (Kernodle et al. 1997) counterselection rare <br />     S. aureus adapt widely varying conditions, including <br />                                                                                       recombination events, e.g., using dominant strepto- <br />     hostile environment infected animal tissues,                                                                                       mycin-sensitivity allele (Chen Novick 2006)     develop resistance antibiotics. present, <br />                                                                                       antisense RNA blocking expression essential gene, <br />     urgent need understand pathobiology                                                                                       secA (Bae Schneewind 2006). methods     organism develop new anti-bacterial agents                                                                                       advantages disadvantages. Transposon mutagenesis     counter  major approach objectives                                                                                       fairly easy, transposons hot spots useless <br />     identify targetable metabolic processes evaluating                                                                                       essential genes. Counterselection strategies require labo- <br />     function specific genes targeted gene inactiva- <br />                                                                                       rious gene-specific constructions   useless                                                                                       essential genes. Antisense RNAs target gene                                                                                       significant failure rate  repress        Reprint requests  Alan M. Lambowitz, Institute Cellular     Molecular Biology, Department Chemistry Biochemistry, Section           eliminate gene function, reveal essentiality. <br />     Molecular Genetics Microbiology, University Texas Austin,              different strategy overcomes     Austin, TX 78712, USA; e-mail: lambowitz@mail.utexas.edu; fax: (512)              disadvantages gene targeting mobile group II <br />     232-3420. <br />        Article published online ahead print. Article publication <span id='am-5' about='oboInOwl:date' typeof='owl:Thing'>date</span>   introns (&#226;&#8364;&#732;&#226;&#8364;&#732;targetrons&#226;&#8364;&#8482;&#226;&#8364;&#8482;)  review, Lambowitz et al. <br />     http://www.rnajournal.org/cgi/doi/10.1261/rna.68706.                              2005). targetron based Lactococcus lactis Ll.LtrB <br />  <br />     RNA (2006), 12:1271&#226;&#8364;&#8220;1281. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2006 RNA Society.                                 1271 <br />  <br />  <br />  <br />  <br />   rna687 Yao et al. ARTICLE RA <br />  JOBNAME: RNA 12#7 2006 PAGE: 2 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />    intron used targeted gene disruption site-       algorithm generally identifies multiple rank-ordered target <br />    specific DNA insertion Escherichia coli Gram-        sites gene. required modifications intro- <br />    negative enterobacteria (Karberg et al. 2001; Perutka et al.      duced donor plasmid PCR (Guo et al. 2000; <br />    2004), Gram-positive bacteria L. lactis     Karberg et al. 2001). E. coli, targetrons commonly insert <br />    Clostridium perfringens (Frazier et al. 2003; Chen et al.         frequencies 1%&#226;&#8364;&#8220;80%, insertions detected <br />    2005). utility group II introns gene targeting         colony PCR incorporating genetic marker <br />    derives unique DNA integration mechanism            intron (Zhong et al. 2003; Perutka et al. 2004). <br />    intron RNA uses ribozyme activity insert          targetrons potentially used disrupt gene <br />    (reverse splice) directly DNA target site      sequence available. <br />    reverse-transcribed intron-encoded protein (IEP)              addition targeted gene disruption, targetrons     review, Lambowitz Zimmerly 2004).                used insert (&#226;&#8364;&#732;&#226;&#8364;&#732;knock &#226;&#8364;&#8482;) cargo genes site-specifically <br />    mobility reactions carried ribonucleoprotein         desired chromosomal locations (Frazier et al. 2003)    (RNP) particle formed RNA splicing             introduce targeted double-strand breaks    contains IEP excised intron lariat RNA. RNPs              repaired homologous recombination cotrans- <br />    initiate mobility using IEP base-pairing          formed DNA fragment, enabling introduction point <br />    intron RNA recognize DNA target sequences.          mutations (Karberg et al. 2001). Additionally, targetron <br />    base-pairing interactions span 13- 16-nt region           randomized target-site recognition sequences    DNA target site, involve intron RNA sequences             used generate library random disruptants, analogous <br />    denoted exon-binding sites 1 2 (EBS1 EBS2) d          global transposon mutagenesis (Zhong et al. 2003). <br />     sequence adjacent EBS1) DNA target site                 Individual targetrons isolated library <br />    sequences denoted intron-binding sites 1 2 (IBS1          DNA PCR used modification obtain <br />    IBS2) 59-exon, d9 39-exon             desired single disruptants (Yao et al. 2005). <br />    (Eskes et al. 1997; Guo et al. 1997, 2000; Mohr et al. 2000).        principle, targetrons used obtain    base-pairing interactions contribute        unconditional conditional disruptions, including condi- <br />    DNA target specificity, feature makes possible     tional disruptions essential genes.    retarget group II introns insert desired sites simply     targetron integrate orientation <br />    modifying EBS1, EBS2, d sequences               relative target gene transcription selecting target <br />    intron RNA (Lambowitz et al. 2005).                               sequences sense antisense strands. targetron <br />       bacteria, targetrons generally expressed           inserted antisense orientation transcribed    donor plasmid, RNPs assembled intracellularly           complement intron RNA sequence, spliced, <br />    (Guo et al. 2000; Karberg et al. 2001). commonly used         yields unconditional disruption. contrast, tar- <br />    E. coli donor plasmid pACD3 specifies precursor RNA             getron inserted sense orientation transcribed    containing 0.9-kb Ll.LtrB-DORF intron short flank-          functional intron RNA potentially spliced    ing exon sequences, IEP (denoted LtrA protein)           precursor RNA IEP, restoring mRNA (Karberg et <br />    expressed position just downstream 39-exon          al. 2001; Frazier et al. 2003). splicing reaction    (Guo et al. 2000; Karberg et al. 2001). IEP binds          dependent IEP, suggested conditional dis- <br />    intron precursor RNA, promotes splicing            ruptants obtained controlling expression    stabilizing catalytically active RNA structure,        IEP separate construct inducible promoter <br />    remains bound excised intron lariat RNA RNPs            (Karberg et al. 2001; Frazier et al. 2003). <br />    promote mobility. mobility, Ll.LtrB intron               constructed targetron donor plasmids    reverse-splices strand target DNA,      S. aureus used disrupt essential hsa <br />    IEP cleaves opposite strand uses 39 DNA end           gene nonessential seb gene high frequencies <br />    cleavage site primer reverse transcription     selection. hsa gene disrupted using <br />    inserted intron RNA. resulting intron cDNA             targetron inserts sense orientation relative    integrated recipient DNA using host DNA               target gene transcription spliced IEP- <br />    replication repair enzymes (Matsuura et al. 1997;             assisted manner precursor RNA, resulting    Cousineau et al. 1998; Smith et al. 2005).                        production functional HSa protein.        Ll.LtrB intron targeted desired sites aid   IEP-assisted splicing reaction inherently tem- <br />    algorithm, scans target sequence          perature-sensitive, consequently, procedure yields <br />    best matches positions recognized IEP          conditional hsa disruptant grow 32&#194;&#176;C    designs PCR primers optimal base-pairing             43&#194;&#176;C. results demonstrate time    interactions intron RNA&#226;&#8364;&#8482;s EBS1, EBS2, d           targetrons used disrupt essential genes,    sequences IBS1, IBS2, d9 sequences DNA         suggest novel general strategy obtaining step <br />    target site (Perutka et al. 2004). positions recognized       conditional gene disruptions organism. nature, <br />    IEP sufficiently flexible             temperature sensitivity group II intron splicing <br />  <br />    1272         RNA, Vol. 12,  7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 3 OUTPUT: Wednesday June 7 03:20:16 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                          Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />     reactions limit temperature range organisms           night 37&#194;&#176;C, colonies screened targetron  <br />     containing group II introns inserted essential genes.       sertion PCR, using primers flanking predicted <br />                                                                        insertion site target gene. initial hsa disruption <br />                                                                        avirulent S. aureus strain RN4220. seb <br />     RESULTS <br />                                                                        disruption strain RN8908, contains                                                                        SaPI3 pathogenicity island encoding SEB, condition- <br />     Use Ll.LtrB targetron gene disruption <br />                                                                        ing donor plasmid passage RN4220,     S. aureus <br />                                                                        does contain SaPI3  Materials Methods). <br />     gene targeting S. aureus, constructed intron-               Pcad promoter low basal level transcription <br />     donor plasmids patterned intron-donor plasmid            absence cadmium (Charpentier et al. 2004),     pACD3 used E. coli (Guo et al. 2000; Karberg et al. 2001).      outset, clear cadmium induction <br />     pNL9161 contains 0.9-kb Ll.LtrB-DORF intron flank-           necessary obtain disruptants.  hsa, <br />     ing exons cloned cadmium-inducible promoter               PCR-screened 10 colonies cells cadmium <br />     (Pcad) S. aureus vector pCN37 (Charpentier et al.           induction 10 desired disruption <br />     2004). LtrA ORF deleted normal location            (Fig. 3A, left panel). Insertion targetron correct <br />     intron inserted just downstream 39-          site confirmed sequencing PCR products (data <br />     exon. E. coli, Ll.LtrB-DORF intron susceptible   shown). seb, 24 colonies cells <br />     nuclease degradation length intron, leading    induced cadmium desired <br />     substantially higher integration frequencies (Guo et al. 2000).    disruption, cadmium induction, 24 <br />        candidate genes chosen test intron targeting        colonies desired disruption (Fig. 4A; left panel <br />     S. aureus: hsa, encodes member HU                shows PCR data subset colonies). correct <br />     family bacterial histone-like proteins (Viter et al. 1999),     disruption confirmed sequencing colony <br />     seb, encodes enterotoxin B (Shafer Iandolo           PCR products (data shown). hsa seb,     1978).  analyzed genes algo-            randomly chosen disruptants analyzed Southern hybrid- <br />     rithm identify potential insertion sites Ll.LtrB        ization using intron probe showed just single band     targetron  Materials Methods), selected             size expected integration desired target site, <br />     single target sites potential target sites        larger band corresponding donor <br />     seb potential target sites hsa.         plasmid, additional bands nonspecific <br />     laboratories independently failed obtain hsa             insertions (Figs. 3A, 4A, right panels figure legends <br />     knockout   Cheung, pers. comm.; Y. Fang R.P.                 molecular weight range spanned blot).     Novick, unpubl.), suspected gene             experiments, Hsa-24s targetron used disrupt <br />     essential, selected anti-Hsa targetron       hsa gene S. aureus strain RN7206     inserts sense orientation potentially          plasmid-carrying derivatives RN4220 RN7206,     removed RNA splicing, enabling production                disruption small number <br />     functional HSa protein. additional challenge             colonies analyzed (data shown). <br />     hsa gene small size (270 bp) (Viter et al. 1999), <br />     limits choice target sites. <br />                                                                        Ll.LtrB gene targeting S. aureus using <br />        Donor plasmids containing Ll.LtrB targetrons insert <br />                                                                        temperature-sensitive vector <br />     selected sites hsa seb genes     constructed PCR using primers designed algo-            donor plasmid expresses LtrA protein,     rithm modify intron&#226;&#8364;&#8482;s EBS1, EBS2, d sequences           potentially remobilize splice inserted Ll.LtrB- <br />     base-pair optimally DNA target site sequences IBS1,          DORF intron, desirable cure disruptants <br />     IBS2, d9. IBS1 IBS2 sequences 59-exon           prior analysis. facilitate curing donor plasmid, <br />     donor plasmid modified comple-              recloned Ll.LtrB targetron constructs vector <br />     mentary intron&#226;&#8364;&#8482;s retargeted EBS1 EBS2 sequen-           backbone pCN39, temperature-sensitive <br />     ces efficient RNA splicing donor construct        replicon (Fig. 1B). resulting donor plasmid containing <br />     Fig. 1; Materials Methods). pCN37-based donor              wild-type Ll.LtrB intron denoted pNL9162,     plasmids containing targetrons Hsa-24s Seb-123a            donor plasmids containing targetrons Hsa-24s Seb- <br />     denoted pNL9163 pNL9165, respectively (Table 1).               123a denoted pNL9164 pNL9166, respectively <br />     predicted base-pairing interactions              (Table 1). donor plasmids gave targeted dis- <br />     targetrons target sites shown Figure 2.           ruptions hsa seb genes high <br />        S. aureus strains containing donor plasmids            efficiency specificity pCN37-based donor plasmids <br />     grown early log phase plated directly        (Figs. 3B, 4B). Hsa-24s targetron gave 100% <br />     BHI medium containing erythromycin induced                 disruptants (24/24) cadmium induction,     CdCl2 prior plating. incubating plates           Seb-123a targetron gave 7/19 disruptants cadmium <br />  <br />                                                                                                         www.rnajournal.org       1273 <br />  JOBNAME: RNA 12#7 2006 PAGE: 4 OUTPUT: Wednesday June 7 03:20:17 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />  <br />    FIGURE 1. Intron-donor plasmids S. aureus gene targeting.   pNL9161 derivative S. aureus expression vector pCN37,    employs cadmium-inducible promoter (Pcad) cadmium regulator (cadR) express precursor RNA containing Ll.LtrB-DORF <br />    intron flanking 59- 39-exon sequences (E1 E2, respectively), plus LtrA ORF downstream E2. plasmid contains <br />    staphylococcal pT181 replicon colE1 replicon, erythromycin-resistance (ermC) ampicillin-resistance (ampR) genes. (B) <br />    pNL9162 derivative S. aureus expression vector pCN39 similar pNL9161  , contains temperature- <br />    sensitive replicon (pT181 cop-634ts repC4) place wild-type pT181 replicon. (C) PCR reactions used modify targetron insertion <br />    different genes. targeting, EBS1, EB2, d sequences Ll.LtrB-DORF intron complementary IBS1, IBS2, d9    DNA target site, IBS1 IBS2 E1 donor plasmid complementary intron&#226;&#8364;&#8482;s retargeted EBS1 EBS2 sequences <br />    efficient RNA splicing. modifications introduced donor plasmid step PCR (PCR1 2), using primers p1 + p2    p3 + p4, diagrammed figure described Materials Methods. PCRs yield 353-bp HindIII/BsrGI fragment,    swapped corresponding fragment donor plasmid. <br />  <br />  <br />    induction (Figs. 3B, 4B; left panels PCR data                    determine splicing reaction tempera- <br />    subset colonies). pCN39-based donor plasmid                   ture-sensitive, splicing Ll.LtrB-DORF intron <br />    readily cured seb disruptants plating cells                 inserted hsa gene assayed RT-PCR, using <br />    43&#194;&#176;C (Fig. 4B, Southern blot, right-hand lanes).                   primers flanking intron-insertion site (Fig. 5B).    donor plasmid cured hsa                          32&#194;&#176;C, wild-type (WT) strain, predominant <br />    disruptants, presumably IEP-assisted splicing                  product 0.1-kb band corresponding hsa mRNA <br />    inserted intron required produce HSa protein.                 (Fig. 5B, lane 1), band, corresponding    inability cure donor plasmid confirms                       mRNA spliced exons, predominant <br />    essentiality hsa gene.                                              product hsa disruptants obtained donor <br />                                                                               plasmids pNL9163 pNL9164 (Fig. 5B, lanes 2 3, <br />    Targetron insertion creates conditional <br />    hsa disruptant <br />                                                                                TABLE 1. S. aureus intron-donor plasmids used study <br />    noticed hsa disruptants containing pCN37- <br />                                                                                Plasmid              Vector backbone                 Targetron <br />    pCN39-based donor plasmids temperature-sensitive <br />    grow plates 32&#194;&#176;&#226;&#8364;&#8220;41&#194;&#176;C 43&#194;&#176;C (Fig.                  pNL9161                 pCN37                    WT Ll.LtrB-DORF <br />    5A; seb disruptants temperature-sensitive; data                pNL9162                 pCN39 (ts              WT Ll.LtrB-DORF <br />                                                                                pNL9163                 pCN37                    Hsa-24s <br />    shown). temperature sensitivity hsa                  pNL9164                 pCN39 (ts)               Hsa-24s <br />    disruptants observed plates contained                pNL9165                 pCN37                    Seb-123a <br />    erythromycin force retention donor plasmid                   pNL9166                 pCN39 (ts)               Seb-123a <br />    supplies IEP, suspected reflect                                                                                                    ts, temperature-sensitive. <br />    temperature sensitivity IEP-assisted splicing reaction. <br />  <br />    1274         RNA, Vol. 12,  7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 5 OUTPUT: Wednesday June 7 03:20:21 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                 Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />                                                                               RT-PCR analysis wild-type Ll.LtrB-DORF intron tran- <br />                                                                               scripts expressed donor plasmids pACD2X E. coli <br />                                                                               pNL9161 S. aureus. shown Figure 6,                                                                               observed similar inhibition splicing 1 1.5 h <br />                                                                               temperature shift 30&#194;&#176;C (E. coli) 32&#194;&#176;C <br />                                                                               (S. aureus) 37&#194;&#176;C 43&#194;&#176;C, accumulation <br />                                                                               unspliced precursor RNA decrease level                                                                               spliced-exon RNA longer incubation higher <br />                                                                               temperature. experiments, real-time RT-PCR <br />                                                                               showed splicing length wild-type Ll.LtrB <br />                                                                               intron expressed plasmid pACD-LtrB E. coli <br />     FIGURE 2. DNA target sites base-pairing interactions     Hsa-24s Seb-123a targetrons. figure shows DNA target site         HMS174(DE3) temperature-sensitive (unspliced pre- <br />     sequences positions &#195;&#191;30 +15 intron-insertion site        cursor RNA, 2.57 6 0.05 32&#194;&#176;C 11.1 6 0.6 43&#194;&#176;C; <br />     intron RNA/DNA target site base-pairing interactions          spliced exons, 4.2 6 0.2 32&#194;&#176;C 2.10 6 0.05 43&#194;&#176;C, <br />     wild-type (WT) Ll.LtrB intron targetrons Hsa-24s Seb-123a, <br />     designed insert S. aureus hsa seb genes, respectively. <br />                                                                               overnight incubation indicated temperatures). <br />     Targetrons named <span id='am-6' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> position 59 insertion      IEP-assisted splicing reaction Ll.LtrB group <br />     site target gene&#226;&#8364;&#8482;s coding sequence, followed &#226;&#8364;&#732;&#226;&#8364;&#732;s&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732; &#226;&#8364;&#8482;     II intron appears inherently temperature-sensitive. <br />     indicating sense antisense strands, respectively. intron  <br />     sertion site strand   strand cleavage site <br />     IEP (CS) indicated arrowheads. Nucleotide residues       DISCUSSION <br />     DNA target site match wild-type Ll.LtrB intron target site <br />     highlighted gray.                                                  Ll.LtrB targetron used                                                                               efficient chromosomal gene disruption human <br />                                                                               pathogen S. aureus. Targetron Hsa-24s, used disrupt <br />     respectively). disruptants lighter 1.0-kb                 hsa gene, exceptionally efficient, giving 100% <br />     band corresponding unspliced precursor RNA,                      disruptant colonies, cadmium induction     prominent cells containing pNL9163                        targetron transcription, targetron Seb-123a, used     pCN37-based donor plasmid) (Fig. 5B, lane 2)                      disrupt seb gene encoding enterotoxin B, somewhat <br />     containing pNL9164  pCN39-based donor plas-                     efficient, giving z37% disruptant colonies     mid) (Fig. 5B, lane 3). difference reflect different             cadmium induction. difference frequency     plasmid copy numbers (20&#226;&#8364;&#8220;25 300&#226;&#8364;&#8220;400 copies                        Hsa-24s Seb-123a targetrons likely reflects primarily <br />     cell vectors pCN37 pCN39, respectively)                       retargeted introns recognize new target <br />     (Charpentier et al. 2004), resulting different levels               sites, seen previously different targetrons E. coli <br />     expression IEP needed precursor RNA splicing.                  (Perutka et al. 2004). Ll.LtrB targetron used <br />        Strikingly, cells grown 32&#194;&#176;C shifted               previously targeted gene disruption     43&#194;&#176;C 2 h, wild-type strain showed single band             Gram-positive bacteria, L. lactis C. perfringens.     corresponding hsa mRNA (Fig. 5B, lane 4),                    L. lactis, different Ll.LtrB targetrons expressed     disruptants showed substantial accumulation larger                 nisin-inducible promoter gave disruptants mleS gene <br />     band corresponding unspliced precursor RNA,                        100% screened colonies, targetrons <br />     concomitant reduction intensity band corre-                 used site-specifically insert cargo gene cloned <br />     sponding mRNA spliced exons (Fig. 5B,                         intron  abiD gene conferring phage  <br />     lanes 5,6). Quantitation precursor mRNA levels                  sistance) (Frazier et al. 2003). C. perfringens, Ll.LtrB <br />     real-time RT-PCR confirmed accumulation pre-                       targetron expressed constitutive promoter gave <br />     cursor RNA reduction hsa mRNA levels                        disruptants toxin gene (plc) 38 screened <br />     disruptants shift 43&#194;&#176;C (Table 2).             colonies (Chen et al. 2005). Based examples,     findings indicate temperature sensitivity                 likely Ll.LtrB targetron generally useful <br />     hsa disruptants temperature sensitivity                  gene targeting Gram-positive bacteria. <br />     IEP-assisted splicing reaction inserted                           results time targetron     Ll.LtrB-DORF intron.                                                      inserts sense orientation removed                                                                               precursor RNA RNA splicing used disrupt <br />                                                                               essential gene.  IEP-assisted splic- <br />     IEP-assisted splicing wild-type Ll.LtrB-DORF <br />                                                                               ing reaction Ll.LtrB intron inherently temperature- <br />     intron temperature-sensitive                                                                               sensitive, resulting step conditional <br />     E. coli S. aureus <br />                                                                               disruption. note ability obtain conditional <br />     determine IEP-assisted splicing reaction                disruptant dependent factors, including     inherently temperature-sensitive, carried similar                  splicing efficiency inserted intron, stability  <br />                                                                                                                www.rnajournal.org       1275 <br />  JOBNAME: RNA 12#7 2006 PAGE: 6 OUTPUT: Wednesday June 7 03:20:22 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                                                       principle, temperature sensi- <br />                                                                                                    tivity RNA splicing reaction <br />                                                                                                    reflect temperature sensitivity                                                                                                    IEP, intron RNA,                                                                                                    combination  Previous stud- <br />                                                                                                    ies showed purified LtrA protein                                                                                                    rapidly irreversibly denaturated <br />                                                                                                    incubated splicing reaction <br />                                                                                                    medium 37&#194;&#176;C absence                                                                                                    intron RNA lower temper- <br />                                                                                                    atures favor expression active <br />                                                                                                    LtrA protein E. coli (Saldanha et al. <br />                                                                                                    1999). Additionally, immunoblot ex- <br />                                                                                                    periments E. coli HMS174(DE3) <br />                                                                                                    expressing donor plasmid pACD2X <br />                                                                                                    showed decreased levels soluble LtrA <br />                                                                                                    protein 43&#194;&#176;C (J. Yao M. <br />                                                                                                    Lambowitz, unpubl.).  tem- <br />                                                                                                    perature sensitivity IEP likely <br />                                                                                                    major factor temperature sen- <br />                                                                                                    sitivity splicing reaction.  <br />                                                                                                    based vitro results, expect                                                                                                    temperature sensitivity exacer- <br />                                                                                                    bated IEP expressed                                                                                                    ectopic location bind <br />                                                                                                    immediately intron RNA                                                                                                     Saldanha et al. 1999; Cui et al. 2004). <br />                                                                                                    thermal stability IEP <br />    FIGURE 3. Use Ll.LtrB targetron disrupt S. aureus hsa gene.  B) Disruption    hsa gene S. aureus RN4220 using targetron Hsa-24s expressed pNL9163 intron RNA structure    pNL9164, respectively. steps, cells containing pNL9163 (pCN37-derivative) grown differ different hosts,    37&#194;&#176;C, containing pNL9164 (pCN39-derivative; temperature-sensitive replicon) influenced location <br />    grown 32&#194;&#176;C. Donor plasmids electroporated RN4220, cells grown intron-insertion site    BHI medium containing erythromycin 1 h, plated medium. left <br />    panels colony PCR wild-type (WT) RN4220 disruptants obtained target gene. Consequently,    donor plasmids. right panels Southern blots MfeI-digested donor plasmids necessary define optimal temper- <br />    pNL9163 pNL9164 (left lane) genomic DNA randomly chosen hsa disruptants ature range conditional disruption <br />    obtained corresponding donor plasmid (right lanes). blots hybridized                                                                                                    case. future,    32P-labeled intron probe. bands shown ones visible blots (molecular <br />    weight range 100 bp&#226;&#8364;&#8220;12 kb). schematic shows diagram hsa gene, possible change operational tem- <br />        integrated targetron, including location primers perature range genetically selecting <br />    pf pr used detect targetron integration colony PCR, MfeI sites used variants IEP having greater    Southern hybridization analysis. <br />                                                                                                    lesser temperature sensitivity. <br />                                                                                                       Notably, bacteria contain natu- <br />                                                                                                    ral group II introns inserted essen- <br />    target gene mRNA, gene product                           tial genes (e.g., groEL Azotobacter) (Ferat et al. 2003).    needed sustain growth. factors differ                     determine    different genes different organisms,                     group II introns display temperature-sensitive splicing <br />    possible obtain conditional disruptants cases.                similar Ll.LtrB intron.  temperature <br />    splicing efficiency inserted targetron                    range bacteria grow limited    pendent sequences flanking insertion site,                 inherent temperature sensitivity splicing reaction <br />    EBS/IBS d&#226;&#8364;&#8220;d9 interactions                   inserted group II intron.  sporadic  <br />    intron flanking exon sequences                         sertion group II introns nature rapidly change <br />    flanking exon sequences potentially form obstructing                   fitness organisms grow different temperatures. <br />    RNA structures. Ultimately, possible assess                    bacterial group II introns    parameters systematically obtain general rules                splice elevated temperatures, possibly reflecting adapta- <br />    predicting splicing efficiency targetrons inserted               tion intron host insertion (Chee    different sites.                                                           Takami 2005). <br />  <br />    1276         RNA, Vol. 12,  7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 7 OUTPUT: Wednesday June 7 03:20:26 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                      Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />                                                                                                        sufficient express targetrons levels <br />                                                                                                        highly efficient gene disruption. <br />                                                                                                        caveat studies used <br />                                                                                                        short Ll.LtrB-DORF intron                                                                                                        selectable marker,                                                                                                        indication longer forms  <br />                                                                                                        tron containing genetic marker                                                                                                        expressed efficiently pro- <br />                                                                                                        moters (J. Yao M. Lambowitz, <br />                                                                                                        unpubl.). efficiency translation <br />                                                                                                        IEP, required                                                                                                        RNA splicing intron integration,                                                                                                        parameter affect tar- <br />                                                                                                        getron expression different hosts.                                                                                                        parameter influenced Shine- <br />                                                                                                        Dalgarno sequences                                                                                                        translation initiation site,                                                                                                        codon usage LtrA protein com- <br />                                                                                                        pared optimal host organ- <br />                                                                                                        ism.  work                                                                                                        S. aureus Chen et al. (2005) <br />                                                                                                        C. perfringens Shine-Dal- <br />                                                                                                        garno sequence optimized spe- <br />                                                                                                        cific host. far, targetrons <br />                                                                                                        introduced donor plas- <br />                                                                                                        mids replicate desired host,                                                                                                        possible introduce <br />                                                                                                        conjugation (Belhocine et al. <br />                                                                                                        2004) nonreplicating DNA  <br />     FIGURE 4. Use Ll.LtrB targetron disrupt S. aureus seb gene.  B) Disruption      haps pre-assembled RNPs.     seb gene S. aureus RN8098 using targetron Seb-123a expressed pNL9165               anticipate procedures     pNL9166. steps, cells containing pNL9165 (pCN37-derivative) grown 37&#194;&#176;C,     containing pNL9166 (pCN39-derivative; temperature-sensitive replicon) grown          broadly applicable use targe- <br />     32&#194;&#176;C. Donor plasmids electroporated S. aureus RN4220, reisolated                trons wide variety prokaryotes. <br />     electroporated RN8098  Materials Methods). Cells grown BHI medium <br />     containing erythromycin, induced 10 mM CdCl2 90 min, plated BHI <br />     medium containing erythromycin. left panels colony PCR wild-type RN8098                MATERIALS METHODS <br />     (WT) representative colonies obtained donor plasmids. right panels <br />     Southern blots AseI-digested donor plasmids pNL9165 pNL9166 (left lane)     randomly chosen seb disruptants obtained corresponding donor plasmid (right lane).        Bacterial strains growth <br />     blots hybridized 32P-labeled intron probe. samples rightmost        conditions <br />     lanes Southern blot panel B cells pNL9166 cured     disruptants overnight incubation 43&#194;&#176;C BHI plates. bands shown ones     S. aureus strains RN4220     visible blots (molecular weight range 100 bp&#226;&#8364;&#8220;12 kb). schematic shows    RN8098. RN4220 restriction-defective, <br />     diagram seb gene,     integrated targetron, including <br />                                                                                                      nonlysogenic derivative NCTC8325     location primers pf pr used detect targetron integration colony PCR     location AseI sites used Southern hybridization analysis.                             efficient recipient E. coli DNA (Kreis- <br />                                                                                                      wirth et al. 1983). RN8098 contains                                                                                                      SaPI3 pathogenicity island encodes        methods used adapting targetrons use                    staphylococcal superantigen enterotoxin B (SEB) (Novick et al. <br />     S. aureus  study) C. perfringens (Chen et al. 2005)               2001). S. aureus strains grown brain heart infusion (BHI) <br />     relatively straightforward. Principally, targetron             trypticase soy agar (TSA) media, erythromycin added                                                                               10 mg/mL. Unless specified  cells containing pCN37- <br />     recloned donor plasmid promoter suitable                                                                               based plasmids grown 37&#194;&#176;C, containing pCN39- <br />     expressing intron desired host. E. coli, targetrons <br />                                                                               based plasmids, temperature-sensitive replicon,     expressed phage T7 promoter, using T7                    grown 32&#194;&#176;C. <br />     RNA polymerase introduced lDE3 lysogen                         E. coli strain DH5a used cloning, HMS174 (DE3)     separate plasmid (Guo et al. 2000; Karberg et al. 2001).                used RT-PCR analysis Ll.LtrB intron splicing. E. coli strains <br />      results S. aureus C. perfringens                grown LB medium, ampicillin added 100 mg/mL, <br />     native host promoter, inducible constitutive,            chloramphenicol 25 mg/mL, erythromycin 200 mg/mL. <br />  <br />                                                                                                                     www.rnajournal.org       1277 <br />  JOBNAME: RNA 12#7 2006 PAGE: 8 OUTPUT: Wednesday June 7 03:20:30 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                                                     just downstream 39-exon,                                                                                                     Ll.LtrB-DORF intron additional <br />                                                                                                     phage T7 promoter inserted intron RNA <br />                                                                                                     domain IV use E. coli genetic assays                                                                                                     intron mobility  Guo et al. 2000; Karberg <br />                                                                                                     et al. 2001). pACD-LtrB related plasmid <br />                                                                                                     expresses length wild-type <br />                                                                                                     Ll.LtrB intron LtrA ORF encoded <br />                                                                                                     intron domain IV (Guo et al. 2000). <br />                                                                                                        S. aureus targetron donor plasmids <br />                                                                                                     used work summarized Table 1. <br />                                                                                                     pNL9161 contains Ll.LtrB targe- <br />                                                                                                     tron construct pACD2X cloned  <br />                                                                                                     stream cadmium-inducible promoter <br />                                                                                                     (Pcad) S. aureus vector pCN37 <br />                                                                                                     (Charpentier et al. 2004). construct                                                                                                     plasmid, 3-kb XbaI/XhoI fragment                                                                                                     pACD2X, containing Ll.LtrB-DORF  <br />                                                                                                     tron flanking exon sequences plus                                                                                                     downstream LtrA ORF, gel-purified, <br />                                                                                                     blunt-ended T4 DNA polymerase,                                                                                                     cloned SmaI site pCN37.                                                                                                     facilitate introduction modifications <br />                                                                                                     needed gene targeting, XbaI site <br />                                                                                                     downstream Pcad replaced                                                                                                     NheI site cleaving pCN37 multiclon- <br />                                                                                                     ing site PstI EcoRI, ligating                                                                                                     DNA oligonucleotide containing NheI <br />                                                                                                     site flanked PstI EcoRI sites.                                                                                                     addition, HindIII site pT181 repC <br />                                                                                                     eliminated altering amino <br />                                                                                                     acid sequence (AAGCTT AAGCTA; Lys- <br />                                                                                                     Leu Lys-Leu), BsrGI site  <br />                                                                                                     stream repC removed BsrGI <br />                                                                                                     digestion, followed T4 DNA <br />                                                                                                     polymerase, religation. <br />    FIGURE 5. hsa disruptants temperature sensitive growth splicing Ll.LtrB-         pNL9162 contains Ll.LtrB <br />    DORF intron.   Plating assay. Wild-type RN4220 hsa disruptants obtained               targetron construct cloned pCN39 <br />    pNL9163 pNL9164 (ts) streaked TSA agar plates temperatures 32&#194;&#176;        (Charpentier et al. 2004),    43&#194;&#176;C. plates incubated 24&#226;&#8364;&#8220;48 h photographed. temperature-sensitive          temperature-sensitive replicon.  <br />    plasmid pNL9164 expressing LtrA (pCN39-derivative) present higher copy number    pNL9163 (pCN37-derivative) (Charpentier et al. 2004), likely resulting somewhat       structed follows:  Pcad promoter <br />    residual growth temperature shift 43&#194;&#176;C. (B) RT-PCR assay Ll.LtrB-DORF intron     cut pCN37 digestion SphI <br />    splicing hsa gene. Cells grown BHI medium overnight, diluted 1:20     PstI, cloned SphI    fresh medium, incubated 4 h 32&#194;&#176;C 2 h 43&#194;&#176;C. RNA extracted            PstI sites pCN39.  XbaI site <br />    analyzed RT-PCR, using primers pE1 pE2 flanking Ll.LtrB-DORF intron inserted       downstream Pcad replaced    hsa gene. smaller RT-PCR product (0.1 kb) derived hsa mRNA           NheI site HindIII BsrGI sites <br />    inserted intron, larger product (1.0 kb) derived precursor RNA containing <br />                                                                                                     removed, described    inserted intron. identities RT-PCR products confirmed sequencing <br />    (data shown). Controls showed products initial reverse transcription step      pNL9161. Finally, pACD2X digested <br />    omitted reactions carried absence RNA substrate (data shown),   XbaI EcoRV, 3-kb frag- <br />    indicating amplified bands experimental samples derived cellular RNA.   ment containing Ll.LtrB-DORF intron <br />    schematic right gel shows hsa precursor mRNAs, including           flanking exons plus downstream <br />    location primers pE1 pE2 expected RT-PCR products.                                LtrA ORF cloned NheI <br />                                                                                                     SmaI sites pCN39. <br />  <br />    Recombinant plasmids                                                      Retargeting Ll.LtrB intron insert                                                                              S. aureus genes <br />    pACD2X contains 0.9-kb Ll.LtrB-DORF intron flanking <br />    exons cloned downstream T7lac promoter CamR                     Ll.LtrB targetron retargeted insert S. aureus genes <br />    pACYC184-based vector (San Filippo Lambowitz 2002).                   using algorithm identifies potential insertion <br />    ORF encoding LtrA protein expressed position            sites designs PCR primers modifying intron RNA  <br />  <br />    1278         RNA, Vol. 12,  7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 9 OUTPUT: Wednesday June 7 03:20:35 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                  Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />       TABLE 2. Real-time RT-PCR analysis levels hsa transcripts S. aureus strains <br />  <br />                                                           Unspliced precursor RNAb                                   hsa mRNAb <br />  <br />       Straina                                           32&#194;&#176;C                      43&#194;&#176;C                     32&#194;&#176;C                      43&#194;&#176;C <br />  <br />       WT                                                N                        N                 0.23 6 0.02               0.25 6 0.02 <br />       RN4220 hsar::LtrB/PNL9163                     2.12 6 0.10                2.73 6 0.07             0.17 6 0.01              0.048 6 0.002 <br />       RN4220 hsar::LtrB/PNL9164 (ts)c               0.29 6 0.03                2.97 6 0.06             0.24 6 0.03              0.013 6 0.002 <br />              WT, wild-type S. aureus RN4220; RN4220 hsa::LtrB/pNL9163, hsa disruptant obtained pNL9163; RN4220 hsa::LtrB/pNL9164, hsa <br />       disruptant obtained pNL9164. <br />       b <br />         Amounts hsa transcripts expressed percentage 16S rRNA. <br />       c <br />        ts, temperature-sensitive. <br />       N , applicable. <br />  <br />  <br />  <br />     base-pair sites (Perutka et al. 2004). Single target sites     1992). E. coli DNAs difficult transform directly     hsa seb chosen potential target sites                  S. aureus RN8098 restriction. Consequently, seb <br />     identified algorithm gene. Donor plasmids              targetron plasmid propagated S. aureus RN4220,  <br />     constructed intron RNA&#226;&#8364;&#8482;s EBS1, EBS2,                 isolated using Qiaquick Miniprep Kit (Qiagen), electro- <br />     d sequences modified optimally base-pair DNA target             porated RN8098. Cells grown overnight BHI <br />     site sequences IBS1, IBS2, d9. IBS2 IBS1 sequences          containing erythromycin, diluted 1:100 5 mL fresh <br />     59-exon donor plasmid modified com-             medium, grown early log phase (OD595 = 0.5),     plementary retargeted EBS2 EBS1 sequences effi-             plated directly BHI containing erythromycin induced <br />     cient RNA splicing (Perutka et al. 2004).                                  10 mM CdCl2 90 min prior plating. Targetron <br />        required modifications introduced donor               integration assayed colony PCR, using primers flanking <br />     plasmid step PCR diagrammed Figure 1C (Karberg                target gene referred generically pf pr.     et al. 2001). step, segments donor plasmid         hsa, pf pr primers 59-CGGAATCAGGAGGTGAA <br />     amplified PCRs. PCR used primers p1 (59-exon                   TGTCTAATGA 59-CGGAATTCACTTAGAAGAACATTACAA <br />     positions &#195;&#191;25 +18 modifications positions &#195;&#191;12 &#195;&#191;1             TTTAT; seb, pf pr primers 59-GGATAAAG <br />     IBS1 IBS2; Seb-123a donor plasmids, positions &#195;&#191;25              GAGATAAAAAATGTATAAGAG 59-AGGTACTCTATAAGTGC <br />     &#195;&#191;13 changed ATAATTATCCTTA prevent                       CTGCCTTAATTAC. <br />     self-targeting) plus p2 (59-CGAAATTAGAAACTTGCGTTCAGTA <br />     AAC). PCR used primers p3 (intron positions +198              Southern hybridization <br />     +246 modifications positions +223 +227 EBS2) plus <br />     p4 (intron positions +259 +326 modifications positions          Cells grown BHI medium overnight, DNA     +276 +285 EBS1 d). second step, PCR              isolated using Bacterial Genomic DNA Prep kit (QIAGEN). <br />     products step gel-purified, mixed             Southern hybridization described (Perutka et al. 2004), <br />     amplified outside primers p1 p4 generate 353-bp          using 32P-labeled intron probe generated PCR pACD2X <br />     PCR product corresponding 59-exon 59-end                 primers 59-TCTTGCAAGGGTACGGAGTA 59-GTAGG <br />     intron (nucleotide positions E1&#226;&#8364;&#8220;25 + 326), modified              GAGGTACCGCCTTGTTC. probe labeled using High <br />     IBS1/2, EBS2, EBS1/d sequences. final PCR product              Prime DNA Labeling kit (Roche Diagnostics). <br />     purified 0.8% agarose gel cloned BsrGI     HindIII sites donor plasmid. initial experiments             RT-PCR qRT-PCR analysis RNA splicing <br />     disruption hsa seb, modifications introduced <br />     pACD2X order validate targetron, using E. coli           S. aureus RN4220 strains containing pCN37- pCN39-based <br />     plasmid assay targetron integration (Guo et al. 2000; Karberg          plasmids grown overnight BHI medium containing <br />     et al. 2001). modified pACD2X segment recloned                erythromycin 37&#194;&#176;C 32&#194;&#176;C, respectively, 200 mL     pCN37 (3-kb XbaI + XhoI fragment pACD2X blunt-ended                overnight culture inoculated 20 mL fresh medium     T4 DNA polymerase ligated SmaI site                   incubated conditions specified individual experiments <br />     pCN37) generate S. aureus donor plasmids pNL9163                Figures 5 6. incubations, RNA extracted     pNL9165, contain targetrons Hsa-24s Seb-123a,                 1 mL culture using RNeasy mini Kit (Qiagen). RT-PCR <br />     spectively (Table 1). pCN39 derivatives, modifica-            carried 300 ng RNA using SuperScript  <br />     tions targeting intron hsa seb genes introduced        Step RT-PCR (Invitrogen), primers flanking     directly pNL9162 generate donor plasmids pNL9164               inserted intron. Ll.LtrB-DORF intron inserted hsa <br />     pNL9166, respectively (Table 1).                                           (Fig. 5), primers 59-AGGTGAATGTCTAATGAA <br />                                                                                CAAAACAGA 59-CATCTACTGCTGAACCAGCTTCTT,     Gene targeting S. aureus                                                RT-PCR 50&#194;&#176;C, 30 min; 94&#194;&#176;C, 5 min; 20 <br />                                                                                cycles 94&#194;&#176;C, 30 sec; 55&#194;&#176;C, 30 sec; 72&#194;&#176;C, 75 sec; followed     donor plasmids containing retargeted introns                  72&#194;&#176;C, 7 min. Ll.LtrB-DORF intron pNL9161 (Fig. 6), <br />     electroporated S. aureus RN4220 (Schenk Laddaga                   primers 59-CTAGCAGCACGCCATAGTG  <br />  <br />                                                                                                                   www.rnajournal.org        1279 <br />  JOBNAME: RNA 12#7 2006 PAGE: 10 OUTPUT: Wednesday June 7 03:20:35 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />    Yao et al. <br />  <br />  <br />  <br />                                                                           primers conditions used analyze splicing                                                                           Ll.LtrB-DORF intron pNL9161. <br />                                                                              Real-time reverse transcription PCR (qRT-PCR) S. aureus <br />                                                                           RNA carried 300 ng cellular RNA samples using <br />                                                                           SYBR green RT-PCR kit (Applied Biosystems), primers: <br />                                                                           59-ATGAACAAAACAGATTTAATCAATGCAG [PF(E1&#226;&#8364;&#8220;E2)] plus <br />                                                                           59- CATCTACTGCTGAACCAGCTTCTT (PR) ligated exons, <br />                                                                           59-AGGCGGTACCTCCCTACTTC [PF(Pre)] plus PR                                                                           unspliced precursor RNA. forward primer precursor <br />                                                                           RNA corresponds 39-end intron, ending 2 nt <br />                                                                           upstream 39-splice site, forward primer ligated <br />                                                                           exons spans splice junction, 24 nt 59-exon 4 nt <br />                                                                           39-exon, ensuring amplify ligated <br />                                                                           exons precursor RNA. common reverse primer                                                                           complementary sequence 39-exon. specificity                                                                           primers intended RNAs checked RT-PCR <br />                                                                           (data shown). <br />                                                                              qRT-PCR using universal thermal cycling conditions, <br />                                                                           specified manufacturer (48&#194;&#176;C, 30 min; 95&#194;&#176;C, 10 min; <br />                                                                           40 cycles 95&#194;&#176;C, 15 sec, 60&#194;&#176;C, 1 min) ABI 7900HT fast <br />                                                                           real-time PCR analyzed using SDS software <br />                                                                           (Applied Biosystems). Serial twofold dilutions (1:5&#226;&#8364;&#8220;1:1280)                                                                           S. aureus wild-type RN4220 cellular RNA used generate <br />                                                                           standard curve fluorescence versus cycle number 16S <br />                                                                           rRNA using primers 59-CTAATCAGAAAGCCACGGCTAAC                                                                           59-CGGATAACGCTTGCCACCTA. amounts hsa precur- <br />                                                                           sor RNA mRNA calculated plotting correspond- <br />                                                                           ing CT (cycle number fluorescence crosses threshold <br />                                                                           value set SDS program) standard curve                                                                           expressed percent 16S rRNA. CT values 16S rRNA <br />                                                                           similar RNA samples, control reactions    FIGURE 6. Temperature sensitivity splicing wild-type         addition RT RNA substrate showed nonspecific <br />    Ll.LtrB-DORF intron E. coli S. aureus.   E. coli              amplification. <br />    HMS174(DE3) containing intron-donor plasmid pACD2X grown <br />    log phase, 100 mM IPTG added induce intron                 qRT-PCR E. coli RNAs similarly using primers: <br />    expression, cultures incubated 1 h overnight       59-GATCGTGAACACATCCATAACCATA plus 59-AAATTGTT <br />    (O/N) 30&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C. (B) S. aureus strain RN4220 containing     TGCCAGTATAAAGATTCG ligated exons; 59-CAAGGCGG <br />    pNL9161 grown overnight, diluted 1:20 fresh BHI          TACCTCCCTACTTC plus 59-AAATTGTTTGCCAGTATAAAGA <br />    containing erythromycin. 10 mM CdCl2 added induce          TTCG unspliced precursor RNA; 59-CGTGTTGTGAAAT <br />    intron expression, cultures incubated 90 min 6 h   GTTGGGTTAA 59-CCGCTGGCAACAAAGGATA 16S <br />    32&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C. RNA extracted analyzed RT-PCR, <br />                                                                           rRNA. <br />    using primers pE1 pE2, hybridize 59 (E1) 39 (E2) <br />    exon sequences flanking Ll.LtrB-DORF intron pACD2X    pNL9161. schematic beneath gel shows precursor    mRNAs, location primers pE1 pE2 expected <br />    RT-PCR products.                                                       ACKNOWLEDGMENTS <br />                                                                           work supported NIH grant GM37949 Welch <br />                                                                           Foundation grant F-1607 M.L., NIH grant AI30138                                                                           R.P.N. thank David Goerlitz, Hong Cui, Sijiong Mou    59-TCGTGAACACATCCATAAC, RT-PCR                        contributions initial experiments S. aureus gene <br />    52&#194;&#176;C, 30 min; 94&#194;&#176;C, 5 min; 40 cycles 94&#194;&#176;C, 30 sec, 58&#194;&#176;C,    targeting; Jiri Perutka help targetron design; Greg <br />    30 sec, 72&#194;&#176;C, 90 sec; followed 72&#194;&#176;C, 7 min. PCR products        Davis (Sigma-Aldrich) helpful suggestions. <br />    analyzed 2% agarose gel, stained ethidium <br />    bromide.                                                               Received February 17, 2006; accepted April 17, 2006. <br />       E. coli, strain HMS174 (DE3), contains IPTG- <br />    inducible phage T7 RNA polymerase, transformed    pACD2X pACD-LtrB plated LB medium containing                 REFERENCES <br />    chloramphenicol. single colony grown LB medium <br />                                                                           Bae, T. Schneewind, O. 2006. Allelic replacement Staphylococ- <br />    containing chloramphenicol OD595 = 0.2&#226;&#8364;&#8220;0.3, 100 mM             cus aureus inducible counter-selection. Plasmid 55: 58&#226;&#8364;&#8220;63. <br />    IPTG added induce T7 RNA polymerase transcription,          Belhocine, K., Plante, , Cousineau, B. 2004. Conjugation <br />    culture incubated 30&#194;&#176;, 37&#194;&#176;, 43&#194;&#176;C 1 h overnight.      mediates transfer Ll.LtrB group II intron different <br />    RNA extracted analyzed RT-PCR  using             bacterial species. Mol. Microbiol. 51: 1459&#226;&#8364;&#8220;1469. <br />  <br />  <br />    1280         RNA, Vol. 12,  7 <br />  JOBNAME: RNA 12#7 2006 PAGE: 11 OUTPUT: Wednesday June 7 03:20:39 2006 <br /> csh/RNA/118163/rna687 <br />  <br />  <br />  <br />  <br />                                                                                                       Use targetrons Staphylococcus aureus <br />  <br />  <br />  <br />     Charpentier, E., Anton, , Barry, P., Alfonso, B., Fang, Y.,           Kreiswirth, B.N., Lo&#194;&#168;fdahl, S., Betley, M.J., O&#226;&#8364;&#8482;Reilly, M., <br />         Novick, R.P. 2004. Novel cassette-based shuttle vector            Schlievert, P.M., Bergdoll, M.S., Novick, R.P. 1983. toxic <br />         Gram-positive bacteria. Appl. Environ. Microbiol. 70: 6076&#226;&#8364;&#8220;6085.             shock syndrome exotoxin structural gene detectably trans- <br />     Chee, G.-J. Takami, H. 2005. Housekeeping recA gene interrupted              mitted prophage. Nature 305: 709&#226;&#8364;&#8220;712. <br />         group II intron thermophilic Geobacillus kaustophilus.          Lambowitz, M. Zimmerly, S. 2004. Mobile group II introns. <br />         Gene 363: 211&#226;&#8364;&#8220;220.                                                           Annu. Rev. Genet. 38: 1&#226;&#8364;&#8220;35. <br />     Chen, J. Novick, R. 2006. SvrA, multidrug exporter (MepA) does          La <br /> </body></html>