<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Functional Annotation Class Lysyl-tRNA Synthetase Phylogeny <br />               Indicates Limited Role Gene Transfer <br />  <br />             Functional comparative genomic studies previously shown essential protein lysyl-tRNA <br />          synthetase (LysRS) exists unrelated forms. prokaryotes eukaryotes contain class II <br />          LysRS, archaea bacteria contain common class LysRS. bacteria class <br />          LysRS &#226;&#65533;&#163;-proteobacteria scattering groups, including spirochetes,          class protein far common form LysRS archaea. investigate unusual <br />          distribution functionally annotated representative phylogenetic sampling LysRS proteins. Class          LysRS proteins variety bacteria archaea characterized vitro ability recognize <br />          Escherichia coli tRNALys anticodon mutants. Class LysRS proteins fall distinct groups, <br />          preferentially recognize anticodon <span id='am-6' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> tRNALys (U36) recognize <br />          second positions (U35 U36). Strong recognition U35 U36 confined          pyrococcus-spirochete grouping archaeal branch class LysRS phylogenetic tree, U36 <br />          recognition seen archaea example &#226;&#65533;&#163;-proteobacteria. corre- <br />          sponding phylogenetic relationships, results suggest despite comparative rarity distribution <br />          class LysRS conforms canonical archaeal-bacterial division. exception, suggested          functional phylogenetic data, appears horizontal transfer class LysRS pyrococcal <br />          progenitor limited number bacteria. <br />  <br />  <br />    Aminoacyl-tRNA (aa-tRNA) synthesis process                        comparable indirect pathways exist production Asn- <span id='am-19' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> esterified 3&#226;&#172;&#732; end tRNA                 tRNAAsn selenocysteinyl-tRNASec (reviewed reference <br /> containing corresponding anticodon sequence (15).                 11). use pathways initially assumed synthesis, aa-tRNAs screened selected elon-                idiosyncratic small minority organisms, recent anal- <br /> gation factor Tu (16) taken ribosomal decoding              yses completed microbial genome sequences shown <br /> site, paired complementary mRNA                     widespread prokaryotes corresponding <br /> codons (19). Consequently, availability cellular pool           direct pathways absent. GlnRS absent characterized <br /> correctly aminoacylated tRNAs essential faithful                 archaea majority bacteria, asparaginyl-tRNA <br /> translation genetic information encoded mRNA                 synthetase (AsnRS) absent archaea signif- corresponding polypeptide sequence (14). Aminoacyl-                     icant number bacteria (31). Initial analyses complete <br /> tRNAs primarily aminoacyl-tRNA synthetase                   genome sequences archaea Methanococcus jannaschii <br /> (aaRS) protein family (9). accuracy aa-tRNA synthesis                (3) Methanobacterium thermoautotrophicum (24) sug- aaRSs ensured extremely high substrate                   gested addition AsnRS GlnRS, genes encoding <br /> specificity, enhanced cases                 cysteinyl-tRNA synthetase (CysRS) lysyl-tRNA syn- <br /> existence proofreading mechanisms. 20 aaRS proteins,                 thetase (LysRS) absent organisms. example Escherichia coli (6), divided              Experimental studies showed CysRS (17, 26) mutually exclusive structural groups 10 members                 LysRS (13) present archaea, encoded termed class class II (reviewed references 1 5).               novel reassigned genes, making original annotation assignment aaRS specific particular amino acid              corresponding open reading frames difficult. case structural classes completely               LysRS experimental approaches identified gene encoding served, reflecting ancient evolutionary origin family           class aaRS (lysK) archaeon Methanococcus maripaludis, proteins (21).                                                           contrast previously characterized LysRS proteins    20 canonical aaRS proteins represent sole                 belonged class II. Comparative genomic analyses showed <br /> means cellular aa-tRNA synthesis. observed               class II LysRS-encoding genes (lysS) absent Bacillus megaterium Gln-tRNAGln synthesized                   majority archaea bacteria, lysK Glu-tRNAGln use glutaminyl-tRNA syn-                     instead cases (e.g., reference 10). Functional (12) thetase (GlnRS) (32), subsequent studies showed                    structural (29) characterizations shown class                                                                             II LysRS proteins functionally equivalent structur- <br />                                                                             ally unrelated. <br />   * Corresponding author. Mailing address: Department Microbi- <br /> ology, Ohio State University, 484 West 12th Ave., Columbus, OH <br />                                                                                Primarily result sequencing numerous micro- <br /> 43210-1292. Phone: (614) 292-2120. Fax: (614) 292-8120. E-mail:             bial genomes, 30 lysK gene sequences known. ibba.1@osu.edu.                                                             distribution organisms containing lysK,  <br />                                                                      4594 <br />  VOL. 184, 2002                                                                             EVOLUTION LYSYL-tRNA SYNTHETASES                                 4595 <br />  <br />                                                                                    nas aromaticivorans cells gift Margaret Romine (Pacific Northwest <br />                                                                                    National Lab, Richland, Wash.) Ferroplasma acidarmanus genomic DNA <br />                                                                                    gift Paul F. Predki (DOE Joint Genome Institute, Walnut Creek, <br />                                                                                    Calif.). S. aromaticivorans cells grown 50% Luria broth 36 h 30&#194;&#176;C <br />                                                                                    genomic DNA prepared according standard procedures (23). <br />                                                                                       Cloning lysK lysS genes. LysRS encoding genes amplified                                                                                    genomic DNAs indicated standard PCR conditions using fol- <br />                                                                                    lowing primers (engineered restriction enzyme recognition sites indicated <br />                                                                                    underlined primer): S. aromaticivorans lysK, Forward: (NdeI) <br />                                                                                    5&#226;&#172;&#732;-GCATATGACAGACGAGATCCGCACC, Reverse: (BamHI) 5&#226;&#172;&#732;-GG <br />                                                                                    ATCCTCAGTCGAGCGCCTCGGC; M. barkeri lysK, Forward: (XhoI) <br />                                                                                    5&#226;&#172;&#732;-GCTCGAGATGGCTGACACAATTCACTGG, Reverse: (BlpI) 5&#226;&#172;&#732;-G <br />                                                                                    CTGAGCTTATGCCTGTTTTTCGGGGC; M. barkeri lysS, Forward: (Nd <br />                                                                                    eI) 5&#226;&#172;&#732;-GCATATGAGCATGGAAATTAACAATGAAAAAATGTC,  <br />                                                                                    verse: (BamHI) 5&#226;&#172;&#732;-GGATCCTCAGTCTTCCCTTTTCATCTGTGG; F. <br />                                                                                    acidarmanus lysK, Forward: (XhoI): 5&#226;&#172;&#732;-GCTCGAGATGTACTGGGCAGAT <br />                                                                                    GCACTG, Reverse: (BamHI): 5&#226;&#172;&#732;-GGATCCTTACTGTGCAGAAT- <br />                                                                                    TATTCTGGATAAG; M. magnetotacticum lysK, Forward: (NdeI) 5&#226;&#172;&#732;- <br />                                                                                    GCATATGTCCTCCGATTCCCGC, Reverse: (BamHI) 5&#226;&#172;&#732;-GGATCCC <br />                                                                                    TACATATCCTCGCCCGC). <br />                                                                                       amplified fragments cloned Topo TA (Invitrogen) vector                                                                                    subsequently subcloned expression vector pET15b (Invitrogen) using                                                                                    restriction sites introduced PCR. Clones sequenced prior transfor- <br />                                                                                    mation E. coli BL21 C&#226;&#171;&#185; strain. procedure allowed overproduction <br />                                                                                    N-terminally His6-tagged derivatives, previously                                                                                    perturb substrate recognition LysRS (14). <br />                                                                                       Overexpression purification LysRS proteins. Transformants                                                                                    grown Luria broth supplemented ampicillin (100 &#226;&#65533;&#174;g/ml) chloram- <br />                                                                                    phenicol (34 &#226;&#65533;&#174;g/ml) 37&#194;&#176;C optical density 600 nm 0.6. expression <br />                                                                                    recombinant protein induced IPTG (isopropyl-&#226;&#65533;&#164;-D-thiogalac- <br />                                                                                    topyranoside) (final concentration 1 mM) 3 h 37&#194;&#176;C. cells                                                                                    harvested centrifugation washed phosphate-buffered saline;                                                                                    subsequent steps performed 4&#194;&#176;C. cell paste resuspended                                                                                    lysis buffer (50 mM Na2H2PO4, 300 mM NaCl, 5 mM 2-mercaptoethanol, pro- <br />                                                                                    tease inhibitor [Roche]) sonicated 10 cycles 30 s. cell extract    FIG. 1. Secondary structure undermodified E. coli tRNALys. Ab-               obtained ultracentrifugation 100,000 &#226;&#171;&#187; g 1 h subsequently applied <br /> breviations: D, dihydrouridine; &#226;&#65533;&#186;, pseudouridine; m7G, 7-methyl-                   Ni-nitrilotriacetic acid agarose column (Qiagen), according standard <br /> <span id='am-1' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="guanosine" datatype="xsd:string"></span>guanosine</span>; X, putative 3-(3-amino-3-carboxypropyl) <span id='am-56' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-57' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uridine</span> modifica-               procedure. protein eluted elution buffer (containing 750 mM <br /> tion. undermodified variants used study contain uridine             imidazole), dialyzed overnight storage buffer (100 mM Tris-HCl [pH 8.0], <br /> place 5-methylaminomethyl-2-thiouridine position 34 aden-                50 mM KCl, 25 mM MgCl2, 5 mM 2-mercaptoethanol, 50% glycerol), osine place N6-threonylcarbamoyladenosine position 37.                    stored &#226;&#171;&#186;20&#194;&#176;C. purity recombinant enzyme preparations Changes anticodon nucleotides (nucleotides 34 36) indi-              judged 95% greater sodium dodecyl sulfate-polyacrylamide gel <br /> cated.                                                                             electrophoresis followed staining Coomassie brilliant blue. <br />                                                                                       Cloning vitro transcription tRNALys genes. tRNA genes vitro <br />                                                                                    T7 RNA polymerase transcription cloned annealing cDNA <br /> archaeal Methanosarcina group exclusion lysS,                      oligonucleotides direct ligation pUC18 plasmid using BamHI Hin- <br />                                                                                    dIII restriction sites. clones transformed E. coli DH5&#226;&#65533;&#163;, sequenced, <br /> compasses archaea scattering bacteria                                                                                    amplified. 3&#226;&#172;&#732; end transcription template digested overnight eukaryotes. Previous studies attributed unusual                       BstNI 60&#194;&#176;C NsiI 37&#194;&#176;C. vitro T7 RNA polymerase runoff <br /> evolutionary pattern horizontal gene transfer (12, 33,                   transcription conducted according standard procedure (18). tRNALys <br /> 34) selective retention common ancestor                        transcripts purified denaturing 12% polyacrylamide gel recovered <br /> particular lineages (22). investigate evolution                  electroelution (Elutrap; Schleicher Schuell), followed desalting                                                                                    Nap-5 Column (Pharmacia Biotech). Plasmid T7-911 preparation lysK gene family, extended previous                            His6-tagged recombinant T7 RNA polymerase gift T. Shrader   <br /> preliminary studies attempted differentiate range                      partment Biochemistry, Albert Einstein College Medicine, New York, <br /> class LysRS proteins based mode recognition                     N.Y.). <span id='am-14' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span>Nucleoside</span> triphosphates Sigma. <br /> tRNALys. Comparison results revised phyloge-                          Genomic phylogenetic analyses. predicted amino acid sequences                                                                                    obtained publicly available databases. Sequences initially aligned <br /> netic tree class LysRS sequences suggests selective <br />                                                                                    using program CLUSTALX (30). alignment used generate <br /> retention major factor evolution protein                    neighbor-joining phylogeny using PHYLIP 3.57c package (distributed family horizontal gene transfer playing relatively minor                    author [J. Felsenstein]; Department Genetics, University Washington, <br /> role.                                                                              Seattle) parameters previously used aaRS phylogenies (20). Maximum- <br />                                                                                    likelihood phylogeny generated using program Tree-Puzzle 5.0 (www <br />                                                                                    .tree-puzzle  (28) (10,000 puzzle steps, JTT substitution matrix). <br />                        MATERIALS METHODS <br />    General. vivo overproduced E. coli tRNALys variants, antico- <br /> don loop nucleotides 34 37 previously shown unmodified <br />                                                                                                                      RESULTS <br /> (Fig. 1) (4), prepared described (4), Borrelia burgdorferi <br /> class LysRS protein (10). Aminoacylation assays performed previously       Aminoacylation activity tRNA specificity recombi- <br /> described (12). Methanosarcina barkeri fusaro genomic DNA gift Kevin Sowers (University Maryland Biotechnology Institute, Baltimore), Mag- <br />                                                                                    nant LysRS proteins. recombinant LysRS proteins netospirillum magnetotacticum genomic DNA gift Elizabeth Bertani        tested vitro activity determined ability (Division Biology, California Institute Technology, Pasadena), Sphingomo-    aminoacylate transcripts (cognate) tRNALys genes <br />  4596        AMBROGELLY ET AL.                                                                                                        J. BACTERIOL. <br />  <br />  <br />       TABLE 1. vitro lysylation tRNALys substrates LysRS                 tRNA (Fig. 2C), mutation U35 C did result                                 proteins                                          change activity (Fig. 2D). significant reduction <br />                                        Formation Lys-tRNALyse                  activity F. acidarmanus LysRS observed LysRSa protein source                                                             tRNALys U36 replaced C, change virtu- <br />                          Cognate                                   E. coli <br />                                        Noncognate transcriptc                     ally eliminated recognition. <br />                         transcriptb                             tRNALysUUUd <br />                                                                                      Anticodon recognition M. barkeri class II lysyl-tRNA syn- <br /> F. acidarmanus             &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185; <br />                                                                                   thetase. Recognition tRNALys anticodon class II <br /> M. magnetotacticum         &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185; <br /> M. barkeri                 &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#186;           LysRS M. barkeri investigated described M. barkeri class II        &#226;&#171;&#186;          &#226;&#171;&#186;                               &#226;&#171;&#185;           class LysRS proteins. Mutation nucleotide U34 G <br /> S. aromaticivorans         &#226;&#171;&#185;          &#226;&#171;&#185; (M. magnetotacticum)          &#226;&#171;&#185;           (Fig. 3A) led significant reduction recognition M. <br /> B. burgdorferi             &#226;&#171;&#185;f         &#226;&#171;&#185; (F. acidarmanus)              &#226;&#171;&#185;f          barkeri class II LysRS, replacement U36 C <br />        LysRS proteins class indicated M. barkeri.           (Fig. 3B) caused slight change, indicating relative <br />   b <br />      vitro-transcribed tRNALys genes (UUU anticodon sequence) derived          importance positions. strongest effects amino- organism LysRS. <br />    c <br />      vitro-transcribed tRNALys genes (UUU anticodon sequence) derived     acylation observed changes U35 tRNALys, organism different LysRS derived, indicated.   G35 C35 variants showing significant reductions    d <br />      Synthesized vivo; text details.                                   activity compared wild-type (Fig. 3B). <br />    e <br />      Lysylation tRNA performed 37&#194;&#176;C 15 min 1 &#226;&#65533;&#174;M enzyme, <br /> 100 &#226;&#65533;&#174;M lysine, 4 &#226;&#65533;&#174;M tRNA. Symbols: &#226;&#171;&#186;, detectable formation                 Molecular phylogenies class LysRS. Phylogenetic anal- <br /> Lys-tRNALys compared enzyme-free control; &#226;&#171;&#185;, detectable formation        yses class LysRS protein family performed using <br /> Lys-tRNALys. <br />    f <br />      Shown comparison, taken reference 13. <br />                                                                                   28 31 sequences currently available public databases. <br />                                                                                   excluded sequences Borrelia genospe- <br />                                                                                   cies 75% amino acid identity B. burg- <br />                                                                                   dorferi (N. Mejlhede,  Montha&#194;&#180;n, M. Thesien, M. Ibba, noncognate tRNALys species (Table 1). pro-                        submitted publication). extensive tRNA anticodon rec- <br /> teins tested, class LysRS S. aromaticivorans                    ognition data available class LysRS B. burgdor- <br /> able aminoacylate vitro-transcribed tRNALys.                     feri (25), sequence chosen inclusion study. <br /> investigate lack aminoacylation cases                     Structural studies revealed high degree similarity <br /> resulted inactive transcripts inactive                      subclass Ib aaRSs LysRS glutamyl-tRNA <br /> zymes, cross-charging experiments performed. B. burg-                        synthetase (GluRS) (29), suggesting GluRS sequences <br /> dorferi class LysRS, previously shown                         appropriate rooting LysRS phylogenetic <br /> aminoacylate variety tRNALys species, ami-                      trees. generated separate sequence alignments <br /> noacylate vitro-transcribed F. acidarmanus tRNALys, indi-                      using CLUSTALX 28 class LysRS 11 GluRS proteins <br /> cating tRNA active. addition, F. acidarmanus                     (chosen representative sampling based existing phy- <br /> LysRS able charge E. coli tRNALys, indicating                     logenies [33]) manually merged single <br /> enzyme active. Similar tests M. barkeri LysRS                    alignment using structure-based sequence alignment Py- <br /> proteins vitro-transcribed M. barkeri tRNALys showed                       rococcus horikoshii LysRS Thermus thermophilus GluRS class LysRS tRNA inactive class II                        (29). Positions containing gaps sequences enzyme active. Investigation M. magnetotacticum                          removed, leaving total 287 aligned residues present showed class LysRS active, albeit significantly                  sequences. sequence alignment used gen- <br /> lower level active proteins tested, revealed                   erate neighbor-joining maximum-likelihood phylogenies. vitro-transcribed M. magnetotacticum tRNALys                     cases GluRS sequences formed single distinct <br /> charged S. aromaticivorans LysRS.                                              outgroup high confidence corresponding node <br />    Anticodon recognition class LysRSs. Anticodon recog-                      (84.3% neighbor joining, 93% maximum likelihood). nition investigated using undermodified E. coli tRNALys                       maximum-likelihood tree rooted GluRS generated <br /> variants generated vivo, previously shown                      class LysRS sequences (Fig. 4). overall class efficient substrates variety class class                LysRS phylogeny resembles somewhat-reduced version II LysRSs (4, 25). Initial tests using M. magnetotacticum LysRS                   universal phylogenetic tree distinct bacterial ar- <br /> showed able charge wild-type tRNA                        chaeal groupings. exceptions placement nucleotide 34 mutant (UUU GUU anticodons,                              crenarchaeon Cenarchaeum symbiosum major bac- <br /> spectively), level activity mutants                      terial grouping subgroup bacteria close insufficient detect charging background. M.                    pyrococcal proteins archaeal group. <br /> barkeri class LysRS shown inactive, pat- <br /> tern anticodon recognition investigated                                                                                                           DISCUSSION F. acidarmanus S. aromaticivorans proteins <br /> (Fig. 2). Mutation U34 G tRNALys                          Comparative genomics tRNALys recognition. effect recognition S. aromaticivorans LysRS (Fig. 2A),                   LysRSs, exception class protein M. <br /> changes U35 resulted reduced charging                          barkeri, active heterologously pro- <br /> pronounced effect observed replacement U36,                      duced, recognize cognate vitro- completely eliminated activity (Fig. 2B). F. aci-                     transcribed tRNA substrate. Previous studies showed darmanus LysRS replacements U34 U35 G                        B. burgdorferi (12) P. horikoshii (29) class LysRSs led improvements recognition compared wild-type                          recognized cognate vitro-transcribed tRNALys, M. <br />  VOL. 184, 2002                                                                  EVOLUTION LYSYL-tRNA SYNTHETASES                     4597 <br />  <br />  <br />  <br />  <br />    FIG. 2. Aminoacylation E. coli tRNALys variants class His6-LysRS proteins. Aminoacylation reactions performed described text (20-&#226;&#65533;&#174;l samples) presence 4 &#226;&#65533;&#174;M tRNA 100 &#226;&#65533;&#174;M [14C]lysine. Comparison charging wild-type mutant tRNALys using <br /> class LysRS enzymes S. aromaticivorans (200 nm enzyme)  B) F. acidarmanus (2 &#226;&#65533;&#174;M enzyme) (C D). <br />  <br />  <br />  <br /> maripaludis enzyme did (12). differences tRNA               vitro did respect potential nucleotide <br /> gene transcript recognition result absence               modification sites. comparing known posi- <br /> essential RNA modifications necessary recognition fold-           tions nucleotide modifications described tRNALys ing, reported human mitochondrial tRNALys (8).                 E. coli (Fig. 1) Haloferax volcanii (7) positions  <br /> attempted correlate sequence differences tRNALys              served &#226;&#8364;&#339;active&#226;&#8364;&#65533; &#226;&#8364;&#339;inactive&#226;&#8364;&#65533; tRNALys gene sets, <br /> genes (UUU anticodon) gave rise active transcripts            (Fig. 5). comparisons identified notable <br />  <br />  <br />  <br />  <br />    FIG. 3. Aminoacylation E. coli tRNALys variants M. barkeri class II His6-LysRS. Aminoacylation reactions performed described text (20-&#226;&#65533;&#174;l samples) presence 4 &#226;&#65533;&#174;M tRNA, 100 &#226;&#65533;&#174;M [14C]lysine, 2 &#226;&#65533;&#174;M enzyme.   Position 34 variants; (B) position 35 36 variants. <br />  4598     AMBROGELLY ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                                                          recognition class LysRS S. aromati- <br />                                                                          civorans, U36 important F. <br />                                                                          acidarmanus enzyme. predominant role U36                                                                          tRNALys recognition observed class LysRS                                                                          M. maripaludis differs B. burgdorferi (25) P. <br />                                                                          horikoshii (29) enzymes, U35 U36                                                                          critical recognition elements. Taken                                                                          previous studies, data presented suggest class <br />                                                                          LysRSs divided broad groups defined                                                                          U36, U35 U36 preferred anti- <br />                                                                          codon nucleotides recognition. functional divergence <br />                                                                          class LysRSs correlated                                                                          overall phylogeny family (Fig. 4, discussed ,                                                                          contrast class II LysRS family pattern                                                                          anticodon recognition highly conserved. <br />                                                                             Phylogenetic conservation anticodon recognition class <br />                                                                          II LysRSs. Previous studies activity tRNALys antico- <br />                                                                          don variants bacterial (E. coli [4] Helicobacter pylori <br />                                                                          [25]) class II LysRSs shown positions                                                                          anticodon recognized, U35 predominant <br />                                                                          recognition site. similar pattern observed                                                                          studies human class II LysRS (27), U35 U36 <br />                                                                          recognized main recognition <br />                                                                          element. data presented (Fig. 3) indicate                                                                          archaeal class II LysRS recognizes anticodon tRNALys <br />                                                                          way bacterial eukaryotic counterparts  <br />                                                                          U35 important position. suggests  <br />                                                                          contrast class counterparts, class II LysRSs share <br />                                                                          pattern tRNA anticodon recognition conserved                                                                          primary domains universal phylogenetic tree. <br />                                                                          difference evolutionary divergence function <br />                                                                          duplicated LysRS classes supports earlier evidence <br />   FIG. 4. Maximum-likelihood phylogenetic tree class LysRS          (12) suggesting families functionally sequences. tree rooted using GluRS sequences. Numbers       mechanistically equivalent. <br /> indicate percentage occurrence nodes 10,000 puzzle steps.      Evolution class LysRS family. phylogenetic tree <br />                                                                          class LysRS sequences contains major branches <br />                                                                          broadly corresponding bacteria archaea (Fig. 4). <br /> difference position 26, N2,N2-dimethyl-             inclusion GluRS sequences places root tree <br /> guanosine H. volcanii <span id='am-26' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> E. coli.                bacterial archaeal branches, indicating tRNALys genes transcripts recognized cog-             evolutionary pattern class LysRS sequences agree- <br /> nate class LysRSs contain A26, exception M.                ment universal phylogenetic tree derived magnetotacticum, recognized contain             rRNA sequences   example, reference 2). G26. possible role modified G26 tRNALys                class LysRS tree appears canonical somewhat restricted, <br /> folding recognition requires extensive cat-            lacking known eukaryotic examples mainly lim- <br /> aloguing vivo RNA modification patterns, strong                ited &#226;&#65533;&#163;-proteobacteria bacterial branch euryarcha- <br /> correlation experimental comparative genomic                 eota archaeal branch. observation limited <br /> approaches provides testable hypothesis observed               canonical phylogeny class LysRSs suggests differences activity vitro transcribed tRNALys              selective retention lysK gene certain lineages genes. supported observation             contributed substantially present distribution, pre- <br /> distribution versus G position 26 broadly correlates             viously suggested (22). molecular phylogeny class LysRSs (Fig. 4 5).              Closer examination class LysRS phylogeny,  <br />    Evolutionary divergence anticodon recognition                suggests limited gene transfer contributed class LysRSs. data presented (Fig. 2) support                phylogenetic distribution class LysRSs. possible ex- <br /> earlier proposals (e.g., reference 25) antico-         ception canonical universal phylogeny class don crucial site tRNALys recognition class              LysRSs seen C. symbiosum protein, posi- <br /> LysRSs. agreement studies using class LysRSs              tioned main bacterial branch. Earlier analyses based bacterium B. burgdorferi archaeon M. maripaludis,            smaller data sets placed C. symbiosum branch pre- nucleotide anticodon (position 34)              ceding &#226;&#65533;&#163;-proteobacteria (33, 34), possibly indicating hori- play significant role recognition. Nucleotides         zontal transfer ancestral crenarchaeote. U35 especially U36 shown important                   present analysis suggests horizontal transfer  VOL. 184, 2002                                                                      EVOLUTION LYSYL-tRNA SYNTHETASES                                4599 <br />  <br />  <br />  <br />  <br />    FIG. 5. Alignment tRNALys sequence (UUU anticodon) organisms encoding class LysRSs. Nucleotide 26 indicated shown boldface type, anticodon sequence underlined. Abbreviations: Rpa, Rhodopseudomonas palustris; CC, Caulobacter crescentus; MMa, M. <br /> magnetotacticum; RP, R. prowazekii; Wol, Wolbachia sp.; SA, S. aromaticivorans; ML, Mesorhizobium loti; RS, Rhodobacter sphaeroides; PF, <br /> Pyrococcus furiosus; PA, Pyrococcus abysii; PH, Pyrococcus horikoshii; BB, B. burgdorferi; SC, Streptomyces coelicolor; TD, Treponema denticola; TP, <br /> Treponema pallidum; HB, Halobacterium sp.; AP, Aeropyrum pernix; MT, M. thermoautotrophicum; MJ, M. jannaschii; MM, M. maripaludis; AF, <br /> Archaeoglobus fulgidus; MB, M. barkeri; FA, F. acidarmanus; TV, Thermoplasma volcanium; TA, Thermoplasma acidophilum. Sequences genes <br /> encoding tRNALys (UUU) obtained Institute Genomic Research (http://www.tigr.org), Sanger Center (http://www.sanger <br /> .ac.uk/Projects/), public databases (http://www.ncbi.nlm.nih.gov/cgi-bin/Entrez/genom_table_cgi). <br />  <br />  <br />  <br />  &#226;&#65533;&#163;-proteobacteria, reliable interpretation                Parts work supported funds Ohio State <br /> exact positioning C. symbiosum protein class              University (M ) National Institutes Health (GM-22854                                                                             Dieter                                                                                     &#194;&#168;ll). LysRS tree, possible gene transfer events, dependent availability closely related archaeal                                              <br /> sequences. archaeal branch class LysRS <br />                                                                              <br /> tree, group containing Pyrococcus species contains                    bacterial examples LysRS. Functional analyses              <br />                                                                             <br /> shown bacterial (B. burgdorferi) archaeal (P. horiko-                <br /> shii) enzymes group share common tRNALys                     <br /> anticodon recognition properties   seen class               <br />                                                                           LysRSs phylogenetic tree, supporting            <br /> possible common origin. positioning group              canonical archaeal branch suggests corresponding              <br />                                                                            <br /> bacterial LysRSs (Borrelia, Treponema, Streptomyces) orig-               <br /> inated gene transfer event pyrococcal progen-         <br />                                                                          <br /> itor ancestral bacterium.  selective retention   <br /> appears main determinant distribution               class LysRSs, horizontal gene transfer played                                                                          <br /> significant widespread role.                                       <br />   <br /> </body></html>