<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />     Mcm10 Required Maintenance Transcriptional Silencing                           Saccharomyces cerevisiae <br />  <br />                                                              ABSTRACT <br />                 Mcm10 essential protein participates initiation elongation DNA <br />              replication. study demonstrate role Mcm10 maintenance heterochromatic <br />              silencing telomeres HM loci budding yeast. mcm10 mutants drastically reduce silencing              URA3 ADE2 reporter genes integrated silent loci. exposed factor, mcm10 <br />              mutant cells display &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo-cluster&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype associated defect maintenance silencing. <br />              addition, combined defect establishment silent chromatin, mcm10 mutants dem- <br />              onstrate synergistic defect HML silencing. Consistent direct silencing function, Mcm10p shows <br />              hybrid interaction Sir2p Sir3p destroyed mcm10-1 mutation dependent              C-terminal 108 amino acids. Tethering GBD-MCM10 defective HMR-E silencer sufficient              restore silencing. Furthermore, mutations MCM10 inhibit ability GBD-SIR3 restore silencing <br />              tethered defective HMR-E. Suppressor mutations MCM2, suppress temperature <br />              sensitivity mcm10-1, fail overcome mcm10-1 silencing defect, suggesting MCM10&#226;&#8364;&#8482;s role              transcriptional silencing separate essential functions DNA replication. <br />  <br />  <br />  <br />  <br /> T     RANSCRIPTIONAL silencing budding yeast <br />       Saccharomyces cerevisiae renders large regions genome inactive fashion similar hetero- <br />                                                                            bound telomeric DNA (Aparicio et al. 1991; Kyrion <br />                                                                            et al. 1993; Moretti et al. 1994; Rusche et al. 2003). <br />                                                                            Silencing thought consist phases, establish- <br /> chromatic regions higher eukaryotes (Grewal                         ment maintenance (Miller Nasmyth 1984; <br /> Elgin 2002). Transcriptional silencing occurs yeast                     Pillus Rine 1989). establishment phase, <br /> telomeres silent mating-type (HM) loci, HML                         silent chromatin formed nascent DNA HMR (reviewed Rusche et al. 2003), house                      maintenance phase sustained silent endogenous copies mating-type genes                          establishment event. Mutations genes SIR1 <br /> (reviewed Haber 1998). HM loci flanked                       RAP1 lead defects establishment cis-acting regulatory elements called silencers (Feldman                   maintenance silent chromatin (Pillus Rine <br /> et al. 1984; Brand et al. 1985, 1987). elements                      1989; Mahoney et al. 1991; Sussel Shore 1991), <br /> consist  E, B sites bound origin                  mutations genes CAC1 SIR3 cause <br /> recognition complex (ORC) (Bell et al. 1993; Micklem                       maintenance establishment defects (Enomoto <br /> et al. 1993; Loo et al. 1995), Rap1 (Shore Nasmyth                     Berman 1998; Enomoto et al. 2000). <br /> 1987), Abf1 (Diffley Stillman 1988), respec-                          link transcriptional silencing DNA <br /> tively. bound, factors recruit silent infor-                replication suggested observation mation regulator (Sir) proteins, Sir1&#226;&#8364;&#8220;4. Orc1                     passage S phase required establish- <br /> directly interacts Sir1 Rap1 Sir4 es-                     ment transcriptional silencing (Miller Nasmyth <br /> tablish silencing complex includes Sir2 Sir3,                   1984). connection silencing DNA unclear proteins recruited                   replication evidenced fact silencers silencer independently  stepwise                    used initiate replication plasmids (Brand et al. 1987) <br /> spreading Sir2/3/4 complex silencer                        origins silencer activity (Grunweller <br /> leads silencing nearby genes (Hecht et al. 1996;                 Ehrenhofer-Murray 2002).  stud- <br /> Hoppe et al. 2002; Rusche et al. 2002). Telomeric si-                      ies demonstrated S-phase requirement <br /> lencing similar silencing HM loci, excep-                does involve DNA replication (Kirchmaier tion Sir2/3/4 complex recruited Rap1                        Rine 2001; Li et al. 2001). S. cerevisiae, DNA replica- <br />                                                                            tion tightly regulated process initiated early S <br />   1 <br />                                                                            phase specific loci called origins replication au- <br />    Corresponding author: Department Molecular Biology Genetics, <br /> 325 Biotechnology Bldg., Cornell University, Ithaca, NY 14853.             tonomously replicating sequences (ARSs) (reviewed E-mail: bt16@cornell.edu                                                   Dutta Bell 1997). Budding yeast  400  <br /> Genetics 171: 503&#226;&#8364;&#8220;515 (October 2005) <br />  504                                               Liachko B. K. Tye <br />  <br /> replication origins, shown           strate novel function MCM10 maintenance <br /> play role silencing. connection           silent chromatin separate essen- <br /> tween DNA replication transcriptional silencing         tial role DNA replication. completely understood, DNA replication <br /> proteins implicated silencing.  <br />                                                                               MATERIALS METHODS <br /> clude S. cerevisiae ORC (Bell et al. 1993; Micklem et al. <br /> 1993), PCNA, RF-C, Cdc45 (Ehrenhofer-Murray et al.                Strains plasmids: Strains used study listed 1999; Zhang et al. 2000), Mcm5 (Dziak et al. 2003).        Table 1. strains isogenic derivatives W303-1A, unless <br /> Similarly, DNA replication factors im-          indicated. procedures performed accord- <br />                                                                ing standard yeast methodology (Sherman 1991). Strains <br /> plicated organization chromatin structure            carrying silencing reporters crossing strain higher eukaryotes fruit fly Drosophila          YB541 YB697 appropriate mutant strain selecting <br /> melanogaster (Pflumm Botchan 2001; Christensen             desired segregants conditional phenotypes Tye 2003) Schizosaccharomyces pombe (Bailis et al.     auxotrophy. Genotypes confirmed PCR plas- <br /> 2003). proteins, Mcm10, physically interacts      mid complementation applicable. Plasmids used                                                                study listed Table 2. Plasmids used expression heterochromatin factor HP1 (Christensen           tethering fusions constructed Gateway Tye 2003; Kellum 2003) D. melanogaster, indicating        (Invitrogen, San Diego). create pGBKgw, kanr gene possible direct role chromatin structure.                   plasmid pGBKT7 (CLONTECH, Palo Alto, CA) replaced <br />    MCM10 originally identified yeast screens            ampr gene Gateway (Invitrogen) cassette searched mutants DNA replication (Dumas            inserted multiple-cloning sequence. Gateway cassettes <br />                                                                ligated plasmids pBTM116 pGAD2F, creating <br /> et al. 1982; Solomon et al. 1992) minichromosome           pBTMgw pGADgw, respectively. Gateway cassette B maintenance (Maine et al. 1984). Mcm10 essential         ligated SmaI site pGBT9.C (Bartel et al. 1996), <br /> chromatin-associated protein (Homesley et al. 2000; Ricke      creating plasmid pGBT9gw. pDONR201 entry clones contain- Bielinsky 2004; Yang et al. 2005). Temperature-            ing MCM10 SIR1 ready N-terminal fusions  <br /> sensitive mcm10 mutants sluggish passage          structed according Invitrogen instructions sequenced. <br />                                                                LR recombination reactions (Invitrogen) set S phase (Kawasaki et al. 2000), inability complete S     pGBT9gw, pGADgw, pBTMgw, pGBKgw phase (Merchant et al. 1997), replication fork pause     aforementioned entry clones. yeast transformations phenotype inactive origins replication (Merchant         carried using standard lithium acetate protocols. <br /> et al. 1997). MCM10 implicated               Silencing assays: Strains bearing URA3 reporter initiation DNA replication (Homesley et al. 2000;           grown overnight appropriate dropout media. Tenfold serial <br />                                                                dilutions set sterile 96 plates  25 ml Wohlschlegel et al. 2002; Sawyer et al. 2004).       dilution spotted appropriate plate using exact roles DNA replication completely       multipipettor. 5-Fluoroorotic acid (5-FOA) used  <br /> derstood, Mcm10 shows physical genetic interactions        centration 1 mg/ml unless stated. cases subunits ORC (Merchant et al. 1997; Homesley           ADE2 reporter gene used, cultures diluted et al. 2000; Kawasaki et al. 2000; Douglas 2003),     300 cells/dropout plate. Colonies grown 3 days                                                                30&#194;&#176; incubated 3 days 4&#194;&#176; color development  implicated heterochromatin-induced silencing.          fore counted. <br /> Intriguingly, estimated  40,000 copies         Factor confrontation (shmooing) mating assays: Mcm10p cell (Kawasaki et al. 2000), far exceeding       shmooing assay, 50 ml saturated MATa culture abundance  400 DNA replication origins budding          streaked middle YPD plate incubated 30&#194;&#176; <br /> yeast. reason abundance clear,    4&#226;&#8364;&#8220;6 hr factor diffuse media. small <br />                                                                sample MATa culture tested spotted near likely Mcm10  previously          edge plate dissecting microscope needle used <br /> unidentified functions.                                        line individual unbudded MATa cells approximately  <br />    study demonstrate Mcm10 plays role        half optical field  3160 magnification) away heterochromatin silencing S. cerevisiae. mutant      MATa streak. Cells incubated 30&#194;&#176; 16&#226;&#8364;&#8220;18 hr alleles MCM10 manifest significant defects silenc-       resulting phenotypes MATa cells scored. patch- <br />                                                                mating assays, strains tested streaked par- <br /> ing telomeric HMR loci. Consistent        allel pattern YPD plates, grown overnight 30&#194;&#176;, observation, mcm10 mutant cells exposed          replica plated tester strains 6697-1 6697-3, opposite factor, &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo-cluster&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype, sig-     mating type minimal media plate. Plates incu- <br /> nifying defect maintenance silent chromatin        bated 2 days 30&#194;&#176;. HML locus (Enomoto Berman 1998).                    Protein-protein interactions: pGADgw SIR RAP1 <br />                                                                hybrid plasmids constructed using Invitrogen hybrid interactions Mcm10               Gateway cloning  pGAD2F pBTM116 plasmids components silencing machinery             (Fields Song 1989) converted destination interactions disrupted Mcm10-1            vectors insertion Gateway recombination cassette <br /> Mcm10-43 mutant proteins. Interestingly, temperature-      BamHI site. length pBTM.MCM10  <br /> sensitive phenotype mcm10 mutant cells genet-        scribed Merchant et al. (1997). pGAD2F pBTM116 <br />                                                                constructs transformed hybrid strain EGY40 <br /> ically separated silencing defect second-site      carrying pSH18-34 reporter plasmid (Fields Song <br /> mutations suppress temperature sensitivity        1989). Interactions assessed appearance blue silencing defect. findings demon-         colonies plates containing X-gal (Sigma, St. Louis).                                 Mcm10 Gene Silencing                                        505 <br />  <br />                                         TABLE 1 <br />                                 Strains used study <br />  <br /> Strains                                   Genotype                               Source <br />                                     Isogenic W303 <br /> W303-1A           MATa ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3-1 <br /> W303-1B           MATa ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 ura3-1 <br /> BTY100            W303 MATa mcm10-1                                           lab <br /> BTY101            W303 MATa mcm10-1                                           lab <br /> BTY103            W303 MATa mcm10-43                                          lab <br /> BTY102            W303 MATa mcm10-43                                          lab <br /> YB541             MATa adh4TURA3 Tel (VII-L)                                  B. Stillman <br /> YB697             MATa hmrTADE2                                               B. Stillman <br /> ILY171            W303 MATa hmrTADE2 adh4TURA3 Tel (VII-L)                    study <br /> ILY180            ILY171 mcm10-1                                              study <br /> ILY270            ILY171 mcm10-43                                             study <br /> GCY23             W303 MATa RDN1TMET15 DAhmrTTRP1                             D. Shore <br />                     adh4TURA3 Tel (VII-L) sir2D met15 ADE2 <br /> ILY163            W303 MATa adh4TADE2 Tel (VII-L)                             study <br /> ILY165            ILY163 mcm10-1                                              study <br /> ILY167            ILY163 mcm10-43                                             study <br /> YJB1838           W303 MATa cac1TLEU2                                         J. Berman <br /> YJB1940           W303 MATa sir1THIS3                                         J. Berman <br /> ILY153            W303 MATa sir1THIS3                                         study <br /> ILY154            W303 MATa cac1TLEU2                                         study <br /> ILY155            YJB1940 mcm10-1                                             study <br /> ILY156            ILY153 mcm10-1                                              study <br /> ILY157            YJB1940 mcm10-43                                            study <br /> ILY158            ILY153 mcm10-43                                             study <br /> ILY159            YJB1838 mcm10-1                                             study <br /> ILY160            ILY154 mcm10-1                                              study <br /> ILY161            YJB1838 mcm10-43                                            study <br /> ILY162            ILY154 mcm10-43                                             study <br /> ILY155            YJB1940 mcm10-1                                             study <br /> ILY157            YJB1940 mcm10-43                                            study <br /> ILY159            YJB1838 mcm10-1                                             study <br /> ILY161            YJB1838 mcm10-43                                            study <br /> ILY212            W303 MATa sir1THIS3 cac1TLEU2                               study <br /> ILY235            YB541 sir1THIS3                                             study <br /> ILY237            YB541 cac1TLEU2                                             study <br /> ILY239            ILY235 mcm10-1                                              study <br /> ILY241            ILY237 mcm10-1                                              study <br /> ILY220            ILY171 Mcm2-S619Y                                           study <br /> ILY221            ILY171 Mcm2-S619F                                           study <br /> ILY222            ILY171 Mcm2-S619Y mcm10-1                                   study <br /> ILY223            ILY171 Mcm2-S619F mcm10-1                                   study <br /> YEA80             MATa AebTUASg hmrTURA3                                      R. Sternglanz <br /> YEA82             MATa aeBTUASg hmrTURA3                                      R. Sternglanz <br /> ILY137            YEA80 MATa mcm10-1                                          study <br /> ILY139            YEA80 MATa mcm10-43                                         study <br />  <br />                                   backgrounds <br /> EGY40[pSH18-34]   MATa ura3-52 trp1-1 leu2-3,112 [pSH18-34]                   E. Golemis <br /> ILY267            EGY40[pSH18-34] sir4THIS3                                   study <br /> ILY268            EGY40[pSH18-34] sir3THIS3                                   study <br /> ILY269            EGY40[pSH18-34] sir2THIS3                                   study <br />                     MATa ura3-52 his3-D200 leu2-3,112 trp1-901 gal4D gal80D <br /> PJ69-4a           LYS2TGAL1-HIS3 GAL2-ADE2 met2TGAL7-LacZ                     P. James <br /> 6697-1            MATa met4                                                   lab <br /> 6697-3            MATa met4                                                   lab <br />  506                                                     Liachko B. K. Tye <br />  <br />                                                                 TABLE 2 <br />                                                       Plasmids used study <br />  <br />          Plasmid                                          Description                                Source <br />          pRS315                                  YCP LEU2                                          New England Biolabs <br />          pRS315MCM10                             YCP LEU2 MCM10                                    lab <br />          pGAD2F                                  2m LEU2 GAD4-AD                                   S. Fields <br />          pBTM116                                 2m TRP1 LEXA-DBD                                  S. Fields <br />          pSH18-34                                URA3 LacZ LEXA binding sites                 S. Fields <br />          pGADgw                                  pGAD2F Gateway cassette                      study <br />          pBTMgw                                  pBTM116 Gateway cassette                     study <br />          pGBKgw                                  pGBKT7 Gateway cassette ampr                 study <br />          pBTMMCM10                               pBTMgw MCM10                                      study <br />          pBTMmcm10-1                             pBTMgw mcm10-1                                    study <br />          pBTMmcm10-43                            pBTMgw mcm10-43                                   study <br />          pBTMMCM10-N128                          pBTMgw MCM10 (128-571)                            study <br />          pBTMMCM10-RIGHT                         pBTMgw MCM10 (292-571)                            study <br />          pBTMMCM10-C108                          pBTMgw MCM10 (1-463)                              study <br />          pBTMMCM10-LEFT                          pBTMgw MCM10 (1-291)                              study <br />          pBTMMCM10-MIN                           pBTMgw MCM10 (128-463)                            study <br />          pBTMSIR2                                pBTMgw SIR2                                       study <br />          pBTMSIR3                                pBTMgw SIR3                                       study <br />          pBTMSIR4                                pBTMgw SIR4                                       study <br />          pGADSIR1                                pGADgw SIR1                                       study <br />          pGADSIR2                                pGADgw SIR2                                       study <br />          pGADSIR3                                pGADgw SIR3                                       study <br />          pGADSIR4                                pGADgw SIR4                                       study <br />          pGADRAP1                                pGADgw RAP1                                       study <br />          pGADMCM10                               pGADgw MCM10                                      study <br />          pGADmcm10-1                             pGADgw mcm10-1                                    study <br />          pGBKMCM10                               pGBKgw MCM10                                      study <br />          pGBKmcm10-1                             pGBKgw mcm10-1                                    study <br />          pGBT9.C                                 YCP GBD cloning vector TRP1                       P. Bartel <br />          pGBT9gw                                 pGBT9.C Gateway cassette                     study <br />          pGBT9MCM10                              pGBDgw MCM10                                      study <br />          pGBT9SIR1                               pGBDgw SIR1                                       study <br />          pM165                                   GBD-Sir1p HIS3 2mm                                D. Shore <br />          pM2046                                  GBD-Sir2p HIS3 RAP1prom                           D. Shore <br />          pM1588                                  GBD-Sir4p HIS3 RAP1prom                           D. Shore <br />  <br />  <br />  <br /> microliters saturated culture started 10 20 colo-         containing recessive, temperature-sensitive mcm10 mu- <br /> nies transformation spotted X-gal plates              tations (mcm10-1 mcm10-43) reporter genes <br /> photographs. <br />                                                                        (URA3 ADE2) integrated telomeric HMR <br />    Microcolony growth rate assay: Individual, unbudded YB541 <br /> cells overnight culture lined row parallel     silent loci yeast genome. URA3 constructs, row ILY100 cells, using dissecting microscope. Cells       silencing assayed measuring sensitivity cells <br /> incubated media plates indicated pictures taken         5-FOA, toxic precursor <span id='am-1' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> kills cells ex- 3160 magnification.                                                 pressing URA3 gene product. URA3 silenced, <br />    mcm10-1 suppressor screen: microliter sam- <br />                                                                        cells survive 5-FOA silencing ples overnight cultures mcm10-1 strain plated YPD plates incubated 2 days 37&#194;&#176;. Resulting colonies         defective, cells inviable. strains  screened recessive cold sensitivity 14&#194;&#176;. Tetrad anal-      taining telVIILTURA3 reporter, mcm10 mutant cells <br /> ysis used identify single-gene second-site suppressors.         showed sensitivity 5-FOA approximately 4 mutations cloned transforming YCp50 yeast                orders magnitude greater observed wild- <br /> genomic library (Rose et al. 1987) screening comple- <br />                                                                        type cells (Figure 1A). addition, larger proportion <br /> mentation cold-sensitive defect followed sequencing. <br />                                                                        mcm10 cells telomeric URA3 reporter                                                                        able form colonies media lacking uracil                            RESULTS <br />                                                                        observed MCM10 cells reporter (data <br />   mcm10 mutations affect silencing telomeric                    shown). Derepression URA3 reporter HMR loci: investigate effects mcm10 mutants                  experiments indicates mcm10 mutants silencing, constructed series isogenic strains              defective telomeric silencing. ensure                                                      Mcm10 Gene Silencing                                                      507 <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;MCM10 required proper telomeric HMR silencing.   Telomeric silencing assays mcm10 mutants. Strains <br /> ILY171, ILY180, ILY270, GCY23, transformed pRS315  rows row) pRS315MCM10 (rows <br /> labeled &#226;&#8364;&#732;&#226;&#8364;&#732;1MCM10&#226;&#8364;&#8482;&#226;&#8364;&#8482;) shown. Tenfold serial dilutions overnight cultures plated CM-LEU media  <br /> dicated concentration 5-FOA. Plates grown 2 days 30&#194;&#176;. (B) Silencing assays mcm10 mutants using ADE2 reporter <br /> gene. Cultures grown overnight liquid selective media, diluted, plated CM-URA plates  300 cells/plate. Plates grown 3 days 30&#194;&#176; incubated 4&#194;&#176; 3 days colonies scored color. Strains ILY171, ILY180, ILY270 used left ILY163, ILY165, ILY167 right. strains transformed pRS315 (&#195;&#191;) pRS315MCM10 <br /> (1). data points averages trials. Error bars appear missing small displayed. <br />  <br />  <br /> silencing defect mcm10 mutants promoter                (Figure 1, B). findings specific, assayed telomeric silencing utilizing           silencing defects caused mcm10-1 telVIILTADE2 reporter (Figure 1B, left). ADE2               mcm10-43 mutations recessive independent gene silenced, resulting yeast cells form pink           reporter genes used global effect colonies, loss ADE2 silencing rise           telomeric HMR loci alike. <br /> white colonies. wild-type cells, vast majority            Mcm10p plays role maintenance colonies pink white/pink sectored,                establishment silent chromatin HML: address <br /> mcm10 colonies exclusively white. results              role MCM10 HML silencing, asked experiment showed mcm10 mutations engen-                   MCM10 required proper silencing endog- <br /> der dramatic derepression ADE2 telomere.                 enous MATa1 MATa2 genes present HML <br />    investigate silencing defect                locus. wild-type MATa cell responds pheromone <br /> mcm10 mutants applies silent loci, inte-             factor) produced MATa cells arresting grated ADE2 HMR locus. Results ex-                forming projection called &#226;&#8364;&#732;&#226;&#8364;&#732;shmoo.&#226;&#8364;&#8482;&#226;&#8364;&#8482; MATa cell <br /> periment similar observed telomeric             HMLa derepressed expresses information <br /> locus; mcm10 strains hmrTADE2 reporter gave                   mating types longer responsive rise exclusively white colonies, wild-             factor (Pillus Rine 1989).  possible <br /> type strains reporter predominantly               measure silencing HMLa individual MATa <br /> pink (Figure 1B, right). results indicate             cells assaying response factor.  <br /> silencing defect mcm10 mutants affects HMR                    comparing mcm10 mutant cells respond telomeric loci.                                                  factor known silencing mutants possible    investigate silencing defects caused               determine mcm10 mutants defective mcm10 mutations dominant recessive,                  establishment maintenance HML silencing. <br /> single-copy ARS/CEN plasmids wild-                    sir1 mutant, defect establishment, <br /> type copy MCM10 transformed                 maintenance silent state (Pillus reporter strains. presence MCM10-expressing                 Rine 1989; Rusche et al. 2002), formed shmoos plasmid restored silencing mcm10 mutant strains               colonies representing silenced derepressed wild-type levels vectors did              populations cells, respectively (Figure 2A).  508                                                Liachko B. K. Tye <br />  <br />  <br />  <br />  <br />                                                                                         Figure 2.&#226;&#8364;&#8221;Mcm10 involved                                                                                      maintenance HML silencing.   <br />                                                                                      Shmoo phenotype analysis mcm10-1. <br />                                                                                      Individual, unbudded cells strains <br />                                                                                      W303-1A, YJB1940, YJB1838, BTY100, <br />                                                                                      ILY155, ILY159, ILY212 ex- <br />                                                                                      posed factor overnight YPD <br />                                                                                      plates shmooing pheno- <br />                                                                                      types scored. Approximately 150 <br />                                                                                      cells strain analyzed                                                                                      independent trials. (B) Patch-mating <br />                                                                                      assays various mcm10 silenc- <br />                                                                                      ing mutants. Cells indicated rele- <br />                                                                                      vant genotype mated tester <br />                                                                                      strains. MATa strains mated tes- <br />                                                                                      ter strain 6697-1  lawn) MATa <br />                                                                                      strains mated 6697-3  lawn). <br />                                                                                      Mated cells allowed grow                                                                                      SD plates 30&#194;&#176; 2 days. (C) mcm10 <br />                                                                                      mutants defective mainte- <br />                                                                                      nance/inheritance telomeric silenc- <br />                                                                                      ing. rates colony size increase <br />                                                                                      mutant   wild-type   <br />                                                                                      strains compared lining indi- <br />                                                                                      vidual unbudded cells synthetic com- <br />                                                                                      plete media 5-FOA. <br />                                                                                      BTY100   left), W303-1A   <br />                                                                                      left), ILY180   right panels), <br />                                                                                      YB541   right panels) <br />                                                                                      shown. right panels                                                                                      pictures colonies dif- <br />                                                                                      ferent time intervals. 30                                                                                      cells analyzed. Plates incu- <br />                                                                                      bated 30&#194;&#176;. <br />  <br />  <br />  <br />  <br /> study,  30% sir1D cells formed colonies, consis-         cells form shmoos able stably maintain <br /> tent previous results (Pillus Rine 1989). Cac1         silent state prolonged period time, largest subunit chromatin assembly factor 1       cells form shmoo clusters  mcm10 <br /> (CAF-1) complex deposits histones H3/H4               mutants  mcm10-1 data shown) displayed pheno- <br /> newly replicated DNA (Verreault et al. 1996). cac1D           type similar cac1D strain strain, defective maintenance si-        sir1D strain (Figure 2A). mcm10 cells did lent state, establishment (Enomoto et al.        form colonies, likely establishment pathway <br /> 1997; Enomoto Berman 1998) did form                 mutants intact, cells able colonies, formed increased number shmoo               respond factor. results suggest Mcm10p <br /> clusters (Figure 2A). phenotype indicative           plays role maintaining HML silent chromatin, <br /> cells able respond factor, able    establishing  maintain arrested phenotype extended                confirm previous results, tested  <br /> period time divide arresting becom-           sponse double mutants factor. sir1D mcm10 cells <br /> ing shmoos  shmoos shmoo clusters                showed striking ablation silencing, just like sir1D <br /> represent cells respond factor, indicating      cac1D strain (Figure 2A). strains, cells stable establishment silencing. difference       completely unresponsive factor,  85% <br />                                                  Mcm10 Gene Silencing                                              509 <br />  cells colonies,  5% cells            investigate mcm10 mutant cells properly able form shmoos  10% formed shmoo                propagate telomeric silent state, placed individ- <br /> clusters. observation indicates greater           ual wild-type mutant unbudded cells containing derepression HMLa locus double mutants          telVIILTURA3 reporter media containing 5-FOA single mutants. strains bearing      compared growth rates colony size (Figure 2C). cac1D mcm10 mutations 10&#226;&#8364;&#8220;20%             MCM10 ura3 cells bearing telomeric URA3 reporter <br /> increase number shmoo clusters formed               proliferated rate comparable isogenic cells <br /> (Figure 2A), signifying slightly greater defect      reporter 5-FOA regular <br /> maintenance silent state     complete media. Similarly, mcm10-1 ura3 cells grew mutants  absence colonies formed          colonies fast MCM10 ura3 cells 5-FOA. mcm10 cac1D double mutant supports notion            contrast, mcm10-1 cells bearing URA3  proteins play role maintenance,             porter proliferated rate comparable establishment silencing HML.                            wild-type counterparts media 5-FOA, 5-FOA <br />    way measure HML silencing measure         significantly retarded proliferation. Similar results ability MATa cells bearing appropriate mutations       observed mcm10-43 cells  shown). mate MATa cells patch-mating assays (Figure        results suggest mcm10 mutants defective 2B). HMLa region derepressed MATa cells,         maintenance inheritance silent telomeric cells mate efficiently. experiment      chromatin. reverse measure silencing HMRa         Mcm10 interacts Sir2 Sir3: Mcm10 plays locus assaying ability MATa cells mate.       direct role silencing, interact compo- studies single mutants MCM10, SIR1,        nents silencing machinery. test possibility, CAC1 showed mating defects mating type,         used yeast hybrid assays investigate double mutants did. sir1D mcm10 double            Mcm10 interacts Sir1, Sir2, Sir3, Sir4, Rap1. mutants showed significant MATa-specific mating            combinations BTM-MCM10 GAD-SIR2 GAD- <br /> fects (Figure 2B), consistent abolishment          SIR3 showed activation LacZ reporter indicated <br /> HMLa silencing observed Figure 2A. cac1D mcm10             blue color colonies formed media <br /> double mutants showed dramatic MATa-specific           containing X-gal (Figure 3A), signifying Mcm10 inter- <br /> mating defect. Similar effects observed          actions Sir2 Sir3. interactions fore maintenance mutants (Enomoto et al.           disrupted single-amino-acid substitution muta- <br /> 2000). findings consistent role     tions, mcm10-1 (P269L) mcm10-43 (C320Y). Western Mcm10 maintenance silencing.                    blot analysis BTM-mcm10-1 BTM-mcm10-43  <br />    mcm10 mutants defect maintenance             dicated fusion proteins stably expressed inheritance telomeric silencing: Pillus         (Figure 3B). results largely confirmed Rine (1989) showed HM chromatin sir1 mutant           switching bait/prey orientation hybrid <br /> cells stably exist epigenetic states&#226;&#8364;&#8221;-      experiment (Figure 3C). lack interaction  <br /> silent active. cells able propagate     tween BTM-SIR3 GAD-MCM10 probably state progeny generations. stably        interference BTM tag Sir3 function pre- <br /> propagate silencing generation, mother          viously observed   Lustig, personal communication). <br /> cell maintain silent chromatin      plasmid combinations showed inherited daughter cell. Inheritance silent     increase LacZ activity (Figure 3A). <br /> state assayed measuring silenced           set hybrid experiments mother cell gives rise derepressed daughter cell          using GBK-tagged proteins bait HIS3 <br /> (Monson et al. 1997). case telomeric           gene reporter (Figure 3D).  GBK- <br /> URA3 silencing reporter used, cell divides,      MCM10 showed interactions GAD-SIR2 GAD- silencing properly inherited daughter       SIR3 GBK-mcm10-1 disrupted interactions, <br /> cell, Ura1 die presence    confirming results observed Figure 3A. <br /> 5-FOA.  telVIILTURA3 cell defect            identify Sir2- Sir3-interacting domain inheritance maintenance silencing lead        Mcm10, carried hybrid analysis trunca- <br /> slower growing colony, proportion progeny      tions Mcm10 (Figure 3E). shown  length dying 5-FOA. observing rate colony       Mcm10 showed strong interaction Sir2 signif- <br /> growth 5-FOA possible assay maintenance/          icant interaction Sir3. Interactions Sir2 inheritance telomeric silencing cells bearing         Sir3 uncompromised N-terminal deletion URA3 reporter.  assay able ade-        128 amino acids Mcm10 (128-571) somewhat <br /> quately distinguish defects inheritance        compromised deletion Mcm10 (292-571). maintenance silent chromatin, si-        contrast, C-terminal deletion 108 amino acids lencing lost maintenance defect        Mcm10 (1-463) completely destroyed interactions inheritance step takes place.                             Sir2 Sir3. Western blot analysis indicated  510                                                  Liachko B. K. Tye <br />  <br />  <br />  <br />  <br />   Figure 3. C terminus Mcm10 interacts silencing proteins. Yeast hybrid bait/prey constructs transformed reporter strains EGY40[pSH18-34]   B, D, E), PJ69-4a (C).   B, D, E) Cultures spotted selective X-gal <br /> plates. interaction indicated blue color resulting activation LacZ reporter. (B) Cultures EGY40[pSH18-34] <br /> expressing BTM-Mcm10 fusions grown overnight CM-TRP media. Cells 1-ml aliquots cultures boiled run Western blots according standard protocols. blot probed rabbitt anti-LexA antibody  gift Jeff <br /> Roberts), stripped, reprobed rabbit anti-actin antibody  gift Tim Huffaker). (C) Saturated cultures serially <br /> diluted spotted selective plates. interaction indicated growth CM media caused activation HIS3 <br /> reporter. (E) hybrid experiments performed Dsir2, Dsir3, Dsir4 deletion strains ILY267, ILY268, ILY269. <br />  truncation fusions stably expressed                mediated Sir3 vice versa interactions (Figure 3B). results indicate interacting            Mcm10, carried hybrid analysis strains <br /> domain Mcm10 Sir2 Sir3 located                deleted SIR2, SIR3, SIR4 (Figure 3F). C-terminal region. speculate disruption             Sir2 Sir3 interacted Mcm10 independent <br /> interaction mcm10-1(P269L) result               of Sir proteins. <br /> overall structural change Mcm10 protein                   Tethering Mcm10p defective HMR-E silencer compromises C-terminal interacting domain.                      sufficient restore silencing. Directly recruiting <br />    finding Sir2 Sir3 interact                Sir proteins Rap1 defective silencer <br /> Mcm10 C-terminal domain Mcm10                        restore silencing (Chien et al. 1993; Buck result indirect interaction Sir              Shore 1995; Lustig et al. 1996; Marcand et al. 1996). <br /> protein mediated  investigate Sir2              investigate tethering Mcm10 sufficient <br />                                                  Mcm10 Gene Silencing                                                511 <br />  <br />                                                                                            Figure 4.&#226;&#8364;&#8221;Tethering Mcm10 <br />                                                                                         defective silencer suf- <br />                                                                                         ficient restore silencing.   <br />                                                                                         GBD-Mcm10 able restore <br />                                                                                         silencing HMR-E (depicted                                                                                         schematic). Strains YEA80   <br />                                                                                         YEA82   trans- <br />                                                                                         formed pGBT9, pGBT9SIR1, <br />                                                                                         pGBT9MCM10. Tenfold serial <br />                                                                                         dilutions plated indi- <br />                                                                                         cated media grown 30&#194;&#176;                                                                                         2 days. (B) GBD-MCM10 construct <br />                                                                                         able restore temper- <br />                                                                                         ature sensitivity silencing <br />                                                                                         defect mcm10 strains. Strains <br />                                                                                         YB541, ILY180, ILY102                                                                                         transformed pGBT9 <br />                                                                                         pGBT9MCM10                                                                                          rows). Tenfold serial dilutions <br />                                                                                         plated selective media <br />                                                                                         5-FOA 30&#194;&#176;   <br />                                                                                         missive temperature) com- <br />                                                                                         plete media 37&#194;&#176; (restrictive <br />                                                                                         temperature). <br />  <br />  restore silencing, used strains si-        complement defect adding plasmid containing <br /> lencer disrupted replaced Gal4 binding sites       single copy MCM10 (Figure 5B). Interestingly, <br /> (Andrulis et al. 2002). mating-type gene cassette      mutations MCM10 did effect targeted HMR replaced URA3, providing way            silencing GBD-SIR1, GBD-SIR2, GBD-SIR4, measure silencing locus. plasmid            require participation Sir3 silenc- <br /> construct expressing GBD-MCM10 fusion. hybrid          ing. findings suggest silencing <br /> gene unable restore silencing HMR-E,        function MCM10 lie downstream Sir Rap1 Abf1 binding sites (Aeb, Figure 4A,       proteins, Mcm10 regulating silencing activity  ORC Rap1 binding sites (aeB, Figure 4A,       Sir3.  deleted, expression GBD-SIR1            temperature sensitivity silencing defects able restore silencing cases (Figure 4A).    mcm10-1 genetically separated: Mcm10 possible fusion protein unstable oth-    plays role DNA replication, possible erwise defective, checked ability complement        silencing defect caused mcm10 mutations  <br /> mcm10 mutations. GBD-MCM10 able complement             product replication defect overall defective temperature sensitivity silencing         S phase. address point asked temper- <br /> fect mcm10-1 mcm10-43 (Figure 4B), suggesting          ature sensitivity  attributed defect hybrid protein retains silencing function.       replication) silencing defects mcm10-1 Taken  data recruiting Mcm10         separated. studies Mcm10 function  sufficient restore silencing defective         ducted screen second-site suppressors silencer.                                                     temperature-sensitive phenotype mcm10-1. screened <br />    Mutations MCM10 inhibit ability GBD-SIR3         spontaneous reversion mcm10-1 temperature restore silencing: Normally, tethering Sirs     sensitivity conferred second phenotype (cold defective silencer sufficient restore silencing    sensitivity) mutation easily identi- <br /> (Chien et al. 1993; Lustig et al. 1996; Marcand et al.        fied. isolated mutants, MCM2-S619Y MCM2- <br /> 1996). Mcm10 functions downstream Sirs,        S619F. suppressors capable overcoming mcm10 mutation inhibit ability GBD-SIR           temperature sensitivity mcm10-1 (Figure 6D) constructs restore silencing defective HMR-E.      dominant fashion (data shown). constructed <br /> constructed tethering yeast strains containing         strains contain HMR telomeric silencing <br /> mcm10-1 mcm10-43 mutations defective        reporters, mcm10-1, suppressor mutations silencer GBD binding sites HMR marked           assay mutations suppress URA3. expected, tethering Sirs MCM10       telomeric silencing defect conferred mcm10-1. strain restored silencing approximately wild-type          mutations Mcm2-S619Y Mcm2-S619F did confer levels (Figure 5A, . hand, targeted          silencing defect (Figure 6, B). silencing GBD-SIR3 defective mcm10             mcm2 alleles able suppress temperature- <br /> mutant strains (Figure 5A, . able           sensitive phenotype caused mcm10-1 mutation, <br />  512                                                    Liachko B. K. Tye <br />  <br />                                                                     Recently, involvement silencing proteins Sir2 <br />                                                                     Sir3 regulation replication initiation                                                                     reported (Pappas et al. 2004). relation- <br />                                                                     ship multistep processes                                                                     participating components remains enigma. Mcm10 <br />                                                                     DNA replication factor involved initi- <br />                                                                     ation elongation steps DNA replication (Homesley <br />                                                                     et al. 2000; Wohlschlegel et al. 2002; Lee et al. 2003; Fien <br />                                                                     et al. 2004; Sawyer et al. 2004). Mcm10                                                                     previously shown interact DNA replication <br />                                                                     proteins implicated silencing (Homesley et al. 2000; <br />                                                                     Kawasaki et al. 2000), direct silencing function                                                                     described. results presented study suggest <br />                                                                     Mcm10 interacts Sir2 Sir3 mainte- <br />                                                                     nance transcriptional silencing function <br />                                                                     appears separable functions essential                                                                     DNA replication. <br />                                                                        demonstrated recessive alleles                                                                     MCM10 able derepress URA3 ADE2 reporter <br />                                                                     genes telomeric HMR loci (Figure 1). addition, <br />                                                                     mutations able derepress endogenous <br />                                                                     genes HML (Figure 2, B).  Mcm10                                                                     ubiquitous role silencing, contrast ORC, <br />                                                                     plays role HM silencing. si- <br />                                                                     lencing S. cerevisiae occurs ribosomal DNA <br />                                                                     loci, effects Mcm10 rDNA silencing                                                                     examined study. Using reporter genes HMR, <br />                                                                     mcm10 mutants clearly showed derepression HMR <br />                                                                     (Figure 1B).  mating assays mcm10 sir1D <br />                                                                     double mutants showed MATa-specific mating defect <br />                                                                     supporting notion mutants derepress <br />                                                                     HML efficiently HMR (Figure 2B). phe- <br />                                                                     nomenon observed mainte- <br />                                                                     nance mutants (Enomoto et al. 2000)    Figure 5.&#226;&#8364;&#8221;Mutations MCM10 inhibit ability GBD-         likely attributed fact silencing stronger <br /> Sir3 restore silencing tethered defective silencer.    HMR HML. <br />   Tethering assays presence mcm10 mutations.               possibility Mcm10 separate <br /> Strains YEA80 (WT), ILY137 (mcm10-1), ILY139 (mcm10-            silencing function supported observation 43) transformed pM165, pM2046, pM1587, pM1588, respectively. Tenfold dilutions overnight cultures <br />                                                                     dominant extragenic suppressors (MCM2-S619Y spotted appropriate dropout media           MCM2-S619F ) temperature sensitivity mcm10-1 <br /> 5-FOA. strain background (Strain) tethering plasmid         able suppress silencing defect (Figure 6). <br /> used (Teth.) indicated. strains transformed       fact, possible Mcm10 execute si- vector pRS315 (GBD) controls. strains        lencing function replication fork silencing defect indicated asterisk. (B) MCM10 <br /> restores silencing strains transforming                                                                     silencing function uncoupled replica- <br /> strains pRS315MCM10. Silencing assayed             tion activity.  rule unlikely <br />                                                                     possibility apparently independent phe- failed restore proper telomeric (Figure 6,            notypes manifestation defect dif- <br /> B) HMR (data shown) silencing mcm10-1                  ferent levels severity. Future experiments aid <br /> strain. results suggest temperature-                 separation function mutations MCM10 sensitive phenotype mcm10-1 linked                 clarify point. <br /> silencing defect.                                                      Tethering silencing proteins defective silencer <br />                                                                     restore silencing.  tethering GBD-Mcm10 <br />                                                                     defective silencer restore silencing <br />                         DISCUSSION <br />                                                                     (Figure 5). finding suggests Mcm10 functions <br />   Involvement numerous DNA replication proteins                  downstream establishment silent chromatin ORC, PCNA, RF-C, Cdc45, Mcm5 tran-               propagation maintenance silent chromatin. <br /> scriptional silencing implicated S. cerevisiae.         hypothesis supported epistasis analysis                                                   Mcm10 Gene Silencing                                                        513 <br />  <br />                                                                                           Figure 6. temperature sensitiv- <br />                                                                                        ity mcm10-1 separated                                                                                        silencing defect. MCM2-S619Y MCM2- <br />                                                                                        S619F suppress temperature sensitivity <br />                                                                                        mcm10-1, silencing defect. <br />                                                                                        Tenfold serial dilutions overnight cul- <br />                                                                                        tures strains ILY171, ILY180, ILY220, <br />                                                                                        ILY221, ILY222, ILY223 plated <br />                                                                                        complete media plates 37&#194;&#176; (D)                                                                                        30&#194;&#176;  C). B growth                                                                                        synthetic complete media containing <br />                                                                                        0.5 mg/ml (low FOA) 1 mg/ml (high <br />                                                                                        FOA). Plates grown  2 days. <br />  <br />  <br />  <br /> sir1 cac1 shmoo assay reporter              interaction Mcm10 Sir2 Sir3. <br /> silencing assays (Figure 2, B). MCM10          Deposition Mcm10 propagating heterochromatin epistasis group CAC1, encodes subunit          mediated elongation machinery (Figure histone chaperone CAF-1 known involved         7A) drawn directly soluble Mcm10 (Figure 7, maintenance heterochromatin. different       B).  function Mcm10 simply <br /> epistasis group SIR1, role silencing          stabilize established silent chromatin, mat- <br /> restricted establishment. Previous tethering experi-        ter GBD-SIR fusion used initiate silencing ments Sir proteins showed recruiting          tethering experiments. data suggest Sirs defective silencer restore silencing    absence wild-type Mcm10, GBD-Sir3 difficulty wild-type levels (Chien et al. 1993; Marcand et al.         nucleating spread silent chromatin. fact 1996; Triolo Sternglanz 1996).                 GBD-Sir3-mediated silencing affected mcm10 <br /> ducted similar experiments mcm10 strains, noted          strains indicate defect occurs GBD-Sir3 constructs lose ability properly        steps GBD-Sir3 binds DNA initiates <br /> restore silencing (Figure 5). result suggests        silencing Sirs recruited. Mcm10 modulate Sir3 activity consistent        mcm10 mutations inhibiting step hybrid interaction Mcm10 Sir3              artificial establishment silencing GBD-Sir3 <br /> (Figure 3).  Mcm10 interacts             trouble recruiting Sirs begin assem- Sir2, silencing GBD-Sir2 inhibited.         bly spreading Sir complexes silencer <br /> explanation GBD fusion altering         (Figure 7C). words, GBD-Sir3 structure Sir3 way make hypersensitive      completely fulfill proper function mcm10 strains mutations MCM10. explanation             transition establishment maintenance <br /> Sir2 Mcm10 interact indirectly, Sir3            requires participation Sirs Mcm10 unidentified factor.  deletion          compromised. Furthering idea (Figure 7C), SIR3 effect hybrid interaction Sir2       believe Mcm10 cooperates Sir3 Mcm10, suggesting Sir3 medi-            normal transition nucleation spreading hetero- <br /> ator. Taken  data indicate nature      chromatin silent mating-type telomeric loci Mcm10-Sir2 interaction different            (Figures 1 2) independent GBD-Sir3. ex- Mcm10-Sir3 interaction. Sir3 shown        planation consistent established model homology Orc1, member origin               cooperative mechanism silencing Sir <br /> recognition complex (Zhang et al. 2002). Mcm10            proteins. explanation consistent interacts members ORC including Orc1               hypothesized role Mcm10 replication origins (Kawasaki et al. 2000), Orc2 (Kawasaki et al. 2000;            stabilizing scaffold transition pre-replication <br /> Christensen Tye 2003), Orc3, Orc5, Orc6                complex pre-initiation complex (IC) subse- <br /> (Douglas 2003). possible functional          quently pre-IC elongation complex <br /> structural similarities silencing replication      (Homesley et al. 2000; Wohlschlegel et al. 2002; Lee <br /> proteins mediate Mcm10-Sir3 interaction.               et al. 2003; Fien et al. 2004; Sawyer et al. 2004). study- <br />    fact Mcm10&#226;&#8364;&#8482;s function lies mainte-          ing roles Mcm10 DNA replication gene <br /> nance silent chromatin tethering Mcm10              silencing begin elucidate connection  <br /> does restore silencing suggests Mcm10 act         tween DNA replication heterochromatin assembly. stabilize silent chromatin structures, Sir2-4 <br /> complexes, recruited. Stabilization         thank Nancy Douglas Erik Andrulis strains, <br />                                                                plasmids, reagents used study discussions accomplished Mcm10 binding assembled              critical reading manuscript. thank Bruce Stillman, <br /> Sir2-4 complexes chromatin (Figure 7A) escorting         Rolf Sternglanz, Erica Golemis, Judith Berman, David Shore, Ravi <br /> soluble Sir2-4 chromatin (Figure 7B), consistent       Ram, Jeff Roberts, Tim Huffaker providing strains, plasmids, <br />  514                                                        Liachko B. K. Tye <br />  <br />  <br />  <br />  <br />                                                                                                           Figure 7.&#226;&#8364;&#8221;Hypotheses                                                                                                        Mcm10 function maintenance <br />                                                                                                        silencing.   Mcm10 associ- <br />                                                                                                        ates chromatin-bound <br />                                                                                                        Sir2-4 complex stabilize silent <br />                                                                                                        chromatin. Mutant Mcm10 pro- <br />                                                                                                        teins able maintain <br />                                                                                                        interaction Sirs. Dis- <br />                                                                                                        sociation Sirs chromatin <br />                                                                                                        results derepression silent <br />                                                                                                        loci. (B) Mcm10p acts recruit <br />                                                                                                        Sir proteins silent chromatin <br />                                                                                                        needed. case                                                                                                        mcm10 mutants, inability                                                                                                        Mcm10-1p facilitate bind- <br />                                                                                                        ing sufficient number Sir <br />                                                                                                        complexes silent region <br />                                                                                                        leads degradation silent <br />                                                                                                        chromatin time. (C) Mcm10 <br />                                                                                                        facilitates transition                                                                                                        nucleation Sir2-4 complexes <br />                                                                                                        silencer spreading <br />                                                                                                        Sir2-4 complexes                                                                                                        silenced region. defective <br />                                                                                                        Mcm10 does allow proper <br />                                                                                                        spreading silent chromatin. <br />  <br />  <br />  <br />  antibodies.  <br /> </body></html>