<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Arabidopsis UVH6, Homolog Human XPD Yeast <br /> RAD3 DNA Repair Genes, Functions DNA Repair Essential Plant Growth1 <br />  <br /> Zongrang Liu2, Suk-Whan Hong3, Mindy Escobar, Elizabeth Vierling, David L. Mitchell, <br /> David W. Mount, Jennifer D. Hall* <br /> Departments Molecular Cellular Biology  L., D.W.M., J.D.H.) Biochemistry Molecular <br /> Biophysics (S.-W.H., M.E., E.V., D.W.M., J.D.H.), University Arizona, Tucson, Arizona 85721; Department Carcinogenesis, University Texas M.D. Anderson Cancer Center, 1515 Holcombe <br /> Boulevard, Houston, Texas 77030 (D.L.M.) <br />  <br />  evaluate genetic control stress responses Arabidopsis, analyzed mutant (uvh6-1) exhibits <br /> increased sensitivity UV light, yellow-green leaf coloration, mild growth defects. mapped uvh6-1 locus chromosome identified candidate gene, AtXPD, corresponding region. gene shows sequence <br /> similarity human (Homo sapiens) XPD yeast (Saccharomyces cerevisiae) RAD3 genes required <span id='am-2' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> excision <br /> repair. propose UVH6 equivalent AtXPD uvh6-1 mutants carry mutation conserved residue AtXPD transformation uvh6-1 mutants wild-type AtXPD DNA suppresses UV sensitivity defective phenotypes. Furthermore, UVH6/AtXPD protein appears play role repair UV photoproducts uvh6-1 mutant exhibits moderate defect excision UV photoproducts. defect suppressed transformation UVH6/AtXPD DNA. identified T-DNA insertion UVH6/AtXPD gene (uvh6-2). <br /> Plants carrying homozygous insertions detected analyses progeny plants heterozygous insertion.  homozygous insertions appear lethal. conclude UVH6/AtXPD gene required UV resistance essential gene Arabidopsis. <br />  <br />  <br />    DNA damage challenge organisms ex-                     damaged strand sides lesion,  <br /> posed UV irradiation. UV photoproducts consist                      moval damaged fragment, repair gap <br /> primarily cyclobutane pyrimidine dimers py-                     filling ligation (Batty Wood, 2000; Boer <br /> rimidine (6-4) pyrimidinone dimers (Mitchell                       Hoeijmakers, 2000; Prakash Prakash, 2000). <br /> Nairn, 1989; Pfeifer, 1997). lesions inhibit DNA                 potential plant homologs human yeast <br /> replication transcription promote mu-                     NER genes identified. Genetic analyses tagenesis (McGregor, 1999). effects UV irradi-                  plant genes, including studies pheno- <br /> ation especially detrimental plants, sun-                 types plants carrying mutations light source damage requirement                     genes, provide support idea NER photosynthesis.                                                    pathway conserved plants. <br />    Increasing evidence suggests plants repair UV-                    Lesion recognition NER involves ho- <br /> damaged chromosomes using mechanisms similar                        mologous heterodimers XPC:HR23B (human) humans (Homo sapiens) yeast (Sac-                   RAD4:RAD23 (yeast; Balajee Bohr, 2000; Batty <br /> charomyces cerevisiae). mechanisms include                   Wood, 2000; Boer Hoeijmakers, 2000; <br /> nucleotide excision repair (NER) pathway, process                    Prakash Prakash, 2000). Arabidopsis genome involves recognition UV lesions, incision                  contains potential homologs XPC/RAD4                                                                        HR23B/RAD23 (Arabidopsis Genome Initiative, 2000). <br />   1 <br />      work supported National Science Foundation        HR23B expression occurs Arabidopsis, rice (Oryza <br /> (grant  MCB&#226;&#8364;&#8220;9728125 D.W.M.) U.S. Department        sativa), carrot (Daucus carota), carrot gene <br /> Agriculture National Research Initiative Competitive Grants Pro-       complements repair defect yeast rad23 mutant <br /> gram (grant  99 &#226;&#8364;&#8220;3515100 E.V.).                                  (Schultz Quatrano, 1997; Sturm Lienhard, <br />    2 <br />      Present address: Appalachian Fruit Research Station, 45 Wilt-     1998). <br /> shire Road, Kearneysville, WV 25430.                                      Damage-induced incision NER involves    3 <br />      Present address: Department Applied Plant Sciences, Chon-      nucleases. nuclease makes cuts 5&#226;&#172;&#732; nam National University, 300 Yang-Bong Dong, Buk-Gu, Gwang-ju <br /> 500 &#226;&#8364;&#8220;757, South Korea. <br />                                                                        lesion consists homologous heterodimers XPF: <br />    * Corresponding author; e-mail jdhall@u.arizona.edu; fax 520 &#226;&#8364;&#8220;      ERCC1 (human) RAD1:RAD10 (yeast). homol- <br /> 621&#226;&#8364;&#8220;3709.                                                              ogous complex implicated plant DNA <br />    Article, publication <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, citation information    repair based UV sensitivity Arabidopsis www.plantphysiol.org/cgi/doi/10.1104/pp.103.021808.                 derivatives carrying mutation plant <br /> Plant Physiology, July 2003, Vol. 132, pp. 1405&#226;&#8364;&#8220;1414, www.plantphysiol.org &#194;&#169; 2003 American Society Plant Biologists     1405 <br />  Liu et al. <br />  <br />  <br /> XPF homolog (Fidantsef et al., 2000; Liu et al., 2000)     UV sensitive, indicative repair defect, antisense construct gene (Gallego et al.,    hypothesized uvh6-1 mutation maps 2000). Arabidopsis XPF mutants defective       AtXPD gene. test possibility, amplified <br /> repair UV photoproducts (Liu et al., 2000; Li et al.,   AtXPD cDNA wild-type (C10) uvh6-1 mu- <br /> 2002). Homologs ERCC1 component             tant plants  described &#226;&#8364;&#339;Materials Methods&#226;&#8364;?) nuclease occur Arabidopsis (Arabidopsis Ge-       determined corresponding sequences. <br /> nome Initiative, 2000) Lilium longiflorum (Xu et       Compared wild-type sequence, al., 1998), L. longiflorum gene complements      single-nucleotide change codon 521 uvh6-1 <br /> repair defect ERCC1-deficient Chinese hamster           mutant cDNA sequence. mutation  <br /> (Cricetulus griseus) ovary cells. nuclease     firmed sequencing mutant genomic DNA (data <br /> cises 3&#226;&#172;&#732; DNA lesions called XPG humans        shown). depicted Figure 1B, mutation <br /> RAD2 yeast. mutation Arabidopsis XPG           result <span id='am-17' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-18' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>Gly</span> Glu missense substitution. <br /> homolog results UV sensitivity (Liu et al., 2001b).      likely UVH6 gene equiv- addition, second mutation (uvr1), fails      alent AtXPD sequence change  <br /> complement AtXPG mutation   Liu D.               tected AtXPD gene uvh6-1 mutation. <br /> Mount, unpublished data) results defective repair          UVH6/AtXPD sequence obtained 6-4 photoproducts (Britt et al., 1993).                 wild-type plants nearly identical unpub- <br />    Finally, NER requires DNA unwinding              lished AtXPD mRNA sequences GenBank (acces- <br /> DNA helicases, called XPB XPD humans            sion nos. AF188623 AY062471) contains RAD25 RAD3 yeast. potential Arabidop-            possibly polymorphic, differences sis homologs XPB/RAD25 identified             coding region. comparison AF188623, (Ribeiro et al., 1998; Arabidopsis Genome Initiative,      observed differences, codon 541 (AGC/ <br /> 2000). AtXPB1 complements repair defect yeast         AGT), did alter encoded Ser amino <br /> rad25 mutants, plants carrying T-DNA insertions        acid, codon 554 (GAA/AAG), gene sensitive alkylating agents (Costa     changed encoded Glu Lys sequence. et al., 2001b). Arabidopsis encodes presumed        comparison AY062471, silent change ob- <br /> homolog XPD/RAD3 genes (Arabidopsis Ge-             served codon 282 (CGA/CGG). residues nome Initiative, 2000).                                    observed positions identical    AtXPD gene subject present study.     Arabidopsis consensus genome examined Arabidopsis mutation (called           sequence (Arabidopsis Genome Initiative, 2000), <br /> uvh6-1) dramatically increases sensitivity UV       like sequence, derived plants light. present evidence mutation results      Columbia ecotype. cDNA sequence missense change Arabidopsis XPD/           submitted GenBank (accession  AY090788) RAD3 gene. findings support possi-         depicted Figure 1B relation correspond- <br /> bility AtXPD/UVH6 gene product functions          ing genomic sequence. plant NER pathway. addition, ob- <br /> serve T-DNA insertion AtXPD (uvh6-2)          Complementation UV Sensitivity uvh6-1 <br /> lethal, suggesting gene serves essential      Mutant AtXPD DNA <br /> function plant development. <br />                                                              confirm identification UVH6 gene,                                                            characterized phenotypes uvh6-1 mutant <br /> RESULTS                                                    plants transformed wild-type <br /> Identification UVH6 Gene                    (C10) AtXPD genomic DNA. Primary T1 transfor- <br /> uvh6-1 Mutation                                            mants examined UV sensitivity, 22 26 <br />                                                            transformants displayed UV-resistant phenotype. <br />   previously identified UV-sensitive mutant        phenotype depicted Figure 2 T2 gener- <br /> Arabidopsis (originally called uvh6 renamed            ation plants homozygous introduced AtXPD <br /> uvh6-1 study) used map UVH6          transgene. shown, uvh6-1 mutant plants ab- <br /> locus 6 centimorgans chromosome            normally sensitive UV-C irradiation, compared <br /> (Jenkins et al., 1995). shown Figure 1A,     wild-type parent, exhibit severe brown- refined location gene region       ing death rosette leaves 3 d treat- physical markers nga59 AtEAT1.                 ment (compare Fig. 2A, middle left).  region, identified candi-       ing confirms previous report uvh6-1 plants <br /> date gene locus At1g03190 (Bac locus F15K9).         hypersensitive UV-C UV-B irradia- designated gene AtXPD ex-          tion (Harlow et al., 1994). contrast, transformed <br /> hibits strong similarity ERCC2 (XPD) Chi-          plants exhibit resistance irradiation <br /> nese hamster RAD3 (yeast; Arabidopsis           shown wild-type parents (right).  formation Resource, http://www.arabidopsis.org).           mutant phenotype complemented wild-type <br />   mammalian XPD yeast RAD3 genes               AtXPD DNA, indicating UVH6/AtXPD gene required NER uvh6-1 mutant         required resistance UV irradiation. <br /> 1406                                                                                     Plant Physiol. Vol. 132, 2003 <br />                                                                                     Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br />  <br />  <br />          Figure 1. Mapping UVH6 locus organization corresponding UVH6(AtXPD) gene.  3.3 megabases <br />          chromosome shown corresponding physical markers nga59, AtEAT1, nga63. UVH6 locus localized <br />          markers nga59 AtEAT1, described &#226;&#8364;&#339;Materials Methods,&#226;&#8364;? based analysis DNA 380 <br />          UV-sensitive F2 progeny (760 chromosomes) obtained cross uvh6-1 (Columbia ecotype) wild-type <br />          (Landsberg ecotype) plants. UV-sensitive recombinants carrying Landsberg nga59 polymorphism identified, <br />          lacked Landsberg nga63 AtEAT1 markers. recombinants carrying Landsberg nga63          obtained. carried Landsberg AtEAT1 marker, carried Landsberg nga59 marker.          position Bac clone F15K9, carries candidate repair gene AtXPD (gene F15K9.20, locus At1g03190),          indicated. B, UVH6/AtXPD gene diagrammed, based comparison known genomic cDNA <br />          sequences. diagram depicts exons  bars), introns  lines), initiation termination codons, uvh6-1 <br />          mutation codon 521. C, Alignment central portion human XPD (accession  CAA36463) UVH6/AtXPD <br />          (accession  AY090788) protein sequences shown, encompassing conserved helicase motifs III IV. uvh6-1 <br />          mutation <span id='am-10' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> 521 indicated, sites known human mutations (designated Hs), produce          diseases xeroderma pigmentosum trichothiodystrophy, indicated. <br />  <br />  <br />  <br /> Growth Defects uvh6-1 Mutant                                      grown normal (non-stress) conditions   addition radiation sensitivity, uvh6-1 mutant                    result low chlorophyll content plants exhibit growth defects sup-                         plants described previously (Jenkins et al., 1997). pressed plants transformed wild-type UVH6/                       addition, mutant uvh6-1 plants substantially <br /> AtXPD genomic DNA. shown Figure 2B (mid-                           smaller compared wild-type plants dle), leaves uvh6-1 mutant plants yellow-                     age, seen Figure 2C (left). contrast, <br /> green, previously reported (Jenkins et al., 1997).                    uvh6-1 plants complemented phenotype observed mutant plants                        AtXPD genomic DNA dark-green exhibit <br /> Plant Physiol. Vol. 132, 2003                                                                                                         1407 <br />  Liu et al. <br />  <br />  <br />                                                                       sequences (accession nos. CAA36463 AAB64698, <br />                                                                       respectively). Arabidopsis sequence exhibits <br />                                                                       strong similarity homologs entire <br />                                                                       length, residue mutated uvh6-1 mutant <br />                                                                       conserved comparisons. Figure 1C depicts <br />                                                                       alignment human sequence, includ- <br />                                                                       ing uvh6-1 mutation site. Additional alignments <br />                                                                        shown) XPD homologs Schizosaccha- <br />                                                                       romyces pombe, Caenorhabditis elegans, Drosophila <br />                                                                       melanogaster (accession nos. CAA93221, AAK95892, <br />                                                                       AAD33587, respectively) reveal strong se- <br />                                                                       quence similarity conservation G residue <br />                                                                       position uvh6-1 mutation. strong <br />                                                                       conservation uvh6-1 residue ho- <br />                                                                       mologs suggests mutation disrupts <br />                                                                       function UVH6/AtXPD protein. <br />                                                                          Human XPD yeast RAD3 genes encode DNA <br />                                                                       helicases essential repair (Winkler et al., 2000). <br />                                                                       Seven highly conserved motifs sequences                                                                       required helicase activity (Gorbalenya et al., <br />                                                                       1989). comparison AtXPD sequence                                                                       human yeast homologs revealed especially <br />                                                                       strong similarity regions seven motifs <br />                                                                       (data shown). Sequence conservation involving <br />                                                                       motifs III IV illustrated Figure 1C.                                                                        appears likely AtXPD plant gene acts                                                                       helicase performs role DNA repair <br />                                                                       previously demonstrated humans yeast. <br />  <br />  <br /> Figure 2. Recovery UV resistance normal growth properties      Repair 6-4 Photoproducts uvh6-1 Mutant complementation uvh6-1/uvh6-1 homozygous mutant UVH6/AtXPD DNA. Mutant uvh6-1 plants transformed              UV sensitivity uvh6-1 mutant suggests <br /> UVH6/AtXPD genomic DNA  C10 wild-type plants)              mutant defective repair UV <br /> scribed &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? C10 wild-type (UVH6/UVH6),        photoproducts. test possibility, examined <br /> uvh6-1 mutant (uvh6/uvh6), T2 generation-complemented mu-         repair 6-4 photoproducts DNA irradi- <br /> tant plants grown 2 3 weeks subsequently exposed UV-C irradiation (300 J m2) described &#226;&#8364;&#339;Materials Methods&#226;&#8364;? <br />                                                                       ated plants, comparing wild-type, uvh6-1 mutant, <br />   maintained normal growth conditions 3 weeks (B),     uvh3 mutant plants. uvh3 mutant UV <br /> 4.5 weeks (c, left), 6.5 weeks (c, right).                         sensitive carries nonsense mutation                                                                       UVH3/AtXPG gene, encoding homolog                                                                       human XPG 3&#226;&#172;&#732;-incision endonuclease (Jenkins et al., <br /> normal growth rates (Fig. 2B, right).  ad-                 1995; Liu et al., 2001b). shown Figure 3A, <br /> ditional mutant phenotypes appear result                                                                       wild-type plants exhibited significant removal (62%) mutation causes UV sensitivity. <br />                                                                       6-4 products DNA 24 h irradi- <br />                                                                       ation. comparison, uvh6-1 mutant (Fig. 3C) <br /> Relationship AtXPD Presumed Human                           exhibited reduced levels repair 24 h (47% <br /> Yeast Homologs                                                        photoproduct removal), suggesting mod- <br />   originally identified AtXPD gene based                    erate repair defect. photoproduct removal similarity encoded protein sequence hu-                 detected uvh3 mutant (Fig. 3B). man XPD yeast RAD3 proteins,                      result confirms previous report (Britt et al., 1993) established roles NER (Batty Wood, 2000;                    defective repair mutant (uvr1) fails com- <br /> Boer Hoeijmakers, 2000; Prakash Prakash,                      plement uvh3 mutant genetic tests. Finally, 2000). performed protein sequence                     uvh6-1 uvh3 mutants, complementation <br /> comparisons assess similarity gain                     appropriate wild-type genes resulted resto- <br /> insight function Arabidopsis protein.                 ration wild-type level repair. comple- <br /> Pair-wise alignments (data shown) indicate                   mented uvh3 mutant showed 66% photoproduct  Arabidopsis sequence (accession  AAF14582)                     moval (Fig. 3B) complemented uvh6-1 56% 50% identical human yeast                       mutant showed 65% removal (Fig. 3C). <br /> 1408                                                                                                Plant Physiol. Vol. 132, 2003 <br />                                                                                     Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br />                                                                          vealed plants heterozygous                                                                          homozygous wild-type allele. ho- <br />                                                                          mozygous T-DNA insertions detected, suggest- <br />                                                                          ing homozygous insertions lethal. <br />                                                                             investigate possibility, analyzed                                                                          heterozygous plants (called Ga-2 Ga-3) <br />                                                                          determine frequency progeny <br />                                                                          inherit T-DNA insert. heterozygote parents <br />                                                                          allowed produce seeds self-fertilization, <br />                                                                          DNA individual F1 seedlings subjected <br />                                                                          PCR analysis described  total, 97 <br />                                                                          progeny plants examined. seen Table  <br />                                                                          56.7% plants heterozygous 43.3% <br />                                                                          carried wild-type alleles, plants ho- <br />                                                                          mozygous insert detected. result <br /> Figure 3. Repair 6-4 photoproducts DNA UV-irradiated          clearly deviates theoretical expectation plants. DNA extracted irradiated plants assayed photo- <br />                                                                          25% progeny carry homozygous <br /> products described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? plants studied C10 wild-type (F), mutant uvh3 (&#228;&#161;&#186;), complemented mutant <br />                                                                          T-DNA insertions conditions inser- <br /> uvh3 (f), mutant uvh6-1 (&#226;&#8364;&#353;), complemented mutant uvh6-1 (&#197;&#8217;).         tion lethal (P &#226;&#172;? 0.01). result rules point represents mean separate assays, SDs   possibility lethality caused second <br /> means shown. difference 24-h data points         mutation unlinked UVH6/AtXPD locus, al- <br /> uvh6-1 complemented uvh6-1 plants statistically signifi-      presence lethal mutation closely (P &#226;&#172;? 0.025).                                                        linked UVH6 possible. likely,                                                                          results indicate homozygous uvh6-2 insertions <br /> Characterization T-DNA Insertion                                 UVH6/AtXPD lethal UVH6/ <br /> UVH6/AtXPD Effect Plant Viability                             AtXPD gene serves essential function plant <br />                                                                          growth. <br />   identified line Arabidopsis plants <br /> (called 825 BO5) carrying T-DNA insertion                                                                           Expression UVH6/AtXPD Gene designate uvh6-2) UVH6/AtXPD <br /> gene. site insertion confirmed DNA                           mRNA expression pattern UVH6/AtXPD <br /> sequencing, depicted Figure 4. Insertion oc-                       gene analyzed shown Figure 5. Total RNA <br /> curred sixth 11 introns UVH6/                       extracted plant tissues, specific <br /> AtXPD gene presumably results truncated                         mRNA sequences amplified RT-PCR, UVH6/AtXPD protein altered splicing                        PCR products visualized electrophoresis. <br /> UVH6/AtXPD mRNA. Based site insertion,                         previously use 50 ng input <br /> 26% coding sequence highly                        plant RNA gave PCR product propor- <br /> conserved helicase motifs deleted.                             tional input RNA concentration, using 25 30 <br />   preliminary analysis, asked 825 BO5 line homozygous heterozygous T-DNA insert. analysis, 10 plants grown initial seed stock, genomic DNA plant subjected PCR analysis distinguishes wild-type T-DNA- <br /> inserted alleles UVH6/AtXPD. locations primers used analysis shown Figure 4. <br /> Gene-specific primers anneal sites flanking insertion site used identify wild-type <br /> UVH6/AtXPD alleles. test detects wild-type <br /> alleles presence insertion results                 Figure 4. Site T-DNA insertion (uvh6-2) UVH6/AtXPD inter-primer distance (greater 10 kb)                    gene. position insert confirmed DNA sequencing, long PCR amplification. detect alleles car-                   described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? site shown rying insertion, second PCR reaction                       respect UVH6/AtXPD genomic DNA, UVH6/AtXPD <br /> formed using UVH6-specific primers                      DNA sequence adjacent left border (LB) shown. UVH6/ <br />                                                                          AtXPD gene depicted exons  lines) introns  primer anneals T-DNA left border. <br />                                                                          lines). Positions primers (UVH6-F, UVH6-R, LB3, LB3D)  DNA plants heterozygous insert                        used PCR reactions DNA sequencing indicated. amplified sets primers,                           results shown Table  primers UVH6-F UVH6-R (gene spe- DNA plants homozygous                        cific) used detect wild-type alleles; primers UVH6-R LB3 <br /> wild-type inserted allele amplified                      (Ga-3 experiment) LB3D (Ga-2 experiment) used detect set primers. Results analysis                     T-DNA-inserted alleles. <br />  <br /> Plant Physiol. Vol. 132, 2003                                                                                                           1409 <br />  Liu et al. <br />  <br />  <br />  <br /> Table  PCR analysis DNA progeny Ga-2 Ga-3 plants carrying heterozygous T-DNA insertion (uvh6-2) UVH6/AtXPD <br /> genea <br />                                                                          Progeny Plants Specified Genotype <br />          Genotype Progeny Plantsb <br />                                                   Progeny Ga-3 parent                Progeny Ga-2 parent              Total progenyc <br />        Homozygous wild type                                 18                                       24                          42(43.3%) <br />        Heterozygous T-DNA insertion                         29                                       26                          55(56.7%) <br />        Homozygous T-DNA insertion                            0                                        0                           0(0.0%) <br />       Genomic DNA extracted leaves F1 seedlings plants (Ga-3 Ga-2) carrying heterozygous T-DNA insertions                                                                                                                     b <br /> UVH6/AtXPD, DNA subjected PCR analysis, described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;?                       Genotypes scored homozygous wild type (amplification observed gene-specific primers), T-DNA insertion heterozygous (amplification observed sets primers), T-DNA insertion homozygous (amplification observed T-DNA/gene-specific primer pair). Figure 4 shows primer annealing sites.    c <br />                                  &#226;?&#185;2 analysis results generated value P &#226;&#172;? 0.01, indicating results consistent expected Mendelian genetic distribution 1:2:1. <br />  <br />  <br /> PCR cycles (Liu et al., 2001b). seen Figure 5,                         Glu position 521 UVH6/AtXPD protein. conditions gave clearly detectable levels                           mutation appears cause extreme UV sen- <br /> product UBQ10 control, gene ex-                              sitivity exhibited uvh6-1 plants mutant <br /> presses relatively high level. control                        plants transformed wild-type UVH6/AtXPD <br /> serves verify equivalent quantities RNA                           DNA regain wild-type radiation resistance. Mutant present PCR reaction.                             sensitivity UV light likely result defi- <br /> tect PCR product UVH6/AtXPD, ex-                                 ciency NER, based finding mutant <br /> presses relatively low level, required addi-                         plants exhibit moderate defect removal 6-4 <br /> tional input RNA (200 ng) 40 PCR cycles.                           photoproducts DNA. conditions, observed UVH6/AtXPD                                  addition, examined mutant carrying mRNA expressed various degrees leaf, root,                          T-DNA insertion (uvh6-2) UVH6/AtXPD <br /> stem, flower bud, meristem tissues.                                gene. insertion predicted produce trun- <br /> UVH6/AtXPD appears expressed low levels                             cated UVH6 gene product. findings indicate tissues, expected gene required                          homozygous insertions UVH6/AtXPD le- <br /> general transcription.                                                       thal, suggesting gene encodes essential <br />                                                                              function required plant development. <br /> DISCUSSION <br />  <br />   identified mutation (uvh6-1) Ara-                         Role AtXPD DNA Repair <br /> bidopsis AtXPD gene characterized phenotypes uvh6-1 mutant plants. AtXPD pre-                               Genes involved NER highly  <br /> dicted homolog human XPD yeast                            served yeast humans (Prakash Prakash, <br /> RAD3 genes, required NER DNA.                               2000), suggesting common mechanism. mech- uvh6-1 mutation missense change Gly                            anism appears conserved plants, based <br />                                                                              characterization presumed ho- <br />                                                                              mologs NER genes Arabidopsis (Britt et al., <br />                                                                              1993; Fidantsef et al., 2000; Gallego et al., 2000; Liu et <br />                                                                              al., 2000, 2001b; Li et al., 2002). findings                                                                              uvh6-1 mutant carries mutation presumed <br />                                                                              Arabidopsis homolog XPD (human) RAD3 <br />                                                                              (yeast) mutant sensitive UV <br />                                                                              light moderate repair defect provide addi- <br />                                                                              tional evidence NER conserved plants. <br />                                                                                 Human XPD yeast RAD3 proteins function                                                                              helicases DNA repair, presumably unwind- <br /> Figure 5. Expression UBQ10 UVH6 mRNA Arabidopsis                   ing DNA surrounding target lesions facilitate  <br /> tissues, measured reverse transcriptase (RT)-PCR. Total RNA            cision (Sung et al., 1988; Winkler et al., 2000). isolated C10 wild-type plant tissues indicated              think highly likely Arabidopsis homolog <br /> assayed, described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? RT-PCR products            UVH6/AtXPD acts repair helicase. detected agarose gel electrophoresis, product <br />                                                                              UVH6/AtXPD protein sequence contains seven <br /> amplified fully spliced message shown case. <br /> Controls RT run experiment  shown)                                                                              highly conserved motifs required helicase <br /> failed produce product, ruling contribution             function (Gorbalenya et al., 1989). uvh6-1 muta- <br /> genomic DNA amplification. RNA prepared unbolted                 tion lies motifs III IV, suggesting flower buds, roots, stems, leaves, using 4-week-old plants        impair helicase activity. addition, uvh6-1 <br /> 2-week-old meristem tissue.                                                  mutation occurs G residue highly  <br /> 1410                                                                                                                 Plant Physiol. Vol. 132, 2003 <br />                                                                       Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br /> served humans, yeast, S. pombe, C. elegans, D.        heat resistance.  rule melanogaster, suggesting residue        possibility uvh6-1 mutants impaired important normal enzymatic function               transcription se  repair defect, <br /> UVH6/AtXPD.                                                  accumulate DNA damage blocks transcription. <br />  <br />  <br />                                                              Interaction AtXPD Predicted <br /> Potential Role AtXPD Regulating Gene <br />                                                              TFIIH Complex <br /> Expression Plant Development <br />                                                                 humans yeast, TFIIH transcription ini- <br />    Mutant uvh6-1 plants exhibit phenotypic defects <br />                                                              tiation complex contains core components grown normal lighting conditions, sug- <br />                                                              additional proteins associated kinase <br /> gesting UVH6/AtXPD protein role <br />                                                              CAK complex. components, plant development addition DNA repair. <br />                                                              core proteins CAK protein appear normal, non-stress growth conditions, mutant <br />                                                              conserved Arabidopsis. Strong matches Arabi- <br /> plants small compared wild-type plants                                                              dopsis sequences observed human core com- age. yellow-green appearance, <br />                                                              ponents XPD, XPB  matched loci), p44. <br /> contain subnormal levels chlorophyll. (Jenkins et <br />                                                              matches occur loci At1g03190, At5g41370, <br /> al., 1997), exhibit poorly organized grana stacks <br />                                                              At5g41360, At1g05050, respectively. exis- thylakoid membranes (M. Jenkins, unpub- <br />                                                              tence expressed AtXPB genes (Costa et al., <br /> lished data). Furthermore, mutants abnor- <br />                                                              2001b) suggests multiple forms TFIIH mally sensitive prolonged exposure high tem- <br />                                                              exist plants. addition, CAK catalytic subunit <br /> perature (37&#194;&#176;C; Jenkins et al., 1997). addition,                                                              appears conserved, based clustering observed T-DNA insertions                                                              Arabidopsis kinase (locus At1g73690) human <br /> UVH6 gene lethal, based failure isolate <br />                                                              (cdk7) yeast (KIN28) CAK sequences phy- <br /> plants homozygous T-DNA-insertion <br />                                                              logenetic tree cyclin-dependent protein kinase se- <br /> (uvh6-2) gene.  UVH6/AtXPD appears                                                              quences (http://kinase.ucsf.edu/ksd/). contrast, <br /> serve essential function plant growth. <br />                                                              sequences similar core    understand role UVH6/AtXPD                                                              CAK components Arabidopsis. <br /> plant growth, informative consider known <br />                                                              recent report (Costa et al., 2001a) claimed <br /> functions human yeast homologs. Human <br />                                                              components (p52 XPD homologous yeast RAD3 proteins                                                              cyclinH) Arabidopsis sugarcane (Saccharum <br /> dual roles repair transcription initiation                                                               officinarum Saccharum spontaneum), sequence <br /> Boer Hoeijmakers, 2000; Prakash Prakash, <br />                                                              analyses suggest report represent <br /> 2000; Lehmann, 2001). role, facili- <br />                                                              matches isolated protein domains.  tate initiation transcription RNA polymerase II <br />                                                              potential TFIIH core CAK compo- initiation factor TFIIH. Considerable evi- <br />                                                              nents identified Arabidopsis, suggesting <br /> dence suggests human XPD mutations cause <br />                                                              complexes exist com- <br /> developmental abnormalities defects tran- <br />                                                              posed novel highly diverged subunits plants. <br /> scription. mutations produce different <br /> genetic diseases (xeroderma pigmentosum, Cock- <br /> ayne syndrome, trichothiodystrophy; Boer              Analysis T-DNA Insertions Suggests Hoeijmakers, 2000; Lehmann, 2001) result        UVH6/AtXPD Essential Gene sensitivity UV light repair deficiencies distinguished additional, distinct phenotypes.          contrast subtle growth defects uvh6-1 example, trichothiodystrophy associated          mutant plants, failed detect plants carry- <br /> brittle hair Cockayne syndrome skeletal             ing homozygous T-DNA insertions UVH6/ <br /> deformations. Substantial evidence supports idea         AtXPD gene. specific insertion charac- additional phenotypes result tran-           terized (uvh6-2) predicted cause truncation scription defects differentiated tissues,     UVH6/AtXPD protein, deleting 26% length <br /> just loss repair capacity (Bootsma Hoeijmak-          seven conserved helicase domains. <br /> ers, 1993; Boer et al., 1998; Lehmann, 2001; Liu et        overall structure presumed helicase <br /> al., 2001a; Viprakasit et al., 2001; Keriel et al., 2002).   activity protein disrupted. fail- <br />    Reminiscent patients human diseases,        ure isolate plants carrying homozygous insertions <br /> uvh6-1 mutants exhibit developmental stress-             UVH6/AtXPD suggests gene essential <br /> response defects. phenotypes consistent            plant growth. observation UVH6/AtXPD possibility UVH6/AtXPD gene             expressed plant tissues supports possi- <br /> required transcription plants.  hy-          bility gene important general <br /> pothesize transcription abnormal uvh6-1       transcription. <br /> mutant, causing defects expression genes              observed apparent reduction quired chlorophyll synthesis, normal growth rate,        transmission T-DNA insertion prog- <br /> Plant Physiol. Vol. 132, 2003                                                                                   1411 <br />  Liu et al. <br />  <br />  <br /> eny plants carrying heterozygous insertion.                                Genetic Physical Mapping UVH6 Gene progeny heterozygous                                  Recombinant plants generated crossing uvh6-1/uvh6-1 homozy- <br /> sertion lost insertion wild type.                           gous mutant plant (Columbia ecotype) wild-type Landsberg erecta  observed ratio heterozygotes:wild                               plant. F2 progeny screened identify exhibited UV <br /> type close 1:1, expected 2:1.                            sensitivity mutant (Columbia) parent, using protecting foam shield <br />                                                                                 meristem treatment, described (Harlow et al., 1994). UV- finding suggests transmission                               sensitive progeny examined presence Landsberg- <br /> serted allele impaired female                         associated simple sequence length polymorphism physical markers nga59, <br /> male gametes  . currently testing                                AtEAT1, nga63. Polymorphisms detected agarose gel analysis possibilities backcrossing heterozygous                                PCR assays (Bell Ecker, 1994). <br /> plants wild type. <br />   model explain differences observed                                Mutant Complementation <br /> tween uvh6-1 point mutant uvh6-2 <br /> T-DNA insertion mutant suggested stud-                                  5.6-kb fragment AtXPD genomic DNA, carries UVH6/ <br />                                                                                 AtXPD coding region, 3&#226;&#172;&#732;-untranslated sequences, 5&#226;&#172;&#732;-flanking region (1.2 <br /> ies yeast RAD3 mutations. mutations exist                                                                                 kb) containing AtXPD promoter, amplified PCR wild-type classes. Mutations class lethal                        (C10) DNA, using primer pair H6U186 (5&#226;&#172;&#732;-CAACATTCCGATTTTCCGT- <br /> result severe defects RNA polymerase II-                                  CACCT) XDL2532 (5&#226;&#172;&#732;-CCTACAGTGAAAATTTGAGCTCCAACAATT). <br /> dependent transcription restrictive conditions                            amplified fragment cloned pCRII vector, using TA <br />                                                                                 Cloning Kit (Invitrogen, Carlsbad CA), transferred plasmid <br /> (Guzder et al., 1994; Prakash Prakash, 2000), <br />                                                                                 pBIN19. resultant recombinant (pAtXPD) introduced Agrobac- <br /> suggesting transcription defect causes                             terium strain GV3101 electroporation, described Mozo Hooykaas <br /> lethality. contrast, second class mutations                           (1991). Transformed bacteria selected Luria-Bertani medium (Sam- <br /> does impair viability, mutants                          brook et al., 1989) containing 30 mg L&#226;&#171;&#186;1 gentamycin 60 mg L&#226;&#171;&#186;1 kana- <br />                                                                                 mycin. DNA selected bacteria transferred uvh6-1 mutant <br /> sensitive UV irradiation exhibit defects                                                                                 plants using floral dip method (Clough Bent, 1998), transformed <br /> NER (Prakash Prakash, 2000).  second                              kanamycin-resistant plants selected generation seed class appears affect DNA repair, leaving                               media containing half-strength Murashige Skoog basal salt mix <br /> transcription largely intact. Presumably, XPD muta-                             (Gibco/Life Technologies, Grand Island, NY), 1&#226;&#171;&#187; B5 vitamins (Bamborg et al., <br /> tions produce diseases humans                               1968), 1% (w/v) Suc, 0.8% (w/v) Bacto-agar, 60 mg L&#226;&#171;&#186;1 kanamycin. <br />                                                                                 Primary transformants tested, described Liu et al. (2001b) second type cause repair defects subtle                           identify UV-resistant, complemented transgenic lines. UV-resistant phe- <br /> impacts transcription. accord model,                            notypes confirmed T2 generation plants. uvh6-1 point mutation resemble second class, causing repair defect minor ef- <br /> fects transcription, uvh6-2 T-DNA                                DNA Sequencing <br /> insertions resemble class, producing                               sequence AtXPD/UVH6 cDNA, partial cDNA fragments lethal phenotype severe transcription defect.                          amplified C10 wild-type uvh6-1 mutant plants RT-PCR,                                                                                 described  fragment sequenced. length cDNA <br />                                                                                 sequence assembled, uvh6-1 mutation identified com- <br />                                                                                 parison wild-type mutant sequences. primers used amplifi- <br /> MATERIALS METHODS                                                           cation  XDU169, 5&#226;&#172;&#732;-GCCAATTTCGAGATCTAGGTAGGAGGAA; <br />                                                                                 XDU740, 5&#226;&#172;&#732;-TAAGGGCGTTTGGTAAGAATC; H6U1375, 5&#226;&#172;&#732;-TACGCCATC- <br /> Strains Growth Conditions                                                   TCGGCACCCAGGCT; H6L2699, 5&#226;&#172;&#732;-AAGTACTGAAGTAATCGGCGT; <br />                                                                                 XDL1740, 5&#226;&#172;&#732;-ACTTCTCATGTCAAATTTGGTGCT; XDL2382, 5&#226;&#172;&#732;-CCCATTG- <br />    wild-type Columbia (C10) used derived single plant            TACCCGCCTTATCAT; H6U1534, 5&#226;&#172;&#732;-AACCGGTGTTTGATCGTTTCCAGT; <br /> isolate Arabidopsis served parent uvh6-1 mutant,     H6U4835, 5&#226;&#172;&#732;-CTGGTTGGATACTTTCGCATCTGC; H6L5357, 5&#226;&#172;&#732;- isolated described (Harlow et al., 1994; Jenkins et al., 1995). Plants   AGGAGCCGACGAAGTATTTCTTGA. sequence T-DNA insertion normally grown room temperature (22&#194;&#176;C&#226;&#8364;&#8220;24&#194;&#176;C) continuous            UVH6, DNA isolated plant carrying insertion, sub- <br /> lighting consisting combination 40WT12 Excella F40 agro bulbs       jected PCR amplification using primers UVH6-R LB3 amplify (Philips Lighting Company, Somerset, NJ) positioned approximately 35 cm         region surrounding insertion site  Fig. 4) sequenced using plants (30 &#226;?&#174;mol photons m&#226;&#171;&#186;2 s&#226;&#171;&#186;1).                                      LB3 primer (5&#226;&#172;&#732;-TAGCATCTGAATTTCATAACCAATCTCGATACAC). <br />    Seeds carrying T-DNA insertion (uvh6-2) UVH6 gene (garlic line <br /> 825 BO5) obtained Torrey Mesa Research Institute (San <br /> Diego). Seeds surface sterilized germinated agar plates      Repair Assays <br /> (Haughn Somerville, 1986) germinated soil grown growth chamber 22&#194;&#176;C 16 h light (240 &#226;?&#174;mol photons m&#226;&#171;&#186;2 s&#226;&#171;&#186;1)            Assays performed described Liu et al. (2000). brief, 22&#194;&#176;C 8 h dark 21&#194;&#176;C 75% 90% humidity.                           procedure involved irradiating seedlings sprouted agar <br />                                                                                 plates, isolating seedling DNA 0 24 h irradiation, quanti- <br />                                                                                 fying 6-4 photoproducts present DNA lesion-specific radio- <br />                                                                                 immunoassay (Mitchell, 1996). <br /> UV Light Treatment <br />     3-week-old plants irradiated UV-C light (254 nm)        PCR Analysis UVH6 Alleles <br /> described Liu et al. (2001b). incubated 3 d F40 gold fluorescent lights (Philips Lighting Company, Somerset, NJ)             Genomic DNA isolated leaves seedlings grown agar avoid reversal UV photoproducts photoreactivation, mechanism            leaves older plants grown soil. prepare seedlings, seeds activated visible light, transferred standard growth condi-      surface sterilized, plated agar medium (containing 2.5 mm potassium <br /> tions 7 d. Sensitivity assessed extent leaf yellowing     phosphate [pH 5.5], 5 mm KNO3, 2 mm MgSO4, 2 mm Ca[NO3]2, 49 &#226;?&#174;m <br /> tissue death.                                                                   ethylenediamine-tetraacetic acid micronutrients, 5 g L&#226;&#171;&#186;1 Suc), incu- <br />  <br /> 1412                                                                                                                     Plant Physiol. Vol. 132, 2003 <br />                                                                                              Arabidopsis Homolog Human XPD Yeast RAD3 <br />  <br />  <br /> bated 4&#194;&#176;C, germinated growth chamber. DNA isolated          Britt AB, Chen JJ, Wykoff D, Mitchell D (1993) UV-sensitive mutant seedlings described Klimyuk et al. (1993) older plants          Arabidopsis defective repair pyrimidine-pyrimidinone(6-4) described Bell Ecker (1994). PCR reactions conducted using           dimers. Science 261: 1571&#226;&#8364;&#8220;1574 <br /> 42 cycles (30 s melting 94&#194;&#176;C, 1 min annealing 65&#194;&#176;C, 2-min        Clough SJ, Bent AF (1998) Floral dip: simplified method extension 72&#194;&#176;C) experiments Ga-3 progeny 35 cycles (1 min           Agrobacterium-mediated transformation Arabidopsis thaliana. Plant J 16: melting 95&#194;&#176;C, 1 min annealing 57&#194;&#176;C, 2.5-min extension            735&#226;&#8364;&#8220;743 <br /> 72&#194;&#176;C) experiments Ga-2 progeny. Primers used  UVH6-F,              Costa RMA, Lima WC, Vogel CIG, Berra CM, Luche DD, Medina-Silva R, <br /> 5&#226;&#172;&#732;-ATCGTCACTGAATTCTCAGGC; UVH-6-R, 5&#226;&#172;&#732;-CATGACGGCTGTATC-                              Galhardo RS, Menck CFM, Oliveira VR (2001a) DNA repair-related <br /> TGCAAG; LB3,  LB3-D, 5&#226;&#172;&#732;-ATTTCATAACCAATCTCGA-                          genes sugarcane expressed sequence tags (ESTs). Genet Mol Biol 24: <br /> TACAC. Reaction products detected 1.5% (w/v) agarose gels.                  131&#226;&#8364;&#8220;140 <br />                                                                                  Costa RMA, Morgante PG, Berra CM, Nakabashi M, Bruneau D, Bouchez <br />                                                                                     D, Sweder KS, Van Sluys MA, Menck CFM (2001b) participation RT-PCR Analysis Gene Expression                                                  AtXPB1, XPB/RAD25 homologue gene Arabidopsis thaliana,                                                                                     DNA repair plant development. Plant J 28: 385&#226;&#8364;&#8220;395 <br />    Total RNA isolated C10 wild-type plant tissues described       Boer J, Wit J, van Steeg H, Berg RJ, Morreau H, Visser P, Lehmann <br /> Liu et al. (2001b). RT-PCR conducted using OneStep RT-PCR Kit               AR, Duran M, Hoeijmakers JH, Weeda G (1998) mouse model (Qiagen , Valencia, CA) 50 (UBQ10) 200 (UVH6) ng              basal transcription/DNA repair syndrome trichothiodystrophy. Mol Cell <br /> RNA. 26 (UBQ10) 40 (UVH6) PCR cycles used amplifi-               1: 981&#226;&#8364;&#8220;990 <br /> cation. microliters 25-&#226;?&#174;L reaction subjected electro-       Boer J, Hoeijmakers JH (2000) Nucleotide excision repair human <br /> phoresis 1% 3% (w/v) agarose, identity RT-PCR                 syndromes. Carcinogenics 21: 453&#226;&#8364;&#8220;460 <br /> product confirmed sequencing. Primers UBQ10U60 (5&#226;&#172;&#732;-GA-                    Fidantsef AL, Mitchell DL, Britt AB (2000) Arabidopsis UVH1 gene CTCTCACCGGAAAGAAAT) UBQ10L573 (5&#226;&#172;&#732;-TTGTCTTGGATCTT-                               homolog yeast repair endonuclease RAD1. Plant Physiol 124: <br /> GGCTTTCA) used amplify UBQ10 gene. Primers XDU740                       579&#226;&#8364;&#8220;586 <br /> (5&#226;&#172;&#732;-TAAGGGCGTTTGGTAAGAATC) XDL2382 (5&#226;&#172;&#732;-CCCATTGTACC-                          Gallego F, Fleck O, Li  Wyrzykowska J, Tinland B (2000) AtRAD1, CGCCTTATCAT) used amplify UVH6 gene.                                    plant homologue human yeast nucleotide excision repair endo- <br />                                                                                     nucleases, involved dark repair UV damages recombination. <br />                                                                                     Plant J 21: 507&#226;&#8364;&#8220;518 <br /> Bioinformatics                                                                   Gorbalenya AE, Koonin EV, Donchenko AP, Blinov VM (1989)                                                                                     related superfamilies putative helicases involved replication, recom- <br />    Multiple sequence alignments obtained using ClustalW pro-               bination, repair expression DNA RNA genomes. Nucleic <br /> gram default parameters (Higgins et al., 1996). BLAST server               Acids Res 26: 4713&#226;&#8364;&#8220;4730 <br /> (http://www.ncbi.nlm.nih.gov/) used database similarity searches         Guzder SN, Qiu H, Sommers CH, Sung P, Prakash L, Prakash S (1994) <br /> (Altschul et al., 1997). Pair-wise alignments alignment AtXPD            DNA repair gene RAD3 S. cerevisiae essential transcription cDNA genomic sequences obtained using LALIGN (Pearson                  RNA polymerase II. Nature 367: 91&#226;&#8364;&#8220;94 <br /> Miller, 1992), minor adjustments make exon/intron junctions          Harlow GR, Jenkins  Pittalwala TS, Mount DW (1994) Isolation consistent consensus sequences Arabidopsis splice junctions.           uvh1, Arabidopsis mutant hypersensitive ultraviolet light                                                                                     ionizing radiation. Plant Cell 6: 227&#226;&#8364;&#8220;235 <br />                                                                                  Haughn GW, Somerville C (1986) Sulfonylurea-resistant mutants Arabi- <br />                                                                                     dopsis thaliana. Mol Gen Genet 204: 430&#226;&#8364;&#8220;434 <br /> Distribution Materials <br />                                                                                  Higgins DG, Thompson JD, Gibson TJ (1996) Using CLUSTAL multiple <br />    request, novel materials described publication          sequence alignments. Methods Enzymol 266: 383&#226;&#8364;&#8220;402 available timely manner noncommercial research purposes,           Jenkins  Harlow GR, Liu  Shotwell MA, Ma J, Mount DW (1995) <br /> subject requisite permission party owners           Radiation-sensitive mutants Arabidopsis thaliana. Genetics 140: 725&#226;&#8364;&#8220;732 <br /> parts material. Obtaining permissions responsibility   Jenkins  Suzuki TC, Mount DW (1997) Evidence heat ultra- requestor.                                                                      violet radiation activate common stress-response program plants                                                                                     altered uvh6 mutant Arabidopsis thaliana. Plant Physiol 115: <br />                                                                                     1351&#226;&#8364;&#8220;1358 <br />                                                                                  Keriel  Stary  Sarasin  Rochette-Egly C, Egly JM (2002) XPD muta- <br /> ACKNOWLEDGMENT                                                                      tions prevent TFIIH-dependent transactivation nuclear receptors    authors gratefully acknowledge Torrey Mesa Research Institute            phosphorylation RARalpha. Cell 109: 125&#226;&#8364;&#8220;135 <br /> (San Diego) 825 B05 seeds carrying UVH6 T-DNA insertion.             Klimyuk VI, Carroll BJ, Thomas CM, Jones JD (1993) Alkali treatment                                                                                     rapid preparation plant material reliable PCR analysis. Plant J 3: <br /> Received February 8, 2003; returned revision March 30, 2003; accepted           493&#226;&#8364;&#8220;494 <br /> April 17, 2003.                                                                  Lehmann AR (2001) xeroderma pigmentosum group D (XPD) gene:                                                                                     gene, functions, diseases. Genes Dev 15: 15&#226;&#8364;&#8220;23 <br />                                                                                  Li  Schuermann D, Gallego F, Kovalchuk  Tinland B (2002) Repair LITERATURE CITED                                                                    damaged DNA Arabidopsis cell extract. Plant Cell 14: 263&#226;&#8364;&#8220;273 <br />                                                                                  Liu J, Akoulitchev S, Weber  Ge H, Chuikov S, Libutti D, Wang XW, <br /> Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang  Miller W,                    Conaway JW, Harris CC, Conaway RC et al. (2001a) Defective interplay <br />   Lipman DJ (1997) Gapped BLAST PSI-BLAST: new generation                  activators repressors TFIIH xeroderma pigmentosum. Cell <br />   protein database search programs. Nucleic Acids Res 25: 3389&#226;&#8364;&#8220;3402                 104: 353&#226;&#8364;&#8220;363 <br /> Arabidopsis Genome Initiative (2000) Analysis genome sequence          Liu  Hall JD, Mount DW (2001b) Arabidopsis UVH3 gene homolog   flowering plant Arabidopsis thaliana. Nature 408: 796&#226;&#8364;&#8220;815                     Saccharomyces cerevisiae RAD2 human XPG DNA repair genes. <br /> Balajee  Bohr VA (2000) Genomic heterogeneity nucleotide excision             Plant J 26: 329&#226;&#8364;&#8220;338 <br />   repair. Gene 250: 15&#226;&#8364;&#8220;30                                                        Liu  Hossain GS, Islas-Osuna MA, Mitchell DL, Mount DW (2000) <br /> Bamborg OL, Miller RA, Ojima K (1968) Nutrient requirements suspen-              Repair UV damage plants nucleotide excision repair: Arabidopsis <br />   sion cultures soybean root cells. Exp Cell Res 50: 151&#226;&#8364;&#8220;158                     UVH1 DNA repair gene homolog Saccharomyces cerevisiae Rad1. <br /> Batty DP, Wood RD (2000) Damage recognition nucleotide excision                  Plant J 21: 519&#226;&#8364;&#8220;528 <br />   repair DNA. Gene 241: 193&#226;&#8364;&#8220;204                                               McGregor (1999) DNA repair, DNA replication, UV mutagenesis. <br /> Bell CJ, Ecker JR (1994) Assignment 30 microsatellite loci linkage        J Invest Dermatol Symp Proc 4: 1&#226;&#8364;&#8220;5 <br />   map Arabidopsis. Genomics 19: 137&#226;&#8364;&#8220;144                                       Mitchell DL (1996) Radioimmunoassay DNA damaged ultraviolet <br /> Bootsma D, Hoeijmakers JH (1993) DNA repair: engagement tran-                  light. GP Pfeifer, ed, Technologies Detection DNA Damage <br />   scription. Nature 363: 114&#226;&#8364;&#8220;115                                                    Mutations. Plenum, New York, pp 73&#226;&#8364;&#8220;83 <br />  <br /> Plant Physiol. Vol. 132, 2003                                                                                                                            1413 <br />   <br /> </body></html>