<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1991 Genetics Society America <br />  <br />  <br />  <br />           Genetic Analysis ofa Major Segment [LGV(left) Genome                                Caenorhabditis elegans <br />  <br />                                              Robert C. Johnsen&apos; David L. Baillie <br />               Institute Molecular Biology Biochemistry, Departmentof Biological Sciences, Simon Fraser University, <br />                                               Burnaby, British Columbia, Canada V5A S 6 <br />                                                        Manuscript received April19, 1991 <br />                                                       Accepted publication July 27, 1991 <br />  <br />                                                                    ABSTRACT <br />                  10,900 F1progenyofethylmethanesulfonate(EMS)-mutagenized                Caenorhabditiselegans <br />               nematodes, isolated 194 lethal mutations left arm LGV, regionbalanced               reciprocal translocationof eT1, analysis 166 mutations resulted identification <br />               deficiency alleles 78 genes including 38 new genes, increasing     number               identified essential genesto 101. estimate thatthere minimum 120 essential genes                  region, comprises approximately 7% recombinational distance,               4.2% genes, C. elegans. calculate minimum 2850 essential genes               thegenome. leftarm LGV recombinational gene clusters separated high- <br />               recombination essential gene-sparse region. gene region, let-330, largest <br />               EMS target left arm LGV, twice alleles (16) EMS-mutable <br />               genes, let-332 rol-3. gene sparse region, lin-40, region near lin-40               major targets T c l mohilization-induced mutagenesis. analysis essentialgenesinlarge <br />               regions help defineC. elegans terms genes aidin understanding               relationship genome structure genome function. <br />  <br />                                                                            mutations. dividing resulting mutations W      E attempted genetically identify         localize majority essential genes large regionof Caenorhabditis elegansgenome:                                                                            complementationgroups <br />                                                                            phenotypes, andthencombining <br />                                                                                                         characterizingtheir <br />                                                                                                                  thisinformation <br /> 23 map units(m...) arebalanced reciprocal                      sequence information, functions large <br /> translocation eTI(ZZZ;V) (ROSENBLUTH      BAILLIE                      classof genes deduced.      Regions 1981) left arm linkage group V [LGV(left)].                      LGV(left) beingstudiedinclude:        LGI(1eft) <br /> Recombinationally, largest region                         (ROSEand BAILLIE1980; HOWELLand ROSE 1990; <br /> higher eukaryote forwhich attempthas                          MCKIM ROSE 1990);LGII(right) (SICURDSON,  regionrepresentsabout          7% C.                      SPANIERand HERMAN 1984); unc-22(N) region <br /> elegans genetic map.                                                       (ROGALSKI,    MOERMANand BAILLIE1982; ROGALSKI <br />    motives work. T h e                     BAILLIE1985; CLARK 1990);       duplication <br /> based ultimate goal investigators                        mnDpl balanced region LGX (MENEELYand  <br /> define C. elegans terms genes. genomic                     MAN 1979, 1981). <br /> cosmid-Yac contig map C. elegans nearing com-                           second motive study pletion (COULSON   et al. 1986, 1988 thesystematic                     relationship genome structure genome    func- <br /> sequencing entire C. elegans genome begun                       tion. regard, genetic analysis genes   COULSON,   J. E. SULSTON   R. H. WATERSTON,                        alargechromosomalregion         address personal communication). sequencing data                         problems. Genes distributed evenly allow identification genes,                      chromosomes C. elegans (BRENNER 1974) (forthe yield muchinformation abouttheir biological                            C. elegans genetic map EDCLEY   RIDDLE1990). <br /> functions. existence genetic mutations                        Identification essential genes supply functional information. Essential genes consti-                     precisely delineate gene clusters fromthe gene sparse <br /> tute thelargest class genesfor mutant alleles                     regions. Furthermore, results afew smallregions <br /> causing discontinuous phenotypes detected                        coding elements identified <br /> clear-cut Mendelian ratios genetic crosses.                      known length, physically way obtain mutations large numberof genes                                                                            recombinationally, suggest thatgene spacing screen genome, region region, lethal <br />                                                                            chromosome recombination distances    &apos; Current address: Division Biological Sciences, 310Tucker   Hall,   strong correlation (GREENWALD et al. 1987; <br /> University Missouri, Columbia, Missouri 6521 1 .                        PRASAD   BAILLIE1989; STARRet al. 1989). Genetics 129: 735-752 (November, 1991) <br />  736                                 C.       R. <br />                                              Baillie <br />                                                   Johnsen <br />                                                       L. D. <br />  <br />   recombination unit length DNA          cultured Petri dishes simple agar nematode <br /> stant. Genetic mapping    genes given region     growth medium streaked Escherichia coli (OP50).  aid aligning physical contig mapwith        details procedures observing han- <br />                                                               dling worms BRENNER        (1974). <br /> genetic map identify regions relative low            Mutations: wild-type C. eleguns (var. Bristol) N2 high rates recombination analysis.         strain strains carrying following mutations Finally, vague idea location         obtained MRC, Cambridge, England, structural features governing chromosome behavior,            Caenorhabditis Genetics Center University Mis- <br />                                                               souri, Columbia, Missouri, cited: dpy-ll(e224)V, dpy- meiotic pairing. Mapped genes serve                                                                18(e364)III, emb-29(g52)V, lin-40(e2173)V [isolated S. W. <br /> landmarks localization.                             EMMONS,     supplied J. HODGKIN      (MRC Cambridge)], <br />     Recombination LGV(left) efficiently balanced        rol-3(e754)V,unc-23(e25)V,unc-34(e566)V,unc-42(e270)V, eTl(ZZZ;V), balances right arm           unc-46(el77)V, unc-60(e677 m35)V, unc-62(e644)V, unc- <br /> linkage group I11 [LGIII(right)] (ROSENBLUTHand               68(e540)V, unc-70(e524)V,unc-76(e91 l)V, eTl(IIZ;V).  <br />                                                               ficiencies mDf1 mDf3 (BROWN1984) D. L. <br /> BAILLIE   1981). region subdivided          RIDDLE&#226;&#8364;&#8482;S   laboratory (Columbia, Missouri). nDf32 (PARK        zones sets overlapping rearrangements            HORVITZ     1986b), nDfl8 nDf31 originated inR. H. <br /> deficiencies) (JOHNSEN BAILLIE1988; ROSEN-                HORVITZ&#226;&#8364;&#8482;S    laboratory (MIT). mutations denoted BLUTH et d . 1988; CLARK et d . 1990; STEWART,                s prefix arose laboratory. <br /> ROSENBLUTH      BAILLIE1991). Prior paper,           Characteristics eTl(ZZZ;V) eTl(III;V) reciprocal <br />                                                               translocation recombinationally balances right half <br />  17 putative nonessential genes  laboratory         LGIII theleft half LGV total 43 m.u. <br /> ditions) 63 essential genes identifiedon         (14% genome) balanced (ROSENBLUTH           BAILLIE <br /> LGV(1eft) (JOHNSEN BAILLIE1988; ROSENBLUTH                1981). balanced regions chromosome ap- <br /> et al. 1988; CLARKet al. 1990; EDCLEYand RIDDLE               proximately size recombinationally. LGV,  <br />  1990; STEWART,     ROSENBLUTHand BAILLIE1991).               combination eT1 heterozygotes appears completely <br />                                                               suppressed left end region dpy-11 <br />     Mutations genes       isolated     (zone 13)  Figure 1 Figure 2 locations zones) <br /> variety screening procedures. T h e largest class       unc-42 (zone 21) near center chromosome. <br /> mutations recoveredby screening overthe            sixteenths progeny +/eTl(ZZZ);+/eTl(V)  <br /> entire eTI balanced region treatment ethyl         maphrodites stop maturing blocked early devel- <br /> methanesulfonate (EMS). intent study          opment, considered result animals <br />                                                               having aneuploid genomes. aneuploidy increase number lethal mutations using EMS          crossing markers eT1 breakpoints, <br /> mutagenesis majority  thirds)        markers LGIII(right) LGV(left) pseudolinked. essential genes LGV(1eft) identified.      breakpoint eT1 LGIII mapped close Based calculations ROSENBLUTH      et al. (1 988)    unc-36. <br /> estimated screening 10,000 mutagenized chro-                BC2200, thestrain used EMS mutagenesis:  <br />                                                               maphrodites homozygous dpy-18;unc-46 strain mosomes identify thirds essen-          mated dpy-l8/eTl;unc-46/eTl males. Wild-type maleoff- <br /> tial genes.                                                   spring mated homozygous eT1 hermaphrodites.     screened10,900 eTI balanced chromo-               individual wild-type hermaphrodite progeny somes treated withEMS isolated 751 recessive              cross used establish strain BC2200. lethal [including sterile adult me1 (maternal        genotype dpy-18(e364)/eTI(III);unc-46(el77)/eTl(V)                                                               early death aneuploids fact effect lethal)] mutations close balanced         dpy-18 unc-46 balanced regions, hermaphrod- <br /> region. compared numberof lethal mutations             ites withthis genotype rise progeny phenotypic mapped eTI balanced chromosomes.              ratio Dpy Unc wild-type Unc-36. used complementation tests tomapthe               lethal      <span id='am-8' about='obo:IAO_0000115' typeof='owl:Thing'>Definition</span> lethal, sterile mel (maternal effect <br /> mutations LGV(1eft) set deficiencies          lethal) mutations: recessive lethal mutation                                                               whenhomozygouscauses           worm tomature (egg divide LGV(1eft) zones. Oncethe lethal              lethal, hatches, early-, mid- late-larval lethal); mature <br /> mutations mapped zones, used comple-               butnotproduce        fertilized eggs (sterile); orto produce <br /> mentation tests discern lethal mutations iden-      fertilized eggs mature (me1 mutations). Second <br /> tified new genes alleles known genes.           generation mels considered lethal mutations. <br />                                                                  EMS mutagenesis: Mutagenesis BC2200 ac- <br /> noted approximate developmentalblocking stages <br />                                                               cording BRENNER(1974) dose  lethals 20 O tested    creased 0.012 M EMS basis dose-response <br /> mutations temperature sensitive ( t s )         curve ROSENBLUTH,         CUDDEFORD     BAILLIE(1983). <br /> missive temperatures 15&#226;&#8364;? 25&#226;&#8364;?. tested            Mutagenesis room temperature (approximately sterile adults mels putative sperm pro-        21-23&#226;&#8364;&#8482;) 4 hr. <br /> duction sperm defects mating wild-               Screening mutations: mutagenesis, egg bearing <br />                                                               young adult Pas allowedto recover hours type males.                                                   wild-type hermaphrodites individually plated                                                               left lay eggs 21 hr ( ?brood) room temperature, <br />             MATERIALS METHODS                             45 hr 15 fresh plates (&#226;&#8364;&#339;B&#226;&#8364;? Brood),                                                                                 O <br />  <br />  <br />   General: nomenclature follows uniform        Pes discarded. Wild-type Fls picked individ- <br /> adopted forC . elegans (HORVITZ <br />                              et al. 1979). nematodes      ually  Poused        Fls picked). Fl&#226;&#8364;&#8482;s <br />                737       LGV(1eft)          Genetic Analysis <br /> progenywerescreened absence healthyegg-                blockers, 0.3-0.5 mm mid blockers, 0.5-0.6mm late block- <br /> bearing Dpy Uncs (lethal, sterile me1 mutations          ers greater 0.6 mm internal fertilized eggs picked . absence fertile Dpy Unc F ~ indi- s       werecalled sterile adults. Worms internal fertilized <br /> cated presence lethal mutation eTI balanced       eggs developmentally blocked progeny considered <br /> regions LGIII LGV  outside close              behomozygous me1 mutations.Worms balanced region).                                                 fertile, developed slowly, noted called &quot;slow.&quot; <br />    Mapping mutations LGIII           LGV Replacing         Test temperature sensitive mutations: Worms balancer ( e T ) wild-type chromosomes allows       15&quot; 25&quot; generation map lethal chromosome calculate distance      tested. dpyr-18/eTl(III);let unc-46/eTI(V)hermaphro- appropriate marker [dpy-I8(III unc-46(V)].        dites brooded 15&quot; 25&quot; andtheir progeny <br /> lethal bearing strains crossed N2 males       scored presence ofDpy Uncs.IfDpyUncswere <br /> wild-type L4 F1 hermaphrodites picked. Fls          present, brooded individually scored <br /> gaveUnc-36progenywere            discarded. progeny     fertility. <br /> lethal bearing dpy-18/+;unc-46/+ approximate              Test cross-fertilization wild-type males: Sterile adult wild-type Dpy-18 ratio lethal mutation         me1 mutations tested cross-fertilization LGV(1eft) strain retained.       wild-type male sperm attempting mate Dpy Unc <br /> strain discarded ( e . , strains mutations     hermaphrodites homozygous lethal mutation, LGIII LGIII LGV discarded).                   N2 males. presence wild-type maleand fertile <br />     Mapping location LGV(1eft): LGV(1eft)          wild-type hermaphrodite progeny indicated defective <br /> divided zones set rearrangement breakpoints,         production sperm defective sperm hermaphro- deficiencies (ROSENBLUTH       et al. 1988). subset   dite. deficiencies (sDf26, sDf30, sDf33, sDf34 sDf35) <br /> uncover allof region exceptfor zone betweensDf30 sDf35  Figure 1). deficiencies                                 RESULTS <br /> balanced eTI strains carry dpy- <br />  18, strain containing sDf35 carry unc-46.           screened10,900 EMS-treated chromosomes Males containing lethal mutations LGV(1eft) [dpy-18/           presence lethal mutations LGIII(right) eTl(III);let unc-46/eTl(V)] crossed set      LGV(1eft). isolated 751 mutations: 242 strains <br /> deficiencies. absence fertile Dpy Unc (Dpy-18 sDf-35) progeny indicated failure complement. Appro-           contained mutations LGV,      45 1only LGIII <br /> priate deficiencies used, necessary         29 strains carried mutations LGV  Figures 1 2), identify zone       LGIII. 64% mutations mapped LGIII <br /> mutation falls. lethal mapped zone,       36% mapped LGV. data support complementation tested representative alleles      previous findings regarding distribution known genes zone (ges (zone 4A), mec (zone <br />  15) unc-83 (zone 12) complementation tested         LGIII LGV     EMS-induced lethals balanced lethals) necessary alleles neighbouring      eT1 (JOHNSEN, ROSENBLUTH       BAILLIE   1986). zones; e.g.,  mutation inzone 15, zone 16            regions recombinationally approximately <br /> mutations tested; b) mutation zone        equal insize, essential gene density LGIII(right) <br />  17 gamma-irradiation-induced mutations (putative          twice LGV(left). strains ones <br /> ficiencies) zone 18 tested. results com- <br /> plementation tests allowed assignment muta-        carrying mutations LGV discarded. <br /> tions appropriate genes.                                          screen, expected recover lethal muta- <br />     Developmental blocking stage:      developmental block-   tions close eT1 balanced <br /> ing stages noted 20&quot;. Prior analysis, worms        regions. analyzing 242 strains containing set 20&quot; generation eliminate      mutations onLGV(left), 50 (27%)mutations werelost <br /> possible temperature dependent maternal effects. Worms <br /> containing relatively early developmental blocking recessive      segregation outside         bal- <br /> lethal mutations werehandled follows: dpy-I8/eTl(III);let      anced region. strains proved unc-46/eTI(V) hermaphrodites        mated      dpy-18/    lethal mutations theeT1 balanced region <br /> eTl(III);unc-46/eTl(V) males. Dpy Unc Fls            LGV(1eft).  total 194 mutations set plates hours number eggs <br /> ( F ~ s )laid counted. day F2s scored      eT1 balanced region LGV uncovered. presence unhatched eggs (putative egg lethal           gave 2% forward lethal mutation rate 0.012   mutations). day, F2s counted        M EMS inthe eT1 balancedregion             LGV maturing nonlethal F ~removed, <br />                            s         thefourth seventh     calculation includes 29 mutations dis- <br /> days homozygouslethal bearing DpyUncs wereob-                 carded strains carried <br /> served measured ocular micrometer. Worms <br /> containing late-larval me1 mutationswere handled            lethals LGIII(right)]. <br /> follows: approximately 20 dpy-I8/eTl(III);let unc-46/eTI(V)          T h e eT1 breakpoints LGIII LGV hermaphrodites set plates 6 hr. Dpy Unc           gene cluster near center respective <br /> progeny weretransferred new plates observation         chromosomes.Assumingthatthesamenumberof <br /> measurement fourth seventh days.                       mutations (50) outside eT1 bal- <br />    homozygous lethalcontaining Dpy Unc worms classified according lengths. Dpy Unc-46s, carrying      anced region LGIII       lost LGV, lethal, reach mature length 0.8-0.9 mm. Worms             calculated forward mutation rate EMS 5.8% <br /> shorter 0.2 mm considered developed         region balancedby eT1. agreement hatching. Worms0.2 0.3 mm called early            ROSENBLUTH, CUDDEFORD         n d BAILLIE(1983), <br />  738                                                        R. C. Johnsen D. L. Baillie <br />  screening 1662 chromosomes yielded                              genes. Mutations defining 12 genes came <br /> forward mutation rate 6.6%.                                                 screens selected regionsof LGV(1eft) <br />    10,900 chromosomes screened pres-                            (ROSENBLUTHet al. 1988) screens using mu- <br /> ence lethal mutation, 4750 (43.6%)came                             tagens cause rearrangements [y-ray irradia- brood 6150 (56.4%) came B brood.                                tion (ROSENBLUTH,                CUDDEFORD            BAILLIE1983) 194 mutations recovered LGV(left), 84                                <span id='am-10' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> (JOHNSEN                       BAILLIE1988)l (43.3%) came brood 110 (56.7%)                                  mutagen preferred sitesofaction <br /> came B brood. signif-                           [mobilizationof transposon Tcl (CLARKet al. <br /> icant difference mutation rates early                         1990)]. <br /> late broods. Furthermore, gene                                     Table 1 , developmental blockingstages allele originating given Po e.,                           (based measured lengths worms) evidence premeiotic events.                                         various allelesare listed. changed canon- <br />    mapped 166 194 lethal mutations iso-                              ical allele assignments essential genes <br /> lated. 28 difficult work ( e . ,                      previously identified laboratory. canonical <br /> male strains established maintained)                           alleles            changed reflect earlier blocking   mapped. analysis                           recessive lethal phenotype EMS-induced allele <br />  166 recessive lethal mutations yielded deficiency                         gene previously identified 7-ray irradi- <br /> (sDf56which deletes zones 1-4B inclusive, Figure                           ation-induced mutation. developmental blocking <br /> l), alleles 78 genes including 38 newly identi-                         stages ofthe canonical allelesrange egg lethality <br /> fied essential genes lethal allele unc-                       slow development. number genes identified <br /> 60 (zone 4A). 38 genes, previ-                             category shownin Figure 3. ously detected genes, provide total 101 genes                             screened recessive lethal mutations, recessive lethal mutations                             me1 mutations, me1 class beunderrepresented. <br /> mapped LGV(1eft). 101 essential genes                                   Previously ROSENBLUTHet al. ( 1 988) screened EMS <br /> plus 17 putative nonessentialgenes total 118 genes                             mutagenized eT1 balancedchromosomes,  <br /> mapped zones LGV(1eft) number genes                               tained Fls Dpy Uncsweretested                              zone average 3.3, range 12.                                fertility, selecting me1 mutations.  majority genes LGV(1eft)                             screen, me1 mutations wereisolatedbecausethey right/left positioned respect                          caused phenotypic effects, slow growth deficiency mapping.                                                         thinness eggs, subsequently    genes mapped zone                              shown mel. similar fraction me1 <br /> LGV(1eft listed Table 1. number EMS-                             mutations isolated screen (6 10,900 <br /> induced alleles screens entire eT1 bal-                          mutagenized chromosomes) isolated RO- <br /> anced region total number alleles                          SENBLUTH et al. (1988) (2 2738 mutagenized <br /> gene shown columns3 4, respectively.                            chromosomes), concluded mostof me1 <br /> canonical allele mutation listed                         mutations generated isolated. iso- <br /> gene (column 5). ofthe results ROSEN-                            lation mel mutations implies large <br /> BLUTH et al. (1988).                                                           fraction me1 mutations havephenotypiceffects <br />    Figure 1 Figure 2 genetic maps                                   inviable progeny. <br /> LGV(1eft). genes shown line rep-                                  screened mutations LGV(1eft) tem- <br /> resenting chromosome; rearrangements                               perature sensitivity testing permissive <br /> [deficiencies duplication (sDp30)]                               temperatures 15 O 25 O . Worms homozy- <br /> breakpoints divide region zones                             gous mutations [ l e t - 4 2 2 ( ~ 1 5 7 8(zone         )    11B5), line. zone numbers thebottom                             l e t - 4 7 6 ( ~ 1 6 2 1(zone <br />                                                                                                          )      13), l e t - 4 7 0 ( ~ 1 6 2 9(zone <br />                                                                                                                                               )     20B) figure. newly identified genes helped                                 l e t - 4 4 1 ( ~ 1 4 1 4(zone <br />                                                                                                          )     20B)] fertile 15&quot;. 2 0 &#194;&#176; , distinguish breakpoints. addition,                           l e t - 4 4 1 ( ~ 1 4 1 4 )escapes,(anescaper defined y-ray irradiation-induced mutations                             occasional individual, carried lethal factor described l e t - 4 0 2 ( ~ 9 9 2 ) , l e t - 3 3 6 ( ~ 7 4and <br />                                                                     1 ) let-   effective dose, overcomes crisis develops <br /> 336(s521) (ROSENBLUTH        et al. 1988), shown                      [HADORN(1945) citedin HADORN(1961)l) deficiencies sDf37, 38 39, respectively.                             occasionally worm fertile. 15   O f <br />  <br />  <br />    Fifty-fourof mutations analyzed ROSEN-                               homozygous l e t - 4 4 1 ( ~ 1 4 1 4worms  )            tested werefertile; <br /> BLUTH et al. (1988) recovered screens                                25&quot;, worms healthy similar   screens entire                         fertile. Worms homozygous l e t - 4 6 1 ( ~ 1 4 8 6 )(zone <br /> eT1 balanced region EMS-induced recessive lethal                           7), l e t - 4 1 8 ( ~ 1 6 1 7(zone <br />                                                                                                              )    11B4) l e t - 4 4 2 ( ~ 1 4 3 0(zone <br />                                                                                                                                                       ) <br /> mutations. Combiningtheir mutations gave                              15) blocked development later stage 15              O <br />  <br />  <br /> 220 mutations 89 101 identified essential                            20&quot; 25 mutants tested <br />                                                                                                            O , <br />                                                    Genetic Analysis LGV(1eft)                                                739 <br />                                                                TABLE 1 <br />                                                 Genes mapped zone LCV(1eft) <br />  <br />                    EMS    Total  Zone     Gene     duced alleles&apos;      alleles       Mutation <br />                                                   Mutapen*                      Phenotwe                 Comments <br />  <br /> 1A     let-450          0                 1          s2160                              Lost <br />   3 <br /> 1B1    let-336&apos;         3                            s1413      EMS      Early larval <br />                                                      s1420      EMS      Early larval <br />                                                      s1495      EMS      Early larval <br />        let-447           1                2          s1457      EMS      Me1 (egg lethal)   Multicell pre-lima bean period <br />                                                                                               eggs  <br />                                                      s1654           Me1 (egg lethal)   Multicell pre-lima bean period <br />                                                                                               eggs  <br />        let-458&quot;         1                 1          s1443      EMS      Early larval <br /> 1B2    let-448          0                 1          s1363      T c1     Mid larval <br /> 1 B3   let-437&apos;          1                1          s1405      EMS      Mid larval         Abnormal tail hermaphrodites <br /> 1c     let-453&apos;          1                1          s2167      EMS      ND                 strain let-417(~1424) <br /> 2      let-431           2                2          s1044      EMS      Sterile adult <br />                                                      s1049      EMS      Sterile adult <br />        unc-34            0                3          e566       EMS      Unc <br /> 4A     ges-1             0                6                              Gesd <br />        let-326           2                2          sl404      EMS      Early larval <br />                                                      s238       EMS      Mid larval <br />        unc-60            1               10          e677       EMS      Unc                visible unc allele <br />                                                      m35        EMS      Unc                visible unc allele <br />                                                      s1586      EMS      Early larval <br /> 4B     emb-29            1                5          s819       EMS      Egg lethal <br />                                                      g52        EMS      Egg lethal         Temperature sensitive <br />                                                      s1613      EMS      Egg lethal <br />                                                      s1666           Egg lethal <br /> 5      let-426           1                2          s1527      EMS      Early larval <br />                                                      s826       EMS      Mid larval <br /> 6A3    let-327           3                           s247       EMS      Slow               Translucent, cold sensitive <br />                                                      s1485      EMS      Slow               Translucent <br />                                                      s1496      EMS      Slow               Morphological abnormalities, <br />                                                                                               lethal 25&apos; <br />        let-478&apos;         1                1           s1620      EMS      Early larval       Me1 egg lethal sDfS0 <br /> 6B     let-347          1                1           s1035      EMS      Late larval <br /> 7      let-330         16               17           s1425      EMS      Early larval <br />                                                      si73       EMS      Mid <br />                                                      sl429      EMS      Early larval       Occasionally midlarval <br />                                                      sl433      EMS      Mid larval         Slow developing heterozygote <br />                                                      s1449      EMS      Mid larval <br />                                                      sl450      EMS      Early-mid larval <br />                                                      sl463      EMS      Early larval <br />                                                      sl468      EMS      Early larval <br />                                                      s1497      EMS      Early-mid larval   Sterile adult-Me1 sDf34 <br />                                                      s1515      EMS      Early larval <br />                                                      s1517      EMS      Early larval <br />                                                      s1518      EMS      Mid larval <br />                                                      s1531      EMS      Mid larval <br />                                                      s1543      EMS      Mid larval         Hypomorph? slow sickly <br />                                                                                              progeny Df <br />                                                      s1583      EMS      Early-mid larval <br />                                                      sl638      EMS      Mid larval         Me1 larval lethal                                                                                              let-330(~1433) <br />                                                      sl702      SPO      Mid larval <br />        let-461&apos;         1                3           sl486      EMS      Early larval       Mid larval 15 , Sel&apos; Lag&apos; <br />                                                                                              alleles <br /> BAl    lin-40           6               16          e21 7 3     EMS      Sterile adult      Vulva protrudes <br />                                                     s1053       EMS      Sterile adult      Vulva protrudes <br />                                                     s1345       Tc1      Early-mid larval <br />                                                     s1351       Tc1      Early-mid larval <br />                                                     s1352       Tc1      Early-mid larval <br />                                                     sl358       Tc1      Early-mid larval <br />                                                     sl360       Tc1      Early-mid larval <br />                                                     s1373       Tc1      Early-mid larval <br />    740                                                 R. C. Johnsen D. L. Baillie <br />                                                             TABLE Continued <br />  <br />  <br />                             EMS    Total  alleles&quot; <br />   duced <br />       Gene <br />          Zone                                 alleles Phenotype <br />                                                          Mutagenb <br />                                                           Mutation                                            Comments <br />  <br />                                                           s1506      EMS    Mid.larval <br />                                                           S 593    EMS    Late larval          Vulva protrudes <br />                                                           s161     EMS    Sterile adult        Vulva protrudes <br />                                                           s1634      EMS    Sterile adult        Vulva protrudes <br />                                                           s1669         Me1 (egg lethal)     Vulva protrudes  eggs laid) <br />                                                           s1675         Me1 (hatches)        Vulva protrudes  eggs laid) <br />                                                           sl704      EMS    Sterile leaky/Df     Vulva protrudes  strain                                                                                                   let-403(~246)] <br />                                                           s1916      uv     Early-mid larval <br />      8A2        let-338                                   s1020      EMS    Mid larval <br />                                                           s503       w&quot;y    Mid larval <br />                 let-455&apos;                                  SI447      EMS    Early larval <br />                                                           s1511      EMS    Early larval <br />                 let-466&apos;                                  s990       EMS    2nd generation Me1   Early larval lethal Fs generation <br />                                                           s1063      EMS    Me1                  Sterile adult F:! generation <br />      9A         let-344                                   s376       EMS    Egg lethal <br />                                                           s 1555     EMS    Mid larval <br />                 let-348                                   s1436      EMS    Early larval <br />                                                           s998       way    Mid larval <br />                                                           s1448      EMS    Me1 (egg lethal)     Slow developing <br />                                                           s1622      EMS    Early larval <br />                 let-430          1                        s1042      EMS    Sterile adult <br />                 let-459c         2                        s1432      EMS    Mid larval <br />                                                           ~1615      EMS    Me1 (egg lethal) <br />      9B         let-341          7                        sl03l      EMS    Egg lethal <br />                                                           s1415      EMS    Early larval <br />                                                           s1421      EMS    Early larval <br />                                                           s1454      EMS    Early larval <br />                                                           sl516      EMS    Hatches <br />                                                           S 534    EMS    Egg lethal <br />                                                           s1571      EMS    Early larval <br />                                                           s2118         ND <br />                 let-342          5               5        s1442      EMS    Early larval         Complements sDf50 <br />                                                           s 1029     EMS    Mid larval           Sterile adult sDf50 <br />                                                                                                    (cross-fertilitynot tested) <br />                                                           s1487      EMS    Early larval         Sterile adult sDf50 <br />                                                                                                    (cross-fertile) <br />                                                           s1549      EMS     Early larval        Sterile adult sDf50 <br />                                                                                                    (cross-fertile) <br />                                                           s1616      EMS     Mid larval          Sterile adult sDf50 <br />                                                                                                    (cross-fertilitynot tested) <br />                 let-345          4               5        s578       EMS     Mid larval <br />                                                           s1452      EMS     Mid larval <br />                                                           s1509      EMS     Late larval <br />                                                           s1510      EMS     Mid larval <br />                                                           sl690          ND <br />                 unc-62           0               2        e644       EMS     Unc                 Viable unc allele <br />                                                           s472       SPO     Egg lethal <br />       10        let-331          1                1       s427       EMS     Mid larval          Slow, cold sensitive <br />                 let-350          1               2        s250       EMS     Sterile adult <br />                                                           s2126          ND <br />                 let-415          2               3        s1525      EMS     Early larval <br />                                                           s129       EMS     Late larval <br />                                                           s1505      EMS     Late larval <br />                 let-41 7         2               4        s204       EMS     Early larval <br />                                                           s1313      EMS     ND                  strain rol-3(~10?0) <br />                                                           s1424      EMS     ND                  strain let-453(~2167) <br />                                                           s1679          ND <br />                 let-419          3                3       s219       EMS     Mid larval <br />                                                           sl483      EMS     Early-mid larval <br />                                                           s 1539     EMS     Early-mid larval <br />                 let-420          6                7       s 1046     EMS     Sterile adult       Vulva protrudes <br />                                                           Genetic Analysis LGV(1eft)                                                     74 1 <br />  <br />                                                              TABLE l-Continued <br />  <br />                     ~ <br />  <br />  <br />  <br />  <br />                          EMS     Total  Zone       Gene <br />          duced                 alleles&quot;      alleles       Mutation   Mutagenb         Phenotype                    Comments <br />  <br />                                                            s723       -V&quot;Y       Sterile adult          Vulva protrudes <br />                                                            s1058      EMS        Sterile adult          Vulva protrudes <br />                                                            s1478      EMS        Sterile adult          Vulva protrudes <br />                                                            s1573      EMS        Sterile adult          Vulva protrudes <br />                                                            s1584      EMS        Sterile adult          Vulva protrudes <br />                                                            s1603      EMS        Sterile adult          Vulva protrudes, abnormal tail <br />          let-428               2                2          s1490      EMS        Late larval            Dumpy, extrudes viscera <br />                                                            s1070      EMS        Sterile adult <br />          let-440&apos;              5                5          s1411      EMS        Hatches <br />                                                            s1440      EMS        Hatches <br />                                                            s 1552     EMS        Hatches-early larval <br />                                                            s1560      EMS        Hatches-early larval <br />                                                            s1589      EMS        Hatches-early larval <br />          let-452&apos;                                1         s1434      EMS        Hatches <br />          let-477&apos;                                1         sl608      EMS        Slow <br />          let-481                                 1         sl636      EMS        Hatches <br />          unc-46                                 3          el 77      EMS        Unc <br /> 1 1A 1   let-443&quot;                               1          s1417      EMS        Early larval <br /> 1 1A2    let-401                                1          sl93       EMS        Mid larval <br /> 1 1B 1   let-349                                3          s217       EMS        Early larval <br />                                                            s502       W&quot;Y        Early larval <br />                                                            s5 72      EMS        Late larval            Cold sensitive <br /> 1 1B2    let-429               2                 2         s584       EMS        Sterile adult <br />                                                            s1597      EMS        Slow                   Morphological abnormalities <br />          let-439&apos;              4                4          s1407      EMS        Early larval <br />                                                            s 1503     EMS        Early larval <br />                                                            s1522      EMS        Early larval <br />                                                            sl524      EMS        Early larval <br />          let-462&apos;              3                 3         s 1594     EMS        Early-mid larval       Blocks earlier sDf36                                                                                                           sDf&apos;s 20, 26, 30 nDf32 <br />                                                             s1481     EMS        Mid larval             Molting problem <br />                                                             s 1590    EMS        Mid larval <br />          let-473&apos;                                1          s 1602    EMS        Early larval           Trapped old cuticle <br />          let-479&apos;                                1          s1576     EMS        Sterile adult          Lays unfertilized eggs <br /> 11 B3    let-329                                 1          s575      EMS        Early larval <br />          let-463&apos;                                1          s2168     EMS        ND                     strain rol-3(~1473) <br /> 11B4     let-418                                 1          s1045     EMS        Sterile adult/Mel      Vulva protrudes 24&quot; <br />                                                             s1617     EMS        Sterile adult          Vulva protrudes, partial rescue <br />                                                                                                           15&quot; <br />          let-421               4                 4          s1477     EMS        Egg lethal <br />                                                             s288      EMS        Leaky <br />                                                             s1460     EMS        Mid larval             Dumpy <br />                                                             s1632     EMS        Hatches                Dumpy, curls, escapers late lar- <br />                                                                                                          val vulva protrusion <br /> 1 1 B5   let-422               4                 7         s194       EMS        Early larval <br />                                                            s738       7-ray      Early larval <br />                                                            s739       way        Early larval <br />                                                            s1312      EMS        ND                     strain let-337(~382) <br />                                                            s1563      EMS        Early larval <br />                                                            s1578      EMS        Mid larval             Permissive temperature 15&quot; <br />                                                            s1548      EMS        Early larval <br /> 12       unc-83                0               11          e1408      EMS        Unc <br /> 12A      let-402               1                3          s1526      EMS        Early larval <br />                                                            s127       EMS        Mid larval <br />                                                            s500       ?-ray      Early larval <br />          let-444&apos;              3                3          s1418      EMS        Early larval <br />                                                            sl459      EMS        Mid larval <br />                                                            s1569      EMS        Me1 (egg lethal) <br /> 12B      let-403               2                4          s1482      EMS        Early larval <br />                                                            s120       EMS        Mid-late larval <br />                                                            s246       EMS        ND                     strain aslin-40(~1704) <br />                                                            s498       way        Late larval <br />  742                                                       R. C. Johnsen D. L. Baillie <br />                                                                TABLE 1-Continued <br />  <br />  <br />                           EMS     Total    Zone        duced <br />                 Gene            alleles&quot;      alleles       Mutation   Mutagenb          Phenotype                  Comments <br />  <br />  13           dpy-11           0               10           e224       EMS        DPY <br />               let-337          4                5           s1426      EMS        Early larval <br />                                                             s382       EMS        ND                   strain let-422(~1312) <br />                                                             s825       EMS        Mid larval <br />                                                             sI018      EMS        Me1 (egg lethal) <br />                                                             sl024      EMS        Me1 (egg lethal) <br />               let-476&quot;         1                            s1621      EMS        Sterile adult        Permissive temperature 15&quot; <br />  14           let-410          1                            s815       EMS        Mid larval <br />                                                             s1565      EMS        Mid larval <br />               let-469c                                      s1582      EMS        Sterile adult <br />               let-471&apos;                                      s1570      EMS        Me1 (early larval) <br />               let-472&apos;                                      s1605      EMS        Mid larval           Morphological abnormalities <br />               unc-70                                        e524       EMS        Unc <br />                                                             s115       EMS        Mid larval           unc-70 recessive alleles                                                                                                         dominantly weak Uncs <br />                                                             s I406     EMS        Early larval <br />                                                              s 1502    EMS        Hatches <br />                                                             s1532      EMS        Early larval <br />                                                             s1557      EMS        Hatches <br />                                                             s1639      EMS        Hatches <br />  15           let-332          8                 8          s234       EMS        Egg lethal <br />                                                             s369       EMS        Egg lethal <br />                                                             s1021      EMS        Early larval         Escapers <br />                                                             s1441      EMS        Egg lethal <br />                                                             s1464      EMS        Egg lethal           developed worms eggs <br />                                                             s1475      EMS        ND <br />                                                             s1498      EMS        Egg lethal <br />                                                             s1567      EMS        Egg lethal <br />               let-339                                       s1444      EMS        Early larval <br />                                                             s1019      EMS        Me1                  Cold sensitive <br />                                                             s1469      EMS        Early-mid larval     egg lethals <br />               let-343                                       s1025      EMS        Egg lethal <br />                                                             s816       EMS        Early-mid larval <br />                                                             s1410      EMS        Early larval <br />                                                             s1428      EMS        Early-mid larval <br />                                                             s1465      EMS        Slow                 Cold sensitive <br />                                                             s1579      EMS        Early larval <br />               let-346          6                 7          s1619      EMS        Mid larval <br />                                                             s373       EMS        Late larval <br />                                                             s1026      EMS        Late larval <br />                                                             s1575      EMS        Mid-late larval <br />                                                             s 1580     EMS        Mid-late larval      protruding vulvae <br />                                                             s I630     EMS        Sterile adult <br />                                                             s2166             ND <br />               let-404          0                 1          s119       EMS        Mid larval <br />               let-425          1                 1          s385       EMS        Sterile adult <br />               let-438          0                 1          s2114             ND <br />               let-442&apos;         3                 3          s1416      EMS        Early-mid larval <br />                                                             s 1430     EMS        Early-mid larval     Sterile adults 15&quot; <br />                                                             s I535     EMS        Early-mid larval <br />       1       let-46gC         1                            s 1533     EMS        Early larval <br />               mcc- 1           0               50                                 Mec <br />               unc-68           0                8            e540      EMS        Unc <br />   166         let-335          6                             s 1439    EMS        Early larval <br />                                                              s232      EMS        Mid larval <br />                                                              s1412     EMS        Late larval <br />                                                              s1476     EMS        Early larval         Tends coil <br />                                                              s1520     EMS        Early larval         Escapers (sterile adults vulva <br />                                                                                                          protrusion) <br />                                                              s1523     EMS        Early larval <br />           3   let-405           1                            s116      EMS        Early larval <br />                                                              s829      EMS        Mid larval <br />                                                       Genetic Analysis LGV(1eft)                                                   743 <br />                                                          TABLE 1 4 n t n u e d <br />  <br />  <br />                       EMS    Total   Zone       Gene     duced alleles&apos;      alleles       Mutation <br />                                                      Mutagenb                      Phenotype                     Comments <br />  <br />                                                        s388       EMS       Mid larval <br />         let-406            0                 1         s514       ?-ray     Mid larval <br />         let-41 1           4                 4         s1595      EMS       Mid larval <br />                                                        s223       EMS       Late larval <br />                                                        s1453      EMS       Mid larval <br />                                                        s1553      EMS       Mid larval <br />         let-423             1                2         s818       EMS       Early larval <br />                                                        s1550      EMS       Hatches/escapers      mid larval lethals <br />         let-449             0                1          s1343     Tc1       Early larval <br />         let-474&quot;            1                1          s1577     EMS       Early larval <br />         let-480&apos;            1                1          s1607     EMS       Sterile adult <br /> 17      let-408             0                2          s827      EMS       Egg lethal <br />                                                         s195      EMS       Late larval <br />         let-413             3                4          s128      EMS       Egg lethal <br />                                                         s1431     EMS       Egg lethal <br />                                                         s1451     EMS       Egg lethal <br />                                                         s 1455    EMS       Early larval <br />         let-414             0                2          s114      EMS       Mid larval <br />                                                         s207      EMS       Mid larval <br />         let-424             3                3          s384      EMS       Sterile adult <br />                                                         s248      EMS       Sterile adult <br />                                                         s1587     EMS       Sterile adult <br />         let-436&quot;            1                1          sl403     EMS       Early larval <br />         let-445&apos;            1                1          s1419     EMS       Me1 (larval lethal)   Tight coiler <br />         let-456&apos;            1                1          s1479     EMS       Early larval <br /> 18      let-412             2                2          s1598     EMS       Sterile adult         Roller <br />                                                        s5 79      EMS       Me1 (egg lethal) <br />         let-464c            2                2         s1504      EMS       Early larval          Approximately % hatch <br />                                                         s1530     EMS       Early larval <br />         rol-3              8               13          e754       EMS       Roller <br />                                                        s126       EMS       Early larval <br />                                                        s422       EMS       Mid larval <br />                                                        s501       ?-ray     Early larval <br />                                                        s742       ?-ray     Mid larval <br />                                                        s833       EMS       Mid larval <br />                                                        s1030      EMS       ND                    strain lct-417(~1313) <br />                                                        s1040      EMS       Mid larval            Temperature sensitive, roller <br />                                                        sl408      EMS       Early larval <br />                                                        sl409      EMS       Early larval <br />                                                        s1473      EMS       ND                    strain let-463(~2168) <br />                                                        s1494      EMS       Early larval <br />                                                        s1519      EMS       Early larval <br /> 19      let-334            2                 2         s908       EMS       Early larval <br />                                                        s383       EMS       Mid larval <br />         let-340            2                2          s1508      EMS       Early larval <br />                                                        s1022      EMS       Mid larval <br />         let-409            5                7          s 1480     EMS       Hatches               Tends tocurl <br />                                                        s823       EMS       Early larval <br />                                                        s206       EMS       Early larval <br />                                                        s1507      EMS       Hatches <br />                                                        s1528      EMS       Hatches <br />                                                        s 1546     EMS       Early larval <br />                                                        s1547      EMS       Hatches <br />         let-416            0                1          s113       EMS       Late larval <br /> 20A     let-460&apos;           0                1          s1664            ND <br />         let-467&apos;           1                1          s1521      EMS      Hatches <br />         unc-23             0                8          e25        EMS      Unc <br /> 20B     let-407            1                4          s1631      EMS      Hatches <br />                                                        s830       EMS      Early larval <br />                                                        sl18       EMS      Early larval <br />                                                        s2122           ND <br />         let-44 1 &apos;         1                1          s1414      EMS      Early larval           Escapers, permissive temperature <br />                                                                                                     15&quot; <br />  744                                                      R. C. Johnsen D. L. Baillie <br />                                                              TABLE l-Continued <br />  <br />  <br />  <br />                          EMS     Total     Zone ducedGene              alleles&apos;      alleles       Mutation   Mutagenb          Phenotype                  Comments <br />  <br />             let-454&apos;           1                 1          51423     EMS        larval <br />                                                                                   Mid                  Double cuticle, pumping <br />             let-470&apos;           2                2           51581     EMS         Mid larval <br />                                                             $1629     EMS         Slow                 Permissive temperature 15&quot; <br />       1     let-475&quot;           1                            s1606     EMS         Sterile adult <br />     * alleles isolated screening entire eT1 <br />                                                             balanced region included  EMS-induced alleles screens selected <br /> regions LGV(left) included. <br />      = formaldehyde; Spo = spontaneous; T c l = mobilization transposon T c l ; UV = ultraviolet irradiation; y-ray = y-ray <br /> irradiation. <br />     &apos; Essential genes identified paper. <br />       Ges = gut esterase. <br />     &apos; Sel = supressor lin-12 (lin = lineage defective). <br />    &apos;Lag = lin-12 glp-1 double mutant phenotype (glp = germline proliferation defective). ND =  <br />  <br />  <br />  lower temperatures. mutations permissive                        (Table 2). genes, let-479 (zone 11B2) <br /> temperature 25&quot;, 25 &quot; let-343(~1465)(zone                         let-469 (zone 14), good candidates spe <br />  15) blocked development later stage 20&quot;.                      genes. <br />    Table 2 results testing sperm                           10 genes LGV(left), production sperm defects mating hermaphro-                            identified recessive nonlethal phenotype ( e . , <br /> dites containing homozygous mutations resulted                         Rol, Unc) recessive lethal alleles  sterile adults, maternal-effect lethals slow devel-                   (1) recessive lethal allele ( ~ 1 5 8 6of <br />                                                                                                                               ) unc-60 <br /> opers wild-type males. Hermaphrodites contain-                         (zone 4A) (predicted MCKIM et al. 1988); (2) ing mutations produced apparently                           spontaneous recessive lethal allele ofunc-62 (zone 9B) <br /> mal progenywhen mated withwild-typemales. Zet-                              isolated ROSENBLUTHet al. (1988); (3) 471(s1570) (zone 14) maternal effect-larval lethal                     recessive lethal alleles unc-70 (zone 14) (predicted cross-fertile. Zet&quot;f(s1569) (zone 12A) let-                     PARKand HORVITZ        1986a), ROSENBLUTH <br /> 346(s1630) (zone 15) twoweakallelesofgenes                              et al. (1988) new alleles( ~ 1 4 0 6 , ~ 1 5 <br />                                                                                                                                 s1532, <br />                                                                                                                                   02, canonical alleles blockdevelopment early                        s1557 ~ 1 6 3 9 )Worms <br />                                                                                                    .      heterozygous stage (let-444) mid-larval stage (let-346).                        recessivelethalallelesof        unc-70 appear mutations, let-470(~1629)(zone 20B), let-                          weakly uncoordinated. Worms homozygous 476(s1621) (zone 13)  genes repre-                        alleles unable develop- <br /> sented allele), putative sperm                         mentally arrested larval stage. Occasionally, <br /> defective andtemperature sensitive. Zet-470(~1629)                          worms homozygous unc-70(~1502)escape. causes slowdevelopment 20 &quot; . mutations,                        mature early-larval blockedprogeny let-479(~1576)(zone 11B2) Zet-469(~1582)(zone                           occasional escaper. lethal alleles com- <br /> 14), arecross-fertile sterile adults; Zet-479(~1576)  lays                  plementation tested e524  visible allele). <br /> unfertilized oocytes.                                                       tested s115 (anearlyrecessive <br />    96 essential genes LGV(left)                        lethal allele) tested e524 ( R e <br /> mutant phenotypes noted, canonical                           SENBLUTH et al. 1988); (4) 12 recessive lethal alleles alleles maternal effect lethal phenotypes [let445                      rol-3 (zone 18). identified alleles( ~ 1 4 0 8 , <br /> (zone 17), Zet-447 (zone lBl), let-459 (zone 9A), Zet-                      s1409, s1473,s1494 s1519), otherseven 466 (zone 8A2) let471 (zone 14)].                            identified ROSENBLUTHet al. (1988). tested cross-fertility: let-447 Zet-459                    alleles  possible exception s1473)  <br /> fertile let-471   genes                      cessive early-larvallethal phenotype. s1473 <br /> appeared pure maternal effect genes.                                  strain recessive lethal mutation <br />    Testing cross-fertility mating wild-type                     [Zet-463(~2168)  (zone 11B3)], developmental <br /> males useful identifying meZ genes                      blocking stages individual mutations identifying mutations cause defective                         determined. Oneof rol-3 alleles, ~ 1 0 4 0which <br />                                                                                                                               ,     spermatogenesis (Spe). Mutations spe    genes ones                   identified ROSENBLUTH     et al. (1988), temperature result hermaphroditic worms becomingsterile                         sensitive. 20&quot; rol-3(~1040 recessive lethal adults make functional oocytes. identi-                        15&quot; fertile roller. lethalalleleswere <br /> fied mutations cross-fertile,                        complementation tested s442 ~ 1 2 6 <br /> whichhavebeenmapped              genes LGV(left)                       recessive lethal mutations). ROSENBLUTH         et al. <br />                          Genetic Analysis LGV(1eft)                                                               745 <br />                              ..                                                                     -2 <br />                     5        t::                                                                        rig <br />  <br />  <br />  <br />  <br />                                                                                                                            -3                                                                                                    : u              $2 <br />                                                                                                               U <br />  <br />  <br />  lPR                  :                     p                &gt; <br />                                                                                     !!            82 <br />          4!!ztilj <br />          04&apos; Si?&apos; <br />   s : g fgt<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;f&amp;g <br /> wBBf3iri5         1 <br />                                    4 !!     j                 ]!e                      -2 <br />                                                                                            f        2; <br />                                                                                                     mc, <br />                                                                                                     2-0 <br />                                                                                                     -K <br />                                                                                         t           ;*u c y <br />                                                                                        1% <br />                                                                                         5               u &apos; c9 <br />                                                                                                              , <br />                                    -                                                                    - <br />  <br />                     rae                     BR                                                                 :              E? <br />  <br />                         rn <br />                              34 <br />                              12:        7yi]s <br />                                            q # <br />                                                               = <br />                                                                    Ll <br />                                                                                        -7i <br />                                                                                        &quot; <br />                                                                                            22 <br />                                                                                                L <br />                                                                                                         $5 55 <br />                                                                                                         8d 8 <br />  <br />                         &amp;2 q 8                                                                    .08 <br />                                                                                                     52 <br />                                                                                                     s+ <br />                                                                                                         K T <br />  <br />  <br />  <br />  <br />                      +  5.                                                                                                                                                                   82 <br />                                                                                                     E6 <br />                                                                                                     qg <br />                                                                                                         0 <br />                                                                                                         m-cl <br />  <br />  <br />  <br />                         E                                                                  P        -g <br />                                                                                                     9,. <br />  <br />  <br />                 allfI!8&apos; j         - <br />                                    - <br />                                              7 <br />  <br />  <br />  <br />  <br />                                                         ~ <br />  <br />  <br />  <br />  <br />                                                                                 - <br />                                                                                            2 <br />  <br />                                                                                            3 <br />                                                                                                     Yw <br />                                                                                                     .4 <br />                                                                                                         x * <br />  <br />                                                                                                         ; <br />                                                                                                     .-M K <br />                                                                                                         6 2 <br />                                                                                                     -* .9 <br />  <br />                              1-                                                                     .K 8 <br />                                                                                            E       .$       g <br />                                                                                                    2% <br />                                         84 <br />                                                                                                     &quot;E <br />                                                                                                    5&quot; <br />                                                                                                    ClA <br />  <br />                                                                    1 <br />                                                                             7 <br />                                                                                                     2s <br />                                                                                                    .-$                                                                                                     .d <br />                                                                                                     cL                                                                                                                                                                                       .       &apos;E cl <br />                                                                                                     85 <br />                           8                                -, <br />                                                                                                    noa <br />                                                                                                    m <br />                                                                                                    4$ <br />                                                                   B <br />                              f-                                                            s <br />                                                                                                    = X <br />                                                                                                    5.&quot; <br />                                                                                                    eld <br />                                                                                                    Clz <br />                                                        1           -      $m g- <br />                                                                                                     o w <br />                                                                                                     e 8 <br />                                                                                                    2M M <br />                                                                         - 3                         IG <br />                                                               8                          2 <br />                                                                                                    u                                                                                                     q           6 <br />  <br />                          R-                                             $88                        3 <br />                                                                                                     2$$ <br />                                                                                                             g$ <br />                                         - <br />                          !8--                                                         &quot;            Gg; <br />  <br />  <br />                                                                   L <br />                                         - 7 <br />                                                                                       -s           &quot;23, <br />                                                                                                    5 wv <br />                                                                                                    L w <br />                                                                                                     O % L <br />                                                                                     5            az2 <br />  <br />                     +: u! <br />                                                                                                    2;$ <br />                                                  *                                     .y 2 2 <br />                                                                                       - N  w 2 <br />                                                                                        0 * ., <br />  <br />  <br />                     &quot;iil1,1{1 <br />                                                                                        -2  U % <br />  <br />                                                                                       -  153 <br />                   !s                    8I <br />                                                Iaa <br />                                                  O     .,,, <br />                                                                                        E <br />                                                                                        s <br />                                                                                                    Awe, <br />                                                                                                                                                                                                       ,3dS <br />                                                                                                    z22 <br />                                                                                                             2 5 <br />  746                                                                R. C. Johnsen D. L. Baillie <br />  <br />                                                                                                                    TABLE 2 <br />                         1 map unit <br />                                                                                         Results testing mutations cross-fertility mating                                                                                                               wild-type males <br />  <br />                                                                                         Zone    Gene    Allele       Blocking stage (20&quot;)       Cross-fertile <br />  <br />                                                                                        1B1  let-447 s1457    Maternal egg lethal                                               nurjil                                                   SA1  lin-040 s1611    Sterile adult, vulva protrudes         <br />                         SDns                                                                                                              9A   let-348 <br />                                                                                                 s1448        Maternal egg lethal, slow              <br />           L    L                        SLY30                                                                                                                               9A <br />                                                                                        10 <br />                                                                                                 s1615 <br />                                                                                             let-459 <br />                                                                                             let-420 <br />                                                                                                 s1478 <br />                                                                                                              Maternal egg lethal <br />                                                                                                              Sterile adult, vulva protrudes <br />                                                                                                                                                                                                                                                                                                       Nob <br />                                              l    nDil8        ,                       11B2 let-429sI597 <br />                 , sDf3@,                     usDf37 <br />                                                                                                              Slow, morphological                                   , a1442 ,                                                                                 abnormalities <br />                                                                                        llB2 let-479s1576     Sterile adult, lays unfertilized      Yes <br />                                                                                                                eggs <br />                       ,                                                              llB4 let-418s1617     Sterile adult, vulva protrudes         1011Al <br />   1185 <br />    1184 <br />    1182 <br />    1181                                      12A 128      13        14       15                                (partial t s ) <br />           HA2       1183 <br />                                                                                        12A let-444 s1569 Maternal egg lethal                   Yes <br />    FIGURE2.-Expansion zones 1 1Al-15. unc-83 mapped                                                     canonical) J. THOMAS  (personal communication). &apos;let-349 let-422                     13   let-476 ~ 1 6 2 1Sterile adult ( t s )                 Yes zone 11B1 11B5, locations map                         14   let-469 s1582 Sterile adult                            Yes <br /> based factor mapping distances unc-46 (zone 10).                           14   let-471 s1570 Maternal larval lethal                   Yes <br /> &apos;sDf36 overlap sDf44 right, 11B1                    15   let-346 s1630 Sterile adult, canonical             Yes <br /> right 1182 1 1 B3 left 1182.                                                         allele <br /> (1988) shown s226 fails complement e754                                    16   let-480 s1607 Sterile adult                                                                                                                   17   let-424 s1587 Sterile adult                             allele roller phenotype),                                 18   let-412 s1598 Sterile adult, roller                    lethal allele s422.                                                                    20B let-470 s1629 Slow, canonical allele                Yes <br />    genes LGV(1eft) deserve <span id='am-9' about='rdfs:comment' typeof='owl:Thing'>comment</span>. <br />                                                                                          alin-40@1634) sterile adult allele, male rescued. <br />    let-447 (zone lB1): let-447 alleles ( ~ 1 4 5 7                                alleles let-420(~1573, ~2584,     ~ 1 6 0 3                                                                                                                                                          ) ~ 2 6 5 4 ) . s 2 6 5was <br />                           4 isolated screenof form-                            male rescued. Note: strains contained aldehyde treated chromosomes         (JOHNSEN BAILLIE                             lethal mutation tested cross-fertility. <br />  1988). alleles recessive maternal effect <br /> egglethalphenotype.Generally,hermaphrodites                                           cation) alleles aresemi-dominant suppressors <br /> homozygous allele lay eggs,                                   lin-12 dominant alleles (J. THOMAS,     personal com- s2457/Df s2654/Df hermaphrodites consist-                                    munication). allele, (s1486), recessive EMS- <br /> ently lay eggs. T h e developmental blocking                                     induced lethal mutation results Lag (lin-22 <br /> stages eggs cases (homozygous                                   glp-2) phenotype 20&quot; blocks development hemizygous s1457 ~ 2 6 5 4appear)       thesame:                           advancedstage 15&quot;, causes mid- <br /> multicellular mass fewer 550 cells  did                                  larval lethality. suggests s1486 null. reachthe lima bean (startof morphogenesis) stage,                                    let-330 (zone 7): let-330 large genewith respect VON EHRENSTEIN         SCHIERENBERC     1980; SULS-                           EMS mutagenesis, EMS forward mutation <br /> TON et ul. 19831. propose possible expla-                                  rate 1.1 X lo-&apos;. greater rateof 6.4 <br /> nations theseresults: (1) let-447 functions:                              X 10-4 unc-22 (unpublished results cited MOER- <br />   involved production germ cells;                                 MAN BAILLIE1981). unc-22 spans 47 kb ge- (b) maternal effect egg development.                                  nomic DNA  19 kb coding DNA) alleles let-447 hypermorphic                                 encodes muscle protein twitchin function probably null second. (2)                                  homology vertebrate striatedmuscle pro- <br /> T h e increased number eggs duetothe                                        tein titin (BENIANet al. 1989). Twitchin molec- <br /> hemizygosity let-447 hemizygosity                                   ular weight approximately 700,000 daltons (BEN- let-336, deleted currently known                               IAN et al. 1989). Figure 1, let- <br /> deficiencies delete let-447. let-336(~1413)domi-                                 461 separated let-330. alleles <br /> nantly effects brood size. Nematodesheterozygous                                      let-330 ( ~ 1 4 9                                                                                                            7 ~ 2 5 2 8are <br />                                                                                                                        ) severe oversDf34 s1413 produce average 20% progeny                                      sDfs 26 42. Worms heterozygous wild-type hermaphrodites  unpublished                                     s1497 o r ~ 1 5 1 8over let-330(~573), sDj26 o r <br /> sults). deficiency deletes let-447 let-336                             sDf42 block early-mid larvae, sDf34, ma- help choosing explanations.                                  ture adulthood andgive occasional larval progeny. <br />    let-461 (zone 7): gene called lug-2 J.                                 sDf34 probably breaks let-330. let- <br /> KIMBLEand sel-3 J. THOMAS.         let-462 alleles                        330(~1634)/let-330(~1433)    Mel-larval lethal (imply- phenotype lin-I2 glp double mu-                                    ing possible interallelic complementation) let- <br /> tants (J. KIMBLEand E. LAMBIE,personal communi-                                       33O(s1638)/sDfs 27, 34 42 larval lethal.                                              Genetic Analysis LGV(1eft)                                               747 <br /> data imply  (1) let-330 large gene; (2) let461     let-342 allele, mature andbecome cross-fertilester- internal let-330 left; (3) let-   ile adults. sDf50 does break close let- <br /> 330 multidomain gene encodes pro-            342, molecularly identifying right break- <br /> teins t.hat function multimers. sDf34 does break       point sDf50 able clone let-342. let-330, molecular identification        let-420 (zone 10): Worms homozygous right breakpoint sDf34 lead thecloning        sevenallelesof      let-420 sterile adults let-330.                                                     protruding vulvae. let-420 appears large target <br />    lin-40 (zone 8A1): developmental blocking             EMS mutagenesis. allele induced stages 16 recessive lin-40 alleles range         gamma-irradiation induced EMS. early-mid larval lethality maternal effect early-lar-     EMS-induction frequency approximately times <br /> Val lethality.Wormshomozygous               higher average hit frequency essential <br /> alleles develop adults (sterile maternal effect      genes LGV(1eft). <br /> lethal) protruding vulvae. results interal-         let418 (zone llB4): Bothallelesof let418 lelic complementation tests lin-40 com-       temperature sensitivephenotypiceffects.           ter- <br /> plex locus classes alleles (unpub-     minal phenotypes ofwormshomozygous                  let- <br /> lished results). Alleles complementation         418(s1045) include mid-larval lethality 15&quot;, mater- <br /> class cause similar phenotypes. alleles lin-40       nal effect early-larval lethality 20&quot; 24&quot; deficiencies delete lin-40, dominantly cause        adults sterile protruding vulvae (R.E. increase (approximately 40%) number             ROSENBLUTH,      personal communication), interestingly <br /> self-progeny C. elegans hermaphrodites.            middle temperature         permissive. <br /> implies lin-40 alleles affect         Worms homozygous let-418(~1617)         15&quot; vulva-lineage delay switch sperma-         maternal effect lethal phenotype, giving larval <br /> togenesis oogenesis hermaphrodites.                    progeny at20&quot; 25&quot; sterile adults    let-341 (zone 9B):let-341 fourthbiggest target     protruding vulvae. EMS mutagenesis LGV(left). canonical                 genes [let418 (zone 11B4), let-420 (zone 10) <br /> allele let-341(~1031) recessive early-larval le-     lin-40 (zone SAl)] recessive allelesthat cause <br /> thal. let-341 alleleshavebeenshown                adult worms sterile andto protruding <br /> dominant suppressors multivulva lin X) muta-           vulvae genes putatively affect    vulva- <br /> tions (S. G . CLARKand H. R. HORVITZ,personal                lineage. <br /> communication). addition, mutation let-bO(lV),          Zone 16: region deleted <br /> C. eleguns ras gene (HAN STERNBERG        1990), sup-    deficiencies possible haplo- <br /> press lethality alleles oflet-341 including      insufficient gene region prevents sI03I (S. G . CLARKand H. R.HORVITZ, personal                isolation deficiencies. location genes <br /> communication). let-341 probably         zone 16 region confirmed 3-factor <br /> lin-15-let-60 vulva induction pathway.                       mapping (ROSENBLUTH       et al. 1988), 3-factor map- <br />    Zet-342 (zone 9B): alleles let-342,     ping used confirm position let- EMS-induced. alleles           449, let-474 let-480. 2-factor mapping data <br /> block development early-larvalstage ( ~ 1 4 4 2 ,     let-449 let-474 unc-46 (zone lo), got <br /> s1487 s1549), (s1029 s1616)            distances 4.3 (2.1-9.1) 4.8 (2.7-7.6)m.u., <br /> block atthe mid-larvalstage.Occasionallyworms                respectively. distances consistent homozygous s1616 mature andhave internal larval          genes zone 16. factor mapping Zet- <br /> progeny. alleles sDj27, nDf32 let-        480  adult sterile)  genes <br /> 342(s1029) appear block early-  mid-) larval        represented allele  possible <br /> stages. s1442 complements sDf50, worms hetero-           alleles hypermorphic <br /> zygous otheralleles sDf50 block        (generally rare alleles) worms heterozygous <br /> sterile adults. Worms heterozygous s1487          appropriate allele deficiency appear s1549 sDf50 werecross-fertile (s1029             normal. case mutations s1616 tested cross-fertility). T o explain      type LGV(1eft). data proposals (1) sDf50 breaks               rob3 (zone 18): Wehave identified fiveEMS  <br /> near promoter region let-342 modifies         duced recessive lethal alleles r o f - 3 ( ~ 1 4 0 8 , ~ 1 4 0 9 , <br /> level transcription (2)let-342 transcribed right    s1473, s1494 ~ 1 5 1 9                                                                                             .    increases number left sDf50 breaks let-342 near       recessive lethal alleles rol-3 <br />                                                                                             12  otherseven <br /> prime end. proposals chromosome              identified ROSENBLUTH        et al. 1988). taining sDf50 produces normal levels oflet-342        new alleles (withthe possible exception ~ 1 4 7 3 ) <br /> product normal product. allows worms          recessive early-larvallethal phenotype. <br /> containing sDf50 chromosome putative null         s1473 strain recessive lethal mu- <br />  748                                            R. C. Johnsen D. L. Baillie <br /> tation [let-463(~2168)(zone 11B3) individual             586 genes, 77 LGV. Assuming developmental blocking stages mu-               biases distribution, tations individually determined.             essential genes followthe distribution, 13% <br /> rol-3 alleles, identified earlier screen         C. elegans&apos; genes LGV. 77 genes <br />  ROSENBLUTH      et al. 1988), temperature sensitive.     LGV 21 LGV(left), 45 LGV(right), 20&quot; recessive lethal 15&quot; fertile       11 mapped position. Ignor- <br /> roller. existenceof thetemperature sensitive                 ing 11 genes positioned, 32% genes allele causes rolling thepermissive temperature          LGV LGV(1eft) LGV(1eft)  <br /> allows argue weak alleles cause rolling           tains 4.2% essential genesin genome. Using <br /> stronger allelescauselethality. Interestingly let-412            minimum estimate 120 essentialgenes (zone 18), separated rol-3            LGV(left), calculate minimum chromosomal rearrangement breakpoint,                   2850 essential genes C. elegans genome. BREN- <br /> allele ( ~ 1 5 9 8                    )     isrecessive     sterile adult     NER&apos;S (1974) original estimate 2000,HOWELL        roller phenotype. s1598 complementa-               ROSE(1990) estimated 3300 CLARK(1990) esti- <br /> tiontestedagainsttwo           rol-3 alleles, s833 andthe        mated 3500. <br /> &quot;visible&quot;allele e754, complemented                     calculate forward mutation rate possibility rol-3-let-412 complex locusis        average gene, removed highly mutable genes <br /> examined W. B. BARBAZUK           (personal communica-        sample  did CLARK(1990) HOWELL <br /> tion). possibility rol-3 let-412         ROSE (1990)l. rate, based 14,545 chro- <br /> neighboring genes affect related                mosomes (determined removing geneswith <br /> developmental pathway($.                                         alleles), 6.7 x          mutations                                                                  gene chromosome. compare rates <br />                       DISCUSSION                                  adjusted forward mutation <br />                                                                  rates cited BRENNER(1974), HOWELLand ROSE <br />    results presented paper brought                                                                  (1990), CLARK(1990) rates expected 101 number ofessentialgenes               identified                                                                  0.012 M EMS, using dose-response curve (ROSEN- <br /> LGV(1eft). 89  EMS-induced <br />                                                                  BLUTH, CUDDEFORD BAILLIE1983). BRENNER <br /> mutations isolated screening eT1 region <br />                                                                  (1974) estimated forward mutation rate genes recessive lethals. used data                                                                  C. elegans withvisiblephenotypes.Hisestimate <br /> truncated Poisson equation derive minimum esti- <br />                                                                  2.4 X           adjusted EMS dosage. <br /> mates total number ofessential genes                                                                  HOWELLand ROSE (1990) noted estimate LGV(1eft) total number C. <br /> elegans genome. the89 identified genes                   based small sample sizeand alow estimate                                                                  total number genes argued estimate <br /> determined m  mean number mutations                                                                  high. estimated average forward <br /> essential gene). Knowing f  fraction genes <br />                                                                  mutation rate peressential gene hDf6 region represented allele) 47/89,                                                                  LGI. estimate     4.3    X        got m value 1.35 hits gene graph m  f wheref = (1 - e-m - m)/l - e-m.                                                                  adjusted EMS dosage. CLARK&apos;S(1990) esti- <br /> calculated 1   -   e-&quot;&apos;  frequency geneswith           mated adjusted rate sDf2 region LGIV                                                                  5.8 x          Ourrate similar mutation) approximately 0.74, giving minimum estimate 120 essential            genes          CLARK&apos;S;   half timesthat HOWELL <br /> LGV(1eft).                                                       ROSE fold lower    estimate number essentialgenes C.            BRENNER.     support HOWELLand ROSE&apos;S(1990) <br /> elegans genome number essential genes                argument BRENNER&apos;S       (1974) gene number esti- <br /> identified LGV(left), know fraction             mate low. total number genes LGV(left). EDGLEY               determined frequency temperature sen- RIDDLE (1990) list 830 genes C . elegans              sitive mutations EMS-induced mutations. <br /> genome, 244 lets (essential genes) come             tested 190 recessivelethal mutations devel- screens specific regions  LGI, ROSE              opment blocking stageat 15O . Worms containing BAILLIE    (1980), HOWELL      et al. (1987)         (2.1%) mutations fertile worms ELL ROSE(1990); LGII: SIGURDSON,           SPANIER       mutations blocked development later <br /> HERMAN(1984); LGIV: ROGALSKI,               MOERMANand           stage 20&quot;. seven (3.7%) BAILLIE   (1982), ROGALSKI      BAILLIE  (1985), CLARK       partiallyheat sensitive. SUZUKI et al. (1967) et al. (1988) ROGALSKI       RIDDLE( 988); LGV:        similar level (6.3%) EMS-induced lethal mu- <br /> ROSENBLUTH       et al. (1988), JOHNSEN BAILLIE              tations X chromosome D. melanogaster (1 988) CLARK      et al. (1990); LGX: MENEELY           temperature sensitive. 177 mutations HERMAN      (1979, 198l)J. Ignoring lets           permissive temperature 25&quot;,                                       Analysis      Genetic           LGV(1eft)                                                 749 <br />  <br />   50 <br />        Number nene8                                                   identified 17 genes strict maternal effect muta- <br />                                                                          tions divided classes: 13 genes <br />    40 <br />                                                                          ortwo alleles                                                                          alleles. smaller class forward <br />    30 <br />                                                                          mutation    rate comparable BRENNER&apos;S           (1 974)rate, <br />                                                                          considered smaller class pure ma- <br />    20 <br />                                                                          ternal genes. led themtoanestimate                                                                                          number (12 pure maternal genes C. elegans <br />     10 <br />                                                                          genome lower  <br />                                                                             interestingthat halfof essential genes <br />      0 <br />                                                                          appear    needed strictly early-larval develop- <br />          Egg    Early   Mid      Late   Sterile   MEL       Slow         ment. Thismay amisleading result reasons: <br />                            Blocking stage <br />                                                                          (1 products genes  <br />    FIGURE3.-Plot number genes whosemostsevere <br />                                                                          quired earlier maternally supplied <br /> alleles block development indicated developmental stages. stages areegg lethal, early- mid- late-larval <br />                                                 lethal, sterile adults,  (2 early-larval blocking stage bedue MEL (maternal effect lethal) slow developing fertile.            metabolic demands rapid growth cell divi- <br />                                                                          sion moulting, e., sickly worms live blocked development later stage 25&quot;                      awhile make moult, 20&quot; genes LGV(1eft) ts alleles                    genes needed strictly moulting wild type. <br /> emb-29 (zone 4B) (CASSADAet al. 198           1) rol-3 (zone-        decreasing number genes necessary develop <br />  18). let-327 (zone 6A) cold sensitive dominant                    mid- late-larval stages reasonable, <br /> allele ( ~ 2 4 7 ) .                                                     probably representatives relatively small <br />     Figure 3 shows number genes identified                     class genes involved specifics later <br /> LGV(1eft) canonical alleles block development <br />                                                                          moults. relatively large numberof genes required indicated stages. assumes LGV(1eft) <br />                                                                          past mid-larval stage relatedto representative genome distri- <br />                                                                          fact developmentalchoice  bution phenotypes shown Figure 3 reflects                                                                          conditions starvation overcrowding C. elegans <br /> proportion genes needed different develop- <br />                                                                          nematodes enter dispersive larval stage called <br /> mental stages, thentheproportionandminimum <br />                                                                          dauer (GOLDENand RIDDLE1982, 1984;                   OHBAand <br /> number genes needed stage                       cal- <br /> culated.Fromtheseassumptions,                   wild-type genes          ISHIBASHI    1982).      class dauer-constitutive <br /> needed zygote normal egg development                          mutations cause worms locked consist 8% (230)      essential genes.              blocked    dauer development pathway. early-larval development requires 50%                      (1425) worms       grow longer length      essential genes. T o past mid-larval                  assigned     mid-larval        called mid-larval <br /> stage requires 17% (485 essential genes. T o                      lethal mutations. T h e large number genes needed past late-larval development requires 2% (60)                     strictly  make fertile animals probably reflects essential genes. 16% (455)          essential genes           complexity     gonad. needed make fertilized eggs. include                       Generally,    genes allele, genes needed formation development                        developmental       blocking stages approximately viable oocytes viable sperm. estimate                      suggesting      thatmost lethal mutations number genes necessary fortheformationand                             nulls.     consistent   findings described development ofviable sperm 60. 36 spe                          (SHANNON     et al. 1972;    ROSEand BAILLIE1980; <br /> genes identified(L&apos;HERNAULT,               SHAKESand           MENEELY         HERMAN       198   1; HOWELLand ROSE <br /> WARD1988 sources cited C. elegans                     1990;CLARK1990) CLARKfound somewhat <br /> genome. L&apos;HERNAULT,SHAKESand WARD used                                 variability. later arresting <br /> truncated Poisson equation calculate                   mutations blocked earlier hemizygotes homo- minimum 14 spe genes LGI (42m.u.). LGI                        zygotes, indicating hypomorphs.  representative 300 m.u. genome,                    cause lethal mutations appear cause null phe- estimate minimum 100 spe genes                       notypes, genes alleles different phe- genome. estimate (based genes)                      notypes aregood candidatesfor complex loci. minimum 60 spe genes similar estimate.                  example Zin-40 (zone SAl) variation appears spe genes randomly                         developmental blocking stages appear beconsistent <br /> located C . elegans genome. 5% (140)                    intragenic complementation map   <br /> ofthe essential genes neededtomake                         viable    published results). cases did note progeny, me1 genes 2%                    phenotypes mutations overdeficiencies, (60) purelymaternal. KEMPHUESet al. (1988)                           did anomalouslate blocking alleles <br />  750                              C.      R.   Johnsen D. L. Baillie <br />  temperature sensitive, possibly indicating       structure analysis DNA sequencing deter- hypomorphs.                                intragenic recombination high <br />    Information regarding biological function   genes.  consider genes [let461 (zone 7), let-341 (zone YB), rol-3 (zone    ability reshuffle domains ofallelesofthese <br />  18), unc-60 (zone 4A), unc-62 (zone 9B) unc-70       genescouldbeevolutionarily            important. (zone 14)] obtained virtue map posi-      analysis  genes regions high <br /> tions. lethal mutations allelic    recombination, lead insights nature mutations mapped positions        nonrandom recombinational distribution researchers, described terms phenotypes         genes noted C. elegans BRENNER      (1974). lethality. unc (uncoordinated) genes          Interestingly, let-347 (zone 6B) lin-40 (zone <br /> generally involvedinmuscle nervous          8A1) gene sparse region highly subject controlling muscles. null phenotype         mutated mobilization Tcl transposable identified muscle genes early-larval lethality.   element (CLARK     et al. 1990; JOHNSEN 1990). tet-75 (gene encoding myo-1) early-larval lethal      shown &#226;&#8364;&#339;hotspot&#226;&#8364;? T c l induced pu- <br /> recessive mutations (ROSEand BAILLIE1980).           tative point mutations lin-40 &#226;&#8364;&#339;hotregion&#226;&#8364;? dominant missense alleles unc-54 recessive      T c l induced deficiencies near lin-40. seven  <br /> lethal phenotype (MACLEOD       et al. 1977; DIBBet al.   ficiencies CLARKet al. (1990) identified pu- <br />  1985; BEJSOVEC   ANDERSON       1988). WATERSTON     tative tip deletions delete left (1989) showed minor myosinheavy chain            known gene chromosome. tip deficiencies <br /> (mhcA) alsohasrecessivelethal       mutations. WATER-     explained mobilization resident <br /> STON demonstrated worms homozygous          Tcl elements, excision followed repair <br /> mutations progress folds         chromosomal breaks. ENGELSet al. (1990) pro- <br /> egg  worm elongates egg fold     posed homolog-dependent model P element loss <br /> accommodate length, wild-types progress        Drosophila. modelthey proposed P folds). wormswith mhcA recessive lethal         transposition created double strand break mutations hatched remained folded state     repaired using homolog chromosome locomotion. unc-70 recessive lethal mutations      template. similar mechanismcould explain high <br /> cause similar phenotype implying unc-70       incidence tip deficiencies eT1 translocation <br /> gene involved muscle development.                      region C. elegans homolog available <br />    mapping large numbers essential genes        effect repair. T c l induced mutations picked deficiencies alsoof withrespect        translocation ( n T ) acts crossover <br /> genomic organization chromosomal behavior.            suppressor. translocations thought map positions genes deline-       pair meiosis  elegans (ROSENBLUTH      BAILLIE <br /> ated regions recombinationally &#226;&#8364;&#339;clustered&#226;&#8364;? genes       1981).  thehomolog wouldnot available gene &#226;&#8364;&#339;sparse&#226;&#8364;?regions. gene cluster          T c l excisionwould accom- <br /> center LGV  e., right end LGV(left)    panied tip deficiency. propose Tcl evident map positions non-essen-      lin-40 induced mutations arise excision tial genes(BRENNER   1974; EDCLEY     RIDDLE1990).    subsequent insertion local Tcl element. second smallercluster evident theleft end     method test hypothesiswouldbe isolate <br /> (zones 1-6A), gene sparse region, let-347 (zone     mobilization ofT c l induced recessive lethalmutations <br /> 6B) lin-40 (zone SAl), lies betweenthe clusters    chromosome region presence (Figure 1). Comparing deficiency sizes cluster     absence ofa translocation. substantially higher ratio gene sparse region, leads argue   putative tip deficiencies putative point mutations difference gene clustered gene       screen translocation support sparse regions reflects difference recombination     hypothesis. <br /> rates unit length chromosome            conclusion, approximately 70% essential <br /> difference physical gene spacings. thelet-347   genes LGV(left) identified. (zone 6B) lin-40 (zone 8A1) region ap-       largest characterized genomic region proximately 6 m.u.withsixgenes(based              higher eukaryote. HOCHMAN          (1970) analyzed chro- <br /> knowngenes andthe fact thatthe region 70%              mosome 4 D. melanogaster identified 33 essen- <br /> saturated essentialgenes) C. elegans        tial loci estimated approximately 2850 essential genes 300 m.u., region       40 total. attempted saturate regions recombination rate atleast 10 times greater    D.melanogaster, recent example 17 polytene average genome rate. interesting note   bands 73A2-73B7 BELOTEet al. (1990). putative complex genes, let-330 (zone 7) lin-     identified 18 complementation groups including 13 <br /> 40 (zone 8Al), liewithin region. Genetic tine        previously identified. cornplex- <br />                                                       Genetic Analysis LGV(1eft)                                                     751 <br />  <br /> ity higher organisms makes task map-                         Complex Genomes, Ed. 5, edited S. J. O&#226;&#8364;&#8482;BRIEN.Cold Spring <br />                                                                             Harbor Laboratory, Cold Spring Harbor, N.Y. <br /> ping essential genes formidable. ease                  ENGELS,  W. R., D. M. JOHNSON-SCHLITZ,      W. B. EGGLESTON    J. maintenance C. elegans rapidity                        SVED,1990 High-frequency P element loss Drosophila genetic analysis makesit feasible saturate entire                    homolog-dependent. Cell 6 2 515-525. <br /> genome essential genes. mapping char-                       GOLDEN,  J. W., D. L. RIDDLE,1982 pheromone influences <br /> acterization majority essential genes                       larval development nematode Caenorhabditiselegans. <br />                                                                             Science 218: 578-580. <br /> provide C. elegans researchers collectionof                      GOLDEN,  J. W., D. L. RIDDLE,1984 Caenorhabditis elegans <br /> identified genes quickly used study                         dauer larval: developmental effects pheromone, food, various developmental pathways.                                         temperature. Dev. Biol. 102: 368-378. <br />                                                                         GREENWALD,     . ,  COULSON,  J. SUISTONand J. PRIESS,1987 <br />    like thank R. E. ROSENBLUTH,       M. ROSEand K.           Correlation physical genetic mapsin lin-12 <br /> S. MCKIM helpful discussions preparation manu-           region Caenorhabditis elegans. Nucleic Acids Res. 15: 2295- <br /> script, H. STEWART  maintaining stocks technical assistance,        2307. M. ROGAMKI    D. RANLY   technical assistance. Supported    HADORN,   E., 1945 Zur Pleiotropie der Genwirkung. Arch. Julius Medical Research Council Canada Studentship R. C. J.             Klaus-Stift. Erganzungsb. Band 20: 82-95. grant fromthe Natural Sciences Engineering Council          HADORN,  E., 196 1 Developmental Genetics Lethal Factors (trans- Canada D. L. B. Maintenance C . elegans strains        lated U. MITTWOCH).      Metheun &amp;  , London, Eng- <br /> Laboratory funded grant Muscular Dystrophy                 land. <br /> Association Canada D. L. B. nematode strains used         HAN,M., P. W. STERNBERG,       1990 let-60, gene specifies work provided Caenorhabditis Genetics Center,              cell fates C. elegans vulval induction, encodes ras funded National Institutes Health National Center        protein. Cell 63: 92 1-93 1 . Research Resources (NCRR).                                           HOCHMAN,    B., 1970 Analysis chromosome 4 Drosophila <br />                                                                             melanogaster. 11. Ethyl methanesulfonate induced lethals. Ge- <br />                    LITERATURE CITED                                         netics 67: 235-252. <br />                                                                          HORVITZ,H. R., S. BRENNER,       J. HODCKINand R. K. HERMAN, <br /> BEJSOVEC,  , P. ANDERSON,     1988 Myosinheavy chain mu-               1979 uniform genetic nomenclature forthe nematode <br />    tations disrupt C. elegans filament assembly. Genes           Caenorhabditis elegans. Mol. Gen. Genet. 175: 129-133. <br />    Dev. 2: 1307-1317.                                                    HOWELL M.,  M. ROSE, 1990 Essential genes BELOTE, J. M., F. M. HOFFMAN,    M. MCKEOWN,     R. L. CHORSKY          hDf6 region chromosome Caenorhabditiselegans. Ge- <br />    B. S. BAKER,1990 Cytogenetic analysis chromosome                   netics 126: 583-592. <br />    gion 73AD Drosophila melanogaster. Genetics 125: 783-793.          HOWELL,   M., S. G. GILMOUR,     R.  MANCEBOand  M. ROSE, <br /> BENIAN, G. M., J. E. KIFF, N. NECKELMANN,      D. G. MOERMANand              1987 Genetic analysis large autosomal region Caenor- <br />    R. H. WATERSTON,        1989 Sequence unusually large              habditis elegans use free duplication. Genet. Res. 4 9 <br />    protein implicated regulation myosin activity C. elegans.       207-2 13. <br />    Nature 3 4 2 45-50.                                                  JOHNSEN, R. C., 1990 Genetic analysis left half linkage <br /> BRENNER,   S., 1974 genetics Caenorhabditiselegans. Ge-              group V Caenorhabditis elegans. Ph.D. thesis, Simon Fraser <br />    netics 77: 71-94.                                                        University, Burnaby B.C., Canada. <br /> BRO <br /> </body></html>