<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>      Rct1, Nuclear RNA Recognition Motif-Containing Cyclophilin, <br />            Regulates Phosphorylation RNA Polymerase II <br />                          C-Terminal Domain <br />   <br />              Phosphorylation C-terminal domain (CTD) RNA polymerase II (RNAP II) dynamic process <br />           regulates transcription coordinates pre-mRNA processing. Rct1,           nuclear multidomain cyclophilin Schizosaccharomyces pombe, encoded essential gene           interacts CTD regulates phosphorylation vivo. Downregulation Rct1 levels results           increased phosphorylation CTD Ser2 Ser5 commensurate decrease RNAP <br />           II transcription. contrast, overexpression Rct1 decreases phosphorylation sites. close <br />           association Rct1 transcriptionally active chromatin suggests role regulation RNAP II <br />           transcriptional activity. data, pleiotropic phenotype Rct1 deregulation, <br />           suggest multidomain cyclophilin important player maintaining correct phosphory- <br />           lation code CTD regulating CTD function. <br />  <br />  <br />    Eukaryotic RNA polymerase II (RNAP II), enzyme                           associates phosphorylation-dependent manner directly <br /> responsible transcription protein-coding genes,                     indirectly variety pre-mRNA-processing factors, <br /> multisubunit complex. accurate transcription, RNAP II                       including 5&#226;&#172;&#732; capping enzymes, splicing factors, controlled factors protein-protein inter-                    components 3&#226;&#172;&#732; end-processing machinery cycle (1, 17, <br /> actions. RNAP II transcripts undergo extensive processing,                      35, 37&#226;&#8364;&#8220;39, 41, 46, 53). <br /> including capping, splicing, polyadenylation,                        Recently, cis-trans isomerization prolyl <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> bonds <br /> tightly coordinated transcription. strong                     heptapeptide repeats revealed addi- <br /> evidence RNAP II coordinates processing reac-                        tional level regulation CTD structure function, tions couples transcription. Specifically,                      shown CTD-interacting proteins bind <br /> C-terminal domain (CTD) largest RNAP II subunit                          CTD trans proline conformation (10, 31, critical role targeting pre-mRNA processing factors                    32, 34, 48). proteins parvulin-type pep- emerging pre-mRNAs.  CTD coordinates tran-                          tidyl-prolyl cis-trans isomerase (PPIase) Pin1, pre- <br /> scription pre-mRNA processing sub-                        viously shown important keeping correct <br /> ject tight regulation transcription cycle (1, 17,                 CTD phosphorylation status mouse human cells (51, <br /> 35, 37, 39, 41, 46, 53).                                                        52). addition, mutation ESS1, Saccharomyces cerevi- <br />    eukaryotes, CTD consists variable numbers                   siae homolog Pin1, resulted defects pre-mRNA 3&#226;&#172;&#732; <br /> YSPTSPS heptapeptide repeats. transcription                          end formation (14). data suggest enzymes cata- <br /> cycle, CTD undergoes dynamic phosphorylation                                                                                 lyzing cis-trans isomerization proline peptide bonds dephosphorylation Ser2 Ser5. Transcription initia- <br />                                                                                 important correct CTD conformation phosphor- <br /> tion requires unphosphorylated CTD. Following initia- <br />                                                                                 ylation consequently recognition diverse pre- <br /> tion, CTD phosphorylated Ser5, modifica- <br />                                                                                 mRNA-processing factors.  generation CTD <br /> tion necessary initiate transcript elongation. <br />                                                                                 code requires phosphorylation cis-trans <br /> Finally, extensive phosphorylation Ser2 accompanied <br />                                                                                 isomerization prolines follow Ser2 Ser5 transcript elongation termination. addition                                                                                 heptapeptide repeats (3, 24, 32, 34). <br /> regulating RNAP II transcription, CTD phosphorylation <br />                                                                                    majority cyclophilins (PPIases) small proteins <br /> status important coordination transcription pre-mRNA processing, particular recruitment                      containing PPIase domain 120 amino acids. <br /> RNA-processing factors nascent transcripts. CTD                           multidomain cyclophilins different  <br />                                                                                 ganisms described (36, 42). com- <br />                                                                                 plex multidomain cyclophilins characterized far    * Corresponding author. Mailing address: Max F. Perutz Laborato-             Arabidopsis AtCyp59 (12) orthologue Paramecium <br /> ries, Medical University Vienna, Department Medical Biochem-              tetraurelia, Kin241p (22). characterized unique <br /> istry, Bohrgasse 9/3, 1030 Vienna, Austria. Phone: 43 1 4277 61642.           domain organization, consisting PPIase domain N <br /> Fax: 43 1 4277 9616. E-mail: zdravko.lorkovic@univie.ac  <br />                                                                                 terminus, followed RNA recognition motif (RRM)    &#226;&#8364;&#160; Supplemental material article http://mcb <br /> .asm.org/.                                                                      C-terminal domain enriched charged amino acids    &#228;&#338;&#164; <br />      Published ahead print 5 March 2007.                                  serines RS/RD dipeptide repeats. Homologous proteins <br />  <br />                                                                          3601 <br />  3602       GULLEROVA ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  majority eukaryotes, Sac-                          manipulator. Spores grown yeast extract-supplement (YES) medium charomyces cerevisiae Saccharomycotina spe-                            4 days 32&#194;&#176;C. <br />                                                                                          pMG derived pREP3X (11) inserting kanamycin resistance <br /> cies encode kind protein (12, 22). Kin241p                         cassette, cut pSMRG2&#226;&#171;&#185; (25), SacI site. plasmid expressing <br /> identified Paramecium tetraurelia protein involved                         Rct1 fused hemagglutinin (HA) tag constructed cloning rct1 gene, <br /> cell morphogenesis (reference 22 references .                             amplified PCR oligonucleotides 5&#226;&#172;&#732;-GTCAGTCTCGAGATGTCTG <br /> AtCyp59 nuclear protein identified yeast                            TACTAATTGAA-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;-GTCAGTGTCGACTCATGCGTAGTC <br />                                                                                       AGGCACATCATACGGATATCGATATCTATCATCTCTATAACG-3&#226;&#172;&#732; (reverse). hybrid screen interacting partner SR proteins,                                                                                       PCR fragment ligated pMG opened XhoI/SalI, resulting important family splicing regulators (43). inter-                       pMG1. <br /> acted CTD RNAP II, function protein                            generate different cell strains rct1-HA gene expressed interface transcription pre-mRNA splicing                             plasmid, WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells transformed pMG1. haploid proposed (12). Interestingly, Arabidopsis cells ectopically                       rct1&#226;&#338;&#172;pMG1 strain (h&#226;&#171;&#186; leu&#226;&#171;&#185; ura&#226;&#171;&#185; ade6-704) obtained rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells <br />                                                                                       tetrad dissection. <br /> expressing AtCyp59 characterized slow growth,                                Conditions PCR genotyping. Loopfuls freshly growing cells trans- <br /> presumably decreased phosphorylation RNAP                             ferred Eppendorf tubes boiled microwave 3 min. Cells II CTD (12). Arabidopsis mutant gene encoding                            resuspended 50 &#226;&#65533;&#174;l sterile water used template PCR AtCyp59 exists, homologous protein present S.                         following conditions: 95&#194;&#176;C 3 min; 30 cycles 94&#194;&#176;C 45 s, 55&#194;&#176;C 45 s,                                                                                       72&#194;&#176;C 1 min; 72&#194;&#176;C 5 min. presence WT rct1 cerevisiae, switched alternative genetically treatable <br />                                                                                       disrupted rct1&#226;&#338;&#172;::ClonNat alleles checked PCR using oligonucleo- <br /> organism, Schizosaccharomyces pombe. S. pombe homolog                             tide pairs 5&#226;&#172;&#732;-GAGGCAGAAGCAGAGGCTGTTACA-3&#226;&#172;&#732; (forward)&#226;&#8364;&#8220;5&#226;&#172;&#732;-AACGTG 49% identical 66% similar AtCyp59,                            CCGCATTTATGGAG-3&#226;&#172;&#732; (reverse) 5&#226;&#172;&#732;-GCGTGGGGACAATTCAACGC-3&#226;&#172;&#732; <br /> domain organization AtCyp59  Fig. 1A) Zn                          (forward)&#226;&#8364;&#8220;5&#226;&#172;&#732;-AACGTGCCGCATTTATGGAG-3&#226;&#172;&#732; (reverse), respectively. knuckle located downstream RRM plant                           presence pMG1Rct1-HA construct cells analyzed PCR using <br />                                                                                       oligonucleotides 5&#226;&#172;&#732;-GGAATCCGATTGTCATTCGGC-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;- <br /> homologs (12). sequence conservation suggests                             CCCGGGGATCCTCTAGAGTC-3&#226;&#172;&#732; (reverse). <br /> highly conserved function.                                                               Nuclear transcription run analysis. Gene Screen Plus membrane (Perkin- <br />     Rct1  &#226;&#8364;&#339;RRM-containing cyclophilin                           Elmer) soaked 1&#226;&#171;&#187; SSC (1&#226;&#171;&#187; SSC 0.15 M NaCl plus 0.015 M sodium <br /> regulating transcription&#226;&#8364;&#65533;), S. pombe ortholog Arabidop-                        citrate) fixed MINIfold slot blot apparatus (Schleicher Schuell)                                                                                       apply DNA. micrograms double-stranded DNA prepared PCR sis AtCyp59, encoded essential gene. Reduced levels                                                                                       denatured applied slot. DNA fixed membranes UV cross- <br /> Rct1 rct1 heterozygous cells result strong growth                           linking Stratalinker 2400 (Stratagene). Membranes prehybridized morphological defects impact meiotic differ-                           hybridization buffer (10 mM Tris-HCl [pH 7.5], 250 &#226;&#65533;&#174;g tRNA, 10&#226;&#171;&#187; Denhardt&#226;&#8364;&#8482;s <br /> entiation. Furthermore, phosphorylation CTD                              solution [0.2% bovine serum albumin, 0.2% polyvinylpyrrolidone {molecular <br /> creased ongoing transcription RNAP II reduced.                              weight, 40,000}, 0.2% Ficoll 400], 0.5% nonfat dry milk, 0.3 M NaCl, 1% sodium <br />                                                                                       dodecyl sulfate [SDS], 10 mM EDTA) 2 h 60&#194;&#176;C. S. pombe WT Rct1&#226;&#171;&#185;/&#226;&#171;&#186; observed phenotypes rescued complemen-                             cells grown YES medium reaching exponential phase. optical <br /> tation assays. addition, higher levels Rct1 resulted                         density 600 nm (OD600) measured, cells decreased phosphorylation CTD reduced                             harvested centrifugation 4,000 rpm 5 min room temperature. Pellets <br /> growth. close association active transcription com-                      washed 1 ml ice-cold TMN buffer (10 mM Tris-acetate [pH 7.0], 12 <br />                                                                                       mM MgCl2, 50 mM NH4Cl) resuspended 0.5 ml TMN buffer plex places essential protein central position regu- <br />                                                                                       equilibrated 10 min ice. TMN buffer removed, pellets lating CTD phosphorylation transcription cy-                           resuspended 950 &#226;&#65533;&#174;l cold water 50 &#226;&#65533;&#174;l 10% sodium sarcosyl cle. recent structural data CTD-interacting                          incubated ice additional 20 min. Detergent removed, perme- <br /> proteins, data emphasize importance PPIases                         abilized cells resuspended 0.5 ml run buffer (50 mM Tris-HCl [pH <br /> formation CTD code.                                                            7.9], 80 mM MgCl2, 500 mM KCl, 1 mM dithiothreitol [DTT], 1 mM <span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>, 0.5 <br />                                                                                       mM <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span>, 0.5 mM <span id='am-30' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span>CTP</span>, 100 units RNasin, 100 &#226;&#65533;&#174;Ci [32P]<span id='am-25' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span>UTP</span>).                                                                                       protocol used http://www.hsph.harvard.edu/wolflab/Protocols <br />                                                                                       /Protocols/Fission%20Yeast/Nurse%20Lab%20Manual.htm. Cells incu- <br />                         MATERIALS METHODS <br />                                                                                       bated 30&#194;&#176;C 10 min washed TMN buffer, total RNA    S. pombe strains handling cells. diploid S. pombe strain P136           isolated using TRIzol reagent (Invitrogen). Isolated RNA dissolved 100 <br /> (h&#226;&#171;&#185;/h&#226;&#171;&#186; leu1-32/leu&#226;&#171;&#185; ura4-27/ura&#226;&#171;&#185; ade6-704/ade&#226;&#171;&#185;) used wild-type             &#226;&#65533;&#174;l Tris-EDTA (TE), denatured 3 min 95&#194;&#176;C, added prehy- <br /> (WT) strain experiments. stated  WT strains (WT,         bridized membrane 3 ml hybridization buffer. Hybridization performed <br /> WTpMG WTpMG1) used diploid. Media, growth conditions, standard            60&#194;&#176;C 2 days. Membranes washed twice 2&#226;&#171;&#187; SSC&#226;&#8364;&#8220;1% SDS genetic methods used described previously (33; http://www-rcf      10 min 60&#194;&#176;C exposed MR Kodak films. addition, blots analyzed <br /> .usc.edu/&#226;&#172;&#402;forsburg/). Geneticin (G418; Sigma) ClonNat (BioAgents)            PhosphorImaging quantified ImageQuant, version 5.1. list genes <br /> used final concentrations 155 100 &#226;&#65533;&#174;g/ml, respectively. nmt1 pro-        analyzed nuclear transcription run oligonucleotides used amplifi- <br /> moter repressed addition thiamine (final concentration, 100 &#226;&#65533;&#174;g/ml).     cation corresponding DNA fragments, reverse transcriptase PCR <br />    Construction rct1 knockout allele plasmids generation rct1&#226;&#180;&#8482;/&#226;&#180;&#8482;pMG1,   (RT-PCR)  , available Table S1 supplemental material. <br /> rct1&#226;&#180;&#8482;/&#226;&#180;&#353;pMG1, rct1&#226;&#338;&#172;pMG1 strains. Fragments corresponding 5&#226;&#172;&#732; 3&#226;&#172;&#732;          Fragment sizes 400 500 nucleotides (nt) originated middle <br /> untranslated regions (UTR) rct1 gene amplified PCR. 5&#226;&#172;&#732;         coding region. case U1 snRNA, U6 snRNA, tRNASer, amplified <br /> UTR, oligonucleotides 5&#226;&#172;&#732;-GACTAGGGATCCTCTAGAAACGAATGTATCA                              DNA fragments cover entire sequences mature RNAs. <br /> CATTGC-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;-GACTAGAAGCTTTTTCAATTAGTACAGAC                                 RNA isolation, RT-PCR, real-time PCR. Total cellular RNA S. <br /> ATG-3&#226;&#172;&#732; (reverse), introduce BamHI HindIII site, respectively,           pombe cells exponential phase isolated TRIzol (Gibco). RNA used. 3&#226;&#172;&#732; UTR, oligonucleotides 5&#226;&#172;&#732;-GACTAGCTGCAGCATCGT                     dissolved proper sterile water treated RNase-free <br /> GACGATCAAAGTTCA-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;-GACTAGTCTAGACAAGTAT                               DNase (Promega) 30 min 37&#194;&#176;C. microgram total RNA used AAGCAATGTAGATTTC-3&#226;&#172;&#732; (reverse), introduce PstI XbaI site,               reverse transcription, performed according manufacturer&#226;&#8364;&#8482;s <br /> respectively, used. PCR fragments cloned pClonNat1.                instructions (Reverse Transcription  Promega) using oligo(dT)15 <br /> pClonNat15&#226;&#172;&#732;UTR3&#226;&#172;&#732;UTR construct linearized XbaI trans-                primer. cDNA diluted 100 &#226;&#65533;&#174;l TE buffer, 10 &#226;&#65533;&#174;l used PCR. <br /> formed WT cells lithium acetate method. Correct integration            Conditions PCRs 1 cycle 95&#194;&#176;C 3 min; 30 cycles 94&#194;&#176;C 1 min, <br /> pClonNat15&#226;&#172;&#732;UTR3&#226;&#172;&#732;UTR rct1 gene analyzed Southern blotting.             50&#194;&#176;C 1 min, 72&#194;&#176;C 1 min; 1 cycle 72&#194;&#176;C 5 min. Oligonucleo- <br /> Heterozygous rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells grown malt extract sporulation medium              tides used amplification corresponding DNA fragments (genes 1 10 plates 3 days 26&#194;&#176;C, tetrads dissected Singer micro-            Fig. 5B) available Table S1 supplemental material. Conditions  VOL. 27, 2007                                                         CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                                       3603 <br />  <br /> RT-PCR amplification meiotically induced genes exactly described          PLRSPSPTPEQC-COOH) P2 (H2N-CERRYRYDRRYRDDRYR-COOH) <br /> previously (2). Oligonucleotides used amplification meiosis-unrelated         (Eurogentec). Antibodies affinity purified SulfoLink (Pierce) column <br /> genes nak1, (AF091345), hob1 (AF275637), wsp1 (AF038575) fol-            coupled peptides. purification, antibodies concentrated lows: nak1for, 5&#226;&#172;&#732;-CTGGAAATGTCAAGCTATGTG-3&#226;&#172;&#732;; nak1rev, 5&#226;&#172;&#732;-CCTGCA                       1 &#226;&#65533;&#174;g/&#226;&#65533;&#174;l used 1:1,000 dilutions. Secondary antibodies, goat anti-rat <br /> AACAAGAGGCAATG-3&#226;&#172;&#732;; hob1for, 5&#226;&#172;&#732;-CAAAGCGTGAAGAAGCAGCA-3&#226;&#172;&#732;;                              (Sigma-Aldrich) goat anti-mouse (Bio-Rad) immunoglobulin G (IgG) hob1rev, 5&#226;&#172;&#732;-ACCTTGTGCACCGTTAAGTC-3&#226;&#172;&#732;; wsp1for, 5&#226;&#172;&#732;-CGATAGCTGA                          goat anti-mouse (Sigma-Aldrich) IgM, conjugated horseradish peroxi- <br /> ATTGCCTCAAC-3&#226;&#172;&#732;; wsp1rev, 5&#226;&#172;&#732;-ATCGTCGTCTTCATCCTCTTC-3&#226;&#172;&#732;. Condi-                         dase, used 1:10,000 dilutions. chemiluminescence kit (Amersham <br /> tions PCRs 1 cycle 95&#194;&#176;C 3 min; 30 cycles 94&#194;&#176;C 1 min, 52&#194;&#176;C     Pharmacia Biotech) used developing blots. 1 min, 72&#194;&#176;C 1 min; 1 cycle 72&#194;&#176;C 5 min. PCR products analyzed 1.5 2% agarose gels. <br />    Real-time PCRs Corbett Research Rotor-Gene GG-3000 <br />                                                                                                                      RESULTS <br /> machine. samples run triplicate ensure accuracy data. PCR 45 cycles QuantiTect SYBR green PCR Master Mix (QIAGEN), 2 &#226;&#65533;&#174;l             Rct1, encoding Schizosaccharomyces pombe ortholog of precipitated DNA, 1 &#226;&#65533;&#174;M oligonucleotides. Cycling parameters 95&#194;&#176;C 15 min followed 45 cycles 94&#194;&#176;C 2 s, 55&#194;&#176;C 11 s, 68&#194;&#176;C 20 s. <br />                                                                                      AtCyp59, essential gene. analyze knockout mutants Oligonucleotides used follows: spo4PROfw, 5&#226;&#172;&#732;-TGGTTATGACAGGTGT                rct1, performed rct1 gene disruption diploid S. pombe <br /> GCTG-3&#226;&#172;&#732;; spo4PROrev, 5&#226;&#172;&#732;-CCATGATCATCATTAAACGG-3&#226;&#172;&#732;; spo4ORFfw, 5&#226;&#172;&#732;-                      strain, confirmed Southern blotting PCR geno- <br /> ATGCTACATGAACTACGAGG-3&#226;&#172;&#732;; spo4ORFrev, 5&#226;&#172;&#732;-GGGTGTAAGGATTCC                              typing (data shown). Integration pClonNat, containing AAGCA-3&#226;&#172;&#732;; spo4TERfw, 5&#226;&#172;&#732;-TTGTATCGATTAACTCTTAA-3&#226;&#172;&#732;; spo4TERrev, 5&#226;&#172;&#732;-                     selectable antibiotic gene, resulted heterozygous cells  <br /> ATGACTTGACGCTACAGGCAT-3&#226;&#172;&#732;; meu13PROfw, 5&#226;&#172;&#732;-CCCTTCGCAAGGTTG <br /> GAAA-3&#226;&#172;&#732;; meu13PROrev, 5&#226;&#172;&#732;-ACTAAGGAAGAAAATCCATG-3&#226;&#172;&#732;; meu13ORFfw, <br />                                                                                      taining rct1&#226;&#338;&#172;::ClonNat rct1 allele. cells 5&#226;&#172;&#732;-GCTTTAAATAACTCACTCAG-3&#226;&#172;&#732;; meu13ORFrev, 5&#226;&#172;&#732;-TTCCAAGGAATCC                            referred heterozygous rct1&#226;&#171;&#185;/&#226;&#171;&#186; short rct1&#226;&#171;&#185;/&#226;&#171;&#186; <br /> GTAA-3&#226;&#172;&#732;; meu13TERfw, 5&#226;&#172;&#732;-GCAGACAACTGCTCTAATAT-3&#226;&#172;&#732;; meu13TERrev,                        cells. Tetrad dissection rct1&#226;&#171;&#185;/&#226;&#171;&#186; heterozygous diploid cells <br /> 5&#226;&#172;&#732;-TCATATCCGTTCCGAAATT-3&#226;&#172;&#732;; mfr1PROfw, 5&#226;&#172;&#732;-GTGGTCAACTGATCAT                            resulted growing spores (Fig. 1B). PCR genotyping <br /> TGTT-3&#226;&#172;&#732;; mfr1PROrev, 5&#226;&#172;&#732;-TGAAACCAGGTGGCTTGAAA-3&#226;&#172;&#732;; mfr1ORFfw, 5&#226;&#172;&#732;- <br />                                                                                      revealed presence WT allele growing spores. <br /> TAATGTCTTGGCAGTCGGAC-3&#226;&#172;&#732;; mfr1ORFrev, 5&#226;&#172;&#732;-TTTCGTAGACTCGATA <br /> TCCC-3&#226;&#172;&#732;; mfr1TERfw, 5&#226;&#172;&#732;-CAATCATCTCAACCCCATCA-3&#226;&#172;&#732;; mfr1TERrev, 5&#226;&#172;&#732;                     addition, viable spores resistant ClonNat, <br /> ATACAGCCTCAAGCCTGAA-3&#226;&#172;&#732;; mei2PROfw, 5&#226;&#172;&#732;-CTCCTCCTCTAACGTTTG                             indicating WT rct1 essen- <br /> TT-3&#226;&#172;&#732;; mei2PROrev, 5&#226;&#172;&#732;-ACAATAAAGCTGGCCGATAA-3&#226;&#172;&#732;; mei2ORFfw, 5&#226;&#172;&#732;-CC                      tial gene. Microscopic analysis nongrowing spores showed <br /> GCTAAATCACTGCGATCT-3&#226;&#172;&#732;; mei2ORFrev, 5&#226;&#172;&#732;-GTGGACCGAACGTCTGA                              spores deleted rct1 germinate cell division stops <br /> AGT-3&#226;&#172;&#732;; mei2TERfw, 5&#226;&#172;&#732;-CGAAAGAGATCCTGTATTG-3&#226;&#172;&#732;; mei2TERrev, 5&#226;&#172;&#732;-GA <br /> TACTCAGAGTGAAGTTGA-3&#226;&#172;&#732;; act1PROfw, 5&#226;&#172;&#732;-GGTGGTATGAAGCCGTTG <br />                                                                                      generations. failed grow cells ATTAC-3&#226;&#172;&#732;; act1PROrev, 5&#226;&#172;&#732;-TTCTGCCGTGAAGTGCTA-3&#226;&#172;&#732;; act1ORFfw, 5&#226;&#172;&#732;-TG                     different temperatures liquid medium (data shown). <br /> GGAACAGTGTGGGTAACA-3&#226;&#172;&#732;; act1ORFrev, 5&#226;&#172;&#732;-AGCACCCTTGCTTGTTGA                              results indicate rct1 disruption lethal <br /> CT-3&#226;&#172;&#732;; act1TERfw, 5&#226;&#172;&#732;-CCGGACTCGAGAAGAAACAT-3&#226;&#172;&#732;; act1TERrev, 5&#226;&#172;&#732;-AAC                     temperature sensitive. <br /> CACCTTTTTCCGCTCTT-3&#226;&#172;&#732;. Fluorescence intensities plotted                                                                                         cellular localization Rct1 analyzed indirect <br /> number cycles using algorithm provided manufacturer. <br />    Cell wall actin staining, immunofluorescence, microscopy. Cell wall       immunofluorescence rct1&#226;&#171;&#185;/&#226;&#171;&#186; rct1&#226;&#338;&#172; strains harboring growth analyzed staining living cells aniline blue. Actin staining   plasmid expressing Rct1 fused HA tag (pMG1; . <br /> performed described (5, 6) using Alexa Fluor 488-phalloidin (Molecular         Cells grown 18 h minimal medium (EMM) Probes). Immunofluorescence performed exactly described previously            derepress nmt1 promoter driving transcription Rct1-HA <br /> (40). Rat anti-HA monoclonal antibody (3F10; Roche) used 1:200 <br /> dilution. Secondary antibody goat anti-rat antibody&#226;&#8364;&#8220;Alexa Fluor 563 (Mo- <br />                                                                                      plasmid, followed fixation indirect immuno- <br /> lecular Probes) 1:200 dilution. DNA visualized Hoechst 33342             fluorescence anti-HA antibody. shown Fig. 1C <br /> (Molecular Probes) staining. stated  cells grown         (shown rct1&#226;&#338;&#172;pMG1 cells), anti-HA antibody detected <br /> mid-exponential phase YES EMM media analyzed microscopy                Rct1-HA nucleus. Staining fixed cells (Axioplan epifluorescence microscope; Zeiss) differential interference  <br />                                                                                      Hoechst showed Hoechst Alexa 563 signals trast (DIC) optics 63&#226;&#171;&#187; oil objective. Images acquired charge- <br /> coupled-device (CCD) camera processed Adobe Photoshop.              overlapping (Fig. 1C). Analysis mitotic cells revealed    Immunoprecipitation. rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells grown 25 ml EMM             Rct1 enters new nuclei rapidly division, prior 15 h, collected centrifugation, washed ice-cold water. Cells   septum formation (Fig. 1C, lower panels). <br /> resuspended 500 &#226;&#65533;&#174;l PEB400 buffer (50 mM HEPES-KOH [pH 7.9], 400                   Disruption rct1 allele results growth, morpholog- <br /> mM KCl, 2.5 mM MgCl2, 1 mM EDTA, 1 mM DTT, 0.1% Triton X-100) (26), <br />                                                                                      ical, meiotic defects rescued episomal expres- <br /> supplemented EDTA-free protease inhibitor cocktail (Roche). cell <br /> volumes glass beads added, cells vortexed times 15 s.     sion Rct1. observed rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells grow Finally, cells sonicated subsequently centrifuged 4&#194;&#176;C 15 min    slowly WT cells, decided analyze rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells 13,000 rpm Eppendorf centrifuge. Supernatant mixed 500 &#226;&#65533;&#174;l       thoroughly. Microscopic analyses revealed rct1&#226;&#171;&#185;/&#226;&#171;&#186; <br /> PEB KCl, immunoprecipitation anti-HA antibody,          cells shorter slightly thicker WT cells. described previously (26). <br />    Chromatin immunoprecipitation. Chromatin immunoprecipitation experi- <br />                                                                                      phenotype observed cells grown rich medium <br /> ments described previously (47). Precipitated DNA resus-            (YES) plates liquid YES medium (Fig. 1D). Growth <br /> pended 100 &#226;&#65533;&#174;l TE buffer, 10 &#226;&#65533;&#174;l used PCR. Conditions PCRs       analysis YES medium revealed rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells grow  1 cycle 95&#194;&#176;C 3 min; 31 cycles 94&#194;&#176;C 1 min, 52&#194;&#176;C 1 min,     siderably slowly WT cells (Fig. 1E). Growth rates <br /> 72&#194;&#176;C 1 min; 1 cycle 72&#194;&#176;C 5 min. PCR products analyzed                                                                                      EMM affected, extent 1.5 2% agarose gels. Oligonucleotide sequences used PCRs shown Fig. 4A available request.                                                  YES (data shown). growth rate unrelated <br />    SDS-PAGE Western blotting. Proteins separated SDS-polyacryl-          knockout strain (termed 93) comparable WT <br /> amide gel electrophoresis (PAGE) (10% gels) transferred polyvinylidene        cells, indicating slower growth rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells difluoride membranes (Millipore), followed Western blotting according          presence ClonNat medium. Surprisingly, <br /> standard procedures. Rat anti-HA (3F10) (Roche), mouse antitubulin (Sigma- <br /> Aldrich), mouse anti-CTD monoclonal antibodies (H14, H5, 8WG16;  <br />                                                                                      cells grown overnight minimal medium nitrogen <br /> vance) used 1:1,000 dilutions. Anti-Rct1 polyclonal antibodies          source (EMM-N) strongly sporulated 32&#194;&#176;C (Fig. 1D). <br /> raised rabbits immunized simultaneously peptides, P1 (H2N-DLVE           Sporulation highly enhanced compared WT cells,  3604      GULLEROVA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Analysis rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells.   Schematic representation domain structure Rct1. (B) Tetrad analysis rct1&#226;&#171;&#185;/&#226;&#171;&#186; diploid S. pombe <br /> cells. asci dissected, spores grown YES plates 32&#194;&#176;C 5 days. (C) Rct1 nuclear protein. rct1&#226;&#338;&#172;pMG1 cells grown <br /> overnight EMM processed immunostaining anti-HA antibody. Rct1 localizes nuclei interphase cells (upper panels) dividing cells  lower panels). Hoechst staining performed determine position nuclei, DIC images acquired determine cell shapes. (D) Microscopic analysis WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells grown overnight liquid YES EMM media cells grown 2 days YES plates. Images acquired CCD camera using DIC optics 63&#226;&#171;&#187; oil objective. (E) Growth analysis WT, <br /> rct1&#226;&#171;&#185;/&#226;&#171;&#186;, unrelated S. pombe 93 strains. Cell grown overnight YES medium diluted OD600 0.1 YES medium, OD600 measured 2.5 h. (F) Analysis Rct1 levels WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. Analysis tubulin level (right) used loading <br /> control. <br />  <br />  <br />  <br /> virtually cells sporulated 34 h, sporula-            ated potential overexpressing strain, WTpMG1,  tion WT cells 50% (Fig. 2D data                       control, WTpMG strain transforming WT diploid cells shown).                                                                     plasmids pMG1 (tagged Rct1) pMG  vector),  <br />    WT S. pombe cells rod-shaped (cylindrical),                spectively. Tetrad dissection rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells resulted polarity expressed pattern cell growth,               growing spores. PCR genotyping, growth restricted ends long axis cell (16, 29). WT         presence ClonNat G418, revealed cells grown YES EMM, aniline blue staining                  rct1&#226;&#338;&#172; haploid cells harboring pMG1  rct1&#226;&#338;&#172;pMG1). <br /> actin localization revealed patterns characteristic dividing             nmt1 promoter repressed thiamine present <br /> asynchronous cells; strongest staining cell tips             YES medium  active sites septum synthesis. contrast, rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells           extent, detectable amounts protein produced (Fig. stained asymmetrically cell end               2A). rct1&#226;&#338;&#172;pMG1 cells grown overnight YES EMM <br /> cell tips (aniline blue). addition, rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells, actin          size shape expected haploid WT cells (Fig. cell curved cables running                2C,  Fig. S1 supplemental material), long axis cell curling nucleus            irregularly shaped cells observed (data cell surface  Fig. S1 supplemental material).            shown; . indicates low expression growth morphological changes rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells ob-                Rct1 plasmid YES medium (Fig. 2A) sufficient served presence absence ClonNat, indicating             complement rct1&#226;&#338;&#172; cells. recovered WT spores presence antibiotic.              expressing additional Rct1 plasmid unclear. <br /> fore, compared Rct1 protein levels WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells                  Growth analysis showed rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 rct1&#226;&#338;&#172;pMG1 Rct1 levels considerably lower rct1&#226;&#171;&#185;/&#226;&#171;&#186;                grow approximately rate, somewhat slower cells (Fig. 1F, left), obviously leading haploinsufficiency.             WT faster rct1&#226;&#171;&#185;/&#226;&#171;&#186; (Fig. 2B). addition thiamine <br />    confirm observed phenotypes rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells             (100 &#226;&#65533;&#174;g/ml) significantly reduced growth rct1&#226;&#338;&#172;pMG1 consequence reduced levels Rct1, vivo                 cells,  unexpectedly, did stop completely (Fig. 2B). <br /> complementation performed. plasmid expressing HA-                     Analysis Rct1-HA protein cells revealed tagged Rct1 (pMG1) control thiamine-repressible                presence low levels HA-tagged Rct1 24 h nmt1 promoter (Fig. 2A,  transformed rct1&#226;&#171;&#185;/&#226;&#171;&#186;                   addition thiamine (Fig. 2C, insert left panel). <br /> diploid cells, forming rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 strain. gener-               explains haploid rct1&#226;&#338;&#172;pMG1 cells growing <br />  VOL. 27, 2007                                               CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3605 <br />  <br />  <br />  <br />  <br />    FIG. 2. Episomal expression Rct1 complements rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells.     Schematic presentation plasmid pMG1, expressing HA-tagged <br /> Rct1. nmt1 promoter repressed thiamine active minimal medium (EMM), does contain thiamine. <br />   Expression analysis Rct1-HA rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1, rct1&#226;&#338;&#172;pMG1, WTpMG1 cells grown overnight EMM (left) YES (right) medium. <br /> Western blotting performed anti-HA antibody. lower right panel represents analysis Rct1 levels; band marked asterisk represents antibody cross-reacting peptide. (B) Growth analysis WT, WTpMG, WTpMG1, rct1&#226;&#171;&#185;/&#226;&#171;&#186;, rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1, rct1&#226;&#338;&#172;pMG1 cells liquid YES medium. Cells grown overnight YES medium, diluted OD600 0.1, grown 25 h presence ClonNat G418. Rct1&#226;&#338;&#172;pMG1 cells grown presence thiamine (rct1&#226;&#338;&#172;pMG1 &#226;&#171;&#185; T) inhibit expression Rct1-HA plasmid. <br /> (C) Actin, DNA, cell wall (Hoechst aniline blue [H&#226;&#171;&#185;AB]) staining rct1&#226;&#338;&#172;pMG1 cells grown overnight YES   24 h presence thiamine  . Note mycelial, branched, unseparated rct1&#226;&#338;&#172;pMG1 cells asymmetric aniline blue staining (AB). <br /> Cells panels  left) scale. cells right panel scale. Images acquired CCD camera using DIC optics 63&#226;&#171;&#187; oil objective. insert left panel represents analysis Rct1-HA <br /> level rct1&#226;&#338;&#172;pMG1 cells grown presence thiamine. (D) Analysis sporulation defects rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. WT, rct1&#226;&#171;&#185;/&#226;&#171;&#186;, rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1, <br /> rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG, WTpMG1 cells inoculated EMM-N medium OD600 0.1 grown 32&#194;&#176;C 24 h. Sporulation scored 24 h counting 100 cells strain. (E) RT-PCR analysis meiotically induced genes cells grown EMM-N. Samples taken indicated time points, RT-PCR performed described previously (2). Positions unspliced spliced <br /> transcripts indicated right. <br />  3606     GULLEROVA ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br />  shows low levels Rct1 able support     phase (Fig. 3C). Analysis cytoplasmic tubulin levels (Fig. cell growth.  rct1&#226;&#338;&#172;pMG1 cells grown 24 h       3B C, lowest panels), staining membranes longer presence thiamine media          Ponceau S (data shown), revealed equal amounts wider (swollen) shorter WT cells, variety      protein loaded lane. addition, analysis growth shapes ranging normal rod-like                levels RNAP II subunits, Rpb3 Rpb7, showed <br /> rounded, pear-like, irregularly branched, highly elongated       total levels RNAP II WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells (Fig. 2C data shown).  cells grow short        similar (Fig. 3B).  results described clearly <br /> chains, indicating defects cell separation. worth     demonstrate CTD phosphorylation status impaired <br /> noting WTpMG1 cells overexpress Rct1 grow            rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells reduced levels Rct1 result CTD <br /> slowly, accordance previous data AtCyp59 (12),       hyperphosphorylation.  results consistent morphological defects observed. Finally,       studies AtCyp59, overexpression led CTD <br /> results shown Fig. S1 supplemental material clearly       hypophosphorylation (12). <br /> indicate phenotypes observed rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells       Episomal expression Rct1-HA rescues CTD phosphor- <br /> rescued episomal expression Rct1-HA.                          ylation defect.  analyzed CTD phosphorylation <br />    life cycle, diploid S. pombe cells heterozygous     status rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 rct1&#226;&#338;&#172;pMG1 strains compared mating-type genes undergo mitotic prolifer-      WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. Cells grown overnight ation enter meiosis. decision devel-       YES medium (OD600, 0.5 0.7 [depending strain]), <br /> opmental states depends nutrient availability, particularly      expression Rct1-HA plasmid low, nitrogen, growth temperature. reg-          CTD phosphorylation analyzed using H14 H5  <br /> ulation meiotic differentiation perturbed rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells    tibodies. Episomal expression Rct1-HA rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 grown 32&#194;&#176;C EMM-N (EMM nitrogen source)              rct1&#226;&#338;&#172;pMG1 cells resulted decrease CTD phosphorylation <br /> (Fig. 1D). addition, genes important        compared Rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells (Fig. 3D; compare lanes 2 3 <br /> entrance meiosis regulated rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells        lane 1).  CTD phosphorylation levels turned demonstrated activated splicing (Fig. 2E). Ectopic expression       lower WT cells. effect Rct1-HA cells (rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 strain) resulted          observed WT cells ectopically expressing Rct1-HA (Fig. 3D; <br /> complementation defect. Interestingly,        compare lane 5 lanes 2 3), analyzed Rct1 levels 24 h EMM-N, &#226;&#172;&#402;15% rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells sporulated,           trains. shown Fig. 3D, Rct1 levels strains <br /> compared &#226;&#172;&#402;35% WT cells. Similarly, overexpression          somewhat higher WT cells  Fig. 2A, lower <br /> Rct1 WT background (WTpMG1) reduced sporula-            panel right).  data consistent tion efficiency (Fig. 2D).  data indicate       results ectopic expression AtCyp59 Arabidopsis <br /> precise regulation Rct1 levels necessary correct mei-     cells, showed decreased CTD phosphorylation otic differentiation. analyzed spliced transcripts    slight increase protein level (12)  explain induced meiosis (nak1, hob1, wsp1).          cells grow lower rate WT cells. <br /> shown Fig. 2D, splicing affected underex-          investigate effects Rct1 CTD phosphor- <br /> pression Rct1, indicating Rct1 probably       ylation, analyzed CTD phosphorylation rct1&#226;&#338;&#172;pMG1 cells <br /> general splicing regulator.                                          Cells grown overnight YES EMM <br />    CTD phosphorylation status affected rct1&#226;&#180;&#8482;/&#226;&#180;&#353; cells.     medium diluted OD600 0.05 YES contain- <br /> previous results obtained Arabidopsis Rct1 homolog,        ing 100 &#226;&#65533;&#174;g/ml thiamine, amounts cells AtCyp59, showed interacted CTD RNAP II          taken analysis indicated time points. Western blot ectopic expression AtCyp59 cell suspension           analysis anti-HA antibody revealed efficient repression culture resulted highly reduced growth rate. effect   Rct1-HA expression, resulting low levels Rct1-HA <br /> attributed reduced phosphorylation CTD             12 h later (Fig. 3E, lane 4 [HA panel]). Analysis RNAP II (12).                                                       blots H5 H14 antibodies revealed concomitant <br />    immunoprecipitation experiments HA-antibodies            increase CTD phosphorylation Ser2 Ser5, respec- <br /> using rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cell extracts, Rct1 copre-      tively (Fig. 3E).  equal protein loading controlled cipitates RNAP II (Fig. 3A).        anti-tubulin antibody (Fig. 3E).  described data <br /> analyze CTD phosphorylation status rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells.       clearly underexpression Rct1 increases night cultures grown YES medium mid-log phase            Rct1 overexpression decreases CTD phosphorylation. <br /> (OD600, 0.7), equal amounts cells directly resus-          Rct1 associated transcriptionally active chromatin <br /> pended SDS-PAGE loading buffer. RNAP II detected             RNAP II transcription cycle. able Western blotting using antibodies specific unphos-        demonstrate interaction Rct1 CTD RNAP II, phorylated CTD (8WG16), phosphorylated Ser2 (H5),               asked Rct1 associates active chromatin <br /> phosphorylated Ser5 (H14) (7, 20, 35). Notably, H5         stage transcription cycle. rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells H14 antibodies revealed higher phosphorylation rct1&#226;&#171;&#185;/&#226;&#171;&#186;       grown 16 h nmt1-inducible conditions, cells, 8WG16 antibody showed modest              chromatin immunoprecipitation (ChIP) performed increase RNAP II signal (Fig. 3B, lanes 2). consistent   anti-HA antibodies. RNAP II genes analyzed with specificities H5 H14 antibodies compared         oligonucleotides covering promoters, open reading frames, 8WG16 (7, 20, 35). Time course analysis cells grown      3&#226;&#172;&#732; UTRs. Figure 4A demonstrates Rct1 associated YES medium showed CTD hyperphosphorylation              chromatin promoter region remains asso- <br /> sists low OD cells reach stationary         ciated transcriptional cycle, oligonucleo- <br />  VOL. 27, 2007                                              CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3607 <br />  <br />  <br />  <br />  <br />    FIG. 3. Phosphorylation status RNAP II CTD rct1&#226;&#171;&#185;/&#226;&#171;&#186; heterozygous cells.   Rct1 coprecipitates RNAP II. Immunoprecipitation Rct1-HA rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cell extracts results coprecipitation RNAP II, determined anti-CTD monoclonal antibody H14. Lane 1, input <br /> protein extract; lane 2, protein extract incubated protein Sepharose beads; lane 3, protein extract incubated anti-HA monoclonal <br /> antibody protein Sepharose. Antibodies used Western blotting indicated right. Immunoprecipitation anti-HA antibody protein extract WT cells did result precipitation detectable amounts RNAP II (lowest panel, analyzed anti-CTD H14 <br /> antibody). (B) CTD phosphorylation increased rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. Overnight cultures mid-exponential phase (OD600 0.7) grown YES <br /> medium collected directly resuspended 2&#226;&#171;&#187; SDS-PAGE loading buffer, proteins separated 8% SDS-PAGE. Blots probed monoclonal antibodies CTD RNAP II, Rpb3, Rpb7 (indicated right panel). Analysis cytoplasmic &#226;&#65533;&#163;-tubulin level performed loading control. (C) Time course CTD phosphorylation WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. Overnight <br /> cultures grown YES medium diluted OD600 0.1 grown  Cells taken indicated ODs, CTD phosphorylation analyzed monoclonal antibody H14 (specific Ser5). Analysis cytoplasmic tubulin level used loading control (lower <br /> panel). Details panel B. (D) CTD phosphorylation defect rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells rescued episomal expression Rct1-HA. Protein blots probed antibodies indicated right. (E) Repression Rct1-HA expression increases CTD phosphorylation. Protein blots rct1&#226;&#338;&#172;pMG1 cells grown presence thiamine taken indicated time points analyzed Western blotting antibodies <br /> indicated right. <br />  <br />  <br />  <br /> tide pairs resulted comparable amplification corre-               (act1), affected meiotic differentiation, did sponding DNA fragments input coprecipitated DNA                    difference 10- 24-h time points (Fig. <br /> samples (compare lanes 3 1). Control PCRs performed                   4B).  data consistent increased tran- oligonucleotides corresponding promoter coding                scriptional activity spo4, meu13, mfr1, mei2 genes regions mitochondrial cox1 gene nuclear U6                  24-h time point rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells (Fig. 2E) clearly <br /> snRNA gene, transcribed RNAP III, did result              demonstrate association Rct1 chromatin  significant amplification, indicating Rct1 specifically           pends transcriptional activity. <br /> associates RNAP II genes. results agree-              RNAP II transcription reduced rct1&#226;&#180;&#8482;/&#226;&#180;&#353; cells. CTD <br /> ment observation mainly RNAP II transcription               undergoes complex pattern phosphorylation/dephosphor- reduced rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells  .                                  ylation transcription cycle  introduction).    investigate association Rct1 chroma-            order enable entry RNAP II new initiation com- <br /> tin correlates transcriptional activity, tested mei-         plex, CTD dephosphorylated release osis-specific genes (spo4, meu13, mfr1, mei2 [2, 30])            chromatin.  investigated functional  <br /> showed enhanced transcription rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells                 sequence(s) Rct1 depletion RNAP II transcriptional <br /> grown 24 EMM-N medium (Fig. 2E). performed                      activity. measured levels nascent transcription ChIP analysis rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells grown 10 24 h             known RNAP II transcription units using nuclear <br /> EMM-N. Figure 4B clearly demonstrates 10 h inoc-               run analysis. Equal amounts exponentially growing WT <br /> ulation, Rct1 associated extent promoter,             rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells permeabilized Na-sarcosyl, coding region, 3&#226;&#172;&#732; UTRs analyzed              ongoing transcription measured incorporation genes.  24 h inoculation, significant increase         [32P]UTP. DNA fragments corresponding coding  <br /> coprecipitation promoter, coding regions, 3&#226;&#172;&#732; UTRs              gions 13 randomly selected protein-coding genes PCR observed (Fig. 4B). contrast, analysis actin gene          amplified slot blotted membranes. addition, DNA <br />  3608     GULLEROVA ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Rct1 associated transcriptionally active chromatin RNAP II transcription cycle.   Association Rct1 chromatin analyzed ChIP. ChIP performed anti-HA antibody rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells grown 32&#194;&#176;C nmt1-inducible conditions <br /> (EMM medium). RNAP II genes previously analyzed nuclear run analyzed (upper panels). lower panels controls <br /> performed genes transcribed mtRNAP (cox1) RNAP III (U6 snRNA). Lanes 1, PCR immunoprecipitated chromatin; lanes 2, <br /> PCR chromatin bound protein Sepharose  lanes 3, PCR input chromatin. (B) ChIP analysis meiosis-specific genes. rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells grown EMM-N 10 24 h, ChIP meiotically induced genes spo4, meu13, mfr1, mei2 performed described panel  Coimmunoprecipitated DNA analyzed real-time PCR. samples run triplicate ensure <br /> accuracy data. <br />  <br />  <br />  <br />  <br /> fragments genes transcribed RNAP III (tRNASer U6 snRNA), RNAP (17S rRNA), mitochondrial <br /> RNA polymerase (mtRNAP) (cox1 gene) employed controls. Filters hybridized total [32P]UTP-labeled <br /> RNA WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells quantified exposure PhosphorImager. Filters subsequently exposed X-ray film (Fig. 5A). Quantification results  <br /> dependent experiments clearly showed transcription 13 RNAP II genes reduced 20 50% rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells <br /> (Fig. 5A, genes 1 13). contrast, RNAP III (samples <br /> 14 16) mtRNAP (sample 18) transcript synthesis affected, RNAP transcription (sample 15) slightly inhibited (Fig. 5A).  transcription run data partial depletion Rct1, results hy- <br /> perphosphorylation CTD, affects RNAP II transcription.  additional effect RNAP transcription excluded  Discussion). <br />    ongoing RNAP II transcription reduced rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells, wanted analyze impact steady- <br /> state levels RNAP II transcripts.  total RNA <br /> isolated WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells reverse transcribed used PCR amplification 10 RNAP II genes previously analyzed nuclear run assay (Fig. 5A, genes 1 10). semiquantitative RT-PCR experiment                         FIG. 5. RNAP II transcription reduced rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells.   Au- <br />                                                                            toradiograph representative nuclear run experiment. Samples <br /> peated times newly isolated RNA. Quantification               1 13, RNAP II transcripts. addition, DNA fragments results shown Fig. 5B revealed RNA steady-state             RNAP gene (17S rRNA; sample 15), RNAP III genes (tRNASer levels 10 genes reduced 15 40% rct1&#226;&#171;&#185;/&#226;&#171;&#186;              U6 snRNA genes, samples 14 16), RNAP II gene U1 <br />                                                                            snRNA (sample 17), mtRNAP (cox1 gene; sample 18), used <br /> cells. Measurements ongoing transcription steady-state <br />                                                                            controls. Slot blots exposed X-ray film. Quantification levels mRNAs provide strong evidence reduction              independent nuclear run experiments shown  Inten- <br /> Rct1 level results inhibition RNAP II transcription.           sities y axis arbitrary units. (B) Steady-state levels 10 <br /> results, increased phosphorylation status             RNAP II genes (genes 1 10 panel  analyzed semi- <br />                                                                            quantitative RT-PCR. Total RNA isolated WT rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells CTD, predicted influence interaction                                                                            reverse transcribed oligo(dT) primer, resulting proteins, cause observed growth              cDNA used PCR gene-specific primers. Intensities morphological defects rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells.                                    y axis arbitrary units. <br />  VOL. 27, 2007                                           CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3609 <br />  <br />  <br />                         DISCUSSION <br />  <br />    data presented  previous data Arabidopsis AtCyp59, provide genetic biochemical evi- <br /> dence S. pombe nuclear multidomain cyclophilin Rct1 required fine tuning RNAP II CTD phosphorylation. <br /> Specifically, rct1 essential gene.   deletion rct1 allele diploid strain results elevated phosphorylation CTD, reduced RNAP II tran- <br /> scriptional activity, severe growth morphological  <br /> fects. data, phenotypes heterozy- <br /> gote, provide evidence Rct1 levels tight dosage <br /> control correct levels Rct1 absolutely required        FIG. 6. Model Rct1 influence CTD structure balanced CTD phosphorylation/dephosphorylation cycle.         function. propose &#226;&#8364;&#339;CTD code&#226;&#8364;&#65533; result coordinated <br /> Finally, presence Rct1 entire length actively   activities CTD kinases, phosphatases, PPIases,                                                                     generate high-affinity binding sites factors involved regula- <br /> transcribed genes consistent direct interaction        tion RNAP II transcription coupling pre-mRNA pro- <br /> tween Rct1 elongating RNAP II complex.             cessing. <br /> results suggest multidomain cyclophilin essential <br /> regulator RNAP II transcription. <br />    Cyclophilins ubiquitous proteins possess PPIase ac-     particular, shown Pcf11 binds CTD tivity; e.g., catalyze cis-trans isomerization peptide      phosphorylation-dependent manner, phosphorylation <br /> bonds preceding proline. eukaryotic cells, cyclophilins     adjacent repeats contributes increase cellular compartments variety func-     affinity Pcf11 CTD (34). new crystallographic tions ascribed  including protein trafficking     biochemical data provide evidence changes phosphor- <br /> maturation, receptor signaling, receptor complex stabilization,     ylation proline isomerization contribute struc- <br /> apoptosis, RNA processing, spliceosome assembly, tran-          tural plasticity CTD, important recruitment scription  references 15, 19, 26, 27, 28, 44, 45, 49, 52    different proteins CTD transcription complex references . Despite obviously widespread im-       (3, 24, 32). prerequisites suggest kinases, phosphatases, <br /> portance PPIases, genetic analysis 13 immunophilin genes      peptidy-prolyl isomerases important regulators S. cerevisiae revealed ESS1 (homolog human           CTD functions (3, 24, 32, 34). <br /> Pin1) essential (13).  recent reexamination              kinases phosphatases shown influ- <br /> ESS1 disruption revealed absolutely essential    ence phosphorylation/dephosphorylation CTD (32, 34, <br /> cell viability deletion causes severe tem-    38), immunophilin, Pin1/ESS1, impli- <br /> perature-sensitive growth phenotype (18). ESS1          cated regulation processes (50, 51). Pin1 essential S. pombe (18) Cryptococcus neoformans,      small parvulin-type immunophilin shown  fungi, e.g., Candida albicans Aspergillus      teract different proteins involved wide variety nidulans. Similarly, Rct1 essential S. pombe, ho-     cellular processes (4, 27, 28, 52). pin1&#226;&#171;&#186;/&#226;&#171;&#186; cells accumulate <br /> molog P. tetraurelia, Kin241, (22). clear   hypophosphorylated RNAP II, moderate overexpres- <br /> deletion particular PPIase lethal organisms     sion Pin1 resulted appearance novel hyper-  possibility certain genetic      hyperphosphorylated form RNAP II (51), influence grounds, high expression levels PPIases compensate         CTD phosphorylation investigated. vitro experiments loss s), acting natural suppressors.   suggested human Pin1 inhibits CTD phosphatase indicate functions certain PPIases      FCP1 stimulates CTD phosphorylation cdc2/cyclin B partially redundant overlapping. Alternatively, pos-   (51).  reduced levels cyclophilin Rct1 <br /> sible PPIases essential certain        result increased CTD phosphorylation Ser2 ditions.                                                            Ser5, increased levels Rct1  AtCyp59) <br />    highly repetitive primary sequence CTD suggests       (12) reduced CTD phosphorylation. indicates  adopt different conformations. significance      physiological conditions, Rct1 stimulates CTD dephosphoryla- <br /> Rct1 regulating CTD function convincing light         tion.  PPIases entirely opposite effects recent structural data suggest free CTD,        CTD phosphorylation status; Rct1p positive effect contains multitude heptapeptide repeats (YSPTSPS),          CTD dephosphorylation  work), opposed Pin1, highly flexible. Significantly, peptidyl-proline isomerase      negative effect (51).  genetic evidence <br /> Pin1, capping enzyme guanylyltransferase Cgt1, Pcf11,       S. cerevisiae indicating ESS1 stimulates dephosphor- essential component mRNA cleavage factor IA,          ylation CTD (50), Rct1 does S. pombe. Nonethe- <br /> bind CTD peptides trans conformation vitro,     cases clear effects brought <br /> providing direct evidence importance cis-       Rct1 bind CTD, facilitating <br /> trans isomerization prolyl bonds CTD repeats protein-      access CTD-modifying enzymes,  alternatively, protein interactions (10, 31, 32, 34, 48). Furthermore, interac-    CTD kinases phosphatases direct targets Rct1 (Fig. <br /> tions CTD regulatory factors         6). Intriguingly, PTPA, unusual PPIase, recently depend Ser2 Ser5 phosphorylation (23, 37, 38).            shown function activator PP2A phosphatase (21). <br />  3610     GULLEROVA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  increase/decrease CTD phosphorylation reducing/           changes binding proteins regulating splicing meiosis- <br /> elevating level Rct1 indicates Rct1         specific genes. meiosis-specific genes transcribed  <br /> volved inhibiting CTD phosphatase stimulating      der vegetative growth;  pre-mRNAs spliced <br /> CTD kinase activity.  experiments neces-        meiotic induction (2, 30). factors regulating <br /> sary answer questions precisely.                      events bind CTD specific conditions. <br />    cis-trans isomerization peptide bonds takes place spon-     deregulation CTD phosphorylation observed taneously, lower rate (42, 44, 49),        rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells induce binding vegetative <br /> possible PPIases absolutely required vivo.          growth, consequently leading splicing induction mei-  generally required selective prolyl       osis. Similarly, recently demonstrated CTD <br /> peptide bond isomerization processes fast changes       promotes alternative splicing fibronectin pre-mRNA  <br /> protein-protein interactions protein phosphorylation          cruiting SR protein SRp20 (9). Interestingly, human ho- <br /> phosphorylation place, e.g., coupled transcription         molog Rct1, PRIL4, recently identified affinity- <br /> pre-mRNA processing. genetic biochemical data              purified spliceosomal B complex (8), supporting accumulated decade indicate phys-       hypothesis Rct1 plays important role interface ical functional coupling RNAP II transcription          transcription pre-mRNA processing. Rct1  <br /> pre-mRNA processing chromatin structure             deed involved pre-mRNA processing    <br /> CTD RNAP II. highly dynamic CTD phosphorylation                tributes splicing regulation requires investigation. <br /> pattern required coordination transcription nu- <br /> clear pre-mRNA processing (1, 4, 17, 35, 37, 39, 46, 53).                                 <br /> addition, recently demonstrated cis-trans <br />                                                                      <br /> isomerization prolyl peptide bonds contribute       <br /> establishing disrupting binding interactions CTD        <br /> (3, 24, 32, 34).  data presented     <br /> previously Pin1, highlights general importance            <br /> PPIases regulation CTD structure function (Fig. 6). <br />    able Rct1 required correct                                    <br /> CTD phosphorylation consequently efficient RNAP II         <br /> transcription.  highly pleiotropic phenotype ob-            <br /> served rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells, observed tight dos-      <br /> age control Rct1, unusual PPIases hints             unknown function(s) RNAP II                  <br /> transcription regulation.  RNAP tran-           <br /> scription reduced rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells. reason       <br /> decrease transcription 17S rRNA clear         <br /> moment.  consistent previous obser-            <br /> vation green fluorescent protein-tagged AtCyp59 localizes          nucleolus extent (12), possibly      <br /> affect transcription RNAP  note            <br /> deletion rct1 allele, results reduced Rct1 pro-     <br /> tein levels, causes strong growth morphological pheno-      <br /> types, presumably deregulated CTD phosphorylation             RNAP II transcription.  possibility Rct1      involved regulation factors important cell           <br /> morphology differentiation excluded. Episomal ex-           <br /> pression Rct1-HA rct1&#226;&#171;&#185;/&#226;&#171;&#186;pMG1 cells partially restored    <br /> growth morphological defects observed rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells,           moderate overexpression WT cells caused strong      <br /> negative effect growth meiotic differentiation,            cell size, polarity, mitotic division WT    <br /> cells.  Rct1 levels clearly tight dosage control.       <br />    Rct1 unusual cyclophilin, contains RRM         <br /> C-terminal domain rich RS/RD dipeptide repeats                  <br /> charged amino acids. high conservation multido-       <br /> main proteins, particular RRM region, suggests ad-            <br /> ditional possible functions apart PPIase activity.    observation AtCyp59 interacts SR proteins            Arabidopsis suggested regulate events          <br /> interface transcription splicing (12). fact, ob-     <br /> served meiotic phenotype rct1&#226;&#171;&#185;/&#226;&#171;&#186; cells explained         deregulation CTD phosphorylation concomitant            <br /> </body></html>