<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Sept. 1998, p. 4406&#226;&#8364;&#8220;4412                                                                                     Vol. 180,  17 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#194;&#169; 1998, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />          Contribution Pmra Promoter Expression Genes                Escherichia coli mra Cluster Cell Envelope <br />                      Biosynthesis Cell Division Genes <br />                         DOMINIQUE MENGIN-LECREULX,1* JUAN AYALA,2 AHMED BOUHSS,1 <br />                                 JEAN VAN HEIJENOORT,1 CLAUDINE PARQUET,1 <br />                                                             3 <br />                                             HIROSHI HARA &#226;&#8364;&#160; <br />  <br />                  Laboratoire des Enveloppes Bacte&#194;&#180;riennes, Centre National la Recherche Scientifique, Universite&#194;&#180; <br />                       Paris-Sud, 91405 Orsay Cedex, France1; Centro Biolog&#196;&#177; Molecular Severo Ochoa, <br />                               Consejo Superior Investigationes Cient&#196;&#177;&#194;&#180;ficas, Universidad Auto <br />                                                                                                &#194;&#180;noma <br />                                   Madrid, Canto Blanco, 28049 Madrid, Spain2; National <br />                                      Institute Genetics, Mishima, Shizuoka-ken 411, Japan3 <br />                                                Received 7 1998/Accepted 1 July 1998 <br />  <br />             Recently, promoter essential gene ftsI, encodes penicillin-binding protein 3 Escherichia coli, <br />          precisely localized 1.9 kb upstream gene, beginning mra cluster cell division          cell envelope biosynthesis genes (H. Hara, S. Yasuda, K. Horiuchi, J. T. Park, J. Bacteriol. 179:5802&#226;&#8364;&#8220;5811, <br />          1997). Disruption promoter (Pmra) chromosome replacement lac promoter <br />          (Pmra::Plac) led isopropyl-b-D-thiogalactopyranoside (IPTG)-dependent cells lysed absence          inducer, defect complemented region Pmra ftsW, fifth gene <br />          downstream ftsI, provided trans plasmid. present work, levels various proteins <br />          involved <span id='am-42' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-48' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> synthesis cell division precisely determined cells Pmra::Plac <br />          promoter expression repressed fully induced. confirmed Pmra promoter required          expression genes mra cluster: mraZ (orfC), mraW (orfB), ftsL (mraR), ftsI, murE, murF, <br />          mraY, murD, ftsW. Interestingly,  sixfold-decreased levels MurG MurC enzymes          observed uninduced Pmra::Plac cells. correlated accumulation <span id='am-29' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-30' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> precursors <br />          <span id='am-96' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-97' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span>UDP</span>&#226;&#8364;&#8220;N-acetylglucosamine UDP&#226;&#8364;&#8220;N-acetylmuramic acid, substrates enzymes, depletion <br />          pool UDP&#226;&#8364;&#8220;N-acetylmuramyl pentapeptide, resulting decreased cell wall peptidoglycan synthesis. <br />           expression ftsZ, penultimate gene cluster, significantly reduced Pmra <br />          expression repressed. concluded transcription genes located downstream ftsW <br />          mra cluster, murG ftsZ, mainly  exclusively) dependent Pmra promoter. <br />  <br />  <br />    rigid, shape-determining material bacterial cell walls            crucial question genes large mra cluster giant polymer periodic structure named peptidoglycan                transcribed open. promoter ftsI gene <span id='am-53' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-54' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-62' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span>murein</span>. biosynthesis complex process involving              named pbpB), encodes penicillin-binding protein 3 <br /> different cytoplasmic membrane steps (33). Conditional-                 (PBP3) (7, 19, 28), genes upstream lethal mutants Escherichia coli altered different levels           recently identified (13). Interestingly, inactivation metabolic pathway described previously,                  promoter (Pmra) chromosome replacement mutations mapped regions                 inducible promoter (Plac) led isopropyl-b-D-thiogalactopy- <br /> chromosome (10, 27, 36).  2 min E. coli               ranoside (IPTG)-dependent cells grow ab- <br /> map, studied great contained large               sence inducer plasmid carrying cluster genes, mraZ envA, code proteins                 mraZ-ftsW region present (13). concluded involved cell envelope biosynthesis cell division.            Pmra promoter essential expression designated mra murein region (7, 19, 27) dcw                genes mra cluster. fact, report uninduced division cell wall (1, 9, 34, 35). earlier work          Pmra::Plac cells significantly depleted products laboratories, complete physical map                   different genes located downstream ftsW, indicating <br /> DNA sequence 17-kb region determined                      main proportion transcription genes function genes present cluster                murG ftsZ) derives Pmra promoter. identified.  exception cell divi- <br /> sion genes ftsQ, ftsA, ftsZ, transcription                                      MATERIALS METHODS <br /> investigated great (4, 9, 11, 29, 31, 34, 35, 37),                                                                                Materials. Acetyl coenzyme (acetyl-<span id='am-24' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>), glucosamine-1-phosphate (GlcN- <br />                                                                             1-P), <span id='am-81' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-82' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span>, UDP&#226;&#8364;&#8220;N-acetylglucosamine (UDP-GlcNAc) bought                                                                             Sigma. Peptidoglycan nucleotide precursors dipeptide D-Ala&#226;&#8364;&#8220;D-Ala   * Corresponding author. Mailing address: Laboratoire des Envel-           prepared described previously (20&#226;&#8364;&#8220;24). UDP-[14C]GlcNAc (7.4 GBq oppes Bacte&#194;&#180;riennes, Centre National la Recherche Scientifique,          mmol21), [3H]<span id='am-76' about='obo:IMR_0200409' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-77' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:id" content="IMR:0200409" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="UMP" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00105" datatype="xsd:string"></span>UMP</span> (0.5 TBq mmol21), L-[14C]alanine (5.6 GBq mmol21) <br />                                                                             purchased Amersham; [14C]<span id='am-86' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-87' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span>acetyl-CoA</span> (1.9 GBq mmol21) Universite&#194;&#180; Paris-Sud, Ba&#203;&#8224;timent 432, 91405 Orsay Cedex, France. <br />                                                                             ICN; D-[14C]<span id='am-64' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-65' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span>glutamic acid</span> (1.9 GBq mmol21) obtained American <br /> Phone: 33-1-69-15-61-34. Fax: 33-1-69-85-37-15. E-mail: dominique           Radiolabeled Chemicals (St. Louis, Mo.); [meso-14C]diaminopimelic acid <br /> .mengin-lecreulx@ebp.u-psud.fr.                                             ([meso-14C]A2pm; 11.5 GBq mmol21) CEA (Saclay, France). <br />   &#226;&#8364;&#160; Present address: Department Biochemistry Molecular Bi-              Bacterial strains plasmids. JE7968 (Pmra::Plac) JE7970 (Pmra::Plac <br /> ology, Saitama University, Urawa, Saitama 338-8570, Japan.                  recA1) derivatives W3110 carry Plac cassette inserted  <br />                                                                      4406 <br />  VOL. 180, 1998                                                                  EXPRESSION GENES E. COLI mra CLUSTER                                     4407 <br />  <br />  <br />  <br />  <br />    FIG. 1. lacZ operon fusion experiments Pmra-murG region mra cluster. orientation approximate sizes mra cluster genes (black arrows) rho-independent terminator  bracket T beneath  shown   filled bars denote chromosomal fragments cloned promoter <br /> assay mini-F vector pFZY1DH. left ends, cut HpaI, joined filled EcoRI site vector. Short open bars denote SmaI-BamHI V <br /> interposon polylinker (8). Open arrowheads P&#226;&#8364;&#8482;s beneath represent Pmra, disrupted Pmra (Pmra::4 bp), Pmra replaced Plac (Pmra::Plac). plasmid <br /> numbers left Pmra, Pmra::4 bp, Pmra::Plac, respectively. b-Gal activities averages measurements transformants MC1061-5 <br /> grown buffered L broth-glucose-<span id='am-37' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span>thymine</span> medium (13) containing ampicillin, (1) (2) IPTG concentration 1 mM. figure drawn scale galK9-9lacZ fusion gene (large white arrows) short upstream region, including polylinker (dotted line) vector (13, 17). Abbreviations restriction sites  relevant sites shown): B, BamHI; E, EcoRI; Ev, EcoRV; H, HindIII; S, SmaI; Sn, SnaBI. <br />  <br />  <br />  <br /> HindIII site Pmra chromosome (13).  dependent           presence absence 1 mM IPTG. Strains requiring IPTG growth lac inducer growth. Plac cassette composed cat gene followed       grown presence, cultures diluted 100-fold transcriptional terminators rrnB operon, lacIq gene           prewarmed medium lacking inducer. effects depletion orientation opposite Plac orientation Pmra. MC1061-5             IPTG cell morphology (loss rod shape) cell growth (arrest growth <br /> (DlacX74) (17) used host lacZ operon fusion experiment. pHR416         followed onset cell lysis) observed time period plasmid derived pSY396 carries entire mra cluster (21-kb           depended strain used: 2 h JE7968 JE7970 3 h <br /> AatII fragment) (13). pHR477, pHR478, pHR479, pHR431, pHR439, pHR427,                  JE7970(pHR477/pHR485). 1-h delay observed strain pHR426 mini-F plasmids carrying chromosomal fragment Pmra             fact biosynthetic activity (MurD) depleted mraY, murD, ftsW, murG, ftsQ, ftsA, ftsZ, respectively. pHR485 carries          strain IPTG deprivation. time  final optical density culture <br /> ftsW control aadA promoter vector pGB2 (13).               approximately 0.7), cells harvested washed 40 ml cold <br /> mini-F plasmids used operon fusion experiments depicted Fig. 1.          20 mM potassium phosphate buffer (pH 7) containing 0.3 mM MgCl2 1 mM vector pFZY1DH (13), derivative pFZY1 (17).                               b-mercaptoethanol. wet cell pellet suspended 5 ml buffer <br />    Recombinant DNA procedures. methods essentially based                 disrupted sonication (Sonicator 150; T. S. Ultrasons, Annemasse, France) Sambrook et al. (30). b-galactosidase (b-Gal) assay unit          5 min cooling. resulting suspension centrifuged 4&#194;&#176;C 30 <br /> <span id='am-7' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> used described Koop et al. (17).                          min 200,000 3 g Beckman TL100 centrifuge. supernatant    Growth conditions. Unless noted, 2YT medium (26) used             dialyzed overnight 4&#194;&#176;C 100 volumes phosphate buffer, growing cells. Cell growth monitored 600 nm spectrophotometer            resulting solution (40 50 mg protein 5 ml), designated crude <br /> (model 240; Gilford Instrument Laboratories, , Oberlin, Ohio). strains         soluble enzyme, stored 220&#194;&#176;C. pellet, consisting membrane pro- <br /> carrying drug resistance genes, antibiotics used following concentra-      teins, resuspended 1 ml buffer. Protein concentrations tions  micrograms milliliter): ampicillin, 100; kanamycin, 30; spectinomy-      determined method Lowry et al. (18), using bovine serum albumin cin, 30; tetracycline, 12.5; chloramphenicol, 10.                                  standard. <br />    Preparation crude enzyme. Exponential-phase cells (0.5-liter cultures)           Enzymatic assays.   GlcN-1-P acetyltransferase (GlmU). step  different strains listed Table 1 grown 37&#194;&#176;C 2YT medium,       mation UDP-GlcNAc GlcN-1-P, catalyzed bifunctional enzyme <br />  <br />  <br />  <br />                                   TABLE 1. Specific activities peptidoglycan-synthesizing enzymes E. coli strainsa <br />                                                                                                          Relative amt <br />             Strain                      IPTG <br />                                                        MurE            MurF           MraY            MurD           MurG            MurC            Ddl          GlmU <br />  <br /> W3110                                     2             1.0             1.0            1.0            1.0               1.0           1.0            1.0           1.0 <br /> JE7968                                    2             0.25            0.2            0.35           0.25              0.4           0.3            0.7           1.0 <br />                                           1             3.0             2.0            1.8            2.1               2.0           3.4            1.3           1.0 <br /> JE7970                                    2             0.20            0.3            0.4            0.3               0.4           0.4            0.7           1.0 <br />                                           1             2.5             2.1            2.2            2.1               2.2           3.5            1.2           1.0 <br /> JE7970(pHR479)                            2             1.0             1.1            1.0            1.2               0.3           0.15           0.8           1.0 <br />                                           1             2.5             2.2            1.5            2.7               1.5           3.2            1.2           1.0 <br /> JE7970(pHR477/pHR485)                     2             1.0             1.2            0.9            0.05              0.4           0.4            0.8           1.0 <br />                                           1             2.5             2.2            1.5            2.0               2.0           3.2            1.3           1.0 <br /> JE7970(pHR439)                            2             1.0             1.1            1.1            1.0               1.1           1.1            1.1           1.0 <br />         Cells grown exponentially 37&#194;&#176;C 2YT medium supplemented (1) supplemented (2) 1 mM IPTG. IPTG-requiring strains grown presence inducer diluted prewarmed medium IPTG. Growth continued effects cell growth observed, 2 3 h later, depending strain used  text). Crude extracts (soluble membrane fractions) prepared analyzed different enzyme <br /> activities. value 1 wild-type strain W3110 corresponds  nanomoles minute21 milligram protein21) 1.5 MurE, 3 MurF, 0.6 MurD, 0.04 MraY, 0.02 MurG, 0.06 MurC, 0.8 Ddl, 103 GlmU, respectively. specific activity GlmU, enzyme involved early steps peptidoglycan <br /> synthesis encoded gene separate chromosomal region (25), used internal standard control absence variability extracts. <br />  4408       MENGIN-LECREULX ET AL.                                                                                                          J. BACTERIOL. <br />  <br />  <br /> GlcN-1-P acetyltransferase (GlmU), standard assay mixture containing 50                                      RESULTS <br /> mM Tris-HCl buffer (pH 8.0), 2 mM UTP, 3 mM MgCl2, 0.5 mM [14C]acetyl- <br /> CoA (700 Bq), 2 mM GlcN-1-P, enzyme (10 mg protein) final volume 100 ml monitored. <br />                                                                                           Levels peptidoglycan-synthesizing enzymes Pmra::Plac <br />    (ii) <span id='am-8' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-Alanine</span>-adding enzyme (MurC). formation UDP&#226;&#8364;&#8220;N-acetyl-                 cells. examine far mra cluster function muramyl (MurNAc)&#226;&#8364;&#8220;L-Ala standard assay mixture containing 100 mM Tris-             Pmra promoter required, recently disrupted HCl buffer (pH 8.6), 5 mM <span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span>, 20 mM MgCl2, 2 mM L-[14C]alanine (2 KBq),               promoter chromosome replaced induc- <br /> 1 mM UDP-MurNAc, enzyme (250 mg protein) final volume 50 ml             ible lac promoter (13). IPTG-dependent strain, JE7968, monitored. <br />    (iii) D-Glutamic acid-adding enzyme (MurD). formation UDP&#226;&#8364;&#8220; <br />                                                                                        resulted construction; strain shown lyse <br /> MurNAc&#226;&#8364;&#8220;L-Ala&#226;&#8364;&#8220;D-Glu standard assay mixture containing 100 mM Tris-HCl              deprived lac inducer exponential growth, buffer (pH 8.6), 5 mM ATP, 5 mM MgCl2, 25 mM D-[14C]glutamic acid (500 Bq),            probably expression genes <br /> 25 mM UDP&#226;&#8364;&#8220;MurNAc&#226;&#8364;&#8220;L-alanine, enzyme (5 mg protein) final volume             involved peptidoglycan synthesis repressed. control 50 ml monitored.                                                                point, levels enzymes encoded different <br />    (iv) meso-A2pm-adding enzyme (MurE). formation UDP-MurNAc <br /> tripeptide standard assay mixture containing 100 mM Tris-HCl buffer (pH <br />                                                                                        genes mra cluster determined cells strain <br /> 8.6), 5 mM ATP, 100 mM MgCl2, 0.1 mM [meso-14C]A2pm (500 Bq), 0.2 mM                   JE7968 grown continuously presence UDP&#226;&#8364;&#8220;MurNAc&#226;&#8364;&#8220;L-Ala&#226;&#8364;&#8220;D-Glu, enzyme (50 mg protein) final volume             1 mM IPTG depleted lac inducer approximately 2 h <br /> 75 ml monitored.                                                                    effects cell growth observed).    (v) D-Alanyl&#226;&#8364;&#8220;D-alanine-adding enzyme (MurF). formation UDP-                  cases  MurE, MurF, MraY, MurD, MurG, MurC), MurNAc pentapeptide standard assay mixture containing 100 mM Tris-HCl <br /> buffer (pH 8.6), 5 mM ATP, 100 mM MgCl2, 70 mM D-[14C]Ala&#226;&#8364;&#8220;D-Ala (500 Bq), <br />                                                                                        levels enzymes observed induced JE7968 cells  <br /> 70 mM UDP&#226;&#8364;&#8220;MurNAc&#226;&#8364;&#8220;L-Ala&#226;&#8364;&#8220;g-D-Glu&#226;&#8364;&#8220;meso-A2pm, enzyme (20 mg pro-                    threefold higher parental W3110 cells <br /> tein) final volume 100 ml monitored.                                       (Table 1). exception concerned Ddl, activity    (vi) D-Alanine:D-alanine ligase (Ddl). standard assay mixture contained         increased 20 30%. expected, inverse variations <br /> 50 mM Tris-HCl buffer (pH 8.6), 5 mM ATP, 20 mM MgCl2, 50 mM D-[14C]Ala                observed IPTG-depleted cells, contained  <br /> (1 KBq), 0.12 mM UDP&#226;&#8364;&#8220;MurNAc&#226;&#8364;&#8220;L-Ala&#226;&#8364;&#8220;g-D-Glu&#226;&#8364;&#8220;meso-A2pm, enzyme (20 <br />                                                                                        fivefold different enzymes mentioned mg protein) final volume 50 ml. reaction product, D-Ala&#226;&#8364;&#8220; <br /> D-Ala, inhibits activity D-alanine:D-alanine ligase (Ddl) (20), quan- <br />                                                                                        wild-type strain W3110, exception Ddl, <br /> titatively converted UDP-MurNAc pentapeptide coupling Ddl activity           decreased 30% (Table 1). concomitant MurF present extract.                                            similar variations enzymes clearly indicated <br />    (vii) Phospho-MurNAc pentapeptide translocase (MraY). reaction          transcription corresponding genes mainly  <br /> substitution [3H]UMP UMP moiety UDP-MurNAc pentapeptide <br />                                                                                        pendent Pmra promoter. used assay translocase activity 20-ml reaction mixture consisting 50 mM Tris-HCl buffer (pH 7.5), 12.5 mM MgCl2, 13 mM [3H]UMP (500 Bq),                 reported previously, IPTG requirement strain <br /> 0.16 mM UDP-MurNAc pentapeptide, membranes (60 mg protein).                     JE7968 complemented chromosomal fragment <br />    cases (assays vii), reaction mixtures incubated 37&#194;&#176;C        extending Pmra ftsW (inclusive) provided 30 min, reactions terminated addition 10 ml <span id='am-91' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-92' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span>,         trans plasmid (13). constructs reaction products separated high-voltage electrophoresis Whatman <br />                                                                                        expression genes impaired 3MM filter paper 2% <span id='am-101' about='obo:IMR_0200110' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-102' property="oboInOwl:id" content="IMR:0200110" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Formic acid" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="KEGG:C00058" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formic acid</span> (pH 1.9) 1 1.5 h 40 V/cm, using LT36 apparatus (Savant Instruments, Hicksville, N.Y.). radioactive spots           IPTG absent instance, ftsW strain JE7970(pHR478) located overnight autoradiography type R2 films (3M, St. Paul,            murD strain JE7970(pHR477/pHR485). Minn.) radioactivity scanner (Multi-Tracermaster model LB285;                strains required IPTG growth filamented lysed G Wallac/Berthold, Evry, France). radioactive spots cut            deprived lac inducer, depending function gene radioactivity counted Betamatic IV liquid scintillation <br />                                                                                        product cell division peptidoglycan synthesis. <br /> spectrophotometer (Kontron Instruments) solvent consisting 2 <br /> ml water 13 ml Aqualyte mixture (J. T. Baker Chemicals, Deventer,            instance, observed cells JE7970(pHR477/ Netherlands).                                                                      pHR485) stopped growing lysed 3 h expo- <br />    (viii) N-Acetylglucosaminyltransferase (MurG). standard reaction mix-           nential growth absence IPTG. analysis cell <br /> ture contained, final volume 25 ml, 100 mM Tris-HCl buffer (pH 7.5),           content time consistent defective expression <br /> 40 mM MgCl2, 30 mM ATP, 0.7 mM UDP-MurNAc pentapeptide, 2 mM UDP- <br />                                                                                        D-glutamic acid-adding enzyme (MurD): 20-fold-reduced <br /> [14C]GlcNAc (1 KBq), membranes (150 mg protein). Membranes incubated UDP-MurNAc pentapeptide 10 min 35&#194;&#176;C generate            level enzyme detected (Table 1), large amounts undecaprenyl-pyrophosphoryl MurNAc pentapeptide  MraY) addi-                UDP&#226;&#8364;&#8220;MurNAc&#226;&#8364;&#8220;L-Ala  nucleotide substrate MurD) <br /> tion time zero radioactive substrate 10-min incubation 35&#194;&#176;C.        accumulated, precursors located downstream reaction stopped placing tubes boiling-water bath 2 min,       pathway depleted. resulted 40% lower pep- reaction mixtures analyzed descending chromatography 16 h           tidoglycan content, probably minimum value <br /> Whatman 1 filter paper isobutyric acid&#226;&#8364;&#8220;1 M NH4OH (5:3, vol/vol). Spots <br /> corresponding products (peptidoglycan <span id='am-1' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> intermediate) remaining         compatible cell viability (data shown). <br /> UDP-GlcNAc substrate detected radioactivity counted                 Cells JE7968 JE7970  recA derivative  <br /> described                                                                        IPTG independent growth transformed    Pool levels peptidoglycan precursors. Cells W3110 JE7970 deriva-          plasmid (pHR479) carrying minimal complementing <br /> tives (1-liter cultures) grown exponentially 37&#194;&#176;C 2YT medium         fragment Pmra-ftsW region.  growth rate absence presence IPTG. optical density cultures reached <br /> 0.7, cells rapidly chilled 0&#194;&#176;C harvested cold. extraction   strain absence IPTG relatively low compared peptidoglycan nucleotide precursors boiling water cold trichloroace-       wild-type strain rate presence <br /> tic acid analytical procedure used quantitation previ-       IPTG (generation times 37&#194;&#176;C 70, 35, 39 min, <br /> ously described (20, 21).                                                              respectively). expected, JE7970(pHR479) cells contained <br />    Isolation sacculi quantitation peptidoglycan. Exponential-phase cells     wild-type levels MurE, MurF, MraY, MurD enzymes <br /> (0.5-liter cultures) W3110 JE7970 derivatives grown described  absence presence IPTG. Harvested cells washed           sixfold-reduced levels MurG MurC ac- <br /> cold 0.85% NaCl solution centrifuged  Bacteria rapidly             tivities, respectively (Table 1). cells grown  <br /> resuspended vigorous stirring 20 ml hot (95 100&#194;&#176;C) aqueous 4%          tinuously absence IPTG, values represented sodium dodecyl sulfate solution 30 min. standing overnight room           production enzymes conditions expres- <br /> temperature, suspensions centrifuged 30 min 200,000 3 g        sion Pmra promoter maximally repressed. pellets washed times water. final suspensions, 2-ml <br /> volumes water, homogenized brief sonication. Aliquots hydro-           cells viable conditions, implying lyzed analyzed previously described (24), peptidoglycan content         MurG MurC activities somewhat excess wild- sacculi expressed terms muramic acid content.                     type strain residual activity sustain <br />  VOL. 180, 1998                                                                     EXPRESSION GENES E. COLI mra CLUSTER                                  4409 <br />  <br />  <br />       TABLE 2. Abnormal peptidoglycan content pools                             Pmra activity (13), displaced insertion Plac cassette <br />              peptidoglycan precursors uninduced                                     HindIII site (Pmra::Plac; pHR582, pHR584, pHR585 <br />                     JE7970(pHR479) cellsa                                              respectively). <br />                                         Amt (nmol/liter culture)                    b-Gal assays, absence functional Pmra (Pmra::4bp) <br />        Strain          IPTG <br />                                                                                        Pmra::Plac activity repressed, indicated                                 UDP-       UDP-       UDP-MurNAc          Peptido-     level transcription detectable EcoRV site low <br />                                GlcNAc     MurNAc      pentapeptide         glycan <br />                                                                                        compared background b-Gal activity measured W3110                   2         70          25           230             2,600       pFZY1DH vector (Fig. 1). Interestingly, Boyle et al. recently <br /> JE7970(pHR479)          2        240         150            90             1,900       localized promoter ftsW gene EcoRI                         1         75          30           190             3,000       BglII sites mraY-murD intergenic region (3). JE7970(pHR439)          2         75          30           230             2,500       region just upstream EcoRV site (Fig. 1), contri- <br />       Cells grown exponentially 37&#194;&#176;C 2YT medium absence (2)            bution promoter low-level transcription detected presence (1) 1 mM IPTG. cases, cells harvested optical         likely. SnaBI SmaI sites, hand, <br /> density 0.7, corresponding approximately 230 mg bacterial cell (dry           small significant transcription detected. <br /> weight) liter culture. Cell peptidoglycan extracted quantitated      likely promoter(s), terms muramic acid content (20, 24). Nucleotide precursors extracted, <br /> purified, quantitated previously described (20, 21).                            EcoRV site murD SnaBI site ftsW,                                                                                        responsible residual expression murG  <br />                                                                                        stream genes observed uninduced Pmra::Plac cells. <br />                                                                                           presence functional Pmra operon fusion <br /> peptidoglycan synthesis, rate sufficient cell                        plasmids, higher b-Gal activity detected SmaI site <br /> integrity. analyses showed cells accumulated                        murG (Fig. 1). transcription originating Pmra <br /> UDP-GlcNAc UDP-MurNAc, nucleotide substrates                                proceed murG probably  MurG MurC enzymes, respectively (Table 2).                                necessarily required growth division cell. <br /> finding confirmed partial depletion enzymes                          consistent increase MurG, MurC, Ddl, cell content indicated significant reduction                          enzymatic activities observed induced JE7970 cells. <br /> flow metabolites pathway peptidoglycan synthesis                         activity Pmra high Plac fully induced enzymatic steps. Consequently, pool UDP&#226;&#8364;&#8220;N-                              1 mM IPTG. <br /> acetylmuramyl pentapeptide, end product cytoplas-                              finding b-Gal activity detected SmaI site, <br /> mic steps, decreased twofold cell peptidoglycan                            presence functional Pmra Pmra::Plac ac- <br /> content decreased 30% (Table 2). previously                              tivity induced, half EcoRV established peptidoglycan content E. coli cells                      SnaBI sites intriguing. Transcriptional attenuation dif- reduced 40 50% loss cell integrity                           ferential degradation mRNA occurring. <br /> (reference 25 references . Table 2 shows                             Relative amounts MraW, PBP3, FtsZ E. coli mu- <br /> pools precursors peptidoglycan content strain                            tant strains. characterized expression genes <br /> JE7970(pHR439), carries murG murC addition                                encoding MraW, PBP3, FtsZ strain JE7970 deple- plasmid, normal. results observed                            tion IPTG inducer 2 h, just lysis cells, strain JE7970(pHR479) grown presence                                measuring relative amounts proteins detected IPTG. finding confirmed variations described                             specific antibodies immunoblotting assay. Increases clearly correlated partial depletion                        levels proteins, compared levels MurG MurC enzymes cell content. ap-                              parental strain, W3110, induction  <br /> proximation, reduced growth rate attributed                            ditions, decreases observed depletion depletion enzymes, strain carrying                          inducer (Table 3). Note case protein, addition murG murC plasmid, JE7970(pHR439), <br /> grew faster, generation time 50 min. <br />    results taken suggested,  tran- <br />                                                                                                  TABLE 3. Relative amounts MraW, PBP3, scription gene located Pmra-ftsW region                                           FtsZ E. coli mutant strainsa control sole Pmra promoter  second, transcription genes distal ftsW originated                                                                        Relative amt 6 SD  <br />                                                                                               Strain             IPTG <br /> Pmra promoter(s). finding 70                                                                MraW             PBP3              FtsZ <br /> 80% MurG MurC activities lost repression Plac Pmra::Plac cells indicated Pmra ab-                    W3110                       2        100             100               100 <br /> solutely required  governed main  tran-                                                    1        106 6 13         88 6 9           109 6 16 <br /> scription corresponding genes.                                                  JE7970                      2         71 6 11         49 6 3            70 6 12 <br />    promoter Pmra contributes expression                                                     1        207 6 23        660 6 192         148 6 11 <br />                                                                                        JE7970(pHR431)              2        114 6 20        334 6 36           72 6 8 murG murC genes. examine contribution                                JE7970(pHR439)              2         88 6 2         361 6 28           59 6 10 <br /> Pmra expression genes mra cluster,                            JE7970(pHR427)              2        141 6 22         75 6 3           209 6 13 fusions proximal region mra cluster                            JE7970(pHR426)              2        132 6 15         70 6 5           217 6 16 promoterless galK9-9lacZ gene promoter assay                              JE7970(pHR416)              2        117 6 19         71 6 13          208 6 31 <br /> mini-F vector, pFZY1DH (Fig. 1). cloned DNA                                                                                                                            Cells grown exponentially 37&#194;&#176;C 2YT medium supplemented (1) HpaI site just upstream Pmra, EcoRV site                          supplemented (2) 1 mM IPTG. Crude extracts (total MraW 59 end murD (pHR571), SnaBI site                              FtsZ membrane fractions PBP3) prepared differential centrifu- <br /> middle ftsW (pHR577), SmaI site 59                            gation disrupting cells French press. Proteins fractionated end murG (pHR578). Pmra disrupted                                   Tricine&#226;&#8364;&#8220;sodium dodecyl sulfate&#226;&#8364;&#8220;8% acrylamide gel detected immuno- <br />                                                                                        blotting specific antibodies chemiluminiscence method. HindIII site filling reaction T4 DNA                            absorbances corresponding bands film different <br /> polymerase (Pmra::4bp; pHR586, pHR588, pHR589                                   experiments measured expressed relative spectively), shown completely abolish                            protein detected parental strain W3110. <br />  4410     MENGIN-LECREULX ET AL.                                                                                             J. BACTERIOL. <br />  <br />  enzymatic activity, measured. twofold           mainly  completely) dependent Pmra promoter. increase MraW sevenfold increase                particular, provided evidence repression evident PBP3, suggesting specific regulation          Pmra promoter resulted dramatic depletion murD genes level transcription translation. FtsZ        gene product, arrest peptidoglycan synthesis showed small (1.5-fold) increase presence              attendant cell lysis. suggested expression inducer. Levels gene products depletion              chromosomal murD gene exclusively dependent conditions induction IPTG agreement              promoter  consequently, putative promoters results obtained mur gene products                region 59 murD functional vivo present suggested transcription Pmra proceed ftsZ.          copy. Strains defective product ftsI <br />    revertants strains JE7967 JE7970          ftsW gene previously constructed (13). chloramphenicol resistant IPTG independent              case, depletion lac inducer resulted cell division <br /> isolated simply growing cells absence IPTG           defect, corrected addition plasmid car- <br /> stationary phase plating 2YT-chloramphenicol                rying defective gene. Strains conditionally defective <br /> medium. Expression genes strains             expression murE, murF, mraY gene IPTG independent, levels proteins            constructed, likely phenomena observed quite similar absence presence inducer (data           strains confirmed genes essential shown). result indicated defect rever-         exclusively expressed Pmra promoter. <br /> tant strains modification lac promoter                confirmed Pmra promoter absolutely operator did allow correct binding ex-           required expression genes distal ftsW mra <br /> pression LacI. changes strains,           cluster.  clear major proportion confirmed expression Pmra::Plac promoter              transcription genes originated promot- <br /> proceed ftsZ.                                                     er, murG murC genes, expression <br />    Expression gene products studied        reduced  sixfold, respectively, repression <br /> Pmra::Plac strain carrying plasmids different fragments         Pmra. residual expression gene products sug- mra cluster. results presented Table 3. MraW             gested presence efficient promoter. shown expressed level strains.       lacZ fusion experiments described work, pro- small increase expression (15 30%) extracts        moter EcoRV site lying  JE7970(pHR431), JE7970(pHR427), JE7970(pHR426),                  murD SnaBI site ftsW. ftsW gene JE7970(pHR416) cells, small decrease expression           essentially dependent Pmra promoter expression, 10%) evident JE7970(pHR439) difficult           promoter probably resides sequence correlate presence plasmids.  expres-           ftsW, future work devoted identification. sion PBP3 FtsZ showed opposite variations               noteworthy genes mra cluster strains. strains JE7970(pHR431) JE7970(pHR439), PBP3             tightly packed 100% DNA increased  fourfold FtsZ decreased              chromosomal region (EMBL entry EC2MIN) coding. 30 40%. Inversely, strains JE7970(pHR427), JE7970              particular, murG gene, coding N-acetylglucos- <br /> (pHR426), JE7970(pHR416), FtsZ increased twofold                aminyltransferase, shown overlap preceding ftsW gene level PBP3 reduced 25 30%. main                4 bases (15, 23). gene cluster, murC, <br /> difference plasmids presence absence          separated murG 100 bases (14), ftsA-encoding fragment main ftsZ promoters            tempting consider murG gene cotranscribed <br /> previously identified (4, 9, 29, 37).                                   exclusively Pmra, preceding genes. <br />                                                                         case, assumed residual transcrip- <br />                           DISCUSSION                                    tion occurred IPTG deprivation Pmra::Plac <br />                                                                         strain sufficient provide MurG molecules    Previous data transcription genes mra             support normal cell growth.  discussed previously <br /> cluster mainly concerned distal  including cell division       (13), basal level expression genes depending genes ftsQ, ftsA, ftsZ envA (2, 4, 9, 11, 29, 32, 37).   Pmra promoter unlikely sufficient absence mur genes proximal region expressed                IPTG, considering presence lacIq strain known recent demonstration pro-              addition glucose growth medium used, moter Pmra, originally identified required ex-         lead maximal repression promoter. fact pression ftsI, directs expression downstream        JE7970(pHR479) cells contained  sixfold-lower levels <br /> genes (13). Earlier suggested existence pro-          MurG MurC enzymes consistent moters ftsI-murC region (22), based observation           promoter contributing expression. multicopy plasmids (pUC18 derivatives) individual                conditions IPTG induction Pmra::Plac <br /> genes cloned orientation opposite vector             promoter strain JE7970, levels MraW PBP3 proteins <br /> promoter fully complemented specific defects thermo-             increased largely different factors,  sevenfold,  <br /> sensitive mutant strains allowed significant overproduction         spectively. effect observed pro- corresponding enzymes. observed particular           teins. simple explanation greater stability PBP3 murE, murD, murG, murC genes. exception                accounts differential expression unlikely concerned murF, expression apparently required                turnover PBP3 shown high (unpublished sequence upstream gene murE, suggesting               observation).  data suggest specific regulation <br /> cotranscription occur case (22).            expression genes level transcription cases, conceivable cumulative effect        translation. sense, weak promoters identi- <br /> weak promoters high copy numbers transcriptional read-            fied genes mra cluster (12), promoter plasmid vector resulted            involvement differential expression sufficient expression functional complementation.         proteins remains established. <br /> present work, expression genes shown             observed specific activity Ddl  <br />  VOL. 180, 1998                                                      EXPRESSION GENES E. COLI mra CLUSTER                                     4411 <br />  <br />  <br /> creased 30% Pmra::Plac cells deprived IPTG                                                REFERENCES <br /> versely, increased 30% induced cells. levels          1. Ayala, J. , T. Garrido, M.  Pedro, M. Vicente. 1994. Molecular <br /> variation low compared enzymes encoded                biology bacterial septation. New Compr. Biochem. 27:73&#226;&#8364;&#8220;101. <br />                                                                          2. Beall, B., J. Lutkenhaus. 1987. Sequence analysis, transcriptional orga- genes mra cluster. showed expression              nization, insertional mutagenesis envA gene Escherichia coli. J. ddlB   dependent Pmra promoter,              Bacteriol. 169:5408&#226;&#8364;&#8220;5415. demonstrated genes murG murC.                      3. Boyle, D. V., M. M. Khattar, S. G. Addinall, J. Lutkenhaus, W. D. noted genes, ddlA ddlB, ligases              Donachie. 1997. ftsW essential cell-division gene Escherichia coli. <br />                                                                             Mol. Microbiol. 24:1263&#226;&#8364;&#8220;1273. identical kinetic properties identified              4. Dai, K., J. Lutkenhaus. 1991. ftsZ essential cell division gene E. coli (38), theoretically contributing Ddl activity           Escherichia coli. J. Bacteriol. 173:3500&#226;&#8364;&#8220;3506. <br /> determined cell extracts. ddlA gene belongs            5. Dai, K., J. Lutkenhaus. 1992. proper ratio FtsZ FtsA completely separate chromosomal region, variations ob-                  required cell division occur Escherichia coli. J. Bacteriol. 174:6145&#226;&#8364;&#8220; <br />                                                                             6151. <br /> served likely represented specific contribution            6. Dewar, S. J., K. J. Begg, W. D. Donachie. 1992. Inhibition cell division DdlB enzyme total activity. null mutation             initiation imbalance ratio FtsA FtsZ. J. Bacteriol. 174: ddlA ddlB apparent effect cell growth             6314&#226;&#8364;&#8220;6316. <br /> morphology, laboratory growth conditions,             7. Donachie, W. D. 1993. cell cycle Escherichia coli. Annu. Rev. Micro- <br />                                                                             biol. 47:199&#226;&#8364;&#8220;230. <br /> reason duplication remains unclear.                             8. Fellay, R., J. Fray, H. Krisch. 1987. Interposon mutagenesis soil    levels FtsZ, essential cell division protein               water bacteria: family DNA fragments designed vitro insertional <br /> coded penultimate gene cluster, varied               mutagenesis Gram-negative bacteria. Gene 52:147&#226;&#8364;&#8220;154. <br /> repression induction Pmra promoter. variations           9. Flardh, K., T. Garrido, M. Vicente. 1997. Contribution individual <br />                                                                             promoters ddlB-ftsZ region transcription essential cell- <br /> paralleled mur gene products, levels            division gene ftsZ Escherichia coli. Mol. Microbiol. 24:927&#226;&#8364;&#8220;936. <br /> comparatively lower (Table 3). suggested portion         10. Fletcher, G., C.  Irwin, J. M. Henson, C. Fillingim, M. M. Malone, transcription genes cluster derived             J. R. Walker. 1978. Identification Escherichia coli cell division gene sep Pmra promoter. Dai Lutkenhaus (4) previously                   organization cell division-cell envelope genes sep-mur-ftsA- <br />                                                                             envA cluster determined specialized transducing lambda bacterio- <br /> reported strain null allele ftsZ chromo-            phages. J. Bacteriol. 133:91&#226;&#8364;&#8220;100. complemented lambda phage (l16-2)                11. Garrido, T., M. Sanchez, P. Palacios, M. Aldea, M. Vicente. 1993. <br /> carrying 6 kb DNA upstream ftsZ, including ftsA               Transcription ftsZ oscillates cell cycle Escherichia coli. <br /> ddlB genes, promoters ftsZ                EMBO J. 12:3957&#226;&#8364;&#8220;3965. <br />                                                                         12. &#194;&#180;mez, M. J. 1991. Doctoral thesis. Universidad Auto      &#194;&#180;noma Madrid, <br /> identified (9, 29, 37). suggested involvement                 Madrid, Spain. <br /> far-upstream promoter, contribution ftsZ expression            13. Hara, H., S. Yasuda, K. Horiuchi, J. T. Park. 1997. promoter estimated 30 40% (4). Pmra participates                  genes Escherichia coli mra cluster cell division cell apparently essential ftsZ expression,              envelope biosynthesis genes, including ftsI ftsW. J. Bacteriol. 179:5802&#226;&#8364;&#8220; <br />                                                                             5811. <br /> Pmra::Plac strain carrying region ftsW plas-        14. Ikeda, M., M. Wachi, H. K. Jung, F. Ishino, M. Matsuhashi. 1990. <br /> mid grew absence IPTG. 59 end insert              Nucleotide sequence involving murG murC mra gene cluster E. <br /> present l16-2 contained ftsZ failed complement               coli. Nucleic Acids Res. 18:4014. null allele ftsZ middle ftsW gene (3, 15).       15. Ishino, F., H. K. Jung, M. Ikeda, M. Doi, M. Wachi, M. Matsuhashi. <br />                                                                             1989. New mutations fts-36, lts-33, ftsW clustered mra region considering data obtained lacZ fusions,              Escherichia coli chromosome induce thermosensitive cell growth divi- assumed promoter, responsible                 sion. J. Bacteriol. 171:5523&#226;&#8364;&#8220;5530. <br /> portion expression genes murG ftsZ,               16. Jung, H. K., F. Ishino, M. Matsuhashi. 1989. Inhibition growth present upstream SnaBI site sequence ftsW.               ftsQ, ftsA, ftsZ mutant cells Escherichia coli amplification                                                                             chromosomal region encompassing closely aligned cell division cell shown  levels PBP3 FtsZ proteins showed                 growth genes. J. Bacteriol. 171:6379&#226;&#8364;&#8220;6382. <br /> opposite variations Pmra::Plac strain carrying plasmids          17. Koop,  H., M. E. Hartley, S. Bourgeois. 1987. low-copy-number different fragments mra cluster. main differ-               vector utilizing b-galactosidase analysis gene control elements. <br /> ence plasmids presence absence                    Gene 52:245&#226;&#8364;&#8220;256. <br />                                                                         18. Lowry, O. H., N. J. Rosebrough,  L. Farr, R. J. Randall. 1951. Protein <br /> fragment carrying ftsA main promoters ftsZ.                 measurement Folin phenol reagent. J. Biol. Chem. 193:265&#226;&#8364;&#8220;275. propose expression FtsA absence Pmra           19. Matsuhashi, M., M. Wachi, F. Ishino. 1990. Machinery cell growth <br /> promoter elicits regulatory mechanism involves repres-               division: penicillin-binding proteins proteins. Res. Microbiol. <br /> sion specific promoter PBP3 FtsA. regard,             141:89&#226;&#8364;&#8220;103. <br />                                                                         20. Mengin-Lecreulx, D., B. Flouret, J. van Heijenoort. 1982. Cytoplasmic previously shown amplification fragment cor-               steps peptidoglycan synthesis Escherichia coli. J. Bacteriol. 151:1109&#226;&#8364;&#8220; <br /> responding genes cluster modified plating           1117. <br /> capacity thermosensitive ftsA mutant restrictive            21. Mengin-Lecreulx, D., B. Flouret, J. van Heijenoort. 1983. Pool levels temperature (16) elevated levels FtsA protein              UDP-N-acetylglucosamine UDP-N-acetylglucosamine-enolpyruvate                                                                             Escherichia coli correlation peptidoglycan synthesis. J. Bacteriol. <br /> blocked cell division early stage (5, 6). imbalance              154:1284&#226;&#8364;&#8220;1290. FtsA FtsZ similarly affect expression            22. Mengin-Lecreulx, D., C. Parquet, L. R. Desviat, J. Pla      &#194;&#180;, B. Flouret, J.  genes cluster, including ftsI.                             Ayala, J. van Heijenoort. 1989. Organization murE-murG region <br />                                                                             Escherichia coli: identification murD gene encoding D-glutamic- <br />                                                                             acid-adding enzyme. J. Bacteriol. 171:6126&#226;&#8364;&#8220;6134. <br />                                                                         23. Mengin-Lecreulx, D., L. Texier, M. Rousseau, J. van Heijenoort. 1991. <br />                      ACKNOWLEDGMENTS                                        murG gene Escherichia coli codes UDP-N-acetylglucosamine: <br />                                                                             N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetyl- <br />    work supported grant Centre National la        glucosamine transferase involved membrane steps peptidoglycan <br /> Recherche Scientifique (URA 1131) grant &#226;&#8364;&#339;Biotechnologies&#226;&#8364;?             synthesis. J. Bacteriol. 173:4625&#226;&#8364;&#8220;4636. Ministe l&#226;&#8364;&#8482;Education Nationale la Recherche et la   24. Mengin-Lecreulx, D., J. van Heijenoort. 1985. Effect growth condi- <br /> Technologie   97.C.0177), France. Work J &#226;&#8364;&#8482;s laboratory         tions peptidoglycan content cytoplasmic steps biosynthesis                                                                             Escherichia coli. J. Bacteriol. 163:208&#226;&#8364;&#8220;212. supported grant BI094-0789 Comision Interministerial <br />                                                                         25. Mengin-Lecreulx, D., J. van Heijenoort. 1993. Identification glmU Ciencia y Tecnologia, Spain. J  acknowledges institutional         gene encoding N-acetylglucosamine-1-phosphate uridyltransferase Esch- <br /> help Fundacio&#194;&#180;n Ramo &#194;&#180;n Areces Centro Biolog&#196;&#177; Mo-        erichia coli. J. Bacteriol. 175:6150&#226;&#8364;&#8220;6157. <br /> lecular Severo Ochoa.                                                   26. Miller, J. H. 1992. short course bacterial genetics: laboratory manual <br />    than <br /> </body></html>