<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                Rad53 Phosphorylation Site Clusters Important                          Rad53 Regulation Signaling <br />  <br />             Budding yeast Rad53 essential protein kinase phosphorylated activated MEC1-           TEL1-dependent manner response DNA damage. studied role Rad53 phosphorylation           mutation consensus phosphorylation sites upstream kinases Mec1 Tel1. Alanine substitution           Rad53 amino-terminal TQ cluster region reduced viability impaired checkpoint functions. substi- <br />           tution mutations spared basal interaction Asf1 DNA damage-induced interactions Rad9. <br />            caused decrease DNA damage-induced Rad53 kinase activity impaired interaction <br />           protein kinase Dun1. Dun1 FHA (Forkhead-associated) domain recognized amino-terminal TQ <br />           cluster Rad53 DNA damage replication blockade.  phosphorylation Rad53 upstream <br />           kinases important Rad53 activation creation interface Rad53 Dun1. <br />  <br />  <br />   Failure maintenance genomic integrity allows ac-                  minal BRCT domains phosphorylated checkpoint <br /> cumulation mutations promote carcinogenesis.                      activation include 53BP1 (6, 12, 64), MDC1 (21, 33, 46, 55, 56, <br /> Cells employ mechanisms ensure complete accu-                     68), BRCA1 (70). <br /> rate transmission genomic DNA daughter cells. ex-                      Activated Rad53 involved cell cycle arrest, transcrip- <br /> ample, DNA replication checkpoint pathways prevent cells                      tional induction repair genes, inhibition late replication incompletely replicated DNA entering mitosis,                   origin firing, stabilization stalled replication forks (10, <br /> DNA damage checkpoint pathways slow arrest cell cycle                  32, 49, 50, 61). conditions replicative stress, Rad53 DNA damage. Activation pathway results                      phosphorylates negatively regulates Dbf4/Cdc7 kinase, <br /> induction transcription genes required DNA repair                   required DNA replication origin firing (13, 14, 27, <br /> (43). Inherited mutations DNA checkpoint pathways                      67). Rad53 phosphorylates activates Dun1, FHA multiple cancer-predisposing disorders, exam-                   domain-containing kinase, DNA checkpoint activation <br /> ples including mutation ATM ataxia telangiectasia                (7, 10, 76), resulting increased expression activity TP53 CHK2 Li-Fraumeni syndrome (8, 35, 51).                            ribonucleotide reductase (26, 71&#226;&#8364;&#8220;74). <br />   DNA checkpoint pathways conserved yeasts                      activation Rad53  orthologs SpCds1 humans. conserved mechanisms include protein kinase                       Chk2) key steps DNA checkpoint pathways, <br /> cascades large <span id='am-20' about='obo:IMR_0001650' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-21' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001650" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="inositol phospholipid" datatype="xsd:string"></span>phosphoinositide</span> 3&#226;&#172;&#732;-kinase-like kinases               detailed mechanism Rad53 activation remains  <br /> (PIKKs), Mec1 Tel1 Saccharomyces cerevisiae, Rad3                   clear. Activation Rad53-like protein kinases occurs  <br /> Schizosaccharomyces pombe, ATM ATR mammals,                        comitantly phosphorylation (11, 45, 48, 58). Mutation <br /> phosphorylate effector kinases act parallel, ScChk1               genes thought act upstream DNA checkpoint <br /> ScRad53, SpChk1 SpCds1, mammalian Chk1                            pathways, including MEC1 RAD9, prevents phos- <br /> Chk2, respectively (40).                                                      phorylation Rad53 (11, 57, 65). Phosphorylated Rad53 <br />   budding yeast, signal initiated DNA damage                                                                               isolated cells activated DNA replication dam- <br /> conveyed effector kinases Rad53 Chk1                                                                               age checkpoint pathways associated elevated kinase <br /> Rad9 (note ScRad9 unrelated SpRad9 human <br />                                                                               activity, protein denaturation, separation Rad9) (47, 59). DNA damage, Rad9 undergoes MEC1- <br />                                                                               proteins sodium dodecyl sulfate (SDS)-polyacryl- TEL1-dependent phosphorylation enables binding                                                                               amide gels, renaturation (45). kinase-defective <br /> Rad53 Rad53 FHA (Forkhead-associated)  <br />                                                                               Rad53 partially phosphorylated DNA damage mains, modular phosphopeptide binding domains, <br />                                                                               replicative stress (45, 48, 58), Rad53 phosphor- occurs concomitantly binding Chk1 (16, 53, 59, 62). <br />                                                                               ylations occur trans. similar phenotypes associated <br /> Replicative stress activates Rad53 intermediary <br />                                                                               mutation PIKKs Mec1  lesser extent, <br /> distinct Rad9, possibly Mrc1 (4). DNA damage S phase <br />                                                                               Tel1 dependency Rad53 phosphorylation signal Rad9-dependent -independent path- <br /> ways (4, 42). mammals, single Rad9 homolog,                kinases suggested PIKKs directly phos- <br /> candidate orthologs similarly situated carboxyl-ter-                phorylate Rad53. Consistent idea, carboxyl-ter- <br />                                                                               minal fragment Rad53 phosphorylated vitro Mec1 <br />                                                                               (63).  Rad53, like protein kinases,    * Corresponding author. Mailing address: Department Pathology,          autophosphorylate, checkpoint activation Rad53 <br /> School Medicine, Yale University, 310 Cedar St., BML 342, New <br /> Haven, CT 06510. Phone: (203) 785-4832. Fax: (203) 785-7467. E-mail: <br />                                                                               likely involves PIKK-dependent transphosphorylation <br /> df.stern@yale.edu.                                                            intra- intermolecular autophosphorylation. <br />    &#226;&#8364;&#160; Present address: Wistar Institute, Philadelphia, PA 19104.              Critical phosphorylation sites identified  <br />                                                                        6300 <br />  VOL. 23, 2003                                                                                Rad53 PHOSPHORYLATION SITES                       6301 <br />  <br />  <br /> Rad53 orthologs SpCds1 mammalian Chk2, respective-               upstream kinases mediates direct interactions ly. Threonine 11 SpCds1 phosphorylated PIKK             Rad53 FHA domain Dun1. propose Mec1 <br /> SpRad3 required SpCds1 activation (60). ATM           Tel1 play important role catalytic activation <br /> ATR phosphorylate Chk2 partially overlapping sites        Rad53 create binding interface Rad53 <br /> located cluster PIKK consensus sites amino termi-        Dun1 phosphorylation amino-terminal TQ <br /> nal FHA domain. Mutation sites,           cluster. <br /> tably threonine 68, impedes Chk2 activation (3, 38, 39).  <br /> phosphorylation Rad53 Mec1 similar cluster                                MATERIALS METHODS <br /> crucial Rad53 activation function.  efforts          Plasmids strains. Wild-type kinase-defective RAD53 (rad53kd; <br /> activate Rad53 orthologs solely transphosphor-       rad53K227A,D339A) expressed control endogenous promoter <br /> ylation PIKKs unsuccessful (38).                        pRS316 pRS315, low-copy-number CEN plasmid, described previously <br />    hand, Rad53 autoactivate DNA           (18, 58). Expression regulation  phosphorylation) plasmid- <br />                                                                      encoded Rad53 comparable genomic RAD53. plasmids <br /> damage. Rad53 expressed high levels bacteria heavily        encoding alanine replacements Rad53 produced PCR-based mu- <br /> phosphorylated absence yeast proteins, presum-       tagenesis  primer PCR internal mutagenic primers). RAD53 open <br /> ably autophosphorylation (20). proposed previously         reading frames (ORFs) (wild type <span id='am-67' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-68' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-70' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> substitutions) ampli- phosphorylated Rad9 serves adaptor delivers          fied PCR primers introduced EcoRI SmaI sites 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends <br />                                                                      ORF. sequences inserted corresponding restriction sites <br /> Rad53 Mec1 (59), Mec1-dependent phosphorylation <br />                                                                      pBluescript SKII (&#226;&#171;&#185;) (pSK) 3xFLAG, contains sequences encoding Rad9 enables phosphorylated Rad9 bind Rad53.               carboxyl-terminal FLAG (3xFLAG) tag, yielding pSK RAD53 3xFLAG  Rad9 play alternative additional role         alleles. pET28b RAD53 3xFLAG plasmids constructed inserting centrating bound Rad53 (20). Binding Rad53 phosphor-           EcoRI-NotI fragment containing RAD53 3xFLAG ORF correspond- <br /> ylated Rad9 enable Rad53 cross-phosphorylation               ing sites pET28b. make pRS316 RAD53 3xFLAG, HpaI-SpeI fragment <br />                                                                      pSK RAD53 3xFLAG inserted sites pRS316 RAD53 <br /> activation. Work Chk2 suggests  activated, Rad53        13xMYC, contains SpeI site RAD53 13xMYC ORF 3&#226;&#172;&#732; activate naive Rad53 molecules PIKK-             untranslated region (UTR). Sequences encoding hemagglutinin (2xHA) tags RAD9-independent mechanism. DNA damage-activated                 introduced similarly 3&#226;&#172;&#732; end RAD53 produce pRS316 RAD53 <br /> phosphorylated Chk2 form homo-oligomers mediated              2xHA. <br />                                                                         pRS314 DUN1 13-MYC (13xMYC) tag (pRS314 DUN1 13xMYC) interaction phosphorylation sites [S/T]Q <br />                                                                      control endogenous DUN1 promoter described (54). <br /> cluster domain molecule FHA domain             Mutations FHA domain DUN1 (Dun1R60A/N103A; Dun1RN)  (2, 69). reconstructed vitro using bacteri-     troduced replacing NotI-StuI PCR fragments PCR products having <br /> ally expressed Chk2, autophosphorylates sites includ-       directed mutations conserved amino acids FHA domain. pGEX 4T3 <br /> ing threonine 68 (2, 69). Chk2 activated PIKKs,           GST-DUN1 FHA dun1 FHA domains mutations (residues 1 199; <br />                                                                      Dun1 FHAR60A/N103A; Dun1 FHARN) constructed follows. Mutagenic <br /> activated Chk2 bind inactive Chk2 FHA             primers used introduce EcoRI sites 5&#226;&#172;&#732; initiation codon DUN1 <br /> domain  phosphorylate, activate molecule.     PCR pRS314 DUN1 13xMYC (wild type FHA mutations) result transition PIKK-dependent PIKK-         templates. EcoRI-StuI PCR fragments cloned EcoRI-SmaI sites independent regulation Chk2.                                      pGEX 4T3. Plasmids encoding <span id='am-39' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-46' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutathione</span> S-transferase (GST) CHK2 FHA <br />                                                                      domain described previously (69). Plasmids GST RAD53 FHA1    observations suggest models                                                                      GST RAD53 FHA2 (BamHI-HpaI) described (53, 54). <br /> mechanism Rad53 activation.  phosphorylation                Plasmids carrying 18xMYC MEC1 pRS315 constructed replacing <br /> Mec1-Tel1 critical phosphorylation sites Rad53 ac-         SpeI-AatII fragments pRS315 MEC1 SpeI-AatII fragments amplified tivate Rad53 directly. case, Rad9 work adap-       PCR genomic DNA DMP3084/3C template. DMP3084/3C <br /> tor deliver Rad53 Mec1. second model, Rad53          strain, genomic Mec1 tagged 18xMYC, described previ- <br />                                                                      ously (44). pRS315 MEC1 containing &#226;&#172;&#402;800 bp 5&#226;&#172;&#732; UTR &#226;&#172;&#402;500 bp 3&#226;&#172;&#732; <br /> activated intermolecular autophosphorylation            UTR constructed recombination repair gapped plasmid. plasmids <br /> DNA damage. main role Mec1-Tel1 phos-             constructed PCRs sequenced tested expression function. <br /> phorylate Rad9, PIKKs required                 strains used W303 background: U960-5C (MATa ade2-1 activation Rad53. Rad9 participate scaf-       can1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1 sml1-1 rad53-XB::HIS3), yJKD 103 <br />                                                                      (MATa cdc13-1 cdc15-2 ade2-1 can1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1 <br /> fold concentrates Rad53. model phos- <br />                                                                      sml1&#226;&#338;&#172;::TRP1 rad53&#226;&#338;&#172;::HIS3), yJKD 201 (MATa cdc15-2 ade2-1 can1-100 his3- <br /> phorylation Rad53 Mec1-Tel1 required Rad53              11,15 leu2-3,112 trp1-1 ura1-3 sml1&#226;&#338;&#172;::TRP1 rad53&#226;&#338;&#172;::HIS3 dun1&#226;&#338;&#172;::Kanr), yJKD <br /> activation Rad9 required coactivator.               403 (MATa ade2-1 can1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1 sml1-1 <br />    Despite overall conservation DNA checkpoint pro-           rad53-XB::HIS3 ASF1-3xHA::Kanr), yJKD 415 (MATa ade2-1 can1-100 his3- <br /> teins, Rad53 unique FHA-containing protein kinases,         11,15 leu2-3,112 trp1-1 ura3-1 sml1-1 rad53-XB::HIS3 dun1&#226;&#338;&#172;::Kanr). DMP3084/ <br />                                                                      3C (MATa ade2-1 can1-100 his3-11,15 leu2-3,112 trp1-1 ura3 MEC1-MYC18:: <br /> including SpCds1 Chk2, having FHA domains            LEU2::med1 DDC2-HA3::URA3) U960-5C generously provided M. <br /> flank kinase catalytic domain, single amino-       Longhese R. Rothstein, respectively. Construction yeast strains  <br /> terminal FHA domain. cluster potential PIKK phosphor-           scribed (54). Strains containing rad53-XB::HIS3 deletion express ylation sites located amino terminal Rad53         residual carboxyl-terminal fragment Rad53. yJKD 103 yJKD 201                                                                      complete deletion RAD53. <br /> FHA domains. investigated role phosphorylation                                                                         yEF569HA-212 (MATa leu2 trp1 ura3 his3 mec1::TRP1 sml1-1 RAD9-3xHA <br /> Rad53 activation function vivo. clusters    pipo [pRS315 18xMYC-MEC1]) A364a background. yEF569, paren- <br /> PIKK sites Rad53 required checkpoint-        tal strain yEF569HA-212, generously provided E. Foss. DZ6-1 <br /> dependent phosphorylation Rad53 associated biological         (MATa/MAT&#226;&#65533;&#163; ura3-53/ura3-53 leu2-3,112/leu2-3,112 his3-&#226;&#338;&#172;200/his3-&#226;&#338;&#172;200 lys2- <br /> functions. mutation second cluster sites         801/&#226;&#171;&#185; ade4/&#226;&#171;&#185; rad53&#226;&#338;&#172;XB: &#226;&#171;&#185; [pRS316-RAD53]) DZ71 (MATa rad53&#226;&#338;&#172;XB:: <br />                                                                      [pRS316-RAD53]; segregant sporulation DZ6-1) previously <br /> mild effects, mutation cluster sites significantly   described (75). experiments performed different <br /> impaired Rad53 function activation. addition,        strains, consistent results observed. appropriate, strains phosphorylation Rad53 amino-terminal TQ cluster          transformed plasmids described  Yeast strains cdc13-1 <br />  6302       LEE ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  cdc15-2 grown 23&#194;&#176;C selectable medium YPAD (1% yeast               Viability checkpoint assays. genotoxin sensitivity assays, cells extract, 2% Bacto Peptone, 2% dextrose, 0.05% <span id='am-51' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span>). strains           diluted fivefold serially spotted YPAD plates containing various  grown 30&#194;&#176;C.                                                                 centrations HU MMS. Cells incubated 30&#194;&#176;C 2 days. <br />    Cell lysis, immunoprecipitation (IP), Western blot analysis. Cells treated      DNA replication checkpoint monitored measuring percentage 0.1% methyl methanesulfonate (MMS) 1 h 200 mM                elongated spindles visualization indirect immunofluorescence, hydroxyurea (HU) 2 h washed washing buffer (phosphate-buffered        previously described (5). Cells arrested G1 phase using &#226;&#65533;&#163;-factor <br /> saline [PBS], 10% glycerol, 1% Triton X-100). Cell pellets resuspended     (Sigma) 2 h 30 min. Cells washed times water 700 &#226;&#65533;&#174;l lysis buffer (PBS, 10% glycerol, 1% Triton X-100, 1 mM EDTA, 1%        released YPAD containing 200 mM HU. Elongated spindles counted <br /> aprotinin, 1 mM phenylmethylsulfonyl fluoride, 10 mM NaF, 20 mM &#226;&#65533;&#164;-glycero-          visualization indirect immunofluorescence, using rat antitubulin <br /> phosphate, 5 mM sodium vanadate, protease inhibitor cocktail [Roche]).          monoclonal antibody YOL1/34 fluorescein isothiocyanate-conjugated don- <br /> Cells mechanically disrupted presence zirconium beads (Biospec)      key anti-rat secondary antibody (Jackson Immunoresearch). mini-Bead Beater-8 (Biospec). extract clarified microcentrifuge      G2/M checkpoint function examined scoring cells arrest 4&#194;&#176;C 10 min. milligrams protein used <span id='am-6' about='obo:IMR_0100698' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001385 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="D-myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200368" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="1D-myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="1D-myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0100698" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="D-myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="IP2" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Inositol 1,4-bisphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C01220" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="myo-Inositol 1,4-bisphosphate" datatype="xsd:string"></span>IP 2</span>.5      anaphase response DNA damage described previously (19, 34, 53). <br /> &#226;&#65533;&#174;g purified antibody 40 &#226;&#65533;&#174;l protein G- plus protein agarose beads        Briefly, cdc13-1 cdc15-2 strains grown 23&#194;&#176;C arrested G1 phase (Oncogene). IP mixtures rotated 2 4 h 4&#194;&#176;C, centrifuged,          3 h &#226;&#65533;&#163;-factor YPAD. Cells washed extensively YPAD washed times washing buffer. SDS-polyacrylamide gel electrophoresis        shifted 37&#194;&#176;C. Samples collected 0, 3, 6 h temperature shift; <br /> loading buffer added sample, samples boiled 5 min.     fixed; stained DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phenylindole); counted sin- <br /> Proteins separated 6 5 15% acrylamide gradient gels trans-        gle double nuclei. <br /> ferred polyvinylidene difluoride membrane (Millipore). 1 h blocking      Pull experiments. Expression histidine (6xHis) Rad53 3xFLAG 5% milk TBST (20 mM Tris [pH 7.5], 150 mM NaCl, 0.1% Tween 20),             (wild type kinase defective) BL21(DE3)pLysS cells induced adding <br /> membranes incubated primary antibodies overnight secondary            1 mM IPTG (isopropyl-&#226;&#65533;&#164;-D-thiogalactopyranoside). 4 h induction, antibodies 1 h detect endogenous Rad53 Rad9. Antibodies Rad53       cells pelleted lysed sonication Tris lysis buffer (25 mM Tris [pH <br /> Rad9 affinity-purified polyclonal rabbit antibodies, previously described   7.5], 300 mM NaCl, 10 mM imidazole, 10% glycerol, 1% Triton X-100). <br /> (53, 54, 75). Phosphorylated forms Rad53 detected using rabbit poly-      centrifugation 4&#194;&#176;C, supernatant rotated 4 8 h 4&#194;&#176;C clonal antibody recognizes phosphorylated [S/T]Q motifs (Cell Signaling        Ni-nitrilotriacetic acid beads (Qiagen) 6xHis Rad53 3xFLAG. beads <br /> Technology). HA-, MYC-, FLAG-tagged proteins detected incubation         washed times Tris washing buffer (25 mM Tris [pH 7.5], 300 mM 1 h horseradish peroxidase (HRP)-conjugated antibodies (anti-HA-HRP        NaCl, 20 mM imidazole, 10% glycerol, 1% Triton X-100) eluted Roche, anti-MYC-HRP Santa Cruz, anti-FLAG-HRP Sigma).            imidazole elution buffer (25 mM Tris [pH 7.5], 300 mM NaCl, 250 mM imida- <br />    Kinase assays. situ autophosphorylation assay (ISA) performed      zole, 10% glycerol, 1% Triton X-100). Rad53 wild type dephosphory- <br /> described previously (45). Rad53 kinase assays immune complexes (IP          lated lambda phosphatase 1 h 30&#194;&#176;C repurified kinase assays) (58), cell lysates strains expressing FLAG-tagged Rad53         procedure described  Glycerol added 50%, final purified prepared using HEPES lysis buffer (25 mM HEPES [pH 7.5], 10% glycerol,         proteins stored &#226;&#171;&#186;20 &#226;&#171;&#186;80&#194;&#176;C. <br /> 0.1% Triton X-100, 1 mM EDTA, 1% aprotinin, 1 mM phenylmethylsulfonyl                  Expression GST FHA fusion proteins BL21 cells induced 1 mM <br /> fluoride, 10 mM NaF, 20 mM &#226;&#65533;&#164;-glycerophosphate, 5 mM sodium vanadate,            IPTG. cells pelleted 4 h lysed sonication bacterial PBS <br /> protease inhibitor cocktail [Roche]). milligrams protein incubated       lysis buffer (PBS, 10% glycerol, 1% Triton X-100 protease inhibitor 2 3 h mouse anti-FLAG antibody (Sigma) protein G- plus protein      cocktail [Roche]). Cell lysates clarified centrifugation 4&#194;&#176;C rotated agarose beads. Beads washed times kinase washing buffer (25       overnight 4&#194;&#176;C <span id='am-82' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span>glutathione</span> Sepharose beads (Amersham) purify GST <br /> mM HEPES [pH 7.5], 10% glycerol, 0.1% Triton X-100). IP kinase assays           fusion proteins. beads washed times resuspended bacterial carried ice 1 h kinase reaction buffer (25 mM HEPES [pH 7.5],    PBS lysis buffer. Glycerol added 50%, purified proteins 1 mM MnCl2, 1 mM MgCl2, 10 &#226;&#65533;&#174;Ci [&#226;&#65533;&#165;-32P]<span id='am-56' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-57' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>, 1 &#226;&#65533;&#174;M nonradioactive ATP)              beads stored &#226;&#171;&#186;20&#194;&#176;C. 4 &#226;&#65533;&#174;g histone H1 substrate reaction. Kinase reactions stopped       pull assay, 10 &#226;&#65533;&#174;g 6xHis Rad53 3xFLAG 20 &#226;&#65533;&#174;g GST FHA adding SDS-PAGE loading buffer. Samples resolved SDS&#226;&#8364;&#8220;5 15%            fusion protein incubated 4 h 2 mg yeast lysate. beads polyacrylamide gels. portion gel 50-kDa protein marker     washed times IP washing buffer, SDS-PAGE loading buffer cut, stained Coomassie blue visualize histone H1, dried, exposed     added. Approximately half sample used Western blot analysis. <br /> film. Proteins 50 kDa transferred polyvinylidene difluoride mem- <br /> brane (Millipore), exposed film autoradiography, immunoblotted Rad53 Rad9.                                                                                                       RESULTS <br />    Mec1 IP kinase assays performed similarly 18xMyc Mec1 immunoprecipitated using mouse anti-Myc antibody (Sigma) recombi-            Rad53 major phosphorylation site clusters vivo. <br /> nant Rad53kd 3xFLAG Rad53kd1-4/9-12AQ 3xFLAG used substrate.              Rad53 16 potential phosphorylation sites [S/T]Q <br />    Plasmid shuffle assay. pRS315 RAD53, wild type alanine mutations,        sequence favored phosphorylation PIKKs (numbered 1 introduced strain DZ71, carries RAD53 URA3 plasmid <br /> (pRS316) genomic rad53 deletion background. Cells diluted fivefold <br />                                                                                     16, Fig. 1A). include TQ cluster amino terminal serially, spotted plates containing 0.1% 5-fluoroorotic acid (5-FOA),      FHA1 situated similarly regulatory sites Chk2 Cds1 <br /> incubated 30&#194;&#176;C 2 4 days.                                                  (1-4 TQ sites; Fig. 1A), SQ cluster amino terminal  <br />  <br />  <br />  <br />    FIG. 1. Phosphorylation Rad53 [S/T]Q mutations.   Schematic diagram Rad53 showing [S/T]Q site. SCD, [S/T]Q cluster <br /> domain. (B) Western blot analysis Rad53, Rad9, Dun1 cells expressing kinase-active Rad53 combined alanine substitutions. Strains <br /> analyzed yJKD 415 background plus pRS314 DUN1 13xMYC pRS316 RAD53, wild type mutations. Rad53   Rad9 (middle) detected using polyclonal rabbit antibodies protein, Dun1 13xMYC   detected anti-Myc-HRP. Cells mock <br /> treated (lanes C) treated 200 mM HU 2 h (lanes H) 0.1% MMS 1 h (lanes M) prior analysis, trichloroacetic acid lysates prepared Western blotting. (C) Western blot analysis phosphorylated forms Rad53 RAD53, rad53kd, rad531-4/9-12AQ cells. <br /> pRS314 DUN1 13xMYC pRS316 RAD53 3xFLAG, wild type alanine substitution mutations, introduced strain yJKD 415. Cells treated way described panel B, Rad53 immunoprecipitated mouse anti-FLAG antibody. Phosphorylated forms Rad53 detected immunoblotting polyclonal rabbit anti-phosphorylated [S/T]Q antibody. (D) vitro phosphorylation Rad53 Mec1. Mec1 immunoprecipitated MMS-treated yEF569HA-212 cell extracts mouse anti-Myc antibody incubated 30&#194;&#176;C recombinant Rad53kd 3xFLAG Rad53kd1-4/9-12AQ 3xFLAG presence [&#226;&#65533;&#165;-32P]ATP. gel exposed film overnight autoradiography (middle). Mec1 recombinant Rad53 proteins detected Western blotting   respectively). (E) Western <br /> blot analysis Rad53 cells expressing kinase-defective Rad53 alanine substitutions. Shown strain U960-5C pRS316 RAD53 alleles marked. experiment performed described panel B. WT, wild type. <br />  VOL. 23, 2003   Rad53 PHOSPHORYLATION SITES   6303 <br />  6304     LEE ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br /> FHA2 (9-12 SQ sites; Fig. 1A). Phosphorylation Rad53                altered phosphorylation profiles Rad53 [S/T]Q site <br /> correlates elevated kinase activity (55), evaluated         mutants, examined Rad53 phosphorylation cells express- <br /> candidate regulatory site mutants alterations Rad53 phos-       ing kinase-defective Rad53 (rad53kd), 1-4 TQ 9-12 <br /> phorylation.                                                           SQ alanine substitution (rad53kd1-4AQ, rad53kd9-12AQ,    Cells wild-type mutated Rad53 exposed HU            rad53kd1-4/9-12AQ). Cluster mutations kinase-defective <br /> (DNA replication-blocking agent) MMS (alkylating agent).            Rad53 caused similar changes Rad53 phosphorylation  <br /> Consistent previous reports (58), Rad53 moderately            gardless Rad53 kinase activity. Combined TQ SQ clus- <br /> phosphorylated (moderate mobility shift) response HU             ter site mutations abolished slowly migrating forms hyperphosphorylated MMS treatment wild-type                Rad53 produced HU MMS treatment (Fig. 1E, lanes <br /> RAD53 strains (Fig. 1B, lanes 2 3 panel),             11 12). rad53kd rad53kd1-4AQ strains showed enhanced <br /> strains expressing kinase-defective Rad53 (rad53kd) showed             Rad53 phosphorylation HU treatment similar man- <br /> enhanced phosphorylation HU (Fig. 1B, lanes 17 18             ner (note basal forms Rad53 apparent Fig. panel). Rad53 hyperphosphorylation HU treat-               1E, lanes 2 5). rad53kd9-12AQ cells, Rad53 displayed <br /> ment rad53kd strain resulted amplifica-           reduced phosphorylation HU treatment, implying tion DNA damage signals presence replicative              phosphorylation 9-12 SQ sites responsible slow-mi- <br /> stress loss RAD53 function strain.           grating forms Rad53 rad53kd rad53kd1-4AQ strains. <br /> extent MMS-dependent mobility shift Rad53                 overall phosphorylation reduced significantly diminished rad531-4AQ  strain expressing          strains (Fig. 1B E). observations suggested, min- <br /> Rad53 alanine substitutions T5, T8, T12, T15; Fig.         imum, trans-phosphorylation upstream kinases occurs <br /> 1B, lane 9 panel) rad539-12AQ  strain express-       major clusters [S/T]Q sites Rad53 vivo ing Rad53 alanine substitutions S473, S480, S485,          vitro Rad53 autophosphorylation, possibly S489; Fig. 1B, lane 12 panel).  Rad53               1-4 TQ 9-12 SQ sites, required HU-dependent shift enhanced rad531-4AQ cells (Fig. 1B,          DNA checkpoint-induced phosphorylation Rad53 vivo. <br /> compare lanes 2 8; note basal form Rad53              amino-terminal TQ cluster important Rad53 <br /> decreased majority Rad53 hyperphosphor-            functions. investigated Rad53 function ylated), similarly rad53kd strains (Fig. 1B, lane 17     compromised substitutions cluster phosphorylation panel). significant change rad539-12AQ          sites. Rad53 essential gene, examined effect <br /> strains compared RAD53 strains (Fig. 1B, lane 11             alanine substitutions Rad53 essential function panel). MMS- HU-dependent mobility shifts                   plasmid shuffle assay. Strains wild-type Rad53 alanine <br /> Rad53 largely eliminated rad531-4/9-12AQ cells (Fig. 1B,       substitutions able grow 0.1% 5-FOA plates, lanes 14 15 panel), expressed Rad53              strains vector barely grew (Fig. 2A).  combined substitutions 1-4 TQ sites 9-12 SQ sites.           cells recovered 0.1% 5-FOA plates replated strains expressing Rad53 single double alanine substi-         YPAD plates, rad531-4/9-12AQ strains showed slightly slower <br /> tutions remaining [S/T]Q sites (sites 5, 6, 7, 8, 13, 14,   growth (data shown). concluded alanine substitu- <br /> 15-16, Fig. 1A), significant effects migration        tions Rad53 spare essential function. Rad53 comparison wild-type Rad53 HU MMS                  Rad53 phosphorylated activated DNA damage <br /> treatment (data shown).                                            replicative stress, Rad53 activation essential DNA <br />    HU- MMS-induced phosphorylation Rad53                    damage intra-S-phase checkpoint pathways. indica- <br /> detected using antibody recognizes phosphorylated              tion loss Rad53 function genotoxin treatment [S/T]Q sequences (PIKKs&#226;&#8364;&#8482; consensus phosphorylation sites)           alanine substitution mutants enhanced Rad53 phos- <br /> strains RAD53 rad53kd (Fig. 1C, lanes 5, 6, 8, 9)          phorylation rad531-4AQ cells HU treatment. Loss detected rad531-4/9-12AQ strains (Fig. 1C, lanes        Rad53 function (e.g., mutations eliminate kinase <br /> 11 12).                                                            activity) similarly enhances Rad53 phosphorylation (Fig. 1B, <br />    simplest explanation substitutions reduce mo-          compare lanes 2 17 panel), probably activation bility shift phosphorylation [S/T]Q sites          DNA damage checkpoint pathways S phase result eliminate vivo phosphorylation upstream PIKKs.              secondary DNA damage caused replication fork collapse examined vitro phosphorylation recombinant Rad53kd            improper late origin firing (32, 49, 50, 61). Consistent 3xFLAG Rad53kd1-4/9-12AQ 3xFLAG Mec1 immune                      conclusion, mutation Rad53 1-4 TQ cluster caused Rad9 <br /> complexes. Phosphorylation Rad53kd1-4/9-12AQ 3xFLAG              hyperphosphorylated associated Rad53 Mec1 significantly reduced compared recombi-            response HU (Fig. 1B, compare lanes 2 8 middle <br /> nant Rad53kd 3xFLAG (Fig. 1D, compare lanes 3 6                 panel), hallmark DNA damage checkpoint activation. <br /> middle panel), despite presence phosphor-            data indicate rad531-4AQ strain deficient ylatable amino acids, including consensus PIKK             response replicative stress. <br /> phosphorylation sites.                                                    examined effect alanine substitutions Rad53    Alternative explanations include possibility         viability cells spotted MMS- HU-containing 1-4 TQ sites 9-12 SQ sites regulatory sites           plates. assays conducted strains sml1-1 <br /> affect Rad53 autophosphorylation mutations           allele, bypasses essential requirement RAD53 affect localization protein          transcriptionally activate production ribonucleotide  <br /> targeted phosphorylation upstream kinases.            ductase S phase (71). rad531-4AQ strains highly sen- <br /> termine Rad53 autophosphorylation activity contributes             sitive HU moderately sensitive MMS treatment, <br />  VOL. 23, 2003                                                                                   Rad53 PHOSPHORYLATION SITES                  6305 <br />  <br />  <br />  <br />  <br />    FIG. 2. 1-4 TQ sites important Rad53 checkpoint function.   Viability cells expressing Rad53 alanine substitutions. DZ71 <br /> strains rad53 alleles derived pRS315 spotted 0.1% 5-FOA plates. Plates scored 3 days 30&#194;&#176;C. (B) Cells Rad53 <br /> mutated 1-4 TQ sites sensitive HU MMS. yJKD 415 strains pRS314 DUN1 13xMYC pRS316 RAD53 rad53 alleles used experiments (B C). Cells diluted serially spotted YPAD HU (50 mM) MMS (0.02%). Plates photographed 2 days 30&#194;&#176;C. (C) S/M checkpoint assay. G1-arrested cells released HU-containing YPAD medium. Cells collected fixed 2.5 h, elongated spindles scored visualization indirect immunofluorescence. Experiments performed times independently. Averages standard deviations graph representative experiment performed times. (D) G2/M checkpoint assay. cdc13-1 cdc15-2 cells (yJKD 103) exposed DNA damage shifted restrictive temperature 3 6 h release G1 arrest. time point, cells collected, fixed, stained DAPI visualize nuclear morphology. <br /> WT, wild type. <br />  <br />  rad539-12AQ strains resistant strains              lication mediated activation DNA <br /> wild-type RAD53 (Fig. 2B). rad531-4/9-12AQ cells                 checkpoint pathways, determined rad53 <br /> sensitive HU MMS rad531-4AQ cells.                   strains DNA checkpoint defects. replication <br /> Rad53 1-4 TQ sites important resistance HU                  checkpoint assay, cells synchronized G1 phase MMS, 9-12 SQ sites largely redundant                  &#226;&#65533;&#163;-factor released YPAD containing 200 mM HU. <br /> assays unless 1-4 TQ sites eliminated.                                 150 min, (90%) cells wild-type Rad53 <br />   RAD53-dependent resistance DNA damage rep-                  arrested large budded cells short spindles,  6306     LEE ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. 1-4 TQ sites work redundantly. yJKD 415 strains pRS314 DUN1 13xMYC pRS316 RAD53 rad53 alleles used experiments.   Viability test HU plates. experiment performed described Fig. 2B. (B) Western blot analysis indicated <br /> proteins. Cells mock treated (lanes C) treated HU (lanes H) MMS (lanes M) prior analysis. WT, wild type. <br />  <br />  <br />  <br /> approximately 33% rad531-4AQ cells elongated spindles,             resistance, add Rad53 proteins displayed HU- similar rad53 deletion strains (28%) (Fig. 2C). rad539-12AQ            MMS-dependent phosphorylation similar cells slightly impaired DNA replication check-               wild-type Rad53, e., smaller HU-dependent mobility <br /> point function.                                                           shift seen rad531-4AQ cells greater MMS- <br />    G2/M DNA damage checkpoint function assayed                     dependent mobility shift (Fig. 3B). observations suggest <br /> monitoring nuclear division DNA damage caused                individual phosphorylation sites Rad53 1-4 TQ <br /> temperature-sensitive cdc13-1 allele. Cells synchronized             cluster functionally redundant. &#226;&#65533;&#163;-factor released restrictive temperature (34).                Effects phosphorylation site mutations Rad53 kinase <br /> Cells escaping G2/M checkpoint arrest arrest instead              activity. Alteration 1-4 TQ phosphorylation sites caused <br /> telophase cdc15-2 mutation. RAD53 strains exhib-               defects Rad53 phosphorylation, DNA checkpoint functions, <br /> ited delayed nuclear division accumulated cells              viability response HU MMS. Loss [S/T]Q <br /> nucleus large budded cells, rad531-4AQ cells proceeded           clusters alter cellular responses impairing Rad53 ac- mitosis rate rad53 deletion cells (Fig. 2D).          tivation interference coupling Rad53  <br /> rad539-12AQ cells normal G2/M checkpoint function.              stream responses. <br /> results indicate Rad53 1-4 TQ sites important                determined contribution Rad53 [S/T]Q <br /> replication DNA damage checkpoint functions. Rad53                    cluster sites Rad53 kinase activity regulated activation. <br /> 9-12 SQ sites minor role replication checkpoint             used immune complex kinase assays measure autophos- <br /> arrest required G2/M checkpoint.                      phorylation transphosphorylation Rad53 isolated    1-4 TQ sites work redundantly Rad53 function geno-              strains Rad53 mutations (Fig. 4Aa Ab). wild-type <br /> toxin sensitivity. simultaneous alanine substitution               RAD53 cells, MMS, lesser extent HU, enhanced total <br /> Rad53 1-4 TQ sites resulted HU sensitivity. determine               Rad53 autophosphorylation transphosphorylation activity individual sites cluster important           (Fig. 4Aa Ab, lanes 2 3). HU MMS treat- Rad53 function, &#226;&#8364;&#339;add  alleles produced,               ment, rad531-4AQ cells retained kinase activity showed reg- individual threonine cluster restored.           ulated activation measured autophosphorylation histone <br /> individual reintroduction site restored viability HU           H1 phosphorylation trans (Fig. 4Aa Ab, lanes 5 6). <br /> (Fig. 3A), slight defect TQ site add              experiments, observed extent allele. Concomitant restoration partial complete HU            Rad53 autophosphorylation transphosphorylation  <br />  VOL. 23, 2003                                                                                Rad53 PHOSPHORYLATION SITES                6307 <br />  <br />  <br />  <br />  <br />    FIG. 4. Kinase activity Rad53 mutants.   Rad53 immune complex kinase assays. yJKD 415 strains pRS314 DUN1 13xMYC pRS316 RAD53 3xFLAG alleles used experiments. (Aa) trans-phosphorylation activity Rad53 histone substrate. Rad53 immunoprecipitated antibody FLAG cells treated HU MMS incubated ice histone H1 presence [&#226;&#65533;&#165;-32P]ATP. Gels exposed film overnight  . Histone H1 visualized Coomassie blue staining SDS&#226;&#8364;&#8220;5 15% <br /> PAGE  . (Ab) Autophosphorylation activity Rad53.   Autoradiogram Rad53 autophosphorylation activity;   Western <br /> blot Rad53. (Ac) Rad9, coimmunoprecipitated Rad53 phosphorylated Rad53 immune complexes.   Autoradiogram Rad9 <br /> phosphorylated Rad53;   Rad9 Western blot IP anti-Rad53 antibodies. Cells mock treated (lanes C) treated HU <br /> (lanes H) MMS (lanes M) prior analysis. (B) Rad53 ISA. Experiments performed described Fig. 1B strains. Rad53 detected Western blotting following ISA. WT, wild type. <br />  <br />  <br />  <br /> tone H1 partially decreased mutation TQ cluster            lanes 8 9), slightly different results relative wild-type Rad53. reduction kinase             IP kinase assays. possible 1-4 TQ sites impor- <br /> activity rad531-4AQ cells significant response         tant phosphorylation sites ISA, 1-4 substitutions im- <br /> MMS (Fig. 4Aa Ab, compare lanes 3 6).                      pair refolding situ, proteins present immune <br /> rad531-4AQ cells displayed hypershifted forms Rad53              complexes account difference.  results <br /> HU treatment, possibly owing strong phosphorylation                indicate combined phosphorylation 1-4 TQ 9-12 SQ <br /> 9-12 SQ sites, possible phosphorylation 9-12 SQ            sites required Rad53 activation,  mu- <br /> sites partially compensate absence phosphory-             tation 1-4 TQ sites greater effect Rad53 activation. <br /> lation 1-4 TQ sites Rad53 activation.                                 Rad53 1-4 TQ mutants interact Asf1 Rad9.    rad539-12AQ cells retained comparable levels Rad53 kinase          investigated Rad53 [S/T]Q cluster mutations <br /> activation showed increased kinase activity HU treat-            interfere binding proteins Rad53. DNA dam- <br /> ment (Fig. 4Aa Ab, lanes 8 9). rad531-4/9-12AQ cells             age induces interaction Rad53 Rad9, required <br /> displayed significant decreases kinase activation,           couple Rad53 damage response (53, 59, 62). residual regulation HU MMS retained (Fig.                MMS-induced Rad9 phosphorylation coimmunoprecipita- <br /> 4Aa Ab, lanes 11 12).                                            tion Rad53 largely normal rad531-4AQ, rad539-12AQ, <br />    Similar results obtained using ISA measure Rad53              rad531-4/9-12AQ cells (Fig. 4Ac). HU treatment, <br /> activation (Fig. 4B). assays measure Rad53 phosphory-              Rad53 interacted Rad9 rad531-4AQ rad531-4/9-12AQ <br /> lation denaturing gel electrophoresis renaturation          cells, lesser extent rad539-12AQ cells, Rad53 situ. assay Rad53 activation severely impaired             [S/T]Q mutations caused secondary DNA damage  1-4 alanine substitutions (Fig. 4B, compare lanes 2 3           fective checkpoint functions. addition, Rad9 coimmunopre- <br />  6308     LEE ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. 1-4 TQ sites required interaction Rad53 Dun1. Alanine substitutions disrupt basal interaction Rad53 Asf1.   Asf1 coimmunoprecipitated Rad53. yJKD 403 strains transformed rad53 alleles derived pRS316. Rad53 immunoprecipitated polyclonal rabbit antibody Rad53, copurified Asf1 detected using mouse anti-HA antibody conju- <br /> gated HRP. (B) Rad53 coimmunoprecipitated Dun1. pRS314 DUN1 13xMYC pRS316 RAD53 3xFLAG mutations intro- <br /> duced yJKD 415. Dun1 immunoprecipitated mouse anti-Myc antibody, Rad53 detected using mouse anti-FLAG antibody <br /> conjugated HRP. Cells mock treated (lanes C) treated HU (lanes H) MMS (lanes M) prior analysis. WT, wild type. <br />  <br />  <br />  <br /> cipitated Rad53 phosphorylated Rad53 immune                rad531-4AQ (Fig. 1B, lanes 8 9). Deletion DUN1 slightly <br /> complex kinase assays rad531-4AQ rad539-12AQ strains,           decreased viability HU treatment sml1-1  reduced phosphorylation rad531-4/9-12AQ cells (Fig.            ground, suppresses DNA damage sensitivity 4Ac), consistent results assays (Fig. 4Aa Ab).       deletion DUN1, severe rad531-4AQ <br />    Rad53 interacts chromatin assembly factor Asf1          cells (Fig. 2B). Restoration single TQ sites Rad53 1-4 absence DNA damage (17, 25). basal interaction              TQ cluster partially restored phosphorylation regulated dissociation Rad53 Asf1              Dun1, suggesting single-site phosphorylation por- <br /> greatly impaired cells Rad53 alanine substitutions (Fig.       tion Rad53 sufficient induce Dun1 phosphor- <br /> 5A). results indicate Rad53 [S/T]Q clusters         ylation (Fig. 3B). results interpreted mean required interaction Rad53 Rad9 Asf1.           Rad53 1-4 TQ mutations does vivo <br />    1-4 TQ site mutations Rad53 disrupt interaction          kinase activity mutations cause disruption Rad53 Dun1. Dun1 protein serine/threonine kinase              DNA damage-induced interaction Dun1 Rad53. single FHA domain (7, 76). Dun1 phosphorylated               Rad53 kinase partially activated rad531-4AQ RAD53-dependent manner DNA damage replica-                  cells (Fig. 4A), examined interactions Dun1 tion blockade forms complex phosphorylated Rad53            Rad53. interaction largely eliminated rad531-4AQ <br /> (24). Mutation FHA domain Dun1 reduces DNA                   cells (Fig. 5B, lanes 8 9) unaffected rad539-12AQ <br /> damage-dependent Dun1 phosphorylation impairs check-               cells (Fig. 5B, lanes 11 12). rad531-4/9-12AQ strains, HU- <br /> point functions survival presence HU MMS              MMS-induced interactions Rad53 Dun1 <br /> (7). Dun1 allows activation transcriptional targets          attenuated (Fig. 5B, lanes 14 15). results sug- <br /> DNA damage response pathway phosphorylation inhi-               gest phosphorylation Rad53 1-4 TQ sites bition transcriptional repressor Crt1 (26). Dun1           required coupling Dun1 Rad53. <br /> phosphorylates Sml1, inhibitor large subunit ri-            Dun1 FHA domain interacts phosphorylated 1-4 TQ <br /> bonucleotide reductase, targeting degradation (71, 74).         cluster Rad53 Rad53 kinase-independent manner.  <br />    Dun1 likely vivo substrate Rad53, mea-          cause Dun1 interacts Rad53 phosphorylation-depen- <br /> sured Dun1 phosphorylation indicator vivo Rad53            dent manner (24), FHA domains generally recognize phos- <br /> kinase activity. HU- MMS-induced phosphorylation                phorylated proteins, determined FHA domain Dun1 normal rad539-12AQ strains (Fig. 1B, lanes 11              Dun1 interacts phosphorylated 1-4 TQ cluster region 12).  Dun1 phosphorylation inhibited                Rad53. <br />  VOL. 23, 2003                                                                              Rad53 PHOSPHORYLATION SITES               6309 <br />  <br />  <br />  <br />  <br />   FIG. 6. Binding Rad53 mutants Dun1 FHA domains. yJKD 415   yJKD 201 (B) pRS314 DUN1 13xMYC pRS316 RAD53 <br /> 3xFLAG used.   Rad53 coimmunoprecipitated Dun1. (B) GST-Dun1 FHA Dun1 FHA mutation purified bacteria. micrograms GST fusion proteins used pull Rad53 2 mg yeast lysate. GST fusion proteins visualized Coomassie blue staining. Cells mock treated (lanes C) treated HU (lanes H) MMS (lanes M) prior analysis. WT, wild type. <br />  <br />  <br />    mutation Rad53 1-4 TQ sites inactivating            induce interaction (Fig. 6A, lanes 9 10, 6B, lanes <br /> <span id='am-75' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-76' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-81' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> substitution Dun1 FHA domain disrupted               13 15).  PIKKs required Rad53 <br /> HU-dependent interaction Rad53 Dun1 (Fig. 6A,              activation creation interface joining Rad53 <br /> lanes 8 16). Furthermore, GST-Dun1 FHA domain fusion               Dun1. increased association kinase-defective Rad53 <br /> proteins expressed bacteria sufficient pull            Dun1 reflect secondary activation DNA dam- <br /> Rad53 HU- MMS-treated cell lysates                age checkpoint pathways result loss Rad53 func- <br /> control cell lysates (Fig. 6B, lanes 7 9). GST-Dun1 FHA           tion. <br /> fusion protein inactivating RN mutation (Dun1                    Specificity sequence context dependence FHA  <br /> FHARN; Dun1 FHAR60A/N103A) did react phosphor-                mains. Chk2, mammalian homolog Rad53, oligomerizes <br /> ylated Rad53 (Fig. 6B, lanes 4 6), confirming FHA          DNA damage Chk2 phosphorylation-dependent <br /> domain Dun1 responsible interaction Dun1             manner involves FHA domain Chk2 molecule phosphorylated Rad53. precipitation Rad53            interacting phosphorylated [S/T]Q cluster GST-Dun1 FHA domain fusion protein reduced sub-                 Chk2 molecule (2, 69). Similarly, isolated carboxyl-termi- <br /> stitution Rad53 1-4 TQ cluster (Fig. 6B, lanes 10 12).       nal fragment Rad53 (beginning residue 432)  formally possible 1-4 TQ region harbors         hybrid screen using Rad53 bait   Sun D. Stern,  <br /> basal inhibitory domain keeps Rad53 interacting              published data) observed Rad53 FHA1 preferentially Dun1 prior checkpoint activation, determined                interacts phosphorylated Rad53 compared Rad53 deletion 1-4 TQ sites restore interaction         FHA2 (Fig. 7A) (54). observed endogenous Rad53 Rad53 Dun1 HU treatment. Rad53                   yeast interacted dephosphorylated recombinant <br /> deletion mutant did interact efficiently Dun1 vivo         Rad53 bacteria MMS treatment (Fig. 7B). <br /> (Fig. 6A, lane 14), GST-Dun1 FHA proteins (data              determine Rad53 FHA1, like Chk2 FHA domain, <br /> shown), suggesting 1-4 TQ sites directly involved      interacts PIKK phosphorylation site cluster,  <br /> interaction Dun1 Rad53.                                        formed pull assays GST-Rad53 FHA1 domain <br />    Interestingly, interaction Dun1 Rad53 does          (Fig. 7A). Rad53 FHA1 domain bound Rad53 require Rad53 kinase activity, implying phosphoryla-          HU-treated MMS-treated cells control cells <br /> tion kinases, possibly Mec1 Tel1, sufficient        (Fig. 7A, lanes 8 10). 1-4 TQ substitution Rad53 <br />  6310   LEE ET AL.   MOL. CELL. BIOL. <br />  VOL. 23, 2003                                                                                 Rad53 PHOSPHORYLATION SITES                6311 <br />  <br />  <br /> slightly impaired isolation Rad53 HU- MMS-               Rad53 FHA1 Dun1 FHA.  vitro <br /> treated cells FHA1 domain, affecting inter-           studies <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> libraries shown FHA domains <br /> action phosphorylated Rad9 cell extracts              prefer phosphorylated threonine serine. alternative ex- <br /> (Fig. 7D, lanes 2 6). Mutation Rad53 9-12 SQ sites            planation rad539-12AQ cause phosphorylation minor effect, HU treatment, increased binding              remaining TQ sites, creating binding sites FHA2 Rad53 FHA1 Rad53, despite substantial contribution             Chk2 FHA domains. sites MMS-dependent Rad53 mobility shift (Fig. 7C,                 interactions GST fusion FHA domains compare lanes 11 13; Fig. 7D, lane 10). Double mutation              Rad53 Rad53 phosphorylation dependent,  1-4 TQ 9-12 SQ sites inhibited interaction            teractions greatly diminished rad531-4/9-12AQ cells (Fig. <br /> Rad53 Rad53 FHA1 domain (Fig. 7D, lane 14). Taken                7D, lanes 14 16 24). Interactions FHA domain  data suggest Rad53 1-4 TQ sites             occurred kinase-defective Rad53 (Fig. 7D, lanes 17 important regulated Rad53-Rad53 interaction, probably                19 25), phosphorylation kinases sufficient direct binding FHA domain phospho-                induce interactions. Overall, observations peptide, minor contribution 9-12 SQ sites.               FHA domains recognize phosphorylated threonines,    binding specificity FHA domains studied vitro           possibly serines, sequence context vivo FHA <br /> using phosphopeptide libraries (9, 15, 23, 66),        domains discriminate DNA checkpoint proteins vivo binding sites known FHA domain proteins (53, 69).            sites phosphorylated vivo. Rad53 FHA1 Dun1 FHA apparently recognize phos- <br /> phorylated threonine residues 1-4 TQ cluster.  <br />                                                                                                    DISCUSSION <br /> trast, FHA2 similar ability recognize phosphorylated <br /> Rad9 weakly detected hyperphosphorylated Rad53                     conducted mutational analysis determine func- <br /> MMS-treated wild-type cell lysates (Fig. 7A, lane 13, 7D,            tion consensus PIKK phosphorylation sites Rad53. <br /> lane 3) (54). relatively weak binding Rad53 FHA2               Rad53 amino-terminal 1-4 TQ cluster  <br /> Rad53 barely affected 9-12 SQ substitution dou-            gion important Rad53 function. Alanine substitution <br /> ble substitution Rad53 [S/T]Q clusters slightly           mutations sites caused decreased viability HU <br /> increased 1-4 TQ cluster substitution HU treatment (Fig.           treatment impaired replication blockade DNA dam- <br /> 7D, lanes 3, 7, 11, 15, 18). results demonstrate               age checkpoint function Rad53. mutations caused site-specific binding preferences FHA domains vivo             reduction DNA damage-induced Rad53 kinase activation  unlike Rad53 FHA1 Dun1 FHA, Rad53 FHA2                    impaired interactions Rad53 Dun1 discriminates vivo phosphorylated Rad9 phos-              DNA damage replication blockade. Phosphorylated Rad53 <br /> phorylated Rad53.                                                        1-4 TQ sites directly recognized FHA domain    compare interactions phosphorylated                Dun1, regardless Rad53 kinase activity.  Mec1 proteins FHA domains different proteins,                 Tel1 important Rad53 activation formed pull experiments human Chk2 FHA                     create Rad53-Dun1 interaction interface. <br /> domain. human Chk2 FHA domain displayed mixed char-                     Different FHA domains different specificities acteristics assays (Fig. 7D, lanes 21 25). Like          vivo-phosphorylated proteins. proteins involved homologous Rad53 FHA1, Chk2 FHA bound Rad53                     DNA checkpoint signaling pathways contain FHA domains. <br /> HU- MMS-treated cells.  Chk2 FHA domain                  binding FHA domains artificial phosphopep- <br /> interacted better Rad53 HU-treated rad531-4AQ cells            tides extensively characterized vitro, HU-treated wild-type cells (Fig. 7D, HU              FHA-binding sites identified vivo. include <br /> panel, lanes 21 22), conditions FHA1                 sites interaction Chk2 SCD ([S/T]Q <br /> Dun1 FHA binding diminished. simplest interpreta-               cluster domain) (2, 69), Rad53 phosphorylated Rad9 <br /> tion Chk2 FHA domain preferentially recognizes           (53, 59), Rad53 FHA1 Dun1 Rad53 <br /> 9-12 SQ sites 1-4 TQ sites 9-12 SQ              1-4 TQ cluster. <br /> sites hyperphosphorylated HU treatment rad531-4AQ               Earlier work vitro peptide binding assays suggested <br /> cells (Fig. 7D, lanes 21 22). Consistent interpre-         different FHA domains different binding specificities <br /> tation, rad539-12AQ cells, interaction Chk2 FHA           (9, 15, 23, 66). provide vivo evidence im- <br /> Rad53 reduced MMS- HU-treated cells,                  portance FHA domain specificity, suggested mutation little impact interaction Rad53             fact different FHA domains recognize different substrates <br />  <br />  <br />  <br />   FIG. 7. Specific binding FHA domains Rad53 mutants. yJKD 415 pRS314 DUN1 13xMYC pRS316 RAD53 3xFLAG used. <br />   C, D) Rad53 Rad9 pulled GST-Rad53 FHA1 (FHA1), GST-Rad53 FHA2 (FHA2), GST-Dun1 FHA (Dun1 FHA), GST-Chk2 FHA (Chk2 FHA). micrograms GST fusion protein used pull Rad53 Rad9 2 mg yeast lysate cells expressing wild-type RAD53   rad53 mutations (C D) treatment. Rad53 Rad9 detected Western <br /> blotting polyclonal rabbit antibody protein   C, D). GST fusion proteins visualized Coomassie blue staining  D). Rad53 lysates cells wild-type mutated Rad53 shown panel C. Rad53 Rad9 Western blotting GST pull- assays lysate  panels) Coomassie blue staining GST proteins   shown panel D. (B) Interaction endogenous Rad53 yeast dephosphorylated recombinant Rad53 bacteria DNA damage. U960-5C strains trans- <br /> formed pRS316 RAD53 2xHA. micrograms recombinant Rad53 3xFLAG used pull endogenous Rad53 2xHA 2 mg yeast lysate. Cells mock treated (lanes C) treated HU (lanes H) MMS (lanes M) prior analysis (m, marker). WT, wild type. <br />  6312     LEE ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br />  vivo. differences binding nature specificity    enable transition upstream PIKK-dependent acti- <br /> predicted previously (15): interaction Rad53           vation Chk2 PIKK-independent activation mediated FHA2 Rad9 decreased presence phosphor-        active Chk2 molecules (2, 69). results suggest ylated peptides compete interaction Rad53         similar interaction Rad53 FHA1, FHA2, FHA1 Rad9. Based biological evidence results       phosphorylated Rad53 (Fig. 7) (54). [S/T]Q GST pull experiments, appears Rad53 FHA2           cluster sites required interaction. Dephosporylated vivo binding preferences distinct Rad53       recombinant Rad53 produced bacteria bind phos- <br /> FHA1 Dun1 FHA domain. conserved Rad53             phorylated Rad53 HU- MMS-treated yeast cells (Fig. <br /> FHA1 Dun1 FHA evidently prefer phosphorylated              7B). Finally, Rad53 phosphorylate Rad53 molecules <br /> Rad53 1-4 TQ cluster. divergent Rad53 FHA2             efficiently.  Rad53-dependent activation Rad53 <br /> main prefers highly phosphorylated form Rad53,          fostered binding Rad9 oligomers (20; <br /> presumably includes phosphorylation 9-12 SQ cluster         S.-J. Lee D. F. Stern, unpublished data). Phosphorylated sites. support vivo FHA domain          Rad9-Rad53 interactions intact Rad53 1-4 TQ site mu- <br /> specificity, recently FHA1,         tants, severe phenotypes. 1-4 TQ sites <br /> FHA2, mediates interactions Rad53 Asf1             involved exclusively Rad53-dependent, Rad9-, PIKK- Ptc2/3 (30, 54). Chk2 FHA domain showed mixed             independent activation Rad53 molecules occur <br /> characteristics seen Rad53 FHA1 FHA2             Rad53 released sites DNA damage. <br /> domains (Fig. 7D). vitro, FHA domains tested            Mutation amino-terminal Rad53 TQ cluster impairs <br /> strong preference phosphorylated TQ phosphorylated        Rad53 function activation, mutation 9-12 <br /> SQ sites, consistent importance Rad9 phosphory-        SQ sites minor effects. effects Rad53 9-12 SQ <br /> lated TQ sites binding Rad53 FHA1 FHA2           substitutions evident absence Rad53 1-4 Rad53 1-4 TQ cluster binding Rad53 FHA1             TQ sites, suggesting play partially redundant roles. <br /> Dun1 FHA.  determine        important functional differences phosphorylated Rad53 9-12 SQ cluster recognized          Rad53 phosphorylation site clusters independent vitro Rad53 FHA2 Chk2 FHA domains.                          overall extent Rad53 phosphorylation. Curiously, rad539-12AQ <br />    Phosphorylation Rad53 [S/T]Q clusters important       cells consistently showed increased Rad53 kinase activity Rad53 activation function. Alanine substitution         HU treatment (Fig. 4). enhanced phos- <br /> Chk2 amino-terminal [S/T]Q cluster including threonine 68 (3,      phorylation functionally superior Rad53 1-4 TQ sites 38, 39) SpCds1 threonine 11 (60), lie [S/T]Q          absence 9-12 SQ sites overall signal amplification <br /> clusters similarly situated 1-4 TQ cluster Rad53,        resulting indirect DNA damage loss 9-12 SQ <br /> reduces phosphorylation upstream PIKKs ATM Rad3,            sites. <br /> phosphorylation downstream targets, functional activity        Overall, mutation Rad53 1-4 TQ sites greater effects vivo. results indicate phosphorylation [S/T]Q   did mutation 9-12 SQ sites, including moderate effect <br /> clusters Rad53 similar function.                      catalytic activation Rad53. amino-terminal TQ <br />    Mutation sets Rad53 [S/T]Q cluster sites greatly     sites important activation function SpCds1 (60), <br /> affected HU- MMS-dependent Rad53 phosphorylation.              mammalian Chk2 (3, 38, 39), ScRad53  study), Based precedent Chk2 (2, 3, 38, 39, 69), [S/T]Q cluster    underlying mechanism clear. suggested  <br /> site phosphorylation likely mediated combina-        tramolecular interaction basally inhibits Chk2 tion trans-phosphorylation PIKKs homotypic intra-        intramolecular interaction relieved phosphorylation intermolecular autophosphorylation. greater effect       threonine 68.  deletion Rad53 1-4 TQ domain <br /> Rad53 9-12 SQ site substitutions MMS-dependent mobility         did rescue viability HU led severe phe- <br /> shift kinase-defective Rad53 indicate sites      notype did deletion DUN1, just mutation 1-4 preferential targets PIKKs. Consistent idea,   TQ sites did (Fig. 2) (Lee Stern, unpublished). Chk2 <br /> carboxyl-terminal fragment Rad53 phosphorylated vitro     threonine 68 phosphorylation required initial activation Mec1 (63).  alternative interpretation       Chk2 leads autophosphorylation catalytic <br /> phosphorylation 9-12 SQ sites greater impact     activation loop (29), Chk2 activated, phosphoryla- electrophoretic mobility Rad53 phosphorylation       tion site required maintenance elevated 1-4 TQ sites does.                                             Chk2 kinase activity (1). similar way, Rad53 1-4 TQ sites <br />    Substitution phosphorylation site clusters eliminated   involved initiation maintenance Rad53 kinase <br /> HU- MMS-dependent Rad53 mobility shift          activity. <br /> biological activities measured. substitutions did           Substitution Rad53 1-4 TQ sites severely affected resis- <br /> compromise basal binding Rad53 Asf1 DNA           tance HU MMS DNA checkpoint functions. damage-dependent binding Rad53 phosphorylated Rad9.          component effects contributed failure finding indicates phosphorylation site mu-     activate Dun1.  sml1-1 mutation, used <br /> tations directly affect modulation upstream reg-    maintain viability rad53 strains, suppresses genotoxin <br /> ulator DNA damage checkpoint.                               sensitivity caused loss DUN1 (73).     Phosphorylation directly regulates Rad53 activity (45)      clear causes greatly decreased viability rad531-4AQ <br /> affects formation phosphorylation-dependent homo-           cells HU MMS treatment. Rad53 activation response heteromeric complexes. DNA damage enables Chk2 oli-            HU treatment rad531-4AQ cells moderately  <br /> gomerization FHA-phosphopeptide interactions.         creased compared wild type (Fig. 4A).  <br />  <br />  <br /> activation Rad53 insufficient carry checkpoint func-                   <br /> tions Rad53 rad531-4AQ cells? phosphorylation                         <br /> Rad53 1-4 TQ sites involved binding interactions                     <br /> dissociation Rad53 activators downstream tar-                      <br /> gets? determine mutation                        Rad53 1-4 TQ sites affects phosphorylation Rad53                        <br /> downstream targets impairs dynamic interactions                    <br /> Rad53.                                                                              <br />    Mec1 Tel1 creates binding interface Rad53                     Dun1. PIKKs roles DNA damage                           <br />                                                                                      <br /> replicative stress checkpoint pathways. involved                      direct recognition types DNA lesions,                       Mec1 localizes sites DNA damage absence               <br />                                                                               checkpoint proteins (28, 41). phosphorylate cou-                     <br /> pling-activator proteins like ScRad9 (16, 59, 62). PIKK-depen-                      <br />                                                                                <br /> dent phosphorylation important catalytic activation                        <br /> Rad53, Chk2 (3, 38, 39; study), Chk1 (22, 31, 36)           <br /> finally Rad53 Chk2 oligomerization (2, 69).                            <br />                                                                             upstream kinases create binding interface                     <br /> tween Rad53 downstream Dun1 phosphorylating           <br /> Rad53 1-4 TQ sites. novel function Rad53 extends              <br /> known functionality PIKKs demonstrates Rad53-                 <br /> like kinases serve stable platforms recruitment            <br /> substrate proteins.                                                 <br />    DNA damage signaling cascades hypothesized              <br /> resemble signaling pathways regulated receptor kinases.                    analogy apt, known DNA damage                <br /> induces formation dynamic signaling complexes stabilized          protein-phosphopeptide interactions. Oligomerization-            <br /> induced autophosphorylation receptor kinases creates bind-            <br /> ing sites adaptors signaling proteins containing                     <br /> SH2 PTB domains, recognize phosphorylated ty-                        <br /> rosine residues (52). transforming growth factor &#226;&#65533;&#164; signaling <br />                                                                           <br /> pathways, ligand-activated transforming growth factor &#226;&#65533;&#164; recep-                   <br /> tor type II activates receptor type phosphorylating GS              <br /> domain, creates binding interface R-Smads (37).               similar way, phosphorylation PIKKs enables activating                      <br /> interactions Rad9 Rad53 plays direct role             Rad53 activation.  creates interface bind-          <br /> ing effector kinase Dun1 phosphorylating Rad53 1-4                  <br /> TQ sites. Signal amplification cascades accelerated                  polyvalency interactions fact                  interactions, scaffolding kinase-kinase complexes, cre-                  <br /> ate additional active kinase molecules.                                          <br />   <br /> </body></html>