<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />       essential role Orc6 DNA replication <br />       maintenance pre-replicative complexes <br />  <br />  <br />                                                        G1 phase, ORC promotes origin-association clamp <br />                                                        loading protein Cdc6 <span id='am-18' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span>-dependent manner (Speck <br />                                                 et al, 2005). factor, Cdt1 (Devault et al, 2002; Tanaka <br />                                                     Diffley, 2002) directs nuclear import MCM <br />        <br />                                                                                               (minichromosome maintenance) family proteins, Mcm2&#226;&#8364;&#8220;7, <br />         <br />                                                                                               act replication fork helicases (reviewed Bell        <br />                         Dutta, 2002). ORC Cdc6 present origins, Cdt1&#226;&#8364;&#8220; <br />                              Mcm2&#226;&#8364;&#8220;7 associate origin DNA. Hydrolysis                                                                                               ATP Cdc6 thought result Cdt1 dissociation       heterohexameric origin recognition complex (ORC)                                    stronger Mcm2&#226;&#8364;&#8220;7 binding origins. Subsequent ATP hydro- <br />       acts scaffold G1 phase assembly pre-replica-                            lysis ORC catalyzes loading additional Mcm2&#226;&#8364;&#8220;7 <br />       tive complexes (pre-RC). Orc1-5 subunits appear                                complexes (Kawasaki et al, 2006; Randell et al, 2006).       required origin binding budding yeast,                                 tightly bound, continued association Mcm2       Orc6 essential protein cell proliferation. Imaging                            chromatin G1 phase requires presence ORC- <br />       Orc6-YFP live cells revealed punctate pattern                                   associated protein, Mcm10 (Homesley et al, 2000). <br />       consistent organization replication origins                                 Collectively, assemblage proteins known       subnuclear foci. Orc6 detected site                              pre-replicative complex (pre-RC). addition pre-RC  <br />       division mother daughter cells, contrast                              mation, initiation DNA replication requires activa- <br />       observations metazoans, required mitosis                             tion kinase complexes, Clb5/Cdc28 Dbf4/Cdc7, <br />       cytokinesis. essential role Orc6 DNA replica-                              promote origin-association Cdc45 (Nougarede <br />       tion identified depleting specific cell cycle                              et al, 2000; Zou Stillman, 2000). Cdc45 turn recruits <br />       stages. Interestingly, Orc6 required entry S                               DNA polymerases origins (Mimura Takisawa, 1998; <br />       phase pre-RC formation, contrast previous                                   Aparicio et al, 1999; Zou Stillman, 2000). <br />       models suggesting ORC dispensable point                                   Curiously, ORC subunits required <br />       cell cycle. Orc6 depleted late G1, Mcm2                                 origin recognition binding vitro (Lee Bell, <br />       Mcm10 displaced chromatin, cells failed                                    1997). Orc6 essential protein budding <br />       progress S phase, DNA combing analysis                                      yeast (Li Herskowitz, 1993), appears dispensable <br />       following bromodeoxyuridine incorporation revealed                                 functions role cell cycle progression       efficiency replication origin firing severely                                determined. Clearly, Orc6 association       compromised.                                                                            budding yeast ORC subunits suggests function DNA <br />       EMBO Journal (2006) 25, 5150&#226;&#8364;&#8220;5158. doi:10.1038/                                     replication. Li Herskowitz disrupted copy ORC6 <br />       sj.emboj.7601391; Published online 19 October 2006                                      diploid yeast strain  following sporulation, able <br />       Subject Categories: cell cycle                                                          observe rounds cell division spores <br />       Keywords: DNA replication; minichromosome maintenance                                   inheriting ORC6 knockout. Arrested cells large <br />       protein; origin recognition complex; pre-replicative complex                            budded phenotype observed DNA replication mu- <br />                                                                                               tants, stage cell cycle arrest deter- <br />                                                                                               mined FACS analysis insufficient number                                                                                               cells. Studies involving replication Xenopus sperm <br />       Introduction                                                                            DNA Drosophila egg extracts indicate Orc6 pro- <br />       origin recognition complex (ORC) plays essential role                            mote DNA replication vitro (Chesnokov <br />       initiation DNA replication binding origin                               et al, 2001). human cancer cells, depletion Orc6       sequences cell cycle acting scaffold                            transfection siRNA duplexes resulted significant <br />       association additional protein factors G1 phase                           reduction number positive cells BrdU incorpora- <br />       (reviewed Bell, 2002). Originally isolated characterized                         tion assays, consistent replicative function (Prasanth <br />       budding yeast Saccharomyces cerevisiae (Bell                                 et al, 2002).  research human fruit fly <br />       Stillman, 1992), ORC composed distinct subunits,                              cells point mitotic cytokinetic functions addition <br />       orthologs wide range                            role Orc6 DNA replication (Prasanth et al, 2002; <br />       eukaryotic species (reviewed DePamphilis, 2005). early                            Chesnokov et al, 2003). <br />                                                                                                    demonstrate Orc6 required initia- <br />       *Corresponding author. Department Biology, University Waterloo,                   tion DNA replication budding yeast cells dis- <br />       200 University Ave., Waterloo, Ontario, Canada N2L 3G1.                                 pensable progression mitosis cytokinesis.       Tel.: &#195;&#190; 1 519 888 4567 x 33957; Fax: &#195;&#190; 1 519 746 0614;                                  Orc6 required maintenance       E-mail: bduncker@sciborg.uwaterloo.ca <br />                                                                                               MCM protein association chromatin, depletion <br />       Received: 5 2006; accepted: 19 September 2006; published                            Orc6 pre-RC formation inhibits replication origin <br />       online: 19 October 2006                                                                 firing. <br />  <br /> 5150 EMBO Journal VOL 25 | 21 | 2006                                                                           <span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; 2006 European Molecular Biology Organization <br />                                                                                                        Orc6 required pre-RC maintenance <br />                                                                                                                                JW Semple et al <br />  <br />  <br />  <br /> Results <br /> Orc6 localizes subnuclear foci living yeast cells step characterizing essential role Orc6, examined cellular localization. accomplished tagging genomic ORC6 sequence encoding GFP <br /> variant eYFP haploid yeast strain (DY-1), resulting expression length Orc6 C-terminal eYFP tag, control natural ORC6 promoter (DY-41). <br /> Studies human cancer cells indicate  addition expected nuclear localization interphase, cyto- <br /> plasmic pools Orc6 exist localization observed cell periphery mitosis midbody cytokinesis (Prasanth et al, 2002). Similar results reported Drosophila cells (Chesnokov et al, 2003). followed single cells progressed cell <br /> cycle, nuclear Orc6 localization observed <br /> (Figure 1A). confirm S. cerevisiae Orc6 confined nucleus, constructed double-tagged haploid strain <br /> expressing Orc6 C-terminal eCFP fusion nuclear pore protein Nup49 C-terminal eYFP fusion <br /> (DY-65). Irrespective cell cycle stage, Orc6 consistently <br /> bounded nuclear membrane, designated signal Nup49 (Figure 1C). rule                                                                    Figure 1 Orc6 localizes subnuclear foci cell <br /> additional pools Orc6 threshold detection,      cycle. Orc6-eYFP, Orc6-eCFP Nup49-eYFP cell preparation conclude significant localization budding     live imaging performed described Materials methods. <br /> yeast Orc6 cell periphery mother-bud         single cell imaged 3 h time period. bright <br />                                                                    field single fluorescent image taken 10 min; <br /> neck. <br />                                                                    representative images different cell cycle stages shown. <br />    investigate subnuclear localization Orc6   Multiple fluorescent images taken time point <br /> live cells, 20 images taken rapid succession stacked    reduce background (B), increased exposure light <br /> increase signal described Materials methods.     source results cell bleaching 3 h time course. (B)                                                                    representative field cells shown. expression Orc6-eYFP resultant higher resolution images revealed pattern <br />                                                                    low endogenous promoter, stronger signal punctate Orc6 foci (Figure 1B), similar        reduced background obtained taking 20 fluorescent images <br /> previously observed fixed cells presumed replica-     low gain rapid succession stacking using ImageJ <br /> tion factory constituents Orc2 (Pasero et al, 1999;        1.30v. (C) localization Orc6 nuclear membrane protein <br /> Lengronne et al, 2001). Chromatin immunoprecipitation              (Nup49) series Orc6-eCFP/Nup49-eYFP cells shows Orc6 <br />                                                                    exclusively nucleus stages cell cycle <br /> (ChIP) analysis indicated Orc6 specifically binds     observed, judged bud morphology. scale bars correspond replication origin sequence (Supplementary Figure 1).           5 mm. <br />  <br />  <br /> Depletion Orc6 leads S-phase arrest <br /> Given cellular localization Orc6 indicative     contrast, 3 h GAL1-ORC6 cells showing role DNA replication, provided evidence            defects S phase progression, accumulation cells <br /> additional cellular function(s), wanted determine        DNA content 1C 2C. significant <br /> point cell cycle cells arrest depleted     accumulation cells 2C greater DNA content Orc6. haploid yeast strain (DY-36) generated replacing       observed, expected mitotic cytokinetic natural ORC6 promoter glucose-repressible GAL1          defects.  6 h, size 2C peak markedly <br /> promoter. strain construction, sequence          reduced compared earlier time points, wt control. <br /> encoding copies HA epitope fused start ORC6 ORF, facilitate monitoring Orc6        Orc6 required entry S phase <br /> levels. Cultures DY-36 isogenic wild-type (wt) strain       order determine Orc6 required entry DY-26, grown 2% galactose/1% raffinose (YPG/R)             progression S phase, Orc6 depleted medium overnight, washed ddH2O, resuspended              GAL1-ORC6 cells synchronized G2/M initially arresting <br /> 2% glucose (YPD) medium. Following 4 h growth YPD,           cells YPG/R supplemented nocodazole, washing Orc6 GAL1-ORC6 cells fallen         culture resuspending cells YPD/nocodazole. normal endogenous levels judged immunodetection              origins rendered competent DNA replication G1 cell extracts (Figure 2A), cells clearly        phase, Orc6 essential process expect showed growth defects subsequent time points, relative       formation 1C peak Orc6-depleted cells fail wt controls (Figure 2B). compared GAL1-ORC6         efficiently origins following release block. wt strain growth YPD removing aliquots 3-h            1C peak observed GAL1-ORC6 strain <br /> intervals FACS analysis. time points following      following Orc6 depletion (Figure 3A). <br /> shift YPD, wt strain exhibited prominent 1C 2C             central function ORC complex promoting DNA <br /> peaks characteristic asynchronous cultures (Figure 2C).      replication facilitate loading MCM proteins  <br /> &amp; 2006 European Molecular Biology Organization                                              EMBO Journal       VOL 25 | 21 | 2006 5151 <br />        Orc6 required pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />  <br />                                                                             Figure 3 Orc6 required S-phase progression.   <br />                                                                             Asynchronous cultures GAL1-3HA-ORC6 cells (DY-36)                                                                             wt parental strain (DY-26; labeled wt) grown 5  106 cells/ <br />                                                                             ml (Exp) arrested G2/M nocodazole (15 mg/ml)                                                                             3 h YPG/R. blocked, cells washed transferred <br />                                                                             YPD nocodazole (12 mg/ml) 4 h maintain cell <br />                                                                             cycle arrest. cells washed released fresh <br />                                                                             YPD arrest agents samples removed                                                                             indicated intervals FACS analysis. (B) GAL1-3HA-ORC6 wt <br />                                                                             cells arrested YPG/R nocodazole transferred <br />                                                                             YPD/nocodazole 4 h described  . Cultures                                                                             subsequently released YPD supplemented factor                                                                             arrest cells late G1 phase. Samples culture taken <br />                                                                             2 h factor chromatin binding assays. Thirty microliters <br />       Figure 2 Depletion Orc6 results accumulation S phase      chromatin-associated pellet (P) 15 ml supernatant <br />       cells.   asynchronous GAL1-3HA-ORC6 (DY-36) culture          (S) analyzed immunoblot. HA-tagged Orc6 detected <br />       grown YPG/R (2% galactose, 1% raffinose) medium 106 cells/      using anti-HA antibody (Roche); anti-Mcm2 (1:500; goat poly- <br />       ml, washed resuspended YPD (2% glucose) medium.          clonal, Santa Cruz) anti-Orc2 (1:1000; rabbit polyclonal) anti- <br />       cell extracts prepared culture aliquots taken prior      bodies used analyze pellet supernatant fractions <br />       (GAL) indicated time points following resuspension      isogenic strains DY-36 (GAL1-3HA-ORC6) DY-93 (ORC6- <br />       YPD. cell extract prepared asynchronous      3HA). Cdt1 detected using (anti-Myc; Sigma) isogenic <br />       exponential culture isogenic strain (DY-93) Orc6    strains DY-83 (CDT1-13Myc) DY-91 (GAL1-3HA-ORC6, CDT1- <br />       tagged number HA epitopes, remains     13Myc), Mcm10 detected antibodies       control endogenous promoter (wt). Eighty micrograms    strains DY-84 (MCM10-13Myc) DY-92 (GAL1-3HA-ORC6, <br />       extract used immunoblot analysis. HA-tagged Orc6          MCM10-13Myc). (C) Liquid hybrid assays performed <br />       detected using anti-HA antibody (Roche) fluorescent sec-       using pEG-Mcm2, pEG-Cdc6, pEG-Cdt1 pEG-Mcm10 baits <br />       ondary antibody (Invitrogen). Ponceau S staining region        combination ORC subunit (pJG-ORC1-6)       detected blot verify equal loading cell extracts   pJG4-6 vector (E) preys. hybrid results representative <br />       shown. (B) Asynchronous GAL1-3HA-ORC6 (DY-36)             assays error bars represent standard deviation. <br />       isogenic wt (DY-26) cultures grown described  . Cell <br />       counts cultures taken 2 h 8 h following <br />       transfer YPD. Growth defects observed GAL1-3HA-        appears reduction Mcm2 loading       ORC6 strain 4 h, compared seen wt cells.        specific absence Orc6. <br />       (C) FACS analysis culture aliquots removed indicated time <br />       points transfer YPD. <br />                                                                             Orc6 interacts members pre-RC involved <br />                                                                             MCM loading maintenance <br />       origins (reviewed Bell Dutta, 2002). explore        determine Orc6 physically interacts Mcm2       role Orc6 rendering origins competent, moni-             factors play role MCM chromatin association,       tored Mcm2 loading means chromatin fractionation              length MCM2, CDC6, CDT1 MCM10 coding sequences <br />       assay (Donovan et al, 1997; Liang Stillman, 1997) using           cloned hybrid bait vector pEG-202 <br />       GAL1-ORC6 cells depleted Orc6 G2/           (Ausubel et al, 1995). constructs separately <br />       M arrest. Mcm2 association chromatin pellet              transformed wt yeast strain DY-1, harboring <br />       reduced, Orc6-depleted cells relative wt cells, following       lacZ reporter-plasmid pSH18-34, prey <br />       release G1 block factor (Figure 3B). confirm         plasmids expressing Orc1-6. Mcm2 tested       decline Mcm2 loading following Orc6 depletion             bait, strongest b-galactosidase signal obtained       G2/M result ORC destabilization,            Orc2, indicating ORC subunit Orc6,       monitored level Orc2 examine overall state       likely associate directly Mcm2 (Figure 3C).       ORC complex. GAL1-ORC6 cells depleted Orc6 G2/M             evaluated ORC subunit interactions Cdc6 Cdt1, <br />       released G1 block levels chromatin-              required MCM loading chromatin   <br />       bound Orc2 comparable wt cells (Figure 3B), consistent             viewed Bell Dutta, 2002). strongest signal Cdc6 <br />       previous vitro studies indicating Orc6 required        obtained Orc1 subunit, consistent       ORC bind replication origins (Lee Bell, 1997).          previously reported Orc1&#226;&#8364;&#8220;Cdc6 interaction yeast <br />  <br /> 5152 EMBO Journal VOL 25 | 21 | 2006                                                      &amp; 2006 European Molecular Biology Organization <br />                                                                                                      Orc6 required pre-RC maintenance <br />                                                                                                                              JW Semple et al <br />  <br />  <br />  <br /> (Wang et al, 1999) human cells (Saha et al, 1998).  Orc2, Orc5 Orc6 preys produced positive  <br /> hybrid signals expressed Cdt1 bait, Orc6 highest  Finally, tested ORC <br /> subunits Mcm10, required preserve interaction Mcm2 chromatin G1 phase (Homesley <br /> et al, 2000).  noticed similar pattern observed Cdt1, Orc6 exhibiting highest hybrid signal. Cdt1 Mcm10 interacted Orc6, evaluated effect G2/M Orc6 depletion chromatin association G1. appreciable <br /> effect degree chromatin binding protein <br /> (Figure 3B). <br />  <br /> Orc6 required maintain MCM chromatin <br /> association following pre-RC formation <br /> Studies Xenopus egg extracts indicate MCM proteins assembled chromatin, ORC Cdc6 removed affecting putative DNA <br /> helicase function (Hua Newport, 1998; Rowles et al, <br /> 1999).  interested determine Orc6 required efficient initiation DNA <br /> replication MCM proteins loaded chro- <br /> matin. carried synchronizing cultures GAL1-ORC6 parental strain growing YPG/R late <br /> G1 phase adding factor. Following initial arrest, cells transferred YPD factor deplete Orc6 GAL1-ORC6 strain. release block, accumulation 1C cells observed Orc6-depleted <br /> GAL1-ORC6 culture 30 min, isogenic wt strain <br /> displayed prominent 2C peak time point, <br /> consistent role Orc6 S phase progression <br /> (Figure 4A). additional control, GAL1-ORC4 strain similarly assessed. Orc4 depletion strain occurs similar kinetics observed Orc6 depletion <br /> (Supplementary Figure 2), case cells able                                                                    Figure 4 Depletion Orc6 late G1 destabilizes MCM association <br /> replicate DNA, albeit slower rate wt control   chromatin impedes S phase entry   wt (DY-26), GAL1- <br /> (Figure 4A).  appears Orc6, opposed    3HA-ORC6 (DY-36) GAL1-3HA-ORC4 (DY-103) strains entire ORC complex, required late G1 phase subse-        grown 5  106 cells/ml (Exp) arrested G1 factor <br />                                                                    (50 mg/ml) 2&#226;&#8364;&#8220;3 h YPG/R. cells pelleted, washed <br /> quent DNA replication. likely explanation                                                                    ddH2O transferred YPD containing factor (50 mg/ml) lack DNA replication following Orc6 depletion late G1      4 h. Cells subsequently washed released YPD <br /> phase destabilization MCM complex            factor Culture aliquots taken indicated <br /> origins, monitored chromatin association Mcm2            times FACS analysis. (B) Isogenic wt, GAL1-3HA-ORC6 prior following Orc6 depletion. equivalent            GAL1-3HA-ORC4 cells arrested factor YPG/R, <br />                                                                    washed incubated YPD factor, described  ; <br /> chromatin pellet supernatant Mcm2 ratios observed          DY-109 (ORC4-3HA) used wt control experiments initial factor arrest wt GAL1-        GAL1-3HA-ORC4 cells. Chromatin fractionation immuno- <br /> ORC6 strains (GAL; Figure 4B), clear displace-         blotting performed described Figure 3, culture <br /> ment Mcm2 pellet supernatant fraction           aliquots removed following initial arrest (GAL),                                                                    end YPD factor incubation (GLU). (C) ChIP carried <br /> following Orc6 depletion GAL1-ORC6 strain (GLU;             mixture Mcm2 Mcm5 antibodies described Figure 4B). Densitometry indicated close         Materials methods, DY-26 DY-36 cells cultured 50% reduction Mcm2 pellet:supernatant ratio            described  . PCR using sample aliquots taken sonication GAL1-ORC6 strain relative isogenic wt strain            (INPUT) following purification immunoprecipitated DNA <br />                                                                    (IP) carried using primers specific ARS1 origin <br /> (Supplementary Figure 3). Interestingly, observed        sequence, region 9 kb upstream ARS1 (ARS1&#226;&#8364;&#8221;9 kb). <br /> dramatic reduction level chromatin-associated Orc2 <br /> following Orc6 depletion. rule possibility overall destabilization ORC respon-      chromatin fraction (Randell et al, 2006). contrast, high <br /> sible displacement Mcm2, observation         levels chromatin-bound Mcm10 initially detected robust Mcm2 chromatin association maintained cells           markedly reduced following Orc6 shut (Figure 4B depleted Orc4 (Figure 4B) argues  case   Supplementary Figure 3). results consistent Cdt1, little difference observed cells        mechanism Orc6 promotes Mcm10 chromatin asso- <br /> mal Orc6 levels depleted,       ciation, Mcm10 turn stabilizes MCM complex case, expected, little Cdt1 detected       (Homesley et al, 2000). Finally, confirmed MCM <br />  <br /> &amp; 2006 European Molecular Biology Organization                                             EMBO Journal      VOL 25 | 21 | 2006 5153 <br />        Orc6 required pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />       proteins actually displaced replication origins fol- <br />       lowing late G1 Orc6 depletion, carrying ChIP analysis <br />       (Figure 4C), demonstrated dramatic drop level <br />       ARS1 DNA immunoprecipitating MCMs Orc6 <br />       absent. <br />  <br />       Depletion Orc6 late G1 reduces efficiency       DNA replication initiation <br />       inability cells lacking Orc6 progress S phase <br />       reduction origin firing, inhibition <br />       elongation, combination  investigate       effect Orc6 depletion initiation events, constructed       TK &#195;&#190; GAL1-ORC6 strain (DY-79), allow incorporation <br />       BrdU newly synthesized DNA. Cultures DY-79       parental TK &#195;&#190; strain (DY-67) initially synchronized <br />       late G1 factor YPG/R, shifted YPD       factor, deplete Orc6 DY-79 cells. cells       subsequently washed resuspended YPD supplemented <br />       BrdU (0.4 mg/ml) ribonucleotide reductase <br />       inhibitor hydroxyurea (HU; 0.2 M), normally results <br />       mid-S phase arrest half replication origins <br />       fired. Following 90 min HU, cells harvested, <br />       genomic DNA isolated single molecule DNA comb- <br />       ing carried  previously described (Versini et al, <br />       2003). Comparison inter-origin distances GAL1-ORC6 <br />                                                                          Figure 5 Orc6 required initiation DNA replication <br />       wt cells revealed approximately half number        following pre-RC formation.   Initiation frequency monitored <br />       origins fired cells depleted Orc6 (Figure 5A), consistent    wt (DY-67) GAL1-3HA-ORC6 (DY-79) TK &#195;&#190; cells. Cells <br />       role Orc6 promoting initiation events. length    synchronized G1 using factor transferred YPD <br />       individual BrdU tracks double               medium 4 h deplete Orc6 arrested                                                                          pheromone. Cells released YPD containing HU <br />       observed wt controls. explained        (0.2 M) block elongation, BrdU monitor new strand <br />       fact <span id='am-29' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-30' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> pools greatly reduced HU          synthesis. DNA plugs prepared 90 min transfer. <br />       administered, cells fewer origins firing       FACS profiles stage experiment presented       nucleotides available migration indivi-       left. DNA combing performed silanized coverslips                                                                          described Materials methods. Representative DNA fibers <br />       dual replication fork. examine possibility origin      wt GAL1-3HA-ORC6 strains shown. BrdU <br />       firing inhibition caused Orc6 depletion triggering G1     incorporation appears green, DNA red. Inter-origin distances       checkpoint, monitored abundance phosphoryla-            BrdU tract lengths following analysis 200 origins       tion status Rad53 did detect upregulation higher     shown histograms, case average value                                                                          indicated. Scale bar represents 50 kb. Mann&#226;&#8364;&#8220;Whitney rank-sum <br />       mobility forms immunoblots samples taken             test used differences wt GAL1- <br />       immediately following Orc6 depletion hour release      ORC6 cells significant (Po0.001). (B) wt (DY-93) GAL1- <br />       factor block (Supplementary Figure 4).                  3HA-ORC6 (DY-36) strains grown YPG/R 5  106 cells/ml <br />           Orc6 play role elongation stage         arrested S phase HU (0.2 M). cells washed <br />                                                                          ddH2O transferred YPD HU (0.36 M) 4 h. Cells <br />       initiation stage DNA replication        washed twice ddH2O released fresh YPD. <br />       investigated. GAL1-ORC6 isogenic wt cells arrested        Aliquots culture taken 20 min FACS analysis. <br />       YPG/R supplemented HU, washed resus- <br />       pended YPD/HU 4 h. Following release HU <br />       block YPD, cultures progressed S phase       essential protein cell proliferation (Li Herskowitz, <br />       comparable rates (Figure 5B), suggesting Orc6         1993), previous studies suggest budding yeast <br />       required efficient replication fork progression.      ORC subunits important functions cell <br />       exclude possibility residual Orc6 following     cycle stages (Bell et al, 1993; Dillin Rine, 1998). Recent <br />       shut regime played role replication fork     attention focused additional roles Orc6 subunit <br />       progression, rule cause S-phase        mitosis cytokinesis metazoans (Prasanth et al, 2002; <br />       arrest length depletion (Figure 2).             Chesnokov et al, 2003). reasonable ask <br />                                                                          budding yeast Orc6 required DNA replication <br />                                                                          cell cycle events. localization Orc6-YFP/ <br />       Discussion                                                         CFP subnuclear foci living cells provides compelling <br />       essential cell cycle role Orc6 specific S phase       evidence replicative function, similar       original biochemical purification budding yeast     punctate pattern previously shown immunolocalization       (Bell Stillman, 1992), ORC shown play            Orc2 fixed cells (Pasero et al, 1999), detection <br />       central role initiation DNA replication (reviewed     discrete zones new DNA synthesis TK &#195;&#190; yeast cells <br />       Bell, 2002). ORC lacking Orc6 fully competent bind           following BrdU pulse (Lengronne et al, 2001).       origin sequences, omission subunit abro-          knowledge, visualization Orc6 foci represents       gates capability (Lee Bell, 1997). Orc6   detection protein constituent thought <br />  <br /> 5154 EMBO Journal VOL 25 | 21 | 2006                                                  &amp; 2006 European Molecular Biology Organization <br />                                                                                                    Orc6 required pre-RC maintenance <br />                                                                                                                            JW Semple et al <br />  <br />  <br />  replication factories involving clusters replication      type large bud morphology associated forks (reviewed Laskey Madine, 2003) living              mutations affecting DNA replication. recently, hy- <br /> S. cerevisiae cells. Intriguingly,   perphosphorylation Orc6 Orc2 following START <br /> ORC subunits, Orc5 Orc6 able homodimerize          shown series overlapping mechanisms conceivably contribute formation               prevent DNA replication, include Cdc6 replication compartments bridging different origin           degradation export Mcm2&#226;&#8364;&#8220;7 nucleus (Nguyen <br /> sequences (unpublished data).                                      et al, 2001). processes Clb/Cdc28 (CDK) <br />    contrast seen human fruit       mediated, direct interaction Clb5 Orc6 <br /> fly cells, observed accumulation Orc6 mid-         reported budding yeast (Wilmes et al, 2004). <br /> plane division mother daughter cells (Figure        mutation Orc2 Orc6 consensus CDK <br /> 1A C). Given nuclear envelope budding yeast        phosphorylation sites abrogation Clb5/Orc6 inter- <br /> remains intact cell cycle, possible      action predisposes cells replication combination Orc6 play karyokinetic role analogous       stabilized Cdc6 constitutively nuclear Mcm2&#226;&#8364;&#8220;7 <br /> cytokinetic function metazoan orthologs. results        (Nguyen et al, 2001; Wilmes et al, 2004; Green et al, 2006; <br /> argue   observe popula-        Tanny et al, 2006).  change effect tion Orc6 site nuclear division cells   cell cycle progression presence wt Cdc6 MCM observed (Figure 1A results shown).            proteins.  prevention replication essen- <br /> FACS analysis following depletion Orc6 asynchro-        tial function Orc6. <br /> nous culture shows accumulation cells X2C DNA              present report, provide strong evidence Orc6 <br /> content (Figure 2C). fact significant 2C DNA peak      required efficient initiation DNA replication. Cells observed 6 h Orc6 depleted suggests        released G2/M block Orc6 chromosome segregation compromised reported             depleted, arrested cell cycle 1C DNA human cells treated Orc6 siRNA (Prasanth et al,           content (Figure 3A). lack Orc6 inhibiting <br /> 2002).  cells depleted Orc6 G2/M block         DNA replication? possibility absence Orc6 fully capable completing mitosis proceeding       prevents proper pre-RC assembly, normally culminates <br /> G1 phase following release (Figure 3A).         chromatin loading MCM complex. rule residual levels Orc6 capable carrying      modest drop level chromatin-associated Mcm2 mitotic cytokinetic functions remain following          cells depleted Orc6 nocodazole released depletion protocol, clear marked accumulation       subsequently blocked factor (Figure 3B). strik- cells DNA content 1C 2C following Orc6         ing,  strong displacement previously <br /> depletion asynchronous culture, cell cycle          loaded Mcm2 depleting Orc6 extended factor <br /> arrest observed conditions primarily            arrest (Figure 4B C). pointed role Orc6 S phase defects.                                            maintaining origin association MCM complexes    metazoan roles Orc6 cytokinesis            associated pre-RC. Possible mechanisms <br /> mitosis present budding yeast? important              Orc6 facilitates MCM chromatin association consideration answering question S. cerevisiae     suggested observations Orc6 interacts Orc6 shares low conservation Drosophila (5%          Cdt1 Mcm10, factors required stable association <br /> identity, 19% similarity) human (6% identity, 15%              Mcm2 chromatin (Homesley et al, 2000; Devault et al, <br /> similarity) orthologs. Furthermore, sequence comparison         2002; Tanaka Diffley, 2002). appeared Orc1&#226;&#8364;&#8220;Orc6 species suggests Orc6 evolved            significant effect chromatin levels Cdt1 quickly ORC subunits (Dhar          Mcm10 following G2/M Orc6 depletion, chromatin <br /> Dutta, 2000).  appears metazoan Orc6              pellet supernatant ratio Mcm10 dropped approxi- <br /> conserved original role DNA replication, acquir-      mately 50% Orc6 expression shut late G1 <br /> ing cytokinetic  case human Orc6,           phase (Figure 4B, Supplementary Figure 3), suggesting <br /> mitotic functions. proposed Orc6 helps coordi-      maintenance role. Interestingly, Mcm10 nate processes DNA replication chromosome               shown regulate chromatin association DNA poly- <br /> segregation cytokinesis metazoans (Prasanth et al,         merase (Ricke Bielinsky, 2004), providing way <br /> 2002), critical multicellular     displacement inhibit DNA replication. unicellular organisms. fact additional func-        contrast seen Orc6 depletion earlier <br /> tions present S. cerevisiae makes advanta-        time point, Orc2 displaced chromatin (Figure 4B). <br /> geous characterize role Orc6 DNA        general destabilization ORC responsible replication, complication mitotic cyto-         MCM displacement, subsequent replication defects; <br /> kinetic phenotypes.                                                 fact Orc4 depletion displaces Mcm2 <br />                                                                    chromatin (Figure 4B) prevents progression Orc6 mediates G1 phase chromatin association                   S phase (Figure 4A) argues points role Mcm2                                                            Orc6 distinct ORC subunits. knowledge, substantial evidence essential role S. cerevisiae Orc6 DNA replication           Orc6 plays role initiation DNA replication <br /> comes original work Li Herskowitz (1993)           pre-RC formation following sporulation tetrad          Previous models pre-RC assembly function dissection ORC6 &#195;&#190; /  heterozygotes, haploid ORC6                proposed MCM proteins loaded spores undergo rounds cell division, arresting        chromatin, ORC dispensable DNA replication. <br />  <br /> &amp; 2006 European Molecular Biology Organization                                           EMBO Journal      VOL 25 | 21 | 2006 5155 <br />        Orc6 required pre-RC maintenance <br />       JW Semple et al <br />  <br />  <br />  <br />       Treatment budding yeast late G1 chromatin pre-              Plasmid construction <br />                                                                                Genomic tagging ORFs performed homologous recombi- <br />       RCs assembled magnetic beads high salt                                                                                nation linear PCR fragments amplified using plasmid <br />       reported remove ORC, MCM proteins remain present                templates. create eYFP tagging vector, DNA sequence <br />       (Donovan et al, 1997; Bowers et al, 2004). case,                 encoding GST tag removed pFA6a-GST-KanMX6 <br />        reduction level associated                (Longtine et al, 1998) digestion PacI AscI.                                                                                gene encoding eYFP amplified pEYFP (Clontech) using <br />       MCM proteins consequences ORC removal                                                                                High Fidelity PCR Kit (Roche), forward reverse primers <br />       DNA replication evaluated. Subsequent DNA repli-                incorporating 50 PacI AscI sites, respectively. Following <br />       cation determined unaffected salt extraction             digestion, EYFP ligated pFA6a-KanMX6 vector       ORC pre-RC formation Xenopus cell-free                 create pFA6a-eYFP-KanMX6. Similarly, ECFP amplified       successful removal Orc1 Orc2                pECFP (Clontech) cloned  creating pFA6a-eCFP-TRP1. <br />                                                                                fluorescent strains created integration PCR-based <br />       subunits monitored (Rowles et al, 1999). contrast              cassette amplified plasmids created 30 -end       results, data clearly points role Orc6             genes tagged (Longtine et al, 1998). <br />       promoting initiation events pre-RC formed,              bait prey constructs hybrid analysis       depletion late G1 severely reduced number origins          developed amplifying entire coding sequence gene <br />                                                                                using High Fidelity PCR Kit (Roche), cloning purified <br />       fired (Figure 5A) clearly inhibited S phase progres-            products vector frame fusion cassette. <br />       sion (Figure 4A). explanation varying observations <br />       budding yeast Xenopus simply                  hybrid assays <br />       differences species respect cell             hybrid assays performed described previously <br />                                                                                (Ausubel et al, 1995; Duncker et al, 2002). LacZ-reporter (pSH18- <br />       cycle window ORC required ensure origin               34), bait (derived pEG202; Ausubel et al, 1995) prey <br />       firing. possibility different ORC subunits           (derived pJG4-6, variant pJG4-5 extensive <br />       required varying degrees initiation pre-RC forma-           polylinker; Gyuris et al, 1993) plasmids serially transformed <br />       tion. favor idea, observed depletion            yeast strain DY-1. Exponential cultures grown SC (2% <br />                                                                                glucose) medium lacking <span id='am-72' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span>, histidine tryptophan       Orc4 late G1 does result Mcm2 chromatin displace-              concentration 5  106 cells/ml, washed ddH2O       ment prevent cells traversing S phase, contrast            resuspended SC (2% galactose/1% raffinose) lacking       seen Orc6 turnover (Figure 4A B). recent              components 6 h induce prey expression. Following <br />       study (Gibson et al, 2006) reported orc1-161, orc2-1            induction, 5  106 cells harvested permeabilized. <br />                                                                                relative strength interaction quantified       orc5-1 temperature-sensitive mutants exhibited slower               b-galactosidase assay utilizing substrate o-nitrophenyl-b-D-galac- <br />       progression S phase wt control, shift          topyranoside (ONPG) (Burke et al, 2000) formula: <br />       restrictive temperature factor, subsequent release           b-galactosidase activity &#194;&#188; 1000   A420 nm/(t   v   A600 nm) t       temperature.  effects consider-              time reaction (min) v volume culture used                                                                                assay (ml). <br />       ably severe observed following Orc6 <br />       depletion. potential late G1 requirement Orc2                  Microscopy <br />       examined number additional studies,                    images taken live cells. Prior imaging cells       varying outcomes. Strong inhibition S phase progression               grown B5  106 cells/ml SC medium (2% glucose, 0.02% <br />       observed orc2-1 strain shifted non-                   <span id='am-13' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span>), washed ddH2O resuspended fresh medium. <br />                                                                                Cultures diluted fresh medium B1  106 cells/ml,       permissive temperature late G1 (Zhang et al, 2002). Late              500 ml added growth chambers imaging. Chambers       G1 Orc2 depletion permissive temperature using GAL1-                created using 0.500 <span id='am-71' about='oboInOwl:id' typeof='owl:Thing'>ID</span> glass tubing cut 0.500 glued       orc2-1 strain little effect S phase progression            standard glass (22   66 mm) coverslip medical grade silicon <br />       report (Shimada et al, 2002), study using              adhesive. Cells imaged Zeiss Axiovert 100   63, <br />                                                                                1.4 N objective lens. image eCFP eYFP used filter sets <br />       strain Orc2 depletion resulted efficient         consisting exciter D436/   20; dichroic 455DCLP; emitter D480/ <br />       DNA replication (Weinberger et al, 2005). possible expla-              40 m exciter HQ500/   20; dichroic Q515LP; emitter HQ520LP <br />       nation differing outcomes depletion studies           respectively (Chroma Technology, Rockingham, VT). Images       Orc2 removal pre-RC destabilization               collected Sony SX700 CCD (1024   768) processed                                                                                ImageJ 1.30v (NIH, Bethesda). maximum exposure time       efficient set conditions used (2 h, 231C)              digital camera 2 s result imaging FP- <br />       (1 h, 301C);  remains tested                tagged proteins endogenous promoters resulted       experimentally.                                                          weak signals. obtain higher signal, 20 fluorescent images <br />           Clearly, better understanding functions indivi-          taken succession subsequently summed using ImageJ <br />                                                                                software. Camera gain adjusted maximize signal noise <br />       dual ORC components required fully characterize                ratios individual frames. <br />       multiple roles complex cell cycle progression.       time, demonstrated essential function           Chromatin binding assay <br />       Orc6 subunit promoting DNA replication mediating               Approximately 2.5  107 cells harvested 1000 g                                                                                spheroplasted performed previously (Pasero et al, 1999),       stability pre-RCs formation.                                                                                modifications. Cells washed ddH2O incubated <br />       considerable determine function              301C 10 min gentle mixing 10 ml/g pre-spheroplasting <br />       conserved eukaryotes, investigate precise               buffer (100 mM EDTA-KOH (pH 8), 10 mM DTT), followed       cell cycle roles ORC subunits.                                  incubation 10 ml/g spheroplasting buffer (0.5XYPD, 1.1 M <br />                                                                                sorbitol) containing 0.5 mg/ml Zymolyase 20T (Seikagaku Corp., <br />                                                                                Japan) 301C 10&#226;&#8364;&#8220;15 min gentle mixing. Cells washed <br />                                                                                20 ml spheroplasting buffer containing 0.5 mM PMSF <br />       Materials methods                                                    followed resuspension 1 ml ice-cold wash buffer (5 mM Tris&#226;&#8364;&#8220; <br />                                                                                HCl (pH 7.4), 20 mM KCl, 2 mM EDTA-KOH (pH 7.4), 1 M sorbitol, <br />       Yeast strains                                                            1% thiodiglycol, 125 mM <span id='am-37' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>spermidine</span>, 50 mM <span id='am-1' about='obo:IMR_0200092' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00750" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Spermine" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200092" datatype="xsd:string"></span>spermine</span>) protease <br />       genotypes strains used study listed Table    inhibitors (0.1 mM benzamidine HCl, 1 mg/ml pepstatin, 2 mg/ml <br />       Unless indicated, DY-1 used standard wt             antipain, 2 mg/ml leupeptin, 0.5 mM PMSF). Cells pelleted       strain.                                                                  400 g 2 min microcentrifuge 41C, washed twice 1 ml <br />  <br /> 5156 EMBO Journal VOL 25 | 21 | 2006                                                         &amp; 2006 European Molecular Biology Organization <br />                                                                                                           Orc6 required pre-RC maintenance <br />                                                                                                                                   JW Semple et al <br />  <br />  <br />  <br /> Table Yeast strains used study <br /> Strain         Genotype                                                                                                     Source <br />  <br /> DY-1           MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2                        S. Gasser <br />                                                                                                                                                                                                                                                         (GA1020) <br /> DY-26          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0                                                             ATCC (BY4733) <br /> DY-36          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1                                   study <br /> DY-39          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-            study <br />                13Myc/TRP1 <br /> DY-41          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-EYFP/       study <br />                kanMX6 <br /> DY-65          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc6HORC6-ECFP/       study <br />                TRP, nup49HNUP49-EYFP/kanMX6 <br /> DY-67          MATa, ura3HURA3[GPD-TK(7x)] ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 GAL psi+                            E Schwob <br />                                                                                                                             (E1000) <br /> DY-79          MATa ura3HURA3[GPD-TK(7x)] ade2-1 trp1-1 can1-100 leu2-3,112 his3-11,15 GAL psi+, orc6HPgal1-                study <br />                3HA-ORC6/TRP1 <br /> DY-81          MATa, ade2-1, can1-100, trp1-1, his3-11, his3-15, ura3-1, leu2-3, leu2-112, pep4HLEU2, orc2HORC2-            study <br />                13Myc/TRP1 <br /> DY-83          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, cdt1HCDT1-13Myc/HIS3                                       study <br /> DY-84          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, mcm10HMCM10-13Myc/HIS3                                     study <br /> DY-91          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1, cdt1HCDT1-13Myc/                 study <br />                HIS3 <br /> DY-92          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HPgal1-3HA-ORC6/TRP1, mcm10HMCM10-                     study <br />                13Myc/HIS3 <br /> DY-93          MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc6HORC6-3HA/TRP1                                         study <br /> DY-103         MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc4HPgal1-3HA-ORC4/TRP1                                   study <br /> DY-109         MATa, his3D200, leu2D0, met15D0, trp1D63, ura3D0, orc4HORC4-3HA/TRP1                                         study <br />  <br />  <br />  <br />  <br /> ice-cold wash buffer, resuspended 0.4 ml ice-cold breakage       incubation YPD pheromone deplete Orc6. cells buffer (5 mM Tris&#226;&#8364;&#8220;HCl (pH 7.4), 20 mM KCl, 2 mM EDTA-KOH (pH            released YPD containing HU (0.2 M) stop S phase <br /> 7.4), 0.4 M sorbitol, 1% thiodiglycol, 125 mM spermidine, 50 mM         progression, BrdU (0.4 mg/ml) mark nascent DNA synthesis. <br /> spermine) protease inhibitors  Cells lysed       90 min, genomic DNA plugs (800 ng DNA/plug) 0.5 ml ice-cold breakage buffer containing 2% Triton X-100          stored 41C combing. plugs initially stained <br /> incubated ice 5 min occasional mixing. lysed cells      YOYO-1 (Molecular Probes), digested agarase spun 16 000 g 5 min microcentrifuge 41C.        (Roche) resuspended 150 ng/ml 50 mM MES pH 5.7. <br /> chromatin pellet digested ice 10 min 100 ml ice-cold      Isolated DNA strands combed silanized coverslips. <br /> breakage buffer containing 5 mM MgCl2 5 ml DNaseI (1 mg/ml).        Appropriate antibodies used detect DNA molecules <br /> Digestion stopped adding EDTA-KOH (pH 7.4) 10 mM.             (Argene) BrdU incorporation (Sera Labs). Images                                                                         captured Leica DM6000B microscope CoolSNAP <br /> Chromatin immunoprecipitation assay                                     HQ CCD camera (Roper). Analysis track lengths performed <br /> Chromatin immunoprecipitation (ChIP) performed pre-              using MetaMorph (Universal Imaging Corp.). Adenovirus DNA viously described (Tanaka et al, 1997) modifications. Cross-       used standard convert distances base pairs. <br /> linking carried 1% <span id='am-66' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> 20 min 301C reaction quenched 125 mM <span id='am-42' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-43' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span>. Cells             <br /> harvested, washed ice-cold PBS pellet snap             <br /> frozen liquid nitrogen stored  801C. Cell lysis         <br /> performed glass beads 500 ml lysis buffer containing <br /> protease inhibitors. soluble fraction cell extract discarded insoluble pellet resuspended 500 ml       <br /> lysis buffer containing protease inhibitors. extract sonicated times 25 s 5&#226;&#8364;&#8220;6 W achieve average DNA     <br /> fragment size 500 bp using MicrosonTM XL 2000 (Misonix).           <br /> Samples spun twice 2 min 4500 g remove cell         <br /> debris. Immunoprecipitation carried 30 ml anti-goat        <br /> IgG agarose beads (Sigma, A9294), preincubated          <br /> 2 mg anti-Mcm2 (Santa Cruz, sc-6680) anti-Mcm5 (Santa       <br /> Cruz, sc-6686) antibodies. Samples incubated overnight         <br /> 41C rotator.                                                       <br />                                                                    <br /> DNA combing                                                             <br /> DNA combing performed Versini et al (2003). brief, wt   GAL1-ORC6 cultures released synchronously G1               <br /> following initial arrest factor subsequent 4 h          <br />  <br />  <br />  <br />  <br /> </body></html>