<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>        YsxC, Putative <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span>-Binding Protein Essential Growth                             Bacillus subtilis 168 <br />  <br />              YsxC member family GTP-binding proteins carried diverse range organisms bacteria <br />           yeasts, plants, humans. resolve issue ysxC Bacillus subtilis essential growth, <br />           attempted construct mutants ysxC inactivated placed control           inducible promoter. Viable mutants obtained case, inducer dependent, <br />           demonstrating unambiguously ysxC essential gene. <br />  <br />  <br />    GTP-binding proteins frequently involved regulatory                 ferase, ydiE encodes putative metalloprotease (probably <br /> pathways ubiquitous molecular switches (6), operating                   o-sialoglycoprotein endopeptidase). conserved essen- <br /> various stages growth life cycles. majority               tial bacterial genes nonessential orthologs yeast repre- <br /> proteins <span id='am-40' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span> <span id='am-25' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-26' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>-binding proteins,                sent potential targets novel broad-spectrum antimicrobial <br /> use GTP substrate phosphorylation guanylation                 agents (6). <br /> (10). Bacillus subtilis, GTP-binding proteins involved                 Construction integrational mutations ysxC. deter- <br /> translation initiation elongation; cell division; protein se-             ysxC essential growth B. subtilis 168, <br /> cretion signal recognition particle pathway; biosynthesis             integrational mutants constructed ysxC ei- flagella; synthesis adenylosuccinate, pyrimidine, folic             ther inactivated (knockout mutant) placed control <br /> acid, riboflavin; oxidation thiophene (7).                 tightly regulated Pspac promoter (23) (fusion mutant). function small putative GTP-binding proteins (e.g.,                case knockout mutant, 303-bp internal fragment Obg, Bex, EngA, YloQ, YsxC, YyaF)   unknown.                    ysxC (bp 2879356 2879054 [7]) cloned integra- <br /> YloQ essential growth B. subtilis (2), Bex            tional plasmid pMUTIN4 (20), resulting plasmid pYSXCK shown complement Escherichia coli essential gene                  (Table 1). construction fusion mutant, 164-bp <br /> era (EMBL accession  U18532, available http://www.embl                  fragment 5&#226;&#172;&#732; end ysxC, incorporating ribosome <br /> -heidelberg srs5/). Obg essential growth                   binding site (RBS) start codon ysxC, cloned <br /> sporulation B. subtilis (18, 21) required acti-             pMUTIN4, resulting plasmid pYSXCF (Table 1). <br /> vation &#226;&#65533;&#180;B stress (17).                                                     ysxC essential growth B. subtilis 168,    putative GTP-binding protein YsxC  called OrfX                    possible isolate integrational mutant [13]) encoded bicistronic operon includes lonA (Fig.             pYSXCK, integration pYSXCF lead isopro- <br /> 1A), encodes cytosolic <span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>-dependent serine endo-                     pyl-&#226;&#65533;&#164;-D-thiogalactopyranoside (IPTG)-dependent growth.  <br /> peptidase (22). ysxC likely transcribed                peated unsuccessful attempts generate lonA, start codon overlaps lonA coding sequence                 pYSXCK-based knockout mutant, irrespective presence ysxC-specific promoter transcriptional initiation site              IPTG. case pYSXCF-based fusion mutant, detected. Furthermore, lonA ysxC showed similar                  100 erythromycin-resistant (Emr) lincomycin-resis- <br /> transcription patterns, including induction heat                 tant (Lmr) transformants &#226;&#65533;&#174;g pYSXCF DNA iso- <br /> stresses (13).                                                                lated presence IPTG, isolated    insertional inactivation ysxC previously reported               absence. data indicate ysxC essential effect growth, did result demon-                  growth. authenticity integration event strable phenotype (16).  recent genome-                     mutant (BFA2414) (Fig. 1A) confirmed PCR (Fig. 1B). based approach designed identify essential genes E. coli,               ensure tight regulation Pspac promoter, BFA2414 B. subtilis, Saccharomyces cerevisiae, ysxC B. subtilis            transformed plasmid p65, provides multiple copies <br /> reported essential (2). study, lacked exper-              E. coli lacI gene. <br /> imental data, genes unknown function es-                     Expression ysxC essential growth. IPTG  <br /> sential growth E. coli, orthologs essential              pendence BFA2414(p65) confirmed growing B. subtilis essential S. cerevisiae.             organism exponential phase (optical density 600 nm essential B. subtilis genes,  obg, yloQ, ysxC,            [OD600] &#226;&#171;&#189; &#226;&#172;&#402;0.3) Luria-Bertani (LB) medium (15) contain- <br /> encode putative GTP-binding proteins, yrrA  called                  ing 1 mM IPTG 0.3 &#226;&#65533;&#174;g erythromycin 10 &#226;&#65533;&#174;g trmU, SWISS-PROT accession  O35020) encodes putative                     kanamycin ml. cells washed twice pre- <br /> tRNA, (5-methylaminomethyl-2-thiouridylate)-methyltrans-                      warmed LB medium diluted 10&#226;&#171;&#186;5-fold prewarmed LB <br />                                                                               media containing erythromycin, kanamycin, range   * Corresponding author. Mailing address: Department Microbi- <br />                                                                               concentrations IPTG 0 1 mM. shown Fig. 2, <br /> ology Immunology, Medical School, University Newcastle             growth eventually observed cultures.  Tyne, Framlington Pl., Newcastle Tyne, NE2 4HH, United               time growth observed increas- <br /> Kingdom. Phone: 44 191 222 7708. Fax: 44 191 222 7736. E-mail: Colin          ingly delayed decreasing IPTG concentrations. case <br /> .Harwood@ncl.ac.uk.                                                           culture 0.1 mM IPTG, growth delayed  <br />                                                                        6819 <br />  6820         NOTES                                                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Construction fusion B. subtilis mutant ysxC.   Schematic representation ysxC region BFA2414 integration pYSXCF, pMUTIN4-based integration plasmid. Filled arrows indicate structural genes, putative &#226;&#65533;&#179;-independent terminators shown stem-loop structures. promoters upstream lonA (PlonA) marked fine broken arrows. Striped boxes tandem duplication RBS 5&#226;&#172;&#732; end ysxC. ysxC&#226;&#172;&#732; 5&#226;&#172;&#732; <br /> end ysxC. Plasmid pMUTIN4 shown line. lacZ reporter gene, lacI, ampicillin resistance (Apr) Emr genes marked fine arrows, promoter Pspac marked fine broken arrow pMUTIN4. region pMUTIN4 used replication E. coli labeled &#226;&#8364;&#339;ori,&#226;&#8364;&#65533; terminators (t1t2t0) upstream Pspac indicated stem-loop structure. arrows genes indicate location orientation primers, dashed lines indicate expected PCR products. positions primers specific B. subtilis 168 respect entire genome (7, 11) follows: <br /> DS-REV, 2879355 2879372; FS-REV, 2879433 2879450; FS  2879596 2879580;  2879970 2879954. 5&#226;&#172;&#732; ends forward   <br /> primers included 10-bp linker HindIII restriction site, reverse (REV) primers included 9-bp linker BamHI site. Positions primers <br /> specific pMUTIN4 follows: MUT  147 165, MUT-REV, 361 379. numbers dashed lines correspond lanes agarose <br /> electrophoresis gel shown panel B. (B) Diagnostic PCR confirming correct integration pYSXCF ysxC mutant BFA2414. Lane 1, 100-bp ladder <br /> (Amersham Pharmacia Biotech , Little Chalfont, United Kingdom). PCR performed BFA2414 (lanes 2 7) B. subtilis 168 (lane 8) chromosomal <br /> DNA. primers  expected product lengths) follows: lane 2, FS DS-REV (261 bp); lane 3, MUT DS-REV (399 bp); lane 4, FS-REV (557 bp); lane 5, MUT-REV (611 bp); lane 6, MUT MUT-REV  PCR product expected primers oriented away ; lane 7, DS-REV  PCR product expected polymerization reaction time 50 s short synthesis 9,409-bp <br /> fragment); lane 8, DS-REV (635 bp). <br />  <br />  <br /> approximately 2 h respect culture containing 1 mM                          sidase production increased exponential phase, reach- <br /> IPTG, growth culture added IPTG                               ing peak ca. 22 nmol ONP/min/OD600 unit delayed 20 h. exponential phase,                               transition exponential stationary growth <br /> mean generation times cultures smaller amounts (&#226;&#177;&#8226;0.1                           phase;  values declined. <br /> mM) IPTG increased &#226;&#172;&#402;50 min, compared &#226;&#172;&#402;30                                  determine delayed growth decreased <br /> min culture 1 mM IPTG.                                                    growth rate BFA2414(p65) lower  e., &#226;&#177;&#8226;0.1 mM) <br />   transcription ysxC monitored (9) fusing                            IPTG concentrations (Fig. 2) overgrowth sup- <br /> spoVG-lacZ reporter gene pMUTIN4 native promoter                             pressor mutants, stationary-phase samples plated  <br /> (Fig. 1A). Irrespective IPTG concentration, &#226;&#65533;&#164;-galacto-                          termine ratio IPTG-independent colonies (putative <br />  <br />  <br />                                                            TABLE 1. Bacterial strains plasmids <br />       Strain plasmid                                                 Relevant characteristic(s                                               Source reference <br />  <br /> B. subtilis <br />   168                                 trpC2                                                                                                       1 <br />   BFA2414                             trpC2 ysxC::pYSXCF Emr                                                                                      study (Fig. 1) <br />   BFA2414(p65)                        trpC2 ysxC::pYSXCF p65 Emr Kmr                                                                              study <br />   BFA2414SUP(p65)                     trpC2 ysxC::pYSXCF p65 Emr Kmr, constitutive Pspac promoter                                                 study <br />  <br /> Plasmids <br />   pMUTIN4                             Apr Emr spoVG-lacZ Pspac (8.61 kb)                                                                          20 <br />   pYSXCK                              pMUTIN4 containing 303-bp internal fragment ysxC (8.913 kb)                                            study <br />   pYSXCF                              pMUTIN4 containing 164-bp fragment RBS 5&#226;&#172;&#732; end ysxC (8.774 kb)                                 study <br />   p65                                 pUB110-based, Kmr PpenP::lacI (4.962 kb)                                                                    S. D. Ehrlich <br />         Kmr, kanamycin resistance. <br />  VOL. 182, 2000                                                                                                                 NOTES       6821 <br />  <br />  <br />                                                                              taken selective pressure applied control- <br />                                                                              lable promoter  <br />                                                                                 YsxC protein family. YsxC member family small <br />                                                                              GTP-binding proteins carried diverse range                                                                              organisms bacteria yeast, plants, humans. anal- <br />                                                                              ysis 29 members YsxC protein family (Fig. 3) shows <br />                                                                              classified distinct phylogenetic groups. <br />                                                                              Group includes YsxC orthologs gram-negative bacteria <br />                                                                              Mycoplasma species, group II includes orthologs                                                                              gram-positive bacteria (e.g., Bacillus, Clostridium, Staphy- <br />                                                                              lococcus), group III includes orthologs Archaea,                                                                              group IV includes orthologs S. cerevisiae, Homo sapiens, <br />                                                                              Arabidopsis thaliana,  interestingly, hyperthermophilic <br />                                                                              bacteria, Aquifex aeolicus Thermotoga maritima. <br />                                                                                 group II organisms, ysxC located downstream lonA                                                                              clpX, code class III ATP-dependent heat <br />    FIG. 2. OD600 BFA2414(p65) LB media following concentra- <br /> tions IPTG: 1 mM (}), 0.1 mM (&#226;&#8211;&#160;), 0.01 mM (&#197;&#8217;), 0.001 mM (F), 0.0001 mM    shock proteases. case Bacillus Brevibacillus, ysxC <br /> (&#226;&#8364;&#353;), 0 mM (E).                                                           located downstream lonA, start codon overlaps                                                                              3&#226;&#172;&#732; end lonA coding sequence nucleotide.                                                                              Clostridium difficile Clostridium acetobutylicum, coding <br /> suppressor mutants) total CFU. marked differ-                 sequence ysxC overlaps 3&#226;&#172;&#732; end lonA nucleo- <br /> ence plating efficiencies various cultures.             tides. Streptococcus spp., Enterococcus faecalis, Staphylococ- <br /> case culture 1 mM IPTG, plating efficiency               cus aureus, Lactococcus lactis, ysxC putative operon <br /> (4.9 &#226;&#171;&#190; 1.2) &#226;&#171;&#187; 10&#226;&#171;&#186;7. number IPTG-independent                       clpX. L. lactis start codon ysxC overlaps 3&#226;&#172;&#732; <br /> colonies increased IPTG concentration decreased,                  end clpX coding sequence nucleotide. cultures 0.001 mM IPTG exhibited plating effi-                phylogenetic groups   III, IV), lonA clpX <br /> ciency close 1. data suggest growth                 located distal site chromosome respect cultures accumulation derivatives              ysxC. Linkage ATP-dependent protease ysxC <br /> suppressor mutations.                                                        observed Pyrococcus horikoshii group III,    determine location putative suppressor muta-                    ysxC ortholog, PH0200, located 59 bp downstream <br /> tion(s), chromosomal DNA IPTG-independent colonies                      gene encoding putative regulatory subunit ATP- <br /> [BFA2414SUP(p65)] used transform B. subtilis 168.                  dependent 26S protease. addition YsxC orthologs eu- case, similar numbers (ca. 103/&#226;&#65533;&#174;g DNA) Emr Lmr                    karyotes, bacterial homologs  aeolicus T. ma- <br /> transformants observed, irrespective presence                 ritima belong group IV.  aeolicus earliest <br /> IPTG. contrast, chromosomal DNA IPTG-                       diverging thermophilic bacteria known, dependent BFA2414 used, transformants (ca. 9 &#226;&#171;&#187; 102/&#226;&#65533;&#174;g                    chemolithoautotroph, grow hydrogen, oxygen, carbon DNA) obtained presence IPTG.                    dioxide, mineral salts (5). T. maritima deepest <br /> results suggested observed suppression linked                slowly evolving lineages Eubacteria. integrated pYSXCF.                                                       core T. maritima eubacterial, quarter <br />    possibility suppressor mutation(s) occurred              genome archaeal nature (12).  thaliana &#226;&#8364;&#339;oid&#226;&#8364;&#65533; lac operator (14) Pspac promoter, leading             homologs YsxC, 219 318 amino constitutive expression. oid, ideal, lac operator             acid residues length, encoded chromosomes II V, <br /> perfect symmetry 10-fold-higher affinity Lac                respectively (8). length smaller homolog pressor native lac operator. Consequently, native               range prokaryotic YsxC proteins (190 219 amino acids). <br /> lac operator associated Pspac promoter integra-                  length longer YsxC homolog similar tional vector pMUTIN2 replaced oid operator                    YsxC orthologs S. cerevisiae H. sapiens. respect <br /> pMUTIN4 reduce noninduced level expression                 YsxC B. subtilis, proteins 100- 110-amino- <br /> promoter (20). Pspac promoter regions IPTG-                 acid extension amino terminus. N termini, independent BFA2414SUP(p65) mutants PCR amplified                       highly conserved S. cerevisiae H. sapiens using primers MUT DS-REV (Fig. 1A). Sequencing                        thaliana, showed homology bacterial PCR products revealed BFA2414SUP                      archaeal proteins. case  thaliana, N terminus <br /> (p65) mutants contained single C3T transition nucleotide                contained putative transmembrane helix (19) amino <br /> 10 oid lac operator. identical base pair substitution              acid residues 11 29 protein currently nucleotide native lac operator                   putative membrane-bound member YsxC family. <br /> shown decrease affinity Lac repressor 96%                 homologs ysxC observed complete ge- generate constitutive phenotype (3).                   nome sequences Borrelia burgdorferi, Chlamydia trachomatis, <br /> cluded observed IPTG-independent growth                   Chlamydia pneumoniae, Deinococcus radiodurans, Mycobacte- selection clones mutation integrated                    rium tubercolosis, Treponema pallidum using databases pYSXCF severely reduces capacity lac opera-                  Institute Genomic Research (TIGR; http://www.tigr <br /> tor upstream functional ysxC gene bind <span id='am-9' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span>lactose</span>                 .org/tdb) Pasteur Institute (http://genolist.pasteur.fr). <br /> repressor. single spontaneous mutation lac op-                   alignment YsxC 28 homologs (data shown) <br /> erator pMUTIN4 result loss IPTG depen-                  revealed regions conservation:   (G/R)X(S/T)N(V  <br /> dence target gene expression, lead underes-                GKS(S/T), putative GTP-binding motif located timation number essential genes gene fusion                 amino terminus; (ii) PGXTXXX(N , located 15 23 resi- gene knockout strategy initially used isolate            dues downstream region; (iii) DXPG(Y/F)G(Y/F), mutants. results indicated appropriate care needs              second putative GTP-binding motif located 10 20 residues <br />  6822       NOTES                                                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Radial phylogenetic tree YsxC protein family. Multiple-sequence alignment phylogenetic analysis performed using web site Institut National la Recherche Agronomique (http://www.toulouse.inra.fr/multalin.html; reference 4). symbol comparison table Blosum62, gap <br /> weight 12, gap length weight 2. root tree marked triangle. Proteins SWISS-PROT accession numbers  parentheses) follows: B. subtilis YsxC (P38424), E. coli YihA (P24253), Haemophilus influenzae HI1118 (P46453), Helicobacter pylori HP1567 (O26087), Mycoplasma genitalium <br /> MG335 (P47577), M. pneumoniae MP359 (P75303), Methanococcus jannaschii MJ0320 (Q57768), Rickettsia prowazekii RP102 (Q9ZE46),  aeolicus AQ_1815 <br /> (O67679), Caulobacter crescentus CgpA (Q9ZG89). Proteins GenBank accession number follows: Archaeoglobus fulgidus AF1326 (AAB89919), <br /> P. horikoshii PH0200 (BAA29269), T. maritima TM1466 (AAD36534), L. lactis (AAF63739), Neisseria meningitidis NMB1806 (AAF42143), Zymomonas mobilis CgpA <br /> (AAD56911), S. cerevisiae Ydr336wp (AAB64772), H. sapiens HSPC135 (AAF29099),  thaliana 219 At2g22870 (AAC32434) length 219 <span id='am-33' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-34' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> <br /> residues,  thaliana 318 F15N18_70 (CAB87708) length 318 amino acid residues. YsxC orthologs B. anthracis, B. stearothermophilus, C. aceto- <br /> butylicum, C. difficile, E. faecalis, S. aureus, S. pneumoniae, S. pyogenes obtained unfinished genome sequencing projects TIGR, Sanger Centre, <br /> Genome Therapeutics Corporation, University Oklahoma. B. stearothermophilus, 130 amino acids N terminus YsxC homolog deduced sequence contig 552. B. brevis, 162 amino acids N terminus deduced GenBank sequence (D00863) inserting nucleotide positions 2745 2746 (16). PAM, percent accepted mutation. <br />  <br />  <br />  <br />  <br /> downstream second region; (iv) KXDK, located 56                                                          74 residues downstream motif. second motif                         shorter S. cerevisiae (GXTXXXN), threonine                           <br /> residue conserved case H. sapiens.                                       <br />                                                                                          <br />    able generate viable mutant ysxC                             <br /> pYSXCK,  integration B. subtilis chromo-                             generates strain carrying YsxC protein trun-                        <br /> cated C terminus just 23 amino acids.                                <br />                                                                                          <br /> protein includes conserved motifs described                            indicates highly charged (5 K residues, 2 E resi-                          <br /> dues, 1 D residue, 1 R residue, serine dyad) C terminus                        essential function.                                                               <br />   thank S. D. Ehrlich gift plasmids pMUTIN4 p65.                     acknowledge release preliminary sequence data                           <br /> YsxC orthologs Bacillus anthracis, Bacillus stearothermophilus, C.                 <br /> acetobutylicum, C. difficile, E. faecalis, S. aureus, Streptococcus pneu-               <br /> moniae, Streptococcus pyogenes, obtained using                        <br /> TIGR website (http://www.tigr.org) unfinished genome sequenc- <br />                                                                                          <br /> ing projects TIGR, Sanger Centre, Genome Therapeutics Cor-                        <br /> poration, University Oklahoma.                                               <br />   work funded European Commission (BIO4-CT95-                              <br /> 0278). <br /> </body></html>