<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Htm1p, mannosidase-like protein, involved glycoprotein degradation yeast <br />  <br /> Misfolded proteins recognized endoplasmic reticulum                   N-linked oligosaccharides processed glucosidase  <br /> (ER), transported cytoplasm degraded                      glucosidase II mannosidase (Herscovics, 1999a). proteasome. Processing intermediates N-linked oligo-                          shown oligosaccharide processing protein folding <br /> saccharides incompletely folded glycoproteins                         linked processes coordinating protein folding, ER retention/ <br /> important role folding/refolding,                     retrieval protein degradation (Ellgaard et al., 1999; <br /> targeting proteolytic degradation. Saccharomyces                           Herscovics, 1999b; Jakob Burda, 1999). <br /> cerevisiae, identified gene coding non-essential                    Mechanisms discern misfolded secretory membrane <br /> protein homologous mannosidase (HTM1)                      proteins discovered past years (Bonifacino required degradation glycoproteins. Deletion                        Weissman, 1998). Misfolded proteins ER lumen HTM1 gene does affect oligosaccharide trimming.                          specifically recognized unknown mechanisms  deletion HTM1 does reduce rate degradation                    subjected cytosolically located proteolytic process termed mutant glycoproteins carboxypeptidase Y, ABC-                     ER-associated degradation (ERAD). targeted proteins transporter Pdr5-26p oligosaccharyltransferase subunit                       transported cytoplasm mechanism Stt3-7p, mutant Sec61-2p, non-glycoprotein.                     formed, partially, constituents ER import <br /> results indicate Htm1p involved                          machinery (R&#195;&#182;misch, 1999). retrotranslocated proteins <br /> processing N-linked oligosaccharides, required                polyubiquitin modified action cytosolically <br /> proteolytic degradation. propose mannosidase                        located enzymes Ubc6p, Ubc7p Hrd1/Der3p, followed homolog lectin recognizes Man8GlcNAc2 oligosaccharides                 degradation proteasome (Brodsky McCracken, 1997; serve signals degradation pathway.                                Kopito, 1997; Sommer Wolf, 1997; Bays et al., 2001; Deak <br />                                                                                  Wolf, 2001). <br />                                                                                     degradation misfolded mutant carboxypeptidase Y [CPY* <br />                      INTRODUCTION                                                (prc1-1); Finger et al., 1993] shown N-glycan secretory membrane proteins translationally                      dependent (Knop et al., 1996)   structure translocated lumen endoplasmic reticulum (ER)                    oligosaccharide plays important role rapid protein <br /> (Rapoport et al., 1996) acquire N-linked oligo-                       degradation (Jakob et al., 1998). Specifically, glycoproteins <br /> saccharides, catalyzed oligosaccharyltransferase.                 carrying Man8GlcNAc2 sugars rapidly degraded ER, secretory membrane proteins attain correct                    finding supported &#226;&#8364;&#732;mannose timer hypothesis&#226;&#8364;&#8482; <br /> dimensional structure concerted action chaperones,                     (Su et al., 1993; Helenius et al., 1997) led isomerases folding mediators (Ellgaard et al., 1999).                    proposal ER lumen contains Man8GlcNAc2-binding <br />  <br />  <br />  +Corresponding   author. Tel: +41 1 632 3327; Fax: +41 1 632 1148; E-mail: jakob@micro.biol.ethz.ch <br />  &#226;&#8364;&#161;Present  address: Clinic Neurosurgery, University Hospital Zurich, CH-8091 Zurich, Switzerland <br />  &#194;&#167;Present address: Cytos Biotechnology AG, CH-8952 Schlieren-Zurich, Switzerland <br />  &#194;&#182; Present address: Biochemistry Department, McGill University, Montreal, PQ, H3G 1Y6, Canada <br />  <br />  C  Jakob D. Bodmer contributed equally work <br />  <br />  <br />  <br /> &#194;&#169; 2001 European Molecular Biology Organization                                                         EMBO reports vol. 2 |  5 | pp 423&#226;&#8364;&#8220;430 | 2001 423 <br />                                                         scientific reports <br />      C  Jakob et al. <br />  <br />      protein initiates glycoprotein ERAD (Liu et al., 1997; Jakob <br />      et al., 1998; Yang et al., 1998). <br />         yeast genome identified good candidate      Man8GlcNAc2-specific lectin (Htm1p) homologous      mannosidase (Camirand et al., 1991). strains <br />      deleted HTM1 defective degradation mutant <br />      soluble protein (CPY*), mutant membrane proteins <br />      (Pdr5-26p, Stt3-7p). data strongly suggest Htm1 <br />      protein putative Man8GlcNAc2-binding lectin. <br />  <br />               RESULTS DISCUSSION <br />      HTM1 codes gene homologous mannosidase <br />           involved protein degradation <br />      searched Saccharomyces cerevisiae genome open <br />      reading frames homology ER mannosidase      hypothesized &#226;&#8364;&#732;degradation lectin&#226;&#8364;&#8482; contain similar <br />      domains. reading frame YHR204w (termed gene homo- <br />      logous mannosidase; HTM1) identified, showing 40% <br />      <span id='am-12' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> sequence similarity yeast mannosidase. <br />         test HTM1 involved protein degradation, <br />      deleted gene yeast strain containing CPY* (Wolf      Fink, 1975) Pdr5* [mutant Pdr5 protein (pdr5-26); Egner et <br />      al., 1998], substrates previously shown      undergo ubiquitin-dependent proteasomal degradation.      controls, used mutant strains deficient      biosynthesis complete oligosaccharide chains (&#226;&#710;&#8224;alg12)      trimming oligosaccharides (&#226;&#710;&#8224;mns1). mutations <br />      prevent biosynthesis Man8GlcNAc2 oligosaccharide      formation processing, respectively, stabilize misfolded <br />      CPY* (Jakob et al., 1998). Equal numbers cells inocu- <br />      lated novo protein synthesis stopped addition <br />      cycloheximide; levels CPY* Pdr5* determined <br />      immunoblotting quantified different time points.      CPY* degradation rate &#226;&#710;&#8224;alg12 &#226;&#710;&#8224;mns1 cells reduced <br />      comparison &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; cells, HTM1- <br />      deficient cells (Figure 1). addition, higher <br />      steady-state level CPY* &#226;&#710;&#8224;alg12, &#226;&#710;&#8224;mns1 &#226;&#710;&#8224;htm1 cells      compared wild-type cells (time point 0; Figure 1A, compare <br />      lanes 1, 4, 7 10). mutant membrane protein Pdr5* <br />      showed similar change degradation: steady-state levels      Pdr5* increased &#226;&#710;&#8224;alg12 &#226;&#710;&#8224;mns1, &#226;&#710;&#8224;htm1 <br />      cells (Figure 1), degradation rates significantly reduced <br />      (Figure 1). <br />         test HTM1 involved general protein degra- <br />      dation, studied mutant Sec61-2 protein, membrane <br />      protein N-glycosylation sites. sec61-2 mutant cells      Fig. 1. Oligosaccharide-dependent degradation mutant proteins CPY* <br />      display temperature-sensitive growth phenotype. Biederer et        Pdr5*.   Isogenic cells grown mid-log phase. Equal cell <br />                                                                           numbers incubated 100 &#194;&#181;g/ml cycloheximide aliquots <br />      al. (1996) showed Sec61-2 protein degraded restric-      removed given time points. Crude protein extract separated SDS&#226;&#8364;&#8220;PAGE <br />      tive temperature. Impairing protein degradation deleting      analyzed immunoblotting using anti-CPY (CPY*) anti-HA <br />      UBC6 UBC7 genes allowed cells survive restrictive         (Pdr5*) antisera. (B) Protein amounts independent <br />      growth conditions (Figure 2A). deleted MNS1 HTM1          experiments quantified plotted time, setting quantity                                                                           time point 0 (steady-state level) 100%. <br />      genes sec61-2 mutants tested      genes influenced degradation Sec61-2 protein cell <br />      growth selective temperature western blot analysis. <br />      mutant cells (sec61-2 &#226;&#710;&#8224;mns1 sec61-2           (Figure 2A) reduced degradation rate Sec61-2 protein <br />      &#226;&#710;&#8224;htm1) improved growth plates restrictive temperature          (Figure 2B). <br />  <br />  <br /> 424 EMBO reports vol. 2 |  5 | 2001 <br />                                                            scientific reports <br />                                                                                                  Role Htm1p glycoprotein degradation yeast <br />  <br />  <br />  <br />  <br /> Fig. 2. HTM1 gene does influence degradation mutant Sec61-2 protein.   &#226;&#710;&#8224;mns1 &#226;&#710;&#8224;htm1 suppress temperature-sensitive growth sec61-2 mutant cells. Isogenic sec61-2 single- double-mutant cells plated serial dilutions rich medium grown 30 37&#194;&#176;C 3 days. <br /> (B) Degradation Sec61-2 protein affected mns1- htm1-deficient cells. Cells grown 30&#194;&#176;C mid-log phase shifted 37&#194;&#176;C 2 h. Equal <br /> cell numbers incubated 100 &#194;&#181;g/ml cycloheximide, aliquots removed cells broken given time points. Extracted proteins separated SDS&#226;&#8364;&#8220;PAGE, analyzed quantified immunoblotting using anti-Sec61 antiserum. protein quantity time point 0 (steady-state level) set 100%. <br />  <br />  <br />  <br />  <br />    results support hypothesis HTM1 gene                       sequences similar Htm1 protein fungal, plant, <br /> required efficient secretory membrane glycoprotein                          insect, worm mammal genomes. Phylogenetic analysis degradation. Degradation proteins lacking N-glycans                      sequences revealed distinct sequence groups, affected. Glycoprotein degradation require specific                         contained class &#206;&#177;1,2-mannosidases lectin recognizes N-linked Man8GlcNAc2 oligosaccharides                        proteins homologous Htm1p (data shown). (Jakob et al., 1998). reasoned lectin                 amino acids reported involved oligosaccharide binding, similar ER mannosidase  HTM1 gene codes                           catalysis Ca2+ coordination Mns1p (Vallee et al., 2000) <br /> polypeptide 796 amino acids significant sequence                            conserved Htm1 protein homologs. similarity yeast mannosidase (Figure 3A), amino acid                       characteristic features Htm1 protein family group  <br />  <br />                                                                                                                           EMBO reports vol. 2 |  5 | 2001 425 <br />                                                                 scientific reports <br />      C  Jakob et al. <br />  <br />  <br />  <br />  <br />      Fig. 3. Similarity Mns1p Htm1 proteins.   Areas homology      Mns1p Htm1p. Areas high amino acid similarity drawn boxes.        Fig. 4. Normal N-linked oligosaccharide processing htm1-deficient yeast <br />      (B) Htm1 family proteins lack highly conserved cysteine residue. Amino        cells. N-linked oligosaccharides isogenic strains metabolically labeled <br />      acid sequences &#206;&#177;-mannosidases Htm1 family proteins plant,             [3H]mannose, extracted, released PNGase F separated HPLC. <br />      mammalian, insect, worm yeast cells aligned Clustal_W.      sequences grouped sequence similarity. arrowhead indicates      position alanine residue Htm1 proteins position Cys340       S. cerevisiae) conserved likely essential maintaining      correct structure &#206;&#177;1,2-mannosidases. letters indicate      obtained cells deleted HTM1 showed Man8GlcNAc2 <br />      species:  Arabidopsis thaliana; ce, Caenorhabditis elegans; dm, Drosophila   Man9GlcNAc2 oligosaccharides. results confirm <br />      melanogaster; hs, Homo sapiens; sc, S. cerevisiae; sp, Schizosaccharomyces      normal processing N-glycans occurs &#226;&#710;&#8224;htm1 cells  <br />      pombe. following numbers refer enzyme DDBJ/EMBL/ <br />                                                                                       Htm1 protein processing mannosidase. <br />      GenBank accession  <br />                                                                                      Htm1p displays sequence similarity class manno- <br />                                                                                      sidases, involved specific trimming N-glycans <br />                                                                                      ER, process requires mannosidase <br />      absence conserved cysteine residues 340 385 (position                activity yeast (Dean, 1999; Gemmill Trimble, 1999; <br />      given Mns1p S. cerevisiae), form disulfide bond.                 Herscovics, 1999a). consistent conclusion      cysteine residues conserved &#206;&#177;1,2-mannosidase                   Htm1p mannosidase, inference      family members likely important main-                    lectin. <br />      taining correct structure processing mannosidases (Lipari                   reports indicate distinct <br />      Herscovics, 1996) (Figure 3B).                                              function N-glycan trimming mannosidase glycoprotein <br />                                                                                      degradation (Liu et al., 1997, 1999; Yang et al., 1998; Cabral et <br />                HTM1 involved processing                                 al., 2000; Mancini et al., 2000; Wang Androlewicz, 2000; <br />                    N-linked oligosaccharides                                         Wang White, 2000). reports good <br />                                                                                      agreement results S. cerevisiae, specific <br />      Processing N-glycans create Man8GlcNAc2 glycan                                                                                      Man8GlcNAc2 glycan required efficient protein degradation      required efficient degradation yeast (Jakob et al., 1998                                                                                      ER (Knop et al., 1996; Jakob et al., 1998). specific <br />      . determined altered processing                                                                                      oligosaccharide degradation substrates recognized <br />      oligosaccharides reason reduced glycoprotein <br />      degradation &#226;&#710;&#8224;htm1 cells.  oligosaccharides                     Htm1 protein facilitate glycoprotein degradation. <br />      metabolically labeled vivo. N-linked sugars extracted <br />      released protein <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> N-glycosidase F                                   Degradation Stt3-7 protein      (PNGase F) treatment analyzed HPLC. Attention                                  oligosaccharide specific mediated <br />      focused sugars processed Golgi mannosyl-                              ubiquitin&#226;&#8364;&#8220;proteasomal pathway <br />      transferases (Figure 4)  expected, wild-type cells <br />      contained Man8GlcNAc2 oligosaccharides, &#226;&#710;&#8224;mns1 cells                         yeast, oligosaccharyltransferase (OST), enzyme complex <br />      contained Man9GlcNAc2 (Figure 4).  oligosaccharides                     transfers oligosaccharides <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span>-linked precursors  <br />  <br /> 426 EMBO reports vol. 2 |  5 | 2001 <br />                                                            scientific reports <br />                                                                                             Role Htm1p glycoprotein degradation yeast <br />  <br />  <br />  <br />  <br />                                                                                 Fig. 6. Oligosaccharide-dependent suppression conditional growth stt3-7 <br />                                                                                 deletion MNS1 HTM1 genes. Isogenic stt3-7 single- double-mutant <br />                                                                                 cells plated serial dilutions rich medium grown 23 30&#194;&#176;C <br />                                                                                 3 days. <br />  <br />  <br />  <br />  <br />                                                                                 OST activity, leading incomplete N-glycosylation nascent <br />                                                                                 glycoproteins restrictive temperature (Spirig et al., 1997).                                                                                 observed overexpression deletion UBC6 gene <br />                                                                                 suppress temperature-sensitive phenotype mutant <br />                                                                                 stt3-7 strains (Figure 5A). <br />                                                                                    UBC6 encodes E2 ubiquitin-conjugating enzyme situated <br />                                                                                 cytosolic face ER membrane plays pivotal <br />                                                                                 role degradation misfolded proteins ER (Sommer <br />                                                                                 Wolf, 1997). Overexpression deletion UBC6 gene <br />                                                                                 increased steady-state level mutant Stt3-7 protein                                                                                 levels OST subunits Ost3p Ost6p, shown                                                                                 western blot analysis (Figure 5B). N-linked glycosylation                                                                                 strongly affected stt3-7 mutant cells (visualized                                                                                 presence hypoglycosylated CPY) slightly <br />                                                                                 improved reducing protein degradation using UBC6 mutants <br />                                                                                 (Figure 5B). finding shows mutant Stt3-7 protein Fig. 5. UBC6-dependent degradation stt3-7 protein.   Overexpression     degraded ubiquitin&#226;&#8364;&#8220;proteasome pathway restrictive <br /> deletion UBC6 gene suppressed temperature-sensitive growth           temperature. speculated reducing protein degradation <br /> phenotype stt3-7 mutant cells. Isogenic stt3-7 single- double-mutant     deleting UBC6 gene increased steady-state level cells plated serial dilutions rich medium grown 23 30&#194;&#176;C   Ost3p Ost6p mutant Stt3-7 protein, permitting 3 days. (B) Overexpression deletion UBC6 gene stabilized mutated Stt3-7 protein. Cells shifted 37&#194;&#176;C protein extracted.   integration Ost3p, Ost6p Stt3-7p partially functional <br /> Steady-state protein levels CPY glycosylation determined            oligosaccharyltransferase complex. partially restored enzymatic <br /> western analysis using indicated antisera. Hexokinase used       activity increases N-linked glycosylation allow mutant <br /> control equal protein loading.                                              cells survive restrictive temperature. <br />  <br />                                                                                       Deletion gene (HTM1) homologous                                                                                      mannosidases suppresses stt3-7 mutation <br /> nascent polypeptides N-X-S/T, composed membrane proteins (Knauer Lehle, 1999), Stt3p                                                                                       stabilizes misfolded protein ER largest subunit essential N-linked glyco-               UBC6-dependent growth stt3-7 strains convenient <br /> sylation process vivo (Zufferey et al., 1995). Assembly               method monitor effect mutant genes process. OST complex multistep process, integration               reduced protein degradation rescue mutant cells, Ost3p Ost6p subunit dependent presence Stt3p                  reasoned similar results creating (U. Spirig, D. Bodmer, M. Wacker M. Aebi, preparation).                  stt3-7 &#226;&#710;&#8224;mns1 stt3-7 &#226;&#710;&#8224;htm1 double mutants Stt3 protein stt3-7 conditional mutant shown severely impair                 N-glycosylated. validation assay, &#226;&#710;&#8224;mns1 suppressed <br />  <br />  <br />                                                                                                                      EMBO reports vol. 2 |  5 | 2001 427 <br />                                                                  scientific reports <br />      C  Jakob et al. <br />  <br />      Table  Yeast strains used <br />  <br />      Strain                     Genotype                                                                                  Reference <br />      SS328                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801                                                   Vijayraghavan et al. (1989) <br />      YG543                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 leu2 stt3-7                                       Spirig et al. (1997) <br />      YG885                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;ubc6::HIS3                                       study <br />      YG950                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 &#226;&#710;&#8224;mns1::KanMX                               study <br />      YG894                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 leu2 stt3-7 Dubc6::HIS3                           study <br />      YG618                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 prc1-1                                            Jakob et al. (1998) <br />      YG777                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 tyr1 &#226;&#710;&#8224;mns1::KanMX prc1-1                                   Jakob et al. (1998) <br />      YG807                      MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;alg12::KanMX prc1-1                              Jakob et al. (1998) <br />      YG1211                     MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;mns1::KanMX prc1-1                               study <br />      YCJ1                       MAT&#206;&#177; ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 &#226;&#710;&#8224;htm1::KanMX prc1-1                               study <br />      YCJ82                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 Dhtm1::KanMX                               study <br />      YCJ88                      MATa ade2-101 ura3-52 his3&#226;&#710;&#8224;200 lys2-801 stt3-7 Dhtm1::KanMX                               study <br />      RSY533                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52                                          R. Schekman <br />      YG1350                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;ubc6::KanMX                             study <br />      YG1351                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;ubc7::KanMX                             study <br />      YG1352                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;mns1::KanMX                             study <br />      YG1353                     MAT&#206;&#177; sec61-2 ade2 leu2-3 leu2-112 pep4-3 ura3-52 &#226;&#710;&#8224;htm1::KanMX                             study <br />  <br />  <br />  <br />  <br />      Table II. Primers gene deletions <br />  <br />      Deletion                                                         Sequence 5&#226;&#8364;&#178; &#226;&#8224;&#8217; 3      HTM1                              HR204forKan                         tcgatcatgtagcatgcgaagacgatgcgtactcattcacttctatcgatgaattcgagctc <br />                                        HR204revKan                         gccattggaagtgagcacaggactatgtttcttgatttgtacaccgtacgctgcaggtcgac <br />                                        HR204&#226;&#8364;&#8220;90u                           gcaataaagaaggacgcg <br />                                        HR204+350L                          ccggtcttccagtatagc <br />      UBC6                              UBC6forKan                          gatggtggaaaaccctccaccatatattcttgctcgccccaacgaagattcgatgaattcgagctc <br />                                        UBC6revKan                          tgaagaactatcattaggttctttgccatctttcttggaattattttctgcgtacgctgcaggtcgac <br />                                        UBC6&#226;&#8364;&#8220;180u                           actaccatcgcatatcgc <br />                                        UBC6+130L                           gttatcattctgatagccg <br />      UBC7                              UBC7forKan                          ctcctcaaggagcttcaacagttaattaaagattctccacctggtatatcgatgaattcgagctc <br />                                        UBC7revKan                          ctcactcaacatgctcataacacttaatagaattttttctacactttgcgtacgctgcaggtcgac <br />                                        UBC7&#226;&#8364;&#8220;70u                            ggcgtttagcgtacgaag <br />                                        UBC7+130L                           agcgtatggcgtatctgg <br />                                    KanMXu                              gtattgatgttggacgag <br />  <br />      aBold    letters represent sequences specific plasmid pFA6aKanMX4 amplification KanMX resistance gene (Wach et al., 1994). <br />  <br />  <br />  <br />      temperature sensitivity stt3-7 cells 30&#194;&#176;C, indicating                   process. Detailed biochemical studies required      direct role N-glycan trimming degradation Stt3-7                       confirm hypothesis. <br />      protein (Figure 6).  stt3-7 &#226;&#710;&#8224;htm1 double mutant <br />      grow elevated temperature, similar stt3-7 &#226;&#710;&#8224;mns1 <br />      strain (Figure 6). result supported interpretation <br />                                                                                                                      METHODS <br />      Htm1 protein plays important role efficient degradation                  Materials. Yeast strains used detailed Table  strains <br />      mutant glycoprotein Stt3-7p.                                                 harboring prc1-1 allele described previously (Jakob <br />         phenotypic analysis &#226;&#710;&#8224;htm1 mutant strains compatible                    et al., 1998). pRS313 HA-pdr5-26 plasmid kindly <br />      idea Htm1 protein fact lectin involved                   provided Dr D.H. Wolf (University Stuttgart, Germany). <br />  <br />  <br />                                                                                            <br />  western blot analysis, antisera raised CPY, Sec61p,                <br /> Ost3p, Ost6p (Dr R. Knauer, University Regensburg,                             <br /> Germany), hexokinase (Dr S. Schr&#195;&#182;der, G&#195;&#182;ttingen, Germany)                           anti-hemagglutinin (anti-HA) (Santa Cruz, CA) used.                     <br /> Yeast manipulations. Standard protocols followed yeast <br />                                                                             <br /> manipulation (Guthrie Fink, 1991). Cells grown                          <br /> permissive temperature YPD (2% bactopeptone, 1%                          <br /> yeast extract, 2% glucose)  maintaining plasmids,                    <br /> minimal medium (0.67% yeast nitrogen base, 2% glucose,               appropriate supplements allow growth). cell                             <br /> protein extracted previously described (Jakob et al., 1998).              <br /> Extracts containing HA-tagged Pdr5* gene incubated                <br /> 50&#194;&#176;C 15 min instead boiled. cycloheximide chase                    <br /> performed described Plemper et al. (1998). Serial dilutions                cells growth assays, starting 5 &#195;&#8212; 105 cells, spotted <br />                                                                                     YPD plates incubated given temperatures                                                                             <br /> Construction yeast strains. loci YHR204w/HTM1, UBC6                     UBC7 deleted replacing major gene                  KanMX cassette (Wach et al., 1994) using primers                        <br /> given Table II. correct integration verified PCR.                   <br /> Deletion MNS1 gene described previously (Jakob et                       <br /> al., 1998).                                                                     <br /> Labeling N-linked oligosaccharides. procedures                         <br /> metabolic labeling, recovery glycoproteins release                   <br /> N-glycans treatment PNGase F (Roche Diagnostics, <br />                                                                                      <br /> Rotkreuz, Switzerland) described previously (Jakob et <br />                                                                         <br /> al., 1998).                                                                            <br />   <br /> </body></html>