<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Mar. 2005, p. 2288&#226;&#8364;&#8220;2296                                                                       Vol. 25,  6 <br /> 0270-7306/05/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.25.6.2288&#226;&#8364;&#8220;2296.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />    Inactivation Pre-mRNA Cleavage Polyadenylation Factor <br />           Pfs2 Fission Yeast Causes Lethal Cell Cycle Defects <br />    Shao-Win Wang,1&#226;&#8364;&#160; Kazuhide Asakawa,2&#226;&#8364;&#160;&#226;&#8364;&#161; Thein  Win,1&#226;&#8364;&#160; Takashi Toda,2 Chris J. Norbury3* <br />       Department Zoology1 Sir William Dunn School Pathology,3 University Oxford, Oxford, Cell Regulation <br />                     Laboratory, London Research Institute, Cancer Research UK, London,2 United Kingdom <br />                    Received 12 October 2004/Returned modification 17 November 2004/Accepted 14 December 2004 <br />  <br />             Faithful chromosome segregation fundamentally important maintenance genome integrity          ploidy. isolating conditional mutants defective chromosome segregation fission yeast Schizosaccha- <br />          romyces pombe, identified role essential gene pfs2 chromosome dynamics. absence <br />          offunctionalPfs2,chromosomalattachmenttothemitoticspindlewasdefective,withconsequentchromosomemis- <br />          segregation. circumstances, multiple intracellular foci spindle checkpoint proteins Bub1          Mad2 seen, deletion bub1 exacerbated mitotic defects loss cell viability resulted <br />          loss pfs2 function. Progression G1 S phase following release nitrogen starvation          required pfs2&#226;&#180;&#8482; function. product orthologous Saccharomyces cerevisiae gene PFS2 component          multiprotein complex required 3&#226;&#180;&#8226;-end cleavage polyadenylation pre-mRNAs  keeping          conservation essential function, S. pombe pfs2 mutant defective mRNA 3&#226;&#180;&#8226;-end processing. <br />          Mutations pfs2 suppressed overexpression putative mRNA 3&#226;&#180;&#8226;-end cleavage factor Cft1.          data suggest unexpected links mRNA 3&#226;&#180;&#8226;-end processing chromosome replication segregation. <br />  <br />  <br />    Following DNA replication eukaryotic cells, accurate mi-              origins replication establishment totic chromosome segregation requires bivalent attach-                  maintenance cohesion sister chromatids follow- <br /> ment replicated chromosomes spindle sister                ing DNA replication. integrity chromosome  <br /> kinetochores, followed anaphase movement chromo-                  sociated complexes potentially threatened traversal somes opposite spindle poles. Defects process result             chromosomes DNA replication forks RNA polymerases. catastrophic failure mitosis cell death              view emerged identification budding <br /> aneuploidy. Spindle checkpoint mechanisms provide protec-                   yeast Saccharomyces cerevisiae DNA elements serve tion eventualities imposing delay               prevent transcriptional read centromeres ori- <br /> onset anaphase chromosomes established sym-               gins replication (27). Maintenance heterochromatin metrical, bivalent attachments tense spindle microtubules                centromeres fission yeast Schizosaccharomyces pombe (14). molecular basis checkpoint incompletely                required repression transcription understood, proteins involved iden-               accurate chromosome segregation (1). Furthermore, recent <br /> tified characterized yeast models shown perform               study demonstrated cohesin, multiprotein complex <br /> analogous checkpoint functions diverse species, including                maintains sister chromatid cohesion, accumulates metazoans plants. proteins include Bub1, Bub3,                     3&#226;&#172;&#732; ends convergent active transcription units (13). <br /> Mps1, Mad1, Mad2, Mad3, thought form                        mutual incompatibility transcription centromere <br /> heterooligomeric complex unattached kinetochores (2).                    function capacity read transcription  progression normal, unperturbed mitosis,                terfere expression downstream genes emphasize <br /> complexes expected short-lived                    importance appropriate termination transcription. <br /> highly dynamic interactions spindle microtubules                RNA polymerase II (Pol II) transcription units, termina- <br /> kinetochores. kinetochores achieved bivalent                  tion generally coupled endonucleolytic cleavage attachment, reduction Mad- Bub-dependent signal-                 primary transcript site defining 3&#226;&#172;&#732; terminus ing allows activation multisubunit ubiquitin ligase known <br />                                                                             mRNA. Termination typically occurs zone lacks anaphase-promoting complex/cyclosome subse- <br />                                                                             distinctive features level primary sequence quent progression anaphase. <br />                                                                             situated bases downstream cleavage <br />    Normal chromosome behavior requires assembly spe- <br />                                                                             site. cases, cleavage followed polyadenylation cialized protein-DNA complexes telomeres, centromeres, <br />                                                                             nascent mRNA, cleavage polyadenylation <br />                                                                             steps coordinated multisubunit protein complexes. <br />   * Corresponding author. Mailing address: Sir William Dunn School <br />                                                                             complexes conserved overall organization Pathology, University Oxford, South Parks Rd., Oxford OX1             diverse eukaryotes (23) comprise cleavage 3RE, United Kingdom. Phone: 44 1865 275540. Fax: 44 1865 275501.            polyadenylation specificity factor (CPSF) cleavage stim- <br /> E-mail: chris.norbury@path.ox.ac.uk.                                        ulation factor (CstF) mammals. S. cerevisiae, orthologues <br />   &#226;&#8364;&#160; S.-W.W., K , T W. contributed equally work. <br />   &#226;&#8364;&#161; Present address: Division Molecular Developmental Biol- <br />                                                                             subunits CPSF CstF identified <br /> ogy, Department Developmental Genetics, National Institute            components cleavage polyadenylation factor <br /> Genetics, Mishima, Shizuoka, Japan.                                         (CPF). absence efficient cleavage polyadenyla- <br />  <br />                                                                      2288 <br />  VOL. 25, 2005                                                                                                        Pfs2 CELL CYCLE                     2289 <br />  <br />  <br />                                                          TABLE 1. S. pombe strains used study <br />        Strain                                                                  Genotype                                                                 Source <br />                                                 &#226;&#171;&#186; <br /> 972                                            h                                                                                                Laboratory stock <br /> 428h                                           h&#226;&#171;&#186; ade6-M210 leu1-32 ura4-D18 his7                                                               Laboratory stock <br /> 429h                                           h&#226;&#171;&#185; ade6-M216 leu1-32 ura4-D18 his7                                                               Laboratory stock <br /> KZ2                                            h&#226;&#171;&#185; leu1-32 ade6-M210 bub1&#226;&#171;&#185;-GFP-kanr Ch16                                                         study <br /> KZ50                                           h&#226;&#171;&#186; ade6-M210 leu1-32 pfs2-3169 bub1&#226;&#171;&#185;-GFP-kanr                                                    study <br /> KZ52                                           h&#226;&#171;&#186; his2 pfs2-3169 mad2&#226;&#171;&#185;-GFP-LEU2 bub1&#226;&#171;&#185;-HA3-kanr                                                  study <br /> KZ67                                           h&#226;&#171;&#186; ade6-M216 leu1-32 ura4-D18 pfs2-3169 bub1::ura4&#226;&#171;&#185;                                              study <br /> pfs2&#226;&#338;&#172;                                          h&#226;&#171;&#185;/h&#226;&#171;&#186; pfs2::ura4&#226;&#171;&#185;/pfs2&#226;&#171;&#185; ade6-M210/ade-M216 leu1-32 ura4-D18                                      study <br /> pfs2&#226;&#171;&#185;-GFP                                      h&#226;&#171;&#186; pfs2&#226;&#171;&#185;-GFP::kanr leu1-32                                                                       study <br /> P41-pfs2                                       h&#226;&#171;&#186; pfs2::ura4-nmt41-pfs2 ade6-M210 leu1-32 ura4-D18 his7                                         study <br /> P81-pfs2                                       h&#226;&#171;&#186; pfs2::ura4-nmt81-pfs2 leu1-32 ura4-D18                                                        study <br /> pfs2-11                                        h&#226;&#171;&#186; pfs2-11                                                                                       study <br /> bub1&#226;&#338;&#172;                                          h&#226;&#171;&#186; bub1::ura4&#226;&#171;&#185; ade6-M210 leu1-32 ura4-D18 his1                                                   Jean-Paul Javerzat <br /> pfs2-11 bub1&#226;&#338;&#172;                                  h&#226;&#171;&#186; pfs2-11 bub1::ura4&#226;&#171;&#185; ade6-M210 leu1-32 ura4-D18 his1                                           study <br /> bub1-GFP                                       h&#226;&#171;&#186; bub1&#226;&#171;&#185;-GFP::kanr leu1-32 ura4-D18                                                              Laboratory stock <br /> pfs2-11 bub1-GFP                               h&#226;&#171;&#186; pfs2-11 bub1&#226;&#171;&#185;-GFP::kanr leu1-32 ura4-D18                                                      study <br /> Ch16                                           h&#226;&#171;&#186; ade6-M210 ura4-D18 Ch16                                                                       Laboratory stock <br /> pfs2-11 Ch16                                   h&#226;&#171;&#186; pfs2-11 ade6-M210 leu1-32 ura4-D18 Ch16                                                       study <br />  <br />  <br />  <br /> tion, Pol II continues transcribe normal termi-                           Disruption modification pfs2. step gene disruption modification <br /> nation zone, potentially disrupting functions genes                       homologous recombination performed following PCR-mediated gener- <br />                                                                                      ation ura4&#226;&#171;&#185; kanMX selectable cassettes flanked 80-bp segments chromosomal features downstream. <br />                                                                                      appropriate regions pfs2&#226;&#171;&#185; using oligonucleotides described Table 2. <br />    characterization fission yeast mu-                      Following transformation diploid strain (428h/429h), Ura&#226;&#171;&#185; progeny tant identified inability execute mitosis faithfully                  screened desired integration pattern diagnostic PCRs primer primary biochemical defect mutant                       pairs spanning presumptive recombination sites (details additional component cleavage polyadenylation machin-                           primers used purpose available authors request). Meiosis <br /> ery. data indicate proper regulation mRNA                          sporulation induced plating malt extract agar, tetrad dissec- <br />                                                                                      tion performed MSM micromanipulator (Singer Instruments, <br /> 3&#226;&#172;&#732;-end processing essential normal chromosome                          Watchet, United Kingdom) described (15). pfs2&#226;&#171;&#185; gene segregation.                                                                         disrupted using primers DELA DELB, construction chromosomal- <br />                                                                                      ly GFP-tagged pfs2&#226;&#171;&#185; strain (pfs2&#226;&#171;&#185;-GFP) accomplished using primers <br />                         MATERIALS METHODS                                        TAGA TAGB, nmt1 promoter shutoff strains P41-pfs2 P81-pfs2 <br />    Fission yeast methods. Conditions growth, maintenance, genetic        constructed using primers PSOA PSOB (Table 2). order manipulation fission yeast described previously (15). complete list     generate ts alleles pfs2, low-fidelity PCRs performed primers strains used study given Table 1.        MUTA MUTB, flank pfs2&#226;&#171;&#185; locus, genomic S. pombe <br /> stated, strains grown 30&#194;&#176;C yeast extract (YE) Edinburgh minimal        DNA template. resulting pool mutagenized pfs2 sequences used <br /> medium (EMM2) appropriate supplements. necessary, gene expres-             transform strain P81-pfs2, transformants selected growth 25&#194;&#176;C <br /> sion nmt1 promoter repressed addition 5 &#226;?&#174;M thiamine        YE agar containing thiamine, allowing growth cells nmt1 growth medium.                                                                   promoter-driven pfs2 allele replaced homologous recombination <br />    pfs2-3169 mutant identified course large-scale mu-        mutagenized PCR product. resulting colonies screened ts <br /> tagenesis screen identify essential genes required accurate chromosome        growth replica plating checked loss P81-pfs2 allele PCR, <br /> segregation. Parental strain KZ2 (Table 1), contains minichromosome            sequence change(s) pfs2 identified following PCR amplification Ch16 (19) Bub1-green fluorescent protein (GFP), mutagenized             mutant alleles. <br /> nitrosoguanidine previously described (24), spread rich yeast extract        Microscopy. Cells fixed 3.8% <span id='am-6' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span> washed phosphate- supplements (YE5S) plates, incubated 27&#194;&#176;C. Colonies replica         buffered saline stained 4&#226;&#172;&#732;,6-diamidino-2-phenylindole (DAPI) plated YE5S plates 36&#194;&#176;C YE plates (rich medium lacking exogenously         examination fluorescence microscopy. Images acquired using Zeiss <br /> added supplements) 32&#194;&#176;C, cells lost Ch16 turned red             Axioskop microscope equipped Planapochromat &#226;&#171;&#187;100 objective, Axio- chromosomal ade6-210 mutation. total 221 isolates                                                                                      cam cooled charge-coupled device camera, Axiovision software (Carl Zeiss <br /> 500,000 mutagenized colonies showed minichromosome loss tempera- <br />                                                                                      , Welwyn Garden City, United Kingdom). Images assembled using <br /> ture-sensitive (ts) phenotypes; furthermore, 39 mutants showed cosegregation                                                                                      Adobe Photoshop. experiments, living cells growing EMM2 medium <br /> minichromosome loss ts phenotypes backcrossing. iso- <br />                                                                                      stained addition 5 &#226;?&#174;g/ml bis-benzimide (Hoechst 33342; Sigma) <br /> lates, mutations 10 complementation groups (cin1 cin10) showed <br />                                                                                      examination fluorescence microscopy. Visualization Pfs2-GFP hypersensitivity thiabenzadole compared wild-type strain. cin2-3169                                                                                      living cells embedded 0.6% low-melting-point agarose performed room <br /> allelic pfs2. analysis pfs2-3169, strain KZ52, contains <br /> Mad2-GFP Bub1-hemagglutinin (HA), used.                                      temperature (&#226;&#172;&#402;22&#194;&#176;C). <br />    Minichromosome loss assays performed strains containing non-           Flow cytometry. Cells fixed 70% ethanol rehydrated 10 mM <br /> essential ade6-M216-marked Ch16 minichromosome derivative chromosome 3            EDTA (pH 8.0)&#226;&#8364;&#8220;0.1 mg RNase ml&#226;&#8364;&#8220;1 &#226;?&#174;M Sytox green incubated 37&#194;&#176;C <br /> (19). ade6-M216 allele complements unlinked ade6-M210 marker         2 h. Cells analyzed using Epics XL-MCL flow cytometer (Coulter, <br /> strains remain Ade&#226;&#171;&#185; long minichromosome main-            Fullerton, Calif.). <br /> tained. Chromosome loss measured progeny single Ade&#226;&#171;&#185; cell             RT-PCR. Total RNA isolated hot phenol extraction purified known number generations selection <span id='am-1' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span> prot-         using RNeasy kit (Qiagen). total 0.5 &#226;?&#174;g total S. pombe RNA otrophy relaxed growth YE agar. Rates chromosome loss          reverse transcribed SuperScript reverse transcriptase (RT) (Invitrogen) generation calculated described (16, 28) formula 1 &#226;&#171;&#186;      random hexamers according manufacturer&#226;&#8364;&#8482;s instructions. Reverse-tran- <br /> e(1/n)lnRn, Rn proportion Ade&#226;&#171;&#185; cells n generations         scribed S. pombe cDNA products (2 &#226;?&#174;l) PCR amplified described <br /> removal selection. strain tested, mean rates calculated   (9) specific primers (Table 2) 25 cycles 95&#194;&#176;C 1 min, <br /> independent measurements.                                                            64&#194;&#176;C 1 min, 72&#194;&#176;C 3 min. <br />  2290     WANG ET AL.                                                                                                 MOL. CELL. BIOL. <br />  <br />                                            TABLE 2. Oligonucleotides used study <br />                                                                Sequence (5&#226;&#172;&#732;-3&#226;&#172;&#732;) <br />  <br /> DELA .........................TTTAAGTTGGTTTTATTTAAATTAAAGATTGATTTTATTGCTGCTAAATAAGGAATTTCACATCTTTGGGGAA <br />                                  TTGGTTACAAAGCGACTATAAGTCAGAAAGTG <br /> DELB..........................GGATGATATTGCATACTCCAAATACCCAAATCATGAGCAAATGGAATTTCAGCAGTAGGGTATAGTAAATA <br />                                  AGAAGACGCAATTCTAAATGCCTTCTGAC <br /> TAGA .........................ACGCTGAGCCGTCAACTCAAAACTCTTTTATCCCAGGTCTCACCTCAAAGTCACAAGATGGGTATCCTCAAA <br />                                  ACTATCGACGGATCCCCGGGTTAATTAA <br /> TAGB .........................GCTGCGGTGTGAATTTTTTTTTTTTTAAATCTAATTAATGCCAAGTAAATATGTTTTTCAAATAACAAAGAAA <br />                                  TGTATTAGAATTCGAGCTCGTTTAAAC <br /> PSOA ..........................CTAGGTGGATTAATTTATTTGGCGTGCCTAATTTGGAACGAAATTAATATCATCTTCAACATCCCACCGATTT <br />                                  ACTTATAATACTAATTCTAAATGCCTTCTGAC <br /> PSOB ..........................TTTGATAGCCCTGATCCATAGTCCACAGTCCTACGAGTCATAGGCTTTTGAATAATTCTAGCATTTTCAGCTC <br />                                  TTTCCATCAAAGCGACTATAAGTCAGAAAGTG <br /> MUTA ........................ATCAAGCATGGATAGATGATCAAAGATATC <br /> MUTB.........................TCCTCTCGCCGAATCGTAATTTCTTAA <br /> URA4F .......................GTCCCCTGGTATCGGCTTGGATGTTAAAGG <br /> URA4R1 ....................TGCCTTCTGACATAAAACGCCTAG <br /> URA4R2 ....................TGTAGGAGCATGTTTAATAAATTACTATAGCAAATTACTTTTTATT <br /> URA4R3 ....................CATACATCTTTCATTGGCTTTGTAC <br /> URA4R4 ....................GTCTACATGGTATTTTACATTCATC <br /> URA4R5 ....................CCAACACCAATGTTTATAACCAAG <br /> URA4R6 ....................GCTTGTGATATTGACGAAACTTTTTGACATC <br /> CDT1F........................CTCATCTCGCTCAGATTTTAACCGTATGGC <br /> CDT1R .......................CTCGCGACTTGTTGATTTCTCCAAATTGAG <br /> CDC18F......................CATATCGTTCAGCAGGAGTCGTGGGTCGTG <br /> CDC18R .....................CGGACTGGGTGAAATGCGACTCTAGTTCAC <br /> CDC18F1....................GGTCTTCAACAGAAAGCCATCCTTTGTACG <br /> CDC18R1 ...................GGTACCAATGTCTCCAACAGCTGTAATGAC <br /> CDC18R2 ...................CGGCAAAATTTGAAAACCGTAACGCGTGAA <br /> CDC18R3 ...................CAACCTGAAAAGTTTAAACACCAAGTTGGA <br /> CDC18R4 ...................ACCACTTGAGACTGAACCACGGGTATGCGT <br /> CDC18R5 ...................CCCCAAAATTGAGAAGTGCTTTTGCTACGT <br /> CDC18R6 ...................TTCCGCAGCTTATATGATTACTATGTCGTC <br />  <br />  <br />  <br />                          RESULTS                                   category frequently contained aberrantly segregated chromo- <br />                                                                    somes (data shown). <br />    Fission yeast Pfs2 essential nuclear protein required <br />                                                                       Complementation cin2-3169 mutant genomic normal chromosome segregation. order identify novel <br />                                                                    DNA library (kindly provided Taro Nakamura Chikashi <br /> genes essential roles chromosome segregation, used <br />                                                                    Shimoda) yielded 10 transformants grow fluorescence microscopy-based screen search collec- <br />                                                                    restrictive temperature.  contained plasmid  <br /> tion ts S. pombe mutants strains displaying chromosome <br />                                                                    sert spanning S. pombe orthologue S. cerevisiae PFS2 <br /> missegregation unusually high frequency Mad2 Bub1 nuclear dots restrictive temperature. screen      gene, term pfs2&#226;&#171;&#185;, contained insert <br /> yielded cin2-3169 strain, number         spanning cft1&#226;&#171;&#185;, S. pombe orthologue S. cerevisiae <br /> mutant strains  complete account screen         CFT1 gene. PFS2 CFT1  known YHH1)  <br /> characterization mutants form basis fu-     code components S. cerevisiae CPF clear coun- <br /> ture report). asynchronous cin2-3169 cells expressing        terparts metazoan CstF CPSF (Cstf50 CPSF160, <br /> Mad2-GFP shifted 26&#194;&#176;C restrictive temper-         respectively) (23). Backcrossing cin2-3169 strain ature 36&#194;&#176;C, cells containing Mad2-GFP dots accumulated          strain pfs2&#226;&#171;&#185; gene tagged GFP  use accounted 8% population 4 h            kanamycin resistance gene marker) demonstrated temperature shift (Fig. 1A). presence            pfs2 gene mutated cin2-3169 cells, cft1&#226;&#171;&#185; repre- <br /> occasionally Mad2-GFP foci cells correlated     sents extragenic suppressor. assigned des- cases gross defects chromosome segregation,     ignation pfs2-3169 pfs2 allele identified cin2-3169. <br /> judged DAPI staining (Fig. 1A). Visualization HA                identification ts pfs2 allele suggested pfs2&#226;&#171;&#185; epitope-tagged Bub1 cin2-3169 cells expressing Mad2-       likely essential cell viability. address point, GFP revealed variety combinatorial phenotypes       constructed ura4-D18/ura4-D18 diploid fission yeast strain shift 36&#194;&#176;C. cells Bub1-HA foci contained         pfs2 alleles disrupted ura4&#226;&#171;&#185; <br /> Mad2-GFP spots, case,      marker. Following meiosis tetrad dissection, cases numbers Bub1-HA Mad2-GFP foci                set haploid spores viable, individual cells equal (Fig. 1B C).      case Ura&#226;&#171;&#186; (Fig. 2A).  pfs2&#226;&#171;&#185;, like <br /> incubation 4 h 36&#194;&#176;C, 85% cin2-3169 cells         PFS2 S. cerevisiae, essential gene, pfs2-3169 microtubular arrays typical interphase, 10% post-     loss function allele. <br /> anaphase arrays 4% contained mitotic spindles.         monitor localization Pfs2 living fission yeast,  VOL. 25, 2005                                                                                          Pfs2 CELL CYCLE               2291 <br />  <br />  <br />                                                                             tion living pfs2&#226;&#171;&#185;-GFP cells fluorescence microscopy <br />                                                                             showed Pfs2-GFP   inference, Pfs2) localized <br />                                                                             nucleus excluded nucleolus (Fig. 2B). <br />                                                                             obvious differences pattern cells <br />                                                                             different cell cycle stages. <br />                                                                                examine influence pfs2 function progression <br />                                                                             mitosis  pfs2-3169 control pfs2&#226;&#171;&#185; <br />                                                                             cells expressing GFP localized centromere chromo- <br />                                                                             2 (cen2-GFP) synchronized arrest hydroxyu- <br />                                                                             rea (HU) 26&#194;&#176;C released HU-free medium                                                                             36&#194;&#176;C. Progression subsequent mitosis mon- <br />                                                                             itored fluorescence microscopy examine cen2-GFP sep- <br />                                                                             aration nuclear morphology (Fig. 3). condi- <br />                                                                             tions, control pfs2&#226;&#171;&#185; cells exhibited uniformly symmetrical <br />                                                                             segregation cen2-GFP signal daughter nuclei,                                                                             peak binucleate cells seen 120 min release                                                                             HU block; 180 min, cells completed <br />                                                                             mitosis cytokinesis (Fig. 3A B). contrast, approxi- <br />                                                                             mately 40% pfs2-3169 cells initiated cen2-GFP separa- <br />                                                                             tion, cells displayed severe chromosome <br />                                                                             missegregation 240 min release HU block (Fig. <br />                                                                             3A C). remaining 60% cells, cen2-GFP <br />                                                                             remained unseparated. population cells failed                                                                             complete mitosis 240 min release HU block <br />                                                                             (Fig. 3C). data suggested HU block  <br />                                                                             lease, pfs2-3169 cells showed defects. mitotic <br />                                                                             sister chromatid missegregation, accompanied  <br />                                                                             creased frequencies Mad2 Bub1 foci,                                                                             block interphase, resulting failure enter mitosis. <br />                                                                             cells exhibited cen2-GFP separation, timing                                                                             event significantly altered pfs2 mutant, <br />                                                                             suggesting gross defect sister chromatid  <br />                                                                             hesion. <br />                                                                                initially independent investigation pfs2&#226;&#171;&#185; <br />  <br />  <br />  <br />  <br />    FIG. 1. Chromosome missegregation accumulation Mad2 Bub1 foci cin2-3169 (pfs2-3169) cells.   asynchronous <br /> culture cin2-3169 mad2&#226;&#171;&#185;-GFP (KZ52) cells growing 26&#194;&#176;C shifted restrictive temperature 36&#194;&#176;C 4 h, stained DAPI reveal DNA, examined fluorescence microscopy. (B) Termi- <br /> nal phenotypes cin2-3169 mad2&#226;&#171;&#185;-GFP bub1&#226;&#171;&#185;-HA strain (KZ52) 4 h incubation 36&#194;&#176;C. Fixed cells processed anti-HA <br /> immunofluorescence DAPI staining. Individual cells containing fluorescent spots assigned categories: <br /> cells containing multiple spots, larger number Mad2-GFP <br /> spots Bub1-HA spots  ; equal numbers multiple <br /> (II) single (V) spots type; multiple spots, larger number Bub1-HA spots Mad2-GFP spots (III); single Mad2-GFP spot (IV) single Bub1-HA spot <br /> (VI). (C) Quantification phenotypes indicated panel B; total 55 cells fluorescent spots scored. <br />                                                                               FIG. 2. Fission yeast Pfs2 essential nuclear protein.   Tet- <br />                                                                             rads derived h&#226;&#171;&#185;/h&#226;&#171;&#186; pfs2::ura4&#226;&#171;&#185;/pfs2&#226;&#171;&#185; diploid strains micro- step gene replacement method used generate                     dissected YE5S agar. Colonies resulting tetrads <br />                                                                             photographed 7 days growth 30&#194;&#176;C. genotypes strain (pfs2&#226;&#171;&#185;-GFP) C-terminally GFP-tagged version                segregants determined replica plating. (B) Merged images <br /> Pfs2. GFP-tagged protein appeared functional,                  fluorescence micrographs showing Pfs2-GFP DNA (Hoechst <br /> judged viability pfs2&#226;&#171;&#185;-GFP strain. Examina-              33342) localization living cells. Bar, 10 &#226;?&#174;m. <br />  2292      WANG ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />                                                                           spindle checkpoint-defective strains genetic interactions <br />                                                                           pfs2-11 cells. Strains deletion bub1 (bub1&#226;&#338;&#172;)                                                                           inherently ts, combination bub1&#226;&#338;&#172; pfs2-11 <br />                                                                           resulted reduction restrictive temperature relative                                                                           seen pfs2-11 (Fig. 4A). genetic interaction <br />                                                                           reflected rapid loss viability double <br />                                                                           mutant compared single mutants  <br />  <br />  <br />  <br />    FIG. 3. Chromosome segregation defects pfs2-3169 cells.   <br /> Segregation patterns sister centromeres. Wild-type (KZ72) pfs2- <br /> 3196 (KZ71) cells carrying cen2-GFP grown 26&#194;&#176;C treated HU 3 h. HU washed away, cultures shifted 36&#194;&#176;C incubation continued. cen2-GFP localization DAPI <br /> staining visualized fluorescence microscopy. Representative <br /> examples unseparated  , equally segregated (b), missegre- <br /> gated (c d) centromeres shown. Bar, 5 &#226;?&#174;m. (B C) Quan- <br /> tification sister centromere segregation patterns following cen2-GFP DAPI staining wild-type (B) pfs2-3169 (C) cells.  <br /> centage type segregation pattern (open squares, unsepa- <br /> rated; blue circles, equally segregated; red triangles, missegregated) plotted time point. <br />  <br />  <br />  <br /> function, used targeted homologous recombination  <br /> place endogenous pfs2&#226;&#171;&#185; promoter  <br /> tenuated versions thiamine-repressible nmt1 promoter, <br /> P41 P81 (3, 4). resulting strains, P41-pfs2 P81- <br /> pfs2, grew healthily absence thiamine unable grow attenuated nmt1 promoters repressed addition thiamine medium (data shown). availability promoter shutoff strains possi- <br /> ble generation ts alleles pfs2, following tar- <br /> geted recombination randomly mutated chromosomal pfs2 <br /> sequences simultaneous restoration pfs2&#226;&#171;&#185; promoter <br />  Materials Methods). approach gave rise second tightly ts allele, pfs2-11. cells ceased divide 1 h shifted restrictive temperature 36&#194;&#176;C  like pfs2-3169 cells, displayed chromosome segrega- <br /> tion defects later time points (Fig. 4 data shown). data indicate phenotypes associated                                                                              FIG. 4. spindle checkpoint required cell survival alleles resulted similar losses pfs2 function.            inactivation pfs2.   Wild-type strain 972 strains <br /> nection pre-mRNA 3&#226;&#172;&#732;-end processing chromo-                    indicated streaked parallel YE5S agar plates photo- segregation suggested data pfs2-3169 cells             graphed incubation 3 5 days indicated temperatures. based unrepresentative, allele-specific effects.         (B) 2-h intervals, samples cultures pfs2-11, bub1&#226;&#338;&#172;, pfs2- <br />    spindle checkpoint required cell survival             11 bub1&#226;&#338;&#172; cells incubated restrictive temperature 36&#194;&#176;C                                                                           taken score percentage aberrant mitosis culture. inactivation pfs2. Despite chromosome segregation               thousand cells indicated times plated YES agar plates <br /> fects described  pfs2 mutants retained viability             assess cell survival. Colonies counted 5 days growth protracted incubation restrictive temperature (Fig. 4B).           26&#194;&#176;C. (C) Fluorescence micrographs DAPI-stained pfs2-11, bub1&#226;&#338;&#172;, increased frequencies Mad2 Bub1 foci cells            pfs2-11 bub1&#226;&#338;&#172; cells grown permissive temperature 26&#194;&#176;C <br />                                                                           shift 36&#194;&#176;C 6 h. Bar, 10 &#226;?&#174;m. (D) Merged images suggested involvement aspects spin-            fluorescence micrographs showing wild-type pfs2-11 cells carrying <br /> dle checkpoint machinery events following loss              Bub1-GFP. Cells grown 36&#194;&#176;C 2 h, fixed methanol, pfs2 function. investigate notion  tested              stained DAPI. <br />  VOL. 25, 2005                                                                                        Pfs2 CELL CYCLE                2293 <br />  <br />  <br />                                                                           results earlier study cleavage polyadenyl- <br />                                                                           ation gene (9), transcripts extending 3&#226;&#172;&#732; end <br />                                                                           mature ura4&#226;&#171;&#185; mRNA undetectable control <br />                                                                           pfs2&#226;&#171;&#185; cells grown 3 h 36&#194;&#176;C (Fig. 6B, upper panel).                                                                           contrast, steady-state ura4&#226;&#171;&#185; transcripts extended 300 <br />                                                                           bases normal 3&#226;&#172;&#732; end pfs2-11 cells grown                                                                           conditions (Fig. 6B, lower panel). data sug- <br />                                                                           gested S. pombe, S. cerevisiae, pfs2&#226;&#171;&#185; function                                                                           required normal 3&#226;&#172;&#732;-end processing pre-mRNA. <br />                                                                              Pfs2 required entry S phase. defective <br />                                                                           chromosome segregation striking aspect                                                                           pfs2 mutant phenotype cellular level, 85% cells <br />                                                                           arrested interphase pfs2 inactivated asyn- <br />                                                                           chronous populations (data shown). Similarly, 60% pfs2- <br />                                                                           3169 cells released HU (early-S-phase) block 36&#194;&#176;C <br />                                                                           unable progress mitosis (Fig. 3). interphase <br />                                                                           function pfs2&#226;&#171;&#185; investigated using nitrogen starvation <br />                                                                           arrest pfs2-11 pfs2&#226;&#171;&#185; cells G1 26&#194;&#176;C releas- <br />                                                                           ing starved cells nitrogen refeeding 36&#194;&#176;C (Fig. 7). <br />                                                                           Flow cytometry indicated pfs2&#226;&#171;&#185; population  <br />                                                                           tered S phase 2 h nitrogen refeeding com- <br />                                                                           pleted DNA replication 4 h (Fig. 7A). contrast,                                                                           pfs2-11 cells left G1 4-h time point,                                                                           substantial numbers G1-arrested cells present                                                                           6 h.  pfs2&#226;&#171;&#185; function required transit normal <br />                                                                           rate G1 phase S phase. likely requirement                                                                           Pfs2 general pre-mRNA processing suggest ni- <br />                                                                           trogen-starved cells lacking pfs2&#226;&#171;&#185; function simply fail    FIG. 5. Increased chromosome segregation failure rate pfs2-11 <br /> cells. (Upper panel) Rates minichromosome loss calculated pfs2&#226;&#171;&#185; pfs2-11 cells grown semipermissive temperature 32&#194;&#176;C. Mean loss rates 9.6 &#226;&#171;&#187; 10&#226;&#171;&#186;4 7.5 &#226;&#171;&#187; 10&#226;&#171;&#186;5 generation pfs2-11 cells control pfs2&#226;&#171;&#185; cells, respectively. (Lower <br /> panels) Examples pfs2&#226;&#171;&#185; colonies (left) Ade&#226;&#171;&#186; pfs2-11 colonies <br /> indicative chromosome loss (right). <br />  <br />  shift cells 36&#194;&#176;C (Fig. 4B). Loss viability correlated appearance &#226;&#8364;&#339;cut&#226;&#8364;? cells, septa- <br /> tion occurred absence nuclear division (Fig. 4B C). Consistent results obtained pfs2-3169 <br /> cells (Fig. 1), cells multiple Bub1-GFP foci observed 2 h pfs2-11 cells shifted 36&#194;&#176;C (Fig. 4D). <br />    failure chromosome segregation seen pfs2 mu- <br /> tants shifted restrictive temperature suggested partial loss pfs2 function severely compromise fidelity chromosome transmission. pfs2&#226;&#171;&#185; (control) pfs2-11 cells containing nonessential minichromosome (Ch16) <br /> carrying adenine biosynthetic marker used measure rates chromosome loss growth semi- <br />                                                                              FIG. 6. Pfs2 required transcriptional termination.   Sche- <br /> permissive temperature 32&#194;&#176;C (Fig. 5). assay showed           matic representation 547-bp region encompassed spontaneous loss minichromosome 13-fold               EcoRV-HindIII fragment downstream ura4&#226;&#171;&#185; gene. po- <br /> frequent pfs2-11 cells control pfs2&#226;&#171;&#185; cells. conclude        sitions translation stop codon (TAA) major site proper functioning pfs2&#226;&#171;&#185; makes important contri-           pre-mRNA 3&#226;&#172;&#732;-end cleavage polyadenylation (vertical arrowhead) <br />                                                                           indicated, oligonucleotides used RT-PCR anal- <br /> bution maintenance genome stability S. pombe. <br />                                                                           ysis (horizontal arrows). (B) Electrophoretic separation RT-PCR <br />    Pfs2 required prevent transcriptional read               products total RNA isolated cultures wild-type (upper Pfs2 previously implicated pre-mRNA cleavage                 panel) pfs2-11 (lower panel) cells grown restrictive temper- polyadenylation S. cerevisiae, likely           ature 36&#194;&#176;C 3 h. RT-PCR products separated 1.5% <br /> essential function pfs2&#226;&#171;&#185; S. pombe involve                agarose gel. Lane numbers correspond 3&#226;&#172;&#732; oligonucleotides <br />                                                                           (URA4R1 URA4R6) used RT PCR amplification;  <br /> pre-mRNA processing. investigate possibility, steady-             actions included common 5&#226;&#172;&#732; primer (URA4F), indicated panel <br /> state RNA characterized downstream region                 negative control reactions (lane C), reverse transcriptase ura4&#226;&#171;&#185; gene examined RT-PCR (Fig. 6). keeping                added RT-PCR. Lane M, size markers. <br />  2294      WANG ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                             achieve level macromolecular biosynthesis needed                                                                             commitment mitotic cell cycle start. inter- <br />                                                                             pretation   supported measurements                                                                             RNA transcripts cdt1&#226;&#171;&#185; cdc18&#226;&#171;&#185; (Fig. 7B), genes <br />                                                                             transcribed start-dependent manner (8). RT-PCR <br />                                                                             analysis indicated levels transcripts                                                                             genes peaked 2 h nitrogen refeeding control <br />                                                                             pfs2&#226;&#171;&#185; cells; pfs2-11 cells, transcript levels increased                                                                             similar kinetics 2 h continued rise, transcripts <br />                                                                             maintained high levels 6 h release. <br />                                                                             data suggested  conditions, pfs2-11 cells <br />                                                                             substantially delayed cell cycle progression point <br />                                                                             start close initiation S phase. Al- <br />                                                                             start-dependent transcription activated                                                                             cells, findings ura4&#226;&#171;&#185; gene (Fig. 6) suggested                                                                             3&#226;&#172;&#732;-end processing transcripts defective. <br />                                                                             able address notion transcripts                                                                             cdc18&#226;&#171;&#185;, 3&#226;&#172;&#732; end cDNA mapped                                                                             gene (11). RT-PCR indicated steady-state cdc18&#226;&#171;&#185; tran- <br />                                                                             scripts extended 200 bases normal mRNA <br />                                                                             3&#226;&#172;&#732; end pfs2-11 cells released nitrogen starvation 2 h <br />                                                                             36&#194;&#176;C (Fig. 7C D). <br />  <br />                                                                                                        DISCUSSION <br />                                                                                                    &#226;&#171;&#185; <br />                                                                                S. pombe pfs2 gene described encodes 509- <br />                                                                             amino-acid protein containing seven WD (&#226;?&#164;-transducin)  <br />                                                                             peats. Previous studies identified CstF50, human counter- <br />                                                                             Pfs2, involvement pre-mRNA 3&#226;&#172;&#732;-end <br />                                                                             processing S. cerevisiae Pfs2p component CPF <br />                                                                             complex (22, 29). Mutation pfs2 S. pombe caused                                                                             accumulation transcripts extending 3&#226;&#172;&#732; ends                                                                             mature mRNAs ura4&#226;&#171;&#185; cdc18&#226;&#171;&#185; genes (Fig. 6 7). <br />                                                                             formally possible finding represents                                                                             stabilization read RNAs normally present <br />                                                                             low levels, straightforward explanation                                                                             pfs2 function required pre-mRNA cleavage                                                                             polyadenylation. identification cft1&#226;&#171;&#185; multicopy <br />                                                                             suppressor adds weight idea. <br />                                                                                study revealed unexpected connections                                                                             pfs2 cell cycle progression. point cycle                                                                             pfs2 function appears critical onset                                                                             S phase activation start-dependent transcrip- <br />                                                                             tion (Fig. 7). start-dependent transcripts accumulated <br />    FIG. 7. Pfs2 required entry S phase.   pfs2&#226;&#171;&#185; (strain       pfs2-11 cells released nitrogen starvation restric- <br /> 972) pfs2-11 cells arrested G1 nitrogen starvation       tive temperature did appear appropriately processed <br /> released nitrogen-rich medium nonpermissive temperature         (Fig. 7C D). impaired function transcripts 36&#194;&#176;C. Cells harvested hourly intervals processed DNA         contributed observed cell cycle arrest, <br /> staining flow cytometry. (B) (Upper panels) Total RNA samples cultures shown panel isolated indicated           Cdt1 Cdc18 required assembly times. RT-PCR analysis performed primers specific cod-         functional prereplicative complexes chromosomal origins ing sequences cdt1&#226;&#171;&#185; (CDT1F CDT1R; left panels) cdc18&#226;&#171;&#185;            replication (10, 11, 18). observation cells released <br /> (CDC18F CDC18R; right panels), products separated              G1 arrest nitrogen refeeding able progress 1.5% agarose gel. (Lower panels) Normalized levels RT-PCR <br /> products. (C D) Transcriptional read cdc18&#226;&#171;&#185; pfs2-11 <br />                                                                             start  judged activation cdt1&#226;&#171;&#185;/cdc18&#226;&#171;&#185; <br /> cells. (C) Schematic representation 743-bp region containing StyI-HinDIII fragment encompassing 3&#226;&#172;&#732; end cdc18&#226;&#171;&#185; gene. positions translation stop codon (TAG) site pre-mRNA 3&#226;&#172;&#732;-end cleavage polyadenylation (vertical arrowhead)           Lane numbers correspond 3&#226;&#172;&#732; oligonucleotides (CDC18R1 (11) indicated, oligonucleotides used RT-PCR         CDC18R6) used RT PCR amplification; reactions included <br /> analysis (horizontal arrows). (D) RT-PCR products obtained total       common 5&#226;&#172;&#732; primer (CDC18F1), indicated panel C. Negative <br /> RNA isolated cultures wild-type (upper panel) pfs2-11           control reactions (lane C) panels B D contained reverse <br /> (lower panel) cells harvested 2 h nitrogen refeeding panel    transcriptase. <br />  VOL. 25, 2005                                                                                      Pfs2 CELL CYCLE                    2295 <br />  <br />  <br /> transcription) absence pfs2 function surprising,        link specifically pre-mRNA cleavage poly- efficient pre-mRNA 3&#226;&#172;&#732;-end processing expected         adenylation chromosome segregation recently estab- prerequisite gene expression needed cell         lished affinity purification cleavage polyadenylation <br /> cycle commitment. possible nitrogen-starved cells         complexes S. cerevisiae S. pombe (6, 7, 17, 25). retain stable stockpile mature mRNAs facilitate G1         yeasts, serine-threonine protein phosphatase Glc7/Dis2 <br /> progression nutritional status improves. Alterna-           identified component complexes puri- <br /> tively, loss pfs2 function affect processing         fication variety tagged subunits, including Cft1 different transcripts different degrees, efficient ex-    Pfs2. S. pombe, dis2&#226;&#171;&#185; gene identified pression genes required G1 transit comparatively       screen mutants defective mitotic chromosome separation <br /> independent pfs2. experiments required         (21). screen identified dis3&#226;&#171;&#185;, encodes pro- <br /> distinguish possibilities. Defects cell cycle     tein homologous S. cerevisiae protein Dis3p/Rrp44p, machinery, opposed defects cell growth, generally lead      component 3&#226;&#172;&#732;-5&#226;&#172;&#732; exoribonuclease termed exosome cell elongation S. pombe. Cells lacking pfs2 function          (12, 20, 21). interesting determine  result shift ts strain restrictive   chromosome segregation defects dis2 dis3 mutants, like <br /> temperature promoter shutoff) did appre-         pfs2 mutants, associated aberrant RNA pro- <br /> ciably elongated,  observation suggested           cessing. <br /> general macromolecular synthesis partially com-            partial loss pfs2 function fission yeast gave rise promised circumstances, arguing highly         dramatic elevation rate chromosome loss (Fig. 5). specific involvement pfs2 regulated expression cell     interesting determine scenario cycle-related genes.                                                 occurs mammals; case, defects cleavage    Chromosome segregation second cell cycle stage dur-         polyadenylation machinery contribute chromo- <br /> ing pfs2 mutant phenotype manifested.              somal instability characterizes cancer cells. <br /> general explanations account defective mitosis <br />                                                                                               ACKNOWLEDGMENTS <br /> seen loss pfs2 function. realistic possibility loss transcriptional termination direct effect      grateful Nick Proudfoot members group aspect chromosome structure kinetochore function.         valuable discussions, Paul Russell bringing attention                                                                      reannotation pfs2 cid14 genes, Stephen Kearsey recent observation cohesin clusters sites conver-     comments manuscript, Jean-Paul Javerzat providing gent transcription (13) suggests intimate relationship         bub1 deletion strain, Taro Nakamura Chikashi Shimoda tween sister chromatid cohesion distribution active       providing genomic DNA library. <br /> Pol II complexes. addition, centromere function S. pombe         work supported Cancer Research UK, Human perturbed deregulated transcriptional read-           Frontiers Science Program (research grant T.T.), Association <br />                                                                      International Cancer Research, Wellcome Trust (research  S. cerevisiae (27). Alternatively, impact   career development award S.-W.W.). loss pfs2 function mitosis indirect,                                                                                                     REFERENCES <br /> impaired expression genes required faithful <br />                                                                       1. Allshire, R. C., E. R. Nimmo, K. Ekwall, J. P. Javerzat, G. Cranston. <br /> chromosome segregation. situation analogous             1995. Mutations derepressing silent centromeric domains fission yeast phenotype previously described S. cerevisiae, muta-           disrupt chromosome segregation. Genes Dev. 9:218&#226;&#8364;&#8220;233. <br /> tion pre-mRNA splicing factors seen cause spindle           2. Amon,  1999. spindle checkpoint. Curr. Opin. Genet. Dev. 9:69&#226;&#8364;&#8220;75. <br />                                                                       3. Bahler, J., J. Q. Wu, M. S. Longtine, N. G. Shah,  McKenzie III,  B. <br /> checkpoint activation secondary consequence defective            Steever,  Wach, P. Philippsen, J. R. Pringle. 1998. Heterologous <br /> tubulin gene expression (5). general interpretations           modules efficient versatile PCR-based gene targeting Schizosac- <br />                                                                          charomyces pombe. Yeast 14:943&#226;&#8364;&#8220;951. mutually exclusive.                                           4. Basi, G., E. Schmid, K. Maundrell. 1993. TATA box mutations    chromosome segregation defects pfs2 mutant cells               Schizosaccharomyces pombe nmt1 promoter affect transcription efficiency <br /> resulted accumulation multiple foci Mad2                transcription start point thiamine repressibility. Gene 123: <br />                                                                          131&#226;&#8364;&#8220;136. <br /> Bub1, presumably marking unattached kinetochores (Fig. 1              5. Dahan, O., M. Kupiec. 2002. Mutations genes Saccharomyces 4). cells multiple Mad2 Bub1 foci            cerevisiae encoding pre-mRNA splicing factors cause cell cycle arrest <br /> proceeded anaphase-like state, suggesting               activation spindle checkpoint. Nucleic Acids Res. 30:4361&#226;&#8364;&#8220; <br />                                                                          4370. <br /> spindle checkpoint properly enforced pfs2 inac-         6. Dichtl, B., D. Blank, M. Sadowski, W. Hubner, S. Weiser, W. Keller. <br /> tivation. Nonetheless, severity mitotic abnormalities         2002. Yhh1p/Cft1p directly links poly  site recognition RNA poly- <br />                                                                          merase II transcription termination. EMBO J. 21:4125&#226;&#8364;&#8220;4135. loss viability pfs2 mutant cells           7. Gavin,  C., M. Bosche, R. Krause, P. Grandi, M. Marzioch,  Bauer, J. <br /> accentuated deletion bub1 (Fig. 4). observa-             Schultz, J. M. Rick,  M. Michon, C. M. Cruciat, M. Remor, C. Hofert, M. <br /> tions suggested spindle checkpoint partially           Schelder, M. Brajenovic, H. Ruffner,  Merino, K. Klein, M. Hudak, D. <br />                                                                          Dickson, T. Rudi, V. Gnau,  Bauch, S. Bastuck, B. Huhse, C. Leutwein, <br /> activated pfs2 mutant cells mitotic abnormalities,             M.  Heurtier, R. R. Copley,  Edelmann, E. Querfurth, V. Rybin, G. <br /> conclusion supported observation Mad2 Bub1               Drewes, M. Raida, T. Bouwmeester, P. Bork, B. Seraphin, B. Kuster, G. colocalized presumptive unattached kinet-             Neubauer, G. Superti-Furga. 2002. Functional organization yeast <br />                                                                          proteome systematic analysis protein complexes. Nature 415:141&#226;&#8364;&#8220;147. <br /> ochores (Fig. 1).                                                     8. Hofmann, J. F., D. Beach. 1994. cdt1 essential target    Additional connections RNA processing chro-               Cdc10/Sct1 transcription factor: requirement DNA replication inhi- <br />                                                                          bition mitosis. EMBO J. 13:425&#226;&#8364;&#8220;434. <br /> mosome segregation suggested characterization          9. Humphrey, T., C. E. Birse, N. J. Proudfoot. 1994. RNA 3&#226;&#172;&#732; end signals S. pombe mutants defective 5&#226;&#172;&#732;-3&#226;&#172;&#732; exoribonuclease               S. pombe ura4 gene comprise site determining efficiency element. <br /> coded dhp1 (26). mutants accumulated polyadenyl-                EMBO J. 13:2441&#226;&#8364;&#8220;2451. <br />                                                                      10. Kearsey, S. E., S. Montgomery, K. Labib, K. Lindner. 2000. Chromatin <br /> ated RNA nuclei exhibited chromosome segrega-               binding fission yeast replication factor mcm4 occurs anaphase <br /> tion defects similar reported pfs2 mutants.          requires ORC cdc18. EMBO J. 19:1681&#226;&#8364;&#8220;1690. <br />  2296       WANG ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br /> 11. Kelly, T. J., G. S. Martin, S. L. Forsburg, R. J. Stephen,  Russo, P.            implicated mitotic control, binds directly Ran enhances GEF <br />     Nurse. 1993. fission yeast cdc18&#226;&#171;&#185; gene product couples S phase                  activity RCC1. EMBO J. 15:5595&#226;&#8364;&#8220;5605. <br />     START mitosis. Cell 74:371&#226;&#8364;&#8220;382.                                              21.   Ohkura, H., Y. Adachi, N. Kinoshita, O. Niwa, T. Toda, M. Yanagida. <br /> 12. Kinoshita, N., M. Goebl, M. Yanagida. 1991. fission yeast dis3&#226;&#171;&#185;                1988. Cold-sensitive caffeine-supersensitive mutants Schizosac- <br />     gene encodes 110-kDa essential protein implicated mitotic control. Mol.           charomyces pombe dis genes implicated sister chromatid separation dur- <br />     Cell. Biol. 11:5839&#226;&#8364;&#8220;5847.                                                              ing mitosis. EMBO J. 7:1465&#226;&#8364;&#8220;1473. <br /> 13. Lengronne, , Y. Katou, S. Mori, S. Yokobayashi, G. P. Kelly, T. Itoh, Y.       22.   Ohnacker, M., S. M. Barabino, P. J. Preker, W. Keller. 2000.     Watanabe, K. Shirahige, F. Uhlmann. 2004. Cohesin relocation                  WD-repeat protein Pfs2p bridges essential factors yeast <br />     sites chromosomal loading places convergent transcription. Nature             pre-mRNA 3&#226;&#172;&#732;-end-processing complex. EMBO J. 19:37&#226;&#8364;&#8220;47. <br />     430:573&#226;&#8364;&#8220;578.                                                                     23.   Proudfoot, N., J. O&#226;&#8364;&#8482;Sullivan. 2002. Polyadenylation: tail com- <br /> 14. Millband, D. N., L. Campbell, K. G. Hardwick. 2002. awesome                    plexes. Curr. Biol. 12:R855&#226;&#8364;&#8220;R857. <br />     power multiple model systems: interpreting complex nature spindle      24.   Radcliffe, P., D. Hirata, D. Childs, L. Vardy, T. Toda. 1998. Identifica- <br />     checkpoint signaling. Trends Cell Biol. 12:205&#226;&#8364;&#8220;209.                                    tion novel temperature-se <br /> </body></html>