<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Mar. 2005, p. 604&#226;&#8364;&#8220;614                                                                                            Vol. 4,  3 <br /> 1535-9778/05/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.4.3.604&#226;&#8364;&#8220;614.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />                           Genomic Analysis PIS1 Gene Expression <br />                                 Mary E. Gardocki, Margaret Bakewell, Deepa Kamath, <br />                                 Kelly Robinson, Kathy Borovicka, John M. Lopes* <br />                             Department Biological Sciences, Wayne State University, Detroit, Michigan <br />                                          Received 18 September 2004/Accepted 20 December 2004 <br />  <br />              Saccharomyces cerevisiae PIS1 gene essential required final step novo synthesis <br />           <span id='am-21' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-22' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span>phosphatidylinositol</span>. Transcription PIS1 gene uncoupled factors regulate yeast <br />           <span id='am-1' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span>phospholipid</span> biosynthetic genes. phospholipid biosynthetic genes regulated response           inositol choline regulatory circuit includes Ino2p:Ino4p activator complex Opi1p <br />           repressor. PIS1 regulated response carbon source anaerobic growth conditions.           regulatory responses modest, entirely surprising PIS1 essential.  modest <br />           regulation PIS1 expression shown affect phosphatidylinositol metabolism affect cell cycle <br />           progression. prompted present study, employed genomic screen, database mining,           traditional promoter analysis identify genes affect PIS1 expression. screen viable yeast deletion <br />           set identified 120 genes affect expression PIS1-lacZ reporter. gene set included peroxi- <br />           somal genes, silencing genes, transcription factors. Factors suggested database mining, Pho2 <br />           Yfl044c, affect PIS1-lacZ expression. PIS1 promoter deletion study identified           upstream regulatory sequence element required carbon source regulation located downstream           previously defined upstream activation sequence elements. Collectively, studies demonstrate           collection genomic traditional strategies implemented identify set genes affect           regulation essential gene. <br />  <br />  <br />    Yeast excellent model study phos-                 inostamycin reduces PI levels inhibits induction S phase <br /> pholipid biosynthesis (Fig. 1) (11, 12, 28, 33). Phosphatidylino-           (18, 36). <br /> sitol (PI) essential phospholipid eukaryotic cells (3,            PI synthase membrane-associated enzyme catalyzes <br /> 11, 12, 28, 33, 38, 54). yeast, PI synthesized novo         condensation <span id='am-44' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="CDP" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span>CDP</span>-<span id='am-104' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-105' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="DAG" datatype="xsd:string"></span>diacylglycerol</span> inositol PI (23, <br /> product PIS1 gene, PI synthase (18, 23, 24, 36, 53&#226;&#8364;&#8220;56),              54) (Fig. 1). Disruption PIS1 gene results lethality represents 12 27% total phospholipid composi-                 (54). PIS1 essential, entirely surprising tion (11, 12, 28, 33). addition structural role, PI            yeast cells extensively regulate PIS1 expression PI <br /> precursor phosphoinositides, sphingolipids, inositol                 synthase levels (2, 24, 25). PIS1 gene expression coregu- <br /> polyphosphates (11, 12, 28, 33). PI metabolites               lated phospholipid biosynthetic genes  <br /> required diverse set processes include glycolipid             sponse inositol choline (2, 11, 12, 28, 33) instead <br /> anchoring proteins (69), signal transduction (21, 58), mRNA              regulated carbon source oxygen. PIS1 expression export (57, 64&#226;&#8364;&#8220;66), vesicle trafficking (17). spite           repressed response glycerol aerobic conditions (2, <br /> importance PI metabolites, relatively little known            25). Promoter deletion analysis identified upstream ac- factors regulate PIS1 expression.                                tivation sequence (UAS) elements (UAS1 UAS3) required <br />    understanding role PIS1 expression regu-                PIS1 gene expression (25).  element required <br /> lating PI synthesis conflicted. study reported             glycerol repression identified. region expression human PIS1 gene COS-7 cells yielded                  includes UAS3 element contains upstream regula- <br /> significant increase PI synthase activity (25-fold) mod-           tory sequence (URS) binds Rox1p exert anaerobic <br /> est increase PI levels (8.2%) (50).  report              regulation (25). significance anaerobic regulation indicated overexpression rat PIS1 gene NIH3T3                evidenced altered membrane composition. PI levels cells yielded elevated levels PI, PI-4,5-P2, PI-3,4,5-P3             elevated cells grown anaerobically, phosphatidylcholine <br /> (19). PIS1 overexpression decreased doubling time               (PC) CDP-diacylglycerol levels affected oxygen <br /> transformed cells accelerated G1 progression (19).                  (25). <br /> sistent effect G1 progression, cyclin D1 cyclin                PIS1 gene expression insensitive inositol choline; <br /> E levels elevated (19). Furthermore, Rous sarcoma virus-                inositol does affect PI synthase activity. High levels infected NIH3T3 cells activated erbB2-transformed                       inositol increase rate PI synthesis Km PI <br /> NIH3T3 cells overexpress PIS1 elevated PI levels              synthase inositol (0.21 mM) ninefold greater (37). Finally, specific inhibition PI synthase activity using            intracellular concentration inositol (24 &#226;?&#174;M) (41).                                                                             cells grown inositol, PI levels double expense                                                                             PC synthesis inositol noncompetitive inhibitor   * Corresponding author. Mailing address: Department Biological <br /> Sciences, Wayne State University, 5047 Gullen Mall, Detroit, MI <br />                                                                             <span id='am-6' about='obo:IMR_0001358' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-7' property="rdfs:label" content="phosphatidylserine" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200250" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001358" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:18303" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C02737" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span>phosphatidylserine</span> synthase (CHO1 gene product), 48202. Phone: (313) 993-7816. Fax: (313) 577-6891. E-mail: jlopes           enzyme PC branch phospholipid biosynthesis (Fig. 1) <br /> @sun.science.wayne.edu.                                                     (41). <br />  <br />                                                                       604 <br />  VOL. 4, 2005                                                                                                                     PIS1 GENOMICS             605 <br />  <br />                                                                                             TABLE 1. Oligonucleotides used study <br />                                                                                 Oligonucleotide <br />                                                                                                                                   Sequence <br />                                                                                    (position) <br />  <br />                                                                                 PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) ........................5&#226;&#172;&#732;-GGATCCCTTGTACTATCACAC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;918) .........................5&#226;&#172;&#732;-AGATCTGAATTCCCTGCACGC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;621) .........................5&#226;&#172;&#732;-AGATCTAAGCTTTTAGCCATG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;521) .........................5&#226;&#172;&#732;-AGATCTTAAATCCAAATCTTC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;325) .........................5&#226;&#172;&#732;-AGATCTTGTGCTTGAGGCTCA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;224) .........................5&#226;&#172;&#732;-AGATCTTTTTTTCATTTTATA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;205) .........................5&#226;&#172;&#732;-AGATCTTTCACGTCGCTCCGC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;184) .........................5&#226;&#172;&#732;-AGATCTATCCCTATTACGGAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;149F) ......................5&#226;&#172;&#732;-AGATCCGCCCCTCCTATTGTT-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;138) .........................5&#226;&#172;&#732;-AGATCTGTTTTTTCCGTCTCG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;127) .........................5&#226;&#172;&#732;-AGATCTCTCGAGGATTTTTCA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;185) .........................5&#226;&#172;&#732;-GGATCCTTACGTGCGGAGCGA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;149B) ......................5&#226;&#172;&#732;-GGATCCTTTCCCTATTGAGAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;126B) ......................5&#226;&#172;&#732;-GGATCCACGGAAAAAACAATA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;101B) ......................5&#226;&#172;&#732;-GGATCCATACTAAAAGTGAAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;76B) ........................5&#226;&#172;&#732;-GGATCCCTCATGTTTTTTATG-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;51B) ........................5&#226;&#172;&#732;-GGATCCTGACGGCAAATAAAC-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;101F) ......................5&#226;&#172;&#732;-AGATCTATAAGTAAAACATAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;76F) ........................5&#226;&#172;&#732;-AGATCTAGGTGGTATGGTTTA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;51F) ........................5&#226;&#172;&#732;-AGATCTCTGTGGAGCCTTCAA-3&#226;&#172;&#732; <br />                                                                                 PIS1 (&#226;&#171;&#186;26F) ........................5&#226;&#172;&#732;-AGATCTTAAGAGGGAAAGTGT-3&#226;&#172;&#732; <br />                                                                                 PIS1 URS (&#226;&#171;&#186;76) .................5&#226;&#172;&#732;-CTCGAGAGGTGGTATGGTTTA-3&#226;&#172;&#732; <br />                                                                                 PIS1 URS (&#226;&#171;&#186;51) .................5&#226;&#172;&#732;-CTCGAGTGACGGCAAATAAAC-3&#226;&#172;&#732; <br />                                                                                 TCM1 (&#226;&#171;&#186;572)......................5&#226;&#172;&#732;-AAGCTTCTTACGTTATCATTC-3&#226;&#172;&#732; <br />                                                                                 TCM1 (&#226;&#171;&#186;1)..........................5&#226;&#172;&#732;-GAATTCGATTGATTGTTGTAG-3&#226;&#172;&#732; <br />   FIG. 1. Schematic depiction S. cerevisiae phospholipid bio-            pBM2015-SEQ.....................5&#226;&#172;&#732;-TCTCGTGCGGAGATGACTGC-3&#226;&#172;&#732; <br /> synthetic pathway. CDP-choline pathway, known &#226;&#8364;&#339;Kennedy&#226;&#8364;? &#226;&#8364;&#339;salvage pathway,&#226;&#8364;? noted broken arrow. Genes designated boldface italic type. Abbreviations: DHA, dihy- <br /> droxyacetone; DHAP, dihydroxyacetone phosphate; PA, phosphatidic                cated, 75 &#226;?&#174;M inositol 1 mM choline added  C&#226;&#171;&#185; medium) acid; DGPP, diacylglycerol <span id='am-34' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span>pyrophosphate</span>; <span id='am-93' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-94' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span>DAG</span>, diacylglcerol.                   NaCl added (0.7 M final concentration). <br />                                                                                    Genomic studies. Plasmid pMA107 (2), containing 629 bp PIS1 pro- <br />                                                                                 moter fused lacZ gene, transformed BY4742-based VYDS <br />                                                                                 (Res Gen Invitrogen Corp., Carlsbad, Calif.) standard lithium acetate yeast <br />                                                                                 transformation procedure (14) 96 microtiter plates. Transformants    PIS1 transcript levels established unusual combi- <br />                                                                                 selected complete synthetic medium lacking <span id='am-88' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> (Ura&#226;&#171;&#186;), replicated nation low transcription initiation rates stable                    Ura&#226;&#171;&#186; 2% glucose X-Gal (5-bromo-4-chloro-3-indolyl-&#226;?&#164;-D-galactopyranoside) <br /> mRNA. PIS1 gene expressed weak promoter                      Ura&#226;&#171;&#186; 3% glycerol X-Gal plates allowed grow 30&#194;&#176;C 48 h. VYDS comparable activity GAL4, INO2, INO4                   transformants screened altered PIS1-lacZ expression, indicated transcription factor genes (1).  weakness                    light-blue phenotype 2% glucose X-Gal plates dark-blue phenotype                                                                                 3% glycerol X-Gal plates relative isogenic wild-type (BY4742) strain. PIS1 promoter compensated stable transcript (half-life,                 eliminate false positives, pMA107 transformants exhibited phenotype 58 min) (1). fact, PIS1 transcript                     transformed plasmids pJH330 (543 bp INO1 gene 132 codons stable yeast mRNAs (34). combination low transcrip-                      INO1 gene fused frame lacZ reporter gene YEp357R) (22) tion initiation rates high transcript stability ensure                  YEp357R-TCM1 (TCM1-lacZ) (described . Mutant alleles VYDS <br />                                                                                 confirmed sequencing bar codes marked mutant. PIS1 transcripts change slowly response environmen- <br />                                                                                    Plasmid construction. construction PIS1-cat reporters tal changes.                                                                    previously reported (25). Briefly, nested set PIS1 promoter deletions <br />    study, utilized genomic strategy identify genes              created PCR using appropriate oligonucleotides (Table 1). 5&#226;&#172;&#732;-ter- affect regulation PIS1 expression. Analysis viable               minal deletion mutants created using reverse primer PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) yeast deletion set (VYDS) identified genes affect                  forward primer set PIS1 (&#226;&#171;&#186;918) PIS1 (&#226;&#171;&#186;127) (Table 1). indi- <br />                                                                                 vidual PCR products cloned pGEM-T (Promega, Madison, Wis.). PIS1 <br /> PIS1 expression regulation. examined role                    promoter fragments excised pGEM-T digestion BamHI transcription factors defined &#226;&#8364;&#339;ChIP chip&#226;&#8364;? (chromatin                    BglII inserted pBM2015 create fusion cat reporter gene <br /> immunoprecipitation microarray chip) approach (62)                        (29). create PIS1 promoter internal deletions, regions position &#226;&#171;&#186;325 <br /> PIS1 expression.  identified URS needed PIS1                      various downstream points amplified using forward primer PIS1 (&#226;&#171;&#186;325) <br />                                                                                 reverse primer set PIS1 (&#226;&#171;&#186;185) PIS1 (&#226;&#171;&#186;51B) (Table 1). repression response glycerol. <br />                                                                                 fragments sequentially cloned pGEM-T pBM2015 described <br />                                                                                  regions downstream internal deletion amplified using <br />                                                                                 reverse primer PIS1-3&#226;&#172;&#732; (&#226;&#171;&#186;1) forward primer set PIS1 (&#226;&#171;&#186;101F) <br />                        MATERIALS METHODS <br />                                                                                 PIS1 (&#226;&#171;&#186;26F) addition PIS1 (&#226;&#171;&#186;149F) PIS1 (&#226;&#171;&#186;127) (Table 1).   Strains, media, growth conditions. Saccharomyces cerevisiae strains   fragments sequentially cloned pGEM-T pBM2015 containing used study BRS1001 (MATa ade2-1 his3-11,15 leu2-3,112 can1-100     relevant regions deletion endpoints described  ura3-1 trp1-1), BY4742 (MAT&#226;?&#163; his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0), BY4741           plasmid, indicates deletion endpoints. pBM2015 derivatives <br /> (MATa his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 met15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0) complete VYDS BY4742         sequenced Wayne State University Core Sequencing Facility using BY4741 backgrounds. Yeast cultures grown 30&#194;&#176;C complete syn-      primer pBM2015-SEQ. Yeast strains containing promoter-cat plasmids <br /> thetic medium (42) (containing 2% glucose) lacking uracil. appropriate,   integrated GAL4 locus created transformation characterized <br /> glucose concentration varied substituted 3% glycerol, 3% acetate,   Southern blot hybridization previously described (4). <br /> 3% ethanol, 3% lactate, 0.1% oleic acid 0.2% Tween 80. indi-         Plasmid YEp357R-TCM1 contains 572 bp sequences upstream  606       GARDOCKI ET AL.                                                                                                                  EUKARYOT. CELL <br />  <br />  <br />                                                       TABLE 2. Genes affect PIS1-lacZ expression <br />                      Reduced expression glucose                                                      Increased expression glycerol <br />  <br />   PIS1                   PIS1 INO1                   PIS1 TCM1                   PIS1                 PIS1 INO1                  PIS1 TCM1 <br />  <br /> APT1                       APG17                           ACE2                       AAD4                     ACB1                           NYV1 <br /> CPD1                       COQ4                            CIK1                       DCP3                     ADH2                           PAC1 <br /> CPT1                       DYN1                            CPS1                       ENT4                     CTF19                          RIF1 <br /> CRH1                       ENT3                            DFG10                      HST3                     CVT19                          YDR445C <br /> CWH41                      HIR3                            EAP1                       HST4                     EGD1                           YDR521W <br /> FAB1                       PDX3                            GSH1                       OGG1                     HSP104                         YGL250W <br /> GCS1                       REG2                            LSB3                       PEX3                     NTG2                           YLL032C <br /> GPD1                       RMD1                            MUD1                       PEX4                     PEX22                          YLL054C <br /> MUM2                       RPB9                            NHX1                       PEX17                    RIM8                           YLL057C <br /> NRG2                       RPS25A                          PEX7                       PMT3                     THI12                          YPL144W <br /> OSM1                       VTA1                            PHO2                       PTC3                     TIR4 <br /> PET130                     YBR053C                         PTC1                       ROT2                     VPS33 <br /> PIM1                       YDL119C                         RPB4                       RPL8B                    VTS1 <br /> POG1                       YDL129W                         SIN3                       SMY1                     YTP1 <br /> RDH54                      YDL173W                         SIT4                       STM1                     YCR076C <br /> SPO75                      YGL165C                         TTR1                       YDR124W                  YDR417C <br /> SRT1                       YJL178C                         YAL068C                    YGR022C                  YOL153C <br /> WH14                       YNR025C                         YDL025C                    YKR043C                  YOR315W <br /> YBR250W                    YOL107W                         YDL057W                    YOR029W                  YPL162C <br /> YDR332W                    YPR123C                         YDL063C                    YOR322C                  YPL166W <br /> YGL050W                                                    YKL147W                                             YPR151C <br /> YGR242W                                                    YLR254C <br /> YIL028W                                                    YLR366W <br /> YMR073C                                                    YNL134C <br /> YPL077C <br />  <br />  <br />  <br /> TCM1 gene fused frame lacZ reporter gene YEp357R (52).            lacZ expression strains grown different carbon sources. <br /> plasmid constructed amplifying 572 bp TCM1 promoter using S.          PIS1 regulated threefold response cerevisiae genomic DNA (Res Gen Invitrogen Corp.) using primers TCM1 <br /> (&#226;&#171;&#186;572) TCM1 (&#226;&#171;&#186;1) (Table 1). 572-bp PCR product cloned                                                                                     carbon source, regulation clearly discernible X-Gal <br /> pGEM-T (Promega) excised digestion HindIII EcoRI           medium containing glucose (dark blue) glycerol (light <br /> inserted YEp357R. Insert orientation sequence confirmed            blue). PIS1-lacZ reporter (pMA107) (2) transformed <br /> DNA sequencing.                                                                     BY4742-based VYDS (MAT&#226;?&#163; set), transformants <br />   Plasmid pLG&#226;&#338;&#172;312 &#226;&#171;&#185; PIS1 URSGLY contains 25-bp fragment PIS1 <br /> promoter (&#226;&#171;&#186;76 &#226;&#171;&#186;51) inserted upstream lacZ reporter gene pLG&#226;&#338;&#172;312 <br />                                                                                     screened light-blue colonies X-Gal glucose  <br /> (32). plasmid constructed amplifying 25 bp PIS1 gene pro-        dium dark-blue colonies X-Gal glycerol medium. moter (&#226;&#171;&#186;76 &#226;&#171;&#186;51) plasmid pPIS1(&#226;&#171;&#186;325) using primers PIS1 URS (&#226;&#171;&#186;76)            rounds screening identified 178 mutants yielded  PIS1 URS (&#226;&#171;&#186;51) (Table 1). 37-bp PCR product cloned                 duced expression glucose 59 mutants increased <br /> pGEM-T (Promega) excised digestion XhoI inserted                                                                                     expression glycerol. eliminate mutants nonspecifi- <br /> plasmid pLG&#226;&#338;&#172;312. Insert orientation sequence confirmed DNA <br /> sequencing.                                                                         cally affected lacZ expression, rescreened mutants using <br />   Enzyme assays. &#226;?&#164;-Galactosidase chloramphenicol acetyltransferase              INO1-lacZ (phospholipid biosynthetic gene) (22) TCM1- <br /> (CAT) assays performed described previously (4, 48).        lacZ (ribosomal protein gene) reporters. Strains exhibiting al- &#226;?&#164;-galactosidase experiments performed using microtiter plate plat-       tered expression reporters (109 strains glucose <br /> form. Units &#226;?&#164;-galactosidase activity defined follows: (A420/minute/ <br /> milligram total protein) &#226;&#171;&#187; 1,000. Units CAT activity defined counts   8 strains glycerol) removed data set. minute organic phase expressed percentage total counts   remaining mutants affected PIS1 specifically, PIS1 INO1, minute (percent conversion) divided protein assayed         PIS1 TCM1 (Table 2). <br /> micrograms) time incubation  hours). Protein concentration       strains grown 2% glucose, 25 mutant strains displayed <br /> extract determined using Bio-Rad (Rockville Center, N.Y.) protein <br /> assay kit. minimum independent measurements                                                                                     decreased PIS1-lacZ expression, 20 mutant strains affected <br /> data point.                                                                         PIS1-lacZ INO1-lacZ expression, 24 mutant strains <br />   Growth phase assays. Expression PIS1 gene monitored               yielded altered PIS1-lacZ TCM1-lacZ expression (Table <br /> BRS1001 transformants harboring pPIS1-918 (cat) reporter construct. Cells       2). strains grown 3% glycerol, PIS1-lacZ expression grown C&#226;&#171;&#186; C&#226;&#171;&#185; media absence presence 0.7 M <br />                                                                                     specifically altered 20 mutant strains, 21 mutant strains NaCl. milliliter aliquots removed intervals span 100 h monitored density (using Klett-Summerson colorimeter). CAT          altered PIS1-lacZ INO1-lacZ expression, 10 strains <br /> activity (described  determined samples simultaneously.          yielded altered PIS1-lacZ TCM1-lacZ expression (Table <br />                                                                                     2). <br />                                                                                        mutant set contains overrepresentation mutants <br />                                  RESULTS <br />                                                                                     affect specific biological processes. genomic <br />    genome-wide screen genes affect PIS1-lacZ expres-                     screen identified genes involved peroxisome biogenesis <br /> sion glucose glycerol. obtain comprehensive                          (pex3, pex4, pex17, pex22) function (acb1 gpd1). <br /> derstanding regulation PIS1 expression, screened                       GPD1 gene involved early step phospho- VYDS (&#226;&#172;&#402;4,800 viable mutant strains) changes PIS1-                        <span id='am-15' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span> biosynthesis (Fig. 1). screen yielded genes <br />  VOL. 4, 2005                                                                                                         PIS1 GENOMICS          607 <br />  <br />  <br />  <br />  <br />    FIG. 2. Peroxisomal biogenesis mutants affect PIS1-lacZ gene expression. PIS1-lacZ gene expression assayed using plasmid pMA107 <br /> containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). Plasmid pMA107 transformed wild-type (WT) strain (BY4742; MAT&#226;?&#163;) isogenic strains containing pex&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity measured. Cultures grown Ura&#226;&#171;&#186; medium containing 2% <br /> glucose (solid bars), 3% glycerol  bars), 0.1% oleic acid 0.2% Tween 80 (hatched bars). mutants divided classes <br /> described text. Note y-axis range differs bar graphs. <br />  <br />  <br /> required chromatin silencing (rif1, hst3, hst4, sin3).            present VYDS identified genomic overrepresentation mutants DNA repair                   plate screen. performed liquid assays quantify effect <br /> (ptc3, ogg1, ntg2, rpb4, rpb9) recently             known pex&#226;&#338;&#172; mutants grown medium containing 2% <br /> sociated PI metabolism (84). interesting              glucose 3% glycerol PIS1-lacZ expression. addition, <br /> subunits RNA polymerase involved transcription-coupled              quantified PIS1-lacZ expression medium containing <br /> repair identified (rpb4 rpb9). genomic                  0.1% oleic acid, carbon source causes peroxisomes screen yielded mutants involved carbon source                 proliferate (70). <br /> regulation play role glycerol repression (nrg2             expected wild-type strain, PIS1-lacZ expression <br /> reg2).  genes required PI metabolism iden-             repressed 3% glycerol unaffected 0.1% oleic acid <br /> tified (FAB1 ACB1 required synthesis phosphoi-             (Fig. 2). results pex&#226;&#338;&#172; mutants af- <br /> nositides sphingolipids, respectively). assume            fected expression PIS1-lacZ reporter genomic screen described did identify gene                 growth condition. included mutants affecting PIS1-lacZ expression  ;  did               identified initial plate screen.  PIS1-lacZ gene <br /> provide wealth information                  expression cultures grown 3% glycerol elevated possible traditional genetic screens used.                   pex3&#226;&#338;&#172;, pex4&#226;&#338;&#172;, pex22&#226;&#338;&#172; mutant strains. important    Peroxisomal biogenesis affects glycerol regulation PIS1-            observed elevated expression pex12&#226;&#338;&#172; pex13&#226;&#338;&#172; <br /> lacZ expression. Peroxisome biogenesis conserved process             mutants grown 3% glycerol (Fig. 2). mutants eukaryotes, involving 32 known peroxins (20,               present original plate screen eliminated subse- <br /> 60, 70, 72&#226;&#8364;&#8220;74, 77, 78, 80, 81). Peroxisomes membrane-                 quent screens, suggesting screening criteria bound organelles function metabolic pathways involved             stringent.  observed pex17&#226;&#338;&#172; inactivation toxic substances (<span id='am-115' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-116' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span>H2O2</span>-based respira-              mutant identified plate screen did reveal difference <br /> tion), regulation cellular oxygen levels, metab-           relative wild type quantified liquid assay. <br /> olism lipids, nitrogen bases, carbohydrates including              discrepancy explained fact plate <br /> fatty acid &#226;?&#164;-oxidation (60, 72, 74, 77, 78). Genomic analysis          assay reports accumulation &#226;?&#164;-galactosidase VYDS strain collection identified PIS1-lacZ expres-              phases growth, liquid assay quantifies &#226;?&#164;-galac- <br /> sion cells grown 3% glycerol X-Gal plates signifi-              tosidase activity single stage growth. <br /> cantly increased pex3&#226;&#338;&#172;, pex4&#226;&#338;&#172;, pex17&#226;&#338;&#172;, pex22&#226;&#338;&#172; mutant                  effect pex&#226;&#338;&#172; mutants PIS1-lacZ expression fell <br /> strains.  assumed screen               classes. Class contained mutants generally <br /> VYDS relied relatively modest phenotype, probably did             displayed pattern PIS1-lacZ expression identify genes regulate PIS1-lacZ expression. Given          carbon sources wild-type strain, elevated possibility, tested known pex&#226;&#338;&#172; mutants             expression glucose oleic acid decreased expression <br />  608     GARDOCKI ET AL.                                                                                                 EUKARYOT. CELL <br />  <br />  glycerol (pex2&#226;&#338;&#172;, pex3&#226;&#338;&#172;, pex10&#226;&#338;&#172;, pex11&#226;&#338;&#172;, pex17&#226;&#338;&#172;, pex29&#226;&#338;&#172;) <br /> (Fig. 2).  pex3&#226;&#338;&#172;, pex10&#226;&#338;&#172;, pex11&#226;&#338;&#172; mutants <br /> clearly yielded altered expression levels relative wild-type control. Class II included mutants yielded levels expression oleic acid wild- <br /> type expression levels oleic acid (pex1&#226;&#338;&#172;, pex6&#226;&#338;&#172;, pex15&#226;&#338;&#172;, <br /> pex19&#226;&#338;&#172;, pex21&#226;&#338;&#172;, pex31&#226;&#338;&#172;, pex32&#226;&#338;&#172;) (Fig. 2). Class III  <br /> cluded mutants yielded levels expression glycerol wild-type expression levels glycerol  <br /> creased levels expression oleic acid (pex4&#226;&#338;&#172;, pex8&#226;&#338;&#172;, <br /> pex12&#226;&#338;&#172;, pex13&#226;&#338;&#172;, pex14&#226;&#338;&#172;, pex18&#226;&#338;&#172;, pex22&#226;&#338;&#172;, pex25&#226;&#338;&#172;, pex27&#226;&#338;&#172;, <br /> pex28&#226;&#338;&#172;, pex30&#226;&#338;&#172;) (Fig. 2). Class IV included mutants generally yielded low levels expression carbon <br /> sources (pex5&#226;&#338;&#172; pex7&#226;&#338;&#172;) (Fig. 2). <br />    results clearly screening VYDS             FIG. 3. Chromatin silencing gene mutants affect PIS1-cat expres- <br /> yield valuable information regarding regulation gene expres-     sion. PIS1 promoter fragment (pPIS-325) fused cat <br />                                                                     reporter gene integrated single copy GAL4 locus sion, modestly regulated gene. pex mutants         strains BY4742 (MAT&#226;?&#163;) BY4741 (MATa) isogenic rif1&#226;&#338;&#172;, possibly identified traditional         hst3&#226;&#338;&#172;, hst4&#226;&#338;&#172; strains. PIS1-cat expression assayed transfor- <br /> means. importantly, results provide evi-       mants grown medium containing 2% glucose (MAT&#226;?&#163;, solid <br /> dence coordination synthesis peroxisomes          bars; MATa, hatched bars) 3% glycerol (MAT&#226;?&#163;, bars; MATa, <br />                                                                     horizontal bars). WT, wild type. phospholipids. <br />    Chromatin silencing genes affect PIS1-lacZ expression. <br /> Genomic analysis VYDS identified rif1&#226;&#338;&#172;, hst3&#226;&#338;&#172;,      strain (37%) cells grown 2% glucose (Fig. 4). hst4&#226;&#338;&#172; mutant strains elevated PIS1-lacZ expression 3%       expected VYDS screen, gpd1&#226;&#338;&#172; strain yielded glycerol X-Gal plates. mutant strains involved     20% decrease &#226;?&#164;-galactosidase activity (Fig. 4).  chromatin silencing telomeres. HST3 HST4 genes           gpd2&#226;&#338;&#172; mutant strain indistinguishable wild-type homologs SIR2 involved regulating short-     control (Fig. 4). <br /> chain fatty acid metabolism (8, 71). Hst3p required        PIS1 promoter binding proteins identified ChIPs silencing origin replication endogenous 2&#226;?&#174;m      required PIS1 expression. effort identify additional <br /> plasmid (31). determine PIS1-lacZ plate phenotype         factors regulate PIS1 expression, mined ChIP  genuine effects transcription effect    chip database containing information yeast DNA-binding <br /> plasmid copy number, assayed PIS1-cat reporter stably          proteins (47). analysis identified Pho2, Yfl044c, Ste12 <br /> integrated single copy GAL4 locus. deter-         weak candidate PIS1 promoter binding proteins (47). Pho2 <br /> mined mating type affected expression comparing CAT           involved response phosphate starvation (5, 39, 40, <br /> activity MATa (BY4741 background) MAT&#226;?&#163; (BY4742               43, 51, 67, 68), Ste12 involved pheromone background) transformants. PIS1-cat expression         pseudohyphal regulation (16, 35, 62, 83). YFL044c encodes mutant strains elevated glucose glycerol         hypothetical open reading frame (ORF) unknown func- <br /> gardless mating type, hst3&#226;&#338;&#172; hst4&#226;&#338;&#172; mutants       tion. Interestingly, pho2 mutant identified plate obvious effect (Fig. 3). results       screen yielding reduced expression 2% glucose X-Gal plate phenotype indirect effect       medium (Table 2). Previously, UAS elements (UAS1 <br /> plasmid copy number. evidence chromatin <br /> silencing affecting PIS1 expression. <br />    Mutants glycerol utilization affect regulation PIS1-lacZ <br /> expression. Genomic analysis VYDS identified gpd1&#226;&#338;&#172; mutant strain decreased PIS1-lacZ expression 2% glucose X-Gal plates. GPD1 encodes <span id='am-39' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="NADH" datatype="xsd:string"></span>NADH</span>-depen- <br /> dent cytosolic glycerol-3-phosphate dehydrogenase, key  <br /> zyme initial stages glycerol metabolism (Fig. 1) (15, <br /> 61). interesting result given PIS1 gene expres- <br /> sion repressed glycerol. led determine addi- <br /> tional enzymes involved early stages glycerol utiliza- <br /> tion affect PIS1-lacZ expression.  quantified <br /> &#226;?&#164;-galactosidase activity gpd1&#226;&#338;&#172; mutant strain gpd2&#226;&#338;&#172;, gut1&#226;&#338;&#172;, gut2&#226;&#338;&#172; mutant strains grown 2% glucose. <br /> GPD2 encodes isoform glycerol-3-phosphate dehydroge-             FIG. 4. Mutants glycerol utilization affect regulation PIS1- <br /> nase (Fig. 1) (59), GUT1 encodes glycerol kinase,         lacZ expression. PIS1-lacZ expression assayed using plasmid <br /> GUT2 encodes mitochondrial glycerol-3-phosphate dehydro-            pMA107 containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). <br />                                                                     Plasmid pMA107 transformed wild-type (WT) strain <br /> genase (Fig. 1) (26, 27).                                           (BY4742; MAT&#226;?&#163;) isogenic strains containing gpd1&#226;&#338;&#172;, gpd2&#226;&#338;&#172;, gut1&#226;&#338;&#172;, <br />    PIS1-lacZ expression increased substantially gut2&#226;&#338;&#172;        gut2&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity measured. <br /> strain (176%) lesser extent gut1&#226;&#338;&#172; mutant       Cultures grown Ura&#226;&#171;&#186; medium containing 2% glucose. <br />  VOL. 4, 2005                                                                                                     PIS1 GENOMICS        609 <br />  <br />  <br />                                                                          YFL044c ORF  5&#226;&#172;&#732;- 3&#226;&#172;&#732;-flanking sequences)                                                                          yfl044c mutant strain assayed PIS1-lacZ expres- <br />                                                                          sion. presence YFL044c pRS200 plasmid  <br />                                                                          stored PIS1-lacZ expression level &#226;&#172;&#402;3-fold higher                                                                          wild-type strain (data shown). result suggests <br />                                                                          Yfl044c limiting PIS1 expression wild-type <br />                                                                          strain conditions tested  <br />                                                                             PIS1 promoter deletion analysis identifies regulatory ele- <br />                                                                          ment required glycerol repression. nested set dele- <br />                                                                          tions PIS1 promoter fused cat reporter gene                                                                          plasmid pBM2015 transformed wild-type strain <br />                                                                          (BRS1001) targeting integration GAL4 locus single <br />                                                                          copy single orientation. previously reported <br />                                                                          analysis deletion subset grown glucose identi- <br />    FIG. 5. Transcription factor mutants affect PIS1-lacZ gene expres-    fied UAS elements positions &#226;&#171;&#186;224 &#226;&#171;&#186;205 (UAS1), <br /> sion. PIS1-lacZ expression assayed using plasmid pMA107 <br /> containing 629 bp PIS1 gene promoter (&#226;&#171;&#186;629 &#226;&#171;&#186;1). Plasmid            &#226;&#171;&#186;184 &#226;&#171;&#186;149 (UAS2), &#226;&#171;&#186;149 &#226;&#171;&#186;138 (UAS3) (Fig. 6A) <br /> pMA107 transformed wild-type (WT) strain (BY4742;             (25). Deletion UAS1 decreased PIS1-cat expression half <br /> MAT&#226;?&#163;) isogenic strains containing yfl044&#226;&#338;&#172;, ste12&#226;&#338;&#172;, pho2&#226;&#338;&#172;, pho4&#226;&#338;&#172;,     (Fig. 6A, compare pPIS-224 pPIS-205), deletion bas1&#226;&#338;&#172;, swi5&#226;&#338;&#172; mutant alleles, &#226;?&#164;-galactosidase activity mea-   UAS2 exacerbated decrease PIS1-cat expression  <br /> sured. Cultures grown Ura&#226;&#171;&#186; medium containing 2% glucose. <br />                                                                          90% (Fig. 6A, compare pPIS-205 pPIS-149). PIS1- <br />                                                                          cat expression detectable levels UAS3                                                                          deleted (Fig. 6A, compare pPIS-225 pPIS-127).  <br /> [&#226;&#171;&#186;224 &#226;&#171;&#186;205], UAS2 [&#226;&#171;&#186;184 &#226;&#171;&#186;149], UAS3 [&#226;&#171;&#186;149                    data showed UAS elements &#226;&#171;&#186;127]) required PIS1 gene expression identified (25).           required response glycerol. Curiously, deleting Consistent results, potential Ste12p Pho2p bind-            UAS elements increased degree  <br /> ing sites identified PIS1 UAS1, PIS1 UAS2                  pression caused growth glycerol  &#226;&#172;&#402;3-fold &#226;&#172;&#402;8-fold <br /> contains additional potential Pho2p binding site.                     [Fig. 6A]). explanation result    results prompted quantify PIS1-lacZ expression            UAS elements deleted, glycerol repression factor pho2&#226;&#338;&#172;, yfl044c&#226;&#338;&#172;, ste12&#226;&#338;&#172; mutant strains. PIS1-lacZ ex-             effective repressing transcription promoter <br /> pression generally decreased mutant strains,            activity lower. effect obvious pho2&#226;&#338;&#172; yfl044c&#226;&#338;&#172; mutants               UAS elements required expres- <br /> (Fig. 5). homeodomain transcription factor Pho2 known             sion, necessary generate internal deletions order interact Swi5, Pho4, Bas1 vivo activate               analyze promoter sequences downstream UAS <br /> transcription HO gene (7, 9), phosphate utilization genes         elements. Analysis PIS1 promoter downstream (7, 82), genes involved purine histidine biosynthesis         UAS elements identified repressor site (URSGLY), <br /> (7), respectively. Pho2 associates              located promoter proximal, required glycerol repression. <br /> proteins, quantified PIS1-lacZ expression swi5&#226;&#338;&#172;,              data identified 25-bp region (&#226;&#171;&#186;76 &#226;&#171;&#186;51)  pho4&#226;&#338;&#172;, bas1&#226;&#338;&#172; mutant strains. PIS1-lacZ expression                 deleted, abolishes PIS1 glycerol repression (Fig. 6A).  <br /> creased 34% bas1&#226;&#338;&#172; strain, pho4&#226;&#338;&#172; swi5&#226;&#338;&#172; mu-               ganization regulatory elements reminiscent GAL4 gene <br /> tant strains exhibited lower levels PIS1-lacZ expression              expression repressed fourfold growth glucose <br /> (53% 34%, respectively) relative wild-type            requires URS element located downstream UAS <br /> strain (Fig. 5). Pho4 Pho2 required induc-           element (29, 30). heterologous employed tion target genes response low phosphate concentra-             examine PIS1 URSGLY function. 25-bp region <br /> tions (5, 39, 40, 43, 51, 67, 68), quantified effect      (&#226;&#171;&#186;76 &#226;&#171;&#186;51) defined deletion analysis inserted phosphate PIS1-lacZ expression.                 plasmid pLG&#226;&#338;&#172;312, previously utilized test URS function <br /> PIS1-lacZ expression significantly affected phos-             (32, 49). Plasmid pLG&#226;&#338;&#172;312 contains CYC1 promoter fused <br /> phate concentration (data shown).                                    lacZ reporter gene. 25-bp PIS1 URSGLY caused <br />    ste12&#226;&#338;&#172; mutant did dramatically affect PIS1-lacZ               CYC1-lacZ expression decrease dramatically (Fig. 6B). expression condition tested response               result indicates PIS1 URSGLY sequence functions pheromone induction (data shown).  Ste12                 URS element. URSGLY effect CYC1-lacZ expression affect PIS1 expression conditions tested           cells grown glycerol assessed CYC1 Ste12 known regulate diverse set genes               promoter regulated carbon source. <br /> response different signals (16, 35, 62, 83).                Glycerol repression PIS1 expression reversed glu- <br />    effect yfl044c&#226;&#338;&#172; mutant PIS1-lacZ expression          cose. Glycerol-mediated repression PIS1 transcription evidence regulation gene expression              unusual does appear involve previ- <br /> putative transcription factor. role Yfl044c             ously identified regulators carbon source regulation (1, 2). <br /> defined ChIP chip database, evidence            determine glycerol-mediated repression reversible, suggest regulates PIS1 expression &#226;?&#164;-galacto-          PIS1-cat activity wild-type strain (BRS1001) containing <br /> sidase liquid assay. corroborate role Yfl044c PIS1          pPIS-325 assayed culture grown medium contain- <br /> expression, transformed pRS200 derivative containing            ing 2% glucose 3% glycerol various concentrations  610     GARDOCKI ET AL.                                                                                                     EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 6.  Identification PIS1 promoter element required glycerol-mediated repression. BRS1001 (wild-type) transformants containing <br /> various promoter deletions grown synthetic medium lacking uracil containing 2% glucose 3% glycerol. Yeast extracts transformants prepared assayed CAT activity. MRD1 divergent ORF. locations MCEs (Mcm1p binding sites) <br /> (black box), Rox1p binding site (gray box), potential TATA boxes (bars), UAS elements, URS element shown. B.D., detection. B. PIS1 URSGLY element functions URS element. Plasmid pLG&#226;&#338;&#172;312 &#226;&#171;&#185; URSGLY, containing 25 bp PIS1 gene promoter (&#226;&#171;&#186;76 &#226;&#171;&#186;51), transformed wild-type strain (BRS1001), &#226;?&#164;-galactosidase activity measured. Cultures grown Ura&#226;&#171;&#186; 2% glucose <br /> medium. <br />  <br />  <br />  <br /> glucose (range, 0.1 2%). data glucose               C&#226;&#171;&#186; medium (46, 63). discovered growing cells <br /> centrations low 0.1% able partially reverse glyc-          hyperosmotic medium (0.7 M NaCl) eliminates growth <br /> erol repression concentrations 1 2% com-                phase regulation affecting inositol-choline-medi- <br /> pletely reversed repression PIS1-cat expression (Fig. 7).             ated regulation (63). observations prompted exami- <br />    PIS1 expression elevated fermentable carbon sources             nation effect growth phase hyperosmotic condi- <br /> (glucose galactose) relative nonfermentable carbon              tions PIS1 expression. examination source (glycerol) (2). quantified effect nonfer-         quantifying CAT activity pPIS1-918 transformants mentable carbon sources PIS1-cat expression using              BRS1001. experiments revealed growth <br /> pPIS-325 construct. addition glycerol, PIS1-cat expression         phase hyperosmotic medium affected expression reduced &#226;&#172;&#402;50% cells grown acetate, ethanol,              PIS1-cat gene (Fig. 9). lactate carbon source (Fig. 8). <br />    PIS1 expression affected growth phase hyperos- <br />                                                                                                    DISCUSSION <br /> motic stress. reported phospholipid <br /> biosynthetic genes regulated response growth phase               recently, PIS1 gene expression believed (46). example, CHO1-lacZ expression levels C&#226;&#171;&#186;                regulated (2, 25, 45, 75, 76).  previously <br /> dium low lag phase increase log                shown PIS1 expression regulated carbon source (2) <br /> phase, peaking beginning stationary phase (46, 63).            response oxygen availability (25, 45, 75, 76). Multiple stationary phase, expression levels decrease precipi-            approaches applied identify genes affect PIS1 <br /> tously reach initial expression levels seen lag        expression. genomic screen VYDS identified 69 <br /> phase. expression pattern similar C&#226;&#171;&#185; medium;                mutants reduced PIS1 expression glucose 51 mu-  expression repressed medium,             tants increased PIS1 expression glycerol. PIS1 pro- <br /> overall expression levels lower observed           moter deletion analysis coupled mining databases  VOL. 4, 2005                                                                                                         PIS1 GENOMICS           611 <br />  <br />  <br />  <br />  <br />    FIG. 7. Glycerol repression PIS1-cat expression reversed glucose. BRS1001 transformants containing pPIS-325 grown Ura&#226;&#171;&#186; synthetic medium containing 3% glycerol 0, 0.1, 0.25, 0.5, <br /> 1.0, 1.5, 2% glucose, CAT activity assayed. <br />  <br />  <br />  <br /> DNA-binding proteins identified UAS elements URS element putative cognate transcription factors. example, promoter deletion analysis identified UAS <br /> element potential Ste12 Pho2 binding sites (UAS1 <br /> [&#226;&#171;&#186;224 &#226;&#171;&#186;205]), additional UAS element potential <br /> Pho2 binding site known Mcm1 binding sites (UAS2 <br /> [&#226;&#171;&#186;184 &#226;&#171;&#186;149]), potential Gcr1 binding site                   FIG. 9.  PIS1-cat expression regulated growth phase UAS element (UAS3 [&#226;&#171;&#186;149 &#226;&#171;&#186;138]),                 hyperosmotic stress. BRS1001 transformants containing pPIS1-918 <br /> Rox1 binding site essential anaerobic regulation PIS1          grown Ura&#226;&#171;&#186; synthetic medium presence inositol (25). Comparison promoters related Saccharomyces                  choline  C&#226;&#171;&#185;) (solid squares), absence inositol choline <br />                                                                            C&#226;&#171;&#186;) (open squares), presence 0.7 M NaCl (solid circles), <br /> yeast species revealed strong sequence similarity               presence inositol, choline, 0.7 M NaCl (open circles). UAS regions, including potential Ste12 binding sites,           Yeast extracts transformants prepared assayed known Mcm1 Rox1 binding sites S. cerevisiae               CAT activity. B. Growth patterns cultures used described S. castellii, S. bayanus, S. kluyveri, S. kudriavzevii, S.         . <br /> mikatae (25). Mining yeast ChIP chip database iden- <br />  <br />  <br />                                                                           tified Yfl044c Mcm1 candidate PIS1 promoter binding <br />                                                                           proteins (47). <br />                                                                              study demonstrated genomic approaches provide <br />                                                                           excellent means identify regulatory mechanisms  <br />                                                                           trol modestly regulated essential genes PIS1.  <br />                                                                            single strategies provide excellent starting point, <br />                                                                           combination multiple genomic approaches                                                                           essential. example, Rox1 identified regulator <br />                                                                           PIS1 promoter ChIP chip study (P value                                                                           0.19) (47) VYDS screen;  suggested                                                                           searches databases DNA-binding proteins coupled                                                                           promoter deletion (25) microarray studies (45, 75, 76). <br />                                                                           previous studies clearly shown Rox1 binds                                                                           bends PIS1 promoter exert anaerobic regulation                                                                           PIS1 gene (25). Conversely, VYDS screen employed <br />                                                                           did identify Yfl044c, examination PIS1 <br />                                                                           promoter sequence does reveal obvious binding site    FIG. 8. Analysis PIS1-cat expression response different       Yfl044c (TTCTTKTYYTTTT) (47).  ChIP  <br /> carbon sources. PIS1 promoter fragment (pPIS-325) created PCR        chip study placed Yfl004c cusp acceptable bind- fused cat reporter gene integrated single copy   ing threshold (P value 0.006) (47), shown GAL4 locus strain BRS1001 (wild type). PIS1-cat expression (pPIS- <br /> 325) assayed BRS1001 transformants grown Ura&#226;&#171;&#186; medium <br />                                                                           Yfl044c does regulate PIS1-lacZ expression. containing 2% glucose, 3% glycerol, 3% acetate, 3% ethanol, 3%         obvious PIS1 promoter binder defined ChIP chip <br /> lactate.                                                                  study Mcm1 (P value 0.000003), protein  612     GARDOCKI ET AL.                                                                                                            EUKARYOT. CELL <br />  <br />  <br /> previously shown bind PIS1 promoter (44).           affect PIS1 expression (REG1, GLC7, MIG1,  [M. E.  VYDS screen identified MCM1          Gardocki J. M. Lopes, unpublished results]). did, essential gene. use multiple genomic approaches      REG2 NRG2 affected PIS1-lacZ expres- expected generate overlapping information.        sion genes role carbon source regu- case Pho2  known Bas2 Grf10),         lation (6). Nrg2 binds DNA directly, levels  suggested regulator PIS1 promoter       creased glycerol- ethanol-grown cells.  Nrg2 deletion analysis potential binding sites        repressor expected increase PIS1 <br /> UAS1 UAS2 (25). VYDS screen identified Pho2            expression glycerol (6).  PIS1 promoter does regulator PIS1-lacZ expression.  ChIP        contain binding site Nrg2.  possible chip database did identify Pho2 potential binder (P        novel transcription factor involved glycerol-medi- <br /> value 0.5) (47). results underscore benefits        ated PIS1 expression. factor present collec- <br /> employing multiple genomic screens coupled database            tion genes defined VYDS screen. <br /> mining traditional approaches.                                knowledge PIS1 expression, regulation,     study demonstrated genomic approaches         various biological processes intimately identify biological functions affect expression   intertwined begin discerned direct gene, illuminating interplay biological pro-      result results presented  Genomic studies coupled <br /> cesses. VYDS screen identified overrepresentation         database mining classical strategies mutants affect biological processes: peroxisome bio-     obviously successfully combined study modestly regu- <br /> genesis (pex3, pex4, pex17, pex22) function (acb1       lated genes. course, challenge interpreting gpd1), chromatin silencing (rif1, hst3, hst4, sin3), DNA    wealth information genomic approaches provide. <br /> repair (ptc3, ogg1, ntg2, rpb4, rpb9). Alterations PIS1      critical process distinguish expression 26 assayed pex mutants extremely inter-       direct indirect effects. important issue esting given cellular role peroxisomes play, specifically    need addressed determine genes cellular oxygen regulation, metabolism lipids, nitro-    processes defined affect PI metabolism. PI metabolism <br /> gen bases, carbohydrates, fatty acid &#226;?&#164;-oxidation (60, 72, 74,   shown affected anaerobic conditions 77, 78). finding suggests changes peroxi-      Rox1 protein (25) carbon source (D. Kamath J. M.  potentially alterations fatty acid metabolism       Lopes, unpublished results), suggesting regulation PIS1 <br /> (60, 72, 74, 77&#226;&#8364;&#8220;79), responsible alterations PIS1   expression does affect PI metabolism. Furthermore, long <br /> expression, reminiscent seen CIT2 cells     half-life PIS1 transcript (57 min) (1) AUG codon <br /> altered mitochondrial activity (10, 13). finding       located upstream PI synthase AUG codon (25) supports existence pathway retrograde regulation       contribute regulation PIS1 gene expression. peroxisomes nucleus suggestive existence novel mechanism cell adjust changes                              ACKNOWLEDGMENTS peroxisomal activity. <br />                                                                        thank Meng Chen, Linan Chen, Leandria Hancock, Ying  <br />     Note identification chromatin silencing DNA          Niketa Jani, Sangeeta Oza, Jimmy Lo efforts editing <br /> repair mutants VYDS screen. dual roles HST3          manuscript. thank Edward Golenberg George Brush <br /> HST4 play chromatin silencing regulating short-chain         helpful discussions. <br /> fatty acid metabolism suggest relationship             work initially supported NSF grant (MCB-0110408) <br /> PIS1 expression fatty acid metabolism. understanding        completed support NSF grant (MCB-0415511)                                                                     J.M.L., William  Turner, Jr., Memorial Foundation scholarship role Hst3 Hst4 silencing devel-     M.E.G., undergraduate research grant Wayne State <br /> oped silencing proteins.  studies     University Undergraduate Research Council K.B. distinguish direct indirect effects PIS1 <br /> expression. Additional studies focus pro-                                      REFERENCES <br /> teins bind PIS1 promoter directly. DNA repair genes         1. Anderson, M. S. 1996. Analysis PIS1 gene expression Saccharomyces <br /> defined VYDS screen included RPB4 RPB9.               cerevisiae. Loyola University Chicago, Maywood, Ill. <br />                                                                     2. Anderson, M. S., J. M. Lopes. 1996. Carbon source regulation PIS1 <br /> genes affect PIS1 expression play            gene expression Saccharomyces cerevisiae involves MCM1 gene regulatory role transcription PIS1 role      component regulatory gene, SLN1. J. Biol. Chem. 271:26596&#226;&#8364;&#8220;26601. <br />                                                                     3. Antonsson, B. 1997. Phosphatidylinositol synthase mammalian tissues. DNA repair. Regardless mechanism               Biochim. Biophys. Acta 1348:179&#226;&#8364;&#8220;186. genes affect PIS1 expression, clear DNA        4. Ashburner, B. P., J. M. Lopes. 1995. Autoregulated expression repair genes affected PIS1 expression. identification         yeast INO2 INO4 helix-loop-helix genes effects cooperative regulation <br />                                                                        target genes. Mol. Cell. Biol. 15:1709&#226;&#8364;&#8220;1715. link damage checkpoint pathway PI             5. Barbaric, S., M. Mu  &#194;&#168;nsterko&#194;&#168;tter, C. Goding, W. Ho&#194;&#168;rz. 1998. Cooperative <br /> tabolism suggested genome-wide studies               Pho2-Pho4 interactions PHO5 promoter critical binding (84). entirely unexpected given PIS1 PI           Pho4 UASp1 efficient transactivation Pho4 UASp2. Mol. <br />                                                                        Cell. Biol. 18:2629&#226;&#8364;&#8220;2639. role cell cycle progression (18, 19, 36, 37).             6. Berkey, C. D., V. K. Vyas, M. Carlson. 2004. Nrg1 Nrg2 transcrip- <br />     promoter deletion study identified URS element,              tional repressors differently regulated response carbon source. <br />                                                                        Eukaryot. Cell 3:311&#226;&#8364;&#8220;317. <br /> downstream UAS elements, required         7. Bhoite, L. T., J. M. Allen, E. Garcia, L. R. Thomas,  D. Gregory, W. P. Voth, <br /> glycerol repression. Examination sequence defined            K. Whelihan, R. J. Rolfes, D. J. Stillman. 2002. Mutations Pho2 URSGLY element did reveal obvious protein-bind-            (Bas2) transcription factor differentially affect activation partner <br />                                                                        proteins Bas1, Pho4, Swi5. J. Biol. Chem. 277:37612&#226;&#8364;&#8220;37618. <br /> ing site. observation consistent findings     8. Brachmann, C., J. M. Sherman, S. E. Devine, E. E. Cameron, L. Pillus, genes involved carbon source regulation yeast            J. D. Boeke. 1995. SIR2 gene family, conserved bacteria humans, <br />  VOL. 4, 2005                                                                                                                                 PIS1 GENOMICS               613 <br />  <br />       functions silencing, cell cycle progression, chromosome stability.               phosphatidylinositol synthesis regulation S phase induction. Exp. <br />       Genes Dev. 9:2888&#226;&#8364;&#8220;2902.                                                                 Cell Res. 215:228&#226;&#8364;&#8220;233. <br />  9.   Brazas, R. M., D. J. Stillman. 1993. Swi5 zinc-finger Grf10           37.   Imoto, M., Y. Taniguchi, H. Fujiwara, K. Umezawa. 1994. Enhancement <br />       homeodomain proteins bind DNA cooperatively yeast HO promoter.                   CDP-DA:inositol transferase activity src- erbB2-transformed cells. <br />       Proc. Natl. Acad. Sci. USA 90:11237&#226;&#8364;&#8220;11241.                                              Exp. Cell Res. 212:151&#226;&#8364;&#8220;154. <br /> 10.   Butow, R. , N. G. Avadhani. 2004. Mitochondrial signaling: retro-       38.   Jackson, M., D. C. Crick, P. J. Brennan. 2000. Phosphatidylinositol       grade response. Mol. Cell 14:1&#226;&#8364;&#8220;15.                                                      essential phospholipid mycobacteria. J. Biol. Chem. 275:30092&#226;&#8364;&#8220;30099. <br /> 11.   Carman, G. M., S.  Henry. 1999. Phospholipid biosynthesis yeast      39.   Kaffman, ,  Herskowitz, R. Tjian, E. K. O&#226;&#8364;&#8482;Shea. 1994. Phosphoryla- <br />       Saccharomyces cerevisiae interrelationship metabolic pro-                tion transcription factor PHO4 cyclin-CDK complex, PHO80- <br />       cesses. Prog. Lipid Res. 38:361&#226;&#8364;&#8220;399.                                                    PHO85. Science 263:1153&#226;&#8364;&#8220;1156. <br /> 12.   Carman, G. M., G. M. Zeimetz. 1996. Regulation phospholipid                40.   Kaffman, , N. M. Rank, E. K. O&#226;&#8364;&#8482;Shea. 1998. Phosphorylation regulates <br />       synthesis yeast Saccharomyces cerevisiae. J. Biol. Chem. 271:13293&#226;&#8364;&#8220;              association transcription factor Pho4 import receptor Pse1/ <br />       13296.                                                                                  Kap121. Genes Dev. 12:2673&#226;&#8364;&#8220;2683. <br /> 13.   Chelstowska, , R.  Butow. 1995. RTG genes yeast function       41.   Kelley, M. J.,  M. Bailis, S.  Henry, G. M. Carman. 1988. Regulation <br />       communication mitochondria nucleus required                phospholipid biosynthesis Saccharomyces cerevisiae inositol. Inositol <br />       expression genes encoding peroxisomal proteins. J. Biol. Chem. 270:                  inhibitor phosphatidylserine synthase activity. J. Biol. Chem. 263: <br />       18141&#226;&#8364;&#8220;18146.                                                                            18078&#226;&#8364;&#8220;18085. <br /> 14.   Chen, D.-C., B.-C. Yang, T.-T. Kuo. 1992. step transformation          42.   Kelly, B. L., M. L. Greenberg. 1990. Characterization regulation       yeast stationary phase. Curr. Genet. 21:83&#226;&#8364;&#8220;84.                                       phosphatidylglycerolphosphate phosphatase Saccharomyces cerevisiae. <br /> 15.   Costenoble, R., L. Adler, C. Niklasson, G. Liden. 2003. Engineering              Biochim. Biophys. Acta 1046:144&#226;&#8364;&#8220;150. <br />       metabolism Saccharomyces cerevisiae anaerobic production            43.   Komeili, , E. K. O&#226;&#8364;&#8482;Shea. 1999. Roles phosphorylation sites       mannitol. FEMS Yeast Res. 3:17&#226;&#8364;&#8220;25.                                                      regulating activity transcription factor Pho4. Science 284:977&#226;&#8364;&#8220;980. <br /> 16.   Crosby, J. , J. B. Konopka, S. Fields. 2000. Constitutive activation     44.   Kuo, M.-H., E. Grayhack. 1994. library yeast genomic MCM1 <br />       Saccharomyces cerevisiae transcriptional regulator Ste12p mutations           binding sites contains genes involved cell cycle control, cell wall       amino-terminus. Yeast 16:1365&#226;&#8364;&#8220;1375.                                                 membrane structure, metabolism. Mol. Cell. Biol. 14:348&#226;&#8364;&#8220;359. <br /> 17.   Czech, M. P. 2000. <span id='am-49' about='obo:IMR_0001350' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001348 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-50' property="rdfs:label" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="PIP2" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="PIP2" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0001350" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-57' property="obo:IAO_0000115" content="PI(4,5)P2 is cleaved by phospholipase C-beta or gamma to generate IP3 and diacylglycerol." datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200477" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C04637" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="PI(4,5)P2" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span>PIP2</span> <span id='am-70' about='obo:IMR_0001352' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001348 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-71' property="oboInOwl:id" content="IMR:0001352" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="1-Phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="PIP3" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="IMR:0200396" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="KEGG:C05981" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Phosphatidylinositol-3,4,5-trisphosphate " datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="PI(3,4,5)P3" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="PIP3" datatype="xsd:string"></span><span id='am-80' property="obo:IAO_0000115" content="PI(3,4,5)P2 is generated by phosphorylation of PI(4,5)P at the 3 position of the inositol ring catalyzed by PI3-kinase. NOT cleaved by phospholipase C." datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Phosphatidylinositol-3,4,5-trisphosphate " datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate" datatype="xsd:string"></span>PIP3</span>: complex roles cell surface. Cell         45.   Kwast, K. E., L.-C. Lai, N. Menda, D. T. James III, S. Aref, P. V. Burke. <br />       100:603&#226;&#8364;&#8220;606.                                                                            2002. Genomic analyses anaerobically induced genes Saccharomyces <br /> 18.   Deguchi, , M. Imoto, K. Umezawa. 1996. Inhibition G1 cyclin                    cerevisiae: functional roles Rox1 factors mediating anoxic <br />       expression normal rat kidney cells inostamycin, phosphatidylinositol            response. J. Bacteriol. 184:250&#226;&#8364;&#8220;265. <br />       synthesis inhibitor. J. Biochem. (Tokyo) 120:1118&#226;&#8364;&#8220;1122.                           46.   Lamping, E., J. Lu   &#194;&#168;ckl, F. Paltauf, S.  Henry, S. Kohlwein. 1995. <br /> 19.   Deguchi, , K. Segawa, K. Hosaka,  B. Weinstein, K. Umezawa. 2002.               Isolation characterization mutant Saccharomyces cerevisiae       Overexpression phosphatidylinositol synthase enhances growth G1                  pleiotropic deficiencies transcriptional activation repression. Genetics <br />       progression NIH3T3 cells. Jpn. J. Cancer Res. 93:157&#226;&#8364;&#8220;166.                            137:55&#226;&#8364;&#8220;65. <br /> 20.   Distel, B., R. Erdmann, S. J. Gould, G. Blobel, D.  Crane, J. M. Cregg, G.      47.   Lee, T. , N. J. Rinaldi, F. Robert, D. T. Odom,  Bar-Joseph, G. K. Gerber, <br />       Dodt, Y. Fujiki, J. M. Goodman, W. W. Just, et al. 1996. unified nomen-               N. M. Hannett, C. T. Harbison, C. M. Thompson,  Simon, et al. 2002. <br />       clature peroxisome biogenesis factors. J. Cell Biol. 135:1&#226;&#8364;&#8220;3.                       Transcriptional regulatory networks Saccharomyces cerevisiae. Science <br /> 21.   Divecha, N., R. F. Irvine. 1995. Phospholipid signaling. Cell 80:269&#226;&#8364;&#8220;278.           298:799&#226;&#8364;&#8220;804. <br /> 22.   Elkhaimi, M., M. R. Kaadige, D. Kamath, J. C. Jackson, J. H. C. Biliran,      48.   Lopes, J. M., S.  Henry. 1991. Interaction trans cis regulatory <br />       J. M. Lopes. 2000. Combinatorial regulation phospholipid biosynthetic                elements INO1 promoter Saccharomyces cerevisiae. Nucleic Acids <br />       gene expression UME6, SIN3, RPD3 genes. Nucleic Acids Res.                   Res. 19:3987&#226;&#8364;&#8220;3994. <br />       28:3160&#226;&#8364;&#8220;3167.                                                                     49.   Luche, R. M., R. Sumrada, T. G. Cooper. 1990. cis-acting element <br /> 23.   Fischl,  S., G. M. Carman. 1983. Phosphatidylinositol biosynthesis             present multiple genes serves repressor protein binding site       Saccharomyces cerevisiae: purification properties microsome-associ-              yeast CAR1 gene. Mol. Cell. Biol. 10:3884&#226;&#8364;&#8220;3895. <br />       ated phosphatidylinositol synthase. J. Bacteriol. 154:304&#226;&#8364;&#8220;311.                    50.   Lykidis, , P. D. Jackson, C. O. Rock, S. Jackowski. 1997. role 24.   Fischl,  S., M. J. Homann, M.  Poole, G. M. Carman. 1986. Phos-                 CDP-diacylglycerol synthetase phosphatidylinositol synthase activity lev- <br />       phatidylinositol synthase Saccharomyces cerevisiae. J. Biol. Chem. 261:            els regulation cellular phosphatidylinositol content. J. Biol. Chem. <br />       3178&#226;&#8364;&#8220;3183.                                                                              272:33402&#226;&#8364;&#8220;33409. <br /> 25.   Gardocki, M. E., J. M. Lopes. 2003. Expression yeast PIS1 gene         51.   McAndrew, P. C., J. Svaren, S. R. Martin, W. Ho&#194;&#168;rz, C. R. Goding. 1998. <br />       requires multiple regulatory elements including Rox1p binding site. J. Biol.          Requirements chromatin modulation transcription activation       Chem. 278:38646&#226;&#8364;&#8220;38652.                                                                  Pho4 acidic activation domain. Mol. Cell. Biol. 18:5818&#226;&#8364;&#8220;5827. <br /> 26.   Grauslund, M., J. M. Lopes, B. R&#195;&#184;nnow. 1999. Expression GUT1,              52.   Myers,  M.,  Tzagoloff, D. M. Kinney, C. J. Lusty. 1986. Yeast shuttle <br />       encodes glycerol kinase Saccharomyces cerevisiae, controlled             integrative vectors multiple cloning sites suitable construction       positive regulators Adr1p, Ino2p, Ino4p, negative regulator             lacZ fusions. Gene 45:299&#226;&#8364;&#8220;310. <br />       Opi1p carbon source-dependent fashion. Nucleic Acids Res. 27:4391&#226;&#8364;&#8220;           53.   Nikawa, J. , T. Kodaki, S. Yamashita. 1988. Expression Saccha- <br />       4398.                                                                                   romyces cerevisiae PIS gene synthesis phosphatidylinositol Esche- <br /> 27.   Grauslund, M., B. R&#195;&#184;nnow. 2000. Carbon source-dependent transcrip-                  richia coli. J. Bacteriol. 170:4727&#226;&#8364;&#8220;4731. <br />       tional regulation mitochondrial glycerol-3-phosphate dehydrogenase         54.   Nikawa, J. , T. Kodaki, S. Yamashita. 1987. Primary structure       gene, GUT2, Saccharomyces cerevisiae.  J. Microbiol. 46:1096&#226;&#8364;&#8220;                  disruption phosphatidylinositol synthase gene Saccharomyces <span id='am-120' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-121' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span>cer</span>- <br />       1100.                                                                                   evisiae. J. Biol. Chem. 262:4876&#226;&#8364;&#8220;4881. <br /> 28.   Greenberg, M. L., J. M. Lopes. 1996. Genetic regulation phospholipid       55.   Nikawa, J. , S. Yamashita. 1984. Molecular cloning gene  <br />       biosynthesis Saccharomyces cerevisiae. Microbiol. Rev. 60:1&#226;&#8364;&#8220;20.                      coding CDPdiacylglycerol-inositol 3-phosphatidyltransferase Saccharomy- <br /> 29.   Griggs, D. W., M. Johnston. 1993. Promoter elements determining weak                ces cerevisiae. Eur. J. Biochem. 143:251&#226;&#8364;&#8220;256. <br />       expression GAL4 regulatory gene Saccharomyces cerevisiae. Mol.          56.   Nikawa, J. , S. Yamashita. 1997. Phosphatidylinositol synthase       Cell. Biol. 13:4999&#226;&#8364;&#8220;5009.                                                               yeast. Biochim. Biophys. Acta 1348:173&#226;&#8364;&#8220;178. <br /> 30.   Griggs, D. W., M. Johnston. 1991. Regulated expression GAL4            57.   Odom,  R.,  Stahlberg, S. R. Wente, J. D. York. 2000. role       activator gene yeast provides sensitive genetic switch glucose repres-         nuclear inositol 1,4,5-<span id='am-83' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-84' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span>triphosphate</span> kinase transcriptional control. Science <br />       sion. Proc. Natl. Acad. Sci. USA 88:8597&#226;&#8364;&#8220;8601.                                          287:2026&#226;&#8364;&#8220;2029. <br /> 31.   Gru&#194;&#168;nweller, ,  E. Ehrenhofer-Murray. 2002. novel yeast silencer:       58.   Ohanian, J., V. Ohanian. 2001. Sphingolipids mammalian cell signal- <br />       2&#226;?&#174; origin Saccharomyces cerevisiae HST3-, MIG1- SIR-depen-               ling. Cell. Mol. Life Sci. 58:2053&#226;&#8364;&#8220;2068. <br />       dent silencing activity. Genetics 162:59&#226;&#8364;&#8220;71.                                      59.   Overkamp, K. M., B. M. Bakker, P. Kotter,  van Tuijl, S. Vries, J. P. van <br /> 32.   Guarente, L., T. Mason. 1983. Heme regulates transcription                   Dijken, J. T. Pronk. 2000. vivo analysis mechanisms       CYC1 gene S. cerevisiae upstream activation site. Cell 32:1279&#226;&#8364;&#8220;1286.          oxidation cytosolic NADH Saccharomyces cerevisiae mitochondria. J. <br /> 33.   Henry, S. , J. L. Patton-Vogt. 1998. Genetic regulation phospholipid           Bacteriol. 180:2823&#226;&#8364;&#8220;2830. <br />       metabolism: yeast model eukaryote. Prog. Nucleic Acid Res. Mol. Biol.        60.   Purdue, P. E., P. B. Lazarow. 2001. Peroxisome biogenesis. Annu. Rev. <br />       61:133&#226;&#8364;&#8220;179.                                                                             Cell Dev. Biol. 17:701&#226;&#8364;&#8220;752. <br /> 34.   Herrick, D., R. Parker,  Jacobson. 1990. Identification comparison      61.   Remize, F., B. Cambon, L. Barnavon, S. Dequin. 2003. Glycerol forma- <br />       stable unstable mRNAs Saccharomyces cerevisiae. Mol. Cell. Biol.              tion wine fermentation mainly linked Gpd1p partly <br />       10:2269&#226;&#8364;&#8220;2284.                                                                           controlled HOG pathway. Yeast 20:1243&#226;&#8364;&#8220;1253. <br /> 35.   Holstege, F. C. P., E. G. Jennings, J. J. Wyrick, T.  Lee, C. J. Hengartner,    62.   Ren, B., F. Robert, J. J. Wyrick, O. Aparicio, E. G. Jennings,  Simon, J. <br />       M. R. Green, T. R. Golub, E. S. Lander, R.  Young. 1998. Dissecting               Zeitlinger, J. Schreiber, N. Hannett, E. Kanin, et al. 2000. Genome-wide <br />       regulatory circuitry eukaryotic genome. Cell 95:717&#226;&#8364;&#8220;728.                       location function DNA binding proteins. Science 290:2306&#226;&#8364;&#8220;2309. <br /> 36.   Imoto, M., T. Morii,  Deguchi, K. Umezawa. 1994. Involvement             63.   Robinson, K. , J.  Koepke, M. Kharodawala, J. M. Lopes. 2000.  614         GARDOCKI ET AL.                                                                                                                          EUKARYOT. CELL <br />  <br />       network yeast basic helix-loop-helix interactions. Nucleic Acids Res. 28:     74. Terlecky, S. R., M. Fransen. 2000. peroxisomes arise. Traffic <br />       4460&#226;&#8364;&#8220;4466.                                                                           1:465&#226;&#8364;&#8220;473. <br /> 64.   Saiardi, , J. J. Caffrey, S. H. Snyder, S. B. Shears. 2000. inositol   75. Ter Linde, J. J. M., H. Y. Steensma. 2002. microarray-assisted <br />       hexakisphosphate kinase family. J. Biol. Chem. 275:24686&#226;&#8364;&#8220;24692.                      screen potential Hap1 Rox1 target genes Saccharomyces cerevisiae. <br /> 65.   Saiardi, , J. J. Caffrey, S. H. Snyder, S. B. Shears. 2000. Inositol           Yeast 19:825&#226;&#8364;&#8220;840. <br />       polyphosphate multikinase (ArgRIII) determines nuclear mRNA export            76. Ter Linde, J. J. M., H. Liang, R. W. Davis, H. Y. Steensma, J. P. Van <br />       Saccharomyces cerevisiae. FEBS Lett. 468:28&#226;&#8364;&#8220;32.                                      Dijken, J. T. Pronk. 1999. Genome-wide transcriptional analysis 66.   Saiardi, , H. Erdjument-Bromage,  M. Snowman, P. Tempst, S. H.               aerobic anaerobic chemostat cultures Saccharomyces cerevisiae. J. <br />       Snyder. 1999. Synthesis diphosphoinositol pentakisphosphate newly            Bacteriol. 181:7409&#226;&#8364;&#8220;7413. <br />       identified family higher inositol polyphosphate kinases. Curr. Biol.          77. Titorenko, V. , R.  Rachubinski. 2001. life cycle peroxi- <br />       9:1323&#226;&#8364;&#8220;1326.                                                                          Nat. Rev. Mol. Cell Biol. 2:357&#226;&#8364;&#8220;368. <br /> 67.   Shao, D., C. L. Creasy, L. W. Bergman. 1998. cysteine residue           78. van den Bosch, H., R. B. H. Schutgens, R. J.  Wanders, J. M. Tager. <br />       helixII bHLH domain essential homodimerization yeast            1992. Biochemistry peroxisomes. Annu. Rev. Biochem. 61:157&#226;&#8364;&#8220;197. <br />       transcription factor Pho4p. Nucleic Acids Res. 26:710&#226;&#8364;&#8220;714.                       79. van Roermund, C. W. T., H. R. Waterham, L. Ijlst, R. J.  Wanders. <br />                                                                                            2003. Fatty acid metabolism Saccharomyces cerevisiae. Cell. Mol. Life Sci. <br /> 68.   Shao, D., C. L. Creasy, L. W. Bergman. 1996. Interaction Saccharo- <br />                                                                                            60:1838&#226;&#8364;&#8220;1851. <br />       myces cerevisiae Pho2 Pho4 increases accessibility activation <br />                                                                                        80. Vizeacoumar, F. J., J. C. Torres-Guzman, D. Bouard, J. D. Aitchison,       domain Pho4. Mol. Gen. Genet. 251:358&#226;&#8364;&#8220;364. <br />                                                                                            R.  Rachubinski. 2004. Pex30p, Pex31p, Pex32p form family 69.   Shields, D., P. Arvan. 1999. Disease models provide insights post-          peroxisomal integral membrane proteins regulating peroxisome size       Golgi protein trafficking, localization processing. Curr. Opin. Cell Biol.       number Saccharomyces cerevisiae. Mol. Biol. Cell 15:665&#226;&#8364;&#8220;677. <br />       11:489&#226;&#8364;&#8220;494.                                                                      81. Vizeacoumar, F. J., J. C. Torres-Guzman, Y. Y. C. Tam, J. D. Aitchison, 70.   Smith, J. J., M. Marelli, R. H. Christmas, F. J. Vizeacoumar, D. J. Dilworth,        R.  Rachubinski. 2003. YHR150w YDR479c encode peroxisomal <br />       T. Ideker, T. Galitski, K. Dimitrov, R.  Rachubinski, J. D. Aitchison.         integral membrane proteins involved regulation peroxisome num- <br />       2002. Transcriptome profiling identify genes involved peroxisome            ber, size, distribution Saccharomyces cerevisiae. J. Cell Biol. 161:321&#226;&#8364;&#8220; <br />       sembly function. J. Cell Biol. 158:259&#226;&#8364;&#8220;271.                                      332. <br /> 71.   Starai, V. J., H. Takahashi, J. D. Boeke, J. C. Escalante-Semerena. 2003.    82. Vogel, K., W. Ho&#194;&#168;rz,  Hinnen. 1989. positively acting regulatory <br />       Short-chain fatty acid activation acyl-coenzyme synthetases requires            proteins PHO2 PHO4 physically interact PHO5 upstream activa- <br />       SIR2 protein function Salmonella enterica Saccharomyces cerevisiae.           tion regions. Mol. <br /> </body></html>