<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>et al. <br /> Freimoser <br /> 2006 <br /> Volume 7, Issue 11, Article R109                                                                                             Open Access <br /> Research <br /> Systematic screening polyphosphate (poly P) levels yeast <br />  <br />  <br />  <br />  <br />                                                                                                                                                              <span id='am-51' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br /> mutant cells reveals strong interdependence primary <br /> metabolism <br /> Florian M Freimoser*, Hans Caspar H&#195;&#188;rlimann*, Claude Jakob&#226;&#8364;&#160;, <br /> Thomas P Werner* Nikolaus Amrhein* <br /> Addresses: *Institute Plant Sciences, ETH Zurich, 8092 Zurich, Switzerland. &#226;&#8364;&#160;Institute Microbiology, ETH Zurich, Zurich, Switzerland. <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reviews <br /> Correspondence: Florian M Freimoser. Email: ffreimoser@ethz.ch <br />  <br />  <br /> Published: 15 November 2006                                                   Received: 2 August 2006 <br />                                                                               Revised: 4 October 2006 <br /> Genome Biology 2006, 7:R109 (doi:10.1186/gb-2006-7-11-r109) <br />                                                                               Accepted: 15 November 2006 electronic version article complete online http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reports <br /> &#194;&#169; 2006 Freimoser et al.; licensee BioMed Central  open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br /> involved <br /> &lt;p Polyphosphate <br />         systematicmaintenance <br />            metabolism <br />                     analysis ofin  polyphosphate <br />                                    ofyeast <br />                                       normal polyphosphate      content <br />                                                     levels yeast knockout  strains                                                                          provides  insights                                                                                             almostinto phosphate <br />                                                                                                         non-essential <br />                                                                                                                   homeostasis.&lt;/p&gt; <br />                                                                                                                          gene identified 255 genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br />                  Abstract <br />  <br />                  Background: Inorganic polyphosphate (poly P) occurs universally organisms bacteria <br />                  man. functions, example, phosphate energy store, involved activation <br />                  regulation proteins. Despite ubiquitous occurrence important functions, unclear <br />                  poly P synthesized poly P metabolism regulated higher eukaryotes. work <br />                  describes systematic analysis poly P levels yeast knockout strains mutated                  non-essential gene. <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />                  Results: consecutive screens, 255 genes  4% yeast genome)                  involved maintenance normal poly P content. genes encoded proteins <br />                  functioning cytoplasm, vacuole transport transcription. reduced poly P <br />                  content, strains exhibited reduced total phosphate content, showed altered <span id='am-27' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span>                  glycogen levels disturbed secretion acid phosphatase. <br />  <br />                  Conclusion: Cellular energy phosphate homeostasis suggested result                  equilibrium poly P, ATP free phosphate cell. Poly P serves buffer                  ATP free phosphate levels   essential consequently                                             interactions <br />                  variable component network.  strains reduced poly P levels                  affected ATP phosphate content, components depend ATP <br />                  free phosphate content, glycogen secreted phosphatase activity. <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br /> Background                                                                     buffer [2-4] activation enzymes [5,6] regu- <br /> Inorganic polyphosphate (poly P) linear polymer                      lation chromatin condensation, gene expression trans- <br /> consists phosphoanhydride linked phosphate residues                     lation [1,7,8]. Poly P involved bacterial pathogenicity <br /> occurs ubiquitously organisms living cells [1].                 [9,10], survival stationary phase bacteria yeast <br /> functions poly P range role phosphate store                   [9,11,12], adaptation alkaline osmotic stress <br />  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.2 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                             http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         [13-15]. slime mold Dictyostelium discoideum, poly P                  proteins) cell homeostasis (2 proteins) biological <br />         regulates development predation behavior [16],                    function terms; transporter activity (30 proteins)         humans blood coagulation accelerated fibrinolysis                  significantly overrepresented molecular function term <br />         delayed poly P [17]. cellular level, poly P               (Figure 1b,c). proteins encoded identified genes <br />         detected cytosol, nucleus, mitochondria,               participate biological processes organelle cell <br />         endoplasmic reticulum [7,18,19]. Poly P particularly                  wall organization biogenesis (24 10 proteins, respec- <br />         prominent acidocalcisomes trypanosomatids [20],                    tively), protein biosynthesis modification (17 12 pro- <br />         Saccharomyces cerevisiae entire poly P                     teins, respectively), DNA RNA metabolism (10 7 <br />         pool resides vacuole [21,22].                                         proteins, respectively) involved response                                                                                      stress (11 proteins). Additional molecular functions         Despite universal occurrence broad functions,               prevalent discovered proteins included activities <br />         little known poly P metabolism regulation.               hydrolases, transferases, transcription regulators         S. cerevisiae  higher eukaryotes)  example,          structural molecules (18, 17, 16 14 proteins, respectively) <br />         known poly P synthesized, despite fact                   protein DNA binding proteins (15 8 proteins, <br />         20% organism&apos;s dry weight com-                      respectively) (Figure 1c). <br />         posed poly P [1]. gain broad view poly P metabolism <br />         yeast identify pathways involved regulation              Vacuolar proteins important determining <br />         poly P levels, extracted quantified poly P knock-              poly P levels <br />         strains non-essential yeast genes.                         255 genes ranked according impact poly P <br />                                                                                      content. mutant cells strongly affected poly <br />                                                                                      P content highly enriched knockouts genes         Results                                                                      encode vacuolar proteins (6 20 vacuolar proteins <br />         Altogether, 4,765 strains YKO collection [23],                  10 strongly affected mutants). Vacuolar func- <br />         sisting knockouts non-essential yeast genes, ini-                 tion depends acidification vacuolar lumen, <br />         tially screened. Strains hypo- hyper-                         mediated vacuolar H+-ATPase (V-ATPase) <br />         accumulated poly P subjected consecutive                       complex. poly P screen, 9 14 V-ATPase sub- <br />         rounds screening. round, 255 strains                 units required normal poly P content (Vma5 (14), <br />         YKO collection altered poly P levels exper-             Vma8 (10), Vma10 (13), Vma13 (34), Vph1 (1), Cup5 (5), Tfp3 <br />         iments  complete list data available online Addi-             (15), Ppa1 (20), Vma6 (27); ranks given parentheses).         tional data file 1). 255 strains reduced             addition, regulators (Rav1 (43), Vps34 (30), Fab1 (150)         poly P levels occasionally, specific time-points                 Vac14 (213)) assembly factors (Vma22 (16), Vph2 (23))         growth, poly P hyper-accumulation relative                     V-ATPase discovered important         wild-type strain observed.                                               normal poly P levels. <br />  <br />         cellular compartments involved maintenance                        VPH1 knockout, VTC4 VTC1 deletion <br />         poly P levels                                                             strains second affected mutants,         255 discovered genes categorized Gene                      knockout VTC2 hypo-accumulated poly P (rank <br />         Ontology   Slim terminology significantly overrepre-                  219). Vtc3, subunits         sented terms (relative occurrence yeast                vacuolar transporter chaperon (Vtc) complex, pre- <br />         genome) determined (Figure 1a-c). cellular com-                     viously shown required maintenance poly P levels <br />         partments highest number proteins important                  [3]. proteins involved hypo-accumulation poly <br />         poly P content cytoplasm (59 proteins), nucleus                 P involved membrane docking fusion Golgi- <br />         (56 proteins), mitochondrion mitochondrial                       endosome endosome vacuole steps (Vps33, <br />         envelope (33 9 proteins, respectively), vacuole (20                  ninth affected mutant), represent alternative path- <br />         proteins) endoplasmic reticulum (13 proteins) (Fig-                  way Golgi vacuole (effected AP-3 com- <br />         ure 1a). genes important poly P content                plex consists Apm3, Apl5, Apl6 Aps3; ranks 21, 59, <br />         encoded membrane ribosomal proteins (12 11 pro-                      57 55, respectively). Vam3, vacuolar t-SNARE protein <br />         teins, respectively) proteins functioning Golgi appa-              traffics vacuole AP-3 complex [24],         ratus (5 proteins), peroxisome cell wall (4 proteins              important maintenance poly P levels (rank 8). <br />          (Figure 1a). terms &apos;vacuole&apos; &apos;mem- <br />         brane&apos; significantly overrepresented 255 genes                   strongly affected strains, genes encoding         important maintenance poly                  phosphofructokinase subunits (PFK1 PFK2, ranks 28 <br />         P levels. representation cellular compartments                    17) pyruvate kinase PYK2 (rank 7) knocked <br />         reflected highly overrepresented biological                   additional strains, components glycoly- <br />         molecular function terms: transport vesicle mediated                     sis identified (GCR2, GPM3, YOR283W; 63, 201         transport (36 22 proteins, respectively), transcription (16              181, respectively). <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                     Genome Biology 2006,     Volume 7, Issue 11, Article R109      Freimoser et al. R109.3 <br />  <br />  <br />  <br />  <br />                  Cellular component                             (b) Biological process                         (c) Molecular function <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      comment <br />                   Cytoplasm <br />  <br />                      Nucleus <br />                                                                    *Transport <br />  <br />                                                                     Unknow n <br />                                                                                                                     Unknow n <br />  <br />                                                                                                              *Transporter act. <br />                                                                                                                                               // <br />               Mitochondrion                         Organelle org. <span id='am-20' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; biogen.                                   Hydrolase act. <br />                    *Vacuole                    *Vesic le --mediated transport                                Transferase act. <br />                           ER                            Protein biosynthesis                              Transcript. reg. act <br />                  *Membrane                                     *Transcription <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      reviews <br />                                                                                                                Protein binding <br />                    Unknown                              Protein modification                               Structural mol. act. <br />  <br />                     Ribsome                              Response stress                                      DNA binding <br />  <br />                          PM                          Cell wall org. &amp; biogen..                                     Ligase act. <br />  <br />       Mitochond. envelope                                  DNA metabolism                                Oxidoreductase act. <br />  <br />        Endomembrane sys.                                             Meiosis                               Protein kinase act. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      reports <br />             Golgi apparatus                       Membrane org. &amp; biogen..                               Translation reg . act. <br />  <br />                     Cell w                             RNA metabolism                                   Enzyme reg . act. <br />  <br />                  Nucleolous                                      Sporulation                                   Peptidase act. <br />  <br />                 Peroxis ome                                 <span id='am-7' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="lipid" datatype="xsd:string"></span>Lipid</span> metabolis m                           Signal transducer act. <br />  <br />                                                                                                              <br />  <br />  <br />  <br />                                                                                                                                                                      deposited research <br />                                                                                                                                                // <br />                                  0       30      60                              0       20         40                           0    20        60 <br />                                      Number genes                                 Number genes                              Number genes <br />                                        category                                    category                                 category <br />  Figure <br />     cellular <br />          1 compartments involved maintenance poly P levels cellular compartments involved maintenance poly P levels. Categorization 255 genes important poly P levels using Slim terminology    cellular component, (b) biological process (c) molecular function. Categories significantly overrepresented (P &#226;&#8240;&#164; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      refereed research <br /> 0.05; red bars) marked asterisk. 15 abundant categories shown  genes summarized  ). <br />  <br />  <br />  <br /> Poly P profiles identified mutant strains                                 higher 2 h 24 h (Figure 2b). Cluster 2 significantly 255 discovered strains poly P data different                    enriched (compared complete genome) cytoplasmic <br /> experiments/time-points available: second                     mitochondrial proteins proteins functioning screen (performed 4 h time-point) samples                    organelle organization biogenesis (Figure 2a). mito- screen (2 h, 4 h, 8 h 24 h samples). Hier-                    chondrial proteins cluster 2 comprised, example, archical clustering experiments log2-trans-                  mitochondrial phosphate transporter (Mir1) mito- <br /> formed poly P data revealed 4 h samples                    chondrial ribosomal proteins (Mrpl33, Mrp51 Mrpl27).                            interactions <br /> experiments grouped (Figure 2d). 2 h time-                          Cluster 4 contained genes caused dramatic <br /> point similar 4 h time-point 8 h                     effect poly P levels deleted: 22 genes cluster 24 h samples (Figure 2). Clustering genes resolved                     total 39) 30 important genes <br /> distinct groups genes similar poly P patterns.                       maintenance poly P levels. group genes cluster genes (Figure 2, cluster 1) comprised seven genes                     included V-ATPase subunits (Vma8, Ppa1, caused elevated poly P levels 2 h 4 h time-                      Cup5, Vma10, Vph1, Vma5) discovered points deleted. seven genes cluster                    additional components vacuole. Correspondingly,                                                                                                                                                                      information <br />  <br />  <br />  <br />  <br /> (KRE1, ECM33 RIM21) encode proteins function                         cluster significantly enriched vacuolar, membrane cell wall biosynthesis organization, Slim cate-                     cytoplasmic proteins involved transport, cell <br /> gory significantly enriched cluster (Figure 2a).                  homeostasis vesicle mediated transport exhibit trans- clusters (Figure 2, clusters 2 4) relative poly                porter hydrolase activity (Figure 2a). distinct cluster <br /> P content  average, minimal 4 h time-point                     3, membrane proteins function transport  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.4 Genome Biology 2006,      Volume 7, Issue 11, Article R109                         Freimoser et al.                                                            http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />                                                        2nd screen (4h) <br />                                      1st screen (4h) <br />               <br />                                                                                                                                            Poly P ratio (log2) <br />  <br />  <br />  <br />                                                                               24h <br />                                                                          4h <br />  <br />  <br />                                                                                     8h <br />                                 2h <br />  <br />  <br />  <br />  <br />                                                                                                                                     -3.0           0             3.0 <br />  <br />                      1 <br />  <br />                                                                                              Cell. component: cytoplasm, mitochondrium <br />                       2                                                                      Biol . process : organelle org. &amp; biogen. <br />  <br />  <br />  <br />  <br />                                                                                              Cell. component: membrane <br />                       3                                                                      Biol . process : transport <br />  <br />  <br />  <br />                                                                                             Cell. component : vacuole, membrane, cytoplasm <br />  <br />                       4                                                                      Biol . process : transport, cell homeostasis, vesicle <br />                                                                                              mediated transport <br />                                                                                              Mol. function : transporter act., hydrolase act. <br />  <br />  <br />                                                                                            (b)                                        1 <br />                                                                                                   Relative poly P content (log 2) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        Cluster 1 <br />  <br />                                                                                                                                                                                        Cluster 2 <br />                                                                                                                                       0 <br />                                                                                                                                                                                        Cluster 3 <br />                                                                                                                                      -1                                                Cluster 4 <br />  <br />                                                                                                                                      -2 <br />  <br />                                                                                                                                      -3 <br />  <br />                                                                                                                                      -4 <br />                                                                                                                                               2h       4h        8h        24h <br />  <br />  <br />         Figure 2 255 genes poly P screen resolved distinct groups similar poly P profiles <br />         Clustering <br />         Clustering 255 genes poly P screen resolved distinct groups similar poly P profiles.   Hierarchical clustering (uncentered Pearson <br />         correlation, complete linkage) log2-transformed data (relative wild-type). Clusters significantly enriched (P &#226;&#8240;&#164; 0.05) Slim category <br />         marked colored bars branches. (b) Average poly P values (log2-transformed, relative wild type) genes distinct clusters <br />           time-points. <br />  <br />  <br />  <br />  <br />                                                                                          Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                              Genome Biology 2006,    Volume 7, Issue 11, Article R109   Freimoser et al. R109.5 <br />  <br />  <br />  <br />  <br /> significantly overrepresented,  average, relative                               3). Poly P content total phosphate levels strongly <br /> poly P content varied slightly increased                               reduced 30 non-essential mutants knock- <br /> growth corresponding deletion strains (Figure 2b).                                  TFP3  slightly reduced poly P total phosphate <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           comment <br />                                                                                            levels) &#206;&#8221;ecm14, &#206;&#8221;mrp51,&#206;&#8221;apm3 &#206;&#8221;sir3 strains strains altered poly P levels affected                                  normal content total phosphate (Figure 3). ATP total phosphate content, ATP levels, acid                                         levels exhibited highest variability 0 h time-point, <br /> phosphatase activity glycogen accumulation                                             cluster knockouts having increased content <br /> Comparison 255 non-essential ORFs                                       (knockouts ERG6, VTC4, VTC1, PYK2, VPH1, CUP5 important poly P content results published                               ECM14). Strongly reduced ATP levels measured large scale analyses revealed 56 strains affected                           knockouts dubious open reading frame (ORF) glycogen accumulation [25], 4 strains                                YPR099c uncharacterized ORF YOL019w (Figure 3). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           reviews <br /> reduced poly P levels previously screen                                30 knockouts hyper-accumulated glycogen, altered PHO5 (acid phosphatase (rAPase)) regulation,                                   2 h 0 h 24 h time-points. exception serves indicator activation state PHO                            mutation dubious ORF YPR099c, pathway [26,27]. verify data screen,                                undetectable glycogen levels 0 h 2 h time- <br /> test poly P phenotype accompanied complex                                 points  explained effect MRPL51, <br /> phenotypes, measured cell density, poly P levels, total                                 encoded opposite strand). 30 <br /> phosphate content, ATP concentrations, glycogen levels                                 knockout mutants exhibited strongly increased rAPase <br /> rAPase activity 30 affected knockout strains.                                  activity 0 h 24 h time-points. Exceptions  <br />  <br />  <br />  <br />                                                                                                                                                                           reports <br />                                                                                            knockouts ECM14, VTC4, VTC1 VAM3 (Figure 3). <br /> Overall, cell density ATP content affected <br /> characteristics, poly P total phosphate content gen- <br /> erally reduced, glycogen levels rAPase activity                               Discussion <br /> increased (Figure 3).                                                                      Previous screen described  yeast <br />                                                                                            mutant cells analyzed respect poly P <br /> Cell density slightly reduced  roughly half OD600)                              content  knowledge, time  <br />  <br />  <br />  <br />                                                                                                                                                                           deposited research strains, 2 h 5 h time-points (Figure                           specific metabolite,  poly P, isolated quantified <br />  <br />  <br />                                    OD600          Poly P                 Pi tot            ATP         Glycogen       rAPase <br />  <br />                                 Time [h]              Time [h]          Time [h]           Time [h]     Time [h]      Time [h] <br />                                0   2   5 24       0    2   5 24         0 2   5 24     0     2 5 24    0 2   5 24    0   2 5 24 <br />                                                                                                                                   ypr099c <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           refereed research <br />                                                                                                                                   reg1 <br />                                                                                                                                   vma22 <br />                                                                                                                                   vps33 <br />                                                                                                                                   ykl118w <br />                                                                                                                                   vma8 <br />                                                                                                                                   vma10 <br />                                                                                                                                   vma5 <br />                                                                                                                                   yol019w <br />                                                                                                                                   tfp3 <br />                                                                                                                                   pfk2 <br />                                                                                                                                   pfk1 <br />                                                                                                                                   ecm14 <br />                                                                                                                                   mrp51 <br />                                                                                                                                   apm3 <br />                                                                                                                                   sir3 <br />                                                                                                                                   ppa1 <br />                                                                                                                                   pep5 <br />                                                                                                                                   vph2 <br />                                                                                                                                   vma6 <br />                                                                                                                                   tat2 <br />                                                                                                                                   erg6 <br />  <br />  <br />  <br />                                                                                                                                                                           interactions <br />                                                                                                                                   vtc4 <br />                                                                                                                                   vtc1 <br />                                                                                                                                   pyk2 <br />                                                                                                                                   vph1 <br />                                                                                                                                   cup5 <br />                                                                                                                                   vam3 <br />                                                                                                                                   ypt7 <br />                                                                                                                                   vps34 <br />  <br />  <br />  <br />  <br />                                        Ratio (log2) <br />  <br />                             -2.0              0                   2.0 <br />                                                                                                                                                                           information <br />  <br />  <br />  <br />  <br /> Figure strains <br />         3     altered poly P levels affected total phosphate content, ATP levels, rAPase activity glycogen accumulation strains altered poly P levels affected total phosphate content, ATP levels, rAPase activity glycogen accumulation. measurements (OD600, total phosphate content, ATP levels, rAPase activity glycogen content) 30 strains affected poly P levels given relative wild type (log2-transformed). data hierarchically clustered (uncentered Pearson correlation, average linkage). <br />  <br />  <br />  <br />                                                                     Genome Biology 2006, 7:R109 <br />  R109.6 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                             http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         complete mutant cell collection.                    data impossible conclude V-ATPase activity <br />         consecutive screens, 255 knockout mutant strains                        regulates poly P levels directly indirectly. poly P levels <br />         altered poly P levels identified. genes affected                 VMA4 knockout strain slightly restored         mutant cells encode proteins intracellular com-               growing cells acid buffered growth medium, Ogawa et <br />         partments components functioning processes                    al. [3] concluded V-ATPase itself essential         primary metabolism. broad analysis defined novel                   poly P metabolism. Instead, authors suggested         biological functions 250 yeast genes allowed,                  Vtc complex (consisting Vtc1, Vtc2, Vtc3 Vtc4),         time, global view pathways processes              regulates vacuolar membrane fusion, morphology func- <br />         affecting poly P metabolism yeast.                 tion [37,38], directly involved synthesis poly P         genes required maintenance normal poly P                    yeast [3]. fact mutants affected various stages         tent, strains hypo- hyper-accumulated poly P                     retrograde forward vesicle trafficking,         reduced total phosphate content, altered ATP <span id='am-52' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-53' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>gly</span>-                autophagy, indicate intact secretory pathway         cogen levels regulated rAPase secretion.                 required normal poly P content supports hypoth- <br />         analysis conclude poly P content extremely sen-                esis Vtc complex influences poly P levels indirectly <br />         sitive parameter highly intertwined primary                     [38]. secondary effect poly P metabolism         metabolism.                                                                  caused  mislocalization misregulation vacuolar <br />                                                                                      proteins important poly P synthesis storage; <br />          number knockout mutant cells reduced                    impairment poly P synthesis transport         poly P levels probably underestimated:                   secretory pathway.  pathways indirectly <br />         mutants previously identified hypo-accumulate                     affect poly P content impact vacuole secre- <br />         poly P screen. example, knock-                tory pathway   link seemingly unrelated path- <br />         outs VMA4, VTC3, ARG82 KCS1 discovered,                    ways poly P metabolism. reduced poly P content         knockouts members PHO pathway (PHO3,                   mutants subunits AP-3 complex, <br />         PHO4  example, PHO84), shown                       mediates alternative pathway Golgi         convey hypo-accumulation poly P [3,26,28].                     vacuole [39] medically relevant type         indication stringency screening conditions                 Hermansky-Pudlak syndrome [40], case         selection criteria suggests true number non-                 indirect effects; mutations proteins targeted         essential genes involved poly P metabolism higher                 vacuole AP-3 complex contain putative di- <br />         255 genes reported  reliability high-               leucine signal AP-3 targeting hypo-accumulated poly P <br />         throughput screening procedure confirmed                     (data shown). <br />         fact 30 strains YKO collection indi- <br />         vidually tested showed strongly reduced poly P levels.                deletion genes encode vacuolar proteins <br />                                                                                      caused severe reduction poly P levels, frac- <br />         poly P observed functions differ-                tion discovered genes encoded proteins function         ent organelles [7,18,19,29], 90% 99% poly P local-              vacuole. remaining 235 genes encode proteins func- <br />         ized vacuole [1,21,22]. Consequently, measure                      tioning cellular compartments. Assessing poly P metab- <br />         exclusively vacuolar poly P proteins                     olism larger picture phosphate energy <br />         organelles screen affect                  homeostasis help explain involvement multi- <br />         vacuolar poly P levels. Poly P storage central function            tude pathways. PolyP, constitute 50%         yeast vacuole   mutants affected vacuolar                 total phosphate content yeast (data shown),         functions morphology likely impaired                   seen form cellular phosphate free phos- <br />         capability store polyP. Important vacuolar functions                      phate, DNA, RNA, nucleotides phospholipids (Figure 4). <br />         include maturation activation different proteins                  Considering example poly P metabolism Escherichia <br />         physiological functions storage metabolites                coli evidence ATP-dependent poly P kinase <br />         cell homeostasis [30-32] depend V-ATPase.                 activity yeast [7], assumed majority yeast <br />         large multimeric, partly membrane-embedded complex                      poly P synthesized ATP (Figure 4). poly P pool         conserved yeast man relevant human                 direct equilibrium ATP pool,         diseases osteopetrosis distal renal tubular acido-               hydrolyzed buffer free phosphate levels (Figure 4). ATP <br />         sis [33,34]. screen, subunits V-ATPase               itself hydrolyzed  example, V-ATPase         regulators identified. addition, V-                   assure vacuolar acidification). importantly, ATP <br />         ATPase activity glucose regulated; presence                pool directly indirectly sustains cellular activities         glucose, V-ATPase functional absence glu-                processes DNA RNA metabolism, ribosome bio- <br />         cose causes reversible dissociation V0 V1 subunits                genesis assembly, transcription protein biosynthesis, <br />         V-ATPase inactivated [35]. phenomenon                       synthesis storage carbohydrates (Figure 4). <br />         explain decline poly P levels soon growth              pathways indirectly associated         medium depleted glucose [36].  based                 poly P metabolism. Ribosome biogenesis, example, claims <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                       Genome Biology 2006,   Volume 7, Issue 11, Article R109   Freimoser et al. R109.7 <br />  <br />  <br />  <br />  <br />                                                                                   Conclusion <br />            Phosphorylated                Storage                                  PolyP metabolism primary metabolism strongly <br />           sugars &amp; proteins           carbohydrates                               interdependent: hand, poly P levels depend necessar- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   comment <br />                                                                                   ily integrity primary metabolism reflect                                                                                   physiological state cell. hand, poly P itself <br />                                                                                   influences cell metabolism importance energy <br />  Phospholipids <br />                             ATP                          poly P                   phosphate homeostasis. genome-wide analysis <br />                                                                                   poly P content implies poly P indirect link <br />                                                                                   different cellular pathways phosphate, <br />                                                                                   energy <span id='am-13' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span>carbohydrate</span> metabolism transcription                        NTPs <br />                                              Pi                                   translation activities. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   reviews <br />                 DNA / RNA <br />                                                                                   Materials methods <br />                                            Pho4                                   Strains culturing conditions <br />                                        (PHO pathway)                              haploid yeast knock strains (YKO [23], based                                                                                   strain BY4741: MAT his3&#206;&#8221;1 leu2&#206;&#8221;0 met15&#206;&#8221;0 ura3&#206;&#8221;0)                                                                                   grown 96 deep plates 30&#194;&#176;C YPD medium (1% <br /> Figure <br /> Poly <br /> network&apos; <br />      P links <br />          4 energy phosphate metabolism &apos;phosphate-energy-                w/v yeast extract, 2% w/v peptone, 2% w/v glucose,  <br />  <br />  <br />  <br />                                                                                                                                                                   reports <br /> Poly P links energy phosphate metabolism &apos;phosphate-energy-              optional addition 200 mg/l G418). screen, fresh <br /> network&apos;. Enzymatic reactions main phosphate/energy pools,            YPD medium inoculated 40 &#206;&#188;l precultured <br /> ATP poly P (shown black arrows)  partly) hypothesized.        (3d) stationary cells cells harvested 4 h  Flow different organic phosphate/energy pools indicated gray <br /> arrows. contrast ATP, poly P directly interconnected       second round screen) 2 h, 4 h, 8 h 24 h phosphate pools   levels fluctuate                    round screen). Growth cell number moni- <br />                                                                                   tored measuring light scattering 600 nm (OD600)                                                                                   BioTek PowerWave&#226;&#8222;&#162; XS microplate spectrophotometer <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   deposited research <br />                                                                                   (BioTEK Instruments , Vermont, USA). detailed poly <br />                                                                                   P measurements selected strains, culture inocu- <br />                                                                                   lated cell density 1 OD600/ml (approximately 107 cells/ <br />                                                                                   ml), time-point 1 OD600 equivalent cells 60% total transcription, 90% mRNA splicing,                       collected poly P quantification. <br /> requirement RNA polymerases dedication 200 different proteins   considerably                          Poly P purification quantification <br /> affects energy homeostasis [41,42].                                               Poly P extraction, purification quantification  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   refereed research <br />                                                                                   formed described previously [36]. short, cells pel- <br /> Intervention point phosphate-energy-net-                       leted deep plates centrifugation (approximately <br /> work inevitably causes effects require read-                       640 g, 15 minutes, 4&#194;&#176;C), supernatant discarded, 50 &#206;&#188;l <br /> justment allocation resources different                     1 M sulphuric acid added suspension neu- <br /> phosphate energy pools. context, &apos;interventions&apos;                      tralized 50 &#206;&#188;l 2 M NaOH 100 &#206;&#188;l Tris-malate buffer (1 represent genetic changes externally induced                       M, pH 7.5, 6% neutral red solution (0.1% neutral red 70% <br /> physiological changes occur, example,                      ethanol)). Cell fragments pelleted centrifugation <br /> progression different growth phases. types                        (approximately 640 g, 15 minutes, 4&#194;&#176;C) 100 &#206;&#188;l changes affect poly P levels strongly,                     supernatant removed. Poly P purified, enzymati- characteristics altered. mutants failed                      cally digested quantified using colorimetric assay [36].                  interactions accumulate poly P invest accumula- <br /> tion storage carbohydrates (glycogen) longer                      Screening procedure data analysis <br /> buffer expression phosphate regulated genes                             poly P measurements poly P raw data normal- <br /> indicated elevated rAPase activity). contrast, total phos-                  ized cell density (OD600) measured corre- <br /> phate content closely followed poly P levels,                    sponding culture. poly P content expressed <br /> case correlation coefficient relative poly P                  relative median cultures 96 total phosphate values (log2-transformed) 0.65.                      plate  screen) poly P content wild-type <br />                                                                                                                                                                   information <br />  <br />  <br />  <br />  ATP levels se significantly altered,                      strain BY4741 (second screens). strains compared characteristics meas-                         relative poly P content 1.5-fold higher relative poly P  <br /> ured, changes smaller, confirming, partly,                    tent 0.67-fold lower selected second general notion  viable growing cells, ATP levels                  round screening. screen, data collected remain constant [43].                                                             different time points (2 h, 4 h, 8 h, 24 h). strains <br />  <br />                                                               Genome Biology 2006, 7:R109 <br />  R109.8 Genome Biology 2006,   Volume 7, Issue 11, Article R109    Freimoser et al.                                 http://genomebiology.com/2006/7/11/R109 <br />  <br />  <br />  <br />  <br />         fulfilled following criteria selected          GmbH, Mannheim, Germany) relative light units emitted <br />         subsequent data analysis: relative poly P content                measured luminometer (Lumat LB 9507, Berthold <br />         24 h time-course varied fold;                 Technologies GmBH  KG, Bad Wildbad, Germany). <br />         time-point differed fold wild- <br />         type poly P content. Strains did grow excluded.                quantify glycogen cell pellet frozen glyco- <br />         strains, affected gene reintroduced                      gen extracted described [49]. extracted glycogen <br />         expressed plasmid, restored poly P levels (data                 digested adding 10 U alpha-amylase 12.6 U <br />         shown).                                                                  amyloglucosidase sodium acetate buffer (pH 4.8, 220 <br />                                                                                      mM) 55&#194;&#176;C 16 h. Released glucose quantified         poly P data selected strains screens                  <span id='am-46' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>D-Glucose</span> HK kit (Megazyme International Ireland , <br />         time-points combined clustered hierarchically                  Bray, Ireland). <br />         Pearson uncentered Pearson correlation distance <br />         measure, respectively, complete average linkage                    Acid phosphatase activity assayed according Huang <br />         Genesis [44]). selected strains ranked accord-                 O&apos;Shea [27] 50 &#206;&#188;l cell suspension adding 200 &#206;&#188;l p- <br />         ing strongly poly P levels affected (calculated               nitrophenyl-phosphate (20 mM). 15 minutes room <br />         sum squares log2-transformed poly P ratios                 temperature, 200 &#206;&#188;l 10% ice-cold trichloracetic acid         time-points screen).                                400 &#206;&#188;l sodium carbonate solution (2 M) added                                                                                      OD420 measured. results expressed relative         genes deleted strains altered poly P               respective values wild-type cells (BY4741). <br />         content categorized studied different computa- <br />         tional tools comparison available data sets.         genes categorized according terminology                    Additional data files <br />         using annotation given go_slim_mapping.tab file                   following additional data available online <br />         available Saccharomyces Genome Database [45].                       version paper. Additional data file 1 contains poly P <br />         Significantly overrepresented GOSlim terms (compared                      data screens. <br />         representation complete yeast genome)                                                                                                           Click <br />                                                                                          poly    data <br />                                                                                               P data <br />                                                                                      Additional <br />                                                                                            file <br />                                                                                                      offile <br />                                                                                                          the1 screens. <br />                                                                                                                     screens <br />         determined using Cytoscape BiNGO plugin (set- <br />         tings: hypergeometric test statistic, false discovery rate cor-              Acknowledgements <br />         rection multiple tests, P &#226;&#8240;&#164; 0.05 confidence limit) [46,47].              thank S Eicke, M Stadler, S Ernst, C Neupert S Clerc help         data manipulations performed Microsoft Excel                     different stages research. M Aebi acknowledged helpful                                                                                      motivating discussions, SZeeman group providing spectropho- <br />         FileMaker.                                                               tometer help glycogen quantification, W Gruissem J F&#195;&#188;t- <br />                                                                                      terer making luminometer available. M Peter M Sohrmann                                                                                      given thanks providing yeast strains. work supported         Measurement total phosphate, ATP, glycogen                            Swiss National Science Foundation (grant 3100A0-112083/1) ETH <br />         acid phosphatase activity                                                    Zurich. <br />         Detailed measurements metabolites activities <br />         performed selected strains various time-points (0 <br />         h, 2 h, 5 h, 24 h). Individual cultures time-point             References <br />         inoculated OD600 = 1 (approximately 107 cells/ml).                  1.   Kornberg  Rao NN, Ault-Rich&#195;&#169; D: Inorganic polyphosphate:                                                                                           molecule functions. Annu Rev Biochem 1999, 68:89-125. <br />         time point 1OD600 equivalent cells harvested                 2.   Neef DW, Kladde MP: Polyphosphate loss promotes SNF/SWI- <br />         used subsequent measurements. data obtained                          Gcn5-dependent mitotic induction PHO5. Mol Cell Biol <br />                                                                                           2003, 23:3788-3797. <br />         duplicate measurements.                                                 3.   Ogawa N, DeRisi J, Brown PO: New components                                                                                           phosphate accumulation polyphosphate metabolism         measurement total phosphate, cells pelleted,                          Saccharomyces cerevisiae revealed genomic expression <br />                                                                                           analysis. Mol Biol Cell 2000, 11:4309-4321. <br />         resuspended 200 &#206;&#188;l 1 MH2SO4 heated boiling                    4.   Thomas MR, O&apos;Shea EK: intracellular phosphate buffer fil- <br />         water bath 20 minutes. Released phosphate quanti-                         ters transient fluctuations extracellular phosphate levels. <br />         fied molybdate malachite green described [36].                        Proc Natl Acad Sci USA 2005, 102:9565-9570. <br />                                                                                      5.   Kuroda  Nomura K, Ohtomo R, Kato J, Ikeda T, Takiguchi N, <br />         ATP quantification neutralized cell extract                   Ohtake H, Kornberg  Role inorganic polyphosphate pro- <br />         poly P quantification prepared, ATP quantified                        moting ribosomal, protein degradation Lon protease <br />                                                                                           E. coli. Science 2001, 293:705-708. <br />         method adapted Hyswert et al. [48]. neutral-                   6.   Wang L, Fraley CD, Faridi J, Kornberg  Roth RA: Inorganic <br />         ized sample (20 &#206;&#188;l) added 80 &#206;&#188;l Tris buffer (20 mM,                       polyphosphate stimulates mammalian TOR, kinase <br />         pH8, 2 mM EDTA) 4 &#206;&#188;l phosphoenol pyruvate                                involved proliferation mammary cancer cells. Proc <br />                                                                                           Natl Acad Sci USA 2003, 100:11249-11254. <br />         added (2.5 mM, pH8, 0.125 M MgSO4, 0.312 M K2SO4).                       7.   Kulaev  Vagabov VM, Kulakovskaya TV: Biochemistry Inorganic <br />         quantification ATP, 5 &#206;&#188;l sample added 45                    Polyphopshates 2nd edition. Chichester, West Sussex: John Wiley                                                                                           Sons,  2004. <br />         &#206;&#188;l luciferase buffer (10 mM Tris-H2SO4, pH 7.4, 3.5 mM                       8.   McInerney P, Mizutani T, Shiba T: Inorganic polyphosphate inter- <br />         MgSO4). addition 50 &#206;&#188;l luciferase solution (Roche                    acts ribosomes promotes translation fidelity vitro <br />         ATP Bioluminescence Assay kit CLS II, Roche Diagnostics                           vivo. Mol Microbiol 2006, 60:438-447. <br />                                                                                      9.   Kim KS, Rao NN, Fraley CD, Kornberg  Inorganic <br />  <br />                                                                  Genome Biology 2006, 7:R109 <br />  http://genomebiology.com/2006/7/11/R109                                     Genome Biology 2006,      Volume 7, Issue 11, Article R109      Freimoser et al. R109.9 <br />  <br />  <br />  <br />  <br />       polyphosphate essential long-term survival viru-                     JF: Characterization yeast ionome: genome-wide <br />       lence factors Shigella Salmonella spp. Proc Natl Acad Sci                analysis nutrient mineral trace element homeostasis <br />       USA 2002, 99:7675-7680.                                                         Saccharomyces cerevisiae. Genome Biol 2005, 6:R77. <br /> 10.   Rashid MH, Rumbaugh K, Passador L, Davies DG, Hamood                   33.   Sobacchi C, Frattini  Orchard P, Porras O, Tezcan  Andolina M, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       comment <br />       Iglewski BH, Kornberg  Polyphosphate kinase essential                  Babul-Hirji R, Baric  Canham N, Chitayat D, et al.: mutational <br />       biofilm development, quorum sensing, virulence Pseu-                     spectrum human malignant autosomal recessive <br />       domonas aeruginosa. Proc Natl Acad Sci USA 2000, 97:9636-9641.                  osteopetrosis. Hum Mol Genet 2001, 10:1767-1773. <br /> 11.   Crooke E, Akiyama M, Rao NN, Kornberg  Genetically altered              34.   Stover EH, Borthwick KJ, Bavalia C, Eady N, Fritz DM, Rungroj N, <br />       levels inorganic polyphosphate Escherichia coli. J Biol                   Giersch AB, Morton CC, Axon PR, Akil  et al.: Novel ATP6V1B1 <br />       Chem 1994, 269:6290-6295.                                                       ATP6V0A4 mutations autosomal recessive distal <br /> 12.   Sethuraman  Rao NN, Kornberg  endopolyphosphatase                       renal tubular acidosis new evidence hearing loss. J <br />       gene: essential Saccharomyces cerevisiae. Proc Natl Acad Sci                 Med Genet 2002, 39:796-803. <br />       USA 2001, 98:8542-8547.                                                   35.   Parra KJ, Kane PM: Reversible association V1 13.   Pick U, Weiss M: Polyphosphate hydrolysis acidic vacu-                   V0 domains yeast vacuolar H+-ATPase unconven- <br />       oles response amine-induced alkaline stress halo-                  tional glucose-induced effect. Mol Cell Biol 1998, 18:7064-7074. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       reviews <br />       tolerant alga Dunaliella salina. Plant Physiol 1991, 97:1234-1240.        36.   Werner TP, Amrhein N, Freimoser FM: Novel method 14.   Rohloff P, Montalvetti  Docampo R: Acidocalcisomes                    quantification inorganic polyphosphate (iPoP) Saccharo- <br />       contractile vacuole complex involved osmoregulation                      myces cerevisiae shows dependence iPoP content       Trypanosoma cruzi. J Biol Chem 2004, 279:52270-52281.                        growth phase. Arch Microbiol 2005, 184:129-136. <br /> 15.   Weiss M, Bental M, Pick U: Hydrolysis polyphosphates               37.   Cohen  Perzov N, Nelson H, Nelson N: novel family yeast <br />       permeability changes response osmotic shocks cells                     chaperons involved distribution V-ATPase       halotolerant alga Dunaliella.            Plant Physiol 1991,             membrane proteins. J Biol Chem 1999, 274:26885-26893. <br />       97:1241-1248.                                                             38.   M&#195;&#188;ller O, Neumann H, Bayer MJ, Mayer  Role Vtc pro- <br /> 16.   Zhang H, Gomez-Garcia MR, Brown MR, Kornberg  Inorganic                       teins V-ATPase stability membrane trafficking. J Cell <br />       polyphosphate Dictyostelium discoideum : influence                        Sci 2003, 116:1107-1115. <br />       development, sporulation, predation. Proc Natl Acad Sci               39.   Odorizzi G, Cowles CR, Emr SD: AP-3 complex: coat  <br />  <br />  <br />  <br />                                                                                                                                                                       reports <br />       USA 2005, 102:2731-2735.                                                        colours. Trends Cell Biol 1998, 8:282-288. <br /> 17.   Smith SA, Mutch NJ, Baskar D, Rohloff P, Docampo R, Morrissey JH:         40.   Huizing M, Boissy  Gahl WA: Hermansky-Pudlak syndrome: <br />       Polyphosphate modulates blood coagulation fibrinolysis.                     vesicle formation yeast man. Pigment Cell Res 2002, <br />       Proc Natl Acad Sci USA 2006, 103:903-908.                                       15:405-419. <br /> 18.   Beauvoit B, Rigoulet M, Guerin B, Canioni P: Polyphosphates          41.   Rudra D, Warner JR: better measure ribosome <br />       source high energy phosphates yeast mitochondria: 31                    synthesis? Genes Dev 2004, 18:2431-2436. <br />       P NMR study. FEBS Lett 1989, 252:17-21.                                   42.   Warner JR: economics ribosome biosynthesis yeast. <br /> 19.   Harold FM, Miller  Intracellular localization inorganic                    Trends Biochem Sci 1999, 24:437-440. <br />       polyphsophate Neurospora crassa. Biochim Biophys Acta 1961,            43.   Ditzelm&#195;&#188;ller G, Wohrer W, Kubicek CP, Rohr M: <span id='am-38' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-39' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Nucleotide</span> pools <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       deposited research <br />       50:261-270.                                                                     growing, synchronized stressed cultures Saccharo- <br /> 20.   Docampo R, Souza W, Miranda K, Rohloff P, Moreno SN: Acido-                  myces cerevisiae. Arch Microbiol 1983, 135:63-67. <br />       calcisomes - conserved bacteria man. Nat Rev                      44.   Sturn  Quackenbush J, Trajanoski  Genesis: Cluster analysis       Microbiol 2005, 3:251-261.                                                      microarray data. Bioinformatics 2002, 18:207-208. <br /> 21.   Indge KJ: Polyphosphates yeast cell vacuole. J Gen                 45.   Saccharomyces Genome Database                    [ftp://genome-ftp.stan <br />       Microbiol 1968, 51:447-455.                                                     ford.edu/pub/yeast/data_download/literature_curation/] <br /> 22.   Urech K, Durr M, Boller T, Wiemken  Schwencke J: Localization           46.   Maere S, Heymans K, Kuiper M: BiNGO: Cytoscape plugin       polyphosphate vacuoles Saccharomyces cerevisiae.                       assess overrepresentation Gene Ontology categories       Arch Microbiol 1978, 116:275-278.                                               biological networks. Bioinformatics 2005, 21:3448-3449. <br /> 23.   Winzeler EA, Shoemaker DD, Astromoff  Liang H, Anderson K,              47.   Shannon P, Markiel  Ozier O, Baliga NS, Wang JT, Ramage D, <span id='am-64' about='obo:IMR_0001693' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-65' property="rdfs:label" content="amin" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0001693" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="CHEBI:32952" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="MeSH:D000588" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="obo:IAO_0000115" content="A group of compounds derived from ammonia by substituting organic radicals for the hydrogens. (From Grant &amp; Hackh&apos;s Chemical Dictionary, 5th ed)" datatype="xsd:string"></span>Amin</span> <br />       Andre B, Bangham R, Benito R, Boeke JD, Bussey H, et al.: Func-                 N, Schwikowski B, Ideker T: Cytoscape: software environment <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       refereed research <br />       tional characterization S. cerevisiae genome gene                     integrated models biomolecular interaction networks. <br />       deletion parallel analysis. Science 1999, 285:901-906.                      Genome Res 2003, 13:2498-2504. <br /> 24.   Darsow T, Burd CG, Emr SD: Acidic di-leucine motif essential              48.   Hyswert DW, Morrison NM: Studies ATP, <span id='am-1' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ADP</span>, AMP <br />       AP-3-dependent sorting restriction functional                    concentrations yeast beer. J Soc Brew Chem 1977, <br />       specificity Vam3p vacuolar t-SNARE. J Cell Biol 1998,                    35:160-167. <br />       142:913-922.                                                              49.   Wang  Wilson WA, Fujino MA, Roach PJ: Antagonistic controls <br /> 25.   Wilson WA, Wang  Roach PJ: Systematic identification                   autophagy glycogen accumulation Snf1p, yeast <br />       genes affecting glycogen storage yeast Sac <br /> </body></html>