<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, 2003, p. 2961&#226;&#8364;&#8220;2966                                                                                Vol. 185,  9 <br /> 0021-9193/03/$08.00&#226;&#171;&#185;0 DOI: 10.1128/JB.185.9.2961&#226;&#8364;&#8220;2966.2003 <br /> Copyright &#194;&#169; 2003, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />                                                             NOTES <br />  <br />               Essentiality clpX, clpP, clpL, clpC, clpE,                             Streptococcus pneumoniae R6 <br />                           Gregory T. Robertson,1&#226;&#8364;&#160; Wai-Leung Ng,1,2 Raymond Gilmour,1 <br />                                           Malcolm E. Winkler1,2* <br />                          Lilly Research Laboratories, Eli Lilly Company, Indianapolis, Indiana 46285,1 <br />                             Department Biology, Indiana University, Bloomington, Indiana 474052 <br />                                          Received 16 September 2002/Accepted 19 February 2003 <br />  <br />             using regulated promoter clpX Streptococcus pneumoniae R6 essential, clpP, <br />          clpL, clpC, clpE disrupted. essentiality clpX initially missed duplication          rearrangement region chromosome containing clpX. Depletion ClpX resulted rapid loss          viability overt changes cell morphology. Essentiality clpX, clpP, reported <br />          previously. <br />  <br />  <br />    reported previously clpP-mediated proteolysis plays              rangements low frequency. clpX required temper- important role pneumococcal stress responses,                 atures tested, depletion ClpX resulted rapid loss development competence, virulence (22). ClpP pro-                cellular viability. <br /> teases consist ATPase regulatory subunit                        Duplications rearrangements clpX region Hsp100 family chaperones ClpP protease subunit,                  S. pneumoniae R6 chromosome. putative ClpX polypep- contains serine protease active site (18; reviewed               tide S. pneumoniae R6 shows high identity similarity <br /> references 8, 9, 11, 21, 24). Streptococcus pneumoniae                  entire length ClpX subunits defined bac- <br /> contains putative orthologs ATPase specificity factors              teria (e.g., 58 76, 60 76, 65 81, 75 87% <br /> (ClpC, ClpE, ClpL, ClpX) (14, 28) single clpP gene                identity similarity, respectively, ClpX Escherichia coli monocistronic operon (22). clpP mutants showed                 K-12, Caulobacter crescentus, Bacillus subtilis, Lactococcus <br /> temperature sensitivity growth 40&#194;&#176;C Trypticase soy                lactis, respectively, GenBank comparisons). intercis- <br /> agar blood (TSA-BA) plates appreciably im-                     tronic regions separate clpX surrounding genes paired growth 37&#194;&#176;C (22). observation                minimal, obvious factor-independent tran- <br /> dependently Chastanet coworkers, showed                  scription terminators region (Fig. 1A).  clpX clpE mutants temperature sensitive, suggesting role              multifunctional operon extends dpr ClpE ClpEP temperature tolerance (5). Roles                   hypothetical gene spr1422 (Fig. 1A). open reading <br /> ClpL ClpX S. pneumoniae discovered,                frame immediately downstream clpX designated spr1426 function ClpC specificity factor present          recently shown essential encode <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span> <br /> question (4, 5, 22). <br />                                                                             binding protein (29). dfr gene upstream clpX (Fig. <br />    described construction &#226;&#338;&#172;clpX::ermAM <br />                                                                             1A) encodes dihydrofolate reductase, required mutation (erythromycin resistance; Ermr) S. pneumoniae R6 <br />                                                                             growth number bacterial species (12). known <br /> (strain EL873; Table 1) mutant did                                                                             gene upstream clpX dpr, implicated temperature-sensitive growth like &#226;&#338;&#172;clpP mutant (22). <br />                                                                             <span id='am-35' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O2</span> resistance (33) critical growth S.  reported  reasons did                                                                             pneumoniae, produces H2O2 aerobic growth  <br /> understand, &#226;&#338;&#172;clpX::ermAM amplicon <br />                                                                             der laboratory conditions (26). S. pneumoniae D39 (Table 1) transformation (22) transfer &#226;&#338;&#172;clpX::ermAM mutation R6                    Previously, confirmed presence &#226;&#338;&#172;clpX::ermAM <br /> D39 using chromosomal DNA isolated strain                      allele EL873 PCR primers located immediately <br /> EL873 (22). analyses problem shown                outside clpX reading frame (22). thought clpX actually essential S. pneumoniae R6 clpX            sufficient resolution gels resolve &#226;&#338;&#172;clpX:: <br /> region chromosome forms duplications rear-                 ermAM construct clpX&#226;&#171;&#185; gene, later analyses fur- <br />                                                                             ther digestion PCR products restriction enzymes <br />                                                                             revealed copies &#226;&#338;&#172;clpX::ermAM clpX&#226;&#171;&#185; <br />   * Corresponding author. Mailing address: Department Biology,           genes present EL873 (Fig. 2, lane 5).  EL873 Indiana University, Jordan Hall 142, Bloomington, 47405. Phone: <br />                                                                             merodiploid likely arose recombination (317) 508-7812. Fax: (812) 855-6705. E-mail: mwinkler@bio.indiana <br /> .edu.                                                                       &#226;&#338;&#172;clpX::ermAM allele copy duplication clpX <br />   &#226;&#8364;&#160; Present address: Cumbre, , Dallas, TX 75235.                        region occurred spontaneously chromosome S. <br />  <br />                                                                      2961 <br />  2962       NOTES                                                                                                                                    J. BACTERIOL. <br />  <br />                                                                  TABLE 1. Bacterial strainsa <br />  Strain                 Genotype phenotype                                                   Description                                      Source reference <br />  <br /> EL59         S. pneumoniae R6, avirulent unen-                  Derived D39 isolate                                                           Tomasz (14) <br />                 capsulated parent strain <br /> EL161        S. pneumoniae D39, virulent encap-                 Subclone original clinical isolate                                             J. Yother (25) <br />                 sulated type 2 parent strain <br /> EL539        EL59 &#226;&#338;&#172;clpP::aad9 (Spcr)                            EL59 transformed linear &#226;&#338;&#172;clpP::aad9 amplicon                                 22 <br /> EL854        EL59 &#226;&#338;&#172;clpC::ermAM (Ermr)                           EL59 transformed linear &#226;&#338;&#172;clpC::ermAM amplicon                                22 <br /> EL873        EL59 &#226;&#338;&#172;clpX::ermAM clpX&#226;&#171;&#185; (Ermr)                     EL59 containing previously undetected merodiploid &#226;&#338;&#172;clpX::ermAM                 22 <br />                                                                  clpX&#226;&#171;&#185; <br /> EL1039       EL161 &#226;&#338;&#172;clpX::ermAM clpX&#226;&#171;&#185; (Ermr)                    EL161 transformed genomic DNA EL873;                                    22 <br />                                                                  contains previously undetected merodiploid &#226;&#338;&#172;clpX::ermAM <br />                                                                  clpX&#226;&#171;&#185; <br /> EL1082       EL59 &#226;&#338;&#172;clpL::ermAM (Ermr)                           EL59 transformed linear &#226;&#338;&#172;clpL::ermAM amplicon                                study <br /> EL1259       EL59 &#226;&#338;&#172;clpE::aad9 (Spcr)                            EL59 transformed linear &#226;&#338;&#172;clpE::aad9 amplicon                                 study <br /> EL1383       EL59 &#226;&#338;&#172;bgaA::(Pc::tetL::T1T2::PfcsK-clpX&#226;&#171;&#185;)          EL59 transformed linear &#226;&#338;&#172;bgaA::(Pc::tetL::T1T2::PfcsK-clpX&#226;&#171;&#185;)                 study <br />                (Tetr)                                            amplicon <br /> EL1387       EL1383 &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM (Ermr Tetr)                    EL1383 transformed linear &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM amplicon                              study <br /> EL1392       &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM clpX&#226;&#171;&#185; (Ermr)                          EL59 transformed &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM, resulting partially                      study <br />                                                                  characterized duplication &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM clpX&#226;&#171;&#185; <br />       Strains constructed transformation indicated recipients linear double-stranded synthetic PCR amplicon DNA. Following single colony isolation, constructions confirmed PCR analysis chromosomal DNA using flanking internal primers additional restriction digestion amplicons (details <br /> available http://www.streppneumoniae.com). <br />  <br />  <br /> pneumoniae R6 (e.g., reference 2). conclusion                           duced synthetic competence stimulatory <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> 1  ref- <br /> fully supported observation EL873 reverted rapidly                       erence 13). frequency transformation amplicon sensitivity erythromycin antibiotic selection                         (&#226;&#172;?2.0 &#226;&#171;&#187; 10&#226;&#171;&#186;6 &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermA transformants) removed cultures (Table 2), reversion ac-                          level detection compared control genomic DNA <br /> companied restoration wild-type clpX&#226;&#171;&#185; chromo-                           imparting resistance novobiocin (7.1 &#226;&#171;&#187; 10&#226;&#171;&#186;4 Norr transfor- (Fig. 2, lanes 5 6).                                                        mants) R6 parent strain EL59. frequency transfor- <br />    potential polarity clpX insertions                       mation calculated based number recovered  <br /> expression essential downstream spr1426 gene (Fig. 1A),                       tibiotic-resistant colonies 1 ml transformation mixture attempted cross &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM amplicon depicted                             divided total CFU milliliter.  Ermr Fig. 1A, clpX reading frame replaced exactly                        colony did appear (EL1392; Table 1), contained start stop codon ermAM reading frame,                        kind rearrangement clpX region. Analysis clpX chromosome S. pneumoniae R6 transformation                              region EL1392 PCR showed presence anom- <br />  <br />  <br />  <br />  <br />    FIG. 1. Organization clpX operon S. pneumoniae R6 construction nonpolar replacement clpX reading frame ermAM (Ermr)   strain containing ectopic copy clpX&#226;&#171;&#185; control fucose-inducible PfcsK promoter located bgaA <br /> locus (B)  text Table 1 details). Genes predicted directions transcription depicted arrows (drawn scale). lytC trxB genes likely mark boundaries clpX region based large intergenic region direction transcription, respectively. shaded regions depict flanking genes surrounding clpX, hatched region corresponds ermAM open reading frame. Pc-tetL imparts <br /> constitutive resistance 0.25 &#226;?&#174;g tetracycline ml used selection marker. T1T2 indicates transcription terminators protect <br /> regulated expression PfcsK. <br />  VOL. 185, 2003                                                                                                                         NOTES         2963 <br />  <br />  <br />  <br />  <br />    FIG. 2. ScaI restriction maps clpX regions genomic DNA S. pneumoniae strains EL59, EL873, EL1392 amplified PCR oligonucleotide primer pair clpX clpX-c (22). Lanes 1 7, Gibco 1-kb Plus standard ladder; lane 2, wild-type clpX&#226;&#171;&#185; locus parent <br /> strain EL59; lane 3, duplicated spr1426 locus EL1392, contains copy clpX&#226;&#171;&#185; clpX&#226;&#172;?&#226;&#172;&#381;ermAM cassette; lane 4, Erms revertant EL1392 containing wild-type clpX&#226;&#171;&#185; region; lane 5, clpX::ermAM duplicated clpX&#226;&#171;&#185; locus EL873; lane 6, Erms revertant EL873 <br /> containing wild-type clpX&#226;&#171;&#185; region. Shaded arrows, clpX reading frame; hatched arrows, ermAM reading frame; open arrows, genes surrounding <br /> clpX (Fig. 1); black dots, &#226;&#172;&#402;0.9 kb unknown DNA present EL1392 clpX&#226;&#171;&#185; region. ScaI restriction sites indicated letter S horizontal line. break denotes unknown chromosomal DNA EL873 merodiploid. Predicted restriction fragment sizes indicated DNA standards kilobase pairs. <br />  <br />  <br /> alous &#226;&#172;&#402;5-kb amplicon contained ermAM cassette,                    peared significantly lower frequency &#226;&#177;&#8226;10&#226;&#171;&#186;6.   <br /> clpX&#226;&#171;&#185;, duplication flanking downstream gene spr1426,              mation &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM clpX&#226;&#171;&#185; duplications occurred uncharacterized region &#226;&#172;&#402;0.9 kb containing DNA                 low detectable frequency comparable essen- <br /> unknown origin (Fig. 2, lane 3). arrangement probably                 tial genes, &#226;&#338;&#172;gyrB::ermAM gyrB&#226;&#171;&#185;, chro- <br /> arose aberrant recombination event clpX region.               mosome S. pneumoniae R6. <br /> Unlike EL873, reversion EL1392 erythromycin sensitivity <br /> occurred low frequency antibiotic selection removed cultures (Table 2),  reversion accompanied restoration wild-type clpX&#226;&#171;&#185; chromo-                    TABLE 2. Reversion frequency S. pneumoniae mutants (Fig. 2, lanes 3 4).                                                  containing duplications clpX region chromosome <br />    determined frequency &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM                                                                                 (%) Ermr clpX&#226;&#171;&#185; merodiploids arose transformations compared                            Selection                            Erms isolates following <br />                                                                                                                                    spontaneous resolution unlinked essential gyrB gene (29). independent                  Strain              CFU/mlb        colonies <br />                                                                                                                                        duplications <br />                                                                                          growtha                     screened <br /> exponential cultures EL59 (R6 parent strain) brain <br /> heart infusion (BHI) medium diluted 1:20 1 ml                                                                             Ermr           Erms <br />  <br /> BHI medium containing 10% heat-inactivated horse serum                     EL873                 4.7 &#226;&#171;&#187; 106         31          4 (13)        27 (87) <br /> (Sigma), 10 mM glucose, 100 ng competence stimu-                                   ERY         4.9 &#226;&#171;&#187; 105         31         31 (100)        0 (0) <br /> latory peptide 1 cell density &#226;&#172;&#402;7.0 &#226;&#171;&#187; 105 CFU/ml               EL1392                2.0 &#226;&#171;&#187; 107        130        129 (99.2)       1 (0.8) <br />                                                                                           ERY         7.9 &#226;&#171;&#187; 106         61         61 (100)        0 (0) <br /> (22). Competent cell suspensions mixed nonpolar &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM amplicon, &#226;&#338;&#172;gyrB::ermAM dele-                                                                                                        Bacteria grown statically BHI containing lacking 0.3 &#226;?&#174;g eryth- <br />                                                                            romycin (ERY) ml &#226;&#172;&#8225;7 h 37&#194;&#176;C 5% <span id='am-14' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-15' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span>CO2</span>, serially diluted 0.9% <br /> tion-insertion amplicon, control genomic DNA bearing                  (wt/vol) NaCl, spread TSA-BA plates containing lacking 0.3 &#226;?&#174;g point mutation confers resistance novobiocin                      erythromycin ml, respectively. Random colonies screened erythro- <br /> (Novr) (16). Novr transformants recovered                  mycin resistance (Ermr) sensitivity (Erms) patched TSA-BA <br />                                                                            plates containing lacking 0.3 &#226;?&#174;g erythromycin ml. <br /> transformations average frequency &#226;&#172;&#402;6 &#226;&#171;&#187; 10&#226;&#171;&#186;4,                        b <br />                                                                                Number CFU milliliter culture cells recovered &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM &#226;&#338;&#172;gyrB::ermAM transformants ap-                     time plating. <br />  2964      NOTES                                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Impaired growth S. pneumoniae R6 strain EL1387 containing regulated replacement clpX (&#226;&#338;&#172;clpX PfcsK::clpX&#226;&#171;&#185;) medium <br /> lacking fucose.   EL59 (clpX&#226;&#171;&#185; parent), EL1383 (regulated merodiploid clpX&#226;&#171;&#185; PfcsK::clpX&#226;&#171;&#185;), EL1387 (regulated replacement &#226;&#338;&#172;clpX PfcsK:: <br /> clpX&#226;&#171;&#185;) streaked TSA-BA plates containing 0.25% (wt/vol) fucose (inducing) lacking additional fucose (noninducing). Plates photographed 24 h incubation 37&#194;&#176;C atmosphere 5% CO2. (B) EL1387 (regulated replacement &#226;&#338;&#172;clpX PfcsK::clpX&#226;&#171;&#185;) grown <br /> statically overnight BHI broth containing 0.1% (wt/vol) fucose 37&#194;&#176;C 5% CO2. Cultures diluted 100-fold fresh BHI containing lacking 0.1% (wt/vol) fucose, static incubation continued 37&#194;&#176;C 5% CO2. Filled circles squares represent optical densities viable cell counts, respectively, culture containing 0.1% (wt/vol) fucose. Open circles squares represent optical densities viable cell counts, respectively, culture lacking fucose (0.001% [wt/vol]) carryover fucose starting culture). Results representative independent experiments. OD620 (1.4 cm), optical density 620 nm tube 1.4-cm diameter. <br />  <br />  <br />  <br />    clpX essential S. pneumoniae R6. fact             construction, yield strain EL1383 (Table 1). Tetr transfor- obtain simple transformant replaced clpX                     mants selected TSA-BA plates containing 0.25% (wt/ <br /> &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM suggested clpX essential. dem-                      vol) <span id='am-30' about='obo:IMR_0200303' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:id" content="IMR:0200303" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C01019" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="L-Fucose" datatype="xsd:string"></span>L-fucose</span> 37&#194;&#176;C atmosphere 5% CO2. loca- <br /> onstrate essentiality, needed construct merodiploid                  tion construction confirmed PCR (data clpX&#226;&#171;&#185; transcription control regulatable              shown). <br /> promoter. catabolic promoters shown                      &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermAM amplicon crossed regulated sugars S. pneumoniae related                 chromosome EL1383, Ermr mutants selected species, including galactose, maltose, fucose uti-             TSA-BA plates containing inducer 0.25% (wt/vol) L-fucose. <br /> lization (1, 3, 3a, 20; P. F. Chan, K.  Ingraham, C. Y.  M.             frequency recovery Ermr transformants  <br /> Lonetto, M. Rosenberg, D. J. Holmes, M. Zalacain, Abstr.                 creased dramatically (2.1 &#226;&#171;&#187; 10&#226;&#171;&#186;3 &#226;&#338;&#172;clpX&#226;&#172;?&#226;&#172;&#381;ermA transfor- <br /> 100th Gen. Meet.  Soc. Microbiol., abstr. H-97, p. 370,                   mants compared &#226;&#172;?2.0 &#226;&#171;&#187; 10&#226;&#171;&#186;6 EL59  ) 2000). chose promoter upstream fucose ki-                    comparable Novr control (4.5 &#226;&#171;&#187; 10&#226;&#171;&#186;4). nase gene (fcsK spr1973) (PfcsK) (14), previously             resulting regulated replacement EL1387 (Table 1) grew like reported provide regulation S. pneumoniae (3a; W.                 EL59 (clpX&#226;&#171;&#185; parent) EL1383 [&#226;&#338;&#172;bgaA::(Pc-tetL::T1T2::Pf- <br />                                                                                       &#226;&#171;&#185;       &#226;&#171;&#185; <br /> Bae, P. Chan, L. Palmer, V. Clausen, J. Throup, M.                           csK-clpX ) clpX     merodiploid] plates containing L-fucose <br /> Noordewier, K. Koretke, D. Lunsford,  Bryant, K. Ingraham,                 completely failed grow L-fucose omitted D. Holmes, M. Rosenberg, M. Burnham, Abstr. 101st                        plates (Fig. 3A). lack growth EL1387 absence <br /> Gen. Meet.  Soc. Microbiol., abstr. B-381, p. 126, 2001).                 L-fucose consistent polar effect expression constructed cassette shown Fig. 1B                    essential downstream spr1426 gene, expres- <br /> promoter region clpX&#226;&#171;&#185; replaced PfcsK, pro-                sion clpX&#226;&#171;&#185; gene control L-fucose tected transcription terminators T1T2 connected                  &#226;&#338;&#172;bgaA locus.  expression clpX&#226;&#171;&#185; essential tetL (tetracycline resistance [Tetr]) gene driven consti-             growth S. pneumoniae R6. <br /> tutive promoter (Pc) (16). Pc-tetL::T1T2::PfcsK-clpX&#226;&#171;&#185; cas-                  clpX essential clpP  formally <br /> sette crossed bgaA locus, encodes dispens-              possible essentiality clpX depend func- <br /> able &#226;?&#164;-galactosidase (34) partly deleted                     tional clpP&#226;&#171;&#185;, clpC&#226;&#171;&#185;, clpE&#226;&#171;&#185;, clpL&#226;&#171;&#185; gene. example, ClpX <br />  VOL. 185, 2003                                                                                                                         NOTES          2965 <br />  <br />  interact ClpP protease subunit                  Implications. finding clpX essential, clpP ATPase specificity subunits limit ClpP proteolysis                unusual, ClpX usually functions regulatory <br /> activity. test notion, transformed EL1387 grown             subunit ClpXP protease (10, 32). constructed fucose-containing medium clpP::kan::rpsL&#226;&#171;&#185;, clpC::kan::             confirmed presence single-copy insertion-deletion mu- <br /> rpsL&#226;&#171;&#185;, clpE::kan::rpsL&#226;&#171;&#185;, clpL::kan::rpsL&#226;&#171;&#185; Janus cassette               tations inactivate clpP, clpC, clpL, clpE (strain EL539, <br /> amplicon (27). transformation, recovered hun-                 EL854, EL1082, EL1259, respectively [Table 1]). dreds transformants TSA-BA plates containing fucose                 essential growth S. pneumoniae R6 37&#194;&#176;C (data kanamycin transformants TSA-BA plates                   shown), clpP clpE imparted temperature sensi- <br /> taining kanamycin lacking fucose (data shown).               tivity BHI liquid medium reported previously (5, 22; data <br /> clpX essentiality did depend functional clpP&#226;&#171;&#185;, clpC&#226;&#171;&#185;,            shown). Curiously, clpE mutant temperature <br /> clpE&#226;&#171;&#185;, clpL&#226;&#171;&#185; gene.                                                     sensitive TSA-BA plates 40&#194;&#176;C (data shown),    Phenotypes clpX underexpression. examined pheno-                 clpP mutant (22). cyanobacterium Synechococcus <br /> types bacteria depleted ClpX learn               sp. strain PCC 7942 genes encoding ClpP isoenzymes <br /> basis essentiality clpX. Repetition experiment          single copy ClpX. Viable deletion-insertion mutants shown Fig. 3A 30&#194;&#176;C showed clpX expression                  Synechococcus obtained clpPI clpPII, sug- <br /> essential lower temperature 37&#194;&#176;C bac-            gesting clpPIII clpX indispensable growth (6, <br /> teria spread TSA-BA plates (data shown).                23). ClpX ClpP essential C. crescentus (7, 15), 0.05% (wt/vol) fucose minimum concentration                  chaperone functions ascribed ClpX supported growth regulated replacement mutant                     role ClpXP proteolysis (17, 24, 30). recent report shows <br /> EL1387 BHI broth 37&#194;&#176;C;  concentration               E. coli K-12 lacking functional ClpX ClpP lost viability <br /> fucose led lower growth yields EL1387 reg-              rapidly did wild-type parent strains extended <br /> ulated merodiploid EL1383 (final culture optical density 620           stationary phase (31). <br /> nm, 0.17 compared 0.47, respectively). determined                 results reported indicate ClpX play es- viability cells regulated replacement mutant                sential roles independent ClpP S. pneumoniae R6 (Fig. <br />                                                                           3A; . Furthermore, depletion ClpX leads EL1387 following downshift low fucose concentration <br />                                                                           rapid cell death overtly affecting cell morphology (Fig. <br /> (Fig. 3B). Addition 0.1% (wt/vol) fucose supported                                                                           3B; data shown). Unlike clpP, clpC, clpE, clpL, clpX <br /> growth viable cells BHI broth 37&#194;&#176;C (Fig. 3B, closed <br />                                                                           appears member heat shock regulon symbols). contrast, cultures EL1387 downshifted                                                                           S. pneumoniae (22), microarray analyses indicated 0.001% (wt/vol) fucose prematurely stopped increasing op- <br />                                                                           clpX transcript significantly changed tical density concomitantly rapidly lost cellular viability <br />                                                                           phase growth chemically defined medium suble- <br /> (Fig. 3B). plateau optical density suggested EL1387 <br />                                                                           thal concentrations antibiotic triclosan novobiocin undergoing rapid autolysis. Consistent obser- <br />                                                                           common translation inhibitors, macro- <br /> vation, examination EL1387 phase-contrast microscopy <br />                                                                           lides erythromycin roxithromycin, caused marginal various times fucose downshift did reveal signifi- <br />                                                                           1.7-fold induction clpX transcript amounts (19; data changes cellular morphology compared EL59              shown). Genetic physiological experiments progress EL1383 controls (data shown). 360 min fucose             learn essential function(s) ClpX understand <br /> downshift, &#226;&#172;?1% EL1387 cells irregularly shaped                organization possible regulation clpX operon large, especially end short chains, compared           (Fig. 1) S. pneumoniae. EL59 EL1383 controls (data shown);   point significant death occurring                  thank colleagues listed Table 1 bacterial strains; D.  <br /> EL1387 cultures.                                                          Morrison Pc-Tetr cassette used studies; K. Kaz- <br />    rapid cell death EL1387 ClpX depletion (Fig.               mierczak, D. Mytelka T. Nicas, G. Zhao comments 3B) analysis global transcription patterns microar-            manuscript. <br />                                                                             work supported resources provided Lilly Research <br /> rays (19, 22) problematic. Instead tested potential            Laboratories Indiana University Bloomington. Gregory T. Rob- <br /> overexpression ClpX regulated merodiploid EL1383                ertson supported Lilly Postdoctoral Fellowship. <br /> changed transcription pattern. compared microarray <br /> patterns RNA isolated cultures EL1383 grown                                              REFERENCES <br /> optical density 620 nm &#226;&#172;&#402;0.2 BHI 37&#194;&#176;C lacking               1. Ajdic, D., J. J. Ferretti. 1998. Transcriptional regulation Strepto- <br /> induced 60 min 0.2% (wt/vol) fucose. basis                coccus mutans gal operon GalR repressor. J. Bacteriol. 180:5727&#226;&#8364;&#8220; <br />                                                                              5732. independent experiments, unable detect sig-                2. Anderson, R. P., J. R. Roth. 1977. Tandem genetic duplications phage <br /> nificant differences transcript patterns EL1383 cultured               bacteria. Annu. Rev. Microbiol. 31:473&#226;&#8364;&#8220;505. fucose, exception clpX tran-             3. Bettenbrock, K., C.  Alpert. 1998. gal genes Leloir pathway <br />                                                                              Lactobacillus casei 64H. Appl. Environ. Microbiol. 64:2013&#226;&#8364;&#8220;2019. <br /> script (increased &#226;&#172;&#402;1.7-fold) genes spr1963                3a.Chan, P. F., K. M. O&#226;&#8364;&#8482;Dwyer, L. M. Palmer, J. D. Ambrad, K.  Ingraham, <br /> spr1973, comprise fucose regulon (fcsK, fucA, fucU,                C.  M.  Lonetto, S. Biswas, M. Rosenberg, D. J. Holmes, M. <br />                                                                              Zalacain. 2003. Characterization novel fucose-regulated promoter <br /> PTS-EII [spr1970], PTS-EII [spr1969], PTS-EII [spr1968], PTS-                (PfcsK) suitable gene essentiality antibacterial mode action studies <br /> EII [spr1967], spr1966, spr1965, fucI, adh2 increased &#226;&#172;&#402;23-,              Streptococcus pneumoniae. J. Bacteriol. 185:2051&#226;&#8364;&#8220;2058. <br /> 16-, 15-, 11-, 39-, 25-, 10-, 10-, 36-, 28-, 3-fold, respectively).   4. Charpentier, E., R. Novak, E. Tuomanen. 2000. Regulation growth <br />                                                                              inhibition high temperature, autolysis, transformation adherence possible overexpression clpX sufficient             Streptococcus pneumoniae clpC. Mol. Microbiol. 37:717&#226;&#8364;&#8220;726. experiments elicit phenotype.                                  5. Chastanet, , M. Prudhomme, J. P. Claverys, T. Msadek. 2001. Reg- <br />  2966         NOTES                                                                                                                                         J. BACTERIOL. <br />  <br />       ulation Streptococcus pneumoniae clp genes role competence        21. Porankiewicz, J., J. Wang,  K. Clarke. 1999. New insights       development stress survival. J. Bacteriol. 183:7295&#226;&#8364;&#8220;7307.                         <span id='am-19' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>-dependent Clp protease: Escherichia coli  Mol. Microbiol. <br />  6.   Clarke,  K., J. Schelin, J. Porankiewicz. 1998. Inactivation clpP1       32:449&#226;&#8364;&#8220;458. <br />       gene proteolytic subunit ATP-dependent Clp protease         22. Robertson, G. T., W. L. Ng, J. Foley, R. Gilmour, M. E. Winkler. 2002. <br />       cyanobacterium Synechococcus limits growth light acclimation. Plant               Global transcriptional analysis clpP mutations type 2 Streptococcus <br />       Mol. Biol. 37:791&#226;&#8364;&#8220;801.                                                                pneumoniae effects physiology virulence. J. Bacteriol. 184: <br />  7.   Fuchs, T., P. Wiget, M. Osteras, U. Jenal. 2001. Precise amounts             3508&#226;&#8364;&#8220;3520. <br />       novel member phosphotransferase superfamily essential growth         23. Schelin, J., F. Lindmark,  K. Clarke. 2002. clpP multigene family <br />       normal morphology Caulobacter crescentus. Mol. Microbiol. 39:679&#226;&#8364;&#8220;              ATP-dependent Clp protease cyanobacterium Synechococcus. <br />       692.                                                                                  Microbiology 148:2255&#226;&#8364;&#8220;2265. <br />  8.   Gottesman, S. 1996. Proteases targets Escherichia coli. Annu.        24. Schirmer, E. C., J. R. Glover, M.  Singer, S. Lindquist. 1996. HSP100/ <br />       Rev. Genet. 30:465&#226;&#8364;&#8220;506.                                                               Clp proteins: common mechanism explains diverse functions. Trends Bio- <br />  9.   Gottesman, S. 1999. Regulation proteolysis: developmental switches.                chem. Sci. 21:289&#226;&#8364;&#8220;296. <br />       Curr. Opin. Microbiol. 2:142&#226;&#8364;&#8220;147.                                                 25. Smith, M. D., W. R. Guild. 1979. plasmid Streptococcus pneumo- <br /> 10.   Gottesman, S., W. P. Clark, V. Crecy-Lagard, M. R. Maurizi. 1993.              niae. J. Bacteriol. 137:735&#226;&#8364;&#8220;739. <br />       ClpX, alternative subunit ATP-dependent Clp protease Esch-          26. Spellerberg, B., D. R. Cundell, J. Sandros, B. J. Pearce,  Idanpaan-Heik- <br />       erichia coli. Sequence vivo activities. J. Biol. Chem. 268:22618&#226;&#8364;&#8220;22626.        kila, C. Rosenow, H. R. Masure. 1996. Pyruvate oxidase, determi- <br /> 11.   Gottesman, S., M. R. Maurizi. 1992. Regulation proteolysis: energy-            nant virulence Streptococcus pneumoniae. Mol. Microbiol. 19:803&#226;&#8364;&#8220;813. <br />       dependent proteases targets. Microbiol. Rev. 56:592&#226;&#8364;&#8220;621. <br />                                                                                         27. Sung, C. K., H. Li, J. P. Claverys, D.  Morrison. 2001. rpsL <br /> 12.   Hartman, P. G. 1993. Molecular aspects mechanism action dihy- <br />                                                                                             cassette, Janus, gene replacement negative selection Strepto- <br />       drofolate reductase inhibitors. J. Chemother. 5:369&#226;&#8364;&#8220;376. <br />                                                                                             coccus pneumoniae. Appl. Environ. Microbiol. 67:5190&#226;&#8364;&#8220;5196. <br /> 13.   Havarstein, L. S., P. Gaustad,  F. Nes, D.  Morrison. 1996. Identi- <br />                                                                                         28. Tettelin, H., K. E. Nelson,  T. Paulsen, J.  Eisen, T. D. Read, S. Peterson, <br />       fication streptococcal competence-pheromone receptor. Mol. Micro- <br />                                                                                             J. Heidelberg, R. T. DeBoy, D. H. Haft, R. J. Dodson,  S. Durkin, M. <br />       biol. 21:863&#226;&#8364;&#8220;869. <br />                                                                                             Gwinn, J. F. Kolonay, W. C. Nelson, J. D. Peterson, L.  Umayam, O. White, <br /> 14.   Hoskins, J., W.  <br /> </body></html>