<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2000 Genetics Society America <br />  <br />  <br />  <br />      Proteasome Mutants, pre4-2 ump1-2, Suppress Essential Function <br />               Mitochondrial RNase P Function                       Saccharomyces cerevisiae Gene RPM2 <br />  <br />                                   Mallory S. Lutz, Steven R. Ellis Nancy C. Martin <br />                     Department Biochemistry Molecular Biology, University Louisville, Louisville, Kentucky 40292 <br />                                                      Manuscript received August 25, 1999 <br />                                                   Accepted publication November 2, 1999 <br />  <br />  <br />                                                               ABSTRACT <br />                 Saccharomyces cerevisiae nuclear gene RPM2 encodes component mitochondrial tRNA- <br />              processing enzyme RNase P. Cells grown fermentable carbon sources require mitochondrial <br />              tRNA processing activity, require RPM2, indicating additional function Rpm2 protein. <br />              RPM2-null cells arrest 25 generations fermentable media. Spontaneous mutations suppress <br />              arrest occur frequency &#226;&#181;&#8216;9 &#226;&#171;&#187; 10&#226;&#171;&#186;6. resultant mutants grow nonfermentable carbon <br />              sources. identified loci responsible suppression, encode proteins influence <br />              proteasome function assembly. PRE4 essential gene encoding &#226;?&#164;-7 subunit 20S proteasome <br />              core. Val Phe substitution highly conserved region Pre4p disrupts proteasome function <br />              suppresses growth arrest RPM2-null cells fermentable media. locus, UMP1, encodes <br />              chaperone involved 20S proteasome assembly. nonsense mutation UMP1 disrupts proteasome <br />              function suppresses &#226;&#338;&#172;rpm2 growth arrest. RPM2 wild-type background, pre4-2 ump1-2 strains <br />              fail grow restrictive temperatures nonfermentable carbon sources. data link proteasome <br />              activity Rpm2p mitochondrial function. <br />  <br />  <br />  <br /> M      ITOCHONDRIA vital organelles eukaryo- <br />         tic cells. addition role respiration oxidative phosphorylation, mitochondria                                                                           organelle formation maintenance disrupt <br />                                                                           growth carbon sources  Pinto et al. 1999). <br />                                                                             Yeast mitochondrial RNase P ribonucleoprotein <br /> site diverse cellular functions heme biosyn-                  complex required removing 5&#226;&#172;&#732; leader sequences <br /> thesis, metabolite transport, <span id='am-36' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-37' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> biosynthesis,                    mitochondrial tRNA precursors (Morales et al. <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span> catabolism (Attardi Schatz 1988; Pon                        1992; Dang Martin 1993). RNA component Schatz 1991). Mitochondrial biogenesis func-                      mitochondrial RNase P (Rpm1r) encoded tion require input nuclear mitochondrial                    mitochondrial genome (Shu et al. 1991). protein <br /> genomes. yeast Saccharomyces cerevisiae, mitochon-                 component mitochondrial RNase P (Rpm2p)  <br /> drial DNA codes proteins required oxidative                       coded nucleus. Biochemical genetic analyses <br /> phosphorylation, electron transport, mitochondrial pro-                   established Rpm2p subunit mitochondrial RNase <br /> tein synthesis, mRNA processing. Ribosomal RNA,                       P (Morales et al. 1992; Dang Martin 1993). <br /> tRNA genes, RNase P RNA gene necessary                         Rpm2p major protein highly purified <br /> mitochondrial protein synthesis coded                     preparations mitochondrial RNase P (Morales et <br /> organelle genome. maintenance wild-type mi-                      al. 1992).  antibodies raised Rpm2p <br /> tochondrial genome dependent mitochondrial                          immunoprecipitated RNase P activity Rpm1r <br /> protein synthesis (Myers et al. 1985).                                    mitochondrial extracts (Dang Martin 1993). <br />   facultative aerobes like S. cerevisiae, mutations                 Finally, strains insertional disruption midway <br /> mitochondrial DNA tolerated cells grown                     RPM2 reading frame accumulated mito- <br /> fermentable carbon sources. Consequently, nuclear                         chondrial tRNA precursors 5&#226;&#172;&#732; extensions, Rpm1r <br /> genes encoding respiratory components factors                      precursors, unable grow respiratory car- <br /> quired mitochondrial gene expression affect respira-                  bon sources (Morales et al. 1992; Stribinskis et al. <br /> tory growth growth fermentable carbon                          1996). data demonstrate RPM1 RPM2 <br /> sources. contrast, nuclear mutations interfere                    encode subunits RNase P activity                                                                           ribonucleoprotein complex required respiratory <br />                                                                           fermentative growth. <br />                                                                             Unexpectedly, cells null allele RPM2   Corresponding author: Nancy Martin, Department Biochemistry          grow respiratory fermentable carbon Molecular Biology, University Louisville, 319 Abraham Flexner <br /> Way, Bldg.  Rm. 708, Louisville, KY 40292.                              sources, indicating RPM2 function addition <br /> E-mail: ncmart01@gwise.louisville.edu                                     role subunit mitochondrial RNase P (Kas- <br /> Genetics 154: 1013&#226;&#8364;&#8220;1023 (March 2000) <br />  1014                                         M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br /> senbrock et al. 1995). function provided                tions ) added autoclaved plate media concen- N-terminal half Rpm2p, explaining strains             trations reported results. <br />                                                                        Construction null allele RPM2: YMW1 YMW2 insertional disruption grew fermentable                 haploid derivatives W303 cells generous gift <br /> carbon sources. nature second function               Dr. Michael Walberg (Zieler et al. 1995). Deletion unclear, related isolation RPM2           RPM2 performed YMW1/YMW2 diploid, YML9. high-copy suppressors temperature-               complete RPM2 open reading frame (ORF) replaced sensitive (ts) allele TOM40/ISP42 (Kassenbrock et                heterologous dominant G418 resistance gene kanMX using <br />                                                                     SFH-PCR-based gene disruption technique (Wach et al. <br /> al. 1995). TOM40 encodes essential membrane com-                 1994; Wach 1996). RPM2/kanMX primers 5&#226;&#172;&#732; CAA <br /> ponent mitochondrial import machinery (Vest-                 CAAACAGAAAAGAAAAAACAAGCATACACAAACGAAAAT <br /> weber et al. 1989; Moczko et al. 1992; Rapaport et al.              GCGTACGCTGCAGGTCGAC 3&#226;&#172;&#732; 5&#226;&#172;&#732; GTTATAACAGTG <br /> 1997; Hill et al. 1998). N-terminal portion RPM2             AAATAAATAAAATATCAAATTGTAAAGGTCAATCGAT sufficient growth fermentable carbon                 GAATTCGAGCTCG 3&#226;&#172;&#732;. Underlined sequences oligonucle- <br />                                                                     otides mark parts sequence complemen- <br /> sources suppresses TOM40 mitochondrial im-                 tary pFA6-MCS plasmid pFA6-kanMX6 (Wach 1996). <br /> port defect (Kassenbrock et al. 1995). Rpm2p                    Southern blot analysis PCR confirmed proper inser- <br /> Tom40p appear interact assayed                  tion kanMX RPM2 locus, specifically starting hybrid  suggesting RPM2 sup-                <span id='am-28' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span> (nt) 3&#226;&#172;&#732; ATG RPM2 ORF press mutant allele physically interacting              penultimate nt 5&#226;&#172;&#732; STOP codon. <br />                                                                        Suppressors RPM2-null mutation: different Tom40p (Groom 1995).  requirement                     diploid RPM2/&#226;&#338;&#172;rpm2::kanMX strains, YML34, YML37, RPM2 growth remains unclear.                                YML41, sporulated haploid spores separated <br />    RPM2-null mutants display significant rate pheno-           tetrad analysis YPD plates. sporulation pattern ob- <br /> typic reversion glucose-containing media. rate              served cases 2:2 ratio big/small colonies indicative spontaneous second-site loss function           YPD 30&#226;&#172;&#353; 3 days. small colonies increase <br />                                                                     size additional week 30&#226;&#172;&#353;. total 14 small <br /> suppressor mutations genes functionally linked                &#226;&#338;&#172;rpm2 colonies individually suspended 200, 300, essential growth function RPM2. labeled             400 &#226;?&#174;l YPD. small aliquot cells counted using second-site suppressors, suppressors arrest (soa) mu-            hemocytometer. remaining cells plated YPD tants.                                                              30&#226;&#172;&#353;. Suppression &#226;&#338;&#172;rpm2-arrested growth results pheno- <br />     independent soa suppressor                typic revertant colonies, likely occurring sponta- <br />                                                                     neous second-site suppressors called suppression arrest <br /> strains fall different complementation              alleles (soa alleles). maximum different colony sizes <br /> groups, suppress lack growth &#226;&#338;&#172;rpm2               seen revertant populations. Individual colonies <br /> strains glucose media. demonstrate                picked replated score homogeneous growth. mutant alleles affect 26S proteasome function.                &#226;&#338;&#172;rpm2 soa colonies grew homogeneously PRE4 identified wild-type allele soa1             studied  determine soa mutations reces- <br />                                                                     sive dominant &#226;&#338;&#172;rpm2, &#226;&#338;&#172;rpm2 soa cells mated <br /> encodes &#226;?&#164;-7 subunit 20S proteasome core <br />                                                                     &#226;?&#179;&#226;&#171;&#185; &#226;&#338;&#172;rpm2 SOA strains containing wild-type copy RPM2 <br /> (Groll et al. 1997). UMP1 wild-type allele soa2           YEp352 covering plasmid. 5-FOA shuffle performed encodes chaperone necessary assembly               lose YEp352 RPM2 plasmid (Sikorski Boeke 1991). 20S proteasome core complex (Ramos et al.                       Identification wild-type SOA alleles: Seventeen indepen- <br /> 1998).  defects proteasome activity compensate              dent &#226;&#338;&#172;rpm2 soa phenotypic revertants backcrossed lack RPM2.                                               isogenic wild-type strains, YMW1 YMW2, isolate soa <br />                                                                     mutations wild-type background deter- <br />                                                                     suppression originated single-site mutations. Dip- <br />               MATERIALS METHODS                                 loids sporulated haploid progeny separated                                                                     tetrad analysis. growth haploid progeny evaluated <br />    Strains, plasmids, media: Standard yeast manipulations       rich minimal glucose glycerol media 16&#226;&#172;&#353; used (Sherman 1991; Sherman Hicks 1991; Kaiser             37&#226;&#172;&#353;. Tight conditional growth phenotypes observed et al. 1994). Yeasts transformed plasmid DNA using        independent backcrosses, permitting pursue lithium acetate method (Chen et al. 1992). yeast strains      soa strains  Tetrad analysis patterns parental, nonpa- <br /> used study listed Table 1. used centro-       rental, tetratype indicated mutations originated <br /> meric pRS300 series plasmids subcloning (Sikorski        single, nuclear loci. ability RPM2 SOA/RPM2 soa <br /> Hieter 1989).                                                       diploids grow restrictive conditions demonstrated <br />    Rich glucose media, YPD, included 1% Bacto-yeast extract,        soa1, soa2, soa3, soa4, soa5 recessive. <br /> 2% Bacto-peptone, 2% glucose. Rich glycerol-ethanol             RPM2 soa1 yeast (37B1.14OCD3) transformed dia, YPGE, contained 1% Bacto-yeast extract, 2% Bacto-pep-          pRS200 centromeric genomic library, YPH1, (Sikorski tone, 3% (v/v) glycerol, 2% (v/v) ethanol. Synthetic com-       Hieter 1989; American Type Culture Collection (ATCC)  <br /> plete (SC), synthetic complete lacking specific amino acids         77165) screened suppression cold-sensitive (cs) <br /> (SC trp&#226;&#171;&#186;), synthetic dextrose minimal media (SD) contain        phenotype 16&#226;&#172;&#353; YPGE. plasmid containing 8.8-kb <br /> 0.67% Bacto-yeast nitrogen base amino acids, 2% glu-        insert suppressed conditional growth phenotype. Subclon- <br /> cose, appropriate supplements (Kaiser et al. 1994). Solid       ing identified 2.2-kb fragment containing single ORF, <br /> media plates included 2% Bacto-agar. Culture media           PRE4, supported growth RPM2 soa1 cells 16&#226;&#172;&#353; agents Fisher Scientific (Pittsburgh) Difco (Detroit)       YPGE. Linkage mutant phenotype PRE4 locus <br /> brand. G418, canavanine, CdCl2 aqueous solutions           confirmed integrating PRE4 URA3 linearized plasmid <br /> sterilized passage 0.2-&#226;?&#174;m filter (Micron Separa-       PRE4 locus RPM2 soa1 cells (Rothstein 1991).                                                   Proteasome Mutants Suppress &#226;&#338;&#172;rpm2                                               1015 <br />  <br />                                                                 TABLE 1 <br />                                                     Yeast strains used study <br />  <br /> Strain                                               Genotype                                             Source reference <br /> YMW1                    MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             M. Walberg; Zieler et al. (1995) <br /> YMW2                    MATa ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             M. Walberg; Zieler et al. (1995) <br /> YMW9                    MATa/MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1                 study <br />                            can1-100 <br /> YML34                   MATa/MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11                               study <br /> YML37                   trp1-1 ura3-1 can1-100 RPM2/&#226;&#338;&#172;rpm2::kanMX <br /> YML41 <br /> 37B1.14OCD3             MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             study <br />                          pre4-2 <br /> 34D1.51OCC1             MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             study <br />                          ump1-2 <br /> 41B3.2OCC1              MAT&#226;?&#163; ade2-1 ade3&#226;&#338;&#172;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             study <br />                          soa3 <br />  <br />  <br />  <br /> cells mated isogenic YMW2, haploid progeny            inhibitors diluted 1:10 (Complete mini cocktail tablets; <br /> scored <span id='am-48' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> auxotrophy growth restrictive          Boehringer Mannheim, Indianapolis). equal volume conditions.                                                            sterile glass beads added, cells disrupted 3 <br />    RPM2 soa2 yeast (34D1.51OCC1) similarly trans-                 min mini-bead beater (Biospec Products) followed 15 <br /> formed YCp50 centromeric genomic library, CEN BANK              min &#226;&#171;&#186;70&#226;&#172;&#353; 3 min bead beater. Samples (Rose et al. 1987; ATCC  37415), screened 37&#226;&#172;&#353;              boiled cleared particulate matter centrifu- YPGE. Plasmid-based suppression seen 11-                gation 12,000 &#226;&#171;&#187; g microfuge. concentration kb insert. Subcloning identified UMP1 wild-type allele          proteins supernatant determined (Bio-Rad protein <br /> suppressing soa2 ts phenotype. Linkage mutant               assays). total 1 &#226;?&#174;g protein loaded 10% <br /> phenotype UMP1 locus confirmed integrating             SDS/polyacrylamide gel, run, blotted Immobilon-P UMP1 URA3 linearized plasmid UMP1 locus RPM2               (Millipore, Bedford, MA) membranes. Probes included rab- <br /> soa2 cells (Rothstein 1991). cells mated iso-            bit polyclonal antiubiquitin antibody (Haas Bright <br /> genic YMW1, haploid progeny scored uracil                 1985) generously provided Dr.  Haas, mouse mono- <br /> auxotrophy growth restrictive conditions.                clonal antiactin antibody (Boehringer Mannheim). Enhanced <br />    Nucleic acid techniques DNA sequencing: Standard                chemiluminescence (Amersham Life Sciences, Arlington <br /> DNA manipulations methodologies used (Sam-                    Heights,  used signal detection. <br /> brook et al. 1989). Restriction enzymes T4 DNA ligase                 Microscopy: Nuclear mitochondrial DNA stained New England Biolabs (Beverly, MA) MBI Fer-                vital dye DAPI (4&#226;&#172;&#732;,6-diamidino-2-phenylindole) mentas. PCR DNA polymerase enzymes used include TAQ                    independent experiments strain (Kaiser et <br /> (Fisher), Klentaq-LA polymerase mix (Sigma, St. Louis),            al. 1994). Approximately 107 cells pelleted microfuge <br /> Vent (New England Biolabs). primers PCR recovery               5-sec pulse resuspended 70% ethanol ice. sequence PRE4 locus upstream primer 5&#226;&#172;&#732; CATT             Cells fixed 10&#226;&#8364;&#8220;30 min ice washed twice CCGTGTAGTGCCAATA 3&#226;&#172;&#732; downstream primer 5&#226;&#172;&#732;                           <span id='am-23' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span>. Cells resuspended 1 &#226;?&#174;g/ml DAPI GGTTACCCGTCAATCCTTTT 3&#226;&#172;&#732;. equivalent PCR prim-                      mounted using aqueous Light antifade Prolong anti- <br /> ers UMP1 locus upstream primer 5&#226;&#172;&#732; CGGCAC                  fade (Molecular Probes, Eugene, . Observations TAATCTAATCTTTCAA 3&#226;&#172;&#732; downstream primer 5&#226;&#172;&#732;                           Nikon Optiphot light microscope &#226;&#171;&#187;60 Plan- <br /> TGGCAATTTTATCCTACCTGAG 3&#226;&#172;&#732;.                                             Apo 1.4 NA objective. <br />    DNA sequencing performed University Wiscon- <br /> sin Biotechnology Center, DNA Synthesis Sequencing <br /> Facility (Dr. Charles Nicolet). PCR products sequenced <br /> directly cloning PCR-TA cloning vector                                       RESULTS <br /> vitrogen). strands PCR product completely <br /> sequenced assure accuracy. Plasmid DNA isolated using              Identification isolation spontaneous second- Bio-Rad (Richmond, CA) Qiagen miniprep kits.             site suppressors &#226;&#338;&#172;rpm2: Sporulation RPM2/ <br /> BLAST searches compared aligned nucleotide sequences               &#226;&#338;&#172;rpm2::kanMX diploids subsequent growth hap- <br /> (Altschul et al. 1990), SIM alignment portion             loid progeny rich glucose yielded 2:2 ratio small/ ExPASy home page used amino acid comparisons. <br /> Amino acid numbering mature Pre4p based infor- <br />                                                                        large colonies. 90% cells mation Protein Data Bank (Brookhaven, NY)               small colonies &#226;&#338;&#172;rpm2 allele lacked buds, indicat- <br /> accession  1RYP (Groll et al. 1997).                                ing ceased growth. <br />    Protein analysis: Protein isolated log-phase cul-             step understanding phenotype, <br /> tures. Cells grown 30&#226;&#172;&#353; OD640 0.5             determined number cells small colo- <br /> transferred 16&#226;&#172;&#353; 3.5 hr. Cells collected centrifuga- <br /> tion stored &#226;&#171;&#186;70&#226;&#172;&#353;. Frozen pellets resuspended            nies. average &#226;&#181;&#8216;4.4 &#226;&#171;&#187; 107 cells small colony 200 &#226;?&#174;l NET-NP (150 mm NaCl, 5 mm EDTA, 50 mm Tris-                     obtained direct counting cells 14 independent <br /> HCl, pH 7.5, 0.5% NP-40) plus protease inhibitors. Protease            colonies. number consistent &#226;&#181;&#8216;25 rounds  1016                                        M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br />                                                             TABLE 2 <br />                                   Growth phenotypes RPM2 soa complementation groups <br />  <br />          Strain genotype                    YPD 16&#226;&#172;&#353;             YPD 37&#226;&#172;&#353;             YPGE 16&#226;&#172;&#353;              YPGE 37&#226;&#172;&#353; <br />          37B1.14OCD3 (RPM2 soa1)                    &#226;&#171;&#185;                  &#226;&#171;&#185;&#226;&#171;&#185;                   &#226;&#171;&#186;                    &#226;&#171;&#185; <br />          34D1.51OCC1 (RPM2 soa2)                   &#226;&#171;&#185;&#226;&#171;&#185;                  &#226;&#171;&#185;&#226;&#171;&#185;                   &#226;&#171;&#185;                    &#226;&#171;&#186; <br />          41B3.2OCC1 (RPM2 soa3)                    &#226;&#171;&#185;&#226;&#171;&#185;                  &#226;&#171;&#185;&#226;&#171;&#185;                   &#226;&#171;&#185;                    &#226;&#171;&#186; <br />            Cultures grown late log phase YPGE 30&#226;&#172;&#353; diluted OD640 &#226;&#171;&#189; 0.1. Aliquots (4-&#226;?&#174;l) 10- <br />          fold serial dilutions spotted appropriate plates. Growth colonies monitored 1 wk. Values <br />          comparisons wild-type YMW2 growth (RPM2 SOA), &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;. <br />  <br />  <br /> cell division initial spore. mean spontane-           soa complementation groups (Table 2). Analysis <br /> ous phenotypic reversion rate 14 independent                  &#226;&#338;&#172;rpm2 soa/&#226;&#338;&#172;rpm2 SOA diploid strains showed &#226;&#338;&#172;rpm2::kanMX colonies, plated rich glucose 30&#226;&#172;&#353;,              mutations complementation groups soa1 soa3 1.5 &#226;&#171;&#187; 10&#226;&#171;&#186;7 1.4 &#226;&#171;&#187; 10&#226;&#171;&#186;6 revertants/                  recessive unable suppress &#226;&#338;&#172;rpm2 pheno- <br /> original spore. Phenotypic revertants designated              type presence wild-type alleles. soa2 <br /> suppressor arrest (soa). 30&#226;&#172;&#353; rich glucose media           allele, hand, semidominant par- <br /> revertant colonies grew different rates               tially able suppress &#226;&#338;&#172;rpm2 presence wild- isogenic wild-type strains. Seventeen               type SOA2. <br /> &#226;&#338;&#172;rpm2 soa double mutants demonstrated growth                     SOA1 locus PRE4, encodes &#226;?&#164; subunit <br /> defects YPD, 16&#226;&#172;&#353; 37&#226;&#172;&#353;, selected            20S proteasome: RPM2 soa1 cells trans- study.                                                     formed low-copy centromeric genomic library <br />    17 soa mutants, 5 demonstrated tight temperature-            (YPH1) screened complementation Cs&#226;&#171;&#186; <br /> sensitive (Ts&#226;&#171;&#186;) cold-sensitive (Cs&#226;&#171;&#186;) phenotypes              respiratory growth phenotype (materials meth- <br /> rich YPGE media linked soa phenotype                   ods). single plasmid containing 8.8-kb insert com- <br /> separate &#226;&#338;&#172;rpm2 mutation. Subsequent crosses               plemented restrictive growth 16&#226;&#172;&#353; YPGE. Subclon- <br /> demonstrated soa mutations reces-            ing ORFs insert revealed PRE4 sive (materials methods). conditional phe-                 responsible complementation. PRE4 encodes <br /> notypes used screening yeast genomic libraries             &#226;?&#164;-7 subunit 20S proteasome core complex wild-type alleles soa loci.                             essential cell viability (Hilt et al. 1993; Groll <br />    Pairwise crosses soa haploids        et al. 1997). Figure 1A shows PRE4 suppression Cs&#226;&#171;&#186;  diploids plated restrictive growth            Ts&#226;&#171;&#186; respiratory growth phenotypes RPM2 soa1 <br /> conditions. crosses indicated                cells. determine soa1 allele PRE4, plasmid <br />  <br />  <br />                                                                                                   Figure 1.&#226;&#8364;&#8221;Permissive                                                                                                   restrictive growth condi- <br />                                                                                                   tions RPM2 pre4-2                                                                                                   RPM2 ump1-2 cells YPGE <br />                                                                                                   media. Genomic library <br />                                                                                                   screens led identifi- <br />                                                                                                   cation PRE4 wild- <br />                                                                                                   type allele soa1 sup- <br />                                                                                                   pression RPM2 soa1 cs <br />                                                                                                   phenotype UMP1                                                                                                   identified wild-type <br />                                                                                                   allele soa2 suppression <br />                                                                                                   RPM2 soa2 ts pheno- <br />                                                                                                   type. Logarithmically grow- <br />                                                                                                   ing cultures diluted                                                                                                   OD640 &#226;&#171;&#189; 0.1 4 &#226;?&#174;l 10- <br />                                                                                                   fold serial dilutions                                                                                                   spotted YPGE plates. <br />                                                                                                   Plates incubated                                                                                                   given temperatures fol- <br />                                                                                                   lows: 16&#226;&#172;&#353;, 5 days; 30&#226;&#172;&#353;, 2 days; <br />                                                                                                   37&#226;&#172;&#353;, 3 days.   Wild-type <br /> cells original parent strain YMW2; soa1 represents strain 37B1.14OCD3; pRS314 PRE4 CEN plasmid PRE4 <br /> locus. (B) Wild-type cells original parent strain YMW1; soa2 represents strain 34D1.51OCC1; pRS316 CEN plasmid UMP1 locus. <br />                                               Proteasome Mutants Suppress &#226;&#338;&#172;rpm2                                          1017 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 2.&#226;&#8364;&#8221;Hypersensitivity pre4-2 <br />                                                                                     ump1-2 arginine analogue ca- <br />                                                                                     navanine oxidative damage <br />                                                                                     caused CdCl2. pre4-2, ump1-2, soa3 <br />                                                                                     cultures actively growing YPD                                                                                     spotted 1:10 serial dilutions mini- <br />                                                                                     mal media plates.   Canavanine (30 <br />                                                                                     &#226;?&#174;g/ml); (B) CdCl2 (30 &#226;?&#174;m). Incubation <br />                                                                                     permissive temperatures 3 <br />                                                                                     days. higher concentrations cana- <br />                                                                                     vanine used study                                                                                     can1-100 mutant allele strains <br />                                                                                     derived W303 background.                                                                                     can1-100 allele confers resistance low <br />                                                                                     levels canavanine. <br />  <br />  <br />  <br />  <br /> containing wild-type PRE4 URA3 integrated            crease mutant proteins normally degraded PRE4 locus soa1 cells. cells crossed          proteasome. Relative wild-type cells, pre4-2 wild type, sporulated, tetrads dissected.       ump1-2 cells showed hypersensitivity canavan- <br /> haploid progeny exhibited 4:0 ratio wild-type/mu-          ine, phenotypic characteristic cells defective <br /> tant segregation pattern YPGE growth 16&#226;&#172;&#353;, demon-          proteasome activity (Figure 2; Chen Hochstrasser <br /> strating soa1 mutant allele PRE4. soa1         1996; Ramos et al. 1998). RPM2 pre4-2 cells  <br /> allele second allele PRE4 identified          creased colony size relative wild-type strains 5 &#226;?&#174;g/ <br /> termed pre4-2 (Hilt et al. 1993). previously identi-        ml canavanine growth 30 35 <br /> fied pre4-1 allele produces truncated protein         &#226;?&#174;g/ml canavanine (Figure 2A). Similar results seen <br /> creased peptidylglutamyl peptidase hydrolyzing (PGPH)           RPM2 ump1-2 mutants grow activity does affect cell growth (Hilt et al. 1993).   media containing 40 &#226;?&#174;g/ml canavanine. contrast, <br />    SOA2 locus UMP1, encodes 20S protea-          RPM2 soa3 cells grow near-wild-type levels maturation factor: overlapping plasmids com-           spotted SD-canavanine plates. combined <br /> plementing soa2 defect isolated (materials             effect null mutation RPM2 soa methods). Subcloning possible ORFs               mutations led poor growth SD media vealed UMP1 complementing gene. Figure             growth presence low concentrations 1B shows UMP1 suppression Ts&#226;&#171;&#186; respiratory                canavanine (5&#226;&#8364;&#8220;10 &#226;?&#174;g/ml; data shown). implies <br /> growth phenotype RPM2 soa2 cells. UMP1 sup-             cells lacking RPM2 increased sensitivity presses Ts&#226;&#171;&#186; respiratory phenotype mem-         stressful conditions. <br /> ber soa2 complementation group. UMP1, like                  second test proteasome function sensitivity <br /> PRE4, necessary function 20S core pro-        cells oxidative damage caused CdCl2. RPM2 <br /> teasomal complex. UMP1 encodes chaperone involved             pre4-2 RPM2 ump1-2 cells grow poorly presence 20S proteasome assembly (Ramos et al. 1998).              30 &#226;?&#174;m CdCl2 (Figure 2B). double &#226;&#338;&#172;rpm2 soa <br /> determine soa2 mutations alleles UMP1,             mutants failed grow conditions (data <br /> plasmid containing wild-type UMP1 integrated             shown). Similar results obtained canavanine, UMP1 locus soa2 cells. cells crossed          RPM2 soa3 cells sensitive CdCl2. wild type, sporulated, tetrads dissected.       canavanine CdCl2 sensitivities overcome haploid progeny revealed 4:0 ratio wild-type/mu-           RPM2 pre4-2 RPM2 ump1-2 cells transformed <br /> tant segregation pattern growth YPGE growth            PRE4 UMP1, respectively, low-copy vectors <br /> 37&#226;&#172;&#353;, demonstrating soa2 locus UMP1.              (Figure 2, B). soa3 complementation group <br /> named soa2 allele ump1-2.                                   does evidence defective 26S proteasome <br />    Sensitivity soa mutants canavanine CdCl2:          activity. investigations relationship soa3 determine proteasome function compromised             suppressor &#226;&#338;&#172;rpm2 yeast ongoing. pre4-2, ump1-2, soa3 strains, cultures incu-           Degradation ubiquitinated proteins inhibited bated variety stressful conditions           pre4-2 yeast: test impaired proteasome <br /> crease cellular dependence proteasomal degrada-              activity pre4-2 cells, examined level ubiquiti- <br /> tion. Exposure cells increasing concentrations         nated proteins permissive restrictive growth  arginine analogue canavanine causes general            ditions (Haas 1988; chen Hochstrasser 1995; <br />  1018                                         M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br />  <br />  <br />  <br />                                                                        Figure 4.&#226;&#8364;&#8221;Morphology &#226;&#338;&#172;rpm2 soa mutants  <br />                                                                     missive growth conditions. Cultures grown YPD 30&#226;&#172;&#353; <br />                                                                     OD640 &#226;&#171;&#189; 0.5.   Wild-type YMW2; (B) &#226;&#338;&#172;rpm2 pre4-2; <br />                                                                     (C) &#226;&#338;&#172;rpm2 ump1-2; (D) &#226;&#338;&#172;rpm2 soa3. Fluorescence images reveal <br />                                                                     nuclear mitochondrial DNA stained DAPI. &#226;&#338;&#172;rpm2 <br />                                                                     pre4-2 &#226;&#338;&#172;rpm2 ump1-2 cultures interconnected cells <br />                                                                     appear contain nuclear DNA (B C). &#226;&#338;&#172;rpm2 pre4-2 cells <br />                                                                     &#226;?&#179;&#226;&#171;&#186; (B) &#226;&#338;&#172;rpm2 ump1-2 &#226;&#338;&#172;rpm2 soa3 cells &#226;?&#179;o <br />                                                                     (C D). Bar, 5 &#226;?&#174;m. <br />    Figure 3.&#226;&#8364;&#8221;Accumulation ubiquitinated proteins pre4-2 strain 37B1.14OCD3. Western blot total yeast ex- <br /> tracts probed polyclonal antibody ubiquitin. <br /> Yeast cell extract (1 &#226;?&#174;g) mid-log-phase cultures              Sequence ump1-2 ump1-3 reveals contain <br /> loaded 10% polyacrylamide gel. (Lanes 1 3) YMW2 wild-type cells; (lanes 2 4) 37B1.14OCD3 <br />                                                                     nonsense mutations: Cells lacking UMP1 viable (pre4-2) cells. Lanes 1 2 grown 30&#226;&#172;&#353; cultures      grow slowly (Ramos et al. 1998). Ump1p postulated reached OD640 &#226;&#171;&#189; 0.5. Lanes 3 4 shifted           interact proPre2p block protease maturation <br /> restrictive temperature 16&#226;&#172;&#353;. shift 16&#226;&#172;&#353; 3 hr       proteasome assembly complete. ump1-2 identical 30&#226;&#172;&#353; OD640 &#226;&#171;&#189; 0.5 cultures. High-molecular-mass        ump1-3 contained nonsense mutations resulting ubiquitin-protein conjugates indicated left. Molecu- <br /> lar mass standards kilodaltons indicated right.       premature stop codons codons 10 36, respectively. <br />                                                                        Morphology &#226;&#338;&#172;rpm2 soa RPM2 soa mutants: <br />                                                                     Cells lacking RPM2 fail produce mature mitochon- <br /> Rinaldi et al. 1998). Wild-type mutant cells               drial tRNA, leading absence mitochondrial pro- <br /> grown 30&#226;&#172;&#353; YPGE media. Aliquots log-phase                 tein synthesis petite S. cerevisiae strains. state <br /> cultures transferred 16&#226;&#172;&#353; 3 hr main-           mitochondrial genome analyzed using com- <br /> tained 30&#226;&#172;&#353;. Total protein isolated, fractionated             bination DAPI staining genetic crosses. DAPI SDS-PAGE, transferred polyvinyldifluoride mem-                staining revealed mitochondrial DNA &#226;&#338;&#172;rpm2 pre4-2 <br /> brane, probed antiubiquitin antibody. blot             cells, &#226;&#338;&#172;rpm2 ump1-2 &#226;&#338;&#172;rpm2 soa3 showed showed increased signal RPM2 pre4-2 extracts vs.              extranuclear fluorescent staining (Figure 4). Mitochon- <br /> wild-type cells 16&#226;&#172;&#353; (Figure 3; Haas Bright 1985).            drial DNA distributed elongated, diffuse signal molecular weight range 35&#226;&#8364;&#8220;250             interconnected &#226;&#338;&#172;rpm2 pre4-2 cells nuclear <br /> kD, reflects general increase ubiquitinated              DNA did appear segregating properly. proteins, typical cells impaired proteasome             determine &#226;&#338;&#172;rpm2 pre4-2 maintained wild-type mito- <br /> activity.                                                           chondrial genome, cells mated YMW2 &#226;?&#179;o, <br />    pre4-2 allele contains missense mutation:              isogenic strain containing wild-type nuclear ge- <br /> pre4-2 mutation <span id='am-43' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Guanine" datatype="xsd:string"></span>guanine</span> <span id='am-53' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span>thymidine</span> base change               nome lacking mitochondrial DNA. resultant nucleotide 157 PRE4 ORF, results               diploids unable grow YPGE, demonstrating <br /> valine phenylalanine substitution amino acid (aa)             &#226;&#338;&#172;rpm2 pre4-2 cells &#226;?&#179;&#226;&#171;&#186;. <br /> 12 mature protein. Sequence comparison pre4-                 Proteasome degradation cyclins involved cell <br /> 2p closely related 20S proteasome               cycle regulation documented (Chun et <br /> &#226;?&#164;-7 subunits archtype &#226;?&#164; subunit Ther-               al. 1996; King et al. 1996; Hershko 1997). &#226;&#338;&#172;rpm2 <br /> moplasma acidiphilium showed amino acid substi-           pre4-2 &#226;&#338;&#172;rpm2 ump1-2 mutants displayed range tution resides highly conserved region.               morphologies indicative cells defective cell cycle <br /> change introduces bulky Phe start &#226;?&#164;-strand              progression (Figure 4). Approximately 5&#226;&#8364;&#8220;10% &#226;&#338;&#172;rpm2 <br /> 2 &#226;?&#164;-sandwich tertiary structure described             pre4-2 cells grown 30&#226;&#172;&#353; YPD multiply budded <br /> Groll et al. (1997).                                                showed interconnected, elongated morphologies <br />                                                Proteasome Mutants Suppress &#226;&#338;&#172;rpm2                                        1019 <br />  <br />                                                                  soa mutants respiratory competent &#226;?&#179;&#226;&#171;&#185; mito- <br />                                                                  chondrial genomes. used DAPI staining observe <br />                                                                  nuclear mitochondrial DNA patterns                                                                  soa mutants. log-phase growth 30&#226;&#172;&#353;, pre4-2                                                                  ump1-2 cultures noticeably different wild-type <br />                                                                  cells contain multiply budded elongated <br />                                                                  cells (Figure 5,  C, E).  mutant <br />                                                                  phenotype displayed degree                                                                  &#226;&#338;&#172;rpm2 pre4-2 &#226;&#338;&#172;rpm2 ump1-2 cultures. Figure <br />                                                                  4, cell bodies contain nuclear DNA, mito- <br />                                                                  chondrial DNA interconnected <br />                                                                  cells. restrictive conditions (3&#226;&#8364;&#8220;16 hr 16&#226;&#172;&#353;                                                                  37&#226;&#172;&#353;, respectively) RPM2 pre4-2 RPM2 ump1-2 <br />                                                                  cultures contained elongated, multiply budded cells <br />                                                                  similar &#226;&#338;&#172;rpm2 counterparts. Consistent                                                                  results double mutant, RPM2 soa3 retains                                                                  wild-type morphology permissive restrictive <br />                                                                  conditions, supporting conclusion soa3                                                                  proteasome-functional mutant (Figure 5, G H). <br />  <br />  <br />                                                                                        DISCUSSION <br />                                                                     results presented demonstrate growth <br />                                                                  fermentable carbon sources restored null <br />                                                                  mutants RPM2 mutations different genes <br />                                                                  disrupt 20S proteasome function. encodes                                                                  proteasome &#226;?&#164;-7 subunit; encodes chaperone <br />                                                                  necessary assembly active 20S proteasome com- <br />                                                                  plex (Ramos et al. 1998). <br />                                                                     Proteasome function essential eukaryotic cells <br />                                                                  regulated degradation targeted proteins.                                                                  <span id='am-12' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>- ubiquitin-dependent proteasome pathway <br />                                                                  plays key role degradation misfolded/mutant <br />                                                                  proteins, controlling cell growth divi- <br />                                                                  sion degradation regulatory proteins <br />   Figure 5.&#226;&#8364;&#8221;DAPI staining RPM2 soa mutants  <br /> missive restrictive growth conditions. Cultures grown   (Chun et al. 1996; Hilt Wolf 1996; King et al. 1996; YPD 30&#226;&#172;&#353; mid-log phase observation permissive     Hershko 1997). 26S proteasome proteolytic <br /> conditions   C, E, G). Identically grown cultures     component ubiquitin-mediated protein degrada- <br /> shifted 16&#226;&#172;&#353; (B H) 37&#226;&#172;&#353; (D, F,  3&#226;&#8364;&#8220;16    tion pathway. dumbbell-shaped symmetrical struc- <br /> hr observation fluorescent microscopy.  B)       ture composed barrel-shaped 20S proteolytic core <br /> RPM2 pre4-2, (C D) RPM2 ump1-2, (E F) RPM2 soa3, <br /> (G  wild-type YMW2 cells. Bar, 5 &#226;?&#174;m.                           capped 19S regulatory particles. yeast 20S protea- <br />                                                                  crystalized recently contains 14 dis- <br />                                                                  tinct structurally related subunits: 7 type &#226;?&#163; (Figure 4B). overnight shift restrictive         7 type &#226;?&#164; organized stacked heptomeric rings <br /> temperature 16&#226;&#172;&#353;, entire culture displayed aber-           configuration &#226;?&#163;&#226;?&#164;&#226;?&#164;&#226;?&#163; (Groll et al. 1997). 20S <br /> rant morphologies dead, serially connected             core complex assembled proprotein precursors <br /> ghosts observed (data shown). &#226;&#338;&#172;rpm2 ump1-2 cul-              including inactive precursors proteasome prote- <br /> tures interconnected cells continue divide 37&#226;&#172;&#353;          ases (Nandi et al. 1997). &#226;?&#164; subunits encoded quite slowly decreased viability (data        essential genes. <br /> shown). 30&#226;&#172;&#353; interconnected, elongated morphol-                Subunits 20S core complex ogy evident &#226;&#338;&#172;rpm2 ump1-2 cultures              structural catalytic roles proteasome function. <br /> reached stationary phase (Figure 4C). &#226;&#338;&#172;rpm2 soa3 cul-            different &#226;?&#164; subunits provide distinct N-ter- <br /> tures appeared wild-type cells permissive         minal threonine proteases proteasome catalytic <br /> restrictive temperatures (Figure 4D).                            core. PUP1, PRE2, PRE3 genes encode &#226;?&#164;-sub- <br />    Mutant morphologies cells permissive             unit proteases trypsin-like, chymotrypsin-like, strictive conditions prominent            PGPH specificities, respectively (Arendt Hoch- <br /> soa1 soa2 strains wild-type RPM2. single        strasser 1997; Heinemeyer et al. 1997; Ciechanover <br />  1020                                       M. S. Lutz, S. R. Ellis N. C. Martin <br />  Schwartz 1998). Pre4p catalytically active                Rpm2p multifunctional protein required protease necessary PGPH activity (Hilt            mitochondrial RNase P activity essential <br /> et al. 1993; Heinemeyer et al. 1997). Mutations protea-        function,  like mitochondria, required &#226;?&#164; subunits, Pre4p, thought influence          fermentative growth. amino-terminal 714-aa por- conformation catalytic subunits affect bind-        tion RPM2, supports growth glucose, ing orientation substrate peptides. Consistent             sufficient serve high-copy suppressor mito- critical role noncatalytic subunits protea-         chondrial protein import defect resulting mutant structure observation null mutant           allele TOM40 (Kassenbrock et al. 1995). pre4-2 <br /> PRE4 lethal. requirement PRE4               ump1-2 suppression &#226;&#338;&#172;rpm2 suggests model relat-  solely related role PGPH activity,      ing proteasomal activity essential feature mito- <br /> truncated allele pre4-1 lacks PGPH activity displays          chondrial biogenesis. detectable effects cell growth degrada-              proteasome mutants suppress mito- <br /> tion reporter protein vivo (Hilt et al. 1993).            chondrially related phenotypes reported <br />    work suggests Val Phe substitution          (Campbell et al. 1994; rinaldi et al. 1998). mutant <br /> &#226;?&#164;-7 global effects proteasome activity. Cells         allele RPT3 (ynt1-1) identified suppressor pre4-2 allele exhibit biochemical morphological             enhanced rate escape mitochondrial DNA stress-related phenotypes characteristic cells         nucleus strains mutant alleles YME1 <br /> duced proteasome activity.  pre4-2 cells accu-           (Thorsness et al. 1993; Campbell et al. 1994). mulated higher levels ubiquitin-protein conjugates             RPT3 mutant suppressed abnormal mitochondrial <br /> relative wild type. interpret data indicate        morphologies growth defects associated yme1 pre4-2 mutation affects just PGPH          mutations. Yme1p ATP- zinc-dependent prote- <br /> activity proteasome. Biochemical, immunologi-              ase component AAA proteolytic complex, cal, crystallographic studies shown Pre4p           localized inner mitochondrial membrane close proximity Pre3p Pup1p (Groll et al.            (Thorsness et al. 1993; Leonhard et al. 1996; Weber <br /> 1997; Heinemeyer et al. 1997; Kopp et al. 1997).             et al. 1996). Unassembled subunits ATP synthase pre4-2 mutation potentially disrupt multiple            respiratory chain components degraded,  <br /> proteolytic activities proteasome           pathway involving Yme1p (Pearce Sher- <br /> overall structural integrity. Similar &#226;&#8364;&#339;poisoning&#226;&#8364;?          man 1995; Weber et al. 1996). RPT3, hand, <br /> Drosophila melanogaster 20S proteasome mutant &#226;?&#164;-2              encodes ATPase activities base &#226;?&#164;-6 subunits reported (Smyth Belote               subcomplex proteasome 19S regulatory particle <br /> 1999).  unlike D. melanogaster mutants,               (Glickman et al. 1998). Like cells pre4-2 allele, <br /> pre4-2 recessive PRE4 presence absence            mutant allele RPT3 leads cold-sensitive growth RPM2, indicating wild-type protein prefer-         nonfermentable carbon sources (Campbell et al. <br /> entially incorporated 20S proteasome              1994). unlikely RPT3 mutant sup- <br /> assembly.                                                         presses YME1 mutations replacing mitochondrial <br />    Proper proteasome assembly critical proteasome           protein functional component AAA com- <br /> function. Unlike Pre4p, Ump1p integral com-             plex. Instead, RPT3 function suppress YME1 <br /> ponent 20S core complex.  Ump1p              mutants disrupting pathway used degrade ab- involved assembly maturation 20S             normal mitochondria (Campbell Thorsness <br /> core complex, mutations affect expression             1998). Recent studies indicated vacuolar <br /> function global effects proteasome function               protease, Pep4p, plays important role escape <br /> (Ramos et al. 1998).  ump1-2, like pre4-2, appears           mitochondrial DNA nucleus YME1 mutants. suppress lack growth fermentative carbon             Mutants PEP4 suppress DNA transfer mitochon- <br /> sources &#226;&#338;&#172;rpm2 cells disrupting proteasome func-             dria nucleus YME1 mutants suppressing <br /> tion. Yeast carrying null alleles UMP1 accumulated             growth defects abnormal mitochondrial morpholo- <br /> 15S precursors 20S core complex proPre2p               gies. suggests vacuole function (Ramos et al. 1998). grow slower rates               degradation abnormal mitochondria YME1 strains sensitive stressful conditions. agreement           (Campbell et al. 1994; Campbell Thorsness results, ump1-2 mutant reported           1998). contrast, RPT3 mutants suppress pheno- <br /> sensitivities canavanine CdCl2 characteristic           types associated YME1 mutation, indicating <br /> proteasome mutants, presumably cells              proteasome mutants help promote mitochondrial <br /> fewer fully assembled, proteolytically active 20S protea-         integrity. complexes. semidominant nature                       proteasome mutant shown suppress <br /> ump1-2 allele suppressing RPM2-null mutations sug-             mutation mitochondrial genome (Rinaldi et <br /> gests single wild-type UMP1 gene &#226;&#338;&#172;rpm2/               al. 1994, 1995). search MnCl2-generated suppres- <br /> &#226;&#338;&#172;rpm2 diploid does support assembly wild-              sors mitochondrial tRNAASP point mutant led type levels 20S proteasome complexes.                          isolation yeast proteasomal gene RPN11/MPR1 <br />                                             Proteasome Mutants Suppress &#226;&#338;&#172;rpm2                                       1021 <br />  <br /> (Zennaro et al. 1989; Rinaldi et al. 1994, 1998; Glick-       tion maintenance mitochondria. model ac- <br /> man et al. 1998). RPN11/MPR1 non-ATPase subunit          counts results encompassing  lid portion 19S proteasomal regulatory          sults obtained YME1 tRNAASP mutants <br /> particle (Glickman et al. 1998). Similar pre4-2, mpr1-1    (Zennaro et al. 1989; Campbell et al. 1994; Campbell <br /> cultures grown stationary phase accumulate ubiquiti-       Thorsness 1998). <br /> nated proteins growth phenotypes prom-             Nuclear gene products comprise majority mito- <br /> inent glycerol medium. Cells mpr1-1 allele        chondrial proteins needed respiratory nonrespi- <br /> contain increased quantity total mitochondrial          ratory growth. mitochondrial import integral <br /> DNA cell, suggesting decrease proteasome        transport nuclear-encoded proteins activity elevate copy number mitochondrial         different mitochondrial compartments. Increased <br /> DNA (Rinaldi et al. 1998).                                    stabilization mitochondrial outer membrane proteins <br />    proteasome mutants isolated unique            increased rates mitochondrial import affect respect work described         growth characteristics yeast respira- <br /> specifically affect 20S proteolytic core vs. 19S      tory fermentative conditions. RPM2 essential <br /> regulatory cap subunits 26S proteasome. sup-         growth function directed level compen- <br /> pression mechanism stabilizes mitochondrial DNA          sated increasing half-life specific protea- loss proteasome function, observed          targets. hypothesis supported previous Rinaldi et al. (1998), account suppression      results RPM2 isolated high-copy suppressor &#226;&#338;&#172;rpm2 yeast &#226;&#338;&#172;rpm2 ump1-2 &#226;&#338;&#172;rpm2 soa3-1        defective allele essential mitochondrial im- <br /> cells &#226;?&#179;o. Alternatively, ynt1-1 proteasome mu-     port channel protein Tom40p (Kassenbrock et al. <br /> tant suppresses yme1 supporting global mitochondrial       1995). <br /> integrity, bypassing vacuolar degradation            Proteasomes regulate levels proteins tar- <br /> pathway, similar mechanism stabilizing mitochondria         geted mitochondrial compartments. case presence decreased proteasomal degradation          control mitochondrial protein levels cystosolic account phenotypic suppression &#226;&#338;&#172;rpm2 pre4-       proteasome activity established. Regulation 2 &#226;&#338;&#172;rpm2 ump1-2 strains.                                   achieve proper levels isoforms cytochrome c, <br />    suggest mechanism proteasome          encoded different nuclear genes localized mutants suppress RPM2-null allele indirect, affect-    intermembrane space, occurs ubiquitin- <br /> ing steady-state level key protein(s) compen-     proteasome degradation mechanism (Pearce Sher- <br /> sate loss RPM2. Unlike mammalian cells, lower      man 1997). likely proteins <br /> eukaryotes, including S. cerevisiae, degrade long-     destined mitochondria regulated protea- <br /> lived proteins ubiquitin-proteasome path-         activity mitochondrial import. <br /> way (reviewed Goldberg et al. 1997).  pre4-2         direct turnover mitochondria utilizing path- <br /> ump1-2 strains accumulate short-lived proteins,        way balancing proteasomal vacuolar degradation <br /> including regulatory proteins known profound          activities provide cells mechanism mon- <br /> effects cell growth. Consistent interpreta-      itoring removing unneeded improperly function- <br /> tion observation null allele RPM2      ing mitochondria (Campbell Thorsness 1998; suppressed multiple copies YBL066,        Sutovsky et al. 1998). hypothesis mitochondrial <br /> encodes putative transcription factor, PAB1,      turnover involves ubiquitin-tagged proteins supported <br /> encodes poly  -binding protein (Groom et al.            studies ubiquitin localization. Ubiquitin conjugates <br /> 1998; H. C. Heyman N. C. Martin, personal com-            ubiquitin-conjugating enzymes localized <br /> munication). genes potential        mitochondrial membrane fractions mito- <br /> increase steady-state level certain proteins,    chondrial outer membrane (Zhaung McCauley turn, suppress RPM2 mutants.                         1989; Magnani et al. 1991). Fisk Yaffe (1999) dem- <br />    Proteasome degradation outer mitochondrial              onstrated loss E3 ubiquitin ligase affects <br /> membrane proteins act general       mitochondrial inheritance. mechanism protein mitochondrial maintenance similar described        ubiquitination cytosolic mitochondria ER membrane proteins, insuring improperly        useful regulated degradation mitochon- <br /> folded proteins misassembled complexes              drial proteins proteasome targeted turn- <br /> graded rapidly (Kopito 1997; Plemper Wolf                 mitochondria vacuole. <br /> 1999). Essential functions mitochondria depend           Cell fractionation experiments previously shown outer mitochondrial membrane proteins include im-          detectable Rpm2p localized mitochondrial <br /> port, fusion, distribution. Decreased proteasome          fraction (Kassenbrock et al. 1995). Rpm2p, compo- <br /> activity impact mitochondrial function          nent mitochondrial RNase P activity, localized increasing longevity mitochondrially associated        matrix.  presence low levels proteins, potentially suppressing lethal phenotypes        Rpm2p mitochondrial fractions cyto- <br /> strains containing mutations adverse proper func-         sol ruled  limited growth  1022                                                M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br /> small &#226;&#338;&#172;rpm2 colonies suggests proteasome target                         Structure 20S proteasome yeast 2.4 resolution. Nature <br />                                                                                386: 463&#226;&#8364;&#8220;471. <br /> affecting mitochondrial function, mitochondria                          Groom, K. R., 1995 Genetic approaches yield new insights  diluted 25 generations. spe-                      functional expression biogenesis Rpm1r Rpm2p <br /> cific role Rpm2p cell growth remains                              subunits yeast mitochondrial RNase P. Thesis, University                                                                                Louisville, Louisville. <br /> known clearly demonstrated decreased pro-                      Groom, K. R., H. C. Heyman, M. C. Steffen, L. Hawkins N. C. <br /> teasome function provide resources necessary                           Martin, 1998 Kluyveromyces lactis SEF1 Saccharomyces cere- absence RPM2. results suggest Rpm2p                         visiae homologue bypass unknown essential function,                                                                                mitochondrial RNase P function, S. cerevisiae RPM2 role regulating synthesis turnover                          gene. Yeast 14: 77&#226;&#8364;&#8220;87. <br /> proteins important growth. role                         Haas,  L., 1988 Immunochemical probes ubiquitin pool dynam- <br /> passed novel proteasome mutants character-                          ics, pp. 173&#226;&#8364;&#8220;206 Ubiquitin, edited M. Rechsteiner. Plenum <br />                                                                                Press, Salt Lake City. <br /> ized                                                                  Haas,  L., P. M. Bright, 1985 immunochemical detec- <br />   thank Ms. Clarissa Moxely Mr. Michael Mindrum techni-             tion quantitation intracellular ubiquitin-protein conju- <br /> cal assistance complementation assays library screens.            gates. J. Biol. Chem. 260: 12464&#226;&#8364;&#8220;12473. <br />                                                                            Heinemeyer, W., M. Fischer, T. Krimmer, U. Stachon D. H. thank Dr.  Haas generous donation affinity purified <br />                                                                                Wolf, 1997 active sites eukaryotic 20S proteasome <br /> polyclonal antiubiquitin antibody Dr. M. Walberg gener-            involvement subunit precursor processing. J. Biol. <br /> ous donation YMW1 YMW2 yeast strains. work sup-                Chem. 272: 25200&#226;&#8364;&#8220;25209. <br /> ported National Institutes Health grant 992332 N.C.M.             Hershko, , 1997 Roles ubiquitin-mediated proteolysis cell <br />                                                                                cycle control. Curr. Opin. Cell Biol. 9: 788&#226;&#8364;&#8220;799. <br />                                                                            Hill, K., K. Model, M. T. Ryan, K. Dietmeier, F. Martin et al., <br />                                                                                1998 Tom40 forms hydrophilic channel mitochon- <br />                       LITERATURE CITED                                         drial import pore preproteins. Nature 395: 516&#226;&#8364;&#8220;521. <br />                                                                            Hilt, W., D. H. Wolf, 1996 Proteasomes: destruction pro- <br /> Altschul, S. F., W. Gish, W. Miller, E. W. Myers D. J. Lipman,             gramme. Trends Biochem. Sci. 21: 96&#226;&#8364;&#8220;102. <br />     1990 Basic local alignment search tool. J. Mol. Biol. 215: 403&#226;&#8364;&#8220;        Hilt, W., C. Enenkel,  Gruhler, T. Singer D. H. Wolf, 1993 <br />     410.                                                                       PRE4 gene codes subunit yeast proteasome <br /> Arendt, C. S., M. Hochstrasser, 1997 Identification                 necessary peptidylglutamyl-<span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>-hydrolyzing activity. Muta- <br />     yeast 20S proteasome catalytic centers subunit interactions            tions link proteasome stress- ubiquitin-dependent pro- <br />     required active-site formation. Proc. Natl. Acad. Sci. USA 94:         teolysis. J. Biol. Chem. 268: 3479&#226;&#8364;&#8220;3486. <br />     7156&#226;&#8364;&#8220;7161.                                                             Kaiser, C., S. Michaelis  Mitchell, 1994 Methods Yeast <br /> Attardi, G., G. Schatz, 1988 Biogenesis mitochondria.                   Genetics. Cold Spring Harbor Laboratory Press, Cold Spring Har- <br />     Annu. Rev. Cell Biol. 4: 289&#226;&#8364;&#8220;333.                                          bor, NY. <br /> Campbell, C. L., P. E. Thorsness, 1998 Escape mitochon-             Kassenbrock, C. K., G. J. Gao, K. R. Groom, P. Sulo, M. G. Douglas <br />     drial DNA nucleus yme1 yeast mediated vacuolar-            et al., 1995 RPM2, independently mitochondrial RNase P <br />     dependent turnover abnormal mitochondrial compartments.                 function, suppresses ISP42 mutant defective mitochondrial <br />     J. Cell Sci. 111: 2455&#226;&#8364;&#8220;2464.                                               import essential normal growth. Mol. Cell. Biol. 15: <br /> Campbell, C. L., N. Tanaka, K. H. White P. E. Thorsness, 1994              4763&#226;&#8364;&#8220;4770. <br />     Mitochondrial morphological functional defects yeast            King, R. W., R. J. Deshaies, J. M. Peters M. W. Kirschner, 1996 <br />     caused yme1 suppressed mutation 26S protease                proteolysis drives cell cycle. Science 274: 1652&#226;&#8364;&#8220;1659. <br />     subunit homologue. Mol. Biol. Cell 5: 899&#226;&#8364;&#8220;905.                         Kopito, R. R., 1997 ER quality control: cytoplasmic connection. <br /> Chen, D. C., B. C. Yang T. T. Kuo, 1992 step transforma-               Cell 88: 427&#226;&#8364;&#8220;430. <br />     tion yeast stationary phase. Curr. Genet. 21: 83&#226;&#8364;&#8220;84.             Kopp, F., K. B. Hendil, B. Dahlmann, P. Kristensen,  Sobek et <br /> Chen, P., M. Hochstrasser, 1995 Biogenesis, structure                  al., 1997 Subunit arrangement human 20S proteasome. <br />     function yeast 20S proteasome. EMBO J. 14: 2620&#226;&#8364;&#8220;2630.               Proc. Natl. Acad. Sci. USA 94: 2939&#226;&#8364;&#8220;2944. <br /> Chen, P., M. Hochstrasser, 1996 Autocatalytic subunit pro-             Leonhard, K., J. M. Herrmann, R.  Stuart, G. Mannhaupt, W. <br />     cessing couples active site formation 20S proteasome             Neupert et al., 1996 AAA proteases catalytic sites oppo- <br />     completion assembly. Cell 86: 961&#226;&#8364;&#8220;972.                                  site membrane surfaces comprise proteolytic Chun, K. T., N. Mathias M. G. Goebl, 1996 Ubiquitin-depen-                 ATP-dependent degradation inner membrane proteins mi- <br />     dent proteolysis cell cycle control yeast. Prog. Cell Cycle         tochondria. EMBO J. 15: 4218&#226;&#8364;&#8220;4229. <br />     Res. 2: 115&#226;&#8364;&#8220;127.                                                       Magnani, M., G. Serafini,  Antonelli, M. Malatesta G. <br /> Ciechanover, ,  L. Schwartz, 1998 ubiquitin-protea-                Gazzanelli, 1991 Evidence particulate location ubiqui- <br />     pathway: complexity myriad functions protein               tin conjugates ubiquitin-conjugating enzymes rabbit brain. <br />     death. Proc. Natl. Acad. Sci. USA 95: 2727&#226;&#8364;&#8220;2730.                           J. Biol. Chem. 266: 21018&#226;&#8364;&#8220;21024. <br /> Dang, Y. L., N. C. Martin, 1993 Yeast mitochondrial RNase P.           Moczko, M., K. Dietmeier, T. Sollner, B. Segui, H. F. Steger et <br />     Sequence RPM2 gene demonstration product            al., 1992 Identification mitochondrial receptor complex <br />     protein subunit enzyme. J. Biol. Chem. 268: 19791&#226;&#8364;&#8220;19796.          Saccharomyces cerevisiae. FEBS Lett. 310: 265&#226;&#8364;&#8220;268. Pinto, B., S. B. Malladi N. Altamura, 1999 MitBASE pilot:           Morales, M. J., Y. L. Dang, Y. C. Lou, P. Sulo N. C. Martin, <br />     database nuclear genes involved mitochondrial biogenesis           1992 105-kDa protein required yeast mitochondrial <br />     regulation Saccharomyces cerevisiae. Nucleic Acids Res.         RNase P activity. Proc. Natl. Acad. Sci. USA 89: 9875&#226;&#8364;&#8220;9879. <br />     27: 147&#226;&#8364;&#8220;149.                                                           Myers,  M., L. K. Pape  Tzagoloff, 1985 Mitochondrial <br /> Fisk, H. , M. P. Yaffe, 1999 role ubiquitination                protein synthesis required maintenance intact mitochon- <br />     mitochondrial inheritance Saccharomyces cerevisiae. J. Cell Biol.       drial genomes Saccharomyces cerevisiae. EMBO J. 4: 2087&#226;&#8364;&#8220;2092. <br />     145: 1199&#226;&#8364;&#8220;1208.                                                        Nandi, D., E. Woodward, D. B. Ginsburg J. J. Monaco, 1997 <br /> Glickman, M. H., D. M. Rubin, V.  Fried D. Finley, 1998              Intermediates formation mouse 20S proteasomes: impli- <br />     regulatory particle Saccharomyces cerevisiae proteasome. Mol.       cations assembly precursor beta subunits. EMBO J. 16: <br />     Cell. Biol. 18: 3149&#226;&#8364;&#8220;3162.                                                 5363&#226;&#8364;&#8220;5375. <br /> Goldberg,  L., T. N. Akopian,  F. Kisselev, D. H. Lee              Pearce, D. , F. Sherman, 1995 Degradation cytochrome <br />     M. Rohrwold, 1997 New insights mechanisms                     oxidase subunits mutants yeast lacking cytochrome c     importance proteasome intracellular protein degrada-             suppression degradation mutation yme1. J. Biol. <br />     tion. Biol. Chem. 378: 131&#226;&#8364;&#8220;140.                                            Chem. 270: 20879&#226;&#8364;&#8220;20882. <br /> Groll, M., L. Ditzel, J. Lowe, D. Stock, M. Bochtler et al., 1997          Pearce, D. , F. Sherman, 1997 Differential ubiquitin-depen- <br />                                                        Proteasome Mutants Suppress &#226;&#338;&#172;rpm2                                                      1023 <br />     dent degradation yeast apo- cytochrome c isozymes. J. Biol.      Sikorski, R. S., J. D. Boeke, 1991 vitro mutagenesis     Chem. 272: 31829&#226;&#8364;&#8220;31836.                                                     plasmid shuffling: cloned gene mutant yeast. Methods <br /> Plemper, R. K., D. H. Wolf, 1999 Retrograde protein transloca-              Enzymol. 194: 302&#226;&#8364;&#8220;318. <br />     tion: ERADication secretory proteins health disease.          Sikorski, R. S., P. Hieter, 1989 shuttle vectors     Trends Biochem. Sci. 24: 266&#226;&#8364;&#8220;270.                                           yeast host strains designed efficient manipulation DNA Pon, L., G. Schatz, 1991 Biogenesis yeast mitochondria, pp.              Saccharomyces cerevisiae. Genetics 122: 19&#226;&#8364;&#8220;27. <br />     333&#226;&#8364;&#8220;406 Molecular Cellular Biology Yeast Saccharo-    Smyth, K. , J. M. Belote, 1999 dominant temperature- <br />     myces: Genome Dynamics, Protein Synthesis, Energetics, edited        sensitive lethal DTS7 Drosophila melanogaster encodes altered <br />     J. R. Broach, J. R. Pringle E. W. Jones. Cold Spring Harbor             20S proteasome beta-type subunit. Genetics 151: 211&#226;&#8364;&#8220;220. <br />     Press, Cold Spring Harbor, NY.                                          Stribinskis, V., G. J. Gao, P. Sulo, Y. L. Dang N. C. Martin, <br /> Ramos, P. C., J. Hockendorff, E. S. Johnson,  Varshavsky                  1996 Yeast mitochondrial RNase P RNA synthesis altered     R. J. Dohmen, 1998 Ump1p required proper maturation                  RNase P protein subunit mutant: insights biogenesis <br />     20S proteasome substrate comple-                mitochondrial RNA-processing enzyme. Mol. Cell. Biol. 16: <br />     tion assembly. Cell 92: 489&#226;&#8364;&#8220;499.                                     3429&#226;&#8364;&#8220;3436. <br /> Rapaport, D., W. Neupert R. Lill, 1997 Mitochondrial protein            Sutovsky, P., R. Moreno G. Schatten, 1998 Sperm mitochon- <br />     import. Tom40 plays major role targeting translocation             drial ubiquitination model explaining strictly maternal <br />     preproteins forming specific binding site prese-            mtDNA inheritance mammals. Mol. Biol. Cell 9: 309a. <br />     quence. J. Biol. Chem. 272: 18725&#226;&#8364;&#8220;18731.                                Thorsness, P. E., K. H. White T. D. Fox, 1993 Inactivation Rinaldi, T., S. Francisci, E. Sennaro, L. Frontali M. <br /> </body></html>