<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Nuclear Export 60S Ribosomal Subunits Depends Xpo1p     Requires Nuclear Export Sequence-Containing Factor, Nmd3p, <br />         Associates Large Subunit Protein Rpl10p <br />  <br />  <br />            Nuclear export ribosomes requires subset nucleoporins Ran  specific transport <br />          factors identified. Using large subunit reporter (Rpl25p-eGFP), isolated          temperature-sensitive ribosomal export (rix) mutants. corresponds ribosomal protein <br />          Rpl10p, interacts directly Nmd3p, conserved essential protein associated 60S subunits. <br />          thermosensitive nmd3 mutants impaired large subunit export. Strikingly, Nmd3p shuttles <br />          nucleus cytoplasm exported nuclear export receptor Xpo1p.           export 60S subunits Xpo1p dependent. conclude nuclear export 60S subunits requires          nuclear export sequence-containing nonribosomal protein Nmd3p, directly binds large subunit <br />          protein Rpl10p. <br />  <br />  <br />    steps ribosome synthesis place nucleolus,            tors (6). reported vivo assay ribosomal specialized subnuclear region. process starts               export yeast uses fusion green fluorescent <br /> synthesis pre-rRNA transcripts, 35S pre-5S rRNA,                 protein (GFP) ribosomal protein Rpl25p (15). Rpl25p processed base modified yield mature                   imported nucleus assembles ribosomes 25S/28S, 18S, 5.8S, 5S rRNAs, respectively (18).                 direct binding rRNA inside nucleolus (39). Passage processes 80 ribosomal proteins assemble               free Rpl25p-GFP preribosomal particles <br /> rRNAs yield preribosomal particles, exported              nucleoplasm appears rapid wild-type cells, cytoplasm (41). contrast pre-rRNA processing                  GFP-labeled ribosomes detected fluorescence mi- <br /> modification, little known assembly pathway               croscopy cytoplasm. Mutations causing defects sub- eukaryotic ribosomal subunits features make                 unit export lead readily detectable nucleoplasmic GFP competent nuclear exit  recent reviews, ref-               signal. assay showed subset nucleoporins erences 18 40).                                                         Ran required 60S subunit export. assay <br />    transport macromolecules nuclear pores                nuclear export 40S subunits led similar conclu- thought involve facilitated diffusion soluble transport            sions (26). <br /> factors repeat sequences nuclear pore proteins                 exploited 60S export assay screen bank (nucleoporins) form line nuclear pore complex.                 randomly generated temperature-sensitive (ts) mutants Directionality transport provided small GTPase                 ribosomal export defect (rix) phenotype. report Ran, presence step RanGTP/RanGDP gradient                   characterization rix5-1 mutant, complemented nuclear membrane  review, reference 23).               RPL10 gene  called QSR1 GRC5). gene <br /> RanGTP binds high affinity nuclear import export                encodes ribosomal protein Rpl10p (21),  <br /> receptors (importins exportins, respectively) karyo-             ported assemble late 60S subunits. Rpl10p pherin &#226;&#65533;&#164; superfamily (10). nuclear exit, export cargoes,                reported interact Nmd3p, nonribosomal protein harbor nuclear export sequences (NESs) (e.g., leucine-                associates 60S subunits (3, 12, 13, 16), analyzed wheth- <br /> rich NESs), form intranuclear complex NES                    er Nmd3p plays role nuclear export ribosomes. ceptor Xpo1p/Crm1 presence RanGTP (8, 33).                   nmd3 ts mutants accumulate large subunit reporter <br /> trimeric complex exported nucleus                                                                             Rpl25-eGFP inside nucleus restrictive temperature. <br /> cytoplasm. <br />                                                                             Interestingly, Nmd3p C-terminal NES domain,    Saccharomyces cerevisiae useful                                                                             uses Xpo1p nuclear exit. large subunit reporter <br /> analysis nuclear pore complex transport fac- <br />                                                                             Rpl25-eGFP accumulates inside nucleus Xpo1p                                                                             inhibited, Nmd3p Xpo1p likely factors involved <br />                                                                             nuclear export 60S subunits. Recently, Ho et al. reported <br />    * Corresponding author. Mailing address: Biochemie-Zentrum Hei- <br /> delberg, Im Neuenheimer Feld 328, D-69120 Heidelberg, Germany. <br />                                                                             similar findings dominant negative allele nmd3 im- <br /> Phone: 49-(0)6221-54-4173. Fax: 49-(0)6221-54-4369. E-mail: cg5@ix          pairs 60S subunit export Nmd3p acts adapter .urz.uni-heidelberg                                                      Xpo1p-mediated nuclear export large subunits (14). <br />  <br />                                                                      3405 <br />  3406       GADAL ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />                                                                    TABLE 1. Yeast strains used <br />        Strain                                                                Genotype                                                             Reference origin <br />  <br /> RS453                   MATa/&#226;&#65533;&#163; ade2/ade2 leu2/leu2 ura3/ura3 his3/his3 trp1/trp1                                                             15 <br /> RS453a                  MATa ade2 leu2 ura3 his3 trp1                                                                                        15 <br /> FY23                    MATa ura3 trp1 leu2                                                                                                  Derived    S288C <br /> FY86                    MAT&#226;&#65533;&#163; ura3 his3 leu2                                                                                                  Derived    S288C <br /> rix5-1                  MAT&#226;&#65533;&#163; ura3 his3 leu2 rix5-1                                                                                           Isolated   ts collec- <br />                                                                                                                                                tion (2) <br /> RPL25-shuffle           MAT&#226;&#65533;&#163; ade2 leu2 ura3 his3 trp1 rpl25&#226;&#338;&#172;::HIS3 &#226;&#171;&#185; pRS316-RPL25 (ARS/CEN RPL25 URA3)                                       Derived    RS453 <br /> Rpl25p-GFP              MAT&#226;&#65533;&#163; ade2 leu2 ura3 his3 trp1 rpl25&#226;&#338;&#172;::HIS3 &#226;&#171;&#185; YEplac195ADE-Rpl25p-GFP (2&#226;&#65533;&#174;m URA3 ADE2                                  15 <br />                          RPL25-GFP) <br /> Rpl25p-eGFP             MAT&#226;&#65533;&#163; ade2 leu2 ura3 his3 trp1 rpl25&#226;&#338;&#172;::HIS3 &#226;&#171;&#185; pASZ11-Rpl25p-eGFP (ARS/CEN ADE2 RPL25-eGFP)                            study <br /> NMD3-shuffle            MATa his3 leu2 lys2 ura3 nmd3::kanMX4 (ARS/CEN URA3 NMD3)                                                            study <br /> nmd3-2                  MATa his3 leu2 lys2 ura3 nmd3::kanMX4 (ARS/CEN LEU2 nmd3-2)                                                          study <br /> BY4743                  MATa/&#226;&#65533;&#163; his3&#226;&#338;&#172;1/his3&#226;&#338;&#172;1 leu2&#226;&#338;&#172;0 leu2&#226;&#338;&#172;0 lys2&#226;&#338;&#172;0/LYS2 MET15&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 ura3&#226;&#338;&#172;0 nmd3::kanMX4/NMD3                               EUROSCARF <br /> kap123&#226;&#338;&#172;                 MAT&#226;&#65533;&#163; ura3 leu2 his3 kap123&#226;&#338;&#172;::HIS3                                                                                    29 <br /> JWY1457                 MAT&#226;&#65533;&#163; can1 ade2 his3 lys2 trp1 leu2 ura3 rpl16a&#226;&#338;&#172;::TRP1 rpl16b&#226;&#338;&#172;::LEU2 &#226;&#171;&#185;YCp50-RPL16b (ARS/CEN                           25 <br />                          URA3 RPL16b) <br /> JWY1460                 MAT&#226;&#65533;&#163; can1 ade2 his3 lys2 trp1 leu2 ura3 rpl16a&#226;&#338;&#172;::TRP1 rpl16b&#226;&#338;&#172;::LEU2 &#226;&#171;&#185;pHIS3CEN3-Rplb16b-2                             25 <br />                          (ARS/CEN HIS3 rpl16b-2) <br /> MNY8 (xpo1 LMB          MATa leu2 hs3 trp1 ura3 &#226;&#338;&#172;crm1::KAN (ARS/CEN LEU2 pDC-CRM1T539C)                                                      27 <br />    sensitive) <br /> rrp44-1                 MATa ura3 lys2 ade2 trp1 his3 leu2/rrp44-1                                                                           Mitchell et al., submit- <br />                                                                                                                                               ted publication <br /> rsa1&#226;&#338;&#172;                   MATa his3 leu2 met15 ura3 rsa1&#226;&#338;&#172;::kanMX4                                                                              EUROSCARF <br /> MNY7 (xpo1 LMB          MATa leu2 hs3 trp1 ura3 &#226;&#338;&#172;crm1::KANr (ARS/CEN LEU2 pDC-CRM1) <br />    resistant) <br />  <br />  <br />  <br />                         MATERIALS METHODS                                           generated fragment cut SphI cloned SphI site pUN100- <br />                                                                                         ProtA-NOP1 (4), replacing protein (ProtA) DsRed. pRS314-DsRed- <br />    Yeast strains, DNA recombinant work, microbiological techniques. Yeast           NOP1 generated cloning 2,323-bp-long SacI-SnaBI fragment strains used study described Table 1. Microbiological techniques,        pUN100-DsRed-NOP1 SacI-SmaI site pRS314. <br /> plasmid transformation recovery, mating, sporulation diploids, tetrad           Isolation rix mutants. collection 900 thermosensitive yeast mutants <br /> analysis essentially described (30). DNA recombinant             (2, 11) derived haploid strains FY23 FY86 used screen work performed described reference 22.                                        previously generated laboratory C. Cole (Dartmouth Medical School, <br />    Plasmid constructions. Plasmid YEplac195-ADE2-L25NLS-GFP derived <br />                                                                                         Hanover, N.H.). collection kept 96 plates &#226;&#171;&#186;80&#194;&#176;C obtained YEplac195-ADE2-URA3-RPL25-GFP removing RPL25 <br />                                                                                         Ann Mutvei (University Ume&#195;&#165;, Ume&#195;&#165;, Sweden). Mutants plated coding sequence, leaving 5&#226;&#172;&#732; end encodes 52 residues Rpl25p. <br />                                                                                         YPD (yeast-peptone-dextrose) medium, transformed YEplac 195-ADE2- <br /> YEplac195-ADE2-Rpl25p-GFP previously described (15). Rpl25p- <br />                                                                                         URA3-RPL25-GFP, plated SDC-Ura 5 days, replica plated GFP reporter improved, derived Rpl25p-eGFP (enhanced GFP) <br />                                                                                         YPD plates, incubated 37&#194;&#176;C. 2 h overnight incubation 37&#194;&#176;C, <br /> construct complements rpl25&#226;&#338;&#172;::HIS3 disruption temperatures tested, <br />                                                                                         plate transferred 23&#194;&#176;C 1 h. Cells mounted slide including 37&#194;&#176;C. new Rpl25p-eGFP construct tested nucleoporin                                                                                         inspected fluorescence microscopy. wild-type strain showed <br /> Ran cycle mutants shown behave similar original Rpl25p-GFP <br />                                                                                         Rpl25p-GFP cytoplasmic staining control mutants (rna1-1, prp20-1 construct nuclear accumulation directly seen restrictive <br />                                                                                         nup49-313) showed nuclear accumulation Rpl25p-GFP, phenotype temperature, requiring short regrowth phase permissive tem- <br />                                                                                         ts mutants investigated. Mutants accumulated Rpl25p-GFP distinct <br /> perature. pASZ11-RPl25p-eGFP construct generated steps. <br />                                                                                         regions rechecked procedure subsequent DNA 3&#226;&#172;&#732; untranslated region RPS20 amplified PCR yeast genomic <br />                                                                                         staining fixation. Cells resuspended water containing Hoechst <br /> DNA using primers CAA CGG ATC CTA AGC TGG TTC TAA CTG GAA <br /> ATA TTT TGT CGA CAC ATT GCC TTA TTC AAA GCC GCC.                             H33342 (50 &#226;&#65533;&#174;g/ml) 20 min room temperature continuous shaking, <br /> 386-bp-long fragment generated cut BamHI-SalI cloned                  washed 1 ml water, mounted slide, inspected fluores- <br /> pASZ11 site, generating pASZ11-3&#226;&#172;&#732;-RPS20. eGFP ampli-                cence microscopy. 35 mutants, Rpl25p-GFP accumulation colocalizes fied PCR pEGFP-1 vector (Clontech, Palo Alto, Calif.) using primers           DNA staining. 35 mutants, old reporter construct lost AAA GGA TCC ATG GTG AGC AAG GGC G TTT TTT GTC GAC TTA                               pRS315-RPL25-eGFP introduced transformation selected SDC- <br /> AGA TCT CTT GTA CAG CTC GTC CAT GCC. 745-bp-long generated                          Leu medium. Individual transformants picked, grown liquid SDC-Leu <br /> fragment cut BamHI-SalI cloned pBluescript generate              medium 23&#194;&#176;C optical density 600 nm (OD600) 1, pBS-eGFP vector, sequence eGFP coding region confirmed               shifted 5 h 37&#194;&#176;C liquid YPD medium. centrifugation, cells DNA sequencing. BamHI-BglII fragment pBS-eGFP containing                  resuspended water, mounted slide, viewed fluorescence micro- <br /> eGFP cloned correct orientation BamHI site pASZ11-             scope. mutants showing 50% accumulation phenotype 3&#226;&#172;&#732;-RPS20. RPL25 including authentic RPL25 5&#226;&#172;&#732; untranslated region                accumulation reinduction growth permissive temperature <br /> amplified PCR yeast genomic DNA using primers TAG GAT CCA                     kept. 20 mutants, 50% cells nuclear accu- <br /> ATG TAA CCG ATT CTG TTA GCA ATG TCC TTC CCG GGC AAT                                 mulation Rpl25p-eGFP condition. Accordingly, mutants GAA GAG ATG AGG AGG CAT GG. 1,293-bp-long generated fragment                        defined rix mutants. cut XmaI-BamHI ligated pASZ11-eGFP-3&#226;&#172;&#732;-RPS20, generating                  Cloning RIX5/RPL10. yeast genomic library LEU2-containing ARS/ pASZ11-RPL25-eGFP. open reading frame sequence RPL25                     CEN plasmid transformed rix5-1 strain. colonies growing verified DNA sequencing. Plasmids pRS314-RPL25-eGFP, pRS315-RPL25-                   restrictive temperature (37&#194;&#176;C), plasmids genomic inserts isolated. <br /> eGFP, pRS316-RPL25-eGFP obtained subcloning XmaI-SalI                   Complementing plasmids contained RPL10 gene. pRS315-RPL10 <br /> 2,378-bp-long fragment derived pASZ11-RPL25-eGFP XmaI-SalI                harboring RPL10 gene generated shown complement sites  respectively, pRS314, pRS315, pRS316.                                     thermosensitive growth defect rix5-1 mutant. <br />    DsRed (red fluorescent protein) amplified PCR plasmid pDsRed1-                Generation NMD3-shuffle nmd3 mutant strains. NMD3 isolated N1 (Clontech) using primers AAA AAA GCA TGC GGA TCC ATG GTG                         PCR yeast genomic DNA. haploid NMD3-shuffle strain constructed <br /> CGC TCC TCC AAG AAC GTC TTT TTT GTC GAC CTA GCA TGC                               heterozygous nmd3::kanMX4/NMD3 diploid. nmd3 ts mutants CAG GAA CAG GTG GTG GCG GCC CTC G. derived 717-bp-long                              generated random PCR-mediated mutagenesis described (35). <br />  VOL. 21, 2001                                                          Nmd3p ADAPTER Xpo1p-DEPENDENT EXPORT 60S                      3407 <br />  <br />  <br /> nmd3-2 contains mutations M141V, D255V, F318L, I328M, K413T, E440G,             tionation high salt, Rpl25p-eGFP detected F449L, N464S, E479D, D498G, D501G. Tagging Nmd3p GFP                60S ribosomal subunit fraction (data shown). Im- <br /> performed exchanging RPL25 pRS315-RPL25-eGFP NMD3 <br /> gene including authentic promoter. C-terminal deletions NMD3                                                                                     portant analysis ts mutants, Rpl25p-eGFP <br /> generated using NMD3 promoter, lacking 44 (&#226;&#338;&#172;NES1), 79          fluorescence signal readily detected higher tempera- <br /> (&#226;&#338;&#172;NES1/2), 116 (&#226;&#338;&#172;NES1/2 &#226;&#338;&#172;NLS) C-terminal residues. protein kinase           tures (e.g., 37&#194;&#176;C), nuclear accumulation Rpl25p-eGFP <br /> inhibitor (PKI)-NES attached Nmd3p&#226;&#338;&#172;NES <span id='am-13' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-14' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> sequence               seen restrictive temperature nucleoporin Ran <br /> NELALKLAGLDINKT. nuclear localization signal (NLS) derived                                                                                     cycle mutants (data shown). <br /> Npl3p fused GFP described reference 34; NES domain Nmd3p <br /> corresponding residues 440 518 attached reporter construct.          35 rix mutants rescreened using new Rpl25p- <br />    Expression Nmd3p Rpl10p Escherichia coli. NMD3 cloned         eGFP construct. strains exhibited nuclear accumula- pET24d vector, harboring kanamycin resistance marker allowing           tion Rpl25p-eGFP restrictive temperature (Fig. 1A). <br /> C-terminal tagging NMD3 histidines. NMD3 amplified             defects manifested 30 min 5 h shift <br /> genomic DNA primers 5&#226;&#172;&#732; aaa aaa gct agc ATG GAA TTC ACA CCT <br /> ATA GA 3&#226;&#172;&#732; ttt ttt ttg cgg ccg cCT GCT GAG ATT CAA CGG GTG                                                                                      restrictive temperature, depending rix mutant. <br /> lowercase letters indicate restriction site) inserting pET-24d   Notably, patterns nuclear accumulation differed (Novagen), cut NheI NotI. pGEX-4T-3 (Amersham Phar-              various rix mutants, independent Rpl25p-GFP reporter <br /> macia Biotech) harboring ampicillin resistance marker used gluta-       used, rix1-1 rix5-1 mutants showed generalized nuclear <br /> thione S-transferase (GST) expression. pHFF17, harbors ampi- <br />                                                                                     accumulation, observed nucleoporin mutants, cillin resistance marker, contains GST-RPL10 fusion construct (38). coexpression, E. coli BL21-codon plus-RIL strain (Stratagene) trans-        rix2-1, rix3-1, rix7-1 mutants exhibited stronger accumu- <br /> formed   pHFF17, (ii) pHFF17 plus pET24d-NMD3, (iii) pGEX-4T-3            lation nucleolus nucleoplasm (Fig. 1A). <br /> plus pET24d-NMD3. inoculated 0.5 liter minimal medium plus                rix5-1 mutation alters ribosomal protein Rpl10p. ampicillin-kanamycin 1/100 overnight culture grown OD600       mutant designated rix5-1, Rpl25p-eGFP detected <br /> 0.6 16&#194;&#176;C; recombinant proteins induced 6 h 16&#194;&#176;C 0.1 mM <br /> isopropyl-&#226;&#65533;&#164;-D-thiogalactopyranoside. Purification GST-tagged proteins                                                                                     nucleoplasm 30 60-min shift described (20). E. coli cell pellet resuspended MU     restrictive temperature (Fig. 1B), indicating export buffer (0.5% Tween 20, 100 mM potassium acetate, 20 mM HEPES [pH 7.0], 2            60S subunit blocked predominantly nuclear stage. mM magnesium acetate, 10% glycerol, 2.5 mM dithiothreitol), cells          structure nucleolus checked rix5-1 strain  <br /> lysed sonication. supernatant incubated 100 &#226;&#65533;&#174;l <span id='am-1' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span>glutathione</span>- <br />                                                                                     sure apparent nucleoplasmic accumulation Rpl25p- <br /> Sepharose 4B (Amersham Pharmacia Biotech), beads washed MU <br /> buffer, proteins bound beads eluted sample buffer.      eGFP enlargement fragmentation detect Rpl10p Nmd3p Western blotting, QSR1-specific antibody (&#226;&#65533;&#163;-             nucleolus. restrictive temperature, Rpl25p-eGFP accu- <br /> QSR1) monoclonal antibody (&#226;&#65533;&#163;  (BAbCo, Richmond, Calif.), respec-            mulated entire nucleus DsRed-Nop1p <br /> tively, used.                                                                   nucleolar marker) exhibited typical crescent staining    Miscellaneous. Sodium dodecyl sulfate-polyacrylamide gel electrophoresis <br /> (SDS-PAGE) Western blot analysis performed described                                                                                     nucleolus (data shown), suggesting nucleolus <br /> (32); isolation ribosomes low- high-salt conditions sucrose         remains intact rix5-1 cells. characterize pu- <br /> gradient centrifugation described reference 37. cell lysates   tative ribosomal subunit export factor, wild-type RIX5 gene fractions sucrose gradient separated SDS-PAGE              cloned complementation ts phenotype rix5- <br /> analyzed Western blotting using indicated antibodies. Pulse-chase labeling <br />                                                                                     1 cells. Unexpectedly, RIX5 corresponds rRNA analysis rRNA processing Northern hybridization performed described previously (36, 37). GFP-tagged fusion proteins         RPL10, gene encoding essential large ribosomal subunit <br /> detected vivo Zeiss Axioskop fluorescence microscope, pictures     protein Rpl10p. Accordingly, rix5-1 mutant called <br /> obtained Xillix Microimager charge-coupled device camera. leptomy-       rpl10-1 (Fig. 1B). Consistent findings, number cin B (LMB) treatment LMB-sensitive (CRM1-T539C) LMB-resis-              qsr1 mutant alleles RPL10, previously iden- <br /> tant (CRM1) S. cerevisiae strains, 100 ng LMB/ml added culture <br /> medium (27). <br />                                                                                     tified (7), impaired nuclear export Rpl25p-eGFP <br />                                                                                     (e.g., qsr1-2 [data shown]). tested ts mutation                                                                                     essential 60S subunit protein, Rpl16p (25)                                  RESULTS <br />                                                                                     detect nuclear accumulation Rpl25p-eGFP  <br />    Isolation rix mutants. identify factors required                       strictive temperature (data shown). anticipated, ribosomal export, GFP tagged 60S ribosomal subunit                             rix5-1 mutant, shifted restrictive temperature, ex- <br /> protein (Rpl25p-GFP) used fluorescence-based                            hibits half-mer polysome profile like seen previ- <br /> assay monitor large subunit export. screened                           ously characterized qsr1 mutants (data shown). <br /> bank 900 randomly generated ts mutants rix phenotype.                         earlier studies, Rpl10p reported associate 60S analyzed mutants nuclear accumulation                               ribosomal subunits cytoplasm, functions Rpl25p-GFP restrictive temperature followed short                       mediate joining 60S 40S subunits (7, 19).  <br /> regrowth phase permissive temperature (15); 35 rix mu-                       Rpl10p nuclear function  indicated tants identified assay.                                                observed defect ribosome export rpl10 ts mutants.    Rpl25p-GFP construct did fully complement                          indication nuclear function Rpl10p, tested rpl25&#226;&#338;&#172;::HIS3 disruption mutant, fluorescence sig-                       nucleolar pre-rRNA processing defective rpl10 ts <br /> nal Rpl25p-GFP decreased cells incubated                            mutants. Northern hybridization showed rapid inhibition higher temperatures (e.g., 37&#194;&#176;C; reference 15),                         pre-rRNA processing following transfer rix5-1 cells 37&#194;&#176;C <br /> constructed modified Rpl25p reporter  Materials                           (data shown). Pre-rRNA processing assessed Methods). new construct, Rpl25p-eGFP, complemented                             pulse-chase labeling [methyl-3H]methionine (Fig. 1C). rpl25&#226;&#338;&#172;::HIS3 disruption temperatures tested, includ-                     Consistent Northern analysis rix5-1 cells, rpl10 ts <br /> ing 37&#194;&#176;C, apparent growth difference compared                          cells showed processing defects 35S pre-rRNA, wild-type strain (data shown). Following subcellular frac-                      retarding synthesis 27S pre-rRNAs. Production  3408      GADAL ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. genetic screen rix mutants identifies ribosomal protein Rpl10p.   rix mutants (rix1-1, rix2-1, rix3-1, rix5-1, rix7-1) isogenic wild-type RIX&#226;&#171;&#185; strain expressing Rpl25p-eGFP grown 23&#194;&#176;C shift 5 h 37&#194;&#176;C inspection fluorescence <br /> microscopy. (B) rix5-1 complemented RPL10.  time course nuclear accumulation Rpl25p-eGFP rix5-1 (&#226;&#171;&#189;rpl10-1) RIX5&#226;&#171;&#185; <br /> (&#226;&#171;&#189;RPL10&#226;&#171;&#185;) cells shifted 1 2 h 37&#194;&#176;C;  growth curve rix5-1 cells rix-5 strain complemented cloned RIX5 (&#226;&#171;&#189;RPL10) 37&#194;&#176;C. (C) Pre-rRNA processing rpl10 ts mutant shifted 37&#194;&#176;C 6 h. rRNAs pulse-labeled 2 min [methyl-3H] methionine <br /> followed 5-min chase. rRNA analyzed autoradiography. (D) Rpl10p contains NLS N domain. length Rpl10p (residues <br /> 1 221), Rpl10p-NLS (residues 1 64), Rpl25p-NLS (residues 1 52), tagged GFP, analyzed KAP123&#226;&#171;&#185; kap123&#226;&#338;&#172; strains fluorescence microscopy. (E) Nuclear accumulation Rpl10p-eGFP Rpl25p-eGFP rrp44-1 wild-type (wt) cells grown 23&#194;&#176;C transfer 37&#194;&#176;C 5 h. <br />  <br />  <br />  <br /> 20S pre-rRNA delayed reduced, appearance                      nonribosomal proteins, Rsa1p Nmd3p, linked aberrant 23S rRNA.                                                    Rpl10p required 60S subunit export. Previous work <br />    Cytoplasmic Rpl10p contains NLS accumulates                     showed nonribosomal proteins Nmd3p Rsa1p nucleus exosome mutant. observation                       genetically linked Rpl10p (16, 17). Rsa1p nuclear <br /> Rpl10p required efficient intranuclear pre-rRNA pro-                  protein implicated facilitating association Rpl10p cessing suggested Rpl10p enters nucleus participate              60S ribosomal subunits (17). Nmd3p essential preribosomal assembly. looked NLS                  conserved protein located cytoplasm associ- <br /> Rpl10p. steady state, Rpl10p located cytoplasm                  ated 60S subunits (3, 13, 16). Rsa1p nonessential, (Fig. 1D).  64 amino acids                 rsa1 &#226;&#338;&#172; strains exhibited retarded growth, particularly 37&#194;&#176;C <br /> Rpl10p target GFP reporter nucleus (Fig. 1D).                   (Fig. 2A; reference 17). Expression Rpl25p-eGFP Kap123p import receptor ribosomal proteins (29)                rsa1 &#226;&#338;&#172; strain showed clear nucleoplasmic accumulation nuclear import Rpl10p(1&#226;&#8364;&#8220;64)-GFP, uptake Rpl25p                  37&#194;&#176;C (Fig. 2A). test Nmd3p functions nuclear <br /> (1&#226;&#8364;&#8220;52)-GFP strongly inhibited kap123&#226;&#338;&#172; mutant (Fig.                 export ribosomes, generated nmd3 ts mutants (Fig. 2B). <br /> 1D) karyopherin mutants tested                 Strikingly, large subunit reporter Rpl25p-eGFP signifi- <br /> (data shown).  analyzed Rpl10p-eGFP                      cantly accumulates inside nucleus nmd3-2 ts cells accumulates nucleus nucleocytoplasmic transport mu-                shift 37&#194;&#176;C (Fig. 2C). Nmd3p Rsa1p, observed tants rna1-1 nup49-313 accumulation                  Rpl10p, required nuclear export 60S ribosomal sub- <br /> (data shown). exosome mutant rrp44/dis3-1                    units. phenotype intranuclear 60S subunit accumula- <br /> does Rpl10p-eGFP exhibit distinct nuclear accumulation                  tion explain half-mer polysome phenotype (poly- restrictive temperature similar Rpl25p-eGFP (Fig. 1E).                somes containing unjoined 40S subunits initiation site) <br /> Rrp44p/Dis3p component exosome, large protein                    previously observed rsa1 (17) nmd3 mutants (16). <br /> complex involved processing degradation                      Recently, nmd3-4 allele tested defects 60S <br /> RNAs (24) including 5.8S rRNA rRNAs (1).                       subunit export, nuclear accumulation Rpl25p-GFP  Rpl25p-eGFP accumulates inside nucleus                    observed nmd3 ts mutant restrictive temper- classes transport mutants, nuclear accumulation                ature (14).  ts alleles nmd3 generated Rpl10p seen exosome mutant  Discus-                    laboratory rise nuclear accumulation Rpl25p- <br /> sion).                                                                       eGFP restrictive temperature (D. Strau&#195;&#376; O. Gadal, <br />  VOL. 21, 2001                                           Nmd3p ADAPTER Xpo1p-DEPENDENT EXPORT 60S                     3409 <br />  <br />  <br />  <br />  <br />                                                          FIG. 1&#226;&#8364;&#8221;Continued. <br />  <br />  <br />  <br /> unpublished data). reason difference clear,    60S peak high-salt conditions (Fig. 3B, right), Ho et al. used Rpl25p-GFP control GAL10         Nmd3p,  released 60S subunits high-salt <br /> promoter (13, 14), assay RPL125-GFP        treatment (Fig. 3B, right). conclude Nmd3p non- control promoter.                                   ribosomal protein loosely associates 60S subunits    Nmd3p binds Rpl10p associates 60S subunits       directly bind Rpl10p. salt-dependent manner. Genetic evidence suggested              Nmd3p contains NESs carboxy-terminal domain. <br /> RPL10 NMD3 functionally linked (16). test       results shown suggest Nmd3p enter <br /> Nmd3p directly associates Rpl10p, coexpressed GST-         nucleus. determine Nmd3p shuttles tagged Rpl10p tagged Nmd3p E. coli purified         nucleus cytoplasm, protein tagged eGFP. <br /> Rpl10p-GST <span id='am-20' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span>glutathione</span>-Sepharose beads. length        Nmd3p-eGFP, fully functional (Fig. 4B), located (65 kDa; Nmd3p-His6) prominent N-terminally cleaved          cytoplasm (Fig. 4C). Interestingly, deletion C <br /> (55 kDa; Nmd3p-His6*) Nmd3p copurify GST-Rpl10p               domain previously shown cause dominant-negative <br /> (Fig. 3A, lane 6). contrast, GST coexpressed         phenotype yeast overproduced strong GAL tagged Nmd3p does reveal copurification (Fig.         promoter (3). C-terminal domain Nmd3p contains 3A, lane 7). Furthermore, GST-Rpl10p purification ab-       conserved putative leucine-rich NESs (Fig. 4A). deleted <br /> sence tagged Nmd3p does Nmd3p copurifica-          NES-like sequences expressed truncated <br /> tion, E. coli contaminant approximately 64 kDa       NMD3 construct authentic promoter. case, evident (Fig. 3A, lane 5).  Nmd3p Rpl10p di-        dominant-lethal phenotype observed, strong nuclear <br /> rectly interact coexpressed E. coli (Fig. 3A). yeast    accumulation truncated Nmd3p proteins seen <br /> cells, Nmd3p associated exclusively free 60S subunits,     (Fig. 4C). Deleting NESs results apparent growth 80S ribosomes polysomes (Fig. 3B, left).        nmd3-shuffle strain 5-fluoroorotic acid-containing <br /> trast, ribosomal protein Rpl10p 60S, 80S,        plates 5 days incubation. shows region polysome fractions low-salt conditions (Fig. 3B, left)   Nmd3p required growth.  truncated form <br />  3410     GADAL ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. rsa1 disruption mutant nmd3 ts mutant accumulate Rpl25p-eGFP nucleus.   Left, growth rsa1&#226;&#338;&#172; disruption isogenic wild-type strain YPD plate liquid medium. Equivalent amounts cells (dilution 10&#226;&#171;&#186;1 steps) grown indicated temperatures 3 days. Right, nuclear accumulation Rpl25p-eGFP rsa1&#226;&#338;&#172; mutant. rsa1&#226;&#338;&#172; RSA1&#226;&#171;&#185; strains transformed RPL25-eGFP grown 23&#194;&#176;C incubation 37&#194;&#176;C 8 h. Rpl25p-eGFP localization analyzed fluorescence microscopy. (B) <br /> Growth nmd3-2 ts mutant 23 37&#194;&#176;C. (C) Rpl25p-eGFP accumulates nucleus nmd3-2 ts mutant. Cells shifted 0, <br /> 30, 90, 180 min 37&#194;&#176;C, Rpl25p-eGFP signal analyzed fluorescence microscopy. <br />  VOL. 21, 2001                                                Nmd3p ADAPTER Xpo1p-DEPENDENT EXPORT 60S                           3411 <br />  <br />  <br />  <br />  <br />    FIG. 3. Nmd3p binds directly Rpl10p associates 60S subunits salt-dependent manner.   Nmd3p binds GST-tagged Rpl10p expressed E. coli. Expression purification recombinant Rpl10p-GST absence presence Nmd3p-His6 described Materials Methods. supernatants E. coli cell lysates (Sup) incubated glutathione-Sepharose 4B beads. washing, bound <br /> proteins eluted sample buffer (Bound), fractions analyzed SDS-PAGE Coomassie blue staining   Western <br /> blotting using &#226;&#65533;&#163;-Rpl10p &#226;&#65533;&#163;  . Lanes: 1, protein standard (10-kDa ladder); 2 4, supernatants; 5 7, bound fractions; 2 5, <br /> Rpl10p-GST absence Nmd3p-His6; 3 6, Rpl10p-GST coexpressed Nmd3p-His6; 4 7, GST coexpressed Nmd3p-His6. <br /> Note length Nmd3p-His6 sensitive proteolysis E. coli, yielding stable breakdown product Nmd3p-His6*, apparently <br /> lacks short N-terminal  Indicated unknown E. coli contaminant (?) coisolates Rpl10p-GST. (B) Nmd3p, tagged ProtA fully functional, extracted 60S ribosomes high-salt (500 mM KCl) treatment. Ribosomes analyzed sucrose gradient centrifugation low- high-salt buffer, ribosomal profiles determined OD260 measurement gradient <br /> fractions  . Western blot analysis gradient fractions reveals ProtA-tagged Nmd3p Rpl10p  . <br />  <br />  <br /> lacking NES (&#226;&#338;&#172;NES1) able complement                  NES NLS activities, account shuttling <br /> nmd3 null strain, cells grow slowly temperatures          Nmd3p nucleus cytoplasm. <br /> (Fig. 4B). addition, NES domain Nmd3p                      Previous work shown proteins harboring leucine- <br /> fused characterized NLS Npl3p, fusion pro-             rich NES transported cytoplasm export  <br /> tein exported cytoplasm (Fig. 4C). deletion             ceptor Xpo1p. determine Xpo1p  C terminus abolishes nuclear accumulation, indicat-              volved export Nmd3p, examined ing region contains NLS (Fig. 4C).           Nmd3p-NES replaced characterized <br /> data indicate C-terminal domain Nmd3p exhibits                Xpo1p-dependent NES PKI. Significantly, PKI-NES <br />  3412     GADAL ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. NESs Nmd3p carboxy-terminal domain.   Schematic drawing Nmd3p domains. Sequence comparison classical <br /> NESs (PKI, mitogen-activated protein kinase kinase [MAPKK], human immunodeficiency virus Rev) NES-like sequences present Nmd3p C domains S. cerevisiae (S.C-1 S.C-2), humans (HUM-1, HUM-2), Schizosaccharomyces pombe (S.P-1 S.P-2) Drosophila <br /> (DRO-1 DRO-2). (B) nmd3 null strain complemented length Nmd3p-eGFP, Nmd3p truncation construct lacking C-terminal NESs (&#226;&#338;&#172;NES1) lacking NESs (&#226;&#338;&#172;NES1/2) 5 days 5-fluoroorotic acid-containing plates. (C) <br /> Intracellular location indicated Nmd3p constructs tagged GFP yeast cells. NLS Npl3p (NLSRGG) tagged GFP used test NES activity Nmd3p C domain. (D) Nuclear accumulation Rpl25p-eGFP expressed slow-growing nmd3&#226;&#338;&#172;NES1 mutant <br /> grown 30&#194;&#176;C. <br />  <br />  <br /> fused C-terminal end Nmd3p&#226;&#338;&#172;NES1/2                described Ho et al. functional (14), stores nuclear export Nmd3p complements                nmd3&#226;&#338;&#172;NES construct lacking 44 residues Nmd3p <br /> growth defect nmd3 &#226;&#338;&#172;NES mutant (data shown).                gives viable cells. Nmd3p contains leucine-rich NESs  Nmd3p NES domain important cell growth             C domain deletion impairs nuclear nuclear export protein.                                 export Nmd3p, resulting slow-growth phenotype (Fig. <br />    Consistent findings, Ho et al. identified NLS       4B C, Nmd3p-&#226;&#338;&#172;NES1), sought test nuclear export essential NES Nmd3p carboxy-terminal domain,                60S subunits mutant.  slow-growing <br /> nonfunctional nmd3&#226;&#338;&#172;100 construct restored               Nmd3p-&#226;&#338;&#172;NES1 strain transformed Rpl25p-eGFP. taching heterologous PKI-NES (14).  nmd3&#226;&#338;&#172;50                striking nuclear accumulation large subunit reporter  VOL. 21, 2001                                             Nmd3p ADAPTER Xpo1p-DEPENDENT EXPORT 60S                         3413 <br />  <br />  <br />  <br />  <br />    FIG. 5. Nmd3p 60S large subunits exported Xpo1p-dependent transport pathway.   Nmd3p-eGFP expressed LMB-sensitive xpo1 mutant isogenic wild-type strain. (B) Rpl25p-eGFP expressed LMB-sensitive xpo1 mutant isogenic wild-type <br /> strain. time point 0 min, LMB (100 ng/ml) added culture medium, fluorescence signals GFP fusion proteins observed fluorescence microscopy. <br />  <br />  <br /> seen mutant grown 30&#194;&#176;C (Fig. 4D).                   contrast, nuclear accumulation Nmd3p-eGFP mutations NES domain Nmd3p export                 evident xpo1-1 ts mutant shifted restrictive <br /> receptor Xpo1p cause inhibition 60S subunit export.                 temperature requires longer incubation restrictive <br />    Nmd3p 60S subunit export inhibited             temperature (data shown). nuclear accumulation Xpo1p function blocked. sought test directly             Nmd3p-eGFP seen cse1-2 ts mutant (31), Xpo1 export receptor Nmd3p. Recently,            defective nuclear export Srp1p/importin &#226;&#65533;&#163; (data LMB-sensitive xpo1 mutant S. cerevisiae engineered           shown). reported earlier Rpl25p-GFP (15) site-specific mutagenesis critical residue Xpo1p           Rpl25p-eGFP, reporters signifi- <br /> (27). mutant exhibits strong rapid inhibition            cantly accumulate xpo1-1 mutant shifted 1 h nuclear export NES-containing cargoes addition              restrictive temperature; nuclear accumulation Rpl25p- <br /> LMB. shown Fig. 5A, nuclear export Nmd3p-eGFP              eGFP seen xpo1-1 cells prolonged <br /> strongly inhibited LMB-sensitive xpo1 strain 5 min af-          incubation 37&#194;&#176;C (data shown). possible ter addition LMB. Strikingly, large subunit reporter            observed inhibition nuclear poly &#226;&#171;&#185; RNA export xpo1-1 <br /> Rpl25p-eGFP accumulates inside nucleus             cells shifted 37&#194;&#176;C (33) impairs synthesis ribosomal time range (Fig. 5B). data rapid            proteins, mRNAs encoding ribosomal proteins strong export defect Nmd3p-eGFP Rpl25p-eGFP                longer exported cytoplasm. <br /> occurs LMB-sensitive xpo1 mutant, suggesting Nmd3p 60S ribosomal subunit export coupled.                                          DISCUSSION <br /> conclude 60S ribosomal export requires Xpo1p.  <br /> ings different previous studies              mechanism ribosomal export nucleus shown Rpl25p-GFP accumulates inside nucleus            cytoplasm understood. Compared nuclear export <br /> LMB-sensitive xpo1 mutant later time points  e., 2 h)        cargoes, e.g., small RNAs tRNA, nuclear <br /> (14). Notably, Ho et al. used RPL25-GFP control           export ribosomes likely complicated, view inducible GAL10 promoter (14). Accordingly, slow               extremely complex biogenesis pathway ribosomes. onset Rpl25p-GFP nuclear accumulation xpo1 mutant            particular, modification, processing, conformational rear- fact GAL10 promoter switched             rangement steps, place ribosomal assembly, allow Rpl25p-eGFP expression. study               trigger transport nucleolus expression RPL25-GFP native promoter, leading          nuclear pores cytoplasm. report nuclear accumulation 5 15 min blocking export         identification novel class mutants, defined rix mu- <br /> LMB treatment.                                                         tants, using visual assay nuclear retention Rpl25p- <br />  3414     GADAL ET AL.                                                                                               MOL. CELL. BIOL. <br />  <br />  <br /> eGFP reporter. 2% (20 900) ts mutant strains        NESs deleted, corresponding Nmd3p construct screened gave rise rix phenotype, comparable levels       longer functional. Furthermore, NES domain Nmd3p, <br /> mutations major pathways, e.g., cell cycle pro-      attached NLS, exhibits distinct NES activity. <br /> gression pre-mRNA splicing. know          findings suggest essential role complementation groups mutants represent,             Nmd3p C domain function nuclear export. genes tested fell different complementation          consistent observation archaebacterial homo- <br /> groups (data shown). conclude substantial num-         logues Nmd3p lack NES-containing C domain charac- <br /> ber genes (probably 50) rise rix phe-      teristic eukaryotes (3). <br /> notype. Analysis allow genetic dissection        data strongly suggest Xpo1p NES  <br /> ribosomal assembly transport possibly coupling.        ceptor Nmd3p. Interestingly, LMB-sensitive xpo1 mu- <br />    mutant analyzed, rix5-1,      tant, exhibits inhibition nuclear export NES  A68V mutation ribosomal protein Rpl10p.            taining cargoes addition LMB (27), allows suggests late assembly steps ribosome biogenesis,        observe rapid nuclear accumulation Nmd3p-eGFP occur inside nucleus, crucial subsequent           Rpl25p-eGFP inhibition Xpo1p. contrast, export cytoplasm. Rpl10p conserved, having homo-           classical xpo1-1 ts mutant does reveal strong nuclear <br /> logues humans (28), archea, E. coli (9). Rpl10p         export defect Nmd3p Rpl25p-eGFP. conceiv- absent preparation 66S nuclear preribo-          able fast onset strong inhibition nuclear <br /> somal particle, proposed Rpl10p assembles 60S       poly &#226;&#171;&#185; RNA export xpo1-1 cells (33) impairs ribosomal <br /> subunits cytoplasm (19).  Rpl10p relatively        biogenesis early stage, mRNAs encoding ribo- <br /> weakly associated 60S subunit (5) lost      somal proteins exported cytoplasm. preribosomal particles purification.                 scenario, ribosomal proteins including Rpl25p-eGFP  <br />    findings Rpl10p required nuclear pre-rRNA         porter longer synthesized restrictive tem- <br /> processing hint associates preribosomes      perature. reason Nmd3p enters nucleus. Consistent interpretation,      nucleus subsequently exported cytoplasm NLS Rpl10p uses import receptor Kap123p,           Xpo1p? likely possibility Nmd3p participates ribo- importins (Kap121p Kap123p) involved            somal subunit export, correlates observed <br /> import ribosomal proteins (29). hand,          concomitant nuclear accumulation Rpl25p-eGFP Nmd3p- nuclear accumulation Rpl10p-eGFP nucleoporin           eGFP LMB-sensitive XPO1 strain. Biochemical data <br /> mutants. understood, Rpl10p-eGFP, contrast        support model Nmd3p binds Rpl10p, Rpl25p-eGFP, sensitive proteolysis inside          targeting Nmd3p 60S subunits. course nucleus attached preribosomal particles         work, Ho et al. characterized Nmd3p Crm1p-depen- fully assembled blocked export. known    dent adapter protein nuclear export large ribosomal rRNA processing intermediates ribosomal pro-           subunit (14), agreement data presented <br /> teins quickly degraded assembled assembled            <br /> incorrectly preribosomes. Interesting context      Remarkably, nuclear protein Rsa1p implicated newly synthesized 25S rRNA unstable (half-life 4 min)       facilitating association Rpl10p 60S ribosomal sub- <br /> nmd3 ts mutants restrictive temperature (13). Appar-          units (17). like Rpl10p, Rsa1p required ently, 25S rRNA-containing ribosomal particles accumu-          efficient nuclear export 60S ribosomal subunits 37&#194;&#176;C. <br /> late inside nuclei nmd3 mutants short lifetime.        Interestingly, restrictive temperature free 60S ribosomal  observation Rpl10p-eGFP does accumulate           subunits rsa1 null mutant depleted Rpl10p (17). nuclei transport mutants reflect high turnover     accumulation half-mer polysomes reported rate transport-arrested precursor particles.     mutants underaccumulate 60S subunits, hand, Rpl25p-eGFP reporter appears longer              presence polysomes carrying initiation complex half-life nuclear ribosomal precursor particles     awaits 60S subunit. phenotype seen Rpl10p observed fluorescence microscopy.  remains         mutants strains lacking Rsa1p, strains shown nuclear form Rpl10p-eGFP                 clear change overall 40S/60S ratios. appears possible detected rrp44/dis3-1 strain. possibility inac-   phenotype consequence retention 60S sub- <br /> tivation exosome  functions degradation       units nucleus.  Rsa1p factor (aberrant) pre-rRNAs (1), stabilizes pre-rRNA         mediates assembly Rpl10p 60S ribosomal subunits <br /> ribosomal proteins bound                                       inside nucleus (17). <br />    finding Rpl10p involved late ribosomal           summary, identified factors function <br /> sembly step, coupled nuclear export,       nuclear export 60S ribosomal subunits. described genetic link RPL10 NMD3 (16)          protein 60S ribosomal subunit, Rpl10p, required suggested Nmd3p play role large subunit export.      large subunit export. protein likely participates Nmd3p implicated translational control gene           late intranuclear maturation steps leading export-competent <br /> expression (16) formation, function, maintenance        60S subunits. Rpl10p interacts directly Nmd3p, NES- <br /> stable 60S subunits (3, 13). Strikingly, nuclear accumu-     containing protein specifically associated 60S sub- <br /> lation large subunit reporter Rpl25p-eGFP          units. Strikingly, Nmd3p shuttles nucleus nmd3 mutants, including slow-growing mutant lacking         cytoplasm uses Xpo1p receptor export. Accord- NESs C domain. leucine-rich               ingly, highly conserved Nmd3p act adapter <br />  <br />  <br />  requires general NES receptor Xpo1p export.        consequence, 60S subunits exported Xpo1p.                               essential role Xpo1p yeast appears                              <br /> involvement ribosomal export. similar conclusion                <br /> Nmd3p Xpo1p-dependent adapter protein nuclear                      <br /> export large subunit recently published                                     group (14). Finally, possible ribosomal                <br /> proteins bind Nmd3p adapter proteins                                     <br /> similar Nmd3p exist. exploiting powerful yeast ge-                <br /> netics, possible additional components                           ribosomal export machinery.                                                          <br /> </body></html>