<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mutation yeast Eug1p CXXS active sites CXXC results dramatic <br /> increase protein disulphide isomerase activity <br />  <br /> Protein disulphide isomerase (PDI) essential protein                  determine catalytic capacity wild-type mutant localized endoplasmic reticulum eukaryotic cells.                 forms assayed activity oxidative refolding reduced catalyses formation isomerization disulphide bonds                     denatured procarboxypeptidase Y refolding bovine folding secretory proteins. PDI composed                      pancreatic trypsin inhibitor. wild-type protein showed domains structural homology thioredoxin                          little activity, oxidative refolding assays <br /> CXXC catalytic motifs. EUG1 encodes yeast protein, Eug1p,                       isomerase activity required. surprising, highly homologous PDI.  Eug1p contains                         particular mutant forms Eug1p containing CXXC <br /> CXXS motifs instead CXXC. current model PDI                         motif displayed activity close genuine PDI. function cysteines motif required PDI-                       results lead propose general disulphide isomerization <br /> catalysed oxidase activity. gain insight bio-                    main function Eug1p  o. <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> properties unusual variant PDI purified characterized protein. furthermore <br /> generated number mutant forms Eug1p                    Key words : carboxypeptidase Y, protein folding, Saccharomyces active sites mutated CXXC sequence.                    cere&#194;&#352;isiae, thiol oxidation. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                      purified enzyme. studies appears Eug1p                                                                                   extremely inefficient PDI. surprising mutation endoplasmic reticulum (ER) constitutes folding  <br />                                                                                   active sites Eug1p CXXC motif improves vironment secretory proteins eukaryotic cells. secretory <br />                                                                                   PDI-like function &#194;&#352;itro. characterized various mutant <br /> protein undergo various post-translational modifications, <br />                                                                                   forms Eug1p, using &#194;&#352;itro assay based oxidative <br /> including formation disulphide bonds, vital                                                                                   refolding denatured reduced procarboxypeptidase Y folding stability protein. formation                                                                                   (proCPY). forms containing CXXC active-site <br /> isomerization disulphide bonds catalysed protein di- <br />                                                                                   sequences exhibit significantly increased foldase activity sulphide isomerase (PDI) [1], abundant protein                                                                                   compared wild-type protein. consistent ER. <br />                                                                                   previous results showing mutation Eug1p active sites <br />    PDI composed domains b-bh-ah arrangement <br />                                                                                   CXXC sequence motif improved ability complement [2]. thioredoxin-like ah domains contain active <br />                                                                                   PDI1 deletion [5]. <br /> sites, sequence motif CGHC. ability PDI form disulphide bond proximal cysteines motif enables protein carry oxidizing equivalents transfer protein substrate. EUG1 gene product                    MATERIALS METHODS <br /> highly homologous PDI yeast mammals appears domain organization [3].   unique <br />                                                                                   Media materials PDI-like proteins contains CXXS active-site                           E. coli grown Luria&#226;&#8364;&#8220;Bertani (LB) medium supplemented <br /> sequences. Mutant PDI active-site sequences CXXS                     appropriate antibiotic [6]. Restriction enzymes, T4 oxidase activity, isomerase activity                 DNA polymerase, T4 polynucleotide kinase T4 DNA ligase <br /> affected [4]. Eug1p expected able                  Promega (Madison, WI, U.S ). Oligonucleotides directly introduce disulphide bonds,  principle,               synthesized DNA Technology (Aarhus, Denmark). <br /> able catalyse rearrangement non-native disulphide bonds.                    Sequencing performed using Taq Dye Deoxy4 terminator <br />    unique active-site sequences, Eug1p interesting                 cycle sequencing kit ABI 373A DNA sequencer, terms enzymology, evolution cell biology. Based                     Applied Biosystems (Foster City, CA, U.S ). 1,4-Dithiothreitol <br /> logic summarized suggested Eug1p acts                   o-phthalaldehyde Merck (Darmstadt, Germany). dedicated isomerase yeast ER. investigate                        N-(3-[2-furyl]acryloyl)-Phe-Phe,   tris(carboxyethyl)phosphine <br /> possibility  chosen prepare Eug1p                           PMSF Sigma (St. Louis, MO, U.S ). 5h,5h- <br /> Escherichia coli C-terminal tag characterize                    Dithiobis(2-nitrobenzoic acid) ICN Biomedicals                                         &apos; <br />  <br />  <br />  <br />   Abbreviations used : proCPY, procarboxypeptidase Y ; ER, endoplasmic reticulum ; GS-proCPY, glutathionylated proCPY ; LB, Luria&#226;&#8364;&#8220;Bertani ; PDI, <br /> protein disulphide isomerase. <br />   1 <br />     correspondence addressed (e-mail jrw!crc.dk). <br />  <br />                                                                                                                                # 2001 Biochemical Society <br />  270             P. N&#195;&#184;rgaard J. R. Winther <br />  <br /> Table 1    Plasmids used expression E. coli                  filtration column equilibrated 50 mM acetate buffer, pH 5.0. <br />                                                                     eluate 600 &#194;&#181;l collected. 300 &#194;&#181;l eluate, 200 &#194;&#181;l <br />                              Plasmid   Active sites                 0.1 M potassium phosphate buffer, pH 7.0, 500 &#194;&#181;l 8 M <br />                              pCT65     CXXS-CXXS                    guanidinium chloride 0.1 M phosphate buffer, pH 7.0,                              pPN335    CXXS-CXXC                    added. proCPY concentration measured using <br />                              pPN342    CXXC-CXXC                    fluorescence excitation emission wavelengths 295                              pPN365    CXXC-CXXS                    335 nm, respectively, quantificated using standard curve. <br />                              pPN398    CXXC-SXXS                    250 &#194;&#181;l NAP-5 eluate 10 &#194;&#181;l 10 mM 5h,5h-dithiobis(2- <br />                              pPN406    SXXS-CXXC                    nitrobenzoic acid) added, incubated minimum 1 h, <br />                                                                     peak corresponding thio-nitro-benzoate detected <br />                                                                     using reversed-phase HPLC. concentration free thiols                                                                     determined relating area peak standard curve. <br /> (Aurora, OH, U.S ). Isopropyl &#206;&#178;-&#239;&#65533;&#164;-thiogalactoside        number free cysteines residues proCPY molecule Biological (Swampscott, MA, U.S ).                             subsequently calculated. <br />  <br /> Strains plasmids                                                Glutathionylation proCPY <br /> pBH1865 described N&#195;&#184;rgaard et al. [7]. pCT58 described     GSSG added concentration 30 mM solution Holst et al. [5]. EUG1 mutated using conven-                 containing denatured reduced $ 12 &#194;&#181;M proCPY 6 M <br /> tional techniques [8,9] subcloned pBH1865, giving          guanidinium chloride\0.1 M Tris\HCl, pH 7.4. represents <br /> pCT56 (containing EUG1CXXS-CXXC) pCT61 (containing              $ 250-fold molar excess GSSG proCPY thiols. EUG1CXXC-CXXS). construction expression plasmid          mixture incubated 5 h. subsequent 60-fold dilution <br /> pCT65, coding sequence signal sequence ampli-       mixture 2 mM GSH, initiates refolding, <br /> fied PCR using primers oCT10 (5h-CCC GGG CAT ATG             resulted GSSG concentration 0.5 mM GSH <br /> GCT AGC ACG CCG GGA TCA GAT TTA CTC-3h)                         concentration 2 mM. <br /> oCT11 (5h-CCC GGG GCG GCC GCC TCG AGT ACG GTA                          determine degree glutathionylation, gluta- <br /> TCT TCA GTA GAC-3h). PCR product digested              thionylated proCPY (GS-proCPY) separated non- <br /> NheI XhoI cloned corresponding sites pET-       bound <span id='am-11' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-12' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span>glutathione</span> applying sample HiTrap <br /> 24a(j) (Novagen). pPN335, pPN342 pPN365               (Amersham Pharmacia Biotech) desalting column equilibrated subcloning NcoI\BstBI fragments pCT56, pCT58            50 mM acetate buffer, pH 5.0. 1&#226;&#8364;&#8220;1.5 ml eluate pool, pCT61, respectively, corresponding sites            300 &#194;&#181;l used determine proCPY concentration pCT65. pPN398 pPN406 mutagenesis                   described  eluate fraction (100 &#194;&#181;l) mixed 2 &#194;&#181;l pPN335 pPN365. sequence EUG1 open                 1 mM tris(carboxyethyl)phosphine, reduces reading frame cases verified.                            disulphide bonds. incubation 30 min,                                                                     liberated GSH determined fluorometrically deriva- <br /> Purification Eug1p                                               tization o-phthalaldehyde described Senft et al. [11]. <br />                                                                        relating GSH concentration proCPY  <br /> E. coli BL21(DE3) carrying relevant expression plasmid                                                                      centration, GS-proCPY preparations contain <br /> Table 1) grown LB medium containing kanamycin <br />                                                                     9.4p1 mol GSH\mol proCPY, corresponding $ 85 % <br /> (30 &#194;&#181;g\ml). cells grown 30 mC D                                                                     &apos;!!       glutathionylation. <br /> 0.2&#226;&#8364;&#8220;0.3, production Eug1p induced addition isopropyl &#206;&#178;-&#239;&#65533;&#164;-thiogalactoside final concentration 1 mM. culture harvested 4&#226;&#8364;&#8220;5 h 30 mC, cells     Assay insulin reduction resuspended $ 5 ml binding buffer (20 mM Tris\HCl,       assay mixture contained 0.1 M potassium phosphate, pH 7.0, <br /> pH 8.0\0.5 M NaCl\5 mM imidazole) litre culture. PMSF        2 mM EDTA, $ 0.7 mg\ml human insulin 0.2 M NaCl added concentration 1 mM, cells            final volume 200 &#194;&#181;l. assays performed presence <br /> sonicated 6i30 s. Cell debris removed centrifugation,    absence 0.5 &#194;&#181;M yeast PDI Eug1p. reaction lysate added pre-equilibrated Ni#+-chelating       started addition 10 &#194;&#181;l 0.2 M 2-mercaptoethanol. column (Novagen), eluted described manu-        reactions carried microtitre plates, turbidity <br /> facturer. eluate dialysed 25 mM Tris\HCl,           (630 nm) reaction mixtures read, 2 min intervals, <br /> pH 8.0, dialysate applied MonoQ <span id='am-5' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span>-exchange       using plate reader. <br /> column (Amersham Pharmacia Biotech) equilibrated buffer. Eug1p eluted $ 0.3 M NaCl. Relevant <br /> fractions pooled dialysed 25 mM Tris\HCl,          RESULTS <br /> pH 8.0, frozen storage. purity Eug1p analysed    Oxidative refolding reduced denatured proCPY SDS\PAGE estimated   90 %.                                marginally enhanced presence wild-type Eug1p <br />                                                                     prerequisite ability PDI function isomerase <br /> Determination disulphide-bond formation proCPY         active site CXXC motif reduced state. oxidative refolding                                                 oxidation isomerization protein thiols ER, Refolding carried <span id='am-23' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span>glutathione</span> redox buffer          prerequisite puts cell dilemma. hand high <br /> described Westphal et al. [10]. Samples 500 &#194;&#181;l removed   capacity oxidation essential efficient production refolding mixture various times. refolding          disulphide-bonded secretory proteins. evidence reaction quenched addition 10 &#194;&#181;l 1 M HCl       suggests protein oxidation mediated oxidized PDI [12]. final pH $ 2. Glutathione removed applying          hand, efficient oxidative machinery  <br /> sample NAP4-5 (Amersham Pharmacia Biotech) gel-               likely incorrect disulphide bonds formed. <br />  <br /> # 2001 Biochemical Society <br />                                                                                                              Yeast Eug1p disulphide isomerase activity            271 <br />  <br />                                                                                              active sites PDI-like CXXC motifs [5,7]. obtain                                                                                              quantitative measure activity wild-type mutant <br />                                                                                              forms assayed activity proteins produced E. coli <br />                                                                                              using assay based oxidative refolding unfolded                                                                                              reduced proCPY [10]. <br />                                                                                                 proCPY  o PDI substrate yeast. native form, <br />                                                                                              proCPY contains disulphide bonds. folding matu- <br />                                                                                              ration protein followed &#194;&#352;itro  o. <br />                                                                                              &#194;&#352;itro, oxidative refolding proCPY studied dilution <br />                                                                                              denatured reduced protein redox buffer,                                                                                              includes reduced oxidized glutathione. refolding, <br />                                                                                              samples withdrawn treated proteinase K,                                                                                              activates proCPY removal propeptide [13]. Proteinase <br />                                                                                              K degrades unfolded partially folded proCPY.                                                                                              activity carboxypeptidase Y subsequently measured                                                                                              compared activity non-denatured sample. <br />                                                                                                 spontaneous oxidative refolding proCPY leads                                                                                              $ 40 % recovery activity 5 h (Figure 1A). progress <br />                                                                                              curve appears sigmoidal, lag phase $ 1 h. Addition                                                                                              wild-type Eug1p surprisingly small effect. catalysed <br />                                                                                              wild-type Eug1p, lag phase spontaneous <br />                                                                                              refolding, recovery folded proCPY higher ($ 55 % <br />                                                                                              5 h, Figure 1A). Conversion active-site sequences <br />                                                                                              CXXC, hand, dramatically increases efficiency <br />                                                                                              enzyme. lag phase significantly shorter,                                                                                              recovery reaches $ 90 % 5 h (Figure 1A). data                                                                                              consistent effect observed  o, CXXS  <br />                                                                                              CXXC mutations cause markedly improved growth rate                                                                                              EUG1-complemented strain [5]. noted,  <br />                                                                                              &#194;&#352;itro oxidative refolding proceeds significantly <br />                                                                                              efficiently yeast PDI Eug1pCXXC-CXXC                                                                                              conditions concentrations enzyme (Figure 1A). <br />                                                                                                 compare relative importance active sites,                                                                                              mutant forms Eug1pSXXS-CXXC Eug1pCXXC-SXXS com- <br />                                                                                              pared. SXXS active sites catalytically inert PDISXXS-SXXS <br />                                                                                              does catalyse proCPY refolding [10]. ah domain far <br />                                                                                              plays important role, seen decreased lag <br />                                                                                              phase, higher recovery Eug1pSXXS-CXXC- <br />                                                                                              catalysed refolding reaction (Figure 2). particularly <br />                                                                                              evident observation progress curves                                                                                              CXXC-CXXC SXXS-CXXC mutant forms                                                                                              similar (Figures 1A 1C). spite fact                                                                                              CXXC-CXXC mutant double concentration functional <br /> Figure 1     Oxidative refolding reduced denatured proCPY                             sites compared SXXS-CXXC mutant form. <br />                                                                                                 Instead inert SXXS motif, mutant forms Eug1p <br /> Reduced denatured proCPY (0.15 &#194;&#181;M) refolded absence presence 1 &#194;&#181;M       investigated wild-type motif CXXS, Eug1p. refolding carried 0.1 M Tris/HCl, pH 7.4/0.2 M KCl/0.5 mM GSSG/2 mM <br /> GSH 25 mC.   Refolding absence Eug1p ($) presence                                                                                              single active-site cysteine contribute oxidative refolding Eug1pCXXS-CXXS ( <br /> ) Eug1pCXXC-CXXC (4). dotted line shows refolding presence   proCPY, observed effect solely CXXC 1 &#194;&#181;M yeast PDI [10]. (B) Refolding absence Eug1p ($) presence     active site ? answer question, compared refolding Eug1pCXXC-CXXS ( <br /> ) Eug1pCXXC-SXXS (4). (C) Refolding absence Eug1p ($)     presence Eug1p mutants CXXC motif plus presence Eug1pCXXS-CXXC ( <br /> ) Eug1pSXXS-CXXC (4).                                CXXS SXXS motif. Assay Eug1pCXXC-CXXS                                                                                              Eug1pCXXC-SXXS mutant forms showed $ 75 % recovery 5 h. <br />                                                                                              progress curves arising presence of                                                                                              Eug1p mutant forms superimposable, indicating highly oxidizing environment inherently disfavours pres-                               presence single cysteine ah site does contribute <br /> ence free thiols PDI necessary maintenance                          significantly activity (Figure 1B). hand, isomerizing activity. Yeast probably eukaryotic                                comparison mutant forms Eug1pCXXS-CXXC organism disulphide-bond-forming apparatus                                  Eug1pSXXS-CXXC shows presence single cysteine extensively characterized. obvious role                                site small effect, leading slightly reduced lag phase <br /> Eug1p maintain isomerase activity hyper-                                (Figure 1C).  main contribution foldase <br /> oxidizing conditions. tested hypothesis assaying                              activity appears stem CXXC active-site <br /> properties purified Eug1p.                                                                sequence. <br />    o, EUG1 high-copy suppressor PDI1 deletion [3,7].                               PDI previously associated disulphide  suppression poor, ability EUG1                              dependent chaperone activity [14]. gauge activity complement PDI1 deletion highly improved changing                                asked Eug1p facilitate refolding denatured <br />  <br />                                                                                                                                             # 2001 Biochemical Society <br />  272               P. N&#195;&#184;rgaard J. R. Winther <br />  <br />  <br />  <br />  <br /> Figure 2     Recovery activity 1 5 h oxidative refolding proCPY <br /> Reduced denatured proCPY refolded absence presence various mutant forms Eug1p. Carboxypeptidase Y activity measured 1 (closed bars) 5 (open bars) h. <br /> Refolding 1 h presence of mutant forms CXXS CXXC mutations shows dramatically higher activity compared refolding catalysed wild-type Eug1p.  recovery 5 h higher. S.E.M. based 2&#226;&#8364;&#8220;7 independent measurements shown. <br />  <br />  <br />  <br />  <br />                                                                                                  proCPY forms incorrect disulphide bonds                                                                                                  substrates Eug1p. <br />  <br />  <br />  <br />                                                                                                  Unfolded oxidized proCPY substrate Eug1p <br />                                                                                                  experiments described investigated potential                                                                                                  Eug1p oxidative folding.  wild-type Eug1p                                                                                                  reasonably expected suited carrying                                                                                                  disulphide shuffling reactions, N-proximal cysteine                                                                                                  CXXC motif PDI necessary  Denatured <br />                                                                                                  glutathionylated proteins previously exploited                                                                                                  model substrates PDI [15,16]. substrates                                                                                                  oxidized form, reshuffling necessary protein                                                                                                  obtain native disulphide configuration. wanted explore <br />                                                                                                  GS-proCPY uniformly oxidized substrate.                                                                                                  experimental conditions used, spontaneous oxidation proCPY <br /> Figure 3     Refolding non-reduced proCPY                                                     occur formation mixed disulphide bond <br /> Refolding denatured proCPY, absence presence Eug1p. Oxidative refolding                                                                                                  protein cysteine glutathione. lag phase <br /> absence Eug1p  comparison ; $) ; non-oxidative refolding absence Eug1p (4)      observed progress curve spontaneous oxidative non-oxidative refolding presence Eug1pCXXC-CXXC ( <br /> ).                               refolding represent initial oxidation cysteine <br />                                                                                                  residues. Firstly, estimate time scale reaction,                                                                                                  followed disappearance free thiol groups proCPY <br />                                                                                                  spontaneous oxidative refolding taking samples proCPY possessing native disulphide bonds. seen                                        various time points. proCPY separated free <br /> Figure 3, refolding denatured proCPY containing                                       glutathione, number free thiol groups proCPY <br /> native disulphide bonds faster oxidative refolding.                                 molecule determined. experiment showed level  presence Eug1p effect refolding                                    free protein thiol decreases rapidly $ 20 min <br /> rate yield conditions, demonstrating                                    (Figure 4). conditions, essentially folded proCPY <br />  <br /> # 2001 Biochemical Society <br />                                                                                                                        Yeast Eug1p disulphide isomerase activity                      273 <br />  <br />  <br />  <br />  <br />                                                                                                   Figure 6     Insulin-reduction assay yeast PDI wild-type Eug1p <br />                                                                                                   precipitation insulin B-chain, caused reduction insulin, followed <br />                                                                                                   spectrophotometrically 630 nm. assay performed absence PDI Eug1p <br />                                                                                                   ( <br /> ) presence Eug1pCXXS-CXXS ($) yeast PDI (X). <br /> Figure 4 Disappearance free thiol groups spontaneous oxidative <br /> refolding proCPY number free protein thiol groups various time points spontaneous oxidative <br /> refolding proCPY measured related protein concentration  curve fitted   suggesting Eug1p poor isomerase (Figure 5). exponential equation). Fully reduced proCPY 11 free thiol groups. time course       consequence specific substrate used, wanted oxidative refolding shows establishment plateau 1.5&#226;&#8364;&#8220;2 thiol groups         test isomerase activity wild-type Eug1p using different <br /> proCPY molecule 20&#226;&#8364;&#8220;30 min.                                                                  substrate.  chose folding intermediate bovine <br />                                                                                                   pancreatic trypsin inhibitor, called N*, yeast PDI previously <br />                                                                                                   shown catalyse rearrangement particular <br />                                                                                                   substrate [10].  consistent results obtained <br />                                                                                                   using GS-proCPY substrate, wild-type Eug1p  <br />                                                                                                   tectable activity assay (results shown). lack                                                                                                   isomerase activity consequence non-reactive active- <br />                                                                                                   site cysteines. test  used wild-type Eug1p insulin- <br />                                                                                                   reduction assay [17], precipitation B-chain caused <br />                                                                                                   reduction disulphide bonds chains                                                                                                   followed. Wild-type Eug1p small significant activity <br />                                                                                                   assay, suggesting cysteines reactive <br />                                                                                                   (Figure 6). activity   extremely small compared <br />                                                                                                   activity similar concentration yeast PDI. <br />                                                                                                      presence Eug1pCXXC-CXXC refolding GS- <br />                                                                                                   proCPY gives rise progress curve identical                                                                                                   obtained refolding fully reduced proCPY (Figure 5). <br />                                                                                                   suggests shuffling reaction rate-limiting step <br />                                                                                                   catalysed refolding proCPY. <br />  <br />                                                                                                   DISCUSSION <br /> Figure 5      Refolding GS-proCPY <br />                                                                                                   Contrary generally believed, Eug1p does appear free thiol groups fully reduced proCPY oxidized formation mixed disulphide      good PDI. argued substrates bonds GSH prior initiating refolding. Refolding proCPY absence Eug1p ($)     ones chosen present study suitable.  presence Eug1pCXXC-CXXC ( <br /> ). Refolding GS-proCPY absence Eug1p         proCPY principle native yeast PDI substrate, fact <br /> (&gt;), presence Eug1pCXXC-CXXC (4) Eug1pCXXS-CXXS  .                            Eug1p converted efficient PDI simply <br />                                                                                                   changing active-site motifs CXXS CXXC argues <br />                                                                                                   proCPY inherently poor substrate Eug1p. <br /> appears time period (Figure 1A).  lag                                     &#194;&#352;itro, impact wild-type Eug1p phase likely represents initial formation disulphide                                  oxidative refolding proCPY small (Figure 1A). <br /> bonds, glutathione intra-molecularly,                                    CXXS CXXC mutations result dramatically <br /> subsequent phase showing regain activity mainly consists                                    increased effect oxidative refolding proCPY. Spontaneous <br /> disulphide rearrangement final folding native                                       oxidative refolding proCPY progresses long lag phase <br /> proCPY structure. Consistent interpretation, lag                                    (Figure 1). conditions, oxidation proCPY phase spontaneous refolding reaction GS-proCPY                                       occur reaction protein thiol groups GSSG, <br /> significantly shorter fully reduced proCPY                                       assay functions donor oxidizing equivalents. <br /> (Figure 5). Surprisingly, progress curves refolding GS-                                lag phase represents mainly oxidation, seen dis- <br /> proCPY wild-type Eug1p similar,                                         appearance free proCPY thiol groups (Figure 4). lag <br />  <br /> phase markedly shortened refolding catalysed                        Eug1p mutant CXXC active sites.                                                                                <br /> suggests higher oxidase activity mutants compared                            wild-type Eug1p. consistent  o                         <br /> observation EUG1CXXC-CXXC, unlike wild-type gene,                       <br /> able complement yeast strain auxiliary PDI-like                              <br /> proteins present [7].                                                     <br />    Lyles Gilbert [18]  case PDI, N- <br />                                                                                     C-terminal domains functionally different. saturating                         <br /> RNase concentrations, PDI mutant inactivated C-                          <br /> terminal active site retains near wild-type activity.                         <br /> mutant inactivated N-terminal site shows $ 30 %                           <br /> activity compared wild-type. non-equivalence                       <br />                                                                                         shown yeast PDI, ah site                                                                                          <br /> active [10]. oxidative refolding proCPY                    presence Eug1p CXXC active-site sequence                        <br /> ah site, similar yeast PDI, efficient presence                 <br /> Eug1p CXXC active site.                                                         <br />    rate-limiting step Eug1pCXXC-CXXC-catalysed folding                   <br />                                                                                       <br /> appears reshuffling incorrectly formed disulphide bonds. <br />                                                                                         supported notion progress curve                        <br /> refolding GS-proCPY identical curve arising                             refolding fully reduced proCPY (Figure 5). GS-proCPY                         used homogeneously oxidized substrate useful                     assay isomerase activity. Surprisingly, wild-type Eug1p did                 activity refolding GS-proCPY (Figure 5). <br />                                                                                         suggests CXXC enzymes better                         <br /> oxidases, better isomerases.                                                <br />    remarkable, evolutionary biochemical                          <br /> point view, activity Eug1pCXXC-CXXC mutant                        <br /> form 30&#226;&#8364;&#8220;50 % native yeast PDI.                       reasonable assume Eug1p evolved perform <br />                                                                                        highly specialized function yeast ER high-                 <br /> copy suppression PDI1 deletion does directly bear                           normally does ER. light, possibility                specialized function relate retrograde transport                    proteins ER tantalizing (M. Zhou R.                       <br />                                                                                  <br /> Schekman, personal communication). <br />  <br /> </body></html>