<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Genetics Cryptopleurine Resistancein Saccharomyces cerevisiae: <br />             Expression Ribosomal Protein Gene Family <br />  <br />                                                               ABSTRACT <br />                   Saccharomyces cereuisiae CRYl gene encodes 40s ribosomal subunit protein rp59 andconfers <br />                sensitivity theprotein synthesis inhibitor cryptopleurine. yeast strain containing cryl-Al::URA3 <br />                null allele viable, cryptopleurine sensitive  ) expresses rp59 mRNA, suggesting                second functional gene. CRY2 gene isolated yeast genomic library <br />                cloned bacteriophage X, using CRYl DNA probe. DNA sequence CRY2 gene contains <br />                open reading frame encoding ribosomal protein 59 thatdiffers residues rp59 encoded <br />                CRYl gene. CRY2 gene mapped left arm chromosome X , centromere- <br />                proximal cdc6 immediately adjacent ribosomal protein genes RPS24A RPL46. Ribosomal <br />                protein 59 essential protein; sporulation diploid doubly heterozygous cryl-A2::TRPl <br />                cryZ-Al::LEU2 null alleles, spore clones containing null alleles recovered. results <br />                indicate CRY2 expressed, lower levels CRYl: (1) Introduction CRY2 high copy <br />                plasmids CryR yeast genotype cryl CRY2 confers CryS phenotype. Transformation                CryR yeast CRY2 low copy CEN plasmid does confer  phenotype. (2) Haploids <br />                containing cryl-A2::TRPl null allele deficit 40s ribosomal subunits, cryZA1::LEUZ <br />                strains wild-type amounts 40s ribosomal subunits. (3) CRY2 mRNA present lower levels <br />                CRYl mRNA. (4) Higher levels <span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>;galactosidase expressed CRYl-lacZ gene fusion <br />                CRY2-lacZ gene fusion. Mutations alter eliminate <span id='am-25' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> rp59 <br />                encoded CRYl CRY2 result resistance cryptopleurine. CRY2  cry2)                expressed lower levels CRYl  cryl), CryRphenotype cry2 mutants expressed <br />                strains containing cryl null allele. <br />  <br />  <br />                                                                                  gene acry 1 resistance allele havebeen cloned T       0 facilitate studies yeastribosome biogen- <br />          esis, previously isolated ribosomal protein <br />  gene conferring readily assayable phenotype, resist- <br />                                                                                  shown encode 40s ribosomal subunit protein <br />                                                                                  rp59(LARKINand WOOLFORD           1983;HIMMELFARB, <br /> ance plant alkaloid cryptopleurine (LARKIN                            VASSAROTTIand FRIESEN      1984). <br />  WOOLFORD       1983;LARKIN, THOMPSON         WOOL-                             Homologs ofrp59 beenidentified number <br />  FORD 1987). Cryptopleurine therelated antibiotic                            organisms. inferred amino acid sequence <br /> emetine inhibit protein synthesis yeast                             yeast rp59 approximately 80% identical  eukaryotes binding t o high affinity site                           homolog rpS14 human, hamster, rat, Dro- <br />  40s ribosomal subunit blocking elongation                               sophila, maize, trypanosomes, Neurospora. Ri- <br />  factor EF-2 <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span>-dependenttranslocation pep-                              bosomal protein 59 50% identical sequence  tidyl-tRNA site P site ribo-                           Halobacterium cutirubrum ribosomal protein S 1 1  (BARBACID,FRE~NOand V ~ Q U E Z1975;                                     37% identical Escherichia coli ribosomal protein <br />  BUCHERand SKOCERSON            1976; DOLZ,V ~ Q U Eand <br />                                                                                S11 (CHENet al. 1986; RHOADS,DIXIT ROUFA <br /> J M ~ N E 1982). Cryptopleurine resistanceinyeast                            1986; TANAKA,     ISHIKAWA OCATA1986; LARKIN, results recessive mutation cryl,                            THOMPSON     WOOLFORD    1987; BROWNet al. 1988; gene approximately 2 cM centromere-proximal                                 LARKIN   et al. 1989; PERELMAN  BOOTHROYD     1990; mating type locus (MAT chromosome HI (SKO-                                TYLER   HARRISON    1990). striking homolo- <br /> CERSON, MCLAUCHLIN           WAKATAMA 1973; GRANT,                           gies proteins widely divergent species, S ~ N C H Eand J M ~ N E 1974). wild-type CRYl                         wellas associationof cryptopleurine resistance <br />                                                                                  mutations genes encoding rp59, attest    &#226;&#8364;&#8482; Present address:Department Genetics, SK-50,University Washing-        functional importance rp59 translation sug- <br /> ton, Seattle, Washington 98195. <br />    * Present address: MerckResearch Laboratories, P.O. Box 2000, 80Y-            gest proteins involved similar  <br /> 230,Rahway, New Jersey 07065-0900. <br />    &#226;&#8364;&#8482; Present address: Division Biological Sciences, University Nebraska, <br />                                                                                  served reactions required elongation steps 308 Manter Hall, Lincoln, Nebraska 68588-01 18.                                  protein synthesis. <br /> Genetics 135: 7 19-730 (November, 1993) <br />  720                                                 G. Paulovich et al. <br />  <br />    resistant cryl alleles recessive t o t h e    somes probed using methods recommended wild type CRYl allele heterozygous diploids,       supplier (Clontech Laboratories, lnc., Palo Alto, California). <br />                                                               Libraries ofyeast genomic fragments cloned bacterio- <br /> unusualfeaturesofthegeneticsofcryptopleurine                  phage X cosmids kindly provided Drs. L. Riles <br /> resistancesuggestthat      simpleMendeliandomi-             M. Olson, (Washington University, St. Louis, Missouri, <br /> nance/recessiveness relationship inadequate          University Washington, Seattle, Washington), scribe behavior  T h e striking       probed described BENDER       PRINGLE(1991). <br /> observation diploid yeast strains monosomic           5&#226;&#8364;&#8482; ends CRY2 transcripts mapped primer <br />                                                               extension analysis,using oligonucleotide primer 5&#226;&#8364;&#8482;- chromosome ZZZ sensitive cryptopleurine            GACTTGAGAGTTATGGCG-3&#226;&#8364;&#8482;, described LARKIN, remaining copyof chromosome 111 carries           THOMPSON     WOOLFORD(1987). Oligonucleotides 5&#226;&#8364;&#732;- cryl resistance allele (MEADE, RILEY n d MANNEY           TACAAGTACAATACATGCA-3&#226;&#8364;&#8482; 5&#226;&#8364;&#65533;AAGCCGTAT <br />  1977; CLARKE    CARBON1983). elegant series        TTCCATGCG-3&#226;&#8364;&#8482;, complementary unique sequences experiments, MEADE, RILEYa n d MANNEY(1977)                3&#226;&#8364;&#732; untranslated regions CRYl CRY2, respectively, <br />                                                               end-labeled usedasgene-specific hybridization <br /> extended observation toa series aneuploid         probes detect CRYl CRY2 mRNAsinRNAblot <br /> polyploid yeast strains demonstrated crypto-         assays. Steady-state levels CRYl CRY2 mRNA pleurine resistance function ratio         measured quantitative S1 nuclease protection assays number cryl resistance alleles t o ploidy. O n e       distinguish CRYl CRY2 mRNAs. 1.1-kb HindIII- <br /> interpretation theseresults level           Bglll fragment containing 5&#226;&#8364;&#732; thirds CRYl                                                               usedas diagnostic probe CRY1 CRY2 mRNAs. <br /> resistance depends interaction     cryl n d   <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> sequences 3&#226;&#8364;&#8482; exons CRYl geneorgenesnotlocatedonchromosome                   ZZZ.    CRY2 diverge 5&#226;&#8364;&#732; ends, shorter CRYl DNA <br /> Alternatively, CRYl locus represent          fragment protected S1 nuclease digestion member ribosomal protein gene family encoding            hybridized CRY2 mRNA hybridized CRYl <br /> rp59 member family located           mRNA. 0.9-kb XbaI-BglIl fragment used hybrid- <br />                                                               ization probe R P L l mRNA (DESHMUKH       et al. 1993). S1 <br /> o n chromosome chromosome           ZZZ.      nuclease protection assays carried described cause halfof yeast ribosomal protein genes studied        OSLEYet al. (1986), using DNA probes labeled <br /> thusfar duplicatedand essential(reviewedin                32P 5&#226;&#8364;&#8482; ends BgllI sites, hybridized <br /> WOOLFORD WARNER 199 1; WOOLFORD 199 ,                20 Pg total yeast RNA. <br /> favored secondpossibility. report, dem-           Isolation !genomic clones yeast recom- <br />                                                               binantphagelibrary: thousand plaques onstrate second gene encoding rp59,           library EcoRI partial genomic fragments yeast DNA <br /> CRY2, located chromosomeX , providing expla-            cloned bacteriophage X Charon 4A (WOOLFORDand <br /> nation genetic behavior cryptopleurine          ROSBASH    1981) screened plaque hybridization sistance Saccharomyces cerevisiae. note cryl       32P-labeledplasmid pJL32 containing 3.5-kb EcoRl frag- <br /> resistance mutations examples recessive muta-          ment includes CRYl (LARKIN       WOOLFORD <br />                                                               1983). Positive plaques counterscreened SzP-la- <br /> tions single member multigene family. T h e         beled plasmid pJL37 containing 1.5-kb EcoRI fragment <br /> CryR phenotype conferred     cryl detectable incryl     includes 5&#226;&#8364;&#8482; exon 5&#226;&#8364;&#8482; end intron CRY2 haploids CRYl expressed higher             CRYl. <br /> levels CRY2.                                                DNA sequencing: series nested deletions gen- <br />                                                               erated 3.0-kb EcoRI-BamHl CRY2 genomic frag- <br />                                                               ment subcloned Bluescript SK+ vector (Stratagene, <br />              MATERIALS METHODS <br />                                                               LaJolla, CA) usingexonuclease I11 mung bean nuclease. <br />    Strains media:S. cerevisiae strains used study Single-stranded templates prepared DNA sequenc- listedin Table 1. E. coli strains HBlOl NM522         ing performed using Sequenase recom- <br /> (Stratagene, LaJolla, California) used propagate      mended manufacturer. sequence CRY2 plasmids, strain Dp50supF (MANIATISet al. 1978)       submitted GenBank database accession <br /> used propagate bacteriophage X. Standard methods      number L12564. <br /> employed yeast genetics (MORTIMERand HAWTHORNE               Construction plasmids cyZA c y 2 null al- <br /> 1969; SHERMAN,     FINKand HICKS1986). Yeast trans-      leles: 4.6-kb EcoRI CRY2 genomic fragment cloned formed LiAc procedure ITO et al. (1983). Cryp-      bacteriophage X subcloned standard methods topleurine phenotypes assayed medium Containing       EcoRl site plasmidpBR325 (BOLIVAR      1978). 3.0- <br /> 5-1 5 PM cryptopleurine obtained Corkwood Enter-         kb EcoRI-BamHI fragment containing CRY2 inserted <br /> prises, Unit F, 61 Waratah St., Kirrawee, New South Wales,    EcoRI BamHl sites plasmids pBR322 <br /> Australia.                                                    YEpl3 (BROACH,      STRATHERN     HICKS1979) pro- <br />    Manipulation nucleic acids: Methods preparation     duce plasmids pBR322CRY2RB YEpl3CRY2RB,  electrophoresis DNA RNA, transfer nylon         spectively. Plasmid pCRY2 identified hybridization <br /> membranes (Schleicher &amp; Schuell, Keen, New Hampshire)         ofthe  3.0-kb EcoRI-BamHl CRY2 library yeast partial hybridization radiolabeled DNA probes      Sau3A fragments cloned vector YEp24 (CARLSON            described previously (MANIATIS,    FRITSCHand SAMBROOK BOTSTEIN1982). Plasmid pJT3O derivative YCp50 <br />  1982; FEINBERG   VOGELSTEIN   1983; LARKIN   WOOL-   contains cryl resistance allele SUP1 1. <br /> FORD 1983;  STAVENHAGEN         WOOLFORD       1984;    cryl-Al::URA3 deletion-insertion plasmidpcry 1- <br /> LARKIN,THOMPSON        WOOLFORD1987;SAMBROOK,             AURAwas constructed inserting l.l-kb Hind111 <br /> FRITSCH   MANIATIS1989).                                  fragment containing yeast URA3 gene    blotof electrophoretically separated yeast chromo-       EcoRI BglII sites internal CRYl follows: 2.2-kb <br />                                                  Cryptopleurine Resistance Yeast                               1           72 <br />                                                               TABLE 1 <br />                                           Saccharomyces cerevisiae strains used study <br />  <br />    Strain                                         Genotype                                                   Source <br />  <br /> A364A       MATaadel-1 ade2-1 uraltyrl-1 his7 lys2-1 gall                                       HARTWELL    (1968) <br /> BJ423       MATa cryl hisl-1                                                                    E. W. JONES <br /> DBY747      MATa his3-A1 leu2-3,112 trpl-289 ura3-52                                            Yeast Genetics Stock Center, <br />                                                                                                   Berkeley, California <br /> JLYlO     MATa cryl trpl-289 ura3-52                                                            Spore clone BJ423 X DBY747 <br /> JLYl1     MATa <br />              leu2-3,112 <br />                      ura3-52                                                                    Spore clone BJ423 X DBY747 <br /> JLY 12    MATa/MATacyl/CRYl leu2-3,112/LEU2 trpl-289/TRPIura3-52/ura3-52                        JLYlO X JLYll <br /> RL92      MATaleu2-3,112 <br />                       prp2                           ura3-52                                                               LASTet al. ( 1 984) <br /> JLYS      MATa cryl leu2-3,112ura3-52 trpl                                                      study <br /> BJ3344    MATa his3-A2OO leu2-AI lys2-801 ura3-52 trpl-A101 can1                                E. W. JONES <br /> BJ3345    MATa ade2-101 his3-A200 leu2-A1 lys2-801 trpl-A101 ura3-52                            E. W. JONES <br /> MMY1401   MATaIMATa adeP-lOI/ADEZ his3-A200/his3-A200 leu2-Al/leu2-A1 lys2-801/lys2-801         BJ3344 X BJ3345 <br />             trpl-AlOl/trpl-A101 ura3-52/ura3-52 canl/CANl <br /> MPY1402 MATa cryl-A2::TRPI CRY2 ade2-101 his3-A200 leu2-AI lys2-801 trpl-AI01 ura3-52           study <br />  MPY1403 MATa CRYl cry2-AI::LEU2 his3-A200 leu2-AI lys2-801 trpl-AI01 ura3-52                   Study <br />  MPY 1404 MATa/MATa cryl-A2::TRPI/CRYI cry2-Al::LEU2/CRY2 adeZ-lOI/ADEP his3-A200/              MPY 1402 X MPY 1403 <br />             his3-AZO0 leu2-Al/leu2-Al lys2-801/lys2-801 trpl-AlOl/trpl-AI01 ura3-52/ura3-52 <br /> JTY69     MATaIMATa his3-A200/his3-A200 trpl-Al/trpl-A2 leu2-A2/leu2-A2ura3-167/ura3-167        Study <br /> JTY78     MATaIMATa  cryl-A2::TRPlICRYI      cry2-Al::LEU2/CRY2 his3-A200/his3-A200 t r f l -   Study <br />             Al/trpl-Al leu2-A2/leu2-A2 ura3-167/ura3-167 <br /> JTY8l     MATa <br />            CRYl        cry2-Al::LEU2 his3-A200 trpl-AI leu2-A2 <br />                                                              ura3-167                           Spore clone JTY78 <br /> JTY82     MATa <br />            cryl-A2::TRPl <br />                      CRY2             his3-A200 trpl-A1 leu2-A2 ura3-167                        Spore clone JTY78 <br /> JTY83     MATa <br />           CRYl         CRY2 his3-A200 trpl-A1 leu2-A2 <br />                                                   ura3-167                                      Spore clone JTY78 <br /> DBY745 cryMATa cryl leu2-3,112 <br />                       adel       ura3                                                           J. HABER <br />  <br />  <br />  <br />  <br /> CRYl HindIII fragment plasmid pCRYl (LARKIN                  CRY2 coding sequence pcry2ALEU codons WOOLFORD1983) cloned HindIII site ofa                    short 5&apos; exon final 49 codons 3&apos; exon. <br /> derivative ofplasmid pBR322 (BOLIVAR        et al. 1977) bearing      T h e deletion-insertion endpoints crylA cry2A null filled  inactivated EcoRI site create plasmid pJL59.          allele construct confirmed DNA sequencing. <br /> pJL59 digested EcoRI BglII. T h e resulting                 step replacements yeast CRYl CRY2 genes <br /> ends repaired DNA polymerase Klenow fragment                performed method ROTHSTEIN(1983). Lin- XhoI linkers ligated ends. DNA               ear HindIII fragments(5-10 rg) containing cryl-AI::URA3 <br /> digested XhoI recircularized ligation create           cryl-AZ::TRPI, AatII-BamHI fragment contain- CRYl deletion plasmid pJL127. T h e 1.1-kb HindIII                ing cry2-A1::LEUZ transformedinto yeast diploids <br /> fragment bearing URA3 purified plasmid YEp24,                JLY12, MMY1401, orJTY78. Hind111 ends filled  XhoI linkers ligated           Analysis yeast ribosomal subunits: Yeast cells ends. resulting fragment ligated              grown density 1.0-1.5 X lo8 cells/ml 200 ml XhoI site pJL 127 createplasmid pcryl AURA.                     YEPD medium 30&quot;. Lysates prepared buffer <br />    T h e deletion-insertion allele cryl-AP::TRPI derived          containing 0.5 M KCI. Total amounts cellular ribosomal pJL127 follows. T h e EcoRI site CRY1              subunits quantified centrifugation cell extracts regenerated construction ofpJL127. Plasmid                  27,000 rpm   5 hr 13&#194;&#176;C  7% 47%linear sucrose <br /> pJL127 digested EcoRI, andthe 1.45-kb yeast                  gradients containing 0.5 M KCI, described Y.-F. TSAY, <br /> TRPl EcoRI fragment purified       plasmid YRp7 (STRUHL          G . SHANKWEILER,     J. LAKE J. L. WOOLFORD,      JR.   <br /> et al. 1979) insertedtocreatepcryl              ATRP. T h e       published data). <br /> pcry 1AURA pcryl ATRP constructs          lack       Construction assay CRYI-lac2 CRYZ-lac2 <br /> CRYl coding sequence codons present              gene fusions: T h e 1.1-kb HindIII-BglII fragment contain- short 5&apos; exon final 29 codons. second             ingthe 5 thirdsof       CRYlwas ligated disruption CRYl using TRPl constructed tofacilitate            HindIII BamHI sites plasmid YIp356R (MYERSet al. <br /> subsequent experiments employing crylA null mutants               1986) create frame fusion CRYl E. coli <br /> taining CRYl plasmid bearing URA3 (MORITZ et al.                 lac2 gene. Likewise, 1.3-kb EcoRI-BglII fragment  <br /> 1990).                                                                taining 5&apos; thirds CRY2was ligated    deletion-insertion allele cry2-A1::LEUZwas derived             EcoRI BamHI sites YIp357 (MYERSet al. 1986) plasmid pBR322CRY2RB digesting DNA                  fuse CRY2 frame l d . construct directed <br /> EcoRV Asp718, repairing Asp718 end Klenow                integrate ura3-167locus yeast strain JTY83 fragment DNA polymerase, ligating XhoI linkers              linearization theunique StuI site plasmid-borne <br /> ends, recircularizing plasmid ligation create           URA3 marker transformation selecting Ura+. Ge- plasmid pcry2A. pcry2A digested XhoI                nomic Southern hybridization confirmed expected  2.2-kbXhoI-Sal1 yeast LEU2fragment inserted              tegration events. Expression gene fusion assayed <br /> ligation create plasmid pcry2ALEU. remains             solid medium containing X-gal extracts fromcells <br />  722                                                         G. Paulovich et al. <br />  <br />  <br />  <br />  <br />                                           X                                           X <br />   FIGURE 1.-Replacement CRYI gene cryI-Al::URA3 null allele. horizontal arrow depicts CRY1 gene. Black <br /> regions noncoding sequences, white regions coding sequences, shaded region intron (notdrawn perfectly scale). lines adjacent genomic sequences. Restriction enzymes sites: Bg, BgllI; Hd. HindIII; RI, EcoRI; X, Xhol. DNA sequences EcoRI BglII sites CRYJ replaced 1.1-kb HindIII fragment containing yeast URA3 gene. Synthetic Xhol linkers attached filled EcoRI Bglll sites CRY1 filled Hind111 sites endsof LIRA3 fragment. DNAs ligated  2.8-kb Hind111 cryI-A1::CJRA3 fragment construct transformed veast diploid JLY 12, selecting Ura&#226;&#8364;&#8482;, C R Y l allele replaced cryl-AJ::URA3. <br />  <br /> grown liquid medium, described DESHMUKHet al.                    carrying cryl resistance allele (MEADE, RILEY (1 993).                                                                  MANNEY1977; CLARKE CARBON 1983).    Isolation sequencing   cry2 cry2 mutant alleles <br /> conferring cryptopleurine resistance cryl mutant al-                  exceptional resistant transformant showed aberrant <br /> lele cloned described LARKINand WOOLFORD                        segregation markers tested sporulated, <br /> (1983). isolate cry2 mutants resistant cryptopleurine,              examined  Nineteen tetrads 1.4 X lo9 cells strain JTY82 (genotype cryl-A2::TRPI                   dissected CryS transformants, C R M ) plated YEPD, colonies replica- <br /> plated YEPD containing 10 PM cryptopleurine CryR                  viable spores obtained  Ura <br /> colonies identified. Asymmetric polymerase chain                  phenotype segregated 2+:2-. Ura+ cosegregatedwith <br /> action (PCR) employed obtain single-stranded cry2                  cryptopleurine sensitivity, tightly linked MATa <br /> DNA strain sequencing cry2 alleles (GYL-                 segregated repulsion cryptopleurine <br /> LENSTEN 1989; ALLARD, ELLSWORTH HONEYCUTT <br />                                                                           resistance (data shown). Genomic Southern blots <br /> 1991) rg genomic DNA extracted mutant <br /> strain used PCR, 50 pmol                demonstrated integration event produced oligonucleotide       primers     5&#226;&#8364;&#732;-AACA-                new HindIII fragment expected size (2.8 kb) ACTCCCAAT-3&#226;&#8364;&#8482; (CRY2-2) 5&#226;&#8364;&#65533;AAGCCGTATTTCC                                 CRYl locus hybridized CRYI ATGCG3&#226;&#8364;&#8482; (CRY2-5). Onetwentieth DNA produced                        URA3 probes andthatcosegregated Ura+ reaction subjected PCR using oligonucleotides 50:l ratio (50 pmol CRY2-2 1                    phenotype tetrads(LARKIN1985). conclude <br /> pmol CRY2-5). Oligonucleotide CRY2-5 subsequently                     CRYl essential gene. <br /> used primer DNA sequencing reactions. nu-                       Becauseyeast ribosomal protein genes cleotide sequence mutant allele determined                    duplicated essential (reviewed WOOLFORD     twice.                                                                    WARNER1991; WOOLFORD1991) mostlikely <br />                                                                           explanation viability haploids carrying                            RESULTS                                        cryl-Al::URA3 mutation second gene <br />    Yeastcontaina second gene conferring C$                              encoding functional rp59. Thispossibility supported <br /> phenotype: yeast ribosomal proteins examined                         observation CRYl genomic cDNA <span id='am-9' about='oboInOwl:date' typeof='owl:Thing'>date</span> essential viability (reviewed WOOL-                    clones hybridize yeast genomic DNA restriction <br /> FORD WARNER 1991       ; WOOLFORD     199 1).                   fragments addition derived theCRYl <br /> termine CRYI encodes essential protein,                        locus [LARKIN andWOOLFORD         (1983) Figure 2, disrupted CRYI gene replacing sequences                         lanes 1 21 haploids containing c r y l - EcoRI BglII sites CRYl                      Al::URA3 disruption allele express RNAspecies <br /> 1. l-kb fragmentcontaining yeast URA3 gene (Fig-                      homologous CRYl probes identical size ure 1). T h e cryl-AI::URA3 allele lacks CRYl                  mature wild-type CRYl mRNA (Figure 3). <br /> coding sequence codons                          second functional gene isolated final 29 codons. HindIII fragment containing                        library yeast genomic DNA cloned bacterio- <br /> cryl-AI::URA3 disruption used transform dip-                       phage X, using CRYI probes. Phage hybridized loid yeast strain JLY 12 Ura+. JLY 12 homozygous                    CRYI-containing plasmid pJL32 counter- ura3-52 heterozygous cryl CRYI                        screened plasmidpJL37, contains short Ura- CryS. T h e CRYI allele JLY 12                      5&#226;&#8364;&#732; exon 5&#226;&#8364;&#8482; end intron CRYI (LARKIN linked MATa, cryl allele linked                       WOOLFORD1983). yeastDNA MATa. seven                            pJL37 hybridizes CRYI locus (LARKIN stable Ura+ transformants examinedwere CryS,                        WOOLFORD1983), phagethat hybridized consistent behavior monosomic diploids                        pJL32 did hybridize pJL37 likely                                                       Cryptopleurine Resistance Yeast                                    723 <br />  <br />                                                                         bridized C R Y l DNA (data shown). <br />                                                                            Sequences 4.6-kb EcoRI fragment                                                                         hybridized theC R Y l probe localized South- <br />                                                                         ern blot analysis 0.7-kb EcoRV-BglII fragment <br />           5.6 Kb -                               - CRY2                 andanadjacent         0.6-kb BglII fragment(data           4.6 Kb -                               - CRY2                 shown). nucleotide sequences frag- <br />                                                                         ments adjacent EcoRI-EcoRV fragment           3.5 Kb -                               - CRYl                 determined. sequence contains open reading <br />                                                                         frame 414 nucleotides interrupted                                                                         fourth codonby intron bounded consensus yeast <br />           2.2 Kb -                               - CRYl                 5&apos; 3&apos; splice sites containing conserved <br />                                                                         branch point sequence TACTAAC (Figure 4).                                                                         ORF encodes protein 96% identical rp59 <br />                                                  - CRYl                 encoded C R Y l (Figure 4).  designate <br />            1.6 Kb- <br />                                                                         gene CRY2. C R Y l gene interrupted              Probe: <br />                 CRY2CRYl                                                codon 307 nucleotide intervening se- <br />    FIGURE2.--GenomicDNA        fragments hybridize CRYI <br />                                                                         quencethat spliced C R Y l pre-mRNA CRY2 DNA.DNA prepared CryS yeast haploid                   (LARKIN,THOMPSON           WOOLFORD1987). A364A (genotype: CRYl CRY2). digested withEcoRI HindIII,             CRY2 intron 408 nucleotides long ho- <br /> subjected electrophoresis % agarose gels. transferred        mologous C R Y l intron, exceptforthe          <br /> Nytran, andprobed &quot;P-labeled 2.2-kb CRYI Hind111               served intron boundaries branchpoint sequences. <br /> fragment 4.6-kb CRY2 EcoRI fragment. Lanes 1 3 contain 5 <br /> rg DNA digested EcoRI. lanes 2 4 contain 5 rg        Primer extension analysis indicated 5&apos; ends <br /> DNA digested HindIII. DNA lanes 1 2 hybridized          CRY2 transcripts map nucleotides -24, -27, CRY1 probe lanes 3 4 CRY2 probe.          -33 respect theCRY2 translation initiation <br /> sizesin kilobases (kb) origins genomic fragment         codon. <br /> indicated. <br />                                                                             confirmed functional rp59 expressed <br />                                             Tetrad <br />                                                                         CRY2 demonstrating gene tran- <br />              CRY1 Genotype $          -    -   +   +1                 scribed vivo confer aCrySphenotype <br />                                                                         transformedintoaCryRstrain.              3.0-kb <br />                                                                         EcoRI-BamHI fragment containing CRY2 2.2- <br />                                                                         kb HindIII fragment containing C R Y l hybrid- <br />                                                                         ized blot RNA isolated temperature- <br />                                                                         sensitive splicing mutant RL92(genotype: CRYl CRY2 <br />                                                                         p r p 2 ) grown permissive restrictive temper- <br />                                                                         atures. restrictive temperature, unspliced pre- <br />                       11-10- .I_                                        mRNA accumulates prp2 mutant (ROSRASH           et <br />                                         7                               al. 1981).Notethatboththe          C R Y l andthe CRY2 <br />                                                                         probes hybridized tothemRNA              expressed  <br />                       CRYI- <br />                                                                       gene, equal efficiency.                                                                         approximately <br />                                                                         (LARKIN, <br />                                                                                        960-nucleotide <br />                                                                                      THOMPSON <br />                                                                                                               C R Y l pre-mRNA <br />                                                                                                   WOOLFORD1987)                                                                         approximately 1 100-nucleotide transcript comple- <br />                                                                         mentary CRY2 accumulate cells shifted 23&quot; <br />   FIGURE%-RNA complementary C R Y l DNAis present                 37&quot; (Figure 5), consistent observed block <br /> haploids containing cryl-AI::URA3 null allele. Total RNAwas           step 1 splicing prp2 temperature-sensitive <br /> extracted parental wild-type diploid JLY 12 (lane ,                                                                         mutants (LIN,LUSTIGand ABELSON         1987). diploid heterozygous cryl-AI::URA3 disruption (lane 2). haploid spores tetrad derived heterozy-        data w e inferthat CRY2 transcribed 1100 <br /> gous diploid (lanes 3-6). RNA subjected electrophoresis      nucleotide pre-mRNA 408 nucleotide 1.2% agarose-<span id='am-32' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> gel, transferred Nytran,           intron spliced produce mature CRY2 mRNA. <br /> probed &quot;P-labeled 2.2-kb C R Y l HindIII fragment. blot          T o directly demonstratethatthe         CRY2 locus simultaneously probed S2P-labeledplasmid 11-10. encodes abundant yeast transcript, ensure equal masses <br />                                                                         expressed, tested introduction extra RNA loaded lane.                                        copies CRY2 gene CryR strain conferred <br />                                                                         CrySphenotype. CRY2 gene cloned yeast <br /> contain DNA duplicated sequence                         high-copy number plasmid YEpl3 YEp24 C R Y l itself. phage identified                transformed CryR strain   JLYS (genotype: cry1 <br /> contained common 4.6-kb EcoRI fragment hy-                        CRY2). T h e cells transformed high-copy <br />  724                                                             G. Paulovich et al. <br />   -580               -560               -540                -520               -500               -480 <br />  CCTTCGTCATCTTGAAACATGGTCGTAGATGTGTAAGGATATTCTGTCGA~ATTGCCGATCTGATAAAGTTTGAAACTTGTTCTAGTTGAAACAAAGCCGGTGATGTTACTTGCTGCT <br />  <br />   -460                -440               -420                -400               -380                -360 <br />  AAGTCTAAGCGACCCTTAATGGTCCCTCTCCTGGCCGTAGCACTGGCGACATAAATAAATGCACTGAGTAAAAGAACAATCTGTGAAAACATCGTCACACTGCTGTTTCACTTGTTGATC <br />                                                                                                                                ............. <br />   -340                -320                -300                 -280                -260                -240 <br />  AAATTTACTTTGATGATATTTAATGGCTTGACAAATTCGTTCATCTTAATTTCATACATAATTTCATCCTGAAATCCTCGATCATTTTCGAGACTGTTTTCGAGCTTGATTTTATATGTT <br />  <br />   -220               -200               -180               -160               -140               -120 <br />  GGTGTAAACACATCATTGTACATCCTTGCACCACACTGAAAAGAATATGGAAAAAATTTCAAACTTTTTAATGGGTGATAGACTTGAAAAAC~CGGAAT~AGATATAGTCGCAAG <br />                        ............. <br />   -100                -80                -60                -4 0               -20                  1 <br />  AACTTGCCTTCTACGAGTATAAAAGCGGTACTATTCCTTCTGTATGCATCTTCATCGCAAGAGAGAAAAAGACTGAAATCAACAACTCCCAATAACAATTAAGAATGGCTAACG~ <br />                                                                                            *           *       *                                        M     N <br />                                                                                                                                                             M   S <br />    20                 40                 60                  80                100                120 <br />  GAAAGGGGTGATATCCTGTTT~CCATTTAGTTATTCTTTTTTTCAGATGGAGACTGTGATCGGGTAGCGCATGATGTTATTATTGGAATTTTATTTTTTTTGCAACTTTGAAGCACT <br />  <br />   140                                                          200                 220                 240 <br />  CCTAACTATATTGCAATAGTGTTATWLGATGCTACTTATGTTTGTAAGGCCATGAAATTCTGTAATACGGAGTTAAAATTTCTGCAATAAAGCACTTCAATGCTATTGTGTTATAGAGGC <br />  <br />   2                                      300                 320                 340                 360 <br />  TCTGCGCTCACCCTTAATACTGTATCATCTGTAATAAAGAATGTTTATCTTAAAAGGATTGTAACTAAATGGTTGTTAGATAGTGACTCATTCGTTTATTTTTTAATAATCGAAAAAGCA <br />  <br />   380              400               420            60 440              4                 480 <br />  GAAGTTCTGTT~CTATACCTCTTATCATGT~CCTTGTTCAAGCTCGCGATAACTCTCAAGTCTTTGGTGTTGCCAGAATCTACGCCTCCTTTAACGATACCTTTGTAC <br />                                                  D     L   V   Q     R   D   N   S   Q       V   F       G       V     R      Y     S   F   N   D   T   F   V   H <br />                                                  N V <br />   500                                                          560                 580                 600 <br />  ATGTTACCGATTTATCTGGTAAGGAAACTATCGCCAGAGTTACTGGTGGTATGAAAGTCAAGGCCGACAGAGACGAATCATCTCCATACGCTGCTATGTTGGCTGCCCAAGATGTTGCCG <br />    V T D L S G K E T R V T G G M K V K D R D E S S P Y M L Q D V  <br />  620                 640                 660                  680                 700                 720 <br /> CTAAGTGTAAGGAAGTCGGTATCACTGCCGTTCACGTTAAGATCAGAGCTACTGGTGGTACCAGAACCAAGACCCCAGGACCAGGTGGTCAAGCCGCTTTGAGAGCTTTGGCCAGATCTG <br />   K C K E V G T V H V K R T G G T R T K T P G P G G Q L R L R S G <br />         R <br />  740                 7 60                780                  800                 820                 840 <br /> GTTTGAGGATTGGCCGTATCGAAGACGTTACTCCAGTCCCATCAGACTCCACCAGAAAGAAG~TGGTAGAAGAGGTAGAAGATTATGATTTCTTTTTTTTTTACTCTATTAACGCATGG <br />   L R G R E D V T P V P S D S T R K K G G R R G R R L &#226;&#8364;&#8482; <br />                                          C <br />  860                 880                 900                  920                 940                 960 <br /> AAATACGGCTTTACCTTCTCACATAAATATAGTTTGTTTAATTAAT~GCAATATAAACATTTGCATATTCGACTTCGTGCATGCAG~TGCGGCAGAT~TTCATTTCTGTCTAAT <br />  <br />  980                1000 <br /> CTTTTGACAGTTTTAAGTACAATATA <br />      FIGURE4.-DNA sequence CRY2 gene inferred amino acid sequence rp59 encoded CRY2. nucleotide sequence coding strand CRY2 shown. open reading frame CRY2 translated amino acids line DNA sequence. amino acids encoded CRY1 differ encoded CRY2 indicated CRY2 amino acid sequence. nucleotides CRY2 numbered respect thetranslation initiation codon, designated + 1. U S R ~sequences <br />                                                                                                                        C        nucleotides <br /> -377 -364 -206 -193 underlined (. ... .). Transcription initiation sites nucleotides -24, -27, -33 underlined. nucleotides CRY2 intron italicized. conserved 5&#226;&#8364;&#8482; splice site, branchpoint, 3&#226;&#8364;&#8482; splice site sequences underlined <br /> (-). <br />  <br />  <br /> number CRY2 plasmids CryS replicated                               adjacent ribosomal protein genes RPS24A selectivemedium containing 5 PM cryptopleurine                                 RPL46 (LEERet al. 1985). example (data shown).Cells transformed high                               tightlylinked    eukaryotic ribosomal protein <br /> copy YEpl3 YEp24 vectors remain CryR                                 genes (Figure 6). <br /> clude CRY2 encodes functional ribosomal pro-                              onlytwo genes: 4.6-kb EcoRI <br /> tein 59 capable expressing CrySphenotype.                               restriction fragment containing CRY2 2.2-kb <br />    Mapping CRY2 CRY2 gene localized                                 HindIII fragment containing CRYl hybrid- left arm chromosome X, centromere-proximal                              ized blots ofyeastgenomic DNA digested cdc6 hybridizationof 1.5-kb EcoRI-SphI                               EcoRI HindIII. fragments CRYl restriction fragment contains CRY2 blot                           CRY2 loci detected. CRYl probe hybridized <br /> electrophoretically resolved yeast chromosomes                             1.6-kb 3.5-kb EcoRI fragments 2.2-kb set cloned <br />                       overlapping genomic restriction                          HindIIIfragment, allofwhich       include CRYI, fragments constructed L. RILESand M. OLSON                                  hybridized weakly 4.6-kb EcoRI fragment (BENDERand PRINGLE1991). CRYl                                     5.6-kb HindIII fragment containing CRY2 (Figure <br /> CRY2 genes different chromosomes. Compar-                               2). cloned4.6-kb EcoRI CRY2 fragment gave <br /> ison nucleotide sequences flanking CRY2                            strong hybridization signals withthe 4.6-kb EcoRI sequence data bases revealed CRY2 immediately                          5.6-kb HindIII CRY2 genomic fragments weak <br />                                                Resistance <br />                                                  Cryptopleurine              Yeast                                      725 <br />                       23&apos;37&apos;     37&apos; <br />                                   23&apos;                                  AI::LEU2 disruption allele (data shown). <br />                                                                           cryl-A2::TRPI haploid MPY 1402 cry2- <br />                                                                        Al::LEU2 haploid MPY 1403, derived gene <br />                                                                        disruption experiments described  mated <br />                P-                            -P <br />                                                                        form diploid strain MPY 1404 doubly het- <br />               M-                             &quot;                         erozygous cryl-A2::TRPI cry2-Al::LEU2 null <br />                                              - RPS24A                  alleles. doubly heterozygous diploid sporu- <br />                                                                        lated subjected tetrad analysis. Trp+Leu+ <br />                                                                        cells (genotype: cryl-A2::TRPlcry2-AI::LEU2)                                                                        recovered cases predicted genotypes                        13 24 <br />                                                                        spores tetrad. 21 tetrads dis- <br />             Probe:    CRYl         CRY2 <br />                                                                        sected, 16 tetratype, 3 non-parental ditype,   FIGURE5. C R Y 2 gene transcribed primary tran-        2 parental ditype respect cryI-A2::TRPI <br /> script containing intron larger CRYZ gene. Total <br /> RNA prepared 20 ml exponentially growing prp2 <br />                                                                        cry2-Al::LEU2 mutations. Thisratio tetrad <br /> mutant cells grown YEPD 23&quot; (lanes 1 3), grown         classes close 4: 1: 1 ratio expected 23&quot; shifted nonpermissive temperature 37&quot;        segregation unlinked genes, predicted hr accumulate unspliced intron-containing pre-mRNAs (lanes 2        CRYl CRY2 chromosomes 111 X , respec- 4). RNA subjected electrophoresis 1.2%agarose-       tively. Southern blot hybridization demonstrated formaldehyde gels, transferred Nytran, hybridized &quot;P- <br /> labeled 2.2-kb CRYZ HindIII fragment (lanes 2) 3.0-kb <br />                                                                        parental diploid haploid spore clone DNAs <br /> CRY2 EcoRI-BamHI fragment (lanes 3 4). CRYZ            contained expected wild-type mutant al- CRY2 DNAs cross-hybridize mRNA expressed              leles (data shown). Identical results obtained respective gene, ratios pre-mRNA mRNA          sporulation strain JTY78, doubly heter- <br /> apparent blots accurate underestimates.     ozygous cryl-A2::TRPI cry2-A1::LEU2. P, intron-containing pre-mRNA; M, spliced mRNA; RPS24A <br /> RPS24A transcript hybridizes 3.0-kb EcoRI-BamHI frag-      diploid constructed sequential transformation <br /> ment contains CRY2.                                          linear DNAs containingthe null alleles. <br />                                                                        JTY78 transformed plasmid pJT30 <br /> signals 3.5-kb EcoRI 2.2-kb HindIII                       containing URA3 l, sporulated, andsubjected <br /> fragments containing CRYl (Figure 2).                                  tetrad dissection, Trp+Leu+Ura+meiotic <br />    Ribosomal protein 59 encoded CRYl                         products obtained . cryl-A2::TRPI <br /> CRY2 genes essential yeast protein: cryl-                      cry2-A2::LEU2 pJT30 cells unable grow A2::TRPI null allele constructed inserting                  medium containing 5-fluoro-oroticacid, selects <br /> T R P l gene place 106 137 codonsof C R Y l .              loss plasmids bearing URA3. Taken  linear 3.1-kb HindIIIfragmentcontaining cryll-                       results indicate thatrp59encoded         A2::TRPI transformed yeast diploid                            CRYl CRY2 essential mitotic growth yeast. <br /> MMY 140 1 replace     onethe wild-type CRYI alleles              cryl-A2::TRPI CRY2 CRYl cry2-Al::LEUZ hap- cryll-A2::TRPI. Trp+ transformants se-                        loids recovered meiosis of doubly <br /> lected, sporulated, subjected tetraddissection.                  heterozygous diploids MPY 1404 JTY78 grew viable CryS spores recovered te-                     identical rates wild-type cells, suggesting nei- <br /> trads, markerssegregated expected andthe                         thermutant significant functional deficit T R P l marker tightly linked mating type                    rp59. lethality cryl-A2::TRPI cry2-AI::LEU2 <br /> locus, MATa MATa. Southern blot hybridization                        doublemutants consistent demonstratedthatthe        TRPI markercosegregated                      functional genes yeast cryl-A2::TRPI restriction fragments (data                  constructeda     cryl-A2::TRPIcry2-AI::LEU2GALI: <br /> shown) confirmed CRYl gene replacement.                         CRYl strain conditional synthesis rp59 dem- <br />    cry2-AI::LEU2 null allele constructed                    onstrate cessation synthesis rp59, 40s <br /> serting LEU2 EcoRV Asp71 8 sites                        ribosomal subunits longer assembled CRY2, thatcodons 4-89 deleted                          cells stop dividing (MORITZ et al. 1990). <br /> (Figure 7). AatII-BamHI fragment containing cry2-                       cryl-AP::TRPl mutants modestly deficient AI::LEU2 used transform diploid strain                       40s ribosomal subunits, cry2-Al::LEU2 mutants <br /> MMY 1401. Leu+ transformants selected, spor-                       contain wild-type levels 40s ribosomal subunits: <br /> ulated, subjected tetrad analysis Southern                   T o examine effect deleting CRYl CRY2 blot hybridization. tetrads yielded viable CryS                assembly ribosomes, examined translational <br /> spores, markerssegregated 2:2. T h e Leu+                       apparatus cryl-A2::TRPI    CRY2, <br />                                                                                                           CRYl             cry2- <br /> phenotype cosegregatedwith replacement wild-                     Al::LEU2, wild-type haploid strains JTY82, <br /> type 4.6-kb EcoRI CRY2 fragment predicted                       JTY8 1, JTY83,respectively. Free ribosomal sub- <br />  2.25-kb 5.25-kb EcoRI fragments CRY2-                      units, 80s monoribosomes, polyribosomes  726                                                         G . Paulovich et al. <br />                                             CRY2 RPS24A             RPL46 <br />  <br />  <br />  <br />  <br />                        ..........,-11 <br />                                                     *. . <br />                                                        :..#* <br />                                                     cdc6 <br />                                                                &apos; <br />  <br />                                                                             ura2 tif2 <br />                                                                                            11-(       -..-....-   x <br />   FIGURE6. tightly-linked cluster ribosomal protein genes CRY2, RPS24A RPL46 left arm chromosome X , <br /> centromere proximal cdc6. parentheses indicate orientation EcoR1 fragment containing gene cluster unknown. <br />  <br />                       Aa R1                    RV                  Nr                                           0a <br />  <br />  <br />  <br />  <br />                                                                L EU2 <br />                           X                                                                              S <br />   FIGURE7.-Replacement CRY2 gene cryZAI::LEU2 deletion-insertion allele. disruption depicted Figure 2. <br /> Restriction enzyme cleavage sites:  Asp718; Aa, AntII; Ba, BamH1; Bg, BgllI; RI, EcoRI; RV, EcoRV; Nr, NruI; S,SalI; X, Xhol. deletion CRY2 gene constructed removing sequences EcoRV site intron CRY2 Asp718 site codon 89 CRY2. 2.9-kb XhoISalI fragment containing yeast LEU2 gene inserted EcoRV Asp718 sites CRY2 shown. AatII-BamHI fragment containing cry2-AI::LEU2 construct transformed diploid &quot;1401        selecting Leu+. <br />  <br /> separated sucrose gradient sedimentation ex-                                 relative expression CRYl CRY2 tracts prepared fromthese strains. observed                            assayed using fusions E. coli lacZ gene codons modest decrease relative free 40s                            107 108 CRYl CRY2, respectively. Single <br /> ribosomal subunits compared free 60s ribosomal                               copiesof gene fusion individually inte- <br /> subunits cryl-A2::TRPlCRY2 mutant strain                                 grated genome yeast strain JTY83 compared wild-typecells (data shown).                                ura3-167 locus, confirmed genomic Southern hy- <br /> ratio total 40s ribosomal subunits total 60s                              bridization. units &amp;galactosidaseper mg pro- <br /> ribosomal subunits measured subjecting cell                              tein expressed strains containing CRYl-lacZ <br /> extracts centrifugation high-salt sucrose gra-                            7 units mg protein strains bearing CRY2- <br /> dients dissociate polyribosomes monoribo-                              lacZ, consistent observed relative levelsof <br /> somes subunits. ratio 40s 60s ribosomal                            CRYl CRY2 mRNAs. <br /> subunits cryl-A2::TRPl CRY2 mutant 80%                                Isolation sequencing my1 c y 2 alleles <br /> wild type (Figure 8). contrast, gra-                           conferring CfyR phenotype: mu- <br /> dient assays revealed CRYlcry2-Al::LEU2                                tations result CryR phenotype yeast map <br /> mutant detectable deficit 40s ribosomal                               cryl locus, tightly linked mating type <br /> subunits (Figure 8 datanot shown).                                          locus MAT chromosome ZZZ. Mutations cause <br />    CRY1 isexpressed higherlevelsthan C R Y 2                                 cryptopleurine resistance linked MAT deficiency 40s ribosomal subunits cryl-                           isolated, genes defined A2::TRPlCRY2 mutant CRYl cry2-                                   mutations mapped characterized <br /> Al::LEU2 mutant suggests eitherthat CRYl ex-                                   K. HOPPER,personal communication).  <br /> pressed thanCRY2, rp59 expressed                              cause rp59 products CRYl CRY2 nearly <br /> CRYl functional rp59 expressed                                identical, thoughtthat possible CRY2. relative levels CRYl CRY2 mRNAs                                obtain CryR mutants resulting mutations wild-type cells measured S1 protection                            CRY2.  mutations difficult assay using DNA fragment complementary C R Y l .                           isolate CRYl CRY2 strain. cryl mutations shownin Figure 9, CRYl mRNA present                                    isolated date recessive. higher expression <br /> 5-10-fold excess CRY2 mRNA. CRYl                                 CRYl compared CRY2 mask phenotype <br /> probe hybridizes CRY2 mRNA                                 cry2 mutation.  determinewhether <br /> CRYl mRNA, assay likely provides slight                             CryR mutants result mutations CRY2, <br /> underestimate CRY2 mRNA levels. Similar results                              selected spontaneous CryRmutants using haploid obtained hybridization RNAblotswith                                  strain containing cryl-A2::TRPl null allele. CryR <br /> gene-specific oligonucleotide probes complementary                              mutants selected plating haploid strain JTY82 thedivergent sequences 3&apos; untranslated                            (cryl-A2::TRPlCRY2) YEPD containing 10 pM <br /> regions CRYl CRY2 (data shown).                                      cryptopleurine. CRY2 DNA recovered  <br />  <br />                                                                          FIGURE9.-CRYI mRNA present higher levels CRY2 <br />                                                                       mRNA wild-type cells. Levels CRY1 CRY2 m R N        E                                                              measured S1 nuclease protection assay. RNA extracted <br />        g     0.5                                                      wild-type strain JTY83, hybridized &quot;P end-labeled <br />        cu                                                             Hindlll-BglI1 CRYl fragment XbaI-Bglll RPLI fragment. di- <br />                                                                              gested nuclease S , subjected electrophoresis denatur- <br />                                                                       ing polyacrylamide gel, exposed X-ray film. positions <br />                                                                       protected DNAs corresponding CRYl,CRY2 R P L f <br />                                                                       mRNAs indicated. shorter CRYI DNA fragment protected <br />              0.0 <br />                                                                       hybridization CRY2 mRNA (-1 50 nucleotides) compared                                                                       protected CRYI mRNA (3 16 nucleotides). CRY2 <br />                                                                       nucleotide sequence diverged CRY1 positions <br />                                                                       Bglll site intron. <br />         C                 CRYl     CfY2-Al::LEU2 <br />                                  60s <br />                                                                        mined. cry2 DNAs contain T T                                                                        TGA nonsense mutation codon CRY2. <br />                                                                        fourth cry2 mutation, L138S, results  <br />                                                                        placement amino acid rp59, Leu-138, <br />                                                                        Ser. <br />                                                                           Previously, cloned cryl resistance allele                                                                        CryR strainDPB745cry integration-excision <br />                                                                        method (LARKIN      WOOLFORD     1983). CryS strains <br />                                                                        (genotype: CRYI CRY2) transformed DNA <br />                                                                        high copy plasmid CryR, e., able <br />                                                                        grow 10 ~ L cryptopleurine. <br />                                                                                            M                cryl allele                                                                        sequenced contain L137S mutation <br />    FIGURE8. 40s ribosomal subunits decreased          codon, identical cry2 missense muta- cryl-A2::TRP1 mutant cry2-AI::LEU2 mutant.         tion described  <br /> CRY1CRY2 (wild type), cryl-A2::TRP1CRY2, CRYl cry2- <br /> AI::LEU2 cells (200 ml) grown YEPD medium mid-log <br /> phase, harvested, lysate culture subjected                                                                                                 DISCUSSION <br /> centrifugation high salt sucrose gradients. gradients collected OD260 monitored using lSC0 gradient fractiona-           report demonstrate yeast ribo- <br /> tor. positions 40s 60s ribosomal subunits indicated     somal protein rp59 encoded CRYI optical density profiles.                                    gene second, unlinked gene CRY2.                                                                        cloned CRY2 based homology previ- independently isolated CryR strains                  ously cloned CRYl gene demonstrated func- integration-excision method using CRY2 cloned                   tional rp59 expressed CRY2. CRY2 locus <br /> Y Ip5 (ROEDER   FINK 1980) amplification                  encodes protein 96% identical encoded polymerase chain reaction. nucleotide se-                      CRYI. T w o differences polypeptides  <br /> quence complete cry2 mutant allele deter-                  coded CRYI CRY2 conservative substitu- <br />  728                                            G. Paulovich et al. <br />  <br /> tions: arginine codon 72 CRY1 lysine codon      ferences expression CRYl CRY2 CRY2, cysteine codon 123 CRYl aserine            result post-transcriptional mechanisms. <br /> codon CRY2 (Figure 4). T h e drastic differ-          CRYl contributes 80% rp59 ences betweenthe rp59polypeptides encodedby          wild-type cells, observe cryl 1 null mutants locilie fouramino acids          contain 80% wild type levels 40s ribosomal theiraminotermini.       T h e inferredaminoter-        subunits grow wild-type rates. contrasts <br /> minal sequence rp59     encoded        CRYl       situation previously observed duplicated <br /> MSNVVQAR. . ., MANDLVQAR. .  rp59               essential yeast ribosomal protein genes. Disruptions encoded CRY2 (Figure 4). overall           gene duplicate pair typically <br /> primary structure rp59 highly conserved        result proportional decrease evolution, obvious divergence          gene product ribosomal subunit occurred amino termini    rp59 homologues     proteinis component(reviewed WOOL- organisms. instance, rp59 proteins        FORD WARNER1991; WOOLFORD            1991). coded yeast CRYl CRY2 genes rpSl4        presently examining coordinate regulation protein encoded mammalian RPS14 gene            CRYlICRY2 genepair. Current evidence indicates <br /> 80% identical, mammalian protein contains          expression CRY2 increases disruption additional 14 amino acids amino terminus         CRYl   LI,  G. PAULOVICH,     J. R. THOMPSON,    F. notpresent yeast rp59 (RHOADS,    DIXITand       JACOBSON J. L. WOOLFORD,       manuscript RHOUFA1986; LARKIN,THOMPSON             WOOLFORD       preparation). studyof duplicate genepair <br /> 1987).                                                     provide insight cellular mechanisms <br />    Consistent high degree evolutionary         coordinating expression ribosomal proteins. <br /> conservation exhibited rp59 family, dem-            existence second yeast gene encoding <br /> onstrated rp59 essential viability yeast    rp59 suggested mutations conferring CryR phe- required stable accumulation     notype obtained CRYl gene <br /> 40s ribosomal subunits (MORITZet al. 1990         CRY2. confirmed expectation work). Previously used cryl-A2::TRPl cry2-            isolating CryRmutants using crylACRY2 strain. <br /> Al::LEU2 GAL2::CRYI strain conditional forsynthesis        Sequencing cry1 cry2 mutant alleles revealed rp59todemonstratethat,upontermination                changing amino acid rp59 fromleucine <br /> synthesis rp59, componentsof 40s ribo-        serine, eliminating carboxy-terminal amino <br /> somal subunit continue synthesized wild-type      acid nonsense mutation causes CryR phenotype. <br /> rates d6 assemble stable 40s subunit        Similarly, mutations codons 3&#226;&#8364;&#8482; end degraded (MORITZet al. 1990).          Chinese hamster ovary gene encoding rp59hom- <br /> conditions, componentsof 60s ribosomal subunits        ologue rpS14 result emetine resistance synthesized assemble functional subunits.      cultured mammalian cells (CHANGand WASMUTH paper, determined disruption        1983; MADJARet al. 1983; RHOADS      ROUFA1985); <br /> CRYl CRY2 does measurably affect cellular           single Arg-149 Cys-149 mutation (EmR-2)causes <br /> growth rates. Cells containing cryl-A2::TRPI null        resistance low levels emetine, double <br /> allele contain modest deficit 40s subunits,     Arg-149 Cys-149 Arg-150 150 mutation containing cry2-AI::LEU2 null allele       (EmR2-2)confers resistance high levels emetine. <br /> noticeably deficient 40s subunits.                      Interestingly, 40s ribosomal subunits CHO <br />    determined CRYl expressed signifi-        EmR-2-2double mutant unstable, consistent cantly higher levels CRY2 wild-type haploid        observations thatrp59 necessary stable <br /> yeast. previously identified mutational analysis     accumulation 40s subunits (MORITZ et ul. 1990; CRYl promoter elements: UASR~G 257-nucleo- <br />                                                         study). <br /> tide richregion(LARKIN, THOMPSON                   Previously DOLZ,VAZQUEZ J M ~ N E (1982) <br /> WOOLFORD     1987). sequencesare present     demonstrated existence high affinity binding <br /> upstream yeast r p genes  tested,         site cryptopleurine 40s ribosomal subunit. shown necessary transcription        Binding cryptopleurine ribosomes isolated rp genes (reviewed WOOLFORD        1991).     CryR yeast reduced compared       CryS <br /> expect differences conserved promoters       ribosomes, account CryR resultin different levels oftranscription CRYl    phenotype.  DOLZet al. suggested cry1 <br /> compared CRY2. putative UASRPG            mutations affect binding crypto- <br /> sequences upstream CRY2 nts -377 -206,           pleurine ribosomes result conforma- <br /> followed T-rich sequence. tested      tional change ribosomes impaired translational sequences upstream CRY2          inhibition cryptopleurine bound ribosome. <br /> function transcriptional promoters. course, dif-     findings, MEADE et al.,  <br />  <br /> consistent hypothesis explain previ-                 <br />                                                                         <br /> ously confusing observations: (1) T h e original obser-                   <br /> vations CryR phenotype completely reces-                      <br /> sivein cryl/CRYl diploids obtain                          <br /> recessive CryR haploid mutants incompatible             existence duplicate, functional genes.  clear CryR phenotype              <br /> cryl CRY2 haploids results cryl                       <br /> highly expressed CRY2. Binding cryptopleu-             <br /> rine <br />         wild-type sensitive ribosomes effectively                    <br /> freezes polyribosomes containing (DOLZ,Viz-                  <br /> QUEZ J M ~ N E 1977). extent                                                                         <br /> sensitivity cryptopleurine function ratio                  sensitive resistant ribosomes. measurements                       relative expression CRYl CRY2 suggest                         cryl CRY2 strain contains 80%-90% CryR ri-               <br /> bosomes, sufficient sustain efficient translation                                                                             presence cryptopleurine, CRYl cry2                        <br /> straincontains 10-20% CryR ribosomes                                 CryS. cryl/CRYl diploid predicted       <br /> contain CryS ribosomes CryS. (2) T h e            <br />                                                             <br /> existence oftwo genes, CRY1 chromosomeI11 CRY2 chromosome X , explains diploid                    <br /> yeast monosomic chromosome I11 carrying cryl                  phenotypically CryS (MEADE, RILEYand MANNEY                         <br /> 1977; CLARK CARBON        1983). suggest            <br /> unequal expression ofthe CRYl CRY2 genes                         <br /> explains previously isolated mutations                    <br /> conferring resistance tocryptopleurinemaptothe                    <br /> CRYl locus.                                                            <br /> </body></html>