<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Aug. 2003, p. 4490&#226;&#8364;&#8220;4498                                                                               Vol. 185,  15 <br /> 0021-9193/03/$08.00&#226;&#171;&#185;0 DOI: 10.1128/JB.185.15.4490&#226;&#8364;&#8220;4498.2003 <br /> Copyright &#194;&#169; 2003, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />          Essential Nature mreC Determinant Bacillus subtilis&#226;&#8364;&#160; <br />                                            Joong-Chul Lee George C. Stewart* <br />                            Department Diagnostic Medicine/Pathobiology, College Veterinary Medicine, <br />                                          Kansas State University, Manhattan, Kansas 66506 <br />                                                Received 10 March 2003/Accepted 29 April 2003 <br />  <br />              mre genes Escherichia coli Bacillus subtilis cell shape determination genes. Mutants affected <br />           mre function spheres instead normal rods. mre determinants required           viability E. coli, mreB determinant essential gene B. subtilis. Conflicting results  <br />           ported membrane-associated proteins MreC MreD essential proteins.  <br />            MreB protein studied  roles MreC MreD proteins <br />           cell shape determination unknown. constructed strain B. subtilis expression mreC <br />           determinant dependent addition isopropyl-&#226;?&#164;-D-thiogalactopyranoside culture medium. <br />           Utilizing conditional strain, shown mreC essential gene B. subtilis. Furthermore,           shown cells lacking sufficient quantities MreC undergo morphological changes,  swelling           twisting cells, followed cell lysis. Electron microscopy utilized demonstrate           polymeric material accumulated division septum cells presence material <br />           correlated bending cell. best explanation results MreC protein involved <br />           control septal versus long-axis <span id='am-29' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-30' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="murein" datatype="xsd:string"></span>peptidoglycan</span> synthesis, cells lacking MreC perform aberrant <br />           septal peptidoglycan synthesis, lysis results deficiency long-axis peptidoglycan synthesis. <br />  <br />  <br />    Bacteria characteristic shapes, important                    prevalence site septum formation leads  <br /> taxonomic features. bacteria possess fixed shape, al-                   mation coccobacilli cocci. Bacteria char- members genus Arthrobacter transition rods                  acteristic coccal shape fall categories. possess cocci depending growth conditions (12). bacterial                    site septum formation exist cocci, <br /> cell wall thought major determinant cell shape,                carry peptidoglycan biosynthetic sites purified <span id='am-40' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-41' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-46' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span>murein</span> sacculus retains characteristic mor-                transition rods septum-specific site blocked <br /> phology cell, specific changes <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> com-                 mutation antibiotic treatment (21). <br /> position peptidoglycan noted shape-altered                    Mutations genome Escherichia coli result mutants (25).  shape determination complicated                    altered cell morphology identified, precise <br /> process, involves topology peptidoglycan                 biochemical functions affected genes biosynthesis contributions components                 determined. mre operon E. coli identified locus <br /> cell envelope involvement cell division ma-                  associated cell shape determination sensitivity chinery. mutation tagF (rodC) determinant Bacillus                 amdinocillin (11, 27&#226;&#8364;&#8220;30). operon consists genes, <br /> subtilis gives rise shift rod-shaped spherical cells,             mreBCD, orfE, cafA. mutation mreB deletion indicating contribution teichoic acid determination                mreBCD produces morphological shift cell population, cell morphology (4, 16, 23).                                                resulting spherical cells. MreB protein shows sequence <br />    model bacterial shape regulation, competing-                  similarity members Hsp70 superfamily, sites model, relates cell shape balance cell                FtsA cell division protein (6, 14). Inactivation mreB wall synthesis long axis rod-shaped cell                    associated increased activity FtsI (penicillin <br /> septum-specific peptidoglycan synthesis, proposed <br />                                                                                binding protein 3) protein, responsible septal pep- <br /> (21, 24). model proposes shape determination                                                                                tidoglycan synthesis (30). proposed increase <br /> bacterial rods depends activity peptidoglycan <br />                                                                                leads hyperseptation activity spherical cell formation. <br /> synthesis reactions (sites) mutually exclusive.  <br />                                                                                Introduction mre genes multicopy plasmid led sequently, lateral wall extended septum <br />                                                                                reduction FtsI activity associated filamentation phe- <br /> formation step cell division, septal peptidoglycan syn- <br />                                                                                notype, presumably increased production MreB. <br /> thesis does occur lateral wall elongation. shape <br />                                                                                basis observations, postulated MreB cell determined balance competing <br />                                                                                negative regulator FtsI (30). roles MreC, <br /> reactions. normal balance produces rods, abnormal <br />                                                                                MreD, OrfE proteins shape determination remain  <br /> prevalence lateral wall elongation yields filaments,                                                                                defined. <br />                                                                                   MreB homolog B. subtilis contained    * Corresponding author. Mailing address: Department Diagnos-             divIVB minicell operon (20, 26). operon contains tic Medicine/Pathobiology, College Veterinary Medicine, Kansas              mreBCD determinants minCD minicell-associ- <br /> State University, 1800 Denison Ave., Manhattan, KS 66506-5603. Phone: <br /> (785) 532-4419. Fax: (785) 532-4039. E-mail: stewart@vet.ksu.edu. <br />                                                                                ated genes. mutant allele mreD, rodB1, induces  <br />    &#226;&#8364;&#160; Contribution 03-298-J Kansas Agricultural Experiment             mation spherical cells.  B. subtilis mre genes Station.                                                                       associated cell shape determination.   <br />                                                                         4490 <br />  VOL. 185, 2003                                                                                                                B. SUBTILIS mreC             4491 <br />  <br />  <br />                                               TABLE 1. Bacterial strains plasmids used study <br />   Strain plasmid                                               Genotype description                                                   Source reference <br />  <br /> B. subtilis 168 strains <br />   KSS1001                       trpC2                                                                                                       K. Bott <br />   KSS1570                       amyE::pdivIVB mreD-minCD                                                                                    study <br />   KSS1571                       &#226;?&#8364; pLEE100, amyE::pdivIVB mreD-minCD                                                                         study <br />   KSS1572                       trpC2, &#226;?&#8364; pLEE100                                                                                            study <br />   KSS1573                       KSS1571 (pLEE504)                                                                                           study <br />   MO1099                        amyE::erm                                                                                                   9 <br />  <br /> E. coli strains <br />   DH5&#226;?&#163;                          endA1 gyrA96 hsdR17 &#226;&#338;&#172;lacU169 (&#226;?&#190;80lacZ&#226;&#338;&#172;M15) recA1 supE44 thi-1                                               BRLc <br />   M15                           Strr lacZ&#226;&#338;&#172;M15 (pRep4 Kmr)                                                                                   Qiagen <br />  <br /> Plasmids <br />   pCHA2489                      mreC ORFa PCR product cloned pT7Blue                                                                   study <br />   pCHA2506                      NotI  mreC, blunted)-SalI (vector site) mreC DNA fragment pCHA2489 cloned                      study <br />                                   EcoICRI-SalI site pQE30 <br />   pCR2.1                        PCR fragment cloning vector, Apr Kmr                                                                        Invitrogen <br />   pDG364                        amyE locus integration vector                                                                               BGSCd (9) <br />   pDG3053                       Shuttle expression plasmid utilizing S. aureus <span id='am-5' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span>lactose</span> operon promoter; Apr Kanr                        study <br />   pLEE50                        pUS19 lacI spac EcoRI-BamHI insert pSI-1                                                      study <br />   pLEE100                       MreC-inducible expression integration vector                                                                study <br />   pLEE200                       pDG364 containing pdivIVB mreC-minCD                                                                        study <br />   pLEE504                       pDG3053 containing truncated mreCb                                                                          study <br />   pQE30                         histidine tag vector, Apr                                                                               Qiagen <br />   pSI-1                         IPTG-inducible expression shuttle plasmid                                                                   D. Henner (2, 33) <br />   pT7Blue                       PCR product cloning vector, Apr                                                                             Invitrogen <br />   pUS19                         pUC19 Spr expressed B. subtilis                                                                     3 <br />       ORF, open reading frame. <br />   b <br />     Truncated mreC (contains bp 1 540). <br />   c <br />     BRL, Bethesda Research Laboratories. <br />   d <br />     BGSC, Bacillus Genetic Stock Center, Ohio State University. <br />  <br />  <br />  <br /> reported B. subtilis mre genes appear essential                 cell. studies confirmed earlier suggestions cell viability, unlike E. coli counterparts (20, 26).            mreB determinant essential cell viability additional determinants products substantial se-                    mreC mreD determinants inactivated loss <br /> quence identity MreB identified B. subtilis                viability &#226;&#8364;&#339;mild phenotypic consequences.&#226;&#8364;? chromosome. mbl determinant appears play role                      Streptomyces coelicolor mreB determinant shown cell morphology, inactivation nonessential determi-                essential viability, mreC gene inactivated <br /> nant does result spherical cell formation (1). mreBH                gene disruption resulted mutants viable determinant, maps near kinC B. subtilis chromo-                   showed significant growth retardation comparison  characterized (GenBank accession number                     wild type (7). better understand biological role D37799). B. subtilis Mre proteins studied               MreC protein B. subtilis reconcile conflicting E. coli counterparts. sequence similarity              conclusions regarding essential nature determinant, <br /> E. coli Mre proteins, phenotype rodB1 allele,                   created strain B. subtilis conditionally expresses <br /> suggests function involves interaction                 MreC protein examined effect depletion septum-specific peptidoglycan biosynthetic enzyme,                    protein growth morphology cell. <br /> ly, Pbp2B (20, 26, 32). <br />    Introduction mreC mreD genes Bacillus <br /> stearothermophilus B. subtilis resulted overexpression                                      MATERIALS METHODS <br />  protease (18). effect protease expression required                     Bacterial strains, plasmids, growth conditions. bacterial strains presence mre determinants; effect seen                 plasmids used study listed Table 1. Media growth conditions                                                                                E. coli B. subtilis described previously (8, 26). Phleomycin (zeocin) determinant introduced host.                 utilized 10 7 &#226;?&#174;g/ml selection E. coli B. subtilis, respec- determined effect protease expression                  tively; kanamycin utilized 20 5 &#226;?&#174;g/ml selection E. coli direct indirect effect production B. stearo-                B. subtilis, respectively; spectinomycin utilized concentration 100 <br /> thermophilus MreCD proteins B. subtilis. MreC protein                   &#226;?&#174;g/ml, chloramphenicol utilized 10 &#226;?&#174;g/ml. <br />                                                                                   grow B. subtilis strain KSS1571, tryptic soy broth (TSB) (Difco) tryptic shown localize division septum dividing <br />                                                                                soy agar (TSA) used. culture media supplemented specti- <br /> cells B. subtilis (19).                                                     nomycin 2 mM isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG). cells <br />    Jones coworkers indicated MreB Mbl pro-                    grown broth overnight, harvested centrifugation, washed <br /> teins B. subtilis filamentous helical structures              equal volume phosphate-buffered saline, diluted 100-fold fresh cytoskeleton-like role bacterial shape determination                 broth, grown 6 h 100 &#226;?&#174;g spectinomycin ml various <br />                                                                                amounts IPTG. incubations performed 37&#194;&#176;C shaking 200 <br /> (17). proposed model MreB protein                       rpm, cell density monitored spectrophotometrically 500 nm. concerned cell width control, Mbl im-                     end 6-h cultivation period, absorbance values determined. cells <br /> portant maintaining linearity longitudinal axis               harvested centrifugation, cells resuspended iden- <br />  4492       LEE STEWART                                                                                                                             J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1.   Integration pLEE100 B. subtilis chromosome. result plasmid integration, mreB determinant divIVA <br /> operon remains control natural promoter (pdivIVA), mreB-distal genes operon control IPTG-inducible spac promoter (pspac). (B) mreD-minC-minD determinants, expressed divIVA operon promoter, positioned amyE locus (9). Abbreviations: Apr, ampicillin resistance determinant; Spr, spectinomycin resistance determinant; pspac, spac promoter; <br /> pdivIVB, divIVB operon promoter. <br />  <br />  <br /> tical cell densities. milliliter samples collected electron microscopy      eliminate possible polar effects mreC-distal genes divIVB <br /> observation methylene blue staining conventional microscopy. remainder     operon, merodiploid strain B. subtilis constructed extra copy culture utilized Western blot analyses muramic acid assays.        mreD minCD, controlled divIVB operon promoter, present.    induce switch teichoic acid teichuronic acid synthesis, low-        divIVB promoter amplified PCR (forward primer, 5&#226;&#172;&#732; CGGAATTCATC <br /> phosphate medium used described previously (31).                               TATTGTCCACCCGCG 3&#226;&#172;&#732;; reverse primer, 5&#226;&#172;&#732; GCTCTAGAGCAAAAATAC <br />    General methods. Competent E. coli B. subtilis cells prepared         CCTAAAGGG 3&#226;&#172;&#732;) cloned using EcoRI XbaI sites incorporated transformed methods Dagert Ehrlich (10) Erickson                5&#226;&#172;&#732; ends primers. promoter fragment positioned Copeland (13), respectively. Plasmid DNA E. coli prepared             mreD-minC-minD determinants (amplified using 5&#226;&#172;&#732; GCTCTAGAACAGAG <br /> alkaline lysate procedure Birnboim Doly (5). Isolation chromosomal          GAGGAAGGATCG 3&#226;&#172;&#732; 5&#226;&#172;&#732; CGGGATCCATTAAGATCTTACTCCG 3&#226;&#172;&#732; plasmid DNAs B. subtilis performed described previously (8, 26).      PCR primers, introducing XbaI BamHI sites, respectively)    Construction MreC-inducible strain. HincII-HindIII sequence           inserted plasmid pDG364 (9) EcoRI-BamHI fragment. cloned multiple cloning site pUS19 (3) removed, DNA fragment              PCR products sequenced ensure constructs constructed <br /> containing spac promoter lacI coding region excised              correctly. resulting plasmid, pLEE200, transformed B. subtilis <br /> plasmid pSI-1 (2, 33) EcoRI BamHI ligated modified         strain MO1099 create strain KSS1570, merodiploid mreD, <br /> pUS19 plasmid cleaved enzymes, generating plasmid pLEE50.             minC, minD genes. second copy determinants positioned <br /> 500-bp fragment encoding 3&#226;&#172;&#732; end mreB generated PCR                  amyE locus (9) (Fig. 1B). <br /> positioned HindIII-SphI fragment downstream IPTG-inducible spac              express truncated form MreC, sequence corresponding promoter plasmid pLEE50 create pLEE100 (Fig. 1). plasmid               ribosome binding site 504 bp mreC open reading frame replicate B. subtilis cells, contains spectinomycin resistance     amplified PCR (forward primer, 5&#226;&#172;&#732; GAGCTCGAGTTCAATAGAAGAG 3&#226;&#172;&#732;; <br /> determinant expressed gram-positive host. Integration      reverse primer, 5&#226;&#172;&#732; GCATGCCAGACGTAAAATTGTTAA 3&#226;&#172;&#732;). PCR prod- <br /> plasmid B. subtilis chromosome mreB locus results mreB             uct cloned SacI-SphI fragment immediately downstream Staph- <br /> expression divIVB operon promoter, expression mreCD di-            ylococcus aureus lactose operon promoter plasmid pDG3053 create plasmid <br /> rected spac. distal determinants minC minD driven         pLEE504. plasmid shuttle plasmid gram-positive replication func- spac promoter weak internal promoter divIVB operon (26).         tions provided staphylococcal plasmid pUB110 ampicillin resistance <br /> 3&#226;&#172;&#732; terminus mreB positioned downstream spac, instead 5&#226;&#172;&#732;              replication functions E. coli provided pUC19. Expression terminus mreC. reason avoid duplication       promoter constitutive plasmid lacks functional lacR repressor- <br /> mreC determinant integration plasmid. preferable 5&#226;&#172;&#732;      encoding determinant. <br /> untranslated leader mreC-containing transcript expressed          Scanning electron microscopy. B. subtilis cells cultured mid- <br /> truncated mreC, include transmembrane domain (19).                    logarithmic growth phase, 1 ml culture fixed overnight 4&#194;&#176;C  VOL. 185, 2003                                                                                                               B. SUBTILIS mreC        4493 <br />  <br /> 20 volumes modified Karnowsky&#226;&#8364;&#8482;s fixative (2% paraformaldehyde, 2.5% glu- <br /> taraldehyde, 1.7 mM CaCl2 0.1 M cacodylate buffer [pH 7.4]). cells fixed, washed twice 0.1 M cacodylate buffer (pH 7.4) postfixed 1% osmium tetroxide 0.1 M cacodylate buffer overnight 4&#194;&#176;C. cells washed twice double-distilled water, resus- <br /> pended double-distilled water, mounted coverslips coated poly-L- <br /> lysine, dehydrated series alcohol solutions (30, 50, 70, 95, 100%). <br /> Dehydrated samples treated 5 min hexamethyldisilazane (Electron <br /> Microscopy Supplies, Ft. Washington, Pa.), dried, mounted, sputter coated gold (S150A sputter coater). Samples viewed Hitachi H-300 <br /> electron microscope 3010 scanning image accessory. <br />     Transmission electron microscopy. B. subtilis cells cultured mid- <br /> logarithmic growth phase. cells pelleted fixed modified Kar- <br /> nowsky&#226;&#8364;&#8482;s fixative, washed, postfixed 1% osmium tetroxide described  5-min washes cold double-distilled water, cells immobilized 2% TSA dehydrated series ethanol solutions (50, 70, <br /> 70, 95, 100%). Dehydrated specimens stained 60 min bloc 5% uranyl acetate 70% ethanol, dehydration completed 20-min rinses 100% acetone, used transitional solvent. cell blocks infiltrated increasing concentrations resin acetone, ending 100% resin periods 2 12 h 8 12 h. samples embedded Epon LX112 embedding medium 18 20 h 42&#194;&#176;C 24 h 60&#194;&#176;C. Embedded samples trimmed sectioned            FIG. 2. Growth MreC expression MreC-inducible strain. <br /> Ultracut E-Reichert-Jung ultramicrotome (C. Reichert Optische Werke AG,               B. subtilis KSS1571 cultured TSB containing 2 mM IPTG  <br /> Vienna, Austria). sections (approximately 80 &#226;?&#174;m  sputter coated       night 37&#194;&#176;C, diluted 1:100 fresh medium supplemented dif- <br /> silver gold retrieved copper grids stained uranyl acetate      ferent concentrations IPTG, incubated 6 h. MO1099, lead citrate. Grids examined viewed Hitachi H-300 electron        wild-type MreC control strain, cultured TSB IPTG. <br /> microscope.                                                                           Growth determined absorbance 500 nm (B). cells <br />     Muramic acid assay. peptidoglycan content bacterial cells          harvested lysed, cell lysates electrophoresed <br /> determined method Hadzija (15). B. subtilis cells cultured TSB      Western blotted. blots probed rabbit anti-MreC late-logarithmic growth phase, harvested centrifugation, washed      antibody  . panel B, mean values determinations <br /> TE buffer (10 mM Tris-HCl, 1 mM EDTA [pH 8.0]). cell amounts            shown, range values indicated. <br /> adjusted culture densities (A500 &#226;&#171;&#189; 1.8). cells lysed <br /> using mini beadbeater-8 (Biospec Products, Bartlesville, Okla.). Following cell <br /> disruption, samples incubated 15 min 75&#194;&#176;C. Samples kept       defined. examine effect loss MreC produc- <br /> &#226;&#171;&#186;80&#194;&#176;C assayed. cell wall samples resuspended 20 ml                                                                                       tion B. subtilis, created strain conditional expres- <br /> chloroform-saturated TE buffer. DNase (0.5 ml 1-mg/ml solution) RNase (1.0 ml 5-mg/ml solution) added incubated 37&#194;&#176;C 30          sion MreC protein. construct pLEE100 min. Trypsin (100 &#226;?&#174;g) added incubated 37&#194;&#176;C 6 h. cell wall          depicted Fig. 1. eliminate possible polar effects fraction recovered centrifugation (12,000 &#226;&#171;&#187; g, 45 min) washed          mreC-distal genes divIVB operon, merodiploid strain <br /> time TE buffer.                                                                  B. subtilis, KSS1571, constructed. strain, extra <br />     Aliquots lysed cells brought 0.5-ml volume 1.0 M NaOH incubated 38&#194;&#176;C 30 min. 0.5 ml 0.5 M H2SO4 5 ml                                                                                       copy mreD minCD, controlled divIVB operon pro- <br /> concentrated H2SO4 added, samples placed boiling water         moter, inserted amyE locus chromosome bath 5 min. samples cooled, 0.05 ml 4% (wt/vol) CuSO4        B. subtilis. Western blot analysis utilizing rabbit polyclonal <br /> &#194;&#183; 5H2O 0.1 ml 1.5% (wt/vol) p-hydroxydiphenyl  95% ethanol)            anti-MinC antiserum indicated MinC produced added. samples incubated 30&#194;&#176;C 30 min, absorbance <br />                                                                                       normal levels IPTG-independent fashion, indicating determined 560 nm. standard curve consisting 0 20 &#226;?&#174;g muramic <br /> acid (Sigma Chemical , St. Louis, Mo.) utilized assay.                  determinants amyE locus expressed (data     Preparation polyclonal antiserum MreC. Histidine-tagged MreC           shown). <br /> protein purified, anti-MreC polyclonal antiserum raised rabbits        B. subtilis strain KSS1571 fails grown TSA plates lack- <br /> previously described (19).                                                            ing inducer MreC expression. residual growth     Western blot analysis. B. subtilis cells cultured TSB IPTG mid-logarithmic growth phase. cultures densities deter- <br />                                                                                       initial area streak plate obtained, presum- <br /> mined spectrophotometrically 500 nm, culture volumes adjusted         ably intracellular carryover IPTG  equivalent cell densities. Equivalent amounts cells cultures      night broth culture used inoculate plates.  harvested centrifugation resuspended 1 ml TE buffer. cells      strain conditional mutant dependent lysed using mini beadbeater-8, lysate loaded sodium <br />                                                                                       presence IPTG culture medium growth. dodecyl sulfate&#226;&#8364;&#8220;4 20% polyacrylamide gradient gels (Ready Gels; Bio-Rad). electrophoretically resolved proteins transferred nitrocellulose        result suggests MreC essential B. subtilis viability. <br /> filter probed anti-MreC antiserum. Goat anti-rabbit immunoglob-             Effect MreC concentration cell growth. mreC <br /> ulin G conjugated alkaline phosphatase (Sigma Chemical Company) used           appeared required growth, B. subtilis KSS1571 secondary antibody, immunoreactive proteins detected using        cultured TSB containing different concentrations IPTG <br /> nitroblue tetrazolium 5-bromo-4-chloro-3-indolylphosphate substrates. <br />                                                                                       (0, 10, 50, 250 &#226;?&#174;M) investigate growth yields                                                                                       B. subtilis strain function IPTG concentration. addi- <br />                                                                                       tion, Western blots cellular proteins prepared, utilizing <br />                                   RESULTS <br />                                                                                       rabbit polyclonal anti-MreC antibody. growth <br />   Construction MreC-inducible strain.                              achieved concentration IPTG correlated MreB protein proposed helical cytoskeletal                           MreC protein produced, indicated Western <br /> element involved cell length determination B. subtilis (17),                    blot analysis (Fig. 2). Low levels MreC observed functions mreC mreD determinants                              absence IPTG induction result leaky unin- <br />  4494     LEE STEWART                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />  FIG. 3. Morphology B. subtilis cells. Smears MO1099 (wild-type parent)  , KSS1571 cultured presence 250 &#226;?&#174;M IPTG (B), KSS1571 cultured absence IPTG (C) stained crystal violet examined microscopy. images magnification. <br />  <br />  <br />  <br /> duced expression spac promoter carryover IPTG            cell morphology, phenotypic changes B. subtilis inoculum culture. growth level,             KSS1571 examined light microscopy differ- MreC present cells, higher wild-type strain          ent IPTG conditions, 250 0 &#226;?&#174;M, compared B. subtilis achieved strain KSS1571 pres-         &#226;&#8364;&#339;wild-type&#226;&#8364;? MO1099 (Fig. 3). MO1099 normal rod <br /> ence 250 &#226;?&#174;M IPTG. Increasing IPTG concentration                  shape (Fig. 3A), altered cell morphologies ob- 250 &#226;?&#174;M did noticeably improve level growth             served KSS1571 cultures. 250 &#226;?&#174;M IPTG, cells MreC production, did eliminate changes             shorter, evidence curvature cells, <br /> cell morphology described (data shown).                especially ends (Fig. 3B). absence IPTG  <br /> pendence cell growth MreC production supports                 duction, swollen cells observed, cells conclusion MreC protein essential protein             spherical, presence debris cell ghosts indicated <br /> B. subtilis. Integration pLEE100 B. subtilis lacking        lysis occurred (Fig. 3C). <br /> extra copies mreD minCD determinants (KSS1572) gives                Effect MreC cell morphology B. subtilis. growth profile KSS1571, indicating             better define morphological changes obtained different <br /> increased expression divIVB-distal determinants             IPTG concentrations, cells prepared scanning elec- responsible observed growth deficit.                        tron microscopy (Fig. 4). Reduction cellular MreC levels <br />    determine effects reduced mreC expression levels           resulted shift normal rod shape bent <br />  <br />  <br />  <br />  <br />   FIG. 4. Scanning electron microscopy MreC-depleted B. subtilis cells. Images strain MO1099   KSS1571 (B F). MreC-inducible strain cultured presence 250 &#226;?&#174;M (B), 50 &#226;?&#174;M (C), 10 &#226;?&#174;M (D) IPTG absence IPTG (E F). <br /> (F) MreC-inducible strain cultured presence 0.3 M sucrose. Bars micrographs indicated sizes micrometers. <br />  VOL. 185, 2003                                                                                                       B. SUBTILIS mreC           4495 <br />  <br />  <br />  <br />  <br />   FIG. 5. Transmission electron microscopy MreC-depleted B. subtilis cells. Images strain MO1099   KSS1571 (B F). MreC-inducible strain cultured presence 250 &#226;?&#174;M (B), 50 &#226;?&#174;M (C), 10 &#226;?&#174;M (D) IPTG absence IPTG (E F). <br /> (F) MreC-inducible strain cultured presence 0.3 M sucrose. arrows panels C D indicate areas polymer <br /> accumulation. arrowhead panel E indicates region cell wall thinner ruptured. Bars, 1 &#226;?&#174;m. <br />  <br />  <br />  <br /> rod, bend usually occurring near end cell,        ent concave bent cell. basis giving &#226;&#8364;&#339;shepherd&#226;&#8364;&#8482;s crook&#226;&#8364;? appearance (Fig. 4B C).            location cell envelope appearance 250 &#226;?&#174;M IPTG, hook observed end cell,            electron microscope, likely accumulated material rest cell having normal appearance (Fig. 4B).        peptidoglycan. concentration IPTG reduced, bent rods begin            lower IPTG concentrations, accumulation poly- <br /> swell, indicated increased cell diameter (Fig. 4C),      meric material increased. cells began evidence twist irregular shapes (Fig. 4C D). reduction        swelling rounding, twisting rod-shaped cells oc- IPTG concentrations produced cells irregular spher-        curred (Fig. 5C D). Removal IPTG inducer ical shape reduction cell length increased           B. subtilis KSS1571 resulted cells irregular-appearing <br /> diameter relative control strain MO1099. Culture        septa, giving appearance different-sized daughter cells <br /> absence IPTG results little growth. condi-          (Fig. 5E).  result sectioning <br /> tions, B. subtilis cells predominantly spherical, resid-     cells twisted. frequency ual rods tended remain associated chains, cell         septation affected, producing different- <br /> lysis evident production ghost cells (Fig. 4E).       sized progeny cells. micrographs suggest mislocaliza- <br /> MreC-depleted B. subtilis appear vulnerable osmotic          tion misdirection septum formation occur pressure-induced lysis weakened cell wall structure.        MreC-depleted cells. Cell lysis typical observation minimize cell lysis osmotic imbalance, 0.3 M sucrose           absence IPTG. cell walls resulting ghosts did <br /> incorporated growth medium containing 0 &#226;?&#174;M IPTG.             uniform thickness (Fig. 5E). apparent rupture condition, majority cells assumed             point cell wall thinner rest wall (Fig. <br /> spherical shape (Fig. 4F). observed morphological shifts          5E). Sucrose supplementation growth medium reduced stabilization cells iso-osmotic environment      apparent cell lysis (Fig. 5F). Growth suggest reduction MreC levels associated deficiency cell wall structure bacteria. <br />    Transmission electron microscopy used examine                                                                               TABLE 2. Muramic acid content MreC-deficient cells <br /> ultrastructure MreC-deficient strain (Fig. 5). IPTG concentration 250 &#226;?&#174;M, rod-shaped cells showed               Strain <br />                                                                                                  IPTG concn                   Muramic acid content                                                                                                     (&#226;?&#174;M)                     (&#226;?&#174;g/mg cells [dry wt]) <br /> evidence bending. bending appeared cen- <br /> tered site septum formation (Fig. 5B). Examination         KSS1001                          0                            1.9 &#226;&#171;&#190; 0.1 septum site indicated accumulation polymeric         KSS1571                         10                            1.8 &#226;&#171;&#190; 0.1 <br /> material appeared cell site septa-       KSS1571                         50                            2.0 &#226;&#171;&#190; 0.2 <br /> tion. Furthermore, accumulated material pres-                                                                                       Values means &#226;&#171;&#190; standard deviations determinations. <br />  4496     LEE STEWART                                                                                                            J. BACTERIOL. <br />  <br />  <br />                                                                          conditions allowed progression cellular defect pro- <br />                                                                          ceed longer period time. longer time,                                                                          accumulation polymeric material progressed                                                                          septum site spread surface spherical cells <br />                                                                          (Fig. 5F). <br />                                                                             Muramic acid content MreC-depleted B. subtilis. MreC <br />                                                                          depletion resulted morphological changes lysis B. <br />                                                                          subtilis, suggesting defect cell wall biosynthesis                                                                          involved. Peptidoglycan major structural component                                                                          gram-positive cell wall comprises 40 90% cell <br />                                                                          wall. determine deviation pepti- <br />                                                                          doglycan content cells conditionally express MreC <br />                                                                          grown medium containing various amounts <br />                                                                          IPTG, muramic acid assay used. Muramic acid assays <br />                                                                          study suffer certain limitations. lower  <br />                                                                          centrations IPTG, cell death lysis occur cultures, <br />                                                                          affect muramic acid recovered                                                                          dry weight harvested cells. Despite limitations, <br />                                                                          muramic acid content remained constant different <br />                                                                          IPTG concentrations did differ wild-type <br />                                                                          level (Table 2). <br />                                                                             conclusion analysis polymeric material <br />                                                                          detected electron microscopy represents substance                                                                          peptidoglycan material peptidoglycan                                                                          distribution, total cell wall material, <br />                                                                          altered MreC-deficient strain. <br />                                                                             evaluate possibility teichoic acid                                                                          peptidoglycan accumulated polymer, B. subtilis <br />                                                                          KSS1571 cultured reduced phosphate medium.                                                                          conditions, teichoic acid replaced teichuronic <br />                                                                          acid. morphological changes associated defective <br />                                                                          teichoic acid resulting tagF (rodC) mutation elim- <br />                                                                          inated replacement teichoic acid teichuronic acid <br />                                                                          growth conditions (31). KSS1571 cul- <br />                                                                          tured phosphate-limiting phosphate-replete growth <br />                                                                          conditions, differences seen MreC-dependent <br />                                                                          growth, morphology, polymeric material accumulation                                                                          cells (data shown).  switch synthesis <br />                                                                          teichoic acid teichuronic acid did override                                                                          MreC-associated defect, appears accumulated ma- <br />                                                                          terial teichoic acid. <br />                                                                             Lack complementation MreC activity truncated <br />                                                                          mreC mutant. mreC mutant described Jones et al.                                                                          created integrating plasmid containing internal frag- <br />                                                                          ment mreC  242-bp fragment initiating unique <br />  <br />  <br />  <br />  <br />                                                                          KSS1573 cultured presence 1 mM IPTG; E, KSS1573 cul- <br />                                                                          tured absence IPTG. (C) Western blot B. subtilis cell lysates <br />                                                                          probed rabbit anti-MreC antiserum. Lanes: 1, MO1099 (wild-type <br />                                                                          control); 2, molecular size markers; 3, KSS1571 grown presence <br />                                                                          250 &#226;?&#174;M IPTG; 4, KSS1571 grown absence IPTG; 5, <br />                                                                          KSS1573 grown presence 250 &#226;?&#174;M IPTG; 6, KSS1573 grown <br />    FIG. 6.   Map pLEE504, shuttle plasmid expresses         absence IPTG. positions length MreC (32 kDa) 168-residue truncated form MreC S. aureus lac pro-       truncated form MreC (19 kDa) indicated. moter. ampicillin resistance determinant (<span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>), replication pro-     cell lysate loaded times used blot <br /> tein gene (rep), kanamycin resistance determinant (kan), phleomycin      shown Fig. 2A. (D) Photomicrographs methylene blue-stained <br /> (zeocin) resistance determinant (ble), NdeI (Nd) PvuII (Pv)      cells. B. subtilis KSS1571  c) KSS1573 (b d) restriction sites shown. (B) Growth B. subtilis strains TSB.   cultured presence 1 mM IPTG  b) absence Symbols: &#226;&#8364;&#353;, KSS1001 (wild type); &#226;&#8211;&#160;, KSS1571 cultured presence     IPTG (c d). arrows panel b indicate cells appear 1 mM IPTG; &#228;&#352;?, KSS1571 cultured absence IPTG; F,             swollen bent. <br />  VOL. 185, 2003                                                                                                      B. SUBTILIS mreC             4497 <br />  <br />  <br /> HindIII site) B. subtilis chromosome (17). inte-        water cell. swelling cell exaggerates gration event generate truncated expressed copy           twisting cell. Eventually, cell wall ruptures, mreC positions 1 504 open reading frame,           cell lyses. loss wall integrity loss sulting loss 122 <span id='am-22' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> residues protein.     regulated switch septum synthesis lateral wall exten- unable obtain viable transformants B.             sion abnormal septal wall synthesis pattern. Preven- <br /> subtilis utilizing procedure.                                   tion cell lysis culturing cells iso-osmotic envi- <br />    evaluate biological activity truncated MreC         ronment leads complete progression thickened wall <br /> protein, truncated allele mreC (contains bp 1 504)     perimeter cell. cells positioned downstream constitutive S. aureus lactose op-        spherical shape, consistent loss lateral wall extension <br /> eron promoter (22) plasmid pDG3053 create pLEE504              capability. <br /> (Fig. 6A). plasmid introduced mreC expres-            septation process B. subtilis involves formation sion strain KSS1571 produce KSS1573. isolate exhibits        FtsZ ring, structure recruits division- limited growth ultimately lyses inclusion       associated proteins (2). division ring forms leading IPTG culture medium, indicating truncated         edge constriction progresses division <br /> MreC protein lacks sufficient biological activity support         process.  evidence sidedness <br /> growth (Fig. 6B). Western blot analysis indicated trun-     cytokinesis ring. electron microscopic observations <br /> cated MreC protein expressed cell, lack          reported indicate septal ring associated expression explanation failure comple-        cytokinesis B. subtilis does sidedness  ment MreC deficiency (Fig. 6C). fact, presence         overexpression wall polymeric material occurs plasmid resulted higher level expression wild-type    cell. differences cell envelope, MreC cell. results presence pUC          ring formed FtsZ division proteins, accounts <br /> plasmid-derived lactose operon operator site plasmid,         septal ring asymmetry unknown. able bind LacI repressor protein titrate       mreC determinant essential determinant B. repressor spac promoter operator. Despite          subtilis appears function septation process. MreC <br /> increased production wild-type MreC, culture grew       shown localize division septum B. subtilis <br /> poorly (Fig. 6B) displayed characteristic cell swell-   (19). Truncation protein destroys proper functioning <br /> ing twisting MreC-depleted cells (Fig. 6D). strain        cell, 168-amino-acid polypeptide does ap- <br /> expressing truncated MreC protein grew poorly          pear able compete length version KSS1571 absence presence IPTG.                protein. truncated protein able compete truncated MreC protein able complement pheno-          wild-type protein interaction compo- <br /> type associated loss MreC protein cells,      nent(s) septation apparatus, MreC multimeric appears able compete wild-type protein.         protein, able included MreC complex, gives cell appearance severe MreC-            inclusion result loss biological activity. <br /> deficient phenotype warranted level length <br /> MreC cell.                                                                             ACKNOWLEDGMENTS <br />                                                                         thank David George technical assistance operation                                                                      DNA sequencer, Lloyd Willard electron microscopy,                          DISCUSSION                                  Patricia W. Stewart DVM assistance production                                                                      anti-MreC antiserum. <br />    competing-sites model peptidoglycan assembly             work supported grant GM57049 Na- <br /> predicts peptidoglycan assembly long axis      tional Institutes Health. <br /> cell septal peptidoglycan synthesis division mu- <br />                                                                                                      REFERENCES <br /> tually exclusive processes control activi- <br />                                                                      1. Abhayawardhane, Y. K., G. C. Stewart. 1995. Bacillus subtilis possesses <br /> ties influence cell morphology (21, 24). Hyperexpression         second determinant extensive sequence similarity Escherichia <br /> septal peptidoglycan biosynthetic apparatus leads spheri-          coli mreB morphogene. J. Bacteriol. 177:765&#226;&#8364;&#8220;773. <br /> cal shape, hyposeptation activity results filamenta-      2. Beall, B., J. Lutkenhaus. 1991. FtsZ Bacillus subtilis required                                                                         vegetative septation asymmetric septation sporulation. Genes <br /> tion. Wachi Matsuhashi concluded MreB                  Dev. 5:447&#226;&#8364;&#8220;455. <br /> protein play regulatory role septal synthesis regu-      3. Benson,  K., W. G. Haldenwang. 1993. Regulation &#226;?&#180;B levels                                                                         activity Bacillus subtilis. J. Bacteriol. 175:2347&#226;&#8364;&#8220;2356. <br /> lating expression FtsI (30). results extend findings   4. Bhavsar,  P., T. J. Beveridge, E. D. Brown. 2001. Precise deletion Wachi Matsuhashi suggesting MreC transmem-              tagD controlled depletion product, glycerol 3-phosphate cytidylyl- <br /> brane protein function control network               transferase, leads irregular morphology lysis Bacillus subtilis grown <br />                                                                         physiological temperature. J. Bacteriol. 183:6688&#226;&#8364;&#8220;6693. determines peptidoglycan biosynthesis process         5. Birnboim, H. C., J. Doly. 1979. rapid alkaline extraction procedure active. MreC production turned  cells begin          screening recombinant plasmid DNA. Nucleic Acids Res. 7:1513&#226;&#8364;&#8220;1523. <br /> synthesize cell wall material division site aberrant    6. Bork, P., C. Sander,  Valencia. 1992. ATPase domain common                                                                         prokaryotic cell cycle proteins, sugar kinases, actin, Hsp70 shock pro- <br /> manner. Overproduction wall polymer occurs                teins. Proc. Natl. Acad. Sci. USA 89:7290&#226;&#8364;&#8220;7294. cell, consequence bending       7. Burger, , K. Sichler, G. Kelemen, M. Buttner, W. Wohlleben. 2000. <br />                                                                         Identification characterization mre gene region Streptomyces <br /> cell, concave overproduction         coelicolor A3(2). Mol. Gen. Genet. 263:1053&#226;&#8364;&#8220;1060. material. Continued synthesis cell wall leads       8. Cha, J.-H., G. C. Stewart. 1997. divIVA minicell locus Bacillus <br /> progressive spread material septum         subtilis. J. Bacteriol. 179:1671&#226;&#8364;&#8220;1683. <br />                                                                      9. Cutting, S. M., P. B. Vander Horn. 1990. Genetic analysis, p. 27&#226;&#8364;&#8220;74. eventually covers cell.  process,      C. R. Harwood S. M. Cutting (ed.), Molecular biological methods overall cell wall appears weaken. leads influx        Bacillus. John Wiley &amp; Sons, Chichester, United Kingdom. <br />  4498       LEE STEWART                                                                                                                              J. BACTERIOL. <br />  <br /> 10. Dagert, M., S. D. Ehrlich. 1979. Prolonged incubation calcium chlo-         22. Oskouian, B., G. C. Stewart. 1990. Repression catabolite repression <br />     ride improves competence Escherichia coli cells. Gene 6:23&#226;&#8364;&#8220;28.                  lactose operon Staphylococcus aureus. J. Bacteriol. 172:3804&#226;&#8364;&#8220;3812. <br /> 11. Doi, M., M. Wachi, F. Ishino, S. Tomioka, M. Ito, Y. Sakagami,  Suzuki,          23. Pooley, H. M., F.-X. Abellan, D. Karamata. 1992. <span id='am-17' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="CDP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CDP</span>-glycerol:poly- <br />     M. Matsuhashi. 1988. Determinations DNA sequence                     (glycerophosphate) glycerophosphotransferase, involved syn- <br />     mreB gene gene products mre region function                  thesis major wall teichoic acid Bacillus subtilis 168, encoded     formation rod shape Escherichia coli cells. J. Bacteriol. 170:4619&#226;&#8364;&#8220;          tagF (rodC). J. Bacteriol. 174:646&#226;&#8364;&#8220;649. <br />     4624.                                                                              24. Satta, <br /> </body></html>