<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    RNA Polymerase II Elongation Factors Saccharomyces cerevisiae: <br />                  Targeted Proteomics Approach <br />  <br />              physically characterize web interactions connecting Saccharomyces cerevisiae proteins suspected <br />           RNA polymerase II (RNAPII) elongation factors, subunits Spt4/Spt5 Spt16/Pob3 (corresponding <br />           human DSIF FACT), Spt6, TFIIF (Tfg1, -2, -3), TFIIS, Rtf1, Elongator (Elp1, -2, -3, -4, -5,           -6) affinity purified conditions designed minimize loss associated polypeptides           identified mass spectrometry. Spt16/Pob3 discovered associate distinct complexes:  <br />           tones; Chd1/casein kinase II (CKII); Rtf1, Paf1, Ctr9, Cdc73, previously uncharacterized protein, <br />           Leo1. Rtf1 Chd1 previously implicated control elongation, sensitivity           6-azauracil strains lacking Paf1, Cdc73, Leo1 suggested proteins involved elongation           RNAPII  Confirmation came chromatin immunoprecipitation (ChIP) assays demonstrating           components complex, including Leo1, cross-linked promoter, coding region, 3&#226;&#180;&#8226; end           ADH1 gene. contrast, subunits TFIIF cross-linked promoter-containing fragment           ADH1. Spt6 interacted uncharacterized, essential protein Iws1 (interacts Spt6), Spt5 <br />           interacted Spt4 truncated form Spt6. ChIP Spt6 novel protein Iws1 resulted <br />           cross-linking proteins regions ADH1 gene, suggesting Iws1 likely           Spt6-interacting elongation factor. Spt5, Spt6, Iws1 phosphorylated consensus CKII sites vivo, <br />           conceivably Chd1/CKII associated Spt16/Pob3. elongation factors Elongator copurified <br />           RNAPII. <br />  <br />  <br />    synthesis functional mRNA precursors eukaryotic                 sylbenzimidazole [DRB] sensitivity-inducing factor [DSIF], <br /> RNA polymerase II (RNAPII) multistage process consist-                  required activation elongation human im- <br /> ing major steps: initiation, elongation, termina-               munodeficiency virus Tat) (46), Elongator (30), Spt6 (15), tion. learned steps involved                 Spt16/Pob3 (corresponding human FACT [facilitates chro- <br /> initiation, including formation initiation complex, initi-            matin transcription]) (28). Elongator copurified elon- <br /> ation RNA chain, promoter clearance,                 gating form RNAPII, shown bind preferentially <br /> known later stages transcription, especially chain                  hyperphosphorylated form RNAPII vitro (30). elongation. known,  transition tran-             Elp3 subunit Elongator histone acetyltransferase (HAT) <br /> scriptional initiation elongation associated change             activity vitro (48), suggesting HAT activity Elp3 factors associated RNAPII.                              assist RNAPII transcription elongation chro- <br />    general transcription factors required pro-               matin template. Human FACT stimulates transcriptional elon- <br /> moter-directed transcription initiation, multisubunit                gation chromatin template partially defined vitro <br /> Srb/Mediator complex associated RNAPII holoen-                   shown interact nucleosomes zyme binds promoters yeast Saccharomyces <span id='am-63' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-64' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span>cer</span>-                 histone H2A/H2B dimers (28, 29). Recently, FACT evisiae, RNAPII elongate transcript unaided.                 identified complex containing human casein kinase II factors discovered stimulate elongation               (CKII) phosphorylates tumor suppressor protein p53 nonchromatin, &#226;&#8364;&#339;naked&#226;&#8364;&#65533; DNA templates (e.g., TFIIS                      (18). <br /> TFIIF) (32, 47). Recently, factors identified                   Purification proteins interacting affecting thought affect RNAPII transcript elongation                  activity RNAPII generally involved conventional <br /> effects chromatin. include yeast factors Spt4                                                                              chromatography RNAPII affinity chromatography, proce- <br /> Spt5 (corresponding human 5,6-dichloro-1-&#226;&#65533;&#164;-D-ribofurano- <br />                                                                              dure identifies human TFIIF (RAP30/RAP74) TFIIS <br />                                                                              (38).  conventional affinity purifications usually <br />                                                                              require <span id='am-18' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> exchangers hydrophobic columns    * Corresponding author. Mailing address: Banting Best Depart- <br /> ment Medical Research, University Toronto, 112 College St., <br />                                                                              case) high salt concentrations  case) <br /> Rm. 210, Toronto, Ontario, Canada M5G 1L6. Phone: (416) 978-4141.            reduce nonspecific protein binding column matrix. <br /> Fax: (416) 978-8528. E-mail: jack.greenblatt@utoronto.ca.                    conditions potentially disrupt interactions  <br />                                                                       6979 <br />  6980      KROGAN ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />             TABLE 1. Yeast strains used study                                           MATERIALS METHODS <br />  <br />                                                           Reference        Yeast strain construction. strains used study (Table 1)  Strain                      Genotype                                   congenic W303-1A (42). Tags fused genes PCR-based <br />                                                           source <br />                                                                         step vivo tagging strategy (3) using TAP tag vector pBS1479 (33). <br /> W303-1A       MATa ura3-1 leu2-3,112 his3-11,15 trp1-1   39             Transformed cells (11) plated synthetic medium lacking tryptophan. <br />                 ade2-1 can1-100                                         Integration tag-encoding DNA confirmed PCR Western <br /> YCK245        W303-1A trp1 &#226;&#338;&#172;                      23             blotting nonspecific rabbit immunoglobulin recognize protein YNJK1         W303-1A, trp1 &#226;&#338;&#172; elp1-TAP::TRP1      23             component TAP tag. Strains following genes replaced YNJK2         W303-1A, trp1 &#226;&#338;&#172; elp3-TAP::TRP1      23 <br />                                                                         kanamycin resistance cassette obtained Research Genetics (http: <br /> YNJK3         W303-1A, trp1 &#226;&#338;&#172; spt4-TAP::TRP1        study <br />                                                                         //sequence-www.stanford.edu/group/yeast_deletion_project/deletions3.html): <br /> YNJK4         W303-1A, trp1 &#226;&#338;&#172; spt5-TAP::TRP1        study <br /> YNJK6         W303-1A, trp1 &#226;&#338;&#172; spt6-TAP::TRP1        study   CHD1, DST1, PAF1, CDC73, RTF1, LEO1. Sensitivity 6-azauracil YNJK7         W303-1A, trp1 &#226;&#338;&#172; spt16-TAP::TRP1       study   tested plating strains harboring pRS316 (37) plates lacking <span id='am-53' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> YNJK8         W303-1A, trp1 &#226;&#338;&#172; pob3-TAP::TRP1        study   containing 25 &#226;&#65533;&#174;g 6-azauracil/ml. <br /> YNJK9         W303-1A, trp1 &#226;&#338;&#172; dst1-TAP::TRP1        study      Protein purification. Tagged complexes implicated transcriptional elonga- <br /> YNJK11        W303-1A, trp1 &#226;&#338;&#172; iws1-TAP::TRP1        study   tion purified immunoglobulin G (IgG) calmodulin columns YNJK12        W303-1A, trp1 &#226;&#338;&#172; rpb3-TAP::TRP1        study   extracts yeast cells (3 liters) grown yeast extract-peptone-dextrose (YEPD) <br /> YNJK15        W303-1A, trp1 &#226;&#338;&#172; tfg1-TAP::TRP1        study   medium optical density 600 nm (OD600) 1.0 1.5. Cell pellets (7 YNJK16        W303-1A, trp1 &#226;&#338;&#172; tfg2-TAP::TRP1        study   10 g) frozen liquid nitrogen lysed grinding dry ice Krups <br /> YNJK17        W303-1A, trp1 &#226;&#338;&#172; tfg3-TAP::TRP1        study   coffee grinder (model 203-70). equal volume YEB (250 mM KCl, 100 mM <br /> YNJK23        W303-1A, trp1 &#226;&#338;&#172; chd1-TAP::TRP1        study   HEPES-KOH [pH 7.9], 1 mM EDTA, 2.5 mM dithiothreitol [DTT]) added, <br /> YNJK44        W303-1A, trp1 &#226;&#338;&#172; elp2-TAP::TRP1      23             following centrifugation Beckman 70 Ti rotor 4&#194;&#176;C 2 h 34,000 <br /> YNJK45        W303-1A, trp1 &#226;&#338;&#172; elp4-TAP::TRP1      23             rpm, supernatant dialyzed IPP buffer (10 mM Tris-Cl [pH 7.9], <br /> YNJK46        W303-1A, trp1 &#226;&#338;&#172; elp5-TAP::TRP1      23 <br />                                                                         0.1% Triton X-100, 0.5 mM DTT, 0.2 mM EDTA, 20% glycerol containing 100, <br /> YNJK74        W303-1A, trp1 &#226;&#338;&#172; rtf1-TAP::TRP1        study <br />                                                                         125, 150, 200 mM NaCl). dialysis, extract centrifuged YNJK78        W303-1A, trp1 &#226;&#338;&#172; leo1-TAP::TRP1        study <br /> YNJK81        W303-1A, trp1 &#226;&#338;&#172; cdc73-TAP::TRP1       study   70 Ti rotor 4&#194;&#176;C 30 min 34,000 rpm, supernatant mixed YNJK91        W303-1A, trp1 &#226;&#338;&#172; paf1-TAP::TRP1        study   3 h 200 &#226;&#65533;&#174;l IgG-Sepharose (Pharmacia) equilibrated IPP buffer. <br /> YNJK130       W303-1A, trp1 &#226;&#338;&#172; ctr9-TAP::TRP1        study   Following binding, IgG-Sepharose washed 1 ml IPP buffer <br />                                                                         followed 400 &#226;&#65533;&#174;l TEV protease cleavage buffer (50 mM Tris-Cl [pH 7.9], <br />                                                                         1 mM DTT, 0.1% Triton X-100, containing 200, 150, 125, 100 mM NaCl). <br />                                                                         beads incubated overnight 4&#194;&#176;C 100 U TEV protease <br />                                                                         (Life Technologies) 200 &#226;&#65533;&#174;l TEV cleavage buffer. eluate combined <br />                                                                         200-&#226;&#65533;&#174;l wash TEV cleavage buffer. added 200 &#226;&#65533;&#174;l stabilize protein complexes. circumvent prob-             calmodulin binding buffer (10 mM Tris-Cl [pH 7.9], 10 mM &#226;&#65533;&#164;-mercaptoethanol, <br /> lems, affinity tags fused selected target proteins     2 mM CaCl2, 0.1% Triton X-100 containing 200, 150, 125, 100 mM NaCl) widely used purify protein complexes. cases,            200 &#226;&#65533;&#174;l calmodulin beads (Pharmacia) equilibrated buffer. <br />                                                                         binding 1 2 h 4&#194;&#176;C, calmodulin beads washed 200 &#226;&#65533;&#174;l  necessary use high salt concen-           calmodulin binding buffer 200 &#226;&#65533;&#174;l calmodulin wash buffer (10 mM <br /> trations reduce nonspecific binding proteins solid sup-        Tris-Cl [pH 7.9], 10 mM &#226;&#65533;&#164;-mercaptoethanol, 0.1 mM CaCl2, 0.1% Triton <br /> ports.                                                                  X-100, containing 200, 150, 125, 100 mM NaCl). purified protein com- <br />    Recently, tandem affinity purification (TAP) scheme,            plexes eluted calmodulin beads 5 times 100 &#226;&#65533;&#174;l                                                                         calmodulin elution buffer (10 mM Tris-Cl [pH 7.9], 10 mM &#226;&#65533;&#164;-mercaptoethanol, adjacent tags (calmodulin-binding <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>                                                                         3 mM EGTA [pH 8.0], 0.1% Triton X-100, containing 200, 150, 125, 100 <br /> Staphylococcus aureus protein  separated tobacco etch            mM NaCl), eluted protein lyophilized concentrated precipi- <br /> virus (TEV) protease site fused protein         tation acetone trichloroacetic acid (TCA). Purified proteins sepa- described (33). use tags  consequently,          rated sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS- <br /> independent purification steps allowed protein purification          PAGE) gels containing 10% polyacrylamide  indicated <br />                                                                         , proteins visualized silver staining. Western blotting carried native conditions near-physiological            performed using standard techniques monoclonal antibody 8WG16 <br /> ionic strengths maintaining low background             (43). <br /> fined protein yield extract derived                 Protein identification. Protein bands reduced, alkylated, subjected untagged strain). circumstances, protein com-               gel tryptic digestion, peptides purified identified                                                                         matrix-assisted laser desorption ionization&#226;&#8364;&#8220;time flight (MALDI-TOF) mass <br /> plexes containing proteins weakly associated vivo <br />                                                                         spectrometry PerSeptive STR instrument (26). tandem mass greater chance remaining associated pu-            spectrometry, TCA-precipitated protein resuspended 100 mM <br /> rification procedure.                                                   NH4HCO3/1 mM CaCl2 buffer, pH 8.5, digested overnight 37&#194;&#176;C 2 &#226;&#65533;&#174;l <br />    RNAPII elongation factors S. cerevisiae           immobilized Poros trypsin beads (PerSeptive). entire digest fraction- <br /> poorly characterized.  conventional purifications           ated described previously (10) 7.5-cm (inner diameter, 100 &#226;&#65533;&#174;m) reverse- <br />                                                                         phase C18 capillary column attached line Finnigan LCQ-Deca ion trap <br /> human proteins previously implicated transcriptional elon-           mass spectrometer ramping linear gradient 2 60% solvent B 90 <br /> gation resulted loss associated          min. Solvent consisted 5% acetonitrile, 0.5% <span id='am-58' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span>, 0.02% hep- <br /> proteins. order systematically characterize yeast tran-       tafluorobutanoic acid (HFBA), solvent B consisted acetonitrile-water <br /> scriptional elongation factors, devised modification        (80:20) containing 0.5% acetic acid 0.02% HFBA. flow rate tip                                                                         needle set 300 nl/min programming high-performance liquid <br /> TAP procedure (33) dilution yeast extract <br />                                                                         chromatography pump using split line. mass spectrometer cycled purification procedure minimized high salt           scans gradient progressed. mass scan <br /> concentrations avoided. cases, purification using          followed successive tandem mass scans intense ions. procedure unveiled new protein-protein interactions            dynamic exclusion list used limit collection tandem mass spectra new polypeptides involved transcriptional elon-            peptides eluted long period time. tandem mass spectra                                                                         searched using SEQUEST algorithm complete yeast <br /> gation S. cerevisiae. Furthermore, data revealed        protein database (6/2,000). high-scoring peptide sequence manually <br /> salt concentrations used purification critical           compared corresponding tandem mass spectrum ensure certain cases.                                                          match correct. <br />  VOL. 22, 2002                                                                                     YEAST RNAPII ELONGATION FACTORS                       6981 <br />  <br />  <br />                                                                                         FIG. 1. Isolation protein complexes containing Spt4, Spt5,                                                                                      Spt6.   TAPs Spt4/Spt5 complex carried strains <br />                                                                                      containing tagged protein TAP-tagged versions Spt4                                                                                      Spt5. Protein complexes purified presence 100 mM NaCl <br />                                                                                      analyzed SDS-PAGE silver staining. Spt4, Spt5, <br />                                                                                      subunits RNAPII, truncated form Spt6 identified                                                                                      trypsin digestion MALDI-TOF mass spectrometry. purifi- <br />                                                                                      cation tagged Spt5, presence Spt4 identified subjecting <br />                                                                                      aliquot eluate final column directly trypsin <br />                                                                                      digestion tandem mass spectrometry. (B) TAP Spt6. Purifica- <br />                                                                                      tion using extract containing TAP-tagged version Spt6 100 <br />                                                                                      mM NaCl resulted isolation forms tagged protein                                                                                      previously uncharacterized protein encoded ORF YPR133c, <br />                                                                                      gene product called Iws1 (interacts Spt6). (C) TAP <br />                                                                                      Iws1. Tagging isolation Iws1 100 mM salt resulted                                                                                      copurification stoichiometric form Spt6.                                                                                      Spt6-TAP Iws1-TAP run gel,                                                                                      concluded Iws1 copurified slowest-migrating form                                                                                      Spt6. <br />  <br />  <br />  <br />  <br />    ChIP. Chromatin immunoprecipitation (ChIP) PCRs performed             contained tagged protein. protein bands absent <br /> described previously (22). yeast strains grown 30&#194;&#176;C OD595 0.4   control preparation corresponding tagged 0.6. Sequences ADH1 3&#226;&#172;&#732; untranslated region primer pair follows: <br /> ADH11231, ACCGGCATGCCGAGCAAATGCCTG; ADH11400, CCCAACT <br />                                                                                      protein associated proteins identified GAAGGCTAGGCTGTGG.                                                                    MALDI-TOF ion trap mass spectrometry. Proteins                                                                                      copurified tagged proteins                                                                                      tagged, purified, analyzed confirm presence <br />                                   RESULTS <br />                                                                                      complex create picture interaction relation- <br />    Purification identification yeast elongation fac-                      ships putative elongation factors S. cerevisiae. <br /> tors. used single-step transformation PCR products                              Spt5 phosphoprotein associates Spt4 Spt6. TRP1 selective marker place TAP tags containing                           Human DSIF NELF (negative elongation factor) calmodulin-binding peptide S. aureus protein  separated                        previously identified purified inhibitory elongation TEV protease cleavage site, C termini TFIIS,                      factors make transcription sensitive DRB (46, 49). subunits TFIIF, known subunits Elongator                         DSIF consists 160- 14-kDa subunits stim- <br /> (23), Spt4, Spt5, Spt6, Spt16, Pob3. confirm success                      ulate transcriptional elongation certain conditions. tagging procedure, Western blotting performed                         yeast homologues DSIF subunits, Spt4 Spt5, extracts derived tagged strains, making use                           form complex, mutations genes encoding irrelevant immunoglobulin recognize protein compo-                          proteins result sensitivity 6-azauracil (15), consid- <br /> nent TAP tag. tagged protein,                          ered important criterion mutations yeast genes <br /> associated proteins, purified IgG calmodulin                         encode elongation factors (2, 9). recently, Dro- <br /> columns analyzed SDS-PAGE followed staining                           sophila homologue Spt5 shown colocalize stalled, <br /> silver. Control purifications carried parallel                    hypophosphorylated RNAPII Drosophila heat shock genes using extracts derived congenic strain                          absence induction hyperphosphorylated, <br />  6982    KROGAN ET AL.                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> actively elongating RNAPII salivary gland develop-         transcriptional elongation.  yeast SPT16 <br /> ment, implying Spt5 elongation factor (1, 17).    gene interacts genetically genes encoding yeast <br />    Purification tagged Spt4 yeast extracts pres-   elongation factors TFIIS Spt4 (41). <br /> ence 100 mM NaCl resulted copurification forms        yeast counterparts FACT, Spt16 Pob3, Spt5 substoichiometric RNAPII (Fig.     tagged subsequently purified yeast extracts 1A). electrophoretically distinct forms Spt5    buffers containing 150 mM NaCl, large approximately <br /> consistently observed, probably caused C-ter-     stoichiometric amounts Spt16 Pob3 isolated minal degradation, tagging purification        proteins detected silver staining (Fig. 3A). <br /> Spt5 resulted isolation form tagged Spt5     NaCl concentration used (Fig. 1A). Copurifying tagged Spt5 Spt4, sub-     purification reduced 125 mM, proteins <br /> stoichiometric RNAPII,  surprisingly, trun-       copurified Spt16 Pob3 substoichiometric amounts <br /> cated form Spt6. small untagged Spt4 polypeptide ran       (Fig. 3B). NaCl concentration reduced gel shown Fig. 1A, presence Spt4     100 mM (Fig. 3C D), yields proteins associ- <br /> sample applied gel confirmed tandem mass spec-      ated Spt16 Pob3 increased substantially. case, <br /> trometry. results implied Spt5 associate    Spt16/Pob3-associated proteins detected Spt4 modified form Spt6. Analysis        parallel purifications carried extracts derived complexes containing tagged Spt4 Spt5 use tandem         untagged strain (Fig. 3A, B, C). <br /> mass spectrometry served identify phosphorylation site       Tandem MALDI-TOF mass spectrometry used Spt5 serine 188 sequence SDDE, start    identify following proteins copurified Spt16 consensus CKII phosphorylation site [(S/T)-(D/E)-X-(D/E)]       Pob3: Chd1, Ctr9, Paf1, Cdc73, Rtf1, CkaI, CkaII, CkbI, CkbII, <br />  Fig. 2B) (12).                                               histones, proteins encoded uncharacterized ORFs, <br />    Spt6 phosphoprotein interacts uncharac-     YOL054w YOR123c  known LEO1) (25).  <br /> terized phosphoprotein Iws1. SPT6 originally identified       ingly, Paf1 Cdc73, Ctr9, previously SPT4 SPT5 screen suppressors cis-      identified components complex binds RNAPII trans-acting mutations affected promoter function (15,   (21, 36). Chd1 member chromodomain-helicase- <br /> 41). recently, Drosophila Spt6 shown colocalize       DNA-binding family shown involved hyperphosphorylated, actively elongating RNAPII           <span id='am-24' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>-dependent nucleosome remodeling (44). Recently,  <br /> polytene chromosomes (1, 17).                                     component complex, Rtf1, implicated genet- <br />    Tagging purification yeast Spt6 resulted copurifi-   ically transcriptional elongation (8). <br /> cation substoichiometric amounts previously uncharac-        plethora proteins identified association terized protein, encoded essential open reading frame      Spt16 Pob3 raised possibility distinct pro- <br /> (ORF) YPR133c, called Iws1 (interacts          tein complexes associated Spt16 Pob3.  <br /> Spt6) (Fig. 1B). Conversely, tagging purification Iws1     solve complexes, tagged purified yeast ex- <br /> resulted isolation Spt6, confirming   tracts various proteins associated Spt16 interaction occurs vivo (Fig. 1C). electro-   Pob3. Tagging Chd1 resulted copurification approxi- <br /> phoretically distinct forms tagged Spt6, form     mately stoichiometric amounts subunits CKII lowest electrophoretic mobility copurified tagged   (CkaI, CkaII, CkbI, CkbII), substoichiometric <br /> Iws1. clear higher-mobility forms        amounts Spt16 Pob3 (Fig. 3E), tagged Spt6 consequence posttranslational modifica-      polypeptides copurified Spt16 Pob3. Tagging <br /> tion vitro vivo degradation.                          purification Ctr9 resulted coisolation Rtf1, <br />    Tandem mass spectrometry used identify tryptic    Leo1, Paf1, Cdc73, Spt16 Pob3, fragments N-terminal region Spt6 phos-        histones CKII Chd1 (Fig. 3F).  tagging <br /> phorylated serine residues (serines 95 [SEDD], 135 [SESD],     purification histone H3 did result copurification 207 [SDED]) (Fig. 2A). case Spt5,         Spt16 Pob3 (data shown). results indicated <br /> phosphorylated serine residue Spt6 beginning     separable complexes associate consensus CKII site. putative CKII sites exist     Spt16 Pob3: histones nucleosomes, Chd1/CKII, N-terminal region Spt6, tandem mass spectrometry       complex containing Ctr9, Paf1, Cdc73, Rtf1, Leo1. <br /> did detect phosphorylation sites. did detect    Confirmation came experiments labeled phosphorylation site serine 89 (SDDD) Iws1,        purified Leo1 Rtf1 (Fig. 3F), Paf1 (Fig. 3G), Cdc73 putative CKII site. Interestingly, acidic N-terminal   (Fig. 3H). case, observed untagged com- <br /> regions Spt5 Iws1 sequence similarity (Fig. 2B),      ponents subcomplex, amounts Spt16 surprisingly, phosphorylated serine residues align      Pob3 varied near-stoichiometric clearly substoi- <br /> perfectly, explaining Spt6 interact     chiometric. certain cases, distinct forms of                                                           Paf1, Cdc73, Ctr9, determined <br />    Spt16/Pob3 associates multiple protein complexes. Hu-     consequences degradation, processing, man FACT purified factor overcame nucleo-         posttranslational modification. Yol054w copuri- <br /> somal block elongation RNAPII vitro composed      fied tagged Spt16 Pob3, did observe copurifi- human homologues yeast Spt16/Cdc68 Pob3 (28, 29).       cation Yol054w tagged Paf1, Ctr9, Cdc73, Rtf1, FACT interacts nucleosomes histone H2A-H2B               Leo1, suggesting Yol054w directly associated dimers   promote nucleosome disruption        Spt16 Pob3. <br />  VOL. 22, 2002                                                                               YEAST RNAPII ELONGATION FACTORS                        6983 <br />  <br />  <br />  <br />  <br />    FIG. 2. Detection phosphorylation putative CKII sites Spt5, Spt6, Iws1.   Identification phosphorylation sites Spt6. Tandem <br /> mass spectrometry used identify tryptic fragments N-terminal region Spt6 phosphorylated serine residues (se- <br /> rines 95 [SEDD], 135 [SESD], 207 [SDED]). phosphorylated serine residue Spt6 beginning consensus CKII site. (B) <br /> Phosphorylation conserved serine present Spt5 Iws1. Mass spectrometry successfully detected phosphorylation site serine 89 <br /> (SDDD) Iws1 serine 188 (SDDE) Spt5; occurs start putative CKII site. acidic N-terminal regions Spt5 Iws1 sequence similarity, phosphorylated serine residues align perfectly. regions compared 23% identity 40% similarity. <br />  <br />  <br />    determine polypeptides copurifying Spt16/                   reductions <span id='am-48' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span>UTP</span> <span id='am-10' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span>GTP</span> concentrations yeast cells, leads <br /> Pob3 involved transcriptional elongation vivo,                 induction PUR5 gene (35), shown strains harboring knockouts genes encoding                     affect elongation efficiency RNAPII (9).  <br /> nonessential subunits Spt16/Pob3 complex tested                   sensitivity compound used indicator sensitivity 6-azauracil. drug 6-azauracil results               involvement transcriptional elongation. Studies using <br />  6984   KROGAN ET AL.   MOL. CELL. BIOL. <br />  VOL. 22, 2002                                                                               YEAST RNAPII ELONGATION FACTORS                        6985 <br />  <br />  <br />  <br />  <br />    FIG. 3. Spt16/Pob3 associates protein complexes.   TAPs Spt16 Pob3 TAP-tagged strains buffers containing 150 <br /> mM NaCl. Purified preparations analyzed SDS-PAGE silver staining. Spt16 Pob3 identified mass spectrometry. <br /> (B) Purification Spt16/Pob3 complexes buffers containing 125 mM NaCl. Analysis SDS-PAGE silver staining revealed number polypeptides copurified tagged Spt16 Pob3. (C D) Tagged Spt16 purified buffers containing 100 mM NaCl, resulting increased yield proteins associated Spt16/Pob3. purified preparation electrophoresed gels containing 10% (C) 15% <br /> (D) polyacrylamide. additional proteins identified mass spectrometry Chd1, Ctr9, Paf1, Cdc73, Rtf1, Leo1, Yol054w, subunits CKII, histones. (E) TAP tagged Chd1 presence 100 mM salt. purified protein analyzed SDS-PAGE, silver staining, mass spectrometry. Tagged Chd1 copurified seemingly stoichiometric amounts subunits CKII (CkaI, CkaII, CkbI, CkbII) substoichiometric amounts Spt16 Pob3. (F H) TAPs Leo1, Rtf1, Ctr9 (F), Paf1 (G), Cdc73 (H)  presence 100 mM NaCl) resulted isolation complexes containing Rtf1, Leo1, Ctr9, Paf1, Cdc73, Spt16 Pob3, histones.   Sensitivities deletion strains 6-azauracil. Strains containing rtf1&#226;&#338;&#172;, paf1&#226;&#338;&#172;, cdc73&#226;&#338;&#172;, leo1&#226;&#338;&#172;, chd1&#226;&#338;&#172;, dst1&#226;&#338;&#172; deletion, plasmid pRS316, plated SD-uracil medium drug 6-azauracil (50 &#226;&#65533;&#174;g/ml) grown 30&#194;&#176;C 2 4 days. WT, <br /> wild type. <br />  <br />  <br /> 6-azauracil helped genes like DST1,                      yeast, TFIIF comprised subunits Tfg1, <br /> encodes TFIIS, encode factors role transcrip-                  Tfg2, Tfg3. subunit tagged purified <br /> tional elongation (2). dst1&#226;&#338;&#172; used control,                   yeast extracts buffers containing 150 mM NaCl, letions rtf1, paf1, cdc73, leo1, chd1, resulted             complex containing subunits TFIIF isolated <br /> significant marked sensitivity 50 &#226;&#65533;&#174;g 6-azauracil/ml (Fig.            nearly stoichiometric RNAPII (Fig. <br /> 3I).                                                                          4A). <br />    Purification TFIIF TFIIS. TFIIF general tran-                     certain arrest sites encoded template DNA, RNA- <br /> scription factor interacts RNAPII recruit                   PII slides backwards, resulting loss contact 3&#226;&#172;&#732; <br /> RNAPII preinitiation complex containing TFIID                        end nascent transcript catalytic center TFIIB (14). TFIIF participates promoter clearance                 polymerase (34, 45, 47). transcriptional elongation factor, associate RNAPII vitro chain elongation,                   TFIIS, facilitates escape arrested RNAPII enhancing polymerization rate processivity (7, 32).             pause site promoting cleavage nascent transcript thought TFIIF acts early stages elongation                 catalytic center RNAPII, followed reextension prevents abortive transcription, increasing                RNA newly created 3&#226;&#172;&#732; end. TFIIS tagged <br /> rate <span id='am-35' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> addition.                                                  purified yeast extracts, predominantly free form  6986     KROGAN ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />  protein isolated (Fig. 4B), TFIIS known      immunoglobulin G-agarose. Finally, reversal cross-links interact directly RNAPII (38).                              carried  followed PCR analyses coprecipi- <br />    Isolation holo-Elongator complex.         tated DNA. gene investigated ADH1, encoding alco- <br /> subunit Elongator complex originally associated           hol dehydrogenase, sets primer pairs used elongating form RNAPII shown pref-           analyze gene: directed promoter region, <br /> erentially bind hyperphosphorylated form RNAPII        directed downstream coding region, vitro (30), hypothesized Elongator replaces         finally, used amplify region corresponding DNA mediator switch transcription initiation         3&#226;&#172;&#732; untranslated region. <br /> elongation. using extracts individual strains containing       subunits TFIIF,  Tfg1, Tfg2, Tfg3, <br /> tagged versions known Elongator subunits, Elp1,        cross-linked promoter region ADH1 Elp2, Elp3, subunit yeast holo-Elongator complex          coding region 3&#226;&#172;&#732; untranslated region (Fig. 5A).  <br /> containing additional polypeptides, Elp4, Elp5, Elp6,     sults suggested  TFIIF stable component subsequently isolated (23). purification carried       initiation complex play role early elongation, higher salt concentrations, Elongator separated     does remain stably associated elongating RNAPII subcomplexes, consisting Elp1, -2, -3       vivo. contrast, components Paf1 complex consisting Elp4, -5, -6 (23). Interestingly,    cross-link specifically regions ADH1 <br /> RNAPII detectable silver staining present Elon-       gene tested, including novel, uncharacterized <br /> gator preparations, microgram quantities Elon-         protein Leo1 (Fig. 5B);  fragments ADH1 <br /> gator loaded gel (23) (data shown).               gene enriched immunoprecipitation relative DNA <br />    Copurification yeast elongation factors RNAPII.      fragment nontranscribed region, matter sub- <br /> Human TFIIS, TFIIF, Spt5 known bind directly          unit Paf1/Ctr9/Cdc73/Rtf1/Leo1 complex tagged. <br /> RNAPII (32, 38, 46).  Ctr9, Paf1, Cdc73            Similarly, ChIP patterns Spt6 Iws1 identified components yeast protein complex binds      identical, displayed degree cross-linking RNAPII (21, 36), Elongator copurified chromatin-        regions ADH1 gene (Fig. 5C). Interestingly, <br /> bound elongating RNAPII (30). RNAPII copurified              ChIP assays demonstrated Elongator subunits did tagged TFIIF, Spt4, Spt5 detect          cross-link ADH1 gene number largest subunits, Rpb1 Rpb2, silver staining identify    genes tested  unpublished data). mass spectrometry (Fig. 1A 4A). used sensitive Western blot technique monoclonal                                  DISCUSSION <br /> tibody directed Rpb1 determine yeast elon- <br /> gation factors associated weakly RNAPII (Fig.           outset study, number genes known 4C). Purification RNAPII itself strain containing       suspected encode elongation factors RNAPII S. cere- <br /> TAP tag Rpb3, largest subunit RNAPII,               visiae. TFIIS shown stimulate elongation yeast <br /> showed soluble yeast extracts used purifica-       RNAPII vitro (34, 45, 47). TFIIF human Drosoph- <br /> tions contained RNAPII recognized monoclonal              ila cells shown stimulate rate elongation antibody 8WG16 (Fig. 4C). tagged elongation factor       respective RNAPII enzymes vitro (4, 32), subunits copurified RNAPII Chd1           true purified yeast TFIIF  Elongator. Chd1 copurify RNAPII simply             unpublished data).  TFIIF stably association RNAPII indirect: weakly   associated promoter region associated Spt16/Pob3, turn weakly associated      downstream transcribed region ADH1 gene vivo, Paf1/Ctr9/Cdc73/Rtf1/Leo1, turn weakly asso-       similar observations reported recently Pokholok et al. <br /> ciated RNAPII  Fig. 6). Remarkably, detectable          (31).  clear TFIIF behaves elon- <br /> RNAPII copurified Elongator Elp1 Elp5           gation factor vivo spite vitro activity. Spt4 tagged purified, 1 &#226;&#65533;&#174;g purified      Spt5 yeast homologues subunits DSIF, Elongator loaded gel (Fig. 4C), similar results    human factor positive negative effects chain obtained tagged Elp2 Elp3 (data shown).        elongation human RNAPII vitro (46), purified yeast experiments (data shown), used antibodies spe-         Spt4/Spt5 stimulates elongation yeast RNAPII vitro <br /> cific phosphorylated C-terminal domain (CTD)              unpublished data). genetic properties yeast <br /> Rpb1. purification tagged Rpb3 revealed        SPT6 gene similar SPT4 SPT5 (15, 41). <br /> extracts contained phosphorylated Rpb1, phosphory-              Drosophila homologues Spt5 Spt6 appear <br /> lated Rpb1, like unphosphorylated Rpb1 recog-            associated Drosophila elongation complexes poly- <br /> nized antibody (8WG16) used Fig. 4C, did copu-        tene chromosomes (1, 17). Spt16 Pob3 yeast homo- <br /> rify Elongator tagged Elp1 Elp5.                         logues subunits human FACT, factor    ChIP assays. Analysis vivo distribution novel   stimulates elongation human RNAPII chromatin tem- <br /> elongation factors described transcribed gene      plate partially reconstituted vitro (28, 29). Ge- <br /> carried ChIP assays proteins cross-        netic studies yeast RTF1 gene implicated Rtf1 linked DNA vivo use <span id='am-43' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> (22). Yeast cells      possible RNAPII elongation factor vivo (8). Finally, bio- lysed, following isolation shearing chro-      <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> copurification experiments indicated Elongator <br /> matin, presence proteins near specific DNA se-           associated elongating RNA polymerase II vivo (30). <br /> quences monitored immunoprecipitation rabbit               Collectively, previously published studies implicated <br />  VOL. 22, 2002                                                                           YEAST RNAPII ELONGATION FACTORS                     6987 <br />  <br />  <br />  <br />  <br />    FIG. 4. TAPs TFIIF TFIIS detection RNAPII interactions Western blot analysis.   Purification TFIIF using TAP- <br /> tagged versions Tfg1, Tfg2, Tfg3. Purified proteins analyzed SDS-PAGE, silver staining, mass spectrometry. Purification of subunits TFIIF buffers containing 150 mM NaCl results isolation components TFIIF substoichio- <br /> metric RNAPII. (B) Tagging purification TFIIS. Purification TFIIS (encoded DST1 gene) buffers containing 100 <br /> mM NaCl resulted isolation predominantly TFIIS. development silver stain revealed background proteins copurified TFIIS. (C) Western blot analysis transcription elongation factors purified presence 100 mM NaCl using <br /> monoclonal antibody 8WG16, binds CTD largest subunit RNAPII, Rpb1. positive control, RNAPII purified strain TAP tag largest subunit, Rpb3. Equal volumes column eluates various purifications loaded gel, approximately 10 times Rpb3 purification loaded. RNAPII, presence 100 mM NaCl, copurified TFIIF (Tfg1, -2, -3), Spt4/Spt5, Spt6/Iws1, TFIIS, Spt16, Paf1/Rtf1, Elongator (Elp1 Elp5) Chd1. <br />  <br />  <br /> 13 polypeptides having possible roles chain elongation            unknown function, Leo1 possibly Yol054w. relatively <br /> yeast RNAPII.  TFIIS, TFIIF, Elongator                    small change NaCl concentration used purifica- purified;  yeast elongation             tion, 150 125 100 mM, enabled identify factors purified conditions designed mini-               Spt16/Pob3-associated proteins just Spt16 <br /> mize subunit loss purification procedure, raising           Pob3. Tagging proteins associated possibility intersubunit associations new              Spt16/Pob3 revealed Spt16/Pob3 complex associates <br /> elongation factor subunits discovered. Based            distinct protein complexes: core histones, Chd1/ experiments described experi-                CKII, Rtf1/Paf1/Ctr9/Cdc73/Leo1  Fig. 6). ments (N. J. Krogan J. F. Greenblatt, unpublished data),               clear protein complexes associate simul- believe complexes containing proteins weakly              taneously Spt16/Pob3. Simultaneous association associated vivo greater chance remaining associ-              imply Spt16/Pob3 scaffold coassembly ated purification dilution minimized puri-              factors. <br /> fication carried using near-physiological ionic strengths.             interaction Spt16/Pob3 Chd1 core  web interactions RNAPII elongation factors                tones accord similar findings human FACT (19, uncovered procedures illustrated Fig. 6.            28, 29). Kelley et al. (19) showed human homologues <br />    importantly, procedures enabled discover               Pob3 Chd1 interact physically vitro vivo. Spt16/Pob3 associates 14 polypeptides,               chd1&#226;&#338;&#172; mutation suppressed growth defect conferred <br /> Rtf1 core histones, Chd1, subunits             particular pob3 point mutation (8) result yeast CKII, protein complex (Paf1/Ctr9/Cdc73)                   resistance 6-azauracil strain backgrounds  known associate RNAPII, proteins                 particularly used , implicating <br />  6988     KROGAN ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br /> Chd1 transcriptional elongation (44). FACT ob-               components Paf1 complex tested sensitivity served copurify human CKII complex phos-               6-azauracil. strains containing tagged proteins did phorylated p53 (18), clear human CHD1              confer significant sensitivity drug, implying tag complex. results obtained                detrimental function complex (data tagging purifying Chd1 imply yeast Chd1 directly             shown). similarly inferred C-terminal tags associated CKII  Fig. 6). possibility Chd1          Spt16, Pob3, Spt5, Spt6, Iws1, Tfg1, Tfg2 controls elongation controlling phosphorylation             detrimental, proteins encoded essen- <br /> elongation factor CKII. Definitive proof Chd1              tial genes, tagged haploid strains viable grow elongation factor awaits experiments showing di-             cases. uncharacterized polypeptide Yol054w <br /> rectly affect elongation RNAPII vivo vitro.                  detected tagged Spt16 purified buffers  <br />     recently shown Paf1 (RNAPII-associated factor),          taining 100 mM NaCl, observed copurify Cdc73, Ctr9 form complex associates RNAPII               tagged proteins. Efforts tag Yol054w  <br /> (21, 36).  roles proteins transcription          successful, conclude Yol054w bona RNAPII unclear. observation combining                 fide subunit of complexes. possible,  <br /> PAF1 deletion deletion mediator subunit SRB5              Yol054w directly associated Spt16 Pob3  Fig. <br /> lethal used argue Paf1 complex involved            6). <br /> initiation (36).  Paf1, Cdc73, Ctr9               subunits TFIIF (Tfg1, Tfg2, Tfg3) associated Spt16/Pob3 complex contains           tagged purified buffers containing 150 mM NaCl, <br /> Rtf1 Leo1, likely complex mediates        proteins copurified near-stoichiometric interaction Spt16/Pob3 RNAPII. Rtf1 (restores                RNAPII case. showed TFIIF interacts TBP function) originally shown affect function             RNAPII vivo, suggested experiments systems TATA-binding protein (TBP) (40).  recent work                (7, 38) demonstrated TFIIF interacts RNAPII suggested likely play role elongation          vitro.  Tfg3 reported associated <br /> RNAPII, strain deletion RTF1 sensitive           number complexes, including transcription <br /> 6-azauracil, phenotype associated defects transcrip-           initiation factor TFIID chromatin-modifying com- <br /> tional elongation (8).  deletion RTF1 shown           plexes SWI/SNF NuA3 (5, 13, 16). extract genetically interact point mutations essential            Tfg3-tagged strain used purify smallest subunit genes SPT16 POB3. view genetically defined role            TFIIF buffers containing 75 100 mM NaCl, poly- Rtf1 elongation association Paf1/Ctr9/Cdc73             peptides detected silver staining SDS-polyacryl- RNAPII, likely Rtf1/Paf1/Ctr9/Cdc73/            amide gel, correspond subunits Leo1 complex important role behavior Spt16/             factors (Krogan Greenblatt, unpublished). <br /> Pob3 elongation factor. Consistent                 Elongator complex reported copurify RNAPII paf1&#226;&#338;&#172;, leo1&#226;&#338;&#172;, cdc73&#226;&#338;&#172; strains sensitive 6-aza-            originally shown subunits, Elp1, Elp2, uracil, implicating products genes transcrip-            Elp3, known additional subunits, Elp4 <br /> tional elongation vivo. confirmation Paf1/               (YPL101w), Elp5 (YHR187w), Elp6 (YMR312w), Cdc73/Ctr9/Rtf1/Leo1 involved elongation came                 separated Elp1/Elp2/Elp3 buffers containing <br /> ChIP experiments showing proteins associated              200 mM NaCl (23, 30). Genetic DNA microarray analysis transcribed regions ADH1 genes                 strains containing deletions Elongator subunits  <br /> data reference 31) recently published genetic               vealed likely components func- coimmunoprecipitation experiments examining roles                tional complex. Interestingly, Western blot analysis showed proteins (27, 39).  contrast work,           detectable RNAPII copurified tagged Elp1, Elp2, study able detect stable physical association           Elp3, Elp5. Given observation purified Elongator, Paf1 complex Spt16/Pob3.                 unlike purified TFIIF, did stimulate elongation finitive proof Paf1, Cdc73, Ctr9, Rtf1, Leo1 elon-          RNAPII vitro  unpublished data), result implies <br /> gation factors awaits experiments showing directly             Elongator, unlike TFIIS, TFIIF, Spt4/Spt5, Spt6/ elongation activity vivo vitro.                            Iws1, Spt16/Pob3 complex, does truly associate <br />     Tagging purification Rtf1, Paf1, Ctr9, Cdc73            RNAPII Elongator associates RNAPII <br /> sulted copurification set polypeptides            RNAPII engaged transcription  <br /> included proteins Leo1, Spt16, Pob3,                haps genes. hands, deletion genes <br /> confirming truly associated Spt16/Pob3.          encoding subunits Elongator does confer resistance possible presence TAP tag protein         sensitivity 6-azauracil (23), suggesting Elongator impair function vivo, strains harboring individual tagged         generally elongation factor. Yeast Elongator sub- <br />  <br />  <br />    FIG. 5. ChIP assays TFIIF, Paf1/Leo1/Ctr9/Cdc73/Rtf1, Spt6/Iws1. ChIP performed strains containing TAP-tagged versions TFIIF subunits Tfg1, Tfg2, Tfg3  , Paf1, Cdc73, Rtf1, Leo1, Ctr9 (B), Spt6 Iws1 (C). monitor presence protein ADH1 gene, chromatin immunoprecipitated rabbit IgG-agarose PCR amplified primer pairs recognizing <br /> promoter (P), coding (C), 3&#226;&#172;&#732; untranslated (U) regions  schematic panel . PCR contained second primer pair amplified region chromosome V devoid ORFs (marked asterisks), providing internal control background. Input, signal chromatin immunoprecipitation. Primer pairs used follows: P, ADH1-235 ADH1-13; C, ADH1844 ADH11013; U, ADH11231 ADH11400; nontranscribed region, Intergenic V-1 Intergenic V-2. <br />  VOL. 22, 2002   YEAST RNAPII ELONGATION FACTORS   6989 <br />  6990     KROGAN ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                     FIG. 6. Comprehensive interaction map RNAPII elongation factors. details, text. <br />  <br />  <br />  <br /> units mainly localized cytosol (24, 31), ChIP          copurified Spt6, confirming bona fide  <br /> experiments did reveal association Elongator subunits          teraction. Western blotting demonstrated Spt6-Iws1, like transcribed region ADH1 gene  unpub-              Spt4/Spt5, copurifies RNAPII. Conversely, purification lished data).  human Elongator shown              tagged RNAPII resulted copurification Spt4, Spt5, Iws1, <br /> weakly stimulate transcription human RNAPII vitro (20),         Spt6, confirming associations (data shown). subunits shown               ChIP experiments showing Iws1 associated cytosol nucleus response interleukin-6 treatment         transcribed region ADH1 gene, ob- <br /> (6).  Elongator regulate transcription           servations Iws1 associated Spt6 RNAPII, sug- <br /> specific genes response signaling pathways.       gest Iws1, like Spt6, RNAPII elongation factor. Proof <br /> consistent microarray experiments indicating Elon-          Spt6 Iws1 elongation factors awaits experiments <br /> gator affects expression small set yeast genes       showing proteins influence elongation RNAPII (23).                                                                 vivo vitro. <br />    Tagging purification Spt4 resulted copurifica-          Interestingly, sequence acidic N-terminal region <br /> tion Spt5. purification carried buffers            Iws1 displays similarity N-terminal region Spt5. <br /> containing 100 mM NaCl, Spt4/Spt5 copurified RNAPII               tandem mass spectrometry revealed  detectable silver staining. Similarly, tagged Spt5         served serine residue consensus CKII phosphorylation site <br /> copurified Spt4 RNAPII. findings accord         region phosphorylated Spt5 Iws1. previous observations mutated form RPB2 ge-            phosphorylated serines detected mass spectrometry netically interacts SPT5 Spt5 coimmunoprecipi-          N-terminal region Spt6 located consensus CKII <br /> tates Rpb1 (15), largest subunit RNAPII. Similarly,       sites (12).  phosphorylation CKII impor- <br /> DSIF, human counterpart Spt4/Spt5, physically interacts        tant association Spt5 Iws1 Spt6 human RNAPII (46).                                               regulatory role activities Spt5, Spt6, Iws1.    Tagged Spt5 copurified truncated form Spt6.       interesting possibility CKII associated Chd1 physical interaction Spt5 Spt6 detected        Spt16/Pob3 elongation factor complex involved previously published study (41) follow study          phosphorylation Spt5, Spt6, Iws1  Fig. 6). <br /> (15). tagging purification Spt6 did result         targeted proteomics approach led description <br /> copurification Spt5, probably form Spt6        web interactions, comprising 25 polypeptides (31 copurified tagged Spt5 missing C-terminal region      Elongator included), putative transcriptional used C-terminal tags exclusively.  did             elongation factors S. cerevisiae. Construction inter- <br /> discover previously uncharacterized 40-kDa polypeptide,             action web possible use affinity tagging Iws1, associated Spt6. Iws1 tagged purified,        purification polypeptides, followed electro- <br />  <br /> phoresis, silver staining, mass spectrometry. Determining                    <br />                                                                                       interactions direct indirect              possible simply relied sensitive                 <br /> Western blot technique. cases,  example, TFIIS,                           <br />                                                                     <br /> TFIIF, Spt4/Spt5), results confirmed known sus-                        <br /> pected interactions. cases, tagging purification    known elongation factor resulted discovery novel                            <br />                                                                                   <br /> information regarding factor. particular, Iws1, Paf1,                              <br /> Cdc73, Ctr9, Leo1 previously suspected                            <br />                                                                                      <br /> RNAPII elongation factors, Paf1, Cdc73, Ctr9, Leo1,                          <br /> Rtf1 known associate Spt16/Pob3.                               <br /> triguing probably elongation factors                     <br />                                                                                           <br /> RNAPII. known RNAPII elongation                             <br /> factors associate simultaneously elongating tran-                           <br /> scription complex, known given factor ac-                  <br />                                                                                            <br /> companies RNAPII way initiation site                              <br /> termination site. nonessentiality yeast of                   <br /> elongation factor polypeptides imply                                   <br />                                                                                         functionally redundant.  possible                       specifically regulate genes expression                                 <br />                                                                                              essential laboratory conditions. yeast genetic                           ChIP assays make possible resolve                         issues.                                                                       <br /> </body></html>