<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>&#239;&#8250;&#8482; 1996 Oxford University Press                                            Nucleic Acids Research, 1996, Vol. 24,  20 4009&#226;&#8364;&#8220;4014 <br />  <br />  <br /> Allelism PSO4 PRP19 links pre-mRNA <br /> processing recombination error-prone DNA <br /> repair Saccharomyces cerevisiae <br /> Martin Grey, Andreas D&#195;&#188;sterh&#195;&#182;ft1, Joao P. Henriques2 Martin Brendel* <br />  <br /> Institut f&#195;&#188;r Mikrobiologie der J.W.Goethe Universit&#195;&#164;t, Theodor-Stern-Kai 7, Haus 75, 60590 Frankfurt/Main, <br /> Germany, 1Qiagen GmbH, Max-Volmer-Stra&#195;&#376;e 4, 40724 Hilden, Germany 2Department Biophysics/Biotechnology Center&#226;&#8364;&#8220;UFRGS, 91507-970 Porto Alegre, RS, Brazil <br />  <br /> Received June 19, 1996; Revised Accepted August 27, 1996                                DDBJ/EMBL/GenBank accession  X99770 <br />  <br />  <br />  <br /> ABSTRACT                                                              yeast mutants sensitive photoactivated psoralens (pso mutants; <br />                                                                       9) allocated epistasis group, radiation-sensitive mutant pso4-1 Saccharo-                    depending mutagen applied biological end point <br /> myces cerevisiae shows pleiotropic phenotype,                       scored respective double mutants (8). mutant allele including sensitivity DNA cross-linking agents,                    gene PSO4, involved error-prone repair nearly blocked sporulation reduced mutability.                 types recombination, e. gene conversion, crossing cloned putative yeast DNA repair gene PSO4                   intrachromosomal recombination (10,11,12), originally genomic library complementation                      isolated X-ray-sensitive mutant xs9 (13); blocked UV-induced mutagenesis sporulation                     particularly sensitive photoactivated bifunctional 8-methoxy- diploids homozygous pso4-1. Sequence analysis                  psoralen renamed pso4-1 (10). yeast mutant revealed gene PSO4 consists 1512 bp located                   phenotypically similar recA mutant Escherichia coli upstream UBI4 chromosome XII encodes                      combines mutagen radiation sensitivity block putative protein 56.7 kDa. PSO4 allelic PRP19,               recombination induced mutagenesis. pleiotropic repair gene encoding spliceosome-associated protein,                     phenotype caused singly existing pso4-1 mutant led shares significant homology yeast                   allocation PSO4 epistasis group (8,14). <br /> genes. Gene disruption destroyed reading frame                    recent studies showed heterologous expression recA PSO4 clone resulted death haploid cells,                 increased resistance UV ionizing radiation S.cerevisiae <br /> confirming finding PSO4/PRP19 essential                wild-type recombination-deficient rad52-1 mutant <br /> gene.  PSO4 essential DNA repair                    cells (15,16).  recA-like yeast mutant pso4-1 showed <br /> gene yeast S.cerevisiae.                                 restored induced mutability transformation multicopy <br />                                                                       vector containing E.coli recA gene (17). Biochemical analysis <br /> INTRODUCTION                                                          proved normal incision 8-MOP + UVA-induced interstrand 50 genes encoding proteins involved DNA repair        cross-links (ICLs), failure DNA strand rejoining pso4-1 <br /> known yeast Saccharomyces cerevisiae (1,2)           (16), depend recombinational step. molecular studies revealed close               apparent role gene PSO4 DNA repair processes attributed <br /> relationship DNA repair enzymes humans                genes belonging epistasis group II (recombination) III <br /> mammals, simple eukaryote advanced model             (error-prone) makes interesting candidate future complex mechanisms underlying DNA repair           understanding interconnections repair processes <br /> unraveled near future. DNA damage-sensitive           yeast. Progress elucidating function Pso4 protein <br /> yeast mutants initially grouped phenotypic groups     necessitated molecular cloning PSO4, wish (3,4)  double mutant analysis measured synergistic      report communication. epistatic interactions, allocated epistasis groups <br /> (5&#226;&#8364;&#8220;7). classification holds today,       MATERIALS METHODS <br /> group comprising &#226;&#710;&#188;20 genes RAD3-like encode            Yeast strains plasmids <br /> proteins <span id='am-20' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> excision repair, group II comprising &#226;&#710;&#188;12 <br /> genes RAD52-like encode proteins           yeast strains used listed Table 1. Single copy vector <br /> involved repair DNA double-strand breaks recombina-         pRS316 (18) multicopy vector pRS426 (19), containing <br /> tional mechanisms largest, far                URA3 selectable marker, used subcloning, comple- <br /> heterogeneous, group III, defined RAD6, comprising &#226;&#710;&#188;20             mentation tests sequencing. original complementing <br /> genes, responsible error-prone DNA repair       isolate pMG470 based multicopy plasmid YEp24 (20). <br /> (1,2,8). DNA damage-sensitive mutants, e.g.        subcloning performed using multicopy vector pRS426 <br />  <br />  <br /> *   correspondence addressed <br />  4010 Nucleic Acids Research, 1996, Vol. 24,  20 <br />  <br />  <br />  <br />  <br /> Figure 1.   Subcloning pso4-complementing gene passenger DNA pMG470 multicopy plasmids. restriction maps passenger DNAs pMG470-derived plasmids shown. pMG470: original clone isolated gene bank (20) harboring 16 kb passenger. 6.5 kb area overlaps second positive clone, pMG490  shown), depicted  pMG471, pMG473 pMG475: non-complementing subfragments pMG470. <br /> pMG480: fragment containing just gene, PRP19, able complement phenotypes pso4-1 mutant. Abbreviations cleavage sites: B/S, ligation <br /> site BamHI Sau3A; E, EcoRI; L, SalI; X, XbaI; Y, BstYI. +/&#226;&#8364;&#8220;: Plasmid complements/does complement pso4-1/pso4-1 pso4-1/prp19-0 mutant. <br /> (B) Non-complementation pso4-1 prp19-0 alleles. Diploids containing different allele combinations constructed tested survival 30 36 C streaking cell suspensions twoYEPD plates subsequently incubated 36 30 C respectively. (C) Survival mutagenesis irradiation UV254 light heteroallelic pso4-1/prp19-0 diploid MG5101 transformed plasmid pRS426 (triangles) pMG480 containing justPRP19 (squares) heterozygous pso4-1/PRP19 diploid MG5103 transformed plasmid (circles) positive control. <br />  <br />  <br />  allow screening possible multicopy suppressor                        CAN1/can1 marker possible select forward mutants <br /> PSO4 (Fig. 1A).                                                                          homozygous can1 replica-plating synthetic medium <br />                                                                                          lacking arginine <span id='am-35' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> supplemented 20 mg/l <br /> Cloning PSO4 <br />                                                                                          canavanine (SynCo&#226;&#8364;&#8220;Arg&#226;&#8364;&#8220;Ura . irradiation Gene PSO4 cloned transforming diploid strain MG5128                               UV254 light (20 J/m2) plates incubated 30 C homozygous pso4-1 yeast genomic library (20)                                  dark 3&#226;&#8364;&#8220;5 days. Clones papillae SynCo <br /> according lithium acetate-based protocol described                             &#226;&#8364;&#8220;Arg&#226;&#8364;&#8220;Ura isolated corresponding master <br /> Gietz et al. (21). previously introduced heterzygous                          plate tested ability sporulate. <br />  <br /> Table 1. Saccharomyces cerevisiae strains used study <br />  <br />  Strain         Genotype                                                                                                                           Source <br />  xs9            MATa lys2 pso4-1                                                                                                                   C.Schenberg <br />  W303           MATa/MAT&#206;&#177; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/PSO          R.Rothstein <br />  MG5100         MATa/MAT&#206;&#177; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/pso4::HIS3   study <br />  MG5101         MATa/MAT&#206;&#177; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15 can1-100 pso4::HIS3/pso4-1                           study <br />  MG5103         MATa/MAT&#206;&#177; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15 can1-100 PSO4/pso4-1                                 study <br />  MG5128         MATa/MAT&#206;&#177; ura3-52/ura3 ade2/ADE can1 pso4-1/pso4-1                                                                             study <br />                                                                                                               4011 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24  <br />                                                                                                               Vol.22,  120         4011 <br />  <br /> Mutagenicity assay survival                                         putative Pso4 protein <br /> Exponentially growing cells cultivated liquid synthetic medium       determined size PSO4/PRP19 gene 1512 bp, <br /> supplemented appropriate amino acids bases            corresponds 56.7 kDa protein consisting 503 amino <br /> washed resuspended phospate buffer (67 mM, pH 7)            acids, instead 502, originally described PRP19 Cheng <br /> final density 107 cells/ml irradiated different UV254       et al. (22), owing additional proline position 239. <br /> doses. determination survival cells plated       Sequence analysis putative protein revealed myb-like <br /> complete medium (SynCo) determination forward            DNA binding domain (W[ST]X2E X2[LIV]) C-terminus <br /> mutations (can1) SynCo arginine plus canavanine          Pso4/Prp19 protein (positions 457&#226;&#8364;&#8220;465, WTKDEESAL). <br /> (20 mg/l) respectively.                                                 retroviral oncogene v-myb cellular counterpart c-myb <br />                                                                         encode nuclear DNA binding proteins. S.cerevisiae, DNA sequencing                                                          myb-related genes include DNA-binding proteins REB1 (23) <br /> DNA sequencing performed framework            BAS1 (24).  Pso4/Prp19 protein does share <br /> European Union BIOTECH program sequence entire                   considerable homology putative proteins encoded chromosome XII. set random subclones plasmid cloning            yeast genes. nucleotide sequence PSO4 <br /> vector pUC18 generated insert region cosmid             appear EMBL, GenBank DDBJ Nucleotide <br /> 1F17. subclones served starting points primer        Sequence Databases accession  X99770. <br /> walking strategy. Double-stranded template DNA cosmid DNA subclones purified using QIAwell                 Construction phenotype pso4 null allele <br /> Purification (Qiagen). Cycle sequencing reactions carried using Taq DNA polymerase fluorescent terminators.       vitro disruption Pso4/Prp19-encoding ORF Gel electrophoresis automated base calling          SalI site inserting HIS3 (cf. Fig. 1), performed step <br /> ABI 373A Stretch DNA Sequencer (Applied Biosystems).                    disruption experiments haploid diploid wild-type cells. <br /> Contig assembly DNA sequence editing using            haploid transformants correct insertion DNASTAR software package (Lasergene).                                    diploid disruptants (MG5100, Table 2) showed                                                                         surviving auxotrophic ascospores sporulation tetrad <br /> RESULTS                                                                 analysis. absence prototrophic spores confirmed <br />                                                                         previously reported essentiality PSO4/PRP19 gene <br /> Cloning DNA repair gene PSO4                                         (22).  demonstrated                                                                         essentiality restricted spore germination.  Diploids homozygous mutant allele pso4-1 higher                                                                         heterozygous diploid MG5100 transformed pMG480 <br /> wild-type UV sensitivity, lower wild-type UV-induced <br />                                                                         (URA3; PSO4), resulting viable spores sporulation. <br /> mutation mitotic recombination sporulate poorly <br />                                                                         Plasmid loss experiments spores containing HIS3 <br /> (&lt;1% asci) appropriate medium (8). PSO4 molecularly <br />                                                                         disruption marker did yield viable Ura auxotrophs, cloned screening transformants diploid MG5128 <br />                                                                         haploid auxotrophic non-disruptants showed plasmid loss homozygous pso4-1 heterozygous CAN1 (Table 1),                                                                         &#226;&#710;&#188;20% 36 h. indicates essentiality PSO4 inducibility mutagenesis UV254 restored <br />                                                                         restricted spore germination, loss plasmid <br /> (CAN1&#226;&#8224;&#8217;can1r). 6000 transformants                                                                         results lethality haploid disruptants. essentiality wild-type-like UV-induced mutagenesis furthermore <br />                                                                         verified construct using URA3 reporter gene  restored sporulation (40% transformants respective <br />                                                                         shown), disrupted alleles generally named pso4-0 wild-type versus maximal number &lt;1% asci mutant) <br />                                                                         purpose simplification discussion. restored resistance DNA cross-linking mutagens. Restriction <br /> analysis complementing plasmids, named pMG470                PSO4 allellic PRP19 pMG490, showed passengers contained identical 6.5 kb overlapping fragment (depicted Fig. 1A).            fact gene PRP19 sufficient Hybridization 32P-labeled 2 kb EcoRI fragment            complement pso4-1 mutation, final evidence allelism <br /> area (Fig. 1A, pMG475) revealed passengers        cloned ORF PSO4 obtained showing null <br /> chromosome XII  shown). subfragments obtained                allele prp19::HIS3 complement phenotype pso4-1 <br /> restriction plasmid pMG470 singular SalI site yielded     mutant heteroallelic diploid constructed crossing plasmids pMG471 pMG473 (Fig. 1A), unable                original pso4-1 mutant xs9 haploid prp19::HIS3 <br /> restore PSO phenotype (Table 2A), indicating SalI           disruptant MG5100-1C opposite mating-type, located pso4-1 complementing open reading frame              containing PSO4-harboring plasmid pMG480 ensure viability (ORF) promoter. determined DNA sequence           haploid disruptant. plasmid loss, resulting Ura <br /> area match gene PRP19 (22), containing SalI        auxotrophic diploid MG5101, heteroallelic pso4-1/prp19::HIS3, <br /> site. sequencing revealed PSO4/PRP19 located            viable showed pleiotropic phenotype typical &#226;&#710;&#188;1.5 kb upstream UBI4 shares promoter region        homozygous pso4-1 diploid, including strong reduction &#226;&#710;&#188;320 bp ORF (L0916, Fig. 1A) upstream PRP19             sporulation induced mutagenesis sensitivity <br /> encoding hitherto uncharacterized protein containing putative       DNA cross-linking mutagens (Table 2 Fig. 1C). <br /> <span id='am-9' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span>/<span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> binding site. Subcloning 2.4 kb fragment                   prp19-0 allele unable complement pso4-1 mutant <br /> containing gene PRP19 (pMG480, Fig. 1A) revealed         phenotype, final proof genes allelic. gene sufficient restore wild-type-like phenotype   terms pso4::HIS3 (pso4-0) prp19::HIS3 (prp19-0) transformed pso4 mutant (Fig. 1C Table 2A).                  considered synonymous. <br />  4012 Nucleic Acids Research, 1996, Vol. 24,  20 <br />  <br /> Table 2.   Complementation analysis different subclones pMG470  Fig. 1A); (B) complementation analysis set plasmids transformed heteroallelic diploid MG5101; (C) non-complementation prp19 pso4 mutant alleles (MG5101) evidence allelism  Fig. 1B C) <br />  <br />  Strain            Relevant genotype      Plasmid              Sporulation          HN2                   30 C                 36 C              Mutability <br />    <br />  <br />  MG5128            pso4-1/pso4-1          pRS426               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG470               45%                  R                     +                    +                 + <br />                                           pMG471               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG473               &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (B) <br />  <br />  MG5101            prp19-0/pso4-1         pRS426               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG470               40%                  R                     +                    +                 + <br />                                           pMG471               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG473               0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (C) <br />  <br />  W303              PSO/PSO                                35%                  R                     +                    +                 + <br />  MG5100            prp19-0/PSO                            33%                  R                     +                    +                 + <br />  MG5128            pso4-1/pso4-1                          &lt;1%                  S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />  MG5103            pso4-1/PSO                             32%                  R                     +                    +                 + <br />  MG5101            prp19-0/pso4-1                         0.1%                 S                     +                    &#226;&#8364;&#8220;                 &#226;&#8364;&#8220; <br />  <br /> R, wild-type-like resistance; S, pso4 mutant-like sensitivity HN2. <br /> +/&#226;&#8364;&#8220; corresponds viability/non-viability 30 36 C mutability/reduced mutability respectively. plasmids pMG470 pMG480 complement pso4-1 mutation. <br /> Sporulation efficiency, sensitivity DNA cross-linking mutagen nitrogen mustard (HN2), mutability determined 30 C. <br />  <br /> pso4-1 non-lethal allele essential gene                                  mutagenesis, lowered spontaneous induced recombination <br /> PSO4/PRP19                                                                           nearly totally blocked pre-meiotic DNA synthesis                                                                                      sporulation (8,12,27). isolated gene PSO4, disruption PSO4/PRP19 lethal, mutants containing                    member RAD52 epistasis group complementation pso4-1 allele viable mentioned                            pso4-1 mutant&#226;&#8364;&#8482;s drastically reduced induced mutability phenotype.  viable mutants essential genes isolated                     blocked sporulation. Characterization DNA repair gene PSO4 sensitve temperatures &gt;30 C. tested                           showed allelism yeast gene PRP19, encodes 502 <br /> diploids harboring different alleles PSO4 possible temperature                <span id='am-28' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-29' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> spliceosome-associated protein (22,28). contrast, <br /> sensitivity. fact, diploids homoallelic pso4-1                 PSO4 sequence contains additional proline-coding CCC <br /> heteroallelic (pso4-1/prp19-0) sensitive 36 C.                     triplet encodes 503 amino acid protein.  prp19-0 allele failed complement sensitivity,                      discrepancy strain-specific difference confirming allelism genes (Fig. 1B).                               presently likely irrelevant discussion putative <br /> comparing growth yield diploid strains 30 C                         protein. heteroallelic pso4-1/pso4-0 diploid MG5101 grow                          allelism PSO4 PRP19 implicates yeast gene 30 C (cf.  severely retarded growth rate                     encodes protein functions RNA splicing generation time &gt;3 h YEPD medium, isogenic                            error-prone DNA repair, recombination sporulation PSO4/pso4-0 diploid MG5103 showed generation time &#226;&#710;&#188;1.5 h,                       organism. DNA repair gene function <br /> comparable wild-types  shown).                                        linked processing RNA essentiality underscores <br />                                                                                      vital function process. RAD3 DISCUSSION                                                                           SSL2/RAD25, PSO4/PRP19 DNA repair gene                                                                                      essential growth viability S.cerevisiae. <br /> Repair DNA double-strand breaks S.cerevisiae involves                             Prp19 protein demonstrated essential genes RAD52 epistasis group. corresponding proteins                       splicing pre-mRNA (28).  biochemical characterization supposed act recombinational mechanism                         revealed tightly associated snRNAs, shown constitute protein complex                        associated spliceosome splicing reaction (22). termed recombinosome (25,26). pso4-1 mutant                         Interestingly, Cheng et al. discussed fact Prp19 distinct <br /> previously shown exhibit pleiotropic phenotype, e.                       Prp proteins spliceosomal components <br /> sensitivity DNA damaging agents, nearly blocked induced                      regarding protein sequence does contain                                                                                                                 4013 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24  <br />                                                                                                               Vol.22,  120              4013 <br />  motifs Prp sequences.      (35) require splicing proper function. demonstrated Prp19 protein associated protein       pso4-1 mutants able incise DNA (14), known <br /> complex different spliceosome, consisting         splicing-dependent repair gene functional, pointing <br /> seven proteins addition Prp19, itself likely present     normal splicing activity pso4-1 mutants 30 C.  oligomeric form (29).  Prp19-associated complex,        non-functional splicing resulted repair deficiency pheno- <br /> itself big spliceosome, unlikely bind     types, &#226;&#710;&#188;50 DNA repair genes cloned ought complex (29). authors suggest Prp protein      encode proteins needed splicing, e. turned released associated splicing complex.       allelic PRP genes. <br /> event concomitant just dissociation         severe handicap growth complete medium 30 C of U4 snRNA ATP-dependent conformational change             heteroallelic pso4-1/pso4-0 heterozygous pso4-1/PSO4 formation functional spliceosome, suggesting          diploids indicates quantity protein produced Prp19 function step spliceosome assembly (30).          pso4-1 mutant allele sufficient normal spliceosome <br />     hand, function               activity, protein encoded PSO4 wild-type <br /> PRP19/PSO4-encoded protein recombinational error-prone          allele normal growing PSO4/pso4-0 diploid apparently <br /> repair? evidence gathered genetic biochemical             satisfies cellular demand spliceosome-associated protein <br /> experiments using allele pso4-1 point late function repair     function. context partial complementation yeast DNA lesions missing mutant strain. excision           pso4-1 mutant&#226;&#8364;&#8482;s sensitivity photoactivated 8-MOP + UVA 8-MOP + UVA-induced ICLs DNA thought proceed                overexpression RecA protein (17) new significance. RecA <br /> normally, production DNA double-strand breaks repair        protein bind single double-stranded DNA, intermediates   rejoined pso4-1 (14).      facilitating initiation recombination (36). Interstrand cross-link suggested repair DNA lesions ICL type          repair E.coli yeast shown proceed require modes repair (31,32),             double-strand breaks (cf. 2,31); secondary lesions removal damage repairosome postulated                substrate RecA RecA-like proteins  helical <br /> nucleotide excision repair (33) second enzyme          filament formation, initiate recombinational repair. RecA <br /> complex repair strand breaks, PSO4/PRP19           Rad51 proteins share considerable sequence homology encoded protein function. existence S.cerevisiae   strong evidence Rad51 binds Rad52 protein DNA double-strand repair-specific recombinosome,           (26). sensitivity pso4-1 mutant 8-MOP + UVA, complex containing proteins encoded RAD51, RAD52,          partial complementation overexpressed RecA protein RAD55 RAD57, recently suggested (26). suggest          epistatic interactions pso4-1 mutant rad51 rad52 Prp19/Pso4-associated protein complex Tarn           mutant alleles (17) lead hypothesis functions workers (29) mentioned recombino-            repair primary secondary induced DNA strand breaks                                                                   recombinational mechanism. association functional <br />    fact existing viable alleles essential gene    protein complex, forming recombinosome (26), <br /> PSO4/PRP19, represented previously described prp19           function Pso4/Prp19 protein;  mutant (22,34) pso4-1 mutant respectively, exhibit        essential yeast cell, role spliceosome- <br /> temperature sensitivity implies 503 amino acid protein        associated protein pre-mRNA processing. Recently, similar <br /> encoded PSO4/PRP19 functional                way association described yeast transcription <br /> domains, active protein     factor TFIIH: participates formation holocomplex active <br /> expressed haploid diploid pso4-1 mutants:        RNA polymerase II transcription initiation obvious growth retardation 30 C pso4-1 haploid            repairosome, specific nucleotide excision <br /> homoallelic diploid mutants assume RNA splicing           repair (37,38). <br /> proceeds efficiency close identical wild-type. hypothesis underscored fact         ACKNOWLEDGEMENTS <br /> viable prp19 mutant accumulates unprocessed pre-mRNA non-permissive permissive temperature (34),             thank Dr C.Schenberg pso4-1 mutant Drs <br /> suggesting viability pso4/prp19 mutant dependent       J.Brozmanova W.Siede helpful discussion. Sequencing functional splicing. Survival pso4-1 mutant pso4-0     left arm chromosome XII supported grant AD suggests mutant allele expresses     European Union BIOTECH program coor- <br /> partially functional protein 30 C; role spliceosomal   dination Joerg Hoheisel, DKFZ, Heidelberg. Research assembly function normal function            supported Qualit&#195;&#164;tsweine D.Brendel. Brazilian&#226;&#8364;&#8220;German <br /> repair DNA single- double-strand breaks recombinational     cooperation facilitated CNPq-DAAD travel grants. <br /> processes significantly perturbed. suggest phenotypes disturbed binding proteins Pso4 multienzyme complex Tarn et al. (29),          REFERENCES <br /> indicating hitherto unidentified proteins    1 Haynes,R.H. Kunz,B. (1981)  Strathern,J.N., Jones,E.W. belong RAD52 epistasis group.                                  Broach,J.R (eds), Molecular Biology Yeast Saccharomyces <br />    alternative, opinion highly unlikely, explanation        cerevisiae. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, repair phenotype pso4-1 mutant                pp. 341&#226;&#8364;&#8220;414. <br />                                                                         2 Friedberg,E.C., Walker,G.C. Siede,W. (1995) DNA Repair incorrect processing transcripts DNA repair genes.          Mutagenesis. ASM Press, Washington, DC. repair genes yeast  RAD14,            3 Brendel,M., Khan,N  Haynes,R.H. (1970) Mol. Gen. Genet., 106, involved incision damaged DNA, intron-containing              289&#226;&#8364;&#8220;295. <br />  4014 Nucleic Acids Research, 1996, Vol. 24,  20 <br />  <br />  4   Game,J.C. Cox,B.S. (1971) Mutat. Res., 12, 328&#226;&#8364;&#8220;331.                       23 Ju,Q., Morrow,B.E. Warner,J.R. (1990) Mol. Cell. Biol., 10, <br />  5   Game,J.C. Cox,B.S. (1972) Mutat. Res., 16, 353&#226;&#8364;&#8220;362.                          5226&#226;&#8364;&#8220;5234. <br />  6   Brendel,M. Haynes,R.H. (1973) Mol. Gen. Genet., 125, 197&#226;&#8364;&#8220;216.             24 Tice-Baldwin,K., Fink,G.R. Arndt,K.T. (1989) Science, 246, 931&#226;&#8364;&#8220;935. <br />  7   Game,J.C. Cox,B.S. (1973) Mutat. Res., 20, 35&#226;&#8364;&#8220;44.                         25 Firmenich , Elias-Arnanz,M. Berg,P. (1995) Mol. Cell. Biol., 15, <br />  8   Henriques,J P. Brendel,M. (1990) Curr. Genet., 18, 387&#226;&#8364;&#8220;393.                1620&#226;&#8364;&#8220;1631. <br />  9   Henriques,J P. Moustacchi,E. (1980) Genetics, 95, 273&#226;&#8364;&#8220;288.              26 Hays,S.L., Firmenich  Berg,P. (1995) Proc. Natl. Acad. Sci. USA, <br /> 10   Henriques,J P., Vicente,E.J., daSilva,K.V.C.L. Schenberg C.G.            92, 6925&#226;&#8364;&#8220;6929. <br />      (1989) Mutat. Res., 218, 111&#226;&#8364;&#8220;124.                                             27 daSilva,K.V.C.L, Morais,M ,Jr, Henriques,J P. (1995) Curr. <br /> 11   Andrade,H.H.R., Marques,E.K., Schenberg C.G. Henriques,J P.              Genet., 27, 207&#226;&#8364;&#8220;212. <br />      (1989) Mol. Gen. Genet., 217, 419&#226;&#8364;&#8220;426.                                        28 Tarn,W.-Y., Lee,K.R. Cheng,S.-C. (1993) Mol. Cell Biol., 13, <br /> 12   Meira,L.B., Fonseca,M.B., Averbeck,D., Schenberg C.G.                      1883&#226;&#8364;&#8220;1891. <br />      Henriques,J P. (1992) Mol. Gen. Genet., 235, 311&#226;&#8364;&#8220;316.                       29 Tarn,W.Y., Hsu, C.H., Huang,K.T., Chen,H.R., Kao,H.Y., Lee,K.R. 13   Benathen  Beam,C  (1977) Radiat. Res., 69, 99&#226;&#8364;&#8220;116.                     Cheng,S.C. (1994) EMBO J., 13, 2421&#226;&#8364;&#8220;2431. <br />                                                                                    30 Tarn,W.Y., Lee,K.R. Cheng,S.C. (1993) Proc. Natl. Acad. Sci. USA, <br /> 14   Morais,M ,Jr, Vicente,E.J., Brozmanov&#195;&#161;,J., Schenberg C.G.                                                                                       90, 10821&#226;&#8364;&#8220;10825. <br />      Henriques,J P. (1996) Curr. Genet., 29, 211&#226;&#8364;&#8220;218. <br />                                                                                    31 Jachymczyk,W.J., von Borstel,R.C., Mowat,M.R.C. Hastings,P.J. <br /> 15   Brozmanov&#195;&#161;,J., Cern&#195;&#161;kov&#195;&#161;,L., Vlckov&#195;&#161;,V., DurajJ. Fridrichov&#195;&#161;  <br />                                                                                       (1981) Mol. Gen. Genet., 182, 196&#226;&#8364;&#8220;205. <br />      (1991) Mol. Gen. Genet., 227, 473&#226;&#8364;&#8220;480.                                        32 Maga&#195;&#177;a-Schwencke,N., Henriques,J P.,Chanet,P. Mousacchi,E. <br /> 16   Cernakov&#195;&#161;,L., Fridrichov&#195;&#161; , Pirsel,M., Kleibl,K., Duraj,J.                 (1982) Proc. Natl. Acad. Sci. USA, 79,1722&#226;&#8364;&#8220;1726. <br />      Brozmanov&#195;&#161;,J. (1991) Biochimie, 73, 285&#226;&#8364;&#8220;288.                                  33 Bardwell J., Bardwell,L., Iyer,N., Svefstrup,J.Q., Feaver,R.D., <br /> 17   Morais,M ,Jr, Brozmanov&#195;&#161;,J., Benfato,M., Duraj,J., Vlckov&#195;&#161;,V.              Kornberg,R.D. Friedberg,E.C. (1994) Mol. Cell. Biol., 14, 3569&#226;&#8364;&#8220;3576. <br />      Henriques,J P. (1994) Mutat. Res., 314, 209&#226;&#8364;&#8220;220.                            34 Vijayraghavan,U., Company,M. Abelson,J. ( <br /> </body></html>