<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2000 Genetics Society America <br />  <br />  <br />  <br />    Chromo-domain Proteins Schizosaccharomyces pombe Differentially <br />           Repress Transcription Various Chromosomal Locations <br />  <br />                                          Genevie`ve Thon Janne Verhein-Hansen <br />              Department Genetics, Institute Molecular Biology, University Copenhagen, DK-1353 Copenhagen K, Denmark <br />                                                           Manuscript received July 9, 1999 <br />                                                       Accepted publication February 7, 2000 <br />  <br />  <br />                                                              ABSTRACT <br />                 Transcription repressed regions fission yeast genome close centromeres, telomeres,              silent mating-type cassettes mat2-P mat3-M. repression involves chromo-domain proteins <br />              Swi6 Clr4. report chromo-domain proteins, Chp1 Chp2, important <br />              position effects. Chp1 showed specificity centromeric regions. essentiality              transcriptional repression centromeric markers correlates importance chromosome stability. <br />              Chp2 appeared pleiotropic. effects centromeric silencing pronounced              Chp1, participated telomeric position effects transcriptional silencing mating-type <br />              region. PolII-transcribed genes repressed placed Schizosaccharo- <br />              myces pombe rDNA clusters, situation analogous budding yeast Saccharomyces cerevisiae. Chp2, <br />              Swi6, Clr4,  lesser extent, Chp1 participated repression. <br />  <br />  <br />  <br /> E   UKARYOTIC chromosomes organized  <br />      mains distinctive properties, potential transcription recombination. <br />                                                                              cause partially symmetrical arrangement relative <br />                                                                              central core, centromeric sequences                                                                              proposed fold hairpin. rDNA forms parameters vary chromosomes ac-                          large clusters &#226;&#181;&#8216;500&#226;&#8364;&#8220;1000 kb ends chromo- <br /> cording DNA sequence, proximity special-                       3 (Umesono et al. 1983; Toda et al. 1984; Allshire <br /> ized structures telomeres centromeres,                       et al. 1987; Hoheisel et al. 1993). Telomeres consist specific state individual cells. Repetitive sequences                 degenerate repeats sequence TTACAGG. proteins interact define                        repeats &#226;&#181;&#8216;300 bp long. adjacent regions transcription recombination                         rDNA genes ends chromosome 3 duced. example, heterochromatic areas,                         telomere-associated sequences ends chromo- <br /> predominantly composed repeated sequences,                            somes 1 2 (Sugawara 1989). <br /> poorly transcribed. Drosophila silence eu-                          Position effects fission yeast transcription chromatic genes translocated nearby (reviewed                             observed near centromeres, telomeres, Weiler Wakimoto 1995). yeast Saccharo-                            vicinity mat2-P mat3-M silent mating-type cas- <br /> myces cerevisiae, recombination PolII transcription                      settes right arm chromosome 2. mat2-P <br /> occur differently rDNA, chromosomal region                          mat3-M cassettes linked 4.3 kb DNA repeats occur naturally eukaryotes, com-                       separates displays striking sequence homol- <br /> pared genomic regions  references                          ogy centromeric repeat (Grewal Klar <br /> Smith et al. 1999).                                                          1997). Marker genes introduced near centromeres, telo- <br />   interested position effects fission yeast                 meres, mating-type region repressed, Schizosaccharomyces pombe determining                         stringently variegated fashion, mechanisms <br /> repeated sequences influence effects. S. pombe                     utilize trans-acting factors Clr1, Clr2, Clr3, Clr4, chromosomes respectively 5.7, 4.6, 3.5 Mb                       Swi6, Rik1 (Lorentz et al. 1992; Thon Klar <br /> (Fan et al. 1988; Hoheisel et al. 1993; Mizukami et                          1992; Allshire et al. 1994, 1995; Ekwall Ruusala <br /> al. 1993). chromosome contains 40- 100-kb                          1994; Nimmo et al. 1994; Thon et al. 1994). addition centromeric region comprising 4- 7-kb central core                      repressing transcription, factors inhibit meiotic unique sequence flanked large inverted repeats                         recombination mating-type region (Egel et al. repeats placed tandem (Takahashi et al. 1992;                         1989; Klar Bonaduce 1991; Lorentz et al. 1992; <br /> Steiner Clarke 1994 references .                          Thon et al. 1994), indicating position effects                                                                              transcription recombination involve related <br />                                                                              structures mechanisms. silencing fac- <br />    Corresponding author: Genevie`ve Thon, Department Genetics,         tors responsible reduced recombination <br /> stitute Molecular Biology, University Copenhagen, &#195;&#732;ster Fari- <br /> magsgade 2A, DK-1353 Copenhagen K, Denmark.                                  observed centromeric areas (Nakaseko et al. 1986; <br /> E-mail: gen@biobase.dk                                                       Chikashige et al. 1989), repress tran- <br /> Genetics 155: 551&#226;&#8364;&#8220;568 ( June 2000) <br />  552                                          G. Thon J. Verhein-Hansen <br />  <br /> scription recombination genomic locations,           Bacterial strains used cloning: Cloning plasmid DNA known. addition position effects          performed Escherichia coli strains DH5 (Hanahan <br />                                                               1983) S1754 (Thon et al. 1999). <br /> transcription, position effects recombination            Cloning chp1: chp1 contained 4.3-kb BamHI- documented S. pombe transplacement study         Ecl 136I fragment genomic DNA. fragment puri- ade6-M26 hot spot recombination              fied cosmid ICRFc60G0618 (Reference Library, ICRF; <br /> showed ability M26 mutation promote        Lehrach 1990) cloned Bluescribe (Stratagene, La <br /> recombination ade6 gene depends chro-           Jolla, CA) digested BamHI HincII, create pGT121. <br />                                                               presence chp1 pGT121 ascertained restriction <br /> mosomal context (Virgin et al. 1995).                         mapping partial sequencing. <br />    proteins associate chromatin          Partial replacement chp1 LEU2 ura4&#226;&#180;&#8482;: EcoRV <br /> determine properties proteins containing chro-      cleaves pGT121 75 bp downstream predicted initiating <br /> matin organization modifier (chromo) domain            ATG chp1 open reading frame (ORF) Bgl II cleaves <br /> view Cavalli Paro 1998). motif         pGT121 1355 bp upstream predicted stop codon.                                                               total 1450 bp pGT121 DNA EcoRV identified Drosophila HP1 Pc proteins (Paro        Bgl II sites replaced 2.2-kb EheI-BamHI fragment Hogness 1991) subsequently &#226;&#172;&#381;40 proteins            pJJ250 (Jones Prakash 1990), contains S. various organisms (Aasland Stewart 1995;             cerevisiae LEU2 gene, create pGT122, 1.8-kb <br /> Koonin et al. 1995; current sequence databases).        HincII-BamHI fragment pON94  gift Olaf Nielsen),  combination           contains S. pombe ura4&#226;&#171;&#185; gene, create pGT162. <br />                                                               pON94 constructed filling ends 1.8-kb <br /> mains, repressors activators transcription <br />                                                               HindIII fragment, contains ura4&#226;&#171;&#185; (Grimm et al. 1988), proteins localize pericentric het-     adding SphI linkers, ligating product SphI <br /> erochromatin. chromo-domain related               site modified version pUC19 (Yannish-Perron et al. <br /> chromo-shadow domain mediate formation pro-            1985) HindIII site converted BamHI site. <br /> tein complexes association chromatin           insert pGT122 released digesting BamHI <br /> (Messmer et al. 1992; Powers Eissenberg 1993;             ScaI used transform diploid strain produced                                                               mating PG9 PG1141. chp1&#226;&#171;&#185;/chp1&#226;&#338;&#172;::LEU2 diploids <br /> Saunders et al. 1993; Platero et al. 1995; Strutt         identified Leu&#226;&#171;&#185; transformants Southern <br /> Paro 1997). potential partners pro-            blot genomic DNA restricted SphI SphI Nsi posed interact chromo-domain proteins, includ-        hybridized chp1 probe described (Southern <br /> ing components nuclear membrane (Ye                Northern blot analyses). diploids subjected <br /> Worman 1996; Ye et al. 1997) components            tetrad dissection. insert pGT162 released                                                               BamHI FspI used transform SP837. Ura&#226;&#171;&#185; trans- <br /> origin recognition complex (Pak et al. 1997).                 formants chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele identified South- <br />    chromo-domain proteins, Swi6 Clr4, influ-          ern blot genomic DNA digested NsiI SphI SacI <br /> ence position effects S. pombe (Lorentz et al. 1992;       probed chp1 probe. <br /> Ekwall Ruusala 1994; Thon et al. 1994; Allshire              Deletion chp1 substituted marker: pGT121 et al. 1995). Swi6 contains chromo- chromo-             digested BsaAI AccI, filled ends                                                               Klenow fragment E. coli DNA polymerase  religated, <br /> shadow domain (Lorentz et al. 1994; Aasland                                                               create pGT187. entire chp1 ORF deleted pGT187, <br /> Stewart 1995). Clr4 contains chromo- SET             33 nucleotides 5&#226;&#172;&#732; 77 nucleotides 3&#226;&#172;&#732; chp1 <br /> main [Su(var)3-9, Enhancer zeste, trithorax; Tschiersch    ORF. insert pGT187 released BamHI SphI <br /> et al. 1994; Ivanova et al. 1998]. swi6        used transform PG1544. DNA FOAR trans- <br /> clr4 mutated deleted, genes normally subject         formants digested NsiI BamHI Ecl 136III transcriptional silencing near centromeres, telomeres,        hybridized chp1 probe, allowed identify <br />                                                               transformants chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele PG1544 mating-type region derepressed. Consistent      replaced chp1&#226;&#338;&#172; allele. phenotypes, Swi6 localizes centromeres,            Replacement chp2 ura4&#226;&#180;&#8482;: 1.8-kb HindIII frag- <br /> telomeres, mating-type region (Ekwall et           ment containing S. pombe ura4&#226;&#171;&#185; gene (Grimm et al. 1988) <br /> al. 1995). Clr4 important localization Swi6     cloned Bluescript KSII(&#226;&#171;&#185;) (Statagene) EcoRV <br /> (Ekwall et al. 1996)  expressed plas-         site ura4&#226;&#171;&#185; gene close EcoRV site polylinker, <br />                                                               create pGT188. oligonucleotides GTO-150 (5&#226;&#172;&#732; TCCCC <br /> mid, itself seen rDNA (Sawin Nurse 1996).       CCGGGAGCTCAGATCGTTATACACTTTACGTATCTAGG 3&#226;&#172;&#732;) tested chromo-domain proteins            GTO-151 (5&#226;&#172;&#732; CGGGATCCGTCGACTATTAAGACTTTC S. pombe, Chp1 (Doe et al. 1998) Chp2,            CAGATATACCAAC 3&#226;&#172;&#732;) used amplify 777 bp geno- <br /> involved processes Swi6 Clr4.           mic DNA chp2 ORF. GTO-152 (5&#226;&#172;&#732; AACTGC <br /> addition assaying effects Chp1 Chp2          AGTCGACTTGATCTTTGGACTTAATAATTAGAATTAC <br />                                                               G 3&#226;&#172;&#732;) GTO-153 (5&#226;&#172;&#732; TCCCCCGGGAAGCTTCTCGAGATG regions silencing previously observed,                                                               GCGCTTGAAGGGCTTAGTGCGC 3&#226;&#172;&#732;) used amplify <br /> investigated Swi6, Clr4, Chp1, Chp2 modi-          776 bp genomic DNA chp2 ORF. <br /> fied efficiency PolII transcription rDNA.       oligonucleotides purchased Operon Technolo- <br />                                                               gies (Alameda, CA). PCR performed Native <br />                                                               Pfu DNA polymerase (Stratagene) 80 &#226;?&#174;l containing              MATERIALS METHODS                            buffer provided manufacturer, 1 &#226;?&#174;g S. pombe genomic <br />                                                               DNA, 1 &#226;?&#174;m primer, 0.5 mm dNTPs. amplifica- <br />   Sequence chp1 chp2: Chp1 (GenBank accession       tion cycles follows: 3 min 94&#226;&#172;&#353;; 15 times [1 min 94&#226;&#172;&#353;, 1 <br /> Q10103) encoded cosmid SPAC18G6 (Z68198)        min 60&#226;&#172;&#353;, 1 min 72&#226;&#172;&#353;]; 7 min 72&#226;&#172;&#353;. PCR products gel <br /> Chp2 (CAA16917) SPBC16C6 (AL021767).                       purified. product amplified GTO-150 GTO-151 <br />                                                Fission Yeast Chromo-domain Proteins                                           553 <br />  digested Sac BamHI. product amplified           Amersham (Piscataway, NJ), following templates: GTO-152 GTO-153 digested Sal SmaI.         BamHI-SphI fragment pGT121 chp1; purified PCR fragments ligated way ligation       product amplified genomic DNA GTO-150 Bluescript SKII(&#226;&#171;&#186;) (Stratagene) digested SacI SmaI        GTO-153 chp2; 1.8-kb HindIII fragment ura4 1.8-kb BamHI-Sal fragment pGT188,          (Grimm et al. 1988); 2-kb BamHI-Sal pJJ250 contains ura4&#226;&#171;&#185;. clone containing ura4&#226;&#171;&#185; gene flanked          LEU2 ( Jones Prakash 1990). Antisense ura4 riboprobes sequences normally flanking chp2 ORF obtained           Northern hybridization Riboprobe II ligation named pGT190. insert pGT190          core (Promega), 3000 Ci/mmol [&#226;?&#163;-32P]<span id='am-20' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span>UTP</span> (Amer- released SacI XhoI used transform SP837.       sham), T7 RNA polymerase (Promega), pON140  gift DNA Ura&#226;&#171;&#185; transformants digested HindIII         Olaf Nielsen) linearized BamHI template. pON- <br /> BamHI PstI hybridized chp2 probe described          140 clone ura4&#226;&#171;&#185; gene pGEM4 (Promega). (Southern Northern analyses). identified             Stability Ura&#226;&#171;&#186; phenotype: FOAR colonies purified <br /> strain chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele, PG1744.                       rounds streaking FOA-containing plates    Replacement chp2 LEU2: 3.7-kb BamHI-PstI            patched YES plates. Following growth nonselective <br /> fragment 0.8-kb Pst SalI fragment pGT190         conditions YES &#226;&#181;&#8216;15 generations, cells suspended <br /> ligated way ligation 2-kb BamHI-SalI        water used spot tests selective media DNA <br /> pJJ250, contains S. cerevisiae LEU2 gene ( Jones     preparation. <br /> Prakash 1990). clone similar pGT190, containing              Chromosome stability: Mitotic loss minichromosome <br /> LEU2 instead ura4&#226;&#171;&#185;, obtained ligation,            Ch16m23::ura4&#226;&#171;&#185;-Tel[72] assayed minichro- <br /> pGT193. insert pGT193 released digesting        mosome Ch16 (Allshire et al. 1995): cells containing Sac XhoI used transform diploid strain produced     ade6-M210 allele chromosome 3, complementing allele mating PG1636 PG1637. chp2&#226;&#171;&#185;/chp2&#226;&#338;&#172;::LEU2              ade6-M216 Ch16m23::ura4&#226;&#171;&#185;-Tel[72], mutations  <br /> diploids identified Leu&#226;&#171;&#185; transformants         terest propagated medium lacking <span id='am-7' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-8' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> (AA-ade) <br /> Southern blot analysis similar described     33&#226;&#172;&#353;, plated medium containing small ade- <br /> chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele diploids subjected        (MSA supplemented 100 mg <span id='am-30' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span>, 200 mg l-leu- <br /> tetrad dissection.                                                 cine, 15 mg adenine liter), incubated 33&#226;&#172;&#353;.    Deletion chp2 substituted marker: pGT190         loss rate Ch16m23::ura4&#226;&#171;&#185;-Tel[72] determined digested Sal  3.7-kb Sal fragment, contains     number colonies red sector equal greater Bluescript 777 bp DNA flanking chp2 centromere-      half colony divided sum white sectored <br /> proximal  ligated 0.8-kb Sal fragment,   colonies. <br /> contains 776 bp DNA flanking chp2 centromere-distal  clone precise deletion chp2 ORF obtained ligation, pGT191. insert pGT191                                   RESULTS released digesting SacI XhoI used replace chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185; allele PG1753. DNA FOAR <br />                                                                       Deletion chromo-domain protein genes, chp1 <br /> transformants digested ClaI BamHI PstI         chp2, S. pombe: gene capable encoding hybridized chp2 probe. identified strain    109-kD protein N-terminal chromo-domain chp2&#226;&#338;&#172; allele, PG1776.                                              present left arm S. pombe chromosome 1.    Introduction ura4&#226;&#180;&#8482; rDNA repeats: 1.8-kb           gene subject previous study named <br /> HindIII fragment containing ura4&#226;&#171;&#185; (Grimm et al. 1988)                                                                    chp1 (chromo-domain protein S. pombe; Doe et al. <br /> cloned pUC8 (Viera Messing 1982). resulting <br /> construct linearized Bsr FI used transform         1998). gene, refer chp2, PG1634. Ura&#226;&#171;&#185; transformants analyzed Southern blots         located right arm chromosome 2 encodes genomic DNA restricted EcoRV, HindIII, BamHI            43-kD protein displaying extensive sequence similarity hybridized LEU2 ura4 probe. transformant      S. pombe Swi6 protein (Figure 1). Like Swi6 pUC8 ura4 integrated pBR322 sequence present         family proteins related Drosophila HP1, Chp2 rDNA repeat PG1634 saved PG1661. <br />    Yeast media culture conditions: YES (Thon Friis         contains chromo- chromo-shadow domain. <br /> 1997) used rich medium; MSA (Egel et al. 1994) supple-      chp1 chp2 ORFs discovered mented 100 mg adenine, 100 mg uracil, 200 mg              S. pombe genome sequencing project (Sanger Center). <br /> <span id='am-35' about='obo:IMR_0200492' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-36' property="oboInOwl:id" content="IMR:0200492" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="L-Leucine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00123" datatype="xsd:string"></span>l-leucine</span> liter used sporulation medium; drop          created deletion alleles chp1 chp2: <br /> media (Rose et al. 1990), FOA medium (Thon et al. 1999),           chp1 entire chp2 ORF replaced YE (Moreno et al. 1991) used test auxotrophies. <br /> Plates incubated 33&#226;&#172;&#353;.                                      S. cerevisiae LEU2 gene, functional homolog <br />    Yeast transformation strain construction: lithium ace-    S. pombe leu1&#226;&#171;&#185; gene, S. pombe ura4&#226;&#171;&#185; <br /> tate protocol adapted Heyer et al. (1986) Moreno          gene. Deletion entire ORF substituted <br /> et al. (1991) used transforming S. pombe. Strains trans-   markers obtained. deletions originally <br /> formed constructs containing LEU2 ura4&#226;&#171;&#185; se-           introduced chromosome diploid cells. lected drop media lacking leucine uracil. FOA  <br />                                                                    chp1 chp2 deletions proved viable hap- <br /> dium used select loss ura4&#226;&#171;&#185;. strains produced transformation progeny subsequent crosses        loid progeny transformed diploids. listed Table 1.                                             observed independently chp1 (Doe et al. 1998). chp1- <br />    Southern Northern blot analyses: S. pombe DNA           deleted cells formed colonies significantly smaller prepared according Moreno et al. (1991) RNA ac-             wild type, chp2-deleted cells formed colo- <br /> cording Schmitt et al. (1990) liquid YES cultures          nies large wild type. <br /> incubated 30&#226;&#172;&#353;. Blotting hybridization conditions described previously (Thon et al. 1999). probes used           Deleting chp1 chp2 alleviates centromeric position Southern hybridization random priming         effects: S. pombe centromeres organized large <br /> kit (Promega, Madison, WI), 3000 Ci/mmol [&#226;?&#163;-32P]<span id='am-25' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="dCTP" datatype="xsd:string"></span>dCTP</span>          inverted repeats (Takahashi et al. 1992; Steiner                                                                                                                                       554 <br />  <br />  <br />                                                                TABLE 1 <br />                                                        Strains genotypes <br />  <br /> Strain                                                  Genotype                                                   Sourcea <br /> FY336           h&#226;&#171;&#185; cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210                                         Allshire et al. (1994) <br /> FY340           h&#226;&#171;&#185; TM1(NcoI)-ura4&#226;&#171;&#185; Random int. leu1-32 ura4-DS/E ade6-210                                   Allshire et al. (1994) <br /> FY489           h&#226;&#171;&#186; leu1-32 ura4-DS/E ade6-210                                                               Allshire et al. (1995) <br /> FY501           h&#226;&#171;&#185; imr L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                      Allshire et al. (1995) <br /> FY520           h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (ch16 ade6-216)                       Nimmo et al. (1995) <br /> FY521           h&#226;&#171;&#186; Ch16 m23::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216)                               Allshire et al. (1995) <br /> FY611           h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115              Allshire et al. (1995) <br /> FY648           h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210                                            Allshire et al. (1995) <br /> FY939           h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                 Allshire et al. (1995) <br /> FY967           h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                                Allshire et al. (1995) <br /> FY986           h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210                              Allshire et al. (1995) <br /> PG9             mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216                                             Thon Klar (1992) <br /> PG445           mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-M216                                                  Thon et al. (1998) <br /> PG596           mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 swi6-115                                    Thon et al. (1994) <br /> PG912           mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M210 clr4-681                                    Thon et al. (1994) <br /> PG1032          mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 clr4-681                         study <br /> PG1141          h90 leu1-32 ura4-D18, ade6-210                                                              Thon et al. (1999) <br /> PG1402          mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-M216                                     study <br /> PG1419          mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-M216 clr4-681                            study <br /> PG1454          mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 chp1&#226;&#338;&#172;::LEU2                                           study <br /> PG1544          h90 leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  study <br /> PG1572          mat1-Msmt-0 leu1-32 ura4-D18 Ylp2.4 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                            study <br /> PG1594          mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-M216 clr4-681                                         study <br /> PG1600          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E -D18 ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2   study <br />                                                                                                                                      G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br /> PG1605          mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp1&#226;&#338;&#172;::LEU2                      study <br /> PG1634          h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 swi6-115                                                           study <br /> PG1636          h90 leu1-32 ura4-DS/E ade6-210                                                              study <br /> PG1637          h90 leu1-32 ura4-DS/E ade6-216                                                              study <br /> PG1639          mat1-Msmt-0 &#226;&#338;&#172;(BglII-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp1&#226;&#338;&#172;::LEU2       study <br /> PG1640          mat1-Msmt-0 &#226;&#338;&#172;(BglII-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 swi6-115          study <br /> PG1661          h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 swi6-115                                                study <br /> PG1691          h&#226;&#171;&#185; imr L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                          study <br /> PG1699          h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                     study <br /> PG1703          h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                    study <br /> PG1709          h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                  study <br /> PG1717          h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                                study <br /> PG1721          h&#226;&#171;&#185; cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp1&#226;&#338;&#172;::LEU2                             study <br /> PG1731          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681              study <br /> PG1732          h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2           study <br /> PG1734          h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7                                                        study <br /> PG1736          h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 clr4-681                                               study <br />   (Continued) <br />                                                                                  TABLE 1 <br />                                                                               (Continued) <br />  <br /> Strain                                                                  Genotype                                                                          Sourcea <br /> PG1744                       h90 ura4-D18 leu1-32 ade6-216 chp2&#226;&#338;&#172;::ura4                                                                            study <br /> PG1753                       h90 ura4-DS/E leu1-32 ade6-210 chp2&#226;&#338;&#172;::ura4                                                                           study <br /> PG1763                       h90 cnt1/TM1(NcoI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                     study <br /> PG1764                       h&#226;&#171;&#185; otr1R(SphI)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                         study <br /> PG1765                       h&#226;&#171;&#185; imr1 L(NcoI)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                  study <br /> PG1767                       h&#226;&#171;&#185; otr1L(dh/Bgl II)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                              study <br /> PG1769                       h&#226;&#171;&#185; otr1L(dh/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                             study <br /> PG1771                       h&#226;&#171;&#185; otr1L(dg1a/HindIII)::ura4&#226;&#171;&#185; oriII leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                           study <br /> PG1773                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp2&#226;&#338;&#172;::LEU2                                   study <br /> PG1774                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) chp2&#226;&#338;&#172;::LEU2                                    study <br /> PG1776                       h90 ura4-DS/E leu1-32 ade6-210 chp2&#226;&#338;&#172;                                                                                 study <br /> PG1778                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  study <br /> PG1779                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  study <br /> PG1781                       mat1-P&#226;&#338;&#172;17::LEU2 &#226;&#338;&#172;(482)mat3-M leu1-32 ura4-D18 ade6-216 swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                  study <br /> PG1783                       mat3-M(EcoRV)::ura4&#226;&#171;&#185; leu1-32 ura4-DS/E ade6-210 chp2&#226;&#338;&#172;::LEU2                                                          study <br /> PG1787                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp2&#226;&#338;&#172;::LEU2                               study <br /> PG1790                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 chp2&#226;&#338;&#172;::LEU2                                               study <br /> PG1792                       h&#226;&#171;&#185; ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 chp1&#226;&#338;&#172;                                                                            study <br /> PG1794                       h90 ura4-DS/E leu1/YIp2.4 pUCura4&#226;&#171;&#185;-7 chp2&#226;&#338;&#172;                                                                           study <br /> PG1796                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-216 chp1&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               study <br /> PG1797                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-210 swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               study <br /> PG1799                       mat1-P&#226;&#338;&#172;17::LEU2 leu1-32 ura4-D18 ade6-216 chp2&#226;&#338;&#172;::ura4&#226;&#171;&#185;                                                               study <br /> PG1970                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 chp1&#226;&#338;&#172;::LEU2                          study <br /> PG1972                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681 chp1&#226;&#338;&#172;::LEU2                          study <br />                                                                                                                                                                              Fission Yeast Chromo-domain Proteins <br />  <br />  <br />  <br />  <br /> PG1974                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 chp2&#226;&#338;&#172;::LEU2                           study <br /> PG1976                       h90 Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) clr4-681 chp2&#226;&#338;&#172;::LEU2                          study <br /> PG1978                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-D18 ade6-210 (Ch16 ade6-216) chp1&#226;&#338;&#172;::LEU2 chp2&#226;&#338;&#172;::LEU2                         study <br /> PG1984                       h&#226;&#171;&#185; Ch16 m23::ura4&#226;&#171;&#185;-TEL[72] leu1-32 ura4-DS/E ade6-210 (Ch16 ade6-216) swi6-115 clr4-681                              study <br /> SP837                        h90 leu1-32 ura4-D18 ade6-216                                                                                        Thon et al. (1994) <br /> SP1124                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M216                                                                   Thon et al. (1994) <br /> SP1126                       mat1-Msmt-0 mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M210 swi6-115                                                          Thon et al. (1994) <br /> SP1151                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; ura4-D18 ade6-M216                                                   Thon et al. (1994) <br /> SP1165                       mat1-Msmt-0 &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; leu1-32 ura4-D18 ade6-M216 clr4-681                                  Thon et al. (1994) <br /> YIp2.4-1                     h&#226;&#171;&#186; leu1/YIp2.4                                                                                                       Toda et al. (1984) <br />       complete genealogy strains constructed study available request. Mutated engineered alleles used strain constructions described following studies: mat1-P&#226;&#338;&#172;17::LEU2: Arcangioli Klar (1991); mat1-Msmt-0 : Engelke et al. (1987) Styrkarsdottir et al. (1993); mat3-M(EcoRV)::ura4&#226;&#171;&#185;: <br /> Thon Klar (1992); mat3-P(EcoRV)::ura4&#226;&#171;&#185;: Thon Klar (1993); mat2-P(XbaI)::ura4&#226;&#171;&#185; &#226;&#338;&#172;(Bgl II-BssHII)mat2-P(XbaI)::ura4&#226;&#171;&#185; : Thon et al. (1994); &#226;&#338;&#172;(482)mat3-M: Thon <br /> et al. (1999); ura4-D18: Grimm et al. (1988); ade6-M210 ade6-M216: Gutz (1963); swi6-115: Gutz Schmidt (1985); swi6&#226;&#338;&#172;::ura4&#226;&#171;&#185;: Lorentz et al. (1994); clr4-681: <br /> Thon et al. (1994). <br />                                                                                                                                                                              555 <br />  556                                            G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br />                                                                                                   Figure      1.&#226;&#8364;&#8221;Sequence <br />                                                                                                 comparison Chp2                                                                                                 Swi6. predicted protein <br />                                                                                                 sequences Chp2 Swi6 <br />                                                                                                 aligned using                                                                                                 CLUSTAL W program <br />                                                                                                 (Thompson et al. 1994). <br />  <br />  <br />  <br />  <br /> Clarke 1994 references . ura4&#226;&#171;&#185; gene             difference FOA-containing media used. placed locations centromere           ura4&#226;&#171;&#185; transcript relative control chromosome 1 Allshire et al. (1994, 1995). Inser-         transcript ura4-DS/E similar studies. <br /> tion locations results level expression         noted (Allshire et al. 1995), inferior ura4&#226;&#171;&#185; gene wild-type location    ura4&#226;&#171;&#185; transcript inferior wild type variegated phenotype clonally derived            support effective growth medium lacking <br /> fractions cell population express ura4&#226;&#171;&#185;,         uracil. rest cells repressed. Swi6 Clr4         Deletion chp1 completely derepressed centro- <br /> required centromeric repression (Allshire et           meric loci caused small derepression cnt1, al. 1995).                                                      central centromeric insertion site (Figure 2,    tested effect Chp1 Chp2 expres-         B). Deletion chp2 partially derepressed ura4&#226;&#171;&#185; sion ura4&#226;&#171;&#185; centromeric integration sites (Figure      centromeric sites tested. increased growth  <br /> 2). experiment, compared ability            dium lacking uracil reduced growth medium wild-type, chp1-, chp2-deleted cells containing         containing FOA (Figure 2A). increased centromeric ura4&#226;&#171;&#185; gene form colonies medium               ura4&#226;&#171;&#185; transcript detected Northern blot <br /> lacking uracil medium containing FOA, toxigenic            relative arising sites substrate Ura4 protein (Grimm et al. 1988; Figure        wild-type cells (Figure 2B).  derepression <br /> 2A). second experiment, examined                  strikingly small compared caused dele- ura4&#226;&#171;&#185; transcript present cells         tion chp1. true sites examined strains Northern blot analysis (Figure 2B). Tran-            possibly cnt1. <br /> scripts originating ura4DS/E allele (Allshire             FOA resistance strains containing ura4&#226;&#171;&#185; cen- <br /> et al. 1994), truncated ura4 gene occupying natural       tromeric marker generally reduced deletion chromosomal location, used internal control.         chp1 chp2, noted approximately cell <br />    wild-type cells, tightest repression ura4&#226;&#171;&#185;     thousand able generate sized colony <br /> observed outer centromeric repeats (otr1L)               FOA-containing medium; frequency cells dg1 region. Insertion central area        varied culture culture. chp&#226;&#171;&#185; strains gave rise (cnt) inner repeats (imr1L) allowed               similar kind papillation (Figure 2A). investi- <br /> higher expression ura4&#226;&#171;&#185;. phenotypes agree              gated nature FOA resistance propagating reported Allshire et al. (1994, 1995).            cells growing FOAR colonies nonselec- strains (FY939, FY967, FY501, FY336) grew             tive conditions measuring ability form <br /> poorly FOA medium experiments origi-             colonies absence uracil. performed <br /> nally observed (compare Figure 2A Figure 1C            Southern blots test possible alterations ura4 <br /> Figure 2B Allshire et al. 1995). believe        gene. independent FOAR colonies strain <br />                                             Fission Yeast Chromo-domain Proteins                                        557 <br />  <br />  <br />  <br />  <br />                                                                                               Figure 2.&#226;&#8364;&#8221;Effect Chp1 <br />                                                                                            Chp2 gene expression <br />                                                                                            near centromere 1.   Expres- <br />                                                                                            sion ura4&#226;&#171;&#185; gene assayed <br />                                                                                            plating efficiency selec- <br />                                                                                            tive media. Serial 10-fold dilu- <br />                                                                                            tions cell suspensions                                                                                            spotted indicated  <br />                                                                                            dia. cells contained cen- <br />                                                                                            tromeric insertions ura4&#226;&#171;&#185; <br />                                                                                            depicted B (1&#226;&#8364;&#8220;6), <br />                                                                                            random integration ura4&#226;&#171;&#185; <br />                                                                                            (R ), length ura4 <br />                                                                                            gene (&#226;&#171;&#186;). strains  1: <br />                                                                                            &#226;&#171;&#185;, FY939; chp1&#226;&#338;&#172;, PG1699; chp- <br />                                                                                            2&#226;&#338;&#172;, PG1767; 2: &#226;&#171;&#185;, FY967; chp1&#226;&#338;&#172;, <br />                                                                                            PG1703; chp2&#226;&#338;&#172;, PG1769; 3: &#226;&#171;&#185;, <br />                                                                                            FY986; chp1&#226;&#338;&#172;, PG1709; chp2&#226;&#338;&#172;, <br />                                                                                            PG1771; 4: &#226;&#171;&#185;, FY648; chp1&#226;&#338;&#172;, <br />                                                                                            PG1717; chp2&#226;&#338;&#172;, PG1764; 5: &#226;&#171;&#185;, <br />                                                                                            FY501; chp1&#226;&#338;&#172;, PG1691; chp2&#226;&#338;&#172;, <br />                                                                                            PG1765; 6: &#226;&#171;&#185;, FY336; chp1&#226;&#338;&#172;, <br />                                                                                            PG1721; chp2&#226;&#338;&#172;, PG1763; R , <br />                                                                                            FY340; &#226;&#171;&#186;, FY489. (B) Northern <br />                                                                                            blot analysis ura4&#226;&#171;&#185; tran- <br />                                                                                            script. RNA prepared                                                                                            strains displayed                                                                                            ura4&#226;&#171;&#185; transcript <br />                                                                                            estimated hybridization <br />                                                                                            ura4 riboprobe. <br />  <br />  <br />  <br />  <br /> displayed Figure 2, 1&#226;&#8364;&#8220;6, tested manner.        kb telomere, constructed steps FOA resistance loss          (Niwa et al. 1989; Nimmo et al. 1994). ura4&#226;&#171;&#185; expressed ura4&#226;&#171;&#185; gene strains examined FY986        variegated manner Ch16m23::ura4&#226;&#171;&#185;-Tel[72] FY648. FY986 FY648, FOA-resistant state         (Nimmo et al. 1994). Clr4  lesser extent, Swi6 <br /> returned efficiently Ura&#226;&#171;&#185; ura4&#226;&#171;&#185; gene ap-            contribute repression ura4&#226;&#171;&#185; location <br /> peared unchanged Southern blots. epigenet-             (Allshire et al. 1995). crossed Ch16m23::ura4&#226;&#171;&#185;- <br /> ically controlled resistance likely masked dele-    Tel[72] minichromosome  respectively, chp1- tion events occurring strains       chp2-deleted strains. Crosses swi6-115 clr4-681 <br /> tested.                                                        performed time comparison.    Deletion chp2, chp1, alleviates telomeric     assayed expression ura4&#226;&#171;&#185; mutant position effects: Ch16 530-kb S. pombe minichromo-        wild-type strains issued cross spot derived chromosome 3 radiation-induced            tests selective media. Strains identical genotypes <br /> breakage (Matsumoto et al. 1987). contains ade6-        proved similar plating efficiencies M216 allele, allows monitor presence       various selective plates (data shown). Spot tests cells bearing ade6-M210 allele intragenic       repeated strain genotype complementation ade6-M216 ade6-                ura4&#226;&#171;&#185; transcript estimated Northern <br /> M210 alleles (Gutz 1963). truncated version Ch16,         blot. seen Figure 3, deleting chp1 Ch16m23::ura4&#226;&#171;&#185;-Tel[72], ura4&#226;&#171;&#185; gene &#226;&#181;&#8216;1             influence telomeric position effect, deleting chp2 <br />  558                                         G. Thon J. Verhein-Hansen <br />  <br />                                                              mosome loss (materials methods; Allshire et <br />                                                              al. 1995). mutant strains tested, noticed                                                              presence colonies red narrow <br />                                                              white sector (Table 2). phenotype indicative                                                              multiple consecutive missegregation events ex- <br />                                                              pected occur infrequently according calcu- <br />                                                              lated rates chromosome loss. existence                                                              phenotype suggests mutations tested destabilize <br />                                                              chromosomes heritable manner fraction                                                              cell population. property account                                                              relatively large variations loss rates observed                                                              independent cultures strain (Table 2). Alter- <br />                                                              natively, red colonies narrow white sector                                                              produced Ade&#226;&#171;&#185; cells temporarily stopped dividing <br />                                                              missegregation event. <br />                                                                 Deletion chp2, chp1, derepresses tran- <br />                                                              scription mating-type region: mating-type <br />                                                              region S. pombe comprises linked cassettes, mat1, <br />                                                              transcribed, mat2-P mat3-M,                                                              silenced (Egel Gutz 1981; Beach 1983). Silencing <br />                                                              extends regions flanking mat2-P mat3-M                                                              10.9-kb region separates cassettes <br />                                                              (Thon Klar 1992; Thon et al. 1994; Grewal                                                              Klar 1997). affects ura4&#226;&#171;&#185; gene placed                                                              XbaI site located &#226;&#181;&#8216;400 bp centromere-distal                                                              mat2-P cassette [mat2-P(XbaI)::ura4&#226;&#171;&#185; allele; Thon et al. <br />                                                              1994; Figure 4A] EcoRV site located &#226;&#181;&#8216;150 bp <br />   Figure 2.&#226;&#8364;&#8221;(Continued). <br />                                                              centromere-distal mat3-M cassette [mat3-M(Eco- <br />                                                              RV)::ura4&#226;&#171;&#185; allele; Thon Klar 1992; Figure 4B]. <br />                                                              products clr1, clr2, clr3, clr4, clr6, swi6, rik1, esp1, <br /> caused increased expression ura4&#226;&#171;&#185;. increased       esp2, esp3 essential repression. expression caused chp2 deletion similar        factors act partially redundant pathways, caused mutation clr4.                           double mutations required completely abol- <br />    Role S. pombe chromo-domain proteins chromo-        ish silencing mating-type region. Double mu- segregation: measured rate loss        tations strong synergistic effects reside <br /> minichromosome Ch16m23::ura4&#226;&#171;&#185;-Tel[72] wild-               trans-acting factors belonging type, chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, swi6-115, clr4-681       class consist combination cis- trans-acting <br /> cells. results presented Table 2. swi6-115    mutations (Thon et al. 1994, 1999; Thon Friis <br /> chp1&#226;&#338;&#172;::LEU2 increased loss Ch16m23::ura4&#226;&#171;&#185;-Tel[72],        1997; Grewal et al. 1998). examined Chp1 accordance previous reports (Allshire et al.         Chp2 contributed repression ura4&#226;&#171;&#185; placed <br /> 1995; Doe et al. 1998). contrast, deletion chp2     near mat2-P mat3-M, repression mat- clr4-681 mutation modest possibly          ing-type genes mat2-P mat3-M. tests  <br /> significant effect rate loss compared     formed strains deletion cis-acting <br /> wild type. constructed strains containing pairwise        silencing elements. <br /> combinations chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, swi6-              assayed ura4&#226;&#171;&#185; expression mating- <br /> 115, clr4-681 alleles. double mu-      type region cells lacking chp1 chp2 spot tests tants viable rate loss Ch16m23::     selective media (Figure 4). Wild-type, swi6-115, clr4- <br /> ura4&#226;&#171;&#185;-Tel[72] greater chp1&#226;&#338;&#172;::LEU2 swi6-115        681 cells spotted plates compari- chp1&#226;&#338;&#172;::LEU2 clr4-681 double mutants          son. Cells lacking Chp1 grew FOA single mutants (Table 2).        poorly medium lacking uracil cells containing combinations tested, rate loss similar    functional protein, indicating Chp1 effect <br /> caused stronger mutation pair. chp2        transcription ura4&#226;&#171;&#185; gene placed mating- <br /> deletion did increase chromosome loss      type region. contrast, lack Chp2 allowed increased <br /> backgrounds tested. minichromosome loss assay         growth absence uracil, indicating Chp2 <br /> used relies measuring changes white Ade&#226;&#171;&#185;           required repression ura4&#226;&#171;&#185; near mat2-P <br /> phenotype displayed cells containing minichro-        mat3-M. addition forming colonies mosome red Ade&#226;&#171;&#186; phenotype displayed chro-         absence uracil, cells deleted chp2 containing <br />                                              Fission Yeast Chromo-domain Proteins                                               559 <br />  <br />  <br />  <br />  <br />                                                                              Figure 3.&#226;&#8364;&#8221;Effect Chp1, Chp2, Swi6, Clr4                                                                            gene expression near telomere.   Spot test analysis. <br />                                                                            Suspensions cells minichromosome <br />                                                                            Ch16m23::ura4&#226;&#171;&#185;-Tel[72] indicated mutations <br />                                                                            spotted Figure 2 estimate level                                                                            expression ura4&#226;&#171;&#185;. m23-TEL[72] strains: &#226;&#171;&#185;, FY520; <br />                                                                            chp1&#226;&#338;&#172;, PG1732; chp2&#226;&#338;&#172;, PG1773; swi6-115, FY611; clr4- <br />                                                                            681, PG1731; m23, FY489; &#226;&#171;&#186;, FY521. (B) Northern blot <br />                                                                            analysis. RNA prepared strains plated                                                                            hybridized ura4 riboprobe. <br />  <br />  <br />  <br />  <br /> ura4&#226;&#171;&#185; near mat2-P form colonies FOA-                    deletion chp2 appeared equivalent swi6 containing medium, indicating deletion chp2                   clr4 mutants (Figure 4B). <br /> did derepress ura4&#226;&#171;&#185; near mat2-P efficiently                    Wild-type S. pombe cells efficiently switch content <br /> mutations swi6 clr4 (Figure 4A, . stronger                mat1 cassette P M copying derepression ura4&#226;&#171;&#185; obtained simultaneously                  silent information present mat2-P mat3-M. Follow- <br /> deleting chp2 mat2-P cis-acting element (Figure                 ing switches, cells mate form zygotic asci <br /> 4A, . derepression seen mat3-M following                isolated colonies. Strains unable switch  <br />  <br />                                                            TABLE 2 <br />                           Effect Chp1, Chp2, Swi6, Clr4 stability minichromosomea <br />  <br />                                         Half-sectored   red               Colonies           Loss rate cell   Mutant/ <br /> Strain             Background             colonies       colonies               examined              division (%)       WT ratio <br /> FY520        WT                          1,0,0,3        0,0,0,0          1098,1081,1022,2642               0.07               1 <br /> PG1600       chp1&#226;&#338;&#172;::LEU2                 6,4,7,9,12     2,4,9,6,9        1105,867,1085,421,1791            1.29              18 <br /> PG1774       chp2&#226;&#338;&#172;::LEU2                 1,1,1,1,6,3    0,0,2,0,0,1      896,984,1097,977,3519             0.21               3 <br /> FY611        swi6-115                    4,3,12,4,40    3,2,0,0,13       956,1016,568,663,1804             1.52              22 <br /> PG1731       clr4-681                    3,0,0,2,0      0,0,0,1,0        1230,973,998,854,651              0.13               2 <br /> PG1978       chp1&#226;&#338;&#172;::LEU2 chp2&#226;&#338;&#172;::LEU2     4,6,2          1,4,0            536,723,536                       0.95              14 <br /> PG1970       chp1&#226;&#338;&#172;::LEU2 swi6-115        25,5,24        11,4,31          896,269,618                       5.60              80 <br /> PG1972       chp1&#226;&#338;&#172;::LEU2 clr4-681        21,14,8        7,21,23          749,707,542                       4.75              68 <br /> PG1974       chp2&#226;&#338;&#172;::LEU2 swi6-115        15,4,2,11      1,3,0,0          774,456,231,1150                  1.38              20 <br /> PG1976       chp2&#226;&#338;&#172;::LEU2 clr4-681        2,0,2,3        0,0,0,0          1116,486,884,905                  0.21               3 <br /> PG1984       swi6-115 clr4-681           6,10,9         7,3,4            690,969,785                       1.60              23 <br />         independent cultures assayed described materials methods. WT, wild type. <br />  560                                            G. Thon J. Verhein-Hansen <br />  <br />  <br />  <br />  <br />   Figure 4.&#226;&#8364;&#8221;Effects Chp1, Chp2, Swi6, Clr4 expression ura4&#226;&#171;&#185; mating-type region.   Effects near <br /> mat2-P. Cells ura4&#226;&#171;&#185; XbaI site centromere-distal mat2-P (Xb) spotted indicated media Figure 2. <br />   Strains intact mat2-P silencing elements.   Strains mat2-P centromere-proximal deletion &#226;&#181;&#8216;1.5 kb Bgl II (Bg) BssHII (Bs) site. mat2-P(XbaI)::ura4 strains: &#226;&#171;&#185;, SP1124; chp1&#226;&#338;&#172;, PG1605; chp2&#226;&#338;&#172;, PG1790; swi6-115, <br /> SP1126; clr4-681, PG1032; &#226;&#338;&#172;(B-B)mat2-P(XbaI)::ura4 strains: &#226;&#171;&#185;, SP1151; chp1&#226;&#338;&#172;, PG1639; chp2&#226;&#338;&#172;, PG1787; swi6-115, SP1640; clr4- <br /> 681, SP1165. (B) Effects near mat3-M. Cells ura4&#226;&#171;&#185; gene EcoRV site centromere-distal mat3-M spotted Figure 2. &#226;&#171;&#185;, PG9; chp1&#226;&#338;&#172;, PG1454; chp2&#226;&#338;&#172;, PG1783; swi6-115, PG596; clr4-681, PG912. <br />                                              Fission Yeast Chromo-domain Proteins                                             561 <br />  <br />  <br />  <br />  <br />   Figure 5.&#226;&#8364;&#8221;Effects Chp1, Chp2, Swi6, Clr4 expression mat2-P mat3-M.   Effects mat2-P. Sporulated <br /> colonies stable mat1-M strains grown MSA plates supplemented leucine, uracil, adenine exposed iodine <br /> vapors photographed. Strains row intact mat2-P cis-acting elements. Strains row 1.5-kb <br /> deletion depicted Figure 4A. mat2-P strains: &#226;&#171;&#185;, SP1124; chp1&#226;&#338;&#172;, PG1605; chp2&#226;&#338;&#172;, PG1790; swi6-115, SP1126; clr4-681, PG1032; <br /> &#226;&#338;&#172;(B-B)mat2-P strains: &#226;&#171;&#185;, SP1151; chp1&#226;&#338;&#172;, PG1639; chp2&#226;&#338;&#172;, PG1787; swi6-115; PG1640; clr4-681, SP1165. (B) Effects mat3-M. <br /> Sporulated colonies stable mat1-P strains photographed  Strains row intact mat3-M cis-acting <br /> elements. Strains row 482-bp deletion depicted Figure 4B. mat3-M strains: &#226;&#171;&#185;, PG445; chp1&#226;&#338;&#172;, PG1796; <br /> chp2&#226;&#338;&#172;, PG1799; swi6&#226;&#338;&#172;, PG1797; clr4-681, PG1594; &#226;&#338;&#172;(482)mat3-M strains: &#226;&#171;&#185;, PG1402; chp1&#226;&#338;&#172;, PG1779; chp2&#226;&#338;&#172;, PG1778; swi6&#226;&#338;&#172;, <br /> PG1781; clr4-681, PG1419. <br />  <br />  <br /> mat1 allele usually form colonies contain              proximal silencing element. qualitatively similar effect <br /> spores. particular circumstances,               observed swi6 clr4 mutant strains (Thon et al. mating types coexpressed single cells, hap-            1994, 1999; Figure 5), following deletion chp1 <br /> loid cells sporulate mating process               (Figure 5).  Chp2, Chp1, appears play <br /> referred haploid meiosis.  sporulation             role similar chromo-domain proteins Swi6 unswitchable mat1-M cells used monitor                Clr4 silencing transcription mating-type <br /> mat2-P expression sporulation unswitchable                  region. <br /> mat1-P cells used monitor mat3-M expression.                constructed mat1-Msmt-0 strains containing examined various strains presence haploid            pairwise combinations chp1&#226;&#338;&#172;::LEU2, chp2&#226;&#338;&#172;::LEU2, <br /> sporulation microscopic examination (data                   swi6-115, clr4-681 alleles. strains used <br /> shown) exposing colonies iodine vapors,                  monitor levels expression mat2-P, revealed selectively stains spores black (Bresch et al. 1968;         haploid meiosis growth limiting nitrogen. <br /> Figure 5). According assay, deletion            strains examined iodine staining micro- <br /> chp1 chp2 significantly derepressed silent cas-            scopic examination. combinations led settes. Deletion chp2,  synergistic effect        easily observable derepression mat2-P (data deletion mat2-P mat3-M centromere-           shown). consistent previous conclusion <br />  562                                             G. Thon J. Verhein-Hansen <br />  Chp1 does affect transcriptional silencing             LEU2 gene plasmid DNA integrated mating-type region indicates Chp2, Swi6,              rDNA Toda et al. (1984) purpose mapping Clr4 act single pathway.                                  rRNA genes. observed LEU2 marker <br />    Effects mating-type switching: Switching              expressed poorly strain opposite mating type regulated extremely effi-            insertion, YIp2.4-1. replicated complete  <br /> cient process homothallic strains mat1 mat2-P            dium medium lacking leucine, colonies mat3-M mating-type region, designated h90. Swi6           YIp2.4 insertion did grow evenly (Figure 6A). <br /> Clr4 required process (Egel et al. 1984;              Growth limited sectors colonies, giving <br /> Gutz Schmidt 1985; Thon Klar 1993; G.                      rise wheel-like patterns. phenotype sup- <br /> Thon  J. S. Klar, unpublished results; Ivanova               pressed deleting chp1, chp2, swi6, mutation <br /> et al. 1998). specifically, Swi6 Clr4             clr4 (Figure 6A; data shown). important choice silent cassette               introduced S. pombe ura4&#226;&#171;&#185; gene near LEU2 <br /> converts mat1 mutations swi6 clr4             gene YIp2.4 (Figure 6B). integration ob- <br /> reduce switching h90 cells, increase switching           tained swi6-115 background tested Southern <br /> cells engineered mat1 mat2-M mat3-P mating-                blots (data shown). expected, integrated <br /> type region (h09 cells; Thon Klar 1993). Iodine                ura4&#226;&#171;&#185; gene displayed tight linkage LEU2 tetrad <br /> staining microscopic examination type                dissections (0 recombinants/45 viable spores exam- <br /> asci produced colony allow estimating             ined). compared expression expression <br /> efficiency mating-type switching. High switching rates          ura4&#226;&#171;&#185; integrated random site (Allshire et al. <br /> lead homogenous mixtures P M cells                1994; Figure 6, C D) expression colonies high efficiency zygotic ascus           truncated ura4-DS/E allele. <br /> formation dark iodine staining phenotype.                 background known silencing defect, ex- hand, low switching rates lead reduced              pression ura4&#226;&#171;&#185; rDNA supported limited <br /> frequency zygotic asci speckled iodine staining          growth medium lacking uracil (Figure 6C). phenotypes. resemblance Chp2                cells form colonies FOA-containing plates (Fig- Swi6, investigated deleting chp2                ure 6C). tested FOA resistance gene affected mating-type switching h90 h09 cells,           genetic alteration ura4&#226;&#171;&#185; gene different backgrounds known affect switching,            epigenetic phenomenon isolating FOA-resistant colo- <br /> swi6-mod&#226;&#171;&#185; swi6-mod&#226;&#171;&#186; (Thon Klar 1993).                   nies 24 independent cultures determining iodine staining sporulation pheno-              rise Ura&#226;&#171;&#185; colonies types wild-type chp2-deleted strains similar           growth nonselective medium. 24 independent <br /> (data shown), indication mating-type switch-           FOA-resistant cultures, 17 displayed phenotype undis- <br /> ing occur efficiently absence Chp2. Mating-          tinguishable parental Ura&#226;&#171;&#185; strain  <br /> type switching occurs normally absence              plating selective media (Figure 6C, row 1; data Chp1 judged iodine staining phenotype                 shown). <br /> chp1-deleted h90 colonies (Doe et al. 1998; data                  chromo-domain proteins <br /> shown).                                                            Chp1, Chp2, Swi6, Clr4 participated repres- <br />    Effect swi6, clr4, chp1, chp2 transcription       sion LEU2 ura4&#226;&#171;&#185; integrated rDNA (Figure PolII genes rDNA repeats: Genes 25S,             6). Altering genes encoding proteins dele- <br /> 18S, 5.8S ribosomal RNAs present 70&#226;&#8364;&#8220;100                 tion mutation improved growth media lacking <br /> copies genome S. pombe (Hoheisel et al. 1993).           leucine uracil (Figure 6C) led accumulation genes organized large clusters            ura4&#226;&#171;&#185; transcripts (Figure 6D). Chp2, Swi6, Clr4 <br /> located ends chromosome 3 consisting                obvious effects Chp1 smaller tandem repeats 10.4-kb unit (Schaak et al. 1982;           consistently reproducible effect. ura4&#226;&#171;&#185; transcript <br /> Toda et al. 1984; Hoheisel et al. 1993). S. cerevisiae         originating rDNA size  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;Position effects rDNA repeats.   Expression S. cerevisiae LEU2 gene chp1&#226;&#171;&#185; chp1&#226;&#171;&#186; cells assayed replica. Colonies chp1&#226;&#171;&#185;   YIp2.4-1) chp1-deleted   PG1572) cells integration LEU2 rDNA obtained complete AA medium replicated medium lacking (AA-leu) containing (AA) leucine indicated. <br /> (B) Integration ura4&#226;&#171;&#185; rDNA. S. pombe ura4&#226;&#171;&#185; gene contained pUC8 vector allowed recombine homology YIp2-4 allele (Toda et al. 1984). locations BamHI (Bam), Bsr FI (Bs), HindIII (Hin) restriction <br /> sites indicated. (C) Expression LEU2 ura4&#226;&#171;&#185; assayed plating efficiency. Cells containing LEU2 ura4&#226;&#171;&#185; gene rDNA indicated mutations spotted selective plates Figure 2. 1, PG1734; 2, PG1792; 3, PG1794; 4, <br /> PG1661; 5, PG1736. (D) Northern blot analysis. RNA prepared strains displayed C probed antisense <br /> ura4 probe: 1&#226;&#8364;&#8220;5, C; 6, FY489; 7, FY340. (E) Southern blot analysis. DNA prepared cultures used D, digested HindIII, blotted, probed D. <br />  Fission Yeast Chromo-domain Proteins   563 <br />  564                                           G. Thon J. Verhein-Hansen <br />  wild-type ura4&#226;&#171;&#185; gene, indicating initiation      contrast effects transcription, swi6 termination transcription occurred normally.         chp2 affected mating-type switching differently. Swi6 determined Southern blot analysis increased        important choice silent cassette used expression ura4&#226;&#171;&#185; caused amplification         convert specific mat1 alleles, lack Swi6 causes ura4&#226;&#171;&#185; gene (Figure 6E). tested stability         characteristic switching defects switching  Ura&#226;&#171;&#186; phenotype chp1&#226;&#338;&#172;, chp2&#226;&#338;&#172;, swi6-115, clr4-         duced strains wild-type mating-type region 681 colonies formed FOA. 24 independent FOA-            increased cells swapped silent cassettes (Egel et <br /> resistant derivatives tested chp2&#226;&#338;&#172;, swi6-115,     al. 1984; Gutz Schmidt 1985; Thon Klar <br /> clr4-681 frequency reversion Ura&#226;&#171;&#185; lower       1993). chp2-deleted cells able form colonies 1 106 cells 9 tested Southern blot proved         sporulation phenotype similar wild type, indicat- lost ura4&#226;&#171;&#185; gene. 24 independent             ing  unlike Swi6, Chp2 absolutely required <br /> FOA-resistant isolates chp1-deleted cells, 13         efficient mating-type switching. explanation frequency reversion Ura4&#226;&#171;&#185; lower 1 106        different effects Swi6 Chp2 reside <br /> 11 return Ura&#226;&#171;&#185; efficiently.            degree protein contributes FOA resistance chp1-deleted cells result       structure mating-type region. Lack Chp2 mutations loss ura4&#226;&#171;&#185; gene            deleterious effect lack Swi6.  <br /> case, loss tested Southern blot),           derepression transcription caused deleting <br /> reversible silencing ura4&#226;&#171;&#185; gene. phenotype         chp2 marked caused deleting swi6 similar phenotype chp&#226;&#171;&#185; cells different        (Figures 4 5).  organization remaining phenotype chp2&#226;&#338;&#172;, swi6-115, clr4-681 cells,      chp2-deleted cells sufficient support efficient reversible silencing ura4&#226;&#171;&#185; ob-           mating-type switching, albeit sufficient complete <br /> served. conclude experiment epi-         transcriptional silencing. possibility Swi6 <br /> genetic repression ura4&#226;&#171;&#185; rDNA dependent           interacts protein required switching small extent Chp1 significantly            Chp2 does  differences swi6- Chp2, Swi6, Clr4.                                          chp2-deleted cells seen mitotic mei- <br />                                                                otic viability mutant strains. Cultures swi6 mutant <br />                                                                cells contain higher proportions dead cells                       DISCUSSION <br />                                                                wild-type cultures produce increased number    examined contribution chromo-           spored asci. chp2-deleted strains healthy <br /> domain proteins, Chp1, Chp2, Swi6, Clr4, posi-          wild type respects (data shown). <br /> tion effects transcription genome fission         Consistent phenotypes, mitotic loss mini- <br /> yeast. determined PolII-transcribed genes         chromosome increased lack Chp2 repressed placed rDNA cluster lo-           absence Swi6 (Table 2). <br /> cated right arm chromosome 3,              Effects chromosome segregation&#226;&#8364;&#8221;Epistasis analy- <br /> repression transcription dependent pres-          sis: Chromo-domain proteins association <br /> ence chromo-domain proteins.             pericentric heterochromatin eukary- shown Chp1 crucial centromeric silencing,      otes. addition repressing transcription, possi- transcriptional silencing mating-type        bly participate formation structures important <br /> region near telomeres. Conversely, major effects        chromosome function, particular kineto- Chp2, addition observed rDNA,            chore. Consistent function, S. pombe mating-type region near telomeres.          chromo-domain proteins Swi6, Clr4, Chp1  compare phenotypes reported          quired efficient chromosome segregation (Allshire <br /> mutations swi6 clr4 discuss light       et al. 1995; Ekwall et al. 1996; Bernard et al. 1998; Doe reported subcellular localization pro-         et al. 1998). Swi6 present centromeres (Ekwall et <br /> teins.                                                         al. 1995) proteins known required <br />    Chp2 vs. Swi6: chp2 swi6 encode closely        localization, Rik1 Clr4 (Ekwall et al. 1996). <br /> related known S. pombe chromo-domain proteins.          mutant allele clr4 used study, clr4-681, did ORFs size contain          increase rate chromosome loss chromo- chromo-shadow domain, structure simi-          extent previously tested allele clr4-S5 (Table 2; <br /> lar HP1 protein flies mammals.          Allshire et al. 1995; Ekwall et al. 1996). clr4-681 chp2 swi6 overlapping distinguishable         allele contains point mutation causing amino-acid <br /> functions. Altering gene alleviated centromeric         change Clr4 SET domain (G486D mutation; Iva- telomeric silencing (Allshire et al. 1995;            nova et al. 1998). interesting determine <br /> study). allowed transcription mating-type       allele clr4 mutations affecting region derepression observed near mat2-P            SET domain protein influence localization <br /> mat3-M cases increased simultaneous         Swi6 extent clr4-S5 allele. clr4- <br /> deletion cis-acting elements located near cassettes.    681 swi6-115 allele synergistic effect                                             Fission Yeast Chromo-domain Proteins                                      565 <br />  <br /> deletion chp1 (Table 2).  Swi6 Clr4          somal regions close centromeres (Nakaseko et al. act pathway different Chp1          1986; Chikashige et al. 1989). inhibition possibly <br /> participates. observed synergy reveals       reflects existence intramolecular secondary <br /> clr4-681 allele does reduce efficiency chromosome       structure compete pairing ho- <br /> segregation effect readily          mologs, steric hindrance seen wild-type background. Consistent            kinetochore. discovered significant fraction <br /> epistasis analysis, Chp1 localizes discrete    cells containing centromeric ura4&#226;&#171;&#185; insertion spots nucleus, coincide centro-        lose ura4&#226;&#171;&#185; gene mitotic growth. likely <br /> meres, manner does depend Rik1             mechanism leading loss ura4&#226;&#171;&#185; placed Clr4 (Doe et al. 1998), centromeric localiza-      centromeric repeat intramolecular gene conver- <br /> tion Swi6 depends Rik1 Clr4. expected,            sion involving untouched repeat placed symmetri- <br /> swi6-115 epistatic clr4-681 (Table 2). Deleting         cally containing ura4&#226;&#171;&#185;. interactions chp2 increased chromosome loss threefold com-             proposed responsible high  <br /> pared wild type, relatively modest effect (Ta-      gree sequence conservation centromeric  <br /> ble 2). experiments required assess         peats presence identical <span id='am-12' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> role chromosome segregation.                            changes central repeats occupying symmetrical posi- <br />    Chp1 target recognition: contrast pleiotropic     tions relative centromere 1 strains (Taka- <br /> effects Chp2, Chp1 displayed specificity centro-      hashi et al. 1992). ura4&#226;&#171;&#185; lost meric regions. DNA segment homologous centro-           insertion site located central core chromo- <br /> meric repeat mating-type region        1. central core unique sequence mat2-P mat3-M (Grewal Klar 1997).              expected target mitotic gene <br /> area, sequence similarity centromere 2 reaches        conversion. determined cen- <br /> 96% 4.3 kb DNA. Deletions mating-type           tral core altered cells having lost ura4&#226;&#171;&#185; region include 4.3-kb repeat lead partial        cnt1 integration site. <br /> derepression transcription,  ?  ?               Transcriptional silencing rDNA: Transcrip- <br /> states expression clonally inherited, defects    tional silencing cases clear function mating-type switching, covariegate            example, silencing mat2-P mat3-M <br /> defects transcription (Grewal Klar 1996; Thon           mating-type cassettes crucial yeast sexual cycle Friis 1997). ura4&#226;&#171;&#185; gene silenced intro-        yeast survival absence nitrogen source. <br /> duced repeat mating-type region              cases, transcriptional silencing  <br /> (Grewal Klar 1997) homologous                sequence phenomena, structural local- <br /> repeat near centromere 1 (Allshire et al. 1995).               ization requirements. role transcriptional silenc- Swi6 Clr4 proteins required            ing rDNA subject speculation. repression locations (Allshire et al. 1995;           Ty1 elements prototrophic markers introduced <br /> Grewal Klar 1997), showed Chp1                rDNA S. cerevisiae poorly transcribed (Bryk required silencing centromeric insertions           et al. 1997; Fritze et al. 1997; Smith Boeke 1997). <br /> ura4&#226;&#171;&#185;, silencing insertions ura4&#226;&#171;&#185;        transcription increased disruption mating-type region, mating-type genes. lack        silencing gene SIR2, ubiquitin-conjugating  effect chp1 deletion gene expression         zyme gene UBC2, reduced levels histone H2A mating-type region simply explained          H2B, mutations large number genes <br /> dundancy silencing mechanism operating          including genes involved DNA replication regula- <br /> region, occasionally masks silencing defects,      tion chromatin structure (Bryk et al. 1997; Smith cumulative effect observed following combina-           Boeke 1997; Smith et al. 1999). Repression PolII <br /> tion chp1 deletion silencing deficien-       transcription rDNA repeats product <br /> cies (Figure 5; data shown). simplest interpreta-      PolI regulation (Smith Boeke 1997), tion results Chp1 does act           rDNA control mitotic meiotic recombination <br /> mating-type region, rule           (Gottlieb Esposito 1989 references , localized action locations       life span regulation (Kennedy et al. 1997). tested. restriction repressive effects       reflect functions nucleolus distinct centromeres suggests tethered pro-       production rRNA linked constituting <br /> tein/DNA complex associated           discrete nuclear subcompartment. function 4.3-kb sequence. Chp1 exert effect            proposed recently RENT complex, nucleolar 4.3-kb repeat distance, possibly pro-       protein complex associated Sir2, sequesters <br /> teins Swi6 Clr4, spread          Cdc14 nucleolus cell cycle DNA oligomerization.                                    releases telophase (Shou et al. 1999; Straight <br />    Loss ura4&#226;&#180;&#8482; gene centromeric regions:            et al. 1999). <br /> Meiotic recombination inefficient S. pombe chromo-          transcriptional repression observed  566                                           G. Thon J. Verhein-Hansen <br />  <br /> rDNA S. pombe reminiscent S. cerevisiae.      Nurse 1996) clr4&#226;&#171;&#186; cells, Swi6 does associ- S. pombe ORF encodes protein related         ate centromeres, nucleolus <br /> S. cerevisiae Sir2 (CAB38511 cosmid SPBC16D10)          (Ekwall et al. 1996). conceive interesting investigate participation     nucleolar sites normally occupied Clr4 recog- <br /> rDNA silencing. chromo-domain proteins                nized Swi6, absence Clr4 titrated roles assayed homologs              away centromeres nucleolus. wild- <br /> S. cerevisiae. S. cerevisiae chromo-domain        type cells, Clr4 repress transcripion PolII genes <br /> proteins: member CHD family (Woodage et al.           rDNA. clr4 mutant cells, Swi6 mislocalize <br /> 1997; AC P32657) S. pombe closest relatives          sites normally occupied Clr4, fail Hrp1 (Jin et al. 1998) uncharacterized ORF (AC          exert repression Clr4. swi6 mutant cells, <br /> O14139) Esa1 (Hilfiker et al. 1996; Clarke et al.          Clr4 titrated sites normally occupied 1999). S. pombe proteins         Swi6 fail repress transcription rDNA. chromo-domain proteins described impor-           indirect effects alterations clr4 tant transcriptional repression rDNA,        possible clr4 controls transcription cluding histone deacetylase Clr3 (data shown).         genes, positively negatively (Ivanova et al. S. cerevisiae, speculate repression    1998). type model, imagine transcription growth rate regulation,       changes centromere telomere structure sterically <br /> S. pombe PolI template preferences different          affect nucleolus, possibly facilitating entrance PolII, nucleolus constitutes com-       PolII. Additional localization studies overexpres- <br /> partment special properties, suggested          sion experiments ought help distinguish ultrastructure (Leger-Sylvestre et al. 1997). regula-        models. <br /> tory function dynamic sequestering proteins             thank Olaf Nielsen gift plasmids, Robin Allshire involved mitosis envisioned S.          strains centromeric telomeric insertions ura4&#226;&#171;&#185; gene, <br /> pombe nucleolus. Clr4, protein important chromo-         Kohta Takahashi Mitsuhiro Yanagida strain containing segregation rDNA silencing, localized           YIp2.4 allele. thank Inga Sig Nielsen Stanley Brown <br />                                                                comments manuscript. reported experiments nucleolus interphase expressed plas-        supported grants Novo Nordisk Foundation mid green fluorescent protein (GFP)-fusion pro-           Danish Natural Science Research Council. <br /> tein. mitosis, fraction Clr4 pool relocal- <br /> izes mitotic spindle (S26 Sawin Nurse <br /> 1996).                                                                              LITERATURE CITED <br />    Direct indirect effects&#226;&#8364;&#8221;Localization: rDNA  <br />                                                                Aasland, R.,  F. Stewart, 1995 chromo shadow domain, <br /> sertions LEU2 ura4&#226;&#171;&#185; gene transcribed               second chromo domain heterochromatin-binding protein efficiently swi6-115 wild-type cells.          1, HP1. Nucleic Acids Res. 23: 3163&#226;&#8364;&#8220;3173. <br /> swi6-115 mutation substitutes arginine tryp-        Allshire, R., 1996 Transcriptional silencing fission yeast:                                                                    manifestation higher order chromosome structure func- <br /> tophan position 269, residue conserved                tions, pp. 443&#226;&#8364;&#8220;466 Epigenetic Mechanisms Gene Regulation, <br /> chromo-shadow domains (Allshire 1996). swi6-                   edited V. E.  Russo, R.  Martienssen  D. Riggs. <br /> 115 mutation results undetectable levels            Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. <br />                                                                Allshire, R., G. Cranston, J. R. Gosden, J. C. Maule, N. D. Hastie <br /> Swi6 protein (Ekwall et al. 1996) proba-             et al., 1987 fission yeast chromosome replicate autono- <br /> bly constitutes loss function.  observation         mously mouse cells. Cell 50: 391&#226;&#8364;&#8220;403. <br /> indicates wild-type Swi6 protein represses PolII      Allshire, R., J. P. Javerzat, N. J. Redhead G. Cranston, 1994 <br />                                                                    Position effect variegation fission yeast centromeres. Cell 76: <br /> transcription rDNA. interpretation              157&#226;&#8364;&#8220;169. <br /> repression straightforward Swi6          Allshire, R., E. R. Nimmo, K. Ekwall, J. P. Javerzat G. Cran- <br /> localized nucleolus wild-type cells (Ekwall          ston, 1995 Mutations derepressing silent centromeric domains <br />                                                                    fission yeast disrupt chromosome segregation. Genes Dev. 9: <br /> et al. 1995, 1996). possible explanation Swi6          218&#226;&#8364;&#8220;233. <br /> recognizes silences PolII-transcribed genes           Arcangioli, B.,  J. S. Klar, 1991 novel switch-activating <br /> placed context rDNA. model, local-           site (SAS1) cognate binding factor (SAP1) required                                                                    efficient mat1 switching S. pombe. EMBO J. 10: 3025&#226;&#8364;&#8220;3032. <br /> ization Swi6 nucleolus precede intro-      Beach, D. H., 1983 Cell type switching DNA transposition duction marker gene                 fission yeast. Nature 305: 682&#226;&#8364;&#8220;688. observed past experiments. Alternatively,         Bernard, P., K. Hardwick J. P. Javerzat, 1998 Fission yeast <br />                                                                    Bub1 mitotic centromere protein essential spindle influence swi6-115 mutation transcription            checkpoint preservation correct ploidy mito- rDNA indirect. absence Swi6            sis. J. Cell Biol. 143: 1775&#226;&#8364;&#8220;1787. <br /> factor normally present rDNA able repress        Bresch, C., G. Muller R. Egel, 1968 Genes involved meiosis <br />                                                                    sporulation yeast. Mol. Gen. Genet. 102: 301&#226;&#8364;&#8220;306. <br /> PolII transcription synthesized,         Bryk, M., M. Banerjee, M. Murphy, K. E. Knudsen, D. J. Garfinkel titrated away sites normally occupied Swi6,          et al., 1997 Transcriptional silencing Ty1 elements centromeres. Titration effects account            RDN1 locus yeast. Genes Dev. 11: 255&#226;&#8364;&#8220;269. <br />                                                                Cavalli, G., R. Paro, 1998 Chromo-domain proteins: linking observed phenotypes. mentioned  GFP-              chromatin structure epigenetic regulation. Curr. Opin. Cell <br /> Clr4 fusion protein localizes nucleolus (Sawin              Biol. 10: 354&#226;&#8364;&#8220;360. <br />                                                      Fission Yeast Chromo-domain Proteins                                                         567 <br /> Chikashige, Y., N. Kinoshita, Y. Nakaseko, T. Matsumoto, S. Mur-                MOZ human genes SAS gene yeast, required <br />     akami et al., 1989 Composite motifs repeat symmetry S.               dosage compensation Drosophila. EMBO J. 16: 2054&#226;&#8364;&#8220;2060. <br />     pombe centromeres: direct analysis integration NotI restric-     Hoheisel, J. D., E. Maier, R. Mott, L. McCarthy,  V. Grigoriev <br />     tion sites. Cell 57: 739&#226;&#8364;&#8220;751.                                               et al., 1993 High resolution cosmid P1 maps spanning Clarke,  S., J.  Lowell, S. J. Jacobson L. Pillus, 1999 Esa1p           14 Mb genome fission yeast S. pombe. Cell 73: 109&#226;&#8364;&#8220;120. <br />     essential histone acetyltransferase required cell cycle      Ivanova,  , M. J. Bonaduce, S. V. Ivanov  J. S. Klar, 1998 <br />     progression. Mol. Cell. Biol. 19: 2515&#226;&#8364;&#8220;2526.                                chromo SET domains Clr4 protein essential <br /> Doe, C. L., G. Wang, C. Chow, M. D. Fricker, P. B. Singh et al.,                silencing fission yeast. Nat. Genet. 19: 192&#226;&#8364;&#8220;194. <br />     1998 fission yeast chromo domain encoding gene chp1&#226;&#171;&#185;               Jin, Y. H., E. J. Yoo, Y. K. Jang, S. H. Kim, M. J. Kim et al., 1998 Isola- <br />     required chromosome segregation shows genetic                  tion characterization hrp1&#226;&#171;&#185;, new member SNF2/ <br />     interaction alpha-tubulin. Nucleic Acids Res. 26: 4222&#226;&#8364;&#8220;4229.           SWI2 gene family fission yeast Schizosaccharomyces <br /> Egel, R., H. Gutz, 1981 Gene activation copy transposition               pombe. Mol. Gen. Genet. 257: 319&#226;&#8364;&#8220;329. <br />     mating-type switching homothallic fission yeast. Curr.         Jones, J. S., L. Prakash, 1990 Yeast Saccharomyces cerevisiae se- <br />     Genet. 3: 5&#226;&#8364;&#8220;12.                                                             lectable markers pUC18 polylinkers. Yeast 6: 363&#226;&#8364;&#8220;366. <br /> Egel, R., D. H. Beach  J. S. Klar, 1984 Genes required           Kennedy, B. K., M. Gotta, D.  Sinclair, K. Mills, D. S. McNabb <br />     initiation resolution steps mating-type switching fission         et al., 1997 Redistribution silencing proteins telomeres <br />     yeast. Proc. Natl. Acad. Sci. USA 81: 3481&#226;&#8364;&#8220;3485.                            nucleolus associated extension life span S. <br /> Egel, R., M. Willer O. Nielsen, 1989 Unblocking meiotic                  cerevisiae. Cell 89: 381&#226;&#8364;&#8220;391. <br />     crossing silent mating-type cassettes fission      Klar,  J. S., M. J. Bonaduce, 1991 swi6, gene required     yeast, conditioned recessive, pleiotropic mutant rik1. Curr.         mating-type switching, prohibits meiotic recombination     Genet. 15: 407&#226;&#8364;&#8220;410.                                                         mat2-mat3 &#226;&#8364;&#339;cold spot&#226;&#8364;? fission yeast. Genetics 129: 1033&#226;&#8364;&#8220;1042. <br /> Egel, R., M. Willer, S. Kj&#195;&#166;ligrulff, J. Davey O. Nielsen, 1994         Koonin, E. V., S. Zhou J. C. Lucchesi, 1995 chromo <br />     Assessment pheromone production response fission                  superfamily: new members, duplication chromo domain <br />     yeast halo test induced sporulation. Yeast 10: 1347&#226;&#8364;&#8220;1354.           possible role delivering transcription regulators chroma- <br /> Ekwall, K., T. Ruusala, 1994 Mutations rik1, clr2, clr3              tin. Nucleic Acids Res. 23: 4229&#226;&#8364;&#8220;4233. <br />     clr4 asymmetrically derepress silent mating-type loci fission   Leger-Sylvestre, , J. Noaillac-Depeyre, M. Faubladier N. <br />     yeast. Genetics 136: 53&#226;&#8364;&#8220;64.                                                 <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="gas" datatype="xsd:string"></span>Gas</span>, 1997 Structural functional analysis nucleolus <br /> Ekwall K., J. P. Javerzat,  Lorentz, H. Schmidt, G. Cranston                  fission yeast Schizosaccharomyces pombe. Eur. J. Cell Biol. 72: <br />     et al., 1995 chromodomain protein Swi6: key component                 13&#226;&#8364;&#8220;23. <br />     fission yeast centromeres. Science 269: 1429&#226;&#8364;&#8220;1431.                  Lehrach, H., 1990 Hybridization fingerprinting genome map- <br /> Ekwall, K., E. R. Nimmo, J. P. Javerzat, B. Borgstrom, R. Egel et               ping sequencing, pp. 39&#226;&#8364;&#8220;81 Genome Analysis Volume 1: <br />     al., 1996 Mutations fission yeast silencing factors clr4&#226;&#171;&#185;            Genetic Physical Mapping, edited K. E. Davies S. M. <br />     rik1&#226;&#171;&#185; disrupt localisation chromo domain protein             Tilghman. Cold Spring Harbor Laboratory Press, Cold Spring <br />     Swi6p impair centromere function. J. Cell Sci. 109: 2637&#226;&#8364;&#8220;               Harbor, NY. <br />     2648.                                                                  Lorentz, , L. Heim H. Schmidt, 1992 switching gene <br /> Engelke, U., L. Grabowski, H. Gutz, L. Heim H. Schmidt, 1987                swi6 affects recombination gene expression mating- <br />     Molecular characterization h&#226;&#171;&#186; mutants Schizosaccharomyces             type region Schizosaccharomyces pombe. Mol. Gen. Genet. 233: <br />     pombe. Curr. Genet. 12: 535&#226;&#8364;&#8220;542.                                            436&#226;&#8364;&#8220;442. <br /> Fan, J.-B., Y. Chikashige, C. L. Smith, C. L. Niwa, M. Yanagida et         Lorentz, , K. Osterman, O. Fleck H. Schmidt, 1994 Switch- <br />     al., 1988 Construction NotI restriction map fission             ing gene swi6, involved repression silent mating-type loci <br />     yeast Schizosaccharomyces pombe genome. Nucleic Acids Res. 17:              fission yeast, encodes homologue chromatin-associated <br />                                                                                 proteins Drosophila mammals. Gene 143: 139&#226;&#8364;&#8220;143. <br />     2801&#226;&#8364;&#8220;2818. <br />                                                                            Matsumoto, T., K. Fukui, O. Niwa, N. Sugawara, J. W. Szostak et <br /> Fritze, C. E., K. Verschueren, R. Strich R. E. Esposito, 1997 <br />                                                                                 al., 1987 Identification healed terminal DNA fragments     Direct evidence SIR2 modulation chromatin structure                                                                                 linear minichromosomes Schizosaccharomyces pombe. Mol. Cell. <br />     yeast rDNA. EMBO J. 16: 6495&#226;&#8364;&#8220;6509. <br />                                                                                 Biol. 7: 4424&#226;&#8364;&#8220;4430. <br /> Gottlieb, S., R. E. Esposito, 1989 new role yeast <br />                                                                            Messmer, S.,  Franke R. Paro, 1992 Analysis functional <br />     transcriptional silencer gene, SIR2, regulation recombina- <br />                                                                                 role Polycomb chromo-domain Drosophila melanogaster. <br />     tion ribosomal DNA. Cell 56: 771&#226;&#8364;&#8220;776. <br />                                                                                 Genes Dev. 6: 1241&#226;&#8364;&#8220;1254. <br /> Grewal, S.  S.,  J. S. Klar, 1996 Chromosomal inheritance <br />                                                                            Mizukami, T., W.  Chang,  Garkavtsev, N. Kaplan, D. Lombardi <br />     epigenetic states fission yeast mitosis meiosis. <br />                                                                                 et al., 1993 13 kb resolution cosmid map 14 Mb fission <br />     Cell 86: 95&#226;&#8364;&#8220;101.                                                            yeast genome nonrandom sequence-tagged site mapping. Cell <br /> Grewal, S.  S.,  J. S. Klar, 1997 recombinationally                73: 121&#226;&#8364;&#8220;132. <br />     pressed region mat2 mat3 loci shares homology           Moreno, S.,  J. S. Klar P. Nurse, 1991 Molecular genetic <br />     centromeric repeats regulates directionality mating-type             analysis fission yeast Schizosaccharomyces pombe. Methods Enzy- <br />     switching fission yeast. Genetics 146: 1221&#226;&#8364;&#8220;1238.                        mol. 194: 795&#226;&#8364;&#8220;823. <br /> Grewal, S.  S., M. J. Bonaduce  J. S. Klar, 1998 Histone           Nakaseko, Y., Y. Adachi, S. Funahashi, O. Niwa M. Yanagida, <br />     deacetylase homologs regulate epigenetic inheritance tran-               1986 Chromosome walking shows highly homologous repeti- <br />     scriptional silencing chromosome segregation fission                 tive sequence present centromere regions fission <br />     yeast. Genetics 150: 563&#226;&#8364;&#8220;576.                                               yeast. EMBO J. 5: 1011&#226;&#8364;&#8220;1021. <br /> Grimm, C., J. Kohli, J. Murray K. Maundrell, 1988 Genetic              Nimmo, E. R., G. Cranston R. Allshire, 1994 Telomere-associ- <br />     engineering Schizosaccharomyces pombe : gene disrup-        ated chromosome breakage fission yeast results variegated <br />     tion replacement using ura4 gene selectable marker.            expression adjacent genes. EMBO J. 13: 3801&#226;&#8364;&#8220;3811. <br />     Mol. Gen. Genet. 215: 81&#226;&#8364;&#8220;86.                                           Niwa, O., T. Matsumoto, Y. Chikashige M. Yanagida, 1989 <br /> Gutz, H., 1963 Untersuchungen zur Feinstruktur der Gene ad7 und                 Characterization Schizosaccharomyces pombe minichromosome <br />     ad6 von Schizosaccharomyces pombe LIND. Habilitationsschrift, Tech-         deletion derivatives functional allocation centro- <br />     nische Universita&#194;&#168;t Berlin, Berlin.                                         mere. EMBO J. 8: 3045&#226;&#8364;&#8220;3052. <br /> Gutz, H., H. Schmidt, 1985 Switching genes Schizosaccharo-          Pak, D. T., M. Pflumm,  Chesnokov, D. W. Huang, R. Kellum et <br />     myces pombe. Curr. Genet. 9: 325&#226;&#8364;&#8220;331.                                       al., 1997 Association origin recognition complex Hanahan, D., 1983 Studies transformation Escherichia coli            heterochromatin HP1 higher eukaryotes. Cell 91: 311&#226;&#8364;&#8220;323. <br />     plasmids. J. Mol. Biol. 166: 557&#226;&#8364;&#8220;580.                                  Paro, R., D. S. Hogness, 1991 Polycomb protein shares Heyer, W. D., M. Sipiczki J. Kohli, 1986 Replicating plasmids               domain heterochromatin-associated protein Drosophila. <br />     Schizosaccharomyces pombe : improvement symmetric segrega-            Proc. Natl. Acad. Sci. USA 88: 263&#226;&#8364;&#8220;267. <br />     tion new genetic element. Mol. Cell. Biol. 6: 80&#226;&#8364;&#8220;89.              Platero, J. S., T. Hartnett J. C. Eissenberg, 1995 Functional <br /> Hilfiker, , D. Hilfiker-Kleiner,  Pannuti J. C. Lucchesi,               analysis chromo domain HP1. EMBO J. 14: 3977&#226;&#8364;&#8220;3986. <br />     1996 mof1, putative acetyltransferase gene related Tip60      Powers, J., J. C. Eissenberg, 1993 Overlapping domains  568                                                      G. Thon J. Verhein-Hansen <br />     heterochromatin associated protein HP1 mediates nuclear             Thon, G., T. Friis, 1997 Epigenetic inheritance transcrip- <br />     localization heterochromatin binding. J. Cell Biol. 120: 291&#226;&#8364;&#8220;            tional silencing switching competence fission yeast. Genet- <br />     299.                                                                         ics 145: 685&#226;&#8364;&#8220;696. <br /> Rose, M., F. Winston P. Hieter, 1990 Methods Yeast Genetics:         Thon, G.,  J. S. Klar, 1992 clr1 locus regulates expres- <br />     Laboratory Course Manual. Cold Spring Harbor Laboratory Press,             sion cryptic mating-type loci fission yeast. Genetics 131: <br />     Cold Spring Harbor, NY.                                                      287&#226;&#8364;&#8220;296. <br /> Saunders, W. S., C. Chue, M. Goebl, C. Craig, R. F. Clark et al.,           Thon, G.,  J. S. Klar, 1993 Directionality fission yeast <br />     1993 Molecular cloning human homolog Drosophila het-                 mating-type interconversion controlled location     erochromatin protein HP1 using anticentromere autoantobodies                 donor loci. Genetics 134: 1045&#226;&#8364;&#8220;1054. <br />     anti-chromo specificity. J. Cell Sci. 104: 573&#226;&#8364;&#8220;582.                Thon, G.,  Cohen  J. S. Klar, 1994 additional linkage <br /> Sawin, K. E., P. Nurse, 1996 Identification fission yeast nu-             groups repress transcription meiotic recombination     clear markers using random polypeptide fusions green fluo-              mating-type region Schizosaccharomyces p <br /> </body></html>