<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1988 Genetics Society America <br />  <br />  <br />  <br /> Fine-Structure Geneticsof ama-1, Essential Gene Encoding Amanitin- <br />      Binding Subunit RNA Polymerase I1 Caenorhabditis elegans <br />  <br />                                          Anne M. E. Bullerjahn DonaldL. Riddle <br />                              Division Biological Sciences, Universityof Missouri, Columbia, Missouri 65211 <br />                                                   Manuscript received March 18, 1988 <br />                                                   Revised copy accepted June 22, 1988 <br />  <br />                                                               ABSTRACT <br />                  fine-structure genetic map constructed ama-1 N essential gene Caenorhabditis <br />               elegans encoding amanitin-bindingsubunit RNA polymerase11. Sixteen EMS-induced recessive- <br />               lethal mutations positioned gene determining intragenic recombination <br />               frequencies m118, mutation confers dominant resistance amanitin. 16 mutants, <br />               isolated ama-l(mll8)background, include 13 early larval lethals               mid-larval lethals, 2 5 &quot; . mutants exhibit temperature-dependence severity               phenotype. Intragenic recombination betweenthe lethal siteand theparental resistance mutation               detected means resistance amanitin. Recombinants detected frequencies low 2 <br />               x         segregation closely linkedflanking markers, unc-17 unc-5, revealed               lethal mutation left right m l l 8 . adding distances betweenthe extreme <br />               left right mutations, ama-1 gene estimated 0.01 1 map unit long, m l l 8 positioned <br />               0.004 map unit left lethalmutation. order thelethal mutationswith respect                viable heteroallelicstrains constructed using freeduplication, mDpl[unc-l7(e113)dpy- <br />               13(+)ama-1(+)]. heteroallelic strains weresensitive amanitin, recombinationevents <br />               lethal mutations werespecifically selected means dominant amanitin resistance <br />               encoded recombinant chromosome. segregation outside markers revealed left-right <br />               order lethal mutations. position mutations gene nonrandom. Functional <br />               domains ama-1 gene indicated various lethal phenotypesare discussed. <br />  <br />  <br />                                                           1987), Drosophila (MORTINand KAUFMAN1984) T      HEenzymeRNA          polymerase I1 synthesizes <br />         mRNA eukaryotic cells. DNA-dependent RNA polymerases,                                                           Chinese hamsterovary cells (INCLES 1978). number <br />                                                           nonconditional lethal polymerase mutants Dro- distinguished onthe basis function,subunit          sophila <br />                                                                                                                                                                                                      characterized                     biochemically <br /> composition, sensitivity fungal toxin,       (GREENLEAF     et al. 1980). C. elegans, amanitin- <br /> amanitin (reviewed ROEDER1976). nema-           binding subunit encoded ama-1 N.Worms car- <br /> tode Caeno-rhabditis elegans species, rying certain dominant mutations     gene enzyme hasa complex subunit composition,        100-foldincreaseinresistancetoamanitin            vivo <br /> complicates vitro studies (SANFORD,PRENCER         (ROCALSKI,    BULLERJAHN      RIDDLE1988), andlike- <br /> GOLOMB1985; SENTENAC        1985). genetic    wise, RNA polymerase I1 purified animals <br /> analysis RNA polymerase I1 useful         shows 100-fold increase resistance amanitin understanding functional consequencesof various       vitro (SANFORD,    GOLOMBand RIDDLE1983). <br /> alterations subunit structure. Mutational studies      nematode,C. elegans, particularly suited organisms focused amanitin-bindingfor large-scale genetic selections required <br /> subunit, hasbeenshownto             large    detectionof rare mutantsor recombinants. (210,000 D) subunit RNA polymerase I1 (GREEN-          addition,thereare       manyduplications,deficiencies <br /> LEAF 1983; BARTOLOMEI       CORDEN1987).        visible markers available aid genetic mapping r e amanitin-resistant mutants cultured cell (EDCLEYand RIDDLE1987). primary reproduc- <br /> lines (CHAN,WHITMOREand SIMINOVITCH            1972;   tive state self-fertilizing hermaphrodite. MERS,PEARSON INCLES 1975; AMATIet al. 1975;           able generate large populations self- <br /> WULFand BAUTZ1976) amanitin-resistant          ing, property especially useful forfine-structure <br /> mutants Drosophila (GREENLEAF       et al. 1979)   mapping C.elegans, fine-structuremaps C. elegans (SANFORD,GOLOMBand RIDDLE 1983).               nonessentialgenes, unc-22 (MOERMANand BAILLIE  temperature-sensitive lethal mutations    1979; ROCALSKIa n d BAILLIE1985), unc-54 (WATER- <br /> affecting large subunit    RNA polymerase I1       STON, SMITHand MOERMAN            1982), andunc-15 (ROSE yeast (HIMMELFARB,SIMPSONand FRIESEN                 BAILLIE1980) generated. case <br /> Genetics 1 2 0 423-434 (October, 1988) <br />  424                                        M. E. Bullerjahn D. L. Riddle <br />  unc-54, encodes myosin heavy chain,                                              TABLE 1 <br /> physical map mutant sites beeh elucidated                  Mutant phenotypes mapped ama-1 lethal alleles <br /> (DIBBet al. 1985). <br />     large subunit RNA polymerase I1 presum-                                              Lethal phenotype <br /> ably interacts withseveralsmallersubunits,                Allele&quot;                       20&quot;                              25&quot; <br /> thought function DNA binding (ALLISONet al.                m252                     Embryonic                     L1 <br />  1985).  genetic analysis subunit            m398                         L1                        L1 <br /> reveal numerous insights structure-function              m370                         L1                        L1 <br /> lationships important enzyme.               m365                         L2                        L1 <br />                                                                  m397                         L1                        L1 <br /> end, than30 mutant alleles ama-1                                                                  m235      Mid-larval maternal-effect embryonicb Mid-larval <br /> obtained, including amanitin resistant n d lethal al-          m368                    Sterile adult              Early larval <br /> leles (ROGALSKI,    BULLERJAHN    RIDDLE1988),           m396            Maternal-effect embryonicb         Mid-larval wild-type ama-1 gene beenclonedand se-                   m236                    Sterile adult              Early larval <br /> quenced (D. BIRDand D. RIDDLE,unpublished data).                 m332                         L1                        L1 <br />                                                                  m364                         L1                        L1 <br /> T h e fine-structure map used guide                                                                  m22                        LI                        L1 <br /> DNA sequence analysis mutant genes.                           m367                         L1                        L1 <br />     lethal ama-1 mutations induced            m369             Mid-larval sterile adult       Mid-larval <br /> ethyl methanesulfonate (EMS) chromosome car-                m328                         L1                        L1 <br /> ryingthedominantamanitin-resistance           allele ama-        m329                         L1                        L1 <br />                                                                  m335      Mid-larval maternal-effect embryonicb Mid-larval <br /> l ( m l l 8  theclosely linked, semidominant dumpy <br /> (short body) marker, dpy-13, 0.05 map unit               exception m335, wasinduced ama m322), <br />  left ama-1 (ROGALSKI       RIDDLE 1988).         mutations m - l ( m l l 8 ) background, listed                                                                left-right map order. <br /> Lethal mutations detected semi-Dpy <br />                                                   (dpy-          Adults lay eggs hatch. <br /> 13 ama ++) F1 hermaphrodites failed pro- <br /> duce viable, fertile Dpy progeny. semi-Dpy             vidually production resistant progenyin microtiter <br /> animals carried lethal mutationsclosely linked dpy-         wells containing 50 pl S medium 1% OP50 40 <br /> 13, semi-Dpy self-progeny tested            pg/ml amanitin (ROGALSKI           RIDDLE1988). <br />                                                                   Genetic markers: genetic nomenclature used loss dominant resistance amanitin    (ROCALSKI <br />                                                                described HORVITZ           et al. (1979), following RIDDLE1988; ROGALSKI,BULLERJAHN              RID-      additions. Doubly mutant genes named according DLE 1988). Loss amanitin resistance indicated          mutation, e.g., ama-l(ml18m369) genotypes lethal mutation ama-1 gene. Homozy-             chromosomalduplications givenin brackets, e.g., <br /> gous lethalama-1 mutants exhibit phenotypes ranging            mDpl[unc-l7(ell3) ama +)] mutant strains  <br />                                                                rived wild-type N 2 strain (BRENNER1974). Muta- nonconditional arrest early development             tions usedinthisstudyinclude                  unc-l7(e113) N , unc- <br /> temperature-sensitive sterility.                               17(e245) N , dpy-l3(e184) IV, unc-5(e53) IV, unc-8(e15) IV, <br />     study, fine-structure genetic map       m - l ( m l l 8 )N , ama-l(m322) IV, lethal ama-1 al- <br /> constructed. Sixteen lethal mutations             leles listed inTable l . <br /> dered respect resistance mutation, m l l 8 ,          isolation characterization ama-1 alleles <br />                                                                mDpl havebeendescribed                   (ROGALSKI   RIDDLE <br /> ninemutationshavebeen           ordered withrespect         1988; ROGALSKI,           BULLERJAHN   RIDDLE   1988).Mutations  recombination distances le-            convey larval lethal sterile adult (genetically lethal) <br /> thalsiteshave      determined usingnovel proce-           phenotypes classed lethals. exception dures based selection rare amanitin-resistant            m221 m335, lethal mutations induced recombinants amanitin-sensitive nonrecombi-              m - l ( m l l 8 ) allele chromosomemarked dpy- <br />                                                                13(e184),providing common genetic backgroundfor vir- <br /> nants. T h e positive selection growth amanitin         tually crosses. 772,221 mutation induced permitted screening animals detect            m - l ( m l l 8 ) allele chromosome marked unc-8, <br /> recombinationinintervalsas        small 2 X        map      m335 mutation induced ama-l(m322) <br /> units, corresponding toless 200 bp DNA.                allele. m l 1 8 m322 mutations confer dominant <br />                                                                resistance amanitin (SANFORD,          GOLOMB   RIDDLE1983; <br />             MATERIALS METHODS                              ROGALSKI,          BULLERJAHNand RIDDLE 1988). dpy- <br />                                                                13(e184) unc-8(e15) alleles semidominant;    Culture conditions: Nematodes grownon NG agar          mutations recessive. <br /> plateswith OP50 strain Escherichia coli (BRENNER           Strainconstructions: maplethal ama-1 mutations <br /> 1974). Amanitin-resistant wormswereselectedinliquid            relative m118 mutation, strains genotype + dpy-13 <br /> cultures containing 1 ml 2.5% (w/w)suspensionof           ama-l(ml18mx)+/unc-l7                ++ unc-5 constructed, <br /> OP50 S medium (SULSTON      BRENNER1974) 20         mx lethalmutation m - l ( m l l 8 ) gene. <br /> pg/mlamanitin  amanitin, Sigma).Dueto expense            lethal mutants maintained heterozygotes toxicity amanitin, liquid cultures smallest      balanced nT1, reciprocal translocation practicalvolumewereused.Populationdensities          ranged    chromosomes N V (FERGUSON                  HORVITZ 1985). approximately 10,000 150,000 larvae/ml. Herma-         Balanced strains crossed withN2 males, semi-Dpy <br /> phrodites grew liquid cultures tested indi-   malecross-progenywerematedwith                    unc-17unc-5  <br />                                                         Fine-Structure ama-1 Map                                                     425 <br /> maphrodites. double unc animals mated poorly,                                    + <br />                                                                         mDpl[unc-l7]; dpy-13 ama ml18mx) unc-5/unc-l7 semi-Dpyhermaphrodite cross-progeny picked                                          +. <br />                                                                         dpy-13 ama-l(mll8my) Hermaphrodites lattergen- L4 larvae propagated generation.                   otype segregated semi-Dpy Unc, Wild progeny, maphrodite producing unc-17unc-5                  self-progeny      kept. Complementation tests unc-5/+ firmed complementation tests unc-l7/+ unc-5/                unc-l7/+ males performed insure recombi- <br /> +   males) used source 20 hermaphrodites            nation outside markers occurred. unc-5 establish populations screened recombinants                unc-17 mutations carried strain, onesemi- <br /> described                                                         Dpy hermaphrodite placed 60 large (100 X <br />     map lethal mutations relative  free            15 mm) Petri plates begin growing large populations chromosomal duplication, m D p l ( N ; f )  used construct       mapping experiments, described  <br /> viable heteroallelic strains genotype mDpl[unc-l7(el13)                 Mapping lethal mutations relative m118: follow- <br /> dpy-13(+)ama-l(+)unc-5(+)];unc-lT(e245)dpy-13ama-                       ing procedure used mapping lethal ama-1 mutations <br /> l(ml18my)   +/+        dpy-13 ama-l(ml18mx) unc-5 mx          example, m328) relative resistance mutation, <br /> lethal ama-1 mutation believed right                                                                 + <br />                                                                         m118. hermaphrodite (Po genotype dpy-13 ama- <br /> lethal  based position respect m118.              l(m118m328) +/unc-17      ++       unc-5 placed 20 <br /> duplication marked unc-l7(e113) cross                large (100 X 15 mm) plates lawns OP50 incu- <br /> demonstrated recombination duplication                  bated 20&quot; 1 week  generations). bac- normal chromosome N (ROGALSKIand RIDDLE1988).                       teria consumed, worms washed unc-17-marked duplication permitted identification               large (150 x 18 mm) sterile culture tubes M9 buffer <br /> self-progeny homoallelic              unc-17dpy-13ama-          (BRENNER1974). Populations plates com- <br /> l(ml18my)    +        chromosome Unc phenotype             bined tube,containing 5 ml M9 buffer, . duplication marked unc-17 unc-5                 tubes shaken 20&quot; 3 days accumulate L1 apreferable balancer heteroallelic                  larvae. larvae (Fs generation) tube stocks, duplication available.                       separated gravity adults      Construction heteroallelic lethal strains involved           dead), divided small (1 00 X 13 mm) sterile culture <br /> eral steps.  hermaphrodites lethal ama mutations           tubes 1 mlof 2.5% (w/w) OP50 S medium 20 <br /> balanced nT1 mated N2 males. Semi-Dpy male                 pg/ml amanitin, shaken 20&quot; 4 days. Selections <br /> cross-progeny mated unc-17 unc-5                  performed way L1 L2 stages <br /> hermaphrodites, semi-Dpy hermaphrodite cross-prog-                  sensitive amanitin (SANFORD,         GOLOMBand <br /> eny pickedas L4 larvae. selfing,                    RIDDLE1983). typical selection experiment included <br /> maphrodites segregated homozygousUncs  low                     independent populations, consisting frequency, semi-DpyUnc recombinants. recombi-                     pooled progeny semi-Dpy Pohermaphrodites. nants maintained heterozygous stocks genotype                4 days, adults grown tubes containing aman- <br /> dpy-13 ama-1 unc5/++ unc-5, unc-17 dpyl3 ama-llunc-                  itin retested microtiter wells containing solution <br />  17 ++,     crossed Wild males ofgenotype            1% OP50 S medium 40 pg/ml amanitin. mDpl[unc-I7];unc-17 dpy-13/++. stock carried unc- <br />                                                                         genotypes resistant progeny confirmed 5 , semi-Dpy cross-progeny picked asL4 larvae.                                                                         complementation tests unc-5/+ unc-l7/+ males. stock carried unc-17, semi-Dpy Unc cross-progeny picked. case stocks carrying unc-5, semi-            mapping temperature-sensitive mutations (m235, <br /> Dpy progeny different genotypes: dpy-13                m368, m396, m236 m369), selection amanitin ama-l(ml18mx) unc-5/+++, unc-17 dpy-13           +/++       unc-5,   verification resistant worms microtiter wells  <br /> mDpl[unc-171; unc-17 dpy-13 ++/+ dpy-13 ama-l(mll8mx)                   formed 25&quot;, mutants grow adult <br /> unc-5. desired hermaphrodites lattergenotype               stage 20&quot; amanitin. <br /> segregated Dpy Dpy Unc progeny, Unc-5                       determine numberof animals population, <br /> Wild progeny. Semi-Dpy hermaphrodites segregated                   5-pI samples taken small culture tube appropriate progeny classes selfeduntil true-                diluted 45 pl M9 buffer. 5-pl samples taken <br /> breeding semi-Dpy stock genotype mDpl[unc-17]; dpy-     +            dilution grown small (60 X 15 mm) Petri <br />                                                                         plates OP50. 3 days 20&quot;  2 days 25&quot;), <br /> 13 m - l ( m l l 8 m x )unc-5 established. <br />     case stocks carrying unc-17, semi-Dpy Unc             adults plate counted. counts  progeny different genotypes: unc-17 dpy-               8 1 adults plate) included heterozygotes 13 ama-l(mll8my)/unc-17       ++        (selfed), unc-17 dpy-lj/unc-    unc-17 unc-5 segregants grew slowly. <br /> 17  +,    mDpl[unc-l7];unc-17 dpy-13 +/uric-1 7 dpy-13 ama-             recombinant phenotypes depended relative ( m l l 8 m y ) . genotype desired,             positions m l 8 mx. mx right m118, <br /> maphrodites genotype recognized                recombinant chromosome restored amanitin <br /> segregated Dpy Unc, homozygous Unc-17 progeny.                  resistance dpy-13 m - l ( m l l 8 ) unc-5,  semi-Dpy Unc hermaphrodites segregated                  resistant recombinants Dpy semi-Dpy <br /> appropriate progeny classes selfed true-breed-             Unc-5 phenotype (Figure la). mx left ing, semi-Dpy Unc stock genotype mDpl[unc-l7];unc-17                 m l l 8 , recombinant chromosome dpy-13 ama-l(mll8my established.                                    unc-17 ama-l(ml18), resistant recombinants     final step construction heteroallelic lethal          semi-Dpy Unc-17 phenotype (Figure 1b). <br /> strains, hermaphrodites genotype mDpl[unc-17];+ dpy-                    strain constructed map m221 relative m118 13 m - l ( m l l 8 m x ) unc-5 crossed N2 males,      genotype + ama-l(ml18m221) unc-8 / dpy-13         ++    Wild male progeny selected. males, genotype               semi-Dpy semi-Unc phenotype. m221 right mDpl[unc-l7];+ dpy-13 ama-l(ml18mx) unc-5/++++                     m118, recombinant chromosome crossed withmDpl[unc-l7];unc-17 dpy-13 ama ml18my)+                   m - l ( m l l 8 ) ++, resistant recombinants wouldhave <br /> semi-Dpy Unc hermaphrodites, semi-Dpy non-Unc L4                    semi-Dpy semi-Unc phenotype. m221 cross-progeny selected. progeny                  left m l l 8 , recombinant chromosome different genotypes: unc-17dpy-13ama-l(ml18my)/+++,                     havebeen dpy-13ama-l(mll8)unc-8,                resistant <br />  426                                                  M. E. Bullerjahn D. L. Riddle <br />  <br />                                                                          b <br />            LETHAL <br />                 TOTHE                       RIGHT                                     m178                                                                                                                           LEFT <br />                                                                                  LETHAL                                 ml18 <br />  <br />  <br />  <br />  <br />               +           dpy-13 <br />                                            m -1 <br />  <br />                                          m l l 8 m328          +                    +     m252 <br />                                                                                             13dpy- <br />                                                                                                              -  ma-1 <br />  <br />  <br />  <br />                                                                                                                     &quot; <br />                                                                                                                      ml18 <br />  <br />                                                                                                                                                                                                                                                           + <br />                                                                                &quot;&quot;&quot;                                                                                  unc-17           +                +             unc-5 <br />  <br />  <br />  <br />  <br />                                    $r <br />               +           dpy-13         m 1 18                unc-5            unc-17            +                 m118           + <br />                                                          ~~ <br />  <br />  <br />  <br />  <br />               +           dpy-13         m l l 8 m328          +                unc-17            +                    +         unc-5 <br />  <br />                                    DPY                                                               UNC-17 <br />  <br />  <br />  <br />  <br />                +          dpy-13         m 1 18     unc-17 unc-5                                  +                 mll8           + <br />  <br />             unc-17           +             +                   unc-5               +          dpy-13         m252 m l 1 8          + <br />                                    UNC-5                                                             SEMI-DPY <br />      FIGURE 1.-Genotypes strains used position lethal alleles relative mZZ8.   lethal mutation right m118 (e.g., <br /> m 3 2 8 ) , recombinant chromosome resulting recombination event m l 8 lethal mutation carries dpy-13, ama- <br /> l ( m l 8 ) , unc-5. Animals carrying recombinant chromosome resistant amanitin, Dpy semi-Dpy Unc-5 phenotype. . - (b) lethal mutation left mZ18 (e , <br />                                                                    - m252), recombinant chromosome carries m - l ( m l 8 ) unc-17, recombinant animals semi-Dpy Unc-17 phenotype. <br />  <br /> recombinants semi-Unc Dpy Unc semi-                       maphrodites came parent             genotype <br /> Dpy phenotype.                                                            mDpl[unc-17];unc-17dpy-13ama-l(mll8my)                     +/+ <br />                                                                                                                                        dpy-13 <br />    recombination frequency ( p ) calculated using                    ama-l(ml18mx) unc-5. selfed, hermaphrodites classes progeny: number resistant Dpy                     segregated classes homozygous progeny addition <br /> recombinants  semi-Dpy mx left m l l 8 )                 tothe desired heteroallelic class couldgiverise           number sensitive parental-type heterozygotes.                   recombinants  Table 3). homozygous class, Punnett square shows expected frequencies                     genotype mDpl[unc-17]; unc-17 dpy-13 ama-l(m1 lamy),         classes total zygotes p( 1 - p ) ( 1 - p)&apos;               semi-DpyUnc phenotype waseasily distinguished <br /> respectively, p 2 negligible.  define R                heteroallelic semi-Dpy class.  second <br />                                              + ratio: resistant Dpy/resistant Dpy parentals, R =                   homozygous class, genotype mDpl[unc-l7]; dpy-13 ama- <br />                       + <br /> p(1 - p ) / p(l - p ) (1 - p)&apos;. solved p = R .                l ( m l 1 8 ) u n c - 5was <br />                                                                                                     , semi-Dpy distinguished map unit corresponds 1O O p .                                         heteroallelic classby appearance. Consequently, <br />    Mapping lethal mutations relative                       60 plates screened,    days, plates <br /> procedure mapping lethal ama-1 mutations relative                     containing semi-Dpy Unc progeny (usually 35-40% essentially sameas mapping                       total plates) discarded. <br /> mutations relative m118,                         Using orderof lethal alleles predicted factor <br /> larger scale. semi-Dpy Pohermaphrodite placed                     distances mutations m l l 8 , &quot;rightward&quot; lethal 60 large plates lawns OP50. Po                       (mx) linked unc-5, &quot;leftward allele                                                            Fine-Structure ama-1 Map                                                       427 <br />                                                                        b <br />          unc-17        +                     +                   +               unc-17      +                   +                     + <br /> mDp 1                                                                   mDp 1 <br />  <br />                                                                                                      c                       -         + <br />                                                                                  unc-17     dpy-73             m398 m178 <br />                                                                           N <br />                                                                                 &quot;&quot;&quot;1- <br />                                                                                                               L&quot;&quot;&quot;&quot; <br />                                                                           IV <br />            +        dpy-13    L       ml18       m329   1     unc-5                     +   dpy-13 <br />                                                                                                      -m25.2          mil8 <br />                                                                                                                              L <br />                                                                                                                                       unc-5 <br />  <br />  <br />  <br />  <br />                                                          l +                     unc-17     dpy-73   r               mll8     1       unc-5 <br />  <br />  <br />  <br />  <br />             +        dpy-13       L   mll8       m329   1      unc-5                    +   dpy-13   LM52            mll8           unc-5 <br />  <br />  <br />                                       DPY                                                                DPY UNC <br />    FIGURE2,&quot;Genotypes strains used position lethal alleles relative  Viable heteroallelic strains constructed using duplication, mDpl[unc-I7(el13)].Using information obtained mapping lethal mutations relative m l 1 8 , chromosome carrying rightward lethal mutation mx (e.g., m329) marked unc-5, chromosome carrying leftward lethal mutation, (e.g., m328), marked unc-17.   order lethal mutations predicted mapping relative m118, recombination lethal mutations resulted chromosome           + d p y - 3 ama-1(m1l8) +. Recombinants carrying chromosome resistant amanitin Dpy. (b) order lethal mutations reversed, recombination resulted recombinant chromosome unc-17 dpy-13 ama-l(m118) unc-5. Resistant recombinants carrying chromosome Dpy Unc. <br />  <br /> linked unc-17, recombinant chromosome                alleles positioned relative  restored amanitin    resistance        carried dpy-Z3 ama-           strategiesutilized selection recombinants l(m118). Resistant recombinants Dpy, semi-Dpy carried mDpl, Dpys                  means amanitin resistance. thefirst strategy, semi-Dpys segregate Dpy Uncs (Figure 2a).               ama + parental animals unableto make func- <br /> order lethal alleles reversedfrom predicted              tional amanitin-resistant RNA polymerase 11, earlier mapping experiments, recombinant chro-                  presence lethal mutation resistance <br /> mosome carry unc-17 unc-5 dpy-                 gene. functional large subunit ani- <br /> 13 m - l ( m l l 8 ) shown Figure 2b. Recombinant <br /> animals Dpy Unc, semi-Dpy, semi-Dpy Unc                   mals product wild-type (sensitive) allele <br /> phenotype, semi-Dpys semi-Dpy Uncs                  ama  selecting amanitin-resistant worms, segregate Dpy Uncs. crosses calculated               recombinationeventsbetweenthelethalmutations <br /> recombination frequencies based resistant Dpy                andthe resistancemutation, m118, weredetected <br />  Dpy Unc order mx -  sensitive parental             (Figure 1). second strategy, selected pairs       classes. respective frequenciesamong total zygotes predicted 54 p( 1 - p ) 54 (1 - p)&apos;. defined           lethal mutations placed trans, strain  <br />                               + <br /> R ratio, Dpy / Dpy parental. Wherep&apos; negligible,               bility conveyed wild-type, ama-l(+), allele <br /> R = p shown  predicted frequencies assume                   carried free duplication, mDpZ[unc-l7(e113)] <br /> equal viabilities ofthe recombinants mDp1               (ROGALSKIand RIDDLE1988). Briefly, mDpl dupli- parental types carry mDpl.                                   cates left arm chromosome ZV                                                                           including unc-8, carries       ama +).                            RESULTS                                       heteroallelic strains sensitive amanitin,                                                                          lethal alleles harbored resistance mutation, m118. <br />    Mapping strategies strategieswere employed                        Recombination lethal sites restored construct fine-structure map   ama                     dominant m - ( m l 8 ) allele, recombinants <br /> lethalalleles werepositionedrelativeto      m118, n                    specifically selected background aman- <br /> amanitin-resistance mutation. Second, selected lethal                    itin-sensitive animals (Figure 2). <br />  428                                          M. E. Bullerjahn <br />                                                                 D. L. Riddle <br />  <br />    Mapping lethals relative amanitin-resistance:               Dpy recombinants Unc recombinants (Table 2). 16 lethal ama-1 alleles positioned relative m l 1 8        suggests Dpy semi-Dpy recombinants listed Table 1 , adapted ROGALSKI,BUL-                grew better liquid cultures did Unc   <br /> LERJAHN RIDDLE(1988). Strains genotype              +      coordinated)recombinants,      swim dpy-13 ama-l(mll8mx) +/unc-17         ++unc-5, mx            Data thatsupported interpretation came <br /> represents lethal mutation, constructed              mapping m221, lethal mutation linked experiments. strains semi-Dpy                unc-8. Semi-Unc worms better thesemidominant nature dpy-l3(e184),               Unc worms, expected survive better sensitive amanitin. animals segregated                liquid culture. essentially equal num- unc-17unc-5 doublemutant, grew                     bers semi-Dpy semi-Unc resistant recombinants <br /> slowly. phenotype genotype resistant                                                + <br />                                                                   issuing parentsof genotype ama-l(mI18m221) <br /> recombinants indicated lethal mutation                unc-8 / dpy-I3   ++ m221 mapping experiment right     left m118 (Figure 1).           (Table 2). Based onthe assumption thatthe Unc <br /> Genotypes recombinants confirmed prog-                 recombinants mappingexperimentswere <br /> eny testing complementation tests. different             represented did grow  <br /> strain used mapping m221 allele             map distances  form 3 2 2 ) calculated based linked semi-dominant unc-8 marker,              solely numbers semi-Dpy Dpy recombinants, dpy-13  MATERIALS METHODS).                    using formula derived MATERIALS METH- <br /> exception involved lethal mutation, m335,               ODS. linked amanitin resistance mutation m322,                  cross determined distance m l l 8 (ROGALSKI, BULLERJAHN     RIDDLE1988).            m252 m118, total 40 semi-Dpy  m335 mapped relative m322                recombinants (Table 2). Thirty distributed <br /> manner lethal mutations mapped                  populations, popu- <br /> relative m118, position determined             lations contained zero recombi- respect lethal mutations.                       nantseach. case, assumed thatthe <br />    mutants grew sterile adults 20&quot;,             amanitin-resistant animals anomalous pop- arrested development early mid-larval              ulations siblings arisingfromrecombination <br /> stages grown 25&quot; (Table 1).                  events occurred earlier generation.  <br /> selection foramanitin-resistantrecombinants                   m252 cross, anomalouspopulations 25&quot; cases, larvae hatched               scored containingonly recombinant  <br /> eggs produced 20 .  temper- <br />                        O                                          totalof seven semi-Dpy recombinants distributed <br /> ature differences experiments             independent populations exper- affect comparison recombination frequencies              iment. <br /> (ROSE BAILLIE 1979).  acontrol                      mapping experiment, amanitin-resist- <br /> experiment determineif selection 25&quot;               ant animals recovered unexpected pheno- <br /> alteredthefrequency recombinants                    types flanking markers. result recovered. T o end, m329 mapped relative              eithergene     conversion doublerecombination <br /> m l I8 using selections performed both20 &quot; 25 &quot; .          events. data cannotformally distinguish <br /> Essentially results obtained                 believe excep- <br /> experiments (Table 2).                                            tional animals convertants based number <br />    results 18 mapping experiments given                animals relative number recombi- Table 2. number m - l ( m l l 8 m x ) / + heter-    nants. lethal mutations relatively far away ozygotes experiment ranged 148,000                m118, m328 m329, 843,000, mean 542,000 experiment.                 recombinantsthan apparent convertants (Table 2). <br /> total 9.6 million heterozygotes subjected              mutations closer m l l 8 , proportion  <br /> selection amanitin, 229 recombinants                  vertants relative recombinants increased (e.g., m365 <br /> used todeterminebothrecombinationfrequencies                      m332). Finally, numberof recombinants mutation order. Table 2 gives distance               reached zero, case m235, 16 different lethal mutations m l l 8             marked decrease number convertants. 90% confidence intervals (STEVENS 1942),                      relationship recombinants convertants lethal mutation left right            observed fine-structure mapping <br /> m l l 8 . combining distances leftward-           experiments C . elegans (MOERMAN BAILLIE andthe rightward lethal mutations                     1979; WATERSTON, SMITHand MOERMAN1982), m l l 8 , determined gene definedby          organisms (FOGEL MORTIMER <br /> mutant set 0.01 1 f 0.003 map unit long.                        1969; CHOVNICK, BALLANTYNE        HOLM197 1). <br />    case moreDpy semi-                 cases, recombinant genotypes best <br />                                                                           TABLE 2 <br />                                                     Mapping lethal alleles relative ama-l(mll8) <br />  <br />                                                                                                                                 Convertants dou- <br />          Allele         Heterozygotes                   Mau units           Recombinants <br />                                                                                  Left/Rieht                                      ble recombinants <br />         m252               543,000             0.0013 (0.0006-0.002)                  L                7 Semi-Dpy&quot;               4 Wild Type&apos; <br />                                                                                                         4 Unc- 17 <br />         m398               460,000             0.0026 (0.002-0.004)                   L                12 Semi-Dpy               2 DPY <br />                                                                                                         2 Unc-I7                 1 Unc-17 <br />                                                                                                                                 21 Wild Type <br />                                                                                                                                  3 Semi-Dpy <br />         m370               57 1,000            0.003 (0.002-0.005)                    L                19 Semi-Dpy               3 Wild Type <br />                                                                                                                                  3 DPY <br />                                                                                                                                  1 Semi-Dpy Unc <br />                                                                                                                                 11 Semi-Dpy <br />         m365               8 19,000             0.0006 (0.0002-0.001)                 L                 5 Semi-Dpy               3 DPY <br />                                                                                                                                  3 Wild Type <br />                                                                                                                                  8 Semi-Dpy <br />         ,397               438,000              0.002 (0.001-0.004)                   L                 9 Semi-Dpy              20 Semi-Dpy&apos; <br />         m235               365,000              0 (0-0.0002)                                            0                        &apos;   DPY <br />         m368               843,000             0 (0-0.0001)                                             0                        4 DPY <br />         m396               35 1,000             0 (0-0.0002)                                            0                        3 Semi-Dpy <br />                                                                                                                                  2 DPY <br />                                                                                                                                  1 Dpy Unc <br />         m236               488,000              0.0002 (0.0001-0.001)                 R                 1 DPY                    1 DPY <br />                                                                                                                                  2 Semi-Dpy <br />         m332               613,000              0.0003 (0.0001-0.001)                 R                 2 Unc-5                  4 DPY <br />                                                                                                                                  6 Semi-Dpy <br />         m364               806,000              0.0012 (0.0007-0.002)                 R                10 DPY                    3 Semi-Dpyd <br />                                                                                                         1 Unc-5                  1 Wild Type <br />                                                                                                                                  1 Dpy Unc <br />         m221               732,000              0.002 (0.001-0.003)                   R                12 Semi-Unc               2 DPY <br />                                                                                                        11 Semi-Dpy               2 Unc-28 <br />          m367              382,000              0.003 (0.002-0.005)                   R                12 DPY                    1 Semi-Dpy <br />                                                                                                         1 Unc-5                  1 Dpy Unc <br />                                                                                                                                  9 Wild Type <br />          m369              328,000              0.005 (0.003-0.007)                   R                15 DPY                    2 Semi-Dpy <br />         m328               148,000              0.006 (0.003-0.01)                    R                 9 DPY                    1 WildType <br />                                                                                                         2 Unc-5 <br />         m32F               647,000              0.007 (0.005-0.009)                   R                44 DPY                   12 Semi-Dpy <br />                                                                                                        18 Unc-5                 17 Wild Type <br />                                                                                                                                  1 Dpy Unc <br />          m329&apos;             511.000              0.006 (0.005-0.008)                   R                33 DPY                    4 Semi-Dpy <br />                                                                                                                                  2 Semi-Dpy Unc <br />                                                                                                                                  1 Dpy Unc <br />          m3358             383,000                                                                      0                        2 DPY <br />                                                                                                                                     _. <br />     described text, 40 semi-Dpy recombinants actually recovered, distribution independent populations <br /> indicated tworecombination events occurred earlier generation. Ofthe 40semi-Dpy recombinants, 35 distributed populations. populations counted containing recombinant  recombinant animals <br /> (distributed independent populations) scored independent events. <br />       Wild-type animals genotype      ++                    +          +++ <br />                                                        m - l ( m l l 8 ) / unc-17               unc-5, recombinant chromosome arisen crossover interval unc-17 dpy-13, inteval lethal m118 sites ama-1. true semi-Dpy animals genotype      ++                  + + <br />                                       m - l ( m l l 8 ) / dfy-13 ama-l(mll8mx)     +. <br />     &apos; 20 semi-Dpy animals, 14 genotype           ++         m - l ( m l l 8 ) + / + dpy-13 ama-l(m397mll8)+ genotype <br />         +                         +                                 +, <br /> unc-17 m - l ( m l l 8 ) unc-5 / dpy-13 ama-l(m397m118) indicating recombination conversion ama-1 accompanied recombination unc-17 dpy-13. <br />       distribution resistant animals independent populations suggested conversion recombination events occurred earlier generation. 45 semi-Dpy, Wild, Dpy Unc animals actually recovered, 43 semi-Dpy convertants wild-type Dpy Unc animals population. anomalous population scored having event type. <br />     e Selection performed 20&quot;. <br />  <br />    &apos;Selection performed 25&quot;. <br />       m335 mutation linked amanitin-resistance mutation, m322, positioned relative sites gene. <br />  <br />                                                                              429 <br />  430                                                                    M. E. Bullerjahn D. L. Riddle <br />                                                                                          TABLE 3 <br />                                                              Mapping lethal ~ m - alleles <br />                                                                                     1       relative  <br />                                             mDpl [unc-                  mDpl [unc- <br />                                            171; + dpy-13                171; unc-17 <br />                                                 ama-1                   dpy-13 ama- <br />          Allele pair&quot;      Total Fa          ( m l 1 8l (mmxl)l 8 m y )           +&apos;   Heteroallelic                                                 Distance            m*)           worms              ~nc-5~                       (%)         Recombinants <br />                                                                                         Fa wormsd                   Convertants&apos;       L-R&apos;units)(mapalleles <br />                           1,328,000        15/60 = 0.25          657,000 (49)           503,000         3 Dpy Unc        24        m252-m398            0.0006 <br />                           1,026,000        24/60 = 0.40          284,000 (28)           445,000         1 Dpy Unc        18        rn398-m370           0.0002 <br />                           1,739,000        22/60 = 0.37          534,000 (31)           763,000         0                 1                            &#226;&#8218;&#172;0.0001 <br />                           1,166,000        17/55 = 0.31          159,000 (14)           695,000         9 DPYB           9         m252-m365            0.001 <br />                           1,077,000        21/60 = 0.35          333,000 (31)           484,000        13 DPY            7         m364-m367            0.003 <br />                           1,474,000        24/60 = 0.40          560,000 (38)           548,000         4 DPY             8        m367-m328            0.0007 <br />                           1,190,000        26/60 = 0.43          393,000 (33)           454,000         0                 2                            &#226;&#8218;&#172;0.0002 <br />                           1,385,000        18/52 = 0.35          424,000 (31)           625,000        19 DPY            14*       m328-m329            0.003 <br />  ~               ~~   ~     ~         ~~                             ~~                      ~~~ <br />  <br />  <br />  <br />  <br />      genotypes heteroallelic animals mDpl[unc-17]; unc-17 dpy-13 <br />                                                                                                                    + <br />                                                                               ama mll8my) / + dpy-13ama-l(mll8mx)unc-5. <br />      fraction frequency semi-Dpy Pohermaphrodites homoallelic dpy-13ama-l(ml18mx)unc-5chromosome, identifiedby failure segregate Unc-17 progeny. <br />    &apos; T h e total number percentage total population animals homoallelic unc-17 dpy-13 <br />                                                                                                           ama-l(mll8my)chromosome, determined scoring phenotypes animals diluted samples. animals semi-Dpy Unc phenotype. <br />      (Total worms-semi-Dpy Unc worms) X (1 - frequency dpy-13ama-l(ml18mx)unc-5homoallelic worms semi-Dpys). <br />      Convertant phenotypes: semi-Dpys segregated Dpy Uncs, Dpys segregated Dpy Uncs, semi-Dpy Uncs, Dpy Uncs. <br />    f Left-right order mutations. <br />  <br />    8 19 Dpy recombinants actually recovered, 1 1 population considered arisen single <br />  <br /> event earlier generation. <br />      &apos; <br />      28 convertants actually recovered, 15 Dpy convertants population considered arisen single event earlier generation, andwere counted  <br />  <br /> explained recombination outside markers                                                 selection true-breedingstocks  MA- <br /> occurred addition recombination conversion                                                TERIALS METHODS). T h e percentage pop ama  example, resistant animals                                              ulation homozygous unc-5 chromosome (usu- wild-type movement body shape                                                   ally 35-40%) calculated based  <br /> selected m252 mapping experiment (Table 2)                                                  centage P,s belonged class.  P,s <br /> carried             recombinant <br />                      chromosome,                    ama-                     ++                    did produce Unc-17 progeny dis- <br />                   +, ( m l J 8 ) generated                                                carded.) T h e percentage population homozy- <br /> events, recombination 3.0-mapunit                                                   gous unc-17 marked chromosome deter- <br /> interval unc-17 dpy-13, recombina-                                                 mined countingthe semi-Dpy Uncs diluted <br /> tion m252 m l l 8 . events need                                              samples populationprior treatment occurred generation,                                                amanitin. <br />                                                                                                           Heterozygous          animals genotype <br /> selection amanitin resistance applied FB                                            mDpI[unc 71;unc 7 dpy-13 ama-l(ml18my)+/+ dpy- <br /> progeny constructedheterozygotes,                                                     13 ama-l(mll8mx)unc-5 expected segregate <br /> scribed MATERIALS METHODS.                                                                  homozygous animals genotypes mDpl [unc-I71;unc- <br />     Mapping lethals relative   second                                                                            + <br />                                                                                                    17 dpy-13 ama-l(ml18my) mDpl[unc-17]; dpy-              + <br /> mapping strategy involved positioning lethal alleles                                               I3 ama-l(mll8mx) unc-5 roughly equal numbers. <br /> relative nearest neighbors. Strains                                              Usually, percentage population homozy- <br /> structed lethal mutation (mx linked                                               gous unc-I7 marked chromosome slightly dpy-I3 unc-5, lethal mutation                                                 smaller percentage population homo- <br />   linked unc-17 dpy-13 (Figure 2).                                               zygous unc-5 marked chromosome, heteroallelic strains genotype mDpI[unc 71;                                         +         difference rarelylarge(Table    3). cases <br /> dpy-13 ama-l(ml18mx)     unc-5/unc-l7 dpy-I3        ama-                                           involving m252, large fold) difference ml18my) + hada semi-Dpy non-Uncphenotype.                                                        number unc-5 homozygotes unc- <br /> Recombinants selected means resist-                                               17 homozygotes. presumably result ance amanitin.                                                                                  slowed development mop1 strains carry     left-right order neighboring lethal mutations                                           doses ama-l(ml18m252) allele. Generation time determined, distance                                                  (egg egg) slowed 30% +/m118m252/ <br /> estimated. necessary determine number                                                ml18m252 strains comparison +/+/ml18m252 heteroallelic worms experiment order                                              strains (ROGALSKI, BULLERJAHN RIDDLE 1988). <br /> estimate recombination frequencies (Table       3),                                             Infour cases, neighboring lethal mutations hermaphrodites used establish populations                                               separated, order deduced                                                   Fine-Structure ama-1 Map                                                  43  <br />  <br />                               ,m332 <br />  <br />  <br />   0 <br />            m397 <br />                        e <br />                                m236- <br />  <br />                               fl <br />                                 396 <br />                               f1360 <br />                                        -                                                   FIGURE 3.-Fine-structure      genetic <br />                                                                                            map ama-1. Mutations                                                                                            positioned relative                               fla35                                fl*g                    nearest neighbors placed    fl f139836                                                                              line, allele numbers stacked ver- <br />            $83                r&lt;\\I0                        f136&apos; ($25                     tically mutations <br />                             1                                                                         1            1          sepa- <br />                                                                                            rated. Alleles separated <br />                                                                                            m l l 8  <br />       }.00I t &quot;    -003+- <br />                         .oo12+&quot;                .m.-~.0007+                .m__1            positioned respect near- <br />                                                                                            est neighbors placed                                                                                            line 90% confidence  <br />  <br />                                +                 +                                       tervals (U           ). <br />                                                                                                               map  <br />  <br />    7\ <br /> .om6       .oO02               &quot;007- <br />                                                                                            given distances various <br />                                                                                            mutations  drawn scale) aswell <br />                                                                                            distances certain mu- <br />                                                                                            tations m118. <br />  <br />  <br />  <br />  predicted mapping lethal sites relative            Mapping lethal alleles region relative m l l 8 (Table 3). cases, m369/m328 m370/            determined    order correct. T h e <br /> m365, mutations separated,              m328 mutation given position m369 <br /> cases, m370/m398 m398/m252, order mu-                alleles separated, tations probable predicted dis-             position agreed data mapping <br /> tance m118. T h e 90% confidence intervals             m328 relative m329. T h e m367 mutation posi- data obtained mapping mutations               tioned 0.004 map unit right m l l 8 relative m118 overlap (Table 2), surpris-       position agreed data mapping <br /> ing differences.                                   m367 relative m364 m328,    Drawing map: fine-structure map ama-              90% confidenceintervaldetermined          mapping <br /> 1 shown Figure 3. general, comparisonof map            m367 relative m118. T o left m118, m370 <br /> distances Tables 2 3 goodadditivity                 mutation placed onthemap          according map. m118 mutation located                distancefrom m118 (Table 2). order &apos;/s way left end gene.          m370 m398 differed predicted map- adult-sterile mutations couldnot separated          ping mutations relative m118, m398 posi- m118. T h e adult-sterilemutations,              tioned map based recombination frequency <br /> m236 m369, maptotheright            m118. T h e          m370. order m398 m252 embryonic lethal mutation, m252, leftward           differed predicted mapping muta- <br /> mutation       early larval lethal mutationsmap             tions relative m118, m252was positioned according gene.                                            recombination data relative m398. T h e m365 <br />    data Tables 2 3 reconciled            mutation given position m370 obtain themost consistent position lethal sites         alleles separated, posi- respect respect m118             tion agreed wellwith data mapping m365 <br />                                                                 relative m252. <br /> (Figure 3). general, recombination frequencies ob- <br /> tained mapping mutations relative    m l 1 8                                                                                     DISCUSSION <br /> considered reliable. Estimation number heteroallelic worms crosses designed position             T h e ama-1 fine-structure map linear, addi- lethal mutations relative            tive, based dataobtainedfrommapping           lethal inaccurate number homoallelic               alleles relative m118, data obtained progeny segregated generations              mapping lethal alleles relative  growth prior selection recombinants couldonly            cases, recombination lethal mutations <br /> beroughlyestimated.       T o theright m118,             detected, order lethal mutations  <br /> mutations m364, m369 m329 placed                spect inferred mapaccording distances m l 1 8 .              mapping lethal alleles relative m118. total 20 <br />  432                                     M. E. Bullerjahn <br />                                                             D. L. Riddle <br />  <br />  million worms screened recombinantsin          bination smaller intervals. WATERSTON,          SMITH <br />  26 crosses used generate fine-structure map.         MOERMAN(1982)observed recombination <br />  average 11 recombinants convertants           event unc-54 mutations, s291 e1328, <br />  detected cross. resolution genetic       later shown 26 bp apart (DIBBet al. <br /> map sufficient tell mutations        1985). Intragenic recombination frequencies         small rearrangements deficiencies, prelim-        determined study. <br /> inary Southern blot analyses suggest           early larval lethal mutations map mutations mapped duplications deficien-          gene. Arrested development death L1 <br /> cies larger 50 bp (L. NAEGER, D. BIRDand D.             stage ama-1 null phenotype. Evidence  RIDDLE,unpublished data).                                   comes m D f l 0 deficiency, conveys    addition recombinants, resistant animals        L1-lethal phenotype deficiency homozygotes (Ro- recovered unexpectedcombinations                GALSKI RIDDLE1988). mutations clus- <br /> flanking markers. exceptionalprogeny               tered veryclose tothe resistance mutation, m118, <br /> likely arisenfromgene         conversion events,     maps approximately &apos;/s way based ontheirfrequency.          FOGELand MORTIMER           gene dpy-I3 (left) end. Molecular analysis <br /> (1969) studied gene conversion arg4 locus          shown dpy-I3 end 5&apos; end yeast andfoundthat         distance       gene (D. BIRDand D. RIDDLE,unpublished data). alleles decreased, number convertants relative        position resistance mutation agrees recombinants increased. CHOVNICK,       BALLANTYNE        position resistance mutation mouse HOLM(197 1) observedthe phenomenon               RNA polymerase I1 large subunit gene (BARTOLOMEI study rosy locus Drosophila.         CORDEN1987).  distance alleles              m252 embryonic lethal mutation po- <br /> extremely small, recombinants            sitioned left end map. data suggest <br /> vertants recovered. results similar.            allele encodes product interferes    general additivity map observed        normal enzyme function (ROGALSKI,           BULLERJAHN    results crosses utilized mDpI[unc 71 com-      RIDDLE1988). T h e m l 1 8 m 2 5 2 allele unique pared results crosses did involve       conveys anembryonic lethal phenotype at20&#194;&#176;, duplication. indicates duplication         null phenotype ofama L1-lethal. itself does significantly affect recombination fre-      m l 1 8 m 2 5 2 product apparently interfereswith ma- <br /> quencies ama  noted,           ternally encoded wild-type RNA polymerase I1 m o p 1 recombines normal chromosome           sufficient completion em- <br /> IV frequency 20-fold reduced            bryonic development. Furthermore, mDp [ama- <br /> observed normal homologs (ROGALSKI               +)] strain genotype m 2 5 2 / m 2 5 2 / + slower RIDDLE1988). recombinants detected            developing 20 O m252/+ m 2 5 2 / + / + strains, <br /> study readily attributed torecombination         suggesting predominance m252 product <br /> gene conversion events involving m D p l .                   antagonizes function wild-type product.    ama-1 gene cloned C. elegans,           interference reduced 25&quot;,        indicating 8.7 kb long (D. BIRDand D. RIDDLE,           abnormalfunction thermolabile. does <br /> unpublished data). Given genetic size        encode altered, deleterious product, mu- <br /> gene 0.01 1 map unit, estimate DNA            tation islikely theproteincoding se- <br /> map unit ratio 790 kb/map unit, assuming            quence gene.               lethal <br /> thatthe map spans theentiregene.           T h e average     mutations affect sequences upstream         coding DNA map unit C. elegans 266 kb,          region. amountincreases 900 kb/map unitin               m335 sterile mutation allelic LGIII gene cluster decrease recombi-         ama-1 lethals. linked m322 resistance allele <br /> nation frequency cluster (GREENWALD         et    ama-1 m l l 8 . m335 mutation al. 1987). ama-1 gene edgeof LGIV          notbeseparatedfrom           m322, just sterile <br /> genecluster (EDGLEYand RIDDLE1987),                 mutations separated m l 8 . consistent higher average kb/map unit            mapping data support theconclusion value.                                                       dominant amanitin-resistance mutations, m 1I8    specific, positive selection recombinants       m322, allelic (ROGALSKI,           BULLERJAHN <br /> means amanitin-resistance allowed detection         RIDDLE1988). possible m l l 8 m322 <br /> combination interval small 0.0002 map            mutations site close. <br /> unit (betweenm370 m398, m l 1 8             T h e amanitin binding site RNA polymerase I1 m 2 3 6 ) , corresponds 170 bp. Screening     highly conserved divergent species (ROEDER <br /> larger populations allow detection recom-          1976). themouse, amanitin-resistance mutation <br />                                                        Fine-Structure ama-1 Map                                                    433 <br />  region protein strongly homolo-                                   LITERATURE CITED <br /> gous yeast protein, adjacent region <br />                                                                      AHEARN,  J. M., M. S. BARTOLOMEI,  M. L. WEST,L. S. CISEKand J. highly conserved E. coli (BARTOLOMEI                   L. CORDEN,    1987 Cloning sequence analysis mouse <br /> CORDEN1987; AHEARNet al. 1987).  struc-                       genomic locus encoding largest subunit RNA polymerase <br /> ture ofthisregion       thoughttobe     essential                11. J. Biol. Chem. 262: 10695-10705. <br /> polymerase function. interesting outof               ALLISON,L., M. MOYLE, M. SHALESand C. J. INGLES, sterile mutations map close m118. Aman-                    1985 Extensive homology largest subunits eu- <br />                                                                         karyotic prokaryotic RNA polymerases. Cell 42: 599-610. <br /> itin interferes elongation reaction tran-                AMATI,P., F. BLASI,U.DI PORZIO  RICCIOand C. TRABONI, <br /> scription (COCHET-MEILHAC       CHAMBON1974),                       1975 Hamster amanitine-resistant RNA polymerase I1 able RNA polymerase I1 isolated amanitin-resist-                    transcribe polyoma virus genome somatic cell hybrids. <br /> ant Drosophila reduced elongation ratein vitro                   Proc. Natl. Acad. Sci. USA 72: 753-757. <br /> (COULTERand GREENLEAF          1985). mutations                BARTOLOMEI,    M. S., J. L. CORDEN,1987 Localization                                                                         amanitin resistance mutation gene encoding largest slow elongation result RNA polym-                      subunit mouse RNA polymerase 11. Mol. Cell. Biol. 7: 586- <br /> erase I1 satisfy demand forhigh tran-                    594. <br /> scription levels particular tissues particular              BRENNER,  S., 1974 genetics Caenorhabditiselegans. Ge- <br /> times development. mutations ex-                       netics 77: 71-94. <br /> pected result slow growth, inability              CHAN, V. L., F.G.           WHITMOREand L. SIMINOVITCH, <br />                                                                         1972 Mammalian cells withaltered forms RNA polymerase <br /> synthesize components necessary oo-                      11. Proc. Natl. Acad. Sci. USA 6 9 31 19-3123. <br /> genesis. describes phenotype sterile                 CHOVNICK,  , G. BALLANTYNE    D. HOLM,1971 Studies mutantsthatmap        close m118,                   gene conversion relationship linked exchange possible affect RNA chain elongation. Bio-                    Drosophila melanogaster. Genetics 6 9 179-209. <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> study mutants aimed obtain-                COCHET-MEILHAC,    M., P. CHAMBON,     1974 AnimalDNA- <br />                                                                         dependent RNA polymerases. 11. Mechanism inhibition <br /> ing information protein structure      activity               RNA polymerase B amatoxins. Biochem. Biophys. Acta <br /> vitro, complement genetic studies                       353: 160-184. <br /> reported M. GOLOMB,      personal communication).               COULTER,   D. E.,  L. GREENLEAF,    1985 mutation    study describesthe mappingof lethal alleles                  largest subunit RNA polymerase I1 alters RNA chain elon- essential gene, demonstrates feasibility                  gation vitro. J. Biol. Chem. 2 6 0 13190-13198. <br />                                                                      DIBB,N. J., D. M. BROWN,    J. KARN,D. G. MOERMAN,    S. L. BOLTEN <br /> using duplication construct viable heteroallelic                   R. H. WATERSTON,       1985 Sequence analysis mutations <br /> lethal strains organism. T h e ability detect                affect synthesis, assembly, enzymatic activity rare amanitin-resistantrecombinants intervals                      unc-54 myosin heavy chain Caenorhabditis elegans.J. Mol. <br /> estimated small 170 nucleotides provides                    Biol. 183: 543-551. basis high-resolution map. Mutations                  EDGLEY, M., D.L. RIDDLE,1987 Caenorhabditiselegans. pp. <br />                                                                         351-365.  Genetic Maps, Vol. 4, Edited S. J. O&#226;&#8364;&#8482;BRIEN. <br /> affect RNA polymerase I1 large subunitare useful <br />                                                                         Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. identify regions polypeptide                FERGUSON,   E. L., H. R. HORVITZ,1985 Identification important forvarious aspects enzyme structure                    characterization 22 genes affect vulval cell lineages function. T h e fine-structure map     ama-1                 nematode Caenorhabditis elegans. Genetics 1 1 0 17-72. <br /> gene provides essential information                  FOGEL,S., R. K. MORTIMER, 1969 Informational transfer                                                                         meiotic gene conversion. Proc. Natl. Acad. Sci. USA 62: 96- <br /> ducing function different domains                                                                          103. <br /> resistant subunit, guiding sequence anal-               GREENLEAF,  L., 1983 Amanitin-resistant RNA polymerase I1 <br /> yses mutantalleles. Finding point mutations                   mutations enzyme&#226;&#8364;&#8482;s largest subunit. J. Biol. Chem. 8.7-kb gene difficult sort                      258: 13403-13406. guide,andthefine-structuremap           provides              GREENLEAF,   L., L. BORSETT,P. JIAMACHELLO D. COULTER, <br />                                                                         1979 Amanitin-resistant D. melanogaster altered <br /> needed link genetic molecular analyses. <br />                                                                         RNA polymerase 11. Cell 18: 613-622. advantage mapping      physical methods              GREENLEAF,   L., J. R. WEEKS,R.  VOELKER,   S. OHNISHIand identify base-pair changes DNA (MYERS, LARIN                    B. DICKSON,   1980 Genetic biochemical characterization MANIATIS1985) thefunctionally important                     mutants RNA polymerase I1 locus D. melanogaster. <br /> alterations distinguishedfrom        possible                 Cell 21: 785-792. <br /> changes.                                                             GREENWALD,    L.,  COULSON,     J. SULSTONand J. PRIESS, <br />                                                                         1987 Correlation physical genetic maps lin- <br />                                                                         12 region Caenorhabditiselegans. NucleicAcidsRes.     15: <br />    work supported Department Health Human             2295-2307. <br /> Services grant GM36674; B. predoctoral trainee supported     HIMMELFARB,     H. J., E. M. SIMPSON      J. D. FRIESEN, GM07494. strains obtained Caenorhabditis          1987 Isolation characterization temperature-sensitive <br /> Genetics Center, supported contract NO1 RR-4-2111           RNA polymerase I1 mutants Saccharomyces cerevisiae. Mol. National Institutes Health Division Research          Cell. Biol. 7: 2155-2164. <br /> sources. thank T . ROGALSKI,  G. SMITH,M. GOLOMB,    K. NEW-      HORVITZ,H. R., S. BRENNER,     J. HODCKINand R. K. HERMAN, <br /> TON E. COE helpful discussions, R. ROSENBLUTH critical       1979 uniform genetic nomenclaturefor nematode <br /> comments manuscript, D. BIRDfor sharing unpublished              Caenorhabditis elegans. Mol. Gen. Genet. 175: 129-133. <br /> results, M. AUDSLEYfor processing manuscript.                INGLES, C. J., 1978 Temperature-sensitive RNA polymerase I1 <br />  434                                             M .D. <br />                                                       E. Bullerjahn <br />                                                                         L. Riddle <br />    mutations Chinese hamster ovary cells. Proc. Natl. Acad.        ROSE  M., D. L. BAILLIE,1980 Genetic organization    Sci. USA 75: 405-409.                                                 region unc-Z5(1), gene affecting paramyosin Cae- <br /> MOERMAN,   D. G., D. L. BAILLIE,1979 Genetic organization            norhabditis elegans. Genetics 96 639-648. <br />    Caenorhabditis elegans: fine-structure analysis unc-22   SANFORD,   T., M. GOLOMBand D. L. RIDDLE,1983 RNA polym- <br />    gene. Genetics 91: 95-103.                                            erase I1 wild typeanda-amanitin-resistantstrains        MORTIN,M. , T. C. KAUFMAN,      1984 Developmental effects         Caenorhabditis elegans. J. Biol. Chem. 2 5 8 12804-12809. <br />    atemperature-sensitiveRNApolymerase           I1 mutationin     SANFORD,   T.,J. PRENCER   M. GOLOMB,1985 Purification    Drosophila melanogaster. Dev. Biol. 103: 343-354.                     immunological analysis RNA polymerase11 Caenorhab- <br /> MYERS,R.M.,  LARINand T. MANIATIS,1985 Detectionof                     ditis elegans. J. Biol. Chem. 260: 8064-8069. <br />    single base substitutionsby ribonuclease cleavage mismatches    SENTENAC , 1985 EukaryoticRNApolymerases.CRCCrit. <br />    RNA:DNA duplexes. Science 2 3 0 1242-1246.                         Rev. Biochem. 18: 3 1-9 1 . <br /> ROEDER,R.G., 1976 EukaryoticnuclearRNApolymerases.pp.                 SOMERS,D. G., M. L.PEARSONand C. J. INCLES, 1975 Isolation <br />    285-329.  RNA Polymerase, Edited R. LOSICKand M.                characterization amanitin-resistant rat myoblast <br />    CHAMBERLIN. Cold Spring Harb <br /> </body></html>