<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 6 1995 Genetics Society America <br />  <br />  <br />  <br />             Sodium Orthovanadate-Resistant Mutants Saccharomyces cerevisiue <br />                  Defects Golgi-Mediated Protein Glycosylation, <br />                          Sporulation Detergent Resistance <br />  <br />                                Cynthia Kanik-Ennulat, Ermelinda Montalvo Norma Neff <br />              Program Molecular Biology Memorial Sloan-Kettering Cancer Center Cornell University Graduate School <br />                                           Medical Sciences, New Ymk, New York 10021 <br />                                                         Manuscript received January 3, 1995 <br />                                                       Accepted publication March 31, 1995 <br />  <br />  <br />                                                              ABSTRACT <br />                   Orthovanadate smalltoxic molecule competes biologically important oxyanion <br />                <span id='am-63' about='obo:IMR_0200102' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00009" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Orthophosphate" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200102" datatype="xsd:string"></span>orthophosphate</span>. Orthovanadate resistance arises spontaneously Saccharomycescerevisiae haploid cells <br />                mutation number genes. Mutations selected 3 mM sodium orthovanadate different <br />                degrees vanadate resistance, hygromycinsensitivity, detergent sensitivity sporulation defects. <br />                Recessive vanadate-resistant mutants belong genetic loci. mutants defective                outer chain glycosylation secreted invertase ( v n l , van2, van4, van5, van6, VAN7-116 , <br />                phenotype &quot;vor lrRG mutants. phenotypes vanadate-resistant mutants <br />                consistent alteration permeability specificity Golgi apparatus. previously <br />                published VAN1 gene product 200 <span id='am-51' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-57' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> domain 40% identity MAN9 gene <br />                product 70% identity w l gene product. Cells containing vanl-18, mnn9 (wg6                anpl mutations phenotypic similarities. VAN2 gene isolated coding region <br />                identified reported. essential gene chromosome XV translated amino acid <br />                sequence predicts unique 337 amino acid protein multiple transmembrane domains. <br />  <br />  <br />  <br /> V    ANADIUM trace element eukaryotic <br />       cells required growth (WILLSKY  1990). biologically significant molecule orthovanadate, <br />                                                                             mers (WILLSKY    et al. 1984, 1985; WILLSKY   1990). Poly- <br />                                                                             oxovanadate (IV form number higher order <br />                                                                             structures including nanoscale magnetic particles (GAT- structural similarities orthophosphate.                        TESCHI et al. 1994). particles large threedi- <br /> Oxyanions orthovanadate arsenate                          mensional molecular clusters oxometal ions toxic cells competing endogenous oxyanions                       form multiple magnetic centers. phosphate sulfate. Monomeric orthovana-                             magnetic particles investigation <span id='am-19' about='oboInOwl:date' typeof='owl:Thing'>date</span> interfere anynumber orthophosphate-                        physical pharmaceutical properties. fea- <br /> mediated processes cell ionic mimicry                      ture magnetic centers attraction mechanism (CLARKSON 1993) Ita potent vitro                         capture heavy metals. best studied mole- <br /> hibitor enzymes bind <span id='am-32' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>                        cules named VI5 [VI5          (H20 ,whichhas cap- <br /> recognize utilize high-energy phosphate residues                         tured arsenic atoms. feature orthovanadate <br />  (WILLSKY  1990).                                                           suggests secretory vacuolar network    Sodium orthovanadate potent inhibitor P-type                       lowered pH environment favor Polyoxovanadate <br /> ATPases thought energy poison                         (IV) formation potential site synthesisof added cells. vivo orthovanadate                        actual toxic compound vivo. documented numerous effects, including                            laboratories isolated vanadate-resistant increase oftyrosine phosphorylation proteins                         mutants S. cereuisiae (WILLSKY et aL 1985;KANIK-ENNU- induction transformed insulin-mimic phe-                        LAT NEFF1990; BALLOUet aL 1991). studies <br /> notype mammalian cells (WALTON     DIXON1993).                       shownthat number genes mutate tovana- <br /> Sodium orthovanadate reported RIEZMAN                           date-resistance. WILLSKY   colleagues studied ortho- <br />  (1985) increase uptake Lucifer yellow Saccharo-                   vanadate metabolism isolated complementa- <br /> myces cereuisiae.                                                           tion groups vanadate-resistant mutants. isolated <br />    important <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> property orthovana-                        vanadate-resistant mutants reported sequence <br /> date spontaneous polymerization response                       locus ( VAN1), screening yeast mutants <br /> low pH (POPE1983).Vanadium exist yeast cells                         altered protein phosphorylation, especially tyrosine number oxidation states, including higher poly-                     protein phosphorylation (KANIK-ENNULAT         NEW <br />                                                                             1990). BALLOU colleagues spontane- <br />   Corresponding author: Norma Neff, Box 86,Memorial Sloan-Ketter-           ous vanadate-resistant mutants ( IRG) noncondi- <br /> ing Cancer Center, 1275 York Ave., New York, NY 10021.                      tional defect secreted invertase glycosylation (BAL <br /> Genetics 140 933-943 (July, 1995) <br />  anik-Ennulat, <br />       C.    934                                                          E. Montalvo N. Neff <br />  <br />                     1991) wgmutations            LOU et al.                             allelic mnn              striction enzyme EcoRI dilute religation. Plasmid pNV93- <br />            mutations, isolatedby laboratory using biochemical               12 contains 3kb Hind111 fragment pNV93-6 ligated <br />                                                                                 Hind111 site yips. PlasmidpNV93-12swas         <br />            screens forviable cells defectivein outer chain          synthesis <br />                                                                                 structed digestion pNV93-12 DNA restriction  <br />             (BALLOU1990) subset mutants            (mnn7, mnn8,      zyme SnuB1, followed dilute religation. Plasmid pNV93- <br />            mnn9and mnnl0) secrete proteins             cell wall manno-      14 contains 2.2-kb EcoRI-Hind111 fragment pNV93- <br />            proteins significant deficiencies outer chainad-             6 ligated EcoRI Hind111 sites ofYCp50,            dition. orthovanadate-resistant         alleles <br />                                                                          pNV93-15 contains fragment EcoRIHindIIIdi- <br />                                                                                 gested Stratagene vector pBluescript SK( -). <br />            MNN8, MNN9and MNNlOgenes showed similar defects                         create null allele VAN2 locus, mostof open <br />            outer chain addition (BALLOUet al. 1991).                         reading frame deleted plasmid pNV93- 15 using                   Recently MNN9 MNNl genes                        SnuBl sites (Figures 3 4). TRPI+ gene isolated <br />            cloned sequenced (YIP et al. 1994).                YRpl7 using restriction enzymes leave blunt <br />            gionofsignificant        amino acididentitybetween               ends  (Ssp1and StuI) ligated deleted vector DNA. <br />                                                                                 null construction excised vector polylinker <br />            MNN9 VAN1 predicted gene products. Based o n                     sequences using restriction enzymes SalI BamHI            results laboratories, examined                 adding lithium acetate-treated cells. <br />            glycosylation secreted invertase mutants,                 Polymerasechainreactionamplification:        Diploid trans <br />            compared phenotypes               van mutants        formants van2 TRP+ null construction analyzed <br />            vrg/mnn mutants.                                             reactions containing -10&#226;&#8364;&#8482; cellequivalents genomic <br />                                                                                 DNA, 100 pm primer reagents Perkin <br />                                                                                 Elmer Cetus &#226;&#8364;&#339;GeneAmp&#226;&#8364;? kit. Thirty cycles 1&#226;&#8364;&#732; 94&#226;&#8364;?, 1&#226;&#8364;&#8482;                            MATERIALSANDMETHODS                                  50&#226;&#8364;? 2&#226;&#8364;&#732; 72&#226;&#8364;&#339; Perkin-Elmer DNA Thermal Cycler               Strains: S. cerevisiue strains used study listed  used. sequences oligonucleotides VAN2sense <br />            Table 1. haploid strains NF134,NF147, diploid             = 5&#226;&#8364;&#8482;-CAAGTTCGTCGTCAACTT-3&#226;&#8364;&#732; VAN2 antisense = 5&#226;&#8364;&#8482;- <br />            strain NF3623 S288c background (SCHATZ           et al. 1986; CACTCGAGACAGGTATCA-3&#226;&#8364;&#8482;. mutant van2-93 gene <br />            SHIHet al. 1988). NFV mutant strains derived fromboth           amplified total nucleic acid NFV293 cells  <br />            NF134 NF147.NFV220-224were                 selectedfrom vrg      scribed sequenced directly. sequence            strains (BALLOU   et al. 1991).                                      oligonucleotides  VAN2 sense = 5&#226;&#8364;?ACGCGTITCAGA- <br />               Media genetic techniques: Media growth, selection         TCCTCT-3&#226;&#8364;&#8482; VAN2 antisense = 5&#226;&#8364;&#8482;-CAATACGTTGCCTGA- <br />            <span id='am-58' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> auxotrophs sporulation wereas described             GAT-3&#226;&#8364;&#732;. <br />            ROSE et al. (1990), yeast plates containing sodium          Invertase activity gels: Active forms invertase  <br />            orthovanadate; plates weredescribed previously (KANIK-         solved  25 X 37 cm 5% polyacrylamide gels. Sample prepa- <br />            ENNULATand NEFF 1990).Yeast mating, sporulation tet-             ration, gel solutions assay conditions described <br />            rad analysis performed using standard methods (ROSE             BALLOU(1990). Gels run 4&#226;&#8364; 10 V 18-24 hr. <br />            et al. 1990). Vanadate resistance phenotypes                                                                                 experiments, cell extracts 0.1% glucose <br />            scored spotting suspensions cells disposable micro-       grown cells used. Spheroplasts (ESMON      et al. <br />            titer wells plates using 48-prong metal replicator.           1981) distinguishinternal external forms invertase. <br />               Isolation sodium orthovanadate-resistant mutantsSpon- <br />            taneous sodiumorthovanadate-resistant mutants wereiso- <br />            lated individual colonies strains NF134 NF147.                                       RESULTS <br />            small colony ( -lo6 cells) suspended 200 ml            sterile water spread YPD plate containing 3 mM                 Isolation genetic analysis sodium orthovana- <br />            sodium orthovanadate. plates incubated 26&#226;&#8364;?           dateresistant mutants: Spontaneous vanadate-resistant <br />            5 days. vanadate-resistant colonywas chosen            mutants appeared YPD+3mM sodium orthovana- <br />            plate. purification YPD plates, individual colo-      date plates frequency         primary <br />            nies retested vanadate resistance growth YPD <br />                                                                                 resistant colonies isolated retested. total            plates 14,26 35&#226;&#8364;? vanadate-resistant mutations tested <br />            behaved single mutations crosses wild-typeand        104 mutant strains haploid parents demon- <br />            mutants. phenotypes cosegregated vana-          stratedconsistentvanadate          resistance            date resistance alleles tested.                            showed leaky phenotype andwere studied  <br />               Dominance complementation tests: test domi-            temperature-sensitive vanadatedependent mu- <br />            nance recessiveness vanadate mutations, mu- <br />            tant mixed 100 ml saturated liquid YPD culture         tants  <br />            wild-type parent opposite mating type ina micro-          remaining 81 strains mated opposite <br />            titer  mating mixtures spotted SD +              wild-type haploid parent heterozygous diploids            uracil plateto select diploid cells auxotrophic complemen-        tested vanadate resistance/sensitivity.  <br />                                                           + <br />            tation. Ura- diploids purified SD uracil plates          cessive strains formed vanadate-sensitive diploids            colonies diploid tested YPD + <br />                                                                                 wild type. recessive mutants mated            vanadate plates. <br />               Plasmidconstructions: Bacterialstrains,plasmidvectors             combinatorial manner determine number                          YCp50-based yeastgenomic plasmid library           genes represented group. Thirty dominant <br />            described previously (SHIHet al. 1988; KANIK-ENNULAT             strains formed vanadate-resistant diploids wild type. <br />            NEW 1990). Yeast transformation recombinant DNA                  results complementation analysis summa- <br />            methods described (KANIK-ENNULAT          NEFF1990). <br />                                                                                 rized Table 2. collection vanadate-resistant <br />            PlasmidpNV93-9 constructed digesting pNV93-6 <br />            restriction enzyme SphI dilute religation. Plasmid          mutants similar reportedby labora- <br />            pNV93-10 constructed digesting pNV93-9with                 tories groups isolated recessive vanadate-resis- <br />                                                      Vanadate-Resistant Yeast Mutants                                        935 <br />  <br />  <br />                                                                TABLE 1 <br />                                                                strain list <br /> ber                     Strain <br />                            NF134                        MATa his4-539 lys2-801 ura3-52 <br />                            NF147                        MATa ade2-101 ura3-52 <br />                            NF3623                       MATa ade2-101 <br />                                                         -        ~     his3-200 leu2-3,112 <br />                                                                                    _       lys2-801- <br />                                                                                            &quot;        trpl-1 ura3-52 <br />                                                                                                     _                   _ <br />                                                         MATa     + his3-200 leu2-3,112          +   t q l - 1 ura3052 <br />                            NFVl1                        MATa his4539 lys2-801 ura3-52 van61 1 <br />                            NFVl4                        MATa his4539 lys2-801 ura3-52 uan2-14 <br />                            NFVl8                        MATa his4-539 lys2-801 ura3-52 uanl-18 <br />                            NFV93                        MATO ade2-101 ura3-52 uan2-93 <br />                            NFV96                        MATa ade2-101 ura3-52 van5-96 <br />                            NFV97                        MATa ade2-101 ura3-52 van497 <br />                            NFVl12                       MATa his4-539 lys2-801 ura3-52 uan5-112 <br />                            NFvl17                       MATa his4-539 lys2-801 ura3-52 van3-117 <br />                            NFvl20                       MATa his4-539 lys2-801 ura3-52 uan5-120 <br />                            NFvl12                       MATa his4-539 lys2-801 ura3-52 van4-122 <br />                            NFVl28                       MATa his4-539 4.52-801 ura3-52 van5-128 <br />                            NFVl33                       MATa his4-539 lys2-801 ura3-52 uan5-133 <br />                            NFV146                       MATa his4-539 lys2-801 ura3-52 uan5-146 <br />                            NFV150                       MATa his4-539 lys2-801 ura3-52 uan5-150 <br />                            NFVl52                       MATa his4-539 lys2-801 ura3-52 van4152 <br />                            NFV153                       MATa his4539 lys2-801 ura3-52 uan4-153 <br />                            NFV2 18                      MATa ade2-101 ura3-52 vanl-18 <br />                            NFV220                       MATa urg6 (mnn9) mnn2 ura3 <br />                            NFV221                       MATa vrg2 (mnnlO) mnn2 ura3 <br />                            NFv222                       MATa urg3 (mnn8) mnn2 ura3 <br />                            NFV223                       MATa urgl mnnl ura3 <br />                            NFV224                       MATa wg4 mnn2 ura3 <br />                            NFV293                       MATa his4539 ura3-52 van2-93 <br />                            NFV294                       MATa his4539 lys2-801 <br />                                                         ~                   ~  ura3-52 van2-93 _     +                  _ _ <br />                                                         MATa     +           + ura3-52      +    ade2-101 <br />                            NFv917                       MATa ade2-101 ura3-52 van3-117 <br />                            NFv920                       MATa ade2-101 ura3-52 van6920 <br />                            NFV923                       MATa ade2-101 ura3-52 uanl-923 <br />                            NFV936                       MATa ade2-101 ura3-52 van4936 <br />                            NFV941                       MATa ade2-101 ura3-53 vanl-941 <br />                            NFV949                       MATa ade2-101 ura3-52 uan5-949 <br />                            NFV956                       MATa ade2-101 ura3-52 van6956 <br />                            NFV963                       MATa ade.2-101 ura3-52 uan4-963 <br />                            NFV969                       MATa ade2-101 ura3-52 van6969 <br />                            NFV980                       MATa ade2-101 ura3-52 vanl-980 <br />                            NFV981                       MATa ade2-101 ura3-52 uan3-981 <br />                            RSY271                       MATa secl8-1 ura3-52 his4-619 <br />  <br />  <br />                                                                         tant mutants genes. collection different <br />                                  TABLE 2                                contains large percentage dominant mutants <br />                                                                         reported group, unusual <br />               Classification vauadatexesistant mutants               type recessive mutation (van3 117, . <br />                                                                            ?&apos;ANI+ gene product significant amino acid <br />                Mutation class                 isolates <br />                                                                         identity MNN9+ ANp1+ gene products:Work <br />       mutations <br />              Dominant                              32                   MacKay (YIP et al. 1994) BALLOU <br />             VAN1                                    4                   (BALLOUet al. 1991) laboratories prompted comparison <br />             VAN2                                    3                   vanadate-resistant mutants mnn <br />             VAN3                                   10                   phenotypes. &quot; V 9 + gene isolated             VAN4                                    8                   sequenced (YIP et al. 1994). predicted VANl+             VAN5                                   14 <br />             VAN6                                    7 <br />                                                                         MNN9+ gene products limited amino acid iden- <br />                                               3                   tity, especiallyin carboxyl-terminal ends                                                                         predicted amino acid sequences (Figure 1) (YIP et al. <br />  936                                                                   C.   Kanik-Ennulat, E. Montalvo N. Neff <br />  <br />  <br />  <br />   gpcHm <br />  <br />   gpVANl <br />                           r&#226;&#8364;&#732;                                                                                                 535 aa&#226;&#8364;&#8482;s <br />  <br />  <br />  <br />                                                                                                                              493 s <br />  <br />  <br />                                                                                                                                             FIGURE1.-Amino acid sequence identitv <br />                                                                                                                                           predicted gene products cloned <br />                                                                                                                                           genes MNN9,VANlandANPI.          Position                                                                                                                                           region predicted amino acid identity   <br />                                                                                                                                           open reading frame gene. <br />                                                                                                                                           Q22 marked carboxyl-terminal  <br /> gpMNN9 199   ERHALDVQKERRAAMALARNELLFSTIGPKTSWVLWLDAIIETPPSLIQDMTKHNK-AI                                                                  gion gpANPl indicates presence o f 22 <br />                .. .. .. .. .. .. .. .. .. .. ..                                 .. .. .. .. .. .. .. .. .. .. .. .. ..                    consequtiveglutamineresidues       t h e pre- <br /> @AN1 324     DRHAVKV~IRRKLMGRARNWITANAIKPYHSWV~D~VELCPGS~QDLM-SKNYDV                                                                      dicted protein sequence. amino acid se- <br />              .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. ....................        .. .. .. .. ..        quence alignments abetween thc genes <br /> gpANPl 167   DRHGFGAQGPRRKLMARARNLTANALKPYHSWVYWRDVDVELIPTTIMEDLMQSGQGNV                                                                  displayed  . Amino acid identities <br />                                                                                                                                           indicated colons. original database <br />              LAANIYQRFYDEEKKQPSIRPYDLNNWQESDTGLEIASQMGDDEIIVEGYAEIATYRPLM                                                                 submission VANZ gene sequence                                                          .. .. .. .. .. .. ..... .. .. . . .. .. .. . . . . . . .                         missing &#226;&#8364;&#339;C&#226;&#8364;? residue sense strand  <br />              IVPNVWRPL: - -PTFLGTEQPYDLNSWMESQEALALMTLDEDDVIVEGYAEYPTWRVHL                                                                tween nucleotides 1748 1749 (USINK  <br />              .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. ..   NUIAT NEFF 19%)).   second open read- <br />              IVPNVWRPL- - -PMJLGNTQPYDFNSWKESEGGLQLADSLDEDAVIVEGYPEYATWRPHL                                                               ing frame (ORF2) identified KIN1                                                                                                                                           ATRl (KANMAWA et nl. 1988). predicted <br />                                                                                                                                           transmembrane protein gene mutate <br />              AHFYDANGVPGEEMALDGVGGGCTLVKAEVHRDGAMFPNFPFYHLIETEGFAKMAKRLNY                                                                 aminotriazole resistance. <br />              ..     ..    .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. <br />              AYIRDAEGDPNEAVDLDGVGGVSIW(AK1FRNGVQFPAFTFENHAETEAFGKMAKKm;Y <br />               ..   ..    .. .. .. ..        .............................. <br />                                               ............................                                    .. .. <br />              AYMRDPNGNPEDEMELffiIGGVSIW(AKVLRTGSHFPAFSFEKHAETEAFGRLSRRMNY <br />  <br />  <br />              DVFGLPNYLVYHIEEENH-                           395 gpMNN9 <br />               .. .. .. .. .. .. .. .. .. <br />              RVGGLPHYTIWHIYEPSDDDLK 521 gpVANl <br />                ................... <br />                  ................. <br />              NVIGLPHYVIWHIYEPSSDDLK 365 gpANPl <br />  <br />  <br /> 1994). gene products unexpected                                                                       initial report (KANIK-ENNUIAT    NEFF 1990) <br /> identity predicted gene product  ANPI+                                                                  consistent alterations posttranslational mod- <br /> gene (MELNICKand SHERMAN         1993). genes                                                                 ificationof membrane-proteincomplexes. anti- nonessentialforhaploid     cell growth. VAN1                                                                    body cross-reacted sugar-phosphate linkages ANPl gene products 70% identical                                                                    screen picked cells accumulating unknown <br /> region both40% identical toMNN9. VAN1                                                                       phosphorylated sugar-protein modification. Changes MNN9 gene productshave similar carboxyl-termi-                                                                    posttranslational modifications proteins lead <br /> nal position homologous region                                                                    misfolding alterations specific activity open reading frames, ANPl gene prod-                                                                     membrane-associated proteins (HAMMOND        e/ nl. 1994). <br /> uct region positioned centrally open                                                                    Cells containing mutations ANPl gene reading frame (Figure 1).                                                                                               phenotypic similarities unnl mnn9 mu- <br />    sequence identity pheno-                                                                   tant$ drugresistant (aminonitrophenyl pro- <br /> types commonbetween         mnn9 mutants (BAI OU                                                                   panediol), clumpy osmotically sensitive (MCKNIGHT <br />  1982) van1 mutants (KANIK-ENNUIAT        NEFF                                                                  P/ nl. 1981). conserved regionof -200 amino acids <br />  1990) (Table 3, Figure 2). types mutant cells                                                                  common phenotypes recessive mutations sug- resistant high levels sodium orthovanadate,                                                                   gests domain nonessential dis- <br /> clumpy, osmoticor detergentsensitive, hygromycin sen-                                                                   tinct genes similar o r overlapping osmotic f h c - <br /> sitive secrete underglycosylated invertase (Table 3,                                                                tion cell. <br /> Figure 2). combinations ofalleles sporulate                                                                      Analysis invertase vanadate-resistant mutants: viable deformed nonviable spores(BALLOU                                                                     Mutants collection vanadate-resistant mu- <br />  1990; KANIK-ENNULAT    NEFF1990). phenotypic                                                                   tants (Figure 2, lanes d-f h), o r mnn. <br /> alterations ourvanadate-resistant mutants described                                                                  mutants (lanes b c) analyzed active  <br />                                                    \&apos;anadate-Resistant Yeast Mutants                                              937 <br />  <br />                                                                 TABLE 3 <br />                                                   Phenotypes van zng mutants <br />  <br />                                                                                                 YPD+          SD+ <br />                              YPD+           YPD+        YPD +         YPD+          YPD+         \Q      :&apos;   \&apos;o.l:&apos;   Invertase <br />   Genotype         YPD       10 Hyg        2.5 H y g    50 Hyg       100 Hyg      0.1% Doc       (nnl)        (mu)        cl;1ss <br />  <br /> M&apos;t MA Ta           +           +             +            +              -            +         1-3           1-2         IVt <br /> M&apos;t &apos;\M Ta          +           +             +           +/-             -            +         1-3           1-2         IVt <br /> 7 l N n 1-18        +           -             -            -              -            -         6- O       6-8        n/ rrml-923            +           -             -            -              -            -         4-6          3-4        9/ 1 q 6 (m n n 9)     +           -             -            -              -            -         6- 10        6-8        9/ 1 <br /> 71rrn2-93           +           +             -            -              -           +/- <br />                                                                                       2-3        3-4                     9/ r1nn2-14            +           +             -            -              -           +/-        3-4          2-3        9/ 1 <br /> 11nn3-1I7           +           +             +            -              -            +         3-4          2-3          \VI <br /> lInn3-98I           +           +             +            -              -            +         3-4          2-3          \$?t <br /> lI/lIl4-97          +           +             +            -              -           +/-        4-6          3-4        9/ 1 <br /> lMlll5-96           +           +             +           +/-             -            +         3-4          2-3        9/ 1 <br /> 7111 n 5- I50       +           +             +           +/-             -            +         3-4          2-3        9/ 1 <br /> vnnh-l            +           +             +           +/-             -            +         3-4          2-3        8/ 10 <br /> wrnh-920            +           +             +            -              -            +         3-4          2-3        8/ O <br />  V W - I6       +          +/-            -            -              -            -         6-10         6-8        9/ 1 <br /> 71rg.2(mnn O )    +           +             -            -              -            +         3-4          2-3        8/10 <br /> vrg3 (mn18)         +           +             -            -              -            +         3-4          2-3        8/10 <br /> urgI/vrg7           +           +           +/-            -              -           +/-        3-4          2-3        9/ 1 <br /> z1rc4/71rc5         +           +           +/-            -              -           +/-        4-6          3-4        9/ 1 <br />   Strains wcre grown o n 2.5-mI YPD plates SD plates 26&quot; additions. Hygromvcin (Hyg) addrrl platesfrom O 100 pg/ml. Sodium rleoxycholate (Doc) addedto plates 0.01-0.5%. Sodium orthovanatdate <br /> (V0.T:&apos;) addccl plates 1 10 m x t . concentrations reported highest level vanadate cells <br /> c m grow 26&apos;. invertase class indicates approximate degreeof invertase glycosylation. Wt, wild-typeheterogeneous <br /> profile; 8/10, intcrmediate truncation glycosylation profile similar seen m n n 8 mnnl0 mutants; 9/1, <br /> greater truncation o f invertase glycosylation profile similar seen mnn9and mnnl mutants (BAIMX.PI 01. 1991). <br />  <br />                                                                      vertase native gel electrophoresis. Mutant swl8                 b c d e f g h                                      cells grown permissive (26&quot;; lane  restrictive <br />                                                                      (57&quot;; lane g) temperatures grown parallel                                                                      controls. Wild-type cells, secl8-1 mutant cells grown <br />                                                                      permissive temperature, secretea heterogeneous <br />                                                                      population glycosylated invertase (Figure 2, lane . <br />                                                                      heterogeneous size secreted invertase                                                                      theaddition 2 5 0 mannoseunits ouhvard <br />                                                                      bound protein proceeds        Golgi (KL&apos;Nw,- <br />                                                                      ZINSKA et nl. 1987). restrictive temperature cells <br />                                                                      sec18 mutation blocked secretory <br />                                                                      process level endoplasmic reticulum Golgi <br />                                                                      transport accumulate core ER glycosylated  <br />                                                                      vertase (ESMONet nl. 1981) (Figure 2, lane g). <br />                                                                         shown BAILOU associates ( B , , o 1982, <br />                                                                                                                            ~~ <br />                                                                      1990; BAILOUpt nl. 1991), cells containing mnn9 ( 7 q 6 ) <br />                                                                      mutations secrete underglycosylated invertase relative <br />                                              V                       wild-type cells (Figure 2, lane b). mnn9 mu- <br />                                                                      tants, secretedinvertase core orendoplasmic reticu- <br />                                                                      lum modification, blocked early stage                                                                      outer chain elongation. Cells containing mnn8 ( v r g l ) <br />                                                                      mnnlO (vrg2) mutations secrete invertase,   FIG~IU    2.--Native <br />               ~.        gel electrophoresis activein-         somewhat larger mnn9 cells sig- <br /> vertase n cell extracts representative strains.       nifican tly underglycosylated (BAI OL~ et crl. 1991) (Fig- <br /> swl8 cells induced 26&quot;; (g) m 1 8 - cells induced 37&quot;; <br />                                                                      ure 2, lane c). <br /> (h) vrg6 (mnn9) cells; (c) vrg2 (mnnl0) cells; (d) unn5-150 <br /> cells; (e) rmn2-93 cells; ( f ) v n n l - 1 8 cells; (h) VAN7-116       distinct invertase glycosylation pro- <br /> cells. internal form invertase rims dye        files classes vanadate-resistant mutants. Cells shown.                                                  containing van1 mutations produce invertase (Figure <br />  938                                     C. Kanik-Ennulat, E                                                              Montalvo <br />                                                                   Neff   N. <br />  <br /> 2, lane f ) similar pattern underglycosylation       (1991). resistant mutations v n l - 1 8 mnn9 vrg6 (lane b). Cells containing van2        mnn9 (vrgb). strains resistant 210 mM <br /> mutations (lane e), van4  shown), VAN7-116 (lane           Na3V04 YPD media. classes mutants h) mutations secrete underglycosylated invertase       intermediate levels resistance (Table 3). strains  vanadate-resistant mutants (vun3-117           resistant SD media (1-2 mM Na3V04lower, <br /> van3-981) showed wild-type distribution glycosyl-         Table 3). dominant andrecessive mutants high <br /> ated invertase  shown). Cells containing van5          levels vanadate resistance severely af- <br /> vun6 (Figure 2, lane d) mutations produced invertase           fected level cell viability, sporulation, hygro- nearly identical distribution somewhat              mycin-sensitivity pattern secreted protein glyco- <br /> smaller wild type, similar glycosylation pro-      sylation. <br /> file seen mnn8 mnn10 mutants. results                VAN3 class recessive mutations  phenotypes summarized Table 3.                usual alleles: vanadate-resistant mutants    van mutants hygromycin sensitive           lowest level resistance highest levels (Table 3 ) vrg mutants reported BALLOU          spore viablilty, secrete invertase, iswildtype et al. (1991). strains tested grew onYF&apos;Df1Msorbitol       appearance (Tables 3 4). van3 mutants plates showed differential sensitivity WD+O.l%          wild type phenotypes tested  <br /> sodium deoxycholate plates strains tested resis-      usual complementation alleles. <br /> tant toYPD+O.l% Triton X-100 0.5% Tween20 plates.              recessive mutation (van3- 11 7) failed comple- significance sensitivity mutants     ment nearly recessive mutation. type <br /> known. hygromycin deoxycholate sensitivities           mutation times collection leaky incorrectly assembledmembrane-          NF134derived mutants twice NF147-derived <br /> protein complexes. phenotypes           mutants. tetrad analysis mutation unlinked reflection general osmotic-sensitivity asseen loss     mutants tested NFV981 (Table 4). Multi- viability long-term liquid culture.                      ple examples NFV981 mutation (complements <br />    Spheroplasts wild-type, secl8-1 (26          classes NFVl17) 37&quot;), vanl -18, van2- 93 mnn9 cells test            NF147-derived mutations (NFV929). unexpected underglycosylated invertase           complexity complementation analysis intracellular periplasmic compartment. wild-         particular allele class vanadate-resistant muta- <br /> type vanadate-resistant strains tested secreted glyco-     tions partial dominant phenotype sylated invertase. unglycosylated intracellular       reflect phenotypic interaction dis- <br /> form invertase cell wall di-        tinct genetic loci. Thetetrad analysisshown gested, 37&quot; blocked secl8-1 cells,         totally exhaustive does demonstrate accumulate core glycosylated intermediates intracellu-         phenotypic complementation vanadate resistance larly (data shown). Thevanadate-resistant mutants          simple. <br /> export underglycosylated invertase    accumu-          van mutations cause sporulationdefects late intracellularly.                                       spontaneous mutations isolated showed domi- <br />    effect vanadate secretion invertase        nant vanadate-resistance phenotype. class ofmuta- wild-type cellswas tested. kinetics profile      tion reported investigators  invertase compared wild type, NFV18 NFV93           SKY et al. 1984;BALLOU et al. 1991).Tetrad analysis <br /> induced presence absence 2 mM sodium              proved difficult sporulation efficiency ~ 1 0 % , <br /> orthovanadate. Addition vanadate wild-type         sporeviability 10-25%. dominant mutant <br /> sistant mutant cells effect time course        (NFV116) analyzed tetrad analysis invertase induction glycosylation pattern      linked vanl, van2, van3 van4 mutants tested (Ta- <br /> 4 h r induction time. invertase induced          ble 4) heterozygous diploid didnotsporulate appreciably different strains          mated van5 van6 mutant strains. <br /> tested presence absence vanadate (0.5-0.8         mutation interesting phenotypic properties <br /> pmol glucose released lo6 cells). results             (Figure 2, Table 3) tentatively designated VAN7- <br /> agree observations BALLOU      (1990)       16. strain showed high level vanadate resis- <br /> derglycosylation secreted invertase mnn mu-          tance clumpy cellsand growth properties simi- <br /> tant cells does notaffect amountor specific activity.      lar van1 -18 mnn9 mutants (Table 4)    results collection mutants        unable say precisely genes represented <br /> scribed havesimilar invertase glycosylation pat-          dominantmutants inconsistent aspects <br /> terns seen mnn mutants.             poor sporulation. <br /> general inverse correlation degree              Heterozygous diploid strains wild-type vanadate-resistance extent outer chain           dominant mutations sporulated YPD YF&apos;D + 3 <br /> elongation. Different degrees vanadate resistance       mM vanadate plates low frequency (&lt;() ). different mutants seen BALLOU      et al.   weeks plates 26&quot; red sectors appeared                                                 Vanadate-Resistant Yeast Mutants                                           939 <br />  <br />                                                            TABLE 4 <br />                                         Tetrad analysis vanadateresistant mutations <br />  <br />                                                       complete tetrads <br />           strain <br />              Diploid                    4:O         3: 1       2:2           1:3        0:4        Spore <br />                                                                                                viability         (%) <br />         WT (Y x uan3-117                  0          0          16            0          0                  93 <br />         uan3-117 X uanl-18                2          7           1            0          0                 100 <br />         uan3-117 X uan2-93                1          8           1            0          0                 100 <br />         uan3-117 x uan4-97                2          6           2            0          0                 100 <br />         uan3-117 X uan5-96                2          5           2            0          0                 85 <br />         uan3-117 X van6920                1          5           0            0          0                 75 <br />         uan3-117 X uan3-981              10          0           0            0          0                 100 <br />         WT (Y X VAN7-116                  0          0           4            0          0                 50 <br />         VAN7-116 X uanl-18                0          2           0            0          0                 20 <br />         VAN7-116 X uan2-93                0          2           0            0          0                 20 <br />         VAN7-116 X uan3-981               0          0           3            0          0                 25 <br />         VAN7-116 X uan4-97                0          2           1            0          0                 25 <br />              Tetrad columns vanadate-resistant: vanadate-sensitive spores. <br />  white diploid cell patches heterozygous forthe                   taining 1 mM sodium orthovanadate. large white 2- 101 marker. vanadate-resistant mutations                  colonies thousands smaller diploid cells allow normal sporulation                yellow ones. retesting URA&apos; plasmid trans- <br /> trols bypassed low frequency. Heterozygous                formant colonies showed consistent sensitivity SD + homozygous diploids containing van1 - 18                 2 mM sodium orthovanadate. transformants uan2-93 mutations, diploids containing null               grown total nucleic acid extracted. Aliquots alleles genes, showed phenotype               preparations transformed CaC12-treated <br /> exact cause sporulation phenotype unknown.                Escherichia coli cells plasmids recovered possible sporulationdefects seen com-             selection ampicillin-resistant colonies. <br /> binations vanadate-resistant mutants struc-               related plasmids wereisolated library. <br /> tural defects spore coats ascus walls,               Restriction enzyme analysis revealed overlapping yeast <br /> allow premature germination. BALLOU(1982) char-                  genomic Sau3A DNA fragments plasmids <br /> acterized defects ascus walls mnn mu-              (pNV93-3,4,6, Figure 3). complete restriction <br /> tants mutate tovanadate resistance. yeast             map plasmid pNV93-6 generated, series <br /> mutants sporulate presence glucose                dropout deletions (pNV93-9,lO) subclones reported (ESPOSITOand KLAPHOLTZ 1981).                      (pNV93-12,14,15) generated. resulting plas- <br />    diploids vanadate-resistant genes                 mids transformed NFV93 tested vana- <br /> identified identified BALLOU       et al.          date sensitivity/resistance. plasmids tested comple- <br /> (1991) sporulated poorly  Independent Ura-                 mented uan2-93 mutation. data showed isolates ofurgstrains isolated tested confirm            complementation activitywas contained 2.2-kb <br /> vanadate resistance. strains (NFV220-224)                 EcoRI/HindIII fragment. fragment direction- <br /> transformed   YCp50       (vector control),pNVll                ally cloned phage vectors Mp18and Mp19 DNA <br />          + <br /> (YCp50 VANlf) pNV93-3 (Figure 3, YCp50 +                         sequence analysis. <br /> VAN2+) plasmid DNAs. Ura3&apos;trans-                            single-stranded phage DNAs sequenced <br /> formants vanadate sensitive. vanadate-              sequential oligonucleotides spaced approximately <br /> resistant mutations urgstrains tested compli-            every300 nucleotides. open reading frame <br /> mented wild-type VANI+ VAN2+ genes.                        &gt;25 codons predicts protein 337 amino acids (Fig- <br />    Isolation VAN2+ gene: Haploid cells                    ure 4). predicted gene productof VAN2+ open <br /> taining uan2-93 mutation turn yellow SD plates                reading frame showed significant amino acid iden- <br /> containing 1-3 mM sodium orthovanadate, grow                     tity protein sequence time sequence slowly wild type. Heterozygous diploid cells               submission. open reading frame      predicts protein <br /> (uan2- 93/ +) white SD plates grow plates              multiple potential transmembrane domains. containing 1 mM sodium orthovanadate. YCp50li-                     EcoRI/HindIII fragment used hybridization <br /> brary Sau3A wild-type yeastgenomic DNA fragments                  probe yeast chromosomes transferred nylon <br /> (SHIHet aL 1988) transformed NFV293 cells,                  filter. Chromosome XVDNA bound probe (hybrid- <br /> selecting URA+ colonies SD media. resulting                   ization data shown). <br /> 25,000 transformant colonies washedoff plates                  DNA fragment identilied library  pooled. cells plated SD plates                 tains VAN2+ gene: PlasmidpNV93-12  <br />  940                                         C. Kanik-Ennulat, E. Montalvo N. Neff <br />  <br />  <br />  <br /> 4 ; <br />       P <br />           H <br />           lkb         H B E <br />                                                                                                                               1 <br />                               1 <br />                                   - <br />                                   Sn <br />                                                                                                            Sn <br />                                                                                                                             H P S p B <br />                                              1 <br />                                              II 11 <br />                                                        1 <br />                                                              X <br />                                                                                                                                                                                            X <br />                                                                  1 <br />                                                                  1 <br />                                                                      X <br />                                                                                                                                           1 <br />                                                                            * <br />  <br />  <br />  <br />  <br />                                                                            pNV93-3           FIGURE3.-Restriction map                                                                                           KAN2 locusandplasmidconstruc- <br />                                                                            DNV93-4 <br />                                                                              ~~ <br />                                                                                           tions. Theupper   represents <br />                                                                                                          line                                                                                                   region yeast genomic&apos;DNA fomd <br />                                                                            pNV93-6                                                                                                                                                                                                         complementing            library <br />                                                                                           plasmids. Restriction enzyme ahbre- <br />                    P                                                       pNV93-9        viations P, PUuII; H, HindIII; l3, <br />                                                                                           BgflI; E, EmRI; Sn, SnnB1; Sp, SphI; X, <br />                                                                            pNVg3&quot;10       Xl,nl. T h e position <br />                                                                                                                                                                                                                                                                                                                                               predicted <br />                                                                                           open reading frame VAN2 gene <br />                                                                            pNV93-12       shown small arrow. <br />                                                                            pNV93-14.15 <br />  <br />                               7                                            pNV93-16 <br />  <br />  <br />  <br /> structed cloning 3 k b Hind111 fragment                  gonucleotides flanking SnnBlsiteswereused           integrating vector Mp5. plasmid contains unique                   analyze genomic DNA diploid transformants <br /> BglII site yeast genomic DNA outside pre-                  polymerase chain reaction technique. dicted coding region VAN2&apos; gene (Figure 3,                       diploid transformants examined contained pNV93-12). PlasmidpNV93-12 DNA linearized                            wildtype (1.3kb DNA fragment) null allele restriction enzyme BglII used transform                      (1.5-kb DNA fragment; data shown). NFV294 selecting Ura+ transformants. indepen-                        contained wild-type size allele likely <br /> dent diploid transformants sporulated tetrads                   represent conversion events t @ - l locus. <br /> dissected. cases tetrads (20/20) contained                      diploid transformant strains sporulated poorly Ura&apos; Ura- spores. case, spores                    (&lt; 10% relative wild-type diploid). 20 vanadate sensitive. case, Ura+ spores                  tetrads analyzed diploid, vanadate sensitive ura- spores vana-                   viable spores. Sixteen tetrads viable <br /> date resistant. auxotrophic markers MAT                      spores noviable spores. invia- <br /> locus segregated 2:2. case integration                  ble spores failed germinate YPD dissection <br /> event occurredonthechromosomecontainingthe                               plate, remaining small refractile. viable spores <br /> van2-9? mutation. second case, integration oc-                    trp- (34/34). auxotrophic        markers curred chromosome containing        wild-type                 2:2. result demonstrates essential nature gene.Thesedatademonstratethatthefragment                              open reading frame identified VAN2+ gene yeast genomic DNA isolated white-yellow colony                    haploid cell viability. <br /> color screen contains VAN2+ gene.                                       Sequence vun2-93: van2-93 gene ampli- <br />    predicted open reading frame      VAN2&apos;                   fied NFV293 cells. comparison wild-type <br /> gene contains SnaBl restriction enzyme sites                  (NF134) mutant sequences revealed single G DNA sequence. sites unique plas-                    change position 1010. wild-type protein se- <br /> midpNV93-12. Dilute religation SnaBl digested                         quence predicts arginine residue (CGT = Arg) pNV93-12 DNA creates deleted VAN2 + gene                        position 274 mutant sequencepredict5 histi- <br /> predicted protein product approximately 60 amino                      dine residue (CAT = . charged amino acid <br /> acids (pNV93-12s). Plasmid DNAs pNV93-12                             anchor transmembrane domain (Figure 4). pNV93-12s digested restriction enzyme BglII                    alteration arginine histidine position used transform vanadate-resistant strain NFV293                   effect overall function structure Ura+. Ura&apos; transformants tested                   essential gene product, butmay create leaky unsta- <br /> integration pNV93-12 DNA strain NV293were                          ble structure. histidine residue potential <br /> vanadate sensitive (20/20). Ut  trans-                      hydrophobic character does arginine residue, <br /> formants integrationof pNV93- 12s DNAwere                       effective transmembrane anchor <br /> vanadate resistant (20/20). results demonstrate                    stoptransfer sequence. open reading frame identified VAN2&apos; <br /> gene contains complimentingactivity reces-                                          DISCUSSION <br /> sive mutant van2-9? vanadate-resistant gene. <br />    V M 2 + gene essential haploid cell viabil-                  Sodium orthovanadate toxic yeast cellsand spon- <br /> ity Strain NF3623 transformed selecting Trp&apos;                      taneous mutations vanadate resistance arise ing van2 TRP+ null allele construction. Twooli-                      number genes. target structure                                                      Vanadate-Resistant Yeast Mutants                                           941 <br />  <br />  <br />  <br />   61   AACGAAAAACAUTATATATMCGCTGCCAAGCCTATTTCTTGTTGCCGATTAATTGTT <br />  <br />  121   GACATCCTCTTAATTACCAAAAGAWCTAAGAUACAAACACACTMCCACACAGTACCT <br />  <br />  181   TTCWCCGAATGTCTWTTG~CAGGTCATGCAWCATAACCCm~TTCAGTT <br />    1                M   S   E   L   K   T   G   H     G   H   N   P   W      S   Y <br />  <br />  241   GCACCAATTCCGGTCCGATCTCTATTTTATCCTATCCTACTGTGGTTCCTCCATTTTAATGACGGTG <br />   18    <span id='am-20' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; N S G P S L S Y C G S S L ~ T Y <br />  <br />  301   ACTAACAAGTTCGTCGTCAACTTGAAGGATTTCAACATGAACTTTGTTATGCTTTTCGTG <br />   38    U K F V V N L K D P N M N F V M L F V <br />  <br />  361   CAATCTTTGGTTTGTACTATAACCTTGATTATC-TACTWCTATWWGTTC <br />   58    Q   S   L   V   C   T     T   L       L   R     L   G   Y      K   F <br />  <br />  421   CGTTCATTAAACAAAACAGACGCCAAGAACTGGTTCCCTATTTCCTTTTTACTGGTCTTG <br />   78    R S L N K T D K N W F P S F L L V L <br />  <br />  481   ATGATCTACACCTCTTCOAAGGCTTTACAATACTTWTGTTCCAATTTACACCATTTTC <br />   98    M     Y   T   S   S   K     L   Q   Y   L     V   P                                                                          FIGURE4.-DNA sequence VAN2 <br />                                                                         Y    T     F <br />                                                                     gene. DNA sequence predicted trans- <br />  541 AAOMTTTTGACTATTATCTTGATTGCTTATOOTOAOOTTCTCTTTTTT~TGGCTCTGTC lation open reading frame shown. <br />  118  K N L T I L Y G E V L F F G G S V                       SnaBl sites usedto make null construc- <br />                                                                     tion (pNV93-16) dropout deletion <br />  601 ACCTCCATGGAATTGTCATCATTTTTGTTGATGGTCCTTTCTTCTGTCGTTGACCAACTTGG (pNV93-12s) underlined. Seven potential <br />  13 8 T S M E L S S F L L Y V L S S V V T W                       transmembrane domains 20-25 amino acids <br />                                                                     underlined  Potential Nglycoslylation <br />  661 GGTGACCAGCAAGCTGTGGCTGCCAAGGCTGCTTCATTTGGCTGAAGQAGCAGCCGGTGCT  sites (NXT/S) marked asterisks. po- <br />  158  G D Q O V K S L E G G                                                                     sition van2-93 mutation <span id='am-43' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-44' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-50' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> <br />  721   GTTGCCTCCTTTAACCCAGGTTATTTCTGQATGTTCACCAACTGTATCACTTCTGCATTA                      &apos;010                   DNA sequence <br />  178    V     S   F   N   P   G   Y   P   W   W   F   T   N   C     T    S     L <br />                                                                                          deposited GenBankAccession <br />                                                                                          Number U15599. sequence  781   TTCGTTCTTATMTGAGUAGAGAATTAAGTTAACTAACTTCAAWATTTCGACACTATG                         benearly identical L33915 (GOG5). <br />  198    p V L l 4 R K R K L T N F K D F D T y <br />  <br />  841   TTTTACAACAATGTTTTGGCTCTACCTATTCTATTATCTGCTGTTTTCTTTCTGTGTGGMGAT <br />  218    F Y N N V L L P L L L F S F C Y E D <br />  <br />  901   TGGTCTTCAGTTAATTTGACACCAATAACTTTTCTMCGATTCGCTAACTGCTATGATCATC <br />  238    W   S   S   V   N   L   T   N   N   F   S   N   D   S   L   T      M                             *               * <br />  961   AGTGGTOTTGCATCCGTCGGTA~TCTTACTGTTCCOOTTTTCG <br />  258    S   G   V     S   V   G     S   Y   C   S   G   W   C   V   R    V   T   S <br />  <br /> 1021   TCTACTACATATTCGATGGTAGGGGCTTTGMCAAGCTGCCAATTGCCTTGTCTGGTTTG <br />  278    S   T   T   Y   S   Y   V   G     L   N   K   L   P       L    S   G   L <br />  <br /> 1081   ATTTTCTTTOAGTCTCCCTTCTTATCTATCTATT~CTCCATTTTTATTGGTTTCCTA <br />  298      F   F   D   S   P   R   N   F   L   S     L   S     F      G   F   L <br />  <br /> 1141   TCAGGTATTATTTATGCTGTTGCCAAAUGAAGCAACAAGCCCAACCTT- <br />  318    S   G       Y     V     K   Q   K   K   Q   Q     Q   P    L   R   K <br />  <br /> 1201   TGAGAACTTACGGGGGGTOTTATTTTTTTTTTTTTGGTTTATTTTATTTTTATA <br />  <br />  <br />  <br /> uivo toxic molecule unknown, logically mono-                   mechanism accumulates toxic molecule. <br /> meric orthovanadate compete monomeric                           Growth yeast cellsin 5 mM orthovanadate causes thophosphate active uptake metabolism                         accumulation oftoxic vanadium compounds  <br /> mimics structure. enzymes form phos-                         known structure wild-type cells vanadate- <br /> phoenzyme intermediates sensitivityto orthovana-                       resistant cells. work ourwork (KANIK-ENNULAT <br /> date. Given large number enzymes                         NEFF1990) demonstrate representative vana- <br /> cell, unlikely mutationof single                    date-resistant mutants alterations phos- create overall cellular resistance.                                phate uptake. results make unlikely vana- <br />    work WILLSKYand collaborators (WILLSKY           et               date resistance yeast isdue inhibition <br />                                                                                                                            uptake <br /> al. 1984, 1985) demonstrates vanadate resistance                       orthovandate. yeast likely arise alteration cellular                 phenotype underglycosylated invertase inthe <br />  942                                    C. Kanik-Ennulat, E                                                             Montalvo       N. Neff <br />  <br /> vanadate-resistant mutants arise number         receptors, known involved se- <br /> molecular defects. Secretion underglycosylated          cretion andmembrane         fusion processes (WARREN <br /> vertase reported yeast genes        1993) potential target molecules orthovandate mnn vrgand van mutants (ERDl, HARWICK et al.           considered possible alternative genes 1990; KRE2, HILLet al. 1992; PMRl, RUDOLPH           et   mutationto      vanadate resistance inthisspecificity <br /> al. 1989). genes knownor poten-       model. <br /> tial roles secretory pathway. Mutations           model toxicity consistent vanadate <br /> <span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span>-binding proteins secretory pathway produce         resistance arising alteration loss function <br /> underglycosylated invertase (FERRO-NOVICK              certain integral membrane-associated proteins. <br /> VICK 1993). otherphenotypes hygromycin sensi-          phenotypes vanadate-resistant mutants tivity, detergent sensitivity, sporulation defects low    consistent models. major orthophos- <br /> viability prolonged liquid culture consistent      phate storage compartment yeast vacuole, weakness plasma membrane cell wall           polymerizes polyphosphate (URECHet defects membrane-protein complexes.                 al. 1978). Polyphosphate chelator cations <br />    support model resistance elimi-      Mg2 Ca2+ basic amino acids, helping nation toxic molecule formed intracellularly.    concentrate store molecules vacuole. toxic molecule formed late endoplas-        low pH environment vacuole vacuolar <br /> mic reticulum early Golgi apparatus ionic        network convert orthovanadate polyvanadate metabolic environment. Alternatively Golgi appara-        vanadyl (WILLSKY   1990) providing competitor  <br /> tus contain sensitive target toxic        mal vacuolar phosphate metabolism. Cells molecule. observation vanadate-resistant        sensitive vanadate toxic molecule accu- <br /> mutants secrete underglycosylated invertase          mulates acidic compartments vacuo- <br /> cell wall plasma membrane defects, supports model        lar network.Cells vanadate resistant pre- vanadate-resistant cells eliminate toxic mole-       venting concentrationof vanadate laterstages <br /> cule cell alterations secretory pathway    secretory pathway. exact molecular role stage Golgi apparatus.                          MNN9+, VANl+ VAN2+ genes known <br />    aretwo simple models fit current data       DNA sequence analysis predicted gene toxic vanadate molecule eliminated        products does point known function. cell. change permeability model.     VAN2 gene identified N. DEANand col- wild-type Golgiapparatus creates toxic molecule       leagues (J. B. POSTER   N. DEAN,unpublished data) directtarget, dilution organelle space    gene, called GOG5, mutate vanadate loss permeability control compart-      resistance. cells containinga vanadate-resistant <br /> ment eliminate toxicity. model          GOG5 mutation hygromycin sensitive, accumulate <br /> normal posttranslational modifications secreted pro-       secreted proteins endoplasmic reticulum teins truncated pools soluble molecules     Golgi-related defects. localization diluted aswell (ABEIJON et al. 1989).          study function proteinsidentified study <br /> create general plasma membrane problems, struc-            help understand structure function tures containing proteins proper modification         membrane-associated proteins Golgi apparatus. inaccurately assembled membrane                thank members yeast laboratoriesat Sloan-Ketter- <br /> integrity itself compromised. model,         ing Institute comments especially MONA SPECTOR    genes mutate vanadate resistance            help dissection sporulation. thank VIVIANMACKAY encodetransporters membrane-associated              CLINTBALLOU     communicating unpublished results, LUN <br /> proteins regulating intraorganellesoluble molecule        BALLOU   IRENE GRIFTfor yeast strains thank BOBBY  BAUMfor <br />                                                               information ATRl open reading frame (ORF2) adja- <br /> content.                                                      cent VANI, noting identity VANl ANPl. <br />    second model requires change vesicle           thank PHILROBBINSfor advice information VANl <br /> membrane targeting (FRANZUSOFF         SCHEKMAN   1989;   ANPl identity  <br /> PRYER   et al. 1992), toxic molecules shunted secretory pathway level Golgi                               LITERATURE CITED <br /> apparatus. model requires alteration Golgi pro- <br />                                                               ABEIJON,C., P. ORLEAN,  P. W. ROBBINS     C. B. HIRSCHBERG,   1989 <br /> teins normal pathway constricted, setting           Topology glycosylationin yeast: characterizationof <span id='am-13' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="GDP" datatype="xsd:string"></span>GDP</span>-man- accumulation intermediates. inter-              nose transport lumenal <span id='am-27' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span>guanosine</span> diphosp <br /> </body></html>