<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, July 2007, p. 4784&#226;&#8364;&#8220;4795                                                                           Vol. 27,  13 <br /> 0270-7306/07/$08.00&#226;&#171;&#185;0 doi:10.1128/MCB.00494-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />   Role Histone Deacetylase 1 Human Tumor Cell Proliferation&#228;&#338;&#164; <br />           Silvia Senese,1,2 Katrin Zaragoza,1 Simone Minardi,2 Ivan Muradore,1 Simona Ronzoni,1 <br />                   Alfonso Passafaro,1 Loris Bernard,2 Giulio F. Draetta,3 Myriam Alcalay,2 <br />                                   Christian Seiser,4 Susanna Chiocca1,2* <br />   European Institute Oncology, Department Experimental Oncology, 20141 Milan, Italy1; IFOM-IEO Campus, Adamello 16, <br />       20139 Milan, Italy2; Cancer Research, Merck Research Laboratories, Basic Research, 33 Avenue Louis Pasteur, Boston, <br />              Massachusetts 021153; Max F. Perutz Laboratories, Medical University Vienna, Vienna Biocenter, <br />                                           Dr. Bohr-Gasse 9/2, 1030 Vienna, Austria4 <br />                            Received 21 March 2007/Returned modification 11 April 2007/Accepted 18 April 2007 <br />  <br />              Posttranslational modifications core histones central regulation gene expression. Histone <br />           deacetylases (HDACs) repress transcription deacetylating histones, class HDACs crucial role <br />           mouse, Xenopus laevis, zebra fish, Caenorhabditis elegans development. role individual class           HDACs tumor cell proliferation investigated using RNA interference-mediated protein knockdown.           absence HDAC1 cells arrest G1 phase cell cycle G2/M <br />           transition, resulting loss mitotic cells, cell growth inhibition, increase percentage           apoptotic cells. contrary, HDAC2 knockdown showed effect cell proliferation unless concur- <br />           rently knocked HDAC1. Using gene expression profiling analysis, inactivation HDAC1 <br />           affected transcription specific target genes involved proliferation apoptosis. Furthermore, HDAC2 <br />           downregulation did cause significant changes compared control cells, inactivation HDAC1, <br />           HDAC1 plus HDAC2, HDAC3 resulted distinct clusters. Loss HDACs impair cell <br />           cycle progression affecting transcription specific target genes biological <br />           processes. data support idea drug targeting specific HDACs highly beneficial           treatment cancer. <br />  <br />  <br />    eukaryotic cells, balance acetylation                      shown induce polyploidy human cancer cell lines <br /> deacetylation critical gene transcription func-               (55). learn HDACs biological pro- <br /> tions different cellular proteins. Histone acetylation regu-               cesses regulate, strong idea identities lated histone acetyltransferases histone deacetylases                relevant target deacetylases need determined aid (HDACs), conserved yeast (Saccharomyces <span id='am-51' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span>cer</span>-                      development HDAC inhibitors anticancer agents evisiae) humans. Mammalian HDACs grouped                            emerging. clear example demonstration HDAC2 classes according homology yeast pro-                     induction loss adenomatosis polyposis coli tumor <br /> teins. class enzymes HDAC1 HDAC2 ubiqui-                           suppressor colorectal tumorigenesis, providing reason tously expressed involved general                         treating certain carcinomas HDAC inhibitors (58). Inter- <br /> cellular processes. According sequence, HDAC3                        estingly, demonstrated previously HDAC <br /> belongs class family, interact               inhibitor valproic acid selectively induces polyubiquitination <br /> class class II enzymes represent functional                   proteasomal degradation HDAC2 (35). Lately, HDAC3 <br /> link families (11). Class II enzymes                        required normal mitotic progression tissue-specific functions, class III enzymes NAD                      maximal phosphorylation histone 3 (H3) Ser 10 (S10) <br /> pendent involved control life span                      mitosis (38). Phosphorylation H3 S10  <br /> certain organisms (15, 16, 33). Recently, class IV,                     characterized evolutionary conserved mitotic event (6, <br /> cludes HDAC11-related enzymes, added (12, 15).                          17, 22, 23, 45). connection transcriptional repression                        Recent experiments small interfering RNA (siRNA) HDAC recruitment, HDAC inhibitors reverse si-                            implicated HDAC1 HDAC3 regulation prolifer- <br /> lenced genes (reviewed reference 30).                                         ation survival cancer cells (13). particular, idea <br />    HDAC inhibitors new class drugs anticancer                      HDAC1 potential target therapeutic <br /> potential. shown induce apoptosis effec-                       intervention certain cancers stems series studies. <br /> tively cancer cells currently clinical trials                HDAC1 shown inhibit estrogen receptor alpha <br /> variety cancers (4, 40). example includes HDAC                        protein expression transcriptional activity, suggesting inhibitor suberoylanilide hydroxamic acid, recently                    modulate breast cancer progression (32). Overexpres- <br />                                                                                  sion HDAC1 mRNA protein levels reported                                                                                  human gastric prostate tumors (5, 18, 34, 44),   * Corresponding author. Mailing address: European Institute              overexpression HDAC3 seen colon tumors (54). <br /> cology, Department Experimental Oncology, Ripamonti 435, <br /> 20141 Milan, Italy. Phone: 39-02-57489835. Fax: 39-02-57489851. E-mail: <br />                                                                                  Furthermore, published data indicate role mouse HDAC1 <br /> susanna.chiocca@ifom-ieo-campus                                               regulation proliferation development.  <br />   &#228;&#338;&#164; <br />     Published ahead print 30 April 2007.                                   stance, expression HDAC1 induced growth <br />  <br />                                                                           4784 <br />  VOL. 27, 2007                                                            ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                                              4785 <br />  <br />  <br /> factor activation mouse T cells fibroblasts (3, 19)                              Western blotting, primary antibodies anti-HDAC1 (monoclonal mouse <br /> HDAC1 levels elevated highly proliferative                           antibody; Upstate), anti-HDAC3 (ab 7030; Abcam), anti-caspase-3 (H-277; <br />                                                                                          Santa Cruz Biotechnology) incubated membrane overnight 4&#194;&#176;C; <br /> tissues, embryonic stem (ES) cells, transformed                              anti-HDAC2 (ab 7029; Abcam), anti-p21  kind gift K. Helin&#226;&#8364;&#8482;s lab), <br /> cell lines (3, 37), suggesting link HDAC1 function                             anti-acetylated H3/H4  house antibody), antitubulin (DM1A; Sigma), proliferation. accordance idea, disruption                           anti-H3 (ab 1791; Abcam) incubated 2 h room temperature. Proteins HDAC1 gene resulted reduced proliferation mouse                                visualized chemiluminescence ECL Western blotting detection <br /> embryos ES cells (37), overexpression HDAC1                               reagents (Amersham Biosciences). <br />                                                                                             Growth curve. determine effects siRNA knockdown cell prolif- <br /> led impaired cell proliferation murine fibroblasts (3).                            eration, 24 h transfection, cells collected trypsinization, counted Taken  results suggest class HDACs                             use hemacytometer trypan blue dye, plated 3,000 viable cells crucial controlling proliferation state mammalian                              U2OS MCF7 2,000 viable cells MCF10A 96 tissue <br /> cells.                                                                                   culture dish final volume 200 &#226;?&#174;l. 24 h, 96 tissue culture dish <br />    HDAC1 HDAC2 heterodimerize gener-                                   stained crystal violet solution left dry.  crystal violet <br />                                                                                          incorporated cellular membrane solubilized solution 10% <br /> ally large multiprotein complexes: Sin3, NuRD,                              <span id='am-41' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span>&#226;&#8364;&#8220;phosphate-buffered saline (PBS) absorbance 595 nm CoREST complexes (16). Numerous transcription factors,                               measured MRX microplate reader (Dynatech). calculated  <br /> including regulators cell cycle, differentiation, develop-                        crease (n-fold) absorbance ratio absorbance time absorbance <br /> ment, shown associate directly HDAC1                               time zero  time 48, 72, 96 h siRNA transfection time <br /> HDAC2 HDAC1/HDAC2 complexes, mediat-                                     zero 48 h transfection). experimental point average                                                                                          independent experiments, respective standard deviation. <br /> ing repression specific target genes (1, 7, 42).                             Colony formation assay. hours transfection, cells col-  loss Sds3, component Sin3/HDAC1/                                   lected trypsinization, counted use hemacytometer trypan blue dye, <br /> HDAC2 complex putative recruiting factor, led cell                              plated 3,000 viable cells U2OS MCF7 2,000 viable cells/ <br /> death missegregation chromosomes mitosis                                MCF10A tissue culture dish. days plating, cells <br /> (8).  HDAC1- HDAC1/HDAC2-mediated                                     stained crystal violet solution (1% crystal violet, 35% ethanol)                                                                                          colonies counted. calculated relative CFU follows: number chromatin modifications important cell cycle                              colonies sample/number colonies control. average relative <br /> control development. light evidence,                               CFU plates calculated, respective standard deviation. <br /> ducted study analyze effect depleting class deacety-                          Fluorescence-activated cell sorter (FACS) analysis cell cycle. detect <br /> lases human tumor cells RNA interference (RNAi).                                HDAC1, phosphorylated S10 H3 (H3-P), DNA HDAC1, cyclin  ablating HDAC1 HDAC3 protein                                 DNA, 72 h cycle transfection, attached cells superna- <br />                                                                                          tant collected resuspended 106 cells/ml 1% <span id='am-11' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> 15 <br /> expression inhibition tumor cell proliferation.                           min. PBS washing, pellet resuspended ice-cold 70% ethanol Clearly, HDAC1 knockdown abolishes ability tumor                                  stored 4&#194;&#176;C. immunodetection, cells washed twice PBS  <br /> cells proceed mitosis. contrary, HDAC2                                 meabilized 0.1% Triton X-100 PBS 10 min. blocked 5% <br /> knockdown shows effect, unless concurrently knock                                  normal goat serum 20 min, cells incubated anti-HDAC1 mouse HDAC1 HDAC2. addition,                                      monoclonal antibody (Upstate), 1:250, anti-H3-P rabbit polyclonal antibody <br />                                                                                          (Upstate) anti-cyclin (Pharmingen), 1:100, PBS plus 1% bovine serum <br /> ducted analysis changes gene expression U2OS                             albumin 2 h room temperature. Cells rinsed PBS incubated cells response knockdown deacetylases demon-                                1 h Cy5-conjugated goat anti-mouse (1:50; Jackson Immunoresearch) strated large number HDAC target genes involved                            fluorescein isothiocyanate (FITC)-conjugated goat anti-rabbit antibodies. Cells proliferation apoptosis.                                                          washed resuspended 1 ml solution containing 2.5 &#226;?&#174;g/ml <br />    overall evidence points idea                           propidium iodide (PI) (Sigma) PBS 250 &#226;?&#174;g/ml RNase PBS incu- <br />                                                                                          bated overnight 4&#194;&#176;C acquisition. <br /> individual class HDAC plays different role regulating                                Samples acquired FACSCalibur (Becton Dickinson) flow cytometer. <br /> genes involved various biological processes.                                          10,000 events acquired. Analysis performed using CellQuest <br />                                                                                          3.3 (Becton Dickinson). <br />                                                                                             FACS analysis activation caspase-3. active caspase-3 analysis, 96 h <br />                          MATERIALS METHODS                                           cycle transfection, attached cells supernatant    Cell culture. U2OS MCF7 cell lines grown Dulbecco&#226;&#8364;&#8482;s modified              collected, fixed 1% formaldehyde PBS 70% cold ethanol, Eagle&#226;&#8364;&#8482;s medium (Cambrex) supplemented 10% fetal bovine serum (Cam-                  permeabilized 0.1% Triton X-100 10 min. Prior incubation anti- <br /> brex), antibiotics, 2 mM <span id='am-6' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span>L-glutamine</span> (Cambrex). MCF10A cell line             caspase-3 polyclonal antibody (9661L; Cell Signaling), cells incubated maintained F12-Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s medium (1:1) supplemented                  30 min 10% goat serum PBS. washed, cells incubated 5% horse serum, 20 ng/ml epidermal growth factor, 0.5 &#226;?&#174;g/ml hydrocorti-             anti-rabbit FITC antibody (Sigma) 1 h room temperature (light <br /> sone, 50 ng/ml cholera toxin, 10 &#226;?&#174;g/ml insulin, antibiotics. cell lines     protected). labeled cells stained overnight 4&#194;&#176;C 2.5 &#226;?&#174;g/ml PI <br /> cultured humidified 37&#194;&#176;C incubator 5% <span id='am-1' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span>CO2</span>.                                     RNase 250 &#226;?&#174;g/ml PBS. cellular suspension passed    Transfection cells siRNA, electrophoresis, Western blotting.              FACSCalibur (Becton Dickinson) flow cytometer. 10,000 events siRNAs ordered MWG BIOTECH prepared according                       acquired, analysis performed using CellQuest 3.3 (Becton Dickinson) <br /> manufacturer&#226;&#8364;&#8482;s instructions (stock concentration 20 &#226;?&#174;M molecular-biology-          ModFit LT 3.1 software. <br /> grade water). Oligofectamine used siRNA transfections (Invitrogen,              Cell synchronization. hours cycle siRNA trans- <br /> San Diego, CA). following siRNA sequences used: 5&#226;&#172;&#732;-CGUACGCG                     fection  , cells synchronized G1/S transition treatment <br /> GAAUACUUCGATT-3&#226;&#172;&#732; siRNA LUC (control), 5&#226;&#172;&#732;-CAGCGACUGUUUG                               2 mM <span id='am-21' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span> 14 h, released 8 h, treated AGAACCTT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CUAAUGAGCUUCCAUACAATT-3&#226;&#172;&#732; siRNA                                    thymidine 14 h. washes PBS, cells cultured fresh <br /> HDAC1, 5&#226;&#172;&#732;-UCCGUAAUGUUGCUCGAUGTT-3&#226;&#172;&#732; siRNA HDAC2,                                  medium medium 0.1 &#226;?&#174;g/ml nocodazole different times  indicated <br /> 5&#226;&#172;&#732;-GAUGCUGAACCAUGCACCUTT-3&#226;&#172;&#732; siRNA HDAC3.                                             experiment) harvested. Cells synchronized prometaphase    Oligofectamine reagent diluted used according manufacturer&#226;&#8364;&#8482;s           treatment 0.1 &#226;?&#174;g/ml nocodazole 16 h released instructions. siRNA used optimized fixed                  drug-induced cell cycle block washed times PBS. <br /> knockdown experiment. second cycle siRNA transfection performed                    Time-lapse microscopy. Time-lapse microscopy performed use Cell <br /> 24 h later. Cells harvested 72 h cycle transfection stored   R imaging station live-cell microscopy (Olympus) mounted IX81 &#226;&#171;&#186;80&#194;&#176;C lysis <span id='am-62' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-63' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> buffer (8 M urea, 0.1 M NaH2PO4, 0.01 M Tris,               inverted microscope (Olympus) equipped incubation chamber (Evotec). <br /> pH 8).                                                                                   includes Hamamatsu ORCA-ER charge-coupled-device camera <br />  4786       SENESE ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. siRNA-mediated knockdown HDAC1, HDAC2, HDAC3 U2OS, MCF7, MCF10A cells. loss HDAC1 HDAC2 produced hyperacetylation histones H3 H4.  C) Relative (rel.) expression levels indicated mRNAs determined real-time PCR, described Materials Methods. Averages independent experiments shown, standard deviations indicated error bars. (D F) Western blots U2OS cell extracts transfection siRNA HDAC1, HDAC2, HDAC3, HDAC1 plus HDAC2. Cells transfected processed described Materials Methods, 30 &#226;?&#174;g loaded lane. control <br /> (CTRL), used transfection antiluciferase siRNA. <br />  <br />  <br />  image acquisition. movie montage performed using ImageJ image            establish arbitrary threshold used discriminate HDAC1-positive (gate II) <br /> analysis software (W. Rasband, National Institutes Health, Bethesda, MD).       HDAC1-negative (gate  cells. H3-P fluorescence distribution evalu- <br />    Statistical immunofluorescence. hours siRNA transfection,     ated gate gate II fix value cells H3-P positive cells plated chamber slides pretreated 15 &#226;?&#174;g/ml poly(D)lysine    negative. evaluated gate gate II (Sigma) treated 0.1 &#226;?&#174;g/ml nocodazole 16 h enrich mitotic       percentages double positive cells. <br /> population. Slides fixed 4% paraformaldehyde&#226;&#8364;&#8220;1&#226;&#171;&#187; PIPES [piperazine-            Extraction RNA reverse transcription-PCR. validation mi- <br /> N,N&#226;&#172;&#732;-bis(2-ethanesulfonic acid)] 10 min, permeabilized 0.1% Triton        croarray results, employed independent RNA preparations samples <br /> X-100 0.2% bovine serum albumin PBS, processed using standard           described  Total RNA isolated using QIAGEN RNeasy Protect <br /> protocols. primary antibodies used anti-HDAC1 monoclonal (Upstate)        mini kit described manufacturer. cDNA generated reverse 1:250 anti-H3-P rabbit polyclonal (Upstate) 1:100, conjugated    transcription-PCR PE Applied Biosystem TaqMan reverse transcription <br /> secondary antibodies Cy3-conjugated goat anti-mouse (Jackson Laborato-        reagents. Relative levels specific mRNA determined 5&#226;&#172;&#732; nuclease <br /> ries) FITC-conjugated goat anti-rabbit (Sigma). DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamino-2-         assay (TaqMan) chemistry  PCRs performed ABI <br /> phenylindole) (Sigma) used nuclear counterstaining. Fluorescence mi-       7900HT sequence detection  GAPDH (glyceraldehyde-3-phosphate <br /> croscopy images acquired using BX61 (Olympus) motorized fluorescence        dehydrogenase) gene used control gene normalization. <br /> microscope. Acquired images visualized using browser developed        Microarray analysis. Biotin-labeled cRNA targets obtained 5 &#226;?&#174;g ImageJ macro language. acquisition DAPI, Cy3, FITC channels,          total RNA derived samples described  cDNA synthesis  <br /> DAPI staining employed identify cell nuclei build mask         formed Gibco SuperScript custom cDNA synthesis kit, biotin-labeled <br /> successive calculations. mean nuclear levels DAPI, HDAC1, H3-P          antisense RNA transcribed using vitro transcription (Ambion, stored memory allow later statistical analysis. HDAC1 fluores-      , Austin, TX) including Bio-11-<span id='am-16' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span> Bio-11-<span id='am-46' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CTP</span> reaction (NEN <br /> cence distribution evaluated control- HDAC1-interfered cells order   Life Sciences, Boston, MA). GeneChip hybridization, washing, staining,  VOL. 27, 2007                                                   ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                                4787 <br />  <br />  <br />  <br />  <br />   FIG. 2. Absence HDAC1 HDAC3 causes defective proliferation.  C) Growth curves U2OS, MCF7, MCF10A transfected HDAC siRNAs. increase absorbance 595 nm directly correlated cell proliferation, described Materials Methods. curve represents average independent experiments, standard deviation indicated error bars. (D F) CFU assay U2OS, <br /> MCF7, MCF10A cells transfected HDAC siRNAs. ratio CFU number sample/CFU number control calculated. column represents average independent experiments, respective standard deviations. CTRL, control. <br />  <br />  <br />  <br /> scanning performed according Affymetrix protocols (Santa Clara, CA).   lated function HDAC1 prevent inappropriate ex- copies HG-U133Plus GeneChip hybridized target.       pression cyclin-dependent kinase (CDK) inhibitors p21 details microarray methods described supplementary data <br /> posted http://bio.ifom-ieo-campus supplementary/MCB49407/. <br />                                                                                p27 (37). accordance model, promoters                                                                                genes p21 p27 hyperacetylated HDAC1 null <br />                                                                                cells (37). addition, HDAC1 recruited p21 <br />                                RESULTS                                         gene transcription factor SP1 (36) Tbx2 (51)    siRNA-mediated knockdown HDAC1 HDAC3. es-                          p21 genes upregulated HDAC inhibitor <br /> tablished methodology HDAC class knockdown                        treatment (29, 30, 47, 48).  analyzed endogenous <br /> various cell lines: U2OS  osteosarcoma cell line), MCF7                   p21 protein levels RNAi treatment. Figure 1D indicates <br /> mammary carcinoma cell line), MCF10A  epithelial                        p21 levels increase absence HDAC1 mammary gland cell line). shown Fig. 1A F,                        HDAC1 plus HDAC2 U2OS cells, accordance expression levels HDACs efficiently knocked                      HDAC1 null embryo results (37) effect HDAC respective siRNAs. interesting note                  inhibitors (47). Conversely, MCF7 MCF10A, levels endogenous HDAC1 HDAC2 cell                     did observe changes p21 levels HDAC1 lines increase opposition (Fig. 1D F), e.,                HDAC1 HDAC2 knockdown (data shown).  knockdown increases protein                   ingly, absence HDAC1 HDAC2 did <br /> level. hand, observed decrease                       Western blot analysis modification-specific antibodies  <br /> HDAC1 HDAC2 HDAC3 knockdown,                              veal increased acetylation levels subset histones H3 protein level. know reason                    H4 (Fig. 1D F) cell lines. result result, preliminary data indicate proteasome                 indicate histones substrates HDACs <br /> pendent.  reduced expression HDAC1                         concurrent double knockdown HDAC1 <br /> HDAC2 consequence reduced cellular                      plus HDAC2 leads hyperacetylation. noted <br /> viability HDAC3 knockdown (Fig. 2).                                      specific knockdown HDAC did    absence HDAC1 induces p21 expression                     affect basal levels p53 acetylation (data shown), <br /> U2OS cell line, absence HDAC1 HDAC2                        reflecting idea multiple deacetylases involved increases histone acetylation. important proliferation                  regulating p53 acetylation (28, 39, 52). <br />  4788     SENESE ET AL.                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />    siRNA-mediated knockdown HDAC1 HDAC3 dimin-                 panel Fig. 4A represents biparametric anal- <br /> ishes cellular viability proliferation. known          ysis, using anti-H3-P PI staining levels U2OS cells. <br /> HDAC inhibitors lead p21 induction. induce cell          analysis shows increase G1 cells absence cycle arrest G1 G2/M apoptosis, killing tumor        HDAC1 time points thymidine release, accompa- <br /> cells (4, 29, 48). pathway leading tumor cell death   nied lower percentage mitotic cells. <br /> unknown, unlikely single molecular pathway          released U2OS cells double thymidine identified cell types HDAC inhibitors.      block nocodazole-containing culture medium enrich <br /> decided investigate cellular viability proliferation        mitotic population. Biparametric analysis using anti-cyclin <br /> U2OS, MCF7, MCF10A cells HDAC knockdown.                     G2 marker, PI staining (Fig. 4B) showed 24 h <br /> Clearly, absence HDAC1, HDAC3, HDAC1 plus                  control cells blocked mitosis expected HDAC2 generated remarkable defect proliferation          HDAC1 knockdown cells unable remain mitosis. U2OS MCF7 cell lines (Fig. 2A B)            fact, time, larger G1/S HDAC1- <br /> unnoticeable effect MCF10A cell line (Fig. 2C). lack       negative cells, accompanied significant decrease mitotic HDAC2 pronounced effect cells (Fig.         cells increase sub-G1 cells. <br /> 2A C). inconsistent published data prob-          analyzed proportion sub-G1 cells  <br /> ably different cell lines used (24). analyzed      lease nocodazole block. shown Fig. 4C, effect absence HDACs life spans        observed massive apoptosis HDAC1-negative U2OS cells. cell lines. correlation results Fig. 2A   Taken  data demonstrate proliferative C, cells lacking HDAC1, HDAC3, combination                  defect HDAC1 knockdown cells partial G1 <br /> HDAC1 plus HDAC2 diminished capacity form col-               arrest cell death G2/M transition onies plated low density (Fig. 2D F). data         beginning mitosis. Alternatively, HDAC1-deficient <br /> suggest HDAC1  combination          cells insensitive nocodazole grow HDAC2) HDAC3 important roles prolifera-               G1 tetraploids.  followed living cells time- <br /> tion tumor cells, agreement published data indicating      lapse microscopy, observed inability HDAC1-nega- relevant role murine HDAC1 regulation prolif-         tive cells condense enter mitosis, suggesting eration development (37). HDAC3-AKAP95/HA95-                   cells die G2/M transition (Fig. 4D). data confirm <br /> Aurora B pathway regulating mitosis demonstrated             extend published findings showing HDAC inhibitors <br /> recently, explaining mitotic defect observed HDAC3          induce G1 G2/M arrest apoptosis (4). <br /> siRNA (38). interesting note cell line              Induction apoptosis HDAC1-deficient U2OS cells. MCF10A, untransformed counterpart MCF7, ab-                observation differences numbers apop- <br /> sence HDACs effect cellular prolifer-        totic cells HDAC1 knockdown prompted confirm <br /> ation.                                                                apoptotic cell death examining caspase-3 activation,    HDAC1 knockdown cells marked reduction H3                FACS analysis (Fig. 5A) Western blot analysis (Fig. 5B) <br /> S10 phosphorylation. H3-P mitotic marker. explore              Materials Methods). results indicate defect proliferation HDAC1 knockdown                caspase-3 activation involved apoptosis HDAC1 <br /> cells mitotic defect, applied single-cell statis-     knockdown. <br /> tical analysis control-interfered HDAC1-inter-               Comparison gene expression profiles depletion fered cells using anti-HDAC1 antibody anti-H3-P          HDAC1, HDAC2, HDAC3, HDAC1 plus HDAC2. antibody  Materials Methods). analyzed           analyzed gene expression profiles U2OS cells loss <br /> cell lines mitotic cells presence absence           HDAC1, HDAC2, HDAC3, HDAC1 plus HDAC2 HDAC1 (Fig. 3). Figure 3A, B, C histograms report-           using HG-U133 Plus Affymetrix oligonucleotide chips, ing distributions HDAC1 levels control-interfered           explore expression approximately 54,000 human tran- <br /> populations HDAC1-interfered populations, described            scripts. Results analyzed using GCOS elab- Materials Methods. shown upper panels,            orated GenePicker GeneSpring software  able discriminate HDAC1-positive cells (gate          supplementary data posted http://bio.ifom-ieo-campus II) HDAC1-negative cells (gate . middle panels              /supplementary/MCB49407/) (2, 10). detailed description represent distributions H3-P levels gates        microarray methods, data analysis elaboration, com- <br /> order define basal level H3-P-positive cells.        parative results supplementary data lower panels, correlation HDAC1 H3-P levels           URL mentioned  gates represented. Interestingly, knockdown             Ablation HDAC1, HDAC2, HDAC3, HDAC1 plus <br /> HDAC1 led clear reduction number H3-P-                 HDAC2 resulted altered expression 987, 283, 1,317, <br /> positive cells, suggesting HDAC1 important          601 genes, respectively  supplementary Table 1 posted <br /> proper cell cycle progression.  effect           http://bio.ifom-ieo-campus supplementary/MCB49407/). <br /> drastic MCF10A cell line (Fig. 3C).                            large degree overlap lists genes <br />    HDAC1 knockdown cells arrest G1 die               regulated siRNA HDAC1 regulated siRNA G2/M transition. tested HDAC1                  HDAC1 plus HDAC2. Ablation HDAC2 results mild <br /> knockdown affects cell cycle progression. U2OS cells             phenotype  compared conditions, associated <br /> synchronized double thymidine block G1/S transi-          deregulated expression genes, 86% tion, cell cycle progression removal thymidine        deregulated mentioned  <br /> monitored FACS analysis PI-stained DNA (Fig. 4A, .          ditions. 31% genes deregulated HDAC3 siRNA  VOL. 27, 2007                                              ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                                 4789 <br />  <br />  <br />  <br />  <br />   FIG. 3. Absence mitotic marker H3-P HDAC1-deficient cells. U2OS, MCF-7, MCF10A cells analyzed presence H3-P.   Distribution HDAC1 signal siRNA-control (CTRL)-transfected siRNA-HDAC1-transfected populations. dashed line represents arbitrary threshold used distinguish HDAC1-negative (gate  HDAC1-positive (gate II) cells. (Middle) <br /> Distribution H3-P signals gate gate II. dashed line represents arbitrary threshold used distinguish positive negative H3-P cells.   Scatter plot HDAC1 signal versus H3-P signal. percentages inside plot indicate percentages double <br /> positive cells. <br />  <br />  common regulated siRNA HDAC1                        deregulated knockdown HDAC1 HDAC1 plus <br /> HDAC1 plus HDAC2  supplementary data sup-                      HDAC2, according Gene Ontology Biological Process cat- <br /> plementary Table 1 posted URL mentioned .                     egories  complete description, supplementary <br /> Hierarchical clustering using lists regulated genes          data posted http://bio.ifom-ieo-campus supplementary <br /> confirms higher degree similarity gene                /MCB49407/). investigate molecular basis HDAC1- <br /> expression profiles resulting ablation HDAC1, HDAC2,               mediated growth arrest, focused attention func- HDAC1 plus HDAC2 resulting ablation                  tional categories: apoptosis proliferation. 36 HDAC3 (Fig. 6).                                                         probe sets (corresponding 28 nonredundant genes)   performed functional classification genes                   classified relevant apoptosis 100 probe sets <br />  4790     SENESE ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. HDAC1-deficient cells arrest G1 die G2/M transition.   Asynchronous   siRNA-control (CTRL)-transfected (black) siRNA-HDAC1-transfected (gray) U2OS cells blocked G1/S transition use thymidine released fresh medium.   <br /> DNA content analyzed FACS analysis PI-stained cells different time points release fresh medium.   Biparametric <br /> FACS analysis using anti-H3-P PI staining performed calculate percentages cells G1, S, G2, M phases time point. <br /> (B) U2OS control- HDAC1-interfered cells blocked G1/S transition use thymidine released medium containing <br /> nocodazole.   DNA content analyzed FACS analysis PI-stained cells different time points. (Middle) Biparametric FACS analysis <br />  VOL. 27, 2007                                                ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                                   4791 <br />  <br />  <br />  <br />  <br />    FIG. 5. HDAC1 knockdown cells undergo apoptosis.   Effect siRNA HDAC1 caspase-3 activation FACS analysis. bars <br /> represent percentages cleaved caspase-3-positive cells control <br /> (CTRL) siRNA-HDAC1-transfected U2OS cells 96 h. values correspond averages independent experiments, error bars indicate associated standard deviations. <br /> (B) Western blots U2OS cell extracts transfection control siRNA HDAC1. Cells collected 120 h transfec- <br /> tion, 50 &#226;?&#174;g lysates loaded lane. control, used injection antiluciferase siRNA. anti-caspase-3 antibody used able recognize length inactive cleaved active <br /> forms. <br />  <br />  <br /> (corresponding 72 nonredundant genes) relevant prolif-                   FIG. 6. Gene expression profiles. Supervised hierarchical clustering <br /> eration. genes present categories (F2R,                 (according Pearson&#226;&#8364;&#8482;s correlation) performed using comparative <br />                                                                            expression values gene lists indicated dendrogram  interleukin-1A [IL1A], SPP1 genes). Figure 7 shows                   gene lists, supplementary Table 1 posted http://bio.ifom-ieo-campus <br /> graphical representation behavior genes                  supplementary/MCB49407/). cases, expression value U2OS cells lacking HDAC1 HDAC1 plus HDAC2 com-                          control cells used baseline (value 1). <br />  <br />  <br />  <br />  <br /> using anti-cyclin (CYCA) PI staining performed calculate percentages cells G1/S, G2, M phases time point. <br />   DNA content analyzed FACS analysis evaluate percentages sub-G1 cells. (C) U2OS control- HDAC1-interfered <br /> cells blocked G2/M transition use nocodazole released fresh medium.   DNA content analyzed FACS analysis PI-stained cells different time points.   DNA content analyzed FACS analysis evaluate percentages sub-G1 cells. (D) <br />   Percentages cells control- HDAC1-interfered population able undergo mitosis 30 h.   Images time-lapse experiments U2OS siRNA-CTRL-transfected siRNA-HDAC1-transfected cells. Control cells experience normal mitosis <br /> (white arrowheads), HDAC1 knockdown cells enter mitosis die (yellow black/white arrowheads). <br />  4792     SENESE ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Gene expression profiles quantitative real-time PCR.   Graphic representation expression patterns genes functionally <br /> involved apoptosis proliferation regulated ablation HDAC1 HDAC1 plus HDAC2 U2OS cells. (B) Validation microarray data TaqMan quantitative real-time PCR. error bars indicate standard deviations independent experiments. CTRL, control. <br />  <br />  <br />  <br />  <br /> pared behavior vector-transfected U2OS cells                supplementary/MCB49407/. verify changes gene <br /> trol). Clearly, genes involved apoptosis induced            expression detected microarray analysis, using repressed RNAi HDAC1 HDAC1 plus                        independent RNA preparations described experimental <br /> HDAC2. Multiple proapoptotic genes, including FAS, BCL2-                 conditions, performed quantitative real-time PCR analysis <br /> like 1, members tumor necrosis factor (TNF) family            19 genes expression altered regulate <br /> receptors ligands (TNFRSF10B, TNFRSF11B, TNF                     proliferation apoptosis (cyclin D2 [CCND2], CCNE2, <br /> RSF12A), activated. Conversely, approximately equal                 CDKN1A [p21], HDAC9, IL6R, 9, MYC, ubiquitin-specific <br /> numbers genes involved cell proliferation              peptidase 16 [USP16], CCNG2, cell division cycle 25C [CDC25C], <br /> downregulated. complete gene lists sup-              CDKN2C, nuclear factor 1A [NF1A], NFIB, septin 6 [SEP6], <br /> plementary Tables 1 2 posted http://bio.ifom-ieo-campus           transforming growth factor &#226;?&#164;2 [TGFB2], TNFRSF10B, <br />  VOL. 27, 2007                                            ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                        4793 <br />  <br />  <br /> TNFRSF11B, 24, IL1A genes). shown Fig. 7,          lack mitotic cells absence HDAC1 (Fig. 3 high correlation microarray real            4). result implies HDAC1 essential mitosis, -time data 19 genes.                                          undescribed observation. expected published data <br />    consistency recent study identi-         showing HDAC6 major tubulin deacetylase fying genes apoptosis cell proliferation      HDAC1 specific tubulin deacetylase activity (25, 57), pathways regulated different HDAC inhibitors            did observe changes tubulin acetylation following <br /> 16-h culture treatment (46). siRNA experi-              siRNA-mediated knockdown HDAC1 HDAC3 (data ments difficult discern early late events,    shown). cells interfered HDAC1 nuclear interesting note study identified  exam-     appearance typical interphase apoptosis. Taken  <br /> ple, cell division cycle gene CDC25C genes encoding           data suggest human tumor cells lacking HDAC1 <br /> TNF family receptors ligands modulated               enter mitosis undergo apoptosis HDAC inhibitors (46). Altogether, findings support          caspase-3 activation. Future work characterize idea HDAC inhibition affects expression genes             mechanism mitotic arrest. data   consistent pathways regulate tumor cell growth survival.         studies pointing importance histone deacetyla- <br />                                                                       tion formation pericentric heterochromatin (9, 50) <br />                                                                       showing loss component Sin3/HDAC1/ <br />                          DISCUSSION <br />                                                                       HDAC2 complex leads cell death missegregation    HDAC inhibitors induce cell cycle arrest, apoptosis, dif-       chromosomes mitosis (8). Interestingly, loss HDAC1 <br /> ferentiation tumor cells, number entered        untransformed cells, ES cells, fibroblasts, neu- <br /> early-phase clinical trials. Based phenotype HDAC1-         rons, does induce apoptosis (37; G. Lagger, R. Grausen- <br /> deficient embryos ES cells, HDAC1 implicated               burger, C. Seiser, unpublished data), proliferation control (37). study, HDAC1         possible checkpoint activation cells compensation important role proliferation human tumor cells       HDAC2. influences capacity undergo mitosis. Al-            Based microarray results, analysis global gene study shows absence HDAC3           expression profiles U2OS cells absence HDAC1, <br /> drastic effect cell proliferation, directed attention       HDAC2, HDAC3, HDAC1 plus HDAC2 yielded large HDAC1 knockdown phenotype. Clearly,                 number genes present altered expression levels  microarray data indicate HDAC3 favors            supplementary data posted http://bio.ifom-ieo-campus uniquely expressed genes, compared HDAC1 HDAC1                 /supplementary/MCB49407/). HDAC1 HDAC1 plus HDAC2 <br /> plus HDAC2 (Fig. 6)  supplementary Table 1 posted            knockdown regulated largely overlapping set genes,  http://bio.ifom-ieo-campus supplementary/MCB49407/),            cluding regulated HDAC2 knockdown. <br /> suggesting different specific pathways action. Recent          HDAC3 knockdown caused deregulation vast number <br /> work clearly shown HDAC3 novel HDAC             genes. Interestingly, 50% genes induced HDAC3 <br /> pathway regulating mitosis Aurora B kinase activity modu-          knockdown induced ablation HDAC1 lation (38). aim study define role   HDAC1 plus HDAC2 expression, 21% genes HDAC1 tumor cells  parallel, provide molecular        repressed HDAC3 knockdown common insight HDAC1 functions conducting global gene              conditions. results consequence expression study U2OS cell line. known          protein reduction HDAC1 HDAC2 observed effects HDAC inhibitors cell line specific          HDAC3 knockdown (Fig. 1). <br /> HDAC inhibitor selective (14), providing ground detailed              analyzed greater genes regulated studies HDAC responsible phenotype           HDAC1. accordance proposed role HDAC1 cell line  demonstrated     transcriptional repressor, HDAC1 target genes expression profiles induced HDAC inhibitors cell line          upregulated downregulated absence HDAC1. <br /> dependent  expression patterns HDACs            Interestingly, result evened HDAC1 vary cell line cell line, anticipate different         HDAC2 ablated. line reports effects knockout particular HDAC (4, 14).              demonstrated independent recruitment HDAC1 <br /> U2OS MCF7 cells, absence HDAC1 remark-               HDAC2 target genes (26, 49). <br /> able effect apoptosis proliferation phe-            phenotype observed absence notype exacerbated simultaneous lack HDAC2.             HDAC1  HDAC1 plus HDAC2), validated genes  <br /> Interestingly, MCF10A, untransformed counterpart           volved apoptosis proliferation. Notably, number MCF7 cells, sensitive lack HDAC1. fact,         cyclins, cyclin-dependent kinases, cyclin-dependent ki- effects proliferation cell cycle          nase inhibitor p21, expression tightly coordinated evident (Fig. 2 3). results agreement       regulate appropriate cell cycle progression, induced idea HDAC inhibitors antiproliferative proapop-         RNAi treatment HDAC1. p21 regarded totic properties mainly transformed cells (4, 41).         attractive therapeutic target human cancer. identified caspase-3 mediator apoptosis provoked       plethora data regarding role multifunctional absence HDAC1. Caspase-3 proapoptotic enzyme              protein cellular pathways emerged, precise triggers apoptosis intrinsic death       mechanism p21 regulates cell cycle progression ligand pathways (20, 21). good apoptosis       unknown, partly p21 act positive indicator.  striking effect observed          negative fashion cell proliferation (53). Given p21 <br />  4794      SENESE ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> expression increased HDAC1 knockdown U2OS                                 10. Finocchiaro, G., P. Parise, S. P. Minardi, M. Alcalay, H. Muller. 2004. <br />                                                                                          GenePicker: replicate analysis Affymetrix gene expression microarrays. <br /> cells MCF7 cells, argue mitotic                             Bioinformatics 20:3670&#226;&#8364;&#8220;3672. <br /> phenotype observed tumor cell lines linked                       11. Fischle, W., F. Dequiedt, M. J. Hendzel, M. G. Guenther, M.  Lazar, W. p21. microarray data showed repression                                    Voelter, E. Verdin. 2002. Enzymatic activity associated class II <br />                                                                                          HDACs dependent multiprotein complex containing HDAC3 CDC25C septin 6. Saccharomyces cerevisiae, septins                            SMRT/N-CoR. Mol. Cell 9:45&#226;&#8364;&#8220;57. family cell division cycle regulatory proteins                     12. Gao, L., M.  Cueto, F. Asselbergs, P. Atadja. 2002. Cloning associated actin stress fibers interphase cells,                        functional characterization HDAC11, novel member human <br />                                                                                          histone deacetylase family. J. Biol. Chem. 277:25748&#226;&#8364;&#8220;25755. <br /> cleavage furrow dividing cells, bud neck budding                       13. Glaser, K. B., J. Li, M. J. Staver, R. Q. Wei, D. H. Albert, S. K. Davidsen. <br /> yeast (reviewed reference 31). human homologue                                 2003. Role class class II histone deacetylases carcinoma cells using <br />                                                                                          siRNA. Biochem. Biophys. Res. Commun. 310:529&#226;&#8364;&#8220;536. <br /> septin 6 fused MLL cases                         14. Glaser, K. B., M. J. Staver, J. F. Waring, J. Stender, R. G. Ulrich, S. K. <br /> infant acute myeloid leukemia (43). Cdc25C phosphatase                              Davidsen. 2003. Gene expression profiling multiple histone deacetylase <br /> major cell cycle regulator mammals. C-terminal catalytic                          (HDAC) inhibitors: defining common gene set produced HDAC inhi- <br />                                                                                          bition T24 MDA carcinoma cell lines. Mol. Cancer Ther. 2:151&#226;&#8364;&#8220;163. <br /> phosphatase domain dephosphorylates Cdc2 promote G2/M                             15. Gregoretti,  V., Y. M. Lee, H. V. Goodson. 2004. Molecular evolution <br /> transition (27), N-terminal regulatory domain                            histone deacetylase family: functional implications phylogenetic <br /> tains multiple phosphorylation sites. Multisite phosphorylation                          analysis. J. Mol. Biol. 338:17&#226;&#8364;&#8220;31. <br />                                                                                      16. Grozinger, C. M., S. L. Schreiber. 2002. Deacetylase enzymes: biological Cdc25C different kinases recruits specific proteins,                           functions use small-molecule inhibitors. Chem. Biol. 9:3&#226;&#8364;&#8220;16. <br /> trolling normal cellular mitotic progression (reviewed                        17. Gurley, L. R., R.  Walters, R.  Tobey. 1973. Histone phosphorylation <br /> reference 56). Clearly, Septin6 Cdc25C impor-                              late interphase mitosis. Biochem. Biophys. Res. Commun. 50:744&#226;&#8364;&#8220; <br />                                                                                          750. <br /> tant roles mitosis. Finally, activation interleu-                     18. Halkidou, K., L. Gaughan, S. Cook, H. Y. Leung, D. E. Neal, C. N. <br /> kins HDAC ablation postapoptotic                             Robson. 2004. Upregulation nuclear recruitment HDAC1 hormone <br />                                                                                          refractory prostate cancer. Prostate 59:177&#226;&#8364;&#8220;189. <br /> activation inflammation response.                                              19. Hauser, C., B. Schuettengruber, S. Bartl, G. Lagger, C. Seiser. 2002. <br />    summary, defined role HDAC1                                   Activation mouse histone deacetylase 1 gene cooperative histone <br /> proliferation human tumor cells  importantly,                                phosphorylation acetylation. Mol. Cell. Biol. 22:7820&#226;&#8364;&#8220;7830. <br />                                                                                      20. Henderson, C., C. Brancolini. 2003. Apoptotic pathways activated identified mitotic defect human tumoral cells knock-                          histone deacetylase inhibitors: implications drug-resistant phenotype. HDAC1. results confirm impor-                               Drug Resist. Updates 6:247&#226;&#8364;&#8220;256. <br /> tance characterizing biological function HDAC1,                            21. Henderson, C., M. Mizzau, G. Paroni, R. Maestro, C. Schneider, C. <br />                                                                                          Brancolini. 2003. Role caspases, Bid, p53 apoptotic response light potential significance target cancer                       triggered histone deacetylase inhibitors trichostatin (TSA) sub- <br /> therapy.                                                                                 eroylanilide hydroxamic acid (SAHA). J. Biol. Chem. 278:12579&#226;&#8364;&#8220;12589. <br />                                                                                      22. Hendzel, M. J., Y. Wei, M.  Mancini,  Van Hooser, T. Ranalli, B. R. <br />                          ACKNOWLEDGMENTS                                                 Brinkley, D. P. Bazett-Jones, C. D. Allis. 1997. Mitosis-specific phos- <br />                                                                                          phorylation histone H3 initiates primarily pericentromeric hetero- <br />   grateful use IFOM-IEO Campus facilities                      chromatin G2 spreads ordered fashion coincident thank members laboratories. especially thank Luca                          mitotic chromosome condensation. Chromosoma 106:348&#226;&#8364;&#8220;360. <br /> Giorgetti, Roberta Carbone (Sr.), Anna Antoni, Mario Faretta,                 23. Hsu, J. Y.,  W. Sun, X. Li, M. Reuben, K. Tatchell, D. K. Bishop, J. M. <br />                                                                                          Grushcow, C. J. Brame, J.  Caldwell, D. F. Hunt, R. Lin, M. M. Smith, Saverio Minucci experimental suggestions. thank Manuelo <br />                                                                                          C. D. Allis. 2000. Mitotic phosphorylation histone H3 governed Malizia Stefano Leva technical help.                                             Ipl1/aurora kinase Glc7/PP1 phosphatase budding yeast nema- <br />   work supported grants Associazione Italiana                       todes. Cell 102:279&#226;&#8364;&#8220;291. la Ricerca sul Cancro (AIRC) S.C., GEN-AU program                    24. Huang, B. H., M. Laban, C. H. Leung, L. Lee, C. K. Lee, M. Salto-Tellez, Austrian Ministry Science C.S., Italian                        G. C. Raju, S. C. Hooi. 2005. Inhibition histone deacetylase 2  <br /> Ministry Health.                                                                      creases apoptosis p21Cip1/WAF1 expression, independent histone <br />                                                                                          deacetylase 1. Cell Death Differ. 12:395&#226;&#8364;&#8220;404. <br />                                  REFERENCES                                          25. Hubbert, C.,  Guardiola, R. Shao, Y. Kawaguchi,  Ito,  Nixon, M. <br /> 1. Ahringer, J. 2000. NuRD SIN3 histone deacetylase complexes devel-              Yoshida, X. F. Wang, T. P. Yao. 2002. HDAC6 microtubule-asso- <br />    opment. Trends Genet. 16:351&#226;&#8364;&#8220;356.                                                     ciated deacetylase. Nature 417:455&#226;&#8364;&#8220;458. <br /> 2. Alcalay, M., N. Meani, V. Gelmetti,  Fantozzi, M. Fagioli,  Orleth, D.        26. Humphrey, G. W., Y. Wang, V. R. Russanova, T. Hirai, J. Qin, Y. Nakatani, <br />    Riganelli, C. Sebastiani, E. Cappelli, C. Casciari, M. T. Sciurpi,  R.              B. H. Howard. 2001. Stable histone deacetylase complexes distinguished <br />    Mariano, S. P. Minardi, L. Luzi, H. Muller, P. P. Di Fiore, G. Frosina,           presence SANT domain proteins CoREST/kiaa0071 Mta-L1. <br />    P. G. Pelicci. 2003. Acute myeloid leukemia fusion proteins deregulate genes          J. Biol. Chem. 276:6817&#226;&#8364;&#8220;6824. <br />    involved stem cell maintenance DNA repair. J. Clin. Investig. 112:         27. Hutchins, J. R., P. R. Clarke. 2004. fingers mitotic trigger: <br />    1751&#226;&#8364;&#8220;1761.                                                                            post-translational regulation Cdc25C phosphatase. Cell Cycle 3:41&#226;&#8364;&#8220;45. <br /> 3. Bartl, S., J. Taplick, G. Lagger, H. Khier, K. Kuchler, C. Seiser. 1997.      28. Ito, , Y. Kawaguchi, C. H. Lai, J. J. Kovacs, Y. Higashimoto, E. Appella, <br />    Identification mouse histone deacetylase 1 growth factor-inducible            T. P. Yao. 2002. MDM2-HDAC1-mediated deacetylation p53    gene. Mol. Cell. Biol. 17:5033&#226;&#8364;&#8220;5043.                                                  required degradation. EMBO J. 21:6236&#226;&#8364;&#8220;6245. <br /> 4. Bolden, J. E., M. J. Peart, R. W. Johnstone. 2006. Anticancer activities      29. Johnstone, R. W. 2002. Histone-deacetylase inhibitors: novel drugs    histone deacetylase inhibitors. Nat. Rev. Drug Discov. 5:769&#226;&#8364;&#8220;784.                  treatment cancer. Nat. Rev. Drug Discov. 1:287&#226;&#8364;&#8220;299. <br /> 5. Choi, J. H., H. J. Kwon, B.  Yoon, J. H. Kim, S. U. Han, H. J. Joo, D. Y.   30. Johnstone, R. W., J. D. Licht. 2003. Histone deacetylase inhibitors    Kim. 2001. Expression profile histone deacetylase 1 gastric cancer              cancer therapy: transcription primary target? Cancer Cell 4:13&#226;&#8364;&#8220;18. <br />    tissues. Jpn. J. Cancer Res. 92:1300&#226;&#8364;&#8220;1304.                                        31. Kartmann, B., D. Roth. 2001. Novel roles mammalian septins: 6. Cobb, J., M. Miyaike,  Kikuchi, M.  Handel. 1999. Meiotic events           vesicle trafficking oncogenesis. J. Cell Sci. 114:839&#226;&#8364;&#8220;844. <br />    centromeric heterochromatin: histone H3 phosphorylation, topoisomer-          32. Kawai, H., H. Li, S. Avraham, S. Jiang, H. K. Avraham. 2003. Overex- <br />    ase II alpha localization chromosome condensation. Chromosoma 108:                pression histone deacetylase HDAC1 modulates breast cancer progres- <br />    412&#226;&#8364;&#8220;425.                                                                              sion negative regulation estrogen receptor alpha. Int. J. Cancer 107: <br /> 7. Cress, W. D., E. Seto. 2000. Histone deacetylases, transcriptional            353&#226;&#8364;&#8220;358. <br />    trol, cancer. J. Cell Physiol. 184:1&#226;&#8364;&#8220;16.                                      33. Khochbin, S.,  Verdel, C. Lemercier, D. Seigneurin-Berny. 2001. <br /> 8. David, G., G. M. Turner, Y. Yao,  Protopopov, R.  DePinho. 2003.              Functional significance histone deacetylase diversity. Curr. Opin. Genet. <br />    mSin3-associated protein, mSds3, essential pericentric heterochroma-           Dev. 11:162&#226;&#8364;&#8220;166. <br />    tin formation chromosome segregation mammalian cells. Genes Dev.           34. Kim, J. H., Y. K. Choi, H. J. Kwon, H. K. Yang, J. H. Choi, D. Y. Kim. <br />    17:2396&#226;&#8364;&#8220;2405.                                                                         2004. Downregulation gelsolin <span id='am-26' about='obo:IMR_0100041' typeof='owl:Thing obo:IMR_0000877 obo:IMR_0001656 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001'><span id='am-27' property="oboInOwl:hasExactSynonym" content="Retin A" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="vitamin A acid" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Tretinoin" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Retin A" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="all-trans-Vitamin A acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0100041" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="all-trans-Vitamin A acid" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="vitamin A acid" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="retinoic acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00777" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Tretinoin" datatype="xsd:string"></span>retinoic acid</span> receptor beta expression <br /> 9. Ekwall, K., T. Olsson, B. M. Turner, G. Cranston, R. C. Allshire. 1997.           gastric cancer tissues histone deacetylase 1. J. Gastroenterol. <br />    Transient inhibition histone deacetylation alters structural func-         Hepatol. 19:218&#226;&#8364;&#8220;224. <br />    tional imprint fission yeast centromeres. Cell 91:1021&#226;&#8364;&#8220;1032.                   35. Kramer, O. H., P. Zhu, H. P. Ostendorff, M. Golebiewski, J. Tiefenbach, <br />  VOL. 27, 2007                                                           ROLE HDAC1 HUMAN TUMOR CELL PROLIFERATION                                            4795 <br />  <br />       M.  Peters, B. Brill, B. Groner,  Bach, T. Heinzel, M. Gottlicher.        47. Richon, V. M., T. W. Sandhoff, R.  Rifkind, P.  Marks. 2000. Histone <br />       2003. histone deacetylase inhibitor valproic acid selectively induces             deacetylase inhibitor selectively induces p21WAF1 expression gene- <br />       proteasomal degradation HDAC2. EMBO J. 22:3411&#226;&#8364;&#8220;3420.                               associated histone acetylation. Proc. Natl. Acad. Sci. USA 97:10014&#226;&#8364;&#8220;10019. <br /> 36.   Lagger, G.,  Doetzlhofer, B. Schuettengruber, E. Haidweger, E. Simboeck,        48. Richon, V. M., Y. Webb, R. Merger, T. Sheppard, B. Jursic, L. Ngo, F. Civoli, <br />       J. Tischler, S. Chiocca, G. Suske, H. Rotheneder, E. Wintersberger, C.            R. Breslow, R.  Rifkind, P.  Marks. 1996. Second generation hybrid <br />       Seiser. 2003. tumor suppressor p53 histone deacetylase 1                  polar compounds potent inducers transformed cell differentiation. <br />       antagonistic regulators cyclin-dependent kinase inhibitor p21/WAF1/            Proc. Natl. Acad. Sci. USA 93:5705&#226;&#8364;&#8220;5708. <br />       CIP1 gene. Mol. Cell. Biol. 23:2669&#226;&#8364;&#8220;2679.                                         49. Sasaki, S., L.  Lesoon-Wood,  Dey, T. Kuwata, B. D. Weintraub, G. <br /> 37.   Lagger, G., D. O&#226;&#8364;&#8482;Carroll, M. Rembold, H. Khier, J. Tischler, G. Weitzer, B.           Humphrey, W. M. Yang, E. Seto, P. M. Yen, B. H. Howard, K. Ozato. <br />       Schuettengruber, C. Hauser, R. Brunmeir, T. Jenuwein, C. Seiser. 2002. <br />                                                                                             1999. Ligand-induced recruitment histone deacetylase negative- <br />       Essential function histone deacetylase 1 proliferation control CDK <br />                                                                                             feedback regulation thyrotropin beta gene. EMBO J. 18:5389&#226;&#8364;&#8220;5398. <br />       inhibitor repression. EMBO J. 21:2672&#226;&#8364;&#8220;2681. <br /> 38.   Li, Y., G. D. Kao, B.  Garcia, J. Shabanowitz, D. F. Hunt, J. Qin, C. P <br /> </body></html>