<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS CHEMOTHERAPY, June 2006, p. 2178&#226;&#8364;&#8220;2184                                                                   Vol. 50,  6 <br /> 0066-4804/06/$08.00&#226;&#171;&#185;0 doi:10.1128/AAC.00140-06 <br /> Copyright &#194;&#169; 2006, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />         Molecular Validation LpxC Antibacterial Drug Target <br />                          Pseudomonas aeruginosa <br />        Khisimuzi E. Mdluli,1&#226;&#8364;&#160; Pamela R. Witte,1&#226;&#8364;&#161; Toni Kline,2&#194;&#167; Adam W. Barb,3 Alice L. Erwin,1* <br />         Bryce E. Mansfield,1&#194;&#182; Amanda L. McClerren,3&#227;&#8250;&#179; Michael C. Pirrung,4&#194;&#167;&#194;&#167; L. Nathan Tumey,3# <br />                     Paul Warrener,1&#226;&#8364;&#160;&#226;&#8364;&#160; Christian R. H. Raetz,3 C. Kendall Stover1&#226;&#8364;&#161;&#226;&#8364;&#161; <br />         Departments Research Biology1 Chemistry,2 Chiron Corporation, 201 Elliott Avenue West, Suite 150, Seattle, <br />              Washington 98119, Departments Biochemistry3 Chemistry,4 Duke University Medical Center, <br />                                                Durham, North Carolina 27710 <br />  <br />                          Received 2 February 2006/Returned modification 2 March 2006/Accepted 24 March 2006 <br />  <br />              LpxC [<span id='am-29' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="UDP" datatype="xsd:string"></span>UDP</span>-3-O-(R-3-hydroxymyristoyl)-GlcNAc deacetylase] metalloamidase catalyzes           committed step biosynthesis <span id='am-1' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> component <span id='am-7' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-8' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipopolysaccharide</span>. previous study (H. R. <br />           Onishi, B.  Pelak, L. S. Gerckens, L. L. Silver, F. M. Kahan, M. H. Chen,   Patchett, S. M. Galloway, S.  <br />           Hyland, M. S. Anderson, C. R. H. Raetz, Science 274:980&#226;&#8364;&#8220;982, 1996) identified series synthetic <br />           LpxC-inhibitory molecules bactericidal Escherichia coli. molecules did inhibit           growth Pseudomonas aeruginosa developed antibacterial drugs.           inactivity LpxC inhibitors P. aeruginosa raised possibility LpxC activity           essential gram-negative bacteria. placing lpxC gene P. aeruginosa tight control           arabinose-inducible promoter, demonstrated essentiality LpxC activity P. aeruginosa.           compound L-161,240, potent inhibitor previous study, active P. aeruginosa <br />           construct endogenous lpxC gene inactivated LpxC activity supplied           lpxC gene E. coli. Conversely, E. coli construct growth dependent P. aeruginosa lpxC <br />           gene resistant compound. differential activities L-161,240 bacterial species           result primarily greater potency E. coli enzyme differences intrinsic <br />           resistance bacteria antibacterial compounds permeability efflux. data validate P. <br />           aeruginosa LpxC target novel antibiotic drugs help direct design inhibitors           clinically important gram-negative bacteria. <br />  <br />  <br />    Lipopolysaccharide critical function gram-nega-                    membrane permeable small molecules tive bacterial membrane integrity resistance host                       E. coli, P. aeruginosa multidrug efflux pumps. <br /> defenses,  conserved lipopolysaccharide                      result factors, P. aeruginosa biosynthetic enzymes attractive targets novel anti-                    susceptible E. coli antibiotics (24). bacterial drugs. drug targeting enzymes biosyn-                      laboratories focused metalloenzyme LpxC <br /> thetic pathway need active Pseudomonas                     [UDP-(3-O-acyl)-N-acetylglucosamine deacetylase], aeruginosa nonfermenting gram-negative bacterial                     catalyzes committed step lipid synthesis (Fig. 1) <br /> species, Escherichia coli enteric                 demonstrated essential growth E. <br /> bacteria, clinically useful. P. aeruginosa outer                     coli (3, 12, 38). P. aeruginosa LpxC similar sequence <br />                                                                                (Fig. 2) catalyzes activity (11). essen- <br />                                                                                tiality LpxC activity P. aeruginosa formally <br />   * Corresponding author. Present address: Microbial Pathogens Pro-            proven, lpxC gene inactivated saturating trans- <br /> gram, Seattle Biomedical Research Institute, 307 Westlake Ave. N.,             poson mutagenesis study (15). data suggest Seattle, WA 98109. Phone: (206) 256-7431. Fax: (206) 256-7229. E-              possible discover LpxC inhibitors active E. <br /> mail: alice.erwin@sbri.org.                                                    coli P. aeruginosa.  early LpxC  <br />   &#226;&#8364;&#160; Present address: Global Alliance TB Drug Discovery, New <br />                                                                                hibitors, showed antibacterial activity E. <br /> York, NY 10004. <br />   &#226;&#8364;&#161; Present address: deCODE Biostructures, Bainbridge Island, WA               coli certain organisms, able inhibit growth 98110.                                                                         P. aeruginosa (5, 12, 27&#226;&#8364;&#8220;29). tempting assume   &#194;&#167; Present address: Department Genome Sciences, University              reason failure intrinsic resistance P. <br /> Washington, Seattle, WA 98195.                                                 aeruginosa antibiotics. Challenging assumption,   &#194;&#182; Present address: Department Pathobiology, University Wash- <br /> ington, Seattle, WA 98195.                                                     undertook studies described evaluate basis <br />   &#227;&#8250;&#179; Present address: Department Chemistry, University Illinois,          refractory nature P. aeruginosa LpxC inhibitors <br /> Urbana-Champaign, 61801.                                                    effective E. coli. focused com- <br />   &#194;&#167;&#194;&#167; Present address: Department Chemistry, University Cali-             pound L-161,240 (Fig. 1), active LpxC  <br /> fornia Riverside, Riverside, CA 92521. <br />   # Present address: Wyeth Research, Pearl River, NY 10965. <br />                                                                                hibitors reported researchers Merck (4, 27).   &#226;&#8364;&#160;&#226;&#8364;&#160; Present address: Rosetta Inpharmatics LLC, Seattle, WA 98109.             critical reason inactivity compound <br />   &#226;&#8364;&#161;&#226;&#8364;&#161; Present address: Pfizer Corporation, Ann Arbor, MI 48105.                 P. aeruginosa failure inhibit enzyme activ- <br />  <br />                                                                         2178 <br />  VOL. 50, 2006                                                                                                                     P. AERUGINOSA LpxC                    2179 <br />  <br />  <br />  <br />  <br />   FIG. 1.   LpxC catalyzes deacetylation UDP-(3-O-acyl)-N-acetylglucosamine, second step biosynthesis lipid  step reversible, LpxC reaction committed step conversion UDP-N-acetylglucosamine lipid  (B) Structure LpxC inhibitor L-161,240 (27). <br />  <br />  <br />  <br /> ity. findings implications designing effective                        &#226;?&#174;g plasmid pKD6 containing E. coli lpxC gene (34). lpxC genes strategies identify LpxC inhibitors developed                         amplified using Pwo DNA polymerase (Roche) 100-&#226;?&#174;l reaction mixture <br />                                                                                      containing 200 &#226;?&#174;M concentration deoxynucleoside <span id='am-24' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span>triphosphate</span> novel antibacterial drugs.                                                        0.5 &#226;?&#174;M concentration primer 30 cycles (94&#194;&#176;C denaturation, 55&#194;&#176;C <br />                                                                                      annealing, 72&#194;&#176;C polymerization). PCR products purified                         MATERIALS METHODS                                        QIAquick PCR purification kit QIAGEN digested NdeI                                                                                      EcoRI restriction enzymes. bands sizes predicted lpxC genes <br />    Reagents bacterial cultivation. bacterial strains plasmids used                                                                                      identified following gel electrophoresis excised gel. listed Table 1. P. aeruginosa strains grown 37&#194;&#176;C Luria-Bertani (LB) <br />                                                                                      excised DNA purified using QIAquick gel extraction kit QIAGEN. <br /> broth (Difco) plated sheep blood agar (Remel). E. coli grown LB <br />                                                                                      purified DNA cut NdeI EcoRI ligated T7 <br /> broth LB agar. EDTA, bis-Tris buffer, sucrose, arabinose, dimethyl <br /> sulfoxide (DMSO) purchased Sigma ultrapure agents. Yeast extract        expression vector (36) pET21b (Novagen) cut multiple tryptone obtained Difco. Restriction enzymes, T4 DNA ligase,           cloning site NdeI EcoRI. ligation mixture transformed reaction buffers obtained New England Biolabs. Polymyxin         DH5&#226;?&#163;, plated LB agar containing ampicillin (250 &#226;?&#174;g/ml). B nonapeptide, tetracycline, ampicillin, carbenicillin, gentamicin, kanamycin    inserts resulting clones sequenced subcloned. E. purchased Sigma. Compound L-161,240 synthesized              coli P. aeruginosa inserts subcloned pDN19 produce plasmids <br /> scribed previously (4). Antibacterial compounds dissolved DMSO            pEC-lpxC1 pPA-lpxC1, respectively, low-copy-number complementation <br /> make stock solutions polymyxin B nonapeptide 3 mg/ml, L-161,240 10          E. coli JBK-1 subcloned pUCP30T produce plasmids pEC-lpxC2 <br /> mg/ml, tetracycline 125 mg/ml. growth curves, DMSO added           pPA-lpxC2, respectively, complementation P. aeruginosa promoter <br /> control tubes needed DMSO concentrations             replacement mutation. <br /> cultures experiment.                                                        Construction P. aeruginosa strain lpxC expression tightly controlled <br />    Enzyme inhibition assays. LpxC activity measured previously described      araBAD promoter. Promoter replacement carried using homol- <br /> (13, 20), using crude cell extracts (38) E. coli W3110 P. aeruginosa    ogous recombination strategy, recombination pBEM10 P. <br /> PAO1 purified enzyme E. coli BL21/DE3/pLysS/pJEJ1 (14) P. aerugi-         aeruginosa removed native lpxC promoter placed tightly regulated <br /> nosa PAO1 (16) enzyme source. Assays 25 mM phosphate             araBAD promoter upstream lpxC chromosome (Fig. 3). preliminary <br /> buffer pH 7.4 5 &#226;?&#174;M substrate 30&#194;&#176;C, enzyme concentrations             experiments (data shown) promoter controlled expression (typically 0.5 10 nM) adjusted conversion 10% time     luciferase gene lux (9), P. aeruginosa, low <br /> course assays.                                                                background level expression absence arabinose expression <br />    DNA manipulations. Standard recombinant DNA procedures used (30).            eliminated addition glucose E. coli (22). Background levels primers amplification coding region lpxC genes included        promoter expression successfully reduced altering sequence NdeI EcoRI restriction sites subsequent cloning. E. coli gene,   ribosome binding site original sequence AGGAG CTTCT. <br /> primers 5&#226;&#172;&#732;-GGGAATTCCATATGATCAAACAAAGGACACTTAAAC                                    Plasmid pPW101 constructed ligating RP4 origin transfer oriT <br /> GT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CCGGAATTCTTATGCCAGTACAGCTGAAGGCGCT-3&#226;&#172;&#732;,                              pSP72. oriT amplified plasmid pEX100T (33) introduction P. aeruginosa gene, 5&#226;&#172;&#732;-GGGAATTCCATATGATGATCAAA                     NdeI AatII restriction sites. create pBEM10, following DNA <br /> CAACGCACCTTGAAGAACAT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CCGGAATTCCTACACTGCCG                                  pieces amplified sequentially ligated pPW101: tetracycline <br /> CCGCCGGGCGCATATAG-3&#226;&#172;&#732;. primers used PCR mixture                       resistance marker plasmid pUCP26 (26), araBAD promoter (8) containing template 10 50 &#226;?&#174;g P. aeruginosa genomic DNA 1         plasmid pBAD HisB (Invitrogen), altered ribosome binding site,  2180        MDLULI ET AL.                                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />    FIG. 2. Comparison predicted LpxC sequences E. coli P. aeruginosa. 82% similarity 57% identity shared sequences entire length protein. Key residues indicated color symbols residues, follows. novel zinc binding motif characteristic known LpxC enzymes, HKXXD, shown red (stars). conserved histidines shown blue (&#226;&#8211;&#160;), E. coli histidines absent P. aeruginosa shown cyan (&#228;&#352;?). conserved phenylalanines providing hydrophobic patch (7) shown orange (&#197;&#8217;). Analogous  aeolicus, E. coli histidine 265 stabilize oxyanion intermediate transition state. Significant differences enzyme fold participating residues suggested differences key residues, example, nonconserved histidines E. coli. <br />  <br />  <br /> araC repressor gene (17, 31), pBAD HisB, 340 base pairs         culture diluted 5 &#226;&#171;&#187; 105 cells/ml. total 200 &#226;?&#174;l diluted culture P. aeruginosa lpxC gene PA4406. tetracycline resistance marker               added wells containing 4 &#226;?&#174;l inhibitor. 96 plates incubated <br /> amplified using forward primer introduced BglII site (5&#226;&#172;&#732;-AGATCTCA              37&#194;&#176;C overnight OD600s determined using Spectramax Plus <br /> AGGGTTGGTTTGCGCA-3&#226;&#172;&#732;) reverse primer introduced EcoRI                      plate reader (Molecular Devices). <br /> site (5&#226;&#172;&#732;-GAATTCTAATTCTCATGTTTGACA-3&#226;&#172;&#732;). araBAD promoter araC gene amplified piece pBAD HisB vector. forward <br /> primer introduced XhoI site (5&#226;&#172;&#732;-CTCGAGGCATGCATAATGTG CCTGT                                                             RESULTS <br /> C-3&#226;&#172;&#732;), reverse primer introduced HindIII site (5&#226;&#172;&#732;-AAGCTTCTCCTG <br /> TTAGCCCAAAAAAACG-3&#226;&#172;&#732;). primer set used alter ribosome                         Compound L-161,240, potent LpxC inhibi- <br /> binding site introduce upstream BssHII site (5&#226;&#172;&#732;-GCGCGCGGACGAA <br />                                                                                         tors described Onishi et al., active E. coli, AGTAAACCCACTGG-3&#226;&#172;&#732;) downstream HindIII site (5&#226;&#172;&#732;-AAGCTTATT <br /> CAGAAGGTTAGCCCAAAAAAACGGG-3&#226;&#172;&#732;). 340 bases PAO1                              MIC 1 &#226;?&#174;g/ml, activity P. aeruginosa (MIC &#226;&#172;&#381; <br /> lpxC amplified PAO1 genomic DNA. forward primer introduced                50 &#226;?&#174;g/ml) (27). compound inactive HindIII site (5&#226;&#172;&#732;-AAGCTTATGATCAAACAACGCACCTT-3&#226;&#172;&#732;),                           wild-type strains P. aeruginosa (PAO1 ATCC <br /> verse primer introduced XbaI site (5&#226;&#172;&#732;-TCTAGAAGCGCTGCCATCCATG                         27853), strain PAO200, genes mexAB oprM, <br /> ATCGG-3&#226;&#172;&#732;). pieces ligated pPW101 form final <br /> product, pBEM10. Transformation pBEM10 P. aeruginosa removed                                                                                         encoding major multidrug efflux pump, deleted (32), <br /> native lpxC promoter replaced tightly regulated araBAD promoter         hypersusceptible strain Z61 (ATCC 35151) (2). <br /> just upstream chromosomal copy lpxC. Growth resulting strain,          Treatment PAO1 PAO200 polymyxin B non- <br /> designated PAO1-PBAD-lpxC, dependent arabinose.                                  apeptide (3 &#226;?&#174;g/ml) increase permeability mem- <br />    Growth curves. Bacterial cultures prepared diluting overnight cultures                                                                                         brane (23) failed render strains susceptible optical density 600 nm (OD600) 0.1 5 ml LB. inhibitor L-161,240 added bacterial cultures final concentration 50 &#226;?&#174;g/ml 10 &#226;?&#174;g/ml.   L-161,240 (data shown). observations suggested cultures incubated shaking, 0.8 ml taken OD600 readings       failure L-161,240 reach target P. course experiment. DH5-&#226;?&#163;, PAO1, PAO200 grown            aeruginosa cells primary reason 37&#194;&#176;C. cases temperature-sensitive JBK-1 strains assayed,    failure inhibit growth P. aeruginosa. <br /> cultures grown 42&#194;&#176;C overnight time course cultures. <br />    Microtiter assay growth inhibition promoter replacement mutants                                                                                           L-161,240 38 times potent E. coli L-161,240. Single colonies DH5&#226;?&#163;, PAO1, mutant strain picked            P. aeruginosa vitro assay LpxC activity grown LB 37&#194;&#176;C, shaking approximately 4 h,              crude bacterial extracts (Table 2). E. coli, inhibitor  VOL. 50, 2006                                                                                                       P. AERUGINOSA LpxC                2181 <br />  <br />  <br />                                                           TABLE 1. Bacterial strains plasmids <br />         Strain plasmid                                          Relevant characteristic(s                               Source reference(s) <br />  <br /> P. aeruginosa strains <br />   PAO1                                  Wild type; sequenced strain                                                      10, 35 <br />   PAO200                                mexAB-oprM                                                                       32 <br />   ATCC 35151 (Z61)                      Chemically mutagenized strain hypersusceptible antibiotics                    18, 39 <br />   PAO1-PBAD-lpxC                        lpxC promoter replaced araBAD promoter                                        work <br />  <br /> E. coli strains <br />   W3110                                 K-12 derivative                                                                  E. coli Genetic Stock Center <br />   DH5&#226;?&#163;                                  Cloning host                                                                     Gibco/Life Technologies <br />   BL221/DE3/pLysS/pJEJ1                 T7 RNA polymerase-driven expression E. coli lpxC                      13 <br />   JBK-1/pKD6                            Chromosomal lpxC disrupted Km cassette; lpxC temp-sensitive              34 <br />                                           plasmid; <span id='am-17' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>Amp</span> <br />  <br /> Plasmids <br />   pET21b                                Expression plasmid; Amp                                                          Novagen <br />   pDN19                                 Low-copy-number plasmid; Tet                                                     25 <br />   pUCP26                                Tet                                                                              26 <br />   pUCP30T                               Replicates E. coli P. aeruginosa; Gm                                 33 <br />   pSP72                                 Amp                                                                              Promega <br />   pBAD B                            araC&#226;&#171;&#185;; ParaBAD                                                                   Invitrogen <br />   pEC-lpxC1                             lpxC E. coli cloned pDN19                                              work <br />   pPA-lpxC1                             lpxC P. aeruginosa cloned pDN19                                        work <br />   pEC-lpxC2                             lpxC E. coli cloned pUCP30T                                            work <br />   pPA-lpxC2                             lpxC P. aeruginosa cloned pUCP30T                                      work <br />   pPW101                                Derived pSP72; oriT &#226;&#171;&#185;                                                       work <br />   pBEM10                                Derived pPW101; recombination P. aeruginosa chromosome                 work <br />                                            results placing lpxC control ParaBAD; Amp; Tet <br />       Antibiotic resistance markers: Amp, ampicillin; Gm, gentamicin; Km, kanamycin; Tet, tetracycline. <br />  <br />  <br />  <br />  <br /> equally active purified LpxC                         resistant LpxC inhibitor temperature (Fig. 4). <br /> LpxC bacterial extract.  P. aeruginosa, puri-                      indicated relative insensitivity P. aerugi- <br /> fication LpxC increased susceptibility L-161,240.                         nosa enzyme inhibitor sufficient confer resistance <br />    observations indicate P. aeruginosa LpxC                      growth inhibition. <br /> resistant L-161,240 LpxC E. coli                             constructed mutant P. aeruginosa native <br /> reducing effect intrinsic resistance mechanisms does                     promoter lpxC replaced inducible promoter (Fig. <br /> render P. aeruginosa susceptible growth inhibition                            3). Modification E. coli araBAD promoter render L-161,240. data allow determine relative                     tightly regulated P. aeruginosa described Materials contributions aspects P. aeruginosa&#226;&#8364;&#8482;s resistance                    Methods. resulting P. aeruginosa mutant, designated compound. order assess effect L-161,240                      PAO1-PBAD-lpxC, fully capable growth pres- <br /> P. aeruginosa LpxC bacterial cell known allow                       ence arabinose did grow absence entry compound, use E. coli construct                       inducer. confirmed P. aeruginosa similar E. coli growth 42&#194;&#176;C dependent presence                                                                                     contains functional copy lpxC functional lpxC gene E. coli P. aeruginosa.                                                                                     activity essential growth. PAO1-PBAD-lpxC chromosomal lpxC gene E. coli strain JBK-1/pKD6                                                                                     transformed plasmid containing P. aeruginosa <br /> inactivated; wild-type copy E. coli lpxC provided                                                                                     lpxC E. coli lpxC (pPA-lpxC2 pEC-lpxC2, respectively). <br /> temperature-sensitive replicon pKD6, confers  <br />                                                                                     transformants incubated various concentra- <br /> picillin resistance (34). lpxC essential growth,                                                                                     tions LpxC inhibitor. absence arabinose, P. <br /> strain viable 42&#194;&#176;C functional copy temperature-sensitive replicon. transformed JBK-1/pKD6                           aeruginosa strains expressing E. coli LpxC nearly low-copy, non-temperature-sensitive replicon carrying                        susceptible inhibitor E. coli DH5&#226;?&#163; strains, wild-type lpxC E. coli P. aeruginosa. Selection                  transformants lpxC gene P. aeruginosa tetracycline resistance 42&#194;&#176;C yielded ampicillin-susceptible                      resistant, parent P. aeruginosa strain PAO1 (Fig. 5). <br /> transformants E. coli P. aeruginosa                      experiment carried presence 0.2% <br /> struct, designated JBK-1/pEClpxC1 JBK-1/pPAlpxC1,                            arabinose induce expression chromosomal lpxC gene, <br /> spectively. result indicated lpxC P. aeruginosa                      transformant susceptible L-161,240 (Fig. 5). expressed E. coli background capable                        confirmed P. aeruginosa lpxC enzyme itself substituting inactivated chromosomal copy.                               primary factor resistance P. aeruginosa L-161,240. <br />    incubated 42&#194;&#176;, JBK-1/pEC-lpxC1 failed grow                            intrinsic resistance mechanisms P. aeruginosa, presence LpxC inhibitor L-161,240, ex-                             form membrane impermeability inhibitor efflux, play pected.  isogenic strain JBK-1/pPA-lpxC1                            relatively minor role. <br />  2182      MDLULI ET AL.                                                                                         ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 3. Construction PAO1-PBAD-lpxC, P. aeruginosa mutant lpxC controlled araBAD promoter.   promoter <br /> replacement vector pBEM10 derived pPW101 PCR products 5&#226;&#172;&#732; region P. aeruginosa lpxC described Materials Methods. <br /> (B) Promoter replacement mutagenesis. Transformation pBEM10 P. aeruginosa removed native lpxC promoter replaced tightly <br /> regulated araBAD promoter just upstream chromosomal copy lpxC. lpxC expression occurs presence arabinose. <br />  <br />  <br />  <br />                              DISCUSSION                                         construct active lpxC gene E. coli <br />                                                                                 complementary construct growth E. coli <br />    data indicate LpxC P. aeruginosa refractory <br />                                                                                 depended lpxC gene P. aeruginosa.  inhibition compound L-161,240 concentrations 10-fold <br />                                                                                 clearly difference enzymes, differences spe- <br /> higher completely inhibit LpxC E. coli. <br />                                                                                 cies membrane structure specificity efflux pumps, demonstrated vitro assays enzyme ac- <br />                                                                                 primary mediator differential susceptibilities tivity bacterial growth experiments using P. aeruginosa <br />                                                                                 bacterial species inhibitor. <br />                                                                                    remains,  cytoplasmic process P. <br />                                                                                 aeruginosa reduces activity L-161,240 (Table <br />   TABLE 2. L-161,240 inhibition LpxC enzyme                                                                                 2). experiments reveal existence  char- <br />                   purified crude sources                                                                                 acterize, process. inhibitor sequestered                         IC50 (&#226;?&#174;M) L-161,240          IC50 ratio      inactivated P. aeruginosa. Alternatively, P. aeruginosa <br />  Enzyme source                              P. aeruginosa     (P. aeruginosa/   LpxC itself complexed molecules                         E. coli DH5&#226;?&#163;                              E. coli) <br />                                                 PAO1                            way limit access inhibitor enzyme. Nonethe- <br /> Crude extracts         0.037 &#226;&#171;&#190; 0.002        1.40 &#226;&#171;&#190; 0.07            37.8          different potencies L-161,240  <br /> Purified enzyme        0.023 &#226;&#171;&#190; 0.003        0.22 &#226;&#171;&#190; 0.03             9.6         zymes appear single factor contributes         IC50, 50% inhibitory concentration. Data means &#226;&#171;&#190; standard deviations   lack activity inhibitor P. aeruginosa. replicate assays.                                                       current knowledge structure biochemistry  VOL. 50, 2006                                                                                                      P. AERUGINOSA LpxC                2183 <br />  <br />  <br />  <br />  <br />   FIG. 4. lpxC gene P. aeruginosa confers L-161,240 resis-        FIG. 5. Treatment promoter replacement mutant P. aerugi- <br /> tance E. coli lacking functional endogenous lpxC gene. strain   nosa various concentrations LpxC inhibitor L-161,240 JBK-1/pKD6 contains chromosomal lpxC gene disrupted           absence   presence (B) arabinose. Regardless pres- <br /> kan element wild-type copy lpxC temperature-sensitive    ence arabinose, wild-type E. coli DH5&#226;?&#163; susceptible inhibitor <br /> replicon pKD6. strains JBK-1/pEC-lpxC1 JBK-1/pPA-lpxC1,           wild-type (wt) PAO1 resistant. PAO1-PBAD-lpxC trans- <br /> pKD6 replaced plasmids  temperature sensitive) bear-     formed plasmid carrying lpxC gene E. coli <br /> ing wild-type lpxC E. coli P. aeruginosa, respectively.     (pEC-lpxC2) P. aeruginosa (pPA-lpxC2). absence arabi- <br /> bacterial strains incubated LB medium containing 10 &#226;?&#174;g/ml        nose  , chromosomal lpxC gene expressed strain <br /> L-161,240 drug, bacterial growth determined moni-       carrying pEC-lpxC2 susceptible inhibitor strain <br /> toring change absorbance 600 nm.                               carrying pPA-lpxC2 resistant. presence 0.2% arabinose (B), <br />                                                                          chromosomal lpxC gene PAO1-PBAD-lpxC expressed,  <br />                                                                          ferring resistance LpxC inhibitor affected LpxC enzymes limited species. relatively          plasmid-borne lpxC. <br /> high degree primary sequence similarity E. coli P. aeruginosa enzymes (Fig. 2) demonstrates inade- <br />                                                                          P. aeruginosa gram-negative pathogens,  <br /> quacy primary sequences accurate prediction func- <br />                                                                          tails presented (1, 16, 19). <br /> tional similarity proteins. Subtle differences, nonconserved histidines E. coli sequence,                                       ACKNOWLEDGMENTS significant consequences overall fold binding <br />                                                                             work supported PathoGenesis Corporation, Seattle, WA <br /> properties. unexpected differences spe-              (later merged Chiron Corporation). C.R.H.R. supported cies&#226;&#8364;&#8482; LpxC enzymes highlight importance protein struc-            NIH grant GM-51310. <br /> ture drug design. Nuclear magnetic resonance (6) X-ray                                              REFERENCES <br /> (37) structures LpxC Aquifex aeolicus                1. Andersen, N., J. Bowman,  Erwin, E. Harwood, T. Kline, K. Mdluli, S. Ng, <br /> termined, zinc sites  aeolicus P. aeruginosa            K. Pfister, R. Shawar. November 2004. Antibacterial agents. Patent <br /> enzymes studied using extended X-ray absorption                    application WO 2004/062601 A2. <br />                                                                           2. Angus, B. L.,  M. Carey, D.  Caron,  M. B. Kropinski, R. E. W. <br /> fine-structure spectroscopy (21). Structural studies LpxC                 Hancock. 1982. Outer membrane permeability Pseudomonas aeruginosa: species addition  aeolicus provide struc-               comparison wild-type antibiotic-supersusceptible mutant. Anti- <br />                                                                              microb. Agents Chemother. 21:299&#226;&#8364;&#8220;309. <br /> tural rationale differential susceptibilities spe-       3. Beall, B., J. Lutkenhaus. 1987. Sequence analysis, transcriptional orga- <br /> cies given inhibitor facilitate design               nization, insertional mutagenesis envA gene Escherichia coli. J. <br /> inhibitors broader specificity previously                 Bacteriol. 169:5408&#226;&#8364;&#8220;5415. <br />                                                                           4. Chen, M. H., M. G. Steiner, S. E. Laszlo,   Patchett, M. S. Anderson, possible.                                                               S.  Hyland, H. R. Onishi, L. L. Silver, C. R. H. Raetz. 1999. Carbo- <br />    data suggest strategies identify broad-spectrum               hydroxamido-oxazolidines: antibacterial agents target lipid biosynthe- <br /> LpxC inhibitors challenging ac-                  sis. Bioorg. Med. Chem. Lett. 9:313&#226;&#8364;&#8220;318. <br />                                                                           5. Clements, J. M., F. Coignard,  Johnson, S. Chandler, S. Palan,  Waller, <br /> count structural differences LpxC enzymes differ-                J. Wijkmans, M. G. Hunter. 2002. Antibacterial activities charac- <br /> ent gram-negative bacterial species. finding nonen-                 terization novel inhibitors LpxC. Antimicrob. Agents Chemother. 46: <br />                                                                              1793&#226;&#8364;&#8220;1799. <br /> zyme components cell extracts affect activity inhibitors        6. Coggins, B. E., X. Li,  L. McClerren, O. Hindsgaul, C. R. H. Raetz, P. bacterial cell adds additional level complexity.            Zhou. 2003. Structure LpxC deacetylase bound substrate-analog <br /> Target-based antibiotic discovery successful               inhibitor. Nat. Struct. Biol. 10:645&#226;&#8364;&#8220;651. <br />                                                                           7. Coggins, B. E.,  L. McClerren, L. Jiang, X. Li, J. Rudolph, O. Hindsgaul, possible evaluate separately factors               C. R. Raetz, P. Zhou. 2005. Refined solution structure LpxC-TU- <br /> contribute bacterial growth inhibition: penetration                514 complex pKa analysis active site histidine: insights cell wall, resistance efflux, inactivation sequestration            mechanism inhibitor design. Biochemistry 44:1114&#226;&#8364;&#8220;1126. <br />                                                                           8. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu- <br /> intracytoplasmic components, potency target.                  lation, modulation, high-level expression vectors containing arabi- use multiple molecular biochemical approaches                     nose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130. <br />                                                                           9. Hickey, M. J., T. M. Arain, R. M. Shawar, D. J. Humble, M. H. Langhorne, <br /> allowed evaluate LpxC inhibitor                  J. N. Morgenroth, C. K. Stover. 1996. Luciferase vivo expression <br /> L-161,240 case, potency                  technology: use recombinant mycobacterial reporter strains evaluate <br /> target primary factor limiting activity P. aeruginosa.           antimycobacterial activity mice. Antimicrob. Agents Chemother. 40:400&#226;&#8364;&#8220; <br />                                                                              407. conclusion critical discovery potent small-           10. Holloway, B. W. 1955. Genetic recombination Pseudomonas aeruginosa. <br /> molecule LpxC inhibitors antibacterial activity                  J. Gen. Microbiol. 13:572&#226;&#8364;&#8220;581. <br />  2184       MDLULI ET AL.                                                                                                      ANTIMICROB. AGENTS CHEMOTHER. <br />  <br /> 11. Hyland, S. , S. S. Eveland, M. S. Anderson. 1997. Cloning, expression,     26. Olsen, R. H., G. DeBusscher, W. R. McCombie. 1982. Development     purification UDP-3-O-acyl-GlcNAc deacetylase Pseudomonas                 broad-host-range vectors gene banks: self-cloning Pseudomonas <br />     aeruginosa: metalloamidase lipid biosynthesis pathway. J. Bacte-          aeruginosa PAO chromosome. J. Bacteriol. 150:60&#226;&#8364;&#8220;69. <br />     riol. 179:2029&#226;&#8364;&#8220;2037.                                                             27. Onishi, H. R., B.  Pelak, L. S. Gerckens, L. L. Silver, F. M. Kahan, M. H. <br /> 12. Jackman, J. E., C.  Fierke, L. N. Tumey, M. Pirrung, T. Uchiyama, S. H.            Chen,   Patchett, S. M. Galloway, S.  Hyland, M. S. Anderson,     Tahir, O. Hindsgaul, C. R. H. Raetz. 2000. Antibacterial agents             C. R. H. Raetz. 1996. Antibacterial agents inhibit lipid biosynthesis. <br />     target lipid biosynthesis gram-negative bacteria. Inhibition diverse         Science 274:980&#226;&#8364;&#8220;982. <br />     UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacetylases sub-          28. Pirrung, M. C., L. N. Tumey,  L. McClerren, C. R. H. Raetz. 2003. <br />     strate analogs containing zinc binding motifs. J. Biol. Chem. 275:11002&#226;&#8364;&#8220;             High-throughput catch release synthesis oxazoline hydroxamates. <br />     11009.                                                                               Structure-activity relationships novel inhibitors Escherichia coli LpxC: <br /> 13. Jackman, J. E., C. R. H. Raetz, C.  Fierke. 1999. UDP-3-O-(R-3-                vitro enzyme inhibition antibacterial properties. J.  Chem. Soc. <br />     hydroxymyristoyl)-N-acetylglucosamine deacetylase Escherichia coli           125:1575&#226;&#8364;&#8220;1586. <br />     zinc metalloenzyme. Biochemistry 38:1902&#226;&#8364;&#8220;1911.                                   29. Pirrung, M. C., L. N. Tumey, C. R. H. Raetz, J. E. Jackman, K. Snehalatha, <br /> 14. Jackman, J. E., C. R. H. Raetz, C.  Fierke. 2001. Site-directed mu-             L. McClerren, C.  Fierke, S. L. Gantt, K. M. Rusche. 2002. Inhi- <br />     tagenesis bacterial metalloamidase UDP-(3-O-acyl)-N-acetylglu-                bition antibacterial target UDP-(3-O-acyl)-N-acetylglucosamine <br />     cosamine deacetylase (LpxC). Identification zinc binding site. Bio-           deacetylase (LpxC): isoxazoline zinc amidase inhibitors bearing diverse <br />     chemistry 40:514&#226;&#8364;&#8220;523.                                                                metal binding groups. J. Med. Chem. 45:4359&#226;&#8364;&#8220;4370. <br /> 15. Jacobs, M. ,  Alwood,  Thaipisuttikul, D. Spencer, E. Haugen, S. Ernst,    30. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning:     O.  R. Kaul, C. Raymond, R. Levy, L. Chun-Rong, D. Guenthner, D.                laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring <br />     Bovee, M. V. Olson, C. Manoil. 2003. Comprehensive transposon mutant             Harbor, N.Y. <br />     library Pseudomonas aeruginosa. Proc. Natl. Acad. Sci. USA 100:14339&#226;&#8364;&#8220;         31. Schleif, R. 1992. DNA looping. Annu. Rev. Biochem. 61:199&#226;&#8364;&#8220;223. <br />     14344. <br />                                                                                      32. Schweizer, H. P. 1998. Intrinsic resistance inhibitors fatty acid biosyn- <br /> 16. Kline, T., N. H. Andersen, E.  Harwood, J. Bowman,  Malanda, S. <br />                                                                                          thesis Pseudomonas aeruginosa efflux: application novel <br />     Endsley,  L. Erwin, M. Doyle, S. Fong,  L. Harris, B. Mendelsohn, K. <br />                                                                                          technique generation unmarked chromosomal mutations study <br />     Mdluli, C. R. H. Raetz, C. K. Stover, P. R. Witte,  Yabannavar, S. Zhu. <br />                                                                                          efflux systems. Antimicrob. Agents Chemother. 42:394&#226;&#8364;&#8220;398. <br />     2002. Potent, novel vitro inhibitors Pseudomonas aeruginosa <br />                                                                                      33. Schweizer, H. P., T. T. Hoang. 1995. improved gene <br />     deacetylase LpxC. J. Med. Chem. 45:3112&#226;&#8364;&#8220;3129. <br />                                                                                          replacement xylE fusion analysis Pseudomonas aeruginosa. Gene 158: <br /> 17. Lee, N. 1980. Molecular aspects ara regulation, p. 389&#226;&#8364;&#8220;410. J. H. Miller <br />                                                                                          15&#226;&#8364;&#8220;22. <br />     W. S. Reznikoff (ed.), operon. Cold Spring Harbor Laboratory, <br />     Cold Spring Harbor, N.Y.                                                         34. Sorensen, P. G., J. Lutkenhaus, K. Young, S. S. Eveland, M. S. Anderson, <br /> 18. Li, X. , D. M. Livermore, H. Nikaido. 1994. Role efflux pump(s)          C. R. H. Raetz. 1996. Regulation UDP-3-O-[R-3-hydroxymyristoyl]- <br />     intrinsic resistance Pseudomonas aeruginosa: resistance tetracycline,          N-acetylglucosamine deacetylase Escherichia coli. second enzymatic <br />     chloramphenicol, norfloxacin. Antimicrob. Agents Chemother. 38:1732&#226;&#8364;&#8220;             step lipid biosynthesis. J. Biol. Chem. 271:25898&#226;&#8364;&#8220;25905. <br />     1741.                                                                            35. Stover, C. K., X. Q. Pham,  L. Erwin, S. D. Mizoguchi, P. Warrener, M. J. <br /> 19. McClerren,  L., S. Endsley, J. L. Bowman, N. H. Andersen,  Guan, J.              Hickey, F. S. Brinkman, W. O. Hufnagle, D. J. Kowalik, M. Lagrou, R. L. <br />     Rudolph, C. R. Raetz. 2005. slow, tight-binding inhibitor               Garber, L. Goltry, E. Tolentino, S. Westbrock-Wadman, Y. Yuan, L. L. <br />     zinc-dependent deacetylase LpxC lipid biosynthesis antibiotic              Brody, S. N. Coulter, K. R. Folger,  Kas, K. Larbig, R. Lim, K. Smith, D. <br />     activity comparable ciprofloxacin. Biochemistry 44:16574&#226;&#8364;&#8220;16583.                   Spencer, G. K. Wong,  Wu,  T. Paulsen, J. Reizer, M. H. Saier, R. E. <br /> 20. McClerren,  L., P. Zhou,  Guan, C. R. Raetz, J. Rudolph. 2005.               Hancock, S. Lory, M. V. Olson. 2000. Complete genome sequence     Kinetic analysis zinc-dependent deacetylase lipid biosynthetic       Pseudomonas aeruginosa PA01, opportunistic pathogen. Nature 406:959&#226;&#8364;&#8220; <br />     pathway. Biochemistry 44:1106&#226;&#8364;&#8220;1113.                                                  964. <br /> 21. McClure, C. P., K. M. Rusche, K. Peariso, J. E. Jackman, C.  Fierke,       36. Studier, F. W.,  H. Rosenberg, J. J. Dunn, J. W. Dubendorff. 1990. Use <br />     J. E. Penner-Hahn. 2003. EXAFS studies zinc sites UDP-(3-O-                T7 RNA polymerase direct expression cloned genes. Methods  <br />     acyl)-N-acetylglucosamine deacetylase (LpxC). J. Inorg. Biochem. 94:78&#226;&#8364;&#8220;85.           zymol. 185:60&#226;&#8364;&#8220;89. <br /> 22. Miyada, C. G., L. Stoltzfus, G. Wilcox. 1984. Regulation araC         37. Whittington, D. , K. M. Rusche, H. Shin, C.  Fierke, D. W. Christianson. <br />     gene Escherichia coli: catabolite repression, autoregulation, effect       2003. Crystal structure LpxC, zinc-dependent  <br /> </body></html>