<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNA (2001), 7 :1397&#226;&#8364;&#8220;1402+ Cambridge University Press+ Printed USA+ <br /> Copyright &#194;&#169; 2001 RNA Society+ <br /> DOI: 10+1017+S1355838201011074 <br />  <br />  <br />  <br />  <br /> Distinct roles RDE-1 RDE-4 RNA <br /> interference Caenorhabditis elegans <br />  <br />  <br /> SUSAN PARRISH 1,2 ANDREW 1 <br /> 1 <br />     Department Embryology, Carnegie Institution Washington, Baltimore, Maryland 21210, USA <br /> 2 <br />     Biology Graduate Program, Johns Hopkins University, Baltimore, Maryland 21218, USA <br />  <br />  <br />  <br />  <br /> ABSTRACT <br /> RNA interference (RNAi) cellular defense mechanism uses double-stranded RNA (dsRNA) sequence- <br /> specific trigger guide degradation homologous single-stranded RNAs. RNAi multistep process involving proteins type RNA intermediate, population small 21&#226;&#8364;&#8220;25 nt RNAs (called siRNAs) initially derived cleavage dsRNA trigger. Genetic screens Caenorhabditis elegans identified <br /> numerous mutations cause partial complete loss RNAi. work, analyzed cleavage injected <br /> dsRNA produce initial siRNA population animals mutant rde-1 rde-4, genes essential RNAi required organismal viability fertility. results suggest distinct roles RDE-1 RDE-4 interference process. null mutants lacking rde-1 phenotypic response dsRNA, siRNAs generated injected dsRNA trigger comparable wild-type. contrast, <br /> mutations rde-4 substantially reduced population siRNAs derived injected dsRNA trigger. Injection chemically synthesized 24- 25-nt siRNAs circumvent RNAi resistance rde-4 mutants, bypass observed rde-1 mutants. results support model RDE-4 involved production siRNAs, RDE-1 acts siRNAs formed. <br /> Keywords: dsRNA; Piwi; PTGS; RNAi; silencing; siRNA <br />  <br />  <br />  <br /> INTRODUCTION                                                            Bosher <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; Labouesse, 2000)+ ancient origin RNAi <br />                                                                         PTGS processes suggests mecha- <br /> Defense mechanisms protect deleterious <br />                                                                         nisms represent primitive nucleic-acid-based im- <br /> gene expression essential long-term survival                                                                         mune response+ species+ genome surveillance mechanisms <br />                                                                            precise molecular mechanisms RNAi <br /> allow selective silencing destruction aberrant nu- <br />                                                                         remain elucidated, recent observations <br /> cleic acids, including molecules endogenous <br />                                                                         provided beginnings working model+ parasitic origin+ RNA interference (RNAi), ex- <br />                                                                         particularly significant observation Ham- <br /> ample mechanism, uses double-stranded RNA <br />                                                                         ilton &amp; Baulcombe (1999), identified population <br /> (dsRNA) trigger sequence-specific degradation <br />                                                                         small (;25-nt) RNAs  referred siRNAs), homologous transcripts  et al+, 1998; reviewed <br />                                                                         presence correlates ongoing silencing+ Sharp, 2001)+ dsRNAs frequently byproduct                                                                         siRNAs observed Drosophila extracts car- <br /> viral transposon gene expression+ proposed <br />                                                                         rying RNAi-like reaction (Hammond et al+, 2000; <br /> function RNAi recognize dsRNAs  <br />                                                                         Zamore et al+, 2000)+ siRNA formation vitro dicators unwanted gene expression, providing <br />                                                                         vivo shown result cleavage means silence parasitic genetic elements   <br />                                                                         dsRNA trigger (Parrish et al+, 2000; Yang et al+, <br /> viewed Baulcombe, 1996, 1999; Ketting et al+, 1999; <br />                                                                         2000; Zamore et al+, 2000)+ Intriguingly, Hamilton &amp; Baul- <br /> Tabara et al+, 1999)+ Aspects RNAi related post- <br />                                                                         combe (1999) reported example ap- <br /> transcriptional silencing (PTGS) mechanisms                                                                         pearance strong siRNA signal dependent conserved diverse phyla eukary- <br />                                                                         presence RNA target+ suggests otic kingdoms (reviewed Cogoni &amp; Macino, 2000; <br />                                                                         population siRNAs formed initial cleavage                                                                         trigger RNA subjected subsequent   <br />                                                                         amplification target-dependent copying process, <br />    Reprint requests  Andrew  Carnegie Institution Washing- <br /> ton, Department Embryology, 115 West University Parkway, Bal-        (b) stabilization presence mRNA <br /> timore, Maryland 21210, USA; e-mail: ciwemb+edu+                   target+ <br />                                                                     1397 <br />  1398                                                                                              S. Parrish   <br />    siRNAs possess characteristic structure,            description molecular intermediates  59 phosphate, 39 hydroxyl,  base          action, combined availability numerous mu- <br /> 39 overhang strand duplex (Elbashir             tants affecting mechanism provides excellent <br /> et al+, 2001b)+ siRNA structure bears striking          starting point construction functional path- <br /> resemblance products formed RNase III cleavage           way (Edgar &amp; Wood, 1966)+ straightforward long dsRNAs+ Consistent hypothesis         starting point analysis genes  <br /> RNase III-like activity involved RNAi, Bernstein         quired RNAi essential viability+ cases colleagues (2001) demonstrated pro-            allow clear analysis reaction living tein homology RNase III (DICER) present          completely lacking specific gene product+ Mutations <br /> Drosophila extracts capable producing short          partial loss RNAi accom- <br /> RNA products longer dsRNA trigger+ Impor-               panying fertility defects somewhat chal- <br /> tantly, synthetic 21- 22-nt RNA duplexes mimic         lenge analysis organism+ case, structure siRNAs capable effectively trig-       rescue maternal dowry, overlapping gene function, <br /> gering RNAi vitro vivo, implicating siRNAs        considerations prevent complete loss RNAi <br /> direct participants RNAi process (Caplen et al+,        activity achieved living  <br /> 2001; Elbashir et al+ 2001a, 2001b)+ siRNAs, prob-            work, utilized assays production ably assembled ribonucleoprotein complex,            siRNAs interference siRNAs rde-1 rde-4 shown direct cleavage cognate mRNA              mutant animals begin constructing pathway region homology, single cleavage        RNAi reaction+ <br /> site located center protected area (Zamore <br /> et al+, 2000; Elbashir et al+, 2001b)+ <br />                                                                RESULTS DISCUSSION <br />    approach understand silencing mecha- <br /> nism relied genetic screens genes                                                                Genetic requirements production mutate silencing-deficient phenotype (Cogoni &amp; <br />                                                                siRNAs RNA interference <br /> Macino, 1997; Elmayan et al+, 1998; Tabara et al+, 1999)+ class mutations Caenorhabditis elegans,          previously described vivo assay fol- <br /> sisting alleles genes rde-1 rde-4 (rde:          low fate dsRNA trigger C. elegans (Parrish <br /> RNAi defective), shows global RNAi resistance          et al+, 2000)+ Briefly, C. elegans adults injected                                                                32 evidence additional phenotypic consequences                P-labeled dsRNA harvested RNA extraction <br /> (Tabara et al+, 1999)+ rde-1 gene cloned          12 h growth 16 8C+ RNA molecules encodes member large PIWI/STING/                  injected animals resolved denaturing polyacryl- <br /> Argonaute/Zwille/eIF2c protein family (e+g+, 24 rep-      amide gels visualized phosphorimager analy- <br /> resentatives C. elegans; Tabara et al+, 1999)+ Analyses     sis+ wild-type recipient animals, assay detects PIWI/STING/Argonaute/Zwille/eIF2c homologs               small proportion (approximately 1%) dsRNA trig- systems suggested similar             ger processed small RNAs (Parrish et al+, rde-1 exhibiting specific role RNAi/PTGS,           2000)+ small RNAs approximately 25 nt involved developmental regula-           length, similar siRNAs observed RNAi di- <br /> tion gene expression (Cox et al+, 1998; Catalanotto         verse species (Hamilton &amp; Baulcombe, 1999; Ham- <br /> et al+, 2000; Fagard et al+, 2000; Grishok et al+, 2001)+      mond et al+, 2000; Parrish et al+, 2000; Yang et al+, <br /> Certain members family dual function          2000; Zamore et al+, 2000)+ investigate genetic development genome surveillance (Fagard et al+,         requirements production siRNAs, utilized 2000)+                                                         vivo assay assess siRNA formation rde-1    Consistent hypothesis mechanistic com-          rde-4 mutants+ <br /> monality RNAi developmental control pro-              mutant strains carrying different rde-1 <br /> cesses, large class mutations shown           alleles (rde-1 (ne219 ) rde-1 (ne300 ); Tabara et al+, <br /> affect RNAi developmental progression+ C.          1999), levels ;25 nt siRNAs produced elegans, mutations genes rde-2, rde-3, mut-2,           dsRNA trigger comparable wild-type N2 <br /> mut-7, mut-8, mut-9, ego-1, exhibit variable            worms (ne300 : Fig+ 1; ne219 : data shown)+ grees RNAi resistance accompanied host              indicates RDE-1 required initial pro- phenotypes, defective chromosome seg-            cessing trigger RNA siRNAs+ rde-1 <br /> regation  , increased transposon mobilization (rde-2,      alleles used distinct molecular lesions: rde- <br /> rde-3, mut-7, -8, -9), chromatin condensation          1 (ne300 ) predicted null contains premature <br /> defects (mut-7 ) (Ketting et al+, 1999; Tabara et al+, 1999;   stop codon, rde-1 ( ne219 ) harbors mis- <br /> Dernburg et al+, 2000; Ketting &amp; Plasterk, 2000; Smar-         sense mutation conserved PIWI domain (Tabara <br /> don et al+, 2000)+ mutations produce steril-      et al+, 1999)+ <br /> ity, does loss function C. elegans DICER            rde-4 ( ne299 ) animals (Tabara et al+, 1999) homolog dcr-1 (Grishok et al+, 2001)+                          examined assay, reduction  Functions RDE-1 RDE-4 RNAi                                                                                               1399 <br />  <br />  <br />  <br />  <br />                                                                          FIGURE 2. Reduced levels ;25-nt siRNA products dsRNA <br />                                                                          trigger rde-4 mutants+ Labeled dsRNA injected N2                                                                          rde-4(ne299) adults (grown 16 8C 20 8C) analyzed  <br />                                                                          scribed Figure 1+ End-labeled ssRNA oligos (zr1 zr2,                                                                          26 nt length zr10, 27 nt length; Parrish et al+, 2000)                                                                          utilized markers+ <br /> FIGURE 1. Processing injected dsRNA ;25-nt siRNAs impaired rde-1 mutants+ 32 P-labeled dsRNA injected N2 <br /> (wild-type) rde-1(ne300) C. elegans adults+ RNAs uni- <br /> formly labeled vitro synthesis presence 32 P-<span id='am-8' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="UTP" datatype="xsd:string"></span>UTP</span>  specific activity 15 Ci/mMol UTP) injected concentration     temperatures paralleled overall RNAi sensitivity+ 16 8C, ;2+5 mg/mL+ Animals allowed recover 12 h 16 8C         siRNA products injected RNA detected subsequently harvested RNA extraction+ Samples           rde-4 ( ne299 ) animals, present level <br /> solved electrophoresis 12% denaturing polyacrylamide gels visualized phosphorimager analysis+ different images       substantially levels wild-type (Fig+ 2)+ gel shown; high gain image located left low    animals grown 20 8C (Fig+ 2) 23 8C <br /> gain image right+ t 5 0 sample contained mixture      (data shown), levels small RNA generated C. elegans labeled dsRNA subjected stan- <br /> dard RNA extraction+ End-labeled ssRNA oligos (zr1 zr2,         rde-4 ( ne299 ) consistently detection <br /> 26 nt length; Parrish et al+, 2000) utilized markers+         threshold assay+ results suggest  <br />                                                                          volvement (direct indirect) RDE-4 initial <br />                                                                          processing dsRNA trigger+ <br /> siRNA generated trigger observed, dramatic decrease seen elevated tempera- <br />                                                                          Assays interference siRNAs <br /> tures (Fig+ 2)+ RNAi process rde-4 (ne299 ) ap- <br />                                                                          distinguish roles RDE-1 <br /> pears somewhat temperature sensitive, allowing <br />                                                                          RDE-4 RNAi interference 20 8C 23 8C, permitting weak <br /> interference grown 16 8C (data shown)+                       experiments performed Drosophila embryo ex- present data distinguish                        tracts, siRNAs derived dsRNA trigger temperature effects ne299 allele reflect                      shown possess characteristic structure temperature-sensitive protein  e+, residual RDE-4 pro-                 59 phosphate, 39 hydroxyl, 39   <br /> tein activity 16 8C) temperature-sensitive pro-                  base overhang strand duplex (Elbashir <br /> cess (ability RNAi reaction proceed low                   et al+, 2001b)+ Synthetic dsRNA oligos siRNA <br /> level absence RDE-4 16 8C)+ Assays                      structure shown capable triggering <br /> initial siRNA formation rde-4 mutant different                 RNAi injection C. elegans (Caplen et al+, 2001)+ <br />  1400                                                                                                           S. Parrish   determine roles RDE-1 RDE-4                  tion 24-nt 25-nt siRNA+ injection bypassed precleavage trigger RNA,                      siRNAs produced interference rde-4 mutant <br /> examined chemically synthesized siRNAs                   background, bypass block interference ap- <br /> produce phenotypic effect rde-1 rde-4 mutants+                 peared incomplete+ Evidence incomplete resto- <br />   siRNAs able bypass RNAi                       ration interference comes comparison fects rde-1 (ne300 ) animals (Fig+ 3)+ siRNAs contain-              fraction progeny affected wild-type versus rde-4 <br /> ing 24 25 nt unc-22 sequence, produced                     mutant animals injected 25-nt siRNA (approx- characteristic specific twitching phenotype                 imately threefold increase wild-type animals)+  <br /> dicative unc-22 interference wild-type animals              terestingly, blunt-ended 26-nt dsRNA did trigger <br /> et al+, 1998; Caplen et al+, 2001), produced interfer-              significant interference rde-4 mutant animals,  <br /> ence rde-1 (ne300 ) animals+ Similarly, blunt-ended               spite triggering modest interference wild-type worms+ <br /> dsRNA oligo containing 26 nt mediate inter-                  result implies structural modifications im- <br /> ference rde-1 ( ne300 ), dsRNA oligo 81 nt              parted RNase III cleavage event important length+ inability siRNAs bypass RNAi                  generating efficient RNAi+ rde-4 ( ne299 ), resistance rde-1 mutants suggests defect                slight background level interference mutant lies downstream trigger processing+ hy-                 26-nt 81-nt dsRNAs; resulted pothesis consistent observation (Fig+ 1)              degradation RNA preparations, result- ;25-nt siRNAs present injection la-                  ing small population RNAs resembling struc- <br /> beled dsRNA rde-1 mutants+                                        ture siRNAs+ ability siRNAs bypass   synthetic siRNAs able partially                RNAi defect rde-4 mutant, addition  <br /> bypass requirement RDE-4 interference                   duced ;25-nt siRNAs generated RNAi <br /> response (Fig+ 3)+ alleles rde-4 (ne299                 mutant (Fig+ 2), suggests RDE-4 involved <br /> ne301 ), interference induced 20 8C injec-                generating siRNAs dsRNA trigger molecules+ <br />  <br />  <br />  <br />  <br />        FIGURE 3. siRNAs bypass RNAi resistance rde-4 rde-1 mutants+ Shown structures        synthetic RNAs injected C. elegans (Parrish et al+, 2000, Caplen et al+, 2001)+ Shown interference <br />        activities synthetic RNAs wild-type mutant C. elegans (wild-type data described, Caplen et al+, 2001)+ Double- <br />        stranded RNAs (5 mg/mL) injected gonad C. elegans adults grown 20 8C+ Percentages represent        fraction F1 progeny exhibiting unc-22 twitching phenotype 0+33 mM levamisole+ Numbers parentheses indicate <br />        total number animals scored+ <br />  Functions RDE-1 RDE-4 RNAi                                                                                 1401 <br />  <br /> Sequential activity RDE-4                                   subsequent steps utilize short segment RDE-1 RNAi                                              guide RNA means recognize act                                                                target transcript+ data consistent sequential model RDE-4 involved step prior initial <br /> cleavage dsRNA trigger produce siRNA-type            MATERIALS METHODS <br /> molecules, RDE-1 involved subsequent <br /> step interference (Fig+ 4)+ complex <br />                                                                Strains <br /> multiprotein/multi-RNA assemblies, events RNAi occur short time range large           C. elegans stock N2 (Brenner, 1974) used complex (e+g+, Bernstein et al+, 2001)+          representative wild-type strain+ Strains carrying RNAi- <br /> likely degree freedom formation           defective mutations rde-1 ( ne219 ), rde-1 ( ne300 ), rde- action protein:RNA complexes execute RNAi+         4 ( ne299 ), rde-4 ( ne301 ) (Tabara et al+, 1999) pointed experiments           extensively backcrossed wild-type  gifts H+ <br />                                                                Tabara C+ Mello)+ strain contained genetic <br /> rule association RDE-1 precleavage <br />                                                                markers+ <br /> RNAi complex, rule continued  <br /> sociation RDE-4 complex cleavage+ <br />    Figure 4 shows working model roles           Assays fate dsRNA trigger <br /> RDE-1, RDE-4, siRNAs interference pro- <br /> cess: RDE-4 invoked recognition long              vivo fate radioactively labeled trigger molecules duplex RNA molecules aberrant RNAs (allowing                analyzed described previously (Parrish et al+, 2000)+ Ra- <br />                                                                dioactive RNA synthesized incorporation 32 P- <br /> cleavage form characteristic siRNA guide), <br />                                                                UTP (specific activity, 15 Ci/mMol UTP) standard T3/ T7 RDE-1  homologous factors) invoked <br />                                                                polymerase reactions followed RNA extraction, sense/ <br />                                                                antisense hybridization, injection C. elegans                                                                concentration ;2+5 mg/mL+ injected dsRNA mol- <br />                                                                ecules contained sequence gfp (326 nt), unc-22 (743 <br />                                                                nt), restriction enzyme linkers (103 nt)+ Following <br />                                                                growth 12 h 16 8C, injected animals harvested                                                                RNA extraction+ RNAs resolved 12% denaturing poly- <br />                                                                acrylamide gels visualized using Molecular Dynamics <br />                                                                Storm Phosphorimager+ <br />  <br />  <br />                                                                Assays interference small RNAs <br />                                                                RNA oligonucleotides containing sequences unc-22 <br />                                                                gene (Fig+ 3) chemically synthesized using 29-O -(tri- <br />                                                                isopropyl)silyloxy-methyl chemistry Xeragon AG (Zurich, <br />                                                                Switzerland; Parrish et al+, 2000; Caplen et al+, 2001)+                                                                26-nt 81-nt RNA oligomers purified anion <br />                                                                exchange HPLC+ RNA oligos annealed injected <br />                                                                C. elegans strains (grown 20 8C) concentration                                                                ;5 mg/mL+ degree interference measured                                                                fraction progeny exhibiting Unc-22 twitching phenotype <br />                                                                0+33 mM levamisole+ <br />  <br />  <br />                                                                ACKNOWLEDGMENTS <br />                                                                thank Hiroaki Tabara, Craig Mello, Lisa Timmons, Rich- <br />                                                                ard Morgan, Natasha Caplen numerous suggestions <br />                                                                continued collaboration, R+ Alcazar, J+ Fleenor, S+ <br />                                                                Kostas, K+ Liu, J+ Yanowitz, T+ Tuschl, J+ Bender, C+ Fan, P+ <br />                                                                Sharp, R+ Plasterk valuable discussions+ Research <br />                                                                support National Institutes Health (Grants <br />                                                                GM37706 GM07231) Carnegie Institution                                                                Washington gratefully acknowledged+ <br />  <br />                                                                Received June 18, 2001; returned revision <br /> FIGURE 4. working model showing roles RDE-1, RDE-4,   June 27, 2001; revised manuscript received <br /> siRNAs interference reaction+                           July 9, 2001 <br />  1402                                                                                                              S. Parrish   <br /> REFERENCES                                                                   Vaucheret H+ 1998+ Arabidopsis mutants impaired cosuppres- <br />                                                                              sion+ Plant Cell 10 :1747&#226;&#8364;&#8220;1757+ <br /> Baulcombe D+ 1996+ RNA target initiator post-             Fagard M, Boutet S, Morel J, Bellini C, Vaucheret H+ 2000+ AGO1, <br />    transcriptional gene silencing transgenic plants+ Plant Mol Biol       QDE-2, RDE-1 related proteins required post- <br />    32 :79&#226;&#8364;&#8220;88+                                                                transcriptional gene silencing plants, quelling fungi, Baulcombe D+ 1999+ Viruses gene silencing plants+ Arch Virol          RNA interference animals+ Proc Natl Acad Sci USA 97 :11650&#226;&#8364;&#8220; <br />    Suppl 15 :189&#226;&#8364;&#8220;201+                                                        11654+ <br /> Bernstein E, Caudy  Hammond S, Hannon G+ 2001+ Role               Xu S, Montgomery M, Kostas S, Driver S, Mello C+ 1998+ <br />    bidentate ribonuclease initiation step RNA interf <br /> </body></html>