<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access identification essential genes using targeted genome <br /> sequencing comparative analysis <br /> Adam M Gustafson*&#226;&#8364;&#160;1, Evan S Snitkin*&#226;&#8364;&#160;1, Stephen CJ Parker1, <br /> Charles DeLisi1,2 Simon Kasif1,2,3 <br />  <br /> Address: 1Bioinformatics Graduate Program, Boston University, Boston, MA 02215 USA, 2Department Biomedical Engineering, Boston <br /> University, MA 02215 USA 3Children&apos;s Hospital Informatics Program Harvard MIT Division Health Sciences Technology, Boston, <br /> MA, USA <br /> Email: Adam M Gustafson* - gustafad@bu.edu; Evan S Snitkin* - esnitkin@bu.edu; Stephen CJ Parker - parker@bu.edu; <br /> Charles DeLisi - delisi@bu.edu; Simon Kasif - kasif@bu.edu <br /> * Corresponding authors &#226;&#8364;&#160;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 19 October 2006                                                     Received: 18 August 2006 <br />                                                                                Accepted: 19 October 2006 <br /> BMC Genomics 2006, 7:265     doi:10.1186/1471-2164-7-265 article available  http://www.biomedcentral.com/1471-2164/7/265 <br /> &#194;&#169; 2006 Gustafson et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />             Abstract <br />             Background: identification genes essential survival theoretical importance             understanding minimal requirements cellular life, practical importance identification <br />             potential drug targets novel pathogens. great time expense required experimental <br />             studies aimed constructing catalog essential genes given organism, computational approach <br />             identify essential genes high accuracy great value. <br />             Results: gathered numerous features generated automatically genome sequence <br />             data assessed relationship essentiality, subsequently utilized machine learning construct <br />             integrated classifier essential genes S. cerevisiae E. coli. looking single features, <br />             phyletic retention, measure number organisms ortholog present              predictive essentiality. Furthermore, construction phyletic retention feature             time explored evolutionary relationship set organisms presence             gene predictive essentiality. E. coli S. cerevisiae optimal sets             contain host-associated organisms small genomes closely related reference. Using <br />             optimally selected organisms, able improve predictive accuracy compared using             available sequenced organisms. hypothesize predictive power genomes consequence <br />             process reductive evolution, parasites symbionts evolved gene content. <br />             addition, essentiality measured rich media, condition resembles environments             organisms hosts nutrients provided. Finally, demonstrate integration             highly predictive features using probabilistic classifier resulted accuracies surpassing             individual feature. <br />             Conclusion: Using features obtainable directly sequence data, able construct classifier <br />             predict essential genes high accuracy. Furthermore, analysis set genomes             presence gene predictive essentiality suggest ways targeted <br />             sequencing used identification essential genes. summary, methods presented             aid reduction time money invested essential gene identification targeting genes <br />             experimentation predicted essential high probability. <br />  <br />  <br />  <br />                                                                                                                                        Page 1 16 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                      http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br /> Background                                                        essential genes [8-10]. essential genes fundamental step understanding cells function            unique evolutionary pressure, likely share comprehension minimal gene set required          characteristics gleaned sustain life. core requirements cellular life      genome sequence data. understood, necessary identify compo- <br /> nents set diverse organisms. <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, prediction       practical theoretical importance discovery essential genes supported          identification essential genes mind, set significant experimental work. Procedures               construct effective classifier essential genes single gene knockouts [1], RNA interference [2],          exploited various genomic descriptors gen- conditional knockouts [3] used discov-           erated directly form sequence data. Previous works aimed <br /> ery mechanisms, techniques require            understanding properties human disease genes <br /> large investment time skill perform.           taken similar approach [11,12]. Interestingly, <br /> increase availability gene knockout data, stud-        predictive descriptors human disease genes <br /> ies undertaken attempt decipher            identified Kondrashov et al. [12] identified characteristics essential genes. analysis       study predictive microbial essential genes. <br /> essential genes numerous organisms, fundamental evo- <br /> lutionary mechanisms genomic fingerprints              initial step construction classifier, uncovered aid essential gene identification         explored various experimental genomic metrics organisms lacking experimental validation.                     assess relate essentiality E. coli S. <br />                                                                   cerevisiae. metric shown highly studies taken advantage abundance          predictive essentiality previous studies reten- <br /> experimental data available model organisms order          tion genes different phyla. order extract understand properties essential genes. exam-        metric identified subsets organisms <br /> ple, groups suggested relation-      achieving highest accuracy prediction essential <br /> ship degree protein-protein interaction                genes. predictive sets contained host-associated <br /> networks essentiality [4,5]. implication      organisms small genomes closely related <br /> hubs networks increased importance          query organism. result directly suggests abundance interaction partners. studies         targeted sequencing genomes used predic- revealed relationships essentiality          tion essential genes. rapidly decreasing invest- <br /> number transcription factor binding sites upstream        ments money time required gene [6]. demonstrated genes          sequencing microbial genome, approach iden- <br /> complex regulation enriched dispensable genes.             tifying essential genes targeted sequencing High accuracy predictions essential genes            itself viable alternative identification using flux balance analysis [7]. method            essential genes. advantage generating hypotheses regarding genes likely essential wide variety       addition phyletic retention, assessed rel- hypothetical conditions. little doubt       ative performance previously reported indicators plethora experimental data generated, addi-          essentiality protein interaction degree, protein <br /> tional properties essential genes documented        size codon bias. S. cerevisiae, protein interaction coming years.                                                 degree highly predictive essentiality, <br />                                                                   protein size codon bias predictive genome-wide experimental data abundant                organisms. Additionally, S. cerevisiae observed model organisms S. cerevisiae E. coli, informa-       high counts certain individual charged amino acids <br /> tion limited newly sequenced organisms,              predictive size  implying high precludes use data identification        counts amino acids distinguish proteins way <br /> genes essential survival. ability identify essential   completely captured size. <br /> genes absence experimental data added <br /> importance, allows rationally          Interestingly, aforementioned metrics select possible drug targets newly sequenced patho-           predictive essentiality organisms, relative <br /> gens. Fortunately, addition relationships               importance sets genes identified varied. essentiality various experimental measures,           striking difference relationship pro- good deal research aimed             tein size essentiality. E. coli, small proteins standing genomic features essential genes. Metrics         enriched essential genes, S. cerevisiae essential codon bias, number paralogs, phyletic              genes underrepresented smallest proteins. <br /> retention shown distinguishing             hypothesize indicative pressure <br />  <br />  <br />                                                                                                                       Page 2 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                     http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  E. coli maintain small proteins absence          Selection organisms phyletic retention measure functional constraints, previously sug-        plummeting cost genome sequencing making <br /> gested [13].                                                     comparative genomics attractive technique, crea- <br />                                                                  tive bioinoformatic methods advantage tar- identifying genomic features predictive        geted sequencing prevalent [14-16]. <br /> essentiality E. coli S. cerevisiae, quantified     vein, set understand evolutionary <br /> predictive limits assembled genomic characteristics       properties sets organisms presence integrating using probabilistic classifier      ortholog indicative essentiality <br /> junction feature selection criterion novel     gene, hopes sequencing appropri- <br /> bioinformatic applications. Using easily obtainable         ate genomes, high accuracy predictions essential genes <br /> genomic features S. cerevisiae E. coli,          Retention gene long evolutionary ability predict essential genes competitive       periods form allows recognition using sequence classifiers included experimental features.       similarity based techniques suggests performing <br /> fact able construct classifier using    critical function [17]. previous studies, sets organ- <br /> descriptors generated sequence data allow broad        isms varying distantly related organisms  <br /> application technique organisms,           eral closely related organisms utilized gene annotation required. ability        examination relationship retention potential impact understanding essential          genes essentiality [8,9]. knowledge, sys- <br /> genes different organisms, search drug     tematic analysis aim understanding nature <br /> targets poorly understood pathogens.                          set organisms presence gene                                                                  predictive essentiality performed. <br /> Results Discussion <br /> Experimental <span id='am-2' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> essentiality                          order identify set organisms analysis relationship various          retention gene predictive essentiality, genomic features essentiality, important        selected small group organisms likely pre- results context definition essentiality.    dictive E. coli essential list, computed definition used experiments based          accuracy subsets reduced set. strategy <br /> growth, lack thereof, mutants rich media         employed large number sequenced <br /> ditions. Clearly, conditions representative     organisms prevented examining possible sub- wild type environments organ-            sets. order assemble reduced set ranked <br /> isms inhabited evolved gene content. Study-        organisms based conditional mutual information <br /> ing metabolic pathways S. cerevisiae using flux balance       maximization criteria (CMIM), described Fleuret <br /> analysis indicates thirds metabolic       [18] detailed Methods section. Briefly, CMIM <br /> enzymes essential condition,             selection iterative feature selection procedure experiments presence rich media result              features selected highest <br /> roughly 20% genes labeled essential [7]. Fur-        mutual information class variable, conditioned <br /> thermore, analysis set genes           features selected. labeled dispensable knockout experiments,            procedure implemented organism ubiquitously present different phyla,        feature, feature vector stores occurrence revealed overrepresentation biosynthetic pathways          orthologs E. coli genes. defining class variable <br /> [8]. likely pathways essential wild   essentiality corresponding genes, CMIM <br /> type conditions, suggested retention         selection returns organisms highly predictive evolution, surplus nutrients provided          essentiality set. Using method selected 20 identified dispensable. Undoubtedly,        organisms set 180 sequenced prokaryotes room debate meaningful definition essenti-        looked predictive accuracy possible subsets ality,  general, genes       various sizes. utility approach validated required rich media conditions vital         comparing distribution Receiver Operating Charac- conditions.  feel despite           teristic (ROC) scores intelligently selected subsets inconsistencies, valuable understand    random sets organisms (Figure 1), sets <br /> properties artificial set essential genes,    organisms selected using maximal mutual information comprehensiveness set ques-           (data shown). compared performance tioned, accuracy limited experimen-        optimal subset organisms larger sets, tal bias.                                                        seen Figure 2 Table 1, little lost despite <br />                                                                  small number organisms used.  using                                                                  optimally selected organisms able achieve <br />  <br />  <br />                                                                                                                      Page 3 16 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                     http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br /> 45 <br />  <br />  <br /> 40               Subsets 5 CMIM 20 <br />  <br />                  Random Subsets 5 Set 180 <br /> 35 <br />  <br />  <br /> 30 <br />  <br />  <br /> 25 <br />  <br />  <br /> 20 <br />  <br />  <br /> 15 <br />  <br />  <br /> 10 <br />  <br />  <br />  5 <br />  <br />  <br />  0 <br />  0.5                    0.55                     0.6           0.65                             0.7                         0.75 <br />                                                   Area Curve <br /> Figure <br /> Distribution <br />          1 ROC areas curves E. coli essential gene prediction using CMIM selected subsets <br /> Distribution ROC areas curves E. coli essential gene prediction using CMIM selected subsets. Prob- <br /> ability density functions areas curves ROC analysis 100,000 randomly selected subsets organisms <br /> selected total 180 sequenced prokaryotes (blue) subsets 5 organisms selected set 20 turn selected based conditional mutual information selection procedure described text (red). Sensitivity specificity computed based numbers true positives (essential genes) false positives (non-essential genes) <br /> identified number organisms ortholog given E. coli gene present decreases. <br />  <br />  <br /> better performance using 180 sequenced            1%, compared subsets prokaryotes  Additional file 5 organisms used       selected reduced set 20. seen Fig- analysis].                                              ure 3A, organisms included                                                                 frequently best performing sets Buchnera evidence organism sets best        aphidicola Wigglesworthia glossinidia. possible, examined distribution organisms        organisms host-associated organisms belonging predictive sets. identified        Gamma proteobacteria, E. coli belongs. sets organisms achieved ROC score         explanation presence orthologs organ- <br />  <br />  <br />                                                                                                                     Page 4 16 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                 1 <br />  <br />               0.9 <br />  <br />               0.8 <br />  <br />               0.7 <br />  <br />               0.6 <br /> Sensitivity <br />  <br />  <br />  <br />  <br />               0.5 <br />  <br />               0.4 <br />  <br />               0.3                                                                   CMIM5 (AUC = 0.7004 ) <br />  <br />               0.2                                                                   CMIM7 (AUC = 0.6984) <br />  <br />                                                                                     Set 180 (AUC = 0.6481) <br />               0.1 <br />  <br />                 0 <br />                  0   0.1        0.2         0.3         0.4     0.5      0.6               0.7         0.8           0.9             1 <br />                                                           1 - Specificity <br /> Figure <br /> ROC curves <br />          2 different sets organisms E. coli essential gene prediction <br /> ROC curves different sets organisms E. coli essential gene prediction. ROC curves based methodology used Figure 1. red optimal set 5 organisms selected CMIM set 20, green optimal set 7 selected CMIM set 20, blue set 180 prokaryotes used study. area curve (AUC) indicated graph. <br />  <br />  <br /> isms predictive essentiality E. coli       genome partly dependant sequence loss. clear considering evolutionary history life-        words, presence alternative pathways allows <br /> style.  organisms share relatively recent com-       viable minimal sets arise process <br /> mon ancestor E. coli, adjustment          reductive evolution, making intersection gene host associated lifestyle underwent massive gene            sets organisms informative [21]. <br /> loss, minimal genetic novelty [19,20]. pres-           second property organisms accounts <br /> ence great selective pressure minimize genome           superiority prediction essentiality relates size, majority genes retained absolutely         working definition essentiality survival essential. reason organisms perform bet-           rich media conditions. rich media conditions <br /> ter gene set given reduced          argued unrepresentative wild type condi- <br />  <br />  <br />                                                                                                                        Page 5 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Table 1: Influence organism composition phyletic retention performance <br />  <br />       Organisms Present                 Cumulative PPV             Number Genes Group        Number Essential Genes <br />  <br />                                                       CMIM Selected Set 5 <br />                5                              74.8%                             139                               104 <br />                4                              66.8%                             271                               181 <br />                3                              55.6%                             435                               242 <br />                2                              42.0%                             690                               290 <br />                1                              28.9%                            1302                               376 <br />                0                              16.8%                            3636                               612 <br />                                                      CMIM Selected Set 7 <br />                7                              77.0%                          122                                  94 <br />                6                              73.3%                          206                                  151 <br />                5                              65.3%                          311                                  203 <br />                4                              54.1%                          455                                  246 <br />                3                              43.0%                          633                                  272 <br />                2                              34.8%                          952                                  331 <br />                1                              25.1%                         1613                                  405 <br />                0                              16.8%                         3636                                  612 <br />                                          Set 27 Non-parasitic Gamma Proteobacteria <br />                27                             59.0%                          305                                  180 <br />                24                             46.6%                          545                                  254 <br />                20                             34.5%                          861                                  297 <br />                16                             27.3%                         1247                                  341 <br />                12                             23.9%                         1697                                  405 <br />                 8                             20.3%                         2235                                  453 <br />                 4                             18.3%                         2845                                  521 <br />                                                      Set 179 prokaryotes <br />               179                             75.0%                          12                                    9 <br />               161                             75.3%                          77                                   58 <br />               143                             64.8%                          196                                  127 <br />               126                             54.4%                          294                                  160 <br />               108                             48.4%                          419                                  203 <br />               90                              39.7%                          572                                  227 <br />               72                              33.0%                          817                                  270 <br />               54                              26.8%                         1172                                  314 <br />               37                              22.4%                         1659                                  371 <br />               19                              19.2%                         2379                                  457 <br />                1                              17.0%                         3516                                  596 <br />  <br />  <br />  <br />  <br /> tions organisms, conditions fairly            Performance individual features yeast <br /> accurate representation wild type conditions organ-         extended analysis essential genes gathering <br /> isms resides inside host, utilizing available          various data sets representative different aspects <br /> nutrients.  having organisms preferable,       yeast biology quantify abilities exact set nutrients provided host vary,         distinguish essential non-essential genes. intersection gene sets best captures generic           value ability predict essentiality studied <br /> host-associated lifestyle [20].                                    organisms, data sets focused features                                                                    easily obtainable;  features generated <br /> Performing similar analysis S. cerevisiae using set 26     need extensive laboratory work. fea- <br /> sequenced eukaryotes returned Schizosaccharomyces pombe,           tures fall general categories: (1) genomic fea- <br /> Encephalitozoon cuniculi, Eremothecium gossypii       tures, based solely sequence data, contain abundant organisms highly predictive              features open reading frame (ORF) size, upstream <br /> sets (Figure 3B). E. cuniculi E. gossypii lead    size, phyletic retention; (2) protein features, host-associated lifestyles, corroborating interpretation       based solely protein characteristics contain optimal organism sets E. coli. Furthermore, E. gos-      data sets like <span id='am-3' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-4' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> composition, codon bias, sypii smallest known eukaryotic genomes,              hydrophobicity. reference comparing  9.2 Mb nuclear genome [22].                                 formance genomic protein features,  <br />  <br />                                                                                                                         Page 6 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                                                                                                  Wigglesworthia brevipalpis <br />                                                                                     Haemophilus ducreyi <br />                                                                                     Neisseria gonorrhoeae <br />                                                                                     Blochmannia floridanus <br />                                                                                     Wolinella succinogenes <br />                                                                                     Chlamydophila caviae <br />                                                                                     Phytoplasma OY <br />                                                                                     Bartonella henselae <br />                                                                                     Mycoplasma penetrans <br />                                                                                     Campylobacter jejuni <br />                                                                                     Helicobacter pylori <br />                                                                                     Francisella tularensis <br />                                                                                     Buchnera aphidicola <br />                                                                                     Tropheryma whipplei <br />                                                                                     Treponema pallidum <br />                                                                                     Mycoplasma mobile <br />                                                                                     Methylococcus capsulatus <br />                                                                                     Coxiella burnetii <br />                                                                                     Zymomonas mobilis <br />                                                                                     Fusobacterium nucleatum <br />             160     140     120     100      80      60        40     20        0 <br />  <br />              B <br />                                                                                     Homo sapiens <br />                                                                                     Ashbya gossypii <br />                                                                                     Encephalitozoon cuniculi <br />                                                                                     Schizosaccharomyces pombe <br />                                                                                     Trichomonas vaginalis <br />                                                                                     Cryptosporidium parvum <br />                                                                                     Entamoeba histolytica <br />                                                                                     Dictyostelium discoideum <br />                                                                                     Candida albicans <br />                                                                                     Plasmodium falciparum <br />                                                                                     Cryptosporidium hominis <br />                                                                                     Arabidopsis thaliana <br />                                                                                     Drosophila melanogaster <br />                                                                                     Tetrahymena thermophila <br />                                                                                     Giardia lamblia <br />                                                                                     Trypanosoma brucei <br />                                                                                     Cyanidioschyzon merolae <br />                                                                                     Cryptococcus neoformans <br />                                                                                     Neospora crassa <br />                                                                                     Caenorhabditis elegans <br />  <br />              150                   100                    50                    0 <br /> Figure 3 organism occurrence performing organism sets <br /> Distribution <br /> Distribution organism occurrence performing organism sets. Histograms display frequency CMIM selected 20 organisms 1% organism sets 5   E. coli (b) S. cerevi- <br /> siae. Note organisms displayed histograms left right based order selection CMIM <br /> selection procedure. seen graphs trend organisms selected earlier present best performing sets, validates use selection method. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 7 16 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> included data sets derived lab-intensive experiments         present largest 10% proteins. protein-protein interaction cellular localiza-        individual function attained significant p- <br /> tion data. positive predictive value (PPV) selected        value, functions present involved cat- <br /> single features varying coverage displayed Figure        alytic activity interaction nucleic acids. Charged <br /> 4A. complete list features used,          amino acids present active sites rationale including  detailed Methods          enzymes, participate catalytic mechanisms. <br /> Additional file 3 actual feature matrix used].                Additionally, charged amino acids make associations                                                                   charged substrates favorable electrostatic <br /> Genomic features                                                  interactions. Based observations, hypothesize expected, best performing genomic feature          enrichment essential genes pro- <br /> phyletic retention measure, construction                teins high counts charged amino acids described  clarity, term &apos;phyletic retention&apos;       functional capabilities amino used presence ortholog         acids. <br /> organisms, place term &apos;conservation,&apos; order prevent confusion measures substitution rate.           Experimental features <br /> second feature genomic set predictive          addition examining predictive power features essentiality total upstream size gene. Genes      genomic protein set, measured largest upstream sizes markedly enriched          prediction accuracy experimentally derived fea- <br /> dispensable genes. result explained              tures previously reported indicative essentiality. <br /> considering recent results Yu et al. showing          observed genes high degree protein <br /> genes complex regulation enriched dispensa-           interaction network likely essential, ble genes, conjunction possibility genes         agreement previous work [5,24,25]. complex regulation larger upstream             proteins 5% degree, 42% essential, <br /> regions order accommodate increased number            considerable enrichment essential genes com- <br /> cis elements [8]. connection previously             pared ~17% expected chance  shown valid Caenorhabditis elegans Drosophila        noted stated literature rela- <br /> melanogaster [23]. results suggest relationship        tionship degree essentiality partly <br /> exists regulatory complexity intergenic dis-          biases data [26]. Specifically, Coulumb et al. <br /> tance S. cerevisiae relationship accounts    state protein interaction dataset Data- association intergenic distance essential-        base Interacting Proteins (DIP), used ity. Given number transcription factor binding        analysis, biased essential genes accu- <br /> sites present promoter gene determined             mulation interactions small scale experiments arduous experimental procedures, beneficial         partial essential genes. authors able use upstream size proxy regulatory         state partiality accounts significant <br /> complexity.                                                       component relationship degree essen- <br />                                                                   tiality. contention substantiated showing <br /> Protein features                                                  disappearance relationship essentiality <br /> Examining single features protein subset                 connectivity using unbiased genome <br /> revealed highly predictive essentiality.      yeast hybrid experiments. recently stated addition previously discussed descriptors           lack reliability completeness yeast  <br /> codon bias protein size, identified enrich-           hybrid data responsible absence relation- <br /> ment essential genes proteins abun-              ship connectivity essentiality [27]. <br /> dance certain amino acids. Specifically, proteins highest counts aspartate, glutamate, lysine        Performance individual features E. coli <br /> enriched essential genes PPVs 29.6%, 31.5%             Despite large evolutionary distance fundamental 30.0% respectively 10% predictions.             biological differences, characteristics essential genes trend partially explained observing large         S. cerevisiae E. coli largely similar. seen Fig- <br /> proteins general enriched essential genes,       ure 4B, strongest predictor essentiality E. coli <br /> PPV 25.8% largest 10% proteins.                   phyletic retention, CAI. Number paralogs relationship explains high amino            protein size predictive essentiality acid counts predictive essentiality, fails fully   organisms. Features E. coli genes used clarify specific amino acids predictive         study available Additional files  Additional file  order gain insight phenomenon,         4]. <br /> looked enrichment molecular function catego- <br /> ries proteins high counts charged amino acids <br />  <br />  <br />                                                                                                                        Page 8 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                          http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                                                        S. cerevisiae <br />           60 <br />                                                                                                               Degree <br />           50                                                                                                  <span id='am-10' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-11' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span>Glutamic Acid</span> <br />                                                                                                               Lysine <br />           40                                                                                                  <span id='am-22' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Glycine</span> <br /> PPV (%) <br />  <br />  <br />  <br />  <br />           30                                                                                                  ORF Size <br />                                                                                                               Paralogs <br />           20                                                                                                  CAI <br />           10                                                                                                  TFBS Count <br />                                                                                                               Upstream Size <br />           0                                                                                                   Random <br />                              1           5              10             15             20 <br />                                     Percentage Predictions <br />  <br />  <br />           B <br />                                                    E. coli <br />           60 <br />  <br />           50 <br />  <br />           40 <br />                                                                                                               CAI <br /> PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                               Paralogs <br />           30                                                                                                  ORF Size <br />                                                                                                               Serine <br />           20 <br />                                                                                                               Random <br />           10 <br />  <br />            0 <br />                             1           5              10             15              20 <br />                                      Percentage Predictions <br /> Comparison <br /> Figure 4 single feature classification accuracies essential genes <br /> Comparison single feature classification accuracies essential genes. Displays accuracy single features yeast   E. coli (B) classifying essential genes. Predictions shown cutoffs 1, 5, 10, 15 20% (x-axis), accuracy, measured PPV, shown y-axis. predicting essentiality, single features ordered descending order, exception ORF size Serine E. coli figure (B). <br />  <br />  <br />  <br />  <br /> Differences feature performance S. cerevisiae             example, features protein size E. coli                                                          codon bias predictive essentiality, accuracy features performed comparably                sets proteins identified varied organisms, noticeable differences.              organisms. <br />  <br />  <br />                                                                                                                           Page 9 16 <br />                                                                                                     (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Protein size predictive essentiality organ-        conserved essential genes. set, designated isms. analysis determined small pro-         SC_All, composed features require extensive <br /> teins E. coli enriched essential genes, small     experimentation, addition easily obtainable fea- <br /> proteins S. cerevisiae slightly enriched dispensa-      tures, assess impact neglecting <br /> ble genes. order gain insight discrepancy         experimental data prediction accuracy. benefit organisms examined distribution            using na&#195;&#175;ve Bayes feature integration clas- essentiality small large proteins organ-        sification assigned probability, making natural isms context phyletic retention. seen       rank predictions, allows direct comparison <br /> Figure 5, marked difference distribution     results using individual features. <br /> large small essential genes dissected bins <br /> based phyletic retention. E. coli, virtually   Feature selection accomplished ranking features largest proteins essential ubiquitously         using conditional mutual information maximization <br /> present 27 sequenced non-parasitic                 (CMIM), described Methods  Additional file 1 Gamma proteobacteria assembled (Figure 5B).                 actual ranking]. phyletic retention feature <br /> hypothesize enrichment essential genes           achieved highest mutual information essential- conserved large proteins indica-            ity, consistent results single feature <br /> tive pressure reduce size individual proteins.     performance. using 21 informative features Larger proteins come increased cost cell          SC_All, 11 SC_GenProt 13 SC_GenProt_No, terms raw materials energy expenditures dur-          able improve prediction accuracy inclu- <br /> ing protein synthesis. idea consistent findings      sion features set  Additional file 2]. Lipman et al., observed increased conser- <br /> vation largest proteins genomes          Overall performance seen Figure 6A, attribute pressure maintain small proteins        positive predictive value (PPV) 1, 5, 10, 15 absence functional constraints [13].        20% predictions shown reference selected single enrichment essential genes small proteins         features. Unexpectedly, SC_All outperforms levels conservation conserved large           SC_GenProt, small (~5% difference <br /> proteins supportive hypothesis.                  15 20% predictions, Figure 5A),                                                                   indicates losing little ignoring features <br /> Figure 5C yields insight lack essential genes         derived experimental data. expected, smallest S. cerevisiae ORFs. enrichment          SC_GenProt_No greatly outperformed dispensable genes amid small ORFs S. cerevisiae          feature sets. consistent results shown consequence abundance small species spe-         single feature analysis, phyletic retention <br /> cific genes. organisms species specific genes         measure far predictive feature. <br /> enriched dispensable genes, expected <br /> based predictive power phyletic retention            SC_GenProt_No performing significantly worse identifying essential genes.  abundance           feature sets, use identifying espe- <br /> small species specific genes leads apparent trend       cially interesting genes. assess ability dispensability smallest S. cerevisiae genes.         SC_GenProt_No identify essential genes noted organisms used phyletic          conserved, looked broader conservation pattern <br /> conservation analysis set 16 sequenced fungi            yeast genes set 16 fungi protists. Based protists, diverse set genomes         set 16 organisms, 285 essential yeast used optimal predictive set.                          genes orthologs 5 organisms                                                                    15% predictions SC_GenProt_No, 24.6% Performance integrated features yeast                       285 conserved essential genes identified. determine limits predictive abilities        contrast, 3.5% 285 conserved essential <br /> classification essential genes integrating multiple       genes identified SC_GenProt cutoff. <br /> features, utilized na&#195;&#175;ve Bayes classifiers. assigned         SC_GenProt_No lowest accuracy features different overlapping sets,     integrated feature sets, useful order assess relative contributions different sub-      increased ability predict conserved essential genes. <br /> sets features. set, designate SC_GenProt, composed features              Performance integrated features E. coli <br /> obtained directly sequence data. second set,             yeast, feature sets integrated na&#195;&#175;ve Bayes clas- designated SC_GenProt_No, identical             sifier used predict essentiality E. coli. sets <br /> SC_GenProt, lacks phyletic retention measure.          containing easily obtainable features analyzed, <br /> included set order assess ability identify      EC_GenProt EC_GenProt_No. experimental fea- <br />  <br />  <br />                                                                                                                       Page 10 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                          http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                                                                           B <br />            40                                                               45 <br />  <br />            35                                                               40 <br />  <br />                                                                             35 <br />            30 <br />                                                                             30 <br /> # Genes <br />  <br />  <br />  <br />  <br />            25 <br />                                                                             25 <br />            20 <br />                                                                             20 <br />            15 <br />                                                                             15 <br />            10 <br />                                                                             10 <br />             5                                                                 5 <br />  <br />             0                                                                 0 <br />                 0 2 4 6 8 10 12 14 16 18 20 22 24 26                              0 2 4 6 8 10 12 14 16 18 20 22 24 26 <br />  <br />  <br />  <br />            C                                                                D <br />           120                                                               45 <br />  <br />                                                                             40 <br />           100 <br />                                                                             35 <br />  <br />            80                                                               30 <br /> # Genes <br />  <br />  <br />  <br />  <br />                                                                             25 <br />            60 <br />                                                                             20 <br />  <br />            40                                                               15 <br />  <br />                                                                             10 <br />            20 <br />                                                                               5 <br />  <br />             0                                                                 0 <br />                 0   2   4   6    8    10 12 14 16                                 0    2     4    6      8    10 12 14 16 <br />  <br />                         Phyletic Retention                                                 Phyletic Retention <br /> Figure <br /> Phyletic 5distribution smallest largest essential genes S. cerevisiae E. coli <br /> Phyletic distribution smallest largest essential genes S. cerevisiae E. coli. Displays number essential genes (blue) superimposed total number genes (gray), binned number organisms gene present (phyletic retention)   300 smallest proteins E. coli, (B) 300 largest proteins E. coli, (C) 300 small- <br /> est proteins S. cerevisiae (D) 300 largest proteins S. cerevisiae. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 11 16 <br />                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br />  <br />                                          S. cerevisiae <br />                  100 <br />  <br />                      90 <br />  <br />                      80 <br />  <br />                      70 <br />  <br />                      60                                                                        SC_All <br />            PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                SC_GenProt <br />                      50                                                                        SC_GenProt_No <br />                      40                                                                        Random <br />  <br />                      30 <br />  <br />                      20 <br />  <br />                      10 <br />  <br />                       0 <br />                                 1        5         10           15         20 <br />                                        Percentage Predictions <br />  <br />  <br />  <br />  <br />                      B                         E. coli <br />                  100 <br />  <br />                      90 <br />  <br />                      80 <br />  <br />                      70 <br />                                                                                                EC_GenProt <br />                      60 <br />            PPV (%) <br />  <br />  <br />  <br />  <br />                                                                                                EC_GenProt_No <br />                      50                                                                        Random <br />                      40 <br />  <br />                      30 <br />  <br />                      20 <br />  <br />                      10 <br />  <br />                      0 <br />                                1        5          10           15         20 <br />                                       Percentage Predictions <br />  <br /> Figure <br /> Comparison <br />         6 accuracies na&#195;&#175;ve Bayes classifiers <br /> Comparison accuracies na&#195;&#175;ve Bayes classifiers. predictive accuracies integrating optimal feature sets using na&#195;&#175;ve Bayes classifier shown yeast   E. coli (B). Accuracy, measured PPV (y-axis) shown 1, 5, <br /> 10, 15 20% predictions different feature sets. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 12 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                      http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> ture set used lack available genome wide         process constructing integrated classifier, analyses.                                                        relationship various genomic characteristics                                                                  essentiality explored. E. coli S. cerevisiae, phyletic retention measure  yeast,        phyletic retention, protein size, codon bias iden- informative feature ranking condi-           tified single features predictive <br /> tional mutual information [Additional file 1].           essentiality. Furthermore, showed pre- yeast obtained best PPV using 13            dictive groups organisms used phyletic retention <br /> features, E. coli required  phyletic reten-    measure contain host-associated organisms tion, serine, tryptophan paralog count (9 features           closely related reference organism. Despite optimally classify EC_GenProt_No). Figure          influence artificial definition essentiality 6B shows performance integrated features E.        selection optimal genome sets, result coli, PPV shown 1, 5, 10, 15 20%        useful suggesting targeted sequencing used predictions.                                                  identification essential genes organisms. <br />                                                                  addition phyletic retention codon bias, Conclusion                                                       related essentiality previous studies, identification essential genes largely        identified relationship protein size essenti- <br /> experimental effort, achieved genome               ality, knowledge explored <br /> knockout techniques. organisms C.           Specifically, observed nature elegans, possible devise highly effective screens   relationship differed E. coli S. cerevisiae, small <br /> essential genes using siRNAs [28], cost ineffec-      protein size indicative essentiality E. coli tiveness technique organisms makes          true large proteins S. cerevisiae.  <br /> broad application currently infeasible. paper          largest E. coli proteins, assessed potential effectiveness methodology         conserved essential. hypothesize genes prioritized based likelihood      observations indicative pressure testing positively lethality screen      maintain small proteome E. coli. <br /> subsequent small scale knockout screens  <br /> formed predictions obtain experimentally           summary strides prediction <br /> validated genes.                                                 essential genes based solely sequence data                                                                  fronts.  gained insight properties investigated efficacy strategy using avail-    sets organisms presence ortholog able knockout experiments assess predictive power         predictive essentiality. Second, assessed features easily obtainable sequence data        predictive power sequence based features, integrating using machine learning meth-           achieved superior prediction accuracy inte- <br /> odologies. integrating genomic protein character-         gration probabilistic framework intelligent fea- <br /> istics varying predictive power using probabilistic         ture selection. <br /> classifier feature selection, able achieve overall predictive accuracy S. cerevisiae E. coli    Methods superior performance individual           Sets essential genes <br /> feature. use descriptors make clas-      Essential gene definitions taken Giaever et al[1] <br /> sifier robust using individual features          Gerdes et al[30] S. cerevisiae E. coli respec- <br /> predictive power likely vary great deal differ-    tively. Additionally, E. coli ORFs removed <br /> ent organisms. example, codon bias strong predic-       80 amino acids. S. cerevisiae tor essential genes organisms studied        ORFs removed FASTA headers contained study 80 bacterial genomes revealed 30%           key words &quot;transposable&quot; &quot;mitochondrial&quot;. total codon bias [29]. Furthermore, able classify        4,728 yeast genes used, 966 essential. essential genes reasonable accuracy          E. coli, 3569 genes used, 611 essential. use gene conservation measure phyletic <br /> retention, providing added benefit identifying            Selection Features <br /> essential genes organism specific. abil-        S. cerevisiae E. coli model organisms ity identify essential genes sequence data         studied years. capitalized potential great practical importance         fact assessing relationship variety <br /> guiding investigations researchers searching          gene properties essentiality. Following list potential drug targets newly sequenced pathogens.             predictors rational including  Note                                                                  features used just S. cerevisiae marked                                                                  star, used E. coli marked stars. <br />  <br />  <br />                                                                                                                      Page 13 16 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                       http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Experimental Characteristics                                      comparison (downloaded UCSC Genome following parameters require large exper-         Browser; conserved track). <br /> imental work obtain. features used classifiers integrating features, excluded       Phyletic retention measure using &apos;easily obtainable&apos; features.               Genes ubiquitously present different taxa <br />                                                                   likely essential [1]. detailed *Protein interaction network degree                               Results section, yeast E. coli separately, set Generated curated interactions accumulated            organisms selected optimally predicted essenti- Database Interacting Proteins (DIP) [31].              ality. count gene reference <br /> degree protein computed summing number             organism (yeast E. coli) represents number of unique interactions participates  Degree,          orthologs present organisms. Bi-directional related metrics network position,         best BLAST hits used define orthologous rela- <br /> documented literature indicative essen-        tionship (using E-value cutoff 0.1). <br /> tiality [5,24,25]. Protein interaction data included E. coli low coverage.                                  Number paralagous genes <br />                                                                   Genes duplicates likely *Protein Localization                                             essential [1]. rational duplicates func- known proteins transcriptional regula-         tion backup capacity presence knockout <br /> tion annotations enriched essential genes [1]. Based       mutation original gene. Paralogs defined result tested nuclear localiza-         genes present genome <br /> tion, protein localization categories,       BLASTP E-value 10-20. addition <br /> useful predicting essentiality. Localization information       ratio larger gene smaller exceed obtained previous large-scale study [32].              1.33. <br />  <br /> *Recombination Rate                                               **Strand bias <br /> Clusters essential genes known regions         Essential genes likely encoded lead- genome characterized lower recombina-           ing strand circular chromosome [35]. <br /> tion rate [33]. gene recombination rates acquired Gerton et al. [34], analyzed according          Protein Characteristics procedures used Pal Hurst [33].                         parameters fall &apos;easily obtainable&apos; category                                                                   require coding sequence  labo- <br /> Genomic Characteristics                                           ratory experiments necessary. represent protein consider following parameters &apos;easily obtainable&apos;          characteristics terms following metrics: amino automatically generated sequence         acid composition, codon bias, codon adaptation index <br /> data.                                                             (CAI), frequency optimal codons (FOP), isoelectric <br />                                                                   point (PI), hydropathicity score, hydrophobicity <br /> Gene size                                                         score. yeast data downloaded Saccha- trend proteins larger         romyces Genome Database [36]. E. coli metrics <br /> evolution. expected gene size            generated using CodonW software package [37]. <br /> indicative essentiality, especially E. coli, ancestral <br /> genes likely essential.                                       Integration features <br />                                                                   Classification essential genes using *Regulatory complexity                                            Orange machine learning package&apos;s implementation Different genes S. cerevisiae exhibit wide variation        na&#195;&#175;ve Bayes. [38]. features, exception regulatory complexity. recently docu-           optimized phyletic retention measure binary <br /> mented relationship regulatory            localization features, discretized using Fayyad complexity essentiality [6]. measured regulatory           Irani&apos;s entropy discretization method [39], imple- <br /> complexity using following parameters: upstream size,         mented Orange. <br /> downstream size, upstream conservation  <br /> stream conservation. sizes measured dis-          discretization, conditional mutual information max- <br /> tance nearest gene. Conservation measured           imization criteria (CMIM), described Fleuret [18], number bases   1000 bp upstream            used rank features. Briefly, CMIM iterative ORF start site   300 bp downstream            method feature vectors selected designated open reading frame, overlap ele-         highest mutual information class vector ments identified conserved seven species            conditioning previously selected features vectors. <br />  <br />  <br />                                                                                                                       Page 14 16 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> Let Y class vector, Xn feature, Y;Xn) mutual information Y Xn, Y; Xn | Xm)                            Additional File 3 conditional mutual information Y Xn                            Feature matrices S. cerevisiae. raw data feature matrix, conditioned Xm. CMIM implemented follows,                               entropy discretized feature matrix included. <br />                                                                                   Click file alternate implementations described Fleuret [18]. <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- score table s initialized values Y ; Xn).                     2164-7-265-S3.xls] <br /> algorithm picks iteration feature Xm highest score, refreshes score s [n] taking                     Additional File 4 minimum s [n] (Y ; Xn | Xm). algorithm                           Feature matrices E. coli. raw data feature matrix, run features selected.                                     entropy discretized feature matrix included. <br />                                                                                   Click file given feature set, optimal number features                        [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                   2164-7-265-S4.xls] <br /> include classification determined empirically. Fea- <br /> tures ranked CMIM, iteratively removed                            Additional File 1 time. interval classification accuracy                       CMIM feature ranking. excel file includes tables showing measured. cutoff optimal feature set iden-                        CMIM feature ranking. <br /> tified highest PPV 5% pre-                        Click file <br /> dictions, requirement PPV 1%                            [http://www.biomedcentral.com/content/supplementary/1471- higher  Additional file 2].                                           2164-7-265-S1.xls] <br />  <br />  assign probability essentiality genes,                                                                                   Additional File 2 <br />                                                                                   Performance na&#195;&#175;ve Bayes classifiers using subsets features. following procedure used. Half essential                           set features analyzed paper (e.g. SC_GenProt, <br /> half non-essential genes randomly chosen                           EC_GenProt, ..), CMIM calculated features included training set. classifier                          ranked order informative  PPV 1, 5, 10 tested remaining genes, assigns probability                        15% predictions shown na&#195;&#175;ve Bayes classifier constructed essentiality. Training/testing bootstrapped 100                            using N features. <br />                                                                                   Click file <br /> times, gene probability essentiality                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> taken average probabilities                            2164-7-265-S2.xls] <br /> assigned  Additional file 6]. <br />                                                                                   Additional File 6 KEGG enrichment analysis                                                   Na&#195;&#175;ve Bayes classification results. feature sets used E. GeneMerge software used calculate enrichment                           coli S. cerevisiae, probability gene essential, KEGG annotations, using background genes                              reported na&#195;&#175;ve Bayes, provided. <br />                                                                                   Click file <br /> used given organism [40]. P-values given Bon- <br />                                                                                   [http://www.biomedcentral.com/content/supplementary/1471- <br /> ferroni corrected.                                                                2164-7-265-S6.xls] <br />  <br /> Authors&apos; contributions <br /> AMG, SCJP, ESS SK conceived project designed study. AMG ESS drafted manuscript.                      Acknowledgements <br /> SCJP ESS performed analysis single feature                        authors like thank Rich Roberts comments formance. AMG performed integrated classifier analysis.                      manuscript, Joseph Mellor Dustin Holloway valuable <br /> ESS performed analysis identifying optimal organism                       insights project. work supported NIH grants <br /> sets. CD SK directed research, provided feed-                    1P20GM066401, 1T32GM070409, R01 HG003367-01A1 NSF grant project. authors read                            ITR-048715 <br /> approved final manuscript. <br />                                                                              References <br />                                                                              1.      Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br /> Additional material                                                                  Lucau-Danila  Anderson K, Andre B, Arkin AP, Astromoff  El- <br />                                                                                      Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br />                                                                                      M, Davis K, Deutschbauer  Entian KD, Flaherty P, Foury F, Garfinkel <br />  Additional File 5                                                                   DJ, Gerstein M, Gotte D, Guldener U, Hegemann JH, Hempel S,  <br />                                                                                      man  Jaramillo DF, Kelly  Kelly SL, Kotter P, LaBonte D, Lamb <br />  List organisms used calculate phyletic retention. list organ-            DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard <br />  isms used calculation phyletic retention shown. KEGG             P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Sche- <br />  letter codes used represent organisms, unless noted.           rens B, Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, <br />  Click file                                                                 Storms RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang CY, <br />  [http://www.biomedcentral.com/content/supplementary/1471-                           Ward TR, Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu <br />                                                                                      K, Bussey H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnston <br />  2164-7-265-S5.xls]                                                                  M: Functional profiling Saccharomyces cerevisiae <br />                                                                                      genome. Nature 2002, 418:387-391. <br />  <br />  <br />                                                                                                                                             Page 15 16 <br />                                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:265                                                                        http://www.biomedcentral.com/1471-2164/7/265 <br />  <br />  <br />  <br /> 2.    Cullen LM, Arndt GM: Genome-wide screening gene func-                25.   Wuchty S: Evolution topology yeast protein inter- <br />       tion using RNAi mammalian cells. Immunol Cell Biol 2005,                    action network. Genome Res 2004, 14:1310-1314. <br />       83:217-223.                                                              26.   Coulomb S, Bauer M, Bernard D, Marsolier-Kergoat MC: Gene <br /> 3.    Roemer T, Jiang B, Davison J, Ketela T, Veillette K, Breton  Tandia          essentiality topology protein interaction net- <br />       F, Linteau  Sillaots S, Marta C, Martel N, Veronneau S, Lemieux S,           works. Proc Biol Sci 2005, 272:1721-1725. <br />       Kauffman S, Becker J, Storms R, Boone C, Bussey H: Large-scale           27.   Batada NN, Hurst LD, Tyers M: Evolutionary physiological <br />       essential gene identification Candida albicans applica-                 importance hub proteins. PLoS Comput Biol 2006, 2:e88. <br />       tions antifungal drug discovery.              Mol Microbiol 2003,     28.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana- <br />       50:167-181.                                                                    pin  Le Bot N, Moreno S, Sohrmann M, Welchman DP, Zipperlen <br /> 4.    Maslov S, Sneppen K: Protein interaction networks arti-                 P, Ahringer J: Systematic functional analysis Caenorhab- <br />       facts. FEBS Lett 2002, 530:255-256.                                            ditis elegans genome using RNAi. Nature 2003, 421:231-237. <br /> 5.    Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central-        29.   Sharp PM, Bailes E, Grocock RJ, Peden JF, Sockett  Variation       ity protein networks. Nature 2001, 411:41-42.                               strength selected codon usage bias bacteria. <br /> 6.    Yu H, Greenbaum D, Xin Lu H, Zhu X, Gerstein M: Genomic anal-                  Nucleic Acids Res 2005, 33:1141-1153. <br />       ysis essentiality protein networks. Trends Genet 2004,         30.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />       20:227-231.                                                                    MD, Somera AL, Kyrpides NC, Anderson  Gelfand MS, Bhattacharya <br /> 7.    Papp B, Pal C, Hurst LD: Metabolic network analysis                      Kapatral V, D&apos;Souza M, Baev MV, Grechkin Y, Mseeh F, Fonstein <br />       causes evolution enzyme dispensability yeast.                         Overbeek R, Barabasi AL, Oltvai ZN, Osterman AL: Experi- <br />       Nature 2004, 429:661-664.                                                      mental determination level analysis essential <br /> 8.    Fang G, Rocha E, Danchin  essential nonessential                     genes Escherichia coli MG1655.                  J Bacteriol 2003, <br />       genes? Mol Biol Evol 2005, 22:2147-2156.                                       185:5673-5684. <br /> 9.    Chen Y, Xu D: Understanding protein dispensability               31.   Xenarios  Rice DW, Salwinski L, Baron MK, Marcotte EM, Eisenberg <br />       machine-learning analysis high-throughput data. Bioinfor-                   D: DIP: database interacting proteins. Nucleic Acids Res <br />       matics 2005, 21:575-581.                                                       2000, 28:289-291. <br /> 10.   Gu  Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH: Role            32.   Huh WK, Falvo JV, Gerke LC, Carroll  Howson RW, Weissman <br />       duplicate genes genetic robustness null mutations.                  JS, O&apos;Shea EK: Global analysis protein localization bud- <br />       Nature 2003, 421:63-66.                                                        ding yeast. Nature 2003, 425:686-691. <br /> 11.   Smith NG, Eyre-Walker  Human disease genes: patterns               33.   Pal C, Hurst LD: Evidence evolution gene order       predictions. Gene 2003, 318:169-175.                     <br /> </body></html>