<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Inhibition HIV-1 reverse transcriptase RNA aptamers Escherichia coli <br />  <br /> ABSTRACT better understanding aptamer function bacteria help establish simple model systems screening RNA&#226;&#8364;&#8220;protein <br /> interactions intracellular context. Escherichia coli DNA polymerase mutants (Pol  fail grow 37&#194;&#176;C unless exogenous DNA polymerase HIV-1 reverse transcriptase (RT) expressed cell.  RNA <br /> aptamers inhibit HIV-1 RT vitro block complementation HIV-1 RT expressed vivo. essential <br /> functions impaired aptamer expression temperature. Intracellular aptamer RNA concentrations induced <br /> cultures measured range 76 180 nM, comparable exogenously expressed HIV-1 RT levels cells. RT polymerase activity reduced background levels cell-free extracts prepared cultures expressing HIV-1 RT 70.28 aptamer, compared extracts cultures expressing HIV-1 RT  Intracellularly expressed <br /> RNA aptamers used generate conditional null mutants bacteria titrating essential protein. <br /> Keywords: SELEX; aptamer; RNA&#226;&#8364;&#8220;protein interactions; bioassay; HIV-1 reverse transcriptase <br />  <br />  <br />  <br /> INTRODUCTION                                                           exerts observable influence biology cells <br />                                                                        (Burke Nickens 2002). known aptamer <br /> Treatment infection human immunodeficiency virus <br />                                                                        function bacterial cells. translational regulation <br /> (HIV) generally includes small molecule inhibitors                                                                        small molecules observed engineered <br /> viral reverse transcriptase (RT).  drug-resistant <br />                                                                        (Werstuck Green 1998) natural systems (Nahvi et <br /> forms RT selected prolonged treatment, prompt- <br />                                                                        al. 2002; Winkler et al. 2002a,b), enzyme antagonism ing widespread efforts generate new anti-RT drugs.                                                                        aptamers bacteria reported. Bacteria <br /> RNA ssDNA aptamers bind HIV-1 RT high <br />                                                                        offer advantages studying RNA&#226;&#8364;&#8220;protein interac- <br /> affinity identified SELEX protocol <br />                                                                        tions, including ease genetic manipulation lower cost <br /> (Tuerk et al. 1992; Schneider et al. 1995; Burke et al. 1996; <br />                                                                        maintaining cell lines. time, number Andreola et al. 2001). different RNA <br />                                                                        characteristics bacterial microenvironment af- <br /> aptamers expressed human T293 cells, suscepti- <br />                                                                        fect aptamer function, including short average lifetime <br /> bility cells HIV infection reduced,                                                                        bacterial mRNA transcripts, counterion concentrations, <br /> infectivity viruses produced infected T293 cells  <br />                                                                        macromolecular constituents. better understanding <br /> duced 90%&#226;&#8364;&#8220;99% compared cells expressing <br />                                                                        aptamer function bacteria help establish aptamers (Joshi Prasad 2002). Human T-lymphoid cell <br />                                                                        extent model systems used rapid <br /> lines expressing 33-nt TPK1.1 aptamer demon- <br />                                                                        screens intracellular aptamer&#226;&#8364;&#8220;protein interactions. <br /> strated intracellular inhibition RT protection                                                                           study utilizes Escherichia coli strain BK148, HIV-1 infection (Chaloin et al. 2002). <br />                                                                        carries temperature-sensitive DNA polymerase (polA12) <br />    aptamer-RT interaction represents                                                                        allows normal growth 30&#194;&#176;C restricts forma- dozen examples aptamer retains target- <br />                                                                        tion isolated colonies 37&#194;&#176;C (Witkin Roegner- <br /> binding function expressed eukaryotic cells                                                                        Maniscalco 1992). Expression exogenous DNA poly- <br />                                                                        merase HIV-1 RT (Kim Loeb 1995; Kim et al. <br />   Reprint requests  Donald H. Burke, Department Chemistry,    1996) rat DNA polymerase &#226;&#65533;&#164; (Sweasy Loeb 1992) <br /> diana University, Bloomington, 47405-7102, USA; e-mail: dhburke@    restores replication normal growth 37&#194;&#176;C. Small mol- <br /> indiana.edu; fax: (812) 855-6590. <br />   Article publication http://www.rnajournal.org/cgi/doi/    ecule <span id='am-6' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> prodrug inhibitors HIV-1 RT estab- <br /> 10.1261/rna.5550103.                                                   lished temperature sensitivity Pol HIV-1 RT strain <br />  <br /> RNA (2003), 9:1029&#226;&#8364;&#8220;1033. Published Cold Spring Harbor Laboratory Press. Copyright &#194;&#169; 2003 RNA Society.                      1029 <br />  Nickens et al. <br />  <br />  <br />  37&#194;&#176;C (Kim Loeb 1995; Kim et al. 1996), demonstrat-                dilutions freshly saturated cultures resulted resatura- <br /> ing DNA polymerase activity RT respon-                tion dilutions assayed HIV-1 RT expressed <br /> sible growth restrictive temperature. demon-               strain. contrast, orders strate RT complementation Pol defect               magnitude higher initial density BK148 required blocked inhibitory RNA aptamers, creating conditional               growth culture strain did express HIV-1 RT <br /> null phenotype suitable evaluating libraries HIV-1 RT             (Table 1). <br /> mutants aptamer-resistant variants. <br />  <br /> RESULTS                                                                  Temperature sensitivity restored RNA aptamers <br />                                                                            aptamers cloned individually expression <br /> Complementation Pol HIV-1 RT <br />                                                                            vector pERLAC (Fig. 1). Aptamers 70.28 70.24 118- <br /> Plasmid pRT5 constructed direct low-level expres-                   nt isolates selected bind HIV-1 RT, 70.28min sion HIV-1 RT (Fig. 1A). duplicate concentric                      34-nt pseudoknot core isolate 70.28 (Burke et al. <br /> spiral streak plates E. coli strain BK148 (Pol                 1996). Aptamer TPK1.1 related 33-nt pseudoknot <br /> incubated 30&#194;&#176;C 37&#194;&#176;C, cells expressing HIV-1 RT                      aptamer (Tuerk et al. 1992), reported plasmid pRT5 formed isolated colonies edge                 bind HIV-1 RT solution enhanced affinity (Kd &#226;&#8240;&#710; 25 spiral temperatures, cells                 pM; Kensch et al. 2000). Plasmid-derived transcripts trol cultures plasmid pHSG576 formed isolated colo-                   contain 28-nt 5&#226;&#172;&#732; leader 50-nt 3&#226;&#172;&#732; tail, 40 nt nies permissive temperature (Fig. 2A,B). Growth                form stable rrnB transcriptional terminator (Fig. 1). center spiral streak plates observed irrespective          design intended minimize nucleolytic degrada- <br /> temperature plasmid. results consistent                  tion maximize probability nascent tran- <br /> previously observed density-dependent growth (Kim                      scripts present aptamers intracellular  <br /> Loeb 1995; Kim et al. 1996; Kim 1997). End-point serial                    vironment native conformations. Nitrocellulose <br />                                                                                                 filter-binding assays transcripts synthe- <br />                                                                                                 sized vitro flanking se- <br />                                                                                                 quences gave Kd values (KdTPK1.1 = 1.1 <br />                                                                                                 nM; Kd70.28 = 5.3 nM; Kd70.28min = 25 <br />                                                                                                 nM; Kd70.24 = 43 nM) similar                                                                                                 originally reported core <br />                                                                                                 aptamers (Tuerk et al. 1992; Burke et al. <br />                                                                                                 1996). <br />                                                                                                     Aptamer expression plasmids                                                                                                 introduced BK148 harboring plas- <br />                                                                                                 mid pRT5. RT-expressing strains carry- <br />                                                                                                 ing aptamer cassette (pERLAC) <br />                                                                                                 grew edges spiral streak plates <br />                                                                                                 30&#194;&#176;C 37&#194;&#176;C, indicating                                                                                                 HIV-1 RT remains active pres- <br />                                                                                                 ence transcripts pERLAC. Cells <br />                                                                                                 expressing aptamer 70.28min (Fig. 2C) <br />                                                                                                 aptamers (data                                                                                                 shown) grew normally 30&#194;&#176;C,                                                                                                 restricted high-density centers                                                                                                 spiral streak 37&#194;&#176;C. Similarly, end- <br />                                                                                                 point dilution assays demonstrated                                                                                                 strains required                                                                                                 orders magnitude higher initial cell <br />                                                                                                 density growth culture did <br /> FIGURE 1.   Sequences plasmid pRT5 controlling translation HIV-1 RT expression. cells expressing HIV-1 RT (Table 1), <br /> Restriction sites used cloning indicated. Translational start sites indicated arrows; Shine-Dalgarno sequence translation HIV-1 RT double-underlined. (B) suggesting little HIV-1 RT activity <br /> Aptamer RNA expressed construct p70.28min. Nucleotides corresponding EcoRI cells expressing inhibitory RNA ap- <br /> SacI sites used cloning aptamers underlined. AptQuant oligonucleotide tamers. HIV-1 RT complementation used probe aptamer detection shown adjacent hybridization target,                                                                                                 BK148 37&#194;&#176;C blocked indi- <br /> present aptamer expression constructs. (C) Sequences aptamer inserts used study. aptamers 70.28 70.24, notations x y represent following sequences: vidual expression aptamer 70.24, 70.28, <br /> (x) 5&#226;&#172;&#732;-gggaaaagcgaaucauacacaaga-3&#226;&#172;&#732;, (y) 5&#226;&#172;&#732;-gggcauaagguauuuaauuccaua-3&#226;&#172;&#732;.                         70.28 min, TPK1.1. <br />  <br /> 1030       RNA, Vol. 9,  9 <br />                                                                                     HIV reverse transcriptase RNA aptamers bacteria <br />  <br />  <br />  <br />                                                                          Consistent interpretation, RT activity cell-free <br />                                                                          extracts reduced HIV RT&#226;&#710;&#8217; background level                                                                          cultures expressing RT aptamers (Table 3). <br />  <br />  <br />                                                                          DISCUSSION <br />                                                                          HIV-1 RT inhibited RNA aptamers inside bacte- <br />                                                                          rial cells, reported loss growth DNA polymerase <br />                                                                          complementation assay. extension, pos- <br />                                                                          sible monitor aptamer&#226;&#8364;&#8220;protein interactions es- <br />                                                                          sential proteins assessing growth conditional null <br />                                                                          phenotypes. RNA aptamers study ex- <br />                                                                          pressed components small transcripts (94&#226;&#8364;&#8220;208 nt). <br />                                                                          inhibit HIV-1 RT polymerase activity vitro,                                                                          prevent HIV-1 RT complementation BK148                                                                          nonpermissive temperatures plates serial <br />                                                                          dilution assays (Fig. 2; Table 1). Aptamer-expressing BK148 <br />                                                                          cultures grew 30&#194;&#176;C, did isogenic Pol+ strain <br />                                                                          (JS295) carrying aptamer expression constructs                                                                          30&#194;&#176;C 37&#194;&#176;C.  essential cellular function                                                                          significantly inhibited aptamers tempera- <br />                                                                          ture. <br /> FIGURE 2. Effect aptamer expression complementation Pol mutation HIV-1 RT. Photographs concentric dilution          binding affinity relative expression levels plates strain BK148 following plasmids:   pHSG576, (B)     aptamer target protein profoundly influence <br /> pRT5, (C) pRT5 p70.28.                                               biological effects aptamer expression (Burke                                                                          Nickens 2002). high-affinity interactions, aptamer RNA <br />                                                                          concentration important precise value Quantification intracellular aptamer concentrations <br />                                                                          Kd determining fraction RT bound aptamer. <br /> Quantitative measurements intracellular aptamer RNA                   Western blot analysis BK148 cells carrying plasmid <br /> levels help delineate parameters govern vivo             pHIVRT (essentially identical pRT5) gave estimates inhibition. end, total cellular RNA (RNAT)                   &#226;&#710;&#188;100 copies HIV-1 RT induced cell (Kim Loeb <br /> isolated strain, applied membrane,                    1995; Kim 1997). Assuming mean cell volume 1.5 &#194;&#181;m3/ <br /> probed radiolabeled oligonucleotides. Aptamer RNA                   cell (Kubitschek Friske 1986), corresponds consistently detected using probe complementary                 concentration 110 nM RT. Intracellular concen- 5&#226;&#172;&#732; leader aptamer expression cassette common               trations aptamer control RNA determined constructs (Fig. 1B). Comparison purified RNA                   range 76 180 nM (Table 2). Given Kd <br /> standards revealed intracellular concentrations ranging                  values reported aptamers context 76 &#194;&#177; 10 nM 180 &#194;&#177; 47 nM (70&#226;&#8364;&#8220;160 copies cell)                  5&#226;&#172;&#732; 3&#226;&#172;&#732; flanking sequences (1&#226;&#8364;&#8220;40 nM), approxi- aptamers (Table 2). values similar             mately 50%&#226;&#8364;&#8220;98% expressed HIV-1 RT expected RT concentrations measured previously (100 copies/cell;                  sequestered aptamers. known Kim Loeb 1995). Control RNA <br /> expressed aptamer-free vec- <br /> tor (pERLAC) measured        TABLE 1. Effects aptamer expression growth HIV-1 RT complemented pol mutants concentration 179 &#194;&#177; 11 nM;           liquid culture end-point dilution assays  growth cells expressing                                                               Dilution initial culturesa plasmid 37&#194;&#176;C attrib- <br />                                           Plasmids                 RT gene/aptamer 10&#226;&#710;&#8217;1 10&#226;&#710;&#8217;3 10&#226;&#710;&#8217;4 10&#226;&#710;&#8217;5 10&#226;&#710;&#8217;6 10&#226;&#710;&#8217;7 10&#226;&#710;&#8217;8 <br /> uted failure accumulate RNA, absence aptamer      pHSG576                     none           +       +       +       &#194;&#177;        &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217; expressed RNA. RNA        pRT5                        Yes            +       +       +       +        +       +      + <br /> concentrations excess      pRT5 pERLAC             Yes/vector          +       +       +       +        +       +      + <br />                                           pRT5 PERTPK1.1          Yes/TPK1.1          +       +       +       &#226;&#710;&#8217;        &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217; measured Kd values            pRT5 p70.24             Yes/70.24           +       +       +       &#226;&#710;&#8217;        &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217; <br /> aptamer-RT interactions, inhibitory       pRT5 p70.28             Yes/70.28           +       +       +       +        &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217; <br /> RNA concentrations          pRT5 p70.28min          Yes/28min           +       +       +       +        &#226;&#710;&#8217;       &#226;&#710;&#8217;      &#226;&#710;&#8217; <br /> high sequester                                                  Growth 37&#194;&#176;C 48 hr following dilution initial cultures densities 8 &#195;&#8212; 108 cfu/mL. HIV-1 RT produced cell. <br />  <br />                                                                                                             www.rnajournal.org          1031 <br />  Nickens et al. <br />  <br />  <br />  <br />                                                                         mented chloramphenicol (30 &#194;&#181;g/mL), carbenicillin (200 &#194;&#181;g/ <br />  TABLE 2. Steady-state concentration (nM) aptamer RNAa <br />                                                                         mL), tetracycline (10 &#194;&#181;g/mL), IPTG (0.1 mM) appropriate. <br />  Plasmids present            Normalized           Normalized  BK148                     cell number             rrnB signal <br />                                                                         Plasmid construction <br />  pERLAC + pRT5                  174 &#194;&#177; 14                179 &#194;&#177; 11 <br />  pERTPK1.1 + pRT5               108 &#194;&#177; 22                112 &#194;&#177; 17 <br />                                                                         plasmid constructs confirmed DNA sequencing. Vec- <br />  p70.24 + pRT5                  200 &#194;&#177; 52                180 &#194;&#177; 47        tor pERLAC derivative pUHE-21.2 (Lanzer Bujard <br />  p70.28 + pRT5                   75 &#194;&#177; 10                 76 &#194;&#177; 10        1988), transcription-translation vector itself derivative  p28min + pRT5                   91 &#194;&#177; 6                 105 &#194;&#177; 7         ColE1 replicon pBR322. Chloramphenicol resistance  <br />                                                                          moved divergent PCR using oligonucleotides pUHE-PCR1 (5&#226;&#172;&#732;- <br />   concentrations assume cell volume 1.5 &#194;&#181;m3.                   GGGGTACCCGGGATCTAAGTATCATTGTT-3&#226;&#172;&#732;) pUHE- <br />                                                                         PCR-2 (5&#226;&#172;&#732;-GGGGTACCTGCAGAGCTCATAAAACGAAAGGCT <br />                                                                         CAGTC-3&#226;&#172;&#732;). Digestion KpnI (underlined) ligation <br /> DNA polymerase activity HIV-1 RT needs provide                   yielded 2502-bp plasmid designated pER1 retains &#226;&#65533;&#164;-lac- <br />                                                                         tamase gene selection. lacO1 operator restored using complementation Pol phenotype, inversely, <br />                                                                         synthetic oligonucleotides create plasmid pERLAC. Transcrip- fraction RT needs blocked inhibit <br />                                                                         tion aptamers cloned pERLAC driven heterologous <br /> growth.  lack RT activity cell-free extracts          phage T7 promoter A1  utilizes E. coli RNA polymerase), cells expressing HIV-1 RT aptamer relative                combined lac operators (PA1-04/03) generate transcripts cells expressing HIV-1 RT (Table 3), suggests              form shown Figure 1B. RT bound aptamer.                                        gene HIV-1 RT amplified PCR plasmid <br />    Bioassays screening new drugs long history,               p83-2 (AIDS Research Reference Reagent Program), use testing aptamer efficacy logical devel-          carries pol gene HIV-1 clinical isolate NY5 (Adachi et al. <br /> opment. aptamers HIV-1 RT                     1986; Gibbs et al. 1994). fragment cloned available 10 yr, bioactivity recently           appropriate oligonucleotides HindIII&#226;&#8364;&#8220;BamHI-digested <br /> reported. noted  RNA aptamers potent anti-                pHSG576, low-copy number plasmid utilizes Pol inde- <br />                                                                         pendent pSC101 origin replication (Takeshita et al. 1987), viral activity human cell culture. speculated                                                                         create HIV-1 RT expression plasmid pRT5 (Fig. 1A). HIV-1 RT able mutate aptamer resistant <br />                                                                         construct nearly identical low-level RT expression <br /> forms adversely affecting viral replication activity            vector, pHIVRT (Kim Loeb 1995). Transcription HIV-1 <br /> (Burke et al. 1996). hypothesis appears               RT gene control lacP/O pHSG576 <br /> born instance (Fisher et al. 2002).             vector. Plasmid copy number pRT5 steady  described offers time methodol-             ditions used  copy number aptamer-encoding <br /> ogy test hypothesis screening libraries RT mu-            vectors diminished 80% BK148 compared aptamer <br /> tants resistance aptamer inhibitors.                             plasmid levels cloning strain NM522. <br />  <br />                                                                         RNA detection quantification <br /> MATERIALS METHODS <br />                                                                         Total RNA (RNAT) prepared overnight cultures using                                                                         MasterPure RNA purification kit (Epicentre), resuspended Strains medium <br />                                                                         ddH2O RNasin (Promega) 1 U/&#194;&#181;L stored &#226;&#710;&#8217;80&#194;&#176;C. <br /> Genetic complementation experiments used Pol E. coli B/r        RNA standards transcribed vitro PCR-amplified <br /> strain BK148 (recA718 polA12 fad751&#226;&#172;&#8249;Tn10 trpE65 uvrA155 mal&#226;&#710;&#8217;            templates derived plasmids pERLAC  RNA T7-Lac) lon&#226;&#710;&#8217; sulA11; Witkin Roegner-Maniscalco 1992). E. coli strain        pERTPK1.1  RNA T7-TPK1.1). Slot blots prepared using <br /> NM522 (Stratagene) used cloning plasmid             Bio-Dot SF microfiltration apparatus Zeta-Probe blotting <br /> struction. Luria broth Difco nutrient broth supple-            membranes using slight modification Epicentre protocol. <br />                                                                         RNAT, standard RNA mixed yeast tRNA,                                                                         slot grid received 6 &#194;&#181;g RNA. Cell number microgram <br />                                                                         RNAT determined optical density culture assuming <br />  TABLE 3. RT activity cell-free extracts                             5 &#195;&#8212; 108 cells&#194;&#183;mL&#226;&#710;&#8217;1 A600nm = 1.0. Blotted membranes  Cell-free extracts            Total RT activitya   Relative activity   probed radiolabeled AptQuant oligonucleotide (Fig. 1A). Af- <br />                                                                         ter washing nonspecifically bound probe, blot dried <br />  BK148  plasmids)          1.9 &#194;&#177; 1.2              1.0          analyzed using Molecular Dynamics PhosphorImager  BK148 (pRT5)                      12.1 &#194;&#177; 2.9              6.4          ImageQuant software. Calibration curves converting Phosphor- <br />  BK148 (pRT5, p70.28)               1.9 &#194;&#177; 0.1              1.0 <br />                                                                         Imager pixels picomoles target RNA plotted using  BL21DE3 (pRT(p))                 253.0 &#194;&#177; 46             133.0 <br />                                                                         signal standard RNA. faint signal RNAT prepared <br />    Unit activity defined fmole <span id='am-1' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dTTP</span> incorporated TCA          BK148 carrying plasmid pRT5 defined nonspecific <br />  precipitable material &#226;&#173;&#710; min&#226;&#710;&#8217;1 &#226;&#173;&#710; &#194;&#181;g protein&#226;&#710;&#8217;1.                          background associated AptQuant probe, sub- <br />  Values reported mean &#194;&#177; standard deviation reported          tracted sample calculating aptamer concentra- <br />  based assays. <br />                                                                         tions strain. <br />  <br />    calculating picomoles aptamer RNA present          <br /> applied sample, blots stripped reprobed rrnB1               <br /> probe specific ribosomal RNA, allowing rRNA serve          internal loading standard. Molar concentrations aptamers <br />                                                                              <br /> inside cells calculated (1) number cells             <br /> contributed RNAT given blot, (2) calculated moles          <br /> aptamer RNA detected, (3) assumed average cell volume             <br /> 1.5 &#194;&#181;m3 exponentially growing cultures nutrient broth (Ku-         <br /> bitschek Friske 1986). Excluded volume intracellular        <br /> molecular crowding omitted calculation.                <br /> Reverse transcriptase assays Kd determination                         <br /> Duplicate reverse transcriptase assays performed using cell-           <br /> free extracts prepared duplicate cultures described previ-         <br /> ously (Hizi et al. 1988) using 20 &#194;&#181;g total protein assay. Cells     <br /> carrying plasmid pRT(p), RT-overexpressing plasmid (gift S.           <br /> Hughes, National Cancer Institute), served positive control         <br /> RT activity. Dissociation constants (Kd) determined                                                                      <br /> nitrocellulose filter-binding assays described (Burke et al. 1996)     <br /> using gel-purified, radiolabeled aptamer RNA purified HIV-1               <br /> RT carrying His6 tag carboxyl terminus.                         <br /> </body></html>