<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biophysical Journal: Biophysical Letters <br />  <br />  <br /> Properties Metabolic Networks: Structure versus Function <br />  <br /> R. Mahadevan*y B. O. Palssony <br /> *Genomatica , San Diego, California 92121; yDepartment Bioengineering, University California, La Jolla, California 92093 <br />  <br />  <br /> ABSTRACT Biological data high-throughput technologies describing network components (genes, proteins, <br /> metabolites) associated interactions driven reconstruction study structural (topological) properties large-scale biological networks. article, address relation functional structural properties using <br /> extensively experimentally validated genome-scale metabolic network models compute observable functional states microorganism compare &#226;&#8364;&#732;&#226;&#8364;&#732;structure versus function&#226;&#8364;&#8482;&#226;&#8364;&#8482; attributes metabolic networks. observed  functionally <br /> speaking, essentiality reactions node correlated node connectivity structural analyses biological <br /> networks suggested. &#239;&#172;?ndings illustrated analysis genome-scale biochemical networks species distinct modes metabolism. results suggest fundamental differences different biological <br /> networks arising representation functional constraints. <br />  <br />                     Received publication 4 November 2004 &#239;&#172;?nal form 18 November 2004. <br />                Address reprint requests inquiries R. Mahadevan, E-mail: rmahadevan@genomatica.com. <br />  <br />  <br />    combination various high-throughput technologies            2. Flow networks, specific variable mass enabled coordinated study cellular components              energy flow conserved node, meta- level genes, proteins, metabolites, associated            bolic networks.  fundamental properties bio- <br /> interactions  studies resulted              logical networks classes significantly <br /> generation large-scale data sets served              different. <br /> foundation reconstruction metabolic, regulatory, <br />                                                                          report, argue metabolic networks signaling, protein-protein interaction networks. avail- <br />                                                                       unique properties resulting , conservation  <br /> ability interaction networks spurred analysis <br />                                                                       straints satisfied node, b), way structural, e., topological, properties networks <br />                                                                       metabolic networks represented, nodes using quantitative approaches (1). functional states                                                                       metabolites links reactions catalyzed metabolic networks  correspond phenotypic functions) <br />                                                                       specific gene products. representation different assessed, compare important <br />                                                                       protein-protein interaction networks, nodes structural properties networks comes                                                                       gene products links correspond interactions. interpreting functional states. <br />                                                                       analysis protein-protein interaction networks suggested <br />    Network analysis suggested biological networks <br />                                                                       deletion highly connected proteins important structural properties.                                                                        correlates lethal phenotype (2). contrast, node <br /> shown networks, including metabolic net- <br />                                                                        e., metabolite) metabolic networks deleted works, scale-free possess &#226;&#8364;&#732;&#226;&#8364;&#732;small world&#226;&#8364;&#8482;&#226;&#8364;&#8482; property <br />                                                                       genetic techniques, links  <br /> (1). Second, scale-free networks suggested high <br />                                                                          topological properties derived network <br /> error tolerance (tolerance random failure) low <br />                                                                       structure, functional, phenotypic, states. <br /> attack tolerance (vulnerability failure highly <br />                                                                       Recent studies indicated metabolic networks connected nodes).  biological networks                                                                       flux distributions average path length longer <br /> differences functional states. general, network <br />                                                                       length obtained consideration network interactions biological entities (genes, proteins, <br />                                                                       structure (3) functional states metabolites, ) classified basis nature                                                                       scale-free characteristics. proposed error attack interaction broad categories: <br />                                                                       tolerance properties metabolic networks assessed 1. Influence networks, nature interactions           context functional states using flux balance analysis    &#226;&#8364;&#732;&#226;&#8364;&#732;influence-based&#226;&#8364;&#8482;&#226;&#8364;&#8482; protein-protein interaction        genome-scale metabolic networks (4). silico models <br />    signaling networks, links represent          currently available organisms (4). silico <br />    interaction present  class extended        models Escherichia coli Saccharomyces cerevisiae    include cases type interaction important        particular, extensively validated physiological <br />    addition presence/absence interaction  ex- <br />    ample, gene regulatory networks transcription fac-                                                2005 Biophysical Society <br />    tors activate repress genes expression).                                                 doi: 10.1529/biophysj.104.055723 <br />  <br /> Biophysical Journal: Biophysical Letters                                                                                              L07 <br />  Biophysical Journal: Biophysical Letters <br />  <br />  <br /> data including data knockout phenotypes (5, 6).                  results, observed lethality fraction <br /> genome-scale models metabolic networks                    (fL) connected metabolites higher compute observable functional network states (7,              highly connected metabolites irrespective 8) used assess systematically attack            size complexity metabolic network. fact, error tolerance nodes high number            surprisingly, networks studied, nections relative low number               points fall narrow range 0.2&#226;&#8364;&#8220;0.5. nections.                                                              metabolites connectivity    biological networks, metabolic networks            lethality fraction 1. metabolites occur satisfy functions simultaneously. support growth,              linear pathway, end metabolite essential genome-scale metabolic network produce              biomass component (e.g., <span id='am-1' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> histidine). 35 different compounds (9) (Fig. 1, right panel).                  relation lethality fraction connectivity genome-scale silico models E. coli S. cerevisiae             illustrated Fig. 1, right panel, representative used analyze lethality gene deletion           metabolites. Additionally, investigate shown correctly predict experimental                  behavior metabolic network specific metab- <br /> phenotype 78.7% (6) 82.6% (5) 13,750                 olite eliminated network removing 4,154 cases, respectively. computed lethal gene knock-             reactions consume/produce metabolite. realize <br /> outs used compare connectivity, Ci, node          experimentally, genes corresponding  <br /> metabolite) lethality links (reactions)       actions simultaneously deleted. number lethal reactions  connections, CL             results analysis depicted  <br /> </body></html>