<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Luminescence Resonance Energy Transfer-Based High-Throughput <br />       Screening Assay Inhibitors Essential Protein-Protein <br />               Interactions Bacterial RNA Polymerase <br />  <br />              binding sigma factors core RNA polymerase essential specific initiation transcription <br />           eubacteria critical cell growth. responsible protein-binding regions highly <br />           conserved eubacteria differ significantly eukaryotic RNA polymerases, sigma factor binding <br />           promising target drug discovery. homogeneous assay sigma binding RNA polymerase <br />           (Escherichia coli) based luminescence resonance energy transfer (LRET) developed using eu- <br />           ropium-labeled &#226;&#65533;&#180;70 IC5-labeled fragment &#226;&#65533;&#164;&#226;&#180;&#8226; subunit RNA polymerase (<span id='am-24' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-25' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-26' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> residues <br />           100 309). Inhibition sigma binding measured loss LRET decrease IC5 <br />           emission. technical advances offered LRET resulted robust assay suitable high-throughput <br />           screening, LRET successfully used screen crude natural-product library. illustrate           method powerful tool investigate essential protein-protein interaction basic research drug <br />           discovery. <br />  <br />  <br />    need new antimicrobial drugs obvi-                   tures Thermus aquaticus holoenzyme (14, 22). protein <br /> ous scientific challenge inappropriate use clinical                 remarkably conserved (&#226;&#172;&#381;80%) amino acid sequence (11, <br /> conditions favored selection strains resistant                 13) known eubacteria regions. increasing number antibiotics. order accelerate                  homologies suggest highly conserved structure function <br /> pace compete nature, high hopes lie                   holoenzyme form RNAP, crucial exploitation recent advances genomic proteomic                   correct initiation transcription. inhibitor inter- <br /> search. increasingly detailed structural mech-                 action expected broad-spectrum antibiotic. <br /> anistic information proteins involved central pro-               supported fact induction expression cesses life cycle cell, replication,                  &#226;&#65533;&#164;&#226;&#172;&#732; fragment (residues 260 309) stops cell growth transcription, translation, rationally designed assays             E. coli. (2). &#226;&#65533;&#180;70 homologue archaea developed inhibitors specific vulnerable                  mammalian cells sigma factors mitochondria (21) <br /> targets biochemical machines. paper,                chloroplasts (1).  significant <br /> illustrate development luminescence resonance energy                 homology bacterial counterparts. fact implies transfer (LRET)-based high-throughput screen antimicro-                   little chance potential new antibiotic inter- <br /> bial drugs based identification crucial protein-                  fering eukaryotic RNAP assembly, protein interaction RNA polymerase (RNAP).                                 lead effects used drug. <br />    Like ribosome DNA polymerase, bacterial tran-                     order screen inhibitors RNAP assembly scription machinery appears offer attractive target                 sigma, assay simpler, faster, reliable ones <br /> drug discovery rational drug design. inhibitor                 currently available (6) developed formation core RNAP assembly major sigma factor form                      &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; complex E. coli. decided use LRET, recent <br /> holoenzyme inhibit initiation transcription                  modification fluorescence resonance energy transfer prevent growth eventually survival                 (FRET) create desired signal protein bind- <br /> cell. Researchers previously identified biochemical                   ing (15, 17, 18). general term &#226;&#8364;&#339;luminescence&#226;&#8364;&#65533;  <br /> methods regions Escherichia coli RNAP  coiled-coil                   stead &#226;&#8364;&#339;fluorescence&#226;&#8364;&#65533;  FRET) indicates lanthanide <br /> region amino acid residues 260 309 &#226;&#65533;&#164;&#226;&#172;&#732;                      emission technically considered fluorescence  e., arising <br /> subunit E. coli) (3, 4) major sigma factor &#226;&#65533;&#180;70 (region             singlet singlet transition). details LRET 2.2) largely responsible sigma-core interactions                 elegantly described recent reviews Selvin (8) (10, 12). results confirmed struc-                 Heyduk (16) covered briefly  quanti- <br />                                                                               tative description effect based Fo &#194;&#168;rster theory <br />                                                                               describes decrease energy transfer inversely pro- <br />   * Corresponding author. Mailing address: McArdle Laboratory Cancer Research, University Wisconsin&#226;&#8364;&#8221;Madison, 1400 University <br />                                                                               portional sixth power distance Ave., Madison, WI 53706. Phone: (608) 263-2635. Fax: (608) 262-2824.          dyes applicable FRET LRET. advantages E-mail: burgess@oncology.wisc.edu.                                            LRET result prolonged fluorescent lifetimes  <br />                                                                        1492 <br />  VOL. 69, 2003                                                                                       LRET ASSAY DRUG DISCOVERY                               1493 <br />  <br />  <br /> lanthanide-based donor fluorophores like Eu Tb                               LRET assay test inhibition protein-protein interaction labeled microseconds milliseconds) compared                      &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#180;&#8226;. LRET assay performed NTG buffer (200-&#226;&#65533;&#174;l total <br />                                                                                    volume) plus 2.5% DMSO  library samples used) 40 nM &#226;&#65533;&#180;70* <br /> short lifetime organic-based fluorophores used                          (labeled protein) 30 nM &#226;&#65533;&#164;&#226;&#172;&#732;* fragment (labeled protein). assays FRET, like Cy5 IC5 (picoseconds microsecond),                        performed refolded &#226;&#65533;&#180;70*. standard assay, &#226;&#65533;&#164;&#226;&#172;&#732;* fragment LRET offers mainly technical advantages FRET.                            added denatured form directly reaction mixture,  10-fold <br />    Heyduk coworkers used LRET measure DNA bind-                             dilution GuHCl 0.15 M final assay, allowed instant refolding.                                                                                    prevent precipitation refolded labeled protein storage ing &#226;&#65533;&#180;70 holoenzyme using Eu chelate donor <br />                                                                                    assay (5). order assure refolding occur confirm results Cy5 <span id='am-37' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span>acceptor</span> (7). able appli-                      experiments denatured protein added, assays cability  adapted principle ex-                       carried adding &#226;&#65533;&#164;&#226;&#172;&#732;* fragment refolded state. <br /> changing Cy5-labeled polynucleotide IC5-labeled                           stock solution (200 nM) &#226;&#65533;&#180;70* prepared prior assay 1:200 <br /> &#226;&#65533;&#164;&#226;&#172;&#732; fragment (residues 100 309 N-terminally fused                      dilution labeled protein (40 &#226;&#65533;&#174;M) NTG stored &#226;&#171;&#186;20&#194;&#176;C storage buffer. <br />                                                                                    stock solution (1.25 &#226;&#65533;&#174;M) &#226;&#65533;&#164;&#226;&#172;&#732;* fragment prepared dilution heart muscle kinase [HMK] recognition site His6 tag).                        labeled denatured protein (75 &#226;&#65533;&#174;M TNTw buffer) 1 M GuHCl NTG resulting homogenous assay, labeled &#226;&#65533;&#180;70                          NTG plus 6 M GuHCl.  10 &#226;&#65533;&#174;l &#226;&#65533;&#180;70* stock solution mixed europium&#226;&#8364;&#8220;diethylenetriaminepentateic acid (DTPA)&#226;&#8364;&#8220;7-ami-                            NTG buffer  adjusted final volume 200 &#226;&#65533;&#174;l), potential 4-methylcoumarin-3-<span id='am-42' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acetic acid</span> (AMCA) complex                           inhibitor (dissolved 1 ml methanol, diluted 1:50 DMSO, 5 &#226;&#65533;&#174;l HMK-His6-&#226;&#65533;&#164;&#226;&#172;&#732;(100-309) fragment Cy5                             applied reaction mixture), salt, solvent added, finally, <br />                                                                                    5 &#226;&#65533;&#174;l denatured, labeled &#226;&#65533;&#164;&#226;&#172;&#732;* stock solution added. Salts denaturants <br /> analogue IC5-maleimide (Dojindo) (purification labeling                        dissolved NTG buffer according desired final concentration procedures described Bergendahl et al. [5]). possible                  maintain standard buffer conditions. Mixing (pipetting times) <br /> monitor complex formation &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#172;&#732; simply                             addition component important reproducible  <br /> looking long-lived emission acceptor (IC5)                    sults. mixture incubated 30 min room temperature measured <br />                                                                                    96 plate (Costar 3650) multiplate reader (VictorV2 1420; Wal- <br /> LRET optically measurable signal complex formation. <br />                                                                                    lac). time-resolved fluorescence measurement, manufacturer&#226;&#8364;&#8482;s pro- <br /> Measuring time-resolved fluorescence allows start sig-                      tocol (LANCE high count 615/665) used (excitation occurred 1,000 <br /> nal acquisition background fluorescence (potentially                     flashes 325 nm, measurement delayed 100 &#226;&#65533;&#174;s, data acquired natural product) intrinsic acceptor fluorescence                        50 &#226;&#65533;&#174;s 615 665 nm). decayed short-lived background fluorescence                          fluorimetric measurements, common use second emission wave- <br />                                                                                    length internal standard. allows correction instrument noise excluded measurement. leads highly                       normalizes signal actual donor, particular <br /> favorable signal noise ratio higher sensitivity                   case. possible, donor acceptor emission wavelengths fidence. especially important working natu-                      separated acquired separately multiplate reader. IC5 <br /> ral-product samples, produce high background                         emission corrected small signal Eu emission <br />                                                                                    band  cross-talk measurement standard) divided intensity <br /> fluorescence. principle assay structures                                                                                    Eu signal. normalization included overall measure- dyes described Fig. 1.                                                  ment protocol described manufacturer multiplate reader <br />    assay performed multiwell plate mea-                        (Wallac). Fortunately, nature method allowed differentiate sured multiplate reader accomplish high-throughput                       loss signal inhibition protein-protein binding simple ab- <br /> screening large number samples <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span>                           sorption caused inner-filter effect substance. helped                                                                                    identify false positives actual high-throughput screen. <br /> library automated way. Typical reaction volumes 10                         vitro transcription. conditions procedure vitro transcrip- 200 &#226;&#65533;&#174;l components, including test substances,                     tion assay according published Landick et al. (9) mixed directly multiwell plate plate                    omitting bovine serum albumin, bind potential hydrophobic inhib- <br /> measured reading device. sensitive nature                       itors. assay conditions follows: 100 mM NaCl, 25 mM Tris HCl (pH fluorescence-based assay (typically low nanomo-                      7.9), 10 mM MgCl2, 1 mM dithiothreitol, 0.1 mM EDTA, 5% glycerol, 0.15 mM <br />                                                                                    ApU (dinucleotide), 20 nM template DNA, 20 nM core RNAP, inhibitor lar range) provides good accuracy signal noise ratio,                       final concentration 2% DMSO, 10 nM &#226;&#65533;&#180;70, 25 &#226;&#65533;&#174;M concentrations avoiding false-positive hits measurement. cho-                      <span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span> <span id='am-47' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span>CTP</span>, 10 &#226;&#65533;&#174;M <span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span>, 20 &#226;&#65533;&#174;Ci [&#226;&#65533;&#163;-32P]GTP/100 &#226;&#65533;&#174;l, added sen screen marine sponge library (19)                       order listed. template, amplified 694-bp fragment plasmid <br /> successfully screened cytotoxic compounds. com-                     pCL185 PCR primers 5&#226;&#172;&#732;-GTT TTC CCA GTC ACG AC-3&#226;&#172;&#732;                                                                                    5&#226;&#172;&#732;-CAG TTC CCT ACT CTC TCG CAT G-3&#226;&#172;&#732;. Omitting <span id='am-32' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span> reaction <br /> plexity, demanded high performance assay                                                                                    mixture initiating ApU dinucleotide resulted halted complex suited evaluate applicability.                   position &#226;&#171;&#185;16 transcribed RNA oligonucleotide AUG GAG AGG GAC <br /> robust nature assay suitability use high-                  ACG G. assay tested ability RNAP holoenzyme (core plus <br /> throughput screen.                                                                 sigma factor) recognize &#226;&#65533;&#180;70-dependent promoter (T7A1; located bp <br />                                                                                    280 694-bp PCR fragment) initiate elongate yield                                                                                    16-mer oligonucleotide. Failure sigma binding core resulted lack                        MATERIALS METHODS <br />                                                                                    product. <br />   Materials chemicals. Overproduction, labeling, purification HMK- <br /> His6-&#226;&#65533;&#164;&#226;&#172;&#732;(100-309) &#226;&#65533;&#180;70(442C) described previously (5). origin natural-product library comprises privately collected samples marine sponges <br />                                                                                                                      RESULTS <br /> (19) extracted acetone ethylacetate. samples (2 5 mg [dry <br /> weight]) dissolved 1 ml methanol diluted 1:50 dimethyl sulfox- <br /> ide (DMSO) assay. chemicals used purchased Sigma unless        Labeled unlabeled proteins shown able indicated text. multiplate reader (VictorV2 1420; Wallac)   form complex electrophoretic mobility shift (EMS) <br /> used perform LRET assay. following buffers used: NTG buffer        assays using native polyacrylamide electrophoresis gels (5). <br /> (50 mM NaCl, 50 mM Tris, 5% glycerol [pH 7.9]), storage buffer (50 mM              different scanning techniques confirmed identities <br /> Tris-HCl, 10 &#226;&#65533;&#174;M EDTA, 0.5 M NaCl, 10% glycerol [pH 7.5]), TNTw buffer <br /> (6 M GuHCl, 50 mM Tris-HCl [pH 7.9], 500 mM NaCl, 0.1% [vol/vol] Tween 20, <br />                                                                                    bands EMS assay. Furthermore, EMS assay <br /> 400 mM imidazole). Core RNAP prepared according method               showed unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment compete bind- <br /> scribed Thompson et al. (20).                                                   ing labeled &#226;&#65533;&#180;70.  unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment itself <br />  1494      BERGENDAHL ET AL.                                                                                            APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Principle LRET assay.   scheme shows principle LRET signal assay created binding labeled proteins &#226;&#65533;&#164;&#226;&#172;&#732; (residues 100 309) &#226;&#65533;&#180;70. fluorescence IC5-labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment decayed delay data acquisition 50 &#226;&#65533;&#174;s excitation 320 nm. Eu emission labeled &#226;&#65533;&#180;70 sensitized IC5 emission complex observed delay characteristic long Eu luminescence lifetime 1 ms. (B) diagram, luminescence plotted versus time. graphs shows intensity luminescence emitted 615 665 nm time. intrinsic IC5 emission decayed <br /> long data acquisitions starts, emission IC5 sensitized LRET goes 1 ms. IC5 slightly faster decay Eu emission. way, IC5 emission LRET acquired data acquisition window, background emission intrinsic IC5 fluorescence excluded. (C) structures fluorophores used derivatize proteins shown. Eu <br /> chelate DTPA-AMCA-maleimide used <span id='am-31' about='rdfs:label' typeof='owl:Thing'>label</span> &#226;&#65533;&#180;70. IC5-PE-maleimide served label &#226;&#65533;&#164;&#226;&#172;&#732; fragment. <br />  <br />  <br />  <br /> represents positive control agent able interfere            beled proteins. LRET assay provides fast reproduc- binding labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment &#226;&#65533;&#180;70 assay.                 ible alternative EMS assay (5) monitor formation <br />   Validation LRET assay test inhibition                 protein-protein interaction labeled &#226;&#65533;&#180;70 protein-protein interaction labeled &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#180;&#8226; unla-                 &#226;&#65533;&#164;&#226;&#172;&#732; fragment inhibition. results EMS <br />  VOL. 69, 2003                                                                       LRET ASSAY DRUG DISCOVERY                     1495 <br />  <br />  <br />  <br />  <br />   FIG. 2. Competition unlabeled &#226;&#65533;&#180;70, nonbinding &#226;&#65533;&#180;70 mutant <br /> (D403R) (3), bovine serum albumin (BSA) labeled &#226;&#65533;&#180;70 binding &#226;&#65533;&#164;&#226;&#172;&#732; fragment. &#226;&#65533;&#180;70 (wild type) competes binding &#226;&#65533;&#164;&#226;&#172;&#732; fragment, proteins non- <br /> specific binding resulting slightly lower LRET count. <br />  <br />  <br />  <br /> assay reproducible LRET assay, exam- <br /> ple, able competition labeled &#226;&#65533;&#180;70 binding &#226;&#65533;&#164;&#226;&#172;&#732; fragment increasing amounts unlabeled &#226;&#65533;&#180;70 <br /> (Fig. 2). important feature assay, signal- noise ratio 10 11. limit detection assay, using described instrument, 1 nM <br /> labeled &#226;&#65533;&#180;70. maximum 250 nM labeled &#226;&#65533;&#180;70 applied assay diffusion-controlled LRET oc- <br /> curred. <br />    Influence salt organic solvents LRET assay. experiments, dependence salt (NaCl GuHCl) solvents (methanol, ethanol, DMSO) characterized (Fig. 3). seen, salt concentration major effect assay, signal drops 50% NaCl concentration increased 100 400 <br /> mM. known &#226;&#65533;&#164;&#226;&#172;&#732; fragment interaction &#226;&#65533;&#180;70           FIG. 3. Dependence salt concentration effect solvent  <br /> weakened increased NaCl concentration.                centration LRET assay.   increasing amounts salt <br /> hand, DMSO common solvent natural products             (NaCl, GuHCl), LRET signal significantly decreases, tested significant effect assay. signal      decreased &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; complex formed. (B)  <br /> LRET assay critically affected               creasing amounts DMSO, methanol, ethanol (0 5%) mixed <br />                                                                       assay buffer prior addition proteins shown. <br /> DMSO present, 2.5%. experiment, ethanol methanol showed significant effect range 1 5%. <br />    Performance LRET assay preliminary screen              Confirmation hits LRET assay vitro <br /> 100 sponge extracts. concentration inhibitor            transcription. concentration inhibitor extracts extracts marine sponges estimated 1 &#226;&#65533;&#174;M         marine sponges examined range 30 nM 2 <br /> (assuming 2-mg sample 100% single component                &#226;&#65533;&#174;M LRET assay. Samples A5, B12, E1, E12, G1, H4, <br /> having approximate molecular mass 1,000 Da).             H6 resulted minor loss signal LRET 100 samples tested, sample D7 turned           assay. Sample D7 turned extract extract reduce signal roughly 90% assay. Sam-        clear inhibition curve examined range (Fig. 5A). ples E1, E12, G1, H4, H6 showed loss signal          confirmed vitro transcription assays. Samples A5, 50%, served arbitrary threshold  55          B12, E1, E12, G1, H4, H6 resulted loss activity 75%). samples (like A7 B12) significantly           vitro transcription assay, judged densitometry active inhibition binding, judged loss   electrophoretograms gels (Fig. 5B). assay, <br /> LRET signal (Fig. 4). variance obtained values         D7 showed dose-dependent inhibition transcriptional majority samples spread wide range 40%         activity, 50% inhibitory concentration (IC50) loss signal.                                                       1 &#226;&#65533;&#174;M, assuming mass content sample <br />  1496      BERGENDAHL ET AL.                                                                                        APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Screen 100 extracts marine sponges LRET <br /> assay. Inhibition &#226;&#65533;&#180;70 binding &#226;&#65533;&#164;&#226;&#172;&#732; fragment tested 100 <br /> extracts marine sponges (G. Marriott J. Tanaka). Binding &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#172;&#732; fragment significantly inhibited fraction D7, fractions lower inhibition conditions. <br /> Assuming molecular mass 1,000 Da 100% active content 2-mg sample, concentrations used low micromolar range. <br />  <br />  <br /> mentioned  significant difference IC50s determined LRET derived vitro transcription assays. <br />  <br />                            DISCUSSION <br />    use LRET investigate protein-protein interactions homogenous assay described  advantages challenges. spite  publications development assay                  FIG. 5. Validation hits LRET   vitro transcrip- <br /> based technique actually document final appli-            tion (B) assays.   candidates (D7, H4, E1, G1) resulting <br /> cation. work, straightforward approach           screen sample effect (A5) serving control <br /> designing robust high-throughput assay exploited            tested LRET assay range 33 nM 2 &#226;&#65533;&#174;M <br />                                                                           (estimated concentration). y axis represents fluorescence sig- pair proteins appears valid target          nal resulting LRET binding labeled &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#172;&#732; drug discovery.                                                       fragment. Data samples E1 G1 represented H4    reasons believe primary pro-             behaved similar LRET assay. (B) determine tein-protein interaction bacterial core RNAP                  functional effect inhibitor &#226;&#65533;&#180;70-&#226;&#65533;&#164;&#226;&#172;&#732; binding, tested  <br />                                                                           didates vitro transcription assay. assay requires  <br /> sigma factors represents prime target drug discovery.           corporation &#226;&#65533;&#180;70 holoenzyme RNAP tests ability key potential target absolute requirement           initiate transcription elongation. autoradiogram se- sigma binding core RNAP initiation transcrip-            quencing gel used quantify transcription product tion; bacterial cell grow uptake inhibitor         halted complex using T cassette template omitting UTP <br /> effectively blocks interaction. addition high           reaction mixture  Materials Methods). resulting <br />                                                                           16-mer oligonucleotide separated gel quantified bioactivity, good specificity expected            using Molecular Dynamics software. estimated concentrations binding region proteins highly conserved                 assay 0.1 50 &#226;&#65533;&#174;M (data samples A5, E1, bacteria significantly different known eukary-            G1 represented H4 behaved similar assay; <br /> otic analogue. implies low probability               data D7 indicated triangles). <br /> effects occur interference human RNAP. site itself offers advantage potential specific targets. binding site &#226;&#65533;&#164;&#226;&#172;&#732; subunit             need new antibiotics, recently major issue <br /> RNAP suspected interact sigma           drug discovery. <br /> factors bacterium, development resistance point              Using LRET measure sigma binding core RNAP mutations inhibitor binds &#226;&#65533;&#164;&#226;&#172;&#732; subunit            shown Heyduk coworkers effective RNAP binding site unlikely, impair                 sensitive method. used characterized <br /> binding essential sigma factors.              &#226;&#65533;&#180;70(442C) mutant (7) natural cysteine residues <br /> creasing incidence antibiotic resistance growing               mutated serine residues shown fully active  VOL. 69, 2003                                                                         LRET ASSAY DRUG DISCOVERY                             1497 <br />  <br />  vitro transcription assays (7). mutant derivatized       substance. contributed enormously avoiding false-pos- DTPA-AMCA-maleimide Eu complex serves               itive readings.  good compatibility assay <br /> donor LRET assay. fragment (residues 100            use DMSO solvent add test substances <br /> 309) &#226;&#65533;&#164;&#226;&#172;&#732; subunit RNAP N-terminal HMK                augments applicability high-throughput screen. recognition site His6 tag fusion derivatized IC5-       natural products peptides small molecules <br /> maleimide LRET acceptor (5). able             combinatorial libraries low solubility water, EMS assays (5) spectrometric measurements           necessary use organic solvents. respect, DMSO rep- <br /> using time-resolved fluorescence labeled proteins       resents versatile potent solvent, bind combinations           used libraries, compatibility crucial assay. <br /> label. controls, unlabeled proteins tested         sample (D7) identified LRET assay termine compete labeled counterparts.       inhibitor containing extract, confirmed assays, EMS LRET, unlabeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment          vitro functional assay. assumptions estimate able compete labeled &#226;&#65533;&#164;&#226;&#172;&#732; fragment binding          concentration inhibitors clearly overestimate labeled &#226;&#65533;&#180;70. data, concluded assay      active compound present fair expect <br /> used monitor &#226;&#65533;&#180;70 &#226;&#65533;&#164;&#226;&#172;&#732; binding used screen          IC 10 100 nM  Initial studies ability inhibitors protein-protein interaction. assay        D7 extract inhibit cell growth E. coli cells represents fast sensitive probe particular complex    culture plates inconclusive limited amounts formation. Substrates material readily available      available material (data shown). Considering complex- prepared simple efficient procedures.       ity samples, plan carry automated screen labeled protein components excellent stability           1,500 sponge extracts deciding storage, great advantage screening large libraries       samples characterize  <br /> 10,000 100,000 substances. Furthermore, LRET             case natural-product libraries, positive hit assay high sensitivity measurements        represent new class antibiotic, substance performed low protein concentrations 1 100 nM,         mode action known. hand, screen <br /> resulting low cost screen.                        help identify known antibiotics mode    suitability assay high-throughput screen-      activity identified reveal ing natural combinatorial libraries demonstrated       activity. positive hits combinatorial librar- screen 100 extracts marine sponges. high demand          ies, substances serve lead structures design performance assay screening natural-prod-          tailor new compound desirable characteristics, uct libraries  containing complex mixtures      high activity, specificity, stability, ability enter cell <br /> possibly 30 compounds, obvious looking           hand, low effects, costs, chance, variance obtained values LRET assay.       likelihood resistance development. addition, assay <br /> variance occurs mainly fluorophores mixtures      serve powerful tool investigate relative binding absorb crucial wavelengths assay     different sigma factors sigma factor mutants core. <br /> generally referred inner-filter effect. effect led response samples B12, E1, E12, G1,                                <br /> H4, H6 LRET assay (Fig. 4   5A),          vitro transcription assay (Fig. 5B) did        <br /> inhibitory effect transcriptional activity. conceiv-    <br /> able compounds binds region &#226;&#65533;&#164;&#226;&#172;&#732;          <br /> subunit inaccessible core RNAP leads                                                                                                     <br /> impaired binding distortion protein structure.                                                                      <br /> underlines technical biochemical differences            described assays compounds active         LRET assay effect vitro      <br />                                                                         <br /> transcription assay. LRET assay monitors particularly       <br /> binding sigma fragment &#226;&#65533;&#164;&#226;&#172;&#732; subunit RNAP,           <br />                                                                     vitro transcription assay tests ability      <br /> holoenzyme RNAP transcriptional initiation elonga-               <br /> tion. simplicity suitability use high-  <br />                                                                      <br /> throughput mode, LRET assay used select poten-            <br /> tial inhibitors large number candidates present            library compounds. way, number samples        <br />                                                                      investigated informative signif-           <br /> icantly time-consuming assay, like vitro transcrip-      <br />                                                                      <br /> tion assay, minimized.                                         <br />    LRET assay turned sensitive reliable.        good signal noise ratio 10 internal         <br />                                                                      <br /> standard method helped distinguish inhibition      <br /> fluorescence quenching inner-filter effect test      <br /> </body></html>