<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1992 Genetics Society America <br />  <br />  <br />  <br />  Drosophila couch potato Gene: Essential Gene Required Normal <br />                               Adult Behavior <br />  <br />           Hugo J. Bellen,**&apos;Harald Vaessin?&apos; Ethan Bier?3 Alex Kolodkin: Diana D&apos;Evelyn,* <br />                                  Sandra Kooyer Yuh Nung Jant <br />        Howard Hughes Medical Institute, *Institutefor Molecular Genetics, Baylor College Medicine, Houston, Texas 77030, <br />              tDepartment Physiology, University Calijornia Sun Francisco, Sun Francisco, Calijornia 94143,              $Department Molecular Cell Biology, University Calijornia Berkeley, Berkeley, Calijornia 94720 <br />                                              Manuscript received January 9, 1992 <br />                                          Accepted publication February 26, 1992 <br />  <br />                                                              ABSTRACT <br />                  enhancer detection screens isolated 14 insertions essential gene                expressed embryonic sensory mother cells (SMC), cells    mature embryonic peripheral <br />                nervous (PNS), glial cells PNS central nervous (CNS). Embryos <br />                homozygote amorphic alleles die, obvious defects cuticle, PNS CNS.                gene named couch potato (cpo) insertional alleles alteradult behavior. <br />                Homozygous hypomorphic cpo flies recover slowly ether anaesthesia, aberrant flight <br />                behavior, fail light exhibit normal negative geotactic behavior.  <br />                flies able groom walk, able fly prodded, indicating                processes required forbehavior severely affected. molecular analysis showsthat 14 insertions <br />                confined nucleotides probably contain key regulatory sequences                gene. orientation insertions position withinthisDNA             fragment play                important role couch potato phenotype. situ hybridization mount embryos suggest <br />                insertions affect levels transcription cpo cells whichit expressed. <br />  <br />  <br />      ELLS nervoussystem require theaction                           peripheral nervous (PNS). T h e goal C      genes acquire morphological func- <br /> tional identity number genes involved nervous <br />                                                                               studies identify genes play impor- <br />                                                                               tant role differentiation orfunction development function       identified                    PNS furtherunravel genetic cascade partially characterized Drosophila (forreviews                         leads fully mature PNS.  CAMPOS-ORTEGA       KNUST 1990; JAN JAN                           isolation allelic series newly isolated gene <br /> 1990; GHYSEN     DAMBLY-CHAUDIERE         1989; RUBIN                     expressed SMC, embryonic 1988; GANETZKY     WU 1986; HALL1985). Genetic                            adult mutant phenotypes, molecular defects screens mutants affect nervous                            underlie mutations, cloning typically focussed isolating mutants                         gene. defective development, exhibiting mor- <br /> phological aberrantstructures          (e.g., Notch, <br />                                                   scute,                                   MATERIALS METHODS <br /> achaete,slit,sim, ) mutants display quite <br />                                                                                   Fly stocks: Generation flies carry enhancer detec- <br /> specific behavioral defects adults (e.g., Shaker, ru-                      tors described BELLENet al. (1989), BIER et al. <br /> tabaga, , exhibiting subtle physiological                        (1989). insertion strains described biochemical alterations.                                                      papers renamed. cpl insertion strain corresponds <br />    T h e adventofenhancerdetection           (O&apos;KANEand                       P[IArB] B52.1M3 strain described BELLEN        et al. <br /> GEHRINC1987; BELLENet al. 1989; BIERet al. 1989)                              (1989) WILSONet al. (1989). 13 insertion corre- <br />                                                                               sponds 1(3)5D4 BIER et al. (1989). strains provides new means study genes areexpressed                           90D/E deficiencies obtained KEVIN MOSES required nervous (MLODZIK            et                  (MOSES,ELLISand RUBIN1989), Dfl3R)DG4 al. 1990; DOE et al. 1991; review BELLEN,                             obtained DONGAILEY(GAILEY            HALL 1989) <br /> WILSONand GEHRING1990). screened                                  Dfl3R)P14, obtained Bowling Green <br /> enhancer detector insertion strains express @-                       Drosophila stock center (Ohio) Umea Drosophila <br />                                                                               stock center (Sweden). <br /> galactosidase gene sensory mother cells (SMC)                             lAcpll lAcp12 alleles recovered remo- <br />                                                                               biking P[IArB], isogenization, described BELLEN <br />    &apos; Corresponding author.                                                    et al. (1989). <br />    &apos; Present address: Ohio State Biotechnology Center Department          n situ hybridization immunocytochemistry situ <br /> Molecular Genetics, Ohio State University, Columbus, Ohio 43210. <br />     Present address: Department Biology, University California San   hybridizations mount embryos performed <br /> Diego, La Jolla, California 92093-0322.                                       according modified protocol TAUTZ         PFEIFLE <br /> Genetics 131: 365-375 (June, 1922) <br />  366                                                    H. J. Bellen et al. <br /> (1989). Probes werelabeled digoxigenin-<span id='am-14' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-15' property="rdfs:label" content="dUTP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span>dUTP</span>               P-galactosidase activity (BIERet al. 1989; BELLEN <br /> hybridization detected monoclonal antibody digoxigenin coupled alkaline phosphatase (Boeh- <br />                                                                    et al. 1989; WILSONet al. 1989). Fourteen insertion <br /> ringer Mannheim). situ hybridization polytene chro-          strains stain   precursor cells mosomes performed according modification           developing embryonic PNS selected. protocol described LANGER-SAFER,LEVINEand           WARD        strains identified screens express (1982). protocol adapted digoxigenin-labeled          lacZ gene SMC, suggesting specific <br /> probes. <br />                                                                    genes active cells. prompted     @-Galactosidase activityin embryos detected  <br /> scribed BELLEN  et al. (1989). /3-Galactosidaseprotein      analyze insertion strains. Surprisingly, proteins localized immunohistochemically using           14enhancerdetectorstrainscarryaninsertionat polyclonal rabbit anti-<span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;galactosidase antibody (Cappel)       cytological subdivision 90D. observations indi- <br /> monoclonal antibodies  text) described BIER et al.       cated enhancer detectors controlled (1989). <br />    Phototaxis assay: Phototactic tests performed         regulatory elements gene, seven-trial countercurrent apparatus described BEN-          90D division possibly hottest spot forP element <br /> ZER (1967). 1-min equilibration period, groups          insertions every300 insertions maps at90D <br /> 20-25 flies given 1 min white light          (BIERet al., 1989). insertion strains homozy- <br /> source dark room. end assay,         gous viable, strainsare homozygous lethal. <br /> flies tube counted, number flies tube multiplied numberof times            separate screen homozygous viable muta- <br /> moved light. sum ofthese values divided         tions cause reduced flight abilities   KOLODKIN total number flies. theoretical range        n d COREY GOODMAN,      personal communication), P <br /> 0-7 (0 = flies light, 7 = flies 7          element-enhancerdetectorstrain          isolated times light). Wild-type flies, e.g. Canton-S, typically     maps 90D. Embryos strain express P-galac- <br /> score higher 4. assay performed times. (+) = index 2 4, (+/-) = 3 5 index &lt; 4,(-) = index          tosidase essentially cells 90D <br /> &lt; 3, nt = tested.                                              insertion strains. <br />    Flight response assay: Groups 20 flies given 30           &amp;Galactosidase expression enhancer detector <br /> sec fly away Petri dish 95 X 28 mm            strainswithinsertions    90D: T h e lac2 staining <br /> vial kept 1 hr anaes-        pattern embryos 14 90D strains  thesia. assays repeated times. (+) = <br /> 61-loo%, (+/-) = 31-60%, (-) = 0-30% fly away.                     exception number midgut cells, confined    Geotaxis assay: 20flieswere transferred             nervous  lacZ expression firstseen 95 X 28 mm vial subsequently capped           stage 8 embryos cellsat anterior end vial. Flies shaken         embryo (Figure 1A)  staging CAMPOS- vialsin dark room number flies movebeyond vialin 15 sec                ORTEGAand HARTENSTEIN        1985).  observe <br /> counted. assays repeated times. (+)        staining SMC differentiating cells = 61-loo%, (+/-) = 31-60%, (-) = 0-31%  nt =               PNS stage 9-1 1 embryos  Figure 1, B, C tested.                                                        D). pattern consists gradual increase    Ether recovery assay: 20 flies anaesthetized         cell presumptive segment set 1 min etherizer containing 2 ml ether. flies placed 100-mm Petri dish recovery           cell clusters. extended germ band (stage <br /> time (RT) recorded 50% flies walk away          11) pattern rapidly complex. start grooming (RT,,), fliesrecovered (RTloo).        subsequent stages known PNS neurons Fifty percent wild-type flies (Canton-S) typically      sibling non-neuronal supportcells express theP- <br /> cover 15 min, 100% recover 25 <br />                                                                    galactosidase (Figure 1 , E F)  Figure 6, B <br /> min. (+) = RT5o &lt;15 min, RTloo&lt; 25 min, (+/-) = 15 min <br /> &lt; RT5(,C 20 min, 25 min &lt; RTloo &lt; 30 min, (-) = 20 rnin            C, BIER et al. 1989). lacZ expression &lt; RT5(,,30 min C RTloo.                                            PNS thoracic abdominal segments resem- <br />    Recombinant DNA: recover DNA sequences flanking              bles expression pattern observed A37  enhancer detectors used plasmid rescue tech-            hancerdetectorstrain          GHYSENand O&apos;KANE <br /> niques described WILSONet al. (1989) BIER et al. <br /> (1 989). Genomic fragments recovered dif-          1989). enhancer detector maps 80A. <br /> ferent strains. fragments used            T h e cells organs thePNS head region <br /> screen lambda DASH genomic library (kindly provided           embryo describedin RON DAVIS).Maps constructed EcoR1,                   thoracic abdominal segments (CAM- <br /> Hind111 XbaI restriction enzymes.                              POS-ORTEGA     HARTENSTEIN      1985; JURGENS   et al. <br />                                                                    1986), making assignments particular clusters                                 RESULTS <br />                                                                    cells labeledin region difficult.T h e <br />   Screening enhancer detectors express 8-                 number cell clusters expressing lacZ gnathal <br /> galactosidase PNS: attempt identify                segmentsandtheprocephalongraduallyincrease <br /> new genes expressed onset nervous               fromoneduringlategastrulation           (stage 7) t o 1 3 development, embryos 4,600 strains carry-                early head involution (stage 14; Figure 2). <br /> ing single P element-enhancer detectors stained               Based o n t h e topological position clusters                                                              Genetics couch potato                                                            367 <br />  <br />  <br />  <br />  <br />          <br />  <br />  <br />  <br />         D                                            E <br />    FtGURE -fl-Galactosidase expression pattern homozygous cporP&#226;&#8364;&#8482; embryos various stages development. embryos shown thrir anterior end left. don71   Stage 8 embryo. lac2 weakly expressed anterior end rmhwo ( s e e arrow). R. Stage 10 embryo, IacZ expression confined cells segment. C . Stage 12 embryo. IacZ expression <br /> Imttern a1rr;tdy quite conlplex relatively large clusters cells segment lalx-led. D, Stage I3 embryo. note staining tlre head region wrll regular pattern PNS aMominal thoracic segments. E. Stage 16 ernhwo. cells tlw srrppnrt cells external sensory organs PNS express @-galactosidase.F, Detailed view stage 16 embryo. Focus &lt;:NS cells PNS focus. CNS hottom embryo. Note glial cells l m t e d tlle don71 CNS  rectangle). <br />  <br />  <br />  <br />  <br />     FIGUREP.-Det; fl-galactosidaseexpression pattern head region head involution (stage 14).  lnteral view ceph;dic region rpofP&#226;&#8364;&#8482; embryo stained l~munocytocllemicallv          anti-@-galactosidaseantibody j u s t head involution. R. <br /> &lt;:onpsitr schrnwtic drawing lateral view ceplralic region embryo. prior head involution. drawing based olwrvations tnatlr wve1~1            em1,ryos does correspnnd exactly picture shown left. clusters labeled                          ; m t m l y numbered theCNS background hatched. simplicity chosen nomenclature used <br /> ;~llti-fl-~:,ll;lctosid;lsc <br /> b y JURGENS rt ai. ( 986). , Central group nlaxillary sense organ. 2. Lower lip organ. 3, Lahial sense organ. 4, Ventral organ. 5. D o r s l <br /> l:ltcral papilla nlaxilliary sense organ. 6 . Hvpopharyngcal organ. 7. Antennal sense organ. 8. Inhral sense o r g n . 9. Dorsn-medial papilla nraxi1l;wy sense orpan. 1 0 - 1 3, Clusters cells probably helong PNS head Ixen described previously knowledge. <br />  <br /> stage 14 embryos, believe 13                            believe cell clustersalso belong tothe PNS. <br /> clusters described previously  compre-                           addition neurons supportcells hensive summary seeTable 7 JURGENS et al. 1986).                           PNS. l d expression observed somecells clusters described pre-                          dorsal ventralarea ventral nerve <br /> viously quite small                          cord CNS, cells alongthe base ofthe bilateral <br /> sist 2-5 cells compared 10-40 cells                          peripheral nerve roots cells laterally clusters. Becausethe cells clusters                     anterior fascicle peripheral nerves. 8-Galacto- epidermalandsubepidermal, mostof                                  sidaseexpressioncanclearlybeseen          thesecells <br /> staining cells confined cellsof PNS,                            completion germ band retraction (stage <br />  Jfi8                                                                                          1-1. J . t4lrnr l      01. <br />  <br />  <br />  <br />  <br />        FKX&#226;&#8364;&#8482;RK :~. (;;ll;lc-tositf;lsc csprcssion ~ ; I ~ ~inY l;~rv;~l          h l l . ~ Y Tinwgin;ll disc- ; d ; t c l u l t h c ; ~ t lo l - r l ~ c r. p inserrioll \ t ~ . t n .  &#226;&#8364;&#732;lllir(l inst:tr <br /> l:~t.valtwain. UCNC                cspwssion h r m;litl c - l ~ ~ s     c rc-dls. <br />                                                                                         s         .l&#226;&#8364;&#732;t~c  t w o w111ra1rlu\tcrs c-orrcsl~ot~~l             n osition ~ M Isin. IO 111ccell oclic t l l c . I ~ ~ O I01-  \ 1111- m d 1 r o o m Iwtlics. . h t \ w II~T:II nwonsll;qwd c111strrsc o r r q m n d t o r l w ;IW w h e v ~rhc tlotorc.c-c.l)t&#226;&#8364;&#732;,r. c-c*llsof 111c~ v c <br /> in1agin:aIdisc- s v n p ~ .w t h neurons 111cl)r;iin. 51;111vc-c*Ilssc;trtrrctl t l ~ r o u g l ~ o ~r tr~tc . I C ~ V O I Ss v s ~ c m;llso cuprcss /nr%. 1%. .~ -;IIII(.IIII  <br /> imtgitxtl disc-. Uocc IIII. ~IainiugI ) c ~ h l ~ trt~c      l   rllorl&#226;&#8364;&#8482;llogcllc.ric. f ~ ~ r r ~ (1.    n v Sew <br />                                                                                                                .                              :Ill st;lining n ~ ~ c - h111. <br />                                                                                                                       <span id='am-1' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> ; t d u t c;~tl.                                      r : I I ~ ~ O O I Ih l w . <br />                                                                ~ ; ~ g ~ m~ pI , l ~ cwl l l s r c m n 1 n o u t h ptrrs. <br /> l ; n l n ; t : 11. l&gt;r:ainrind: s. s u l ~ o c s c ~ ~ ~ganglion: <br />  <br />  <br />  <br /> 1 3 ) . Ikwd morpl~ologyand position,                                                                    n t h c ;ldrtlt flv w ex;lmincd @-g;ll;lctosid;~sc <br /> cells prolxlbly corrcspontl t o glial cc*lls.so111cof                                                       expression n sc-ctionso f h c w l . s sllown n Figure <br /> I1;1vc hccn descri1)ed previously ( ~ACO B Sand (;MI&gt;-                                                            X ; . staining observed n lamina, l n n rind, <br /> MAS 1989:.],\COnS PI / . 1989: ~ l , \ 4 l 3 &#226;&#8364;&#8482; l ~ ; l l (d; 0 O D S N                                  sul,ocsoph;lgc;~lganglion ant! clusters cc*lls n t h c . <br />  1 9 9 ; FREI)IEC ;mtl \ ~ H O W , D 1989).                                                               11lo11t hp;lrts. lac% e s p r c s d m ; l n y nuclei <br />     ~j-(;;ll;lctosid;lsc expression \\. esanlined n                                                     retina, mctlull;l. lol,rlla ;1nd l o l ~ l ; plate,    ~ r ocelli instar- lrv;rcbrains ant! imaginal discs. s sho~vn                                                      ocellar ganglion. Hcncc, cells  t hc visual n Iigurc 3  lac% expressed n wval brain n                                                        systcrn cxl)rcss marker gene.  lny cells n f ~r nr; t n clusters cells. set h s ;l n1oonsh;lpcd                                                 ; l r l l ~ ~ l l l 1 ascgrllc&#226;&#8364;&#8482;nts <br />                                                                                                                                           l          ;1lso strongly C&#226;&#8364;&#8482;spr&#226;&#8364;&#8482;ss nlilrkcr <br /> fi)rrn n hcnlisphcrc  1)rain. l&#226;&#8364;&#8482;hesc cells                                                          gcne. Furthcrmore. lacZ cxpr-cssccd n cells o f <br /> prolxtl)ly correspond cclls w t h pho-                                                         muslwoom oclics othcr parts bin. <br /> torcccptor cc4ls f - o m eye im;lginal disc synapse n                                                     Finally, t w o cells stain  t h c h s e m111v h n . T l l c scconcl bilatcrallv symmctr-ic cluster                                                   n~;lcrocl~;~et;w              n hcad. nl;~rkcrgcv1c c . e ~ ~corrcsponcIs <br />          s                size n c t ,c;Ition wit11 t11c cell                                              csprtwc&#226;&#8364;&#732;d m ; l n y cells ;ldult PSS ; d <br /> 1)otlic.s nlushroo111 bodies. S ~ I C I                                                              ~ C                                                                                                                   C S S (PY~NG-IAV                 IAN,  personal conlmrrnic;ltiorl). <br /> brain thought t o play key role n olfhctory learning                                                             Enhancerdetectorinsertions                   90D affect nlc*mory. n ;ldtlition thcsc cell clusters                                                          essential gene: &#226;&#8364;&#732;1&#226;&#8364;&#8482;0 map Icth;ll mutations (a1)Ix-c- <br /> 111;lny scattcretl cells n brain csprcss 1acZ. <br />                                                                                                                   viatcd lO, 1 = Icth;ll)0 1 1 t h c chromosomcs t l u t cxrry <br /> !:inally, clusters o f cells arc associated                                                           tllc 9 0 1 ) inscv-tions, wc pcrfort1lcd complcmcnt;lriorl <br /> w t I1 t h c h n  ! loc;ltcd bet wen 1,0111 ccrclml <br />                                                                                                                   IC-sts w t h  series dcficicmcic.s t h t lack cytological <br /> Iwrnisphcrcs c ~ p r c s slarZ ( n o t shown). clus- <br />                                                                                                                   1);tntls w t h n ;1rountl t h c 9OD/E subdivision. Somc <br /> ter snlall, loc;~tctlccntr;Illy ;Ind b l l sh;qml.                                                                                                                   o f rcstrlts tlwsc conlplcnlent;ltion tc&#226;&#8364;&#8482;sts arc t w o arc largc~,      form ;t pair, ;1nd arc pear slwpcd. <br />                                                                                                                   slwwn n iglrrc 4 ;lnd l&#226;&#8364;&#8482;aldc 1 . 1,ethaI rnrrt;ltion(s) pl&#226;&#8364;&#8482;l~esc   clusters prolx1l)ly clong t o ringgl;\ntl <br />                                                                                                                   ;ill tc1stc.d cIII.oIllosoIllcs t h ; ~c;lrry 901)/1~. inscrtiorls <br /> ;tnd stomatogastric nervous s;&#226;&#8364;&#8482;stcrn (&lt; IPOS- <br />                                                                                                                   arc uncovcrcd h y tllc tlcficicmcics. ji&#226;&#8364;&#8482;31&lt;)1&#226;&#8364;&#8482;-14. <br /> O R T E G ;IIlt! I1ARTESS ElS 198.7). <br />     8-(;;llactosirI;tsc csprcssion n i11laginal discs  w t h <br />                                                                                                                   f(3R)g111SIO, J&#226;&#8364;&#8482;31&lt;)/(3)11S6;111tl j?31&lt;)1)G4( S C C \ f - <br />                                                                                                                   TERIAI. ANI)  TllOl~S),l\&#226;&#8364;&#8482;hct-&lt;&#226;&#8364;&#8482;;lS ji&#226;&#8364;&#8482;3/&lt;)gl11.Y-1 C o 1 1 l ) k - <br /> t h e exception o f eye disc, confirlcd t o singlc cells <br />                                                                                                                   merits $101) insrrtion ~ ~ l l r o ~ l l o s oJiorcovcr,~~~cs. <br /> o r snl;lll medium size clusters ofcclls.\Ye compared <br /> t hc lac% expression p t t c r n n thc wing imaginal disc <br />                                                                                                                   thc c t h l insertion chron1oso11ws (11-9 ) fail t o com- cpI insertion strain w t h p t t c ~ ntlccril,cd                                                   plcmc1lt c;lch othcr. Ic11cc* inscrtions ilffcct  b y GIWSES O&#226;&#8364;&#8482;KANE( 1 OXO) f X J enhancer                                                             co11111lon             cssenti;ll gtvw t h t 1n;1ps t o t h c 901)2-90!~6 <br /> tlctcctor cspress&#226;&#8364;?d n SS. Rot11 patterns                                                         cytological Ixlnds  1:igurc ) . <br /> arc similar n 11otum triple row bristles                                                                  1 1 deficiencies klil complcmcnt t h c <br /> precursor cells n wing region ;\t&#226;&#8364;&#8482;c n o t lbclctl n                                                     cIlromoson1c~scontaining 901) insertion ;11sof l t o c p l wing disc. n t h c c*ye imaginal disc nlany                                                           complcnlcnt sr&#226;&#8364;&#732; (GAIIXY IIAI.1. W?): s c e 1:igul-e <br /> photoreceptor cells express @-g;ll;lctositl;1scI)chind                                                        4). Iolnozygo1rs .w&#226;&#8364;&#8482; flies h;~ve;1 gray str-ipc nlorphogcnctic f \ l ~ ~ o(SCC   \ \ . 1;igurc 313). Fi11illly. lac%                                              thorax resr~lt lack t h c dorso <br /> expression restricted PSS CNS                                                               longitudinal indirect flight musclc. (:onlplcnlcnt;ltion n instar l r ~  e salivary glands csl)rcss                                                    tests w t h sr&#226;&#8364;&#8482; ; n d 901) irlsc*rtioncIlro11wsonl(*s <br /> ~~-g;ll;lctosi&lt;l;lsc..                                                                                            indicate t h ; l t csscnti;ll gene ; 901) complements <br />                                                                                            Genetics couch potato                                              369 <br />                                         90D/E <br />                                                                                                         allow order different alleles pro- <br />             B                  C                                               E <br />                                          D          E          F                              C <br />                                                                                                         gressive allelic series partially complement <br />                                                                              1                          homozygous lethal insertions: 11-8 &gt; 19 &gt; c p l = <br />       90 <br />                                                                                  CDO   lr                                                                                                                                 cp2. T h e complementation data indicate                                                3-620                     J-bl                         insertional alleles null alleles example <br />                                                93C2-01.91BI-2                          P-f4 <br />                                                                                                         c p l l l l cp1/12 flies reduced viability c p l l <br />                                                9OC7 . 9181                       g18XfO               deficiency flies sub-lethal Illdeficiency 121 <br />                                               90DI. 9OP10-12                 1(3)~y,5                   deficiency flies lethal (Tabel 1). <br />                                                                                                            T o define embryonic mutant null phenotype                                              9302-4.9OF3-6                064 <br />                                                                                                         necessary recoveramorph alleles.                    9001-2.90~1-2                         srl6                                            performed A2-3-transposase excision mutagenesis <br />      sr-(                                      9 x 7 - 8 . 90EI-2   D T ] s r ~ 4 6 ~           &apos;                                  (BELLEN      et al. 1989). 500 ry- flies lost                                                                                                         ofthe 90Dc p l enhancer detectorwere screened, <br />                                                                                                         newly isolated mutants failed complement <br />      sr+    &lt;                                    90F8-II.91B1                     gJBYI <br />                                                                                                         11-2 Dj(&apos;3R)Pl4. new alleles                                                                                                         named 1Acpl l c p 1 2 . shown Table 1,    FIGURE4,&quot;Extent cytological visible deficiencies                                      l c p l l mutation fails complement lethality OI)/E region. cytological extent deficiencies                                      alleles tested amorph <br /> shown. provided KEVIN MOSE~(personal communi-                                              allele. otherallele l c p 1 2 behaves relatively <br /> cation) DON GAIIXY(GAILEYand HAIL 1989), determined n work (Dj73R)wf6 Dj73R)sr&quot;&quot;).                                                                                                         weak hypomorph various complementation <br /> 110 known breakpoint SY cpo demarcation                                         tests. Based complementation data presented genes unknown.                                                                               Table 1 reordered alleles decreasing <br />                                                                                                         strength following allelic series: Dj(&apos;3R)~r&apos;~~&apos;      = sr&apos; allele gray stripe observed transhet- <br />                                                                                                         Dj(&apos;3R)d6= l c p 1 1 &gt; 11-8 &gt; 19 &gt; l c p 1 2 &gt; c p l = cp2. <br /> erozygous flies.  sri6 sr1461,diepoxybu- <br />                                                                                                            cpo required embryonic viability, cpo <br /> tane (DEB)- ethyl methanesulfonate (EMS)  <br /> duced alleles  LA POMPA,GARCIAandFERRUS                                                              mutant adults couch potatos: anticipated 1989), f l complement lethality chro-                                                 flies homozygous hypomorphic alleles cpo mosomes thatcarry90Dinsertions             Table 1).                                                adult behavioral defects amutant Cytological examination stripe alleles reveals                                                 isolated screen reduced flight ability. small cytologically visible deficien-                                                 noticed homozygous c p l adultsrecovermore <br /> cies: srI6 = Dj(&apos;3R)9ODI; 9081-2 sr&apos;461 =                                                           slowly ether anaesthesia heterozygous <br /> Dj(&apos;3R)90C7-8;9081-2  Figure 4). Based                                                       balanced flies, indicating behavioral  <br /> data SORSA(1988) estimate deficien-                                                    fects associated insertion chromo- <br /> cies larger 70 kb probably 300                                                   somes.  decided performa series kb.  essential gene described                                                    simple behavioral experiments flies carrying  <br /> stripe genemap 90D1; E1-2 cytological                                                        dependently isolated alleles different ge- <br /> interval.                                                                                               netic screens. selected simple assays meas- <br />     Insertions 90D hypomorphic alleles:                                                     ure phototaxis (BENZER 1967), geotaxis, flight  <br /> enhancer detector strainswith insertion 90D                                                   sponse recovery time ether anaesthesia  homozygous viable. ofthese strains isolated                                                   MATERIAL METHODS). <br /> o n basis expression pattern,                                                      Table 2 summarizes result behavioral <br /> strain isolated screen mutations thatcause                                                 tests. assay results easily reduced flight abilities. Complementation tests                                                    subdivided groups: high (+), interme- strains, various deficiencies shown                                                  diate ( ),   low scoring (-) class  MATERIALS Figure 4, homozygous lethal alleles                                                  METHODS). data homozygous 4 1 , <br /> indicate viable alleles subdivided                                                 c p 2 v3 flies exhibit behavioral defects assays. groups (Table 1). insertions, named c p l                                                   addition 11 12 insertional allelesfail c p 2 ( c prefers couch potato phenotype, ,                                              complement c p l c p 2 alleles fail complement poorly complement defi-                                                      assays.  homozygous v3 flies ciencies partially complement 11 12                                                    behaviorally defective assays, v3 inser- <br /> insertions cause lethality.  c p l c p 2 ,                                           tion fully complements 11, 12 c p 2 insertions. hypomorphic viable alleles essential comple-                                                 suggests v3 insertion complements mu- <br /> mentation group defined 11-9 insertional al-                                                     tations gene, v3 affects leles. homozygous viable 90D inser-                                                     nearbygene absence reduction causes tions, V , v2 v3, reduced viabilityin                                                     similar phenotype. T o determine mutations <br /> various complementation tests. c p l c p 2 alleles                                              associated insertions contribute <br />  370                                                                    H. J. Bellen et al. <br />                                                                            TABLE 1 <br />                          Complementation datafor insertional alleles deficiencies cytological subdivision90D <br />  <br />                                    19           11          12            C P1          CP2       V l       v2       v3           IAcp I       1Acp I2 <br />         19                         - <br />         11                         -            - <br />         12                         -            -           - <br />         CPl                        +          +/-          +/-             + <br />         CP2                        +          +/-          +/-             +             + <br />         V                        +            +           +              nt            +        + <br />         v2                         +            +           +              +             +        +         + <br />        v3                         nt            +           +              +             +        +         nt       + <br />        IAcpll                      -            -           -              -             -        +         +        +              - <br />        lAcpl2                    +/-          +/-          +/-             +             +        +         +        +              -              - <br />        DfljR)P14                   -            -nt         - <br />                                                             nt           -/+ <br />                                                                           nt            nt                                          -              - <br />        Dfl3R)glEXI 0              nt            -           -              -             -        +         +        +              -              - <br />        Df3R)glBX6                 nt            -           -            -/+             -        +         +        +              -              - <br />        Dfl3R)DG4                   -           -            nt           -/+            -/+       +         +        nt             -              - <br />        DJ3R)srf6                   -           -            -nt            <br />                                                                           nt            nt                           nt             -              - <br />        Df3R)~r&apos;~~&apos;                -            -            -            -/+ <br />                                                                            nt           nt                 nt        nt             -              - <br />        Dfl3R)glEXI                +            +            +     nt       +       nt    +        nt                                +              + <br />        sr &apos;                       +            +            +              +             +        +         +        +    ~~~~~ <br />                                                                                                                                     +        ~ <br />                                                                                                                                                    + <br />    - = transheterozygous progeny recovered; + = 50% expected            transheterozygous progeny recovered; +/- = 50% 10% expected transheterozygous progeny recovered;        -/+    = 10% expected transheterozygous progeny recovered; nt = tested. <br />  thebehavioral defects, c p l , v3 11 insertions                          MAb6D5 (data shown). T o screen defects excised using A2-3-transposase mutagenesis                                 CNS andthe PNS cpo mutantstrains, scheme. 10 independent homozygous viable                                   immunocytochemically stained nervous strains insertion strains                              embryos homozygous different cpo alleles <br /> recovered andfurther            analyzed. viable fly                          anti-HRP (JAN JAN 1982), anti-cut (BLOCH- <br /> strains tested behavioral assays.                               LINGER et al. 1990) monoclonal antibody 22C 10 original insertion independent rev-                               (CANALand FERRUS1986). obvious defects ertant strains recovered scored positive                                 observed nervous systems embryos strains <br /> (+) assays. observations indicatethat                            carryinginsertional alleles C ~ O &apos; ~ PThus,  ~ ~ . cpo insertions responsible forthe behavioral                                  essential observe obvious morphological <br /> defects lethality case I1 insertion.                           defects cuticle, muscles nervous  summary,thesedataindicatethat                                            Enhancer detector insertions cpo map 250- <br /> covered mutants essential gene                                  bpfragment: Genomic DNA flanking inde- insertional alleles affect function gene                              pendently isolated insertions near cpo iso- required normal behavior.                                               lated using plasmid rescue technique  WILSON <br />     Viable mutant flies (e.g., c p l l c p l , c p 2 / c p 2 , c p l l l l ,        et al. 1989). rescued fragments, map c p 2 / 1 1 ) abnormal adult behavior                             cytological subdivision 90D1-6, used screen development significantly delayed                               lambda DASH library, 50kb contiguous flies eclose 1 usually 2 days                          sequences genomic DNA 90D region balanced flies. addition, flight assays                            isolated  Figure 5 ) . impression cpo mutants unable fly,                                identify functionally important domains able fly limited distance prodded.                               cpo, mapped positions 13 enhancer <br /> Finally, c p l c p 2 flies groom time                         detectorinsertions Southern analyses. Ali tested behavioral assays.Based                                  insertions mapped far located 250-bp <br /> abnormal hypoactive behavior mutant flies                                genomic fragment. insertion causes homo- named gene couch potato (cpo).                                          zygous lethality (19) maps left half ge- <br />     cpo alleles cause lethality late embryonic                              nomic fragment (Figure 5). T h e 19 insertion behaves <br /> development.Forexample,              homozygous cpo&quot;                            relatively weak hypomorph complementation <br /> cpoiACPI1first instar larvae fail hatch aresluggish                          tests (cpo&quot; Table 1). insertions map 150- extracted manually.  did ob-                                    250 bp right      19 insertion (11-6,8, V , v2, <br /> serve obvious defects reduction muscle size                               v3, c p l c p 2 ) . insertions homo- cpoiAcP1I,cpo&quot; Df(3R)DG4 embryos using                                    zygous viable (vl-v3) map different orientation <br /> immunocytochemistry muscle-specificantibody                                  insertions. T h e v 3 insertion causes                                                       Genetics couch potato                                                                                                      37 1 <br />                                                              TABLE 2 <br />                                                         Behavioral assays <br />  <br />                   CP2            CPl   11      12                                                     U3                    cs             TM3                      ry <br />  <br /> Phototactic response <br />   CP1 <br />                                        -       -             -          -                             nt                    +              +/-                      + <br />   cP2 <br />                                        -       -             -          -                             +                     +              +/-                     nt <br />   V3                                   +        +            nt         +                             -                     +              +/-                     nt <br />   (3                                   +        +            +          +                             + nt                  +              nt <br /> Flight response <br />   C P1 <br />                                        -        -            -         nt                             nt                    +              +/-                      + <br />   cP2 <br />                                        -        -            nt        -                              +                     +              +/-                      + <br />   v3        nt                         nt                               +                             -                     +              +/-                      + <br />   cs                                   +        +            +          +                             +                     +               +                      nt <br /> Geotactic response <br />   CP1 <br />                                        -        -       nt   -          nt                                                  +              +/-                     +/- <br />   cP2 <br />                                        -        -            nt         -                             +                     +              +/-                      + <br />   v3                                   +       +/-           nt         +                             -                     +              +/-                     +/- <br />   cs                                   +        +            +          +                             +                     +              nt                      nt <br /> Ether recovery <br />   CP <br />   CP2 <br />        &#226;&#8364;&#8482;                               - <br />                                        - <br />                                                 - <br />                                                 - <br />                                                              - <br />                                                              - <br />                                                                         - <br />                                                                         - <br />                                                                                                       nt <br />                                                                                                       + <br />                                                                                                                          +/- <br />                                                                                                                             + <br />                                                                                                                                            +/- <br />                                                                                                                                            +/- <br />                                                                                                                                                                      <br />                                                                                                                                                                     + <br />   V3                                   +        +            nt         +                             -                     +              +/-                       <br />   CS                                   +        +            +          +                             +                     +               -                      nt <br />   MATERIAL    METHODS   symbols. <br />  <br /> cpo phenotype complements cpo alleles,                                                                     SI&#226;&#8364;&#8482; <br />  <br />  <br /> including deficiencies. insertions map <br /> approximately 40-60 bp left cluster viables. homozygous lethal. Fi-                                                                   \      nally, cpo&#226;&#8364;&#732;p&#226;&#8364;&#732; cpocp2 insertions map close               EH    xn H xn <br />                                                                               / <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 H <br />                                                                                                                                                                                                 , <br />                                                                                                       nx <br />                                                                                                       , <br />                                                                                                       ! <br />                                                                                                                  E <br />                                                                                                                                                                                                                                                                                                                                                        f /P        EX <br />                                                                                                                                                                                                                                                                     I <br />                                                                                                                                        n <br />                                                                                                                                        , <br />                                                                                                                                                                                                                                                                                     HXxn <br />                                                                                                                                                 / / <br />                                                                                                                                                          n <br />                                                                                                                                                                                                                                                                                                                       nn <br />                                                                                                                                                              I <br />                                                                                                                                                                     EEH <br />                                                                                                                                                                     l l <br />                                                                                                                                                                             n <br />                                                                                                                                                                                                                                                                                                                                                                    E E <br />                                                                                                                                                                                        I <br /> vl-3 11-6 insertions. noted mapping data tentative based                                       -                            -                                               - <br />                                                                                                                                                                          - <br />                                                                                                                                                                          -      61.1 <br />                                                                                                                                                                                 61.2 <br />                                                                                                                                                     Y <br />  genomic Southern, andplasmid rescue data.                           lnurtlon rlta IQ                                                      1kb <br />  <br />                                                                              Insertion  11. 12.13,  6,II insertions null alleles cpo,               v     insmlon sit-       VI.   VZ,   v3 probably located regulatory region                                  site cpt, CPZ <br />                                                                              ~n.srt  gene.                                                              FIGURE5.-Molecular map cpo mapping position    Genomic Southern analysis revealed                     P element insertions hybridization sites cpoiAcf&#226;&#8364;? deletion possibly combined              longest cDNAs. ~ p o &#226;&#8364;? ~ &#226;&#8364;?  severe allele, deficiency. rearrangement P element sequences  Figure                  excision event lead loss BamHI site located <br />                                                                     200 bp left c p l insertion. precise extent 5),                                       internal deletion c p l           deficiency unknown restricted boxed region enhancer detector (data shown). Quantitative ge-                figure. c ~ o &#226;&#8364;?                                                                                          - ~ hypomorphic lethal alleles, map <br /> nomic Southern analysis showed Dx3R)sri6and                    50 bp. cpo&#226;&#8364;? allele weaker hypomorph cpo&#226;&#8364;?. <br /> Dx3R)~r&#226;&#8364;&#8482;~~&#226;&#8364;&#8482;  lack DNA corresponding entire               x alleles insertion maps different position. Note                                                                     insertions homozygous viable ( v l , v2 v 3 ) inserted <br /> genomic walk shown Figure 5.  <br />                                                                     opposite orientation insertions. insertions <br /> firmed situ hybridization polytene chromo-                 behave differently complementation tests c p l cp2 <br /> somes carrying deficiencies using various frag-               insertions inserted opposite orientation. data <br /> mentsfromthegenomic           walk probes(datanot                indicate position insertion, possibly orienta- <br /> shown).                                                             tion size insertion mayplay role themutant <br />                                                                     phenotype. E = EcoRI, H = Hind111 X = XbaI. <br />    90D insertions intron cpo: 5- <br /> kb NotI-Hind111 plasmid rescued genomic fragment                    screen embryonic lambda gt 10 cDNA library 3&#226;&#8364;&#732; end cpo&#226;&#8364;&#732;p&#226;&#8364;&#8482; insertion previously                (provided L. KAUVAR) cDNAs 1.6 <br /> shown contain repetitive DNA hybridize                 and1.8kb     isolated, thesewere used major 2.7-2.8-kb transcript    possibly           screen size selected 9-12-hr-old embryonic cDNA <br /> minor transcripts 1.6- 4.4-kb range; cpo@&#226;&#8364;&#8482;               library (ZINN, MCALLISTER  GOODMAN 1988).  corresponds strainB52.1 M3 Figure 4 WILSON                 longest cDNAs, 17, 61.1 61.2,   62 <br /> et al. 1989; fragment used genomic Southern                 positives selected forfurther     work. Northern blots figure 5-kb NotI-                 cDNAs 3.0, 3.6and 2.5 kblong, respectively. <br /> Hind111 fragment). genomic fragment used                   Southern analysis shows cDNAs  372                                                 H.1. Rellen et al. <br />       v                                                                         - <br />                /           . .. <br />  <br />  <br />  <br />  <br />                                                               .. .. <br />                                                                              <br />  <br />  <br />                                                           *           0- <br />  <br />  <br />                                           B                                   C <br />        4 <br />                                   &quot; <br />  <br />  <br />  <br />  <br />                                        t <br />                                                                                                                         4 <br />                                                                                 4 <br />  <br />       D <br />  <br />  <br />            . <br />                             -               E                                   F <br />  <br />  <br />  <br />  <br />       G <br />  <br />  <br />  <br /> dcrivetl fronl  message w t h different splicing pat-     positioned peripheral nerves express <br /> tern. 61.2cDNAsstronglvhybridizes                  cpo. glial cells form sheath periph- <br /> genomic fr;Igments sides insertions           eral nerves, sheath easilv seen conclude enhancer detectors likely            vicinitv CNS. cytoplasm glial cells <br /> inserted n intron transcription unit cpo.        ;IStheir projections clearly labeled, masking <br /> Genomic Southern analyses, Northern analvsis                possible staining axons. embrvos reverse Northern analysisindicate          thethree        able identify strong staining cells located <br /> &lt;:DNAs contain highly repetitive D N             dorsal pair cells ventral <br /> fined  2.2-kb Hind111 fragment shared               CNS. cells correspond cDNAs  Figure.5). Reverse Northern exper-             location cells express @-galactosidase <br /> imentscovering entire walk using 9- 12-hr-old               enzyme insertion strains mostlikely <br /> poIV )R N revealed arc             glial cells. <br /> transcripts detected w t h n introns shown Fig-         comparisonof       embryonic@-galactosidase <br /> ure 5 (di1t;t sho\\r\.n).                                   staining cpo enhancer detector strains w t h     shown n Figure 6 , situ hybridization experi-       expression pattern cpo mRNAs shows ments mount embryos w t h digox-           enhancer detectors quite hithfullv reproduce igcnin labeled cDNAsshow transcripts               expression pattern transcripts. onlvob- <br /> cxpressed n SJ4C PNS  n           vious difference patterns shown Figure <br /> differentiated cells PNS (Figure 6 , B C).           1 Figure 6 canbe         attributed nuclear cell clusters, example cells lateral            localization @-galactosidasestaining (BEILEN PI <br /> cluster, <span id='am-19' about='rdfs:label' typeof='owl:Thing'>label</span> intensely (Figure 6E).          al. 1989: M&apos;IISON ~t al. 1989) predominantly <br />  hecytoplasmof neurons isweakly labeled,                 cvtoplasmic staining cpo transcripts. m n y axon bundles nerves labeled, espc-           earlydevelopmentalstagesboth           patterns ;Irevery <br /> cially ventral embryo              similar. example, stage 7-8 embryos express pcriphcr;II nerves connect w t h CUS (Figure 61:).        l d transcript fewcells t clear transcripts n neurons contrib-       anterior end embryo. stage 9-10 embryos ute labeling axons glial cellsthat        cclls presumptive segment express                                                   Genetics couch potato                                            373 <br />  <br /> transcriptatabout         4:20-4:40 hr development,         detailed analysis lacZ expression pattern 0.5 hr earlier @-galactosidaseexpres-           enhancerdetectorstrains        showed lacZ ex- <br /> sion cells, addition, types           pressed glial cells CNS PNS staining compared, subtle differences beseen           low levelsin cellsof midgut. n situ intensity cells cell clusters        hybridizations mountembryos             cpo PNS labeled. T h e ZacZ expression staining         cDNAs indicate cpo expression confined PNS <br /> intensity similar cellsof        lateembryos        glia, possibly CNS.  transcript abundantin             unable establish expression CNS specific cells, especially chordotonal organs.         significantly higher background. major difference betweenthe expression patterns            convincing evidence cpo expressed faint staining observed CNS          midgut. defects observed muscles, situ hybridizations.  label         PNS CNS amorphic mutant embryos        possible nonspecific homozygous DJ3R)srI6 embryos              cpo required embryonic nervous stain CNS.                                         anessential behavioral, biochemical, physiolog- <br />    Expression cpo altered somemutants:                ical functionwithoutaffecting       morphology Mutant embryos thatlack ormost cpo gene            CNS PNS. cpo required PNS alterations cpo expression. com-             viability, conclude pared expression cpo mutant (cpox/cpox) wild         chordotonalorgansare        essential viable <br /> type (cpox/+, +/+) embryos using mount n            mutations scute-achaete complex cause loss situ hybridizations. Expression cpo clearly altered      PNS organs chordotonals. Future molec- embryos homozygous deficiencies cpo. Em-             ular genetic experiments address ques- <br /> bryos homozygous DJ3R)srI6 embryonic lethal            tions. lack sequences cpo               insertional alleles 90D mapped far cDNAs hybridize. quarter embryos              confined 250-bp interval sequences appear D J ~ R ) s ~ &apos; ~ / Tstrain <br />                           M ~ labeling       intron cpo transcript, 5&apos; CNS. Homozygous lethal cpo&quot; embryos larvae              start cDNA (data shown). ob- obvious defects PNS CNS              servations indicate neighboring se- &apos;/4 embryos balanced strains exhibit       quences play important role regulation cpo severe reductionof amountof label cells          transcription.  insertional mutations 90D PNS older embryos (stage 14- 17) label           tested cause lethality behavioral defects  seen thechordotonalneurons          (compare        v l v2 insertional strainshave testedyet). <br /> Figure 6G Figure 6E). Similar observations           Second, overall decrease level cpo transcript embryos        cp0&quot;~~&quot;/TM3flies, indicating    PNS observed insertion strains, 90D insertional mutationsand deficiency          homozygous 11 embryos transcript confined <br /> insertion site affect level cpo mRNAs           chordotonal neurons stage 14- 17 embryos spatial temporal expression cpo. <br />                                                                  small deletion theinsertion site leads <br />                                                                expression pattern observed                      DISCUSSION                                homozygous 11 embryos. conclude     work genetic phenotypic        sequences essential proper expression characterization alongwith preliminary molecular           cpo PNS theglial cellsand thatthey contain <br /> analysis newly identified gene namedcouch potato          key regulatory sequences. <br /> ( 9 0 ) . Insertions cpo isolated         number cpo alleles cause severe behavioral <br /> independent   enhancerdetector <br />                             screens                    defects adults. behavioral abnormal- <br /> expression lacZ reportergene essentially             ities enhanced uncharacterized mu- <br /> confined theSMC differentiated      cells      tations results various behavioral assays PNS embryonic development.                 revertants clearly indicate cpo requiredfor <br /> insertionstrains interesting             normal adult behavior.  behavioral  <br /> enhancer detector strainsat positions 90D           fects indicative general sickness label sensory mother cells progeny.           specific behavioral deficits. analysis lacZ <br /> addition, sensory organs dispensi-           pattern various 90D enhancer detector strains <br /> ble viability, number cpo insertional mutations       suggests cpo expressed CNS, PNS foundto beembryonic lethal suggesting            salivary glands instar larvae adults gene plays important role development             tissues. conclude function PNS.                                           cpo gene probably required mature nerv- <br />    observation cpo essential raises im-         ous normal adult behavior. <br /> portant question: tissue cpo required?              Complementation behavioral tests showed  374                                                  J. Bellen et al. <br />  v 3 insertion causes behavioral defects butthat           iological role. cpo gene ispossibly defects observed trans-heterozygous          hottest spot P element-enhancer detector inser- <br /> flies carry av 3 insertion anothercpo insertion      tions genetic evidence indicates genomic affects behavior causes lethality. excision     region P elements inserted key v 3 allowed recover strains exhibit     regulatory region gene. molecular wild type behavior, conclude v 3 insertion        genetic analyses progress determine causes cpo phenotype, uncharacter-           role cpo nervous  <br /> ized mutations unlikely play arole                                                                  grateful CLIVEWILSONfor helping initial <br /> phenotype. possible v 3 affect         steps project. thank ROLF BODMERfor teaching H. <br /> gene near cpo plays similar role cpo,          BELLEN   intricacies embryonic PNS, RON DAVIS   unlikely v 3 inserted         KAI ZINN providing libraries. thank COREYGOODMAN <br /> insertions cause lethality affect         RICHARD    DORSKY    communicating unpublished results, Su- <br />                                                               SAN SHEPERD     ED GRELLfor advice, KEVIN MOSES, GERRY <br /> behavior. conclude v 3 insertion <br />                                                               RUBIN,DON GAILEY,       JEFF HALL,ALBERTO    FERRUSand Dro- <br /> complements insertions cause cpo phe-            sophila stock centers Bloomington (Indiana), Bowling Green <br /> notype. intragenic complementation            (Ohio), Umea (Sweden) sending fly strains  thank transvection-like phenomena insertions           BETHELDON,PATRICK LAWINGER,         YUN-TAIKKIM andJuAN BOTAS <br /> disrupt somatic pairing chromosomes trans-          comments manuscripts, anonymous reviewers <br />                                                               fortheir constructive helpful criticism. research heterozygous flies.                                           supported grant Muscular Dystrophy Association    Mutations cpo stripe fail complement         H.B.H.B. Assistant Investigator Howard Hughes <br /> tested deficiencies cytological subdivisions 90D/E.        Medical Institute. mutants gene      fail fly, preliminary evidence suggests stripe required                             LITERATURE CITED nervous (COSTELLO      WYMAN 1986;          BELLEN,   H. J., C. WILSON W. GEHRING, 1990 Dissecting LA POMPA, GARCIA FERRUS         1989), investi-          complexity nervous enhancer detection. <br /> gatedthe possible relationshipbetweenthese                     BioEssays 12: 199-204. <br /> genes greater  Based following obser-        BELLEN,   H., J., C. J. O&apos;KANE,C. WILSON,U. GROSSNIKLAUS,        R. K. <br />                                                                    PEARSON    W. GEHRING,1989 P-element mediated  <br /> vations believe cpo stripe different               hancer detection: versatile method study development genes map close proximity              Drosophila. Genes Dev. 3: 1288-1300. <br /> mutations genes complement             BENZER,S . , 1967 Behavioral mutants Drosophila isolated typical stripe phenotype, e., sr&apos;lcpo flies           countercurrent distribution. Proc. Natl. Acd.Sci. USA 58: <br />                                                                    1112-1119. adarkthoracicstripe;second                   BIER,E., H. VAESSIN,    S. SHEPHERD,   K. LEE,K. MCCALL,   S . BARBEL, <br /> behavioral defects observedin cpo mutants observed              L. ACKERMAN,     R. CARRETTO,   T. UEMURA,   E. GRELL,L. Y. JAN trans-heterozygous srlcpo flies;  genomic frag-           Y. N. JAN, 1989 Searching pattern mutation ments adjacent enhancer detector insertionsin               Drosophila genome P-lacZ vector. Genes Dev. 3: <br /> stripe gene contained 50 kb genomic                  1273-1287. <br />                                                               BLOCHLINGER,     K., R. BODMER,   R., J. W. JACK, L. Y. JAN Y. N. <br /> walk shown Figure 4; fourth, situ hybridization             JAN, 1988 Primary structureand expression aproduct <br /> experiments polytene chromosomes se-                 cut, locus involved specifying sensory organ identity <br /> quences 90D walk revealed                  Drosophila. Nature 333: 629-635. <br /> sequences walks hybridize respectively             CANAL . ,  FERRUS,1986 pattern early neuronal <br />                                                                    differentiation Drosophila melanogaster. J. Comp. Neurol. <br /> 90E1-2 (stripe) 90D2-6 (cpo), agreement                177: 707-720. mapping positions respective enhancer           CAMPOS-ORTEGA,      J. , V. HARTENSTEIN,   1985 Embryonic <br /> tector insertions; fifth, P-galactosidase expression           Development Drosophila melanogaster.Springer Verlag, Berlin. <br /> pattern enhancer detector strains inserted         CAMPOS-ORTEGA,       J. , E.KNUST, 1990 Genetics early <br /> o r stripe different expression pattern             neurogenesis Drosophila melanogaster. Annu. Rev. Genet. 24: <br />                                                                    387-407. <br /> (data shown). observations provide com-             COSTELLO,    W. J., R. WYMAN,     1986 Development indi- <br /> pelling evidence genes different.                rect flight muscle muscle-specific mutant Drosophila  muscle defects observed               melanogaster. Dev. Biol. 11.8: 247-258. <br /> trans-heterozygous stripelcpo flies, suggesting          LA POMPA,    J. L., J. R. GARCIA    FERRUS, 1989 Genetic <br />                                                                    analysis muscledevelopment Drosophila melanogaster.Dev. genes interact (data shown).                  Biol. 131: 439-454. <br />    conclusion, shown cpo required          DOE, C., Q. CHU-LAGRAFF,           D. M. WRIGHTand M. SCOTT, embryonic viability fornormal adult behavior.              1991 prosper0 gene specifies cell fates Drosophila expression pattern indicates mayplay                central nervous  Cell 65: 451-464. <br /> importantrole PNS glia.                 FREDIEU,   J. R.,  P. MAHOWALD,1989 Glial interactions <br />                                                                    neurons Drosophila embryogenesis. Development <br /> absence gene product does cause ob-                 1 0 6 749-757. <br /> vious morphological defects cells indicating         GAILEY,D., J. C. HALL, 1989 Behavior cytogenetics cpo play unknown biochemical phys-                 fruitless Drosophila melanogaster: different courtship defects <br />                                                         Genetics couch potato                                                    375 <br />      caused separate, closely linked lesions. Genetics 121: 773-       glia axon pathway formation Drosophila em- <br />      785.                                                                 bryo. Glia  press). <br /> GANETZKY,     B., C. F. Wu, 1986 Neurogenetics membrane         LANGER-SAFER,P. R., M. LEVINE C.D.                     WARD, <br />      excitability Drosophila. Annu. Rev. Genetics 20: 13-44.           1982 Immunological method mapping genes Drosoph- <br />  GHYSEN , C. J. O&#226;&#8364;&#8482;KANE,1989 Detection neural                  ila polytene chromosomes. Proc. Natl. Acad. Sci. USA 79: <br />                                                                           4381-4385. <br />      hancer-like elements genome Drosophila. Develop- <br />                                                                        MLODZIK,M., Y . HIROMI,U. WEBER,C . S . GOODMAN        G. M. <br />      ment 105: 35-52. <br />                                                                           RUBIN,1990 Drosophila seven gene, member  GHYSEN , C. DAMBLY-CHAUDIERE,          1989 Genesisof                                                                           steroid receptor gene superfamily, controls photoreceptor cell <br />      Drosophila peripheral nervous  Trends Genet. 5: 251-          fates. Cell 6 0 21 1-224. <br />      255.                                                              MOSES,K., M. ELLISand G. RUBIN1989 glass gene encodes <br />  HALL,J., 1985 Genetic analysis behavior ininsects. Compr.             zinc-finger proteinrequired Drosophila photoreceptor <br />      Insect Physiol. Biochem. Pharmacol. 9: 287-373.                      cells. Nature 3 4 0 531-536. <br /> JACOBS, J. R., C. S. GOODMAN,    1989 Embryonic development        O&#226;&#8364;&#8482;KANE,C. J., W. GEHRING,1987 Detection insitu ge- <br />      axon pathways Drosophila CNS.  glial scaffold          nomic regulatory elements Drosophila. Proc. Natl. Acad. Sci. <br />      appears growth cones. J. Neurosci. 9 2402-          USA 8 4 9 123-9127. <br />      2411.                                                             RUBIN,G. M., 1988 Drosophilamelanogaster experimental <br /> JACOBS,J. R., Y . HIROMI,N. H. PATELand C. S. GOODMAN,                    organism. Science 240: 1453-1459. <br />      1989 Lineage, migration, morphogenesis longitudinal        SORSA,V., 1988 Chromosome Maps Drosophila, Vol. 2, p. 146. <br />      glia Drosophila CNS revealed molecular lineage        CRC Press, Boca Raton, Fla. <br />                                                                        TAUTZ, D., C. PFEIFLE1989 non-radioactive situ hybrid- <br />      marker. Neuron 2: 1625-1 631. <br />                                                                           ization method localization specific RNA&#226;&#8364;&#8482;s Drosophila <br /> JAN, L. Y., Y . N. JAN, 1982 Antibodies horseradish  <br />                                                                           embryos reveals translational control segmentation <br />      oxidase specific neuronal markers Drosophila gras-      gene hunchback. Chromosoma 9 8 81-85. <br />      shopper embryos. Proc. Natl. Acad. Sci. USA 7 9 2700-2704.        WILSON,C., R. K. PEARSON,H. J. BELLEN,C. J. O&#226;&#8364;&#8482;KANE,U. <br /> JAN, Y . N., L. Y .JAN, 1990 Genes required specifying cell       GROSSNIKLAUS     W. GEHRING1989 P-element mediated <br />      fates Drosophila embryonic sensory nervous  Trends         enhancer detection: anefficient method isolating char- <br />      Neurosci. 21: 493-498.                                               acterizing developmentally regulated genes Drosophila. <br /> JURGENS,G., R. LEHMANN,         M. SCHARDIN  C.NUSSLEIN-VOL-          Genes Dev. 3: 1301-1313. <br />      HARD, 1986 Segmental organization head em-          ZINN,K., L. MCALLISTERand C. S. GOODMAN,         1988 Sequence <br />      bryo Drosophila melanogaster. Roux&#226;&#8364;&#8482;s Arch. Dev. Biol. 195:        analysis neuronal expression fasciclin grasshopper <br />      359-377.                                                             Drosophila. Cell 53: 577-587. <br />  KLAMBT, C.,   C. S. GOODMAN,   1991 diversity pattern                                Communica <br /> </body></html>