<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Differences evolutionary history disease genes affected dominant recessive mutations <br /> Simon J Furney1,3, M Mar Alb&#195;&#160;2 N&#195;&#186;ria L&#195;&#179;pez-Bigas*1 <br />  <br /> Address: 1Genome Bioinformatics Laboratory. Centre Genomic Regulation, Universitat Pompeu Fabra, Pg. Maritim la Barceloneta 37-49, E- <br /> 08003, Barcelona, Spain, 2ICREA &#226;&#8364;&#8220; Institut Municipal d&apos;Investigaci&#195;&#179; M&#195;&#168;dica. Universitat Pompeu Fabra, Dr. Aiguader 80, 08003, Barcelona, Spain 3Conway Institute Biomolecular Biomedical Research, University College Dublin, Belfield, Dublin 4, Ireland <br /> Email: Simon J Furney - simon.furney@ucd  M Mar Alb&#195;&#160; - malba@imim.es; N&#195;&#186;ria L&#195;&#179;pez-Bigas* - nuria.lopez@upf.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 03 July 2006                                                        Received: 03 March 2006 <br />                                                                                Accepted: 03 July 2006 <br /> BMC Genomics 2006, 7:165    doi:10.1186/1471-2164-7-165 article available  http://www.biomedcentral.com/1471-2164/7/165 <br /> &#194;&#169; 2006 Furney et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Global analyses human disease genes computational methods yielded <br />                  important advances understanding human diseases. Generally studies treated <br />                  group disease genes uniformly, ignoring type disease-causing mutations (dominant <br />                  recessive). report present comprehensive study evolutionary history                  autosomal disease genes separated mode inheritance. <br />                  Results: examine differences protein coding sequence conservation dominant <br />                  recessive human disease genes. analysis shows disease genes affected dominant <br />                  mutations conserved affected recessive mutations.                  consequence fact recessive mutations remain hidden selection heterozygous. <br />                  Furthermore, employ functional annotation analysis investigations disease severity                  support hypothesis. <br />                  Conclusion: study elucidates important differences dominantly- recessively- <br />                  acting disease genes terms protein DNA sequence conservation, paralogy essentiality. <br />                  propose division disease genes mode inheritance enhance                  understanding disease process prediction candidate disease genes future. <br />  <br />  <br />  <br />  <br /> Background                                                                     reports analysed conservation pattern Understanding genetic basis human inherited dis-                        disease genes compared rest genes [2-7]. Human <br /> orders primary goals medical genetics.                     disease genes conserved applying knowledge gleaned research, dis-                        protein level, general, remainder ease prevention treatment improved. Cur-                            human proteome [2], pos- <br /> rently, 1600 human genes known                          sess homologues mouse rat genomes [3]. Fur- associated particular Mendelian disease pheno-                         thermore, study human genes involved disease <br /> types. analysis group genes global                       lower non-synonymous substitution rate (KA) perspective revealed interesting insights                    disease genes compared generic genes [5]. nature human disease [1-5].                                             study sequence conservation <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> level <br />                                                                                human rat small difference <br />  <br />  <br />                                                                                                                                        Page 1 11 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                   http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  KA/KS ratio human disease genes non-        Sanchez et al. showed diseases caused genes cod- <br /> disease genes, significantly elevated syn-     ing enzymes predominantly recessive, onymous substitution rate (KS) disease genes        disorders causative gene coded tran- <br /> discovered [3]. Analysis chimpanzee genome         scription factor  main, dominantly-acting [1]. higher synonymous substitution rate disease         Lopez-Bigas et al. reported similar results genes attributed higher relative frequency    demonstrate dominant human disease genes CpG dinucleotides [6]. Recently work reported          closer paralogues recessive disease genes [14]. Kon- analysis human-rodent KA/KS ratios disease genes     drashov Koonin reported haploinsufficient <br /> compared housekeeping rest genes, finding        (dominant) disease genes paralogues housekeeping genes conserved group           human genome haplosufficient (recessive) genes disease genes showing intermediate values         [15]. housekeeping genes [7]. <br />                                                                 studies investigated differ- studies treated disease genes homogene-      ences sequence conservation disease genes <br /> ous datasets doing neglected inherent        depending mode inheritance. Smith Eyre- <br /> difference sets genes, molecular      Walker [4] analyse gene evolution Jimenez-Sanchez <br /> genotype underlying disease state. autosomal          et al. dataset report higher conservation dominant <br /> Mendelian disease phenotypes understood              disease genes compared recessive disease genes, terms dominant recessive mutations affecting          study observes selective constraints disease <br /> particular gene. cases mutation said     genes compared non-disease genes, result  dominantly-acting, mutation single allele     trary evidence reports, gene sufficient cause disease state. Genes suscep-   small number genes analysed [5]. <br /> tible disease-causing mutations way haploinsufficient. mutation cause loss function     undertaken comprehensive study molec- protein dominant negative effects, reduced     ular evolution autosomal human disease genes <br /> compromised functioning. Diseases caused recessive          depending mode inheritance; genes <br /> mutations require alleles affected, pro-     affected dominant mutations recessive muta- <br /> duction functional protein allele       tions. investigated differences coding satisfy physiological requirements (haplosuffi-      sequence divergence, protein conservation, human paral- <br /> ciency).                                                       ogy, C. elegans lethality, protein function, gene structure <br />                                                                severity disease. comparison evolution- phenomenon dominance recessitivity dip-          ary patterns dominant recessive disease genes <br /> loid organisms recorded quantitatively Gregor     reveals important differences sets Mendel 1866 [8], evolution dominance         genes understood terms different topic debate geneticists cen-   hereditary nature, giving insights  <br /> tury [9]. Fisher, 1928, reported mutations        standing hereditary human diseases. <br /> observed Drosophila recessive wild-type <br /> alleles [10]. attributed balance recur-   Results <br /> rent mutations elimination natural selection.     Differences level protein conservation  Wright (1929) challenged hypothesis stat-        disease proteins affected dominant recessive <br /> ing unrealistically high levels selective pressure     mutations necessary evolution dominance          division disease genes reveals depending manner [11]. proposed dominance inher-          mode inheritance level conservation ent consequence physiology. Kascer Burns (1981)         protein different. Human proteins encoded genes <br /> developed metabolic model dominance congruent            affected dominant mutations conserved Wright&apos;s physiological model [12].            mouse encoded genes affected recessive <br /> metabolic model criticised solely       mutations (p-value Mann-Whitney (M-W) test = 7.59 <br /> based kinetic properties metabolic enzymes           &#195;&#8212; 10-6; Tables 1 <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;2). plot frequency distribu- <br /> [13].                                                          tions protein conservation scores (cs, Methods) <br />                                                                different sets genes, observe set Previous studies analysed human disease genes          dominant disease genes (DD) significantly different <br /> terms nature mutation underlying disease     distribution recessive disease genes (DR) (p-value phenotype [1,2,5]. focus studies      Kolmogorov-Smirnov (K-S) test = 1.95 &#195;&#8212; 10-6, Table 2 mainly differences functional classification    Figure 1) sets disease genes significantly dominant recessive disease genes. Jimenez-              different non-disease genes (p-value K-S <br />  <br />  <br />                                                                                                                   Page 2 11 <br />                                                                                             (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                               http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 1: Molecular evolution paralogy disease genes involved dominant recessive diseases protein level. Average values mouse paralogues conservation score different sets genes. <br />  <br />                                    CS mouse                        n                      CS paralogues                          n <br />  <br />       Non disease                     0.730                     16789                         0.385                           17962 <br />         Disease                       0.795                     1105                          0.349                            1108 <br />    Disease Dominant                   0.810                      474                          0.420                            478 <br />    Disease Recessive                  0.783                      631                          0.295                            630 <br />  <br />  <br />  <br /> tests &lt; 2.2 &#195;&#8212; 10-16). Recessive disease genes display sharp          dominant disease genes 0.420, higher <br /> decrease frequency high levels protein conserva-              non-disease genes.  exam- <br /> tion scores (cs &gt; 0.9), trend apparent dominant               ine sequence conservation pattern paralogues disease non-disease genes. Consistent results                   closely, observe non-disease genes higher <br /> observed analysis cs vertebrates,                 proportion highly similar paralogues (cs &gt; 0.8) Rattus norvegicus, Gallus gallus, Takifugu rubripes,            disease gene groups (Figure 2). differ- Danio rerio  Additional file 1].                               ence pronounced recessive dominant <br />                                                                        disease genes. assess differences statistically Level protein conservation paralogues dominant                calculated scores p-values dominant reces- recessive disease genes                                            sive disease genes 10,000 randomly generated sets <br /> Previously, reported genes involved dis-           proteins. Recessive disease genes significantly <br /> ease conserved paralogues human genes                lower expected number highly conserved para- <br /> general [2], presumably highly similar paralogues              logues  score = -2.76, p-value = 2 &#195;&#8212; 10-3 cs &gt; 0.8). potentially compensate mutated protein [16],              number dominant disease gene paralogues cs &gt; case disease observed.                0.8   significantly different expected <br /> firmed analysis, average conservation              random sampling proteins. conclusion, score (cs) paralogues 0.349 human disease genes              lower number conserved paralogues disease ver- 0.385 non-disease genes (p-value M-W test =                sus non-disease genes entirely attributed 5.24 &#195;&#8212; 10-4, Tables 1 &amp;2).  classifying dis-            recessive disease genes. <br /> ease genes mode inheritance phenotype, significant differences observed                   Selective pressures acting dominant recessive genes involved dominant recessive diseases               disease genes DNA level <br /> (p-value M-W test = 4.44 &#195;&#8212; 10-16; Tables 1 2),              analysed level conservation DNA level congruence previous results [14]. average cs                  sets genes order obtain better understand- <br /> value paralogues recessive disease genes 0.295,               ing selective pressures acting  exam- <br />  <br /> Table 2: p-values Mann-Whitney Kolmogorov-Smirnoff test asses statistical significance differences averages distributions groups. <br />  <br />                                                          Mann-Whitney Test <br />  <br />                             CS mouse              CS paralogs                 KA                         KS                          KI <br />  <br />       nD vs Dis              &lt;2.2 e-16              3.38 e-04                0.170                    1.36 e-10                  0.260 <br />       nD vs DD               &lt;2.2 e-16              5.24 e-04              6.37 e-04                  8.34 e-07                  0.567 <br />       nD vs DR               2.08 e-11              5.34 e-15              2.99 e-06                  1.05 e-05                  0.309 <br />       DD vs DR               7.59 e-06              4.44 e-16              7.17 e-11                    0.212                    0.798 <br />  <br />                                                      Kolmogorov-Smirnoff Test <br />  <br />                             CS mouse              CS paralogs                 KA                         KS                      KA/KS <br />  <br />       nD vs Dis              &lt;2.2 e-16              1.56 e-05              4.70 e-05                  1.12 e-09                &lt;2.2 e-16 <br />       nD vs DD               &lt;2.2 e-16              8.70 e-08              7.98 e-05                  3.06 e-04                &lt;2.2 e-16 <br />       nD vs DR               &lt;2.2 e-16              1.86 e-10              2.66 e-10                  2.20 e-07                &lt;2.2 e-16 <br />       DD vs DR               1.95 e-06              1.84 e-13              1.26 e-11                    0.122                  2.65 e-10 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Page 3 11 <br />                                                                                                            (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                                        http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br />             0.45 <br />                                                            Average cs                             vation scores (Figure 1) representation <br />                                Disease Dominant <br />              0.4 <br />                                                            0.810                                  recessive disease genes observed high conser- <br />                                Disease Recessive           0.783 <br />             0.35 <br />                                Non Disease                 0.730                                  vation scores. <br />              0.3 <br />                                                                                                   Disease genes higher KS values, dominant <br /> frequency <br />  <br />  <br />  <br />  <br />             0.25 <br />                                                                                                   recessive genes, rest human genes (p-value <br />              0.2                                                                                  M-W test disease versus non-disease = 8.34 &#195;&#8212; 10-7 Fig- <br />             0.15 <br />                                                                                                   ure 3B, Table 3). differences ascribed                                                                                                   varying mutation rates genome, simi- <br />              0.1 <br />                                                                                                   larity mean KI values (Table 3) non-significant dif- <br />             0.05 <br />                                                                                                   ferences Mann-Whitney test (Table 2)               0                                                                                   groups negate  Huang et al. reported                    0.1   0.2     0.3      0.4        0.5       0.6      0.7   0.8     0.9     1 <br />                                                                                                   significantly elevated KS level human disease genes <br />                                  conservation score mouse <br />                                                                                                   compared non-disease genes analysis Conservation <br /> Figure level <br /> protein  1     dominant recessive disease genes                                         human rat genes [3]. phenomenon dis- <br /> Conservation dominant recessive disease                                                    covered analysis chimpanzee genome [6]. <br /> genes protein level. Distribution conservation score mouse dominant recessive disease genes versus                                       differences observed KS disease rest genes.                                                                                    non-disease genes necessary analyse differences <br />                                                                                                   ratio KA/KS groups genes.                                                                                                   analysis confirms  observed conservation <br /> ined non-synonymous (KA) synonymous (KS) coding                                               score KA analyses, dominant disease genes exhibit sequence substitution rates, intron sequence                                           lowest evolutionary rates (Figure 3C, Table 3), dis- <br /> substitution rates (KI), human-chimpanzee ortho-                                             ease recessive genes similar average KA/KS value logues [6]. KA indicative selective pressure acting                                     rest genes (nD), distribution KA/KS sites involve change <span id='am-16' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>, KS                                         values groups genes dif- reflective background mutation level. Disease                                         ferent (p-value K-S test &lt; 2.2 &#195;&#8212; 10-16). <br /> genes taken lower KA values non-dis- <br /> ease genes (average KA 0.00295 vs. 0.00318, Table 3),                                          Consistent results analysis KA, KS, agreement previous results [5].  strong-                                         KA/KS human-mouse-dog orthologues  Additional <br /> est differences types disease                                        file 2]. <br /> genes (M-W test p-value = 7.17 &#195;&#8212; 10-11), average <br /> KA 0.0026 dominant disease genes 0.0032                                         Essentiality dominant recessive disease genes <br /> recessive disease genes (Table 3, Figure 3A). Dominant                                            analysis shows recessive disease genes  <br /> disease genes evolving slowly, general,                                          represented highly conserved genes. possibil- recessive disease genes non-disease genes,                                          ity highly conserved proteins occurrence indicates subjected stronger selective                                       double mutations, observed recessive disease genes, <br /> straints. Interestingly, average value KA                                         associated lethality, case disease disease recessive non-disease genes                                           condition observed. assess mapped <br /> similar (0.00320 0.00318), distribution KA val-                                        genes study C. elegans gene orthologues <br /> ues sets clearly different (p-value K-S                                      previously classified Wild Type (WT), <br /> test = 2.66 &#195;&#8212; 10-10 Figure 3A). fact, observe                                         disease (D) Lethal (L) according RNAi data [17]. lower proportion disease recessive genes                                             Genes lethal disrupted considered low KA value (&lt;0.001) intermediate KA values                                           &quot;essential                                          genes&quot;. represented group genes (Figure 3). agreement observed distribution conser- <br />  <br /> Table 3: Molecular evolution disease non-disease genes. Average values KA, KS, KI KA/KS different sets genes. <br />  <br />                                                      KA                               KS                 KI                 KA/KS                       n <br />  <br />        Non disease                                 0.00318                          0.01400           0.01260                0.220                    10914 <br />          Disease                                   0.00295                          0.01644           0.01270                0.194                     886 <br />     Disease Dominant                               0.00260                          0.01790           0.01274                0.157                     367 <br />     Disease Recessive                              0.00320                          0.01540           0.01266                0.222                     519 <br />  <br />  <br />  <br />  <br />                                                                                                                                                       Page 4 11 <br />                                                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                                                  http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br />                                                      Average cs <br />             0.35               Disease Dominant      0.420 <br />                                                                                                                          Disease Recessive     0.295 <br />                                                                                                          0.35                                                                                           Average Ka <br />              0.3               Non Disease           0.385                                                0.3 <br />                                                                                                                                                                    Disease Dominant                     0.00260 <br />             0.25                                                                                         0.25                                                      Disease Recessive                    0.00320 <br />  <br />  <br />  <br />  <br />                                                                                         frequency <br />                                                                                                                                                                    Non Disease                          0.00318 <br /> frequency <br />  <br />  <br />  <br />  <br />              0.2 <br />                                                                                                           0.2 <br />  <br />  <br />  <br />                                                                                                          0.15 <br />             0.15 <br />  <br />  <br />                                                                                                           0.1 <br />              0.1 <br />  <br />  <br />                                                                                                          0.05 <br />  <br />             0.05 <br />  <br />                                                                                                               0 <br />                                                                                                                       0     0.001         0.002          0.003     0.004     0.005     0.006    0.007   0.008     0.009    &gt;0.009 <br />               0 <br />                    0.1   0.2     0.3      0.4      0.5    0.6     0.7   0.8   0.9   1 <br />  <br />  <br />                                  conservation score paralogues                                                                                                      Ka <br /> ease <br /> Figure  2 protein <br /> Conservation <br />      genes            level dominant recessive dis- <br />               paralogues                                                                                                                                                          Average Ks <br />                                                                                         B <br /> Conservation paralogues dominant reces-                                                         0.3 <br />  <br /> sive disease genes protein level. Distribution  <br /> servation score paralogues dominant recessive                                                 0.25                          Disease Dominant                                0.01790 <br /> disease genes versus rest genes.                                                                                               Disease Recessive                               0.01540 <br />                                                                                                           0.2 <br />                                                                                                                                                                                       0.01400 <br />                                                                                            frequency <br />                                                                                                                                       Non Disease <br />                                                                                                         0.15 <br />  <br /> Firstly determined essential genes significantly <br />                                                                                                           0.1 conserved WT genes (cs human-mouse: WT = <br /> 0.80, L = 0.84, p-value M-W test = 6.79 &#195;&#8212; 10-6; KA human-                                               0.05 <br />  <br /> chimpanzee orthologues: WT = 0.0019, L = 0.0013, p- <br /> value M-W test = 2.19 &#195;&#8212; 10-8). question                                                          0 <br />                                                                                                                       0    0.0025     0.005             0.0075     0.01     0.0125     0.015   0.0175   0.02     0.0225   &gt;0.0225 <br />  recessive disease genes represented essential genes. data set reces-                                                                                                           Ks <br /> sive disease genes lower proportion C. elegans <br /> essential genes (31%) set dominant disease                                  C <br /> genes (36%). addition, 47% dominant disease                                                  0.3 <br />                                                                                                                                                                                                                Average Ka/Ks <br /> orthologues phenotypic effect (WT)                                                                                                                                 Disease Dominant <br />                                                                                                        0.25                                                                                                        0.157 <br /> mutated, comparison 61% recessive disease ortho-                                                                                                                       Disease Recessive                  0.222 <br /> logues 58% non-disease gene orthologues.                                                         0.2 <br />                                                                                                                                                                                                                    0.220 <br />                                                                                           frequency <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 Non Disease <br />                                                                                                        0.15 <br /> Evolutionary conservation rates functional annotations <br /> Previously proportions dif-                                               0.1 <br />  <br />  <br /> ferent functional annotations dominant recessive <br />                                                                                                        0.05 <br /> disease genes [1,14,15]. Diseases caused mutations genes coding enzymes transport-                                                  0 <br />                                                                                                                   0       0.1       0.2           0.3        0.4      0.5       0.6      0.7     0.8    0.9       1        &gt;1 <br /> ers recessive, mutations transcription <br /> regulators, structural molecules, nucleic acid binding <br />                                                                                                                                                                       Ka/Ks <br /> genes signal transducers primarily dominant.  differences evolutionary                             Evolutionary <br />                                                                                         dominant <br />                                                                                         Figure 3 andrates                                                                                                          recessive disease <br />                                                                                                                       human <br />                                                                                                                           genes <br />                                                                                                                              andatchimpanzee <br />                                                                                                                                    DNA level genes rates dominant recessive disease genes                                  Evolutionary rates human chimpanzee dissimilarities different types                               genes dominant recessive disease genes functional genes distinct evolutionary                               DNA level.   Distribution KA disease genes versus <br /> tories sets genes. order rule                            rest genes. (B) Distribution KS disease genes <br /> possibility human disease gene classified accord-                              versus rest genes. (C) Distribution KA/KS ratio ing molecular function protein product                                 disease genes versus rest genes. <br /> determined Gene Ontology   &apos;slim&apos; terms [18], conservation score mouse KA/KS values human chimpanzee dominant reces-                                     tion values observed different functional clas- <br /> sive disease genes functional annotations                                sifications, observe consistently higher conservation assessed (Table 4). Despite variation conserva-                             scores lower KA/KS values (Table 4) set dom- <br />  <br />  <br />                                                                                                                                                                                                                Page 5 11 <br />                                                                                                                                                                           (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                       http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 4: Mean CS mouse, paralogues KA/KS values dominant recessive disease genes slim category <br />  <br />                                                                          CS mouse <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.778                     0.790                     0.818                        0.755 <br />   Transcription regulation          0.800                     0.860                     0.871                        0.824 <br />     Nucleic acid binding            0.795                     0.835                     0.851                        0.803 <br />          Catalysis                  0.809                     0.819                     0.832                        0.815 <br />     Signal transduction             0.751                     0.748                     0.776                        0.708 <br />     Structural molecule             0.828                     0.831                     0.838                        0.815 <br />         Transporter                 0.824                     0.848                     0.862                        0.841 <br />           Carrier                   0.838                     0.881                     0.884                        0.880 <br />  <br />                                                                        CS paralogues <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.374                     0.272                     0.300                        0.236 <br />   Transcription regulation          0.353                     0.422                     0.440                        0.359 <br />     Nucleic acid binding            0.358                     0.370                     0.431                        0.250 <br />          Catalysis                  0.381                     0.355                     0.430                        0.331 <br />     Signal transduction             0.383                     0.347                     0.406                        0.261 <br />     Structural molecule             0.449                     0.482                     0.595                        0.207 <br />         Transporter                 0.424                     0.400                     0.539                        0.331 <br />           Carrier                   0.425                     0.432                     0.571                        0.402 <br />  <br />                                                                       KA/KS chimpazee <br />  <br />                                      nD                     Disease              Disease dominant            Disease recessive <br />  <br />      Enzyme regulation              0.163                     0.233                     0.223                        0.251 <br />   Transcription regulation          0.170                     0.144                     0.109                        0.253 <br />     Nucleic acid binding            0.188                     0.164                     0.149                        0.190 <br />          Catalysis                  0.185                     0.179                     0.150                        0.189 <br />     Signal transduction             0.213                     0.173                     0.138                        0.238 <br />     Structural molecule             0.201                     0.183                     0.155                        0.250 <br />         Transporter                 0.176                     0.166                     0.082                        0.213 <br />           Carrier                   0.170                     0.197                     0.123                        0.213 <br />  <br />  <br />  <br /> inant disease genes compared recessive disease             reduction selective pressure decreasing severity genes. results confirm evolutionary differ-        disease (Table 5). pattern clearly evident ences observed dominant recessive disease             recessive disease genes, appears complicated <br /> genes different selective pressures acting          dominant disease genes.  disease genes sets genes history simply       modes inheritance result severe pheno- varying proportions functionally dissimilar             types  e. death reproductive age)  <br /> genes group.                                              served genes consequent disease does <br />                                                                   severely affect reproductive fitness individ- addition, paralogues dominant recessive            ual. <br /> disease genes similarly analysed, higher conserva- <br /> tion dominant disease gene paralogues evident            Gene structure autosomal dominant recessive functional group (Table 4).                                  disease genes <br />                                                                   Previously, genes involved Evolutionary conservation rates severity disease            hereditary diseases different gene structure proper- <br /> Analysis evolutionary constraints disease genes,        ties compared rest genes human genome indicated human-chimpanzee KA/KS values,               [2,19]. particular, disease genes longer coding <br /> categorised severity disease displays trend        sequences, exons alternative splicing [19]. <br />  <br />  <br />                                                                                                                        Page 6 11 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                         http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> Table 5: KA/KS mean values mode inheritance according severity disease:  Mild (death &gt;60 years), Moderate (death puberty 60), Severe (death puberty). <br />  <br />                                       n                       Disease               Disease dominant             Disease recessive <br />  <br />                                  99                       0.186                     0.133                         0.224 <br />           Mild                        36                       0.167                     0.114                         0.215 <br />         Moderate                      90                       0.182                     0.152                         0.209 <br />          Severe                       78                       0.168                     0.110                         0.186 <br />  <br />  <br />  specific sequence properties thought            inversely correlated evolutionary rate,  higher probability suffering disease-causing              older proteins evolve slowly proteins mutations genes longer com-               recent origin [20].   spite <br /> plex splicing patterns [19]. analysed structure           fact recessive disease genes higher evolu- <br /> genes involved autosomal dominant recessive dis-             tionary rates, better represented old genes <br /> eases elucidate differences groups.          dominant disease genes  Additional file 3]. <br />  confirmed previous results, finding                Overall, results presented mode group disease genes  average, higher number             inheritance gene important determinant exons (D = 13.53, nD = 9.77), higher number alter-          rate evolution. According nearly neutral theory native transcripts (D = 5.03, nD = 4.47), longer gene         evolution slightly deleterious mutations sequence (D = 62818 nD = 52066) coding                      fixed populations [21]. Recessive genes accumu- <br /> sequence (D = 657.2 nD = 509.6) compared                 late larger number mutations dominant <br /> complete set human genes. differences             genes, mutant &quot;hidden&quot; tested using Mann-Whitney Kolmogorov-                  selection heterozygous [22]. higher fixation rate <br /> Smirnov tests significant. similar analysis com-          slightly deleterious mutations recessive genes paring set genes involved autosomal recessive          result higher KA/KS ratios type gene [4]. <br /> dominant diseases shows significant differences protein gene length number alternative              Analyses conservation levels genes involved transcripts sets genes.                recessive diseases shown significant decrease fre- marginally statistically significant difference       quency high levels conservation (high conservation number exons (DD = 13.65, DR = 14.63; p-value            score (Figure 1) low KA/KS ratio (Figure 3C)). M-W test = 8.5 &#195;&#8212; 10-4).                                             plausible explanation effect genes high <br />                                                                     degree conservation enriched human essen- <br /> Discussion                                                          tial genes, involved recessive disease analysis reveals selective pressures human          double mutation lethal. autosomal disease genes involved dominant reces-             analysis RNAi phenotypes C. elegans human <br /> sive disorders differ significantly. Dominantly-acting dis-         orthologues groups genes analysed sup- <br /> ease genes appear conserved recessively-            ports  slightly higher proportion acting genes. addition, confirmed previous               lethal genes set dominant disease genes results showing disease genes generally evo-          higher proportion genes effect (WT) lutionarily conserved human genes [2,5,6].               recessive disease set.  considerable <br />                                                                     proportion genes essential C. elegans Previously proportions dif-           classified recessive disease genes human. ferent functional annotations dominant recessive             fact  set essential genes disease genes differ substantially [1,14,15].          C. elegans likely enriched human essential nature conservation pattern             genes, cases gene essen- <br /> determined solely function protein.           tial C. elegans humans. <br /> analysis conservation levels groups disease genes slim functional category               suggested essential genes evolve (Table 4) appear refute  categories,        slower rates non-essential genes [23]. Certainly, pre- <br /> dominant disease genes consistently conserved              vious works shown essential genes bacteria recessive disease genes. factors               [24] yeast [25] highly conserved, shown correlate protein evolutionary rate.         controversial results [26,27]. recent report shown age protein        level conservation human genes tends <br />  <br />  <br />                                                                                                                          Page 7 11 <br />                                                                                                    (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                    http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  higher corresponding orthologue C. ele-       disease gene gene duplication. Alternatively, gene <br /> gans essential lethal. supports     duplications dominant disease genes hypothesis sharp decrease recessive disease     recent similar events recessive disease genes. genes high conservation levels related       seen Table 4, difference conservation <br /> paucity human essential genes recessive disease       level paralogues determined protein function. <br /> set. <br />                                                                 study focused autosomal genes  possible greater degree conser-    affected recessively- dominantly-acting mutations <br /> vation exhibited dominant disease genes vs. recessive        lead disease phenotype. readily acknowledge <br /> disease genes effect higher likelihood fixa-    highlight relevant groups genes <br /> tion slightly deleterious non-synonymous mutations        X-linked disease genes imprinted genes recessive disease genes coupled repre-        [28,29].  cursory evidence X-linked disease <br /> sentation highly conserved genes dataset.       gene evolution included  Additional file 4]. <br />  addition, analysed conservation disease genes      Conclusion <br /> according severity disease (Table 5).         highlight significant differences disease <br /> analysis reveals conservation dominant         genes, terms protein DNA sequence conserva- recessive disease genes highest category       tion, paralogy essentiality, categorised disease results death onset puberty,     mode inheritance. e. reproductive fitness equal zero population <br /> genetics perspective. acknowledge partition      analysis reveals genes affected dominant dis- <br /> disease genes rudimentary groups simplifi-      ease mutations conserved recessive disease <br /> cation,  serves insight relation-   genes. attribute fact recessive mutations <br /> ship disease severity selective constraints.        remain hidden selection heterozygous,                                                                 allow recessive disease genes accumulate larger confirmed disease genes higher KS        number slightly deleterious mutations eventually <br /> values, dominant recessive genes,         fixed populations. <br /> rest human genes (Figure 3B, Table 3). Huang et al. <br /> reported similar results analysis human           number studies attempted predict disease rat genes [3], phenomenon discov-         genes computationally using features sequence <br /> ered analysis chimpanzee genome [6]. Fur-         length, paralogy, sequence conservation, range tissue <br /> thermore, analysis authors ascribe higher KS    expression, amino-acid composition splicing signals <br /> level disease genes abundant presence         [2,4,5,30]. studies treated dis- mutation-susceptible CpG dinucleotide disease            ease genes homogeneous dataset. suggest genes. report CpG dinucleotides              differences dominant recessive disease genes <br /> excluded analysis, difference KS rates          accounted future disease gene-prediction sets genes non-significant.          studies. <br />  tendency non-disease genes closer para-          Methods <br /> logues disease genes previously attributed        Data fact close paralogue able ameliorate    Genes involved hereditary disease catalogued effect loss function mutation gene vir-     Online database Mendelian Inheritance Man <br /> tue similar functionality [16]. recessive   (OMIM) [31]. retrieved list genes &apos;mor- <br /> disease genes, contain mainly loss function            bid map&apos; table OMIM database. Using NCBI <br /> mutations, deficit close paralogues        LocusLink [32] database Ensembl database [33], <br /> compensate loss function. contrary,           located corresponding gene sequence records. dominant disease genes, possessing closer          result list 1647 genes associated human dis- functionally similar paralogues,          ease. Ensembl protein-coding genes classi- <br /> reprieved disease state existence close par-   fied non-disease (nD). Disease genes classified <br /> alogues dominant-negative effects. addi-        according mode inheritance disease tion, haploinsufficient genes  e. dominant disease)       cause using text mining automatic extraction dosage-dependent, case retention            clinical synopsis section OMIM database manual <br /> paralogues advantageous [15].  dif-          curation reported [14]. Genes classified auto- <br /> ference observed groups disease genes        somal Disease Dominant (DD; n = 498) mutation selective pressure recessive     single allele reported causative genetic <br />  <br />  <br />                                                                                                                     Page 8 11 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                    http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> disorder, autosomal Disease Recessive (DR; n = 662)          rates KA, KS KI data genes taken mutations alleles required disease phe-     Chimpanzee Sequencing Analysis Consortium sup- <br /> notype, (e.g. X-linked diseases) described        plementary data [6]. KA/KS values gene datasets previously. Genes type mutations (dom-         calculated summing non-synonymous inant recessive) used       synonymous sites dataset reported study.                                                          [6]. Mann-Whitney Kolmogorov-Smirnov tests                                                                 conducted using KA, KS KI data individual <br /> Calcultation conservation score                              gene, KA/KS excluding genes KS = 0. <br /> Conservation score (cs) estimation divergence occurred pair proteins evolu-      Essentiality data C. elegans <br /> tion, independent length proteins [2].     C. elegans genes classified depending pheno- score provides estimation non-syn-      type RNAi experiments groups, Wild Type <br /> onymous nucleotide substitution rate pair          (WT), Lethal (L) Disease (D). RNAi phenotypes proteins, takes account conservativeness      extracted Wormbase Biomart [17]. mapped amino acid substitutions. Conservation scores cal-      human orthologue C. elegans gene using ortho- <br /> culated using WUBLASTP (version 2.0) [34],             logues pairwise data Ensembl Compara [39]. based public domain NCBI BLAST version 1.4 [35].         number human genes corresponding WT, L D <br /> Hits E-values &gt; 1 &#195;&#8212; 10-10 discarded. Smith-           phenotypes RNAi experiments C. elegans Waterman [36] alignment performed pairs              counted groups analysed (DD, DR gave significant BLAST hit. value cs cal-     nD). <br /> culated datasets DD, DR nD using relevant human protein-coding sequence Ensembl ver-          Evolutionary rates functional classification <br /> sion 34 WUBLASTP score closest homologue          dominant recessive disease genes classified Mus musculus divided WUBLASTP score            according molecular function protein protein itself, reported [37]. Sim-    determined Gene Ontology &quot;slim&quot; terms [18]. Pro- <br /> ilarly, conservation score paralogues calculated     tein conservation scores mouse paralogues WUBLASTP score closest paralogue divided          calculated described previously. KA/KS values cal- WUBLASTP score protein itself.            culated gene dataset summing non- <br />                                                                 synonymous synonymous sites dataset. assess statistical significance results com- <br /> puted non-parametric statistical tests, Kol-            Evolutionary rates severity disease <br /> mogorov-Smirnov (K-S) test Mann-Whitney (M-             Data severity disease genes obtained W) test, using R statistics package [38]. test   Jimenez-Sanchez et al. [1]. total 303 genes set significant deviation random expectation       classified following groups according number paralogues dominant recessive           disease severity:  Mild (death &gt;60 years), Moderate <br /> disease genes (X) high levels conservation (cs &gt; 0.8)     (death puberty 60), Severe (death randomly generated 10,000 datasets proteins            puberty). KA/KS values group genes calcu- <br /> identical sample size dominant recessive      lated explained  <br /> disease gene datasets counted set number genes paralogues cs higher threshold      Gene structure analysis <br /> (0.8). scores {Zx = (X - &#206;&#188;x)/&#207;&#402;x} p-values {px = &#206;&#163;(nx &#226;&#8240;&#165;    Gene length, coding sequence length number X)/N} calculated  mean = &#206;&#188;x, standard devia-         introns obtained Ensembl database [33]. tion = &#207;&#402;x, nx number sets score        number alternative transcripts obtained Alter- <br /> threshold X N total number datasets)                 native Splicing Database [40]. Differences                                                                 sequences properties different groups (D, nD, <br /> Calculation nucleotide substitution rates DNA level       DD DR) assessed using Kolmogorov-Smir- set 13,454 human-chimpanzee orthologues used             nov test. gene evolution analysis Chimpanzee Sequenc- <br /> ing Analysis Consortium [6] translated Ent-        Authors&apos; contributions <br /> rez gene entries Ensembl gene IDs [33]. dataset         SJF carried analysis statistical tests, merged set dominant recessive OMIM          participated design study drafted disease genes compiled earlier. resulted non-         manuscript. MMA performed gene age analysis redundant dataset 886 disease genes, consisting 367       revised manuscript critically. NLB conceived study, <br /> dominant disease (DD) 519 recessive disease (DR)            participated design coordination, helped genes, 10,914 &quot;non-disease&quot; (nD) genes. Substitution <br />  <br />  <br />                                                                                                                    Page 9 11 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                         http://www.biomedcentral.com/1471-2164/7/165 <br />  <br />  <br />  <br /> draft manuscript. authors read approved                          8.    Mendel: Experiments Plant Hybridization. Cambridge, MA., <br />                                                                                        Harvard University Press, 1965; 1866. <br /> final manuscript.                                                                9.    Porteous JW: Dominance- years                                                                                        Mendel&apos;s paper. J Theor Biol 1996, 182:223-232. <br /> Additional material                                                              10.   Fisher RA: possible modification response                                                                                        wild type recurrent mutations. Amer Nat 1928, 62:115-126. <br />                                                                                  11.   Wright S: Fisher&apos;s theory dominance. Amer Nat 1929, <br />                                                                                        63:274-279. <br />      Additional file 1                                                           12.   Kacser H, Burns JA: molecular basis dominance. Genetics <br />      contains supplementary figure 1.                                               1981, 97:639-666. <br />      Click file                                                         13.   Phadnis N, Fry JD: Widespread correlations domi- <br />                                                                                        nance homozygous effects mutations: implications      [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                        theories dominance. Genetics 2005, 171:385-392. <br />      2164-7-165-S1.pdf]                                                          14.   Lopez-Bigas N, Blencowe BJ, Ouzounis CA: Highly consistent pat- <br />                                                                                        terns inherited human diseases molecular level. <br />      Additional file 2                                                                 Bioinformatics 2006, 22:269-277. <br />                                                                                  15.   Kondrashov FA, Koonin EV: common framework  <br />      contains supplementary tables 1 &amp; 2.                                           standing origin genetic dominance evolutionary <br />      Click file                                                               fates gene duplications. Trends Genet 2004, 20:287-290. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                   16.   Strachan T, Read AP: Molecular Pathology. Human Molecular <br />      2164-7-165-S2.pdf]                                                                Genetics 3 ed Oxford, BIOS Scientific; 2004. <br />                                                                                  17.   Wormbase [http://www.wormbase.org/biomart/martview] <br />                                                                                  18.   Harris MA, Clark J, Ireland  Lomax J, Ashburner M, Foulger R, Eil- <br />      Additional file 3                                                                 beck K, Lewis S, Marshall B, Mungall C, Richter J, Rubin GM, Blake JA, <br />      contains supplementary figure 2.                                               Bult C, Dolan M, Drabkin H, Eppig JT, Hill DP, Ni L, Ringwald M, <br />      Click file                                                               Balakrishnan R, Cherry JM, Christie KR, Costanzo MC, Dwight SS, <br />      [http://www.biomedcentral.com/content/supplementary/1471-                         Engel S, Fisk DG, Hirschman JE, Hong EL, Nash RS, Sethuraman  <br />      2164-7-165-S3.pdf]                                                                Theesfeld CL, Botstein D, Dolinski K, Feierbach B, Berardini T, Mun- <br />                                                                                        dodi S, Rhee SY, Apweiler R, Barrell D, Camon E, Dimmer E, Lee V, <br />                                                                                        Chisholm R, Gaudet P, Kibbe W, Kishore R, Schwarz EM, Sternberg <br />      Additional file 4                                                                 P, Gwinn M, Hannick L, Wortman J, Berriman M, Wood V, la Cruz <br />      contains supplementary table 3.                                                N, Tonellato P, Jaiswal P, Seigfried T, White R: Gene Ontology <br />      Click file                                                                 database informatics resource. Nucleic Acids Res <br />                                                                                        2004, 32:D258-61. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                   19.   Lopez-Bigas N, Audit B, Ouzounis C, Parra G, Guigo R: splicing <br />      2164-7-165-S4.pdf]                                                                mutations frequent cause hereditary disease? <br />                                                                                        FEBS Lett 2005, 579:1900-1903. <br />                                                                                  20.   Alba MM, Castresana J: Inverse relationship evolution- <br />                                                                                        ary rate age mammalian genes. Mol Biol Evol 2005, <br />                                                                                        22:598-606. <br /> Acknowledgements                                                                 21.   Ohta T: Slightly deleterious mutant substitutions evolu- <br />                                                                                        tion. Nature 1973, 246:96-98. thank Roderic Guig&#195;&#179;, Desmond G. Higgins Arcadi Navarro val- <br />                                                                                  22.   Charlesworth B, Coyne JA, Barton NH: relative rates evo- <br /> uable help, comments discussion. thank Ensembl-Compara                 lution sex chromosomes autosomes. Amer Nat 1987, <br /> group pairwise similarity data. N. L.-B. supported long-term         130:113-146. <br /> post-doctoral fellowship Human Frontiers Science Program. S.J.F.        23.   Wilson AC, Carlson SS, White TJ: Biochemical evolution. Annu <br /> acknowledges support EMBO Short Term Fellowship ASTF 83-2005                      Rev Biochem 1977, 46:573-639. <br />                                                                                  24.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes Instituto Nacional Bioinform&#195;&#161;tica (INB). M.M  acknowledges                evolutionarily conserved nonessential genes <br /> support Fundaci&#195;&#179; ICREA Spanish Ministerio Edu-                    bacteria. Genome Res 2002, 12:962-968. <br /> caci&#195;&#179;n y Ciencia grant number BIO2002-04426-C02-01.                              25.   Hirsh AE, Fraser HB: Protein dispensability rate evolu- <br />                                                                                        tion. Nature 2001, 411:1046-1049. <br />                                                                                  26.   Hurst LD, Smith NG: essential genes evolve slowly? Curr Biol <br /> References                                                                             1999, 9:747-750. <br /> 1.      Jimenez-Sanchez G, Childs B, Valle D: Human disease genes.               27.   Yang J, Gu  Li WH: Rate protein evolution versus fitness <br />         Nature 2001, 409:853-855.                                                      effect gene deletion. Mol Biol Evol 2003, 20:772-774. <br /> 2.      Lopez-Bigas N, Ouzounis CA: Genome-wide identification                28.   Charlesworth B: effect life-history mode inherit- <br />         genes likely involved human genetic disease. Nucleic                  ance neutral genetic variability. Genet Res 2001, 77:153-166. <br />         Acids Res 2004, 32:3108-3114.                                            29.   Spencer HG: Population genetics evolution genomic <br /> 3.      Huang H, Winter EE, Wang H, Weinstock KG, Xing H, Goodstadt L,                 imprinting. Annu Rev Genet 2000, 34:457-477. <br />         Stenson PD, Cooper DN, Smith D, Alba MM, Ponting CP, Fechtel K:          30.   Adie EA, Adams RR, Evans KL, Porteous DJ, Pickard BS: Speeding <br />         Evolutionary conservation selection human disease                       disease gene discovery sequence based candidate prioriti- <br />         gene orthologs rat mouse genomes. Genome Biol                       zation. BMC Bioinformatics 2005, 6:55. <br />         2004, 5:R47.                                                             31.   Hamosh  Scott AF, Amberger J, Bocchini C, Valle D, McKusick VA: <br /> 4.      Smith NG, Eyre-Walker  Human disease genes: patterns                     Online Mendelian Inheritance Man (OMIM), knowledge- <br />         predictions. Gene 2003, 318:169-175.                                           base human genes genetic disorders. Nucleic Acids Res <br /> 5.      Kondrashov FA, Ogurtsov AY, Kondrashov  Bioinformatical                     2002, 30:52-55. <br />         assay human gene morbidity. Nucleic Acids Res 2004,                   32.   Pruitt KD, Maglott DR: RefSeq LocusLink: NCBI gene-cen- <br />         32:1731-1737.                                                                  tered resources. Nucleic Acids Res 2001, 29:137-140. <br /> 6.      Consortium CS: Initial sequence chimpanzee genome                 33.   Hubbard T, Barker D, Birney E, Cameron G, Chen Y, Clark L, Cox T, <br />         comparison human genome. Nature 2005,                             Cuff J, Curwen V, T, Durbin R, Eyras E, Gilbert J, Hammond M, <br />         437:69-87.                                                                     Huminiecki L, Kasprzyk  Lehvaslaiho H, Lijnzaad P, Melsopp C, <br /> 7.      Tu  Wang L, Xu M, Zhou X, Chen T, Sun F: understand-                 Mongin E, Pettett R, Pocock M, Potter S, Rust  Schmidt E, Searle S, <br />         ing human disease genes comparing housekeeping                         Slater G, Smith J, Spooner W, Stabenau  Stalker J, Stupka E, Ureta- <br />         genes genes. BMC Genomics 2006, 7:31.                                Vidal  Vastrik  Clamp M: Ensembl genome database <br />                                                                                        project. Nucleic Acids Res 2002, 30:38-41. <br />  <br />  <br />  <br />                                                                                                                                            Page 10 11 <br />                                                                                                                       (page number citation purposes) <br />  BMC Genomics 2006, 7:165                                                                   http://www.biomedcentral.com/1471-2 <br /> </body></html>