<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNA (1995), 1:437-448. Cambridge University Press. Printed USA. <br /> Copyright &#194;&#169; 1995 RNA Society. <br />  <br />  <br />  dual-specificity pseudouridine synthase: Escherichia coli synthase purified cloned basis specificity *746 23S RNA specific *32 tRNAPhe <br />  <br />  <br /> JAN WRZESINSKI,1 KELVIN NURSE,1&apos; 3 ANDREY BAKIN,1&apos;3 BYRON G. LANE,2 JAMES OFENGAND1 <br /> 1   Roche Institute Molecular Biology, Roche Research Center, Nutley, New Jersey 071 1 0, USA <br /> 2   Department Biochemistry, University Toronto, Toronto, Ontario M5S 1 A8, Canada <br />  <br />  <br />  <br /> ABSTRACT Escherichia coli pseudouridine (*) synthase, forms *746 E. coli 23S ribosomal RNA *32 tRNAPhe, isolated cloned. enzyme contains 219 amino acids calculated MW 24,432 Da. <span id='am-56' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-57' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-59' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>Amino acid</span> sequence comparison * synthases cloned <span id='am-24' about='oboInOwl:date' typeof='owl:Thing'>date</span>, tRNA 16S RNA, did reveal common sequence motifs, despite catalysis com- <br /> mon reaction. gene cloned  6 leader affinity purification. overexpression, enzyme remained soluble cell cytoplasm purified homogeneity Ni2+- <br /> containing resin. enzyme reacted length 23S RNA fragment residues 1-847, form- <br /> ing 1 mol *Imol RNA position 746, normal site *. enzyme dependence Mg2+. yield obtained 1 mM EDTA 10 mM Mg2+, rate faster EDTA Mg2+. length <br /> 1 6S RNA fragments 1-526 1-678, tRNAVaI transcripts, modified EDTA Mg2+. <br /> tRNAPhe transcripts,  modified yield mol tImol transcript rate EDTA like 23S RNA. Sequencing showed * position 32, normal site * tRNA. 23S <br /> rRNA *746 tRNA *32 occur single-stranded segments sequence, *UGAAAA, closed stem.  synthase require recognition short stretch primary sequence 3&apos; site pseudouridylation. example dual-specificity modifying enzyme RNA,  specific single site RNA, equally site-specific second class RNA. essen- <br /> tiality * residues assessed disruption synthase gene. <br /> Keywords: <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> RNA sequencing; large subunit rRNA; modified nucleosides; overexpression; polyhisti- <br /> dine tag; RNA modifying enzyme <br />  <br /> INTRODUCTION                                                         procedures (Maden, 1990). development                                                                       rapid primed reverse transcriptase approach (Bakin number variety modified nu-                       <span id='am-25' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; Ofengand, 1993), sequencing residues lon- <br /> cleosides ribosomal RNA (rRNA) considerable                     ger problem. Using approach, deter- <br /> (Maden, 1990; Limbach et al., 1994), role rRNA               mined location single Escherichia coli 16S <br /> function largely unknown.                   RNA (Bakin et al., 1994a), E. coli 23S RNA lack site-specific localization modified nu-            (Bakin &amp; Ofengand, 1993; Bakin et al., 1994b), cleosides respective RNA sequences.                  30 yeast 28S RNA (Bakin et al., 1994b), 13 particularly true pseudouridine (5-i3-D-ribofurano-               yeast 18S RNA (Bakin &amp; Ofengand, 1995),  <br /> syluracil, ,  recently, lo-                residues large subunit (LSU) <br /> cated labor-intensive conventional sequencing                      RNAs Halobacter halobium, Bacillus subtilis, Drosphila <br />                                                                       melanogaster, Mus musculus, Homo sapiens (Ofen- <br />                                                                       gand et al., 1995). <br />    Reprint requests  James Ofengand, Roche Institute Molec-        main conclusion analyses &apos; <br /> ular Biology, Roche Research Center, Nutley, New Jersey 07110, <br /> USA; e-mail: ofenganj@rnisdO.dnet.roche.com.                          residues    clustered discrete areas LSU <br />    3These authors contributed equally work.                   RNAs, contain 9 residues, E. coli, <br />                                                                   437 <br />  438                                                                                                          J. Wrzesinski et al. 57 residues, M. musculus,      synthases. Screening cell extracts areas  near  peptidyl trans-      [5-3H]<span id='am-79' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span>-labeled 23S RNA transcripts showed ferase center (PTC).  clustering paralleled      3H release activity (&gt;70%) S200- clustering methylated residues         streptomycin supernatant fraction eluted data available (Bakin &amp; Ofengand, 1993; Brima-            sharp peak DEAE column. activity combe et al., 1993; Bakin et al., 1994b). cluster-       separated distinct peaks MonoS <br /> ing indicates nonrandom selection U residues          column. Preliminary studies showed peak converted implies existence                   &apos;,                                          reacted enzyme-dependent manner yield ap- <br /> specific functional purpose localization.             proximately 1 mol 3H released/mol RNA,    previously postulated functional role        activity released 3H affected <br />    PTC (Lane et al., 1992), subsequent          changes Mg2+ salt concentrations reports discussed alternate explanations        assay. peak purified based available data (Bakin &amp; Ofengand, 1993;          MonoQ column (Fig. 2). stage, polypeptide <br /> Bakin et al., 1994b; Lane et al., 1995). believe      ca. 30 kDa identified staining intensity useful way approach question func-         paralleled enzymatic activity. specific activity <br /> tion choose organism relatively res-        pooled fractions 40-43 1.6 x 106 units/mg pro- <br /> idues, E. coli, attempt block       tein assayed 10 mM Mg2+ RNA (1-847). <br /> formation,  inactivation appropri- <br /> ate synthase. approach predicated  <br /> sumption residue require            Site specificity enzyme specific synthase, far         Unit stoichiometry (0.9-1.3) observed true rRNA synthases known (Wrzesinski             assays 10 mM EDTA, 2 mM Mg2+, et al., 1995; work), synthase form      10 mM Mg2+ (Table 1). values mea- <br />    adjacent nearby sites,        sured plateau reaction. Table 1 shows truA (hisT) synthase forms positions         stoichiometry lack depen- <br /> 38, 39, 40 anticodon stem loop tRNA         dence Mg2+ observed fragment (Kammen et al., 1988).                                        23S RNA transcript residues 1 847 used. <br />     embarked program iden-          results purified synthase reacted <br /> tify clone 23S RNA synthases E. coli, char-         single site 23S RNA molecule, acterize site action properties,       site remained exposed presence 10 mM carry gene disruption analysis vivo      Mg2+, Mg2+ required enzyme, effects loss specific residues. report,     lastly site question 847 efforts, identification   residues. Sequence analysis (Bakin &amp; Ofengand, enzyme responsible formation I746              1993) enzyme-treated 23S RNA RNA (1-847) <br /> (Fig. 1). Unexpectedly, purified recombinant syn-         showed clearly synthase formed *I746 thase formed I32 tRNAPhe. ex-       cases (Fig. 3). method, RNA reacted <br /> ample dual-specificity modifying enzyme.                 1-cyclohexyl-3-(2-morpholinoethyl)carbodiimide <br />                                                               metho-p-toluenesulfonate (CMC), followed alkaline <br /> RESULTS                                                       hydrolysis remove CMC G, U,                                                               U-derived bases  Subsequent reverse tran- <br /> Purification 23S RNA *746 synthase                     scription results stop base 3&apos; residue <br />                                                               attachment CMC group N-3  <br /> Previous work identified location 8 resi- <br /> dues plus modified E. coli 23S RNA (Bakin &amp; <br /> Ofengand, 1993). enzymes             TABLE 1. Effect Mg2+ EDTA yield 3H release known, initial experiments designed            length 23S RNA 1-847 fragment. <br /> tect synthase activity capable forming      residues. simplicity, chose      Substrate         Conditions          Mol   3H/mol RNA&apos;        SDb <br /> initial substrate free 23S RNA vitro (Weitzman        23S RNA         10 mM Mg2+                  1.27 (10)          0.29 <br /> et al., 1990) [5-3H]<span id='am-74' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-75' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span> order use 3H        (1-2,904)       2 mM Mg2+                   1.17 (19)          0.07 <br /> lease assay Cortese et al. (1974) measure                          10 mM EDTA                  0.93 (4)           0.03 <br /> activity. assay completely specific  <br />                                                     &apos;,        23S RNA         10 mM Mg2+                  1.14 (3)           0.09 <br /> cause release 3H 5-position <span id='am-68' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span>uridine</span>        (1-847)         2 mM Mg2+                   1.14 (8)           0.11 <br /> ring characteristic m5U synthesis pos-                       1 mM EDTA                   1.14 (3)           0.09 <br /> sibly formation 5-substituted pyrim-        Blank subtracted. Value parentheses number <br /> idines. practice, conditions,      determinations. activity detected eventually shown              b Standard deviation. <br />  Cloning dual-specificity E. coli synthase                                                                                                                                                                          439 <br />  <br />                                             U <br />                                                     LA                  AISCAC                                     CA <br />                                                            AA <br />                                                                                                                        AUCCoGAAA                    CA%                    u o <br />                       lIIII III                         II   I                                                        I I I IIle            II1 *1 <br />  <br />                       MUC nMYnAU                               OSAM0,t <br />                  SA <br />                              I4:y S <br />  <br />                                                                                                     _cA <br />                                                     -U <br />                                                                                                                               %e,AC6G                   ?*s0UG               &apos;PA <br />  <br />                                                                                    C                                        C.AG <br />                                    <br />                                        CCC_ <br />                                          U                                                                                                                      u~~~~~au <br />                                                                                                                                                                          %                         <br />                                                                                                                                                                                  -U        <br />  <br />  <br />  <br />                                                                                             6-C <br />                                                 r-c <br />  <br />  <br />  <br />  <br />                                                 U *   5- <br />                                                                                                               U <br />                                                                                                               -ue <br />                                                                                                                         U                                                         C                                                                                                                                                                                   o   6-c <br />                                                                                                                   6c-CA                                                          .C            <br />                                                                                                                                                                                             -    AP e                            C                               C <br />                               O                                              LOGO4       IBU C     666                    UU6SMUMUACU                         A6S   6Bis <br />                                                                             III                                                                                               c <br />                               U                                                                                                                           UC                       <br />                                                                                      &quot;%AG    AAU                                                                     AAAAA6 <br />                                                                                                                 G-Ciew                                     <br />                                             8v}AU&#194;&#162;                                                                     ^o <br />                                                                                                                   0-6 aa <br />                                                                                II   1-1411                                   CA <br />                                                                              MUGG6CCCMSMCCU                                                                                                                      AA%        AU    UPTPGA_UA                                                                                                                                                 f <br />                                                                                                               AA=.U%&apos; <br />                                                                                                                                                                                                                                                       U <br />                                                                                                  AA               6-CIUMA <br />                                                                                                                          BGUA      0A <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              *   C <br />  <br />                                                                                                   O 1: <br />  <br />  <br />  <br />  <br />                                                                                                                                                             vv &apos;Pi <br />                                                                                                                                                         CAA a9&apos;C   4r                                                GA <br />  <br />                                                                                                                                                                   &amp;; <br />                                                                                                                                                 - <br />                                                                                                                                                                                                                        <br />  <br />  <br />  <br />  <br />                                                                                                                                                 | <br />                                                                                                                                                 ^                     o o&amp;                ~~~~~UAA =AUVUU&quot;^ <br />                                                                                                                                                                                     UMGAA  <br />                                                                                                                                                                                                         B        CU <br />  <br />                                                                                                                                                                            ~GAGC       CZA GMGM&amp;GMSMBM   AG <br />                                                                                                                                                                                                   cA             ~~~C-4 <br />  <br />                                                                                                                              u                                                                                       u~~~~~~~~~~ <br />  <br />                                                                                                                                                                                                                  C        U <br />  <br />                                                                                                                                                                                                                      CA <br />  <br />  <br />  <br />  <br />        FIGURE 1. Location *746 E. coli 23S RNA. 5&apos;-half 23S RNA secondary structure (Gutell, 1993) shown <br />        segment residues 696 766 expanded form. mU m5U747, mG m1G745. Arrow marks position <br />        *746 (Branlant et al., 1981; Bakin &amp; Ofengand, 1993). <br />                                                                     _*:57t;fro.8R&lt;RSd_vh-_sL: 4z4l^ea!wpX&apos;&#194;&#162;sSj?&quot;@*myeM4125fv.lsw ! <br />                                                                    -    M. eS <br /> 440                                                                                                                                                                  J. Wrzesinski et al. <br />       12                                                               1w847                                                                 1w847 23S <br />                                                                        - - &quot;                                                                 + + + + + + enzYme <br />       10                                                               - ++ACUG                                                              - + + - + + CMt <br />                                                                        424 <br />                                                                        12 ..          <br />                                                                                       *s ... <br />                                                                                                                                              4 2 4 4 2 4 OH- <br />                                                                         jt:  r- <br />  <br />                                                                         _Pldfl <br />                                                                              &gt;S <br />  <br />  <br />                                                                         _             m.Ss. <br />                                                                             4        ;&apos; <br />                                                                                   -d&apos;S&apos; <br />                                                                             ?? &apos;     F&apos;rSE&apos;g? +&apos; <br />                                                                             YM       satC, &apos; ?. ; <br />                                                                    t23                iNi-?:^ fi <br />  <br />                                                                                                ,,.&apos;.4,tz <br />                                                                         ..&gt;..,b,,s ws.&lt;6. <br />                                                                                                      #,X;s,&amp; &apos; <br /> 0                                                                                ; ;&apos;w&apos;                                                                      *.&apos;S S?t,;; ,&quot; &quot;j.Y?S&#194;&#171; <br />                                                                      .$. .Ni-u;&gt;:,ss <br /> ._0 <br /> 02 <br />                                                                    .&apos;..,.X,-t&gt;.&apos;-4 L; <br />                                                                     , t iu ii r .. <br />                                                                    j ||                       tB <br />                                                                    s         | |                     <br />                                                                    ] ll <br />                                                                     | s                    g <br />                                                                    ti 1?3    P |       g     R <br />                                                                    3se <br />                                                                    ?@x <br />                                                                        fi 2,C <br />                                                                    X&lt;r4zN <br />                                                                    i3M.  ^ <br />                                                                    &gt; Ws_ <br />  <br />  <br />  <br />                                                                     Se.F:]YIS <br />                                                                     Xl&amp;ss <br />                                                                    .Aft-M <br />                                                                    .&lt;w}s-y&apos;.w <br />                                                                    flXLS <br />                                                                    *e f5,S2@jZfrf <br />                                                                                 w <br />                                                                    ;&quot;_ <br />  <br />                                                                    ?_ <br />  <br />  <br />                                                                    - <br />                                                                    - <br />                                                                    - <br />  <br />        36      38       40       42       44      46       48         !_ <br />                    Fraction              number                    :9tE <br />                                                                    w t <br />                                                                    &apos;3 <br />                                                                    &apos;@_ <br />                                                                     _a <br />                                                                    .FC.<span id='am-10' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span>gaS</span>;,&#194;&#167;,wis:a1:;!W <br />                                                                    Mf*,ffis                                                                    &apos;peg,7, <br />                                                                    :iY!P$:# <br />                                                                    :_ <br />                                                                    gE <br />                                                                    __Sa <br />   45.                                                              _ <br />                                                                    S <br />                                                                    ._4ti <br />                                                                    _SME <br />                                                                                                    sii <br />  <br />  <br />  <br />  <br />                                                                    4Zffl <br />                                                                    |_: <br />                                                                    StSS..E. &lt;                                                                    :nSS SCNS: <br />                                                                    b5xpRwD,^.er <br />                                                                     4,s.;isn..^&gt;; <br />                                                                    .\&apos;, 74B.St <br />                                                                     *{Jitit ti} -&gt; <br />                                                                             St22&apos;; <br />                                                                      t. ,[.rt,,2Se &apos; <br />  <br />  <br />  <br />   31                                                               aSH*^s&apos;Aa <br />                                                                    h..S_ <br />                                                                    .?J,?DA <br />                                                                    5&apos;5ffi..4 <br />  <br />                                                     _              _                                                                                                          ST ^ ] / <br />                                                                    &quot;X                                                                                                     .    .   , _, . <br />                                                                    _ <br />                                                                    w?.^ <br />                                                                    &apos;Xs;F <br />                                                                    &apos;&quot;SY,sCL;0;&#194;&#162;;&apos;&apos;SF: <br />                                                                    sF^.f&gt;eXQwg <br />  21                                                                , 82sft -&apos; %g j <br />                                                                        *@X. ;|? &apos;t &apos;s +. 9. <br />                                                                        &apos;&gt;4sSX.    Nslitg <br />                                                                      *&apos;&apos;..|s t.;hEi; APe-g <br />                                                                      ..:..}w.&apos;.&apos; ;&apos;T4E <br />                                                                    *&gt;- &apos;SU&apos;     3x- <br />                                                                     F. t-:.&gt; .&lt;.; ?Ss <br />  <br />             St 39 40 41 42 4344 4                                  *4e. <br />                                                                    *.S-stt&#194;&#163;t!t &#194;&#162;t jti <br />                                                                    f, ,.?, ;f <br />                                                                    y.,f, <br />                                                                    %itZs 4, s^&apos;tF .,5 <br />                                                                              # S6.sS <br />  <br />                                                                    w5;NeK&apos;svf *d*f.&apos;S.@: <br />                                                                    srf+.et:&amp;. ;w^ii <br />                                                                                                          __ <br />                                                                                                                                    l   _ <br />                                                                                                                                                                ,2, <br />  <br />  <br />  <br />  <br />                                                                    ^+;^=    &gt;* W .. <br />                                                                                      .g&lt;&gt;fi3 <br /> FIGURE 2. Final step purification 23S RNA *746 synthase   lSvv:jt.4w, &apos;t.;X <br />                                                                    ; WCs. w^-|h.p,%t <br /> MonoQ. Upper panel, chromatography enzyme FPLC         lS?H;aRt&gt;2-slZi}; <br />                                                                    .^Wr.      ..m.s <br />                                                                    sureu* .1 h:, <br />                                                                    ;&#194;&#167;s2t&quot;=.w.u <br />                                                                                  C&gt;2.} <br />                                                                                                    ,., <br /> MonoQ column described Materials meth-      :z2;tiO9waw-s <br />                                                                         &lt;vK , ^?9, <br />                                                                     ?N &apos;,F: .:4 <br /> ods. Lower panel, column fractions indicated electropho-            &gt;Kiise.Ssstl. <br /> resed SDS gels stained Coomassie Blue. St, MW          ,,.s; <br />                                                                                   ;M s g <br />                                                                             &apos;.;&apos; &quot;2, ,fl, <br />                                                                            .t44 standards values indicated.                                 ?, ,.l <br />  <br />  <br />                                                                      s-X?g;51t <br />                                                                                     ,.; _ <br />                                                                                             ,-,s,j,ef <br />                                                                                             r.ops I_ <br />                                                                                                                               |_ <br />                                                                                                                                                          ,,w,&amp;s,&gt; <br />  <br />                                                                    sXRf fJ;&apos; ig&apos;Bi ;4.             lil_ <br />                                                                    $&apos;G5;alEj&#194;&#163;=ffids <br />                                                                        %wcZi _- <br />                                                                    :=v9a&apos;x&apos;5S <br />                                                                    .^fXSaB,+-e.-e .7x7set <br />                                                                    tE <br />                                                                    .ESi&apos;.tA                                                                                    _E blocks base-pairing. residues                  5 te**s441- &apos;g; ..r <br />  <br />  <br />                                                                    W*:-sE \ rgVtf,]&apos;,., <br />                                                                                   9.,. tC _-_^S <br />                                                                                            ; ,. <br />  <br />  <br /> 647-782 shown figure, residues 437-787                                S <br />                                                                    .5 BS ZA.,D,. - <br />                                                                             &apos;f&apos;v&apos; <br />                                                                                  2s. <br />                                                                                                w <br />                                                                                                .s ,J ew,-- <br />                                                                                              &apos;-, <br />                                                                                            s &apos;.&apos;. &apos; ^,.R <br />                                                                                                           &lt;wPxpA tfn = . -                 ! 9   * .e. <br />  <br /> (41%) screened experiments.                  .&#194;&#167;:.. .; &apos;,4S <br />                                                                    ;&#194;&#163;.                                   .44&gt; -erv .+ ,,,P__fj ,g,jes <br /> Considering unit stoichiometry release              &quot;f 2&quot;t&apos;;                                   o&apos;&apos;   &gt;g$|W Ef ;&apos; <br />  <br />  <br />  <br />  strong bands observed Figure 3, likely synthase reacted U746. 4746              FIGURE 3. Reverse transcription analysis site + formation. &apos; naturally residues 1-847 (Bakin &amp;                  vitro transcript 23S RNA 1-847 fragment                                                                    incubated (+) (-) enzyme treated w*h (+) Ofengand, 1993).                                                   (-) CMC followed 2 4 h incubation pH 10.4 (OH). <br />                                                                    methodology according Bakin Ofengand (1993).  <br />                                                                    C, U, G, sequencing lanes using vitro transcript. Arrow shows <br /> Identification gene                                         position t746. <br /> N-terminal amino acid sequencing gel-purified <br /> protein band shown Figure 2 yielded following               assignment  unpromising <br /> sequence: emxnYnPpQxpxxv, upper case                     data set containing reliable assignments, letters denote reliable assignments, lower case letters            possible select GenBank <br /> denote questionable assignments, x indicates              database putative open reading frame (ORF)  Cloning dual-specificity E. coli synthase                                                                                       441 <br /> TABLE 2. 1 synthase activity induced cell extracts              contained correct-size insert. Plasmids ORF insert.                                                     clones used transform BL21/DE3 <br />                            Units enzyme                                 cells. induction, S15 extracts cells                              mL extract                                  assayed 3H release activity. 40- 50-fold stim- <br /> Source                         (x 10-3                       Ratiob       ulation obtained dependent pres- <br /> pET-15b                            5.5                            1         ence plasmid containing insert (Table 2). Clone 2                          235                            43          figure probably minimum value Clone 4                          263                            48          3H-releasing activities contribute pET <br />                                                                             control value. shown Figure 4, large increase <br />         S15 cell extract prepared described Materials        activity correlated production large <br /> methods induced cells grown 37 &#194;&#176;C containing  <br /> dicated plasmid. extract assayed formation 3H          protein 27 kDa size. release assay using RNA (1-847) fragment 10 mM Mg2 .                 band visible induced pET control (data <br />    bRatio activity/mL clones 2 4, respectively,            shown). majority overexpressed pro- <br /> pET-15b. <br />                                                                             tein S15 supernatant fraction (Fig. 4, <br />                                                                             lane 1), smaller protein S15 <br />                                                                             pellet (lane 2). protein supernatant <br /> deduced N-terminal amino acid sequence, MGMENY                              fraction appeared mainly free S200 (lane 3). <br /> NPPQEPWLV. . ., matched experimental data,                           smaller amounts associated ribosomes <br /> cluding questionable residues, completely.                       (lanes 4, 5). Extracts cells grown 30 &#194;&#176;C behaved gene cloned  described                             similarly. <br /> shown code 23S *746 synthase.                            gene cloned translation prod- <br /> gene 0-2.4 min region chromo-                       uct contain  6 sequence  accession number D10483, residues 59,346-                          leader <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> order affinity purify protein 60,005 reverse orientation.                       Ni2+-containing resin column. synthase activity normal base Shine-Dalgarno sequence separated                          affinity-purified protein constitute  nucleotides initiating AUG, sin-                      finitive proof gene cloned gene gle UAA termination codon. gene codes 219-                        synthase. Figure 4 (lane P) shows Ni2+- <br /> amino acid protein calculated MW 24,432 Da.                       column effectively removed contaminating <br />                                                                             protein S15 supernatant fraction, resulting                                                                             single protein band monomer MW 27.0 <br /> Cloning gene, overexpression,                                        kDa. 5&apos;-tag increases MW 2,179 Da, affinity purification                                                   expected value recombinant protein, 26.6 ORF cloned pET-15b standard meth-                           kDa, based MW 24.4 kDa calculated ods. clones selected 15 recovered                         ORF, good agreement  <br />  <br />  <br />  <br />                               S          1       2         3            4       5                P        S <br />                                                                                                                     *-100 <br />               75-&gt;                                                                                                  v-+75 <br />               50-&gt; <br />                                                                                                                     &apos;-50 <br />               35-&gt; <br />               25-&gt;                                                                                                  v35 <br />                                                                                                                    &apos;3 <br />  <br />  <br />  <br />  <br />           FIGURE 4. Overexpression, cellular localization, affinity purification 23S RNA *746 synthase gene product. <br />           Cells grown 37 &#194;&#176;C broken sonication fractionated S15 supernatant pellet. supernatant <br />           fractionated S200, ribosomal 1 M NH4Cl ribosomal wash (HSW), ribosomes. Lane 1, S15 su- <br />           pernatant; lane 2, solubilized S15 pellet; lane 3, S200; lane 4, HSW; lane 5, solubilized ribosomes; lane P, 0.3 ug           affinity-purified recombinant protein. Extracts approximately equivalent amounts cells loaded           lane P. S, molecular weight standards (Novagen) values indicated. <br />  442                                                                                                               J. Wrzesinski et al. synthase enzymatic activity resides                              TABLE 3. Specificity recombinant &apos;746 synthase various affinity-purified protein                                     RNAs                                                                                                1 mM EDTA              10 mM Mg2+ <br /> Treatment [5-3H]uracil-containing RNA (1-847) <br />                                                                                             Rate        Yield       Rate        Yield recombinant protein yielded 1.1 mol 3H                     RNA transcripts     (pmol/min) (pmol/pmol) (pmol/min) (pmol/pmol) <br /> released/mol substrate plateau reaction  Fig. 6 Table 3). verify &apos;746              23S RNA (1-847)        33.12       1.27         7.69       1.19 <br /> formed, chemical sequencing performed (Fig. 5).                  23S RNA (1-2,904)       7.23       0.97        4.33         - <br />                                                                      16S RNA (1-526)         0.04        -          0.02         - residues 671-786 shown fig-                 16S RNA (1-678)        &lt;0.01        -         &lt;0.01         - <br /> ure, sequence 440 796, 42% 1-847               16S RNA (1-1,542)       0.04        -         &lt;0.01         - <br /> fragment, screened. Clearly &apos;746 formed                 30S subunits            -           -         &lt;0.01         - <br />  substrate treated affinity-                    tRNAPhe                 6.21       0.78         0.77        - <br />  <br /> purified protein. absence enzyme treatment,                tRNAVal                  0.02       -           0.02        - <br />  <br />       affinity-purified pro-                                                                                            3H release assays performed described Materials <br />                                                                      methods 400 nM substrate 10 mM Mg2+ 100 IAL <br />                                                                      reactions. 1 mM EDTA used, Mg2+ substrate <br />                                                                      buffer present follows. RNA (1-847), 0.14 mM Mg2+; <br />                                                                      RNA (1-2,904), 0.43 mM Mg2+; RNA (1-526), 0.14 mM Mg2+; RNA <br />          _1- EaL                    t   Enza                         (1-678), 0.08 mM Mg2+; RNA (1-1,542), 0.12 mM Mg2+; tRNAPhe, <br />          4 24                                                        &lt;0.01 mM Mg2+; tRNAVal, 0.02 mM Mg2+. synthase concentra- <br />                         4 2 4 OH                                     tion 0.45 Ag/mL. -, determined. <br />              + +    ACUG + + CMC <br />  <br />                                                                      tein  fact, *I746 synthase gene                                                                      identified. <br />                                                                         specific activity EDTA RNA(1-847)                                                                      22.1 x 106 units/mg, corresponding turnover <br />                                                                      number 2.77 x 10-2 mol/min/mol enzyme.                                                                      comparison, equivalent turnover number                                                                      tRNA P55 synthase 1.05 x 10-2, threefold <br />                                                                      difference. enzymes carry                                                                      reaction single-stranded loop RNA closed <br />                                                                      stem, reasonable turnover numbers <br />                                                                      similar. specific activity isolated enzyme <br />                                                                      (Fig. 2) 1.6 x 106 units/mg,                                                                      compared value 5.1 x 106 recombinant <br />                                                4-746                 enzyme measured Mg2+ condi- <br />                                                                      tions. purity isolated enzyme estimated <br />                                                                      ca. 70% (Fig. 2), specific activity pure  <br />                                                                      zyme 2.3 x 106, half                                                                      recombinant enzyme.  recombinant enzyme <br />                                                                      appears active isolated material. <br />  <br />                                                                      Evidence dual specificity &apos;746 <br />                                                                      synthase effect Mg2+ <br />                                                                      specificity synthase tested tran- <br />                                                                      scripts 16S RNA, length fragments, <br />                                                                      tRNA transcripts available labora- <br />                                                                      tory. reaction length 16S <br />                                                                      RNA fragments 1-526 1-678, tRNAVS, <br />                                                                      10 mM Mg2+ 1 mM EDTA. 30S subunits <br />                                                                      10 mM Mg2+ unreactive. surprise, <br />                                                                      tRNAPhe reacted quite  presence EDTA, <br /> FIGURE 5. Reverse transcription analysis site formation    tRNAPhe transcript reacted virtually recombinant enzyme. RNA (1-847) reacted enzyme level 1.1 mol 3H released/mol RNA treated   rate length 23S RNA (Fig. 6A), reached <br /> Figure 3. Arrow shows position &apos;746.                          yield. 23S RNA (1-847) fragment <br />  Cloning dual-specificity E. coli synthase                                                                                      AA43 <br />                                                                                     ENZ                                          HYD <br />                                                                                     +   -   AG          GA       -     +   -   ENZ <br />                                                                        U45- <br />                                                                                                                                  -U20 <br /> E                                                                      U47- <br /> 5CIL <br />        0.5                                         L                   U50- <br />                                                                        U51- <br />  <br />        0.o <br />              0   5   10     15        0        5   10     15      20   U54-                                                      -U32 <br />                                  Time (mins)                           U55-                                                      -U33 <br /> FIGURE 6. Evidence dual function I746 synthase. <br /> [5-3H]uracil-containing vitro transcripts *, RNA (1-847); 0, <br /> RNA (1-2,904);  tRNAPhe incubated fixed affinity-purified recombinant enzyme formation  <br /> sayed release 3H described Materials methods ex- <br /> cept 400 nM substrate   presence *, 1 mM EDTA <br /> plus 0.14 mM Mg2+; 0, 1 mM EDTA plus 0.43 mM Mg2+;  1 mM <br /> EDTA; (B) 10 mM Mg2+. Pm/pm, pmol 3H released/pmol RNA              U59-                                                      -U39 <br /> added reaction. <br />                                                                        U60- <br />  better substrate, rate times <br /> faster, yielding 1 mol 3H released/mol                                                                           -U45 <br /> RNA. presence Mg2+ (Fig. 6B), reactions slowed, RNA (1-847) <br /> fragment rapid reach completion                                                                             -U47 <br /> allotted time. results summarized Table 3. substrates, RNA (1-847) RNA (1-2,904), <br /> rates measured 5 mM EDTA did differ 1 mM EDTA values. Note shift                                                                            -U50 EDTA Mg2+ affected active sub-                       U66-                                                      -U51 <br /> strates differently. 4.3-fold reduction RNA (1-847) fragment, 8.1-fold reduction                                                            B <br /> tRNAPhe transcript, 1.7-fold drop length 23S RNA. Mg2+, stim-                   FIGURE 7. Sequence analysis site formation tRNAPhe <br />                                                                        recombinant I746 synthase. [5-3H]uracil-containing ulator reaction, inhibitory. main result,               vitro transcript tRNAPhe reacted recombinant synthase  tRNAPhe, tRNAVal,                    1 mM EDTA level 0.8 mol 3H released/mol RNA se- <br /> act synthase.                                                 quenced.  Chemical sequencing (Peattie, 1979) using 3&apos;-end <span id='am-73' about='rdfs:label' typeof='owl:Thing'>label</span>- <br />                                                                        ing [5&apos;-32PJpCp. reaction U hydrazine    data Table 3 indicated single site             15 min 0 &apos;C. Enz +, enzyme-treated RNA; Enz -, control RNA <br />  tRNAPhe transcript, sequencing analysis                  incubated enzyme.  G, sequencing lanes using enzyme- <br /> used determine location (Fig. 7). Standard chem-                treated transcript. U residues indicated. B:                                                                         hydrazinolysis (HYD) performed 7 M <span id='am-84' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> room tem- <br /> ical sequencing (Fig. 7A) showed absence                 perature indicated times. <br /> U66  3&apos;-U) U45. particular, U55 converted  method, presence band <br /> denotes U absence indicates m5U (Peattie, <br /> 1979). band corresponding U39 visible                   U32. U16, U12, U8 bands experiment, apparently region                  equal intensity enzyme treatment (data highly structured allow reaction U39                   shown). results clearly U32 standard conditions. 7 M urea included                    U residue converted  particular, U39, hydrazinolysis room temperature (Fig. 7B),                 tRNAPhe (Steinberg et al., 1993), reaction U39 obtained                     modified. <br /> enzyme treatment. case U33                     Comparison sequence surrounding U32 U20. U residue did yield U band af-                tRNAPhe U746 23S RNA (Fig. 8) <br /> ter enzyme treatment, did treatment,                  revealed reason reactivity U32  444                                                                                                           J. Wrzesinski et al. <br />                                                                     DISCUSSION <br />                                                                        Substrate specificity <br /> 745GU          U G A752                        23S RNA <br />                                                                        work, described purification                                                                        cloning enzyme site specifically forms                                                                        E. coli 23S RNA. second description                                                                        synthase ribosomal RNA,       32U      U G Jb                          tRNAPhe           16S RNA (Wrzesinski et al., 1995), fourth <br />                                                                        example cloned synthase RNA, despite <br />                                                                        ubiquitious occurrence tRNA, rRNA,                                                                        sn(o)RNA. enzyme formed position 746       32U      U   G AA3                         tRNACYS           23S RNA. rigorously proven                                                                        direct analysis entire 23S RNA molecule, fact <br />                                                                        approximately unit stoichiometry 3H release <br />                                                                        obtained 23S RNA RNA(1- <br />       32U      U   2 A38                       tRNALeu4          847) (Table 1) argues strongly forma- <br />                                                                        tion occurred 847 nucleotides. Residues <br />                                                                        437-796, 43% nucleotides, directly <br />                                                                        screened.  native 23S RNA, *746                                                                        * 1-847 sequence (Bakin et al., 1993). Taken <br />       32U U                    38            tRNALOU6           facts provide strong evidence FIGURE 8. Sequence single-stranded segments vitro tran-   enzyme described work site-specific  <br /> scripts 23S RNA tRNAPhe recogni zed 23S         gard 23S RNA. <br /> RNA I746 synthase. Segments additional titNA transcripts          enzyme reacted tRNAPhe shown differ base (underlined). tRNA gene se- <br /> quences Komine et al. (1990). <br />                                                                        transcripts form *32. reason ap- <br />                                                                        parent sequences sites com- <br />                                                                        pared. shown Figure 8, sequences                                                                        identical * residue residues. <br /> tRNAPhe. sequences identical d                 Apparently, enzyme needs single-stranded <br /> single-stranded loops. survey known E. coli tRNA                  region appropriate sequence, need <br /> (Steinberg et al., 1993) tRNA gene seqluences (Ko-                 base paired stem close loop ruled et al., 1990) identified tR]NAs (shown                 event, location *746 base  Fig. 8) similar sequences. b e interesting             moved stem, *32 adjacent determine *746 synthase ccan form *32                   stem, does matter. equivalent transcripts.                                                sequences known * site 23S RNA <br />                                                                        (Bakin &amp; Ofengand, 1993; Bakin et al., 1994b) 16S <br />                                                                        RNA (Bakin et al., 1994a), keeping evidence <br /> Comparison amino acid sequence                           reactive site 23S 16S RNA U746 <br /> synthase known proteins                                     23S RNA. known * sites E. coli <br />                                                                        tRNAs bear sequence similarity *32 search GenBank database using TFASTA                      tRNAcys, tRNALeu4, tRNALeU6, dif- <br /> module GCG Sequence Analysis Sc)ftware suite                    fers single base 2 3 residues * programs (Devereux et al., 1984) did n ot reveal                (Fig. 8). *746 synthase capable *32 <br /> existence gene product similarity                 formation transcripts tRNAs known. *746 synthase.  coomparison                     example RNA modifying enzyme enzyme (rluA)                     dual specificity,  completely cloned * synthases known, tthe rsuA syn-                 specific single site RNA, case U746 <br /> thase forms *516 E. coli small su bunit rRNA                    23S RNA, equally specific single site  <br /> (Wrzesinski et al., 1995), truA  k:nown                 class RNA, U32 tRNAPhe. <br /> hisT) gene product converts U residue?s anti-                  Despite fact *746 located codon arm tRNA * (Arps et al., 19485; Kammen                    modified nucleotides mlG745 m5U747  Fig. 1), <br /> et al., 1988), truB, gene produ(ct forms *55                  requirement modified bases tRNA (Nurse et al., 1995), virtually nc3 amino acid                  * formation. vitro transcripts 23S RNA <br /> sequence correspondence detecAted (Fig. 9).                    RNA(1-847) excellent substrates ab- <br /> Despite common catalytic reaction cairried                   sence modified bases.  reciprocal enzymes, sequence homology pri-                       need true. Formation m1G m5U mary level evident.                                             require prior * formation. <br />  Cloning dual-specificity E. coli synthase                                                                                                   445 <br />  <br />                     1                                                     50 <br />               truB MSRPRRRGRD INGVLLLDKP QGMSSNDALQ KVKRIYNANR AGHTGALDPL <br />               rsuA <br />               truA                         .M... . . ...... .       . .M ..     0        ..*   0   0           0   0       0   0 <br />  <br />  <br />               rluA .... .     .. ... <br />                                     .. <br />                                                . M . . . . . e . . .. m <br />                                                                                       0    0   0                   &amp;       0             <br />  <br />  <br />  <br />                          51                                                                                                                  100 <br />               truB       ATGMLP ICLG EATKFSQYLL                DSDKRYRVIA             RLGQRTDTSD                 ADGQIVEERP <br />               rsuA                   ..MRLDKFIA                QQLGVSRAIA             GREIRGNRVT                 V-DGEIVRNAA <br />               truA       SDQQQPPVYK IALGIEYDGS                 KYYGWQRQNE             VRSVQEKLEK                 ALSQVANEP               ri uA      GMENYNPPQE PWLVILYQDD                 HIMVVNKPSG             LLSVPGRLEE                 HKDSV ..... <br />                    101                                                 150 <br />               truB VTFSAEQLAA ALDTFRGDIE QIPSMYSALK YQGKKLYEYA RQGIEVPREA <br />               rsuA FKLLPEH.DV AYDGNPLAQQ HGPRYFMLNK PQG...YVCS TDDPDHP... <br />               truA TVFCAGRTDA GVHGTGQVVH FETTALRKDA AWTLGVNANL PGDIAVRWVI <br />               rluA  ... MTRIQR DYPQAESVHR LDMAT..... .......... SGVIVVALTK <br />                          151                                                                                                                 200 <br />               truB       RPITVYELL.         .... FIRHEG NELELEIHCS KGTYIRTIID DLGEKLGCGA <br />               rsuA       ... TVLYFLD        EPVAWKLHAA GRLDID...T TGLVLMTDDG QWSHRITSPR <br />               truA       TVPDDFHARF         SATARRYRYI ... IYNHRL RPAVLSKGVT HFYEPLDAE. <br />               rluA       AAERELKRQF         REREPKKQYV AR.VWGHPS PAEGLVD... ...LPLICD. <br />                          201                                                                                                                 250 <br />               truB HVIYLRRLAV SKYPVERMVT LEHLRELVEQ AEQQDIPAAE LLD. ... P.. <br />               rsuA HHCEKTYLVT LESPVAD.DT AEQFAKGVQL HNEKDLTKPA VLEVITP ... <br />               truA . . ... .RMHRAAQ...CLLG ENDFTSFRAV QEQSRTPWRN VMHINVTRHG <br />               rluA    .W.P...NR PKQKVCYETG KPAQTEYEVV EYAADNTARV VLKP ITGRSH <br />                          251                                                                                                                 300 <br />               truB       . . LLMPKDSP ASDYPVVNLP LTSSVYFKNG NPVRTSGAPL EGLVRVTEGE <br />               rsuA       . . TQVRLTIS EGRYHQVKRM FAAV.... GN HVVELHRERI GGITLDADLA <br />               truA       PYVVVDZKAN AFVHHMVRNI VGSLMEVGAH NQPESWIAEL LAAKDRTLAA <br />               rluA       QLRVHML... ALGHPILGDR FYASPEARAM APRLLLHAEK LT ITHPAYGN <br />                     301                               334 <br />               truB NGKFIGMGEI DDEGRVAPRR LVVEYPA ... ... <br />               rsuA PGEYRPLTEE EIASVV.... <br />               truA TAKAEGLY LVAVDYPDRY DLPKPPMGPL FLAD <br />               rl uA SMTFKAPADF <br />         FIGURE 9. Sequence comparison 23S RNA *746 synthase known synthases. Sequences tRNA <br />         &apos;55 synthase (truB), anticodon arm synthase (truA), 16S &apos;516 synthase (rsuA), enzyme (rluA)         compared using PILEUP module GCG Sequence Analysis Software programs (Devereux et al., 1984). Bold let- <br />         ters identity high similarity. C residues double underlined. Numbers left right margins amino <br />         acid sequence numbers truB protein. <br />  <br />  <br />  <br />  <br /> Gene protein      sequence                                        unique <span id='am-48' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-49' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-53' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence recognized                                                                       enzyme adjacent U modified, gene identified long sequenced                common amino acid elements catalysis site <br /> gion near origin chromosome. propose                    enzyme obscured unique amino acid rluA gene (ribosomal large subunit                  residues needed recognition. search GenBank <br /> pseudo U formation U modification). Comparison                  analogs enzyme species sequence synthase sequences                   unsuccessful. <br /> known did reveal conserved primary sequence                      Cysteine residues known important elements, despite fact enzymes               truA  T) synthase tRNA (Kammen et al., 1988), catalytic center.                 general feature synthases. <br />  446                                                                                                       J. Wrzesinski et al. <br /> Mechanistically, role SH group readily         obtained Sigma. Deoxyoligonucleotide primers envisaged. known enzymes contain 2-3            prepared described previously (Bakin &amp; Ofengand, 1993). <br /> cysteine residues, comparison location        DEAE Sepharose CL6B, MonoS, MonoQ FPLC columns primary sequence (Fig. 9) did reveal com-           Pharmacia. <br /> mon features suggest functional role. <br />                                                                 Buffers <br /> Importance enzyme                                        Buffer  10 mM Hepes, pH 8.0, 10 mM MgCl2, 5 mM mer- <br />                                                                 captoethanol, 0.1 mM EDTA. Buffer B: 20 mM Hepes, pH 8.0, significance enzyme E. coli depends          20 mM NH4Cl, 5 mM mercaptoethanol, 0.1 mM EDTA, 10% role *746 ribosome structure function.         glycerol. Buffer C: buffer B, pH 7.8. Buffer D: 20 mM region 5&apos;-half molecule,          Tris, pH 8.0, 20 mM NH4Cl, 5 mM MgCl2, 5 mM mercapto- believed  physically near,      ethanol, 0.1 mM EDTA, 10% glycerol. Buffer E: 20 mM Tris, PTC (Bakin &amp; Ofengand, 1993; Bakin et al., 1994b)           pH 8.0, 20 mM NH4C1, 6 M urea. Buffer LB: 50 mM Tris- envisaged play role pep-         HCI, pH 6.8, 100 mM DTT, 2% SDS, 10% glycerol, 0.1% <br /> tide bond formation special properties          bromphenol blue. Binding buffer: 20 mM Tris-HCl, pH 7.9, <br />                                                                 0.5 M NaCl, 5 mM imidazole. Elution buffer: binding buffer <br /> (Lane et al., 1992, 1995; Bakin &amp; Ofengand, 1993; Bakin         imidazole concentration raised 1.0 M. <br /> et al., 1994b).  position universally modified. Analyses Halobacter halo- <br /> bium, Bacillus subtilis, Drosphila melanogaster, Mus mus-       Pseudouridine formation assay <br /> culus, Homo sapiens large subunit ribosomal RNAs <br /> did detect position (Ofengand et al., 1995).      Reactions contained 50 mM Hepes, pH 7.5, 100 mM NH4Cl,  *746 important               2 mM indicated Mg(OAc)2, 80-100 nM  indi- <br /> essential E. coli. Gene disruption studies           cated, [5-3H]uracil-containing length 23S RNA                                                                 (1-847) fragment, 5 mM DTT, 400 units/mL RNasin,  <br /> progress answer question.                       zyme. Mixtures preincubated 5 min 37 &#194;&#176;C    aspect function enzyme         adding enzyme. Incubation continued 37 &#194;&#176;C mation *32 tRNAPhe, potentially               various times. Reactions stopped addition 97-,uL <br /> tRNAcys, tRNALeu4, tRNALeu6.             aliquots 1.0 mL 12% Norit 0.1 N HCI. Samples tRNAs E. coli known contain I32,            mixed allowed stand room temperature 5 min, sequences suggest modified           centrifuged remove charcoal, supernatant *746 synthase. clear synthase        passed Acrodisc filter assembly (0.2 um, cat #4192, capable *32 formation,            Gelman Sciences). earlier experiments, 0.5-mL sam-  tRNA-specific, synthase cell.          ple counted value multiplied 2. Later,   known essentiality            tire supernatant removed, charcoal pellet <br /> *32 tRNA processing/biosynthesis tRNA             washed twice 1.0 mL 0.1 N HCl, time passing <br />                                                                 supernatant wash filter assembly function. Disruption *746 synthase gene           rinse. washes supernatant combined answer questions                                     counted. washes sufficient elute 97%    need consider &quot;real&quot; function           recoverable 3H. methods gave essentially  enzyme *32 formation, *746             sults, reproducible. unit ac- <br /> mation harmless adventitious event. gene disrup-        tivity defined enzyme catalyzing tion effect, question trivial.           release 1 pmol 3H supernatant 30 min 37 &#194;&#176;C.  gene disruption deleterious results loss *32 tRNAs question, loss *746, determination * responsible           RNA transcripts difficult.                                               rRNA transcript length 23S RNA (1-2,904)                                                                 prepared linearization pCW1 (Weitzmann et al., 1990) <br />                                                                 transcription 40 mM Hepes, pH 7.8, 20 mM MgCl2, <br /> MATERIALS METHODS                                           40 mM NaCl, 4 mM <span id='am-63' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>spermidine</span>, 10 mM DTT, 5 mM                                                                 ATW, CTTW, UTP, <span id='am-16' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GTP</span>, 2 units/mL inorganic pyrophosphatase, <br /> Materials                                                       1,000 units/mL RNasin, 22 nM linearized plasmid, 5,000 <br />                                                                 units/mL T7 RNA polymerase, 300 ,uCi/mL [5-3H]UTP <br /> [5-3H]UTP [_x-32P]<span id='am-32' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="dATP" datatype="xsd:string"></span>dATP</span> Amersham. RNasin           37 &#194;&#176;C 7-10 h. synthesis fragment 1-847 run- Promega. Restriction enzymes T4 RNA ligase         transcription, pCW1 linearized EcoR tran- New England Biolabs. T7 RNA polymerase            scribed described. Transcripts 16S RNA fragments Ambion,  Plasmid pET-15b, BL21/DE3             tRNAval tRNAPhe prepared described <br /> Novablue strains E. coli, Bind resin obtained   previously (Nurse et al., 1995; Wrzesinski et al., 1995). Novagen,  T4 DNA ligase shrimp phosphatase         RNA samples purified phenol extraction, ethanol U.S. Biochemical. Norit washed HCl         precipitation, gel filtration. <br />  Cloning dual-specificity E. coli &apos; synthase                                                                            447 <br /> Purification 23S*746 synthase                                overexpression, transformed BL21/DE3 cells                                                                  grown   M9ZB (Studier et al., 1990) 30 &#194;&#176;C 37 &#194;&#176;C grams E. coli MRE600 frozen cell paste (Grain <br /> Processing Corp.) harvested mid-log phase washed,         A6w 0.6. IPTG (1 mM) added cells grown 30 &#194;&#176;C <br />                                                                  37 &#194;&#176;C 4 h. Cells disrupted sonication 0.15 thawed 4 &#194;&#176;C 30 mL buffer  removal        times original culture volume (OCV) buffer D plus liquid centrifugation, cells suspended     1 mM phenylmethylsulfonylfluoride, centrifuged 250 mL buffer plus 10% glycerol, disrupted son-      15,000 x g obtain S15 supernatant pellet fractions. <br /> ication. Ribosomes cell debris removed centri-       pellet dissolved 0.04 OCV buffer E. S15 su- <br /> fugation 35,000 rpm 3 h Spinco Ti45 rotor.    pernatant centrifuged 45,000 rpm 3 h Spinco <br /> supernatant (S200) added seventh volume 20%           Ti6O rotor obtain S200 supernatant ribosomal <br /> streptomycin sulfate adjusted pH 7.5, mixture         pellet. 3/4 supernatant taken S200 <br /> stirred 4 &#194;&#176;C 30 min. streptomycin supernatant         extract. ribosomal pellet suspended 0.025 OCV <br /> (270 mL) recovered centrifugation precipitated        20 mM Hepes, pH 8.0, 1 M NH4Cl, incubated 0 &#194;&#176;C ammonium sulfate (151 g). mixture adjusted       2 h, centrifuged 200,000 x g 14 h. super- <br /> pH 7-7.5 NH40H stirred overnight 4 &#194;&#176;C.           natant taken ribosomal high salt wash (HSW), precipitate collected centrifugation, pellet dis-     ribosomal pellet dissolved 0.04 OCV buffer E. <br /> solved 30 mL buffer B, dialyzed buffer remove ammonium sulfate. <br />    dialyzed sample loaded 0.15 mL/min            Affinity purification synthase <br /> 1.5 x 90 cm column DEAE Sepharose CL6B equilibrated buffer C eluted linear gradient 1 mM/mL       S15 supernatant dialysed versus binding buffer im- <br /> NH4Cl buffer C. Synthase activity eluted 280 mM            mediately application 2.5 mL column Bind <br /> NH4Cl recovery 60% input. Peak fractions        resin. Conditions preparation operation column pooled concentrated dialysis buffer C        described pET Manual, 4th edition, <br /> plus 20% polyethylene glycol 6000 loss activity.      Novagen,  addition elution buffer, tagged sample (13.4 mL) loaded 8-mL MonoS            protein released. A280-containing fractions column equilibrated buffer C eluted 1 mL/min      pooled dialyzed versus 20 mM Tris-HCl, pH 7.9, 0.5 M linear gradient 1.9 mM/mL NaCl buffer C.          NaCl, plus imidazole 0.5 M (1 h), 0.25 M (overnight), 0 M <br /> enzyme eluted 210 mM NaCl recovery 35%           (1 h), finally versus 20 mM Tris-HCl, pH 7.9, 0.35 M NaCl input activity. Pooled samples dialysed buf-    1 h. regimen, protein stayed soluble. fer C, applied 1 mL column MonoQ eluted           lower concentrations salt, higher protein concentra- <br /> 0.5 mL/min gradient 9.3 mM/mL NaCl buffer C.        tions, protein tends precipitate. protein solution enzyme eluted sharp peak  Fig. 2) 110 mM         diluted equal volume glycerol stored NaCl buffer C recovery 60% input activity.      -20 &#194;&#176;C concentration 30 tg/mL. pooled fractions concentrated placed buffer C Centricell filtration. Glycerol added 50% <br /> final concentration samples stored -20 &apos;C.            Polyacrylamide gel electrophoresis <br />                                                                  SDS gels 12% contained 0.375 M Tris-HCl, pH 8.8, <br /> Cloning overexpression 23S*746                        0.1% SDS. 5% stacking gel contained 0.127 M Tris- <br /> synthase gene                                                    HCI, pH 6.8, 0.1% SDS. Samples heated 95 &#194;&#176;C                                                                  5 min buffer LB quenched ice loading. candidate ORF amplified prepared insertion       Gels stained Coomassie Blue using pET-15b PCR. N-terminal primer extended         silver stain reagent kit protocol Biorad Laboratories, <br /> -21 +19 initiating AUG +1,          <br /> changes -1, -3, -4, -5, create Xho site adja- <br /> cent initiating AUG. C-terminal primer,  <br /> verse orientation, extended +671 +710,         Protein sequencing sense nucleotide 657, contained mismatches 694, <br /> 695, 696, 697 order create BamH site. Primers      Glycerol removed purified enzyme dialysis <br />                                                                  protein precipitated volumes cold ace- removed membrane filtration (Amicon Microcon 100) <br />                                                                  tone -20 &#194;&#176;C overnight 1 h -70 &apos;C. pre- amplified product digested Xho BamH  <br /> zymes. pET vector digested Xho BamH         cipitate dissolved buffer LB plus 3.5 M urea dephosphorylated shrimp phosphatase, purified gel        electrophoresed  Samples electroblotted                                                                  PVDF membrane (Millipore Corp.) following standard pro- <br /> electrophoresis, incubated gene insert li-     cedures (Matsudaira, 1987). N-terminal sequencing car- <br /> gation mixture containing 50 mM Tris-HCl, pH 7.9, 10 mM          ried described previously (Denman et al., 1989). <br /> MgCl2, 25 Ag/mL nuclease-free bovine serum albumin, 10 mM <br /> DTIT, 1 mM <span id='am-37' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span>, 500 units/mL T4 DNA ligase, 2.2 Lg/mL vec- <br /> tor, 12 Ag/mL insert 20 h 16 &apos;C. Transformation    Protein determinations <br /> Novablue cells standard methods yielded <br /> 15 clones 25 tested correct insert pET vec-   Protein content assayed modified Bradford proce- <br /> tor. Plasmids clones transferred BL21/DE3     dure (Bio-Rad Protein Assay cat# 500-0006) using bovine se- <br /> cells.                                                           rum   albumin standard. <br />  448                                                                                                                         J. Wrzesinski et al. <br /> ACKNOWLEDGMENTS                                                                gle base substitutions, insertions, deletions  <br />                                                                                coding site (C1400). Biochemistry 28:1002-1011. thank Kurt Hoilfelder Yu-Ching Pan Depart-                    Devereux J, Haeberli P, Smithies 0. 1984. comprehensive set ment Protein Biochemistry, Hoffmann-La Roche, ,                     sequence analysis programs VAX. Nucleic Acids Res 12: sequencing analysis. Supported grant (MRC-                    387-395. <br />                                                                             Gutell RR. 1993. Collection small subunit (16S- 16S-like) <br /> MT-1226) Medical Research Council Canada                        ribosomal RNA structures. Nucleic Acids Res 21:3051-3054. <br /> B.G.L.                                                                      Kammen HO, Marvel CC, Hardy L, Penhoet EE. 1988. Purification, <br />                                                                                structure properties Escherichia coli tRNA pseudouridine <br />                                                                                synthase  Biol Chem 263:2255-2263. <br /> Received 3, 1995; returned revision 24, 1995;                   Komine Y, Adachi T, Inokuchi H, Ozeki H. 1990. Genomic organi- <br /> revised manuscript received June 7, 1995                                       zation physical mapping transfer RNA genes Esch- <br />                                                                                erichia coli K12. J Mol Biol 212:579-598. <br />        <br /> </body></html>