<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br />  <br />                            high-efficiency method replace essential genes mutant <br />                            alleles yeast <br />  <br />  <br />                            Abstract <br />                                 Temperature-sensitive (TS), internally deleted truncated alleles important tools facilitate <br />                                 characterization essential genes. developed straightforward method replace                                 wild-type gene mutant allele endogenous locus. method efficient alternative <br />                                 step method integration alleles compromised function contain multiple <br />                                 mutations. strain constructed essential gene disrupted selectable <br />                                 marker. Strain viability maintained plasmid carrying copy essential wild-type gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 theADE3 gene. mutant allele cloned integratable vector carrying selectable/ <br />                                 counter-selectable marker, asURA3. plasmid linearized transformed, directing <br />                                 integration 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; region flanking essential open reading frame (ORF). Transformants                                 integrated mutant gene endogenous locus lose autonomous plasmid carrying <br />                                 wild-type copy essential gene theADE3gene. transformants identifiable                                 white sectoring colonies, display mutant phenotype characterized. optional second <br />                                 selection step 5-fluoroorotic acid (5-FOA) selects popouts integrating vector sequences, <br />                                 leaves mutant allele endogenous locus, recycles selectable markers. used                                 method integrate TS allele SPC110 integrated standard methods. <br />  <br />  <br />                            Keywords <br />                                 temperature-sensitive; gene replacement; integration; plasmid shuffle; conditional allele; <br />                                 counterselection <br />  <br />  <br />                            Introduction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Mutant alleles important tools provide insight functions essential genes.                                             number simple high-efficiency methods developed replacement                                             non-essential genes mutant alleles yeast (Struhl, 1983; Szent-Gyorgyi, 1996).  <br />                                             working essential genes, classic step gene replacement method                                             main method used, functional copy gene question present times <br />                                             maintain strain viability (Scherer Davis, 1979; Winston et al., 1983; Boeke et al., <br />                                             1987; Rothstein, 1991). basic strategy introduce mutant allele linearizing                                             integratable plasmid containing mutant allele URA3, selectable/counter-selectable <br />                                             marker. Linearization directs integration endogenous locus gene, resulting                                             tandem array wild-type mutant allele flanking URA3 marker. counter- <br />                                             selection 5-FOA, strains carrying mutant allele identified screening. <br />  <br />                                             notable drawbacks step approach. screen mutant allele <br />                                             time consuming mutant does display obvious phenotype. Alternatively, <br />  <br />  <br />                            *Correspondence  Trisha N. Davis, Department Biochemistry, Box 357350, University Washington, Seattle, WA 98195-7350, <br />                            USA. E-mail: tdavis@u.washington.edu <br />                             Widlund Davis                                                                                             Page 2 <br />  <br />  <br />                                                screen large number colonies isolating strain                                                gene-replaced allele mutant allele selected mutant fit <br />                                                compared wild-type. case, extremely difficult replace wild-type NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                mutant allele.  mutations spread significant portion essential <br />                                                ORF, number lost integration recombination event required <br />                                                growth 5-FOA. gene replacement strategies work cases,                                                susceptible gene conversion wild-type sequence present site integration <br />                                                (Shortle et al., 1984; Erdeniz et al., 1997). <br />  <br />                                                developed high-efficiency method overcomes difficulties allows <br />                                                replacement essential gene non-lethal mutation. <br />  <br />                               Materials methods <br />                               Media <br />                                                YPD medium SD medium described (Sherman et al., 1986). SD complete <br />                                                SD medium supplemented 50 &#206;&#188;g/ml <span id='am-1' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span>, 25 &#206;&#188;g/ml <span id='am-13' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span>, 100 &#206;&#188;g/ml tryptophan, <br />                                                0.1% casamino acids. SD-tryptophan SD complete lacking tryptophan. SD-uracil low <br />                                                adenine SD complete lacking uracil contains 5 &#206;&#188;g/ml adenine. 5-FOA medium SD <br />                                                complete 100 &#206;&#188;g/ml uracil supplemented 1 mg/ml 5-FOA (Boeke et al., 1987). <br />                                                YP glycerol medium YPD medium 2% glycerol substituted glucose. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Plasmids <br />                                                Plasmids used study listed Table 1. Plasmid pHS55 integratable version                                                plasmid pHS54. Plasmid pHS54 carries spc110-227 allele, recovered TS <br />                                                screen described previously(Sundberg Davis, 1997). <br />  <br />                               Strains <br />                                                yeast strain used study HSY2-12C. genotype MATa ade2-1oc ade3&#206;&#8221; <br />                                                can1-100 his3-11,15 leu2-3,112 lys2&#206;&#8221;::HIS3 spc110&#206;&#8221; ::TRP1 trp1-1 ura3-1 carrying <br />                                                plasmid pHS26. haploid isolate HSY2 derivative W303(Geiser et al., <br />                                                1993). Escherichia coli strain used plasmid preparations XL-1 Blue (Stratagene). <br />  <br />                               Transformation selection <br />                                                pHS55 (1 &#206;&#188;g) linearized SnaBI. linearized DNA transformed                                                equivalent 5 ml culture HSY2-12C 100 Klett units, using LiAc TRAFO method <br />                                                (Schiestl Gietz, 1989). transformation reaction plated SD-uracil low <br />                                                adenine plates incubated room temperature 7 days. Sectored colonies isolated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                streaked SDuracil low adenine obtain single, uniformly white colonies. Single <br />                                                white colonies sectored isolate tested growth YP glycerol eliminate <br />                                                isolates petite. remaining white isolates streaked YPD duplicate. <br />                                                set incubated room temperature 2 days allow recombination loss                                                URA TRP markers SPC110 locus, set incubated 37 &#194;&#176;C                                                2 days test temperature sensitivity. sample room-temperature streak struck <br />                                                heavily 5-FOA plates incubated room temperature 5 days. Single colonies                                                isolated 5-FOA plates tested lost ability grow                                                medium lacking tryptophan. <br />  <br />                               PCR <br />                                                Colony PCR using pairs primers performed steps gene <br />                                                replacement. Primer positions illustrated Figure 2d. Primer 1 anneals middle  <br />  <br />                                                      Yeast. Author manuscript; available PMC 2006 December 12. <br />                             Widlund Davis                                                                                                  Page 3 <br />  <br />  <br />                                                TRP1 ORF sequence 5&#226;&#8364;&#178;-TAATTTCACAGGTAGTTC-3&#226;&#8364;&#178;. Primer 2 primer <br />                                                T7 sequence 5&#226;&#8364;&#178;-TAATACGACTCACTATAGGG-3&#226;&#8364;&#178;. Primers 3 4 span SPC110 <br />                                                ORF sequences 5&#226;&#8364;&#178;-TGCGCGGATATCTTAAGCAA-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                TTGAAGATCAGTCTGCGTAGG-3&#226;&#8364;&#178;, respectively. Colony PCR performed 50 &#206;&#188;l <br />                                                reactions containing 1 &#195;&#8212; GC buffer, 200 &#206;&#188;M dNTPs, 0.5 &#206;&#188;M primer, 0.5 &#206;&#188;l Phusion&#226;&#8222;&#162; <br />                                                DNA polymerase (Finnzymes). Reactions cycled 30 times 98&#194;&#176;C 10 s, 50&#194;&#176;C 20 <br />                                                s 72&#194;&#176;C 2 min, final extension step 72&#194;&#176;C 7 min. PCR products separated <br />                                                1% agarose gel withm 1 kb ladder (NEB). <br />  <br />                               Sequencing <br />                                                Colony PCR performed using primers 3 4 described  3 kb products                                                purified QIAquick&#194;&#174; PCR purification kit (Qiagen). Sequencing primer <br />                                                5&#226;&#8364;&#178;-TGAAATCTGAACAGAGTA-3&#226;&#8364;&#178;. <br />  <br />                               Results discussion <br />                                                developed new method gene replacement based plasmid shuffle <br />                                                strategy (Bender Pringle, 1991). essential ORF replaced                                                selectable marker URA3 diploid strain homozygous ade2-1oc mutation <br />                                                ade3&#206;&#8221;. PCR cassette based knock strategies using kanamycinor hygromycin resistance <br />                                                genes process simple efficient (Wach et al., 1994; Goldstein McCusker, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                1999). strain transformed plasmid containing essential ORF flanking <br />                                                upstream downstream sequence (typically &#226;&#710;&#188;500 bp 5&#226;&#8364;&#178; 3&#226;&#8364;&#178;), ADE3 gene.                                                ADE3 gene used selection medium lacking histidine genes                                                compromised.  selectable marker URA3 included ADE3 <br />                                                plasmid. routinely use plasmid carrying 2 &#206;&#188;m origin replication, ADE3 LYS2                                                selection transformation, using plasmids pLI831, pTD29 essential <br />                                                gene (Muller 1996). Selection ADE3 plasmid followed sporulation                                                isolation haploid carrying disrupted ORF ADE3 plasmid. strain                                                maintain ADE3 plasmid viability, carries copy essential ORF. <br />                                                maintenance ADE3 ade2-1oc background, colonies strain red, <br />                                                especially supplied minimal amounts adenine. strain called &#226;&#8364;&#732;plasmid shuffle <br />                                                strain&#226;&#8364;&#8482; (Figure 1a). <br />  <br />                                                Following strain construction, mutant ORF flanking sequence cloned                                                integratable vector containing URA3 gene, pRS306 (Sikorski Hieter, 1989). <br />                                                plasmid linearized unique restriction site 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; region essential <br />                                                gene. unique site available, engineered. generally use unique site <br />                                                3&#226;&#8364;&#178; region. Transformation linearized plasmid plasmid shuffle indicator <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                strain directs integration 3&#226;&#8364;&#178; region essential gene, present                                                endogenous locus 3&#226;&#8364;&#178; region copy gene ADE3 plasmid (Figure <br />                                                1b). Integration endogenous locus allows loss ADE3 plasmid,                                                integration ADE3 plasmid makes loss lethal, carries functional copies <br />                                                essential gene. Consequently, transformation reaction plated SD-uracil <br />                                                low adenine plates, event result white sectored colonies,                                                solid red colonies (Figure 1c). Solid white colonies isolated sectored colony <br />                                                streaking single colonies. Loss mitochondrial DNA cause white sectoring <br />                                                colony, colonies easily identified inability grow medium                                                glycerol sole carbon source. white colony able grow YP glycerol                                                URA3 marker flanked disrupted ORF mutant gene endogenous locus <br />                                                (Figure 1c). mutant phenotype apparent stage. Plating medium containing <br />                                                5-FOA selects recombination events remove URA3 marker (Figure 1d).                                                events remove sole functional copy essential gene, resulting non-viable <br />  <br />                                                      Yeast. Author manuscript; available PMC 2006 December 12. <br />                             Widlund Davis                                                                                                 Page 4 <br />  <br />  <br />                                                strain. event result viable strain carries desired mutant allele                                                endogenous locus. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                proof principle, integrated spc110-227, temperature-sensitive allele                                                gene encoding spindle pole body component (Sundberg Davis, 1997). allele falls <br />                                                complementation group spc110-226 allele, defects spindle <br />                                                assembly achieve bioriented spindle (Yoder et al., 2005). Previous attempts                                                integrate spc110-227 allele classical step method failed. <br />  <br />                                                plasmid shuffle indicator strain described previously (Geiser et al., 1993).                                                case, TRP1 replaces SPC110 ORF plasmid pHS26 ADE3 plasmid. <br />                                                Linearization transformation 1 &#206;&#188;g plasmid pHS55, contains spc110-227 allele, <br />                                                yielded 715 transformants.  21 sectored white (Figure 2a). Sectored colonies  <br />                                                streaked isolate single, uniformly white colonies. Seven isolates generated white <br />                                                colonies unable grow YP glycerol, indicating colonies petite                                                 remaining 14 isolates temperature-sensitive 37&#194;&#176;C (Figure 2b). 14 <br />                                                isolates able generate colonies 5-FOA subsequently unable                                                grow medium lacking tryptophan, confirming loss spc110::TRP1 allele                                                recombination. monitored process colony PCR (Figure 2c,d). Finally, 14 isolates <br />                                                contained mutations present SPC110 ORF plasmid pHS55, confirmed                                                sequencing. spc110-227 allele seven mutations spanning approximately 350 base pairs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                seven mutations (A2305G, T2405C, A2434G, G2452A T2491A) result                                                published <span id='am-6' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> changes (R784G, M802T, K812E, E818K Y831N) (Sundberg                                                Davis, 1997). remaining mutations (A2322G A2673G) silent. Base pairs                                                residues numbered start SPC110 ORF. <br />  <br />                                                increased efficiency method compared classic step method                                                attributed mainly extent selection mutant allele. step approach, <br />                                                mutations lost integration wild-type sequence present site                                                recombination. Degradation ends linearized integratable vector increases                                                possibility. selection integration event, allow sufficient cell divisions <br />                                                popout event occur. Selection mutant allele occurs directly                                                recombination, progressively decreasing fraction total cells mutant allele <br />                                                cell division. Depending severity selection, make possibility <br />                                                isolating mutant low. method minimizes problems. integration, <br />                                                mutations ORF mutant allele lost wild-type sequence present <br />                                                endogenous locus. Gene conversion wild-type sequence autonomous <br />                                                plasmid possible event infrequent, rarely plasmid shuffle <br />                                                strain sector rare event result URA+ colony picked <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                screen. transformation, selection mutant allele affects extent <br />                                                sectoring colony arose cell integrated mutant allele                                                endogenous locus. colony containing exceptionally compromised allele                                                endogenous locus eventually sector extent seen, making assay                                                sensitive. soon sectoring occurs, selection mutant allele occur                                                reversions possible, wild-type sequence present mutant white colony. <br />  <br />                                                method used step step method. white colonies display <br />                                                mutant phenotype; consequently, allele characterized single <br />                                                transformation selection. optional second popout step 5-FOA                                                time useful needs make selectable markers available. method <br />                                                readily available using red/white screen TS mutants.                                                needs convert recovered mutant alleles ARS/CEN plasmids integratable <br />                                                versions. Conveniently, modified use URA-based TS screen,  <br />  <br />                                                      Yeast. Author manuscript; available PMC 2006 December 12. <br />                             Widlund Davis                                                                                                            Page 5 <br />  <br />  <br />                                                2 &#206;&#188;m ARS/CEN plasmid contains URA3 wild-type allele, ARS/CEN vector <br />                                                mutant allele selectable marker (Boeke et al., 1987). Essentially, <br />                                                integration performed described, selection integrants genomic locus <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                occurs 5-FOA medium. popout integratable plasmid desired,                                                integratable vector contain selectable/counter-selectable marker URA3,                                                LYS2. <br />  <br />   <br />  <br />  <br />                                                Figure 1. <br />                                                  plasmid shuffle strain: strain ORF replaced selectable <br />                                                marker URA3. example shown  TRP1 used. 5&#226;&#8364;&#178;- 3&#226;&#8364;&#178; regions flanking <br />                                                ORF indicated. 2&#206;&#188;m plasmid carrying wild-type gene required maintain <br />                                                strain viability. 2&#206;&#188;m plasmid carries strain&apos;s sole copy ADE3. strain carries <br />                                                ade2-1oc mutation remains red, cells losing 2&#206;&#188;m plasmid                                                viable. (b) Integration: integratable plasmid carrying URA3 selection mutant <br />                                                gene linearized unique restriction site marked asterisk. Linearization directs <br />                                                integration 3&#226;&#8364;&#178; region endogenous locus ADE3 plasmid. (c) <br />                                                Organization essential locus: cells losing ADE3 plasmid viable integration <br />                                                occurs endogenous locus, resulting sectored colony. white colony deleted <br />                                                ORF, followed URA3 mutant gene endogenous locus. (d) Popout: white <br />                                                colony 5-FOA select URA3 marker. Recombination 3&#226;&#8364;&#178; <br />                                                regions results inviable strain, recombination 5&#226;&#8364;&#178; regions results NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                viable strain desired mutations <br />  <br />  <br />  <br />  <br />  <br />  <br />  <br />                                                Figure 2. <br />                                                  Sectoring: single transformation linearized integration vector gave 21 sectored <br />                                                colonies. sample transformants shown. arrow indicates sectored colony. (b) <br />                                                Phenotype: temperature sensitivity representative white colonies compared wild- <br />                                                type (wt) strain. Strains incubated 37&#194;&#176;C 2 days. (c) Colony PCR using sets                                                primers performed stages replacement. Lane 1, plasmid shuffle strain <br />                                                template, using primers 1 2; lane 2, URA+ white isolate template, using <br />                                                primers 1 2; lane 3, TRP&#226;&#8364;&#8220;URA&#226;&#8364;&#8220;5-FOA isolate template, using primers 1 2; <br />                                                lane 4, plasmid shuffle strain template, using primers 3 4; lane 5, URA+ white <br />                                                isolate template, using primers 3 4; lane 6, TRP&#226;&#8364;&#8220;URA isolate template, using <br />                                                primers 3 4. Lane M contains 1 kb ladder. (d) Primer positions SPC110 locus. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions SPC110 green blue, respectively. mutant spc110-227 <br />                                                ORF yellow <br />  <br />  <br />  <br />  <br />                                                      Yeast. Author manuscript; available PMC 2006 December 12. <br />                                Widlund Davis                                                                                                            Page 8 <br />  <br />                                                                                           Table 1 <br />                               Plasmids used study <br />  <br />                            Plasmid                Parent plasmid                       Relevant markers                              Source reference <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            pHS26                  pTD29                        SPC110 LYS2, ADE3, 2 &#206;&#188;m origin             Geiser et al., 1993 <br />                            pHS54                  pHS32                        spc110-227                                 Sundberg Davis, 1997 <br />                            pHS55                  pRS306                       spc110-227                                 Sundberg Davis, 1997 <br />                            pLI831                 DK202                        ADE3                                       Muller, 1996 <br />                            pRS306                                              URA3, f1 origin                            Sikorski Hieter, 1989 <br />                            pRS316                                              CEN6 ARSH4, URA3, f1 origin                Sikorski Hieter, 1989 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br /> </body></html>