<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Metabolite essentiality elucidates robustness Escherichia coli metabolism <br /> Pan-Jun Kim*&#226;&#8364;&#160;, Dong-Yup Lee&#226;&#8364;&#161;&#194;&#167;&#194;&#182;&#229;&#8218;&#168;, Tae Yong Kim*&#194;&#167;&#194;&#182;, Kwang Ho Lee*&#194;&#167;&#194;&#182;**, Hawoong Jeong*&#226;&#8364;&#160;&#226;&#8364;&#161;&#226;&#8364;&#160;&#226;&#8364;&#160;, Sang Yup Lee*&#226;&#8364;&#161;&#194;&#167;&#194;&#182;&#226;&#8364;&#160;&#226;&#8364;&#160;&#226;&#8364;&#161;&#226;&#8364;&#161;, Sunwon Park&#226;&#8364;&#161;&#194;&#182; <br /> *Center Systems Synthetic Biotechnology, Institute BioCentury, &#226;&#8364;&#160;Department Physics, &#226;&#8364;&#161;Bioinformatics Research Center, &#194;&#167;Metabolic Biomolecular Engineering National Research Laboratory, BioProcess Engineering Research Center, &#194;&#182;Department <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> Biomolecular <br /> Engineering (BK21 Program), &#226;&#8364;&#161;&#226;&#8364;&#161;Department BioSystems, Korea Advanced Institute Science Technology, 373-1 Guseong-dong, <br /> Yuseong-gu, Daejeon 305-701, Korea <br />  <br /> Edited Philip P. Green, University Washington School Medicine, Seattle, WA, approved July 2, 2007 (received review April 9, 2007) <br />  <br /> Complex biological systems robust genetic envi-                 various genotypic environmental conditions using ronmental changes levels organization. biological                genome-scale Escherichia coli metabolic model consisting 762 <br /> functions Escherichia coli metabolism sustained               metabolites 932 biochemical reactions (9&#226;&#8364;&#8220;13)  Materials <br /> single-gene multiple-gene mutations using redundant                  Methods). alternative pathways.  limited number genes identified lethal cell. regard, reaction-         Results <br /> centric gene deletion study limitation understanding               Characterization Vivo Validation Metabolite Essentiality. metabolic robustness.  report use flux-sum,                explore robustness E. coli metabolism metabolite summation incoming outgoing fluxes                    perspective, classified intracellular metabolites particular metabolite pseudo-steady state conditions,                categories, essential nonessential metabolites, mon- <br /> good conserved property elucidating robustness E. coli              itoring cell growth consumption rate given metabolite point view. functional behavior,                  metabolite suppressed zero (Materials Methods). structural evolutionary properties metabolites               resultant list essential metabolites given supporting <br /> essential cell survival, investigated means                  information (SI) Table 1 19 different environmental <br /> constraints-based flux analysis perturbed conditions.                 conditions specified different combinations C, P, N, <br /> essential metabolites capable maintaining steady flux-sum              S sources, aerobic vs. anaerobic conditions (SI Table 2). severe perturbation actively redistributing                 results obtained glucose-minimal medium aerobic <br /> relevant fluxes. Disrupting flux-sum maintenance               condition used representative examples.  <br /> suppress cell growth. approach analyzing metabolite es-                 ingly, relatively unstudied metabolites, hexadecenoyl-ACP, <br /> sentiality provides insight cellular robustness concomitant            phosphatidylglycerol, 2-isopropylmaleate, fragility, used applications, including            essential. metabolite essentiality does depend development new drugs treating pathogens.                                environmental conditions 87.8% total essential <br />                                                                                 metabolites essential different growth conditions <br /> cellular robustness &#229;&#8230;&#169; flux-sum &#229;&#8230;&#169; metabolic networks <br />                                                                                 (Fig. 1a). <br />                                                                                    essentiality given metabolite demonstrated                                                                                 vivo means multiple gene disruptions metab-     vailability complete genome sequences       characterized organisms led reconstruction genome-scale metabolic networks, represent complex <br />                                                                                 olite. disrupting multiple nonlethal reactions                                                                                 particular metabolite suppresses cell growth, metabolite                                                                                 regarded essential deletion individual <br /> web metabolites interconversions catalyzed                  reaction itself nonlethal. conducted gene <br /> gene products. Robustness, inherent property metabolic                   deletion experiments neighboring reactions networks, enables maintenance cellular functions                   tetrahydrofolate, identified essential metabolite <br /> various internally externally perturbed conditions.                    silico. reactions (genes), phosphoribosylglyci- <br /> robustness experimentally demonstrated                  namide formyltransferase (purN), <span id='am-91' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-92' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> cleavage disruption considerable portion genes affect              (lpdA), <span id='am-103' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-104' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycine</span> hydroxymethyltransferase (glyA) se- cell viability (1, 2). studies topological              lected disruption targets. single double gene dele- <br /> functional properties metabolic networks achieved                  tion mutant (&#226;&#338;&#172;purN, &#226;&#338;&#172;lpdA, &#226;&#338;&#172;glyA, &#226;&#338;&#172;purN&#226;&#338;&#172;lpdA) able <br /> progress (3&#226;&#8364;&#8220;6), provide limited understanding metabolic robustness. conventional attempt study robustness relies identification genes reactions               Author contributions: P.-J.K. D.-Y.L. contributed equally work; P.-J.K., D.-Y.L., <br />                                                                                 T.Y.K., K.H.L., H.J., S.Y.L., S.P. designed research; P.-J.K., D.-Y.L., T.Y.K., K.H.L. <br /> indispensable cell.  universal metabolic pathways                  performed research; P.-J.K., D.-Y.L., T.Y.K., K.H.L., H.J., S.Y.L., S.P. wrote paper. species, tricarboxylic acid (TCA) cycle                                                                                 authors declare conflict  <br /> glycolytic pathways, relatively lethal reactions (1, 7). <br />                                                                                 article PNAS Direct Submission. fact indicates important reaction                                                                                  Freely available online PNAS open access option. <br /> higher chance backup pathway (7).                                                                                  &#229;&#8218;&#168;Present  addresses: Department Chemical Biomolecular Engineering, National <br /> functionally important reactions necessarily lethal,                                                                                  University Singapore, 4 Engineering Drive 4, Singapore 117576; Bioprocessing point places limitation reaction-centric approach                  Technology Institute, Agency Science, Technology Research  STAR), 20 Biopolis studying lethality observing gene deletion effects.               Way,  06-01, Centros, Singapore. regard, investigated interplay cellular                **Present address: R <span id='am-11' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; D Center Bioproducts, CJ Corporation, Seoul 157-724, Korea. <br /> robustness underlying metabolism metabolite                    &#226;&#8364;&#160;   correspondence addressed. E-mail: hjeong@kaist.ac.kr point view, robustness accomplished                  leesy@kaist.ac.kr. <br /> level metabolites, fundamental entities (4,                article contains supporting information online www.pnas.org/cgi/content  <br /> 8) generated, consumed, recycled metabolic pro-                      0703262104/DC1. <br /> cesses. Constraints-based flux analysis carried                   &#194;&#169; 2007 National Academy Sciences USA <br />  <br />  <br />  <br /> 13638 &#226;&#8364;&#8220;13642 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  34                                                            www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703262104 <br />  Fig. 1. Characteristics essential nonessential metabolites E. coli metabolism.   Metabolic network including central cell envelope <br /> metabolism. Cofactors shown number associated reactions large visual examination. size circle/box <br /> corresponds flux associated metabolite/reaction, thickness line denotes flux line. (b) Experimental measure growth rate relative wild type disrupting genes essential metabolite tetrahydrofolate (blue), nonessential <br /> metabolite 1-deoxy-<span id='am-115' about='obo:IMR_0200169' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-116' property="oboInOwl:id" content="IMR:0200169" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="D-Xylulose 5-phosphate" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="KEGG:C00231" datatype="xsd:string"></span>D-xylulose 5-phosphate</span> (red). deleted genes: P, purN; L, lpdA; G, glyA; PL, purN/lpdA; PLG, purN/lpdA/glyA; DX, dxs/xylB. (c) Distributions <br />                                                                                                                            k&#226;&#171;&#185;&#226;?&#166; degree k essential nonessential metabolites. vertical axis represents P(k) defined fraction degree, &#229;&#8230;&#176;k&#226;&#171;&#186; &#226;?&#166; P(k&#226;&#172;&#732;)dk&#226;&#172;&#732; &#226;&#171;&#189; f(k) f(k) fraction <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              BIOPHYSICS metabolites k &#226;&#171;&#186; &#226;?&#166; k &#226;&#171;&#185; &#226;?&#166;. distributions follow power-law P(k) &#226;&#172;&#8364; k&#226;&#171;&#186;&#226;?&#165; &#226;?&#165; &#226;&#171;&#189; 1.97 essential metabolites &#226;?&#165; &#226;&#171;&#189; 3.06 nonessential <br /> ones. (d) horizontal axis represents number different organisms N, vertical axis represents fraction metabolites conserved <br /> phylogenetically &#226;&#172;&#381;N different organisms. EMC/NMC denotes metabolites E. coli essential/nonessential half growth conditions. majority EMC (66.1%) present organisms (&#226;&#177;&#8211;79.3%), contrary case NMC  21.2% phylogenetic range). <br />  <br />  <br />  survive growth rate changes,                              number reactions (degree) participated simultaneous deletion genes                                        metabolite calculated (5). degree distributions (&#226;&#338;&#172;purN&#226;&#338;&#172;lpdA&#226;&#338;&#172;glyA) prevented cell growth completely, indicat-                           essential nonessential metabolites follow ing tetrahydrofolate essential cell growth                      power-law distribution broad range degrees (Fig. 1c). <br /> (Fig. 1b). contrast, 1-deoxy-D-xylulose 5-phosphate iden-                       surprisingly, degree distribution essential metabolites <br /> tified nonessential metabolite silico, experimental                        right-skewed compared nonessential ones, indicat- <br /> disruption reactions producing metabolite                         ing essential metabolites connected reactions <br /> constructing &#226;&#338;&#172;dxs&#226;&#338;&#172;xylB slightly changed increased                         nonessential ones.  highly connected  growth rate compared wild type (Fig. 1b).                           tabolites essential; include essential molecules <br /> results indicate multiple gene knockout mutants                           cofactors, e., H, <span id='am-44' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span>, <span id='am-33' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>, Pi, <span id='am-5' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span> NAD, reactions essential metabolites suffer                             participate &#226;&#172;&#381;76 reactions. metabolites having detrimental impact cellular function,                          degree &#226;&#172;?3, 34% essential. nonessential metabolites negligible influence growth                         noticed nonessential metabolites mani- <br /> capability. experiments, measured growth                          fest inactive state fluxes metabolites <br /> rates gene deletion mutants relative wild type                   remain zeros. inactive metabolites nonessential consistent silico predictions (Note 1 SI                           metabolites considered, 88.6% active metabolites <br /> Appendix).                                                                             having degree &#226;&#172;?3 essential. <br />                                                                                           loss essential metabolites directly threatens cell <br /> Structural Evolutionary Properties Metabolites.                         viability, expect metabolites essential <br /> investigated inherent network properties essential metab-                       various growth conditions conserved olites elucidate correlation structural prop-                       species. investigated evolutionary conservation erty functional behavior metabolite perspective.                          essential metabolites 227 organisms fully sequenced <br />  <br /> Kim et al.                                                                                                  PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  34 &#229;&#8230;&#169; 13639 <br />  genomes (SI Table 3).  metabolites essential growth conditions E. coli present different <br /> organisms, showing higher degree conservation nonessential ones evolutionary process (Fig. 1d). <br />  <br /> Stability Metabolite Flux-Sum. understand robustness cellular metabolism quantitatively metabolite  <br /> spective, strength fluxes metabolite quantified. end, flux-sum (&#226;&#338;&#189;) metabolite defined summation incoming outgoing fluxes follows: <br />  <br />               &#226;&#171;&#189;   &#229;&#8224;&#732; <br />                     j&#226;?&#167;Pi <br />                            Sijvj &#226;&#171;&#189; &#226;&#171;&#186;     &#229;&#8224;&#732; <br />                                          j&#226;?&#167;Ci <br />                                                 Sijvj &#226;&#171;&#189; <br />                                                           1 <br />                                                           2 <br />                                                               &#229;&#8224;&#732; <br />                                                               j <br />                                                                   &#229;&#8230;&#169;Sijvj&#229;&#8230;&#169;, <br />  <br />  Sij stoichiometric coefficient metabolite reaction j, &#226;?&#175;j flux reaction j. Pi denotes set reactions producing metabolite  Ci denotes set reactions consuming metabolite  pseudo-steady state <br /> assumption,  mass flow contributed fluxes <br /> producing (consuming) metabolite  <br />   robustness E. coli metabolism examined deter- <br /> mining sensitivity flux-sum genetic perturbation given metabolite. quantified evaluating relative fluctuation  response deletion active <br /> nonlethal reactions: <br />  <br />                              &#229;&#8224;&#8216;&#229;&#8230;&#183;&#226;&#338;&#189;i2&#229;&#8230;&#184; &#226;&#171;&#186; &#229;&#8230;&#183;&#226;&#338;&#189;i&#229;&#8230;&#184;2/&#229;&#8230;&#183;&#226;&#338;&#189;i&#229;&#8230;&#184; &#229;&#8230;&#183;. . .&#229;&#8230;&#184; denotes average deletion active <br /> nonlethal reactions. noted interested trivial cases deletion inactive reactions. low relative fluctuation values, number essential metab- <br /> olites greater nonessential metabolites <br /> (Fig. 2a). result indicates flux-sums essential <br /> metabolites relatively insensitive genetic perturbation <br /> compared nonessential ones.  94.3% total metabolites fluctuation range &#226;&#172;?0.0875              Fig. 2. Stability metabolite flow genetic perturbations.   ratio essential, nonessential metabolites            essential metabolites metabolites function flux-sum fluctuation. highest ranked ones relative fluctuations.             (b) Flux-sum fluctuation versus flux-vector fluctuation essential/ concluded essential metabolites resistant internal            nonessential metabolites. (c d) number essential metabolites (c) variations maintaining basal mass flow correspond-             nonessential ones (d) metabolite takes relative change flux-sum <br />                                                                              &#229;&#8230;&#169;&#226;&#338;&#172;&#226;&#338;&#189;i&#229;&#8230;&#169;/ active nonlethal reaction j maximum contribution ing metabolite, leading robustness cellular <br />                                                                              flux-sum (maximum &#229;&#8230;&#169;Sij&#226;?&#175;j&#229;&#8230;&#169;/  removed. cases   metabolism.                                                                  diagonal indicate additional flux compensation (loss) reactions. <br />    mechanism contribute resistance                     (e) Illustration neighboring reactions <span id='am-135' about='obo:IMR_0200109' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-136' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Carbamoyl phosphate" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:id" content="IMR:0200109" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="KEGG:C00169" datatype="xsd:string"></span>carbamoyl phosphate</span> (cbp). essential metabolites internal perturbations? explicitly               flux-sum carbamoyl phosphate compensated carbamoyl- <br /> tackle question, monitored individual flux values                phosphate synthase (CBPS) reaction highest flux, carbamate essential metabolites genetic perturbations.                 kinase (CBMK), removed. Thickness arrow represents defined flux-vector (&#226;&#338;&#191;  &#225;&#382;&#8225; ) metabolite collection           flux, shown reaction. <br />                                                         &#225;&#382;&#8225; &#226;&#171;&#189; {Sij&#226;?&#175;j}, <br /> individual fluxes Sij&#226;?&#175;j linked reactions j, &#226;&#338;&#191; evaluated flux-vector fluctuation, represents relative deviation flux values given metabolite            nonessential ones. clarify resistance deleting reactions:                                                     essential metabolites internal perturbations, severe pertur- <br />                                                                              bation conducted deleting reaction contributes <br />                            &#229;&#8224;&#8216;&#229;&#8230;&#183;&#229;&#8230;&#169;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#169;2&#229;&#8230;&#184; &#226;&#171;&#186; &#229;&#8230;&#169;&#229;&#8230;&#183;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#184;&#229;&#8230;&#169;2/&#229;&#8230;&#169;&#229;&#8230;&#183;&#226;&#338;&#191;&#225;&#382;&#8225; i&#229;&#8230;&#184;&#229;&#8230;&#169;                   flux-sum given essential metabolite.                                                                              noted deleted reaction active nonlethal reaction linked &#229;&#8230;&#169;. . .&#229;&#8230;&#169; denotes magnitude given vector. Apparently,       metabolite. Fig. 2 c d shows effects kind observed variation relevant fluxes metabolite directly         severe perturbation flux-sum changes essential contributes change flux-sum metabolite. Scatter        nonessential metabolites, respectively. essential metabolites <br /> plot flux-sum fluctuation corresponding                  located diagonal, indicating extent flux-vector fluctuation nonessential metabolites clearly shows         flux-sum change flux loss caused deleting linear relationship, indicating flux-sum metabolites         contributing reaction. Accordingly, reaction having affected perturbed variations individual fluxes (Fig.        relatively high flux eliminated, flux-sum compen- <br /> 2b). hand, considerable number essential                sated fluxes essential metabolite, recovering <br /> tabolites small fluctuation flux-sums despite          flux loss immediately. Remarkably, essential metab- increased fluctuations individual flux values (Fig. 2b).          olites, flux loss recovered fluxes result implies flux-sums essential metabolites          reactions, resulting small change flux-sum, affected flux variations  compared               dominant reaction high flux value (6)  <br /> 13640 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0703262104                                                                                                 Kim et al. <br />  Fig. 3. Phenotypic effect attenuating flux-sum level essential metabolites.   Changes cell growth rate flux-sum continuously decreases. growth rate flux-sum scaled relative wild type. growth profile type case metabolite, phosphatidylglycero- <br /> phosphate, type B <span id='am-23' about='obo:IMR_0200133' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Oxidized thioredoxin" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200133" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00343" datatype="xsd:string"></span>oxidized thioredoxin</span>, type C <span id='am-49' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-50' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-55' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span>adenosine</span> diphosphate. (b c) Cell growth rate relative wild type reducing flux-sum half, shown function basal flux-sum level attenuated essential metabolites. results glucose-minimal aerobic condition <br /> (b) 19 environmental conditions (c) presented. metabolites b colored way according growth profile. metabolites belong type  refer Note 4 SI Appendix. <br />  <br />  <br />  <br /> removed (Fig. 2c). metabolites include carbamoyl phosphate,                          basal flux-sums, type B high basal flux-sums, <span id='am-120' about='obo:IMR_0200008' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-121' property="oboInOwl:hasDbXref" content="KEGG:C00365" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:id" content="IMR:0200008" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="dUMP" datatype="xsd:string"></span>dUMP</span>, <span id='am-125' about='obo:IMR_0200494' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-126' property="rdfs:label" content="CMP" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:id" content="IMR:0200494" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="KEGG:C00055" datatype="xsd:string"></span>CMP</span>, <span id='am-79' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-80' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span>L-glutamate</span> 5-semialdehyde (Note 3 SI                                   type C ultra-high basal flux-sums (Fig. 3 b c). <br /> Appendix). example, carbamoyl phosphate key metabolite                           turns 83.8% essential metabolites belong type  <br /> involved arginine proline metabolism purine                             metabolites adjust cell growth rate proportionally pyrimidine biosynthesis. flux-sum carbamoyl phosphate                              flux-sum, act like acclimators affecting cell maintained alteration fluxes largest                        growth; acclimator metabolites allow cell growth rate flux reaction catalyzed carbamate kinase blocked                             finely adjusted flux-sums, provide <br /> completely (Fig. 2e); carbamoyl-phosphate syn-                          effective control cell growth. classification essential <br /> thase compensate large flux loss caused knocking-                        metabolites according growth profile flux-sum carbamate kinase, resulting recovery 98.9%                           attenuation described Note 4 SI Appendix. <br /> basal flux-sum. regulation carbamoyl-phosphate syn- <br /> thase response deletion carbamate kinase actually                         Discussion <br /> inferred gene expression profile data (Note 3 SI                              functional robustness metabolic networks outcome <br /> Appendix). results suggest maintenance                              long evolutionary process reflects resistance flux-sum serve good indicator metabolic robustness.                      internal external fluctuations (14&#226;&#8364;&#8220;17). example, fact motivated predict efficiently candidate reactions                    existence alternative pathways flux redistributions implies <br /> activated flux-sum recovery severe gene knockout                        backup pathways possibly activated perturbations.  using stoichio-similarity, developed                        perform function various genetically envi- algorithm predict probable reaction                            ronmentally perturbed conditions (3, 18). fault-tolerance <br /> recover flux-sum gene knockout perturbation (Note 3                         robustness key cell survival SI Appendix).  believe cellular robustness                       perturbations. regard, metabolite-based perspective elucidated functional property metabolic network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      BIOPHYSICS <br />                                                                                           provide new guidelines interpreting cellular robustness. <br /> manifesting resilience essential metabolites                           Essential metabolites substantial cell survival capable disturbed flux conditions.                                                                rerouting metabolic fluxes sustaining usage level. <br />                                                                                           capability essential metabolites leads quite dra- <br /> Attenuation Metabolite Flux-Sum. Essential metabolites play                          matic tolerance wide range internal disturbances. pivotal role cell survival, steadily maintaining mass flow                         possible essential metabolites produce consume metabolites internal                                characteristics presented  inaccuracy arise  <br /> disturbances cell. sense, metabolite                             cause regulatory mechanisms fully considered perspective robustness E. coli provides cellular-level                          current analysis. examined effects fragility: failure maintain flux-sum single essential                     incorporating limited number regulatory mechanisms dur- <br /> metabolite drastically suppress cellular growth.                            ing study, shown difference (Note 1 malfunction multiple genes metabolite                                 SI Appendix). expected better classification essential <br /> cause critical decrease flux-sum. Especially,                        nonessential metabolites performed genome- <br /> essential metabolites (85%), reducing flux-sum half                         wide regulatory mechanisms incorporated genome- <br /> basal level led suppression growth rate half                        scale flux analysis.  28.9% active nonessential metabolites                                  number applications envisaged using concept <br /> showed behavior.                                                                     metabolite essentiality. example, used develop <br />    effects reducing flux-sum cell growth                               metabolic engineering strategies enhanced production  <br /> examined  flux-sum gradually decreased, <br />                                                                                           sired bioproducts suitably implementing desired flux values. essential metabolite exhibited characteristic profile                                                                                           used identify new drug targets. Disruption <br /> cell growth rate, belonged types:  B,                                                                                           (knockout) multiple nonlethal reactions (genes) C, Fig. 3a.&#194;&#167;&#194;&#167; growth rate sensitive extent                                                                                           essential metabolite lead fatal cell damage flux-sum reduction types C, type <br />                                                                                           attenuation (knockdown) reactions B. characteristic growth rate correlated <br />                                                                                           effect. case treating superbacteria resistant basal flux-sum values; metabolites type low <br />                                                                                           multiple antibiotics, design drugs inhibit enzymes <br />                                                                                           catalyzing multiple nonlethal reactions essential metab- <br /> &#194;&#167;&#194;&#167;Exceptionally, <br />                <span id='am-18' about='obo:IMR_0200466' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Ubiquinol" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00390" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200466" datatype="xsd:string"></span>ubiquinol</span>-8, <span id='am-28' about='obo:IMR_0200483' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00399" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200483" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Ubiquinone" datatype="xsd:string"></span>ubiquinone</span>-8, L-malate exhibit different growth   olite. Alternatively, synthetic lethal mutations (7, 19, 20)  profile termed type D. details, refer Note 4 SI Appendix.              systematically identified enzymes various screening <br />  <br /> Kim et al.                                                                                                 PNAS &#229;&#8230;&#169; August 21, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  34 &#229;&#8230;&#169; 13641 <br />  techniques (21), implemented siRNA anti-sense                  linked reactions producing metabolite M techniques.                                                                     consistently taken account, preventing phenotypic effect <br />                                                                                 irrelevant essentiality given metabolite M. Materials Methods                                                           scaled resultant change cell growth rate relative Constraints-Based Flux Analysis. genome-scale silico E. coli             growth rate wild type calculating essentiality metabolic model iJR904 used slight modifications                       metabolite. reactions metabolite based publicly available information databases                       inactive specific growth condition, considered  <br /> (9&#226;&#8364;&#8220;12); consists 762 metabolites (including external metab-               tabolite nonessential. essentiality metab- <br /> olites) 932 biochemical reactions (including transport pro-                 olites follows clear bimodal distribution (SI Fig. 5), essential <br /> cesses). Cell growth quantified biomass equation                       metabolite easily identified absence leads derived drain biosynthetic precursors relevant                  decrease cell growth rate half wild type, <br /> cofactors E. coli biomass appropriate ratios (13).                absence nonessential metabolite minimal stoichiometric relationships metabolites                      effect cell growth. tried criteria reactions genome-scale silico E. coli model                      essential/nonessential metabolites according essentiality <br /> balanced steady-state hypothesis. resultant bal-                  did difference. <br /> anced reaction model   underdeter- <br /> mined calculating flux distribution insufficient              Construction Gene Knockout Mutant Strains. Mutant strains measurements constraints.  unknown fluxes                       constructed step gene inactivation method (24). metabolic reaction network calculated linear                 wild-type E. coli W3110 strain transformed pKD46 programming-based optimization objective function                    contains phage &#226;?&#173; recombination  E. coli W3110 cells <br /> maximizing growth rate, subject constraints pertaining               carrying pKD46 transformed electroporation mass conservation, reaction thermodynamics, capacities                   PCR product produced using plasmid pKD3 follows:                                                                     pKD4 templates. PCR product 50- 56-bp <br />                       &#229;&#8224;&#732; <br />                       j&#226;?&#167; J <br />                              Sijvj &#226;&#171;&#189; bi,   &#226;?&#163; j &#226;&#177;&#8226; v j &#226;&#177;&#8226; &#226;?&#164; j, <br />                                                                                 homology upstream downstream DNA immediately <br />                                                                                 adjacent specific target gene knocked                                                                                  contained Flp recombinase target site (FRT). Recombinant Sij represents stoichiometric coefficient metabolite               strains selected growing cells presence chlor- reaction j, &#226;?&#175;j flux reaction j, J set reactions,         amphenicol kanamycin, inserted cassette elimi- bi net transport flux metabolite  metabolite            nated using helper plasmid pCP20. knockout mutant intermediate, bi zero. &#226;?&#163;j &#226;?&#164;j lower                  confirmed PCR analysis using primers upper bounds flux reaction j, respectively.                region gene deletion. knockout mutants flux irreversible reaction considered positive:                 grown Luria-Bertani (LB) broth LB agar plates 37&#194;&#176;C. negative flux signifies reverse direction reaction. intracellular fluxes quantified elucidate robust-               Measurement Specific Growth Rate. strains grown M9 <br /> ness E. coli metabolism response genetic perturbations                 minimal medium containing 5 g/liter glucose determine various environmental conditions (SI Table 2).                    growth kinetics using Microbiology Reader Bioscreen C <br /> performed simulation additional regulatory constraints                 analyzer (Oy Growth Curves AB  Helsinki, Finland).  <br /> (22) optimization scheme, MOMA (minimization                     tailed procedures described Note 1 SI Appendix. <br /> metabolic adjustment) (23), qualitative difference results presented                                                 thank C.-M. Ghim, S. B. Sohn, H. U. Kim, J. S. Yang, H. Yun <br />                                                                                 providing valuable information linguistic advice S. T. Kim <br /> Characterization Metabolite Essentiality. metabolite essen-              J.-H. Pak illustrative assistance. pKD46 plasmid used                                                                                 gene-disruption experiments kindly provided B. L. Wanner <br /> tiality defined phenotypic effect metabolite M               (Purdue University, West Lafayette, . work supported cell growth consumption rate set zero. fluxes             Korean Systems Biology Program Grant M10309020000-03B5002-00000 metabolite M restricted produce                Ministry Science Technology LG Chem Chair <br /> metabolite, balancing constraint mass conservation                 Professorship. H.J. acknowledges financial support Korea relaxed allow nonzero values incoming fluxes               Science Engineering Foundation (KOSEF) Grant R17- outgoing fluxes limited zero.  metabolites             2007-073-01001-0. <br />  <br />  <br />  1. Gerdes SY, Scholle MD, Campbell JW, Bala &#194;&#180;zsi G, Ravasz E, Daugherty MD,    13. Neidhardt FC, Curtiss R, Ingraham JL, Lin ECC, Low KB, Magasanik B, <br />     Somera AL, Kyrpides NC, Anderson  Gelfand MS, et al. (2003) J Bacteriol       Reznikoff W, Riley M, Umbarger (1996) Escherichia coli Salmonella <br />     185:5673&#226;&#8364;&#8220;5684.                                                                  (ASM Press, Washington, DC). <br />  2. Goebl MG, Petes TD (1986) Cell 46:983&#226;&#8364;&#8220;992.                                  14. Wagner (2005) Robustness Evolvability Living Systems (Princeton Univ <br />  3. Papp B, Pal C, Hurst LD (2004) Nature 429:661&#226;&#8364;&#8220;664.                              Press, Princeton). <br />  4. Raymond J, Segre` D (2006) Science 311:1764&#226;&#8364;&#8220;1767.                           15. Stelling J, Sauer U, Sz <br /> </body></html>