<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The missing piece type II fatty acid synthase Mycobacterium tuberculosis <br /> Emmanuelle Sacco*, Adrian Suarez Covarrubias&#226;&#8364;&#160;, Helen M. O&#226;&#8364;&#8482;Hare&#226;&#8364;&#161;, Paul Carroll&#194;&#167;, Nathalie Eynard*, T. Alwyn Jones&#226;&#8364;&#160;, <br /> Tanya Parish&#194;&#167;, Mamadou Daffe&#194;&#180;*, Kristina Ba&#194;&#168;ckbro&#226;&#8364;&#160;, Anna&#196;&#177;&#194;&#168;k Que&#194;&#180;mard*&#194;&#182; <br />  partement des canismes Mole&#194;&#180;culaires des Infections Mycobacte&#194;&#180;riennes, Institut Pharmacologie et Biologie Structurale, Centre National la <br /> Recherche Scientifique, 31077 Toulouse, France; &#226;&#8364;&#160;Department Cell Molecular Biology, Uppsala University, Biomedical Center, SE-751 24 Uppsala, <br /> Sweden; &#226;&#8364;&#161;Institute <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Sciences Engineering, Ecole Polytechnique Fe rale Lausanne, CH-1015 Lausanne, Switzerland; &#194;&#167;Centre Infectious Disease, Institute Cell Molecular Science Barts London, London E1 2AT, United Kingdom <br />  <br /> Edited Christian R. H. Raetz, Duke University Medical Center, Durham, NC, approved July 20, 2007 (received review 4, 2007) <br />  Mycobacterium tuberculosis fatty acid synthase type II (FAS-II)             metabolic pathway represents valuable source potential unique property producing unusually long-chain                new pharmacological targets (2). <br /> fatty acids involved biosynthesis mycolic acids, key molecules          Isoniazid inhibits elongation process leading  tubercle bacillus. enzyme(s) responsible dehydration          mation main (meromycolic) chain MAs. steps <br /> (3R)-hydroxyacyl-ACP elongation cycles mycobac-               elongation cycles monitored acyl carrier protein <br /> terial FAS-II remained unknown. step classically catalyzed           (ACP)-dependent FA synthase type II (FAS-II) (5). <br /> FabZ- FabA-type enzymes bacteria, proteins               FAS-II systems plants, bacteria, parasites, present mycobacteria. Bioinformatic analyses essentiality             mitochondria, usually perform novo biosynthesis <br /> study allowed identification candidate protein cluster,                (6).  Corynebacterineae unique <br /> Rv0635-Rv0636-Rv0637. expression recombinant Escherichia coli            elongates long-chain FAs (C12&#226;&#8364;&#8220;C18) long- <br /> strains leads formation heterodimers, Rv0635-Rv0636               chain FAs (5). mycobacteria, enzymes catalyzing (HadAB) Rv0636-Rv0637 (HadBC), occurs Mycobac-                elongation steps characterized [supporting  <br /> terium smegmatis, shown split-Trp assays. heterodimers               formation (SI) Fig. 7]: &#226;?&#164;-ketoacyl-ACP synthetases KasA <br /> exhibit enzymatic properties expected mycobacterial FAS-II              KasB (7, 8), &#226;?&#164;-ketoacyl-ACP reductase MabA (9), dehydratases: marked specificity long-chain (&gt;C12)               trans-2-enoyl-ACP reductase InhA (10).   <br /> ACP-linked substrates. Furthermore, function 3-hydroxyacyl              zyme(s) involved step cycle, corresponding dehydratases coupled MabA InhA enzymes                   (3R)-hydroxyacyl-ACP dehydratase(s)  ,  <br /> M. tuberculosis FAS-II  HadAB HadBC long-sought              known. peptidic domains proteins carrying function <br /> (3R)-hydroxyacyl-ACP dehydratases. correlation                  multifunctional synthases FAS-II systems substrate specificities enzymes, organization            generally difficult identify (11, 12). <br /> thologous gene cluster different Corynebacterineae, struc-             FAS-II systems, activity classically catalyzed ture mycolic acids suggests distinct roles het-               FabZ (dehydratase) FabA (dehydratase&#226;&#8364;&#8220;isomerase) enzymes <br /> erodimers elongation process. work describes                    (6). previously shown FabZ/FabA-type <br /> bacterial monofunctional (3R)-hydroxyacyl-ACP dehydratases                   protein present tubercle bacillus (13). indicated <br /> longing hydratase 2 family. original structure fact        HADs mycobacteria present different essential M. tuberculosis survival make              distantly related catalytic sequence motif. investigation zymes good candidates development antimycobacte-                Mtb genome using bioinformatic tools, pool 11 putative <br /> rial drugs.                                                                     (R)-specific enoyl hydratases/3-hydroxyacyl dehydratases                                                                                 identified (13, 14). Structural determination modeling (3R)-hydroxyacyl-ACP dehydratase &#229;&#8230;&#169; hydratase 2 &#229;&#8230;&#169; mycolic acid biosynthesis &#229;&#8230;&#169;    (13&#226;&#8364;&#8220;15) suggested belong fatty acid elongation &#229;&#8230;&#169; hot dog fold                                            related hydratase 2 protein family, underlying <br />                                                                                 3D structure FabA type enzymes maintained                                                                                 called hot dog fold catalytic site distinct (16). T   uberculosis kills nearly million people year  e., <br />     person 20 seconds). Emergence drug-resistant <br /> Mycobacterium tuberculosis (Mtb) main concerns                                                                                 family members, Rv0636, proposed                                                                                 candidate FAS-II basis essentiality treatment disease; prevalence cases resistant             prediction (14). purpose present work use main drugs (isoniazid, rifampicin, streptomycin, etham-                complementary experimental strategies determine  butol) high regions world  30% previously treated cases).  development new drugs <br />                                                                                 Author contributions: E.S. S.C. contributed equally work; H.M.O., T.P., K.B., <br /> effective resistant strains priority (1).              Q. designed research; E.S., S.C., H.M.O., P.C., N.E., T.P., K.B., Q. performed <br />    Mycobacteria possess <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span>-rich envelope,                 research; H.M.O. contributed new reagents/analytic tools; E.S., S.C., H.M.O., T J., T.P., <br /> typically contains long-chain (C60&#226;&#8364;&#8220;C90) &#226;?&#163;-alkylated &#226;?&#164;-                     M.D., K.B., Q. analyzed data; T J., T.P., M.D., Q. wrote paper. <br />  <br /> hydroxylated fatty acids (FAs) called mycolic acids (MAs) (2).                  authors declare conflict  original lipids universally suborder                  article PNAS Direct Submission. <br /> Corynebacterineae (including Mycobacterium, Corynebacterium,                    Abbreviations: ACP, acyl carrier protein; FA, fatty acid; FAS, FA synthase;  (3R)- <br /> Rhodoccocus, Nocardia, ) genus holds molecules                   hydroxyacyl-ACP dehydratase; H-HadA, tagged HadA; H-HadB, tagged HadB; MA, <br />                                                                                 mycolic acid; Mtb, M. tuberculosis. specific chain lengths. MAs crucial architecture               correspondence addressed  Institut Pharmacologie et permeability mycobacterial envelope. play                   Biologie Structurale, Centre National la Recherche Scientifique, 205 Route Nar- role virulence persistence tubercle bacillus             bonne, 31077 Toulouse Cedex, France. E-mail: annaik.quemard@ipbs.fr. infected organisms (3, 4). biosynthetic pathway,                   article contains supporting information online www.pnas.org/cgi/content  essential survival mycobacteria, target              0704132104/DC1. frontline antituberculous drug, isoniazid.                  &#194;&#169; 2007 National Academy Sciences USA <br />  <br />  <br />  <br /> 14628 &#226;&#8364;&#8220;14633 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  37                                                        www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104 <br />  Table 1. silico analysis putative (R)-specific hydratases/dehydratases Mtb <br />                 Size, Hydratase 2                         Ubiquity                                                   Nocardia       Known predicted <br /> Protein         kDa     motif*           Proximity&#226;&#8364;&#160;        mycobacteria&#226;&#8364;&#161; Corynebacterium&#194;&#167; Rhodococcus sp.&#194;&#167;                 farcinica&#194;&#167;         structure&#194;&#182; <br />  <br /> Rv0130          16.0        &#226;&#171;&#185;               fbpC                 &#226;&#171;&#186;            NCgl0284,&#229;&#8218;&#168; 44%        Ro03023, 60%        Nfa34100, 58%              SHD <br /> Rv0216          35.8        &#226;&#171;&#186;                                    &#226;&#171;&#185;                                  Ro02873, 68%                                   DHD <br /> Rv0241c         30.2       &#226;&#171;&#185;&#226;&#171;&#185;            fabG4                   &#226;&#171;&#185;             JK0821,** 42%        Ro05198 , 51%       Nfa54620, 55%              DHD <br /> Rv0504c         18.4        &#226;&#171;&#185;            cmaA2                   &#226;&#171;&#185;                                                                                 SHD <br /> Rv0635 HadA     17.4        &#226;&#171;&#186;         mmaA1-mmaA4                &#226;&#171;&#185;                                   Ro01984, 53%       Nfa51180, 53%              SHD <br /> Rv0636 HadB     14.9       &#226;&#171;&#185;&#226;&#171;&#185;         mmaA1-mmaA4                &#226;&#171;&#185;                                   Ro01983, 63%       Nfa51180, 63%              SHD <br /> Rv0637 HadC     18.9        &#226;&#171;&#186;         mmaA1-mmaA4                &#226;&#171;&#185;                                                                                 SHD <br /> Rv2499c         20.2        &#226;&#171;&#186;                                    &#226;&#171;&#186;             JK1548,** 75%         Ro06098, 76%       Nfa50400, 78%              SHD <br /> Rv3389c         30.3       &#226;&#171;&#185;&#226;&#171;&#185;              cmaA1                 &#226;&#171;&#186;                                                                                 DHD <br /> Rv3538          30.2       &#226;&#171;&#185;&#226;&#171;&#185;                                    &#226;&#171;&#186;                                   Ro04531, 62%       Nfa4620, 59%               DHD <br /> Rv3542c         33.9       &#226;&#171;&#185;                                     &#226;&#171;&#186;                                  &#229;&#8218;&#168;Ro04486, 63%       Nfa4510, 65%               DHD <br />  <br /> *&#226;&#171;&#185;&#226;&#171;&#185;, strictly conserved motif &#229;&#8230;&#179;YF&#229;&#8230;&#180;-x(1,2)-&#229;&#8230;&#179;LIVG&#229;&#8230;&#180;-&#229;&#8230;&#179;STGC&#229;&#8230;&#180;-G-D-x-N-P-&#229;&#8230;&#179;LIV&#229;&#8230;&#180;-H-x(5)-&#229;&#8230;&#179;AS&#229;&#8230;&#180;; &#226;&#171;&#185;, motif including conserved basic catalytic motif &#229;&#8230;&#179;D&#229;&#8230;&#180;-x(4)-H; &#226;&#171;&#186;, <br />  absent. <br /> &#226;&#8364;&#160;Presence ORF(s) demonstrated putatively involved FA/MA biosynthesis transfer. <br /> &#226;&#8364;&#161;&#226;&#171;&#186;, absent present pseudogene; &#226;&#171;&#185;, conserved considered sequenced mycobacterial genomes  SI Materials Methods). <br />  orthologous putative protein percentage identity given BlastP alignment &#229;&#8230;&#179;on (quasi-)totality sequence&#229;&#8230;&#180;. following <br />  <br />  genomes analyzed: Rhodococcus sp. RHA1, Nocardia farcinica IFM10152, Corynebacterium diphtheriae NCTC13129, Corynebacterium glutamicum ATCC <br />  13032, Corynebacterium jeikeium K411, Corynebacterium efficiens YS-314. <br /> &#194;&#182;SHD, single hot dog fold [known (underlined) SHD, ref. 15]; DHD, double hot dog fold [known (underlined) OHO, ref. 14]. <br /> &#229;&#8218;&#168;Detected C. glutamicum. <br />  <br /> **Detected C. jeikeium. <br />  <br />  <br />   putative proteins missing piece                 tity, poorly related Rv0636 (13&#226;&#8364;&#8220;15% <br /> FAS-II puzzle.                                                                    identity) (SI Fig. 8), suggesting Rv0635 Rv0637                                                                                   similar role, distinct Rv0636. <br /> Results <br /> Selection Candidates Silico Analyses. select              Essentiality Rv0635-Rv0637 Cluster. used step <br /> silico candidates dehydration step FAS-II cycle                    homologous recombination strategy demonstrate putative (R)-specific hydratases/dehydratases previ-                    Rv0635-Rv0637 gene cluster essential vitro. deletion <br /> ously identified Mtb (13, 14), defined different criteria              delivery vector entire operon deleted (Table 1):   ubiquity protein mycobacteria                     constructed. addition, merodiploid strain constructed related genera (Nocardia Rhodococcus), (ii) absence                   entire operon integrated using L5-based Corynebacterium devoid FAS-II  Search                    vector. Attempts generate unmarked deletion HadA, HadB, HadC Orthologs Corynebacteri-                        genes merodiploid WT backgrounds <br /> neae), (iii) occurrence catalytic motifs, (iv)                     (SI Fig. 9). unable isolate deletion <br /> proximity chromosome genes potentially involved                      mutants. double crossover (DCO) strains analyzed FA/MA biosynthesis transfer.                                                   PCR, WT gene. contrast, merodiploid <br />    proteins genes present myco-                    background able delete normal chromosomal <br /> bacterial genomes sequenced far, Rv0216 obvious                         copy; DCOs WT (integrated) catalytic motif (14), Rv0504c ortholog Rhodococ-                    deletion alleles (SI Fig. 9), confirmed Southern blot <br /> cus Nocardia (Table 1). contrast, Rv0241c Rv0636                    analysis (data shown). provided direct evidence represented genera. bear characteristic                  genes Rv0635-Rv0637 cluster essential vitro. conserved catalytic sequence, called &#226;&#8364;&#732;&#226;&#8364;&#732;hydratase 2 motif&#226;&#8364;&#8482;&#226;&#8364;&#8482;               contrast, Rv0241c deletion easily obtained WT <br /> (Table 1 SI Fig. 8), includes catalytic residues                     background using methodology (T.P., unpublished <br /> D-x(4)-H (13, 14) observed hydratase 2 family (16).                     results). MAs components essential mycobac-  Rv0241c adjacent ORF encoding putative                       terial growth, gene(s) encoding s) FAS-II <br /> &#226;?&#164;-ketoacyl-ACP reductase (FabG4) distinct FAS-II,                    likely essential, like kasA, mabA, inhA <br /> MabA (FabG1), Rv0636 located near mmaA1-4 gene                         (18&#226;&#8364;&#8220;20).  data consistent Rv0635- <br /> cluster involved biosynthesis oxygenated MAs (3).                       Rv0636-Rv0637 cluster best candidate long-  Rv0241c ortholog Corynebacte-                        sought dehydratase. greater convenience, proteins                                                                                                                                                                     BIOCHEMISTRY <br />  <br />  <br />  <br /> rium species (Table 1).  Rv0636 satisfies selection                called HadA, HadB, HadC, respectively. <br /> criteria represents best candidate.  Mtb <br /> chromosomes, Rv0635 overlaps Rv0636 Rv0637 4 bp away                       Expression Purification HabA-C Proteins: Analysis Rv0636, implying genes form operon.                      Quaternary Structure Vitro. HadB gene hadA-hadB-hadC structural prediction data Rv0636 (14), Rv0635,                       hadB-hadC gene clusters cloned TOPO expression <br /> Rv0637 (Table 1) suggested putative proteins                   vector downstream tag coding sequence. Proteins 15&#226;&#8364;&#8220;19 kDa composed single hot dog fold.                          produced Escherichia coli purified using step <br /> Proteins topology form dimers                      chromatography procedure Ni Sepharose Superdex 75 <br /> basic structural motif) (16, 17). Rv0635 Rv0637 lack                  columns. noteworthy  hadB expressed  hydratase 2 catalytic motif (Table 1 SI Fig. 8),                       tagged product  tagged HadB (H-HadB)] <br /> hypothesized Rv0636 protein needs associate het-                       insoluble, purified protein relatively unstable erodimers Rv0635 Rv0637 functional.                    solution. contrast, hadA-hadB-hadC clone, significant <br /> noteworthy Rv0635 Rv0637 share 45% sequence iden-                        protein isolated. Strikingly, tagged <br />  <br /> Sacco et al.                                                                                       PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  37 &#229;&#8230;&#169; 14629 <br />  Fig. 1. Protein purification. SDS/PAGE gel filtration chromatograms. cloned genes mentioned. Lane  total soluble proteins; lane b, Ni Sepha- <br /> rose fraction; lanes c d, Superdex 75 fractions. Monomeric sizes pro- <br /> teins: H-HadA, 18.3 kDa; HadB, 14.8 kDa; H-HadB, 15.9 kDa; HadC, 18.9 kDa. <br />  <br />  <br />  <br /> HadA (H-HadA) copurified untagged HadB, presence HadC undetectable (Fig. 1).  H-HadB HadC coproduced, H-HadB coeluted untagged HadC (Fig. 1). data strongly suggested interactions HadA HadB HadB HadC. <br />   gel filtration chromatography, performed 4&#194;&#176;C, pool Ni-column fractions containing H-HadB HadC <br /> displayed profile major peak estimated 65 kDa minor peak &#226;&#172;&#8225;33 kDa (Fig. 1). Analyses SDS/PAGE (Fig. <br /> 1) <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> mapping demonstrated elution peak <br /> corresponded equimolar amounts H-HadB HadC, second corresponded H-HadB  similar <br /> chromatogram obtained fractions containing H- <br />                                                                                  Fig. 3. Chain length specificity profiles HadAB HadBC heterodimers. <br /> HadA HadB, major peak  65 kDa) holding                                                                                  Assays performed fixed concentrations substrate enzyme.  proteins minor peak holding H-HadA (Fig. 1).                        B) HadAB 2.5 &#226;?&#174;M enoyl-<span id='am-32' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span>   25 &#226;?&#174;M enoyl-CoA (B). (C) HadBC <br />   Data suggested heterocomplexes Rv0635-Rv0636                          25 &#226;?&#174;M enoyl-CoA. Data means &#226;&#171;&#190; SD. activities measured (HadAB) Rv0636-Rv0637 (HadBC) formed tetramers.                          heterodimers presence short chain substrates (C4 C8)                                                                                  significantly control values  enzyme). large SDs ob- <br />                                                                                  tained long-chain substrates solubility problems. <br />  <br />  <br />  <br />                                                                                  quaternary structure HadAB examined dy- <br />                                                                                  namic light scattering, confirmed tetramer                                                                                  low temperature (8&#194;&#176;C) (SI Table 2).  higher <br />                                                                                  temperature (20&#194;&#176;C) behaved dimer. <br />  <br />                                                                                  Study Protein&#226;&#8364;&#8220;Protein Interactions Vivo Using Split-Trp <br />                                                                                  Method. investigate occurrence interactions                                                                                  proteins encoded hadA-C cluster mycobacteria vivo, <br />                                                                                  split-Trp method used Mycobacterium smegmatis. <br />                                                                                  Briefly, tryptophan auxotrophic &#226;&#338;&#172;hisA strain M. smegmatis <br />                                                                                  cotransformed pair plasmids, encoding                                                                                  target protein fusion N-terminal fragment <br />                                                                                  (Ntrp) C-terminal fragment (Ctrp) Saccharomyces <br />                                                                                  cerevisiae protein Trp1p involved Trp biosynthesis. vivo <br /> Fig. 2. Split-Trp growth assay M. smegmatis. Recombinant M. smegmatis         interaction occurs target proteins encoded &#226;&#338;&#172;hisA-expressing pairs proteins fused Ntrp Ctrp diluted 1-                                                                                  pair plasmids, Trp1p function restored 10-fold water spotted parallel minimal medium medium <br /> plus Trp. pair Ntrp-Esat6 Cfp10-Ctrp used positive control <br />                                                                                  recombinant strain grow exogenous Trp supply.  early secreted T cell antigens Esat6 Cfp10 Mtb form tight, 1:1   experiments, heterotypic interactions  25&#194;&#176;C) ob- <br /> complex). Negative controls extra positive controls performed       served M. smegmatis HadA HadB SI Materials Methods). Images taken 3-week incubation.          HadB HadC, HadA HadC (Fig. 2). <br />  <br /> 14630 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104                                                                                             Sacco et al. <br />  Fig. 4. Comparison specific activities presence octenoyl-CoA octenoyl-ACP. Assays performed 2 &#226;?&#174;M substrate fixed  <br /> centrations enzymes HadAB   HadBC (B). Data means &#226;&#171;&#190; SD. activities measured heterodimers presence C8:1-CoA significantly control values  enzyme). <br />  <br />  <br />  <br /> Weaker homotypic interactions detected HadA HadB, consistently  <br />   results agreement vitro data. altogether led conclusion HadB interacts vitro vivo HadA HadC form heterodimers  temperature &#226;&#177;&#8211;20&#194;&#176;C). <br />  <br /> Enzymatic Activity HadAB HadBC Heterodimers. Enzymes <br /> belonging (R)-specific enoyl hydratase/hydroxyacyl dehy- <br /> dratase family preferentially catalyze hydration reaction isolated enzymatic complex (6).  activities studied presence enoyl <br /> derivatives vitro. activities HadAB HadBC het- <br /> erodimers measured presence short-chain <br /> CoA derivatives, 3-hydroxybutyryl-CoA (C4), <span id='am-37' about='obo:IMR_0200121' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Crotonoyl-CoA" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200121" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00877" datatype="xsd:string"></span>crotonoyl-CoA</span> <br /> (C4:1), trans-2-octenoyl-CoA (C8:1). HadBC did display significant activity substrate experimental <br /> conditions used, varying enzyme concentration large range  SI Materials Methods). slight activity HadAB detected presence C8                          Fig. 5. Coupled assay HadAB presence MabA InhA. Shown <br /> substrate: 0.506 &#226;&#171;&#190; 0.001 &#226;?&#174;mol/min milligram protein               results MALDI-TOF MS analyses reaction media containing <br /> highest enzyme concentration tested (280 nM).                                   3-ketododecanoyl-CoA, <span id='am-27' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADPH</span>, <span id='am-22' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span>NADH</span>, MabA plus InhA  , plus 280 <br />                                                                                 nM HadAB (B), plus HadAB InhA (C). B, peaks m    Kinetic experiments realized presence                                                                                 966, 988, 1,010, 1,032, 1,054 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185;, [M&#226;&#171;&#185;Na]&#226;&#171;&#185;, [M-H&#226;&#171;&#185;2Na]&#226;&#171;&#185;, <br /> longer-chain derivatives, C12&#226;&#8364;&#8220;C20 trans-2-enoyl-CoAs. mini-                  [M-2H&#226;&#171;&#185;3Na]&#226;&#171;&#185;, [M-3H&#226;&#171;&#185;4Na]&#226;&#171;&#185; ions 3-hydroxydodecanoyl-CoA (product <br /> mize solubility problems encountered amphipathic                  MabA), respectively. B, minor peaks m 948, 970, 992, 1,014, molecules, experiments performed fixed low                  1,036 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185; <span id='am-16' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> monosodium tetrasodium adducts substrate concentration (2.5 &#226;?&#174;M). HadAB heterodimer proved                      unsaturated species, dodecenoyl-CoA. C, peaks m 950, 972, 994, active presence medium- long-chain                     1,016, 1,038 stand [M&#226;&#171;&#185;H]&#226;&#171;&#185; ion monosodium tetrasodium <br /> molecules (Fig. 3A). contrast, signal detected                   adducts saturated species, dodecanoyl-CoA. Insets C display HadBC conditions; exhibited activity higher               kinetics respective reactions followed 340 nm (oxidation InhA <br />                                                                                 coenzyme, NADH). spectra correspond time points 3 min <br /> substrate concentration (25 &#226;?&#174;M). activity HadBC <br />                                                                                 reaction. <br /> increased increasing chain length (Fig. 3C), HadAB behaved differently specificity centered hexadecenoyl-CoA (C16) (Fig. 3 B). <br />                                                                                 SI Materials Methods). MS analyses demonstrate    behaviors enzymes presence ACP CoA <br />                                                                                 dodecanoyl-CoA, saturated product InhA, derivatives compared using trans-2-octenoyl- <br />                                                                                                                                                                   BIOCHEMISTRY <br />  <br />  <br />  <br /> ACP trans-2-octenoyl-CoA substrates. heterodimers                    quickly formed  3 min) presence HadAB (Fig. <br /> displayed strict specificity enoyl-ACP (Fig. 4).                   5C). biosynthesis did occur absence Interestingly, HadB homodimer able metabolize                        heterodimer (Fig. 5A). absence InhA, small propor- <br /> C8:1-ACP conditions identical used het-                     tion intermediate dodecanoyl-CoA, product erodimers (data shown).                                                     dehydration reaction catalyzed HadAB, appeared (Fig. 5B). <br />    Finally, heterodimers tested ability                  type profile reminiscent described function coupled reactions presence MabA InhA,                 HADs (6): absence enoyl reductase, reaction FAS-II reductases catalyze reactions upstream                  equilibrium favor hydration reaction. comparison, <br /> downstream dehydration step cycle (SI Fig. 7).                    complete reaction presence HadBC heterodimer MabA reaction performed presence                        went slowly: product InhA began appear 30 <br /> 3-ketododecanoyl-CoA (C12),  completion                     min  reaction arrived completion 3-h <br /> action, heterodimers plus InhA added. Aliquots                  24-h time points (SI Fig. 10). weak activity correlates reaction mixtures taken different time points                relatively poor specific activity HadBC  <br /> Sacco et al.                                                                                     PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  37 &#229;&#8230;&#169; 14631 <br />  Fig. 6. Model roles HadAB HadBC heterodimers MA biosynthesis pathway. HadAB  like KasA, early FA elongation <br /> cycles catalyzed FAS-II  leading formation intermediate-size (C22&#226;&#8364;&#8220;C42) meromycolic chains consequently medium-length MAs Rhodococcus Nocardia. mycobacteria, HadBC, like KasB, elongate intermediate-size meromycolic chains size molecules <br /> (C52&#226;&#8364;&#8220;C64) late elongation cycles performed FAS-II, resulting synthesis eumycolic acids (C74&#226;&#8364;&#8220;C90). clarity, proteins FAS-II mentioned. <br />  <br />  <br />  <br /> presence medium-chain CoA derivatives   earlier                      Discussion paragraph).                                                                     typical FabZ FabA enzymes universally far   conclusion, properties HadAB HadBC het-                              bacterial FAS-II systems present Mtb sequenced <br /> erodimers reminiscent described mycobac-                        mycobacterial genomes. previous investigations seek <br /> terial FAS-II individual enzymes                    potential (R)-specific hydratases/dehydratases tubercle complex, e., marked specificity medium-                           bacillus led (13, 14) protein family long-chain substrates ACP derivatives (7&#226;&#8364;&#8220;10).                        include bacterial eukaryotic enoyl-CoA hydratases 2 (16,  exhibit (3R)-hydroacyl dehydratase activity                      17). shares global topology probably presence reductases FAS-II complex.                                   reaction mechanism FabA/FabZ family, exhibits                                                                                      poor sequence similarity catalytic motif Search HadA, HadB, HadC Orthologs Corynebacteri- <br />                                                                                      distinct (17). Nonetheless, domain multifunc- <br /> neae. gather putative information respective <br />                                                                                      tional FAS polypeptides Corynebacterineae, Fungi, roles HadAB HadBC FAS-II cycle,                                                                                      Archaeoglobus hydratase 2 catalytic motif (13, 14, 21). <br /> consequently formation meromycolic chain,                                                                                      Similarly, imagine (3R)-hydroxyacyl-ACP <br /> presence organization hadA-C cluster examined <br />                                                                                      dehydratation step elongation cycles monitored Corynebacterineae genera, genomes                                                                                      mycobacterial FAS-II catalyzed enzyme <br /> available  Table 1 legend). chromosomal region <br /> surrounding hadA-C gene cluster conserved                           hydratase 2 family. mycobacteria related genera, Coryne-                        putative hydratases 2 related proteins iden- <br /> bacterium, Rhodococcus, Nocardia (SI Fig. 11).                        tified Mtb, protein cluster HadA-HadB-HadC, probably <br /> Corynebacterium ORFs corresponding hadA, hadB,                            encoded operon, appeared good candidate hadC missing, orthologs detected                          FAS-II. complete set experimental data reported <br /> region BLAST searches (Table 1), known FAS-II                          present study points clearly HadAB HadBC <br /> enzymes.  Corynebacterium produces short MAs (C22&#226;&#8364;&#8220;                            heterodimers missing pieces elongation  <br /> C36), result simple condensation classical FAs                     Unlike Rv3389c, previously studied Mtb hydratase 2 protein <br /> (C8&#226;&#8364;&#8220;C18), FAS-II-linked FA elongation steps.                             (13), HadB-containing enzymes display ACP dependence <br /> Rhodococcus Nocardia possess &#226;&#8364;&#732;&#226;&#8364;&#732;intermediate-size&#226;&#8364;&#8482;&#226;&#8364;&#8482; MAs                           specificity long-chain substrates character- <br /> (C34&#226;&#8364;&#8220;C48 C44&#226;&#8364;&#8220;C60, respectively) medium-length                          istic properties FAS-II components (5). <br /> meromycolic chain (C22&#226;&#8364;&#8220;C30 C32&#226;&#8364;&#8220;C42, respectively), shorter                          coproduced E. coli M. smegmatis, HadB protein mycobacteria (C52&#226;&#8364;&#8220;C64) (2). Consistently,                         associates heterodimers HadA HadC. genes orthologous hadA hadB, inhA, mabA,                       partnerships substantially change behavior, HadAB <br /> kasA, present Rhodococcus Nocardia (Table 1 SI                        HadBC heterodimers exhibit distinct chain length specificity <br /> Fig. 11). Interestingly, orthologs hadA hadB fused                    profiles. Strikingly, observed genome analyses long unique gene Nocardia. contrast, hadC ORF                          hadA hadB genes conserved missing Nocardia Rhodococcus (SI Fig. 11),                        mycolate-producing genera, Rhodococcus Nocardia.  <br /> ortholog genomes (Table 1).                        sistently, genes overlap fuse long unique gene, <br /> Likewise, BLAST searches revealed genera                           suggesting cotranscription association hold protein orthologous KasB (data shown).                               products crucial physiology Corynebacterineae. <br />  <br /> 14632 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0704132104                                                                                                       Sacco et al. <br />  Furthermore, demonstrated gene                        topology equivalent predicted structure Rv3389c (13, <br /> hadA-C cluster essential Mtb survival, case                 14), intervening bridge domains. genes encoding FAS-II enzymes (18&#226;&#8364;&#8220;20). consistent                      present work description bacterial mono- transposon mutagenesis-based predictions essentiality                     functional HADs belonging hydratase 2 enzyme family. hadA hadB genes (22).  like KasA (4, 23), HadAB                      Thanks elegant experimental strategy, eukaryotic likely involved early FA elongation cycles catalyzed               discovered recently, yeast mitochon- FAS-II mycobacteria, Nocardia Rhodococ-                     drial FAS-II (12). Strikingly, corresponds hydratase <br /> cus, leading formation intermediate-size meromycolic                    2-related double hot dog enzyme. high level sequence <br /> chains (Fig. 6). preference HadBC longer substrates,                   divergence mycobacterial eukaryotic enzymes, compared HadAB, absence HadC ortholog                       importance essential step metabolic <br /> genera bearing medium-chain MAs, KasB ortholog                        pathway specific Mycobacterium related genera, makes absent, strongly suggest HadBC implicated,                  HadAB HadBC exciting new targets drug discovery. described KasB (4, 23), late steps meromycolic               Materials Methods <br /> chain biosynthesis occurring mycobacteria (Fig. 6). Addition- <br />                                                                                   Sequence analyses described (13). Mtb Rv0635- <br /> ally, hadC gene does essential Mtb growth <br />                                                                                   Rv0637-KO strain constructed based previously  vitro (22). Consistently, kasB shown nonessential                                                                                   ported strategy (20). DCOs isolated single <br /> Mtb Mycobacterium marinum (4, 23). Furthermore, HadC                          crossover (SCO) merodiploid strains previously described ubiquitous mycobacteria. data,                          (24). Protein production purification HadB, HadAB, propose HadBC necessary formation                          HadBC performed described (14), expres- <br /> complete meromycolic chains MA mycobacteria (Fig. 6).                       sions induced 37&#194;&#176;C, Triton added lysis buffers <br />    sequence analysis, HadB bears hydratase 2                       SI Table 3), Ni Sepharose FF (GE Healthcare) <br /> motif (13, 14) catalytic subunit                    column used. split-Trp experiments adapted HadAB HadBC heterodimers. Furthermore, structural                         procedure described yeast (25). Enzyme assays MS <br /> predictions strongly suggest subunits                  analyses reaction media performed described (13). single hot dog fold.  organize asymmetrical                  Short-chain CoA derivatives used 25 &#226;?&#174;M  75 <br /> heterodimers reminiscent asymmetrical double                      &#226;?&#174;M hydroxybutyryl-CoA). octenoyl-ACP prepared <br /> hot dog fold R-hydratase domain MFE-2 enzymes                       using commercially available E. coli holo-ACP (Sigma, St. <br /> probably Rv3389c R-hydratase/dehydratase protein (13, 17).                     Louis, MO), trans-2-octenoic acid, E. coli acyl-ACP synthase double hot dog folds, domain contains catalytic                   reported (26). mycobacterial FAS-II use apparatus necessary stabilize                   E. coli ACP instead natural ACP, AcpM (SI Fig. 7) (5). long acyl chain substrates. role analogous                 additional details SI Materials Methods. proposed HadA HadC subunits. Consistently, <br /> structural predictions suggest HadAB HadBC                         thank F. Laval, H. Montrozier, F. Viala (Institut Pharmacologie <br /> hold open active site tunnels (required long-chain                        et Biologie Structurale), L. Henry (Ecole Polytechnique Federale <br />                                                                                   Lausanne) precious help MS analyses, substrate <br /> meromycolate precursors), detailed description                                                                                   synthesis, figure preparation, split-Trp experiments, respectively;  <br /> await crystallographic data. contrast, closed substrate-                     Hartridge (Institute Cell Molecular Science) technical assis- <br /> binding pocket predicted HadB homodimer (15).                        tance Expression Proteomics Facility (Uppsala University) Furthermore, absence activity HadB heterodimer                   MS analyses; J. Shanklin (Brookhaven National Laboratory, Upton, NY) <br /> presence octenoyl-ACP suggests heterotypic partner-                   J. Walker (Medical Research Council, Cambridge, U.K.) ships HadB HadA HadC play role                       kind gifts plasmid pAasH E. coli C41(DE3), respectively; S. interaction ACP moiety substrate.                      Cole (Institut Pasteur, Paris, France) total Mtb DNA cosmid <br />                                                                                   MTCY20H10 work supported doctoral fellowship <br /> knowledge, asymmetrical hot dog folded heterodimer                          E.S.) Fondation Pour la Recherche dicale, Marie reported far. organization single fused                 Curie fellowship, grants European Community (QLK2- <br /> gene, deduced orthologs HadA HadB                       CT-2001-02018 LSHP-CT-2005-018923), Foundation Stra- Nocardia covalently bound. adopt                    tegic Research, Swedish Research Council. <br />  <br />  <br />  1. World Health Organization (2004) Anti-Tuberculosis Drug Resistance     13. Sacco E, Legendre V, Laval F, Zerbib D, Montrozier H, Eynard N, Guilhot C, <br />     World, Report  3   Geneva).                                                Daffe&#194;&#180; M, Que&#194;&#180;mard (2007) Biochim Biophys Acta 1774:303&#226;&#8364;&#8220;311. <br />  2. Barry CE, III, Lee  Mdluli K, Sampson AE, Schroeder BG, Slayden RA,        14. Castell  Johansson P, Unge T, Jones TA, Ba&#194;&#168;ckbro K (2005) Protein Sci <br />     Yuan Y (1998) Prog Lipid Res 37:143&#226;&#8364;&#8220;179.                                          14:1850&#226;&#8364;&#8220;1862. <br />  3. Dubnau E, Chan J, Raynaud C, Mohan VP, Lane   &#194;&#180;elle MA, Yu K, Que&#194;&#180;mard      15. Johansson P, Castell  Jones TA, Ba&#194;&#168;ckbro K (2006) Protein Sci 15:2300&#226;&#8364;&#8220;2309. <br />     Smith  Daffe&#194;&#180;, M (2000) Mol Microbiol 36:630&#226;&#8364;&#8220;637.                           16. Hisano T, Tsuge T, Fukui T, Iwata T, Miki K, Doi Y (2003) J Biol Chem <br />  4. Bhatt  Fujiwara N, Bhatt K, Gurcha SS, Kremer L, Chen B, Chan J, Porcelli       278:617&#226;&#8364;&#8220;624. <br />     SA, Kobayashi K, Besra GS, Jacobs WR, Jr (2007) Proc Natl Acad Sci USA        17. Koski MK, Haapalainen  Hiltunen JK, Glumoff T (2004) J Biol Chem <br />     104:5157&#226;&#8364;&#8220;5162.                                                                    279:24666&#226;&#8364;&#8220;24672. <br />                                                                                                                                                                       BIOCHEMISTRY <br />  <br />  <br />  <br />  5. Bloch K (1977) Adv Enzymol Relat Areas Mol Biol 45:1&#226;&#8364;&#8220;84.                      18. Vilche`ze C, Morbidoni HR, Weisbrod TR, Iwamoto H, Kuo M, Sacchettini JC, <br />  6. Rock  Cronan JE (1996) Biochim Biophys Acta 1302:1&#226;&#8364;&#8220;16.                         Jacobs WR, Jr (2000) J Bacteriol 182:4059&#226;&#8364;&#8220;4067. <br />  7. Schaeffer ML, Agnihotri G, Volker C, Kallender H, Brennan PJ, Lonsdale JT     19. Bhatt  Kremer L, Dai AZ, Sacchettini JC, Jacobs WR, Jr (2005) J Bacteriol <br />     (2001) J Biol Chem 276:47029&#226;&#8364;&#8220;47037.                                               187:7596&#226;&#8364;&#8220;7606. <br />  8. Kremer L, Dover LG, Carrere S, Nampoothiri KM, Lesjean S, Brown AK,           20. Parish T, Roberts G, Laval F, Schaeffer M, Daffe&#194;&#180; M, Duncan K (2007) J <br />     Brennan PJ, Minnikin  Locht C, Besra GS (2002) Biochem J 364:423&#226;&#8364;&#8220;              Bacteriol 189:3721&#226;&#8364;&#8220;3728. <br />     430.                                                                          21. Qin YM, Haapalainen  Kilpela&#194;&#168;inen SH, Marttila MS, Koski MK, Glumoff <br />  9. Marrakchi H, Ducasse S, Labesse G, Montrozier H, Margeat E, Emorine L,            T, Novikov DK, Hiltunen JK (2000) J Biol Chem 275:4965&#226;&#8364;&#8220;4972. <br />     Charpentier X, Daffe&#194;&#180; M, Que&#194;&#180;mard (2002) Microbiology 148:951&#226;&#8364;&#8220;960.          22. Sassetti CM, Boyd DH, Rubin EJ (2003) Mol Microbiol 48:77&#226;&#8364;&#8220;84. <br /> 10. Que&#194;&#180;mard  Sacchettini JC, Dessen  Vilche <br />                                                `ze C, Bittman R, Jacobs WR, Jr,   23. Gao LY, Laval F, Lawson EH, Groger RK, Woodruff  Morisaki JH, Cox JS, <br />     Blanchard JS (1995) Biochemistry 34:8235&#226;&#8364;&#8220;8241.                                    Daffe&#194;&#180; M, Brown EJ (2003) Mol Microbiol 49:1547&#226;&#8364;&#8220;1563. <br /> 11. Joshi AK, Smith S (1993) J Biol Chem 268:22508&#226;&#8364;&#8220;22513.                         24. Parish T, Stoker NG (2000) Microbiology 146:1969&#226;&#8364;&#8220;1975. <br /> 12. Kastaniotis AJ, Autio KJ, Sormunen RT, Hiltunen JK (2004) Mol Microbiol       25. Tafelmeyer P, Johnsson N, Johnsson K (2004) Chem Biol 11:681&#226;&#8364;&#8220;689. <br />     53:1407&#226;&#8364;&#8220;1421.                                                                 26. Rock  Garwin JL (1979) J Biol Chem 254:7123&#226;&#8364;&#8220;7128. <br />  <br />  <br />  <br />  <br /> Sacco et al.                                                                                       PNAS &#229;&#8230;&#169; September 11, 2007 &#229;&#8230;&#169; vol. 104 &#229;&#8230;&#169;  37 &#229;&#8230;&#169; 14 <br /> </body></html>