<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />               Cdc1 Vacuole Coordinately Regulate Mn21 Homeostasis <br />                            Yeast Saccharomyces cerevisiae <br />  <br />   <br />                                                                ABSTRACT <br />                  yeast CDC1 gene encodes essential protein implicated regulation cytosolic <br />               [Mn2 1]. identify factors impinge Cdc1 Cdc1-dependent process, isolated second- <br />               site suppressors conditional cdc1-1(Ts) growth defect. Recessive suppressors define 15 COS (CdcOne <br />               Suppressor) genes. Seven COS genes required biogenesis vacuole, organelle <br />               known sequester intracellular Mn21 . eighth gene, COS16, encodes vacuolar membrane protein <br />               involved Mn2 1 homeostasis. results suggest mutations block vacuolar Mn2 1 <br />               sequestration compensate defects Cdc1 function. Interestingly, Cdc1 dispensable cos16D <br />               deletion strain, cdc1D cos16D double mutant exhibits robust growth medium supplemented               Mn2 1.  single, essential function Cdc1 regulate intracellular, probably cytosolic, Mn21. <br />  <br />  <br />  <br /> C     ELLS yeast Saccharomyces cerevisiae divide        budding. Bud development complex process appears require metal <span id='am-11' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> Mn21. Early studies <br />                                                                            glycosylation shown affect bud growth. Mu- <br />                                                                            tants och1D lack Mn21-dependent Golgi glycosyl- <br />                                                                            transferase exhibit conditional defect bud ion chelators Ca21 ionophores (Iida et al. 1990)                  growth, Och1 essential standard growth <br /> suggested Ca21 essential cell growth                      conditions (Nagasu et al. 1992).  clear <br /> proliferation, subsequent report suggested Mn21,                     Och1, Golgi Mn21 , related Mn21 Ca21, limiting ion (Youatt McKinnon                        requirement identified depletion studies. <br /> 1993). Yeast cells depleted Mn21 Ca21 ex-                         proteins implicated intracellular <br /> hibit defect bud growth, Mn21 500&#226;&#8364;&#8220;1000-fold                    Mn21 homeostasis. Mn21 uptake plasma mem- effective Ca21 reversing growth defect,                   brane appears mediated Smf1, integral <br /> suggesting Mn21 physiologically important                      plasma membrane protein (Supek et al. 1996). second <br /> ion (Loukin Kung 1995).                                                protein, Smf2, shares significant homology Smf1 <br />    Yeast Mn21 present subcellular compart-                   involved Mn21 uptake (Eide ments, represent, influence,                     Guerinot 1997). intracellular organelles, Mn21 pool essential bud growth. cytosol                        Golgi characterized  <br /> contains Mn21 Mn21 -dependent enzymes                      spect Mn21 flux. structurally unrelated proteins, pyruvate carboxylase, glutamine synthetase, argi-                   Pmr1  Ca21-ATPase homologue) Ccc1, nase (Wedler 1994). Mn21 present Golgi,                     play role transport Mn21 cytosol activates glycosyltransferases involved                  Golgi (Lapinskas et al. 1995; Lapinskas et al. processing secreted proteins (Wedler 1994).                      1996). protein, Atx2, implicated Mitochondrial Mn21 required enzymes citric                    release Golgi Mn21 cytosol (Lin acid cycle (Wedler 1994) proteases involved                     Culotta 1996). proteins (Pmr1 mitochondrial protein import (Supek et al. 1996).                       Smf1) essential growth Mn21 -limiting <br /> Finally, Mn21 yeast vacuole (Okorokov                      conditions, phenotypes Mn21-depleted pmr1 et al. 1977), acidic, membrane-bound organelle                     smf1 strains reported.  clear implicated Mn21 detoxification.                       activity protein affects Mn21 pool <br /> known Mn21-dependent enzymes processes,                           required bud growth. <br />                                                                               factor does affect growth-related Mn21 <br />                                                                            pool essential gene CDC1. Conditional cdc1(Ts) <br />   Corresponding author: Stephen Garrett, Department Microbiology        mutants originally identified basis  Molecular Genetics, UMDNJ-New Jersey Medical Center, 185               fect bud growth (Hartwell et al. 1970), South Orange Ave., University Heights, Newark, NJ 07103-2714. <br /> E-mail: garretst@umdnj.edu <br />                                                                            lines evidence implicate Cdc1 regulation   1 <br />     Present address: Department Biochemistry, University Connecti-   cytosolic Mn21.  Mn21 supplement partially sup- <br /> cut Health Center, Farmington, CT 06030.                                   presses cdc1(Ts) growth defect (Loukin Kung <br />  <br /> Genetics 148: 1787&#226;&#8364;&#8220;1798 (April, 1998) <br />  1788                                                M. Paidhungat S. Garrett <br />  <br /> 1995; Paidhungat Garrett 1998), suggesting                 lated patching independent colonies FY11 FY12 <br /> Mn21 limiting cdc1 mutants. Second, defects                YEPD agar plates, incubated                                                                     308 368 2 days. Temperature-resistant revertants arose <br /> CDC1 cause cells sensitive depletion                papillae patches; single papilla picked cytosolic Mn21 (Supek et al. 1996; Paidhungat Gar-              patch ensure revertants independent. rett 1998). Finally, CDC1 overexpression makes certain              238, revertants gave rise small, temperature-resistant <br /> chelator-sensitive mutants tolerant Mn21 deple-             colonies large, temperature-sensitive colonies. <br /> tion (Paidhungat Garrett 1998).  pro-                   Crosses showed small colonies disomic chro- <br />                                                                     mosome IV, large colonies contained single <br /> pose bud-growth defect cdc1(Ts) mutants                 copy chromosome IV. <br /> results depletion cytosolic Mn21, implies,               Cloning COS4, COS5, COS15: COS4, COS5, COS15 turn, cytosolic Mn21 critical bud growth             cloned YCp50-based yeast genomic library (Rose <br /> (Paidhungat Garrett 1998).                      et al. 1987) ability complement temperature- mechanism Cdc1 regulates cytosolic Mn21                sensitive growth defects cos4-8 cdc1-1 [YEp13-CDC1], cos5- <br />                                                                     17 cdc1-1 [YEp13-CDC1], cos15-118 cdc1-1 [YEp13-CDC1] Mn21-dependent processes known.                         strains, respectively. COS4 plasmids, pFB77&#226;&#8364;&#8220;pFB80,  <br />   identify cellular components affect essen-            tained distinct, overlapping, genomic regions tial Cdc1 function, isolated suppressors cdc1-            reversed temperature-resistant growth cos4-8 cdc1- <br /> 1(Ts) temperature-sensitive growth defect.               1(Ts) strain. Plasmid pFB87 generated subcloning cessive suppressors define genes previously implicated              4.3-kb HindIII-BglII genomic fragment pFB78                                                                     HindIII BamHI sites low-copy URA3 vector pRS316 biogenesis vacuole, organelle known                 (Sikorski Hieter 1989). fragment complemented <br /> sequester divalent cations Mn21. addition,           cos4 mutant phenotypes physically linked <br /> suppressor gene, COS16, encodes vacuolar membrane                 PEP3 region (Riles et al. 1993). PEP3 gene protein appears involved vacuolar Mn21                disrupted inserting HIS3 marker unique BamHI <br /> sequestration. results suggest mutations ei-             site pFB78 generate plasmid pFB107. pep3::HIS3 <br />                                                                     disruption excised pFB107 using EcoRI. <br /> ther directly indirectly block vacuolar Mn21 sequestra-             Seven (pFB94&#226;&#8364;&#8220;pFB100) COS5 plasmids recovered <br /> tion compensate defects Cdc1 function.                contained distinct, overlapping, genomic regions. Plasmid <br /> sults Cdc1 dispensable cells lacking           pFB109 constructed inserting 4.6-kb BglII-BamHI <br /> Cos16, suggesting Cdc1 function required             fragment pFB99 BamHI site pRS316. cells exhibit defect vacuolar Mn21 sequestra-             plasmid complemented cos5 mutant phenotypes. <br />                                                                     Plasmid pFB162 used mark COS5 locus tion.  likely essential function          constructed inserting 1.9-kb EcoRI fragment Cdc1 maintain cytosolic [Mn21] minimum                pFB109 derivative (pFB119) integrating vector <br /> threshold required growth.                              pRS306, lacked SpeI XbaI sites multiple <br />                                                                     cloning sequence (MCS). Plasmid pFB162 linearized                                                                     unique XbaI site insert integrated homology- <br />                MATERIALS METHODS                                directed recombination yeast genome. <br />                                                                        identical COS15 plasmids, pFB135 pFB136,  <br />    Media: Standard yeast media prepared previously          tained PEP5 gene, determined comparing restric- <br /> described (Paidhungat Garrett 1998). LiCl MgCl2             tion map YCp50-PEP5 plasmid obtained E. added rich yeast medium (YEPD) autoclaving.           Jones (Carnegie Mellon University, Pittsburgh). <br /> Medium containing CaCl2 autoclaving YEPD                Cloning disruption COS8: COS8 gene origi- <br /> dium, buffering pH 5.5 50 mm succinate/NaOH             nally cloned high-copy suppressor cdc1-1(Ts) growth adding CaCl2 indicated concentration.                   defect. Plasmid pFB57 isolated high-copy (pRS202) <br />    Strains plasmids: Revertants derived strains       yeast genomic library (C. Connelly P. Hieter, unpub- <br /> FY11 [MATa ade1 trp1 leu2 his3 ura3 cdc1-1(Ts)] FY12            lished results) ability suppress temperature-sensi- <br /> [MATa ade8 trp1 leu2 his3 ura3 cdc1-1(Ts)]. Strain FY 70 (FY11      tive growth defect strain FY11 (cdc1-1) 308. 1.8-kb <br /> CDC1) derived FY11 reversion contains           HindIII genomic fragment pFB57 subcloned wild-type CDC1 allele. Strain FY 71 (MATa ade8 trp1 leu2 his3       HindIII site pRS202 generate plasmid pFB67, ura3 CDC1) described (Paidmungat Garrett               suppressed cdc1-1(Ts) temperature-sensitive growth  <br /> 1998) used determine suppressor linkage CDC1.         fect. Interestingly, plasmid pFB67 complemented salt sen- <br /> Diploid strain FY1 (MATa/MATa cdc1D::HIS3/CDC1 ade2/                sitivity cos8 mutants. determine pFB57 contained <br /> ADE2 trp1/TRP1 ura3/ura3 leu2/leu2 his3/his3 lys2/lys2)         COS8, 1.8-kb HindIII fragment subcloned cos16D derivative (FY1 cos16D::LEU2/COS16) derived           low-copy vector pRS316. resulting plasmid, pFB161, com- strain Y1029 (Garrett Broach 1989) transfor-            plemented cos8 mutant phenotypes. 1.8-kb Hin- <br /> mation. Bacterial strains MC1066 DH5a used             dIII fragment physically mapped region chromo- <br /> plasmid manipulations described (Casadaban            XVI (Riles et al. 1993 references  et al. 1983; Woodcock et al. 1989).                                 contained single, complete open reading frame, VPS4/ <br />    Plasmid YEp13-CDC1 (pFB28) described (Paid-             END13. <br /> hungat Garrett 1998). Plasmid YIp5-CDC1 (pFB56)                    1.8-kb HindIII COS8 fragment inserted pUC8 constructed inserting 3.5-kb HindIII CDC1 fragment       (NEB catalogue) generate plasmid pFB217. different HindIII site YIp5 (NEB catalogue; New England           derivatives pFB217 used disrupt chromosomal <br /> Biolabs, Beverly, MA). resulting plasmid integrated         COS8 gene. construct, URA3 marker  CDC1 locus linearizing XbaI yeast transfor-   serted unique MscI site COS8 generate plasmid <br /> mation.                                                             pFB277. cos8::URA3 disruption liberated plasmid <br />    Genetic manipulations: cdc1(Ts) revertants iso-         pFB277 KpnI, transplaced yeast genome.                                                       Mn21 Homeostasis Yeast                                                1789 <br />  <br /> confirm cos8::URA3 disruption pFB277 represented           CDCFDA used instead CFDA. CPY secretion  complete loss Cos8 function, cos8D::URA3 deletion          tected colony hybridization assay (Roberts et al. 1991). constructed pFB217 replacing EcoRV-SpeI             Isolation vacuoles Cos16 localization: Crude fraction- <br /> gion COS8 URA3 generate plasmid pFB275.              ation cell extracts performed described (Cowles et <br /> cos8D::URA3 disruption liberated HindIII trans-        al. 1994), 35S-labeling step omitted. Vacu- <br /> placed yeast genome. transformation fre-          oles purified described (Cardenas et al. 1995; Ohsumi <br /> quencies using second construct reduced             Anraku 1981), exception SW40Ti rotor <br /> limited flanking homology, transformants derived        used ultracentrifugation (rpms adjusted gener- <br /> construct exhibited identical phenotypes.                           ate required g force). Protein content extracts    Cloning COS9 : recessive, cold-sensitive (188) growth        measured Bradford assay (Bio-Rad, Richmond, CA) <br /> defect cos9-26(Cs) revertant exploited select COS9    BSA standards. antigen detection quantitation, <br /> clones low-copy yeast genomic library (Rose et al. 1987).    extracts diluted sample buffer, heated 658 10 plasmids, pFB188 pFB190, containing overlapping             min, separated 10% SDS-PAGE, blotted nitrocellu- <br /> regions complemented conditional growth defect           lose (Shleicher Schuell, Keene, NH). Cos16-LacZ <br /> cos9-26(Cs) strain reversed temperature-resistant           fusion protein detected b-Gal antiserum (Cappel <br /> growth cdc1-1(Ts) cos9-26(Cs) double mutant 308. 1.0-     Research Products). CPY Vph1 detected mono- <br /> kb EcoRI-XbaI complementing fragment plasmid pFB188            clonal antibodies Molecular Probes (Eugene, . Kar2 subcloned pRS316 generate plasmid pFB220.          Pma1 detected using antisera obtained M. D. <br /> region physically linked ARF1 region chromo-          Rose (Princeton University, Princeton, NJ) M. Cardenas IVL (Riles et al. 1993 references . frame-       (Duke University Medical Center, Durham, NC), respectively. <br /> shift mutation introduced ARF1 gene digesting       HRP-conjugated secondary antibodies (Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span>, St. <br /> pFB220 BglII filling staggered ends generate     Louis; Promega, Madison, WI) used detection plasmid pFB254. arf1D::HIS3 construct obtained         ECL (Amersham, Arlington Heights, . <br /> Dr. R. Kahn (Emory University, Atlanta).                               Mn21 estimation: cells prepared previously  <br />    Cloning COS16: COS16 gene cloned low-             scribed (Lapinskas et al. 1995), concentrated centrifu- <br /> copy yeast genomic library (Rose et al. 1987) screening      gation 20&#226;&#8364;&#8220;80 A600 units/ml. cell slurry (25 ml, 1&#226;&#8364;&#8220;2 A600 <br /> temperature-sensitive transformants cdc1-1(Ts) cos16-57        units) digested 50% nitric acid 2 hr 958  mutant. plasmids (pFB163 pFB164) overlapping           solution cleared) acid-rinsed Eppendorf tube, cooled <br /> inserts recovered. Plasmids pFB168 pFB169             room temperature, diluted 1 ml deionized <br /> derived low-copy LEU2 vector pRS315 (Sikorski          water. Atomic absorption spectroscopy performed Hieter 1989) contained 2.7-kb PstI-BamHI 3.2-kb             L&#226;&#8364;&#8482;vov platform furnace Z3030 (Perkin-Elmer, Norwalk, <br /> HindIII-PstI fragments, respectively, pFB163. Plasmid          CT) instrument 25 mg Mg 3)2 (Sigma Chemical, ACS <br /> pFB179 generated inserting complementing 2.1-kb            grade) matrix modifier. furnace temperature program <br /> Bgl II-SpeI  SpeI site MCS) fragment           Dry 1408 (10 sec ramp, 60 sec hold); Char 14008 (8 sec <br /> pFB168 BamHI XbaI sites pRS316. Physical         ramp, 25 sec hold); Atomize reading 22008 (0 sec ramp, <br /> analysis 2.1-kb fragment (Riles et al. 1993 references   5 sec hold); Clean 26508 (1 sec ramp, 4 sec hold); Cool 208  showed contained single, complete open reading        (1 sec ramp, 30 sec hold). Argon <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span>gas</span> flow maintained frame, YCR44c. frameshift mutation introduced            300 ml/min, atomization (0 ml/min). YCR44c linearizing pFB179 unique XbaI site         digested sample analyzed twice averaged. results <br /> YCR44c, filling staggered ends, religating gener-     independent experiments shown. Mn21 content <br /> ate plasmid pFB182.                                                 sample normalized A600. <br />    cos16D::HIS3 disruption generated steps.  2.7-kb PstI-BamHI insert pFB168 inserted XbaI XhoI sites pRS306, using SpeI SalI sites pFB168 MCS. left unique XhoI XbaI sites                                RESULTS <br /> insert resulting plasmid pFB211. 0.6-kb XhoI-XbaI <br />                                                                        Suppressors cdc1-1(Ts) temperature-sensitive <br /> fragment COS16 coding region pFB211  <br /> placed XhoI-XbaI HIS3 fragment generate plasmid           growth defect: Suppressors cdc1-1(Ts) growth  <br /> pFB218. cos16D::HIS3 allele transplaced yeast      fect isolated selecting spontaneous, tempera- <br /> genome liberating plasmid pFB218 EcoRI.      ture-resistant revertants MATa cdc1-1(Ts) MATa <br /> cos16D::LEU2 allele constructed replacing XhoI-SnaBI     cdc1-1(Ts) strains  materials methods). Fifty- <br /> region YCR44c plasmid pFB211 SalI-HpaI LEU2 <br />                                                                     independent revertants isolated 308, fragment generate plasmid pFB371. cos16D::LEU2 allele liberated pFB371 BamHI Bgl II.                    revertants recovered 368.  <br />    COS16-lacZ fusion generated inserting MfeI        vertants isolated 308 grew 368, suggesting fragment carrying COS16 open reading frame             growth defect 308 severe 368 <br /> EcoRI site YEp356R (Myers et al. 1986) generate plasmid       (Figure 1). 54 revertants isolated 308, <br /> pFB325. fusion lacked carboxy-terminal residue      revertants isolated 368, contained dominant <br /> Cos16 protein. integrating version (pFB333) COS16- <br /> lacZ fusion plasmid constructed inserting AatII          suppressors. Temperature-resistant growth segregated <br /> COS16-lacZ fusion fragment pFB325 place ho-          2:2 cases, indicating revertant carried mologous AatII fragment YIp356R (Myers et al. 1986).             single nuclear suppressor mutation. <br /> Plasmid pFB333 integrated URA3 locus lineariz-        Intragenic suppressors identified crossing <br /> ing StuI.                                                      cdc1-1(Ts) cos (cdcone suppressor) strain wild- <br />    Vacuolar staining carboxypeptidase Y (CPY) secretion: <br /> Log-phase cells stained vital vacuolar dye            type CDC1 COS strain, examining tetrads tem- <br /> CDCFDA (Molecular Probes, Eugene, , using procedure            perature-sensitive recombinants [cdc1-1(Ts) COS]. Sup- <br /> described previously (Manolson et al. 1992),            pressors revertants isolated 368,  1790                                                  M. Paidhungat S. Garrett <br />  <br />                                                                                                  TABLE 1 <br />                                                                                Recessive suppressors cdc1-1(Ts) define 15 <br />                                                                                          complementation groups <br />  <br />                                                                        Locus                       Mutant allele(s) recovered <br />                                                                        cos1                 cos1-1, cos1-124 <br />                                                                        cos2                 cos2-2, cos2-6, cos2-13, cos2-16, cos2-19, <br />                                                                                                cos2-22, cos2-27 <br />                                                                        cos3                 cos3-3, cos3-109, cos3-115, cos3-127, <br />                                                                                                cos3-129 <br />                                                                        pep3 (cos4)          pep3-8, pep3-112, pep3-122 <br />                                                                        vph4 (cos5)          vph4-17 <br />                                                                        cos6                 cos6-18 <br />                                                                        cos7                 cos7-20, cos7-31 <br />                                                                        vps4 (cos8)          vps4-21, vps4-23, vps4-130 <br />                                                                        arf1 (cos9)          arf1-26 (Cs), arf1-32, arf1-132 <br />                                                                        cos10                cos10-28 <br />                                                                        cos11                cos11-29, cos11-113 <br />                                                                        cos13                cos13-101, cos13-125 <br />                                                                        cos14                cos14-102, cos14-103, cos14-116, cos14-120, <br />                                                                                                cos14-131 pep5 (cos15) pep5-118 <br />                                                                        cos16                cos16-57, cos16-128 <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;Suppressors cdc1-1(Ts) growth defect. parent cdc1-1(Ts) strain seven spontaneous revertants          cos isolates growth 308. analysis, 40 streaked YEPD agar incubated 238, 308, 368. Strains  C DC1 (FY 70 5 FY 11 C DC1); cos2 (FY 11 <br />                                                                        48 suppressors assigned 15 different comple- <br /> cos2-2); cos11 (FY 11 cos11-29); cos4 (FY 11 cos4-8); cos16 (FY11      mentation groups (Table 1). <br /> cos16-57); cos3 (FY 11 cos3-3); cos1 (FY 11 cos1-1); cdc1-1 (FY 11).      cdc1-1(Ts) cos double mutants sensitive <br />                                                                        salts 238. examine salt sensitivity                                                                         transformed representative mutant recessive revertant isolated 308, failed recombine               complementation group YIp5-CDC1 plas- cdc1-1(Ts) lesion (.30 tetrads examined).                 mid, tested growth medium containing 200 mm <br /> cdc1-1(Ts) allele resulted single C T transition             CaCl2, 200 mm MgCl2, 100 mm LiCl. Seven CDC1 <br /> [Pro351 (CCT) Leu351 (CTT)]. CDC1 gene                     cos mutants (cos4, cos5, cos15, cos6, cos11, cos8, cos9) revertants (FY70) isolated 368 contained                 exhibited varying degrees sensitivity salts wild-type CCT sequence codon 351; recessive                 tested (Figure 2 data shown; cos6 cos11 <br /> intragenic suppressor, cdc1-130, retained original                 mutants sensitive cos4 mutants salts, <br /> cdc1-1(Ts) lesion second C T transition [Thr231               sensitivity 200 mm MgCl2 readily <br /> (ACA) Ile231 (ATA)].                                                apparent figure). Interestingly, cos mutants <br />   Aberrant segregation ade8 marker suggested                    (cos1, cos3, cos14) sensitive CaCl2 recessive revertants disomic chromosome                   LiCl, cos mutants (cos13 cos16) specifically <br /> IV. CDC1 located chromosome IV,                       sensitive MgCl2 (Figure 2 data shown), likely cdc1-1(Ts) duplication sufficient               cos mutants (cos2, cos7, cos10) did exhibit allow growth 308. Consistent idea, cdc1-             measurable sensitivity salts tested (Figure <br /> 1(Ts) haploid strain carrying extra copy cdc1-               2 data shown). cases, salt sensitivity 1(Ts) allele integrating plasmid grew 308 (data               cdc1-1(Ts) suppression tightly linked  recombi- shown). contrast, diploid strain containing              nation .10 tetrads), implying single mutation copies cdc1-1(Ts) allele failed grow 308 (data            responsible phenotypes. Finally, sensitivity shown).  diploid cells need disproportionately                specific cations probably reflects differences cellu- Cdc1 activity haploid cells. explain                lar functions particular COS genes intragenic suppressors (cdc1-130)                   mutants complementation chromosome IV duplications recessive.                         groups (cos1, cos3, cos14, cos13, cos16, cos2, cos7,   Recessive suppressors define 15 genes fall                 cos10) exhibited sensitivity range cations phenotypic groups: 54 suppres-                 representative isolate (Figure 2 data sors isolated 308 recessive, unlinked CDC1.             shown). Accordingly, CDC1 cos mutants divided placed complementation groups                       broad phenotypic groups, IV (Table 2). <br /> testing diploids pairwise matings cdc1(Ts)                 Mutations group III genes, COS4, COS5,                                                     Mn21 Homeostasis Yeast                                              1791 <br />  <br />  <br />  <br />  <br />                                                                                                      Figure 2.&#226;&#8364;&#8221;Salt tolerance <br />                                                                                                   cos mutants. indi- <br />                                                                                                   cated strains trans- <br />                                                                                                   formed YIp5-C DC1 <br />                                                                                                   plasmid, streaked <br />                                                                                                   YEPD agar, YEPD pH <br />                                                                                                   5.5 agar containing 200 mm <br />                                                                                                   CaCl2, YEPD agar containing <br />                                                                                                   200 mm MgCl2, YEPD agar <br />                                                                                                   containing 100 mm LiCl, <br />                                                                                                   incubated 308 3&#226;&#8364;&#8220;5 <br />                                                                                                   days. Strains  cos2 (FY11 <br />                                                                                                   cos2-2); cos11 (FY11 cos11- <br />                                                                                                   29); cos6 (FY11 cos6-18); cos4 <br />                                                                                                   (FY11 cos4-8); cos16 (FY11 <br />                                                                                                   cos16-57); cos3 (FY11 cos3-3); <br />                                                                                                   cos1 (FY11 cos1-1); 1 (FY11). <br />  <br />  <br />  <br />  <br /> COS15, blocked accumulation red pigment             vacuolar protein sorting (vps/pep) mutants (Banta et <br /> ade1 background conferred temperature-sensitive             al. 1988). group IIIA cos mutants displayed <br /> (368) growth defect CDC1 strain (data shown).            additional phenotypes exhibited subset  group III genes subclassified group IIIA            vps/pep mutants, including sensitivity 500 mm <br /> (COS4, COS5, COS15), group IIIB (COS6, COS8,              NaCl, 4 mm ZnCl2, 8 mm MnCl2, failure <br /> COS9, COS11).                                                 sporulate homozygous diploids (data shown). <br />   Group IIIA COS genes identical class C VPS genes:        determine group IIIA COS genes related ade1 pigment-accumulation defect, temperature                 known VPS genes, isolated COS4, COS5, sensitivity, salt sensitivity group IIIA cos mu-       COS15 genes yeast genomic library comple- <br /> tants resembled phenotypes exhibited subset               menting temperature-sensitive growth defect                                                                   CDC1 cos4, CDC1 cos5, CDC1 cos15 strains, respec- <br />                                                                   tively  materials methods). examined <br />                            TABLE 2 <br />                                                                   ability cos4, cos5, cos15 mutants complement <br />             Phenotypic grouping COS genes                      phenotypes known vps/pep mutants. studies <br />                                                                   showed (Table 1) COS4 identical VPS18/PEP3 <br />               Complementation                                     (Preston et al. 1991; Robinson et al. 1991), COS5 Group           groups/Loci                Sensitivity                                                                    identical VPS16/VPH4 (Horazdovsky Emr           cos, cos3, cos14 b         CaCl2, LiCl                1993; Preston et al. 1992), COS15 identical II          cos13, cos16               MgCl2                      VPS11/PEP5 (Woolford et al. 1990). Finally,  <br /> III         Group IIIA: pep3           CaCl2, LiCl, MgCl2,        signments confirmed showing mutations <br />                (cos4), vps16 (cos5),     MnCl2, ZnCl2             inactivating VPS18 (pep3::LEU2), VPS16 (vph4-5)                pep5 (Cos15) <br />                                                                   VPS11 (pep5D::URA3), fourth class C VPS <br />             Group IIIB: vps4           CaCl2, LiCl, MgCl2, <br />                (cos8), cos6, cos11,      MnCl2, ZnCl2;            gene VPS33 (pep14-5) (Banta et al. 1990) suppressed <br />                arf1 (cos9)               sensitivity         growth defect cdc1-1(Ts) strain 308 (data                                          severe              shown).  mutations class C VPS genes <br />                                          group IIIA               suppressed cdc1-1(Ts) growth defect 308. <br /> IV          cos2, cos7, cos10                                   Group IIIB cos mutants mislocalize vacuolar pro- <br />       Sensitivity  200 mm CaCl2; 200 mm MgCl2; 100 mm LiCl;      tein CPY: Group IIIB cos mutants shared pheno- <br /> 4 mm ZnCl2; 8 mm MnCl2.                                           types class C vps mutants, including general <br />   b <br />     strains CDC1.                                        salt sensitivity sporulation defect, class C <br />  1792                                            M. Paidhungat S. Garrett <br />  <br />  <br />  <br />  <br />   Figure 3.&#226;&#8364;&#8221;CPY secretion. Patches indicated strains replica plated YEPD agar, overlaid nitrocellulose <br /> membrane, incubated 238. 12 hr, membranes rinsed water probed CPY antibodies. strains rows C carried YIp5-CDC1 plasmid. Strains  left right, row  FY11 cos16-57, FY11 cos3-3, FY11 cos1-1, FY11; row B: FY11 cos2-13, FY11 cos11-29, FY11 cos6-18, FY11 pep3-8; row C: FY11 arf1-32, FY11 pep3-8, FY11 vps4-21, FY11; <br /> row D: FY70 [pRS202-COS4], FY70 [pRS202], FY11 [pRS202-COS4], FY11 [pRS202]. <br />  <br />  <br />  <br />  <br /> vps mutants  cos9 mutants did exhibit             cdc1-1(Ts) mutant 308 (data shown).  loss <br /> sporulation defect). determine group IIIB COS              class D Vps function suppressed cdc1(Ts) growth <br /> genes involved vacuolar protein sorting,         defect. COS6 COS11 distinct PEP7 examined localization vacuolar protein CPY.               define class D VPS genes. <br /> Wild-type strains efficiently target CPY vacuole.            COS8 identical VPS4/END13, class E VPS gene: <br /> Mutants defects vacuolar protein sorting             cos8 cells exhibited single, prominent vacuolar struc- <br /> pathway, contrast, secrete significant amounts CPY         ture surrounded small, CDCFDA-stain- <br /> (Raymond et al. 1992). Similar class C vps18 (cos4-       ing vesicles (Figure 4). vacuolar morphol- <br /> 8) mutant, group IIIB cos mutants (cos6, cos8,        ogy resembled wild-type cells (Figure 4), class cos9, cos11 mutants) secreted significantly CPY         class E vps mutants contain normal-looking wild-type strain (Figure 3, B C). sorting       vacuoles (Raymond et al. 1992).  molecular char- <br /> defect apparent, obvious,           acterization  materials methods) showed cos9 mutant (Figure 3C).  group IIIB COS                COS8 identical VPS4/END13, class E VPS gene <br /> genes necessary efficient vacuolar protein sort-         implicated vacuolar protein sorting (Munn Riez- <br /> ing. contrast, cos mutants groups II IV did          man 1994). Furthermore, vps4D::URA3 disruption sup- exhibit significant CPY secretion (Figure 3),            pressed cdc1-1(Ts) growth defect 308 (data group cos mutants displayed slight defect CPY             shown), indicating loss class E Vps function sup- <br /> secretion (Figure 3A) severe older           pressed cdc1-1(Ts) growth defect. <br /> colonies.                                                           VPS4 identified high-copy sup- <br />    Mutations class D VPS genes suppress cdc1-1(Ts)        pressor cdc1-1(Ts) growth defect  materials <br /> growth defect: determine group IIIB cos mutations          methods), determined Vps4 overproduction <br /> affected vacuolar biogenesis, examined vacuolar               blocked vacuolar protein sorting. shown Figure 3D, <br /> morphology using vital stain CDCFDA. contrast             increase VPS4 dosage caused CDC1 cdc1-1(Ts) multilobed vacuolar structure wild-type cells          cells secrete CPY, consistent idea (Figure 4), cos6 cos11 mutant cells displayed single       Vps4 overproduction suppressed cdc1-1(Ts) growth <br /> vacuolar lobe (Figure 4 data shown)             defect impeding vacuolar protein sorting. <br /> similar abnormal vacuolar morphology class D              Loss Arf1 function suppresses cdc1-1(Ts) growth <br /> vps mutants (Raymond et al. 1992). Like class D vps mu-          defect: group IIIB mutant, cos9, displayed numer- <br /> tants, cos6 cos11 cells failed form vacuolar segrega-     ous (.15) CDCFDA-staining vesicles, indicative vacuo- <br /> tion structures exhibited defect vacuolar inheri-       lar fragmentation (Figure 4; phenotype clearer <br /> tance (Figure 4 data shown). Finally, disruption         Nomarski optics vesicles outside focal class D VPS gene, VPS19/PEP7/VAC1 (Weisman                plane interfered visualization CDCFDA fluo- Wickner 1992), alleviated growth defect             rescence). Interestingly, cos9 mutants exhibited abnor- <br />                                                       Mn21 Homeostasis Yeast                                              1793 <br />  <br />                                                                     major role protein turnover. vacuolar hy- <br />                                                                     drolases, including proteinase B CPY, activated <br />                                                                     vacuolar Pep4 proteinase (proteinase .                                                                     result, loss Pep4 function results 90&#226;&#8364;&#8220;95% reduc- <br />                                                                     tion vacuolar hydrolase activity ( Jones et al. 1982). <br />                                                                      cdc1-1(Ts) pep4D::LEU2 strain failed                                                                     grow 308 (data shown).  loss <br />                                                                     Pep4 accompanying reduction vacuolar <br />                                                                     hydrolase activity account cdc1(Ts) suppres- <br />                                                                     sion defects vacuolar biogenesis. <br />                                                                        Group II gene COS16: group II cos mutants exhib- <br />                                                                     ited subset group III cos mutant phenotypes, <br />                                                                     including Mg21 sensitivity (Figure 2), failure sporu- <br />                                                                     late, slight growth defect 368 (data shown). <br />                                                                      group II cos mutants did secrete CPY (Fig- <br />                                                                     ure 3). cloned COS16 gene yeast genomic <br />                                                                     library complementation temperature-resis- <br />                                                                     tant growth cdc1-1(Ts) cos16-57 mutant. com- <br />                                                                     plementing fragment mapped region chro- <br />                                                                     mosome IIIR (data shown), contained single <br />                                                                     1071-bp open reading frame, YCR44c. frameshift mu- <br />                                                                     tation introduced YCR44c coding region abol- <br />                                                                     ished ability 2.1-kb fragment complement <br />                                                                     cos16-57 mutation (data shown). addition,                                                                     YCR44c disruption suppressed cdc1(Ts) temperature- <br />                                                                     sensitive growth defect (data shown) conferred <br />                                                                     Mg21 sensitivity temperature-sensitive growth  368) <br />                                                                     CDC1 strain (data shown). Finally,                                                                     spontaneous cos16 alleles tightly linked (,2.5 cM <br />    Figure 4.&#226;&#8364;&#8221;Vacuolar morphology cos mutants visualized          20 tetrads) MAT locus, predicted CDCFDA. Exponentially growing cells stained        physical proximity YCR44c MAT.  COS16 <br /> vital dye CDCFDA visualized differential interference        identical open reading frame YCR44c. <br /> contrast (DIC) fluorescence (CDCFDA) microscopy. Strains: <br /> COS1 (FY 11 [YIp5-C DC1]); cos4 (vps18) (FY 11 cos4-8 [YIp5- <br />                                                                        Subcellular localization Cos16: predicted pri- <br /> C DC1]); cos6 (FY11 cos6-18 [YIp5-CDC1]); cos8 (vps4) (FY11 cos8-   mary structure Cos16 protein did exhibit sig- <br /> 21 [YIp5-CDC1]); cos9 (arf1) (FY11 cos9-32 [YIp5-CDC1]).            nificant homology proteins available databases. <br />                                                                      hydropathy analysis (Kyte Doolit- <br />                                                                     tle 1982) showed Cos16 contained 8 putative mem- <br /> malities cell size shape (Figure 4), suggesting Cos9         brane-spanning helices, suggesting inte- affect cellular processes. cloned COS9             gral membrane protein. examine Cos16 localization, identical ARF1 gene  materi-            constructed hybrid gene COS16 lacZ. <br /> als methods), encodes GTPase implicated                 single copy COS16-lacZ fusion integrated transport endoplasmic veticulum (ER)             URA3 locus cos16D::HIS3 mutant. (.80%) <br /> Golgi compartments (Stearns et al. 1990). expected,              Cos16-LacZ fusion protein detected par- arf1D::HIS3 disruption allowed cdc1-1(Ts) mutant           ticulate fraction (100,000 g 1 hr), solubil- <br /> grow 308 (data shown). ARF1                 ized Triton X-100, 1 m NaCl 2 m <span id='am-17' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span> previously described VPS gene, arf1D::HIS3            (data shown).  Cos16-LacZ fusion protein mutant secreted CPY, displayed abnormal vacuolar mor-               probably integral membrane protein. <br /> phology, exhibited decreased tolerance salt.                 subcellular localization Cos16-LacZ fusion <br /> cause Arf1 function required normal vacuolar bio-            protein initially examined crude fractionation. <br /> genesis, cos9 mutations probably suppress cdc1-             Cell lysates fractionated sequential centrifuga- <br /> 1(Ts) growth defect debilitating vacuolar function.              tion (Cowles et al. 1994), fusion protein   Proteinase deficiency does suppress cdc1-1(Ts)          fraction detected Western blotting. Greater <br /> growth defect: group III cos mutations                  90% Cos16-LacZ fusion protein cofrac- <br /> blocked formation normal vacuole, cdc1(Ts)                 tionated vacuolar membrane marker, Vph1, growth defect probably alleviated absence             ER lumenal marker, Kar2, P13 fraction normal vacuole, defect specific stage          (data shown). determine COS16 encoded vacuolar protein sorting. yeast vacuole plays              vacuolar membrane protein, vacuoles purified  1794                                             M. Paidhungat S. Garrett <br />  <br />                                                                  uolar localization Cos16-LacZ fusion protein <br />                                                                  probably reflects subcellular location native <br />                                                                  Cos16 protein. <br />                                                                     Cos16 Mn21 homeostasis: vacuole <br />                                                                  sequesters Mn21 cytosol (Okorokov et al. 1977), <br />                                                                  examined Cos16 involved Mn21 homeo- <br />                                                                  stasis. cos16D::HIS3 deletion modest effect, <br />                                                                  similar vps4 mutation, Mn21 sensitiv- <br />                                                                  ity wild-type strain (data shown). Muta- <br />                                                                  tions Golgi Mn21 transporter gene, PMR1, increase <br />                                                                  sensitivity Mn21 result Mn21 accumulation <br />                                                                  cytosol (Lapinskas et al. 1995). cos16D::HIS3 <br />                                                                  deletion exacerbated sensitivity (Figure 6A).  <br />                                                                  deletion COS16 altered tolerance Mn21. <br />                                                                     examined effect COS16 chelator <br />                                                                  sensitivities Mn21-homeostasis mutants. Strains <br />                                                                  lacking Pmr1 function sensitive EGTA,                                                                  sensitivity ameliorated increasing Mn21 influx <br />                                                                  plasma membrane (Paidhungat Gar- <br />                                                                  rett 1998). shown Figure 6B, cos16D::HIS3 <br />                                                                  deletion allowed pmr1D::LEU2 strain grow  <br />                                                                  dium containing 0.5 mm EGTA. Mutations SMF1                                                                  cause cells sensitive EGTA, case                                                                  result loss high-affinity Mn21 uptake <br />                                                                   EGTA sensitivity alleviated Mn21 sup- <br />                                                                  plement (Supek et al. 1996), overproduction <br />                                                                  Cdc1 (Paidhungat Garrett 1998).    Figure 5.&#226;&#8364;&#8221;Subcellular localization Cos16-b-gal fusion       cos16D::HIS3 smf1D::URA3 double mutant tolerated <br /> protein. Protein cell lysates (Total) purified <br />                                                                  4 mm EGTA medium (Figure 6C). Finally, vacuoles (Vacuole) separated SDS-PAGE trans- <br /> ferred nitrocellulose membrane. b-gal lane contains       EGTA sensitivity (1.5 mm) cdc1-1(Ts) strain purified b-galactosidase. Vacuoles purified previously   alleviated cos16D::HIS3 deletion (Figure 6D). <br /> described (Cardenas Heitman 1995). membrane              findings implicate Cos16 Mn21 homeostasis probed antibodies bacterial b-gal protein,   suggest Cos16 involved sequestration yeast proteins CPY, Kar2, Pma1. <br />                                                                  Mn21 vacuole. result consistent <br />                                                                  notion cos16 mutations alleviate                                                                  cdc1(Ts) conditional growth defect effect <br /> flotation Ficoll gradient (Cardenas Heitman             Mn21 homeostasis. <br /> 1995), analyzed presence Cos16-LacZ                  Mn21 content cdc1 cos16 mutants: genes <br /> fusion protein. intact Cos16-LacZ fusion              affect Mn21 homeostasis, including PMR1 ATX2, <br /> riched vacuolar extract extent             alter intracellular Mn21 content (Lapinskas et al. 1995; <br /> comparable vacuolar lumenal marker, CPY (Fig-             Lin Culotta 1996). contrast, intracellular Mn21 <br /> ure 5). contrast, ER protein, Kar2,               levels unaffected overexpression CCC1, high- <br /> plasma-membrane protein, Pma1, significantly            copy suppressor pmr1 Mn 21 defect (Lapinskas <br /> enriched vacuolar fraction (Figure 5). Similar            et al. 1996). results suggested Cdc1 studies using vacuolar vesicles purified different          Cos16 affected intracellular distribution Mn21, method (Ohsumi Anraku 1981) yielded identical                examined mutations CDC1 COS16 altered <br /> results (data shown). results suggest             intracellular Mn21 levels judged atomic absorption <br /> Cos16 integral membrane protein vacuole.            spectroscopy. Mn21 content cdc1-1(Ts) (38.9 6 <br />    Misfolded membrane proteins default vacuole            8.3 pmols/A600) cos16D::HIS3 (41.4 6 12.3 pmols/ <br /> (Stack Emr 1993).  unlikely            A600) mutants similar Mn21 content explanation vacuolar localization Cos16-          isogenic wild-type (41.2 6 7.0 pmols/A600) strain, sug- <br /> LacZ fusion protein, single-copy COS16-lacZ              gesting depletion Cdc1 Cos16 complemented cos16D phenotypes (data              significant effect cell Mn21 content. Cellular <br /> shown).  different epitope-tagged COS16               Mn21 levels unaffected CDC1 overexpres- <br /> fusion, myc epitope fused C              sion (data shown).  Cdc1 Cos16 terminus Cos16, functional, exhibited simi-         affect intracellular distribution Mn21 lar localization pattern (data shown).  vac-        changing overall cellular content. <br />                                                     Mn21 Homeostasis Yeast                                                 1795 <br />  <br />  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;COS16 Mn21 homeostasis.   cos16D exacerbates Mn21 sensitivity pmr1D mutant. indicated strains streaked YEPD pH 5.5 agar, supplemented 0 mm, 2 mm, 4 mm MnCl2, incubated 308 3&#226;&#8364;&#8220;5 days. <br /> Strains  cos16D pmr1D (FY 70 cos16D::HIS3 pmr1D::LEU2); COS16 pmr1D (FY70 pmr1D::HIS3), cos16D PMR1 (FY70 cos16D::HIS3); <br /> COS16 PMR1 (FY70). (B) cos16D suppresses EGTA sensitivity pmr1D mutant. strains panel streaked YEPD agar, supplemented 0 mm 0.5 mm EGTA, incubated 308 3&#226;&#8364;&#8220;5 days. (C) cos16D suppresses EGTA <br /> sensitivity smf1D mutant. 10-fold dilutions exponentially growing cultures indicated strains spotted YEPD <br /> agar YEPD agar containing 4 mm EGTA, incubated 308 3 days. Strains  1 (FY70), cos16D (FY70 cos16D::HIS3), <br /> smf1D (FY70 smf1D::URA3), cos16D smf1D (FY 70 cos16D::HIS3 smf1D::URA3). (D) cos16D suppresses EGTA sensitivity cdc1-1(Ts) mutant. indicated strains streaked YEPD agar YEPD agar containing 1.5 mm EGTA incubated 308 3 days. Strains  cdc1 cos16D (FY11 cos16D::HIS3); cdc1 (FY11); CDC1 (FY70); cos16D (FY70 cos16D::HIS3). <br />  <br />  <br />  <br />    Cdc1 dispensable cos16D mutants: Cos16             .90% expected cdc1D::HIS3 cos16D::LEU2 prog- involved vacuolar Mn21 homeostasis,              eny formed healthy colonies supplemented determined cdc1D cos16D double mutant grow               2 mm 4 mm Mn21 (Figure 7).  Mn21 augments presence exogenous Mn21. Isogenic diploids                ability cos16D deletion bypass Cdc1 function. <br /> (cdc1D::HIS3/CDC1 COS16/COS16 cdc1D::HIS3/ <br /> CDC1 cos16D::LEU2/COS16) sporulated dis- <br /> sected YEPD medium agar   Mn21                                        DISCUSSION <br /> supplement. viable His2 (CDC11) segregants recovered tetrad cdc1D::HIS3/CDC1                     Previous studies implicated Cdc1 regulation COS16/COS16 diploid (Figure 7), 2:0 segrega-                intracellular Mn21 (Loukin Kung 1995; Paidhun- <br /> tion affected Mn21 (data shown).              gat Garrett 1998). hypothesis based trast, tetrads cdc1D::HIS3/CDC1 cos16D::LEU2/              observations, including: (1) Mn21 supplement <br /> COS16 diploid yielded 60% expected cdc1D::HIS3               partially alleviated conditional cdc1(Ts) growth  <br /> cos16D::LEU2 segregants (Figure 7).  Cdc1               fect; (2) CDC1 overexpression ameliorated EGTA <br /> essential strains lacking Cos16 function.               sensitivity  unrelated Mn21-homeostasis mutants. <br />  1796                                              M. Paidhungat S. Garrett <br />  <br />                                                                   block vacuolar Mn21 sequestration                                                                   vacuolar biogenesis mutations relieve                                                                   EGTA-sensitive phenotype cdc1(Ts) mutants (data <br />                                                                   shown). results eliminate vacuole                                                                   essential, Cdc1-dependent Mn21 store suggest                                                                   vacuolar biogenesis mutations suppress cdc1(Ts) <br />                                                                   growth defect attenuating depletion cytosolic <br />                                                                   Mn21. <br />                                                                      Lesions vacuole biogenesis (VPS) genes affect                                                                   vacuolar functions, including proteolysis, Ca 21 accumu- <br />                                                                   lation, proton uptake Mn21 sequestration.  <br />                                                                   conditional cdc1(Ts) growth defect suppressed <br />                                                                   mutations affect processes. <br />                                                                   particular, cdc1(Ts) growth defect suppressed <br />                                                                   inactivation protein, Cos16, appears <br />                                                                   involved Mn21 sequestration vacuole. <br />    Figure 7.&#226;&#8364;&#8221;Cdc1 dispensable cos16 mutant. Heterozy- <br />                                                                   lines evidence support assertion.  <br /> gous cdc1D::HIS3/C DC1 COS16/COS16 (FY 1) cdc1D::HIS3/         Cos16 integral membrane protein vacuole, <br /> C DC1 cos16D::LEU2/COS16 (FY 1 cos16D::LEU2) diploids        judged subcellular localization pattern sporulated dissected rich yeast medium      functional Cos16-LacZ fusion protein (Figure 5).  Mn21 supplement  materials methods). Tiny            observation consistent presence puta- <br />  small) colonies cdc1D::HIS3 cos16D::LEU2 deter- <br />                                                                   tive transmembrane domains predicted cod- <br /> mined growth minimal medium agar lacking histi- <br /> dine leucine.   b, c, d refer individual spores   ing region.  Cos16 appropriately positioned tedrad.)                                                        affect transport cytosol vacuole. Sec- <br />                                                                   ond, cos16D deletion suppresses chelator sensitivity <br />                                                                   strain (smf1D) compromised Mn21 uptake                                                                   cytosol (Figure 6C), suggesting loss Cos16  certain genetic backgrounds, Mn21           function compensates low influx cytosol. <br /> supplement completely bypasses essential Cdc1              cos16 mutations relieve chelator sensitivity quirement.  cdc1D cos16D double mutant exhibits          mutant (pmr1D) efficiently suppressed robust growth medium supplemented                genetic manipulations raise cytosolic Mn21 (Figure <br /> Mn21 (Figure 7). COS16 involved intra-         6B; Paidhungat Garrett 1998). Finally, Cos16 <br /> cellular Mn21 homeostasis, results best explained       inactivation exacerbates Mn21 sensitivity model Mn21 regulation single, essential      wild-type (data shown) pmr1D mutant strains <br /> function Cdc1. results rule possibility     (Figure 6A), presumably aggravating accumula- Mn21 dependence cdc1 strains reflects fact        tion cytosolic Mn21.  Cos16 appears antago- Cdc1 Mn21-dependent enzyme (Supek et al. 1996;          nize cdc1(Ts) growth defect sequestering cytosolic <br /> Paidhungat Garrett 1998).                                     Mn21 vacuole. <br />    mechanism Cdc1 affects intra-               COS16 CDC1 involved homeostasis cellular Mn21 levels clear, evidence favors          intracellular Mn21, intracellular Mn21 levels model Cdc1 functions maintain cytosolic               vary changes Cos16 Cdc1 function? previous <br /> Mn21. Mn21 depletion Golgi elicits protein             report (Okorokov et al. 1977) suggested intra- <br /> glycosylation defect observed cdc1(Ts) mu-             cellular Mn21 vacuole. According tants, cdc1(Ts) growth defect exacerbated,             report, defect cytosolic Mn21 retention, predicted alleviated, alterations increase Mn21 flux            cdc1(Ts) mutants, affect total intracellular cytosol Golgi (Paidhungat Gar-                Mn21 content.  harder reconcile fact <br /> rett 1998).  suggested cytosol,           cos16D deletion did alter level intracellu- Golgi apparatus, likely site             lar Mn21. think possible explanations <br /> cdc1(Ts) Mn21 depletion (Paidhungat Garrett                   apparent paradox.  Cos16 1998). major intracellular Mn21 store              involved regulation ions, affect organ- <br /> vacuole (Okorokov et al. 1977). vacuole              elle copurify vacuole. known play role Mn21 detoxification,           formally ruled possibilities, known cellular function(s) vacuolar           easily account observation cos16 mutations <br /> Mn21 pool.  mutations inhibit vacuolar               affect growth Mn21 homeostasis mutants. Alterna- <br /> biogenesis  function) distinct steps sup-          tively, Cos16 inactivation block cytosolic vacuole <br /> press, exacerbate, cdc1(Ts) growth             Mn21 transfer having measurable effect vacu- <br /> fect. Suppression conditional growth probably results          olar Mn21 content. <br />  <br />  <br />    Finally, COS16 vacuolar biogenesis                 <br />                                                                            <br /> genes characterized study, identified                    suppressors cdc1(Ts) growth defect.                              <br /> suppressor, cos13, falls phenotypic                   <br />                                                                      <br /> epistasis group cos16   identify                         component complex pathway involved vacuo-                      <br /> lar Mn21 homeostasis. contrast, remaining reces-            <br /> sive suppressors (cos1, cos3, cos14, cos2, cos7, cos10)         <br /> confer phenotypes obviously related                          <br /> Mn21 regulation.  genes corresponding                suppressors specify receptors Cdc1            <br />  Mn21?)-dependent growth process. Future studies                  genes help elucidate role Cdc1                          Mn21 homeostasis growth.                                             <br /> </body></html>