<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>2304                                                                       Biophysical Journal   Volume 91   September 2006      2304&#226;&#8364;&#8220;2311 <br />  <br />  <br /> Using Topology Metabolic Networks Predict Viability Mutant Strains <br />  <br /> Zeba Wunderlich* Leonid  Mirnyy <br /> *Biophysics Program, Harvard University, Cambridge, Massachusetts; yHarvard-MIT Division Health Sciences Technology, <br /> Cambridge, Massachusetts <br />  <br />  <br /> ABSTRACT Understanding relationships structure (topology) function biological networks central <br /> question systems biology. idea topology major determinant systems function attractive highly disputed hypothesis. structural analysis interaction networks demonstrates correlation topological properties node (protein, gene) network functional essentiality, analysis metabolic networks <br /> fails &#239;&#172;?nd correlations. contrast, approaches utilizing topology biochemical parameters metabolic <br /> networks, e.g., &#239;&#172;&#8218;ux balance analysis, successful predicting phenotypes knockout strains. reconcile seemingly con&#239;&#172;&#8218;icting results showing topology metabolic networks Escherichia coli Saccharomyces cerevisiae  fact, suf&#239;&#172;?cient predict viability knockout strains accuracy comparable &#239;&#172;&#8218;ux <br /> balance analysis large, unbiased mutant data sets. surprising result obtained introducing novel topology-based <br /> measure network transport: synthetic accessibility. popular topology-based characteristics node degree, graph diameter, node usage (betweenness) fail predict viability E. coli mutant strains. success synthetic accessibility demonstrates ability capture essential properties metabolic network, branching <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> reactions directed transport material inputs outputs. results strongly support link topology function biological networks  agreement recent genetic studies, emphasize minimal <br /> role &#239;&#172;&#8218;ux rerouting providing robustness mutant strains. <br />  <br />  <br /> INTRODUCTION suggested debated idea topology                    single-gene knockout strains. EMA predict viability <br /> determines network function. structures               mutant strains comparable accuracy. biological networks available, remains hard separate            methods use network topology, stoichiometry contributions topology contributions                   metabolic chemical transport reactions, cases, <br /> kinetic equilibrium parameters.                 maximal rates reactions, established structures wealth related experimental             separate role topology role played data, Escherichia coli Saccharomyces cerevisiae                   parameters network function. addition, metabolic networks perfect model systems explore               complexity method results, EMA techniques <br /> role network topology. topology metabolic network              computationally expensive (12) provide little insight <br /> sufficient predict viability knockout mutants?                  certain mutations lethal,    Metabolic networks modeled extensively using                 tolerated. <br /> steady-state flux balance approaches (1&#226;&#8364;&#8220;8). test                      untangle topology stoichiometry capabilities metabolic network models, groups                metabolic network topology sufficient <br /> compared predicted experimentally measured effects                 predict viability E. coli S. cerevisiae <br /> gene deletions cell growth. effective                   mutant strains accurately FBA large, unbiased sets <br /> methods flux balance analysis (FBA) (3,4,6&#226;&#8364;&#8220;9),                 mutants (9,13,14). result supports claim lated minimization metabolic adjustment (MOMA) method                  topology plays central role determining network func- <br /> (10), elementary mode analysis (EMA) (11).                   tion malfunction (15,16). employ novel network methods shown useful understand-                 property, synthetic accessibility, intuitive trans- <br /> ing structure dynamics metabolic fluxes,                  parent way understanding effects metabolic muta- <br /> deliver different experimentally testable predictions. FBA                tion (Fig. 1). define synthetic accessibility, S, total accurately predict fluxes individual reactions             number reactions needed transform given set input wild-type mutant strains viability              metabolites set output metabolites predict                                                                           increases S result alterations topology                                                                           metabolic network adversely affect growth. term <br /> Submitted December 30, 2005, accepted publication 24, 2006.   &#226;&#8364;&#732;&#226;&#8364;&#732;synthetic accessibility&#226;&#8364;&#8482;&#226;&#8364;&#8482; borrowed field drug <br /> Address reprint requests Leonid Mirny, Harvard-MIT Div. Health      design, defined smallest number chem- <br /> Sciences <span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; Technology, Massachusetts Institute Technology, 77          ical steps needed synthesize drug common labo- <br /> Massachusetts Ave., Cambridge, MA 02139. Tel.: 617-452-4862; Fax:         ratory reactants similar spirit &#226;&#8364;&#732;&#226;&#8364;&#732;scope&#226;&#8364;&#8482;&#226;&#8364;&#8482; 617-253-2514; E-mail: leonid@mit.edu. <br /> &#195;&#8220; 2006 Biophysical Society <br /> 0006-3495/06/09/2304/08 $2.00                                                                                doi: 10.1529/biophysj.105.080572 <br />  Topology-Based Metabolic Predictions                                                                                                                         2305 <br />  <br />  <br />                                                                                    Construction graphic metabolism model <br />                                                                                    reactions included E. coli metabolic network taken                                                                                    Edwards Palsson (4), reactions included yeast metabolic <br />                                                                                    network taken Duarte et al. (8). updated <br />                                                                                    version E. coli metabolic network available (6), chose use                                                                                    previous version enable comparison synthetic accessibility perfor- <br />                                                                                    mance previous studies (4,9&#226;&#8364;&#8220;11). reaction metabolite repre- <br />                                                                                    sented node, directed edges connect reactants reactions                                                                                    reactions products, accounting reversibility reactions. <br />  <br />  <br />                                                                                    Selection input output metabolite sets <br />                                                                                    input metabolites E. coli minimal medium, E. coli rich medium,                                                                                    various yeast medium conditions listed Supplementary Material, <br />                                                                                    Tables S1&#226;&#8364;&#8220;S4. E. coli minimal medium consists energy source <br />                                                                                    (glucose, acetate, glycerol, succinate), components minimal medium, <br />                                                                                    sulfur source, carbon dioxide oxygen, <span id='am-41' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nicotinamide</span> mononucleotide, <br />                                                                                    regulatory protein thioredoxin (Supplementary Material, Table S1). <br />                                                                                    input metabolites chosen match real composition minimal <br /> FIGURE 1 illustration synthetic accessibility approach.          medium closely possible. Nicotinamide mononucleotide thiore- <br /> representation metabolic network, circular nodes represent metabo-          doxin included ensure  wild-type network, components <br /> lites, rectangular nodes represent reactions, directed edges indicate    output biomass accessible. chosen specifically relationships. Nodes outline (green blue) synthetically        upstream metabolites biomass synthesis pathways. <br /> accessible, nodes outline (red orange) accessible.      E. coli rich medium consists metabolites minimal medium algorithm begins identifying reactions neighbor input      biotin, riboflavin, pantoate, <span id='am-18' about='obo:IMR_0200344' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00314" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Pyridoxine" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200344" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyridoxine</span>, thiamin, dihydrofolate, <br /> metabolites (nodes A&#226;&#8364;&#8220;D) marking reactions                p-aminobenzoic acid, 20 amino acids, <span id='am-28' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-35' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> bases <br /> reactants available accessible (reactions 1 2). products     included external metabolites metabolic network (external <span id='am-36' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span>thymine</span> reactions marked accessible (nodes F&#226;&#8364;&#8220;H). algorithm              metabolic network). Rich medium difficult model <br /> examines neighboring reactions newly marked metabolites           accurately, using slightly different input metabolite sets step continues new metabolites marked accessible.       significant effect results (results shown). inset demonstrates happens gene produces enzyme               input metabolites yeast based descriptions Duarte catalyzes reaction 2 deleted: metabolites H K reaction 5         et al. (8) include histidine, leucine, <span id='am-46' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> compensate accessible  define synthetic accessibility, S,         deletions 3/Leu-2/Ura-3 mutant strains. Additionally, number reactions required transform set inputs set        thioredoxin (oxidized), H1  endoplasmic reticulum), <span id='am-23' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>  outputs. Synthetic accessibility analogous diameter directed        endoplasmic reticulum), dolichol included inputs, graph, contrast graph diameters, synthetic accessibility takes       components biomass producible, account branching nature chemical reactions purpose              wild-type network. <br /> metabolic networks, produce outputs inputs.                                    E. coli output metabolites taken components E. coli <br />                                                                                    biomass (Supplementary Material, Table S5) (19). yeast output metabo- <br /> metabolites (17,18). demonstrate network                        lites components biomass reaction reported (8). <br /> characteristics node degree change graph <br /> diameter unable predict viability E. coli mutant <br />                                                                                    Synthetic accessibility algorithm <br /> strains better random predictions, suggesting synthetic <br /> accessibility appropriate characteristic networks                    determine synthetic accessibility outputs given inputs, directed transport, metabolic networks.                               use type iterative breadth search, similar previously described <br />                                                                                    &#226;&#8364;&#732;&#226;&#8364;&#732;forward-firing&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Fig. 1) (20). algorithm starts examining                                                                                    reactions require given input metabolites reactant. MATERIALS METHODS                                                              marks reactions reactants available &#226;&#8364;&#732;&#226;&#8364;&#732;accessible&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                                    marks metabolites produced reactions &#226;&#8364;&#732;&#226;&#8364;&#732;accessible&#226;&#8364;&#8482;&#226;&#8364;&#8482; De&#239;&#172;?nition synthetic accessibility                                                algorithm examines reactions require newly <br />                                                                                    marked metabolites starting material, determines reaction <br /> Consider metabolic network access certain inputs: substrates         accessible based availability reactants, consumed environment (e.g., sugars, oxygen, nitrogen),           new metabolites marked accessible. Concurrently, number aim producing certain outputs amino acids, nucleotides,         steps needed reach accessible metabolite j, synthetic accessibility components collectively called biomass. define synthetic          Sj, recorded; synthetic accessibility network S calculated accessibility Sj output j minimal number metabolic reactions       summing synthetic accessibilities outputs. <br /> needed produce j network inputs (Fig. 1). Sj set infinity j synthesized network inputs. Summing synthetic <br /> accessibility components biomass, obtain total              Comparison experimental results <br /> synthetic accessibility S &#194;&#188;  Si biomass. propose enzyme knockout does change S, e., biomass produced               compare results approach experimental data sets, extra metabolic cost, mutant viable. S &#194;&#188; N,         create adjacency matrix, represents wild-type metabolic net- <br /> essential component biomass produced network                 work topology.  mutant strain, create &#226;&#8364;&#732;&#226;&#8364;&#732;mutated&#226;&#8364;&#8482;&#226;&#8364;&#8482; adja- <br /> inputs, predict lethal phenotype.                               cency matrix removing reactions catalyzed gene.  <br />                                                                                                                       Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  2306                                                                                                                                       Wunderlich Mirny <br />  <br /> E. coli predictions, previous papers, delete corresponding           unrelated reactions. measure performance synthetic acces- <br /> genes reactions catalyzed multiple isozymes. calculate              sibility erroneous network plotting accuracy percentage <br /> viability mutant compare results experimental data          shuffled assignments. <br /> Supplementary Material). Smutant &#194;&#188; Swild-type, predict mutant viable; predict inviable. E. coli insertional mutant data set, <br /> phenotype data given competitive growth rates. mutant                      RESULTS <br /> considered negatively selected  inviable) twofold decrease growth rates 30 generations (9). Gerdes et al. data set (21),       Performance synthetic accessibility create mutated adjacency matrices genes included metabolic             E. coli metabolic network <br /> network model, resulting 598 mutated adjacency matrices. yeast <br /> experimental data, use preprocessed data set created Duarte et al.           study performance synthetic accessibility (8) simultaneously delete isozymes.                                         predicting viability knockout strains compare                                                                                        previous studies, applied method E. coli <br />                                                                                        metabolic network. initially tested data sets <br /> Calculation topology-based predictions <br />                                                                                        used previous studies: large, unbiased data set explore number topology-based measures predictors                  insertional (transposon-induced) mutants (9) smaller <br /> E. coli mutant viability, including node degree, diameter, node usage. <br />                                                                                        data set collected FBA analysis (4), mainly degree enzyme calculated summing degrees reactions catalyzed enzyme isozymes. define network                  contained knockouts enzymes involved central metab- <br /> diameter sum metabolites versus metabolites&#226;&#8364;&#8482; shortest paths,         olism. mutants grown minimal medium. mutant, calculate change network diameter wild-            used data sets specifically used type. define node usage enzyme number times                  previous studies (4,9&#226;&#8364;&#8220;11), compared results. <br /> reactions catalyzed enzyme used produce biomass wild- <br />                                                                                        used union data sets refer type strain, according synthetic accessibility approach, essentially analogous betweenness (22,23). measure, degree,                combined data set. applied combined data <br /> diameter, usage, predict enzyme essential                set, approach performed (62% accuracy, p &#194;&#188; 6 3 corresponding mutant stain inviable), measure greater            10 8) FBA approach (62%, p &#194;&#188; 3 3 10 8)  Table given cutoff. vary cutoff entire range possible         1, Fig. 2 details). large data set 487 insertional <br /> values value gives optimal performance, measured                                                                                        mutants (9), synthetic accessibility approach performed accuracy significance x 2 statistic. <br />                                                                                        (60% accuracy, p &#194;&#188; 3 3 10 5) FBA                                                                                        MOMA approaches (58% 59% accuracy, p &#194;&#188; 1 3 10 3 <br /> Quantitative analysis performance                                                   1 3 10 4, respectively), somewhat higher <br />                                                                                        statistical significance. smaller data set 79 mutants assess performance synthetic accessibility methods predicting phenotype mutant stains, use measures: accuracy,             (4), FBA correctly predicted 86% cases, sensitivity, specificity, p-value x 2 statistic. define accuracy     topology-based synthetic accessibility approach 71% (TP 1 TN)/(TP 1 TN 1 FP 1 FN), TP number true                       accuracy, providing correct predictions 53/68 &#194;&#188; 78% positives, TN number true negatives, FP number false               cases predicted correctly FBA. <br /> positives, FN number false negatives. define positives negatives terms experimentally measured phenotypes,                                                                                           difference performance synthetic <br /> positives viable strains negatives inviable strains,            accessibility approach compared FBA assignment arbitrary reversed. similar fashion, define           data sets probably related way data sets <br /> sensitivity TP/(TP 1 FP) specificity TN/(TN 1 FN). calculate              interpreted cases included data sets. x 2 statistic, use two contingency tables sort mutant          smaller data set, mutant strains classified viable strain based silico vivo phenotypes calculate appropriate p-value. <br />                                                                                        inviable, insertional data set, mutants                                                                                        labeled negatively selected  population mutant <br />                                                                                        strain half wild-type population 30 <br /> Assessment synthetic accessibility robustness                                       generations competitive growth) negatively selected. test robustness approach, introduce random mistakes              synthetic accessibility approach deems mutant E. coli network randomly reassigning certain fraction enzymes           strain inviable negatively selected based path <br />  <br /> TABLE 1 Comparison accuracy statistical signi&#239;&#172;?cance FBA, MOMA, EMA, synthetic accessibility methods <br /> applied E. coli metabolic network <br />                                                                                                           Method <br />  <br />   Mutant data source               cases         Synthetic accessibility               FBA                        MOMA                         EMA <br />                                                                                                     11                       <br /> Collected literature               79                 71%, 8 3 10 *  -5 <br />                                                                                       86%, 7 3 10    (4)                                     90%, 3 3 10 14 (11) <br /> Insertional mutants                    481                 60%, 3 3 10 5              58%, 1 3 10 3 (9)          59%, 1 3 10    4 <br />                                                                                                                                     (10)             &#226;&#8364;&#8220; <br /> Combined data sets                     560                 62%, 6 3 10 8              62%, 3 3 10 8                      &#226;&#8364;&#8220;                           &#226;&#8364;&#8220; <br /> Gerdes data set                        598                 74%, 1 3 10 5                      &#226;&#8364;&#8220;                          &#226;&#8364;&#8220;                           &#226;&#8364;&#8220; <br /> *Accuracy, p-value x 2 statistic. <br />  <br /> Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                            2307 <br />  <br />  <br />  <br />  <br /> FIGURE 2 Performance synthetic accessibility compared FBA, MOMA, EMA, topology-based measures using E. coli metabolic <br /> network. graphs illustrate relative performance techniques using measures, accuracy, (TP 1 TN)/(TP 1 TN 1 FP 1 FN), negative <br /> log x 2 statistic&#226;&#8364;&#8482;s p-value, indicates correlation silico predictions vivo observations E. coli mutant strain viability. x2 statistic calculated using contingency table like ones Fig. 3 smaller data set (79 data points, 90 data points EMA), insertional <br /> mutant data set (487 data points), combined data set (560 data points) (4,9,11). larger, representative insertional mutant data set combined data set used, synthetic accessibility accurate statistically significant FBA.  synthetic accessibility performs poorly smaller data set, probably data set data points covers central metabolism, small fraction metabolic network. topology-based measures, degree diameter, perform worse FBA, MOMA, EMA, synthetic accessibility, indicating poorly <br /> characterize functioning metabolic network. random predictions using expected values produced FBA x 2 test represent expected performance correlation silico vivo predictions. vary little expected values x 2 tests used. <br />  <br />  <br /> lengths inputs outputs accessibility outputs,                     2. approach correctly predicted strains classification scheme correspond closely                            longer rerouted pathways inviable, suggesting synthetic accessibility approach: longer path lengths                            rerouting metabolic fluxes plays small role rescuing correspond reduced growth rates invia-                               mutant strains. result consistent results bility.                                                                                 FBA analysis yeast mutants (24). <br />    number type data points included data                           3. viable mutants untouched primary <br /> sets different. insertional data set larger                        synthetic pathways short rerouting (e.g., (487 vs. 79 data points) includes fairly random col-                              isozymes). <br /> lection insertions metabolic genes, smaller <br /> data set contains data enzymes used                               used previous FBA studies, central metabolism (glycolysis, pentose phosphate pathway,                           applied synthetic accessibility approach large- <br /> <span id='am-5' about='obo:IMR_0200473' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00158" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Citric acid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200473" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>citric acid</span> cycle, respiration processes) (4).                           scale knockout study Gerdes et al. (21), identified <br /> central metabolism contains number alternate pathways,                          genes essential robust growth rich medium using require fewer steps commonly                              genetic footprinting technique based transposon-based <br /> used pathways, surprising synthetic acces-                        mutagenesis. synthetic accessibility approach performed <br /> sibility approach performs poorly FBA                                 data set (74% accuracy, p &#194;&#188; 1 3 10 5). <br /> applied smaller data sets. <br />    regard combined data set, synthetic accessibility greater sensitivity, indicating better FBA                      Performance synthetic accessibility MOMA predicting strains viable,                            yeast metabolic network <br /> lower specificity, indicating good                                                                                      ensure success synthetic accessibility <br /> predicting inviable strains (Figs. 3 4). success                                                                                      method limited E. coli metabolic network, synthetic accessibility combined data set demonstrates <br />                                                                                      tested method metabolic network S. cerevisiae, important results, making transparent difference <br />                                                                                      metabolic network reconstructed viable nonviable strains. <br />                                                                                      hand (8). reconstruction extensively validated <br /> 1. nonviable mutants simply lack pathway                                   use FBA predict phenotypes large <br />    synthesize biomass components (S &#194;&#188; N),                              number single-gene knockout yeast strains grown    e., essential metabolites produced                        variety conditions (13,14). conditions include glucose <br />    network inputs (Table 2).                                                     minimal medium (MMD) rich medium defined <br />  <br />                                                                                                                          Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  2308                                                                                                                                   Wunderlich Mirny <br />  <br />  <br />  <br />  <br /> FIGURE 3 Results synthetic accessibility approach, divided type prediction. contingency graphs allow exploration types errors common. Results reported   combined E. coli data set, (B) Gerdes et al. (21) E. coli data set, (C) combined yeast <br /> data set (metabolic genes . x axis represents phenotypes predicted synthetic accessibility method, y axis represents experimental <br /> phenotypes. green blocks correspond cases prediction matches experiment, red, hashed blocks correspond errors. area box proportional number cases category. diagram, common type error synthetic accessibility <br /> approach predicts mutant viable actually inviable. <br />  <br />  <br /> carbon source (YPGal, galactose; YPD, glucose; YPDGE,                              essential slow sets, synthetic accessibility performs <br /> glucose-ethanol-glycerol, YPG, glycerol; YPE, ethanol;                         comparably FBA. conditions considered <br /> YPL, lactate). Sets essential slow-growth genes                        simultaneously, synthetic accessibility predicts phenotype identified experimentally genes                           83.7% accuracy, compared FBA 82.6% <br /> mutant strains constructed genes pro-                         accuracy. <br /> duced slow-growing mutant strains rich (YPD) medium.                               believe higher overall accuracy synthetic results (Table 3) gene sets                        accessibility FBA applied yeast metabolic <br />                                                                                    network probably largely result way data sets <br />                                                                                    used. E. coli data sets, predictions                                                                                    knockout strains involved genes                                                                                    included metabolic network model. yeast data <br />                                                                                    sets, following protocol previous FBA study (8), <br />                                                                                    predictions strains, gene                                                                                    included metabolic network model                                                                                     genes nonessential, predict knockouts                                                                                    genes absent metabolic network model viable, <br />                                                                                    inflates accuracy. report accuracies                                                                                    predictions metabolic gene knockouts Table 3,                                                                                    accuracies higher cases. <br />  <br />                                                                                    Performance topology-based measures <br />                                                                                    E. coli metabolic network <br />  <br /> FIGURE 4 Plot sensitivity specificity synthetic accessibility           tested ability topology-based graph charac- <br /> (SA) prediction methods. combined E. coli data set (560          teristics, node degree, graph diameter, node <br /> data points), sensitivity, TP/(TP 1 FP), specificity, TN/(TN 1 FN),        usage  Materials Methods), predict viability calculated predictions using synthetic accessibility, FBA, <br /> degree, diameter. cutoff values degree diameter selected <br />                                                                                    TABLE 2 E. coli mutants predicted inviable minimize x 2 test p-value. random values calculated using                                                                                    synthetic accessibility approach combined data set, <br /> expected values calculated x2 test FBA essentially                                                                                    divided reason predicting inviability values synthetic accessibility used instead. degree diameter good sensitivity, specificity quite low.    Reason predicting inviability    Correct (percent)    Incorrect (percent) <br /> synthetic accessibility FBA moderate values sensitivity <br />                                                                                     accessible outputs ,              89 (59%)              63 (41%) specificity. cases, sensitivity greater, implying                                                                                      wild-type (S &#194;&#188; N) <br /> viable predictions reliable inviable predictions,                                                                                    S . wild-type                            10 (67%)               5 (33%) seen Fig. 3. <br />  <br /> Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  Topology-Based Metabolic Predictions                                                                                                           2309 <br />  <br /> TABLE 3 Accuracy synthetic accessibility FBA methods predicting viability yeast deletion strains <br />                                                                                       Data set <br />  <br />                                         Essential    Slow    MMD      YPGal      YPD        YPDGE        YPG         YPE         YPL           cases                            118         83       564      564       565         565         565         565        565         4154 <br /> FBA (8)                                  31.4%      19.3%     84.0%    85.6%     84.4%       85.3%       86.5%       85.7%      86.4%        82.6% <br /> Synthetic accessibility                  11.9%       1.20%    94.0%    97.2%     85.3%       84.4%       83.7%       84.1%      84.1%        83.7%  cases  metabolic enzymes)   100         45       459      459       462         462         462         462        462         3373 <br /> FBA                                      33.0%       4.44%    95.0%    97.6%     87.9%       87.9%       89.2%       87.4%      88.3%        87.6% <br /> Synthetic accessibility                  14.0%       2.22%    94.3%    96.9%     88.5%       88.3%       89.0%       88.7%      88.7%        87.1% <br />  <br />  <br />  <br /> E. coli mutant strains. studies suggested            topological characteristics node degree diameter <br /> nodes higher degrees important              predictive information-carrying networks (e.g., network, removal nodes biological networks              Internet, protein&#226;&#8364;&#8220;protein interaction networks), results likely lead lethal phenotype (15,16). test          suggest characteristics synthetic accessi- hypothesis, computed degree enzyme              bility appropriate transport directed networks, number metabolites participating reactions cata-             metabolic networks. <br /> lyzed enzyme. strain predicted inviable degree knocked enzyme certain <br />                                                                        Robustness synthetic accessibility <br /> cutoff. Fig. 2 shows optimized cutoff value, procedure predicts viability worse random prediction.           Metabolic networks incomplete    theoretical studies focused graph diameter          contain errors. study predictions using measure network performance, defining graph                  synthetic accessibility depend errors network, <br /> diameter mean shortest paths pair             performed robustness analysis using E. coli meta- <br /> nodes (15,25,26). test graph diameter predictor             bolic network. Errors modeled random reassignment <br /> viability, predicted mutant inviable increase         certain percentages enzymes different reactions. Fig. <br /> graph diameter exceeded cutoff. Fig. 2 shows  similar           5 shows accuracy prediction decreased node degree, graph diameter did perform better              increased fraction introduced mistakes. method random predictions.                                               tolerated assignment error rates 5&#226;&#8364;&#8220;10%, accuracy <br />    Similarly, tested topology-based measure, <br /> enzyme usage, defined number times reactions catalyzed enzyme used produce <br /> biomass wild-type strain according synthetic <br /> accessibility approach. Enzyme usage analogous node <br /> betweenness, number shortest paths pairs nodes node (22,23). Enzyme <br /> usage performed somewhat better random predictions worse synthetic accessibility, surprising basically used subset data produced synthetic accessibility approach. <br />    summary, popular topology-based measures performed poorly synthetic accessibility.  node <br /> degree diameter accurate simply pre- <br /> dicting mutants viable, gives accu- <br /> racy 53.8%, node usage performed better                FIGURE 5 Accuracy synthetic accessibility approach node degree diameter, worse predictor           percentage enzyme-reaction assignments shuffled. assess robust- <br />                                                                        ness synthetic accessibility method errors topology synthetic accessibility  Supplementary Material). <br />                                                                        E. coli metabolic network, randomly shuffle given percentage    characteristics ignore essential properties              assignments enzymes reactions calculate accuracy metabolic network, directionality branching reactions,          synthetic accessibility method 10 trials. plot average accuracy directed transport material cellular substrates            percentage assignments shuffled, error bars noting (sugars, oxygen, ) products (biomass). Synthetic                minimum maximum observed accuracy. horizontal line denotes                                                                        accuracy predicting mutants viable, best expected result accessibility, contrast, takes account properties <br />                                                                        random network. approach relatively robust random errors metabolic network.  synthetic accessibility             enzyme-reaction assignments, clear expected trend thought generalization concept graph          lower accuracy great variability accuracy number diameter directed transport networks. certain             shuffled assignments increases. <br />  <br />                                                                                                          Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  2310                                                                                                              Wunderlich Mirny <br />  <br />  <br /> dropped level random predictions ;50%               tary Material) demonstrates known complexities metab- <br /> enzyme-reaction assignments shuffled.                            olism: metabolic pathway used produce specific <br />                                                                       product shortest                                                                        completely characterized sets input output meta- <br /> DISCUSSION <br />                                                                       bolites. Similar rates failure flux balance techniques study, topology function          suggest importance regulation adaptation muta- <br /> metabolic network intimately related. introducing            tions possible role undiscovered metabolic novel topology-based measure, synthetic accessibility,             transport reactions. able correctly predict viability 443 598 mutant            explore popular network characteristics <br /> E. coli strains comprehensive, reliable data set (21)          graph diameter, node degree, betweenness (usage) 3477 4154 mutant yeast strains grown             predictors mutant viability. results demonstrate conditions (13,14). Synthetic accessibility, S, essentially      characteristics fail predict mutants&#226;&#8364;&#8482; viability. network diameter specifically tailored transport networks,        conclude, agreement recent similar study (28), increase S correlated inviable        node degree used predict viability metabolic <br /> phenotype. significant increase S mutation suggests           knockout strains. <br /> increased metabolic costs, leading reduction growth            lack predictive utility node degree graph <br /> rate death. apparent success synthetic accessibility        diameter metabolic networks easy understand. attributed contribution network                 concepts widely applied information exchange <br /> topology information used                networks, Internet social networks, predictions.                                                    pair nodes potentially interact. contrary,    Synthetic accessibility rapidly computed given        metabolic network transport network products network, adjustable parameters,  contrast            synthesized set initial substrates. Performance FBA, MOMA, EMA, does require knowledge                 network determined ability synthesize pro- <br /> stoichiometry maximal uptake rates metabolic               ducts,  paths inputs final products transport reactions. E. coli insertional data set,         central importance, contrast diameter, pair accuracy synthetic accessibility approach comparable        nodes considered. chemical reactions require FBA MOMA. performance synthetic                substrate yield product, linear path used accessibility compared FBA EMA smaller E.            information networks needs replaced tree coli data set worse, smaller data set data   required substrates. Considering aspects naturally leads mutants affecting central metabolism            concept synthetic accessibility study metabolic biased, large data set insertional mutants       similar transport networks, e.g., signaling networks, fairly unbiased representative. Synthetic accessibility           webs reactions, input chemical performs comparably FBA yeast data sets.               physical stimulus output group chemical  <br />    Unlike FBA, synthetic accessibility does assume           sponses stimulus. Synthetic accessibility defined way <br /> optimality regard biomass production. model           generalization graph diameter directed, branching <br /> assumes long rerouted fluxes efficient native      chemical reactions input&#226;&#8364;&#8220;output transport network. <br /> ones, predicting mutants longer fluxes (larger synthetic            summary, topology metabolic <br /> accessibility) inviable. assumption fails         network central determining viability mutant <br /> certain cases, similar success rates FBA ap-           strains, success widely used flux balance techniques <br /> proach suggest assumption holds true vast ma-           predicting viability primarily attributed jority mutant strains. conclude, agreement recent       topology. addition stoichiometric parameters <br /> studies (24,27), rerouting does contribute signifi-          does significantly improve accuracy predictions, <br /> cantly robustness knockout mutants.                             used FBA predict fluxes  <br />    Similar accuracy achieved techniques based flux              dividual reactions. introduce concept synthetic <br /> balance synthetic accessibility points network topol-      accessibility, allows fast, accurate, easily inter- <br /> ogy primary determinant viability predictions          pretable analysis metabolic networks. results suggest <br /> FBA MOMA. results suggest network               rerouting metabolic fluxes plays minimal role topology sufficient predict strain viability use   providing viability mutant strains. Importantly, results stoichiometric coefficients flux balances does im-         strongly support central role network topology prove prediction accuracy, detailed prediction fluxes     determining phenotypes biological systems. individual reactions FBA/MOMA does require knowl- <br /> edge stoichiometric coefficients maximal uptake rates. <br />                                                                       SUPPLEMENTARY MATERIAL <br />    Importantly, flux balance synthetic accessibility <br /> fail predict viability significant number mutants.         online supplement article visiting <br /> Analysis incorrect predictions E. coli  Supplemen-          BJ Online http://www.biophysj.org. <br />  <br /> Biophysical Journal 91(6) 2304&#226;&#8364;&#8220;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                    2311 <br />  thank V. Spirin help processing raw data comments        13. Giaever, G.,  M. Chu, L. Ni, C. Connelly, L. Riles, S. Veronneau, manuscript, M. Kardar suggestions testing robustness,           S. Dow,  Lucau-Danila, K. Anderson, B. Andre,  P. Arkin, V. Berube M. Slutsky comments manuscript.                   Astromoff, et al. 2002. Functional profiling Saccharomyces <br /> thank  Trusina K. Sneppen useful discussions.                               cerevisiae genome. Nature. 418:387&#226;&#8364;&#8220;391. <br />                                                                                   14. Steinmetz, L. M., C. Scharfe,  M. Deutschbauer, D. Mokranjac, W. recipient Howard Hughes Medical Institute Predoctoral <br />                                                                                        S. Herman, T. Jones,  M. Chu, G. Giaever, H. Prokisch, <br /> Fellowship. L M. Alfred P. Sloan Research Fellow.                             P. J. Oefner, R. W. Davis. 2002. Systematic screen human <br />                                                                                       disease genes yeast. Nat. Genet. 31:400&#226;&#8364;&#8220;404. <br />                                                                                   15. Albert, R., H. Jeong,  L. Barabasi. 2000. Error attack <br />                                                                                       tolerance complex networks. Nature. 406:378&#226;&#8364;&#8220;382. <br /> REFERENCES                                                                        16. Jeong, H., S. P. Mason,  L. Barabasi,  N. Oltvai. 2001. <br />                                                                                       Lethality centrality protein networks. Nature. 411:41&#226;&#8364;&#8220;42. <br />  1. Varma, , B. O. Palsson. 1994. Metabolic flux balancing&#226;&#8364;&#8221;basic <br />     concepts, scientific practical use. Biotechnology. 12:994&#226;&#8364;&#8220;998.            17. Myatt, G. J. 1994. aided estimation synthetic accessibility. <br />                                                                                       PhD Thesis. University Leeds, Leeds, UK. <br />  2. Edwards, J. S., B. O. Palsson. 1999. Systems properties     Haemophilus influenzae Rd metabolic genotype. J. Biol. Chem.                  18. Handorf, T., O. Ebenhoh, R. Heinrich. 2005. Expanding metabolic <br />     274:17410&#226;&#8364;&#8220;17416.                                                                  networks: scopes compounds, robustness, evolution. J. Mol. <br />                                                                                       Evol. 61:498&#226;&#8364;&#8220;512. <br />  3. Edwards, J. S., B. O. Palsson. 2000. Metabolic flux balance <br />     analysis silico analysis Escherichia coli K-12 gene             19. Neidhardt, F. C., H. E. Umbarger. 1996. Chemical composition     deletions. BMC Bioinformatics. 1:1.                                               Escherichia coli. Escherichia coli Salmonella: Cellular                                                                                       Molecular Biology. F. C. Neidhardt R. Curtis, editors. ASM Press, <br />  4. Edwards, J. S., B. O. Palsson. 2000. Escherichia coli MG1655              Washington, DC. 13&#226;&#8364;&#8220;16. <br />     silico metabolic genotype: <span id='am-17' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, characteristics, capabil- <br />     ities. Proc. Natl. Acad. Sci. USA. 97:5528&#226;&#8364;&#8220;5533.                              20. Romero, P. R., P. Karp. 2001. Nutrient-related analysis                                                                                       pathway/genome databases. Pac. Symp. Biocomput. 471&#226;&#8364;&#8220;482. <br />  5. Forster, J.,  Famili, P. Fu, B. O. Palsson, J. Nielsen. 2003. <br />     Genome-scale reconstruction Saccharomyces cerevisiae meta-             21. Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz, <br />     bolic network. Genome Res. 13:244&#226;&#8364;&#8220;253.                                            M. D. Daugherty,  L. Somera, N. C. Kyrpides,  Anderson, M. S. <br />                                                                                       Gelfand,  Bhattacharya, V. Kapatral, et al. 2003. Experimental deter- <br />  6. Reed, J. L., T. D. Vo, C. H. Schilling, B. O. Palsson. 2003.               mination level analysis essential genes Escherichia <br />     expanded genome-scale model Escherichia coli K-12 (iJR904 GSM/ <br />                                                                                       coli MG1655. J. Bacteriol. 185:5673&#226;&#8364;&#8220;5684. <br />     GPR). Genome Biol. 4:R54. <br />                                                                                   22. Newman, M. E. 2001. Scientific collaboration networks.  Network <br />  7. Burgard,  P., C. D. Maranas. 2001. Probing performance <br />                                                                                       construction fundamental results. Phys. Rev. E. 64:016131. <br />     limits Escherichia coli metabolic network subject gene addi- <br />     tions deletions. Biotechnol. Bioeng. 74:364&#226;&#8364;&#8220;375.                           23. Newman, M. E. 2001. Scientific collaboration networks. II. Shortest <br />                                                                                       paths, weighted networks, centrality. Phys. Rev. E. 64:016132. <br />  8. Duarte, N. C., M. J. Herrgard, B. O. Palsson. 2004. Reconstruction <br />     validation Saccharomyces cerevisiae iND750, fully compart-           24. Papp, B., C. Pal, L. D. Hurst. 2004. Metabolic network analysis     mentalized genome-scale metabolic model. Genome Res. 14:1298&#226;&#8364;&#8220;1309.                causes evolution enzyme dispensability yeast. Nature. <br />                                                                                       429:661&#226;&#8364;&#8220;664. <br />  9. Badarinarayana, V., P. W. Estep 3rd, J. Shendure, J. Edwards, S. Tavazoie, <br />     F. Lam, G. M. Church. 2001. Selection analyses insertional             25. Jeong, H., B. Tombor, R. Albert,  N. Oltvai,  L. Barabasi. <br />     mutants using subgenic-resolution arrays. Nat. Biotechnol. 19:1060&#226;&#8364;&#8220;               2000. large-scale organization metabolic networks. Nature. <br />     1065.                                                                             407:651&#226;&#8364;&#8220;654. <br /> 10. Segre, D., D. Vitkup, G. M. Church. 2002. Analysis optimality          26. Ma, H.,  P <br /> </body></html>