<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1989 Genetics Societyof America <br />  <br />  <br />  <br />   Identification Characterizationof New Geneof Escherichia coli K-12 <br />                   Involved Outer Membrane Permeability <br />  <br />                              Barbara  Sampson,&apos; Rajeev Misra Spencer  Benson&apos; <br />                               Department Biology, Princeton University, Princeton, New Jersey 08544-101 4 <br />                                                Manuscript received November 10, 1988 <br />                                                Accepted publication March 17, 1989 <br />  <br />                                                            ABSTRACT <br />                  Using geneticselection mutations allow large maltodextrins cross outer membrane <br />               Escherichia coli absence LamB maltoporin, obtained characterized               mutations definea new locus E. coli. designated locus <span id='am-48' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="IMP" datatype="xsd:string"></span>imp</span> increased membrane <br />               permeability. Mapping studies imp gene resides approximately 1.2 min E. coli <br />               chromosome. T h e mutations alter permeability outer membrane resulting increased <br />               sensitivity detergents, antibiotics dyes. T h e mutations nonreverting codominant. <br />               Genetic analysis mutationssuggest imp gene essential gene. general <br />               cloning strategy used clone dominant recessive alleles. Using technique, <br />               cloned wild-type mutant imp alleles low copy number plasmid. <br />  <br />  <br />  <br /> T     outer membrane Escherichia coli forms        permeability barrier cell. sub- <br /> stances enter cell cross barrier.                                                           largerthanmaltotriose                                                           porin required (WANDERSMAN, <br />                                                                                             functional LamB malto- <br />                                                                                                  SCHWARTZ <br />                                                           ENCI 1979). starting LamB- strainand <br />                                                                                                             FER- <br />   provides protection variety agents  selecting growth mixture maltodextrins, <br /> including degradative enzymes, detergents, <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span>       identified different classes mutants <br /> agents antibiotics. T h e structure mem-      (MISRA et al. 1987). report ge- <br /> brane consists <span id='am-26' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-27' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipopolysaccharide</span> (<span id='am-16' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-17' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span>LPS</span>) (outer leaf-   netic biochemical characterization let), phospholipids (predominantly inner leaf-     classes mutants  addition conferring let), integral peripheral proteins     Dex+ phenotype, confers dramatic increase sen- <br /> NIKAIDO    VAARA (1985, 1987) afor     review  sitivity variety antibiotics, detergents anddyes. <br /> structure]. permeability outer membrane        mutations define new locus E. coli (imp) altered mutations treatment    appears essential gene. <br /> cellswith various chemical agents EDTA <br /> (LEIVE 1965), local anesthetics (LABEDAN 1988                       MATERIALSANDMETHODS <br /> inhibitors LPS biosynthesis (GOLDMAN     et al. 1987). <br /> Mutations number genetic loci alter outer        Bacterial strains: Bacterial strains plasmids used                                                           study listed Table 1. <br /> membrane&apos;s permeability. includemutations              Media chemicals: Minimal (M63) Luria(L) agar <br /> affecting LPS structure (COLEMAN       DESPHANDE broth media asdescribed SILHAVY, BERMAN            1985), <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span> biosynthesis (RAETZ andFOULDS1977),         ENQUIST(1 984). Maltodextrins purchased Pfan- structure porins (BENSON,   OCCIand SAMP-      stiehl Laboratories,Waukegan, Illinois, andpurified SON 1988; MISRA BENSON1988a,b SAMSON              dialysis described previously (BENSON, OCCIand SAMPSON <br />                                                           1988). Antibiotic sensitivity disks purchasedfrom <br /> BENSON1985), mutations acrA (HENSON           Difco Laboratories, Detroit,Michigan, BBL Microbiolog- WALKER 1982),    abs (CLARK 1984)   tolC (WHIT-   ical Systems, Cockeysville, Maryland. [&apos;4C]Maltose pur- <br /> NEY 1971), colicin tolerant mutations    chased Amersham Corp., Arlington Heights, Illinois. <br /> (NAGELDE ZWAIC andLURIA1967;                DAVIESand     Novobiocin (Nov), erythromycin (Em), kasugamycin <br /> REEVES1975) result increased permeability        (Ksg) purchased fromSigma Chemical Corp., St. Louis, <br />                                                           Missouri, used 50 pg/ml (Nov) 25 pg/ml (Em unknown mechanisms.                                    Ksg). Arginine <span id='am-43' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> added minimal media <br />    described genetic selection mutants      concentration 33 pg/ml. increased outer membranepermeability (BENSON            Geneticmanipulations: Procedures involving F&apos; mat- DECLOUX     1985). selection relies fact  ings, Hfr mapping, P1 transductions TnlO pools order E. coli grow maltodextrins        described SILHAVY,BERMANand ENQUIST(1984)                                                                             MILLER(1972). T h e ksgA560::TnlO insertion isolated <br />                                                                             TnlO pool prepared MC4100. capitalized <br />    &apos; Present address: Rockefeller University, New York, New York 10021-     fact insertions ksgA gene confer resist- <br /> 6329. <br />      Present address: Department Microbiology, University Maryland,   ance kasugamycin (Ksg&apos;) (ANDRESSON     DAVIES1980). <br /> College Park, Maryland 20742-445 1.                                         P1 lysate prepared MC4100 TnlO pool used Genetics 122: 491-501 (July, 1989) <br />      492                                          B.  Sampson, R. Misra S.  Benson <br />                                                                     TABLE 1 <br />                                                           Bacterial strains plasmids <br />  <br />                      Strain <br />                    plasmid                                  Genotype                                                                   Source <br />  <br />                 Strains <br />                   Jef8                   HFR thrA carB8 relA1 metBl spoTl                          GICOTet al. (1984) <br />                   SE14                   F&#226;&#8364;&#8482; 104/ thr-1 leuB6 gpt-proA62) recAargE3 <br />                                                                                  13                M. SCHMIDT <br />                                             thi-1 ara-14 lacy1 galK2 xyl-7 mtl-1 tsx-33 <br />                                             supE44 hisG4 rpsL3 1 <br />                   MC4 100                F- araD 139 argF-1ac)U1698 rpsL 150 relA                CASADABAN (1 976) <br />                                            JbB5301 ptsF25 deoCl <br />                   MC 1000                F- araDl 39 araABC-leu)7679galU galK                    M. CASADABAN <br />                                             lac)X74 rpsL thi <br />                     MCR106 <br />                      100 <br />                       MC4                           AlamB 106                                      EMRand SILHAVY <br />                                                                                                                 (1 980) <br /> 6   MCR             DME553                           AompF80                                       study <br />                     ECB4623              araDl 39 galK lac)X74 rpsL thi imp-42                                                                                     13                  study <br />           ThisF&#226;&#8364;&#8482;104/MCR106 <br />                imp-4213 <br />                 thr-43::Tn/U <br />                     BAS803                                                                              study <br />                                             sr15 17::Tn5 recA56 <br />                   BAS849                  MCR <br />                                            This106 imp-42 13                                            study <br />                   BAS  1329 <br />                      MCRlO6                                           araABC-leu)7679 <br />                         zab::TnlU       imp-4213                                                        study <br />                                             sr15 17::Tn5 recA56 <br />                   BAS1335 <br />                      MCRlO6 <br />                        1eu::TnlO                              imp-208 sr1517::Tn5 recA56           study <br />                   BAS                        1340         1329 F&#226;&#8364;&#8482; 104/BAS                                                     study <br />                   BAS                        1347         1335 F&#226;&#8364;&#732; 104/BAS                                                     study <br />                   BAS1 <br />                     F&#226;&#8364;&#8482;104-1/MCR106 <br />                         365 zab::Tn/O <br />                                araABC-leu)7679 <br />                                                                                                    study <br />                                             imp-421 3 recA::Km <br />                   BAS 1362 <br />                        MCRlO6 <br />                             imp-4213 <br />                                 srl300::Tn/O                               recA56                  study <br />                   BASMCRlO6 <br />                        1363 <br />                          sr1300::TnIO                            recA56                            study <br />                   RAMMCRlO6 <br />                         105                         AompF80 zei06::TnlO                            MISRAand BENSON(1988b) <br />                   RAMMCRlO6 <br />                         127                         AopmF80 imp-208                                MISRAand BENSON( 1 988a) <br />                 Plasmids <br />                   PRAM 1005               Km&#226;&#8364;&#8482;, pSC 10 1 replicon                                   study <br />                   pSAB5402 <br />                         KmR,                    vector pRAM1005 based, <br />                                                                     cloned      imp gene           study <br />                   pSAB5420                Km&#226;&#8364;&#8482;, vector PRAM 1005 basedThisimp-42 <br />                                                                             3 1                         study <br />  <br />  <br />  <br />     transduce MCRIO6 tetracycline resistance (Tc&#226;&#8364;&#732;). ap-                electrophoresis described (MISRA BENSON <br />     proximately 2000 transductants, Ksg&#226;&#8364;&#732;. deter-                1988a). <br />     ksgA::TnlO insertions, checked link-                  DNA methods: DNA methods described MISRA <br />     age thecarB (1 &#226;&#8364;&#8482;) leu (2&#226;&#8364;&#8482;) genetic markers. case           BENSON(1 988b). <br />     (ksgA560::TnlO) demonstrate TnlO                       Construction pRAM1005 chromosomal clone <br />     terminant linked c d leu genes showed               bank: partial restriction map pRAM1005 shown     &gt;90% linkage imp gene. T h e presence carB               Figure 1. PRAM 1005 unique BamHI EcoRI cloning <br />     mutation assayed plating cells glucose minimal                 sites pSClOl replicon  constructed     plates plus minus supplementation arginine                  steps pRAM1001 (MISRAand BENSON1988b) <br />     uracil (GIGOTet al. 1980).                                               basically inserting HindIII-BamHI fragment carrying <br />        F&#226;&#8364;&#8482; 104-1 constructed recombining thr                   kanamycin resistance determinantfrompNEO          (P-L <br />     carB8 mutations Jef8 ontoF&#226;&#8364;&#8482; 104 series genetic              Biochemicals, Milwaukee, Wisconsin) HinDIII <br />     crosses (datanot      shown). F&#226;&#8364;&#8482;104-1 complement                  EcoRI sites PRAM 1001. Chromosomal DNA     1eu::TnlO insertion does complement carB8                    prepared RAM105, digested BamHI     mutation. F&#226;&#8364;&#8482;104-1 crossed BAS1314 yield                 ligated intopRAM1005 hadbeendigested                strain BAS1 361. introduced recA11l::Km                   BamHI treated alkaline phosphatase. T h e ligated <br />     sertion strain generate strain BAS1365. T h e               mixture transformedinto        MC1000. Transformants <br />     recA 1 11 ::Km insertion kindly provided GREGPHIL-                selected L plates containing kanamycin. Approxi- <br />     LIPS insertion 1.4-kb fragment carrying                                                                              mately 10,000 transformants pooled plasmid DNA <br />     kanamycin determinantfrom pUC-4-K (PharmaciaInc.,                        extracted pool. <br />     Piscataway, New Jersey) PstI site recA gene <br />     (G. PHILLIPS, personal communication).                                                          RESULTS <br />        Phenotypic tests:Detergent andantibiotic sensitivity tests <br />     described previously (BENSONand DECLOUX                        Isolation imp-4213 imp-208 mutants: <br />     1985). Growth tests described (MISRAand                     previously described  selection obtaining <br />     BENSON1988a) dialyzed maltodextrin used <br />     instead defined size maltodextrins.                                   mutations affect outer membrane permeability. <br />        Biochemical tests: [&#226;&#8364;&#732;4C]Maltose uptake assays, prepara-               T h e selection growth maltodextrins     tions  cell envelopes SDS-polyacrylamide gel                absence LamB maltoporin, yields      pre- <br />                                                     Permeability Mutants E. coli                                                    493 <br />                                                                                                   TABLE 2 <br />                                                                                       Sensitivity various compounds <br />  <br />                                                                              zone <br />                                                                                 lnhibition <br />                                                                                    Compound                                    (mm) <br />                                                                                  (concentration <br />                                                                                                 MCRlOGimp+ BAS849(imp-4213) <br />  <br />                                                                       Chemicals, Dyes Detergents <br />                                                                                          14 <br />                                                                         C u S 0 4 (1.19 mg)                    11 <br />                                                                         Deoxycholate ( 1 500 pg)                0               13 <br />                        Digemt    BomHI                                                                     0                8 <br />                                                                         Triton X-100 (750 pg) <br />                        T r o t w t h phonphat m 0                 SDS (750 pg)                            0               12 <br />                        Ligatowith  B o m H digamtad                   Eosin Y (5%)                            0                7 <br />                        olwomomomal D N ( &lt; - + ) <br />  <br />  <br />                                                                                          Antibiotics <br />                        l s c t ~ m rt ranmi ormant 6               Tetracyclines <br />                        Saroon f o r Nov <br />                                              r                                            20 (30 pg)           14 <br />                                                                         Chlorotetracycline <br />                                                                         Oxytetracycline16  (30 pg)             15 <br />                                                                         Tetracycline (5 pg)                     9               13 <br />                                                                       Betalactams <br />                                                                         Ampicillin (50 pg)                     14              27 <br />                                                                                           29 pg) <br />                                                                         Carbenicillin ( 1 00                   22 <br />                                                                         Cephalothin (3015  pg)                 13 <br />                                                                         Penicillin G (10 units)                 0                0 <br />                                                                                                                                               Bacitracin ( 1 0 units)                 0              22 <br />                                                                         Chloramphenicol (5 fig)                 7              13 <br />                                                                         Erythromycin (15 pg)                    0              19 <br />                                                                         Naladixic Acid (5 pg)                  11              17 <br />                                                                         Novobiocin (30 pg)                      0              22 <br />                                                                         Rifampin (1 5 pg)                       8              26 <br />   FIGURE 1.-Partial    restriction <br />                                  maps pRAM1005 <br />                                                                  Trimethdprim (5   26 pg)               17 <br /> pSAB5402. T h e details isolationof pSAB5402 given text. plasmids drawn scale. T h e multicloning site      Sensitivity expressed size (mm) zone growth <br />                                                                      inhibition center disc.T h e size disc 6 mm. <br /> (MCS) present pRAM1005 following restriction sites; <br /> EcoRI-SalI-KpnI-SamI-B~mHI.    T h e chromosomal insert containing imp gene cloned BamHI site. T h e sizes      1.2&#226;&#8364;&#8482; region E . coli chromosome linked <br /> plasmids approximate.                                            known TnlO markers region. 50 <br />                                                                      independent transductions involving mutant imp <br /> dominantly mutants alterations major po-                 alleles pleiotropic effects conferred altera- <br /> rins, OmpF OmpC(MISRAet d . 1987). addition                    tions, e., antibiotic sensitivity, detergent sensitivity types mutations, obtained isolates,               Dex+ phenotype moved single <br /> 421 3 208, Dex+ mutation did                    unit. suggests various phenotypes map porin gene conferred sensitivity                imp-42 13 andimp-208 mutations caused single <br /> MacConkey agar (Mac&#226;&#8364;&#8482;)), Nov (Nov&#226;&#8364;&#8482;), Em (Em&#226;&#8364;&#8482;).                    genetic alterations. <br /> Isolate 421 3 200 spontaneous                        sensitivity profile BAS849 (imp-4213) dependent Dex+ mutantsobtainedfromMCRlO6,                            identical tothat original isolate 421 3. mutant did contain                     increased permeability extendstoa          diverse set ompF(Dex) mutation. Isolate 208                      compoundsincludingdetergents,          metals, dyes, 70 spontaneous independent Dex+ mutants obtained                     numerous antibiotics (Table 2).  did RAM105. exact frequency                      detect increase sensitivity aminoglyco- <br /> isolates occurred difficult obtain ap-              sides; amikacin, gentamycin, kanamycin neomy- <br /> peared colonies 4 6 days plating.                    cin polymyxin, clindamycin, EDTA. estimated frequencyof ompF(Dex) ompC(Dex)                    detergents; deoxycholate SDS, large antibi- <br /> mutations 5 X lo-&#226;&#8364;&#8482; (BENSON,     OCCIand SAMPSON                  otics erythromycin, novobiocin, bacitracin, <br /> 1988     1 X         (MISRA BENSON 1988b),                   rifampin increase sensitivity dramatic. <br /> respectively. basis, frequency 4213               T o quantify degree change determined 208 isolates 1 X 10&#226;&#8364;?&#226;&#8364;&#8482;.                        twofold dilution method minimum inhibitory <br />   Phenotypic characterization imp mutants:                    concentrations (MICs) rifampin (Rif), novobiocin, ensure phenotypes represented     effects            erythromycin, ampicillin (<span id='am-36' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="AMP" datatype="xsd:string"></span>Amp</span>) chloroampheni- single genetic alteration, transferred imp               col (Cm). MIC values BAS849  Rif, &lt;0.5 <br /> mutations P1 transductionintostrainMCRlO6                         pg/ml; Nov, 2 pg/ml; Em, &lt;0.5 pg/ml; Amp, &lt;0.5 <br /> using linked TnlO marker.Genetic          studies              pg/ml; andCm,2pg/ml.           MICvalues  showed imp mutations mapped                   parentstrain     MCRlO6  Rif, &gt;64pglrnl;        Nov, <br />  494                                  B.  Sampson, R. Misra S.  Benson <br />  <br /> &gt;128 pg/ml; Em, &gt;128 pg/ml; Amp, 2 pg/ml;                                           TABLE 3 <br /> Cm, 6 pg/ml.  relative change sensitivity                      factor mappingdata large hydrophobic antibiotics (Nov, Em, Rif) large orders magnitude.                                    Genetic Markers <br />                                                                                                                       trast, theincrease sensitivityto Amp Cm,             Cross      cad     krgA::TnlO    imp     leu      lsolates <br /> predominately cross outer membrane po- <br />                                                                Donor         -        Tc&apos;        4312    + <br /> rins, 4- 6-fold.                                      Recipient     +        Tc&apos;         Wt     - <br />    T o determine general structure outer                        -        Tc&apos;         wt      -                   23 <br /> membrane disrupted, assayed sensitivityto                          +        Tc&apos;         wt      -                   48 <br /> following bacteriophages: X, K20,Hy2, SS4, @80,                               +        Tc&apos;         wt      +                    4 <br /> Tu2, Tu4, T6, P1,       U3. case did detect                      -        Tc&apos;         wt      +                    6 difference sensitivity compared control                        +        Tc&apos;        4213     +                   35 <br /> strain MCRlO6. imp-208 allele results nearly <br />                                                                               +        Tc&apos;        4213     -                   20 <br />                                                                               -        Tc&apos;        4213     +                    8 <br /> identical phenotypes introduced MCRlO6.                             -        Tc&apos;        4213     -                   14 <br />    Geneticcharacterization imp-4213 muta-                                                                  Total   158 <br /> tion: initial genetic mapping indicated          B. Donor        -         Tc&apos;        4313 <br /> mutation present isolate 421 3 mapped              Recipient     +        Tcs         Wt <br />  <br /> ompF omPC. T o determine map position                              +        Tc&apos;        4213                     60 mutation conferred Dex&apos; increased                            +        Tc&apos;         Wt                      25 <br />                                                                               -        Tc&apos;        4213                     33 <br /> permeability phenotypes, did Hfr mapping using                             -        Tc&apos;         Wt                       4 <br /> Hfrdonor       strains <br />                      BW6164, BW6169,  BW6175                                  -        Tc&apos;        4213                      5 <br /> (WANNER1986)andHfrG6(HOFNUNG, HATIFIELD                                                                          Total    3 SCHWARTZ 1974).       results indicated      C. Donor                  Tc&apos;        4313    + <br /> imp mutation nearthe thr geneat 0&apos;                    Recipient              Tcs         Wt     - <br /> chromosome (data shown). T o define                                    Tc&apos;         wt      -               27 <br /> map location, did , ,andfour-factor                                       Tc&apos;        4213     -               13 <br /> crosses using generalized transducing phage P 1                                 Tc&apos;        4213     +               17 sample mapping results shown Table                                     Tc&apos;         wt      +                2 <br /> 3 cotransduction linkage imp gene                                                                                                                   Total    59 <br /> adjacent loci given Figure 2. noted          Transductions described MATERIALS METH- <br />                                                            ODS.  cases, selection Tc&apos; transductants. imp <br /> linkage imp-42 13 andwild-type alleles flank-    alleles scored sensitivity Nov Em Dex+ <br /> ing markerswas thesame. linkage imp&apos;        phenotype, wt means wild type, 4213 means imp-4312 confered <br /> allele higher    linkage imp-    sensitivities. carB leu alleles scored appropriate <br /> 42 13 allele markers, ksgA leu (Figure <br />                                                                                         + <br />                                                            auxotropy, - = auxotropic, = prototropic. <br />  <br /> 2). ksgA leu genes lie opposite sides <br />                                                            208/F&apos; 104 imp&apos;). mutant Dex+ MacS phe- imp gene, interpret mean                                                            notypes recessive NovSand Ems pheno- <br /> selective bias mutant imp alleles                                                            types dominant (Table 4). noted acquisition imp&apos; allele. <br />                                                            strain wasrecA&apos;     wild-type (imp&apos;)Nov&apos;    MacrDex- <br />    gene order, based mapping studies,                                                            colonies quickly accumulated culture. carB-ksgA-imp-ara-leu estimate map posi- <br /> tion imp approximately 1.2&apos; E. coli     colonies presumably arise recombinational loss  chromosome. position rab4292::Tn5           homogenization) mutant allele wild type. zab::TnlO shown informational purposes.            mapping studies, suggests    imp-208 allele maps position     selective bias mutant allele. <br /> imp-4213 mutation. T o determine ifwe               detergent drug sensitivities conferred recombination mutant alleles,           imp mutations provided convenient       means crossed mutation selecting       select revertants. did ways; selection adjacent KsgA560::TnlO o r leu+ marker. did         Mac&apos;, Novr Em&apos; colonies. Spontaneous Macr <br /> notdetect wild-type recombinantsamong              colonies occur frequency approximately 2 X <br /> &gt;900 transductants screened. suggests        10&quot;j. 20 spontaneousand 2-amino-purine induced mutations close          Mac&apos; revertants tested, pseudorevertants. identical alterations, overlapping deletions.           able rescue imp-4213 allele    Diploid reversion analysisof imp alleles:       thesestrains (D. WOODWORTH,          unpublisheddata). <br /> T o determine imp-42 13 andimp-208 mutations        Selection spontaneous Nov&apos; Em&apos; revertants <br /> aredominantor        recessive, constructed strains     37&quot; true revertants pseudorever- <br /> BASl 340 (imp-4213/F&apos; 104imp&apos;) BASl 347 (imp-          tants.  mutagenized cultures pos- <br />                                                             Permeability Mutantsof E. coli                                                       495 <br />  <br />  <br />  <br />  <br />                                                                                                               FIGURE2.-Linkage         map E. <br />                                                                                                            coli chromosomalregionsurround- <br />                                                                                                            ing imp gene. Linkages  <br />                                                                                                            termined twofactor PI cotrans- <br />                                                                                                            ductions.Numbers        brackets    ) <br />                                                                                                            represent frequency thewild- <br />                                                                                                            type imp allele. numbers <br />                                                                                                            determined using theimp-42 1 3 <br />                                                                                                            allele. cases, numbers rep- <br />                                                                                                            resenttheaverageofseveralinde- <br />                                                                                                            pendent transduction experiments. <br />  <br />  <br />  <br />  <br />                                                                                                                                          <br />  <br />  <br />                                  TABLE 4                                                                 TABLE 5 <br />                      Diploid analysis imp alleles                                    Reversion analysis imp alleles <br />  <br />                                                      Phenotypes                                                                      Reversion <br />                                                                                                                      Reversion      L-Tc-Nov <br />                       Strain                   Dex   Mac    Nov      Em                      Strain               L-Nov plates      plates <br />  <br />       MCRlO6 (wild type)                       -     R     R         R         BASl 329 <br />                                                                                (imp-4213 <br />                                                                                  zab::TnlO) <br />                                                                                    &gt;4                                    X   10-9         ND <br />       BAS1329 <br />                                                                                BASl 340 <br />      zab::TnlO) <br />       (imp-42 13                               +      s      s       s         (imp-4213 <br />                                                                                        zab::TnlO/F&apos;104imp+)5.4           X   lo-&apos;   4.6   X <br />       BASl 340                                                                 BAS1335 <br />           zab::TnIO/F&apos;104imp+) <br />       (imp-4213                                -     R     S         S         (imp-208 1eu::TnlO) <br />                                                                                           &gt;6                             X   lo-&apos;         ND <br />  <br />        BAS1335                                                                 BAS 1347 <br />        (imp-208 <br /> 1eu::TnlO)                                     +      ss       s                                         4.6 <br />                                                                                (imp-208 leu::TnlO/F&apos;104imp+)             X          5.4   X   lo-* <br />       BAS1347                                                                    strains recA56. L-Nov = Luria medium 50 m/ml <br />       (imp-208 leu::TnlO/F&apos;104imp+)            -     R     S         S        novobiocin, L-Tc-Nov = Luria medium 25 p/ml tetracycline <br />                                                                               plus 50 p/ml novobiocin. ND =  <br />          strains MCRlO6 recA56. Phenotypes tested <br />       described MATERIALS METHODS. S = sensitive, R = resist-       maintained selection Tc&apos; frequency <br />                            + <br />       ant, - = growth, = good growth 36 hr. <br />                                                                               Nov&apos; colonies reduced orders mag- <br />                                                                               nitude (Table 5 ) . explanation findings <br />       sible obtain rare Nov&apos; Em&apos; pseudorevertants <br />                                                                               Nov&apos; colonies arise deletion events       (R. FORTIER, personal communication). <br />                                                                               eventsare stimulated adjacent TnlO. <br />          imp gene appears essential: noted <br />                                                                               T o test  constructed strain BASl 365,  <br />       diploid strains BAS1340 BAS1347,  <br />                                                                               scribed MATERIALS METHODS. strain car- <br />       like haploid strain derived, <br />                                                                               ries F&apos;104-1 carB8 allele allows <br />       segregated Novr colonies highfrequency.                         detect loss chromosomal curB+ allele       strains yield Nov&apos; colonies frequencies high                   shown Figure 3. Strain BAS1365 yields Novr Tcs <br />       lo-&apos; (Table 5) cases Ems phenotype                    CarB- colonies atfrequency <br />                                                                                                                                 simultaneously lost. explanation unusual                   showing Nov&apos; colonies arise deletions <br />       finding colonies arise inactivation                 remove TnlO thecarB gene. <br />       mutant chromosomalimp allele. Nov&apos;                         observations suggest imp mutation, <br />       colonies Tc&apos;. fact, able detect                 unusual property TnlO,       Tcs colonies imposing Nov&apos; selection                     results apparent deletion events. Strains       simple replica plate techniques(datanot        shown).                  diploid wild type imp allele yield <br />  496                                      B.  Sampson, R. Misra S.  Benson <br />  <br />  <br />  <br />  <br />                                                                   - <br />   t hr <br />           +                  carfl   -                 mx) <br />                                                               +                  IOU <br />                                                                                            + <br />  <br />   t hr <br />           +                  carB <br />                                      t <br />                                          306:   rnlO <br />                                                                   p   ara-l ou <br />                                      1    1 <br />                                                                                         0&#226;&#8364;&#8482;                                     &#226;&#8364;? mp-42 13                                         2&#226;&#8364;&#8482; <br />  <br />  <br />                                                                                                        FIGURE3.&#226;&#8364;?Spontaneous          inacti- <br />                                                                           rrcA:: Ktn                vation mutant imp allele                                                                                                                                                                                      deletion event. T h e rationale                                                                                                     results thedepictedexperiment <br />                                                                                                     described    text.                                                                                                          straight lines relevant genes <br />                                                                                                     present F&#226;&#8364;&#8482; 104-1,                                                                                                      oval depicts relevant genes pre- <br />                                                                                                     senton chromosome. T h e phe- <br />                                                                                                     notype starting diploid strain <br />                                                                                                     represented upper                                                                                                     figure Km&#226;&#8364;&#8482; Novs Erns Tc&#226;&#8364;&#8482;                                                                                                     CarB+. T h e lower figure <br />                                                                                                     shows thepredictedstructureafter <br />                                                                                                     deletion event. T h e phenotype                                                                                                     resulting diploid deletion strains <br />  <br />   t hr+                      carfl   -                 mp <br />                                                               +              1       B U <br />                                                                                            +        shown lower dia- <br />                                                                                                     gram Km&#226;&#8364;&#8482; <br />                                                                                                                   Nov&#226;&#8364;&#8482; Em&#226;&#8364;&#732; Tc&#226;&#8364;&#8482;                                                                                                     CarB-. <br />  <br />   thrt <br />  <br />  <br />  <br />  <br />                                                                           recA:: B r <br />                                                                                   <br />  <br />  <br />  <br /> Tc&#226;&#8364;?colonies, different TnZOs (zab::TnlO                   minimal media (maltodextrins glucose) porin <br /> leu::TnlO) stimulate frequency Nov&#226;&#8364;&#8482; col-                  minus strains regardless imp allele carried <br /> onies (Table 5).  close byTnZOs                    reduced growth rate. slower growth phe-  Strain BAS803 (thr::TnlO imp-4213/F&#226;&#8364;&#8482;104 imp&#226;&#8364;&#8482;)                     notype indicative porin-deficient phenotype. <br /> does yield Tcs colonies, frequency                    Biochemicalanalysis imp mutants: Novr &#226;&#8364;&#339;revertants&#226;&#8364;? orders magnitude                 crude assay outer membrane structure analyzed <br /> thanthat     seen similar diploids carrying                  protein profile cell envelopes soluble <br /> zab::TnlO 1eu::TnlO.                                           fractions prepared fromstrains carrying theimp-42 13, <br />    observations, coupled fact hap-                imp-208, imp+ alleles. detect differ- <br /> loid strains revert yield Tcs colonies                  ence protein profiles SDS-PAGE gels gardless imp TnlOalleles carry, suggest                fractions (data notshown). imp gene essential explain                     T o quantitatethe    changes outermembrane inactivate mutant imp allele                permeability imp mutations, did growth <br /> diploids.                                                             tests [&#226;&#8364;&#732;4C]maltose uptake assays. complex  <br />    Porin independence imp phenotypes:                       dia strain BAS849(imp-4213) andthe imp&#226;&#8364;&#8482; parent <br /> determine phenotypesconferred imp                       MCR 106 nearly identical growth rates (Fig. 4A). <br /> mutations operated altering properties                  Growth tests using 10 mM maltodextrin carbon <br /> OmpF OmpCporins, constructed strains lack-                     source presence imp mutation <br /> ing bothporins. Strains lack                 increases growth rate (Figure 4B). higher  <br /> porins retain imp associated phenotypes e.,              centrations maltodextrins (16 mM) strains <br /> Nov&#226;&#8364;&#8482;, Em&#226;&#8364;&#8482;, Macs Dex+ (datanot shown).                        identical growthrates(datanot        shown). sensitivities conferred imp mutation                    imp mutations increase therateat        labelled <br /> require presence porins. noted                 maltose taken cell (Figure 5). best <br />                                                     Permeability Mutants E. coli                                                 497 <br />  <br />  <br />  <br />                                                                             16 <br />  <br /> 0 <br /> 0                                                                           12 <br /> (0 <br />  <br />  <br /> Q0                                                                           8 <br />  <br />  <br />                                                                               4 <br />  <br />  <br />      0.104                                                        1           0- <br />          0      30     60      90     120    150     180    210                   0       20          40          60                                                                                7 <br />                                                                                                 Time (sec) <br />      1 .oo <br />                                                                         FIGURE5.-Rate       [&apos;4C]maltoseuptake imp mutants. <br />                                                                       Uptake experiments described       MATERIALS                                                                       METHODS. RAM105      (OmpF-        OmPC&apos; imp&apos;), open triangles; <br /> 0                                                                     MCRlO6(OmpF&apos;OmpC&apos;            imp&apos;), opencircles;BAS849(OmpF&apos; <br /> 0                                                                     OmpC&apos;imp-4213), filled circles; RAM127 (OmpF- O m p C <br /> (0 <br />                                                                       imp-208), filled triangles. <br /> Q0                                                                    frequency. plasmids colonies iden- <br />      0.1 0 <br />                                                                       tical BamHIrestriction patterns (data notshown).                                                                       plasmid, pSAB5402, chosen analysis. <br />                                                                       Apartialrestrictionmap        ofplasmid pSAB5402      0.05                                                             given Figure 1. plasmid carries insert                                                                       approximately16       kb. provethat      plasmid <br />                              Time (min)                               carried imp gene, recombined imp-4213 <br />    FIGURE4.--Growth test imp&apos; imp-4312 strains.   Resultsof      mutation plasmid follows. introduced <br /> growth tests LB. (B) Results growth test 10 mM maltodex-     pSAB5402 BAS849 (imp-4213), extracted plas- <br /> trin. MCRlO6 (imp&apos;) = open circles BASS49 (imp-4213) = filled         midDNA       andtransformed       DNA circles. <br />                                                                       BAS849 selecting plasmid encoded Km&apos; determi- <br />                                                                       nant screened transformantswhich seen strain DME553 (LamB- OmpF- OmpC+ <br />                                                                       Dex+ Mac&apos; (diploid theimp-42 13 allele). Recall <br /> imp&apos;) compared strain RAM 127 (LamB- OmpF- <br /> OmpC+ imp-208). StrainMCRlO6 (LamB- OmpF+                             Dex+ MacSphenotypes recessive OmpC+ imp&apos;) higher basal rate uptake                      imp-4213 allele present comparedto DME553 presence                          plasmid expected phenotypes OmpF porin, functionally larger porin                 transformants.Among350          Km&apos; transformants, OmpC (NIKAIDOand VAARA 1987; MISRA BEN-                           obtained colonies Dex+ Mac&apos;. <br /> SON 1988a). cases, rate uptake conferred                analysis plasmid (pSAB5420) fromone imp mutations 10 15% ob-                    clones showed restriction pattern <br /> served LamB+ strains (data shown).                           identical pSAB5402, introduced <br />    Cloning imp gene low copy number                     BAS1363(imp+) conferred Nov&apos; Dex+ phe- <br /> plasmid: clone bank constructed using low                   notype (Table 6). proved physically geneti- <br /> copy number vector pRAM1005 described MA-                       cally plasmid carried imp-42 13 allele. TERIALS METHODSTO        identify clone                 fact restriction map identical pSAB5402 <br /> carried plasmid encoding imp gene,                      suggests imp-4213 mutation does result use fact imp-42 13 mutation rapidly                gross rearrangement DNA. estimated lost recombinational correction  . Plas-                 detect deletionsor insertions 200 bpor  <br /> midDNA fromthe BamHI clone bank trans-                               codominance imp mutations suggest formed strain ECB4623. strain derived                    result production altered gene product, MClOOO whichhas increased transformation                         phenotypes seen F&apos; diploids reflect <br /> efficiency plasmid DNA. necessary use                  contributions mutant wild-type gene background imp mutationsreduce                       products.  reasoned gene dosage <br /> transformation efficiency 30- 50-fold (data                 alleles increased result shown). approximately 2500 Km&apos; transform-                      stronger representationof phenotype conferred <br /> ants screened, resistant (grew  Mac-                   gene product. case (Table <br /> Conkey agar segregated      Nov&apos; colonies high                 6). imp&apos; allele present insix         copies <br />  498                                              B.  Sampson, R. Misra S.  Benson <br />                             TABLE 6                                    accurately assign frequency mutations.   Effects mutant wild-type imp gene low copy           likely, mutations occurred    imposition                            plasmid                                     selection.  possible selection <br />                                                                        itselfmay influenced type mutants                  imp genotype      Disc sensitivity <br />                                                 zones <br />                                                                        obtained. Recent work CAIRNS, OVERBAUCH                                                                  Growth    StrainChromosome      Plasmid SDS D E 0 Eos Nov Em maltodextrin     MILLER(1988) suggest nonlethal selection BAS1362      imp-4213                       1410                  10 19 15        +          selection influence type mutation obtained. <br />                                                                           believe mutations simple missence <br /> BAS 1362 imp-42 13 imp&apos;            0    0    0 10 (15)     -/+ <br /> pSAB5402                                                               null mutations based rarity      nonre- <br />                                                                        verting codominate nature, suggest  <br /> BAS1362 imp-4213 imp-4213 13 <br />                           13                10 21 21       ++          sult similar identical genetic events resulting <br /> pSAB5402 <br />                                                                        gene product partial acquired func- <br /> BAS1363 imp&apos;                  0    0    0    0 (8)     - <br /> BAS1363 imp&apos;imp&apos;                   0    0    0    0 (14)    - <br />                                                                        tion(s). explanation account low <br /> pSAB5402                                                               frequency obtained mutations, BAS1363 imp&apos;            imp-4213 10     0    0 15 (18)      +          fact unable detect recombination <br /> pSAB5420                                                               mutations. <br />                                                                           imp mutations specific base pair sub- <br />    numbers indicate size (mm) killing zone. Number      stitutions, deletions insertions. possibility <br /> given ( ) indicate zone growth inhibition, killing. + = <br /> growth, - = growth, -/+ = poor growth 36hr. SDS =                unlikely unable obtain true rever- <br /> sodium dodecyl sulfate, D E 0 = deoxycholate, Eos = eosin Y .          tants eitherspontaneously following treatment                                                                        2-AP, NTG, EMS (data shown). favor (BAS1362/pSAB5402) detergent, Nov&apos;, theDex+                        possibility mutationsresult specific dele- <br /> phenotypes reduced relative haploid strain                  tion events able detect <br /> BAS1362. appears presence        imp&apos;               exact excisionof insertion reversion test. <br /> allele plasmid itself alter outer membrane                  Analysis imp-42 13 mutation pSAB5420  <br /> permeability slightly. observe weak Dex+ phe-                     dicates mutation deletion, small <br /> notype BAS1362/pSAB5402  copies imp&apos;)                      (e200 bp) reside acoding wild type gene provided single                  region fuse adjacent ones. <br /> copy F&apos; 104the diploid Dex-. Inaddition,                            Mapping mutations places imp gene BAS1362/pSAB5402 shows slight increase Ems                        approximately 1.2 min E. coli chromosome <br /> compared BAS1362 (Table 6). imp-4213                        ksgA m . gene order obtained <br /> present plasmid (BAS1363/pSAB5402)                        factor mappingand consistent  increase sensitivity SDS, Nov Dex+                 factor cotransduction frequencies (Table3, Figure 2). <br /> phenotype (Table 6). presence single imp&apos; <br />                                                                         high linkage ksgA, <br /> gene sufficient eliminate sensitivity deoxycho- <br />                                                                        zab4292::Tn5, zab::TnlO, imp gene late reduce NOV&apos; (Table 6). imp-4213 <br />                                                                        selective bias wild type imp allele, relative <br /> allele present plasmid chro- <br />                                                                        position suggested gene order viewed <br /> mosome (BAS1362/pSAB5420) phenotypes  <br />                                                                        tentative confirmed physical analysisof ferred mutation accentuated (Table 6). <br />                                                                        cloned gene. nogenes region                                                                        chromosome known affect outer mem- <br />                          DISCUSSION                                    brane functions.  themap position does    utilized genetic selection gain insights               insight nature imp gene product barrier functions outer membrane                    physiological role cell. does E. coli. selection based uptake                   mutations define new locus. <br /> large sugar, maltodextrin,  yields classes                 mapping studies, fact strains dip- <br /> mutants unequal frequencies (MISRA et al. 1987;                     loid region segregate Novr colonies high <br /> BENSON,OCCIand SAMPSON        1988). rare class                 frequency, suggest bias mutant <br /> mutations, imp, results sensitivity variety                 allele. basis selection mutant <br /> unrelated compounds including detergents, antibiot-                    imp alleles known. fact haploid <br /> ics dyes. imp mutations characterized                   strains, carry single copy mutant imp study nonreverting codominant muta-                      gene, significant growth disadvantage <br /> tions define new genetic locus appears                 complex media growth advantage essential genethat affects outermembrane                         maltodextrin clouds issue. possibility <br /> permeability.                                                          disadvantage imposed imp mutation <br />    imp mutants appeared Dex+ colonies                   exponential growth phase  <br /> days afterplating  difficult                    diates effects phase ofthe <br />                                              Permeability Mutants E. coli                                      499 <br />  <br /> growth cycle, viability stationary phase.        structure (MISRA BENSON198813; BENSON,OCCI <br />    appearance Novr colonies dip-          SAMPSON     1988). theimp mutations cause <br /> loid strains easily explained inactivation      similar alteration, impart analogous phenotype mutant imp allele. suggestion sup-        different mechanism known. porin <br /> ported fact colonies lose imp          independence imp phenotypes indicates associated phenotypes (Dex+ NovS Ems Mac&apos;).           porins required increased phenotype. contrast pseudorevertants obtained fromhaploid          imp mutants exhibit reduced growth strains whichlose imp associated             absence bothporins, suggests thatthe imp <br /> sensitivities. factors contributetothe            mutations result production new porin <br /> apparent high frequency diploid strains seg-         type protein consistent fact altera- <br /> regate Novr colonies. include strongselection         tion <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> profile envelope. addition, loss mutant allele, stimulation deletions       suggests diffusion certain substances adjacent TnlOs transposons      (KLECKNER,REI-       outer membraneis limiting growth imp <br /> CHARDT BOTSTEIN          1979) technical diffi-   porin minus mutants. <br /> culty starting population cells does           explanation observed sensitivities contain preexisting Novr colonies.         Dex+ phenotype general property suggested frequency    (1 O-*         small inno-     outer membrane altered. culums cells contain preexisting Novr cells.          mutantsretain sensitivity bacteriophages <br />     lines evidence suggest imp gene      tested significant reduction essential. imp mutations codominant         growth rate, defect appears result simple null mutations. Diploid strains inacti-        major disruption structure. unable vate themutant allele high frequency              aspecific outer membrane biochemical alteration <br /> eliminates increased permeability pheno-           imp mutants. explanation types;  haploid strains unable         imp mutations alter minor component outer direct selection reversal increased            membrane present assays sensitive <br /> permeability phenotypes yields pseudorevertants.          case tolC mutants, Lastly, attempts obtain amber mutations, gene fu-          increased permeability phenotype,cloningand <br /> sions  transposons imp gene            overexpression tolC geneproduct  minor <br /> unsuccessful.                                                 outer membrane protein) required order     Diploid reversion analysis studies          biochemically detect alteration (MORONA, MAN- <br /> certain phenotypes conferred imp alterations           NING REEVES      1983). separated, e., Dex+ MacSfrom NovSand                fact detect changein sensitivity <br /> Em&apos;. explanation forthis theimp alteration        LPS requiring bacteriophages, EDTA poly- <br /> effects different barrier functions outer          myxin suggests LPS structure dramatically <br /> membrane, recessive fashion             altered. supported silver strain- <br /> dominant fashion. favor alternative simpler            ing LPS imp-208 mutant gives <br /> explanation thevarious phenotypes reflect differ-        apattern identical parentalstrain(datanot <br /> ent levelsof increased permeability single           shown).  preclude possibility <br /> barrier function. present possible know       theimp mutations alterthe LPS subtlefashion   alternatives correct.          detected assays. Recent work Roque <br />     Phenotypic analysisof imp mutantsdemon-               et al. (1988) hyperpermeableabs mutation stratesthe outermembranebarrierfunctions                 shown alterations polycationic binding altered large number     compounds (Table       LPS result hyperpermeable phenotype. 2 ) . Disc sensitivity tests imp mutations      present, know imp mutations alter <br /> increases sensitivity antibiotics.        property LPS. <br /> MIC determinations greater             T h e suggestion imp gene encodes essen- <br /> 100-fold increase sensitivity certain hydrophobic       tial cellular function implies gene involved <br /> antibiotics normally cross outer membrane           fundamentalcellular process. present, theonly <br /> poorly small increase sensitivity      known component outer membrane essential antibiotics predominately cross outer            cell viability lipid  fatty acid component <br /> membrane porins. Interestingly, general           LPS (RAETZ 1987).T h e possibility lipid  <br /> pattern sensitivities similar conferred      tabolism altered imp mutants attractive <br /> small frame deletionswithin porin genes    (MISRA       atthis time know case. BENSON1988b; BENSON,OCCI SAMPSON                         cloned imp&apos; geneontoa            low copy <br /> 1988). suggested porin mutations            number plasmid used plasmid-chromosomal  <br /> conferthesephenotype          alteringthemembrane          combination obtain imp-4213 allele. T h e clon- <br />  500                                              Sampson, <br />                                                 Misra <br />                                                                                            B.      R.                    S.  Benson <br />  <br /> ing strategy general allows cloning                  CLARK, D., 1984 Novel antibiotic hypersensitive mutants Esch- dominant recessive alleles. case,                    erichia coli genetic mapping chemical characterization. <br />                                                                         FEMS Microbiol. Lett. 21: 189-195. <br /> wild type allele (Nov&apos; Mac&apos;) recessive mutant              COLEMAN,   W. G., JR., K. DESHPANDE,      1985 New cysE-pyrE- <br /> NovSphenotype, dominant Mac&apos; pheno-                         linked rfa mutation Escherichia coli K-12 result type. able identify correct clone                     heptoseless lipopolysaccharide.J. Bacteriol. 161: 1209- 1214. <br /> screeningfor thedominant,         Mac&apos;, plasmid deter-               DAVIES,J. K., P. REEVES, 1975 Genetics resistance mined phenotype recombinational loss                      colcins Escherichia coli K-12: cross-resistance colicins <br />                                                                         group  J. Bacteriol. 123: 102-1 17. <br /> chromosomal NovSphenotype. importantly,                     EMR,S., T . J. SILHAVY, 1980Mutations affecting localization allowed demonstrate directly                   Escherichia coli outer membrane protein, bacteriophage <br /> cloned correct gene showing mutant                      lambda receptor. J. Mol. Biol. 141: 63-90. <br /> imp alleles berecombined ontothe plasmid                       GIGOT,D., M. CRABEEL,      FELLER,   D. CHARLIER,   W. LISSENS,   N. <br /> genome. eliminated possibility ob-                    GLANSDORFF      PIERARD,     1980Patterns Polarity                                                                         Escherichia coli carAB gene cluster. J. Bacteriol. 143: 914- <br /> served phenotype resulted form overexpression                      920. <br /> unrelated cloned gene. plasmids used                   GOLDMAN,    R., W. KOHLBRENNER,       P. LARTEYand  PERNET, demonstrate codominant natureof                       1957 Antibacterial agents specifically inhibiting lipopolysac- <br /> mutations gene dosage mutant                       charide synthesis. Nature 3 2 9 162-164. <br />                                                                      HENSON, J. M., J. R. WALKER,     1982 Genetic analysis arcA wild-type alleles correlated degreeof                                                                         lir mutations Escherichia coli. J. Bacteriol. 152: 1301- <br /> imp conferred phenotypes.                                                1302. <br />    present, understand molecular                    HOFNUNG,  M., D. HATFIELDand M. SCHWARTZ, 1974malB region <br /> basis phenotypes observed imp mutants,                   Escherichia coli K-12: characterization ofnew mutants. J. mechanisms mutations cause                     Bacteriol. 117: 40-47. <br />                                                                      KLECKNER, N.,   K. REICHARDT   D. BOTSTEIN,    1979 Inversions <br /> increased permeability.  time,                                                                         deletions Salmonella chromosome generated rule possibility effect outer                   translocatable tetracycline element TnIO. J. Mol.Biol. 127: <br /> membrane permeability indirect                      89-1 15. <br /> imp mutations alter process directly related               LABEDAN,  B., 1988 Increase permeability Escherichia coli aspect envelope structure                     outer membrane local anesthetics penetration anti- <br />                                                                         biotics. Antimicrob. Agents Chemother. 32: 153-155. cell wall inner membrane.Nonetheless,                                                                      LEIVE,L., 1965 Releaseof lipopolysaccharide EDTA treat- <br /> believe mutants define anew essential gene                   ment E. coli. Biochem. Biophys.Res. Commun. 21: 290- E. coli provide new tools genetic analysis               296. complexities outer membrane. Specifi-                  MILLER,J. H., 1972 Experiments Molecular Genetics.Cold Spring <br /> cally, able localized mutagenesis                   Harbor Laboratory, Cold Spring Harbor, NY. <br />                                                                      MISRA,M., S.  BENSON,      1988a Isolation characteriza- <br /> obtain additional imp mutations, use imp  <br />                                                                         tion OmpC porin mutants altered Pore properties. J. <br /> ferred phenotypes develop selections pseudo-                     Bacteriol. 1 7 0 528-533. <br /> revertants suppress moreof sensitiv-                MISRA,M., S.  BENSON,    1988b Genetic identification ities conferred mutations. pseudorever-                     pore domain OmpC porin Escherichia coli K-12. J. <br /> tants right provide novel insights                     Bacteriol. 1 7 0 361 1-3617. outer membrane specifically excludes                    MISRA,R.,B. <br />                                                                                         SAMPSON,    J. L. OCCI S.  BENSON, <br />                                                                         1987 Identification mutants altered outer membrane <br /> hydrophobic compounds cell,                      permeability, pp. 436-442, Antibiotic Inhibition Bacterial imp mutations alter outer membrane perme-                       Cell Surface Assembly andFunction, edited P. ACTOR, L. <br /> ability.                                                                DANA-MOORE,      M. HIGGINS,  M. SALTON   G. D. SCHOCKMAN. <br />                                                                          SOC.Microbiol., Washington, DC. <br />  work supported U.S. Public Health Service grant         MORONA,  R.,P  MANNINGand P. REEVES,1983 Identification <br /> GM34810 National Institutes Health.                         characterization TolC protein, outer membrane <br />                                                                         protein Escherichia coli. J. Bacteriol. 153: 693-699. <br />                    L T E R T U R E CITED                         NAGELDE ZWAIG, R., S. E. LURIA,1967 Genetics physi- <br /> ANDRESSON,    0. S., andJ. E. DAVIES,1980 Organization              ology colicin-tolerant mutants Escherichia coli. J. Bacteriol. <br />    Restriction Enzyme Clevage Map ksgA-pdxA region            94: 1112-1123. <br />    Escherichia coli chromosome. Mol. Gen. Genet. 179: 21 1-      NIKAIDO,H., M. VAARA,1985 Molecular basis bacterial <br />    216.                                                                 outer membrane permeability. Microbiol. Rev. 49: 1-32. <br /> BENSON, S. , DECLOUX, 1985Isolation characterization       NIKAIW, H., M. VAARA,1987 Outer membrane, pp. 7-22, <br />    outer membrane permeability mutants Escherichia coli K-        Escherichia coli andSalmonellatyphimuriumCellularand <br />    12. J. Bacteriol. 161: 361-367.                                      Molecular Biology, edited F. NEIDHARDT,      J. INGRAHAM,     K. <br /> BENSON,S. , J. L. Occrand B.  SAMPSON,      1988 Mutations          B. BROOKS,   B. MAGASANIK,    M. SCHAECHTER    H. E. UMBAR- <br />    alter pore function OmpF porin Escherichia        GER. American Society Microbiology, Washington, D.C. <br />    coli K12. J. Mol. Biol. 2 0 4 961-970.                            RAETZ,C.R. H., 1987 Structure         biosynthesisoflipid CAIRNS, J., J. OVERBAUGH     S. MILLER, 1988 origins         Escherichia coli, pp. 498-503 Escherich <br /> </body></html>