<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Evidence binding C5 protein P RNA <br /> enhances ribozyme catalysis influencing <br /> active site metal <span id='am-6' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> affinity <br />  <br /> ABSTRACT RNA subunit (P RNA) bacterial RNase P ribonucleoprotein ribozyme catalyzes Mg-dependent hydrolysis pre-tRNA, requires essential protein cofactor (P protein) vivo enhances substrate binding affinities catalytic rates substrate dependent manner. Previous studies Bacillus subtilis RNase P, containing Type B RNA subunit, <br /> showed cognate protein subunit increases affinity metal ions important catalysis, functional role ions unknown.  demonstrate Mg2+ dependence catalytic step Escherichia coli RNase P, contains common Type RNA subunit, modulated cognate protein subunit (C5), indicating property fundamental P protein. monitor specifically binding active site metal ions, analyzed quantitatively rescue Cd2+ inhibitory Rp phosphorothioate modification pre-tRNA cleavage site. results binding C5 protein increases apparent affinity rescuing Cd2+, providing evidence C5 protein enhances metal <br /> ion affinity active site, likely contribute significantly rate enhancement physiological metal ion <br /> concentrations. <br /> Keywords: ribozyme; catalysis; ribonucleoprotein; RNase P; active site <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                   collaborate achieve specific physiological function                                                                                important goal. biological functions RNA protein intimately <br />                                                                                   ribonucleoprotein enzyme protein sub- <br /> related extensive structural functional studies <br />                                                                                unit plays key role specificity catalysis ribonu- <br /> reveal RNA binding proteins modulate RNA <br />                                                                                clease P, catalyzes 59-end maturation tRNA <br /> function binding stabilizing specific conforma- <br />                                                                                (Hsieh et al. 2004; Kazantsev Pace 2006). bacteria, <br /> tions (e.g., Weeks Cech 1995; Ho Waring 1999),                                                                                RNase P composed z400 <span id='am-24' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-26' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> (nt) RNA <br /> facilitating necessary conformational changes (Jankowsky <br />                                                                                subunit (P RNA) z100 <span id='am-32' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-35' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> protein subunit Bowers 2006; Caprara et al. 2007), pro- <br />                                                                                (termed C5 Escherichia coli) (Fig. 1; Evans et al. 2006). <br /> viding unique surfaces RNP targeting regulation <br />                                                                                P RNA subunit able recognize pre-tRNAs (Egea et al. 2005; Hopper 2006). coordinated function <br />                                                                                catalyze hydrolysis reaction vitro itself; RNA protein enzyme catalysis fundamental                                                                                substrate affinity, specificity, catalytic rate strongly <br /> ribosome catalyzed protein synthesis numerous <br />                                                                                dependent assembly cognate protein subunit <br /> intron splicing RNA processing reactions.                                                                                (Loria et al. 1998; Niranjanakumari et al. 1998; Kurz essential functional interplay, understanding                                                                                Fierke 2000; Buck et al. 2005a; Sun et al. 2006). Binding RNA proteins form specific complexes                                                                                P protein dramatically increases pre-tRNA affinity,                                                                                making direct contact 59-leader sequence <br />    Reprint requests  Michael E. Harris, Center RNA Molecular            (Niranjanakumari et al. 1998).  E. coli C5 <br /> Biology Department Biochemistry, Case Western Reserve Univer-           protein subunit enhance tRNA binding sity School Medicine, Cleveland, OH 44106, USA; e-mail: meh2@               thought contact 59-leader sequence directly. <br /> cwru.edu; fax: (216) 368-2110. <br />    Article published online ahead print. Article publication <span id='am-5' about='oboInOwl:date' typeof='owl:Thing'>date</span>   Comparative equilibrium binding analyses different http://www.rnajournal.org/cgi/doi/10.1261/rna.571007.                       E. coli pre-tRNAs indicate protein differential <br />  <br /> RNA (2007), 13:1505&#226;&#8364;&#8220;1515. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2007 RNA Society.                                1505 <br />  Sun Harris <br />  <br />  <br />  <br />                                                                                                 effects tRNA interactions, <br />                                                                                                 catalysis, specificity likely result <br />                                                                                                 direct contacts 59 <br />                                                                                                 leader protein-induced <br />                                                                                                 conformational changes occur                                                                                                 catalytic core ribozyme (Buck <br />                                                                                                 et al. 2005b). understanding                                                                                                 mechanistic basis effects                                                                                                 currently limited lack detailed <br />                                                                                                 information structural conse- <br />                                                                                                 quences protein binding,                                                                                                 quantitative description ac- <br />                                                                                                 companying changes enzyme speci- <br />                                                                                                 ficity catalysis. <br />                                                                                                    P RNA considered metal- <br />                                                                                                 loenzyme key insight                                                                                                 demonstration protein subunit <br />                                                                                                 Bacillus subtilis RNase P increases                                                                                                 apparent affinity metal ions impor- <br />                                                                                                 tant catalysis (Kurz Fierke 2002). <br />                                                                                                 foundational struc- <br />                                                                                                 tural kinetic  B. subtilis <br />                                                                                                 enzyme represents minor subclass                                                                                                 RNase P enzyme contains                                                                                                 variant Type B RNA subunit (Haas <br />                                                                                                 et al. 1994). Relatively small changes                                                                                                 RNA structure result significantly <br />                                                                                                 different Mg2+ requirements folding <br />                                                                                                 catalysis (Lehman Joyce 1993; <br />                                                                                                 Frank Pace 1997; Nelson et al. 2005; <br />                                                                                                 Roychowdhury-Saha Burke 2006). <br /> FIGURE 1.   Secondary structure diagram E. coli P RNA. Helices designated P  important question (paired) numbered 59 end RNA. Segments RNA connecting helices previously documented contribu- designated J (joining) numbered according helices connect. helices P1&#226;&#8364;&#8220; <br /> P18 make secondary structure E. coli P RNA labeled. secondary structure <br />                                                                                                 tions RNase P protein catalysis organized according dimensional structure P RNA subunit, arrows E. coli enzyme, contains <br /> depict connections define path RNA chain. (B) Cartoon diagram  common Type RNA sub- <br /> dimensional structure Type P RNA pre-tRNA bound. structure diagram unit, enhancement based crystal structure Thermotoga maritime P RNA (Torres-Larios et al. 2005).            2+ <br /> Individual helices shown cylinders colored according secondary structure Mg binding.  follows diagram panel  bound pre-tRNA depicted black ribbon P protein enhancement direct <br /> subunit shown sphere. 59-leader sequence pre-tRNA shown dashed line. effect increasing affinities active <br /> (C) Sequence secondary structure pre-tRNAMet608. location RNase P cleavage site metal ions. <br /> site nucleotides N(+1) N( 1) indicated arrow. (D) proposed active site metal ion interactions reactive phosphate pre-tRNA           lines evidence indi- <br /> transition state. described text, metal ions proposed, interact cate P RNA positions active site pro-Rp phosphate oxygen reactive phosphate. position substituted metal ions. Detailed structural sulfur atom Rp(+1) pre-tRNA substrate.                                                   functional studies protein endoribo- <br />                                                                                                 nucleases catalyze chem- <br />                                                                                                 istry P RNA presence effects thermodynamic contribution tRNA                       active site metal ions (Cowan 1998). Additionally, leader portions substrate low affinity               presence active site metal ions Group intron tRNA bodies compensated combining                       ribozyme, catalyzes similar intermolecular nucleo- 59 leaders increase affinity                  philic attack phosphodiester, strongly supported order reach uniformity binding (Sun et al. 2006).                     detailed biochemical high resolution structural data <br /> Additionally, protein subunit E. coli RNase P                    (Hougland et al. 2005; Stahley Strobel 2006).  <br /> shown significantly enhance rate substrate                    rate enhancement P RNA clearly dependent cleavage step, substrates &gt;800-fold.                  Mg2+ ion concentration conditions  <br /> 1506       RNA, Vol. 13,  9 <br />                                                                                    Protein modulation ribozyme metal affinity <br />  <br />  <br />  <br /> RNA subunit folded enzyme concentrations                substrate phosphorothioate.  data provide <br /> saturating (Smith et al. 1992; Beebe et al. 1996).         evidence important function bacterial RNase  phosphorothioate modifications pre-tRNA               P protein subunit enhance rate catalysis cleavage site inhibit P RNA catalysis, effects          increasing metal ion affinity active site pro-Rp phosphate oxygen modification sulfur              catalytic RNA subunit. <br /> rescued addition thiophilic metal ion Cd2+ <br /> (Warnecke et al. 1996). general metal ion mecha- <br />                                                                    RESULTS <br /> nism RNase P catalysis assumed  Fig. 5,  catalytic metal ions positioned            quantify contribution P protein subunit coordinate pro-Rp oxygen reactive phosphate             rate enhancement RNase P, determined effect (Steitz Steitz 1993; Warnecke et al. 1996).         C5 protein binding single turnover rate constant ions helps position water molecule nucleophile         cleavage E. coli pre-tRNAMet608. pre-tRNAMet608 assist deprotonation, second proposed        substrate chosen representative coordinate stabilize 39O leaving group. Analyses        &#226;&#8364;&#732;&#226;&#8364;&#732;typical&#226;&#8364;&#8482;&#226;&#8364;&#8482; E. coli pre-tRNA substrate contains bonding environment nucleophilic water           optimal recognition elements defined <br /> transition state using kinetic isotope effects consistent       sequence comparisons structure function studies. direct metal ion catalysis nucleophilic activation          mechanistic features substrate recognition (Cassano et al. 2004).  previously observed            catalysis defined using substrate anticipated effects C5 protein enhancing catalytic rates results         reflect general features RNase P enzymology. Analyses general property P protein influencing            performed typically conditions 100 mM <br /> metal ion dependence catalysis, raises              NaCl, 17.5 mM Mg2+, determined optimal <br /> possibility effects related enhanced        holoenzyme multiple turnover reaction (Guo et al. <br /> active site metal ion affinity.                                    2006). Reaction pH Mg2+ concentration varied     contributions metal ions RNA                 described  shown Figure 2A, identical <br /> function complex, relating binding specific         reaction conditions holoenzyme z10-fold higher <br /> ions specific functional roles continues       rate constant cleavage pre-tRNAMet608 compared vexing problems field. RNAs bind numerous              P RNA (0.14 min 1 versus 1.3 min 1). lower rate <br /> metal ions solution, interact site specifically       constant P RNA conditions direct coordination H-bonding electrostatic           slower, noncatalytic rate-limiting step  <br /> interactions (DeRose 2003; Draper et al. 2005). classes       hanced protein subunit. scenario protein make significant contributions function            subunit change mechanism reflected Mg2+ titration experiments, difficult       substrate cleavage, slower upstream step, interpret terms thermodynamics individual          rate limiting. Alternatively, protein subunit metal ion interactions. Additionally, established       enhance catalysis direct fashion altering ribozyme systems Mg2+ binding increase           affinity metal ions necessary catalysis suggested <br /> catalytic rate constants indirectly stabilizing active RNA      previously results B. subtilis RNase P. <br /> conformations promoting substrate docking steps                 assess protein changes rate-limiting <br /> (e.g., Shan Herschlag 2002; Tinsley et al. 2004).         step pre-tRNA cleavage reaction, compared contribution protein subunit P RNA             pH dependence single turnover rate constants catalysis appears related binding Mg2+ ions,       P RNA presence absence C5 protein. Analysis binding sites functional roles ions           pH dependence cleavage rate constant probe <br /> fundamentally difficult establish.                              rate limiting step RNase P reaction based <br />    address issues, compared pH Mg2+            rationale deprotonation yield hydroxide <br /> dependence catalytic step P RNA subunit          ion nucleophile essential step RNase P catalyzed <br /> E. coli RNase P presence absence cognate         phosphodiester bond hydrolysis reaction. Accordingly, protein subunit (C5 protein) identical reaction              rate substrate cleavage increases increasing hydrox- <br /> conditions. Additionally, used quantitative analysis         ide ion concentration follows rate law specific Cd2+ rescue cleavage site Rp phosphorothioate (PS)        base catalysis (Smith Pace 1993; Cassano et al. 2002; <br /> modification isolate binding thermodynamics              Persson et al. 2003).  plots log(kobs) versus pH active site metal ions (Peracchi et al. 1997; Shan et al. 2001).   linear slope unity suggest <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> comparative kinetic analyses C5 protein              step rate limiting, alteration rate- <br /> reduces apparent affinity Mg2+ ion essential            limiting step reaction likely result absence <br /> catalysis changing rate-limiting step. Impor-          significant decrease pH dependence. shown <br /> tantly, binding C5 protein P RNA results higher       Figure 2B, rates P RNA holoenzyme <br /> apparent affinity binding rescuing Cd2+ ions       cleavage pH sensitive log-linear relationship <br />  <br />                                                                                                     www.rnajournal.org       1507 <br />  Sun Harris <br />  <br />  <br />  <br />                                                                         kobs pH. slopes reactions                                                                         approximately equal 1 (1.1 1.2 RNase P                                                                         P RNA, respectively), consistent deprotonation                                                                         functional group transition state reactions. <br />                                                                          data indicate essentially                                                                         difference rate-limiting steps holoenzyme <br />                                                                         P RNA reactions reaction conditions. <br />                                                                             tested binding C5 protein influences <br />                                                                         Mg2+ concentration dependence single turnover <br />                                                                         rate constant cleavage pre-tRNAMET608. pH <br />                                                                         titration experiments, reactions performed satu- <br />                                                                         rating concentration enzyme order minimize                                                                         contribution binding steps observed rate constant. <br />                                                                         Additionally, titrations performed Mg2+ concen- <br />                                                                         trations greater concentration required P RNA <br />                                                                         folding (&gt;5 mM) indicated sensitivity chemical                                                                         oligonucleotide probing (Zarrinkar et al. 1996; Pan                                                                         Sosnick 1997; Kent et al. 2000). shown Figure 2C,                                                                         observed single turnover rate constants holo- <br />                                                                         enzyme P RNA reactions increase Mg2+ <br />                                                                         concentration increases. concentrations monovalent <br />                                                                         ions adjusted maintain constant ionic strength, <br />                                                                         site-specific diffuse metal ion interac- <br />                                                                         tions contribute observed metal ion dependence. <br />                                                                         P RNA  apparent Mg2+ affinity compar- <br />                                                                         atively low, estimate apparent <br />                                                                         affinity (Kd,Mg) rate saturating Mg2+ concen- <br />                                                                         trations (kobs(max)) obtained.  binding                                                                         protein subunit sensitive high ionic strength <br />                                                                         (&gt;0.5 M), limiting upper range data                                                                         collected (Day-Storms et al. 2004). Nonetheless,                                                                         data indicate kobs(max) values P RNA <br />                                                                         holoenzyme reactions differ twofold <br />                                                                         (z0.07 s 1 holoenzyme; z0.13 s 1 P RNA . <br />                                                                         Importantly, Mg2+ titration data reactions fit <br />                                                                         noncooperative binding model E. coli holoen- <br />                                                                         zyme displays significantly lower overall Kd,Mg (32 mM) <br />                                                                         compared P RNA reaction (&gt;250 mM).                                                                         highest concentration tested P RNA                                                                         twofold higher apparent Kd,Mg value repre- <br />                                                                         sents lower limit. 10 mM Mg2+, rate enhancement <br />                                                                         presence C5 protein &gt;55-fold,  <br />                                                                         lower concentrations, encountered FIGURE 2. Kinetic analyses pH Mg2+ dependence         cellular environment (z1 mM) (e.g., Froschauer et al. <br /> single turnover rate constants P RNA RNase P holoenzyme         2004; Farruggia et al. 2005), extent enhancement cleavage pre-tRNAMet608.   Single turnover reactions RNase P <br /> holoenzyme (d) P RNA (s) compared saturating           greater. <br /> enzyme concentration  MES 50 mM (pH 5.75), NaCl 100 mM,                 data important establishing MgCl2 17.5 mM, Triton 0.005%. (B) Plots log(kobs) versus pH      contributions C5 protein RNase P function RNase P (d) P RNA (s) fit linear equation (slope=1.1   modulating apparent metal ion affinity, raise obvious 1.2 RNase P P RNA reactions, respectively). (C) <br /> Plot kobs values P RNA RNase P holoenzyme versus Mg2+        questions regarding specific role  roles) metal <br /> concentration. data fit equation cooperative       ion interactions play enzyme catalysis binding mechanism (Equation [2], Materials Methods). Hill       bind enzyme&#226;&#8364;&#8220;substrate complex. Previous studies coefficients unity reactions;  apparent <br /> dissociation constant holoenzyme smaller P RNA <br />                                                                         large small ribozymes clearly demonstrate (KMg [RNase P]=32 mM KMg [P RNA  &gt;250 mM).              metal ions bind distance active site inset shows closeup data low Mg2+ concentrations.       promote folding provide large apparent contribution <br />  <br /> 1508       RNA, Vol. 13,  9 <br />                                                                                         Protein modulation ribozyme metal affinity <br />  <br />  <br />  catalysis (DeRose 2003; Draper et al. 2005).               modynamics (Christian 2002, references . concentrations divalent ions used Figure 2 (17.5 mM),          test Cd2+ rescue catalytic defect caused <br />         2+ Mg requirement P RNA global folding satisfied             substrate modification, single turnover cleavage reac- <br /> (z5 mM) (Zarrinkar et al. 1996; Fang et al. 2000);           tions performed presence absence Cd2+ <br />                                                                2+ possible C5 protein enhances affinity Mg           background 17.5 mM Mg2+ ensure appropriate <br /> ions necessary local folding transition affects       RNA folding minimize effects Cd2+ active site necessary substrate positioning       folding. experiments, substrate containing (Christian et al. 2006). Additionally, described          Rp phosphorothioate modification N(+1) position <br /> simple model C5 protein directly affects           pre-tRNAMet608 (Rp[+1] pre-tRNA) created ligat- <br /> affinity metals ribozyme active site.                  ing 11 nt oligo encompassing cleavage site 39 <br />    strategy observing effect protein subunit        fragment containing remainder tRNA portion specific binding catalytic metal ions suggested         substrate. shown Figures 3 4, PS observation replacing pro-Rp oxygen           modification results significant defect rate substrate phosphate sulfur results &gt;1000-fold              substrate cleavage (kOobs/kSobs&gt;1000) Rp(+1) pre- <br /> defect catalysis, adding &#226;&#8364;&#732;&#226;&#8364;&#732;thiophilic&#226;&#8364;&#8482;&#226;&#8364;&#8482; metal          tRNA substrate. defect deleterious effect <br /> ion, like Mn2+ Cd2+, reaction rescues cleavage           substrate binding, assessed measuring <br /> (Warnecke et al. 1996; Chen et al. 1997).                  substrate dissociation constants gel filtration. Apparent <br /> appropriate conditions result interpreted           Kd z1 nM binding pre-tRNA Rp(+1) pre- <br /> indicating rescuing ion coordinated directly          tRNA, respectively, RNase P. modification site-specific sulfur modification analogy            minor effect binding P RNA (Kd Mg2+ ions coordinated analogous oxygen atom               1 mM 3 mM pre-tRNA Rp[+1] pre-tRNA, unmodified RNA (Pecoraro et al. 1984; Piccirilli et al.       respectively). Importantly, catalytic defect 1993). Group intron, Group II intron,                  rescued addition Cd2+ P RNA holo- <br /> hammerhead ribozymes, quantitative analysis metal                 enzyme reactions. data illustrate ion dependent PS rescue provided means                     second criterion met presence Cd2+ <br /> analyzing relative affinity specific active site metal        does result significant miscleavage (Fig. 3).  <br /> ions (Peracchi et al. 1997; Shan et al. 1999; Gordon             presence phosphorothioate modification Piccirilli 2001). analysis, nonspecific effects           Rp(+1) pre-tRNA does engender miscleavage reaction rate caused presence Cd2+ accounted            absence Cd2+. result important, miscleavage evaluation krel, ratio observed        arises parallel reactions <span id='am-12' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> <br /> rates oxygen-containing substrate sulfur-                obscure intrinsic cleavage rate (Loria Pan 1999; <br /> containing substrate (krel=kSobs/kOobs). Plotting krel versus    Zahler et al. 2005).  rate constants determined <br /> concentrations rescuing Cd2+ provides thermodynamic             absence Cd2+ contain inherent inaccuracy. <br /> signature binding ions substrate cleavage site (Shan et al. <br /> 1999, 2001).  comparison concentration dependence Cd2+ <br /> rescue holoenzyme P <br /> RNA reactions, principle, permit contribution C5 protein active site metal affinity measured. <br />    interpret data terms specific metal ion binding, crite- <br /> ria met:  addition Cd2+ provide significant rescue modified substrate  <br /> ground Mg2+ ions; second, added <br /> Cd2+ cause miscleavage                                                FIGURE 3. Cd2+ rescues cleavage N(+1) PS substrate modification P RNA limits accuracy measured rate RNase P holoenzyme. Single turnover reactions performed standard conditions <br /> constants;  (50 mM MES pH 5.75, 100 mM NaCl, 17.5 mM MgCl2, 0.005% Triton X-100)                                                                                             2+ <br /> change rate limiting step absence (panels 1,3) presence 5 mM Cd (panels 2,4) RNase P holoenzyme (panels <br /> presence absence rescuing metal 3,4) P RNA (panels 1,2) reactions. Aliquots removed predetermined time points                                                analyzed 20% PAGE. positions products resulting correct cleavage ( 1/+1) ion, invalidate quan- miscleavage ( 2/ 1) indicated. product marked asterisk results correct <br /> titative evaluation Cd2+ binding ther- cleavage minor population substrate containing additional nucleotide 59 end. <br />  <br />                                                                                                           www.rnajournal.org       1509 <br />  Sun Harris <br />  <br />  <br />  <br />                                                                            micromolar concentrations Cd2+ entirely <br />                                                                           suppress miscleavage;  comparison rate  <br />                                                                           stants determined conditions accurately <br />                                                                           reflect intrinsic substrate cleavage rate. addition,                                                                           test presence Cd2+ changes rate limiting <br />                                                                           step cleavage Rp(+1) pre-tRNA unmodified <br />                                                                           pre-tRNA, effect pH rate constants                                                                           substrates examined. experiments catalytic <br />                                                                           rate constant pH dependent presence Cd2+ <br />                                                                           15 mM reactions, indicating                                                                           change rate limiting step (data shown). <br />                                                                              criteria Cd2+ dependent rescue, accurate <br />                                                                           cleavage, change mechanism Rp(+1) pre- <br />                                                                           tRNA substrate established, possible employ analysis <br />                                                                           krel   probe active site metal ion affinities <br />                                                                           P RNA presence absence C5 protein. <br />                                                                            determined effect Cd2+ rate <br />                                                                           constant cleavage native oxygen containing <br />                                                                           substrate (kOobs) P RNA presence absence <br />                                                                           C5 protein. shown Figure 4, Cd2+ inhibits                                                                           reactions concentration-dependent, saturable fashion. <br />                                                                           tritation data RNase P P RNA fit                                                                           cooperative binding mechanism classes <br />                                                                           Cd2+ ions bind simultaneously inhibit RNase P <br />                                                                           catalysis. Furthermore, apparent affinity inhib- <br />                                                                           itory Cd2+ ions similar P RNA RNase <br />                                                                           P (Kd,Cd=1.9 mM 3.0 mM P RNA RNase P, <br />                                                                           respectively). contrast, Cd2+ results increase                                                                           rate constant cleavage Rp(+1) pre-tRNA (kSobs)                                                                           P RNA holoenzyme. large degree <br />                                                                           inhibition P RNA reaction, stimulatory <br />                                                                           effect Cd2+ kSobs clearly observed lower <br />                                                                           concentrations (Fig. 4C).  effect Cd2+ ions                                                                           rate constant cleavage Rp(+1) pre-tRNA                                                                           combination site-specific rescue effect                                                                           inhibition. <br />                                                                              isolate thermodynamics binding                                                                           rescuing metal ions, krel utilized described                                                                           (krel=kSobs/kOobs), normalizes inhibitory effect <br />                                                                           Cd2+ cleavage reactions catalyzed P RNA                                                                           presence absence C5 protein. plot krel versus <br /> FIGURE 4. Quantitative analysis Cd2+ dependent rescue           concentration rescuing Cd2+ ions fits (+1 Rp) PS substrate modification. Single turnover rate constants         cooperative binding model holoenzyme determined series increasing Cd2+ concentrations                                                                           P RNA reactions (Fig. 5A). apparent dissociation <br /> background 17.5 mM Mg2+.   Plots kobs (RNase P) versus Cd2+ <br /> concentration shown PS modified (s) unmodified (d)            constant catalytic metal ion binding (Kd,app) substrates. (B) Plots kobs (P RNA  versus Cd2+ concentration     holoenzyme reaction 9.3 mM Hill coefficient (n) shown modified (s) unmodified (d)substrates. (C)          2, indicating metal ions interact Rp <br /> kobs (P RNA  data PS modified substrates shown separately better view Cd2+ stimulation low Cd2+ concentrations. Data <br />                                                                           sulfur modification Rp(+1) pre-tRNA, consistent represent average values independent trials.       metal ion model described (Fig. 5B).                                                                           P RNA  Cd2+ concentration significantly higher <br />                                                                           15 mM required reach saturation.  <br /> Nonetheless, rate constants small                    precipitation RNA higher Cd2+ concentrations limits <br /> relative rate constants cleavage unmodified             precise measurement metal ion affinity substrate (&gt;1000-fold), inaccuracy 0 mM Cd2+                  cooperativity, Kd,app estimated significantly <br /> data point little effect subsequent krel analysis.             &gt;30 mM.  apparent dissociation constant  <br /> 1510       RNA, Vol. 13,  9 <br />                                                                                                 Protein modulation ribozyme metal affinity <br />  <br />  <br />  <br />                                                                               model protein subunit significant effect                                                                               catalysis influencing affinity active site metal <br />                                                                               ions, increasing apparent rate constant pre- <br />                                                                               tRNA cleavage (Fig. 6). <br />                                                                                  important observation regarding differential <br />                                                                               functional effects RNA metal ion interactions                                                                               significant inhibition P RNA RNase P holo- <br />                                                                               enzyme increasing concentrations Cd2+.                                                                               PS-modified substrate used, activation inhibi- <br />                                                                               tion contribute observed Cd2+ dependence                                                                               krel approach permits affinity activating ion binding <br />                                                                               deconvoluted.  underlying quantitative <br />                                                                               relationship observed inhibitory Cd2+ binding <br />                                                                               affinity active site metal affinity PS-modified <br />                                                                               substrate difficult evaluate. Inhibition arise                                                                               competition active site metal ions indirectly                                                                               affecting active site geometry distance. Figure 4                                                                               shows significant different magnitude <br />                                                                               effect increasing Cd2+ concentration rate <br />                                                                               constants P RNA RNase P reactions.                                                                               order interpretation geometry RNA                                                                               Cd2+ bound different C5 protein bound                                                                               P RNA, difference structure accounts FIGURE 5. Comparison krel function Cd2+ concentration              difference degree inhibition observed. P RNA RNase P holoenzyme shows C5 enhances binding affinity catalytic metal ion.   Relative reaction rates (krel) <br /> calculated ratio observed reaction rates substrates sulfur substitution  e., krel=kSobs/kOobs) plotted <br /> versus Cd2+ concentrations. Data holoenzyme reaction represented (d) P RNA represented (s). Fitting Hill equation reveals n=2 holoenzyme reaction, KCd (holo)=9 mM KCd (P RNA &gt;30mM. (B) Interpretation concentration dependent rescue Rp phosphorothioate <br /> modification Cd2+. krel analysis controls non- <br /> specific effects Cd2+, cooperative nature Cd2+-dependent <br /> rescue suggests rescuing metal ions interact single <br /> sulfur modification, reflects metal ions binding P RNA active site native, unmodified substrate. <br />  <br />  <br />  <br /> active site metal ion binding threefold higher P RNA RNA&#226;&#8364;&#8220;protein complex. <br />  <br /> DISCUSSION Cd2+ rescue experiments accurately reflect interactions Mg2+ interactions reactive phos- <br /> phate, observed increase apparent Mg2+ affinity <br /> observed Figure 2 reflects active site metal ion <br /> binding. close correspondence metal ion <br /> interactions Group intron ribozyme active site <br /> identified quantitative metal ion rescue experiments                   FIGURE 6. P protein increases affinity metal ions active <br />                                                                               site P RNA. Interpretation Cd2+ rescue data supports high resolution X-ray crystallography indicates                     model Mg2+ ions bind active site enzyme&#226;&#8364;&#8220; <br /> assumption likely valid (Shan et al. 1999; Stahley                   substrate complex ground state. apparent affinities Strobel 2005).  simple interpretation                     rescuing ion interactions indicated.  affinities                                                                               corresponding active site Mg2+ ions significantly different. <br /> data supports model RNase P catalysis                                                                                indicated text, evaluation Mg2+ titration <br /> divalent metal ions bind reactive phosphate                     data consistent cooperative binding high affinity class enzyme&#226;&#8364;&#8220;substrate complex essential catalysis.                    metal ions contributes catalysis indicated diagram. <br />  <br />  <br />                                                                                                                    www.rnajournal.org          1511 <br />  Sun Harris <br />  <br />  <br />  <br /> Additionally, titration data distinguish             C5 protein make relatively little contribution single inhibitory Cd2+ ion bound            catalysis (Beebe Fierke 1994; Sun et al. 2006) multiple Cd2+ ions binding noncooperatively similar          low degree sequence conservation P protein compared <br /> affinity.  second possibility P RNA        catalytic core P RNA (Haas Brown 1998; <br /> additional inhibitory Cd2+ binding interactions          Jovanovic et al. 2002).  presence distinct RNase P holoenzyme.                             unique structural motifs protein cofactor,    illustration multicomponent nature         conceivable polypeptide backbone itself RNA&#226;&#8364;&#8220;metal ion interactions provided differences         contribute metal ion binding. Alternatively, P protein apparent binding thermodynamics &#226;&#8364;&#732;&#226;&#8364;&#732;catalytic&#226;&#8364;&#8482;&#226;&#8364;&#8482; metal         act indirectly, inducing conformational change ion binding total Mg2+ titration versus active site metal     P RNA results optimal alignment RNA <br /> ion binding obtained krel analysis. apparent binding       functional groups contact metal ions. Previous studies <br /> thermodynamics rescuing Cd2+ ions cooperative,          shown protein decreases Mg2+ metal ion binding bulk Mg2+ titration experi-            requirement folding (Buck et al. 2005a).  ments, contain significant contribution            effects manifest lower concentrations active site metal ion binding,  regard       titration analyses presented chemical oligo- important note additional Mg2+ ions          nucleotide probing confirm P RNA folded necessary catalysis, bind active      range Mg2+ concentrations tested (Zarrinkar <br /> site, detected PS rescue experiments,     et al. 1996; Sun et al. 2006).  single turnover report interaction metal ions       analyses performed saturating concentrations introduced sulfur modification. Based physical studies         enzyme control differences active model systems, likely bulk Mg2+ titration       fraction enzyme different metal ion concentrations. <br /> results reflect binding multiple classes metal ions,    Alternatively, protein contribute substrate <br /> including diffuse ions interact RNA          docking step, initial substrate binding, site-specific fashion. data Figure 2 bulk         binding sites active site metal ions formed. Mg2+ titration fit model binding single class    catalysis slow relative equilibration activating Mg2+ ions noncooperative manner. Inspec-          docked undocked forms, pH dependence tion titration data low Mg2+ ion concentrations         P RNA holoenzyme reactions reveals observed rate constants            phosphodiester bond cleavage rate limiting cases <br /> accounted complex model                (Loria Pan 1999), consistent data reported <br /> classes ions necessary catalysis: high affinity       model docking results formation class binds cooperatively lower affinity class    metal ion binding sites associated catalysis, binds noncooperative fashion.  inaccuracies          shifting equilibrium docked state result current data set measuring small rate constants        decrease concentration Mg2+ required controlling depletion Mg2+ high              reach saturation, consistent observations <br /> concentrations P RNA required achieve saturation            reported  Previous studies Fierke colleagues <br /> make difficult distinguish models.            showed shortening leader sequence model pre- <br /> Additionally, differences size charge distribution     tRNA substrate decreased apparent affinity metal ion Cd2+ versus Mg2+ changes electrostatic envi-         interactions necessary cleavage (Kurz Fierke 2002). <br /> ronment active site introduction PS           preliminary results truncation modification result differences apparent         59-leader sequence significant effect extent <br /> Mg2+ Cd2+ affinity. Nonetheless, results provide        catalytic enhancement C5 protein binding evidence specific effect RNase P protein   P RNA (L. Sun, F.E. Campbell, M.E. Harris, prep.) <br /> subunit active site metal ion binding physio-          consistent indirect effect protein logical divalent metal ion concentrations (0.5&#226;&#8364;&#8220;1 mM)              direct effect contribution functional groups (e.g., Froschauer et al. 2004; Farruggia et al. 2005)     substrate positioning interactions. large.                                                          ability RNA-binding proteins alter appar- <br />    Given importance understanding cooperative          ent metal ion affinity cognate RNA ligands function RNA protein, mechanism               documented, providing direct experimental linkages <br /> RNase P protein acts enhance active site metal ion             metal ion binding function remains challeng- <br /> affinity significant  effect protein        ing. Group Group II self-splicing introns require <br /> binding active site metal ion affinity principle      essential Mg2+ ion cofactors, binding maturase <br /> result P protein contributing protein functional             proteins essential processing vivo (Lambowitz groups serve additional ligands metal ions         Zimmerly 2004; Stoddard 2006) results significant active site. possibility unlikely given       decreases levels Mg2+ necessary achieve optimal substrates reaction conditions          catalytic rates vitro. effects generally <br />  <br /> 1512      RNA, Vol. 13,  9 <br />                                                                                         Protein modulation ribozyme metal affinity <br />  <br />  <br />  <br /> attributable metal ions involved RNA folding,           products resolved denaturing PAGE (20%). conver- <br /> necessarily coupled active site metal ion affinity,           sion substrate product quantified phosphorimaging <br /> formation active site obviously requires folding.               using Molecular Dynamics ImageQuant software proteins result single round self-                 (Amersham). data plotted versus time using Kaleida- <br />                                                                        Graph software (Synergy) fit single exponential function: <br /> splicing, differential effects P protein cleavage <br /> step different substrates provide mechanism                                                                                                  F c &#194;&#188; 2 Be2kt                        (1) <br /> substrate-dependent regulation. Clearly, integral roles P protein substrate recognition catalysis <br /> rationalize necessity protein subunit vivo.              Fc fraction cleaved, maximal extent                                                                        reaction, B amplitude exponential, k observed  mechanism protein alters <br />                                                                        cleavage rate constant (kobs), t time. Rate constants reported <br /> catalytic metal ion affinity determined.  <br />                                                                        average independent determinations, standing mechanistic basis likely help illuminate           &lt;10% error curve fit primary data. important interplay protein binding                       Cleavage reactions presence increasing (Mg2+) RNA function.                                                          performed essentially described  kobs plotted <br />                                                                        function (Mg2+) concentrations fitted form                                                                        Hill equation: <br /> MATERIALS METHODS E. coli P RNA pre-tRNAMet608 generated vitro                                      kmax &#195;&#176;&#194;&#189;Mg2+  n &#195;&#382; <br />                                                                                            kobs &#194;&#188;        n            n                 (2) <br /> transcription previously described (Sun et al. 2006). C5                                     K Mg + &#195;&#176;Mg2+ &#195;&#382; <br /> protein expressed E. coli isolated affinity purifica- <br /> tion described (Guo et al. 2006). (+1 Rp) PS modified pre-      kobs rate constant measured Mg2+ concen- <br /> tRNA (Rp[+1] pre-tRNA) generated ligation ( 5, +6)         tration, kmax cleavage rate constant saturating Mg2+ <br /> 11 nt oligo (Dharmacon) (+7, +74) 39 fragment pre-            concentration, (Mg2+) Mg2+ concentration, KMg tRNAMet generated vitro transcription (Zahler et al. 2005).      apparent dissociation constant, n Hill coefficient. shown Figure 3 substrate gives rise primarily cleavage      pseudo order rate constants determined correct phosphodiester bond 59 N(+1) tRNA.             described series increasing [Cd2+]. relative rate <br /> additional product migrating slower correct cleavage          krel calculated ratio observed reaction rates product observed, marked asterisk         substrates sulfur substitution (e.g., krel = <br /> Figure 3. established product artifact arising      kSobs/kOobs). Plots kobs versus (Cd2+) fit Hill ligation reaction. Observation specifies           equation described Mg2+ titration analyses. <br /> independent presence phosphorothioate modifica- <br /> tion observed substrates generated vitro <br /> transcription. Formation product correlates           ACKNOWLEDGMENTS <br /> appearance correct cleavage product. minor <br />                                                                        <br /> product (&lt;5%), rate determinations corrected                                                                        <br /> account formation. <br />                                                                        <br />    kinetic thermodynamic studies, pre-tRNAs 59-end <br />                                                                         <br /> labeled [g-32P]-<span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span> (MP Biomedicals) T4 polynucleo- <br />                                                                        <br /> tide kinase dephosphorylation alkaline phosphatase. <br />                                                                         <br /> Single turnover kinetic analyses performed essentially described (Sun et al. 2006) following modifications. Rates <br />                                                                       measured following conditions, pH metal <br /> ion concentration varied indicated text: 50 mM MES (pH <br /> 5.75), 100 mM NaCl, 17.5 mM MgCl2, 0.005% Triton X-100. <br />                                                                      <br /> Enzyme concentrations determined saturating observed change rate double enzyme       <br /> 10%. Enzyme (1 mM RNase P 10&#226;&#8364;&#8220;30 mM P RNA) 59-32P-                 <br /> end-labeled pre-tRNA (1&#226;&#8364;&#8220;4 nM) renatured separately    <br /> absence divalent metal ion incubation 95&#194;&#176;C 3 min,         <br /> followed decreasing temperature 37&#194;&#176;C 0.2&#194;&#176;C          <br /> second incubating 10 min. MgCl2 added          <br /> desired concentration mixture incubated         <br /> 10 min. holoenzyme reactions, C5 protein added              <br /> (final concentration equal P RNA), incubation          <br /> continued 10 min. Enzyme substrate mixed            start reaction, aliquots removed           <br /> predetermined intervals quenched EDTA twice         concentration divalent metal ion reaction. Reaction        <br /> </body></html>