<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 20  24 pp. 7209&#194;&#177;7219, 2001 <br />  <br />  <br /> RTG-dependent mitochondria nucleus signaling negatively regulated seven WD-repeat <br /> protein Lst8p <br />  <br />  <br /> Zhengchang Liu, Takayuki Sekito1,                                      maintain supply key TCA cycle intermediates, e.g. <br /> Charles B.Epstein2 Ronald Butow3                                 oxaloacetate, severely <br />                                                                        limiting cells mitochondrial ciencies. Department Molecular Biology, University Texas Southwestern <br /> Medical Center, 5323 Harry Hines Blvd, Dallas, TX 75390-9148, USA      expression genes retrograde pathway, <br /> 1 <br />                                                                        CIT2 (encoding glyoxylate cycle isoform  Present address: Department Cell Biology, National Institute   citrate synthase), DLD3 (encoding D-lactate dehydro- <br /> Basic Biology, Nishigonaka 38, Myodaiji 444-585, Aichi, Japan <br /> 2 <br />  Present address: Aventis Pharmaceuticals, , 26 Landsdowne         genase) PDH1  gene involved propionate <br /> Street, Cambridge, MA 02139, USA                                       metabolism) directly dependent regulatory <br /> 3                                                                      genes, RTG1, RTG2 RTG3 (Liao Butow, 1993; Jia <br />  Corresponding author <br /> e-mail: butow@swmed.edu                                                et al., 1997; Chelstowska et al., 1999). <br />                                                                           cells respiratory cient reduced cells reduced mitochondrial function, RTG1, 2                  respiratory activity, RTG genes regulate 3 required expression genes involved                 expression CIT1, ACO1, IDH1 IDH2 (Liu glutamate synthesis. Glutamate negatively regulates                    Butow, 1999), encode enzymes catalyzing &#194;&#174;rst <br /> RTG-dependent gene expression upstream Rtg2p,                       steps TCA cycle lead synthesis   turn, acts upstream bHLH/Zip tran-                    ketoglutarate, precursor glutamate. cells scription factors, Rtg1p Rtg3p. report                     robust mitochondrial respiratory activity, expression mutations [lst8-(2&#194;&#177;5)] LST8, essential                 TCA cycle genes control HAP <br /> gene encoding seven WD40-repeat protein required                     transcription complex (Forsburg Guarente, 1989; targeting <span id='am-34' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> permeases (AAPs)                    Gangloff et al., 1990; Rosenkrantz et al., 1994). Glu- <br /> plasma membrane, bypass requirement Rtg2p                      tamate homeostasis, central functions RTG- abolish glutamate repression RTG-dependent                      dependent gene expression, exempli&#194;&#174;es retro- <br /> gene expression. lst8-1 mutation,                    grade pathway serves adapt cells mitochondrial <br /> reduces plasma membrane expression AAP,                      dysfunction. levels glutamate regulate retro- <br /> bypass Rtg2p requirement, suppresses                     grade response negative feedback loop, low <br /> glutamate repression RTG target gene expression.                    glutamate levels activate high glutamate levels repress Lst8p negatively regulates RTG gene                       RTG-dependent gene expression (Liu Butow, 1999). <br /> function, acting sites, upstream Rtg2p,                   cells compromised mitochondrial function, <br /> affecting glutamate repression RTG-dependent gene                   mutations RTG genes result glutamate <br /> expression Ssy1p, AAP-like sensor                        auxotrophy (Liao Butow, 1993; Jia et al., 1997). <br /> external amino acids, Rtg2p                         exact mechanism glutamate signaling Rtg1p&#194;&#177;Rtg3p. data, genome-                     unclear, respiratory-competent cells cells grown wide transcription pro&#194;&#174;ling, reveal pathways regu-                     presence high levels glutamate medium, lated glutamate, provide insight                       basic helix&#194;&#177;loop&#194;&#177;helix/leucine zipper (bHLH/Zip) tran- <br /> regulation cellular responses mitochondrial                      scription factors, Rtg1p Rtg3p, present dysfunction.                                                           inactive complex cytoplasm (Sekito et al., 2000). Keywords: LST8/mitochondria/RTG genes/WD40 repeat/                     respiratory cient cells cells grown medium <br /> yeast                                                                  lacking glutamate, factors accumulate nucleus <br />                                                                        processes requiring Rtg2p, novel cytoplasmic protein <br />                                                                        N-terminal <span id='am-18' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span>-binding domain similar                                                                        hsp70/actin/sugar kinase superfamily ATP-binding <br /> Introduction                                                           proteins (Bork et al., 1992). Rtg2p acts proximal budding yeast, Saccharomyces cerevisiae, mito-                  sensor mitochondrial dysfunction, gluta- <br /> chondrial dysfunction, respiratory ciency                mate levels, regulate subcellular localization blocks tricarboxylic acid (TCA) cycle, results               Rtg1p Rtg3p (Sekito et al., 2000). <br /> increased expression subset nuclear genes involved                Recent studies shown RTG pathway metabolism, small molecule transport pathways,                      responds quality nitrogen source peroxisomal biogenesis stress responses (Liao et al.,              medium target <span id='am-6' about='obo:IMR_0002818' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C07909" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0002818" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="rapamycin" datatype="xsd:string"></span>rapamycin</span> (TOR) kinase <br /> 1991; Chelstowska Butow, 1995; Traven et al., 2001;                pathway (Komeili et al., 2000; Shamji et al., 2000). Epstein et al., 2001b). pathway inter-organelle                particular, cells grown poor nitrogen source, <br /> communication, called retrograde regulation, accommo-                  TOR kinase pathway inhibited dates cells mitochondrial defect. example,                rapamycin, Rtg1p Rtg3p translocate reactions catalyzed target genes                     cytoplasm nucleus Rtg2p-dependent manner retrograde pathway serve, directly indirectly,               activate target gene expression, similarly  <br /> &#195;&#163; European Molecular Biology Organization                                                                                     7209 <br />  Liu et al. <br />  <br />  <br /> Rtg2p-dependent nuclear accumulation Rtg1p Rtg3p cells dysfunctional mitochondria (Sekito <br /> et al., 2000).  studies pathways <span id='am-11' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span>carbohydrate</span> nitrogen metabolism linked RTG-dependent gene expression. <br />    identify additional regulatory genes RTG <br /> pathway, searched mutations bypass requirement RTG2 CIT2 expression. search led identi&#194;&#174;cation LST8, essential gene encoding seven WD40-repeat protein implicated amino acid permease transport Golgi cell <br /> surface (Roberg et al., 1997a).   mutations LST8 bypass requirement Rtg2p Rtg1p/Rtg3p-dependent gene expression. conclude Lst8p functions negative regulator       Fig. 1. RTB2/LST8 negative regulator CIT2 expression.   RTB2 <br /> RTG-dependent gene expression, acting          regulates CIT2 expression upstream RTG3, downstream                                                                RTG2. Wild-type (PSY142), rtg2D (PSY142-rtg2), rtg2D lst8-5 <br /> sites, upstream Rtg2p level external          (RBY426) rtg2D rtb2-5 rtg3D (RBY427) cells grown YPD <br /> glutamate sensing Rtg2p              medium mid-log phase collected b-galactosidase activity <br /> Rtg1p&#194;&#177;Rtg3p.                                                   analysis determine transcriptional activation integrated <br />                                                                CIT2-lacZ reporter gene. b-galactosidase assays cell extracts <br />                                                                carried triplicate described Materials methods. <br /> Results                                                        (B) RTB2 LST8. Wild-type PSY142 cells rtg2D lst8-5 <br />                                                                transformed pRS416-LST8, lst8-5, rtg2D lst8-1 lst8-1 <br /> Isolation RTG2 bypass mutants                               derivatives grown YNBcasD medium plus <span id='am-77' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> (omitted identify new genes retrograde pathway,            cells transformed pRS416-LST8) mid-log phase, CIT2-lacZ <br /> carried genetic screen identify mutants    reporter gene expression determined b-galactosidase activities. <br /> bypass requirement Rtg2p CIT2 expression. <br /> Using rtg2D strain CIT2-lacZ reporter gene           suf&#194;&#174;cient abolish reporter gene expression integrated URA3 locus, cells mutagenized       rtg2D rtb2 isolates, shown rtg2D rtb2-5 cells <br /> screened colonies appeared blue X-gal plates.      (Figure 1B). verify RTB2 LST8  screen yielded 14 candidates selected        carried complementation tests examining CIT2- <br /> potential rtg2D bypass mutants, designated rtb (Rtg        lacZ expression diploids obtained cross bypass). Northern blot analysis revealed rtb   rtg2D rtb2-5 strain rtg2D lst8D::LEU2 strain <br /> isolates high levels expression             carrying wild-type copy LST8 centromeric <br /> endogenous CIT2 mRNA (data shown), indicating              plasmid. resulting diploids cured plasmid rtg2D bypass phenotype peculiar        tested CIT2-lacZ expression. CIT2- <br /> reporter gene. Standard genetic tests showed rtg2D    lacZ expression high diploids (data bypass phenotype isolate recessive      shown), indicating RTB2 LST8. Finally, single gene mutation.  isolates         described section, identi&#194;&#174;ed point mutations <br /> placed complementation groups, rtb1 rtb2.         LST8 coding region rtb2 mutant independent rtb2 mutants obtained,           alleles identi&#194;&#174;ed screen.  refer properties described  details      RTB2 LST8. lst8 mutant alleles rtb1 complementation group described          examined singly, shown lst8-5 (Figure 1B),                                                          CIT2-lacZ expression nearly 10-fold higher    Preliminary characterization rtb2 isolates    expression wild-type cells. Altogether, data (rtb2-5) showed block CIT2-lacZ expression         LST8 negative regulator RTG gene function. rtg2D mutation completely reversed rtb2-5 mutation (Figure 1A). Similar results obtained rtb2 mutants. exclude possibility    lst8-1 mutant allele does bypass rtg2D rtb2 mutations activated alternate pathway CIT2     mutation <br /> expression, introduced rtg3 null mutation rtg2D     LST8 shown previously encode essential seven <br /> rtb2-5 mutant cells. CIT2-lacZ expression abolished        WD-repeat protein functions delivery Gap1p, triple mutant (Figure 1A), suggesting rtg2D    possibly amino acid permeases, cell <br /> bypass does involve recruitment alternative         surface (Roberg et al., 1997a). study, mutant <br /> transcriptional activators.                                    allele LST8 (lst8-1) identi&#194;&#174;ed synthetic lethal <br />                                                                sec13-1 mutation. determine RTB2 LST8                                                   lst8-1 mutation similar effects RTG-dependent identify RTB2 gene, rtg2D rtb2-5 mutant cells      gene activity, lst8-1 allele transplaced transformed centromere-based wild-type yeast            LST8 locus wild-type rtg2D cells resultant <br /> genomic library screened transformants            transformants assayed CIT2-lacZ expression <br /> restored CIT2-lacZ expression low level observed        (Figure 1B). Surprisingly, unlike lst8 mutants wild-type rtg2D cells. Complementing              obtained, lst8-1 allele unable bypass rtg2D <br /> plasmids obtained, subsequent subcloning              mutation, itself resulted ~4-fold <br /> yielded single gene, LST8, necessary           activation CIT2-lacZ expression. &#194;&#174;ndings raise <br />  <br /> 7210 <br />                                                                                                 Lst8p negatively regulates RTG gene function <br />  <br />  <br />  <br />  <br />                                                                              Fig. 3. glutamate auxotrophy rtg2D mutation rescued                                                                              lst8-5 lst8-1 mutation. Wild-type PSY142 cells,                                                                              rtg2D, lst8-5, rtg2D lst8-5, rtg2D rtb2-5 rtg3D, lst8-1 rtg2D lst8-1 <br />                                                                              derivatives streaked YNBD medium supplemented                                                                              0.02% glutamate indicated, uracil, leucine lysine,                                                                              incubated 30&#194;&#176;C 2&#194;&#177;3 days. <br />  <br />                                                                              conserved seven WD repeats Lst8p,                                                                              WD repeats G protein b-subunit (Sondek et al., <br />                                                                              1996). designated different lst8 mutant alleles <br />                                                                              obtained study lst8-2 lst8-5, collect- <br />                                                                              ively lst8-(2&#194;&#177;5). Structural modeling Lst8p suggests <br />                                                                              lst8-(2&#194;&#177;5) mutations clustered localized                                                                              surface protein, lst8-1                                                                              internal position opposite protein <br />                                                                              (Figure 2B). <br /> Fig. 2. Analysis lst8 mutations.   Alignment seven WD            lst8-5, lst8-1, rescues glutamate <br /> repeats Lst8p. conserved residues seven WD <br /> repeats highlighted gray, consensus sequence formed        auxotrophy rtg2D mutant <br /> shown alignment; j designates hydrophobic residue          hallmark rtg mutant cells glutamate <br /> y designates hydrophilic residue; T designates serine threonine.      auxotrophs mitochondrial function reduced <br /> Based alignment Lst8p G protein b1 subunit            compromised (Liao Butow, 1993; Liu Butow, <br /> (ter Haar et al., 1998), WD repeat likely consist b-strands, indicated arrows  B, C D. row gives                                                                              1999). Given striking difference ability canonical numbering residues WD repeat. red              lst8-(2&#194;&#177;5) mutant alleles versus lst8-1 allele type indicates mutations identi&#194;&#174;ed screen: lst8-2, Gly146          suppress block CIT2 expression rtg2D cells, glutamate; lst8-3, Gly138 aspartate; lst8-4 lst8-6, Gly181         asked difference lst8 <br /> aspartate; lst8-5, Gly171 aspartate. yellow type indicates        mutants ability reverse glutamate auxo- <br /> lst8-1 mutation identi&#194;&#174;ed synthetic lethal sec13 (Roberg et al., <br /> 1997a). (B) dimensional model Lst8p. model created          trophy rtg2D strain grown glucose medium. Swiss-Model (Peitsch, 1996), automated comparative protein             shown lst8-5 (Figure 3), lst8-(2&#194;&#177;5) mutant <br /> modeling server, based homology Lst8p G protein           alleles restored glutamate prototrophy rtg2D cells, <br /> b1 subunit. image prepared using programs Gl_render (http://         depended presence RTG3. contrast, <br /> www.hhmi.swmed.edu/external/Doc/G1_render.html), BOBSCRIPT <br /> (Esnouf, 1997), MOLSCRIPT (Kraulis, 1991) Raster3D (Meritt, <br />                                                                              lst8-1 mutation unable restore glutamate <br /> 1997). positions b-strands D WD repeat 6              prototrophy rtg2D cells, consistent inability <br /> indicated. N C designate N- C-terminal ends Lst8p.            lst8 allele bypass loss CIT2 expression Locations lst8-(2&#194;&#177;5) mutations indicated red dots,     rtg2D cells. cases CIT2 reporter gene lst8-1 mutation yellow dot.                                       expression high, Rtg3p localized nucleus <br />                                                                              (data shown). &#194;&#174;ndings underscore                                                                              difference lst8-(2&#194;&#177;5) lst8-1 mutants, possibility Lst8p separable functions, notion                 suggest site negative regulation RTG-depend- developed later report.                                 ent gene expression Lst8p Rtg2p    Sequencing &#194;&#174;ve lst8 mutants isolated showed                     Rtg1p&#194;&#177;Rtg3p transcription complex. contained single missense mutation <span id='am-53' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-54' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> <br /> residue, WD repeat 4 remainder WD                          lst8-5 lst8-1 mutant cells largely <br /> repeat 5 (Figure 2A). mutants contained                       insensitive glutamate repression <br /> identical mutation position 181 WD repeat 5. Based                     investigate regulation RTG-dependent design rtg2D bypass screen  Materials                      gene expression Lst8p, examined effects methods), mutants likely arisen                       addition glutamate growth medium CIT2-lacZ <br /> independently. mutated <span id='am-65' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-66' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Gly" datatype="xsd:string"></span>glycine</span> residues                      expression LST8, lst8-5 lst8-1 cells. LST8 cells <br /> localized tight turns protein. WD repeat 5                  grown minimal medium, CIT2-lacZ expression divergent seven, lacking conserved                        repressed &gt;100-fold addition 0.2% glutamate His2, Ser/Thr20 Asp24, containing residue                     medium (Figure 4A). lst8-5 mutant cells,  <br /> insertion linker region leading WD repeat 6.                    CIT2-lacZ expression largely insensitive glutamate determined mutation lst8-1 allele identi&#194;&#174;ed                  repression, decreasing ~2-fold addition Roberg et al. (1997a),                         0.2% glutamate. loss sensitivity glutamate <br /> leucine serine change position 300, residue                          repression result bulk impairment  <br />                                                                                                                                               7211 <br />  Liu et al. <br />  <br />  <br />                                                                           glutamate uptake, raise possibility glutamate <br />                                                                           repression involve external glutamate sensing. <br />  <br />                                                                           Ssy1p&#194;&#177;Ptr3p signal transduction cascade <br />                                                                           mediates glutamate repression CIT2 expression <br />                                                                           candidate external glutamate sensor Ssy1p,                                                                           plasma membrane protein predicted structure <br />                                                                           similar amino acid permeases (Didion et al., <br />                                                                           1998; Iraqui et al., 1999; Klasson et al., 1999). Previous <br />                                                                           studies shown amino acid induction AGP1, <br />                                                                           encodes broad-speci&#194;&#174;city amino acid permease,                                                                           genes encoding amino acid <br />                                                                           permeases, abolished ssy1 mutant cells (Didion <br />                                                                           et al., 1998; Iraqui et al., 1999). amino acid <br />                                                                           inductive effects shown require Ptr3p,                                                                           peripheral plasma membrane protein, proposed <br />                                                                           Ptr3p interacts Ssy1p signal <br />                                                                           transduction cascade external amino acid sensing.                                                                           addition, induction probably requires Shr3p, protein <br />                                                                           function facilitate exit Ssy1p                                                                           endoplasmic reticulum (Klasson et al., 1999)   <br />                                                                           targeting plasma membrane. <br /> Fig. 4. insensitivity glutamate repression CIT2-lacZ reporter      investigate Ssy1p mediate glutamate <br /> gene expression lst8-5 mutant cells result defect   repression RTG target gene expression, &#194;&#174;rst <br /> glutamate uptake.   Lst8-5 lst8-1 mutants largely insensitive glutamate repression CIT2-lacZ reporter gene expression. Wild- <br />                                                                           determined glutamate repress type (PSY142), lst8-1 lst8-5 mutant strains integrated copy   activity CIT2-lacZ reporter gene ssy1D, ptr3D CIT2-lacZ reporter gene grown YNB5%D medium            shr3D mutant cells. Figure 5A shows strong 0.2% glutamate. cell extracts prepared             repression CIT2-lacZ activity glutamate wild-type <br /> b-galactosidase activities determined described Materials      cells essentially blocked mutant strains. methods. (B) Glutamate uptake wild-type lst8-5 mutant <br /> cells. Wild-type (PSY142) lst8-5 mutant strains grown         lack signi glutamate repression CIT2-lacZ <br /> YNBD medium glutamate uptake assayed described              expression ssy1D, ptr3D shr3D mutant cells Materials methods. Glutamate present &#194;&#174;nal concentration      similar lst8 mutants.  unlike 0.01%. (C) Glutamate uptake wild-type gap1D dip5D             lst8-(2&#194;&#177;5) mutations, loss CIT2 reporter gene <br /> mutant strains. Wild-type (S288C) gap1D dip5D derivative strain grown YNBD medium, glutamate uptake assay                                                                           expression rtg2D cells reversed ssy1D, <br /> carried described (B). (D) CIT2-lacZ reporter gene expression   ptr3D shr3D mutations (Figure 5A), double repressed glutamate gap1D dip5D mutant cells. Wild-type    mutants rtg2D ssy1D, rtg2D ptr3D rtg2D shr3D <br /> (S288C) gap1D dip5D mutant derivative strain grown          remained glutamate auxotrophs (Figure 5B). data <br /> YNB5%D medium glutamate indicated. cell          suggest  like lst8-1 mutation, ssy1D, ptr3D extracts prepared b-galactosidase activities determined described Materials methods.                                    shr3D mutations affect RTG gene function upstream                                                                           RTG2. <br />                                                                              measured rate glutamate uptake ssy1D <br />                                                                           cells, glutamate uptake inhibited                                                                           ~50% (Figure 6A). data agreement glutamate uptake, lst8-5 mutant cells showed                  previous report showing modest decrease glutamate kinetics glutamate uptake wild-type cells                      uptake ssy1 mutant cells (Klasson et al., 1999). (Figure 4B). Interestingly, lst8-1 mutation did              provide additional evidence glutamate repression bypass requirement Rtg2p, greatly reduced                  CIT2-lacZ expression function glutamate glutamate repression CIT2-lacZ expression.                         uptake, overexpressed Dip5p Gap1p ssy1D cells <br />   asked glutamate repress                     determined rate glutamate uptake cells. <br /> CIT2-lacZ reporter gene expression mutant cells               Overexpression amino acid permeases ssy1D <br /> defective glutamate transport. yeast,                 cells restored rates glutamate uptake ~80% major pathways glutamate uptake, involving                    wild-type rate (Figure 6A), CIT2-lacZ expression <br /> general amino acid permease encoded GAP1                       remained insensitive glutamate repression (Figure 6B). speci&#194;&#174;c glutamate/aspartate permease encoded                   level glutamate repression CIT2-lacZ DIP5 gene (Regenberg et al., 1998). Accordingly,                   expression wild-type cells overexpressing Dip5p examined effects inactivation genes                    Gap1p somewhat observed wild-type <br /> glutamate repression CIT2-lacZ expression. gap1D                  control cells, possibly high level expression <br /> dip5D double mutant, rate glutamate uptake                     plasma membrane proteins competes Ssy1p <br /> inhibited ~85% (Figure 4C).  CIT2-lacZ                    targeting, clear ssy1D cells overexpressing <br /> expression double mutant cells sensitive               amino acid permeases, glutamate completely fails glutamate repression wild-type cells (Figure 4D).                   repress CIT2-lacZ reporter gene expression. Altogether, data suggest glutamate repression RTG-                      data support view sensing dependent gene expression strictly dependent                    Ssy1p&#194;&#177;Ptr3p signal transduction pathway,  <br /> 7212 <br />                                                                                            Lst8p negatively regulates RTG gene function <br />  <br />  <br />  <br />  <br /> Fig. 5. Effects shr3D, ssy1D ptr3D mutations glutamate <br /> repression, rtg2D suppression glutamate auxotrophy.   CIT2-lacZ <br /> expression. Transformants wild-type (PLY126), shr3D (PLY151- <br /> ura3), ssy1D (HKY20), ptr3D (HKY31), rtg2D (PLY126-rtg2), shr3D <br /> rtg2D (PLY151-rtg2), ssy1D rtg2D (HKY20-rtg2) ptr3D rtg2D <br /> (HKY31-rtg2) strains containing CIT2-lacZ reporter gene centromere-based plasmid pCIT2-lacZ grown mid-log phase media indicated &#194;&#174;gure, b-galactosidase activities    Fig. 6. loss glutamate repression CIT2-lacZ reporter gene <br /> determined extracts described Materials methods. Strains    expression ssy1D cells uncoupled glutamate uptake. <br /> rtg2D (PLY126-rtg2), shr3D rtg2D (PLY151-rtg2), ssy1D rtg2D                Glutamate uptake ssy1D cells overexpressing Dip5 Gap1. <br /> (HKY20-rtg2) ptr3D rtg2D (HKY31-rtg2) tested            ssy1D (HKY20) cells transformed 2m control plasmid <br /> CIT2-lacZ expression grown YNB5%D medium             vector), 2m-TEF-DIP5 2m-GAP1 plasmids, resultant glutamate auxotrophs. (B) shr3D, ssy1D ptr3D mutations       transformants, including wild-type cells transformed 2m rescue glutamate auxotrophy rtg2D cells. Cells streaked   control plasmid, analyzed glutamate uptake described YNBD medium 0.02% glutamate incubated          Figure 4B. (B) Glutamate repression CIT2-lacZ reporter gene <br /> 30&#194;&#176;C 2&#194;&#177;3 days.                                                       expression ssy1D cells overexpressing Dip5p Gap1p.                                                                          various transformants described   transformed                                                                          pCIT2-lacZ reporter gene construct transformants                                                                          grown YNBD medium 0.2% glutamate. cell <br /> glutamate uptake, major factor responsible                    extracts prepared b-galactosidase activities determined <br /> glutamate repression RTG-dependent gene expression.                   described Materials methods. <br />    determined amino acid speci&#194;&#174;city repression CIT2 reporter gene expression wild-type <br />                                                                          addition glutamate medium. number studies <br /> cells. CIT2-lacZ expression repressed strongly cells grown presence glutamate glutamine,                   shown different quality nitrogen sources expression reduced &lt;3-fold                        rise different patterns gene expression (Cardenas <br /> presence amino acids (Figure 7). ssy1D                    et al., 1999; Hardwick et al., 1999; Komeili et al., 2000). <br /> mutant, repression CIT2-lacZ expression glutamate                  Assessment quality different nitrogen sources glutamine eliminated  instances,                 complicated fact variation modest repression amino acids                      strains response utilization different <br /> eliminated. data suggest speci&#194;&#174;c non-                     nitrogen sources, including ammonia glutamate. <br /> speci&#194;&#174;c repression RTG target gene expression                      Accordingly, experiments <br /> effected Ssy1p.                                                  described effects glutamate RTG- <br />                                                                          dependent gene expression, glutamate added                                                                          medium containing ammonia, avoid <br /> Genome-wide transcription analysis genes                              comparing different, sole nitrogen sources. <br /> regulated glutamate                                                      Strain S288C grown mid-log phase YNB5%D glutamate act repressor                     medium supplementation 0.2% <br /> activator gene expression (Iraqui et al., 1999),            glutamate. mRNA isolated, labeled cDNAs <br /> effects operating Ssy1p&#194;&#177;Ptr3p signal transduc-               prepared reverse transcription presence tion pathway, wished obtain global view                  Cy3- Cy5-labeled nucleotides. labeled cDNAs <br /> glutamate-regulated genes. purpose, carried                  mixed hybridized duplicate dye microarray analysis ~6200 yeast genes determine                gurations yeast genome arrays. Applying relative mRNA abundance affected                     stringent cut average 2.5-fold difference  <br />                                                                                                                                         7213 <br />  Liu et al. <br />  <br />  <br /> individual genes, 45 genes expression affected glutamate (Table ;  31 genes repressed 14 induced. repressed <br /> genes, 12 known subject nitrogen catabolite <br /> repression (NCR), including MEP2, encoding ammonia <br /> permease, GAP1, encoding general amino acid <br /> permease. repression NCR genes glutamate suggests ammonia preferred <br /> nitrogen source strain S288C, repression NCR genes glutamate likely combined effect glutamate ammonia generate <br /> glutamine, preferred nitrogen source. expected, RTG target genes, ACO1, CIT2 IDH2, glutamate-repressible genes. <br />    14 genes induced glutamate, encode               Fig. 7. Effects different amino acids CIT2-lacZ reporter gene <br /> amino acid permeases, &#194;&#174;ve  BAP2, BAP3, AGP1,            expression wild-type ssy1D cells. Wild-type (PLY126) GNP1 VAP1/TAT1, belong Cluster amino acid           ssy1D (HKY20) strains transformed centromere-based plasmid <br /> permease family shows broad substrate speci&#194;&#174;city          pCIT2-lacZ grown mid-log phase YNB5%D medium <br /> (Regenberg et al., 1999). Induction genes            supplemented 0.1% indicated amino acids. cell <br />                                                                  extracts prepared b-galactosidase activities determined <br /> external amino acids Ssy1p dependent (Iraqui et al.,          described Materials methods. <br /> 1999), underscoring conclusion glutamate effect induction repression gene expression. <br />  <br />                                                                  glutamate signaling, binding glutamate Ssy1p <br /> Discussion <br />                                                                  send signal (inhibitory) RTG pathway, identify additional components RTG pathway,            proposed effects amino acids sought mutants bypass dependence            Ssy1p/Ptr3p-dependent activation gene expression <br /> Rtg2p, require Rtg1p&#194;&#177;Rtg3p transcriptional         amino acid inductive pathway (Didion et al., 1998; <br /> activators target gene expression. current work          Klasson et al., 1999; Iraqui et al., 1999). Despite loss <br /> shows certain mutations seven WD40-repeat            glutamate repression RTG pathway ssy1D, <br /> protein, Lst8p, bypass requirement Rtg2p,        ptr3D, shr3D lst8-1 cells, establishes wild-type cells, Lst8p negative         mutations bypass requirement Rtg2p CIT2 <br /> regulator RTG-dependent gene expression. dia-              expression lst8(2&#194;&#177;5) mutations. <br /> grammed model Figure 8, suggest Lst8p             considering effect glutamate RTG-depend- regulate RTG-dependent gene expression sites,         ent gene expression, important distinguish downstream Rtg2p upstream            sensing extra- intracellular glutamate. cells <br /> level external glutamate repression RTG                compromised mitochondrial function, intracellular <br /> pathway.                                                         glutamate supplies maintained RTG regulation    lst8-1 lst8-(2&#194;&#177;5) mutant cells,           expression TCA cycle genes responsible expression CIT2-lacZ reporter gene constitutive       synthesis ketoglutarate (Liu Butow, 1999), insensitive repression glutamate, simple           increased expression genes encoding enzymes way Lst8p regulate RTG pathway              function anaplerotic pathways, small molecule transport <br /> controlling cells&apos; ability glutamate.    peroxisomal activities converge supply acetyl- <br /> previous study demonstrated general amino acid          <span id='am-29' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span>CoA</span>, oxaloacetate citrate TCA cycle, permease, Gap1p, failed targeted plasma             increased expression genes enable cells utilize <br /> membrane lst8-1 mutant cells (Roberg et al., 1997a).          poor nitrogen sources (Epstein et al., 2001b). contrast, results suggest,  glutamate repression       glutamate available medium, RTG- <br /> RTG-dependent gene expression likely occur            dependent gene expression regulated glutamate sensing involving Ssy1p&#194;&#177;Ptr3p              SSY1&#194;&#177;PTR3 signal transduction pathway. <br /> signal transduction pathway processes                RTG pathway insensitive dependent glutamate uptake.  inactivation        glutamate repression lst8-(2&#194;&#177;5) mutant cells, quite <br /> Ssy1p&#194;&#177;Ptr3p pathway eliminated glutamate repression           possible Lst8p regulatory site(s) downstream RTG-dependent gene expression, glutamate uptake           Rtg2p direct connection glutamate <br /> ssy1D cells inhibited ~50%; result     regulation pathway. WD40-repeat motif agreement previous study (Klasson et al., 1999).       identi&#194;&#174;ed originally b-subunit heterotrimeric G <br /> Secondly, glutamate repression RTG pathway            proteins (Fong et al., 1986) subsequently largely unaffected inactivation general           wide spectrum regulatory proteins, (Gap1) glutamate/aspartate-speci&#194;&#174;c (Dip5p) amino             functions mediating protein&#194;&#177;protein interactions. acid permeases. Finally, overexpression Gap1p             Lst8p component complex tethers <br /> Dip5p ssy1D cells restored rate glutamate uptake       Rtg1p&#194;&#177;Rtg3p cytoplasm. model, loss ~80% wild-type cells, CIT2 expression,      glutamate repression RTG-dependent gene expression ssy1D cells  insensitive glutamate repres-       lst8-(2&#194;&#177;5) mutant cells consequence sion. data suggest  Ptr3p-coupled             constitutive localization Rtg1p&#194;&#177;Rtg3p complex  <br /> 7214 <br />                                                                                                  Lst8p negatively regulates RTG gene function <br />  <br />  <br /> Table  Genome-wide effects glutamate gene expression <br /> Gene expression            ORF                 Gene             Gene product/function                                                     Fold change <br />  <br /> Repressed <br /> NCR target genes <br />                            YNL142W             MEP2             ammonia permease low capacity high af&#194;&#174;nity                         14.3 <br />                            YKR039W             GAP1             general amino acid permease                                               14.0 <br />                            YKR034W             DAL80            negative regulator multiple nitrogen catabolic genes                   12.4 <br />                            YLR158C             ASP3C            L-asparaginase II, extracellular                                           9.0 <br />                            YIR032C             DAL3             ureidoglycolate hydrolase                                                  7.7 <br />                            YLR155C             ASP3A            L-asparaginase II, extracellular                                           7.5 <br />                            YLR157C             ASP3B            L-asparaginase II, extracellular                                           7.4 <br />                            YLR160C             ASP3D            L-asparaginase II, extracellular                                           5.8 <br />                            YJR152W             DAL5             allantoate permease                                                        5.1 <br />                            YIR031C             DAL7             malate synthase 2                                                          4.5 <br />                            YLR142W             PUT1             proline oxidase                                                            3.3 <br />                            YOR348C             PUT4             putative proline-speci&#194;&#174;c permease                                          2.7 <br /> RTG target genes <br />                            YLR304C             ACO1             aconitase, mitochondrial                                                   6.5 <br />                            YCR005C             CIT2             non-mitochondrial citrate synthase                                         3.3 <br />                            YOR136W             IDH2             NAD+-dependent isocitrate dehydrogenase                                    3.1                            YKR033C                              protein unknown function                                               11.4 <br />                            YLR348C             DIC1             mitochondrial dicarboxylate transport protein                              5.4 <br />                            YLR349W                              protein unknown function                                                4.4 <br />                            YBR182C             SMP1             probable transcription factor                                              4.3 <br />                            YBR294W             SUL1             probable sulfate transport protein                                         4.2 <br />                            YPL058C             PDR12            multidrug resistance transporter                                           3.6 <br />                            YNL208W                              protein unknown function                                                3.2 <br />                            YOR135C                              protein unknown function                                                2.9 <br />                            YJL172W             CPS1             carboxypeptidase yscS                                                      2.9 <br />                            YGL045W                              protein unknown function                                                2.7 <br />                            YML131W                              putative NAD-dependent oxidoreductase                                      2.7 <br />                            YLR161W                              protein unknown function                                                2.7 <br />                            YER124C                              protein unknown function                                                2.6 <br />                            YLL053C                              aquaporin water channel protein                                            2.6 <br />                            YIL121W                              member MFS-MDR family multidrug permeases                            2.5 <br />                            YGL028C             SCW11            soluble cell wall protein                                                  2.5 <br /> Induced <br /> Amino acid <br /> permeases <br />                            YDR046C             BAP3             valine transporter                                                        22.9 <br />                            YDR068              BAP2             probable amino acid permease leucine, isoleucine valine <br />                            YDR                                                                                                            20.0 <br />                            YCL025C             AGP1             amino acid permease                                                       12.1 <br />                            YBR069C             VAP1             probable amino acid transport protein                                     12.0 <br />                            YDR508C             GNP1             high-af&#194;&#174;nity glutamine permease                                            6.1 <br />                            YGR055W             MUP1             high-af&#194;&#174;nity methionine permease                                           4.3 <br /> Proteases <br />                            YIR039C             YPS6             GPI-anchored aspartic protease                                             6.6 <br />                            YOL154W                              protein similarity zinc metalloproteinases                         2.8                            YDR509W                              protein unknown function                                                4.8 <br />                            YNL160W             YGP1             glycoprotein synthesized response nutrient limitation                3.8 <br />                            YMR011W             HXT2             high-af&#194;&#174;nity hexose transporter-2                                          3.6 <br />                            YBR092C             PHO3             acid phosphatase, constitutive                                             3.2 <br />                            YOL158C             ENB1             enterobactin transporter                                                   3.0 <br />                            YNL065W                              member MFS-MDR family multidrug permeases                            2.7 <br />  independent cultures strain S288C grown mid-log phase YNB5%D 0.2% glutamate. isolation poly + <br /> RNA  labeled cDNAs Cy3 Cy5 mixed oppositely labeled cDNA treated untreated samples. indicated fold changes gene expression determined described Epstein et al. (2001a) using cut 2.5-fold change average expression ratios independent RNA preparations, repeated reversal Cy3 Cy5 labeling. <br />  <br />  <br />  nucleus. alternative model Lst8p bind                       molecular basis proposed dual regulate activity kinase phosphatase,                 sites action Lst8p diagrammed model effecting change phosphorylation state Rtg3p;                     Figure 8? suggest domains  turn, dictate Rtg3p&apos;s nuclear entry exit.                   Lst8p, functioning assembly activity remains determined internal glutamate                         Ssy1p&#194;&#177;Ptr3p signal transduction complex sensed RTG pathway Rtg2p functions                             negative regulation RTG-dependent gene expression transduce glutamate signals Rtg1p&#194;&#177;Rtg3p                             downstream Rtg2p. addition genetic evidence <br /> transcription complex.                                                        summarized, clearly ning sites Lst8p action <br />  <br />                                                                                                                                                  7215 <br />  Liu et al. <br />  <br />  <br />                                                                           Lst8p localized cellular <br />                                                                           compartment. speci&#194;&#174;c intracellular location <br />                                                                           Lst8p established, preliminary cell fraction <br />                                                                           studies suggest Lst8p localized                                                                           different cellular compartments (unpublished observa- <br />                                                                           tions). studies required clarify                                                                           issues. <br />                                                                               Ssy1p shown involved amino acid <br />                                                                           inductive expression amino acid permeases <br />                                                                           (Didion et al., 1998; Iraqui et al., 1999), <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> <br />                                                                           transporter, Ptr2p (Didion et al., 1998),  shown <br />                                                                            glutamate repression. &#194;&#174;ndings prompted                                                                           carry genome-wide transcriptional analysis                                                                           effects glutamate gene expression. Microarray <br />                                                                           experiments revealed major pathways                                                                           affected glutamate: NCR; RTG pathway;                                                                           amino acid inductive pathway. Interestingly, Ssy1p <br />                                                                           appears involved pathways <br />                                                                           (Didion et al., 1998; Iraqui et al., 1999; Klasson et al., <br /> Fig. 8. Model dual regulation RTG gene functions Lst8p.         1999). set genes expression affected cell surface, Ssy1p binds glutamate glutamine sends inhibitory signal Rtg2p Ptr3p. revealed     glutamate microarray analysis amino acid- <br /> lst8-1 mutation, Lst8p affect targeting assembly          inducible genes, BAP2, BAP3, GNP1, VAP1/TAT1 Ssy1p&#194;&#177;Ptr3p  signal transduction function suggested      AGP1. transcription shown dependent Roberg et al. (1997a), act upstream Rtg2p. Lst8p        Ssy1p (Iraqui et al., 1999).  compared proposed function downstream Rtg2p, revealed lst8-(2&#194;&#177;5) mutations. Intracellular glutamate hypothesized                                                                           hydrophobic amino acids, glutamate glutamine sensed Ptr3p (Klasson et al., 1999), additional component      strong repressors RTG gene function, weak Ssy1p signaling pathway. Repressive signals generated           inducers AGP1 expression. consistently observed <br /> glutamate lock Lst8p `RTG inhibition&apos; state     addition 0.01% glutamate growth medium <br /> Lst8p available function Ssy1p&#194;&#177;Ptr3 signaling;   resulted 4- 5-fold greater induction AGP1 effect attenuating glutamate repression RTG gene <br /> functions.                                                                compared addition 0.2% glutamate (data                                                                           shown), supporting Lst8p switching model <br />                                                                           discussed  amino acid induction PTR2 upstream downstream Rtg2p, striking                transcription involves transcriptional repressor Cup9p lst8-(2&#194;&#177;5) mutations clustered localize                  N-end rule pathway component, Ubr1p (Goodman <br /> region Lst8p predicted                  et al., 1997; Byrd et al., 1998). cross- <br /> protein opposite lst8-1 mutation (Figure 2B).              regulation pathways? prelim- <br /> WD40-repeat proteins usually adopt b-propeller struc-                   inary experiments (data shown) ubr1, ure2 gln3 <br /> ture, use blades interact              deletion mutations signi effect proteins affecting blades (Goodman                      expression RTG target gene, CIT2,  conversely, <br /> et al., 1997; ter Haar et al., 1998). propose                 rtg mutations affect expression NCR <br /> distinctive phenotypes types lst8 mutations                 target gene, GLN1. &#194;&#174;ndings suggest defects interaction protein                     little direct cross-regulation pathways <br /> different factors. dual functionality Lst8p                    downstream Ssy1p&#194;&#177;Ptr3p. <br /> facilitate glutamate homeostasis participating                        Recent work shown TOR (target negative feedback loop regulating glutamate levels.                rapamycin) kinase signaling pathway, involved presence high concentrations external glutamate,                nutrient sensing, affects RTG-dependent gene <br /> signals generated Ssy1p `lock&apos; Lst8p form                   expression manner retrograde pathway, functions sequester Rtg1p&#194;&#177;Rtg3p complex                    affecting subcellular localization Rtg1p cytoplasm.  Lst8p available                Rtg3p Rtg2p-dependent manner (Komeili et al., <br /> Ssy1p signal transduction  plausibly                2000). Inhibition TOR kinase pathway treating <br /> function targeting post-translational modi&#194;&#174;cation                cells rapamycin growing poor components  model                   nitrogen source promotes dissociation Ure2p explain observation glutamate blocks targeting               cytoplasmic, phosphorylated form positive regulator Gap1p plasma membrane (Roberg et al.,                           NCR pathway, Gln3p, resulting Gln3&apos;s depho- <br /> 1997a,b), suggests means attenuating                    sphorylation nuclear accumulation (Beck Hall, <br /> glutamate repression concentration glutamate                 1999; Cardenas et al., 1999; Hardwick et al., 1999). medium decreases. Consistent `switching&apos;                    Dephosphorylation Gln3p believed controlled <br /> model, observed rtg mutant cells grown                 type 2A-related phosphatase, Sit4p, rich medium, ~10-fold induction AGP1                       inhibitor, Tap42p, effector protein TOR pathway <br /> expression (data shown). &#194;&#174;nding indicate                   (Beck Hall, 1999). sit4 deletion mutation,  Lst8p released site(s) downstream                does affect CIT2 expression (unpublished data), <br /> Rtg2p protein available                    suggesting different TOR effectors serve SSY1&#194;&#177;PTR3 pathway. Altogether, &#194;&#174;ndings imply                   regulate NCR RTG pathways. <br />  <br /> 7216 <br />                                                                                               Lst8p negatively regulates RTG gene function <br />  <br />  <br /> Table II. Saccharomyces cerevisiae strains used <br /> Strain                   Genotype                                                                                              Source <br />  <br /> PSY142                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ                                                                R.Butow <br /> PSY142-rtg2              MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3                                                    R.Butow <br /> RBY346                   MATa ura3-52 leu2::LEU2-RTG2 lys2 ura3::CIT2-lacZ rtgD::ura3 lst8-5                                   study <br /> RBY426                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3 lst8-5                                             study <br /> RBY427                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3 lst8-5 rtg3D::LEU2                                 study <br /> RBY254                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ lst8-1                                                         study <br /> RBY264                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ lst8-1 rtg2D::kanMX4                                           study <br /> RBY418                   MATa ura3 leu2 lys2                                                                                   study <br /> RBY428                   MATa ura3/ura3 leu2/leu2 lys2/lys2 LST8/lst8D::LEU2 [pLST8]                                         study <br /> RBY419                   MATa ura3 leu2 lys2 rtg2D::kanMX4                                                                     study <br /> RBY420                   MATa ura3 leu2 lys2 lst8D::LEU2 pLST8                                                                 study <br /> RBY421                   MATa ura3 leu2 lys2 lst8D::LEU2 rtg2D::kanMX4 pLST8                                                   study <br /> RBY422                   MATa ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 RTG2/rtg2D::ura3 LST8/lst8-5                          study <br /> RBY423                   MATa ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 rtg2D::ura3/rtg2D::kanMX4 LST8/lst8-5                 study <br /> RBY424                   MATa ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 rtg2D::ura3/rtg2D::kanMX4 lst8D::LEU2/lst8-5          study <br /> ECY408                   MATa ura3 lst8-1                                                                                      C.Kaiser <br /> S288C                    MATa ura3-52 <br /> M4276                    MATa ura3 gap1D dip5D::kanMX                                                                          M.Kielland-Brandt <br /> PLY126                   MATa ura3-52 lys2D201                                                                                 P.Ljungdahl <br /> PLY151-ura3              MATa ura3-52 lys2D201 ade2 shr3D1::ura3                                                               P.Ljungdahl <br /> HKY20                    MATa ura3-52 lys2D201ssy1D13                                                                          P.Ljungdahl <br /> HKY31                    MATa ura3-52 lys2D201 ptr3D15                                                                         P.Ljungdahl <br /> PLY126-rtg2              MATa ura3-52 lys2D201 rtg2::kanMX4                                                                    study <br /> PLY151-rtg2              MATa ura3-52 lys2D201 alde2 shr3D1::ura3 rtg2::kanMX4                                                 study <br /> HKY20-rtg2               MATa ura3-52 lys2D201 ssy1D13 rtg2::kanMX4                                                            study <br /> HKY31-rtg2               MATa ura3-52 lys2D201 ptr3D15 rtg2::kanMX4                                                            study <br />  <br />  <br />  <br /> Materials methods                                                       AAAGCGGGAAGAAGTCA-3&#194;&#162; used amplify 2.1 kbp LST8 <br />                                                                             sequence. ECY408 genomic DNA used amplify lst8-1 mutant <br /> Strains                                                                     allele. ampli&#194;&#174;ed DNA digested SpeI SalI ligated <br /> Strains used study listed Table II.                          Yip352 pRs416 form, respectively, Yip352-lst8-1 pRS416- <br />                                                                             lst8-1. Similarly, LST8 ampli&#194;&#174;ed PSY142 strain Growth media, growth conditions                                             isogenic rtb2 mutant strains construct pRS416-LST8, pRS416-lst8- <br /> Yeast strains grown 30&#194;&#176;C YPD (1% yeast extract, 2%               2&#194;&#177;6, respectively. pCIT2-lacZ described previously (Liao et al., <br /> bactopeptone 2% dextrose), YNBcasD (0.67% yeast nitrogen base,          1991; Liu Butow, 1999). obtain plasmid 2m-LYS2, pRS426 <br /> 1% casamino acids 2% dextrose) minimal YNB medium (0.67%             plasmid digested EcoRV StuI remove URA3 <br /> yeast nitrogen base) supplemented 2 5% glucose (YNBD             sequence replaced 5.2 kbp LYS2 sequence SmaI- YNB5%D, respectively), Na glutamate  indicated        PstI-digested YDp-K (Berben et al., 1991). construct plasmid 2m- text &#194;&#174;gures), supplemented required nutrients.         TEF-DIP5, primer pair 5&#194;&#162;-gtcaACTAGTCTCTAAGTAATGAA- <br />                                                                             GATGCC-3&#194;&#162; 5&#194;&#162;-gtcaGTCGACGTGATACCTGTACACTATGG-3&#194;&#162; <br /> Strain constructions                                                        used amplify DIP5 open reading frame 413 bp 3&#194;&#162;- <br /> Standard genetic manipulations carried described Rose        untranslated region. resultant PCR product digested SpeI <br /> et al. (1990). RBY426 isolated rtg2 bypass screen described    SalI ligated pRS426-TEF plasmid (Mumberg et al.,  RBY426 transformed rtg3D::LEU2 disruption cassette        1995). URA3 sequence pRS426-TEF-DIP5 replaced form RBY427. RBY254 lst8-1 mutant obtained          LYS2, similarly construction 2m-LYS2. construct 2m-GAP1 <br /> step gene replacement. RBY418 derived PSY142 lacking           plasmid, 3.5 kbp GAP1 sequence obtained SpeI- SalI- <br /> integrated CIT2-lacZ reporter gene mating-type switching PSY142       digested pPL247 (Ljungdahl et al., 1992) ligated pRS426. cells transformed plasmid pGAL-HO (Herskowitz Jensen,          resultant pRS426-GAP1 digested NcoI StuI remove 1991). RBY418 transformed rtg2::kanMX4 disruption               URA3 sequence replaced LYS2 sequence <br /> cassette form RBY419. RBY420 meiotic segregant obtained             described  <br /> following sporulation diploid RBY428 transformed pRS416- <br /> LST8. RBY421 constructed RBY420 transforming cells              Rtg2 bypass mutant screen pUC18-rtg2::kanMX4 digested PstI. Matings carried        Ethyl methanesulfonate mutagenesis carried essentially RBY426 RBY418, RBY419 RBY421 form RBY422,                 described previously (Liao Butow, 1993). Cells plated RBY423 RBY424, respectively. RTG2 disruptions strains             YNBD plus X-gal amino acid supplements. Fourteen rtb mutants <br /> PLY126-rtg2, PLY151-rtg2, HKY20-rtg2 HKY31-rtg2 carried            obtained grouped complementation groups, transformation rtg2::kanMX4 disruption cassette    rtb1 rtb2, described text. rtb mutants arose respective parental strains.                                                frequency ~1 106 cells. <br />  <br /> Plasmid constructs                                                          Isolation RTB2 (LST8) gene <br /> Construction rtg3D::LEU2 disruption cassette described previ-        URA3 centromeric CIT2-lacZ reporter gene transformed ously (Liu Butow, 1999). construct rtg2D::kanMX4 cassette          RBY285 (rtg2D rtb2-5) derivative lacking integrated version derivative, primer set 5&#194;&#162;-AGGCACACTCTTCTTCAC-3&#194;&#162; 5&#194;&#162;-                 reporter gene. isolate RTB2 gene, wild-type genomic DNA <br /> gactaagcttTGAACAACAAGAAGGTGCCC-3&#194;&#162; used amplify                       library constructed centromeric vector p366 carrying LEU2 <br /> RTG2. digestion BglII HindIII, RTG2 DNA ligated          marker screened transformants RBY285 abolish BamHI HindIII sites pRS416 form pRS416-RTG2.           CIT2-lacZ expression. 8000 LEU2+ transformants 643 bp NdeI fragment pRS416-RTG2 cloned NdeI site        obtained overlapping restriction maps. Subcloning pUC18 form pUC18-RTG2. kanMX4 disruption cassette module           centromeric plasmid pRS415 retransformation revealed inserted SpeI site form pUC18-rtg2::kanMX4. primer set      fragment containing LST8 gene necessary suf&#194;&#174;cient 5&#194;&#162;-gactactagtACCACCAGGACCAAAGCTTG-3&#194;&#162; 5&#194;&#162;-gactgtcgacCA-                   complement rtb2-5 mutant phenotype. <br />  <br />  <br />                                                                                                                                             7217 <br />  Liu et al. <br />  <br /> b-galactosidase assays                                                           Doolittle,R.F. Simon,M  (1986) Repetitive segmental structure <br /> b-galactosidase assays carried previously described (Liu         transducin b subunit: homology CDC4 gene Butow, 1999). plasmid&#194;&#177;strain combination, assays                   identi&#194;&#174;cation related mRNAs. Proc. Natl Acad. Sci. USA, 83, <br /> conducted triplicate independent experiments carried             2162&#194;&#177;2166. times.                                                           Forsburg,S.L. Guarente,L. (1989) Communication                                                                                  mitochondria nucleus regulation cytochrome genes Assays amino acid uptake                                                     yeast Saccharomyces cerevisiae. Annu. Rev. Cell Biol., 5, <br /> Amino acid uptake assayed described Roberg et al. (1997a)              153&#194;&#177;180. glutamate uptake, equal volumes 0.2 mM 14C-labeled         Gangloff,S.P., Marguet,D. Lauquin,G.J.M. (1990) Molecular cloning <br /> glutamate 59.1 mM unlabeled glutamate mixed diluted 50-             yeast mitochondrial aconitase gene (Aco1) evidence fold assay mixture. 2 min intervals, samples 100 ml      synergistic regulation expression glucose plus glutamate. Mol. diluted 3 ml ice-cold 5.9 mM glutamate quench                Cell. Biol., 10, 3551&#194;&#177;3561. <br /> reaction, cells collected &#194;&#174;ltration washed 33 5 ml       Goodman,O.B., Krupnick,J.G., Gurevich,V.V., Benovic,J.L. ice-cold water. L-[U-14C]<span id='am-41' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-42' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutamic acid</span> (250 mCi/mmol) obtained            Keen,J.H. (1997) Arrestin/clathrin interaction. Localization Amersham Pharmacia (Buckinghamshire, UK).                                   arrestin binding locus clathrin terminal domain. J. Biol. Chem., <br />                                                                                  272, 15017&#194;&#177;15022. <br /> Microarray analysis                                                           Hardwick,J.S.,      Kuruvilla,F.G.,     Tong,J.K.,     Shamji F.     fresh culture S288C diluted 250 ml YNB5%D               Schreiber,S.L. (1999) Rapamycin-modulated transcription nes 0.2% glutamate. overnight growth OD600 0.8, cells          subset nutrient-sensitive signaling pathways directly <br /> collected mRNA samples prepared described (Epstein et al.,           controlled Tor proteins. Proc. Natl Acad. Sci. USA, 96, <br /> 2001b). Cy3- Cy5-labeled cDNAs prepared hybridized             14866&#194;&#177;14870. <br /> microarray 6219 yeast genes. Replica experiments carried          Herskowitz  Jensen,R.E. (1991) Putting HO gene work: <br /> using independent liquid cultures opposite guration          practical uses mating-type switching. Methods Enzymol., 194, <br /> Cy3 Cy5. data analyzed described (Epstein et al.,                132&#194;&#177;146. <br /> 2001a,b) using cut value average 2.5-fold change         Iraqui , Vissers,S., Bernard,F., DeCraene,J.O., Boles,E., Urrestarazu  <br /> replica hybridizations.                                                          Andre,B. (1999) Amino acid signaling Saccharomyces <br />                                                                                  cerevisiae: permease-like sensor external amino acids F- <br />                                                                                  box protein Grr1p required transcriptional induction Acknowledgements                                                                 AGP1 gene, encodes broad-speci&#194;&#174;city amino acid permease. <br />                                                                                  Mol. Cell. Biol., 19, 989&#194;&#177;1001. thank, G.Fink, C.Kaiser, P.O.Ljungdahl M.C.Kielland-Brandt         Jia,Y., Rothermel,B., Thornton,J. Butow,R  (1997) basic <br /> yeast strains plasmids, Tsan Xiao technical support, members         helix&#194;&#177;loop&#194;&#177;helix zipper transcription complex functions Butow laboratory helpful discussions. work supported         signaling pathway mitochondria nucleus. Mol. Cell. grants GM22525 CA77811 National Institutes Health,            Biol., 17, 1110&#194;&#177;1117. 0642 Robert Welch Foundation.                                Klasson,H., Fink,G.R. Ljungdahl,P.O. (1999) Ssy1p Ptr3p                                                                                  plasma membrane components yeast senses <br />                                                                                  extracellular amino acids. Mol. Cell. Biol., 19, 5405&#194;&#177;5416. <br /> References                                                                    Komeili , Wedaman,K.P., O&apos;Shea,E.K. Powers,T. (2000) <br />                                                                                  Mechanism metabolic control: target rapamycin signaling links <br /> Beck,T. Hall,M.N. (1999) TOR signaling pathway controls                  nitrogen quality activity Rtg1 Rtg3 transcription <br />   nuclear localization nutrient-regulated transcription factors. Nature,      factors. J. Cell Biol., 151, 863&#194;&#177;878. <br />   402, 689&#194;&#177;692.                                                               Kraulis,P.J. (1991) MOLSCRIPT: program produce detailed <br /> Berben,G., Dumont,J., Gilliquet,V., Bolle,P  Hilger,F. (1991)          schematic plots proein structures. J. Appl. Crystallogr., 24, <br />   YD <br /> </body></html>