<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Systems Biology 3; Article number 104; doi:10.1038/msb4100144 <br /> Citation: Molecular Systems Biology 3:104 <br /> <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; 2007 EMBO Nature Publishing Group rights reserved 1744-4292/07 <br /> www.molecularsystemsbiology.com <br />  <br /> REPORT <br />  <br /> Pathway redundancy protein essentiality revealed Saccharomyces cerevisiae interaction networks <br /> Igor Ulitsky Ron Shamir* <br />  <br /> School Science, Tel Aviv University, Tel Aviv, Israel <br /> * Corresponding author. School Science, Tel Aviv University, Tel Aviv 69978, Israel. Tel.: &#195;&#190; 972 3 6405383; Fax: &#195;&#190; 972 3 6405384; <br /> E-mail: rshamir@tau.ac  <br /> Received 8.1.07; accepted 11.2.07 <br />  <br />  <br />  <br />  <br />                biological interpretation genetic interactions major challenge. Recently, Kelley                Ideker proposed method analyze genetic physical networks, explains                known genetic interactions linking different pathways physical network.                 extend method devise novel analytic tools interpreting genetic interactions physical <br />                context. Applying tools large-scale Saccharomyces cerevisiae data set, analysis <br />                reveals 140 pathway models explain 3765 genetic interactions, roughly doubling                previously explained. Model genes tend short mRNA half-lives                phosphorylation sites, suggesting stringent regulation linked pathway redundancy. <br />                identify &#226;&#8364;&#732;pivot&#226;&#8364;&#8482; proteins physical interactions pathways                models, pivots tend essential highly conserved. analysis models                pivots sheds light organization cellular machinery roles individual <br />                proteins. <br />                Molecular Systems Biology 17 April 2007; doi:10.1038/msb4100144 <br />                Subject Categories: metabolic regulatory networks; computational methods <br />                Keywords: essential genes; genetic interactions; pathway analysis; protein interactions; S. cerevisiae <br />  <br />  <br />  <br />  <br /> Introduction                                                                     interact physically (Tong et al, 2001, 2004), protein <br />                                                                                  PIs likely GIs (Ozier et al, <br /> Gene knockout studies shown B18%                               2003). <br /> Saccharomyces cerevisiae genes essential growth                        modular nature cellular organization rich medium (Giaever et al, 2002). Consequently, buffering                    widely recognized (Hartman et al, 2001). methods genetic level believed abundant eukaryotes                       developed detecting functional modules <br /> (Hartman et al, 2001). better understand role                          PI networks. modules, termed pathways, <br /> nonessential genes, large-scale studies performed                        represent physically interacting proteins involved carrying <br /> double knockouts (Pan et al, 2004; Tong et al, 2004)                         particular function. Depending detection <br /> identified events synthetic lethality, mutant                    method, pathways represent molecular complexes <br /> carrying deletions nonessential genes lethal,                      (Bader Hogue, 2003) signaling cascades (Rives synthetic sickness, mutant shows weaker                              Galitski, 2003). Kelley Ideker (2005) defined pathway <br /> phenotype. use term genetic interaction (GI)                    group proteins densely interconnected interaction genetic perturbations affecting                PI network, studied frequency GIs phenotype, lethal sick. graph genes                   pathways. systematic analysis large-scale <br /> nodes edges corresponding GIs called GI network.                   GI PI data, concluded pathway <br />    Recent technologies enable systematic mapping                       explanations GIs B3.5 times abundant protein&#226;&#8364;&#8220;protein (Ito et al, 2001) protein&#226;&#8364;&#8220;DNA (Lee et al,                    pathway explanations, concluded GIs 2002) interactions (physical interactions (PIs)), yielding large                 bridge redundant processes. arguments PI networks. networks larger, need                            prevalence pathway GIs given Ye et al <br /> computational tools dissecting mounting.                         (2005), postulated genes pathway integrated analysis PI GI networks compelling                        expected share common GI partners, used similarity challenge, carry important complementary                             GI patterns successful function prediction. Similar results <br /> biological signals. Initial studies shown proteins                  established recently DNA damage (Pan region GI network slightly likely                    et al, 2006). <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                     Molecular Systems Biology 2007 1 <br />  Pathway redundancy essential proteins Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Results discussion                                                                connected subnetwork PI network  connected <br />                                                                                       pathway, described Materials methods). example <br /> Assembly GI PI networks                                                        sparse pathways presented Figure 2A. assembled GI network (Figure 1A) taking                              buffering mechanism DNA repair BioGRID database version 2.0.19 (Stark et al, 2006) 13 632                            homologous recombination response oxidative <br /> synthetic lethality synthetic fitness interactions                            stress partially recovered using dense <br /> S. cerevisiae, covering 2682 genes. focusing genes                         pathway <span id='am-8' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>  shown). define pathway interactions, obtained GI network 1869                           model (BPM) Kelley Ideker (2005), using new <br /> genes 12 850 interactions. PI network, consisting                          notion pathway (Figure 1C). scoring models protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220;DNA interactions multiple                            model detection algorithm defined Materials sources (Supplementary information 1), contained 68172                                methods. comparison models using <br /> interactions covering 6184 proteins.                                                  dense pathways interaction data (Supplementary <br />                                                                                       information  shows construct  <br />                                                                                       pathway explanations GIs (3765 versus 3117), Pathway definitions pathway models                                        maintaining significant functional content models.                                                                                       models allow direct interpretation specific starting point computational framework Kelley <br />                                                                                       buffering cases congruence methods (Supplementary Ideker (2005) detection pathway inter- <br />                                                                                       information B). <br /> pretations GIs. Kelley Ideker define &#226;&#8364;&#732;pathway&#226;&#8364;&#8482; densely connected set proteins PI network,  pathway model&#226;&#8364;&#8482; disjoint pair pathways densely interconnected GI network (Figure 1B).                             comprehensive model map S. cerevisiae <br /> Models defined probabilistically using                            BPM finding approach generated 140 models greedy algorithm. requirement high PI density                         provided pathway explanations 3765 GIs, 2.7- <br /> appropriate complexes, known biological                                fold increase 1377 interactions explained Kelley <br /> pathways (e.g. linear signaling cascades) induce dense                         Ideker (2005). mainly incorporation subnetworks physical network. chose                            GIs (12 850 instead 4125) lesser extent employ alternative definition, pathway simply                        using PIs (68172 versus 27 604), use  <br />  <br />  <br />  <br /> Figure 1 Study outline methodology.   Overview analysis methods reported results. (B) BPM constructed dense pathways PI <br /> network. (C) BPM constructed connected pathways PI network. (D, E) Examples biological explanations pivot proteins. (D), pivots <br /> correspond shared members linear pathways, signal flow indicated arrow directions, (E), correspond shared complex <br /> members. Note pivots (E) redundant, densely connected pathways physical interactions genetic interaction  <br />  <br /> 2 Molecular Systems Biology 2007                                                                                            &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                         Pathway redundancy essential proteins <br />                                                                                                                                          Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Figure 2 Model examples. Rectangles represent genes, pathways shown different colors. blue ovals pivot nodes. Essentials genes drawn thicker border. main text discussion BPMs  E).  , clarity, 52 genetic interactions pathways shown. <br />  <br />  <br />  <br />  create &#226;&#8364;&#732;scaffolds&#226;&#8364;&#8482; pathways scoring models.                       Po0.001; Supplementary Figure S2). correlation gene content models available supplementary                      pathways BPM higher expected (0.112 versus <br /> archive. description interactive visualization                     0.0378 expected, Po0.001; Supplementary Figure S2). models available http://acgt.cs.tau.ac bpm. <br />                                                                                     Identification pivot proteins <br /> Functional enrichment models                                                     established pathways models frequently <br />                                                                                     represent coherent functional units, functional units utilized TANGO algorithm (Shamir et al, 2005) order <br />                                                                                     cell share components (Krause et al, test functional enrichment models                                                                                     2004), tested possibility computationally detecting <br /> categories (Ashburner et al, 2000). total 71.4 69.3% <br />                                                                                     shared components PI network. end, BPMs enriched functional <br />                                                                                     model, sought proteins densely connected <br /> category &#226;&#8364;&#732;biological process&#226;&#8364;&#8482; &#226;&#8364;&#732;cellular <br />                                                                                     pathways (Figure 1D E, Materials compartment&#226;&#8364;&#8482; ontologies, respectively (Supplementary Table <br />                                                                                     methods), called pivot proteins model. <br /> S1). complexes annotated SGD slim (Cherry et al, <br />                                                                                     Altogether, identified 124 distinct pivots 40 models. 1998), 46.3% enriched BPM. Despite                                                                                     average, 1.09 pivots model, pivot <br /> low coverage GI network, coverage complexes                                                                                     appeared 1.22 models. <br /> comparable achievable direct analysis PI <br />                                                                                        systematically analyzed representation proteins <br /> network (Supplementary information C). <br />                                                                                     known distinct processes                                                                                     group pivots. end, identified proteins participat- <br />                                                                                     ing complexes pathways  Materials Phenotypic coherence                                                                                     methods), used curated set multicomplexed <br /> pathways                                                                            genes (Krause et al, 2004). summarized Table  extent pathways BPM                           pivots enriched sets. example function? answer question, used phenotypic                           overlap BPM 96 (Figure 2B). model containing contribution non-essential genes S. cerevisiae,                            pathways parts SWR1 Ino80 complexes involved measured fitness deletion mutants diverse                              chromatin remodeling, identified pivot proteins Arp4, <br /> treatments (Brown et al, 2006). BPM pathway                         Rvb1 Rvb2, proteins known significantly coherent phenotypic response pattern                   participate SWR1 Ino80 complexes (Shen random connected groups size physical                       et al, 2000; Krogan et al, 2003). BPM 97 (Figure 2C), Sus1, <br /> network (Pearson correlation 0.384 versus average 0.0382,                  shown nuclear pore <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                          Molecular Systems Biology 2007 3 <br />  Pathway redundancy essential proteins Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Table Multiple roles pivot proteins <br />  <br />                                                  proteins                      Pivots                     Expected                        Significance <br />  complexes                                          206                              21                          4.35                        P&#194;&#188;1.27  10 9 <br /> KEGG pathways                                         390                              11                          8.24                        P&#194;&#188;0.200 <br /> Filtered KEGG pathways                                 71                               6                          1.50                        P&#194;&#188;3.68  10 3 <br /> Known complex overlaps                                 39                               8                          0.55                        P&#194;&#188;7.49  10 9 <br />  enrichment proteins known participate multiple physical pathways set pivot proteins. complexes taken SGD &#226;&#8364;&#732;macromolecular <br /> complex slim&#226;&#8364;&#8482; ontology. filtered KEGG pathways KEGG pathways 50% genes formed connected component physical network. <br /> Known complex overlaps taken Krause et al (2004). Significance evaluated using hypergeometric distribution. <br />  <br />  SAGA complex (Rodriguez-Navarro et al, 2004)                          partially redundant (McCleland et al, 2003), identified pivot model representing GIs                     placed different pathways. Note pathways. pivots correspond known                              subunits Ndc80 complex possible GIs PIs <br /> complex pathway overlaps, frequently represent                            pairs, biologically correct partition general purpose genes cooperating multiple pathways.                         complex pathways obtained taking  example, BPM 87, general transcription factor Spt15                    sideration GIs model. example, Mad1 identified pivot model contains components                     physically connects Tid3/Spc25 subunit distinct transcription-related complexes RSC, SWR1                     genetically interacts Nuf2/Spc24 subunit. SAGA (Figure 2D).                                                                 additional external interactions cause biologically correct <br />                                                                                   partition score higher alternatives. <br />                                                                                      Mad1, highly conserved protein specific function Essentiality evolutionary retention pivot                                  spindle cell cycle checkpoint, additional  proteins                                                                          spindle checkpoint, cells arrested metaphase <br />                                                                                   chromosomes successfully attach microtubules. pathways form model partially <br />                                                                                   Tid3 Spc25, members Ndc80 complex, redundant function, pivots represent proteins <br />                                                                                   appear pathway Mad1, specifically linked active pathways, hypothesized                                                                                   spindle checkpoint organisms (summarized pivots frequently correspond essential genes.  72 <br />                                                                                   Bharadwaj et al, 2004).  recruitment Mad1 pivots essential, highly significant <br />                                                                                   shown dependent Spc25 Tid3 Xenopus fraction given total number essential genes                                                                                   human cells (McCleland et al, 2003; Bharadwaj et al, 2004) network (22.6 essentials expected, P&#194;&#188;1.42  10 23).                                                                                   spindle checkpoint shown defective spc25 observed general strong correlation degree                                                                                   mutants (Wigge Kilmartin, 2001). exactly does Mad1 <br /> essentiality (P&#194;&#188;5.87  10 85 using rank-sum test), pre- <br />                                                                                   attach kinetochore currently known. viously reported (Jeong et al, 2001), high prevalence                                                                                   Mad1 shows yeast hybrid interaction Spc25 essential genes pivots far                                                                                   S. cerevisiae (Newman et al, 2000) Tid3 human <br /> explained degrees (39.98 essentials expected, <br />                                                                                   cells (Martin-Lluesma et al, 2002), attempts demonstrate Po10 5). essential pivots tend closer <br />                                                                                   biochemical interaction Ndc80 Mad1 functions BPMs (Supplementary information D). <br />                                                                                   reported unsuccessful (Martin-Lluesma et al, 2002; McCleland <br /> Essentiality recently shown connected                                                                                   et al, 2003). <br /> evolutionary retention genes eukaryotes (Gustafson <br />                                                                                      model, Mad1 linked pivot Smc1, member et al, 2006). Using data Gustafson et al (2006)                                                                                   cohesin complex, required sister chromatid cohesion pivots significantly retained evolution <br />                                                                                   mitosis. Smc1 shown localize kinetochore <br /> (P&#194;&#188;9.79  10 9), controlling large fraction <br />                                                                                   meiosis interact Tid3 yeast human essential genes (P&#194;&#188;0.029). <br />                                                                                   cells (Zheng et al, 1999; Gregson et al, 2002). Furthermore, <br />                                                                                   Smc1 shown required proper assembly                                                                                   mitotic spindle human cells (Gregson et al, 2001), Example: spindle checkpoint model <br />                                                                                   exact function metaphase unknown. findings interesting example using models pivots                             suggest connection Mad1 kinetochore understanding cellular mechanisms BPM 66 (Figure 2E).                          general Ndc80 complex particular, mediated model represents buffering different compo-                          Smc1. Note use pivots provides additional <br /> nents kinetochore, complex bound centromeres                      clues BPM annotation understanding inter- chromosomes mitosis. pivots,                      pathway organization. <br /> model composed members known subcomplexes inner kinetochore: Ndc80, COMA MIND  Wulf <br /> et al, 2003). Specifically, pivots Mtw1 Dsn1 correspond distinct unit MIND complex, bridges                             Physiological properties models <br /> different kinetochore subcomplexes  Wulf et al, 2003;                          Recently, physiological properties PI network hubs <br /> Westermann et al, 2003). subunits Ndc80                            extensively analyzed (Batada et al, 2006). complex, Nuf2/Spc24 Tid3/Spc25, shown                           proteins models hubs, asked  <br /> 4 Molecular Systems Biology 2007                                                                                      &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                Pathway redundancy essential proteins <br />                                                                                                                                 Ulitsky R Shamir <br />  <br />  <br />  <br />  physiological properties differed                  ensemble networks degree sequence counting <br /> hubs.  focus analysis mRNA stability (Wang                 fraction contain interaction b. log- <br />                                                                              odds score et al, 2002) number putative phosphorylation sites <br /> (Obenauer et al, 2003), properties manifesting                                           P&#195;&#176;V1 ; V2 jMBPM &#195;&#382; <br />                                                                                 S&#195;&#176;V1 ; V2 &#195;&#382; &#194;&#188; log <br /> turnover regulation gene. Detailed analysis,                                          P&#195;&#176;V1 ; V2 jMnull &#195;&#382; <br />                                                                                                  Q <br /> covering additional properties, described Supplementary                                log &#195;&#176;a;b&#195;&#382;2V1  V2 bI&#195;&#176;a; b&#195;&#382; &#195;&#190; &#195;&#176;1   b&#195;&#382;&#195;&#176;1   I&#195;&#176;a; b&#195;&#382;&#195;&#382; <br /> information E Supplementary Table S2.                                                   &#194;&#188;Q <br />                                                                                                &#195;&#176;a;b&#195;&#382;2V1  V2 ra;b I&#195;&#176;a; b&#195;&#382; &#195;&#190; &#195;&#176;1   ra;b &#195;&#382;&#195;&#176;1   I&#195;&#176;a; b&#195;&#382;&#195;&#382; <br />    tests BPM genes behave significantly unlike genes. average mRNA half-life BPM genes                     main difference score score described                                                                              Kelley Ideker (2005) structure imposed BPMs 21.4, versus 26.2 genes (P&#194;&#188;9.97  10 10, rank-sum                PI network. score model density PIs test). average number phosphorylation sites 5.1,                  pathway, instead require connected GP. <br /> versus 4.0 genes (P&#194;&#188;3.13  10 9). parameters significantly correlated pathways constitute BPM (intraclass correlations 0.408 0.189,                Model finding algorithm <br /> Po10 4 P&#194;&#188;0.0184, respectively). finding                   model finding procedure described Kelley Ideker (2005) explained high degrees model genes (Supplementary                  greedy network search algorithm uses seeds single GIs. Table S2).                                                                   improve procedure initializing algorithm better seeds <br />                                                                              maximal bicliques GI network  biclique disjoint <br />    Note mRNA half-lives experimentally derived,                 pair node sets node set edges nodes phosphorylation sites computationally                     set). following procedure performed u, v <br /> ferred sequence. 3552 genes                   (u, v)AEP <br /> parameters available, correlated (r&#194;&#188; 0.0182 <br />                                                                              1. Identify B set nodes adjacent u v GG. Proceed <br /> P&#194;&#188;0.276).  parameters significantly corre-                      |B|Xkmin. <br /> lated genes BPMs (566 genes, r&#194;&#188; 0.132,                         2. Partition B connected components GP: B1, B2, y, Bl. <br /> P&#194;&#188;1.6  10 3). results remain highly significant                   3. Bi |Bi|Xkmin, identify set Ai nodes <br /> controlling key functions enriched model partici-                    adjacent nodes Bi GG. <br />                                                                              4. Ai, partition connected components induces pants (Supplementary Figure S4). findings suggest                    GP, A1i , A2i , y,               j                           j <br />                                                                                                     add (Bi, Ai) set seeds |Ai|Xkmin. <br /> genes BPMs experience stringent regulation. possible hypothesis pathways redundant                  algorithm produces maximal bicliques (V1, V2) GG,                                                                              Vi induces connected component GP size Xkmin. mechanisms available, tighter regulation allows cell               produced set seeds filtered overlaps. used kmin&#194;&#188;2 switch alternatives faster.                      tests. Owing relatively sparse nature interaction <br /> conclusion revalidated GIs covering wider                   networks, method efficient practice. <br /> functional range available.                                              optimization phase starts seed candidate model, <br />                                                                              greedily tries improve score current model    results indicate despite limitations today&#226;&#8364;&#8482;s GI <br />                                                                              addition, removal exchange nodes pathways PI networks, integrated analysis powerful                     keeping pathway connected PI network. order approach understanding organization yeast                     efficiently track connectivity requirement, use cellular  expect analysis provide                   notion articulation nodes. articulation node connected graph <br /> insights large picture genetic redundancy higher              node removal disconnects graph. Articulation nodes                                                                              efficiently detected depth search traversal eukaryotes                                                           graph, calculating &#226;&#8364;&#732;lowpoint&#226;&#8364;&#8482; values node (cf.  <br />                                                                              1979). algorithm maintains connectivity pathway                                                                              dynamically updating, pathway, set articulation nodes. <br /> Materials methods                                                        set used ban optimization moves disrupt connectivity. <br />                                                                              optimization, filtering step removes models overlap Scoring models                                                               450% pathways higher scoring models. additional <br />                                                                              model filtering step computes empiric P-value sampling 1000 build probabilistic score described Kelley Ideker <br />                                                                              random gene groups size, retains BPMs (2005) Sharan et al (2005) identify pathway explana- <br />                                                                              Po0.05  Supplementary information F). <br /> tions GIs finding BPMs GI PI networks. Let <br /> GG&#194;&#188;(V,  GI network let GP&#194;&#188;(V, EP) PI network. <br /> Note nodes V represent genes products, <br /> depending context. BPM pair disjoint sets V1, V2,      Identification pivot proteins   |V1|, |V2|X2, (b) Vi induces connected subgraph GP        Given model (V1, V2), seek nodes densely connected (c) unusually GIs V1 V2 (Figure 1C).          pathways physical network. Specifically, vAV, Vi pathway. Property (c) reflects assumption         denote Ni(v) nodes Vi adjacent v GP. v genetic buffering implies dense set GIs pathways.           pivot  1, 2, |Ni(v)|Xlmin |Ni(v)| significant, given    quantify property (c). derive log-odds score reflecting       degree v GP (PoPmax, using hypergeometric test). actual likelihood density GIs pathways           analysis described paper, used lmin&#194;&#188;2 Pmax&#194;&#188;0.05. unusually high. compare hypotheses: BPM                     addition, filter master regulator genes involved hypothesis, pair genes, V1 V2,          processes, protein folding chaperons, considered genetically interact high probability b, independently   pivots proteins degree o250 GP. <br /> gene pairs, likelihood model (V1, V2) P  b V1   V2)bI  b) &#195;&#190; (1 b)(1  b)),  b) equals 1 exists GI b equals 0; null <br /> hypothesis, pair   b) connected probability ra, b,           Statistical analysis <br /> representing chance observing interaction random, given       Correlation analysis performed using non-parametric Spear- degrees b GG. estimate ra, b generating random       man test, unless indicated. P-values reported  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                Molecular Systems Biology 2007 5 <br />  Pathway redundancy essential proteins Ulitsky R Shamir <br />  <br />  <br />  <br /> controlling specific gene class (e.g. essential genes) obtained   Gregson HC, Schmiesing JA, Kim JS, Kobayashi T, Zhou S, Yokomori K random sampling large number gene groups                (2001) potential role human cohesin mitotic spindle aster <br /> fraction genes class. P-values reported                    assembly. J Biol Chem 276: 47575&#226;&#8364;&#8220;47582 <br /> controlling degrees calculated binning         Gregson HC, Van Hooser AA, Ball Jr AR, Brinkley BR, Yokomori K <br /> genes 40 equal-size bins based degree, sampling <br />                                                                                  (2002) Localization human SMC1 protein kinetochores. <br /> genes bins, maintaining proportion genes                                                                                  Chromosome Res 10: 267&#226;&#8364;&#8220;277 bin. <br />                                                                              Gustafson  Snitkin ES, Parker SC, DeLisi C, Kasif S (2006)                                                                                  identification essential genes using targeted genome <br />                                                                                  sequencing comparative analysis. BMC Genomics 7: 265 <br /> Supplementary information                                                    Hartman JLt, Garvik B, Hartwell L (2001) Principles buffering                                                                                  genetic variation. Science 291: 1001&#226;&#8364;&#8220;1004 <br /> Supplementary information available Molecular Systems <br />                                                                              Ito T, Chiba T, Yoshida M (2001) Exploring protein interactome <br /> Biology website (www.nature.com/msb). <br />                                                                                  using comprehensive hybrid projects. Trends Biotechnol 19: <br />                                                                                  S23&#226;&#8364;&#8220;S27 <br />                                                                              Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality                                                                                  centrality protein networks. Nature 411: 41&#226;&#8364;&#8220;42 <br /> Acknowledgements                                                             Kelley R, Ideker T (2005) Systematic interpretation genetic thank members Shamir laboratory helpful                        interactions using protein networks. Nat Biotechnol 23: 561&#226;&#8364;&#8220;566 <br /> discussions. IU fellow Edmond J Safra Bioinformatics             Krause R, von Mering C, Bork P, Dandekar T (2004) Shared <br /> Program Tel-Aviv University. RS supported                  components protein complexes&#226;&#8364;&#8221;versatile building blocks Wolfson foundation, EMI-CD project funded              biochemical artefacts? BioEssays 26: 1333&#226;&#8364;&#8220;1343 <br /> European Commission FP6 Programme, thematic             Krogan NJ, Keogh MC, Datta N, Sawa C, Ryan OW, Ding H, Haw RA, <br /> area &#226;&#8364;&#732;Life Sciences, genomics biotechnology health&#226;&#8364;&#8482;, contract            Pootoolal J, Tong  Canadien V, Richards DP, Wu X, Emili  <br /> number LSHG-CT-2003-503269.                                                      Hughes TR, Buratowski S, Greenblatt JF (2003) Snf2 family <br />                                                                                  ATPase complex required recruitment histone H2A variant <br />                                                                                  Htz1. Mol Cell 12: 1565&#226;&#8364;&#8220;1576 <br />                                                                              Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph  Gerber GK, <br /> References                                                                       Hannett NM, Harbison CT, Thompson CM, Simon  Zeitlinger J, <br />                                                                                  Jennings  Murray HL, Gordon DB, Ren B, Wyrick JJ, Tagne JB, <br /> Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM,                 Volkert TL, Fraenkel E, Gifford DK, Young RA (2002) <br />    Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-         Transcriptional regulatory networks Saccharomyces cerevisiae. <br />    Tarver L, Kasarskis  Lewis S, Matese JC, Richardson JE, Ringwald            Science 298: 799&#226;&#8364;&#8220;804 <br />    M, Rubin GM, Sherlock G (2000) Gene ontology: tool                Martin-Lluesma S, Stucke VM, Nigg EA (2002) Role Hec1 spindle <br />    unification biology. Gene Ontology Consortium. Nat Genet               checkpoint signaling kinetochore recruitment Mad1/Mad2. <br />    25: 25&#226;&#8364;&#8220;29                                                                     Science 297: 2267&#226;&#8364;&#8220;2270 <br /> Bader GD, Hogue CW (2003) automated method finding                    McCleland ML, Gardner RD, Kallio MJ, Daum JR, Gorbsky GJ, Burke <br />    molecular complexes large protein interaction networks. BMC                DJ, Stukenberg PT (2003) highly conserved Ndc80 complex    Bioinformatics 4: 2                                                           required kinetochore assembly, chromosome congression, Batada NN, Hurst LD, Tyers M, Russell R (2006) Evolutionary                  spindle checkpoint activity. Genes Dev 17: 101&#226;&#8364;&#8220;114 <br />    physiological importance hub proteins. PLoS Comput Biol 2: e88         Newman JR, Wolf E, Kim PS (2000) computationally directed screen <br /> Bharadwaj R, Qi W, Yu H (2004) Identification novel                       identifying interacting coiled coils Saccharomyces cerevisiae. <br />    components human NDC80 kinetochore complex. J Biol                     Proc Natl Acad Sci USA 97: 13203&#226;&#8364;&#8220;13208 <br />    Chem 279: 13076&#226;&#8364;&#8220;13085                                                     Obenauer JC, Cantley LC, Yaffe MB (2003) Scansite 2.0: proteome-wide <br /> Brown JA, Sherlock G, Myers CL, Burrows NM, Deng C, Wu HI,                       prediction cell signaling interactions using short sequence <br />    McCann KE, Troyanskaya OG, Brown JM (2006) Global analysis                 motifs. Nucleic Acids Res 31: 3635&#226;&#8364;&#8220;3641 <br />    gene function yeast quantitative phenotypic profiling. Mol          Ozier O, <span id='am-9' about='obo:IMR_0001693' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="A group of compounds derived from ammonia by substituting organic radicals for the hydrogens. (From Grant &amp; Hackh&apos;s Chemical Dictionary, 5th ed)" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D000588" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001693" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amin" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:32952" datatype="xsd:string"></span>Amin</span> N, Ideker T (2003) Global architecture genetic <br />    Syst Biol 2:1                                                                 interactions protein network. Nat Biotechnol 21: 490&#226;&#8364;&#8220;491 <br /> Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, Hester ET, Jia Y,        Pan X, Ye P, Yuan DS, Wang X, Bader JS, Boeke JD (2006) DNA <br />    Juvik G, Roe T, Schroeder M, Weng S, Botstein D (1998) SGD:                   integrity network yeast Saccharomyces cerevisiae. Cell 124: <br />    Saccharomyces genome database. Nucleic Acids Res 26: 73&#226;&#8364;&#8220;79                    1069&#226;&#8364;&#8220;1081 Wulf P, McAinsh AD, Sorger PK (2003) Hierarchical assembly         Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, <br />    budding yeast kinetochore multiple subcomplexes. Genes Dev               Hieter P, Spencer F, Boeke JD (2004) robust toolkit functional <br />    17: 2902&#226;&#8364;&#8220;2921                                                                 profiling yeast genome. Mol Cell 16: 487&#226;&#8364;&#8220;496 S (1979) Graph Algorithms. Potomac, MD: Science                Rives AW, Galitski T (2003) Modular organization cellular networks. <br />    Press                                                                         Proc Natl Acad Sci USA 100: 1128&#226;&#8364;&#8220;1133 <br /> Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S,            Rodriguez-Navarro S, Fischer T, Luo MJ, Antunez O, Brettschneider S, <br />    Lucau-Danila  Anderson K, Andre B, Arkin AP, Astromoff  El-               Lechner J, Perez-Ortin JE, Reed R, Hurt E (2004) Sus1,    Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S,                      functional component SAGA histone acetylase complex    Curtiss M, Davis K, Deutschbauer  Entian KD, Flaherty P, Foury F,           nuclear pore-associated mRNA export machinery. Cell 116: <br />    Garfinkel DJ, Gerstein M, Gotte D, Guldener U, Hegemann JH,                   75&#226;&#8364;&#8220;86 <br />    Hempel S, Herman  Jaramillo DF, Kelly  Kelly SL, Kotter P,           Shamir R, Maron-Katz  Tanay  Linhart C, Steinfeld  Sharan R, <br />    LaBonte D, Lamb DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier             Shiloh Y, Elkon R (2005) EXPANDER integrative program suite <br />    M, Mao R, Menard P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-            microarray data analysis. BMC Bioinformatics 6: 232 <br />    Macdonald P, Scherens B, Schimmack G, Shafer B, Shoemaker DD,             Sharan R, Ideker T, Kelley B, Shamir R, Karp RM (2005) Identification <br />    Sookhai-Mahadeo S, Storms RK, Strathern JN, Valle G, Voet M,                  protein complexes comparative analysis yeast bacterial <br />    Volckaert G, Wang CY, Ward TR, Wilhelmy J, Winzeler EA, Yang Y,               protein interaction data. J Comput Biol 12: 835&#226;&#8364;&#8220;846 <br />    Yen G, Youngman E, Yu K, Bussey H, Boeke JD, Snyder M,                    Shen X, Mizuguchi G, Hamiche  Wu C (2000) chromatin <br />    Philippsen P, Davis RW, Johnston M (2002) Functional profiling             remodelling complex involved transcription DNA <br />    Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391                      processing. Nature 406: 541&#226;&#8364;&#8220;544 <br />  <br /> 6 Molecular Systems Biology 2007                                                                              &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                         Pathway redundancy essential proteins <br />                                                                                                                          Ulitsky R Shamir <br />  <br />  <br />  <br />  <br /> Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz  Tyers M    Wang Y, Liu CL, Storey JD, Tibshirani RJ, Herschlag D, Brown PO <br />    (2006) BioGRID: general repository interaction datasets.          (2002) Precision functional specificity mRNA decay. <br />    Nucleic Acids Res 34: D535&#226;&#8364;&#8220;D539                                         Proc Natl Acad Sci USA 99: 5860&#226;&#8364;&#8220;5865 <br /> Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N,             Westermann S, Cheeseman IM, Anderson S, Yates 3rd JR, Drubin DG, <br />    Robinson M, Raghibizadeh S, Hogue CW, Bussey H, Andrews B,              Barnes G (2003) Arch <br /> </body></html>