<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access understanding human disease genes comparing housekeeping genes genes <br /> Zhidong Tu&#226;&#8364;&#160;, Li Wang&#226;&#8364;&#160;, Min Xu, Xianghong Zhou, Ting Chen Fengzhu Sun* <br />  <br /> Address: Molecular Computational Biology Program, University Southern California, Los Angeles, California 90089, USA <br /> Email: Zhidong Tu - ztu@usc.edu; Li Wang - wang7@usc.edu; Min Xu - mxu@usc.edu; Xianghong Zhou - xjzhou@usc.edu; <br /> Ting Chen - tingchen@usc.edu; Fengzhu Sun* - fsun@usc.edu <br /> * Corresponding author &#226;&#8364;&#160;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 21 February 2006                                                   Received: 04 October 2005 <br />                                                                               Accepted: 21 February 2006 <br /> BMC Genomics 2006, 7:31    doi:10.1186/1471-2164-7-31 article available  http://www.biomedcentral.com/1471-2164/7/31 <br /> &#194;&#169; 2006 Tu et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: studies compared various features heritable disease genes                  called non-disease genes, yielded conflicting results. potential <br />                  problem studies non-disease genes contained large number essential genes <br />                  &#226;&#8364;&#8220; genes indispensable humans survive reproduce. functional disruption <br />                  essential gene fatal consequences, s reasonable regard essential genes                  extremely severe &quot;disease&quot; genes. perform comparative study features human <br />                  essential, disease, genes. <br />                  Results: absence set defined human essential genes, consider set 1,789 <br />                  ubiquitously expressed human genes (UEHGs), known housekeeping genes,                  approximation. demonstrate UEHGs likely contain large proportion                  essential genes. UEHGs, disease genes genes different                  evolutionary conservation rates, DNA coding lengths, gene functions,  findings <br />                  systematically confirm disease genes intermediate essentiality                  housekeeping genes greater human genes. <br />                  Conclusion: human genome contain thousands essential genes having features                  differ significantly disease genes. propose classify unique group                  comparisons disease genes non-disease genes. new way classification                  comparison enables clearer understanding disease genes. <br />  <br />  <br />  <br />  <br /> Background                                                                    Smith et al. [3] disease genes evolved Identification novel genes associated human dis-                      higher nonsynonymous/synonymous substitution rate <br /> eases critical tasks medical research.                   ratios (Ka/Ks) non-disease genes, Huang et al. [4] goal, various features compared                        significant differences. common prob- heritable disease genes non-disease genes [1-                     lem studies human essential genes 4]. findings consistent                          ignored simply grouped non-dis-  conflicting results showed  example,                      ease genes. Essential genes genes functions  <br />  <br />                                                                                                                                        Page 1 13 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                      http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> necessary organism survive reproduce.     genes. validate set UEHGs  disruption essential genes&apos; function cause fatal   tains large fraction essential genes. compar- <br /> consequences, regarded severe       ing groups genes, disease <br /> &quot;disease&quot; genes.  comparing disease genes       genes distinguished essential genes. <br /> mixture essential non-disease genes reduce     UEHGs really contain greater fraction essential <br /> clarity signals disease-related features     genes non-UEHGs  e. disease genes), lead erroneous findings.  beneficial    expect observe followings.  essential genes separate human essential genes non-disease       functionally extremely important, selective pres- <br /> genes comparisons                              sure higher non-essential <br />                                                                genes, UEHGs slower evolutionary rate <br /> Thousands genes identified essential           disease genes [12,13]. Second, genes multiple model organisms, Saccharomyces       Mendelian diseases caused deleterious amino <br /> cerevisiae, Caenorhabditis elegans, Mus musculus [5-7].    acid substitutions, study conservation amino certain human genome       acid level, expect different patterns UEHGs, <br /> contains hundreds thousands essential genes, s        disease genes.  UEHGs mapped <br /> impractical experimentally determine S. <span id='am-24' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-25' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="cer" datatype="xsd:string"></span>cer</span>-     species, homologous genes evisiae C. elegans. absence set defined    likely essential species species evolu- <br /> human essential genes poses challenge studying            tionarily close humans. Fourth, essential proteins urges alternative solutions.                      usually tend hub proteins (highly connected)                                                                protein-protein interaction network [14], UEHGs human genome extremely complex tissue               higher average physical interaction degree expression profile. genes expressed cer-      non-UEHGs. Fifth, functions UEHGs tain tissues specific times, consti-   fundamentally important. verify hypotheses, tutively ubiquitously expressed [8,9].      compile lists UEHGs, disease genes genes, presumed necessary fun-     human genes. collect various features com- <br /> damental cellular physiological processes referred     pare selected features gene classifi- housekeeping genes [9]. Housekeeping genes             cations. studied researchers interesting <br /> observations reported. example, Zhang        Comparison evolutionary features <br /> Li housekeeping genes evolved slowly           compare Ka, Ks ratio (Ka/Ks) based tissue-specific genes [10]. Eisenberg Levanon         way classification human genome. housekeeping genes compact            Ka, Ks Ka/Ks derived human-rat coding lengths, result higher selec-     human-mouse orthologous pairs. results obtained <br /> tive pressure[11]. Based unique properties       human-mouse orthologous pairs indicate ubiquitously expressed human genes (UEHGs),                 UEHGs smallest Ka, Ks Ka/Ks ratio believe suitable candidates essential        groups (P-values UEHGs vs. disease 3.4E-39, <br /> genes. hypothesis intuitive sounds        6.3E-15, 1.7E-38; P-values UEHGs vs. reasonable, efforts required collect support-   5.3E-64, 9.0E-27, 1.3E-57, respectively Ka, Ks ive evidence systematic level.                            Ka/Ks), disease genes lower evolutionary rates <br />                                                                genes (P-values 9.1E-5, 5.5E-4 2.6E-4 study, consider set 1,789 ubiquitously         Ka, Ks Ka/Ks, respectively) (Fig 1). various sta- <br /> expressed human genes (UEHGs) approximation          tistical measurements, UEHGs consistently stand essential genes. demonstrate UEHGs            slowest evolved gene group difference <br /> likely contain large proportion essential genes    UEHGs groups greater approximate human essential genes. perform-        difference disease genes genes <br /> ing way feature comparison UEHGs (presumed          (Table 1). results similar human-rat orthol- <br /> essential genes), disease genes, rest human         ogous pairs used calculate Ka Ks, P-val- <br /> genes (referred genes), dif-    ues slightly significant.  disease genes evolve <br /> ferent aspects evolutionary conserva-      slower rates genes significant differ- <br /> tion rates, DNA coding lengths, gene functions,            ences Ka, Ks Ka/Ks (P-values 0.008, 0.052                                                                0.026, respectively). UEHGs evolve slowest rates <br /> Results                                                        differences Ka, Ks Ka/Ks strongly signif- <br /> Instead dividing human genome disease vs.          icant (P-values UEHGs vs. disease 7.0E-32, 3.2E- <br /> non-disease genes, choose way classification,       13, 2.7E-28; P-values UEHGs vs. 1.1E-  UEHGs (presumed &quot;essential&quot;), disease,             49, 4.8E-23, 1.2E-44, respectively, Ka, Ks Ka/ <br />  <br />  <br />                                                                                                                    Page 2 13 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                         http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                            <br />  <br />  <br />  <br />  <br />                          (b) <br />  <br />  <br />  <br />  <br /> Figure 1 Ka, Ks Ka/Ks <br /> Distribution <br /> Distribution Ka, Ks Ka/Ks   cumulative density Ka, Ks Ka/Ks derived human-mouse orthologous <br /> pairs. Ka, number non-synonymous substitutions non-synonymous sites. Ks, number synonymous substitu- <br /> tions synonymous site, Ka/Ks ratio. groups human genes represented different colors number genes group listed right line symbols. (b) box plots drawn based data. category, central box depicts middle 50% data 25th 75th percentile, enclosed red hor- <br /> izontal line represents median value distribution. Extreme values indicated solid blue dots occur outside main bodies data. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 3 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                   http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 1: Comparison evolutionary rate groups genes <br />  <br />                                 UEHGs                                   Disease genes                               genes <br />  <br />                 Mean(SEM)       Median         Range       Mean(SEM)       Median         Range       Mean(SEM)        Median          Range <br />  <br />  Ka                0.053         0.033        0&#226;&#8364;&#8220;0.48         0.080          0.061         0&#226;&#8364;&#8220;0.70         0.092          0.066          0&#226;&#8364;&#8220;1.37 <br />                  (0.0015)                                   (0.0017)                                   (8.0E-4) <br />  Ks                0.58          0.57        0.076&#226;&#8364;&#8220;2.0        0.63          0.61         0.14&#226;&#8364;&#8220;2.4        0.66           0.62         0.041&#226;&#8364;&#8220;4.2 <br />                  (0.0047)                                   (0.0043)                                   (0.0022) <br />  Ka/Ks             0.088         0.059        0&#226;&#8364;&#8220;0.59          0.12          0.10          0&#226;&#8364;&#8220;0.70         0.13           0.11            0&#226;&#8364;&#8220;1.2 <br />                  (0.0023)                                   (0.0023)                                    (0.001) <br />  <br />  Ka, number non-synonymous substitutions non-synonymous sites. Ks, number synonymous substitutions synonymous site. Ka <br />  Ks values calculated based human-mouse orthologous pairs described main text. column group mean <br />  followed standard error mean (SEM). <br />  <br />  <br /> Ks) (Fig S2).  results different               conservation score specific amino acids based ings Smith [3] Huang [4], described                  large-scale multiple-sequence alignment 8 species  results consistent                formed recently UCSC research group [22]. think partly effect mixing essential                conservation scores derived state phylo- <br /> genes non-disease genes previous studies.                     Hidden Markov Model interpreted proba- reason different groups used different set                bilities base conserved hidden state. disease non-disease genes (Smith et al. studied 387                collect list 6,000 disease mutation sites <br /> disease 2,024 non disease genes, Huang et al. studied                 1,900 polymorphism (neutral) mutation sites <br /> 1,112 disease genes 10,000 non-disease                      SwissProt. compare conservation genes dataset covered &gt;1,700 disease genes                    types sites  <br /> &gt;12,000 non-disease genes, supplementary materials                    ground  background obtained considering detailed comparison). results indicate                     conservation score amino acids coding <br /> UEHGs strongest selection pressure. Disease                 regions). shown Fig 2   b), polymorphism muta- <br /> genes evolve intermediate rate slower                 tion sites significantly biased conserved genes, faster UEHGs. experiments                   sites disease mutation sites significantly biased <br /> sider larger gene sets significant number                   conserved sites (p-values &lt; 10-5 essential genes separated non-disease genes.                 cases). consistent findings Miller  results better reflect relationship             Kumar focused seven disease genes <br /> disease genes genes. slightly different                    [23]. compare UEHGs disease genes results calculated rat mouse likely                 results shown Fig 2(c). UEHGs conserved different evolutionary rates mouse rat                 disease genes conservation score disease divergence 16 million years ago [15].                  mutation sites greater UEHGs. We confirm positive correlation Ka                   don&apos;t information &quot;essential sites&quot;, unable <br /> Ks gene groups, observed previous                  directly compare &quot;essential sites&quot; disease sites. <br /> studies [16-20]. example, Ka Ks disease                    Instead, think possible mechanism distin- <br /> genes correlation coefficient 0.45 (P-value                  guishing essential genes disease genes essen- <br /> 1.9E-86 t-test). synonymous mutation                    tial genes contain larger fraction highly conserved apparent effect protein sequence, affect                 sites  underlying assumption highly  <br /> DNA structure, mRNA structure, biological processes                   served sites correspond functionally important loci). transcription RNA splicing [21]. small                     chance random mutation cause average Ks UEHGs indicates synonymous sites                   severe phenotype higher essential genes UEHGs stronger selection                 disease genes. select conservation score 0.9 groups, suggests functional impor-                  cut value define sites conservation scores <br /> tance UEHGs.                                                           highly conserved sites. cut chosen <br />                                                                           based distribution conservation scores Ka Ks summary statistics <span id='am-7' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-12' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sub-                disease mutation sites. Different cut values tested <br /> stitution rate gene, Mendelian diseases                 results similar. calculate fraction highly <br /> caused <span id='am-15' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> substitutions coding regions,                  conserved sites coding region distri- informative study pattern conserva-                bution fraction UEHGs, disease genes tion individual nucleotides amino acids. obtain                 genes Figure 2(d). s clear UEHGs contains  <br />  <br />                                                                                                                                   Page 4 13 <br />                                                                                                             (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                         80          Disease gene sites                                             80           Background  sites) <br />                                     Disease mutation sites                                                          Polymorphic mutation sites <br />  <br />  <br />  <br />                         60                                                                             60 <br />         Percentage <br />  <br />  <br />  <br />  <br />                                                                                           Percentage <br />                         40                                                                             40 <br />  <br />  <br />  <br />  <br />                         20                                                                             20 <br />  <br />  <br />  <br />  <br />                         0                                                                               0 <br />                              0.0   0.2              0.4           0.6   0.8   1.0                            0.0      0.2             0.4           0.6                0.8            1.0 <br />                                                   Conservation Score                                                                Conservation Score <br />                                                                                   (b) <br />  <br />  <br />  <br />  <br />                                                                                                                       UEHGs <br />                                          UEHGs sites                                               2.0            disease genes <br />                         80                                                                                            genes <br />                                          Disease mutation sites <br />                                          Disease gene sites <br />  <br />  <br />                                                                                                        1.5 <br />                         60 <br />            Percentage <br />  <br />  <br />  <br />  <br />                                                                                           Density <br />  <br />  <br />  <br />  <br />                         40                                                                             1.0 <br />  <br />  <br />  <br />  <br />                         20                                                                             0.5 <br />  <br />  <br />  <br />  <br />                         0                                                                              0.0 <br />                              0.0   0.2              0.4           0.6   0.8   1.0                            0.0     0.2            0.4          0.6             0.8            1.0 <br />                                                    Conservation score                                                       Fraction higly conserved region <br />   (c)                                                                               (d) <br />  <br />  <br /> Figure conservation <br /> Codon   2               gene groups <br /> Codon conservation gene groups. conservation score amino acids groups genes compared.   distribution disease causing mutation sites&apos; conservation score plotted solid line. dotted line drawn based conservation scores sites coding region  e., distribution conservation score sites randomly chosen). (b) distribution polymorphism mutation sites&apos; conservation score vs. random dis- <br /> tribution  . (c) distribution conservation score UEHGs (black line), disease gene (broken blue line) dis- <br /> ease causing mutation sites (red broken line). (d) distribution fraction highly conserved regions (Cons. <br /> Score&gt;0.9). human gene group represented different color. <br />  <br />  higher fraction highly conserved sites                             achieve. Unfortunately, knowledge gene essential- groups, significant difference                          ity high animals limited. example, disease genes genes (p-value = 0.32).                             number mouse genes known mutation pheno- <br />                                                                                     types 10% genome (~2,800 Mouse <br /> Cross-species comparison gene deletion phenotypes                                Genome Informatics database) heavily human diseases studied experimenting                                 biased homologs human disease genes <br /> model organisms mouse. underneath ration-                               (results shown).  S. cerevisiae C. ele- <br /> ale homologous genes similar functions                          gans explored gene essentialities species evolutionarily close. Similarly,                            genome scale [5,6]. compare human <br /> essentiality human genes tested evolution-                          genome  favoured arily close species. approach work                             far evolutionarily distances genes, differences species,                              humans. Human genes mapped yeast worm <br /> closer species  higher accuracy                              based homologous relationship. UEHGs, disease <br />  <br />  <br />                                                                                                                                                                              Page 5 13 <br />                                                                                                                                        (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure <br /> Comparison <br />          3 gene essentiality human C. elegans <br /> Comparison gene essentiality human C. elegans Human genome divided groups described main text 20,488 C. elegans genes mapped group based homology. essentiality C. ele- <br /> gans gene obtained RNAi-interference experiment described main text. Different phenotypes represented different colors number homologs group listed. fraction human genes C. elegans <br /> homologs shown group  <br />  <br />  <br />  <br /> genes genes, examine fraction genes          UEHGs highest fraction, followed disease homologous essential mapped spe-            genes, genes.  shown cies (Table 2 Fig 3). column Table 2 shows      column, conditional homologous genes, UEHGs contain largest fraction genes           fraction essential genes group does strictly homologous counterparts yeast. Disease genes             follow order  e., UEHGs&gt;disease genes smaller fraction UEHGs greater          genes). prominent yeast C. elegans. <br /> genes. order observed C. elegans           phenomenon likely caused shown Fig 3. yeast C. elegans evolutionary      larger evolutionary distance yeast human. distant human, results support UEHGs           say given gene highly conserved  tain greater fraction functionally important genes          human-yeast homologous gene), essentiality disease genes. second column Table         gene longer strongly linked group 2 shows fraction homologous genes           gene belongs  addition, results suggest groups essential yeast.   <br />  <br />                                                                                                                      Page 6 13 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                    http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 2: Cross-species comparison gene essentiality human S. cerevisiae. <br />  <br />                                                Total                                 Essential                     Non-essential       Unknown <br />  <br />  Homologs UEHGs                  384 (20.2% UEHGs)*                138 (35.9%) (7.3% UEHGs)*                242(63.0%)         4(1.1%) <br />  Homologs disease genes        196 (9.8% disease genes)*         51 (26.0%) (2.5% disease genes)*           142(72.5%)         3(1.5%) <br />  Homologs genes          1005 (3.5% genes)*          379 (37.7%) (1.3% genes)*           618(61.5%)         8(0.8%) <br />  homologs                                 4641                                  505 (10.9%)                      3427(73.8%)       709(15.3%) <br />  Total yeast genes                           6179                                 1058 (17.1%)                      4397(71.2%)       724(11.7%) <br />  <br />  Yeast genes mapped human gene groups homologous mapping. rows human gene group fourth <br />  row yeast genes human homologous genes. row summarizes yeast genes. column lists number  homologous genes human gene group fractions respect number human genes corresponding group <br />   brackets marked *). following columns list number yeast essential, non-essential, unknown genes human gene <br />  group. fractions respect number homologs group shown brackets. second column, fraction  homologous essential genes human genes group given bracket *. <br />  <br />  <br />  <br /> highly conserved genes contain large fraction essential                 elders. People usually think genes associated early <br /> genes                                                                  onset diseases higher selection pressure                                                                            associated late onset diseases. correct, <br /> Comparison features                                               essential genes critical, evolutionary rates noticed years  protein                 similar early onset disease genes interaction network, essential proteins tend interact                   late onset ones. verify  divide disease proteins model organisms [14]. Peri et al.                    genes based onset age Jimenez et al. [25] manually collected 27,000 interactions involv-                   compare Ka/Ks ratio. Figure 6 shows evolutionary <br /> ing 18,000 human proteins [24]. data                   rates tend increase onset age larger. <br /> set quite sparse, accuracy assumed high.             correlation coefficient onset age distribution protein physical interaction degree                 Ka/Ks ratio positive P-value 0.02 based UEHGs, disease genes shown Fig-                   weighted squares regression [26]. (Weighted ure 4. average degree &#194;&#177; standard error UEHGs, dis-                 square used different age groups contain une- <br /> ease genes 11.0 &#194;&#177; 0.6,8.4 &#194;&#177; 0.4 6.2 &#194;&#177;                qual number genes non-constant variances, 0.1, respectively. consider proteins                 introducing weight regression, effects interaction, 0 degree mean inter-                reduced.) Base visual inspection, Fig. 6 suggests <br /> action absence data unable make                    UEHGs similar Ka/Ks ratios genes <br /> distinction. acknowledge data collection                       responsible diseases uterus genes biased human disease genes.                       similar Ka/Ks ratios genes associated late onset set UEHGs defined purely based gene                       diseases.  regression performed expression independent source, s unlikely                    group number actual disease onset age heavy bias UEHGs.  high interac-                    original linear relationship different disease <br /> tion degrees UEHGs regarded supporting                      onset ages distorted extent), P- <br /> evidence essentiality.                                           value just passes stringent cut  e., 0.05),                                                                            data analysis needed draw  investigate function annotation                   fident conclusion results. <br /> groups genes. shown Figure 5, UEHGs, disease <br /> genes genes distinct function distribu-                 Discussion <br /> tions. UEHGs enriched protein biosynthesis                      results support UEHGs fundamentally important physiological                        form distinct group disease genes. results <br /> processes, disease genes relevant sens-                  endorse UEHGs contain large proportion <br /> ing responding internal/external signals,                 functionally essential genes. try advanced mechanisms fine tuning certain bio-                    UEHGs good candidates human essential <br /> logical processes.                                                         genes, intention claim                                                                            best gene set representing human essential <br /> Finally, look relationship gene&apos;s conser-                genes. gene needs ubiquitously expressed <br /> vation onset age disease. Different diseases                    considered UEHG, low expressed exhibit symptoms different ages. diseases                    tissue specific expressed essential genes excluded. <br /> develop early utero present                     tissue samples collected mainly  <br />  <br />                                                                                                                                     Page 7 13 <br />                                                                                                               (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                               http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br />                    70% <br />  <br />  <br />  <br />                    60% <br />  <br />  <br />  <br />                    50% <br />                                                                                                                UEHGs(762) <br />                                                                                                                Disease(1168) <br />                    40%                                                                                         4063) <br />         Fraction <br />  <br />  <br />  <br />  <br />                    30% <br />  <br />  <br />  <br />                    20% <br />  <br />  <br />  <br />                    10% <br />  <br />  <br />  <br />                    0% <br />                          1-5   6-10   11-15   16-20   21-25   26-30   31-35   36-40   41-45   46-50    51-55   56-60   61-65   66-70   71-75   &gt;76 <br />  <br />                                                               Protein-protein interaction degree <br />  <br />  <br />  <br />  <br /> Figure 4 protein physical interaction degrees <br /> Distribution <br /> Distribution protein physical interaction degrees. UEHGs, disease genes, genes shown differ- <br /> ent colors histogram. seen interaction degree increases, fraction UEHGs increases. summary statistics, main text. number genes interaction HPRD listed gene group. <br />  <br />  <br />  <br />  <br /> adult individuals, genes essential early stage                      Different previous studies human housekeeping <br /> development missed  revealed cross-                          gene, define UEHGs genes expressed  species comparison, UEHGs failed cover                                  &quot;exactly ) tissues examined. essential genes genes classified                      fluctuation gene expression error genes. study different set genes consider-                       gene expression measurement, tissues ing genes conserved yeast, C. elegans                         examined, fewer genes observed expressed human. results indicate contain large                         tissues. relax criteria allow missing expres- <br /> fraction essential genes (results shown).                          sion small fraction tissues size UEHGs  pointed Chervitz et al. [27], set                        sensitive number tissues examined. <br /> miss human essential genes don&apos;t                                  different cutoff value expression level homologs yeast C. elegans. contrast, UEHGs                         adopted. order verify results sensitive unbiased sample essential genes. combi-                           specific criteria used define UEHGs, prepare <br /> nation UEHGs conserved genes generate                              UEHGs set defined genes expressed more complete set candidates human essential                               300 standard units 79 tissues. leads genes. realize set disease genes                       2,038 genes grouped UEHGs, 1,509 genes <br /> study mainly genes associated Mendelian dis-                             contained original set 1,789 genes. evo- <br /> eases, complex disease genes represented.                         lutionary rates compared new set  <br />  <br />                                                                                                                                                Page 8 13 <br />                                                                                                                          (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                          http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure 5annotation genes groups <br /> Function <br /> Function annotation genes groups. categories described row labels columns classes genes. color scheme (scale shown right) used display significance level representation <br /> (numbered negative logarithm P-value, upper half scale) representation (numbered logarithm P-value, lower half scale) certain gene group function category. Hyper-geometric distribution used cal- <br /> culation P-value. <br />  <br />  <br /> UEHGs, disease genes genes. results              regions shorter genes. unclear dis- identical slight changes        ease genes generally longer genes. pos- <br /> P-values  details supplementary materials).            sible explanation functions disease <br /> indicates findings sensitive criteria      genes performed proteins certain defining UEHGs.                                            lengths. example, <span id='am-1' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="ion" datatype="xsd:string"></span>ion</span> channel proteins (e.g. <br />                                                                    cystic fibrosis transmembrane conductance regulator, <br /> Previous studies shown house-keeping genes               CFTR) need span membrane multiple shorter coding length [11] disease genes usu-           times form pore structure, task ally longer coding length [1]. confirm          fulfilled short protein. studies needed ings way comparison (Table 3). UEHGs              explore general cases  required expressed tissues constitu- <br /> tively, s beneficial intron untranslated <br />  <br />  <br />                                                                                                                         Page 9 13 <br />                                                                                                   (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> Table 3: Comparison length various parts UEHGs, disease genes, genes. <br />  <br />                                   UEHGs (n = 1400)               Disease (n = 1773)            (n = 10304)                   P-value <br />  <br />  Coding sequence length                 1501 &#194;&#177; 38                     2205 &#194;&#177; 73                     1849 &#194;&#177; 15                        3.2E-09 <br />                                           1109                          1537                           1459 <br />  Total exon length                      2545 &#194;&#177; 48                     3250 &#194;&#177; 78                     2752 &#194;&#177; 18                        1.6E-06 <br />                                           2136                          2557                           2343 <br />  Number exons                        10.7 &#194;&#177; 0.2                    13.5 &#194;&#177; 0.3                     9.9 &#194;&#177; 0.1                       9.6E-07 <br />                                             8                             10                             7 <br />  Total intron length                  35698 &#194;&#177; 1558                  60376 &#194;&#177; 2836                   54881 &#194;&#177; 1139                       0.012 <br />                                           15588                         23528                          18540 <br />  5&apos; UTR length                          546 &#194;&#177; 22                      582 &#194;&#177; 21                        560 &#194;&#177; 8                         0.55 <br />                                            238                           251                            245 <br />  3&apos; UTR length                          559 &#194;&#177; 21                      569 &#194;&#177; 21                        575 &#194;&#177; 8                         0.86 <br />                                            247                           243                            254 <br />  <br />  row line gives average value &#194;&#177; s.e.m, second line gives median. UTR stands untranslated region. Pair-wise rank <br />  sum tests performed largest P-values listed column. <br />  want point  shown Fig 6,                  essential genes. believe updated global picture sharp dividing lines essential genes, disease                        disease genes enable better identify genes genes. diseases simply lethal                        future [32]. associated genes essential genes defini- <br /> tion. diseases severe effects s                    Methods <br /> hard distinguish true non-disease genes.                         Compiling lists disease genes UEHGs list  gene essentiality better described                    list disease genes obtained OMIM [29]. <br /> continuous spectrum artificially divided                       3,962 records listed morbidmap (Jun 6, 2005) <br /> groups. complicated situations arise dif-                      entries known sequence (OMIM <span id='am-22' about='oboInOwl:id' typeof='owl:Thing'>ID</span> marked ferent mutation forms considered. different                         *), known sequence phenotype (OMIM ID <br /> mutations usually lead phenotypes different severi-                     marked #), phenotype description, molec- <br /> ties [28,29], disease gene non-essential                  ular basis known (OMIM ID marked +) gene essential gene non-lethal mutation form.                     retained study. total 2,012 genes unique  simple grouping human genome lack                            OMIM Ids finally collected human disease genes. power accurately illustration complex sce- <br /> nario associated human disease genes.                                    Ubiquitously expressed genes obtained                                                                               result recent large scale microarray experiment Conclusion                                                                    human gene expression patterns Su et al. [33]. total studies suggest human essential genes                          33,698 genes sampled 79 tissues interro- <br /> unique group genes simply ignored                        gated experiments. overall gene expression classified non-disease genes studies                      level 776.5 standard Affymetrix average difference <br /> disease genes. disease genes                       units, genes expression level greater 550 <br /> eral properties residing essential genes.                   standard units 73/79 tissues selected notice gene essentiality better described                    UEHGs  conservative estimation percentage continuous spectrum instead assigned class                    essential genes human genome 10%, <span id='am-23' about='rdfs:label' typeof='owl:Thing'>label</span>.  simplicity way classi-                  standards set roughly 2,000 genes fication good purpose research com-                  classified UEHGs). total 1,789 genes parisons performed easily.                                             collected. set UEHGs small overlap dis- <br />                                                                               ease genes 176 genes belong classes. list <br /> Extensive knowledge human essential genes                           UEHGs Additional file 2. <br /> critical understanding human diseases. shown essential genes direct associa-                      Collection gene features <br /> tion diseases cancer [30,31]. Studying human                     mouse rat homologs corresponding synony- <br /> essential genes provide key clues questions                    mous substitution rate (Ks), nonsynomymous substitu- human beings evolved.  limited                            tion rate (Ka) totally 15,726 human genes attentions paid little                      downloaded NCBI HomoloGene [34]. prevent <br /> atic studies  showed picture                        possible contamination paralogous genes, disease genes gets clearer explicitly consider                     considered mapped orthologous pairs. test <br />  <br />  <br />                                                                                                                                        Page 10 13 <br />                                                                                                                   (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                         http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />  <br /> Figure 6 disease onset age Ka/Ks <br /> Correlation <br /> Correlation disease onset age Ka/Ks. correlation disease onset age Ka/Ks. Disease genes divided 5 groups based disease onset age. weighted linear regression applied disease genes (group 2 5) shown dotted line. coefficient onset age +0.0086 P-value 0.02, derived regression. UEHGs genes plotted sides diseases genes visual comparison. standard deviation indicated short horizontal bar mean denoted solid circle. large variation group hints confounding fac- <br /> tors affect Ka/Ks. <br />  <br />  <br />  statistical significance difference Ka, Ks      genes [25]. Weighted square regression used <br /> Ka/Ks distributions groups, Kolmogorov-           correlations disease onset ages Smirnov test used calculate p-value [4]       Ka/Ks ratios [26]. direct comparisons possible. Nucleotide <br /> conservation scores downloaded UCSC                     Yeast genes collected NCBI Entrez Gene Data- <br /> Genome Browser website [35]. Human sequence varia-                base [37] divided groups: UEHG <br /> tion information obtained Swiss-Prot protein             homologs, disease gene homologs, human gene <br /> knowledgebase [36]. original amino acid positions             homologs, genes human homologs. mapped nucleotide positions correspond-            homologies obtained NCBI HomoloGene ing chromosome obtain conservation score.               described  yeast gene deletion phenotype data <br /> study correlation onset age disease      downloaded Saccharomyces Genome Database <br /> conservation, obtained onset ages 900              [38]. Similarly, genes C. elegans collected  <br />  <br />                                                                                                                      Page 11 13 <br />                                                                                                 (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                              http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br /> NCBI Entrez Gene Database divided             Additional material <br /> groups. RNAi phenotypes C. elegans genes retrieved WormBase [39]. RNAi phenotypes divided categories: lethal (including                                                                       Additional File 1 <br />                                                                       Supplementary materials Details methods described file. <br /> embryonic larval lethal), wild type, sick (phenotypes             Click file , unknown. genes anno-               [http://www.biomedcentral.com/content/supplementary/1471- <br /> tated phenotype, severe               2164-7-31-S1.doc] <br /> (assuming lethal&gt;sick&gt;wild) chosen pheno- <br /> types.                                                                Additional File 2 <br />                                                                       List UEHGsThe list UEHGs list contained file. degrees genes protein physical interaction              Click file <br />                                                                       [http://www.biomedcentral.com/content/supplementary/1471- <br /> network retrieved Human Protein Refer-                  2164-7-31-S2.txt] <br /> ence Database (HPRD) [24]. compare function dis- <br /> tribution genes different categories, used <br /> Gene Ontology   Biological Process protein func- <br /> tion annotation. Gene Ontology annotations 12,715             Acknowledgements <br /> human genes downloaded NCBI [40]               thank reviewers great suggestions revision. thank <br /> classifications based biological processes used.         Andrew Su John Hogenesch answering question regarding Similar Zhou et al. [41], node referred inform-    expression cutoff value UEHGs. thank Kim Fechtel pro- <br /> ative covers 500 genes,          viding classification genes compare gene sets. <br /> descendant nodes cover genes. 25 inform-            thank Matthew Lebo proofreading manuscript. research ative nodes defined according criterion. test     supported NIH/NSF joint mathematical biology initiative DMS-0241102 <br />                                                                  NIH P50 HG 002790. UEHGs, disease genes genes  represented 25 function categories, <br />                                                                  References used hyper-geometric distribution calculate p-         1.      Lopez-Bigas N, Ouzounis CA: Genome-wide identification value.                                                                   genes likely involved human genetic disease. Nucleic <br />                                                                          Acids Res 2004, 32:3108-3114. <br />                                                                  2.      Bortoluzzi S, Romualdi C, Bisognin  Danieli GA: Disease genes <br /> Gene length information retrieved UCSC                          intracellular protein networks. Physiol Genomics 2003, <br /> genome table browser [42]. genes                      15:223-227. <br /> mapped refSeq IDs length information                3.      Simth NGC, Eyre-Walker  Human disease genes: patterns                                                                          predictions. Gene 2003, 318:169-175. <br /> retrieval. assess significance difference   4.      Huang H, Winter EE, Wang H, Weinstock KG, Xing H, Goodstadt L, <br /> length genes different categories, Wilcoxon rank sum               Stenson PD, Cooper DN, Smith D, Alba MM, Pointing CP, Fechtel K: <br />                                                                          Evolutionary conservation selection human disease <br /> test used calculate p-value.                                  gene orthologs rat mouse genomes. Genome Biol <br />                                                                          2004, 5:R47. process collecting various features, genes        5.      Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />                                                                          Lucau-Danila  Anderson K, Andre B, Arkin AP, Astromoff  Bakk- annotated certain databases. limited                  oury  Bangham R, Benito R, Brachat S, Campanaro S, Curtiss M, <br /> comparisons genes information. number                     Davis K, Deutschbauer  Entian K, Flaherty P, Foury F, Garfinkel DJ, <br />                                                                          Gerstein M, Gotte D, Guldener U, Hegemann JH, Hempel S, Herman <br /> genes included comparison                    Jaramillow DF, Kelly  Kelly SL, Kotter P, LaBonte D, Lamb DC, <br /> corresponding tables figures. information                 Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard P, Ooi method materials, Additional file 1.                         SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Scherens B, <br />                                                                          Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, Storms <br />                                                                          RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang C, Ward TR, <br /> Authors&apos; contributions                                                   Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu K, Bussey <br /> ZT conceived study, performed way classi-               H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnson M: Func- <br />                                                                          tional profiling Saccharomyces cerevisiae genome. <br /> fication, carried comparison evolutionary                 Nature 2002, 418:387-391. <br /> rate amino acid level, protein physical interaction           6.      Sonnichsen B, Koski LB, Walsh  Marschall P, Neumann B, Brehm M, <br />                                                                          Alleaume M, Artelt J, Bettencourt P, Cassin E, Hewitson M, Holz C, <br /> degrees, drafted manuscript. LW carried com-                 Khan M, Lazik S, Martin C, Nitzsche B, Ruer M, Stamford J, Winzi M, <br /> parison features. MX studied relation-              Heinkel R, Roder M, Finell J, Hantsch H, Jones SJ, Jones M, Piano F, <br /> ship gene expression correlation protein                         Gunsalus KC, Oegema K, Gonczy P, Coulson  Hyman AA, <br />                                                                          Echeverri CJ: genome RNAi profiling early embryogen- <br /> essentiality. XZ, TC FS jointly guided research.            esis Caenorhabditis elegans. Nature 2005, 434:462-469. <br /> authors read approved final manuscript.                  7.      Blake JA, Richardson JE, Bult CJ, Kadin JA, Eppig JT: Mouse <br />                                                                          Genome Database Group: MGD: Mouse Genome Data- <br />                                                                          base. Nucleic Acids Res 2003, 31:193-195. <br />                                                                  8.      Warrington JA, Nair  Mahadevappa M, Tsyganskaya M: Compari- <br />                                                                          son human adult fetal expression identification                                                                          535 housekeeping/maintenance genes. Physiol Genomics 2000, <br />                                                                          2:143-147. <br />                                                                  9.      Butte AJ, Dzau V, Glueck SB: defining housekeeping,                                                                          &quot;maintenance,&quot; genes Foucus  compendium gene <br />  <br />  <br />                                                                                                                               Page 12 13 <br />                                                                                                          (page number citation purposes) <br />  BMC Genomics 2006, 7:31                                                                        http://www.biomedcentral.com/1471-2164/7/31 <br />  <br />  <br />  <br />       expression normal human tissues&quot;. Physiol Genomics 2001,                 sor genes: higher intensities purifying selection       7:95-96.                                                                    genes. Mol Biol Evol 2003, 20:964-968. <br /> 10.   Zhang L, Li W-H: Mammalian housekeeping genes evolve                  32.   Adie EA, Adams RR, Evans KL, Porteous D, Pickard BS: Speeding <br />       slowly tissue-specific genes. Mol Biol Evol 2004,                 disease gene discovery sequence based candidate prioriti- <br />       21:236-239.                                                                 zation. BMC Bioinformatics 2005, 6:55. <br /> 11.   Eisenberg E, Levanon EY: Human housekeeping genes com-            33.   Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, Block D, Zhang J, <br />       pact. Trends Genet 2003, 19:362-365.                                        Soden R, Hayakawa M, Kreiman G, Cooke MP, Walker JR, Hogenesch <br /> 12.   Hirsh AE, Fraser HB: Protein dispensability rate evolu-              JB.: gene atlas mouse human protein-encoding <br />       tion. Nature 2001, 411:1046-1049.                                           transcriptomes. Proc Natl Acad Sci USA 2004, 101:6062-6067. <br /> 13.   Jordan IK, Rogozin IB, Wolf YI, Koonin EV: Essential genes        34.   Wheeler DL, Church DM, Edgar R, Federhen S, Helmberg W, Mad- <br />       evolutionarily conserved nonessential genes                   den TL, Pontius JU, Schuler GD, Schriml LM, Sequeira E, Suzek  <br />       bacteria. Genome Res 2002, 12:962-968.                                   Tatusova TA, Wagner L: Database resources national <br /> 14.   Jeong H, Mason SP, Barbasi L, Oltvai ZN: Lethality central-           center biotechnology information: update. Nuclei Acids Res <br />       ity protein networks. Nature 2001, 411:41-42.                            2004, 32:D35-D40. <br /> 15.   Bourque G, Pevzner PA, Tesler G: Reconstructing genomic           35.   Conservation score              [http://hgdownload.cse.ucsc.edu       archeitechture ancestral mammals: lessons human,                    loads.html#human] <br />       mouse, rat genomes. Genome Res 2005, 14:507-516.                  36.   Human sequence variation                  [http:/ expasy.org/sprot/sp- <br /> 16.   Wolfe KH, Sharp PM: Mammalian gene evolution: nucleotide                    docu.html] <br />       sequence divergence mouse rat. J Mol Evol 1993,           37.   NCBI Entrez gene                  [http://www.ncbi.nlm.nih.gov/entrez/ <br />       37:441-456.                                                                 query.fcgi?db=gene] <br /> 17.   Ohta T, Ina Y: Variation synonymous substitution rates             38.   Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, Hester ET, Jia Y, <br />       mammalian genes correlation syn-                      Juvik G, Roe T, Schroeder M, Weng S, Botstein D: SGD: Saccharo- <br />       onymous nonsynonymous divergences. J Mol Evol 1995,                     myces Genome Database. Nucleic Acids Res 1998, 26:73-79. <br />       41:717-720.                                                           39.   Chen N, Harris TW, Antoshechkin  Bastiani C, Bieri T, Blasiar D, <br /> 18.   Makalowski W, Boguski MS: Evolutionary parameters                    Bradnam K, Canaran P, Chan J, Chen CK, Chen WJ, Cunningham F, <br />       transcribed mammalian genome: analysis 2,820 orthol-                  Davis P, Kenny E, Kishore R, Lawson D, Lee R, Muller HM, Nakamura <br />       ogous rodent human sequences. Proc Natl Acad Sci USA                    C, Pai S, Ozersky P, Petcherski  Rogers  Sabo  Schwarz EM, Van <br />       1998, 95:9407-9412.                                                         Auken K, Wang Q, Durbin R, Spieth J, Sternberg PW, Stein LD: <br /> 19.   Smith NG, Hurst LD: effect tandem substitutions               WormBase: comprehensive data resource Caenorhab- <br />       correlation synonymous nonsynonymous rates                      ditis biology genomics.                 Nucleic Acids Res 2005, <br />       rodents. Genetics 1999, 153:1395-1402.                                   33:D383-D389. <br /> 20.   Castresana J: Estimation genetic distances human          40.   Gene ontology annotation                 [ftp://ftp.ncbi.nlm.nih.gov/gene/ <br />       mouse introns. Genome Biol 2002, 3:R28.                                     DATA/] <br /> 21.   Pagani F, Raponi M, Baralle FE: Synonymous mutations CFTR          41.   Zhou X, Kao MJ, Wong WH: Transitive functional annotation <br />       exon 12 affect splicing neutral evolution. Proc              shortest-path analysis gene expression data. Proc Natl <br />       Natl Acad Sci USA 2005, 102:6368-6372.                                      Acad Sci USA 2002, 99:12783-12788. <br /> 22.   Siepel  Bejerano G, Pedersen JS, Hinrichs  Hou M, Rosenbloom     42.   Karolchik D, Hinrichs  Furey TS, Roskin KM, Sugnet CW, Haussler <br />       K, Clawson H, Spieth J, Hillier LW, Richards S, Weinstock GM, Wil-          D, Kent WJ: UCSC Table Browser data retrieval tool. <br />       son RK, Gibbs RA, Kent WJ, Miller W, Haussler D: Evolutionarily             Nucl Acids Res 2004, 32(Suppl 1):D493-D496. <br />       conserved elements vertebrate, insect, worm, yeast <br />       genomes. Genome Res 2005, 15:1034-1050. <br /> 23.   Miller MP, Kuman S: Understanding human disease mutations <br />       use interspecific genetic variation. Hum Mol <br />       Genet 2001, 10:2319-2328. <br /> 24.   Peri S, Navarro JD, Amanchy R, Kristiansen TZ, Jonnalagadda CK, <br />       Surendranath V, Niranjan V, Muthusamy B, Gandhi TKB, Gronborg M, <br />       Ibarrola N, Deshpande N, Shanker K, Shivashankar HN, Rashmi BP, <br />       Ramya MA, Zhao  Chandrika KN, Padma N, Harsha H <br /> </body></html>