<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Functions Fission Yeast Orp2 DNA Replication Checkpoint Control <br />  <br />  <br />                                                                 ABSTRACT <br />                 orp2 essential gene fission yeast Schizosaccharomyces pombe 22% identity budding yeast <br />              ORC2. isolated temperature-sensitive alleles orp2 using novel plasmid shuffle based selection <br />              <span id='am-31' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span>thymidine</span> kinase. Cells bearing temperature-sensitive allele orp2-2 fail complete DNA replica- <br />              tion restrictive temperature undergo cell cycle arrest. Cell cycle arrest depends checkpoint <br />              genes rad1 rad3. checkpoint functions wild type, orp2-2 mutation causes high rates <br />              chromosome plasmid loss. phenotypes support idea Orp2 replication initiation <br />              factor. Selective spore germination allowed analysis orp2 deletion mutants. experiments showed <br />              absence orp2 function, cells proceed mitosis despite lack DNA replication.              suggests Orp2 protein checkpoint machinery likely DNA <br />              replication initiation required induce replication checkpoint signal. <br />  <br />  <br />  <br /> E    UKARYOTIC DNA replication initiated       replication origins chromosome. budding yeast Saccharomyces cerevisiae, origin sequences <br />                                                                                Orp2 binds replication activator Cdc18 <br />                                                                              (Leatherwood et al. 1996). Cdc18 similar                                                                              Cdc6 proteins budding yeast, Xenopus, humans bound origin recognition complex (ORC),                           (Kelly et al. 1993). Cdc18/Cdc6 activity cell cycle composed subunits, Orc1 Orc6                         regulated, regulation important accurate <br /> (Bell Stillman 1992). eukaryotes,                           regulation DNA replication. fission yeast, overex- <br /> cis-acting sequences replication origins com-                    pression Cdc18 leads DNA rereplication (Nishi- <br /> plex understood (Huberman 1998).                         tani Nurse 1995). appears Cdc18 activity <br /> theless, ORC appears widely conserved.                          regulated Cdc2 kinase (Jallepalli et al. 1997; <br /> fission yeast Schizosaccharomyces pombe, genes similar                    Lopez-Girona et al. 1998). attractive model sequence ORC1, ORC2, ORC5 identi-                           Cdc18/Cdc6 interacts ORC catalyzes asso- <br /> fied (Gavin et al. 1995; Muzi-Falconi Kelly 1995;                        ciation MCM replication factors chromatin <br /> Grallert Nurse 1996; Leatherwood et al. 1996;                            (J. Leatherwood 1998). <br /> Ishiai et al. 1997).                                                           cis-acting sequences used replication origins <br />    ORC2-like factor, Orp2 (Orc2 related pombe),                    budding yeast include binding site ORC particularly interesting interacts proteins                  binding site transcription activator; higher <br /> required coupling DNA replication cell cycle pro-                     eukaryotes, sequences direct replication initia- <br /> gression. Orp2 identified binds                      tion remained elusive (reviewed Rowley et al. cell cycle kinase Cdc2 (Leatherwood et al. 1996).                        1994; Huberman 1998), fission yeast, deter- fission yeast, Cdc2 cyclin-dependent kinase                        minants origin activity appear complex <br /> governing G1 S G2 M transitions                       possibly redundant budding <br /> (Nurse Bisset 1981; Nurse 1990). Initiation                           yeast (Wohlgemuth et al. 1994; Clyne Kelly 1995; <br /> DNA replication requires active complex Cdc2                           Dubey et al. 1996). gain broader understanding B-type cyclins Cdc13, Cig1, Cig2.                          replication initiates, informative (Fisher Nurse 1996; Mondesert et al. 1996).                          learn functions putative fission yeast Cdc2/Cdc13 complex activate replication                        ORC complex vivo. end, isolated essential prevent initiation DNA replica-                   temperature-sensitive alleles orp2. <br /> tion G2 (Broek et al. 1991; Hayles et al. 1994).                        isolate orp2 mutants, developed plasmid-shuf- interaction Cdc2 Orp2 suggests Cdc2                          fle fission yeast. Fission yeast, like fungi, <br /> regulates DNA replication directly replication origins                    lacks thymidine kinase. used thymidine kinase <br /> (Leatherwood et al. 1996).                                                   gene herpes simplex virus new counterselectable <br />                                                                              marker taking advantage fact pres- <br />                                                                              ence thymidine kinase causes sensitivity thymi- <br />                                                                              dine analogue 5-fluoro-29-deoxyuridine (FUdR).   Corresponding author: Janet Leatherwood, Department Molecular <br /> Genetics Microbiology, Life Science, Rm. 130, SUNY, Stony                flood information genome projects Brook, NY 11794-5222. E-mail: janet.leatherwood@sunysb.edu                   increasing number genes identified hybrid <br />  <br /> Genetics 154: 599&#226;&#8364;&#8220;607 (February 2000) <br />  600                                                          J. Kiely et al. <br />  <br />                                                                TABLE 1 <br />                                                            S. pombe strains <br />  <br />                                                Genotype                                                   Source <br />          JLP46          h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 his7-366/his7-366             Leatherwood et al. (1996) <br />                           ura4-D18/ura4-D18 orp21/orp2D::ura41 <br />          JLP125         h2 ade6-M216 leu1-32 his71::padh-TK                                     study <br />          JLP164         h1 ade6-M210 ura4-D18 leu1-32 his7-366 orp2D::ura41                     study <br />                           porp2TKade <br />          JLP166         h2 ade6-M216 ura4-D18 leu1-32 his7-366 orp2D::ura41 porp2TKade            study <br />          JLP208         h2 orp2-2 ura4-D18 leu1-32                                                study <br />          JLP216         h2 orp2-7 ura4-D18 leu1-32                                                study <br />          JLP242         h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 his7-366/his7-366               study <br />                           ura4-D18/ura4-D18 orp2-2/orp2D::ura4 <br />          JLP247         h1 rad1-1 orp21 ade62 ura4-D18 leu1-32 his7-366                         study <br />          JLP248         h1 rad1-1 orp2-2 ade62 ura4-D18 leu1-32                                 study <br />          JLP249         h2 rad11 orp2-2 ade62 ura4-D18 leu1-32                                  study <br />          JLP250         h2 rad11 orp21 ade62 ura4-D18 leu1-32 his7-366                          study <br />          CHP429         h2 ade6-M216 leu1-32 his7-366                                           C. Hoffman <br />          PR109          h2 leu1-32 ura4-D18                                                     P. Russell <br />  <br />  <br />  interaction screens, increasing                   (McLeod et al. 1987). plasmid pJL217 transformed <br /> need engineer conditional mutations. methods                     S. pombe (strain PR109 h2 leu1-32 ura4-D18, P. Russell)                                                                         selection leucine prototrophy; transformants sensitive <br /> used widely useful fission yeast stud-                25 mm FUdR (Sigma, St. Louis, cat. F-0503) indicating <br /> ies. approach applicable budding yeast                   functional thymidine kinase expressed adh1 <br /> (S. B. Haase, unpublished data; Sclafani Fang-                      promotor (data shown). his71, padh -TK plasmid <br /> man 1986), Neurospora crassa, fungi (Gri-                  pJL218 constructed cloning 3.0-kb EcoRI fragment <br /> vell Jackson 1968). Expression thymidine ki-                     containing padh -TK pJL217 EcoRI-digested his71 <br />                                                                         plasmid pEA13 (Apolinario et al. 1993). integrate padh - <br /> nase fission yeast makes possible measure                 TK his7 locus, pJL218 linearized his71 DNA replication BrdU incorporation use                     Cla1 used transform his7-366 mutant strain CHP429 tritiated thymidine (J. Leatherwood, unpublished                     histidine prototrophy. resulting allele designated <br /> data). labeling newly synthesized DNA                    his71::padh-TK (strain JLP125 h2 leu1-32 ade6-M216 his71:: <br /> possible wild-type yeast lack thymidine salvage             padh -TK) results FUdR sensitivity (Figure 2). <br />                                                                            plasmid containing orp21, ade61, padh -TK  <br /> pathway use thymidine BrdU                    structed steps: (1) padh -TK 1.8-kb HindIII/ <br /> precursors DNA synthesis.                                           SmaI fragment pJL217 cloned HindIII/SmaI- <br />    previous studies showed orp2 essential               digested pREP3 (Maundrell 1993) create pJL240; (2) cells lacking orp2 died phenotypes consis-                orp21 2.8-kb HindIII fragment pER27 (Leath- <br /> tent defect DNA replication (Leatherwood                      erwood et al. 1996) cloned HindIII-digested pJL240 <br />                                                                         create pJL243; (3) ade61 3.0-kb PvuII, SpeI <br /> et al. 1996). study investigate S-phase progres-             fragment pAS1 (Szankasi et al. 1988) cloned sion, chromosome stability, replication-checkpoint                  pJL243 digested Nsi  treated T4 DNA <br /> control orp2 mutants.                                                polymerase remove Nsi overhang, purified,                                                                         digested SpeI. resulting plasmid, pJL261, referred <br />                                                                         porp2 TKade text shown Fig- <br />                MATERIALS METHODS                                    ure 1. orp21 expression plasmid pJL207 described <br />                                                                         (Leatherwood et al. 1996). <br />    S. pombe strains media: genetic methods media               orp2 mutagenesis identification conditional alleles: <br /> used S. pombe essentially described Moreno et            orp2 amplified 20 cycles (annealing temperature 678) <br /> al. (1991). YSO rich media low <span id='am-16' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> (0.5% yeast                plasmid pJL207 primers oJL18 (nmt promotor se- <br /> extract, 0.2% casamino acids, 3% glucose; add 2% agar              quence 59 TCT CAC TTT CTG ACT TAT AGT CGC T 39) <br /> making plates). Strains used study listed Table 1.       oJL19 (nmt terminator sequence 59 CTA GCA GTA CTG <br />    Thymidine kinase constructs: thymidine kinase gene               GCA AGG GAG AC 39) using mutagenic PCR conditions  herpes simplex virus (HSV1-TK; McKnight 1980)                  scribed Cadwell Joyce (1992). Reactions contained <br /> cloned expressed adh1 promoter S.               dNTPs divalent cations specified Table 2 pombe ; refered padh -TK. BamHI recognition sites         contained 1 mm <span id='am-21' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span>dCTP</span>, 1 mm <span id='am-6' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span>dTTP</span>, 20 mm Tris pH 8.4, 50 added coding region HSV1-TK PCR amplifi-              mm KCl, 50 ng plasmid template, 1 mg primer, 5 <br /> cation using primers BTK59 CGGGATCCCGTATGGCTTCG                         units TAQ polymerase (GIBCO-BRL, Gaithersburg, MD) TACC 39 BTK39 CGGGATCCCGTGTTTCAGTTAGCC 39.                          final volume 100 ml. PCR products digested Bsi WI <br /> padh -TK expression plasmid pJL217 constructed clon-             BamHI cloned Bsi WI/BamHI-digested pJL207. <br /> ing 1.13-kb TK PCR product BamHI site 39                Approximately 1000 Escherichia coli transformants pooled adh1 promotor fission yeast expression plasmid pART1             PCR reaction condition. Plasmid DNA                                                      Mutants Fission Yeast orp2                                               601 <br />  <br />                                                                      low adenine plates), finally rich media 258 368 <br />                                                                      screen temperature-sensitive phenotype. Ura2, Ade2, <br />                                                                      temperature-sensitive colonies analyzed Southern <br />                                                                      blotting confirm orp2D::ura4 replaced.                                                                      method temperature-sensitive orp2-2 orp2-7 alleles <br />                                                                      isolated stable integrants (allele numbers indicate                                                                      PCR pools mutants obtained). <br />                                                                         Selective spore germination FACS analysis: Diploid <br />                                                                      strain JLP46 (h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 <br />                                                                      ura4-D18/ura4-D18 orp21/orp2D::ura4) described <br />                                                                      (Leatherwood et al. 1996). haploid orp2D::ura4 derived                                                                      JLP46 kept alive orp2 expressed plasmid pJL207 <br />                                                                      (Leatherwood et al. 1996). haploid mated                                                                      orp2-2 temperature-sensitive strain create diploid                                                                      isolate lacking pJL207 plasmid obtained; <br />                                                                      JLP242 (h1/h2 ade6-M210/ade6-M216 leu1-32/leu1-32 <br />                                                                      ura4-D18/ura4-D18 his7-366/his7-366 orp2-2/orp2D::ura4). Se- <br />                                                                      lective spore germination FACS analysis performed <br />   Figure 1.&#226;&#8364;&#8221;Map shuttle vector porp2 TKade (plasmid               previously described JLP46; note ungerminated <br /> pJL261).                                                             spores removed final data presented gating <br />                                                                      small objects based forward scatter assignment <br />                                                                      gate parameters determined analysis pure <br />                                                                      population ungerminated spores (Leatherwood et al. <br /> pooled colonies transformed yeast strain JLP166                                                                      1996). hydroxyurea (HU)-treated samples, 15 mm HU <br /> 258. total 200&#226;&#8364;&#8220;1200 yeast transformants screened <br />                                                                      added media beginning germination. pool replica printing YES YES 1 50 <br /> mm FUdR growing 258 368. percentage nonfunctional temperature-sensitive orp2 alleles PCR pool shown Table 2. total 97 temperature-                                       RESULTS <br /> sensitive candidates picked retested replica print- <br /> ing FUdR plates 258, 328, 35.58, 368. Plasmids         Isolation orp2 mutants: Fission yeast orp2 essen- <br /> rescued 16 candidates gave greatest temperature        tial; recessive orp2 mutants identified <br /> dependence growth. plasmids retrans-          using plasmid shuffle. haploid deleted orp2 formed JLP166 temperature-sensitive pheno-          kept alive wild-type orp2 plasmid, types retested.                                                 carried gene HSV thymidine kinase. Mutagen- <br />   Integration orp2 mutants: experiment thymidine kinase gene, thymidine kinase-based approach         ized orp2 second plasmid introduced. Cells devised integrate selected orp2 * mutants. reasoned        lost wild-type orp2 plasmid isolated integration orp2 * plasmid shuffle strain JLP164    selecting thymidine kinase activity. Recessive <br /> (orp2D::ura4, porp2TKade) allow cells grow           phenotypes mutant orp2 alleles scored, <br /> permissive temperature porp2TKade plasmid            plasmids recovered retested, selected mu- viable media containing FUdR. Integrants selected basis idea. orp2 * linear DNA       tants integrated genome study. <br /> introduced strain JLP164 LiOAc transformation               Selection thymidine kinase gene: Fungi cells grown 2 days 258 rich media allow        including S. pombe, S. cerevisiae, N. crassa lack thymi- <br /> loss porp2TKade plasmid. Transformations            dine kinase use thymidine precur- <br /> plated directly rich media (YES) containing 100 mm FUdR           sor DNA synthesis (reviewed Kornberg incubated 258 3 days. Colonies screened replica printing plates lacking <span id='am-26' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span>  orp2D::ura4        Baker 1992). Thymidine kinase phosphorylates thymi- replaced, fail grow uracil), plates     dine, phosphorylates halogenated thymidine <br /> low adenine  porp2TKade lost, colonies red       derivates FUdR. Phosphorylated FUdR po- <br />  <br />  <br />                                                                TABLE 2 <br />                                                      Mutagenic PCR conditions <br />  <br />                          <span id='am-11' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span>dATP</span> <br />                          <span id='am-1' about='obo:IMR_0200286' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00286" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200286" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="dGTP" datatype="xsd:string"></span>dGTP</span>              Mg21                              % Nonfunctional             % Temperature <br />          PCR             (mm)              (mm)              Mn21                258                     sensitive <br />          1                 0.2               5                                 28                          2.8 <br />          2                 0.2               5              0.5 mm                  70                          2.0 <br />          3                 0.2               8                                 15                          1.2 <br />          5                 1.0               5                                 25                          0.9 <br />          6                 1.0               5              0.5 mm                  45                          6.2 <br />          7                 1.0               8                                 25                          3.5 <br />          8                 1.0               8              0.5 mm                  67                          3.8 <br />  602                                                        J. Kiely et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 3.&#226;&#8364;&#8221;Characterization thymidine kinase-based <br />                                                                       plasmid shuffle. Cells contain LEU2 selectable vector (middle, <br />   Figure 2.&#226;&#8364;&#8221;Expression thymidine kinase causes sensitivity         control) LEU2, orp21 expression plasmid pJL207 (left FUdR. Strain CHP429 (wild type) thymidine kinase               right). middle right, cells contain shuttle <br /> expression strain JLP125 (padh-TK) grown YES plates           vector porp2TKade. cells lack chromosomal orp2 (orp2 <br /> containing 0 mm, 1 mm, 100 mm FUdR.   Cell morphol-             D::ura41) ade6 mutants (ade6-m210).   Cells ogy 1 day 328; (B) overall growth 2 days 328.       grown YSO media cells lacking porp2TKade red. <br />                                                                         Cells grown YSO plus 50 mm FUdR                                                                       cells containing porp2TKade killed, cells tent inhibitor thymidylate synthetase prevents              additional source functional orp2 (pJL207) grow. novo synthesis dTTP (reviewed Kornberg Baker 1992). dTTP, cells replicate DNA eventually die. previously shown                 convenient way identify FUdR resistance plas- <br /> budding yeast expressing thymidine kinase selec-               mid loss. <br /> tively killed FUdR (Sclafani Fangman 1986;                        orp2 shuttle plasmid sole source orp2 S. B. Haase S. Reed, unpublished data).                           strains JLP164 JLP166; chromosomal gene <br />    reasoned fission yeast sensitive             deleted replaced ura41 (orp2D::ura41). killing FUdR. Wild-type fission yeast thymi-            test feasibility plasmid shuffle, JLP166 <br /> dine kinase expression strain JLP125 (materials                   haploid transformed LEU2 orp21 <br /> methods) grown media containing FUdR.                    expression plasmid pJL207 LEU2 vector pRep1. <br /> treatment caused cell cycle arrest thymidine ki-               expected, cells received orp21 plasmid nase strain JLP125: cells failed divide continued              lose shuttle plasmid porp2TKade grow FUdR grow length (Figure 2). cell cycle arrest               medium transformed vector  <br /> similar seen DNA replication blocked                  trol require porpTKade plasmid fail grow hydroxyurea replication mutants. JLP125 cells                FUdR medium (Figure 3). arrested 1 mm FUdR, lowest concentration                     Generation orp2* mutants: orp2 open reading <br /> tested, wild-type strain grows normally                frame mutagenized PCR cloned media containing 100 mm FUdR shows slight                  LEU21 orp2 expression vector pJL207 produce orp2* <br /> decrease growth rate cell cycle delay 600               mutant libraries. Amplified library pools trans- <br /> mm FUdR.                                                              formed fission yeast strain JLP166 selecting    plasmid shuffle based thymidine kinase:                   Leu1. determine transformants received func- <br /> goal plasmid wild-type allele           tional orp2, colonies replica printed FUdR plates <br /> favorite gene  e., orp21) able select cells            select porp2TKade plasmid. Transformants lose plasmid uncover mutant allele             grew FUdR orp2/LEU2 plasmid pro- <br /> gene function mutant determined.                 vided essential functions orp2. Temperature-sensi- case, requires special orp2 shuttle             tive mutants identified transformants plasmid. plasmid named porp2TKade                     grow FUdR plates 258 368  materi- important features: (1) wild-type orp21, (2) count-             als methods details). Sixteen temperature- <br /> erselectable marker thymidine kinase (padh-TK)                sensitive alleles orp2 obtained.  <br /> loss plasmid selected growth                  orp2-2 orp2-7, integrated genome   <br /> FUdR, (3) selectable marker ade61 yeast trans-                placing orp2D::ura41 allele orp2*) study <br /> formation; ade61 detected colony color.                (materials methods). low adenine medium, ade62 mutants form red colo-                      Analysis orp2-2 orp2-7 temperature sensitive <br /> nies, ade62 mutants containing porp2TKade Ade1                mutants: expected mutations interfere form white colonies. case, colony color              DNA replication, orp2 mutants cell cycle  <br />                                                  Mutants Fission Yeast orp2                                                603 <br />  <br /> fects restrictive temperature. 36.58, strains JLP208       heterozygous orp2-2/orp21 orp2-7/orp21 diploids grow <br /> (orp2-2) JLP216 (orp2-7) stop dividing, elongate,            divide restrictive temperature 36.58 indi- lose viability. division arrest orp2-7 hetero-     cating alleles recessive. <br /> geneous cells remain arrested, exper-           studying replication protein, iments use orp2-2. temperature-sensitive pheno-              fact used plasmid-borne alleles original <br /> types JLP208 (orp2-2) JLP216 (orp2-7)            screen allowed selection slightly defec- complemented orp21 plasmid pJL207. Finally,             tive alleles. idea cell dependent par- <br />                                                                  tially defective replication protein lower ef- <br />                                                                  ficiency plasmid replication leading                                                                  protein significant plasmid loss. cases <br />                                                                  mutagenized gene&#226;&#8364;&#8482;s normal function                                                                  affect plasmid copy number stability, screening                                                                  mutant alleles plasmids yield severely defective <br />                                                                  mutants plasmid-borne gene present <br />                                                                  copies; result, defective proteins                                                                  overexpressed, mask subtle defects                                                                  function. <br />                                                                     FACS analysis shows orp2-2 arrests S-phase                                                                  restrictive temperature (Figure 4A). Unlike cdc10-129, <br />                                                                  arrests 1C DNA content, orp2-2 appears                                                                  arrest S-phase 1C 2C DNA content. <br />                                                                  3.5 hr shift restrictive temperature, cell divi- <br />                                                                  sion ceased orp2-2 population; septation <br />                                                                  index ,1%, cells elongated. indicates                                                                  cycle arrest S-phase orp2-2 allele. <br />                                                                     cell division arrest result incomplete DNA <br />                                                                  replication, cell cycle arrest require replication <br />                                                                  checkpoint functions. test  orp2-2 mutant <br />                                                                  crossed replication checkpoint mutants rad1-1 <br />                                                                  rad3D. 368, single mutant orp2-2                                                                  highly elongated, indicative cell cycle arrest.                                                                  arrest checkpoint dependent 368, orp2-2 <br />                                                                  rad1-1 orp2-2 rad3D double mutants continue di- <br />                                                                  vide. divisions aberrant result &#226;&#8364;&#339;cut&#226;&#8364;&#65533; <br />                                                                  cells loss viability expected cells enter <br />                                                                  mitosis incompletely replicated chromosomes (Fig- <br />                                                                  ure 4B). <br />                                                                     checkpoint functions intact, orp2-2 <br />                                                                  mutation results genetic instability. measure chro- <br />                                                                  mosome stability, used Ch16, small derivative                                                                  chromosome III stably maintained fourth <br />  <br />  <br />  <br />                                                                     Figure 4.&#226;&#8364;&#8221;Phenotypic analysis orp2-2.   FACS analysis <br />                                                                  DNA replication orp2-2 mutants: cells grown mid- <br />                                                                  log phase shifted 36.58 0, 2, 3.5 hr. Wild-type <br />                                                                  cdc10-129 mutant strains included controls. (B) <br />                                                                  Checkpoint dependence orp2-2 cell cycle arrest.   Wild- <br />                                                                  type, (b) rad1-1, (c) orp2-2, (d) rad1-1 orp2-2 strains                                                                  grown liquid media mid-log phase (258), shifted                                                                  restrictive temperature orp2-2 (36.58) 3.5 hr, fixed, <br />                                                                  stained 49,6-diamidino-2-phenylindole (DAPI). (C) <br />                                                                  Chromosome loss assay: wild-type orp2-2 mutants containing <br />                                                                  nonessential fourth chromosome (Ch16) grown                                                                  5 hr restrictive temperature (368), plated YSO <br />                                                                  media permissive temperature (258); 10-fold mutant <br />                                                                  cells plated wild type compensate loss                                                                  viability mutant. Red (dark) colonies formed                                                                  fourth chromosome Ch16 lost. <br />  604                                                     J. Kiely et al. <br />  <br /> chromosome (Niwa et al. 1986). Ch16 monitored                   depends inherited protein, spores inherit intragenic complementation ade6-M216                 mutant protein severe DNA replication <br /> Ch16 ade6-M210 chromosome III. Cells                   defects cells inheriting wild-type protein. FACS <br /> Ch16 Ade1 form white colonies media low                 analysis germinating spore population shows adenine (YSO) loss Ch16 results                 initial DNA replication slow spores inherit <br /> red sector colony. Ch16 crossed orp2-2             mutant protein compared spores inherited <br /> mutant, cells incubated 368 5 hr,              wild-type protein.  clear difference plated YSO 258. cells died; z10%               populations 298, permis- <br /> survived formed white red colonies depending                sive temperature replication growth orp2-2 maintenance Ch16 (Figure 4C). experi-            strains (Figure 5). appears inherited mutant <br /> ment, z50% surviving orp2-2 mutants lost Ch16 com-              Orp2-2 protein does support replication pared 1 1000 loss events wild type. Chro-          similarly inherited wild-type protein. shown mosome loss readily accounts loss viability          spores inheriting wild-type orp2 product able orp2-2 restrictive temperature. Finally, plasmid main-          replicate genome approximately twice (Leath- <br /> tenance transformation efficiency poor              erwood et al. 1996). strongly suggests orp2-2 mutant permissive temperature.                  protein stable functional successive cell <br />    Analysis cells lacking orp2 inherit mutant orp2-2       cycles. stable inheritance Orp2 consistent product: Previously, orp2D::ura4 deletion allele            role ORC epigenetic inheritence. <br /> orp2 analyzed using selective spore germination.                  spore germination experiment using spores <br /> Unexpectedly, nearly complete cycles             strain orp2-2/orp2D::ura4 (JLP242), nearly 50% of DNA replication achieved orp2D::ura4                cells entered mitosis failed nuclear division (Figure <br /> cells. hypothesized wild-type              7). resulted anucleate daughter cells  <br /> protein inherited spores. used              equally divided nuclei cut septum formation orp2-2 mutant ask directly cycles DNA                cells multiple aberrant septa. mitotic <br /> replication carried using orp2 protein inherited           phenotypes suggest orp2 function important parent.                                                   proper replication checkpoint control mitosis.    create population cells orp2 gene              observed similar events low frequency deleted, selectively germinated spores containing               orp2D::ura4 cells inherited wild-type protein, orp2D::ura4. diploid strain orp2-2/orp2D::ura4                 effect pronounced spores <br /> (JLP242) constructed comparison pre-              inherit mutant protein. time nearly 50% <br /> viously studied diploid orp21/orp2D::ura4 (JLP46).            cells inherit mutant protein abnormal divi- <br /> diploids grown sporulated permissive tem-              sions, z5% cells inherit wild-type protein <br /> perature (278), spores germinated media                phenotype.  Orp2 defective, <br /> lacking uracil 298 348. restrictive temper-       checkpoint arrest cells aberrant <br /> ature 348 chosen preliminary experi-                mitoses occur. <br /> ments shown germination poor 36.58                  tested checkpoint response HU strains used.                                                  affected loss orp2 function (Figures 6    DNA replication orp2D::ura4 germinating spores            7). repeated selective spore germination  <br />  <br />  <br />  <br />                                                                                Figure 5.&#226;&#8364;&#8221;Selective spore germination orp2 <br />                                                                              deletion spores.   orp21/orp2D::ura41 parent <br />                                                                              JLP46 (B) orp2-2/orp2D::ura41 parent JLP242. <br />                                                                              Spores germinated media lacking uracil                                                                              orp2D::ura4 spores germinate. (Left) <br />                                                                              FACS analysis; (right) cells fixed stained <br />                                                                              DAPI 14-hr, 348 sample. <br />                                                  Mutants Fission Yeast orp2                                              605 <br />  <br />  <br />  <br />  <br />   Figure 6.&#226;&#8364;&#8221;Selective spore germination presence hydroxyurea 348. Spores wild-type/orp2D::ura41 par- <br /> ent JLP46 orp2-2/orp2D::ura41 parent JLP242 germi- <br /> nated 15 mm HU media lacking uracil orp2D::ura41 spores germinate. (Left) FACS analysis; (right) cells fixed stained DAPI. <br />  <br />  <br /> presence HU 348. FACS data confirm HU treatment largely blocked DNA replication populations experment (Figure 6). orp2 deletion <br /> spores orp21 diploid synthesized small amounts DNA HU did enter mitosis. contrast, orp2 deletion spores orp2-2 diploid              Figure 7.&#226;&#8364;&#8221;Selective spore germination orp2 deletion <br /> germinated HU, went aberrant              spores; quantitation normal aberrant divisions mi- <br /> mitosis. results minimal level Orp2         croscopic analysis DAPI stained cells (100&#226;&#8364;&#8220;300 cells function required restraint mitosis         counted sample). Cells 348 experiments <br />                                                                  shown Figures 5 6 analyzed.   Septation index <br /> replication checkpoint. <br />                                                                  counting morphologically normal divisions, &#226;&#8364;&#339;normal&#226;&#8364;&#65533;                                                                  cells single septum middle wild-type-length <br />                                                                  cell (z8&#226;&#8364;&#8220;12 mm long) clearly divided nuclei. Cells                        DISCUSSION                                inherit wild-type orp2 product proceed appar- <br />    Temperature-sensitive alleles orp2 isolated        ently normal division, blocked cells                                                                  treated HU. (B) Septation index counting aberrant divi- new method uses thymidine kinase counterse-          sions, &#226;&#8364;&#339;aberrant&#226;&#8364;&#65533; cells multiple septa, unequally <br /> lectable marker. method applicable          divided nuclear masses, cut nuclei septum situations counterselection required.           bisects nuclear mass. orp2 alleles studied recessive resulted cell division cycle arrest. orp2-2 mutant unable progress S-phase judged FACS             restrictive temperature, orp2-2 mutants arrest  <br /> analysis, requires replication checkpoint function        fore mitosis.  reduction orp2 func- <br /> cell cycle arrest, orp2-2 defect results genomic      tion compromises checkpoint control. selective <br /> instability wild-type strain. pheno-       spore germination experiment used obtain cells <br /> types provide strong evidence orp2 essential         deleted orp2 gene. spores lacking orp2 function DNA replication.                                     germinated analyzed, replication <br />  606                                                     J. Kiely et al. <br />  division fates cells depended greatly            alleles genes allow little <br /> orp2 product inherited diploid parent.               replication fail checkpoint arrest. mutant product inherited, DNA replication                 mutations different factors severely impaired, cells unable             similar defects leads favor model arrest replication checkpoint instead pro-           mutants checkpoint defective fail <br /> ceeded aberrant mitosis. cells              common event, formation replication <br /> unable arrest response HU treatment.                 forks. conclude minimal level orp2 function                cut5 mutant special class, cut5 necessary establish DNA replication checkpoint               cells checkpoint defective replication <br /> control.                                                           forks initiated (Saka et al. 1994). Cut5 <br />    unexplained oddity data           directly link replication machinery check- populations cells spore germination experi-          point machinery checkpoint ma- <br /> ments: cells z1N DNA  e., cells synthesized              chinery. Dbp11, budding yeast homolog Cut5, little DNA) cells z2N DNA (Figure 5,                similarly required replication checkpoint B). 1N cells ungerminated spores                  responses. function Dbp11 appears related (materials methods). know                  polymerase &#206;&#181; (Araki et al. 1995), consistent populations cells.  inter-             idea replication forks generate checkpoint sig- <br /> ested z1N cells synthesized DNA             nals (Navas et al. 1995).  preliminary test, treated germinating                 alternative model prereplication complexes <br /> spores hydroxyurea (Figure 6), idea             mark DNA unreplicated cells replicating small DNA,              signal replication completed. context presence hydroxyurea induce dam-               model, imagine  e.g., Orp2- <br /> age response cause checkpoint arrest.                2 protein defective forming prereplication com- half orp2D cells inherit           plex triggers replication orp2-2 product went aberrant mitosis              checkpoint. existence budding yeast alleles <br /> hydroxyurea present  result encourages             orc5 orc2 inhibit mitosis replication speculate cells appearing             completed (Dillin Rine 1998) consis- <br /> 1N truly 1N replicated DNA             tent idea; alleles form    Furthermore, good correlation            aberrant prereplicative complex-like structures number cells appear 1N number                continue inhibit mitosis replication com- aberrant mitosis. Figure 5A, z5%               pleted normal prereplication complexes orp2D cells inherit orp21 product remain            dissassembled. fission yeast orp2-2 allele 1N, z5% cells undergo aberrant mitosis                  characterized does inhibit mitosis replication (Figure 7). Figure 5B, half opr2D cells            complete, possible screen inherit orp2-2 product remain 1N, half           type mutant fission yeast. cells undergo aberrant mitosis (Figure 7).                     <br />    possible interpretations           <br /> results, favor idea role Orp2            <br /> checkpoint signaling establish replication forks,       turn initiate checkpoint signals prevent <br /> mitosis.  propose checkpoint lack DNA replication;  checkpoint                        <br /> activated ongoing  aberrant) DNA replication.                 cell completely lacking replication, check-          <br /> point triggered mitosis occurs.                         <br />    long list genes affect DNA          <br /> replication replication checkpoint. fission                <br /> yeast, genes include orp2  article), orp1, cdc18,      <br /> cdt1, pol1 (DNA polymerase , rad11 (RPA), rfc2 (RFC),         cut5 (Kelly et al. 1993; Saka Yanagida 1993;              <br /> Saka et al. 1994; Hofmann Beach 1994; D&#226;&#8364;&#8482;Urso                 <br /> et al. 1995; Grallert Nurse 1996; Carlson et al.           <br /> 1997; Parker et al. 1997; Reynolds et al. 1999).           <br /> budding yeast, genes include CDC6  homolog              cdc18), CDC7, DBF4, RFC5 (RFC) (Piatti et al.            <br /> 1995; Toyn et al. 1995; Sugimoto et al. 1996; Tavor-             <br /> mina et al. 1997; Noskov et al. 1998). general,          <br /> </body></html>