<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, June 2007, p. 4325&#226;&#8364;&#8220;4327                                                                            Vol. 189,  11 <br /> 0021-9193/07/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.01854-06 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />           YhdJ, Nonessential CcrM-Like DNA Methyltransferase                    Escherichia coli Salmonella enterica&#228;&#338;&#164; <br />                             Sarah E. Broadbent,1 Roberto Balbontin,2 Josep Casadesus,2 <br />                                 Martin G. Marinus,3 Marjan van der Woude1* <br />    Hull York Medical School Department Biology, University York, P.O. Box 373, York Y010 5YW, England1; <br />        Departamento Gene&#194;&#180;tica, Facultad Biolog&#196;&#177;  Universidad Sevilla, Apartado 1095, Seville 41080, Spain2;              Department Biochemistry Molecular Pharmacology, University Massachusetts Medical School, <br />                                     364 Plantation Street, Worcester, Massachusetts 016053 <br />                                           Received 11 December 2006/Accepted 19 March 2007 <br />  <br />            Caulobacter crescentus DNA <span id='am-6' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> methyltransferase CcrM homologs &#226;?&#163;-Proteobacteria <br />          essential viability. CcrM 34% identical yhdJ gene products Escherichia coli Salmonella <br />          enterica. study provides evidence E. coli yhdJ gene encodes DNA adenine methyltransferase.          contrast earlier report,  yhdJ essential gene E. coli S. enterica. <br />  <br />  <br />    Eubacterial genomes encode DNA methyltransferases                       yhdJ gene essential, overexpression does restriction-modification systems,              alter cell morphology. address yhdJ essential, <br /> ferred orphan solitary DNA methyltransferases.                 yhdJ coding sequence  codons) <br /> best studied enzymes adenine methyltrans-                  replaced (9) chloramphenicol resistance gene (cat) <br /> ferases: Dam, present &#226;?&#165; subdivision Proteobacte-              E. coli strain AB1157 (4), resulting isolate GM8494. ria, CcrM  &#226;&#8364;&#339;cell cycle-regulated methyltransferase&#226;&#8364;?),               &#226;&#338;&#172;yhdJ::cat mutation transduced P1vir (8) species &#226;?&#163; subdivision. methyltransferases             GM8494 E. coli WA802 (15) designated GM8613. E. known involved control cellular events,            coli strain BW25113 yhdJ deletion derivative JW5543, <br /> including gene expression, cell cycle regulation, DNA                E. coli gene knockout collection (1), ana- <br /> pair (reviewed references 7 14).                   lyzed. yhdJ gene disrupted S. enterica serovar <br /> examined, CcrM essential viability, Dam              Typhimurium LT2 procedure Datsenko Wanner <br /> homologs essential species. Escherichia coli           (3). Disruption yhdJ strains confirmed Southern Salmonella enterica, Dam methyltransferase mutants                 blot analysis PCR (data shown). constructs <br /> pleiotropic defects viable.                                         lacking yhdJ viable, significant growth deficiency <br />    E. coli yhdJ gene, b3262, annotated putative             observed based doubling times Luria broth (LB)  methyltransferase-encoding gene (10), predicted pro-                example, doubling times BW25113 &#226;&#338;&#172;yhdJ::Kmr <br /> tein contains conserved methyltransferase motifs S-adeno- <br />                                                                             [JW5543] derivative [1] 49 min 52 min, respectively). <br /> syl-<span id='am-16' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-17' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span>L-methionine</span> (SAM) binding catalysis. order                                                                             difference cell morphology observed domains places YhdJ beta group methyltrans- <br />                                                                             yhdJ mutants isogenic wild-type strains. used <br /> ferases, CcrM homologs (12). E. coli YhdJ <br />                                                                             WA802 strain background  previous study, identity adenine methyltransferases M. AvaIII <br />                                                                             yhdJ deletion constructed background (6). <br /> (55.8%) CcrM Caulobacter crescentus (34.3%). Like <br />                                                                              yhdJ essential gene E. coli S. enterica. <br /> CcrM unlike M. AvaIII, yhdJ does appear                                                                                following experiments, derivatives  restriction-modification  gene encoding close proximity.                                              structed E. coli plasmid pCA24N, carries yhdJ- <br />    existence essential DNA methyltransferase E.                his6-gfp cloned isopropyl-&#226;?&#164;-D-thiogalactopyranoside <br /> coli significant implications understanding              (IPTG)-inducible promoter (5). pMV237 pCA24N DNA replication cell cycle control. recently pub-                NotI fragment contains gfp, pMV238, yhdJ- <br /> lished collection frame E. coli deletions contains yhdJ             his6 insert removed deleting SfiI fragment mutant, suggesting YhdJ essential (1),                pMV237, resulting  ? vector. <br /> previous report concluded gene deletion                    Based observations Kossykh Lloyd (6), strains <br /> obtained yhdJ expression essential viability               overexpressing YhdJ aberrant growth mor- <br /> (6). study, reexamine role function            phology.  examined induction ydhJ ex- yhdJ gene product.                                                   pression plasmid JW5543 (&#226;&#338;&#172;yhdJ::Kmr) (1) caused <br />                                                                             morphological changes. controls, used strain <br />                                                                             noninducing conditions, yhdJ plasmid <br />   * Corresponding author. Mailing address: Department Biology,           pMV238.  did observe change cell <br /> IIU, Area 12, University York, P.O. Box 373, York Y010 5YW, <br />                                                                             morphology result induction YhdJ. doubling <br /> England. Phone: 44 (0)1904 328841. Fax: 44 (0)1904 328844. E-mail: <br /> mvdw1@york.ac.uk.                                                           times wild-type YhdJ-overexpressing strains   &#228;&#338;&#164; <br />     Published ahead print 30 March 2007.                              similar, 5% (results shown). <br />  <br />                                                                      4325 <br />  4326     NOTES                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                         FIG. 2. vitro YhdJ-dependent methylation protects pACYC177 <br />                                                                      NsiI cleavage. Plasmid DNA incubated partially purified <br />                                                                      YhdJ indicated (lanes 1 4) digested NsiI (lanes 1 <br />                                                                      6). &#226;&#8364;&#339;G&#226;&#8364;? (lane 6) indicates addition YhdJ storage buffer; lane <br />                                                                      7 undigested plasmid. <br />  <br />   FIG. 1. Overexpression yhdJ protects DNA NsiI digestion. <br /> Total DNA isolated BW25113/pMV237(6xhis-yhdJ) (lanes 1      37&#194;&#176;C. results (Fig. 2) cleavage NsiI inhib- 3) BW25113/pMV238 (vector) (lanes 4 6) incubated <br />                                                                      ited concentration-dependent manner YhdJ (com- enzyme (lanes 1 4), EcoRV (lanes 2 5) positive control, NsiI (lanes 3 6).                     pare lanes 1 2 lanes 3 4), indicating                                                                      adenines recognition sequence 5&#226;&#172;&#732;-ATGCAT-3&#226;&#172;&#732;                                                                      modified (http://rebase.neb.com/cgi-bin/msget?NsiI). <br />                                                                         Methylation adenine shown directly automated <br />    YhdJ adenine DNA methyltransferase. previous              dye terminator sequencing, modification leads report indicated YhdJ methylation confers protection         increase signal complementary <span id='am-11' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span>thymidine</span> (2, 11). <br /> NsiI digestion (6). examine enzymatic activity     examine YhdJ-dependent modification, strains <br /> YhdJ, methylation putative target sequence exam-         constructed. kanamycin cassette deleted ined determining DNA susceptibility NsiI digestion,       JW5543 (BW25113 yhdJ::Kmr) (1) described (3), resulting vivo vitro. NsiI recognition sequence 5&#226;&#172;&#732;-ATG        MV1061. Strain MV1074 MV1061(&#226;&#338;&#172;yhdJ) pACYC177 <br /> CAT-3&#226;&#172;&#732;. 5&#226;&#172;&#732; adenine methylated        pMV237(6xhis-yhdJ), MV1081 MV1061(&#226;&#338;&#172;yhdJ) <br /> strands, cleavage NsiI reduced 50%. contrast,     pACYC177 pMV238 (vector). Plasmid DNA 3&#226;&#172;&#732; adenine methylated DNA strands, cleavage com-       isolated strains BW25113 containing <br /> pletely inhibited (http://rebase.neb.com/cgi-bin/msget?NsiI) (13).   pACYC177 (MV1070) grown LB presence 3 mM <br /> NsiI digestion patterns chromosomal DNA isolated             IPTG, region pACYC177 flanking NsiI sites stationary-phase cultures wild-type yhdJ deletion         sequenced. Examination sequencing chromatograms <br /> strains JW5543 GM8494 identical, protection         permitted detection N6-methyladenine, described digestion (results shown).            Rao Buckler-White (11). Sections representative trace <br /> analysis performed total DNA isolated culture          showing NsiI sequence presented Fig. 3. shown BW25113 carrying pMV237(yhdJ) grown 1 mM IPTG,               controls GATC sequences, methylation targets protection NsiI cleavage occurred (Fig. 1, lane 3). Pro-        Dam (14), plasmid pACYC177 isolated isogenic Dam&#226;&#171;&#185; <br /> tection NsiI digestion did occur strain         Dam&#226;&#171;&#186; hosts (SV5421 SV5422, respectively). rel- <br /> carried pMV238  vector) (Fig. 1, lane 6). results        ative increase signal 5&#226;&#172;&#732; thymidine residue specif- <br /> indicate level expression wild-type isolate grown   ically DNA incubated YhdJ indicates ad- conditions used low methylation      enine opposing strand  3&#226;&#172;&#732; adenine 5&#226;&#172;&#732;-ATG <br /> detected method.  YhdJ overexpression            CAT-3&#226;&#172;&#732; NsiI recognition sequence) methylated. Similar results <br /> lead methylation NsiI recognition sequence.                obtained second NsiI sequence pACYC177     confirm protection NsiI digestion conferred      shown). <br /> YhdJ, plasmid DNA methylated vitro partially                YhdJ nonessential DNA adenine methyltransferase. <br /> purified YhdJ. Strain DH5&#226;?&#163;(pMV237) grown LB               study provides lines evidence, based DNA 1 mM IPTG added optical density 600 nm 0.3.         digestion DNA sequencing, indicating YhdJ Cells harvested 3 h later, YhdJ partially           adenine methyltransferase.  contrast earlier <br /> purified GraviTrap (GE Healthcare). final frac-         report, analysis suggests YhdJ methylates second, <br /> tion dialyzed 50 mM Tris (pH 8), 1 mM EDTA, 1            3&#226;&#172;&#732;, adenine sequence 5&#226;&#172;&#732;-ATGCAT-3&#226;&#172;&#732; (Fig. 2 3). <br /> mM dithiothreitol (DTT), 100 mM NaCl stored               NsiI sequences pACYC177 flanked &#226;&#171;&#186;20&#194;&#176;C 50% glycerol. vitro methylation carried       identical nucleotides, likely  pACYC177, NsiI target sequences. meth-             contain, YhdJ recognition sequence. DNA methyl- <br /> ylation reaction consisted YhdJ  40 ng pACYC177,           transferase enzymes highly specific target se- <br /> 50 mM Tris (pH 8), 1 mM EDTA, 1 mM DTT, 100 mM NaCl,                 quence (http://rebase.neb.com/cgi-bin/msget?NsiI);  50 &#226;?&#174;M SAM. Methylation carried 37&#194;&#176;C 2 h           unlikely target sequence exists, stopped incubation 65&#194;&#176;C. NsiI (6 units) 10         completely ruled  absence detectable <br /> mM MgCl2 added, DNA incubated 3 h                YhdJ-dependent methylation bacterial cells (Fig. 1) suggests <br />  VOL. 189, 2007                                                                                                                     NOTES         4327 <br />  <br />  <br />                                                                       (reviewed reference 14).  avoid confusion, <br />                                                                       YhdJ referred M.EcoKCcrM (6). <br />  <br />                                                                         thank Richard Roberts (NEB) helpful comments Renata <br />                                                                       Kaminska help cell synchronization studies. <br />                                                                         research supported BBSRC (grant BB/C502849/1) <br />                                                                       (M.V.D.W.), Spanish Ministry Education Science                                                                       European Regional Fund (grants GEN2003-20234-CO6-03                                                                       BIO2004-03455-<span id='am-1' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span>-02) (J.C.), National Institutes Health grant <br />                                                                       GM63790 (M.G.M.). <br />  <br />                                                                                                       REFERENCES <br />                                                                        1. Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura, M. Baba, K.  <br />                                                                           Datsenko, M. Tomita, B. L. Wanner, H. Mori. 2006. Construction                                                                           Escherichia coli K-12 frame, single-gene knockout mutants: Keio <br />                                                                           collection. Mol. Syst. Biol. 2:2006.0008. <br />                                                                        2. Bart, , M. W. van Passel, K. van Amsterdam,  van der Ende. 2005. <br />                                                                           Direct detection methylation genomic DNA. Nucleic Acids Res. 33: <br />                                                                           e124. <br />                                                                        3. Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo- <br />                                                                           somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br />                                                                           Sci. USA 97:6640&#226;&#8364;&#8220;6645. <br />                                                                        4. DeWitt, S. K., E.  Adelberg. 1962. occurrence genetic trans- <br />                                                                           position strain Escherichia coli. Genetics 47:577&#226;&#8364;&#8220;585. <br />                                                                        5. Kitagawa, M., T. Ara, M. Arifuzzaman, T. Ioka-Nakamichi, E. Inamoto, H. <br />                                                                           Toyonaga, H. Mori. 2005. Complete set ORF clones Escherichia <br />                                                                           coli ASKA library. complete set E. coli K-12 ORF archive: unique <br />                                                                           resources biological research. DNA Res. 12:291&#226;&#8364;&#8220;299. <br />                                                                        6. Kossykh, V. G., R. S. Lloyd. 2004. DNA adenine methyltransferase                                                                           Escherichia coli cell cycle regulated essential viability. J. <br />    FIG. 3. 3&#226;&#172;&#732; adenine NsiI recognition sequence modi-       Bacteriol. 186:2061&#226;&#8364;&#8220;2067. (Retraction, 186:6340.) <br /> fied YhdJ. traces 5&#226;&#172;&#732;-ATGCAT-3&#226;&#172;&#732; sequence (left) plasmid    7. Lobner-Olesen, , O. Skovgaard, M. G. Marinus. 2005. Dam methyl- <br />                                                                           ation: coordinating cellular processes. Curr. Opin. Microbiol. 8:154&#226;&#8364;&#8220;160. <br /> pACYC177, isolated MV1070  , MV1081 (middle),                                                                        8. Marinus, M. G. 1973. Location DNA methylat <br /> </body></html>