<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>    Mapping identification essential gene <br />     functions X chromosome Drosophila <br />  <br />     Drosophila melanogaster genome consists                               large-scale P element insertion screen vital gene functions <br />     chromosomes contain 165 Mb DNA, 120 Mb                           BAC tiling map X chromosome. collection     euchromatic. Drosophila Genome Projects,                           501 X-chromosomal P element insertion lines used map <br />     collaboration Celera Genomics Systems, sequenced                        essential genes cytogenetically establish short sequence <br />     genome, complementing previously established                              tags (STSs) linking insertion sites genome.     physical genetic maps. addition, Berkeley                              distribution P element integration sites, identified <br />     Drosophila Genome Project undertaken large-scale                              genes transcription units expression <br />     functional analysis based mutagenesis transposable                          patterns P-element-tagged enhancers described     P element insertions autosomes.  present                           discussed. <br />  <br />                    INTRODUCTION <br /> Drosophila melanogaster model organism genome research 90 years, starting Sturtevant&#226;&#8364;&#8482;s <br /> demonstration recombination frequencies used          Fig. 1. Diagram showing contig distribution X chromosome. <br /> map genes linear order chromosome (Sturtevant,          Contigs indicated boxes schematic representation 1913). Physical maps relate genes physical sites        X chromosome. Contig 1 composed cosmids BACs. <br /> chromosomes (Bridges, 1935), precise <br /> mapping cloned DNA fragments situ hybridization polytene chromosomes, enabled chromosomal maps cloned genomic DNA segments (Wensink et al., 1974). achievements, development procedures molecular              termini cosmid clone inserts. sequence tags <br /> screening (Grunstein Hogness, 1975) assembly            correspond cosmid clones known cytological location, <br /> large chromosomal contigs (Bender et al., 1983) facilitated       apparently single-copy inserts members contigs <br /> systematic attempts construct P1-based physical maps         (Madueno et al., 1995). additional tags determined <br /> Drosophila genome. maps cover &#226;&#710;&#188;90% euchrom-               isolated cosmid clones known cytological location. <br /> atic genome punctuated short sequence tag (STS)             Additional probes derived sequences reported sites  average, 50 kb apart (Kimmerly et al., 1996).       databases, generally corresponding genes known cyto- recently, genome shotgun sequencing project,            logical map position sequences P element <br /> collaborative effort Celera Genomics Berkeley          insertions, known cytological position. Probes used <br /> Drosophila Genome Project (BDGP), provided 95%                   selected partly basis cytological position, order <br /> coverage Drosophila genome (Adams et al., 2000; Rubin           enhance distribution probe sequences et al., 2000b). addition, expressed sequence tag (EST) projects     chromosome. generated sequence information &gt;100 000 transcripts <br /> (Rubin et al., 2000a).                                                                Example inter-contig gaps <br />    assign function DNA sequences, BDGP                                                                        probes (ES00262, mapping 7B1-2, ES00263, <br /> undertaken large-scale functional analysis genome <br />                                                                        mapping polytene chromosome bands 7B1-6) fail hybridize gene-disruption projects based mutagenesis <br />                                                                        BAC clones library. probes derived transposable P element insertion. projects                                                                        mapped P1 clones. BAC clones hybridizing probes focused isolation essential genes autosomes.                                                                        ES00262 ES00263 observed. current collection strains disrupt 1200 different genes, <br />                                                                        explanation sequences probes <br /> representing &#226;&#710;&#188;30% estimated 3600 genes auto- <br />                                                                        derived incorrect polymorphism <br /> somes mutable easily scorable phenotypes, mainly <br />                                                                        Drosophila strains used construction sterility lethality (Spradling et al., 1995, 1999).                                                                         BAC P1 libraries. gap represents separation report large-scale P element insertion screen identify essen- <br />                                                                        contig 1 (1A 7A) contig 2 (7B 10A). gaps gener- <br /> tial genes X chromosome, construction                                                                        ally correspond intensely staining polytene chromosome X-chromosomal BAC tiling map, complements                                                                        bands (e.g. 10B, 11CD, 13A 17A) presumably <br /> ongoing studies Drosophila genome. <br />                                                                        represent long regions probes. number cases, <br />                                                                        clones repeatedly failed signals probes           RESULTS DISCUSSION                                        basis cytological location hybridization data, <br />                                                                        expected hybridize. Possible explanations <br />            Tiling map X chromosome                              include clones absent certain filters, cloned DNA <br />                                                                        segments bearing segmental deletion, failures physical map X chromosome constructed                  hybridization. <br /> hybridizing 521 probes high-density filters bearing EDGP BDGP BAC libraries. 5771 BAC clones identi-                          Generation P element insertions <br /> fied hybridization. contigs assembled using combination manual analysis hybridization data situ       euchromatin Drosophila X chromosome comprises <br /> hybridization. span X chromosome subtelo-           &#226;&#710;&#188;22 Mb DNA, containing 2182 predicted protein coding <br /> meric region 1A 20C near base chromosome arm          genes (Adams et al., 2000). estimated thirds (Figure 1; supplementary information available website          Drosophila genes obvious loss function phenotype <br /> http://www2.open.ac.uk/biology/research/molecular-genetics/            (Miklos Rubin, 1996), &#226;&#710;&#188;800 X-chromosomal genes EDGPmap.html). longest contig, contig 1, spans divisions 1&#226;&#8364;&#8220;6       mutable scorable phenotype sterility lethality. comprises cosmid clones (Siden-Kiamos et al., 1990)           order assess genes systematic manner, generated BAC clones. Divisions 1&#226;&#8364;&#8220;3 mapped yield template            collection lethal P element insertions X chromosome. <br /> clones sequencing (Benos et al., 2000, 2001),            crossing scheme generating X-chromosomal P{lacW} <br /> region BAC libraries screened principally link             insertion lines depicted Figure 2. Females reinsertion <br /> cosmid contigs.                                                        P{lacW} element, shown absence    Probe sequences selected sources.             dominant second chromosome marker Curly white+ eye <br /> EDGP cosmid mapping project determined sequence tags              colour P{lacW}, individually mated FM7c males. <br />  <br />  <br />                                                                                                             EMBO reports vol. 3 |  1 | 2002 35 <br />                                                                    scientific report <br />      Peter et al. <br />  <br />  <br />  <br />  <br />                                                                                        Fig. 3. P{lacW} distribution X chromosome. localization                                                                                        based molecularly defined integration site. Open bars represent     Fig. 2. Crossing scheme isolating lethal X-chromosomal insertion lines.        insertions given polytene region shaded bars indicate number <br />     sex chromosomes (left pair genotype) large                 genes affected. <br />     autosomes shown schematically females left. Relevant <br />     mutations labelled accordingly. P{lacW} insertion indicated     triangle white+ selectable marker represented darker <br />     chromosome. transposase source indicated double-headed <br />     arrow. Balancer chromosomes hatched. Absence crossed males <br />     F3 generation indicates potential lethal P{lacW} insertion.   contain second integration site containing     details, Methods. <br />                                                                                        integrations situ hybridization signal                                                                                        observed. cases, insertion sites &lt;50 kb <br />                                                                                        apart, likely limit resolution                                                                                        situ hybridization technique. closely linked <br />     57 105 crosses initially set 39 900 (69.9%)                          strains, transcription unit hit P elements, <br />     produced offspring. crosses, 501 strains (1.3%)                    indicative local hopping remobilization     derived hemizygously lethal semilethal                         starting P{lacW}. Collectively, results indicate &gt;90% <br />     (defined &lt;20% viability compared sibling balancer                        lines contain single P element insertions.      males). Taking account X chromosome represents                       collection P element insertion strains comparable     roughly seventh genomic target size, 1.3% corres-                       best P element collections generated <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> (Spradling et al., <br />     ponds frequency &#226;&#710;&#188;9% lethal P element insertions, rate                    1999). <br />     similar reported autosomal insertions comparable                    P{lacW} insertion associated lethal phenotype     P element vector (Cooley et al., 1988; Bellen, 1999). addition,                 causally linked excision P{lacW} restores wild type <br />     73 lines isolated carried visible mutation                function. tested total 206 lines combining trans- <br />     sterile sex.                                                                posase source P{ry+ &#226;&#710;&#8224;2-3}(99B) P{lacW} element <br />                                                                                        (Bellen et al., 1989). 134 strains (65.0%) produced fully viable <br />                 Distribution P element insertions                                   male progeny lost white+ gene, suggesting P <br />                                                                                        element excised precisely. 65% value regarded     insertion sites 401 strains determined situ                      minimum, reversion tests performed     hybridization polytene chromosome squashes. addition,                        10 females bearing P{lacW} insertion trans- <br />     DNA insertion sites isolated plasmid                        posase source. P elements difficult excise     rescue inverse PCR generate STS insertion                  exhibit low mobilization frequencies certainly <br />     lines. Figure 3 summarizes distribution P integrations                  missed type protocol. <br />     X chromosome, showing higher frequency     P{lacW} insertions regions closer telomere. observation                                Characterization lines <br />     remains valid considering non-redundant <br />     insertions. skewed distribution mirrored relatively             &#206;&#178;-galactosidase gene P{lacw} report expression <br />     distribution transcription units X chromosome.                   pattern adjacent genes, immunohistochemically stained <br />      likely inherent insertion site preference                  embryos anti-&#206;&#178;-gal antibody. 398 501 strains (79.4%) <br />     P elements (Liao et al., 2000) responsible observed                     expressed enzyme. considerably higher 64% <br />     distribution.                                                                      reported random autosomal insertion P element <br />        quality P element collection depends largely                    (Bier et al., 1989) result fact X-chromo- <br />     proportion strains harbour single P{lacW} insertions.                   somal lines selected insertions affecting essential genes. <br />     situ hybridizations polytene chromosomes carried                       Representative patterns line deposited     401 strains. 89.5% (359 lines) contain single insertions, 9.7%                 FlyView database, http://flyview.uni-muenster  (Janning, <br />     (39 lines) contain insertions 0.7%  lines) contain                   1997), searched various criteria. <br />     P{lacW} insertions.  14 strains                          lethal phase determined 497 501 lines.     showed second hybridization signal polytene region 3A,                     frequent stage development stops larval stage, <br />     unable confirm second insertion site molecular                      29.2% analyzed lines  Supplementary data).     means. Conversely, molecular analysis identified strains                     unexpected, homozygous mutants survive <br />  <br />  <br /> 36 EMBO reports vol. 3 |  1 | 2002 <br />                                                      scientific report <br />                                                                                         Gene functions Drosophila X chromosome <br />  stage relying maternal contribution gene           (Spradling et al., 1999). 60 called warm spots (2&#226;&#8364;&#8220;5 <br /> products deposited heterozygous mother. addition,        insertions gene; Spradling et al., 1999) correspond 169 X- <br /> majority lines probably carry null allele,         chromosomal insertions (34.5%), similar observed hypomorphic mutations.                                          second chromosomes (35.8 34.2%, respec- <br />    generated STS flanking P element insertion           tively). 23 hot spots X chromosome hit <br /> site 496 501 lines plasmid rescue           times, resulting total 225 insertions (45.9%). <br /> inverse PCR  sequences submitted EMBL nucleo-          frequencies second chromosome hot spots <br /> tide database; Supplementary data). integration sites          39.1 38.1%, respectively. genes hit determined BLAST search published Drosophila            frequently  Notch, 22 insertions; Trf2, 17; inx2, 15; ras, <br /> genomic sequence (Adams et al., 2000). cases gene         14; act5C, <span id='am-2' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="CTP" datatype="xsd:string"></span>ctp</span> trol 12  total, hot spots <br /> annotation supported EST data, unambiguously              contained 104 insertions, transcription units identify affected gene. predicted genes open             hit  summary, X-chromosomal collection, single <br /> reading frame annotated assumed insertion       hits represented hot spots represented, <br /> integrated vicinity putative transcription start site      compared autosomal P insertion collections. Taken <br /> upstream nearest translational start codon,       X-chromosomal genes mutated screen strong preference P integrations 5&#226;&#8364;&#178;-end genes          hit, average, 2.7 times, figures (Spradling et al., 1995).                                              autosomal collection 2.3 2.2, respectively. <br />    value collection  ,determined number new genes affected, briefly discuss mapping data. total 513 STS sequences generated <br />                                                                                            CONCLUSIONS 496 strains. strains, repeated attempts isolate        Saturation mutagenesis studies ratio lethal complementa- <br /> flanking sequences failed, 16 identification         tion groups polytene chromosomes bands 0.81 (Ashburner <br /> integration site uninformative, P element inserted    et al., 1999). estimate predicts &#226;&#710;&#188;820 essential genes repetitive DNA. 11 lines, insertion     X chromosome (1012 bands).  collection represents <br /> yoyo retrotransposon (Whalen Grigliatti, 1998).          mutations slightly &lt;25% X-chromosomal essential <br /> 497 STSs generated remaining 480 lines           genes similar data obtained autosomal <br /> derived unique X-chromosomal sequences. insertions           collection lethal P insertions (Spradling et al., 1999). &gt;7 kb away annotated gene unclear              P element screen X-chromosomal genes required adult P integration responsible observed lethal       viability BAC tiling map reported represent valu- <br /> phenotype. additional P insertions occurred vicinity      able tool Drosophila research community. gene known EP insertions distance            disruption collection provides opportunity link &#226;&#710;&#188;130 <br /> annotated gene, suggesting P elements inserted       X-chromosomal genes phenotype. majority putative promotor region unknown gene. Fourteen           lines available Bloomington stock center, P elements, representing gene pairs, affect genes.        situ hybridization data, enhancer trap cases, gene localized large intron    expression patterns strains deposited gene.                                                       FlyView database. Using resources, Drosophila <br />    183 P{lacW} insertions affect 52 different genes          research community utilize collection functional <br /> previously characterized molecular genetic             molecular characterization affected genes. BAC tiling <br /> level, e.g. Notch, pebbled short gastrulation. Phenotypically      map  hand, serve reference point characterized mutations previously unidentified genes          mapping sequences  importantly, essential form second group, consisting loci. trol,           source particular genomic fragment, e.g. germ line known zw1, 12 insertions stardust        transformation. combined efforts community insertion. largest group insertions disrupts genes          increase knowledge model organism Drosophila mutation previously reported. cases,          virtue evolutionary conservation molecular information available genes         genes processes, shed light human gene function. <br /> predicted algorithms (Adams et al., 2000). summary, 301 insertions affect 130 genes (including nested gene pairs) remain characterized.                                                                                                METHODS <br /> class transcription units code proteins putatively        Drosophila strains embryos. following fly strains <br /> involved signaling cascades, e.g. Rala, protein degradation,     (Lindsley Zimm, 1992; FlyBase, 1999) used Rpt4, intracellular transport, Ntf-2,         experiments. starting element, P{lacW} (Bier et al., 1989) <br /> cellular developmental processes.                                  originated male sterile insertion strains, 1040B    determine frequency genes hit,              1260A, contain single transposon chromosome 2. <br /> relied molecular information.  allelism easily        strain used supply transposase activity w; CyO/wgSp; determined genetic criteria, complementation tests            TM6/Sb P{ry+ &#226;&#710;&#8224;2-3}(99B) (Robertson et al., 1988). female X chromsome lethals possible             fertile FM6 female sterile FM7c chromosomes introducing duplication rescues lethality. Genes     employed X chromosome balancers. addition X chromosome hit represent 96 490            fertile sexes, FM6 advantage carrying white <br /> insertions (19.6%). compares rate 25.8 28.8%          null allele, facilitating detection low expression second chromosome collection, respectively            white+ marker gene. <br />  <br />  <br />  <br />     Crossing scheme  Figure 2)                                           <br />     P: w/w; P{lacW}/P{lacW} &#195;&#8212; w/Y; wgSp/CyO; ry Sb P{ry+ &#226;&#710;&#8224;2-3}(99B)/              <br />     TM6B                                                                      <br />     F1: FM6/FM6 &#195;&#8212; w/Y; P{lacW}/CyO; ry Sb P{ry+ &#226;&#710;&#8224;2-3}(99B)/+                 <br />     F2: w/FM6; CyO/+  P{lacW}  &#195;&#8212; FM7c/Y                       <br />     absence non-balancer bearing sons F3 generation              <br />     indicates potential lethal insertion X chromosome. <br />                                                                   <br />     Immunohistochemistry. Enhancer trap expression pattern                                                                                     <br />     analyzed immunohistochemistry. Embryos collected                                                                                   <br />     apple juice agar plates, fixed stained anti-&#206;&#178;-galactosidase              <br />     antibody (Cappel). secondary peroxidase-coupled antibody                <br />     biotinylated anti-rabbit Ig (Organon Teknika).                             <br />     Molecular analysis. DNA isolated adult flies                   <br />     QIAGEN DNeasy according instructions manufac-                    <br />     turer. isolation flanking sequences plasmid rescue                     <br />     (Pirrotta, 1987), EcoRI-digested DNA used, resulting                <br />     DNA 3&#226;&#8364;&#178;-end P{lacW}. Inverse PCR (Silver, 1991)                   <br />     performed essentially described BDGP webpage (http://          <br />     www.fruitfly.org/). construction BAC tiling map           <br />     application described supplementary information         <br />     website ftp://ftp.ebi.ac.uk/pub/databases/edgp/200111/bac_probes.txt;           <br />     P insertion data stored directory ftp://           <br />      <br /> </body></html>