<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Haploinsufficiency Flap endonuclease (Fen1) leads rapid tumor progression <br /> Melanie Kucherlapati*&#226;&#8364;&#160;, Kan Yang&#226;&#8364;&#161;, Mari Kuraguchi&#226;&#8364;&#161;, Jie Zhao&#194;&#167;, Maria Lia&#194;&#167;, Joerg Heyer&#194;&#182;, Michael F. Kane&#229;&#8218;&#168;, <br /> Kunhua Fan&#226;&#8364;&#161;, Robert Russell&#194;&#167;, Anthony M. C. Brown&#226;&#8364;&#161;**, Burkhard Kneitz&#226;&#8364;&#160;&#226;&#8364;&#160;, Winfried Edelmann&#194;&#167;, <br /> Richard D. Kolodner&#229;&#8218;&#168;, Martin Lipkin&#226;&#8364;&#161;, Raju Kucherlapati* <br /> *Departments Medicine Genetics Harvard Partners Center Genetics Genomics, Harvard Medical School, Boston, MA 02115; &#226;&#8364;&#161;Strang <br /> Cancer Research Laboratory Rockefeller University, New York, NY 10021; &#194;&#167;Department Cell Biology Institute Animal Studies, Albert <br /> Einstein College Medicine, 1300 Morris Park Avenue, Bronx, NY 10461; &#194;&#182;Memory Pharmaceuticals Corporation, Montvale, NJ 07645; &#229;&#8218;&#168;Ludwig <br /> Institute Cancer Research, University California San Diego School Medicine, La Jolla, CA 92093; **Department Cell Biology Anatomy, Weill Medical College Cornell University, New York, NY 10021; &#226;&#8364;&#160;&#226;&#8364;&#160;Physologische Chemie  Biozentrum Hubland, <br /> 97074 Wurzburg, Germany <br />  <br /> Contributed Richard D. Kolodner, 29, 2002 <br />  <br /> Flap endonuclease (Fen1) required DNA replication                     FEN1 functions processing 5&#226;&#172;&#732; ends Okazaki <br /> repair, defects gene encoding Fen1 cause increased                 fragments lagging strand synthesis long patch-base excision <br /> accumulation mutations genome rearrangements.                  type repair (32&#226;&#8364;&#8220;37). FEN1 protein binds proliferating nuclear cell <br /> mutations genes involved processes cause cancer              antigen (38&#226;&#8364;&#8220;39) potentially competes p21, xeroderma <br /> predisposition, investigated possibility Fen1                 pigmentosum gene product, 5&#226;&#172;&#732; methyl cytosine methyl transferase, <br /> function tumorigenesis gastrointestinal tract. Using gene           proteins specific binding motif, implying roles knockout approaches, introduced null mutation murine                FEN1 DNA replication, repair, epigenetic inheritance, cell <br /> Fen1. Mice homozygous Fen1 mutation obtained,                cycle control. Null mutations gene  referred rad27) <br /> suggesting absence Fen1 expression leads embryonic lethal-              Saccharomyces cerevisiae cause temperature-sensitive viability, <br /> ity. Fen1 heterozygous animals appear normal.                    increased sensitivity UV light, mutagen sensitivity, genomic combined mutation adenomatous polyposis coli               instability, plasmid loss, destabilization telomeric repeats <br /> (Apc) gene, double heterozygous animals increased numbers                nonviable combination mutations genes involved adenocarcinomas decreased survival. tumors              homologous recombination (40&#226;&#8364;&#8220;46). rad27 mutants exhibit mice microsatellite instability. copy Fen1            complex mutator phenotype. increased frequencies gene remained intact tumors, Fen1 haploinsufficiency appears            accumulating frame-shift mutations  lead rapid progression cancer.                                          suggested rad27 mutations cause partial MMR defect <br />                                                                               (41&#226;&#8364;&#8220;42, 47). rad27 mutants accumulate insertion mutations  <br /> T    forms human colorectal cancer,      frequent sporadic form. Germ line mutations Apc gene responsible rare cancer predisposition syn- <br />                                                                               result duplication sequences flanked repeated sequences <br />                                                                               deletion mutations result deletion sequences <br />                                                                               flanked repeated sequences (42), thought arise <br /> drome, familial adenomatous polyposis (FAP) (1&#226;&#8364;&#8220;7),                   improper processing Okazaki fragments (42, 48). Finally, <br /> additional genetic changes required tumor formation.                  rad27 mutants accumulate extensive genome rearrangements Mutations Apc genes involved tumor progression            suggested result errors DNA replication prerequisite majority sporadic colorectal tumors. Hered-           (43). phenotypes caused rad27 mutations make Fen1 itary nonpolyposis colorectal cancer (HNPCC) colorec-              potential candidate cancer susceptibility gene. <br /> tal cancer predisposition syndrome. Germ line mutations                   examine role FEN1 tumor initiation progression, <br /> DNA mismatch repair (MMR) genes Msh2 Mlh1 major                   generated mice carry null mutation Fen1 gene. cause HNPCC (refs. 8&#226;&#8364;&#8220;12; http:&#229;&#8230;&#190;&#229;&#8230;&#190;www.nfdht.nl&#229;&#8230;&#190;database&#229;&#8230;&#190;                     homozygosity Fen1 mutation leads early <br /> mdbchoice.htm). HNPCC cases accounted                      embryonic lethality. Mice heterozygous mutation <br /> missense, nonsense, frameshift, splice mutations             viable mild tumor predisposition phenotype. genes deletion mutations, primarily Msh2 (9, 13&#226;&#8364;&#8220;16). small <br />                                                                               proportion mice (17%) develop non-Hodgkin&#226;&#8364;&#8482;s lymphoma proportion HNPCC cases appear caused germ line <br />                                                                               B cell type, mice premature thymus involution. <br /> mutations MMR genes, Msh6 Pms2, muta- <br />                                                                               assess role Fen1 GI tumor progression, generated <br /> tions Msh6 familial non-HNPCC cases (17&#226;&#8364;&#8220;21). <br />                                                                               mice double heterozygotes mutations Apc Tumors HNPCC patients exhibit microsatellite insta- <br />                                                                               Fen1 genes. Mice heterozygous Apc mutation <br /> bility (MSI) (22&#226;&#8364;&#8220;26). MSI results expansion contraction mono- multinucleotide repeats failure repair insertion&#229;&#8230;&#190;          (Apc1638N) tumor predisposition phenotype. develop <br /> deletion mismatches DNA. large number HNPCC                  colonic polyps adenomas small intestine kindreds accounted mutations Msh2, Mlh1,                 MSI, median survival 13 mo. Apc1638N MMR genes, reports suspected HNPCC cases                   Fen1null combined, median survival resulting double mutations genes tumors              heterozygous mice reduced 9 mo, tumors MSI, raising possibility mutations genes               advanced, tumors MSI. wild-type (WT) copy involved MMR implicated cases (27&#226;&#8364;&#8220;30).                   Fen1 allele remains intact tumors. results suggest <br />    proportion sporadic tumors intestinal tract         haploinsufficiency Fen1 important contributor tumors sites exhibit MSI (16, 23, 24, 31), suggesting           gastrointestinal (GI) tumor progression mouse models. sporadic defects MMR genes play role initiation progression number tumor types.  <br />                                                                               Abbreviations: FAP, familial adenomatous polyposis; HNPCC, hereditary nonpolyposis <br /> mutations Msh2 Mlh1 silencing Mlh1 detected              colorectal cancer; MMR, mismatch repair; MSI, microsatellite instability; GI, gastrointesti- tumor types (8&#226;&#8364;&#8220;12). observation           nal; ES, embryonic stem; IVTT, vitro transcription translation; WT, wild type. <br /> suggests DNA repair genes involved                 reprint requests addressed. E-mail: mkucherlapati@rics.bwh. <br /> cases. gene Flap Endonuclease 1 (Fen1).                            harvard.edu. <br />  <br />  <br />  <br /> 9924 &#226;&#8364;&#8220;9929 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; July 23, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169;  15                                                               www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.152321699 <br />                                                                                   Fig. 2. Genotyping embryonic day 3.5 blastocysts PCR. Blastocysts                                                                                  placed distilled <span id='am-8' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span>, boiled 10 min, used directly substrate                                                                                  PCR reactions. 260-bp fragment amplified Neomycin gene                                                                                  Fen1null locus, 120-bp fragment amplified Fen1 WT locus. <br />  <br />  <br />  <br />                                                                                  sections flat mucosa adjacent tumors. Statistical analyses                                                                                  performed using Fisher exact probability &#226;?&#185;2 tests                                                                                  analysis tumor incidence Mann&#226;&#8364;&#8220;Whitney test Fig. 1. Strategy production Fen1null mutant mice.   Gene-           binomial exact calculation tumor multiplicity. <br /> targeting strategy. (B) Southern blot liver DNA Fen1 WT (12-kb <br /> fragment) Fen1null mutant mouse (6-kb fragment). DNA digested HindIII hybridized 1,800-bp probe 3&#226;&#172;&#732; flanking          MSI Analysis. DNA extracted tumor tissue used region Fen1 gene. (C) PCR analysis tail DNA Fen1 WT (WT, 2.2-kb   PCR amplification (49) limiting dilution conditions. Approx- <br /> fragment) Fen1null mutant mouse (1.7-kb fragment).                           imately 25&#226;&#8364;&#8220;30 reactions performed tumor DNA <br />                                                                                  sample dilution 10&#226;&#171;&#186;5. D7Mit91 dinucleotide repeat <br />                                                                                  locus (left: TCTTGCTTGCATACACTCACG right: GAGA- <br /> Materials Methods                                                            CAAACCGCAGTCTCCT) amplified using end-labeled <br /> Fen1 Gene Targeting. 9-kb fragment mouse genomic DNA                    left primer. Amplified products separated denaturing <br /> isolated BAC 325J22 (RP-22 BAC library) cloned                     polyacrylamide gel autoradiographed analysis. Instability <br /> pUC19. fragment contained Fen1 coding region, shown                   judged determining proportion PCR reactions DNA sequencing. 1.2-kb KpnI fragment located 5&#226;&#172;&#732; end                  contained microsatellite allele differed WT allele. flanking region deleted facilitate identification target vector insertion embryonic stem (ES) cells. ClaI                   Sequencing Remaining Fen1 Allele Tumors. Primer pairs HindIII sites encompass coding region 826                  constructed spanned Fen1 gene mouse&#229;&#8230;&#190; <br /> bp 3&#226;&#172;&#732; flanking sequence. ClaI site filled converted            human homology blocks implicated transcriptional control XbaI site. Fen1 coding region deleted                      (50). PCR products sequenced. <br /> digestion XbaI HindIII replaced neomycin <br /> gene control phosphoglycerol kinase promoter.                   Analysis Apc Truncation Mutations. Codons 677-1674 826 bp 3&#226;&#172;&#732; flanking region previously deleted reinserted            mouse Apc gene analyzed protein truncating mutations <br /> HindIII site using PCR product containing HindIII sites             PCR vitro transcription translation (IVTT) <br /> end. Proper orientation sequence confirmed. targeting                (51&#226;&#8364;&#8220;52). PCR amplification WT Apc allele performed <br /> vector (Fig. 1A) linearized KpnI transfecting ES cells.             stages eliminate coamplification inactivated <br /> G418-resistant clones screened PCR using primers                      Apc1638N allele. nanograms genomic DNA amplified <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     GENETICS targeting vector 3&#226;&#172;&#732; flanking region. 165 ES cell               10-&#226;?&#174;l reactions containing Pfu (Pyrococcus furiosus) DNA <br /> clones screened, 8 contained correctly targeted mutant allele                polymerase reaction buffer [20 mM Tris&#228;&#161;&#160;HCl (pH 8.8)&#229;&#8230;&#190;2 mM <br /> (Fen1null). Chimeric mice generated standard techniques                  MgSO4&#229;&#8230;&#190;10 mM KCl&#229;&#8230;&#190;10 mM (NH4)2SO4&#229;&#8230;&#190;0.1% Triton X-100&#229;&#8230;&#190;0.1 C57&#229;&#8230;&#190;BL6 blastocysts germ line transmission monitored                 mg&#229;&#8230;&#190;ml nuclease-free BSA] 0.05 units&#229;&#8230;&#190;&#226;?&#174;l Pfu turbo <br /> using coat color markers. Chimeric males bred C57&#229;&#8230;&#190;BL6                    (Stratagene). Cycling conditions cycle 94&#194;&#176;C 5 <br /> females offspring genotyped PCR (Fig. 1C) Southern                     min, followed 20 cycles 94&#194;&#176;C 1 min, 60&#194;&#176;C 1 min, hybridization (Fig. 1B). Offspring intercrossed required.                72&#194;&#176;C 5 min, final extension cycle 72&#194;&#176;C 5 min. <br />                                                                                  procedure resulted amplification products &#226;&#172;&#8225;3.2 kb. <br /> Generation Fen1null&#229;&#8230;&#190;Apc1638N Animals. Mice heterozygous               overlapping segments Apc gene covering codons <br /> Fen1null allele mated Apc1638N animals spontaneously              677-1223 1100&#226;&#8364;&#8220;1674 subsequently amplified develop intestinal colonic polyps colon cancer.                      aliquots reactions using pairs PCR primers <br /> Apc1638N allele C57&#229;&#8230;&#190;BL6 background. Fen1null mice                 specific IVTT. amplification 677-1223 fragment, mixed genetic background estimated 60% C57&#229;&#8230;&#190;BL6,                  forward primer 5&#226;&#172;&#732;-CGGGATCCTAATACGACTCACTAT- <br /> 37.5% 129&#229;&#8230;&#190;SV, 2.5% SJL&#229;&#8230;&#190;J. animals maintained                  AGGGAGACCACCATGGATGCATGTGGAACTTTGTGG <br /> Association Assessment Accreditation Laboratory                    reverse primer: 5&#226;&#172;&#732;-CGAACAGCTAGCATTAGATG- <br /> imal Care-approved facility barrier conditions.                            GAGGTACAGC used. amplification 1100&#226;&#8364;&#8220;1674 <br />                                                                                  fragment, forward primer: 5&#226;&#172;&#732;-GCGGATCCTAATACGACT- <br /> Analysis Tumors. mice killed, GI tract                     CACTATAGGGAGACCACCATGGGTATGATGATGT- <br /> removed, opened longitudinally, fixed 10% neutral-buffered                ATAGGTCAAGGGGAACCAGT reverse primer: 5&#226;&#172;&#732;- <br /> formalin. gross specimens examined dissecting                   CAACTTGCTAGCTCTGACCCCATCTCCAG used. <br /> microscope tumors, number location tumors                     PCR performed 20-&#226;?&#174;l reactions containing 1-&#226;?&#174;l aliquots recorded. Representative tumors chosen histological                stage reactions. Cycling conditions segments molecular analysis. histological diagnosis                  25 cycles PCR performed tissue sections hematoxylin eosin staining. Early invasive             annealing temperature 57&#194;&#176;C. microliter adenocarcinomas invaded submucosa,                  resulting PCR products directly used template 6-&#226;?&#174;l invasive adenocarcinomas invaded                IVTT reactions (TNT T7 Quick PCR DNA, Promega) <br /> muscularis  Microadenomas counted serial                 containing 2.5 &#226;?&#174;Ci [35S]-methionine (Amersham Pharmacia). <br />  <br /> Kucherlapati et al.                                                                                        PNAS &#229;&#8230;&#169; July 23, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169;  15 &#229;&#8230;&#169; 9925 <br />                                                                                   Table 2. Multiplicity GI tumors Fen1null mice <br />                                                                                                                                        Small           Large <br />                                                                                                         Overall       Stomach        intestine       intestine <br />  <br />                                                                                  WT                  0.00 &#226;&#171;&#190; 0.00     0.00 &#226;&#171;&#190; 0.00    0.00 &#226;&#171;&#190; 0.00    0.00 &#226;&#171;&#190; 0.00 <br />                                                                                  Fen1null            0.00 &#226;&#171;&#190; 0.00     0.00 &#226;&#171;&#190; 0.00    0.00 &#226;&#171;&#190; 0.00    0.00 &#226;&#171;&#190; 0.00 <br />                                                                                  Fen1null&#229;&#8230;&#190;Apc1638N   8.44 &#226;&#171;&#190; 7.13*    0.56 &#226;&#171;&#190; 1.13&#226;&#8364;&#160;   7.32 &#226;&#171;&#190; 6.20*   0.56 &#226;&#171;&#190; 1.01&#226;&#8364;&#160; <br />                                                                                  Apc1638N            7.13 &#226;&#171;&#190; 2.85     1.88 &#226;&#171;&#190; 2.10    5.00 &#226;&#171;&#190; 1.85    0.25 &#226;&#171;&#190; 0.71 <br />  <br />                                                                                     Statistical results comparison binomial exact calcula- <br />                                                                                  tion: compared Fen1null: *, P &#226;&#172;? 0.001; &#226;&#8364;&#160;, P &#226;&#172;? 0.02. significant difference <br />                                                                                  Fen1null&#229;&#8230;&#190;Apc1638N double heterozygous Apc1638N mice <br />                                                                                  Mann-Whitney test. Mean &#226;&#171;&#190; SD. <br />  <br />  <br />  <br />                                                                                  severely underrepresented (X2 &#226;&#171;&#189; 26.9, P &#226;&#172;? 0.05) (Fig. 2.).                                                                                  attempted establish cell lines 60 blastocysts obtained                                                                                  intercrosses; able establish 14 (23%).                                                                                  wild type heterozygotes, Fig. 3. Kaplan&#226;&#8364;&#8220;Meier survival plot Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;; Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;; <br />                                                                                  homozygous Fen1null allele. results suggest Fen1 <br /> Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186;; Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186; mice. Time death mice              expression necessary continued cell viability. moribund recorded. colors different genotypes number mice follows: green, (106) Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;; red, (161)         Properties Fen1null Heterozygotes. Mice heterozygous <br /> Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;; dark blue, (33) Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186;; light blue, (25)             Fen1null mutation viable, fertile, life span <br /> Fen1 &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186; Apc &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#186;.                                                                indistinguishable WT littermates.                                                                                  small groups mice Fen1null heterozygotes, mice <br />                                                                                  moribund. group seven mice, total reactions incubated 30&#194;&#176;C 1 h. Aliquots                    heterozygotes WT animals; IVTT reactions analyzed 12% SDS&#229;&#8230;&#190;PAGE                            heterozygotes moribund 11 mo age, fluorography. characterization tumor-specific                     moribund 5 mo age. mice mutations, PCR products cloned vector, indi-                     thymuses times smaller vidual clones screened IVTT identify mutations,                  WT littermates healthy animals comparable age. <br /> DNA sequence determined.                                                     Histological examination revealed severe atrophy thymus, <br />                                                                                  medulla cortex, overall reduction Results <br />                                                                                  density uniformity lymphocyte population. Homozygous Fen1null Animals Viable. generated mouse                   overall observations favor interpretation premature thy- <br /> ES cells copy Fen1 gene replaced null               mus involution mechanism accelerated loss individual <br /> allele entire gene completely deleted (Fig. 1 .               lymphocytes. second cohort contained 36 mice, 12 <br /> Correct gene targeting events identified Southern hybrid-                Fen1null heterozygotes. examined ization (Fig. 1B) PCR (Fig. 1C). obtained gene-                     average age 12.3 mo, heterozygotes non- <br /> targeted cell lines.  designated 126, 111, 102, 146,               Hodgkins lymphoma B cell type, used experiments. ES cells injected mouse                  Paneth cell hyperplasia small intestine. <br /> blastocysts generate chimeric mice. Chimeric mice cell lines <br /> 126 111 transmitted mutant allele (Fen1null)               Effect Fen1 Haploinsufficiency Apc1638N Mice. gener- <br /> germ line. resulting Fen1null heterozygotes viable                  ated mice heterozygous mutation Apc gene <br /> fertile. intercrossed offspring genotyped                 (Apc1638N) (53) heterozygous Fen1null. Kaplan&#226;&#8364;&#8220;Meier <br /> PCR. examined 219 mice 34 matings.  71                     survival curves different groups mice shown Fig. <br /> wild type 148 Fen1null heterozygotes. Fen1null homozy-                  3. Fen1null&#229;&#8230;&#190;&#226;&#171;&#185; mice survival curves indistinguishable <br /> gotes detected. results suggest expression                WT littermates. Apc1638N mice median survival <br /> Fen1 necessary normal embryogenesis, lack                     13 mo. addition Fen1null allele mice <br /> functional copies Fen1 gene leads embryonic lethality.                 reduces median survival 9 mo (P &#226;&#172;? 0.07). <br />    determine lack gene expression leads early                  Mice different genetic compositions killed exam- <br /> lethality, examined 37 blastocysts Fen1null intercrosses.            ined approximately 1 yr age. tumor incidence types detected blastocysts homozygous (Fen1                 tumors observed different types mice summarized <br /> &#226;&#171;&#185;&#229;&#8230;&#190;&#226;&#171;&#185;: Fen1 &#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#185;: Fen1 &#226;&#171;&#186;&#229;&#8230;&#190;&#226;&#171;&#186; &#226;&#171;&#189; 23:10:4), blastocysts                        Table 1. WT Fen1null heterozygous mice tumors <br />  <br />  <br />                        Table 1. Tumor incidence Fen1 mice <br />                                                                                                  , % mice tumors <br />                                                          Sex <br />                        Genotype (n)                     (M&#229;&#8230;&#190;F)          Age, mo            Overall               GI             Extra-GI <br />  <br />                        WT (7)                           1:6           12.6 &#226;&#171;&#190; 1.1         0 (0%)            0 (0%)              0 (0%) <br />                        Fen1null (12)                    1:2           12.3 &#226;&#171;&#190; 2.5         2 (17%)           0 (0%)              2 (17%) <br />                        Fen1null&#229;&#8230;&#190;Apc1638N (9)            1:0.5         11.1 &#226;&#171;&#190; 1.7         9 (100%)          9 (100%)*           0 (0%) <br />                        Apc1638N (8)                     1:0.6         13.0 &#226;&#171;&#190; 1.9         8 (100%)          8 (100%)            0 (0%) <br />  <br />                          n, number mice studied. Statistical results comparison Fisher exact probability test: compared                        Fen1null: *, P &#226;&#172;? 0.001. significant difference Fen1null&#229;&#8230;&#190;Apc1638N double heterozygous                        Apc1638N mice. Mean &#226;&#171;&#190; SD. <br />  <br />  <br /> 9926 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.152321699                                                                                             Kucherlapati et al. <br />  Table 3. Histologic types tumors Fen1null mice <br />                                       Apc1638N     Fen1null&#229;&#8230;&#190;Apc1638N <br />  <br /> Total  tumors           57              76  (%) tumors histological 36 (63%)        48 (63%) <br />     examination <br /> Histologic types tumors: <br />  <br /> Total                              36 (100%)        48 (100%) <br /> Malignant                           1 (2.8%)        12 (25%) <br />   Early invasive carcinoma          0 (0%)           2 (4.2%) <br />   Adenocarcinoma                    1 (2.8%)        10 (20.8%) <br /> Benign <br />   Villous-tubular adenoma           4 (11.1%)        7 (14.6%) <br />   Tubular adenoma                  25 (69.4%)       23 (47.9%) <br />   Microadenoma                      6 (16.7%)        6 (12.5%) <br />  <br />  <br />  GI tract. Apc1638N double heterozygotes      Fig. 5. Invasive adenocarcinoma small intestine Fen1nullApc1638N substantial tumor burden. Tumor multiplicity slightly higher    double mutant mouse. Tumor tissue composed neoplastic glands Fen1null&#229;&#8230;&#190;Apc1638N animals Apc1638N mice (Table 2).         irregular shape size infiltrating muscularis. Mucinous pools formed. <br /> range tumor multiplicity considerably greater Fen1null&#229;&#8230;&#190;     Ulceration shown surface tumor. Desmoplastic response Apc1638N mice Apc1638N mice  range tumors        observed near neoplastic glands invasive area. Hematoxylin&#229;&#8230;&#190;eosin &#226;&#171;&#187;100. animal Apc1638N mice 4&#226;&#8364;&#8220;13 Fen1null&#229;&#8230;&#190;Apc1638N <br /> double heterozygotes, range 3&#226;&#8364;&#8220;20. definitive difference      Remaining Fen1 Allele Tumors Intact. examined strains mice obtained histological     status WT Fen1 allele DNA tumors derived typing tumors (Table 3; Fig. 5). Fen1null&#229;&#8230;&#190;Apc1638N double       Apc1638N allele Fen1null heterozygous mice. DNA 15 <br /> heterozygous mice 9-fold greater incidence malignant          Fen1null Apc1638N tumors 5 Apc1638N tumors exam- <br /> tumors (early invasive invasive adenocarcinomas) compared          ined. entire coding region Fen1 gene additional Apc1638N heterozygotes (P &#226;&#172;? 0.0053). Apc1638N animals         820 bp 5&#226;&#172;&#732; untranslated region contains homology benign tumors (tubular adenomas) (3.13 &#226;&#171;&#190; 1.89)               blocks conserved mouse human Fen1 genes Fen1null&#229;&#8230;&#190;Apc1638N double heterozygotes (2.22 &#226;&#171;&#190; 1.48, P &#226;&#172;? 0.05).     analyzed DNA sequencing. cases, Fen1 gene increase number adenomas large intestine     intact detectable mutations (data shown). <br /> observed double heterozygous mice (0.56 &#226;&#171;&#190; 1.01) compared Apc1638N (0.25 &#226;&#171;&#190; 0.71, P &#226;&#172;? 0.05, Table 2).                             Apc Mutations Tumor DNA. examined fate WT <br />                                                                        Apc allele tumors groups mice. total 41 <br /> Tumors Fen1null&#229;&#8230;&#190;Apc1638N Double Heterozygous Animals         tumors analyzed Fen1null&#229;&#8230;&#190;Apc1638N mice, results <br /> MSI. determined tumors double heterozy-        compared obtained analysis 15 tumors <br /> gotes exhibited MSI using limiting dilution PCR. DNA       Apc1638N mice (51). IVTT analysis indicated 15 (37%) normal intestinal mucosa healthy C57&#229;&#8230;&#190;BL6 mice (0 60          tumors double heterozygotes Apc truncation <br />  <br />  <br />  <br />  <br />                                                                                                                                                          GENETICS <br /> PCR reactions positive), tumors Apc1638N mice (7     mutations region encoding codons 876-1274, se- <br />                                                                        quence alterations shown Table 4. general, Apc <br /> 91 PCR reactions positive) showed instability (Fig. 4 C). <br />                                                                        truncation mutations detected groups tumors point <br /> DNA Mlh1 tumors (42 62 PCR reactions positive)         mutations. Insertion&#229;&#8230;&#190;deletion mutations identified 2 15 tumors Fen1null&#229;&#8230;&#190;Apc1638N heterozygotes (93 123 PCR        tumors double heterozygous mice (Fig. 6). detected <br /> reactions positive) extensive MSI (Fig. 4 B D). Normal liver   insertion&#229;&#8230;&#190;deletions tumors Apc1638N animals. <br /> tissue Apc1638N mouse adenomas showed MSI <br /> (0 31 PCR reactions positive; data shown). Normal liver         Discussion <br /> tissue Fen1null Apc1638N mouse adenocarcinomas did     generated mice loss function mutation Fen1 MSI (0 30 PCR reactions).                                  gene examined phenotype caused mutation. Mice <br />  <br />  <br />  <br />  <br />                                                                                                Fig. 4. MSI Fen1nullApc1638N tumor DNA. Primers <br />                                                                                                D7Mit91 locus used amplify 262-bp <br />                                                                                                fragment limiting dilution conditions (10&#226;&#171;&#186;5). <br />                                                                                                  PCR amplifications (normal intestine)                                                                                                MSI. (B) PCR amplifications                                                                                                Fen1null Apc1638N tumor DNAs MSI indicated. <br />                                                                                                (C) PCR amplifications Apc1638N <br />                                                                                                tumor DNA MSI. (D) Thirty PCR amplifi- <br />                                                                                                cations Mlh1 tumor MSI. <br />  <br />  <br /> Kucherlapati et al.                                                                              PNAS &#229;&#8230;&#169; July 23, 2002 &#229;&#8230;&#169; vol. 99 &#229;&#8230;&#169;  15 &#229;&#8230;&#169; 9927 <br />                       Table 4. Sequence Apc mutations Apc1638N Fen1null&#229;&#8230;&#190;Apc1638N tumors <br />                      Codon      Mutation      Consequence            WT sequence*            Apc1638N      Fen1null&#229;&#8230;&#190;Apc1638N <br />  <br />                      874      C-T               Arg-Stop       TCA AAA CGA GGT CTG                &#226;&#8364;&#8220;                 1 <br />                      921      C-T               Arg-Stop       GCG GCA CGA AGA AGC              1                   1 <br />                      934      ins&#226;&#8364;&#160; TACA         Frameshift     AAC ACA TAC AAC TTC              1                   &#226;&#8364;&#8220; <br />                      939      G-T               Glu-Stop       AAG TCG GAA AAT TCA              1&#194;&#167;                  &#226;&#8364;&#8220; <br />                      941      C               Ser-Stop       GAA AAT TCA AAT AGG               &#226;&#8364;&#8220;                  1 <br />                      956      C-T               Arg-Stop       TAT AAA CGA TCT TCA               &#226;&#8364;&#8220;                  4 <br />                      982      G-T               Glu-Stop       TCA GTT GAA TCC TAT               &#226;&#8364;&#8220;                  1 <br />                      992      del&#226;&#8364;&#161; 8 bp &#226;&#171;&#185;     Frameshift     AAA TTTTGCAGTTAT GGT             1&#194;&#167;                  &#226;&#8364;&#8220; <br />                      995      del&#226;&#8364;&#161; 10 bp        Frameshift     AGT TATGGTCAGTAT CCA              &#226;&#8364;&#8220;                   1 <br />                      1047     G               Trp-Stop       GAA AGG TGG GCA AGA              1&#194;&#167;                  &#226;&#8364;&#8220; <br />                      1112     C-T               Arg-Stop       ACA AAT CGA ATG GGT               &#226;&#8364;&#8220;                  1 <br />                      1154     G-T               Glu-Stop       GAA GAA GAA GAA GAG               &#226;&#8364;&#8220;                  1 <br />                      1202     ins&#226;&#8364;&#160; ATCA         Frameshift     AAT TCA TCA GCA CAA               &#226;&#8364;&#8220;                  1 <br />                      1227     C-T               Gln-Stop       AAA AGG CAG AAT CAG               &#226;&#8364;&#8220;                  1 <br />                      1242     C-T               Gln-Stop       CAG ACT CAA AAA GGC               &#226;&#8364;&#8220;                  1 <br />                      1274     C               Ser-Stop       TGC AGT TCA TTA TCA               &#226;&#8364;&#8220;                  1 <br />                      Total                                                                      5                  15 <br />  <br />                       WT sequence surrounding mutation shown site mutation shown bold. <br />                      &#226;&#8364;&#160;ins, insertion. <br />                      &#226;&#8364;&#161;del, deletion. <br />                      &#194;&#167;Data Kuraguchi et al. (51). <br />  <br />  <br />  <br />  <br /> lacking copies Fen1 gene die early embryogenesis,            mice similar, appears significant increase rate <br /> indicating Fen1 gene product essential development.        progression tumors double mutant mice. relative paucity homozygous blastocysts,          times adenocarcinomas carrying Apc1638N <br /> failure establish Fen1null homozygous ES cell lines, suggests      mutation  results suggest reduction Fen1 does <br /> Fen1 expression necessary cell survival proliferation.     alter tumor initiation process profound effect role FEN1 DNA replication repair explain           tumor progression. tumors examined showed <br /> importance cell survival. Fen1 heterozygotes normal          mutation loss WT Fen1 allele. observation life span exhibit phenotypes. mice            consistent view complete absence Fen1 function <br /> showed features premature thymus involution                 lead impaired proliferation. clue mechanism result early T cell depletion. proposed function             Fen1 heterozygosity results rapid tumor progression came <br /> Fen1 nonhomologous end joining (54) V(D)J                     observation tumors double heterozy- <br /> combination explain observed result. proportion                gotes exhibit MSI level seen tumors Mlh1 Fen1 heterozygotes B cell lymphomas. precise               mutant mice; suggests reduction Fen1 expression <br /> mechanism Fen1 hemizygosity leads B cell lym-                  results impairment DNA repair. <br /> phomas needs investigation.                                           Fen1 heterozygotes tumor susceptibility    obtained strong evidence role Fen1 tumor            reduction copy number sufficient result advanced tumors <br /> progression. Fen1null mutation combined              Apc1638N background suggest conditions, <br /> Apc1638N mutation, double heterozygous mice exhibited signif-          haploinsufficiency Fen1 gene causes mutator phenotype icant differences heterozygous Apc1638N        mammalian cells. total 41 tumors analyzed Fen1null&#229;&#8230;&#190; <br /> mutation  double heterozygotes lower median                Apc1638N mice. (37%) Apc truncation muta- <br /> survival. average tumor burden groups              tions. mutations novel. include 4-bp <br />  <br />  <br />  <br />  <br />                                                                                                                 Fig. 6. Distribution Apc mutations <br />                                                                                                                 Fen1null Apc1638N tumor DNA <br />                                                                                                                 tumor DNA Apc1638N mice. <br />                                                                                                                 Diagram Apc codons 677 <br />                                                                                                                 1674 showing positions char- <br />                                                                                                                 acteristics mutations detected                                                                                                                 Fen1nullApc1638N (Upper) Apc1638N <br />                                                                                                                 (Lower) intestinal tumors. &#226;&#8364;&#353;, deletion; <br />                                                                                                                  , insertion; , substitution. sym- <br />                                                                                                                 bol represents independent muta- <br />                                                                                                                 tion. 15-aa  C)                                                                                                                 20-aa (1&#226;&#8364;&#8220; 4) &#226;?&#164;-catenin-binding repeats <br />                                                                                                                 SAMP (Ser, Ala, Met, Pro se- <br />                                                                                                                 quence) repeat (S1) segment                                                                                                                 Apc indicated. <br />  <br />  <br /> 9928 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.152321699                                                                                   Kucherlapati et al. <br />  insertion 10-bp deletion. truncation                                      individual components make complexes. Haploinsuf- <br /> mutations detected tumors similar alterations                                        ficiency component make difference cell <br /> previously identified remaining Apc allele tumors                                        undergoing normal replication;  cell cycle  <br /> Apc1638N mice, Fen1 heterozygous mice                                         turbed mutations oncogenes tumor suppressor genes (e.g., <br /> appreciable tumor burden, suggest Fen1 does play                                        Apc), additional levels gene products significant role GI tumor initiation. advanced nature                                     necessary accommodate change rates cell division. tumors double heterozygotes suggests greater role                                     critical product necessary replication repair Fen1 tumor progression.                                                                            sufficient quantity, result detrimental    maintenance genomic stability prime importance                                     genomic stability. Replication repair factors involved cell, surprising genes involved replication                             maintenance genomic stability, results described imply <br /> repair contribute process. observed MSI Fen1null&#229;&#8230;&#190;                                        quantitative measure expression gene <br /> Apc1638N haploinsufficient tumors consistent                                        products useful prognostic indicator, particularly genes <br /> increased accumulation frameshift mutations observed S.                                         controlling cell cycle concomitantly genes involved replica- <br /> cerevisiae rad27 mutants. Fen1 WT allele intact                                  tion repair. <br /> tumors suggests quantity product plays important <br /> role tumor progression. Eukaryotic replication initiates 2 &#226;&#171;&#187; 104                                work supported grants R.K. (CA84301-04 ES11040) 1 &#226;&#171;&#187; 105 sites human haploid genome. formation                                        R.D.K. (GM50006 ES11040) National Institutes competent replication complexes leading lagging strands                                    Health, fellowship M.K. American Association DNA depend concentration availability                                       Cancer Research&#226;&#8364;&#8220;Cancer Research Foundation America. <br />  <br />  <br />  1. Yang, K., Edelmann, W., Fan, K., Lau, K., Kolli, V., Fodde, R., Khan, P. M., Kucherlapati,        26. Terdiman, J. P., Gum, J. R., Jr., Conrad, P. G., Miller, G. , Weinberg, V., Crawley, S. C., <br />     R. <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Lipkin, M. (1997) J. Exp. Zool. 277, 245&#226;&#8364;&#8220;254.                                                    Levin, T. R., Reeves, C., Schmitt, , Hepburn, M., et al. (2001) Gastroenterology 120, <br />  2. Bussey, H. J. R. (1975) Familial Polyposis Coli (Johns Hopkins Univ. Press, Baltimore).               21&#226;&#8364;&#8220;30. <br />  3. Bodmer, W. F., Bailey, C. J., Bodmer, J., Bussey, H. J. R., Ellis, , Gorman, P., Lucibello,     27. Park, J. G., Vasen, H. F., Park, K. J., Peltomaki, P., Ponz Leon, M., Rodriguez-Bigas, <br />     F. C., Murday, V. , Rider, S. H., Scambler, P., et al. (1987) Nature (London) 328,                  M. , Lubinski, J., Beck, N. E., Bisgaard, M. L., Miyaki, M., et al. (1999) Dis. Colon <br />     614&#226;&#8364;&#8220;616.                                                                                              Rectum 42, 710&#226;&#8364;&#8220;715. <br />  4. Leppert, M., Dobbs, M., Scambler, P., O&#226;&#8364;&#8482;Connell, P., Nakamura, Y., Stauffer, D.,                  28. Wijnen, J., Khan, P. M., Vasen, H., van der Klift, H., Mulder, , van Leeuwen- <br />     Woodward, S., Burt, R., Hughes, J., Gardner, E., et al. (1987) Science 238, 1411&#226;&#8364;&#8220;1413.                Cornelisse, , Bakker, B., Losekoot, M., Moller, P. &amp; Fodde, R. (1997)  J. Hum. <br />  5. Groden, J., Thliveris, , Samowitz, W., Carlson, M., Gelbert, L., Albertsen, H., Joslyn,             Genet. 61, 329&#226;&#8364;&#8220;335. <br />     G., Stevens, J., Spiro, L., Robertson, M., et al. (1991) Cell 66, 589&#226;&#8364;&#8220;600.                        29. Wu, Y., Berends, M. J., Mensink, R. G., Kempinga, C., Sijmons, R. H., van Der Zee,  G., <br />  6. Nishisho, , Nakamura, Y., Miyoshi, Y., Miki, Y., Ando, H., Horii, , Koyama, K.,                   Hollema, H., Kleibeuker, J. H., Buys, C. H. &amp; Hofstra, R. M. (1999)  J. Hum. Genet. <br />     Utsunomiya, J., Baba, S., Hedge, P., et al. (1991) Science 253, 665&#226;&#8364;&#8220;669.                              65, 1291&#226;&#8364;&#8220;1298. <br />  7. Kinzler, K. W., Nilbert, M. C., Su, L., Vogelstein, B., Bryan, T. M., Levy, D. B., Smith,         30. Farrington, S. M., Lin-Goerke, J., Ling, J., Wang, Y., Burczak, J. D., Robbins, D. J. &amp; <br />     K. J., Preisinger,  C., Hedge, P., McKechnie, D., et al. (1991) Science 253, 661&#226;&#8364;&#8220;665.               Dunlop, M. G. (1998)  J. Hum. Genet. 63, 749&#226;&#8364;&#8220;759. <br />  8. Kuismanen, S. , Holmberg, M. T., Salovaara, R., la Chapelle,  &amp; Peltomaki, P.              31. Eshleman, J. R. &amp; Markowitz, S. D. (1995) Curr. Opin. Oncol. 7, 83&#226;&#8364;&#8220;89. <br />     (2000)  J. <br /> </body></html>