<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Aug. 2002, p. 4555&#226;&#8364;&#8220;4572                                                                                Vol. 184,  16 <br /> 0021-9193/02/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.184.16.4555&#226;&#8364;&#8220;4572.2002 <br /> Copyright &#194;&#169; 2002, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />   Genetic Footprinting Antimicrobial Drug Targets: Examples <br />                  Cofactor Biosynthetic Pathways <br />              Svetlana Y. Gerdes, Michael D. Scholle, Mark D&#226;&#8364;&#8482;Souza, Axel Bernal, Mark V. Baev, <br />                    Michael Farrell, Oleg V. Kurnasov, Matthew D. Daugherty, Faika Mseeh, <br />                           Boris M. Polanuyer, John W. Campbell, Shubha Anantha, <br />                              Konstantin Y. Shatalin, Shamim  K. Chowdhury, <br />                                Michael Y. Fonstein, Andrei L. Osterman* <br />                                             Integrated Genomics , Chicago, Illinois 60612 <br />                                               Received 17 January 2002/Accepted 14 2002 <br />  <br />              Novel drug targets required order design new defenses antibiotic-resistant pathogens. <br />           Comparative genomics provides new opportunities finding optimal targets previously unexplored <br />           cellular functions, based understanding related biological processes bacterial pathogens           hosts. integrated approach identification prioritization broad-spectrum drug targets. <br />           strategy based genetic footprinting Escherichia coli followed metabolic context analysis           essential gene orthologs various species. Genes required viability E. coli rich medium identified <br />           genome scale using genetic footprinting technique. Potential target pathways deduced           data compared panel representative bacterial pathogens using metabolic reconstructions <br />           genomic data. Conserved indispensable functions revealed analysis potentially represent <br />           broad-spectrum antibacterial targets. target prioritization involves comparison corresponding <br />           pathways individual functions pathogens human host. promising targets           validated direct knockouts model pathogens. efficacy approach illustrated using examples <br />           metabolism adenylate cofactors NAD(P), coenzyme  flavin <span id='am-37' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> dinucleotide. drug <br />           targets pathways, including distantly related adenylyltransferases (orthologs E. coli <br />           genes nadD, coaD, ribF), discussed  <br />  <br />  <br />   growing number antibiotic-resistant microbial patho-                  scale high-throughput random transposon mutagenesis. gens (44, 46; http://www.cdc.gov/narms/default.htm) presents                 major advantage technique parallel analysis challenge modern medicine. majority existing                 thousands genes multiple growth conditions. <br /> antibiotics utilize limited number core <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> structures                  transposon-based approach termed &#226;&#8364;&#339;genetic footprinting&#226;&#8364;? target cellular functions, cell wall bio-               originally described Saccharomyces cerevisiae (84, 85). <br /> synthesis, DNA replication, transcription, translation (67).               Genetic footprinting step process:   random trans- <br /> Identification unexplored cellular functions potential tar-              poson mutagenesis large number cells, (ii) competitive <br /> gets prerequisite development novel antibiotic che-                outgrowth mutagenized population various selec- <br /> motypes. Choosing optimal target function crucial step                 tive conditions, (iii) analysis individual mutants surviv- long expensive process drug development                      ing population using direct sequencing various hy- <br /> requires best possible understanding related biological                 bridization PCR-based techniques. Various modifications <br /> processes bacterial pathogens hosts.                              genetic footprinting recently applied   Extensive programs utilizing genomic information search                   microorganisms, including Mycoplasma genitalium Myco- novel antimicrobial targets launched recently                 plasma pneumoniae (49), Pseudomonas aeruginosa (99), Heli- <br /> industry academia (14, 15, 69, 72, 91). Complete genome                    cobacter pylori (52), Escherichia coli (7, 45). ver- <br /> sequences multiple bacterial species, including major                  sion method, termed genomic analysis mapping pathogens, available years,                   vitro transposition, developed Haemophilus projects way (16). abundance                       influenzae (1, 75) Streptococcus pneumoniae (1). Direct <br /> genomic data enabled development novel post-                        application technique usually limited microbial <br /> genomic experimental computational techniques aimed                     species natural competence, transposon mutagene- <br /> drug target discovery. Experimental approaches genome-                      sis performed vitro isolated DNA fragments, wide identification genes essential cell viability           mutations introduced genome transformation <br /> microbial species reviewed recently (23, 43, 54, 59,                 linear DNA fragments followed gene conversion. 67, 77, 79). techniques based systematic                   targeting specific genomic region, approach increases <br /> gene inactivation directed mutagenesis genome                    insertion density, improving resolution genetic footprinting. <br />                                                                                elegant extension method naturally compe- <br />   * Corresponding author. Mailing address: Integrated Genomics, , <br />                                                                                tent species described P. aeruginosa (99). <br /> 2201 W. Campbell Park Dr., Chicago, 60612. Phone: (312) 491-0846,              Genetic footprinting E. coli mini-Tn10 mutagenesis ext. 213. Fax: (312) 491-0856. E-mail: andrei@integratedgenomics.com.          vivo recently reported (7, 45). systematic <br />  <br />                                                                         4555 <br />  4556      GERDES ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br /> analysis E. coli gene essentiality, chosen ap-                       Transposome        formation       transposon      mutagenesis.     Plasmid <br /> proach based use &#226;&#8364;&#339;transposome,&#226;&#8364;? precut trans-                     pMOD&#226;&#172;?MCS&#226;&#172;&#381; containing artificial transposon EZ::TN&#226;&#172;?KAN-2&#226;&#172;&#381; (Epicentre <br />                                                                                 Technologies, Madison, Wis.) isolated strain (MG1655 poson DNA stable complex modified Tn5 transposase                     DH10B) subsequently mutagenized avoid restriction-modification <br /> (40, 47). approach advantages mini-Tn10                   problems. Transposon DNA released PvuII digestion, recommended <br /> mutagenesis:   single irreversible insertions pro-                   manufacturer, gel purified using QIAquick gel extraction columns <br /> duced, source transposase activity pro-                (Qiagen, Valencia, Calif.). Transposomes preformed incubating trans- <br />                                                                                 poson DNA (7 ng/&#226;?&#174;l) hyperactive Tn5 EZ::TN transposase (0.1 U/&#226;?&#174;l; <br /> tein transposome complex; (ii) need                                                                                 Epicentre Technologies generous gift W. Reznikoff) solution <br /> assemble elaborate transposon delivery vector                    containing 40 mM Tris-acetate (pH 7.5), 100 mM potassium glutamate, 0.1 mM <br /> tight regulation replication transposase expression;                 EDTA, 1 mM dithiothreitol, tRNA (0.1 mg/ml). Samples incubated (iii) limit insertion cell achieved based                30 min 37&#194;&#176;C dialyzed 10 mM Tris-acetate, pH 7.5, plus 1 mM ratio transposome complexes competent cells                    EDTA 0.05-&#226;?&#174;m filters (Millipore, Bedford, Mass.) 1 h. Dialyzed samples <br />                                                                                 mixed electrocompetent E. coli 1:2 ratio (vol/vol) transformed <br /> time transformation.                                                         electroporation. Cultures immediately diluted Luria-Bertani- <br />    Computational techniques identification potential                 based rich medium   kanamycin incubated 37&#194;&#176;C 40 <br /> drug targets based genomic data reviewed                        min gentle agitation. efficiency electroporation E. coli strains <br /> cently (35, 37, 67, 79). studies, silico analysis               MG1655 DH10B 5 &#226;&#171;&#187; 104 2 &#226;&#171;&#187; 106 kanamycin-resistant colonies successfully combined experimental techniques (4, 23,                      1 &#226;?&#174;g transposon DNA, respectively. <br />                                                                                    Outgrowth mutagenized population. Half mutagenized population <br /> 36). example, Arigoni coauthors used comparative                        immediately frozen stored time zero sample. rest genome analysis identify previously uncharacterized genes                    culture used inoculate BIOFLO 2000 fermentor (New Brunswick Sci- potential broad-spectrum targets emphasizing genes                        entific, Edison, N.J.) containing 950 ml following medium: tryptone, 10   broadly conserved various bacteria, including                  g/liter; yeast extract, 5 g/liter; 50 mM NaCl, 9.5 mM NH4Cl, 0.528 mM MgCl2, <br />                                                                                 0.276 mM K2SO4, 0.01 mM FeSO4, 5 &#226;&#171;&#187; 10&#226;&#171;&#186;4 mM CaCl2, 1.32 mM K2HPO4. <br /> pathogens; (ii) conserved humans; (iii) likely                                                                                 medium supplemented following micronutrients: 3 &#226;&#171;&#187; 10&#226;&#171;&#186;6 mM <br /> encode soluble proteins. essentiality selected genes                 (NH4)6(MoO7)24, 4 &#226;&#171;&#187; 10&#226;&#171;&#186;4 mM H3BO3, 3 &#226;&#171;&#187; 10&#226;&#171;&#186;5 mM CoCl2, 10&#226;&#171;&#186;5 mM CuSO4, assessed directed knockouts E. coli Bacillus                  8 &#226;&#171;&#187; 10&#226;&#171;&#186;5 mM MnCl2, 10&#226;&#171;&#186;5 mM ZnSO4 (68). following vitamins subtilis (4). silico target identification techniques               added (concentrations milligrams liter): biotin, 0.12; riboflavin, 0.8; <br /> focused formal comparative sequence analysis,                     pantothenic acid, 10.8; niacinamide, 12.0; <span id='am-42' about='obo:IMR_0200344' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00314" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Pyridoxine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200344" datatype="xsd:string"></span>pyridoxine</span>, 2.8; thiamine, 4.0; lipoic <br />                                                                                 acid, 2.0; <span id='am-58' about='obo:IMR_0200356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:id" content="IMR:0200356" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Folic acid" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00504" datatype="xsd:string"></span>folic acid</span>, 0.08; p-aminobenzoic acid, 1.37. Kanamycin added <br /> tempting assess conservation overall biological                   10 &#226;?&#174;g/ml. fermentation temperature held 37&#194;&#176;C, dis- <br /> text various pathogens human host. believe                   solved oxygen held 30 50% saturation, pH held 6.95 <br /> comparative analysis pathways biological subsystems                       titration 5% H3PO4). Media growth conditions designed significantly improve ability select potential targets.              minimize number genes required cell survival. Cells grown                                                                                 batch culture 23 population doublings (12 h) cell density 1.4 &#226;&#171;&#187; 109. <br />    approach identification pri- <br />                                                                                 Genomic DNA isolated used generate genetic footprints. <br /> oritization broad-spectrum antimicrobial targets known                     Detection transposon insertions nested PCR. pairs primers functional roles. use list essential genes inferred                  used consecutively, second pair primers nested  genetic footprinting model microorganism (E. coli)                    Fig. 2A). primer pair contained transposon-specific primer starting point analysis corresponding functional roles               chromosome-specific primer. Chromosome-specific landmark primers  <br />                                                                                 signed ordered set unidirectional primer pairs covering entire E. coli context metabolic pathways microbial pathogens.                    genome using custom software. Pairs separated average 3,500 bp, kind comparative analysis based metabolic recon-                     primers pair separated shortest possible distance <br /> struction genomic data indicates suitable target pathways                  range &#226;&#171;&#186;3 900 bp. Average primer length 27 bp (sequences available target functions pathways,                      request). Transposon-specific primers chosen avoid significant <br />                                                                                 similarity E. coli chromosome, using PrimerSelect software (DNA- <br /> potential range pathogens target. Comparison                                                                                 STAR, , Madison, Wis.). pairs nested, outwardly directed transposon- <br /> microbial pathways individual target proteins                    specific primers  end) used detect transposons inserted human counterparts allows target prioritization. Fi-                    orientations  Fig. 2A). forward primer pair includes external <br /> nally, prospective targets validated directed knockouts                  primer, 5&#226;&#172;&#732;-GTTCCGTGGCAAAGCAAAAGTTCAA-3&#226;&#172;&#732;, internal model pathogens (Staphylococcus aureus H. influenzae).                    primer, 5&#226;&#172;&#732;-GGTCCACCTACAACAAAGCTCTCATCA-3&#226;&#172;&#732;. reverse primer <br />                                                                                 pair includes external primer, 5&#226;&#172;&#732;-CCGACATTATCGCGAGCCCATTTAT- corresponding proteins cloned representative <br />                                                                                 3&#226;&#172;&#732;, internal primer, 5&#226;&#172;&#732;-GCAAGACGTTTCCCGTTGAATATGGC-3&#226;&#172;&#732;. <br /> pathogens human cDNA libraries, expressed analyzed                         consecutive PCR amplifications  external PCR)  advanced target proteins representa-                   performed following conditions: 95&#194;&#176;C 1 min; 94&#194;&#176;C 12 s, 70&#194;&#176;C tive bacterial sources, human counterparts,                6 min (2 cycles); 94&#194;&#176;C 12 s, 69&#194;&#176;C 6 min (2 cycles); 94&#194;&#176;C 12 s, 68&#194;&#176;C crystallized, structures determined assist                  6 min (36 cycles); 68&#194;&#176;C 6 min. Amplification reactions contained 0.3 &#226;?&#174;g                                                                                 template DNA (equivalent 6 &#226;&#171;&#187; 107 E. coli genomes), 0.2 mM concentration drug development.                                                       deoxynucleoside <span id='am-68' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span>triphosphate</span>, 0.4 &#226;?&#174;M concentration primer, <br />    illustrate approach using examples biosyn-                  PCR buffer (40 mM Tricine-KOH [pH 9.2], 15 mM potassium acetate, 3.5 mM <br /> thesis adenylate cofactors, NAD, coenzyme (<span id='am-63' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span>CoA</span>),                     magnesium acetate, bovine serum albumin [3.75 &#226;?&#174;g/ml]), 0.4 &#226;?&#174;l Advan- flavin adenine dinucleotide (FAD), presenting                       tage cDNA Polymerase Mix (Clontech Laboratories, Palo Alto, Calif.) 20 &#226;?&#174;l. <br />                                                                                 second internal PCR performed reaction mix, promising targets development novel broad-spec- <br />                                                                                 DNA templates consisted products PCR diluted 103-fold. <br /> trum antibiotics.                                                               Amplification conditions internal PCR follows: 95&#194;&#176;C 1 min; 94&#194;&#176;C <br />                                                                                 12 s, 69&#194;&#176;C 6 min (2 cycles); 94&#194;&#176;C 12 s, 68&#194;&#176;C 6 min (9 cycles); 68&#194;&#176;C <br />                                                                                 6 min. products internal PCRs (3-&#226;?&#174;l aliquots) size separated <br />                        MATERIALS METHODS <br />                                                                                 0.65% agarose gels. insert detection analysis procedures  <br />   Strains. E. coli strains MG1655 (F&#226;&#171;&#186; lambda&#226;&#171;&#186; ilvG rfb50 rph1) (53) DH10B   formed 96 format. <br /> [F&#226;&#172;&#732; mcrA &#226;&#338;&#172;(mrr-hsdRMS-msrBC) &#226;?&#190;80lacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;lacX74 endA1 recA1 deoR                  Image capture analysis performed 1D Image Analysis Software <br /> &#226;&#338;&#172;(ara-leu)7697 araD139 galU galK nupG rpsL] (42) S. aureus strain RN4220    (Eastman Kodak Company, Rochester, N.Y.). Mapping detected Tn5 <br /> (55) used work.                                                    insertions using custom software, calculates insert positions <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                                       4557 <br />  genome sequence using addresses internal landmark primers               TABLE 1. Genetic footprinting E. coli genes related size corresponding PCR products. Visualization insert locations           metabolism adenylate cofactors NAD(P), CoA, FAD using custom software integrated ERGO database. mi- <br /> crobial genome database WIT, predecessor ERGO, described                                                                     inserts <br />                                                                                        Biosynthetic                      ORF size       ORF strain: Conclusionb <br /> previously (71).                                                                                          Gene <br />                                                                                         subsystem                         (bp) <br />    Optimization experimental conditions low-noise detection transposon                                                       MG1655 DH10B <br /> insertions. Reaction conditions optimized using small set control genes known essentiality, order detect maximum number inserts,       NAD            nadB                 1,620           9        ND           N <br /> keeping level PCR-introduced noise low.  minimum                         nadA                 1,041          12        ND           N genomic DNA contained representative mix mutant chromo-                            nadC                   891           1         4           N <br /> somes consistently yielded reproducible patterns bands PCR                            ushA                 1,650           6        ND           N <br /> determined. Second, products external corresponding internal PCRs                          pnuC                   717           4        ND           N analyzed agarose gel. comparison used                           nadR                 1,251           6         4           N <br /> guide optimizing PCR parameters: cooling rate, number cycles                            pncA                   657           2        ND           N <br /> external versus internal PCR, rate PCR products&#226;&#8364;&#8482; dilutions. PCR                            pncB                 1,200           8        ND           N <br /> conditions fine-tuned yield minimum number false products,                             nadD (ybeN)            639           1         4           Ec  internal PCR bands lacking corresponding external PCR product.                            nadE                   825           0         1           E  genetic footprints 4-kb nadD locus generated independently                           nadF (yfiB)            876           0         0           E <br /> using identical template DNA samples different chromosome-specific <br /> nested primer pairs. footprints yielded coherent patterns, consistently       CoA            panD                   378            2        0           N <br /> visualizing 10 transposon inserts area (data shown). Finally, 12 random                    panB                   792            1        3           N <br /> internal PCR bands gel purified sequenced. bands originated                          panE                   909            5        2           N expected chromosomal loci. Comparison calculated insert loca-                         panC                   849            1        5           N <br /> tions sequencing results indicated mapping error introduced gel                         coaA                   948            0        0           E <br /> electrophoresis 4.5% size band.                                               coaBC (dfp)          1,290            0        0           E <br />    Metabolic reconstruction genomic sequence data. versions ge-                           coaD (kdtB)            477            0        0           E <br /> nomes used work follows: E. coli K-12 MG1655 (18) (GenBank                             coaE (yacE)            618            0        0           E <br /> accession  U00096), P. aeruginosa PAO1 (88) (GenBank accession                                  acpS                   378            0        0           E <br /> AE004091), Bacillus anthracis Ames (http://www.tigr.org/tdb/mdb/mdbinprogress <br /> .html), Mycobacterium tuberculosis H37Rv (24) (GenBank accession  AL123456),        FMN/FAD ribA                          588            0        0           E <br /> H. pylori J99 (3) (GenBank accession  AE001439), S. aureus COL (http://www.tigr             ribD                        1,101            0        0           E <br /> .org/tdb/mdb/mdbinprogress.html/), S. pneumoniae (GenBank accession                         ribB                          651            0        0           E <br /> AE005672), M. genitalium (34) (GenBank accession  L43967), H. influenzae Rd                 ribH                          468            0        0           E <br /> (32) (GenBank accession  L42023), Chlamydia trachomatis (87) (GenBank accession             ribE                          639            0        0           E  AE001273), Homo sapiens (http://www.ncbi.nlm.nih.gov/genome/guide/                     ribF (yaaC)                   939            0        0           E <br /> human/).                                                                                                                                                                              number transposon insertions gene strains MG1655    Comparisons specific biochemical capabilities E. coli, relevant bac-      DH10B detected logarithmic aerobic outgrowth 23 population dou- <br /> terial pathogens, human host based metabolic reconstructions           blings rich medium shown. <br /> performed ERGO database. Metabolic reconstructions ERGO                     b <br />                                                                                             Assessments gene essentiality, based results genetic footprinting tentative projections known biochemical pathways specific organism       strains MG1655 DH10B. E, essential; N, nonessential; ND, determined. <br />                                                                                           c completely sequenced annotated genome (described reference 82).               nadD appeared nonessential genetic footprinting experi- projections based presence absence orthologs specific        ments shown essential directed-knockout technique. <br /> genes known, basis studies organisms, involved corresponding pathways. approach produces estimate metabolic <br /> potential given organism  asserted pathways actually expressed specific conditions) direct experimental <br /> data falls short defining actual metabolic fluxes.                               quence, using database size 106 high-cutoff E value 104.    Examples illustrating type analysis given Tables 2 4    resulting E values indication closely bacterial sequences metabolic subsystems (groups pathways) related biosynthesis ade-          related remote human sequence conserved core nylate cofactors NAD(P), CoA, FAD. define pathway series              bacterial group  higher E value  higher divergence . <br /> consecutive transformations bifurcations (82). Merging alterna-            Verification gene essentiality E. coli. direct verification gene <br /> tive pathways (numbered II, , Table 2) combined higher             essentiality E. coli used pKO3-based allelic exchange (58). briefly <br /> hierarchical blocks, ultimately form subsystem. contrast pathways,       illustrated using nadD example. Deletion gene (predicted subsystems formally defined boundaries, assembled             essential) attempted parallel nadR  nonessential gene based conventions field  cofactor biosynthesis described          metabolic subsystem) control. E. coli nadD (12, 63)  references 5, 12, 13), biological expertise interests   ybeN) nadR genes flanking regions PCR amplified using <br /> particular user.                                                                       MG1655 genomic DNA template following primers: 5&#226;&#172;&#732;-CAGCTG <br />    Profile HMMs. comparisons human bacterial sequences                 ATTCGTAAGCTGCCAAGCATC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GGGGTCGACTCACTCACGG using HMMER 2.2 program (30; http://hmmer.wustl.edu/),              TGATAAGGATGGTTGGTGGTGATG-3&#226;&#172;&#732; nadD 5&#226;&#172;&#732;-TGGCCTGCCG <br /> multiple-sequence alignments obtained CLUSTAL-W (90). built            ACTGACAATCTC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GCTGGAAAACGCCCTGCTGGAGT-3&#226;&#172;&#732; profile hidden Markov model (HMM) target enzyme, using ortholog           nadR. PCR products cloned gene replacement vector pKO3 <br /> sequences pathogens  specified Fig. 5A. HMMs            (58).  650-bp HincII-ClaI fragment nadR replaced 1.7-kb following enzymes built using length protein sequences            Eco57I DNA fragment containing tetracycline resistance cassette <span id='am-104' about='obo:IMR_0200415' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-105' property="rdfs:label" content="Nicotinic acid" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:id" content="IMR:0200415" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="KEGG:C00253" datatype="xsd:string"></span>nicotinic acid</span> mononucleotide adenylyltransferase (NaMNAT), NAD synthetase             pACYC184 (GenBank accession  X06403). Similarly, 434-bp ApaLIBglII <br /> (NADS), NAD kinase (NADK), pantothenate kinase (PK), phosphopantetheine                fragment nadD excised replaced kanamycin resistance cas- <br /> adenylyltransferase (PPAT), dephospho-CoA kinase (DPCK). Protein                sette pUC4K (GenBank accession  X06404). E. coli strain MG1655 mains roughly corresponding <span id='am-86' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-87' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-91' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> residues 1 212 213 413          transformed deletion plasmids. Following cointegration, resolution, E. coli CoaBC protein used phosphopantothenoylcysteine             elimination plasmids, Kmr Sucr colonies screened sensitivity <br /> carboxylase (PPCDC) phosphopantothenoylcysteine synthetase (PPCS)                  chloramphenicol (58). Final verification gene replacement HMMs, respectively. Likewise, protein domains approximately corresponding           PCR analysis primers flanking region recombination. <br /> amino acid residues 1 176 177 214 E. coli RibF protein used         case nadR, gene replacement readily achieved: 98% Kmr Sucr build profile HMMs FAD synthase (FADS) flavokinase (FK), respec-             colonies Cms, Kmr Sucr Cms colonies tested PCR <br /> tively. Obtained HMMs run bacterial human se-               contained expected deletion chromosome. case nadD, Kmr <br />                                                                                                                                                                                                                                                                                   4558 <br />                                                          TABLE 2. Functional reconstruction metabolic pathways NAD/NADP biosynthesisa <br />                                                                                                                                                        Presence ortholog pathway genome  <br />  <br />  <br />                                                                                          Protein <br />                     NAD(P) biosynthesis                                EC  <br />                                                                                          example <br />  <br />  <br />  <br />  <br />                                                                                                             E. colib <br />                                                                                                                         P. aeruginosa <br />                                                                                                                                         B. anthracis <br />                                                                                                                                                          M. tuberculosis <br />                                                                                                                                                                            H. pylori <br />                                                                                                                                                                                        S. pneumoniae <br />                                                                                                                                                                                                        S. aureus <br />                                                                                                                                                                                                                    M. genitalium <br />                                                                                                                                                                                                                                    H. influenzae <br />                                                                                                                                                                                                                                                    C. trachomatis <br />                                                                                                                                                                                                                                                                     H. sapiens <br />                                                                                                                                                                                                                                                                                  GERDES ET AL. <br />  <br />  <br />  <br />  novo pathways NaMN                                                                                  Yes          Yes              Yes             Yes                Yes                                                                       Yes <br />    Tryptophan quinolinate                                                                            &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  (II) Aspartate quinolinate <br />     Aspartate oxidase                                                1.4.3.16         gi&#229;&#8230;&#169;16130499         nadB         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     Quinolinate synthase                                                              gi&#229;&#8230;&#169;16128718         nadA         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />  Quinolinate NaMN <br />     Quinolinate phosphoribosyl transferase                           2.4.2.19         gi&#229;&#8230;&#169;16128102         nadC         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Salvage: niacin NaMN/NMN                                                                          Yes          Yes              Yes             Yese                        Yes             Yes         Yes                                        Yes <br />     <span id='am-93' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-94' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span>Nicotinamide</span> NaMN <br />      Nicotinamide deamidase                                          3.5.1.19         gi&#229;&#8230;&#169;140602           pncA         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />      Nicotinate phosphoribosyl transferase                           2.4.2.11         gi&#229;&#8230;&#169;16128898         pncB         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />   (II) Nicotinamide NMN <br />      Nicotinamide phosphoribosyl transferase (nadVc)                 2.4.2.12         gi&#229;&#8230;&#169;13629024         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Salvage: exogenous NMN NAD                                                                        Yes          Yes                                                                                       Yes                               NMN phosphohydrolase (extracellular) (nadNd)                     3.1.3.5          gi&#229;&#8230;&#169;1573165          ushA         &#226;&#171;&#186;                &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />     Pyridine <span id='am-81' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> transporter                                                   gi&#229;&#8230;&#169;16273640         pnuC         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     Nicotinamide ribose kinasef                                      2.7.1.22         gi&#229;&#8230;&#169;16132207         nadR         &#226;&#171;&#185;                &#226;&#171;&#186;               &#226;&#180;&#8482;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />     NMN adenylyltransferase <br />          NMN specific (archaea, bacteria)                     2.7.7.1          gi&#229;&#8218;&#168;16132207         nadR         &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#180;&#8482;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />        (ii) NaMN/NMN specific (eukarya)                              2.7.7.1/18       gi&#229;&#8230;&#169;11245478         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />  <br /> Common pathway NAD/NADP                                                                                Yes          Yes              Yes             Yes                Yes         Yes             Yes         Yes             Yes                           Yes <br />   NaMN NAD <br />    NaMN adenylyltransferase <br />        NaMN specific  bacteria)                               2.7.7.18         gi&#229;&#8230;&#169;1723307          nadD         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />      (ii) NaMN/NMN specific (eukarya)                                2.7.7.1/18       gi&#229;&#8230;&#169;11245478         &#226;&#171;&#186;            &#226;&#171;&#186;                &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />    NAD synthetase                                                    6.3.5.1          gi&#229;&#8230;&#169;16129694         nadE         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />   NAD NADP <br />    NAD kinase                                                        2.7.1.23         gi&#229;&#8230;&#169;8489010          nadF         &#226;&#171;&#185;                &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />        Subsystems, pathways, individual functional roles (specified Enzyme Classification [EC] numbers) involved adenylate cofactor biosynthesis relevant organisms presented nodes branches (rows) hierarchical tree. Proteins shown representative examples (GenBank <span id='am-103' about='oboInOwl:id' typeof='owl:Thing'>ID</span> numbers). presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) orthologous gene specific genome marked column corresponding organism, E. coli, actual gene names given. Alternative pathways subsystem indicated uppercase roman numerals. Nonorthologous genes functional role shown separate rows indicated lowercase roman numerals. Bifunctional (fused) proteins functional domain shown boldface type. &#226;&#8364;&#339;Yes&#226;&#8364;? indicates particular subsystem asserted given organism presence required genes (enzymatic functions) genome.  ? indicates particular subpathway asserted organism basis genomic data  Functional <br /> roles inferred metabolic context analysis associated specific genes (&#226;&#8364;&#339;missing genes&#226;&#8364;?) indicated question marks. <br />    b <br />      Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />    c <br />      Identified Haemophilus ducreyi; present V factor-independent Pasteurellaceae, H. influenzae (62). <br />    d <br />      Identified H. influenzae (76); closest homolog E. coli ushA. <br />    e <br />      Experimental data suggest pathway functional M. tuberculosis (83). <br />    f <br />      corresponding function E. coli H. influenzae catalyzed C-terminal domain multifunctional protein NadR (O. Kurnasov et al., unpublished data). <br />                                                                                                                                                                                                                                                                                  J. BACTERIOL. <br />  4559 <br /> GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS <br />  <br />  <br />  <br />  <br />                                                                                                              TABLE 3. Functional reconstruction metabolic pathways CoA biosynthesisa <br />                                                                                                                                                                                                        Presence ortholog pathway genome  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                          M. tuberculosis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                        S. pneumoniae <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                    C. trachomatis <br />                                                                                                                                                                                                                                                                    M. genitalium <br />                                                                                                                                                                         P. aeruginosa <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                    H. influenzae <br />                                                                                                                                           Protein <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                         B. anthracis <br />                                                                         CoA biosynthesis                                 EC  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                     H. sapiens <br />                                                                                                                                           example <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        S. aureus <br />                                                                                                                                                                                                                            H. pylori <br />                                                                                                                                                              E. colib <br />                                                  novo pathway pantothenate                                                                           Yes          Yes             Yes             Yes                Yes                      Yes                                                                                                   Aspartate &#226;?&#164;-alanine <br />                                                      Aspartate 1-decarboxylase                                         4.1.1.11        gi&#229;&#8230;&#169;1786323          panD         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                   Ketovalerate pantoate <br />                                                      3-Methyl-2-oxobutanoate hydroxymethyl transferase                 2.1.2.11        gi&#229;&#8230;&#169;1786326          panB         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                      2-Dehydropantoate 2-reductase <br />                                                          2-Dehydropantoate 2-reductase                               1.1.1.169       gi&#229;&#8230;&#169;1100871          panE         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                        (ii) Ketol-acid reductoisomerase                                1.1.1.86        gi&#229;&#8230;&#169;146477           ilvC         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                   &#226;?&#164;-Alanine, pantoate pantothenate <br />                                                      Pantoate-&#226;?&#164;-alanine ligase                                         6.3.2.1         gi&#229;&#8230;&#169;1786325          panC         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                  Pantothenate (<span id='am-19' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>vitamin</span> B5) salvage                                                                         Yes          Yes             Yes             Yes                Yes         Yes             Yes                      Yes                           Yes <br />                                                      Sodium/pantothenate symporter                                                     gi&#229;&#8230;&#169;455654           panF         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                  Common pathway CoA                                                                                     Yes          Yes             Yes             Yes                Yes         Yes             Yes                      Yes                           Yes <br />                                                     Pantothenate kinase                                                                                                 ?f                                                 ?f <br />                                                         bacteria                                                2.7.1.33        gi&#229;&#8230;&#169;1790409          coaA         &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#185;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                       (ii) Eukarya                                                     2.7.1.33        gi&#229;&#8230;&#169;6320740c         &#226;&#171;&#186;            &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Phosphopantothenoylcysteine synthetase                             6.3.2.5         gi&#229;&#8230;&#169;1790070          coaBe        &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#171;&#185;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Phosphopantothenoylcysteine decarboxylase                          4.1.1.36        gi&#229;&#8230;&#169;1790070          coaCe        &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#171;&#185;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#180;&#8482;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />                                                     Pantetheine-phosphate adenylyltransferase <br />                                                         Bacteria                                                     2.7.7.3         gi&#229;&#8230;&#169;1790065          coaD         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#186;                &#226;&#171;&#186; <br />                                                       (ii) Eukarya                                                     2.7.7.3         gi&#229;&#8230;&#169;632171d          &#226;&#171;&#186;            &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#180;&#8482; <br />                                                     Dephospho-CoA kinase                                               2.7.1.24        gi&#229;&#8230;&#169;1786292          coaE         &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#180;&#8482; <br />                                                                                                         Table 2, footnote  details. <br />                                                    b <br />                                                      Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />                                                    c <br />                                                      Eukaryotic pantothenate kinase (21), structurally unrelated corresponding E. coli enzyme (CoaA family). <br />                                                    d <br />                                                      Eukaryotic pantetheine-phosphate adenylyltransferase distantly related corresponding bacterial enzymes  recognition Psi-Blast) (27); forms fusion protein dephospho-CoA kinase                                                  higher eukaryotes. <br />                                                    e <br />                                                      bacteria  S. pneumoniae set), CoaB C-terminal domain, CoaC N-terminal domain bifunctional protein. <br />                                                    f <br />                                                      Pantothenate kinase &#226;&#8364;&#339;missing gene&#226;&#8364;? number bacterial pathogens. <br /> VOL. 184, 2002 <br />                                                                                                                                                                                                                                                                                                        4560 <br />                                                                                                                                                                                                                                                                                                       GERDES ET AL. <br />  <br />  <br />  <br />  <br />                                                          TABLE 4. Functional reconstruction metabolic pathways FMN/FAD biosynthesisa <br />                                                                                                                                                                           Presence ortholog pathway genome  <br />  <br />  <br />                                                                                                            Protein <br />                               FMN/FAD biosynthesis                                         EC           example <br />  <br />  <br />  <br />  <br />                                                                                                                             E. colib <br />                                                                                                                                            P. aeruginosa <br />                                                                                                                                                            B. anthracis <br />                                                                                                                                                                               M. tuberculosis <br />                                                                                                                                                                                                 H. pylori <br />                                                                                                                                                                                                             S. pneumoniae <br />                                                                                                                                                                                                                             S. aureus <br />                                                                                                                                                                                                                                         M. genitalium <br />                                                                                                                                                                                                                                                         H. influenzae <br />                                                                                                                                                                                                                                                                         C. trachomatis <br />                                                                                                                                                                                                                                                                                          H. sapiens <br />  <br />  <br />  <br />  novo pathways riboflavin                                                                                             Yes             Yes             Yes               Yes                Yes         Yes             Yes                      Yes             Yes               Ribuloso-5-phosphate L-3,4-dihydroxy-2-butanone-4-phosphate <br />     3,4-Dioxy-2-butanone-4-phosphate synthetase                                           4.1.2.-        gi&#229;&#8230;&#169;1789420        ribBc           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#171;&#185;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />  <span id='am-11' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span> 5-amino-6-ribitylamino-2,4-(1H,3H)-pyrimidinedione <br />                                                                                                                                        c <br />     GTP cyclohydrolase II                                                                 3.5.4.25       gi&#229;&#8230;&#169;16129238       ribA            &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#171;&#185;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine deaminase                                                                  3.5.4.26       gi&#229;&#8230;&#169;1786616        ribDd           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#353;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine reductase                                                                  1.1.1.193      gi&#229;&#8230;&#169;1786616        ribDd           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#353;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />     Pyrimidine phosphatase                                                                3.1.3                            ?               ?               ?                 ?                  ?           ?               ?           &#226;&#171;&#186;               ?               ?                &#226;&#171;&#186; <br />  L-3,4-Dihydroxy-2-butanone-4-phosphate, 5-amino-6-ribitylamino-2,4- <br />        (1H,3H)-pyrimidinedione riboflavin <br />     6,7-Dimethyl-8-ribityl-lumazine synthase                                              2.5.1.9        gi&#229;&#8230;&#169;16128400       ribH            &#226;&#171;&#185;               &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#171;&#186; <br />     Riboflavin synthase                                                                   2.5.1.9        gi&#229;&#8230;&#169;16129620       ribE            &#226;&#171;&#185;               &#226;&#171;&#185;                 &#226;&#171;&#185;                  &#226;&#171;&#185;           &#226;&#171;&#185;               &#226;&#171;&#185;           &#226;&#171;&#186;               &#226;&#171;&#185;               &#226;&#171;&#185;                &#226;&#171;&#186; <br />  <br /> Riboflavin (vitamin B2) salvage                                                                                                                      Yes                                        Yes             Yes         Yes                                        Yes <br />     Riboflavin transporter (ypaA)e                                                                       gi&#229;&#8230;&#169;16079362       &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#185;                 &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#185;               &#226;&#171;&#185;           ?               &#226;&#171;&#186;               &#226;&#171;&#186;                ? <br />  <br /> Common pathway FMN/FAD                                                                                                  Yes             Yes             Yes               Yes                Yes         Yes             Yes         Yes             Yes             Yes              Yes <br />    Riboflavin kinase                                                                      2.7.1.26       gi&#229;&#8230;&#169;16128019       ribFf           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#185; <br />    FAD synthetase                                                                         2.7.7.2 <br />        Bacteria                                                                                        gi&#229;&#8230;&#169;16128019       ribFf           &#226;&#180;&#8482;               &#226;&#180;&#8482;                 &#226;&#180;&#8482;                  &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;           &#226;&#180;&#8482;               &#226;&#180;&#8482;               &#226;&#180;&#8482;                &#226;&#171;&#186; <br />      (ii) Eukarya                                                                                        gi&#229;&#8230;&#169;6320159g       &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                 &#226;&#171;&#186;                  &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;           &#226;&#171;&#186;               &#226;&#171;&#186;               &#226;&#171;&#186;                &#226;&#171;&#185; <br />     Table 2, footnote  details. <br />  b <br />    Identical patterns pathways similar genes E. coli, Salmonella enterica serovar Typhi, Yersinia pestis subsystems considered. <br />  c <br />    RibA RibB form bifunctional (fused) protein bacteria monofunctional separate proteins E. coli H. influenzae. <br />  d <br />    RibD bifunctional (fused) protein bacteria. <br />  e <br />    Riboflavin transporter originally identified B. subtilis (56). <br />  f <br />    RibF bifunctional (fused) protein bacteria. <br />  g <br />    Eukaryotic FAD synthase originally identified S. cerevisiae (100) detectable sequence similarity corresponding bacterial enzymes. <br />                                                                                                                                                                                                                                                                                                       J. BACTERIOL. <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                        4561 <br />  <br /> Sucr Cms colonies recovered  192 Kmr Sucr colonies screened),              formed E. coli strain DH10B identical experimental <br /> suggesting nadD essential E. coli growth rich medium.                   conditions. <br />    Verification predicted gene essentiality S. aureus. Allelic exchange vector <br /> pBT2 utilized verification gene essentiality S. aureus (20). nadD <br />                                                                                           Preliminary gene essentiality conclusions based <br /> (gi&#229;&#8230;&#169;15927174) pncA (gi&#229;&#8230;&#169;15927492) S. aureus orthologs PCR amplified            semiautomatic analysis number relative posi- <br /> using primers 5&#226;&#172;&#732;-ATTCCTTGTCGCCCGTTATGC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-AACGCGCTTC                            tions inserts retained gene selective  <br /> ATTGTATCCT-3&#226;&#172;&#732; nadD 5&#226;&#172;&#732;-GTCCGTTAATCCCACAAGCATCA-3&#226;&#172;&#732;                               growth 23 population doublings. Failure recover inserts, 5&#226;&#172;&#732;-CGCCGACTTTATCTTTTTCAGC-3&#226;&#172;&#732; pncA. Genomic DNA isolated <br />                                                                                        presence limited number inserts S. aureus strain ATCC 29213 used PCR template. PCR <br /> products cloned plasmid pCR2.1-TOPO (Invitrogen, Carlsbad, Calif.).          end coding sequence, suggested cells carrying trans- 384-bp NheI-AvrII fragment nadD replaced tetracycline resis-           posons gene viable growth  <br /> tance marker (generous gift M. Smeltzer, University Arkansas Med-          ditions.  occur gene constituted ical Sciences, Little Rock), resulting 4.0-kb DNA fragment containing          cold spot transposition insert polar effect inactivated nadD cloned pBT2 (20). plasmid introduced S. aureus RN4220 electroporation. Following cointegration, resolution, <br />                                                                                        essential downstream gene. validated technology elimination plasmid (20), tetracycline-resistant (Tetr) colonies       combination following controls:   genetic foot- <br /> screened sensitivity chloramphenicol. Tetr Cms colonies        printing mutagenized population prior outgrowth case nadD, pncA successfully inactivated control exper-        (time zero sample) number genes previously estab- <br /> iment. case, 38% Tetr colonies Cms, Tetr <br />                                                                                        lished essential (Fig. 2B), (ii) verification consistency <br /> Cms colonies tested PCR contained correct pncA deletion chro- <br /> mosome.                                                                                preliminary essentiality assignments metabolic <br />    S. aureus coaD ortholog (gi&#229;&#8230;&#169;15926709) flanking regions amplified       biological context gene  , (iii) compar- PCR using primers 5&#226;&#172;&#732;-GATTGCCAGTTGTAGGGTTCATA-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GC                           ison observed essentiality data reported GTTGGCTTAATCACAGAATA-3&#226;&#172;&#732;, cloned pBT2  produce plasmid                          literature, (iv) genetic footprinting presence pMF32), subjected vitro transposon mutagenesis. end artificial transposon constructed (M. Farrell et al., unpublished data)                                                                                        complementing DNA fragment  example, reference <br /> cloning Tet M marker derived Tn916 (38) pMOD-2&#226;&#172;?MCS&#226;&#172;&#381;                      27). <br /> (Epicentre Technologies). Plasmid pMF32 mixed 1:1 molar ratio               Global transposon mutagenesis E. coli. Libraries 2 &#226;&#171;&#187; <br /> Tet M-transposon DNA, EZ::TN transposase (Epicentre Technologies)              105 independent insertion mutants generated E. coli <br /> added 0.1-U/&#226;?&#174;l final concentration. mixture incubated 2 h                                                                                        strains MG1655 DH10B using Tn5-based EZ::TN&#226;&#172;?KAN- <br /> 37&#194;&#176;C reaction buffer (50 mM Tris-acetate [pH 7.5], 150 mM potassium acetate, <br /> 10 mM magnesium acetate, 40 mM <span id='am-98' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-99' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span>spermidine</span>) used transform E.                2&#226;&#172;&#381;Tnp transposome (Epicentre Technologies) transposon <br /> coli strain DH10B. Tetr E. coli colonies screened            delivery  utilizes mutant hyperactive form cell PCR, different transposon-containing pMF32 plasmids se-             Tn5 transposase, reported low inser- <br /> lected, including plasmids containing coaD inactivated single transposon      tion site specificity (41, 64). study EZ::TN transposase <br /> insertion middle gene plasmids transposon open <br /> reading frames (ORFs) immediately upstream downstream coaD.                                                                                        sufficiently random yield 1 14 indepen- <br /> plasmids used parallel pBT2-driven allelic exchange S. aureus           dent insertions majority E. coli genes  example, <br /> RN4220  . Tetr Cms (knockout) colonies obtained coaD             Fig. 2). average insertion density experimentally <br /> disruptions, neighboring ORFs (encoding gi&#229;&#8230;&#169;15926708                      determined insert 250 bp  outgrowth). <br /> gi&#229;&#8230;&#169;15926710) successfully inactivated. <br />                                                                                         total number 105 independent insertion mu- <br />    Preliminary characterization bacterial targets human counter- <br /> parts. Representative bacterial target enzymes human counterparts            tants analyzed 23 cell divisions theoretically corresponds expressed E. coli BL21(DE3) N-terminal fusions tag           insert 46 bp genomic sequence  E. coli <br /> using pProEX-HT3a (Invitrogen). Corresponding DNA fragments                 genome size 4,639 kb). believe difference amplified PCR using bacterial genomic DNA (American Type Culture               slight preferences target sequence recognition <br /> Collection, Manassas, Va.) human brain cDNA (Clontech) primers gen- <br /> erating NcoI BspHI site 5&#226;&#172;&#732; end SalI PstI site 3&#226;&#172;&#732; end. <br />                                                                                        modified Tn5 transposase actually used DNA fragments purified, digested, cloned NcoI SalI            numerical measure bias. data  PstI) restriction sites vector. resulting constructs verified   sixfold difference observed insertion density DNA sequencing. Recombinant proteins purified homogeneity using              250 bp versus frequency expected completely step procedure consisting chromatography Ni-nitrilotriacetic acid aga- <br />                                                                                        random insertions  46 bp). insertion density al- <br /> rose (Qiagen) gel filtration (Superdex G-200) described previously (28). <br /> Preliminary enzymatic characterizations performed representative bac-         lowed make essentiality assessments 87% E. coli <br /> terial human enzymes involved biosynthesis NAD (NMN                  ORFs, larger 80 amino acid residues NaMNAT), CoA (PPAT), FAD (FADS) using high-performance liquid                      length. <br /> chromatography analysis reaction products coupled enzymatic                 assess reliability approach, compared  <br /> assays, described reference 27. Enzymatic reactions performed presence 0.1 mM <span id='am-47' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-48' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> 0.1 mM concentration respective <br />                                                                                        sults 50 min E. coli chromosome gene <br /> substrates: NMN  NaMN), 4&#226;&#172;&#732;-phosphopantetheine, flavin mononucleotide             essentiality data compiled literature Genetic <br /> (FMN). High-performance liquid chromatography analysis performed using             Resource Committee Japan (http://www.shigen.nig.ac.jp <br /> <span id='am-24' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-25' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span>-pair chromatography isocratic conditions: 100 mM sodium phosphate            /ecoli/pec/Analyses.jsp). 111 nonessential genes listed <br /> buffer (pH 3.5) 8 mM tetrabutylammoniumbromide methanol (8%                                                                                        region chromosome, 98 genes (88%) NaMNAT, 15% PPAT, 30% FADS), column (50 4.6 mm <br /> [inner diameter]), packed 5 &#226;?&#174;M C18 (Supelco).                                     experimentally determined nonessential (retained  <br />                                                                                        serts) procedure. <br />                                                                                           81 known essential genes located 50 <br />                                   RESULTS <br />                                                                                        min E. coli chromosome, transposon insertions   Genetic footprinting (Fig. 1) systematically applied                          detected 70 genes (86%) experiment. gene determine genes required logarithmic aerobic growth                            81 analyzed, encoding essential cell division protein FtsK E. coli MG1655 enriched Luria-Bertani medium. Genetic                            (11), contained 10 insertions.  inserts footprinting limited chromosomal region                              clustered C-terminal half coding sequence, <br />  4562     GERDES ET AL.                                                                                                   J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                      FIG. 1. General scheme E. coli genetic footprinting procedure. <br />  <br />  <br />  <br /> corresponding amino acid residues 780 1329 (Fig. 2C).            genes sets data good agreement. 13 genes N-terminal domain FtsK required role         lacked transposon insertions outgrowth MG1655, <br /> cell division viability (97). suggests genetic foot-     12 did contain inserts strain DH10B  printing used successfully detect  map)          exception nadE, contained single insert vivo essential domains ORFs, cases essen-          DH10B experiment. seven nonessential genes tial region immediately adjacent translational start.       pathways analyzed strains  10 remaining essential genes single transposon insertion         tained inserts MG1655 DH10B. panD lacked detected outgrowth strains            inserts DH10B, MG1655 experiment (MG1655 DH10B). fact              contained  reflect fact DH10B <br /> genes tolerate transposon inserts certain restricted        experiment transposons inserted possible <br /> loci detrimental effect corresponding gene            orientations monitored, orientations product. Similar phenomena reported ge-             mapped MG1655 experiment. <br /> netic footprinting experiments E. coli (45) H.                Comparison MG1655 DH10B genetic footprints <br /> influenzae (1).                                                        permitted detailed mapping deletions DH10B <br />    Comparison genetic footprints E. coli                chromosome: &#226;&#338;&#172;(ara-leu)7697 (22) &#226;&#338;&#172;lacX74 (10, 89).  <br /> strains. test reproducibility approach, gen-          letion &#226;&#338;&#172;(ara-leu)7697 covers 25.6-kb region, corre- <br /> erated limited genetic footprint E. coli strain DH10B             sponds area 63.4 89.0 kb MG1655 genome <br /> (covering 650 kb chromosome) compared          includes genes polB fruR (Fig. 2E). corresponding region MG1655. Unambiguous es-               Deletion &#226;&#338;&#172;lacX74 corresponds region 340.3 sentiality data strains obtained 530 genes            369.5 kb (total 29.2 kb) MG1655 sequence  <br /> (excluding deleted DH10B). Identical assessments           cludes genes b0324 b0347 possibly <br /> essentiality produced 487 genes (92%) group.          mhpB  shown). results genetic footprinting experiments MG1655                 transposon insertions produce polar effects? Surpris- DH10B genes controlling biosynthesis NAD, CoA,              ingly, cases transposon insertions detected  FAD presented Table 1. majority            stream known essential genes, expected  VOL. 184, 2002                                                    GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                               4563 <br />  <br />  <br />  <br />  <br />    FIG. 2. Detection mapping transposon insertions.   Primer strategy nested PCR. Transposon-specific primers shown gray; <br /> chromosome-specific landmark primers shown black. (B) gel image shows analysis chromosomal loci: aspC (nonessential) lpdA ftsJ (essential) time zero (lanes 1, 3, 5) outgrowth (lanes 2, 4, 6). ORF locations marked relative pair lanes. inserts visible lpdA ftsJ time zero, detected outgrowth. nonessential aspC contains <br /> insertions time points. (C E) Examples genetic footprints. Note scale different panel. length direction gene indicated large horizontal gray arrows. Black diamonds represent transposon inserts. width diamond corresponds mapping error introduced gel electrophoresis. positions landmark PCR primers shown bows crossing genes, arrows genes. (C) Genetic footprinting ftsK locus MG1655. 3&#226;&#172;&#732; half essential gene contains inserts. <br /> (D) Genetic footprinting coaD locus MG1655. transposon insertion immediately upstream essential gene apparently does interfere expression. (E) Mapping &#226;&#338;&#172;(ara-leu)7697 deletion DH10B. genetic footprint corresponding region MG1655 shown comparison. <br />  <br />  <br />  <br /> destroy promoter sequences polar effect              inserts nonessential N-terminal regions protein expression downstream genes. case coaD               tolerated interfere translation  <br /> shown Fig. 2D), argS, frr, rplT, secA,          stream sequences. limits subgenic resolution ge- genes. believe EZ::TN&#226;&#172;?KAN-2&#226;&#172;&#381; transpo-                     netic footprinting cases multifunctional proteins son sequence inserted orientation capable initi-            essential domains proximal translational start  shown ating level transcription sufficient cell survival         Fig. 2C). <br /> cases, specific promoter sequence added                  Analysis gene essentiality data context E. coli structure. Analysis transposon sequence Neu-              metabolic pathways. major advantages genetic <br /> ral Network Promoter Prediction program (95; http://www                     footprinting directed knockouts identification es- <br /> .fruitfly.org/seq_tools/promoter.html) reveals multiple putative            sential genes fact footprinting fast highly <br /> E. coli-type promoters oriented outwards directions.                parallel method. high-throughput technique, probably explains polar effects transposon inser-               limitations, example, cross-feeding outgrowth <br /> tions downstream genes rarely observed study.                complex mutant population. efficient way minimize  translational polarity distal domains genes              erroneous conclusions respect essentiality likely significant limiting factor technique.         given gene consider results context multifunctional protein essential C-terminal domain,              corresponding metabolic pathway subsystem. Metabolic <br />  4564     GERDES ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Simplified diagrams, illustrating biochemical transformations directly involved biosynthesis NAD(P)  , CoA (B), FMN/FAD (C). pathways genes shown present E. coli, exceptions marked dashed lines. Recycling <br /> pathways transformations related genes remain unknown  NMN deamidase) included. <br />  <br />  <br />  <br />  <br /> context analysis allows reconciliation data genetic          chemically genetically. hand, key <br /> footprinting experiments, literature  available),         genes identified recently, essentiality predicted metabolic reconstruction. tech-              directly confirmed. results genetic footprinting <br /> nique efficiently applied E. coli,     majority known E. coli genes involved best-studied model microbial systems. contradictions                biosynthesis salvage pathways producing NAD(P), CoA, types data encountered certain gene,          FAD summarized Tables 2 4. possible formulate probable assertion           Genetic footprinting E. coli genes involved NAD(P) <br /> inspecting behavior E. coli genes           biosynthesis. behavior majority NAD(P) biosyn- <br /> pathway.                                                                 thetic genes genetic footprinting experiments (Table 1)     biosynthesis adenylate cofactors provides good ex-          consistent previously published data meta- <br /> ample utility approach,   cofactors         bolic reconstruction presented Fig. 3A. essential metabolites types organisms, (ii)            expected, genes novo salvage pathways <br /> corresponding pathways E. coli rich essential genes,           appear nonessential, experiments outgrowth oc- (iii) subsystems thoroughly studied bio-             curred rich media containing niacinamide. Nicotinamide <br />  VOL. 184, 2002                                             GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                       4565 <br />  <br />  <br /> riboside (NmR) probably advanced NAD(P) pre-           (kdtB), previously shown essential (36, 45). cursor transported E. coli.       shown Table 1, genes essential genes common pathway essential,       footprinting study, acpS, produces  <br /> inactivation compensated NAD(P) salvage.          zyme responsible covalent attachment CoA acyl car- consistent observations, notable ex-     rier protein, required fatty acid biosynthesis (33). <br /> ception nadD. Multiple insertions gene ob-            Genetic footprinting E. coli genes involved FMN/FAD <br /> served, suggesting nadD dispensable E. coli growth     biosynthesis. FMN/FAD biosynthetic genes rich media. contradicted genetic data Salmonella       essential genetic footprinting experiments enterica serovar Typhimurium (48), exper-       E. coli strains (Table 1). fact genes iments E. coli MG1655, failed produce directed       novo riboflavin biosynthesis essential presence deletion nadD rich medium, presence           riboflavin (0.8 mg/liter) medium consistent 50 &#226;?&#174;M NAD NMN. hands, nadD                 absence riboflavin transporter E. coli K-12 (6). Ribo- <br /> successfully deleted presence plasmid containing     flavin auxotrophs obtained E. coli using specific <br /> functional human nadD ortholog, pyridine <span id='am-73' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-74' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-75' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> adeny-         selection steps facilitate riboflavin transport uncharacter- <br /> lyltransferase 1 (PNAT-1) (gi&#229;&#8230;&#169;12620200; O. Kurnasov, unpub-        ized mutations significantly higher concentrations lished data).                                                      exogenous riboflavin (6, 8, 9). <br />    possible explanation contradiction pres-        enzymes common flavin pathway, consecutively <br /> ence functional NadR E. coli (Table 2). nadR gene     producing cofactors, FMN FAD, form bifunc- transcriptional regulator novo biosynthesis        tional protein encoded essential ribF gene. enzy- <br /> niacin salvage genes E. coli (92). Recently NadR shown      matic activities expected indispensable, ge- possess low levels NMN adenylyltransferase activity (74)     netic footprinting data suggest essentiality NmR kinase activity (O. Kurnasov et al., unpublished    C-terminal domain (encoding FK). general limitation <br /> data). suggests NadR play role salvaging        technique respect multifunctional proteins. <br /> exogenous NMN. NMN adenylyltransferase activity                summary, 13 genes encoding 16 enzymes NadR involved recycling intracellular NMN           vitamin/cofactor biosynthetic pathways shown  <br /> directly NAD, bypassing NAD synthase encoded             quired aerobic growth E. coli enriched media (Table <br /> nadE (Fig. 3A).  flux PnuC-NadR sal-           1). genes identified essential genetic <br /> vage pathway unlikely sufficient compensate        footprinting agreement published data theoretical <br /> inactivation nadD nadE, responsible        analysis. contradictory case (nadD) reconciled bulk NAD production E. coli (73).  explanation     directed gene deletion strategy. addition, 12 genes <br /> contradicts apparent essentiality nadE observed      related metabolism NAD(P), CoA, FAD ana- <br /> genetic footprinting experiments, compensatory role      lyzed genetic footprinting nonessential PnuC-NadR bypass imply nonessentiality nadE        (Table 1).  results good agreement nadD (Fig. 3A). present propose          available experimental data metabolic reconstructions single unambiguous interpretation contradiction          corresponding pathways. <br /> tween classical genetic genetic footprinting data        essential genes representing potential broad-spectrum <br /> respect nadD.  point disagree-      antibacterial targets analyzed respect ment NAD(P) biosynthesis genetic footprinting       genomic data available selected bacterial pathogens data types evidence.                                  human host  . analysis allows extrapolation <br />    Genetic footprinting E. coli genes involved pantothe-     experimental gene essentiality data E. coli nate/CoA biosynthesis. contrast complexity            bacterial pathogens, including S. aureus. <br /> NAD(P) metabolism, CoA biosynthetic pathway topo-              Selected target validation. predicted essentiality logically simple (Fig. 3B). genetic footprinting data   S. aureus nadD ortholog (gi&#229;&#8230;&#169;15927174) confirmed consistent experimental data theoret-      indirect approach. Deletion nadD attempted parallel <br /> ical considerations (Table 1). enzymatic step com-     S. aureus pncA ortholog (gi&#229;&#8230;&#169;15927492), presumed mon pathway pantothenate CoA nonredundant           dispensable presence exogenous nicotinate, using <br /> indispensable, phosphorylated intermediates      allelic exchange vector pBT2 (20). &#226;&#338;&#172;nadD variants transported cell. eliminates potential        obtained, extensive screening (2,248 colonies), cross-feeding possibility exogenous CoA salvage      control experiment pncA successfully inactivated transport phosphorylated precursors growth         38% colonies screened. indicates nadD medium. genes pantothenate biosynthetic path-       essential S. aureus growth complex tryptic soy <br /> way nonessential bypassed exogenous pan-         broth (Difco Laboratories, Detroit, Mich.). <br /> tothenate known transporter encoded panF gene            different strategy used verify predicted essen- <br /> (51, 93).                                                          tiality S. aureus coaD ortholog (gi&#229;&#8230;&#169;15926709). vitro <br />    coaA gene E. coli, encoding pantothenate kinase,     transposon mutagenesis used disrupt coaD enzyme common step pathway pantothe-        adjacent ORFs. conclusion coaD essentiality <br /> nate CoA, previously characterized essential (86, 94).   drawn based fact coaD disrupted <br /> Genes enzymatic steps CoA biosynthesis        neighboring ORFs (gi&#229;&#8230;&#169;15926708 gi&#229;&#8230;&#169;15926710) <br /> (coaBC, coaD, coaE) recently discovered         successfully inactivated Tet M transposon, (39, 65). genes, coaE  yacE) coaD        frequencies 83 5%, respectively. <br />  4566     GERDES ET AL.                                                                                                          J. BACTERIOL. <br />  <br />  <br />  <br />  <br />  FIG. 4. Selection antibacterial drug targets combination genetic footprinting E. coli comparative analysis reconstructed <br /> metabolic subsystems, pathways, individual genes pathogens humans. <br />  <br />  <br />  <br />    Selected bacterial targets human counterparts               genetic footprinting identifying novel antibacterial drug tar- overproduced E. coli, purified, characterized.           gets (Fig. 4). <br /> respective adenylyltransferase activities following               Experimental gene essentiality data determined genetic <br /> combinant enzymes directly confirmed vitro:                      footprinting analyzed terms relevant E. coli met- <br /> NaMNATs E. coli, S. aureus, H. pylori (gi&#229;&#8230;&#169;1786858,              abolic subsystems pathways using ERGO database. <br /> gi&#229;&#8230;&#169;15924584, gi&#229;&#8230;&#169;2314504, respectively); isoforms hu-          contextual analysis refines essentiality assessment man PNAT, e., PNAT-1 (gi&#229;&#8230;&#169;12620200) PNAT-2                          gene addressing following questions. met- <br /> (gi&#229;&#8230;&#169;11245478); PPATs S. aureus, M. tuberculosis, H. influ-          abolic pathway essential gene associated  Does enzae, H. pylori (gi&#229;&#8230;&#169;15924115, gi&#229;&#8230;&#169;560525, gi&#229;&#8230;&#169;1573650,            pathway yield metabolite essential cell viability? gi&#229;&#8230;&#169;15612433, respectively); PPAT domain human                 route produce metabolite?  <br /> PPAT/DPCK protein (27); FADS domain B. subtilis                   tabolite salvaged growth media final form? <br /> bifunctional FK/FADS protein (gi&#229;&#8230;&#169;16078730); human                    advanced precursor  point salvage) <br /> FADS (F. Mseeh, unpublished results).                                    cell acquire environment? kind                                                                          analysis generates hypotheses essentiality non- <br />                                                                          essentiality known genes pathways. Com- <br />                           DISCUSSION <br />                                                                          parison models actual genetic footprinting data <br />    approach identifying ranking antibacterial drug            data available literature (e.g., http://www <br /> targets based combination genetic footprinting              .shigen.nig.ac.jp/ecoli/pec/Analyses.jsp) allows refinement model (E. coli) comparative genome analysis                initial experimental assessments gene essentiality calls schematically illustrated Fig. 4. major components            experimental examination  case nadD). approach, experimental computational, post-                    consistent data compiled E. coli, metabolic <br /> genomic techniques. Genetic footprinting E. coli allows               reconstruction selected pathways extended panel identification essential genes function growth condi-         representative pathogens. hierarchical overview illustrated <br /> tions high-throughput format. relatively new tech-             Tables 2 4 provides convenient way perform com- <br /> nique multiple potential applications functional genom-           parative cross-genome analysis relevant pathways met- <br /> ics. example, monitoring changes pattern gene           abolic subsystems. Questions addressed step essentiality varying growth conditions genetic              analysis include following. pathways present background, possible assess various aspects cell         pathogens consideration? pathways  <br /> metabolism. allow functions previously uncharac-             taining essential genes conserved nonredundant terized genes established allows functional roles          group? functions selected pathways known genes refined. example study             encoded closely related genes? point reflect published recently (7). consider application             likelihood finding small molecule efficiently <br />  VOL. 184, 2002                                                     GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                         4567 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 6. General scheme chemical transformations catalyzed                                                                            adenylyltransferases (ATse) biosynthesis NAD(P), <br />                                                                            CoA, FAD. enzymes&#226;&#8364;&#8221;NaMNAT (encoded E. coli                                                                            nadD), PPAT (gene coaD), FADS (ribF) selected                                                                            promising antibacterial drug targets pathways. <br />  <br />  <br />  <br />                                                                            bind target protein pathogens spectrum <br />                                                                            suppressing functional activity. use parameters                                                                            preliminary measures likelihood, P values pair- <br />                                                                            wise sequence comparisons E values produced                                                                            comparing bacterial sequence profile (HMM) built <br />                                                                            specific panel pathogens  illustrated Fig. 5A    FIG. 5. Prioritization potential broad-spectrum antibacterial        B). structural functional data, elaborate <br /> drug targets biosynthetic pathways NAD(P), CoA, FAD. criteria shown target enzyme (marked abbrevi-        productive criteria applied, conservation ations x axis).   Range pathogens. number patho-       elements active site, relative substrate preferences, <br /> gens  representative set) containing given target enzyme      affinities range natural synthetic ligands. <br />  included building corresponding HMM) represented            corresponding human pathways reconstructed filled bars. Organisms containing target enzyme  <br /> fore excluded spectrum indicated inside open bars                                                                             genomic data assess potential effects letter abbreviations). (B) Compact pathogen subsets outliers.      arise inhibition human counterparts antibac- <br /> Negative logarithms E values indicate closely bacterial    terial targets. criterion used selecting antibacte- <br /> sequence related probabilistic consensus corresponding    rial targets absence corresponding function profile HMM (larger values indicate higher similarity). Subsets         corresponding pathway human host   <br /> pathogens target orthologs closely related consensus shown inside dotted boxes. Outliers indicated letter        stance, bacterial cell wall biosynthesis). approach, al- <br /> organism abbreviations. Organisms shown P. aeruginosa (E) (PA),        historically productive, unnecessarily rejects <br /> B. anthracis ( ) (BA), M. tuberculosis (&#198;&#8217;) (MT), H. pylori (&#226;&#8211;&#160;) (HP), S.    promising targets. targets universal biosynthetic path- <br /> pneumoniae (&#228;&#352;?) (PN), S. aureus (&#226;&#189;&#167;) (SA), M. genitalium (&#227;&#8364;&#171;) (MG),           ways reconsidered new opportuni- <br /> H. influenzae (&#197;&#8217;) (HI), C. trachomatis (&#226;&#8364;&#353;) (CT), E. coli (F). E. coli proteins similar Yersinia pestis Salmo- <br />                                                                            ties provided comparative genomics parallel chemical <br /> nella enterica serovar Typhi. (C) Human counterparts: distance        synthesis. Certain features revealed level host-patho- <br /> bacterial families. negative logarithms E values human       gen comparative genome analysis useful evaluation <br /> counterpart sequences compared corresponding bacterial              targets. instance, pathway unique HMMs (smaller values indicate higher divergence human            pathogens potentially redundant humans  NAD <br /> zyme corresponding bacterial family). &#226;&#180;&#177;&#226;&#180;&#177;, HMM E values human proteins higher threshold (&#226;&#172;&#381;10,000).              biosynthesis), human functional counterpart                                                                            structurally unrelated bacterial proteins. cases  4568     GERDES ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br /> bacterial orthologs target enzyme human version         graded nicotinamide riboside membrane-bound share sequence similarity, prioritize targets based       periplasmic phosphohydrolases, nadN gene prod- <br />   lowest P values generated pairwise compar-       uct (76). addition, step niacinamide salvage pathway <br /> isons target protein homologs bacterial pathogen        gram-positive pathogens S. pneumoniae S. au- human functional counterpart (ii) E value         reus constitute possible narrow-spectrum target. derived comparison human protein sequence             organisms lack genes novo NAD biosynthesis profile HMM built corresponding pathogen pro-            PnuC-NadR pathway nicotinamide riboside salvage <br /> teins  Fig. 5C).                                               (Table 2). <br />    illustrate approach, essential conserved genes           step pathway NaMN NAD NADP, known functions biosynthesis key adenylate        requiring NaMNAT, NADS, NADK, conserved cofactors NAD(P), CoA, FAD investigated.              majority bacterial pathogens (Table 2), representing poten- <br /> biosynthetic pathways fit criteria listed        tially broad-spectrum targets. pathway nonredundant, cofactor utilized multiple enzymes pathogens;      corresponding genes essential E. coli (Table 2). <br /> (ii) bacteria unable import cofactors     NaMNAT NADS present C. trachomatis H. <br /> phosphorylated precursors prior degradation; (iii)       influenzae, excludes pathogens target steps biosynthetic pathways universal        profile. Inhibition NADK cover broadest range nonredundant contain number promising targets.          pathogens, excluding C. trachomatis  Fig. 5A).  <br />    NAD(P) biosynthesis characterized high level            NADK recently characterized H. sapiens (57), <br /> complexity diversity various organisms (60, 66), includ-      revealing high sequence similarity bacterial <br /> ing versions novo biosynthesis          enzymes  P value 2 &#228;&#161;&#160; 10&#226;&#171;&#186;25 NADK P. aeruginosa <br /> routes salvage exogenous precursors (Table 2 Fig.         significantly better P values pairwise comparison <br /> 3A). absence recognizable genes NAD(P) bio-           bacterial orthologs). C-terminal synthase <br /> synthesis genome C. trachomatis (Table 2) suggests         domain human NADS ortholog (gi&#229;&#8230;&#169;10433831) reveals obligate intracellular parasite unique transport       lower significant sequence similarity machinery salvage NAD NADP.  C. tra-            bacterial counterparts  best P value 6 &#228;&#161;&#160; 10&#226;&#171;&#186;13 NADS <br /> chomatis excluded panel pathogens used           M. genitalium). Finally, human counterpart bacte- assess targets NAD biosynthesis.                               rial NaMNAT marginal sequence similarity bac- <br />    Genes controlling novo NAD biosynthesis aspartate         terial enzymes nadD family  best P value &#226;&#172;&#402;0.2 salvage niacin E. coli nonessential rich        NaMNAT H. pylori). E values derived comparing <br /> media. conclusion projected P. aeruginosa, B.       human PNAT, NADS, NADK profile HMMs anthracis, possibly M. tuberculosis based metabolic        corresponding bacterial protein families support conclu- <br /> constructions (Table 2). cases controversial.   sions (Fig. 5C). Prioritization potentially broad- spite fact genes required niacinamide        spectrum targets NAD biosynthesis combination salvage (pncA pncB) present genome, M. tuber-         criteria summarized Fig. 5 yields target <br /> culosis reported strictly dependent novo        preference order follows: NaMNAT better NADS <br /> NAD biosynthesis (83). pncA expressed M. tubercu-        better NADK. <br /> losis (81, 102), interpretation pncB ortholog        NaMNAT activity initially characterized E. coli (26), <br /> functionally inactive M. tuberculosis, labora-     chromosomal locus mapped Salmonella (48). <br /> tory conditions.                                                      identity structural gene remained  <br />    Theoretically, bacterial novo NAD biosynthesis         known recently. identified gene based partate attractive target pathway, absent       clustering chromosome nadE pncB gene  <br /> humans, novo biosynthesis occurs step ox-         thologs microbial genomes (12). gene idative degradation tryptophan (66).           independently identified E. coli Mehl coworkers (63). <br /> set bacterial pathogens, H. pylori expected       cloned expressed nadD E. coli useful targets pathway, lacks niacin salvage        cloned expressed orthologs gram-positive (S. au- <br /> genes (Table 2). step conversion aspartate             reus) gram-negative (H. pylori) pathogens (12), quinolinate make excellent target pathway anti-         purified NadD proteins. Analysis substrate <br /> infective agents specific H. pylori (Table 2 Fig. 3A).      specificity purified recombinant proteins revealed strong <br /> systematic analysis essential genes conserved exclusively      preferences bacterial enzymes NaMN H. pylori (23) did recognize potential pathway    NMN  &#226;&#172;&#381;17,000-fold case S. aureus NaMNAT). target drug development, did          bias consistent major flux NAD biosynthesis <br /> account metabolic context organism.                      bacteria going nicotinic acid mononucleotide <br />    PnuC-NadR pathway nicotinamide riboside salvage            (NaMN) NaAD. H. influenzae provides example narrow spec-             isoforms NadD identified trum target. complete pathway present limited       human genome (gi&#229;&#8230;&#169;12620200, gi&#229;&#8230;&#169;11245478, gi&#229;&#8230;&#169;14029540). <br /> number bacteria, sole route NAD biosyn-         cloned, purified, characterized forms <br /> thesis H. influenzae. H. influenzae requires called V fac-     designated PNAT-1 PNAT-2, tors growth (25, 76), simplest nicotinamide      perform adenylyltransferase reaction equally efficiently <br /> riboside. acceptable V factors, NADP, NAD,             pyridine nucleotides NMN NaMN (O. Kurnasov <br /> NMN  present growth medium), gradually              et al., unpublished data). Human PNAT-2 indepen- <br />  VOL. 184, 2002                                                GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                        4569 <br />  <br />  <br /> dently identified characterized groups (31,          tholog monofunctional PPCDC. Monofunctional <br /> 80). observed dual specificity human PNATs consis-          PPCSs S. pneumoniae B. anthracis reveal high <br /> tent previous experimental data (61) recon-         sequence similarity  quite divergent <br /> struction putative human NAD biosynthetic pathway              PPCS domain bifunctional PPCDC/PPCS pro- genomic data (Table 2).                                          teins. contrast bacterial orthologs, produce    difference substrate preferences human              low reliable similarity score compared human <br /> PNATs bacterial NaMNATs additional incentive               monofunctional PPCS (3 &#228;&#161;&#160; 10&#226;&#171;&#186;23 H. sapiens B. <br /> pursuing NadD selective, relatively broad-spectrum           anthracis). enzyme common pathway, DPCK, antibacterial target. dimensional structures             ubiquitous pathogens set. <br /> NaMNATs E. coli (101) B. subtilis (70)                enzymes human CoA biosynthesis solved association NaMN. Comparison struc-           recently identified characterized (27).  <br /> tures structure human PNAT-2 (103) reveals sig-           PPCDC DPCK, relatively similar bacterial <br /> nificant differences organization active sites,       counterparts (Fig. 5C). human monofunctional PPCS facilitate identification selective antibacterial      distant PPCS domains bacterial bifunctional <br /> inhibitor.                                                            PPCDC/PPCS proteins, closely related    Similar comparative analysis CoA FMN/FAD                bacterial monofunctional PPCDCs. PPAT domain biosynthetic pathways humans bacteria produced addi-           human bifunctional PPAT/DPCK protein reveals signifi- <br /> tional broad-spectrum antibacterial targets (Fig. 5).          sequence similarity bacterial counterpart, novo pathway CoA biosynthesis present bacterial        likely quite dissimilar overall structure. PPAT <br /> pathogens absent humans (Table 3), corre-              probably represents attractive target common <br /> sponding enzymes attractive targets          CoA biosynthetic pathway number reasons:    universal presence pantothenate symporter orthologs.           stitutes target broad range pathogens (Fig. 5A), (ii) pathogens lack novo pantothenate biosynthetic            bacterial orthologs enzyme closely related genes (Table 3) depend solely salvage pantothenate (S.          consensus  outliers; Fig. 5B), (iii) human PPAT pneumoniae H. influenzae) dephospho-CoA (M. geni-              dissimilar bacterial HMM profile (Fig. 5C). <br /> talium C. trachomatis). Bacterial pantothenate symporters         PPAT E. coli (encoded coaD, previously kdtB) (panF family) share high sequence similarity human             previously described, dimensional structure protein implicated vitamin transport (96). panF              solved (39, 50). <br /> thologs present genomes M. genitalium C.                majority bacteria  exception M. geni- <br /> trachomatis. Specific transporters allowing salvage host        talium) considered conserved novo riboflavin <br /> phospho-CoA intracellular pathogens              biosynthetic pathway, present humans (Table 4 <br /> represent narrow spectrum targets) unknown.                 Fig. 3C).  genes novo path- <br />    enzymatic steps common pathway CoA              way essential E. coli (Table 4). observations biosynthesis pantothenate encoded essential genes         brought attention gene products potential E. coli (Table 3). enzymes conserved        antibacterial drug targets (17, 29, 98).  direct exper- <br /> broad range bacterial pathogens. enzyme           imental analysis riboflavin transport salvage pathway, PK, encoded E. coli coaA (86) belongs             pathogens necessary order define useful structural family different recently identified      targets  B. subtilis known effectively salvage exog- <br /> eukaryotic PKs (21). makes coaA-related PKs attrac-           enous riboflavin, bypassing requirement novo <br /> tive target relatively narrow subset list patho-      pathway (19). Orthologs riboflavin transporters, <br /> gens, including H. influenzae, M. tuberculosis, S. pneu-          gene ypaA, recently identified B. subtilis (56) present moniae. S. aureus  staphylococci                 gram-positive pathogens, including B. anthracis, S. au- <br /> enterococci) B. anthracis lack coaA orthologs,           reus, S. pneumoniae, functionality enzymatic function likely performed remote              directly confirmed. Additional structurally dissimilar ri- <br /> thologs eukaryotic PK  tr&#229;&#8230;&#169;Q99SC8 S. aureus).        boflavin transporters likely exist pathogens. known forms PK identified             example, riboflavin transport occur M. genitalium, sequence similarity analysis P. aeruginosa, H. pylori,       novo riboflavin biosynthesis pathway asserted pathogens included set, suggests       organism (Table 4).  ypaA orthologs existence uncharacterized form enzyme.        available mycoplasma ureaplasma ge- <br />    remaining enzymes pathway represent poten-         nomes. <br /> tial antibacterial drug targets relatively broad spectrum (Ta-        common pathway FMN/FAD biosynthesis repre- <br /> ble 3). bacteria, enzymatic activities, PPCS          sented bacteria bifunctional enzyme catalyzes <br /> PPCDC, located C-terminal N-terminal               consecutive reactions. reaction formation domains bifunctional (fused) protein (gene coaBC, previ-         FMN, catalyzed FK domain (C terminus). second <br /> ously dfp E. coli). exception list pathogens S.   reaction conversion FMN FAD FADS domain <br /> pneumoniae  streptococci enterococci)            (N terminus). enzymes represent antibacterial targets <br /> containing separate ORFs, coaB coaC, operon            broad range (Fig. 5A). HMM E values  reversed order relative fusion proteins).         group pathogens high (Fig. 5B), indicating <br /> ingly, B. anthracis  Bacillus cereus) contains      targets divergent.  FADS <br /> bifunctional PPCDC/PPCS monofunctional PPCS                  parameter improved removing outliers, <br />  4570       GERDES ET AL.                                                                                                                                  J. BACTERIOL. <br />  <br />  H. pylori  source divergent pro-                               quired growth survival Haemophilus influenzae. Proc. Natl. Acad. <br />                                                                                              Sci. USA 99:966&#226;&#8364;&#8220;971. <br /> teins sample group pathogens) M. genitalium                           3.   Alm, R. , L. S. L. Ling, D. T. Moir, B. L. King, E. D. Brown, P. C. Doig, <br /> (Fig. 5B).                                                                                   D. R. Smith, B. Noonan, B. C. Guild, B. L. deJonge, G. Carmel, P. J. <br />    contrast known bacteria, eukaryotic FK                             Tummino,  Caruso, M. Uria-Nickelsen, D. M. Mills, C. Ives, R. Gibson, <br />                                                                                              D. Merberg, S. D. Mills, Q. Jiang, D. E. Taylor, G. F. Vovis, T. J. Trost. <br /> FADS identified S. cerevisiae (78, 100) monofunctional                                1999. Genomic-sequence comparison unrelated isolates hu- <br /> proteins. overexpressed, purified, characterized                                 man gastric pathogen Helicobacter pylori. Nature 397:176&#226;&#8364;&#8220;180. human enzymes common FMN/FAD biosynthetic                                   4.   Arigoni, F., F. Talabot, M. Peitsch, M. D. Edgerton, E. Meldrum, E. Allet, <br />                                                                                              R. Fish, T. Jamotte, M. L. Curchod, H. Loferer. 1998. genome-based <br /> pathway (F. Mseeh, unpublished data). Sequence comparison                                    approach identification essential bacterial genes. Nat. Biotechnol. human FK FADS profile HMMs clearly indi-                                     16:851&#226;&#8364;&#8220;856. <br /> cates FADS potentially selective target (Fig.                            5.   Bacher, , S. Eberhardt, W. Eisenreich, M. Fischer, S. Herz, B. Illarionov, <br />                                                                                              K. Kis, G. Richter. 2001. Biosynthesis riboflavin. Vitam. Horm. <br /> 5C). agreement  reliable P values produced                               61:2&#226;&#8364;&#8220;49. pairwise comparisons human FADS bac-                              6.   Bacher, , S. Eberhardt, G. Richter. 1996. Biosynthesis riboflavin, <br /> terial counterparts, human FK sequence similar-                                p. 657&#226;&#8364;&#8220;664. F. C. Neidhardt et al. (ed.), Escherichia coli Salmonella: <br />                                                                                              cellular molecular biology. ASM Press, Washington, D.C. <br /> ity bacterial FK domains higher similarity                             7.   Badarinarayana, V., P. W. Estep, J. Shendure, J. Edwards, S. Tavazoie, F. bacterial FK domains  example,                                   Lam, G. M. Church. 2001. Selection analyses insertional mutants <br />                                                                                              using subgenic-resolution arrays. Nat. Biotechnol. 19:1060&#226;&#8364;&#8220;1065. <br /> M. tuberculosis FK domain gives P value 10&#226;&#171;&#186;9 human <br />                                                                                         8.   Bandrin, S. V., M. Y. Beburov, P. M. Rabinovich,   Stepanov. 1979. <br /> FK P value 4 &#228;&#161;&#160; 10&#226;&#171;&#186;6 H. pylori FK domain).                                  Riboflavin auxotrophs Escherichia coli. Genetika 15:2063&#226;&#8364;&#8220;2065.  Rus- <br />    conclusion, genetic footprinting E. coli combination                             sian.) <br />                                                                                         9.   Bandrin, S. V., P. M. Rabinovich,   Stepanov. 1983. 3 linkage groups comparative genome analysis reconstructed metabolic                                  genes riboflavin biosynthesis Escherichia coli. Genetika 19:1419&#226;&#8364;&#8220; <br /> subsystems pathways allowed identify number                                  1425.  Russian.) antibacterial targets involved biosynthesis ad-                       10.   Beckwith, J. R., E. R. Signer. 1966. Transposition Lac region                                                                                              Escherichia coli.  Inversion Lac operon transduction Lac enylate cofactors NAD(P), CoA, FAD.                                       &#226;?&#190;80. J. Mol. Biol. 19:254&#226;&#8364;&#8220;265. <br /> attractive targets pathways adenylyltrans-                        11.   Begg, K. J., S. J. Dewar, W. D. Donachie. 1995. new Escherichia coli <br /> ferases: NaMNAT, PPAT, FADS (Fig. 6).                                       cell-division gene, ftsk. J. Bacteriol. 177:6211&#226;&#8364;&#8220;6222. <br />                                                                                        12.   Begley, T. P., C. Kinsland, R.  Mehl,  Osterman, P. Dorrestein. <br /> targets (NaMNAT PPAT) validated direct-                                     2001. biosynthesis nicotinamide adenine dinucleotides bacteria. <br /> ed-knockout strategy S. aureus. orthologs target                              Vitam. Horm. 61:103&#226;&#8364;&#8220;119. <br /> enzymes representative pathogens, human                          13.   Begley, T. P., C. Kinsland, E. Strauss. 2001. biosynthesis                                                                                              coenzyme bacteria. Vitam. Horm. 61:158&#226;&#8364;&#8220;171. <br /> counterparts, overproduced, purified, characterized                           14.   Benton, B. L., J. Ving, F. Malouin, P. K. Martin, M. B. Schmid, D. Sun. direct vitro assays. Comparative analysis adenylate                                 March 2000. Methods screening compounds active Staphylococcus <br /> cofactor biosynthesis provides illustration general ap-                              aureus target genes. U.S. patent 6,037,123. <br />                                                                                        15.   Benton, B. L., J. Ving, F. Malouin, P. K. Martin, M. B. Schmid, D. Sun. <br /> proach extended functional subsystems                                2001. Methods screening compounds active Staphylococcus <br /> reveal novel antibacterial targets.                                                          aureus target genes. U.S. patent 6,228,588. <br />                                                                                        16.   Bernal, , U. Ear, N. Kyrpides. 2001. Genomes OnLine Database <br />                                                                                              (GOLD): monitor genome projects world-wide. Nucleic Acids Res. <br />                                 ADDENDUM                                                     29:126&#226;&#8364;&#8220;127. <br />                                                                                        17.   Black, M. T., L. K. Shilling, R. K. Stodola, R. L. Warren,  L. Kosmatka, <br />    manuscript reviewed, genome-wide                                    R. O. Nicholas, L. M. Palmer, M.  Lonetto, J. C. Fedon, J. E. Hodgson, <br />                                                                                              D. J. C. Knowles. June 2001. RibB. U.S. patent 6,252,044. <br /> analysis essential genes H. influenzae published (2).                        18.   Blattner, F. R., G. Plunkett, C.  Bloch, N. T. Perna, V. Burland, M. Riley, <br /> Comparing essential genes divergent organisms use-                                J. ColladoVides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, N. W. <br /> ful  defining common essential pathways life                                 Davis, H.  Kirkpatrick, M.  Goeden, D. J. Rose, B. Mau, Y. Shao. <br />                                                                                              1997. complete genome sequence Escherichia coli K-12. Science potential targets development antimicrobial thera-                                277:1453&#226;&#8364;&#8220;1462. <br /> peutics&#226;&#8364;? (2). genome-scale list essential nonessen-                         19.   Bresler, S. E., E.  Glazunov, D.  Perumov. 1972. Study tial genes E. coli currently preparation.                                           operon riboflavin biosynthesis Bacillus subtilis. IV. Regulation                                                                                              synthesis riboflavin synthetase. Investigation riboflavin transport <br />                                                                                              cell membrane. Genetika (USSR) 8:109&#226;&#8364;&#8220;117. <br />                           ACKNOWLEDGMENTS                                              20.   Bru&#194;&#168;ckner, R. 1997. Gene replacement Staphylococcus carnosus                                                                                              Staphylococcus xylosus. FEMS Microbiol. Lett. 151:1&#226;&#8364;&#8220;8. <br />    grateful William Reznikoff Igor Goryshin (University                  21.   Calder, R. B., R. S. B. Williams, G. Ramaswamy, C. O. Rock, E. Campbell, Wisconsin, Madison); Jerry Jendrisak Les Hoffman (Epicentre                           S. E. Unkles, J. R. Kinghorn, S. Jackowski. 1999. Cloning char- <br /> Technologies); Tadhg Begley (Cornell University, Ithaca, N.Y.); Zol-                         acterization eukaryotic pantothenate kinase gene (panK) Aspergil- <br /> tan Oltvai (Northwestern University, Chicago, Ill.); Henry Burd,                         lus nidulans. J. Biol. Chem. 274:2014&#226;&#8364;&#8220;2020. <br />                                                                                        22.   Casadaban, M. J., S. N. Cohen. 1980. Analysis gene control signals <br /> Niels Larsen, Gregory Kogan, Yuri Kuniver, Yuri Grechkin, Rob- <br />                                                                                              DNA fusion cloning Escherichia coli. J. Mol. Biol. 138:179&#226;&#8364;&#8220;207. <br /> ert Haselkorn (Integrated Genomics) valuable discussions, help,                    23.   Chalker,  F., H. W. Minehart, N. J. Hughes, K. K. Koretke, M.  Lonetto, support project. thank George Church (Har-                            K. K. Brinkman, P. V. Warren,  Lupas, M. J. Stanhope, J. R. Brown, vard Medical School, Boston, Mass.) gift pKO3, Mark                               P. S. Hoffman. 2001. Systematic identification selective essential genes Smeltzer (University Arkansas Medical Sciences) gift                       Helicobacter pylori genome prioritization allelic replacement mu- <br /> pCRII::tet RN4220, Reinhold Bru    &#194;&#168;ckner (Universita&#194;&#168;t Tu <br />                                                              &#194;&#168;bingen,                        tagenesis. J. Bacteriol. 183:1259&#226;&#8364;&#8220;1268. <br /> Tu&#194;&#168;bingen, Germany) pBT2, Don Clewell (University Mich-                     24.   Cole, S. T., R. Brosch, J. Parkhill, T. Garnier, C. Churcher, D. Harris, S. V. <br /> igan, Ann Arbor) plasmid pAM120 containing Tn916.                                    Gordon, K. Eiglmeier, S. <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span>Gas</span>, C. E. Barry, F. Tekaia, K. Badcock, D. <br />                                                                                              Basham, D. Brown, T. Chillingworth, R. Connor, R. Davies, K. Devlin, T. <br />                                                                                              Feltwell, S. Gentles, N. Hamlin, S. Holroyd, T. Hornby, K. Jagels,  Krogh, <br />                                  REFERENCES                                                  J. McLean, S. Moule, L. Murphy, K. Oliver, J. Osborne, M.  Quail, M.  <br />   1. Akerley, B. J., E. J. Rubin,  Camilli, D. J. Lampe, H. M. Robertson,              Rajandream, J. Rogers, S. Rutter, K. Seeger, J. Skelton, R. Squares, S. <br />      J. J. Mekalanos. 1998. Systematic identification essential genes vitro         Squares, J. E. Sulston, K. Taylor, S. Whitehead, B. G. Barrell. 1998. <br />      mariner mutagenesis. Proc. Natl. Acad. Sci. USA 95:8927&#226;&#8364;&#8220;8932.                           Deciphering biology Mycobacterium tuberculosis complete <br />   2. Akerley, B. J., E. J. Rubin, V. L. Novick, K. Amaya, N. Judson, J. J.               genome sequence. Nature 393:537&#226;&#8364;&#8220;544. <br />      Mekalanos. 2002. genome-scale analysis identification genes           25.   Cynamon, M. H., T. B. Sorg,  Patapow. 1988. Utilization  <br />  VOL. 184, 2002                                                                GENETIC FOOTPRINTING COFACTOR BIOSYNTHESIS                                       4571 <br />  <br />       tabolism NAD Haemophilus parainfluenzae. J. Gen. Microbiol 134:            49. Hutchison, C. , S. N. Peterson, S. R. Gill, R. T. Cline, O. White, C. M. <br />       2789&#226;&#8364;&#8220;2799.                                                                           Fraser, H. O. Smith, J. C. Venter. 1999. Global transposon mutagenesis <br /> 26.   Dahmen, W., B. Webb, J. Preiss. 1967. deamido-diphosphopyri-                 minimal mycoplasma genome. Science 286:2165&#226;&#8364;&#8220;2169. <br />       dine nucleotide diphosphopyridine nucleotide pyrophosphorylases           50. Izard, T.,  Geerlof. 1999. crystal structure novel bacterial <br />       Escherichia coli yeast. Arch. Biochem. Biophys. 120:440&#226;&#8364;&#8220;450.                     adenylyltransferase reveals half sites reactivity. EMBO J. 18:2021&#226;&#8364;&#8220;2030. <br /> 27.   Daugherty, M., B. Polanuyer, M. Farrell,  Lykidis, V. Cre&#194;&#180;cy-Lagard,        51. Jackowski, S., J. H. Alix. 1990. Cloning, sequence, expression        Osterman. 2002. Complete reconstitution human coenzyme               pantothenate permease (panF) gene Escherichia coli. J. Bacteriol. <br />       biosynthetic pathway comparative genomics. J. Biol. Chem. 277:21431&#226;&#8364;&#8220;             172:3842&#226;&#8364;&#8220;3848. <br />       21439.                                                                           52. Jenks, P. J., C. Chevalier, C. Ecobichon,  Labigne. 2001. Identifica- <br /> 28.   Daugherty, M., V. Vonstein, R. Overbeek,  Osterman. 2001. Archaeal             tion nonessential Helicobacter pylori genes using random mutagenesis <br />       shikimate kinase, new member GHMP-kinase family. J. Bacteriol.              loop amplification. Res. Microbiol. 152:725&#226;&#8364;&#8220;734. <br />       183:292&#226;&#8364;&#8220;300.                                                                     53. Jensen, K. F. 1993. Escherichia coli K-12 wild types W3110 29.   Debouck, C., J. C. Fedon, D. D. Jaworski, J. Mooney, L. M. Palmer, C. M.             MG1655 rph frameshift mutation leads pyrimidine starvation <br />       Traini, M. Wang, R. L. Warren, Y. Y. Zhong. December 2001. RibH.                 low pyrE expression levels. J. Bacteriol. 175:3401&#226;&#8364;&#8220;3407. <br />       U.S. patent 6,326,462.                                                           54. Judson, N., J. J. Mekalanos. 2000. Transposon-based approaches 30.   Eddy, S. R. 1998. Profile hidden Markov models. Bioinformatics 14:755&#226;&#8364;&#8220;               identify essential bacterial genes. Trends Microbiol. 8:521&#226;&#8364;&#8220;526. <br />       763.                                                                             55. Kreiswirth, B. N., S. Lofdahl, M. J. Betley, M. O&#226;&#8364;&#8482;Reilly, P. M. Schlievert, <br /> 31.   Emanuelli, M., F. Carnevali, F. Saccucci, F. Pierella,  Amici, N. Raffaelli,       M. S. Bergdoll, R. P. Novick. 1983. toxic shock syndrome exotoxin <br />       G. Magni. 2000. Human NMN adenylyltransferase: molecular cloning,                structural gene detectably transmitted prophage. Nature 305: <br />       chromosomal localization, tissue mRNA levels, bacterial expression,              709&#226;&#8364;&#8220;712. <br />       enzymatic properties. J. Biol. Chem. 276:406&#226;&#8364;&#8220;412.                                56. Kreneva, R. , M. S. Gel&#226;&#8364;&#8482;fand,   Mironov, Y.  Yomantas, Y.  Kozlov, <br /> 32.   Fleischmann, R. D., M. D. Adams, O. White, R.  Clayton, E. F. Kirkness,             S. Mironov, D.  Perumov. 2000. Inactivation ypaA gene        R. Kerlavage, C. J. Bult, J. F. Tomb, B.  Dougherty, J. M. Merrick, K.          Bacillus subtilis; analysis resulting phenotypic expression. Russ. <br />       McKenney, G. Sutton, W. Fitzhugh, C. Fields, J. D. Gocayne, J. Scott, R.             J. Genet. 36:972&#226;&#8364;&#8220;974. <br />       Shirley, L.  Liu,  Glodek, J. M. Kelley, J. F. Weidman, C.  Phillips, T.   57. Lerner, F., M. Niere,  Ludwig, M. Ziegler. 2001. Structural       Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T.               functional characterization human NAD kinase. Biochem. Biophys. Res. <br />       Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L.              Commun. 288:69&#226;&#8364;&#8220;74. <br />       Fuhrmann, N. S. M. Geoghagen, C. L. Gnehm, L.  McDonald, K. V.                 58. Link,  J., D. Phillips, G. M. Church. 1997. Methods generating <br />       Small, C. M. Fraser, H. O. Smith, J. C. Venter. 1995. genome               precise deletions insertions genome wild-type Escherichia coli: <br />       random sequencing assembly Haemophilus influenzae Rd. Science                 application open reading frame characterization. J. Bacteriol. 179:6228&#226;&#8364;&#8220; <br />       269:496&#226;&#8364;&#8220;512.                                                                         6237. <br /> 33.   Flugel, R. S., Y. Hwangbo, R. H. Lambalot, J. E. Cronan, C. T. Walsh.        59. Loferer, H.,  Jacobi,  Posch, C. Gauss, S. Meier-Ewert, B. Seiz- <br />       2000. Holo-(acyl carrier protein) synthase phosphopantetheinyl trans-            inger. 2000. Integrated bacterial genomics discovery novel anti- <br />       fer Escherichia coli. J. Biol. Chem. 275:959&#226;&#8364;&#8220;968.                                 microbials. Drug Discov. Today 5:107&#226;&#8364;&#8220;114. <br /> 34.   Fraser, C. M., J. D. Gocayne, O. White, M. D. Adams, R.  Clayton, R. D.        60. Magni, G.,  Amici, M. Emanuelli, N. Raffaelli, S. Ruggieri. 1999. <br />       Fleischmann, C. J. Bult,  R. Kerlavage, G. Sutton, J. M. Kelley, J. L.             Enzymology NAD&#226;&#171;&#185; synthesis. Adv. Enzymol. Relat. Areas Mol. Biol. <br />       Fritchman, J. F. Weidman, K. V. Small, M. Sandusky, J. Fuhrmann, D.                  73:135&#226;&#8364;&#8220;182. <br />       Nguyen, T. R. Utterback, D. M. Saudek, C.  Phillips, J. M. Merrick, J. F. <br />                                                                                        61. Magni, G., M. Emanuelli,  Amici, N. Raffaelli, S. Ruggieri. 1997. <br />       Tomb, B.  Dougherty, K. F. Bott, P. C. Hu, T. S. Lucier, S. N. Peterson, <br />                                                                                            Purification human nicotinamide-mononucleotide adenylyltransferase. <br />       H. O. Smith, C.  Hutchison, J. C. Venter. 1995. minimal gene <br />                                                                                            Methods Enzymol. 280:241&#226;&#8364;&#8220;247. <br />       complement Mycoplasma genitalium. Science 270:397&#226;&#8364;&#8220;403. <br />                                                                                        62. Martin, P. R., R. J. Shea, M. H. Mulks. 2001. Identification 35.   Freiberg, C. 2001. Novel computational methods anti-microbial target <br />                                                                                            plasmid-encoded gene Haemophilus ducreyi confers NAD  <br />       identification. Drug Discov. Today 6:S72&#226;&#8364;&#8220;S80. <br />                                                                                            dependence. J. Bacteriol. 183:1168&#226;&#8364;&#8220;1174. <br /> 36.   Freiberg, C., B. Wieland, F. Spaltmann, K. Ehlert, H. Brotz, H. La- <br />                                                                                        63. Mehl, R. , C. Kinsland, T. P. Begley. 2000. Identification       bischinski. 2001. Identification novel essential Escherichia coli genes <br />                                                                                            Escherichia coli nicotinic acid mononucleotide adenylyltransferase gene. J. <br />       conserved pathogenic bacteria. J. Mol. Microbiol. Biotechnol. <br />                                                                                            Bacteriol. 182:4372&#226;&#8364;&#8220;4374. <br />       3:483&#226;&#8364;&#8220;489. <br /> 37.   Galperin, M. Y., E. V. Koonin. 1999. Searching drug targets           64. Meis, R. 2000. EZ::TN transposon insertions target DNA vitro       microbial genomes. Curr. Opin. Biotechnol. 10:571&#226;&#8364;&#8220;578.                               highly random. Epicentre Forum 7:5. [Online.] http://www.epicentre.com/ <br /> 38.   Gawron-Burke, C., D. B. Clewell. 1984. Regeneration insertionally             forum.asp. <br />       inactivated streptococcal DNA fragments excision transposon             65. Mishra, P. K., P. K. Park, D. G. Drueckhammer. 2001. Identification <br />       Tn916 Escherichia coli: strategy targeting cloning genes          yacE (coaE) structural gene dephosphocoenzyme kinase       gram-positive bacteria. J. Bacteriol. 159:214&#226;&#8364;&#8220;221.                                   Escherichia coli K-12. J. Bacteriol. 183:2774&#226;&#8364;&#8220;2778. <br /> 39.   Geerlof, ,  Lewendon, W. V. Shaw. 1999. Purification charac-         66. Moat,  G., J. W. Foster. 1987. Biosynthesis salvage pathways       terization phosphopantetheine adenylyltransferase Escherichia coli.          pyridine nucleotides, p. 1&#226;&#8364;&#8220;20. D. Dolphin O. Avramovich (ed.), <br />       J. Biol. Chem. 274:27105&#226;&#8364;&#8220;27111.                                                      Pyridine nucleotide coenzymes, B. John Wiley <span id='am-30' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-31' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Sons, New York, <br /> 40.   Goryshin,  Y., J. Jendrisak, L. M. Hoffman, R. Meis, W. S. Reznikoff.          N.Y. <br />       2000. Insertional transposon mutagenesis electroporation released          67. Moir, D. T., K. J. Shaw, R. S. Hare, G. F. Vovis. 1999. Genomics       Tn5 transposition complexes. Nat. Biotechnol. 18:97&#226;&#8364;&#8220;100.                             antimicrobial drug discovery. Antimicrob. Agents Chemother. 43:439&#226;&#8364;&#8220;446. <br /> 41.   Goryshin,  Y., J.  Miller, Y. V. Kil, V.  Lanzov, W. S. Reznikoff.     68. Neidhardt, F. C., P. L. Bloch, D. F. Smith. 1974. Culture medium       1998. Tn5/IS50 target recognition. Proc. Natl. Acad. Sci. USA 95:10716&#226;&#8364;&#8220;              enterobacteria. J. Bacteriol. 119:736&#226;&#8364;&#8220;747. <br />       10721.                                                                           69. Nilsen, T. W. June 2001. Method identifying essential functional <br /> 42.   Grant, S. G. N., J. Jessee, F. R. Bloom, D. Hanahan. 1990. Differential          genes. U.S. patent 6,248,525. <br />       plasmid rescue transgenic mouse DNAs Escherichia coli methy-           70. Olland,  M., K. W. Underwood, R. M. Czerwinski, M. C. Lo,  Aula- <br />       lation-restriction mutants. Proc. Natl. Acad. Sci. USA 87:4645&#226;&#8364;&#8220;4649.                 baugh, J. Bard, M. L. Stahl, W. S. Somers, F. X. Sullivan, R. Chopra. <br /> 43.   Hamer, L., T. M. DeZwaan, M. V. Montenegro-Chamorro, S.  Frank,                2002. Identification, characterization, crystal structure Bacillus sub- <br />       J. E. Hamer. 2001. Recent advances large-scale transposon mutagenesis.            tilis nicotinic acid mononucleotide adenylyltransferase. J. Biol. Chem. 277: <br />       Curr. Opin. Chem. Biol. 5:67&#226;&#8364;&#8220;73.                                                     3698&#226;&#8364;&#8220;3707. <br /> 44.   Hancock, R. E. W., D. Knowles. 1998. approaching end           71. Overbeek, R., N. Larsen, G. D. Pusch, M. D&#226;&#8364;&#8482;Souza, E. Selkov, Jr., N. <br />       antibiotic era? Curr. Opin. Microbiol. 1:493&#226;&#8364;&#8220;494.                                Kyrpides, M. Fonstein, N. Maltsev, E. Selkov. 2000. WIT: integrated <br /> 45.   Hare, R. S., S. S. Walker, T. E. Dorman, J. R. Greene, L. M. Guzman, T. J.           high-throughput genome sequence analysis metabolic  <br />       Kenney, M. C. Sulavik, K. Baradaran, C. Houseweart, H. Y. Yu,  Foldes,             construction. Nucleic Acids Res. 28:123&#226;&#8364;&#8220;125. <br />        Motzer, M. Walbridge, G. H. Shimer, K. J. Shaw. 2001. Genetic             72. Palmer, L. M., J. M. Pratt, M. Rosenberg. October 2000. Method       footprinting bacteria. J. Bacteriol. 183:1694&#226;&#8364;&#8220;1706.                               determining gene essentiality pathogen. U.S. patent 6,139,817. <br /> 46.   Heinemann, J.  1999. antibiotics cause antibiotic resistance. Drug         73. Penfound, T., J. W. Foster. 1996. Biosynthesis recycling NAD, <br />       Discov. Today 4:72&#226;&#8364;&#8220;79.                                                               p. 721&#226;&#8364;&#8220;730. F. C. Neidhardt et al. (ed.), Escherichia coli Salmonella: <br /> 47.   Hoffman, L. M., J. J. Jendrisak, R. J. Meis,  Y. Goryshin, W. S.               cellular molecular biology. ASM Press, Washington, D.C. <br />       Reznikoff. 2000. Transposome insertional mutagenesis direct sequenc-         74. Raffaelli, N., T. Lorenzi, P. L. Mariani, M. Emanuelli,  Amici, S. Rug- <br />       ing microbial genomes. Genetica 108:19&#226;&#8364;&#8220;24.                                        gieri, G. Magni. 1999. Escherichia coli NadR regulator endowed <br /> 48.   Hughes, K. T., D. Ladika, J. R. Roth, B. M. Olivera. 1983. indis-             nicotinamide mononucleotide adenylyltransferase activity. J. Bacteriol. <br />       pensable gene NAD biosynthesis Salmonella typhimurium. J. Bacte-              181:5509&#226;&#8364;&#8220;5511. <br />       riol. 155:213&#226;&#8364;&#8220;221.                                                               75. Reich, K. , L. Chovan, P. Hessler. 1999. Genome scanning Hae- <br />  4572        GERDES ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />       mophilus influenzae identification essential genes. J. Bacteriol. 181:    90. Thompson, J. D., D. G. Higgins, T. J. Gibson. 1994. Clustal-W: im- <br />       4961&#226;&#8364;&#8220;4968.                                                                           proving sensitivity progressive multiple sequence alignment 76.   Reidl, J., S. Schlor,  Kraiss, J. Schmidt-Brauns, G. Kemmer, E.                sequence weighting, position-specific gap penalties weight matrix <br />       Soleva. 2000. NADP NAD utilization Haemophilus influenzae. Mol.               choice. Nucleic Acids Res. 22:4673&#226;&#8364;&#8220;4680. <br />       Microbiol. 35:1573&#226;&#8364;&#8220;1581.                                                         91. Timberlake, W., V. Gavrias. November 1999. Identification essential <br /> 77.   Rosamond, L.,  Allsop. 2000. Harnessing power genome             survival genes. U.S. patent 5,976,828. <br />       search new antibiotics. Science 287:1973&#226;&#8364;&#8220;1976.                           92. Tritz, G. J. 1974. Characterization nadR locus Escherichia coli.  J. <br /> 78.   Santos, M. ,  Jimenez, J. L. Revuelta. 2000. Molecular character-            Microbiol. 20:205&#226;&#8364;&#8220;209. <br />       ization FMN1, structural gene monofunctional flavokinase       93. Vallari, D. S., C. O. Rock. 1985. Isolation characterization       Saccharomyces cerevisiae. J. Biol. Chem. 275:28618&#226;&#8364;&#8220;28624.                            Escherichia coli pantothenate permease (panF) mutants. J. Bacteriol. 164: <br /> 79.   Schmid, M. B. 1998. Novel approaches discovery antimicrobial <br />                                                                                            136&#226;&#8364;&#8220;142. <br />       agents. Curr. Opin. Chem. Biol. 2:529&#226;&#8364;&#8220;534. <br />                                                                                        94. Vallari, D. S., C. O. Rock. 1987. Isolation characterization 80.   Schweiger, M., K. Hennig, F. Lerner, M. Niere, M. Hirsch-Kauffmann, T. <br />                                                                                            temperature-sensitive pantothenate kinase (coaA) mutants Escherichia <br />       Specht, C. Weise, S. L. Oei, M. Ziegler. 2001. Characterization       recombinant human nicotinamide mononucleotide adenylyl transferase                   coli. J. Bacteriol. 169:5795&#226;&#8364;&#8220;5800. <br />       (NMNAT), nuclear enzyme essential NAD synthesis. FEBS Lett.                95. Waibel, , T. Hanazawa, G. Hinton, K. Shikano, K. J. Lang. 1989. <br />       492:95&#226;&#8364;&#8220;100.                                                                          Phoneme recognition using time-delay neural networks. IEEE Transact. <br /> 81.   Scorpio, , Y. Zhang. 1996. Mutations pncA, gene encoding                  Acoustics Speech Signal Process. 37:328&#226;&#8364;&#8220;339. <br />       pyrazinamidase/nicotinamidase, cause resistance antituberculous           96. Wang, H., W. Huang, Y.-J. Fei, H. Xia, T. L. Yang-Feng, F. H. Leibach, L. D. <br />       drug pyrazinamide tubercle bacillus. Nat. Med. 2:662&#226;&#8364;&#8220;667.                         Devoe, V. Ganapathy, P. D. Prasad. 1999. Human placental Na&#226;&#171;&#185;- <br /> 82.   Selkov, E., N. Maltsev, G. J. Olsen, R. Overbeek, W. B. Whitman. 1997.           dependent multivitamin transporter. Cloning, functional expression, gene <br />       reconstruction metabolism Methanococcus jannaschii                  structure, chromosomal location. J. Biol. Chem. 274:14875&#226;&#8364;&#8220;14883. <br />       sequence data. Gene 197:GC11&#226;&#8364;&#8220;GC26.                                               97. Wang, L. L., <br /> </body></html>