<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2006 Genetics Society America <br /> DOI: 10.1534/genetics.106.057893 <br />  <br />  <br />  <br />   Glial Neuronal Functions Drosophila Homolog Human <br />    SWI/SNF Gene ATR-X (DATR-X ) jing Zinc-Finger Gene Specify <br />            Lateral Positioning Longitudinal Glia Axons <br />  <br />                                Xuetao Sun, Tatiana Morozova Margaret Sonnenfeld1 <br />                  Department Cellular Molecular Medicine, University Ottawa, Ottawa, Ontario K1H 8M5, Canada <br />                                                         Manuscript received March 3, 2006 <br />                                                       Accepted publication April 27, 2006 <br />  <br />  <br />                                                              ABSTRACT <br />                 Neuronal&#226;&#8364;&#8220;glial communication essential constructing orthogonal axon scaffold               developing Drosophila central nervous (CNS). Longitudinal glia (LG) guide extending <br />               commissural longitudinal axons pioneer commissural neurons maintain glial survival <br />               positioning.  transcriptional regulatory mechanisms controlling processes               known. Previous studies showed midline function jing C2H2-type zinc-finger transcription <br />               factor partially required axon scaffold formation Drosophila CNS.               screened gain function enhancers jing gain function eye identified Drosophila <br />               homolog disease gene human thalassemia/mental retardation X-linked (ATR-X)               genes potential roles gene expression, translation, synaptic transmission, cell cycle. jing <br />               DATR-X reporter genes expressed CNS neurons glia, including LG. Coexpression <br />               jing DATR-X embryonic neurons synergistically affects longitudinal connective formation. <br />               embryogenesis, jing DATR-X autonomous nonautonomous roles lateral <br />               positioning LG, neurons, longitudinal axons shown cell-specific knockdown gene <br />               expression. jing DATR-X required autonomously glial survival. jing DATR-X mutations <br />               synergistic effects longitudinal axon formation suggesting functionally related. <br />               observations support model downstream gene expression controlled potential <br />               DATR-X&#226;&#8364;&#8220;Jing complex facilitates cellular positioning axon guidance, ultimately allowing proper <br />               connectivity developing Drosophila CNS. <br />  <br />  <br />  <br />  <br /> D    URING central nervous (CNS) develop- <br />        ment, axons navigate long distances faced attractive repulsive guidance cues <br />                                                                             (Kidd et al. 1998a; Kinrade et al. 2001). Signaling                                                                             cell&#226;&#8364;&#8220;cell contact maintain ipsilateral positions                                                                             LG connectives. fact, Slit&#226;&#8364;&#8220;Robo signaling properly interpreted (Tessier-Lavigne                          cancels attraction longitudinal axons Goodman 1996). interneurons, cell                            CNS midline Netrin&#226;&#8364;&#8220;Frazzled (Bhat 2005). Com- <br /> bodies lie CNS midline, project                      missureless (Comm) transmembrane protein axons midline form commissural                            prevents delivery Robo growth cones, <br /> tracts. &#226;&#8364;&#732;&#226;&#8364;&#732;decision&#226;&#8364;&#8482;&#226;&#8364;&#8482; axon cross midline                 specifically commissural neurons, allowing Drosophila ventral nerve cord (VNC)                             axons cross midline (Tear et al. 1996; Keleman <br /> vertebrate spinal cord depends differential                          et al. 2002; Keleman et al. 2005). downregulation response axons midline repellent Slit                      Robo genetic means overexpression comm attractant Netrins (Seeger et al. 1993; Tessier-                        results excess axons CNS midline (Kidd <br /> Lavigne 1994; Battye et al. 1999; Kidd et al. 1999;                         et al. 1998b).  differential localization Long et al. 2004; Bhat 2005). commissural axons                       Comm, Robo, Slit determines directions <br /> cross midline, turn fasciculate                        navigating axons scaffold follow. Slit&#226;&#8364;&#8220; <br /> longitudinal tracts run parallel midline                    Robo important conserved mechanism repelled midline Slit.                                      establish cellular positioning boundaries    ligand Slit Roundabout (Robo),                        developing vertebrate invertebrate nervous systems <br /> located longitudinal glia (LG) associated                        (Kidd et al. 1999; Rajagopalan et al. 2000a,b; Simpson <br /> pioneer neuron growth cones adjacent midline                         et al. 2000a,b; Rasband et al. 2003; Barresi et al. 2005). <br />                                                                                relationship neurons glia   1 <br />                                                                             formation Drosophila CNS axon tracts    Corresponding author: Department Cellular Molecular Medicine, <br /> University Ottawa, Ottawa, Ontario K1H 8M5, Canada.                      extensively studied genetic cell ablation methods <br /> E-mail: msonnenf@uottawa.ca                                                 (Hidalgo Brand 1997, 2000; Booth et al. 2000). <br />  <br /> Genetics 173: 1397&#226;&#8364;&#8220;1415 ( July 2006) <br />  1398                                      X. Sun, T. Morozova M. Sonnenfeld <br />  longitudinal axon tracts constructed ex-        disease gene MR syndromes <br /> tensions pioneer neurons (Bate Grunewald           (Gibbons et al. 1995b; Villard et al. 1996a,b; Abidi <br /> 1981; Jacobs Goodman 1989; Hidalgo Brand               et al. 1999; Xue et al. 2003; Tang et al., 2004). <br /> 1997). form longitudinal fascicle, dMP2              explore transcriptional mechanisms control- <br /> MP2 pioneer neurons extend axons posteriorly          ling axon patterning, investigated cell-specific <br /> contact anteriorly projecting growth cones          roles jing DATR-X regulating axon formation vMP2 pCC neurons ( Jacobs Goodman 1989).               embryonic CNS. roles DATR-X jing hemisegment, LG act mobile guideposts           CNS studied reducing expression specif- migrating axons (Hidalgo Booth 2000). Early            ically neurons glia using RNA interference (RNAi) <br /> ablation LG affects joining descending       analyzing GOF effects gene. ascending pioneer growth cones subsequent              jing DATR-X (1) autonomous roles CNS <br /> fasciculation defasciculation pioneer later         glial survival, (2) autonomous nonautonomous <br /> follower axons (Booth et al. 2000; Hidalgo Booth           roles LG axon positioning, (3) autonomous <br /> 2000). Despite important guidance role LG,          nonautonomous roles longitudinal axon  cells depend pioneer axons survival         growth. phenotypes jing DATR-X mutations <br /> (Hidalgo et al. 2001; Kinrade et al. 2001). addi-           derive perturbation extensive neuronal&#226;&#8364;&#8220; <br /> tion, contralateral neuron cell bodies needed          glial communication mechanisms govern CNS <br /> axon pathfinding longitudinal connective              axon scaffold formation. Early ubiquitous expression <br /> (Whitington et al. 2004). proper migration              DATR-X mRNA enrichment transcripts follower glia fly optic lobe requires preexisting     reporter gene expression embryonic neurons photoreceptor axon scaffold (Hidalgo et al. 2001;              glia reveal similarity expression pattern jing <br /> Kinrade et al. 2001; Dearborn Kunes 2004).                 (Sedaghat et al. 2002). combined effects jing  neuronal&#226;&#8364;&#8220;glial interactions, addition         DATR-X knockdown GOF synergistic guidance molecules, instrumental axon               phenotypes resulting single gene knockdown <br /> patterning (Oland Tolbert 2002).                           similar, providing strong evidence jing    jing gene originally identified genetic      DATR-X work   jing DATR-X <br /> screens regulators border cell migration         function instrumental neuronal&#226;&#8364;&#8220;glial commu- <br /> ovary midline cell development embryo-          nication mechanisms govern CNS axon scaffold <br /> genesis (Liu Montell 2001; Sedaghat et al. 2002).          formation. embryogenesis, jing transcripts accumulate CNS midline, adjacent neuroectoderm, brain, trachea (Sedaghat et al. 2002; Sonnenfeld et al. 2004).                      MATERIALS METHODS CNS midline trachea, Jing functions  <br /> stream basic helix-loop-helix PAS  Arnt&#226;&#8364;&#8220;Sim)-          Drosophila strains: flies raised standard <br />                                                                Drosophila cornmeal medium 25&#194;&#176; (Ashburner 1989). <br /> containing (bHLH-PAS) transcription factors control         collection chromosome transgenic EP strains, <br /> tyrosine kinase signaling epidermal growth         generated described R&#195;&#184;rth (1996), obtained <br /> factor receptor (Egfr) fibroblast growth factor         Hungarian Szeged stock center. GMR-Gal4 (second <br /> ceptor Breathless (Sedaghat et al. 2002; Sonnenfeld            chromosome) obtained J. Nambu (Hay et al. 1994; <br /> et al. 2004). CNS midline, jing required com-    Wing et al. 2002) paired (prd)-Gal4  chromosome) <br />                                                                obtained S. Crews. driver, ELAV-Gal4  X), <br /> missural longitudinal axon formation midline           used drive expression neurons promoter <br /> expression wild-type jing does completely rescue        fusion embryonic lethal abnormal vision gene <br /> axon defects mutants, suggesting functions       (Robinow White 1988; Yao White 1994); Jing contribute axon tract formation (Sonnenfeld         obtained Bloomington stock center. GCM-Gal4 <br /> et al. 2004).                                                  drives expression CNS glia expressing glial cells <br />                                                                missing (GCM) gene obtained M. Freeman <br />    search additional factors important jing       (Freeman et al. 2003; Hosoya et al. 1995; Jones et al. 1995;). <br /> function, carried genetic screen identify          btl-Gal4 used drive expression trachea (Shiga <br /> genes function modify jing gain-      et al. 1996). function (GOF) assay developing eye-imaginal            UAS-jingE, jing EMS alleles, jing deficiency [Df <br /> disc. Seven chromosome enhancer/promoter-                (2R)ST1] previously described (Sedaghat et al. 2002; <br />                                                                Sonnenfeld et al. 2004). UAS-jingU second chromosome <br /> tagged (EP) genes identified GOF                 insertion expresses jing determined situ <br /> hanced jing. group genes specifically          hybridization. <br /> interacts jing ommatidial              Genetic crosses: Balancers detected using Cyo wingless- <br /> formation.  identified Drosophila              lacZ TM3 ubx-lacZ. DATR-X jing overexpressed homolog disease gene human thalassemia/            CNS neurons crossing homozygous ELAV-Gal4 homo- <br />                                                                zygous UAS-jingE, EP(3)0635, UAS-DATR-X. coexpres- <br /> mental retardation (MR) X-linked (DATR-X) (Gibbons             sion experiments, flies carrying ELAV-Gal4 UAS-jingE et al. 1995b). human ATR-X gene encodes zinc-            crossed flies homozygous EP(3)0635. Controls  <br /> finger ATPase involved chromatin remodeling         cluded ELAV-Gal4/1, UAS-jingE/1, EP(3)0635/1.                                                  Lateral Positioning LG Axons                                           1399 <br />  <br /> double RNAi flies carrying ELAV-Gal4 UAS-jing RNAi         sequence tags (ESTs) SD07188 LD28477] obtained <br /> crossed flies carrying UAS-DATR-XRNAi.                           Open Biosystems jing cDNA previously obtained <br />    EP screen: Males EP line crossed virgin        Research Genetics (Sedaghat et al. 2002). DATR-X EST <br /> females carrying GMR-Gal4. eye morphologies F1 male          SD07188 contains 2382 bp DATR-X coding sequence beginning <br /> progeny cross analyzed dissecting            base pair 1968 respect length LD28477 cDNA <br /> microscope male flies straight wings             sequence. DATR-X clones SD07188 LD28477 subjected <br /> crossed virgin females carrying UAS-jingE. copy           DNA sequence analysis compared Gadfly <br /> UAS-jingE associated rough eye phenotype          database (Celera genomics) (Adams et al. 2000). Homology rough eye phenotype combination             searches using BLAST (http://www.ncbi.nlm.nih.gov). <br /> particular EP line GMR-Gal4 represented interaction.         length DATR-X sequence (accession AAL13821; FlyBase) number flies rough eyes determined          aligned Caenorhabditis elegans (AAD55361), mouse <br /> dissecting microscope 600 progeny               (AAC08741), human (AAB49970) using Clustal X (1.83). <br /> analyzed trials 25&#194;&#176;. Controls included       Protein sequences compared using BL2SEQ (http://www. trial included GMR-Gal4/UAS-jingE GMR-Gal4/1               ncbi.nlm.nih.gov/blast/bl2seq/wblast2.cgi) (Tatusova did rough eye 25&#194;&#176;. additional control,      Madden 1999). Protein domains analyzed using <br /> eyes inspected flies heterozygous UAS-jingE       SMART PROSITE (Hulo et al. 2004; Letunic et al. 2004). enhancing EP line. Enhancing lines subjected               length DATR-X cDNA (LD28477) subcloned genetic trials GMR-Gal4 UAS-jingE 25&#194;&#176;.       pUAST vector (Brand Perrimon 1993) EcoRI&#226;&#8364;&#8220; penetrance determined number rough eyes           KpnI fragment used generate germline transformant <br /> divided 25% carried GMR-Gal4, UAS-jingE,        flies standard procedures (Genetic Services, Cambridge, <br /> particular EP strain. total 591 individual chromo-       MA) (Rubin Spradling 1983). Expression DATR-X EP lines screened. Enhancer EP lines tested          confirmed multiple lines situ hybridization embryos interactions inter se additional EP lines expressing   expressing different DATR-X upstream activating sequence <br /> DNA regulatory genes random EP lines                transgenes control prd gene transactivator (prd- <br /> method described                                           GAL4). <br />    Microscopy: scanning electron microscopy (SEM), 1-           situ hybridization reporter gene construction: <br /> 3-day-old adult heads dissected, fixed, dehydrated      mount situ hybridization carried using <br /> 15-min incubations graded ethanol series. Dehydrated           digoxigenin-labeled RNA probes according Janody et al. <br /> heads sent Mount Sinai Bioimaging Center                (2000). DATR-X riboprobe generated linearizing <br /> (Toronto) sputter coating gold-palladium SEM           SD07188 EST HaeII. Sense probes synthesized <br /> examination. eyes sample examined              using T7 promoter antisense using Sp6 according processed using Adobe Photoshop software. light             manufacturers&#226;&#8364;&#8482; specifications (Roche). Probe specificity <br /> microscopy, 1- 3-day-old adult heads dissected,             tested GOF DATR-X embryos carrying prd-Gal4 <br /> analyzed Zeiss Axioskop, images captured           driver EP(3)0635 UAS-DATR-X transgenes. instar Nikon DXM1200 digital camera processed using Adobe            larval nerve cords dissected PBS fixed 4% <br /> Photoshop software. Confocal microscopy carried        <span id='am-19' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> 20 min. Fixed nerve cords rinsed BioRad1024 microscope using 340 water immersion [1.15               times fresh PBS washed 1 hr. Nerve cords numerical aperture (NA)] 320 (1.1 NA) objectives.                subjected situ hybridization described  <br />    Antibodies immunostaining Drosophila embryos:             Digoxigenin-stained nerve cords embryos mounted following antibodies obtained Develop-            80% glycerol examined Zeiss Axioskop 2. <br /> mental Studies Hybridoma Bank (Iowa City, IA): BP102 (1:10),           total 1553 bp 59 region jing obtained <br /> anti-Repo (8D12, 1:10) (Campbell et al. 1994), 1D4 (Fasciclin       PCR using Drosophila w1118 genomic DNA template, <br /> II, 1:10)(Van Vactor et al. 1993), anti-Robo (13C9, 1:10)       sequenced, cloned KpnI NotI sites (Kidd et al. 1998a). rabbit antibody b-galactosidase      pCaSper vector vivo expression. total 593 bp (Promega) used 1:100 dilution. HRP-conjugated goat           obtained PCR 59 DATR-X region surrounding <br /> anti-mouse IgG used 1:100. Texas red FITC-               EP element insertion site. fragment cloned conjugated secondary antibodies used confocal              EcoRI site pCaSper. generate transgenic strains, microscopy. Monoclonal antibody 2A12 used visualize          pCaSper plasmids injected plasmid <br /> tracheal tubules (1:3).                                             encoding D2-3 transposase w1118 embryos P-element- <br />    Embryos fixed PBS containing 4% formaldehyde             mediated transformation (Genetic Services) (Rubin embryos incubated overnight primary anti-             Spradling 1983). Multiple strains analyzed lacZ <br /> bodies PTN (13 PBS, 0.1% Triton X-100, 5% normal goat            expression. <br /> serum). Horseradish peroxidase-conjugated secondary anti-              RNA interference: DATR-X RNAi, 697-bp sequence <br /> bodies used <span id='am-24' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O2</span>/diaminobenzedine histochemis-            generated PCR base pair positions 3181&#226;&#8364;&#8220;3837 <br /> try. Stained embryos dehydrated ethanol             using LD28477 cDNA template. RNAi jing, 597-bp <br /> series, mounted methyl salicylate, examined            sequence generated PCR base pair position 2289 <br /> marski optics Zeiss Axioskop microscope. Images           ATG LD36562 cDNA. DNA fragments <br /> captured Nikon DXM1200 digital camera processed            individually subcloned sequentially, opposite direc- <br /> using Adobe Photoshop software.                                     tions, AvrII NheI sites pWIZ plasmid  gift <br />    Molecular genetic analysis DATR-X generation           R. Carthew) inverted sequence direc- <br /> UAS lines: Genomic cDNA sequences encoding Dro-             tionally cloned using XbaI NheI restriction enzymes (Lee <br /> sophila DATR-X homolog identified searching             Carthew 2003). final inverted repeat containing <br /> Berkeley Drosophila Genome Project (BDGP) database          vectors UAS-DATR-X RNAi UAS-jing RNAi confirmed <br /> EP(3)0635 associated gene (Adams et al. 2000; Rubin et al. 2000).   sequence analysis. generate transgenic strains, UAS- location EP element EP(3)0635 confirmed        DATR-X RNAi UAS-jing RNAi plasmids, injected <br /> PCR using inverted repeat primers flanking genomic DATR-X       plasmid encoding D2-3 transposase w1118 <br /> sequence (CGACGGGACCACCTTATGTTATTTCATCATG) fol-                     embryos P-element-mediated transformation (Genetic <br /> lowed DNA sequence analysis. DATR-X cDNAs [expressed             Services) (Rubin Spradling 1983). <br />  1400                                         X. Sun, T. Morozova M. Sonnenfeld <br />  <br />    independent UAS-DATR-X RNAi UAS-jing RNAi                                       TABLE 1 <br /> transformant lines bred homozygosity, mapped, tested double-stranded RNAi activity crossing CNS-                 EP lines enhanced jing gain function <br /> (ELAV-Gal4) tracheal-specific Gal4 (btl-Gal4) drivers immunostaining monoclonal antibodies 1D4 2A12,            EP line              Gene              Predicted function <br /> respectively. confirm downregulation mRNA expression, <br /> embryos carrying armadillo-Gal4 (Ahmad Henikoff 2001)          EP(3)0635     ATR-X/XNP             Chromatin remodeling UAS-DATR-X RNAi UAS-jing RNAi subjected             EP(3)0473     D1                    Chromatin remodeling situ hybridization using DATR-X jing riboprobes,          EP(3)3145     Dataxin-2             Translational regulation <br /> spectively. number stage 13 embryos reduced            EP(3)3060     lap                   Receptor-mediated <br /> expression transgene- Gal4 driver-carrying embryos                                            endocytosis <br /> compared control embryos carrying          EP(3)3705     CG32137               Microtubule binding <br /> driver transgene                                                                            (cell cycle) <br />    Quantitative analysis longitudinal axon glial cell       EP(3)3354     CG17383 ( JIGR-1)     Transcriptional regulation <br /> defects: number crossovers segment counted          EP(3)3084     CG15507               Unknown stage 16/17 control embryos expressing multiple <br /> jing DATR-X RNAi transgenes driven ELAV-           Rough eyed flies scored population carrying GMR- <br /> Gal4 GCM-Gal4 stained 1D4 anti-Robo.            Gal4, UAS-jing, EP line dissecting micro- independent transgenic lines controls ana-           scope. Crosses repeated times 25&#194;&#176; total- <br /> lyzed. Neuronal expression pWiz-DATR-X022, pWiz-DATR-           ing sample size .1200 flies. <br /> X28-MO1, pWiz-DATR-X26-MO2, pWiz-DATR-X28-MO2 <br /> resulted 20.9% (n &#194;&#188; 200), 22% (n &#194;&#188; 178), 27% (n &#194;&#188; 187), 30% (n &#194;&#188; 145) embryos axon defects, respectively.         enhancer screen carried using Neuronal expression pWiz-jingNC1A, pWiz-jingORO3,               chromosome collection EP lines  materials pWiz-jingNA2, pWiz-jing006 resulted 25% (n &#194;&#188; 180), 29%          methods) (R&#195;&#184;rth 1996). 591 chromosome EP <br /> (n &#194;&#188; 210), 28% (n &#194;&#188; 168), 31% (n &#194;&#188; 201) embryos        lines screened, 7 repeatedly interact <br /> axon defects, respectively. <br />                                                                    genetically coexpressed jing eye    Quantitative analysis glia carried light <br /> microscopy anti-Repo stained mount embryos           control GMR-Gal4 (Table 1; Figure 1). Eye-specific <br /> stage 11 16. Glia counted nerve cord segment       expression copy UAS-jing transgene embryos expressing independent jing DATR-X RNAi         visible effects ommatidial formation (Figure 1B) transgenes driven ELAV-Gal4 GCM-Gal4.           compared eyes heterozygous GMR-Gal4 (Figure <br /> Controls included embryos heterozygous RNAi <br />                                                                    1A).  coexpression jing EP(3)0635, <br /> transgene, Gal4 driver w1118. Standard deviations glial numbers sample determined using              controls expression Drosophila homolog <br /> Microsoft Excel sample sizes exceeded 250 nerve cord           human ATR-X (DATR-X), disrupted ommatidial  <br /> segments. Differences number glia sample         mation (Figure 1D). Eye-specific expression EP(3)0635 <br /> relative controls evaluated significance using         effect (Figure 1C). jing interacted Student&#226;&#8364;&#8482;s paired t-test. space longitudinal  <br />                                                                    4 EP lines, including EP(3)3145, EP(3)3705, <br /> nectives considered longitudinal break. <br />                                                                    EP(3)3354, EP(3)0473, resulting rougher eyes <br />                                                                    (Figure 1, F, K&#226;&#8364;&#8220;M). loss pigmentation occurred                                                                    case EP(3)3354. lines effect                          RESULTS <br />                                                                    expressed eye (Figure 1N; Table 2). <br />    Identification Drosophila homolog human                  additional EP lines interacted jing <br /> ATR-X additional genes involved jing function:              caused rough eye phenotype expressed approach identify genes interacting jing           eye, including EP(3)3084 (Figure 1G) use EP collection strains including genes tagged         EP(3)3060 (Figure 1I). GMR-Gal4-driven expression insertion transposable element containing UAS              EP(3)3084 associated morphological defects, <br /> sites (Szeged stock center, Szeged, Hungary) (Brand                mild reductions pigmentation, glossy eyes Perrimon 1993; R&#195;&#184;rth 1996). reasoned                   100% flies (Figure 1, G N; Figure 2A).  <br /> jing GOF dosage effects larval eye            coexpression EP(3)3084 jing resulted easily scored adult neural              enhancement phenotype eye size tissue sensitive studies transcriptional regulatory       reduced, eyes rougher, pigmentation mechanisms (Rubin 1988; Zipursky Rubin 1994)                   completely lost (Figure 1H; Figure 2B). total jing neuronal function (Sedaghat et al. 2002).           35% eyes phenotype enhanced (n &#194;&#188; 345). <br />  <br />                                                                                                                          &lt; <br />    Figure 1.&#226;&#8364;&#8221;Identification enhancer/promoter (EP) lines enhance jing gain  function eye.  M) Scanning <br /> electron micrographs adult eyes following genotypes shown:   GMR-GAL4/1; (B) GMR-GAL4/1; UAS-jingE/1; (C) <br /> GMR-GAL4 /1; EP(3)0635/1; (D) GMR-GAL4/1; UAS-jingE /1; EP(3)0635/1; (E) GMR-GAL4/1; EP(3)3145/1; (F) GMR- <br /> GAL4/1; UAS-jingE /1; EP(3)3145/1; (G) GMR-GAL4/1; EP(3)3084/1; (H) GMR-GAL4/1; UAS-jingE /1; EP(3)3084/1; <br />   GMR-GAL4/1; EP(3)3060/1; (J) GMR-GAL4/1; UAS-jingE /1; EP(3)3060/1; (K) GMR-GAL4/1; EP(3)3705 /1; UAS- <br /> jingE/1; (L) GMR-GAL4/1; UAS-jingE /1; EP(3)3354/1; (M) GMR-GAL4/1; UAS-jingE /1; EP(3)0473/1. (N) Penetrance <br /> values enhancing phenotypes. Bars, 100 mm. <br />  Lateral Positioning LG Axons   1401 <br />  1402                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br />                                                             TABLE 2 <br />                                          Random screening jing interacting EP lines <br />  <br />                          Eye phenotype            Sample size                             Eye phenotype          Sample size <br />                                                                       P[GMR-Gal4]- <br /> P[GMR-Gal4] 3                                                         EP(3)0473 3 <br />   EP(3)3339            Wild   type                    250               EP(3)3339           Rough                 200   (60) <br />   EP(3)3303            Wild   type                    178               EP(3)3303           Rough                 250   (40) <br />   EP(3)3258            Wild   type                    189               EP(3)3258           Rough                 214   (37) <br />   EP(3)3916            Wild   type                    201               EP(3)3916           Rough                 210   (23) <br />   EP(3)893             Wild   type                    200               EP(3)893            Rough                 276   (16) <br />   EP(3)3377            Wild   type                    215               EP(3)3377           Rough                 234   (42) <br />  <br /> P[GMR-Gal4]-                                                          P[GMR-Gal4]- <br /> EP(3)3084 3            Glossy, Ypigment               400             EP(3)3060 3           Rough                 450 <br />   EP(3)3339            change                      260               EP(3)3339           Enhanced               76   (20) <br />   EP(3)3303            change                      201               EP(3)3303           Enhanced               60   (53.3) <br />   EP(3)3258            change                      192               EP(3)3258           Enhanced               60   (6.7) <br />   EP(3)3916            change                      189               EP(3)3916           change             150 <br />   EP(3)893             change                      234               EP(3)893            change             200 <br />   EP(3)3377            change                      250               EP(3)3377           Enhanced              160   (75) <br />  <br /> P[GMR-Gal4];                                                          P[GMR-Gal4]; <br /> EP(3)0635 3            Wild   type                    650             EP(3)3145 3           Wild   type           630 <br />   EP(3)3339            Wild   type                    251               EP(3)3339           Wild   type           151 <br />   EP(3)3303            Wild   type                    234               EP(3)3303           Wild   type           145 <br />   EP(3)3258            Wild   type                    243               EP(3)3258           Wild   type           130 <br />   EP(3)3916            Wild   type                    267               EP(3)3916           Wild   type           125 <br />   EP(3)893             Wild   type                    154               EP(3)893            Mild   rough          218 (6.4) <br />   EP(3)3377            Wild   type                    134               EP(3)3377           Wild   type           245 <br />  <br /> P[GMR-Gal4];                                                          P[GMR-Gal4]; <br /> EP(3)3354 3            Wild   type                    550             EP(3)3705 3           Wild   type           601 <br />   EP(3)3339            Wild   type                    200               EP(3)3339           Wild   type           160 <br />   EP(3)3303            Wild   type                    250               EP(3)3303           Wild   type           134 <br />   EP(3)3258            Wild   type                    214               EP(3)3258           Wild   type           191 <br />   EP(3)3916            Wild   type                    210               EP(3)3916           Wild   type           230 <br />   EP(3)893             Wild   type                    276               EP(3)893            Wild   type           212 <br />   EP(3)3377            Wild   type                    234               EP(3)3377           Wild   type           213 <br />   Sample sizes include total number progeny. Sample sizes EP lines phenotype, EP(3)3060, EP(3)3084, EP(3)0473, include number eyes phenotype. Percentages parentheses indicate penetrance number enhanced eyes total number eyes phenotype. <br />  <br />  <br /> Eye-specific expression EP(3)3060 caused mild                Courey 2002), translational regulation [EP(3)3145, Datx2] <br /> rough eye (Figure 1I) loss pigmentation                (Satterfield et al. 2002; Ciosk et al. 2004), synaptic <br /> (Figure 2G). Enhancement phenotype                    vesicle transport [EP(3)3060] (Pena-Rangel et al. 2002), <br /> observed EP(3)3060 jing coexpression,             cell cycle control [EP(3)3705] (BDGP). test included rougher eyes (Figure 1J) loss pigmen-             specificity interactions jing, enhancers <br /> tation (Figure 2H). total 38% eyes                 crossed random EP lines, including EP(3) <br /> phenotype enhanced (n &#194;&#188; 301).                                3339, EP(3)3303, EP(3)3258, EP(3)3916, EP(3)893, <br />    Specificity EP interactions: genetic synergy            EP(3)3377, driven eye GMR- jing enhancers suggests                Gal4. enhancers, EP(3)3060 EP(3)0473 <br /> genes function similar processes neural                 interacted multiple EP lines (Table 2; Figure 2, cells.  group jing-interacting genes               J). addition, EP(3)3145 interacted EP(3)0893, involved wide range functions, including              resulting mildly rough eyes (Table 2). <br /> chromatin remodeling (DATR-X D1) (Rodriguez-                      addressed specificity D1 DATR-X <br /> Alfageme et al. 1980; Ashley et al. 1989; Gibbons et al.          enhancement testing interactions reg- <br /> 1995b; Picketts et al. 1996), transcriptional regulation          ulatory genes EP collection. DATR-X [EP(3)635] <br /> [EP(3) 3354] (Ewel et al. 1990; England et al. 1992;              did interact EP elements upstream <br /> Cutler et al. 1998; Brody et al. 2002; Bhaskar                genes encoding DNA-binding proteins, including <br />                                              Lateral Positioning LG Axons                                          1403 <br />  <br />  <br />  <br />  <br />   Figure 2.&#226;&#8364;&#8221;Genetic interactions jing enhancers. Light level images adult eyes following genotypes shown:   <br /> GMR-GAL4 /1; EP(3)3084/1; (B) GMR-GAL4/1; EP(3)3084/1; UAS-jingE/1; (C) GMR-GAL4/1; EP(3)3084/1; (D) <br /> EP(3)3377/1; GMR-GAL4/1; EP(3)0635/1; EP(3)3084/1; (E) GMR-GAL4/1; EP(3)3084/1; EP(3)3354/1; (F) GMR- <br /> GAL4/1; EP(3)3084/1; EP(3)3145/1; (G) GMR-GAL4/1; EP(3)3060/1; (H) GMR-GAL4/1; EP(3)3060/1; UAS-jingE/1; <br />   GMR-GAL4/1; EP(3)3060/1; EP(3)3377/1; (J) GMR-GAL4/1; EP(3)3060/1; EP(3)3303/1; (K) GMR-GAL4/1; <br /> EP(3)3060/1; EP(3)3145/1; (L) GMR-GAL4/1; EP(3)3354/1; (M) GMR-GAL4/1; EP(3)3354/1; EP(3)3303/1; (N) GMR- <br /> GAL4/1; EP(3)0635/1; EP(3)3354/1; (O) GMR-GAL4/1; EP(3)3354/1; EP(3)3145/1; (P) GMR-GAL4/1; EP(3)3145/1; <br /> (Q) GMR-GAL4/1; EP(3)3145/1; EP(3)0893/1; (R) GMR-GAL4/1; EP(3)3145/1; EP(3)0635/1; (S) GMR-GAL4/1; <br /> EP(3)0635/1. Bars, 50 mm. <br />  <br />  <br /> EP(3)1005 (n &#194;&#188; 411) EP(3)1096 (n &#194;&#188; 387). P                EP(3)3205 (35% penetrance, n &#194;&#188; 301), lies <br /> element line EP(3)1005 lies upstream CG15514,             upstream CG7552 encoding protein WW encodes protein containing DNA-binding                  domain interacts genetically cyclin E (Tseng <br /> BED finger chromatin-boundary-element-                   Hariharan 2002). contrast, EP(3)0473 inter- <br /> binding proteins transposases (FlyBase) (Aravind              acted EP(3)1005 (48%, n &#194;&#188; 205), EP(3)3205 (12%, <br /> 2000). P element line EP(3)1096 lies upstream           n &#194;&#188; 150), EP(3)3058 (83%, n &#194;&#188; 256), indicating gene encoding CXXC zinc finger (FlyBase,                      general transcriptional regulator. <br /> CG11033). EP(3)0635 did interact EP(3)3058                 Interenhancer interactions:  determined (n &#194;&#188; 200), lies upstream poly U binding            jing enhancers interacted eye. <br /> factor 68kD (pUf68) gene (Page-McCaw et al. 1999;                 Eye-specific coexpression EP(3)3084 EP(3)0635, <br /> Lasko 2000).  EP(3)0635 did interact                 EP(3)3354, EP(3)3145 resulted severe <br />  1404                                      X. Sun, T. Morozova M. Sonnenfeld <br />  <br />                                                           TABLE 3 <br />                                             Screening jing interacting EP lines <br />  <br /> P[GMR-Gal4];                                                     P[GMR-Gal4]; <br /> EP(3)3145           Eye phenotype         % penetrance      N    EP(3)0635             Eye phenotype       % penetrance    N <br />                                                                                                                     EP(3)0635       Glossy, YY pigment             17.9        217     EP(3)3084         YYY pigment              46.1        256 <br /> EP(3)3060       Bumpy, Y pigment               49.0a       300     EP(3)3354         Glossy, YYY pigment      17.2         93 <br /> EP(3)3354       Glossy, YY pigment             14.1         85     EP(3)3060         Glossy, YY pigment       88.9a        90 <br /> EP(3)3084       Rougher, YYY pigment           30.8a        65     EP(3)3705         Rough                    20          200 <br /> EP(3)3705       Rough                           8.5         94     EP(3)0473         Rough                    18.8        223 <br /> EP(3)0473       Rough                          15.0        167 <br />  <br /> P[GMR-Gal4];                                                     P[GMR-Gal4]; <br /> EP(3)3084           Eye phenotype         % penetrance      N    EP(3)3060             Eye phenotype       % penetrance    N <br />                                                                                                                 EP(3)3060       change                       0          250     EP(3)3354         Smaller, rougher         40           70 <br /> EP(3)3354       Glossy, YYY pigment            21.4a       234     EP(3)473          Rougher                   7.5a        90 <br /> EP(3)0473       Rougher                        24.9a       201     EP(3)3705         Smaller, rougher         18.7a        75 <br /> EP(3)3705       Rougher                        20.8a       120 <br />  <br /> P[GMR-Gal4]; <br /> EP(3)3354           Eye phenotype         % penetrance      N <br /> EP(3)3705               Small                  18.4        250 <br /> EP(3)0473               Rough                  15.6        225 <br />   Penetrance indicated percentage eyes phenotype progeny carrying GMR-Gal4 EP elements. <br /> N, sample sizes. <br />       Penetrance EP lines phenotype EP(3)3060, EP(3)3084, EP(3)0473 percentage enhanced <br /> eyes total eyes phenotype. <br />  <br />  <br /> rough eye phenotypes including losses pigmentation             EP(3)3354 (Figure 2, R O). Collectively, (Figure 2, D&#226;&#8364;&#8220;F; Table 3). Eye-specific expression              results identify group genes related func- <br /> EP(3)3060 resulted rougher eyes coexpressed               tions pertaining specifically jing dur- EP(3)3145, EP(3)3354, EP(3)0635, EP(3)0473,                  ing eye development. EP(3)3705 (Figure 2K; Table 3).                      ATR-X highly conserved: Previous studies <br /> widespread interactions EP(3)3060 suggest           showed ATR-X involved gene regulation results considered nonspecific (Table 2; Figure            chromatin remodeling functions (Cardoso et al. 1998; <br /> 2, J).                                                      Xue et al. 2003; Tang et al. 2004). Given GOF <br />    Eye-specific coexpression EP(3)3354                    interactions DATR-X jing eye, EP(3)3145 EP(3)0635 resulted glossy eyes              proposed regulatory relationship <br /> severely reduced pigmentation representing synergistic            DATR-X Jing embryonic CNS interactions (Table 3; Figure 2, N O).                initiated studies DATR-X. Database searches <br /> designated EP(3)3354 jing interacting gene regula-             genomic sequence flanking EP(3)0635 P ele- <br /> tory 1 ( JIGR1), given potential role regulating           ment confirmed transposon strain gene expression (Brody et al. 2002). EP element            obtained inserted 59-untranslated region <br /> JIGR1 lies upstream transcript CG17383,              predicted gene CG4548 encoding Drosophila identified differential head cDNA screen                 homolog human XNP/DATR-X (GadFly). <br /> (Brody et al. 2002). JIGR1 contains MADF domain                   DATR-X located cytological interval 96E1 shown Adf-1 transcription activator bind DNA            chromosome (BDGP) encodes predicted <br /> specifically developmentally regulated Dro-            polypeptide 1311 amino acids 49.4% overall <br /> sophila gene promoters (Ewel et al. 1990; England                 similarity human ATR-X alignment using <br /> et al. 1992; Cutler et al. 1998; Bhaskar Courey               Clustal X algorithm (Figure 3A). ATR-X homologs <br /> 2002). JIGR1 interacted EP(3)3705 resulting          human, mouse, C. elegans, planarians, reduced eye size (Table 3).                                     Drosophila (Figure 3, C) (Villard et al. <br />    EP(3)3145 lies upstream Drosophila ataxin2 (Datx2)          1999). sequence conservation interacted jing enhancers                 human DATR-X confined helicase screen randomly chosen EP lines.                SNF2 domains. SNF2 domains C. elegans, observed glossy eyes reduced pigmentation                  mouse, human ATR-X 54.9, 66.4, coexpression EP(3)3145 EP(3)635                 65.8% <span id='am-12' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> identity DATR-X, <br />                                               Lateral Positioning LG Axons                                           1405 <br />  <br />  <br />  <br />  <br />    Figure 3.&#226;&#8364;&#8221;ATR-X homologs.   Comparison DATR-X protein domain architecture predicted using SMART PROSITE. <br /> Invertebrate ATR-X truncated comparison mouse human shares high identities C-terminal amino acids proteins. Percentage identity SNF2 helicase C domains C. elegans, mouse, human comparison D. melanogaster. (B) Comparison amino acid identity regions containing mutations ATPase domain ATR-X human patients ATR-X syndrome. mutations include Mu K1600R Ped23 <br /> D2035V highly conserved regions. (C) Phylogenetic analysis ATR-X orthologs. distance sequences sum horizontal branch length separating  Sequences aligned using CLUSTALW (Tree View). sequence <br /> residues column colored basis alignment consensus, calculated automatically  color <br /> information, refer CLUSTAL W online help). Species designation follows protein acronym (Dm, Drosophila melanogaster; <br /> Ag, Anopheles gambiae; Mm, murine; Hs, human; Ce, Caenorhabditis elegans). Bar indicates number substitutions. <br />  <br />  <br /> respectively (Figure 3A). ATPase domains C.                 teins (Figure 3A) (Cardoso et al. 2000). zinc <br /> elegans, mouse, human ATR-X 58.9, 63,                 fingers required nuclear localization DNA <br /> 63% amino acid identity DATR-X, respec-               binding ATR-X (Cardoso et al. 2000).  tively (Figure 3A). ATR-X nucleosome-stimulated                possible invertebrate ATR-X proteins require <br /> ATPase activity used modify chromatin struc-            site-specific DNA targeting carry ATPase <br /> ture common site patient mutations                activities. C terminal ATR-X region rich (Gibbons et al. 1995b; Xue et al. 2003; Tang et al.                poly glutamine repeats (Figure 3A) proposed 2004). Mutations associated MR invariant               involved protein&#226;&#8364;&#8220;protein interactions residues amino acids sites human             present D. melanogaster C. elegans ATR-X <br /> mutations, designated Mu K1600R Ped23 D2035V,                  (Picketts et al. 1996; data shown). Sequence dis- conserved DATR-X (Figure 3B).                               tances ATR-X orthologs different species    differences amino acid se-                schematized Figure 3C. <br /> quences vertebrate invertebrate ATR-X. Human                   DATR-X jing-lacZ expressed CNS glia mouse ATR-X zinc-finger DNA-binding                  neurons: determine expression pattern DATR- <br /> main consisting multicysteine zinc-finger motifs          X development, hybridized mount <br /> (C2-C2) present invertebrate ATR-X pro-             control embryos digoxigenin-labeled antisense <br />  1406                                       X. Sun, T. Morozova M. Sonnenfeld <br />  sense DATR-X riboprobes. Ectopic activation                 sion copies transgene  <br /> EP(3)0635 detected prd-expressing stripes              average 65% segments longitudinal antisense DATR-X probe confirming specificity              connectives jing DATR-X coexpression (Figure probe EP line (Figure 4A). DATR-X transcripts           5D, arrowhead; Figure 5G) comparison expression present low level cellular blastoderm          copies UAS-jing (5% segments, n &#194;&#188; 80) embryos (Figure 4B) gastrulation (Figure                EP(3)0635 (18% segments, n &#194;&#188; 55) (Figure 5G). <br /> 4C). stage 15 embryos, DATR-X transcripts                 jing DATR-X coexpression syn- <br /> enriched neuroectoderm supraesophageal                  ergistic effects specifically embryonic CNS neurons. <br /> ganglion (brain) (Figure 4, D G). mature                   Neuronal-specific functions jing DATR-X ventral nerve cord, DATR-X mRNA expressed cells              required repulsion longitudinal axons glia position longitudinal axon tracts                 CNS midline: targeted DATR-X jing <br /> (Figure 4G, arrows) laterally located glia             mutations discern neuronal glial contribu- <br /> (Figure 4G, arrowhead). Similar CNS expression pat-                tions gene product axon scaffold formation. <br /> terns observed embryos hybridized                 DATR-X jing expression knocked using <br /> sense DATR-X riboprobes (Figure 4F). DATR-X tran-                  conditional RNAi (Hammond et al. 2001; Lee scripts present brains            Carthew 2003). total 697- 567-bp sequences <br /> instar wild-type larvae predominant optic           nonconserved regions DATR-X jing lobe region (Figure 4E).                                           separately subcloned P[UAST] derivative plasmid <br />    better understand pattern expression,              produce intron-spliced hairpin RNA corresponding <br /> costained embryos carrying jing- DATR-X-lacZ                   DATR-X jing genes, respectively. UAS/ <br /> reporter genes anti-b-galactosidase glial-            Gal4 (Brand Perrimon 1993) used <br /> specific anti-Repo antibody. Analysis confocal mi-              allow hairpin RNA conditionally downregulate <br /> croscopy revealed jing-lacZ DATR-X-lacZ               DATR-X jing expression specific cell lineages, <br /> expressed glial neuronal lineages.             confirmed situ hybridization. Pene- DATR-X-lacZ- jing-lacZ-expressing glia pre-            trance values CNS axon phenotypes associated sent positions characteristic LG (Figure 4, R).        neuronal glial knockdown jing DATR-X DATR-X-lacZ- jing-lacZ-expressing glia occupy            ranged 24 30%  materials methods). lateral (Figure 4, K L, arrowheads) ventral               Longitudinal connective formation analyzed <br /> regions (Figure 4, N, O, S).                                       homozygous jing mutant embryos    Pan-neural coexpression DATR-X jing syner-               neuronal-specific knockdown jing DATR-X <br /> gistically disrupts longitudinal axon formation:             stained antibody Fasciclin II (1D4). Neuronal <br /> coexpression jing DATR-X strongly affected adult            specificity directed ELAV-Gal4 driver. neuronal development wanted determine               mutant embryos, FasII-positive longitudinal axons ab- <br /> coexpression similar affects embryonic neurons.             errantly cross CNS midline stage 16 embryos <br /> jing DATR-X expressed postmitotic                  (Figure 6, B&#226;&#8364;&#8220;D, arrowheads; Figure 6M). addition, <br /> neurons ELAV-GAL4. Expression jing               breaks longitudinal connectives sug- <br /> DATR-X resulted subtle defects CNS axon            gesting axon outgrowth defects (Figure 6, C D, <br /> scaffold observed BP102 staining. defects            arrows). embryos simultaneous neuronal knock- <br /> included thinner longitudinal connectives (Figure 5, B             jing DATR-X FasII-positive lateral C, arrowheads) reduced spacing                 fascicles fuse single tract CNS midline <br /> anterior posterior commissures (Figure 5, B C,             (Figure 6E). results establish autonomous <br /> arrows). Additive effects observed expression           neuronal requirement jing DATR-X function copies jing (Figure 5E) DATR-X                 outgrowth lateral positioning longitudinal <br /> (Figure 5F) commissural axons properly              axons genetic synergy process. <br /> separated (Figure 5, E F, arrows).                                Consistent results, pan-neural knockdown <br />    Pan-neural coexpression jing DATR-X resulted             jing DATR-X associated high levels Robo severe defects axonal patterning expres-           CNS midline stage 12 (Figure 6, G <br />  <br />  <br />                                                                                                                                &lt; <br />   Figure 4.&#226;&#8364;&#8221;Drosophila ATR-X jing expressed CNS glia neurons. situ hybridization using DATR-X digoxigenin- <br /> labeled riboprobe showing wild-type mount embryos anterior left  G). Confocal micrographs stage 15 mount embryos (H&#226;&#8364;&#8220;S). Embryos stained polyclonal rabbit anti-b-galactosidase (FITC-conjugated secondary anti- <br /> body, green) anti-Repo (Texas Red-conjugated secondary antibody, red).   validate probe specificity, embryo <br /> ectopically expressing DATR-X ectodermal stripes (arrow) paired gene hybridized DATR-X riboprobe. em- <br /> bryo carries paired-GAL4 EP(3)0635. (B) DATR-X transcripts present cellular blastoderm embryos. (C) 5.5- 6-hr-old <br /> embryo showing DATR-X transcripts supraesophageal ganglion (arrowhead) neuroectoderm (arrow) sagittal view. <br /> (D) sagittal stage 15 embryo showing elevated DATR-X transcripts supraesophageal ganglion (arrowhead) ventral <br /> nerve cord (arrow). (E) instar larval CNS expressing DATR-X brain hemispheres (arrow) ventral ganglion <br />                                                 Lateral Positioning LG Axons                                              1407 <br />  <br />  <br />  <br />  <br /> (arrowhead). DATR-X expressed strongly ventral region ganglion (vg, arrowhead) optic lobe (ol, arrow). <br /> (F) situ hybridization using DATR-X negative control sense probe. (G) DATR-X expression stage 15 ventral nerve cord <br /> (ventral view). Strong expression observed cells lining longitudinal connectives (arrows) lateral cells (arrowhead). <br /> (H&#226;&#8364;&#8220;R) Focal planes DATR-X jing-lacZ expression longitudinal glia (LG) (arrows). (N O) Sagittal views DATR-X-lacZ expressed ventral glia (arrowhead) dorsal longitudinal glia (arrow). DATR-X-lacZ expressed neurons (double arrowhead). (P&#226;&#8364;&#8220;R) jing-lacZ strongly expressed rows longitudinal glial cells. (S) jing-lacZ expressed bilateral glial lineage (arrow) laterally located lineage (arrowhead) ventral focal plane. Bars:  F) <br /> 200 mm; (G) 30 mm; (H&#226;&#8364;&#8220;S) 50 mm. <br />  1408                                    X. Sun, T. Morozova M. Sonnenfeld <br />  H) stage 16 (Figure 6, J K). phenotypes      expression genes required Robo read variable (Figure 6K), severe case    Slit cue. Alternatively, jing DATR-X regulate <br /> Robo-positive axons present midline (Figure      longitudinal positioning Robo-independent man- <br /> 6K) severe cases midline Robo observed       ner (Kinrade et al., 2001). fewer hemisegments (Figure 6L, arrowhead).           Glial functions Jing DATR-X required observed breaks Robo-positive connectives (Figure         ipsilateral positioning longitudinal glia, glial sur- <br /> 6L, arrow). high Robo levels jing DATR-X          vival, patterning longitudinal axons: Genetic <br /> pan-neural mutants suggest genes           cell ablation experiments shown critical <br /> regulate robo expression.  medial axon           role neuronal&#226;&#8364;&#8220;glial communication plays displacement suggests mutations affect        pioneering longitudinal axon tracts (Booth et al. <br />                                                              2000; Hidalgo Booth 2000; Hidalgo et al. 2001; <br />                                                              Kinrade et al. 2001; Whitington et al. 2004).                                                              thought informative examine <br />                                                              jing DATR-X play role glial guidance                                                              longitudinal axons. Glial development examined <br />                                                              pan-glial expression jing DATR-X RNAi <br />                                                              transgenes control glial cells missing (gcm)- <br />                                                              Gal4 (Hosoya et al. 1995; Jones et al. 1995) using <br />                                                              anti-Repo marker assess glial fates (Campbell <br />                                                              et al. 1994; Xiong et al. 1994; Halter et al. 1995). <br />                                                                 Robo-positive Repo-positive LG                                                              misplaced medially stage 12 pan-glial <br />                                                              knockdown jing DATR-X (Figure 7, C&#226;&#8364;&#8220;F)                                                              jing hemizygotes (Figure 7B). Wild-type numbers                                                              glia mutants stage reveal jing                                                              DATR-X involved glial differentiation                                                              division specification cells (Figure 7O). <br />                                                              Later wild-type embryogenesis Robo present <br />                                                              axons  Figure 6I) glial  <br />                                                              ment restricted Robo-independent mechanisms, <br />                                                              including axon contact trophic support (Kinrade <br />                                                              et al., 2001). mature cord, LG <br />                                                              longitudinal axons medially misplaced                                                              CNS midline jing hemizygotes glial knock- <br />                                                              jing DATR-X (Figure 7, H&#226;&#8364;&#8220;J, M, N). Despite <br />                                                              maintenance glial&#226;&#8364;&#8220;axonal contact, numbers <br />                                                              total glia steadily decreased embryogenesis                                                              jing DATR-X glial mutants (Figure 7O).  <br />                                                              glial numbers unaffected neuronal-specific <br />  <br />  <br />                                                                 Figure 5.&#226;&#8364;&#8221;Pan-neural expression DATR-X jing                                                              embryonic CNS neurons synergistically affects axon pattern- <br />                                                              ing. Ventral views ventral nerve cord (VNC) stage <br />                                                              15 embryos stained BP102  F) shown ante- <br />                                                              rior    Control embryo heterozygous elav-Gal4                                                              w1118. longitudinal connectives connect hemiseg- <br />                                                              ment (arrowhead). arrow denotes space                                                              anterior posterior commissural axons. (B C) <br />                                                              overexpression jing (B) DATR-X (C) postmitotic <br />                                                              CNS neurons, longitudinal axons (arrowheads)                                                              space commissural axons smaller (arrows). <br />                                                              (D) Coexpression copy EP(3)0635 UAS-jingE                                                              postmitotic neurons synergistic effects. Note absence <br />                                                              longitudinal connectives (arrowhead). (E F) Additive <br />                                                              effects overexpressing copies UAS-jingE (E)                                                              EP(3)0635 (F) Note commissural axons                                                              separated (arrows) longitudinal connectives                                                              (arrowheads). (G) Quantitative analysis longitudinal  <br />                                                              nective formation. Bars, 50 mm. <br />                                               Lateral Positioning LG Axons                                           1409 <br />  <br />  <br />  <br />  <br />    Figure 6.&#226;&#8364;&#8221;DATR-X jing functions required specifically CNS neurons longitudinal connective formation. DATR-X jing RNAi transgene expression driven neurons ELAV-Gal4.  E F&#226;&#8364;&#8220;L) Shown close views VNC mount stage16 embryos stained anti-Fasciclin II monoclonal antibody (1D4)  E) anti-Robo (F&#226;&#8364;&#8220;L). Stage 12 <br /> embryos stained anti-Robo (F&#226;&#8364;&#8220;H).   wild-type embryos, longitudinal fascicles clearly delineated. (B&#226;&#8364;&#8220;D) jing (B C) DATR-X (D) mutants, longitudinal fascicles misroute midline (arrowheads) broken (arrows). <br /> (E) coexpression jing DATR-X RNAi transgenes neurons, longitudinal fascicles fuse tract midline. <br />                                                                                                                      (continued) <br />  1410                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> jing DATR-X knockdown despite medial misplace-                  Vertebrate ATR-X C2C2 zinc-finger motif ment mispositioning LG (Figure 7, K, L, O).                 amino terminus similar plant homeodomain  jing DATR-X glial-specific mutations                finger previously identified proteins involved perturb autonomous survival function                chromatin-mediated transcriptional regulation (Aasland compensated axon contact.                                et al. 1995; Gibbons et al. 1997). Interestingly, D. mela- <br />                                                                    nogaster C. elegans ATR-X proteins contain                                                                    zinc-finger domain, suggesting structures                       DISCUSSION <br />                                                                    acquired evolution necessity <br />    conserved role ATR-X embryonic CNS?                vertebrate chromatin remodeling mechanisms. Pa- candidates screen, chose             tients identified mutations ATPase <br /> study DATR-X possible involvement Jing CNS             zinc-finger domains ATR-X, confirming function disease relevance. Mutations human             essential functional regions protein (Villard <br /> ATR-X gene associated X-linked MR                 et al. 1996b,c; Gibbons Higgs 2000). <br /> phenotypes lead cognitive delay, facial dysmor-               Given absence zinc-finger domains phism, microcephaly, skeletal genital abnormali-               DATR-X, postulate invertebrate DATR-X proteins <br /> ties, neonatal hypotonia (Gibbons et al. 1995a,b;              complexed proteins containing nuclear <br /> Villard et al. 1996a,b; Gibbons Higgs 2000).                 targeting DNA-binding motif regulate gene <br /> total 87% MR genes fruit fly homolog              expression proper regulatory sites. 76% candidate functional ortholog revealing               role Jing embryonic expression <br /> remarkable conservation humans D. mela-                pattern mutant overexpression pheno- <br /> nogaster (Inlow Restifo 2004). orthologs               types similar DATR-X.  human MR genes cellular phenotypes involving                  ATPase domain DATR-X neurons, glia, neural precursor cells arise           conserved evolution regions <br /> defects proliferation, migration, process exten-            protein evolved suit specific needs <br /> sion arborization (Inlow Restifo 2004).                 cell. summary, different mechanisms ATR-X <br /> example, targeted mutation ATR-X early                   function different binding partners species <br /> forebrain mice leads cortical progenitor cell death          account divergence sequence reduced forebrain size  rube&#194;&#180; et al. 2005).              respect amino terminal Q-rich repeats addition, mutations genes controlling identity              main chromatin remodeling aspects ATR-X forebrain neuronal precursors result holopro-            remain similar. <br /> sencephaly brain hemispheres                      Jing encodes nuclear protein putative DNA- <br /> separate (Wallis Muenke 2000). increased                    binding transcriptional regulatory domains (Liu <br /> understanding molecular cellular bases              Montell 2001; Sedaghat et al. 2002). C2H2 <br /> hereditary MR critical generation drug               zinc fingers Jing similar (50% identical) treatments.                                                        mouse adipocyte enhancer binding protein <br />    ATR-X belongs SWI/SNF group chromatin                 2 (AEBP2) 25% homology remodeling proteins, use energy provided              Kru&#194;&#168;ppel family transcription factors, including <span id='am-1' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> hydrolysis disrupt histone&#226;&#8364;&#8220;DNA associations             encoding gli ZIC2 (Liu Montell 2001). AEBP2 nucleosomes different positions (Kingston                  function implicated chromatin remodeling events Narlikar 1999; Whitehouse et al. 1999).                   (Cardoso et al. 1998; Cao Zhang 2004) chromatin modulation allows access activators           strong expression brain  et al. 1999). repressors DNA binding sites target              Genetic screening identifies related group jing- <br /> genes. helicase C SNF2N domains ATR-X                   interacting genes: utilized background shown DNA translocase nucleo-                sensitive jing function conduct genetic screen remodeling activities (Xue et al. 2003; Tang et al.           eye. GOF screen, hypothesized 2004). Accordingly, mutations ATR-X                   misexpression jing eye combination mapped helicase C SNF2N domains,                  genes involved jing transcriptional regula-  60% homology DATR-X                   tion lead alterations gene expression conserved C. elegans humans. conser-             consequently disrupt ommatidial formation. vation supports conserved role Drosophila ATR-X              genetic relationship DATR-X jing em- chromatin remodeling (Tang et al. 2004; work).             bryonic neurons glia shows screen  <br />  <br /> (F) Wild-type Robo-positive glia present midline. (G H) Medial misplacement Robo-positive glia <br /> (arrowheads) occurs neuronal-specific knockdown jing DATR-X.   Lateral positioning Robo-positive axons wild- <br /> type embryos. (J&#226;&#8364;&#8220;L) Robo-positive axons cross midline (arrowheads) neuronal-specific knockdown DATR-X jing. <br /> Axonal breaks evident (arrow L). Arrow J denotes remaining connective. (M) Graph displaying average number longitudinal axons cross CNS midline embryo. <br />  Figure 7.&#226;&#8364;&#8221;DATR-X jing required glial patterning survival.  D G&#226;&#8364;&#8220;L) CNS glia identified                                                                                                         (continued) <br />  1412                                       X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> successful identifying genes function adult               stage 12, Robo present LG responds <br /> neuronal cells relevant jing function                 repulsive midline Slit molecules maintain lateral <br /> embryonic CNS. consistent previous               positioning (Kinrade et al. 2001). medial misplace- <br /> ings fly eye valid targeting genes        ment Robo- Repo-positive LG stage 12 function tissues (Raymond et al. 2004).              jing DATR-X glial-specific knockdown suggests    EP(3)3084 contains transposon proximity               breakdown Robo-dependent <br /> novel gene known FlyBase transcript identifier           repulsive mechanisms.  fact Robo <br /> CG15507. Despite strong effects EP(3)3084 expres-               protein present jing DATR-X glial sion eye specifically strongly enhanced          neuronal knockdown suggests robo expression coexpression jing, DAtx2, JIGR1.            regulated Jing DATR-X. Alternatively, Jing  gene specifically interacted            DATR-X regulate expression factor randomly chosen EP lines, suggesting                controls Robo reads Slit signal. support, <br /> functional relationship genes. EP             misrouting axons midline presence elements lines located 59 untranslated         Robo occurs calmodulin Son sevenless mutants <br /> region downstream genes, suggesting                proteins required process Sli <br /> result overexpression (BDGP). Given regulatory              signal (Fritz VanBerkum 2000). possible <br /> role MADF domains, possible JIGR1 regu-              jing DATR-X regulate expression factors <br /> lates gene expression Jing DATR-X (England                controlling glial neuronal positioning Robo- <br /> et al. 1992; Bhaskar Courey 2002). Alternatively,              independent fashion (Kinrade et al. 2001). <br /> JIGR1 regulate expression Jing/DATR-X                    jing DATR-X mutations clearly affect target gene. Likewise, DAtx2 involved regu-              Robo-mediated LG positioning.  glial survival lating translation protein essential           affected robo mutant embryos, glia die <br /> component Jing/DATR-X/JIGR1 complex                 despite continuous axonal contact jing DATR-X <br /> stream target genes (Satterfield et al. 2002;             glial-specific mutants.  loss CNS glia <br /> Ciosk et al. 2004). role orthologs transla-           reflect breakdown intrinsic survival pathway <br /> tional regulators MR shown Dro-                mediated jing DATR-X. expression jing <br /> sophila ortholog fragile-X MR 1 (Dfmr1). Dfmr1 regulates        DATR-X reporter genes glia consistent MAP1B homolog Futsch control synaptic                    role. Furthermore, jing DATR-X/ATR-X <br /> structure function embryonic Drosophila                 implicated cell survival processes CNS (Zhang et al. 2001).  genetic screening              CNS midline tracheal cells cortical progen- phenotypic analysis Drosophila power            itors, respectively (Sedaghat et al. 2002; Sonnenfeld <br /> decipher pathways cellular bases MR genes.              et al. 2004; rube&#194;&#180; et al. 2005). <br />    Neuronal glial functions jing DATR-X                    Second, robo mutants central pCC/MP2 <br /> dictate axon tract formation: wild-type Drosophila              fascicle, outer longitudinal fascicles, embryos, LG assume characteristic positions             affected.  jing DATR-X glial neuronal <br /> cross midline adjacent VNC segments (Ito               mutants outer fascicles fused, broken, et al. 1995). multiple mechanisms dif-           seen crossing midline. defects simi- <br /> ferent stages development, including response             lar ablation neurons glia pulsive attractive molecules, cell&#226;&#8364;&#8220;cell contact, trophic       genetic loss glia gcm mutants (Hosoya et al. 1995; <br /> support, axon contact (Kinrade et al., 2001). dis-           Jones et al. 1995; Vincent et al. 1996; Hidalgo Brand <br /> ruption processes perturb formation           1997). observations suggest multiple biological glial axonal scaffolds. expression jing          processes require proper function genes DATR-X reporter genes LG correlates LG             consistent important upstream role jing <br /> phenotypes associated mutations genes.               DATR-X glial neuronal differentiation. <br />  <br />  <br />  <br />  mount embryos stained anti-Repo. (E, F, M, N) Embryos stained anti-Robo. Shown close ventral views nerve cord stage 12/3  F) stage 15 (G&#226;&#8364;&#8220;N) embryos.   wild-type stage 12/3 embryos, longitudinal glia (LG) migrated medially positions guide pioneer longitudinal axons. (B&#226;&#8364;&#8220;D) contrast, LG inappropriately <br /> occupy positions CNS midline hemizygous jing embryos (B) glial-specific knockdown jing (C) DATR-X (D) driven gcm-Gal4. (E F) Robo-positive glia misplaced medially expression jing (E) DATR-X (F) RNAi transgenes CNS glia. (G) wild-type embryos, LG occupy cell wide row lining longitudinal <br /> connectives (arrows). LG observed dorsal plane view. (H&#226;&#8364;&#8220;J) jing hemizygotes (H) jing   DATR-X (J), glial mutants glia misplaced midline (arrows). (K L) jing (K) DATR-X (L) neuronal mutants, <br /> glia cross midline scattered nerve cord. (M N) Glial knockdown jing (M) DATR-X (N) results breaks Robo-positive longitudinal axons (arrows) inappropriate midline crossings (arrowheads). (N) Quantitative analysis glia stages 12 15 expressing jing DATR-X RNAi transgenes glia neurons. Bars, 50 mm. <br />                                                          Lateral Positioning LG Axons                                                     1413 <br />  <br />    Evidence accumulating chromatin accessibility                       Bhat, K. M., 2005 Slit-Roundabout signaling neutralizes netrin-- <br />                                                                                    frazzled-mediated attractant cue specify lateral positioning <br /> plays key role transcriptional regulation cell-                        longitudinal axon pathways. Genetics 170: 149&#226;&#8364;&#8220;159. <br /> type-specific gene expression CNS (Hsieh et al.                         Booth, G. E., E. F. Kinrade  Hidalgo, 2000 Glia maintain <br /> 2004). conservation ATPase domains                                    follower neuron survival Drosophila CNS development. similar phenotype DATR-X jing muta-                                Development 127: 237&#226;&#8364;&#8220;244. <br />                                                                                Brand,  H., N. Perrimon, 1993 Targeted gene expression tions expression patterns raises possi-                           means altering cell fates generating dominant pheno- <br /> bility Jing involved targeting chromatin                       types. Development 118: 401&#226;&#8364;&#8220;415. <br /> remodeling complex containing DATR-X transcrip-                             Brody, T., C. Stivers, J. Nagle W. F. Odenwald, 2002 Iden <br />                                                                                    tification novel Drosophila neural precursor genes using dif- <br /> tional target genes products required                                ferential embryonic head cDNA screen. Mech. Dev. 113: 41&#226;&#8364;&#8220;59. response longitudinal growth cones glia                          Campbell, G., H. Goring, T. Lin, E. Spana, S. Anderson et al., <br /> guidance cues.                                                                     1994 RK2, glila-specific homeodomain protein required                                                                                    embryonic nerve cord condensation viability Drosophila. <br />    summary, identified group genes                            Development 120: 2957&#226;&#8364;&#8220;2966. <br /> pertain jing function specifically genetically                          Cao, R., Y. Zhang, 2004 SUZ12 required histone <br /> interact adult neuronal cells. results                            methyltransferase activity silencing function EED- <br />                                                                                    EZH2 complex. Mol. Cell 15(1): 57&#226;&#8364;&#8220;67. <br /> specific neural glial developmental defects underlie                       Cardoso, C., S. Timsit, L. Villard, M. Khrestchatisky, M. Fontes problems axon guidance associated muta-                                et al., 1998 Specific interaction ATR-X gene product <br /> tions DATR-X jing. studies using targeted                              SET domain human EZH2 protein. Hum. Mol. <br />                                                                                    Genet. 7: 679&#226;&#8364;&#8220;684. <br /> mutations MR genes alleviate view brain                       Cardoso, C., Y. Lutz, C. Mignon, E. Compe, D. Depetris et al., <br /> phenotypes result generic effects height-                            2000 ATR-X mutations cause impaired nuclear location ened sensitivity brain.                                                     altered DNA binding properties ATR-X protein. J. Med. <br />                                                                                    Genet. 37: 746&#226;&#8364;&#8220;751. <br />    thank Marc Freeman gcm-gal4 S. Hayashi btl-gal4.          Ciosk, R., M. DePalma J. R. Priess, 2004 ATX-2, C. elegans <br /> thank Doug Holmyard Bioimaging Centre Mount Sinai Hos-                   ortholog ataxin 2, functions translational regulation pital performing SEM. thank Helga Agah help immu-                  germline. Development 131: 4831&#226;&#8364;&#8220;4841. <br /> nohistochemistry. thank Andrew Ridsdale OGHRI use               Cutler, G., K. M. Perry R. Tjian, 1998 Adf-1 nonmodular confocal microscope training. work supported              transcription factor contains TAF-binding Myb-like motif. <br /> grant Canadian Institutes Health Research M.S.                      Mol. Cell Biol. 18: 2252&#226;&#8364;&#8220;2261. <br />                                                                                Dearborn, R., Jr., S. Kunes, 2004 axon scaffold induced                                                                                    retinal axons directs glia destinations Drosophila optic <br />                                                                                    lobe. Development 131: 2291&#226;&#8364;&#8220;2303. <br />                        LITERATURE CITED                                        England, B. P.,  Admon R. Tjian, 1992 Cloning Dro- <br /> Aasland, R., T. J. Gibson  F. Stewart, 1995 PHD finger:                  sophila transcription factor Adf-1 reveals homology Myb onco- <br />      implications chromatin-mediated transcriptional regulation.               proteins. Proc. Natl. Acad. Sci. USA 89: 683&#226;&#8364;&#8220;687. <br />      TIBS 20: 56&#226;&#8364;&#8220;59.                                                           Ewel, , J. R. Jackson C. Benyajati, 1990 Alternative DNA- <br /> Abidi, F., N. J. Carpenter, L. Villard, M. Curtis, M. Fontes et al.,               protein interactions variable-length internucleosomal regions <br />      1999 Carpenter-Waziri syndrome results mutation                    associated Drosophila Adh distal promoter expression. Nucleic <br />      DATR-X.  J. Med. Genet. 85: 249&#226;&#8364;&#8220;251.                                    Acids Res. 18: 1771&#226;&#8364;&#8220;1781. <br /> Adams, M. D., S. E. Celniker, R.  Holt, C.  Evans, J. D. Gocayne           Freeman, M. R., J. Delrow, J. Kim, E. Johnson C. Q. Doe, <br />      et al., 2000 genome sequence Drosophila melanogaster.                  2003 Unwrapping glial biology: Gcm target genes regulating glial <br />      Science 5461: 2185&#226;&#8364;&#8220;2195.                                                      development, diversification function. Neuron 38: 567&#226;&#8364;&#8220;580. <br /> Ahmad, K., S. Henikoff, 2001 Modulation transcription                 Fritz, J. L., M. F. VanBerkum, 2000 Calmodulin son  <br />      factor counteracts heterochromatic gene silencing Drosophila.              enless dependent signaling pathways regulate midline crossing      Cell 104: 839&#226;&#8364;&#8220;847.                                                            axons Drosophila CNS. Development 27(9): 1991&#226;&#8364;&#8220;2000. <br /> Aravind, L., 2000 BED finger, novel DNA-binding domain                   Gibbons, R. J., D. R. Higgs, 2000 Molecular-clinical spectrum <br />      chromatin- boundary-element-binding proteins transpo-                  ATR-X syndrome.  J. Med. Genet. 97: 204&#226;&#8364;&#8220;212. <br />      sases. Trends Biochem. Sci. 25: 421&#226;&#8364;&#8220;423.                                  Gibbons, R. J., L. Brueton, V. J. J. Buckle Burn, J. Clayton-Smith, <br /> Ashburner, M., 1989 Drosophila: Laboratory Manual. Cold Spring                   B. C. C. Davison et al., 1995a clinical hematological <br />      Harbor Laboratory Press, Cold Spring Harbor, NY.                              features X-linked thalassemia/mental retardation syn- <br /> Ashley, C. T., C. G. Pendleton, W. W. Jennings,  Saxena                      drome (ATR-X).  J. Med. Genet. 55: 288&#226;&#8364;&#8220;299. <br />      C. V. Glover, 1989 Isolation sequencing cDNA clones             Gibbons, R. J., D. J. Picketts, L. Villard D. R. Higgs, <br />      coding Drosophila chromosomal protein D1. repeating motif                1995b Mutations putative global transcriptional regulator <br />      proteins recognize DNA. J. Biol. Chem. 264: 8394&#226;&#8364;&#8220;8401.               cause X-linked mental retardation thalassemia (ATR-X <br /> Bate, C. M., E. B. Grunewald, 1981 Embryogenesis                      syndrome). Cell 80: 837&#226;&#8364;&#8220;845. <br />      sect nervous II: second class neuron precursor cells          Gibbons, R. J., S. Bachoo, D. J. Picketts, S. Aftimos, B. Asenbauer <br />      origin intersegmental connectives. J. Embryol.                 et al., 1997 Mutations transcriptional regulator ATRX estab- <br />      Exp. Morphol. 61: 317&#226;&#8364;&#8220;330.                                                    lish functional significance PHD-like domain. Nat. Genet. <br /> Barresi, M. J. F., L. D. Hutson, C-B. Chien R. O. Karlstrom,                   17: 146&#226;&#8364;&#8220;148. <br />      2005 Hedgehog regulated Slit expression determines commis-                Halter, D. , J. Urban, C. Rickert, S. S. Ner, K. Ito et al., <br />      sure glial cell position zebrafish forebrain. Develop-             1995 homeobox gene repo required differentia- <br />      ment 132: 3643&#226;&#8364;&#8220;3656.                                                          tion maintenance glia function embryonic nervous <br /> Battye, R.,  Stevens J. R. Jacobs, 1999 Axon repulsion                  Drosophila. Development 121: 317&#226;&#8364;&#8220;332. <br />      midline Drosophila CNS requires slit function. Develop-        Hammond, S. M.,   Caudy G. J. Hannon, 2001 Post-tran- <br />      ment 126: 2475&#226;&#8364;&#8220;2481.                                                          scriptional gene silencing double-stranded RNA. Nat. Rev. rube&#194;&#180;, N. G., M. Mangelsdorf, M. Jagla, J. Vanderluit, D. Garrick               Genet. 2(2): 110&#226;&#8364;&#8220;119. <br />      et al., 2005 chromatin-remodeling protein ATRX critical        Hay, B. , T. Wolff G.M. Rubin, 1994 Expression baculovirus <br />      neuronal survival corticogenesis. J. Clin. Invest. 115: 258&#226;&#8364;&#8220;267.       P35 prevents cell death Drosophila. Development 120: 2121&#226;&#8364;&#8220;2129. <br /> Bhaskar, V.,  J. Courey, 2002 MADF-BESS domain fac-                   G.-P., S. Kim H.-S. Ro, 1999 Cloning characterization      tor Dip3 potentiates synergistic activation Dorsal Twist.              novel zinc finger transcriptional repressor. J. Biol. Chem. 274: <br />      Gene 299: 173&#226;&#8364;&#8220;184.                                                            14678&#226;&#8364;&#8220;14684. <br />  1414                                                X. Sun, T. Morozova M. Sonnenfeld <br />  <br /> Hidalgo, ,  H. Brand, 1997 Targeted neuronal ablation:              Page-McCaw, P. S., K. Amonlirdviman P.  Sharp, 1999 PUF60: <br />      role pioneer neurons guidance fasciculation               novel U2AF65-related splicing activity. RNA 5: 1548&#226;&#8364;&#8220;1560. <br />      CNS Drosophila. Development 124: 3253&#226;&#8364;&#8220;3262.                     Pena-Rangel, M. T.,  Rodriguez J. R. Riesgo-Escovar, <br /> Hidalgo, , G. Booth, 2000 Glia dictate trajectories pio-               2002 misexpression study examining dorsal thorax formation <br />      neer neurons Drosophila embryonic CNS. Development                  Drosophila melanogaster. Genetics 160: 1035&#226;&#8364;&#8220;1050. <br />      127: 393&#226;&#8364;&#8220;402.                                                          Picketts, D. J., D. R. Higgs, S. Bachoo, D. J. Blake, O. W. J. <br /> Hidalgo, , E. F Kinrade M. Georgiou, 2001 Drosophila                  Quarrell et al., 1996 ATRX encodes novel member      neuregulin vein maintains glial survival axon guidance           SNF2 family proteins: mutations point common mecha- <br />      CNS. Dev. Cell 1: 679&#226;&#8364;&#8220;690.                                             nism underlying ATR-X syndrome. Hum. Mol. Genet. 5: <br /> Hosoya, T., K. Takizawa, K. Nitta Y. Hotta, 1995 Glial cells                1899&#226;&#8364;&#8220;1907. <br />      missing: binary switch neuronal glial determina-         Rajagopalan, S., E. Nicolas, V. Vivancos, J. Berger B. J. Dick- <br />      tion Drosophila. Cell 82: 1025&#226;&#8364;&#8220;1036.                                    son, 2000a Crossing midline. Roles regulation Robo <br /> Hsieh, J., K. Nakashima, T. Kuwabara, E. Mejia F. H. Gage,                  receptors. Neuron 28: 767&#226;&#8364;&#8220;777. <br />      2004 Histone deacetylase inhibition-mediated neuronal differ-          Rajagopalan, S., V. Vivancos, E. Nicolas B. J. Dickson, <br />      entiation multipotent adult neural progenitor cells. Proc. Natl.        2000b Selecting longitudinal pathway. Robo receptors specify <br />      Acad. Sci. USA 101: 16659&#226;&#8364;&#8220;16664.                                           lateral position axons Drosophila CNS. Cell 103: <br /> Huang, Y., S. J. Myers R. Dingledine, 1999 Transcriptional               1033&#226;&#8364;&#8220;1045. <br />      pression REST: recruitment Sin3A histone deacetylase         Rasband, K., M. Hardy C. B. Chien, 2003 Generating X:  <br />      neuronal genes. Nat. Neurosci. 2(10): 867&#226;&#8364;&#8220;872.                          mation optic chiasm. Neuron 39: 885&#226;&#8364;&#8220;888. <br /> Hulo, N., C. J.  Sigrist, V. Le Saux, P. S. Langendijk-Genevaux, L.       Raymond, K., E. Bergeret,  Avet-Rochex, R. Griffin-Shea      Bordoli et al., 2004 Recent improvements PROSITE                    M.-O. Fauvarque, 2004 screen modifiers RacGAP(84C) <br />      database. Nucleic Acids Res. 32: D134&#226;&#8364;&#8220;D137.                                gain function Drosophila eye revealed LIM kinase <br /> Inlow, J. K., L. L. Restifo, 2004 Molecular comparative                 Cdi/TESK1 downstream effector Rac1 spermato- <br />      genetics mental retardation. Genetics 166: 835&#226;&#8364;&#8220;881.                     genesis. J. Cell Sci. 117: 2777&#226;&#8364;&#8220;2789. <br /> Ito, K., J. Urban G. M. Technau, 1995 Distribution, classifica-         Robinow, S., K. White, 1988 locus ELAV Drosophila mel- <br />      tion development Drosophila glial cells late embry-          anogaster expressed neurons developmental stages. Dev. <br />      onic early larval ventral nerve cord. Roux&#226;&#8364;&#8482;s Arch. Dev. Biol.          Biol. 126: 294&#226;&#8364;&#8220;303. <br />      204: 284&#226;&#8364;&#8220;307.                                                          Rodriguez-Alfageme, G. T., G.T. Rudkin andL. Cohen, 1980 Isolation, <br /> Jacobs, J. R., C. S. Goodman, 1989 Embryonic development                 properties cellular distribution D1, chromosomal protein      axon pathways Drosophila CNS. II. Behavior pioneer               Drosophila. Chromosoma 78: 1&#226;&#8364;&#8220;31. <br />      growth cones. J. Neurosci. 9: 2402&#226;&#8364;&#8220;2411.                               R&#195;&#184;rth, P., 1996 modular misexpression screen Drosophila <br /> Janody, F., J. Reischl N. Dostatni, 2000 Persistence                     detecting tissue- specific phenotypes. Proc. Natl. Acad. Sci. <br />      Hunchback terminal region Drosophila blastoderm              USA 93: 12418&#226;&#8364;&#8220;12422. <br />      embryo impairs anterior development. Dev. 127: 1573&#226;&#8364;&#8220;1582.              Rubin, G. M., 1988 Drosophila melanogaster experimental organ- <br /> Jones, B. W., R. D. Fetter, G. Tear C. S. Goodman, 1995 Glial               ism. Science 240: 1453&#226;&#8364;&#8220;1459. <br />      cells missing: genetic switch controls glial versus neuronal    Rubin, G. M.,  C. Spradling, 1983 Vectors P element-mediated <br />      fate. Cell 82: 1013&#226;&#8364;&#8220;1023.                                                  gene transfer Drosophila. Nucleic Acids Res. 11: 6341&#226;&#8364;&#8220;6351. <br /> Keleman, K., S. Rajagopalan, D. Cleppien, D. Teis, K. Palha et al.,         Rubin, G. M., L. Hong, P. Brokstein, M. Evans-Holm, E. Frise et al., <br />      2002 Comm sorts Robo control axon guidance Dro-                  2000 Drosophila complementary DNA resource. Science 5461: <br />      sophila midline. Cell 110: 415&#226;&#8364;&#8220;427.                                        2222&#226;&#8364;&#8220;2224. <br /> Keleman, K., C. Ribeiro B. J. Dickson, 2005 Comm function  <br /> </body></html>