<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Gene Duplication Zebrafish <br /> kit ligand Partitioning Melanocyte <br /> Development Functions kit ligand Keith  Hultman1, Nathan Bahary2, Leonard  Zon3,4, Stephen L. Johnson1* <br /> 1 Department Genetics, Washington University School Medicine, Saint Louis, Missouri, United States America, 2 Department Molecular Genetics Biochemistry, <br /> University Pittsburgh School Medicine, Pittsburgh, Pennsylvania, United States America, 3 Department Pediatrics, Children&#226;&#8364;&#8482;s Hospital, Boston, Massachusetts, <br /> United States America, 4 Howard Hughes Medical Institute, Boston, Massachusetts, United States America <br />  <br />  retention particular genes genome duplication zebrafish given insights genes evolve partitioning ancestral functions. examine partitioning expression patterns functions zebrafish kit ligands, kit ligand (kitla) kit ligand b (kitlb), discuss possible coevolution duplicated zebrafish kit receptors (kita kitb). situ hybridizations kitla mRNA expressed trunk adjacent notochord middle somite stages melanocyte migration later <br /> expressed skin, receptor required melanocyte survival. kitla expressed regions <br /> complementary kita receptor expression, including pineal gland, tail bud, ear. contrast, kitlb mRNA expressed brain ventricles, ear, cardinal vein plexus, regions generally complementary zebrafish <br /> kit receptor ortholog.  like kitla, kitlb expressed skin stages consistent melanocyte <br /> survival.  appears kita kitla maintained congruent expression patterns, kitb kitlb evolved divergent expression patterns. demonstrate interaction kita kitla morpholino knockdown <br /> analysis. kitla morphants, kitlb morphants, phenocopy null allele kita, defects melanocyte <br /> migration survival. Furthermore, kitla morpholino, kitlb morpholino, interacts genetically sensitized <br /> allele kita, confirming kitla functional ligand kita.  examine kitla overexpression embryos, results hyperpigmentation caused increase number size melanocytes. hyperpigmentation dependent kita function. conclude following genome duplication, kita kitla maintained receptor&#226;&#8364;&#8220;ligand relationship, coevolved complementary expression patterns, functional analysis reveals kita receptor&#226;&#8364;&#8482;s function embryo promoted interaction kitla. <br /> Citation: Hultman KA, Bahary N, Zon LI, Johnson SL (2007) Gene duplication zebrafish kit ligand partitioning melanocyte development functions kit ligand  <br /> PLoS Genet 3(1): e17. doi:10.1371/journal.pgen.0030017 <br />  <br />  <br />                                                                                           mutations neural crest derived pigment cells make Introduction <br />                                                                                           zebra&#239;&#172;?sh attractive model analysis genetic <br />    Vertebrate melanocyte pigment patterns long                                  mechanisms pigment cell development. Additionally, studied model pattern formation morphogenesis.                               zebra&#239;&#172;?sh genome thought duplicated early Melanocytes arise neural crest immediately begin <br />                                                                                           teleost lineage [9&#226;&#8364;&#8220;12].  redundant genes migrate ventrally distinct migratory pathways                                                                                           lost, small percentage duplicates <br /> establish embryonic pigment pattern. Mutations genes <br /> required melanocyte morphogenesis increased                                  retained, presumably evolving independ- <br /> understandings cell&#226;&#8364;&#8220;cell communication, signal trans-                                  ently. orthologous pair zebra&#239;&#172;?sh kit genes <br /> duction, differentiation, cell migration. Mice null                              [13,14]. zebra&#239;&#172;?sh mutant  kita (sparse), <br /> mutations receptor tyrosine kinase Kit (Dominant                                   defects embryonic melanocyte development;  <br /> spotting, W) embryonic lethal homozygotes failure hematopoiesis [1]. Animals heterozygous null                            Editor: Gregory S. Barsh, Stanford University School Medicine, United States homozygous hypomorphic alleles viable,                                America <br /> display white spotting coat color phenotype.                                    Received September 14, 2006; Accepted December 12, 2006; Published January <br /> analysis mutants revealed early failure                                 26, 2007 <br /> melanocyte lineage: fewer melanoblasts arise neural                                Copyright: &#195;&#8220; 2007 Hultman et al. open-access article distributed crest, defects ventral dispersion, die                            terms Creative Commons Attribution License, permits unrestricted <br /> melanization [2&#226;&#8364;&#8220;4]. locus encoding ligand                                use, distribution, reproduction medium, provided original author <br />                                                                                           source credited. <br /> receptor, Kitl (Steel, Sl), displays nearly identical de&#239;&#172;?ciencies <br /> [5&#226;&#8364;&#8220;8].                                                                                    Abbreviations: dpf, days postfertilization; hpf, hours postfertilization; kita, <br />                                                                                           zebrafish kit receptor; kitb, zebrafish kit b receptor; Kitl, mouse kit ligand; kitla, <br />    recently, use zebra&#239;&#172;?sh, Danio rerio,                               zebrafish kit ligand  kitlb, zebrafish kit ligand b; MO, morpholino oligonucleotide; successful extension understanding                                       RACE, rapid amplification cDNA ends <br /> vertebrate pigment pattern biology. optical clarity                                * correspondence addressed. E-mail: sjohnson@genetics.wustl. <br /> zebra&#239;&#172;?sh embryos, oviparous development, genetic                                      edu <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                       0089                                          January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                            Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> Author Summary                                                                 rapidly cleaved membrane. vitro experi- <br />                                                                                ments shown cells expressing variant produce <br /> Gene duplication events provide useful substrate identify             soluble ligand product [17]. Kitl-2 isoform splices effects evolution reshaping genes roles physiology          exon 6, does contain secondary cleavage development.  dozens receptor tyrosine kinases,                site exon 7, expression shown differing roles development, generated animal                  restricted cell membrane Kitl-1 [17,18]. <br /> lineages. clear associated ligands duplicated <br />                                                                                Furthermore, genetic analysis alleles Kitl evolved evolution constrained match cognate receptors. report demonstrates                  reveals splice forms involved different <br /> duplication kit ligand gene zebrafish shows                 functions Kit signaling. Mice Steel-dickie (Sld) <br /> expression function specific development                     mutation, lack transmembrane domain, produce <br /> melanocyte partitioned ligands, kitla.               soluble form Kitl similar Kitl-1 [15,16] means, kitla coevolved duplicates kit           melanocytes migrate properly subse- <br /> receptor tyrosine kinase regulate zebrafish melanocyte develop-             quently die [17]. indicates survival requirement ment. contrast, expression pattern ligand, kitlb,          properly expressed Kitl basolateral surface kitla approximates expression           epidermis. Transgenic mice expressing Kitl lacking <br /> mouse kit ligand gene, evolved independently kit <br />                                                                                exon 6,  appear wild-type pigmentation receptor gene. <br />                                                                                adults qualitative migration de&#239;&#172;?ciency <br />                                                                                [18]. current hypothesis soluble ligand product <br />                                                                                required establish chemotactic chemokinetic <br />                                                                                gradient promoting cell migration, cell survival differences role Kit mice.                 sustained cell&#226;&#8364;&#8220;cell contact membrane- <br /> contrast mouse, zebra&#239;&#172;?sh homozygous kitab5 (null)                   tethered ligand. alternative splicing primary <br /> viable de&#239;&#172;?ciencies primordial germ cells                     cleavage site exon 6 Kitl contribute hematopoiesis. Zebra&#239;&#172;?sh kita mutants retain 60% wild-                       regulation switch development. <br /> type embryonic melanocyte numbers [13]. melano-                             investigation, ortholog tetrapod <br /> cytes fail migrate periphery embryo                       Kitl duplicated zebra&#239;&#172;?sh genome  <br /> remain dorsal regions neural tube                      orthologs undergone splice form degeneration.  otocysts. 4 days postfertilization (dpf),                      ortholog, kitla, retained exon 6, kitlb appears melanocytes undergo programmed cell death                              lost exon. Given suggested roles mouse <br /> completely absent 12 dpf. second kit ortholog, kitb,                    isoform, hypothesized kitla, having retained exon 6, expressed Rohon-Beard neurons, trigeminal gan-                        better suited promote melanocyte migration, <br /> glion, ear does involved                      similar mouse Kitl-1, kitlb, having lost exon 6, melanocyte development [14].                                                   required melanocyte survival, like mouse Kitl-2. test <br />    primary roles kita zebra&#239;&#172;?sh melanocyte migration                  hypothesis &#239;&#172;?rst examining mRNA expression survival separable. Genetic analysis receptor                   gene using situ hybridization targeting <br /> using temperature-sensitive allele kita (kita jle99) revealed             antisense morpholino oligonucleotides (MOs) zebra- kita signaling required migration 2 dpf                 &#239;&#172;?sh ortholog Kitl. required cell survival stage [15].                   kitla expressed trunk Furthermore, identi&#239;&#172;?cation migration-speci&#239;&#172;?c                        medial melanocyte migration pathway early migra- <br /> survival-speci&#239;&#172;?c alleles kita suggests                       tory stages, 18 30 hours postfertilization (hpf), functions involve separate molecular domains kita                       later expressed skin stages (4 5 dpf) [15]. migration-speci&#239;&#172;?c alleles contained                     require kita receptor function survival. MO knockdown <br /> lesions near putative ligand-binding domain, allele                    kitla con&#239;&#172;?rms required melanocyte defective melanocyte survival lesion                 migration survival. overexpression intracellular kinase domain kita. possible explanation                  kitla results hyperpigmented embryo increase &#239;&#172;?nding survival ligand independent                      number size melanocytes. demonstra- <br /> zebra&#239;&#172;?sh.                                                                      tion kitla acting kita pathway achieved <br />    evidence Kitl Kit separable functions                 genetic interaction low dose kitla migration survival revealed mice mutant                       MOs sensitized allele kita receptor, Neuro&#239;&#172;?bromin sustained RAS pathway activity [16].                    revealing hyperpigmentation phenotype kitla mice melanocytes require Kit signaling                     requires kita function. <br /> migration independent Kit signaling melano-                        contrast, kitlb expressed cells lining brain <br /> cyte survival, indicating activating RAS pathway                   ventricles, sensory epithelium ear, suf&#239;&#172;?cient melanocyte survival migration                     cardinal vein plexus regions associated trunk <br /> mouse.                                                                         melanocyte migration.  like kitla, kitlb expressed    Separable functions migration survival                     skin 4 5 dpf larvae. MO knockdowns kitlb indicate <br /> established mouse Kit ligand, Kitl                         required embryonic melanocyte <br /> referred mast cell growth factor [Mgf], steel factor [SlF],              migration survival. Finally, discuss possible signal- stem cell factor [Scf]). Kitl mouse expressed                 ing kit ligands kit receptors zebra&#239;&#172;?sh <br /> cell surface splice variants. larger variant,                partitioning functions gene duplication <br /> Kitl-1, contains primary cleavage site encoded exon 6                     events receptor&#226;&#8364;&#8220;ligand pairs. <br />  <br />       PLoS Genetics | www.plosgenetics.org                              0090                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> Results <br /> Identifying Zebrafish kit ligand Orthologs <br />    interested kita-dependent pigment <br /> pattern larval zebra&#239;&#172;?sh required kit ligand (Kitl mouse, <br /> KITLG human, referred Mgf, Scf). searching zebra&#239;&#172;?sh genetic trace database Kitl ortholog using <br /> protein sequence multiple species, identi&#239;&#172;?ed partial candidate genes Chromosomes 25 4, refer kitla kitlb, respectively. candidate locus ampli&#239;&#172;?ed 24-hpf embryonic cDNA con&#239;&#172;?rm annotated sequence test expressed. length cDNA obtained candidate 59 39 rapid ampli&#239;&#172;?cation cDNA ends (RACE) amplifying exons. test candidates orthologous mammalian Kitl,  <br /> structed phylogenetic tree known tetrapod <br /> sequences, including mouse, human, chick, copies Xenopus, axolotl (Figure 1). able identify candidate copies predicted kitl fugu, medaka, stickleback genomes. copies present &#239;&#172;?sh genomes closely related tetrapod clade, appears zebra&#239;&#172;?sh copies arose duplication event teleost lineage divergence tetrapod lineage teleost <br /> radiation. observation consistent notion genome duplication teleost lineage time <br /> [9&#226;&#8364;&#8220;11]. conclude zebra&#239;&#172;?sh kitla kitlb                      Figure 1. Phylogenetic Tree kitl Protein Sequences <br /> orthologous mammalian Kitl.                                                topology tree suggests kitl locus duplicated                                                                               teleost radiation. Dr, Danio rerio (zebrafish), Fr, Fugu rubripes (fugu), <br /> kitl orthologs Correspond Different Mammalian Kitl                      Ga, Gasterosteus aculeatus (stickleback), Ol, Oryzias latipes (medaka),  <br />                                                                               Ambystoma mexicanum (axolotl), Gg, Gallus gallus (chick), Xl, Xenopus <br /> Isoforms                                                                      laevis (Xenopus), Mm, Mus Musculus (mouse), Hs, Homo sapiens (human). <br />    different mammalian Kitl functions                       Bootstrap values 1,000 replicates labeled. <br /> shown correspond alternative splicing exon 6,                   doi:10.1371/journal.pgen.0030017.g001 <br /> examined genetic structure splice forms zebra&#239;&#172;?sh <br /> kitla kitlb. test zebra&#239;&#172;?sh kitla kitlb                kitla retains exon 6 clear, predicted transcript <br /> alternatively spliced exon 6 larval pigment develop-                terminates exon 5. contrast zebra&#239;&#172;?sh transcripts <br /> ment, RT-PCR conducted gene primers                          con&#239;&#172;?rmed RT-PCR experiments, fugu, <br /> &#239;&#172;&#8218;anking exon 6 larva 1, 2, 3, 4, 5 dpf.                    stickleback, medaka sequences predicted <br /> gene displayed single splice form (unpublished data).                    GENSCAN sequences (http://genscanw.biosino.org http:// aligned complete cDNA coding sequence kitla                   genes.mit.edu/GENSCAN). <br /> kitlb genomic sequence reveal exon&#226;&#8364;&#8220;intron                        &#239;&#172;?nding zebra&#239;&#172;?sh kitla kitlb appear resemble <br /> structure. comparison genomic structure mouse                   mammalian alternative splice forms compelled notion <br /> Kitl reveals kitla contains exons, aligning               genes functioning similarly roles murine exons (Figure 2). contrast, kitlb                      suggested mouse isoforms. mouse, alternative <br /> retained exons, exons 1 5 aligning                    splicing appears regulate ef&#239;&#172;?ciency Kitl murine exons 1 5 exons 6 8                           cleaved cell membrane, isoform exon 6 <br /> aligning murine exons 7 9, suggesting kitlb                 (Kitl-1) contains primary cleavage site rapidly cleaved missing sequence homologous mouse exon 6 (Figure 2).                    cell membrane [17]. site identi&#239;&#172;?ed  duplication kit ligand gene zebra&#239;&#172;?sh                      (<span id='am-8' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-9' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence VAAS) mouse conserved followed splice form degeneration similar                 human chick unrecognizable Xenopus axolotl <br /> shown zebra&#239;&#172;?sh duplicated genes [19].                                [20,21]. examined protein sequences    presence absence exon 6 associated                 zebra&#239;&#172;?sh genes possible cleavage sites. kitla, kitlb, respectively, teleost. stickleback          identify cleavage site zebra&#239;&#172;?sh kitla exon 6  Dataset fugu sequence de&#239;&#172;?ned kitla phylogenetic tree                    S1 multiple sequence alignment). did,  notice predicted contain exon 6, kitlb species                  similar site fugu (amino acid sequence VISS) exon 6. does   predicted sequence medaka kitlb                    addition, secondary cleavage site, present appears contain candidate exon 6. indicate                    mouse isoforms exon 7 cleaved lower ef&#239;&#172;?ciency, splice form degeneration exon 6 occurred                          conserved species.  dif&#239;&#172;?cult independently taxa, radiation teleost               predict zebra&#239;&#172;?sh kitla kitlb cleaved clade.  possible exon falsely                      membrane comparable ef&#239;&#172;?ciency. <br /> identi&#239;&#172;?ed gene-&#239;&#172;?nding algorithm. medaka                           existence copies kit ligand, correspond- <br />  <br />       PLoS Genetics | www.plosgenetics.org                             0091                                     January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />                                                                                        pathway. distinguish expression levels higher <br />                                                                                        background regions lateral migration <br />                                                                                        pathway appropriate stages (18 30 hpf).  <br />                                                                                        &#239;&#172;?nd evidence change kitla expression                                                                                        responsible change migration medial                                                                                        lateral pathway. Instead, appears likely                                                                                        speci&#239;&#172;?c migratory behavior controlled factors Figure 2. Genomic Structure Alignment Mouse Kitl                                  kitla. examined kitla expression Exons (boxes) drawn scale labeled according                    present 4 dpf, kita required melanocyte <br /> homology mouse sequence. Pairwise similarity (percent identical <br /> residues plus conserved amino acid substitutions based                          survival. 4 5 dpf, kitla expression lost Blossum40 matrix) zebrafish protein mouse protein                     medial trunk regions evident skin (Figure 3I). <br /> presented exon parentheses. length values mouse                expression restricted melanocyte stripe regions <br /> zebrafish kitla: 29% identical 43% similar. length values mouse zebrafish kitlb: 20% identical 50% similar. quite                occurs skin, including interstripe <br /> diverged sequence, zebrafish paralogs display conserved                    regions (Figure 3I, black arrowhead). faint <br /> intron site locations mouse ortholog.                 staining dorsal regions somite (Figure 3I, red <br /> best alignment kitlb reveals lost exon 6, alternatively spliced mouse human. Exon 5 kitla expanded                 arrowhead). suggest early expression medial <br /> 39 corresponding contraction exon 6. Kit binding domain                      cells supports medial lateral migration contains residues interact Kit determined crystal                later switch expression skin provides subsequent <br /> structures mouse proteins [26,27]. locations major <br /> cleavage site mouse exon 6 minor cleavage site mouse                     cues melanocyte survival. <br /> exon 7 indicated. identify cleavage site                          addition expression patterns kitla suggestive sequence conservation zebrafish gene  Dataset S1).                  roles pigment pattern, expression places MOs (red bars) targeted overlap ATG start site kitlb exon 3&#226;&#8364;&#8220;intron 3 boundary kitla kitlb. splice site MOs                   role melanocyte development.  &#239;&#172;?rst <br /> resulted splicing exon 2 exon 4 (red lines), resulting shorter,          observe kitla mRNA expression tail bud 18 hpf frame, transcript  Figure 4E 4F).                                           embryos presomitic mesoderm (Figure 3A 3B) doi:10.1371/journal.pgen.0030017.g002 <br />                                                                                        pineal gland 24-h embryo (Figure 3C). <br />                                                                                        regions complementary kita expression ing alternative splice form mouse human, lead                          developing caudal notochord pineal gland [13], suggest- hypothesize zebra&#239;&#172;?sh kitla retains function                         ing possible role communication presomitic <br /> mammalian isoform 1 kitlb, having lost exon 6,                          mesoderm (expressing ligand, kitla) notochord <br /> retain function mammalian isoform 2.                                    (expressing receptor, kita) pineal gland <br /> predicted kitla required melanocyte                                  (expressing ligand receptor).  migration, kitlb required melanocyte                                kita mutant kitla morphants,  display <br /> survival. ideas explored examining                                   tail-bud notochord-related phenotypes. kitla mRNA expression patterns MO knockdown phenotypes                                 expressed sensory epithelium ear, zebra&#239;&#172;?sh kitl orthologs.                                                            prominently ventral otic vesicle (Figure 3D, arrow- <br />                                                                                        head). Interestingly, otic expression near kitla kitlb Expression Patterns                                                    second zebra&#239;&#172;?sh kit receptor, kitb, expressed [14].    explore kitla kitlb involved                             suggests functional communicative role <br /> establishing embryonic pigment pattern,                                 conserved kitla kitb, examined mRNA expression patterns embryonic                               morphological phenotype ear kitla morphants. <br /> development using mount situ hybridization (Figure                               contrast kitla expression, kitlb expressed 3). mouse, Kitl mRNA expression &#239;&#172;?rst trunk                          regions suggest role promoting melanocyte migra- embryonic day 10.5 dorsal somitic dermatome                                  tion trunk. &#239;&#172;?nd kitlb mRNA expressed [22,23]. expression lateral neural crest                         ventricular proliferation zones, pallium, brain 22 <br /> present approximately time kita mRNA-                                 hpf (Figure 3G 3H, black arrowheads). region positive melanocyte precursors begin migrate. Later, Kitl                        described previously proliferative zone neurons <br /> expressed ectoderm, support melanocytes                          glial cells [25]. kitla, kitlb expression migration skin [23].                                                            detected ear (Figure 3G, red arrow).  unlike <br />    Consistent role kitla, observe kitla mRNA                       kitla, sections expresssion limited ventral groups cells somite trunk 20 hpf                        otic vesicle, sections kitlb expression 30 hpf (Figure 3E). mount situ analysis shows                            otic vesicle epithelium (unpublished data). kitla-expressing cells appear center somite,                        observe robust expression, roughly half sections reveal cells adjacent                                 embryos, kitlb cardinal vein plexus 26 hpf (Figure <br /> notochord (Figure 3F). interesting kitla mRNA                               3G, red arrowhead). expression results suggest kitlb <br /> expression restricted middle somite;                             expressed locations appropriate earlier observations melanocyte migration showed                               signaling melanocytes migration melanocytes appear funnel middle somite                           medial lateral pathway trunk.  4 dpf, kitlb migrate medial pathway [24]. kitla mRNA                            mRNA expression evident skin (Figure 3J). kitlb <br /> expression overlaps locations melanocyte migration                             expression stage appears intense kitla medial pathway supports notion kitla                           skin stages. results consistent roles involved promoting migration melanocytes                               kitla, kitlb, melanocyte survival. <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     0092                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 3. kitla kitlb Mount Situ Hybridizations <br />  F) kitla mRNA expression stages migration. <br />   kitla expression seen 19 hpf presomitic mesoderm tail bud (black arrowhead). <br /> (B) Section 19-hpf tail bud. <br /> (C) High magnification shows expression kitla mRNA pineal gland 26 hpf. <br /> (D) High magnification shows kitla mRNA ear 26 hpf sensory epithelium, pronounced staining ventral otic vesicle (black <br /> arrowhead). <br /> (E) kitla mRNA expressed groups cells horizontal myoseptum middle somite (black arrowheads) trunk beginning 22 <br /> hpf 30 hpf (image 26 hpf). observe kitla-positive cells dorsal locations posterior somites (red arrowheads). <br /> (F) Cross section trunk shows expression near notochord 26 hpf. <br /> (G H) kitlb mRNA expression stages migration. kitlb expressed ventricles brain (black arrowheads), ear (red arrow), cardinal vein plexus (red arrowhead) 24 hpf (G). Cross section brain ventricle shows kitlb expressed cells lining brain ventricles (H). <br />  J) kitla kitlb mRNA expression 4 dpf, stage kita-dependent survival. kitla mRNA expressed skin (black <br /> arrowhead) dorsal myotome (red arrowhead)  . kitlb mRNA expressed faintly skin (black arrowhead) (J). <br /> Scale bars:   100 lm, (B) 25 lm, (C) 100 lm, (D) 50 lm, (E) 100 lm, (F) 25 lm, (G) 100 lm, (H) 10 lm,   25 lm, (J) 25 lm. <br /> nt, neural tube; nc, notochord; pm, presomitic mesoderm <br /> doi:10.1371/journal.pgen.0030017.g003 <br />  <br />  <br /> kitla Required Melanocyte Migration Survival                            zebra&#239;&#172;?sh kitla kitlb resembled alternative splice forms <br />   test kitla kitlb provides ligand                       mammalian Kitl foregoing analysis kitlb <br /> function kita melanocyte development, explored                           expression consistent role melanocyte survival, roles embryonic development MO gene                           kitla expression consistent migration survival, <br /> knockdown (Figure 4). &#239;&#172;?nding duplicated                               compelled possibility gene corresponded <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 0093                                    January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 4. kitla Morphant Phenocopies kitab5 Migration <br />   Wild-type embryonic pigment pattern 2 dpf shows melanocytes migrating yolk (red arrowhead). <br /> (B) kitab5 mutants migration phenotype, melanocytes remaining near ear (red arrow) dorsum absent yolk head (black <br /> arrowheads). <br /> (C) Wild-type embryos injected kitla MOs (6.1 ng) exhibit migratory phenotype similar kitab5 melanocytes present near ear (red arrow) absent head yolk (black arrowheads). <br /> (D) Wild-type embryos injected kitlb MOs (6.0 ng) indistinguishable wild-type showing melanocytes present yolk (red arrowhead) 2 dpf. <br /> (E&#226;&#8364;&#8220;G) RT-PCR morphant embryos shows MO specificity: (E) kitla RT-PCR wild-type, kitla MO, kitlb MO 3 dpf; (F) kitlb RT-PCR wild-type, kitla <br /> MO, kitlb MO 3 dpf; (G) kitla RT-PCR kitla MO 2, 3, 4, 5, 6, 7 dpf, revealing aberrant splice product caused MO dominant 5 dpf, wild-type message visible. <br /> (H) Regions embryo used define migrated nonmigrated melanocytes quantitative analysis melanocyte migration. Red areas <br /> indicate nonmigrated melanocytes dorsal lateral stripe hind yolk ear. Green areas define migrated melanocytes head, yolk, ventral yolk sac stripe hind yolk. Note melanocytes migrated positions dorsum horizontal myoseptum, region typically melanocytes, scored nonmigrated embryo, melanocyte migrates past horizontal myoseptum scored migrated, migration appropriate  <br />   Quantitative analysis melanocyte migration negative control MOs (6.8 ng), kitab5, kitla MO (6.1 ng). kitla MO embryos display similar loss migration kitab5. Mean values 95% confidence interval reported, n &#194;&#188; 10. <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g004 <br />  <br />  <br />  <br /> functionally different mammalian splice forms.                           multiple protein sequence alignment different test hypothesis kitla kitlb required                  DNA sequence confer speci&#239;&#172;?city. RT-PCR <br /> melanocyte migration survival, respectively, designed                        analysis MO-injected embryos showed MO splice-blocking MOs targeting E3I3 exon-intron boun-                           gene caused aberrant splicing, joining exon 2 exon 4; <br /> dary speci&#239;&#172;?c gene injected wild-type                         morphant splices exon 3 targeted gene <br /> embryos. chose boundary kit-                          (Figure 4E kitla Figure 4F kitlb). Sequencing binding domain [26,27] conserved region                      PCR product con&#239;&#172;?rmed splice product results  <br />        PLoS Genetics | www.plosgenetics.org                                 0094                                   January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                            Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 5. kitla Morphant Phenocopies kitab5 Survival Phenotype <br />   Wild-type larva 8 dpf. <br /> (B) Higher magnification dorsal melanocyte stripe 8 dpf wild-type larva healthy melanocytes (black arrowhead). <br /> (C) kitla MO shows fewer melanocytes 8 dpf, similar kitab5. <br /> (D) Higher magnification kitla MO showing melanocytes blebbing skin (red arrow), characteristic melanocyte programmed cell death. <br /> (E F) kitlb MO 8 dpf (E) higher magnification (F) kitlb MO 8 dpf, indistinguishable wild-type. <br /> (G) Total melanocyte counts survival phenotypes kitab5 kitla MO similar. <br /> Scale bars: 100 lm. <br /> doi:10.1371/journal.pgen.0030017.g005 <br />  <br />  <br />  frame deletion exon 3 (amino acids 41 59                    result total number melanocytes kitab5. kitla kitlb). Furthermore, result speci&#239;&#172;?c,            indicate migration phenotype kitla kitla MOs did interfere kitlb splicing exon 3                 sensitive abrogation melanocyte number <br /> conversely, kitlb MOs did interfere kitla splicing               phenotype. <br /> (Figure 4E kitla Figure 4F kitlb).                                  addition defects migration, melanocytes kitla MO <br />    examined consequences MO knockdown                       embryos undergo programmed cell death manner similar <br /> zebra&#239;&#172;?sh embryonic melanocyte development. Melanocytes                        kita-null mutants (Figure 5). Wild-type larva wild-type embryos begin migrate neural crest                   distinct stripes 8 dpf (Figure 5A) healthy 18 hpf 2 dpf establish dorsal, lateral, ventral,            melanocytes (Figure 5B). Melanocytes kita mutant larvae <br /> yolk sac stripes (Figure 4A). contrast, melanocytes kitab5              begin die 4 dpf, characteristically blebbing mutants fail migrate remain near ear                 skin larvae completely free melanocytes dorsal stripe, appearing ventral stripe               12 dpf. Melanocytes kitla morphants begin die yolk sac stripe (Figure 4B). Embryos injected kitla MOs (6               4 dpf, fewer melanocytes 8 dpf (Figure 5C), bleb <br /> ng) similar phenotype kitab5 (Figure 4C). Melanocyte                skin kita mutants (Figure 5D). Melanocyte counts <br /> counts regions outlined Figure 4H reveal signi&#239;&#172;?-                    reveal kitla morphants extent cantly fewer migrated melanocytes wild-type, similar                  melanocyte loss kita mutants (Figure 5G). kitab5 (Figure 4I). Interestingly, melanocytes remain                 usually melanocytes kita mutants disappear, dorsal regions near ear kitab5 animals,                melanocytes kitla morphant embryos tend survive. <br /> resulting total melanocyte number similar wild-               possibly partial restoration kitla function, type.  &#239;&#172;?nd slightly higher doses kitla MOs                 MOs tend lose ef&#239;&#172;?cacy time [28,29]. &#239;&#172;?rst  <br />       PLoS Genetics | www.plosgenetics.org                             0095                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                              Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 6. kitla MO Enhances Temperature-Sensitive kit j1e99 Allele <br />   kit j1e99 embryos reared 28 8C appear similar wild-type melanocyte pattern  Figure 4A wild-type) 3 dpf. <br /> (B) submaximal dose (0.5 0.8 ng) kitla MOs shows little effect wild-type embryos. <br /> (C) Submaximal dose kitla MOs shows significant migration phenotype kit j1e99. <br /> (D E) Quantitative analysis kit j1e99&#226;&#8364;&#8220;kitla enhancement. Melanocytes counted 3 dpf scored migrated green regions nonmigrated regions shown red (Figure 4H). Mean values 95% confidence interval reported (n &#194;&#188; 10). <br /> (D) Using metric, wild-type embryos average 94.4 (69% total) migrating melanocytes. submaximal dose kitla MOs results 16.4 fewer <br /> migrated melanocytes wild-type. kit j1e99 embryos reared 28 8C 3.7 fewer migrated melanocytes. number total melanocytes migrating melanocytes kit j1e99 submaximal dose kitla MOs significantly different compared wild-type. <br /> (E) Migration effect defined difference migrating melaocytes compared wild-type. combined effects kit j1e99 submaximal dose kitla MOs additive, expect migration effect  20.1 kit j1e99&#226;&#8364;&#8220;kitla MO larvae ( 16.4 6 3.7). Instead, observe migration effect  40.7. v2 test number migrated melanocytes kit j1e99&#226;&#8364;&#8220;kitla MO larvae expected number reveals difference significantly (p , 0.0002) greater additive. <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g006 <br />  <br />  <br />  <br /> restoration wild-type kitla transcript 3 dpf                     expected additive de&#239;&#172;?cit ( 20.1 melanocytes) (Figure 6E). increasing wild-type transcript 7 dpf (Figure 4G),                     result con&#239;&#172;?rms genetic interaction kita kitla. stages melanocyte survival kita dependent. results indicate kitla promotes migration                  kitla Overexpression Induces Hyperpigmentation <br /> survival melanocytes similar zebra&#239;&#172;?sh kita receptor.                    explore role kitla establishing                                                                                melanocyte pattern, examined kitla overexpression kitla MOs Enhance kita jle99                                                   wild-type embryos (Figure 7). Accordingly, constructed    phenotypic analysis kitla MOs wild-type embryos                   expression vector kitla fusion red &#239;&#172;&#8218;uorescent <br /> suggests kitla functionally orthologous                         protein, JRed [30]. Injection vector, capped mRNA <br /> mammalian Kitl, ligand kita receptor tyrosine                   vector, wild-type embryos results embryos <br /> kinase. test kitla kita pathway,               hyperpigmentation body (Figure 7B). investigated genetic interaction effects                test increase cell number, <br /> kitla MOs sensitized kita allele (Figure 6). kita jle99             melanocytes counted dorsal stripe. Wild-type <br /> temperature-sensitive allele kita, lesion C-                  embryos average 200 dorsal melanocytes <br /> terminal tyrosine kinase domain (Leu754Pro) [15]. kita jle99 &#239;&#172;?sh               (Figure 7A 7E). Hyperpigmented animals phenotypically wild-type permissive temperature                     dramatic increase number dorsal melanocytes, <br /> (25 8C) defects melanocyte migration                          averaging 300 cells increase greater 50% <br /> survival restrictive temperature (33 8C) similar null              (Figure 7E). number melanocytes regions kita mutant. purposes, advantage                        appears increased embryos, intermediate phenotype 28 8C (Figure 6A), slightly                 quanti&#239;&#172;?ed increase. <br /> fewer melanocytes migrate compared wild-type.                             transient injection experiments, typically hypomorphic phenotype likely partial kita signaling                  little red &#239;&#172;&#8218;uorescence cells yolk. <br /> activity used assay testing                     JRed kitla expected act dimers genes act genetic pathway kita.                   [30,31], possible  steric hindrance, kitla analysis shows interaction kita j1e99                 dimerization interferes ability JRed dimerize <br /> kitla MOs synergistic, greater de&#239;&#172;?cit                        correctly visible locations migrating melanocytes ( 41.7) mutant ( 3.7                   expression. mount situ analysis shows abundant melanocytes) morphant ( 16.4 melanocytes),                     highly mosaic mRNA expression (unpublished data), suggest- <br />  <br />        PLoS Genetics | www.plosgenetics.org                             0096                                  January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                              Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 7. kitla Overexpression Causes Hyperpigmentation Wild-type kitab5 Embryos <br />   Wild-type larva. <br /> (B) Larva injected kitla::JRed fusion construct shows hyperpigmentation melanocytes covering larger area wild-type. <br /> (C) kita larva. <br /> (D) kitla::JRed injected kitb5 embryos results kitb5 phenotype. <br /> (E) kitla::JRed hyperpigmented larvae melanocytes dorsal stripe wild-type. kitla expression vector injected kita <br /> embryos, change number melanocytes. larvae 6 dpf (n &#194;&#188; 10). <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g007 <br />  <br />  <br />  <br /> ing kitla protein expressed embryo                   process.  expression data 4 dpf experiments.                                                             kitlb involved melanocyte survival. injected <br />                                                                                kitlb E3I3 MOs block normal splicing kitlb resulting kitla-Induced Hyperpigmentation kita Dependent                              transcripts (approximately 75%) lacking exon 3 (Figure <br />    &#239;&#172;?rmly kitla acts kita receptor,                 4F), effect melanocyte phenotypes. kitlb <br /> explored effect overexpressing kitla kitab5 embryos <br />                                                                                morphant embryos signi&#239;&#172;?cantly different wild- <br /> (Figure 7). kitla expression construct injected <br />                                                                                type melanocyte migration (Figure 4D) survival (Figure kitab5 embryos (Figure 7D), melanocytes develop identi- <br />                                                                                5E 5F). Additionally, kitlb morphants appear morpholog- <br /> cally kita-null phenotype (Figure 7C), including                                                                                ically normal obvious blood defects anemia identical number melanocytes dorsal stripes (Figure <br />                                                                                2 dpf (unpublished data). rule 7E). consistent notion effects                                                                                possibility kitlb functional despite ability E3I3 <br /> ligand, kitla, mediated receptor, kita. <br />                                                                                MOs splice exon 3, designed additional kitlb MOs phylogenetic analysis, MO phenocopy kitab5, MO enhancement kita j1e99, depend-                   designed block translation. showed ence kita kitla-induced hyperpigmentation,                       effect melanocyte migration survival conditions <br /> conclude kitla functional kita ligand zebra&#239;&#172;?sh                  tested. sensitive test, took advantage embryonic melanocyte development kitla promotes                       conditional kita allele injecting kitlb MOs kita-dependent functions embryonic melanocyte                    embryos homozygous kita jle99, examine development. &#239;&#172;?nding kitla MO largely phenocopies                      enhance sensitized kita phenotype. failed kita mutant tends argue kitla ligand                   detect enhancement kita jle99 kitlb MOs, required embryonic melanocyte development,                     independently combined mix (unpublished data). dif&#239;&#172;?cult rule small effect kitlb                   Taken  expression kitlb mRNA, <br /> revealed experiments.                                                   data suggest kitlb does necessary role                                                                                embryonic melanocyte development. <br /> kitlb Required Embryonic Melanocyte Patterning Survival                                                                    Discussion <br />   demonstration kitla necessary melanocyte migration survival roles kita receptor                     identi&#239;&#172;?ed explored roles kitl <br /> does preclude possibility kitlb participates              orthologs zebra&#239;&#172;?sh. orthologs notable  <br />        PLoS Genetics | www.plosgenetics.org                             0097                                  January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                   Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />                                                                                       primary cleavage site alters soluble expression                                                                                       ligand [17,18]. evidence alternative splice <br />                                                                                       forms, Kitl-1, containing primary secondary <br />                                                                                       cleavage sites, Kitl-2, secondary cleavage <br />                                                                                       site, functioning promoting melanocyte migration <br />                                                                                       survival, respectively [22]. shown, <br />                                                                                        exclusive expression short splice form, <br />                                                                                       lacking exon 6, Kitl-2, results normal adult pigment <br />                                                                                       pattern [18].  isoforms appear cleaved                                                                                       membrane [17], albeit different rates, differences <br /> Figure 8. Stylized Drawing Zebrafish Embryo Displaying mRNA                        functions isoforms quantitative, <br /> Expression Patterns kita kitb Receptor Tyrosine Kinases              qualitative. <br /> Candidate Ligands, kitla kitlb                                                       zebra&#239;&#172;?sh kita melanocytic functions appear Coincident expression noted kita (blue) kitla (green)               molecularly separable, explored possibility trunk, tail bud, ear, pineal gland. kitb (red) kitlb (yellow) appear coincident. kitlb does appear coincident receptor                migration survival functions partitioned expression brain ventricles cardinal vein plexus. kitb           zebra&#239;&#172;?sh kitl orthologs. primary cleavage coincident kitla expression trunk.                                     site conserved species, unable doi:10.1371/journal.pgen.0030017.g008 <br />                                                                                       identify site protein sequence &#239;&#172;?sh <br />                                                                                       sequences examined  Dataset S1). entirely <br />                                                                                       surprising given rate evolution ligands. differences genomic structure, expression pattern,                           possible site exists conserved. phenotypes MO knockdown experiments.                                            analysis expression patterns, MO knockdown <br />    clear gene duplications played role                       phenotypes, interaction MO knockdown sensitized <br /> creation large number receptor tyrosine kinases                             kita allele, &#239;&#172;?nding overexpression phenotypes kitla <br /> [32,33]. clear role gene duplication generating                     require kita function argue kitla provides protein ligands, evolve rapidly                         necessary ligand roles kita receptor promoting <br /> phylogenetic relationships characterized.                             embryonic melanocyte development, kitlb  instances, receptor&#226;&#8364;&#8220;ligand pairs                            provide little  roles include establishing arisen duplication                                 initial cell number, promoting migration, promoting <br /> receptor ligand. case Kit/                            survival. Temperature shift experiments kit j1e99 <br /> Kitl Fms/M-Csf. Kit clearly closely related                      temperature-sensitive allele kit revealed melanocytes <br /> Fms [34], protein sequences Kitl M-Csf dif&#239;&#172;?cult                        require kit signaling 2 dpf migration 2 align.  secondary structures ligands                          dpf survival [15]. &#239;&#172;?nding kitla required indicate  short-chain helical cytokines, M-Csf                     indicates involved &#239;&#172;?rst promote similar Kitl [26]. recent duplication                            migration 2 dpf later promote <br /> leading kita/kitb kitla/kitlb zebra&#239;&#172;?sh,                           survival 2 dpf. migration survival <br /> provides opportunity understanding evolutionary                            defect kitl MOs phenocopy kita null suggests <br /> mechanisms coevolution receptor&#226;&#8364;&#8220;ligand pairs.                              difference time functions    kit receptor ligand appear                                differential expression display ligand 2 dpf. duplicated teleost lineage. duplication events                           likely switch mediated downstream kita consistent notion genome duplication                          signaling. note explored individual <br /> event prior teleost radiation [9&#226;&#8364;&#8220;11].                              roles kitla kitlb promoting adult melanocyte develop- <br /> hypothesized gene duplication events followed                          ment, different kita-dependent kita-independent relaxed evolutionary selection result changes                         melanocyte populations [41,42]. expression function orthologs [35]. Preserva-                            Previously, exhaustive noncomplementation screen tion copies result subfunctionalization,                          additional alleles kita resulted migration-speci&#239;&#172;?c multiple functions controlled single ancestral                           alleles lesions near ligand-binding domain, gene partitioned gene duplicates [36]. Subfunc-                           single survival-speci&#239;&#172;?c allele lesion intracellular <br /> tionalization gene expression described                               domain [15]. possible explanation results zebra&#239;&#172;?sh ortholog pairs [37&#226;&#8364;&#8220;39]. Similar case                       migration-promoting function kita required kitl  zebra&#239;&#172;?sh mitf gene                                ligand interaction, melanocyte survival function <br /> duplicated, ortholog generates transcript                          kita ligand independent. &#239;&#172;?nding kitla <br /> corresponds splice forms produced                                MO abrogates melanocyte migration survival rules <br /> mammalian Mitf gene [19,40].                                                          possibility. light results                                                                                        likely intracellular lesion disrupt binding kitla Functional Ligand kita Melanocyte                                 downstream targets kita survival speci&#239;&#172;?c. Migration Survival                                                                appears separable functions kitla/kita <br />    Examination genetic structure kitla kitlb                        pathway diverge kita, activity kitla. <br /> loci revealed likely splice form degeneration followed duplication event, kitla retained exon 6                            Coincident Gene Expression kit&#226;&#8364;&#8220;kitl Pairs <br /> kitlb gene lost exon. Interestingly, presence absence                        interested expressions zebra&#239;&#172;?sh exon 6 mouse transcript regulates inclusion                        kitl orthologs evolved duplication event. <br />  <br />        PLoS Genetics | www.plosgenetics.org                                    0098                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                       Zebrafish kit ligand Duplication/Evolution <br />  <br />  analysis, assume Kitl expression                  include cardinal vein plexus cells lining patterns mouse [22,23,43] chick [43] similar           ventricles brain (Figure 8, yellow shading). tetrapod/&#239;&#172;?sh ancestral pattern examine               obvious morphological phenotypes seen kitlb mor- <br /> gene expression domains partitioned zebra&#239;&#172;?sh                  phants, sensitive assays, instance, behavior Kitl orthologs.  Kitl expressed trunk            blood histology, reveal subtle defects. <br /> lateral intermediate mesoderm mouse chick                    conclude coincident mRNA expression <br /> [23,43,44] dorsal dermomyotome mouse [22].                  patterns kitla kita, similar phenotypes <br /> Kitl weakly expressed brain                 caused mutation MO knockdown, kitla kita <br /> expressed near otic vesicle mesenchyme                 coevolved expression patterns functions, presumably hindbrain surface ectoderm, pro-                    partitioning select expression domains functions nounced expression regions lateral ventral otic             ancestral receptor ligand genes. contrast, vesicle [45].                                                             expression ear, little coincident <br />    composite, early expression domains kitl             expression kitlb kitb embryo. raises genes zebra&#239;&#172;?sh match mouse chick,                possibility kitlb does signal kitb receptor. assumed ancestral form. domains,              logic, kitlb kitb interact receptors domains kit receptors, shown stylized            ligands, related ancestral paralogs. <br /> embryo (Figure 8) representing expression patterns seen                   interesting know structures kitla 18 30 hpf. Additionally, diagram reveals                 kita coevolved maintain effective ligand&#226;&#8364;&#8220;receptor <br /> coincident expression kita kitla, consistant kitla-           interactions  contrast, kitlb kitb structures <br /> positive cells signaling cells expressing kita. impor-            evolved unidenti&#239;&#172;?ed receptor ligand <br /> tant, kita expressed migrating melanocytes (Figure 8, blue          partners. <br /> shading trunk) kitla expressed cells               addition early expression domains, kitla <br /> medial pathway melanocyte migration (green shading                    kitlb mRNAs expressed 4 5 dpf kita- trunk). location kita kitla                   dependent melanocyte survival. stage, kita-expressing <br /> coincident tail bud, kita expressed                melanocytes established embryonic melanocyte <br /> notochord [13] (Figure 8, blue) kitla surrounding           pattern.  kitla (abundantly) kitlb (weakly) <br /> presomitic mesoderm (Figure 8, green shading tail).                    expressed skin restricted Additionally, kitla kita expressed pineal             regions adjacent near melanocyte stripes. presence gland (Figure 8, green/blue regions), suggesting            kitla expression interstripe region skin indicates interacting promote aspect pineal gland                     gene required  permissively) function development. involvement kita                     migration, kitla expression does establish prepattern pathway does appear required normal develop-                melanocyte stripe formation. <br /> ment, observe morphological changes                       partitioning expression domains tissues kitla morphants kitab5 mutants.                zebra&#239;&#172;?sh kit/kitl orthologs result 300 million  like kitla, expression kita subset           years evolution zebra&#239;&#172;?sh genome duplicated [12]. <br /> expression domains observed mouse [13].                        compare recent duplication kitl <br /> conclude coevolution kita kitla results retained           Xenopus (called Xsl-1 Xsl-2) [21]. contrast complementary expression patterns, following partition                 teleost, Xenopus genes appear contain exon 6  ancestral kit kitl expression. expression                   Dataset S1). Overall, difference expression domains <br /> patterns suggest share functional roles embryonic                 Xsl-1 Xsl-2 development.                                                              zebra&#239;&#172;?sh kitla kitlb. Like zebra&#239;&#172;?sh, copies appear <br />     examined possibility expression patterns             expressed otic placode skin. Unlike kitb kitlb coevolved                 zebra&#239;&#172;?sh, Xenopus genes expressed dermo- <br /> previously reported receptor kitb expressed                myotome trunk, Xsl-1 expressed Rohon-Beard neurons, trigeminal ganglion, otic                    length trunk Xsl-2 restricted anterior <br /> vesicle [14] (Figure 8, red shading). possible                   somites. clear expression domains exception otic vesicle, observe kitla                functioning Xenopus relation melanocyte develop- kitlb expression, locations appears             ment, reported ortholog Kit appears coincident expression kitlb. Like kitla, &#239;&#172;?nd kitlb                expressed melanocytes [46,47]. <br /> expressed epithelium lining otic vesicle (Figure 8, <br /> yellow shading). interesting note mouse, Kitl            kitla-Induced Hyperpigmentation expressed mesenchyme near ear,                     &#239;&#172;?rst step understanding morphological otic vesicle epithelium, kitla            roles kitla melanocyte development, begun kitlb. expression domains clearly homologous,               generate animals overexpress ligand. Following similar positioning raises possibility            injection kitla expression vector, embryos  <br /> re&#239;&#172;&#8218;ect homology function.             express kitla hyperpigmented larger <br /> know expression domains evolved different                    melanocytes. &#239;&#172;?nding hyperpigmentation tissues. kita expressed ear, remains               dependent kita, kitla MO knockdown <br /> possible ligands signaling kitb          phenotype interaction sensitized temper- context.                                                             ature-sensitive kita allele, provides compelling evidence    locations kitlb mRNA expression             kitla ligand kita embryonic melanocyte develop- coincident kita kitb receptor expression. areas            ment. interesting know similar <br />  <br />       PLoS Genetics | www.plosgenetics.org                         0099                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> overexpression kitlb results hyperpigmentation,                          amplify kitla kitlb cDNA adapted T7 RNA polymerase shed light kitlb capacity                      recognition sequences 39 end coding sequence: kitla <br />                                                                                     forward primer 59-TCTCGTTCCATATGAAGAAGTCAA-39, reverse bind kita transduce signal.                                                primer 59-taatacgactcactataggTCAGATATCCCCACATCTAATGG-39, <br />     unclear kitla-overexpressing embryos                       kitlb forward primer 59-CACATGAGGGAGGTTAAGATTG-39, melanocytes proliferation precursor                         reverse primer 59-taatacgactcactataggACACCCTAAAGAATCCAGC- <br />                                                                                     CA-39, lowercase letters indicate T7 sequence. <br /> cells, recruitment differentiation additional stem cells,                       Antisense probe synthesized PCR products using <br /> transdifferentiation nonmelanized cells melanocytes,                        DIG-labeled NTPs transcription reaction T7 polymerase. protection cell death. Regardless mechanism,                         sense strand synthesized using T7 adaptors 59 end <br />                                                                                     coding sequenced used negative control. mount result suggests normal development, kitla rate                      situ hybridization modi&#239;&#172;?ed previously published <br /> limiting, correct expression necessary                      methods [55] use Abimed situ robot (Abimed Situ <br /> produce wild-type embryonic pigment pattern. Notably                            Pro; Intavis AG, http://www.intavis.com). Color development absent variety large-scale [48] focused [49,50]                    conducted manually. <br />                                                                                        sectioning, embryos embedded agar [56], 10-lm <br /> mutant screens zebra&#239;&#172;?sh mutants melanocyte                              sections cut using Leica CM 1850 Cryostat (http://www.leica. <br /> hyperpigmentation, increased number cells                           com). <br /> larger cells. interesting phenotype produced                         MO knockdown analysis. Antisense MOs (Gene Tools, http://www. <br />                                                                                     gene-tools.com) designed reduce functional <br /> kitla DNA injection, expected produce mosaic                          protein expressed embryos [57]. chose target mRNA splicing <br /> expression kitla, effect entire                            order test ef&#239;&#172;?cacy MOs RT-PCR analysis [58]. <br /> animal. analysis understand role kitla                             MOs kitla E3I3 (59-CTGGATAACAACACTCACCACTTCT-39) <br />                                                                                     kitlb E3I3 (59-CACATGTATACTTACCACATCCTTT-39) melanocyte development ongoing, making                            designed block processing splice donor site exon 3. <br /> stable transgenic lines expressing construct                         Additional MOs designed target kitlb ATG start site (59- <br /> allow explore role kitla/kita signaling cell                        ATGTCTTAACCTCCCTCATGGGGAACAT-39) 59 UTR (59- <br /> number, cell size control, directing melanocyte                           CCTGTAAATGACTGGAGATAATCAC-39). Negative control MO <br />                                                                                     (59-CCTCTTACCTCAGTTACAATTTATA-39) used control <br /> migration.                                                                          nonspeci&#239;&#172;?c, MO effects. MO injection solution 20 <br />                                                                                     lM MO 13 Danieau buffer 1% Phenol Red visual tracer. <br />                                                                                     Embryos injected MO injection solution yolk Materials Methods                                                                cell stage. <br />    Fish rearing stocks. Embryos obtained vitro                     test ef&#239;&#172;?cacy MO altering transcript kitla fertilization reared 14:10-h dark/light cycle egg water              kitlb, RT-PCR conducted embryos injected kitla E3I3 MOs, <br /> constant temperature 28 8C, unless noted. Wild-type                    kitlb E3I3 MOs, negative control MO (Cont. MO). Primers RT- <br />                                                                                     PCR kitla forward primer 59-CACAGTTGCTGCCTATTCCA-39, <br /> &#239;&#172;?sh obtained sjC inbred line sjA/sjC hybrids <br />                                                                                     kitla reverse primer 59-TGAATCCTCCAAACCAGGTC-39, kitlb  <br /> [51,52]. Mutants kit receptor tyrosine kinase, kit b5 kit j1e99, <br />                                                                                     ward primer 59-ACCTGCTCAGGTGTTTTTGG-39, kitlb reverse previously described [15]. <br />                                                                                     primer 59-CATTCTGTCCTCCAGGTCGT-39. semiquantitative <br />    Cloning kitla kitlb. Candidate zebra&#239;&#172;?sh kitl sequences                                                                                     measure kitla kitlb MO ef&#239;&#172;?cacy measured pixel searching Genome Shotgun Trace Database (http:// <br />                                                                                     intensity calculation ImageJ (Research Services Branch, National <br /> www.ncbi.nlm.nih.gov/Traces/trace.cgi?<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;cmd&#194;&#188;retrieve&amp;val&#194;&#188;species_ <br />                                                                                     Institute Mental Health, http://www.rsb.info.nih.gov/ij). <br /> code%3D%22DANIO%20RERIO%22&amp;retrieve&#194;&#188;Submit; June 2002)                                 kitla overexpression. kitla cDNA sequence cloned frame using tblastn program [43] Kitl protein sequences                     pJRed-N vector (Evrogen, http://www.evrogen.com) produce human, mouse, chick, Xenopus, axolotl query. query                  C-terminal fusion protein. resulted expression vector sequence appeared produce conserved alignments                        early promoter cytomegalovirus driving expression zebra&#239;&#172;?sh genome, signi&#239;&#172; pair human                  kitla::JRed fusion protein. DNA (100 pg) injected directly wild- <br /> KITLG exon 4, E-values 4e 9 1e 5. pair hits                 type kit b5 embryos  cell stage analysis, <br /> mapped LN54 radiation hybrid panel Chromosomes 25 4                   using 1% Phenol Red tracer. Alternatively, injected 59-capped named kitla kitlb, respectively.                                            mRNA (30 pg) vector (mMessage mMachine; Ambion, <br />    obtain length cDNA sequence loci, PCR primers                   http://www.ambion.com). simplicity, refer embryos injected designed amplify candidate kitl                    mRNA vector &#226;&#8364;&#732;&#226;&#8364;&#732;kitla injected,&#226;&#8364;&#8482;&#226;&#8364;&#8482; unless <br /> exons embryonic caudal &#239;&#172;?n poly(T)-primed cDNA. RACE                        noted. used attain 59 39 ends genes. case kitlb,              Melanocyte counts. Melanocyte migration determined RACE failed, primers designed candidate open                        counting melanocytes 48 hpf seven distinct regions reading frames downstream upstream known exons                    embryo: yolk, near ear, head anterior eye, genomic region (assembly z06s024825). Single RT-PCR fragments                       stripes hind yolk. Melanocytes isolated reaction 1% agarose gel electro-                     yolk, head, ventral, yolk sac stripes classi&#239;&#172;?ed migrated <br /> phoresis sequenced.                                                             melanocytes colored green Figure 4H.    Phylogenetic sequence analyses. multiple sequence align-                   remained near ear dorsal lateral trunk stripes, <br /> ment kit ligand protein sequences generated ClustalW                    marked red Figure 4H, considered nonmigratory. adjusted manually  Dataset S1). alignment                Melanocyte survival determined counting total number <br /> protein sequence C-terminal exon 5 ambiguous, Kyte-Doolittle                 melanocytes larvae 4 dpf 11 dpf. Hyperpigmented <br /> hydropathy plots hidden Markov modeling program                             animals kitla overexpression placed 1% epinephrine HMMTOP [53] used con&#239;&#172;?rm alignment trans-                         retract melanosomes, dorsal stripe melanocytes counted <br /> membrane domain exon 7. Exons 6 9 omitted                      compared number dorsal melanocytes wild-type phylogenetic analysis. Phylogenetic relationships                     embryo. <br /> determined using maximum likelihood method Phylip <br /> package [54]. Bootstrap support topology conducted maximum likelihood 1,000 replicates. built trees using                 Supporting Information neighbor joining parsimony methods. Neighbor joining <br /> produced tree zebra&#239;&#172;?sh kitla outgroup &#239;&#172;?sh                 Dataset S1. Multiple Sequence Alignment kit Ligands <br /> genes, parsimony produced tree zebra&#239;&#172;?sh kitlb                   doi:10.1371/journal.pgen.0030017.sd001 (83 KB RTF). <br /> outgroup. maximum likelihood tree presented consistent pattern gene duplication previously <br /> reported teleost lineage require               Accession Numbers <br /> evolutionary changes explain copies &#239;&#172;?sh species.                 GenBank (http://www.ncbi.nlm.nih.gov/Genbank) accession numbers <br />    situ hybridization. kitla kitlb probe template generated              genes discussed paper kitla (AY929068) kitlb PCR ampli&#239;&#172;?cation 24-hpf embryonic cDNA. Primers used                     (AY929069). Protein sequences Entrez protein accession <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  0100                                   January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br /> numbers human KITLG (NP_000890), mouse Kitl                                            Acknowledgments <br /> (NP_038626), chick Kitl (NP_990461), axolotl Kitl (AAD17253), <br /> Xenopus Xsl-1 (AAH44108), Xenopus Xsl-2 (AAT66411)                               thank Jonathan Gitlan, Chao Tsung Yang, Eve Mellgren http://www.ncbi.nlm.nih.gov/entrez/query.                                          useful comments manuscript. thank Scott Higdon fcgi?CMD&#194;&#188;search&amp;DB&#194;&#188;protein.                                                                 help maintaining &#239;&#172;?sh stocks. <br />                                                                                                Author contributions. KAH SLJ conceived designed    Protein sequences Fugu rubripes (fugu), Gasterosteus aculeatus <br />                                                                                             experiments, analyzed data, wrote paper. KAH <br /> (stickleback), Oryzias latipes (medaka) searching                     performed experiments. NB, LIZ, SLJ contributed reagents/ <br /> predicted protein database species                              materials/analysis tools. <br /> Ensembl (http://www.ensembl.org). Fugu kitla (NEWSINFR-                                        Funding. work funded National Institutes Health <br /> UP00000171441), stickleback kitla (ENSGACP00000012357), medaka                              grant GM56988 SLJ. <br /> kitla (ENSORLP00000016705), medaka kitlb                                                Competing interests. authors declared competing <br /> SORLP00000016705).                                                                          interests exist. <br />  <br />  <br /> References                                                                                  23. Matsui Y, Zsebo KM, Hogan BL (1990) Embryonic expression 1. Little CC (1915) inheritance black-eyed white spotting mice.                    haematopoietic growth factor encoded Sl locus ligand     Nat 49: 727&#226;&#8364;&#8220;740.                                                                            c-kit. Nature 347: 667&#226;&#8364;&#8220;669. <br /> 2. Geissler  Ryan MA, Housman (1988) dominant-white spotting                      24. Raible DW, Wood  Hodsdon W, Henion PD, Weston JA, et al. (1992) <br />     (W) locus mouse encodes c-kit proto-oncogene. Cell 55: 185&#226;&#8364;&#8220;192.                  Segregation early dispersal neural crest cells embryonic <br /> 3. Qiu FH, Ray P, Brown K, Barker PE, Jhanwar S, et al. (1988) Primary                          zebra&#239;&#172;?sh. Dev Dyn 195: 29&#226;&#8364;&#8220;42. <br />     structure c-kit: Relationship CSF-1/PDGF receptor kinase                    25. Wullimann MF, Puelles L (1999) Postembryonic neural proliferation     family: Oncogenic activation v-kit involves deletion extracellular                    zebra&#239;&#172;?sh forebrain relationship prosomeric domains. Anat <br />     domain C terminus. EMBO J 7: 1003&#226;&#8364;&#8220;1011.                                                 Embryol (Berl) 199: 329&#226;&#8364;&#8220;348. <br /> 4. Orr-Urtreger  Avivi  Zimmer Y, Givol D, Yarden Y, et al. (1990)                      26. Zhang  Zhang R, Joachimiak  Schlessinger J, Kong XP (2000) Crystal <br />     Developmental expression c-kit, proto-oncogene encoded W                        structure human stem cell factor: Implication stem cell factor <br />     locus. Development 109: 911&#226;&#8364;&#8220;923.                                                            receptor dimerization activation. Proc Natl Acad Sci U S 97: 7732&#226;&#8364;&#8220; <br /> 5. Sarvella PA, Russell LB (1956) Steel, new dominant gene house                       7737. <br />     mouse. J Hered 47: 123&#226;&#8364;&#8220;128.                                                             27. Jiang X, Gurel O, Mendiaz EA, Stearns GW, Clogston CL, et al. (2000) <br /> 6. Anderson DM, Lyman SD, Baird  Wignall JM, Eisenman J, et al. (1990)                        Structure active core human stem cell factor analysis     Molecular cloning mast cell growth factor, hematopoietin active                binding receptor kit. EMBO J 19: 3192&#226;&#8364;&#8220;3203. <br />     membrane bound soluble forms. Cell 63: 235&#226;&#8364;&#8220;243.                             28. Yan YL, Miller CT, Nissen RM, Singer  Liu D, et al. (2002) zebra&#239;&#172;?sh sox9 <br /> 7. Copeland NG, Gilbert DJ, Cho BC, Donovan PJ, Jenkins NA, et al. (1990)                       gene required cartilage morphogenesis. Development 129: 5065&#226;&#8364;&#8220;5079. <br />     Mast cell growth factor maps near steel locus mouse chromosome 10                29. Mellgren EM, Johnson SL (2004) requirement kit embryonic <br />     deleted number steel alleles. Cell 63: 175&#226;&#8364;&#8220;183.                              zebra&#239;&#172;?sh melanocyte differentiation revealed melanoblast delay. Dev <br /> 8. Huang E, Nocka K, Beier DR, Chu TY, Buck J, et al. (1990) hematopoietic                  Genes Evol 214: 493&#226;&#8364;&#8220;502. <br />     growth factor KL encoded Sl locus ligand c-kit              30. Shagin DA, Barsova EV, Yanushevich YG, Fradkov AF, Lukyanov KA, et al. <br />     receptor, gene product W locus. Cell 63: 225&#226;&#8364;&#8220;233.                                (2004) GFP-like proteins ubiquitous metazoan superfamily: Evolution 9. Amores  Force  Yan YL, Joly L, Amemiya C, et al. (1998) Zebra&#239;&#172;?sh hox                     functional features structural complexity. Mol Biol Evol 21: 841&#226;&#8364;&#8220;850. <br />     clusters vertebrate genome evolution. Science 282: 1711&#226;&#8364;&#8220;1714.                       31. Lemmon MA, Pinchasi D, Zhou M, Lax  Schlessinger J (1997) Kit receptor <br /> 10. Taylor JS, Braasch  Frickey T, Meyer  Van Peer Y (2003) Genome                       dimerization driven bivalent binding stem cell factor. J Biol Chem <br />     duplication, trait shared 22000 species ray-&#239;&#172;?nned &#239;&#172;?sh. Genome Res                   272: 6311&#226;&#8364;&#8220;6317. <br />     13: 382&#226;&#8364;&#8220;390.                                                                            32. Rousset D, Agnes F, Lachaume P, Andre C, Galibert F (1995) Molecular <br /> 11. Jaillon O, Aury JM, Brunet F, Petit JL, Stange-Thomann N, et al. (2004)                     evolution genes encoding receptor tyrosine kinase immuno- <br />     Genome duplication teleost &#239;&#172;?sh Tetraodon nigroviridis reveals early              globulinlike domains. J Mol Evol 41: 421&#226;&#8364;&#8220;429. <br />     vertebrate proto-karyotype. Nature 431: 946&#226;&#8364;&#8220;957.                                        33. Grassot J, Gouy M, Perriere G, Mouchiroud G (2006) Origin molecular <br /> 12. Taylor JS, Van Peer Y, Braasch  Meyer (2001) Comparative genomics                    evolution receptor tyrosine kinases immunoglobulin-like domains. <br />     provides evidence ancient genome duplication event &#239;&#172;?sh. Philos                    Mol Biol Evol 23: 1232&#226;&#8364;&#8220;1241. <br />     Trans R Soc Lond B Biol Sci 356: 1661&#226;&#8364;&#8220;1679.                                             34. Mellgren EM, Johnson SL (2002) evolution morphological complex- <br /> 13. Parichy DM, Rawls JF, Pratt SJ, Whit&#239;&#172;?eld TT, Johnson SL (1999) Zebra&#239;&#172;?sh                     ity zebra&#239;&#172;?sh stripes. Trends Genet 18: 128&#226;&#8364;&#8220;134. <br />     sparse corresponds orthologue c-kit required                    35. Ohno S (1970) Evolution gene duplication. New York: Springer-Verlag. <br />     morphogenesis subpopulation melanocytes, essential               36. Force  Lynch M, Pickett FB, Amores  Yan YL, et al. (1999) Preservation <br />     hematopoiesis primordial germ cell development. Development 126:                         duplicate genes complementary, degenerative mutations. Genetics <br />     3425&#226;&#8364;&#8220;3436.                                                                                  151: 1531&#226;&#8364;&#8220;1545. <br /> 14. Mellgren EM, Johnson SL (2005) kitb, second zebra&#239;&#172;?sh ortholog mouse                37. Serluca FC, Sidow  Mably JD, Fishman MC (2001) Partitioning tissue <br />     Kit. Dev Genes Evol 215: 470&#226;&#8364;&#8220;477.                                                           expression accompanies multiple duplications Na&#195;&#190;/K&#195;&#190; ATPase alpha <br /> 15. Rawls JF, Johnson SL (2003) Temporal molecular separation kit                    subunit gene. Genome Res 11: 1625&#226;&#8364;&#8220;1631. <br />     receptor tyrosine kinase&#226;&#8364;&#8482;s roles zebra&#239;&#172;?sh melanocyte migration                   38. Chiang EF, Yan YL, Guiguen Y, Postlethwait J, Chung B (2001) Cyp19 <br />     survival. Dev Biol 262: 152&#226;&#8364;&#8220;161.                                                            (P450 aromatase) genes duplicated zebra&#239;&#172;?sh chromosomes ex- <br /> 16. Wehrle-Haller B, Meller M, Weston JA (2001) Analysis melanocyte                          pressed ovary brain. Mol Biol Evol 18: 542&#226;&#8364;&#8220;550. <br />     precursors Nf1 mutants reveals MGF/KIT signaling promotes                       39. Chiang EF, Pai CI, Wyatt M, Yan YL, Postlethwait J, et al. (2001) sox9 <br />     directed cell migration independent function cell survival. Dev Biol              genes duplicated zebra&#239;&#172;?sh chromosomes: Expression similar tran- <br />     232: 471&#226;&#8364;&#8220;483.                                                                               scription activators distinct sites. Dev Biol 231: 149&#226;&#8364;&#8220;163. <br /> 17. Huang EJ, Nocka KH, Buck J, Besmer P (1992) Differential expression                 40. Lister JA, Close J, Raible DW (2001) Duplicate mitf genes zebra&#239;&#172;?sh: <br />     processing cell-associated forms kit-ligand: KL-1 KL-2.                   Complementary expression conservation melanogenic potential. <br />     Mol Biol Cell 3: 349&#226;&#8364;&#8220;362.                                                                   Dev Biol 237: 333&#226;&#8364;&#8220;344. <br /> 18. Tajima Y, Moore MA, Soares V, Ono M, Kissel H, et al. (1998) Consequences               41. Johnson SL, Africa D, Walker C, Weston JA (1995) Genetic control adult <br />     exclusive expression vivo Kit-ligand lacking major proteolytic                 pigment stripe development zebra&#239;&#172;?sh. Dev Biol 167: 27&#226;&#8364;&#8220;33. <br />     cleavage site. Proc Natl Acad Sci U S 95: 11903&#226;&#8364;&#8220;11908.                                42. Rawls JF, Johnson SL (2001) Requirements kit receptor tyrosine <br /> 19. Altschmied J, Delfgaauw J, Wilde B, Duschl J, Bouneau L, et al. (2002)                      kinase regeneration zebra&#239;&#172;?sh &#239;&#172;?n melanocytes. Development 128: <br />     Subfunctionalization duplicate mitf genes associated differential                   1943&#226;&#8364;&#8220;1949. <br />     degeneration alternative exons &#239;&#172;?sh. Genetics 161: 259&#226;&#8364;&#8220;267.                        43. Reedy MV, Johnson RL, Erickson CA (2003) expression patterns c- <br /> 20. Parichy DM, Stigson M, Voss SR (1999) Genetic analysis steel PG-                 kit Sl chicken embryos suggest unexpected roles genes     M/versican-encoding gene AxPG candidates white (d) pigmenta-                     somite limb development. Gene Expr Patterns 3: 53&#226;&#8364;&#8220;58. <br />     tion mutant salamander Ambystoma mexicanum. Dev Genes Evol 209:                  44. Motro B, van der Kooy D, Rossant J, Reith  Bernstein (1991) Contiguous <br />     349&#226;&#8364;&#8220;356.                                                                                    patterns c-kit steel expression: Analysis mutations W Sl <br /> 21. Martin BL, Harland RM (2004) developmental expression                            loci. Development 113: 1207&#226;&#8364;&#8220;1221. <br />     Xenopus laevis steel homologues, Xsl-1 Xsl-2. Gene Expr Patterns 5: 239&#226;&#8364;&#8220;            45. Steel KP, Davidson DR, Jackson IJ (1992) TRP-2/DT, new early <br />     243.                                                                                        melanoblast marker, shows steel growth factor (c-kit ligand) 22. Wehrle-Haller B, Weston JA (1995) Soluble cell-bound forms steel                     survival factor. Development 115: 1111&#226;&#8364;&#8220;1119. <br />     factor activity play distinct roles melanocyte precursor dispersal               46. Kao KR, Bernstein (1995) Expression Xkl-1, Xenopus gene related     survival lateral neural crest migration pathway. Development 121:                    mammalian c-kit, dorsal embryonic tissue. Mech Dev 50: 57&#226;&#8364;&#8220;69. <br />     731&#226;&#8364;&#8220;742.                                                                                47. Baker CV, Sharpe CR, Torpey NP, Heasman J, Wylie CC (1995) Xenopus c- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                          0101                                       January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br />       kit-related receptor tyrosine kinase expressed migrating stem cells          53. Tusnady GE, Simon (2001) HMMTOP transmembrane topology <br />       lateral line  Mech Dev 50: 217&#226;&#8364;&#8220;228.                                                 prediction server. Bioinformatics 17: 849&#226;&#8364;&#8220;850. <br /> 48.   Kelsh RN, Brand M, Jiang YJ, Heisenberg CP, Lin S, et al. (1996) Zebra&#239;&#172;?sh              54. Felsenstein J (2005) Phylip (Phylogeny Inference Package), version 3.6 <br />       pigmentation mutations proce <br /> </body></html>