<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Microbiology                                                                                                                         BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access authentic methionine aminopeptidase genes differentially expressed Bacillus subtilis <br /> CongHui 1,2, HongYan Lu&#226;&#8364;&#160;3, Agnieszka Sekowska*4, Gang Fang2, <br /> YiPing Wang1, Anne-Marie Gilles*2 Antoine Danchin2 <br />  <br /> Address: 1National Laboratory Protein Engineering Plant Genetic Engineering, College Life Sciences, Peking University, 100871, <br /> Beijing, P.R. China, 2Institut Pasteur, Unit&#195;&#169; G&#195;&#169;n&#195;&#169;tique des G&#195;&#169;nomes Bact&#195;&#169;riens, 28 rue du Docteur Roux, 75724 Paris Cedex 15, France, 3HKU- <br /> Pasteur Research Centre, Dexter H. C. Man Building, 8 Sassoon Road Pokfulam, Hong Kong, S R., P.R. China 4CEA Saclay, Laboratoire Stress <br /> Oxydants et Cancer, DSV/DBJC/SBMS, Bat 142, 91191 Gif sur Yvette Cedex, France <br /> Email: CongHui - cyou@pasteur.fr; HongYan Lu - lv_hongyan@hotmail.com; Agnieszka Sekowska* - AS208663@dsvidf.cea.fr; <br /> Gang Fang - fangg@pasteur.fr; YiPing Wang - wangyp@pku.edu.cn; Anne-Marie Gilles* - amgilles@pasteur.fr; <br /> Antoine Danchin - adanchin@pasteur.fr <br /> * Corresponding authors &#226;&#8364;&#160;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 05 October 2005                                                    Received: 01 August 2005 <br />                                                                               Accepted: 05 October 2005 <br /> BMC Microbiology 2005, 5:57   doi:10.1186/1471-2180-5-57 article available  http://www.biomedcentral.com/1471-2180/5/57 <br /> &#194;&#169; 2005 et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: putative methionine aminopeptidase genes, map (essential) yflG (non- <br />                  essential), identified genome sequence Bacillus subtilis. investigated                  function methionine aminopeptidases explored possible reasons                  essentiality dispensability B. subtilis. <br />                  Results: silico analysis MAP evolution uncovered coordinated pattern MAP                  deformylase did correlate pattern 16S RNA evolution. Biochemical assays <br />                  showed MAP (MAP_Bs) YflG (YflG_Bs) B. subtilis overproduced Escherichia <br />                  coli obtained pure proteins exhibited methionine aminopeptidase activity vitro. <br />                  Compared MAP_Bs, YflG_Bs approximately orders magnitude efficient <br />                  assayed synthetic <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> substrates. map yflG genes expressed multi-copy <br />                  plasmids complement function defective map gene chromosomes E. <br />                  coli B. subtilis. contrast, lacZ gene transcriptional fusions showed promoter activity <br />                  map 50 100-fold higher yflG. Primer extension analysis detected                  transcription start site yflG promoter. work identified YvoA acted possible <br />                  weak repressor yflG expression B. subtilis vivo. <br />                  Conclusion: MAP_Bs YflG_Bs functional methionine aminopeptidases vitro                  vivo. high expression level map low expression level yflG account                  essentiality dispensality B. subtilis, respectively, cells grown laboratory <br />                  conditions. difference activity synthetic substrates suggests different <br />                  protein targets vivo. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 15 <br />                                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                  http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Background                                                      codes methionine aminopeptidase. attempt Ribosome-mediated protein synthesis initiated         better understand functional role MAP, studied methionine  eukaryotes) N-formylme-           evolutionary trend MAPs silico demonstrated <br /> thionine  prokaryotes eukaryotic organelles) [1].        map yflG genes B. subtilis code  removal N-formyl group           methionine aminopeptidases vitro vivo;  <br /> polypeptide peptide deformylase (DEF, EC 3.5.1.88),           finding high expression level map par- N-terminal methionine large number proteins         allel low expression yflG B. subtilis, cleaved methionine aminopeptidase (MAP, EC                account essentiality dispensability, <br /> 3.4.11.18) [2,3]. efficiency removal initiator    respectively. <br /> methionine defined highly conserved local sub- <br /> strate specificity, determined methionine      Results adjacent residue. MAP hydrolytically removes        Sequence alignment evolutionary pattern analyses <br /> N-terminal methionine penultimate resi-           genome sequence B. subtilis, genes bears small uncharged chain (<span id='am-42' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-43' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>Gly</span>, Ala, Ser,      code methionine aminopeptidases, map yflG, <br /> Thr, Pro, Val, Cys) [4-9], residues downstream        respectively [21] (Figure 1). sequence similarity penultimate residue little impact reaction.     identity E. coli MAP (MAP_Ec) B. subtilis MAPs studied <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> reported cobalt-            MAP (MAP_Bs) 65% 46%, respectively. Similarity <br /> dependent metalloproteases [3].  reports           MAP_Ec YflG B. subtilis (YflG_Bs) showed MAPs exhibited activity presence          55% identity 34% similarity divalent ions, Zn(II), Fe(II) Mn(II) [10-      MAP_Bs YflG_Bs 58% identity 36%. <br /> 12]. According sequence homology, MAPs              sequence YflG_Bs differs slightly grouped subtypes, type type II [13,14].       PROSITE consensus type MAPs (PS00680), insertion approximately 60 residues C-terminal        [MFY]-x-G-H-G- [LIVMC]-[GSH]-x(3)-H-x(4)-[LIVM]-x- <br /> domain type II methionine aminopeptidases             [HN]-[YWVH]. particular displays extra residue difference types [14]. Eukaryotes          conserved GHG metal-binding motif.  <br /> possess map genes, type type II      ingly, MAP_Bs similar MAP_Ec YflG_Bs [3], map gene prokaryotic       similar putative methionine ami- <br /> genomes, type (Archaea) type II (Bacteria) [15].    nopeptidase Staphylococcus aureus (labelled &quot;YMAP&quot; <br />                                                                 Figure 1). Overall proteins similar MAP distributed living organisms,        physico-<span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> properties  closely similar <br /> plays important physiological role. deletion      length slightly acidic).  shown Fig- <br /> single map gene prokaryotes Escherichia coli [16]    ure 1, metal (presumably cobalt) binding site (high- Salmonella typhimurium [17] lethal. yeast, dele-   lighted residues) conserved. important <br /> tion map genes (type type II)        investigate proteins methionine ami- <br /> causes slow growth phenotype, deletion          nopeptidase activity. <br /> genes lethal [18]. MAP activity essential role living cells? Previous studies showed    difference evolution proteins considered proteins needed N-terminal methio-           essential related organisms (Firmicutes) intriguing, removed normal biological activity, proper         explored evolution pattern map study <br /> subcellular localization eventual degradation               meant identify persistent genes bacteria [23]. (reviewed [19]). result, possible MAP     matched evolution 16S RNA, map gene <br /> essential essentiality downstream tar-    does follow linear course evolution, gets. essentiality, MAP changes dynamics         ribosomal proteins [23], shows erratic pattern (Fig- sulfur containing metabolites,            ure 2). suggests MAP involved differ- <br /> important role homeostasis cell.                  ent biological processes role methionine <br />                                                                 aminopeptidase according organisms. general, map gene genome         prompted explore evolution MAP pro- <br /> prokaryotes, exception cyanobacteria strain,      teins functionally related, using criteria functional methionine aminopeptidases           used identify persistent genes. defined persistent <br /> [20]. Surprisingly, genomes Bacilli,      genes genes present orthologs 85% putative MAP genes detected sequence             genomes clade  26 Firmicutes), alignment. B. subtilis, putative genes responsible       removing obligatory endosymbionts list  MAP activity, map yflG, identified [21].         Methods). protein divergence (sequence similarity <br /> Kobayashi et al. [22] reported map essential gene    divided sequence length difference) MAP B. subti- B. subtilis yflG  apparent paradox yflG   lis orthologs Firmicutes compared <br />  <br />  <br />                                                                                                                    Page 2 15 <br />                                                                                              (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure 1 alignment MAP E. coli, B. subtilis, S. aureus <br /> Sequences <br /> Sequences alignment MAP E. coli, B. subtilis, S. aureus. line shows protein sequence MAP B. subtilis (MAP_Bs), second shows MAP E. coli (MAP_Ec), shows YflG B. subtilis (YflG_Bs) shows YflG/MAP Staphylococcus aureus (YMAP_Sa). Identical amino acids shadowed dark conserva- <br /> tive amino acids shadowed grey. highlighted residues stand conserved amino acids important MAP <br /> activity (bold italic residues conservative basic ones, bold acidic ones). [MFY]-x-G-H-G-[LIVMC]-[GSH]- <br /> x(3)-H-x(4)-[LIVM]-x-[HN]-[YWVH]  H cobalt ligand) conserved motif protein sequences (placed alignment). <br />  <br />  <br />  <br />  bacterial evolutionary distance (measured 16S            molecular mass MAP_Bs YflG_Bs rRNA distance). Similar analyses performed           27409.17 &#194;&#177; 0.92 Da 27209.35 &#194;&#177; 2.14 Da, respectively. <br /> general protease, ClpP, DEF, ribosomal protein S4, RpsD          Interestingly, proteins purified E. coli retained YkrB, second deformylase present B. subtilis (Fig-      initial methionine, consistent nature ure 2). Interestingly, contrast expected evolution-    second residue, isoleucine cases. <br /> ary pattern RpsD, MAP exhibited erratic <br /> evolutionary way Def YkrB, functional             seen Table 1, enzymes exhibited methionine <br /> deformylases B. subtilis, suggesting evolution          aminopeptidase activity vitro synthetic peptide <br /> methionine aminopeptidase deformylase.                       substrates tested. differed widely,                                                                   extent activity: specific activity YflG_Bs Enzyme activity vitro                                         90- 20-fold higher MAP_Bs recognized gene function identification         tetrapeptide MGMM tripeptide MAS sub- <br /> solely based sequence comparisons mislead-           strates, respectively. enzymes preferred tetrapep- <br /> ing [24,25].  determined map          tide MGMM substrate. MAS MG yflG authentic MAP genes overproducing           substrates, MAP_Bs activity 76.9% 46.2% proteins E. coli characterizing enzymatic activ-    MGMM substrate, respectively, YflG_Bs <br /> ity. products map yflG genes purified        retained 17.7% 0.6% relative activity, respec- 90% homogeneity. Mass spectrometry showed              tively, substrates. <br />  <br />  <br />                                                                                                                      Page 3 15 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                     http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> YkrB 2 <br /> Evolutionary <br /> Figure         trend analyses MAP, ClpP, Def, RpsD Evolutionary trend analyses MAP, ClpP, Def, RpsD YkrB. Map, ClpP, Def, RpsD YkrB picked B. subtilis. BBHs Firmicutes detected. protein divergence (sequence similarity <br /> divided sequence length difference) B. subtilis orthologs Firmicutes compared bacte- <br /> rial evolutionary distance individually (measured 16S <br /> rRNA evolution). <br />  <br />  <br />  <br />  <br /> Table 1: Reaction rate MAP_Bs YflG_Bs different <br /> peptides substrates <br />  <br />                                    Specific Activity (U/mg) <br />  <br />  Peptide (4 mM)                  MAP_Bs               YflG_Bs <br />                                                                             activity <br />                                                                    Figure 3of <br />                                                                    Relative <br />                                                                    present                  MAP_Bs (B) YflG_Bs                                                                                          ions <br />                                                                                        <br />                                                                               different <br />                                                                    Relative activity   MAP_Bs (B) YflG_Bs                                                                    present different ions. MGMM (4 mM) used  MGMM                            2.6 &#194;&#177; 0.3           232.0 &#194;&#177; 8.2 <br />  MAS                             2.0 &#194;&#177; 0.1            41.0 &#194;&#177; 5.5 <br />                                                                    substrate. ions tested 0.2 mM concentration. <br />  MG                              1.2 &#194;&#177; 0.1            1.3 &#194;&#177; 0.1 <br />                                                                    relative activity defined 100%                                                                    presence <span id='am-17' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="CO2" datatype="xsd:string"></span>Co2</span>+ (MAP_Bs: 2.6 U/mg; YflG_Bs: 232.0 U/mg). <br />  measurements proteins different <br />  substrates  <br />  <br />  <br />                                                                    methionine aminopeptidase located C-termi- effect divalent ions (0.2 mM) enzyme activity            nal polypeptide chain [20,26-29], N- investigated (Figure 3). MAP_Bs significant activ-         terminal GFP fusions proteins expected <br /> ity Mn2+ Co2+. MAP_Bs retained                       interfere functions. addition,  <br /> appreciable activity (26&#226;&#8364;&#8220;50%) replacement Co2+            structed C-terminal GFP fusions MAP_Bs Cd2+, Zn2+, Ca2+, Mg2+, Cu2+, Sr2+ Ba2+,          YflG_Bs, possible degradation Ni2+. contrast, YflG_Bs showed strong preference          fusion proteins vivo unknown reasons, Co2+ decrease activity 20%             failed detect fluorescence case (data divalent ions tested.                                        shown). Cells strains BSIP8001 (MAP::GFP fusion) <br />                                                                    BSIP8002 (YflG::GFP fusion) mid-exponential <br /> Subcellular localization                                           growth phase collected visualized fluores- investigate MAP_Bs YflG_Bs dif-             cence microscopy, respectively. GFP-MAP_Bs (Figure <br /> ference subcellular localization, constructed N-       4A) GFP-YflG_Bs (Figure 4B) proteins distributed <br /> terminal green fluorescent protein (GFP) fusions           evenly cells. expected, intensity proteins. previously reported, catalytic domain          fluorescence proteins increased  <br />  <br />                                                                                                                       Page 4 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure 4 GFP-MAP_Bs GFP-YflG_Bs fusions <br /> Localization <br /> Localization GFP-MAP_Bs GFP-YflG_Bs fusions.  Localization GFP-MAP_Bs strain BSIP8001 grown minimal medium 0.5% xylose (w/v); B. Localization GFP-YflG_Bs strain BSIP8002 grown minimal medium 1% <br /> xylose (w/v). <br />  <br />  <br />  <br />  <br /> increasing concentration xylose used induce expres-        map expression considerably higher yflG <br /> sion GFP fusions (0.1%- 1% (w/v)) (data               expression <br /> shown).                                                          fact map essential yflG [22], <br />                                                                  ability yflG gene rescue MAP_Bs <br /> MAP-defective mutants rescued MAP_Bs YflG_Bs           function supplied high level, points substan- <br /> supplied trans                                                tial differences expression level genes. E. coli mutant EM9 contains engineered map gene,          test possibility, expression map yflG genes expression control isopropyl &#206;&#178;-D-          B. subtilis vivo studied using lacZ reporter <br /> thiogalactoside (IPTG), strain grow           gene. <br /> unless IPTG added growth medium [16]. Using replicative vector pBAD, holds arabinose-           Strains BSHP7042 BFS4611 grown minimal <br /> inducible promoter, map yflG genes B.               medium used monitor expression map <br /> subtilis individually introduced strain EM9.        yflG, respectively. map gene showed higher <br /> shown Figure 5, transformants complement            promoter activity (Figure 6). activity increased gradu- map defect IPTG omitted (leading MAP           ally log phase growth nearly fold <br /> deficiency) high concentration arabinose sup-        increase detected (109&#226;&#8364;&#8220;379 U/mg) early <br /> plied, case plasmids carrying B. subtilis map   exponential stationary growth phases. contrast, yflG genes. Lower concentrations arabinose par-       yflG gene showed low activity tially supported growth. demonstrated map         conditions tested (4&#226;&#8364;&#8220;8 U/mg) (Figure 6). summary, yflG B. subtilis supply MAP function E.      map gene promoter showed 50 100-fold higher activity <br /> coli vivo.                                                    yflG using lacZ reporter gene. <br />  B. subtilis, map essential gene, yflG      Promoter localization yflG gene <br /> [22].  map yflG copy         B. subtilis genome, yflG gene located upstream <br /> control IPTG provided multi-copy plasmid pDG148            yflH gene, followed putative Rho-independent <br /> (30&#226;&#8364;&#8220;50 copies using pUB110 replicon B. subtilis         transcription terminator. RT-PCR experiments demon- <br /> [30]), map::lacZ disruptants chromosome B. sub-        strated yflG yflH belonged common tran- <br /> tilis obtained (BSHP7046 BSHP7037).               scription unit, making operon (data shown). <br /> theless, witnessed cases systematic            Furthermore, lacZ transcription fusion yflH gene <br /> formation variegated blue white streaked colonies          (strain BSHP7043) showed activity changes X-gal plates, clones extremely unstable, leading        yflG gene conditions tested (data loss transformants generations (data       shown), substantiating RT-PCR results. order shown).                                                      identify promoter yflG-yflH operon, carried <br />                                                                  primer extension analysis. primers complemen- <br />                                                                  tary different regions inside yflG gene used <br />                                                                  yielding identification transcription start <br />  <br />  <br />                                                                                                                      Page 5 15 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                  http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br />                                                                Expression <br />                                                                Figure 6 map yflG lacZ transcriptional fusions <br />                                                                Expression map yflG lacZ transcriptional <br />                                                                fusions. &#206;&#178;-galactosidase expression map yflG lacZ <br />                                                                transcriptional fusion growth curve strains <br />                                                                BSHP7042 BFS4611 minimal medium. OD600 (&#226;??)                                                                &#206;&#178;-galactosidase activity (&#226;&#8212;?) measured strain <br />                                                                BSHP7042 map lacZ transcription fusion. OD600 (&#228;&#352;?)                                                                &#206;&#178;-galactosidase activity (&#226;&#8211;&#160;) measured strain <br />                                                                BFS4611 yflG lacZ transcription fusion. <br />  <br />  <br />  <br /> Figure 5rescue experiments <br /> Genetic <br /> Genetic rescue experiments. bacteria streaked M63 minimal solid medium plates incubated 37&#194;&#176;C overnight. concentration IPTG     point correlate structure promoter 1 mM. order plates picture described        poor expression yflG conditions tested. left right   EM9 <br /> host strain M63 IPTG; EM9 M63 IPTG. B. <br /> EM9 strain pBAD vector (ECHP1007) M63            YvoA showed slight repression effect yflG expression IPTG; ECHP1007 M63 arabinose (0.5% (W/V));        expression yflG B. subtilis vivo low, <br /> ECHP1007 M63 IPTG arabinose. C. EM9             endeavoured uncover conditions enhance strain caring pBAD map vector (ECHP1005) M63           expression. significant difference arabinose (0.5% (W/V)); ECHP1005 M63 IPTG;             carbon, nitrogen sulfur sources changed, ECHP1005 M63 IPTG; ECHP1005 M63             difference conditions sporulation ger- <br /> arabinose (0.02% (W/V)); ECHP1005 M63 arabinose        mination (data shown). genes proximal (0.1% (W/V)). D. EM9 strain caring pBAD yflG vector            chromosome code proteins related <br /> (ECHP1006) M63 arabinose (0.5% (W/V));                 functions, neighbourhood yflG analysed. Gene <br /> ECHP1006 M63 IPTG; ECHP1006 M63             nagP, encodes putative phosphoenolpyruvate- <br /> IPTG; ECHP1006 M63 arabinose (0.1% (W/V)); <br />                                                                dependent transport N-acetylglucosamine-specific <br /> ECHP1006 M63 arabinose (0.02% (W/V)). <br />                                                                enzyme IICB component, located upstream yflG                                                                transcribed divergently (Figure 8A). share com- <br />                                                                mon control elements. Exploring corresponding inter- <br />                                                                genic region sliding window 19 nt-long                                                                uncovered sequence AATTGGTATAGATCACTAG                                                                significant counterpart (AGCTGGTCTAGAT- <br /> point. shown Figure 7, start located 31 nt        CACTAG) upstream nagAB operon, encoding upstream ATG translation start codon regions        N-acetylglucosamine metabolic genes putative tran- <br /> weakly similar consensus -35 (TTCCTA) -10               scriptional regulator (GntR family) gene, yvoA (Figure <br /> (TAAGCT) regions upstream start point,       8A). prompted investigate YvoA separated 18 nt, rich region. difficult   possible regulator yflG. Using lacZ fusion,  <br />  <br />                                                                                                                    Page 6 15 <br />                                                                                              (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure <br /> Identification <br />          7     transcription start point yflG promoter <br /> Identification transcription start point yflG promoter. Primer extension sequencing reaction performed primer. size extended product compared DNA-sequencing ladder corre- <br /> sponding region. arrow indicates +1 site. Predicted -10 -35 regions bold boxed; experimental tran- <br /> scription start site bold; translational start site underlined. <br />  <br />  <br />  <br />  <br /> monitored expression yflG yvoA-disrupted            vitro (Table 1 Figure 3). Furthermore, gene rescue <br /> strain, BSIP8004.                                               experiments showed map yflG genes <br />                                                                 expressed replicative plasmids supply MAP shown Figure 8B, promoter yflG showed low        function E. coli B. subtilis, demonstrating consistently higher activity yvoA gene disrupted     MAP_Bs YflG_Bs functional methionine ami- <br /> strain BSIP8004 compared wild type par-           nopeptidases vivo (Figure 5). Interestingly, Kobayashi et <br /> ent BFS4611  fold higher activity). suggested          al. [22] reported disruption map gene B. sub- YvoA contribute regulation yflG            tilis lethal, deletion yflG  expression.                                                     present study, obtained B. subtilis map mutants                                                                 map yflG genes expressed multi-copy replicative <br /> Discussion                                                      plasmids control IPTG induced promoter prokaryotes, usually accepted   (strains BSHP7046 BSHP7037). observation sug- <br /> gene responsible methionine aminopeptidase.             gests single copy yflG gene expressed  B. subtilis genome program predicted              chromosome supply MAP function B. <br /> existence methionine aminopeptidase genes, map           subtilis, substantiating conditions yflG [21]. shown MAP_Bs               expression extremely low, making MAP signif- <br /> YflG_Bs functional methionine aminopeptidases            icant methionine aminopeptidase available.  <br />  <br />                                                                                                                     Page 7 15 <br />                                                                                               (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />                                                                  phosphate deaminase activity modulated amino- <br />                                                                  terminal methionine enzyme [31]. tried                                                                  growth conditions involving N-acetylglucosamine                                                                  carbon nitrogen source,  did                                                                  modulation YflG activity (data shown).                                                                  explored expression yflG::lacZ fusions sporula- <br />                                                                  tion germination. enhancement observed <br />                                                                  (data shown). consequence, surmise                                                                  high expression level map accounts essentiality                                                                  B. subtilis. <br />  <br />                                                                  published MAP_Ec activity [6,11] 5&#226;&#8364;&#8220;10 times <br />                                                                  higher MAP_Bs assay conditions.                                                                  related observation map gene E. <br />                                                                  coli does belong highly expressed ribosome pro- <br />                                                                  tein operon, contrast situation B. subtilis [32]. <br />                                                                  result, relative expression level map E. coli <br />                                                                  lower map gene B. subtilis.                                                                  unknown reasons impossible yflG <br />                                                                  gene function main methionine aminopeptidase <br />                                                                  B. subtilis vivo spite high activity YflG_Bs <br />                                                                  vitro. shown Table 2, enzyme activity MAPs <br />                                                                  crude cell extracts E. coli (BL5), B. subtilis (168)                                                                  yflG disrupted B. subtilis strain (BFS4611) showed                                                                  difference vitro condition tested. <br />                                                                  Different expression levels map genes genome car- <br />                                                                  rying methionine aminopeptidase genes <br />                                                                  previously suggested main reason                                                                  different physiological roles vivo, direct <br />                                                                  evidence            provided       [20,33].     Methionine <br />                                                                  aminopeptidases vivo active proteins, e.                                                                  complex polypeptides [34,35] short pep- <br />                                                                  tides vitro assays. specific tar- <br /> Expression <br /> Figure <br /> rupted strain <br />          8 yflG lacZ transcriptional fusions yvoA dis-     gets require specific aminopeptidase. similar <br /> Expression yflG lacZ transcriptional fusions yvoA          conclusion holds apparently paralogous genes. <br /> disrupted strain.  gene organization yflG           example, functional peptide <br /> yvoA. arrows indicate direction gene transcription;   deformylases B. subtilis, probably B. &#206;&#178;-galactosidase expression yflG lacZ transcriptional fusion   predominant deformylase concomitant high gene yvoA disrupted strain BSIP8004 yvoA wild type strain      expression [36]. <br /> BFS4611 grown minimal medium. OD600 (&#226;??) &#206;&#178;-galac- <br /> tosidase activity (&#226;&#8212;?) measured strain BSIP8004. <br /> OD600 (&#228;&#352;?) &#206;&#178;-galactosidase activity (&#226;&#8211;&#160;) measured         finding evolutionary parallel trend strain BSIP8004 strain BFS4611.                          methionine aminopeptidase deformylase Firmi- <br />                                                                  cutes (Figure 2) consistent functional relationship <br />                                                                  vivo, correlates fact deformy- <br />                                                                  lase removes N-formyl groups prerequisite                                                                  subsequent function MAP [33]. evolution corroborated lacZ fusion studies showing         signature protein-protein interactions [37,38], <br /> map gene expressed 50&#226;&#8364;&#8220;100-fold higher yflG          intriguing conjecture, supported conditions tested. Despite weak expression         consistent duplication deformylases genomes yflG,  detected sigma promoter using           methionine aminopeptidase (Table <br /> primer extension (Figure 7). showed         3). <br /> expression modulated YvoA (Figure 8), suggesting possible connection N-acetylglucosamine                  Conclusion <br /> metabolism methionine aminopeptidase.                proved MAP_Bs YflG_Bs functional <br /> respect worth noting E. coli glucosamine-6-        methionine aminopeptidases B. subtilis  <br />  <br />                                                                                                                       Page 8 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 2: MAP activity crude cell extracts E. coli B. subtilis <br />  <br />                                                                                 Specific Activity (U/mg) <br />  <br />          Peptide (4 mM)                           E. coli (BL5)                  B. subtilis wild type (168)        B. subtilis yflG mutant (BFS4611) <br />  <br />               MGMM                                 1.5 &#194;&#177; 0.2                             1.5 &#194;&#177; 0.1                             1.5 &#194;&#177; 0.1 <br />                MAS                                 1.1 &#194;&#177; 0.2                             1.1 &#194;&#177; 0.1                             1.1 &#194;&#177; 0.2 <br />                MG                                  0.6 &#194;&#177; 0.1                             0.8 &#194;&#177; 0.2                             0.4 &#194;&#177; 0.1 <br />  <br />  Cells stationary growth phase strains collected sonically disrupted. Supernatants cell debris used  measure MAP activity. Totally, times measurements carried  <br />  <br />  <br />  <br /> suggested map gene essential high                     experiments performed accordance expression level, yflG nonessential possibly                        European regulation requirements concerning  low expression level making                   tained use Genetically Modified Organisms Group compensate function map map                      (French agreement N&#194;&#176; 2735). expressed, specific targets dedicated MAPs. Conservation map-                       DNA techniques <br /> like genes Bacilli suggests involvement process                      DNA purification, restriction enzyme digestion, ligation <br /> specific group organisms, involve                       transformation E. coli performed according peptide deformylase.                                                    standard protocols [40]. cloning purpose, YieldAce                                                                              Pfu DNA polymerase (Stratagene) used. DNA <br /> Methods                                                                      sequences plasmids constructed work Bacterial strains, plasmids, culture conditions                          determined using dideoxy-chain termination method <br /> E. coli B. subtilis strains plasmids used             Thermo Sequenase Kit (Amersham Pharmacia <br /> work listed Table 4. E. coli TG1 XL1-Blue                    Biotech). B. subtilis cells transformed plasmid <br /> used cloning experiments (TG1 single cross                      DNA following steps protocol described previ- <br /> recombination replicative plasmids propagation                        ously [41]. Transformants selected LB plates  <br /> XL1-Blue double cross recombination) E.coli                     taining corresponding antibiotics IPTG needed. <br /> BL5 IPTG-induced protein overproducing. B. subtilis <br /> 168 used wild type strain study. E. coli                  Plasmid strain constructions <br /> cells grown Luria-Bertani (LB) medium [39]                     order clone genes B. subtilis protein pro- <br /> M63 minimal medium (KH2PO4, 4.4 mM; K2HPO4, 8                                ducing E. coli, map yflG genes amplified <br /> mM; (NH4)2SO4, 15 mM; MgSO4, 2 mM; ferric citrate, 34                        PCR genomic DNA B. subtilis 168 cloned <br /> &#194;&#181;M; VitB1, 0.0001% (w/v); fructose, 0.4% (w/v); sodium                       expression vector pET24b(+) (Novagen) using NdeI <br /> citrate, 0.3 mM; CaCl2, 50 &#194;&#181;M; MnCl2, 5 &#194;&#181;M; ZnCl2, 12 &#194;&#181;M;                    XhoI restriction sites. amplified fragments <br /> CuCl2, 2.5 &#194;&#181;M; CoCl2, 2.5 &#194;&#181;M; Na2MoO4, 2.5 &#194;&#181;M)                               included nucleotides +1 +744 nucleotides +1 supplemented final concentration arabinose                         +747 relative translational start point map 0.02% 0.5% (w/v) needed. B. subtilis cells                      yflG, respectively. resulting constructs, plasmids grown SP medium minimal medium                                 pIPP8003 pIPP8004, transformed E. coli <br /> (K2HPO4, 8 mM; KH2PO4, 4,4 mM; glucose, 27 mM; Na3-                          BL5 giving strains ECIP8003 ECIP8004 overpro- <br /> citrate, 0.3 mM; <span id='am-22' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span>L-glutamine</span>, 15 mM; <span id='am-59' about='obo:IMR_0200307' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-60' property="oboInOwl:id" content="IMR:0200307" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00078" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="L-Tryptophan" datatype="xsd:string"></span>L-tryptophan</span>, 0.244                     ducing MAP YflG, respectively. <br /> mM; ferric citrate, 33.5 &#194;&#181;M; MgSO4 <span id='am-65' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-66' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span>L-methionine</span>, 1 <br /> mM; MgCl2, 0.61 mM; CaCl2, 49.5 &#194;&#181;M; FeCl3, 49.9 &#194;&#181;M;                          construct map yflG GFP fusion plasmids MnCl2, 5.05 &#194;&#181;M; ZnCl2, 12.4 &#194;&#181;M; CuCl2, 2.52 &#194;&#181;M; CoCl2,                       strains, map yflG genes amplified PCR <br /> 2.5 &#194;&#181;M; Na2MoO4, 2.48 &#194;&#181;M). Antibiotics added                         genomic DNA B. subtilis 168 cloned N- following concentrations required: ampicillin,                      terminal GFP fusion vector pSG1729 [42] using XhoI 100 &#194;&#181;g/ml; spectinomycin, 80 &#194;&#181;g/ml; chloramphenicol, 5                       EcoRI restriction sites. amplified fragments included <br /> &#194;&#181;g/ml; phleomycin, 5 &#194;&#181;g/ml; kanamycin, 100 &#194;&#181;g/ml E.                      nucleotides +1 +744 nucleotides +1 +747 <br /> coli 5 &#194;&#181;g/ml B. subtilis. IPTG added 1 mM                     relative translational start point map yflG, <br /> concentration stated needed. Solid media                     respectively. resulting plasmids, pIPP8001 prepared adding 1.5% agar (w/v) respective liq-                    pIPP8002, linearized ScaI restriction enzyme <br /> uid media. Bacteria grown 37&#194;&#176;C. optical den-                     transformed B. subtilis 168. Clones sity (OD) bacterial cultures measured 600 nm. <br />  <br />  <br />                                                                                                                                         Page 9 15 <br />                                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 3: Distribution def map genes selected genomes <br />  <br />                                                                        Numbers Genome <br />  <br />  Genome                                                       def                    map                        EMBL <span id='am-64' about='oboInOwl:id' typeof='owl:Thing'>ID</span> <br />  <br />  Streptomyces coelicolor                                           4                        3                        AL645882 <br />  Streptomyces avermitilis                                          3                        4                        BA000030 <br />  Nocardia farcinica IFM 10152                                      3                        3                        AP006618 <br />  Shewanella oneidensis MR-1                                        3                        1                        AE014299 <br />  Silicibacter pomeroyi DSS-3                                       3                        1                        CP000031 <br />  Bacillus cereus ZK                                                2                        3                        CP000001 <br />  Bacillus thuringiensis serovar konkukian                          2                        3                        AE017355 <br />  Bacillus anthracis &apos;Ames Ancestor&apos;                                2                        3                        AE017334 <br />  Bacillus anthracis Sterne                                         2                        3                        AE017225 <br />  Bacillus cereus ATCC 10987                                        2                        3                        AE017194 <br />  Bacillus anthracis Ames                                           2                        3                        AE016879 <br />  Bacillus cereus ATCC 14579                                        2                        3                        AE016877 <br />  Leifsonia xyli subxyli CTCB07                                     2                        3                        AE016822 <br />  Synechocystis PCC 6803                                            1                        3                        BA000022 <br />  Bacillus clausii KSM-K16                                          1                        3                        AP006627 <br />  <br />  <br />  <br />  <br /> selected amylase deficient phenotype double        introducing SalI cloning site 5&apos; end SphI <br /> cross recombination amyE site [42], giving strains       cloning site 3&apos; end, respectively. fragments <br /> BSIP8001 BSIP8002, respectively.                             inserted SalI SphI sites IPTG- <br />                                                                  inducible replicative vector pDG148 [44], producing obtain map yflG expression plasmids           plasmid pHPP7021 pHPP7022, respectively. map gene rescue analysis conditional map gene dele-         wild type strain B. subtilis 168 transformed tion E. coli strain, EM9 [16], map (nucleotides -30      plasmids, giving strains BSHP7045 BSHP7038, <br /> +744 relative map translation start point) yflG       respectively. <br /> (nucleotides -21 +754 relative yflG translation <br /> start point) genes amplified PCR genomic            disrupt map gene, 246 bp-long fragment DNA B. subtilis 168 creation BamHI            map gene (nucleotides +142 +388 relative BglII sites. purified PCR fragments cleaved         map translation start point) amplified PCR BamHI BglII restriction enzymes inserted                 genomic DNA B. subtilis 168 using primers introducing sites plasmid pBAD containing            EcoRI cloning site 5&apos; end BamHI cloning <br /> arabinose inducible promoter  modified pBAD-&#206;&#177;&#206;&#178;&#206;&#179;                site 3&apos; end fragment. fragment plasmid containing additional BamHI cloning site          inserted EcoRI BamHI sites plasmid <br /> used removing BamHI BglII insert [43]), pro-         pJM783 [45], contains promoterless lacZ reporter <br /> ducing plasmids pHPP1017 pHPP1018, respectively.             gene, producing plasmid pHPP7025. plasmid proper orientation checked ability             introduced chromosome BSHP7045 cloned fragment recovered double digestion              BSHP7038 strains single cross event, giving <br /> using BamHI BglII restriction enzymes. Plasmids              strains BSHP7046 BSHP7037, respectively. ascer- <br /> pHPP1017, pHPP1018 pBAD vector trans-               tain correctly inserted, clones checked <br /> formed E. coli strain EM9 individually selected         PCR. LB plates containing ampicillin, chloramphenicol IPTG, giving strains ECHP1005, ECHP1006                      construct map transcriptional fusion lacZ <br /> ECHP1007, respectively.                                          gene, DNA fragment downstream map gene <br />                                                                  (nucleotides + 423 relative translation start obtain map yflG copies          point gene&apos;s stop codon) amplified PCR <br /> ditional map disruptant construction B. subtilis, map     genomic DNA B. subtilis 168 using primers intro- <br /> gene (nucleotides -43 +744 relative map transla-       ducing EcoRI cloning site 5&apos; end BamHI <br /> tion start point) yflG gene (nucleotides -54 733 rel-     cloning site 3&apos; end fragment, inserted <br /> ative yflG translation start point) amplified      EcoRI BamHI sites plasmid pJM783 [45], <br /> PCR genomic DNA B. subtilis 168 using primers            producing plasmid pHPP7017. plasmid intro- <br />  <br />  <br />                                                                                                                      Page 10 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                        http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 4: Bacterial strains plasmids used created study <br />  <br />  Strains             Genotype                                                                                     Source Reference <br />  <br />  Escherichia coli <br />  <br />  XL1-Blue            K12 supE44 hsdR17 recA1 endA1 gyrA46 thi relA1 lac- F&apos;[proAB+ lacIq lacZ&#226;&#710;&#8224;M15 Tn10(tetR)]     Laboratory collection <br />  TG1                 K12 supE hsd&#226;&#710;&#8224;5 thi &#226;&#710;&#8224; (lac-proAB) F&apos;[traD36 proA+ proB+ lacIq lacZ&#226;&#710;&#8224;M15]                       Laboratory collection <br />  BL5(DE3)            F-                                                                                           Invitrogen <br />  EM9                 PT7-map  chromosome)                                                                      Chiron Culture Collection Centre <br />  ECIP8003            F- pIPP8003                                                                                  work <br />  ECIP8004            F- pIPP8004                                                                                  work <br />  ECHP1005            EM9 pHPP1017                                                                                 work <br />  ECHP1006            EM9 pHPP1018                                                                                 work <br />  ECHP1007            EM9 pBAD                                                                                     work <br />  <br />  Bacillus subtilis <br />  <br />  168                 trpC2                                                                                        Laboratory collection <br />  BFS4611             trpC2 yflG::lacZ                                                                             Functional analysis projecta <br />  BFS817              trpC2 yvoA::lacZ                                                                             Functional analysis projecta <br />  BSHP7042            trpC2 map-lacZ                                                                               work <br />  BSHP7045            trpC2 pDG148-map                                                                             work <br />  BSHP7046            trpC2 map::lacZ pDG148-map                                                                   work <br />  BSHP7037            trpC2 map::lacZ pDG148-yflG                                                                  work <br />  BSHP7038            trpC2 pDG148-yflG                                                                            work <br />  BSHP7043            trpC2 yflH::lacZ                                                                             work <br />  BSIP8001            trpC2 amyE::gfp-map                                                                          work <br />  BSIP8002            trpC2 amyE::gfp-yflG                                                                         work <br />  BSIP8003            trpC2 yvoA::[(lacZ-ery)::pEC23(Kan)]                                                         work <br />  BSIP8004            trpC2 yflG::lacZ yvoA::[(lacZ-ery)::pEC23(Kan)]                                              work <br />  <br />  Plasmids            Description                                                                                  Source Reference <br />  <br />  pET24b(+)           expression vector, KanR                                                                      Novagen <br />  pBAD-abg            genetic rescue vector, AmpR                                                                  [43] <br />  pDG148              expression vector, AmpR KanR                                                                 [44] <br />  pJM783              cloning vector, CmR                                                                          [45] <br />  pSG1729             N-terminal GFP fusion vector, AmpR SpecR                                                     [42] <br />  pEC23               integrative plasmid, KanR                                                                    M. Simon P. Stragier, unpublished <br />  pIPP8003            pET24b map                                                                                   work <br />  pIPP8004            pET24b yflG                                                                                  work <br />  pHPP1017            pBAD map                                                                                     work <br />  pHPP1018            pBAD yflG                                                                                    work <br />  pHPP7017            pJM783 map-lacZ                                                                              work <br />  pHPP7018            pJM783 yflH::lacZ                                                                            work <br />  pHPP7021            pDG148 map                                                                                   work <br />  pHPP7022            pDG148 yflG                                                                                  work <br />  pHPP7025            pJM783 map::lacZ                                                                             work <br />  pIPP8001            pSG1729 map                                                                                  work <br />  pIPP8002            pSG1729 yflG                                                                                 work <br />  <br />    strain constructed frame project functional characterization genome B. subtilis Japan [46]. <br />  <br />  <br />  <br /> duced chromosome B. subtilis 168 single                          nese project functional analysis genome crossover event, giving strain BSHP7042.                                          B. subtilis, 2000 genes dis- <br />                                                                                   rupted fusion lacZ reporter gene [22,46,47]. yflG transcriptional fusion lacZ gene <br /> (yflG::lacZ disruptant, strain BFS4611) constructed                           construct yflH transcriptional fusion lacZ framework European Union Japa-                              gene, DNA fragment downstream yflH gene <br />  <br />  <br />                                                                                                                                            Page 11 15 <br />                                                                                                                       (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                 http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> (nucleotides + 32 +310 relative translation     o-dianisidine dihydrochloride 0.1 M Tris-HCl (pH <br /> start point) amplified PCR genomic DNA          7.4)), tube incubated 37&#194;&#176;C 10 min, B. subtilis 168 using primers introducing EcoRI cloning     absorbance 440 nm recorded. Standard curve site 5&apos; end BamHI cloning site 3&apos; end   known concentration L-methionine colour fragment, inserted EcoRI BamHI sites     development buffer plotted quantitative analysis. plasmid pJM783 [45], producing plasmid pHPP7018.            absorbance 1 &#194;&#181;mol methionine ml 440 plasmid introduced chromosome B.           nm equivalent 8.6 [6]. Unit activity subtilis 168 single cross event, giving strain       defined 1 &#194;&#181;mol methionine produced min BSHP7043.                                                      assay conditions used. Preference metal ions                                                                tested changing cobalt substrate solution obtain disruptant yvoA strain compatible           divalent metal ions final concentration BFS4611, BFS817 strain (yvoA::lacZ disruptant,        0.2 mM. European Union Japanese consortium,  transformed ScaI linearized pEC23 plasmid         Amylase activity detected growth B. subtilis <br />  carries kanamycin resistance gene, M. Simon       strains Tryptose Blood Agar Base (TBAB, Difco) supple- <br /> P. Stragier, unpublished) replacement kan-        mented 10 g/l hydrolyzed starch (Sigma). Starch <br /> amycin resistance gene lacZ Erm genes belong-       degradation detected sublimating iodine ing pMutin plasmid (erythromycin resistant)             plates [49]. <br /> integrated genome, homologous recombina- <br /> tion. resulting clones checked inability    &#206;&#178;-galactosidase specific activity measured described grow erythromycin chloramphenicol.               Miller [50]. Unit &#206;&#178;-galactosidase activity resulting strain named BSIP8003. chromosomal           defined enzyme produced 1 nmol <br /> DNA BSIP8003 prepared used transform         o-nitrophenol min 28&#194;&#176;C. Specific activity BFS4611 strain, producing double yvoA yflG dis-        expressed Units mg protein. experiments rupted strain BSIP8004.                                        performed triplicates. <br />  <br /> Expression purification MAP_BS YflG_Bs              Subcellular localization analysis <br /> proteins                                                       visualize GFP-MAP_Bs GFP-YflG_Bs fusions loca- strains ECIP8003 ECIP8004 grown 37&#194;&#176;C           tions living cells, BSIP8001 BSIP8002 strains LB medium containing kanamycin (25 &#194;&#181;g/ml)               grown minimal medium 0.5% fructose sole <br /> induced 0.5 mM IPTG hours. Cell pellets          carbon source supplemented xylose different final harvested centrifugation stored -20&#194;&#176;C.          concentrations (0.1% 1% (w/v)). Cells mid- <br /> Cells suspended 20 mM Tris-HCl (pH 8.0)            exponential growth phase collected visualized <br /> sonically disrupted. cellular debris removed        fluorescence microscopy. Microscope Axiovert 135 TV <br /> centrifugation 12,000 g 40 min. supernatants        (ZEISS, Germany) used experiment fluo- applied DEAE-sephacel column chromatogra-            rescence filter sets used visualize GFP obtained <br /> phy (Amersham). protein (MAP_Bs YflG_Bs)            Chroma Technology Corp (USA). Cells visual- <br /> eluted 0.1 M NaCl buffer. Protein         ized 1% agarose (w/v) slides [51] using Axiovision 4 <br /> centration determined Bradford&apos;s method [48]        (Allied High Tech Products, ) exposure <br /> using protein assay kit (Bio-Rad Laboratories). purity     time 3&#226;&#8364;&#8220;10 s. isolated proteins determined SDS-PAGE [40]. <br /> <span id='am-11' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>Ion</span> mass spectrometry used measure molecular        Genetic rescue experiment <br /> mass purified protein checking exist-   growth EM9, ECHP1005 (containing B. subtilis map <br /> ence methionine proteins.          gene), ECHP1006 (containing B. subtilis yflG gene)                                                                ECHP1007 (containing pBAD vector , Enzyme assays                                                  compared M63 minimal medium plates (37&#194;&#176;C,  methionine aminopeptidase assay performed           night) presence absence IPTG (1 mM) described Arie Ben-Bassat et al. [6] small modifi-     arabinose (0.02 %, 0.1 % 0.5 %). <br /> cations. 10 &#194;&#181;l protein solution added 90 &#194;&#181;l substrate solution (4 mM peptide 0.1 M K2HPO4               RNA isolation manipulation <br /> (pH7.5), 0.2 mM CoCl2), incubated 37&#194;&#176;C         Total cellular RNA extracted B. subtilis 168 cells <br /> 10 min. reaction stopped incubation 100&#194;&#176;C        growing minimal medium OD600 0.5 using 2 min. addition 900 &#194;&#181;l colour              High Pure RNA Isolation Kit Roche. RNA  <br /> development mixture (containing 0.2 mg L-<span id='am-35' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span>         centration determined light absorption 260 nm <br /> oxidase, 0.03 mg horseradish peroxidase, 0.2 mg      280 nm. 2 &#194;&#181;g RNA loaded 1.2% agarose <br />  <br />  <br />                                                                                                                  Page 12 15 <br />                                                                                             (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                     http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> gel check RNA purity integrity. RT-PCR experi-      enzyme activity analysis, helped draft manu- <br /> ments performed using RT-PCR (Promega)             script. AD designed supervised study. specified manufacturer. <br />                                                                Acknowledgements <br /> Primer extension analysis carried using reverse        initial stage work supported HKU-Pasteur Research <br /> transcriptase AMV (Roche) described [52]. oligo-        Centre (SAR Hong Kong, China) like thank HF Kung nucleotides (+33 +62 +97 +126 relative trans-     KY Yuen early moment. CH supported <br />                                                                EGIDE (France). like thank JD Huang kind gift lation start point yflG) used identification <br />                                                                pBAD-&#206;&#177;&#206;&#178;&#206;&#179; vector, Chiron Culture Collection Center conditional map yflG promoter. primers used gen-      gene deletion E. coli mutant, EM9, U Mechold kind suggestions eration sequence ladders. Reaction products            experiment manuscript drafting, YL Yap contribution sil- <br /> separated 7% denaturing polyacrylamide gel contain-         ico analyses discussion. <br /> ing 8 M <span id='am-81' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-82' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span>. DNA sequences determined using <br /> Sanger&apos;s dideoxy chain-termination method                 References <br /> &quot;Thermo Sequenase radiolabeled terminator cycle                1.    Kozak M: Comparison initiation protein synthesis pro- <br />                                                                      caryotes, eucaryotes, organelles. Microbiol Rev 1983, <br /> sequencing kit&quot; Amersham Pharmacia Biotech.                     47:1-45. <br />                                                                2.    Link AJ, Robison K, Church GM: Comparing predicted DNA sequence silico analysis                                  observed properties proteins encoded genome                                                                      Escherichia coli K-12. Electrophoresis 1997, 18:1259-1313. <br /> DNA sequences analyzed using DNA Strider soft-        3.    Bradshaw RA, Brickey WW, Walker KW: N-terminal processing: <br /> ware [53]. program BLAST [54] used search             methionine aminopeptidase N alpha-acetyl trans- <br /> homologous sequences database. program                    ferase families. Trends Biochem Sci 1998, 23:263-267. <br />                                                                4.    Huang S, Elliott RC, Liu PS, Koduri RK, Weickmann JL, Lee JH, Blair <br /> CLUSTALW used multiple alignment [55]. B. sub-               LC, Ghosh-Dastidar P, Bradshaw RA, Bryan KM: Specificity tilis sequences analyzed using SubtiList database           cotranslational amino-terminal processing proteins                                                                      yeast. Biochemistry 1987, 26:8242-8246. <br /> [56,57]. Accepted consensus sequences extracted           5.    Moerschell RP, Hosokawa Y, Tsunasawa S, Sherman F: specifi- PROSITE [58,59].                                                cities yeast methionine aminopeptidase acetylation                                                                      amino-terminal methionine vivo. Processing altered <br />                                                                      iso-1-cytochromes         c    created           oligonucleotide <br /> Evolutionary pattern analyses                                        transformation. J Biol Chem 1990, 265:19638-19643.  MAP orthologs (orthologs defined using         6.    Ben-Bassat  Bauer K, Chang SY, Myambo K, Boosman  Chang S: <br />                                                                      Processing initiation methionine proteins: prop- <br /> BBH (Bi-directional Best Hit) strategy, e. gene            erties Escherichia coli methionine aminopeptidase <br /> genome similar hit gene b genome B               gene structure. J Bacteriol 1987, 169:751-757. vice versa, gene b regarded pair         7.    Tsunasawa S, Stewart JW, Sherman F: Amino-terminal processing <br />                                                                      mutant forms yeast iso-1-cytochrome c. specifici- <br /> orthologs [60]) retrieved pair bacte-           ties methionine aminopeptidase acetyltransferase. J <br /> rial genomes. Secondly, BBHs, protein sequence              Biol Chem 1985, 260:5382-5391. <br /> similarity &gt;50% protein length difference &lt;1.3,     8.    Miller CG, Strauch KL, Kukral  Miller JL, Wingfield PT, Mazzei GJ, <br />                                                                      Werlen RC, Graber P, Movva NR: N-terminal methionine-spe- <br /> gene map B. subtilis collected. Thirdly,                 cific peptidase Salmonella typhimurium. Proc Natl Acad Sci U <br /> orthologs BBHs collected              S 1987, 84:2718-2722. <br />                                                                9.    Hirel PH, Schmitter MJ, Dessen P, Fayat G, Blanquet S: Extent N- <br /> genomes  total 206 bacterial genomes EMBL              terminal methionine excision Escherichia coli proteins <br /> July 1st, 2005), recursive identification            governed chain length penultimate <br /> members cluster longer extended.              amino acid. Proc Natl Acad Sci U S 1989, 86:8247-8251. <br />                                                                10.   Walker KW, Bradshaw RA: Yeast methionine aminopeptidase protein divergence (sequence similarity divided                   utilize Zn2+ Co2+ cofactor: case mis- <br /> sequence length difference) MAP B. subtilis            taken identity? Protein Sci 1998, 7:2684-2687. <br /> orthologs Firmicutes compared bac-        11.   D&apos;Souza V M, Holz RC: methionyl aminopeptidase                                                                      Escherichia coli function <span id='am-54' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span>iron</span>(II) enzyme. Biochem- <br /> terial evolutionary distance (measured 16S rRNA                   istry 1999, 38:11079-11085. <br /> evolution). analyses carried ClpP, Def,       12.   Wang J, Sheppard GS, Lou P, Kawai M, Park C, Egan DA, Schneider <br />                                                                       Bouska J, Lesniewski R, Henkin J: Physiologically relevant <br /> RpsD YkrB.                                                       metal cofactor methionine aminopeptidase-2                                                                      manganese. Biochemistry 2003, 42:5035-5042. <br /> Authors&apos; contributions                                         13.   Bazan JF, Weaver LH, Roderick SL, Huber R, Matthews BW: <br />                                                                      Sequence structure comparison suggest methio- <br /> CHY carried MAPs enzyme activity analysis, subcellu-             aminopeptidase, prolidase, aminopeptidase P, crea- <br /> lar observation, expression level comparisons, drafted           tinase share common fold. Proc Natl Acad Sci U S 1994, <br />                                                                      91:2473-2477. manuscript. HYL did genetic rescue experiment E.        14.   Arfin SM, Kendall RL, Hall L, Weaver LH, Stewart AE, Matthews BW, <br /> coli primer extension analysis. designed initial          Bradshaw RA: Eukaryotic methionyl aminopeptidases: experiments, completed genetic rescue experiment              classes cobalt-dependent enzymes. Proc Natl Acad Sci U S                                                                      1995, 92:7714-7718. <br /> B. subtilis, constructed lacZ fusion mutants B. subtilis,   15.   Li JY, Cui YM, Chen LL, Gu M, Li J, Nan FJ, Ye QZ: Mutations wrote manuscript. GF did evolutionary            S1 sites methionine aminopeptidases Escherichia <br /> pattern analyses. YPW helped analyse experimental             coli Homo sapiens reveal residues critical sub- <br />                                                                      strate specificity. J Biol Chem 2004, 279:21128-21134. <br /> data draft manuscript. AMG supervised MAPs <br />  <br />  <br />                                                                                                                          Page 13 15 <br />                                                                                                     (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                         http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> 16.   Chang SY, McGary EC, Chang S: Methionine aminopeptidase                      27.   Liu S, Widom J, Kemp CW, Crews CM, Clardy J: Structure       gene Escherichia coli essential cell growth. J Bacteriol                 human methionine aminopeptidase-2 complexed       1989, 171:4071-4072.                                                               fumagillin. Science 1998, 282:1324-1327. <br /> 17.   Miller CG, Kukral  Miller JL, Movva NR: pepM essential              28.   Griffith EC, Su  Turk  Chen S, Chang YH, Wu  Biemann K, Liu <br />       gene Salmonella typhimurium.                         J Bacteriol 1989,          JO: Methionine aminopeptidase (type 2) common <br />       171:5215-5217.                                                                     target angiogenesis inhibitors AGM-1470 ovalicin. <br /> 18.   Li X, Chang YH: Amino-terminal protein processing Saccha-                       Chem Biol 1997, 4:461-471. <br />       romyces cerevisiae essential function requires                29.   Zuo S, Guo Q, Ling C, Chang YH: Evidence zinc fingers <br />       distinct methionine aminopeptidases. Proc Natl Acad Sci U S                      methionine aminopeptidase Saccharomyces <span id='am-70' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-71' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span>cer</span>- <br />       1995, 92:12357-12361.                                                              evisiae important normal growth. Mol Gen Genet 1995, <br /> 19.   Lowther WT, Matthews BW: Structure function                             246:247-253. <br />       methionine aminopeptidases.                 Biochim Biophys Acta 2000,       30.   McKenzie T, Hoshino T, Tanaka T, Sueoka N: <span id='am-27' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> <br />       1477:157-167.                                                                      sequence pUB110: salient features relation 20.   Atanassova  Sugita M, Sugiura M, Pajpanova T, Ivanov  Molecular                replication regulation. Plasmid 1986, 15:93-103. <br />       cloning, expression characterization distinctive                31.   Lara-Gonzalez S, Dixon HB, Mendoza-Hernandez G, Altamirano MM, <br />       genes encoding methionine aminopeptidases cyanobacte-                           Calcagno ML: role N-terminal group allos- <br />       rium Synechocystis sp. strain PCC6803. Arch Microbiol 2003,                        teric function glucosamine-6-phosphate deaminase       180:185-193.                                                                       Escherichia coli. J Mol Biol 2000, 301:219-227. <br /> 21.   Kunst F, Ogasawara N, Moszer  Albertini  Alloni G, Azevedo V,           32.   Suh JW, Boylan SA, Oh SH, Price CW: Genetic transcrip- <br />       Bertero MG, Bessieres P, Bolotin  Borchert S, Borriss R, Boursier                tional organization Bacillus subtilis spc-alpha region. <br />       L, Brans  Braun M, Brignell SC, Bron S, Brouillet S, Bruschi CV,                 Gene 1996, 169:17-23. <br />       Caldwell B, Capuano V, Carter NM, Choi SK, Codani JJ, Connerton              33.   Giglione C, Serero  Pierre M, Boisson B, Meinnel T: Identification <br />        Cummings NJ, Daniel RA, Denizot F, Devine KM, Dusterhoft                     eukaryotic peptide deformylases reveals universality N- <br />       Ehrlich SD, Emmerson PT, Entian KD, Errington J, Fabret C, Ferrari E,              terminal protein processing mechanisms. Embo J 2000, <br />       Foulger D, Fritz C, Fujita M, Fujita Y, Fuma S, Galizzi  Galleron N,             19:5916-5929. <br />       Ghim SY, Glaser P, Goffeau  Golightly EJ, Grandi G, Guiseppi G,            34.   Palmiter RD, Gagnon J, Walsh KA: Ovalbumin: secreted pro- <br />       Guy BJ, Haga K, Haiech J, Harwood CR, Henaut  Hilbert H, Holsap-                 tein transient hydrophobic leader sequence. Proc <br />       pel S, Hosono S, Hullo MF, Itaya M, Jones L, Joris B, Karamata D, Kasa-            Natl Acad Sci U S 1978, 75:94-98. <br />       hara Y, KlaerrBlanchard M, Klein C, Kobayashi Y, Koetter P,                  35.   Jackson R, Hunter T: Role methionine initiation hae- <br />       Koningstein G, Krogh S, Kumano M, Kurita K, Lapidus  Lardinois S,                moglobin synthesis. Nature 1970, 227:672-676. <br />       Lauber J, Lazarevic V, Lee SM, Levine  Liu H, Masuda S, Mauel C,           36.   Haas M, Beyer D, Gahlmann R, Freiberg C: YkrB main pep- <br />       Medigue C, Medina N, Mellado RP, Mizuno M, Moestl D, Nakai S,                      tide deformylase Bacillus subtilis, eubacterium contain- <br />       Noback M, D, OReilly M, Ogawa K, Ogiwara  Oudega B,                        ing functional peptide deformylases. Microbiology 2001, <br />       Park SH, Parro V, Pohl TM, Portetelle D, Porwollik S, Prescott                  147:1783-1791. <br />       Presecan E, Pujic P, Purnelle B, Rapoport G, Rey M, Reynolds S, Rieger       37.   Goh CS, Cohen FE: evolutionary analysis reveals insights <br />       M, Rivolta C, Rocha E, Roche B, Rose M, Sadaie Y, Sato T, Scanlan E,               protein-protein interactions. J Mol Biol 2002, 324:177-192. <br />       Schleich S, Schroeter R, Scoffone F, Sekiguchi J, Sekowska  Seror SJ,      38.   Ramani AK, Marcotte EM: Exploiting evolution inter- <br />       Serror P, Shin BS, Soldo B, Sorokin  Tacconi E, Takagi T, Takahashi              acting proteins discover interaction specificity. J Mol Biol <br />       H, Takemaru K, Takeuchi M, Tamakoshi  Tanaka T, Terpstra P,                      2003, 327:273-284. <br />       Tognoni  Tosato V, Uchiyama S, Vandenbol M, Vannier F, Vassarotti          39.   Bertani G: Studies lysogenesis.  mode phage liber- <br />        Viari  Wambutt R, Wedler E, Wedler H, Weitzenegger T, Win-                    ation lysogenic Escherichia coli. J Bacteriol 1951, 62:293-300. <br />       ters P, Wipat  Yamamoto H, Yamane K, Yasumoto K, Yata K, Yosh-             40.   Sambrook J FEGMT: Molecular Cloning: Laboratory Manual. <br />       ida K, Yoshikawa HF, Zumstein E, Yoshikawa H, Danchin                        NY, Cold Spring Harbor; 1989. <br />       complete genome sequence Gram-positive bacterium                      41.   Kunst F, Rapoport G: Salt stress environmental signal <br />       Bacillus subtilis. Nature 1997, 390:249-256.                                       affecting degradative enzyme synthesis Bacillus subtilis. J <br /> 22.   Kobayashi K, Ehrlich SD, Albertini  Amati G, Andersen KK, Arnaud                 Bacteriol 1995, 177:2403-2407. <br />       M, Asai K, Ashikaga S, Aymerich S, Bessieres P, Boland F, Brignell SC,       42.   Lewis PJ, Marston AL: GFP vectors controlled expression <br />       Bron S, Bunai K, Chapuis J, Christiansen LC, Danchin  Debarbouille               dual labelling protein fusions Bacillus subtilis. Gene <br />       M, Dervyn E, Deuerling E, Devine K, Devine SK, Dreesen O, Err-                     1999, 227:101-110. <br />       ington J, Fillinger S, Foster SJ, Fujita Y, Galizzi  Gardan R, Eschevins   43.   Muyrers JP, Zhang Y, Testa G, Stewart AF: Rapid modification       C, Fukushima T, Haga K, Harwood CR, Hecker M, Hosoya D, Hullo                      bacterial artificial chromosomes ET-recombination. <br />       MF, Kakeshita H, Karamata D, Kasahara Y, Kawamura F, Koga K,                       Nucleic Acids Res 1999, 27:1555-1557. <br />       Koski P, Kuwana R, Imamura D, Ishimaru M, Ishikawa S, Ishio  Le            44.   Stragier P, Bonamy C, Karmazyn-Campelli C: Processing       Coq D, Masson  Mauel C, Meima R, Mellado RP, Moir  Moriya S,                   sporulation sigma factor Bacillus subtilis: morpholog- <br />       Nagakawa E, Nanamiya H, Nakai S, Nygaard P, Ogura M, Ohanan T,                     ical structure control gene expression. Cell 1988, <br />       O&apos;Reilly M, O&apos;Rourke M, Pragai  Pooley HM, Rapoport G, Rawlins                   52:697-704. <br />       JP, Rivas LA, Rivolta C, Sadaie  Sadaie Y, Sarvas M, Sato T, Saxild        45.   Perego M: Integrational vectors genetic manipulation       HH, Scanlan E, Schumann W, Seegers JF, Sekiguchi J, Sekowska                     Bacillus subtilis. Bacillus subtilis Gram-positive bacteria <br />       Seror SJ, Simon M, Stragier P, Studer R, Takamatsu H, Tanaka T,                    Edited  Sonenshein AL HJALR. Washington, D.C, Amercian Society <br />       Takeuchi M, Thomaides HB, Vagner V, van Dijl JM, Watabe K, Wipat                   Microbiology; 1993:615-624. <br />        Yamamoto H, Yamamoto M, Yamamoto Y, Yamane K, Yata K,                     46.   Bacillus Subtilis Genome Database                          [http://bacil <br />       Yoshida K, Yoshikawa H, Zuber U, Ogasawara N: Essential Bacillus                   lus.genome.ad.jp] <br />       subtilis genes. Proc Natl Acad Sci U S 2003, 100:4678-4683.                47.   Micado Release 6.1 [http://locus.jouy.inra.fr/cgi-bin/genmic/mad <br /> 23.   Fang G, Rocha E, Danchin  essential non-essential                        base/progs/madbase.operl] <br />       genes? Mol Biol Evol, press .                                             48.   Bradford MM: rapid sensitive method quantita- <br /> 24.   Brenner SE: Errors genome annotation. Trends Genet 1999,                        tion microgram quantities protein utilizing princi- <br />       15:132-133.                                              <br /> </body></html>