<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>4642&#226;&#8364;&#8220;4648 Nucleic Acids Research, 2000, Vol. 28,  23                                                              &#194;&#169; 2000 Oxford University Press <br />  <br />  <br />  NAD-dependent ligase encoded yerG essential gene Bacillus subtilis <br /> Marie-Agn&#195;&#168;s Petit* S. Dusko Ehrlich <br /> Laboratoire G&#195;&#169;n&#195;&#169;tique Microbienne, INRA, 78352 Jouy Josas, France <br />  <br /> Received August 29, 2000; Revised Accepted October 12, 2000 <br />  <br />  <br /> ABSTRACT                                                                       formed. crystal structure 41 kDa <span id='am-8' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-dependent <br />                                                                                ligase bacteriophage T7 solved (1), DNA ligases grouped families, ATP- <br />                                                                                adenylation domain NAD-dependent DNA ligase dependent NAD-dependent, according                                                                                Bacillus stearothermophilus (2). enabled Wigley cofactor required activity. surprising capa- <br />                                                                                colleagues (2) make structural alignment adenylation <br /> bility kinds ligases complement                          domain proteins, prove similar fold vivo observed. Bacillus subtilis                           share critical residues. recently  <br /> harbours NAD-dependent ligase, YerG,                               dimensional structure complete 76 kDa NAD-dependent <br /> ATP-dependent ligases, YkoU YoqV,                                ligase Thermus filiformis reported (3). encoded 134 kb lysogenic bacterio-                               NAD-dependent ligases uniform size (~70 kDa) phage SP&#206;&#178; consisting single adenylation                               reveal extensive <span id='am-19' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence conservation domain typical ATP-dependent ligases.                               entire protein. ATP-dependent ligases diverse genetics ligases B.subtilis                             size, share common ligase domain, &#226;&#710;&#188;270 residues studied previously, genes encoding ligase                          length, possibly flanked different domains. recently, kind, yerG yoqV, investigated.                             common view NAD-dependent ligases present yerG gene essential B.subtilis.                          eubacteria  ATP-dependent ligases suggests ATP-dependent                                   common living organisms, including bacterio- <br /> ligases able complement yerG defect.                             phages, archeabacteria, viruses eukaryotes.  addition, ATP-dependent ligase encoded                                  different picture emerged rising number yoqV, cloned plasmid appropriate                               complete bacterial genomes sequenced. expression signals, unable rescue yerG                                17 complete genomes eubacteria presently available, mutant strain. B.subtilis ligase genes yerG                        contain gene homologous NAD-dependent <br /> yoqV introduced Escherichia coli                               ligase family, contain copies genes <br /> strain encoding thermosensitive ligase (ligts),                          homologous ATP-dependent ligase family  yoqV did complement ligts defects,                             Bacillus subtilis, Mycobacterium tuberculosis Haemo- <br />                                                                                philus influenzae (Table 1). <br /> yerG fully complemented growth UV sensi- <br /> tivity defects lig mutant. propose rename yerG yoqV genes B.subtilis ligA ligB <br />                                                                                Table 1. Distribution DNA ligases B.subtilis, H.influenzae respectively.                                                                  M.tuberculosis <br />  <br />  <br /> INTRODUCTION                                                                   Bacterium species        NAD-dependent ligase      ATP-dependent ligase <br />                                                                                                         (length amino acids)   (length amino acids) <br /> DNA ligases catalyse joining breaks duplex DNA                       B.subtilis               YerG (668)                YoqV (270) DNA replication, repair recombination.                                                                                                                                   YkoUa (611) <br /> grouped families, ATP-dependent DNA ligases NAD-dependent ligases, according cofactor required                  M.tuberculosis           LigA (691)                LigB (507) activity. Apart common KXDG motif,                                                                         LigC (358) <br /> families share amino acid similarities.  reac-                                                                    O28549 (759) <br /> tions catalysed classes DNA ligase identical, <br />                                                                                H.influenzae             DNLJ (679)                DNL1 (268) fact <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span> group derived different cofactors. implies covalent <br />                                                                                aThisprotein mentionned putative ligase data banks, attachment AMP lysine residue KXDG motif, <br />                                                                                Pfam algorithm protein alignment includes ATP-dependent ligase <br /> followed transfer 5&#226;&#8364;&#178;-phosphoryl group nicked                family. conserved motives ATP-dependent ligase site duplex DNA. AMP group finally released                     present protein  alignment: http//www.sanger.ac.uk/ adenylated DNA phosphodiester bond                          Software/Pfam). <br />  <br />  correspondence addressed. Tel: +33 1 34 65 25 12; Fax: +33 1 34 65 25 21; Email: petit@biotec.jouy.inra.fr <br />                                                                                            Nucleic Acids Research, 2000, Vol. 28,  23 4643 <br />  <br />  <br />    eubacterial species, Escherichia coli Salmonella                 Saccharomyces cerevisiae strain encoding thermosensitive <br /> typhimurium, NAD-dependent ligase gene, encoded                     ligase (cdc9ts) (8). <br /> lig gene, known essential (4&#226;&#8364;&#8220;6).                    high degree complementation species ligase gene present,                 classes enzymes reported literature, known genes essential. question              possible tell priori ligase genes addressed regard B.subtilis. B.subtilis genome             B.subtilis genome overlapping functions, harbours NAD-dependent ligase-like gene, yerG,                     essential viability. ATP-dependent ligase-like genes, yoqV ykoU. yoqV                       decided investigate NAD-dependent ligase <br /> gene encoded 134 kb lysogenic bacteriophage SP&#206;&#178;,                     encoding gene, yerG, essential dispensable B.subtilis. ykoU apparent bacteriophage origin. YoqV                       gene essential B.subtilis. small protein composed 270 amino acids, consist                  addition, ATP-dependent ligase encoded single domain, adenylation domain typical ATP-dependent                 yoqV, cloned plasmid appropriate expression <br /> ligases. YkoU protein 611 amino acids length,                signals, unable rescue yerG mutant strain. indicates <br /> adenylation domain encompasses 292 residues.                ligases substitute  smaller  YoqV, studied               B.subtilis. B.subtilis ligase genes yerG yoqV ATP-dependent ligase genes dispensable, strain                   introduced E.coli ligts strain, cured SP&#206;&#178; prophage viable, ykoU mutant,                 yoqV does complement ligts defects, yerG constructed frame systematic                        fully complements growth UV sensitivity defects analysis unknown genes B.subtilis (Micado database:                      lig mutant. <br /> http://locus.jouy.inra.fr/cgi-bin/genmic/madbase_home.pl; strain <br /> constructed laboratory Dr K. Devine).                                MATERIALS METHODS <br />    Various studies shown heterologous complementation possible ATP-dependent NAD-dependent                            Strains plasmids <br /> enzymes. examples ATP-dependent ligases able                       Strains plasmids listed Table 2. Bacillus subtilis complement bacterial ligase mutants, shown                    E.coli cells grown LB medium unless bacteriophage T4 ligase able rescue lig mutant                  stated. needed, antibiotics added final concen- <br /> S.typhimurium (4), catalytic domain human ligase                  tration 5 &#194;&#181;g/ml kanamycin 0.5 &#194;&#181;g/ml erythro- <br /> restores thermoresistance E.coli strain encoding                   mycin B.subtilis, 100 &#194;&#181;g/ml ampicillin E.coli. thermosensitive ligase (ligts) (7). Conversely, NAD-                       chemically defined medium described (9). <br /> dependent ligase E.coli restores thermoresistance                  Constructions performed described  integrity <br />  <br />  <br />                Table 2. Strains plasmids <br />  <br />                                               Relevant genotype, description                           Reference, source <br />                Strains           JJC75a            lig7ts                                                   B.Michel <br />                                  168               trpC2                                                    C.Anagnostopoulos <br />                                  HVS610            168 insb pMAP149 yerH                                 work <br />                                  HVS609            168 ins pMAP52 yerG                                   work <br />                                  HVS611            168 ins pMAP59 yerG, plasmid pMAP141             work <br />  <br />  <br />                Plasmids          pDG148            pBR322-pUB110 KanR lacI Pspac                            (15) <br />                                  pMUTIN2           pBR322 lacZ lacI EmR Pspac                               (16) <br />                                  pMAP52            pMUTIN2 Pspac:(nt &#226;&#8364;&#8220;20c 275 yerG)                   work <br />                                  pMAP59            pMUTIN2MCS Pspac:(nt 330&#226;&#8364;&#8220;666 yerG)                    work <br />                                  pMAP65            pUB110 KanR lacI                                         (9) <br />                                  pMAP138           pDG148 Pspac:yoqV                                        work <br />                                  pMAP140           pDG148 Pspac:yerG                                        work <br />                                  pMAP149           pMUTIN2MCS Pspac:(nt 540&#226;&#8364;&#8220;854 yerH)                    work <br />                                  pIL253            EmR                                                      (17) <br />                                  pMAP139           pMAP138&#226;&#710;&#8224;(pBR322 lacI) AatII PstI, KanR           work <br />                                  pMAP141           pMAP140&#226;&#710;&#8224;(pBR322 lacI) AatII PstI, KanR           work <br />  <br />                aAB1157  background, hsdR thr+ pro+. <br />                binsindicates insertion given plasmid B.subtilis chromosome. <br />                cGene numbering starts start codon. <br />  4644 Nucleic Acids Research, 2000, Vol. 28,  23 <br />  <br />  <br />  <br />  <br /> Figure 1. Chromosomal structure wild-type strain 168 mutant strains. Strain 168 shown figure. strain HVS609, plasmid pMAP52 integrated yerG, fuse yerG Pspac promoter. strain HVS610, plasmid pMAP149 disrupting yerH gene. <br />  <br />  <br />  sequence DNA fragments generated PCR                        immediately upstream XbaI site followed SalI site. <br /> verified automated dideoxysequencing.                                         DNA fragment containing yoqV ORF flanked artificially <br />                                                                                  introduced XbaI SalI sites generated PCR yerG transcriptional fusion strain HVS609. fuse yerG                    cloned pDG148 E.coli. resulting plasmid, <br /> inducible promoter, plasmid pMUTIN2 used.                          pMAP138, used complementation tests E.coli. pBR322 derivative containing Pspac IPTG inducible promoter                     B.subtilis smaller derivative, pMAP139, deleted pBR322 <br /> upstream multiple cloning site flanked BamHI EcoRI.                  sequences lacI gene, AatII&#226;&#8364;&#8220;PstI deletion promoter preceded transcription terminators.                       pMAP138. Using strategy, yerG ORF downstream plasmid lacI gene                     cloned pDG148 pMAP140, smaller derivative selectable erythromycin resistance marker (EmR)                            devoid pBR322 lacI sequences, pMAP141, B.subtilis. DNA fragment starting 20 bp upstream                        obtained AatII&#226;&#8364;&#8220;PstI deletion pMAP140. <br /> yerG start codon, include ribosome binding site <br /> 296 bp length, flanked artificially introduced EcoRI                   Adenylation test BamHI sites, generated PCR cloned                            Bacillus subtilis E.coli crude extracts prepared pMUTIN2. resulting plasmid, pMAP52, contained tran-                        following way. 100 ml overnight culture centrifuged, <br /> scriptional fusion Pspac                       pellet stored &#226;&#8364;&#8220;80&#194;&#176;C. lyse cells, frozen pellet <br /> nucleotides yerG. generate fusion Pspac                    resuspended 1 ml 50 mM Tris pH 7.5, 5 mM DTT, complete ORF yerG, plasmid pMAP52 introduced                          1 mM EDTA, 2 mg/ml lysozyme, complemented cocktail single crossing chromosome B.subtilis wild-                  protease inhibitors (Roche). suspension incubated type 168 strain. resulting strain, HVS609 (Fig. 1), harbors                  20 min ice. Sonication applied 5 s using 1 s pulse/9 s single copy yerG gene control Pspac                    pause cycles, small probe (Vibracell apparatus). promoter.                                                                        soluble fraction (2 ml) collected 30 min ultracentri- <br />                                                                                  fugation 100 000 g, concentration ~4 mg/ml total <br /> yerH mutant strain HVS610. internal fragment yerH                      protein. <br /> gene, 350 bp length, generated PCR cloned                            test adenylation activity crude extracts, 4 &#194;&#181;l EcoRI BamHI sites pMUTIN2                             extract incubated buffer (50 mM Tris pH 7.5, 10 mM <br /> plasmid pMAP149. disrupt yerH, plasmid pMAP149                            MgCl2, 2 mM DTT, 0.1 mM ATP), final volume 20 &#194;&#181;l, <br /> introduced single crossing chromosome                      presence 5 &#194;&#181;Ci [&#206;&#177;-32P]ATP, 15 min 20&#194;&#176;C  B.subtilis strain HVS610 (Fig. 1).                                       ratio cold- radiolabelled ATP critical                                                                                  optimal detection adenylation activity crude extracts). <br /> Plasmid pMAP59. disrupt yerG gene, 314 bp internal                      Proteins denatured SDS buffer separated segment yerG generated PCR cloned                          polyacrylamide gel electrophoresis (PAGE). gel EcoRI BamHI sites pMUTIN2 plasmid                             fixed, colored dried prior exposition. Radioactivity pMAP59.                                                                          detected using Storm apparatus (Molecular Dynamics). <br />  <br /> Plasmids pMAP138 141. clone yoqV strong                              UV sensitivity test <br /> expression signals, plasmid pDG148 used vector.                   Exponentially growing cells diluted serially plated pBR322-pUB110 shuttle vector, replicates                         LB. Plates irradiated fluence 2 J/m2/s different <br /> E.coli B.subtilis. contains strong promoter active                 times, incubated growth 30&#194;&#176;C  strain JJC75) B.subtilis E.coli canonical ribosome binding site                   42&#194;&#176;C strains. <br />                                                                                                  Nucleic Acids Research, 2000, Vol. 28,  23 4645 <br />  <br />  <br />        Table 3. yerG, yoqV overproducing strain allows selection B.subtilis transformants harbouring yerG disruption <br />  <br />                                                          Proportion disrupted yerG clonesa transformantsb B.subtilis 168 strain containing: <br />        Transforming DNA                                  plasmid                      pMAP141 (yerG)                  pMAP139 (yoqV) <br />        Plasmid DNA pMAP59 (disrupts yerG)                N.R.                            5/15                            0/2 <br />        Total DNA strain <br />        HVS611 (yerG disrupted)                           0/48                            18/18                           0/18 <br />  <br />        aThe proportion transformants effectively containing yerG disruption resulting congression indicated  text details). <br />        bCompetent   cells indicated strains transformed EmR indicated DNA. strains tested similar level        competence, estimated transformation plasmid pIL253  1.3 &#195;&#8212; 105 3.7 &#195;&#8212; 105 transformants &#194;&#181;g DNA obtained). <br />        N.R., relevant  transformants obtained). <br />  <br />  <br /> RESULTS <br /> yerG essential gene NAD-dependent ligase gene located 55&#194;&#176; B.subtilis chromosome, gene putative operon <br /> encoding pcrB, pcrA, yerG yerH (Fig. 1). genes, pcrA, encoding DNA helicase SF1 <br /> family, characterised essential <br /> gene (9). pcrB yerH genes unknown match known protein organisms. <br />   yerG mutant strain constructed examine                          Figure 2. Growth curves strain HVS609 harbouring plasmid pMAP65. encoded NAD-dependent ligase enzyme essential                               Colony forming units (cfu, left) optical density (OD650, right) strain vivo. disrupt yerG gene, B.subtilis transformed                      grown presence (squares) absence (circles) IPTG                                                                                    reported function time. pMAP59, non-replicative plasmid containing internal fragment gene. transformants obtained rich media, chemically defined media. <br />   impossibility disrupting yerG result                         Overexpression yoqV encoded ATP-dependent ligase <br /> polar effect insertion downstream gene                        does rescue yerG mutation <br /> operon (yerH), requirement yerG viability. test alternative, yerH mutant constructed.                  potential ATP-dependent ligases encoded yerG gene, non-replicative plasmid containing                      B.subtilis genome, ykoU yoqV, essential. <br /> internal fragment yerH used transform B.subtilis.                        yerG essential wild-type strain B.subtilis,                                                                                    suggests ATP-dependent ligase complement <br /> Transformants obtained, contained appropriate <br />                                                                                    yerG activity. yoqV gene belongs SP&#206;&#178; <br /> gene disruption, resulting strain (strain HVS610, Fig. 1) <br />                                                                                    prophage, reasoned transcription likely did growth defect. concluded                                                                                    repressed chromosome. possibility existed yerH essential, turn, impossibility                                                                                    ATP-dependent ligase fused appropriate <br /> disrupt yerG likely requirement                           expression signals able complement yerG <br /> viability.                                                                         mutation. test possibility, plasmid constructed, <br />   confirm conclusion, strain HVS609 constructed,                       pMAP139, yoqV ORF cloned pUB110 yerG fused IPTG inducible promoter.                          derivative, strong expression signals, including investigated strain HVS609 able grow                         Pspac promoter canonical ribosome binding site. absence IPTG. sign lethality observed                          control, yerG gene cloned similarly pUB110 approximately 30 generations growth absence                     derivative, plasmid pMAP141. <br /> inducer. low concentration LacI                      order investigate new strain, cell  copy lacI gene inserted                    yoqV ATP-dependent ligase constitutively expressed, <br /> chromosome, Pspac&#226;&#8364;&#8220;yerG fusion), allowing                         survive absence yerG encoded NAD- <br /> leaky yerG expression. plasmid pMAP65, pUB110                               dependent ligase, attempt disrupting yerG non- <br /> derivative encoding LacI, introduced strain HVS609,                       replicative plasmid pMAP59  paragraph) repeated. <br /> arrest growth observed absence inducer,                        Table 3, lane 1, shows proportion transformants shown Figure 2. 2 h, number viable cells ceased                    yerG gene effectively interrupted various B.subtilis increase optical density continued increase                      strains. <br /> additional hour, result cell filamentation,                            B.subtilis strain harbouring yerG encoding plasmid <br /> leveling   strain HVS609 containing pMAP65                        pMAP141, transformants obtained (50/&#194;&#181;g IPTG-dependent growth. confirms yerG essential.                   plasmid DNA). expected resulted  4646 Nucleic Acids Research, 2000, Vol. 28,  23 <br />  <br />  <br /> reduced homology recipient transforming DNA: <br /> plasmid pMAP59 shares 337 bp homology yerG <br /> gene, 293 bp homology plasmids <br /> pMAP139 pMAP141 segment encompassing Pspac promoter. distinguish transformants pMAP59 recombined chromosomal copy yerG, pMAP59 recombined resident plasmid pMAP141, total DNA extracted 15 transformants. 15 clones contained disrupted <br /> copy yerG (Table 3, lane 1) determined Southern blot <br /> hybridisation  shown). 10 clones contained recombinant plasmid pMAP141 pMAP59. analysis concluded possible construct yerG disruption B.subtilis strain containing plasmidic       Figure 3. Presence adenylate activity specific E.coli B.subtilis <br /> copy gene. resulting strain named HVS611.            crude extracts strains containing plasmid overproducing YoqV. Crude <br />                                                                     extracts E.coli strains JM105 (lane 1), JM105 + pMAP138 grown    B.subtilis strain harbouring yoqV encoding            IPTG (lane 2), JM105 + pMAP138 grown IPTG (lane 3), B.subtilis <br /> plasmid pMAP139, transformation plasmid pMAP59                 strains 168 (lane 4) 168 + pMAP139 (lane 5), incubated radio- <br /> yielded transformants  trans-        labelled ATP, separated PAGE, stained Coomassie blue (right, formants contained disrupted copy yerG.         marker lane shown), exposed (left). Lane M, molecular weight markers, mixture plasmids, resident pMAP139,           relevant sizes indicated kDa. 14 kb plasmid recombinant pMAP139 pMAP59. low transformation efficiency, absence transformants harbouring yerG disruption, suggested yerG mutation tolerated strain expressing yoqV.         conclusion, plasmid pMAP141 present                                                                     recipient wild-type strain B.subtilis, chromosomal copy  given low number transformants, possi- <br />                                                                     yerG disrupted.  yoqV encoding <br /> bility disruption tolerated                                                                     plasmid pMAP139 present, transformants containing <br /> completely excluded. <br />                                                                     disrupted copy yerG obtained. <br />    increase chances yerG disruption strain 168 <br />                                                                        verify ATP-dependent ligase cloned harbouring pMAP139, attempt disrupting yerG          pMAP139 active, adenylation test performed  Instead pMAP59 source DNA, total DNA            B.subtilis crude extracts. step ligation process <br /> strain HVS611, pMAP59 integrated           consists covalent attachment AMP lysine <br /> yerG, used (Table 3, lane 2). case, homology      residue ligase, visualised easily resident transforming DNA greatly increased,         protein gel AMP radiolabelled. Extracts treated higher transformation frequencies expected.         RNase incubated presence [&#206;&#177;-32P]ATP 15 min total DNA strain HVS611 contains                room temperature analysed denaturing PAGE. pMAP141 plasmid DNA, possibility exists obtain &#226;&#8364;&#732;false        shown Figure 3, lanes 4 5, radioactive signal positive transformants&#226;&#8364;&#8482;, resulting congression, e.     detected samples extracted YoqV-containing <br /> simultaneous acquisition independent markers,                strain, absent B.subtilis wild-type extract. yerG::EmR plasmid pMAP141. strain 168 harbouring             signal corresponded protein ~30 kDa, close yerG encoding plasmid, high yield transformants            expected size 31 kDa YoqV. YoqV <br /> (3.5 &#195;&#8212; 104/&#194;&#181;g DNA) obtained expected,             ligase present active strain containing plasmid 18 clones analysed, contained interrupted copy      pMAP139. absence signal crude extracts yerG (Table 3, lane 2) determined PCR analysis.        B.subtilis wild-type extracts suggests YoqV <br /> strain 168, 10 times transformants (2 &#195;&#8212; 103/&#194;&#181;g DNA)         YkoU ATP-dependent ligases expressed detect- obtained, 48 clones analysed,               able level strain. <br /> resistant kanamycin, marker plasmid pMAP141. showed transformants &#226;&#8364;&#732;false positives&#226;&#8364;&#8482;.          yoqV ligase does complement ligts mutant strain 168 harbouring pMAP139, 6 &#195;&#8212; 103 transformants &#194;&#181;g         E.coli, yerG ligase does DNA obtained, result similar strain 168.          E.coli, C-terminal human ATP-dependent <br /> suggested transformants resulted congression   ligase able restore thermoresistance ligts strain (7), yerG::EmR pMAP141 DNA. Unlike recipient strain               S.typhimurium bacteriophage T4 ATP-dependent <br /> 168, strain containing plasmid pMAP139              ligase, contains minimal adenylation domain, like <br /> resistant kanamycin congression         yoqV gene product, allows growth NAD-dependent <br /> tested phenotypically. check presence plasmid         ligase disrupted strain (4). investigated pMAP141 transformants, total DNA 18 clones            YoqV able complement ligts strain E.coli.  <br /> extracted, plasmid chromosomal DNA                  plasmid pMAP138, hybrid composed pBR322, lacI analysed Southern blotting. clones, plasmid pMAP141       pMAP139, constructed. plasmid transformed effectively, combination          strain JJC75, containing lig7 ts allele, resulting <br /> plasmid pMAP139. proportion 0 18 clones        strain grown various temperatures. Plates incubated contained yerG disruption (Table 3, lane 2).                      42 30&#194;&#176;C shown Figure 4. ligts, strain <br />                                                                                                Nucleic Acids Research, 2000, Vol. 28,  23 4647 <br />  <br />  <br />  <br />  <br />                                                                                    Figure 5. UV sensitivity wild-type E.coli strain JJC40 grown 42&#194;&#176;C <br />                                                                                    (squares), ligts derivative JJC75 grown 30&#194;&#176;C (circles) JJC75 harbouring <br />                                                                                    yerG encoding plasmid pMAP140, grown 42&#194;&#176;C (triangles). fraction <br /> Figure 4. yerG, yoqV, restores thermoresistance E.coli ligts        surviving clones reported function UV dose. <br /> strain. Petri plates showing streaks LB E.coli ligts strain harbouring plasmid  plasmid), plasmid pMAP138 [pMAP138 (yoqV)] pMAP140 <br /> [pMAP140 (yerG)], grown 42   30&#194;&#176;C  . <br />                                                                                    (squares). conclusion, yerG gene product allowed                                                                                    restoration growth UV resistance defects                                                                                    ligts strain. <br /> harbouring plasmid pMAP138 did grow 42 37&#194;&#176;C  shown), presence absence IPTG. confirm defect, cells grown exponentially 30&#194;&#176;C                    DISCUSSION plating efficiency 42&#194;&#176;C relative 30&#194;&#176;C determined.                    B.subtilis, shown yerG-encoded NAD- 10&#226;&#8364;&#8220;5 survivors 42&#194;&#176;C ligts strain,                      dependent ligase, propose rename ligA, essential pMAP138. shows B.subtilis ATP-                       growth. somewhat unexpected, given pres- <br /> dependent ligase does complement growth defect                      ence potential ligase genes chromosome. <br /> ligts strain.                                                                      ligases, ykoU yoqV, belong family <br />    verify ATP-dependent ligase cloned                             ATP-dependent ligases. Observations shown pMAP138 active, adenylation test performed                           ATP-dependent ligases, originating organisms <br /> E.coli ligts crude extracts (Fig. 3, lanes 1&#226;&#8364;&#8220;3). radioactive                     completely unrelated bacteria, T4 bacteriophage <br /> signal detected samples extracted YoqV                             Homo sapiens, able restore viability containing strain, absent extract did                      S.typhimurium E.coli lig mutant (4,7), contain YoqV. 10-fold intense sample                         suggesting surprisingly broad range activity compati- expression yoqV derepressed addition                        bility enzymes. IPTG. showed ATP-dependent ligase encoded                            yoqV gene SP&#206;&#178; prophage, yoqV active E.coli crude extracts.                                       possibility existed absence complementation    Similar heterologous complementation experiments                           simply lack sufficient level expression. carried parallel yerG encoded NAD ligase.                          increase expression yoqV encoded ATP-dependent <br /> Plasmid pMAP140, hybrid pBR322, lacI pMAP141,                             ligase, propose rename ligB, gene cloned provides IPTG-dependent expression B.subtilis                         plasmid appropriate expression signals. NAD-dependent ligase E.coli, constructed                                case, yerG gene disrupted. investigate <br /> introduced ligts background. contrast yoqV                        activity yoqV, gene cloned E.coli experiment, strain did form individual colonies 42&#194;&#176;C                       ability complement growth defect lig7ts strain (Fig. 4), plated efficiency close 100%                     tested.  42&#194;&#176;C yoqV failed restore growth ligts <br /> temperature, absence IPTG. shows                      mutant. exclude possibility ligase encoded B.subtilis NAD-dependent ligase complements growth                             yoqV fact inactive enzyme, adenylation activity defect ligts strain.                                                        B.subtilis E.coli crude extracts tested    YerG ligase able restore growth                       present. appears  B.subtilis ligts strain 42&#194;&#176;C, ability complement UV sensitivity              E.coli, ATP-dependent ligases provide sufficient ts strain investigated. ligts strain               activity maintaining alive bacterial cells devoid UV sensitive 30&#194;&#176;C (Fig. 5, circles), defect disappeared                 native NAD-dependent ligase. presence yerG, 42&#194;&#176;C (triangles), strain                        cases complementation reported, UV resistant original wild-type strain                            noted phenotypes restored.  4648 Nucleic Acids Research, 2000, Vol. 28,  23 <br />  <br />  Chlorella virus ATP-dependent ligase E.coli NAD-         H.influenzae, harbour ligase gene,  <br /> </body></html>