<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#239;&#8250;&#8482; 2003 Genetics Society America <br />  <br />  <br />  <br />                 Identification Trans-dominant Modifiers Prat Expression <br />                                    Drosophila melanogaster <br />  <br />                                             Nicolas Malmanche Denise V. Clark1 <br />                       Department Biology, University New Brunswick, Fredericton, New Brunswick E3B 6E1, Canada <br />                                                       Manuscript received December 11, 2002 <br />                                                       Accepted publication March 25, 2003 <br />  <br />  <br />                                                                ABSTRACT <br />                 committed step purine novo synthesis pathway performed amidophosphoribosyl- <br />              transferase (EC 2.4.2.14) Prat. Drosophila melanogaster Prat essential gene promoter              lacks TATA-box initiator element multiple transcription start sites predominant start <br />              site. study regulation Prat expression adult eye, used Prat:bw reporter gene,              Prat coding region replaced brown (bw) coding region. pale-orange eye color <br />              single copy Prat:bw prompted use multicopy array Prat:bw derived using P <br />              transposase mutagenesis produces darker-orange eye color bw D; st genetic background.              used 13-copy array Prat:bw tool recover dominant EMS-induced mutations affect              expression transgene. screening 21,000 F1s deviation orange eye color, isolated <br />              23 dominant modifiers: 21 suppressors (1 Y-linked, 5 X-linked, 4 2-linked, 11 3-linked) 2 enhancers <br />              (1 2-linked 1 3-linked). Quantification effect endogenous Prat gene expression, using RT-PCR <br />              young adult fly heads, identifies subset modifiers candidates genes involved regulating <br />              Prat expression. <br />  <br />  <br />  <br /> T    Drosophila melanogaster Prat gene encodes      enzyme purine novo synthesis pathway, <br /> amidophosphoribosyltransferase (EC 2.4.2.14; Clark <br />                                                                             Prat limiting step novo purine pathway <br />                                                                             (Wyngaarden Kelley 1983). <br />                                                                                purine synthesis pathway regulated 1994). Primer extension analysis showed Prat                       Prat enzyme activity, stimulated phospho- <br /> gene multiple transcription initiation sites,                  ribosyl-<span id='am-1' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyrophosphate</span> inhibited purine nucleo- predominant site adult RNA. pro-                        tides (Holmes 1981). addition, regulation <br /> moter lacks TATA-box initiator element (D. Clark,                     pathway occurs transcriptional level unpublished observation). 5&#226;&#172;&#732; untranslated region                        Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae small intron alternative splice <span id='am-34' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acceptor</span>                      (Daignan-Fornier Fink 1992; Zalkin Dixon <br /> site rest gene intronless (Clark et al.                 1992; Rolfes Hinnebusch 1993). bacterial <br /> 1998). Prat mRNA expressed stages develop-                     cases, negative regulation occurs response  <br /> ment, low levels expression                        crease pathway end products  S. cerevisiae, <br /> larval stages high maternal expression ovaries                       positive regulation occurs response decrease leading mRNA accumulation 0- 2-hr-old embryos                      pathway end products. humans, genes encoding <br /> (Clark MacAfee 2000; Malmanche et al. 2003).                            Prat enzyme steps 6 7 pathway <br />    screen ethyl methanesulfonate (EMS)-induced                     divergently transcribed partly transcriptionally <br /> mutations region 84E1-2, Prat alleles                    controlled nuclear respiratory factor 1, phenotype described genes purine                        controls expression cytochrome genes (Chen et al. <br /> novo synthesis pathway recovered (Clark 1994).                         1997). addition, human <span id='am-24' about='obo:IMR_0200194' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00294" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200194" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Inosine" datatype="xsd:string"></span>inosine</span> monophosphate <br /> Analysis Prat alleles showed reduction                     mRNA levels post-transcriptionally controlled enzyme activity 40% wild-type level induces                   availability pathway end products (Glesne et al. &#226;&#8364;&#339;purine syndrome&#226;&#8364;? phenotype. contrast <br />                                                                             1991). Aside cases, little known genes enzymes pathway                                                                             genetic regulation novo purine pathway purine syndrome phenotype penetrant <br />                                                                             multicellular eukaryotes. Drosophila, genes mutants, detectable enzyme <br />                                                                             encoding enzymes novo purine pathway, ade2 <br /> activity (Henikoff et al. 1986c). observation  <br />                                                                             ade3, characterized genetic sistent finding studies different organisms <br />                                                                             molecular levels (Henikoff et al. 1986a,b,c; Tiong et al. <br />                                                                             1989; Tiong Nash 1993), regulation   1 <br />                                                                             genes transcriptional level relation    Corresponding author: Department Biology, Bag Service 45111, <br /> University New Brunswick, Fredericton, NB E3B 6E1, Canada.               pathway end products addressed. <br /> E-mail: clarkd@unb.ca                                                          Drosophila, dosage-sensitive second-site <br />  <br /> Genetics 164: 1419&#226;&#8364;&#8220;1433 (August 2003) <br />  1420                                              N. Malmanche D. V. Clark <br />  <br /> modifiers act trans suppress enhance gene            bw D; st males  F2, recombinant second chromo- <br /> expression described (Rabinow et al. 1991;               somes white-eyed males balanced using bw DSp/CyO; st. <br />                                                                    stocks crossed al dp b pr cn c sp flies Birchler et al. 1994; Bhadra Birchler 1996; Bha-               establish recessive markers present. recombi- <br /> dra et al. 1997b). effect modifiers ex-             nation mapping, modifier stocks crossed pression individual genes variable            appropriate marker chromosome stock. F1, heterozy- <br /> Drosophila life cycle affect various loci             gous females crossed bw D Sp/CyO; st second <br /> differently. genes corresponding                chromosome bw D; st/TM3, Sb st chromo- <br />                                                                     F2, individual heterozygous balanced males car- <br /> modifiers encode factors role expres-          rying potential recombinant chromosomes crossed sion genes affect, sought isolate mod-           bw DSp/CyO; st second chromosome bw D; st/TM3, <br /> ifiers Prat identify trans-acting factors important          Sb st chromosome amplification. Prat expression. Furthermore, Prat expression            200 250 recombinant chromosomes tested limits purine synthesis novo pathway mamma-              individual modifier. members complex comple- <br />                                                                    mentation group chromosome, total 500 <br /> lian cells (Yamaoka et al. 1997), predicted           recombinant chromosomes tested.  F3, het- genes encoding regulatory factors identi-          erozygous balanced males carrying potential recombinant <br /> fied isolation dosage-sensitive modifiers Prat            chromosomes tested crosses: (1) Prat:bw <br /> expression. study isolation 23          stock, test dominant effect eye color; (2) modifiers Prat:bw (2 enhancers 21 suppressors).             original modifier stock, test recessive lethal pheno- <br />                                                                    type; (3) marker chromosome stock, test Results pigment mRNA assays subset              markers present recombinant chromosome. modifiers effect Prat expression, dem-            chromosomal location modifiers determined <br /> onstrating Prat:bw reporter gene effective          using recombination mapping, deficiency mapping tool identifying candidate Prat regulatory genes.              associated recessive phenotype completed using deficiency <br />                                                                    kits second chromosomes Blooming- <br />                                                                    ton Drosophila Stock Center. <br />                                                                       Quantitative RT-PCR: Total RNA extracted 0-               MATERIALS METHODS                                4-hr-old adult fly heads genotype [Mod/&#226;&#171;&#185;] ; bw D; Prat:bw <br />    Drosophila stocks: Drosophila cultured standard         st [Mod/&#226;&#171;&#185;]; bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st. samples RNA extrac- <br /> media 25&#226;&#172;&#353;. Prat alleles P[ry&#226;&#171;&#185;; Prat:bw] transgene   tion collected microcentrifuge tubes ice, frozen described Clark (1994) Clark et al. (1998), respec-     liquid nitrogen, stored &#226;&#171;&#186;70&#226;&#172;&#353;. RNA extracted using <br /> tively. report, used 13-copy Prat:bw array inserted   Trizol reagent (Life Technologies/Invitrogen) reverse 65A10 bw D; st genetic background refer          transcriptase polymerase chain reactions (RT-PCR)  Prat:bw. P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] transgene stock, described       formed using RT-PCR beads (Ambion, Austin, TX). estab- <br /> Dreesen et al. (1991), bw D; st stock obtained   lish quantitative RT-PCR conditions set primers <br /> Steven Henikoff. T(2;3)Sbv/CyO, 1)wm4 1)wm4;         (rp49, Prat, bw), series RT-PCRs different num- <br /> Su(var)205E39A/CyO stocks obtained Amy Csink.        bers PCR cycles concentration RNA  fly stocks used study obtained         formed determine range cycles output <br /> Bloomington (Indiana) Stock Center. Description refer-         linear range amplification (Foley et al. 1993). <br /> ences alleles described article   conditions established using 0- 4-hr-old adult Flybase (FlyBase 1999).                                         fly head RNA bw D; Prat:bw st bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st.    Mutagenesis screen: bw D; st males starved 2 hr    established conditions 16 PCR cycles rp49 primers, 23 fed 1% sucrose solution 25 mm EMS (Sigma)           PCR cycles Prat primers, 23 PCR cycles bw primers 12 hr. mutagen treatment males crossed          bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st RNA, 21 PCR cycles bw primers masse bw D; Prat:bw st/TM6b females. eye colors       bw D; Prat:bw st RNA (Figure 1). primer pairs (Life progeny observed 0- 4-hr-old young adult flies      Technologies/Invitrogen) used 5&#226;&#172;&#732; TCCGTTCCCGAGT dissecting microscope. Males females showing         CTGGGCACAGCGGC 3&#226;&#172;&#732; 5&#226;&#172;&#732; fluo AGGCGGTGCAGT <br /> deviation bw D; Prat:bw st/st eye color individually     GGCCCGTTCCCTTG 3&#226;&#172;&#732; Prat, 5&#226;&#172;&#732; ACTCACGCAACGGACG <br /> backcrossed bw D; st. F2 showed deviation       GTGCAGGACG 3&#226;&#172;&#732; 5&#226;&#172;&#732; fluo AGGACCCCATAGCCAACAG bw D; Prat:bw st eye color, X second chromosome         CCGCCACC 3&#226;&#172;&#732; bw, 5&#226;&#172;&#732; CCAAGGACTTCATCCGC balanced using N 55e11/FM7c; bw D; st stock bw D Sp/    CACC 3&#226;&#172;&#732; 5&#226;&#172;&#732; fluo GCGGGTGCGCTTGTTCGATCC 3&#226;&#172;&#732; CyO; st stock, respectively. F2s showing deviation,     rp49 (Foley et al. 1993). &#226;&#8364;&#339;Fluo&#226;&#8364;? indicates fluorescein conju- chromosome balanced using bw D; st/TM3, Sb st          gated 5&#226;&#172;&#732; end primer pairs. stock chromosomes retested       cDNA synthesis performed 42&#226;&#172;&#353; 30 min using 1 &#226;?&#174;g <br /> effect bw D; Prat:bw st/st eye color. applicable,      total RNA oligo(dT) primer. PCR cycles 1 min <br /> modifiers tested inter se complementation       95&#226;&#172;&#353;, 1 min 66&#226;&#172;&#353;, 1 min 72&#226;&#172;&#353;. Following RT-PCR, respect lethal recessive phenotype.                       products separated agarose gel electrophoresis    Genetic mapping: Chromosome 2- 3-linked modifiers           blotted Nylon membrane (Roche) capillary method <br /> associated interesting effect endogenous Prat       (Sambrook et al. 1989). products detected using gene meiotically mapped using linkage recessive visible    antifluorescein-alkaline phosphatase antibody (NEN-Dupont) <br /> markers. second chromosome marked al dp b Bl pr cn          <span id='am-6' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="rdfs:label" content="CDP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span>CDP</span>-star chemiluminescent substrate (New England Bio- <br /> c bw D chromosome marked ru h th st cu sr e       labs, Beverly, MA). signal detected using Kodak used recombination mapping second-           Ultident X-ray film films scanned Epson <br /> chromosome-linked modifiers, respectively. construct        Expression 626 scanner transparent film adaptor second chromosome mapping bw D; st genetic           measurement using Scion Image Windows.  ground, al dp b Bl pr cn c/CyO females crossed bw D;     modifiers, replicate experiments performed <br /> st males. F1, heterozygous non-Cy females crossed      genotype, beginning independent RNA isolation. <br />                                                   Trans-modifiers Prat Expression                                            1421 <br />  control RNAs extracted aliquot           carrying multicopy array directly related num- <br /> extract used set experiments (beginning         ber transgene copies array (Clark et al. <br /> reverse transcription, PCR, detection). Results RT- <br /> PCR shown Table 2 ratio Prat mRNA/             1998). present study, used 13-copy array rp 49 mRNA control experiment vs. Prat mRNA/rp 49             P[ry&#226;&#171;&#185;; Prat:bw], localized chromosome mRNA modifier background experiment. ratio               65A10, referred Prat:bw. indicates independent measurement. example           13-copy array produces nonsaturated orange eye color, <br /> RT-PCR data analysis, ratios footnote c Table 2          allowing screen enhancers suppres- obtained analysis data shown Figure 2. <br />    Pigment assays: pigment quantification, modifier         sors eye color bw D; st genetic background. crossed bw D; Prat:bw st stock bw D; P[ry&#226;&#171;&#185;;    level pteridine pigment Prat:bw fly heads reflects <br /> bw&#226;&#171;&#185;] st stock. Adult flies grouped appropriate geno-       Prat expression prepupal types, aged 5 days 25&#226;&#172;&#353;, frozen liquid nitrogen,        pupal stages young adult fly heads pigment <br /> vortexed, separating heads bodies. Prior extrac-         deposition occurs (Dreesen et al. 1988). second <br /> tion, heads manually selected collected 30% <br /> ethanol (pH 2) incubated 3 days rotator room         stage screen test isolated modifiers temperature (Henikoff Dreesen 1989). heads               Prat:bw eye color effect mRNA expression <br /> genotype used assay. Following centrifugation,          endogenous Prat gene. modifiers loss- <br /> absorbance supernatant measured 480 nm using           function mutations, correspond Cary spectrophotometer. genotype, 7           genes limiting Prat expression. <br /> 10 assays performed. mean, standard deviation, P value t-test calculated.                        screened &#226;&#181;&#8216;21,000 F1 flies carrying EMS-mutagen- <br />    Variegation: Modifiers crossed bw&#226;&#171;&#185;; st stock       ized chromosomes deviation nonsatu- <br /> study effect bw D/ bw&#226;&#171;&#185; eye-pigment variegation. Adult       rated Prat:bw orange eye color 0- 4-hr-old young <br /> flies appropriate genotype aged 3 5          adult flies  materials methods). isolated <br /> days prior observation. estimation modifier&#226;&#8364;&#8482;s           23 modifiers Prat:bw eye color: 2 enhancers (1 <br /> effect variegation performed visual observation photography. analysis Mod-Y-1 effects wm4, wm4;            2- 1 3-linked) 21 suppressors (1 Y-linked, 5 <br /> Su(var)205E39A/CyO females crossed Mod-Y-1/&#226;&#171;&#185;; bw D;        X-linked, 4 2-linked, 11 3-linked). st males, generating Mod-Y-1/wm4; bw D/CyO; &#226;&#171;&#185;/st males,        modifiers Mod(Prat:bw)m-n, m means linkage crossed wm4; &#226;&#171;&#185;; &#226;&#171;&#185; females. resulting Mod-Y-1/       group n means arbitrary number.  wm4; &#226;&#171;&#185;/CyO; &#226;&#171;&#185;/st  &#226;&#171;&#185;) males observed             refer simply Mod-m-n. X-linked modifiers <br /> modification wm4 phenotype absence bw D st. Control males generated way using           isolated female F1s, supporting idea parental &#226;&#171;&#185;; bw D; st stock males cross. Flies     spontaneously arising modifiers, X chro- <br /> collected aged 25&#226;&#172;&#353; 10 days prior photography.          mosome, undetectable. sorted 23 sup- analysis Mod-Y-1 effects Sbv, Sbv/CyO females         pressors classes reflecting strength crossed Mod-Y-1 ; bw D; st &#226;&#171;&#185;; bw D; st males       effect Prat:bw pale-orange eye color F1, 14 thoracic bristles scored Stubble phenotype <br /> (Csink et al. 1994). Stubble bristles indicate suppres-       young adult flies (Table 1). enhancers  <br /> sion Sbv phenotype higher level expression        creased pigment pale orange light red. Sb allele. data represented box plot data,           21 suppressors, 15 reduced pigment pale  <br /> generated using Minitab software.                                    ange yellow 6 showed weaker suppression                                                                      pale orange. Complementation tests recessive <br />                                                                      lethality revealed complementation groups,                           RESULTS <br />                                                                      second chromosome alleles    Isolation 23 dominant modifiers Prat:bw:                chromosome alleles  Genetic map- <br /> screen designed stages. stage             ping dominant modifiers associated recessive pheno- recover trans-acting dominant modifiers affecting             type). <br /> Prat:bw reporter gene phenotype. used Prat                    Pigment assays modifiers grouped <br /> porter gene, P[ry&#226;&#171;&#185;; Prat:bw], changes eye color            classes: quantified effect 23 white pale orange bw D; st genetic                 dominant modifiers pigment level bw D; <br /> ground (Clark et al. 1998). Prat:bw consists 8.9-              Prat:bw st/st bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st/st stocks. modifier <br /> kb genomic DNA fragment Prat coding                     affects general aspect transcription affects <br /> region replaced frame fusion               expression post-transcriptionally, expect amino-terminal Prat sequence bw coding                   similar effect modifier transgenes. <br /> region cDNA. Prat coding region                   modifier different effect Prat:bw <br /> uninterrupted introns, assume Prat:bw                   P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;], acting effects tained sequences necessary Prat regulation               sequences specific Prat:bw transgene. transcriptional activity Prat gene               main differences exist Prat:bw <br /> monitored adult fly eyes using Prat:bw reporter               transgene P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] transgene.  gene. used multicopy array transgene                P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] transgene construct 8.4-kb generated rounds transposase muta-                 genomic fragment bw locus sufficient <br /> genesis. level pteridine pigment eyes flies             rescue bw mutant phenotype (Dreesen et al. <br />  1422                                              N. Malmanche D. V. Clark <br />  <br />                                                                 TABLE 1 <br />                                      Effect modifiers pteridine eye-pigment levels <br />  <br />                      Screen                                        t-test     Screen                                        t-test <br /> Modifier    Class    effecta     n      bw D; Prat:bw st/st b     P valuec    effecta   n      bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st/st b   P valuec <br /> Control                         10       0.079   &#226;&#171;&#190;   0.003                              10         0.137   &#226;&#171;&#190;   0.005 <br /> Mod-1-1       1         ss      10       0.047   &#226;&#171;&#190;   0.006         10&#226;&#171;&#186;9         ss      10         0.077   &#226;&#171;&#190;   0.005        10&#226;&#171;&#186;15 <br /> Mod-1-2       1         ss      10       0.057   &#226;&#171;&#190;   0.007         10&#226;&#171;&#186;6         ne      10         0.114   &#226;&#171;&#190;   0.007        10&#226;&#171;&#186;7 <br /> Mod-1-3       1         ss      10       0.060   &#226;&#171;&#190;   0.004         10&#226;&#171;&#186;8         ws      10         0.098   &#226;&#171;&#190;   0.014        10&#226;&#171;&#186;6 <br />  <br /> Control                         10       0.067 &#226;&#171;&#190; 0.003                                  10         0.133 &#226;&#171;&#190; 0.008 <br /> Mod-1-4       1         ss      10       0.038 &#226;&#171;&#190; 0.005             10&#226;&#171;&#186;9         ss      10         0.070 &#226;&#171;&#190; 0.002            10&#226;&#171;&#186;10 <br />  <br /> Control                         10       0.067 &#226;&#171;&#190; 0.004                                  10         0.135 &#226;&#171;&#190; 0.005 <br /> Mod-1-5       1         ss      10       0.031 &#226;&#171;&#190; 0.004             10&#226;&#171;&#186;13        ss      10         0.043 &#226;&#171;&#190; 0.002            10&#226;&#171;&#186;17 <br />  <br /> Control                         10       0.070   &#226;&#171;&#190;   0.005                              10         0.136   &#226;&#171;&#190;   0.010 <br /> Mod-2-1       1         ss      10       0.040   &#226;&#171;&#190;   0.0057        10&#226;&#171;&#186;10        ss      10         0.082   &#226;&#171;&#190;   0.0083       10&#226;&#171;&#186;10 <br /> Mod-2-2       1         ss       9       0.040   &#226;&#171;&#190;   0.0096        10&#226;&#171;&#186;5         ss      10         0.088   &#226;&#171;&#190;   0.0096       10&#226;&#171;&#186;9 <br /> Mod-2-5       2         se       9       0.100   &#226;&#171;&#190;   0.011         10&#226;&#171;&#186;6              10         0.190   &#226;&#171;&#190;   0.0087       10&#226;&#171;&#186;10 <br />  <br /> Control                         10       0.068 &#226;&#171;&#190; 0.005                                  10         0.130 &#226;&#171;&#190; 0.010 <br /> Mod-2-3       4         ws      10       0.050 &#226;&#171;&#190; 0.002             10&#226;&#171;&#186;7         ne      10         0.146 &#226;&#171;&#190; 0.009            0.001 <br /> Mod-2-4       1         ss       9       0.049 &#226;&#171;&#190; 0.003             10&#226;&#171;&#186;8         ne      10         0.114 &#226;&#171;&#190; 0.008            0.0006 <br />  <br /> Control                         10       0.069   &#226;&#171;&#190;   0.006                              10         0.122   &#226;&#171;&#190;   0.013 <br /> Mod-3-6       1         ws      10       0.057   &#226;&#171;&#190;   0.005        0.00035       ne       9         0.112   &#226;&#171;&#190;   0.008        0.05 <br /> Mod-3-2       3         ws      10       0.066   &#226;&#171;&#190;   0.006          0.37        ne       8         0.132   &#226;&#171;&#190;   0.010        0.10 <br /> Mod-3-10      3         ws       8       0.070   &#226;&#171;&#190;   0.007          0.78        ne       9         0.126   &#226;&#171;&#190;   0.009        0.53 <br /> Mod-3-7       1         ss      10       0.058   &#226;&#171;&#190;   0.0003        0.0003       ss      10         0.070   &#226;&#171;&#190;   0.007        10&#226;&#171;&#186;7 <br />  <br /> Control                         10       0.063   &#226;&#171;&#190;   0.004                              10         0.136   &#226;&#171;&#190;   0.009 <br /> Mod-3-1       2         se       7       0.075   &#226;&#171;&#190;   0.007       6 &#226;&#171;&#187; 10&#226;&#171;&#186;3       ne       7         0.155   &#226;&#171;&#190;   0.010        0.002 <br /> Mod-3-11      3         ss       7       0.059   &#226;&#171;&#190;   0.004         0.07         ss       8         0.140   &#226;&#171;&#190;   0.008        0.32 <br /> Mod-3-9       3         ws      10       0.063   &#226;&#171;&#190;   0.005         0.99         ne       8         0.135   &#226;&#171;&#190;   0.010        0.80 <br />  <br /> Control                         10       0.072   &#226;&#171;&#190;   0.004                              10         0.139   &#226;&#171;&#190;   0.007 <br /> Mod-3-3       1         ws       9       0.057   &#226;&#171;&#190;   0.006       1.4 &#226;&#171;&#187; 10&#226;&#171;&#186;4     ws       8         0.121   &#226;&#171;&#190;   0.01        3 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br /> Mod-3-4       4         ws       9       0.080   &#226;&#171;&#190;   0.006         0.0071       ss       8         0.107   &#226;&#171;&#190;   0.005       2 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br /> Mod-3-5       1         ss       7       0.058   &#226;&#171;&#190;   0.004          10&#226;&#171;&#186;5        ne       7         0.128   &#226;&#171;&#190;   0.003       6 &#226;&#171;&#187; 10&#226;&#171;&#186;4 <br />  <br /> Control                         10       0.069 &#226;&#171;&#190; 0.004                                  10         0.123 &#226;&#171;&#190; 0.009 <br /> Mod-3-8       3         ss       7       0.063 &#226;&#171;&#190; 0.009             0.175        ne      10         0.125 &#226;&#171;&#190; 0.005             0.55 <br /> Mod-3-12      3         ws       9       0.064 &#226;&#171;&#190; 0.004             0.043        ne       7         0.121 &#226;&#171;&#190; 0.019             0.80 <br />  <br /> Control                         10       0.057 &#226;&#171;&#190; 0.004                                  10         0.148 &#226;&#171;&#190; 0.009 <br /> Mod-Y-1       1         ss      10       0.033 &#226;&#171;&#190; 0.004             10&#226;&#171;&#186;9         ss      10         0.112 &#226;&#171;&#190; 0.007            10&#226;&#171;&#186;9 <br />   ss, strong suppressor; ne, effect; ws, weak suppressor; se, strong enhancer;  weak enhancer. <br />       effect modifiers eye color observed transgenes 0- 4-hr-old young adult fly heads. <br />   b <br />     Pigment values expressed mean absorbance 480 nm &#226;&#171;&#190; standard deviation values obtained n assays. <br />   c <br />     effect modifier pigment levels determined performing t-test. P value represents significance difference pigment value modifier 95% confidence. Sets modifier genotypes grouped control genotype. <br />  <br />  <br />  <br /> 1991), expression bw Prat:bw               Prat:bw transgene array 13 P elements <br /> trol 5&#226;&#172;&#732; 3&#226;&#172;&#732; regulatory sequences Prat                (Clark et al. 1998). transgene arrays <br /> gene. Second, P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] transgene, bw                   exhibit gene silencing (Peng coding region original exon-intron organization               Mount 1995), Prat:bw transgene arrays si- bw locus, sequences removed               lenced copy-number-dependent expression Prat:bw construct. difference               (Clark et al. 1998). absence gene silencing P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st stock single transposon insertion                likely presence sufficient genomic se- <br />                                               Trans-modifiers Prat Expression                                         1423 <br />  <br /> quence flanking Prat:bw, including flanking gene             Mod-2-3 Mod-3-4. Mod-3-4 enhances eye color sequences (Clark et al. 1998).  predicted           Prat:bw suppresses P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] (Table 1), modifiers Prat:bw acting level transcrip-      Mod-2-3 suppresses Prat:bw eye color enhances <br /> tion acting modification gene            P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] (Table 1). Mod-3-4 identified dur- <br /> silencing transgene array.           ing screen suppressor Prat:bw eye color intriguing exceptions  , finding              young adult flies, pigment measurements modifiers affect endogenous Prat gene&#226;&#8364;&#8482;s            inverse effect Prat:bw eye color aged flies. <br /> mRNA levels   support idea.                  summary, visual scoring Prat:bw eye-color <br /> difference transgenes insertion site;       modification young adult flies pigment mea-  P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] Prat:bw transgene           surements 5 days correlate 19 modifiers: 13 position sensitive (Dreesen et al. 1991;         class 1 modifiers isolated strong suppressors <br /> Clark et al. 1998).                                              Prat:bw eye color, 2 class 2 modifiers strong <br />    results pigment assays summarized           enhancers, 4 6 class 3 modifiers weak sup- <br /> Table 1. technique, performed 7              pressors. existence 4 modifiers  10 assays modifier genetic             way illustrates important initially <br /> grounds (Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;]). Assays           screen flies age subsequent <br /> males females modifiers second                analysis pigment levels ages reveal mod- chromosomes  results fe-            ifiers dynamic effects level eye pigment. <br /> males shown), females X chromosome,                  RT-PCR analysis shows subset modifiers males Y chromosome. controls,              affects Prat expression: developed quantitative RT- <br /> used flies derived transgene stocks crossed            PCR assay determine effects modifiers parental bw D; st strain used EMS screen.        gene expression mRNA level. technique calculated mean standard deviation             measured level mRNAs various genetic <br /> pigment value genotype. addition, P              backgrounds: (1) endogenous Prat mRNA lines <br /> values t-tests calculated comparisons            containing transgenes, (2) bw mRNA produced <br /> tween presence absence modifier            Prat:bw transgene, (3) bw mRNA pro- genetic background.                                         duced P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] transgene. addition, rp49 <br />    screen, 23 modifiers fell           mRNA (O&#226;&#8364;&#8482;Connell Rosbash 1984) coampli- <br /> classes eye color observed 0 4              fied mRNAs provide hr eclosion, modifiers fell            internal control (Foley et al. 1993). levels classes tested pigment values 5 days: class          mRNAs measured presence absence <br /> 1 mutations suppress Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] pig-           modifier chromosome adult female fly heads <br /> ment levels (13 modifiers), class 2 mutations enhance            aged 0&#226;&#8364;&#8220;4 hr. exception Y-linked modifier, Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] pigment levels (2 modifi-           Mod-Y-1, males assayed. <br /> ers), class 3 mutations affect Prat:bw eye color 0-           chose investigate effect modifier <br /> 4-hr-old adult flies effect Prat:bw       Prat expression young adult fly heads P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] pigment levels aging flies         reasons.  wanted examine gene expression <br /> 5 days (6 modifiers), class 4 mutations enhance          stage used isolation dominant <br /> transgene suppress (2 modifiers).                  modifiers Prat:bw. Second, wanted determine <br />    comparing pigment levels flies carrying         selection Prat:bw eye-color variants lead modifier  P values t-tests      isolation trans-modifiers Prat expression highly significant (P &#226;&#172;? 0.002) 13 modifiers          mutations involved pigment pathway suppress transgenes (class 1),            post-transcriptional regulation. <br /> enhance transgenes (class 2), class 4              bw D allele null allele 2-Mbp heterochro- <br /> modifiers (Table 1). modifer, Mod-3-6, showed          matic insertion locus (Henikoff et al. 1995). stronger effect Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] (Table          bw primers used RT-PCR correspond <br /> 1), placed class 1.                                  sequences 5&#226;&#172;&#732; heterochromatic <br />    <span id='am-11' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, class 3 modifiers          insertion, possible detecting bw effect pigment level bw D; Prat:bw st        transcript bw D locus RT-PCR experi- effect bw D; P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] st (Table       ments. bw transcript detected RT-PCR <br /> 1). explanation class 3  screen,      homozygous bw D; st adult fly heads, rp49 <br /> mutations suppress enhance Prat:bw eye color         mRNA (data shown), indicating mRNAs selected flies young  0 4             detected Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] trans- <br /> hr)  pigment test, flies aged 5            genes derived transgenes. <br /> days prior analysis aging suppresses modi-          determining number PCR cycles quanti- <br /> fication Prat:bw eye color.                                   fication bw mRNA (Figure 1),    Class 4 composed exceptional modifiers,             bw mRNA abundant Prat:bw P[ry&#226;&#171;&#185;; <br />  1424                                           N. Malmanche D. V. Clark <br />  <br />  <br />  <br />  <br />   Figure 1.&#226;&#8364;&#8221;Determination PCR conditions quantitative RT-PCR assays. graphs yield Prat rp49 mRNAs genotypes rounds PCR. yield bw mRNA corresponds transgene genotype. RT-PCR assays, number PCR cycles primer pair chosen linear range amplification, assays comparable. values y-axis represent band density. <br />  <br />  <br />  <br />  <br /> bw&#226;&#171;&#185;] flies, contradiction level pteri-     (5 suppressors 2 enhancers) strong effect <br /> dine pigments measured assay.           Prat expression genetic backgrounds finding inconsistent, pigment assay measures           correlates original eye-color observation protein activity process pigment deposi-           pigment assay data. 5 class 1 modifiers <br /> tion 5 days specific cells, RT-PCR             (Mod-3-5, Mod-3-6, Mod-2-1, Mod-2-2, Mod-1-5) measures mRNA levels time adult fly heads.             2 class 2 modifiers (Mod-3-1 Mod-2-5). Mod-3-6, explanation Prat expression adult fly             RT-PCR data strong decrease expres- <br /> heads occurs cells bw expression does oc-           sion Prat endogenous gene genetic  <br /> cur. alternative explanation tran-        grounds (Table 2 Figure 2A). suppres- <br /> scripts derived Prat:bw transgene array         sion bw mRNA transgenes;  functional.                                                      suppression Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;].    Overall, results RT-PCR analysis           chromosome 2 complementation group  Genetic <br /> despite inconsistencies pigment assay           mapping dominant modifiers associated recessive <br /> RT-PCR data, original visual identification mod-          phenotype), Mod-2-1 Mod-2-2, modifiers <br /> ifiers Prat:bw eye color effective recovery         Mod-3-5 Mod-1-5, RT-PCR results showed  <br /> trans-dominant modifiers Prat gene expression. Spe-           duction bw mRNA transgenes consis- <br /> cifically, RT-PCR data indicate 7/23 modifiers          tent decrease pigment level shown                                                Trans-modifiers Prat Expression                                             1425 <br />  <br />                                                              TABLE 2 <br />            Effect modifiers Prat, Prat:bw P[ry ; bw&#226;&#180;&#8482;] mRNA expression 0- 4-hr-old adult fly heads <br />                                                          &#226;&#180;&#8482; <br />  <br />  <br />  <br />                                                                  Transgene <br />                                          Prat:bw                                             P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] <br />                       Pigment                                               Pigment <br />         Modifier       effecta      RT-PCR Prat b      RT-PCR bw b           effecta     RT-PCR Prat b      RT-PCR bw b <br />         Mod-1-1          ss          2.1; 2.3           2; 2.2                 ss         1.4; 3            2.1; 2.3 <br />         Mod-1-2          ss          1; 1.4; 1.8        1.5; 1.5               ne         2.3; 2.5          1.6; 3 <br />         Mod-1-3          ss          0.6; 0.8; 0.9      0.7; 0.8               ws         0.6; 0.9          0.2; 0.4 <br />         Mod-1-4          ss          1.6; 3             1.4; 1.5               ss         2; 4              1.4; 3 <br />         Mod-1-5          ss          0.1; 0.2           0.1; 0.2               ss         0.1; 0.2          0.2; 0.2 <br />         Mod-2-1          ss          0.3; 0.5           0.3; 0.5               ss         0.4; 0.4          0.1; 0.1 <br />         Mod-2-2          ss          0.5; 0.7           0.5; 0.6               ss         0.1; 0.1          0.1; 0.2 <br />         Mod-2-3          ws          0.3; 0.7c          0.1; 0.4c              ne         0.1; 0.3c         0.2c; 0.4 <br />         Mod-2-4          ss          0.7; 1.3           0.1; 0.3               ne         0.9; 1            0.1; 0.1 <br />         Mod-2-5          se          1.3; 1.4; 1.5      5; 6.7                         1.5; 1.7          1.5; 2 <br />         Mod-3-1          se          1.5; 1.5           1.8; 2                 ne         1.5; 1.5          1; 1.1 <br />         Mod-3-2          ws          0.2; 0.3           0.9; 1.4               ne         0.2; 0.4; 0.6     0.3; 0.8 <br />         Mod-3-3          ws          0.6; 1.4           1.2; 1.4               ws         0.8; 1.1; 1.3     0.3; 0.3; 0.6 <br />         Mod-3-4          ws          0.2; 0.7           1; 1.3; 2.4            ss         2.5; 2.7; 2.8     0.7; 1.2 <br />         Mod-3-5          ss          0.1; 0.3           0.2; 0.3               ne         0.4; 0.5          0.1; 0.3 <br />         Mod-3-6          ws          0.1; 0.2c          0.1c; 0.3              ne         0.2; 0.5c         0.3; 0.5c <br />         Mod-3-7          ss          1.3; 2.4; 2.7      0.7; 0.7               ss         3.1; 3.2          4.5; 5 <br />         Mod-3-8          ss          0.3; 0.4           0.7; 0.7               ne         0.1; 0.3          0.2; 0.4 <br />         Mod-3-9          ws          0.3; 0.5; 0.7      3; 5                   ne         2; 2.9; 3         1.2; 1.4 <br />         Mod-3-10         ws          0.8; 1; 1.3        2; 2.7                 ne         1.3; 1.3; 1.4     1; 1.2 <br />         Mod-3-11         ss          0.3; 0.4           0.8; 0.8               ss         1.6; 1.8          4.5; 5 <br />         Mod-3-12         ws          0.2; 0.3           0.2; 0.3               ne         0.8; 1            0.8; 1 <br />         Mod-Y-1          ss          1.6c; 1.9          1.2; 1.5c              ss         1.6; 1.8c         3.2; 3.5c <br />           ss, strong suppressor; ne, effect; ws, weak suppressor;  weak enhancer; se, strong enhancer. <br />                       effect modifiers eye color observed transgenes 0- 4-hr-old young adult fly <br />         heads. <br />           b <br />             RT-PCR values expressed ratio level gene x mRNA modifier <br />         relative rp49 mRNA.  rp49 coamplified mRNA level used normalize <br />         yield mRNA detected gene x. Gene x endogenous Prat mRNA different genotypes         gene x bw mRNAs derived Prat:bw bw&#226;&#171;&#185; transgene. <br />           c <br />             Indicates relative values obtained experiments shown Figure 2. <br />  <br />  <br />  <br />  <br /> pigment assays.  suppression affects Prat                 Prat:bw eye color, contradiction pigment <br /> mRNA expression genetic backgrounds.                      assay data (Tables 1 2 Figure 2A). strong <br /> class 2 modifiers Mod-2-5 Mod-3-1 enhance Prat                    effects Prat expression correlation mRNA expression genetic backgrounds                  Prat:bw phenotype suggest modifiers cor- <br /> effect correlates enhancement pigment expres-                 respond genes encoding trans-acting factors involved <br /> sion.  effect Prat mRNA similar             Prat regulation expression. genetic backgrounds, effect bw mRNA                    12 remaining modifiers, 5 enhancers varies, stronger enhancement Prat:bw                Prat expression genetic backgrounds (Mod-1-1, <br /> P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] Mod-2-5 (Table 2)  Mod-3-1,                 Mod-1-2, Mod-1-4, Mod-3-7, Mod-Y-1). interesting effect bw mRNA P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] (Table                   note 5 modifiers strong suppressors 2). spite inconsistencies, overall 5           Prat:bw eye color 0- 4-hr-old adults sup- <br /> class 1 2 class 2 modifiers appear acting                   pressors Prat:bw P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] measured common mechanism Prat bw mRNA                       pigment assays.  enhancement bw <br /> expression similar effects types              mRNA expression transgenes  Mod- <br /> construct promoter.                                               3-7 Prat:bw) suggests post-transcriptional effect asso- <br />    remaining modifiers, (Mod-2-3, Mod-                 ciated modifier chromosomes. <br /> 3-2, Mod-3-8, Mod-3-12) strong suppression                      modifiers display opposite effect Prat Prat expression correlation young adult                expression function genetic background <br />  1426                                          N. Malmanche D. V. Clark <br />  <br />  <br />  <br />  <br />                                                                             Figure 2.&#226;&#8364;&#8221;Quantitative RT-PCR showing   <br />                                                                          effect Mod-3-6 Mod-2-3 (B)                                                                          effect Mod-Y-1 endogenous Prat expression <br />                                                                          bw expression transgenes relative <br />                                                                          level expression rp49. relative <br />                                                                          values obtained experiments foot- <br />                                                                          noted c Table 2. <br />  <br />  <br />  <br />  <br /> (Mod-3-4, Mod-3-9, Mod-3-11). suppressors        ity modifiers gene expression Prat expression Prat:bw genetic background          modifiers PEV, examined effects enhancers P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] genetic background.          bw D/bw&#226;&#171;&#185; variegation phenotype. Variegated expression modifiers, RT-PCR results        wild-type copy bw gene occur trans <br /> bw transgenes correlate initial isolation   bw D allele. inactivation bw thought suppressors Prat:bw eye color.                          mislocalization bw&#226;&#171;&#185; allele    modifiers modify Prat expression.          heterochromatic compartment nucleus (Csink <br /> Mod-1-3, weak suppression Prat mRNA         Henikoff 1996). tests facilitated genetic backgrounds. modifiers Mod-2-4           fact modifiers isolated homozygous <br /> Mod-3-3 reduce mRNA levels transgenes           bw D background. cross modifier stock bw&#226;&#171;&#185;; affecting Prat mRNA levels (Table 2). Mod-3-10 does       st stock allowed visualization bw D/bw&#226;&#171;&#185; varie- affect Prat significantly genetic background    gating pigment phenotype eye allowed shows enhancement bw mRNA Prat:bw.          assessment dominant effect modifier specific effect modifiers bw trans-     model variegation. <br /> gene mRNA expression pigment level suggests             23 modifiers, 4 suppress bw D/bw&#226;&#171;&#185; variega- act common sequences              tion (Mod-Y-1, Mod-2-5, Mod-3-10, Mod-3-12) 3 <br /> structs, bw coding region,         enhance bw D/bw&#226;&#171;&#185; variegation (Mod-1-5, Mod-3-4, ry&#226;&#171;&#185; marker present transgenes.                        Mod-3-5; Figure 3). experiment, note    subset Prat:bw modifiers modifies position-      suppression bw D/bw&#226;&#171;&#185; phenotype indicates  <br /> effect variegation: Drosophila, trans-acting dos-     hancement bw expression enhancement age-sensitive modifiers gene expression mod-      phenotype indicates suppression bw expression. <br /> ifiers position-effect variegation (PEV; Bhadra           Mod-Y-1 suppresses pigment levels trans- <br /> Birchler 1996; Bhadra et al. 1998). findings            genes, enhances mRNA levels transgenes <br /> reflect relationship regulation gene       endogenous Prat gene (Tables 1 2 expression chromatin structure. test possibil-     Figure 2B). discordance observa- <br />                                               Trans-modifiers Prat Expression                                           1427 <br />  <br />  <br />  <br />  <br />                                                                                         Figure 3.&#226;&#8364;&#8221;Effect modifiers <br />                                                                                      bw D/&#226;&#171;&#185; variegation. phenotypes <br />                                                                                      shown genotypes   bw D/&#226;&#171;&#185;; <br />                                                                                      st, (B) bw D Mod-2-5/&#226;&#171;&#185;; st, (C) bw D Mod- <br />                                                                                      2-5r110/&#226;&#171;&#185;; st, (D) bw D/&#226;&#171;&#185;; Mod-3-4 st/&#226;&#171;&#185; st, <br />                                                                                      (E) bw D/&#226;&#171;&#185;; Mod-3-5 st/&#226;&#171;&#185; st, (F) bw D/&#226;&#171;&#185;; <br />                                                                                      Mod-3-10 st/&#226;&#171;&#185; st, (G) bw D/&#226;&#171;&#185;; Mod-3-12 <br />                                                                                      st/&#226;&#171;&#185; st, (H) Mod-1-5/&#226;&#171;&#185;; bw D/&#226;&#171;&#185;; st. <br />  <br />  <br />  <br />  <br /> tions explained difference sampling           color recessive lethal phenotype  , pigment mRNA.  Y-linked mod-               suppression variegation (Figure 3C), indicating <br /> ifier, alternative explanation inconsistency          effect linked modifier Prat:bw pigment suppression mRNA enhance-                    expression. <br /> ment, involving effect gene,              Mod-3-4, Mod-3-5, Mod-1-5 enhanced bw D/bw&#226;&#171;&#185; invoked. interested determining            phenotype (Figure 3, D, E, H, respectively). Mod-Y-1 general effect PEV, investigated          finding consistent suppression bw expres- effect variegating alleles: bw D, wm4, Sbv.     sion observed genetic backgrounds Mod-3-5 <br /> Suppression variegating phenotype observed            Mod-1-5, Mod-3-4 did affect bw mRNA levels. alleles (Figure 4). finding correlates         behavior Mod-3-4 enigmatic, enhancement mRNA levels observed             correlation observed effects transgenes Prat. results indi-         Mod-3-5 Mod-1-5 expression transgenes, <br /> cate Mod-Y-1 enhancer gene expression.             Prat, bw D/bw&#226;&#171;&#185; variegation. <br /> Changes Y chromosome dosage shown                   Mod-3-10 Mod-3-12 strong suppression modify PEV, extra Y chromosome suppress             bw D/bw&#226;&#171;&#185; variegated phenotype (Figure 3, F G, <br /> PEV (Reuter Spierer 1992).  possible              respectively) isolated weak suppres- Mod-Y-1 chromosome rearrangement mul-             sors Prat:bw eye color. Mod-3-10 Mod-3-12 <br /> tiple effects gene expression. Cytological examina-           modify bw expression P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;], Mod-3-10 tion larval neuroblast mitotic chromosomes showed             enhancer Prat:bw mRNA expression Mod-3-12 detectable rearrangement associated Mod-             suppressor Prat:bw mRNA expression. <br /> Y-1 chromosome. addition, Mod-Y-1 chromosome                 Genetic mapping dominant modifiers associated elevated X-Y meiotic nondisjunc-         associated recessive phenotype: Sixteen modifiers tion, did display recessive bobbed phenotypes         associated recessive lethal phenotype, 4 X-linked complementation tests X-linked recessive bobbed          modifiers recessive lethal phenotype, 1 2-linked <br /> alleles (D. Clark, unpublished observations).  al-          suppressor (Mod-2-4) recessive male sterile pheno- Mod-Y-1 behaves general suppressor PEV            type, 1 3-linked suppressor (Mod-3-9) recessive female enhancer gene expression, associ-         sterile phenotype, 1 linked Y chromosome. <br /> ated obvious chromosomal rearrangement               Inter se crosses performed 16 recessive lethal <br /> volving rDNA autosomes.                               modifiers determine number complementa- <br />    Mod-2-5 shows strong suppression bw D/bw&#226;&#171;&#185; var-         tion groups. modifiers fell complementa- <br /> iegated phenotype (Figure 3B). addition,             tion group second chromosome 5 fell combinant chromosomes obtained recombina-                 complex complementation group chro- <br /> tion mapping dominant modifier Prat:bw eye             mosome: subgroup consists Mod-3-1, Mod-3-2, <br />  1428                                             N. Malmanche D. V. Clark <br />  <br />                                                                   complementing alleles (Mod-3-1 Mod-3-2 <br />                                                                   Mod-3-5) similar effect visual isola- <br />                                                                   tion phenotypes, pigment assay, RT-PCR results, <br />                                                                   effects intercomplementing al- <br />                                                                   leles (Mod-3-3 Mod-3-4) relation                                                                   visual isolation phenotype pigment assay data. <br />                                                                   situation suggests second-site muta- <br />                                                                   tions Mod-3-3 Mod-3-4 chromosomes                                                                   affect result obtained alleles. <br />                                                                      Modifiers mapped effect Prat <br />                                                                   mRNA easily scored Prat:bw eye color. focused <br />                                                                   Mod-2-3, Mod-2-5, Mod-3-6, complementa- <br />                                                                   tion groups&#226;&#8364;&#8221;Mod-2-1 Mod-2-2 second chro- <br />                                                                   mosome Mod-3-1, Mod-3-2, Mod-3-3, Mod-3-4,                                                                   Mod-3-5 chromosome determine                                                                   alleles colocalize genetic map. domi- <br />                                                                   nant modifier phenotype mapped recombination <br />                                                                   recessive lethal phenotype recombination <br />                                                                   deficiency mapped. Mapping results ap- <br />                                                                   proaches allowed determine                                                                   second-site lethal mutations modifier chromo- <br />                                                                    cases, recombinant viable dominant <br />                                                                   modifier chromosomes retained allow analysis <br />                                                                   recessive phenotype modifier ab- <br />                                                                   sence second-site mutations. <br />                                                                      Mod-3-6 linked h left arm                                                                   chromosome (data shown). recombinant <br />                                                                   chromosomes displayed dominant modifier pheno- <br />                                                                   type recessive lethal phenotype, indicating <br />                                                                   recessive lethal mutations present <br />                                                                   modifier chromosome associated    Figure 4.&#226;&#8364;&#8221;Effect Mod-Y-1 position-effect variegation.    modifier gene. homozygous viable recombinant phenotypes shown following males:   bw D/&#226;&#171;&#185;;     modifier chromosomes isolated mapping st/st, (B) Mod-Y-1 ; bw D/&#226;&#171;&#185;; st/st, (C) wm4/Y, (D) wm4/Mod-   obvious recessive phenotype. Using deficienc- <br /> Y-1. (E) Box plot showing effect Mod-Y-1 Sbv variega- <br /> tion. degree suppression Sbv measured scoring <br />                                                                   ies chromosome, original Mod-3-6 chro- <br /> 14 thoracic macrochaetae presence stubble          mosome complements deficiencies uncovering h <br /> phenotype (Csink et al. 1994). higher proportion       region visible phenotype;  fails <br /> stubble macrochaetae, suppression       complement deficiencies arms Sbv allele. &#226;&#8364;&#339;y&#226;&#171;&#185;&#226;&#8364;? males (n &#226;&#171;&#189; 64) females (n &#226;&#171;&#189; 74)         chromosome [Df (3L)emc-E12, Df(3R)Antp17, Df(3R)p- non-Cy flies derived cross &#226;&#171;&#185;; bw D; st <br /> parental strain males Sbv/CyO females. Likewise, &#226;&#8364;&#339;y mod&#226;&#8364;? <br />                                                                   XT103], indicating second-site mutations chro- <br /> males (n &#226;&#171;&#189; 58) females (n &#226;&#171;&#189; 87) derived           mosome. <br /> cross Mod-Y-1 ; bw D; st males Sbv/CyO females.          second-chromosome complementation group <br />                                                                   consisting Mod-2-1 Mod-2-2, modifiers mapped <br />                                                                   cn (42D) c (52D) right arm                                                                   chromosome 2. modifiers fail complement Df Mod-3-3, second subgroup Mod-3-3 Mod-                (2R)stan1 (breakpoint: 46D7-9; 47F15-16), deficiency <br /> 3-4, subgroup Mod-3-4 Mod-3-5.           localized cn c, modifiers comple- <br /> 5 modifiers represent single gene               ment overlapping deficiencies, Df(2R)X1 (breakpoint: <br /> complementing alleles second-site               46C; 47A1) Df(2R (breakpoint: 47D3; 48B02), <br /> mutations associated modifier chromo-            suggesting modifier alleles present somes. remaining 9 modifiers did fall              cytological region 47A1-D3. <br /> complementation group.                                               modifiers chromosome complex <br />   second chromosome complementation                 complementation group, Mod-3-1, Mod-3-2, Mod-3-3, <br /> group alleles similar effect Prat bw           Mod-3-4, Mod-3-5, mapped markers th <br /> expression, correlates initial isolation           sr. cu sr Mod recombinant chromosome pigment assay data, chromosome com-             obtained, indicating presence modifiers plementation group presents complex result.                cu region right arm chromosome.                                            Trans-modifiers Prat Expression                                          1429 <br />  <br />                                                                                         Figure 5.&#226;&#8364;&#8221;Second-chromosome <br />                                                                                      deficiency mapping recessive <br />                                                                                      lethality associated Mod-2-3.                                                                                      cytological extents noncomple- <br />                                                                                      menting deficiencies shown ho- <br />                                                                                      rizontal lines cytological <br />                                                                                      division(s). lines represent <br />                                                                                      deficiencies fail complement <br />                                                                                      modifier chromosome.                                                                                      lines represent deficiencies fail <br />                                                                                      complement modifier chromo- <br />                                                                                      colocalize  <br />                                                                                      combination mapping domi- <br />                                                                                      nant modifier Prat:bw. <br />  <br />  <br />  <br />  members complementation group failed           chromosomes complement Df(2R)Px1 (breakpoint: <br /> complement deficiency localized right cu,         60B8-10; 60D1-02), indicating modifier Tp(3,Y)ry 506-85C (breakpoint: 87D1-2; 88E5-6).          Prat:bw wing phenotype localized result consistent idea members      region 60D right bw. group alleles gene. <br />    Mod-2-3 maps Bl (38B) cn (42D) <br />                                                                                    DISCUSSION <br /> markers, flank centromere. Mod-2-3 dominant effect generation time, delay 24           Prat gene encodes enzyme performs hr 25&#226;&#172;&#353;. mapping experiment, ex-            limiting step purine novo biosynthesis <br /> tended generation time recessive lethal pheno-          pathway. Characterization genes encoding  <br /> type comapped modifier Prat:bw (data          zymes pathway prokaryotic <br /> shown). Deficiency mapping showed Mod-2-3            eukaryotic species. studies chromosome uncovers 15 lethal effects distributed        addressed regulation pathway enzy- arms chromosome (Figure 5).           matic level, little information genetic <br /> deficiencies, Df(2L)Tw50, Df(2L)Tw161, Df(2R)            regulation pathway available multicellular  <br /> M41A4, 2R)bwVde2LCyR located               ganisms. D. melanogaster Prat gene previously Bl cn markers.  remains possible         characterized molecular genetic levels dominant effect generation time           (Clark 1994) Prat:bw reporter gene devel- <br /> recessive lethal phenotype independent Prat:       oped changed eye color white pale <br /> bw dominant effect linked non-        orange bw D;st genetic background (Clark et al. <br /> complementing deficiencies present region.        1998). <br /> high number interactions modifier chromo-              alteration eye color reflects Prat expression deficiencies used mapping         prepupal pupal stages early <br /> problematic mapping resolution modifier         adult development pigment deposition occurs Prat:bw. explanation observation      Drosophila (Steller Pirotta 1985; Dreesen et al. deficiencies, failing comple-      1988). Prat developmental profile mRNA expres- <br /> ment recessive lethal second-site mutations Mod-       sion characterized (Clark et al. 1998; Clark <br /> 2-3 chromosome locations, dominant       MacAfee 2000; Malmanche et al. 2003). Prat ex- <br /> synthetic effects delayed generation-time phe-       pression occurs development variation <br /> notype associated modifier chromosome.               expression levels. lowest level occurs    Mod-2-5 shows strong enhancement Prat:bw eye          second larval stage expression abundant <br /> color maps near bw (59E) end right         females males, corresponds Prat <br /> arm second chromosome. recombinant           expression oocytes accumulation young <br /> chromosomes carrying modifier, visible         embryo. Considering complexity Prat expression <br /> recessive phenotype includes modification       development lack molecular data pu- <br /> wing shape (Figure 6). phenotype relation-        rine gene regulation multicellular organisms, <br /> ship wing phenotype seen purine syn-          chose genetic approach identify factors drome (Clark 1994). Subsequently, deficiency map-             role Prat gene expression.  reported ping original modifier chromosome                isolation characterization trans-acting dominant <br /> recombinant chromosome (r110) showed                modifiers Prat:bw reporter gene. <br /> chromosomes displayed wing phenotype (Figure 6)                nonsaturated Prat:bw orange eye color allowed Df(2R)Px2 (breakpoint: 60C5-6; 60D9-10)         screen dominant enhancers suppres- <br />  1430                                         N. Malmanche D. V. Clark <br />  <br />  <br />  <br />  <br />                                                                                          Figure 6.&#226;&#8364;&#8221;Mod-2-5 Mod- <br />                                                                                       2-5 r110 wing phenotype. pheno- <br />                                                                                       types shown following <br />                                                                                       genotypes:  B) Mod-2-5 r110 <br />                                                                                       homozygotes showing range <br />                                                                                       effect wing shape, (C) Mod- <br />                                                                                       2-5r110/Df(2R)Px2, (D) Mod-2-5/ <br />                                                                                       Df(2R)Px2. <br />  <br />  <br />  <br />  <br /> sors Prat:bw eye color. EMS mutagenesis screen      dines synthesized purine precursors dominant modifiers Prat:bw orange eye color,      supplied purine novo synthesis, salvage, isolated enhancers 21 suppressors, 16          interconversion pathways. issue single alleles. Following screen, quanti-   isolated modifiers affect purine pteri- <br /> fied specificity dominant modifiers     dine synthesis pathways, affecting Prat:bw bw expres- <br /> effect Prat:bw Prat expression using pigment          sion indirectly.  recessive mutations Prat assays 5-day-old adult fly heads quantitative RT-     genes involved <span id='am-19' about='obo:IMR_0200292' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="rdfs:label" content="GMP" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200292" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00144" datatype="xsd:string"></span>GMP</span> synthesis reduced <br /> PCR assays 0- 4-hr-old adult-fly-head mRNA.         pteridine eye pigment, pleiotropic pheno- investigated effect chromatin structure     type affecting aspects development called using dominant phenotype associated bw D        purine syndrome.  Prat:bw modifi- <br /> allele.                                                      ers dominant phenotype reminiscent purine <br />    advantage using eye-color reporter gene         syndrome. screen conducted wild-type ge- Drosophila provides simple sensitive     netic background purine pigment pathway <br /> assay gene expression, F1 flies easily   genes, dominant gain function mutation affect- <br /> screened dissecting microscope eye-color         ing pathways isolated. <br /> variants immediately mated need        Presumably, mutation domi- <br /> establish stocks carrying mutagenized chromo-           nant effects phenotype. issue addressed  Eye-color reporter genes used         stage design screen, assess past identify genes involved heterochromatin-          affect modifiers endogenous Prat gene expression <br /> mediated gene silencing (Weiler Wakimoto 1995)           second stage, allowing distinguish intron splicing (Peng Mount 1995),            modifiers effect Prat expression example. difficult assess      effect eye-pigment synthesis genes involved process identified,    deposition. <br /> modifier genes expressed eye development,            considering effect modifiers Prat:bw <br /> screen provides way identify subset        measured pigment assay Prat bw <br /> candidate regulatory genes, starting         measured RT-PCR, important remember <br /> point analysis genetic regulation      pigment synthesis deposition process oc- <br /> Prat purine genes.                                 curs developmental stages prepupa <br />    use bw reporter gene screen raises        adult (Steller Pirotta 1985; Dreesen et al. issue interrelationship purine      1988). modifier affect temporal dynamics eye-pigment pathways. brown protein&#226;&#8364;&#8482;s func-          process differently process Prat:bw <br /> tion eye process pteridine pigment. Pteri-      mRNA accumulation young adults. factor <br />                                               Trans-modifiers Prat Expression                                        1431 <br />  contribute inconsistencies pig-            cussed  Prat likely expressed ment mRNA measurements pigment                  developing eye, evidence coordinated <br /> P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;] expression reflects expression specific         regulation purine novo synthesis pteridine <br /> cell type, mRNA Prat:bw expression likely           transport pathways. relation study, reflects expression wider range cell types        follow pattern bw expression adult head. example, modifiers Prat:bw eye           transgenes comparison wild-type bw Prat <br /> color did affect Prat expression, indicating       expression prepupa adult overview of modifiers isolated screen involved         regulation transgenes different develop- post-translational processes pigment synthesis        mental stages. analysis provide transport regulation Prat           basis extend analysis effect expression.                                                      modifiers different times process    23 dominant modifiers isolated              eye-pigment deposition. <br /> screen, 7 modifiers correlation ob-              regulation gene expression involves  <br /> served Prat:bw eye color, pigment assay, RT-         cruitment trans-acting factors chromatin PCR results. additional modifiers correla-           formation transcriptional complex. recruit- <br /> tion observed Prat:bw eye color Prat         ment trans-acting factors related chromatin struc- bw RT-PCR results Prat:bw genetic background.           ture local nuclear environment region.  consider 11 modifiers Prat:bw             Analysis behavior white modifiers described <br /> eye color correspond trans-acting factors involved         shown mechanisms gene expres- Prat expression. remaining 12 modifiers,            sion chromatin organization exclusive pro- group 5 class 1 modifiers shows enhancement              cesses. examine functional link gene <br /> bw Prat gene expression genetic                 expression chromatin organization respect grounds isolated suppressors            trans-acting dominant modifiers, tested Prat:bw eye color 0- 4-hr-old adult flies      effect bw D/bw&#226;&#171;&#185; variegation. suppressors pigment level 5 days.              modifiers affected bw D/bw&#226;&#171;&#185; variegation.  19 modifiers effect Prat            Mod-2-5 suppressed bw D/bw&#226;&#171;&#185; phenotype,  <br /> expression young adult fly heads, indicating         combinant chromosome showed effect, indi- <br /> use Prat:bw transgene array recover trans-domi-        cating dominant modifier Prat:bw nant modifiers Prat expression successful.                modification trans-dominant variegation linked. <br />    Prat:bw dominant modifiers           addition, deficiency mapping indicated effect P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;], raising possibility        original modifier chromosome recombinant <br /> mutations effect global aspect              chromosome possess recessive mutation Df(2R) <br /> gene expression. yeast, transcription factors           Px2 region wing-vein phenotype. various <br /> play role expression purine synthesis genes.         properties Mod-2-5 useful characteriza- <br /> example, GCN4, controls expression                 tion genetic molecular levels. <br /> <span id='am-12' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> biosynthesis genes, upregulates purine gene              complex complementation group alleles <br /> expression response purine starvation (Mo&#194;&#168;sch et           chromosome, members, <br /> al. 1991; Rolfes Hinnebusch 1993). humans,                Mod-3-4 Mod-3-5, enhance bw D/bw&#226;&#171;&#185; phenotype transcription factor NRF1, controls expres-             Mod-3-1, Mod-3-2, Mod-3-3, <br /> sion cytochrome genes genes involved             obvious modification bw D/bw&#226;&#171;&#185; pheno- <br /> respiration, plays role expression bidirec-          type. Genetic mapping linked dominant mod- <br /> tional promoter PRAT-AIRC locus (Chen et al.              ifiers deficiency, suggesting 1997). likely Drosophila          lesions gene, comple- <br /> factors controlling expression Prat wide-           menting alleles enhancement variegation <br /> ranging effects gene expression.                              does correlate dominant modifica- <br />    Prat:bw modifiers effect            tion Prat:bw. addition, alleles comple- global aspect gene expression acting               mentation group different effects Prat:bw chromatin structure. Multiple effects trans-acting            Prat expression. findings taken ac- <br /> dominant modifiers described               count genetic characterization alleles <br /> modifiers Drosophila white gene (Rabinow et               attempt link dominant effect Prat:bw <br /> al. 1991; Bhadra et al. 1997a,b). white            enhancement bw D/bw&#226;&#171;&#185; phenotype. <br /> modifiers effect chromatin organization,            addition use study Prat regulation demonstrated modification PEV (Csink              itself, Prat:bw transgene array used tool <br /> et al. 1994; Bhadra Birchler 1996; Bhadra et al.             study general phenomenon associated 1998). second possibility Prat bw         variegation transgene arrays (Sabl 1996). classical <br /> transcriptionally coregulated eye,         PEV, phenotype occurs result chromosomal functional relationship genes dis-            rearrangements involving heterochromatin  1432                                                  N. Malmanche D. V. Clark <br />  <br /> followed using eye-pigment gene.  expansion                      expression white eye color locus suppresses posi- <br />                                                                             tion effect variegation Drosophila melanogaster. Genetics 137: transgene multicopy array single site                  1057&#226;&#8364;&#8220;1070. <br /> produce phenotype similar classical heterochroma-                  Chen, S., P. L. Nagy H. Zalkin, 1997 Role NRF-1 bidirec- <br /> tin-induced PEV (Dorer Henikoff 1994).                           tional transcription human GPAT-AIRC purine biosynthe- <br />                                                                             sis locus. Nucleic Acids Res. 25: 1809&#226;&#8364;&#8220;1816. <br /> trast, Prat:bw transgene array does varie-               Clark, D. V., 1994 Molecular genetic analyses Drosophila <br /> gated phenotype polytene chromosomes fly                         Prat, encodes enzyme novo purine biosynthe- <br /> larvae carrying array new unknown struc-                     sis. Genetics 136: 547&#226;&#8364;&#8220;557. <br />                                                                         Clark, D. V., N. MacAfee, 2000 purine biosynthesis  <br /> ture resembling dense puff (Clark et al. 1998).                      zyme PRAT detected proenzyme mature forms cytological observation relevant organiza-                    development Drosophila melanogaster. Insect Biochem. Mol. Biol. <br /> tion Prat:bw transgene array copy-number-                    30: 315&#226;&#8364;&#8220;323. <br />                                                                         Clark, D. V., J. F. Sabl S. Henikoff, 1998 Repetitive arrays <br /> dependent expression.                                                       containing housekeeping gene altered polytene chromo- <br />    did isolate trans-dominant modifiers inducing                     morphology Drosophila. Chromosoma 107: 96&#226;&#8364;&#8220;104. <br /> Prat:bw variegation.  did isolate modifi-                Csink,  K., S. Henikoff, 1996 Genetic modification hetero- <br />                                                                             chromatic association nuclear organization Drosophila. Na- <br /> ers Prat:bw expression effect ob-                       ture 381: 529&#226;&#8364;&#8220;531. <br /> served Prat P[ry&#226;&#171;&#185;; bw&#226;&#171;&#185;].  modifiers             Csink,  K., R. Linsk J.  Birchler, 1994 Lighten suppressors bw D/bw&#226;&#171;&#185; variegation. observations                     (Lip) gene Drosophila melanogaster, modifier retroelement <br /> lead think modifiers involved                        expression, position effect variegation white locus insertion <br />                                                                             alleles. Genetics 138: 153&#226;&#8364;&#8220;163. <br /> chromatin structure control gene                  Daignan-Fornier, B., G. R. Fink, 1992 Coregulation purine <br /> expression. effect modifiers linked                     histidine biosynthesis transcriptional activators BAS1 unusual polytene structure observed                        BAS2. Proc. Natl. Acad. Sci. USA 89: 6746&#226;&#8364;&#8220;6750. <br />                                                                         Dorer, D. R., S. Henikoff, 1994 Expansions transgene  <br /> Prat:bw multicopy array (Clark et al. 1998)                   peats cause heterochromatin formation gene silencing share common mechanism regulation associated                           Drosophila. Cell 77: 993&#226;&#8364;&#8220;1002. Prat:bw insertion site.                                        Dreesen, T. D., D. H. Johnson S. Henikoff, 1988 brown <br />                                                                             protein Drosophila melanogaster similar white protein <br />    conclusion, demonstrated effective-                       components active transport complexes. Mol. Cell. Biol. <br /> ness using Prat:bw transgene array reporter                     8: 5206&#226;&#8364;&#8220;5215. <br /> gene isolation dominant modifiers Prat expres-                Dreesen, T. D., S. Henikoff K. Loughney, 1991 pairing- <br />                                                                             sensitive element mediates trans-inactivation associated <br /> sion. isolated 11 dominant modifiers Prat:bw ex-                      Drosophila brown gene. Genes Dev. 5: 331&#226;&#8364;&#8220;340. <br /> pression display similar effect Prat:bw eye color             FlyBase, 1999 FlyBase database Drosophila genome Prat gene expression young adult fly heads.                       projects community literature. Nucleic Acids Res. 27: 85&#226;&#8364;&#8220;88 <br />                                                                             (http://flybase.bio.indiana.edu). additional 5 modifiers strongly enhance Prat expres-                 Foley, K. P., M. W. Leonard J. D. Engel, 1993 Quantitation <br /> sion contrast suppression Prat:bw eye color.                 RNA using polymerase chain reaction. Trends Genet. 9: continued efforts characterize modifiers,                      380&#226;&#8364;&#8220;385. <br />                                                                         Glesne, D. , F. R. Collart E. Huberman, 1991 Regulation including fine genetic mapping studies                         <span id='am-39' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="IMP" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>IMP</span> dehydrogenase gene expression end products, <span id='am-29' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span> <br /> effects expression genes times devel-                    nucleotides. Mol. Cell. Biol. 11: 5417&#226;&#8364;&#8220;5425. <br /> opment, reveal role regulation Prat                Henikoff, S., T. D. Dreesen, 1989 Trans-inactivation                                                                             Drosophila brown gene: evidence transcriptional repression <br /> expression metabolic pathway genes.                               somatic pairing dependence. Proc. Natl. Acad. Sci. USA 86: <br />   authors thank Amy Csink help analysis Mod-Y-1,            6704&#226;&#8364;&#8220;6708. <br /> Nancy MacAfee technical advice, Andre&#194;&#180; Breton data analysis     Henikoff, S., M.  Keene, K. Fechtel J. W. Fristom, 1986a <br /> advice, Roger Smith help imaging. thank anony-         Gene gene: nested Drosophila genes encode unrelated <br /> mous reviewers helpful comments. research funded         proteins opposite DNA strands. Cell 44: 33&#226;&#8364;&#8220;42. <br />                                                                         Henikoff, S., M.  Keene, J. S. Sloan, J. Bleskan, R. Hards et al., operating grant Canadian Institutes Health Research <br />                                                                             1986b Multiple purine pathway enzyme activities encoded D.V.C.                                                                   single genetic locus Drosophila. Proc. Natl. Acad. Sci. USA <br />                       <br /> </body></html>