<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  Preservation Duplicate Genes Complementary, Degenerative Mutations <br />  <br />  <br />                                                                ABSTRACT <br />                 origin organismal complexity generally thought tightly coupled evolution new <br />              gene functions arising subsequent gene duplication. classical model evolution              duplicate genes, member duplicated pair usually degenerates million years              accumulating deleterious mutations, duplicate retains original function. model <br />              predicts rare occasions, duplicate acquire new adaptive function, resulting              preservation members pair, new function retaining old. <br />               empirical data suggest greater proportion gene duplicates preserved              predicted classical model. present new conceptual framework understanding              evolution duplicate genes help explain conundrum. Focusing regulatory complexity <br />              eukaryotic genes, complementary degenerative mutations different regulatory elements <br />              duplicated genes facilitate preservation duplicates, increasing long-term opportu- <br />              nities evolution new gene functions. duplication-degeneration-complementation (DDC) <br />              model predicts (1) degenerative mutations regulatory elements increase reduce <br />              probability duplicate gene preservation (2) usual mechanism duplicate gene preservation <br />              partitioning ancestral functions evolution new functions. present              examples (including analysis new engrailed gene zebrafish) appear consistent              DDC model, suggest analytical experimental approaches determining              complementary loss gene subfunctions acquisition novel functions likely primary <br />              mechanisms preservation gene duplicates. <br />                  newly duplicated paralog, survival depends outcome race entropic <br />                  decay chance acquisition advantageous regulatory mutation. <br />                                                                                          Sidow (1996, p. 717) <br />                  hand, fix advantageous allele giving slightly different, selectable, function <br />                  original copy. initial fixation provides substantial protection future fixation                  null mutations, allowing additional mutations accumulate refine functional differentiation. <br />                  Alternatively, duplicate locus instead fix null allele, pseudogene. <br />                                                                                                  Walsh (1995, p. 426) <br />                  Duplicated genes persist mutations create new essential protein functions, event                  predicted occur rarely. <br />                                                                              Nadeau Sankoff (1997, p. 1259) <br />                  overall, complex metazoans, major mechanism retention ancient gene duplicates <br />                  appear acquisition novel expression sites developmental genes, <br />                  accompanying opportunity new gene roles underlying progressive extension                  development itself. <br />                                                                                        Cooke et al. (1997, p. 362) <br />  <br />  <br />  <br /> T   genomes organisms contain multiple <br />     copies genes closely related structure function. gene families arise tandem <br />                                                                          events presumed preceded                                                                          origin vertebrates (Ohno 1970; Morizot et al. 1991; <br />                                                                          Lundin 1993; Holland et al. 1994; Amores et al. 1998; <br /> duplications, case HOX, hemoglobin,                     Pe&#194;&#180;busque et al. 1998), brewer&#226;&#8364;&#8482;s yeast (Wolfe keratin clusters animals, polyploidization                Shields 1997; Seoighe Wolfe 1998),                                                                          plant species (Lewis 1979). mechanism pre- <br />                                                                          serves large proportion duplicate genes long <br />   Corresponding author: Allan Force, Department Biology, University   time periods,  unclear. classical model Oregon, Eugene, 97403. E-mail: force@oregon.uoregon.edu            predicts duplicate genes initially fully overlap- <br /> Genetics 151: 1531&#226;&#8364;&#8220;1545 (April 1999) <br />  1532                                                       Force et al. <br />  <br /> ping, redundant functions, copy                   example, tetraploid fish lineages, 30&#226;&#8364;&#8220;75% shield second copy natural selection, gene              duplicate protein-coding genes avoided nonfunc- <br /> dosage critical. deleterious mutations               tionalization time spans order 50 100 <br /> occur frequently beneficial mutations                million yr (Allendorf et al. 1975; Ferris Whitt <br /> (Lynch Walsh 1998), classical model predicts                1979); Xenopus laevis, half duplicate genes common fate duplicate pair             preserved 30 million yr (Bisbee et al. 1977; fixation null allele prevents normal               Graf Kobel 1991; Hughes Hughes 1993); transcription, translation, protein function, e.,          allopolyploidization event maize, annual formation pseudogene duplicate               plant, 72% avoided nonfunctionalization 11 <br /> loci (Haldane 1933; Nei Roychoudhury 1973;                      million yr (Whitkus et al. 1992; Ahn Tanksley <br /> Bailey et al. 1978; Takahata Maruyama 1979;                     1993; White Doebley 1998). fact Li 1980; Watterson 1983). model,                   loci observed lineages appear non- <br /> elucidated Ohno (1970), mechanism                   functional member related tetraploid species permanent preservation duplicate genes                argues idea duplicate genes fixation rare beneficial mutations endowing               retained constraints imposed gene dosage copies novel function, second copy             requirements, enzyme loci investigated. <br /> maintains original function. introductory quo-              highest levels duplicate gene retention <br /> tations illustrate extent model cur-           fish plant lineages incom- <br /> rently central paradigm theory duplicate              plete transition diploid inheritance, similar estimates <br /> gene evolution.                                                     duplicate gene preservation emerged   discuss difficulties ability classical      ancient polyploidization events disomic inher- <br /> model explain preservation gene duplicates                itance clearly reestablished, duplica- evolution propose new model                  tions preceded origin tetrapods (33%; <br /> explain duplicate gene preservation fixation              Nadeau Sankoff 1997). Second, X. laevis, degenerative mutations fixation               tetraploid z30 mya, <span id='am-7' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> substitution pat- <br /> new beneficial mutations.  present ex-              terns consistent action purifying selection <br /> amples, including original data zebrafish               copies duplicated genes (Hughes grailed genes, consistent new model. Finally,           Hughes 1993).  loci avoided non- <br /> suggest series experimental approaches testing             functionalization duplicate copies, new model.                                                      relative paucity null alleles segregating   Problems classical model preservation            extant populations (Ferris Whitt 1977). ob- gene duplicates: simplest model                servations unexpected loci involved ongo- <br /> fate duplicate genes  double-recessive model),               ing degenerative process, suggest possibility rate nonfunctional genes (genes                duplicate loci stabilized populations. make functional protein product) fixed                   attempts explain high populations largely determined random genetic              rate duplicate gene preservation empirical <br /> drift null mutation rate (u), provided prod-            observation.  surviving duplicate loci taxa <br /> uct effective population size u ,0.01.          preserved new gene functions conditions, frequency individuals homozy-              evolve higher rate predicted. gous null duplicate loci negligible, null            aware,  convincing evidence mutations behave essentially neutral alleles.                majority duplicate copies acquired new func- <br /> probability copy nonfunctional              tions did exist ancestral genes z1 2 e22ut, t number generations               (Ferris Whitt 1979). second possible explana- loci functionally diploid              tion long-term effective population sizes respect meiosis (Nei Roychoudhury 1973;                      larger expected, fact large Takahata Maruyama 1979; Li 1980; Watterson                      selection double homozygotes prevents fixa- <br /> 1983). result suggests gene duplicates               tion null alleles locus (Takahata Maru- nonfunctional high probability                yama 1979; Li 1980; Walsh 1995). appears relatively short period time. example, u             account case X. laevis (Hughes Hughes <br /> 1026 generation, mean time nonfunc-                 1993). population size requirements preser- <br /> tionalization order million generations          vation gene duplicates selection double                                                             nulls hundreds millions years prohibi- <br />   general observations involving species derived              tively extreme. possible explanation dis- polyploidization events appear contradict               crepancy theory observation rapid demise gene duplicates predicted classi-            rate gene loss slowed form cal model.  numerous cases, fraction genes          natural selection heterozygous carriers null <br /> preserved higher predicted classic model.            alleles (Bailey et al. 1978; Takahata Maruyama <br />                                                 Preservation Duplicate Genes                                         1533 <br />  <br /> 1979; Li 1980; Hughes Hughes 1993; Clark 1994;               gene specific tissue, cell lineage, developmental <br /> Nowak et al. 1997).                                              stage, individual functional domains poly- <br />    Gene structure duplicate gene preservation:            <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> coding portion gene. <br /> alternative reason failure classical model          model presented outlines degenera- explain fates duplicate loci            tive mutations regulatory subfunctions facilitate <br /> overly simplistic view gene structure. gen-        preservation duplicate genes, absence eral assumption classical model proper-       positive selection beneficial mutations, parti- <br /> ties gene adequately subsumed                tioning repertoire gene expression patterns single function, genes functions,        ancestral alleles. model quite distinct controlled different DNA regulatory           classical model, degenerative mutations <br /> elements  following reviews number              lead gene loss beneficial mutations examples: Piatigorsky Wistow 1991; Hughes                    route gene preservation. <br /> 1994; Kirchhamer et al. 1996; Arnone Davidson <br /> 1997). case point cut locus Drosophila <br /> melanogaster (Jack 1985; Liu et al. 1991; Jack                    GENE PRESERVATION COMPLEMENTARY <br /> DeLotto 1995). Genetic molecular analyses dem-                          DEGENERATIVE MUTATIONS <br />                                                                             (SUBFUNCTIONALIZATION) <br /> onstrate 120-kb region DNA upstream cut promoter drives tissue-specific expression,            Following polyploidization event, genomic redun- spontaneous recessive mutant alleles result            dancies exist levels: duplicate chromosomes, <br /> insertions transposable elements region.        duplicate genes, duplicate regulatory regions driv- <br /> regulatory mutation alleles fall complementa-          ing gene expression. level redundancy subject <br /> tion classes, varying effects tissue-specific expres-    processes mutation random genetic drift, <br /> sion  Malpighian tubules, spiracles, central nervous          lead loss function chromosome loss,  specific portions wing leg imaginal discs,        gene inactivation, loss individual regulatory ele- embryonic adult external sensory organs),             ments. duplicate chromosomes lose different genes, morphology viability. Similar comple-             organism remain viable, chromo- <br /> mentation groups involving regulatory-region muta-               somes complement jointly retaining <br /> tions known developmental genes D.              functional copies genes present original <br /> melanogaster, including cubitus interruptus (Slusarski et        ancestral chromosome. Likewise, duplicate genes lose <br /> al. 1995) Ultrabithorax (Bender et al. 1983).                different regulatory subfunctions, com- <br />    widespread existence complementation classes           plement jointly retaining set eukaryotic gene loci indicates gene expres-          subfunctions present original ancestral gene. sion patterns typically controlled multiple           refer complementary loss duplicate genetic modular independent) regulatory regions             elements degenerative mutation duplication- <br /> sociated distinct protein-coding domains (Arnone            degeneration-complementation (DDC) process. Davidson 1997). explicit assumption            unique feature distinguishes DDC process principles involving complementation al-           classical model degenerative mutations facili- <br /> leles locus extended complementa-          tate hinder preservation duplicate <br /> tion duplicate loci, suggest             functional genes. following discussion, focus <br /> regulatory complexity inherent gene classes              duplications entire chromosomes genomes essential, previously missing, component            tandem gene duplications wish <br /> models evolutionary fate duplicate genes.             exclude complications caused uncertainty justification argument derives sub-        extent original duplication local <br /> stantial evidence showing spatial temporal parti-            homogenization events caused unequal crossing tioning expression patterns gene duplicates          gene conversions (Zhou Li 1996). <br /> wide variety species (Ferris Whitt 1979;                     general DDC model, process dupli- <br /> Hughes Hughes 1993; Ekker et al. 1995; Lee et al.            cate gene evolution occurs phases (Figure 1). <br /> 1996; Raff 1996; Gerhart Kirschner 1997).                 phase  genes experience formally incorporate issue expression pattern             alternative fates, correspond complexity models gene duplication, focus             outcomes classical model.  copy subfunctions affect different gene expres-          incur null mutation coding region, sion domains development. adopt                subsequently drifts fixation, leading gene loss (non- <br /> operational <span id='am-6' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> subfunction specific            functionalization). Nonfunctionalization occur <br /> subset gene&#226;&#8364;&#8482;s function  mutated, estab-           regulatory regions duplicate lishes distinct complementation group, cut           destroyed. Second, copy acquire mutation <br /> example (Liu et al. 1991; Jack DeLotto                 conferring new function, fixed <br /> 1995). subfunction involve expression           positive Darwinian selection (neofunctionaliza- <br />  1534                                                       Force et al. <br />  <br />                                                                                    Figure 1. potential fates duplicate <br />                                                                                    gene pairs multiple regulatory regions. <br />                                                                                    small boxes denote regulatory elements <br />                                                                                    unique functions, large boxes <br />                                                                                    denote transcribed regions. Solid boxes  <br />                                                                                    note intact regions gene, open boxes <br />                                                                                    denote null mutations, triangles denote <br />                                                                                    evolution new function.                                                                                    model focuses mutations fixed popula- <br />                                                                                    tions, diagram shows state single <br />                                                                                    gamete. steps, copies <br />                                                                                    acquires null mutations regula- <br />                                                                                    tory regions. left, fixed muta- <br />                                                                                    tion results absence functional pro- <br />                                                                                    tein product upper copy.                                                                                    gene nonfunctional pseudogene, <br />                                                                                    remaining regulatory regions associated <br />                                                                                    copy eventually accumulate degenera- <br />                                                                                    tive mutations. right, lower copy <br />                                                                                    acquires null mutation regulatory region intact upper copy. copies essential complete gene expression, mutational event <br /> permanently preserves members gene pair future nonfunctionalization. fourth regulatory region,  eventually acquire null mutation copy  center, regulatory region acquires new function preserves copy. beneficial mutation occurs expense essential function, duplicate <br /> copy preserved retains original function. <br />  <br />  <br /> tion). thought mutations               subfunction copy opposed <br /> involve changes regulatory regions (Grenier et al.                purifying selection. <br /> 1997; Shubin et al. 1997; Palopoli Patel 1998).                     Mutations cause subfunctions degrade Assuming new function results loss                 occur mechanisms, including nucleotide sub- <br /> essential ancestral function, neofunctionalization                stitutions, deletions, inversions, insertions transpos- <br /> sures preservation nonmutated copy.  prin-             able elements, slippage/replication errors, unequal <br /> ciple, neofunctionalization occur            crossing repeated transcription-factor <br /> copies acquire new regulatory region altering              binding sites. Transposable elements generate <br /> existing subfunctions, pointed Sidow (1996)].              subfunctional alleles. example, P, copia,  duplicate experience loss reduction               gypsy elements known mutagenic expression different subfunctions degenerative             insert 59 regions Drosophila genes (Kidwell <br /> mutations. case, combined action               Lisch 1997). Species recent history poly- <br /> gene copies necessary fulfill requirements          ploidization, example, maize, appear ancestral locus (subfunctionalization). happens,             insertions commonly 59 39 regions genes, complementation subfunctions dupli-                  species lacking recent polyploidization <br /> cate genes preserve partially degenerated cop-             event, insertions infrequent (White et al. 1994; <br /> ies. phase II DDC model, duplicate genes                   Wessler et al. 1995). transposable element inser- <br /> preserved neofunctionalization subfunc-                 tions, presumably regulatory DNA, tolerated <br /> tionalization undergo random resolution persisting                recently evolved polyploid species redundant subfunctions, accumulation degen-                redundancy regulatory elements. <br /> erative mutations eliminates subfunction                    probability subfunctionalization: argu- copy.                                                   ments presented suggest DDC process <br />    Subfunctionalization occur different                   make gene duplicates essential, routes: qualitative quantitative. qualitative sub-          account high levels duplicate gene preserva- <br /> functionalization, model illustrate               tion observed polyploid lineages? consider Figure 1, duplicate copy goes fixation               simple model suggests  reasonable pa- <br /> complete loss subfunction mutation, second                rameter values, DDC process account sig- <br /> locus subsequently acquires null mutation differ-            nificant fraction preserved duplicate genes. <br /> ent subfunction. contrast, quantitative subfunctional-               Consider situation members ization results fixation reduction expres-            recently duplicated gene independently mutable <br /> sion mutations duplicates. case,                subfunctions, essential, single total regulatory efficiency subfunction             copy, mutate identical rates (ur) copies reduced threshold level determined              alleles lacking relevant subfunction. Letting uc organismal requirements, degradation                  rate null mutations arise coding <br />                                                        Preservation Duplicate Genes                                              1535 <br />  <br /> region, null mutation rate locus uc 1 <br /> zur gene copy. assume conditions functional allele  possible allele copies) given duplicated gene pair sufficient wild- <br /> type function  double recessive model), beneficial mutations rare relative degenerative <br /> mutations. Provided product population size genic mutation rate ,0.01 (Li 1980), frequency double null homozygotes sufficiently low allele frequencies evolve effectively <br /> neutral manner. rate fixation mutation population approximately equal rate mutation level gene (Kimura 1983). <br />    imagine duplicate gene copies <br /> experiences fixation event. Assuming subfunction, probability gene <br /> survives event  does pseudogene) total regulatory-region mutation rate divided total mutation rate copies <br />                                                                            Figure 2.&#226;&#8364;&#8221;Combinations relative null mutation rates                                              zur                        regulatory coding regions (ur/uc) number subfunc- <br /> Prob (survival fixation event) 5         .            (1)      tions   yield various probabilities (PS) duplicate gene <br />                                           uc 1 zur                      preservation DDC process. probablility duplicate <br />                                                                         gene preservation increases number regulatory <br />   Following elimination subfunctions                elements. gene copy, second copy maintain subfunction, complete loss essential <br /> subfunction duplicates lethal.                  tionalization, PS, obtained summing quantity permissible mutation rate second copy                    5 2  <br /> comes  2 1)ur. Additional null mutations occur                                                     remaining  2 1) regulatory subfunctions coding region partially degraded copy. <br />                                                                                                  PS 5    o PS                      (4) <br />                                                                                                         5 <br />                                                                                                         2 <br />   total rate (summed copies) <br />                                                                         probability nonfunctionalization equal second mutational event [uc 1 2 2 1)ur]. <br />                                                                         1 2 PS. expression, probability probability subfunctionalization sec- <br />                                                                         duplicate-gene preservation increases num- <br /> ond event, PS,2, equal probability coding <br />                                                                         ber regulatory regions mutation rate regions survived hit multiplied                                                                         regulatory region (Figure 2). regulatory regions <br /> probability second mutation occurs comple- <br />                                                                         provide targets subfunctionalization mentary subfunction second copy, <br />                                                                         hit penalty, increasing mutation rate <br />  <br />                       1u 1zu zu 2 1u 1 2 2 1)u 2 <br />                                          2 1)u                          subfunction reduces relative probability fixa- <br />            PS,2 5               r                 r <br />                                                      .         (2)      tion null mutation coding region                         c             r       c       r <br />                                                                         complementation. <br /> Following logic, seen  2 1) distinct                 DDC process leads subfunctionalization series mutational events lead duplicate-gene                  high probability given reasonable parameter values. preservation subfunctionalization null                 example, subfunctions mutation <br /> mutations regulatory regions occur different                  rate subfunction 10% coding region null <br /> gene copies, initially occur copy                   rate, probability subfunctionalization 0.1, <br /> followed second copy, ini-                  mutation rate subfunction 30% tially occur copy followed fourth                   null rate, probablitity subfunctionaliza- second copy,  probability of                  tion 30% (Figure 2). Generally, total rate additional pathways subfunctionalization, e.,                subfunctional mutations (zur) exceeds null rate  2 1) consecutive regulatory-region null mutations                 coding region approximately fourfold, copy followed  given                  probability gene preservation subfunc- <br /> generalization Equation 2,                                           tionalization exceeds 50%. complexity size                                     i22                                 regulatory regions eukaryotic genes (Kirchhamer et <br />        PS 5   1u 1zu zu 2 p 1u 1 2                   c <br />                         r <br />  <br />                             r <br />                                      2 j 2 1)u <br />                                     j50  2 j 2 1)u 2 <br />                                           c <br />                                                      .r <br />  <br />                                                           r <br />                                                                (3)      al. 1996; Arnone Davidson 1997) suggests                                                                         conditions met frequently. total probability gene preservation subfunc-                     Time scales subfunctionalization resolution: <br />  1536                                                      Force et al. <br />  <br /> Using model presented  mean time gene preservation conditional actual occurrence obtained treating times mutational <br /> events geometrically distributed variables. rate occurrence initial regulatory-region null muta- <br /> tion 2zur, copies contains mutational targets. noted  subsequent initial event, zero  2 2) additional degenerative <br /> mutations incurred hit copy mutation opposite copy. mean time subfunctionalization conditional occurrence  2 1) consecutive regulatory-region null mutations copy followed                                            i21 <br />                 tS 5 <br />                          1 1 <br />                             1 <br />                          ur 2z <br />                                1 <br />                                            5 <br />                                             o <br />                                                    1 <br />                                            j 1 2 j <br />                                                     2.   (5a) <br />  mean time subfunctionalization  <br />                                                                        tS PS                          tS 5    o       PS <br />                                                .         (5b) <br />                                 5 <br />                                 2 <br />  <br />    classical model, expressions indicate fates duplicate genes generally deter- <br /> mined relatively short period  evolutionary <br /> time scale; Figure 3A). example, ur 5 1027/yr, <br /> tS order 4 million yr provided number regulatory regions greater  , 5 does exceed 12.5 million yr.  DDC model, duplicate genes destined preserved subfunctionalization                  Figure 3.&#226;&#8364;&#8221;  mean time subfunctionalization expected million years.             function number subfunctions  , situation regulatory-region mutation rate x times              ur 5 1027/yr. (B) fates gene pairs deter- <br /> figure, mean time subfunctionalization             mined short time, evolutionary scale, ratio <br /> divided x.                                                      mutation rate regulatory coding regions                                                                    weak determinant expected degree resolution    Unless initial regulatory regions,           time subfunctionalization. regulatory regions  2 2) likely <br /> remain resolved evolutionary time initial subfunctionalization event. fraction regu-             molecular nature subfunctions preser- <br /> latory regions expected resolved time         vation genetic redundancy: preceding theory gene preservation subfunctionalization                    assumes individual regulatory subfunctions  <br />                                                                                    iPS                  dependently mutable, single mutations suf- <br />                      Pr(0) 5           o    zPS <br />                                                  .        (6)      ficient eliminate subfunction. simple <br />                                       5 <br />                                       2 <br />                                                                    scenario, various subfunctions duplicate <br />    fraction depends weakly ratio               genes preserved DDC process expected coding-region regulatory-region mutation rates,             resolved randomly, copy retaining ,0.5 number regulatory regions exceeds           half subfunctions limits binomial <br /> (Figure 3B).  anticipate preserva-          sampling.  define subfunctions tion duplicate genes DDC process, substantial          mutational properties members <br /> fraction regulatory subfunctions typically remain          distinct complementation classes, definition does resolved phase II. Assuming occurrence           subfunctions arranged mutations destroy regulatory regions Poisson          DNA molecule. Regulatory regions different sub- <br /> process, site unresolved time            functions partially overlapping embedded, <br /> gene preservation, probability unre-          leading situation number expression <br /> solved t time units simply P0(t) 5 e22tur.        domains exceeds number complementation number unresolved sites time t follows              groups (Jack DeLotto 1995; Kirchhamer et al. binomial distribution parameter P0(t).                      1996). central issues illustrated Figure <br />                                                    Preservation Duplicate Genes                                         1537 <br />  <br />                                                                     (Pickett Meeks-Wagner 1995). Figure 4C,                                                                     example, regulatory regions 1 3 partially  <br />                                                                     lapping regulatory region 2 embedded                                                                     region overlap. situation, gene duplicate <br />                                                                     suffers degenerative mutation nonoverlapping <br />                                                                     portion regulatory region 1, gene dupli- <br />                                                                     cate experiences degenerative mutation destroying <br />                                                                     nonoverlapping portion regulatory region 3,                                                                     regulatory region 2 sheltered mutation <br />                                                                     events destroy adjacent binding <br />                                                                     sites mutation region                                                                     completely eliminate subfunction 1 3 (Figure 4F). <br />                                                                     Resolution subfunction 2 proceed                                                                     small-scale mutational events partially degrade                                                                     quantitative functionality individual transcription-fac- <br />                                                                     tor binding sites. hand, binding sites <br />                                                                     regulatory region 2 completely shared                                                                     regulatory regions 1 3 (Figure 4, E G),                                                                     mutational events inactivate regions 1 3                                                                     opposite copies indefinitely preserve embedded <br />                                                                     regulatory region 2 copies.  resolution <br />                                                                     overlapping embedded subfunctions pro- <br />                                                                     vides potential explanation maintenance                                                                     redundancy observed duplicate <br />                                                                     genes invoking positive selection duplicate <br />   Figure 4.&#226;&#8364;&#8221;Overlapping embedded regulatory ele-                gene expression (Nowak et al. 1997). <br /> ments. transcription-factor binding sites shown             topology regulatory regions help <br /> sumed essential expression domain.   Inde-         explain unidirectional bidirectional divergence pendent regulatory regions independent transcription-          gene duplicates observed Ferris Whitt (1979). <br /> factor binding sites. (B) Overlapping regulatory regions independent sites. (C) Overlapping embedded regulatory          enzyme-encoding loci, gene duplicates regions independent sites. (D) Overlapping regulatory          unidirectional divergence enzyme products regions shared sites. (E) Embedded regulatory regions          locus predominate tissues loci shared sites. (F) Resolution overlapping regulatory         expressed. situation common <br /> regions independent sites (derived C) leading          gene loci overlapping embedded ar- <br /> quantitative resolution regulatory region 2 1 3 destroyed paralogous copies. (G) Resolution embedded          rangements regulatory regions, elimina- <br /> regulatory regions shared sites leading true redun-         tion embedded element gene copy <br /> dancy regulatory region 2 1 3 destroyed        (element 2 Figure 4E, example) prevent paralogous copies.                                                  fixation alleles overlapping regulatory <br />                                                                     elements (elements 1 3 Figure 4E) destroyed <br />                                                                     gene duplicate. bidirectional diver- <br /> 4. situation modeled equivalent setting           gence, locus highly expressed spatial arrangement transcription-factor            given tissue. situation frequent <br /> binding sites allows independent resolution              gene loci independently arranged regulatory <br /> subfunctions  5 3 Figure 4A). overlapping               regions. <br /> (Figure 4, B D) embedded (Figure 4, C E)                    DDC dosage effects: situations, gene <br /> regulatory regions, transcription-factor binding sites          dosage requirements increase probability interdigitated acting independently               gene duplicates preserved. theoretical <br /> (Figure 4, B C) shared, DNA                    model developed assumes subfunc- <br /> binding site(s) required ex-                tion, activity alleles pression domain (Figure 4, D E). cases,                gene duplicates sufficient survival. possible, mutational events knock                   gene duplication subfunctions <br /> expression domain time, effectively reducing               remain intact alleles <br /> number subfunctions.                                             ensure optimal fitness. instance, consider gene <br />    Complexities involving physical arrangement               separate subfunctions. duplication, regulatory regions DNA help explain,                subfunction sufficient survival intact invoking positive selection, expression            single allele, second subfunction suffi- <br /> domains preserved gene duplicates                    cient alleles, subfunction  1538                                                    Force et al. <br />  <br /> required alleles. case, second subfunctions resolved duplicate gene principles DDC. subfunction,  maintained gene duplicates active alleles. cases, dosage requirements provide initial gene preservation mechanism, complemen- <br /> tary loss subfunctions acquisition new <br /> function reinforce initial preservation event. <br /> Note type dosage effect provides alterna- <br /> tive mechanism shared embedded elements (Figure                  Figure 5.&#226;&#8364;&#8221;Phylogeny engrailed family genes. Sequences <br />                                                                  aligned eye assistance CLUSTAL X, 4) retaining specific expression domain            unambiguous portions alignment em- <br /> gene duplicates.                                                 ployed final analysis. Phylogenetic tree based    cases, gene dosage requirements cause           neighbor-joining method (Saitou Nei 1987) multi- partitioning subfunctions favored positive       ple substitution correction. Numbers bootstrap values <br /> selection. example, consider situation            based 1000 runs. Lamprey enga cloned polymer- <br />                                                                  ase chain reaction using primers [forward, 59ATI AA activity alleles duplicated gene pair      G) ATI TGG TT(T C) CA G) AA(T C) AA39; reverse, <br /> certain tissue time deleterious. case,       59TG(G  TT(G  TAI G CC(T C) TGI GCC AT39] <br /> fixation nonfunctional subfunctional allele        designed amplify conserved Engrailed <span id='am-15' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> se- accelerated positive selection. Note case           quences IKIWFQNK MAQGLYNH, respectively. like differs formal model proposed           primers used screen lamprey cosmid library  <br />                                                                  structed single ammocete larva according Stra- assumes drift purifying selection usu-         tagene SuperCos cloning kit. primers used ally sufficient fixation subfunctional alleles.       obtain additional sequence zebrafish eng1b (Amores et cases, mutations subfunctions          al. 1998). Species abbreviations: Hsa, Homo sapiens; Mmu, Mus <br /> deleterious single-copy genes duplication          musculus; Gga, Gallus gallus; Dre, Danio rerio; Pma, Petromyzon beneficial duplication.          marinus; Bfl, Branchiostoma floridae;  Heliocidaris erythro- <br />                                                                  gramma. GenBank accession nos.: Hsa EN2, J03066; Mmu increase rate fixation subfunctional al-         En2, L12705; Gga En2, L12697; Dre eng2, X68446; Dre eng3, <br /> leles simultaneously increasing rate fixation       X68447; Hsa EN1, L12699; Mmu En1, Y00201, M11987; Gga nonfunctional alleles, overall effect proba-       En1, L12695; Dre eng1b, AF071237; Dre eng1, X68445; Pma <br /> bility duplicate gene preservation clear. Future       enga AF129401; Bfl AmphiEn, U82487; HeEn, U58775. <br /> experimental modeling work help define complex interactions gene dosage, <br /> population size, mutation rates subfunctional,            1992). Zebrafish,  reported coding null, neofunctional alleles, roles            engrailed genes (eng1, eng2, eng3; Ekker et purifying positive selection duplicate gene            al. 1992), report additional sequence preservation. hoped near-neutral DDC              expression data fourth eng gene recently <br /> model provided act null hypothesis             cloned, eng1b (Amores et al. 1998). Phylogenetic analysis <br /> testing complex possibilities.              rooted engrailed genes amphioxus (Holland et <br />    Possible examples DDC process: pre-            al. 1997) new lamprey engrailed gene, enga, shows <br /> sent possible examples general DDC pro-           eng1/eng1b eng2/eng3 ancient duplicate <br /> cess way account observed            gene pairs (Figure 5). tree shows eng1 patterns duplicate gene expression. Additionally,          eng1b orthologous tetrapod En1, suggest experiments falsify DDC model             eng2 eng3 orthologous tetrapod En2. explanation specific cases. consider          phylogenetic tree shows duplication pair duplicate engrailed genes zebrafish        event(s) produced eng1/eng1b eng2/eng3 ZAG1 ZMM2 gene pair maize. Analysis           gene pairs occurred lineage zebrafish cases identify gene duplicates, determine           diverged lineage tetrapods. duplication arose tandem duplication duplication            produced main clades vertebrate engrailed <br /> large chromosome regions, infer ancestral functions           genes occurred divergence. unduplicated parent gene, finally determine                 determine zebrafish eng gene pairs distribution gene functions du-           originated chromosome-scale duplications local <br /> plicated genes explained complementary             tandem duplications, mapped eng1b locus sharing ancestral functions acquisition        compared genome locations engrailed novel functions.                                              genes mammals zebrafish (Logan et al. 1989; <br />    Engrailed genes zebrafish: Tetrapods mem-         Amores et al. 1998; Postlethwait et al. 1998). bers engrailed gene family, called En1 En2            results showed zebrafish genes eng1 eng1b <br /> (Joyner Martin 1987; Gardner Barald                      linked dlx2 dlx5 different chromo- <br />                                                 Preservation Duplicate Genes                                         1539 <br />  <br />                                                                  fish duplicates eng1/eng1b.  En1 used <br />                                                                  outgroup infer ancestral shared expression <br />                                                                  domains eng1 eng1b. <br />                                                                     expression patterns engrailed genes <br />                                                                  complex, focus expression patterns                                                                  engrailed-1 gene family groups cells. Zebra- <br />                                                                  fish eng1 expressed pectoral appendage bud, <br />                                                                  eng1b expressed specific set neurons                                                                  hindbrain/spinal cord (Figure 7).                                                                  expression domains neofunctionalization?                                                                  present progenitor gene dupli- <br />                                                                  cation domain lost duplicate? Examin- <br />                                                                  ing recent unduplicated outgroup allow <br />                                                                  infer state ancestral gene. ab- <br />                                                                  sence information recent outgroup, <br />                                                                  tetrapods provide appropriate data. mouse                                                                  chicken, En1 expressed expression domains, <br />                                                                  developing pectoral appendage bud, specific <br />                                                                  neurons hindbrain spinal cord ( Joyner                                                                  Martin 1987; Davis et al. 1991; Gardner Barald <br />                                                                  1992). appears following duplica- <br />                                                                  tion event produced eng1 eng1b ray- <br />    Figure 6.&#226;&#8364;&#8221;Syntenic relationships engrailed-1 genes. Ze-    finned lineage, eng1 ancestor retained pectoral <br /> brafish appears copies single chromosome        appendage bud expression lost hindbrain/spi- <br /> segment present common ancestor teleosts      nal cord neuron expression, eng1b ancestor <br /> mammals, copy eng1 dlx2 copy eng1b dlx5. copy segment broken    lost pectoral appendage bud expression  chromosome rearrangement, appears LG6            tained hindbrain/spinal cord neuron expression LG1. humans, zebrafish LG9, segment        hypothesized Figure 7. <br /> single chromosome arm, Hsa2q, mouse, segment            case gene preservation subfunctionaliza- chromosomes. Orthologous genes         tion? data suggest complementary loss expres- <br /> horizontal line. Syntenies displayed, gene orders ignored ease comparisons orthologues.                  sion, consistent DDC model. defini- <br />                                                                  tive test hypothesis require identification                                                                  regulatory elements responsible expres- <br /> somes (Figure 5). Prior analysis shown gene         sion domains zebrafish, fish share eng1/eng1b <br /> duplicates dlx2 dlx5 zebrafish orthologues          duplication, fish diverged lineage giving tetrapod DLX2 (Stock et al. 1996; Ellies et al. 1997),        rise zebrafish duplication event, tetra- linked EN1 human chromosome 2q                    pods, including mouse chicken. zebrafish, (Ozcelik et al. 1992). Likewise, eng2 eng3 genes             appear examples similar engrailed, includ- linked gene duplicates shh twhh            ing duplicates msx genes (Ekker et al. 1997), nkx <br /> different zebrafish chromosomes (Postlethwait et al.             genes (Lee et al. 1996), hedgehog genes (Ekker et <br /> 1998), zebrafish hedgehog genes            al. 1995). <br /> orthologous tetrapod SHH (Zardoya et al. 1996),                  ZAG1 ZMM2 maize: second possible exam- linked EN2 human chromosome 2q                    ple preservation gene duplicates subfunc- <br /> (Logan et al. 1989). mapping results             tionalization, consider duplicate genes known zebrafish copies large chromosome segments            ZAG1 ZMM2 maize genome, originated <br /> surrounding zebrafish engrailed genes,        allotetraploidization event closely <br /> syntenic regions conserved mammalian ge-                related grasses 11 mya (Goodman et al. 1980; <br /> nomes (Figure 6). conclude gene pairs            Wendel et al. 1986; Helentjaris et al. 1988; Gaut eng1/eng1b eng2/eng3 arose duplication events             Doebley 1997; White Doebley 1998). ZAG1 involved large chromosome sections, consistent              ZMM2 apparent orthologues single-copy flo- origin genome duplication event ray-          ral homeotic genes known AGAMOUS Arabidopsis <br /> finned fish (Amores et al. 1998; Postlethwait et al.             PLENA Antirrhinum, ex- <br /> 1998).                                                           pressed carpels stamens (Yanofsky et al. 1990; <br />    Note independent data sets, gene phyloge-            Coen Meyerowitz 1991; Bradley et al. 1993). <br /> nies based sequence information chromosomal               ZAG1 expressed high levels maize car- <br /> locations based genetic mapping data, concur             pel development, expressed low levels tetrapod En1 gene outgroup zebra-           stamen primordia (Mena et al. 1996). contrast, <br />  1540                                                        Force et al. <br />  <br />  <br />  <br />  <br />                                                                                       Figure 7.&#226;&#8364;&#8221;Expression pattern zebrafish <br />                                                                                    eng1 eng1b. Expression eng1 (left)                                                                                    eng1b (right) transcript 28.5-hr zebrafish em- <br />                                                                                    bryos. Dorsal view trunk showing pecto- <br />                                                                                    ral appendage bud  , expresses <br />                                                                                    eng1, dorsal view head    <br />                                                                                    ing expression eng1b specific set                                                                                    hindbrain spinal neurons. situ hybrid- <br />                                                                                    ization performed according Jowett et <br />                                                                                    al. (1995) Thisse et al. (1993). eng1b <br />                                                                                    probe derived 39 untranslated <br />                                                                                    region polymerase chain reaction using <br />                                                                                    using primers (forward, GCTCATGGCT <br />                                                                                    CAAGGACTCTA; reverse, AACATTG <br />                                                                                    GACTAAACGTAAAACTT) cloned                                                                                    TA cloning vector (Invitrogen, San Diego), <br />                                                                                    eng1 probe gift Monte <br />                                                                                    Westerfield. figure shows hypothesized <br />                                                                                    evolutionary scenario tissue-specific <br />                                                                                    patterns expression engrailed family <br />                                                                                    members zebrafish left tetra- <br />                                                                                    pods right. Solid open boxes indi- <br />                                                                                    cate expression lack expression, <br />                                                                                    respectively. Blue boxes represent pectoral <br />                                                                                    appendage regulatory element red boxes <br />                                                                                    indicate neuron regulatory element. <br />  <br />  <br />  <br />  <br /> ZMM2 highly expressed maize stamens                 ancestral genes ZAG1 ZMM2 immature carpel, increases late            expressed strongly developing stamens car- <br /> female flower development (Figure 8). phenotype                  pels allotetraploid ancestor maize shortly ZAG1 null mutant limited early carpel develop-            polyploidization event, AGAMOUS PLENA ment, suggesting late ZMM2 expression rescues                   today Arabidopsis Antirrhinum. hypothesize <br /> later carpel development ZAG1 mutants.                            reciprocal regulatory mutations ZAG1/ <br />    DDC model explain data suggesting                ZMM2 duplicates complemented                                                                       preserving genes exist today&#226;&#8364;&#8482;s maize.                                                                      allotetraploidization event, degenerative regulatory <br />                                                                      mutations decreased expression ZAG1 stamens <br />                                                                      carpel, regulatory mutations elim- <br />                                                                      inated expression ZMM2 early carpel                                                                      stamens. hypothesis correct,  <br />                                                                      maize plants doubly homozygous ZMM2 ZAG1 <br />                                                                      null mutations produce plants phenocopy <br />                                                                      AGAMOUS PLENA mutants Arabidopsis  <br />                                                                      tirrhinum. addition, molecular analysis pro- <br />                                                                      moters gene family maize, close relative <br />                                                                      sorghum, Arabidopsis, Antirrhinum identify <br />                                                                      conserved regulatory elements partitioned <br />                                                                      gene duplication. <br />   Figure 8. subfunctionalization hypothesis explain              Hoxa1 Hoxb1 mouse: possible example expression duplicated genes maize. Small solid boxes <br /> indicate active regulatory elements, gray boxes indicate regula-     DDC duplicate genes involves Hoxa1 Hoxb1 <br /> tory elements reduced activity, open boxes indicate         genes mouse (Figure 9). genes reside dupli- <br /> null activity alleles regulatory element.                    cate Hox clusters, groups closely linked genes                                                 Preservation Duplicate Genes                                       1541 <br />  <br />                                                                  rhombomere 4. Judging expression functional <br />                                                                  analysis, analogous element probably drives early <br />                                                                  expression Hoxa1 rhombomere 5 caudal <br />                                                                  segments, element defined molecu- <br />                                                                  larly. DDC model predicts unduplicated <br />                                                                  ancestor genes possessed regulatory ele- <br />                                                                  ments drove expression Hox gene                                                                  late r4 early r5 expression domains,                                                                  duplication, subfunctions resolved divergently <br />                                                                  paralogous copies (Figure 9). <br />                                                                     addition independent roles Hoxa1                                                                  Hoxb1 just discussed, genes early redun- <br />                                                                  dant roles, including expression broadly overlapping <br />                                                                  territories activation  <br />                                                                  stream targets (Studer et al. 1994, 1996; Dupe&#194;&#180; et al. <br />                                                                  1997; Maconochie et al. 1997; Chen Ruley 1998). <br />                                                                  shared roles important inde- <br />                                                                  pendent roles. genes contain <span id='am-22' about='obo:IMR_0100041' typeof='owl:Thing obo:IMR_0000877 obo:IMR_0001656 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001'><span id='am-23' property="oboInOwl:hasExactSynonym" content="Retin A" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="vitamin A acid" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Tretinoin" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00777" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="vitamin A acid" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Retin A" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="all-trans-Vitamin A acid" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0100041" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Tretinoin" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="retinoic acid" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="all-trans-Vitamin A acid" datatype="xsd:string"></span>retinoic acid</span>  <br />   Figure 9. DDC hypothesis explain expression        sponse element (RARE) 39 region initiates duplicated Hox genes mouse? Small solid boxes indicate     expression adjacent gene neuroectoderm conserved RARE enhancer, medium boxes indicate active          mesoderm (Langston et al. 1997; Figure 9). Hoxa1 <br /> rhombomere regulatory elements, large boxes indicate         RARE required Hoxa1 expression transcribed coding regions. &#226;&#8364;&#339;r5 enhancer?&#226;&#8364;&#65533;      rhombomere 5, proper level Hoxa1 <br /> characterized element expected drive expression r3/r4 border, r5 caudal segments,       expression somitic presomitic mesoderm necessary glossopharyngeal nerve develop-     (Thompson et al. 1998). requirement single <br /> ment. Open boxes indicate hypothetical dead regulatory ele-      conserved regulatory element  RARE) expres- <br /> ments. r5 enhancer element Hoxa1         sion different domains (rhombomere 5 meso- <br /> identified predicted present.                       derm) property expected embedded regulatory <br />                                                                  elements diagrammed Figure 4, D, E, G.                                                                  DDC model predicts subfunctions encode family DNA-binding proteins specifies           shared embedded regulatory elements diver- <br /> fate anterior-posterior axis bilaterian ani-        gently resolved, redundant, overlapping expres- <br /> mals (Lewis 1978; Krumlauf 1994). tetrapod              sion domains retained, observed Hox clusters arose sequential duplication               Hoxa1/Hoxb1 duplicated gene pair portions events, probably genome duplications,           neuroectoderm mesoderm. <br /> divergence ray-finned lobe-finned fishes 420 mya             experiments distinguish cur- <br /> (Duboule Dolle&#194;&#180; 1989; Holland Garcia-Fer-                rent subfunctions murine Hoxa1 Hoxb1 dupli- <br /> nandez 1996; Amores et al. 1998; Postlethwait et al.             cates arose subfunctionalization, neofunctionaliza- <br /> 1998). Hoxa Hoxb clusters probably sisters           tion, model? critical issue derived second duplication event (Kappen            r4 r5 enhancers present ancestral <br /> Ruddle 1993; Zhang Nei 1996; Amores et al. 1998).            gene duplication. infer state  case, entire genes regulatory       ancestral gene examining outgroup diverged <br /> elements duplicated, true             lineage tetrapods just duplication tandem duplication.                                            event produced Hoxa1 Hoxb1 genes.    Hoxb1 Hoxa1 cooperate pattern anterior ecto-           lamprey outgroup (Pendleton et al. <br /> dermal mesodermal derivatives vertebrate em-              1993; Amores et al. 1998; Carr et al. 1998) experi- <br /> bryos (Figure 9). Hoxa1 important segment iden-           ments conducted   tity rhombomere 5 (r5) hindbrain           determine corresponding functions development glossopharyngeal nerve             likely present ancestral gene caudal rhombomeres (Dupe&#194;&#180; et al. 1997; Studer et            duplication. <br /> al. 1998). Analogously, Hoxb1 important specifying           summary, examples discussed provide identity rhombomere 4 (r4) development             data consistent DDC model, facial nerve (Po&#194;&#168;pperl et al. 1995; Gavalas et al.           cases readily explained DDC <br /> 1998). 70-bp-long element, called r4 autoregula-           model classical model. experiments <br /> tory enhancer, conserved mouse, chick,            need firmly establish route dupli- <br /> pufferfish (Po&#194;&#168;pperl et al. 1995). element functions        cate gene preservation employed case. elevate expression Hoxb1 restrict               Testing DDC classical models: noted <br />  1542                                                   Force et al. <br />  <br /> earlier, basic premise classical                                          TABLE 1 <br /> model duplicate gene evolution gene dupli-                      Critical values zur/(uc 1 zur) required specific <br /> cates preserved evolution new func-                  probabilities subfunctionalization (PS), given tions tested. deleterious mu-                      different numbers subfunctions   obtained <br /> tations common beneficial                                           Equations 3 4 <br /> mutations, believe DDC process provides reasonable  parsimonious) alternative explanation                                                      PS cases long-term preservation gene                           0.1                 0.3                 0.5 <br /> duplicates. Unlike classical model, mutational <br /> mechanisms lead gene preservation DDC             2                  0.448                0.774               1.000 <br /> distinct responsible origin new            3                  0.384                0.654               0.832 <br />                                                                  5                  0.351                0.598               0.759 <br /> gene functions. hand, expanding                                                                 10                  0.332                0.57                0.728 <br /> time period genes exposed selection,           &#226;&#710;&#382;                   0.316                0.548               0.708 preservation duplicates DDC process facili- <br /> tates subsequent opportunity evolution new <br /> functions. evolution new gene functions mechanism duplicate gene preservation,             larger achieve 50% probability possible empirically reject alternative           gene preservation (Table 1). relative rate subfunctionalization hypothesis. consider              formation subfunctional alleles potentially fruitful avenues future research.                  range, subfunctionalization modeled  <br />                                                                    likely major mechanism duplicate gene <br /> 1. Phylogenetic analysis: subfunctionalization                 preservation. Experiments designed mea- <br />    model predicts sum subfunctions pre-             sure mutation rate subfunctional, neofunc- <br />    served gene duplicates equal total sub-          tional, nonfunctional alleles test critically    functions ancestral gene. prediction             various models. empirical studies demonstrate    clearly distinct position classical             rate mutation subfunctional alleles    model, suggests gene preservation              low relative rate coding null mutations,    pendent acquisition new cis-regulatory              particular subfunctionalization model falsified. <br />    regions driving novel expression patterns          3. Regulatory region complexity: subfunctionaliza- <br />    velopment (Sidow 1996; Cooke et al. 1997). test              tion model predicts probability gene pres- <br />    alternative hypotheses, evolutionary time             ervation higher complex genes <br />    frame short preclude possibil-             larger numbers subfunctions), particularly <br />    ity genes initially preserved subfunctionaliza-         genes regulatory regions subfunc- <br />    tion subsequently acquired new func-             tions spatially independent DNA    tions. requires analysis recently              complex genes provide targets sub- <br />    preserved duplicates cladogram allows            function mutations. Testing prediction requires <br />    inference ancestral expression patterns             molecular characterization regulatory regions <br />    appropriate outgroups. example, explain              various genes species duplicated genes <br />    derivation triplicate Drosophila genes paired,           comparison closely related species    gooseberry, gooseberry-neuro, conserved          duplications. <br />    protein function distinct developmental func-            4. Multiple polyploidization events: subfunctionali- <br />    tions, Li Noll (1994) suggested following              zation model makes specific predictions    duplication &#226;&#8364;&#339;genes acquire new functions                 probability duplicate gene preservation    changes regulatory regions generating               closely spaced polyploidization events,    altered expression&#226;&#8364;&#65533; considering possibil-           thought occurred early vertebrate phylog- <br />    ity genes simply result dif-          eny (Holland et al. 1994; Holland Garcia- <br />    ferential loss subsets expression domains          Fernandez 1996; Nadeau Sankoff 1997; <br />    ancestral gene. phylogenetic analysis closely          Amores et al. 1998). DDC model suggests    related outgroup species single gene copies                duplicate loci preserved subfunctionalization    distinguish classical DDC                 polyploidization event (Figure 1, right)    models.                                                         fewer subfunctions parent locus 2. Mutation rate subfunctional alleles: simple              duplication (Figure 1, .   <br />    subfunctionalization model discussed requires              ory predicts likelihood preservation  <br />    total subfunction mutation rate relative            pends number subfunctions (Figure 2),    total null mutation rate order 0.3             probability duplicate loci preserved <br />    larger achieve 10% probability              second round duplication diminished <br />    duplicate gene preservation, order 0.7            relative polyploidization event.   <br />  <br />    hand, single duplicate survives round      seriously testable model explaining evolu- <br />    original subfunctions intact (Figure         tionary fate duplicate genes. <br />    1, left), second round duplication,         <br />    probability duplicate preservation ap-    <br />    proximately event.           <br />    level gene preservation does change         <br />    polyploidization events, subfunctionalization         <br />    unlikely explanation preservation dupli- <br />    cate genes. Data HOX complexes verte-            <br />    brates suggest level duplicate preservation      <br />    does decline subsequent duplication          <br />    events. Assuming (AB)(CD) model HOX clus- <br />    ter duplication (Kappen Ruddle 1993; Zhang <br />    Nei 1996; Amores et al. 1998), 11 14 <br />                                                                                    <br />    gene pairs (13 HOX cluster genes plus EVX; 79%) <br />    preserved duplication      <br />    (Amores et al. 1998). second duplication,               <br />    lineage leading mouse, 16 25 gene pairs         <br />    (64%) preserved, duplica-          <br />    tion, occurred ray-finned fish lineage        <br />    leading zebrafish (Amores et al. 1998), 13          <br />    45 gene pairs (29%) preserved            <br />    present duplicate copies. caveat            <br />    genes remain discovered zebra-    <br />    fish, genes present duplicate copy      <br />    time second duplication events      <br />    permanently preserved mecha-              <br />    nism, results conform qualitative        <br />    expectations DDC model.                                   <br /> 5. Population size: Finally, subfunctionali-           <br />    zation model assumes selection mutant      <br />    alleles negligible, predictions model      <br />    fulfilled closely populations         <br />    relatively small size, conditions,          <br />    incidence double null homozygotes                <br />    appearance beneficial mutations minimal.        <br />    hand, neofunctionalization relies         <br />    entirely rare beneficial mutations,         <br />    common mechanism gene preservation                <br />    species large effective sizes.                         <br />  <br />    focuses regulatory complexity         <br /> genes preservational nature degenerative                <br /> mutations, DDC model provides different perspec-           <br /> tive evolutionary consequences gene duplica-            <br /> tions classical model, focuses           <br /> primarily nonfunctionalizing properties               <br /> generative mutations coding regions neo-       <br /> functionalizing properties beneficial mutations.             <br /> DDC model organizes disparate collection             <br /> observations principles gene structure,          <br /> believe principles established       <br /> firmly subfunctionalization taken       <br /> </body></html>