<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>In vitro vivo function C-terminus Escherichia coli single-stranded DNA binding protein <br />  <br />  <br />  <br />                                                           constructed deletion mutants Escherichia           role DNA replication, repair recombination, <br /> coli single-stranded DNA binding protein (EcoSSB)                single-stranded DNA binding protein Escherichia coli <br /> lacking different parts C-terminal region.           (EcoSSB) essential survival cell (1&#226;&#8364;&#8220;3). EcoSSB <br /> region EcoSSB composed parts: <span id='am-12' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-13' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycine</span>-            consists 177 amino acids forms stable homotetramer proline-rich sequence &#226;&#710;&#188;60 amino acids followed            (4). Secondary structure prediction indicates sequence acidic region 10 amino acids          EcoSSB divided parts, N-terminal domain <br /> highly conserved bacterial SSB proteins.               (&#226;&#710;&#188;120 amino acids) rich &#206;&#177; helices &#206;&#178; sheets single-stranded DNA binding protein human                 unstructured C-terminus (5). N-terminal thirds <br /> mitochondria (HsmtSSB) lacks region homologous                 contain DNA-binding domain (6). function C-terminal EcoSSB.             C-terminal characterized  importance <br /> investigated chimeric protein consisting                C-terminus illustrated ssb113 mutation (7), complete sequence human mitochondrial                     penultimate residue EcoSSB protein (proline <br /> single-stranded DNA binding protein (HsmtSSB)                176) replaced serine (8). mutation results UV- C-terminal EcoSSB. Fluorescence titrations          temperature-sensitive phenotype. DNA-binding affinity DNA-melting curves showed C-terminal                mutated protein slightly affected (8). Williams et al. (6) EcoSSB essential DNA-binding              showed C-terminal EcoSSB vitro. affinity single-stranded DNA RNA           accessible proteolysis DNA-binding. Removing increased removal 10 amino               C-terminal fragment resulted protein higher affinity <br /> acids. Consequently, nucleic acid binding affinity           single-stranded DNA. HsmtSSB reduced addition C-terminus             C-terminal EcoSSB consists regions: EcoSSB. mutant proteins lacking 10               sequence &#226;&#710;&#188;50 amino acids rich proline <span id='am-24' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-25' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> residues <br /> amino acids unable substitute wild-type EcoSSB            followed acidic region 10 amino acids containing vivo.  nucleic acid binding properties         aspartate residues. contrast proline- glycine-rich depend intact C-terminus, region            sequence, region 10 amino acids highly conserved <br /> essential vivo function. DNA                 procaryotic SSB proteins (1). <br /> binding properties HsmtSSB EcoSSB quite                  characterize functions C-terminal similar, HsmtSSB does function E.coli.               EcoSSB, constructed mutant EcoSSB proteins failure overcome fusing C-terminal              different parts region deleted. EcoSSB HsmtSSB. differences                 Information influence N-terminal DNA <br /> N-terminal parts proteins responsible            binding domain functions C-terminal incompatibility. mutants                obtained replacing domain human mitochondrial <br /> defective tetramerization.  mixed tetramers           single-stranded DNA binding protein (HsmtSSB). HsmtSSB, formed proteins containing             sharing 36% sequence homology N-terminus <br /> N-terminal  reflects structural differences            EcoSSB, homotetrameric single-stranded DNA binding N-terminal parts HsmtSSB                      protein (SSB) vitro DNA binding properties EcoSSB. results indicate region            indistinguishable EcoSSB (9). size HsmtSSB 10 amino acids, highly conserved             (133 amino acids) corresponds size N-terminal domain <br /> bacterial SSB proteins, involved essential protein&#226;&#8364;&#8220;        EcoSSB protein completely lacking sequences <br /> protein interactions E.coli cell.                         homologous C-terminal EcoSSB. <br />  <br />  <br />  <br />  <br /> *   correspondence addressed <br />                                                                                                             2707 <br />  <br />                                                                                Nucleic Acids <br />                                                                                Nucleic       Research,1994, <br />                                                                                        Acids Research, 1996,Vol.  24  <br />                                                                                                              Vol.22,  114       2707 <br />  <br /> MATERIALS METHODS                                                    Nucleic acid concentrations given units monomers                                                                        determined using following absorption coefficients: <br /> Poly(dT), poly(dA&#226;&#8364;&#8220;dT) poly(rU) purchased                 &#206;&#181;max = 8600 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1 poly(dT) (16), &#206;&#181;260 nm = 9200 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1 <br /> Pharmacia (Freiburg). Experiments performed standard         poly(rU) (17) &#206;&#181;260 nm = 6700 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1 poly(dA&#226;&#8364;&#8220;dT). <br /> buffer containing 0.1 mM EDTA, 20 mM potassium phosphate <br /> pH 7.4 sodium chloride indicated. fluorescence titrations <br />                                                                        Physicochemical experiments <br /> 100 p.p.m. Tween 20 (Serva) added. <br />                                                                        Fluorescence titrations carried Schoeffel RRS 1000 <br /> Media                                                                  spectrofluorimeter described earlier (18). Excitation wavelength <br />                                                                        295 nm emission observed 350 nm. Theoretical <br /> LB medium (10) used microbiological experiments.           binding isotherms calculated using model Schwarz concentration antibiotics used selection purposes       Watanabe (19) binding site size n, cooperative <br /> 100 &#194;&#181;g/ml ampicillin 30 &#194;&#181;g/ml chloramphenicol respectively.        binding affinity (K &#207;&#8240;), fluorescence quench (Qf )                                                                        parameters. fluorescence quench difference Site-directed mutagenesis                                              normalized fluorescences free protein protein&#226;&#8364;&#8220;DNA <br />                                                                        complex. <br /> Site-directed mutagenesis performed gapped-duplex             DNA melting curves measured DMR10 (Zeiss) <br /> method using pBR322 derived plasmids pSF1 (11)                 spectrophotometer described previously (20) using heating rate <br /> pSF1(HsmtSSB) (9) conferring ampicillin resistance.           20 K/h. significant differences heating cooling <br /> truncated EcoSSB mutants EcoSSB Q152* EcoSSB G117*                 curve observed confirming reversibility melting. constructed changing codons Q152 G117            Analytical ultracentrifugation carried Spinco/ stop codon TAG TGA respectively. construction           Beckman model E centrifuge evaluated described earlier deletion mutant EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 HsEcSSB,              (21). <br /> introduced cleavage sites blunt end restriction endonucleases respective positions genes. cleavage desired   Subunit exchange experiments <br /> gene fragments purified using preparative agarose gel <br /> electrophoresis ligated. EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 fusion protein       subunit exchange experiments, different SSB proteins amino acids 1&#226;&#8364;&#8220;115 EcoSSB followed         mixed concentration 50 &#194;&#181;M buffer <br /> 10 amino acids (168&#226;&#8364;&#8220;177) EcoSSB. HsEcSSB, complete           containing 0.5 M NaCl, 1 mM EDTA, 20% glycerol 20 mM <br /> sequence HsmtSSB (1&#226;&#8364;&#8220;133) followed amino acids                 potassium phosphate, pH 7.5 volume 10 &#194;&#181;l. mixture <br /> 113&#226;&#8364;&#8220;177 EcoSSB.                                                     allowed react 3 days room temperature                                                                        exchange experiments containing EcoSSB derivatives Protein preparation                                                    2 weeks 37 C exchanges HsmtSSB HsEcSSB. <br />                                                                        products diluted addition 40 &#194;&#181;l 1 M triethanolamine plasmids used overproduction SSB protein carry          pH 8.5, 20 &#194;&#181;l dimethylsuberimidate (30 mg/ml water) 130 &#194;&#181;l <br /> respective gene control &#206;&#187;PL-promotor.               water. Crosslinking performed 3 h incubation transformation E.coli TGE900 (12) carrying thermo-            room temperature products analyzed SDS&#226;&#8364;&#8220;PAGE <br /> sensitive &#206;&#187;cI857 repressor protein, production induced        (12%) (22). <br /> temperature shift 30 42 C cells harvested 3 h induction. sequence ssb gene confirmed       Complementation ssb defective strains <br /> plasmid prepared aliquot cells taken induction.                                                             complementation analysis E.coli strains used.    Wild-type EcoSSB HsEcSSB prepared using                    RDP268 (23) chromosomal ssb gene deleted SSB <br /> poly(ethyleneimine) precipitation protein subsequent        function restored pACYCssb plasmid conferring chloram- <br /> extraction 0.4 M NaCl described Lohman et al. (13).         phenicol resistance. CS149 carries ssb-3 mutation (G15D) deletion mutant EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 prepared like                  (24) chromosome rendering cells unable survive wild-type EcoSSB extracted poly(ethylene-       presence 0.9 &#194;&#181;g/ml mitomycin C. strains transformed <br /> imine) precipitation 0.8 M NaCl. HsmtSSB prepared          plasmids containing respective ssb genes. high copy <br /> described earlier (9). EcoSSB mutants lacking acidic           number plasmids respective overproduction vectors (pSF1 <br /> region 10 amino acids purified like HsmtSSB.          derivatives v.s.) used. defective strains PL <br />                                                                        promotor repressed leading high intracellular SSB <br /> Determination concentrations                                        concentrations. Monocopy plasmids constructed follows: <br />                                                                        pSBL5 (25) derivative mini-F plasmid pRE432 (26) <br /> Protein concentrations given units tetramers      conferring ampicillin resistance contains wild-type text. determined spectrophotometrically using        EcoSSB gene control natural promotor. following absorption coefficients 280 nm: &#206;&#181; = 113 000 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1       plasmid nucleic acid sequence coding 160 C-terminal wild-type EcoSSB (14) EcoSSB Q152*, &#206;&#181; =                        amino acids wild-type EcoSSB replaced respective <br /> 88 800 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1 EcoSSB G117* EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167,                   mutant gene fragments. Unfortunately absence suitable <br /> &#206;&#181; = 76 240 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1 HsmtSSB &#206;&#181; = 99 000 M&#226;&#8364;&#8220;1cm&#226;&#8364;&#8220;1              restriction sites exchanging complete EcoSSB gene <br /> HsEcSSB. Unless stated  absorption coefficients          precluded construction HsmtSSB HsEcSSB derivatives calculated known <span id='am-5' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> composition          pSBL5. pSBL5 monocopy vector ssb gene <br /> absorption coefficients aromatic amino acids (15).              control natural ssb promotor, intracellular SSB <br />  2708 Nucleic Acids Research, 1996, Vol. 24,  14 <br />  <br />                                                                                 fact 10 amino acids render SSB protein net <br />                                                                                 negatively charged DNA binding domains net <br />                                                                                 positively charged. proteins 98% pure judged                                                                                 Coomassie-stained denaturing PAGE, main impurity                                                                                 wild-type EcoSSB expressed chromosomal ssb gene.                                                                                 preparation HsmtSSB impurity removed                                                                                 poly(ethyleneimine) precipitation step. EcoSSB G117*                                                                                 EcoSSB Q152* precipitation did remove wild-type <br />                                                                                 EcoSSB indicating formation mixed tetramers (v ). <br />                                                                                 Sedimentation analysis analytical ultracentrifuge showed <br />                                                                                 proteins homotetramers. <br />  <br />                                                                                 Binding single-stranded DNA <br />  <br />  <br /> Figure 1. Schematic depiction SSB proteins investigated study.   binding SSB proteins poly(dT) observed Hatched: N-terminal (&#226;&#710;&#188;120 aa) EcoSSB. Cross-hatched: HsmtSSB.           inverse fluorescence titrations 0.3 M NaCl (Fig. 2). White: proline- glycine-rich region EcoSSB (&#226;&#710;&#188;50 aa). Black: acidic       binding poly(dT) intrinsic tryptophan-fluorescence C-terminal region EcoSSB (10 aa).                                            quenched &#226;&#710;&#188;90%; exact depending                                                                                 respective protein. highest quench shown proteins <br />                                                                                 contain tryptophan DNA binding domain. <br /> concentrations case correspond SSB level                    Tryptophan 135 EcoSSB resides proline glycine rich <br /> normal E.coli cell.                                                             region fluorescence scarcely quenched DNA <br />   ssb-3 complementation 100&#226;&#8364;&#8220;200 transformed cells                      binding (18). wild-type EcoSSB, EcoSSB Q152* grown plates containing ampicillin. Colonies replica plated             HsEcSSB quench. affinity proteins LB containing 0.9 &#194;&#181;g/ml mitomycin C incubated 37 C                    poly(dT) large allow discrimination binding <br /> overnight.                                                                      properties different proteins shape   &#226;&#710;&#8224;ssb complementation transformed cells grown                  fluorescence titrations (18). binding site sizes 65 nucleotides <br /> presence ampicillin omitting chloramphenicol.                                                                                 tetramer wild-type EcoSSB, EcoSSB Q152* HsEcSSB, <br /> subsequent inoculations 4 ml medium allowing cells                                                                                 59 nucleotides  C-terminal EcoSSB <br /> grow &#226;&#710;&#188;50 generations, 100&#226;&#8364;&#8220;200 cells plated. Clones <br />                                                                                 DNA binding domain (v ). sheer lost helper plasmid pACYCssb identified                                                                                 size region leads requirement larger stretches <br /> replica plating LB plates containing chloramphenicol                                                                                 poly(dT) reach DNA binding sites tetramer. <br /> ampicillin. <br />   plasmid tested complement &#226;&#710;&#8224;ssb &#226;&#710;&#188;25%                          indirect, sensitive, method compare colonies lost helper plasmid indicated chloramphenicol               affinities different SSB proteins single-stranded DNA <br /> sensitivity. case absence wild-type EcoSSB                   measurement ability proteins depress confirmed western blot analysis.                                             melting temperature double-stranded DNA. Differences                                                                                 affinities detectable room temperature amplified                                                                                 melting experiment strong decrease affinity RESULTS                                                                         increasing temperature (27). Figure 3 shows destabilization schematic depiction different deletion mutants                      poly(dA&#226;&#8364;&#8220;dT) different SSB proteins. <br /> EcoSSB, wild-type HsmtSSB chimeric HsEcSSB                                 C-terminally truncated proteins EcoSSB Q152* constructed work shown Figure 1.                                 EcoSSB G117* cause drastic reduction melting temperature <br />   C-terminally truncated proteins EcoSSB Q152*                          poly(dA&#226;&#8364;&#8220;dT). probably removal electrostatic <br /> EcoSSB G117* lack highly conserved region 10                    repulsions acidic C-terminus single-stranded <br /> amino acids complete C-terminal respectively.              DNA. effect observed HsmtSSB deletion mutant EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 amino acids 116&#226;&#8364;&#8220;167                     addition C-terminal EcoSSB strongly diminished <br /> missing, leading protein lacks proline-                      helix destabilization activity (HsEcSSB). <br /> glycine-rich region does contain 10 amino acids.                      deletion mutant EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 negatively <br /> HsmtSSB homologous N-terminal domain EcoSSB                        charged C-terminus directly fused DNA binding domain. corresponds EcoSSB G117*. chimeric protein                      room temperature mutant binds strongly HsEcSSB fusion protein composed complete HsmtSSB                    poly(dT) helix destabilizing activity melting experiment C-terminal EcoSSB. protein corresponds                 lowest proteins study. reflects EcoSSB containing N-terminal DNA binding domain,                        electrostatic repulsion acidic C-terminus proline- glycine-rich region acidic C-terminus 10                 single-stranded DNA strongly pronounced amino acids.                                                                    mutant. <br />   proteins overexpressed E.coli. proteins                   fact HsmtSSB does destabilize poly(dA&#226;&#8364;&#8220;dT) contained 10 amino acids EcoSSB (wild-type EcoSSB,                  strongly EcoSSB Q152* EcoSSB G117* reflection <br /> EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167, HsEcSSB) precipitated                              different structures DNA binding domains poly(ethyleneimine)  probably                proteins. <br />                                                                                                                                  2709 <br />  <br />                                                                                               Nucleic Acids <br />                                                                                               Nucleic       Research,1994, <br />                                                                                                       Acids Research, 1996,Vol.  24  <br />                                                                                                                             Vol.22,  114                   2709 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 3. Destabilization poly(dA&#226;&#8364;&#8220;dT) SSB proteins. Poly(dA&#226;&#8364;&#8220;dT) <br /> Figure 2. Fluorescence titrations poly(dT). Protein (0.3 &#194;&#181;M) titrated         (38 &#194;&#181;M) melted standard buffer containing 0.1 M NaCl presence poly(dT) standard buffer containing 0.3 M NaCl. solid lines                1.27 &#194;&#181;M protein.   Tm values: EcoSSB Q152*  : 35_C; EcoSSB (b): 47_C; <br /> represent theoretical binding isotherms according model Schwarz          HsEcSSB (c): 56_C; protein (d): 61_C. (B) Tm values: protein <br /> Watanabe (19) using following parameters:   EcoSSB (F): n = 65, K &#207;&#8240;              (d): 61_C; EcoSSB G117* (e): 38_C; HsmtSSB (f): 48_C; EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 <br /> = 2.4 &#195;&#8212; 108 M&#226;&#8364;&#8220;1, Qf = 0.88; EcoSSB Q152* (J): n = 65, K &#207;&#8240; = 1.6 &#195;&#8212; 108 M&#226;&#8364;&#8220;1,             (g): 59_C. <br /> Qf = 0.91; HsEcSSB (&#226;&#8482;&#166;): n = 65, K &#207;&#8240; = 1.7 &#195;&#8212; 108 M&#226;&#8364;&#8220;1, Qf = 0.85 (B) EcoSSB <br /> &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 (F): n = 59, K &#207;&#8240; = 1.9 &#195;&#8212; 108 M&#226;&#8364;&#8220;1, Qf = 0.95; HsmtSSB (J): n = 59, <br /> K &#207;&#8240; = 1.7 &#195;&#8212; 108 M&#226;&#8364;&#8220;1, Qf = 0.96; EcoSSB G117* (&#226;&#8482;&#166;): n = 59, K &#207;&#8240; = 1.1 &#195;&#8212; <br /> 108 M&#226;&#8364;&#8220;1, Qf = 0.99. discussed previously (18), relatively strong binding       vivo properties <br /> does allow interpretation figures K &#207;&#8240; exact numbers lower limits. contrast, values n Qf accurate &quot;2   Physiological abilities mutant proteins investigated &quot;0.02 respectively. <br />                                                                                       complementation E.coli strains defective lacking ssb <br />                                                                                       gene.  strain lacking ssb gene viable.  <br /> Binding single-stranded RNA                                                        used strain chromosomal ssb gene deleted                                                                                       substituted ssb gene coded plasmid conferring affinity EcoSSB single-stranded RNA weaker                          chloramphenicol resistance (23). plasmid replaced single-stranded DNA (3). differences affinity                     plasmid carrying mutant ssb gene different resistance <br /> detectable fluorescence titrations poly(dT) observed                      regarded respective mutant able complement &#226;&#710;&#8224;ssb. poly(rU). Figure 4 shows fluorescence titrations                             severe defect ssb gene ssb-3 (24) glycine 15 <br /> different proteins poly(rU).                                                     replaced aspartate leading high UV mitomycin C <br />    proteins missing acidic C-terminus (EcoSSB Q152*,                          sensitivity. Cells carrying defect viable EcoSSB G117*) higher affinity poly(rU) compared                          presence 0.9 &#194;&#181;g/ml mitomycin C. regard complementation wild-type EcoSSB. affinity deletion mutant                            positive plasmid coding mutant SSB confers <br /> EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 drastically reduced. reflects                       resistance concentration mitomycin C. influence electrostatic repulsion nucleic                         Complementation tested high cellular concentrations <br /> acid negatively charged C-terminus.                                           mutant SSB using expression plasmids repression <br />    holds comparison HsmtSSB HsEcSSB                           PL promotor. lower amounts mutant SSB used affinities proteins poly(rU) weak                   monocopy plasmid ssb gene control marginal binding observed fluorescence titrations.                 chromosomal E.coli ssb promotor. <br />  2710 Nucleic Acids Research, 1996, Vol. 24,  14 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 5. Subunit exchange wild-type mutant EcoSSB proteins <br />                                                                                  detected <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> crosslinking. Lane 1: EcoSSB Q152* wild-type <br />                                                                                  EcoSSB, lane 2: EcoSSB Q152*, lane 3: EcoSSB G117*, lane 4: EcoSSB <br />                                                                                  G117* wild-type EcoSSB, lane 5: wild-type EcoSSB, lane 6: molecular <br />                                                                                  mass standard, lane 7: EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167, lane 8: EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167                                                                                  wild-type EcoSSB. Bands corresponding heterodimers indicated arrows. <br />  <br />  <br />                                                                                  envisage EcoSSB G15D complementing                                                                                  lethal mutations EcoSSB Q152* EcoSSB G117* .                                                                                  mechanism, high concentrations EcoSSB Q152*                                                                                  EcoSSB G117* dilute intact C-terminal regions extent <br />                                                                                  correct EcoSSB function suppressed (cf. Table 1). <br />  <br />                                                                                  Table 1. Complementation ssb defective strains <br />  <br />                                                                                   Protein              Complementation <br />                                                                                                        ssb-3                         &#226;&#710;&#8224;ssb <br />                                                                                                        monocopy      high copy       monocopy     high copy <br />                                                                                   EcoSSB-wt            +             +               +            + <br />                                                                                   EcoSSB Q152*         +             &#226;&#8364;&#8220;               &#226;&#8364;&#8220;            &#226;&#8364;&#8220; <br /> Figure 4. Fluorescence titrations poly(rU). Protein (0.3 &#194;&#181;M) titrated    EcoSSB G117*         +             &#226;&#8364;&#8220;               &#226;&#8364;&#8220;            &#226;&#8364;&#8220; poly(rU) standard buffer containing 0.2 M NaCl. Solid lines represent <br /> theoretical binding isotherms calculated according model Schwarz    EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167      +             +               +            + <br /> Watanabe (19) using following parameters:   EcoSSB (F): n = 65, <br /> K &#207;&#8240; = 7.0 &#195;&#8212; 10 6 M&#226;&#8364;&#8220;1 , Qf = 0.64; HsmtSSB (J): n.d.; HsEcSSB (&#226;&#8482;&#166;): n.d.            HsmtSSB              n.d.          &#226;&#8364;&#8220;               n.d.         &#226;&#8364;&#8220; <br /> (B) EcoSSB Q152* (J): n = 65, K &#207;&#8240; = 2.1 &#195;&#8212; 107 M&#226;&#8364;&#8220;1, Qf = 0.67; EcoSSB              HsEcSSB              n.d.          &#226;&#8364;&#8220;               n.d.         &#226;&#8364;&#8220; <br /> &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 (F): n.d.; EcoSSB G117* (&#226;&#8482;&#166;): n = 59, K &#207;&#8240; = 2.6 &#195;&#8212; 107 M&#226;&#8364;&#8220;1, Qf = <br /> 0.71. Note titration points HsmtSSB, HsEcSSB EcoSSB <br /> &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 located nearly straight lines bending saturation accessible range poly(rU) concentration. indicates HsmtSSB EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167 bind weakly             Subunit exchange <br /> reasonable fluorescence quench poly(rU). <br />                                                                                  Formation mixed tetramers demonstrated vitro                                                                                  chemical crosslinking subunits dimethylsuberimidate <br />                                                                                  analyzing products denaturing PAGE. Figure 5 (lanes <br />    Table 1 shows results complementation experiments.               2, 3, 5 7) Figure 6 (lanes 1, 3 6) Wild-type EcoSSB used control complement                     chemical crosslinking homotetramers leads appearence case. mutant showing complementation                     covalently connected dimers, trimers tetramers addition <br /> ability wild-type EcoSSB EcoSSB &#226;&#710;&#8224;116&#226;&#8364;&#8220;167. &#226;&#710;&#8224;ssb                      monomeric band. supports result analytical complemented proteins. conclude                    ultracentrifugation proteins study form stable proline- glycine-rich region C-terminal             homotetramers (v.s.). C-terminal EcoSSB not essential vivo DNA binding domain                         required tetramer formation. <br /> HsmtSSB replace corresponding binding domain                          Formation heterotetramers experiment best EcoSSB.                                                                          demonstrated appearence additional covalently <br />    Small amounts EcoSSB Q152* EcoSSB G117*,                               linked heterodimer. different SSB proteins form tetramers  complement N-terminal mutation G15D.                        possible combinations chemical crosslinking leads likely mechanism complementation formation                     trimeric tetrameric species. products mixed tetramers containing intact N-terminal domains                    resolved gel. C-terminally modified <br /> mutants EcoSSB Q152* EcoSSB G117* respectively                        EcoSSB-proteins able exchange subunits wild-type correct C-terminal region EcoSSB G15D.                      EcoSSB (Fig. 5, lanes 1, 4 8) human mitochondrial SSB <br />                                                                                                                              2711 <br />  <br />                                                                                           Nucleic Acids <br />                                                                                           Nucleic       Research,1994, <br />                                                                                                   Acids Research, 1996,Vol.  24  <br />                                                                                                                         Vol.22,  114                   2711 <br />  <br />                                                                                  mutations localized different regions EcoSSB                                                                                  overcome formation mixed tetramers. <br />                                                                                    number defined functional properties SSB <br />                                                                                  proteins assigned different structural regions. <br />  <br /> Figure 6. Subunit exchange EcoSSB, HsmtSSB HsEcSSB <br /> detected chemical crosslinking. Lane 1: wild-type EcoSSB, lane 2:             <br /> HsmtSSB wild-type EcoSSB, lane 3: HsmtSSB, lane 4: HsEcSSB               <br /> wild-type EcoSSB, lane 5: molecular mass standard, lane 6: HsEcSSB, lane 7:       <br /> HsEcSSB HsmtSSB. band corresponding heterodimer indicated           arrow. Lack heterodimer formation lanes 2 4 indicated     <br /> asterisk.                                                                          <br /> protein form mixed tetramers EcoSSB (Fig. 6, lanes 2                  4).  fusion EcoSSB C-terminal region                         <br /> HsmtSSB does impair heterotetramer formation                          <br /> HsmtSSB HsEcSSB (Fig. 6, lane 7). results                       subunit interaction restricted N-terminal DNA                       <br /> binding domain.  N-terminal domains procaryotic                       eucaryotic mitochondrial SSB proteins different                      <br /> allow subunit interaction.                                                          <br />                                                                                     C-terminal EcoSSB essential                        <br /> binding nucleic acids tetramer formation. negative                <br /> charges 10 amino acids weaken binding                     <br /> protein nucleic acids. electrostatic repulsion depends                distance charges N-terminal binding domain               amino acid sequence.                                                     <br />                                                                             <br />    10 amino acids C-terminal                    <br /> essential vivo function. Mutant SSB proteins missing               <br /> acidic region functional E.coli cell. vital role    10 amino acids function different                    <br /> DNA binding, probably protein&#226;&#8364;&#8220;protein interactions.                      <br />                                                                                    <br /> sequence DNA binding domain 10 amino                   <br /> acids serves spacer keeping negative charges away                 bound DNA.                                                                <br />   DNA binding tetramer formation localized                  <br /> N-terminal domain. Structural differences domain                 <br /> eucaryotic mitochondrial procaryotic SSB proteins prohibit          <br /> interactions heterologous subunits. phenotypical effect       <br /> </body></html>