<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>et al. <br /> Vitkup <br /> 2006 <br /> Volume 7, Issue 5, Article R39                                                                                               Open Access <br /> Research <br /> Influence metabolic network structure function enzyme <br />  <br />  <br />  <br />  <br />                                                                                                                                                              <span id='am-75' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br /> evolution <br /> Dennis Vitkup*, Peter Kharchenko&#226;&#8364;&#160; Andreas Wagner&#226;&#8364;&#161; <br /> Addresses: *Center Computational Biology Bioinformatics, Department Biomedical Informatics, Columbia University, Russ Berrie <br /> Pavilion, St Nicholas Avenue, New York, NY 10032, USA. &#226;&#8364;&#160;Department Genetics, New Research Building, Ave Louis Pasteur, Harvard Medical <br /> School, Boston, MA 02115, USA. &#226;&#8364;&#161;Department Biology, Castetter Hall, University New Mexico, Albuquerque, NM 87131, USA. <br />  <br /> Correspondence: Dennis Vitkup. Email: dv2121@columbia.edu <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reviews <br /> Published: 9 2006                                                         Received: 6 September 2005 <br />                                                                               Revised: 9 January 2006 <br /> Genome Biology 2006, 7:R39 (doi:10.1186/gb-2006-7-5-r39) <br />                                                                               Accepted: 7 April 2006 electronic version article complete online http://genomebiology.com/2006/7/5/R39 <br />  <br /> &#194;&#169; 2006 Vitkup et al.; licensee BioMed Central  <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reports open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br /> ture <br /> &lt;p      function <br />              enzyme <br />       andanalysis <br /> Metabolic          ofof  metabolicconstraints, <br />                      evolutionary <br />                       evolution         network shapes <br />                                                     gene                                                          duplication <br />                                                               evolutionand <br />                                                                          essentiability <br />                                                                             enzymes.&lt;/p&gt; <br />                                                                                            yeast metabolic network demonstrates struc- <br />  <br />  <br />  <br />  <br />                  Abstract <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br />                  Background: studies molecular evolution focused individual genes proteins. <br />                   understanding design principles evolutionary properties molecular networks <br />                  requires wide perspective. present work connect molecular evolution                  gene level properties cellular metabolic network. contrast protein interaction <br />                  networks, previous studies investigated molecular evolution proteins, <br />                  metabolic networks relatively defined global function. ability consider fluxes                  metabolic network allows relate functional role enzyme network rate <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />                  evolution. <br />  <br />                  Results: results, based yeast metabolic network, demonstrate important <br />                  evolutionary processes, fixation single <span id='am-60' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-61' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-65' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> mutations, gene duplications,                  gene deletions, influenced structure function network. Specifically, central                  highly connected enzymes evolve slowly connected enzymes.  enzymes <br />                  carrying high metabolic fluxes natural biological conditions experience higher evolutionary <br />                  constraints. Genes encoding enzymes high connectivity high metabolic flux higher <br />                  chances retain duplicates evolution. contrast protein interaction networks, highly <br />                                                                                                                                                              interactions <br />                  connected enzymes likely essential compared connected enzymes. <br />  <br />                  Conclusion: presented analysis evolutionary constraints, gene duplication, essentiality <br />                  demonstrates structure function metabolic network shapes evolution                  enzymes. results underscore need systems-based approaches studies molecular <br />                  evolution. <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br /> Background                                                                     duplications influence structure evolution Molecular networks genes encoding building                       networks. Conversely, network function constrain blocks represent different levels biological organiza-                  kinds mutations tolerated, genes <br /> tion interact evolution. hand, genetic                      evolve. Existing work structure evolution molec- <br /> changes point mutations, gene deletions, gene                      ular networks mainly focused protein interaction <br />  <br />                                                               Genome Biology 2006, 7:R39 <br />  R39.2 Genome Biology 2006,   Volume 7, Issue 5, Article R39   Vitkup et al.                                          http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        networks [1-6]. networks heterogeneous:                                6 <br />        contain large macromolecular complexes, regulatory interac- <br />        tions, signaling interactions, interactions proteins        provide structural support cell. result, difficult <br />                                                                                                  5 <br />        ascertain network structure reflects network function. <br />        large fraction false positives false negatives pro- <br />  <br />  <br />  <br />  <br />                                                                                Mean centrality <br />        tein interaction networks [7,8] complicates struc- <br />        ture function analysis. contrast, cellular metabolic                                4 <br />        networks relatively characterized model <br />        organisms Saccharomyces cerevisiae [9,10]        Escherichia coli [11]. function - biosynthesis energy                           3 <br />        production - understood, relationship        network structure network function. <br />  <br />        present study, ask topology metabolic                              2 <br />                                                                                                      0   10     20           30         40        50 <br />        network metabolic fluxes  metabolic flux rate <br />                                                                                                               Enzyme connectivity <br />        <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> reaction converts reactants prod- <br />        ucts) reactions network influence evolution                                                                                         Figure <br />                                                                                    correlation <br />                                                                                metabolic1 network <br />                                                                                                enzyme connectivity centrality yeast <br />        metabolic network genes point mutations                  correlation enzyme connectivity centrality yeast <br />        gene duplication. results suggest network                 metabolic network. Spearman&apos;s rank correlation r = -0.74, P &lt; 0.0001; <br />        structure function need understood fully appre-            Pearson&apos;s correlation r = -0.67, P &lt; 0.0001. centrality enzyme                                                                                equal mean length network distances enzyme        ciate metabolic networks constrain evolution           enzymes networks (pairs enzymes connected        parts. present study possible recent            path network excluded calculation). <br />        publication comprehensive compendium metabolic <br />        reactions yeast Saccharomyces cerevisiae [10].        compendium comprises 1,175 metabolic reactions 584                  connected enzymes direct access network <br />        metabolites, involves 16% yeast genes.                 nodes consequently shorter path lengths                                                                                enzymes network. evolutionary constraints        Using stoichiometric equations chemical               metabolic enzyme estimated normalized <br />        reactions, calculate connectivity enzyme            ratio non-synonymous synonymous substitutions        number metabolic enzymes produce consume               nucleotide site (Ka/Ks) occurred gene coding        enzyme&apos;s products reactants  Materials meth-             enzyme [13]. small Ka/Ks ratio suggests higher evolu- <br />        ods Additional data file 1). words, metabolic            tionary constraints enzyme,  smaller fraction <br />        enzyme metabolic enzyme B connected                 accepted <span id='am-68' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-69' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-71' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> substitutions. analysis, used <br />        share metabolite product reactant.              average ratio Ka/Ks unambiguous orthologs        Highly connected enzymes representation enzymes             sequenced Saccharomyces species: S. cerevisiae, S. para- <br />        share metabolites enzymes. Including               doxus, S. bayanus, S. mikatae [14]. average Ka/Ks <br />        highly connected metabolites cofactors             values used main analysis taken study        <span id='am-44' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-45' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span> hydrogen network representation render                Kellis et al. [14]. recalculated average ratios using <br />        network structure dominated nodes,                 maximum-likelihood method Yang Nielsen [15] <br />        obscure functional relationships enzymes.              obtained qualitatively similar results. <br />        excluded 14 highly connected metabolites: <br />        ATP, H, <span id='am-5' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span>, <span id='am-11' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span>pyrophosphate</span>, <span id='am-76' about='obo:IMR_0200102' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-77' property="oboInOwl:id" content="IMR:0200102" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Orthophosphate" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="KEGG:C00009" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orthophosphate</span>, <span id='am-28' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CO2</span>, NAD,                   Figure 2 demonstrates statistically significant negative cor- <br />        glutamate, NADP, <span id='am-16' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADH</span>, <span id='am-39' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>, <span id='am-21' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>AMP</span>, <span id='am-33' about='obo:IMR_0200075' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="rdfs:label" content="NH3" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200075" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00014" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NH3</span>, <span id='am-55' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-56' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span> [12].                   relation metabolic connectivity enzyme        results report qualitatively insensitive        ratio Ka/Ks (Spearman&apos;s rank correlation r = -0.20, P = 1.1 <br />        exact number removed metabolites.                                    &#195;&#8212; 10-4; Pearson&apos;s correlation r = -0.18, P = 7 &#195;&#8212; 10-4). inset <br />                                                                                Figure 2 shows negative association holds                                                                                broad range connectivities, caused        Results                                                                 small number highly connected proteins. Additional data <br />        Highly connected enzymes evolve slowly                                  file 2 demonstrates weaker negative correlation        discuss network structure - specifically,          non-synonymous (amino acid changing) substitutions Ka        enzyme&apos;s position network - influences enzyme evolu-             gene connectivity (Spearman&apos;s rank correlation r = -0.13, P = <br />        tion. Generally, enzymes central parts metabolism            1.6 &#195;&#8212; 10-2). reason using Ka, instead        tricarboxylic acid cycle neighbors           preferable Ka/Ks, measure evolutionary constraints <br />        enzymes peripheral metabolic pathways (Figure 1).                does compensate gene-specific differences synony- <br />        correlation shown Figure 1 arises fact            mous substitution rates introduces additional noise <br />  <br />                                                                Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                             Genome Biology 2006,              Volume 7, Issue 5, Article R39   Vitkup et al. R39.3 <br />  <br />  <br />  <br />  <br />                                                                                                      0.075 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            comment <br />               0.40 <br />  <br />  <br />  <br />  <br />                                                                                     Ka/Ks (median) <br />               0.35                                                                                   0.050 <br />  <br />  <br />  <br />  <br />               0.30                                                                                   0.025 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            reviews <br />               0.25 <br />                                                                                                      0.000 <br />    Ka/Ks <br />  <br />  <br />  <br />  <br />                                                                                                              0-10        10-20      20-30        30-40 <br />  <br />               0.20                                                                                             Enzyme connectivity <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            reports <br />               0.15 <br />  <br />  <br />               0.10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            deposited research <br />               0.05 <br />  <br />  <br />               0.00 <br />                          0               10               20               30                          40           50              60               70 <br />  <br />                                                              Enzyme connectivity <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            refereed research Figure <br />     relationship <br />         2        enzyme connectivity yeast metabolic network evolutionary constraint quantified Ka/Ks ratio relationship enzyme connectivity yeast metabolic network evolutionary constraint quantified Ka/Ks ratio. Spearman&apos;s rank <br /> correlation r = -0.20, P = 1.1 &#195;&#8212; 10-4; Pearson&apos;s correlation r = -0.18, P = 7 &#195;&#8212; 10-4. connectivity metabolic enzyme equal total number network enzymes producing consuming enzyme&apos;s reactants products. Ka fraction amino acid replacement substitutions amino acid replacement site DNA; Ks fraction silent substitutions silent site DNA. inset shows histogram binned enzyme <br /> connectivity versus median evolutionary constraint Ka/Ks (using data main figure). standard errors bin shown. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            interactions data. Additional data file 3 shows synonymous                        ratio Ka/Ks mRNA expression levels (Spearman&apos;s <br /> (silent) substitutions Ks enzyme connectivity sig-                   rank correlation r = -0.33, P = 5.5 &#195;&#8212; 10-10; Pearson&apos;s correla- <br /> nificantly correlated (Spearman&apos;s rank correlation r = 0.056,                    tion r = -0.30, P = 3.6 &#195;&#8212; 10-8). Information mRNA expres- <br /> P = 0.30). expected, synonymous substitutions                   sion metabolic genes obtained study cause amino acid changes selectively neu-                    Holstege et al. [16] number mRNA molecules <br /> tral purpose analysis.                                            cell estimated based microarray data.                                                                                  relatively weak correlation connectivity                                                                                                                                                                            information <br />  <br />  <br />  <br />  highly connected enzymes greater evolutionary                        expression levels (Spearman&apos;s rank correlation r = 0.11, P = <br /> constraint (smaller Ka/Ks)? possibility corre-                  4.6 &#195;&#8212; 10-2).  partial correlation analysis -  <br /> lation primarily mediated corresponding gene                           trolling mRNA expression levels - gene connec- <br /> expression level [3].  confirming previous observa-                       tivity evolutionary constraint Ka/Ks shows enzymes <br /> tions [3], significant negative correlation                   highly connected parts network evolve slowly <br />  <br />                                                                Genome Biology 2006, 7:R39 <br />  R39.4 Genome Biology 2006,    Volume 7, Issue 5, Article R39    Vitkup et al.                                              http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        Table 1 <br />  <br />        Correlation enzymatic flux magnitude evolutionary constraint Ka/Ks <br />  <br />        Uptake                                 Maximum uptake rates       Spearman&apos;s rank correlation (P value) zero      Spearman&apos;s rank correlation (P value) <br />                                                  (mmol/gDW/h)                                fluxes                                  zero fluxes <br />  <br />        Glucose/oxygen                               15.3/2.4                          -0.28 (P = 3.8 &#195;&#8212; 10-3)                          -0.25 (P = 3.6 &#195;&#8212; 10-6) <br />  <br />        Glucose/oxygen                               15.3/0.2                          -0.31 (P = 1.7 &#195;&#8212; 10-3)                          -0.22 (P = 5.7 &#195;&#8212; 10-5) <br />        Glucose/oxygen                               15.3/0.01                         -0.26 (P = 9.3 &#195;&#8212; 10-3)                          -0.21 (P = 1.2 &#195;&#8212; 10-4) <br />        Fructose/oxygen                              15.3/6.0                          -0.27 (P = 6.4 &#195;&#8212;   10-3)                        -0.20 (P = 2.5 &#195;&#8212; 10-4) <br />        Fructose/oxygen                              15.3/0.2                          -0.25 (P = 1.3 &#195;&#8212;   10-2)                        -0.20 (P = 1.8 &#195;&#8212; 10-6) <br />        Acetate/oxygen                               10.0/2.4                            -0.08 (P = 0.45)                              -0.21 (P = 9.2 &#195;&#8212; 10-5) <br />        Acetate/oxygen                                5.0/5.0                            -0.010 (P = 0.39)                             -0.19 (P = 3.7 &#195;&#8212; 10-4) <br />  <br />        correlation enzymatic flux magnitude evolutionary constraint Ka/Ks calculated enzymes carrying zero fluxes. <br />        gDW, grams dry weight. <br />        independent expression levels (Spearman&apos;s partial correla-                 tive correlation flux individual enzymatic <br />        tion r = -0.18, P = 1.4 &#195;&#8212; 10-3; P value Spearman&apos;s partial            reactions ratio Ka/Ks (Spearman&apos;s rank correlation r <br />        correlation estimated randomization).                                  = -0.31; P = 1.7 &#195;&#8212; 10-3; Pearson&apos;s correlation r = -0.24, P = 1.7 <br />                                                                                      &#195;&#8212; 10-2). correlation clearly non-linear, expo- <br />        Enzymes carry large metabolic fluxes evolve                              nential shape. results summarized Table 1        slowly                                                                        similar associations exist flux evolutionary  <br />        metabolic network supports cell growth                      straints Ka/Ks growth conditions glucose fruc- <br />        computationally quantified apparatus meta-                     tose - natural carbon sources yeast. Interestingly,        bolic flux analysis [17]. flux balance analysis,                   correlations evolutionary constraint Ka/Ks flux <br />        straints imposed stoichiometry reversibility                        substantially lower, statistically insignificant, ace- <br />        chemical reactions used restrict space feasible                 tate, carbon source dominate natural yeast <br />        metabolic fluxes. constrained subjected                  environment [21]. correlation        optimization procedure obtain flux distribution                  flux magnitude connectivity (results shown), evo- <br />        maximizes desirable metabolic property. cellu-                   lutionary constraints high fluxes complementary        lar growth-rate important component fitness                 connectivity constraints described (Figure 2). <br />        single-cell organism, biomass production used        property optimized. predictions flux balance                     Gene duplication correlation connectivity flux <br />        analysis good agreement experimental                        Gene duplications effects opposite        results E. coli [18,19] S. cerevisiae [20].                           amino acid changes: increase reduce <br />                                                                                      flux enzymatic reaction. established        relate metabolic flux rate enzyme evolution,                 highly connected enzymes enzymes high associated <br />        used flux balance analysis calculate metabolic fluxes               flux especially sensitive amino acid changes (Figures 2 <br />        yeast metabolic network [10], maximizing growth                    3). enzyme-coding genes, conversely,        different carbon sources (Table 1). Specifically, asked                    likely undergo duplication? Figure 4 shows        flux enzymatic reactions associated                   case enzyme connectivity. figure demon- <br />        evolutionary constraint Ka/Ks corresponding                        strates association enzyme-coding gene&apos;s <br />        enzyme-coding genes. analysis, enzymes catalyzing                 number duplicates enzyme connectivity  enzymes <br />        different chemical reactions, used reaction                   sequence identity higher 40% considered        largest flux; enzyme isoenzymes (enzymes                    duplicates). Mean connectivity genes duplicates        catalyzing reaction), used isoenzyme                 15.0, genes duplicates 19.2 (non-parametric <br />        smallest ratio Ka/Ks. growth conditions used vary                   Wilcoxon test, P = 1.4 &#195;&#8212; 10-4). result suggests dupli- <br />        available carbon sources different uptake rates                cates enzymes producing consuming widely used metab- <br />        oxygen (Table 1). calculated distribution flux values               olites likely retained evolution. Figure 5        metabolic network highly non-uniform [11]                 Additional data file 5 demonstrate similar association <br />        fluxes - usually representing glycolytic enzymes -                  exists non-zero enzymatic flux reaction <br />        orders magnitude larger rest (Additional data                 number duplicates respective enzyme&apos;s cod- <br />        file 4). eliminate disproportionate effect large              ing gene. Specifically, higher flux reaction, <br />        fluxes, removed fluxes orders magni-                   duplicates enzyme-coding gene  Qualitative <br />        tude larger median metabolic flux network                     association enzymatic flux gene duplication        (similar results obtained fluxes). Figure 2 demon-               recently shown Papp et al. [22]. <br />        strates, aerobic growth glucose, significant nega- <br />  <br />                                                                  Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                                                            Genome Biology 2006,                                               Volume 7, Issue 5, Article R39            Vitkup et al. R39.5 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             (b) <br />                                              1.2                                                                                                                1 <br />  <br />  <br />  <br />  <br />                                                                                                                    Metabolic flux magnitude (mmol/gDW/hour) <br />   Metabolic flux magnitude (mmol/gDW/hour) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     comment <br />                                              1.0 <br />  <br />  <br />  <br />                                              0.8                                                                                                               0.1 <br />  <br />  <br />                                              0.6 <br />  <br />  <br />  <br />                                              0.4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     reviews <br />                                                                                                                                                               0.01 <br />  <br />  <br />                                              0.2 <br />  <br />  <br />  <br />                                              0.0 <br />                                                                                                                                                                  0.00       0.05         0.10           0.15           0.20         0.25 <br />                                                0.00      0.05      0.10           0.15   0.20         0.25 <br />  <br />                                                                           Ka/Ks                                                                                                                 Ka/Ks <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     reports <br /> Figure  relationship <br />          3        metabolic flux evolutionary constraint relationship metabolic flux evolutionary constraint.  relationship metabolic flux values evolutionary constraint Ka/Ks aerobic growth glucose. (maximal uptake rate glucose 15.3 mmol/g dry weight (DW)/h; maximal oxygen uptake 0.2 mmol/gDW/h). Spearman&apos;s <br /> rank correlation r = -0.30; P = 2.7 &#195;&#8212; 10-3; Pearson&apos;s correlation r = -0.24, P = 1.7 &#195;&#8212; 10-2. metabolic fluxes calculated using flux balance analysis maximize cell growth rate. Fluxes orders magnitude larger median non-zero flux - representing large glycolytic fluxes - excluded analysis. (b)   using log coordinates metabolic flux magnitude. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     deposited research <br />                                                                                                                                                                      1.6 <br />                                                    1.4                                          &gt;30 <br /> Average number gene <br />  <br />  <br />  <br />  <br />                                                                                                                Average number gene <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      1.4 <br />                                                                                                                  duplications  bin) <br />  <br />  <br />  <br />  <br />                                                    1.2 <br />                                                                                                                                                                      1.2 <br />                                                    1.0 <br />       duplications <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      1.0 <br />                                                                               15-30 <br />                                                    0.8 <br />                                                            0-15                                                                                                      0.8 <br />                                                    0.6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     refereed research <br />                                                                                                                                                                      0.6 <br />  <br />                                                    0.4 <br />                                                                                                                                                                      0.4 <br />  <br />                                                    0.2                                                                                                               0.2 <br />  <br />                                                    0.0                                                                                                               0.0 <br />                                                             1                 2                 3 <br />                                                                                                                                                                            0.001-0.01     0.01-0.1             0.1-1          &gt;1 <br />                                                                   Enzyme connectivity                                                                                       1      Metabolic <br />                                                                                                                                                                                          2   flux magnitude <br />                                                                                                                                                                                                      3                         4 <br />  Figure <br />     relationship  enzyme <br />          4 corresponding <br /> duplications                  enzyme-coding <br />                                   connectivitygenes <br />                                                average number relationship enzyme connectivity average number                                         reaction <br />                                                                                                                                                                                                                               gene <br />                                                                                                                Figure <br />                                                                                                                    relationship betweenofthe <br />                                                                                                                     and5 magnitude     thenumber <br />                                                                                                                                                 metabolic <br />                                                                                                                                                        duplicates <br />                                                                                                                                                           flux                                                                                                                                                                                                                                                                                                                                              enzyme-coding <br />                                                                                                                                                                             enzymatic <br />                                                                                                                                                                                                                                                     interactions <br /> duplications corresponding enzyme-coding genes. Enzymes                                                relationship number duplicates enzyme-coding <br /> sequence identity larger 40% 100 aligned amino acids                                         gene magnitude metabolic flux enzymatic considered duplicates.                                                                                 reaction. results shown aerobic growth glucose (maximal <br />                                                                                                                uptake rate glucose 15.3 mmol/gDW/h; oxygen 0.2 mmol/gDW/h). <br />                                                                                                                Putative duplicate pairs 40% amino acid similarity Connectivity, essentiality, metabolic robustness                                                           100 aligned amino acid residues excluded. <br /> Evolutionary constraints enzymes indirect indicators metabolic robustness amino acid changes, changes                                                    fitness studied [23,24]. studies determine essen- metabolic network tolerated millions years                                                  tial genes,  genes elimination lab- <br />                                                                                                                                                                                                                                                     information <br />  <br />  <br />  <br />  evolution. type biological robustness                                                    oratory environments effectively lethal. use available complete gene deletions. Robustness gene                                                       essentiality data motivated observation highly <br /> deletions derived laboratory studies                                                  connected proteins protein interaction networks effects gene deletions growth rate indicators                                                  likely essential cell [1]. carried analyses <br />  <br />  <br />                                                                                                 Genome Biology 2006, 7:R39 <br />  R39.6 Genome Biology 2006,                         Volume 7, Issue 5, Article R39   Vitkup et al.                                http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />                                                                                                      essential. highly connected parts metabolic network, <br />                                                                                                      metabolic fluxes rerouted alternative path- <br />        Fraction essential genes <br />  <br />  <br />  <br />                                      0.25 <br />                                                                                                      ways loss function mutation [19,28,29]. does <br />                                                                                                      hold linear metabolic pathways metabolic net- <br />                                      0.20 <br />                                                                                                      work&apos;s periphery, loss function mutation                                                                                                      fatal rerouting possible. <br />                                      0.15 <br />  <br />  <br />  <br />                                      0.10 <br />                                                                                                      Discussion <br />                                                                                                      sum, demonstrate highly connected enzymes <br />                                      0.05 <br />                                                                                                      enzymes carry high metabolic fluxes yeast <br />                                                                                                      metabolic network tolerated fewer amino acid substitu- <br />                                      0.00                                                            tions evolutionary history. enzymes carrying <br />                                             0-12                  12-24              &gt;24 <br />                                                                                                      larger fluxes constrained? likely answer comes <br />                                                      Enzyme connectivity <br />                                                              2                        3              observation mutations affecting enzy- <br />                                                                                                      matic activity reduce increase flux. Enzymes <br />        Figure <br />        relationship <br />                6         enzyme connectivity gene essentiality                           carrying high fluxes tend reaction products enter <br />        relationship enzyme connectivity gene essentiality.                       large number metabolic pathways. Consequently, muta- <br />        connectivity metabolic enzyme equal total number        network enzymes producing consuming enzyme&apos;s reactants                             tional reduction activity enzymes        products. information gene essentiality obtained                          detrimental reduction activity enzymes <br />        systematic gene deletion study Giaever et al. [23] using SGD                           lower flux. <br />        database [25]. <br />                                                                                                      genes encoding enzymes high flux <br />        using data essential genes derived large scale gene                                 duplicates. Importantly, argue        deletion study Giaever et al. [23], used Saccharo-                                 duplications arise frequently genes products <br />        myces genome database (SGD) [25] collect essentiality                                  carry high flux, duplications likely        data.                                                                                         preserved evolution, advantage - higher <br />                                                                                                      flux - provide. gene&apos;s duplicates initially                                                                                                      preserved advantageous increase metabolic <br />        analyses essential enzyme-coding genes                                                                                                      flux, divergence provide functional ben- <br />        clearest deviations behavior observed protein interac- <br />                                                                                                      efits [30]. Divergence metabolic genes expression <br />        tion networks. metabolic network                                                                                                      regulation established gene intensely stud- <br />        highly connected metabolites  ATP hydrogen)                                                                                                      ied parts metabolism, tricarboxylic acid cycle <br />        excluded, mean connectivity essential enzymes                                                                                                      enzymes [31]. <br />        significantly smaller mean connectivity non- <br />        essential enzymes (Figure 6). yeast network, mean <br />                                                                                                      association predicted enzymatic <br />        connectivity essential enzymes 13.2, non-essen- <br />                                                                                                      flux evolutionary rate pronounced carbon <br />        tial enzymes 17.5 (non-parametric Wilcoxon test, P = 4.0 &#195;&#8212; 10- <br />        4). statistically significant difference connectivity                                sources dominate natural environment yeast.                                                                                                      suggests use association flux        essential non-essential enzymes observed                                                                                                      evolutionary constraint search conditions domi- <br />        metabolites used establish network connections.  <br />                                                                                                      nated evolution metabolic networks. Similar analyses, <br />        sequently, highly connected metabolic enzymes                                                                                                      use genomic data infer environment        likely essential low connected enzymes. Similarly, <br />                                                                                                      shaped organism&apos;s evolution, used        Mahadevan et al. [26] demonstrated recently, removal                                                                                                      carbon limitation influenced evolu- <br />        highly connected metabolites essential                                                                                                      tion E. coli metabolic network strongly        removal low connected metabolites.  <br />                                                                                                      nitrogen limitation [19], yeast evolution <br />        highly connected enzymes enzymes carrying high fluxes <br />                                                                                                      favored fermentation respiration [32]. <br />        likely duplicates  sim- <br />        ilar biochemical function). suggests highly  <br />                                                                                                      previous study Hahn et al. [6] reported  based        nected enzymes likely essential                                                                                                      amino acid divergence, E. coli metabolic network        duplicates compensate loss  <br />                                                                                                      exists statistically significant association enzyme <br />        function mutations [27].  average <br />                                                                                                      connectivity evolutionary constraint. emphasize        number duplicates essential metabolic enzymes 0.19 <br />                                                                                                      contradiction earlier work results        average number duplicates non-essential <br />                                                                                                      apparent.  earlier study based        enzymes 0.8 (non-parametric Wilcoxon test, P = 8 &#195;&#8212; 10-49). <br />                                                                                                      smaller set enzymes (n = 108 opposed n = 350 , <br />        addition gene duplications, flux rerouting provide <br />                                                                                                      statistical power.  differ- <br />        mechanism make highly connected genes  <br />                                                                                      Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                 Genome Biology 2006,   Volume 7, Issue 5, Article R39   Vitkup et al. R39.7 <br />  <br />  <br />  <br />  <br /> ent statistical measures previous study showed, like       Materials methods  negative association connectivity evo-        Metabolic network <br /> lutionary constraint, albeit P &lt; 0.05. Second,        used comprehensive collection yeast S. cerevisiae <br />  <br />  <br />  <br />  <br />                                                                                                                                                     comment lack sufficient sequence information closely         metabolic reactions Foster et al. [10] calculate metabolic <br /> related sister species E. coli, previous study used      enzyme connectivities. addition enzymatic reactions <br /> amino acid divergence Ka preferable Ka/Ks             assigned 671 open reading frames (ORFs), collection <br /> indicate evolutionary constraint. fact, correlation           contains reactions unassigned known ORFs, transport connectivity Ka similar              reactions, reactions represented large macromolecu- <br /> present study previous work (Spearman&apos;s rank corre-          lar complexes. reactions used calculate lation r = -0.13, P = 1.2 &#195;&#8212; 10-2 versus Spearman&apos;s rank         enzyme connectivities excluded main anal- <br /> correlation r = -0.15, P = 7 &#195;&#8212; 10-2 study Hahn et al.).    ysis. Large macromolecular complexes (containing  <br />  <br />  <br />  <br />                                                                                                                                                     reviews <br />                                                                      ORFs) represented single enzymatic nodes cal- surprising observed associations       culation connectivities metabolic enzymes. weak magnitude. reason low magnitude          order include functional relationships calcula- factors influence evolution enzyme-coding          tion enzyme connectivities, excluded 14 highly <br /> genes. factors gene expression levels (dis-         connected metabolites factors  described cussed paper) constraints stemming ter-          main text). result exclusion, small fraction (5%) <br /> tiary quaternary structure enzymes, differ          network enzymes disconnected network enzymes (little known constraints).           zero connectivity). enzymes  <br />  <br />  <br />  <br />                                                                                                                                                     reports <br /> key point factors, metabolic         included analysis. <br /> network function structure clear influence protein evolution.                                                   Flux balance analysis <br />                                                                      Flux balance analysis (FBA) used obtain metabolic flux results yeast metabolic network relate          distribution described previously [10,17,19]. network <br /> earlier work protein interaction networks?  similar       Forster et al. [10] used flux balance calculations. <br /> relationship protein connectivity evolutionary           silico network yeast metabolism includes central <br />  <br />  <br />  <br />  <br />                                                                                                                                                     deposited research <br /> constraint suggested [4,5];  association       carbon metabolism, transmembrane transport reactions, <br /> exists different reasons. Highly connected proteins pro-      pathways responsible synthesis degradation tein interaction networks evolve slowly larger         amino acids, nucleic acids, vitamins, cofactors, lipids. fraction highly connected protein&apos;s sequence involved        total, network consists 733 metabolites 1,175 met- protein interactions evolutionarily           abolic reactions. flux-balance analysis, constraints <br /> strained [4]. contrast, high protein connectivity met-     limiting nutrient uptake, reaction irreversibility, steady- <br /> abolic network established protein-protein            state conservation metabolite concentrations applied. <br /> interactions, consumption production               fluxes optimal growth obtained maximi- <br />  <br />  <br />  <br />  <br />                                                                                                                                                     refereed research <br /> widely used metabolites. metabolic networks, mutations         zation biomass production using linear optimization. Lin- <br /> enzyme-coding genes - changing reaction rates concen-            ear optimization performed using GNU Linear <br /> trations - especially deleterious consequences          Programming Kit [33]. <br /> widely used metabolites. Consequently, highly connected <br /> metabolic enzymes evolve slowly functional             Molecular evolution <br /> opposed structural constraints. ability consider           identified duplicates S. cerecisae genome using fluxes enzymes metabolic network allows           previously described genome analysis tool [34]. <br /> relate functional role enzyme network        Briefly, tool locates gene duplicates genome using <br /> rate evolution. functional analysis genome-scale      BLASTP [35] aligns globally Needleman <br /> network counterpart genome-scale net-            Wunsch dynamic programming alignment algorithm                             interactions <br /> work studied far.                                               [36]. Putative duplicate pairs 40% amino acid <br />                                                                      similarity 100 aligned amino acid residues conclusion, analysis evolutionary constraints, gene        excluded; remaining pairs calculated number duplication, essentiality demonstrates structure        substitutions synonymous site (Ks) number function metabolic network shapes evolution          substitutions non-synonymous site (Ka) using maxi- enzymes. long run, analyses biological          mum likelyhood models Muse Gaut [37] Goldman <br /> networks allow increasingly place evolution        Yang [38]. <br />                                                                                                                                                     information <br />  <br />  <br />  <br />  <br /> genes larger context operate, building <br /> blocks cellular networks.                                         average Ka/Ks, Ka, Ks values used analysis                                                                      obtained study Kellis et al. [14]. complemen- <br />                                                                      tary approach, recalculated average ratios using <br />  <br />  <br />  <br />                                                       Genome Biology 2006, 7:R39 <br />  R39.8 Genome Biology 2006,             Volume 7, Issue 5, Article R39               Vitkup et al.                                          http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        maximum-likelihood method Yang Nielsen [15]                                                   Comparative assessment large-scale data sets protein- <br />        obtained qualitatively similar results.                                                              protein interactions. Nature 2002, 417:399-403. <br />                                                                                                           8.    Spinzak E, Sattah S, Margalit H: reliable experimental <br />                                                                                                                 protein-protein interaction data ? J Mol Biol 2003, 327:919-923. <br />                                                                                                           9.    Karp PD, Paley S, Romero P: Pathway Tools software. Bioin- <br />                                                                                                                 formatics 2002, 18:S225-S232. <br />        Additional data files                                                                              10.   Forster J, Famili  Fu P, Palsson BO, Nielsen J: Genome-scale <br />        following additional data available online                                              reconstruction Saccharomyces cerevisiae metabolic <br />        version paper. Additional data file 1 figure showing                                        network. Genome Res 2003, 13:244-253. <br />                                                                                                           11.   Edwards JS, Palsson BO: Escherichia coli MG1655 silico <br />        examples metabolic connectivity.   example                                                metabolic genotype: <span id='am-38' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, characteristics,        metabolic reaction network sphingoglycolipid metabo-                                                capabilities. Proc Natl Acad Sci USA 2000, 97:5528-5533. <br />                                                                                                           12.   Kharchenko P, Vitkup D, Church GM: Filling gaps metabolic <br />        lism; metabolites drawn small circles (DHSP, sphinga-                                             network using expression information. Bioinformatics 2004, <br />        1-phosphate; PETHM, ethanolamine phosphate; <span id='am-81' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-82' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>SPH</span>,                                                    20:I178-I185. <br />        sphinganine; CDPETN, CDPethanolamine; ETHM, eth-                                                   13.   Li W-H: Molecular Evolution Sunderland: Sinauer Associates; 1997. <br />                                                                                                           14.   Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES: Sequencing <br />        anolamine) enzyme-encoding genes shown rectan-                                                comparison yeast species identify genes regu- <br />        gles. (b) Metabolic connectivity dpl1 gene (solid edges),                                         latory elements. Nature 2003, 423:241-254. <br />        defined reactions shown  . dpl1 gene                                      15.   Yang  Nielsen R: Estimating synonymous nonsynony- <br />                                                                                                                 mous substitution rates realistic evolutionary models. <br />        total metabolic connections: established                                              Mol Biol Evol 2000, 17:32-43. <br />        ethanolamine phosphate (red edges); sph-                                          16.   Holstege FC, Jennings  Wyrick JJ, Lee TI, Hengartner CJ, Green <br />                                                                                                                 MR, Golub TR, Lander ES, Young RA: Dissecting regulatory <br />        inganine 1-phosphate (blue edges). Metabolic connections                                                 circuitry eukaryotic genome. Cell 1998, 95:717-728. <br />        enzymes dashed edges. Additional                                         17.   Varma  Boesch BW, Palsson BO: Biochemical production capa- <br />        data file 2 demonstrates relationship enzyme                                                 bilites Escherichia coli. Biotech Bioeng 1993, 42:59-73. <br />                                                                                                           18.   Edwards JS, Ibarra RU, Palsson BO: silico predictions        connectivity average amino acid divergence Ka.                                                   Escherichia coli metabolic capabilites consistent        Spearman&apos;s rank correlation r = -0.13, P = 1.6 &#195;&#8212; 10-2. Addi-                                             experimental data. Nat Biotechnol 2001, 19:125-130. <br />                                                                                                           19.   Segre D, Vitkup D, Church GM: Analysis optimality natural <br />        tional data file 3 shows relationship enzyme                                             perturbed metabolic networks. Proc Natl Acad Sci USA 2002, <br />        nectivity average silent divergence Ks. Spearman&apos;s                                               99:15112-12117. <br />        rank correlation r = -0.056, P = 0.30. Additional data file 4                                   20.   Foster J, Famili  Palsson BO, Nielsen J: Large-scale evaluation                                                                                                                 silico gene deletions Saccharomyces cerevisiae. OMICS <br />        histogram calculated metabolic fluxes yeast                                              2003, 7:193-202. <br />        network aerobic growth glucose (maximal uptake rate                                         21.   Strathern JN, Jones EW, Broach JR: Molecular Biology        glucose 15.3 mmol/g dry weight/h; oxygen 0.2 mmol/g                                                  Yeast Saccharomyces. Metabolism Gene Expression Cold <br />                                                                                                                 Spring Harbor Press, NY; 1982. <br />        dry weight/h). Note small number fluxes - representing                                      22.   Papp B, Pal C, Hurst LD: Metabolic network analysis        glycolysis - disproportionately large magnitudes. Similar                                           causes evolution enzyme dispensability yeast. <br />                                                                                                                 Nature 2004, 429:661-664. <br />        flux distributions obtained growth condi-                                      23.   Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />        tions. Additional data file 5 shows correlation                                              Lucau-Danila  Anderson K, Andre B, et al.: Functional profiling <br />        non-zero enzymatic flux reaction number                                                Saccharomyces cerevisiae genome.                 Nature 2002, <br />                                                                                                                 418:387-391. <br />        duplicates respective enzyme&apos;s coding gene. Addi-                                        24.   Steinmetz LM, Scharfe C, Deutschbauer  Mokranjac D, Herman <br />        tional data file 6 provides connectivity evolutionary                                                ZS, Jones T, Chu  Giaever G, Prokisch H, Oefner PJ, Davis RW: <br />        parameters (Ka/Ks, Ka, Ks) yeast metabolic enzymes.                                                  Systematic screen human disease genes yeast. Nat <br />                                                                                                                 Genet 2002, 31:400-404. <br />        Histogram <br />        colipid <br />         growth <br />        Connectivity <br />         gene. <br />         tion <br />        0.30. <br />               Maximal <br />        -oxygen <br />         tudes. <br />         gene <br />         metabolic <br />         edges. <br />         nections: <br />         edges); <br />         Metabolic <br />           representing <br />            1.6 <br />                           aerobic <br />               enzyme-encoding <br />               correlation <br />               relationship <br />                              &#195;&#8212; <br />                shown <br />                         <br />        sphinganine <br />        silent <br />               Additional <br />        Click <br />        amino <br />        tions <br />        connectivity <br />               sphinganine; <br />        Examples <br />        =        example <br />                  acid                   10 <br />                 Similar <br />                   0.2 <br />                                                                               -2     file <br />                   conditions. <br />                     uptake <br />                         growth <br />                         mmol/g <br />                                                                               number <br />                                                                                                divergence <br />                  metabolism; <br />                           flux <br />                       enzymes. <br />                       enzymes <br />                       connections <br />                                              data                                 theevolutionary <br />                         divergence <br />                           1-phosphate; <br />                           metabolic <br />                            CDPETN, <br />                             established <br />                              glycolysis <br />                                . <br />                                                               file <br />                                 rate <br />                                                                                                  K <br />                                  calculated <br />                                                                      distributions <br />                                                                       dpl1 <br />                                    sThe <br />                                     .3 <br />                                      16 <br />                                       5 <br />                                       4 <br />                                       2dry <br />                                        metabolites <br />                                                                                Spearman&apos;s <br />                                                                                  metabolic <br />                                          glucose <br />                                          genes <br />                                           duplicates <br />                                           K <br />                                                                                     connectivity <br />                                            dpl1 <br />                                            CDPethanolamine; <br />                                            gene <br />                                            weight/h. <br />                                             aglucose <br />                                                    gene <br />                                                                                                        (solid <br />                                                       parameters <br />                                                                                                .sphinganine <br />                                                  enzyme <br />                                                  PETHM, <br />                                                                                                   Spearman&apos;s <br />                                                    metabolic <br />                                              non-zerowere <br />                                                                                                                  rank <br />                                                                                                                                                                             Note <br />                                                           enzymatic <br />                                                                                                               reaction <br />                                                        shown <br />                                                        15.3                                                            edges), <br />                                                             drawn <br />                                                               aalso <br />                                                              mmol/g <br />                                                                                                                                  rank         sphosphate <br />                                                                               number <br />                                                                               ,weight/h <br />                                                                     rectangles. <br />                                                       disproportionately <br />                                                           connectivity <br />                                                           ethanolamine <br />                                                            ethanolamine <br />                                                               enzymes <br />                                                                 1-phosphate <br />                                                                 correlation <br />                                                                 fluxes <br />                                                                  total <br />                                                                   respective <br />                                                                                                                                      network <br />                                                                      obtained <br />                                                                      (K <br />                                                                                                                                             small <br />                                                                       ETHM, <br />                                                                        correlation <br />                                                                                                                                                 ain <br />                                                                          flux <br />                                                                          dry <br />                                                                           small <br />                                                                           defined <br />                                                                           /K <br />                                                                                                                                                                                                                                                                                                              rK <br />                                                                                                                                                                   large <br />                                                                                   (blue <br />                                                                                   = <br />                                                                                  metabolic <br />                                                                                phosphate; <br />                                                                                   circles <br />                                                                                                                                                                enzyme&apos;s <br />                                                                                                                                                                                                                                                    yeast <br />                                                                                    (b) <br />                                                                                    forby   = <br />                                                                                           byof <br />                                                                                   ethanolamine) <br />                                                                                       -0.056, <br />                                                                                       Ksphingogly- <br />                                                                                         average <br />                                                                                         rother <br />                                                                                          smagni- <br />                                                                                           )Metabolic <br />                                                                                                                                                                                       edges). <br />                                                                                             network <br />                                                                                                                                                                                          DHSP, <br />                                                                                                                                                                                               dashed <br />                                                                                                coding <br />                                                                                                 (red <br />                                                                                                 reac- <br />                                                                                                 fluxes <br />                                                                                                  reac- <br />                                                                                                   <br />                                                                                                  yeast <br />                                                                                               -0.13, <br />                                                                                                   P                                                                                                     =P <br />                                                                                                   SPH,    25.   Dwight SS, Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, <br />                                                                                                                 Engel SR, Feierboch B, Fisk DG, Hirchman J, Hong EL, et al.: Saccha- <br />                                                                                                                 romyces genome database: underlying principles        Acknowledgements                                                                                         organisation. Brief Bioinform 2004, 5:9-22. <br />        thank Dr Andrey Rzhetsky, Dr Uwe Sauer, Dr Eugene Koonin                                26.   Mahadevan R, Palsson BO: Properties metabolic networks: <br />        valuable discussions. thank anonymous reviewers                                  structure versus function. Biophys J 2005, 88:L07-L09. <br />        helpful suggestions.                                                                          27.   Gu  Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH: Role                                                                                                                 duplicate genes genetic robustness null mutations. <br />                                                                                                                 Nature 2003, 421:63-66. <br />                                                                                                           28.   Wagner  Robustness mutations genetics net- <br />        References                                                                                               works yeast. Nat Genet 2000, 24:355-361. <br />        1.     Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central-                           29.   Edwards JS, Palsson BO: Robustness analysis Esherichia <br />               ity protein networks. Nature 2001, 411:41-42.                                                  coli metabolic network. Biotechnol Prog 2000, 16:927-939. <br />        2.     Hirsh AE, Fraser HB: Protein dispensability rate                                     30.   Kuepfer L, Sauer U, Blank LM: Metabolic functions duplicate <br />               evolution. Nature 2001, 411:1046-104 <br /> </body></html>