<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Letter <br />  <br /> Origins impact constraints evolution gene families <br /> Boris E. Shakhnovich1,3 Eugene V. Koonin2 <br /> 1 <br />  Bioinformatics Program, Boston University, Boston, Massachusetts 02215, USA; 2National Center Biotechnology Information, <br /> National Institutes Health, Bethesda, Maryland 20894, USA <br />  <br />  <br />         Recent investigations high-throughput genomic phenomic data uncovered variety significant         relatively weak correlations gene&#226;&#8364;&#8482;s functional evolutionary characteristics. particular, essential genes <br />         genes paralogs slight propensity evolve slowly nonessential genes singletons, <br />         respectively.  given weakness multiplicity associations, biological relevance remains <br />         uncertain.  existence essential paralog used specific strong gauge         selection. partition gene families genomes classes: include essential gene <br />         (E-families) essential genes (N-families). weaker purifying selection causes N-families <br />         evolve dynamic regime higher rates duplicate fixation pseudogenization. genes <br />         E-families subject significantly stronger purifying selection N-families, survive longer         exhibit greater sequence divergence. Longer average survival time allows divergence upstream regulatory <br />         regions, resulting change transcriptional context paralogs E-families. findings compatible <br />         differential division ancestral functions (subfunctionalization) emergence novel functions <br />         (neofunctionalization) prevalent modes evolution paralogs E-families opposed         pseudogenization (nonfunctionalization), typical fate paralogs N-families. Unlike         characteristics genes, essentiality, existence paralogs, expression level, membership E-family         N-family strongly correlates level selection appears major determinant gene&#226;&#8364;&#8482;s <br />         evolutionary fate. <br />         [Supplemental material available online www.genome.org.] <br />  nature connections organismal molecular                           theory molecular evolution, remains open question. evolution remains fundamental  generally, unanswered                           substantial disagreement correlation question. relationship evolution gene                       artifact caused transitive relationships expression level <br /> organism characterized change fitness precipi-                      (Pal et al. 2001) abundance (Drummond et al. 2005). tated deletion mutation gene (Drake et al. 1998;                       recent, sophisticated statistical analyses indepen- <br /> Keightley Eyre-Walker 1999). simple defi-                     dent groups suggest correlation gene&#226;&#8364;&#8482;s knockout <br /> nition, quantifying fitness effect gene deletions muta-                    effect purifying selection does exist; studies tions remains hard problem. principal difficulties                   shown correlation probably quite weak (Drummond changes deleterious conditions neutral                      et al. 2005; Zhang 2005; Wall et al. 2005). impor- beneficial conditions (Macarthur Levins                           tantly, observed correlations taught little evo- <br /> 1964; Levins 1968). result, notions fitness essen-                   lutionary dynamics governing gene duplication divergence. <br /> tiality inherently ambiguous.                                                          Gene duplication followed divergence pri- <br />       Driven recent availability complete genome                      mary driving forces functional innovation evolu- <br /> sequences, results genome-wide functional                         tion (Ohno 1970; Lynch Katju 2004). Currently, diver- <br /> says, researchers observed significant correlations                     gence paralogs duplication thought follow functional characteristics genes, essentiality (Hurst                 routes. common outcome duplication <br /> Smith 1999; Hirsh Fraser 2001; Yang et al. 2003), number                      event nonfunctionalization copy protein-interaction partners (Fraser et al. 2002), expression                     pseudogene  eventually, goes extinct (Nei Roychoud- <br /> level (Pal et al. 2001), rate evolution (equivalently                 houry 1973; Petrov Hartl 2000), second copy <br /> strength purifying selection).  reported correla-                     retains original function.  common  <br /> tions relatively weak, contribution character-                   theless crucial evolutionary scenarios neofunctionalization <br /> istic remains subject vigorous debate (Jordan et al. 2003; Wall                 subfunctionalization. case neofunctionalization, <br /> et al. 2005; Drummond et al. 2006). particular, validity                   paralog retains original function, observed negative correlation gene&#226;&#8364;&#8482;s knockout ef-                      evolves new function phase rapid, nearly neutral <br /> fect (essentiality) evolutionary rate, originally predicted                   evolution (Ohno 1970). subfunctionalization model, <br /> Kimura Ohta (Kimura 1981) corollary neutral                          multiple functions ancestral gene divided                                                                                      paralogs, evolve purifying selection (Force <br /> 3 <br />  Corresponding author.                                                               et al. 1999; Lynch Force 2000; Zhang 2005). Given <br /> E-mail Borya@acs.bu.edu; fax (617) 353-4814. <br /> Article published online print. Article publication <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> http://   sufficiently detailed information, probability functional <br /> www.genome.org/cgi/doi/10.1101/gr.5346206.                                           divergence occurred speciation event estimated <br />  <br />  <br /> 16:1529&#226;&#8364;&#8220;1536 &#194;&#169;2006 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/06; www.genome.org                           Genome Research          1529 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />  <br />  <br /> Figure 1. largest E- N-gene families yeast. family left contains essential genes (N-family), family right includes essential members (E-family). red circles represent essential genes, blue squares represent nonessential genes. E-family represented consists various mitochondrial membrane transporters, N-family includes seripauperins (poorly characterized multigene family repre- <br /> sented, primarily, subtelomeric regions yeast chromosomes). <br />  <br />  <br /> using statistical models sequence evolution (Gu 1999,                       true essential genes paralogs (Yang et <br /> 2001a,b).  general, relationship selec-                 al. 2003). independent line analysis indicated <br /> tion neofunctionalization understood. ex-                  genes paralogs, average, evolve slowly ample, paralogs experience period relaxed purify-                singletons (Davis Petrov 2004; Jordan et al. 2004), nature <br /> ing selection immediately duplication, genes paralogs               relationships variables gene&#226;&#8364;&#8482;s greater evolutionary conservation singletons (Davis                  essentiality, existence paralogs, rate evolution&#226;&#8364;&#8221; Petrov 2004; Jordan et al. 2004). unexpected              remains unclear. sought explore essentiality ing taken indicate  average, duplications               paralogy necessary sufficient primary determinant genes critical biological functions retained               purifying selection. end, need compare strength duplications biologically important genes.                        selection essential nonessential members       goal present study elucidate general               gene family. <br /> relationships functional constraints, differential                           Paralogy relationships genes genomes strengths purifying selection, gene duplication.               conveniently represented general form Diver- <br /> observe membership gene family good predictor                gence Diffusion Graph (DDG). vertices represent genes, <br /> selection. example, genes essential paralog               edges represent homology relationships weighed according stronger selection genes essential paralogs.                sequence similarity scores (Fig. 1; Supplemental ma- families paralogs include es-                terial). genes partitioned paralogous families (Har- <br /> sential gene (E-families) consist entirely non-              rison Gerstein 2002; Enright et al. 2003) finding strongly <br /> essential genes (N-families) evolve dramatically different regi-            connected components DDG,  sets vertices mens. genes E-families, average, evolve substan-                path exists pair  Supplemental mate- <br /> tially slowly genes N-families, E-families               rial). Partitioning paralogous families (strongly connected com- greater divergence paralogs. attrib-                ponents DDG) classes include uted significantly longer average survival time paralogs             essential gene (E-families) essential mem- E-families compared N-families. E-families appear              bers (N-families)&#226;&#8364;&#8221;allows compare essential nonessen- <br /> comprise reservoir genes evolution new functions               tial members family  Methods). species subfunctionalization neofunctionalization routes. Fi-                  genome-wide data gene essentiality available, nally, relationship evolution               E-family N-family sets included substantial numbers genes open reading frames upstream regions. Specifically,                    (Table 1), providing statistical comparison evolutionary <br /> genes E-families stronger selection evolve novel             characteristics essential nonessential genes transcriptional regulatory contexts.                                           E-families comparisons sets genes                                                                                E-families N-families. <br />  <br /> Results                                                                        Table 1. Numbers essential genes paralogs <br />                                                                                sizes E-families N-families <br /> Homogeneity strength selection paralogous <br /> families                                                                                                       Essential     Nonessential <br />                                                                                                essential   genes       genes         Genes essential genes tend evolve slowly  implication,               Species             genes       E-families     E-families       N-families <br /> appear stronger purifying selection nonessential <br /> ones (Hirsh Fraser 2001; Jordan et al. 2002), trend                S. cerevisiae      1032             95             183             656 <br />                                                                                C. elegans          861            158             119            1300 <br /> relatively weak. Furthermore, published reports suggest                   E. coli             688             24             328             521 negative correlation selection essentiality <br />  <br />  <br /> 1530      Genome Research <br />              www.genome.org <br />                                                                                                               Evolution gene families <br />  <br />  <br /> Table 2. Average strength purifying selection essential     strength selection correlates probability pseudo- <br /> nonessential genes, E-families N-families                  genization M. leprae. observed E- <br />                     Essential genes   Nonessential genes       P      families N-families evolve different strengths puri- <br />                                                                       fying selection, hypothesized exhibit <br /> SFP density            0.01567              0.02158          1e-20    different pseudogenization duplication rates. Identification <br /> Ka/Ks &#226;&#8364;&#8220; Yeast          0.10                 0.13             1e-8     pseudogenes (Harrison et al. 2001, 2002; Harrison Ger- <br /> Ka/Ks &#226;&#8364;&#8220; E. coli        0.054                0.099            1.6e-6   stein 2002) derived E-families N-families S. cerevisiae <br />                                                                       showed estimated pseudogenization rate E-families nonessential <br /> genes families      E-families          N-families                 approximately seven times lower N-families. <br /> SFP density            0.012               0.027             1e-40    high-precision estimate Ka/Ks &#226;&#8364;&#8220; Yeast          0.08                0.12              4e-11    small number pseudogenes E-families, difference Ka/Ks &#226;&#8364;&#8220; E. coli        0.056               0.1               1e-8     pseudogenization rates highly statistically significant <br />                                                                       (Table 3). table shows essential nonessential members E- <br /> families stronger purifying selection members N-          larger total number paralogs (Table 1) N-families, <br /> families. P-values calculated using tailed t-test.     coupled observation higher pseudogenization rate, <br />                                                                       suggests recent duplicates N-families enjoy <br />                                                                       higher duplication rate. test prediction, identified <br />       example, partitioning paralogous gene sets E-          pairs orthologs lineage-specific paralogs using InPara- <br /> families N-families used test essentiality   noid algorithm (Remm et al. 2001) closely related yeast major determinant selection. strength selection   species (S. cerevisiae, Candida glabrata, Kluyveromyces lactis, Aphis <br /> characteristic gene family membership opposed essenti-       gossypii, Debaryomyces hansenii). data num- <br /> ality, predict genes E-families, including nonessen-   ber paralogs fixed species diverg- <br /> tial ones, subject significantly stronger purifying       ing common ancestor. example, given gene selection members N-families. assessed strength        paralog S. cerevisiae, ortholog paralogs C. purifying selection standard measures: single feature   glabrata, infer lineage-specific duplication occurred <br /> polymorphism (SFP) densities Saccharomyces cerevisiae genes        divergence C. glabrata. Using approach, (Winzeler et al. 2003), K /K s ratios S. cerevisiae&#226;&#8364;&#8220;        calculated number duplicate fixation events mem- <br /> Saccharomyces paradoxus orthologs Escherichia coli K12/           bers E-families N-families. N-families fix <br /> CFT073 orthologs. genes stronger purifying         duplication events rate approximately twice E- <br /> selection expected lower SFP densities lower Ka/Ks    families (Table 4) example, 90% E-family members <br /> ratios (Nei 1987).                                                    S. cerevisiae exactly ortholog C. glabrata,       essential nonessential members        80% N-families group. agreement E-families substantially stronger purifying selection        17% E-families &gt;46% N-families fixed members N-families, independent species analyzed       duplicate  produce extra paralog) S. cerevisiae method used estimate selection (Table 2).          divergence common ancestor D. hansenii.  employed criteria, difference strength     validity analysis hinges assumption selection nonessential genes E-families N-             duplication rate E-families N-families. Alter- <br /> families considerably significant difference        natively, differences duplication rates explain essential nonessential genes species      observed pattern invoking differential fixation dupli- <br /> (Table 2).  strength selection        cates;  hard imagine genes E-families <br /> salient characteristic gene family membership essential-      N-families duplicate different rates. <br /> ity.  data Table 2 suggest essentiality se         Taken  duplication pseudogenization data  necessary sufficient impose purifying selection.    dicate N-families evolve significantly &#226;&#8364;&#339;dynamic&#226;&#8364;? <br /> Instead, given E-families exhibit significant biases   regime E-families.  weaker purifying se- <br /> characteristics responsible transitive correla-     lection (Table 2), members N-families higher rate tions strength selection, Codon Adaptation        pseudogenization (Table 3) duplicate fixation (Table 4). <br /> Index protein abundance (Hahn Kern 2005; Supplemental          course, observations entirely indepen- <br /> Table S4), conclude existence essential paralog     dent greater duplicate fixation rate result used specific marker genes subject strong purifying     higher rate pseudogenization. relevant issue effect selection. systematic differences protein func- <br /> tions E-families N-families; classes fami- <br />                                                                       Table 3.     Genes pseudogenes E-families N-families lies showed comparable functional diversity, E- <br />                                                                       yeast <br /> families included greater fraction molecular chaperones proteins related functions, N-families                                                 Pseudogene/gene <br /> riched metabolic enzymes transporters  annotated lists                       genes         pseudogenes                 ratio E-family N-family membership, Supplemental ma-         E-families        278           4                            0.014 <br /> terial).                                                              N-families        656           62                           0.095 <br />                                                                       E/N ratio         0.42          0.06                    P-value &lt; 1e-20 <br /> Selection dynamics duplications divergence <br />                                                                       ratio pseudogenes/genes seven times E-families prob- implication observation sets    ability P &lt; 1e-20 difference chance. calcula- <br /> genes related evolution share characteristics        tion comparing ratio genes types impose selection. recently showed (Shakhnovich 2006)          families pseudogenes. <br />  <br />  <br />                                                                                                              Genome Research             1531 <br />                                                                                                                 www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 4. Rate duplicate fixation E-families N-families              thermore, observed difference distributions synony- S. cerevisiae yeasts                                        mous site divergence E-families N-families compatible <br />                  C. glabrata      K. lactis     gossypii     D. hansenii    notion characteristic half-life paralogs                                                                               E-families longer N-families. Taking Ks <br /> Exactly 1 ortholog pair                                                       value measure evolutionary time, estimated pairs <br />   E-families         0.9             0.86          0.87            0.81       paralogs E-families survive, average, times <br />   N-families         0.8             0.6           0.53            0.41       longer (mean Ks = 3.25) paralogs N-families (mean <br />                                                                               Ks = 0.15; P &lt; 1e-40).  increased evolution- <br /> +1 paralog S. cerevisiae <br />   E-families          0.08           0.12          0.11            0.17       ary dynamism results shift shorter life spans genes <br />   N-families          0.17           0.33          0.41            0.46       N-families. <br />  <br /> +2 paralogs S. cerevisiae                                                  Longer life span duplicates E-families allows greater <br />   E-families        &lt; 0.01         &lt; 0.01        &lt; 0.01          &lt; 0.01       divergence sequence transcriptional regulation <br />   N-families          0.02           0.05          0.03            0.07 <br />                                                                                   observation longer average time survival duplicates used InParanoid software estimate number species-                E-families carries significant implications divergence specific paralogs S. cerevisiae yeast species.       protein sequences families. fact, visual examination difference E-families N-families significant            graph representation largest paralogous families P-value &lt; 1e-10 using &#226;?&#185;2 tests. <br />                                                                                   S. cerevisiae immediately reveals striking difference                                                                                   E-families N-families (Fig. 1). sizes prodigious rate evolution typical fate para-               families shown Figure 1 similar, N-family logs. Does higher pseudogenization rate N-families (Table                  greater number connections node (node degree) 3) offset higher duplicate fixation rate (Table 4) resulting             (&#226;&#710;&#188;10 compared &#226;&#710;&#188;1). observed substantial <br /> shorter overall survival pairs paralogs? test               difference mean clustering coefficients charac- <br /> assume synonymous sites approximately neutral                        terize density connections genes types <br /> evolve clock-like fashion. assumption, distri-                families (&#226;&#710;&#188;0.55 N-families compared &#226;&#710;&#188;0.21 E-families; <br /> bution synonymous substitutions paralogous gene families                    P &lt; 1e-3). number connections node mirror age distribution paralogs.                               clustering coefficient measure transitivity sequence space, sug- <br />       N-families, distribution Ks shows best fit            gesting  average, E-family paralogs diverged farther away exponential decay curve. consistent approxi-                   N-family paralogs. <br /> mately constant probability pseudogenization unit time.                           calculated distribution sequence divergence  sharp contrast, number paralogs E-families correlates                tween pairs paralogs E-families N-families using linearly increase synonymous site divergence (Fig. 2).                standard measures nonsynonymous substitution rate, shape distribution Figure 2 E-families                   number nonsynonymous substitutions nonsynonymous <br /> explained model pseudogenization rate                        site (Ka), <span id='am-10' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence identity  Supplemental families drops paralogs diverge (data shown). Fur-                   material).  criteria, paralogs E-families                                                                                                             characterized greater diver- <br />                                                                                                             gence (Fig. 3A,B). example, S. ce- <br />                                                                                                             revisiae, average amino acid se- <br />                                                                                                             quence identity paralogs E- <br />                                                                                                             families &#226;&#710;&#188;40% compared &#226;&#710;&#188;73% <br />                                                                                                             N-families (Fig. 3B). Qualitatively <br />                                                                                                             similar results observed E- <br />                                                                                                             families N-families E. coli                                                                                                             Caenorhabditis elegans (Supplemental <br />                                                                                                             Table S6a,b). Furthermore, differ- <br />                                                                                                             ence function distribution <br />                                                                                                             family size shown Supplemental <br />                                                                                                             Table S7a,b. Finally, nearly identical  <br />                                                                                                             sults obtained broad range                                                                                                             cutoffs used classify genes paralogs <br />                                                                                                             (Table 5; Supplemental Table S6a,b). <br />                                                                                                             results unequivocally                                                                                                              average, paralogs E-families                                                                                                             separated protein se- <br />                                                                                                             quence space paralogs N- <br />                                                                                                             families. <br />                                                                                                                   fact, difference sequence <br />                                                                                                             divergence distributions large <br /> Figure 2. Distributions synonymous substitution rates (Ks) pairs paralogs E-familes (gray      wanted assess pre- <br /> circles) N-families (black squares). x-axis Ks values y-axis number pairs. Ks bins 0.3 E-families 0.1 N-families. black dashed line fit          dictive value differentiating exponential decay paralogs N-families (R2 = 0.96, P &lt; 0.001). solid gray line       E-families N-families absence <br /> linear fit Ks divergence paralogs E-families (R2 = 0.86, P &lt; 0.001).                          essentiality data. performed  <br />  <br />  <br /> 1532      Genome Research <br />              www.genome.org <br />                                                                                                                        Evolution gene families <br />  <br />  <br />  <br />  <br /> Figure 3. Sequence divergence E-families N-families.   Distributions nonsynonymous substitution rates (Ka) pairs paralogs N-families (black squares) E-families (red circles). mean Ka values &#226;&#710;&#188;0.5 E-families &#226;&#710;&#188;0.14 N-families (P &lt; 1e-50). (B) Distributions sequence identity pairs paralogs E-families (red circles) N-families (black squares). mean sequence identity &#226;&#710;&#188;40% pairs paralogs E-families &#226;&#710;&#188;73% pairs paralogs N-families (P &lt; 1e-40). <br />  <br />  <br /> ciprocal analysis,  examined characteristics                contrast, paralogs N-families appeared coregu- <br /> sequence divergence differentiate E-families                  lated TFs (Fig. 5).  average, E-family <br /> N-families. end, used Receiver Operating Characteris-                members diverged substantially farther N- <br /> tic (ROC) statistics based average separation paralogs             family members sequence, regulatory <br /> families classify genes E-families N-families               context. consistent notion longer time explicitly invoking essentiality marker. ROC curve                  survival paralogs E-families allows modification upstream <br /> Figure 4 shows &#226;&#710;&#188;80% E-families paralogs                regions increases chances acquisition new regula- <br /> far diverged &#226;&#710;&#188;20% N-families. better separation                tory mechanisms. <br /> E-families N-families obtained using clustering <br /> coefficient classification criterion: 73% E-families            Discussion captured single false positive (N-family),              present evidence gene family membership general analysis covered larger families                  reliable indicator strength purifying selection acting members), resulting classification 11 E-families 18 <br /> N-families yeast.  separation paralogs sequence <br /> space E-families N-families dissimilar classifica- <br /> tion based solely sequence divergence,  reproduces partitioning based using essential paralogs markers. <br />       results E-families explore protein se- <br /> quence space, duplication divergence paralogs, greater depth N-families. important bear- <br /> ings impact family membership functional diver- <br /> gence. far, presented evidence paralogs E- <br /> families survive longer, enjoy lower rate pseudogenization, diverge sequence farther paralogs N-families. Sub- <br /> functionalization (division ancestral pleiotropy) character- <br /> ized strong purifying selection paralogs dupli- <br /> cation (Force et al. 1999; Lynch Force 2000) aided long survival duplicates E-families. sequence divergence taken characteristic sub- <br /> functionalization, reasonably hypothesize paralogs E-families preferentially evolve subfunctionaliza- <br /> tion scenario. <br />       diverging paralogs E-families follow subfunc- <br /> tionalization  neofunctionalization) routes, expect                Figure 4. ROC curves predicting family belongs E-                                                                                 N-class based average sequence divergence paralogs clustering observe divergence sequence expression               coefficient. Paralogous gene families (strongly connected components regulation paralogs adapt new biological niches.                   DDG) sorted average sequence divergence pairs compared extent transcription factor (TF) sharing                paralogs (black squares) clustering coefficients (green triangles). <br /> paralogs classes families. accord notion              families members analyzed, families <br />                                                                                 clustering coefficient 0 excluded uninformative. greater functional diversification genes E-families,              results 23 E-families 63 N-families sequence divergence paralogs families common transcription                   calculation 12 E-families 18 N-families clustering coeffi- <br /> factors binding upstream regions (Harbison et al. 2004).               cient calculation. red diagonal line represents random expectation. <br />  <br />  <br />                                                                                                                        Genome Research              1533 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 5. Robustness sequence divergence E-families                       magnitude observed differences N-families Ks cutoff used building DDG graph S.              types families sharply contrasts previous observations weak <br /> cerevisiae <br />                                                                               moderate correlations various functional evolu- <br />               genes       genes       &lt; Ka&gt;      &lt;Ka&gt;       tionary characteristics genes determined genome scale <br /> Ks cutoff    E-families     N-families     E-families    N-families     (Hurst Smith 1999; Hirsh Fraser 2001; Pal et al. 2001; <br />                                                                               Jordan et al. 2003; Drummond et al. 2005). believe 5                 275               658          0.509344       0.147632 <br />                                                                               case E-families N-families different 4.4               270               659          0.509814       0.147657 <br /> 3.8               264               642          0.507339       0.144439      partitioning attains sharp separation genes classes, <br /> 3.2               246               625          0.500473       0.140768      measured characteristics evolution reflect collective <br /> 2.6               204               585          0.490572       0.128384      behaviors distinct gene sets. tight link 2.3               159               568          0.445489       0.124553 <br />                                                                               gene family membership strength purifying selection <br /> 2                 114               491          0.404539       0.104033 <br /> 1.7                69               434          0.307063       0.091604      likely reflect relationship function  <br /> 1.4                34               369          0.20307        0.070387      straints. Gene families share set related functions <br />                                                                               (Todd et al. 2001; Shakhnovich Max Harvey 2004) Average nonsynonymous sequence divergence (Ka) calculated            similar range functional constraints, translates pairs paralogs. differences &lt;Ka&gt; E-families         characteristic strength purifying selection. typical range N-families statistically significant (P &lt; 0.001).  expanded version included Supplemental Table S1.)                                        functions associated certain level purifying selection,                                                                               turn, determines subsequent fate duplicates,  gene. Purifying selection, linked functional constraints,             likelihood gene fixed duplication, diverges affect course molecular evolution                 particular degree paralog, undergoes subfunctional- <br /> influencing speed divergence, affecting fate           ization neofunctionalization. investigations paralogs duplication. Specifically, paralogs            origins homogeneity selection acting members paralo- E-families include essential genes subject                gous families uncover novel determinants selection <br /> stronger purifying selection genes N-families (Table 2)               rooted commonalities shared members family essential paralogs.  salient observation            specific individual genes. genes families paralogs experience similar levels                 possible conclusion study subfunc- <br /> selection. clear-cut difference dis-             tionalization transient phase gene evolution, genes <br /> tribution protein functions E-families N-families,             divide ancestral functions soon undergo neofunctionaliza- <br /> appears presence essential genes corre-             tion. hypothesis consistent observations lates greater biological importance E-family members                  apparently decreasing rate pseudogenization passage (Table 2). spite stronger selective constraints, paralogs            time (Fig. 2) change transcriptional regulation paralogs <br /> E-families exhibit greater sequence divergence N-families                E-families (Fig. 5). similar sub-, neofunctionalization model <br /> (Fig. 3). Furthermore, difference exploration se-               recently proposed Zhang study quence space E-families N-families large              evolution protein&#226;&#8364;&#8220;protein interaction networks  use average sequence divergence paralogs cluster-              Zhang 2005). <br /> ing coefficient predictive manner classify large propor-                  results presented uncover surprisingly strong link <br /> tion families using existence essential para-            gene&#226;&#8364;&#8482;s membership paralogous family,  <br /> log criterion (Fig. 4).                                                  straints imposed evolution purifying selection,       difference average sequence divergence members E-families N-families attributed differential dynamics molecular evolution families. <br /> Specifically, N-families evolutionarily dynamic,  <br /> duplicates families tend pseudogenes shortly duplication (Table 3) enjoy higher fixation rate <br /> (Table 4). results compatible recent evidence  <br /> dicating  yeast, functionally important genes tend duplicate  Zhang 2006). net outcome difference evolutionary dynamics E-families N-families paralogs E-families longer life spans <br /> (Fig. 2).  average, paralogs E-families older,  evolving longer time, paralogs N-families <br /> (Fig. 2). likely  thanks longer average life <br /> span, paralogs E-families follow subfunctionalization neofunctionalization paths paralogs N- <br /> families. hypothesis supported demonstration extensive transcriptional rewiring (divergence TF- <br /> binding sites) paralogs E-families N-families <br /> (Fig. 5). findings robust respect method family identification  importantly, species              Figure 5. Shared transcription factors regulating pairs paralogs                                                                               E-families N-families. E-families, 92% pairs TF- <br /> families analyzed similar results obtained                 binding sites common, 29% pairs N-families shared <br /> yeast, nematode C. elegans, bacterium E. coli             sites TFs (P &lt; 1e-50). distribution N-families Supplemental material).                                                       characterized heavy tail pairs sharing TFs. <br />  <br />  <br /> 1534        Genome Research <br />              www.genome.org <br />                                                                                                                     Evolution gene families <br />  <br />  <br /> characteristics gene family evolution duplication. clas-                generation protein database search programs. Nucleic Acids Res. <br />                                                                                   25: 3389&#226;&#8364;&#8220;3402. <br /> sification genes E-families N-families useful <br />                                                                               Conery, J.S. Lynch, M. 2001. <span id='am-2' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Nucleotide</span> substitutions starting point variety future studies relationships             evolution duplicate genes. Pac. Symp. Biocomput. 2001: 167&#226;&#8364;&#8220;178. evolution genes, genomes, phenotypes.                        Cormen, T.H. 2001. Introduction algorithms, 2nd ed. MIT Press, <br />                                                                                   Cambridge, MA. <br />                                                                               Davis, J.C. Petrov, D  2004. Preferential duplication conserved <br />                                                                                   proteins eukaryotic genomes. PLoS Biol. 2: e55. <br /> Methods                                                                       Drake, J.W., Charlesworth, B., Charlesworth, D., Crow, J.F. 1998. <br />                                                                                   Rates spontaneous mutation. Genetics 148: 1667&#226;&#8364;&#8220;1686. construct DDG species (S. cerevisiae, C.             Drummond, D , Bloom, J.D., Adami, C., Wilke, C.O., Arnold, F.H. <br /> elegans, E. coli), complete sets protein sequences                2005. highly expressed proteins evolve slowly. Proc. Natl. Acad. <br />                                                                                   Sci. 102: 14338&#226;&#8364;&#8220;14343. respective genomes extracted GenBank data-                  Drummond, D , Raval, , Wilke, C.O. 2006. single determinant <br /> base, all sequence comparison performed                    dominates rate yeast protein evolution. Mol. Biol. Evol. <br /> using BLAST program (Altschul et al. 1997; ftp://                             23: 327&#226;&#8364;&#8220;337. <br /> ftp.ncbi.nih.gov). protein node DDG,                 Enright, J., Kunin, V., Ouzounis, C  2003. Protein families                                                                                   TRIBES genome sequence space. Nucleic Acids Res. 31: 4632&#226;&#8364;&#8220;4638. <br /> results comparisons, represented BLAST                     Force, , Lynch, M., Pickett, F.B., Amores, , Yan, Y.L., scores, expectation (e) values, amino acid sequence identities,                Postlethwait, J. 1999. Preservation duplicate genes edges connecting nodes. edges assigned                       complementary, degenerative mutations. Genetics 151: 1531&#226;&#8364;&#8220;1545. <br /> weights using BLAST scores, amino acid sequence identities,                Fraser, G., Kamath, R.S., Zipperlen, P., Martinez-Campos, M., <br />                                                                                   Sohrmann, M., Ahringer, J. 2000. Functional genomic analysis ratio nonsynonymous (Ka) synonymous (Ks)                       C. elegans chromosome systematic RNA interference. Nature <br /> substitution rates. calculate Ka Ks values, amino acid                 408: 325&#226;&#8364;&#8220;330. <br /> sequence alignments traced corresponding                     Fraser, H.B., Hirsh, E., Steinmetz, L.M., Scharfe, C., Feldman, <br /> nucleotide sequence alignments, Ka Ks calculated                     M.W. 2002. Evolutionary rate protein interaction network. <br />                                                                                   Science 296: 750&#226;&#8364;&#8220;752. <br /> using PAML (Yang 1997; Lynch Conery 2000; Yang                        Gerdes, S.Y., Scholle, M.D., Campbell, J.W., Balazsi, G., Ravasz, E., <br /> Nielsen 2000; Conery Lynch 2001). similar procedure                     Daugherty, M.D., Somera, L., Kyrpides, N.C., Anderson, , <br /> used Enright et al. (2003).                                                    Gelfand, M.S., et al. 2003. Experimental determination       Paralogous families identified finding strongly                 level analysis essential genes Escherichia coli MG1655. J. <br />                                                                                   Bacteriol. 185: 5673&#226;&#8364;&#8220;5684. <br /> connected components DDG described Cormen                        Giaever, G., Chu, M., Ni, L., Connelly, C., Riles, L., Veronneau, S., <br /> (2001)  Supplemental Table S1). define strongly                      Dow, S., Lucau-Danila, , Anderson, K., Andre, B., et al. 2002. <br /> nected component set nodes path exists                    Functional profiling Saccharomyces cerevisiae genome. Nature pair. principal results presented obtained                   418: 387&#226;&#8364;&#8220;391. <br />                                                                               Gu, X. 1999. Statistical methods testing functional divergence ing BLAST cutoff 1e-15 dS = 5.  nearly identical                 gene duplication. Mol. Biol. Evol. 16: 1664&#226;&#8364;&#8220;1674. <br /> results obtained broad range cutoffs (Table 3)              Gu, X. 2001a. Mathematical modeling functional divergence After identifying strongly connected components DDG,                   gene duplication. J. Comput. Biol. 8: 221&#226;&#8364;&#8220;234. <br /> high-throughput essentiality data used divide fami-               Gu, X. 2001b. site-specific measure rate difference gene <br />                                                                                   duplication speciation. Mol. Biol. Evol. 18: 2327&#226;&#8364;&#8220;2330. <br /> lies E- N-classes. yeast gene essentiality data              Hahn, M.W. Kern, D. 2005. Comparative genomics centrality obtained high-throughput knockout experiments                           essentiality eukaryotic protein-interaction networks. <br /> (Giaever et al. 2002); C. elegans gene essentiality data                 Mol. Biol. Evol. 22: 803&#226;&#8364;&#8220;806. genome-wide RNAi knockdown experiments (Fraser et                    Harbison, C.T., Gordon, D.B., Lee, T , Rinaldi, N.J., Macisaac, K.D., <br />                                                                                   Danford, T.W., Hannett, N.M., Tagne, J.B., Reynolds, D.B., Yoo, J., et <br /> al. 2000; Kamath et al. 2003; Simmer et al. 2003), E. coli                al. 2004. Transcriptional regulatory code eukaryotic genome. <br /> essentiality data GenBank (ftp://ftp.ncbi.nih.gov)                      Nature 431: 99&#226;&#8364;&#8220;104. <br /> (Gerdes et al. 2003; details, Supplemental material).             Harrison, P.M. Gerstein, M. 2002. Studying genomes data yeast SNPs Winzeler et al. (2003).                      aeons: Protein families, pseudogenes proteome evolution. J. Mol. <br />                                                                                   Biol. 318: 1155&#226;&#8364;&#8220;1174. <br /> transcription factor data ChIP-chip experiments (Lee                Harrison, P.M., Echols, N., Gerstein, M.B. 2001. Digging dead <br /> et al. 2002; Harbison et al. 2004).                                               genes: analysis characteristics pseudogene <br />       used InParanoid program (http://inparanoid.                          population Caenorhabditis elegans genome. Nucleic Acids Res. <br /> cgb.ki.se/) identify orthologs species-specific paralogs                   29: 818&#226;&#8364;&#8220;830. <br />                                                                               Harrison, P., Kumar, , Lan, N., Echols, N., Snyder, M., Gerstein, yeast species (S. cerevisiae, C. glabrata, K. lactis,             M. 2002. small reservoir disabled ORFs yeast genome gossypii, D. hansenii). genomes obtained                    implications dynamics proteome evolution. J. Mol. Biol. <br /> NCBI (http://www.ncbi.nlm/nih.gov).                                               316: 409&#226;&#8364;&#8220;419. <br />                                                                                X. Zhang, J. 2005. Rapid subfunctionalization accompanied                                                                                   prolonged substantial neofunctionalization duplicate gene <br />                                                                                   evolution. Genetics 169: 1157&#226;&#8364;&#8220;1164. <br /> Acknowledgments                                                                X. Zhang, J. 2006. Higher duplicability important genes <br />                                                                                   yeast genomes. Mol. Biol. Evol. 23: 144&#226;&#8364;&#8220;151. authors thank Eugene Shakhnovich invaluable sup-                  Hirsh, E. Fraser, H.B. 2001. Protein dispensability rate port review manuscript. thank Matthew Hahn                     evolution. Nature 411: 1046&#226;&#8364;&#8220;1049. proposing duplication rate study helpful                 Hurst, L.D. Smith, N.G. 1999. essential genes evolve slowly? <br /> comments regarding manuscript. Additionally, extend                        Curr. Biol. 9: 747&#226;&#8364;&#8220;750. <br />                                                                               Jordan, K., Rogozin, B., Wolf, Y , Koonin, E.V. 2002. Essential appreciation Charles DeLisi, Tim Reddy, Joe Mellor, Julian                 genes evolutionarily conserved nonessential genes <br /> Mintseris, Bioinformatics program Boston                     bacteria. Genome Res. 12: 962&#226;&#8364;&#8220;968. <br /> University discussion insights.                                       Jordan, K., Wolf, Y , Koonin, E.V. 2003. simple dependence <br />                                                                                   protein evolution rate number protein&#226;&#8364;&#8220;protein <br />                                                                                   interactions: prolific interactors tend evolve <br />                                                                                   slowly. BMC Evol. Biol. 3: 1. <br /> References                                                                    Jordan, K., Wolf, Y , Koonin, E.V. 2004. Duplicated genes evolve <br />                                                                                   slower singletons despite initial rate increase. BMC Evol. <br /> Altschul, S.F., Madden, T.L., Schaffer, , Zhang, J., Zhang, , Miller,       Biol. 4: 22. <br />     W., Lipman, D.J. 1997. Gapped BLAST PSI-BLAST: new              Kamath, R.S., Fraser, G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., <br />  <br />  <br />  <br />                                                                                                                     Genome Research              1535 <br />                                                                                                                        www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />     Kanapin, , Le Bot, N., Moreno, S., Sohrmann, M., et al. 2003.          Remm, M., Storm, C.E., Sonnhammer, E.L. 2001. Automatic <br />     Systematic functional analysis Caenorhabditis elegans genome         clustering orthologs paralogs pairwise species <br />     using RNAi. Nature 421: 231&#226;&#8364;&#8220;237.                                            comparisons. J. Mol. Biol. 314: 1041&#226;&#8364;&#8220;1052. <br /> Keightley, P.D. Eyre-Walker,  1999. Terumi Mukai riddle        Shakhnovich, B.E. 2006. Relative contributions structural <br />     deleterious mutation rates. Genetics 153: 515&#226;&#8364;&#8220;523.                       des <br /> </body></html>