<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The candidate tumor suppressor ING4 represses <br /> activation hypoxia inducible factor (HIF) <br /> Abdullah Ozer, Leeju C. Wu, Richard K. Bruick* <br /> Department Biochemistry, University Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, TX 75390-9038 <br />  <br /> Communicated Steven L. McKnight, University Texas Southwestern Medical Center, Dallas, TX, April 1, 2005 (received review January 31, 2005) <br />  hypoxia inducible factor (HIF) plays important role              HIF regulated posttranslational hydroxylation (11) progression number pathophysiological processes including              <span id='am-41' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="O2" datatype="xsd:string"></span>O2</span>-dependent hydroxylase, factor inhibiting HIF 1 (FIH-1) <br /> tumorigenesis. addition characterized oxygen-               (12&#226;&#8364;&#8220;14). <br /> dependent modes regulation, function HIF transcrip-                 hypoxic response pathway recognized tion factor influenced action                 important contributor number cancers, increased <br /> regulatory pathways. Misregulation factors resulting               levels HIF associated increased tumor aggres- <br /> inappropriate HIF expression activity contribute                 siveness, therapeutic resistance, mortality (15). HIF progression human cancers induction genes                    induced result intratumoral hypoxia stemming rapid <br /> promoting angiogenesis, glycolysis, cell survival, metastasis,             growth and&#229;&#8230;&#190;or inefficient O2 delivery tortuous vasculature. processes. candidate tumor suppressor protein                  Alternatively, HIF function altered O2-independent <br /> inhibitor growth family member 4 (ING4) recently                   manner genetic alterations activate signaling <br /> implicated repressor angiogenesis tumor growth                     pathways inactivate tumor suppressors (15). Understanding association NF-&#226;?&#172;B. demonstrate sup-                  mechanisms oncogenes tumor suppressor genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              BIOCHEMISTRY <br /> pression ING4 induces HIF transcriptional activity          affect HIF function sheds light misregu- <br /> ING4 directly associates HIF prolyl hydroxylase, Fe(II)-           lation promotes cancer reveals physiologically relevant <br /> dependent oxygenase previously shown mediate HIF stability               mechanisms cells fine-tune HIF activity function oxygen availability.  affecting              borderline hypoxic conditions encountered &#226;&#8364;&#732;&#226;&#8364;&#732;normoxic&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> HIF&#226;&#8364;&#8482;s stability, ING4 mediates HIF&#226;&#8364;&#8482;s activity. data support            tissues (16&#226;&#8364;&#8220;18). <br /> model  addition regulating HIF stability, HIF prolyl               Recently, inhibitor growth family member 4 (ING4) hydroxylases modulate HIF function recruitment                 emerged strong candidate tumor suppressor protein,  ING4, likely component chromatin-remodeling complex.                 pressing tumor growth angiogenesis (19) loss                                                                                contact inhibition (20). ING4 member family proteins <br /> prolyl hydroxylase &#229;&#8230;&#169; inhibitor growth <br />                                                                                characterized highly conserved C-terminal plant homeodo- <br />                                                                                main (PHD)-like zinc-finger domain implicated variety <br />  <br /> T    o maintain adequate levels aerobic respiration, mammalian <br />      cells able sense status O2 availability respond changes O2 levels fall critical levels. <br />                                                                                processes including oncogenesis, apoptosis, DNA repair,                                                                                cell cycle control (21). best-studied ING family members <br />                                                                                reside nucleus positively negatively regulate key component hypoxic response pathway hypoxia                 gene expression interactions N terminus <br /> inducible factor (HIF). Decreased O2 cell results HIF              chromatin-remodeling complexes containing histone acetyl- <br /> induction subsequent transcription number target               transferases, histone deacetyltransferases, factor acetyltrans- <br /> genes promote adaptation environmental stress. list           ferases (21). HIF targets includes genes affecting metabolism, O2 delivery,                  ING4&#226;&#8364;&#8482;s precise mode action elucidated, <br /> angiogenesis, cellular survival (1). Consequently, HIF plays            ING4 shown interact RelA subunit NF-&#226;?&#172;B <br /> important role physiological pathophysiological states               suppress expression angiogenesis-related genes including O2 supply limiting (1, 2).                                            6, 8, Cox-2 (19). ING4 serves     HIF obligate heterodimer composed HIF-&#226;?&#164; subunit                 suppress expression HIF target genes hypoxic conditions. essentially insensitive O2 availability HIF-&#226;?&#163; subunit         Furthermore, HPH-2 directly interacts ING4, providing accumulation activity acutely responsive O2 levels             possible mechanism ING4 recruitment HIF. Interestingly, <br /> (reviewed refs. 1&#226;&#8364;&#8220;3). Briefly, normoxic conditions HIF-&#226;?&#163;              ING4 association HPH-2 does affect hydroxylase targeted rapid proteosomal degradation association           activity HIF stability. Instead, data consistent protein ubiquitin ligase complex containing product von           model ING4, recruited HPH-2 HIF hypoxic <br /> Hippel&#226;&#8364;&#8220;Lindau tumor suppressor gene (pVHL). pVHL fails bind                 conditions, acts adapter protein recruit transcriptional &#226;?&#163;-subunit hypoxic conditions, allowing HIF accumu-                repressors mediate HIF activity. <br /> late. Differential recognition HIF-&#226;?&#163; VHL mediated hydroxylation conserved proline residues HIF-&#226;?&#163;                   Materials Methods <br /> O2-dependent degradation domain (4). normoxic condi-                     Recombinant Protein Expression Purification. Protein coding <br /> tions, HIF prolyl hydroxylases (HPHs; called PHDs EGLNs)               sequences HPH-2 (GenBank accession  AF229245) efficiently modify residues (5&#226;&#8364;&#8220;7), prerequisite VHL                ING4 (GenBank accession  NM&#227;&#8250;&#173;016162) amplified PCR <br /> binding HIF-&#226;?&#163; (8, 9). HPH active site contains Fe(II)                   confirmed sequencing. Recombinant FIH-1 prepared coordinated Xaa-Asp . . . triad. enzymes turn <br />                                                                                described (13). sequence encoding HPH-2 residues 181&#226;&#8364;&#8220;426 <br /> bind 2-oxoglutarate, HIF-&#226;?&#163;, O2 effect hydroxylation HIF-&#226;?&#163; subunit 2-oxoglutarate. Hydroxylated 2-oxogluta- <br /> rate undergoes decarboxylation produce succinate <span id='am-19' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span>CO2</span>.                   Abbreviations: HIF, hypoxia inducible factor; VHL, von Hippel&#226;&#8364;&#8220;Lindau tumor suppressor enzymes require O2 substrate hydroxy-               gene; HPH, HIF prolyl hydroxylase; FIH-1, factor inhibiting HIF-1; ING, inhibitor growth <br /> lation reaction, implicated direct O2 sensors         family member; PHD, plant homeodomain; siRNA, short interfering RNA. <br /> hypoxic response pathway (10). Independent protein stability,                correspondence addressed. E-mail: bruick@biochem.swmed.edu. <br /> coactivator recruitment C-terminal transactivation domain               &#194;&#169; 2005 National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                        PNAS &#229;&#8230;&#169; 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169;  21 &#229;&#8230;&#169; 7481&#226;&#8364;&#8220;7486 <br />  (HPH-2C) subcloned pHIS parallel vector (22)         2,400 &#226;&#171;&#187; g 4&#194;&#176;C 5 min. Pellets washed 25 mM <br /> purified following procedure used FIH-1 (13).         Hepes-KOH (pH 7.5), 3 mM MgCl2, 1 mM DTT, 0.1% Nonidet <br />   ING4 expressed Escherichia coli G&#226;?&#164;1 fusion protein.     P-40, 1:200 Protease Inhibitor Mixture resuspended Cells lysed buffer containing 20 mM Tris&#228;&#161;&#160;HCl (pH 8.0), 50     buffer. Cells lysed passage 25-gauge needle <br /> mM NaCl, 5 mM 2-mercaptoethanol, 100 &#226;?&#174;M ZnCl2, 1:200              centrifuged 30 s. cytosolic supernatant removed, <br /> Protease Inhibitor Mixture (Sigma) purified Source Q         nuclear pellets washed times resuspen- <br /> resin (Amersham Pharmacia). G&#226;?&#164;1 tag removed TEV            sion 20 mM Hepes-KOH (pH 7.5), 400 mM KCl, 1 mM DTT, <br /> protease digestion followed <span id='am-6' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> exchange chromatography using      10% glycerol incubated 4&#194;&#176;C 30 min. samples <br /> Mono S resin (Amersham Pharmacia). ING4 stored buffer          centrifuged 200,000 &#226;&#171;&#187; g 45 min 4&#194;&#176;C separate containing 50 mM NaPO4 (pH 5.9), 300 mM NaCl, 1.0 mM                  extracted nuclear proteins insoluble material. <br /> 2-mercaptoethanol, 10 &#226;?&#174;M ZnCl2 4&#194;&#176;C. <br />                                                                       [14C]-2-Oxoglutarate Decarboxylation Assay. HPH activity mea- <br /> Yeast Hybrid Assay. yeast strain L40 transformed     sured modified [14C]-2-oxoglutarate decarboxylation assay <br /> vectors encoding HPH-2C fused LexA DNA-binding domain          derived ref. 24. Briefly, 1-ml reaction containing 3.0 &#226;?&#174;M <br /> (DBD) (HPH-2C&#229;&#8230;&#190;pVJL10) ING4 fragments fused                 <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> substrate (HIF-1&#226;?&#163; residues 556&#226;&#8364;&#8220;574) incubated GAL4 activation domain (ING4&#229;&#8230;&#190;pGAD-NotII). Positive protein&#226;&#8364;&#8220;           recombinant HPH-2C enzyme presence 50 mM Tris&#228;&#161;&#160;HCl <br /> protein interactions denoted LacZ reporter expression.        (pH 8.0), 2 mg&#229;&#8230;&#190;ml BSA, 0.2 mg&#229;&#8230;&#190;ml catalase, 5 mM KCl, 1.5 mM <br />                                                                       MgCl2, 1.0 mM DTT, 64 &#226;?&#174;M [14C]-2-oxoglutarate [14.6 nCi&#229;&#8230;&#190;nmol <br /> Western Blot Analysis. Cells resuspended SDS sample buffer,   specific activity (1 Ci &#226;&#171;&#189; 37 GBq)], 50 &#226;?&#174;M ascorbate, 20 &#226;?&#174;M proteins resolved SDS&#229;&#8230;&#190;PAGE Western blot            FeSO4 sealed 15-ml tube. 1-h incubation room <br /> analysis. Polyclonal antiserum obtained rabbits immu-        temperature gentle shaking, pH solution nized HPH-2C, ING4, 140 amino acids          lowered &#226;&#172;&#8225;2 diluted HClO4. Released [14C]CO2 cap- <br /> HIF-&#226;?&#164; subunit. HPH-2 antiserum failed detect endogenous        tured 3M Whatman paper saturated 10 M NaOH HPH-1 HPH-3. ING4 antiserum displayed minimal cross-           measured scintillation counting. <br /> reactivity ING family members, weak <br /> detection ING1 ING5 seen overexpression. Mouse       RNA Interference. Cells plated 24 plates (3 &#226;&#171;&#187; 104 cells <br /> monoclonal antibodies p-ATF-2, annexin (Santa Cruz Bio-           16 h transfection, 200 nM small interfering <br /> technology), HIF-1&#226;?&#163; (BD Transduction Laboratories, Lexing-        RNA (siRNA) duplexes (Dharmacon, Lafayette,  trans- <br /> ton, KY) purchased. Immune complexes detected            fected using Oligofectamine reagent (Invitrogen). Media enhanced chemiluminescence using peroxidase-conjugated sec-           replaced 16&#226;&#8364;&#8220;20 h transfection. Cells maintained ondary antibodies (Jackson ImmunoResearch).                           normoxic conditions (20% O2) 72 h transfection                                                                       normoxic conditions 57 h followed 12&#226;&#8364;&#8220;15 h hypoxia <br /> Cell Culture. HeLa cell lines maintained Dulbecco&#226;&#8364;&#8482;s           (1% O2). Total RNA preparation Northern blot analysis modified Eagle&#226;&#8364;&#8482;s medium (HyQ DME, HyClone) containing                 performed according methods ref. 23. transient <br /> high glucose supplemented 10% FBS (Gemini Biolog-            transfections, HeLa cells transfected 3XHRE-tk-Luc <br /> ical Products, Calabasas, CA) presence 5.0% CO2          reporter construct 51 h siRNA transfection using Lipo- <br /> 37&#194;&#176;C. Cells maintained hypoxic conditions 37&#194;&#176;C          fectamine Plus (Invitrogen). siRNA duplexes composed humidified hypoxic chamber (Coy Laboratory Products,         following oligonucleotides: ING4#1, UGAGGGACCUAGAC- <br /> Ann Arbor, MI) filled 1% O2&#229;&#8230;&#190;5.0% CO2 balanced           CAAAGTT CUUUGGUCUAGGUCCCUCATT; ING4#2, <br /> N2. Luciferase assays performed according method          GAACGGAAGAAGAAAUAGATT UCUAUUUCU- ref. 23.                                                           UCUUCCGUUCTT; GFP, GGCUACGUCCAGGAGCG- <br />                                                                       CACC UGCGCUCCUGGACGUAGCCUU. <br /> Coimmunoprecipitation Assay. HeLa cells transfected ING4 p3XFLAG-CMV10 vector (Sigma) using Lipo-               Chromatin Immunoprecipitation Assay. Briefly, HeLa cell line <br /> fectamine plus reagent (Invitrogen). 24 h, cells lysed     stably transfected 3XHRE-tk-Luc reporter construct 20 mM Tris&#228;&#161;&#160;HCl (pH 7.5), 100 mM NaCl, 5 mM 2-mercap-             maintained hypoxic conditions (1% O2) 15 h. Crosslink- <br /> toethanol, 1% Nonidet P-40, Protease Inhibitor Mixture            ing, lysate preparation, immunoprecipitations, DNA purifica- <br /> (Sigma). Lysates precleared Protein agarose beads         tion performed according experimental procedures <br /> (Roche Diagnostics) incubated presence preimmune        described ref. 25. 227-bp region 3XHRE promoter serum HPH-2C immune serum overnight 4&#194;&#176;C. Antibody&#226;&#8364;&#8220;              amplified using following primers: 5&#226;&#172;&#732;-AGTGCAGGTGC- <br /> protein complexes precipitated Protein agarose beads,     CAGAACATT-3&#226;&#172;&#732; 5&#226;&#172;&#732;-CGGTAGGTCGAGAGGTCAGA-3&#226;&#172;&#732;. <br /> washed extensively lysis buffer, eluted SDS loading     PCR products separated electrophoresis 1.5% agarose <br /> buffer. Coimmunoprecipitated 3XFLAG-tagged ING4                gel stained ethidium bromide. Relative band intensities <br /> tected Western blot analysis using anti-FLAG M2 antibody           quantitated QUANTITY (Bio-Rad) software. <br /> (Sigma). <br />                                                                       Results <br />                   35S-labeled <br /> GST Pull-Downs.               proteins prepared TNT       ING4 Represses Hypoxic Induction HIF Target Genes. siRNAs Coupled Reticulocyte Lysate (Promega). Lysates contain-        used efficiently suppress expression ING4 HeLa cells ing 35S-labeled proteins incubated GST fusion proteins      determined Western blot analysis (Fig. 1A). remaining <br /> immobilized <span id='am-24' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-25' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span>glutathione</span> Sepharose 4B resin (Amersham Phar-         protein signal low levels unsuppressed ING4 macia) 1 h 4&#194;&#176;C. washes buffer containing    potential cross-reactivity endogenous ING5. siRNA <br /> 20 mM Tris&#228;&#161;&#160;HCl (pH 7.5), 200 mM NaCl, 1 mM DTT, 0.5%              treatment, cells incubated normoxic conditions Nonidet P-40, bound proteins eluted boiling SDS sample     subjected hypoxia promote induction HIF-responsive genes. <br /> buffer resolved SDS&#229;&#8230;&#190;PAGE.                                      cells treated control siRNA targeting GFP,                                                                       characterized HIF target genes, Nip3 (23) adenylate kinase <br /> Subcellular Fractionation. HeLa cells washed ice-cold PBS   3 (AK3) (26), induced response hypoxia assessed harvested scraping. Cell pellets resuspended 10 mM     Northern blot analysis (Fig. 1B). ING4 expression Hepes-KOH (pH 7.5), incubated ice 10 min, centrifuged      suppressed ING4-targeted siRNAs, levels  <br /> 7482 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                                                Ozer et al. <br />                                                                                                                                                                   BIOCHEMISTRY <br /> Fig. 1. siRNA suppression ING4 enhances HIF target gene expression hypoxic conditions.   Western blot analysis indicating relative protein <br /> levels HPH-2, ING4, nuclear HIF-1&#226;?&#163; normoxic (20% O2) hypoxic <br /> (1% O2) conditions 12 h treatment HeLa cells siRNA duplexes <br /> specific ING4 (ING4#1 ING4#2) GFP control. HIF-2&#226;?&#163; undetectable Western blots HeLa cell extracts. (B) Northern blot analysis endogenous HIF target genes Nip3 AK3 siRNA-mediated suppression ING4. Actin mRNA levels shown confirm equivalent RNA loading. results representative multiple experiments. <br />  <br />  <br />  endogenous HIF target genes hypoxic condi- <br /> tions typically increased additional 2- 3-fold relative GFP siRNA control-treated cells (Fig. 1B). increases                                                                                   Fig. 2. ING4 interacts HPH-2 C-terminal PHD (residues <br /> HIF-responsive gene expression increase                                                                                   191&#226;&#8364;&#8220;249).  Upper) Yeasts transformed vectors encoding HPH-2C <br /> nuclear HIF-1&#226;?&#163; levels (Fig. 1 , suggesting HIF activity,            fused LexA DNA-binding domain (DBD) ING4 fused GAL4 HIF stability, suppressed ING4. expected,                 activation domain. Positive protein&#226;&#8364;&#8220;protein interactions denoted LacZ <br /> effect ING4 suppression observed HIF target genes                reporter expression. Transformations performed duplicate. (Lower) conditions (normoxia) HIF-1&#226;?&#163; does accumulate                   Yeast hybrid analysis HPH-2C interactions N- C-terminal <br /> (Fig. 1 . Consistent ING4-dependent phenotype ob-                      truncations ING4 shown. (B) 35S-labeled length ING4 (residues <br /> served NF-&#226;?&#172;B-responsive genes (19), inverse rela-                 1&#226;&#8364;&#8220;249) ING4 lacking PHD (residues 1&#226;&#8364;&#8220;198) incubated immo- <br /> tionship ING4 levels expression HIF target genes                   bilized GST GST&#229;&#8230;&#190;HPH-2C fusion protein. Bound 35S-labeled ING4                                                                                   visualized SDS&#229;&#8230;&#190;PAGE. (C) Lysates HeLa cells transfected N- cells HIF induced hypoxia. <br />                                                                                   terminal 3XFLAG-tagged ING4 immunoprecipitated anti-HPH-2 <br />                                                                                   serum preimmune serum control. ING4 coimmunoprecipitated <br /> HPH-2 Interacts ING4. hypothesized ING4                     HPH-2 detected Western blot analysis using anti-FLAG M2 interact directly HIF, unable detect                antibody. IP, immunoprecipitate. <br /> interaction variety assays (data shown).  did <br /> detect interaction ING4 factor known regulate <br /> HIF-&#226;?&#163; stability, HPH-2. interaction proteins                 ING4 weakly ING4 lacking PHD vitro GST initially observed yeast hybrid screen                  pull assays (Fig. 2B). interaction HPH-2 LexA DNA-binding domain (DBD) fused catalytic                          ING4 detected coimmunoprecipitation assay <br /> hydroxylase domain HPH-2 (HPH-2C). itself, LexA                         (Fig. 2C). <br /> DBD&#229;&#8230;&#190;HPH-2C fusion protein capable activating                           demonstrate direct association ING4 porter genes driven LexA promoter (Fig. 2A). yeast                HPH-2 vitro, proteins expressed bacteria                                                                                   purified near homogeneity. shown Fig. 3, HPH-2C mi- <br /> cells cotransformed construct expressing ING4 fused                                                                                   grated predicted 27-kDa monomer size exclusion GAL4 activation domain, activation LexA-responsive                     column (Fig. 3B), behavior ING4 (predicted mo- <br /> LacZ reporter gene observed, indicative positive interac-                lecular mass 29 kDa) consistent multimeric <br /> tion HPH-2C ING4 (Fig. 2 .                                          species (Fig. 3A). proteins incubated  <br />    identify region ING4 associates HPH-2C,                  observed migrate larger complexes (Fig. <br /> series N- C-terminal deletion constructs assayed.                  3C). contrast, incubation ING4 regulatory HIF <br /> shown Fig. 2  residues 191&#226;&#8364;&#8220;249 ING4, encompassing                     hydroxylase, FIH-1, did result appearance larger <br /> PHD, sufficient interact HPH-2C. interaction                protein complex (Fig. 3E).  independent lines observed 198 residues ING4. Consistent                 evidence demonstrate direct association ING4 results, HPH-2C able interact strongly length              HPH-2. <br />  <br /> Ozer et al.                                                                                              PNAS &#229;&#8230;&#169; 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169;  21 &#229;&#8230;&#169; 7483 <br />  Fig. 3. Purified ING4 HPH-2C directly associate vitro.         Fig. 5. ING4 HPH-2 substrate vitro. Substrate utilization micrograms purified recombinant ING4, HPH-2C, and&#229;&#8230;&#190;or FIH-1 proteins        HPH-2C assessed [14C]CO2 generation presence control <br /> incubated resolution Superdex 200 10&#229;&#8230;&#190;30             HIF-1&#226;?&#163; peptide substrate recombinant ING4 protein. Assays  <br /> column (Amersham Pharmacia Biotech). Eluted fractions resolved            formed triplicate. <br /> SDS&#229;&#8230;&#190;PAGE visualized Coomassie blue staining. elution profile protein standards given  <br />                                                                                   lation substrate HPH-2, employed hydroxylase assay using <br />                                                                                   [14C]-2-oxoglutarate cosubstrate (24). [14C]-2-oxoglutarate HPH-2 ING4 Reside Nucleus. results indicate             efficiently hydroxylated HPH-2C presence HPH-2 directly interacts ING4. cells,  ING                   substrate HIF-1&#226;?&#163; peptide. Hydroxylated [14C]-2- <br /> family members predicted reside nucleus (27).                   oxoglutarate undergoes decarboxylation release [14C]CO2,  localization studies performed overexpressed GFP&#226;&#8364;&#8220;                   captured quantified. levels emitted <br /> HPH protein fusions indicate HPH-2 resides primarily                  [14C]CO2 reflect relative utilization putative <br /> cytoplasm (28, 29). determine endogenous ING4                      polypeptide substrate hydroxylation reaction (Fig. 5). Unlike <br /> HPH-2 reside subcellular compartment,                     results obtained HIF-1&#226;?&#163; substrate, incubation examined distribution HeLa cell extracts (Fig. 4).                 HPH-2C ING4 did result increased levels [14C]CO2 <br /> incubation, cells lysed, soluble cytosolic proteins                 production, indicating ING4 HPH-2 substrate. <br /> separated insoluble cellular material, including intact nuclei, subjected high-salt extraction nuclear proteins.             ING4 Does Affect HPH-2 Activity HIF Stability. predicted, ING4 exclusively fraction containing               ING4 did appear HPH-2 substrate affect HPH-2 <br /> extracted nuclear proteins. Contrary GFP&#226;&#8364;&#8220;HPH-2 fusion                      HIF accumulation (Fig. 1 , reasoned ING4 association <br /> results, endogenous HPH-2 equally distributed frac-                   HPH-2 directly affect hydroxylase activity. test tions. data provide evidence substantial portion                                                                                   hypothesis, hydroxylation HIF peptide substrate HPH-2C <br /> HPH-2 does reside nucleus, promote O2- <br />                                                                                   followed presence increasing amounts purified <br /> dependent degradation nuclear HIF associate                                                                                   ING4. ability HPH-2C hydroxylate HIF-&#226;?&#163; assay nuclear factors ING4. <br />                                                                                   assessed subsequent interaction peptide substrate                                                                                   35S-labeled VHL pull assay (6). shown Fig. 6, <br /> ING4 HPH-2 Substrate. addition ING4, hybrid <br /> interactions observed HPH-2C known                         recombinant ING4 stimulated inhibited hydroxylation <br /> HPH substrates, HIF-1&#226;?&#163; HIF-2&#226;?&#163; (data shown).                     peptide substrate HPH-2C. lack effect ING4 speculated positive interaction reflect enzyme&#226;&#8364;&#8220; <br /> substrate relationship HPH-2 ING4. Known targets HPHs contain conserved LXXLAP motif (30). sequence present ING family members, <br /> mutations tolerated adjacent proline residue peptide substrates (31). test ING4 hydroxy- <br />  <br />  <br />  <br />  <br />                                                                                   Fig. 6. ING4 does affect HPH-2C activity vitro. Hydroxylase activity                                                                                   recombinant HPH-2C measured 35S-labeled VHL pull assay Fig. 4.    ING4 HPH-2 salt-extracted nuclear             presence HIF-1&#226;?&#163; peptide substrate increasing amounts recom- <br /> proteins. HeLa cells incubated normoxic (20% O2) hypoxic (1%        binant ING4  ING4 protein relative HPH-2C indicated). <br /> O2) conditions 12 h. Soluble cytoplasmic (C) salt-extracted nuclear (N)   Reaction conditions chosen provide approximately half-maximal <br /> proteins separated analyzed Western blot analysis antibod-       substrate hydroxylation observe stimulation inhibition hydrox- <br /> ies raised ING4 HPH-2. Antibodies p-ATF-2 annexin used    ylase activity. 35S-labeled VHL binding fully hydroxylated peptide (&#226;&#171;&#185; assess integrity nuclear cytoplasmic samples, respectively.     control) shown reference. Assays performed triplicate. <br />  <br />  <br /> 7484 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0502716102                                                                                                   Ozer et al. <br />                                                                                     Fig. 8. ING4, HPH-2, HIF associate HRE promoter. HeLa cells <br />                                                                                    stably transfected 3XHRE-tk-Luc reporter construct incubated <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  BIOCHEMISTRY <br />                                                                                    hypoxic (1% O2) conditions 15 h followed crosslinking, sonica- <br /> Fig. 7. ING4 affects HIF activity chromatin-dependent manner.               tion, immunoprecipitation using antibodies specific ING4, HIF-&#226;?&#164;, Suppression ING4 increases expression stably transfected HIF-driven        HPH-2. (Upper) Primers flanking 3XHRE promoter element used luciferase reporter gene. HeLa cell line stably expressing HIF-responsive    amplify associated chromatin DNA PCR. (Lower) Relative band intensities <br /> 3XHRE-tk-Luc reporter gene transfected siRNA duplexes specific        indicated. Ab, antibody control; Pre Immune, preimmune serum ING4 (duplex ING4#1) GFP followed incubation normoxic           control. <br /> (20% O2) hypoxic (1% O2) conditions 15 h. Similar results obtained independently isolated stably transfected HeLa cell lines (data shown). (B) siRNA suppression ING4 does affect hypoxic induction                                                                                    factors expected affect transcription <br /> transiently transfected HIF reporter gene. Wild-type HeLa cells trans- <br /> fected siRNA duplexes specific ING4 GFP control followed <br />                                                                                    transiently transfected (nonchromosomal) DNA. 3XHRE-tk-Luc HIF reporter construct incubation normoxic hypoxic conditions. assays performed triplicate, results   Discussion <br /> representative multiple experiments.                                            HIF-dependent hypoxic response pathway present virtu- <br />                                                                                    ally cell body participates physiological processes <br />                                                                                    disease states ischemia cancer (1, 2, 15). hydroxylase activity vitro consistent lack change             discovery hydroxylases mediate HIF-&#226;?&#163; stability nuclear HIF-1&#226;?&#163; protein levels ING4 levels suppressed                   coactivator recruitment provided mechanism HIF siRNA (Fig. 1 .                                                               function directly regulated O2. availability <br />                                                                                    O2 serve hydroxylase substrate plays major role ING4 Suppresses HIF Activity Chromatin-Dependent Manner.                      determining status HIF induction given cell (10), ING family members interact chromatin-                          HIF expression hydroxylase function likely subject remodeling factors (21), speculated ING4 likewise                    additional layers regulation involving signaling pathways, post- <br /> suppress HIF activity recruiting factors restrict accessibility <br />                                                                                    translational modification, feedback loops, protein&#226;&#8364;&#8220;protein inter- transcriptional machinery DNA. begin test                                                                                    actions, protein localization (1). extensive dynamic <br /> model, compared effect siRNA-mediated ING4 suppres- <br />                                                                                    regulation HIF reflects critical biological role sion hypoxic induction HIF-responsive luciferase <br />                                                                                    need fine-tune HIF induction physiologically relevant <br /> reporter driven multiple HIF responsive elements, 3XHRE-tk- <br /> Luc (32), stably (Fig. 7A) transiently (Fig. 7B)                O2 concentrations encountered &#226;&#8364;&#732;&#226;&#8364;&#732;normoxic&#226;&#8364;&#8482;&#226;&#8364;&#8482; conditions transfected. siRNA treatment, stably transfected cells                  HIF transcription factors partially induced. <br /> incubated hypoxic conditions promote induction                 relationship O2 availability hydroxylase <br /> luciferase reporter gene. cells treated control                   activity serve general determinant HIF induction, <br /> GFP siRNA, relative luciferase activity induced 10-fold                 multiple layers regulation likely required subtly adapt HIF response hypoxia (Fig. 7A). ING4 expression                         function variety dynamic microenvironments. <br /> suppressed, luciferase reporter activity hypoxic conditions                     physiological importance potential therapeutic utility typically increased additional 2-fold relative GFP-targeted              hypoxic response pathway driven search additional <br /> siRNA controls. result entirely consistent data                   regulatory components. end, investigated candidate <br /> presented Fig. 1B endogenous HIF target genes. Consistent                   tumor suppressor ING4. inverse relationship ING4 hypothesis, ING4, HPH-2, HIF subunit                      expression tumor growth vascular volume human associate 3XHRE element hypoxic                        glioblastomas mouse xenograft models observed. <br /> ditions chromatin immunoprecipitation assay (Fig. 8).                         hybrid coimmunoprecipitation experiments    contrast results observed stably transfected                 revealed interaction ING4 NF-&#226;?&#172;B (19), precise <br /> reporter, hypoxic induction transiently transfected reporter                mechanism ING4 suppresses NF-&#226;?&#172;B function unaffected ING4-targeted siRNA (Fig. 7B).                         determined. HIF transcription factors fre- <br /> results consistent model ING4 suppresses HIF                   quently induced cancers,   promote tumor- <br /> activity recruitment chromatin-remodeling factors,                  igenesis, aforementioned cell culture studies change  <br /> Ozer et al.                                                                                               PNAS &#229;&#8230;&#169; 24, 2005 &#229;&#8230;&#169; vol. 102 &#229;&#8230;&#169;  21 &#229;&#8230;&#169; 7485 <br />  expression levels proangiogenic HIF target genes  e., VEGF)                                   ING4. case HIF-responsive genes HeLa cells, observed manipulation ING4 (19).                                   putative ING4-associated factors modestly suppress <br /> noted studies performed hypoxic                                      transcription. local changes histone <br /> ditions expect HIF induction. Suspecting                                    acetylation state changes acetylation state ING4 suppress HIF-mediated effects tumor                                        nonhistone proteins including HIF itself. support chromatin- <br /> growth angiogenesis, examined effects ING4                                    remodeling model, suppression ING4 did affect expres- <br /> levels HIF target gene induction normoxic                                        sion transiently transfected HIF reporter subject hypoxic conditions.                                                                                regulation function chromatin structure. <br />    Consistent hypothesis, suppression ING4 HeLa                                        far focused attention ING4 association cells led significant reproducible increase                                        HPH-2. Given high conservation C-terminal regions <br /> induction endogenous HIF target genes hypoxic                                         HPH proteins, possible ING4 ditions. similar result obtained using stably trans-                                    interact members HPH family. Likewise, <br /> fected reporter gene driven minimal HIF-responsive pro-                                       PHD highly conserved members ING <br /> moter, supporting contention effects ING4                                     family, HPHs interact ING proteins mediated HIF transcriptional regu-                                       turn recruit different sets proteins HIF HIF-dependent <br /> lators. transcriptional targets NF-&#226;?&#172;B                                        promoters.  precise consequence(s) ING associ- <br /> reported promote HIF induction increasing HIF-1&#226;?&#163; sta-                                        ation HPHs requires additional study likely bility (33), believe ING4 indirectly regulates HIF                                  complicated differences function, protein recruitment, NF-&#226;?&#172;B mechanism. studies, change                                       expression patterns ING HPH family members. HIF-1&#226;?&#163; stability observed ING4 suppression,                                           data demonstrate candidate tumor effects ING4 suppression pronounced                                         suppressor protein ING4 represses function factors <br /> hypoxic conditions, unlike effects NF-&#226;?&#172;B HIF,                                      mediating tumor growth angiogenesis addition NF-&#226;?&#172;B. observed normoxia (33).                                                       Furthermore, direct interaction ING4    ING4 directly recruited NF-&#226;?&#172;B,                                  HIF regulatory factor, suggesting mechanism ING4 recruit- <br /> unable detect direct association ING4 HIF. <br />                                                                                                    ment HIF additional role HIF prolyl <br /> Instead, ING4 associated directly known HIF regulator, <br />                                                                                                    hydroxylases mediating HIF function hypoxic conditions. <br /> HPH-2, multiple assays. ING4 substrate HPH-2,                                                                                                    Lastly, provide evidence repressive effects ING4 <br /> did affect HPH activity like recently reported HPH- <br />                                                                                                    depend context transcribed target genes, implying interacting factor OS-9 (34). Instead, study provides evidence <br />                                                                                                    ING4&#226;&#8364;&#8482;s mode action depend ability recruit substantial fraction HPH-2 resides nucleus                                                                                                    chromatin-remodeling factors. candidate chromatin- mediate HIF degradation reoxygenation associate ING4 affect HIF activity.                                                        remodeling factors proposed interact ING4 (35), <br />    ING family members shown promote chromatin                                         additional studies required identify physiological <br /> remodeling recruitment factors promote                                      complexes ING4 resides. <br /> suppress transcriptional activation (21). O2 levels high, <br />                                                                                                    thank B. Horazdovsky, C. Dann, P. Erbel, J. Rutter helpful <br /> HPH-2 hydroxylates proline residues O2-dependent <br />                                                                                                    advice. work supported Burroughs Wellcome Fund Career <br /> degradation domain subsequently displaced product                                    Award, University Texas Southwestern President&#226;&#8364;&#8482;s Research VHL (pVHL), leading rapid HIF-&#226;?&#163; degradation                                      Council, Robert  Welch Foundation, American Cancer <br /> transcriptional repression pVHL-mediated recruitment                                    Society Young Investigator Award  R.K.B.) conducted histone deacetylases (12). O2 levels low, HIF-&#226;?&#163; recog-                                    facility constructed support Research Facilities Improvement <br /> nition HPH-2 instead promote recruitment chromatin-                                      Program Grant C06 RR-15437. R.K.B. Michael L. Rosenberg <br /> remodeling factors promoter regions HIF target genes                                 Scholar Medical Research. <br />  <br />  <br />  1.   Wenger, R. H. (2002) FASEB J. 16, 1151&#226;&#8364;&#8220;1162.                                                 19. Garkavtsev, , Kozin, S. V., Chernova, O., Xu, L., Winkler, F., Brown, E., Barnett, <br />  2.   Maxwell, P. H. <span id='am-12' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Ratcliffe, P. J. (2002) Semin. Cell Dev. Biol. 13, 29&#226;&#8364;&#8220;37.                       G. H. &amp; Jain, R. K. (2004) Nature 428, 328&#226;&#8364;&#8220;332. <br />  3.   Bruick, R. K. (2003) Genes Dev. 17, 2614&#226;&#8364;&#8220;2623.                                               20. Kim, S., Chin, K., Gray, J. W. &amp; Bishop, J. M. (2004) Proc. Natl. Acad. Sci. USA 101, <br />  4.   Huang, L. E., Gu, J., Schau, M. &amp; Bunn, H. F. (1998) Proc. Natl. Acad. Sci. USA 95,              16251&#226;&#8364;&#8220;16256. <br />       7987&#226;&#8364;&#8220;7992.                                                                                   21. Campos, E. , Chin, M. Y., Kuo, W. H. &amp; Li, G. (2004) Cell. Mol. Life Sci. 61, <br />  5.   Epstein,  C. R., Gleadle, J. M., McNeil, L. , Hewitson, K. S., O&#226;&#8364;&#8482;Rourke, J., Mole,           2597&#226;&#8364;&#8220;2613. <br />       D. R., Mukherji, M., Metzen, E., Wilson, M. , Dhanda, , et al. (2001) Cell 107, 43&#226;&#8364;&#8220;54.   22. Sheffield, P., Garrard, S. &amp; Derewenda,  (1999) Protein Expression Purif. 15, 34&#226;&#8364;&#8220;39. <br />  6.   Bruick, R. K. &amp; McKnight, S. L. (2001) Science 294, 1337&#226;&#8364;&#8220;1340. <br />                                                                                                    23. Bruick, R. K. (2000) Proc. Natl. Acad. Sci. USA 97, 9082&#226;&#8364;&#8220;9087. <br />  7.   Ivan, M., Haberberger, T., Gervasi, D. C., Michelson, K. S., Gu   &#194;&#168;nzler, V., Kondo, K., <br />                                                                                                    24. Kivirikko, K.  &amp; Myllyla, R. (1982) Methods Enzymol. 82, 245&#226;&#8364;&#8220;304. <br />       Yang, H., Sorokina, , Conaway, R. C., Conaway, J. W. &amp; Kaelin, W. G. (2002) Proc. <br />       Natl. Acad. Sci. USA 99, 13459&#226;&#8364;&#8220;13464.                                                        25. Wang, F., Zhang, R., Beischlag, T. V., Muchardt, C., Yaniv, M. &amp; Hankinson, O. <br />  8.   Ivan, M., Kondo, K., Yang, H., Kim, W., Valiando, J., Ohh, M., Salic, , M.,  J.,             (2004) J. Biol. Chem. 279, 46733&#226;&#8364;&#8220;46741. <br />       Lane, W. S. &amp; Kaelin, W. G. (2001) Science 292, 464&#226;&#8364;&#8220;468.                                     26. O&#226;&#8364;&#8482;Rourke, J. F., Pugh, C. W., Bartlett, S. M. &amp; Ratcliffe, P. J. (1996) Eur. J. Biochem. <br />  9.   Jaakkola, P., Mole, D. R., Tian, Y.-M., Wilson, M. , Gielbert, J., Gaskell, S. J., von         241, 403&#226;&#8364;&#8220;410. <br />       Kriegsheim, , Hebestreit, H. F., Mukherji, M., Schofield, C. J., et al. (2001) Science     27. Feng, X., Hara, Y. &amp; Riabowol, K. (2002) Trends Cell Biol. 12, 532&#226;&#8364;&#8220;538. <br />       292, 468&#226;&#8364;&#8220;472.                                                                         <br /> </body></html>