<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1988 Genetics Society America <br />  <br />  <br />  <br />  Conditional Mutants RPCl60, Gene Encoding Largest Subunit               RNA Polymerase C Saccharomyces cerevisiae <br />  <br />  Rosmarie Gudenus,&apos; Sylvie Mariotte, Alejandra Moenne, Anny Ruet, Sylvie Memet, Jean-Marie <br />                        Buhler, Andre Sentenac Pierre Thuriaux <br />  Service Biochimie, Dbpartement Biologie, C.E.N. Saclay, Commissariat Energie Atomique, F91191 G$ sur Yvette (cedex), <br />                                                                France <br />                                             Manuscript received September 24, 1987 <br />                                                   Revised copy accepted March 12, 1988 <br />  <br />                                                               ABSTRACT <br />                  18.4-kb fragment yeast genome containing gene largest subunit RNA <br />               polymerase C (RPCI6O) cloned hybridization previously isolated fragment gene. <br />               RPG160 maps chromosome XV, tightly linked allelictotheessentialgene             TSM8740. <br />               Temperature sensitive (ts) mutant alleles constructed vitro mutagenesis NaHSOs <br />               substituted wild-type allele chromosome.     wereunambiguously <br />               identified rpcl60 mutants failure complement fully defective mutation rpcl60:: URA3. <br />               faithful transcription yeast tRNA gene mutant cell-free extracts strongly reduced               compared wild-type. n vivo, rpcl60 mutations specifically affect synthesis tRNA               temperature sensitive way, comparatively little effect synthesis 5s rRNA               effect on5.8s rRNA. unlinked mutation (pcil-3)suppresses thetemperature sensitive phenotype <br />               rpc160-41 mutation. <br />  <br />  <br />    N yeast eukaryotes, genes      small RNAs tRNA 5 s rRNAare <br /> specifically transcribed nuclear <br />                                                                         yeast RNA polymerase C participates                                                                         transcription genes corre- <br />                                                                         sponding tRNA 5s rRNA. Given impor- <br /> RNA polymerases, RNA polymerase C  111).                        tance yeasts model systems tostudymajor <br /> enzyme extensively purified contains                       cellular processes eukaryotes, undertaken <br /> fourteen distinct polypeptides  called sub-                   isolation characterization mutantspro- <br /> units), sharedwith RNA polymer-                       ducing conditionally defective RNA polymerase C <br /> ase RNA polymerase B (SEN-                         S. cerevisiae. genes potentially corresponding <br /> TENAC 1985). T h e promotersrecognized               RNA             distinct subunits RNA polymerase C polymeraseC       intragenic        transfer RNAs               cloned immunological screening (RIVAet al. 1986). <br /> corresponding tononsensesuppressors,mutations                           (RPC160) encodes largest subunit defective promoters lead antisuppressor                      C160 identical gene previously identified <br /> phenotype relatively easily                     homology structural gene largest <br /> isolated characterized yeasts Saccharomyces                  subunit RNA polymerase B Drosophila melano- <br /> cerevisiae (ALLISON   et al. 1983) SchizosaccharomyceJ              gaster (INGLES    et al. 1984). [RNA polymerase subunits <br /> pombe (PEARSON    et al. 1985) intragenic promoters                 symbolized capital  B C followed directly recognized RNA polymerase C                         number correspondingto estimated molecular <br /> itself, additionalproteins(transcription           fac-          weight x 10&apos;. symbols havebeen used tors) added thepurified RNA                            designate corresponding genes (RUET      et al. 1980; <br /> polymerase C achieve faithful transcription vitro                 INCLES   et al. 1984; RIVAet al. 1986; NONETet al. 1987). <br /> (KLEKAMP     WEIL1982; KOSKIet al. 1982; RUETet                     Pending general agreement  al. 1984; CAMIER    et al. 1985) thereforeessen-                (MANNet al. 1987) adopted nomenclature <br /> tial gene         specificity RNA polymer-                    proposed RIVAet al. (1986).] genes ase C.                                                                  striking sequence homology gene encoding <br />    n spite extensive enzymological immuno-                     largestsubunit         bacterial RNA polymerase <br /> logical studies (SENTENAC     1985), little known              (ALLISON    et al. 1985) report theconstruction interaction RNA polymerase C                        temperature-sensitive mutant alleles RPCl60 <br /> cognate transcription factors.        clear          affect synthesis tRNAs vivo vitro, <br />                                                                         isolation extragenic suppressor restoring <br />    &apos; <br />    Present address: Immuno AG, Uferstrasse 15, A2304, Orth der       growth restrictive temperature theprecise <br /> Donau, Austria.                                                         localization RPC160 gene chromosome <br /> Genetics 119 517-326 (July, 1988). <br />  518                                                      R. Gudenus et al. <br />  <br />                            TABLE 1                                  described SHERMAN, FINK HICKS          (1979).                           Yeast strains                             proteinase phenotype pep4-3 mutation fol- <br />                                                                     lowed method JOHNSTON (1977) using 2% com- <br />                                                                     mercial powder milk caseine source.Strain D27-7C <br />                                  Genotype <br />                                                                     carries pep4::HIS3 null allele (kindly sent C. WOOL- <br />                                                                                         + <br />  <br />  <br />  <br />  <br /> YNN281&quot;           ura3-52 his3-200 trpl-901 lys2-801 ade2-101       FORD) instead ofpep4-3.      Free spores canRICAN+ <br />                      MATa                                           heterozygous diploids obtained digesting asci <br /> YNN281canRb       ura3-52 his3-200 trpl-901 lys2-801 ade2-101       100-fold diluted Helix pomatia juice (Industrie Biolo- <br />                      canR MAT                                       gique Francaise) forat 1 hrat 30&#194;&#176;, checking YNN282&quot;           ura3-52 his3-200 trpl-901 lys2-801 ade2-101       complete digestion microscope separating <br />                      MAT                                            aggregated sporesby sonication. spores plated <br /> YNN281-15         uru3-52 his3-200 trpl-901 lys2-801 ade2-I01       selective mediumcontaining canavanine isolate <br />                      rpc160-15 MATa                                 canR haploid segregants (KLAPHOLZ      EASTON-ESPOSITO <br /> YNN281-20         ura3-52 3-200 trpl-901 lys2-801 ade2-101       1982), occasional formation MATclIMATa diploids <br />                      rpcl60-20 MATa                                 mating adjacent spores plating. Yeast <br /> YNN281-31         ura3-52 his3-200 trpl-901 lys2-801 ade2-101       transformation according ITO et al. (1983). <br />                      rpcl60-31 MATa                                 Integrative transformants symbolized :: YNN281-41         ura3-52 his3-200 trpl-901 lys2-801 ade2-I01       symbol  integrated <br />                                                                                                                                                                                DNA fragment     (e.g., <br />                      rpcl60-41 MATa                                 rpcl60:: URA3 ). Transformants distinguished                                                                                   + <br />  <br />  <br />  <br />  <br /> D27-7C            ura3-52 his3-200 trpl-901 lys2-801 ade2-I01       potential revertants suppressors  obtained                      rpc160-41 MATa pep4 ::HIS3+ MAT                transforming amber lys2-801 mutants) testing YNN281-41-3       urd-52 his3-200 trpl-901 lys2-801 ade2-I01        mitotic instability transformedcharacter                       rpcI60-41 MATa pcil-3 MATa                     needed, genomic hybridization analysis appro- <br /> D25-13A           his3-200 trpl ade2-I01 pep4-3 rpc160-31 MATa      priate &quot;P-labeled probe. <br /> 382-23A&apos;          spoll ura3-52 can1 cyh2 ade2hi57 h o d MATa          Plasmidsandphages: Phage plasmids EMBL3 <br /> 393-35C&apos;          spoll ura3-52 his2 leu1 lysl met4 pet8 MAT        (FRISCHAUF   et al. 1983), pSP65 (MELTON   et al. 1984), YRp7 <br /> 381-9D&quot;           spol1 ura3-52 ade6 arg4 a707 asp5 met14 lys2      (PARENT,    FENIMORE    BOSTIAN1985) YCp631 <br />                     pet1 7 trpl MAT                                 (BARNESandTHORNER           1986) previously  <br /> 396-22B&apos;          spoll ura3-52 adel his1 leu2 lys7 met3trp5        scribed. pEMBLYi32 similar pEMBLYi21 contains <br />                     MAT                                             pUC19 polylinker (BALDARI CESARENI           1985 Be3 14d           tsm 8740 MATa                                     personal communication). Plasmid p02 pPBR322  <br /> NC93SPl&apos;          ura3-351,273,328 MATa                             rivate 1881 bp EcoRV PvuII fragment T12-2B(pSP4)      ura3-52 his3-200 trip1 -901 lys2-801 ade2-IO1     deleted tandem du lication 635-bp ClaI <br />                     rpcl60 ;: URA3+ MAT (pSP4)                      fragment carrying tRNA$yeast           gene[originating <br /> 20B-12f           pep4-3 trpl MAT                                   plasmid pY20 (FELDMAN,OLAHandFIEDENREICH <br /> D25-13A           pep4-3 trpl his3-200 ade2-I01 MAT                 1982)] inserted ClaI site (0.       GABRIELSEN <br /> CMY214&apos;           YNN281 canR X YNN282                              C. MARCK,personal communication). plasmids <br /> D8                382-23A X 393-35C                                 carrying inserts overlappingwith RPC160 constructed <br /> D9                382-23A X 396-22B                                 study (Figure 1). pSPl derived pSP65 Dl0               382-23A X 381-9D                                  carries 2.5-kb EcoRI insert RPC160 interrupted T2                rpcl60 :: URA3 + transformants CMY2 14         1. l-kb Hind111 URA3 fragment inserted                                                                                             + <br />  <br />  <br />  <br />  <br />                                                                     blunt-end ligation StuI site. YIp32-1 derives TDU               rpcl60::URA3 + transformants DU <br /> TD9                              + <br />                   rpcl60 :: URA3 transformants D9                   pEMBLYi32 insertion 5.3-kb XbaI fragment <br />                                                                     starts 1.2 kb upstream initiation codon ofRPCI60 <br /> TDlO              rpcl60 :: URA3 + transformants D 10 <br />                                                                     ends coding sequence. Finally, pSP4 carries <br />     Yeast Genetic Center, Berkeley.                                 14.4-kb fragment containingthe RPC160 gene <br />   &apos; MANNet al. (1987).                                              insertedat Sal1 site YCp631. fragment    KLAPHOLZand EASTON-ESPOSITO  (1982).                             isolated genomic library (M. SNYDER,       unpublished <br />    HILGER,PREVOTand MORTIMER(1982).                                 data) yeast strain X2180C contains Sau3A <br />               unpublished data. <br />    F. LACROUTE, <br />                                                                     partialfragments cloned BamHIsite   JONES (1976).                                                     EMBL3  library obtained C. MANN).Recom- <br />                                                                     binantphage plasmids obtained standard <br />                                                                     techniques (MANIATIS,    FRITSCH  SAMBROOK     1982). Plas- <br /> X V . n parallel work, mutations RPC40 gene              mids kept amplified Escherichia coli strains <br />  encodes subunit shared RNA polymerase                   RR1 HB 101 (MANIATIS,      FRITSCH SAMBROOK      1982). C) obtained, shown t o                     NaHSOs mutagenesis, used E. coZi strain JM107 <br /> affect vivo assembly RNA polymerases                 (YANISCH-PERRON,VIEIRA                                                                                                                   MESSING   1985) prepare <br /> (MANN et al. 1987). Temperature-sensitivemutants                    single stranded DNA ung strain BW310 [defective <br />                                                                     <span id='am-37' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> N-glucosylase (DUNCAN,    ROCKSTROH     WARNER thegeneencodingthelargestsubunit            yeast              1978)] amplification mutagenized DNA. <br /> RNA polymerase B described recently                     Media andgrowth conditions S. cerevisiae: Yeast <br /> (HIMMELFARB, SIMPSON       FRIESEN1987; NONETet                 strains grown 30&quot; using standard media growth <br /> al. 1987).                                                          conditions (SHERMAN, FINK HICKS      1979). Temperature <br />                                                                     sensitive mutants grown YPD 25&quot; tested                                                                      37&quot;. Nutritional requirements determined onminimal <br />              MATERIALS METHODS <br />                                                                      medium SD supplemented appropriate nutrients <br />    Yeast strains: yeast strains used study           (SHERMAN, FINK HICKS      1979). tryptophan, <br /> listed Table 1. standard genetic techniques yeast        <span id='am-30' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> requirements met 0.5% casamino <br />                                      Polymerase <br />                                           RNA                Mutants Yeast                                                             5 19 <br />  <br /> acid. Respiratory deficient mutants, resistance canavan-                   E         E    E E            E            E            E <br />                                                                  B    B <br /> ine cycloheximidewere tested described                   KLAPHOLZ    EASTON-ESPOSITO     (1982). Colonies resistant   1                                                                          1 5-fluoroorotate selected described BOEKE, <br />                                                                                                                                                             n <br />                                                                                   I <br />                                                                                   nn <br />                                                                                                                                  I1 <br />                                                                                                                                  n <br /> LACROUTE     FINK (1985) using 50-mm Petri dishes                  K                                         K            s <br /> cubated 5 days 25&quot;. Sporulation solid <br /> medium K-acetate (10 g/liter) supplemented appropriate nutrients quarter concentration <br />                                                                                                  Fii                    pspl <br />  <br />  <br /> used SD medium.                                                                            URA 3 <br />    NaHSOSmutagenesis: Single-stranded DNAwas pre- <br /> pared 100-ml culture J M l O l (YIp32-1) super-                                   1 <br />                                                                                                                       yIp32- <br />  <br /> infected phage IRl described DENTE,CESARENI                                       E E       St   E <br />                                                                                                                     C CORTESE    (1983). 10 pg DNA treated 10 min 37&quot; 3.0 M NaHSO3 (SHORTLE       BOTSTEIN   1983) mutagen removed spun-column chromatogra- <br /> phy 1-ml Sephadex G-50 medium column. transformation E. coli strain BW310, 2000           L                                                                  PSP4 <br />                                                                       1                                         1 <br /> pooled ampicillin-resistant transformants harvested              B                                         B                 S LB supplemented ampicillin double-stranded <br />                                                                     FIGUREI.-Simplified restriction map 18.4-kb region <br /> plasmid DNAwas prepared alkaline lysis CsCl <br />                                                                  spanning RPCl60. black arrow corresponds coding <br /> density gradient (MANIATIS,    FRITSCH  SAMBROOK    1982). <br />                                                                  sequence RPCl60 (ALLISON   et al. 1985). Endonuclease cleavage uru3-52 yeast strain YNN281 transformed                                                                  sites symbolized B (BamHI), K (KfinI), H (HindIII), E 10 pg mutagenized plasmidDNA linearized                                                                  (EcoRI), S (SalI), X (XbaI), St (StuI) C (ClaI). inserts <br /> theunique CZaI site promote genomic integration                                                                  corresponding plasmids (pSP1, YIp32-1, YRp7-160 targeted crossing R P C l 6 0 gene (ORR-WEAVER, <br />                                                                  pSP4) used study drawn scale lower SZOSTAK   ROTHSTEIN     1981). 5000 transformants <br />                                                                  figure. <br /> selected uracil omission medium scored tem- <br /> perature-sensitivity replica plating YPD 25&quot; 37&quot;. <br />    n vivo labeling [14C]lysine [5,6-&apos;H]uracil:Cul- <br />                                                                  Cells washed distilled waterand disrup- <br /> tures (25 ml) yeast 2 X lo7 cells/ml) exponentially     tion buffer (200 mM Tris-HC1 pH 8.0, 1 mM EDTA, 10 <br /> growing 25&quot; SD medium (supplemented casa-             mM mgC12, 10 mM 2-mercaptoethanol, 10% v/v glycerol, 4 <br /> mino acids 20 mg/ml <span id='am-14' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-15' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>, trypotophan            mM phenylmethylsulfonylfluoride finally resus- <br /> uracil) cultivated 30 min water baths        pended 40 mlof thatbuffer. disruption 25&quot; 37&quot;. cellswere filtered 0.45-pm            Manton-Gaulin homogenizer, ionic strength ad- <br /> filters transferred medium       justed 300 mM ammonium sulfate. cell lysate uracil concentration 0.5 mg/ml. [&apos;HIUracil (5.104 Bq/        left 15 min ice centrifuged 90 min 35,000 <br /> ml) [&apos;4C]lysine(2.104Bqiml)         addedandthe         rpm. transcription assaywas final volume <br /> cultures grown 45 min. pulse-chase           40 pl, presence reaction mixture experiments labeling pulse 15 min             contained 40 mM HEPES pH 7.8,70 mM KCl, 5 mM MgC12, <br /> chased 30 min adding unlabeled uracil 200         10 mM dithiotreitol, 0.1 mM EDTA, 10 v/v glycerol, 3 mM <br /> mg/ml.Aliquots (1 ml precipitated 4 ml 10%               <span id='am-19' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span>, 3 mM <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span>GTP</span>, 3 mM <span id='am-47' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="CTP" datatype="xsd:string"></span>CTP</span>, 0.15 mM [&quot;PIUTP (6.7 lo3 <br /> trichloracetic acid, filtered 0.45-prn filters, thoroughly    Bq/nmol), 0.1 pg plasmid p02 containing yeast <br /> washed 5% trichloracetic acid counted scintil-       tRNA?&apos;&quot; gene varying concentrations cell-free ex- <br /> lation acid-precipitable 14Cand &apos;H activity. incor-       tract. Incubation 25&quot; 45 min. synthesized <br /> poration &apos;H insmallRNA          (5.8S, 5 s tRNA)       RNA separated electrophoresis 6% polyacryla- <br /> measured extracting total RNA follows. Cells resus-         mide gelin presence 7 M ureaand subjected pended 5 ml extraction buffer (50 mM Tris-HC1 pH <br />                                                                   autoradiography described  <br /> 7.4, 10 mM EDTA, 0.10 M NaCl 5% sodium dodecyl <br /> sulfate) crushed Eaton press extracted twice 5 ml phenol-chloroform-isoamyl alcohol mixture                                         RESULTS <br /> (24:24: 1). nucleic acids aqueous phase precipated ethanol resuspended 100 to200 pI <br />                                                                    Cloning genetic localization 18.4-kb  water. Aliquots corresponding 5 20 pg RNA          gion spanning gene RPCl60 onchromosome X V : <br /> vacuum dried, dissolved 20pl 90% formamide             2.5-kb EcoRI fragment RPClGO described run 6% polyacrylamidegel containing 7 M <span id='am-52' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span>             RIVAet al. (1986) corresponds half <br /> 0.09 M Tris-Borate (pH 8.3 2.5 M EDTA 90 min            coding sequence (Figure 1 ALLISON    et al. 200 V. gel fixed 7% <span id='am-42' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span>, immersed 30 min Amplifier T M (Amersham) dried 80&quot;          1985). isolate gene, fragment vacuum. Labeled RNA bands detected                 used probe genomic library strain X2 18OC <br /> autoradiography using kodak X-Omat S film,            S . cerevisiae phage EMBL3 (M. SNYDER, <br /> quantified densitometry.                                      unpublisheddata)      partly overlapping clones <br />    Preparation cell-free extracts vitro assay      analyzed restriction mapping using 18 dif- <br /> specific transcriptionof tRNA genes: Cultures (5-liter)      ferent enzymes. define 18.4-kb region strains 20B-12 (pep4-3,trpl), D25-13A (rpc160-31, <br /> pep4-3, trpl) D27-7C (rpc160-41, pep4::HZS?+ )          bears gene. simplified restriction map grown YPD collected centrifugation optical       given Figure 1  detailed map available density 2.0 600 nm, yielding 30 g wet cells.      request). <br />  520                                                 R. Gudenus et al. <br />     3.6-kp EcoRI fragment containing proxi-                                              TABLE 2 <br />  mal R P C l 6 0 interrupted 1.1-kb URA3  + <br />  <br />  <br />  <br />  <br />                                                                 Localization rpcl60 right arm chromosome XV  insert (Figure 1) subsituted                                    tetrad analysis <br />  wild-type alleles R P C l 6 0 ura3-52 diploid <br />  strain CMY214. experiment yeastcells                               Segregation genetic markers <br />  transformed linearized EcoRI fragment <br />                                                                                     RPCl60                RPCl60               tsm8740 URA3 transformants selected onthe <br />             + <br />  <br />                                                                                          X                   X                         X <br /> appropriate medium (ROTHSTEIN          1983). trans-                           tsm8740                 ade2                    ade2 <br /> formant (strain T2) analyzed hybrid-                               P T N P                 T N P           T        N <br />  ization ofits genomic DNA EcoRI digestion,              data     68           4       0       142 21    0          59    13     0 <br />  using 2.5-kb EcoRI fragment RPCl6O ra-              HILGER, PR~VOT                                                 38    13     0 <br /> dioactive probe. theuntransformedstrain,                 hybridization revealed 2.5-kb band corre-               MORTIMER <br /> sponding R P C l 6 0 + alleles, whereasan           Pooled data                                                    97    26     0 <br /> additional 3.6-kb band corresponding inter-             Average map        (1.2-7.3) <br />                                                                                  2.8     6.4                (3.0-9.8) <br />                                                                                                                    10.6            (7.0-14.2) <br /> rupted rpcl60:: URA3 allele present strain <br />                         + <br />                                                                 distances <br /> T 2 (data shown). Fourteen tetrads      dissected        Map distances given centimorgans (% recombinants) sporulationof T2. showed 2:2 segregation           confidence interval 95% given parentheses. P stands <br />                                                                parental ditypes, T tetratypes N nonparental viable spores (auxotrophic uracil) andtwo           ditypes. Confidence intervals calculated assuming Poisson <br /> lethal spores expected rpcl60 : : URA3 allele <br />                                                 +              distribution. Average map distances calculated data pooled haplolethal agreement equivalent              crosses using ts alleles small duplication <br />                                                                R P C l 6 0  RESULTS) <br />                                                                                           including 41 tetrads analyzed results INCLES et al. (1984). sporelethality      HILGER,PR~VOT       MORTIMER(1982), ignoring possible <br /> observed reflect defective germination          heterogeneities differences genetic  afailuretoundergo            vegetative growth,        grounds. <br /> constructed haploid strain T12-2B (pSP4)            chromosomes. 100 viable haploid segregants <br /> carries rpcl60 :: URA3 chromosomal markerbut <br />                        + <br />                                                               isolated transformed diploids viable contains functionalRPClGO         described KLAPHOLZand EASTON-ESPOSITO <br /> geneonthecentromeric              plasmid pSP4 (CEN3,         (1982). mutantand wild-type allele LYS2+, RPCl6O+).T12-2B cultivated               marker segregated roughly equal frequencies       generations liquid YPD, plated YPD              offspring diploid transformants TD8,TD9 2 X 10&apos; colonies wereexamined sponta-               TD10, ade2 pet1 7 neous plasmid loss leading lysine auxotrophy.           located chromosome XV. markers, <br /> auxotrophic colony detected, control            offspring 100% wild type 100% <br /> experiment RPCl6O strain carrying pSP4 <br />                                 + <br />                                                               mutant, depending diploid strain considered. <br /> yielded about5%auxotrophs             indicates     indicates RPCl60 itself located chro- <br /> pSP4 belost rpcl60 : : URA3 background, <br />                                         + <br />                                                               mosome XV. <br /> showing R P C l 6 0 allele essential <br />                             +                                    T o precisely localize RPCl6O chromosome XV, viability vegetatively growing cells.               constructed haploid strain T4-1 (ura3- <br />          map          RPCl60&apos; chromosome, <br />                               ato                          52, his3-A200,ade2-101) carrying haplolethal <br /> rpcl60 :: URA3 allele constructed tech- <br />                  +                                            allele rpcl60::URA3          andthe <br />                                                                                              +       wild-type allele <br /> nique described diploid strains (D8,           R P C l 6 0 + tandemduplication.     strain, <br /> D9 D10) homozygous ura3-52              R P C l 6 0 genetically marked URA3 insert                 + <br />  <br />  <br />  <br />  <br /> s p o l l , heterozygous variety genetic         butthe haploid mutant viable markers located 16chromosomes                  adjacent RPCl60 allele. Preliminary results indi- <br />                                                                                      + <br />  <br />  <br />  <br />  <br /> identified S . cerevisiae (Table 1). spoll muta-       catedthat RPCl6O meiotically linked tion eliminates meiotic crossing com-            ADE2. prompted cross T4-1 ura3-52 <br /> pletely preventing production viable haploid           tsm 8740 partner cross heterozygousfor <br /> spores meiotic offspring, providesa              RPC160, ADE2, TSM 8740 HIS3 simple chromosome allocation technique (KLAPHOLZ              chromosome XV, TSM8740 close ADE2. EASTON-ESPOSITO        1982). particular case,    T h e segregation RPCl6O followed viable haploid spores harboring            segregation URA3 character. Seventy                                                                                                      + <br />  <br />  <br />  <br />  R P C l 6 0 allele consequently auxo- <br />              +                                                tetrads analyzed: 55were parental R P C l 6 0 , <br /> trophic uracil. meiotic crossing        ADE2 TSM8740. undergone crossing- R P C l 6 0 + allele present viable         RPCl6O TSM8740, remained <br /> spores cosegregate genetic markers            parental theRPCl60-ADE2 interval. remain- <br /> present cis chromosome,             ing 13 undergone crossing ADE2 linkage markers located             R P C l 6 0 remained parental theRPCIGO- <br />                                                      RNA Polymerase Mutants Yeast                                               52 1 <br />  <br />                              URA 3                                    FRIESEN 1987; NONET            et al. 1987). sumarized <br />                                                                       Figure2, 5.3-kbXbaI fragment RPCl60 lacking <br />                                                                       386 nucleotides C-terminal end  <br />  <br />                        w <br />                                                                       including N-terminal endwas prepared single- <br />                                                                       stranded DNA plasmid YIp32-1 (URA3+), tak- <br />                                                                                                           ing advantage       f l origin replication harbored <br />                                                                       plasmid (BALDARI CESARENI              1985).                 ,                                                   DNA treated NaHS03and amplified                                                                       transformation ung strain E . coli generate <br />                        t.rg.1.d Intogntlon <br />                                                                       pool 2000 clones mutagenized double- <br />                                                                       stranded plasmids. mutagenized plasmids                                                                       integrated genome          ura3-52 yeast strain <br />                                URA3          5&apos;       3&apos; <br />                                                                       YNN281 transformation targeted crossing- <br />                                                                       (ORR-WEAVER,          SZOSTAK     ROTHSTEIN      1981) <br />                Cla                                Cldl <br />                                                                       RPC160 unique ClaI site. generated                                                                     tandem duplication RPCl60, inactive <br />  <br />  <br />  <br />  <br />               - <br />                                                                       copy lacking C-terminal end functional <br />                                1                                      copy 3.7 kb coding sequence <br />                                                                        ( . e , , region proximal ClaI) arosefrom                                                                       mutagenized plasmid. plasmid-born mutation                   5&apos;                3&apos; <br />                                                                       phenotypically expressed gen- <br />                                                                       omic integration provided located upstream <br />                mutrnt 811010    Ch10mOWm8l   bCUS                     ClaI site recombine functional <br />     FIGURE2.-Construction      fo chromosomal mutant alleles       copy RPCl60. Mutant            excised URA3   + <br />  <br />  <br />  <br /> RPC160 integrative crossover plasmid-born RPC160            plasmid fromthechromosome                   aspontaneous <br /> insert. single-stranded preparation plasmid YIp32-1 contain-     crossover direct repeats RPC160 <br /> ing XbaI insert RPC160 lacking 3&apos; end mutagenized NaHSOs amplified ung E . coli strain generate     selected described BOEKE, LACROUTE              pool double-stranded mutant plasmids. Plasmid integration        FINK (1984). Provided thatthe               excisive crossing <br /> targeted crossover (ORR-WEAVER,    SZOSTAK  ROTHSTEIN 1981)        located upstream ts mutation ClaI site generates tandem duplication RPC160        RPCl60, mutation remain chromo- truncated complete RPC160 sequence. Potential ts <br />                                                                        somal copy RPC160 plasmid excision. mutations [symbolized (* present plasmid trans- <br /> mitted nontruncated RPC160 copy provided          final result precise substitution wild-type <br /> located upstream ClaI site (ORR-WEAVER,      SZOSTAKand        RPCl60 gene mutagenized RPCl60 fragment <br /> ROTHSTEIN1981; SHORTLE,NOVICKand BOTSTEIN1984).                    (Figure 2). <br /> resulting transformant express ts phenotype.            5000 URA3 transformants isolated <br />                                                                                                        + <br />  <br />  <br />  <br />  <br /> Spontaneous excision plasmid intergenic crossover <br />                                                                        1 week incubation 25&quot;, showed little RPCIGO repeats restores single copy RPC160 harbors ts mutation provided excisive crossing      growth replica plated YPD 37&quot; occured upstream mutated site.                             grew 25&quot;. temperature-sensitive character                                                                        uracil auxotrophy. Tetrad analysis TMS8740 interval. Ninety tetrads various                      cross ura3 tester strainNC93SP1 showed factor crosses ADE2 ts alleles                  mutants ts charactersegregated <br /> RPCI6O   analyzed.                      independentlyfromthe              URA3 markerand                                                                                                               + <br />  <br />  <br />  <br />  <br /> data, sumarized Table 2, allow precise localiza-                  born integrated plasmid.  <br /> tion RPCl60 chromosome X V , TSM                          genetic evidence establishes  <br /> 8740 ADE2, 3 cM marker                        maining ts mutations located integrated <br /> (Table 2). tsm8740 mutation complemented                    YIp32-1 plasmid (Table3).(1)Theyare                  recessive, transformation thecentromeric LYS2 +,                          2:2 segregation, cosegregate URA3      + <br />  <br />  <br />  <br />  <br /> RPCl60 plasmid pSP4 notallelic           + <br />                                                                        strongly linked ade2. (2) Spontaneous plasmid <br /> RPCl60.                                                                excision strongly correlates reversion ts <br />    Construction temperature-sensitive mutant al-                    character. (3)Conversely, reversion ts mutation <br /> leles RPC160 random mutagenesis vitro:                        frequent (between5 X                   andlop3per <br /> Temperature-sensitive (ts) mutant alleles RPCl60                    cell) accompanied isolated anapproach previously usedto                          plasmid excision. Reversion plasmid excision <br /> obtain ts mutants genes encoding actin,                     ascribed tointergenic conversion unequal <br /> poisomerase I1 largest subunit RNA poly-                     crossing directrepeats                  merase B S. cerevisiae (SHORTLE, NOVICK   BOT-                  RPCI6O (KLEINand PETES1981). (4 trans- <br /> STEIN 1984; HOLMet al. 1985; HIMMELFARB,      SIMPSON                  formant, obtained spontaneous subclones      522                                                                 R. Gudenus et al. <br />                                                                             TABLE 3 <br />                                                                    Properties t s mutants <br />  <br />                                        Doubling time                              Meiotic linkage&apos;                     Mitotic instability <br />                                                YPD <br />                                                hr)&quot;                            URA3    +       add                                 Fre uency (%)                                                                              ~~ <br />  <br />                                                                                                                  Frequency           Jasmid loss <br />                                                                                                                 reversion                                                                                                                                                                                                                                                                              L   S   + <br />  <br />  <br />  <br />  <br />               Strain                    25&quot;              37&quot;                      P T N                          plasmid loss            revertants&apos; <br />  <br /> YNN281-15 (rpcl60-15)                  4.0               15                       160     0    151 0       17       22125 <br /> YNN281-20 (96160-20) 16            8.8 3.5                                           1*   0    16 1 0               21/25                       26 <br /> YNN281-31 (rp~160-31)                  4.3          &gt;20                            8 0    0     8 0 0               2 1/25                      62 <br /> YNN281-41 (rpc160-41)                  4.5            1230 <br />                                                     &gt;20                           150     0        29               24/25 <br />        YNN288: 2.5 hr (25&quot; 2.7 hr (37&quot;). <br />      &apos; P, T N defined Table 2. lJRA3+ character allows follow segregation chromosomally integrated plasmid <br /> bearing ts mutation original construction. Tetrad analysis based asci viable spores. tetrad labeled (*) classified &quot;tetratype&quot; corresponded ascus temperature resistant spore prototrophic uracil  , harbored plasmid YiP32-1) segregant resulted spontaneous reversion ts allele concomitant <br /> plasmid excision, possibly intergenic conversion (KLEINand PETES1981). <br /> ~   Average thiee experiments set 50 revertants  <br />  <br />  <br />  excised plasmid remained ts                                       generations. Minor differences growth rates (Table <br /> crossover localized upstream ts mutation).                                   3) suggest mutants correspond    todistinct <br /> expected, ts mutation stable                                      mutations. Cell morphology normal cell subclones  revertant lo6 cells),                               appeared arrested atvarious cell cycle stages intergenic conversion unequal crossing                                  judged percentageof budded cells longer possible copy                                      size buds. Diploid cells YNN281-41 X D27-7c <br /> RPCl6O left chromosome.                                                         (homozygous rpcl60-41) precultured 25&quot; <br />    constructions yielded recessive mutant                                  liquid YPD plated sporulation medium alleles (rpcl60-15, rpcl60-20, rpcl60-31 rpcl60-                                  temperature. yielded 5% asci <br /> 4 1 ) result precise substitution                                 mature spore asci) compared <br /> wild-type RPCl60 gene mutated gene.                            +                                                          45 asci observed RPCl60+ diploid <br /> contrast initial transformants, genet-                                YNN281 X YNN282 experimental <br /> ically stable harbor foreign    DNA sequence                                   conditions. Practically asci observedwhen <br /> (originatingfromthetransforming         vector) inthe                                 mutant diploid precultured 30&quot; vicinityof RPCl6O. Admittedly          ts mutations                               25&quot; followed 16-hr shift 37&quot;. map gene fragment immediately proximal                                        measuredtheincorporation        labeled RPCl6O mutagenized insert. T o eliminate                                    uracil tRNAs, 5s rRNA 5.8s rRNA wild- possibility, established rpcl60                                 type,rpc160-31 rpc160-41 cells exponentially <br /> mutants define single complementation group                                      growing 25&quot; cells shifted 37&quot; 1 hr. <br /> mating rpcl60-41 segregant andshowing                                     used eithera45imin pulse a15-min pulse resulting diploids temperature-sensitive.                                followed 30-min chase 400-fold excess crossed lys2-801, rpcl60-15 mutant                                       unlabeled uracil. conditions, strain T12-2B (pSP4 rpcl60:: URA3                   + <br />                                                                                       effect rpcl60-41 mutation protein syn- <br /> viable harborsthecentromeric        plasmid                                thesis total RNA synthesis measured  pSP4 (LYS2 +, R P C 1 6 0 + ) .   resulting                                      incorporation [14C]lysine [3H]uracil acid <br /> rpcl60 : : URA3 +lrpc160-15 (pSP4) diploid tem-                                   precipitablematerial(datanotshown).       incor- <br /> perature-resistant RPCl60 allele             +                      poration [3H]uracil tRNAs, 5 s rRNA 5.8s <br /> pSP4.  spontaneous curing plasmid                                      rRNA 45-min pulse measured au- <br />  (leading tolysine auxotrophy) yielded ts phenotype,                                toradiography quantified densitometry (Fig- <br /> indicating rpcl60-15 notcomplemented                                       ure 3). compared wild type, accumulation <br /> rpcl60 :: URA3 mutations <br />                        + <br />                                                                                       tRNAs reduced times rpcl60- <br />  define gene.                                                                41 mutant 25&quot; reduced factor <br />     Phenotype characterizationof ts mutants:                                  cells shifted to37&quot;  ts mutantshardly grow 37&quot; anabout                                         1 hr 16 hr. effect 5.8s rRNA <br />  twofold reduced growth rate 25&quot; compared                                    accumulation 25&quot; o r 37&quot;  surprisingly, <br />  RPCl60+ strain YNN281. temperature                                        little effect 5s rRNA accumulation 25&quot;  shift, growth  measured    turbidometry 600                                 37&quot; 1 hr, rRNA known  nm) arrestedafterthreetofour           cell                              synthesized RNA polymerase C. 5srRNA <br />      huI[  W T(16-c) <br />                                           LA <br />                                                   RNA <br />                                                    Mutants <br />                                                       Polymerase <br />  <br />  <br />  <br />  <br />                                           41 (irh 37&apos;C) <br />                                                                           ~                                                                                     Yeast <br />  <br />                                                                                              llill               M <br />                                                                                                                  B <br />                                                                                                                                     323 <br />  <br />  <br />  <br />  <br />                                                                                                                  0                                            n                                                                                   #   - <br />  <br />  <br />  <br />  <br />     k Lk J ,4v   (15.c)                  41   (th .t 37&apos;C) <br />                                                                               1 2 3 4 <br />                                                                          FIGLRE   4. vitro transcription yeast tRNA$&quot; gene                                                                       cell-free extracts RPC160 strain D20B- 12                                                                       rpc160-31 strain D25-13A <br />                                                                                               5 6 7 8 <br />  <br />                                                                                                         + <br />  <br />  <br />  <br />  <br />                                                                                                 collected <br />                                                                                                                   9101112 <br />  <br />  <br />                                                                                                              log phase cultureson <br />                                                                       YPD 25&quot;. Panel  5 , 15, 25 35 pg protein wild- <br />   FIGURE3.-Incorporation    [&apos;HH-5,6]uracil small-sized       type cell-free extract (lanes 1-4) mutant cell free extract <br /> RNA 45 min vivo labeling 25&quot; 37&quot; strain      (lanes 5-8) assayed specific transcription tRNA&quot;&apos;&quot; <br /> YNN281 ( R P C 1 6 0 + ) D27-7C (rpc160-41). R N        gene plasmid p02 25&quot;. R N synthesized separated <br /> prepared, separatedona         polyacrylamide gel autoradi-       6% polyacrylamide gel  MATERIALS ASD METHODS). ographed described YATERIALS ASD METHODS. Autoradi-             arrow points tRNA band. Panel B: assay ograms quantified densitometry. Left panel: wild type         presence 20 pg wild-type proteins (lane 9) 20 k g <br />   mutant   exponential cultures 25&quot;. Right     mutant proteins (lane lo), mixture (lane 11) panel: mutant cells shifted 37&quot; 16 hr  1 hr  .   40 pg wild-type proteins (lane 12). <br />  <br />  <br /> synthesis strongly reduced cells                    Selection extragenicsuppressorisolated <br /> transferredfor 16 hrat 37&quot;. Similar results                      therpcl60-41 strain: exponential culture obtained 15-min pulse chase,                 strain YNN281-41 (rpcl60-41) UV mutagenized <br /> indicating 5.8s rRNA, 3s rRNA, rRNA                      toabout 30% survival plated YPD 37&quot;. stable mutant andwild-type cells (data                Colonies growing temperature appeared shown).  differences observed                    1 week frequency 1 10&apos; viable cells. accumulation tRNA 5 s rRNA                          reisolated doublingtime <br /> mutant cells likely entirely differences              YPD measured 37&quot;. grew rapidly rate synthesis molecules. Similar                  wild-type strain gave ts segregants results obtained rpcl60-31 mutant D25-                  backcrossed RPCI6O&apos; strain YNN282. 13A (data shown).                                                 likely result reversion rpcl60-41    RNA polymerase C activity cell-free extracts:                   R P C l 6 0 &apos;. seven thatgrew slowly mutants (rpc160-31 rpcl60-41) tested                     YPD 37&quot; (doubling times 360-400 min), activity RNA polymerase C cell-free                     particular revertant (Y NN28 1-41-3) backcrossed <br /> extracts cells exponentially growing 25&quot;, using                 rpcl60-41 strain D27-7C YNN282 specific assay transcription tRNA genes.                     (RPC160 &apos;). Tetrad analysis showed thecapacity <br /> Figure 4 shows thatthetranscription         activity             grow 37&quot; monogenic (2:2) segregation cell-free extract rpcl60-31 strain strongly                cross, YNN281-41-3 reduced comparedto RPC160&apos;  similar                          crossed YNN282, ts phenotype recovered <br /> somewhat markedeffect          observed                 25% spores. results indicated <br /> rpcl60-41; datanot       shown). Mixing experiments                   capacity grow 37&quot; monogenic <br /> showed reduced activity                  suppressor rpcl60-41 genetically unlinked <br /> presence inhibitor transcription mutant               RPC160. corresponding mutation extract (Figure 4, lanes 9-12). assay                    called pcil-3  polymerase C interaction). 25&quot;, similar results obtained                    growth rate pcil-3 RPC160&apos; segregant YPD <br /> extract incubated 37&quot;, showing marked                  25&quot; 37&quot; indistinguishable wild type <br /> temperature sensitivity mutant enzyme                    showing pcil-3 mutation  experimental conditions (data shown).                       tectable growth phenotype. segregant  524                                              R. Gudenus et al. <br /> identified capacity restore original sup-    integrated plasmid, close linkage ADE2 pressorphenotype crossed intoa rpc160-41              chromosome XV, frequentreversion spon- <br /> background.                                                taneous plasmid excision failure com- <br />                                                            plement rpcl60:: URA3 null allele.                       DISCUSSION <br />                                                                                        + <br />  <br />  <br />  <br />  <br />                                                            alleles distinct mutations judged <br />    Previous work (INGLES     et al. 1984; ALLISONet al.    differences reversion patterns  1985;RIVA et al. 1986; THURIAUX        et al. 1986)   growth rates 25&quot; 37&quot;. <br /> identified RPC160 auniquegeneencodingthe                   ts mutants analyzed (rpcl60- <br /> largest subunit RNA polymerase C. gene         31 rpc160-41) defective tRNA synthesis <br /> amarked homology onesencodingthe                p&apos;  vivo cell-free extracts. synthesis <br /> subunit E . coli RNA polymerase (ALLISON         et al. 5.8s rRNA catalyzed RNA polymerase  1985) large subunitof RNA polymerase         affected. anticipated, fully defective rpcl60 <br /> B yeast (ALLISON     et al. 1985; RIVAet al. 1986).     mutants unable undergo spore germination <br /> R P C l 6 0 maps right arm chromosome XV,        (INGLES  et al. 1984) vegetative growth  work). <br /> close allelic TSM8740, gene encoding         ts mutant rpcl60-41 fivefold <br /> presumably essential hitherto unidentified func-       reduction tRNA synthesis vivo 25&quot; tion (HILGER,PREVOTand MORTIMER,1982). SPO-                twofold reduction growthrate rich <br /> T 4  involved sporulation (TSUBOI mediumatthesametemperature,indicatingthat <br />  1983)] maps veryclose TSM8740 (possibly           tRNA synthesis  step, R P C l 6 0 ) allelic    transcription thecorrespondinggenes         RNA <br /> RPC160 homozygous rpcl60 diploidshavea               polymerase C) major rate limiting factor strongly reduced level sporulation 25&quot;. RIVAet       cellular growth. <br /> al. (1986) cloned 22 genes potentially             Temperature-sensitive mutants need ther- <br /> encoding various subunits RNA polymerase  B           mosensitive forthe activity thecorresponding C. restriction maps overlap         enzyme, instead affect synthesis  18.4-kb insert cloned present work          stability), especially case complex multimeric <br /> unpublished results).  data        far  enzymes RNApolymerases. ts mutants <br /> suggest remarkable feature organization         defective assembly RNA polymerase surrounding region chromosome,           described E. coli (KIRSCHBRAUM       et al. <br /> certainly argue cluster genes encoding        1975). yeast, MANNet al. (1987) isolated <br /> RNA polymerase subunits.                                   plasmid-born ts alleles RPC40, gene encoding <br />    Temperature-sensitive rpcl60 mutants ob-           AC 40 subunit common RNA polymerase tained vitro mutagenesis NaHS03            C. corresponding mutations prevent non-replicative plasmid subsequently          vivo assembly RNA polymerase C serted chromosome integrative transfor-        affect activity preassembled enzyme. <br /> mation (SHORTLE,    NOVICKand BOTSTEIN        1984).  similar conclusion reachedforts mutations <br /> procedure somewhat cumbersome              gene encoding        largestsubunit ofRNA <br /> alternativeapproach      based plasmid exchange         polymerase B S. cerevisiae (HIMMELFARB,      SIMPSON <br /> (MANNet al. 1987;BUDD CAMPBELL1987)                FRIESEN 1987).     contrast, mutation isolated advantage yielding mutations         gene NONET et al. (1987) appears precisely substituted wild-type allele     affect activity intracellular accu- chromosome, thanyielding mutations born         mulation enzyme, completely lacks RNA <br /> onan artificial extrachromosomalreplicon. al-           polymerase B activity vitro cells grown <br /> readyobserved(SHORTLE, NOVICK BOTSTEIN permissive temperature. <br />  1984) ts mutants unlinked mutagenized gene            present case, ts mutants analyzed generated (2 5000 transformants).         additional fourfold decrease tRNA syn- explanation unlinked ts mutants        thesis vivo hour shift 37&quot;.  integrative transformation restricted small      unlikely mutations merely prevent cell subpopulation competentfor mitotic           assembly RNA polymerase C affecting recombination (HENACT LUZZATI1972; MINET,              properties enzyme present temper- <br /> GROSSENBACHER-GRUNDER                                                    THURIAUX    1980)   ature shift. case, preexisting RNA prone high level sponta-         polymerase molecules continuetoproduce <br /> neous mutation.  observed mutagenic          RNA hoursbeforebecominglimiting, <br /> effect transformation spontaneous mutation        assuming RNA polymerase C unusually <br /> monitored resistance canavanine. addition         labile enzyme.  exclude unlinked ts mutants, ts alleles RPClGO      mutation itself temperature-sensitive effect identified cosegregation       stability enzyme  associated                                          RNA Polymerase Mutants Yeast                                                    525 cognate transcription complexes)          possibility pczl-3 encodes protein directly <br /> direct effect transcriptional activity. partic-   interacting C160 subunit, example ular, hypothesis explain reduced        transcription factor subunit poly- <br /> RNA polymerase C activity mutant cell-free           merase. <br /> extracts particularly temperature-sensitive.            thank CARLMANNand PIERRE      FROMAGEOT   stimulating <br />    striking feature mutants analyzed        discussions encouragement. work partly supported largelyspecific forthe synthesis           long term EMBO Fellowship R.G. grant tRNA, synthesis 5s rRNA eventually          Centre National la Recherche Scientifique (ATP 033-542). arrested doubling times 37&#226;&#8364;?. indicates mutantenzyme   possibly, wild type form  discriminates                           LITERATURE CITED <br /> structurally related initiation complexes formed         ALLISON,D. S., S. H. GOH B.D. HALL,1983 promoter <br /> tween tRNA 5s rRNAgenes                    sequence yeast tRNA Tyr gene. Cell 34: 655-664. <br /> proteins acting transcription factors (SENTENAC          ALLISON,L. <br />                                                                      ,      M. MOYLE,M. SHALESand C. J. INGLES, <br />                                                                1985 Extensive homology amongthe largest subunits  1985). possibility mutationally modi- <br />                                                                eukaryotic prokaryotic RNA polymerases Cell 42: 599- <br /> fied C160 subunit specifically altered recogni-           610. <br /> tion site initiation complexes. Alternatively,      BALDARI,   C., G. CESARENI,     1985 Plasmids pEMBLY:new affinity wild type RNA polymerase C            single stranded shuttle vectors recovery analysis lower tRNA initiation complexes,             yeast DNA sequences. Gene 35: 27-32. <br /> account reduced synthesis tRNAs ts        BARNES,   D , J. THORNER,     1986 Genetic manipulation                                                                 Saccharomyces cerevisiae use LYSZ gene. Mol.Cell. <br /> mutations lead decreased intracellular concen-             Biol. 6: 2828-2838. <br /> tration enzyme.                                      BOEKE,J. D., F. LACROUTE       G.R. FINK,1984 positive <br />    ribosomal repeat unit, 5s rRNA                selection mutants lacking orotidine -5&#226;&#8364;&#8482;-phosphate decar- <br /> gene lies major promoter element                boxylase activity yeast: 5-fluoro-<span id='am-9' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orotic acid</span> resistance. Mol. <br />                                                                 Gen. Genet. 197: 345-346. initiation site large rRNA gene,          BUDD,M., J. L. CAMPBELL,     1987 Temperature-sensitive mu- <br /> transcribed opposite orientation (ELIONand WAR-              tations yeast DNA polymerase gene. Proc. Natl. Acad. <br /> NER 1984). suggests transcription          Sci. USA 84: 2838-2842. <br /> 5s rRNA gene RNA polymerase C berequired             CAMIER,  S., 0. GABRIELSEN,  R. BAKER    SENTENAC,     1985 thesynthesis ofthe 35s rRNA RNA polymerase               split binding site transcription factor tRNAsG&#226;&#8364;&#8482;&#226;&#8364;?  ensuring coordinate production              gene. EMBO J. 4: 491-500. <br />                                                             DENTE,L.,G. CESARENI      R. CORTESE,    1983 pEMBL: new <br /> types rRNA.  data establish             family single stranded plasmids. NucleicAcidsRes. 11: <br /> synthesis 5.8s rRNA   consequently,               1645-1655. <br /> 35s rRNA) affected severe reduction          DUNCAN,K.,  B. <br />                                                                         P.            ROCKSTROH       H. R. WARNER, rate 5s rRNA synthesis.  pe-               1978 Escherichia coli K12 mutants deficient Uracil-DNA <br /> culiar organization rRNA repeat unit S .              glycosylase.J. Bacteriol. 134: 1039-1046. <br />                                                             ELION,E. , J. R. WARNER,1984 major promoter <br /> cerevisiae does result tight coordination           element rRNA transcription yeast lies 2 kb upstream. transcription rDNA genes byRNA poly-                 Cell 39: 663-673. <br /> merases C.                                            FELDMAN,   H., J. OLAH andH. FIEDENREICH,      1982 Sequence    conclusion, present work provides             yeast DNA fragment containing achromosomal replicator characterization conditional mutants specifically            tRNA Glu gene. Nucleic Acids Res. 9: 2249-2959. <br />                                                             FRISCHAUF,    M., H. LEHRBACH,      POUTSKA    N. MURRAY, <br /> defective RNA polymerase C. properties                1983 Lambda replacement vectors carrying polylinker se- mutant RNA polymerase C enzymes                   quences. J. Mol. Biol. 170: 827-842. <br /> analyzed biochemically order identify mo-         H~NAUT     , <br />                                                                          , M. LUZZATI,     1972 ContrBle I&#226;&#8364;&#8482;aptitude lecular defect leads temperature-sen-           recombiner pendant la phase vkgktativechez Saccharomyces <br /> sitive phenotype, parallel study            cerevisiae. Mol. Gen. Genet. 116: 26-34. <br />                                                             HILGER,F., M. P R ~ V Oand  T R. K. MORTIMER,         1982 Genetic <br /> gene RPC40 (MANNet al. 1987). Yeast genes corre-               mapping arg, cpa, car tsm genes Saccharomyces cerevisiae. <br /> sponding subunits RNA polymerase C                 Curr. Genet. 6: 93-98. cloned immunological screening (RIVA           HIMMELFARB,      H. J., E. M. SIMPSONand J. D. FRIESEN, <br /> et al. 1986) currently characterized           1987 Isolation characterization temperature-sensitive laboratory. genes related RNA              RNA polymerase I1 mutants Saccharomyces cerevisiae. Mol. Cell. <br />                                                                Biol. 7: 2155-2164. <br /> polymerase C function identified selecting        HOLM,C., T. GOTO,J. C. WANCand D. BOTSTEIN,1985 DNA extragenic suppressors rpcl60 ts mutants.            topoisomerase I1 required time mitosis yeast. <br /> deed, ts alleles described (rpcl60-41)         Cell 41: 169-173. suppressible extragenic suppressor mutation,       INGLES, C. J., H. J. HIMMELFARB,   M. SHALES,   L. GREENLEAF   pcil-3, itself detectable growth phe-          J. I3. FRIESEN,1984 Identification, molecular cloning                                                                mutagenesis Saccharomyces cereviriae RNA polymerase genes. <br /> notype. indication asto molecular               Proc. Natl. Acad. Sci. USA 81: 2157-216. <br /> mechanismofthis        suppression, obvious         ITO, H., Y. FCKUDA,K. MURATA  KIMURA. <br />  526                                                      R. Gudenus et al. <br />     1983 Transformation intact yeast cells treated alcali     PEARSON, D.,     WILLIS,H. HOTTINGER, J.       BELL  KUMAR,      U. <br />     ions J. Bacteriol. 153: 163-168.                                     LEUPOLD     D. SOLL,1985 Mutations preventing expression <br /> JOHNSTON, L. H., 1977 use novel plate assay search         sup3 tRNASernonsense suppressors Schizosaccharomyces <br />     yeast mutants defective desoxyribonucleases. Mol. Gen.        pombe. Mol. Cell. Biol. 5: 808-815. <br />     Genet. 152: 219-222.                                              RIVA,M., S. MEMET,     J. Y. MICOUIN,J. HUET  TREICH,       J. DASSA, <br /> JOSES, E., 1976 Proteinase mutants Saccharomyces cerevztiae.          R. YOUNG,J. M. BUHLER,           SENTENAC      P. FROMAGEOT, <br />     Genetics. 85: 23-33.                                                 1986 Isolation structural genes yeast RNA polymerases <br />  KIRSCHBAUM,   J. B.,  V. CLAEYS,  S. NASI,B. MOLHOLTand J. H.         immunological screening. Proc. Natl. Acad.Sci. USA 83: <br />     MILLER,1975 Temperature-sensitive RNA polymerase mu-                  1554-1558. <br />     tants alteredsubunit synthesis degradation. Proc.        ROTHSTEIN,   R.L. 1983 step gene disruption yeast. Meth- <br />     Natl. Acad. Sci. USA 72: 2375-2379.                                  ods Enzymol. 101: 202-210. <br />  KLAPHOLZ,   S., R. EASTON-ESPOSITO,     1982 new mapping       RUET ,  SENTENAC,        P. FROMAGEOT,    B. WINSORand F. LAC. <br />     method employing meiotic rec mutant ofyeast. Genetics              ROUTE,1980 mutation B 220 subunit gene affects <br />     100: 387-412.                                                        structural functional properties yeast RNA poly- <br />  KLEIN, H. L., T. D. PETES,1981 Intrachromosomalgene                 merase B vitro. J. Biol. Chem. 255: 6450-6455. <br />     conversion yeast. Nature 289: 144-148.                         RUET, , S. CAMIER,        W. S. MAGOWICZ  SENTENACandP. <br />  KLEKAMP,  M. S., P  WEILL,1982 Specific transcription          FROMAGEOT,       1984 Isolation class C transcription factor <br />     homologous class 111 genes yeast-soluble cell-free extracts.      forms stable complex tRNA genes. EMBO J. 3: <br />     J. Biol. Chem. 257: 8432-8441.                                       343-350. <br />  KOSKI,R. , D. S. ALLISON,     M. WORTHINGTON     B. D. HALL,   SENTENAC,   , 1985 Eukaryotic RNA polymerases. CRC Crit. <br />      1982 vitro RNA polymerase 111 Saccharomyces        Rev. Biochem. 18: 31-90. <br />     cereuisiae: effects deletions point mutations SUP4    SHERMAN,   F., G. R. FINK J.B. HICKS,1979 Methods Yeast <br />     transcription. Nucleic Acids Res. 10: 8127-8143.                     Genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, <br />  MANIATIS,T.,E.F. FRITSCHand J. SAMBROOK,            1982 Molecular      N.Y. <br />     Cloning: Laboratory Manual. Cold Spring Harbor Laboratory,      SHORTLE,   D., D. BOTSTEIN,     1983 Directed mutagenesis     Cold Spring Harbor, N.Y.                                              sodium bisulfite Methods Enzymol. 100: 457-468. <br />  MANN, C., J. M. BUHLER  TREICHand  SENTENAC,                     SHORTLE,   D., P. NOVICKand D. BOTSTEIN,           1984 Construction <br />      1987 RPC40, auniquegene asubunitshared                  genetic characterization temperature-sensitive mutant <br />     yeast RNA polymerases C. Cell 48: 627-637.                      alleles yeast actin gene. Proc. Natl. Acad. Sci. USA 81: <br />  MELTON,D , P.  KRIEG,M. R. REBAGLIATI,         T. MANIATIS, K.      4889-4893. <br />     ZINN M.R. GREEN,1984 Efficient invitro synthesis           THURIAUX    P., C. MANN,    J. M. BUHLER,  1. TREICH,   R. GUDENUS,  S. <br />     biologically active RNA RNA hybridization probes             MARIOTTE,      M. RIVAand  SENTENAC,        1986 Gene cloning <br />      plasmids containing bacteriophage SP6 promoter. Nucleic            mutant isolation subunits RNA polymerases     Acids Res. 12: 7035-7056.                                             yeast Saccharomyces cerevisiae. pp. 5 19-53 1 . Extrachromosomal <br />  MINET, M.,  M. GROSSE <br /> </body></html>