<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1997 Genetics Society America <br />  <br />  <br />  <br />        New Enhancer Position-Effect Variegationin Drosophila melunogaster <br />             Encodes Putative RNA Helicase Binds Chromosomes <br />                         Regulated Cell Cycle <br />  <br />                     Daniel F. Eberl,**&#226;&#8364;&#8482;Lori J. lor ,*^^^&apos; Michael B. Melnick,*&#226;&#8364;?72Vanita S O O ~ , ~ ~ ~ <br />                                         Paul h k o t Norbert Perrimon*7: <br />    *Department Genetics, Harvard Medical School, Boston, Massachusetts 021 15, tDepartment Biology, McGill University, <br /> Montreal, Quebec, Canada H3A B :Howard Hughes Medical Institute, Harvard Medical School, Boston, Massachusetts 02115 <br />                                                        Manuscript received February 3, 1997 <br />                                                       Accepted publication March 31, 1997 <br />  <br />                                                                ABSTRACT <br />                   Urosophila melanogaster, positioneffect variegationof white gene beena useful phenomenon <br />                study chromosomestructure andthe genes modify        identified new enhancer <br />                variegation locus, Dmrnahel (hel).Deletion mutation hel enhances white variegation, thiscan <br />                reversed transformed copy  presence endogenous copies, transformed <br />                behaves suppressor variegation. he1 essential gene functions maternally                zygotically. HEL protein similar known RNA helicases, butcontains unusual variant (DECD) <br />                DEAD motif common proteins. Potential     HEL homologues mammals, <br />                yeast worms.HEL proteinassociateswithsalivaryglandchromosomes                 andlocates nuclei                embryos ovaries, disappears mitotic domains embryos chromosomes condense.                propose HEL protein promotes open chromatin structure favors transcription                development regulating spreadof heterochromatin, andthat HEL regulated  andmay                role  mitotic cell cycle embryogenesis. <br />  <br />  <br />  <br /> P    OSITION-EFFECT variegation (PEV euchro- <br />       matic gene expression occurs chromosome <br /> rearrangements place euchromatic genes het- <br />                                                                             genetic properties facilitate identification                                                                             loci involved PEV. genes act                                                                             haplo-suppressors PEV cloned,  <br /> erochromatic regions. Variable spreading hetero-                          appear encode proteins associate chromatin juxtaposed euchromatic     genes dur-                    modify chromatin (EISSENBERG       et al. 1990,1992; REU- <br /> ing development thought inactivate genes, cre-                    TER et al. 1990; GARZINO     et al. 1992; TSCHIERSCH    et al. <br /> atinga    clonally inherited variegated, mottled,                        1994). HP-1 heterochromatin-associated protein  <br /> pattern gene expression (EISSENBERG    1989; TARTOF                      coded Su(var)205  known asSuvar(2)5);Su(var)7 <br /> et al. 1989; HENIKOFF   1990; REUTER SPIERER   1992).                    called Suvar(3)7) encodes putative zinc-finger <br /> PEV manifested complete inactivation gene                        protein modulo gene produces DNA-binding <br /> expression individual cells thought occur                      protein. Su(var)3-9contains regions similarityto level transcription (HENIKOFF 1981; RUSHLOW                       negative positive transcription regulators. addi- <br /> et al. 1984; KORNHERand KAUFFMAN 1986).                                     tion, protein phosphatase encoded Suvar(3)G acts <br />    product white gene Drosophila                        haplo-suppressor PEV postulated <br /> quired redeye pigmentation sensitive PEV.                     modify structural components heterochromatin, chromosomalinversion, Zn(l)wm4,places white                       zinc-finger proteinencoded Su(var)7 heterochromatic region Xchromosome                           (BAKSAet al. 1993). cloned haplo-enhancersof PEV <br /> causes variegation white gene expression, resulting                   appear encode chromatin-associated proteins. mosaic pattern red eye pigmentation (SCHULTZ                           E(var)?-93D encodes protein contains domain <br /> 1936). Deletions mutations number loci                          transcriptional regulators dominantly suppress enhance Zn(l)wm4variegation.                         localized polytene chromosomes (DORNet al. 1993a), <br />                                                                             Trithmax-like encodes putative zinc-finger <br />   Corresponding author: Daniel F. Eberl, Department Genetics, Har- <br /> vard Medical School, 200 Longwood Ave., Boston, MA 02115.                   protein likely Drosophila GAGA factor <br /> E-mail: eberl@rascal.med.harvard.edu                                        (FARKAS   et al. 1994). GAGA factor, initially deemed <br />   authors contributed equally.                                      transcription factor, implicated <span id='am-26' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>  <br />   &#226;&#8364;&#732; Present address: New England BioLabs, , Beverly, MA 01915-          pendent chromatin remodeling (Lu et al. 1993; TSUKI- <br /> 55510. <br />                                                                             YAMA et al. 1994). Recently, modiJer mdg4, turns <br />   &#226;&#8364;&#339;Present address: Department Cell Biology, Harvard Medical <br /> School, Boston, MA 02115.                                                   identical E(var)3-93D shown interact <br />     Present address: Department Molecular Medical Genetics,          supPressor Hairy wing (su(Hw)) impart <br /> University Toronto, Toronto, Ontario, Canada M5S 1A8.                    directionality onthe chromatin insulating ability Genetics 146: 951-9fi3 (July, 1997) <br />  952                                                          D. F. Eberl et al. <br />  <br /> su(Hw) (GERASIMOVA     et al. 1995). enhancer                 clones D. he1 sequenced using Sequenase (United <br /> PEV, Evurl, encodes aubiquitin-specificprotease (HEN-                    States Biochemical) described DEL SAL. et al. (1989). <br />                                                                          Similarities DEAD box family members, rathomologue, <br /> CHOZ et d . 1996). <br />                                                                          recently, human pig homologues    cloning identification                    searching protein databases NCBI using BLASTP <br /> Dmrnahel (hel),encoding protein,HEL, extensive                    program (ALTSCHUL et al. 1990). D. MILLERdiscovered similarity known ATPdependent RNA helicases.                       homology HEL yeast Saccharomyces cmmisiae <br /> demonstrate he1 gene new haploenhancer                     protein, HERl, homology search GenBank.                                                                          pointed possible homologue worm, <br /> ofPEV Drosophila,                        Caenwhabditis ekgans. partial sequence worm homo- <br /> HEL protein locates nucleus associates                   logue available expressed sequence tag Gen- <br /> chromosomes, intimating arole aputative RNA heli-                    Bank accession  21479&apos;7. Thecorresponding C. elegans <br /> case chromatin structure. results suggest                  cDNA, cm5d4 (WATERSTON            et al. 1992), provided byR. HEL protein promotes open chromatin           struc-              WATERSTON,       sequencing cm5d4 cDNA com- <br />                                                                          pleted study. Comparisons homologues ture favors transcription preventing spread                  using GCG Sequence Analysis Software Package (GENET- heterochromatin. Furthermore,                      ICS         GROUP 1991) Drosophila complete HEL protein present transcriptionally compe-                    C. ekgans sequences GenBank accession <br /> tent nuclei  interphase), disappears chro-                 nos. LO6018 U08102, respectively. <br /> mosomes condense enter mitosis. remarkable                          heyf5&apos; mutation cloned homozygous <br />                                                                          mutant pupae (identified animals failed differenti- <br /> disappearance HEL protein mitotic domains                      ate abdominal structures) PCR. he1 clones se- embryos ofparticular light finding              quenced described according Sequenase WARBRICK     GLOVER(1994) thata he1 cDNA                          manual provided United States Biochemical. <br /> (called WM6) suppress premature mitosis caused                          P element-mediatedtransformation: he1 gene  loss weel mikl function fission yeast,                  cluded 4.5-kb EagI/BgllI genomic DNA fragment.                                                                          fragment cloned fromcosmid 8 theNotI/BamHI <br /> Schizosaccharomyces pornbe. weel mikl encode inhibi-                 sites pCaSpeR2 (THUMMEL         et al. 1988) P-element trans- <br /> torykinasesof     CDC2 protein kinase,                      formation Drosophila. transformation construct  <br /> gether associated cyclin, play central role                 tains -700 bp genomic DNA 5&apos; putative translation <br /> controlling entry mitosis.                                          initiation site HEL protein, -1300 bp 3&apos;                                                                          putative translation termination site. Flies transformed <br />                                                                          injection y w embryos containing endogenously ex- <br />                 MATERIALSAND          METHODS                            pressed A2-3 transposase (ROBERTSON           et al. 1988) recov- <br />   Genetic stocks. Deficiency stocks containing Df(2L)cP              ered described (SPRADLING           1986) trans- <br /> Df(2L)cP4(SZIDONYA REUTER 1988), theZn(l)wm4mu-                  formant lines contained second chromosome insertion tant stock provided Mid-America Drosophila                   pCaSpeR2 construct suitable studying effects PEV <br /> Stock Center Bowling Green, Ohio. Thebalancer chromo-                 concert Zn(l)wm&apos; chromosome. pCaSpeR2 deficiency stocks replaced CyO, w f &quot;             vector expresses low levels white miniwhite gene  <br /> balancer chromosome (KASSIS et al. 1992), expresses p-             struct (w+&quot;&apos;C transformation marker, produces galactosidase wingless pattern. EMS- X ray-induced            orange background eye transformant <br /> lethal stocks region 25E (SZIDONYA    REUTER 1988)           line. Visual observation ofPEVwas possible trans- <br /> provided byJ. SZABAD.   following alleles complemen-            formant line containing pCaSpeR2, orange <br /> tation groups jf25-jf30 SZIDONYA       REUTER lethal          background spectrophotometric quantitation PEV <br /> mutant collection confirmed genetic complementa-                 difficult, second transformation construct using <br /> tion lie Df(2L)cP4: b24, sz15&apos;, sz9, sz18&apos;, a7, sz63, b2&apos;,     Y.E.S. vector (PATTONet al. 1992), expresses b7, a26 b23. alleles tested heterozygotes              yellow body pigment gene (y&apos;) transformation marker. Df(2L)cP4  rescue lethality transformed h e 1        4.5-kb Eu$/BglII genomic DNA fragment mutantchromosomes contain additional lethal               removed pCaSpeR2 EugI/XbaI fragmentand <br /> mutations positioned outsideof Df(2L)cC4sz15&apos; tested            cloned NotI/XbuI sites Y.E.S. vector obtain Df(2L)~l&quot;~ modification PEVwith Zn(l)wm4chro-            transformants background eye color trans- <br /> mosome. het&quot;l5&apos; lesion he1 recombined away               formation vector. transformant line containing E.S. second sz15 lesion tkv gene   recombi-            construct used measure PEV effects he1 white gene <br /> nant         recombinants balanced Cy0,wf&apos;&quot;                expression l)wff14.     Similar visual results obtained <br /> balancer chromosome.                                                     using pCaSpeR2 transformation vector, expressing low <br />    Molecular cloning localization   hel gene: cDNA          levels w + &quot; &apos;  transformation markerand yielding orange <br /> clone encoding nuclear lumin (GRUENBAUM           et al. 1988)       eyes  shown), obtained Y.E.S. vector, <br /> provided P. FISHERand used screen dp bw genomic               expresses y+ body pigmentation gene marker <br /> DNA library thecosmid vector pWE16 (R. PADGETI W.                  (shown Figure 3). <br /> GELBART,   unpublished results). cosmid clones, cosmids 4               Geneticcrossesandquantitation PEV Transformants 8, isolated used screen a4 8-hr embryonic             containing E.S. construct   used quan- <br /> cDNA library (BROWNand KAFATOS 1988). l.<span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;kb cDNA                       titate effects he1 PEV. assay gene dosage effect <br /> encoding he1 gene recovered.                                     Df(2L)cP4on PEV used spectrophotometric quantita- <br />    he1 cDNA used probe genomic Southern blots                 tion eye pigments female offspring reciprocal <br /> containing DNA wild-type flies flies heterozy-             crosses y l)wm4; Df(2L)cP4/Cy0        y w; Plhel, y + / / + . <br /> gous deficiencies Df(2L)cPf, Df(2L)h2,       Df(2L)cP4to         test effect &quot; PEV assayed eye pigment <br /> locate he1 gene dosage comparison.                                levels female offspring Zn(l)wrn4;1(2)szI5/CyO males <br />    Sequencingandsequence         analysis: Genomic cDNA              crossed y w; Plhel, y+ / + females. Mean OD4H5n,,,    readings <br />                                                   Drosophila RNA Helicase PEV                                                953 <br />  (hePI5&apos;) (Df(2L)cp4)separate eye pigment ex-          ondary antibodies (Vector Laboratories ;     1250 dilu- <br /> tractions genotype determined. extraction,           tions). <br /> heads 12 flies aged 5-7 days homogenized 200                   Conventionalcolor fluorescent images photo- <br /> pl chloroform 200 pl 0.1% ammonium hydroxide                   graphed Zeiss Axiophot fluorescent microscope.  <br /> centrifuged 2 min, 100 pl aqueous phase             sulting photographic slides scanned p r e <br /> used forspectrophotometry withoutdilution            (ASHBURNER        cessed using Adobe Photoshop software assembled 1989). extracts normalized relative dose           printing Deneba Canvas software. Confocal imaging value trials subtracting         BioRad MRGGOO Laser Scanning   <br /> basal reading  y 7u heads).                                       hanced Adobe Photoshop assembled using Deneba <br />    Generation mutant hel germlineclones: Homozygous                 Canvas. <br /> mutant hel clones generated germline using                Western blot analyses, total protein 0-4hr em- <br /> autosomal FLP recombinasedominant femalesteriletech-                   bryos recovered homogenization PBS containing 1 <br /> nique (CHOU andPERRIMON           1996). technique employs        mM PMSF 0.1% SDS. Proteins separated 10% yeast FLP recombinase, induces recombination              polyacrylamide SDS gels electroblotted BioTrans nitro- <br /> homozygous FRT sites, ovo&quot;&apos; dominant female sterile            cellulose membranes immunodetection.    Membranes mutation selection clones. original he1 mu-         blocked 5% nonfat dry milk PBS 0.1 % Tween-20 <br /> tant chromosome, szZ5, carries mutation close1           15 min room temperature incubation anti- <br /> linked veins (tkv) gene, used independenthe$&apos;   J       HEL antibody  preimmune serum) ata 1:lOO dilution recombinants, remove tkfl&quot; mutation,             15 min room temperature      4&quot; overnight. Mem- FRT&apos;OAchromosome. generate        clones, females geno-      branes reblocked antibody detection accom- <br /> type &apos; P{neoFRT4OAI/ CyOwere crossed males geno-             plished described tissueanalyses (seeabove type y w F W 2 / Y; P{zu+&apos;&apos;&apos;~~&apos; <br />                              O V ~ &apos; P{neo <br />                                      }     FRT&apos;&quot;}/CyO. Progeny         secondaryantibodyincubations      1 hr. Developed heat shocked 1 hr at37&quot;     larval instar,   blots scanned using Adobe Photoshop females genotype y w FLP&apos;*/+; he1&quot;&quot; P{neo FRT&apos;OAJ/              labeled printing Deneba Canvas. <br /> P{7ufrr&apos;(&apos;ovoI&quot;] PJneo FRT&quot;&apos;;&apos;) tested presence germline clones. <br />    Antibody generation, tissue immunodetection Western                                      RESULTS <br /> blot analysis: produce HEL protein bacteria, 1.4kb <br /> restriction fragment hel cDNA extending SpeI               Molecular cloning he1 gene: Genomic site (bp 429-434 Figure 2A) Not1 site pNB40             cDNA clones encoding he1 isolated vector subcloned Bluescript SK&apos; (Stratagene).            efforts characterize RNA transcripts 25E-F  <br /> fragment excised SmuI Not1 subcloned             gion second chromosomeof Drosophila. chro- <br /> intothe <span id='am-14' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-15' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span>glutathione</span>-Stransferase (GST) expression vector <br /> pGEX-5X-1 (Pharmacia). Fusion protein purifiedfrom <br />                                                                        mosomal walk nuckarlamin locus 25F1 <br /> Escherichia coli lysates affinity chromatography glutathi-       (GRUENBAUM      et al. 1988; Figure 1) initiated Sepharose-4B elution 10 mM <span id='am-90' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-91' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span>glutathione</span>.               using nuclear lamin cDNA screen dp cn bw genomic <br /> 76kD fusion protein, containing GST, 18 novel amino acids              DNA library cosmid vector pWE16 (R. PADCETT <br />   plasmid-encoded, he1 5&apos; UTR)              W. GELBART,       unpublishedresults). cosmid <br /> entire HEL polypeptide used immunize rabbit.Injec- <br />                                                                        clones, cosmids 4 8, -35 kb length overlap- <br /> tions 100 n1g purified fusion proteinin Freund&apos;s <br /> incomplete adjuvant biweekly intervals.                                      - <br />                                                                        ping 17 kb (data shown) isolated. 1.&amp; <br />    enzymatic immunodetection HEL, larvalsalivary                kb cDNAwas recovered 4 8-hr Drosophila <br /> glands dissected gently squashed 50% <span id='am-95' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-96' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acetic acid</span>,          embryonic cDNA library (BROWNand KAFATOS 1988), <br /> 3.7% <span id='am-85' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-86' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> period -2-5min.               Embryos      using cosmids 4 8 probes. Sequencing fixed 50% acetic acid, 3.7% forma1dehyde:heptane <br />                                                                        cDNA clone genomic DNA encompassing <br />  (1:l) for5 min devitellinized shaking embryos hep- <br /> tane methanol ( 1 2 ) ; ovaries fixed 5 min in50%        cDNA indicated mature mRNA transcript acetic acid, 3.7% formaldehyde. antibody incubations           created splicing exons (Fig- <br /> washes phosphate-buffered saline (PBS)               ure 2, B). cDNA hybridized mRNAs <br /> 0.1% Tween-20. 1:50 1:lOO anti-HEL antibody dilution <br /> gave optimal results. Salivary glands incubated anti- <br />                                                                           - <br />                                                                        1.6 and1.8 kb Northern blot analysis poly- <br />                                                                        adenylated RNA various stages embryonic  <br /> HEL  preimmune serum control experiments)  1 hr room temperature; embryos ovaries incu-              velopment (data shown), indicating therecov- <br /> bated 15-30 min (embryos) 1 hr (ovaries) room               ered cDNA length nearly length. <br /> temperature thenovernight 4&quot;. Biotinylated secondary              apparent RNA helicasewith unusualDEAD <br /> antibody (anti-rabbit IgG, Vector Laboratories, ) di-          box (DECD): long open reading frame       luted 1200 incubatedwith tissues 2 hr room           beginning second exon transcript  <br /> temperature. Secondary antibodies detected using avidin/biotinylated-HRP reagent (Vectastain Elite,              coded 1.6-kb cDNA. Encoded open <br /> Vector Laboratories, ) DAB NiCl color devel-          reading frame protein 425 amino acids (Figure <br /> opment.                                                                2A) extensive similarity RNA helicases. pro- <br />    fluorescent images, embryos fixed described             tein, HEL, similar murine translation initiation enzymatic immunodetection  necessary              factor eIF4A otherDEAD box-containing RNA heli- <br /> omit acetic acid microtubule stainingwith anti-tubulin; <br /> HEL detection unaffected omission acetic acid), <br />                                                                        cases regions conserved members labeled anti-HEL anti-tubulin monoclonal             family (Figure 2C; SCHMID   LINDER 1992).    <br /> tibody (Boehringer Mannheim, 1:50 dilution) detected                HEL containsa cysteine place themore fluorescein-conjugated Texas Red-conjugated sec-              common alanine inthe position DEAD box <br />  954                                                     D. F. Eherl et ul. <br />  <br />                                                                                                       nla m <br />                                                            E(var) <br />                         jf23         jf24        j f 2 5j f 2 6     cl j f 2 7j f 2 8j f 2 9j f 3 0 <br />                          a7          SZZO         b24      sz15&apos;             a9      sz63   b7    a26 <br />                          sz36        h25                                     sz18&apos;   b2&apos;          b23 <br />                                      a1                                      a7 <br />                                      b79 <br />  <br /> Df(2L)clh7     +       D2-4                                                                                     F1-2 <br />  <br />  <br />  <br />  <br /> Df(2L)clh4            D6-EI                                                                                            E 5 - F l &quot; <br />   FIGURE1.-Genetic organization 25D-F region contains hel gene. Complementation groups printed hold mutant alleles group listed italic. Complementation mapping jf series (SZIDONYAand REUTER <br /> 1988) deficiencies   encompass hel gene confirmed. Placement nuclear lamin (nlum) based molecular analyses (data shown). relative order complementation groups Df(2L)cp4is undetermined. <br />  <br /> (DECD thanDEAD). otherhelicase-                 Df(2L)cP4 Df(ZL)cP&apos;, act haplo-enhancers like protein, theplum pox virus Cl protein, cysteine            variegation l)wm4in theabsence suppres- position (LAIN et al. 1989), contains           sors (data notshown, Figure 1). Southern blotanaly- histidine place second aspartic acid (DECH),            ses revealed he1 gene removed fall D M family helicases (BORK                 deficiencies (data shown) tested he1 KOONIN1993). membersof DEAD                   influence PEV. <br /> box family, eIF4A, p68, An3 VASA, viral                    Flies transformed genomic DNA encoding <br /> C1 protein shown possess RNA helicase                  he1 P element-mediated transformation.Transforma- <br /> activity (HIRLING   et al. 1989; ROZEN et al. 1990; LAIN et al.     tion constructs contained -700 bp genomic DNA 5&apos; <br /> 1990; GURURAJAN       et al. 1994; LIANGet al. 1994).         putative translation initiation site HEL <br /> members assumed unwind RNA basis                   protein, -1300 bp 3&apos; putative translation <br /> sequence comparison proven helicases.                        termination site. Crosseswere    HEL homologues otherorganisms: amino                      transformed copies hel rescue reduced white <br /> acid sequence HEL protein used search                 gene expression Zn(1)w&quot;&quot;&apos; caused Df(2L)d4en- <br /> databases homologues organisms. possi-               hancement PEV. Transformed he1 capable  <br /> ble functional homologue HEL (based sequence                  storing white expression unmodified Zn wVn4   levels <br /> similarity) therat p47, 82% amino                    presenceof Df(2L)cP4 (Figure 3; cross Table <br /> acid identity; NAIR et al. 1992). submittingthe               1). eye pigment level single dose flies sig- <br /> HEL sequence Genbank, second putative homo-                    nificantly lower flieswith he1 gene doses <br /> logue thebudding yeast, S. cerevisiae, surfaced                (paired t-test; P &lt; 0.001). enhancement variega- <br /> (HERl, 63% identity; D. MILLER,personal communica-                  tion rescued transformed copy he1 level <br /> tion). addition, partial sequence cDNA recov-               significantly different endogenous <br /> ered nematode worm, C. elegans (Cernahel;                  copies.  he1 encodes gene   <br /> WATERSTON     et al. 1992), suggested existence           leted, causes enhancement PEV. identified homologue HEL. completed sequenc-                   mutation hel  section) behaves ing C. elegans cDNA  MATERIALS METH-                 similar way deficiency. addition, transformed <br /> ODS), revealing high degree similarity             he1 mediates increase eye pigment level encoded protein HEL (79% identity). compari-                  endogenous copies ( P &lt; 0.05), suggesting son protein sequences shown Figure                he1 behaves triplo-suppressor PEV (Table 1). <br /> 2D. recentBLAST search homologuesof HEL                      hel essentialgene: addition haplo- <br /> identified additional apparent homologues humans                 enhancer variegation function  pigs (BAT1, 83% identity; PEELMAN          et al. 1995).        homozygous lethal complementation groups <br />    hel modifies PEV Previously, SZIDONYA         REUTER         uncovered Df(2L)cP4 (SZIDONYA         REUTER <br /> (1988) described loci region 25F-26B sec-             1988; Figure 1). foundthat P-element <br /> ond chromosome behave haplo-enhancers                   transformation constructs containing genomic copy triplo-suppressor Zn(1)wm4variegation inthe                  he1 gene fully rescue lethality caused single <br /> presence strong suppressor ofPEV, Su-var(2 &apos;.                homozygous recessive mutation, sz15&apos;, indicating deficiencies delete region 25E,              sz15&apos; mutation resides hel gene he1 <br />                                             Drosophila RNA Helicase PEV                                        955 <br />  essential gene. determined        nation dissected ovaries germline clones. Egg <br />         mutation acts haplo-enhancer wm4         chambers develop late stages showing signs <br /> variegation            heterozygotes (crossTable <br />                                                                                                B             degeneration resorption. Similar results 1; P &lt; 0.05), indicating loss function phenotype         obtained independent he1&quot;&quot;&quot;&apos; recombinants. <br /> similar seen Df(2L)cP4. Furthermore,         results indicate wild-type hel activity  <br /> enhancement variegation caused copy          quired oogenesis consistent &apos; mutation reversed transformed copy he1       maternal effect PEV. <br />  (Table 1) cross B, saw triplo-suppressor        HEL protein nuclear associates   chro- <br /> effect he1 PEV (Table 1;P &lt; 0.01).  heterozy-      mosomes: HEL protein expressed bacteria gosity hel  heF&apos;5&apos;/+ animals) results enhance-       used generate rabbit polyclonal antibody ment ofPEV, whilehomozygosityof            he1  hefzI5&apos;/   HEL. polyclonal anti-HEL antiserum used Df(2L)cP4 animals) lethal. Lethality caused loss      Western blots total protein 0- 4hr-old em- hel occurs embryonic, larval pupal stages       bryos (Figure 4). strongly reacting, major band of development. variability likely related    expectedmolecular weight endogenousHEL strong maternalexpression hel  section).       protein (-47 kD) seen immune serum, possible heFzJ5&apos;mutation com-       preimmune serum. weakly reacting, minor <br /> plete null mutation hep&quot;&quot;&quot; enhance PEV         bands seen immune preimmune <br /> lesser extent does Df(2L)cP4 (Table  recom-         sera. <br /> mend caution comparing results              antibody used determine thecellular <br /> crosses atdifferent times).  alternatively    location endogenous HEL protein embryos, larval explained enhancer PEV lies            salivary glands adult ovaries (Figure 5). Df(2L)clh4 small percentage mutant embryos        casesHEL protein nucleus cells signs ofmorphological defects incomplete         express  case nuclear staining detected <br /> involution theheadand gut.           homozygotes sur-      preimmune serum substituted antibody- <br /> viving pupal stage develop small heads ap-         containing serum. relatively low cytoplasmicback- <br /> pendages, thoraces truncated    posteri-   ground color reaction occurred preimmune <br /> orly, abdominal structures fail differ-    immune sera. <br /> entiate     characteristics used identify          HEL protein clearly association chro- I5&apos;/ bet&quot;&apos;&quot; homozygous pupae forPCR cloning        mosomes salivary glands, ap- <br /> mutant he1 gene. DNA sequencing, glutamic              pears located chromosomes aswell <br /> acid lysine substitution identified position 245     (Figure 5A). appears associated regions mutant HEL protein.                                    chromosomes; evidence HEL <br />    Effects PEV he1 include maternalcompo-             binds particular loci. Strong nuclear staining nent: influence he1 l)wm4variegation          apparent cells outer, largely epidermal, <br /> cludes maternal effect. Male offspring l)wm4;         surface embryos (Figure 5B), notable areas Df(2L)c2h4/+females  dose hel) express little     lower staining intensity, corresponding mitotic <br /> white, notthey receive Df(2L)clh4chromo-       domains described FOE (1989) FOE et al. (data shown), female  progeny express         (1993) seen early-mid embryonic stages white female progeny l)w&quot;&apos; females       mitotic cell cycles asynchronous Dro- doses hel). Different overall levels eye pig-      sophila embryo (Figure 6). HELalso expressed ments female offspring reciprocal             ovaries heavy staining seen large, polyploid <br /> crosses shown Table 1 (cross  illustrate mater-      nurse cell nuclei smaller follicle cell nuclei nal effect associated hel. HEL mRNA supplied       follicle cells migrate nurse cells embryos maternally presentthroughout em-               egg chamber (Figure 5C). <br /> bryogenesis (data notshown). Theseresults suggest           Nuclear expression HEL protein disappears dur- <br /> maternal HEL protein, addition zygoticallyex-           ingmitosis: observe regionally reduced <br /> pressed HEL, promotes transcriptional competence              HEL staining themitotic domains asynchronously development.                                           dividing embryos (described FOE1989 FOEet al. <br />    he1 required oogenesis: Homozygous mu-           1993), examined moredetailtherelationship <br /> tant he1 clones generated female germlines         HEL mitotic cellcycle. visualize <br /> ing autosomal FLP recombinase-dominant female             areas mitosis developing embryos clearly, sterile technique (CHOUand PERRIMON         1996)     used confocal microscopy embryos double-stained <br /> sz15&apos; mutation hel  MATERIALS METHODS).            HEL antibody antibody ,&amp;tu- <br /> Females het&quot;&apos;5&apos;/hel&quot;&apos;r germlines lay eggs,      bulin detect disintegration cytoplasmic microtu- theeggs laid small, collapsed      bule networks subsequent formationof nuclear mi- fused dorsal appendages. obvious            totic spindles  FOE1989 FOE et al. 1993). defect late oogenesis detected exami-         individual composite images generated  956                                                                                                                                                                    D. F. Eberl et al. <br />  <br />           1   gCtt~caaatgg~tttg+tgtgtgcacaa=t~t=gataa=ttaa=gatatc*c=cgg=t=60 <br />                                                                                                                                                                                   1441   tacttctccac~tcgc~tccaccgctatcacgac~att~~ttattattttgttaatg~ca <br />                                                                                                                                                                                                                                                    1500 <br />       61   agCtCgCatgagttCttaaaC9t(lCttcggt tGCTCTGCGCTGTT~TTCCTCCG 120 <br />                                                                                                                                                                                   1501   E g c g t t c g c t t s g t o t t g t c c c o t c c g t D t s g ~ t                               1560 <br />    121     CATCACGTTAAAGATTTTAAAATAGTTAATTTCACTTTAAAAGCTGTATTTTAARG&apos;IGAT                                                                                                   180 <br />                                                                                                                                                                                   1561                                                              1620 <br />                                                                                                                                                                                          ggttaaaogcgtlcaaccaaaogtcsagcgct+ctgtgagoataactcgt~t~gaaaa~=t <br />    181     AAACGTGCACAAAAGttgagtttgacactgaaaarcacaaaaat===cg                                                                                                               240 <br />                                                                                                                                                                                   1621   tagttaatgactatcggaca=cattttctaaatggttaatcgtggaa~ttggcaaccgtt                                                      1680 <br />    241     cttgtaatgggtatttccttttatttcgaaat~cat~aatg~ggtg~ggtg~aaaaaca <br />                                                                     300 <br />                                                                                                                                                                                   1681                                                             1740 <br />                                                                                                                                                                                          gc~aaagcata~acaagcagtaac~g~tgaatgaatcaat~aattgaaatcg~aa~~gatttaa <br />    301     aaatggcggagggaacgccggcaccggttccgcaaaaaaaagcggcattttctttaacaa                                                                                                    360 <br />                                                                                                                                                                                   1741                                                             1800 <br />                                                                                                                                                                                          caataccaacgctagcaaaaaccaatacctaaataaaacta=aag~aaa~ag~aaagatc <br />    361     acacatsraacttatctagtcgcacccattttatttcccagCATTTTAGTTGCGAATTTAT                                                                                                   420 <br />                                                                                                                                                                                   1801   ccgccgactggccgatgagacgagtrgaatcaatccagagcg~~tgcgatgcg~taggc~1860 <br />    421     AGCCGATAACTAGTTAAAGGTAAGAAAGCCGCAGAAATGGCCGACAATGACGATCTTTTG                                                                                                    480 <br />                                               M D N D D L L                                                                                                                     1861   cgctcatctgtaatccgcacac+tctcagagcgatcatcg~t~ct=ggg~gcagtt=~ga1920 <br />    481     G C T C G G W \ T G G G G C G C C G G ~ C ~ C G G T C G ~ ~ G G ~ C C C C A5A4G0                                                                   1921   gactagttgatgagatgtggtggtagcgtggccgfatgtagaatattctatgcatattgt                                                      1980 <br />            D       Y       E       D       E       E       Q       T       E       T       T             V       E       N       Q       E             P       K <br />                                                                                                                                                                                   1981                                                          2040 <br />                                                                                                                                                                                          tgtatgttccgaggogagcct~gt~taagttcgattt~t=tgta~=~=t==taatt~gt <br />    541     AAGGATGTCAAGGCCACCTATGTGTCCATTCACAGTTCCGGCTTCCGCGATTTCCTCCTG 600 <br />            K D V K G T Y V S H S S G F R D F L L                                                                                                                                2041   CStttIaOltgCgcC.Cattat.tttSCC tCc.gCCCATffiA~CTACGTCGACGA                                                       2100 <br />                                                                                                                                                                                                                                                        P     M    E    V       Y       V       D       D <br />    601     AAACCGGAGATCCTGCGGGCCATCGTAGACTGCGGCTTCGAGCATCCCTCGGAGGgtgag                                                                                                    660 <br />            K       P       E             L       R                   V       D       C       G       F       E       H       P       S       E       V                      2101   TGAGGCCAAGCTGACGCTGCACGGACTGCAGCAGCACTACGTCAATCTGAAGGAGAACGA                                                      2160 <br />                                                                                                                                                                                           E K L T L H G L Q Q H Y V N L K E N E <br />    661     tacttgca=tccacctggagcgacgaaaaggta~~gt~gcta~cgatg=tct~cgcttgca <br />                                                                      120 <br />                                                                                                                                                                                   2161   GAAGAACAAGAAACTGTTCGAACTGCTCGACGTGCTCGAGTTCAATCAGgtaggtcattc                                                      2220 <br />    721     gTTCAGCACGAGTGCATTCCGCAGGCCGTACTGGGCATGGACATCCTCTGTCAGGCCAAG                                                                                                    780            K N K K L F E L L D V L E F N Q <br />               Q H E C P Q V L G M D L C Q K <br />                                                                                                                                                                                   2221                                                                  2280 <br />                                                                                                                                                                                          ggtggtcttaacttatatttaccactracrtgatta~ttgatt~t~~=tgt gGTGGTCATCTTT <br />    781     TCCGGCATGGGTAAGACCGCCGTCTTCGTTCTGGCCACGCTGCAGCAGCTGGAGCCGTCG                                                                                                    840                                                                                             V       V             F <br />            S G M G K T V F V L T L Q Q L E P S <br />                                                                                                                                                                                   2281   GTGAAGTCTGTGCAACGTTGCGTGGCTCTGTCGCAGCTCCTGACGSAGCAGAACTTCCCC                                                      2340 <br />    841     GACAACAACACCTGCCACGTCCTGGTCATGTGCCACACCCGCGAGCTGGCCTTCCAGATC 900                                                                                                              V K S V Q R C V L S Q L L T E Q N F P <br />            D N N T C H V L V M C H T R E L F Q                                                                                                                                                                                   2341   GCCATCGGCATCCATCGTGTGACCCAGGAGSAGCGTCTGAATCGCTACCAGCAGTTC 2400 <br />    901     AGCAAGGAGTATGAGCGATTCTCCAAGTACATGCCCACAGTCAAGgtgagtctacgtttt                                                                                                    960                 G       H    R    G    M    T     Q    E    E    R     L     N   R    Y       Q       Q       F <br />            S K E Y E R F S K Y M P T V K <br />                                                                                                                                                                                   2401   AAGGACTTCCAGAAGCGCATTCTGGTGGCCACCAATCTCTTTGGCCGCGGCATGGACATC                                                      2460 <br />    961     tccattgtatataattcaaaaaaaagtttggtttggtt~g~=t~t~tagttactatatatt~~ <br />                                                                         1020                                                                                                             K D F Q K R L V T N L F G R G M D   1021     ttcgatattttttctatttagGTGGCTGTCTTCTTTGGCGGAATffiCTATTCAAAAffiAC 1080                                                                                                    2461   GAGCGTGTGAACATCGTGTTWULCTACGACATGCWGAGGATTCCGACACCTACTTGCAT                                                       2520 <br />                                                                    V             V       F       F       G       G       M                   Q       K       D                     E    R    V    N       V    F    N     Y    D    M    P    E     D     S   D    T       Y       L       H <br />   1081     GAGGAGACCCTCAAGAGCGGCACCCCGCATATTGTGGTGWXACCCCTGGCCGAATTCTC                                                                                                     1140   2521   CGCGTGGCCCGTGCCGGTCGCTTCGGCACCAAGCGACTGGCGATCACATTCGTTTCGGAC                                                      2580 <br />            E E T L K S G T P H V V G T P G R L                                                                                                                                       R V R G R F G T K G L T F V S D <br />   1141     GCCCTCATTCGCAACAAGAAACTTAATCTGAAGCTTTTGAAGCACTTTGTGCTCGACGAG                                                                                                    1200   2581   GAGAACGACGCCAAGATACTTAACGAAGTACAGGATCGTTTCGATGTGAACATCAGTGAG                                                      2640 <br />            L R N K K L N L K L L K H F V L D E                                                                                                                                       E N D K L N E V Q D R F D V N S E <br />   1201                                                              1260 <br />            TGCGACAAGATGCTGGAGCAGCTGGgtaagftgacraara~t=~c=~~atg~cgga=gtt                                                                                                           2641                                                             2700 <br />                                                                                                                                                                                          CTGCCCGAGGAARTCGATCTCTCTACATACAgtaagttg=~=tg~tg=gt ~~==g~ <br />            C       D       K       M       L       E       Q       L       D                                                                                                             L    P    E    E       D    L    S     T    Y      1261     ttactaataaccaaaccattcttttagATATGCGTCGTGAcGTTCAAGAGATTTTCCGTA                                                                                                    1320   2701   C c o g c g t g c t c t t g t t t t o + T t t t t = ~ ~ T G G f f C G C T G ~ T C ~ G2G7T6 ~ <br />                                                                                                                                                                                                                                                                                                           0 <br />                                         M R R D V Q E F R S                                                                                                                                                                                        E G R &apos; <br />  <br />   1321     GCACCCCGCACGGTGATGATGTTCTCTGCCACATTGAGCAAGGACATTCGTC                                                                                                            1380   2761   ATGTTAATCGGAACAAGACTATGCACTTCGAATAGATACAXACCTAAAACCTTTTTTAAA                                                      2820 <br />                T       P       H       G       K       Q       V       M       M       F       S             T       L       S       K       D             R       P <br />                                                                                                                                                                                   2821   ACTCTACTTTATAACTAAATAAAAACTATTCAGAWTGTGTTGACAAACAAAAACGGAA                                                        2880 <br />   1381     CCGTTTGC~GTTCATGGATgtaaatatcaa=gtgtt=ata~= aatg~t cg <br />                                                              1440 <br />              V C K K F M Q D                                                                                                                                                      2881   ttataagtaatgtaattattttgggaatggattcta                              2916 <br />  <br />  <br />    FIGURE 2. he1 gene encodes protein similarity RNA helicases.   DNA sequence deduced protein sequence Drosophila hel gene (GenBank accession number L06018). DNA sequence cDNA genomic clones <br /> (representing exons) uppercase, sequence genomic clones (representing introns lowercase deduced <br /> protein sequence shown DNA sequence. (B) Schematic exon/intron structure he1 gene. Horizontal <br /> lines represent exons, splicing introns illustrated v-shaped lines connecting exons, protein-encoding <br /> regions indicated boxes. Positions motifs commonly RNA helicases shown shaded hatched boxes exons. Alternative nomenclatures ATPase B domains domains VI (WALKER         et al. 1982; HODGMAN 1988) shown. (C) Comparison conserved regions DEAD family (modeled PAUSEand SONENBERG             1992; PAUSEet al. <br /> 1993) proposed DECD subfamily putative RNA helicases basedon HEL apparent homologues. correspondence conserved amino acids withinthe ATPase B domains/domains VI shown. (D) Similarity Drosophila <br /> HEL protein worm (C. ekgans), yeast (S. cerevisiae) rat homologues. Divergent regions shaded. Drosophila <br /> protein 79% identical C. ekgans protein (Cernahel; WATERSTON1992and study;GenBankaccession number <br /> U08102), 63% identical yeast protein (HER1; D. MILLER,     personal communication; GenBank accession  274132; ORF <br /> YDL084W) 82% identical rat protein (p47; NAIRet al. 1992; GenBank accession  M75168). <br />  <br /> anterior (head) portion embryo early mitotic                                                                                                                               seen careful inspection. Various states ofmicrotu- <br /> cycle 14 (Figure 6),  comparing p-tu-                                                                                                                                bule breakdown spindle formation apparent <br /> bulin patterns HEL, HEL disappears                                                                                                                               (Figure 6). Triple staining HEL, @tubulin DAPI mitotic domains. cells withina mitotic                                                                                                                              (data notshown) allowedus interpret themitotic <br /> domain traversemitosis somewhat synchronously,                                                                                                                                    stages structures experimental  <br /> closer look domain reveals different cells                                                                                                                             ditions. preparation mitosis, depolymerization domain various stages mitosis.                                                                                                                                cytoplasmic microtubules identified loss ofp- <br /> Cells near center domain                                                                                                                               tubulin immunoreactivity cytoplasm. occurs enter mitosis mitosis spreads                                                                                                                               detectable formation mitotic spindles periphery adomain (FOE et al. 1993;                                                                                                                                   nuclei evident periphery mitotic <br /> Figure 6), slightly scattered pattern                                                                                                                               domains Figure 6, cells general                                                        Drosophila RNA Helicase PEV                                                                                                 9.57 <br />  <br />  <br />                                                                                                  ... <br />                                                                                                                                                                                . <br />  <br />                                                                                       UADND,           D L L D Y E D ~ E .O . .     . . . . . . . . . . . . . . . . TET               18 <br />                                                                                                  E e E&apos;O L L D Y ,E E E&apos;O E E 0   . . . . , . . .-   ~-~            ~.   . . ~. .    ~ <br />                                                                                                                                                                                       16        ,       . <br />                                                                                         ~ ~ ~ ~ ~ E ~ E D L L ~ Y s                          D   N     E   ~      E   ~ &apos; ~ 39 ~                $   ~   ~     E <br />                                                                                                                       . .V . . . . <br />                                                                                       ~ E N D V D N ~ L L D Y E D D E                     . . . . . .         . .   . . E T 21 <br />  <br />                                                                                       T V ~ N Q E P . K K D V K G T Y V S H S S C P R D F L L K P E L R V 57 <br />                                                                                       D K Q P E V G G ~ ~ D ~ &apos; K ~ K G T Y ~ H S S C P R D F L L K P E L R56A C <br />                                                                                       ECDNNNNAAA.GDKKCSYVCIHSTCPKDFLLKPELSRA1I                                       78 <br />                                                                                       C D G T E P K K D V K C S Y V S H S S C P ~ D F L L K P e L L R ~61 <br />  <br />                                                                                       D   C C P E H P S E V O H E C P O V L C U D L C Q K S C U G K T V F V L , 91 <br />                                                                                       D    ~ C P E H P ~ ~ ~ ~ H , E ~ ~ P ~ ~ L ~ U D ~96 ~ ~ Q K ~ ~ <br />                                                                                       D   C C P E H P S E V O O H T P O S H C T D V L C O K S C L C K T V F V L 118 <br />                                                                                       D   C C P E H P S E V O H E C P O ~ L C ~ D V L C O K S C ~ C K T V F101    V L <br />  <br />  <br />  <br />  <br />                                                                                       P   T V K V V P P G C U ~ K D B B &apos; = L K . ~. ,b &apos; T P H V V G T P C R F L 175 <br />                                                                                       P   C ~ K V V F , F ~ ~ U , K K D ~ E R L N . D . ~ P H ~173~ ~ T P ~ R ~ L                                                                                       P   D V K T V F Y C C T P S K D E L L ~ N K D T P H V V T P C R L K 194 <br />                                                                                       P   NVKVAVPFGGLSIKKDEEVLK1(..NCPHIVVGTPCRiLAll8 <br />  <br />                                                                                       L   R N K K L N L K L L K H F V L D E C D K U L E O L D U R R D V O E F R S T 215 <br />                                                                                       L    R S ~ K L K L D K Y K Y P V L D E C D K ~ ~ D D ~ ~ ~ .D. V209           O ~ . . <br />                                                                                       L   V R e r Y ~ D L S H V . N F V &apos; l D e C D K V L ~ e ~ ~ U R R D V O ~ F 214R T <br />                                                                                       L   R N K S L W L ~ H K H F L D ~ C D K ~ L E Q L D U R R D V O B F R218    U T <br />  <br />                                                                                       P H &apos; G K O V U U P S T L S K D , ~ R P V C K K P U O D P U E V Y V D D E K L T 255 <br />                                                                                       .. <br />  C                                                                                         . K O V U U F S T L P K E L R T V C ~ K R F U O , D , P U E V Y V D D E K L T 2b6 <br />                                                                                       P R d K O V U N P S T L S Q E R P C R R F L O N P L E F V D D E K L T 27b <br />                                                                                       P H ~ K ~ ~ W ~ P S T L S ~ E R P ~ ~ R ~ F ~ Q D P H E                          258F ~ D D E T K L T <br />  DECD SUBFAMILY <br />         AKSGZKT    RaR      GG   TFGR   DEfD   SAT   GRGXD   HRVXRAGR                 L H G L Q ~ H Y V N L K E N ~ K N K K L P E L L D ~ L ~ P N Q195~ ~ F ~ K ~ ~ ~ <br />  NW                                                                     COOH          L H G L Q Q H Y Y K L K ~ ~ K N R R L L N L L D L E F N ~ V V F V K A286 <br />                                                                                                                                                                   V K <br />                                                                                       L H G L Q P Y Y K L B ~ R E K N R K L Q L L D D L ~ F N O V I F V K ~ T T ~ ~ <br />         AXXXXKT   &#226;&#8218;&#172; T R m   GG   TFGR   D&#226;&#8218;&#172;W    SAT   ARGXD   HR1GW.W <br />                                                                                       LHGLOOYYVKLKDWEKNRKLFDLLDVLEPNOVVIFVKSVO                                                        298 <br />  DEAD FAMILY <br />  <br />  <br />  <br />  <br />                                                                                       R L V T N L P C R G U D E R V N V F N Y D U P E D S D T Y L H R V R l 7 5 <br />                                                                                       RILVATDLPGRCUDIERVNXVFNYDUP,E,DSDSYLHRVliRAl66 <br />                                                                                       R C V S T D V F C R C O E R &apos; ~ N L N Y D L T N E D Q ~ L H R V G R194                                                                                        R L V T R L Q C R C W D V R V N F N Y D ~ P E D S D T Y L H R V R 378 <br />                                                                                                                                                            ..    - <br />                                                                                       G R P G T K C L T P V S D E N D K L N ~ V O D R F D , V N S E L P ~ E 414 <br />                                                                                                                                                                        . <br />                                                                                       ~ R F G T K C L T F V S ~ ~ E ~ N D ~ K T L N S ~ Q D R P D 405           ~ T E L P ~ ~ . <br />                                                                                       GRPGTKGLAISPVSSKEDEEVLAKIOERFD~KIAEPPEEG O b <br />                                                                                       GRPCTKCLASHLCOTRTUPRSS                                                                          (00 <br />  <br />  <br />                                                                                       IDLSTYIEGR                                                                                      424 <br />                                                                                       IDVSTYI-E,C,R&apos;GN                                                                                b17 <br />                                                                                       IDPSTYLNN                                                                                       bb3 <br />  <br />  <br />                                                                FIGURE2.- Continupd <br />  enter mitosis, relatively early mitotic                       HEL protein appears present stages. Robust HEL protein expression seen nuclei                        interphase embryos. HEL bepresent G2 cells enter mitosis. Moving slightlymore interi-                      evident nuclei cells prepare mitosis <br /> orly domain (Figure6), HEL detectable                        anembryonic stage includes G2 phase (preced- spindles begin form, althoughat reduced intensity.                       ing M), HEL protein persists nondivid- spindle continues form, condensing chromo-                           ing cells later amnioserosa somes movetoward metaphaseplate, HEL                                   arrest G2  FOEet al. 1993; data shown). HEL follow movement fades. time                         germ cells mitotically chromosomes gatheron metaphase plate, HEL                              quiescent migrate embryo nuclei <br /> immunoreactivity extinguished. HEL remains absent                           throughouttheouter       surface embryos (datanot anaphase, 0-tubulin detection reveals ex-                          shown) cells stopped dividing <br /> panded spindles characteristic gapping atthe                              settled G1. HEL protein accumulates soon <br /> equator, telophase, evidenced lingering 0-                        mitosis  cytoplasmic microtubule networks <br /> tubulin staining whatappears theslowly disinte-                       begin reassemble) epidermis embryos com- <br /> grating spindle overlap microtubules parting                           pleting cycle 14, acquisition G1 phase  cells (described FOE d al. 1993 seen                            FOEet dl. 1993), gaps HEL immunoreactivity recent data SICRIST d al. 1995). HEL immunoreactiv-                         epidermis embryos mitotic <br /> ity resumes interphase cytoplasmic microtubules                          domains arise, suggesting HEL pres- <br /> reform (data shown). note HEL protein                              ent DNA synthesis S phase. Polytene salivary <br /> disappears mitotic domains described FOE                             gland cells, alternate G S cycles, <br /> (1989) far able identify.                            contain HEL protein, yetknow <br />                                                    D. F. Eberl   rt nl. <br />  <br />  <br />  <br />                                                                                                       FIGURE3.-hrl mod- <br />                                                                                                    ifier PEV.Transforma- <br />                                                                                                    tion hrl rescuesthe <br />                                                                                                    Df(212)c/&quot;&quot;-induced  <br />                                                                                                    hancement rohitr varie- <br />                                                                                                    gation. Shown eves                                                                                                    selected flies identi- <br />                                                                                                    cal X chromosome genc- <br />                                                                                                    types (y h(1)7u&quot;&quot;&apos;/y 7lJ)                                                                                                    different hrl gene dosages. <br />                                                                                                    7umf induces variega- <br />                                                                                                    tion 7uhitr gene                                                                                                    presence                                                                                                    endogenous copies hrl <br />                                                                                                    (+/CjO) variega- <br />                                                                                                    tion enhanced                                                                                                    copy he1 deleted (+/ <br />                                                                                                    Df(2L)df). Addition                                                                                                    transformedcopy       Iwl <br />                                                                                                    (Plhrl y+]/Df(2L)~f&apos;~  <br />                                                                                                    stores thevariegation                                                                                                    original level.                                                                                                    copies hrl (P{hrl ?  <br />                                                                                                    CyO) moderately suppress <br />                                                                                                    variegation. Table <br />                                                                                                    1 quantitation                                                                                                    eye pigment.. ge- <br />                                                                                                    notypes. <br />                                              <br />  <br />  <br />  theG S phase           ATP binding, ATPase B/II motif (DEAD) HEL associates chromosomes cells.             involved ATP hydrolysis couplingof ATPase <br /> Taken  result.. suggest HEL protein           helicase activities. Motif 111 (SAT) impli- expressed G1, S G2 phases              cated RNA unwinding, region VI (HRIGRXXR) <br /> cell cycle Drosophila.                                       required forRNA binding andATP hydrolysis. mu- <br />                                                                 tagenesis study VASA protein Drosophila                                                                 revealed <span id='am-61' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-62' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycine</span> <span id='am-37' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-38' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span>glutamic acid</span> change                      DISCUSSION <br />                                                                 ARGXD domain abrogates theability DEAD box- <br />    identified new gene Drosophila, hel,            containing helicase unwind RNA (LIANG        et nl. 1994). encodes protein similar members          DORNet nl. (199%) reported cloning family ATPdependent RNA helicases.                   translation initiation factoreIF4A Drosophila. fly <br /> shown he1 enhancer  PEV anessen-         eIF4A protein 73% identical mouse translation <br /> tial gene. maternal contribution hel influ-         factor. HEL -35% identical mouse fly <br /> ences PEV, examination mutant he1 germline               eIF4A proteins. <br /> clones indicates he1 required oogenesis.            HEL likely function ATPase: DNA encod- <br /> antibody generated bacterially expressed HEL            ing potential homologue HEL rat (p47) <br /> protein recognizes protein expected size West-          obtained screening    expression library <br /> ern blots, reveals HEL expressed nuclei          antibodies ATPase (NAIRet al. 1992). associates withsalivary gland chromosomes.              ATP binding hydrolysis domains  LINDER           et al. <br /> HEL protein disappearsin mitotic domains embryos,            1989; PAUSEand SONENBERG 1992)        putative <br /> indicating HEL regulated cell cycle          HEL homologues (including HEL itself) conserved <br /> opening intriguing question whetherHEL             members DEAD box family ATPdepen- <br /> volved cellcycle control development.                 dent RNA helicases. notableexception                  HEL sharesseveral motifs known RNA heli-                DECD variation itself, HEL homo- <br /> cases: regions conserved HEL             logues. suspect thatthis substitution pre- members DEAD boxcontaining RNA heli-                 clude ATPase activity  . <br /> case family  Figure 2C) implicated                Does HEL unwind RNA? variegation particular molecular functions (refer    LINDERet nl.        thought act change chromatin struc- <br /> 1989; PAUSEand SONENRERC        1992; PAUSEet nl. 1993).        ture, consider possibility HEL act ATPase domain (AXXXXGKT) required                RNA, DNA, similarity RNA <br />                                                Drosophila RNA Helicase PEV                                                  959 <br />  <br />                                                            TABLE 1 <br />                                      Effect he1 gene dosage variegationof Zn(l)Wm4 <br />  <br />                                               Gene dose <br />                                                             Df(2L)cp4 Df(2L)cp4 <br /> Cross          Genotype&quot;            Endogenous <br />                                         mothers&apos;fathers&quot; <br />                                                    Transformed                                                   Normalizedd <br />        y l)wm4.- +                  2                0           0.191 2 0.0855 <br />                                                                                0.0157            5 1.000 <br />                                                                                                     0.0078            5   0 <br />             y w &apos; cy0 <br />          y wm4. +                   1                0           0.095 t 0.0055       0.0424 t 0.0030    0.408   5   0.0349*** <br />             y w &apos; Df(2L)cp&apos; <br />          y l)wm4.P{hel y&apos;}             1                1          0.167   5   0.0129   0.0723 t 0.0043    0.856   5   0.0627 <br />             y w &apos; Df(2L)cp&apos; <br />          y l)wm4.-P{hel y+}           2                 1          0.229 -+ 0.0159      0.0997 t 0.0075    1.270   5   0.1116* <br />             Y W   &apos; cy0 <br />  B       y l)wm4.- +                  2                0           0.206 2 0.0111                          1.000 5 0 <br />              yw      cy0 <br />          y l)wm4.- +                   1               0           0.169 t 0.0197                          0.805 t 0.0729* <br />              y w &apos; 1(2)sz15 <br />          y l)wm4.-P{hel y&apos;}            1                1          0.201 t 0.0094                          0.976 2 0.0125 <br />              y w &apos; 1(2)sz15 <br />          y wm4.-P{hel y&apos;}            2                1          0.256 t 0.0224                          1.246 t 0.0432** <br />              yw   &apos;    cy0 <br />   Female offspring&quot; reciprocal crossesb. y n ( 1 ) ~ &quot;Df(2L)cp4/CyOwith <br />                                                                 ~;                y w; Plhel, y+}/+ cross  fromy l)wm4; <br /> 1(2)sz15/CyO males y w; P{hel, y+)/+ females cross B. extracts normalizedd relative value endogenous gene doses trials  trials cross B) subtracting basal reading  y w heads); normalized values reciprocal crosses cross pooled statistics. Statistical significance normalized values relative dose values indicated (* P &lt; 0.05, ** P &lt; 0.01, *** P &lt; 0.001). <br />  <br />  <br /> helicases, including onlyconservative changes               tissues intimateadirect     role HEL chromatin <br /> HRIGRXXR block, indicative RNA helicase                structure. HEL act  level ofthe chromosomes, <br /> activity required RNA binding (PAUSE              unwinding removing mRNA et al. 1993). second motif, SAT, critical           site transcription, consequently keeping chromo- <br /> RNA unwinding (PAUSEand SONENBERG 1992),                   accessible transcription. Anotherpossi- <br /> conserved HEL. addition, highly conserved <span id='am-49' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>gly</span>-             bility HELmay discourage heterochromatin <br /> cine RGXD domain, shown essential                 spreading associating chromosomes com- <br /> RNA unwinding activity VASA protein (LUNG et                plex include RNA component. Alterna- <br /> al. 1994), conserved HEL.                                    tively, HEL affect PEV indirectly regulating    DEAD box domain implicated ATP                  expression specific genes, chromosomal pro- <br /> hydrolysis coupling ATPaseand helicase activi-           teins. <br /> ties (seeLINDER et al. 1989; PAUSEand SONENBERG                       Clues implicate RNA regulation tran- <br /> 1992). suspect alanine cysteine change              scriptional competence uncovered. DEAD box HEL  DECD) allow                 MALELESS (MLE) protein Drosophila regulates dos- <br /> ATPase helicase activity plum pox virus              age compensation associating X chromo- <br /> C1 protein contains cysteine position           males, resulting increased transcription shown ATP-dependent RNA helicase                  single male X chromosome (KURODA et al. 1991). <br /> activity (LAIN et al. 1990).              MLE likely homologue human RNA helicase able demonstrate helicase activity initial             (LEE HURWITZ 1993),       demonstrated RNA heli- <br /> tempt bacterially expressed glutathione Strans-               case, removed chromosomes ferase-HEL fusion protein incubated uitro             RNase (BASHAW      BAKER1996). recently identi- double-stranded RNA template(provided            N.           fied Drosophila male-specific RNAs, roXl roX2, asso- <br /> METHOT N. SONENBERG).        HEL functional            ciate Xchromosome chromatin (AMREIN AXEL <br /> RNA helicase, require additional factor(s)              1997; MELLER et al. 1997). HELaffects PEV specific template (s) .                                         acting chromatin-associated RNAs  <br />    HEL influence PEW Strong association                  Mammalian dosage compensation occurs X-chromo- HEL protein salivary gland chromosomes                 inactivation females. Anuclear RNA tran- <br /> nuclear localization HEL embryonic ovarian               scribed XIST/Xist locus implicated <br />  960                                                     D. F. Eberl d nl. <br />  <br />  <br />  <br />  <br />             200 Kd   - <br />              97      - <br />              68      -                                               r <br />              43      - <br />  <br />               29      - <br />   FIGURE4.-Western blot analysis HEL. Western blot <br /> analysis protein from0- 4-hr wild-type embryos. preimmune serum (left) anti-HEL immune <br /> serum (right) recognize minor proteins, anti- <br /> HEL immune serumidentifies prominent proteinof molec- <br /> ular weight expected endogenous HEL protein(-47 <br /> kD). <br />  Xinactivation (BROCKDORFF     et nl. 1992; BROWN    et dl. <br /> 1992). XISTRNA complexes inactive X <br /> chromosome interphase (CLEMSON          et nl. 1996). expect RNA helicase  RNAde- <br /> pendent ATPase) prevent complex forming X chromosome remains transcrip- <br /> tionally active. experiments required  <br />                                                                     P <br /> termine HEL affectsPEV mechanisms. <br />    HEL cell cycle: begun character- <br /> ize possible role HEL cell cycle regulation                       --HI  nuclc;w protein. HEL protein localiza- <br />                                                                       FI( KI.: <br />                                                                    tion nuclei salivary glands, embryos ovaries seen <br /> looking forcell cycledefects he1 mutant embryos                                                                    using anti-HEL polyclonal antibody andenzymatic detec- overexpression HEL wild-type embryos,                 tion.   Gently squashed polytene chromosomes sur- approaches led inconclusive results.              rounding cytoplasm nuclear envelopes association heF&apos;5&apos;/Df(21,)c1&apos;4 heterozygotes            HEL protein chromosomes. (B) HEL &apos; homozygotes suffer appear               protein expressed nuclei embryo duringearly <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span>gas</span>- <br />                                                                    trulation. lateral view, intensity <br /> problems morphogenetic movements em- <br />                                                                    HEL protein expression begins vary different <br /> bryogenesis. significant FOE et               nuclei stage. (C) closeup view intact ovary <br /> nl. (1993) report thetiming mitosis important           showing nuclear expression HEL protein large polyploid cells undergo complex morphogenetic                   nurse cell nuclei (*) smaller follicle cell nuclei (+) ments.  he1 mutant defects occur             migrate egg chambers nurse cells. <br /> low frequencies; strongereffects likely masked strong maternal contribution HEL mRNA.                   overexpression experiments, heat shock treat- <br /> Overexpression HEL cycle 14 embryogene-               ment wild-typeembryos cause chromosome  <br /> sis, using transformed helcDNA driven heat shock            densation, cell cycle delay mitotic synchronization <br /> promoter, prompted obvious passage                 (MALDONADO-CODINA      et al. 1993; L. J. LORENZ, <br />                                                                                                                   personal <br /> mitosis. clear mitosis delayed          observations), microtubule breakdown (L. J. <br />                                               Drosophila RNA Helicase PEV                                               961 <br />  <br />  <br />  <br />  <br /> HEL <br />  <br />  <br />  <br />  <br /> TUBULIN <br />  <br />  <br />  <br />  <br />    FIGURE6.-HEL protein disappears mitosis. Confocal images mitotic domains (domains 1 5 FOE 1989) head region embryo, anterior cephalic furrow (CF) early cell cycle 14 shown.   composite series optical sections outer lateral cell layer embryo double stained HEL protein (red) and0-tubulin <br /> (green) clearly indicates HEL disappears mitosis. Bright HEL nuclear signals (red) surrounded cytoplasmic <br /> 0-tubulin staining (green) interphase cells, imparting &quot;pimento green olive&quot; appearance cells. (Yellow halos nuclei produced fusion dimension dimensional cytoplasmic 0-tubulin signals, slightly nuclei, nuclear HEL signals.) Mitotic domains 1 5 (numbered) appear obvious dark regions  HEL <br /> protein fades) containing P-tubulin positive mitotic spindles. Black white photos HEL (B) 0-tubulin (C). <br /> (D) Closeup view domain 1 region boxed showing mitotic details. Cytoplasmic 0-tubulin staining disappears HEL affected (*) subsequently, HEL begins disappear (diamond). (E) Magnified view region boxed D. Dynamics HEL 0-tubulin staining progress following steps: (1) cytoplasmic microtubules break (*), (2) HEL begins fade (diamond), (3) spindle microtubules begin appear HEL gone (+), (4) HEL undetectable anaphase <br />  (filled arrowhead) spindle prominent equatorial gapping ( 5 ) telophase (open arrowhead) spindle <br /> remnants seen daughter nuclei. <br />  <br />  <br /> LORENZ,personal observations),   complicate           GLOVER1994), effect wild- studies.  HEL regulatory             type yeast. uncover role HEL role cell cycle able           cell cycle regulation Drosophila embryos overex- <br /> identify, shown overexpression            pression stable form HEL  heZcDNA fission yeast,S. pornbe, suppress          graded mitosis. approach useful charac- <br /> premature mitosis induced loss known                 terizing roles B CYCLINS (SIGRIST   et <br /> inhibitory kinases CDC2, Wee1and Mikl, (WARBRICK                aZ. 1995). <br />  962                                                               D. F. Eberl et al. <br />  <br />    role HEL cell cycle regulation,                         inactive X-specific transcript containing conserved ORF                                                                                    located nucleus. Cell 71: 515-526. help consolidate possibility link                        BROWN,C.J., B. D. HENDRICH,          J. L. RUPERT,    R. G. LAFRENlhKE, Y. XlNC; <br /> tween PEV cell cycle control. growing evi-                            et al., 199&apos;2 human XIST gene: analysis 17 kb inactive <br /> dence thatthese processes intimately associ-                            X-specificRNA contains conserved repeats highly <br />                                                                                    localized nucleus. Cell 71: 527-5q2. <br /> ated. clonedhaplo-suppressor PEV, Su-var(3}6,                         BROWN,     N. H., F. C . KAFATOS, 1988 Functional cDNA libraries <br /> codes protein <br />                     phosphatase involved mitosis                              Drosophila embryos. J. Mol. Biol. 2 0 3 423-437. <br /> (AXTON et al. 1990; BAKSA et al. 1993). Mutations                          CHOL&apos;,T.-B., N. PERRIMON,           1996 autosomal FLP-DFS tech- <br />                                                                                    nique generating germlinemosaics Drosophi mrlanogaskr. <br /> Su-var(3}6can prompt overly condensed chromosomes                                  Genetics 144: 1673-16&apos;79. abnormal chromosome segregation. Furthermore,                             CIEMSON, C. M., J.  McNErr., H. F. WII.IAW J. B. AwREN(:L.:, <br />  transcriptional activator cell cycle regulator,                            1996 XIST RNA paints inactive X chromosomeat  <br />                                                                                    terphase: evidencefor novel RNA involved nuclear/chromo- <br /> E2F, influences PEV (SEUMet al. 1996). Finally,                           structure. J. Cell Biol. 132: 259-275. <br /> Drosophila gene encodingPCNA recessive suppres-                          DEI.SAI., G., G. MANFIOLETTIand C. SCHNEIDER,                   I989 &lt;:TAB- <br /> sor PEV (HENDERSON      et al. 1994). PCNA functions                            DNA precipitation method: common miniscale preparation                                                                                    template DNA phagemids, phages plasmids suitable DNA replication repair, exists com-                              sequencing. Biotechniques 7: 514-519. <br /> plexes withvarious cyclins CDKS (cdc-like kinases),                       DORN,R., V. m u s s , G. &amp;.UTER andH. SAUMWF.BER, 199 including CYCLINS B CDC2 (ZHANG et al.                                   enhancer position-effect variegation Drosophila, t.:(vc~r)3- <br />                                                                                    93D, codes chromatin protein containing consenzed  <br /> 1993). HEL clearly affectsPEV, understanding                               main common transcriptional regulators. Proc. Natl. <br /> functional basis mitotic disappearance HEL                              Acad. Sci. USA 90: 11376- l 1380. illuminate relationship                        DORN,R., H. MORAWIETZ,             G. REUTER H. SACIMWERLK,                1993h <br />                                                                                    Identification essential Drosophila gene thatis homologous processes.                                                                     translation initiation factor eIF-4A yeast mouse. <br />                                                                                    Mol. Gen. Genet. 237: 233-240. <br />    Mutant fly stocks provided J. SZABAD        Mid- <br />                                                                               EISSENBERG, J. C., 1989 Position effect variegation Drosophila: <br /> America Drosophila Stock Center Bowling Green, Ohio thank                    genetics chromatin assembly. BioEssays 11: 14-17. <br /> P. FISHER providing nuclear lamin cDNA clone, R. PrZDGETI             EISSENBERG,    J. C., T. C.JAMES, D. M. FOSTER-HARTNE~,                T. HARTNI.. T, cosmid library, N. BROWN   cDNA library, K. GWTA              V. NGANrt al., 1990 Mutation aheterochromatin-specific building he1 expression construct. gratefully acknowledge               chromosomal protein associated suppression position- <br /> D. MII ER communicating results prior topublication R.                   effect variegation Drosophila melanogastrr.Proc. Natl. Acad. Sci. <br /> WATERSTON colleagues sending     partially sequenced cDNA              USA 87: 9923-9927. <br /> clone encoding C. eleguns homologue. thank J. RICHTER, N.              EIssmnmc;, J. C . , G. D. MORRIS,        G. R E U T K R T. HAKTNEIT,        1992 <br /> SONENBERG,    W. BENDER,   C.-t. WU, M. HECK,C. BERGand T. ETWCH                   heterochromatin-associated protein HP-1 essential pro- <br />                                                                                    tein Drosophila dosage-dependent effects position- helpful suggestions encouragement, D. HARRISON advice <br />                                                                                    effect variegation. Genetics 131: 345-352. <br /> o n confocal imaging, T. ORR-WEAVER          com-        FARKAS    G., J. GAUSL, M. GAI ONI,G. REUTEK, H. GWRKOVI(:Srt l . , <br /> ments manuscript. work supported Howard                     1994 Trithorale-likegeneencodes thc Drosophila GAGA fac- <br /> Hughes Medical Institute National Cancer Institute                   tor. Nature 371: 806-808. <br /> Canada funds CanadianCancer Society. :].L.                FOE, V. E., 1989 Mitotic domains reveal early commitment cells <br /> supported postdoctoralfellowship theNational Institutesof                Drosophila embryos. Development 107: 1-22, <br /> Health andV.S. supported NaturalSciences Engineering             FOE,V. E., G. M. ODELI.        B.  EDGAR, 1993Mitosis morpho- <br /> Research Council (Canada) University Undergraduate Student                      genesis Drosophila embryo: point counterpoint, pp. <br /> search Award. P.L. Research Scientist National Cancer                  149-300 TheDer1elopment Drosophila mrlanogaslrr,Vol. 1 , ed- <br />                                                                                    ited M.BATE  MARTINEL h S . Chid Spring Harhor <br /> Institute Canada N.P. Howard Hughes Medical Institute <br />                                                                                    hhoratory Press, Cold Spring Harbor, N Y . <br /> Investigaror.                                                                 GAKLINO, V., . PEREIR        P. IAAURENTI,   Y. GILWA,R. W&apos;. LEws et ., <br />                                                                                    1992 Cell lineage-specific expression modulo, dosedepen- <br />                       LITERATURECITED                                              dent modifier variegation Drosophila. EMBO J. 11: 4471- <br />                                                                                    4479. <br />             S . F., W. Gtsl-l,W. MII.L.ER, E. W. MEERS D.J. IJPMAN, <br /> AI,TSC:HLJI.,                                                                 GENFTICS    COMIT~TEK GROUP,         1991 Prugram Mrmual, thP GCG Pack- <br />     1990 Basic local alignment search tool. J. Mol. Biol. 215: 403-                agp, Version 7, April 1991. 575 Science Dr., Madison, WI 53711. <br />     410.                                                                      GEKASIMOVA,        T. , D.  GDL~IA,  D. V. G E R L S M0.    ~ ~SIMONOVA <br />                                                                                                                                                      ,         AMREIN,  H., R. x n , 1997 Genes expressed neurons adult              V. G. CORCES,1995 ADrosophila protein impartsdirec- <br />     male Drosophila. Neuron 88: 459-469.                                           tionality chromatin insulator enhancer position- <br /> ASHBURNER,       M., 1989 Quantitative measurement red-eye pig-                 effect variegation. Cell 82: 587-597. <br />     ments, p. 312 Drosophila: Laboratoly Manual. Cold Spring             GKLTN~AAI:M,Y.,      Y . LANDESMAK, B. DREES,     J. W. BARE,H. SAUMWEBEK          rl <br />     Harhor Laboratory Press, Cold Spring Harbor, N Y .                             d , , 1988 Drosophila nuclear lamin precursor Dm,, translated <br /> AxTc)N,J.M., V. D O M B R h , P. T. W. COHEN D. M. ( O\&apos;F.R, 1990          developmentally regulated mRNA species ap- <br />     protein phosphatase 1 isoenzymes Drosophila                   parently encoded single gene.   Cell Bid. 1 0 6 585-596. <br />     essential mitosis. Cell 63: 33-46.                                    GVRIIRAJAN,      R., . MATHEWS,    F.J. LONGOand D. WFXLS, 1994 An3 <br /> BALSA,K., H. MowwIET%,V. D O M B R ~M.           , AXTON, H. TALIREKT rt         mRNA encodes RNA helicasc colocalires nuclroli <br />     ., 1993 Mutations protein phosphatase 1 gene 878                   n Xenopus oocytes stage-spccific mannel-. Proc. Natl. Acad. <br />     differentially affect suppression position-effect variegation           Sci. USA 91: 2056-2060. <br />     mitosis Drosophila melanogastcn Genetics 135: 117-125.             HENCHOX,      S . , F. Ruswrls, D. PAUI.1 P. SPIERER, 19% BASHAW,G. J., B. S . BAKER,1996 Dosage compensation                        dose putative ubiquitin-specific protease affects position- <br />     chromatin structure Drosophila. Curr. Opin. Genet.Devel. 6:                 effect variegation Drosuphilo mrlanuguslrr. Mol. Cell. Bid. 16: <br />     496-501.                                                                       5717-5725. <br /> BORK,P., E. V. KOONIN, 1993 expanding family helicases              HENl)EKSON, D. S., S . S . BANGA,      T.  ~ R l ~ ~ l , l Aand <br />                                                                                                                                             T l l J. B. 8OYl),1994 <br />     &apos;DEAD/H&apos; superfamily. Nucleic Acids Res. 21: 751 -                  Mutagen sensitivity suppression position-effect variegation <br />     752.                                                                           result mutations mus209, Drosophila gene encoding <br /> BROC:KI)L)ORFF,    N.,  ASHWORTH,  G. F. UY,  V. M. M&lt;:Cz2se,D. P.           P(:NA. EMB0.J. 13: 1450-1459. <br />     RIS rt / . , 1992 product mouse Xist gene 15 kh         HENIKOFF,      S . , 1981 Position-effect variegation andchromosome <br />                                                         Drosophila RNA Helicase PEV                                                             963 <br />     structure heat shock puff Drosophila. Chromosoma 83:              REUTER,G., M. GIARRE,     J. FARAH,    J. GAUSZ  SPIERER      et al., 1990 <br />     381-393.                                                                      Dependence positioneffect variegation Drosophila dose <br /> HENIKOFF,    S., 1990 Position-effect variegation 60 years. Trends          gene encoding unusual zinc-finger protein. Nature 344: <br />     Genet. 6: 422-426.                                                            219-223. <br /> HIKING,H., M. SCHEFFNER,         T. RESTLE andH. STAHL,1989 RNA               REUTER, G., P. SPIERER,1992 Position effect variegation     helicase activity associated human p68 protein. Nature              chromatin proteins. BioEssays 1 4 605-612. <br />     339 562-564.                                                              ROBERTSON,     H. M., C. R. PRESTON,R.W. PHILLIS,D. M. JOHNSON- <br /> HODGMAN,T.C., 1988 new superfamily replicative proteins.                     SCHLITZ,   W. K. BENZ et al. 1988 stable genomic source P <br />     Nature 333: 22-23.                                                           element transposase Drosophila melanogaster. Genetics 118: <br /> Ussrs,J. , E. NOLL, E. P. VANSICKLE,   W. F. ODENWALD    N. PERRI-     <br /> </body></html>