<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                            Genetic analysis requirements SOS induction                            nalidixic acid Escherichia coli <br />  <br />  <br />                            Abstract <br />                                 Nalidixic acid, prototype antibacterial quinolone, induces SOS response mechanism                                 requires RecBCD nuclease/helicase. key step inferred induction pathway                                 conversion drug-induced gyrase cleavage complex DNA break processed                                 RecBC. tried clarify nature step searching additional gene products                                 specifically necessary SOS induction following nalidixic acid treatment. transposon library NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 approximately 19,000 insertion mutants yielded 18 mutants substantially reduced SOS <br />                                 induction following nalidixic acid UV treatment, hypersensitive                                 nalidixic acid. 18 mutants turned insertions recB recC. expected, recA <br />                                 insertion mutants uncovered uninducible nalidixic acid UV treatment. <br />                                 Insertions 11 genes cause partial defects SOS induction                                 pathways, providing possible leads understanding detailed mechanisms SOS induction. <br />                                 Overall, results suggest nalidixic acid-induced DNA breaks generated RecBC <br />                                 itself, redundant activities, essential protein uncovered                                 transposon mutagenesis. <br />  <br />  <br />                            Keywords <br />                                 DNA gyrase; RecBCD; DNA damage; quinolones; transposon mutagenesis <br />  <br />  <br />                            1. Introduction <br />                                               Type II DNA topoisomerases essential enzymes involved numerous cellular processes, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               including DNA replication, DNA segregation transcription. enzymes catalyze <br />                                               changes DNA topology creating transient double strand break segment duplex <br />                                               DNA passing second duplex segment break. clinically important <br />                                               antibacterial antitumor agents target type II topoisomerases altering equilibrium                                               reaction cycle stabilize normally transient intermediate called cleavage complex <br />                                               (Drlica Zhao, 1997; Burden Osheroff, 1998). include antibacterial <br />                                               quinolones (e.g. nalidixic acid) target DNA gyrase  topoisomerase IV certain <br />                                               bacteria), antitumor drugs mitoxantrone, adriamycin etoposide, target <br />                                               mammalian type II topoisomerase. case, drug-stabilized cleavage complex consists <br />                                               topoisomerase linked phosphotyrosine bonds 5&apos; ends staggered double- <br />  <br />  <br />                            * Corresponding author: Kenneth N. Kreuzer, Email: kenneth.kreuzer@duke.edu, Phone: 919 684 6466, Fax: 919 684 2790. <br />                            1Present address: bioM rieux, 100 Rodolphe St., Durham, NC 27712 <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Newmark et al.                                                                                               Page 2 <br />  <br />  <br />                                             strand break. Results numerous systems demonstrate cytotoxicity                                             inhibitors dependent formation cleavage complexes inhibition                                             topoisomerase catalytic activity  reviews, Chen Liu, 1994; Drlica Zhao, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             1997). <br />  <br />                                             research devoted understanding stabilized cleavage complexes <br />                                             lead cell death, detailed mechanism(s) remains unclear. Formation cleavage <br />                                             complexes necessary sufficient cytotoxicity. 5&apos; ends broken DNA <br />                                             cleavage complex safely linked protein, cleavage complexes readily <br />                                             reversible drug removal vivo vitro (Hsiang Liu, 1989; Drlica <br />                                             Zhao, 1997). energy poison dinitrophenol blocks cytotoxicity mammalian type II <br />                                             topoisomerase inhibitors does prevent cleavage complex formation (Kupfer et al., <br />                                             1987). Likewise, dinitrophenol  chloramphenicol) protects Escherichia coli                                             cytotoxic action nalidixic acid (Cook et al., 1966). Evidently, cellular processing event <br />                                             occur convert subset cleavage complexes cytotoxic lesions.                                             event, exact nature cytotoxic lesion? <br />  <br />                                             number observations strongly suggest cytotoxic lesion type double- <br />                                             strand break  review, Chen Liu, 1994; Drlica Zhao, 1997). example, <br />                                             mutational inactivation functionally conserved recombination proteins phage, bacterial <br />                                             eukaryotic systems leads drug hypersensitivity. gene products required repair <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             topoisomerase-mediated DNA damage similar identical required repair <br />                                             endonuclease-generated double-strand breaks. Furthermore, evidence overt <br />                                             chromosomal breaks nalidixic acid treatment, location breaks                                             possibility covalently attached protein determined (Chen et al., 1996). <br />  <br />                                             variety models principle explain relationship drug-stabilized <br />                                             cleavage complexes cytotoxicity.  nuclease SbcCD directly recognize <br />                                             covalent protein-DNA complex cleave DNA nearby (Connelly et al, 2003). Second, <br />                                             replication complex associated helicase able extract lagging strand template <br />                                             cleavage complex collision (Howard et al., 1994).  DNA breaks                                             result &#226;&#8364;&#339;collateral damage&#226;&#8364;&#65533; recombination nucleases act replication fork <br />                                             blockage cleavage complex, phage T4 (Hong Kreuzer 2003).                                             recently quinolone-stabilized gyrase cleavage complexes block E. coli <br />                                             replication forks plasmid pBR322 vivo (Pohlhaus Kreuzer, 2005). study,                                             blocked forks broken, consistent collateral damage model. <br />  <br />                                             approach mechanism cytotoxicity, taken advantage SOS reporter <br />                                             screened E. coli mutants specifically deficient SOS induction nalidixic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             acid treatment. primary target nalidixic acid E. coli DNA gyrase  Maxwell                                             Critchlow, 1998), drug inducers SOS regulon studied                                              SOS regulon consists 30 different genes, involved                                             damage repair bypass reactions (Friedberg et al., 1995). LexA protein normally <br />                                             represses SOS genes, cleaved trigger SOS result DNA damage. Cleavage                                             LexA depends activated form RecA protein, bound single-stranded DNA. <br />  <br />                                             DNA gyrase cleavage complexes stabilized nalidixic acid necessary sufficient <br />                                             induction SOS response. conflicting evidence DNA replication <br />                                             required induction SOS nalidixic acid (Gudas Pardee, 1976; Sassanfar                                             Roberts, 1990). Aside RecA, known protein required SOS induction <br />                                             nalidixic acid multifunctional RecBCD enzyme  review, Myers Stahl, <br />                                             1994). RecBCD generally requires DNA end gain entry DNA, involvement  <br />  <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                             Page 3 <br />  <br />  <br />                                             SOS induction suggests free DNA break generated nalidixic acid- <br />                                             stabilized cleavage complex. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             results provide strong parallels mechanism cytotoxicity                                             mechanism SOS induction nalidixic acid, particularly recA, recB recC mutants <br />                                             hypersensitive drug (McDaniel et al., 1978). highly likely                                             mechanism creates DNA breaks induce SOS pathway mechanism leads                                             cytotoxicity. report, identify analyze E. coli mutants deficiencies                                             SOS response nalidixic acid. <br />  <br />                               2. Materials methods <br />                               2.1. Materials media <br />                                             Kanamycin nalidixic acid purchased Sigma, 5-bromo-4-chloro-3-indolyl- <br />                                             &#206;&#178;-D-galactopyranoside (X-gal) purchased Gold Biotechnology  L-broth (LB) <br />                                             contained Bacto-Tryptone (10 g/L), yeast extract (5 g/L) sodium chloride (10 g/L)                                             LB plates composition plus Bacto-agar (14 g/L). agar contained Bacto- <br />                                             Tryptone (13 g/L), sodium chloride (8 g/L), sodium citrate dihydrate (2 g/L), glucose (5 g/L) <br />                                             Bacto-agar (6.5 g/L). <br />  <br />                               2.2. Transposon mutagenesis primary nalidixic acid screening <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             transposon library created E. coli strain JH39 [F- sfiA11, thr-1, leu-6, hisG4, argE3, <br />                                             ilv(Ts), galK2, srl(?) rpsL31, lac U169, dinD1::MudI(Apr lac); Heitman Model, 1991; <br />                                             O&#226;&#8364;&#8482;Reilly Kreuzer, 2004] using EZ::TN&#226;&#8222;&#162; &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit                                             Epicentre (Madison). Electrocompetent cells pulsed 1.8 kV 0.1-cm cuvettes, <br />                                             followed outgrowth TB hour 37&#194;&#176;C aeration. cells plated                                             appropriate dilutions LB plates containing kanamycin (60 &#206;&#188;g/ml). Following overnight <br />                                             incubation 37&#194;&#176;C, colonies lifted using 85-mm circular nitrocellulose filters <br />                                             (Schleicher Schuell) placed, colony  LB plates containing nalidixic acid <br />                                             (50 &#206;&#188;g/ml) X-gal (60 &#206;&#188;g/ml). lifted colonies incubated overnight 37&#194;&#176;C                                             plates checked color development approximately 4 12 hours. Colonies                                             turned darker blue wild-type marked &#226;&#8364;&#339;dark blue&#226;&#8364;&#65533; white <br />                                             lighter wild-type marked &#226;&#8364;&#339;white&#226;&#8364;&#65533; &#226;&#8364;&#339;intermediate&#226;&#8364;&#65533;, respectively. <br />  <br />                               2.3. DNA techniques sequencing <br />                                             Genomic DNA purified using MasterPure&#226;&#8222;&#162; DNA Purification Kit Epicentre <br />                                             (Madison) essentially described manufacturer  phenol-chloroform extraction <br />                                             followed ethanol precipitation added RNase treatment). final purified <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA pellet resuspended 30 &#206;&#188;l T5E1 buffer (5 mM Tris-HCl, pH 7.8; 1 mM EDTA). <br />                                             DNA samples sequenced Duke University Cancer Center DNA Analysis <br />                                             Facility using modification automated sequencing protocol (60 cycles, 60oC annealing <br />                                             temperature). sequencing primer (10 pmol/&#206;&#188;l) KAN-2 FP-1 (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             ACCTACAACAAAGCTCTCATCAACC&#226;&#8364;&#8220;3&#226;&#8364;&#178;), forward primer EZ::TN&#226;&#8222;&#162; <br />                                             &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit. <br />  <br />                                             PCR-amplification recB gene performed primers recB-FP (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             CGGCAGGATGTTCATCTCCCGCC&#226;&#8364;&#8220;3&#226;&#8364;&#178;) recB-RP (5&#226;&#8364;&#178;&#226;&#8364;&#8220; <br />                                             TCTTGTGCTGCACGAGTCAGCC&#226;&#8364;&#8220;3&#226;&#8364;&#178;). <br />  <br />                               2.4. Phage P1 transductions <br />                                             Transductions performed method Silhavy et al. (1984), selecting                                             kanamycin-resistance gene transposon. <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 4 <br />  <br />  <br />                               3. Results <br />                               3.1. Isolation transposon mutants reduced SOS induction response nalidixic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               acid <br />                                             described Introduction, DNA breaks likely generated nalidixic acid- <br />                                             stabilized cleavage complexes requisite step SOS induction. search genes                                             necessary step, used EZ::TN&#226;&#8222;&#162; &lt;KAN-2&gt; Tnp Transposome&#226;&#8222;&#162; Kit <br />                                             (Epicentre) create library transposon mutants JH39 strain E. coli. strain <br />                                             dinD1::lacZ fusion provides convenient reporter SOS response, allowing <br />                                             non-inducible mutants identified simple color screen (Heitman Model, 1991). <br />                                             strain sfiA (sulA) mutation, reduces filamentation inhibition cell <br />                                             division SOS induction (Friedberg et al., 1995). <br />  <br />                                             transposon-protein complexes transformed JH39 electroporation,                                             approximately 19,000 colonies random transposon insertions selected based                                             kanamycin resistance. colonies lifted nitrocellulose filters,                                             transferred plates containing nalidixic acid X-gal. 4-hour overnight incubation, <br />                                             small fraction colonies displayed unusual color intensities, indicating different amounts <br />                                             &#206;&#178;-galactosidase induction. collected 25 &#226;&#8364;&#339;white mutants&#226;&#8364;&#65533; apparent induction                                             105 &#226;&#8364;&#339;intermediate mutants&#226;&#8364;&#65533; reduced induction relative JH39. isolated 66 &#226;&#8364;&#339;dark <br />                                             blue mutants&#226;&#8364;&#65533; increased induction relative JH39; detailed (O&apos;Reilly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Kreuzer, 2004). <br />  <br />                               3.2. Secondary screens induction nalidixic acid UV <br />                                             began analyzing white intermediate mutants, apparent defects SOS <br />                                             induction nalidixic acid. secondary screen SOS induction performed disk <br />                                             test lawn mutant exposed levels nalidixic acid growth <br />                                             X-gal plates (Figure 1A). zones inhibition measured allow                                             determination drug sensitivity (Table 1). 25 white mutants showed normal-sized <br />                                             zones inhibition displayed blue color presence drug (e.g., compare <br />                                             W16 wild-type JH39 Figure 1A Table 1). intermediate mutants looked <br />                                             indistinguishable parental JH39 test, nalidixic acid sensitivity                                             intensity &#206;&#178;-galactosidase induction. judged false positives                                             discarded. remaining 45 intermediate mutants showed reduced induction <br />                                             compared JH39, 20 displaying noticeably color  20 <br />                                             mutants modestly hypersensitive nalidixic acid, exemplified mutants I78                                             I40 (Figure 1A Table 1). remaining 25 intermediate mutants, (I28) appeared                                             hypersensitive nalidixic acid (Figure 1A Table 1), 24 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             noticeably resistant similar wild-type (Tables 1 2; Figure 1A, I104). <br />  <br />                                             additional secondary screen, 25 white mutants remaining 45 intermediate <br />                                             mutants tested SOS response UV. Like nalidixic acid, UV treatment                                             induces SOS response, pathway induction requires recFOR recBC, <br />                                             presumably UV damage leads single-stranded DNA double-strand <br />                                             breaks (McPartland et al., 1980). Cells spotted twice X-gal plates,                                             spots exposed UV. wild-type control, strong induction evident                                             hours incubation (Figure 1B, JH39). <br />  <br />                                             25 white mutants uninducible UV test (e.g., Figure 1B, W16). <br />                                             Previous analysis JH39 genetic background shown background level &#206;&#178;- <br />                                             galactosidase expression SOS-inducing agent, suggesting low level SOS- <br />                                             independent transcription dinD1::lacZ fusion (Heitman Model, 1991;  <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 5 <br />  <br />  <br />                                             Kenyon Walker, 1980 Figure 1B, JH39 minus UV). Particularly discovery <br />                                             recA, recB recC mutants intermediate class  , surmised                                             white mutants completely unable express &#206;&#178;-galactosidase, likely transposon <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             insertions dinD1::lacZ fusion, did analyze  <br />  <br />                                             intermediate class 45 mutants turned interesting.                                             showed apparently wild-type levels SOS induction response UV (e.g., Figure 1B, I40), <br />                                             16 showed reduced induction (e.g., Figure 1B, I78, I104 I28; Table 2). <br />                                             Mutants unable process drug-stabilized cleavage complexes SOS signal <br />                                             29 normal SOS response UV. deficient SOS response <br />                                             nalidixic acid UV apparently general defect SOS induction. <br />  <br />                                             small &#206;&#178;-galactosidase expression shown intermediate mutants response <br />                                             nalidixic acid indicate mutated gene strictly required SOS <br />                                             response dinD1::lacZ fusion expressed low level SOS response <br />                                              mentioned . <br />  <br />                               3.3. Analysis insertions DNA sequencing P1 transduction <br />                                             interested 29 intermediate mutants showed normal UV responses, <br />                                             particularly 18 hypersensitive nalidixic acid. Nonetheless, avoid missing <br />                                             potentially interesting genes, decided sequence intermediate mutants. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             used MasterPure&#226;&#8222;&#162; DNA Purification Kit (Epicentre) purify genomic DNA                                             mutant used forward primer EZ::TN&#226;&#8222;&#162; transposon kit sequence <br />                                             end transposon. Using modified automatic sequencing protocol,                                             able obtain 100 bp unambiguous sequence mutant. Using                                             sequences, E. coli K12 genome searched using BLAST determine gene <br />                                             interrupted transposon (Table 2 Table 3). <br />  <br />                                             insertion mutants transposon trpB gene,                                             dinD1::lacZ fusion (Casadaban Cohen, 1980). SOS phenotypes caused <br />                                             insertions presumably result polar effects lacZ expression,                                             mutants discarded collection  included Tables). <br />  <br />                                             transposon insertions intermediate class recA gene.                                             recA mutants hypersensitive nalidixic acid light blue nalidixic <br />                                             acid UV tests. Given JH39 produces small &#206;&#178;-galactosidase                                             exposure DNA-damaging agent (Heitman Model, 1991), weak level &#206;&#178;- <br />                                             galactosidase recA insertion mutants probably unrelated SOS pathway. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously constructed recA56 derivative JH39, called JH59 (Heitman                                             Model, 1991), turned light blue nalidixic acid disk test. Furthermore, Kenyon <br />                                             Walker (1980) RecA required induced basal expression                                             din fusion constructs  dinD1 fusion tested). summary, new <br />                                             recA mutants expected phenotype: defective SOS response nalidixic acid <br />                                             UV, hypersensitive nalidixic acid. <br />  <br />                                             expected recB recC mutants mutants specifically defective <br />                                             SOS response nalidixic acid  Introduction).  sequencing revealed                                             insertions recB recC, 16 hypersensitive nalidixic acid  Tables <br />                                             1 2). Finding recB recC mutants group validated mutant hunt,                                             focused attention mutants phenotype search additional <br />                                             genes necessary SOS response nalidixic acid-stabilized cleavage complexes. <br />  <br />  <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                             Page 6 <br />  <br />  <br />                                             16 recBC mutants, intermediate mutants fit category                                             insertion ompA, gene major outer membrane protein, yhcL,                                             product suggested involved dicarboxylate transport (Janausch NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Unden, 1999). protein function suggested role processing cleavage <br />                                             complexes gene screen, suspicious                                             transposon insertions strains causing observed phenotype.                                             resolve uncertainty, performed phage P1 transductions transposon <br />                                             insertion mutation clean JH39 background, using kanamycin selectable marker.                                             transductants behaved like wild-type JH39 nalidixic acid UV tests, indicating <br />                                             transposon mutations ompA yhcL causing interesting phenotype. <br />                                             screened 19,000 colonies phenotype  quite <br />                                             possible acquired spontaneous mutation inactivated recB                                             recC.  PCR amplification genomic DNA ompA yhcL strains revealed <br />                                             recB genes approximately 1000 bp longer wild-type recB gene                                             JH39. likely endogenous insertion sequence interrupted recB <br />                                             genes strains, giving recBC-like phenotype. ompA yhcL insertions <br />                                             discarded collection included Tables. <br />  <br />                                             results ompA yhcL insertions raised concern transposon <br />                                             insertions collection causing phenotype  Insertions 10 <br />                                             different genes, including recA, recB recC, isolated multiple times NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             phenotype, arguing strongly transposon causative cases.                                              addition ompA yhcL, isolated single-hit insertions 6 additional genes/reading <br />                                             frames. test insertions causing SOS phenotype, performed <br />                                             P1 transductions using insertion mutants donors JH39 recipient. Kanamycin- <br />                                             resistant transductants hns, intQ, moaD mobA donors recapitulated SOS <br />                                             phenotypes donor parents, confirming transposon causative case. <br />                                             insertion mutants failed transduction test eliminated                                             collection  included Tables). <br />  <br />                               4. Discussion <br />                                             interested pathway(s) leads DNA breaks treatment inhibitors <br />                                             stabilize topoisomerase cleavage complexes. DNA breaks thought involved <br />                                             cytotoxicity DNA rearrangements caused inhibitors type II topoisomerases                                              Introduction). Elucidation detailed mechanism DNA break formation reveal <br />                                             important aspects replication fork dynamics, implications                                             antibacterial anticancer chemotherapy topoisomerase inhibitors. approached                                             issue experimentally taking advantage SOS response, isolating mutants reduced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             SOS induction nalidixic acid treatment. <br />  <br />                                             Mutational inactivation recB recC blocks SOS induction nalidixic acid UV <br />                                             (McPartland et al., 1980), focused attention mutants phenotype.                                             19,000 transposon insertion mutants, 18 defective induction nalidixic <br />                                             acid normal induction UV. Sixteen transposon insertion recB <br />                                             recC gene, confirms validity genetic screen. insertions <br />                                              strains approximately 1-kb insertion                                             recB gene, explains induction defect. E. coli numerous insertion sequences                                             roughly 1-kb size (Galas Chandler, 1989), surmise mutants                                             undergone transposition event inactivated recB gene time prior genetic <br />                                             screen. competent cells generating 19,000 transposon insertion mutants                                             grown single batch, strains started siblings insertion <br />                                             sequence recB prior transformation transposon. <br />  <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                               Page 7 <br />  <br />  <br />                                              total 18 mutants desired phenotype, 18 defective                                             recB recC. screened 19,000 colonies isolated recBC mutants,                                             screen quite thorough, unlikely novel gene uncovered NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             screening additional mutants using method criteria.                                             transposon mutagenesis strategy, possibility RecBCD non-essential <br />                                             protein involved SOS induction nalidixic acid UV. helicase activity <br />                                             RecBCD necessary SOS induction nalidixic acid treatment (Chaudhury                                             Smith, 1985).  role RecBCD probably limited generation single- <br />                                             stranded DNA break, enzyme likely involved break formation. <br />  <br />                                             results consistent possibility essential cellular proteins involved                                             DNA break formation cleavage complex. example, transcription DNA <br />                                             replication necessary. specific model replication fork run model,                                             replisome directly causes DNA breaks collision cleavage complex <br />                                              Introduction). transcription replication necessary break formation,                                             mutants reduced SOS induction simply slower growth.                                             measured growth rate insertion mutant genes uncovered                                              did significantly decreased growth rates: cyaA, icdA, mcrA/ <br />                                             elbA, crp, tufA (data shown; did test tufB, presumably grow similarly <br />                                             tufA mutant). mutants reduced SOS induction poor <br />                                             growth, note mutants isolated little growth defect, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             expected isolate poor-growth mutants was necessary                                             reduce SOS induction. <br />  <br />                                             second possible explanation finding recBC mutants desired phenotype                                             multiple pathways process cleavage complex SOS-inducing signal.                                             candidates genes involved redundant pathways uncovered mutants <br />                                             modestly defective SOS induction nalidixic acid seemingly normal                                             UV, icdA, intQ, mcrA/elbA, moaD, mobA, tufA/B trkH (Table 2). mutants <br />                                             group altered sensitivity nalidixic acid trkH, modestly <br />                                             resistant drug (Table 1). trkH gene encodes integral membrane protein involved <br />                                             potassium uptake (Silver, 1996) trkH mutations impact nalidixic acid <br />                                             uptake efflux.  decreased SOS response trkH mutants simply reflect                                             lower effective level inhibitor inside cell. genes mentioned                                             function clearly suggests involvement cleavage complex processing.                                              gene products known suspected DNA cleavage activities: McrA,                                             restriction endonuclease, IntQ, encoded cryptic prophage related                                             integrase family proteins. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             inherent limitation genetic screen raises possible explanation finding                                             recBC mutants strong SOS defect specific nalidixic acid treatment.                                             mutants missed screen partially constitutive                                             SOS response.  absence nalidixic acid, desired mutants                                             levels &#206;&#178;-galactosidase similar greater levels wild-type cells                                             nalidixic acid. Insertions variety important DNA metabolism genes cause SOS <br />                                             constitutive phenotype (O&#226;&#8364;&#8482;Reilly Kreuzer, 2004), currently analyzing                                             defective SOS induction nalidixic acid treatment. <br />  <br />                                             collection included insertions genes caused SOS defects                                             nalidixic acid UV treatment. Mutations genes known cause pleiotropic <br />                                             effects altering nucleiod structure gene expression (hns), protease function (clpP)                                             cyclic <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span> regulatory (cyaA crp) (Dorman, 2004; Saier et al., 1996; Gross, <br />                                             1996). SOS defects mutants likely caused  <br />  <br />                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                                                 Page 8 <br />  <br />  <br />                                             indirect effect pleiotropy, results provide clues subtleties SOS <br />                                             regulatory  <span id='am-46' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-47' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span>cAMP</span>  involving products crp cyaA genes,                                             previously linked SOS context cAMP-dependent SOS induction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             resting cells (Taddei et al., 1995). <br />  <br />  <br />  <br />                               Abbreviations <br />                                             LB <br />                                                            L-broth <br />  <br />                                             UV <br />                                                            ultraviolet light <br />  <br />                                             X-gal <br />                                                            5-bromo-4-chloro-3-indolyl-&#206;&#178;-D-galactopyranoside <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Gene. Author manuscript; available PMC 2008 January 15. <br />                             Newmark et al.                                                                                            Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 1. <br />                                             Nalidixic acid disk tests UV tests wild type insertion mutants.  Overnight cultures <br />                                             strain LB diluted OD560 approximately 1.0 200 &#206;&#188;L diluted <br />                                             cultures added 3.0 mL agar. agar mixture poured LB <br />                                             plate containing X-gal 60 &#206;&#188;g/mL. circular glass fiber filters (GF/C; approximately 1 <br />                                             cm diameter) placed cooled plate. solution nalidixic acid                                             delivered filter disks (0, 2, 4 8 uL 5 mg/ml stock 10 mM NaOH, corresponding <br />                                             0, 10, 20 40 &#206;&#188;g), starting left position continuing clockwise NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             plate. plates incubated overnight 37&#194;&#176;C photographed morning.                                             strain given plate, gene interrupted transposon                                             parentheses  parental JH39, transposon, W16,                                             sequenced). B. Overnight cultures strain spotted (8 &#206;&#188;L) duplicate LB <br />                                             plates containing X-gal 60 &#206;&#188;g/mL. hours incubation 37&#194;&#176;C, half plate                                             exposed UV (52.5 J/m2, General Electric G15T8 germicidal bulb)                                             half protected sheet cardboard. plates wrapped foil protect <br />                                             light incubated additional hours 37&#194;&#176;C. plates stored <br />                                             4&#194;&#176;C photographed day. <br />  <br />  <br />  <br />  <br />                                                       Gene. Author manuscript; available PMC 2008 January 15. <br />                                 Newmark et al.                                                                                                                               Page 11 <br />  <br />                                                                                                     Table 1 <br />                                Zones inhibition nalidixic acid. <br />  <br />                                                                                                              Zones Inhibition (mm NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Nalidixic Acid (&#206;&#188;g):                         10                               20                               40 <br />                                         JH39                             14.6 (&#194;&#177;1.0)                      17.2 (&#194;&#177;1.3)                      19.8 (&#194;&#177;1.4) <br />                                         recA                                17.0                             20.5                             24.0                   HS <br />                                         recB                             17.8 (&#194;&#177;0.7)                      20.8 (&#194;&#177;0.9)                      23.6 (&#194;&#177;0.7)               HS <br />                                         recC                             18.3 (&#194;&#177;1.0)                      21.6 (&#194;&#177;1.3)                      24.5 (&#194;&#177;0.9)               HS <br />                                          hns                                17.0                             20.0                             23.0                   HS <br />                                         clpP                                14.0                             16.5                             20.0                   N <br />                                         cyaA                             13.3 (&#194;&#177;2.5)                      17.0 (&#194;&#177;1.7)                      20.3 (&#194;&#177;1.5)               N <br />                                         icdA                                14.0                             16.0                             18.0                   N <br />                                         intQ                                14.0                             16.0                             20.0                   N <br />                                     mcrA/elbA                               13.0                             16.0                             18.5                   N <br />                                        moaD                                 15.0                             17.0                             21.0                   N <br />                                        mobA                                 15.0                             17.0                             20.0                   N <br />                                        tufA/B                               14.5                             16.5                             19.5                   N <br />                                         crpb                             11.7 (&#194;&#177;0.6)                      15.7 (&#194;&#177;0.6)                      19.7 (&#194;&#177;0.6)               r <br />                                         trkH                                12.0                             14.0                             17.0                   R <br />  <br />                                                        diameter zones inhibition measured plates prepared identically depicted Figure 1A. HS, hypersensitive, R, markedly <br />                            resistant, r, slightly resistant, N, normal (similar parental JH39). values shown represent average measurements independent transposon <br />                            insertion mutants. Standard error shown transposon insertion mutants available  Table 3). JH39 values represent                            average standard deviation 18 different measurements parental strain. <br />                            b <br />                             did significantly differ, measurements crp promoter insertion mutant (I42) included calculation. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                  Gene. Author manuscript; available PMC 2008 January 15. <br />                                                                   NIH-PA Author Manuscript                                                  NIH-PA Author Manuscript                                               NIH-PA Author Manuscript <br />                                                                                                                                                                Table 2 <br />                                                              Summary intermediate mutants. <br />  <br />                                                                  Genea                       #                 NAL Colorb                         UV Colorc                     NAL Sensitivityd            Putative known functione <br />  <br />                                                                  recA                        2                      +                                 +                                 HS                  Homologous pairing strand exchange protein <br />                                                                  recB                        8                      +                                +++                                HS                  RecB subunit helicase-nuclease ExoV <br />                                                                  recB  f                  2                      +                                +++                                HS                  RecB subunit helicase-nuclease ExoV <br />                                                                  recC                        8                      +                                +++                                HS                  RecC subunit helicase-nuclease ExoV <br />                                                                                                                                                                                                                                                                                   Newmark et al. <br />  <br />  <br />  <br />  <br />                                                                  hns                         1                      ++                               ++                                 HS                  DNA-binding protein H-NS <br />                                                                  clpP                        2                      ++                               ++                                 N                   <span id='am-8' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span>-dependent serine protease <br />                                                                  cyaA                        3                      ++                                +                                 N                   Adenylate cyclase <br />                                                                  icdA                        2                      ++                               +++                                N                   Isocitrate dehydrogenase (TCA) <br />                                                                  intQ                        1                      ++                               +++                                N                   Putative integrase gene defective prophage Qin <br />                                                                  mcrA/elbAg                  2                      ++                               +++                                N                   Restriction endonuclease (McrA)/isoprenoid biosynthesis (elbA) <br />                                                                  moaD                        1                      ++                               +++                                N                   Molybdopterin converting factor, small subunit <br />                                                                  mobA                        1                      ++                               +++                                N                   Molybdopterin-<span id='am-27' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span>guanine</span> dinucleotide biosynthesis protein                                                                  tufA/B                      2                      ++                               +++                                N                   Protein chain elongation factor EF-Tu <br />                                                                  crph                        4                      ++                                +                                  r                  cAMP receptor protein <br />                                                                  trkH                        2                      ++                               +++                                R                   Membrane protein, K+ transport <br />  <br />                                                                                                                                          gene isolated  P1 transductions demonstrated transposon insertion causative. <br />                                                                     b <br />                                                                      Intensity blue color response nalidixic acid, determined disk test. +++ = normal, ++ = light blue, + = light blue. descriptions relative JH39 directly <br />                                                                     comparable UV tests. <br />                                                                     c <br />                                                                      Intensity blue color response UV radiation, determined UV test. +++ = normal, ++ = light blue, + = light blue. descriptions relative JH39 directly comparable <br />                                                                     nalidixic acid disk tests. <br />                                                                     d <br />                                                                      Drug sensitivity nalidixic acid, determined disk test (Table 1). N = normal (similar parental JH39), HS = hypersensitive, r = slightly resistant, R = markedly resistant <br />                                                                     e <br />                                                                      Functions defined NCBI website E. coli K12. Italicized function putative. <br />                                                                     f <br />                                                                      strains transposon yhcL ompA apparent element recB. <br />                                                                     g <br />                                                                         transposon insertion located genes. <br />  <br />  <br />  <br />  <br /> Gene. Author manuscript; available PMC 2008 January 15. <br />                                                                     h <br />                                                                         crp insertions coding sequence, fourth located 115 nucleotides upstream start codon. <br />                                                                                                                                                                                                                                                                                   Page 12 <br />                                   Newmark et al.                                                                                                                             Page 13 <br />  <br />                                                                                                      Table 3 <br />                                  Locations directions transposon insertions. <br />  <br />                            genea                   strain number                           9 bp direct repeatb                        <span id='am-19' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> transcribed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            clpP                    I35                                     ACTTTGCAC                                  34 <br />                                                    I75                                     ACATGGCTA                                  163 <br />                            crp                     I42                                     TCACATTAC                                  &#226;&#710;&#8217;114c <br />                                                    I100                                    GTGCTTGGC                                  12 <br />                                                    I104                                    GGGTAAAGA                                  176 <br />                                                    I105                                    CGGTAAAGA                                  608 <br />                            cyaA                    I65                                     GTGTGGAAG                                  451 <br />                                                    I102                                    GCATATACT                                  542 <br />                                                    I103                                    CCGCAGGAT                                  1242 <br />                            hns                     I28                                     AACCTGGAC                                  329 <br />                            icdA                    I37                                     GTTTGAAGC                                  1010 <br />                                                    I107                                    CTCCAAGCC                                  418 <br />                            intQ                    I6                                      GGGTTAAGC                                  2 <br />                            mcrA/elbA               I24                                     GTTCAGAGT                                  +361d <br />                                                    I44                                     GGGTTGGAC                                  +248d <br />                            moaD                    I19                                     ACCATCCGC                                  196 <br />                            mobA                    I22                                     GCCTTACAT                                  314 <br />                            recA                    I7                                      GTCCAACAC                                  563 <br />                                                    I78                                     GGTGAAGAC                                  99 <br />                            recB                    I40                                     GGTATGTGC                                  1568e <br />                                                    I57                                     GTTCGATGG                                  1940 <br />                                                    I68                                     GGATGTGGT                                  1922 <br />                                                    I73                                     GATTTACAC                                  3470 <br />                                                    I79                                     GGTATGTGC                                  1568e <br />                                                    I83                                     TTCTTGCAC                                  2868 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    I93                                     ACCAGGCCT                                  491 <br />                                                    I94                                     ATGGTGAAC                                  1365 <br />                            recC                    I32                                     GATAGCAGT                                  1119 <br />                                                    I36                                     CCCGCCACC                                  1558 <br />                                                    I50                                     GCTGGGAAT                                  2039 <br />                                                    I52                                     GTCTGGACG                                  34 <br />                                                    I54                                     ACATGATGG                                  3241 <br />                                                    I85                                     GGGTAAACA                                  699 <br />                                                    I90                                     GCCTGGCAC                                  442 <br />                                                    I97                                     GTTGTTGGG                                  1601 <br />                            trkH                    I23                                     TAACTACGG                                  761 <br />                                                    I27                                     GATTATCCC                                  71 <br />                            tufA                    I29                                     CACGTAGAC                                  959 <br />                            tufB                    I17                                     GGCGTAGAG                                  1047 <br />  <br />                                                           Alternate gene names: elbA (ycgW), intQ (b1579) <br />                            b <br />                             9 bp repeat determined using single primer transposon. base inferred repeat shows intact nucleotide                            gene, reading 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; direction respect transcription. <br />                            c <br />                             insertion located 115 nucleotides upstream start codon crp gene. <br />                            d <br />                             insertions lay 361 248 nucleotides downstream stop codon mcrA gene. <br />                            e <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             insertions, identical, isolated independently clones. <br />  <br />  <br />  <br />  <br />                                                                    Gene. Author manuscript; available PMC 2008 January 15. <br />   <br /> </body></html>