<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Apr. 1996, p. 1557&#226;&#8364;&#8220;1566                                                                         Vol. 16,  4 <br /> 0270-7306/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />    Synthetic Enhancement TFIIB Defect Mutation SSU72, <br />         Essential Yeast Gene Encoding Novel Protein             Affects Transcription Start Site Selection Vivo <br />                                            ZU-WEN SUN&#226;&#8364;&#160;            MICHAEL HAMPSEY* <br />                             Department Biochemistry Molecular Biology, Louisiana State University <br />                                            Medical Center, Shreveport, Louisiana 71130 <br />                     Received 14 November 1995/Returned modification 28 December 1995/Accepted 23 January 1996 <br />  <br />              ssu72 mutant Saccharomyces cerevisiae identified enhancer TFIIB defect (sua7-1)           confers cold-sensitive growth defect downstream shift transcription start site selection.           ssu72-1 allele did affect cold sensitivity  combination sua7-1, created heat-sensitive phenotype. <br />            start site selection ADH1 gene dramatically shifted downstream normal <br />           sites. effects rescued SUA7 SSU72, defining functional relationship <br />           genes. SSU72 single-copy, essential gene encoding novel protein 206 amino acids.           ssu72-1 allele result 30-bp duplication creating sequence encoding Cys-X2-Cys-X6-Cys-X2-Cys <br />           zinc binding motif near N terminus Ssu72p. Mutational analysis demonstrated N terminus           Ssu72p essential function cysteine residues normal mutant proteins critical. <br />           discuss possibility potential zinc binding motif Ssu72p facilitates assembly tran- <br />           scription preinitiation complex effect important accurate start site selection vivo. <br />  <br />  <br />    Transcription initiation RNA polymerase II (RNAP II)                   functional transcription preinitiation complex vitro (7, 71). <br /> involves concerted action large number proteins.                 Assembly complex nucleated association TBP class gene specific binds promoters containing                  TFIID) TATA element, followed binding specific target sequences (reviewed references 33 52).                TFIIB. TBP physically interacts TFIIB, crystal factors stimulate rate transcription response             structure TFIIB-TBP-TATA element ternary complex specific physiological developmental stimuli. contrast,            recently described (45). RNAP II, association TFIIF, <br /> general transcription factors required accurate, basal-              binds complex. TFIIE TFIIH associate level transcription absence gene-specific stimulation              complex binding RNAP II required pro- <br /> (reviewed references 6 77). exception                 moter clearance manner dependent <span id='am-23' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-24' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span> <br /> TATA-binding protein (TBP), general factors bind                  hydrolysis (27). considerable evidence specific DNA sequences. class factors, known                support scheme assembly preinitiation complex <br /> coactivators, mediators, adaptors, functions communica-                vitro, RNAP II appears recruited promoters vivo <br /> tion gene-specific general factors (reviewed                 holoenzyme complex includes general factors. reference 25). class includes TBP-associated fac-                direct support premise yeast mammalian <br /> tors  TBP, make TFIID (26), oth-               systems, holoenzyme complexes somewhat different <br /> er adaptors, including Ada2p-Ada3p-Gcn5p complex                      compositions reported (35, 37, 47). <br /> Saccharomyces cerevisiae (31). Transcriptional activity                 Genetic studies using yeast S. cerevisiae provided <br /> affected chromatin structure DNA template                   valuable new information regarding general transcription <br /> viewed reference 48). biochemical genetic exper- <br />                                                                              factors functions. example, mutations SPT <br /> iments suggest function gene-specific factors                                                                              genes identified suppressors Ty solo d insertions overcome repressive effects chromatin transcrip- <br />                                                                              inhibit transcription adjacent HIS4 LYS2 genes (72). <br /> tion initiation. identities factors mecha- <br />                                                                              genes, SPT15, encodes TBP (15, 29). Mutations <br /> nisms interact confer accurate, regulated <br /> initiation RNAP II subject intense investigation               SPT15 shift initiation d initiation site normal critical understanding regulated cell growth                 initiation site HIS4, establishing role TBP development.                                                                 normal transcription initiation vivo (15). class    general transcription factors initially defined               genes, designated SUA, identified suppressors biochemical studies include TBP (initially component                aberrant open reading frame located transcribed leader TFIID complex), TFIIA, TFIIB, TFIIE, TFIIF, TFIIH,                    region CYC1 gene (50). genes, SUA7 TFIIJ (reviewed references 11 76). factors                  SUA8 (RPB1) encode, respectively, TFIIB largest sub- <br /> assemble DNA template defined order generate                  unit RNAP II (4, 50). Mutations gene shifted <br />                                                                              transcription start site selection downstream normal <br />                                                                              sites. experiments demonstrated TFIIB                                                                              largest RNAP II subunit required accurate transcrip- <br />    * Corresponding author. Present address: Department Biochem- <br /> istry, Robert Wood Johnson Medical School, UMDNJ, 675 Hoes Ln., <br />                                                                              tion start site selection vivo suggested specific <br /> Piscataway, NJ 08854-5635.                                                   interactions TFIIB largest RNAP II subunit <br />    &#226;&#8364;&#160; Present address: Department Biochemistry, Robert Wood John-          affect process (4). <br /> son Medical School, University Medicine Dentistry New                 Genetic analysis transcription initiation po- <br /> Jersey, Piscataway, NJ 08854-5635.                                           tential identify transcription factors undetect- <br />  <br />                                                                       1557 <br />  1558       SUN HAMPSEY                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />                                                              TABLE 1. Yeast strains used study <br />        Strain                                                                                                    Genotype <br /> T16 .............................................................................................MATa cyc1-5000 cyc7-67 leu2-3,112 ura3-52 cyh2 <br /> YDW546....................................................................................MATa cyc1-5000 cyc7-67 leu2-3,112 ura3-52 cyh2 sua7-1 <br /> YMH71-9C................................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 <br /> YZS19 ........................................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 <br /> YZS19/YCplac33 ......................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3] <br /> YZS19/pDW11..........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SUA7] <br /> YZS19/pM385 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU72] <br /> YZS19/pM586 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU72] <br /> YZS19/pM681 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU73] <br /> YZS19-1B ..................................................................................MATa ura3-52 leu2-3,112 sua7-1 ssu72-1 <br /> YMH171 ....................................................................................MATa ura3-52 leu2-3,112 his3 trp1D <br /> YZS89A .....................................................................................MATa ura3-52 leu2-3,112 his3 trp1D ssu72::LEU2 [pM586: URA3 SSU72] <br /> LRB535/536...............................................................................MATa/MATa his3D200/his3D200 leu2-3,112/leu2-3,112 ura3-52/ura3-52 <br />  <br />  <br />  <br /> ed biochemical means. example, SPT3 (13), SPT7                              sua7-1 segregant derived YDW546 (44). YZS19 spontaneous cs1 <br /> (21), SPT8 (14) genes encode novel factors play im-                           revertant YMH71-9C. YZS19/YCplac33 (sua7 ssu72 ssu73), YZS19/pDW11 <br />                                                                                        (SUA7 ssu72 ssu73), YZS19/pM385 YZS19/pM586 (sua7 SSU72 ssu73), portant roles normal transcription initiation                          YZS19/pM681 (sua7 ssu72 SSU73) derivatives YZS19 carrying indi- <br /> detected biochemically. products genes                          cated plasmids. YZS19-1B sua7 ssu72 SSU73 segregant derived functionally related TBP, Spt3p shown phys-                         backcross YZS19 YDW546. Strain YZS89A (SUA7 ssu72::LEU2 [URA3- <br /> ically interact TBP (13). Furthermore, human TAFII18                           SSU72]) used host plasmid shuffle derived structurally similar N-terminal region Spt3p (42).                           YMH171 described  LRB535/536 diploid strain used host                                                                                        disruption SSU72 (65). important example transcription factors                              media prepared according standard procedures (59). YPD refers <br /> initially identified genetic means Srb proteins. Mu-                        rich medium consisting 1% yeast extract, 2% peptone, 2% glucose. <br /> tations SRB genes selected basis                           5-FOA medium contains 5-fluoro-<span id='am-18' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span>orotic acid</span> used positive selection <br /> ability suppress growth phenotypes associated trunca-                          URA31 gene (5). <br />                                                                                           Genetic methods nomenclature. Standard procedures used cross- <br /> tion heptapeptide repeat domain (CTD) located                                                                                        ing yeast strains, selecting diploids, inducing sporulation, dissecting tetrads <br /> C terminus largest subunit RNAP II (46).                               (60). Yeast transformations lithium acetate procedure (23). <br /> factors components RNAP II holoenzyme                                Plasmid DNA recovered S. cerevisiae described previously (30) clude novel proteins associated preinitiation complex                         introduced Escherichia coli electroporation (12). transforma- <br /> (36, 68) kinase-cyclin pair involved phos-                     tions E. coli calcium chloride procedure (40). E. coli <br />                                                                                        XL1-Blue (recA1 endA1 gyrA96 thi-1 supE44 hsdR17 (rK2 mK2) relA1 lac [F9 <br /> phorylation RNAP II CTD (39). SPT, SUA,                                 proAB lacIq lacZDM15 Tn10 (Tetr)]) used host strain plasmids. <br /> SRB systems underscore importance genetic approaches                               Standard yeast genetic nomenclature used  upper- identifying critical factors involved transcription initia-                     lowercase italicized symbols denoting dominant recessive alleles, respec- <br /> tion RNAP II.                                                                       tively (e.g., SSU72 ssu72-1). symbols Ura2, Leu2, Trp2 denote <br />    continuing use genetic methods identify factors                        failure grow media lacking Ura, Leu, Trp (2Ura, 2Leu, 2Trp <br />                                                                                        media, respectively). Cold- heat-sensitive growth phenotypes expressed important transcription initiation RNAP II.                        cs2 ts2, respectively, denote distinctly impaired growth  approaches isolate characterize suppressors                        growth) rich medium (YPD) 16 378C. <br /> defects TFIIB (sua7) largest subunit RNAP II                              Plasmids. Restriction sites pertaining SSU72 constructs depicted Fig. <br /> (sua8) affect start site selection. sua7-1 allele encodes                     2. low-copy-number vectors YCp50 (CEN URA3) (55), pRS314 (CEN TRP1) TFIIB Glu-623Lys (E62K) replacement responsible                              pRS316 (CEN URA3) (62), YCplac33 (24); integrating vector <br />                                                                                        pRS305 (LEU2) (62); high-copy-number vector YEp24 (2 mm URA3) aberrant initiation pattern cold-sensitive                          (55) described previously. plasmids constructed follows. <br /> (cs2) growth defect (51). selecting revertants                             pM264 2.3-kb SalI-SphI fragment encompassing SUA7 high-copy- <br /> sua7-1 cs2 phenotype, uncovered genes desig-                             number vector YEp24. (ii) pM384 original SSU72 clone carrying nated ssu71, ssu72 ssu73  suppressors sua7, genes 1,                        ;12.5-kb DNA insert vector YCp50. (iii) pM385 1.05-kb EcoRI- <br />                                                                                        HindIII SSU72 DNA fragment pRS316. (iv) pM471 3.8-kb HindIII- <br /> 2, 3). SSU71 identical TFG1, gene encoding                           HindIII SSU72 DNA fragment vector pRS316. (v) pM471A identical largest subunit TFIIF (64), SSU73 identical RPB9,                         pM471, HindIII fragment inserted pRS316 gene encoding 14.2-kDa subunit RNAP II (66).                            opposite orientation. (vi) pM523 derived pM539 replacement ssu71 ssu73 suppressors functionally similar                           HindIII-BclI DNA fragment encompassing entire SSU72 open reading frame <br />                                                                                        LEU2 HindIII-BamHI DNA fragment. (vii) pM524 ssu72 DNA lacking compensate cs2 phenotype                                sequence encoding N-terminal 23 amino acids amplified PCR <br /> restore normal initiation pattern sua7-1 background                         primers generate flanking EcoRI SpeI sites. Amplified DNA (64, 66).  SSU72 gene. contrast                         digested EcoRI SpeI ligated YCpIF17 (18). construct <br /> ssu71 ssu73 suppressors, ssu72-1 enhancer                             expresses HA-Ssu72 protein, lacking N-terminal 23 residues Ssu72p, <br />                                                                                        GAL1 promoter. (viii) pM539 2.0-kb ApaI-BamHI SSU72 DNA <br /> sua7-1 defect, conferring synthetic heat-sensitive phenotype                         fragment pRS316. (ix) pM540 identical pM539, carries (ts2) causing start site selection ADH1 CYC1                            ssu72-1 allele. pM540 recovered strain YZS19 gap repair using <br /> genes shift downstream normal sites. Genetic                         pM539 digested EcoRI HindIII. (x) pM550 identical molecular analyses demonstrated SSU72 essen-                            pM524, encodes length Ssu72p HA fusion protein                                                                                        GAL1 promoter. (xi) pM552 2.7-kb XbaI-HindIII DNA fragment <br /> tial gene encoding novel protein affect assem-                        encompassing SSU72 ORF498 LEU2 integrating vector pRS305. (xii) <br /> bly transcription preinitiation complex.                                        pM586 1.05-kb EcoRI-HindIII SSU72 DNA fragment YCplac33. (xiii) <br />                                                                                        pM681 RPB9 (SSU73) gene YCplac33 vector. 3.2-kb PstI- <br />                                                                                        BamHI fragment encompassing RPB9 transferred plasmid Ro406 (20) <br />                     MATERIALS METHODS                                              YCplac33. (xiv) pM699 includes entire SSU72 open reading frame                                                                                        amplified PCR oligonucleotides create flanking XmaI-NcoI sites    Yeast strains media. yeast strains used study listed Table   inserted CUP1-<span id='am-6' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span>glutathione</span> S-transferase (GST) vector pCBGST-1 (41), <br /> 1. Strains T16 YDW546 described previously (50). YMH71-9C                allowing expression GST-Ssu72 fusion protein CUP1 control. (xv) <br />  VOL. 16, 1996                                                                                                           YEAST SSU72 GENE                     1559 <br />  <br />  <br /> pM712 1.05-kb EcoRI-KpnI DNA fragment encompassing SSU72 <br /> wild-type gene TRP1 vector pRS314. (xvi) pM740 identical pM699, N-terminal 4 residues Ssu72p eliminated GST-Ssu72 fusion protein restriction sites flanking SSU72 NotI NcoI. (xvii) pM742 identical pM740, N-terminal 10 amino <br /> acids Ssu72p eliminated GST-Ssu72 fusion protein. (xviii) <br /> pDW11 SUA7 wild-type gene vector pRS316 (50). <br />    Isolation SSU72 gene. SSU72 gene isolated YCp50 <br /> genomic library (56) complementation ts2 phenotype conferred ssu72-1 allele strain YZS19. relationship complementing <br /> DNA genetically defined ssu72 locus established integration cloned DNA genome strain YDW546 (sua7 SSU721 leu2) homologous recombination (55). Plasmid pM552, carries 2.7-kb XbaI- <br /> HindIII DNA fragment encompassing SSU72, digested unique BamHI               FIG. 1. Conditional growth defects associated sua7, ssu72, ssu73 <br />                                                                                      mutations. Strain YZS19 (sua7 ssu72 ssu73) spontaneous cs1 derivative site open reading frame adjacent SSU72. Linearized pM552                                                                                      YMH71-9C (sua7 SSU72 SSU73). strains depicted rows 2 5 introduced YDW546, transformants selected 2Leu  <br />                                                                                      derived YZS19 introducing low-copy-number plasmids carrying dium. tags chromosomal locus cloned DNA LEU2 <br />                                                                                      SUA7, SSU72, SSU73 vector  Consequently, strains isogenic, <br /> marker, confirmed Southern blot analysis (data shown).                                                                                      differing indicated alleles. Strain names follows: YMH71-9C <br /> strain crossed YZS19-1B (sua7 ssu72 leu2), resulting diploid      (sua7 SSU72 SSU73), YZS19/YCplac33 (sua7 ssu72 ssu73), YZS19/pDW11 <br /> sporulated dissected. Prior mating, SUA7 gene introduced         (SUA7 ssu72 ssu73), YZS19/pM586 (sua7 SSU72 ssu73), YZS19/pM681 <br /> YZS19-1B URA3 high-copy-number plasmid (pM264); presence             (sua7 ssu72 SSU73). results demonstrate ssu72-1 enhancer SUA7 wild-type gene improved sporulation germination efficiencies         sua7-1 allele, conferring severe ts2 phenotype (cf. row 5 row 1), homozygous sua7 diploids. Segregants cured plasmid-borne SUA7            ssu73-1 suppressor cs2 phenotype associated sua7-1 (cf. row 4 gene 5-FOA medium subsequently scored Leu ts phenotypes.              row 1). phenotypes associated ssu72-1 ssu73-1 mutations Complete cosegregation Leu1/ts1 Leu2/ts2 phenotypes confirmed             absolutely dependent sua7-1 allele, SUA7 plasmid rescues cloned gene SSU72.                                                       phenotypes (row 3). Columns represent cell growth serial dilutions (left    DNA sequence analysis. Plasmids pM471 pM471A linearized           right). revertant colonies seen rows 2 5 378C indicative SacI NotI sites multiple cloning region, nested sets dele-     high spontaneous reversion frequency ssu72-1 allele. <br /> tions generated exonuclease III S1 nuclease (Erase Base Kit; <br /> Promega). Single-stranded DNA generated selected clones VCSM13 helper phage (Stratagene Cloning Systems) presence kana- <br /> mycin, described previously (58). strands entire 3,785-bp DNA                                           RESULTS <br /> fragment sequenced dideoxy-terminator method M13 uni- <br /> versal primer; sequencing gaps filled SSU72-specific primers.          Isolation ssu72 mutant. sua7-1 allele encodes entire open reading frame ssu72-1 allele sequenced single-           E62K replacement TFIIB confers altered start site <br /> stranded DNA derived plasmid pM540.                                             selection severe cs2 phenotype (51) (Fig. 1, row 1). Cold <br />    Construction ssu72::LEU2 null allele. Plasmid pM523 digested ApaI BamHI, resulting ssu72::LEU2 DNA fragment introduced <br />                                                                                      sensitivity associated defects assembly mul- diploid strain LRB535/536, selecting Leu1 transformants. Integration     tisubunit complexes (reviewed reference 73). suggests SSU72 loci, resulting replacement SSU72 gene           sua7-1 mutant defective assembly LEU2 (ssu72::LEU2), confirmed Southern blot analysis (data shown).        transcription preinitiation complex. Consistent possi- <br />    Determination transcript start sites. Primer extension performed        bility, form TFIIB encoded sua7-1 functional described previously, using total RNA ADH1-specific primer oIP-87 <br /> (50). Primer extension products resolved 6% polyacrylamide DNA se- <br />                                                                                      exhibits normal stability vivo (51). Consequently, suppres- <br /> quencing gels visualized Molecular Dynamics PhosphorImager.           sors enhancers E62K form TFIIB include <br /> molecular size markers SUA7 DNA sequenced                factors facilitate impair complex assembly. <br /> primer oIP-133 (50).                                                                    spontaneous cs1 revertants    Isolation ssu72-1 allele. ssu72-1 allele cloned strain <br /> YZS19 gap repair (57). Plasmid pM539 digested EcoRI HindIII, <br />                                                                                      sua7-1 mutant (YMH71-9C) isolated. effort deleting entire SSU72 open reading frame  Fig. 2). gapped        identify potential pleiotropic phenotypes, cs1 revertants <br /> vector purified following size fractionation agarose gel introduced    scored ts2. revertants severely YZS19. Ura1 transformants selected screened retention       ts2, failing grow rich medium (YPD) 378C. ts2 phenotype associated ssu72-1. Plasmid DNA recovered digested EcoRI HindIII, confirming recovery ssu72 DNA. plasmid                                                                                      strains, YZS14 YZS45, result recessive <br /> designated pM540.                                                                    mutations SSU71 (TFG1), gene encoding largest <br />    vitro construction ssu72 alleles. Site-directed mutations SSU72     subunit TFIIF (64). Genetic analysis strain, <br /> generated synthetic oligonucleotides PCR according technique            YZS19, revealed cs1 ts2 phenotypes result <br /> described previously (43). Template DNA plasmid pM539 (SSU72) pM540 (ssu72-1). Following amplification, DNA digested EcoRI <br />                                                                                      recessive mutations unlinked genes, designated <br /> (SSU72 site) KpnI (vector site) inserted TRP1 vector pRS314         ssu72-1 ssu73-1. ssu73-1 mutation suppressed cs2 <br /> (62). Plasmids expressing N-terminal deletions Ssu72p (4-, 10-, 23-nucle-     phenotype conferred sua7-1 (66), ssu72-1 mu- <br /> otide deletions) constructed described                                 tation combination sua7-1 responsible ts2 <br />    Functional analysis ssu72 alleles. plasmid shuffle (61) set assess function vitro-generated ssu72 alleles. chromosomal <br />                                                                                      phenotype (Fig. 1). <br /> SSU72 locus strain YMH171 (SUA71 SSU721) disrupted ApaI-                Isolation SSU72 gene. ts2 phenotype strain <br /> BamHI DNA fragment encompassing ssu72::LEU2 plasmid pM523. Strain               YZS19 used clone SSU72 wild-type gene. YCp50 <br /> viability maintained introducing plasmid pM586 (URA3 SSU72)              yeast genomic library (56) introduced YZS19, YMH171 prior disruption. resulting strain designated YZS89A (Table <br /> 1). Phenotypes associated vitro-constructed ssu72 alleles assessed                                                                                      transformants selected 378C. ts1 trans- <br /> introducing ssu72 TRP1 plasmids YZS89A subsequently curing Trp1             formants isolated total ;60,000 Ura1 transfor- <br /> transformants pM586 (URA3 SSU72) 5-FOA medium. Failure recover              mants. Plasmid DNA recovered 12 transformants <br /> 5-FOA-resistant (FOAr) transformants indicated ssu72 allele non-         reintroduced YZS19, confirming ts1 pheno- <br /> functional. FOAr strains subsequently scored growth rich medium <br /> (YPD). Plasmids pM712 (SSU72), pM699 (CUP1-GST-SSU72), pM550 <br />                                                                                      type strain conferred plasmid DNA (GAL1-HA-SSU72) served positive controls experiments.                strain reversion. Restriction analysis identified distinct <br />    Sequence comparisons homologies. DNA protein sequence compar-             plasmids common 3.8-kb HindIII restriction fragment. <br /> isons performed BLAST algorithm (2), accessed National         plasmids, designated pM384, chosen Center Biotechnology Information electronic mail service. <br />    <span id='am-34' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. 3.8-kb DNA sequence encom- <br />                                                                                      analysis. <br /> passing SSU72 entered GenBank data library accession              Plasmid pM384 carries insert approximately 12.5 kb number U20390.                                                                       vector YCp50. gene responsible complementation <br />  1560      SUN HAMPSEY                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                                    recently identified ssu73-1 suppressor allele                                                                                 RPB9 (66).  genes represented YZS19 (SUA7, <br />                                                                                 SSU72, SSU73/RPB9) cloned. enabled                                                                                 confirm genetically defined relationships                                                                                 genes introducing low-copy-number vectors carrying SUA7, <br />                                                                                 SSU72, SSU73 strain YZS19 scoring resulting <br />                                                                                 transformants growth 16 378C. results shown <br />                                                                                 Fig. 1. Strain YZS19/pM586 (sua7 SSU72 ssu73) cs1 <br />                                                                                 ts1 (row 4), confirming ssu73-1 suppresses                                                                                 sua7-1 cs2 phenotype does confer ts2. contrast, <br />                                                                                 YZS19/pM681 (sua7 ssu72 SSU73) exhibits cs2 phe- <br />                                                                                 notype parent strain YMH71-9C (sua7 SSU72 SSU73) <br />                                                                                 (cf. rows 1 5), confirming ssu72-1 suppressor <br />                                                                                 sua7-1.  ts2 phenotype YZS19 retained                                                                                 presence SSU73 gene (cf. rows 2 5)                                                                                 eliminated SSU72 SUA7 (rows 3 4), confirming <br />                                                                                 dependence ts2 sua7-1 ssu72-1. results <br />                                                                                 firmly establish ssu72-1 enhancer ssu73-1                                                                                 suppressor TFIIB defect. <br />    FIG. 2. Restriction map SSU72 region. partial restriction map       SSU72 sequence. entire 3.8-kb HindIII fragment <br /> compassing SSU72 gene shown. DNA fragments complement ts2       encompassing SSU72 sequenced. open reading <br /> phenotype associated ssu72-1 suppressor sua7-1 background indicated black bars, open bars denote failure complement.                                                                                 frames 618, 1,482, 894 nucleotides identified (Fig. <br /> 3.8-kb HindIII-HindIII 1.05-kb EcoRI-HindIII DNA fragments vector    2). SSU72 gene subsequently delimited 1.05-kb <br /> pRS316 (62) designated pM471 pM385, respectively. adjacent open     EcoRI-HindIII DNA fragment (pM385) includes reading frames encode potential proteins 494 298 amino acids (aa)   618-nucleotide open reading frame (Fig. 3). Accordingly, <br /> unknown functions. <br />                                                                                 SSU72 encodes potential protein 206 amino acids                                                                                 calculated molecular mass 23,469 Da. Ssu72p acidic (pI 5 <br />                                                                                 4.9) enriched charged residues (27% D, E, K, R). ts2 initially delimited common 3.8-kb HindIII                     N terminus includes half-site C-X2-C potential metal <br /> DNA fragment (plasmid pM471) (Fig. 2). Southern blot anal-                      binding motif  . imperfect direct repeats ysis confirmed fragment derived intact                   identified (residues 69 78 82 90), suggesting yeast genome SSU72 single-copy gene (data                     internal element symmetry Ssu72p. structural <br /> shown). relationship complementing DNA                          features summarized Fig. 4. structural features com- genetically defined ssu72 gene established                    monly associated transcription factors  al- <br /> tegration cloned DNA LEU2 vector                   clustering acidic residues C-terminal <br /> genome YDW546. strain (sua7 leu2 SSU72::LEU2)                       half protein. <br /> crossed strain YZS19-1B (sua7 leu2 ssu72), resulting                    Ssu72p sequence compared protein se- <br /> diploid sporulated dissected. total 18 segregants                  quence databases using BLAST algorithm (2). obtained scored. Leu1 progeny phenotyp-                       homologous proteins  similarity conserved <br /> ically ts1, Leu2 progeny ts2. Segregation                   region ATP-dependent RNA helicases identified. LEU2-tagged locus opposite ssu72-1 marker (ts2) estab-                   similarity Ssu72p consensus residues lished cloned gene SSU72.                                    region especially good (Fig. 4). Nonetheless,    Genetic relationship SUA7 SSU72. Indepen-                        Ssu72p sequence includes DEAD box ATP- <br /> dent segregation cs1 (ssu73) ts2 (ssu72) phenotypes                  binding proteins highly conserved HRIGRXXR motif YZS19 raises question SSU72 function-                    required RNA binding ATP hydrolysis (49). Conse- <br /> ally related SUA7. described  SSU72 cloned                   quently, Ssu72p unlikely ATP-dependent RNA <br /> complementation ts2 phenotype. addition, plasmid                      helicase. <br /> DNA 9 12 ts1 transformants YZS19 exhibited                            SSU72 essential gene. SSU72 locus disrupted <br /> HindIII restriction pattern characteristic SUA7                  replacing HindIII-BclI fragment encompassing  <br /> gene. Plasmid pDW11, carries SUA7 low-copy-                          tire SSU72 open reading frame LEU2 HindIII-BamHI <br /> number plasmid (50), subsequently transform                        DNA fragment (Fig. 2). resulting construct used YZS19 ts1. ability SUA7 SSU72                       disrupt copy SSU72 homozygous leu2 diploid <br /> wild-type gene restore ts1 demonstrates ts2 phe-                    strain LRB535/536. sporulation dissection, notype associated ssu72-1 dependent sua7-1.                        viable spores recovered tetrad (Fig.  ts2 phenotype YZS19 simply conse-                    5) viable spores phenotypically Leu2. Inspection <br /> quence ssu72-1 mutation &#226;&#8364;&#732;&#226;&#8364;&#732;synthetic&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype                 inviable segregants microscope revealed requiring sua7-1 ssu72-1. effect reminiscent                germinated, undergoing cell divisions ssu71-1 suppressor, similarly dependent                   cessation growth. spore viability restored sua7-1 mutation associated ts2 phenotype (64).                   diploid strain transformed plasmid pM385 (SSU72 <br /> SSU71, extremely important result dem-                    URA3) prior dissection. resulting segregants onstrates genetic relationship TFIIB prod-                    replica printed 5-FOA medium, selecting plas- <br /> uct SSU72 gene.  strain YZS19 contains                       mid-borne copy SSU72, segregants, suppressor (ssu73-1) enhancer (ssu72-1)                      Leu2, grew (data shown).  SSU72 essen- <br /> mutation (sua7-1). mutations single                   tial cell growth. <br /> strain consequence selecting phenotype (cs1)                       ssu72-1 mutation affects transcription start site selec- screening (ts2).                                              tion. sua7-1 mutation shifts transcription start site se- <br />  VOL. 16, 1996                                                                                                        YEAST SSU72 GENE                     1561 <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 4. Structural features Ssu72p. essential N terminus (residues 5                                                                                    23) denoted shaded box includes sequence C-X2-C,                                                                                    duplicated ssu72-1 mutant. imperfect repeats (Fig. 3) denoted                                                                                    arrows. black box (residues 122 150) denotes sequence similarity known <br />                                                                                    putative ATP-dependent RNA helicases. region aligned eIF-4A <br />                                                                                    mice; Tif1 Tif2, putative counterparts eIF-4A yeasts; <br />                                                                                    ME31B, maternally expressed Drosophila gene; Dhh1p, Ste13p, Drs1p, <br />                                                                                    putative helicases S. cerevisiae S. pombe; RHLB, putative helicase <br />                                                                                    E. coli. Sc, Mm, Dm, Sp, Ec denote S. cerevisiae, Mus musculus, <br />                                                                                    Drosophila melanogaster, S. pombe, E. coli, respectively. Residues                                                                                    identical Ssu72p highlighted boldface type, conserved residues                                                                                    highlighted italics. Arrowheads denote residues identical                                                                                    eukaryotic proteins (excludes E. coli RHLB). <br />  <br />  <br />  <br />  <br />                                                                                    initiation ADH1 (Fig. 6, lane 4) pattern <br />                                                                                    associated sua7-1 defect (Fig. 6, lane 1). strain <br />                                                                                    YZS19 mutated ssu73 locus, normal initiation <br />    FIG. 3. SSU72 sequence. complete nucleotide sequence             pattern SSU72 transformant suggests ssu73-1 com- <br /> 1,058-bp EcoRI-HindIII DNA fragment encompassing SSU72 shown,             pensates downstream shift caused sua7-1, result deduced <span id='am-42' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-48' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence Ssu72p. EcoRI site (2233)           observed previously ssu71-1 suppressor (64). <br /> located 6 nucleotides downstream termination codon ORF494 (Fig. 2).      Furthermore, ssu71-1 ssu73-1, ssu72-1, sup- 6-bp direct repeats flank duplicated sequence ssu72-1 mutant <br /> (nucleotides 25 54), corresponding amino acid sequence duplicated   press sua7-1 cs2 phenotype.  ssu71 ssu73 <br /> (residues 9 18), region sequence similarity ATP-dependent RNA     suppressors appear functionally related helicases (residues 122 150) underlined. imperfect amino acid repeats   distinct ssu72-1 enhancer. <br /> (residues 69 78 82 90) indicated arrows, identical residues      Characterization ssu72-1 mutation. ssu72-1 allele indicated boldface type. <br />                                                                                    cloned gap repair (57), entire open reading <br />                                                                                    frame sequenced. 30-bp duplication nucleotides 25                                                                                    54 identified mutation (Fig. 7). ssu72-1 <br /> lection downstream normal sites (50), asked                      duplication maintains open reading frame creates ssu72-1 affect process. Transcription start sites                  10-amino-acid direct repeat  partially overlapping 12-ami- mapped ADH1 locus primer extension (Fig. 6).                        acid repeat) (Fig. 8). Interestingly, repeat includes Transcription ADH1 gene normally initiates sites                     amino acid sequence C-T-V-C Cys4 zinc binding <br /> located positions 227 237 relative ATG start                         motif (C-X2-C-X6-C-X2-C) created near N terminus codon (Fig. 6, lane 2). Consistent previous results (50), sua7-1 mutation diminished initiation position 237 enhanced initiation multiple sites located downstream 227 (Fig. 6, lane 1). Strikingly, strain YZS19, initiation nearly eliminated 237 reduced 227 dramat- <br /> ically enhanced multiple sites located downstream <br /> (Fig. 6, lane 3). effect dependent sua7-1 ssu72-1 mutations, normal initiation pattern restored introducing low-copy-number plasmid carrying SSU72 (Fig. 6, lane 4) SUA7 (lane 5) wild-type <br /> gene YZS19. similar enhanced downstream shift observed CYC1 (data shown), establishing ssu72 <br />                                                                                       FIG. 5. SSU72 essential gene. copy SSU72 gene dis- <br /> effect start site selection specific ADH1 pro-                   rupted LEU2 marker diploid strain LRB535/536 (ssu72::LEU2/ <br /> moter. results demonstrate Ssu72p involved                        SSU72 leu2/leu2). sporulation dissection, viable <br /> start site selection TFIIB-dependent manner provide                       spores recovered tetrad (9 25 tetrads depicted) evidence functional relationship                       viable spores phenotypically Leu2. spore viability restored                                                                                    dissections presence plasmid-borne SSU72 gene (data proteins.                                                                      shown). resulting segregants replica printed 5-FOA <br />    plasmid-borne SSU72 gene strain YZS19/pM385                              medium, selecting plasmid-borne SSU72, segregants, (sua7-1 ssu72-1/SSU72 ssu73-1) restored normal pattern                      Leu2, grew. results establish SSU72 essential gene. <br />  1562        SUN HAMPSEY                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />                                                                                         strain carries SUA7 wild-type gene, allowing                                                                                         phenotypes attributed exclusively ssu72 mutations.                                                                                         results shown Table 2. Deletion N-terminal 4 <br />                                                                                         residues effect, deletion 10 23 N-terminal <br />                                                                                         residues abolished Ssu72p function.  N terminus <br />                                                                                         Ssu72p, includes duplicated sequence encoded                                                                                         ssu72-1, critical function. <br />                                                                                            asked cysteine position 15 critical.                                                                                         mutations, encoding serine replacement (C15S)                                                                                         encoding histidine replacement (C15H), created. <br />                                                                                         C15S mutant inviable, C15H mutant grew <br />                                                                                         nearly wild-type strain. inviability                                                                                         C15S mutant establishes amino acid 15 critical,                                                                                         viability C15H mutant suggests C-X2-C se- <br />                                                                                         quence involved zinc chelation. generated <br />                                                                                         comparable replacements cysteine position 25  <br />                                                                                         coded ssu72-1 (Fig. 8). Despite retention normal <br />                                                                                         C-15, C25S mutant inviable, C25H  <br />                                                                                         placement effect. data establish cys- <br />                                                                                         teine residues located near N terminus Ssu72p                                                                                         ssu72-1-encoded protein essential function                                                                                         suggest residues involved zinc chelation. <br />  <br />  <br />  <br />  <br />    FIG. 6. Primer extension analysis ADH1 transcription start sites. Lane 1, <br /> YMH71-9C (sua7-1); lane 2, T16 (wild type); lane 3, YZS19 (sua7-1 ssu72-1 <br /> ssu73-1); lane 4, YZS19/pM385 (sua7-1 SSU72 ssu73-1); lane 5, YZS19/pDW11 <br /> (SUA7 ssu72-1 ssu73-1). Lanes  C, G, T used molecular size <br /> markers correspond sequence ladder SUA7 DNA. Transcription <br /> initiates comparable rates positions 237 227  ATG start <br /> codon designated 11) wild-type strain (lane 2). sua7-1 mutation <br /> causes downstream shift start site selection (lane 1), effect exacerbated sua7-1 ssu72-1 ssu73-1 mutant, normal initiation <br /> sites nearly eliminated favor multiple, discrete downstream sites (lane <br /> 3). normal initiation pattern restored presence SSU72 <br /> (lane 4) SUA7 (lane 5) gene, demonstrating downstream shift YZS19 dependent sua7-1 ssu72-1 mutations. normal <br /> initiation pattern observed sua7-1 SSU72 ssu73-1 strain, indicating ssu73-1 compensates downstream shift associated sua7-1 (cf. lanes 1 4). effect similar ssu71-1 suppressor (64) stark <br /> contrast effect ssu72-1. <br />  <br />  <br />  altered form Ssu72p. intriguing result, motif implicated protein-DNA protein- <br /> protein interactions (53).  factors shown <br /> previously affect start site selection S. cerevisiae (TFIIB Rpb1p Rpb9p subunits RNAP II) contain <br /> C-X2-C-X6&#226;&#8364;&#8220;17-C-X2-C/H zinc binding motifs.                                                 FIG. 7. DNA sequence ssu72-1 allele. Partial DNA sequences                                                                                         SSU72 ssu72-1 alleles shown, cases CGCAAT (base pairs <br />   investigated importance Ssu72p N terminus                            13 18) AACAAT (base pairs 55 60) (Fig. 3). brackets encompass generating specific ssu72 mutations vitro assessing                        region SSU72 duplicated create ssu72-1 allele includes phenotypes using plasmid-shuffle vivo. host                          TCAAAC sequence flanks reiterated sequence. <br />  VOL. 16, 1996                                                                                                                              YEAST SSU72 GENE               1563 <br />  <br />  <br />  <br />  <br />    FIG. 8. ssu72-1 allele encodes direct repeat near N terminus Ssu72p. ssu72-1 allele result 30-bp tandem duplication (Fig. 7). 6-bp <br /> direct repeats flank duplication boxed. 10-amino-acid direct repeats resulting ssu72-1 duplication indicated arrows. cysteine <br /> residues repeated sequence form zinc binding motif (C-X2-C-X6-C-X2-C). <br />  <br />  <br />  <br />                                         DISCUSSION                                                            basis ts2 phenotype. suggests                                                                                                               SSU72 unrelated SUA7, results indicate oth- <br />    Functional relationship Ssu72p TFIIB. SSU71,                                                     erwise.  ssu72-1 ts2 phenotype requires presence <br /> SSU72, SSU73 genes uncovered genetic selection                                                  sua7-1 allele; SUA7 ssu72-1 sua7-1 SSU72 suppressors conditional growth defect (cs2) associated                                               single mutants ts2. effect defined synthetic  TFIIB mutation (sua7-1) alters transcription                                                 hancement indicates genetic relationship start site selection. potential selection identify                                             genes (28). Second, transcription start site selection genes products functionally related TFIIB                                                    shifted significantly downstream normal sites established identity SSU71 gene. SSU71 encodes                                                  YZS19 mutant, effect rescued SUA7 homolog mammalian RAP74 identical TFG1,                                                    SSU72 (Fig. 6, cf. lane 3 lanes 4 5). results <br /> gene encoding largest subunit yeast TFIIF (64).                                                    demonstrate Ssu72p affect transcription initiation ssu71-1 suppressor compensated transcription ini-                                                TFIIB-dependent manner vivo provide strong support <br /> tiation defect associated sua7-1, restoring normal pat- <br />                                                                                                               functional relationship proteins.  <br /> tern start site selection CYC1 gene. SSU73 gene <br />                                                                                                               theless, unable detect direct interaction identical RPB9, encodes 14.2-kDa subunit                                                                                                               proteins hybrid RNAP II (66). Like ssu71-1, ssu73-1 compensates cs2 <br />                                                                                                               association TFIIB GST-Ssu72p HA-Ssu72p <br /> growth defect sua7-1 (Fig. 1) restores normal pat- <br /> tern transcription start site selection ADH1 (Fig. 6, lane                                              fusion proteins (63).  know 4). results imply functional relationship                                                 functional relationship TFIIB Ssu72p involves largest subunit TFIIF Rpb9p subunit RNAP II.                                                    physical interaction. <br />    ssu72-1 allele, contrast ssu71 ssu73,                                                  primary structure Ssu72p does offer clue suppressor sua7-1 cs2 phenotype.  ssu72-1 arose                                                 function. significant sequence similarity spontaneously ssu73-1 strain YZS19 identified                                                 known putative ATP-dependent RNA helicases.                                                                                                               similarity appears significant (Fig. 4), limited                                                                                                               small region protein. absence motifs          TABLE 2. Viability associated Ssu72p alterations                                                characteristic ATP-dependent RNA helicases implies <br />                                                                                                               Ssu72p RNA helicase.  sequence <br /> Ssu72p alteration                                                                             Viabilitya <br />                                                                                                               similarity suggests Ssu72p ATP-dependent RNA  <br />                  b <br />  N-term-D4 ...................................................................................Viable          licases share common, albeit undefined, function. <br />  N-term-D10b .................................................................................Inviable           ssu72-1 mutation. ssu72-1 allele result  N-term-D23b .................................................................................Inviable        30-bp duplication flanked 6-bp direct repeats (Fig. 8).  <br />  C15Sc ............................................................................................Inviable   sequently, ssu72-1 encodes 10-amino-acid repeat creates <br />  C15Hc ...........................................................................................Viable <br />  C25Sd ............................................................................................Inviable   C-X2-C-X6-C-X2-C zinc binding motif (Cys4) near N <br />  C25Hd ...........................................................................................Viable      terminus Ssu72p (Fig. 8). result particularly intrigu- <br />                                                                                                                 ing. largest subunit RNAP II contains similar zinc <br />     Strains expressing indicated Ssu72p alterations constructed plas-                             binding motif (C-X2-C-X6-C-X2-H) (1). structure lo- <br /> mid shuffling described Materials Methods. Viability assessed ability Trp1 transformants YZS89A grow 5-FOA medium. FOAr                                     cated N terminal amino acid replacements encoded colonies purified subcloning subsequently restreaked rich                                   sua8 alleles RPB1, exert effects start site se- <br /> dium (YPD) compare growth rates control strains expressing                                   lection similar sua7-1 (4). TFIIB includes zinc <br /> SSU721. Growth rates viable mutants wild-type control strains comparable. <br />                                                                                                               binding motif (C-X2-C-X17-C-X2-C) near N terminus,   b <br />     indicated alterations denote, respectively, deletion N-terminal 4,                             sua7-1, sua7-2, sua7-3 alleles, shift tran- <br /> 10, 23 residues Ssu72p. proteins expressed plasmids                                    scription start site selection downstream normal sites, <br /> pM740 (D4) pM742 (D10) GST-Ssu72 fusion proteins plasmid                                        encode single amino acid replacements highly conserved <br /> pM524 (D23) HA-Ssu72 fusion protein. Cells expressing length GST- <br /> Ssu72 (pM699) HA-Ssu72 (pM550) fusion proteins viable, demonstrat- <br />                                                                                                               domain immediately adjacent motif (51). Rpb9p <br /> ing fusion proteins functional.                                                                  subunit RNAP II includes zinc binding motifs, near <br />   c <br />     Amino acid replacements denoted single letter code. Accordingly,                               N terminus near C terminus (74). Like <br /> C15S C15H designate, respectively, cysteine3serine cysteine3histidine                                 sua7 (TFIIB), sua8 (Rpb1p), ssu72, alleles RPB9 replacements position 15. <br />   d <br />     Amino acid replacements generated using ssu72-1 allele                                                                                                               affect start site selection. cases, start site selection template.  indicated replacements C-25 occur 10-                                  shifted upstream normal site attributed residue duplication encoded ssu72-1 (Fig. 8).                                                              rpb9 null allele replacement cysteine residue  1564     SUN HAMPSEY                                                                                                MOL. CELL. BIOL. <br />  <br />  N-terminal zinc binding motif (20, 32) nonsense            addition, effects sua7 mutations start site selection <br /> mutation immediately following C-terminal zinc binding             CYC1 ADH1 genes essentially identical motif (66). results point role zinc binding      sua8 mutations, genetic evidence suggests TFIIB <br /> domains transcription start site selection, spe-       interacts directly largest subunit RNAP II (4). <br /> cific functions zinc binding regions remain undefined.          Clearly, RNAP II TFIIB critical determinants start <br />    C-X2-C-X(6&#226;&#8364;&#8220;25)-C-X2-C motif             site selection, presumably consequence specific inter- <br /> transcription factors, including second largest subunit         actions Rpb1p, Rpb9p, TFIIB. <br /> RNAP II (67), TFIIE (16), elongation factor TFIIS (53),               role does Ssu72p play start site selection? adenovirus E1A protein (22). dimensional nuclear           important recognize data indicate magnetic resonance structure revealed zinc binding            SSU72 necessarily plays direct role start site selection. <br /> domain TFIIS does form classical zinc finger motif.          ssu72 mutation enhances TFIIB defect;  TFIIS forms stranded, antiparallel b-sheet,            ssu72-1 allele does affect start site selection SUA7 <br /> designated zinc ribbon (53). suggested motif        wild-type background (Fig. 6, lane 5). result similar inaccessible (cryptic) native protein ex-        effect ssu71 suppressors. Mutations SSU71 restore <br /> posed binding TFIIS elongation complex.          normal pattern start site selection sua7-1  interesting possibility reminiscent proposal          ground (64) little effect SUA71  TFIIB exists latent form N-terminal             ground (63). ssu71, ssu72, ssu73 alleles, domain forms intramolecular interaction C-termi-           ssu73-1 (RPB9) influences start site selection SUA71 <br /> nal domain (54). binding transcriptional activator, <br />                                                                        background (66).  findings consistent zinc binding motif TFIIB N terminus exposed, <br />                                                                        conclusion start site selection determined solely providing surface interaction RNAP II-TFIIF.                                                                        interactions TFIIB RNAP II. <br /> zinc binding motif critical function, cysteine <br />                                                                           possible role Ssu72p assembly transcription <br /> replacements eliminated ability TFIIB recruit RNAP <br /> II-TFIIF preinitiation complex (8).                           preinitiation complex. Ssu72p facilitate assembly    noted ssu72-1-encoded protein,        transcription preinitiation complex. idea based normal Ssu72p, includes C-X2-C-X6-C-X2-C zinc bind-            following rationale. Firstly, E62K form TFIIB encoded <br /> ing motif. Nonetheless, cysteine 15 essential N terminus        sua7-1 impair assembly TFIIB preini- Ssu72p functionally replaced histidine         tiation complex. supported severe cs2 phenotype <br /> serine, suggesting C-X2-C sequence involved zinc        sua7-1 mutants, phenotype associated defects chelation. precedent possibility. Rpb10p       assembly multisubunit complexes (reviewed reference <br /> subunit RNAP II contains similar half-site C-X2-C se-             73), observation TFIIB encoded sua7-1 quence (75) known bind zinc (69). N terminus          functional stable vivo (51). Consequently, suppres- Ssu72p does bind zinc, amino acids               sors (ssu71 ssu73) enhancers (ssu72) sua7-1 <br /> cysteine constitute zinc ligands. Alternatively,         mutation include factors affect assembly Ssu72p exist dimer creates zinc binding              preinitiation complex. Secondly, ssu71 suppressors domain trans, proposed human immunodeficiency            sua7-1 encode altered forms largest subunit TFIIF <br /> virus Tat protein (19).                                                restore normal pattern transcription initiation    appear discrepancy struc-           sua7-1 mutant (64). Ssu71p subunit TFIIF homolo- <br /> ture ssu72-1-encoded protein function.              gous mammalian RAP74. RAP30 subunit ssu72-1 mutation creates, eliminates, structural         TFIIF sufficient initiate transcription (9) recruit <br /> motif (Cys4) altered protein         RNAP II preinitiation complex (17, 34, 70), RAP74 <br /> expected dominant, gain function mutation.              increase efficiency complex assembly (34, 70). <br /> ssu72-1 exacerbates TFIIB defect, causing start site selec-        results suggest RAP74 function regulating <br /> tion shift downstream normal site creating       assembly preinitiation complex (10). Ssu71p synthetic ts2 phenotype. Furthermore, ssu72-1 fully reces-        homologous RAP74 (64), Ssu71p function sive respect SSU721. anomaly rationalized,         preinitiation complex assembly. Thirdly, Ssu72p includes  10-amino-acid duplication encoded ssu72-1           half-site C-X2-C zinc binding motif critical function <br /> impairs protein stability. case,            duplicated ssu72-1 mutant. Zinc binding domains <br /> SUA71 ssu72-1 mutants mildly ts2, albeit lesser          RNAP II proposed stabilize subunit-sub- <br /> extent sua7-1 ssu72-1 double mutants (63) (Fig. 1, rows 2         unit interactions, facilitating assembly enzyme 3). Cys4 zinc binding motif retain               complex (69; reviewed reference 3). Furthermore, zinc <br /> facilitate protein-protein interactions 10-res-        binding motif TFIIB necessary recruit RNAP II idue duplication destabilizes protein. explain          preinitiation complex (8), E62K defect (sua7-1) cysteine 25, component normal            located just downstream motif (51). Finally, protein duplication encoded ssu72-1,          effector sua7-1 uncovered studies, ssu73-1, encodes functionally critical (Table 2).                                       altered form Rpb9p. RNAP II subunit includes    Start site selection yeasts. yeast vitro transcription   zinc binding motifs proposed affect   replacement RNAP II TFIIB S.                  sembly RNAP II complex (3). Taken  cerevisiae functional counterparts Schizosac-          results suggest accurate start site selection involves spe- <br /> charomyces pombe sufficient change initiation pattern        cific interactions RNAP II TFIIB S. cerevisiae pattern characteristic S.         interactions mediated,  zinc binding <br /> pombe (38). basis result, Li et al. (38) concluded      domains. half-site zinc binding domain Ssu72p RNAP II TFIIB sole determinants tran-              facilitate interactions, component scription start site selection yeasts. premise consistent   preinitiation complex gone undetected biochemically effects altered forms TFIIB (sua7), Rpb1p               chaperone-like protein facilitates complex assem- <br /> (sua8), Rpb9p (ssu73) start site selection (4, 20, 32, 50).     bly. <br />  VOL. 16, 1996                                                                                                           YEAST SSU72 GENE                       1565 <br />  <br />  <br />                           ACKNOWLEDGMENTS                                                eukaryotic transcription. Curr. Opin. Cell Biol. 6:403&#226;&#8364;&#8220;409. <br />                                                                                      27. Goodrich, J. , R. Tjian. 1994. Transcription factors IIE IIH   grateful David Gross valuable discussions                     ATP hydrolysis direct promoter clearance RNA polymerase II. Cell 77: <br /> course work thoughtful comments manuscript.                       145&#226;&#8364;&#8220;156. thank Rolf Furter RPB9 clone.                                        28. Guarente, L. 1993. Synthetic enhancement gene interaction: genetic tool <br />   work supported grants National Institutes                      come age. Trends Genet. 9:362&#226;&#8364;&#8220;366. <br /> Health (GM-39484) American Cancer Society (NP-842).                          29. Hahn, S., S. Buratowski, P.  Sharp, L. Guarente. 1989. Isolation                                                                                          gene encoding yeast TATA binding protein TFIID: gene identical                                   REFERENCES                                             SPT15 suppressor Ty element insertions. Cell 58:1173&#226;&#8364;&#8220;1181. <br />  1. Allison, L. , M. Moyle, M. Shales, C. J. Ingles. 1985. Extensive           30. Hoffman, C., F. Winston. 1987. minute DNA preparation     homology largest subunits eukaryotic prokaryotic RNA                yeast efficiently releases autonomous plasmids transformation Esche- <br />     polymerases. Cell 42:599&#226;&#8364;&#8220;610.                                                        richia coli. Gene 57:267&#226;&#8364;&#8220;272. <br />  2. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990.        31. Horiuchi, J., N. Silverman, G.  Marcus, L. Guarente. 1995. ADA3,     Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410.                        putative transcriptional adaptor, consists separable domains  <br />  3. Archambault, J., J. D. Friesen. 1993. Genetics eukaryotic RNA                 teracts ADA2 GCN5 trimeric complex. Mol. Cell. Biol. 15: <br />     polymerases  II, III. Microbiol. Rev. 57:703&#226;&#8364;&#8220;724.                              1203&#226;&#8364;&#8220;1209. <br />  4. Berroteran, R. W., D. E. Ware, M. Hampsey. 1994. sua8 suppressors        32. Hull, M. W., K. Mckune, N.  Woychik. 1995. RNA polymerase II <br />     Saccharomyces cerevisiae encode replacements conserved residues                subunit RPB9 required accurate start site selection. Genes Dev. 9: <br />     largest subunit RNA polymerase II affect transcription start       481&#226;&#8364;&#8220;490. <br />     site selection similarly sua7 (TFIIB) mutations. Mol. Cell. Biol. 14:226&#226;&#8364;&#8220;     33. Johnson, P. F., S. L. McKnight. 1989. Eukaryotic transcriptional regu- <br />     237.                                                                                 latory proteins. Annu. Rev. Biochem. 58:799&#226;&#8364;&#8220;839. <br />  5. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. positive selection        34. Killeen, M., B. Coulombe, J. Greenblatt. 1992. Recombinant TBP, <br />     mutants lacking orotidine-59-phosphate decarboxylase activity yeast:              transcription factor IIB, RAP30 sufficient promoter recognition <br />     5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197:345&#226;&#8364;&#8220;346.                       mammalian RNA polymerase II. J. Biol. Chem. 267:9463&#226;&#8364;&#8220;9466. <br />  6. Buratowski, S. 1994. basics basal transcription RNA polymerase II.     35. Kim, Y.-J., S. Bjorklund, Y. Li, M. H. Sayre, R. D. Kornberg. 1994.     Cell 77:1&#226;&#8364;&#8220;3.                                                                         multiprotein mediator transcriptional activation interaction  7. Buratowski, S., S. Hahn, L. Guarente, P.  Sharp. 1989. interme-           C-terminal repeat domain RNA polymerase II. Cell 77:599&#226;&#8364;&#8220;608. <br />     diate complexes transcription initiation RNA polymerase II. Cell 56:       36. Koleske,  J., S. Buratowski, M. Nonet, R.  Young. 1992. novel <br />     549&#226;&#8364;&#8220;561.                                                                             transcription factor reveals functional link RNA polymerase II <br />  8. Buratowski, S., H. Zhou. 1993. Functional domains transcription               CTD TFIID. Cell 69:883&#226;&#8364;&#8220;894. <br />     factor TFIIB. Proc. Natl. Acad. Sci. USA 90:5633&#226;&#8364;&#8220;5637.                           37. Koleske,  J., R.  Young. 1994. RNA polymerase II holoenzyme <br />  9. Chang, C., C. F. Kostrub,  F. Burton. 1993. RAP30/74 (transcription            responsive activators. Nature (London) 368:466&#226;&#8364;&#8220;469. <br />     factor IIF) required promoter escape RNA polymerase II. J. Biol.       38. Li, Y., P. M. Flanagan, H. Tschochner, R. D. Kornberg. 1994. RNA <br />     Chem. 268:20482&#226;&#8364;&#8220;20489.                                                               polymerase II initiation factor interactions transcription start site selec- <br /> 10. Chibazakura, T., S. Kitajima, M. Yonaha, Y. Yasukochi. 1994. Enhance-            tion. Science 263:805&#226;&#8364;&#8220;807. <br />     ment bacterial transcription initiation vitro 74 kDa subunit     39. Liao, S. M., J. H. Zhang, D.  Jeffrey,  J. Koleske, C. M. Thompson, D. M. <br />     human general transcription factor IIF (RAP74). Biochim. Biophys. Acta               Chao, M. Viljoen, H. J. J. Vanvuuren, R.  Young. 1995. kinase-cyclin <br />     1219:592&#226;&#8364;&#8220;600.                                                                        pair RNA polymerase II holoenzyme. Nature (London) 374:193&#226;&#8364;&#8220;196. <br /> 11. Conaway, R. C., J. W. Conaway. 1993. General initiation factors          40. Mandel, M.,  Higa. 1970. Calcium-dependent bacteriophage DNA <br />     RNA polymerase II. Annu. Rev. Biochem. 62:161&#226;&#8364;&#8220;190.                                   infection. J. Mol. Biol. 53:159&#226;&#8364;&#8220;162. <br /> 12. Dower, W. J., J. F. Miller, C. W. Ragsdale. 1988. High efficiency trans-     41. McDonnell, D. P., J. W. Pike, D. J. Drutz, T. R. Butt, B. W. O&#226;&#8364;&#8482;Malley. <br />     formation E. coli high voltage electroporation. Nucleic Acids Res. 16:         1989. Reconstitution <span id='am-1' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="vitamin" datatype="xsd:string"></span>vitamin</span> D-responsive osteocalcin transcription <br />     6127&#226;&#8364;&#8220;6145.                                                                           unit Saccharomyces cerevisiae. Mol. Cell. Biol. 9:3517&#226;&#8364;&#8220;3523. <br /> 13. Eisenmann, D. M., K. M. Arndt, S. L. Ricupero, J. W. Rooney, F.              42. Mengus, G., M.  X. Jacq,  Staub, L. Tora, P. Chambon,  <br />     Winston. 1992. SPT3 interacts TFIID allow normal transcription            Davidson. 1995. Cloning characterization hTAF(II)18, hTAF(II)20 <br />     Saccharomyces cerevisiae. Genes Dev. 6:1319&#226;&#8364;&#8220;1331.                                    hTAF(II)28: subunits human transcription factor TFIID. <br /> 14. Eisenmann, D. M., C. Chapon, S. M. Roberts, C. Dollard, F. Winston.              EMBO J. 14:1520&#226;&#8364;&#8220;1531. <br />     1994. Saccharomyces cerevisiae SPT8 gene encodes acidic protein       43. Mikaelian, ,  Sergeant. 1992. general fast method generate <br />     functionally related SPT3 TATA-binding protein. Genetics              multiple site directed mutations. Nucleic Acids Res. 20:376&#226;&#8364;&#8220;377. <br />     137:647&#226;&#8364;&#8220;657.                                                                     44. Na, J. G., M. Hampsey. 1993. Kluyveromyces gene encoding 15. Eisenmann, D. M., C. Dollard, F. Winston. 1989. SPT15, gene                  general transcription factor IIB: structural analysis expression Sac- <br />     encoding yeast TATA binding factor TFIID, required normal                 charomyces cerevisiae. Nucleic Acids Res. 21:3413&#226;&#8364;&#8220;3417. <br />     transcription initiation vivo. Cell 58:1183&#226;&#8364;&#8220;1191.                             45. Nikolov, D. B., H. Chen, E. D. Halay,   Usheva, K. Hisatake, D. K. Lee, <br /> 16. Feaver, W. J., N. L. Henry, D.  Bushnell, M. H. Sayre, J. H. Brickner, O.          R. G. Roeder, S. K. Burley. 1995. Crystal structure TFIIB-TBP- <br />     Gileadi, R. D. Kornberg. 1994. Yeast TFIIE&#226;&#8364;&#8221;cloning, expression,              TATA-element ternary complex. Nature (London) 377:119&#226;&#8364;&#8220;128. <br />     homology vertebrate proteins. J. Biol. Chem. 269:27549&#226;&#8364;&#8220;27553.                 46. Nonet, M. L., R.  Young. 1989. Intragenic extragenic suppressors <br /> 17. Flores, O., H. Lu, M. Killeen, J. Greenblatt,  F. Burton, D. Reinberg.         mutations heptapeptide repeat domain Saccharomyces cerevisiae <br />     1991. small subunit transcription factor IIF recruits RNA polymerase          RNA polymerase II. Genetics 123:715&#226;&#8364;&#8220;724. <br />     II preinitiation complex. Proc. Natl. Acad. Sci. USA 88:9999&#226;&#8364;&#8220;10003.     47. Ossipow, V., J.-P. Tassan, E.  Nigg, U. Schibler. 1995. mammalian <br /> 18. Foreman, P. K., R. W. Davis. 1994. Cloning vectors synthesis          RNA polymerase II holoenzyme containing components required     epitope-tagged, truncated chimeric proteins Saccharomyces cerevisiae.         promoter-specific transcription initiation. Cell 83:137&#226;&#8364;&#8220;146. <br />     Gene 144:63&#226;&#8364;&#8220;68.                                                                  48. Paranjape, S. M., R. T. Kamakaka, J. T. Kadonaga. 1994. Role 19. Frankel,  D., D. S. Bredt, C. O. Pabo. 1988. Tat protein human            chromatin structure regulation transcription RNA polymerase II. <br />     immunodeficiency virus forms metal-linked dimer. Science 240:70&#226;&#8364;&#8220;73.                Annu. Rev. Biochem. 63:265&#226;&#8364;&#8220;297. <br /> 20. Furter-Graves, E. M., B. D. Hall, R. Furter. 1994. Role small RNA       49. Pause, , N. Methot, N. Sonenberg. 1993. HRIGRXXR region     pol II subunit TATA transcription start site spacing. Nucleic Acids Res.       DEAD box RNA helicase eukaryotic translation initiation factor 4A     22:4932&#226;&#8364;&#8220;4936.                                                                        required RNA binding ATP hydrolysis. Mol. Cell. Biol. 13:6789&#226;&#8364;&#8220; <br /> 21. Gansheroff, L. J., C. Dollard, P. Tan, F. Winston. 1995. Saccharo-           6798. <br />     myces cerevisiae SPT7 gene encodes acidic protein important           50. Pinto, , D. E. Ware, M. Hampsey. 1992. yeast SUA7 gene encodes <br />     transcription vivo. Genetics 139:523&#226;&#8364;&#8220;536.                                         homolog human transcription factor TFIIB required normal <br /> 22. Geisberg, J. V., W. S. Lee,  J. Berk, R. P. Ricciardi. 1994. zinc          start site selection vivo. Cell 68:977&#226;&#8364;&#8220;988. <br />     finger region adenovirus E1A transactivating domain complexes        51. Pinto, , W. H. Wu, J. G. Na, M. Hampsey. 1994. Characterization     TATA box binding protein. Proc. Natl. Acad. Sci. USA 91:2488&#226;&#8364;&#8220;2492.               sua7 mutations defines domain TFIIB involved transcription start site <br /> 23. Gietz, D.,  St. John, R.  Woods, R. H. Schiestl. 1992. Improved              se <br /> </body></html>