<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 3, 971-980, September 1992 <br />  <br />  <br /> Chromosome Loss, Hyperrecombination, Cell <br /> Cycle Arrest Yeast mcml Mutant <br /> Randolph Elble Bik-Kwoon Tye <br /> Section Biochemistry, Molecular Cell Biology, Cornell University, Ithaca, New York 14853 <br /> Submitted April 1, 1992; Accepted June 30, 1992 <br />  <br />  <br />         original mcml-l mutant identified inability propagate minichromosomes <br />         ARS-specific manner, suggesting defective initiation DNA synthesis <br />         ARSs. mutant defective expression mating-type-specific genes.         genetic biochemical studies confirmed Mcml transcription factor mediates <br />         transcriptional regulation number genes, including genes outside mating <br />         type complement, interacting different cofactors. MCM1 essential <br />         gene, previously characterized mcml mutants exhibits significant growth defects. <br />         assess roles Mcml essential growth, constructed         characterized temperature-sensitive conditional mutant mcml, mcml-llOL. mutant <br />         exhibits temperature-dependent cell-cycle arrest, large, elongated bud single, <br />         undivided nucleus DNA content close 2n. addition, shows elevated <br />         levels chromosome loss recombination. spite severity mcml-llOL <br />         mutation, mutant retains ARS-specific pattern minichromosome instability. <br />         phenotypes precisely exhibited mutants MCM genes, <br />         MCM2, MCM3, MCM5/CDC46, shown play interacting roles         early steps DNA replication. <br />  <br /> INTRODUCTION                                                ARSs indirectly, transcription factor reg- <br />                                                             ulates expression replication initiator proteins, Mcml versatile transcription factor regulates     directly, replication initiator protein binds expression diverse genes interacting         ARSs. Recently, role replication initiator multiple cofactors. acts positive regulator    shown direct binding Mcml multiple sites <br /> specific genes using a1 cofactor (Bender           ARSs (Christ, Chang, Tye, unpublished results). <br /> Sprague, 1987; Passmore et al., 1988, 1989). acts     initiation DNA synthesis commits cell negative regulator specific genes binding      enter complete cell cycle, Mcml operatively a2 (Keleher et al., 1988; Passmore et      play key role coordinating cell division. <br /> al., 1989; Elble Tye, 1991). Recently,         Functional dissection Mcml indicates 80 <br /> shown regulate cell-cycle dependent expression       <span id='am-6' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> DNA binding domain (amino acids 18-97) SW15 gene, encodes transcription factor      Mcml sufficient viability, minichromosome HO endonuclease, interacting SFF,        maintenance, general transcriptional activation <br /> SW15 factor (Lydall et al., 1991). involved   (Christ Tye, 1991). Proteins containing homology regulation arginine metabolic genes interacting   80 amino acid domain identified ARGR proteins (Dubois Messenguy, 1991).        diverse organisms including yeast, plants, humans. <br /> Additional genes regulation Mcml            human serum response factor (SRF) <br /> likely identified. appears combi-     plant homeotic genes, AG DEF, partic- <br /> natorial action multiple cofactors, Mcml play      ularly interesting regulatory role cell pivotal role coordinating cell growth yeast.        growth cell division. SRF cell proliferation factor <br />    MCM1 originally identified gene involved     mediates transcription growth-stimulated propagation minichromosomes ARS             genes response growth factors serum (Norman <br /> pendent manner. phenotype suggests Mcml           et al., 1988). AG gene Arabidopsis (Yanofsky et involved initiation DNA replication     al., 1990) DEF gene Antirrhinum (Sommer et <br />  <br /> &#194;&#169; 1992 American Society Cell Biology                                                                     971 <br />  R. Elble B.-K. Tye <br />  <br /> al., 1990) homeotic genes important                   cdc17-2 his7 leu2 trpl ura3), H17C1B1 (MATa cdc17-1 his7 ural). <br /> flower development. Organ development plants                    cdcl 7-2 strain used plasmid stability experiments,                                                                       cdc17-1 allele used ultraviolet light (UV) sensitivity <br /> quires coordination gene expression DNA                    experiment. tub2 allele used tub2-401, gift T. Huffaker <br /> replication achieve cell differentiation cell divi-            (Cornell University). mcm3-1 strain R61 described (Gib- <br /> sion.                                                                 son et al., 1990). rad9 strain 7859-7-4 isogenic wild-type <br />     MCM1 essential gene appears                strain 7859-10-2 gift T. Weinert (University Arizona). <br /> play key role regulation gene expression DNA replication, currently available mcml                 Determination Cell Doubling Times <br /> mutants exhibits significant growth defects. learn                 Doubling times determined plotting optical densities essential role Mcml cell,                 cultures growing logarithmically 30&#194;&#176;C yeast extract, peptone, <br /> constructed conditional mutant truncating                    dextrose (YPD) medium. <br /> MCM1 gene amino acid codon 110  286 <br /> total). mcml -1 OL mutation results extremely              Microscopic Techniques <br /> unstable MCM1 mRNA truncated protein                     Cell staining DNA-specific dye 4,6-diamidino-2-phenylindole <br /> (Elble Tye, 1991). effect mutation                 (DAPI) carried according published procedures (Williamson <br />                                                                       Fennell, 1975). Cells stained Calcofluor (Sigma, St. <br /> transcriptional regulation mating type-specific genes              Louis, MO) described Pringle et al. (1989). Cells photo- dramatic known mcml                      graphed using Zeiss (Thornwood, NY) microscope equipped mutant alleles; affects expression               lOOX objective. <br /> specific genes, repression acti- <br /> vation specific genes (Elble Tye, 1991).             Flow Cytometry <br /> paper, characterize physiological defects              Cells stained propidium iodide according method mcml-IlOL mutant strain depleted Mcml                       Hutter Eipel (1978) analyzed using Ortho Diagnostics <br /> activity. Properties mutant indicate suffers          2151 machine (Coulter Electronics, Hialeah, FL). multiple defects characteristic mutants lesions DNA replication, cell cycle arrest              Chromosome Loss Recombination Assay <br /> fore nuclear division, increased chromosome loss, hy-                 Loss frequencies determined essentially described Gibson <br /> perrecombination, UV sensitivity (Hartwell                    et al. (1990). Single colonies picked plates containing com- <br /> Smith, 1985). fact, properties mcml-1lOL              plete medium minus <span id='am-13' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>, suspended water, spread YPD <br /> mutant remarkably similar exhibited                   plates. plates incubated 30&#194;&#176;C -25 generations                                                                       growth. Colonies picked, resuspended water, spread <br /> mutants interacting MCM genes, MCM2 (Sinha                   plates containing 5-fluoroorotic acid (FOA) YPD determine <br /> et al., 1986) MCM3 (Gibson et al., 1990), MCM5/                   fraction cells URA3 marker lost. <br /> CDC46 (Hennessy et al., 1991). family structur- <br /> ally related MCM genes, like MCMl, required                Mitotic Plasmid Stability Assay <br /> stable maintenance minichromosomes ARS-                      Plasmid stability determined described Christ Tye <br /> specific manner (Yan et al., 1991; Chen et al., press).            (1991). Cells grown 30&#194;&#176;C tub2 strain,                                                                       grown 20&#194;&#176;C. <br />  <br /> MATERIALS METHODS                                                 RESULTS <br /> Strain Constructions                                                  mcml-llOL Mutation Affects Different ARSs <br /> Construction strains C2-286L C2-11OL strains 14D-      Differently <br /> 286L 14D-1 OL described (Elble Tye, 1991).       previously described phenotype diploid strains 14D-286L/UH5 14D-11OL/UH5 constructed crossing 14D-286L 14D-1 lOL strain UH5-MT (MATa            mcml-l allele (Passmore et al., 1988; Christ Tye, <br /> leu2::URA3 mcmlAXho BamHI [MCMI ARSI CEN5 TRP1]) se-            1991). mcml-l allele ARS-specific mini- <br /> lecting Trp- segregants. Strain UH5-MT contains URA3 gene      chromosome-maintenance defect reduced <br /> serted gene replacement LEU2 locus chromosome III.       ability activate specific genes. observations accomplished cleaving plasmid pLURA Hpa EcoRI, transforming, selecting Ura+Leu- transformants.    suggested mcml-11OL allele represents replacement verified Southern analysis. pLURA          severe mutation mcml-l . instance, doubling <br /> structed replacing Cla EcoRV fragment YIp351 containing   time mcml-IIOL cells YPD medium 30&#194;&#176;C -4 <br /> LEU2 Cla Sma fragment YIp5 containing URA3.        h compared 1.5- 2-h doubling time wild- <br /> determine recessiveness mcml-IlOL allele, C2-llOL <br /> strain transformed pMCM1-TRP YIp5-XB286 (Elble     type mcml-l cells. effects mcml-llOL tran- Tye, 1991). pMCM1-TRP constructed inserting 3.3        scriptional activation extreme (Elble kb <span id='am-18' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span>Sph</span> BamHI fragment containing MCM1 pSE358.                  Tye, 1991).  expected mcml-llOL allele                                                                       greater effects minichromosome stability Strains                                                         sought determine mcml-l1OL mutant cdc strains used K72-1 (MATa cdc7-4 ura3), 6433-4C (MATa     maintains ARS-specificity observed mcml-1. cdc4-1 leu2-3), F391 (MATa cdc14 leu2 ura3 ade2), 4837-5-2 (MAToa     end, transformed plasmids containing ARSI  <br /> 972                                                                                                           Molecular Biology Cell <br />                                                                                                                      Yeast mcml Mutant <br />  <br /> ARS120 mcml-llOL isogenic wild-type strains            c measured plasmid loss rates. ARSI represen-         0 <br /> tative subset ARSs, including ARS131       O <br />                                                                 <br />                                                                       0.5 -          T                          ARS <br /> telomeric ARSs unstable mcml-l          Q                                              * 120 ARS-specific mcm mutants (Maine et al.,                  0.4 -                                     *     1984). Similarly, ARS120 represents subset ARSs, <br /> including ARS121, ARS131 C, ARS206,          L.e)   0.3 - <br />                                                                 <br />                                                                 stable mutants. Figure 1 shows  mcml-llOL mutant strain, ARSI plasmid                     0.2- <br />                                                                Cu <br />                                                                 <br /> dramatically stable ARS120 plasmid, ap-          proaching theoretical maximum 50% loss cell                0.1 - <br />                                                                Cl) <br /> division. specificity mcml defect ARSI       0 <br />                                                                -j     0.0 - absolute. ARS120 plasmid                             MCM1 mcml          cdc17   cdc4       cdc7   cdc14   tub2 affected mcml-11OL mutation                                       mcml <br /> ARSI plasmid, nonetheless 10 times stable                                           St ra n mutant wild-type strain. defects         Figure 1. Mitotic stability minichromosomes containing different ARS stability partially complemented plasmid       ARSs wild-type mcml-llOL mutant cells. minichromosomes <br /> containing wild-type MCMI gene, centromere,          used YCpl20, containing ARSI20, CEN5, LEU2, URA3; TRP1 marker (MCM1/mcml; Figure 1). Incomplete                pCD4, containing ARS1, CEN5, LEU2, URA3. Error bars represent <br />                                                                deviation mean independent assays. theoretical <br /> complementation instability               maximum 50% loss cell division. MCM1/mcml denotes strain <br /> MCM1-containing plasmid. Derivatives 2,u plas-          mcml-lIOL mutation complemented plasmid <br /> mid affected mcml-I1OL mutation.           bearing MCM1 gene, centromere, TRPI marker. Loss rate <br /> plasmid YEp24 loss rate 3% cell division          strain based ratio Trp+Ura+ Trp+ cells. wild-type strain 15% mutant strain, fivefold difference. instability significant 2,u plasmid shares little common natural           cation-defective cdc mutants point chromosomes presence ARS;             specific lesion initiation ARSs. <br /> segregation fidelity apparently based maintenance high copy number association mi-           mcml-11OL Mutation Causes Elevated <br /> crotubules (Som et al., 1988).                                 Frequencies Chromosome Loss Recombination <br />    assess ARS-specificity minichromo-           number mutations arrest nuclear division maintenance general property replication-         yeast enhance chromosome loss cells defective mutants, measured loss rates          grown semipermissive temperature (Hartwell plasmids cdcl 7 strain, defective DNA poly-      Smith, 1985).  mutations interfere <br /> merase (Budd Campbell, 1987)           DNA replication repair lead higher levels mutant strains defects S phase initiation           recombination addition chromosome loss (Hartwell <br /> chromosome segregation. Figure 1 shows cdc17          Smith, 1985). example, mutations DNA <br /> mutant severe mcm defect grown 30&#194;&#176;C,             polymerases (cdc17; Budd Campbell, 1987) semipermissive temperature allele,         (cdc2; Sitney et al., 1989) lead increased levels ARS1- ARS120-containing plasmids equally              chromosome loss recombination, affected. tested mutants       mutation segregation machinery, tub2, affects reported defective maintenance chromosomes         chromosome stability (Huffaker et al., 1988). deter- minichromosomes,  cdc4, cdc7, cdcl4,             truncation allele tub2 (Hartwell Smith, 1985; Huffaker et al., 1988;         properties, constructed wild-type mutant diploid <br /> Palmer et al., 1990). cdc4 cdc7 thought          strains monitored stability MATa-containing <br /> defective late Gl events leading initiation DNA       copy chromosome III. chromosome  <br /> synthesis, cdc14 tub2 defective chro-       tained copy URA3 gene integrated gene- <br /> mosomal segregation (Pringle Hartwell, 1981; Huf-          replacement leu2 locus left arm (Figure <br /> faker et al., 1988). mutants moderate mcm        2B). modification allowed use FOA mon- <br /> defects grown semipermissive temperatures              itor loss chromosome ura3 cells (30&#194;&#176;C cdc4, cdc7, cdc14; 20&#194;&#176;C tub2)            grow FOA-containing media (Boeke et al., 1984). <br /> showed ARS specificity (Figure 1). Loss rates       Ura- cells result recombi- <br /> plasmids mutants ranged 7 13%            nation events centromere leu2::URA3 cell division, compared loss rates 1-2%         resulting loss left arm wild-type controls.  origin-specific plasmid-             entire chromosome. events distinguished <br /> maintenance defects general feature repli-        testing Ura- cells ability mate  <br /> Vol. 3, September 1992                                                                                                               973 <br />  R. Elble B.-K. Tye <br />  <br />                                                                         mcml-IlOL allele causes growth defect; doubling <br />                                LOSS            RECOMBINATION <br />                                                                           time YPD 30&#194;&#176;C twice isogenic wild- <br />                     150.                                                  type strain. Furthermore, mutant cells shifted 370C <br />               U)                                                          arrest growth. determine cells arrested <br />                                                                           particular point cell cycle, cells grown <br />               X     100.                 97                               initially 16&#194;&#176;C, semipermissive temperature                                                                                         mutant, shifted 37&#194;&#176;C various times, <br />                                                                           treated DNA-specific stain (DAPI), examined <br />                   50 -                                                  microscopically. 16&#194;&#176;C, wild-type mutant <br />               w <br />  <br />               LI- <br />                                                                           populations contained unbudded cells normal <br />                                                          23               shape, mutant cells somewhat larger <br />                               2.7                02                       wild type (Figures 3 B, 4).  <br />                                                                           4 h 37&#194;&#176;C, 89% mutant cells large <br />                              MCM1       mcml    MCM1    mcml              budded single nucleus near neck, <br />                                                                           wild-type population contained       B <br />                                                                           unbudded Gi cells (Figures 3, C-F, 4). phe- <br />                                                                           notype indicates mcml truncation mutant                                                                           blocked nuclear division, consistent defects                            leu2::URA3           LEU2::MCMI                DNA replication mitosis. defect recessive. <br />                                                                           Mutant cells wild-type copy MCM1                                                                           integrated URA3 delayed nuclear division <br />                                 MATa             MATs                     indistinguishable wild-type cells cell <br />                                                                           cycle distribution morphology. logarithmic <br />                                                                           culture grown 30&#194;&#176;C, -40% cells unbud- <br /> Figure 2. Frequencies chromosome loss recombination             ded 60% budded, wild-type <br /> wild-type mcml-IIOL mutant strains.   Frequencies loss        control. <br /> chromosome III recombination leu2 CEN3               separate experiment, DNA content determined. bar represents median 5 independent trials        cells measured flow cytometry. Cells wild-type 10 independent trials mutant cells. (B) Schematic <br /> representation MATa- MATa-containing copies chro-           defects DNA replication arrest DNA contents <br /> mosome III. Sites integration URA3 marker wild-         ln 2n, depending S phase mu- <br /> type mutant Mcml alleles shown. CEN3 represented            tant gene product supposed act tightness circles bars.                                              allele; leaky mutants arrest near 2n                                                                           presence checkpoint DNA integrity cells. Mating indicate MATa,                     G2/M boundary (Weinert Hartwell, 1988; Hartwell <br /> right arm chromosome, lost.                       Weinert, 1989; Hennessy et al., 1991). wild- <br /> contrast, cells retain MATa unable mate                   type mutant cells grown semi-permissive lost URA3 recombination event.                             temperature mutant, 16&#194;&#176;C, ln DNA <br />   Wild-type mutant diploid strains grown                      contents, suggesting Gl limiting 30&#194;&#176;C YPD medium -25 generations allow                           cell cycle temperature (Figure 5). DNA peaks <br /> chromosome loss. cells plated YPD                        mutant cells shifted slightly right, FOA-containing medium determine percentage                      possibly accumulation mitochondrial DNA. cells lost URA3 marker, Ura- cells                    Mutant cells shifted 37&#194;&#176;C displayed 2n peak  screened ability mate. frequencies                  large component 2n material dis- chromosome loss recombination                       crete ln peak, suggesting presence partially rep- <br /> higher mcml mutant strain wild-type                    licated DNA DNA degradation. <br /> control (Figure 2A). Chromosome loss -40 times                           Examination cells growing logarithmically 30&#194;&#176;C frequent recombination -90 times fre-                       revealed mcml-1lOL cells develop multiple, <br /> quent. Elevated chromosome loss restricted                      elongated buds (Figure 6, B D). (Cells bearing chromosome III; OFAGE analysis clone cells                   mcml-l allele exhibit large bud morphology lost chromosome III revealed chro-                     37&#194;&#176;C arrest growth develop additional <br /> mosomes lost. hyper-recombination                       buds.) buds appear nucleated;  <br /> phenotype mcml-llOL consistent defect                      nuclei irregular stretched shapes, DNA replication mutant strain                       usually near neck, appear  <br /> segregation defect.                                                       nections nuclei adjacent buds (Figure 6F). mcml-11OL Arrests Nuclear Division                                     ascertain elongated structures true buds, established previously MCM1 gene es-                       treated cells Calcofluor (Sigma), sential yeast cell growth (Passmore et al., 1988).                stains chitin rings occur neck  <br /> 974                                                                                                       Molecular Biology Cell <br />                                                                                                                         Yeast mcml Mutant <br />  <br />  <br />  <br />  <br /> Figure 3. Photomicrographs wild-type   C, E) mcml-llOL mutant (B, D, F) cells. Cells grown 16&#194;&#176;C  B), shifted 37&#194;&#176;C 4 h (C-F), stained DAPI, photographed. Nomarski optics  D) reveals cell morphology, DAPI fluorescence (E F) reveals DNA content. Bar, 10 ,um. <br />  <br />  <br />  <br />  <br /> Vol. 3, September 1992                                                                                                                975 <br />   R. Elble B.-K. Tye <br />  <br />                                                                             bud emerged opposite end <br />                                                                             mother cell. irregular shape cells                                                                             reminiscent shmoo cells arrested Gl mating <br />                                                                             factor mutations CDC28 Gl cyclin genes <br />                                     ,Q       (7)        (2)         &#194;&#163;       (Reed, 1980; Richardson et al., 1989). considered <br />                           0i)       ()      (7         (5          (:)      possibility morphology related                                                                             role Mcml mating type regulator.                   MCMl    75        19         5        0           1       conceivable, e.g., derepression specific  <br /> 160C                                                                        ceptor factor cells (Elble Tye, 1991)                   mcml    56         0        32       7           5        lead autocrine response endogenous factor <br />                                                                              consequently, partial arrest.                                                                              aberrant morphology observed mcml-llOL mutant <br />                                                                             cells mating types.  mcml-IIOL <br />                   MCM1    56        24        12       1          7         cells arrest G2, Gl, DNA content (Figure 5). <br /> 370C                                                                        morphological aberrations                   mcml     1         0        56       33         10        unknown Mcml function functions disrupted <br /> rigure &apos;. yuantitation ot arrest morpnologles. rampies ot uLAli- <br />       r:..t                                                    ;C n ADT   mutant. <br /> stained cells examined Figure 3 analyzed. 200 cells counted sample. Percentages cells unbudded, <br /> small-budded, large-budded indicated. Large-budded cells             Sensitivity UV mcml Mutant <br /> contained single nucleus cell body, undivided nucleus bud neck, divided nuclei segregated cell body.          Inhibition DNA replication leads accumulation <br /> Cells considered large-budded bud            damaged DNA (Hennessy et al., 1991). elevated <br /> fourths size mother cell.                                        levels recombination observed mutant                                                                             consistent possibility. reasoned DNA <br />                                                                             damage present, mutant cells hy- <br /> mother bud. Figure 6H shows constricted                        persensitive additional DNA damage caused mu- <br /> regions cellular compartments stain heavily,                        tagens ultraviolet light. mcml- <br /> indicating compartment does represent                           1lOL strain moderately sensitive UV, mother cell bud;  chitin rings                     level rad9 deletion mutant (Figure 7; Schiestl <br /> elongated like rest cell body. sites bud                  et al., 1989). UV sensitivity mcml mutant <br /> emergence appear abnormal                         related apparent replication defect cells examined. example, buds Figure                      function controlled Mcml. help dis- <br /> 6H occurs end row bud scars (arrowhead),                     tinguish possibilities, examined <br />  <br />                   160C                             370C, 4hr <br />  <br />  <br />  <br />                                                                                          MCM1 <br />  <br />  <br /> cc <br /> Lu <br /> m  ui <br /> 0 <br />  -1 <br /> ul <br />                            Li <br /> w                                                                                        mcml <br />  <br />  <br />  <br />                                                                                                  Figure 5. DNA content arrested cells. <br />                                                                                                  Wild-type mcml-1 lOL mutant cells                                                                                                  grown YPD medium 16&#194;&#176;C                                                                                                  shifted 37&#194;&#176;C 4 h. Aliquots  <br />                                      RELATIVE DNA CONTENT                                        moved analyzed flow cytometry. <br />  <br /> 976                                                                                                          Molecular Biology Cell <br />                                                                                                                         Yeast mcml Mutant <br />  <br />  <br />  <br />  <br />  E                   F                                        G                        H <br /> Figure 6. Abnormal morphology mcml-1lOL cells growing semipermissive temperatures. Wild-type   C, E, G) mcml-lIOL <br /> mutant (B, D, F, H) cells grown 30&#194;&#176;C YPD medium stained DAPI (E F) DAPI plus Calcofluor (G H). <br />  D) Nomarski optics. arrowheads point chitin rings bud scars. Calcofluor quenches DAPI fluorescence, making difficult observe chitin rings DNA simultaneously, especially mutant. Bar, 10 ,um. <br />  mcm mutant, mcm3-1, known replication-                  mutation greatly increases levels chromosome <br /> defective mutant, cdc17-1, sensitivity UV. mcm3-               loss recombination, severely destabilizing <br /> 1 previously reported hypersensitive               minichromosomes extreme bias certain <br /> cells grown permissive temperature, 23&#194;&#176;C.                   ARSs, represented ARS1 (Maine et al., 1984).  strains moderately                   just phenotypes expected Mcml  <br /> hypersensitive shifted restrictive temper-                volved DNA replication, especially level ature briefly UV exposure (Figure 7). temperature shift little effect survival                  100 wild-type strain relative strain grown <br /> continuously 30&#194;&#176;C (Figure 7). appears hypersensitivity UV implicit mcm phe-                     10- <br />                                                                                         KI                           wild-type, C2-286L <br /> notype. moderate nature defects, compared                                                          . <br />                                                                                                                      wild-type, 7859-10-2 [37] rad3 mutant, e.g., suggests Mcm proteins              *                                       .      wild-type, 7859-10-2 directly involved DNA repair (Haynes                                                                   cdc17, H17C1B1 [37] Kunz, 1981). Inspection mcml mutant cells 1 d                                                                 rad9, 7859-7-4 irradiation revealed UV treatment greatly ex-                   1 .1-                            -0-      mcm3, R61 [37] <br /> acerbates elongated, irregular bud morphology;                                                              mcml-1 IOL, C2-1 IOL <br /> mcml-l cells, normally wild-type mor- <br /> phology 30&#194;&#176;C, display aberrant phenotype                    .01 <br /> irradiation isogenic wild-type cells                                    10       20 observations provide additional evidence                              Joules/m 2 <br /> elongated bud phenotype linked DNA damage                    Figure 7. Ultraviolet-light sensitivity wild-type mutant cells. mcml mutants.                                                     Dilutions logarithmically growing cultures plated YPD <br />                                                                       exposed ultraviolet germicidal lamp various times. <br /> DISCUSSION                                                            Cells grown 30&#194;&#176;C mcm3 cdcl 7 strains  <br />                                                                       wild-type control, shifted 37&#194;&#176;C 1.5 h shown mutation MCM1 gene                        exposure incubated room temperature <br /> results conditional arrest nuclear division.             colonies appeared. <br />  <br /> Vol. 3, September 1992                                                                                                                   977 <br />  R. Elble B.-K. Tye <br />  <br /> initiation. 2n DNA content arrested          different effects. believe cells inconsistent initiation defect.   arrest phenotype results largely inability  Hennessy et al. (1991) demonstrated              mutant cells complete DNA replication. different alleles cdc46, allelic      mutant cells hallmarks replication- <br /> ARS-specific mutant mcm5-1 (Chen, unpublished               defective mutants, including elevated levels chro- <br /> sults), arrest DNA contents ln 2n,             mosome loss recombination, sensitivity ultravi- <br /> pending tightness particular allele.       olet light, appropriate arrest morphology. <br /> workers apparently leaky mutations          principal characteristic mcml mutant genes DNA polymerases lead unex-               ties initiation replication defect ARS <br /> pectedly cell cycle arrests 2n, nearly 2n, DNA      function, phenotype shared mcm mu- <br /> contents (Conrad Newlon, 1983; Hartwell, 1973).            tants, including mcm2 (Sinha et al., 1986), mcm3 (Gibson  DNA content arrest reliable indicator        et al., 1990), mcm5/cdc46 (Chen et al., press) execution point allele. expect tighter    abfl mutants (Rhode et al., 1992). Mcm2, Mcm3, <br /> conditional alleles mcml arrest ln DNA           Mcm5 family structurally functionally <br /> content.                                                       related proteins (Hennessy et al., 1991; Yan et al., 1991; <br />    transcription factors mammalian cells ap-        Chen et al., press). subcellular localization pear involved modulating progress Gl          cell-cycle-regulated nuclear late <br /> S phase coordinating cell growth cell division.         M phase Gl/S transition, cytoplasmic act promoting transcription pro-          phases cell cycle (Yan, unpublished results). <br /> liferation-related genes, e.g., SRF, AP1, E2F (Angel       properties proteins suggest act <br /> et al., 1987; Norman et al., 1988; Mudryj et al., 1990;        narrow window Gl/S transition <br /> Chittenden et al., 1991); E2F shown           beginning S phase virtue nuclear <br /> interact directly cell cycle regulator Cyclin       localization effect initiation DNA replication <br /> (Mudryj et al., 1991). appear play direct roles      ARSs. addition similarities phenotypes, replication initiation, especially viral origins         Mcml tied mcm mutants genetically. <br /> (DePamphilis, 1988). NF1 promotes SV40 replication             mcml-l mcm2-1 double mutant forms synthetic binding adjacent origin (Jones et al., 1987;         lethal (Yan, unpublished results), implying Cheng Kelly, 1989), p53 act negatively         Mcml Mcm2 proteins interact functionally. SV40 origin (Bargonetti et al., 1991).             Does Mcml play direct role activation ARSs <br /> mammalian origin characterized,         direct binding, involved indirectly acti- <br /> DHFR origin, contains potential binding sites      vating expression replication initiation factor? <br /> transcription factors, APi, NFIII (Daily        believe Mcml act levels. et al., 1990). Mutations abolish function        observed potential Mcml binding sites upstream transcription factors available;         genes thought involved DNA replication, MCM3 <br /> methods employed demonstrate            RPAl, centered -180 -250, respec- factors required Gl S transition.      tively, translation start codons. RPA1 encodes example, antibodies Fos Jun proteins           yeast homologue single-stranded-DNA binding shown inhibit DNA synthesis cultured cells          protein required SV40 replication vitro (Heyer et <br /> (Kovary Bravo, 1991). Similarly, interference         al., 1990). possibility Mcml regulate <br /> Myc function antisense oligomers causes cells ar-        RPA1 especially interesting terminal <br /> rest Gl (Heikkila et&apos; al., 1987).                           morphology rpal mutants resembles mcml <br />    contrast, mcm1 mutation described causes        mutants. hand, observed <br /> cells arrest later stage cell cycle, nuclear    Mcml binding sites ARSs ARSI division. knowledge, reported         disruption sites reduces ARS function <br /> instance mutation transcription factor causing       (Chang, unpublished data). difference behavior cell cycle arrest nuclear division. Mutations    ARSI ARS120 mcm mutants ex- <br /> essential yeast transcription factors Rapi (Shore      plained differences quality abundance Nasmyth, 1987) Abfl (Rhode et al., 1992)             Mcml binding sites ARSs. cause arrest specific point cell cycle.        inspection vitro binding experiments  case Abf 1, ARS-           ARS120 contains Mcml binding sites, binding protein, prior synchronization abfl mutant          ARSI contains fewer sites, lower affinity (Chang cells factor arrest cells S phase non-     Christ, unpublished data). hypomorphic allele permissive temperature (Rhode et al., 1992), suggesting        mcml-1 1OL, ARSs ARSI likely essential function performed Abf 1     bind Mcml likely initiate replica- G1/S transition DNA replication. Like Abf 1,        tion. Residual initiation events ARSs ARS120 fact Mcml transcription factor means        result incomplete replication yeast arrest phenotype represent composite              genome. <br />  <br /> 978                                                                                              Molecular Biology Cell <br />                                                                                                                                  Yeast mcml Mutant <br />  <br /> ACKNOWLEDGMENTS                                                            Haynes, R. H., Kunz, B.  (1981). DNA repair mutagenesis <br />                                                                            yeast.  Molecular Biology Yeast Saccharomyces: Life thank Tim Huffaker advice, use microscope, critical      Cycle Inheritance, ed. J. N. Strathem, E. W. Jones, J. R. Broach. <br /> reading manuscript. work supported NIH Grant            Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 371-414. <br /> GM34190. <br />                                                                            Heikkila, R., Schwab, G., Wickstrom, E., Loke, S. L., Pluznik, D., <br />                                                                            Watt, R., Neckers, L. (1987). c-myc antisense oligodeoxynu- <br /> REFERENCES                                                                 cleotide inhibits entry S phase progress Gl. <br /> Angel, P., Imagawa, M., Chui, R., Stein, B., Imbra, R., Rahmsorf, H.,      Nature 328, 445-449. <br /> Jonat, C., Herrlich, P., Karin, M. (1987). Phorbolester-inducible      Hennessy, K. M., Lee, , Chen, E., Botstein, D. (1991). group <br /> genes contain common cis element recognized TPA-modulated           interacting yeast DNA replication genes. Genes Dev. 5, 958-969. <br />  trans-acting factor. Cell 49, 729-739. <br />                                                                            Heyer, W., Rao, M., Erdile, L., Kelly, T., Kolodner, R. (1990).  Bargonetti, J., Friedman, P., Kern, S., Vogelstein, B., Prives, C.    essential Saccharomyces cerevisiae single-stranded DNA binding protein <br /> (1991). Wild-type mutant p53 immunopurified proteins bind          homologous large subunit human RP  EMBO J. 9, 2321- sequences adjacent SV40 origin replication. Cell 65, 1083-    2329. <br />  1091. <br />                                                                            Huffaker, T. C., Thomas, J. H., Botstein, D. (1988). Diverse effects <br /> Bender, , Sprague, G. F. (1987). MATal protein, yeast tran-        f,-tubulin mutations microtubule formation function. J. Cell <br /> scription activator, binds synergistically second protein      Biol. 106, 1997-2010. <br /> set ctll-type-specific genes. Cell 50, 681-691. <br />                                                                            Hutter, K. J., Eipel, H. E. (1978). Flow cytometric determinations <br /> Boeke, J. D., LaCroute, F., Fink, G. R. (1984). positive selection   cellular substances algae, bacteria, molds yeasts. Antonie mutants lacking orotidine-5&apos;-phosphate decarboxylase activity       van Leeuwenhoek J. Microbiol. Serol. 44,, 269-282. <br /> yeast: 5-flouro-<span id='am-1' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span>orotic acid</span> resistance. Mol. Gen. Genet. 197, 345-346. <br />                                                                            Jones, K. , Kadonaga, J. T., Rosenfeld, P. J., Kelly, T. J., Tjian, <br /> Budd, M., Campbell, J. L. (1987). Temperature-sensitive mutations      R. (1987). cellular DNA-binding protein activates eukaryotic yeast DNA polymerase gene. Proc. Natl. Acad. Sci. USA 84,         transcription DNA replication. Cell 48, 79-89. <br />  2838-2842. <br />                                                                            Keleher, C., Goutte, C., Johnson,  (1988). yeast cell-type- <br /> Chen, Y., Hennessy, K., Botstein, D., Tye, B.K. CDC46/MCM5,            specific repressor a2 acts cooperatively non-cell-type-specific yeast protein subcellular localization cell-cycle regulated,    protein. Cell 53, 927-936. involved DNA replication ARSs. Proc. Natl. Acad. Sci.  press)                                                                     Kovary, K., Bravo, R. (1991). Jun Fos protein families <br />                                                                            required cell cycle progression fibroblasts. Mol. Cell. <br /> Cheng, L., Kelly, T. J. (1989). Transcriptional activator nuclear      Biol. 11, 4466-4472. <br /> factor stimu <br /> </body></html>