<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Dec. 2009, p. 7456&#226;&#8364;&#8220;7465                                                                             Vol. 191,  24 <br /> 0021-9193/09/$12.00 doi:10.1128/JB.00963-09 <br /> Copyright &#194;&#169; 2009, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />                    SOS-Regulated Type 2 Toxin-Antitoxin System&#228;&#338;&#164; <br />           Larissa  Singletary,1 Janet L. Gibson,1 Elizabeth J. Tanner,1&#226;&#8364;&#160; Gregory J. McKenzie,1&#226;&#8364;&#161; <br />                      Peter L. Lee,1&#226;&#8364;&#160; Caleb Gonzalez,1 Susan M. Rosenberg1,2,3,4* <br />   Departments Molecular Human Genetics,1 Biochemistry Molecular Biology,2 Molecular Virology Microbiology,3 <br />                   Dan L. Duncan Cancer Center,4 Baylor College Medicine, Houston, Texas 77030-3411 <br />                                               Received 22 July 2009/Accepted 1 October 2009 <br />  <br />              Escherichia coli chromosome encodes seven demonstrated type 2 toxin-antitoxin (TA) systems: cassettes <br />           cotranscribed genes, encoding stable toxin protein cause cell stasis death, <br />           encoding labile antitoxin protein, regulatory protein. demonstrate           yafNO genes constitute additional chromosomal type 2 TA upregulated SOS DNA <br />           damage response. yafNOP genes dinB operon, dinB underlies stress-induced <br />           mutagenesis mechanisms. yafN identified putative antitoxin homology known antitoxins, <br />           implicating yafO  yafP) putative toxin. Using phage-mediated cotransduction assays linkage <br />           disruption, yafN essential gene second essential yafO present. <br />            yafP necessary toxin antitoxin. Fourth, DinB required,           yafNOP genes required stress-induced mutagenesis Escherichia coli Lac assay. results <br />           imply yafN encodes antitoxin protects cells yafO-encoded toxin protein-based <br />           TA upregulated SOS response. <br />  <br />  <br />    Toxin-antitoxin (TA) systems modules bacterial ge-                 demonstrated E. coli, including relBE, mazEF, <br /> nomes cause growth arrest programmed cell                    dinJ-yafQ, prlF-yhaV, yefM-yoeB, chpBI-chpBK, hipAB <br /> death cells harboring (1, 18, 19, 54). Type 1 TA                     (1, 3, 19, 20, 39, 45, 50).  relBE mazEF exert <br /> systems consist RNA antitoxin protein toxin,            toxic effects controlled responses RNA antitoxin inhibits translation toxin mRNA.                    <span id='am-26' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> starvation (8, 28). Additionally, mazF kills cells <br /> Type 2 TA systems typically consist genes operon,               response DNA-damaging oxidative stresses (28). <br /> transcriptionally translationally coupled,  usually,            type 2 (dinJ-yafQ) (45) characterized type 1 upstream gene encodes labile antitoxin protein                 (hok, symER, tisAB-istr1) (31, 46, 55) TA systems downstream gene encodes stable toxin protein (Fig. 1A).                    previously reported controlled SOS response Continuous transcription operon, continuous                  DNA damage.  dinJ-yafQ appears SOS <br /> transcription antitoxin, ensures protection ef-              regulated. consensus LexA binding site fects toxin. Interruption transcription operon              identified upstream dinJ-yafQ operon (35) LexA <br /> tips balance favor toxin antitoxin                                                                              bound site study (56) (14), <br /> longer rapidly degraded. <br />                                                                              LexA does regulate operon vivo exposure    TA systems originally identified plasmids &#226;&#8364;&#339;plas- <br />                                                                              mitomycin C (14) UV light (11). role stress  <br /> mid-addiction modules&#226;&#8364;? (34) maintain plasmid                                                                              sponses regulating TA systems interesting problem. <br /> host cell. Failure cells inherit plasmid results rapid <br />                                                                              provide evidence E. coli chromosomal <br /> loss labile antitoxin, unmasking stable preexisting <br /> toxin, death cell process known postsegre-                type 2 TA SOS response control, yafNO. <br /> gational killing (13, 38).                                                      yafNOP originally identified putative TA    recently, TA systems discovered                     homology yafN known antitoxins, implicating yafO <br /> Escherichia coli chromosome, raising question                    yafP) putative toxin(s) (19). yafNOP lies  <br /> function (1, 19). Proposed functions chromosomal                   stream dinB dinB operon (43) (Fig. 1B), TA systems include roles nutritional stress response (8),                 upregulated midway SOS DNA damage response <br /> protection phages (27), formation &#226;&#8364;&#339;persister&#226;&#8364;? cells                  (11). DinB central player stress-induced mutagenesis resist antibiotics (32), selfish genetic elements (38, 54),             mechanisms E. coli (9, 17, 42), Salmonella enterica (48), antiaddiction modules allow bacteria resist plas-                Pseudomonas putida (53), Bacillus subtilis (52). prox- <br /> mid addiction (54). type 2 chromosomal TA systems                    imity yafNOP dinB piqued genes. <br />                                                                              Previously, Brown Shaw demonstrated classical TA sys- <br />                                                                              tem dissection yafO overexpressed growth inhi- <br />   * Corresponding author: Mailing address: Department Molecular           bition occurred (5). additional work regarding yafN Human Genetics, Baylor College Medicine, Baylor Plaza,            yafP conducted (5). contrast notion <br /> Room S809A, Mail Stop BCM225, Houston, TX 77030-3411. Phone:                 yafNOP TA published yafN deletion <br /> (713) 798-6924. Fax: (713) 798-8967. E-mail: smr@bcm.edu. <br />                                                                              strain Keio E. coli knockout collection,   &#226;&#8364;&#160; Present address: Stanford University, Palo Alto, CA 94305. <br />   &#226;&#8364;&#161; Present address: Verenium Corporation, San Diego, CA 92121.              inviable yafN antitoxin (2). Given   &#228;&#338;&#164; <br />     Published ahead print 16 October 2009.                             conflicting results, reinvestigated yafNOP.   <br />                                                                       7456 <br />  VOL. 191, 2009                                                                      SOS-REGULATED TOXIN-ANTITOXIN                                   7457 <br />  <br />                                                                                    1:4 dilution. Phage added multiplicity infection 10 cell, <br />                                                                                    allowed sit room temperature 10 minutes, diluted 10&#226;&#171;&#187; TB, shaken <br />                                                                                    2 hours 32&#194;&#176;C, plated MacConkey-maltose plates seeded                                                                                    108 109 &#226;?&#173;cI 108 109 &#226;?&#173;cIh80 phage kill nonlysogens, red (maltose- <br />                                                                                    positive) colonies picked, purified, verified. &#226;?&#173;cIh80 adsorbs                                                                                    alternate receptor kills nonlysogens mutated maltose uptake <br />                                                                                    receptors, resistant wild-type &#226;?&#173; adsorption. <br />                                                                                       Cloning yafN pBAD24. yafN amplified strain FC36 using <br />                                                                                    primers yafN-F-KpnI (AAAAAAGGTACCATTCTGGTGTGCATTATTATG) <br />                                                                                    yafN-R-HindIII (AAAAAAAAGTTCTTATTCCTTAAAGTCATTG)                                                                                    Platinum Pfx polymerase (Invitrogen) cloned pCR-BluntII-Topo-Kanr <br />                                                                                    using Invitrogen&#226;&#8364;&#8482;s Zero Blunt Topo PCR cloning kit. yafN excised                                                                                    cloning plasmid KpnI XhoI ligated pBAD24 using T4 DNA <br />                                                                                    ligase (Invitrogen), creating plasmid pLS1. Insertion yafN pBAD24 <br />                                                                                    confirmed sequencing (SeqWright, Houston, TX). <br />                                                                                       Quantitative phage P1-mediated cotransduction assays. Phage P1-mediated <br />                                                                                    cotransductions performed based Miller protocol (44)                                                                                    following modifications. Transductants selected LBH-Tet 20 mM CIT <br />                                                                                    patched 100-colony grids LBH-Kan-CIT  LBH-chloram- <br />                                                                                    phenicol-CIT &#226;&#338;&#172;yafN &#226;&#338;&#172;yafP cotransduction) screen recovery markers <br />                                                                                     Plates monitored days incubation possible <br />                                                                                    slowly growing colonies. <br />                                                                                       Sequencing JW0222 yafO. yafO gene strain JW0222 amplified <br />                                                                                    using PCR primers yafN-F-KpnI yafPExtR (AGTTTGTGGAATCAGAAA <br />                                                                                    ACG). PCR product purified Qiagen&#226;&#8364;&#8482;s QIAquick PCR purification kit <br />                                                                                    sequenced using mentioned primers (SeqWright, Houston, TX). <br />                                                                                       Mutagenesis DNA damage sensitivity assays. Stress-induced mutagenesis <br />                                                                                    assays performed according method Harris et al. (25). UV <br />    FIG. 1. Type 2 TA organization dinB operon.                      light kill curves, cultures grown early/mid-log phase (optical density   Type 2 TA organization. Type 2 TA systems usually               600 nm, 0.1 0.3) LBH spread LBH agar plates, exposed gene operons antitoxin (upstream) toxin                      indicated UV light dose, incubated dark 37&#194;&#176;C 16 h, scored stream) genes cotranscribed/translated. antitoxin protein               colonies. bleomycin mitomycin sensitivity assays, saturated LBH cultures <br /> degraded rapidly toxin, cell requires                  diluted grown mid-log phase (optical density 600 nm, 0.3 0.6) <br /> continuous transcription/translation avoid stasis/death induced              spotted fresh drug-containing plates indicated dilutions. Bleomycin toxin. (B) E. coli dinB operon. yafNOP genes, encoding               dissolved 0.9% NaCl. Mitomycin C dissolved sterile <span id='am-16' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O</span> (final pH, <br /> putative TA  lie downstream dinB operon. operon               6.0), manufacturer&#226;&#8364;&#8482;s recommendation. Plates incubated 16 h upregulated SOS DNA damage response (SOS box shown).                     32&#194;&#176;C scored CFU. <br /> yafN, encoding putative antitoxin, 297 bp; yafO, encoding                  Lambda burst size assay UV-induced lysogens. Saturated LBH cultures <br /> putative toxin, 399 bp; yafP, unknown function, 453 bp             E. coli &#226;?&#173; lysogens diluted K-glucose grown 1.5 &#226;&#171;&#187; 108 cells/ml. <br /> length. strains used study (derivatives SMR4562                 aliquot cells removed, lysed lysozyme chloroform, FC40) contain additional copy dinB operon duplication              submitted plaque assay quantify phage spontaneously induced operon F&#226;&#172;&#732; episome. T, toxin;  antitoxin.                          prophage state. Cells UV irradiated (50 J/m2) induce                                                                                    lambda prophage lytic growth shaken &#226;&#177;&#8211;300 rpm 1.5 h 37&#194;&#176;C                                                                                    darkness. Lysozyme (50 mg/ml) chloroform added lyse cells                                                                                    release intracellular phage, debris pelleted, supernatants yafN yafO  yafP) constitute TA                           assayed PFU. Phage titers calculated used determine burst size <br /> explore possible function cell.                                         (PFU/cell, corrected preexisting phage). <br />                                                                                       Flow cytometry cell sorting. Flow cytometry analyses fluorescence- <br />                                                                                    activated cell sorting (FACS) performed method reference 47 <br />                        MATERIALS METHODS                                       following modifications. Log-phase cells analyzed using BD <br />    Media, antibiotics, growth conditions. media used included Luria-       FACSAria cell sorter (BD Biosciences). Data shown means Bertani-Herskowitz (LBH; 1% tryptone, 0.5% NaCl, 0.5% yeast extract, 2 &#226;?&#174;g/ml       experiments  culture/experiment experiments 1 4 cultures/ <br /> <span id='am-21' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span>thymine</span>), tryptone broth (TB; 1% tryptone, 0.5% NaCl), BBL plates (0.5%        experiment experiment 5). 106 nongreen 8 &#226;&#171;&#187; 103 (typically &#226;&#172;&#381;104) <br /> NaCl, 1% BBL Trypticase peptone solidified 1.5% agar). K-glucose consists     green cells sorted culture. Sorting purity controls performed 2&#226;&#171;&#187; M9 (44) following additions: 7.5% Casamino Acids, 5 mM              experiment  method reference 47). Propidium iodide (PI) <br /> MgSO4, 0.05% NaCl, 0.2% glucose, 10 &#226;?&#174;g/ml <span id='am-1' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span>vitamin</span> B1, 3 &#226;?&#174;M FeCl3, 50           staining performed method reference 47. <br /> &#226;?&#174;M CaCl2. Plating culture broth TB following additions: 0.2% mal- <br /> tose, 5 mM MgSO4, 10 &#226;?&#174;g/ml thymine, 10 &#226;?&#174;g/ml vitamin B1. Carbon sources <br /> glucose, maltose, glycerol, <span id='am-6' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lactose</span> used 0.1% unless                                      RESULTS <br /> stated. antibiotics used included tetracycline (Tet; 10 &#226;?&#174;g/ml, 3.33 &#226;?&#174;g/ml sodium citrate [CIT] present), kanamycin (Kan; 30 &#226;?&#174;g/ml), chloram-            yafN essential presence yafO. test phenicol (25 &#226;?&#174;g/ml), bleomycin (various concentrations, specified text),    yafNOP functional TA  examined mitomycin C (various concentrations, specified text), rifampin (100     putative antitoxin gene, yafN, essential gene &#226;?&#174;g/ml). 5-Bromo-4-chloro-3-indolyl-&#226;?&#164;-D-galactoside (X-Gal; 40 &#226;?&#174;g/ml)                                                                                    presence functional putative toxin gene, yafO. used. buffers used included TM (Tris buffer &#226;&#171;&#185; MgSO4, pH 7.5) M9 (44). <br /> Bacteria grown 32&#194;&#176;C 37&#194;&#176;C, stated experiment.                diagrammed Fig. 2A, using cotransduction assay, trans- <br />    Strains new alleles. E. coli K-12 strains plasmids used study   duced E. coli cells phage P1 grown cells listed Table 1. New deletion alleles created study shown    carrying zae-502::Tn10 Tet resistance (Tetr) gene partially <br /> Table 2, PCR primers used construct new deletions short-         linked chromosomal &#226;&#338;&#172;yafN::Kan deletion, conferring <br /> homology recombination methods (12) shown Table 3. Phage P1-mediated <br /> transductions performed described previously (44). construct &#226;?&#173; lyso- <br />                                                                                    Kan resistance (Kanr), mutation  P1 <br /> gens, stationary-phase TB cultures nonlysogens diluted 200&#226;&#171;&#187; plating   donor cells carried yafN&#226;&#171;&#185;-expressing plasmid allow <br /> culture broth, shaken 6 hours 32&#194;&#176;C, pelleted, resuspended TM    viability  essential) prevent accumulation/ <br />  7458        SINGLETARY ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />  <br />  <br />                                       TABLE 1. E. coli strains, plasmids, lambda phage used studya <br />       Strain, phage, <br />                                                      Relevant genotype                                               Reference(s) source <br />        plasmid <br />  <br /> Strains <br />   BT340                    DH5&#226;?&#163;(pCP20)                                                              7  CGSC) <br />   BW25113(pKD46)           pKD46                                                                    12  CGSC) <br />   BW25141(pKD3)            pKD3                                                                     12  CGSC) <br />   BW25141(pKD13)           pKD13                                                                    12  CGSC) <br />   CAG18436                 MG1655 zae-502::Tn10                                                     51 <br />   DH5&#226;?&#163;                     endA1 hsdR17 (rK&#226;&#171;&#186; mK&#226;&#171;&#185;) supE44 thi-1 &#226;?&#173;&#226;&#171;&#186; recA1 gyrA96 relA1                21  W. Bridger &#229;&#8230;&#179;Edmonton, Alberta, Canada&#229;&#8230;&#180;) <br />                               deoR &#226;&#338;&#172;(lacZYA-argF)U169 &#226;?&#190;80dlacZ&#226;&#338;&#172;M15 <br />   FC36                     &#226;&#338;&#172;(lac-proAB)XIII thi ara Rifr                                            6 <br />   FC40                     &#226;&#338;&#172;(lac-proAB)XIII thi ara Rifr &#229;&#8230;&#179;F&#226;&#172;&#732; proAB&#226;&#171;&#185; lacI33&#226;?&#8364;lacZ&#229;&#8230;&#180;                    6 <br />   JW0222                   &#226;&#338;&#172;yafN::FRTKanFRT                                                         2 <br />   MG1655                   Wild type                                                                4 <br />   N2731                    recG258::Tn10 mini-Kan                                                   36  R. Kolodner &#229;&#8230;&#179;San Diego, CA&#229;&#8230;&#180;) <br />   SMR4562                  Genotype FC40, independent construction                  41 <br />   SMR5833                  SMR4562(pKD46)                                                           43 <br />   SMR5889                  SMR4562 &#226;&#338;&#172;dinB50::FRT &#229;&#8230;&#179;F&#226;&#172;&#732; &#226;&#338;&#172;dinB50::FRT&#229;&#8230;&#180;                                   43 <br />   SMR6068                  SMR4562 &#226;&#338;&#172;yafN11::FRTKanFRT &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                                     SMR5833 &#226;&#171;&#187; DNA &#226;&#338;&#172;yafN::FRTKanFRT <br />                                                                                                       amplified pKD13 &#229;&#8230;&#179;from BW25141 <br />                                                                                                       (pKD13)&#229;&#8230;&#180;, location screened mating <br />   SMR6074                  SMR4562 &#229;&#8230;&#179;F&#226;&#172;&#732; &#226;&#338;&#172;yafO14::FRTKanFRT&#229;&#8230;&#180;                                          SMR5833 &#226;&#171;&#187; DNA &#226;&#338;&#172;yafO::FRTKanFRT <br />                                                                                                       amplified pKD13 &#229;&#8230;&#179;from BW25141 <br />                                                                                                       (pKD13)&#229;&#8230;&#180;, location screened mating <br />   SMR6076                  FC36 &#226;&#338;&#172;yafO14::FRTKanFRT                                                  FC36 &#226;&#171;&#187; P1(SMR6074) <br />   SMR6080                  SMR4562 &#226;&#338;&#172;yafP18::FRTKanFRT &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                                     SMR5833 &#226;&#171;&#187; DNA &#226;&#338;&#172;yafP::FRTKanFRT <br />                                                                                                       amplified pKD13 &#229;&#8230;&#179;from BW25141 <br />                                                                                                       (pKD13)&#229;&#8230;&#180;, location screened mating <br />   SMR6082                  FC36 &#226;&#338;&#172;yafP18::FRTKanFRT                                                  FC36 &#226;&#171;&#187; P1(SMR6080) <br />   SMR6221                  FC36 &#226;&#338;&#172;yafP20::FRT                                                        SMR6082 &#226;&#171;&#187; pCP20  BT340) <br />   SMR6233                  MG1655(pKD46)                                                            MG1655 &#226;&#171;&#187; pKD46 &#229;&#8230;&#179;from BW25113(pKD46)&#229;&#8230;&#180; <br />   SMR6353                  MG1655 &#226;&#338;&#172;(yafN-yafP)776::FRTKanFRT                                        SMR6233 &#226;&#171;&#187; DNA &#226;&#338;&#172;(yafN-yafP)::FRTKanFRT <br />                                                                                                        pKD13 &#229;&#8230;&#179;from BW25141(pKD13)&#229;&#8230;&#180; <br />   SMR6669                  MG1655 &#226;&#338;&#172;att&#226;?&#173;::PsulA&#226;?&#8364;gfp-mut2                                             26, 40 <br />   SMR7491                  SMR4562 &#226;&#338;&#172;(yafN-yafP)602 &#229;&#8230;&#179;F&#226;&#172;&#732; &#226;&#338;&#172;(yafN-yafP)602&#229;&#8230;&#180;                             43 <br />   SMR10285                 SMR4562 &#226;&#338;&#172;(yafN-yafO)779::FRTKanFRT &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                             SMR5833 &#226;&#171;&#187; DNA &#226;&#338;&#172;(yafN-yafO)::FRTKanFRT <br />                                                                                                       amplified pKD13 &#229;&#8230;&#179;from BW25141 <br />                                                                                                       (pKD13)&#229;&#8230;&#180; <br />  SMR10287                  FC36 &#226;&#338;&#172;(yafN-yafO)779::FRTKanFRT                                          FC36 &#226;&#171;&#187; P1(SMR10285) <br />  SMR10291                  SMR4562 &#226;&#338;&#172;yafN11::FRTKanFRT zae-502::Tn10 &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                       SMR6068 &#226;&#171;&#187; P1(CAG18436) <br />  SMR10483                  FC36 &#226;&#338;&#172;(yafN-yafO)779::FRTKanFRT zae-502::Tn10                            SMR10287 &#226;&#171;&#187; P1(CAG18436) <br />  SMR10943                  FC36(pBAD24)                                                             FC36 &#226;&#171;&#187; pBAD24  ATCC 87399) <br />  SMR10946                  DH5&#226;?&#163;(pLS1)                                                               DH5&#226;?&#163; &#226;&#171;&#187; pLS1 (initial plasmid construction) <br />  SMR10948                  FC36(pLS1)                                                               FC36 &#226;&#171;&#187; pLS1  SMR10946) <br />  SMR10953                  SMR4562 &#226;&#338;&#172;yafP18::FRTKanFRT zae-502::Tn10 &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                       SMR6080 &#226;&#171;&#187; P1(CAG18436) <br />  SMR10988                  MG1655 &#226;&#338;&#172;att&#226;?&#173;::PsulA&#226;?&#8364;gfp-mut2 &#226;&#338;&#172;(yafN-yafP)776::FRTKanFRT                  SMR6669 &#226;&#171;&#187; P1(SMR6353) <br />  SMR10990                  MG1655 recG258::Tn10 mini-Kan                                            MG1655 &#226;&#171;&#187; P1(N2731) <br />  SMR10995                  SMR4562 &#226;&#338;&#172;yafN11::FRTKanFRT &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180; &#229;&#8230;&#179;pKD46&#229;&#8230;&#180;                             SMR6068 &#226;&#171;&#187; pKD46 &#229;&#8230;&#179;from BW25113(pKD46)&#229;&#8230;&#180; <br />  SMR11008                  SMR4562 &#226;&#338;&#172;yafN11::FRTKanFRT &#226;&#338;&#172;yafP19::FRTcatFRT zae-502::                  SMR11168 &#226;&#171;&#187; P1(CAG18436) <br />                              Tn10 &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180; <br />   SMR11168                 SMR4562 &#226;&#338;&#172;yafN11::FRTKanFRT &#226;&#338;&#172;yafP19::FRTcatFRT &#229;&#8230;&#179;F&#226;&#172;&#732; yaf&#226;&#171;&#185;&#229;&#8230;&#180;                  SMR10995 &#226;&#171;&#187; DNA &#226;&#338;&#172;yafP::FRTcatFRT <br />                                                                                                      amplified pKD3 &#229;&#8230;&#179;from BW25141(pKD3)&#229;&#8230;&#180; <br />   SMR11228                 MG1655(&#226;?&#173;)                                                                MG1655 &#226;&#171;&#187; &#226;?&#173;SR108 <br />   SMR11229                 MG1655(&#226;?&#173;) &#226;&#338;&#172;(yafN-yafP)776::FRTKanFRT                                     SMR11228 &#226;&#171;&#187; P1(SMR6353) <br />   SMR11231                 MG1655(&#226;?&#173;) recG258::Tn10 mini-Kan                                         SMR11228 &#226;&#171;&#187; P1(N2731) <br />  <br /> Phage lambda <br />   &#226;?&#173;SR108                   &#226;?&#173; wild type                                                              F. Stahl (Oregon) <br />  <br /> Plasmids <br />   pBAD24                   Plasmid containing arabinose-inducible promoter PBAD, Ampr               23  ATCC 87399) <br />   pLS1                     Plasmid containing yafN control PBAD                        work <br />   pKD3                     cat-containing plasmid, template PCR                                 12 <br />   pKD13                    kan-containing plasmid, template PCR                                 12 <br />   pCP20                    FLP-containing plasmid, used excise drug markers flanked           12 <br />                              FRT sequences <br />   pKD46                    Red recombinase expression plasmid                                       12 <br />       FRT, FLP recombination target; CGSC, E. coli Genetic Stock Center (Yale University); ATCC, American Type Culture Collection. <br />  VOL. 191, 2009                                                                         SOS-REGULATED TOXIN-ANTITOXIN                          7459 <br />  <br />  <br />                                                         TABLE 2. Description new deletion alleles <br />                                                                                                              Deletion coordinates (distance     Replacement <br />       Strain                        New allele                         Wild-type gene length (bp) <br />                                                                                                               gene translation start)       cassettea <br />  <br /> SMR6068                 &#226;&#338;&#172;yafN11::FRTKanFRT                                       294                         &#226;&#171;&#185;13   &#226;&#171;&#185;279                    FLPable Kan <br /> SMR6074                 &#226;&#338;&#172;yafO14::FRTKanFRT                                       399                         &#226;&#171;&#185;13   &#226;&#171;&#185;384                    FLPable Kan <br /> SMR6080                 &#226;&#338;&#172;yafP18::FRTKanFRT                                       453                         &#226;&#171;&#185;16   &#226;&#171;&#185;438                    FLPable Kan <br /> SMR6221                 &#226;&#338;&#172;yafP20::FRT                                             453                         &#226;&#171;&#185;16   &#226;&#171;&#185;438                    FRT scar <br /> SMR10285                &#226;&#338;&#172;(yafN-yafO)779::FRTKanFRT                   individual genes              &#226;&#171;&#185;13   (yafN) &#226;&#171;&#185;384 (yafO)      FLPable Kan <br /> SMR11168                &#226;&#338;&#172;yafP19::FRTcatFRT                                       453                         &#226;&#171;&#185;40   &#226;&#171;&#185;416                    FLPable cat <br />       replacement cassette, sequence inserted DNA begins ATCC (Kan cassette) TCATA (cat cassette) ends CTACA (Kan cassette) TACAC (cat cassette). Sequences listed 5&#226;&#172;&#732; 3&#226;&#172;&#732;. FRT, FLP recombination target. <br />  <br />  <br />  <br />  <br /> selection extragenic suppressor mutations al-                           hypothesized viable, strain carry low cell mutated essential gene grow. initially                        spontaneous mutation inactivating yafO alternatively du- <br /> selected neutral Tetr marker screened                                plication operon copies yafN <br /> colonies Kan-replaced gene  yafN,                          deleted. mutants likely responsible <br /> yafNO, yafP, yafN yafP simultaneously). expected                           0.16% frequency cotransduction seen &#226;&#338;&#172;yafN:: <br /> frequency cotransduction markers (Kanr                            Kan zae-502::Tn10 cotransduction (Fig. 2B). sequenced <br /> Tetr) approximately 30% based physical distance.                          yafO gene strain JW0222 report yafO gene <br /> Cotransduction frequencies &#226;&#172;&#402;30% imply gene                             contains single base pair deletion <span id='am-44' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span>guanine</span> bp 7,  essential, cotransduction frequencies                         frameshift, creates early stop codon (TAA) significantly lower imply reduced viability trans-                        bp 61 399-bp yafO gene (Fig. 2D). resultant YafO <br /> ductants received deletion,  gene                           protein prematurely truncated likely nonfunc- <br /> deleted essential.                                                                 tional. <br />    &#226;&#338;&#172;yafN::Kan zae-502::Tn10 cells cotrans-                            YafP toxin antitoxin component. duced 0.16% &#226;&#171;&#190; 0.16% Tetr transductants (Fig. 2B).                          sought understand yafP integral function data imply yafN essential gene. Supporting                      antitoxin toxin. yafP integral interpretation, loss chromosomal copy                         antitoxin,  like yafN, essential viability yafN transduction achieved efficiently (23% &#226;&#171;&#190;                             presence yafO. &#226;&#338;&#172;yafP::Kan zae-502::Tn10 <br /> 2.7% &#226;&#338;&#172;yafN::Kan zae-502::Tn10 cotransductants) recipient                           cotransduced efficiently 20% &#226;&#171;&#190; 1.8% strain cells carrying yafN-containing plasmid (pLS1) (Fig. 2B)                         additional plasmid-borne yafP gene 26% &#226;&#171;&#190; 2% cells carrying plasmid vector (Fig. 2B).                        strains harboring extra yafP gene (Fig. 3A).  yafP conclude yafN essential gene.                                           essential presence yafO YafP    yafNO(P) genes TA  yafN                         integral antitoxin. <br /> expected essential putative toxin, YafO,                         yafP integral toxin, loss yafP <br /> present. &#226;&#338;&#172;(yafN-yafO)::Kan zae-502::Tn10                           expected allow efficient cotransduction &#226;&#338;&#172;yafN:: <br /> cells transduced, frequency cotransduction                     Kan linked Tetr marker yafN longer <br /> markers Tetr 29% &#226;&#171;&#190; 5% (Fig. 2C). data                        required viability. frequency  <br /> yafN essential YafO functional support                       transduction double deletion &#226;&#338;&#172;yafN::Kan &#226;&#338;&#172;yafP::cat interpretation yafN yafO TA                                     linked Tetr marker &#226;&#172;?0.33% cells extra copy <br />    contrast results, published yafN dele-                      yafN gene efficient cells carrying yafN <br /> tion strain, JW0222, E. coli Keio deletion collection (2).                     expression plasmid pLS1 (Fig. 3B), indicating yafN  <br />  <br />  <br />                                                     TABLE 3. Linear replacement gene deletion primers <br />                        Linear replacement <br />         Strain                                                                                Sequencea                                             Template <br />                           primer  <br /> SMR6068                   yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafNwR               gctgtaagttgcaggcgaataagttttgttttgaatacccgcatccttattccttaaagtcTGTAGGCTGGAGCTGCTTC <br /> SMR6074                   yafOwL               tatgacggatgatgatttcaatgactttaaggaataaggatgcgggtattcATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafOwR               atagtttcttatttgtatgttattcataatataaattcaaaaacgcatgcgTGTAGGCTGGAGCTGCTTC <br /> SMR6080/6221              yafPwL               gcagaagcgtttcgcatgcgtttttgaatttatattatgaataacatacaaATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafPwR               ataccaggcgggcgttattttcattgcaagctggatttaatgttgcggtttTGTAGGCTGGAGCTGCTTC <br /> SMR6353                   yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafPwR               ataccaggcgggcgttattttcattgcaagctggatttaatgttgcggtttTGTAGGCTGGAGCTGCTTC <br /> SMR10285                  yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafOwR               atagtttcttatttgtatgttattcataatataaattcaaaaacgcatgcgTGTAGGCTGGAGCTGCTTC <br /> SMR11168                  yafP::CAT-L          atgaataacatacaaataagaaactatcagcctggcgatTCATATGAATATCCTCCTTAG                                          pKD3 <br />                           yafP::CAT-R          atgttgcggtttatatcgcatataaaaattagtaaacGTGTAGGCTGGAGCTGCTTC <br />         Lowercase letters refer chromosomal sequence; uppercase letters refer template sequence. Sequences listed 5&#226;&#172;&#732; 3&#226;&#172;&#732;. <br />  7460      SINGLETARY ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Cotransduction assay quantitative determination inviability various mutant strains.   assay design. Phage P1 grown strains <br /> containing gene knockouts (Kanr) known linkage nonlethal Tetr marker, zae-502::Tn10, transduced recipient cells selected Tet. Colonies patched Kan plates determine frequency cotransduction markers, determine gene essential. gene essential, frequency cotransduction Kanr gene deletion Tetr transductants lower predicted distance markers. (B) yafN essential. &#226;&#338;&#172;yafN::Kan deletion cotransduced efficiently zae-502::Tn10 cells <br /> harboring plasmid pLS1, expresses yafN, cells carrying plasmid vector  P1 donor: &#226;&#338;&#172;yafN::Kan zae-502::Tn10 <br /> (SMR10291). Recipient cells: yaf&#226;&#171;&#185; (FC36), yaf&#226;&#171;&#185;[pBAD] (SMR10943), yaf&#226;&#171;&#185;[pLS1] (SMR10948). Average &#226;&#171;&#190; 1 SEM experiments. (C) yafN essential presence yafO. &#226;&#338;&#172;(yafN-yafO)::Kan zae-502::Tn10 cotransduction (P1 donor: SMR10483) yaf&#226;&#171;&#185; strain FC36. Average &#226;&#171;&#190; 1 SEM experiments. (D) yafO gene mutated yafN deletion strain JW0222 E. coli gene knockout Keio collection. Sequencing JW0222 yafO gene revealed &#226;&#171;&#186;1-bp deletion bp 7 results premature stop codon bp 61 399-bp gene. <br />  <br />  <br />  <br /> mains essential yafP deleted. address                      formed &#226;&#338;&#172;yafP recipient strain. frequency cotrans- <br /> possible concern lingering toxin recipient cell                    ductants remained &#226;&#172;?0.33% (Fig. 3B). conclude yafN <br /> (present prior transduction &#226;&#338;&#172;yafP) cause inviability               remains essential despite absence yafP.  YafP cotransduction, additional cotransduction                     necessary toxin. <br />  VOL. 191, 2009                                                             SOS-REGULATED TOXIN-ANTITOXIN                            7461 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 4. yafNOP genes required stress-induced mu- <br />                                                                           tagenesis. Lac assay strains starved lactose days                                                                           monitored day Lac&#226;&#171;&#185; reversion mutant colonies according    FIG. 3. YafP necessary component toxin anti-     method reference 25.   Representative experiment. Lac&#226;&#171;&#185; <br /> toxin.   yafP essential viability   revertants 108 cells plotted time. Average &#226;&#171;&#190; SEM antitoxin. &#226;&#338;&#172;yafP::Kan cotransduced efficiently zae-502::Tn10      cultures. (B) Summary mutation rates multiple experiments. <br /> (P1 donor: SMR10953) recipient cells: yaf&#226;&#171;&#185; (FC36) yaf&#226;&#171;&#185;[F&#226;&#172;&#732;         Mutation rates, calculated according method reference 37, <br /> yaf&#226;&#171;&#185;] (SMR4562). Average &#226;&#171;&#190; SEM experiments. (B) Loss          Lac&#226;&#171;&#185; revertants 108 cells day days 3 5. Mean &#226;&#171;&#190; SEM <br /> yafP does allow recovery &#226;&#338;&#172;yafN cotransductants         experiments. yaf&#226;&#171;&#185; strain SMR4562, &#226;&#338;&#172;yafNOP <br /> expected yafP required toxin function. Cotransduction    strain SMR7491, &#226;&#338;&#172;dinB strain SMR5889. <br /> P1 donor &#226;&#338;&#172;yafN::Kan &#226;&#338;&#172;yafP::cat zae-502::Tn10 (SMR11008)  <br /> cipient cells: yaf&#226;&#171;&#185; (FC36), &#226;&#338;&#172;yafP (SMR6221), yaf&#226;&#171;&#185;[pBAD] <br /> (SMR10943), yaf&#226;&#171;&#185;[pLS1] (SMR10948). Average &#226;&#171;&#190; SEM experiments.                                                              evidence yafNOP serve protective unit                                                                           E. coli phage &#226;?&#173;.  burst size &#226;?&#173; lytic infection                                                                           different yaf&#226;&#171;&#185; cells infected &#226;&#338;&#172;yafNOP <br />                                                                           cells infected; ratio yaf&#226;&#171;&#185; &#226;&#338;&#172;yafNOP 1.4 &#226;&#171;&#190; 0.6 <br />    role stress-induced mutagenesis. dinB,               (mean &#226;&#171;&#190; standard error mean [SEM] experi- gene dinB-yafNOP operon, key player stress-          ments).  test yafNOP genes promote <br /> induced point mutagenesis E. coli Lac (17, 42)              prevent progression &#226;?&#173; induction lyso- imagine roles TA process,            genic state, transition controlled SOS response (49), <br /> tested possible involvement yafNOP stress-induced lac           spotted various dilutions &#226;?&#173; lysogens E. coli yafNOP&#226;&#171;&#185; <br /> reversion. observed yafNOP strain showed slightly             &#226;&#338;&#172;yafNOP cells different concentrations bleo- significantly higher stress-induced mutagenesis              mycin- mitomycin C-containing plates. inducing SOS <br /> did yaf&#226;&#171;&#185; strain, unlike dinB, required (Fig. 4A          response DNA-damaging drugs, simultaneously induced <br /> B).  yafNOP genes required stress-induced            &#226;?&#173; prophage lytic, leading far severe <br /> mutagenesis. yafNOP genes shown previously                killing lysogens (Fig. 6A B) nonlysogens <br /> tribute slightly generation-dependent mutagenesis (43).           (Fig. 5B C), indicating major mode killing    SOS DNA damage response. hopes discovering                 lysogens prophage induction.  possible role yafNOP SOS response, explored             difference survival &#226;&#338;&#172;yafNOP lysogens assays DNA damage affects cell survival test          yaf&#226;&#171;&#185; lysogens (Fig. 6A B). yafNO contribute survival loss survival               SOS-induced senescence caused yafNO. Previously, <br /> phenotypes. used different DNA damage-induc-                   Pennington Rosenberg &#226;&#172;&#402;65% cells ing agents including UV light, bleomycin, mitomycin C                 undergo spontaneous SOS induction unable form col- induce SOS response. saw significant differ-                 onies, despite fact nearly SOS-induced cells <br /> ence &#226;&#338;&#172;yafNOP yaf&#226;&#171;&#185; cells survival UV light               viable, determined ability exclude dye PI <br /> (Fig. 5A); bleomycin, double-strand break-inducing agent                (47), suggesting senescence-like state (47). <br /> (29) (Fig. 5B); DNA cross-linking agent mitomycin C (30)               test senescence-like state result (Fig. 5C).                                                                toxic  possibly bacteriostatic) yafO, repeated ex- <br />    previous report suggested MazEF TA                   periment using FACS sort yaf&#226;&#171;&#185; &#226;&#338;&#172;yafNOP reporter <br /> protects cells phage P1 infection (27).               strains carrying chromosomal gfp gene controlled  7462     SINGLETARY ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Sensitivity nonlysogens DNA-damaging agents.   UV light kill curve. Log-phase cells UV irradiated monitored colony formation. Average &#226;&#171;&#190; 1 SEM experiments shown. (B) Nonlysogen bleomycin kill plates. Log-phase cells plated bleomycin-containing plates monitored viability. Bleomycin creates single- double-strand breaks DNA (29). &#226;&#338;&#172;recG strain <br /> monitored concurrently showed sensitivity  shown). Results shown panels B C representative experiments. (C) Nonlysogen <br /> mitomycin C kill plates. Log-phase cells plated mitomycin C-containing plates (various concentrations) monitored viability. <br /> Mitomycin C powerful DNA interstrand cross-linker (30). &#226;&#338;&#172;recG strain monitored concurrently showed sensitivity  shown). panels, yaf&#226;&#171;&#185; strain MG1655, &#226;&#338;&#172;yafNOP strain SMR6353, &#226;&#338;&#172;recG strain SMR10990. Log dilutions indicated. <br />  <br />  <br />  <br /> SOS-inducible sulA promoter. sorted sponta-              little difference colony-forming abilities <br /> neously SOS-induced green SOS-uninduced nongreen                     spontaneously SOS-induced green populations. ob- <br /> subpopulations, compose &#226;&#172;&#402;1% &#226;&#172;&#402;99% cell                   served 38% &#226;&#171;&#190; 12% yafNOP&#226;&#171;&#185; SOS-induced green cells <br /> population, respectively (reference 47 study).               formed colonies (35% &#226;&#171;&#190; 9% green cells forming colonies <br />  VOL. 191, 2009                                                                SOS-REGULATED TOXIN-ANTITOXIN                      7463 <br />  <br />  <br />                                                                                                       DISCUSSION <br />  <br />                                                                                 shown yafN gene dinB-yafN-yafO- <br />                                                                              yafP operon (43) essential viability presence                                                                              functional yafO gene (Fig. 2B C), providing strong evi- <br />                                                                              dence genes constitute antitoxin toxin <br />                                                                              genes, respectively, type 2 TA pair. previous                                                                              vivo demonstrations type 2 TA systems used                                                                              method separate cloning gene differentially <br />                                                                              inducible plasmids showing toxin induces cell <br />                                                                              stasis solely expressed, antitoxin                                                                              expressed (1, 50), used alternative genetic approach                                                                              allows rule possible effects specific overexpression <br />                                                                              toxin antitoxin. conclude <br />                                                                              YafO exerts toxic effect, YafN quell effect, <br />                                                                              gene expressed normal levels native <br />                                                                              promoter, single copy normal chromosomal position. <br />                                                                                 Typically, chromosomal TA systems consist genes                                                                              operon, toxin antitoxin. pre- <br />                                                                              cedent TA gene element, mazEFG <br />                                                                              (22), yafP, yaf gene operon,                                                                              integral component toxin antitoxin (Fig. <br />                                                                              3A B). conclusion does preclude possibility <br />                                                                              YafP play regulatory role YafNO TA <br />                                                                              detected, mazG does mazEF <br />                                                                              (22). <br />                                                                                 possible function TA controlled SOS <br />                                                                              response interesting problem. unlike dinB <br />                                                                              (42), gene operon (43), yafNOP genes                                                                              contribute stress-induced mutagenesis significantly (Fig. 4), <br />                                                                              process requires SOS-induced levels dinB,                                                                              SOS-induced levels SOS-controlled component <br />                                                                              (17). SOS-controlled genes recA (6, 24) ruvA <br />                                                                              ruvB (15, 25) required stress-induced Lac mutagen- <br />    FIG. 6. YafNOP does affect killing prophage induction           esis, appreciate, constitutive levels expres- <br /> DNA-damaging agents. DNA damage induces SOS response                 sion, induced levels (17). results presented rule <br /> activates &#226;?&#173; lytic cycle, causing killing prophage induction, seen greater sensitivity DNA-damaging agents lysogens <br />                                                                              requirement YafNOP stress-induced mutagenesis <br />  figure) nonlysogens (Fig. 5). YafNOP does affect      constitutive expression levels. Previously, <br /> killing prophage induction.   Lambda lysogen bleomycin kill             YafNOP little effect spontaneous generation-depen- <br /> plates. Log-phase E. coli (&#226;?&#173;) lysogens plated bleomycin-           dent mutagenesis nonstressed growing cells (43). <br /> containing plates (various concentrations) monitored viability.         did detect effects yafNOP survival cells &#226;&#338;&#172;recG strain monitored concurrently showed sensitivity  shown). <br /> (B) Lambda lysogen mitomycin C kill plates. Log-phase E. coli &#226;?&#173;              following various SOS-inducing treatments, nonlysogens <br /> lysogens plated mitomycin C-containing plates (various         (Fig. 5A C) cells harboring wild-type lambda <br /> centrations) monitored viability. &#226;&#338;&#172;recG strain monitored           prophage, induced leading cell lysis SOS <br /> concurrently showed sensitivity  shown). panels, yaf&#226;&#171;&#185;       response activated (Fig. 6A B). Finally, strain SMR11228, &#226;&#338;&#172;yafNOP strain SMR11229, &#226;&#338;&#172;recG <br /> strain SMR11231. Results shown representative experi-          recapitulate previous results Pennington ments.                                                                       Rosenberg, showing spontaneously SOS-induced <br />                                                                              green fluorescent cells (bearing SOS-controlled chromo- <br />                                                                              somal gfp reporter gene) apparently viable unable normalized 94% &#226;&#171;&#190; 10% nongreen cells forming colonies                   form colonies recovered FACS (47), control FACS-induced effects colony formation) (Fig.               yafNOP responsible senescence-like state <br /> 7). Similarly, 26% &#226;&#171;&#190; 5% &#226;&#338;&#172;yafNOP green cells formed col-                   (Fig. 7). <br /> onies (25% &#226;&#171;&#190; 5% green cells forming colonies normalized                      function SOS-controlled TA sys- 95% &#226;&#171;&#190; 2% nongreen cells forming colonies) (Fig. 7).                    tem? TA systems, toxic effects ensue operon&#226;&#8364;&#8482;s  strains&#226;&#8364;&#8482; cells nearly viable, 3.6% &#226;&#171;&#190;                expression decreased, expect possible effects 1.4% yafNOP&#226;&#171;&#185; green cells PI&#226;&#171;&#185; (dead) versus 4.5% &#226;&#171;&#190;                   YafO manifested cells recover SOS response <br /> 1.4% &#226;&#338;&#172;yafNOP green cells PI&#226;&#171;&#185;.  numbers                  return normal DNA repair, cells senescence-like state alive (PI&#226;&#171;&#186;) unable             genes expressed mid-range SOS response (11).  form colonies different yafNOP&#226;&#171;&#185;                      haps YafO induces transient cell stasis recovery. &#226;&#338;&#172;yafNOP cells (Fig. 7).                                                      function extend cell cycle checkpoint caused  7464      SINGLETARY ET AL.                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. yafNOP genes responsible senescence-like state spontaneously SOS-induced cells. Strains chromosomal <br /> gfp gene controlled SOS-inducible sulA promoter sorted SOS-induced green SOS-uninduced nongreen populations FACS plated CFU. decreased ability SOS-induced green cells form colonies reversed &#226;&#338;&#172;yafNOP mutation. Percentage live/dead cells population determined PI staining flow cytometry quantify dead cells, dye. percentage cells senescence-like state, excluding PI unable form colonies, different yaf&#226;&#171;&#185; (SMR6669) &#226;&#338;&#172;yafNOP (SMR10988) cells. <br /> Average &#226;&#171;&#190; SEM experiments. <br />  <br />  <br />  <br />  <br /> SOS-induced expression SulA inhibitor cell division                                         ACKNOWLEDGMENTS <br /> (16). did effects YafNO cells                work supported National Institutes Health grant <br /> SOS induction, rule possibility impor-              R01-GM53158. <br /> tant role SOS recovery assays                      thank N. Fonville, R. S. Galhardo, P. J. Hastings, C. Herman,  <br /> detected. Alternatively addition, dinB, presumably                Al Mamun, C. Shee valuable input course                                                                             work. rest operon, upregulated slightly RpoS <br /> general stress response (33). YafO plays role                                                                                                              REFERENCES <br /> promoting cell stasis recovery RpoS response. <br />                                                                              1. Aizenman, E., H. Engelberg-Kulka, G. Glaser. 1996. Escherichia coli <br /> Additionally, yafNO genes transcribed                     chromosomal &#226;&#8364;&#339;addiction module&#226;&#8364;? regulated <span id='am-11' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span>guanosine</span> [corrected] 3&#226;&#172;&#732;,5&#226;&#172;&#732;- upstream SOS-controlled  RpoS-controlled) dinB pro-                     bispyrophosphate: model programmed bacterial cell death. Proc. Natl. <br /> moter (43), vitro evidence additional pro-                  Acad. Sci. USA 93:6059&#226;&#8364;&#8220;6063. <br />                                                                              2. Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura, M. Baba, K.  <br /> moter exist immediately upstream yafN, possibly cre-                     Datsenko, M. Tomita, B. L. Wanner, H. Mori. 2006. Construction ating SOS  RpoS)-independent yafNOP operon (57).                       Escherichia coli K-12 frame, single-gene knockout mutants: Keio col-  yafNOP act outside contexts SOS                         lection. Mol. Syst. Biol. 2:2006.0008. <br />                                                                              3. Black, D. S.,  J. Kelly, M. J. Mardis, H. S. Moyed. 1991. Structure RpoS) response.                                                                 organization hip, operon affects lethality inhibition    recent report shows DinB SOS-                        <span id='am-33' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-34' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> DNA synthesis. J. Bacteriol. 173:5732&#226;&#8364;&#8220;5739. <br />                                                                              4. Blattner, F. R., G. Plunkett III, C.  Bloch, N. T. Perna, V. Burland, M. <br /> inducible DNA polymerase, Pol V, interact directly                     Riley, J. Collado-Vides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, <br /> NusA transcription antitermination factor (10). au-                     N. W. Davis, H.  Kirkpatrick, M.  Goeden, D. J. Rose, B. Mau, Y. <br /> thors suggest NusA direct translesion DNA                        Shao. 1997. complete genome sequence Escherichia coli K-12. Science <br />                                                                                 277:1453&#226;&#8364;&#8220;1474. <br /> synthesis activity DinB sites active transcription. Sim-            5. Brown, J. M., K. J. Shaw. 2003. novel family Escherichia coli <br /> ilarly, imagine DinB affect NusA-dependent                    toxin-antitoxin gene pairs. J. Bacteriol. 185:6600&#226;&#8364;&#8220;6608. <br /> transcription termination, provide                  6. Cairns, J., P. L. Foster. 1991. Adaptive reversion frameshift muta- <br />                                                                                 tion Escherichia coli. Genetics 128:695&#226;&#8364;&#8220;701. <br /> level SOS control gene expression (negative positive).              7. Cherepanov, P. P., W. Wackernagel. 1995. Gene disruption Esche- YafNO YafP functions process,                          richia coli: TcR KmR cassettes option Flp-catalyzed excision <br />                                                                                 antibiotic-resistance determinant. Gene 158:9&#226;&#8364;&#8220;14. toxic effect YafO related transcription- <br />                                                                              8. Christensen, S. K., M. Mikkelsen, K. Pedersen, K. Gerdes. 2001. RelE, <br /> termination-specific effect.                                                    global inhibitor translation, activated nutritional stress. Proc. <br />    manuscript prepared, group                       Natl. Acad. Sci. USA 98:14328&#226;&#8364;&#8220;14333. <br />                                                                              9. Cirz, R. T., F. E. Romesberg. 2006. Induction inhibition cipro- <br /> reported YafO protein RNase (58). results                  floxacin resistance-conferring mutations hypermutator bacteria. Antimi- <br /> support conclusions provide mechanism toxic                   crob. Agents Chemother. 50:220&#226;&#8364;&#8220;225. <br /> action YafO toxin. Important steps                 10. Cohen, S. E., V. G. Godoy, G. C. Walker. 2009. Transcriptional modu- <br />                                                                                 lator NusA interacts translesion DNA polymerases Escherichia coli. <br /> standing yafNO TA include defining                   J. Bacteriol. 191:665&#226;&#8364;&#8220;672. targets toxin acts.                                             11. Courcelle, J.,  Khodursky, B. Peter, P. O. Brown, P. C. Hanawalt. <br />  VOL. 191, 2009                                                                            SOS-REGULATED TOXIN-ANTITOXIN                                    7465 <br />  <br />       2001. Comparative gene expression profiles following UV exposure wild-          36. Lloyd, R. G., C. Buckman. 1991. Genetic analysis recG locus       type SOS-deficient Escherichia coli. Genetics 158:41&#226;&#8364;&#8220;64.                           Escherichia coli K-12 role recombination DNA repair. J. <br /> 12.   Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo-              Bacteriol. 173:1004&#226;&#8364;&#8220;1011. <br />       somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad.         37. Lombardo, M. J.,  Aponyi, S. M. Rosenberg. 2004. General stress <br />       Sci. USA 97:6640&#226;&#8364;&#8220;6645.                                                                 response regulator RpoS adaptive mutation amplification Esche- <br /> 13.   Engelberg-Kulka, H., G. Glaser. 1999. Addiction modules pro-                   richia coli. Genetics 166:669&#226;&#8364;&#8220;680. <br />       grammed cell death antideath bacterial cultures. Annu. Rev. Micro-          38. Magnuson, R. D. 2007. Hypothetical functions toxin-antitoxin systems. J. <br />       biol. 53:43&#226;&#8364;&#8220;70.                                                                        Bacteriol. 189:6089&#226;&#8364;&#8220;6092. <br /> 14.   Fernandez Henestrosa,  R., T. Ogi, S. Aoyagi, D. Chafin, J. J. Hayes, H.      39. Masuda, Y., K. Miyakawa, Y. Nishimura, E. Ohtsubo. 1993. chpA       Ohmori, R. Woodgate. 2000. Identification additional genes belonging            chpB, Escherichia coli chromosomal homologs pem locus responsible <br />       LexA regulon Escherichia coli. Mol. Microbiol. 35:1560&#226;&#8364;&#8220;1572.                 stable maintenance plasmid R100. J. Bacteriol. 175:6850&#226;&#8364;&#8220;6856. <br /> 15.   Foster, P. L., J. M. Trimarchi, R.  Maurer. 1996. enzymes,          40. McCool, J. D., E. Long, J. F. Petrosino, H.  Sandler, S. M. Rosenberg,       process recombination intermediates, opposite effects                 S. J. Sandler. 2004. Measurement SOS expression individual Esche- <br />       adaptive mutation Escherichia coli. Genetics 142:25&#226;&#8364;&#8220;37.                             richia coli K-12 cells using fluorescence microscopy. Mol. Microbiol. 53: <br /> 16.   Friedberg, E. C., W. Siede, G. C. Walker. 2006. DNA repair mu-                 1343&#226;&#8364;&#8220;1357. <br />       tagenesis. ASM Press, Washington, DC. <br />                                                                                          41. McKenzie, G. J., R. S. Harris, P. L. Lee, S. M. Rosenberg. 2000. 17.   Galhardo, R. S., R.  M. Yamada, E. C. Friedberg, P. J. Hastings, T. <br />                                                                                              SOS response regulates adaptive mutation. Proc. Natl. Acad. Sci. USA <br />       Nohmi, S. M. Rosenberg. 2009. DinB upregulation sole role                                                                                              97:6646&#226;&#8364;&#8220;6651. <br />       SOS response stress-induced mutagenesis Escherichia coli. Genetics <br />                                                                                          42. McKenzie, G. J., P. L. Lee, M. J. Lombardo, P. J. Hastings, S. M. <br />       182:55&#226;&#8364;&#8220;68. <br /> 18.   Gerdes, K., S. K. Christensen,  Lobner-Olesen. 2005. Prokaryotic                 Rosenberg. 2001. SOS mutator DNA polymerase IV functions adaptive <br />       toxin-antitoxin stress response loci. Nat. Rev. Microbiol. 3:371&#226;&#8364;&#8220;382.                  mutation adaptive amplification. Mol. Cell 7:571&#226;&#8364;&#8220;579. <br /> 19.   Gotfredsen, M., K. Gerdes. 1998. Escherichia coli relBE genes              43. McKenzie, G. J., D. B. Magner, P. L. Lee, S. M. Rosenberg. 2003.       belong new toxin-antitoxin gene family. Mol. Microbiol. 29:1065&#226;&#8364;&#8220;1076.             dinB operon spontaneous mutation Escherichia coli. J. Bacteriol. <br /> 20.   Grady, R., F. Hayes. 2003. Axe-Txe, broad-spectrum proteic toxin-                185:3972&#226;&#8364;&#8220;3977. <br />       antitoxin specified multidrug-resistant, clinical isolate Entero-   44. Miller, J. H. 1992. short course bacterial genetics: laboratory manual <br />       coccus faecium. Mol. Microbiol. 47:1419&#226;&#8364;&#8220;1432.                                          handbook Escherichia coli related bacteria. Cold Spring Harbor <br /> 21.   Grant, S. G. N., J. Jessee, F. R. Bloom, D. Hanahan. 1990. Differential            Laboratory Press, Plainview, NY. <br />      <br /> </body></html>