<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, June 1986, p. 746-750                                                                           Vol. 166,  3 <br /> 0021-9193/86/060746-05$02.00/0 <br /> Copyright &#194;&#169; 1986, American Society Microbiology <br />  <br />  <br />          Deletion spf (Spot 42 RNA) Gene Escherichia coli <br />                              GRAHAM F. HATFULL CATHERINE M. JOYCE* <br /> Department Molecular Biophysics Biochemistry, Yale University Medical School, New Haven, Connecticut 06510 <br />                                          Received 1 November 1985/Accepted 10 March 1986 <br />  <br />             investigate function spot 42 RNA, small RNA Escherichia coli, constructed strain          spf, structural gene RNA, deleted. achieved using Aatt phage k carrying DNA <br />          fragment spanning spf region precise deletion spf. integration phage spf locus <br />          subsequent excision, able cross spf deletion bacterial chromosome. fact <br />          deletion obtained indicated spf essential gene. did observe major <br />          defect Aspf cells, strain background deletion caused slight growth impairment. <br />  <br />  <br />    Spot 42 RNA Escherichia coli moderately abundant                             MATERIALS METHODS <br /> (100 200 copies cell) RNA, 109 nucleotides long (14, <br /> 15). encoded single gene (spf) located immediately                Materials. Restriction endonucleases obtained downstream polA 86.6 min E. coli genetic map               New England Biolabs Boehringer Mannheim Biochemi- <br /> (2, 7, 13). function spf gene unknown,             cals. T4 DNA ligase Collaborative Research, , abundance RNA suggested gene                    Boehringer Mannheim Biochemicals. Oligonucleotide <br /> product play important, essential, role                 linkers Collaborative Research, , New <br /> vivo. cellular distribution spot 42 RNA gives little               England Biolabs. Bacterial strains used work listed <br /> clue function; RNA associated                    Table 1. Plasmids general techniques relevant membranes, ribosomes, nucleoid (15).                    work described previously (8).    Currently know ultimate gene                     plasmids derived vector pNG16, deletion <br /> product spf RNA protein. RNA itself               derivative pBR322. <br /> structural catalytic component ribonucleoprotein                     Construction genetically marked spf deletion. particle, like Ml RNA component RNase P (5).                construction outlined Fig. 1. pGH78 contains 2.1- hand, spot 42 RNA sequence features suggesting                  kilobase-pair (kb) SacI-HindIII fragment carrying end mRNA: ribosome-binding site followed                  polA coding region spf gene, BamHI linker initiation codon open reading frame                 inserted AluI site 46 base pairs (bp) coding capacity 15-residue <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> (14). potential              termination codon polA (8). BamHI site defined translated product rich hydrophobic amino acids,                    upstream endpoint deletion spf. downstream <br /> particularly leucine, contains total charged           endpoint defined using plasmid, AN3058, carrying residues (including N- C-terminal amino acids).               vitro-constructed deletion polA (6). plasmid  alternating pattern positively negatively           tains DNA Sau3A site position 3058 charged groups adjacent pairs separated approxi-                  published sequence (7), HindIII site position 3822, <br /> mately turn helix. Consideration hydro-                BamHI site position original Sau3A site. <br /> phobic nature putative peptide gene product led             genetic marker used kanamycin resistance (Kmr) suggest inserted mem-                   gene Tn903; plasmid pGH54 contains gene 950-bp <br /> branes, serving structural role forming channels             BamHI fragment inserted BamHI site pNG16 (8). allow passage certain ions small molecules (7,               800-bp BamHI-EcoRI fragment AN3058 13). Sahagan Dahlberg shown expression                   950-bp BamHI fragment pGH54 purified ligated <br /> spf negatively regulated cyclic <span id='am-11' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span> (15).             large EcoRI-BamHI fragment pGH78, taken circumstantial evidence linking spf gene             structure Kmr gene (transcribed product membrane structure function, cyclic                 direction polA spf) replaces spf. Sall site AMP negative regulator membrane                  vector DNA replaced EcoRI linker envelope proteins (1, 10, 12) cell division (17).           resulting 3.2-kb EcoRI fragment recovered cloning <br />    problem assigning function gene locus             EcoRI site pNG16, pGH84. Kmr Aspf <br /> approached asking gene                   construction inserted EcoRI vector, Xgtl- <br />  available cloned fragment) essential               XB (4). structure resulting phage, XGH84, viability E. coli. determining                 verified Southern blot analysis, using pCJ1 probe. gene replaced chromosome                  control phage, XGH86, carries Kmr gene vitro-constructed deletion, using strategy             deletion spf, constructed analogous way <br /> described previously (8). target gene essential         insertion Kmr fragment BamHI site survival, obtain strain gene             pGH78. <br /> deleted. null mutation used test                     Isolation analysis lysogens. Lysogens XGH84 E. <br /> functions gene.                                                    coli CM4722 selected 30&#194;&#176;C L-broth plates seeded <br />                                                                           Xch80del9 (11) subsequently checked X <br />   * Corresponding author.                                                 immunity Kmr. Chromosomal DNA candidate <br />                                                                     746 <br />  VOL. 166, 1986                                                                                                 DELETION spf GENE             747 <br />  <br />           TABLE 1. Bacterial strains used work                                desired transductants selected plates containing 40 <br />  Strain                   Genotype                         Source                     ,ug kanamycin ml. <br /> CM4722&apos;   F+ gal-bio) thi relAl spoTI     W. S. Kelley <br />                                                                                          Growth CJ200 CJ205 liquid medium com- <br /> CJ200     CM4722 spf&apos; Kmr                     work                               pared using competition experiment. strain CJ205     CM4722 Aspf Kmr                     work                               mixed equal number cells parent strain, <br /> S165      AgalS165 rpsL                   strain                              CM4722 (Kms). period growth, mixture                                                 collection                            plated L broth, proportion Kmr cells (CJ200 LE392  lacY galK galT metB trpR supE supF     strain                              CJ205) determined. <br />          hsdR                                   collection                               sensitivity spf&apos; Aspfstrains salt, detergents, <br /> AW330 lac gal ara xyl thr leu thi fhuA rpsL   R. Macnab                               drugs compared ways. method, 20SOK- galK galP                              R. Macnab                               heavy inoculum strain tested streaked JC1552 leuB trp hisG argG metB lacY gal       B. Bachmann                             sector L-broth plate. substance          malA xyl mtl rpsL fhuA tsx supE        (CGSC)b                               tested placed center plate. AB2557 purF dsdA aroC ilvD lac malA xyl mtl B. Bachmann                               incubation, face-upwards, position inhibition zone <br />          rpsL fhuA tsx supE                     (CGSC) <br /> KL188 pyrD trp thyA thi galK malA xyl mtl B. Bachmann                             isogenic Aspf spf&apos; pairs compared. Alterna- <br />          rpsL                                   (CGSC)                                tively, strains plated agar substance <br />   CM4722 F+ derivative HfrH strain KS302 (16). <br />                                                                                       tested placed filter paper disk lawn  b <br />     CGSC, E. coli Genetic Stock Center, Yale University, New Haven,                   cells, described Benson DeCloux (3). Similar <br /> Conn.                                                                                 results obtained methods. <br />                                                                                          Swarming behavior motile strains examined                                                                                       tryptone plates containing 0.38% agar. <br /> lysogens analyzed Southern blots determine                                                         RESULTS <br /> structure spf locus.                                                             method determining particular gene    Curing lysogens. Lysogens grown overnight                               essential survival (Fig. 2) described 30&#194;&#176;C L broth. Cured cells selected plating                                 previously (8). Briefly, involves insertion Kmr <br /> appropriate dilutions 42&#194;&#176;C L-broth plates containing 1                          marker sequences immediately flanking mM EDTA. proportion survivors retained Kmr                                gene  using standard vitro manipulations. determined replica plating. obtain independent                           genetically marked deletion cloned phage vector <br /> estimate relative frequencies recombination                             Aatt thermoinducible (cI857). permissive <br /> events giving rise Kmr kanamycin-sensitive (Kms)                               temperature, phage lysogenize homologous  <br /> cells, sample lysogen culture induced                         combination cloned bacterial sequences, giving phage lysate. composition lysate determined                            structure integrated prophage flanked forming lysogens strain LE392 determining                              wild-type deleted (Kmr) copy target gene. relative numbers Kmr Kms lysogens.                             Spontaneous curing lysogen regenerate <br /> details, reference 8.)                                                            wild-type configuration leave Kmr-marked dele- <br />    Hybridization probes. Southern blots probed                            tion chromosome, depending position nick-translated pCJ1, carries 5-kb HindIll                               recombinational crossover. Failure obtain Kmr cells <br /> fragment containing polA spf genes (9), spot                          temperature-resistant cured survivors indi- <br /> 42 RNA labeled vitro transcription described                                 cate deletion target gene lethal previously (7).                                                                       necessary recombinational events did place    methods. Transduction spf Kmr E. coli                                high frequency. possibility ad- <br /> CJ200, Aspf Kmr E. coli CJ205, carried                               dressed independent estimate fraction dele- <br /> using P1 kc following standard procedures (11).                               tion-producing events, examining product <br />  <br />  <br />                                          500 bp <br />  <br />                                   S    Sc                            Bspf                          H.,R <br />                                                                              P.II <br />                                                                                                                        pGH78 <br />                                                                              B <br />                                                                                                                        AN3058 <br />                             R    Sc                                 KmR                                   H,R <br />                                                       za <br />                                                            B <br />                                                                     --H1-               B <br />                                                                                                           II <br />                                                                                                                        pGH84 <br />                             R    Sc                                 Km           IH       B                     H,R <br />                                                                                                                  ,r <br />                                                                   . .... <br />                                                                                               -W <br />                                                                                                                       * oGH36 <br />   FIG. 1. Construction plasmids used work. details text. line represents DNA derived E. coli <br /> chromosome. region indicated polA coding sequence (hatched area), spf gene (black arrow), functional <br /> open reading frame 22-kilodalton protein (open arrow) (7). DNA fragment carrying Kmr gene represented wavy line. <br /> DNA derived plasmid vectors drawn line shown entirety. B, BamHI; H, Hindlll; R, EcoRI; S, Sall; Sc, <br /> Sacl. <br />  748     HATFULL JOYCE                                                                                                        J. BACTERIOL. <br />  excision process, phage produced induction                                            XGH84 lysogen, repeating curing addi- <br /> tional copy target gene trans remove                                            R KmR R <br /> selective pressure deletion. <br />    Phage lysogen structure. construction XGH84, spf gene replaced Kmr retaining surrounding sequences, described                                 S                .                S <br /> Materials Methods section Fig. 1. deletion <br /> endpoints carefully chosen remove completely                                               13.3 - spf-transcribed region encroaching extremely close neighboring genes (7). Lysogens XGH84 CM4722 selected, chromosomal                                                      Integration <br /> DNA analyzed Southern blots identify integration taken place spflocus. Digestion EcoRI SalI gives pair fragments  sizes 8.2                     SR                               R KMR               S 8.3 kb, 7.5 9.0 kb, depending position                -4      _                  X            C              | recombihational crossover) integration taken place <br />                                                                                  8.3   *                                   8.2 spf (Fig. 3). Integration chromosome               results 13.3-kb fragnient diagnostic unrearranged                    S              KmR R <br /> spf locus 3.2-kb fragment derived XGHS4.                                                            R.   *S <br /> chose independent lysogens, type                                                                             B <br />                                                                                         9.0        b    *x-m               7.5 <br /> type B, curing experiment. <br />    Curing lysogens shows spf deleted. Cured             FIG. 3. Integration XGH84 spf locus. alternative <br /> cells selected overnight cultures chosen            structures, designated B, result integration lysogens, proportion Kmr colonies deter-              XGH84 homologous recombination spf. Symbols: *, position <br />                                                                      spf gene; mm, upstream flanking DNA carried XGH84; <br /> mined (Table 2). case, substantial proportion           zi, downstream flanking DNA: Chroniosomal DNA external Kmr, spf-deleted, cells obtained.         region, homology XGH84 shown wavy line. lysogen tested, ratio Kmr Kms cured cells              position relevant EcoRI (R) Sall (S) sites indicated, approximately reciprocal ratio Kmr Kms                  sizes  kilobases) fragments hybridize transducing phage obtained induction lysogen,             pCJ1 probe. <br /> indicating selective bias spf-deleted <br /> cells. consistent  fourfold bias favor <br />                                                                      recombination upstream flanking region,                                                                      lysogens structure gave tnore Kmr cells, lysogens                                                                      structure B gave Kms cells (Table 2 Fig. 3).                                                                      probably greater length upstream region                            ., X        _                             flanking homology (1,150 bp opposed 770 bp). <br />                                                                         Southern blot analysis chromosomal DNA cured <br />                                                                      cells obtained way showed Kms cells                                                                      regained 5-kb HindIII fragment diagnostic unrear- <br />                                                                      ranged spf locus (data shown). Kmr cells,                                    Integration                       hand, new HindIII fragments 4.7 1.1 kb (Fig. <br />                                                                      4B, lanes 1 2).  DNA cells did                                                                      hybridize spot 42 RNA (Fig. 4B, lanes 5 6).                   KmR.                     x                         observation confirms hybridization data Rice                                                                      Dahlberg (13) indicated wild-type E. coli <br />                                                                      chromosome contains single locus homologous spot 42 <br />                                        Curing                        RNA. <br />                                                                         curing experiment provided cells lacking                                                                      spf gene, serve tool examline                                                                      phenotype spf null mutation probe               x                                 KmR                  function spf gene. sure phenotype                                                       =P__           attributable Aspf mutation, generated isogenic <br />                                                                   spf&apos; Kmr strain, using control phage XGH86,                                                                      Kmr inserted removing spf (Fig. 1).                      )                           X                   strains CJ200 (spfJ Kmr) CJ205 (Aspf Kmr) identical <br />                                                                      presence CJ200 230 bp spanning spf <br />                                                                      gene (Fig. 4). Southern blot analysis DNA spf+ <br />                                                                      Kmr cells confirmed smnaller HindIII fragment    FIG. 2. Method determining gene deleted.     1.3 kb, compared 1.1 kb Aspf cells (Fig. 4B, lanes 3 <br /> Phage integration excision used cross deletion   4), fragment hybridized spot 42 RNA <br /> target gene (X) bacterial chromosome. details text. tegions homology phage chromo-      (lanes 7 8). boxed: X  upstream flanking DNA; =., downstream             Phenotype Aspf cells. compared growth CJ205 <br /> flanking DNA. Chromosomal DNA external regions           (Aspf) CJ200 (spf) various solid media: L broth homology shown wavy lines. (Note      MacConkey agar, supplemented glucose, <span id='am-6' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Lactose" datatype="xsd:string"></span>lactose</span>, <br /> possible integrated structures shown.)                            galactose, maltose, M9 minimal medium (11) supple- <br />  VOL. 166, 1986                                                                                      DELETION spf GENE              749 <br />  <br />  <br />                                           kb <br />  <br />                                 H                                               H <br />                                                                                                            spf <br />  <br />  <br />                                 H                                  KmR ,                   H <br />                                                     4.7                   &apos;O        1.                     Aspf KmR <br />  <br />  <br />                                 H                                  KmR H                     H <br />                                  &apos;*                 4.7                 -O..*        1.3       w <br />                                                                                                            spf KmR <br />  <br />    B             1 2 3 4              5 6 7 8 <br />                                                                   FIG. 4. Restriction analysis cured cells.   diagram <br />                                                                 compares   structure spf locus wild-type cells (spfJ),   4.7- <br />                                                                 /spf Kmr cells obtained curing XGH84 lysogens, spf&apos; <br />                                                                 Kmr cells derived XGH86. spf gene represented                                                                 black arrow, Kmr insert represented boxed region. <br />                                                                 HindlIl sites (H) shown, sizes  kilobases)                                                                 Hindlll fragments hybridize pCJ1 probe. (B) Southern blot <br />                                                                 analysis Aspf Kmr cells (lanes 1, 2, 5, 6) spfF Kmr cells <br />                                                                 (lanes 3, 4, 7, 8). Chromosomal DNA isolates                                                                 genotype digested HindIII. digest divided <br />                                                                 tracks 1% agarose gel lanes 1 4 5                                                                 8 contain identical sets DNA samples. transfer nitrocel- <br />   1.3-                  -..k   ~.. <br />                                                                 lulose, lanes 1 4 hybridized nick-translated pCJ1,                                                                 lanes 5 8 hybridized spot 42 RNA transcribed vitro. <br />   1.1-                                                          sizes fragments  kilobases) indicated. <br />  <br />  <br />  <br />  <br /> mented glucose glycerol. difference      generations temperature, proportion CJ205 <br /> colony size morphology 37&#194;&#176;C 42&#194;&#176;C.         cells decreased (typically 50% start plates incubated 23&#194;&#176;C, CJ205 (Aspf) gave smaller    experiment 25% 10 generations). suspect colonies CJ200. difference noticeable      growth difference liquid medium 42&#194;&#176;C L-broth plates, amounted twofold difference      peculiar CM4722 background, effect colony diameter 2 days growth. believe      seen S165 strain background.  effect slower growth 23&#194;&#176;C attributable spf deletion   liquid medium 42&#194;&#176;C accompanied observed secondary mutation, observed          difference plating efficiency colony morphology result independent isolates CJ205         temperature. <br /> genotype obtained curing transduction.              tested number properties diagnostic alter-  effect dependent CM4722            ation membrane structure function. Aspf mutation <br /> ground. difference growth 23&#194;&#176;C seen        did affect sensitivity cells NaCl, EDTA, sodium <br /> compared Aspf Kmr spf&apos; Kmr derivatives         dodecyl sulfate, various antibiotics (carbenicillin, tetracy- <br /> strains listed Table 1. CM4722 strain   cline, nalidixic acid, rifampin), 37&#194;&#176;C 23&#194;&#176;C. group F+, transferred F factor         (Increased sensitivity compounds characteristic CM4722 Aspf derivative strain S165. resulting <br /> strain indistinguishable F- parent plates 23&#194;&#176;C, ruling effect F fact <br /> </body></html>