<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Innate Immunity Human Embryonic Stem Cells: <br /> Comparison Adult Human Endothelial Cells <br /> Ga&#194;&#180;bor Fo&#194;&#168;ldes1., Alexander Liu1., Rekha Badiger1, Mark Paul-Clark1, Laura Moreno1, Zsuzsanna <br /> Lendvai2, Jamie S. Wright1, Nadire N. Ali1&quot;, Sian E. Harding1*&quot; , Jane  Mitchell1&quot; <br /> 1 National Heart Lung Institute, Imperial College, London, United Kingdom, 2 Heart Center, Semmelweis University, Budapest, Hungary <br />  <br />  <br />  <br />      Abstract <br />      Treatment human disease human embryonic stem cell (hESC)-derived cells close reality, little known <br />      responses physiological pathological insult. ability cells respond activation Toll like receptors <br />      (TLR) critical innate immune sensing tissues, extends general danger sensing, e.g. oxidative <br />      stress, cardiomyocytes. used biomarker release gene-array analysis compare responses hESC      differentiation, primary human endothelial cells. presence cardiomyocytes endothelial cells <br />      confirmed differentiated cultures immunostaining, FACS-sorting  cardiomyocytes, beating activity. <br />      Undifferentiated hESC did respond CXCL8 release Gram positive Gram negative bacteria, range PAMPs <br />      (pathogen associated molecular patterns) TLRs 1-9 (apart flagellin, activator TLR5). Surprisingly, lack TLR- <br />      dependent responses maintained 4 months differentiation hESC, cultures included cardiomyocytes <br />      endothelial cells. contrast, primary cultures human aortic endothelial cells (HAEC) demonstrated responses      broad range PAMPs. Expression downstream TLR signalling pathways demonstrated hESC, 1b, TNFa      INFc, bypass TLRs, stimulated CXCL8 release. NFkB pathway expression present hESC NFkB      able translocate nucleus. Low expression levels TLRs detected hESC, especially TLRs 1 4, explaining <br />      lack response hESC main TLR signals. TLR5 levels similar differentiated hESC HAEC,      siRNA knockdown TLR5 abolished response flagellin. findings potential implications survival      function grafted hESC-derived cells. <br />  <br />   Citation: Fo&#194;&#168;ldes G, Liu  Badiger R, Paul-Clark M, Moreno L, et al. (2010) Innate Immunity Human Embryonic Stem Cells: Comparison Adult Human <br />   Endothelial Cells. PLoS 5(5): e10501. doi:10.1371/journal.pone.0010501 <br />   Editor: Massimo Federici, University Tor Vergata, Italy <br />   Received November 2, 2009; Accepted April 14, 2010; Published 5, 2010 <br />   Copyright: &#195;&#376; 2010 Fo&#194;&#168;ldes et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported  Wellcome Trust (http://www.wellcome.ac.uk/), British Heart Foundation (http://www.bhf.org.uk/), European Community, <br />   NC3Rs (http://www.nc3rs.org.uk/), BBSRC (http://www.bbsrc.ac.uk/), Rosetrees Trust (http://www.rosetreestrust uk/), SROP (http://www.nfu.hu/), Geron <br />   Corporation. L. recipient Wellcome Trust Vacation Scholarship. M.P.C. recipient Wellcome Trust University Award (083429 07  R.B.   recipient BHF Clinical Research Fellowship. funders role study design, data collection analysis, decision publish, preparation   manuscript. <br />   Competing Interests: Geron: authors non-financial collaborative agreement, does alter adherence PLoS policies   sharing data materials. <br />   * E-mail: sian.harding@imperial.ac.uk <br />   . authors contributed equally work. <br />   &quot; authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            (TLRs) [4]. <span id='am-1' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span>LPS</span> activates TLR4 recruits adapter protein <br />                                                                                         pathways including MyD88, MAL, TRIF TRAM initiate <br />     Human embryonic stem cells (hESC) currently                               signaling events leading activation NFkB induction developed sources tissue-specific cells treatment                      inflammatory genes including CXCL8. 10 TLRs <br /> human disease, including heart failure. hoped hESC-                          expressed human cells specific PAMPs 1&#226;&#8364;&#8220;9 identified. <br /> derived cells seed repair damaged tissues allowing                           TLR10 remains orphan receptor present. addition recovery organ function. immune response                         sensing pathogens PRRs understood sense host <br /> host implanted cells subject                         ligands wider role surveillance danger signals <br /> little known innate immune response grafted                         [5]. phenotypes TLR knock mice studied <br /> cells  cells body express partial                      directly, TLR4 gene deletion, example, associated innate immune response, include endothelial                          immune suppression, chronic inflammation lung [6], <br /> cells [1] cardiomyocytes [2,3] required make                       vascular compromise evidence heart failure [7]. viable cardiac graft. innate immune response                             hand, TLR activation heart involved modeled experimentally activation cells pathogens                         deleterious responses oxidative stress [8], ischemia [9] pathogen associated molecular patterns (PAMPs). best studied                        septic cardiomyopathies [10], various TLR knockout mice PAMPs <span id='am-11' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-12' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span>lipopolysaccharide</span> (LPS) derived Gram                              resistant insults doxorubicin <br /> negative bacteria. PAMPs sensed cells pattern                                cardiomyopathy [11] hypertrophy [12]. HESC originate <br /> recognition receptors (PRRS), include Toll like receptors                         inner cell mass blastocyst, left undisturbed <br />  <br />  <br />        PLoS | www.plosone.org                                                   1                                    2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                    TLR Pathways Human hESC <br />  <br />  develop sterile environment womb. TLR                    gelatinized dishes cultured order allow continued <br /> responses embryo develop relatively late,               differentiation. <br /> suppressed neonatal period [13]. question hESC derivatives culture similar trend e.              Cell Plating Handling retain immature phenotype develop                      Undifferentiated H7 cell colonies 6 plates mature TLR responses, vital question understand            removed subcultured 96 plates previously coated order establish potential repair heart       Matrigel (100 ul .  medium aspirated organs. paper directly compare expression                 spontaneously differentiating cells colonies removed <br /> activity TLRs, downstream signaling components                  treatment collagenase 37uC 10 minutes, NFkB signaling undifferentiated differentiated                   time collagenase aspirated cells human hESC fully differentiated mature human                 washed 2 ml PBS. Colonies undifferentiated H7 cells aortic endothelial cells.                                                    broken mechanically 5-ml pipette tip small clusters <br />                                                                              subcultured .70% confluence. Differentiated hESC                                                                              T175 flasks 10-cm culture dishes removed Materials Methods <br />                                                                              surface treatment Trypsin-EDTA (Sigma-Aldrich), <br /> Materials                                                                    counted plated 96 plates coated 0.5% gelatin. <br />    Heat inactivated E. coli S. aureus prepared described <br /> previously [14]. Synthetic agonists TLR1/2 (Pam3CSK4),                   hESC-Derived Endothelial Cell (hESC-EC) Culture <br /> TLR2/6 (FSL-1), TLR3 [poly C)], TLR5 (Flagellin), TLR7                        Undifferentiated H7 hESC dissociated clumps (Imiquimod), TLR8 (E. coli K12 ssRNA) purchased                    placed ultra low-attachment plates medium containing 2% <br /> InvivoGen  (San Diego, USA). TLR4 ligand (LPS)                        FCS (Endothelial Growth Medium-2, Lonza). described <br /> obtained Sigma-Aldrich (Dorset, UK). 1 purchased                 [16], CD31+ cells sorted sterile cell sorter R<span id='am-21' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;D Systems (Abingdon, UK). reagents, unless                  (BD FACSAria, BD Biosciences) cultures 13 days stated, obtained Invitrogen.                             differentiation propagated endothelial growth medium. <br />                                                                              Passages 3 10 used experiments. <br /> Human Embryonic Stem Cell Culture <br />     experiments performed using H7 line            Matrigel Tubule-Forming Assay <br /> Geron Corporation&#195;&#376; (Menlo Park, CA, USA).  gene                         Matrigel diluted 1:2 endothelial basal medium ice <br /> array analysis undifferentiated stem cells performed H7,          100 ml added 24 plates allowed gel SHEF2, SHEF4 SHEF5 lines. H7 SHEF lines                          layer 37uC. CD31+ cells (50.000  seeded <br /> ethically derived hESC lines. H7 imported                        gels tubes photographed 22 hours. <br /> collaboration agreement Geron, permission UK Stem Cell Bank.                                                       Endothelial Cell Culture <br />     Undifferentiated H7 cells maintained feeder-cell                Primary human aortic endothelial cells purchased free conditions mouse embryonic fibroblast-conditioned medi-              Promocell (Heidelberg, Germany) cultured according um (MEF-CM), supplemented 8 ng/ml recombinant                        manufacturer&#226;&#8364;&#8482;s instructions. human endothelial cell line <br /> human basic fibroblast growth factor Geron&#226;&#8364;&#8482;s protocols                (EAhy-926) cultured Dulbecco&#226;&#8364;&#8482;s Modified Eagle&#226;&#8364;&#8482;s Medium <br /> described previously [15]. brief, mouse embryonic fibroblasts             (DMEM) supplemented 1% <span id='am-33' about='obo:IMR_0200173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00262" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Hypoxanthine" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200173" datatype="xsd:string"></span>Hypoxanthine</span>-Aminopterin- <br /> (MEFs) obtained pregnant mouse embryos MF-1                     <span id='am-38' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span>Thymidine</span>  Sigma-Aldrich), 5 mM <span id='am-28' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span>L-glutamine</span>, 100 U/ml <br /> strain. propagation 10% FCS-containing medium, MEFs                 penicillin, 100 mg/ml streptomycin, 10% heat-inactivated mitotically inactivated 0.01 mg/ml mitomycin C                  FCS. <br /> passage 4 2.5 hours adhered overnight pre- <br /> gelatinized T225 flasks  seeding density 1.886107 cells/             Cell Treatments <br /> flask) medium containing 10% FCS subsequently                      investigate response TLR stimulation, cells replaced 150 ml hESC medium, supplemented                       treated 96 plates PAMPS agonists TLR1- <br /> recombinant human 4 ng/ml basic fibroblast growth factor.                    8, heat killed S. aureus E. coli 1 used Conditioned medium, collected daily 10 days,                   positive control, 24 hours. experiments measure NFkB <br /> passed 0.2 mm low protein attachment cellulose acetate               activation cells treated 1 hour. <br /> filter units (Corning) prior feeding H7 cells. hESC medium <br /> consisted KnockOut DMEM (KO DMEM) supplemented                       Knockdown TLR5 <br /> 20% KnockOut serum replacement (KOSR), 1 mM L-glutamine,                        TLR5 siRNA knockdown, TARGETplus SMART- <br /> 50 U/ml penicillin, 50 mg/ml streptomycin, 1% non-essential                  pool TLR5 siRNA transfection performed using DharmaFect <br /> amino acids (100x stock) 0.1 mM b-mercaptoethanol.                   reagent (100 nM, final incubation volume 100 ml) manufac- <br /> undifferentiated cells cultured Matrigel (BD Sciences)-              turer&#226;&#8364;&#8482;s instructions (Dharmacon, Thermo Scientific). Scrambled, <br /> coated 6 plates (Nunc, Roskilde, Denmark). induction             non-targeting siRNA (100 nM; Dharmacon, Thermo Scientific) differentiation, spontaneously differentiated cells removed          used negative controls. Fluorescent siGLO Red siRNA treatment collagenase 37uC 10 min. hESC                 indicator (100 nM; Dharmacon, Thermo Scientific) used colonies mechanically broken 5 ml pipette tip                optimization documentation transfection efficacy (.90%) cultured 4 days low attachment 6 plates (Nunc),             24&#226;&#8364;&#8220;48 hours. <br /> suspended differentiation medium form embryoid bodies. differentiation medium hESC medium                   Immunocytochemistry KOSR replaced 20% non-heat-                           Cell fixed 4% paraformaldehyde, permeabilized inactivated FCS. Embryoid bodies plated 0.5%                   0.2% Triton X-100, labeled primary antibodies anti- <br />  <br />  <br />        PLoS | www.plosone.org                                        2                             2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                     TLR Pathways Human hESC <br />  <br />  <br /> CD31 (PECAM-1, 1:100 dilution, Santa Cruz Biolegend), anti-              15 minutes 42uC 5 minutes 95uC inactivate/ <br /> CD34 (Abcam, 1:200), anti-myosin heavy chain (Ab15, 1:200,                  degrade RNA reverse transcriptase enzyme. 91 ml RNase- <br /> Abcam, Cambridge), anti-von Willebrand factor (1:100, Dako).                Free water added remaining cDNA. <br /> Primary antibodies detected Alexa 488- (Invitrogen)             Real Time-Polymerase Chain Reaction (RT-PCR) <br /> Alexa 647- (Invitrogen) conjugated secondary antibodies                 Array. RT-PCR preparations performed RT2- <br /> 1:400). DNA visualised DAPI (0.5 mg/ml; Sigma). DiI-               Profiler-PCR-Array kit (SuperArray) according manufacturer&#226;&#8364;&#8482;s <br /> labelled acetylated human low density lipoprotein (Ac-LDL)              protocols. brief, experimental cocktail purchased Invitrogen. Images acquired Zeiss Axio               containing 102 ml diluted strand cDNA, 1.275 ml <br /> Observer Z1 fluorescence microscopy.                                        SuperArray-RT2-SYBR-Green/ROX-MasterMix 1.173 ml <br />                                                                             RNase-Free water. 25 ml cocktail added Enzyme-Linked-ImmunoSorbent-Assays (ELISAs)                                 96 plate, containing forward reverse primers    Concentrations CXCL8/IL8 cell-free supernatants               TLR-related genes, housekeeping genes, human-genomic-DNA- <br /> measured using sandwich ELISA kits (R&amp;D Systems)                        contamination PCR controls. Samples amplified using <br /> calculated using 4-parameter-log-fit curves according manufac-           ABI 7500 Real-Time-PCR (Applied Biosystems, Foster <br /> turer&#226;&#8364;&#8482;s protocols.                                                          City, USA) 40 cycles 15 s 95uC, 30 s 55uC, 30 s                                                                             72uC. Relative levels gene expression (fold differences) Measurement NFkB activation                                              calculated according manufactures instructions. CT values <br />    Undifferentiated hESC cultured 6 plates 90%           35 higher considered indicating surfaces covered H7 colonies              undetectable levels expression. 3/3 2/3 runs produced incubated 1b (1 ng/ml) LPS (1 mg/ml).                CT. = 35 given gene, expression considered hour, cellular nuclear extracts prepared using               undetectable overall. <br /> commercially available nuclear extraction kit (Active Motif,                   Quantitative RT-PCR TLR5. quantifying mRNA <br /> Carlsbad, CA, USA) according manufacturer&#226;&#8364;&#8482;s protocols.               levels TLR5 undifferentiated differentiated hESC brief, cells washed, collected ice-cold PBS           cultures, real-time PCR analyses performed TaqMan <br /> presence phosphate inhibitors centrifuged 5006g               Gene Expression Assay (Hs01920773_s1, Applied Biosystems, <br /> 5 min. resultant pellets suspended excess hypotonic          CA). Human GAPDH Endogenous Control (FAM/MGB probe) <br /> buffer, treated detergent centrifuged briefly 14,0006g          used housekeeping control. PCR performed 30 sec. cytoplasmic fraction collected, nuclei        Rotor-Gene 3000 (Corbett Research) real-time PCR instrument lysed nuclear proteins dissolved cocktail lysis       relative expression determined. <br /> buffer protease inhibitor. Nuclear protein concentrations determined using Bradford assay subsequently analyzed            Statistical analysis NFkB activation using TransAMTM NFkB p65 transcrip-                   Data reported mean6 S.E. mean n experiments. <br /> tion factor assay kit (Active Motif), according manufacturer&#226;&#8364;&#8482;s       Data analyzed using way ANOVA followed Dunnett&#226;&#8364;&#8482;s <br /> instructions. assay based nuclear NF-kB p65 proteins             Multiple Comparison Test sample t-test normalized <br /> binding consensus NF-kB oligonucleotides fixed 96                data described respective legends. <br /> plates. brief, 10 mg nuclear proteins added incubated 1 h allow binding P65 consensus              Results <br /> oligonucleotides. presence resulting complex detected primary antibody. addition                     Phenotype human hESC differentiated include <br /> horseradish peroxidase conjugated secondary antibody, P65               endothelial-like phenotypes <br /> quantified spectrophotometry.                                               Undifferentiated hESC characteristic appearance                                                                             tightly packed cells colonies shown Fig. 1a b. H7 <br /> Gene-Expression Analysis                                                    line, obtained Geron Corp., Menlo Park CA, grown <br />    RNA Extraction. Total RNA isolated hESC using                   feeder-free conditions described previously [15] RNeasy kit (Qiagen, Hilden, Germany) according                       differentiated embryoid body formation 20% FCS. manufacturer&#226;&#8364;&#8482;s protocols. Spin-column samples centrifuged              days differentiation suspension cultures, embryoid bodies 95006g, 20uC 15 seconds unless stated  brief,           plated gelatinized surfaces continued 350&#226;&#8364;&#8220;600 ml lysis-Buffer RLT added cell-pellets                differentiate adherent cultures prolonged periods  4 <br /> homogenized passing 20-gauge needle 5&#226;&#8364;&#8220;10 times                 months). Figure 1C-F shows morphology cultures 1 3 addition equal volume 70% ethanol. solution            months differentiation, demonstrating emergence centrifuged spin-column DNase digestion using               variety features, including clusters beating cardiomyocytes <br /> DNase-Buffer RDD solution (Qiagen) room temperature 25               (video S1) vessel-like structures. Immunocytochemical staining <br /> minutes. Subsequently, RNA washed Buffer RW1                   known markers (Fig. 2) confirms presence cardiomyo- <br /> RPE centrifugation. final wash Buffer RPE 2                cytes endothelial cells mixed population cells. <br /> minutes applied RNA eluted 30&#226;&#8364;&#8220;60 ml                   adjustment differentiation conditions using FACS <br /> RNase-free water. RNA concentration integrity                       sorting CD31 surface antigen, highly expandable population <br /> determined spectrophotometry (Spectramax, Switzerland)                human endothelial-like cells (hESC-EC) obtained obtain A260/A280 ratios later agarose gel electrophoresis.           hESC. Cells took cobblestone pattern culture characteristic <br />    Reverse transcriptase-PCR Strand reaction.                     endothelial cells (Fig. 3A). Cells stained positive strand-complimentary-DNA (cDNA) synthesized                  endothelial-specific CD31 CD34 markers (Fig. 3C&#226;&#8364;&#8220;D) 1 mg total RNA using RT2 Strand Kit (SuperArray                acetylated LDL uptake culture (Fig. 3B). indicating <br /> Bioscience Corporation, MD, USA) according manufacturer&#226;&#8364;&#8482;s                endothelial phenotype function, cells formed tube-like <br /> protocols. Briefly, 10 ml rt-Cocktail, containing RNA mixed          structures solidified Matrigel showed migration 10 ml Genomic-DNA-Elimination Mixture incubated                fibronectin surface wound healing assays (Fig. 3E&#226;&#8364;&#8220;F). <br />  <br />  <br />       PLoS | www.plosone.org                                        3                             2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                     TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 1. Appearance hESC cultures. Undifferentiated H7 cells   b); 1 month (c, d) 3 months (e, f) differentiation. Examples clusters beating cells seen c d, shown Video S1. Vessel-like structures observed (e). <br /> doi:10.1371/journal.pone.0010501.g001 <br />  <br />  <br />  <br /> Relative expression TLRs related genes hESC                      differentiation. Fig. 4A shows expression levels TLRs 1, 3, endothelial cells                                                   4, 5 6 undifferentiated H7 cells 3 SHEF <br />   Expression TLRs downstream signaling effector genes               lines. TLR 1, 3, 4 6 expression consistently lower hESC determined undifferentiated hESC 1&#226;&#8364;&#8220;4 months              compared endothelial cells, TLRs 1 4 particularly low. <br />  <br />  <br />        PLoS | www.plosone.org                                      4                              2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                              TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 2. Presence cardiomyocytes endothelial cells differentiated hESC cultures. Immunocytochemical staining clusters 1 <br /> month differentiated H7 hESC showing   cardiomyocytes myosin heavy chain   b) (green) corresponding brightfield image (b) <br /> endothelial cells identified von Willebrand factor (green), CD31 (red) DAPI (nuclei, blue). <br /> doi:10.1371/journal.pone.0010501.g002 <br />  <br />  <br />  <br />  <br />       PLoS | www.plosone.org                                   5                            2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                          TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 3. Characteristics human embryonic stem cell-derived endothelial cells. hESC-EC cultures showed   cobblestone <br /> morphology. (B) Cells showed DiI-labelled acetylated LDL uptake, stained positive human anti-CD31 antibody (green; C), anti-CD34 <br /> (red; D). DAPI (blue) used nuclear staining. (E) Cells plated solidified Matrigel form hollow-like tubes. (F) wound healing assay, cells migration fibronectin surface (upper panel showing cell free area 0 hours; cells migrate wound site 22 hours). <br /> doi:10.1371/journal.pone.0010501.g003 <br />  <br />  <br /> Levels TLR8, TLR10 undetectable hESC (Cycle                     downstream signalling pathways, majority NFkB <br /> threshold, CT, values 35 lower majority samples                (Fig. 4B) TLR signaling genes (Fig. 4C) expressed (Table S1)). Levels TLR2 similar higher            similar levels endothelial cells hESC, reduced <br /> TLR6 H7 SHEF lines, CT values range                    expression MyD88 TICAM1 contribute 27&#226;&#8364;&#8220;30 (Table S1), levels undetectable endothelial               poor functional response.  reasonable <br /> cells majority samples fold-change calculated.             agreement H7 SHEF lines. <br /> Low levels expression TLR7 TLR9 present hESC undetectable endothelial cells. Interestingly, expres-             Relative expression TLRs related genes following <br /> sion TLR5 robust endothelial cells higher                differentiation hESC <br /> undifferentiated hESC. Agreement good H7                          Expression levels determined 1, 3 4 months SHEF lines TLR expression. Consistent activity             differentiation H7 hESC. Compared undifferentiated H7 <br />  <br />  <br />        PLoS | www.plosone.org                                          6                               2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                         TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 4. Expression TLR TOLL NFkB signalling genes undifferentiated hESC. Relative expression levels undifferentiated H7 <br /> hESC (n = 3, open bars) 3 SHEF lines (grey bars) compared human aortic endothelial cells (HAEC, n = 3).  TLR genes. TLRs 2 7&#226;&#8364;&#8220;10 undetectable majority HAEC. B: NFkB related genes; C: TOLL signalling-related genes. Statistical significance calculated using  <br /> sample t-test averaged H7 values 1.0, *P,0.05, **P,0.01, ***P,0.001, n = 3. <br /> doi:10.1371/journal.pone.0010501.g004 <br />  <br />  <br /> (Fig. 5A), general increase time TLRs 1, 2, 3, 4,         endothelial cells (Fig. 5B) clear TLR1 TLR4 5 6, undifferentiated levels 4 months. Low              low levels, 4 months differentiation. levels TLRs 7, 9 10 sporadically apparent                  modest adjustments expression level NFkB (Table 1) <br /> (Table S1).  comparing differentiated hESC                       levels seen differentiation components  <br />  <br />        PLoS | www.plosone.org                                         7                               2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                                        TLR Pathways Human hESC <br />  <br />  <br />                                                                                         TLR signaling pathways (TICAM1, TICAM2 IRAK1) <br />                                                                                         consistently increased (Table 2). list gene expression <br />                                                                                         changes differentiation Table S2. <br />  <br />                                                                                         Effect PAMPs TLR1-8 bacteria CXCL8 <br />                                                                                         release human hESC primary human endothelial <br />                                                                                         cells <br />                                                                                            Release CXCL8 used study biomarker cell <br />                                                                                         activation. undifferentiated hESC statistically <br />                                                                                         significant increase CXCL8 release cells stimulated 24 h <br />                                                                                         E. coli, S.Aureus array PAMPs TLRs1, <br />                                                                                         2, 3, 4, 6, 7 8 (Fig. 6A). compared                                                                                         differentiated hESC cultures, phenotypic evidence                                                                                         ESC specialization established (Fig. 2). month                                                                                         initiation differentiation, hESC-derived cells did release <br />                                                                                         increased levels CXCL8 response PAMPs TLRs                                                                                         bacteria (Fig. 6B). exception flagellin, activator <br />                                                                                         TLR5, produced increase CXCL8                                                                                         undifferentiated hESC differentiated hESC-derived cells <br />                                                                                         (Fig. 6A B). Remarkably, 4 months differentiation, <br />                                                                                         TLR5 remained PAMP produce response hESC- <br />                                                                                         derived cells (Fig. 6C). 1b, acts independently TLRs <br />                                                                                         TLR adapter protein MyD88, activated undifferen- <br />                                                                                         tiated differentiated hESC release CXCL8 (Fig. 6A&#226;&#8364;&#8220;C).                                                                                         separate experiments protocols repeated compare directly <br />                                                                                         responses purified cultures hESC-EC mixed differenti- <br />                                                                                         ated hESC cultures (Fig. 7A&#226;&#8364;&#8220;B). Similar results seen                                                                                         presented Fig. 6. LPS effect CXCL8 release mixed <br />                                                                                         differentiated cultures hESCs (Fig. 7A) purified hESC-ECs <br />                                                                                         (Fig. 7B). flagellin 1b increased CXCL-8 release                                                                                         experiments (Fig. 7A&#226;&#8364;&#8220;B). Purified hESC-EC expressed <br />                                                                                         TLR5, showing 3-fold increase mRNA levels compared <br />                                                                                         undifferentiated hESC (data shown). contrast <br />                                                                                         results obtained hESC differentiated derivatives, <br /> Figure 5. Expression TLR genes differentiated hESC. <br />                                                                                         primary cultures adult human aortic endothelial cells released <br /> Relative expression differentiated H7 hESC 1 month (n = 2, open                   increased levels CXCL8 response Gram negative bacteria <br /> bar), 3 months (grey bar) 4 months (solid bar) compared                        LPS  activate TLR4), FSL-1  activates averaged undifferentiated H7 hESC (n = 3) B: human aortic                           TLR2/6 heterodimer) 1b (Fig. 6D). Similar results seen <br /> endothelial cells (HAEC, n = 3).                                                        endothelial cell line, EAhy-926, used (data doi:10.1371/journal.pone.0010501.g005                                                   shown). ensure differences CXCL8 release                                                                                         hESC primary endothelial cells different <br />  <br />                                                                                          Table 2. TOLL signalling related gene expression hESC. <br />  Table 1. NFkB related gene expression hESC. <br />                                                                                          Table 2             Differentiated: fold change vs undifferentiated <br />  <br />  Table 1             Differentiated: fold change vs undifferentiated                     Gene           1 month         1 month        3 month     4 month <br />  <br />  Gene           1 month         1 month        3 month     4 month                  BTK                 n.d.d           n.d.d          n.d.d       n.d.d <br />  <br />  NFKB1               1.66            1.92           0.38        0.85                     MYD88               3.82            0.89           0.47        0.89 <br />  NFKB2               2.41            2.28           0.63        2.01                     TICAM2              2.84            3.23           1.18        3.10 <br />  NFKBIA              1.12            0.68           0.26        1.09                     TIRAP               1.56            1.12           0.37        0.87 <br />  NFKBIL1             1.21            0.91           0.47        1.28                     TOLLIP              1.42            1.62           1.74        1.14 <br />  NFRKB               1.61            0.83           0.28        0.91                     TRAF6               2.08            1.76           1.00        1.41 <br />  c-REL               0.75            0.41           0.12        0.76                     TICAM1              3.65            5.53           3.12        3.80 <br />  NFKB3               6.61            2.98           0.86        2.05                     IRAK1               1.34            4.23           1.50        1.84 <br />  IKBKB               0.77            0.41           0.31        0.60                     IRAK2               0.75            0.66           0.26        1.01 <br />  <br />  Fold change differentiated H7 hESC 1&#226;&#8364;&#8220;4 months compared average                 Fold change differentiated H7 hESC 1&#226;&#8364;&#8220;4 months compared average <br />  (n = 3) undifferentiated H7, value 1 represents equivalent gene              (n = 3) undifferentiated H7, value 1 represents equivalent gene <br />  expression 1 represents lower expression differentiated        expression 1 represents lower expression differentiated  undifferentiated.                                                                       undifferentiated. N.d.d = detectable differentiated cells. <br />  doi:10.1371/journal.pone.0010501.t001                                                   doi:10.1371/journal.pone.0010501.t002 <br />  <br />  <br />  <br />          PLoS | www.plosone.org                                                 8                                  2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                        TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br /> Figure 6. Response hESC primary cells bacteria PAMPs. Release CXCL8 undifferentiated  , 1 month- (B) 4 month- <br /> (C) differentiated H7 hESC primary cultures human aortic endothelial cells (D). Cells treated Gram negative E. coli (C, 108 CFU/ <br /> ml), Gram positive S. aureus (SA, 108 CFU/ml), PAMPs TLR2/1 (PAM3CSK4; 300 ng/ml), TLR2/6 (FSL-1; 1 mg/ml), TLR3 (Poly:IC; 10 mg/ml), TLR4 <br /> (LPS; 1 mg/ml), TLR5 (flagellin; 100 ng/ml), TLR7 (imiquimod; 10 mg/ml) TLR8 (E. coli 12 ssRNA; 10 mg/ml) 1b  1; 1 ng/ml) 24 hours CXCL8 measured using ELISA. data mean 6 S.E. mean n = 8211  ; n = 329 (B); n = 3 (C) n = 426 (D). Statistical <br /> significance calculated way analysis variance followed Dunnett&#226;&#8364;&#8482;s Multiple Comparison Test  control). p value 0.05 considered statistically significant denoted *. <br /> doi:10.1371/journal.pone.0010501.g006 <br />  <br />  <br /> media used routine culture, undifferentiated                    Knockdown TLR5 confirms specificity flagellin <br /> differentiated hESC tested DMEM plus 10%                      response <br /> FCS used EAhy-926 endothelial cell culture (Fig. S1). Use                Silencing TLR5 using siRNA resulted significantly reduced EAhy-926 media altered basal CXCL8 levels extent                 CXCL8 release flagellin-treated mixed differentiated hESC cultures did reveal effect bacterial PAMP stimulation             (Fig. 7A) purified hESC-EC (Fig. 7B) compared scrambled <br /> CXCL8 release.  noted effects               siRNA. Interestingly, TLR5 siRNA knockdown general <br /> long term culture highly specialized medium influence            depressant effect CXCL8 release mixed differentiated cultures, <br /> TLR signaling pathways cells.                                   reduction specific flagellin hESC-EC. <br />  <br />  <br />        PLoS | www.plosone.org                                        9                               2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                           TLR Pathways Human hESC <br />  <br />  <br />  <br />  <br />                                                                                  Figure 8. Activation NFkB undifferentiated hESC.  , <br /> Figure 7. Enzyme-linked immunosorbent assay CXCL8                             Undifferentiated H7 hESC activated release CXCL8 production hESC purified hESC-derived endothelial                         24 hours stimulation TNFa (TNF), INFc (INF) 1b  1) cells. Bar graphs showing release CXCL8 1 month-old                      6 (10 ng/ml (n = 6). (B), NFkB activated differentiated H7 hESC cells   hESC-derived CD31+ endothelial               undifferentiated H7 hESC following 1 hour stimulation 1b (1 ng/ <br /> cells (B) transfected scrambled siRNA (open bars) TLR5 siRNA             ml) LPS (1 mg/ml) (n = 3). Statistical significance (solid bars) 48 hours. Transfected cells treated TLR4 (LPS;        calculated way analysis variance followed Dunnett&#226;&#8364;&#8482;s <br /> 1 mg/ml), 1b  1b; 1 ng/ml), TLR5 (flagellin; 100 ng/ml)             Multiple Comparison Test  control). **P,0.01. <br /> 24 hours CXCL8 measured. data mean 6 S.E.                 doi:10.1371/journal.pone.0010501.g008 <br /> mean n = 3. # P,0.05 vs control; * P,0.05 *** P,0.001 vs <br /> scrambled siRNA group. <br /> doi:10.1371/journal.pone.0010501.g007                                            results [17,18,19] NFkB genes functionally <br />                                                                                  active undifferentiated hESC TLR4 PAMP unable                                                                                  produce NFkB translocation cells. <br /> Effect cytokines TLR4 PAMP LPS cell NFkB activation hESC                                                          Discussion <br />    investigate underlying mechanism lack TLR responses hESC, function downstream pathways                          primary findings present study  investigated. Effects 1b CXCL8 release                           undifferentiated hESC immune response associated undifferentiated hESC compared produced                        bacterial TLR activation, range bacterial PAMPs TNFa, INFc 6. cells activated                   TLR5. surprisingly, cultures hESC differentiated release increased levels CXCL8 1b, TNFa, INFc                   4 months detectable immune response TLR2 6 (Fig. 8A). line observations  NFkB                     (Gram positive) TLR4 (Gram negative) activation. TLR4 <br /> activated undifferentiated hESCs following 1 h stimulation               ligand, LPS failed activate NFkB hESCs, consistent 1b whilst treatment cells LPS period                 notion cells functionally active TLR effect (Fig. 8B). Clearly, evidenced data, confirm               responses. population differentiated hESCs enriched  <br />  <br />        PLoS | www.plosone.org                                           10                              2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                      TLR Pathways Human hESC <br />  <br />  <br /> endothelial-like cells (hESC-EC) shared lack response                   lack response bacterial challenge TLR4 active response TLR5.                              surprising undifferentiated hESC, given delay    HESC directly compared endothelial cells cultured              development innate immune sensing embryo. TLR adult tissue, shown display positive                 responses generally develop near bacterial-TLR response assessed increased CXCL8 release               term. results agreement [20] cells stimulated agonists TLR2 TLR4. Gene              reported lack responsiveness undifferentiated mouse ESC expression profiles undifferentiated differentiated             differentiated derivatives LPS. authors hESC revealed relatively low levels TLR1 TLR4 hESC                 reported lack TLR4 expression cells compared levels expressed endothelial cells.  TLR5           epigenetic modulation TLR4 gene promoter (methylated). <br /> expression relatively high differentiated hESC compared            recently study Lee workers [21] <br /> endothelial cells. Whilst gene expression data               demonstrate positive expression TLRs mouse ESCs. <br /> interpreted caution, pattern TLR expression fits          authors demonstrated long term ability hESC respond PAMPs selective            exposure (24 days) increased proliferation differentiation TLRs. noted TLR2 acts dimer              mouse ESCs seen cells stimulated LPS (TLR4) TLR1 TLR6. Loss TLR1 indirectly                       POLY:IC (TLR3). apparently conflicting observations influence activation TLR2.                                      mouse ESCs likely influenced different culture conditions, <br />    TLRs linked TIR domains adapter protein pathways              time LPS exposure, epigenetic factors differing clones MyD88. findings hESC purified hESC-               ESCs used. likely similarities differences human <br /> EC responded 1b, agonist works                       mouse ESC cells respects remain subject <br /> independently TLRs TIR (TLR IL1 receptor) domain               investigation. Clearly data shows hESC resistant linked receptor, releasing increased levels CXCL8                 stimulation PAMPs flagellin. hESC <br /> activation NFkB. HESCs express number                 purified hESC-EC expressed TLR5 knockdown NFkB related genes. levels compared                 TLR5 siRNA inhibited CXCL8 production response endothelial cells relative expression NFkB2, NFkB1A,                 flagellin cultures. suggests TLR5 acts active <br /> NFKNFRkB, REL IKBkB similar hESC.                       sensor bacterial flagellin monomers hESC. Evidence expression levels NFkB1 (gene p50) RELA (gene p65)            cytoprotective role TLR5 comes studies showing lower hESC endothelial cells. lower gene                TLR5 ligation block apoptosis activating downstream <br /> expression clearly did limit NFkB activation CXCL8                   antiapoptotic genes [22]. Activation TLR5 protect induction. interesting note  addition 1b,           tissue injury conditions involving high levels cell death [23]. <br /> hESCs responded releasing increased levels CXCL8                    mechanism responsible lack LPS responsiveness <br /> stimulated cytokines including TNFa, INFc,               hESC determined.  observed 6. data suggests whilst hESC hESC-EC             mesoderm formation embryoid bodies (EB) hESC able sense respond bacterial PAMPs LPS,               inhibited  shown mesoderm specific gene, Brachury clearly intact active inflammatory signaling pathways.            silencing), challenge LPS, TLR4 PAMP [24]. Gene expression analysis adapter proteins related              suggests low TLR4 levels able produce genes showed differences hESC adult                    sufficient response affect differentiation processes pro- <br /> endothelial cells. Whilst levels TIRAP, TOLLIP TRAF6                 longed stimulation, results secondary LPS similar undifferentiated hESC adult endothelial cells,           release cytokines MEF layer present EB levels BTK, MyD88, TICAM1, TICAM2 IRAK2                     formation study. <br /> significantly lower. Interestingly pattern gene expression              Lack innate immune danger-sensing signals different <br /> changed somewhat hESC differentiated, upregu-                implications different cell lineages depending  like <br /> lation TICAM1, TICAM2 IRAK1. Changes expression                   endothelium, distinct role pathogens  like NFkB-related genes differentiation modest            cardiac myocytes, pathways involved directed <br /> consistent, RELA upregulated REL                     general danger signals. Endothelial cells reduced (Table 2).  functional levels MyD88                  provide barrier endocrine functions essential <br /> associated signaling proteins clearly sufficient               innate immune surveillance cells.  endothelial cells differentiated undifferentiated hESC mount robust                  generally cell type pathogens encounter inflammatory response 1b.                                             circulation. target cells, including endothelial cells,    NFkB signaling differentiating hESC somewhat                 essential tissue derived hESCs express functional <br /> controversial, study hES-NCL1 line showing                    bacterial innate immune response. use hESC-derived <br /> expression NFkB pathway components significant levels             endothelial cells tissue repair compromised undifferentiated cells regulating differentiation        lack innate immune sensing. remains seen possibly controlling differentiation process way            vivo environment stimulate maturation [19]. contrast, Kang et al [17] low NFkB                   hESC-derived cells   mix host grafted <br /> undifferentiated SNUhES3 MizES4 hESC lines compared                  endothelium tolerable maintain function. HEK haematopoetic progenitor lines  MyD88                  necessary use pre-treatment strategies accelerate maturation <br /> TRAF2 equivalent), poor CXCL8 induction                  vitro mechanical hormonal stimulation, provision TNFa. Differentiation associated increased NFkB                extracellular matrix culture cell types. cell <br /> IL8 response TNFa, finding replicated mouse ESC            types directly involved immune sensing, consequences [18]. contradictory results studies suggest              relative insensitivity insult positive negative <br /> variation hESC lines, seen robust similar           aspects. Cell therapy  cases, directed areas expression levels key NFkB pathway components various             damage implanted hESC-derived cells introduced undifferentiated hESC (H7 SHEF lines), little         areas hypoxia inflammation. Taking example cardiac <br /> change differentiation H7.                                          myocytes, lack TLR2 4 response predicted  <br />  <br />       PLoS | www.plosone.org                                       11                              2010 | Volume 5 | Issue 5 | e10501 <br />                                                                                                                                                 TLR Pathways Human hESC <br />  <br />  <br /> increase resistance hypoxia [10] improve survival                             Table S1 CT values human aortic endothelial cells (HAEC, <br /> implantation scar border zone, sensitivity                             n = 3), undifferentiated H7 hESC (n = 3); undifferentiated SHEF2, <br /> inflammatory cytokine milieu infarcted heart                                   SHEF4 SHEF5 differentiated H7 1 month (n = 2), 3 <br /> retained. suggest implantation success                                    months 4 months differentiation. Values .35 set optimal later time period infarction, acute                                   35: considered undetectable. <br /> inflammation subsided scar established.                                        doi:10.1371/journal.pone.0010501.s002 (0.36 MB <br />                                                                                               DOC) <br /> Summary                                                                                       Table S2 Fold changes undifferentiated H7 hESC <br />   summary, shown time hESC                               (n = 3) differentiated H7 1 month (n = 2), 3 months 4 <br /> sense respond bacteria bacterial PAMPs activate                             months differentiation. N.d.d. = detectable differen- <br /> TLR4, respond flagellin activates TLR5.                               tiated; n.d.u. = detectable 3/3 2/3 undifferentiated pattern PAMP sensing consistent relative                             n.d. = detectable  <br /> expression TLR genes hESCs. despite having                                  doi:10.1371/journal.pone.0010501.s003 (0.14 MB ability sense LPS, hESC respond robust manner                                   DOC) <br /> cytokines linked MyD88 NFkB transcription pathways. <br />                                                                                               Video S1 1 month differentiated H7 hESC showing clusters observations important suggest whilst                                  beating cardiomyocytes, shown Fig. 1C D. <br /> endothelial   cells produced hESC display                                   doi:10.1371/journal.pone.0010501.s004 (2.26 MB <br /> phenotypic markers express mature immune                                        WMV) <br /> function. implications strategy timing implantation hESC derived cells tissue repair. <br />                                                         <br /> </body></html>