<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Arabidopsis FAB1/PIKfyve Proteins Essential Development Viable Pollen[W] <br /> Paul Whitley1, Steven Hinz, James Doughty1* <br /> Department Biology Biochemistry, University Bath, Bath, Somerset BA2 7AY, United Kingdom <br />  <br />  <br /> <span id='am-37' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-38' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span>Phosphatidylinositol</span> 3,5-bisphosphate [PtdIns(3,5)P2] <span id='am-1' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span>phospholipid</span> role controlling membrane trafficking <br /> events yeast animal cells. function <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span> plants unknown, synthesis shown regulated osmotic stress plant cells. PtdIns(3,5)P2 synthesized PIKfyve/Fab1 family proteins, orthologs, FAB1A FAB1B, present Arabidopsis (Arabidopsis thaliana). study, attempt address role lipid analyzing phenotypes plants mutated FAB1A FAB1B. possible generate plants <br /> homozygous mutations genes, single mutants isolated. homozygous single mutant plant lines <br /> exhibited leaf curl phenotype marked FAB1B mutants. Genetic transmission analysis revealed failure generate double mutant lines entirely inviability pollen carrying mutant alleles FAB1A FAB1B. pollen displayed severe defects vacuolar reorganization following mitotic division development. presence abnormally large vacuoles pollen tricellular stage resulted collapse majority grains carrying mutant alleles. demonstrates crucial role PtdIns(3,5)P2 modulating dynamics vacuolar rearrangement <br /> essential successful pollen development. Taken  results consistent PtdIns(3,5)P2 production central cellular responses changes osmotic conditions. <br />  <br />  <br />  <br />    Phosphoinositides make minor fraction total                    cells addition cellular signaling events (Backer, <br /> membrane lipids eukaryotic organisms.                        2008). <br /> production spatially restricted cytoplasmic                        plants, PtdIns3P essential normal growth <br /> leaflet specific organellar membranes tempo-                       development. Arabidopsis (Arabidopsis thaliana) <br /> rally regulated phosphatidylinositol (PtdIns) ki-                      plants carrying VPS34 antisense construct nases phosphatases. hydoxyl                         severe developmental defects (Welters et al., 1994). <br /> groups PtdIns phosphorylated, singly                     Furthermore, using pharmacological inhibitors combinatorially, produce seven different phos-                      PtdIns3P production analysis transgenic plants <br /> phoinositides. different phosphoinositides                          defective downstream signaling PtdIns3P, recruit activate proteins specific                        shown lipid role play <span id='am-12' about='obo:IMR_0001650' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-13' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="inositol phospholipid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001650" datatype="xsd:string"></span>phosphoinositide</span>-binding domains                            diverse physiological processes, root <br /> implicated regulation important cellu-                     hair growth (Lee et al., 2008a). phenotypes ob- <br /> lar functions, including membrane trafficking, cell                       served studies PtdIns3P function plants growth, cytoskeleton remodeling (Di Paolo                         consistent role endosomal vacuolar Camilli, 2006).                                                        trafficking plants (Kim et al., 2001; Lee et al., <br />    animal cells, phosphorylation 3 position                  2008a), eukaryotes. Recently, attempt <br /> PtdIns phosphorylated derivatives                          generate vps34 homozygous mutant plant lines carried different classes PtdIns 3-kinase                 unsuccessful failure mutant vps34 allele <br /> (classes III; Cantley, 2002). Plants yeast                     transmit male germ line (Lee et al., class III PtdIns 3-kinases orthologs                     2008b). Saccharomyces cerevisiae protein Vps34p (Mueller-                        Importantly, PtdIns3P precursor phos- <br /> Roeber Pical, 2002). Vps34p orthologs thought                     phoinositide, PtdIns 3,5-bisphosphate [PtdIns(3,5)P2], use PtdIns sole lipid substrate produce                   vital roles endosomal trafficking PtdIns 3-phosphate (PtdIns3P). PtdIns3P involved                       eukaryotes (Dove et al., 2009).  possible endosomal/lysosomal protein sorting eukaryotic                      effects plants attributed PtdIns3P                                                                           previous studies actually inability   1 <br />                                                                           cells produce PtdIns(3,5)P2. PtdIns(3,5)P2 pro- <br />      authors contributed equally article.                    duced PtdIns3P 5-kinases PIKfyve Fab1p    * Corresponding author; e-mail bssjd@bath.ac.uk. <br />                                                                           animal yeast cells, respectively. PIKfyve/Fab1p <br />    author responsible distribution materials integral findings presented article accordance policy          proteins N-terminal FYVE domain necessary <br /> described Instructions Authors (www.plantphysiol.org)       binding PtdIns3P-containing membranes, cen- <br /> James Doughty (bssjd@bath.ac.uk).                                         tral Cpn60_TCP1  HSP chaperonin T complex 1) <br />    [W] <br />        online version article contains Web data.         homology domain, C-terminal kinase domain. <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.109.146159                     Arabidopsis, number genes encoding <br /> 1812     Plant Physiology&#195;&#8217;, December 2009, Vol. 151, pp. 1812&#226;&#8364;&#8220;1822, www.plantphysiol.org &#195;&#8220; 2009 American Society Plant Biologists <br />                                                                            Role PtdIns(3)P 5-Kinase Pollen Development <br />  <br />  <br /> putative Fab1p homologs,                  single copy genes encoding PIKfyve/Fab1p. <br /> FAB1A (At4g33240) FAB1B (At3g14270), encode                genes encoding putative PIKfyve/ <br /> proteins having FYVE domains N termini                Fab1p homologs Arabidopsis At4g33240 <br /> (Mueller-Roeber Pical, 2002). likely      (FAB1A), At3g14270 (FAB1B), At1g71010 (FAB1C), <br /> proteins PtdIns3P 5-kinases Arabidopsis.             At1g34260 (FAB1D).  FAB1A spite importance PtdIns(3,5)P2 yeast             FAB1B encode proteins contain predicted FYVE <br /> animals, little known function            domains N termini plants.  shown hyperosmotic           domain organization PIKfyve/Fab1p (Fig. <br /> stress induce rapid synthesis PtdIns(3,5)P2         1B). Phylogenetic analysis indicates proteins sim- cell suspension cultures number plant             ilar FAB1C FAB1D, FYVE domain, species (Meijer Munnik, 2003) pollen tubes          likely exist higher plant species tobacco (Nicotiana tabacum; Zonia Munnik,             proteins cluster plant-specific group distinct 2004). production consistent require-           plant FYVE domain-containing proteins (Fig. 1C). ment PtdIns(3,5)P2 vacuolar membrane reorga-            propose FAB1A FAB1B encode bona fide <br /> nization, water moves vacuole               PtdIns3P 5-kinases, role FAB1C cytosol cells placed hyperosmotic             FAB1D production PtdIns(3,5)P2 stress. animal cells, defective PtdIns(3,5)P2 produc-       debatable. FAB1C FAB1D predicted encode <br /> tion leads cytoplasmic vacuolation endosome-             polypeptides PtdIns3P 5-kinase activity, derived membranes (Ikonomov et al., 2001; Jefferies            lack FYVE domain, unlikely et al., 2008). general require-       efficiently targeted PtdIns3P substrate-containing <br /> ment eukaryotes PtdIns(3,5)P2 production         membranes. <br /> endomembrane remodeling. remodeling                 transcript levels FAB1A FAB1B mediated proteins bind PtdIns(3,5)P2.             investigated wide range tissue types utilizing <br /> number candidates identified, including           publicly available microarray data GENE- <br /> yeast Svp1p (Dove et al., 2004), mammalian homo-           VESTIGATOR software (Zimmermann et al., 2004). <br /> log WIP149 (Jeffries et al., 2004), CHMP3 (Whitley             FAB1A FAB1B transcripts plant <br /> et al., 2003), Ent3p (Friant et al., 2003).                tissues varying degrees (Supplemental Fig. S1). <br />    study, aimed investigate          FAB1A showed highest expression levels pollen, <br /> role PtdIns(3,5)P2 plant physiology              seed, senescent leaves, FAB1B transcript <br /> function PIKfyve/Fab1p orthologs Arabidopsis             levels highest root hair zone, pollen, generating mutant plant lines homozygous                stamens. microarray data reveal FAB1A ex- <br /> T-DNA insertions FAB1A FAB1B. failed            pression levels significantly lower FAB1B generate double homozygous fab1a/fab1b knockout             tissue categories (Supplemental Fig. S1). <br /> plants observed subtle phenotypes fab1a fab1b single homozygous plants. data              Isolation FAB1A FAB1B Knockout Lines pollen fab1a/fab1b genotype abnor- <br /> mal vacuolar phenotype does contribute             order obtain Arabidopsis lines mutations generation. data consistent              FAB1A FAB1B, SALK T-DNA lines carrying <br /> hypothesis male gametophytic defect ob-               insertions genes identified. number served vps34 mutant pollen (Lee et al., 2008b)           lines T-DNA insertions genes inability pollen generate PtdIns(3,5)P2   identified. SALK_013923 SALK_066673 cho- direct result lack PtdIns3P.            sen investigation contained inser- <br />                                                                tions exons FAB1A (At4g33240; exon 4 11)                                                                FAB1B (At3g14270; exon 9 11), respectively (Fig. 2A). <br />                                                                predicted alleles ( fab1a-1 fab1b-1) <br /> RESULTS                                                        unable produce functional protein Phylogeny Expression PtdIns3P 5-Kinases                 T-DNA insertions positioned 5# lipid Arabidopsis                                                 kinase domain. PCR-based genotyping confirmed <br />                                                                T-DNA insertions FAB1A FAB1B. Homozygous <br />    availability complete Arabidopsis ge-            fab1a-1 fab1b-1 mutant plant lines generated <br /> nome sequence allowed identification                stocks checked transcripts enzymes involved <span id='am-24' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-25' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span>phosphoinositide</span> metabolism                reverse transcription (RT)-PCR. FAB1A FAB1B <br /> based comparison wealth data               transcripts detected fab1a-1/fab1a-1 studies animals yeast  review, Mueller-         fab1b-1/fab1b-1 plants, respectively, present <br /> Roeber Pical, 2002). proposed             wild-type plants (Supplemental Fig. S2). proteins Arabidopsis putative       noted primers used amplify cDNA homologs PIKfyve Fab1p, enzymes respon-             RT-PCR 3# T-DNA insertion sites;  <br /> sible conversion PtdIns3P PtdIns(3,5)P2          short transcripts coding truncated proteins (Fig. 1A), animals yeast, respectively (Mueller-        potentially produced homozygous T-DNA <br /> Roeber Pical, 2002). Animals yeast contain           insertion lines.  case, <br /> Plant Physiol. Vol. 151, 2009                                                                                         1813 <br />  Whitley et al. <br />  <br />  <br />  <br />  <br />           Figure 1. Arabidopsis possesses genes encoding FYVE domain-containing protein orthologs PIKfyve/Fab1.  Production <br />           PtdIns(3,5)P2 PtdIns requires sequential action enzymes, PtdIns 3-kinase (VPS34) PtdIns3P 5-kinase <br />           (PIKfyve/Fab1). FAB1 phosphorylates D-5 position phosphoinoside ring. B C, Arabidopsis genes, FAB1A <br />           FAB1B, encoding proteins similar domain structure PIKfyve/Fab1 proteins. PIKfyve/Fab1 proteins           identified broad range species (C) appear ubiquitous eukaryotes. Arabidopsis genes, FAB1C           FAB1D, encoding proteins lack PtdIns3P-binding FYVE domain (B). FYVE-domain-lacking proteins appear           plant lineages (C). Species prefixes C follows: Sp, Schizosaccharomyces pombe; Af, Aspergillus fumigatus; <br />           Kl, Kluyveromyces lactis; Sc, Saccharomyces cerevisiae; Dd, Dictyostelium discoideum; Cr, Chlamydomonas reinhardtii; Os, <br />           Oryza sativa; Vv, Vitis vinifera;  Arabidopsis thaliana; Pt, Populus trichocarpa; Pp, Physcomitrella patens; Mt, Medicago <br />           truncatula; Dm, Drosophila melanogaster; Dr, Danio rerio; Mm, Mus musculus; Hs, Homo sapiens. <br />  <br />  <br />  <br /> truncated proteins lack C-terminal lipid                          greater 95% soil greater 98% 13 <br /> kinase domain unable convert PtdIns3P                        Murashige Skoog medium. mutant lines PtdIns(3,5)P2.  consider fab1a-1/                       appeared healthy developed displaying <br /> fab1a-1 fab1b-1/fab1b-1 plants null mutants                    gross morphological defects.  4 weeks <br /> FAB1A FAB1B, respectively.                                              postgermination, noted leaves                                                                             fab1a-1/fab1a-1 fab1b-1/fab1b-1 mutant lines ap- <br /> fab1a-1 fab1b-1 Lines Display Subtle Leaf-Curling                       peared different wild-type plants. <br /> Phenotypes Reproductively Viable                                    Leaves mutant plants noticeably curled                                                                             margins (Fig. 2B). quantitative analysis leaf <br />   Seeds fab1a-1/fab1a-1 fab1b-1/fab1b-1                            curling (Fig. 2C) revealed statistically <br /> plants exhibited normal germination rates fre-                          significant difference degree curling fab1b-1/ <br /> quencies. Germination frequencies lines                       fab1b-1 compared wild type (P = 0.027). significantly different wild type,                       similar trend data 1814                                                                                                              Plant Physiol. Vol. 151, 2009 <br />                                                                                           Role PtdIns(3)P 5-Kinase Pollen Development <br />  <br />  <br />                                                                            fab1a-1/fab1a-1, phenotype severe (P = <br />                                                                            0.066). fab1a-1/fab1a-1 fab1b-1/fab1b-1 plant <br />                                                                            lines fully reproductively viable, displaying seed <br />                                                                            set values indistinguishable wild type. <br />  <br />                                                                            fab1a-1/fab1a-1;fab1b-1/fab1b-1 Double Mutants                                                                            Generated <br />  <br />                                                                               ubiquitous expression profile FAB1A <br />                                                                            FAB1B lack severe developmental <br />                                                                            defects mutant plant lines, remains feasible                                                                            FAB1A FAB1B proteins functionally redun- <br />                                                                            dant. order generate double mutant (fab1a-1/ <br />                                                                            fab1a-1;fab1b-1/fab1b-1), single mutant lines                                                                            crossed  resulting F1 plants <br />                                                                            (fab1a-1/+;fab1b-1/+) allowed self-pollinate, <br />                                                                            progeny screened PCR                                                                            presence double homozygous mutants. 86 plants <br />                                                                            screened (Supplemental Table S1),                                                                            single double mutant detected, despite                                                                            theoretical 16 probability recovering <br />                                                                            plant (assuming transmission defects).  <br />                                                                             fab1a-1/+;fab1b-1/fab1b-1 fab1a-1/fab1a-1; <br />                                                                            fab1b-1/+ plants identified, lines  <br />                                                                            veloped maturity, severe vegetative <br />                                                                            morphological defects. suggests single copy <br />                                                                            FAB1A FAB1B sufficient support <br />                                                                            growth development. order increase <br />                                                                            chances obtaining fab1a-1/fab1a-1;fab1b-1/fab1b-1 <br />                                                                            double mutants, fab1a-1/+;fab1b-1/fab1b-1 fab1a- <br />                                                                            1/fab1a-1;fab1b-1/+ plants allowed set self <br />                                                                            seed. cases, potentially fourth                                                                            progeny homozygous mutant al- <br />                                                                            leles, assuming transmission defects. Fifty <br />                                                                            plants parent screened PCR                                                                            determine genotype, double mu- <br />                                                                            tants recovered. <br /> Figure 2. Identification leaf curl phenotype fab1a-1/fab1a-1      Importantly, seed set fab1a-1/+;fab1b-1/ <br /> fab1b-1/fab1b-1 plants.  Exon intron structure Arabidopsis        fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants  <br /> FAB1A FAB1B genes. Boxes represent exons intervening lines         mal (Fig. 3). gaps observed devel- <br /> represent introns. locations orientations SALK T-DNA            oping siliques, indicates inability detect <br /> insertions indicated fab1a-1 fab1b-1 alleles. LB      double mutants failures ability RB indicate left border right border T-DNA, respectively. <br />                                                                            plants (1) produce female gametes (2) undergo <br /> Inward-facing arrows indicate positions directions primers <br /> used RT-PCR experiments verify T-DNA insertions <br />                                                                            seed development following fertilization.  <br /> disrupted gene expression. region genes encoding             seeds harvested siliques germinated <br /> catalytic PtdIns3P 5-kinase domain indicated (PIP5K). B, Plant lines <br /> homozygous fab1a-1 fab1b-1 mutant alleles distinct leaf <br /> curl phenotype comparison wild-type (WT) plants; plants <br /> shown 4 weeks old. C, degree curling quantified comparing ratio uncurled leaf area visible curled <br /> surface area line. Leaf areas determined digital image <br /> analysis using ImageJ software. Curling extreme fab1b-1/ <br /> fab1b-1 plants, significantly different wild type (P = <br /> 0.027). fab1a-1/fab1a-1 plants displayed extreme, consistent, phenotype comparison wild type (P = 0.066). <br /> Plotted ratio means 95% confidence intervals indicated vertical bars. Leaves analyzed randomly selected <br /> plants line (numbers leaves measured: wild type, n = 30;      Figure 3. Seed set mature siliques wild-type (WT), fab1a-1/+; <br /> fab1a-1/fab1a-1, n = 32; fab1b-1/fab1b-1, n = 32).                         fab1b-1/fab1b-1, fab1a-1/fab1a-1;fab1b-1/+ plant lines. fab1a-1/+; <br />                                                                            fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants seed set <br />                                                                            indistinguishable wild type. <br />  <br /> Plant Physiol. Vol. 151, 2009                                                                                                               1815 <br />  Whitley et al. <br />  <br />  frequencies comparable wild type (99% 13 Murashige Skoog medium; n = 100). Taken  data strongly indicative trans- <br /> mission defect affecting fab1a-1/fab1b-1 pollen. <br />  <br /> fab1a-1/fab1b-1 Pollen Viable <br />  <br />    Plants carrying mutant alleles (fab1a-1/+;fab1b-1/ <br /> fab1b-1 fab1a-1/fab1a-1;fab1b-1/+) produce <br /> pollen 50% lack wild-type FAB1A FAB1B genes  e. genotype fab1a-1/ <br /> fab1b-1). Pollen derived plants checked viability staining fluorescein diacetate <br /> (FDA). large proportion pollen isolated fab1a-1/+;fab1b-1/fab1b-1 (34%) fab1a-1/fab1a-1; <br /> fab1b-1/+ (43%) plants failed stain FDA likely dead (Fig. 4; Table . grains seen collapsed frequently devoid cytoplasmic content (Fig. 4B). <br /> Staining pollen nucleic acid stain 4#,6- <br /> diamidino-phenylindole (DAPI) supported FDA <br /> data, grains lacked distinct sperm veg- <br /> etative cell nuclei.  grains remained intact, abnormalities number nuclei  sperm nuclei vegetative cell nucleus), suggesting gametogene- <br /> sis progresses normally wild-type mutant <br /> grains. <br />    investigate fab1a-1/fab1b-1 pollen capable effecting fertilization, restricted pollina- <br /> tions carried fab1a-1/+;fab1b-1/ <br /> fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ pollen parents transgenic male-sterile line (A9::barnase) mother. Fifty progeny resulting pollinations screened presence fab1a-1 fab1b-1 mutant alleles, resulting <br /> plants sired fab1a-1/ <br /> fab1b-1 pollen.  conclude failure generate fab1a-1/fab1a-1;fab1b-1/fab1b-1 double mu- <br /> tant plants entirely attributable fab1a-1/fab1b-1 <br /> pollen inviable case         Figure 4. Phenotypic analysis mature pollen wild-type, fab1a-1/+; <br />                                                               fab1b-1/fab1b-1, fab1a-1/fab1a-1;fab1b-1/+ plants. B, <br /> competitive disadvantage pollinations mixed <br />                                                               Fluorescence microscopy FDA-stained pollen derived wild- <br /> pollen genotype.                                              type   fab1a-1/+;fab1b-1/fab1b-1 (B) plants. FDA stains live cells <br />                                                               green imaged UV light. large proportion pollen fab1a-1/fab1b-1 Pollen Displays Abnormal Vacuolar          fab1a-1/+;fab1b-1/fab1b-1 plants collapsed failed stain Phenotype Late Pollen Development                          FDA (white arrowheads B), indicating dead.                                                               observed pollen fab1a-1/fab1a-1;fab1b-1/+ plants (data    investigate fab1a-1/fab1b-1 pollen invia-       shown). C D, DAPI staining pollen nuclei revealed uncollapsed <br /> bility, fresh mature pollen, derived fab1a-1/+;          pollen grains fab1a-1/+;fab1b-1/fab1b-1 plants (D) trinucleate, <br /> fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants,         wild-type pollen (C). observed pollen stained Neutral Red, rapidly accu-             fab1a-1/fab1a-1;fab1b-1/+ plants. White arrows indicate diffuse <br /> mulates vacuoles vesicles. Shortly pollen        vegetative nucleus close proximity intensely fluorescing <br /> hydration stain, approximately 50%            pair sperm nuclei. E L, Neutral Red staining pollen vacuoles ob- <br />                                                               served light differential interference contrast microscopy. Wild-type <br /> pollen grains abnormal vacuolar pheno- <br />                                                               pollen characterized containing small, regular vacuoles (E <br /> type unstained/dead (Table II; Fig. 4, F  K,        J). Pollen samples fab1a-1/fab1a-1;fab1b-1/+ (K) fab1a-1/+; L), vast majority pollen grains       fab1b-1/fab1b-1 (L) plants contained large number collapsed grains <br /> wild-type plants (98%) displayed numerous small               (black arrowheads). small proportion uncollapsed grains vacuoles relatively uniform size morphology            fab1a-1/+;fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants (Table II; Fig. 4, E J). majority grains      range highly abnormal vacuolar phenotypes, typified pres- deemed abnormal collapsed (Fig. 4, K            ence relatively  large, frequently irregular vacuoles (F . <br />  <br /> 1816                                                                                                 Plant Physiol. Vol. 151, 2009 <br />                                                                                               Role PtdIns(3)P 5-Kinase Pollen Development <br />  <br />  <br />  <br /> Table  Analysis pollen viability FDA staining                       generative cell, indication vacu- <br />   effect fab1a-1 fab1b-1 mutant alleles pollen viability <br />                                                                             oles enlarged irregularly sized propor- assessed staining pollen freshly dehisced anthers          tion grains mutant lines compared <br /> FDA described &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? Viable grains stain FDA        wild type (Fig. 5,  C, E). abnormal <br /> positive (Fig. 4, B). Actual numbers grains scored             vacuolar phenotype marked just prior indicated parentheses.                                                   second mitotic division mutant <br />                                       Pollen FDA        Pollen FDA          lines (Fig. 5, G H), vacuoles wild-type <br />             Genotype                                                        grains appeared fragment smaller vacu- <br />                                         Positive         Negative <br />  <br />                                             %               % <br />                                                                             oles disperse transition bicellular <br />    Wild type                           99 (198)           1 (2)             tricellular stage development (Yamamoto <br />    fab1a-1/+;fab1b-1/fab1b-1           66 (145)          34 (75)            et al., 2003). tricellular pollen matured wild-type <br />    fab1a-1/fab1a-1;fab1b-1/+           57 (166)          43 (125)           anthers, pollen cytoplasm took smoother <br />                                                                             appearance, vacuoles difficult detect <br />                                                                             certainty membrane-bound <br />                                                                             structures. contrast, nearly 50% grains                                                                             mutant lines underwent rapid collapse exhibited <br /> L), remained expanded displayed                           extreme vacuolar morphologies tricellular pollen range phenotypes typified presence                           matured (Fig. 5, D F). Taken  relatively vacuoles irregular size                     observations strongly suggest fab1a-1/fab1b-1 pol- <br /> morphology, vacuoles exceedingly                            len grains develop normally just large (Fig. 4, F .                                                        mitotic division.  following     order assess stage pollen develop-                     phase extensive vacuole fragmentation occurs <br /> ment fab1a-1/fab1b-1 mutations exerted ef-                        prior second mitotic division, grains fail fects, chemically fixed floral tissues fab1a-1/+;                      regulate process properly. <br /> fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants                           analysis pollen development trans- sectioned observations                            mission electron microscopy fab1a-1/+;fab1b-1/ <br /> light microscope. evidence aberrant pollen/                           fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants confirmed <br /> vacuolar morphology detected early                           observations obtained light microscopy. velopmental stages, tetrads highly vac-                        ultrastructure &#226;&#8364;&#339;early&#226;&#8364;? bicellular pollen highly <br /> uolated uninucleate microspores looking uniform                         similar wild-type (Fig. 6, B) identical pollen anthers wild-type plants                        mutant (Fig. 6, E, F,  J) lines.  highly <br /> (data shown). Uninucleate microspores di-                          aberrant vacuolar phenotypes observed grains <br /> vide form bicellular pollen consisting large                        mutant plant lines tricellular stage vegetative cell, fills pollen wall cavity,                    development (Fig. 6, G, H, K, L). vacuoles generative cell, resides vegetative                    frequently large irregular shape cell cytoplasm. transition uninucleate                      uneven boundaries (Fig. 6, G H), starkly contrast- bicellular stage development followed imme-                       ing small, uniform vacuoles diately fragmentation large vacuole                          smooth peripheries present wild-type grains (Fig. <br /> predominated uninucleate microspores (McCormick,                         6, C D). large vacuoles contained <br /> 1993; Yamamoto et al., 2003). observed signif-                        granular deposits appeared degraded cyto- <br /> icant differences wild-type pollen pollen                       plasm. cases, vacuoles filled major- fab1a-1/+;fab1b-1/fab1b-1 fab1a-1/fab1a-1;                         ity pollen cavity, suggesting large-scale loss fab1b-1/+ plants stage development.                           cytoplasm degradation cellular components. <br /> development proceeded second mitotic                              grains completely collapsed division, produces sperm cells                           contained little cytoplasm (Fig. 6K). <br />  <br />  <br />  <br />                      Table II. Assessment pollen vacuolar phenotype Neutral Red staining <br />                         effect fab1a-1 fab1b-1 mutant alleles morphology vacuoles pollen collected <br />                      freshly dehisced anthers assessed staining Neutral Red described &#226;&#8364;&#339;Materials                      Methods.&#226;&#8364;? Pollen scored &#226;&#8364;&#339;normal&#226;&#8364;? contained numerous small vacuoles (Fig. 4E), grains scored <br />                      &#226;&#8364;&#339;abnormal&#226;&#8364;? typically contained fewer irregularly sized large vacuoles (Fig. 4, F  collapsed. <br />                      &#226;&#8364;&#339;Unstained&#226;&#8364;? pollen grains lacked cytoplasm frequently collapsed. Actual numbers grains <br />                      scored indicated parentheses. <br />                                  Genotype                   Normal Pollen         Abnormal Pollen          Unstained Pollen <br />  <br />                                                                     %                    %                        % <br />                         Wild type                             98 (191)                1.5 (3)                  0.5 (1) <br />                         fab1a-1/+;fab1b-1/fab1b-1            55.4 (210)              30.1 (114)               14.5 (55) <br />                         fab1a-1/fab1a-1;fab1b-1/+            48.1 (140)              19.6 (57)                32.3 (94) <br />  <br /> Plant Physiol. Vol. 151, 2009                                                                                                            1817 <br />  Whitley et al. <br />  <br />  <br />                                                                               DISCUSSION <br />  <br />                                                                                  PtdIns(3,5)P2 ubiquitous minor membrane <br />                                                                               phospholipid plants, animals, yeast (Dove et al., <br />                                                                               1997; Whiteford et al., 1997; Meijer et al., 1999).                                                                               conserved role regulating endomembrane traffick- <br />                                                                               ing events animals yeast  review, Dove <br />                                                                               et al., 2009). PtdIns(3,5)P2 production tightly regu- <br />                                                                               lated (Sbrissa et al., 2007; Botelho et al., 2008; Ikonomov <br />                                                                               et al., 2009) inducible number different <br />                                                                               stimuli. example, hyperosmotic stress increases <br />                                                                               levels PtdIns(3,5)P2 20-fold yeast (Dove <br />                                                                               et al., 1997) significantly increases levels animal <br />                                                                               cells plants (Meijer et al., 1999; Zonia Munnik, <br />                                                                               2004; Sbrissa Shisheva, 2005). Hormonal stimuli, <br />                                                                               insulin mammalian cells, lead                                                                               increased production PtdIns(3,5)P2 (Ikonomov et al., <br />                                                                               2007). <br />                                                                                  animals yeast, PtdIns(3,5)P2 produced                                                                               PtdIns3P action PtdIns3P 5-kinases called <br />                                                                               PIKfyve Fab1p, respectively (Shisheva, 2008). <br />                                                                               PIKfyve Fab1p similar domain architecture, <br />                                                                               consisting N-terminal FYVE domain recruits <br />                                                                               protein specific membranes binding PtdIns3P <br />                                                                               (Sbrissa et al., 2002), central Cpn60_TCP1 domain <br />                                                                               probably plays regulatory role, kinase <br />                                                                               domain responsible conversion                                                                               PtdIns3P PtdIns(3,5)P2. Arabidopsis,                                                                               genes encoding proteins domain architecture <br />                                                                               FAB1A (At4g33240) FAB1B (At3g14270).                                                                               addition, genes, FAB1C (At1g71010)                                                                               FAB1D (At1g34260), contain Cpn60_TCP1  <br />                                                                               main C-terminal lipid kinase domain lack                                                                               FYVE domain. FYVE domain essential                                                                               yeast Fab1p phenotypically rescue fab1D strain <br />                                                                               (Botelho et al., 2008), possibility FAB1C                                                                               FAB1D PtdIns3P 5-kinases ruled  <br />                                                                                  obvious abnormal feature FAB1A                                                                               FAB1B homozygous T-DNA insertion lines leaf- <br />                                                                               curling phenotype. Given apparent evolutionarily <br />                                                                               conserved role PtdIns(3,5)P2 osmoregulation,                                                                               inability fab1a-1/fab1a-1 fab1b-1/fab1b-1 lines                                                                               produce sufficient levels lipid correctly <br />                                                                               regulate turgor pressure explain leaf curling. <br />                                                                               severity leaf curl pronounced fab1b-1/ <br />                                                                               fab1b-1 plants, consistent FAB1B                                                                               highly expressed FAB1A leaves (Supplemental <br /> Figure 5. Phenotypic analysis light microscopy pollen develop-          Fig. S1). Despite phenotypic differences ment wild-type, fab1a-1/fab1a-1;fab1b-1/+, fab1a-1/+;fab1b-1/ <br />                                                                               homozygous single mutant lines wild-type <br /> fab1b-1 plant lines. Pollen grains anthers bicellular tricellular <br /> stages development shown, respectively, wild-type  B), <br />                                                                               plants, mutants developed maturity ex- <br /> fab1a-1/fab1a-1;fab1b-1/+ (C D), fab1a-1/+;fab1b-1/fab1b-1            hibited normal seed set. broadly overlapping <br /> (E F) plant lines. Wild-type pollen displayed multiple small              expression profiles FAB1A FAB1B, vacuoles bicellular tricellular stages development           similar phenotypes exhibited fab1a-1/ B). bicellular pollen grains mutant lines contained abnor-        fab1a-1 fab1b-1/fab1b-1 mutants, suggest mally large vacuoles (black arrowheads C E). large proportion     FAB1A FAB1B  large extent, functionally <br /> tricellular grains mutant lines collapsed (black stars D F),   redundant. unable assess developmental partially collapsed contained large <br /> vacuoles (black arrowheads F). abnormal vacuolar phenotype noted marked bicellular grains approaching second mitotic division fab1a-1/fab1a-1;fab1b-1/+ (G)             abnormally large vacuoles). GC, Generative cell; SC, sperm cell; Tp, <br /> fab1a-1/+;fab1b-1/fab1b-1 (H) plant lines (black arrowheads indicate          tapetum; VN, vegetative nucleus. Bars = 20 mm. <br />  <br /> 1818                                                                                                              Plant Physiol. Vol. 151, 2009 <br />                                                                         Role PtdIns(3)P 5-Kinase Pollen Development <br />  <br />  <br />                                                                                       Figure 6. Transmission electron mi- <br />                                                                                       croscopy pollen wild-type, <br />                                                                                       fab1a-1/+;fab1b-1/fab1b-1, fab1a-1 <br />                                                                                       /fab1a-1;fab1b-1/+ plants. Pollen                                                                                       observed bicellular (wild-type, <br />                                                                                       B; fab1a-1/fab1a-1;fab1b-1/+, <br />                                                                                       E F; fab1a-1/+;fab1b-1/fab1b-1, <br />                                                                                       J) tricellular (wild-type, C <br />                                                                                       D; fab1a-1/fab1a-1;fab1b-1/+, G <br />                                                                                       H; fab1a-1/+;fab1b-1/fab1b-1, K <br />                                                                                       L) stages development. Dashed <br />                                                                                       boxes indicate enlarged regions ad- <br />                                                                                       jacent micrographs right.                                                                                       small vacuoles present bicellular <br />                                                                                       pollen (white arrowheads B, F,                                                                                       J), abnormally large <br />                                                                                       vacuoles observed propor- <br />                                                                                       tion grains mutant lines                                                                                        J). Wild-type tricellular grains                                                                                       characterized presence                                                                                       small regular vacuoles (white <br />                                                                                       arrowheads D), approxi- <br />                                                                                       mately 50% tricellular grains                                                                                       mutant lines col- <br />                                                                                       lapsed contained large irregular <br />                                                                                       vacuoles (black stars G, H, K,                                                                                       L) addition enlarged regular <br />                                                                                       vacuoles (white arrowhead H). Ex, <br />                                                                                       Exine; GC, generative cell;  intine; <br />                                                                                       LB, lipid body. Bars = 5 mm. <br />  <br />  <br />  <br />  <br /> phenotypes fab1a-1/fab1a-1;fab1b-1/fab1b-1 double        vives maturity, Neutral Red staining revealed mutant plants, attempts generate mutants         population grains aberrant vacuolar unsuccessful. Genetic analyses revealed unam-          morphologies.  grains biguously fab1a-1/fab1b-1 pollen incapable      observed dead account 50% fab1a-1/ <br /> transmitting alleles generation.   fab1b-1 pollen population. notable fea- <br /> role FAB1A FAB1B growth development           ture pollen fab1a-1/+;fab1b-1/fab1b-1 fully elucidated, transgenic approaches aimed      fab1a-1/fab1a-1;fab1b-1/+ plants tended abolishing respective transcripts     aggregate difficult remove anthers. employed.                                                problems noted pollen derived    Importantly, previously reported        fab1a-1/fab1a-1 fab1b-1/fab1b-1 plants. <br /> Arabidopsis pollen lacking PtdIns 3-kinase VPS34           phenotypes observed pollen fab1a-1/+; severe developmental defects (Lee et al., 2008b).      fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants <br /> VPS34 produces PtdIns3P PtdIns; PtdIns3P         similar described VPS34 mutant substrate Fab1/PIKfyve enzymes,         plants (Lee et al., 2008b).  required PtdIns(3,5)P2 synthesis, defects seen   important differences. transmission vps34-1 <br /> VPS34-deficient pollen primarily          allele observed Lee et al. (2008b), despite <br /> lack PtdIns(3,5)P2 production  ab-        extensive efforts, transmission fab1a-1/fab1b-1 pollen <br /> sence PtdIns3P). order test hypothesis,       observed study. addition pollen development plants carrying fab1a-1           abnormal vacuolar phenotype, vps34 mutant plants <br /> fab1b-1 mutant alleles investigated. fab1a-1/+;      produce increased proportion binucleate pollen <br /> fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants,       (Lee et al., 2008b). proposed abnormal <br /> 50% pollen fab1a-1/fab1b-1 genotype.     vacuoles physically prevent normal cell division large proportion pollen plants did      developing mutant pollen alternatively stain FDA considered           production PtdIns3P specific <br /> nonviable/dead pollen (34% 43%, respectively).          role directly regulating cell cycle. fab1a-1/ predict dead pollen mainly fab1a-1/          fab1b-1 pollen abnormal vacuoles cell <br /> fab1b-1 genotype, pollen fab1a-1/fab1a-1         division defect, favor explana- <br /> (fab1a-1/FAB1B pollen) fab1b-1/fab1b-1 (FAB1A/          tions. propose vacuolar defects observed <br /> fab1b-1 pollen) plants appear normal (.98% FDA              pollen vps34-1 mutant plants fab1a-1/ <br /> positive). fab1a-1/fab1b-1 pollen sur-        fab1b-1 pollen entirely attributable inabil- <br /> Plant Physiol. Vol. 151, 2009                                                                                           1819 <br />  Whitley et al. <br />  <br />  <br /> ity mutant pollen produce normal levels            likely extremely high expres- <br /> PtdIns(3,5)P2.                                              sion sperm cells (Supplemental Fig. S1).  <br />    important question consider does        ingly, lack FAB1A FAB1B female <br /> lack FAB1A/FAB1B affect pollen viability?            gametophyte did appear measurable <br /> possibility PtdIns(3,5)P2 production         effect fertility.  gametogenesis se <br /> required pollen tube growth,  large       requires FAB1A FAB1B;  function <br /> proportion developing pollen anthers fab1a-1/      reproduction related specific developmental <br /> +;fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants      processes unique pollen. speculate collapses long dehiscence, apparent   related preparation dehydration pollen <br /> primary defect pollen occurs pollen   prior dehiscence, stresses anthers.  known membrane       developmentally imposed ovule. <br /> dynamics important polarized pollen tube               summary, present, knowledge, elongation phosphoinositides play key            genetic phenotypic analysis Fab1/PIKfyve <br /> role membrane trafficking cytoskeletal reorga-       family proteins plants. conclude activity <br /> nization polarized cell growth (Monteiro et al.,     FAB1 proteins essential development 2005; Samaj et al., 2006; Zonia Munnik, 2008),          viable pollen. results consistent role PtdIns(3,5)P2 role surviving             FAB1 proteins responding osmotic changes fab1a-1/fab1b-1 pollen. Experimental evidence ad-        controlling levels PtdIns(3,5)P2. appar- <br /> dress possibility FAB1A FAB1B              ent pollen, leaf curl phenotype homozy- <br /> required pollen tube growth obtained,        gous single mutant plants hints defect data suggest small proportion pollen     osmoregulation. Future studies include investi- <br /> surviving dehiscence fab1a-1/fab1b-1,            gations cell plant phenotypes grown majority surviving pollen fab1a-1/        normal stress conditions conjunction FAB1B FABlA/fab1b-1 (dependent parent).               biochemical analysis PtdIns(3,5)P2 levels mutant <br />    presence large vacuolar structures late    plants. future, interesting address <br /> bicellular stage sign pollen develop-     plant orthologs known yeast animal <br /> ment perturbed mutant plants. increase          effectors PtdIns(3,5)P2 essential pollen  <br /> PtdIns(3,5)P2 levels increases surface area          bility plant-specific effectors respon- <br /> volume ratio intracellular organelles mediating       sible. <br /> fission large vacuoles  equivalent organelles) smaller vacuoles, protecting cells rapid <br /> dehydration lysis (Weisman, 2003; Sbrissa           MATERIALS METHODS <br /> Shisheva, 2005). fab1a-1/fab1b-1 pollen, unable produce normal levels PtdIns(3,5)P2, inability     Plant Growth Conditions <br /> vacuoles undergo fission explain accumu-            Arabidopsis (Arabidopsis thaliana) lines used study grown <br /> lation large vacuoles late bicellular stage    controlled-environment room 16-h-day/8-h-night cycle. Temper- <br /> development. ultimately lead pollen           ature maintained 22&#194;&#176;C 6 1&#194;&#176;C, relative humidity 60%. Seeds <br /> collapse dehydration phase occurs           sown petri dishes containing fine-grade peat-based compost,                                                             seedlings transferred pots containing compost 5 d just prior anthesis. cue vacuole fission <br />                                                             germination. strictly developmentally regulated induced changes solute concentration late pollen devel- <br /> opment.                                                     Phylogenetic Analysis FAB1/PIKfyve Family <br />    aware interpretation results    Proteins <br /> relies assumption Arabidopsis FAB1A             Protein sequences aligned using ClustalX (Larkin et al., 2007), FAB1B major sources PtdIns(3,5)P2 production,        trees drawn using NJPLOT program distributed ClustalX <br /> particularly pollen. domain structure,     software. following UniProtKB protein sequences used                                                             analysis: Schizosaccharomyces pombe, FAB1_SCHPO; Aspergillus fumigatus, <br /> feel safely assume FAB1A FAB1B         Q4WN65_ASPFU; Kluyveromyces lactis, Q6CS22_KLULA; Saccharomyces cere- PtdIns3P 5-kinases.  ex-         visiae, FAB1_YEAST; Dictyostelium discoideum, B0G126_DICDI; Chlamydomonas <br /> tremely difficult biochemically quantify PtdIns(3,5)P2   reinhardtii, A8J5R5_CHLRE; Oryza sativa, Q6ZJN4_ORYSJ, Q8H3L8_ORYSJ, <br /> levels accuracy fab1a-1/fab1b-1 pollen meta-     Q0DRB6_ORYSJ; Vitis vinifera, A5B6P1_VITVI, A7PDP8_VITVI,                                                             A5C675_VITVI; Arabidopsis, Q0WUR5_ARATH (FAB1A), Q9LUM0_ARATH <br /> bolic labeling methods (Zonia Munnik, 2004),                                                             (FAB1B), Q9SSJ8_ARATH (FAB1C), Q9XID0_ARATH (FAB1D); high mortality fab1a-1/fab1b-1 pollen    Populus trichocarpa, B9HJA2_POPTR B9N7I0_POPTR; Physcomitrella <br /> genotypes fab1a-1/+;fab1b-1/fab1b-1 fab1a-1/       patens, A9SCF9_PHYPA A9RYY6_PHYPA; Medicago truncatula, <br /> fab1a-1;fab1b-1/+ plants.                                   A4Q7Q7_MEDTR; Drosophila melanogaster, FYV1_DROME; Danio rerio, <br />    Regarding importance FAB1A FAB1B           B2KTE1_DANRE; Mus musculus, FYV1_MOUSE; Homo sapiens, FYV1_ <br />                                                             HUMAN. <br /> plant growth development, microarray data  <br /> veal expression levels high pollen comparison tissues. contrasts       Isolation FAB1A FAB1B Mutant Plant Lines FAB1C expression, negligible pollen.         SALK T-DNA insertion lines (Alonso et al., 2003) obtained FAB1D levels relatively high pollen,        Nottingham Arabidopsis Stock Centre. Genotyping plants carried  <br /> 1820                                                                                                 Plant Physiol. Vol. 151, 2009 <br />                                                                                                      Role PtdIns(3)P 5-Kinase Pollen Development <br />  <br />  PCR using genomic DNA template. fab1a-1 alleles (SALK_013923)             Supplemental Figure S1. Expression profiles Arabidopsis FAB1A, <br /> identified using following primers: T-DNA-specific left border primer                FAB1B, FAB1C, FAB1D. <br /> (5#-ATTTTGCCGATTTCGGAAC-3#) gene-specific left (LP; 5#-CCTGT- <br />                                                                                        Supplemental Figure S2. Expression FAB1A FAB1B wild-type <br /> CGTTTCCATCCAATTT-3#) right (RP; 5#-TGGACATGAATCCTTC- <br />                                                                                          mutant Arabidopsis plants. <br /> GACTG-3#) primers. fab1b-1 alleles (SALK_066673) identified using left border primer conjunction gene-specific LP (5#-TGAAC-                Supplemental Table S1. Expected observed genotypes progeny <br /> CATTGCTTCTGCTAGG-3#) RP (5#-CTCTCGTGGACATTCTTGAGC-3#)                                derived self-pollinated fab1a-1/FAB1A;fab1b-1/FAB1B plants. <br /> primers. RT-PCR carried test presence transcripts fab1a-1/fab1a-1 plants using LP (5#-CCAGGCAGCTATCATTTCACG-3#) RP (5#-GAAAGCCATTGATCCGAGGAC-3#) primers. Likewise fab1b-1/ <br /> fab1b-1 plants tested RT-PCR using LP (5#-TGGTTCAGTCGAC-               ACKNOWLEDGMENTS <br /> GAGAGCAT-3#) RP (5#-GGAGACCATTGATCAGGGACC-3#) primers. <br /> Total RNA RT-PCR isolated using RNeasy kit (Qiagen)                thank Dr. Alex Jeffries guidance phylogenetic analysis, <br /> manufacturer&#226;&#8364;&#8482;s instructions followed.                                         Ursula Potter technical assistance electron microscopy, Dr. Baoxiu Qi <br />     fab1a-1/+;fab1b-1/fab1b-1 fab1a-1/fab1a-1;fab1b-1/+ plants iso-        academic technical input, Jiannan Guo initial PCR charac- <br /> lated initially crossing fab1a-1/fab1a-1 fab1b-1/fab1b-1 single mutant       terization plant lines. thank Salk Institute Genomic Analysis <br /> lines generate F1 seeds. F1 plants allowed set &#226;&#8364;&#339;self&#226;&#8364;? seed   Laboratory Nottingham Arabidopsis Stock Centre provision F2 population lines recovered.                               mutant seed. <br />                                                                                     Received August 13, 2009; accepted October 18, 2009; published October 21, <br /> Leaf Curl Analysis                                                                  2009. <br />  <br />    degree leaf curl quantified wild-type, fab1a-1/fab1a-1 fab1b-1/fab1b-1 plants estimating visible leaf area photographs taken <br /> directly plants comparing area actual leaf area. Leaves         LITERATURE CITED removed sequentially plants permit analysis. leaf flattened clear adhesive plastic rephotographed. Image           Abramoff MD, Magelhaes PJ, Ram SJ (2004) Image processing analysis carried using ImageJ software (Abramoff et al., 2004)              ImageJ. Biophotonics International 11: 36&#226;&#8364;&#8220;42 <br /> obtain accurate measurements.                                                       Alonso JM, Stepanova  Leisse TJ, Kim CJ, Chen H, Shinn P, Stevenson <br />                                                                                        DK, Zimmerman J, Barajas P, Cheuk R, et al (2003) Genome-wide inser- <br />                                                                                        tional mutagenesis Arabidopsis thaliana. Science 301: 653&#226;&#8364;&#8220;657 <br /> Pollen Analysis                                                                     Backer JM (2008) regulation function class III PI3Ks: novel roles <br />                                                                                        Vps34. Biochem J 410: 1&#226;&#8364;&#8220;17 <br />    staining methods, pollen initially collected open <br />                                                                                     Botelho RJ, Efe JA, Teis D, Emr SD (2008) Assembly Fab1 phospho- <br /> flowers placed 400 mL 8% (w/v) Suc solution. flowers briefly <br />                                                                                        inositide kinase signaling complex requires Fig4 phosphoinositide <br /> vortexed release pollen removed tweezers. Pollen grains <br />                                                                                        phosphatase. Mol Biol Cell 19: 4273&#226;&#8364;&#8220;4286 pelleted centrifugation 6,000g 2 min resuspended                                                                                     Cantley LC (2002) phosphoinositide 3-kinase pathway. Science 296: appropriate stain microscopical analysis. <br />                                                                                        1655&#226;&#8364;&#8220;1657 <br />    Viability pollen grains assessed staining FDA. Pollen                                                                                     Di Paolo G, Camilli P (2006) Phosphoinositides cell regulation immersed 8% (w/v) Suc containing 0.5 mg mL21 FDA observed <br />                                                                                        membrane dynamics. Nature 443: 651&#226;&#8364;&#8220;657 <br /> immediately. Pollen nuclei stained DAPI described Lee et al. <br />                                                                                     Dove SK, Cooke FT, Douglas MR, Sayers LG, Parker PJ, Michell RH <br /> (2008b), 8% (w/v) Suc added staining solution. Vacuoles <br />                                                                                        (1997) Osmotic stress activates phosphatidylinositol-3,5-bisphosphate visualized pollen staining 0.02% (w/v) Neutral Red 8% (w/v) <br />                                                                                        synthesis. Nature 390: 187&#226;&#8364;&#8220;192 <br /> Suc. Observations carried Nikon ECLIPSE 90i microscope <br />                                                                                     Dove SK, Dong K, Kobayashi T, Williams FK, Michell RH (2009) Phos- <br /> (Nikon Instruments) equipped epifluorescence modules UV micros- <br />                                                                                        phatidylinositol 3,5-bisphosphate Fab1p/PIKfyve PPIn <br /> copy. <br />                                                                                        endo-lysosome function. Biochem J 419: 1&#226;&#8364;&#8220;13 <br />                                                                                     Dove SK, Piper RC, McEwen RK, Yu JW, King MC, Hughes DC, Thuring <br />                                                                                        J, Holmes AB, Cooke FT, Michell RH, et al (2004) Svp1p defines Analysis Pollen Development Light                                            family phosphatidylinositol 3,5-bisphosphate effectors. EMBO J 23: <br /> Transmission Electron Microscopy                                                       1922&#226;&#8364;&#8220;1933 <br />                                                                                     Friant S, Pecheur EI, Eugster  Michel F, Lefkir Y, Nourrisson D, <br />    Material wild-type, fab1a-1/+;fab1b-1/fab1b-1, fab1a-1/fab1a-1;            Letourneur F (2003) Ent3p PtdIns(3,5)P2 effector required fab1b-1/+ plants prepared using method adapted described            protein sorting multivesicular body. Dev Cell 5: 499&#226;&#8364;&#8220;511 <br /> Lee et al. (2008b). Buds anthers fixed solution containing 2.5%       Ikonomov OC, Sbrissa D, Dondapati R, Shisheva (2007) ArPIKfyve- <br /> glutaraldehyde 4% paraformaldehyde 0.1 M PIPES buffer (pH 7.4)             PIKfyve interaction role insulin-regulated GLUT4 translocation <br /> 4&#194;&#176;C overnight. Samples rinsed PIPES buffer. Postfixation              glucose transport 3T3-L1 adipocytes. Exp Cell Res 313: <br /> carried 1% (w/v) osmium tetroxide 4 h 4&#194;&#176;C, following           2404&#226;&#8364;&#8220;2416 <br /> material washed distilled water. Samples stained 1% aqueous          Ikonomov OC, Sbrissa D, Ijuin T, Takenawa T, Shisheva (2009) Sac3 uranyl acetate 1 h dark dehydrated ethanol series            insulin-regulated phosphatidylinositol 3,5-bisphosphate phospha- <br /> (30%, 50%, 70%, 80%, 90%, 95%, 100%) embedded LR White resin                tase: gain insulin responsiveness Sac3 downregulation (London Resin Company). Sections (0.5 mm  cut stained              adipocytes. J Biol Chem 284: 23961&#226;&#8364;&#8220;23971 <br /> Toluidine Blue light microscopy. Sections (100 nm  cut           Ikonomov OC, Sbrissa D, Shisheva (2001) Mammalian cell morphology <br /> transmission electron microscopy. Light microscopy conducted                endocytic membrane homeostasis require enzymatically active <br /> Nikon ECLIPSE 90i microscope (Nikon Instruments). transmission elec-               phosphoinositide 5-kinase PIKfyve. J Biol Chem 276: 26141&#226;&#8364;&#8220;26147 <br /> tron microscopy, observations JEOL 1200EXII transmission           Jefferies HB, Cooke FT, Jat P, Boucheron C, Koizumi T, Hayakawa M, <br /> electron microscope operating 80 kV.                                                Kaizawa H, Ohishi T, Workman P, Waterfield MD, et al (2008)                                                                                        selective PIKfyve inhibitor blocks PtdIns(3,5)P(2) production dis- <br />     Sequence data article GenBank/EMBL data              rupts endomembrane transport retroviral budding. EMBO Rep 9: <br /> libraries accession numbers NM_119478 NM_112285.                             164&#226;&#8364;&#8220;170 <br />                                                                                     Jeffries TR, Dove SK, Michell RH, Parker PJ (2004) PtdIns-specific MPR <br />                                                                                        pathway association novel WD40 repeat protein, WIPI49. Mol Biol <br /> Supplemental Data                                                                      Cell 15: 2652&#226;&#8364;&#8220;2663 <br />                                                                                     Kim DH, Eu YJ, Yoo CM, Kim YW, Pih KT, Jin JB, Kim SJ, Stenmark H, <br />    following materials available online version article.        Hwang (2001) Trafficking phosphatidylinositol 3-phosphate  <br /> Plant Physiol. Vol. 151, 2009                                                                                                                               1821 <br />  Whitley et al. <br />  <br />  <br />    trans-Golgi network lumen central vacuole plant         3-phosphate-interacting domains PIKfyve: binding specificity    cells. Plant Cell 13: 287&#226;&#8364;&#8220;301                                                role PIKfyve. Endomembrane localization. J Biol Chem 277: 6073&#226;&#8364;&#8220;6079 <br /> Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA,                 Sbrissa D, Shisheva (2005) Acquisition unprecedented phosphati- <br />    McWilliam H, Valentin F, Wallace IM, Wilm  Lopez R, et al (2007)           dylinositol 3,5-bisphosphate rise hyperosmotically stressed 3T3-L1 <br />    Clustal W Clustal X version 2.0. Bioinformatics 23: 2947&#226;&#8364;&#8220;2948            adipocytes, mediated ArPIKfyve-PIKfyve pathway. J Biol Chem 280: <br /> Lee Y, Bak G, Choi Y, Chuang WI, Cho HT (2008a) Roles phosphati-             7883&#226;&#8364;&#8220;7889 <br />    dylinositol 3-kinase root hair growth. Plant Physiol 147: 624&#226;&#8364;&#8220;635       Shisheva (2008) PIKfyve: partners, significance, debates paradoxes. <br /> Lee Y, Kim ES, Choi Y, Hwang  Staiger CJ, Chung YY (2008b)                Cell Biol Int 32: 591&#226;&#8364;&#8220;604 <br />    Arabidopsis phosphatidylinositol 3-kinase important pollen       Weisman LS (2003) Yeast vacuole inheritance dynamics. Annu Rev <br />    velopment. Plant Physiol 147: 1886&#226;&#8364;&#8220;1897                                      Genet 37: 435&#226;&#8364;&#8220;460 <br /> McCormick S (1993) Male gametophyte development. Plant Cell 5:                Welters P, Takegawa K, Emr SD, Chrispeels MJ (1994) AtVPS34,    1265&#226;&#8364;&#8220;1275                                                                    phosphatidylinositol 3-kinase Arabidopsis thaliana, essential <br /> Meijer HJ, Munnik T (2003) Phospholipid-based signaling plants. Annu         protein homology calcium-dependent lipid binding domain. <br />    Rev Plant Biol 54: 265&#226;&#8364;&#8220;306                                                   Proc Natl Acad Sci USA 91: 11398&#226;&#8364;&#8220;11402 <br /> Meijer HJG, Divecha N, van den Ende H, Musgrave  Munnik T (1999)            Whiteford CC, Brearley CA, Ulug ET (1997) Phosphatidylinositol 3,5- <br />    Hyperosmotic stress induces rapid synthesis phosphatidyl-D-inositol       bisphosphate defines novel PI 3-kinase pathway resting mouse <br />    3,5-bisphosphate plant cells. Planta 208: 294&#226;&#8364;&#8220;298                         fibroblasts. Biochem J 323: 597&#226;&#8364;&#8220;601 <br /> Monteiro D, Castanho Coelho P, Rodrigues C, Camacho L, Quader H,              Whitley P, Reaves BJ, Hashimoto M, Riley  Potter BV, Holman GD <br />    Malho R (2005) Modulation endocytosis pollen tube growth            (2003) Identification mammalian Vps24p effector phospha- <br />    phosphoinositides phospholipids. Protoplasma 226: 31&#226;&#8364;&#8220;38                  tidylinositol 3,5-bisphosphate-dependent endosome compartmentaliza- <br /> Mueller-Roeber B, Pical C (2002) <span id='am-18' about='obo:IMR_0001650' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="inositol phospholipid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001650" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span>Inositol phospholipid</span> metabolism            tion. J Biol Chem 278: 38786&#226;&#8364;&#8220;38795 <br />    Arabidopsis: characterized putative isoforms inositol phospho-      Yamamoto Y, Nishimura M, Hara-Nishimura  Noguchi T (2003) Behavior <br />    lipid kinase phosphoinositide-specific phospholipase C. Plant            vacuoles microspore pollen development Arabidopsis <br />    Physiol 130: 22&#226;&#8364;&#8220;46                                                           thaliana. Plant Cell Physiol 44: 1192&#226;&#8364;&#8220;1201 <br /> Samaj J, Muller J, Beck M, Bohm N, Menzel D (2006) Vesicular trafficking,     Zimmermann P, Hirsch-Hoffmann M, Hennig L, Gruissem W (2004) <br />    cytoskeleton signalling root hairs pollen tubes. Trends Plant     GENEVESTIGATOR: Arabidopsis microarray database analysis <br />    Sci 11: 594&#226;&#8364;&#8220;600                                                              toolbox. Plant Physiol 136: 2621&#226;&#8364;&#8220;2632 <br /> Sbrissa D, Ikonomov OC, Fu  Ijuin T, Gruenberg J, Takenawa T,               Zonia L, Munnik T (2004) Osmotically induced cell swelling versus cell <br />    Shisheva (2007) Core protein machinery mammalian phosphati-            shrinking elicits specific chan <br /> </body></html>