<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>J protein cochaperone mitochondrial inner <br /> membrane required protein import mitochondrial matrix <br /> Patrick D. D&#226;&#8364;&#8482;Silva*, Brenda Schilke*, William Walter*, Amy Andrew*&#226;&#8364;&#160;, Elizabeth  Craig*&#226;&#8364;&#161; <br /> *Department Biochemistry, 433 Babcock Drive, University Wisconsin, Madison WI 53706; &#226;&#8364;&#160;Graduate Program Biomolecular Chemistry, <br /> 1300 University Avenue, University Wisconsin, Madison, WI 53706 <br />  <br /> Contributed Elizabeth  Craig, September 24, 2003 <br />  major Hsp70 mitochondrial matrix (Ssc1 yeast)               associated J protein Sec63 stimulate ATPase activity critically important translocation proteins              luminal Hsp70 Kar2 essential translocation proteins <br /> cytosol, mitochondrial inner membrane,               cytosol endoplasmic reticulum lumen (9&#226;&#8364;&#8220;11). <br /> matrix. Tim44, peripheral inner membrane protein limited                 J protein mitochondria reported involved <br /> sequence similarity J domain J-type cochaperones, tethers          protein translocation.  mitochondrial J pro- <br /> Ssc1 import channel. report  unlike J protein,         teins studied yeast Saccharomyces cerevisiae. <br /> Tim44 does stimulate ATPase activity Ssc1, does            soluble matrix J protein Mdj1 known function Ssc1 <br /> affect stimulation known mitochondrial J protein          folding imported mitochondrially encoded proteins <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> substrate.  conclude Tim44 does                   does play significant role import (12, 13). J protein <br /> function J protein cochaperone Ssc1;  tethers Ssc1         Jac1 functions minor Hsp70 matrix, Ssq1, import channel interactions independent              biogenesis Fe&#229;&#8230;&#190;S clusters (14, 15). Mdj2 J protein critical J protein function.  previously unstudied             inner membrane.  deletion MDJ2 little essential gene, PAM18, encodes 18-kDa protein contains             phenotypic effect;  function remains obscure (16). <br /> J domain localized mitochondrial inner membrane.                  Tim44, mitochondrial inner membrane protein, Pam18 stimulates ATPase activity Ssc1; depletion Pam18             considered possible J protein partner Ssc1. Tim44, vivo disrupts import proteins mitochondrial matrix.           serves tether Ssc1 import channel, shows weak propose Pam18 J protein partner Ssc1               similarity J domain Sec63, deletion region <br /> import channel critical Ssc1&#226;&#8364;&#8482;s function protein import.        disrupts interaction Tim44 Ssc1 mitochondria (17). <br />                                                                              controversial, recent reports indicate Tim44 <br />                                                                              interacts ATPase domain Ssc1, does J domain <br /> M       ost proteins mitochondrial matrix synthesized         cytosolic ribosomes imported outer inner mitochondrial membranes. Translo- <br />                                                                              known J proteins (18, 19).  regardless                                                                              acts J protein partner Ssc1, Tim44 regulates inter- <br /> cation inner membrane occurs inner                    action Ssc1 import channel. Tim44 tethers <span id='am-36' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>- <br /> membrane channel driven membrane potential                 bound Ssc1 import channel. polypeptide substrates import motor (1&#226;&#8364;&#8220;3). critical components motor               bind Ssc1[ATP], Ssc1 released Tim44 absence major Hsp70 molecular chaperone mitochondria                      ATP hydrolysis, permitting binding (mtHsp70; Ssc1 yeast); peripheral inner membrane pro-                 Ssc1[ATP] channel second cycle interaction (20). <br /> tein Tim44, tethers Ssc1 import channel;                   importance import proteins <span id='am-47' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-48' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-52' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> release factor Ssc1, Mge1. Multiple cycles Ssc1            mitochondria cell function, set test idea binding release translocating polypeptide, driving               Tim44, analogous Sec63, functions J protein partner import process, required import proteins             Ssc1 protein import. Tim44 does mitochondrial matrix (1&#226;&#8364;&#8220;3).                                                  activities associated prototypical J protein.     Ssc1, like Hsp70s, contains C-terminal domain                previously unstudied essential gene, PAM18, encodes J protein <br /> binds short segments unfolded polypeptides rich hydro-                 inner membrane important protein transloca- <br /> phobic amino acids (4, 5). N-terminal ATPase domain                      tion vivo J protein partner Ssc1. <br /> regulates binding interaction <span id='am-31' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span> nucleo- <br /> tides. turn, binding peptide segment C-terminal             Materials Methods <br /> domain stimulates ATPase activity N terminus.              Yeast Strains, Plasmids, Genetic Techniques. PAM18 ob- <br /> ATP-bound state, interaction peptide substrate unstable,             tained PCR-amplifying genomic DNA position &#226;&#171;&#186;303 fast rates. <span id='am-6' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>-bound state,                 &#226;&#171;&#185;702 using Pfu Turbo polymerase (Stratagene) cloned <br /> interaction relatively stable, slow rates.                pRS316 (21). obtain null allele PAM18, coding <br />                                                                                                                                                        BIOCHEMISTRY <br />  <br />  <br />   thought ATP-bound form Hsp70                    sequences &#226;&#171;&#185;71 &#226;&#171;&#185;542 replaced TRP1 gene. <br /> initiates interaction polypeptide substrate,              heterozygous PAM18&#229;&#8230;&#190;pam18 diploid PJ53 (22) carrying <br /> stabilized hydrolysis ATP. Exchange ATP bound               PAM18 plasmid sporulated. &#226;&#338;&#172;pam18 haploids carrying <br /> ADP results release peptide, completing cycle.            PAM18 pRS316 isolated dissecting tetrads YP <br />    Hsp70s rarely,  function independently.               medium (1% yeast extract&#229;&#8230;&#190;2% peptone). Mutants PAM18 <br /> function cochaperones. J proteins, named                 constructed site-directed mutagenesis using <br /> presence signature J domain interacts                   QuikChange protocol (Stratagene) cloned pRS315. <br /> ATPase domain, stimulate Hsp70&#226;&#8364;&#8482;s ATPase activity, facil-                   Pam18 epitope-tagged C terminus tandem <br /> itating interaction substrate polypeptides (6, 7). activ- <br /> ity critical feature J protein function, <span id='am-55' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-56' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-58' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> <br /> alterations J domain decrease ATPase stimulation             Abbreviation: HA, hemagglutinin. <br /> functionally defective vivo (8). example, lumen                correspondence addressed. E-mail: ecraig@wisc.edu. endoplasmic reticulum, ability membrane-                      &#194;&#169; 2003 National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                             PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  24 &#229;&#8230;&#169; 13839 &#226;&#8364;&#8220;13844 <br />  copies hemagglutinin (HA) peptide introducing NotI site termination codon subcloning HA <br /> tag NotI fragment pGTEP1 (23). tagged protein functional; cells carrying tagged protein PAM18 protein grew wild-type cells variety conditions. purposes purification, entire ORF PAM18 cloned <span id='am-19' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-20' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span>glutathione</span> S-transferase expressing <br /> vector pGEX-KG (24), forming fusion protein Pam18 C terminus. modulate expression Pam18, entire <br /> ORF (&#226;&#171;&#185;1 &#226;&#171;&#185;507) cloned pCM189 (25), placing <br /> PAM18 control tetracycline-regulatable promoter, <br /> resulting repression transcription addition           Fig. 1. Effect Tim44 ATPase activity Ssc1. Ssc1:ATP complex (&#226;&#172;&#8225;0.2 <br /> tetracycline derivative doxycycline.                                    &#226;?&#174;M) incubated 23&#194;&#176;C presence various concentrations Mdj1 <br />    Complete synthetic media lacking specific amino acids             peptide P5. rate conversion ADP measured plotted containing 5-fluoroorotic acid prepared described (26).         fold stimulation basal rate determined absence additional chemicals obtained Sigma unless stated.         components.   Mdj1, peptide P5, Tim44, indicated. (B) Reactions                                                                         1 &#226;?&#174;M Mdj1 (Left) 5 &#226;?&#174;M peptide P5 (Right) carried presence <br />                                                                         indicated concentrations Tim44. <br /> Fractionation Mitochondria. Mitochondria fractionated <br /> essentially described ref. 27. Briefly, 25 &#226;?&#174;g mitochondria swollen 500 &#226;?&#174;l EM buffer (10 mM Mops KOH, pH                 purified Ssc1 antibodies crosslinked protein  <br /> 7.2&#229;&#8230;&#190;1 mM EDTA) incubated ice 15 min. Efficiency <br />                                                                         Sepharose beads. micrograms Ssc1 incubated disruption outer membrane assessed incubation <br />                                                                         Ssc1 antibody beads 4&#194;&#176;C 1 h. condition, &#226;&#172;&#8225; 90% proteinase K 50 &#226;?&#174;g&#229;&#8230;&#190;ml ice 15 min. control, <br />                                                                         Ssc1 typically immunoprecipitated. washes, <br /> sodium deoxycholate added 1% disrupt mitochon- <br /> drial inner membrane. Mitochondria separated mem-             Tim44 (0.6 &#226;?&#174;g) added. mixture incubated final <br /> brane soluble fractions described ref. 28. Mitochondrial      volume 150 &#226;?&#174;l 23&#194;&#176;C 30 min point 1.5 &#226;?&#174;l fractions subjected immunoblot analysis using            1 mg&#229;&#8230;&#190;ml solution peptide P5 added reactions, ECL (Amersham Pharmacia Biotech) according                incubation continued additional 30 min. Tim44:Ssc1 <br /> manufacturer&#226;&#8364;&#8482;s suggestion, using polyclonal antibodies spe-          complex formation robust, forming 0&#194;&#176;C cific Tim44, cytochrome b2, Mge1 monoclonal               variety salt buffer conditions (data shown). final <br /> antibody specific HA.                                               concentrations Ssc1, Tim44, peptide P5 0.6, 0.09, <br />                                                                         10 &#226;?&#174;M, respectively. centrifugation, beads Protein Purification. Pam18 purified bacterial strain C41      washed times mitochondrial lysis buffer (250 mM <br /> (29). Expression induced 3 h 37&#194;&#176;C addition 1 mM        sucrose&#229;&#8230;&#190;80 mM KCl&#229;&#8230;&#190;20 mM Mops-KOH, pH 7.2&#229;&#8230;&#190;0.2% Triton <br /> isopropyl-&#226;?&#164;-D-thiogalactopyranoside. affinity chromatog-          X-100&#229;&#8230;&#190;1 mM PMSF) containing appropriate nucleotide. <br /> raphy using <span id='am-62' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span>glutathione</span> agarose, biotinylated thrombin              Tim44 Ssc1 bound beads detected SDS&#229;&#8230;&#190; <br /> added mixture incubated 3 h 20&#194;&#176;C. cleaved        PAGE immunoblot analysis using polyclonal antibodies <br /> Pam18 product recovered removal glutathione               specific Tim44 Ssc1. <br /> agarose beads centrifugation removal thrombin streptavidin&#229;&#8230;&#190;agarose (catalog  69022-3, Novagen).                    Peptide Binding Assay. Briefly, fluorescence anisotropy assays    Tim44His (20), Mdj1His (30), Ssc1His (31), Ssq1His (31)     binding peptide carried described refs. 20 purified described. tagged proteins previously          32. Peptide P5 labeled fluorescein generate F-P5 shown able rescue absence normal protein vivo.   described (33). Various concentrations Ssc1 incubated <br />                                                                         F-P5 (10 nM) 23&#194;&#176;C buffer (25 mM Hepes-KOH, pH <br /> ATPase Assay. Isolation Hsp70:ATP complexes ATPase               7.4&#229;&#8230;&#190;100 mM KCl&#229;&#8230;&#190;11 mM Mg(OAC)2&#229;&#8230;&#190;10% glycerol). assays carried essentially described refs. 20 32,    binding reached equilibrium, anisotropy measurements changes. obtain radiolabeled ATP:Ssc1 complexes,           Beacon 2000 Fluorescence Polarization 50 &#226;?&#174;g Hsp70 incubated 100 &#226;?&#174;Ci [&#226;?&#163;-32P]ATP (3,000          (Panvera, Madison, WI) 23&#194;&#176;C excitation 490 nm Ci&#229;&#8230;&#190;mmol, DuPont) buffer (25 mM Hepes-KOH, pH                       emission 535 nm. <br /> 7.4&#229;&#8230;&#190;100 mM KCl&#229;&#8230;&#190;11 mM Mg(OAC)2&#229;&#8230;&#190;10% glycerol) containing <br /> 25 &#226;?&#174;M ATP 100 &#226;?&#174;l ice 30 sec. complex isolated         Results <br /> immediately NICK column (Amersham Pharmacia Bio- <br />                                                                         Tim44 Does Stimulate ATPase Activity Ssc1. address tech) frozen aliquots &#226;&#171;&#186;70&#194;&#176;C. ATPase assays, <br />                                                                         possibility Tim44 acts J protein cochaperone aliquots thawed ice; time 0 time shift 23&#194;&#176;C, <br />                                                                         import channel, tested ability affect ATPase activity additions peptide P5 (CALLLSAPRR), Tim44, Mdj1, <br /> and&#229;&#8230;&#190;or Pam18 samples. indicated                Ssc1 using single-turnover conditions. complex times, 3-&#226;?&#174;l aliquots removed mixed 1 &#226;?&#174;l stop           Ssc1 radiolabeled ATP isolated, rate ADP <br /> solution containing 4 M <span id='am-67' about='obo:IMR_0200110' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-68' property="oboInOwl:id" content="IMR:0200110" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Formic acid" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00058" datatype="xsd:string"></span>formic acid</span>, 2 M LiCl, 36 mM ATP.           formation determined incubation presence mixture spotted polyethyleneimine-cellulose              absence Tim44. basal rate ATP hydrolysis Ssc1 TLC plate developed 1 M formic acid 0.5 M LiCl.              0.07 min&#226;&#171;&#186;1, consistent previous results (32). effect drying, TLC plate exposed PhosphorImager             ATPase activity Ssc1 observed addition screen (Molecular Dynamics), ATP ADP              Tim44 range concentrations, 35 &#226;?&#174;M, 140 quantified PhosphorImager (Molecular Dynam-             molar excess Ssc1 (Fig. 1A). control, increasing <br /> ics). percentage hydrolysis 0 time (&#226;&#172;&#8225;20%) sub-            concentrations soluble mitochondrial J protein <br /> tracted time points.                                           Mdj1 peptide substrate added. stimulated                                                                         hydrolysis ATP. example, 10-fold molar excess (2 &#226;?&#174;M) <br /> Coimmunoprecipitation Assay. Ssc1 coimmunoprecipitation             Mdj1, activity stimulated 7-fold; 100-fold excess (20 <br /> assay carried essentially described ref. 20. Affinity-    &#226;?&#174;M), peptide activity stimulated 6.5-fold. <br />  <br /> 13840 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                                                      D&#226;&#8364;&#8482;Silva et al. <br />  Fig. 2. Functional defect Ssc1A503D.   ATPase activity. Ssc1:ATP Ssc1A503D:ATP complexes (&#226;&#172;&#8225;0.2 &#226;?&#174;M) incubated 23&#194;&#176;C presence Mdj1 peptide P5. basal rates hydrolysis mutant wild-type protein indistinguishable. (B) Peptide binding measured fluorescence anisotropy: <br /> Fluorescein-labeled F-P5 (10 nM) incubated presence indicated concentrations wild-type A503D mutant protein ADP. Anisotropy <br /> measurements taken 25&#194;&#176;C; raw polarization values fitted site binding (hyperbola) equation determine Kd. (C) Effect peptide Ssc1:Tim44 interaction. Tim44 (0.09 &#226;?&#174;M) incubated Ssc1 (0.6 &#226;?&#174;M) presence ATP ADP, indicated, 30 min. cases, 10 &#226;?&#174;M <br /> peptide P5 added, incubation continued additional 30 min. mixtures immunoprecipitated using Ssc1-specific antibodies described Materials Methods. presence Tim44 Ssc1 precipitate assessed immunoblot analysis using antibodies specific respective <br /> proteins. <br />  <br />  <br />  <br /> Tim44 Interaction Ssc1 Distinct J Protein.              rate Ssc1 observed, vitro, presence high <br /> assess Tim44 interacting Ssc1 sites similar                concentration peptide substrate turnover minute. overlapping contacting J protein, tested                   rate hydrolysis clearly sufficient drive import effect increasing concentrations Tim44 ability                    vivo, rate cycle interaction Mdj1 stimulate ATPase activity Ssc1. effect                       occur imported polypeptide. Assuming rate stimulation seen Tim44 added concentrations                       ATP hydrolysis Ssc1 observe vitro assays 5 35 &#226;?&#174;M, Mdj1 held constant 1 &#226;?&#174;M                         reflective natural state protein, cofactor Ssc1 <br /> (Fig. 1B). Similar experiments carried presence                   stimulates ATP hydrolysis exist. peptide substrates.  effect seen peptide                          Tim44 does stimulate ATPase activity Ssc1, <br /> stimulation ATPase activity addition Tim44.                      searched yeast genome database (www.yeastgenome.org) <br /> results indicate Tim44 does act J protein partner                potential J protein partners Ssc1. ORF YLR008C <br /> Ssc1, does interfere binding substrate peptide.                  (PAM18) essential gene    probe differences Tim44 J protein                   encodes protein containing predicted internal membrane <br /> interactions, used mutant isolated search                   spanning domain (Fig. 3A) J domain 57% identical temperature-sensitive point mutations SSC1. Ssc1A503D                      J protein mitochondrial inner membrane single amino acid alteration peptide-binding domain.                 Mdj2 (16). confirmed deletion PAM18 W303 ATPase activity A503D mutant protein stimu-                        strain background resulted inviability, deletion MDJ2 <br /> lated Mdj1 (Fig. 2A).  interactions Tim44                     little phenotypic effect (data shown). peptide substrate normal. Ssc1A503D bound fluorescein-                        determine J domain Pam18 important tagged peptide affinity wild-type protein,                  function, constructed mutants causing alterations dissociation constant (Kd) 0.3 &#226;?&#174;M (20), measured                       J domain. hallmark J domain highly <br /> fluorescence anisotropy (32) (Fig. 2B). peptide substrate                       conserved tripeptide, histidine, proline, aspartic acid <br /> stimulated ATPase activity Ssc1A503D nearly                     (HPD), constructed mutation results replace- <br /> wild-type protein (4.0- 4.2-fold, respectively) peptide                  ment HPD alanines (Pam18A3). addition, concentration 15 &#226;?&#174;M (Fig. 2 . Tim44 bound                            constructed mutation results substitution glu- <br /> Ssc1[ATP] Ssc1A503D[ADP],  wild-type pro-                         tamine place histidine (Pam18H141Q). test tein, peptide able stimulate release Tim44                       mutants rescue inviability &#226;&#338;&#172;pam18 strain, <br /> Ssc1A503D[ATP] complex (ref. 20 Fig. 2C).  interac-                      plasmids carrying PAM18 mutants transformed tion Tim44 peptide substrate, J protein                      &#226;&#338;&#172;pam18 strain carrying wild-type copy PAM18 cen- <br />                                                                                                                                                                     BIOCHEMISTRY <br />  <br />  <br />  <br /> Mdj1, normal mutant Ssc1 protein. Tim44                      tromeric plasmid having URA3 gene. resulting compete binding Mdj1 peptide interacts                      strains plated media containing 5-fluoroorotic acid, normally Ssc1 A503D, conclude Tim44 J                       selects cells having lost URA3-containing plasmid carrying <br /> protein does bind sites Ssc1 prevent                       wild-type PAM18 gene, viable cells recovered (Fig. 3B). <br /> interaction known J protein.                                                J domain essential PAM18 function. <br />                                                                                      Pam18 J protein partner Ssc1 vivo, Essential J Protein Mitochondrial Inner Membrane, Pam18.                   localized mitochondria J domain interact <br /> Multiple binding release cycles Ssc1 incoming                      Ssc1, matrix. determine cellular <br /> polypeptide chain required import (1&#226;&#8364;&#8220;3).                  location Pam18, mitochondria cells expressing Pam18 <br /> estimated  average, vivo, polypeptide chains pass                HA tag C terminus (Pam18-HA) isolated. import channel minute (34).  turnover                   mitochondria contained protein reacted rate ATP hydrolysis Ssc1 sufficiently fast allow                 HA-specific antibody present mitochondria <br /> numerous interaction cycles time frame. turnover                  expressing Pam18 lacking HA tag (data shown). Mito- <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                      PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  24 &#229;&#8230;&#169; 13841 <br />                                                                                     localize Pam18, mitochondria having intact inner <br />                                                                                    membrane, disrupted outer membrane, formed                                                                                    swelling mitochondria buffer low osmotic strength.                                                                                    swollen mitochondria treated protease,                                                                                    intact mitochondria (Fig. 3C Left). Like Tim44, intermem- <br />                                                                                    brane space protein cytochrome b2, matrix protein Mge1, <br />                                                                                    Pam18 substantially affected protease digestion                                                                                    intact mitochondria, suggesting Pam18 outer <br />                                                                                    membrane.  treatment swollen mitochondria                                                                                    protease resulted production predominant cleavage <br />                                                                                    fragment containing HA tag. results suggest                                                                                    Pam18 located inner membrane. addition,                                                                                    HA tag C terminus resistant protease attack, <br />                                                                                    J domain C-terminal putative membrane- <br />                                                                                    spanning domain predicted matrix                                                                                    inner membrane, N terminus protruding                                                                                    intermembrane space. orientation make J <br />                                                                                    domain accessible Ssc1 matrix. <br />                                                                                       assess Pam18 role mitochondrial protein <br />                                                                                    import vivo, PAM18 placed control                                                                                    tetracycline-repressible promoter. Drug added turn                                                                                    expression, accumulation precursor form nuclear- <br />                                                                                    encoded mitochondrial protein, Hsp60, monitored.                                                                                    case nuclear-encoded matrix proteins, prese- <br />                                                                                    quence Hsp60 cleaved matrix processing protease <br />                                                                                    immediately traversing inner membrane (35);                                                                                    accumulation normally undetectable precursor form                                                                                    used previously vivo indicator function                                                                                    import process (28, 36). Cells harvested 24 h drug <br />                                                                                    addition, growth tester strain begun slow <br />                                                                                    compared control strain. precursor form Hsp60 <br />                                                                                    accumulated strain Pam18 expression                                                                                    repressed, control strain. accumulation                                                                                    indicative role Pam18 import proteins                                                                                    mitochondrial matrix vivo. <br />  <br />                                                                                    Pam18 Specifically Stimulates Ssc1 ATPase Activity. results <br />                                                                                    described previous section suggest Pam18 acts J <br />                                                                                    protein mitochondrial import. purified PAM18 <br />                                                                                    wild-type mutant proteins tested ability stim- <br /> Fig. 3. vivo function J protein Pam18.   Predicted amino acid          ulate ATPase activity Ssc1 using single-turnover assay <br /> sequence Pam18. box drawn putative membrane-spanning            described  Wild-type Pam18 stimulated ATPase ac- <br /> region. region having sequence identity J domain underlined;       tivity 7-fold present 7-fold excess (Fig. 4A). Pam18 highly conserved HPD indicated ***. (B) Requirement vivo J domain      Mdj1 similar concentration dependences stimula- Pam18. Strains carrying plasmids expressing protein (&#226;&#171;&#186;), wild-type           tion Ssc1 ATPase activity, suggesting proteins <br /> Pam18, mutant Pam18, Pam18AAA Pam18H141Q (Pam18HI/Q) spot-             interact similarly Ssc1. case Mdj1, addition <br /> ted 1- fold dilution series media lacking (Left) containing (Right) <br />                                                                                    excess Tim44 did inhibit ability Pam18 stimulate <br /> 5-fluoroorotic acid. Plates incubated 3 days 30&#194;&#176;C. (C) Fractionation mitochondria. Mitochondria purified cells expressing Pam18 <br />                                                                                    Ssc1 (Fig. 4B), indicating binding Tim44 Pam18 having HA tag C terminus. Mitochondria separated               mutually exclusive. <br /> membrane soluble components (Right). Mitochondrial protein (T)                test Pam18 stimulates Ssc1 ATPase activity fractional equivalents membrane pellet (P) supernatant (S)             prototypical J protein, used J domain mutants described <br /> analyzed. Mitochondria (M) subjected swelling disrupt outer         previous section, Ssc1 mutant membrane (SW) (Left). Equivalent amounts treated proteinase K            defective interaction known J protein Mdj1. Pam18 cases treated detergent (sodium deoxycholate), indicated.      mutant proteins altered J domains unable fractions subjected immunoblot analysis using antibodies            function vivo failed stimulate Ssc1&#226;&#8364;&#8482;s ATPase activity <br /> specific HA tag Pam18, antibody specific known <br />                                                                                    (Fig. 4A). addition, tested ability wild-type Pam18 <br /> proteins intermembrane space (cytochrome b2), inner membrane <br /> (Tim44), matrix (Mge1). (D) &#226;&#338;&#172;pam18 cells carrying Pam18 control                                                                                    stimulate ATPase activity A503D Ssc1 mutant (Fig. tetracycline regulatable promoter grown overnight rich media,       4C). Like Mdj1 (Fig. 2D), Pam18 did stimulate mutant point subcultured presence absence tetra-         protein.  stimulation Pam18 specific, cycline analogue doxycycline. 24 h cell lysates prepared      failed stimulate ATPase activity Hsp70 separated SDS&#229;&#8230;&#190;PAGE, followed immunoblot analysis using antibodies         mitochondrial matrix Ssq1 (Fig. 4D).  conclude specific Hsp60.                                                                Pam18 J protein partner Ssc1. <br />  <br />                                                                                    Discussion <br /> chondria containing Pam18-HA separated membrane                          data presented allow better understanding soluble fractions, subjected immunoblot                     mechanism translocation proteins mitochondrial <br /> analysis using HA-specific antibodies. Pam18-HA present                     inner membrane.  Tim44 tethers Ssc1 import membrane fraction, Tim44 (Fig. 3C Right).                         channel, does function J protein cochaperone  <br /> 13842 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                                                       D&#226;&#8364;&#8482;Silva et al. <br />  Fig. 4. Stimulation Ssc1 ATPase activity Pam18. Hsp70:ATP complexes prepared, rate hydrolysis measured 23&#194;&#176;C presence various concentrations Mdj1, wild-type Pam18, mutant Pam18, presence absence Tim44. rate conversion ADP measured plotted fold stimulation basal rate. panel, Hsp70 ATP hydrolysis measured indicated asterisk.   Ssc1:ATP <br /> (&#226;&#172;&#8225;0.2 &#226;?&#174;M) incubated presence various concentrations wild-type Pam18, Mdj1, Pam 18 mutant proteins Pam18AAA Pam18H141Q. (B) <br /> Reactions 1 &#226;?&#174;M Pam18 carried presence indicated concentrations Tim44. (C) Ssc1:ATP Ssc1A503D:ATP complexes (&#226;&#172;&#8225;0.2 &#226;?&#174;M) incubated presence excess Pam18. (D) Ssq1:ATP complexes (0.1 &#226;?&#174;M) presence various concentrations Pam18. <br />  <br />  <br />  <br /> mitochondrial Hsp70 Ssc1, does bind site(s)                 induced interaction peptide substrate rapidly  <br /> Hsp70 J protein. Second, identified J protein                      stabilizes Ssc1:Tim44 interaction (20). Likewise, binding partner Ssc1 inner membrane stimulates Ssc1&#226;&#8364;&#8482;s                     Mge1 Ssc1[ADP] destabilizes interaction Tim44 (20, 37). <br /> ATPase activity. results lead model Ssc1,                      Tim44 evolved conformational-sensitive switch <br /> tethered import channel, stimulated J protein                  governing association Ssc1 import channel. close proximity, ensuring rapid binding incoming                      fact Tim44 J protein identification polypeptide chain.                                                                new J protein partner Ssc1 allow forward revised <br />    Determining mechanism action Tim44 protein                        model protein import. contrast endoplasmic retic- <br /> translocation important, central initial                  ulum, tethering import channel stimulation interaction translocating protein import motor.                 ATPase activity executed protein, J protein Sec63 <br /> inability Tim44 stimulate ATPase activity Ssc1,              (38), mitochondria separate critical functions  concentrations 20-fold higher required 5-fold                      proteins. Tim44 regulates Ssc1&#226;&#8364;&#8482;s interaction stimulation known J protein partner Ssc1, Mdj1, argues                  import channel, Pam18 stimulates Ssc1&#226;&#8364;&#8482;s ATPase activ- <br /> strongly Tim44 functioning J cochaperone Ssc1.                ity. Occupancy biologically relevant form Ssc1, <br /> original idea Tim44 function J-type cochaperone               Ssc1[ATP], predominates import channel (20). Interaction <br /> Ssc1 based limited sequence homology J domains                  incoming polypeptide stimulates dissociation Ssc1 J proteins (17). possible J-like region evolved            Tim44  combination Pam18, stimulates ATP true J domain facilitate interaction Ssc1                   hydrolysis, stabilizing Ssc1:polypeptide substrate interaction. <br /> Tim44. fact, 18-aa deletion completely removing region                 Interestingly, Ssc1&#226;&#8364;&#8482;s release Tim44 substrate binding <br /> resulted synthesis stable mutant protein inserted                independent ATP hydrolysis. Pam18&#226;&#8364;&#8482;s ability correctly membrane defective interaction                   stimulate ATPase activity Ssc1 affected Tim44, <br /> Ssc1. results suggested region Tim44 interacts                 determining relative rates release Tim44 ATP <br /> directly Ssc1 (17).  scenario Tim44 having                   hydrolysis needed definitive model evolved J-like domain interacts Ssc1 J domain                  import forth.  noted binding site unlikely competition functional                    mechanistically important ATP hydrolysis temporally <br /> interaction Tim44 true J proteins Mdj1 Pam18                  coupled release, failure hydrolyze ATP result observed.  interaction sites J proteins Tim44                dissociation polypeptide chain Ssc1 likely distinct; detailed structural information                 extreme instability Ssc1:substrate interaction Ssc1 required clearly delineate exact sites interactions               bound ATP. Pam18 surely plays critical role stimulating proteins.                                                                   hydrolysis rate. play role coordinating    results indicate Pam18 prime candidate                 timing hydrolysis, allowing efficient coupling release J protein partner Ssc1 mitochondrial import. case                Tim44 ATP hydrolysis. <br />                                                                                      Determination exact mechanistic details protein import components import machinery, Tim44, <br />                                                                                   coordination action Ssc1 cofactors awaits Ssc1, Mge1, Pam18 essential. located inner <br />                                                                                   analysis.  precedent additional proteins <br />                                                                                                                                                                    BIOCHEMISTRY <br />  <br />  <br />  <br /> membrane, J domain extending matrix,                                                                                   mediating interactions Hsp70s&#229;&#8230;&#190;J proteins&#229;&#8230;&#190;substrate <br /> positioned interact Ssc1 tethered import <br />                                                                                   polypeptides. Thermus thermophilus stable complex channel. expected bona fide J protein partner,                                                                                   Hsp70 J protein mediated protein, DafA (39). <br /> stimulate ATPase activity Hsp70 Ssc1,                                                                                   binding substrate polypeptide, DafA displaced functions, does stimulate ATPase activity                                                                                   complex. Interactions adapter proteins Tim44 matrix Hsp70, Ssq1. stimulation affected                                                                                   DafA common presently appreciated serve presence Tim44.  Pam18 able stimulate                     important roles regulating interactions Hsp70s, J proteins, ATPase activity Ssc1 tethered import                    substrate polypeptides. <br /> channel Tim44. <br />     Tim44 considered simply membrane                      thank Jarek Marszalek helpful discussions thoughtful <br /> anchor Ssc1. interaction proteins                   comments manuscript. work supported National evolved exquisitely sensitive conformation                   Institutes Health Grants GM278709  E C.) 5T326M08349 Ssc1. example, conformational change Ssc1[ATP]                         ). <br />  <br />  <br /> D&#226;&#8364;&#8482;Silva et al.                                                                                     PNAS &#229;&#8230;&#169; November 25, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  24 &#229;&#8230;&#169; 13843 <br />   1.   Neupert, W. <span id='am-12' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Brunner, M. (2002) Nat. Rev. Mol. Cell Biol. 8, 555&#226;&#8364;&#8220;565.               22.   James, P., Pfund, C. &amp; Craig, E. (1997) Science 275, 387&#226;&#8364;&#8220;389. <br />  2.   Pfanner, N. &amp; Geissler,  (2001) Nat. Rev. Mol. Cell Biol. 5, 339&#226;&#8364;&#8220;349.              23.   Roof, D. M., Meluh, P. B. &amp; Rose, M. D. (1992) J. Cell Biol. 118, 95&#226;&#8364;&#8220;108. <br />  3.   Matouschek, , Pfanner, N. &amp; Voos, W. (2000) EMBO Rep. 5, 404&#226;&#8364;&#8220;410.                  24.   Guan, K. L. &amp; Dixon, J. E. (1991) Anal. Biochem. 192, 262&#226;&#8364;&#8220;267. <br />  4.   Bukau, B. &amp; Horwich,  L. (1998) Cell 92, 351&#226;&#8364;&#8220;366.                                  25.   Gari, E., Piedrafita, L., Aldea, M. &amp; Herrero, E. (1997) Yeast 13, 837&#226;&#8364;&#8220;848. <br />  5.   Hartl, F. &amp; Hayer-Hartl, M. (2002) Science 295, 1852&#226;&#8364;&#8220;1858.                           26.   Sikorski, R. S. &amp; Boeke, J. D. (1991) Metholds Enzymology: Guide Yeast <br />  6.   Cheetham, M. E. &amp; Caplan,  J. (1998) Cell Stress Chaperones 3, 28&#226;&#8364;&#8220;36.                    Genetics Molecular Biology, eds. Guthrie, C. &amp; Fink, G. R., Vol. 194, pp. <br />  7.   Kelley, W. (1999) Curr. Biol. 9, R305&#226;&#8364;&#8220;R308.                                                302&#226;&#8364;&#8220;318. <br />  8.   Genevaux, P., Schwager, F., Georgopoulos, C. &amp; Kelley, W. L. (2002) Genetics         27.   Ryan, M. T., Voos, W. &amp; Pfanner, N. (2001) Methods Cell Biol. 65, 189&#226;&#8364;&#8220;215. <br />       162, 1045&#226;&#8364;&#8220;1053.                                                                      28.   Kang, P. J., Ostermann, J., Shilling, J., Neupert, W., Craig, E.  &amp; Pfanner, <br />  9.   Brodsky, J. L. &amp; Schekman, R. (1993) J. Cell Biol. 123, 1355&#226;&#8364;&#8220;1363.                         N. (1990) Nature 348, 137&#226;&#8364;&#8220;143. <br /> 10.   Matlack, K. E. S., Plath, K., Misselwitz, B. &amp; Rapoport, T.  (1997) Science        29.   Miroux, B. &amp; Walker, J. E. (1996) J. Mol. Biol. 260, 289&#226;&#8364;&#8220;298. <br />       277, 938&#226;&#8364;&#8220;941.                                                                        30.   Horst, M., Oppliger, W., Rospert, S., Schonfeld, H.-J., Schatz, G. &amp; Azem,  <br /> 11.   Corsi,  K. &amp; Schekman, R. (1997) J. Cell Biol. 137, 1483&#226;&#8364;&#8220;1493.                           (1997) EMBO J. 16, 1842&#226;&#8364;&#8220;1849. <br /> 12.   Hermann, J., Stuart, R., Craig, E. &amp; Neupert, W. (1994) J. Cell Biol. 127,           31.   Dutkiewicz, R., Schilke, B., Knieszner, H., Walter, W., Craig, E.  &amp; <br />       893&#226;&#8364;&#8220;902.                                                                                   Marszalek, J. (2003) J. Biol. Chem. 278, 29719&#226;&#8364;&#8220;29727. <br /> 13.   Rowley, N., Prip-Buus, C., Westermann, B., Brown, C., Schwarz, E., Barrell, B.       32.   Liu, Q., Krzewska, J., Liberek, K. &amp; Craig, E.  (2001) J. Biol. Chem. 276, <br />       &amp; Neupert, W. (1994) Cell 77, 249&#226;&#8364;&#8220;259.                                                     6112&#226;&#8364;&#8220;6118. <br /> 14.   Lutz, T., Westermann, B., Neupert, W. &amp; Herrmann, J. (2001) J. Mol. Biol. 307,       33.   Montgomery, D., Morimoto, R. &amp; Gierasch, L. (1999) J. Mol. Biol. 286, <br />       815&#226;&#8364;&#8220;825.                                                                                   915&#226;&#8364;&#8220;932. <br /> 15.   Voisine, C., Cheng, Y. C., Ohlson, M., Schilke, B., Hoff, K., Beinert, H.,           34.   Lim, J. H., Martin, F., Guiard, B., Pfanner, N. &amp; Voos, W. (2001) EMBO J. 20, <br />       Marszalek, J. &amp; Craig, E. (2001) Proc. Natl. Acad. Sci. USA 98, 1483&#226;&#8364;&#8220;1488.                 941&#226;&#8364;&#8220;950. <br /> 16.   Westermann, B. &amp; Neupert, W. (1997) J. Mol. Biol. 272, 477&#226;&#8364;&#8220;483.                      35.   Cheng, M., Hartl, F., Martin, J., Pollock, R., Kalousek, F., Neupert, W., <br /> 17.   Merlin, , Voos, W., Maarse, , Meijer, M., Pfanner, N. &amp; Rassow, J. (1999)              Hallberg, E., Hallberg, R. &amp; Horwich,  (1989) Nature 337, 620&#226;&#8364;&#8220;625. <br />       J. Cell Biol. 145, 961&#226;&#8364;&#8220;972.                                                          36.   Gambill, B. D., Voos, W., Kang, P. J., Miao, B., Langer, T., Craig, E.  &amp; <br /> 18.   Krimmer, T., Rassow, J., Kunau, W. H., Voos, W. &amp; Pfanner, N. (2000) Mol.                  Pfanner, N. (1993) J. Cell Biol. 123, 109&#226;&#8364;&#8220;117. <br />       Cell. Biol. 20, 5879&#226;&#8364;&#8220;5887.                                                           37.   Schneider, H. C., Westermann, B., Neupert, W. &amp; Brunner, M. (1996) EMBO <br /> 19.   Strub, , Rottgers, K. &amp; Voos, W. (2002) EMBO J. 21, 2626&#226;&#8364;&#8220;2635.                           J. 15, 5796&#226;&#8364;&#8220;5803. <br /> 20.   Liu, Q., D&#226;&#8364;&#8482;Silva, P., Walter, W., Marszalek, J. &amp; Craig, E.  (2003) Science 300,   38.   Rapoport, T., Matlack, K., Plath, K., Misselwitz, B. &amp; Staeck, O. (1999) Biol. <br />       139&#226;&#8364;&#8220;141.                                                                                   Chem. 380, 1143&#226;&#8364;&#8220;1150. <br /> 21.   Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19&#226;&#8364;&#8220;27.                             39.   Klostermeier, D., Seidel, R. &amp; Reinstein, J. (1999) J. Mol. Biol. 287, 511&#226;&#8364;&#8220;525. <br />  <br />  <br />  <br />  <br /> 13844 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.1936150100                                                             <br /> </body></html>