<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>IRE1a Disruption Causes Histological Abnormality Exocrine Tissues, Increase Blood Glucose Level, Decrease Serum Immunoglobulin Level <br />  <br />      Abstract <br />      Accumulation unfolded proteins endoplasmic reticulum (ER) causes ER stress. cellular adaptive response ER <br />      stress, unfolded protein response (UPR) activates molecules quality control ER proteins. enzyme plays      important role UPR Inositol Requiring Enzyme-1 (IRE1), highly conserved yeast humans. particular, <br />      mammalian IRE1a activates X-box-binding protein 1 (XBP1) unconventional splicing XBP1 mRNA ER stress. <br />      analysis knockout mice, IRE1a XBP1 shown essential development XBP1      necessary secretory machinery exocrine glands, plasma cell differentiation, hepatic lipogenesis.       essentiality IRE1a specific organs tissues remains incompletely understood.  analyzed phenotype      IRE1a conditional knockout mice IRE1a-deficient mice exhibit mild hypoinsulinemia, hyperglycemia,      low-weight trend.  IRE1a disruption causes histological abnormality pancreatic acinar salivary serous <br />      tissues decrease serum level immunoglobulin produced plasma cells, dysfunction liver. <br />      Comparison report previous reports regarding XBP1 conditional knockout mice provide clues      discovery novel functions IRE1a XBP1. (196 words) <br />  <br />  <br />  <br /> Introduction                                                                           ER-located type transmembrane protein kinase domain <br />                                                                                        RNase domain cytosolic region. exposed ER <br />    majority secretory proteins, antibodies,                       stress, trans-autophosphorylation activation RNase <br /> digestive enzymes, hormones, synthesized cytoplasm                      domain, IRE1 induces unconventional splicing mRNA cotranslationally translocated lumen                           encoding specific transcription factor responsible UPR <br /> endoplasmic reticulum (ER) narrow channel called                             activation [4&#226;&#8364;&#8220;8]. IRE1 highly conserved yeast humans, <br /> translocon ER membrane, initially located                       IRE1 paralogues reported mammals: IRE1a <br /> ER unfolded unmodified nascent polypeptides.                              IRE1b [9&#226;&#8364;&#8220;11]. Specifically, IRE1a induces unconven- <br /> proteins undergo meticulous folding molecular chaper-                          tional splicing XBP1 mRNA ER stress condition [12]. <br /> ones, correct disulfide bond formation protein disulfide                            spliced XBP1 mRNA translated functional <br /> isomerases, proper oligosaccharide modification                             transcription factor induce UPR. IRE1, ER-located <br /> oligosaccharyltransferase complex, sugar trimming enzymes,                         transmembrane proteins, PERK ATF6, play important roles <br /> calnexin/calreticulin cycle ER [1,2].  cells                     mammalian UPR [13,14]. sensing ER stress, PERK induces <br /> produce proteins large amounts, ER thought                       phosphorylation eIF2a translational activation liable overloaded maturation proteins.                      ATF4 [15]. hand, ER stress condition, ATF6 <br /> Accumulation unfolded proteins ER causes ER                             cleaved Site-1 Site-2 proteases, cytoplasmic <br /> stress. adaptively respond ER stress, cell induces                       domain translocated nucleus [16,17]. ATF4 transcriptional activation molecules maturation                          cleaved ATF6 work transcription factors UPR induction, proteins ER. response called unfolded protein                           XBP1 activated IRE1a. <br /> response (UPR) [3].  UPR important cellular response                           described  IRE1a directly catalyzes cleavage mass production functional secretory proteins                          XBP1 mRNA splicing reaction ER stress condition <br /> unfolded proteins cells produce large amounts.                        [12]. knowledge, reaction exclusively dependent    <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, molecules reported play                               IRE1a activity detected IRE1a-deficient cells [18]. <br /> important roles UPR. molecules, IRE1,                            implies IRE1a XBP1 function signal <br />  <br />  <br />        PLoS | www.plosone.org                                                  1                            September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                        Phenotype IRE1a CKO Mice <br />  <br />  <br /> transduction pathway ER stress response.  IRE1a knockout            bromophenol blue), lysate heated 98uC 10 min. <br /> (KO) mice XBP1 KO mice commonly embryonic                          SDS-PAGE (10% gel) used resolve proteins lethality IRE1a XBP1 play essential role            lysate. electrophoresis, proteins electrotransferred <br /> mammalian development [19&#226;&#8364;&#8220;21].  embryonic                  polyvinylidene fluoride microporous membranes. Immuno- <br /> lethality XBP1 KO mice rescued XBP1 transgene                 detection amylase glyceraldehyde 3-phosphate dehydroge- <br /> specifically expressed liver [22], IRE1a KO mice          nase (GAPDH) performed using rabbit polyclonal anti- <br /> rescued endogenous IRE1a specifically expressed                 amylase antibody (Sigma) mouse monoclonal anti-GAPDH <br /> extra-embryonic tissues liver [18]. suggests       antibody (Cell signaling), respectively. known IRE1a-dependent XBP1 function XBP1-independent IRE1a function(s) exists extra-embry-               Analysis B cell differentiation <br /> onic tissues IRE1a-independent XBP1 function(s)                Serum levels IgM IgG1 measured ELISA <br /> exists fetal liver.  comparison analysis                   (Bethyl Laboratories ) according manufacturer&#226;&#8364;&#8482;s <br /> conventional conditional KO mice terms IRE1a                  instruction. Flow cytometry analysis performed FACS <br /> XBP1 provide clues discovery                    vantage SE (Becton Dickinson) FlowJo ver. 6.3.2 software. additional tissue-specific functions molecule. Analysis          analysis, single-cell suspensions spleen prepared <br /> XBP1 conditional KO mice, including XBP1 KO mice rescued                    according protocol Beckman Coulter Japan. Splenocytes XBP1 transgene specifically expressed liver,                 stained PE-conjugated IgD antibody (Becton <br /> previously demonstrated XBP1 required secretory             Dickinson), FITC-conjugated IgM antibody (eBioscience), machinery exocrine glands, plasma cell differentiation,              APC-conjugated B220 antibody (Becton Dickinson). <br /> hepatic lipogenesis [22&#226;&#8364;&#8220;24].  remains unclear IRE1a plays essential function biological phenomena.           Measurement transaminase activity serum elucidate  analyzed phenotype IRE1a conditional              aspartate aminotransferase (AST) alanine amino- <br /> KO mice study.                                                      transferase (ALT) activities serum measured using                                                                             transaminase assay kit (Wako). <br /> Methods <br /> IRE1a conditional KO mice                                                   Quantitative PCR analysis <br />                                                                                Quantitative PCR analysis transcript performed using <br />   previously described, generated viable IRE1a conditional <br />                                                                             TaqMan probe 7900HT (Applied Biosystems) accordance KO mice (Mox2+/Cre; IRE1aDNeo/DR) control mice (Mox2+/+; <br />                                                                             manufacturer&#226;&#8364;&#8482;s instructions GAPDH transcript internal <br /> IRE1aDNeo/DR) breeding Mox2+/Cre; IRE1a+/DR mice                                                                             control. Results expressed mean 6 S.E.M triplicate <br /> Mox2+/+; IRE1aDNeo/DNeo mice [18]. IRE1a conditional KO mice <br />                                                                             experiments using RNA isolated independent tissues. control mice born near-Mendelian ratios. mice <br />                                                                             probe/primer set,  Mm00469005_m1, Mm00499536_m1, <br /> used experiment maintained mixed (C57BL/6 x <br />                                                                             Mm00772290_m1, Mm01204691_m1  Applied Biosys- <br /> 129/SvE) background. Experimental protocols involving animals <br />                                                                             tems) used quantification IRE1b, <span id='am-28' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span>diacylglycerol</span> O- approved Animal Studies Committees RIKEN                                                                              acyltranserase 2 (Dgat2), stearoyl-coenzyme desaturase 1 (Scd1), permit number; H22-1-105) NAIST  permit number; <br />                                                                             acetyl-coenzyme carboxylase 2 (Acc2) transcripts, respectively. <br /> 1011). <br />                                                                             IRE1a transcripts quantified using forward primer: 59-ggt cca <br />                                                                             atc gta cgg cag tt-39, reverse primer: 59-tct ctc aca gag cca cct ttg <br /> Measurement blood glucose insulin                                    tag-39, probe: 59-FAM-cct gca gac aga tct-MGB-39. Total XBP1 <br />    Blood glucose level measured using portable glucose                transcripts quantified using forward primer: 59-gaa tgg aca cgc <br /> measuring device (Arkray). Insulin level determined                  tgg atc ct-39, reverse primer: 59-gcc acc agc ctt act cca ctc-39, enzyme linked immunosorbent assay (ELISA) using mouse insulin               probe: 59-FAM-cct ctg gaa cct cg-MGB-39. Spliced XBP1 transcripts standard (Shibayagi). Glucose tolerance tests performed           quantified using forward primer: 59-gaa tgg aca cgc tgg atc 20-week-old IRE1a conditional KO control mice               ct-39, reverse primer: 59-cag agt cca tgg gaa gat gtt ct-39, probe: fasted 16 hours. Mice orally administered                59-FAM-cac ctg ctg cgg act-MGB-39. <br /> 2 mg/g body weight glucose. Blood glucose level serum <br /> insulin level measured indicated intervals. <br />                                                                             Results <br /> Histological analysis                                                       IRE1a-deficient mice exhibit mild hypoinsulinemia, <br />    tissue fixed 10% formalin embedded               hyperglycemia, low-weight trend <br /> paraffin. Paraffin blocks sliced 5-mm sections                  IRE1a conventional KO mice embryonic lethality stained hematoxylin eosin general histopath-               described earlier, embryo proper-restricted IRE1a conditional KO <br /> ological analysis. Immunohistochemical analysis performed               mice, specifically express IRE1a extra-embryonic <br /> using 6-mm paraffin sections. Immunoreactivity insulin             tissues, avoid embryonic lethality [18,20,21]. initially glucagons detected using guinea pig polyclonal anti-                measured body weight blood glucose level age 4 weeks insulin rabbit polyclonal anti-glucagon antibodies                 age 16 weeks examine growth process health condition Dako), respectively. Immunoreaction secondary                     embryo proper-restricted IRE1a conditional KO mice (hereinafter <br /> antibodies chromogenic reaction 3,39-diaminobenzi-                 IRE1a CKO mice). IRE1a CKO mice exhibited mild hyperglycemia <br /> dine tetrahydrochloride performed according standard                low-weight trend compared control mice free <br /> procedures.                                                                 feeding conditions (Figure 1A), difference                                                                             body length, feed water intake day IRE1a <br /> Western blot analysis                                                       CKO control mice (Table S1). investigate blood glucose <br />   tissues lysed SDS sample buffer (50 mM Tris-HCl               regulatory ability IRE1a CKO mice, performed oral glucose <br /> pH 6.8, 2% SDS, 50 mM DTT, 10% glycerol, 1 mg/ml                        tolerance test (OGTT). measurement blood insulin level <br />  <br />  <br />       PLoS | www.plosone.org                                        2                         September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                           Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 1. Comparison body weight, blood glucose level, plasma insulin level.   Weekly changes body weight (left) blood <br /> glucose level (right) ad libitum feeding condition 4 16 weeks birth. solid line squares broken line circles refer control IRE1a CKO mice data, respectively. upper figures lower figures refer data male female mice, <br /> respectively. plots indicate mean error bars denote S.E.M (n = 7&#226;&#8364;&#8220;10). Statistical significance differences control IRE1a CKO <br /> mice determined Student&#226;&#8364;&#8482;s t-test (*p,0.05, **p,0.01). (B) Changes blood glucose level (left) blood insulin level (right) oral <br /> administration glucose. solid line squares broken line circles refer control IRE1a CKO mice data, respectively. upper figures lower figures refer data male female mice, respectively. Measurement performed 20 weeks old. plots <br /> indicate mean error bars denote S.E.M (n = 7&#226;&#8364;&#8220;10). Statistical significance differences control IRE1a CKO mice determined Student&#226;&#8364;&#8482;s t-test (*p,0.05, **p,0.01). <br /> doi:10.1371/journal.pone.0013052.g001 <br />  <br />  OGTT revealed insulin level peak decreased             (Figure 3A 3B Figure S1). stained sections revealed 30,50% IRE1a CKO mice compared control mice.                      salivary gland serous acini IRE1a CKO mice half hand, measurement blood glucose level OGTT                   size compared control mice;  size showed IRE1a CKO mice significantly higher blood                     salivary gland mucous acini IRE1a CKO mice similar <br /> glucose level peak control mice IRE1a CKO mice              control mice (Figure 3C 3D Figure S2 required longer time blood glucose level recover control           S3). hand, pancreatic acinar tissue IRE1a <br /> mice (Figure 1B).  measured level insulin                     CKO mice showed local spongy lesion loss acinar cells <br /> extracted pancreas islets IRE1a CKO mice ELISA                (Figure 3E Figure S4), started occurring weeks total proteins used internal                    birth. suspected pathological change caused standard. addition, performed immunohistochemical analysis                 apoptosis;  TUNEL analysis showed difference insulin glucagons tissue sections pancreas               IRE1a CKO mice control mice (Table S3).  IRE1a CKO mice. level insulin extracted pancreas                 examined effects IRE1aon production digestive islets IRE1a CKO mice quite similar control                enzymes pancreas salivary glands. Amylase mice (Table S2). signals insulin glucagons pancreas            key digestive enzymes produced pancreatic acinar IRE1a CKO mice normally detected (Figure 2A). Number                     salivary gland serous cells. Western blot analysis performed using size islets ratio insulin+ cells glucagons+ cells       tissue lysate saliva revealed difference islet nearly equal IRE1a CKO control mice                  amylase expression secretion IRE1a CKO mice <br /> (Figure 2B).  histological difference islets             control mice (Figure 3F). IRE1a CKO control mice. <br />                                                                                  IRE1a necessary differentiation B cells IRE1a disruption causes histological abnormality                          plasma cells <br /> pancreatic acinar salivary gland serous tissues                                 Previous studies demonstrated IRE1a activated <br />    described  embryonic lethality XBP1 KO mice                   differentiation B cells plasma cells rescued expression liver-specific XBP1                      XBP1 played essential role process [21,23,25]. <br /> transgene;  mice die days birth. death              investigate roles IRE1a differentiation B cells thought caused dysfunction digestion-associated               mice, serum IgM IgG1 populations IgM+, <br /> exocrine glands, pancreas salivary glands [22].                  IgD+ B220+ cells splenocytes analyzed IRE1a  investigated size histological phenotype            CKO mice. Serum IgM IgG1 serve markers plasma cell <br /> pancreas salivary glands IRE1a CKO mice.                    differentiation, IgM, IgD B220 serve markers size salivary glands IRE1a CKO mice normal,                mature B cells. ELISA revealed serum levels IgM IgG1 pancreas 30,50% smaller control mice                  reduced half IRE1a CKO mice compared  <br />  <br />        PLoS | www.plosone.org                                            3                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                     Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 2. Histological analysis islets IRE1a CKO mice control mice.   upper, middle, lower panels stained <br /> sections, immunostained images using anti-insulin antibodies, immunostained images using anti-glucagon antibodies, respectively. images control IRE1a CKO mice derived serial sections (scale bar: 50 mm). (B) aThe number islets counted 1 cm2 pancreas section shown mean 6 S.E.M (n = 10 pancreas sections). bThe diameter (mm) islet shown mean 6 S.E.M (n = 50). cThe ratio insulin+ cells glucagons+ cells islet shown mean 6 S.E.M (n = 10). data obtained 20 weeks old female mice. <br /> doi:10.1371/journal.pone.0013052.g002 <br />  <br /> control mice. <span id='am-18' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span>Lipopolysaccharide</span> (<span id='am-8' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-12' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="LPS" datatype="xsd:string"></span>LPS</span>) stimulation increases            immunoglobulin produced plasma cells, does concentration immunoglobulins  fold;                  effect differentiation mature B cells.  case LPS stimulation, levels IgM IgG1 approximately half IRE1a CKO mice                      Hepatic function IRE1a-deficient mice normal <br /> compared control mice (Figure 4A). Flow cytometry                     Hepatic abnormality reported embryo analysis splenocytes showed populations positive          IRE1a KO mice XBP1 KO mice [19,21].  XBP1 <br /> negative cells IgM, IgD B220 markers similar                   shown necessary <span id='am-2' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-3' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> metabolism IRE1a CKO mice control mice (Figure 4B).                hepatocytes mature mice [24]. hand, suggests IRE1a disruption causes decrease serum level          histological analysis embryonic liver IRE1a CKO mice <br />  <br />  <br />        PLoS | www.plosone.org                                       4                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                    Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> PLoS | www.plosone.org   5   September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                        Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br /> Figure 3. Effect deletion IRE1a pancreatic exocrine gland salivary glands.   Comparison raw weight (g) organs IRE1a CKO mice control mice (20 weeks old). Data expressed mean 6 S.E.M (n = 7&#226;&#8364;&#8220;10). (B) Pancreas collected mice. pancreas come spleen small intestine (scale bar: 1 cm). (C) stained sections serous tissue salivary glands (scale bar: 200 mm). longest diameter serous acini shown mean 6 S.E.M (n = 50). Statistical significance differences control IRE1a CKO mice determined Student&#226;&#8364;&#8482;s t-test (p,0.005). (D) stained sections mucous tissue salivary <br /> glands (scale bar: 100 mm). longest diameter mucous acini shown mean 6 S.E.M (n = 50). (E) stained sections pancreatic acinar <br /> tissue (scale bar: 200 mm). (F) Western blot analysis amylase pancreas, salivary glands saliva. genotype, samples derived mice used. saliva sample used equal volume. GAPDH used internal standard. data B-F obtained 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g003 <br />  <br /> previously failed abnormality [18], performed                     abnormality insulin secretion. Insulin prevented entering <br /> phenotype analysis liver IRE1a CKO mature mice                  secretory pathway quality control ER proteins measurement serum AST ALT, observation stained                  does receive proper modifications (e.g., folding disulfide <br /> sections, analysis expression lipid metabolism-related         bond formation) biosynthesis secretion. possible <br /> genes.  clear differences                  IRE1a indirectly involved maturation process IRE1a CKO mice control mice (Figure 5).                              insulin ER inducing expression chaperones, <br />                                                                              disulfide isomerase enzymes. hand, based <br /> Functional IRE1a mRNA significantly disrupted IRE1a                    vitro analysis effect IRE1a insulin, suppression CKO mice                                                                     IRE1a reported cause low insulin biosynthesis    Quantitative PCR analysis IRE1a XBP1 various                    translation level little impact insulin secretion [28]. tissues performed determine degree functional                  downregulation IRE1a reported suppress <br /> disruption IRE1a mRNA IRE1a CKO mice.                   insulin mRNA degradation occurs chronic high- <br /> tissues tested, PCR signals IRE1a detected control            glucose environment [29].  performed <br /> mice previously reported [18], weakly detected          experiments confirm findings, easy IRE1a CKO mice (Figure 6A). PCR signals total                    prepare samples mice detailed experiments XBP1 similarly detected control IRE1a CKO               insulin biosynthesis; theory, IRE1a CKO mice used <br /> mice (Figure 6B), spliced XBP1 slightly obviously           25% chance born. future, detected control mice, little IRE1a CKO mice               aim analyze functions IRE1a association (Figure 6C 6D). case mice injected                 insulin biosynthesis b cells islets generating mice tunicamycin, ER streesor, spliced XBP1 level IRE1a                 IRE1a disrupted specifically b cells. previously <br /> CKO mice hardly detected tested tissues,         reported loss XBP1 effect development control mice strongly increased liver kidney slightly         islets [22] XBP1+/2 mice tended high blood <br /> increased tissues (Figure 6C 6D). addition,                  glucose levels [30].  remains unclear IRE1a IRE1b specifically expressed gastrointestinal             XBP1 function terms insulin biosynthesis/secretion tract [26], IRE1b expression examined tissues             control blood glucose level. words, uncertain <br /> quantitative PCR. expected, PCR signals IRE1b                     IRE1a XBP1 function pathway predominantly detected gastrointestinal tract             independently. answer questions, need generate <br /> control IRE1a CKO mice, slightly tissues                mice expressing active (spliced) form XBP1. believe (Figure 6E). results confirm functional IRE1a mRNA                valuable information functional relationship spliced XBP1 mRNA significantly disrupted IRE1a               IRE1a XBP1 obtained analysis insulin <br /> CKO mice suggest IRE1b complement role                  biosynthesis/secretion blood glucose level offspring IRE1a IRE1a CKO mice. hand, IRE1a                     produced crossing IRE1a KO mice XBP1 KO mice mRNA spliced XBP1 mRNA slightly detected                    mice expressing spliced XBP1. <br /> tissues IRE1a CKO mice.  IRE1a completely                    Size reduction acinar tissue partial deletion acinar cells <br /> disrupted IRE1a CKO mice, reason                 pancreas size reduction serous acini salivary phenotype IRE1a CKO mice clear-cut.                            gland observed IRE1a CKO mice. pancreatic <br />                                                                              acinar tissue salivary gland serous tissue representative <br /> Discussion                                                                   organs secretion digestive enzymes produce large <br />                                                                              proteins amylase. insulin, amylase    ER stress associated various diseases [27], vivo       prevented entering secretory pathway quality control functions ER stress-related molecules presently receiving             ER proteins does receive proper modifications ER <br /> attention. regard IRE1a functional analysis              (e.g., folding disulfide bond formation) biosynthesis <br /> molecule adult KO mice lagging KO mice               secretion. IRE1a activated pancreatic acinar tissue suffer embryonic lethality;  using            salivary gland physiological conditions excessive <br /> conditional KO method IRE1a-deficient mice exhibit mild                 expression amylase causes ER stress IRE1a activation <br /> hypoinsulinemia, hyperglycemia, low-weight trend,             [31,32]. Based previous findings, possible IRE1a IRE1a disruption causes histological abnormality pancreatic           involved homeostasis ER development acinar salivary serous tissues decrease serum level            survival exocrine cells pancreatic acinar tissue salivary <br /> immunoglobulin produced plasma cells.                                 gland constantly inducing expression chaperones, disulfide <br />    present study, level insulin protein extracted         isomerase enzymes. described  developmental pancreas islets similar IRE1a CKO mice               disorder pancreatic acinar tissue salivary gland control mice. Presumably, insulin expression level                observed XBP1 CKO mice [22].  signal transduction <br /> normal IRE1a-deficient b cells.  lower level             IRE1a XBP1 important development blood insulin IRE1a CKO mice                         maintenance pancreatic acinar tissue salivary gland. <br />  <br />  <br />        PLoS | www.plosone.org                                        6                        September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                       Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 4. Effect deletion IRE1a B cells.   Serum immunoglobulin levels measured ELISA. Columns indicate mean error bars <br /> denote S.E.M (n = 5). data obtained 8 weeks old female mice. (B) Flow cytometry analysis splenocytes using IgM, IgD B220 markers. values corners chart indicate proportions positive negative cells marker. Reproducible data separate experiments shown. data obtained 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g004 <br />   unlike IRE1a CKO mice, phenotypes severe                indigestion food, death shortly birth [22]. XBP1 CKO mice exemplified observation apoptotic                  indicate importance IRE1a-independent XBP1 functions signals pancreatic acinar tissue, reduced production               development maintenance pancreatic acinar <br /> digestive enzymes pancreatic acinar tissue salivary gland,        tissue salivary gland. <br />  <br />  <br />        PLoS | www.plosone.org                                        7                       September 2010 | Volume 5 | Issue 9 | e13052 <br />                                                                                                                       Phenotype IRE1a CKO Mice <br />  <br />  <br />  <br />  <br /> Figure 5. Effect deletion IRE1a liver.   stained sections liver tissue (scale bar: 50 mm). (B) Comparison serum <br /> AST ALT levels IRE1a CKO mice control mice. Data expressed mean 6 S.E.M (n = 5). (C) Quantitative PCR analysis lipid synthesis genes liver mice fed normal high-fructose feed. Columns indicate mean error bars denote S.E.M (n = 3). data obtained 20 weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g005 <br />  <br />    Like hormones (e.g., insulin glucagons) digestive                 histological abnormalities embryonic liver IRE1a CKO <br /> enzymes, antibodies secretory proteins prevented            mice described [18].  XBP1 shown entering secretory pathway quality control ER               essential lipid metabolism hepatic cells mature mice <br /> proteins receive proper modifications ER (e.g.,        [24]. present study, hepatic abnormalities folding disulfide bond formation) biosynthesis          analysis IRE1a CKO mice. suggest existence secretion.  IRE1a believed play important role            IRE1a-independent XBP1 functions. XBP1 exerts functions plasma cells terms production quality                    depending IRE1a, splicing XBP1 mRNA IRE1a antibodies. present study, serum levels IgM                necessary.  splicing XBP1 mRNA blocked IgG1 IRE1a CKO mice half control                 inactivation IRE1a [18]. Given observations, mice, trend observed regardless presence              functions IRE1a-independent XBP1 brought LPS stimulation. Previous studies revealed IRE1a activation             unspliced XBP1. differentiation B cells plasma cells                Phenotype analysis IRE1a CKO mice study provided <br /> XBP1 played essential role differentiation process                 important information regarding physiological functions [21,23,25]. Based findings, IRE1a contribute             IRE1a. Comparison results phenotypes XBP1 <br /> normal production antibodies activating XBP1 known                  KO mice previously reported indicates possibilities IRE1a <br /> UPR pathways, inducing expression chaperones,                 functions independent XBP1 XBP1 functions <br /> disulfide isomerase enzymes ER.                             independent IRE1a. study provided findings    Hepatic abnormalities embryos IRE1a KO XBP1                 important revealing new roles played IRE1a XBP1 <br /> KO mice reported [19,21], absence                     vivo. <br />  <br />  <br />  <br /> Figure 6. Quantitative PCR analysis IRE1a, IRE1b, XBP1.   IRE1a. (B) Total XBP1. (C) Spliced XBP1. (D) Ratio spliced XBP1 total <br /> XBP1. (E) IRE1b. Tun 2 + indicates p. injection saline tunicamycin (500 ng/g body weight) 16 tissue collection. Li; liver, K; kidney, <br /> Sp; spleen, H; heart, Lu; lung, B; brain, Sa; salivary gland, St; stomach,  intestine,  colon, Pa; pancreas,  islet. data obtained 20 <br /> weeks old male mice. <br /> doi:10.1371/journal.pone.0013052.g006 <br />  <br />    regard, possible discrepancies                        doi:10.1371/journal.pone.0013052.s004 (0.62 MB TIF) present data Glimcher&#226;&#8364;&#8482;s [22,24]                                                                                             Figure S2 Fig. 3C enlarged presentation fine <br /> Kaufman&#226;&#8364;&#8482;s [21] groups depends difference                                                                                             histological structures. distinctive serous acini panel <br /> experimental condition. example, conditional KO mice <br />                                                                                             surrounded dashed line ease comparing size exogenous target gene [21,22], exogenous target gene <br />                                                                                             acini control IRE1a CKO mice. different endogenous target gene expression <br /> level unexpected dominant effects, unlikely                                  doi:10.1371/journal.pone.0013052.s005 (1.06 MB TIF) <br /> conditional KO mice flox/Cre   difference                            Figure S3 Fig. 3D enlarged presentation fine <br /> mouse genetic background raise distinct compensatory                                    histological structures. distinctive mucous acini panel <br /> mechanism UPR pathways IRE1a XBP1                                      surrounded dashed line ease comparing size CKO mice mice commonly generated flox/                               acini control IRE1a CKO mice. <br /> Cre  need carefully analyze                                       doi:10.1371/journal.pone.0013052.s006 (1.03 MB TIF) <br /> phenotypes IRE1a XBP1 disruption identical <br /> experimental condition future.                                                       Figure S4      Fig. 3E enlarged presentation fine <br />                                                                                             histological structures. Ac; acinar tissue,  islet. local spongy <br />                                                                                             lesion (unstained area) loss acinar cells impressive Supporting Information <br />                                                                                             pancreatic tissue IRE1a CKO compared control <br /> Table S1 Body length, feed intake, water intake.                                        mice.  doi:10.1371/journal.pone.0013052.s001 (0.03 MB                                     doi:10.1371/journal.pone.0013052.s007 (0.95 MB TIF) <br /> DOC) <br /> Table S2 Insulin content islet pancreas.                             doi:10.1371/journal.pone.0013052.s002 (0.03 MB                                  <br /> DOC)                                                                                <br /> Table S3 Total number TUNEL positive cells acinar <br /> cells tissue section.                                                           doi:10.1371/journal.pone.0013052.s003 (0.03 MB                                    <br /> DOC)                                                                                        <br /> Figure S1 Fig. 3B enlarged presentation fine                                 <br /> histological structures. Sp; spleen,  intestine, Pa; pancreas, surrounded dashed line. <br /> </body></html>