<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Rad18 Required DNA Repair Checkpoint <br /> Responses Fission Yeast <br />  <br />           survive damage genome, cells respond activating DNA repair           checkpoint responses. Using genetic screens fission yeast Schizosaccharomyces pombe,           recently isolated new genes required DNA damage checkpoint control.           strains defines new allele previously described rad18 gene, rad18-74. rad18           essential gene, absence extrinsic DNA damage. encodes conserved protein <br />           related structural maintenance chromosomes proteins. Point mutations rad18 lead           defective DNA repair pathways responding UV-induced lesions   <br />           double-stranded breaks. Furthermore, rad18p required maintain cell cycle arrest           presence DNA damage, failure leads highly aberrant mitoses. gene encoding <br />           BRCT-containing protein, brc1, isolated allele-specific high-copy suppressor           rad18-74. brc1 required mitotic fidelity cellular viability strains rad18 <br />           mutations essential DNA damage responses. Mutations rad18 brc1           synthetically lethal topoisomerase II mutant (top2-191), indicating proteins play <br />           role chromatin organization. studies role chromatin organization           maintenance activation responses DNA damage. <br />  <br />  <br /> INTRODUCTION                                                           mitosis. activation p34cdc2/cyclinB rate-limiting <br />                                                                        step entry mitosis. Analysis G2 DNA damage <br /> Strict control fidelity cell cycle events important       checkpoint fission yeast identified genes cells prevent heritable damage changes ploidy.           essential DNA damage checkpoint (Al-Khodairy importance control manifest occurrence           Carr, 1992; Enoch et al., 1992; Walworth et al., 1993; cell cycle checkpoints. Checkpoint pathways maintain            Al-Khodairy et al., 1994; Saka et al., 1997; Willson et al., 1997). <br /> interdependency cell cycle transitions pause cell cycle         checkpoint arrests cell cycle maintenance <br /> progression cellular defects detected. Checkpoints            inhibitory Y15 phosphorylation p34cdc2, described monitor DNA damage, completion                achieved chk1p-dependent signaling wee1p DNA replication, spindle integrity, cell mass, delay-           cdc25p (Furnari et al., 1997; O&#226;&#8364;&#8482;Connell et al., 1997; Peng et al., <br /> ing cell cycle progression requirements contin-          1997; Rhind et al., 1997; Sanchez et al., 1997). Chk1p uation cell cycle met (Nurse, 1975, 1994; Hartwell          phosphorylated DNA damage, dependent Weinert, 1989; Murray, 1995).                                      checkpoint rad genes, function upstream chk1 <br />   Entry mitosis controlled activity              (Walworth Bernards, 1996). Similar signaling pathways <br /> cdk/cyclin complex p34cdc2/cyclinB (Nurse, 1990).                 appear activated blocks DNA replication complex accumulates G2 maintained                  kinase, cds1p, involved transducing <br /> inactive state phosphorylation tyrosine 15              signals cell cycle machinery (Boddy et al., 1998; Lind- <br /> (Y15) p34cdc2 wee1p related kinases. p34cdc2              say et al., 1998; Zeng et al., 1998). <br /> comes active Y15 dephosphorylated cdc25p                   Maintenance genomic integrity requires DNA family phosphatases, cell proceeds                held highly structured proteinaceous complexes called <br />                                                                        chromatin. Chromatin undergoes structural alterations  <br />   &#194;&#167; <br />       Corresponding author. E-mail address: m.oconnell@pmci.unimelb.   incident DNA replication chromosome segrega- <br />       edu.au.                                                          tion, proteins involved maintaining  <br />  <br /> &#194;&#169; 1999 American Society Cell Biology                                                                                      2905 <br />  H.M. Verkade et al. <br />  <br />  <br /> altering chromatin structure essential stability        G2 mitosis, addition role sister <br /> integrity genome  review, Heck, 1997).               chromatid resolution (Uemura et al., 1987; Holm et al., 1989). S G2 phases, sister chromatids associate                SMC1&#226;&#8364;&#8220; 4 proteins central chromatin <br /> tightly lengths. Perturbation sister chromatin       organization. class proteins structurally <br /> cohesion results chromosome instability chromosome           related SMC1&#226;&#8364;&#8220; 4 defined rad18p fission yeast breaks cause inability repair double-stranded         Rhc18p budding yeast (Lehmann et al., 1995). Rad18 breaks (Birkenbihl Subramani, 1992; Strunnikov et al.,          essential gene. role inferred phenotype 1993; Guacci et al., 1997; Michaelis et al., 1997; Losada et al.,   point mutant rad18-X, shows sensitivity ionizing <br /> 1998). Xenopus, 9S 14S protein complexes, cohesins,          UV-C irradiation, inability repair UV-C-induced required generate maintain cohesion (Losada         lesions, chromosomal instability accompanied accu- <br /> et al., 1998). Cohesins contain structural maintenance       mulation mitotic defects (Murray et al., 1994; Lehmann et <br /> chromosomes (SMC) proteins XSMC1p XSMC3p, ho-                   al., 1995). Genetic studies place rad18 epistasis <br /> mologues required sister chromatid cohe-           group rad2 fission yeast RAD51 homologue rhp51 <br /> sion Saccharomyces cerevisiae (SMC1 SMC3) (Michaelis         (Lehmann et al., 1995). Mutations genes <br /> et al., 1997). 14S cohesin particle contains XRAD21p,      results chromosomal instability (Murray et al., 1994; homologues budding yeast (Scc1p/                  Lehmann et al., 1995).  rad2p required Mcd1p) fission yeast (rad21p) (Birkenbihl Subra-            repair UV-C-induced lesions, acts independently <br /> mani, 1995; Guacci et al., 1997; Michaelis et al., 1997; Losada     classical excision repair, rhp51p, analogy et al., 1998). budding yeast Scc1p/Mcd1p required          functional studies RAD51 proteins, required establish cohesion S-phase (Uhlmann Nasmyth,                 repair double-stranded breaks. pathways 1998), dissociates chromatin chromatids segregate,          linked requirement recombination pro- degraded (Guacci et al., 1997; Michaelis et al.,        cesses (Lehmann et al., 1995). structural similarity 1997; Ciosk et al., 1998). Mutations fission yeast rad21    SMC proteins, rad18p play role forming chro- <br /> lead phenotypes consistent defective sister        mosomal structures, including required different <br /> chromatid cohesion: chromosome instability, failure dou-         DNA repair mechanisms. <br /> ble-stranded break repair, defects chromosome seg-              DNA damage checkpoint genes described fis- <br /> regation (Birkenbihl Subramani, 1995; Tatebayashi et al.,       sion yeast, cut5 essential absence irradiation. <br /> 1998). key element required stable inheritence       possible genes involved aspects genomic <br /> chromosomes budding yeast scc2p             integrity checkpoint role reper- <br /> stoichimetric subunit cohesin, required           toire function. Proteins required detection DNA <br /> association cohesin chromosomes, suggesting             damage activation checkpoint repair pathways loads cohesin complex chromosomes              described. likely proteins (Toth et al., 1999). Likewise, mutations fission yeast       chromatin associated define essential genes. <br /> homologue mis4 lead defects sister chromatid cohesion          identify genes essential viability chromosome stability, mis4p appears func-           checkpoint function, specific alleles need tion mainly S-phase (Furuya et al., 1998). Strains           isolated, T45M mutation cut5, present containing temperature-sensitive mis4-242 allele           cut5 alleles isolated far (Saka et al., 1997; Verkade sensitivity UV-C DNA replication inhibitor hy-           O&#226;&#8364;&#8482;Connell, 1998). screening approach saturation <br /> droxyurea permissive temperature, G2 check-             genome needed specific mutations points intact (Furuya et al., 1998).                            essential genes lead checkpoint deficiencies    paired, sister chromatids intertwined,           alter  essential, functions. subsequent decatenation requires activity topo-           performed genetic screen fission yeast identify <br /> isomerase II. Failure resolve sister chromatids       new genes involved DNA damage checkpoint (Verk- <br /> segregation results chromosome breakage subse-               ade O&#226;&#8364;&#8482;Connell, 1998). strain <br /> quent instability genome loss genetic material.       isolated screen, previously termed NA74, defines extreme leads increased rates molecular           novel allele rad18 renamed rad18-74. Unlike <br /> evolution cell death (Holm et al., 1985, 1989; Uemura         previously characterized allele, rad18-X, rad18-74 Yanagida, 1986; Uemura et al., 1987; Downes et al., 1991).      defects DNA damage checkpoint, alleles    onset mitosis marked chromatin condensa-            defective DNA repair. Cells deleted rad18 tion, alteration chromatin organization compacts         unable arrest G2 presence DNA-damaging <br /> chromosomes fold. protein complexes chro-             agents. Expression allele mutated putative rad18 <br /> matin condensation activity, condensins, char-        <span id='am-11' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span>-binding domain failed rescue DNA damage sen- <br /> acterized Xenopus (Kimura Hirano, 1997).           sitivity rad18-X rad18-74, overexpression tain SMC proteins XCAP-Ep (SMC2 homologue)                  dominant negative lethal. indicates essential <br /> XCAP-Cp (SMC4 homologue) (Hirano Mitchison, 1994;               role ATP binding hydrolysis rad18p function. <br /> Hirano et al., 1997), homologues          Rad18 mutants phosphorylation chk1p  species. Loss function budding yeast              sponse DNA damage, event proposed XCAP-E homologue (SMC2) fission yeast SMC2               indicative checkpoint initiation.  rad18p  <br /> SMC4 homologues (cut14 cut3) causes inability             quired maintain checkpoint-mediated G2 arrest. condense chromatin complete chromosome segrega-              isolated allele-specific high-copy suppressor rad18-74. <br /> tion (Saka et al., 1994b; Strunnikov et al., 1995). Topoisomer-     gene, brc1, encodes BRCT domain&#226;&#8364;&#8220; containing protein <br /> ase II required chromatin condensation          (Koonin et al., 1996; Callebaut Mornon, 1997), deletion  <br />  <br /> 2906                                                                                                    Molecular Biology Cell <br />                                                                                                                             Fission Yeast rad18 <br />  <br />  leads genomic instability mitotic defects                tion, washed twice CSE (20 mM citrate/phosphate, pH 5.6, 40 defects DNA repair checkpoint function. Genetic                 mM EDTA, 1.2 M sorbitol), incubated 1 h 37&#194;&#176;C 1 ml <br /> interactions rad18, brc1, genes involved               CSE 1.5 mg Zymolyase 20-T (ICN, Costa Mesa, CA). cells <br /> chromatin organization suggest role rad18p                   washed CSE resuspended ;6 3 108 cells/ml regulation chromatin structure essential                   TSE (10 mM Tris, pH 7.5, 45 mM EDTA, 0.9 M sorbitol) 37&#194;&#176;C.                                                                           equal volume 1% low gelling temperature agarose TSE  genomic integrity responses DNA damage.                            55&#194;&#176;C) added, agarose plugs samples                                                                           set ice 15 min. plugs transferred solution 1% <br /> MATERIALS METHODS                                                     sarcosyl, 0.5 M EDTA, pH 9.5, 0.5 mg/ml proteinase K incu- <br />                                                                           bated 55&#194;&#176;C 48 h, readdition proteinase K 24 h. <br /> Fission Yeast Methods                                                     plugs stored 4&#194;&#176;C washed Tris-EDTA                                                                           loading. Sample plugs run 0.6% gel chromosomal strains derivatives 972h2 975h1. Standard procedures       grade agarose (Bio-Rad, Hercules, Ca) 0.53 Tris borate-EDTA media used propagation genetic manipulation              220 h 40 V pulse time 75 min Gene Navigator PFGE <br /> (Moreno et al., 1991). Methods transformation microscopy          apparatus (Pharmacia Biotech, Piscataway, NJ). described previously (O&#226;&#8364;&#8482;Connell et al., 1994). proce- <br /> dure used screen new genes involved DNA damage <br /> checkpoint described previously (Verkade O&#226;&#8364;&#8482;Connell,          Chromosome Loss Assays <br /> 1998). test temperature sensitivity strains genetic <br />                                                                           Mutant alleles crossed ade6-704 background analyses, strains streaked YES placed 25, 30,                                                                           crossed strain containing artificial minichromosome (Ch10), <br /> 36&#194;&#176;C 3&#226;&#8364;&#8220; 4 d. Strains grown 25&#194;&#176;C defined minimal <br />                                                                           contains ade6-704 tRNA suppressor sup3-5 (Niwa et al., <br /> media exponential phase shifted 30 36&#194;&#176;C. Samples                                                                           1989). resulting progeny grown 30&#194;&#176;C selective <br /> fixed ethanol 0.25% gluteraldehyde plus 3.7% <span id='am-29' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> <br />                                                                           conditions (minimal medium lacking <span id='am-6' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>), transferred (O&#226;&#8364;&#8482;Connell et al., 1994) stained DAPI microscopy <br />                                                                           nonselective conditions (YES) 29 generations, samples (Moreno et al., 1991). Immunolocalization rad18p tagged N <br />                                                                           plated minimal media containing 10 mg/l adenine. sample <br /> terminus myc expressed medium-strength nmt1 <br />                                                                           taken beginning time course used promoter performed using 9E10 Cy3-coupled anti-mouse <br />                                                                           background correction. 3 d 30&#194;&#176;C number red colonies <br /> immunoglobulin G (Krien et al., 1998). <br />                                                                           scored percentage total number colonies,                                                                           used determine rate chromosome loss, calculated UV Irradiation                                                            percentage colonies auxotrophic adenine divided                                                                           number generations. <br /> Cells grown exponential phase 30&#194;&#176;C plated YES, <br /> allowed dry, irradiated range doses UV-C using Stratalinker (Stratagene, La Jolla, CA). Colonies counted    Cloning NA74 Recombinant Techniques <br /> 3 d 30&#194;&#176;C. test ability cdc25-22 rescue radiation <br /> sensitivity, strains cdc25-22 background grown 25&#194;&#176;C,         Standard procedures used construction recombinant <br /> plated YES, irradiated  plates placed        plasmids (Sambrook et al., 1989). DNA sequencing performed <br /> directly 25&#194;&#176;C incubated 36&#194;&#176;C 4 h placed        double stranded templates processed ABI377 se- 25&#194;&#176;C. Colonies counted 4 &#226;&#8364;&#8220;5 d incubation 25&#194;&#176;C.           quencer (Applied Biosystems, Foster City, CA). NA74 strain  Checkpoint analysis performed cells synchronized            ura4-D18 background) transformed genomic library <br /> cdc25-22 block release protocol. Strains cdc25-22           pURSP1 (Barbet et al., 1992). transformants replica plated <br /> ground grown 25&#194;&#176;C, shifted 36&#194;&#176;C 3 h, irradiated       YES 0.01% methyl methanesulfonate (MMS) select 50 J/m2 UV-C described (O&#226;&#8364;&#8482;Connell et al., 1997), resuspended       complementation. DNA extracted complementing prewarmed media 25&#194;&#176;C. Cell cycle progression monitored          transformants used electroporate Escherichia coli. DAPI straining ethanol-fixed samples, counting number        plasmids retested complementing activity transforming <br /> cells passing mitosis, assessed cells mitosis, nuclei   NA74 strain assessing complementation. Complementation <br /> undergoing septation. proportion cells failing arrest       assessed sensitivity range doses UV-C irradiation <br /> estimated area curve spanning time            described . deletion versions suppressing clone <br /> control cells entered mitosis.                                            constructed removing internal 1.4-kb HindIII <br />                                                                           fragment 3.8-kb HindIII fragment extending end                                                                           clone religating 2.7-kb PmeI&#226;&#8364;&#8220;SmaI fragment, end filling <br /> Ionizing Radiation                                                        Klenow religating. ATP-binding domain mutation <br /> Exponentially growing cells 30&#194;&#176;C irradiated Varian         (G129A,K130R) constructed vitro mutagenesis (Kunkel et <br /> (Palo Alto, CA) linear accelerator. focus set 100 cm        al., 1987) rad18 cDNA following oligonucleotide: flasks, irradiated 6-MeV electron beam        59-GGTCATAATGGTAGCGCTAGAAGTGCTATACTTACAGGA- dose rate ;22 Gy/min. Samples taken plated             39. Base substitutions underlined create Eco47III site. <br /> YES, colonies counted 3 d 30&#194;&#176;C. Synchronous             Mutant (referred rad18-DN) wild-type rad18 cultures obtained using <span id='am-1' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span>lactose</span> synchronization exponen-          cloned strength nmt1 promoter (pREP1), comple- <br /> tially growing cells (Mitchison Carter, 1975). Ionizing radiation     mentation assessed presence thiamine. synchronous cultures delivered using Gammacell (Gamma <br /> Elektric, Stenl&#195;&#184;se, Denmark) 1000 137Cs source 12 Gy/min. Cell <br /> cycle progression monitored DAPI staining ethanol-fixed <br />                                                                           Spore Germination rad18::ura4 <br /> samples, counting number cells passing mitosis.                    Construction rad18 deletion strain described previ- <br />                                                                           ously (Lehmann et al., 1995). Sporulating colonies wild-type <br />                                                                           strain genotype ura4-D18/ura4-D18 leu1-32/leu1-32 ade6- <br /> Analysis Double-stranded Break Repair                                  M210/ade6-M216 his3-D1/his3-D1 h1/h2 rad18 deleted <br /> Double-stranded breaks induced exponentially growing              strain genotype rad181/rad18::ura4 ura4-D18/ura4-D18 leu1-32/ <br /> cells treating cultures 450 Gy using Varian linear        leu1-32 ade6-704/ade6-704 h1/h90 grown YE exponential <br /> accelerator described  culture placed 30&#194;&#176;C       phase starved minimal media lacking ammonium recover. milliliter samples harvested centrifuga-      30&#194;&#176;C induce sporulation. cell mix digested overnight  <br />  <br /> Vol. 10, September 1999                                                                                                                   2907 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                        0.05% b-glucuronidase washed water times; spores <br />                                                                        harvested layering mix 25% glycerol minimal <br />                                                                        media lacking ammonium, collected centrifugation 1500 3 g <br />                                                                        15 min, washed water. Spores (6 3 106) inocu- <br />                                                                        lated minimal media lacking <span id='am-34' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> rad18::ura4 spore <br />                                                                        culture including uracil wild-type spore culture.                                                                        allowed germinate 30&#194;&#176;C polar growth observed <br />                                                                        marker germination (10 &#226;&#8364;&#8220;12 h) incubated 0.01% <br />                                                                        MMS 0.31 mg/ml 4-nitroquinoline-N-oxide (4-NQ)                                                                        treatment 6 h. Cells fixed ethanol DAPI <br />                                                                        stained counting mitoses fixed glutaraldehyde                                                                        formaldehyde described photography. <br />  <br />  <br />  <br />                                                                        Construction brc1::ura4 Strain <br />                                                                        construction knockout plasmid, 7.5-kb fragment                                                                        subcloned pBluescript KS1 HindIII site                                                                        ablated digestion, end filling, ligation.                                                                         1.4-kb HindIII fragment, spans <span id='am-22' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> 48 amino <br />                                                                        acid 490 brc1 coding open reading frame excised                                                                        replaced 1.8-kb ura4 fragment. diploid strain  <br />                                                                        structed genotype: ura4-D18/ura4-D18 leu1&#226;&#8364;&#8220;32/leu1&#226;&#8364;&#8220;32 ade6- <br />                                                                        M210/ade6-M216 h1/h2. brc1 deletion plasmid used                                                                        transform strain uracil prototrophy. Transformants                                                                        propagated continuous log-phase 4 d allow integration                                                                        replated selective conditions. Colonies screened                                                                        Southern blotting replacement event. appropriate <br />                                                                        strains derived, chosen analysis.                                                                        strain induced sporulate plating MEA medium                                                                        2 d. Tetrads 20 azygotic asci dissected, cases                                                                        spores viable gave 2:2 segregation ura1 <br />                                                                        phenotype. brc1::ura4 strains crossed wild type                                                                        exclude possibility suppressor mutations. <br />  <br />  <br />  <br />                                                                        Phosphorylation chk1p <br />                                                                        strains background chk1::HA (Walworth                                                                        Bernards, 1996). Cultures irradiated UV-C ionizing <br />                                                                        radiation described  samples harvested                                                                        frozen. frozen cells disrupted using bead beater                                                                        glass beads extracted lysis buffer (50 mM Tris, pH 7.5, 150 <br />                                                                        mM NaCl, 50 mM NaF, 2 mM EDTA, 1 mM DTT, 80 mM b-glyc- <br />                                                                        erophosphate, 0.1% NP40, 0.1 mM Na-orthovanadate, 1 mM PMSF, <br />                                                                        plus protease inhibitors). extract cleared centrifugation <br />                                                                        13,000 3 g 15 min, supernatant boiled SDS <br />                                                                        sample buffer. Samples analyzed SDS-PAGE Western <br />                                                                        blotting using 12CA5 antibody. band intensity quanti- <br />                                                                        fied Molecular Dynamics (Sunnyvale, CA) densitometer using <br />                                                                        ImageQuant software. <br />  <br />  <br />  <br />  <br />                                                                        Figure 1 (cont). fluorescence microscopy. (F) Wild-type cells <br /> Figure 1. NA74 defective response DNA damage.           (closed symbols) NA74 (open symbols) synchronized UV-C survival curve wild type (M) NA74 (F). (B) Ionizing       lactose gradient centrifugation irradiated 100 Gy (F <br /> radiation survival curve wild type (M) NA74 (F). (C) DAPI-     E) 500 Gy (&#197;&#8217; &#226;&#8364;&#353;) mock irradiated (f M). Cultures <br /> stained cells NA74 synchronized cdc25-22         incubated 30&#194;&#176;C, ethanol-fixed samples taken block release irradiated 50 J/m2 UV-C. Bar, 10 mm. (D)    DAPI staining. percentages cells passed mitosis <br /> DAPI-stained cells asynchronous NA74 irradiated 450 Gy        assessed sample fluorescence microscopy. (G) <br /> ionizing radiation. Bar, 10 mm. (E) Wild-type cells (f M)      cdc25-22 (f M), cdc25-22 NA74 (F E), cdc25-22 rad3-136 <br /> NA74 (F E) synchronized cdc25-22 block release         (&#197;&#8217; &#226;&#8364;&#353;) grown 25&#194;&#176;C, plated YES medium, irradi- irradiated 50 J/m2 UV-C (closed symbols) mock         ated indicated dose UV-C. Plates placed irradiated (open symbols). Cultures incubated 25&#194;&#176;C,        25&#194;&#176;C (open symbols) 36&#194;&#176;C 4 h shifted 25&#194;&#176;C <br /> ethanol-fixed samples taken DAPI staining. percent-       (closed symbols). Surviving colonies, expressed percentage ages cells passed mitosis assessed sample   unirradiated controls, counted 4 d. <br />  <br />  <br /> 2908                                                                                                          Molecular Biology Cell <br />                                                                                                                         Fission Yeast rad18 <br />  <br />  <br />                                                                       RESULTS <br />                                                                       NA74 Mutation Causes Altered Response                                                                       DNA Damage G2 <br />                                                                       NA74 mutation defines gene recently identi- <br />                                                                       fied screen novel G2 checkpoint genes fission <br />                                                                       yeast (Verkade O&#226;&#8364;&#8482;Connell, 1998). strain MMS <br />                                                                       sensitive enters lethal mitoses presence                                                                       DNA-damaging agent. characterize mutation <br />                                                                       performed phenotypic analysis effect                                                                       response DNA damage. Survival curves UV-C <br />                                                                       ionizing radiation showed NA74 hypersensitive                                                                       DNA damaging agents (Figure 1, B). DAPI <br />                                                                       staining irradiated cells confirmed pro- <br />                                                                       gressing aberrantly mitosis. cells, DNA <br />                                                                       stretched division plane, cytokinesis                                                                       progressed absence completed chromosome segre- <br />                                                                       gation (Figure 1, C D). <br />                                                                          defective mitoses DNA damage consistent <br />                                                                       failure G2 DNA damage checkpoint,  <br />                                                                       sessed cell cycle progression kinetics synchronous cul- <br />                                                                       tures irradiation. wild-type controls, normal check- <br />                                                                       point-induced delay seen doses irradiation                                                                       minimal effects cell survival. response NA74 <br />                                                                       cells   complex. UV-C-irradiated cells, <br />                                                                       ;30% cells culture failed arrest, entering <br />                                                                       mitosis kinetics similar unirradiated controls.                                                                       remainder progressed mitosis kinetics similar <br />                                                                       wild-type cells (Figure 1E). Consequently, cultures                                                                       NA74 cells showed doubling cell number time <br />                                                                       course experiment.  unlike wild-type cells, <br />                                                                       mitoses irradiated NA74 cells highly aber- <br />                                                                       rant, failed chromosome segregation (Figure 1C), sug- <br />                                                                       gesting fully resolved lesions caused UV-C <br />                                                                       irradiation reinitiating mitosis. response                                                                       lack synchrony  assessed unirradiated <br />                                                                       controls) incomplete arrest cdc25-22  Figure 2C). <br />                                                                       data consistent checkpoint defect <br />                                                                       delay maintained sufficient time                                                                       ensure mitotic fidelity. g-irradiated cells, NA74 <br />                                                                       strain showed dose-dependent delay mitosis identical                                                                       wild-type strain (Figure 1F). despite fact <br />                                                                       NA74 cells irradiated 500 Gy showed  <br />                                                                       siderable reduction viability (2%) compared wild- <br />                                                                       type controls (48%), failed faithfully complete mitosis <br />                                                                       (Figure 1D). physiological role check- <br />                                                                       point prevent mitosis presence DNA damage, <br />                                                                       data consistent defective cell cycle <br />                                                                       response DNA damage. <br />  <br />  <br />  <br />                                                                       Figure 2 (cont). indicated times DAPI staining. percent- <br /> Figure 2. Rad18 mutants defective DNA repair.            ages cells passed mitosis assessed indicated strains grown midlogarithmic phase         sample fluorescence microscopy. (C) indicated strains irradiated 450 Gy ionizing radiation. Samples taken   background cdc25-22 arrested late G2 incubation indicated times (hours) processed pulse field gel elec-   36&#194;&#176;C 3 h. Cells irradiated 100 J/m2 UV-C (closed <br /> trophoresis. Unirradiated controls (U) included used      symbols) mock irradiated (open symbols; time 5 0) standards positions chromosomes. (B) Wild-type   maintained 36&#194;&#176;C 2 h irradiation shifted 25&#194;&#176;C. <br /> cells (f M) rad18-X (F E) synchronized            Samples assayed DAPI staining, normal mitoses (f <br /> cdc25-22 block release irradiated 50 J/m2 UV-C      M) aberrant mitoses (F E) percentage total cells <br /> (closed symbols) mock irradiated (open symbols). Cultures     counted samples 100 &#226;&#8364;&#8220;150 cells time point. <br /> incubated 25&#194;&#176;C, ethanol-fixed samples taken        Data represent mean samples; SE # 5%. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                2909 <br />  H.M. Verkade et al. <br />  <br />  <br />    defective G2 DNA damage checkpoint sole           defects rad18-X (Lehmann et al., 1995) (Figure 2C cause radiation sensitivity NA74, imposed        unpublished results). rad18 strains prevented <br /> cell cycle delay irradiation partially       entering mitosis arrest relieved. rescue radiation-induced cell death,            data indicate rad18 mutants require p34cdc2 activa- <br /> scribed checkpoint mutants (Al-Khodairy              tion entry aberrant mitoses, nuclear <br /> Carr, 1992).  rescue observed             abnormalities result passage mitosis NA74 delayed G2 UV-C irradiation imposi-           interphase event.  following mitoses tion cdc25-22 arrest (Figure 1G), rescue          rad18-X using protocol highly aberrant clearly evident checkpoint-defective rad3-136 control.     seen irradiated rad18-74 cells po-  checkpoint defects account ra-         stirradiation arrest (Figures 1E 2C). diation sensitivity NA74, consistent       reported enhancement rad18-X defects 36&#194;&#176;C (Leh- additional defect DNA repair.                                mann et al., 1995) suggests failure maintain                                                                    checkpoint arrest general feature rad18 alleles. NA74 Allele rad18                                         rad18 allele defective DNA replication checkpoint aimed clone NA74 gene complementation          assessed hydroxyurea-induced cell cycle arrest im- <br /> MMS sensitivity isolated seven complementing plas-             position temperature-sensitive S-phase mutants.  <br /> mids, fell classes. set, defined        like checkpoint-proficient mis4-242 strain, rad18-74 <br /> plasmids, included rad18 gene. NA74 subsequently           rad18-X hydroxyurea sensitive  unpub- <br /> shown allele rad18 linkage analysis          lished results). <br /> hitherto referred rad18-74. Sequencing rad18-74              rad18 essential gene, rad18-74 showed single base substitution (G3  leading             rad18-X alleles completely null rad18p function. <br /> amino acid substitution (A151T), located 21 resi-         determine checkpoint status rad18D cells,  <br /> dues C-terminal putative ATP-binding domain.            sessed ability wild-type spores spores deleted remaining plasmids shown restriction              rad18 (rad18::ura4) maintain checkpoint arrest DNA <br /> hybridization analyses possess overlapping inserts          damage induced MMS UV-mimetic 4-NQ. represented high-copy suppressor completely rescued         germination, radiomimetic drugs added cul- radiation sensitivity rad18-74. suppressor, brc1,   tures, followed mitotic figures 6 h. discussed                                                    continuous exposure drugs circumvents asyn- <br />    Rad18p previously shown required             chrony spore germination precludes spore germina- <br /> repair UV-C-induced lesions (Lehmann et al., 1995).          tion experiments using UV-C g-irradiation. pres- <br /> cause rad18-74 previously isolated allele,            ence drugs, wild-type spores maintained G2 <br /> rad18-X, sensitive MMS ionizing radiation,         arrest, assessed fluorescence-activated cell-sorting assayed ability repair double-stranded DNA             analysis duration experiment (Figure 3 breaks using pulse field gel electrophoresis (Figure 2A).          unpublished results). Conversely, rad18::ura4 spores germi- <br /> Wild-type cells efficiently rejoined chromosomal frag-         nated entered defective mitoses presence ments 4 h irradiation 450 Gy, little         drugs. 6 h, aberrant mitoses represented ;30% repair evident rad18 strain 10 h       germinated spores, normal mitoses ob- <br /> irradiation.  rad18p required repair double-        served (Figure 3). 3 h, ;20% aberrant mitoses <br /> stranded DNA breaks, rad18-74 enters mitosis         characterized stretching DNA dose ionizing radiation completing repair.         division plane, remainder resembling classical data indicate g-irradiated rad18-74 cells      &#226;&#8364;&#339;cut&#226;&#8364;&#65533; phenotype. 6 h, cytokinesis nuclear mate- <br /> showed checkpoint kinetics similar wild-type cells,                                                                    rial occurred mitotic cells. absence MMS <br /> cycle G2 mitosis absence successfully <br />                                                                    4-NQ, ;10% rad18::ura4 spores displayed cut completing DNA repair, explains appearance                                                                    &#226;&#8364;&#339;stretched&#226;&#8364;&#65533; phenotype, accounting 75% mitotic fig- <br /> mitotic defects cells. Conversely, rad18-X largely <br /> checkpoint proficient response ionizing radiation (Al-       ures. data indicate rad18p required success- <br /> Khodairy Carr, 1992). rad18 strain          ful passage mitosis absence DNA-dam- <br /> capable repairing DNA damage induced ionizing ra-            aging agents. Germination rad18::ura4 spores 11 mM <br /> diation, difference cell cycle responses dramatic.    hydroxyurea caused prolonged cell cycle arrest  unpub- <br /> assayed UV-C checkpoint rad18-X cdc25-22 block         lished results), indicating replication checkpoint (36&#194;&#176;C) release (25&#194;&#176;C) synchronization           intact, cells capable undergoing interphase <br /> unlike rad18-74, small proportion rad18-X cells          arrest. compared kinetics aberrant mitoses entered mitosis, subsequently little increase     cultures chk1::ura4 strain, completely <br /> cell number (Figure 2B). strains defective nucleo-         lacks G2 DNA damage control (Walworth et al., 1993). tide excision repair (rad13  rad15-P, rad16-U [Murray et      cells accumulated similar levels aberrant mitoses caused <br /> al., 1992; Carr et al., 1993, 1994]) irradiated UV-C,     checkpoint failure presence MMS 4-NQ cells elongate fail enter mitosis, demonstrating    compared rad18::ura4, cut <br /> effect wild-type checkpoint responding unrepairable        phenotype. appearance aberrant mitoses DNA damage (Al-Khodairy Carr, 1992). Like rad18-74,            delayed compared rad18::ura4 strain. UV-C sensitivity rad18-X rescued               earlier appearance rad18::ura4 inherent <br /> cdc25-22 G2 arrest imposed 2 h irradiation syn-       requirement rad18p mitosis partially <br /> chronized G2 cells, consistent DNA repair        affected different germination kinetics strains. <br />  <br />  <br /> 2910                                                                                                  Molecular Biology Cell <br />                                                                                                                           Fission Yeast rad18 <br />  <br />  <br />  <br />  <br />                                                                          Figure 4. Rad18p required phosphorylation chk1p <br />                                                                          DNA damage. indicated strains, background                                                                          hemagglutinin-tagged chk1p, irradiated 100 J/m2, sam- <br />                                                                          ples taken indicated times (minutes). Western blot <br />                                                                          analysis using 12CA5 antibody used detect forms                                                                          hemagglutinin-tagged chk1p. Time 0 5 unirradiated. Essentially <br />                                                                          identical results obtained cultures irradiated 250 Gy <br />                                                                          ionizing radiation. <br />  <br />  <br />  <br />                                                                           data indicate rad18p essential                                                                          prevent entry mitosis presence DNA damage. <br />  <br />                                                                          Rad18-X rad18-74 Normal DNA <br />                                                                          Damage-induced chk1p Phosphorylation <br />                                                                          chk1p protein kinase final element signaling <br />                                                                          cascade activated G2 DNA damage checkpoint. DNA <br />                                                                          damage induces phosphorylation-dependent mobility <br />                                                                          shift chk1p, rad checkpoint gene-dependent manner <br />                                                                          (Walworth Bernards, 1996). phosphorylation pro- <br />                                                                          motes binding 14-3-3 proteins used surrogate <br />                                                                          marker checkpoint activation (Chen et al., 1999). used <br />                                                                          marker investigate defective DNA dam- <br />                                                                          age checkpoint responses rad18 mutant strains                                                                          failed checkpoint activation checkpoint maintenance.                                                                          assayed ability rad18-X rad18-74 cultures pro- <br />                                                                          mote chk1p phosphorylation response ionizing                                                                          UV-C irradiation. strains showed wild-type chk1p <br />                                                                          phosphorylation kinetics response DNA damage, <br />                                                                          phosphorylated forms appearing 15 min,                                                                          maintained, albeit slightly lower levels rad18 <br />                                                                          mutants, duration time course (Figure 4). <br />                                                                          Phosphorylated chk1p continued maintained                                                                          wild-type cells recovered checkpoint arrest <br />                                                                          entered mitosis, suggesting chk1p phosphorylation <br />                                                                          marker checkpoint release. data consis- <br />                                                                          tent model rad18p required checkpoint <br />                                                                          maintenance initiation. possible                                                                          assay chk1p phosphorylation rad18::ura4 background, <br /> Figure 3. Rad18p essential cell cycle arrest DNA dam- <br /> age.   Spores rad181/rad18::ura4 germinated appearance polar growth (10 h) 30&#194;&#176;C medium lacking uracil select rad18::ura4 cells, rad181 spores control. MMS   Figure 3 (cont.) (rad181/rad18::ura4), chk1::ura4 (chk11/ 4-NQ added, cultures continued incubation          chk1::ura4) strains left untreated (M), MMS (l) 3 h. Samples taken, fixed, stained DAPI. Note      4-NQ (F) added time 0. Samples taken hourly, presence cells septa bisecting rad18::ura4 nuclei     fixed, stained DAPI, mean percent cells presence 4-NQ MMS. Bar, 10 mm. (B) Kinetics               aberrant mitoses  cut DNA stretched division <br /> aberrant mitoses spore germination experiments. Spores           plane; text) recorded samples 100 &#226;&#8364;&#8220;150 cells. <br /> germinated wild-type (ura41/ura4-D18), rad18::ura4          SE # 5%. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                 2911 <br />  H.M. Verkade et al. <br />  <br />  <br />  <br />  <br /> Figure 5. Rad18p nuclear protein essential ATP-binding <br /> domain.   Rad18p localizes chromatin compartment nucleus. Cells expressing myc-tagged rad18p stained 9E10 <br /> anti-myc antibody, visualized Cy3, counterstained DAPI. (B) rad18-DN dominant negative overexpressed. Wild- <br /> type colonies grown presence (1T) absence (2T) thiamine 3 d 30&#194;&#176;C. Plasmids: (1) pREP1 vector; (2) pREP1 <br /> wild-type rad18; (3) pREP1 rad18-DN. (C) rad18-DN comple- <br /> ment rad18-X. rad18-X cells grown presence (1MMS) absence (2MMS) 0.0075% MMS thiamine-containing media <br />  represses lethal expression) 3 d 30&#194;&#176;C. Plasmids: (1) pREP1 vector; (2) pREP1 wild-type rad18; (3) pREP1 rad18-DN. <br /> rad18-DN complement rad18-74  unpublished results).             Figure 6. Isolation allele-specific high-copy suppressor                                                                            rad18-74.   Partial restriction map rad18-74-suppressing plas- <br />                                                                            mid. isolates contained open reading frames 19G10.6                                                                            19G10.7 (indicated solid bars; introns 19G10.7 [brc1] indi- presence equal number ungerminated                    cated hatched bars). P, PmeI; H, HindIII. Rescue rad18-74 (rad181) spores makes data impossible interpret.                    assessed UV-C irradiation survival curves. (B) rad18-74 spores G1, time cells competent                rad18-X transformed vector (M), C19G10.7 (brc1) chk1p phosphorylation (Martinho et al., 1998),              (&#226;&#8364;&#353;), rad18 (F). Transformants grown selective media, observed absence phoshorylation attributed                  UV-C survival curves determined percent survival com- <br /> cell cycle status defect signaling chk1p.              pared unirradiated controls. <br />  <br />  <br />  <br /> Rad18p Nuclear Protein Essential                              activity SMC2/SMC4 heterodimer (Kimura ATP-binding Domain                                                         Hirano, 1997). structural similarity rad18p SMC <br /> proteins suggests chromatin organization role rad18p.                Allele-specific High-Copy Suppressor rad18 <br /> Immunostaining cells expressing myc-tagged version                 Encodes BRCT Protein <br /> rad18p showed localization pattern identical DAPI                     complementation rad18-74 genomic DNA li- <br /> staining, indicating  like SMC proteins, rad18p              brary, plasmids carrying high-copy suppressor chromatin compartment nucleus (Figure                  rad18-74 isolated addition rad18-containing <br /> 5A) (Strunnikov et al., 1993, 1995; Hirano Mitchison,                  plasmids. Sequence analysis high-copy suppressor <br /> 1994; Saka et al., 1994b; Saitoh et al., 1995). ATPase             plasmids showed corresponded locus chromo- <br /> activity demonstrated SMC family members,                     covered cosmid C19G10 (GenBank accession num- mutated ATP-binding domain rad18, construct                   ber Z69909). open reading frames present allele rad18-DN. Overexpression rad18-DN wild-                   suppressing plasmids, deletion mapping type cells lethal, indicating allele domi-             plasmids showed suppressing activity resided  nant negative effect essential rad18 function cells                dependent  open reading frame designated <br /> resembling rad18D accumulation mitotic defects                  C19G10.7 (Figure 6A). C19G10.7 encodes protein (Figure 5B unpublished results) (Lehmann et al.,                   BRCT domains, termed gene brc1. <br /> 1995). Lower-level expression wild-type rad18 rescued               suppression rad18-74 brc1 allele specific, MMS sensitivity rad18-X rad18-74, expression                 suppressive activity radiation sensitivity rad18-DN complement rad18-X rad18-74                   rad18-X (Figure 6B). suppression radiation sensi- <br /> (Figure 5C).  data indicate ATP-                   tivity rad18-74 reflected suppression binding domain rad18 essential function, indicating              mitotic defects seen checkpoint failure. effects essential role ATP hydrolysis, described                unlikely perturbation cell cycle distribution, <br />  <br />  <br /> 2912                                                                                                             Molecular Biology Cell <br />                                                                                                                  Fission Yeast rad18 <br />  <br />  high-level overexpression brc1           logue Aspergillus NIMA kinase, implicated nmt1 promoter effect cell cycle progression          control chromatin condensation, activation wild-type rad18 mutant cells  unpublished results).        kinases prematurely induce chromatin  fission yeast BRCT proteins, encoded cut5        densation independently cell cycle progression <br /> rhp9, effect radiation sensitivity rad18-74      (O&#226;&#8364;&#8482;Connell et al., 1994). fin1 deleted, cells expressed multicopy plasmid.  BRCT          similar phenotypes G2 delay mitotic defects. domains insufficient suppress rad18-74.              fact, double mutants fin1D top2-191 syn- <br />                                                                   thetic lethality greatly increased frequency brc1 Required Mitotic Fidelity                             mitotic abnormalities (Krien et al., 1998). suggests                                                                   cooperate chromatin condensation. determine determine function brc1p, deleted brc1           rad18p plays role chromatin condensation, placed ura4 marker homologous recombina-           examined genetic interactions tion diploid cells (Figure 7A). Dissection tetrads          genes involved chromatin organization. data showed 2:2 segregation ura41:ura42, indicating brc1       presented Table 1. essential gene. Given allele-specific interaction         rad18-74 rad18-X synthetically rad18, tested brc1::ura4 cells radiation      lethal top2-191, failing form colonies semiper- <br /> sensitive. Radiation survival curves showed brc1::ura4 cells      missive temperature 30&#194;&#176;C (Figure 8A).  sensitive UV-C irradiation, mildly      ditions top2-191 cells delay G2 mitotic defects sensitive ionizing radiation (Figure 7, B C), did      ;10% cells able form colonies (Krien et al., observe defects DNA damage replication check-            1998). DAPI staining double mutant strains grown point kinetics  unpublished results).           permissive temperature 25&#194;&#176;C shifted 30&#194;&#176;C phenotypes brc1::ura4 cells indicate brc1p plays role   4 h showed dramatic increase frequency aberrant fidelity mitotic chromosome segregation.  <br />                                                                   mitoses. characterized failure com- <br /> assays chromosome stability showed brc1::ura4 cells <br />                                                                   pletely segregate chromosomes cytokinesis (Figure 8, <br /> lost artificial chromosome frequency 200-fold <br />                                                                   C  Table 1). Inactivation topoisomerase II drugs <br /> greater wild-type cell (Figure 7D). Chromosome  <br />                                                                   mitoxantrone leads G2 arrest double- <br /> stability seen rad18-74 (16-fold)                                                                   stranded DNA breaks, drugs highly <br /> described rad18-X (74-fold) (Lehmann et al., 1995).                                                                   toxic strains defective double-stranded break repair described rad18 epistasis group <br />                                                                   budding yeast (Hartwell et al., 1997). rad18-X;top2-191 <br /> members rad2 rhp51 (Murray et al., 1994; Muris et al., <br />                                                                   rad18-74;top2-191 double mutants proceed aberrant <br /> 1996). Second, brc1::ura4 cells showed consistent                                                                   mitosis, rad18p required maintain G2 delay caused <br /> variable delay entry mitosis, dividing 19 6 3.8 mm, <br /> compared 13.8 6 0.4 mm wild-type controls (Figure        reduction top2p activity semipermissive <br /> 7E). Fluorescence-activated cell-sorting analysis combined        temperature. inability repair double-stranded breaks mitotic indices showed delay confined       caused top2p inactivation account G2 period cell cycle. Furthermore, ;6% cells        mitotic phenotypes, repair defect showed variety chromosome segregation defects result-        rad18-X rad18-74 sole cause syn- <br /> ing classical cut phenotypes, nuclear fragmentation, hy-       thetic lethality. Alternatively, synthetic phenotypes <br /> percondensation, mis-segregation, stretching DNA           suggest top2p rad18p cooperate chromatin division axis (Figure 7, F&#226;&#8364;&#8220;J). phenotypes         organization. attempted suppress rad18 mutants <br /> resemble observed rad18::ura4 spores (Figure 3A).        overexpressing top2 nmt1 promotor ob-  data consistent defective organiza-      served effect radiation sensitivity mitotic fidelity. <br /> tion chromatin absence brc1p function. Finally,      Overexpression top2 effect wild-type cells  nonparental ditypes recovered crosses          unpublished results). brc1::ura4 synthetically lethal <br /> brc1::ura4 rad18-74 rad18-X. case,     top2-191 (Figure 8B), increased defects chromo- <br /> presumptive double mutants divided times,            segregation (Figure 8, C&#226;&#8364;&#8220;J, Table 1). <br /> arresting microcolonies elongated         Double mutants constructed rad18-X, rad18-74 <br /> lysed cells, described strains deleted rad18   fin1D showed higher levels mitotic abnormalities <br /> (Lehmann et al., 1995). data indicated mutations       (7- 10-fold), suggesting cooperation function. brc1 rad18 result synthetic lethality,   characterize  investigated functional links brc1 essential rad18 function impaired.                condensin SMC proteins encoded cut3 cut14.                                                                   synthetic phenotypes seen double mutants  <br />                                                                   structed conditional alleles, cut3-477 Interactions Genes Involved Higher-                    cut14-208. specific alleles, Order Chromosome Structure                                        intermediate phenotypes temperatures ranging defects chromosome segregation G2 delay            25 34&#194;&#176;C  unpublished results), seen brc1 rad18 mutants reminiscent pheno-         rule functional links time. functional <br /> types described strains defective chromatin organiza-       interactions top2 fin1 suggest possible role tion. temperature-sensitive allele type II topoisomer-     rad18p chromatin condensation. Double mutants rad18 <br /> ase, top2-191, displays similar phenotypes shifted        alleles rad21-45 strain, analogy homol- <br /> semipermissive temperature (Krien et al., 1998). Top2p ap-        ogous proteins systems likely defective pears play roles chromatin organization, includ-       sister chromatid cohesion (Guacci et al., 1997; Michaelis et al., <br /> ing chromatin condensation. Fin1p, fission yeast homo-        1997; Losada et al., 1998), showed synthetic interactions. <br />  <br />  <br /> Vol. 10, September 1999                                                                                                       2913 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                                 Similarly, genetic interactions observed mis4- <br />                                                                                 242, defective sister chromatid cohesion <br />                                                                                 (Furuya et al., 1998). Furthermore, double mutants                                                                                 rad18 alleles rad21-45 mis4-242 showed  <br />                                                                                 hanced radiation sensitivity compared parent, <br />                                                                                 indication separable function   <br />                                                                                 published results).  genetic interactions                                                                                 observed consistent role rad18p chromatin <br />                                                                                 organization, independent sister chromatid  <br />                                                                                 hesion. <br />  <br />  <br />                                                                                 DISCUSSION <br />                                                                                 Cells respond DNA damage G2 delaying mitotic <br />                                                                                 onset maintenance p34cdc2 Y15 phosphorylation <br />                                                                                 (O&#226;&#8364;&#8482;Connell et al., 1997; Rhind et al., 1997). Studies                                                                                 molecular events controlling checkpoint response                                                                                 beginning unravel biochemical interactions dif- <br />                                                                                 ferent checkpoint proteins checkpoint proteins <br />                                                                                 core cell cycle regulators (Walworth Bernards, 1996; <br />                                                                                 Furnari et al., 1997; O&#226;&#8364;&#8482;Connell et al., 1997; Peng et al., 1997; <br />                                                                                 Rhind et al., 1997; Saka et al., 1997; Sanchez et al., 1997; <br />                                                                                 Kostrub et al., 1998; Kumagai et al., 1998; Chen et al., 1999; <br />                                                                                 Furnari et al., 1999; Kumagai Dunphy, 1999; Lopez- <br />                                                                                 Girona et al., 1999). Classically, DNA damage checkpoint <br />                                                                                 genes defined mutations abolish                                                                                 checkpoint response phenotypic consequence                                                                                 absence DNA damage.  cases, role <br />                                                                                 checkpoint control repertoire func- <br />                                                                                 tion essential gene, case cut5p (Saka                                                                                 Yanagida, 1993; Saka et al., 1994a; Verkade O&#226;&#8364;&#8482;Connell, <br />                                                                                 1998). Extensive genetic screens needed identify <br />                                                                                 specific checkpoint-defective alleles essential genes                                                                                 fully define molecules involved response DNA <br />                                                                                 damage. <br />                                                                                    rad18 example essential gene required <br />                                                                                 checkpoint control. precise essential func- <br />                                                                                 tion rad18p known, structural similarity                                                                                 SMC family proteins, localization chromatin,                                                                                 phenotypes rad18 mutants support role chromatin <br />                                                                                 organization. Deletion rad18 lethal associated <br />                                                                                 mitotic failure. Analysis point mutation rad18, <br />                                                                                 rad18-X, previously shown rad18p required                                                                                 repair UV-induced lesions (Lehmann et al., 1995),                                                                                 showed rad18p essential repair                                                                                 double-stranded DNA breaks. Furthermore,                                                                                 shown rad18p required maintain checkpoint <br />                                                                                 arrest DNA damage, failed checkpoint <br /> Figure 7. Mitotic defects chromosome instability strain deleted                                                                                 response leads highly aberrant mitoses subsequent <br /> brc1.   Strategy creating null allele brc1. MATERIALS        cell death. described isolation analysis <br /> METHODS details. (B) UV-C survival curve wild-type (M)              checkpoint-defective allele rad18, rad18-74.  <br /> brc1::ura4 (F). (C) Ionizing radiation survival curve wild-type (M)     sponse UV-C ionizing radiation, strains carrying <br /> brc1::ura4 (F). (D) Rates loss artificial chromosome 10 generation.   rad18-74 allele failed maintain checkpoint arrest. <br /> Data expressed percentage total colonies corrected       rad18-X cells showed defective checkpoint <br /> loss generation zero. Values wild-type, 0.005; rad18-74, 0.080;          responses held 36&#194;&#176;C irradiation. Importantly, <br /> brc1::ura4, 0.997; rad18-X, 0.368. (E) DAPI phase-contrast image         cells deleted rad18 checkpoint deficiencies compara- <br /> exponentially growing brc1::ura4 cells. Note variability cell length     ble chk1 deleted strain. <br /> mis-segregated chromosomes  left). Bar, 20 mm. (F&#226;&#8364;&#8220;J) Examples mitotic abnormalities brc1::ura4. Bars, 10 mm. (F) Nuclear fragmentation <br />                                                                                    response DNA damage considered mis-segregation. (G) Stretching chromosomes division axis          multistage process involving damage detection, initiation bisection septum. (H) Bisection nucleus septum hyper-          cell cycle arrest DNA repair pathways, maintenance condensation.   Hypercondensation mis-segregation. (J) Fragmenta-         arrest repair, subsequent recovery cell cycle <br /> tion   incomplete segregation chromosomes  left).            reentry. assess checkpoint defect lies rad18 <br />  <br />  <br /> 2914                                                                                                               Molecular Biology Cell <br />                                                                                                                           Fission Yeast rad18 <br />  <br />  <br />  <br /> Table 1. Genetic interactions rad18, brc1, top2, fin1, rad21, mis4 <br />  <br />                     rad18-74          rad18-X               brc1D                fin1D          top2-191         rad21-45          mis4-242 <br />  <br /> Wild type              2                6                    6                      4             11                 7                11 <br /> rad18-74               &#226;&#8364;&#8220;              Allelic          Synthetic lethal            27             52                12                 6 <br /> rad18-X                                 &#226;&#8364;&#8220;              Synthetic lethal            41             52                10                12 <br /> brc1D                                                        &#226;&#8364;&#8220;                      8             40                 6                16 <br />  <br /> Numbers represent percent aberrant mitoses double mutants. strains containing top2-191 mis4-242 grown midlogarithmic <br /> phase 30&#194;&#176;C fixation, DAPI staining, observation fluorescence microscopy. top2-191- mis4-242-containing strains grown midlogarithmic phase 25&#194;&#176;C shifted 30&#194;&#176;C 4 h fixation, DAPI staining, observation fluorescence <br /> microscopy. Data represents mean samples 100 cells. <br />  <br />  <br /> mutants, assayed phosphorylation chk1p protein                  brc1p multicopy plasmid. suppression allele <br /> kinase marker checkpoint activation (Walworth                 specific rad18-74 general feature  <br /> Bernards, 1996). time course experiments, rad18-74                creased expression BRCT proteins. BRCT domains rad18-X showed wild-type kinetics chk1p phosphor-                  homology C-terminus breast ovarian cancer <br /> ylation. observation, kinetics mi-              susceptibility gene BRCA1 (itself G2 checkpoint protein; <br /> totic entry irradiated synchronous cultures, consistent             Xu et al., 1999) variety proteins  model rad18p essential mainte-                   volved DNA repair, DNA replication, checkpoint <br /> nance checkpoint arrest, initiation.         control (Callebaut Mornon, 1997). instances contrast rad3p protein, required initiate           shown act protein interaction domains <br /> checkpoint arrest maintain (Martinho et al.,                (Saka et al., 1997; Grawunder et al., 1998; Taylor et al., 1998). <br /> 1998). analyses DNA damage check-                     possible interpretation suppression rad18-74 point genetically separable initiation mainte-                rad18p brc1p physically interact  <br /> nance events, required survival                  teraction defective rad18-74 mutant. attempted <br /> DNA damage. chk1p phosphorylation persists                     coimmunoprecipitate epitope-tagged derivatives wild-type cells reenter cell cycle, rad18              rad18p brc1p expressed wild-type irradiated cells <br /> mutant cells maintain chk1p phosphorylation despite                  saw evidence physical association despite efficient <br /> defective checkpoint maintenance, chk1p itself                immunoprecipitation protein. Furthermore, saw <br /> specific checkpoint initiation.                                       evidence interaction yeast hybrid analysis    addition role checkpoint maintenance, rad18p              vitro&#226;&#8364;&#8220;transcribed translated proteins  required DNA repair. defects                 unpublished results). likely brc1 acts <br /> unrelated present model explaining response                  downstream rad18 bypass suppressor rad18-74, rad18 mutants DNA damage. similarity                   suppression represents subset rad18p <br /> SMC family DNA-binding chromatin organization                      functions. Deletion brc1 results mitotic defects proteins, rad18p involved establishing main-                 chromosome instability, phenotypes taining chromosomal structures necessary DNA                 primary defect. phenotypes seen repair. presence structures chromosomes               rad18 mutants. Furthermore, deletion brc1 synthetically acts origin signal maintain checkpoint              lethal rad18-X rad18-74, indicating brc1p arrest. scenario, defective rad18p function              required viability rad18 function compromised. observed, result failure complete repair                cells   hypersensitive UV-C maintain arrest. structures presumably essential                ionizing radiation, brc1p essential DNA normal chromatin organization given essential na-                 repair checkpoint responses, seen rad18p.  <br /> ture rad18 mitotic defects occur absence.           presence rad18-X rad18-74, brc1 essential, <br />    phenomenon related checkpoint                  suggesting brc1 plays downstream role, able <br /> monitors completion S-phase particular role              compensate subfunctional rad18. <br /> DNA polymerase (pol1/swi7) response (D&#226;&#8364;&#8482;Urso et al.,                presence BRCT domains, likely 1995). pol1p present, functional, S-                 brc1p facilitates protein&#226;&#8364;&#8220;protein interactions, inter- <br /> phase progression stalled. results checkpoint acti-            acting BRCT domain protein. analogy vation prevents entry mitosis. pol1p                  cohesin condensin complexes, rad18p leted, cells fail recognize incomplete DNA synthesis               larger complexes (Jessberger et al., 1998). brc1p <br /> enter catastrophic mitoses.  presence pol1p                function adaptor molecule pro- stalled replication forks appears signal incomplete                teins, complex involved rad18p-dependent <br /> S-phase activates checkpoint. Similarly, checkpoint               process required mitosis. attempted localize brc1p <br /> pathways monitor presence proteins involved                 indirect immunofluorescence specific local- <br /> chromatin organization, influence timing                 ization cells despite detecting brc1p Western blots. entry mitosis serve alert cell                possible epitopes obscured vivo. positioned cell cycle.                                             attempted localize GFP-brc1p fusions    radiation sensitivity checkpoint defects                    localization fluorescence cells expressing fusion <br /> rad18-74 completely suppressed expression                  protein nmt1 promoter. Multiprotein complexes <br />  <br />  <br /> Vol. 10, September 1999                                                                                                                2915 <br />  H.M. Verkade et al. <br />  <br />  <br />                                                                            tion proteins mitotic fidelity. Mutations                                                                            rad18, brc1 fin1 synthetically lethal tempera- <br />                                                                            ture-sensitive topoisomerase II semipermissive tempera- <br />                                                                            ture. consistent models proposing multiple <br />                                                                            roles topoisomerase II chromatin organization (Heck, <br />                                                                            1997). rad18p play role top2p-dependent <br />                                                                            decatenation chromatin, defects explain <br />                                                                            rad18 phenotypes. Rad18-74 rad18-X mutants, <br />                                                                            unlike top2-191, minor mitotic defects                                                                            absence irradiation. Furthermore, mutants  <br />                                                                            fective maintaining checkpoint arrest irradiated <br />                                                                            late G2 (cdc25-22-arrested cells), time bulk                                                                            decatenation likely completed.  defective <br />                                                                            mitoses presumably result failed repair check- <br />                                                                            point maintenance defect topoisomerase II <br />                                                                            activity.  overexpression top2 rescue                                                                            radiation sensitivity mitotic abnormalities rad18-X <br />                                                                            rad18-74. Brc1-deleted cells delay entry mitosis.                                                                            combination G2 delay mitotic defects seen                                                                            strains carrying mutations rad18 (Lehmann et al., 1995)                                                                            fin1 (Krien et al., 1998) temperature-sensitive top2 <br />                                                                            mutants semipermissive temperature (Krien et al., 1998). <br />                                                                            delay consistent model defects                                                                            chromatin organization monitored mitosis                                                                            lead activation checkpoint influences timing <br />                                                                            mitotic onset. <br />                                                                               increasingly evident regulation                                                                            higher-order chromatin structure vital maintenance <br />                                                                            genomic integrity. SMC1&#226;&#8364;&#8220; 4 proteins central reg- <br />                                                                            ulating chromosome dynamics, functioning key com- <br />                                                                            ponents chromatin condensation sister chro- <br />                                                                            matid cohesion. phenotypes genetic interactions <br />                                                                            rad18-74, rad18-X, rad18::ura4 mutants provide <br />                                                                            link chromatin organization, DNA repair,                                                                            control mitotic entry. conserved structure, lo- <br />                                                                            calization, requirement chromosome stability sug- <br />                                                                            gest rad18p carries related functions SMC1&#226;&#8364;&#8220; 4. <br />                                                                            Genetic interactions rad18 top2 fin1 suggest                                                                            role chromatin condensation, extensive analyses                                                                            rad18p needed determine precise function.  <br />                                                                            fects DNA repair checkpoint maintenance                                                                            absence rad18p function consistent role <br /> Figure 8. Interactions top2, rad18 brc1.   top2-191         rad18p forming essential structures  <br /> synthetically lethal rad18-74 rad18-X. Strains plated        quired DNA repair recognized proteins YES incubated indicated temperatures 4 d <br /> (25&#194;&#176;C) 3 d (30 36&#194;&#176;C). Strains: (1) wild type; (2) rad18-74; (3)     signal checkpoint maintenance. recent years, ex- <br /> rad18-74 top2-191; (4) rad18-X top2-191; (5) rad18-X; 6, top2-191. (B)     tensive progress understanding check- <br /> top2-191 synthetically lethal brc1::ura4. Strains plated      point initiation, analysis rad18p  YES treated  Strains: (1) wild type; (2) brc1::ura4; (3)   cover arrest maintained ensure completion <br /> brc1::ura4 top2-191; (4) top2-191. (C&#226;&#8364;&#8220;J) DAPI phase-contrast im-       DNA repair progression mitosis. <br /> ages cells grown midlogarithmic phase 25&#194;&#176;C shifted 30&#194;&#176;C 4 h. Strains: (C) wild type; (D) rad18-X; (E) <br /> rad18-74; (F) top2-191; (G) brc1::ura4; (H) top2-191 rad18-X;   top2- <br /> 191 rad18-74; (J) top2-191 brc1::ura4. Bar, 10 mm.                          <br />                                                                            <br />                                                                             impede fluorescence GFP, protein                  <br /> diffusely localized cell.                                   <br />   investigated functional links brc1 genes                  <br /> implicated chromatin organization constructing dou-                   <br /> ble mutants combinations lead                     <br /> enhancement phenotype, indicative coopera-                  <br /> </body></html>