<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>cdc12p, Protein Required Cytokinesis Fission Yeast, Component Cell Division Ring Interacts Profilin <br /> Fred Chang,*&#226;&#8364;&#161; David Drubin,&#226;&#8364;&#161; Paul Nurse* <br /> *Imperial Cancer Research Fund, London WC2A 3PX, United Kingdom; &#226;&#8364;&#161;Department Molecular Cell Biology, <br /> University California, Berkeley, California 94720-3202 <br />  <br />  <br />  <br />  <br /> Abstract. eukaryotic cells, cell divi-                    (profilin homologue), cdc15 mutants exhibit sion fission yeast depends assembly actin                  specific cdc12p staining mitosis. cdc4 mutant <br /> ring circumscribes middle cell. Schizo-                    cells exhibit medial cortical cdc12p spot place saccharomyces pombe cdc12 essential gene neces-                      ring. mid1 mutant cells generally exhibit cdc12p spot <br /> sary actin ring assembly septum formation.                         single cdc12p strand extending random di- cdc12p member family                        rection. Based patterns, present model proteins including Drosophila diaphanous, Saccharo-                        ring assembly originates single point myces cerevisiae BNI1, S. pombe fus1,                     cortex molecular pathway volved cytokinesis actin-mediated processes.                   functions cytokinesis proteins suggested. Finally, <br /> Using indirect immunofluorescence,                            cdc12 cdc3 mutants synthetic- <br /> cdc12p located cell division ring                     lethal genetic interaction, proline-rich domain actin structures. overexpressed, cdc12p                      cdc12p binds directly profilin cdc3p vitro, suggest- <br /> located medial spot interphase anticipates                    ing function cdc12p ring assembly future ring site. cdc12p localization altered ac-                bind profilin. <br /> tin ring mutants. cdc8 (tropomyosin homologue), cdc3 <br />  <br />  <br />  <br />  <br /> C <br />          ell division eukaryotic cells accom-                   plasma membrane known           plished action actin-based                 putative signals responsible establishing position           tractile ring  reviews Conrad Schroe-                ring. <br /> der, 1990; Satterwhite Pollard, 1992; Fishkind Wang,                  fission yeast Schizosaccharomyces pombe ex- <br /> 1995). positioning ring, dictates sub-               cellent organism studying eukaryotic cell division sequent division plane, determine cell size, shape,                molecular-genetic level (Fankhauser Simanis, 1994; <br /> orientation. Placement division plane critical              Chang Nurse, 1996). Cell division fission yeast  development, instance, segregation lo-                quires actin-based ring, forms early mitosis calized determinants (Rhyu Knoblich, 1995; Horvitz                     shrinks late mitosis septum cell wall material Herskowitz, 1992). Events cell division                  grows centripetally cell surface (Marks Hyams, <br /> ordinated events nuclear division ensure                   1985). ring play roles cell division.  daughter cell receives intact genome. animal                   mark site cell wall septum formation,  <br /> cells, coordination cellular nuclear division events            second, play contractile role actively closing appears involve mechanism cleavage                       plasma membrane. ring clearly closes (Jo- <br /> plane determined mitotic asters  Rappaport,                  chova et al., 1991; McCollum et al., 1995; work), defin- <br /> 1986). proposed mitotic asters send                   itive evidence actin ring contractile exerts <br /> signals cell surface induce contractile ring               active, contractile force lacking. Alternatively, <br /> mation (Rappaport, 1986). number compo-                      actin ring stabilize guide plasma membrane <br /> nents contractile ring identified, little              septation. Conservation ring components indi- <br /> known molecular mechanism ring assembly                    cate S. pombe actin ring similar molec- <br />                                                                            ular level contractile ring larger eukaryotes. address correspondence Fred Chang, Department Micro- <br />                                                                               fission yeast, proposed actin ring <br /> biology, Columbia University, 701 West 168th Street, New York 10032.       subsequent division site positioned mid- <br /> Fax: (212) 305-1468. e-mail: fc99@columbia.edu                             dle cell signal premitotic nucleus <br />  <br />  <br />  <br /> &#239;&#163;&#169; Rockefeller University Press, 0021-9525/97/04/169/14 $2.00 Journal Cell Biology, Volume 137, Number 1, April 7, 1997 169&#226;&#8364;&#8220;182   169 <br />  (Chang Nurse, 1996). positions nucleus         Table  Schizosaccharomyces pombe Strains Used subsequent division site coincide, mutant          Study <br /> cells displaced nucleus, multiple nuclei, altered                               Genotype                         Reference <br /> cell polarity (Toda, T., personal communication; Marks et al.,                             2 <br /> 1987; Snell Nurse, 1994; Verde et al., 1995).          972          wild type h                                 Leopold, 1970 <br />                                                                   FC43         ura4-D18 h2                                 P. Nurse collection <br /> trast animal cells (Rappaport, 1986), fission yeast, <br />                                                                   FC20         cdc12-112 ura4 h2                           work <br /> ring assembly placement independent mi-            FC127        cdc12-229 h2                                Chang et al., 1996 <br /> totic spindle (Chang et al., 1996).  assembly         FC131        cdc8-346 h2                                 Chang et al., 1996 <br /> medial ring induced certain circumstances         FC55         cdc15-287 ura4-D18 h1                       Chang et al., 1996 <br /> terphase cells (Fankhauser et al., 1995; Ohkura et al., 1995),    FC361        cdc4-377 h2/h2 (diploid)                    Chang et al., 1996 <br /> suggesting signals responsible medial positioning        FC164        mid1-366 h2                                 Chang et al., 1996 present mitosis.                               FC291        ura4-D18 h2 pnmt-cdc12 (ura41)              work <br />    study molecular basis actin ring assembly        FC565        ura4-D18 h2 pnmt-cdc12 integrated           work <br /> placement, genetic screens used identify         FC315        ura4-D18 h2 pUR19 (ura41)                   work <br /> genes required actin ring formation (cdc3, cdc4, cdc8,        FC335        cdc121/cdc12::ura4                          work <br />                                                                                ura4-D18/ura4-D18 leu1-32/leu1-32 <br /> cdc12, cdc15, rng2) gene required ring <br />                                                                                ade6-M216/ade6-M210 h1/h2 <br /> placement (mid1) (Nurse et al., 1976; Marks et al., 1987;         PN1419       cdc25-22 leu1-32 h2                         P. Nurse collection <br /> Chang et al., 1996). Temperature-sensitive mutations genes cause defects cell division actin organi- <br /> zation restrictive temperature. mutants develop large, multinucleate cells ultimately die. Different     small actin-binding proteins shown bind <br /> aberrant patterns actin distribution indicate       monomeric actin, actin-related protein complex, phos- <br /> gene products play different roles actin ring assembly.        pholipid <span id='am-43' about='obo:IMR_0001350' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001348 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-44' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="PIP2" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="PIP2" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="PI(4,5)P2" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="IMR:0200477" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-58' property="obo:IAO_0000115" content="PI(4,5)P2 is cleaved by phospholipase C-beta or gamma to generate IP3 and diacylglycerol." datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001350" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C04637" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span>PIP2</span>, proline-rich peptides poly- <br /> cdc3 cdc8 mutants display disorganized actin struc-           <span id='am-84' about='obo:IMR_0200183' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00148" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0200183" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="L-Proline" datatype="xsd:string"></span>l-proline</span> VASP, regulate actin dynamics tures cell cycle, suggesting cdc3         formation F-actin structures  Machesky Pol- <br /> cdc8, encode profilin tropomyosin, respectively,        lard, 1993; Pollard, 1995; Machesky et al., 1994). organize actin general fashion (Marks et al., 1987;      studies begin address protein&#226;&#8364;&#8220;protein interactions <br /> Balasubramanian et al., 1992, 1994; Chang et al., 1996). cdc12    build actin ring. cdc15 mutants exhibit delocalized actin patches mitosis normal actin distributions interphase          Materials Methods <br /> (Fankhauser et al., 1995; Chang et al., 1996), suggesting gene products play specific roles ring assem-       Yeast Strains Genetic Methods <br /> bly mitosis. cdc4 rng2 mutants display disorga- <br />                                                                   Standard S. pombe genetic techniques media performed pre- <br /> nized actin cables instead ring, suggestive defect      viously described Moreno et al. (1991). S. pombe strains late step ring assembly (McCollum et al., 1995; Chang     isogenic 972 (Leupold, 1970). S. pombe strains used listed Table  <br /> et al., 1996). cdc4 encodes EF hand&#226;&#8364;&#8220;containing protein         FC20 ura4-D18 cdc12-112 h2 isolated cross cdc12-112 h2 properties myosin light chain (McCollum et al.,         h1 ura4-D18 (P. Nurse lab collection). cdc4-377 homozygous diploid <br />                                                                   strain isolated spontaneously FC123; diploid used  <br /> 1995). mid1 necessary placement ring,         cause higher percentage defined actin aster structures mid1 mutants exhibit rings septa located random posi-      diploid, structures present haploid strain tions angles cell surface (Chang et al., 1996).         Yeast transformations performed lithium acetate method <br /> cdc15p polo kinase homologue plo1p              (Moreno et al., 1991) electroporation (Kelly et al., 1993). <br /> implicated important regulators ring assembly cell cycle, overexpression sufficient   DNA Manipulations Hybridizations <br /> generate ring stage cell cycle (Fankhauser        Standard DNA ligation cloning procedures used (Maniatis et al., <br /> et al., 1995; Ohkura et al., 1995).                               1982) GeneClean Bio101 (Vista, CA) DNA prepared    cloning molecular character-          Qiagen plasmid kits (Chatsworth, CA). Genomic yeast DNA pre- <br />                                                                   pared described (Moreno et al., 1991). Enhanced chemiluminescence <br /> ization actin ring genes, cdc12. pheno-         (ECL)1 Southern hybridization carried recommended (ECL <br /> type cdc12ts mutants suggests cdc12 gene product       southern blotting; Amersham Corp., Arlington Heights, . colony hy- <br /> required specifically actin ring assembly mitosis      bridizations, 32P-labeled DNA probes prepared using Prime kit target cell cycle spatial regula-       (Stratagene, La Jolla, CA). <br /> tors ring assembly (Chang et al., 1996). cdc12p member family proteins implicated          cdc12 Cloning cytokinesis, suggesting function cdc12p         cdc12 gene cloned complementation temperature-sensi- <br /> conserved. cdc12p member protein fam-        tive lethal phenotype cdc12-112. pUR19 (ura41)-based S. pombe ge- <br /> ily localized cell division ring. Studies cdc12p    nomic library (Barbet et al., 1992) transformed ura4D18 cdc12- <br />                                                                   112 (FC112), 5 3 104 ura1 transformant colonies screened localization patterns overexpression mutant           replica printing YE5S phyloxin growth 368C. colo- <br /> backgrounds reveal medial cdc12p spot. localization        nies showed plasmid-dependent rescue. Isolation plasmid DNA pattern provide novel insights mechanisms             transformants identified related plasmids, pFC103 ring formation placement. Furthermore, provide <br /> genetic biochemical evidence suggesting func-        1. Abbreviations used paper: DAPI, 49,6-diamidino-2-phenylindole; <br /> tion cdc12p associate gene product       ECL, enhanced chemiluminescence; GST, <span id='am-26' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-27' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="glutathione" datatype="xsd:string"></span>glutathione</span>-S-transferase; PLP, <br /> quired ring formation, profilin cdc3p. Profilins          poly-l-proline. <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                     170 <br />  pFC104. pFC103 complemented cdc12-112 strain retransformation              cdc12 open reading frame pFC103 (ura41). nmt1 promoter contained 6-kb insert pUR19. Deletion analysis showed             fragment generated PCR using 59 nmt promoter oligonucleotide <br /> central region critical activity  Fig. 1 . genetic criteria   OKS3 GGGGGTACCCGCCATAAAAGACAGAATAAG 39 <br /> demonstrated complementing activity cdc12 gene:  hy-           nmt oligonucleotide, cut KpnI BamHI. stable pnmt-cdc12 <br /> bridization ordered cosmid blot (Hoheisel et al., 1993) showed         transformants (FC565) integrated plasmid chromo- <br /> pFC103 mapped left tip chromosome  corresponding location          unstable transformants (FC291) contain free plasmid cdc12 previously mapped genetically. Second, pFC110,            analyzed produced similar cdc12p localization patterns. contains 3-kb EcoRI cdc12 insert pFC103 sup3-5 <br /> marker  pSTA12), integrated genome homologous                Generation Anti-cdc12p Antibody <br /> recombination. Random spore analysis showed plasmid inte- <br /> grated cdc12 locus.  introduction COOH-terminal half          His6-tagged protein fragment cdc12p purified Escherichia coli cdc12 gene cdc12-112 does confer rescue 368C major-         used production polyclonal antibodies. protein ity cells, confers rare stable gene conversion cdc12-112 al-    obtained following manner: 0.9-kb cdc12 fragment encodes <br /> lele, suggesting genetic defect cdc12-112 resides COOH-        cdc12 FH1 domain PCR amplified oligonucleotides 107rev <br /> terminal half protein.                                                     (GGAAGGATCCAACACACTCGTTCATGC) 59 BamPst (GGA- <br />     cDNA additional genomic DNA clones cdc12 isolated              AGGATCCGCTGCAGAGTCAACACTT), cut BamHI,  <br /> delineate ends transcript test existence possible     serted BamHI site tag expression vector pQE9 make <br /> trons. pFC125, contains partial 2.5-kb cdc12 cDNA includes           pFC122. pFC122 (multiple isolates analyzed) transformed E. coli 39 half gene central EcoRI site, identified hy-        M15/REP4. His6-tagged protein E. coli purified Ni agarose <br /> bridization (Maniatis et al., 1982) S. pombe cDNA lYES library             using denaturing protocol insoluble fraction recommended <br /> (generous gift S. Elledge, Baylor University, Houston, TX; Elledge           QIAexpress manual (Qiagen). 2 mg purified protein acryla- <br /> et al., 1991) using central 1.0- 0.9-kb HindIII fragments probe.     mide gel slices eluted PBS injected rabbits. rabbit <br /> insert pFC125 subcloned TZ18R vector produce pFC126              (FC2) produced sera recognized 220-kD protein crude yeast ex- <br /> pFC127, used sequencing subcloning. pFC128,             tracts immunoblot. affinity purification, FC2 sera loaded contains 2.4-kb cdc12 cDNA covering 59 end cdc12, ob-          Sepharose 4B column containing purified denatured His6- <br /> tained PCR approach 59 RACE (Gibco BRL, Gaithersburg, MD).                   tagged cdc12p FH1 protein fragment eluted 4.5 M MgCl2 109ext1 (GCATCATTAGGAATATCA) used initial gene-spe-                     100 mM <span id='am-89' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-90' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> (acid) (Pfeffer et al., 1983). Eluates immediately dia- <br /> cific oligonucleotide 59 anchor oligonucleotide. cycles       lyzed PBS 1 30% glycerol. MgCl2 eluate used Western analy- <br /> PCR internal primers 109ext2 (GGAAGGATCCGTTGACACAG-                          sis, <span id='am-101' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-102' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span> eluate used immunofluorescence (described <br /> TTGAGGG) 109ext3 (CAATTCCATCGTTTGACC) allowed                            . amplification 2.4-kb fragment cloned TA KS clon- <br /> ing vector (gift J. Wuarin, Imperial Cancer Research Fund, London,           Western Analysis <br /> UK). pFC163, contains 5-kb genomic cdc12 fragment covering 59 end putative promoter region gene pDB248, cloned             Crude boiled yeast cell extracts prepared described Cor- hybridization Tamagawa Noguchi S. pombe genomic                   rea-Bordes Nurse (1995). Protein concentrations measured stan- <br /> DNA library (gift T. Toda, Imperial Cancer Research Fund; Hirano             dardized bicinchoninic acid protein assay (Pierce <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> , <br /> et al., 1988).                                                                    Rockford,  Coomassie staining. 5&#226;&#8364;&#8220;50 mg extract lane                                                                                   loaded 6% SDS-PAGE blotted 0.2-mm nitrocellulose using <br />                                                                                   Transfer Buffer (Harlow Lane, 1988) 8 V 16 h 48C. Blots <br /> Deoxynucleotide Sequencing                                                        washed <span id='am-64' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-65' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span>, stained Ponceau S, processed recom- <br /> Deoxynucleotide sequencing performed using ALF automated se-               mended using ECL Western blotting kit (Amersham Corp.) anti- <br /> quence machine using T4 polymerase sequencing kit Pharmacia                cdc12 affinity-purified antibodies 1:500 dilution TBS 1 0.1% <br /> Biotech  (Piscataway, NJ). cdc12 cDNA clones pFC127 pFC128           Tween. 5.5-kb cdc12 genomic clone XhoI BamHI derived pFC103 sequenced, using subclones primer walking. pFC127             Immunofluorescence <br /> cDNA (39 EcoRI) pFC155 genomic DNA (59 cdc12 BamHI EcoRI) sequenced completely strands, sequences              immunofluorescence, cells fixed cold methanol processed used cDNA genomic sequences did                   described (Snell Nurse, 1994). Anti-cdc12 antibody (1:10 dilution) <br /> agree; particular, numerous single base pair changes         sheep anti&#226;&#8364;&#8220;rabbit IgG CY3 conjugate secondary antibody (1:200 dilu- <br /> pFC128 cDNA isolated PCR, incorporated. addi-                tion; Sigma Chemical , St. Louis, MO) used visualize cdc12p. <br /> tion, pFC163 used confirm sequences 59 start site            anti-tubulin mAb TAT1 (1:5; generous gift K. Gull, University <br /> cluded pFC103 delineate start AUG. Sequences aligned               Manchester, Manchester, UK; Woods et al., 1989) anti-actin mAb analyzed using DNA analysis programs Assemblign, DNA                      (1:200; Amersham Corp.) anti&#226;&#8364;&#8220;mouse FITC conjugate secondary  <br /> Strider, DNA-Star.                                                            tibody (1:200) used visualize microtubules actin, respectively. <br />                                                                                   Stained cells mounted 1 mg/ml 49,6-diamidino-2-phenylindole <br /> cdc12::ura4 Deletion                                                              (DAPI) Vectashield (Vector Laboratories, , Burlingame, CA). Rhoda- <br />                                                                                   phalloidin staining <span id='am-113' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span>-fixed cells performed ex- 5.5-kb BamHI&#226;&#8364;&#8220;XhoI cdc12 DNA fragment pFC103 containing                 actly described previously (Chang et al., 1996). Cells photo- <br /> entire cdc12 open reading frame ligated BamHI XhoI               graphed described Chang et al. (1996) Northern Exposure <br /> sites KS1 vector produce pFC161. pFC162 (cdc12::ura4)               imaging (Phase 3 Imaging Systems, Glen Mills, PA). <br /> structed replacement 4.1 kb PstI&#226;&#8364;&#8220;SalI fragments cdc12 FC161 1.8-kb PstI&#226;&#8364;&#8220;SalI ura4 fragment. replace wild-type cdc12         Vitro Binding Experiments <br /> gene cdc12::ura4, pFC162 cut BamHI XhoI,  3.8-kb band gel purified transformed ura4/ura4 leu1/leu1      pKG251 (Balasubramanian et al., 1993) gift K. Gould M. <br /> ade6-M210/ade6-M216 h1/h2 diploid. Multiple cdc12::ura4 heterozygous              Balasubramanian (University Tennessee, Nashville). Soluble <span id='am-118' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-119' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span>glutathione</span>- <br /> diploids confirmed Southern blotting PCR. random spore             S-transferase (GST)&#226;&#8364;&#8220;cdc3p GST proteins expressed pKG251 <br /> analysis, cdc12::ura4 heterozygous diploids grown minimal media           pGAT2 E. coli strain BL21 purified glutathione beads. His6- leucine late exponential phase, washed minimal media            tagged cdc12p expressed pFC122 (described , purified  nitrogen, allowed sporulate 258C 2 d, digested heli-        der denaturing conditions, renatured dialysis 8 M <span id='am-128' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-129' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span>-PBS <br /> case (Moreno et al., 1991). resultant spores germinated mini-         stepwise fashion PBS. cdc12p protein solution precleared  <br /> mal media <span id='am-38' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> leucine 15 h 308C.                              fore binding assays centrifugation 14K rpm 10 min mi- <br />                                                                                   crofuge. Final concentration cdc12p protein 0.6 mg/ml. Approxi- <br /> cdc12 Overexpression                                                              mate molecular masses proteins  GST-cdc3 5 40 kD                                                                                   cdc12p 5 45 kD. Poly-l-proline (PLP; 1,000&#226;&#8364;&#8220;10,000; average mol wt 5,600; <br /> pnmt-cdc12 constructed insertion moderate strength nmt1             n 5 50; Sigma Chemical ) diluted PBS. binding experiments, <br /> promoter pREP41X KpnI&#226;&#8364;&#8220;BamHI site 60 bp upstream                  20 ml appropriate dilution cdc12p PBS, 20 ml 50% beads PBS <br />  <br />  <br />  <br />  <br /> Chang et al. Schizosaccharomyces pombe Cell Division Protein cdc12p               171 <br />  Figure 1. cdc12 encodes protein similar dia/BNI1 family proteins.   Localization cdc12 gene. Restriction map original 6-kb genomic clone insert (pFC103) rescues cdc12-112 mutant cells. (Arrow) Extent orientation single large <br /> open reading frame region. Truncated versions clone ability complement cdc12-112 mutant; 11, rescue; <br /> 1, partial rescue. cdc12::ura4 deletion construct used deleting cdc12. cdc12 open reading frame replaced 1.6-kb <br /> ura4 insert. (B) cdc12 open reading frame. Regions predicted coiled-coil underlined (Lupas et al., 1991), polyproline  <br /> gions boxed. (C) Similarity gene domain structure. Proteins share central FH1 polyproline rich domain (black box), FH2 <br /> domain (stripe box), larger COOH-terminal area homology (gray) diagrammed D. (D) Similarity proteins COOH-terminal region. Proteins aligned using Megalign DNA Star. Residues identical calculated consensus colored black, conserved residues colored gray. sequence data available GenBank/EMBL/DDBJ accession <br /> number Q10059. <br />  <br />  <br /> containing appropriate dilutions protein, 140 ml CB buffer (0.6 M   <span id='am-69' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-70' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-72' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequences genomic cDNA clones <br /> sorbitol, 50 mM Tris, pH 7.5, 140 mM NaCl, 5 mM EDTA, 0.06% Triton            helped delineate ends mRNA transcript X-100) mixed incubated 200 ml tube room temperature inversion 1 h. PLP inhibition experiments, 5 ml PLP buffer     demonstrated cdc12 does contain introns. added binding reactions. Aliquots taken total          Conceptual translation nucleotide sequence pre- <br /> sample. Supernatant pellet fractions collected centrifugation   dicts cdc12 encodes polypeptide 1,841 <span id='am-77' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-83' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> picofuge (Stratagene) 30 s. Fractions analyzed SDS-          residues. Features deduced polypeptide include PAGE gels Coomassie staining. Densitometry bands  <br /> formed 1000 gel documentation (Alpha Innotech Corp., <br />                                                                               predicted coiled-coil domains (Lupas et al., 1991) San Leandro, CA).                                                             central region containing stretches polyproline (Fig. <br />                                                                               1 B). Database searches using BLAST algorithm  <br />                                                                               vealed cdc12 gene product similar emerg- <br /> Results                                                                       ing family proteins include Bni1p S. cerevi- <br />                                                                               siae, diaphanous Drosophila, fus1p S. pombe, <br /> cdc12 Encodes Member Newly Recognized Family                           figA/sepA Aspergillus nidulens, cappuccino Cytokinesis Proteins                                                       Drosophila, formins mouse, human, chicken fission yeast cdc12-112 mutant recessive tempera-                    (Fares, H., J. Pringle, personal communication; Pe- <br /> ture-sensitive lethal mutant specific defect actin                  tersen et al., 1995; Harris et al., 1997; Castrillon ring formation cell division (Nurse et al., 1976; Marks                   Wasserman, 1994; Emmons et al., 1995; Woychik et al., <br /> et al., 1987; Chang et al., 1996). cloned cdc12 gene                   1990; Maas et al., 1991; Trumpp et al., 1992). pro- screening plasmid genomic DNA library                         teins, diaphanous, Bni1p, sepA, cappuccino rescue temperature-sensitive lethality cdc12-               implicated cytokinesis (Castrillon Wasserman, 1994; <br /> 112 mutant. Multiple genetic criteria showed plas-                   Kohno et al., 1996; Harris et al., 1997; Manseau et al., <br /> mid pFC103 contained cdc12 gene  Materials                        1996). <br /> Methods). Sequence analysis clone revealed                            proteins share regions homology,  <br /> large open reading frame (Fig. 1 . Deletion analysis (Fig.                  cluding domains termed FH1 FH2 <br /> 1  showed open reading frame responsible                        (formin homology domains) (Castrillon Wasserman, <br /> complementation cdc12-112 mutation. Comparison                         1994). FH1 domain includes proline-rich sequences <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                                 172 <br />                                                                           NH2-terminal half proteins  shown).  <br />                                                                          letion analysis cdc12 suggests COOH-terminal <br />                                                                          thirds protein, contains conserved <br />                                                                          FH1 FH2 domains, essential sufficient activ- <br />                                                                          ity (Fig. 1 . <br />  <br />                                                                          cdc12 Essential Gene Required Actin Ring <br />                                                                          Septum Formation <br />                                                                          determine function cdc12, examined ef- <br />                                                                          fects cdc12 deletion. cdc12::ura4 deletion construct <br />                                                                          .70% open reading frame,  <br />                                                                          cluding conserved FH1 FH2 domains,  <br />                                                                          placed ura4 gene. construct transformed <br />                                                                          diploid strain, multiple stable ura1 diploid <br />                                                                          transformants, confirmed heterozygous <br />                                                                          homologous replacement cdc12 gene,                                                                          sporulated analyzed. Tetrad analysis showed                                                                          spores dead                                                                          viable haploid ura41 spores, demonstrating                                                                          cdc12::ura4 haploid cells viable. Microscopic ex- <br />                                                                          amination individual spores showed inviable <br />                                                                          spores tetrad germinated grew single <br />                                                                          large swollen dumbbell- pear-shaped cell did                                                                          divide. random spore analysis, introduction plasmid <br />                                                                          containing cdc12 gene rescued lethality                                                                          cdc12::ura4 spores, indicating lethal phenotype  <br />                                                                          sociated cdc12::ura4 disruption Figure 2. Phenotype cdc121 cdc12::ura4 deletion cells.                                                                           cdc12 gene.  cdc12 essential viability. B). Wild-type cells stained rhodamine-phalloidin,  <br /> ing normal interphase   mitotic (B) actin patterns. (C&#226;&#8364;&#8220;L)             Germinated cdc12::ura4 spores examined ef- <br /> cdc12::ura4 cells stained rhodamine-phallodin (C, E, G,         fects actin septum organization. Wild-type cells form , DAPI (D, F, H, J), calcofluor (K L). Diploid ura4/        cortical actin patches ends cells interphase <br /> ura4 leu1/leu1 ade6-M16/ade6-M22 cdc121/cdc12::ura4 cells           actin ring mitosis (Fig. 2, B; Marks <br /> sporulated, spores germinated media containing           Hyams, 1985). cdc12::ura4 cells normal interphase <span id='am-123' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-124' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>, cdc12::ura4 spores germinate. Spores            cortical actin patches (Fig. 2, C E), delocal- grown 15 h 308C, fixed, stained. (C&#226;&#8364;&#8220;F) cdc12::ura4       ized actin patches instead actin ring mitosis <br /> cells interphase, showing normal interphase polarized actin           (Fig. 2, G . mitotic cells, actin dots distribution cell tips. Cell D nuclei, cell F   concentrated zones actin ring proceeded generation contains nu- <br />                                                                          formed (Fig. 2 . cdc12::ura4 cells multinucleate <br /> clei. (G&#226;&#8364;&#8220;J) cdc12::ura4 mitotic cells actin dots instead actin ring. Cell G focused middle cell,    (Fig. 2), indicating nuclear division occurred ab- <br /> cell focused cell surface. Nuclear patterns H    sence cytokinesis. phenotype identical characteristic cells mitosis  Fig. 6 comparison).    seen temperature-sensitive cdc12 alleles restrictive <br /> Calcofluor-stained cells septum cell wall deposition        temperature (Chang et al., 1996; Marks et al., 1987; Nurse multiple nuclear divisions. rounded bulge stains          et al., 1976).  cdc12 required actin ring assem- Calcofluor (K L) cdc12::ura4 cells represents site       bly mitosis, required actin organiza- original spore wall. Bar, 10 mm.                                  tion interphase. <br />                                                                             cdc12::ura4 cells depositions septal material <br />                                                                          (Fig. 2, K L). contrast known tempera- potential binding sites profilin SH3  <br />                                                                          ture-sensitive alleles cdc12, display multiple aber- <br /> mains (Tanaka Shibata, 1995; Mayer Ekd, 1995). <br />                                                                          rant patches septal materials (Chang et al., 1996; FH2 domain conserved region (striped box Fig. <br />                                                                          Streiblova, 1984; Nurse et al., 1976).  cdc12 1 C) located near COOH terminus. detected <br />                                                                          necessary septum formation. additional 300&#226;&#8364;&#8220;amino acid region similarity COOH-terminal proteins encompassing FH2 domain  gray box Fig 1 C; Fig. 1 D). simi- <br />                                                                          cdc12 Component Cell Division Ring <br /> larity strongest fungal members: cdc12p              localize cdc12 gene product, raised polyclonal <br /> 31&#226;&#8364;&#8220;35% identical 56&#226;&#8364;&#8220;60% similar 300&#226;&#8364;&#8220;amino                    rabbit antibodies cdc12 FH1 domain protein frag- <br /> acid domain fungi proteins, 26% identical               ment expressed E. coli. affinity-purified antibody 50% similar proteins. Phylogenetic tree               recognized immunoblot single band z220 kD <br /> analysis shows S. pombe fus1 similar cdc12.               (Fig. 3 , consistent size predicted cdc12p little detectable similarity vertebrate              nucleotide sequence. 220-kD protein  <br /> formins outside FH1 FH2 regions. cdc12p, fus1p,               firmed cdc12p showing level pro- <br /> Bni1p, figAp share limited regions homology                  tein increased z10&#226;&#8364;&#8220;20-fold cells containing plasmid <br />  <br />  <br />  <br />  <br /> Chang et al. Schizosaccharomyces pombe Cell Division Protein cdc12p      173 <br />  Figure 3. cdc12p localizes cell division ring wild-type cells.   Specificity anti-cdc12 affinity-purified antibody. 5 mg crude yeast cell extracts analyzed Western blotting using affinity-purified anti-cdc12 antibody. (Lane 1) Extract wild-type cells. (Lane 2) Extract cells overexpressing cdc12p. Cells transformed plasmid pnmt-cdc12 (FC291) shifted derepressing conditions (minimal media thiamine) 20 h 308C. Molecular weight markers (kD) indicated. Serial dilu- <br /> tions showed pnmt-cdc12 extract contains 10&#226;&#8364;&#8220;20-fold cdc12p wild-type extracts (data shown). (B&#226;&#8364;&#8220;K) cdc12p lo- <br /> calization wild-type cells. Wild-type cells grown YE5S 308C exponential phase, fixed, processed immunofluores- <br /> cence using anti-cdc12 antibody DAPI  Materials Methods details). (B, D, F, H, J) Anti-cdc12 staining. (C, E, G,  K) DAPI staining. (B C) typical field cells containing cells different phases cell cycle. Note examples ring <br /> staining. (e) Cell early mitosis ring beginning form asymmetrical fashion.   Interphase specific cdc12 <br /> staining. (B, inset) Cross-section cdc12p ring. (D E) Cell early mitosis; (F G) cell early anaphase; (H  cell mid- <br /> anaphase; (J K) cell postanaphase ring closure. Patchy cytoplasmic staining seen cells lacking cdc12p, probably represents background staining (data shown). Bar, 10 mm. <br />  <br />  overexpresses cdc12p thiamine-regulated                    small stained weakly wild-type cells. <br /> nmt promoter (Fig. 3 . Measurements cdc12p levels                   moderate strength thiamine-repressible nmt1 promoter synchronized cells cells arrested G2               (derived pREP41x) cloned 59 cdc12 open <br /> (cdc25 block) G1 phases (cdc10 block) showed                    reading frame (pnmt-cdc12; Materials Methods), <br /> cdc12p expressed cell cycle large              construct integrated chromosome. Cells <br /> fluctuations protein levels mobility shifts different           carrying construct expressed cdc12p 10&#226;&#8364;&#8220;20-fold <br /> phases cell cycle (data shown).                               higher wild-type levels, retained normal growth <br />    Immunofluorescence using anti-cdc12p antibody                 rate, cell morphology, cell division, actin distribution <br /> vealed staining fission yeast cell division ring (Fig.            20 h induction (data shown). <br /> 3). cdc12p present early mitotic cells  faint             Overexpression cdc12p revealed structures ring (Fig. 3 D). ring intense ana-                  previously observed wild-type cells (Figs. 4 5; <br /> phase (Fig. 3 H)  completion anaphase, ap-                  Table II). interphase, common pattern <br /> peared contract single dot daughter                  (37% cells) discrete medial spot located  <br /> cells cytokinesis (Fig. 3 J). cdc12p staining            tween nucleus cortex (Fig. 4, cell 1; Table II). <br /> similar staining patterns components                  dot associated similar dot F-actin <br /> ring actin cdc4p (Marks Hyams, 1985; Mc-                 (data shown). 18% cells exhibited dot Collum et al., 1995). cdc12p did localize actin             cell surface near end cell, presumably structures, interphase actin patches ends              midbody remnant previous division (Fig. 4, cell 5; <br /> cells actin dots adjacent septum septation              Table II). 33% cells exhibited specific staining, <br /> (Marks Hyams, 1985). cdc12p staining spe-                    frequent patterns multiple dots cific ring staining cdc12-112 mu-                observed (Fig. 4, cell 6; Table II). Medial spot staining <br /> tant cells  Fig. 5 ,  addition, green fluores-            appeared cell cycle regulated, (96%) <br /> cent protein&#226;&#8364;&#8220;cdc12 fusion located similar structures               cells beginning interphase  short cells) <br /> (Chang, F., unpublished observations).                                   exhibited medial spot staining, (76%) late <br />    interphase cells, weak, diffuse cytoplasmic stain-            interphase cells (long cells) exhibited medial spot. <br /> ing detected (Fig. 3 B, cell . largely               early mitosis, cells exhibited similar medial spot <br /> nonspecific background staining, similar weak cyto-                (17%; Fig. 5 C), spot intersecting faint cdc12 ring <br /> plasmic staining present cdc12-112  Fig. 5               strand (31%; Fig 5, B C), ring similar  cdc12::ura4 cells (data shown).                               wild-type cell (50%; Fig. 5 D). Examination cells dif- <br />                                                                          ferent focal planes (Fig. 5) showed cdc12p spot                                                                           instance, ring seen particular focal plane <br /> Cells Overexpressing cdc12 Exhibit cdc12p Spot                         (compare spot cell ring cell D). cells examined cdc12p localization cells overexpress-               spot ring  strand) observed (Fig. 5, B <br /> ing cdc12p attempt visualize cdc12p structures                C), structures coincided, suggesting <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                            174 <br />                                                                                                           Figure 4. overexpres- <br />                                                                                                          sion, cdc12p localizes                                                                                                          medial spot. FC565 (pnmt- <br />                                                                                                          cdc12) cells induced                                                                                                          derepressing conditions                                                                                                          20 h 308C, fixed cold <br />                                                                                                          methanol, processed                                                                                                          immunofluorescence                                                                                                                   anti-cdc12 antibody (left pan- <br />                                                                                                          els), anti-tubulin antibody <br />                                                                                                          (middle panels), DAPI <br />                                                                                                          (right panels). tubulin <br />                                                                                                          DAPI staining patterns <br />                                                                                                          identify cell cycle stage. <br />                                                                                                           C) panels                                                                                                          representative fields                                                                                                          population cells. <br />                                                                                                          (Cell 1) Interphase cell                                                                                                          medial cdc12p spot. (Cell 2) <br />                                                                                                          Early mitotic cell  <br />                                                                                                          dial spot. (Cell 3) Cell                                                                                                          postanaphase cdc12p <br />                                                                                                          ring. (Cell 4) Cell post- <br />                                                                                                          anaphase closed ring. <br />                                                                                                          (Cell 5) Cell cell divi- <br />                                                                                                          sion cdc12p spot                                                                                                          presumed midbody                                                                                                          cell tip. (Cell 6) Cell                                                                                                          spots, cell tip                                                                                                          cell middle. Bar, <br />                                                                                                          10 mm. <br />  <br />  spot mark rise ring.             mitosis, like cdc12-299 mutant (Fig. 6, D, left <br /> cells, half rings appeared discontinuous                 panels). addition, cdc15 mutant cells overexpressed spot (Fig. 5 C, panel). late            cdc12p did exhibit cdc12p spot ring staining mitosis, nearly cells exhibited cdc12p ring seen          interphase mitotic cells (data shown).  <br /> wild-type cells (Fig. 4, cells 3 4).                              cdc3 (profilin homologue), cdc8 (tropomyosin homo- <br />    corresponding medial spot readily detectable             logue), cdc15 genes necessary cdc12p localiza- wild-type cells overexpress cdc12,         tion ring, cdc15 required known cdc12p resides spot wild-type            cdc12p spot formation. <br /> cells. small faint dot proper location occasion-            contrast, cdc12p localized single medial corti- <br /> ally seen, usually significantly intense          cal spot cdc4-377 mutant cells mitosis (Fig. 6 E; background cytoplasmic dots.                    Table II). early mitotic cells (76%) exhibited observed cells cdc12p ring staining appears             staining, late mitotic cells (76%) exhibited stronger cell  cell e Fig. 3 B).        spot (Table II), suggesting spot formed eral explanations cdc12p spot considered           mitosis. addition, cells (6% late mitotic cells) <br /> Discussion.                                                           possessed spot faint intersecting ring.                                                                       cdc4 mutant, cdc12p accumulated medial spot place <br />                                                                       ring. relationship cdc12p actin exam- <br /> cdc12p Localization Different Actin Ring Mutants <br />                                                                       ined cdc4-377 mutant cells. Instead actin ring,  examined expression localization cdc12p               cells form asterlike array actin cables organized different actin ring mutants.  immunoblot anal-          single cortical point (Fig. 6 F; Chang et al., 1996). <br /> ysis showed expression cdc12p altered              Double staining revealed cdc12p dot located <br /> cdc3, cdc4, cdc8, cdc15, mid1 mutants (data shown).            center larger actin aster (Fig. 6 F). <br /> Second, examined cdc12p localization different mu-                 mid1-366 mutants exhibited highly variable patterns tant backgrounds. cdc3-313, cdc8-346, cdc15-287 mu-               reflect multiple defects spatial organization ring <br /> tants generally showed specific cdc12p staining             (Fig. 7; Table II). Generally, mid1 mutants medial <br />  <br />  <br />  <br />  <br /> Chang et al. Schizosaccharomyces pombe Cell Division Protein cdc12p   175 <br />  Figure 5. positions cdc12p spot ring coincide. FC565 (pnmt-cdc12) cells processed described Fig. 4. Shown representative cells early mitosis, period ring formation. panels different focal planes cell, panels tubulin DAPI staining labeled.   Cell cdc12p spot. (B) Cell cdc12p <br /> associated complete ring. (C) Cell cdc12p spot incomplete ring. Note discontinuity ring adjacent spot panel. (D) Cell cdc12p ring spot. Bar, 10 mm. <br />  <br /> cdc12p spot associated single strand ( Fig. 7, C).         dial spot cdc12p (Fig. 7 D) staining  90% strands associated nucleus            shown). 30% spots positioned away point strand, strands wrapped                  cell middle nucleus (Fig. 7 E), indicating appar- nucleus (Fig. 7 C). cells                ent defect placing spot. (,5%) com- <br />  <br />                                                                                                         Figure 6. Localization                                                                                                         cdc12p different actin ring <br />                                                                                                         mutants mitosis.   <br />                                                                                                         cdc12-299 (FC127); (B) cdc3- <br />                                                                                                         313 (FC114); (C) cdc8-346 <br />                                                                                                         (FC131); (D) cdc15-287 <br />                                                                                                         (FC55); (E F) cdc4-377 <br />                                                                                                         (FC361). Mutant cells                                                                                                         grown YE5S permissive <br />                                                                                                         temperature (258C),                                                                                                         shifted restrictive temper- <br />                                                                                                         ature (35.58C) 4 h. Cells <br />                                                                                                         fixed processed                                                                                                         immunofluorescence                                                                                                                 anti-cdc12 antibody, anti-tubu- <br />                                                                                                         lin antibody anti-actin anti- <br />                                                                                                         body, DAPI. ba- <br />                                                                                                         sis spindle nuclear <br />                                                                                                         morphology, cells pictured <br />                                                                                                         anaphase, cell cycle <br />                                                                                                         period maximum cdc12p <br />                                                                                                         ring staining wild-type cells. <br />                                                                                                          C) Cells mitosis                                                                                                         nuclei dividing <br />                                                                                                         exhibit <br />                                                                                                         double spindles; (E&#226;&#8364;&#8220;F) cells <br />                                                                                                         exhibit single spindles.  C) <br />                                                                                                         Mutants exhibit specific <br /> cdc12p staining; cytoplasmic staining nonspecific background staining. (E F) Note medial spot cdc12p (arrow) cdc4 mutant, localizes origin actin aster (F, middle panel, labeled . Bar, 10 mm. <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                          176 <br />  Table II. cdc12p Localization Patterns Wild-type Mutant Cells <br />                                            Percentage cells particular cdc12p <br />                                                       localization pattern <br />  <br />  <br />  <br />  <br />                         Number Cell type              cells assayed <br />  <br />                                                                 % <br />  <br /> Wild type                   300          100     0     0    0       0    0     0     0 <br />   interphase <br /> Wild type                    50            52    0     0 48         0    0     0     0 <br />   early mitosis <br /> Wild type                    80             2    0     0 98         0    0     0     0 <br />   late mitosis <br /> OP cdc12                    205            33 37       0    0 18        12     0     0 <br />   interphase <br /> OP cdc12                     70             1 17 31 50              0    0     0     0 <br />   early mitosis <br /> OP cdc12                     80             1    0     0 99         0    0     0     0 <br />   late mitosis <br /> cdc8 mitosis                 50          100 0         0 0          0   0     0     0 <br /> cdc15 mitosis                50           88 0         0 6          0   0     0     0 <br /> cdc4 early mitosis           46           76 13        0 11         0   0     0     0 <br /> cdc4 late mitosis            46           15 76        6 2          0   0     0     0 <br /> mid1 early mitosis           70           39 16i       0 0          0   0     3    43 <br /> mid1 late mitosis            70            4 3         0 13         0   0    34    46 <br /> Cells fixed stained anti-cdc12 anti-tubulin antibodies DAPI. <br /> Individual cells screened fluorescence microscopy particular cell <br /> cycle stage basis tubulin staining, examined cdc12p local- <br /> ization. Cells short mitotic spindle nuclear division designated early mitosis. Cells anaphase postanaphase designated late mitosis. Cells interphase microtubule array single nucleus designated interphase. Dots lines indicate cdc12p localization, circle indicates nucleus. OP (overproducing) cdc12: cells (FC566) contain chromosomal integrated copy pnmt-cdc12 induced minimal  <br /> dia 20 h 308C. cdc8-346 (FC131), cdc15-287 (FC55), cdc4-377 (FC361) <br /> mutants grown 4 h restrictive temperature 35.58C. mid1-366 <br /> (FC164) cells grown 2 h 35.58C.   Cells specific cdc12p stain- <br /> ing. (b) Cells medial cdc12p spot. Interphase spots adjacent nu- <br /> cleus. (c) Cells cdc12p spot intersecting ring. (d) Cells normal <br /> cdc12p ring. (e) Cells cdc12p dot near end cell, probably midbody remnant, left previous division. (f) Cells multiple dots      Figure 7. Localization cdc12p mid1 mutant cells early <br /> expressing cells dots covered b e. cells category com-   mitosis. mid1-366 mutant cells grown YE5S permissive <br /> bination patterns seen b e. (g) Cells intact displaced ring. (h) <br /> Cells cdc12p strand. associated dot, .90% associated <br />                                                                                           temperature (258C), shifted restrictive temperature nucleus.   mid1 cells, dots cortex, dots     (35.58C) 2 h. Cells processed immunofluorescence <br /> (z30%) displaced away medial nucleus.                                        anti-cdc12 antibody, anti-tubulin antibody anti-actin anti- <br />                                                                                           body, DAPI. basis spindle nuclear morphol- <br />                                                                                           ogy, cells pictured early mitosis, cell cycle period <br />                                                                                           ring formation. Note different configurations cdc12p <br /> plete rings present early mitosis, complete                                  dot strand (arrows). Bar, 10 mm. <br /> rings displaced tilted (47% cells) seen late mitosis (Table II; Chang et al., 1995), sug- <br /> gesting cdc12p strand early mitosis orga-                                 work) suggests proteins interact nize displaced ring later mitosis.  mid1 mu-                               vivo.  tested interaction tants defects guiding strand                                    gene products using genetic biochemical methods. <br /> circumference cell form ring medial                                    tested cdc3 cdc12 interact geneti- <br /> placement spot.                                                                    cally constructing cdc12 cdc3 double mutants. Fig. <br />                                                                                           8 shows phenotypes wild-type, cdc3, cdc12, cdc3 <br /> Evidence cdc12p Interacts Profilin cdc3p                                        cdc12 mutants. cdc3-313 cdc12-112 double mutants existence proline-rich sequences cdc12p suggests                                severe phenotype single mu- cdc12p bind profilin homologue cdc3p,                               tant. cdc3 cdc12 mutant spores dead 258C profilins known bind proline-rich sequences (Ta-                                produced cells poor growth 258C naka Shibata, 1985). addition, evidence cdc3                                  growth intermediate temperature 298C. com- cdc12 share similar mutant phenotypes (Chang et al.,                                  parison, single cdc3 cdc12 mutants grew 258C <br /> 1996) gene products localize septum                                  298C. single mutants double mutants did <br /> region cell division (Balasubramanian et al., 1994;                                grow 368C.  cdc3 cdc12 genes exhibit  <br />  <br />  <br />  <br /> Chang et al. Schizosaccharomyces pombe Cell Division Protein cdc12p                       177 <br />                                                                         Discussion <br />                                                                        cdc12p Cell Division <br />                                                                        cdc12 encodes member emerging family proteins <br />                                                                        function cytokinesis actin-mediated pro- <br />                                                                        cesses. mutant phenotype cdc12 indicates                                                                        required actin ring formation septum formation <br />                                                                        cell division: cdc12 mutants exhibit actin dots instead                                                                        actin ring, form septum, accumulate                                                                        large, multinucleate cells. cdc12p localization ring  <br />                                                                        dicates likely structural component                                                                        cell division ring. contrast actin-binding pro- <br />                                                                        teins organize multiple actin structures (Balasubra- <br />                                                                        manian et al., 1992), cdc12 function specifically ring <br />                                                                        formation mitosis: cdc12 mutants affected <br />                                                                        actin patch organization interphase (Chang et al., <br />                                                                        1996; Fig. 2), cdc12p located actin <br />                                                                        structures actin patches. cdc12p role                                                                        directing septum cell wall formation outside plasma <br />                                                                        membrane proximal ring involved                                                                        targeting cell wall synthetases ring site. <br /> Figure 8. cdc3 cdc12 exhibit synthetic-lethal genetic inter- <br /> action. Siblings tetratype tetrad cross cdc3-313 cdc12-112 mutants streaked YE5S plates incu-           Interaction cdc12p Profilin <br /> bated 258, 298, 368C. Growth assayed colony forma- <br /> tion. Note cdc3 cdc12 double mutant exhibited markedly        imagine cdc12p component multiprotein <br /> reduced growth 258 298C single mutants        complex organizes actin plasma membrane. <br /> ble. Strains FC322 (cdc12-112), FC323 (wt), FC324 (cdc3-          evidence cdc12p binds directly 313), FC325 (cdc3-313 cdc12-112).                                  actin-binding protein, profilin cdc3p. profilin                                                                        bind sequential prolines (Machesky                                                                        Pollard, 1993; Perelroizen et al., 1994), likely synthetic-lethal interaction, suggests gene           sequential stretches prolines FH1  <br /> products interact level vivo.                           main cdc12 contribute interaction. <br />     tested cdc3p cdc12p interact di-                  vivo observations support vitro interac- <br /> rectly vitro binding assay. GST&#226;&#8364;&#8220;cdc3p fusion pro-             tion data.  cdc3 cdc12 mutants similar <br /> tein GST protein expressed E. coli puri-               phenotypes: mutants defective actin ring  <br /> fied glutathione-coupled agarose beads (Fig. 9  lanes             sembly display disorganized actin dots place 1 2). length cdc12 fusion protein ex-            ring mitosis (Chang et al., 1996; Marks et al., 1985; <br /> pressed E. coli, smaller His6-tagged fragment             Balasubramanian et al., 1994). Second, cdc3p cdc12p <br /> cdc12p included central proline-rich FH1 domain               located regions cell: cdc3 located expressed E. coli purified using Ni-affinity                mitosis band cell middle overlaps <br /> chromatography (Fig. 9  lane 3). cdc12p protein                  broader narrow ring cdc12p (Balasubra- <br /> fragment mixed GST-cdc3 beads GST beads,                   manian et al., 1994).  cdc3 required local- binding assayed determining                   ization cdc12p ring, cdc12p localized cdc12p pelleted beads. Fig. 9, B C,             cdc3 mutant (Fig. 6 B). Fourth, cdc3 cdc12 mutants cdc12p bound beads containing GST-cdc3 did                 exhibit synthetic-lethal genetic interaction.  bind beads GST glutathione  cdc12p           genetic biochemical data support idea cdc12p <br /> binding dependent concentration cdc3p.                profilin interact physically function ac- <br /> test proline-rich region cdc12p responsi-           tin ring assembly. <br /> ble profilin binding, assayed binding pres-             profilin intensely studied, role reg- <br /> ence poly-l-proline competitor. Binding inhibited               ulating actin vivo understood. vitro, addition PLP peptides (Fig. 9, D E). 5:1-              profilin potential inhibit actin polymeriza- <br /> fold molar excess PLP <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> cdc12 suffi-               tion binding actin monomers drive actin <br /> cient inhibition, 1:2 molar deficiency             polymerization, possibly affecting nucleotide-bind- <br /> suggesting inhibition specific. Addition 1%            ing state actin lowering critical concentration <br /> BSA did abolish binding (data shown). ex-              actin polymerization (Pantaloni Carlier, 1993; Ther- <br /> periments necessary derive accurate binding              iot Mitchison, 1993). vivo, profilin located <br /> constant. data indicate cdc12p fragment              sites active actin assembly, instance, end capable binding cdc3p profilin direct specific          Listeria motile bacterium, promote actin <br /> manner interaction likely mediated            polymerization (Theriot et al., 1994). S. pombe, profilin PLP binding site profilin- proline-rich sequence (s)        appears active role actin ring assembly (Bala- <br /> present FH1 domain cdc12p.                                   subramanian et al., 1994).  cdc12p influ- <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                          178 <br />  Figure 9. cdc12p fragment binds profilin cdc3p vitro.   Purified proteins used studies. (Lane 1) Purified GST-cdc3. <br /> (Lane 2) Purified GST. (Lane 3) Purified tagged cdc12p fragment. (B C) cdc12p binds GST-cdc3 GST. 180-ml reaction, cdc12p (1.6 mM final concentration) 20-ml 50% slurries glutathione agarose beads containing indicated concen- <br /> trations GST-cdc3 GST added reaction. Reactions incubated room temperature 1 h, beads pelleted. (B) Coomassie-stained gel total proteins pelleting (lane 1) pellet fractions (lanes 2&#226;&#8364;&#8220;10) equivalent volumes. <br /> Bands quantitated densitometry relative total protein lane 1 plotted C. (D E) Poly-l-proline inhibits binding. <br /> Varying amounts poly-l-proline added binding reactions similar ones described cdc12p 0.8 mM. (D) <br /> Coomassie-stained gel pellet fractions. (E) cdc12p pellets quantitated densitometry relative supernatant fractions <br />  shown). <br />  <br />  <br /> ence actin assembly ring formation              interact rho GTPases (Kohno et al., <br /> teraction profilin.                                               1996; Evangelista et al., 1997). cdc12p FH1 domain <br />    evidence members cdc12p fam-             contains proline-rich consensus sequences binding ily interact profilin. Drosophila, cappuccino             SH3-binding domains. coiled-coil domains profilin mutants possess similar phenotypes oogen-              cdc12p potential sites intramolecular intermolec- <br /> esis, interaction demonstrated              ular interactions cdc12p protein molecules hybrid (Manseau et al., 1996). hybrid, genetic,             coiled-coil proteins cdc15p, myosin, biochemical data suggest S. cerevisiae BNI1              septins. large protein multiple domains, cdc12p <br /> interacts profilin (Evangelista et al., 1997). Recently,          act scaffold brings multiple ligands. proline-rich proteins implicated profilin ligands: VASP Mena related proteins appear regulate actin assembly focal adhesions <br />                                                                        Evidence Pathway Ring Formation actin tail motile bacteria Listeria (Rein-           Analysis cdc12p localization different cell types  <br /> hard et al., 1992, 1995; Chakraborty et al., 1995; Gertler et          vealed intriguing novel patterns suggest site <br /> al., 1996). profilin VASP associated              ring assembly initiated single point (Fig. 10). <br /> nucleating center actin tail Listeria, possibil-        cdc12p-overexpressing cells, cdc12p localized ity cdc12p profilin associated nucle-           single medial spot nucleus cortex dur- <br /> ating center ring.                                             ing late interphase early mitosis (Figs. 4, 5, 7). <br />    cdc12p possess additional binding partners. BNI1p               cdc12p spot located precisely future site  <br />  <br />  <br />  <br /> Chang et al. Schizosaccharomyces pombe Cell Division Protein cdc12p    179 <br />                                                                           Numerous aspects model remain tested.                                                                       instance, cdc12p spot readily visualized                                                                       wild-type cells overexpression, known                                                                       cdc12p spot present interphase wild-type cells,                                                                       cdc12p affinity unidentified medial <br />                                                                       structure does contain cdc12p normally. alter- <br />                                                                       nate model cdc12p spots represent aberrant ag- <br />                                                                       gregates cdc12p. instance, overexpression cdc12p <br />                                                                       interphase induce formation aggre- <br />                                                                       gate ring proteins including myosin contracts                                                                       single tight ball.  models, fact <br />                                                                       cdc12p spot anticipates ring site suggests                                                                       spatial signals position ring present  <br />                                                                       terphase cdc12p associated proteins capa- <br />                                                                       ble responding spatial cues.  cdc12p <br />                                                                       spot regarded marker reveals activity <br />                                                                       spatial regulators position division site. <br />  <br />                                                                       Roles Ring Genes Ring Formation <br /> Figure 10. model establishment division site S.   Analysis different ring mutants suggests differ- <br /> pombe.  B) interphase, marker spot established       ent ring genes play different roles ring formation. cdc12, cell middle, possibly signal nucleus. spot     cdc15, cdc3 (profilin homologue), cdc8 (tropomyosin contain cdc12p; alternatively, overexpression, cdc12p        homologue) appear involved early step ring accumulate structure. (C) early mitosis, spot    formation, possibly nucleation ring. cdc12p remains cortex. (D) early mitosis, ring forms                                                                       localized cdc15, cdc3, cdc8 mutants, suggesting site marked spot extension strand spot circumference cell. (E) anaphase,        role gene products localize cdc12p. <br /> ring robust. (F) nuclear division, ring        cdc15p, SH3 domain&#226;&#8364;&#8220;containing protein, impor- <br /> closes septation cell division. Roles actin ring    tant regulator ring formation cell cycle (Fank- <br /> genes inferred cdc12p localization mutants. Aspects       hauser et al., 1995) candidate partner binding model inferred cdc12p overexpression mu-         SH3-binding domain cdc12p. cdc8p homologue tro- <br /> tant data. Alternate models discussed text.                pomyosin, small actin-binding protein (Balasubramanian <br />                                                                       et al., 1992). Studies antiactin drug Latruculin                                                                       F-actin necessary cdc12p localization <br />                                                                       (Engqvist, , F. Chang, unpublished observations). <br /> ring assembly generally did include F-actin.          proposed bud site determination S. cerevisiae <br /> preceded ring assembly appeared mark future                (Yang et al., 1997), F-actin organized actin-binding pro- <br /> division site.                                                        teins required cortical scaffold binds <br />    similar cdc12p spot observed cdc4                cdc12p cortex. cortex, cdc12p profilin <br /> mid1 mutant cells. cdc4 mutant cells exhibited single spot          organize ring components, including F-actin, instead ring mitosis. mid1 mutants exhib-           site. demonstrated Thyone sperm (Tilney et al., <br /> ited single strand cdc12p appeared emanate               1983), F-actin used nucleation seed cdc12p spot nucleus seemingly random               cdc12p profilin-dependent actin assembly. Finally, <br /> direction. cases, cdc12p overexpressed,           cdc12p ring components required  aspect ring assembly defective.                              gether ring assembly, loss essential <br />    Based observations, propose specula-               components cause delocalization compo- <br /> tive model ring positioning assembly fission yeast         nents. <br /> (Fig. 10). interphase early mitosis, cell                 contrast, cdc4 mid1 necessary lo- <br /> middle marked single spot cdc12p (Fig. 7,            calize cdc12p se, required later B). mitosis, ring form spot sin-          step ring formation. cdc4p putative myosin light chain, <br /> gle strand cdc12p originates spot &#226;&#8364;&#339;mi-           suggesting myosin-related protein important grates&#226;&#8364;? circumference cell (Fig. 10 D).             ring formation (McCollum et al., 1995). mid1 gene prod- strand migration model consistent patterns            uct necessary spatial regulation  <br /> mid1 mutants, described  cells overex-                pects ring formation: guiding cdc12p strand pressing cdc12p, incomplete rings discon-           circumference, placing initial spatial cue tinuous spot observed (Fig. 5 C).           middle cell. mid1 (dmf1) encodes novel protein model contrast alternate novo model             pleckstrin homology domain, PEST sequences, ring arise novo circumferential              nuclear localization signal (Sohrmann et al., 1996).  <br /> structure medial axis. placement          triguingly, mid1 gene product located nucleus <br /> ring fission yeast determined signal              interphase medial ring early mitosis, sug- interphase nucleus (Chang Nurse, 1996), spot              gesting link positions nucleus ring. represent site nucleus signals cortex.           analysis shows  cases, cdc12p <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 137, 1997                         180 <br />  appears associated nucleus earliest                       References <br /> stages ring formation mid1-366 mutant.                                Balasubramanian, M.K., D.M. Helfman, S.M. Hemmingsen. 1992. new <br />                                                                                    tropomyosin essential cytokinesis fission yeast S. pombe. Nature <br />                                                                                    (Lond.). 360:84&#226;&#8364;&#8220;87. <br />                                                                                 Balasubramanian, M.K., B.R. Hirani, J.D. Burke, K.L. Gould. 1994. Role cdc12-like Proteins Cytokinesis                                         Schizosaccharomyces pombe cdc31 gene encodes profilin essential cy- Spatial Organization                                                           tokinesis. J. Cell Biol. 125:1289&#226;&#8364;&#8220;1301. <br />                                                                                 Barbet, N., W.J. Muriel, M. Carr. 1992. Versatile shuttle vectors ge- <br /> cdc12p member family proteins involved cy-                           nomic libraries use Schizosaccharomyces pombe. Gene (Amst.). 114: <br /> tokinesis actin-related processes cell fu-                        59&#226;&#8364;&#8220;66. <br />                                                                                 Bobola, N., R.P. Jansen, T.H. Shin, K. Nasmyth. 1996. Asymmetric accu- <br /> sion. overall amino acid sequence homology                            mulation Ash1p postanaphase nuclei depends myosin restricts proteins low (,30%), similarity                          yeast mating-type switching mother cells. Cell. 84:699&#226;&#8364;&#8220;709. <br /> domain structure, conservation certain small domains                         Castrillon, D.H., S  Wasserman. 1994. diaphanous required cytoki- <br />                                                                                    nesis Drosophila shares domains similarity products formin homology domains (Castrillon                                limb deformity gene. Development (Camb.). 120:3367&#226;&#8364;&#8220;3377. <br /> Wasserman, 1994), roles related cellular pro-                      Chakraborty, T., F. Ebel, E. Domann, K. Niebuhr, B. Gerstel, S. Pistor, C.J. <br />                                                                                    Temm-Grove, B.M. Jockusch, M. Reinhard, U. Walter. 1995. focal <br /> cesses suggest function proteins                          adhesion factor directly linking intracellularly motile Listeria monocytogenes <br /> served. proteins required cytokinesis:                        Listeria ivanovii actin-based cytoskeleton mammalian cells. <br /> like cdc12, diaphanous Drosophila necessary cy-                          EMBO (Eur. Mol. Biol. Organ.) J. 14:1314&#226;&#8364;&#8220;1321. <br />                                                                                 Chang, F., P. Nurse. 1996. fission yeast fission middle. Cell. 84: <br /> tokinesis (Castrillon Wasserman, 1994), sepA/figA                          191&#226;&#8364;&#8220;194. Aspergillus required septation (Harris et al., 1997).                 Chang, F.,  Woollard, P. Nurse. 1996. Identification characterization <br /> BNI1p S. cerevisiae role placement di-                      fission yeast mutants defective actin ring assembly placement. J. <br />                                                                                    Cell Sci. 109:131&#226;&#8364;&#8220;142. <br /> vision site (bud site placement bipolar pattern)                         Conrad, G.W., T.E. Schroeder, editors. 1990. Cytokinesis: Mechanisms (Zahner et al., 1996), displays genetic interaction                         Furrow Formation Cell Division. New York Academy Sciences, <br />                                                                                    New York. <br /> septin, located division ring site                  Correa-Bordes, J., P. Nurse. 1995. p25rum1 orders S phase mitosis polarized growth bud tip (Fares, H., M. Longtine,                           acting inhibitor p34cdc2 mitotic kinase. Cell. 83:1001&#226;&#8364;&#8220;1009. J. Pringle, personal communication).                                        Elledge, S., J. Mulligan, S. Ramer, M. Spottswood, R. Davis. 1991. lYES:                                                                                    multifunctional cDNA expression vector isolation genes com- <br />    members cdc12p family related roles                           plementation yeast Escherichia coli mutations. Proc. Natl. Acad. Sci. localizing proteins determinants certain                         USA. 88:1731&#226;&#8364;&#8220;1735. <br /> gions cell cortex. cappuccino required cy-                   Emmons, S., H. Phan, J. Calley, W. Chen, B. James, L. Manseau. 1995. cap- <br />                                                                                    puccino, Drosophila maternal effect gene required polarity egg <br /> tokinesis establishment polarity Dro-                            embryo, related vertebrate limb deformity locus. Genes <span id='am-19' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Dev. <br /> sophila oogenesis potentially involved signaling                     9:2482&#226;&#8364;&#8220;2494. <br />                                                                                 Erdelyi, M., M. Michon,  Guichet, J.B. Glotzer,  Ephrussi. 1995.  <br /> events nucleus cortex establish po-                           quirement Drosophila cytoplasmic tropomyosin oskar mRNA local- <br /> larity axes (Emmons et al., 1995; Roth et al., 1995). cdc12,                       ization. Nature (Lond.). 377:524&#226;&#8364;&#8220;527. <br /> cdc3, cdc8 play analogous roles cappuccino,                          Evangelista, M., K. Blundell, M.S. Longtine, C.J. Chow, N. Adame, J.R. Prin- <br />                                                                                    gle, M. Peter, C. Boone. 1997. Bni1p, yeast form linking Cdc42p profilin (chickadee), tropomyosin II,                             actin cytoskeleton polarized morphogenesis. Science (Wash. DC). <br /> quired localization posterior determinants                        press. <br /> Drosophila oocyte (Emmons et al., 1995; Erdelyi et al., 1995;                   Fankhauser, C., V. Simanis. 1994. Cold fission: splitting pombe cell                                                                                    room temperature. Trends Cell Biol. 4:96&#226;&#8364;&#8220;101. <br /> Manseau et al., 1996). S. cerevisiae, BNI1                        Fankhauser, C.,  Reymond, L. Cerutti, S. Utzig, K. Hofmann, V. Sima- <br /> quired proper localization determinant (Ash1p)                        nis. 1995. S. pombe cdc15 gene key element reorganization                                                                                    F-actin mitosis. Cell. 82:435&#226;&#8364;&#8220;444. daughter bud (Bobola et al., 1996). cdc12p                       Fishkind, D.J., Y.L. Wang. 1995. New horizons cytokinesis. Curr Opin. <br /> similar S. pombe fus1p, required cell fusion                       Cell Biol. 7:23&#226;&#8364;&#8220;31. nuclear migration mating localized patches                    Gertler, F.B., K. Niebulr, M. Reinhard, J. Wehland, P. Soriano. 1996. <br />                                                                                    Mena, relative VAST Drosophila Enabled, implicated  projection tip (Petersen et al., 1995). fus1 nec-                    trol microfilament dynamics. Cell. 87:227&#226;&#8364;&#8220;239. <br /> essary localize fusiogenic proteins projection tip dur-               Harlow, E., D. Lane. 1988. Antibodies: Laboratory Manual. Cold Spring <br /> ing mating process. Candidate vertebrate homologues                            Harbor Laboratory, Cold Spring Harbor, NY. 726 pp. <br />                                                                                 Harris, S.D., L. Hamer, K.E. Sharpless, J.E. Hamer. 1997. Aspergillus cdc12p formins, implicated                           nidulans scpA gene encodes FH1/2 protein involved cytokinesis limb morphogenesis (Woychik et al., 1990).                             maintenance cellular polarity. EMBO (Eur. Mol. Biol. Organ.) J. press. <br />                                                                                 Hirano, T., Y. Hiraoka, M. Yanagida. 1988. temperature-sensitive muta- <br /> cellular role formins determined. Fur-                         tion Schizosaccharomyces pombe gene nuc21 encodes nuclear <br /> ther study cdc12p homologues help eluci-                         scaffold-like protein blocks spindle elongation mitotic anaphase. J. Cell <br /> <span id='am-18' about='oboInOwl:date' typeof='owl:Thing'>date</span> signals mark sites cell division polarity                      Biol. 106:1171&#226;&#8364;&#8220;1183. <br />                                                                                 Hoheisel, J.D., E. Maier, R. Mott, L. McCarthy, V. Grigoriev, L.C. Schalk- eukaryotes.                                                                   wyk, D. Nizetic, F. Francis, H. Lehrach. 1993. High resolution cosmid <br />                                                                                    P1 maps spanning 14 Mb genome fission yeast Schizosaccha- grateful past present members Nurse laboratory            romyces pombe. Cell. 73:109&#226;&#8364;&#8220;120. <br /> Drubin laboratory help, advice, reagents. thank Bruce Goode          Horvitz, H.R.,  Herskowitz. 1992. Mechanisms asymmetric cell divi- <br />                                                                                    sion: Bs Bs, question. Cell. 68:237&#226;&#8364;&#8220;255. advice binding studies; Jaime Correa-Bordes gift cell cycle ex-   Jochova, J.,  Rupes, E. Streiblova. 1991. F-actin contractile rings proto- <br /> tracts; Elmar Maier assistance cosmid mapping; Charles Boone,           plasts yeast Schizosaccharomyces. Cell Biol. Int. Rep. 15:607&#226;&#8364;&#8220;610. <br /> Steve Wasserman, Lynn Manseau, John Pringle, Hana Fares, Steven Har-            Kanbe, T.,  Kobayashi, K. Tanaka. 1989. Dynamics cytoplasmic  <br /> ris, Takashi Toda communication results publication.             ganelles cell cycle fission yeast Schizosaccharomyces pombe: <br />                                                                                    dimensional reconstruction serial sections. J. Cell Sci. 94:647&#226;&#8364;&#8220;656. thank T. Toda, K. Ayscough, J. Thorner, K. Kozminski,            Kelly, T.J., G.S. Martin, S.L. Forsburg, R.J. Stephen,  Russo, P. Nurse. <br /> viewers helpful comments manuscript.                                    1993. fission yeast cdc181 gene product couples S-phase START     F. Chang supported Helen Hay Whitney Foundation               mitosis. Cell. 74:371&#226;&#8364;&#8220;382. <br /> University California CRCC grant.                                            Kohno, H., K. Tanaka,  Mino, M. Umikawa, H. Imamura, T. Fujiwara, Y. <br />                                                                                    Fujita, K. Hotta, H. Qadota, Y. Watanabe, et al. 1996. Bni1p implicated                                                                                    cytoskeletal control putative target Rho1p small <span id='am-10' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GTP</span> binding protein <br /> Received publication 30 April 1996 revised form 14 February             Saccharomyces cerevisiae. EMBO (Eur. Mol. Biol. Organ.) J. 15:6060&#226;&#8364;&#8220; <br /> 1997.              <br /> </body></html>