<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                         Pivotal role Francisella tularensis heat-shock <br />                                         sigma factor RpoH <br />  <br />                                         Francisella tularensis highly infectious pathogen infects animals humans cause                                         disease tularemia. primary targets bacterium macrophages, replicates                                         cytoplasm escaping initial phagosomal compartment. ability replicate                                         macrophages relies tightly regulated expression series genes.                                         commonly used means coordinating regulation multiple genes bacteria consists                                         association dedicated alternative sigma factors core RNA polymerase (RNAP).                                         silico analysis F. tularensis LVS genome led identify, addition genes encoding <br />                                         RNAP core (comprising a1, a2, b, b9 v subunits), gene (designated rpoD) encoding <br />                                         major sigma factor s70, unique gene (FTL_0851) encoding putative alternative sigma <br />                                         factor homologue s32 heat-shock family (designated rpoH).  F. tularensis represents <br />                                         minority bacterial species possess alternative sigma factor                                         addition main factor s70. present work, FTL_0851 encodes genuine s32 <br />                                         factor. Transcriptomic analyses F. tularensis LVS heat-stress response allowed                                         identification series orthologues known heat-shock genes (including Hsp40, <br />                                         GroEL, GroES, DnaK, DnaJ, GrpE, ClpB ClpP) number genes implicated Francisella <br />                                         virulence. bioinformatic analysis used identify genes preceded putative s32-binding <br />                 site, revealing similarities differences RpoH-mediated gene expression                 Escherichia coli. results suggest RpoH essential protein F. tularensis, positively <br />                   regulates subset genes involved heat-shock response. <br />  <br />  <br />  <br /> INTRODUCTION                                                                arthropod bites, contaminated animal <br />                                                                             carcasses, inhalation bacteria. high <br /> Francisella tularensis Gram-negative, facultative <br />                                                                             infectivity severity disease, F. tularensis intracellular pathogen causes tularemia humans <br />                                                                             classified Centers Disease Control large number animal species. Human infections <br />                                                                             potential biological weapon  reviews, McLendon et occur ingestion contaminated water food, <br />                                                                             al., 2006; Sjostedt, 2006). pathogenicity F. tularensis <br />                                                                             largely dependent ability replicate inside host <br /> Abbreviations: FDR, false discovery rate; FPI, Francisella pathogenicity <br /> island, MFS, major facilitator superfamily; PFM, position frequency <br />                                                                             cells, particular macrophages vivo. matrix; qRT-PCR, quantitative RT-PCR; RNAP, RNA polymerase; TIGR,           factors important virulence identified, Institute Genomic Research.                                         particular genes located Francisella pathogenicity microarray data discussed paper deposited          island (FPI)  Bruin et al., 2007; Golovliov et al., 2003; <br /> ArrayExpress accession numbers E-MEXP-2048 E-                 Lai et al., 2004; Ludu et al., 2008; Nano et al., 2004; Nano <span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; <br /> MEXP-2037.                                                                  Schmerk, 2007; Santic et al., 2007; Tempel et al., 2006; supplementary tables, listing primers used study, genes        Weiss et al., 2007), molecular mechanisms <br /> significantly decreased expression temperature upshift, genes      underlying disease fully understood. used BioProspector analysis, putative binding sites BioProspector used building matrices, genes           Regulation gene expression bacteria occurs primarily <br /> potential s32-binding sequence promoter region, available      transcriptional level. association dedicated online version paper.                                      alternative sigma factors core RNA polymerase <br />  <br /> 2560                                                                                           029058 G 2009 SGM Printed Great Britain <br />                                                                                            Francisella tularensis heat-shock response <br />  <br />  <br /> (RNAP) provides simple efficient way bacteria       RpoH identified genes preceded putative <br /> rapidly adapt various environmental changes.              RpoH-binding site. results demonstrate similarities <br /> RNAP holoenzyme contains subunits core                heat-stress response F. tularensis molecule (comprising subunits, b, b9 v             studied response E. coli, illuminate <br /> subunits) sigma factor enabling holoenzyme          differences organisms. number specifically recognize promoter elements initiate            heat-induced genes previously demonstrated transcription sites. regulon single sigma      importance ability F. tularensis survive <br /> factor comprise hundreds genes. sigma factors        multiply inside host cells, suggesting stress induced response environmental changes         responses significance virulence Francisella. controlled anti-sigma factors  reviews Gourse et al., 2006; Gruber &amp; Gross, 2003). number sigma factors encoded different bacterial species varies      METHODS <br /> considerably. example, seven sigma factors                                                                  Bacterial strains plasmids. Bacterial strains plasmids used <br /> identified Escherichia coli <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, Streptomyces <br />                                                                  study listed Table 1 oligonucleotide sequences coelicolor thought encode 61. bacteria       Supplementary Table S1. F. tularensis LVS grown pre contain single sigma factor, majority express     chocolate agar enriched PolyVitex (bioMe&#194;&#180;rieux), chocolate agar  number generally relates size     plates prepared GC medium base, IsoVitalex vitamins genome environmental variation allows               haemoglobin (BD Biosciences), Schaedler + <span id='am-2' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-3' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>vitamin</span> K3 broth <br /> growth organism (Kill et al., 2005; Rodrigue et al.,      (Shaedler-K3, bioMe&#194;&#180;rieux) 37 uC room temperature. E. coli <br /> 2006).                                                           strains grown Luria&#226;&#8364;&#8220;Bertani (LB) plates LB broth <br />                                                                  (Difco) 37 uC room temperature aerobically. ubiquitous mode bacterial response                                                                  F. tularensis, antibiotics used 5 mg ml21 (kanamycin) environmental changes involves function               1.75 mg ml21 (chloramphenicol). E. coli, antibiotics used component signal transduction systems (TCS). TCS            following concentrations: 25 mg ml21 (kanamycin), 2.5 mg ml21 <br /> consists membrane-bound sensor monitors                (chloramphenicol) 50 mg ml21 (ampicillin). <br /> environmental variations dedicated cytoplasmic <br />                                                                  plasmid used overexpression rpoH, pFNLTP6gro-0851, response regulator mediates adaptive response,           constructed amplifying 957 bp fragment (corresponding e. transcriptional regulation gene expression.            sequence 78 bp upstream rpoH gene entire gene) using <br /> Strikingly, Francisella subspecies possess single       primers 0851_F 0851_R (Supplementary Table S1), followed complete TCS pair (Mohapatra et al., 2007 references         digestion NdeI NotI cloning plasmid pFNLTP6gro , analyses F. tularensis genomes         (Maier et al., 2004; Meibom et al., 2009) downstream groE <br /> revealed presence single gene encoding           operon promoter F. tularensis. pFNLTP6gro-0851 introduced <br />                                                                  LVS strain electroporation. <br /> protein homologous alternative sigma factor, placing F. tularensis relatively small group       plasmid pBADrpoH constructed amplifying LVS rpoH <br /> bacterial species express single alternative   gene using primers 0851_F2 0851_R2, followed digestion sigma factor. absence complete TCS            EcoRI cloning plasmid pBAD downstream arabinose- <br />                                                                  inducible pBAD promoter. rpoH gene verified presence gene encoding alternative sigma <br />                                                                  right orientation PCR using primers 59pBAD 0851_R2. <br /> factor reflect fact Francisella genome relatively small thought deteriorating, based    Functional complementation. E. coli K-12 strain MG1655 high number pseudogenes incomplete                 transformed pRB11 (Yura et al., 2007) pBADrpoH. <br /> biosynthetic pathways (Larsson et al., 2005). absence        Phage P1 used transduce DrpoH : : Kan zhf50 : : Tn10 multiple alternative sigma factors TCS suggests F.    mutation strain CAG48315 (Zhou et al., 1988) MG1655/ <br /> tularensis limited ability respond proliferate   pRB11 MG1655/pBADrpoH. transduction lysate <br />                                                                  preparation, overnight culture CAG48315 diluted 100-fold diverse environmental conditions. <br />                                                                  fresh LB supplemented 5 mM CaCl2 0.2 % glucose, Given small number putative regulatory proteins           grown 37 uC shaking 1 h adding 200 ml P1 phage <br /> encoded F. tularensis, interested examining        lysate 5 ml culture. culture lysed completely, <br />                                                                  drops chloroform added transduction lysate function alternative sigma factor                                                                  kept 4 uC. transduction, recipient strains (MG1655/ <br /> help organism survival stress response.    pRB11 MG1655/pBADrpoH) grown overnight LB present study, alternative sigma       supplemented 50 mg ampicillin ml21 0.2 % arabinose. factor encoded F. tularensis genome, RpoH,           cells harvested resuspended LB supplemented genuine s32 unit, data suggest rpoH           50 mg ampicillin ml21, 0.2 % arabinose, 100 mM MgSO4 5 mM <br /> essential gene F. tularensis. gain insight    CaCl2. 100 ml volume bacterial cells 100 ml transduction RpoH-dependent mechanisms gene expression F.           lysate mixed incubated 37 uC shaking 30 min. <br />                                                                  addition 200 ml 1 M sodium citrate (pH 5.5), cells tularensis performed transcriptomic analysis                                                                  incubated 37 uC shaking 1 ml LB containing 50 mg <br /> heat-stress response F. tularensis LVS evaluated      ampicillin ml21 0.2 % arabinose 1 h, plated LB agar <br /> role RpoH regulation bacterial genome        supplemented 50 mg ampicillin ml21, 25 mg kanamycin ml21 RpoH-overproducing strain. addition, bioinfor-           0.2 % arabinose. Colonies appearing overnight incubation <br /> matic analysis determined consensus binding sequence           verified DrpoH : : Kan zhf50 : : Tn10 mutation  <br /> http://mic.sgmjournals.org                                                                                                       2561 <br />  N. Grall  <br />  <br /> Table 1. Bacterial strains plasmids used study <br />  <br />  Strain plasmid                                             Description genotype                                        Source <br />  <br />  E. coli strains <br />  DH5a                            F2w80lacZ DM15 recA1 endA1 hsdR17 supE44 thi-1 gyrA96 relA1                        Laboratory strain collection <br />                                   (lacZYA-argF) U169 <br />  CAG48315                        l2 phtpg&#226;&#8364;&#8220;lacZ DrpoH : : Kan zhf50 : : Tn10, pRB11                                  Zhou et al. (1988) <br />  MG1655                          K-12                                                                               Arnaud Gutierrez <br />  MG1655/pRB11                    MG1655 containing plasmid pRB11 expressing E. coli rpoH                            study <br />  MG1655/pBADrpoH                 MG1655 containing plasmid pBAD expressing F. tularensis rpoH                       study <br />  MG1655DrpoH/pRB11               MG1655/pRB11 transduced P1 lysate CAG48315                               study <br />  MG1655DrpoH/pBADrpoH            MG1655/pBADrpoH transduced P1 lysate CAG48315                            study <br />  F. tularensis strains <br />  LVS                             F. tularensis subspecies holarctica live vaccine strain                            Laboratory strain collection <br />  LVS/pFNLTP6gro-0851             LVS plasmid pFNLTP6gro_0851 expressing rpoH groES promoter           study <br />  LVS/pFNLTP6gro                  LVS plasmid pFNLTP6gro insert                                         study <br />  Plasmids <br />  pRB11                           pBR322 derivative E. coli rpoH gene downstream IPTG-inducible              Yura et al. (2007) <br />                                   tac promoter <br />  pBAD                            araBAD promoter, araC, pBR322 origin, AmpR                                         Invitrogen <br />  pBADrpoH                        F. tularensis rpoH gene cloned downstream arabinose-inducible promoter          study <br />                                   pBAD <br />  pFNLTP6gro                      E. coli&#226;&#8364;&#8220;F. tularensis shuttle vector groE operon promoter, KmR AmpR,          Meibom et al. (2009); Maier <br />                                   constructed pFNLTP6gro-gfp deleting gfp gene                               et al. (2006) <br />  pFNLTP6gro-0851                 E. coli&#226;&#8364;&#8220;F. tularensis shuttle vector F. tularensis rpoH downstream         study <br />                                   groE operon promoter, KmR AmpR <br />  <br />  <br />  <br /> PCR using primers KanaRtn903_F KanaRtn903_R.                   hybridization buffer (40 % formamide, 56 saline sodium citrate <br /> strain obtained transduction, MG1655DrpoH/pRB11,              (SSC), 0.1 % SDS, 0.6 mg salmon sperm DNA ml21) 42 uC 18 h <br /> grown LB agar supplemented 50 mg ampicillin ml21                 hybridization chamber (Corning). <br /> 25 mg kanamycin ml21 room temperature, 30 37 uC.                                                                              F. tularensis microarrays, provided Pathogen Functional <br /> second strain, MG1655DrpoH/pBADrpoH, grown LB agar <br />                                                                              Genomics Resource Center (PFGRC; Rockville, MD, USA), contain <br /> supplemented 50 mg ampicillin ml21 25 mg kanamy- <br />                                                                              2331 oligonucleotides (70-mer) replicates oligonucleo- <br /> cin ml21 presence absence 0.2 % arabinose room <br />                                                                              tide, representing ORFs F. tularensis Schu S4 F. <br /> temperature, 30 37 uC. <br />                                                                              tularensis LVS. <br /> RNA extraction. RNA extracted exponential-phase bacteria            Microarrays scanned Genepix 4000B scanner (Molecular TRIzol reagent (Invitrogen). extraction chloroform,          Devices). determine signal fluorescence intensities spot, aqueous phase purified using RNeasy Clean protocol            TIFF images analysed using Spotfinder Midas software <br /> (Qiagen). Contaminating DNA removed RNA preparations                (TIGR). Statistical analyses performed using Significance RNA-free DNase (Ambion).                                              Analysis Microarrays (SAM) software (Tusher et al., 2001).                                                                              number genes significantly modified expression defined <br /> Microarray experiments data analyses. prepare RNA                 performing class analysis using false discovery rate (FDR) <br /> microarray analyses, F. tularensis LVS, F. tularensis LVS+                   0 %. <br /> pFNLTP6gro, F. tularensis LVS+pFNLTP6gro-0851 grown 37 uC shaking OD600 ~0.3&#226;&#8364;&#8220;0.4. rpoH overexpression             RNA extractions independent experiments experiments, bacterial cells harvested RNA                 condition sample set hybridized extracted. heat-shock-response experiments, culture F.            microarrays. <br /> tularensis LVS split  culture placed 37 uC shaking, incubated 44 uC            Real-time quantitative RT-PCR (qRT-PCR). validate shaking. 30 min incubation, bacterial cells harvested         microarray results, ORFs selected qRT-PCR analysis <br /> RNA extracted.                                                           RNA samples used microarray hybridizations. <br />                                                                              heat-shock time-course experiment, RNA samples Reverse transcription cDNA fluorescent labelling performed          isolated temperature shift 37 44 uC 10, 20, sample 3 mg RNA according protocol provided            30, 45 60 min incubation 44 uC. analyses, 1 mg RNA Institute Genomic Research (TIGR; http://pfgrc.jcvi.org).         reverse-transcribed using random hexamers Superscript II <br /> Briefly, RNA reverse-transcribed using Superscript II reverse        reverse transcriptase (Invitrogen) according protocol provided <br /> transcriptase presence amino-allyl-<span id='am-30' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="rdfs:label" content="dUTP" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span>dUTP</span>, dNTP                  manufacturer. Real-time PCR performed gene- <br /> random primers, followed coupling cyanine dye (Cy3                  specific primers using ABI PRISM 7700 SYBR green PCR <br /> Cy5). Microarray hybridization post-hybridization washings          master mix (Applied Biosystems). calculate gene- <br /> performed according TIGR protocol. Briefly, pre-                specific transcript, standard curve plotted primer set <br /> hybridization microarray slides, hybridization performed           using series diluted genomic DNA LVS. compare  <br /> 2562                                                                                                                             Microbiology 155 <br />                                                                                                    Francisella tularensis heat-shock response <br />  <br />  <br /> transcript amounts different conditions, amounts        Functional complementation E. coli rpoH <br /> gene transcript normalized DNA helicase (FTL_1656), gene shown change little expression growth            tested ability LVS rpoH gene (Brotcke et al., 2006). expression gene determined          functionally complement temperature-sensitive growth replicates single qRT-PCR experiment.                       E. coli DrpoH mutant lacking s32, grow <br />                                                                             20 uC (Zhou et al., 1988).  Bioinformatics analyses. RpoH-binding site consensus <br /> sequence identified described previously (Slamti et al., 2007) <br />                                                                             introduced DrpoH mutation strain CAG48315 <br /> using BioProspector (Liu et al., 2001) search conserved motifs       (Zhou et al., 1988; Table 1) E. coli strain MG1655 structure similar RpoH-binding sites E. coli      transduction. recipient strain contains plasmid Vibrio cholerae. search parameters input sequences         encoding F. tularensis rpoH gene (pBADrpoH) adjusted motif identified corresponded previously          control arabinose-inducible promoter. predicted RpoH-binding sites upstream F. tularensis grpE             resulting strain did grow agar plates groES (Meibom et al., 2008) (Supplementary Tables S3 S4). sequences 200 bp upstream 400 annotated F. tularensis <br />                                                                             supplement temperature (room temperature, 30 ORFs used background model. final search                    37 uC; Fig. 1, middle row).  F. tularensis rpoH <br /> parameters follows: motif, 2 blocks; width motif       gene promoted growth mutant strain block, 7; width second motif block, 7; minimum gap           arabinose added medium (Fig. 1, row), blocks, 13; maximum gap blocks, 16. Identification       showing F. tularensis rpoH functionally <br /> putative RpoH-binding sites accomplished steps.            complement lack s32 E. coli. control, position frequency matrices (PFMs) constructed. composed PFMs blocks 1 2 identified                                                                             transduced DrpoH mutation recipient strain <br /> BioProspector separated 13&#226;&#8364;&#8220;16 columns, respectively,        containing plasmid encoding E. coli rpoH gene <br /> frequencies nucleotides set identical. Second,   (pRB11; Yura et al., 2007) (Fig. 1, row), program PATSER (van Helden, 2003) used search 200 bp             strain exhibited growth pattern upstream F. tularensis genes sequences corresponding          MG1655DrpoH/pBADrpoH strain grown presence matrices. minimum PATSER score threshold         arabinose. results F. tularensis rpoH <br /> set 9. cases putative binding sites block <br /> 1 block 2 sequences different gap lengths, lower-scoring site <br />                                                                             encodes genuine s32 unit RNAP. discarded. <br />                                                                             F. tularensis transcriptomic response heat <br />                                                                             stress <br /> RESULTS <br />                                                                             studied gene expression changes F. tularensis <br /> Identification functional rpoH gene                                 LVS response heat shock.  extracted RNA <br /> F. tularensis LVS                                                           bacterial cultures grown Schaedler medium <br />                                                                             (supplemented vitamin K3) temperature silico analysis LVS genome allowed                      upshift 30 min 44 uC. cDNA synthesis identify genes involved synthesis core                    labelling, control  culture kept 37 uC) RNAP. reported Charity et al. (2007), F.                          heat-shocked samples paired hybridized tularensis genome contains genes, designated rpoA1                      microarrays. Statistical analysis using SAM software <br /> (FTL_0261) rpoA2 (FTL_0616), encode                           (Tusher et al., 2001) indicated 181 genes exhibited <br /> different subunits RNAP differ                   significant (FDR 0 %) differential expression twofold regions predicted critical dimer                      greater level response temperature upshift formation, promoter recognition, activator inter-                       37 44 uC (Table 2 Supplementary Table S2). action. FTL_1744 FTL_1743 (rpoB rpoC) encode                        wide distribution putative functional roles attributed b b9 subunits, respectively, FTL_1533 (rpoZ)                    differentially expressed genes indicates extent v subunit.                                                              molecular response enables F. tularensis LVS survive major sigma factor (s70) encoded FTL_1050                         eventually adapt thermal stress. transcripts (rpoD). Strikingly, identified gene, FTL_0851,                  112 genes abundant temperature <br /> encoding putative alternative sigma factor sharing                        upshift (Table 2).  15 genes annotated <br /> significant sequence similarity members                   functions protein fate. known heat-shock proteins, <br /> family s32 heat-shock factors  designated rpoH).                   Hsp40, GroEL, GroES, DnaK, DnaJ, GrpE, ClpB, rpoH sequence LVS encodes protein 292 aa,                  ClpX, ClpP HtpG, form major group. <br /> 53.4 % <span id='am-43' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-47' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> identity s32 E. coli K-12,                 major sigma factor RpoD (s70) highly conserved F. tularensis subspecies.                        upregulated, fold-induction 4.6. contrast, rpoH <br /> Multiple sequence alignments s32 family members                     did appear upregulated temperature <br /> led identification highly conserved stretch             upshift. 40 % upregulated genes genes <br /> amino acids Q(R/K)(K/R)LFFNLR designated                        encoding proteins unknown function pseudogenes. <br /> RpoH box (Nakahigashi et al., 1995). note                Transposases composed 21 % upregulated genes, box perfectly conserved F. tularensis s32 sequences,              reflect cross-hybridizations microarrays <br /> strongly suggesting encodes functional s32 factor.                transposases highly homologous. Interestingly, <br />  <br /> http://mic.sgmjournals.org                                                                                                            2563 <br />  N. Grall  <br />  <br />  <br />  <br />        Fig. 1. rpoH gene F. tularensis functionally complements E. coli DrpoH mutant. E. coli stain MG1655DrpoH <br />        harbouring plasmid pBADrpoH (encoding rpoH F. tularensis) pRB11 (encoding rpoH E. coli) streaked LB <br />        agar containing ampicillin 0.2 % arabinose, indicated (+ara), incubated 24 h 30 37 6C, 48 h        room temperature (RT). <br />  <br />  <br />  <br />  genes F. tularensis FPI                     qRT-PCR analyses heat-shock-induced genes <br /> downregulated: FTL_0115/1161, FTL_0116/_1162                         (groEL, dnaK, grpE hsp40) period 60 min <br /> FTL_0124/_1170, encoding proteins unknown function,                   incubation 44 uC (Fig. 2b). genes pdpB1 iglB1 (intracellular growth locus                          observed increased expression times (10&#226;&#8364;&#8220;60 min) <br /> B)(Supplementary Table S2).                        compared heat shock.  transcript <br /> regulated genes annotated functions metabolism,                   levels appeared elevated later time points <br /> biosynthesis cofactors ribonucleotide biosynthesis.             (45 60 min). upregulated genes, genes encoding proteins unknown function pseudogenes represented signifi- <br />                                                                          Role rpoH F. tularensis transcriptional portion downregulated genes. Furthermore, <br />                                                                          regulation genes encoding proteins transport functions downregulated heat shock.                                     spite repeated efforts, unable                                                                          inactivate rpoH gene LVS. tried inactivate verify accuracy microarray data,                                                                          rpoH insertional mutagenesis allelic <br /> performed qRT-PCR analyses transcripts chosen <br />                                                                          exchange temperatures ranging 20 37 uC, genes. experiments, used RNA <br />                                                                          unsuccessful. likely  formally <br /> sample sets used microarray experiments. <br />                                                                          proven) rpoH essential F. tularensis, seen Fig. 2 , relative levels rpoH, dnaK, groEL <br />                                                                          thought bacteria, V. cholerae hsp40 transcripts fully supported microarray data, <br />                                                                          Neisseria gonorrhoeae (Laskos et al., 2004; Slamti et al., dnaK, groEL hsp40 expressed higher levels                                                                          2007). <br /> 44 uC 37 uC expression rpoH changed temperature upshift. investigate                   overcome lack rpoH mutant, evaluated heat-shock response temporal manner, performed                   effect overproduction RpoH LVS transcrip- <br />  <br /> 2564                                                                                                                      Microbiology 155 <br />                                                                                                   Francisella tularensis heat-shock response <br />  <br />  <br /> Table 2. Genes significantly increased expression  twofold) temperature upshift <br />  <br />  Locus*                        Gene product                          Gene               Cellular role categoryD                     Fold <br />                                                                                                                                   changed <br />  <br />  FTL_0034        Hypothetical protein                                       Hypothetical proteins: conserved                       2.02 <br />  FTL_0088        Acetyltransferase protein                                  Unknown function                                       2.38 <br />  FTL_0093        Chitinase                                                  Metabolism                                            10.24 <br />  FTL_0094        ClpB protein                                        clpB   Degradation proteins, peptides glycopeptides    5.51 <br />  FTL_0144        Hypothetical protein                                       Unknown function: general                              2.77 <br />  FTL_0163        MFS transporter                                            Transport binding proteins                         4.31 <br />  FTL_0180        Acyltransferase                                     htrB   Metabolism                                             2.25 <br />  FTL_0186        MFS transport protein, fragment                            Transport binding proteins                         4.01 <br />  FTL_0201        Hypothetical protein                                       Unknown function                                       2.15 <br />  FTL_0202        Conserved hypothetical membrane protein                    Cell envelope:                                   3.29 <br />  FTL_0203        Hypothetical membrane protein                              Unknown function: general                              2.00 <br />  FTL_0219        Hypothetical protein                                       Unknown function: general                              2.86 <br />  FTL_0267        Heat-shock protein 90                               htpG   Protein folding stabilization                      3.01 <br />  FTL_0281        Heat-shock protein, hsp40                                  Protein folding stabilization                      7.13 <br />  FTL_0359        Type IV pili fibre building block protein                  Unknown function                                       2.55 <br />  FTL_0365        2                                                          Pseudogene                                             2.92 <br />  FTL_0390        Type IV pili fibre building block protein                  Unknown function: general                              1.99 <br />  FTL_0645        Hypothetical lipoprotein                                   Cell envelope                                          3.20 <br />  FTL_0662        Prophage repressor protein                                 Mobile extrachromosomal element functions          3.31 <br />  FTL_0663        Hypothetical protein                                       Hypothetical proteins: conserved                       4.02 <br />  FTL_0758        2                                                          Pseudogene                                             2.24 <br />  FTL_0826        2                                                          Pseudogene                                             2.09 <br />  FTL_0844        LysR transcriptional regulator family protein              Regulatory functions                                   2.35 <br />  FTL_0867        Hypothetical protein                                       Hypothetical proteins: conserved                       2.78 <br />  FTL_0891        Trigger factor                                      tig    Protein folding stabilization                      4.38 <br />  FTL_0892        <span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-dependent Clp protease subunit P                clpP   Degradation proteins, peptides glycopeptides    3.70 <br />  FTL_0893        ATP-dependent protease ATP-binding subunit          clpX   Degradation proteins, peptides glycopeptides    2.87 <br />  FTL_0894        DNA-binding, ATP-dependent protease La              lon    Degradation proteins, peptides glycopeptides    2.53 <br />  FTL_0908        2                                                          Pseudogene                                             2.07 <br />  FTL_0909        Hypothetical protein                                       Hypothetical proteins: conserved                       2.29 <br />  FTL_0911        2                                                          Pseudogene                                             2.34 <br />  FTL_0912        2                                                          Pseudogene                                             1.97 <br />  FTL_0964        ATP-dependent protease ATP-binding subunit          hslU   Protein folding stabilization                      4.33 <br />  FTL_0965        ATP-dependent protease peptidase subunit            hslV   Degradation proteins, peptides glycopeptides    5.13 <br />  FTL_0996        AhpC/Tsa family protein                                    Detoxification                                         2.01 <br />  FTL_0997        Hypothetical protein                                       Hypothetical proteins: conserved                       3.13 <br />  FTL_1008        2                                                          Pseudogene                                             2.01 <br />  FTL_1048        Hypothetical protein                                       Unknown function: general                              2.23 <br />  FTL_1049        DNA primase                                         dnaG   DNA replication, recombination repair              2.21 <br />  FTL_1050        RNAP sigma-70 factor                                rpoD   Transcription                                          4.58 <br />  FTL_1125        Transcription regulator                             hipA   Regulatory functions                                   2.75 <br />  FTL_1126        Transcription regulator                                    Regulatory functions                                   2.26 <br />  FTL_1190        Chaperone protein GrpE (heat-shock protein          grpE   Protein folding stabilization                      6.32 <br />                   family 70 cofactor) <br />  FTL_1191        Chaperone protein DnaK (heat-shock protein family   dnaK   Protein folding stabilization                      2.23 <br />                   70 protein) <br />  FTL_1192        Chaperone protein DnaJ (heat-shock protein 70       dnaJ   Protein folding stabilization                      6.19 <br />                   family cofactor) <br />  FTL_1194        Pirin family protein                                       Unknown function: general                              4.17 <br />  FTL_1221        Hypothetical membrane protein                              Cell envelope                                          5.11 <br />  FTL_1222        Conserved hypothetical protein                             Hypothetical proteins: conserved                       4.58 <br />  FTL_1226        2                                                          Pseudogene                                             2.65 <br />  FTL_1257        2                                                          Pseudogene                                             2.31 <br />  FTL_1259        2                                                          Pseudogene                                             3.41 <br />  FTL_1260        2                                                          Pseudogene                                             2.27 <br />  <br />  <br />  <br /> http://mic.sgmjournals.org                                                                                                           2565 <br />  N. Grall  <br />  <br /> Table 2. cont. <br />  <br />  <br />  Locus*                          Gene product                          Gene                 Cellular role categoryD                    Fold <br />                                                                                                                                      changed <br />  <br />  FTL_1265        2-Amino-4-hydroxy-6-hydroxymethyldihydropteri-        folK     Biosynthesis cofactors, prosthetic groups       5.01 <br />                   dine pyrophosphokinase/dihydropteroate synthase                carriers: <span id='am-25' about='obo:IMR_0200356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00504" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Folic acid" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200356" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>folic acid</span> <br />  FTL_1266        Lipase/esterase                                                Metabolism                                              5.82 <br />  FTL_1267        Hypothetical protein                                           Hypothetical proteins: conserved                        4.01 <br />  FTL_1269        2                                                              Pseudogene                                              2.07 <br />  FTL_1270        2                                                              Pseudogene                                              2.68 <br />  FTL_1361        Cold-shock protein                                    cspA     Hypothetical proteins                                   2.30 <br />  FTL_1365        Hypothetical protein                                           Hypothetical proteins: conserved                        2.49 <br />  FTL_1366        Membrane fusion protein, putative                              Transport binding proteins                          2.13 <br />  FTL_1403        Conserved hypothetical membrane protein                        Cell envelope                                           2.90 <br />  FTL_1411        Hypothetical protein                                  ybaB     Hypothetical proteins: conserved                        2.31 <br />  FTL_1412        Recombination protein RecR                            recR     DNA replication, recombination repair               2.07 <br />  FTL_1517        Hypothetical protein                                           Hypothetical proteins: conserved                        2.09 <br />  FTL_1535        2-Dehydro-3-deoxyphosphooctonate aldolase                      Cell envelope                                           1.99 <br />  FTL_1555        Hypothetical membrane protein                                  Cell envelope                                           4.32 <br />  FTL_1575        Hypothetical membrane protein                                  Cell envelope                                           4.09 <br />  FTL_1600        Periplasmic L-asparaginase II precursor               ansB     Metabolism                                              2.04 <br />  FTL_1691        Hypothetical protein                                           Hypothetical proteins: conserved                        2.02 <br />  FTL_1692        Hypothetical protein                                           Cell envelope: surface structures                       2.13 <br />  FTL_1714        Chaperone protein, GroEL                              groL     Protein folding stabilization                       3.05 <br />  FTL_1715        Chaperonin protein, GroES                             groS     Protein folding stabilization                       3.12 <br />  FTL_1770        Type IV pili <span id='am-35' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>-binding protein                        Transport binding proteins                          3.01 <br />  FTL_1771        Type IV pili nucleotide-binding protein               pilT     Transport binding proteins                          3.03 <br />  FTL_1776        Hypothetical membrane protein                                  Cell envelope                                           3.01 <br />  FTL_1777        2                                                              Pseudogene                                              2.13 <br />  FTL_1778        2                                                              Pseudogene                                              3.19 <br />  FTL_1793        Hypothetical protein                                           Hypothetical proteins: conserved                        2.06 <br />  FTL_1805        ATPase                                                         Unknown function                                        2.92 <br />  FTL_1806        MFS transport protein                                          Transport binding proteins                          2.91 <br />  FTL_1807        Histidyl-tRNA synthetase                              hisS     tRNA aminoacylation                                     3.12 <br />  FTL_1808        Ribosome-binding factor                                      Transcription                                           2.96 <br />  FTL_1810        Transcription elongation factor NusA                  nusA     Transcription                                           4.04 <br />  FTL_1811        Hypothetical protein                                           Unknown function: general                               2.74 <br />  FTL_1814        2                                                              Pseudogene                                              2.76 <br />  FTL_1878        component sensor protein KdpD                     kdpD     Regulatory functions:                             2.14 <br />  FTL_1894        Transposase, fragment                                          Mobile extrachromosomal element functions           4.16 <br />  FTL_1913        Sua5/YciO/YrdC family protein                                  Unknown function                                        2.06 <br />  FTL_1945        Hypothetical protein                                           Unknown function: general                               1.95 <br />  FTL_1957        Heat-shock protein                                             Protein folding stabilization                       8.57 <br />  FTL_XXXX&#194;&#167;       Transposase                                           isftu2   Mobile extrachromosomal element functions         2.0&#226;&#8364;&#8220;6.9 <br />  FTL_YYYY||      Transposase                                           isftu1   Mobile extrachromosomal element functions         2.3&#226;&#8364;&#8220;3.2 <br />  <br /> *Genes bold type putative RpoH-binding sequence located start codon  Supplementary Table S3 detailed information). <br /> Gene numbers marked italic type identified screens selections determinants required intracellular survival virulence (Kraemer et al., 2009; Maier et al., 2007; Su et al., 2007; Tempel et al., 2006; Weiss et al., 2007). <br /> DCellular role category according TIGR. <br /> dFold change calculated average expression 44 uC relative 37 uC calculated independent experiments duplicate <br /> hybridizations. <br />  number isftu2-type transposases: FTL_0079, 0220, 0270, 0367, 0441, 0468, 0568, 0575, 0591, 0622, 0861, 0868, 1301, 1368, 1471, 1516, 1655, <br /> 1680 1885. <br /> | isftu1-type transposases: FTL_0128, 0417 0607. <br />  <br />  <br />  <br />  <br /> 2566                                                                                                                          Microbiology 155 <br />                                                                                                         Francisella tularensis heat-shock response <br />  <br />  <br />  <br />  <br />       Fig. 2. Quantification RT-PCR transcript levels selected genes heat shock.   Validation microarray <br />       experiment. Transcript levels normalized DNA helicase (FTL_1656) fold difference (relative 37 6C) <br />       SDs shown dnaK, groEL, hsp40 rpoH genes. (b) Change expression heat-shock genes       60 min incubation 44 6C. Transcript levels normalized DNA helicase (FTL_1656) fold difference <br />       (relative sample taken temperature shift) SDs shown dnaK, groEL, hsp40 grpE genes. Data       triplicate samples. <br />  <br />  <br />  <br />  <br /> tome.  compared transcriptomes LVS                         known heat-shock proteins (Hsp40, Hsp90, GroES, GroEL, <br /> (transformed plasmid pFNLTP6gro                              DnaK DnaJ). transcriptional control RpoH <br /> inserted gene) LVS overexpressing RpoH                          specific heat-shock proteins. <br />  e. transformed plasmid pFNLTP6gro_0851, <br /> expressing FTL_0851 groES promoter).                                                                               Consensus sequence Francisella RpoH- <br /> extracted RNA exponentially grown bacterial cul- <br />                                                                               dependent promoter <br /> tures, cDNA synthesis labelling, control RpoH-overexpressing samples paired hybridized                        identify RpoH-binding site consensus sequence, microarrays. Statistical analysis indicated                  analysed 200 bp sequence upstream 112 heat- <br /> seven genes exhibited significant increased expression                        upregulated genes listed Table 2 using Bioprospector <br /> (FDR 0 %) response overexpression RpoH                               (Liu et al., 2001). analyses failed identify RpoH <br /> (Table 3). expected, rpoH itself upregulated 21-                       consensus binding sequence, reasoned likely <br /> fold. upregulated genes encoded                        fact relatively small proportion heat- <br />  <br />  <br />  <br />  <br /> Table 3. Genes showing change expression presence plasmid overexpressing RpoH <br />  <br />  Locus                           Gene product*                        Gene                    Cellular role categoryD                Fold changed <br />  <br />  FTL_0851        RNAP sigma factor                                    rpoH         Transcription: transcription factors                  20.6 <br />  FTL_0281        Heat-shock protein, Hsp40                                         Protein fate: protein folding stabilization        2.9 <br />  FTL_0267        Heat-shock protein 90                                htpG         Protein fate: protein folding stabilization        2.9 <br />  FTL_1191        Chaperone protein DnaK (heat-shock protein           dnaK         Protein fate: protein folding stabilization        2.6 <br />                   family 70 protein) <br />  FTL_1192        Chaperone protein DnaJ (heat-shock protein           dnaJ         Protein fate: protein folding stabilization        2.0 <br />                   70 family cofactor) <br />  FTL_1714        Chaperone protein, GroEL                             groEL        Protein fate: protein folding stabilization        2.2 <br />  FTL_1715        Chaperone protein, GroES                             groES        Protein fate: protein folding stabilization        2.3 <br />  <br /> *Genes significantly changed expression, upregulated twofold, included table displayed according functional <br /> category. <br /> DCellular role category according TIGR. <br /> dValues average fold change LVS/pFNLTP6gro-0851 LVS/pFNLTP6gro independent experiments duplicate <br /> hybridization. microarray contains spots gene. <br />  <br />  <br /> http://mic.sgmjournals.org                                                                                                                      2567 <br />  N. Grall  <br />  <br />  <br />  <br />        Fig. 3. Consensus sequence F. tularensis s32-binding site displayed WebLogo (http://weblogo.berkeley.edu).        height column indicates sequence conservation position, height symbols stack <br />        indicates relative frequency nucleic acid position. N(13&#226;&#8364;&#8220;16) indicates gap blocks. <br />  <br />  <br />  <br /> regulated genes identified microarray analyses              significant homology RpoH-binding site consensus <br /> directly regulated RpoH, making difficult                    sequences E. coli V. cholerae (Slamti et al., 2007). <br /> Bioprospector identify conserved binding sites. address problem, repeated Bioprospector <br /> analyses using upregulated genes annotated NCBI chaperones, chaperonins, proteases heat-shock <br />                                                                        DISCUSSION <br /> proteins, classes genes known directly regulated           bacterial heat-shock response results transient <br /> RpoH bacteria (Supplementary Tables S3 S4).               induction set highly conserved heat-shock proteins, yielded putative binding sites,          including chaperones proteases. alternative s- <br /> corresponded previously predicted F. tularensis RpoH-               factor s32 (RpoH) primary regulator controls binding sites (Meibom et al., 2008). PFM representing            transcription genes heat shock sites identified Bioprospector constructed              general stress conditions. activity heat-shock <br /> program PATSER (van Helden, 2003) used interrogate              proteins ensures correct folding proteins upstream regions 1754 ORFs NCBI                      normal growth, essential resistance stress annotation LVS putative RpoH-binding sites. total             minimizing damage caused accumulation of 72 non-overlapping putative RpoH-binding sites               misfolded aggregated proteins. examination score 9 identified upstream 71 genes               heat-shock response F. tularensis resulted (Supplementary Table S5), including 13 genes                    identification bacterial heat-shock proteins, upregulated heat shock (Table 4 marked bold                  previously reported increase type Table 2). 13 putative sites consensus             heat oxidative stress (Ericsson et al., 1994; <br /> binding sequence determined (Fig. 3) shares                   Lenco et al., 2005) response mammalian body <br />  <br />  <br />  <br /> Table 4. Heat-induced genes potential s32-binding sequence promoter region <br />  <br />  Distance*    LengthD                Binding site sequence              Locus                        Gene productd <br />  <br />      34          27     CTTGAAAAACAGCAAAGTATCCTTAAG                    FTL_0094    ClpB protein <br />      81          30     CTTGAAAAAACTAACTTCCACTCTCATATA                 FTL_0267    Heat-shock protein 90, HtpG <br />      84          27     CTTGTATTTATATAAAAAGACCTTATA                    FTL_0281    Heat-shock protein, Hsp40 <br />       7          28     CTTGAAAATGATTAGAAAAAGTATTTTA                   FTL_0663    Hypothetical protein <br />     101          29     CTTGTCATATATTTAATAATAAACATATA                  FTL_0891    Trigger factor <br />     162          28     CTTGAAGTGATGTTTCATATACCTTCAT                   FTL_0894    DNA-binding, ATP-dependent protease La <br />      48          29     CTTGTAAAATTTAATATCAATTCCATTTA                  FTL_0965    ATP-dependent protease, peptidase subunit, HslV <br />      82          29     CTTGAAAATACAGAATTAATCACTATCTA                  FTL_1190    Chaperone protein GrpE (heat-shock protein family <br />                                                                                     70 cofactor) <br />        92        29     CTTGAAAAGATTATAAATATGCCCATCTA                  FTL_1191    Chaperone protein DnaK (heat-shock protein family <br />                                                                                     70 protein) <br />        68        30     CTTGAAAATTAAAAAAAAGCCCCCAATTTC                 FTL_1265    2-Amino-4-hydroxy-6-hydroxymethyldihydropteri- <br />                                                                                     dine pyrophosphokinase/dihydropteroate synthase <br />        98        29     CTTGAAAATTTTTTTTTTGACTCAATATC                  FTL_1715    Chaperonin protein, GroES <br />        40        29     CTTGTATTTAGCATGATCGTATCATTCTT                  FTL_1805    ATPase <br />        75        27     CTTGAAATTTTAAAAACCGATCATATA                    FTL_1957    Heat-shock protein <br />  <br /> *Distance ATG start codon. <br /> DLength (nucleotides) putative s32-binding site. <br /> dGene product according TIGR. <br />  <br /> 2568                                                                                                                  Microbiology 155 <br />                                                                                            Francisella tularensis heat-shock response <br />  <br />  <br /> temperature (Horzempa et al., 2008), addition                 Defining RpoH consensus box <br /> various genes. <br />                                                                     Using data obtained transcriptional <br />                                                                     profiling, performed bioinformatics analysis Role s32 heat-stress response                                 determine consensus sequence binding s32 (Fig. <br />                                                                     3). identified putative s32-binding site Control s32 activity E. coli complex <br />                                                                     promoter region 13 genes induced heat performed transcriptional, post-transcriptional <br />                                                                     shock. genes induced RpoH post-translational levels. heat-shock response E.        overexpression, gene presumed gene <br /> coli induced result increased s32 levels, primarily      transcripts dnaK-dnaJ groES-groEL, suggesting increase rpoH mRNA translation                     genes directly s32 control. increased stability activity s32 (Arse`ne et al.,            remainder heat-induced genes, did observe <br /> 2000), control rpoH transcription            putative s32-binding sites, indicating of minor importance. Similarly, did observe          controlled directly s32. consensus <br /> increase transcription rpoH F. tularensis            binding sequence s32 resembles consensus sequence <br /> temperature upshift. contrast, transcription s70          V. cholerae E. coli (Nonaka et al., 2006; Slamti et <br /> increased heat shock, suggesting s70-containing          al., 2007), genes putative operons <br /> RNAPs required heat-shock response                   preceded s32-binding site F. tularensis Francisella, combination gene-specific              E. coli, small variations regulators.                                                         observed (Nonaka et al., 2006) (Fig. 4). observed heat- directly define genes dependent RpoH,             mediated upregulation consecutive genes tig- <br /> performed transcriptional profiling F. tularensis LVS            clpP-clpX-lon, bioinformatics analysis identified s32- <br /> harbouring plasmid expressing RpoH groE                  binding site upstream tig lon genes (Fig. 4a). <br /> promoter compared bacteria carrying             E. coli, clpP, clpX lon induced plasmid. allowed identify genes  addition         heat shock, s32-binding site upstream rpoH itself) expressed increased levels      clpP lon. F. tularensis E. coli, strains overexpressing RpoH. genes encode heat-           transcription unit dnaK-dnaJ induced heat shock <br /> shock proteins Hsp40, HptG, DnaK, DnaJ, GroES                   preceded s32-binding site. F. tularensis, GroEL, induced heat stress         genes adjacent grpE, heat-induced previous studies s32 regulon E. coli, V. cholerae         preceded s32-binding site, E. coli grpE N. gonorrhoeae identified genes (Gunesekere          located genome (Fig. 4b). organiza- <br /> et al., 2006; Nonaka et al., 2006; Slamti et al., 2007; Zhao et     tion gene transcriptional unit groES-groEL <br /> al., 2005), demonstrating common features RpoH                   identical species (Fig. 4d), regulation different species.  number             case hslV-hslU genes, exception genes identified F. tularensis unexpectedly low         E. coli genes followed heat-induced gene, <br /> compared bacterial species. date,         menA (Fig. 4c). protease FtsH carries degradation <br /> promoter allows specific induction transcription        s32 E. coli itself induced heat shock, Francisella, experiments                  transcription unit contains s32-binding site (Fig. 4e). <br /> performed continuous overexpression RpoH.                  Interestingly, F. tularensis did observe increased <br /> Based knowledge regulation s32 E. coli,         expression ftsH gene heat shock, genetic believe lead underestimation genes          region encoding FtsH predicted contain s32- <br /> directly controlled RpoH. current model               binding site (Fig. 4e). region containing ftsH regulation s32 E. coli, called &#226;&#8364;&#732;unfolded protein          duplicated LVS strain, titration model&#226;&#8364;&#8482;, stability activity s32 rely     copy strains F. tularensis level unfolded protein relative chaperone level       sequence information available. Currently, review Guisbert et al., 2008). unfolded protein            know activity s32 F. tularensis levels low compared chaperones, s32 predomi-            controlled mechanism similar &#226;&#8364;&#732;unfolded protein <br /> nantly bound chaperones inactive,           titration model&#226;&#8364;&#8482; degradation s32 high level unfolded proteins titrate chaperones         performed FtsH protease. case, regulatory <br />                                                                     mechanism differ E. coli, FtsH away s32, rendering s32 active.                                                                     heat-shock response F. tularensis. <br /> experimental set  unfolded protein levels expected low  stress imposed bacteria),             observed s32-binding sites number chaperones expressed result             genes did upregulated RpoH <br /> RpoH overexpression. mechanism s32                   overexpression heat shock. possible regulation exists F. tularensis E. coli, set   missed transcriptomic analysis heat <br /> lead chaperone-mediated inactivation s32,            stress induced constant overexpression responsible low number genes                 RpoH. performed transcriptomic analysis observe induced RpoH-overexpressing strain.            30 min heat exposure, probable genes <br />  <br /> http://mic.sgmjournals.org                                                                                                    2569 <br />  N. Grall  <br />  <br />  <br />  <br />        Fig. 4. Schematic representations organization selected genes containing putative s32-binding sequences F. <br />        tularensis E. coli. Black arrows represent genes induced heat stress. Grey arrows represent genes        induced heat stress. Gene symbols indicated arrows locus numbers left panel (numbers <br />        preceded FTL_ designate loci F. tularensis LVS numbers preceded b loci E. coli MG1655). isftu, <br />        transposase; H, hypothetical protein; mfs, major facilitator transporter. locations s32-binding sites E. coli        Nonaka et al. (2006). <br />  <br />  <br />  induced early heat-shock response.                    intracellular multiplication survival. genes <br /> qRT-PCR analysis temporal expression heat-                 include clpB (Gray et al., 2002; Maier et al., 2007; Meibom <br /> shock genes showed increased expression times                   et al., 2008; Su et al., 2007; Tempel et al., 2006; Weiss et al., 10 60 min heat exposure,                    2007), hspG (Weiss et al., 2007), FTL_0663 (Kraemer et al., <br /> elevated transcript levels later time points.                2009; Su et al., 2007), dnaK (Tempel et al., 2006) results indicate heat-shock response initiated                FTL_1806 [encoding major facilitator superfamily (MFS) <br /> promptly continues 1 h, rule                 transport protein] (Maier et al., 2007; Weiss et al., 2007). possibility genes increased                      identification genes, important expression early heat-shock response.                         intracellular survival, indicate stress responses <br /> Interestingly, number genes containing putative                  play central role pathogenesis F. tularensis s32-binding sites  induced heat stress)               ability survive inside host cells. intracellular <br /> induced oxidative stress (N. Grall, unpublished results),              environment hostile bacterium, induction indicating heat-shock response                       stress responses imperative survival mediated s32. Finally, anticipate                  host cells. <br /> putative sites represent true s32-binding sites,                   Given significant number genes (18 %) identified bioinformatics                       induced heat stress implicated analysis.                                                                 virulence Francisella, interesting                                                                           study molecular mechanisms underlying stress relationship heat-stress response                           response. particular, understanding virulence                                                                 activity s32 controlled, including determining                                                                           proteins signals mediate control, lead Remarkably, considerable number genes                  better understanding s32-mediated stress <br /> upregulated heat stress (20 genes, Table 2)              responses contribute survival mammalian <br /> identified screens selections genes required                host. <br /> </body></html>