<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Int J Biochem Cell Biol. 2008 ; 40(11): 2560&#226;&#8364;&#8220;2571. doi:10.1016/j.biocel.2008.05.003. <br />  <br />  <br />  <br />                            Expression, essentiality, microtiter plate assay                            mycobacterial GlmU, bifunctional glucosamine-1-phosphate <br />                            acetyltransferase N-acetylglucosamine-1-phosphate <br />                            uridyltransferase <br />  <br />                            Wenli Zhang1, Victoria C. Jones2, Michael S. Scherman2, Sebabrata Mahapatra2, Dean <br />                            Crick2, Suresh Bhamidi2, Yi Xin3, Michael R. McNeil2*, Yufang Ma1,4* <br />                            1Department Biochemistry Molecular Biology, Dalian Medical University, Dalian 116044, China <br />                            2Department Microbiology, Immunology Pathology, Colorado State University, Fort Collins, <br />                            Colorado 80523, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            3Department Biotechnology, Dalian Medical University, Dalian 116044, China <br />                            4Liaoning Provincial Core Lab Glycobiology Glycoengineering, Dalian 116044, China <br />  <br />                            Abstract <br />                                 <span id='am-168' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-169' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-N-acetyl-<span id='am-47' about='obo:IMR_0200485' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00329" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0200485" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="D-Glucosamine" datatype="xsd:string"></span>D-glucosamine</span> (UDP-GlcNAc) essential precursor <span id='am-97' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-98' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-100' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span>peptidoglycan</span>                                 rhamnose-GlcNAc linker region mycobacterial cell wall. Mycobacterium tuberculosis H37Rv <br />                                 genome, Rv1018c shows strong homology GlmU protein involved formation UDP- <br />                                 GlcNAc bacteria. GlmU bifunctional enzyme catalyzes sequential steps                                 UDP-GlcNAc biosynthesis. Glucosamine-1-phosphate acetyl transferase catalyzes formation                                 N-acetylglucosamine-1-phosphate, N-acetylglucosamine-1-phosphate uridylyltransferase <br />                                 catalyzes formation UDP-GlcNAc. inhibition peptidoglycan synthesis results <br />                                 cell lysis, M. tuberculosis GlmU potential anti-tuberculosis drug target. study cloned <br />                                 M. tuberculosis Rv1018c (glmU gene) expressed soluble GlmU protein E. coli BL21(DE3). <br />                                 Enzymatic assays showed M. tuberculosis GlmU protein exhibits glucosamine-1-phosphate <br />                                 acetyltransferase N-acetylglucosamine-1-phosphate uridylyltransferase activities.                                 investigated effect Mycobacterium smegmatis activity GlmU fully removed NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 reduced genetic approach. results showed activity GlmU required growth                                 M. smegmatis bacteria did grow absence active GlmU enzyme.                                 functional GlmU enzyme gradually reduced temperature shift experiment, M. <br />                                 smegmatis cells non-viable morphology changed normal rod shape stubby- <br />                                 rounded morphology cases lysed. Finally microtiter plate based assay GlmU <br />                                 activity OD340 read developed. studies support                                 development M. tuberculosis GlmU enzyme target new anti-tuberculosis drugs. <br />  <br />  <br />  <br />  <br />                             corresponding authors: Yufang Ma, Department Biochemistry Molecular Biology, Dalian Medical University, Dalian 116044, <br />                            China. Email: yufang_ma@hotmail.com. Fax: +86-411-86110139. Phone: +86-411-86110307; Michael McNeil, Department                            Microbiology, Immunology Pathology, Colorado State University, Fort Collins, Colorado 80523, USA. Email: <br />                            mmcneil@colostate.edu. Fax: (970) 491-1815. Phone: (970) 491-1784. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Zhang et al.                                                                                                  Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     Mycobacterium tuberculosis; Mycobacterium smegmatis; GlmU; glucosamine-1-phosphate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     acetyltransferase; N-acetylglucosamine-1-phosphate uridylyltransferase <br />  <br />  <br />                               1. Introduction <br />                                              emergence drug resistant strains bacteria problem                                              controlling infectious disease (Peterson, 2005). especially true case                                              tuberculosis. Tuberculosis (TB) remains major threat world health; world-wide <br />                                              statistics million people developing active tuberculosis nearly fourth                                              dying disease stark reality (Dye et al.1999).  Global Alliance TB <br />                                              drug Development www.tballiance.org.) TB, public health problem,                                              complicated lack wide array chemotherapeutics causative agent, <br />                                              Mycobacterium tuberculosis,  emergence drug-resistant strains                                              line drugs large problem recent years. current line anti-TB <br />                                              drug regimens achieve 99% efficacy, reduced drug <br />                                              resistance (O&apos;Brien Nunn, 2001). Multiple drug resistant TB (MDR-TB), bacillus <br />                                              resistant line drugs used fearsome type resistance;  <br />                                               strains resistant second line antibiotics appearing (Gandhi et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              2006; Van Rie Enarson, 2006); dubbed extreme drug resistant TB (XDR- <br />                                              TB). <br />  <br />                                              mycobacterial cell wall core consists layers (McNeil Brennan, 1991; Lee et al., <br />                                              2006). highly impermeable outer layer composed mycolic acids lipids <br />                                              containing 70 90 carbons. inner layer consists peptidoglycan. layers                                              covalently tethered connecting <span id='am-7' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span>polysaccharide</span> arabinogalactan (McNeil Brennan, <br />                                              1991; Lee et al., 2006), attached peptidoglycan disaccharide linker, <br />                                              &#206;&#177;-L-rhamnosyl-(1,3)-&#206;&#177;-D-GlcNAc-(phosphate).  GlcNAc essential component                                              peptidoglycan  bacteria) rhamnose-GlcNAc linker cell wall                                              mycobacteria. <br />  <br />                                              E.coli (Mengin-Lecreulx Vanheijenoort, 1994; Riley et al., 2006), enzymes <br />                                              catalyze sequential steps biosynthesis UDP-GlcNAc, activated <span id='am-82' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-83' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-86' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> <br />                                              sugar GlcNAc. glutamine fructose-6-phosphate transferase, encoded glmS gene, <br />                                              catalyzes fructose-6-phosphate glutamine glucosamine-6-phosphate glutamate,                                              phosphoglucosamine mutase, encoded glmM gene, converts glucosamine-6-phosphate <br />                                              (GlcNH2-6-P) glucosamine-1-phosphate (GlcNH2-1-P). GlmU encoded glmU NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              bifunctional enzyme. activity GlmU, glucosamine-1-phosphate acetyltransferase, <br />                                              catalyzes formation N-acetylglucosamine-1-phosphate (GlcNAc-1-P)                                              glucosamine-1-phosphate using acetyl-<span id='am-67' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-68' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> acetyl donor. second activity GlmU, <br />                                              N-acetylglucosamine-1-phosphate uridylyltransferase, catalyzes formation UDP- <br />                                              GlcNAc GlcNAc-1-P <span id='am-143' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-144' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Uridine" datatype="xsd:string"></span>uridine</span> <span id='am-77' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span> (<span id='am-148' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-149' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span>UTP</span>). active sites reside                                              C N terminal regions protein respectively (Olsen Roderick, 2001). Analysis                                              genome sequence Mycobacterium tuberculosis H37Rv shown protein <br />                                              encoded M. tuberculosis Rv3436c, Rv3441c Rv1018c homologous E. coli GlmS, <br />                                              GlmM, GlmU, respectively.  propose mycobacteria                                              biosynthetic pathway UDP-GlcNAc E. coli (Fig. 1). Antibiotics attacking <br />                                              peptidoglycan synthesis, &#206;&#178;-lactams glycopeptides result cell death typically <br />                                              lysis (Holtje, 1995), reason attractive. GlmS GlmM <br />                                              catalyze reactions identical similar eukaryotes. reaction <br />  <br />  <br />  <br />                                               Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                Page 3 <br />  <br />  <br />                                           catalyzed GlmU unique prokaryotes GlmU important potential <br />                                           drug target. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           study, expressed M. tuberculosis GlmU protein E. coli BL21(DE3) identified <br />                                           GlmU glucosamine-1-phosphate acetyltransferase N-acetylglucosamine-1- <br />                                           phosphate uridylyltransferase. determine effect active GlmU mycobacterial <br />                                           growth, prepared M. smegmatis glmU knockout strain complemented M. <br />                                           tuberculosis glmU rescue plasmid temperature sensitive replication. Finally <br />                                           developed microtiter plate based assay activity. <br />  <br />                               2. Materials methods <br />                               2.1 BACTERIAL STRAINS PLASMIDS <br />                                           characteristics bacterial strains plasmids used study described Table <br />                                           1. E. coli DH5&#206;&#177;, NovaBlue BL21(DE3) cells grown Luria-Bertani (LB) broth                                           LB agar 37&#194;&#176;C routinely. M. smegmatis mc2155 cells grown LB broth containing <br />                                           0.05% Tween 80 LB agar 37&#194;&#176;C routinely. Sucrose added LB agar final <br />                                           concentration 10% selecting M. smegmatis glmU knockout strain ZW-2. Incubation <br />                                           M. smegmatis ZW-2 strain carrying temperature sensitive rescue plasmid 30&#194;&#176; <br />                                           C (permissive temperature) 42&#194;&#176;C (non-permissive temperature) respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           final concentration antibiotics used study follows: ampicillin (<span id='am-40' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-41' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="AMP" datatype="xsd:string"></span>Amp</span>), 100 <br />                                           &#194;&#181;g ml&#226;&#710;&#8217;1; gentamicin (Gen), 5 &#194;&#181;g ml&#226;&#710;&#8217;1 E. coli M. smegmatis; kanamycin (Kan), 50 &#194;&#181;g <br />                                           ml&#226;&#710;&#8217;1 E. coli 25 &#194;&#181;g ml&#226;&#710;&#8217;1 M. smegmatis; streptomycin (Str), 20 &#194;&#181;g ml&#226;&#710;&#8217;1 E. <br />                                           coli 10 &#194;&#181;g ml&#226;&#710;&#8217;1 M. smegmatis. <br />  <br />                               2.2 CLONING GENETIC MANIPULATION <br />                                           Cloning procedures performed according standard protocols (Sambrook Russell, <br />                                           2001). M. smegmatis mc2155 genomic DNA prepared previously described (Jackson <br />                                           et al., 2000). Probe Southern blotting labeled digoxygenin (DIG)-labeled <br />                                           <span id='am-62' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-63' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="dUTP" datatype="xsd:string"></span>dUTP</span> using DIG High Prime Labeling Detection Starter Kit (Roche). SmaI- <br />                                           digested M. smegmatis genomic DNA fragments separated 0.8% agarose gel                                           transferred Nytran membrane (Whatman Schleicher Schuell) described (Li et al., <br />                                           2006). DNA hybridization detection performed according Roche&#226;&#8364;&#8482;s instructions. <br />  <br />                               2.3 CLONING, EXPRESSION PURIFICATION M. TUBERCULOSIS GLMU E. COLI <br />                               MURA MURB <br />                                           M. tuberculosis glmU gene (Rv1018c) amplified M. tuberculosis H37Rv <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           genomic DNA (supplied Colorado State University NIH contract &#226;&#8364;&#339;Tuberculosis <br />                                           research materials vaccine testing.&#226;&#8364;?) using Mtb glmU1 primer, 5&#226;&#8364;&#8482; <br />                                           CATATGACGTTTCCTGGTGACAC 3&#226;&#8364;&#8482; (underlined sequence NdeI site) Mtb <br />                                           glmU2 primer, 5&#226;&#8364;&#8482; CTCGAGTCACGGTGTCTGATCAGCGTC 3&#226;&#8364;&#8482; (underlined sequence                                           XhoI site). amplified PCR product ligated pBluescript II KS(&#226;&#710;&#8217;) (Strategene)                                           generate plasmid pBS-Mtb glmU (Table 1). Following sequence confirmation Mtb glmU <br />                                           gene ligated NdeI XhoI sites pET-16b (Novagen), resulting pET16b- <br />                                           Mtb glmU (Table 1). pET16b-Mtb glmU transformed E. coli BL21(DE3) expression <br />                                           GlmU protein. <br />  <br />                                           E. coli BL21(DE3) containing pET16b-Mtb glmU grown 2 liters LB broth containing <br />                                           Amp overnight 37&#194;&#176;C. cells transferred shaker room temperature induced <br />                                           overnight addition isopropyl-D-thiogalactopyranoside (IPTG) final concentration <br />                                           0.1 mM. cells harvested pellets resuspended 20 ml lysis prep buffer (20 <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 4 <br />  <br />  <br />                                           mM Tris-HCl, pH 8.0, 500 mM NaCl, 20% glycerol) 1 &#194;&#181;M leupeptin, 1 &#194;&#181;M pepstatin  <br />                                           1 mM phenylmethylsulphonyl fluoride (PMSF). cells broken using French <br />                                           Pressure method, followed centrifugation 20 000 x g 40 minutes. supernatant NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           applied 1.5 ml column volume Select HF affinity gel (Sigma). column washed <br />                                           40 ml wash buffer (lysis prep buffer 20 mM imidazole, pH 8.0). protein                                           eluted 15 ml elute buffer (20 mM Tris-HCl, 500 mM NaCl, 20% glycerol, 200 mM <br />                                           imidazole, pH 7.0) 1 &#194;&#181;M leupeptin, 1 &#194;&#181;M pepstatin  1 mM PMSF, 5 <br />                                           ml collected testing. <br />  <br />                                           purified protein run 12% SDS-PAGE transferred nitrocellulose membrane <br />                                           (Whatman Schleicher Schuell) blotting buffer (20 mM Tris-base, 150 mM <span id='am-119' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-120' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span>                                           20% methanol). membrane blocked 2% BSA TBST buffer (10 mM Tris-HCl, <br />                                           pH 8.0, 150 mM NaCl, 0.05% Tween 20) incubated (anti)-polyhistidine clone 1 <br />                                           antibody (Sigma) 1 6000 dilution followed washing TBST. membrane                                           incubated antimouse-IgG-AP conjugate (Sigma) protein band visualized                                           BCIP/NBT solution. <br />  <br />                                           gene murA amplified E. coli K12 genomic DNA following                                           primers incorporating NcoI SmaI restriction enzyme sites (underlined): 5&#226;&#8364;&#8482; <br />                                           AACTCCATGGATAAATTTCGTGTTCAG 3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482; <br />                                           AGACCCGGGTTCGCCTTTCACACGC 3&#226;&#8364;&#8482;. PCR products digested NcoI <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SmaI, ligated expression vector pTYB4, previously digested                                           NcoI SmaI. E. coli murB gene amplified E. coli genomic DNA                                           following primers incorporating NcoI SmaI restriction enzyme sites (underlined)                                           cloned pTYB4: GCAGCCATGGACCACTCCTTAAAACC                                           GTTATCCCGGGTGAAATTGTCTCCAC. pTYB4-Ec murA pTYB4-Ec murB                                           transformed ER2566 respectively. expressed E. coli MurA E.coli MurB proteins                                           purified IMPACT&#226;&#8222;&#162; Protein Purification according New England Biolabs&#226;&#8364;&#8482; <br />                                           instructions. <br />  <br />                               2.4 DETERMINATION ACETYL TRANSFERASE URIDYLYLTRANSFERASE <br />                               ACTIVITIES M. TUBERCULOSIS GLMU PROTEIN <br />                                           acetyl transferase uridylyltransferase activities M. tuberculosis GlmU protein                                           determined separately. acetyl transferase activity, reaction mixture contained 100 <br />                                           &#194;&#181;M acetyl CoA glucosamine-1-phosphate 10 mM Tris buffer (0.1 mM MgCl2 <br />                                           0.01% Triton X 100, pH 8). transacetylation carried addition 100 <br />                                           ng GlmU (purified) crude M. smegmatis extract reaction incubated 30 <br />                                           minutes room temperature. control experiment, addition enzyme source, <br />                                           substrates buffer carried  incubation, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           reaction mixture dried cleaned glass tube mild hydrolysis carried                                           0.2 mM TFA (Trifluroacetic acid) 80&#194;&#176;C 1 hour. tubes cooled room <br />                                           temperature dried. drying repeated twice presence methanol.                                           followed acetylation 1:1 deutero acetic anhydride/pyridine, overnight. drying <br />                                           solvents, acetylated products extracted D2O:CHCl3 (1:1) chloroform <br />                                           layer subjected GC/MS. acetylated-derivatives dissolved chloroform prior <br />                                           injection DB-5 column initial temperature 50&#194;&#176;C held 1 min. temperature <br />                                           increased 275&#194;&#176;C rate 5&#194;&#176;C/min. mass spectral analysis carried positive <br />                                           CI mode collision <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span>gas</span> methane. starting material showed M+H-63 <span id='am-34' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> <br />                                           3 AMU higher product non-deuterated methyl group product. <br />  <br />                                           determine uridylyltransferase active, enzyme reaction run reverse <br />                                           direction PPi UDP-GlcNAc GlcNAc-1-P UTP (Mengin-Lecreulx                                           Vanheijenoort, 1994). reaction performed 40 &#194;&#181;l total volume containing 25 &#194;&#181;M <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                           Page 5 <br />  <br />  <br />                                           UDP-GlcNAc, 1 mM <span id='am-29' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyrophosphate</span> (PPi) 42 &#194;&#181;g/ml GlmU protein 50 mM Tris-HCl <br />                                           (pH 8.2) containing 1 mM MgCl2. incubation 37&#194;&#176;C 30 minutes, reaction                                           run Beckman HPLC 30 minute gradient starting 100% 75 mM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           KH2PO4 ending 100% 500 mM KH2PO4. <br />  <br />                                           microtiter plate assay, GlmU activity linked MurA MurB E. coli. <br />                                           assay run combining 4 &#194;&#181;g purified M. tuberculosis GlmU, 0.6 &#195;&#172;g purified E. coli <br />                                           MurA 80 ng purified E. coli MurA 80 &#194;&#181;l 200 mM Tris containing 10 mM MgCl2, 200 <br />                                           &#194;&#181;M <span id='am-57' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-58' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span>, 20 mM KCl 96 plate. reaction started                                           adding 20 &#194;&#181;l 200 mM Tris, 10 mM MgCl2 containing 1 mM UTP, 1 mM acetyl CoA, 1 mM <br />                                           glucosamine-1-phosphate, 1 mM phosphoenol pyruvate. Absorbance 340 nm                                           followed 45 minutes 30&#194;&#176;C Benchmark Plus (Bio-Rad) plate reader data                                           analyzed using Microsoft Excel. <br />  <br />                               2.5 CONSTRUCTION SUICIDE PLASMID RESCUE PLASMID <br />                                           <span id='am-90' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-96' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence M. tuberculosis H37Rv GlmU protein <br />                                           (http://genolist.pasteur.fr/TubercuList/) used query BLASTP identify                                           homologous gene M. smegmatis mc2155 genome. M. smegmatis glmU gene <br />                                           (MSMEG_5426) 75% identical homology Rv1018c. M. smegmatis <br />                                           glmU (1449 bp) upstream region (491 bp) amplified mc2155 genomic DNA <br />                                           using LA Taq DNA polymerase (Takara), glmU1 primer <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (5&#226;&#8364;&#8482;ACTAGTCTCGACACGGACCCGGGAGC 3&#226;&#8364;&#8482;, underlined sequence SpeI site)                                           glmU2 primer (5&#226;&#8364;&#8482;TGCGGCCGCTCAGCTCTCGTCGCCCAACG3&#226;&#8364;&#8482;, underlined <br />                                           sequence NotI site). PCR product 1947 bp purified cloned pMD18-T <br />                                           vector (Takara) generate plasmid plasmid pZW-1 (Table 1). glmU gene disrupted <br />                                           inserting kanamycin resistance cassette (kanR) pUC4K (GE Healthcare), yielding <br />                                           pZW-2 (Table 1). pZW-2 digested NotI SpeI resulting glmU::kanR <br />                                           fragment (3.1 kb) ligated NotI SpeI sites pPR27-xylE (Li et al., 2006), <br />                                           resulting suicide plasmid pZW-3 (Table 1). parent plasmid pPR27 (Pelicic et al., <br />                                           1997) shuttle vector containing replication origins E.coli mycobacteria. <br />                                           Specifically, replication origin mycobacteria mutations resulting plasmid <br />                                           replication temperature sensitive mycobacteria.  pZW-3 replicates                                           permissive temperature (30&#194;&#176;C), replicate non-permissive temperature (42&#194;&#176;C), <br />                                           resulting integration M. smegmatis genome intact glmU gene                                           glmU::kanR present. <br />  <br />                                           M. tuberculosis glmU gene amplified M. tuberculosis H37Rv genomic DNA using <br />                                           Mtb glmU1 primer, 5&#226;&#8364;&#8482; CATATGACGTTTCCTGGTGACAC 3&#226;&#8364;&#8482; (underlined sequence                                           NdeI site)&#226;&#8364;&#8482; Mtb glmU3 primer, 5&#226;&#8364;&#8482;GGATCCTCACGGTGTCTGATCAGCGTC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           3&#226;&#8364;&#8482; (underlined sequence BamHI site). PCR product M. tuberculosis glmU cloned <br />                                           pMD18-T vector generate plasmid pZW-4 (Table 1). M. tuberculosis glmU gene <br />                                           ligated NdeI BamHI sites pET23b-Phsp60 (Li et al., 2006), yielding                                           pZW-5 (Table 1). Phsp60-Mtb glmU ligated XbaI BamHI sites pCG76 <br />                                           (Guilhot et al., 1994), resulting rescue plasmid pZW-6 (Table 1). M. tuberculosis <br />                                           glmU gene transcribed promoter heat shock protein 60 M. bovis BCG <br />                                           M. tuberculosis GlmU protein complemented glmU::kanR M. smegmatis ZW-2.                                           parent plasmid pCG76 temperature-sensitive mycobacterial replication origin                                           pPR27 replicate 30&#194;&#176;C, effectively lost 42&#194;&#176;C. <br />  <br />                               2.6 SELECTION M. SMEGMATIS GLMU KNOCKOUT STRAIN ZW-2 <br />                                           M. smegmatis electrocompetent cells prepared described (Pelicic et al., 1997)                                           suicide plasmid pZW-3 electroporated competent cells Gene Pulser (Bio- <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 6 <br />  <br />  <br />                                            Rad). Transformants grown LB agar plates containing Kan Gen 30&#194;&#176;C.                                            colony propagated LB broth containing 0.05% Tween 80, Kan Gen 30&#194;&#176;C, <br />                                            cells plated LB agar plate containing Kan Gen incubated NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            42&#194;&#176;C colonies appeared. M. smegmatis ZW-1 (Table 1) integrated pZW-3                                            glmU locus selected using Southern blot. <br />  <br />                                            pZW-6 rescue plasmid electroporated ZW-1 cells. Transformants grown                                            LB agar plate containing Kan Str, incubated 30&#194;&#176;C. Transformants                                            plated LB agar plate containing Kan, Str 10% sucrose. M. smegmatis ZW-2 <br />                                            (glmU gene knockout strain) (Table 1) confirmed using Southern blot. <br />  <br />                               2.7 MONITORING GROWTH M. SMEGMATIS ZW-2 <br />                                            growth M. smegmatis ZW-2 followed monitoring optical density (OD)                                            600 nm cultures measuring colony forming units (CFU). M. smegmatis ZW-2                                            inoculated 2 tubes containing 5 ml LB broth 0.05% Tween 80 appropriate <br />                                            antibiotics, cells incubated 30&#194;&#176;C 42&#194;&#176;C respectively. M. smegmatis mc2155 <br />                                            M. smegmatis mc2155 pZW-6 used controls. OD600 detected                                            interval 24 hours growth curves 30&#194;&#176;C 42&#194;&#176;C obtained. <br />  <br />                                            measure CFU M. smegmatis ZW-2 cells grown LB broth containing 0.05% Tween <br />                                            80 Kan 30&#194;&#176;C OD600 reached 0.10, cells transferred 42&#194;&#176;C <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            incubator. OD600 cells grown 42&#194;&#176;C determined interval 24 hours. <br />                                            CFU determinations (taken time points), dilutions spread LB agar <br />                                            plates containing Str Kan. plates incubated 30&#194;&#176;C CFU counted. <br />  <br />                               2.8 DETECTION M.TUBERCULOSIS GLMU PROTEIN M. SMEGMATIS ZW-2 <br />                                            M. smegmatis ZW-2 cells grown 30&#194;&#176;C (OD600 0.10) switched 42&#194;&#176;C incubator. <br />                                            ZW-2 cells kept growing 30&#194;&#176;C controls. cells harvested interval <br />                                            24 hours pellets resuspended lysis prep buffer 1 mM PMSF. cells                                            broken sonication followed centrifugation 20 000 x g 30 minutes. supernatants <br />                                            run 10% SDS-PAGE Western blot performed described 2.3.                                            membrane incubated anti-M. smegmatis GlmU serum mouse,                                            prepared lab (unpublished data), incubated antimouse-IgG-AP conjugate <br />                                            (Sigma). protein band visualized BCIP/NBT solution. <br />  <br />                               2.9 MORPHOLOGY M. SMEGMATIS ZW-2 42 &#194;&#176;C <br />                                            effects decreasing GlmU activity morphology ZW-2 examined                                            scanning electron microscopy (SEM) time points CFUs determined. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cells washed times 0.1 M Phosphate buffer (pH 7.4) fixed 2.5% <br />                                            glutaraldehyde 1% OsO4 followed dehydration graded series ethanol (20, <br />                                            40, 60, 70, 80, 90, 100%). cell pellet resuspended 100% ethanol, applied silicon <br />                                            wafer slide, gold coated thickness 5 nm. cells examined JSM-6500F <br />                                            scanning electron microscope  Central Instrumentation Facility, Colorado State <br />                                            University) using accelerating voltage 15 kV. <br />  <br />                               3. Results <br />                               3.1. INITIAL IDENTIFICATION GLMU MYCOBACTERIA <br />                                            amino acid sequence encoded M. tuberculosis H37Rv gene Rv1018c expected                                            encoded GlmU homology bacterial GlmU proteins. sequence                                            used query BLASTP identify homologous gene M. smegmatis mc2155 <br />                                            genome. M. smegmatis glmU gene (MSMEG_5426) 75% identical <br />  <br />                                            Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                              Page 7 <br />  <br />  <br />                                           homology Rv1018c. sequence alignment mycobacterial GlmU proteins <br />                                           E. coli GlmU protein shown Fig. 2. confirm presence GlmU M. <br />                                           smegmatis, conversion GlcNH2-1-P GlcNAc-1-P, N-acetylase activity GlmU <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           shown occur crude M. smegmatis extracts (Fig. 3c) using GC/MS method shown <br />                                           Fig. 3a. <br />  <br />                               3.2 EXPRESSION PURIFICATION M. TUBERCULOSIS GLMU PROTEIN <br />                                           M. tuberculosis glmU gene cloned NdeI XhoI sites pET16b yielding <br />                                           N-terminal histidine tag GlmU fusion protein expected molecular weight                                           54.10 kD. M. tuberculosis GlmU expressed soluble protein E. coli BL21(DE3) <br />                                           induction 0.1 mM IPTG room temperature purified Ni2+ affinity <br />                                           chromatography. Purified M. tuberculosis GlmU protein determined expected <br />                                           size SDS-PAGE Western blot analysis (Fig. 4). <br />  <br />                               3.3 ACETYL TRANSFERASE URIDYLYLTRANSFERASE ACTIVITIES M. <br />                               TUBERCULOSIS GLMU PROTEIN <br />                                           acetyl transferase purified M. tuberculosis GlmU protein demonstrated using                                           GC/MS assay shown Fig. 3d. uridylyltransferase activity shown reverse <br />                                           direction UDP-GlcNAc converted UTP GlcNAc-1-P. incubation                                           UDP-GlcNAc PPi 30 minutes lead complete disappearance UDP-GlcNAc (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           5c) control reaction (Fig. 5b), PPi included reaction mixture, <br />                                           lead reaction. <br />  <br />                               3.4 GENERATION M. SMEGMATIS GLMU GENE KNOCKOUT STRAIN ZW-2 <br />                                           M. smegmatis mc2155 transformants pZW-3 selected LB agar plates containing <br />                                           Kan Gen 30&#194;&#176;C colonies turned yellow catechol sprayed plates. <br />                                           yellow colonies propagated LB broth containing 0.05% Tween 80, Kan Gen <br />                                           30&#194;&#176;C, spread LB agar plates containing Kan Gen 42&#194;&#176;C. pZW-3 <br />                                           plasmid unable replicate 42&#194;&#176;C, Kan resistant colonies plates necessarily <br />                                           integrated glmU::kanR glmU locus M. smegmatis mc2155 genome. M. <br />                                           smegmatis transformant arose single homologous recombination event                                           glmU locus (ZW-1) confirmed Southern blot. <br />  <br />                                           rescue plasmid pZW-6 electroporated M. smegmatis ZW-1, generation                                           M. smegmatis ZW-2 (glmU knockout strain)  grow LB agar containing Kan 10% <br />                                           sucrose) selected  M. smegmatis ZW-2 strains confirmed Southern blot <br />                                           (Fig. 6). strains showed expected bands 1.0 kb (faint), 1.59 kb 1.79 kb. band <br />                                           1.34 kb missing, confirming loss intact glmU gene. bands 0.65 kb, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           0.80 kb 5.3 kb pZW-6 plasmid. <br />  <br />                               3.5 ESSENTIALITY GLMU GENE MYCOBACTERIAL GROWTH <br />                                           order determine GlmU required mycobacterial growth, growth curves                                           M. smegmatis ZW-2 30&#194;&#176;C 42&#194;&#176;C obtained (Fig. 7A). results clearly showed <br />                                           M. smegmatis ZW-2 grew 30&#194;&#176;C, 42&#194;&#176;C pZW-6                                           replicate. contrast, M. smegmatis mc2155 M. smegmatis mc2155 containing pZW-6 <br />                                           grew 30&#194;&#176;C 42&#194;&#176;C.  results showed M. tuberculosis glmU gene <br />                                           essential mycobacterial growth. <br />  <br />                                           M. smegmatis ZW-2 grown 30&#194;&#176;C 0.10 OD600, culture switched                                           incubator 42&#194;&#176;C. CFU OD600 time obtained shown Fig. 7B.                                           functional GlmU protein produced M. smegmatis ZW-2 cells 30&#194;&#176;C, cells <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                 Page 8 <br />  <br />  <br />                                           multiplied 24 hours temperature shift (Fig. 7B). dramatically,                                           increased incubation time 42&#194;&#176;C, CFUs dropped GlmU decreased. <br />                                           Western blot data (Fig. 7C) confirmed expected loss GlmU protein, 1 day <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           insufficient GlmU detection western blot seen (Fig 7C lane 3). <br />  <br />                               3.6 MORPHOLOGICAL CHANGES M. SMEGMATIS ZW-2 42&#194;&#176;C <br />                                           UDP-GlcNAc important precursor peptidoglycan mycobacterial cell wall, <br />                                           expected change morphology possibly lysis bacteria biosynthesis <br />                                           decreased.  temperature switched 30&#194;&#176;C 42&#194;&#176;C, morphology <br />                                           M. smegmatis ZW-2 grown 42&#194;&#176;C 24, 48, 72, 96 hours observed SEM. <br />                                           morphology M. smegmatis ZW-2 30&#194;&#176;C just switching 42&#194;&#176;C (Fig. 8A)                                           indistinguishable M. smegmatis mc2155 wild type (Fig. 8G). time, M. smegmatis <br />                                           ZW-2 cells 42&#194;&#176;C dramatically shorter rounder (Fig. 8B, C D). M. <br />                                           smegmatis ZW-2 cells 42&#194;&#176;C 96 hours showed evidence lysis (Fig. 8E F).                                           SEM results suggest lack GlmU activity result profound negative effects                                           bacterial cell morphology. <br />  <br />                               3.7 DEVELOPMENT MICROTITER PLATE ASSAY GLMU <br />                                           Given desirability GlmU drug target designed microtiter plate based assay <br />                                           detect activity. assay (Fig. 1) relies enzymes MurA MurB convert <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           UDP-GlcNAc UDP-MurNAc concomitant reduction NADPH. E. coli <br />                                           MurA MurB cloned expressed E coli resulting proteins purified                                           described material methods. evident Fig. 9, enzymes substrates <br />                                           present NADPH oxidized concomitant decrease OD340. <br />  <br />                               4. Discussion <br />                                           UDP-GlcNAc essential precursor peptidoglycan rhamnose-GlcNAc linker region <br />                                           mycobacterial cell wall. pathway UDP-GlcNAc biosynthesis significantly <br />                                           different eukaryotes prokaryotes. eukaryotes, N-acetylation occurs GlcNH2-6-P <br />                                           GlcNH2-1-P. second reaction, formation UDP-GlcNAc UTP                                           GlcNAc-1-P, prokaryotes eukaryotes. eukaryotes, uridylyltransferase <br />                                           activity carried distinct monofunctional enzymes little sequence homology                                           GlmU.  tertiary structure active site eukaryotic enzymes responsible                                           uridylyltranserase activity similar uridylyltransferase active site prokaryotic <br />                                           enzymes (Peneff et al., 2001a; Peneff et al., 2001b). Given similarity, acetylation <br />                                           reaction considered attractive drug target bacteria. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           shown M. tuberculosis GlmU protein distinct activities; <br />                                           glucosamine-1-phosphate acetyltransferase N-acetylglucosamine-1-phosphate <br />                                           uridylyltransferase. mycobacteria, GlmU enzyme clearly essential,                                           knocked M. smegmatis grow presence rescue plasmid containing                                           M. tuberculosis glmU gene, grow rescue plasmid does replicate. <br />                                           Insertional mutagenesis studies using transposon site hybridization (TraSH) methodology <br />                                           reported M. tuberculosis glmU essential gene (Sassetti et al.2003);                                           extended determine fate bacteria gene turned (Fig 7 &amp; Fig 8).                                           experimental approach somewhat limited GlmU enzyme abruptly turned <br />                                            replication rescue plasmid containing functional M. tuberculosis <br />                                           glmU gene stopped. results decrease GlmU present cell                                           time. SEM data showed dramatic change morphology including appeared lysis <br />                                           cells latest time point. cells cleanly broken half happens <br />                                           &#206;&#178;-lactams, cells appear ruptured (Fig. 8E F).   <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                        Page 9 <br />  <br />  <br />                                           essentiality GlmU effect loss activity mycobacterial cells strongly <br />                                           support development target new TB drugs. Potentially, new drugs                                           similar bactericidal properties &#206;&#178;-lactams. assay developed microtiter plates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           allows inhibitors acetyl transferase uridylyltransferase screened <br />                                            <br />  <br />                                           Inhibitors GlmU important terms inhibiting linker region                                           mycobacterium cell wall. previously demonstrated effects knocking                                           rhamnosyl formation enzymes, RmlD (Ma et al., 2002) Rml B RmlC (Li et al., <br />                                           2006). Assays inhibitors RmlC RmlD proceeding NIH Roadmap screening <br />                                           center. work validates develops techniques appropriate similar screening                                           GlmU inhibitors. <br />  <br />                               Acknowledgement <br />                                           work supported funds provided National Basic Research Program China (2006CB504400), <br />                                           National Natural Science Foundation China (30670454) National Institute Health (USA) (AI 33706). <br />  <br />  <br />                               References <br />                                           Dye C, Scheele S, Dolin P, Pathania V, Raviglione MC. Global Surveillance Monitoring <br />                                              Project. Consensus statement. Global burden tuberculosis: estimated incidence, prevalence, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              mortality country. Journal American Medical Association 1999;282:677&#226;&#8364;&#8220;686. <br />                                           Gandhi NR, Moll  Sturm AW, Pawinski R, Govender T, Lalloo U, Zeller K, Andrews J, Friedland G. <br />                                              Extensively drug-resistant tuberculosis cause death patients infected tuberculosis <br />                                              HIV rural area South Africa. Lancet 2006;368:1575&#226;&#8364;&#8220;1580. [PubMed: 17084757] <br />                                           Guilhot C, Otal  Van Rompaey  Martin C, Gicquel B. Efficient transposition mycobacteria: <br />                                              construction Mycobacterium smegmatis insertional mutant libraries. Journal Bacteriology <br />                                              1994;176:535&#226;&#8364;&#8220;539. [PubMed: 8288551] <br />                                           Holtje JV. growth autolysis: <span id='am-108' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-109' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-110' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="PGN" datatype="xsd:string"></span>murein</span> hydrolases Escherichia coli. Archives Microbiology <br />                                              1995;164:243&#226;&#8364;&#8220;254. [PubMed: 7487333] <br />                                           Jackson M, Crick DC, Brennan PJ. <span id='am-16' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-17' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span>Phosphatidylinositol</span> essential <span id='am-11' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-12' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>phospholipid</span> mycobacteria. <br />                                              Journal Biological Chemistry 2000;275:30092&#226;&#8364;&#8220;30099. [PubMed: 10889206] <br />                                           Kostrewa D, D&apos;Arcy  Takacs B, Kamber M. Crystal structures Streptococcus pneumoniae N- <br />                                              acetylglucosamine-1- phosphate uridyltransferase, GlmU, apo form 2.33 resolution                                              complex UDP-N-acetylglucosamine Mg(2+) 1.96 resolution. Journal Molecular Biology <br />                                              2001;305:279&#226;&#8364;&#8220;289. <br />                                           Lee  Wu SW, Scherman MS, Torrelles JB, Chatterjee D, McNeil MR, Khoo KH. Sequencing                                              oligoarabinosyl units released mycobacterial arabinogalactan endogenous arabinanase: <br />                                              identification distinctive novel structural motifs. Biochemistry 2006;45:15817&#226;&#8364;&#8220;15828. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              [PubMed: 17176104] <br />                                           Li W, Xin Y, McNeil MR, Ma Y. rmlB rmlC genes essential growth mycobacteria. <br />                                              Biochemical Biophysical Research Communications 2006;342:170&#226;&#8364;&#8220;178. [PubMed: 16472764] <br />                                           Ma Y, Pan F, McNeil MR. Determination <span id='am-52' about='obo:IMR_0200338' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="rdfs:label" content="dTDP" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00363" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200338" datatype="xsd:string"></span>dTDP</span>-rhamnose formation essential growth                                              mycobacteria. J.Bacteriol 2002;184:3392&#226;&#8364;&#8220;3395. [PubMed: 12029057] <br />                                           Mengin-Lecreulx D, Vanheijenoort J. Copurification glucosamine-1-phosphate acetyltransferase                                                N-acetylglucosamine-1-phosphate uridylyltransferase activities Escherichia coli: characterization <br />                                                glmU gene product bifunctional enzyme catalyzing subsequent steps pathway <br />                                                UDP-N-acetylglucosamine synthesis. Journal Bacteriology 1994;176:5788&#226;&#8364;&#8220;5795. [PubMed: <br />                                                8083170] <br />                                           McNeil MR, Brennan PJ. Structure, function, biogenesis cell envelope mycobacteria                                                relation bacterial physiology, pathogenesis, drug resistance; thoughts possibilities <br />                                                arising recent structural information. Research Microbiology 1991;142:451&#226;&#8364;&#8220;463. [PubMed: <br />                                                1871433] <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                                     Page 10 <br />  <br />  <br />                                           O&apos;Brien RJ, Nunn PP. need new drugs tuberculosis, Obstacles, opportunities,                                               steps. American Journal Respiratory Critical Care Medicine 2001;163:1055&#226;&#8364;&#8220;1058. [PubMed: <br />                                               11316634] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Olsen LR, Roderick SL. Structure Escherichia coli GlmU pyrophosphorylase acetyltransferase <br />                                               active sites. Biochemistry 2001;40:1913&#226;&#8364;&#8220;1921. [PubMed: 11329257] <br />                                           Olsen LR, Vetting MW, Roderick SL. Structure E. coli bifunctional GlmU acetyltransferase active <br />                                               site substrates products. Protein Science 2007;16:1230&#226;&#8364;&#8220;1235. [PubMed: 17473010] <br />                                           Pelicic V, Jackson M, Reyrat JM, Jacobs WR Jr, Gicquel B, Guilhot C. Efficient allelic exchange                                               transposon mutagenesis Mycobacterium tuberculosis. Proceedings National Academy                                               Sciences 1997;94:10955&#226;&#8364;&#8220;10960. <br />                                           Peneff C, Mengin-Lecreulx D, Bourne Y. crystal structures Apo complexed Saccharomyces <br />                                               cerevisiae GNA1 shed light catalytic mechanism amino-sugar N-acetyltransferase. <br />                                               Journal Biological Chemistry 2001;276:16328&#226;&#8364;&#8220;16334. [PubMed: 11278591] <br />                                           Peneff C, Ferrari P, Charrier V, Taburet Y, Monnier C, Zamboni V, Winter J, Harnois M, Fassy F, Bourne <br />                                               Y. Crystal structures human pyrophosphorylase isoforms complexes UDPGlc(Gal) <br />                                               NAc: role alternatively spliced insert enzyme oligomeric assembly active site <br />                                               architecture. EMBO Journal 2001;20:6191&#226;&#8364;&#8220;6202. [PubMed: 11707391] <br />                                           Peterson LR. Squeezing antibiotic balloon: impact antimicrobial classes emerging resistance. <br />                                               Clinical Microbiology Infection 2005;11:4&#226;&#8364;&#8220;16. [PubMed: 16138814] <br />                                           Riley M, Abe T, Arnaud MB, Berlyn MK, Blattner FR, Chaudhuri RR, Glasner JD, Horiuchi T, Keseler <br />                                               IM, Kosuge T, Mori H, Perna NT, Plunkett G III, Rudd KE, Serres MH, Thomas GH, Thomson NR, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Wishart D, Wanner BL. Escherichia coli K-12: cooperatively developed annotation snapshot-2005. <br />                                               Nucleic Acids Research 2006;34:1&#226;&#8364;&#8220;9. [PubMed: 16397293] <br />                                           Sambrook, J.; Russell, DG. Molecular Cloning: Laboratory Manual. 3 rd ed. New York: Cold Spring <br />                                               Harbor Laboratory Press; 2001. <br />                                           Sassetti CM, Boyd DH, Rubin EJ. Genes required mycobacterial growth defined high density <br />                                               mutagenesis. Molecular Microbiology 2003;48:77&#226;&#8364;&#8220;84. [PubMed: 12657046] <br />                                           Van Rie  Enarson D. XDR tuberculosis: indicator public-health negligence. Lancet <br />                                               2006;368:1554&#226;&#8364;&#8220;1556. [PubMed: 17084741] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                        Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 1. <br />                                             Biosynthetic pathway UDP-GlcNAc formation mycobacteria. GlmU (Rv1018c)                                           bifunctional enzyme glucosamine-1-phosphate acetyl transferase N- <br />                                           acetylglucosamine-1-phosphate uridylyl transferase activities. (B) reactions MurA                                           MurB act UDP-GlcNAc product GlmU used linked assay                                           loss NADPH monitored. bacterium, product, UDP-MurNAc, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           converted peptidoglycan. <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                             Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 2. <br />                                           Sequence alignment glmU E. coli, M. tuberculosis M. smegmatis. symbol &#226;&#8364;&#339;.&#226;&#8364;? <br />                                           indicates breaks inserted order obtain best alignment. <br />                                           sequence underlines indicate uridylyl transferase active site (Kostrewa et al., 2001). <br />                                           active site acetylation involves amino acids adjacent, interacting beta <br />                                           helices depicted follows: bold letters Acetyl Coenzyme binding amino <br />                                           acids, bold italics letters amino acids bind glucosamine-1-phosphate <br />                                           (Olsen et al., 2007). numbering end line sequences refers                                           amino acid number species&#226;&#8364;&#8482; sequence. Consensus 1 shows homology M. <br />                                           tuberculosis M. smegmatis sequences (second lines) compared                                           Consensus 2 shows homology sequence organisms   second,                                           lines) compared. symbols used Consensus 1 Consensus 2 follows: <br />                                           &#226;&#8364;&#339;.&#226;&#8364;? means consensus  upper case abbreviation amino acid means                                           perfect consensus present; lower case abbreviation amino acid means imperfect <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           consensus amino acid present sequence. symbols <br />                                           indicate closely related amino acids  &#226;&#8364;&#339;!&#226;&#8364;? V  <br />                                           &#226;&#8364;&#339;$&#226;&#8364;? L M  &#226;&#8364;&#339;%&#226;&#8364;? F Y  &#226;&#8364;&#339;#&#226;&#8364;? N, D, Q,                                           E  <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                            Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 3. <br />                                           Determination GlmU acetyl transferase activity.   preparation tetra-O- <br />                                           perdeuterioacetyl-N-perdeuterioacetyl derivative starting GlcNH2-1-phosphate                                           tetra-O-perdeuterioacetyl-N-acetyl derivative product GlcNAc-1-phosphate. D6Ac2O = <br />                                           hexadeuterioacetic anhydride. (B) M+H-63 ion (63 weight trideuterio <span id='am-163' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-164' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span>) <br />                                           detected GC/MS (CI mode) analysis derivatization reactants incubated buffer <br />                                           enzyme (control reaction). (C) M+H-63 ion detected GC/MS (CI mode) analysis <br />                                           derivatization reactants incubated buffer crude M. smegmatis enzymes. (D) <br />                                           M+H-63 ion detected GC/MS (CI mode) analysis derivatization reactants <br />                                           incubated buffer M. tuberculosis GlmU expressed E. coli. C D starting <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                        Page 14 <br />  <br />  <br />                                           material fully converted product. acetyl transferase activity incubations                                           200 &#194;&#181;l 10 mM Tris buffer (0.1 mM MgCl2 0.01% Triton X 100), pH 8 containing 100 <br />                                           &#194;&#181;M acetyl CoA glucosamine-1-phosphate. 100 ng purified M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tuberculosis GlmU (D), 0.5 &#194;&#181;g crude M. smegmatis extract (C), enzyme (B)                                           added reactions incubated 30 minutes room temperature derivatization. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 4. <br />                                           Analysis purified M. tuberculosis GlmU protein SDS-PAGE   Western blot <br />                                           (B). Lane 1. PageRuler prestained protein ladder (Fermentas,  170, 130, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           100, 70, 55, 40, 35, 25 kD); Lane 2. purified M. tuberculosis GlmU protein  histidine tag) <br />                                           (5 &#194;&#181;g) expected molecular weight 54.10 kD. (anti)-polyhistidine clone 1 <br />                                           antibody 1 6000 dilution used Western blot analysis (B). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                      Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           uridylyl transferase activity GlmU enzyme assayed HPLC.   UDP-GlcNAc <br />                                           standard, (B) UDP-GlcNAc GlmU PPi, (C) UDP-GlcNAc GlmU                                           PPi. enzyme reactions incubated 37&#194;&#176;C 30 minutes 40 &#194;&#181;l 50 mM Tris-HCl <br />                                           (pH 8.2) containing 1 mM MgCl2, 25 &#194;&#181;M UDP-GlcNAc, 1 mM pyrophosphate (PPi)  C <br />                                           , 42 &#194;&#181;g/ml GlmU protein. <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 6. <br />                                           Southern blot analysis M. smegmatis ZW-2 strains. <br />                                             expected DNA fragments M. smegmatis mc2155. (B) expected DNA fragments <br />                                           M. smegmatis ZW-2. (C) Confirmation M. smegmatis ZW-2 strains Southern blot. <br />                                           Lanes 1&#226;&#8364;&#8220;6. M. smegmatis ZW-2 strains hybridized DNA bands 1.0 kb, 1.59 kb, <br />                                           1.79 kb; Lane 7. pZW-6 hybridized DNA bands 0.65 kb, 0.8 kb 5.3 kb; Lane <br />                                           8. M. smegmatis mc2155 wild type hybridized DNA bands 1.34 kb 1.79 kb. (D) <br />                                           map plasmid pCG76-Mtb glmU (pZW-6) rescue plasmid. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                            Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 7. <br />                                           Growth curves M. smegmatis ZW-2 strain. <br />                                             Growth curves M. smegmatis ZW-2 strain 30&#194;&#176;C 42&#194;&#176;C. M. smegmatis ZW-2                                           30&#194;&#176;C (&#226;&#8211;&#178;); M. smegmatis ZW-2 42&#194;&#176;C (&#226;&#8211;&#161;). M. smegmatis mc2155 30&#194;&#176;C (&#226;&#8212;?); M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           smegmatis mc2155 42&#194;&#176;C (&#226;&#8212;&#8249;). M. smegmatis mc2155 carrying pZW-6 30&#194;&#176;C (&#226;&#8211;&#160;); M. <br />                                           smegmatis mc2155 carrying pZW-6 42&#194;&#176;C (&#226;&#8211;&#161;). (B) CFU growth curve ZW-2.                                           cells grown 30&#194;&#176;C 24 hours transferred 42&#194;&#176;C incubator. CFU                                           OD600 M. smegmatis ZW-2 grown 42&#194;&#176;C determined interval 24 hours. Log <br />                                           CFUs M. smegmatis ZW-2 (&#226;&#8211;&#178;); OD600 M. smegmatis mc2155 ZW-2 42&#194;&#176;C (&#226;&#8212;?). <br />                                           (C) Western blot GlmU showing loss temperature shift. antibody                                           M. smegmatis GlmU cross reacts M. tuberculosis GlmU. Lanes  1. purified M. <br />                                           tuberculosis GlmU protein  histidine tag) (54.10 kD); 2. PageRuler&#226;&#8222;&#162; Prestained Protein <br />                                           Ladder (Fermentas), MW protein visualized 55 kD; 3, 5, 7, 9. M. smegmatis ZW-2 <br />                                           grown 42&#194;&#176;C 1, 2, 3, 4 days shifting temperature 30&#194;&#176;C 42&#194;&#176;C; 4, 6, 8, 10. <br />                                           M. smegmatis ZW-2 grown 30&#194;&#176;C 1, 2, 3 4 days. molecular weight M. <br />                                           tuberculosis GlmU expressed pCG76-Mtb glmU M. smegmatis ZW-2 50.29 kD. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                       Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 8. <br />                                           Scanning electron micrographs M. smegmatis ZW-2 strain. <br />                                             M. smegmatis ZW-2 cells grown 30&#194;&#176;C 24 hours just transfer 42&#194;&#176;C (10 <br />                                           000X); (B&#226;&#8364;&#8220;E) M. smegmatis ZW-2 cells 42&#194;&#176;C 24; 48, 72 96 hours (10 000X); (F) <br />                                           M. smegmatis ZW-2 cells 42&#194;&#176;C 96 hours  E) higher magnification (30 <br />                                           000X); (G) Wild type M. smegmatis control. Arrows (E) (F) point areas possible <br />                                           lysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Int J Biochem Cell Biol. Author manuscript; available PMC 2009 January 1. <br />                             Zhang et al.                                                                                          Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 9. <br />                                           GlmU, MurA MurB 96 plate assay. <br />                                           enzymatic activity M. tuberculosis GlmU determined 96 plate format <br />                                           utilizing MurA MurB,  E. coli) NADPH. enzymes present (&#226;&#8212;&#352;) <br />                                           A340 drops (average slope &#226;&#710;&#8217;0.0054, standard deviation 0.00013) loss NADPH. <br />                                           substrates omitted reaction (&#206;&#8221; acetyl Coenzyme (average slope <br />                                           &#226;&#710;&#8217;0.0007, standard deviation 0.00033), &#226;&#8212;&#8249; - glucosamine-1-phosphate (average slope <br />                                           &#226;&#710;&#8217;0.0005, standard deviation 0.00023), &#195;&#8212; - phosphoenol pyruvate (average slope &#226;&#710;&#8217;0.0008, <br />                                           standard deviation 0.00018) &#226;&#8211;&#161; - UTP (average slope &#226;&#710;&#8217;0.0006, standard deviation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           0.00017)) A340 does decrease. Similar lack activity seen (data presented                                           graph) enzymes left  GlmU (average slope &#226;&#710;&#8217;0.00094, <br />                                           standard deviation 0.000012), MurA (average slope &#226;&#710;&#8217;0.00043, standard deviation 0.00012) <br />                                           MurB (average slope &#226;&#710;&#8217;0.0006, standard deviation 0.00019). <br />  <br />  <br />  <br />  <br />                                   <br /> </body></html>