<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.081836 <br />  <br />  <br />  <br /> Genetic Interaction Escherichia coli AcpT Phosphopantetheinyl <br />             Transferase YejM Inner Membrane Protein <br />  <br />                                          Nicholas R. Lay*,1 John E. Cronan&#226;&#8364;&#160;,2 <br />                  *Department Microbiology &#226;&#8364;&#160;Department Biochemistry, University Illinois, Urbana, Illinois 61801 <br />                                                    Manuscript received September 11, 2007 <br />                                                  Accepted publication December 31, 2007 <br />  <br />  <br />                                                               ABSTRACT <br />                Strain LH530, mutant Escherichia coli K-12, reported increased outer membrane <br />              permeability, temperature-sensitive growth, reduced synthesis <span id='am-19' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span>  unmapped mutant gene <br />              suppressed high-copy-number plasmids carrying wild-type acpT gene, encodes              protein catalyzes post-translational protein modification, attachment 49-phosphopantetheine. <br />              mapped strain LH530 mutation gene unknown function, yejM, known encode inner <br />              membrane protein. mutation yejM nonsense mutation produces truncated protein lacking              predicted periplasmic domain. Reconstruction mutation gave strain having phenotypes              LH530. contrast nonsense mutants, deletion entire yejM gene lethal. Suppression AcpT <br />              overexpression yejM nonsense mutants encoding truncated proteins specific AcpT. <br />               AcpT overexpression suppressed lethality deletion entire yejM gene              suppression did require AcpT enzymatically active. mechanism overexpression <br />              specific cytosolic protein bypasses essentiality inner membrane protein unknown. <br />  <br />  <br />  <br /> T     cell envelopes gram-negative bacteria consist <br />        inner membrane, outer membrane, rigid layer <span id='am-55' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-56' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-64' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span>peptidoglycan</span> located                                                                          thesis hypersensitivity hydrophobic large <br />                                                                          hydrophilic antibiotics. mutations occur                                                                          lpxA, lpxC, lpxD genes (Normark 1970; Tsuruoka <br /> membranes. outer membrane provides protec-                         et al. 1988; Galloway Raetz 1990; Kloser et al. 1996; <br /> tive barrier prevents diffusion hydrophobic                  Vaara Nurminen 1999). lpxA lpxD encode large hydrophilic antibiotics cell                  enzymes transfer fatty acid, b-hydroxymyristate, sists inner leaflet composed phospholipids                  nascent lipid species (Crowell et al. 1986; Kelly et al. outer leaflet composed lipopolysaccharides.                    1993) lpxC encodes enzyme deacetylates <br /> structure <span id='am-25' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-26' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span>lipopolysaccharide</span> divided                      precursor lipid (Young et al. 1995). components: lipid  core oligosaccharide,                        Hirvas et al. (1997) isolated E. coli mutant O-antigen. lipid portion lipo-                      strain, strain LH530, hypersensitive hydro- <br /> <span id='am-10' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span>polysaccharide</span> anchors lipopolysaccharide                     phobic large hydrophilic antibiotics showed <br /> outer leaflet outer membrane. synthesis                    temperature-sensitive growth. Strain LH530  <br /> lipid Escherichia coli studied                    ported defective lipid biosynthesis serves model understanding process                  mutant gene mapped isolated.  <br /> (Raetz et al. 2007). lipid molecules synthesized                   attempts isolate responsible gene, E. coli gram-negative bacteria consist                   workers identified gene called ORF195  o195), b-19-6-linked disaccharide N-acetylglucosamine                 called acpT, encodes phosphopantetheinyl modified attachment fatty acids, phos-                      transferase (Lambalot et al. 1996; Flugel et al. 2000; phates, additional saccharides (Raetz et al. 2007).                  Lay Cronan 2006), suppressed temper- <br /> Additional covalent modifications core lipid                    ature-sensitive phenotype mutant strain LH530. <br /> molecule, addition palmitate modi-                    strain LH530 carried wild-type copy acpT <br /> fication 19- 49-phosphates l-4-aminoarabinose             growth 42&#194;&#176; occurred high levels phosphoethanolamine, occur outer mem-                        overexpression (Hirvas et al. 1997).  growth <br /> brane stressed (Raetz et al. 2007).                                   suppression complementation. <br />    Mutations identified lipid bio-                Strain LH530 phenotypes consistent synthetic genes result reduction lipid syn-                 defect lipid synthesis; e., strain leaked peri- <br />                                                                          plasmic enzyme, cytoplasmic enzyme,    1 <br />     Present address: Bldg. 37, Room 5132, National Cancer Institute,     outer membrane contained abnormally low levels Bethesda, MD 20892. <br />    2 <br />                                                                          OmpF porin (Nurminen et al. 1997). <br />     Corresponding author: Department Microbiology, University Illinois, B103 <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> Life Sciences Laboratory, 601 S. Goodwin        studied mutation Ave., Urbana, 61801. E-mail: j-cronan@life.uiuc.edu                   phosphopantetheinyl transferases (Flugel et al. 2000; <br />  <br /> Genetics 178: 1327&#226;&#8364;&#8220;1337 (March 2008) <br />  1328                                                 N. R. Lay J. E. Cronan <br />  <br />                                                                  TABLE 1 <br />                                                         Strains used study <br />  <br />           Strain                                  Relevant features                                 Source reference <br />           CAG12177         MG1655 atoSTTn10                                                    Singer et al. (1989) <br />           DH5a             recA1 endA1                                                         Strain collection <br />           HT253            W3110 pdxJ8TDTn10                                                   Strain collection <br />           JM105            F9 lacIq proAB/endA D(lac-proAB)                                    Yanisch-Perron et al. (1985) <br />           LH530            F9 lacIq proAB/endA D(lac-proAB) Ts                                 Hirvas et al. (1997) <br />           MC1061           araD139 D(ara-leu)7696                                              Strain collection <br />           Top10F9          F9 lacIq Tn10 (TetR)/f80TlacZ DM15 DlacX74 recA1 endA1              Invitrogen (San Diego) <br />           JW0757-1         DaraBAD DlacZ DbioA746Tkan                                          CGSC; Baba et al. (2006) <br />           JW0014-1         DaraBAD DlacZ DdnaJ735Tkan                                          CGSC; Baba et al. (2006) <br />           CY1783           DaraBAD DlacZ bioA746TlacZY                                         study <br />           CY1817           DaraBAD DlacZ DdnaJ735TlacZY                                        study <br />           NRD142           LH530 atoSTTn10                                                     study <br />           NRD148           JM105 atoSTTn10 Ts                                                  study <br />           NRD150           MC1061 DompCTkan                                                    study <br />           NRD151           MC1061 DyojITkan                                                    study <br />           NRD153           MC1061 DrcsCTkan                                                    study <br />           NRD159           MC1061 DatoBTkan                                                    study <br />           NRD161           MC1061 DnapBTkan                                                    study <br />           NRD163           MC1061 DccmHTkan                                                    study <br />           NRD169           MC1061 DyejKTkan                                                    study <br />           NRD178a          MC1061 DyejMTkan                                                    study <br />           NRD183           JM105 DyejMTkan carrying pNRD217                                    study <br />           NRD184           MC1061 yejM569Tkan                                                  study <br />           NRD187           JM105 yejM569Tkan                                                   study <br />             CGSC, Coli Genetic Stock Center (Yale University). <br />                          strain carries chromosomal duplication yejM region  text). <br />  <br />  <br />  Lay Cronan 2006). report mapping                       sensitive phenotype. Strain NRD148 generated trans- mutation responsible temperature-sensitive                  ducing Tn10 strain NRD142 strain JM105                                                                         screening resulting tetracycline transductants strain <br /> phenotype phenotypes strain LH530.                                                                         acquired temperature-sensitive phenotype. mutant strain yejM gene                  DompC strain NRD150 constructed l-Red recombinase- <br /> characterized, including demonstration yejM                    mediated gene replacement (Datsenko Wanner 2000) essential gene.  demonstrate                     using PCR product generated template pKD4 <br /> suppression temperature-sensitive phenotype                   (Datsenko Wanner 2000) OmpCKO                                                                         Rev primers. strains carrying deletions yojI, rcsC, atoB, <br /> strain LH530 overexpression AcpT                                                                         napB, ccmH, yejM, yejK generated l-Red recombi- phosphopantetheinyl transferase activity                nase-mediated gene replacement using PCR product <br /> AcpT overexpression suppresses lethality                    generated template pKD4, using pair primers <br /> yejM null mutants.                                                      named gene (supplemental Table 1S). Strain NRD183 <br />                                                                         constructed transduction kanamycin resistance <br />                                                                         construct strain NRD178 strain JM105 carrying <br />                                                                         plasmid pNRD217. transduction performed                 MATERIALS METHODS                                   presence arabinose, transductants selected LB <br />                                                                         plates containing kanamycin arabinose. wild-type <br />    Bacterial strains plasmids: bacterial strains used        yejM gene modified introducing UGA termination study derivatives strain E. coli K-12.        codon place codon Y190 UGA (opal) codon <br /> E. coli strains listed Table 1 Tn10 insertion     substituting kanamycin resistance cassette yejM <br /> strains used gene mapping, previously                 sequences downstream mutated codon l-Red- <br /> scribed. (Singer et al. 1989; Nichols et al. 1998). plasmids        mediated recombination using PCR product generated used study listed Table 2. primers used      template plasmid pKD4 using primers YejMKO4 study (supplemental Table 1S) purchased Inte-                 YejMKO Rev. Kanamycin-resistant recombinants selected <br /> grated DNA Technologies. Fosmid pCC1Fos obtained                    LB plates containing kanamycin 28&#194;&#176;. Strain NRD187 Epicentre. Site-directed mutagenesis               generated transduction kanamycin resistance  <br /> Quickchange PCR method (Stratagene, La Jolla, CA).                      terminant strain NRD184 strain JM105. lacZY <br />    Strain NRD142 constructed phage P1vir transduc-               transcriptional fusions bioA dnaJ constructed tion Tn10 strain CAG12177 strain LH530                   described Ellermeier et al. (2002) vectors (Miller 1972) followed screening tetracycline-                improved ribosome-binding sites ( J. Slauch, personal com- <br /> resistant transductants maintenance temperature-             munication) used pCY580 (Cronan 2003) used <br />                                                         Phenotype E. coli yejM                                                 1329 <br />  <br />                                                                 TABLE 2 <br />                                                       Plasmids used study <br />  <br /> Plasmid                                         Relevant features                                          Source reference <br /> pBAD24            Ampr, araBAD promoter-based expression vector, pUC-type ori                        Guzman et al. (1995) <br /> pBAD30            Ampr, araBAD promoter-based expression vector, p15a ori                            Guzman et al. (1995) <br /> pBAD322           AmpR, araBAD promoter-based expression vector, complete pBR322 origin              Cronan (2006) <br /> pCR2.1            Ampr, Kanr, cloning vector, pUC ori                                                Invitrogen <br /> pKD4              oriRg, Kanr cassette flanked FRT sites                                          Datsenko Wanner (2000) <br /> pKD46             AmpR, RepA1019(Ts), l, g, b, exo expressed araBAD promotor             Datsenko Wanner (2000) <br /> pQE70-Sfp         Ampr, pQE70-derived plasmid encoding Sfp C-terminal His6 tag                Mofid et al. (1999) <br /> pYon113           Ampr, acpT control araBAD promoter                                 Flugel et al. (2000) <br /> pNRD28            acpS amplified JM105 using primers AcpSBAD Rev                    study <br />                      TOPO cloned pCR2.1 <br /> pNRD83            EcoRI&#226;&#8364;&#8220;HindIII acpS fragment pNRD28 inserted sites              study <br />                      pBAD322 <br /> pNRD192           EcoRI&#226;&#8364;&#8220;XbaI acpS fragment pQE70-Sfp inserted sites              study <br />                      pBAD30 <br /> pNRD203           EcoRI&#226;&#8364;&#8220;HindIII acpS acpS fragment pNRD83 inserted sites            study <br />                      pBAD24 <br /> pNRD204           EcoRI&#226;&#8364;&#8220;XbaI sfp acpS fragment pNRD192 inserted sites            study <br />                      pBAD18 <br /> pNRD205           Encodes AcpT D91E. Site-directed mutation acpT pYon113                   study <br />                      generated using primers AcpTD91E Rev. <br /> pNRD206           Encodes AcpT E137S. Site-directed mutation acpT pYon113                  study <br />                      generated using primers AcpTE137S Rev. <br /> pNRD207           Encodes AcpT K141A. Site-directed mutation acpT pYon113 using                study <br />                      primers AcpTK141A Rev. <br /> pNRD216           yejM amplified JM105 using primers YejM Rev primers               study <br />                      TOPO cloned pCR2.1 <br /> pNRD217           NcoI&#226;&#8364;&#8220;XbaI yejM containing fragment pNRD216 inserted                  study <br />                      sites pBAD322 <br />  <br />  <br />  source Flp recombinase construction. b-       marker replacing yejK, ccmH, napB, yojI, ompC, rcsC, atoB Galactosidase activities determined according Miller          determined. 200 kanamycin-resistant transductants <br /> (1972).                                                               resulting transduction strain LH530 phage P1vir <br />    Culture media growth conditions: Strains grown            lysates grown strains NRD169, NRD163, NRD161, NRD151, LB liquid medium LB agar plates. LB medium               NRD150, NRD153, NRD159 screened tempera- <br /> supplemented final concentrations glucose (0.2%)           ture sensitivity described  closely linked <br /> arabinose (0.08%). Antibiotics used following         marker identified, gene replaced marker,  <br /> centrations: 100 mg liter 1 ampicillin, 50 mg liter 1 kanamy-         gether  2500 bp adjacent genomic DNA cin, 5 mg liter 1 tetracycline.                                   strain LH530 parental strain JM105, sequenced <br />    Gene mapping: mutation conferring temperature                  amplification PCR using YejK, YejL, YejM <br /> sensitivity strain LH530 initially localized E. coli    primers. PCR product directly sequenced K-12 chromosome transduction strain LH530 P1vir            primers used amplification Core Sequencing Unit lysates collection strains harboring Tn10           Keck Center Comparative Functional Genomics sertions spaced E. coli chromosome (Singer et al.          University Illinois. <br /> 1989; Nichols et al. 1998). P1vir lysates nearly strains collection Tn10 insertion 56 min E. coli chromosome. P1vir                                  RESULTS <br /> lysates used transduce Tn10 insertion strains strain LH530. transductants                  Mapping mutation responsible acquired Tn10 selected ability strains      temperature-sensitive phenotype E. coli K-12 grow 28&#194;&#176; LB plates containing tetracycline. 48 102 tetracycline-resistant transductants obtained         strain LH530: Hirvas et al. (1997) previously identified transduction screened loss temperature           temperature-sensitive mutant named strain LH530 <br /> sensitivity streaking transductant LB plates       phenotype similar mutants defec- <br /> taining tetracycline followed incubation plate 28&#194;&#176;       tive lipid synthesis; e., mutant sensitive 42&#194;&#176;. Tn10 insertions cotransduced           hydrophobic large hydrophilic antibiotics  temperature-sensitive mutation strain LH530. Tn10 inserted napA inserted            missive temperatures reported sub- atoS. define location mutation,        normal rates lipid synthesis temperatures. <br /> cotransduction frequencies mutation kanamycin           Hirvas et al. (1997) plasmids carrying gene <br />  1330                                            N. R. Lay J. E. Cronan <br />  <br />  <br />  <br />  <br />    Figure 1. segment E. coli chromosome contains yejM. Transductions using strain LH530 recip- <br /> ient performed using P1vir lysates derivatives strain MC1061, genes denoted solid arrow replaced kanamycin resistance cassette. <br /> Transductants selected LB&#226;&#8364;&#8220;kanamycin plates incu- <br />                                                                    Figure 2. illustration putative structure bated 28&#194;&#176; 2 days. cross 200 kanamy- <br />                                                                  length YejM   truncated form YejM pro- <br /> cin-resistant transductants screened                                                                  duced strain LH530 (B) based data predictions <br /> temperature-sensitive growth. solid arrows <br />                                                                  Daley et al. (2005). YejM inner membrane protein frequency kanamycin marker replacing                                                                  contains putative membrane-spanning helices gene cotransduced gene relieved                                                                  periplasmic C-terminal domain. periplasmic domain <br /> temperature-sensitive phenotype strain LH530. <br />                                                                  YejM predicted sulfatase/phosphatase activity <br />                                                                  UniProtKB/Swiss-Prot database. <br />  <br /> named acpT  called ORF195) suppressed                   resistance cassette inserted place yejK. temperature-sensitive phenotype mutant strain,           amplified genes yejK, yejL, yejM inter- AcpT-encoding plasmids high            genic regions genes strain LH530 <br /> copy number.  acpT gene strain LH530              parental strain JM105 sequenced PCR sequence parental wild-type strain,          products. sequencing data showed single differ- <br /> indicating suppression complementation.              ence genomic sequences: G570A mutation responsible temperature-sensitive           mutation yejM gene strain LH530. G570A <br /> phenotype strain LH530 identified mapped.          mutation resulted replacement UGG codon <br />    mapped mutation responsible tem-               tryptophan 190 UGA termination codon. YejM <br /> perature-sensitive phenotype strain LH530 trans-           previously shown inner membrane <br /> ducing strain LH530 phage P1vir lysates                protein predicted composed transmembrane <br /> collection strains, carried transposon        domain consisting putative membrane-spanning <br /> Tn10 antibiotic resistance element inserted known           helices large periplasmic domain (Daley et al. <br /> chromosomal location (Singer et al. 1989; Nichols                2005) (Figure 2). nonsense mutation trun- <br /> et al. 1998). screening 56 min E. coli     cate expressed protein 190 residues 586 <br /> chromosome, temperature-sensitive mutation                residues cleanly remove predicted peri- <br /> strain LH530  53 29% linked               plasmic domain leaving predicted transmem- <br /> Tn10 insertions napA atoS, respectively (Figure 1).       brane domain intact (Figure 2). resolve location mutation, set              linked yejM mutation strain LH530 nonessential genes located region E. coli      Tn10 insertions napA atoS transduction <br /> chromosome (yejK, ccmH, napB, yojI, ompC, rcsC,              used strains donors transduce <br /> atoB) replaced kanamycin resistance cassettes          strains JM105 MG1655 tetracycline resistance. l-Red recombinase-mediated gene replacement                  resulting transductants screened (Datsenko Wanner 2000) linkage                temperature-sensitive growth. temperature-sensi- <br /> mutation markers determined mea-               tivity profile observed strain NRD187, derivative <br /> suring cotransduction frequencies (Figure 1).                strain JM105 acquired yejM(Ts) muta- <br /> temperature-sensitive mutation strain LH530               tion, identical observed strain LH530 tightly linked (100%) kanamycin             (data shown).  yejM(Ts) <br />                                                    Phenotype E. coli yejM                                               1331 <br />  <br />  <br />  <br />  <br />  Figure 3. growth strains JM105, NRD187,                 Figure 4.&#226;&#8364;&#8221;Complementation temperature-sensitive <br /> LH530 LB plates incubated overnight 28&#194;&#176; 42&#194;&#176;.     strain LH530 plasmid-encoded copy yejM. Strain JM105 <br />                                                                  derived temperature-sensitive mutant, strain LH530, <br />                                                                  carrying vector pBAD322 plasmid containing <br />                                                                  copy yejM control araBAD promoter <br /> derivative strain MG1655 clearly temperature              (pNRD217 called pYejM figure) tested abil- <br /> sensitive, strain showed greater growth 42&#194;&#176;         ity grow 42&#194;&#176; LB plates containing glucose did strain JM105 derivatives (data shown).          arabinose. <br /> difference behavior attributed fact strain MG1655 grew significantly faster strain              resulting plasmid, pNRD217, complement <br /> JM105 faster growth accentuated leakiness           growth strain LH530 42&#194;&#176; plates containing (formation small colonies) shown strain LH530              glucose arabinose. Strain LH530 containing pNRD217 prolonged incubation 42&#194;&#176;.                                grew LB&#226;&#8364;&#8220;arabinose-containing plates, did    Characterization yejM(Ts) mutation:             grow plates containing glucose  represses <br /> firm premature termination codon yejM            expression araBAD promoter) (Figure 4). gene strain LH530 responsible tempera-            introduction plasmid pNRD217 strain NRD187 <br /> ture-sensitive growth strain LH530, used l-Red             similarly allowed growth 42&#194;&#176; LB plates containing <br /> recombinase-mediated gene replacement generate                arabinose containing glucose (data nonsense (G570A) mutation strain LH530             shown). inability recover plasmid-encoding <br /> wild-type gene concomitantly replace yejM             yejM complementation mutation LH530  <br /> sequence downstream mutation kanamycin             dicates clone bank screened Hirvas et al. <br /> resistance cassette. construct transduced          (1997) lacked yejM plasmid presumably bias strain JM105 generate strain NRD187.             construction bank. <br /> compared abilities strains JM105, LH530,                 yejM(Ts) mutation suppressed overexpression <br /> NRD187 grow LB plates incubated 28&#194;&#176; 42&#194;&#176;              active inactive forms AcpT, high- <br /> overnight. strains grew 28&#194;&#176;,          level expression phosphopantetheinyl strain JM105 grew 42&#194;&#176; following overnight             transferases, AcpS Sfp: Hirvas et al. (1997) cubation (Figure 3).  prolonged incuba-             previously shown temperature-sensitive pheno- <br /> tion, strain LH530 formed small colonies 42&#194;&#176;          type strain LH530 suppressed high-copy-number <br /> strain NRD187 did  difference suggested            plasmids carrying acpT, encodes phosphopante- strain LH530 basal readthrough UGA non-                theinyl transferase plays minor role E. coli <br /> sense codon (Bjornsson Isaksson 1993) resulted               K-12 lipid metabolism  Lay Cronan 2006). Phos- low levels length YejM. Strains LH530             phopantetheinyl transferases enzymes transfer <br /> NRD187 essentially identical lipid phenotypes          49-phosphopantetheinyl moiety <span id='am-35' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>    expected previous report (Hirvas         served serine residue carrier protein et al. 1997), strains sensitive           acyl carrier proteins lipid synthesis (Lambalot et al. <br /> rifampicin erythromycin strain JM105.                   1996). Attachment prosthetic group mandatory <br />    performed complementation experiments                   thiol group phosphopantetheinyl <br /> assess expression wild-type copy          moiety carries intermediates fatty acid biosynthe- <br /> yejM native promoter cosmid vector          sis. tested ability phosphopantetheinyl araBAD promoter multicopy plasmid                transferases, AcpT, AcpS, Sfp, suppress allowed growth strain LH530 42&#194;&#176;. fragment             temperature-sensitive phenotype LH530  <br /> DNA contained yejM, upstream              expression. E. coli AcpS trimeric phosphopante- downstream genes yejL proL, respectively,            theinyl transferase catalyzes modification acyl <br /> amplified strain JM105. genome segment             carrier proteins fatty acid biosynthesis (Flugel et al. inserted single-copy F-factor-based cosmid         2000) Sfp monomeric Bacillus subtilis  <br /> pCC1Fos. presence construct strain LH530          zyme catalyzes phosphopantetheinylation carrier <br /> allowed growth nonpermissive temperature (data            proteins involved secondary metabolic processes shown). placed yejM gene               nonribosomal <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> polyketide syntheses <br /> araBAD promoter pBAD322 tested ability                (Lambalot et al. 1996; Mootz et al. 2001). previously <br />  1332                                                  N. R. Lay J. E. Cronan <br />  <br />                                                                         2000; Lay Cronan 2006). high-level ex- <br />                                                                         pression, mutant acpT alleles lacked ability <br />                                                                         functionally replace AcpS wild-type allele <br />                                                                         showed expected strong complementation (Flugel <br />                                                                         et al. 2000; Lay Cronan 2006) (data shown). <br />                                                                          plasmids encoding wild-type                                                                         mutant AcpT proteins suppressed temperature- <br />                                                                         sensitive phenotype strain LH530 (Figure 5).  <br />                                                                         fore, suppression yejM nonsense mutations did                                                                         require AcpT enzymatically active. <br />                                                                            yejM essential gene, essentiality                                                                         suppressed AcpT overexpression: temperature- <br />                                                                         sensitive phenotype strain LH530,                                                                         production truncated form YejM lacking                                                                         C-terminal periplasmic domain, raised question                                                                         effects deletion entire yejM gene. Repeated <br />    Figure 5.&#226;&#8364;&#8221;Suppression analyses strain LH530 plasmids           attempts delete yejM strain MC1061 l-Red <br /> encoding diverse phosphopantetheinyl transferases             recombinase-mediated gene replacement unsuc- <br /> site-directed mutants AcpT phosphopantetheinyl transfer-         cessful despite ready success deleting genes <br /> ase.   Derivatives strain LH530 carrying vector pBAD24         method. eventually isolated single yejM plasmid expressed sfp, acpS, acpTunder control araBAD promoter (pNRD204, pNRD83, pYon113, respec-               deletion candidate, strain NRD178. confirmed tively) tested ability grow 42&#194;&#176; LB plates     presence yejM deletion strain using PCR <br /> taining glucose inducer, arabinose. (B) ability wild-     primers yielded product yejM deletion <br /> type AcpT site-directed mutants AcpT elim-        present.  difficulty experienced <br /> inate phosphopantetheinyl transferase activity vivo (Table 2)        construction strain suggested yejM tested ability suppress defective growth strain <br /> LH530 42&#194;&#176; LB plates containing glucose arabi-           deletion strain second characteristic nose.                                                                   allowed viability deletion strain.                                                                         transduced yejM deletion derivative strain <br />                                                                         JM105 carried second plasmid-borne copy yejM <br /> reported Hirvas et al. (1997), high-level overexpres-                control araBAD promoter. Transduction sion AcpT suppressed temperature-sensitive                       performed presence arabinose trans- <br /> phenotype strain LH530 (Figure 5) low levels                 duction mixtures plated LB plates containing overexpression did (data shown).                     arabinose kanamycin. transductants tested overexpression AcpS high-level expres-                   ability grow LB plates containing glucose <br /> sion Sfp suppressed temperature-sensitive phe-                    expression plasmid-borne yejM repressed) <br /> notype strain LH530 (Figure 5). Similar results                 arabinose  expression plasmid-borne yejM <br /> obtained strain NRD187, carries vitro-                 induced). transductants grew LB&#226;&#8364;&#8220;arabinose <br /> constructed yejM nonsense allele (data shown).                      plates LB&#226;&#8364;&#8220;glucose plates (Figure 6). Similar <br />    lack suppression temperature-sensitive                 results obtained experiment  <br /> phenotype strain LH530 high-level expression                   formed MG1655 DyejM strain carrying pNRD217, <br /> AcpS Sfp suggested phosphopantetheinyl                      yejM plasmid (data shown).  trans- <br /> transferase activity AcpT required sup-                  duction resolved DyejM allele pression.  <span id='am-48' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-49' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-50' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> substitutions                    character present original strain, NRD178. key AcpT residues (D91E, E137S, K141A)                        analysis genome original yejM <br /> conserved phosphopantetheinyl transferases                        deletion strain NRD178, using PCR primers gener- <br /> (Lambalot et al. 1996) involved                       ated products different lengths depending binding CoA substrate Sfp AcpS (Reuter                    presence absence wild-type copy yejM, demon- <br /> et al. 1999; Parris et al. 2000; Mofid et al. 2004).                strated original yejM deletion strain, strain <br /> AcpT amino acid substitutions substitutions pre-                   NRD178, contained duplication region viously shown eliminate Sfp phosphopantetheinyl                      genome includes yejM. duplicated region  <br /> transferase activity (Mofid et al. 2004)                   tained copy wild-type yejM plus copy DyejM <br /> structed pBAD24 vector.  agreement                 allele (data shown). attribute chromosomal prior report (Lambalot et al. 1996),                    arrangement strain NRD178 l-Red-mediated  overexpression, AcpT inactive largely                      combination copy preexisting chromosomal <br /> insoluble, assayed activities proteins               duplication yejM region present cell  abilities plasmids support growth strain             dergoing recombination event. spontaneous <br /> carrying conditional mutation acpS (Flugel et al.                  duplications bacterial chromosomes docu- <br />                                                    Phenotype E. coli yejM                                              1333 <br />  <br />  <br />  <br />  <br />                                                                     Figure 7.&#226;&#8364;&#8221;Suppression lethality yejM deletion <br />                                                                  strain overexpression AcpT. yejM deletion strain <br />                                                                  NRD178 transduced phage P1vir strain JM105 <br />                                                                  containing plasmid expressed yejM (pNRD217) <br />                                                                  acpT (pYon113) arabinose-inducible araBAD pro- <br />                                                                  moter. Transductants acquired yejM deletion se- <br />                                                                  lected LB&#226;&#8364;&#8220;kanamycin&#226;&#8364;&#8220;arabinose plates. transductant <br />    Figure 6.&#226;&#8364;&#8221;Determination essentiality yejM.          cross, parental strains, Strains JM105, LH530, NRD183, derivative strain         streaked plates LB&#226;&#8364;&#8220;glucose LB&#226;&#8364;&#8220;arabinose JM105 containing deletion yejM carries plasmid         incubated overnight 30&#194;&#176;. <br /> pNRD217 encoding yejM (called pyejM   <br /> trol araBAD promoter, tested ability grow overnight 28&#194;&#176; 42&#194;&#176; LB plates containing      null mutation did require AcpT enzymatic activity. <br /> arabinose glucose.                                            yejM null mutation transduced                                                                  derivative JM105 expressing inactive E137S AcpT <br />                                                                  encoded plasmid pNRD206, growth proceeded mented (Roth et al. 1996).  F-factor-based           seen wild-type AcpT (Figure 7). stain expressing <br /> cosmid generated strain NRD178 genomic DNA                  E137S AcpT grew presence arabinose, <br /> contained genome fragment spanned region              failed grow expression blocked ab- fruA ccmF (including wild-type copy             sence arabinose  presence absence <br /> yejM) lacked kanamycin-marked DyejM deletion.            glucose) (data shown). cosmid complemented DyejM deletion                    Does AcpT overexpression induce heat-shock <br /> strain NRD186 (data shown). suggests          response? possibility suppression yejM <br /> duplicated region include genes         mutations AcpT overexpression overexpres- <br /> fruA ccmF, consistent sizes docu-           sion triggers stress response induces synthesis mented duplications (Roth et al. 1996).                          protein replaces  bypasses) YejM function.    test yejM essential, DyejM     stress response known induced high-level ex- <br /> deletion constructed l-Red recombinase-                 pression insoluble proteins AcpT heat- <br /> mediated gene replacement strain DY330 (Yu et al.             shock response.  asked heat-shock <br /> 2000) carrying yejM plasmid pNRD217.              regulon induced AcpT overexpression assay- <br /> recombinants obtained shown contain yejM             ing expression classical heat-shock protein, DnaJ, <br /> deletion PCR recombinants               use chromosomal dnaJ-lacZY fusion construct. shown Figure 6) grew induction            Construction fusion eliminated DnaJ function, <br /> yejM expression plasmid.                                DnaJ functionally replaced    Given yejM essential growth,          cochaperone proteins, DjlA CbpA (Gur et al. <br /> protein lacking periplasmic domain allows growth             2004), DnaJ mutants behave normally regards heat low-growth temperatures, question raised           shock. Use fusion provides assay DnaK AcpT overexpression suppress            expression encoded promoter-proximal <br /> lethal phenotype strains carrying total deletion         gene dnaK-dnaJ operon (Cowing et al. 1985; yejM gene. DyejM deletion mutation         Bardwell et al. 1986). readily transduced derivative JM105              Following construction dnaJ-lacZY fusion strain <br /> carried multicopy plasmid acpT expression           CY1817, behavior strain tested classical inducible (araBAD) promoter. strain grew            30&#194;&#176;&#226;&#8364;&#8220;42&#194;&#176; temperature shift. strain showed inducing conditions conditions             expected rapid increase dnaK-dnaJ expression   <br /> pressed expression plasmid-encoded acpT (Figure           sayed b-galactosidase activity) soon tempera- <br /> 7). observed growth similar given        ture shift reached plateau threefold greater <br /> induction plasmid-borne copy yejM AcpT          30&#194;&#176; expression level  30 min shift (Figure <br /> overproduction fully suppressed effects oth-         8A) expected increases DnaK levels seen <br /> erwise lethal mutation. reported suppres-           temperature-shift regimen (Gross 1996). sion temperature sensitivity strain                control, lacZY fusion housekeeping biosynthetic <br /> expressing truncated YejM, suppression yejM           gene, bioA, assayed parallel. contrast dnaJ <br />  1334                                          N. R. Lay J. E. Cronan <br />  <br /> fusion, bioATlacZY fusion strain showed effect       bypass loss YejM. Strain NRD183, contains temperature; b-galactosidase activity constant         complete deletion yejM plasmid encoding AcpT <br /> (Figure 8A).                                                   arabinose control, grown presence    dnaJ-lacZY fusion strain CY1817 trans-         arabinose 30&#194;&#176; cells early log phase cultures <br /> formed plasmids encoding wild type, E137S         recovered centrifugation. cells mutant AcpT, vector. Cultures            washed suspended medium lacking arabi- <br /> strains grown 30&#194;&#176;, protein expression induced         nose. culture divided halves. half 0.2% arabinose, b-galactosidase activities       received 0.2% arabinose, second received water. <br /> measured. low-growth temperature chosen                cultures incubated 30&#194;&#176; decrease heat-shock protein expression levels           growth followed turbidity. growth increase sensitivity assay.       culture lacking arabinose began slow depletion <br /> cultures showed constant level expression 90-      AcpT, culture split half supple- <br /> min time course (Figure 8B), indicating induction        mented arabinose. time required growth heat-shock response. possible caveat ex-         arabinose-supplemented culture, accelerated rel- <br /> periment time course brief allow     ative culture lacking arabinose, taken induction heat-shock regulon, induction        measure time required accumulation AcpT heat-shock response expression            levels sufficient bypass yejM deletion. Acceleration <br /> insoluble proteins seen roughly comparable          growth addition arabinose seen time intervals (Lesley et al. 2002; Smith 2007).       36 min inducer addition (Figure 8C) time protein different case          course chosen test induction heat-shock  <br /> asked long induction AcpT overexpression takes          sponse deemed appropriate. <br />  <br />  <br />  <br />  <br />                                                                   Figure 8.&#226;&#8364;&#8221;Effects heat shock AcpT overproduction <br />                                                                dnaJ induction.   Behavior dnaJTlacZY fusion                                                                heat-shock response. Cultures dnaJTlacZY fusion <br />                                                                strain CY1817 (squares) bioATlacZY fusion strain <br />                                                                CY1783 (circles) growing exponentially early log phase <br />                                                                30&#194;&#176; shifted 42&#194;&#176; time zero samples taken <br />                                                                b-galactosidase assays times given. b-Galactosidase <br />                                                                activities given Miller units (Miller 1972). (B) Deriva- <br />                                                                tives strain CY1817 carrying plasmids encoding                                                                wild-type (pYon13) mutant (pNRD207) AcpT proteins                                                                vector (pBAD24) grown 30&#194;&#176; LB&#226;&#8364;&#8220;ampicil- <br />                                                                lin. Exponentially growing cultures received arabinose <br />                                                                final concentration 0.2% induce AcpT overexpres- <br />                                                                sion samples taken b-galactosidase assays                                                                times given. Triangles, vector control; squares, pYon13;                                                                circles, pNRD207. b-Galactosidase activities given Miller <br />                                                                units ordinate scale identical  (C)                                                                cells culture strain NRD183 grown LB&#226;&#8364;&#8220;ampicillin <br />                                                                medium containing 0.2% arabinose 30&#194;&#176; harvested <br />                                                                centrifugation, washed, resuspended LB medium. <br />                                                                culture diluted OD600 0.05 split <br />                                                                half. culture received 0.2% arabinose                                                                half remained unsupplemented. cultures                                                                grown overnight 30&#194;&#176;. washed cells cul- <br />                                                                tures recovered resuspended                                                                OD600 0.1 medium                                                                grown. turbidities cultures growth 30&#194;&#176; <br />                                                                monitored periodic dilution cultures <br />                                                                exponential growth growth culture lacking <br />                                                                arabinose began slow   5 hr dilution  <br />                                                                night cultures). Half culture added pre- <br />                                                                warmed flask containing sufficient arabinose final <br />                                                                concentration 0.2%. growth cultures <br />                                                                monitored. Squares denote culture grown                                                                arabinose experiment; circles denote cul- <br />                                                                ture lacking arabinose experiment; triangles <br />                                                                denote culture supplemented arabinose following <br />                                                                growth absence arabinose. <br />                                                   Phenotype E. coli yejM                                              1335 <br />  <br />                       DISCUSSION                                transposon mutagenesis essentially precluded detection <br />                                                                 yejM. Growth strains carrying yejM transposon <br />    YejM inner membrane protein predicted                                                                 insertion alleles assayed 30&#194;&#176; 37&#194;&#176; (Gerdes membrane-spanning helices large C-terminal <br />                                                                 et al. 2003; Kang et al. 2004), temperatures strains <br /> periplasmic domain (Figure 2) (Daley et al. 2005)                                                                 LH530 NRD187 grow reasonably  remains <br /> shares sequence similarities sulfatases phos- <br />                                                                 unclear strains carrying yejM trans- <br /> phatases (Swiss Protein database). topological <br />                                                                 poson insertion alleles temperature sensitive prediction accurate, G205A yejM mutant <br />                                                                 growth. <br /> allele strain LH530 encode YejM protein <br />                                                                    Suppression growth defects strains lacking <br /> containing complete transmembrane domain,                                                                 YejM periplasmic domain entire protein <br /> lacking entire C-terminal periplasmic domain.                                                                 overexpression cytosolic AcpT phosphopante- <br /> constructed yejM mutation vitro resulting        theinyl transferase presents conundrum. strain temperature sensitive.  original        mutation resulting defective outer membrane constructed yejM G205A mutant strains similar           suppressed overexpression enzyme post- <br /> phenotypes respect lipid biosynthesis             translationally modifies cytosolic carrier protein? hypersensitivity antibiotics. Chromatographic            shown phosphopantetheinyl transferase <br /> mass spectral analyses (C. Michael Reynolds, personal           activity AcpT involved yejM mutant suppression <br /> communication) showed strains (LH530              AcpT overexpression act indirectly. <br /> NRD187) essentially normal lipid compositions            possible explanation based properties bulk lipid produced strains       AcpT nature suppression phenomena parent strain JM105 hexa-acylated lipid       suppression involve stress response. 1,49-bis-phosphate species normally E. coli          agreement results Lambalot et al. (1996), (Raetz et al. 2007).  mutant strains            AcpT accumulates insoluble aggregate synthesized small discernible levels hepta-       fairly modest overexpression conditions <br /> acylated lipid species contains additional            (data shown). Aggregation fact <br /> palmitoyl group (C. Michael Reynolds, personal com-             AcpT evolutionarily isolated munication). believe presence lipid       substrates  Lay Cronan 2006) lacks <br /> species mutant strains secondary response        substrates aid proper protein folding.  fairly <br /> problems maintaining outer membrane integrity                high levels AcpT overexpression required (Raetz et al. 2007). Previous studies shown           suppression temperature-sensitive phenotype addition compounds affect outer membrane                strain LH530; low levels overexpression fail sup- <br /> integrity, cationic peptides, result synthe-     press (Hirvas et al. 1997) (data shown). misfold- <br /> sis palmitoylated lipid Salmonella enterica serovar     ing involved suppression YejM mutations, Typhimurium (Zhou et al. 1999). LpxM-deficient               specific AcpT overexpression acpT clones <br /> mutant strain E. coli O157:H7 reported         clones isolated clone bank 8000 accumulate lipid species (Raetz et al. 2007).         wild-type E. coli DNA fragments high-copy-number <br />    nonsense mutations resulted            vector allowed growth strain LH530 42&#194;&#176; <br /> ditionally lethal phenotype, deletion entire yejM        (Hirvas et al. 1997). tested specificity gene unconditionally lethal.  results        AcpT overexpression temperature-sensitive phe- <br /> suggest YejM functions. function carried       notype strain LH530 overexpression AcpH, N-terminal transmembrane domain essential         E. coli protein involved ACP metabolism growth temperatures function          aggregates overexpression  <br /> C-terminal periplasmic domain dispensable low-            ditions tested (Thomas Cronan 2005; Thomaset al. <br /> growth temperatures. functions important            2007). Overexpression AcpH using vector cellular process. work Baba et al.      employed Figure 7 failed allow growth strain <br /> (2006) reported yejM essential gene       LH530 42&#194;&#176; (data shown). Consistent ob- unable construct strain carrying complete           served inferred specificity suppression, AcpT <br /> deletion yejM l-Red recombinase-mediated gene             overexpression failed induce heat-shock regulon. <br /> replacement. contrast, groups reported yejM         Induction heat-shock regulon misfolded pro- essential gene basis transposon muta-       teins frequently observed (Oh Liao 2000; Lesley <br /> genesis (Gerdes et al. 2003; Kang et al. 2004).      et al. 2002; Smith 2007), induction heat reports reported transposon insertions     shock played major role AcpT overproduction located downstream nonsense mutation             suppression phenomenon, large diverse set strain LH530 N-terminal transmembrane             Hirvas clones (instead acpT clones) domain translated. mutant strains          suppressed yejM mutation. <br /> produce YejM proteins 60 residues longer             possible explanation (suggested  produced strain LH530.  nature             viewer) overexpressed AcpT specifically <br />  1336                                                      N. R. Lay J. E. Cronan <br />  <br /> bind normally abundant protein(s).                        Crowell, D. N., M. S. Anderson C. R. Raetz, 1986 Molecular <br />                                                                                 cloning genes lipid disaccharide synthase <span id='am-66' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span>UDP</span>-N- <br /> protein  proteins) trapped                              acetylglucosamine acyltransferase Escherichia coli. J. Bacteriol. <br /> AcpT aggregates, depleting cells func-                          168: 152&#226;&#8364;&#8220;159. <br /> tional AcpT-binding protein(s). weakness                       Daley, D. O., M. Rapp, E. Granseth, K. Melen, D. Drew et al., possible explanations consid-                            2005 Global topology analysis Escherichia coli inner mem- <br />                                                                                 brane proteome. Science 308: 1321&#226;&#8364;&#8220;1323. <br /> ered fail provide plausible mechanism                     Datsenko, K. , B. L. Wanner, 2000 step inactivation overexpression normally cytosolic protein                          chromosomal genes Escherichia coli K-12 using PCR products. <br /> allows growth E. coli cells lack essential mem-                      Proc. Natl. Acad. Sci. USA 97: 6640&#226;&#8364;&#8220;6645. <br />                                                                             Lay, N. R., J. E. Cronan, 2006 genome rearrangement brane protein.                                                                  orphaned Escherichia coli K-12 AcpT phosphopantetheinyl <br />    function YejM remains obscure. loss                           transferase cognate Escherichia coli O157:H7 substrates. periplasmic domain inner membrane                                Mol. Microbiol. 61: 232&#226;&#8364;&#8220;242. <br />                                                                             Ellermeier, C. D.,  Janakiraman J. M. Slauch, 2002  <br /> protein results increased permeability outer                       struction targeted single copy lac fusions using lambda Red <br /> membrane exacerbated increased growth tem-                           FLP-mediated site-specific recombination bacteria. Gene <br /> perature. reported phenotype reduced lipid                             290: 153&#226;&#8364;&#8220;161. <br />                                                                             Flugel, R. S., Y. Hwangbo, R. H. Lambalot, J. E. Cronan, Jr. biosynthesis strain LH530 (Hirvas et al. 1997; Vaara                         C. T. Walsh, 2000 Holo-(acyl carrier protein) synthase Nurminen 1999) complicated possibility                         phosphopantetheinyl transfer Escherichia coli. J. Biol. Chem. <br /> translocation phospholipids outer leaflet                        275: 959&#226;&#8364;&#8220;968. <br />                                                                             Galloway, S. M., C. R. Raetz, 1990 mutant Escherichia coli outer membrane known occur strains carry                         defective step endotoxin biosynthesis. J. Biol. <br /> mutations affecting organization lipopolysaccha-                         Chem. 265: 6394&#226;&#8364;&#8220;6402. <br /> ride cell surface (Wu et al. 2006).                      Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz <br />                                                                                 et al., 2003 Experimental determination level anal- <br /> lipid synthesis assayed using <span id='am-14' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span>phospholipid</span> synthe-                        ysis essential genes Escherichia coli MG1655. J. Bacteriol. 185: <br /> sis standard, results previously reported                     5673&#226;&#8364;&#8220;5684. partially increased phospholipid synthesis                  Gross, C. , 1996 Function regulation heat shock proteins, <br />                                                                                 pp. 1382&#226;&#8364;&#8220;1399 Escherichia coli Salmonella typhimurium: Cel- decreased lipid biosynthesis.                           lular Molecular Biology, edited F. C. Neidhardt, III, J. L. accumulation phospholipids outer leaflet                       Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, <br /> thought render cells sensitive imperme-                         M. Riley, M. Schaechter H. E. Umbarger. American Society <br /> able antibiotics (Raetz et al. 2007), phenotype strain                     Microbiology, Washington, DC. <br />                                                                             Gur, E., D. Biran, N. Shechter, P. Genevaux, C. Georgopoulos <br /> LH530. Increased sensitivity antibiotics                        et al., 2004 Escherichia coli DjlA CbpA proteins substi- <br /> seen lipid mutants.  known lipid                      tute DnaJ DnaK-mediated protein disaggregation. J. Bacteriol. <br /> mutants marked alterations compositions                          186: 7236&#226;&#8364;&#8220;7242. <br />                                                                             Guzman, L. M., D. Belin, M. J. Carson J. Beckwith, 1995 Tight lipid species yejM mutant strains                      regulation, modulation, high-level expression vectors  <br /> essentially normal lipid compositions (C. Michael                             taining arabinose PBAD promoter. J. Bacteriol. 177: 4121&#226;&#8364;&#8220;4130. <br /> Reynolds, personal communication).                                          Hirvas, L., M. Nurminen,  M. Helander, R. Vuorio M. Vaara, <br />                                                                                 1997 lipid biosynthesis deficiency Escherichia coli <br />   thank C. Michael Reynolds Christian Raetz                    antibiotic-supersensitive mutant LH530 suppressed novel <br /> partment Biochemistry, Duke University, lipid analyses         locus, ORF195. Microbiology 143: 73&#226;&#8364;&#8220;81. <br /> Coli Genetic Stock Center strains. work supported           Kang, Y., T. Durfee, J. D. Glasner, Y. Qiu, D. Frisch et al., 2004 Sys- <br /> National Institutes Health grant AI15650 National                   tematic mutagenesis Escherichia coli genome. J. Bacteriol. <br /> Institute Allergy Infectious Diseases.                                   186: 4921&#226;&#8364;&#8220;4930. <br />                                                                             Kelly, T. M., S.  Stachula, C. R. Raetz M. S. Anderson, <br />                                                                                 1993 firA gene Escherichia coli encodes UDP-3-O-(R-3- <br />                                                                                 hydroxymyristoyl)-glucosamine N-acyltransferase. step <br />                                                                                 endotoxin biosynthesis. J. Biol. Chem. 268: 19866&#226;&#8364;&#8220;19874. <br />                       LITERATURE CITED                                      Kloser,  W., M. W. Laird R. Misra, 1996 asmB, suppressor <br />                                                                                 locus assembly-defective OmpF mutants Escherichia coli, Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura et al., <br />                                                                                 allelic envA (lpxC). J. Bacteriol. 178: 5138&#226;&#8364;&#8220;5143. <br />     2006 Construction Escherichia coli K-12 frame, single-gene        Lambalot, R. H.,  M. Gehring, R. S. Flugel, P. Zuber, M. LaCelle <br />     knockout mutants: Keio collection. Mol. Syst. Biol. 2: 2006.0008.       et al., 1996 new enzyme superfamily: phosphopantetheinyl <br /> Bardwell, J. C., K. Tilly, E. Craig, J. King, M. Zylicz et al.,                 transferases. Chem. Biol. 3: 923&#226;&#8364;&#8220;936. <br />     1986 <span id='am-40' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-41' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-42' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence Escherichia coli K12 dnaJ1          Lesley, S. , J. Graziano, C. Y. Cho, M. W. Knuth H. E. Klock, <br />     gene. gene encodes heat shock protein. J. Biol. Chem.              2002 Gene expression response misfolded protein screen <br />     261: 1782&#226;&#8364;&#8220;1785.                                                             soluble recombinant protein. Protein Eng. 15: 153&#226;&#8364;&#8220;160. <br /> Bjornsson, , L.  Isaksson, 1993 UGA codon context             Miller, J. H., 1972 Experiments Molecular Genetics. Cold Spring <br />     spans codons: reversal ms2i6A37 tRNA, mutation               Harbor Laboratory Press, Cold Spring Harbor, NY. <br />     rpsD(S4) streptomycin. J. Mol. Biol. 232: 1017&#226;&#8364;&#8220;1029.                 Mofid, M. R., M.  Marahiel, R. Ficner K. Reuter, <br /> Cowing, D. W., J. C. Bardwell, E.  Craig, C. Woolford, R. W.                  1999 Crystallization preliminary crystallographic studies <br />     Hendrix et al., 1985 Consensus sequence Escherichia coli                Sfp: phosphopantetheinyl transferase modular peptide <br />     heat shock gene promoters. Proc. Natl. Acad. Sci. USA 82: 2679&#226;&#8364;&#8220;             synthetases. Acta Crystallogr. D Biol. Crystallogr. 55: 1098&#226;&#8364;&#8220;1100. <br />     2683.                                                                   Mofid, M. R., R. Finking, L. O. Essen M.  Marahiel, <br /> Cronan, J. E., 2003 Cosmid-based transient expression                2004 Structure-based mutational analysis 49-phosphopante- <br />     absolute transcriptional control Escherichia coli          theinyl transferases Sfp Bacillus subtilis: carrier protein recogni- <br />     genes. J. Bacteriol. 185: 6522&#226;&#8364;&#8220;6529.                                        tion reaction mechanism. Biochemistry 43: 4128&#226;&#8364;&#8220;4136. <br /> Cronan, J. E., 2006 family arabinose-inducible Escherichia coli        Mootz, H. D., R. Finking M.  Marahiel, 2001 49-Phospho- <br />     expression vectors having pBR322 copy control. Plasmid 55:                  pantetheine transfer primary secondary metabolism Ba- <br />     152&#226;&#8364;&#8220;157.                                                                    cillus subtilis. J. Biol. Chem. 276: 37289&#226;&#8364;&#8220;37298. <br />                                                                 Phenotype E. coli yejM                                                       1337 <br />  <br /> Nichols, B. P., O. Shafiq V. Meiners, 1998 Sequence analysis              Thomas, J., J. E. Cronan, 2005 enigmatic acyl carrier pro- <br />     Tn10 insertion sites collection Escherichia coli strains            tein phosphodiesterase Escherichia coli: genetic enzymolog- <br />     used genetic mapping strain construction. J. Bacteriol.                ical characterization. J. Biol. Chem. 280: 34675&#226;&#8364;&#8220;34683. <br />     180: 6408&#226;&#8364;&#8220;6411.                                                           Thomas, J., D. J. Rigden J. E. Cronan, 2007 Acyl carrier protein <br /> Normark, S., 1970 Genetics chain-forming mutant Escheri-                   phosphodiesterase (AcpH) Escherichia coli non-canonical <br />     chia coli: transduction dominance envA gene mediat-                 member HD phosphatase/phosphodiesterase family. Bio- <br />     ing increased penetration antibacterial agents. Genet.                 chemistry 46: 129&#226;&#8364;&#8220;136. <br />     Res. 16: 63&#226;&#8364;&#8220;78.                                                           Tsuruoka, T., M. Ito, S. Tomioka,  Hirata M. Matsuhashi, <br /> Nurminen, M., L. Hirvas M. Vaara, 1997 outer mem-                          1988 Thermosensitive omsA mutation Escherichia coli     brane lipid deficient Escherichia coli mutant LH530                causes thermoregulated release periplasmic proteins. J. Bacter- <br />     duced levels OmpF leaks periplasmic enzymes.                            iol. 170: 5229&#226;&#8364;&#8220;5235. <br />     Microbiology 143: 1533&#226;&#8364;&#8220;1537.                                              Vaara, M., M. Nurminen, 1999 Outer membrane permeability <br /> Oh, M. K., J. C. Liao, 2000 DNA microarray detection met-                   barrier Escherichia coli mutants defective late <br />     abolic responses protein overproduction Escherichia coli.                acyltransferases lipid biosynthesis. Antimicrob. Agents Che- <br />     Metab. Eng. 2: 201&#226;&#8364;&#8220;209.                                                        mother. 43: 1459&#226;&#8364;&#8220;1462. <br /> Parris, K. D., L. Lin,  Tam, R. Mathew, J. Hixon et al., 2000 Crystal       Wu, T.,  C. McCandlish, L. S. Gronenberg, S. S. Chng, T. J. <br />     structures substrate binding Bacillus subtilis holo-(acyl carrier        Silhavy et al., 2006 Identification protein complex  <br />     protein) synthase reveal novel trimeric arrangement mole-                 sembles lipopolysaccharide outer membrane Escherichia <br />     cules resulting active sites. Structure 8: 883&#226;&#8364;&#8220;895.                   coli. Proc. Natl. Acad. Sci. USA 103: 11754&#226;&#8364;&#8220;11759. <br /> Raetz, C. R., C. M. Reynolds, M. S. Trent R. E. Bishop,                   Yanisch-Perron, C., J. Vieira J. Messing, 1985 Improved M13 <br />     2007 Lipid modification systems gram-negative bacteria.                   phage cloning vectors host strains: nucleotide sequences     Annu. Rev. Biochem. 76: 295&#226;&#8364;&#8220;329.                                               M13mp18 pUC19 vectors. Gene 33: 103&#226;&#8364;&#8220;119. <br /> Reuter, K., M. R. Mofid, M.  Marahiel R. Ficner,                        Young, K., L. L. Silver, D. Bramhill, P. Cameron, S. S. Eveland <br />     1999 Crystal structure surfactin synthetase-activating               et al., 1995 envA permeability/cell division gene Escher- <br />     zyme Sfp: prototype 49-phosphopantetheinyl transferase                ichia coli encodes second enzyme lipid biosynthesis. <br />     superfamily. EMBO J. 18: 6823&#226;&#8364;&#8220;6831.                                            UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine deacety- <br /> Roth, J. R., N. Benson, T. Galitski, K. Haack, J. G. Lawrence et al.,              lase. J. Biol. Chem. 270: 30384&#226;&#8364;&#8220;30391. <br />     1996 Rearrangements bacterial chromosome: formation                Yu, D., H. M. Ellis, E. C. Lee, N.  Jenkins, N. G. Copeland et al., <br />     applications, pp. 2256&#226;&#8364;&#8220;2276 Escherichia coli Salmonella ty-         2000 efficient recombination chromosome engi- <br />     phimurium: Cellular Molecular Biology, edited F. C. Neidhardt,          neering Escherichia coli. Proc. Natl. Acad. Sci. USA 97: 5978&#226;&#8364;&#8220; <br />     III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S.              5983. <br />     Reznikoff, M. Riley, M. Schaechter H. E. Umbarger.                    Zhou, , S. Lin, R. J. Cotter C. R. Raetz, 1999 Lipid mod- <br />     American Society Microbiology, Washington, DC.                             ifications characteristic Salmonella typhimurium induced Singer, M., T.  Baker, G. Schnitzler, S. M. Deischel, M. Goel                    NH4VO3 Escherichia coli K12. Detection 4-amino-4-deoxy-L- <br />     et al., 1989 collection strains containing genetically linked             arabinose, phosphoethanolamine palmitate. J. Biol. Chem. <br />     alternating antibiotic resistance elements genetic mapping              274: 18503&#226;&#8364;&#8220;18514. <br />     Escherichia coli. Microbiol. Rev. 53: 1&#226;&#8364;&#8220;24. <br /> Smith, H. E., 2007 transcriptional <br /> </body></html>