<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Pds5p Essential Chromosomal Protein Required Sister <br /> Chromatid Cohesion Condensation Saccharomyces cerevisiae <br /> Theresa Hartman,* Kristen Stead,* Douglas Koshland,&#226;&#8364;&#161; Vincent Guacci* <br /> *Fox Chase Cancer Center, Basic Science Division, Philadelphia, Pennsylvania 19111; &#226;&#8364;&#161;Carnegie Institution Washington, <br /> Department Embryology, Baltimore, Maryland 21210 <br />  <br />  <br />  <br />  <br /> Abstract. PDS5 gene (precocious dissociation                           sequences bound Mcd1p. Furthermore, Pds5p local- <br /> sisters) identified genetic screen designed                       ization chromosomes dependent Mcd1p.  <br /> identify genes important chromosome structure.                            Pds5p, like cohesin complex members, compo- <br /> PDS5 essential gene homologues                            nent molecular glue mediates sister chroma- yeast humans. Pds5p function important                         tid cohesion.  Mcd1p localization chromo- <br /> viability S phase mitosis localizes                       somes independent Pds5p, reflect <br /> chromosomes cell cycle window,                           differences roles cohesion. Finally, Pds5p compasses times sister chromatid cohesion                            required condensation cohesion, exists. Pds5p required maintain cohesion cen-                        confirms link processes revealed <br /> tromere proximal distal sequences. proper-                          analysis Mcd1p (Guacci, V., D. Koshland, <br /> ties identical cohesion complex                       Strunnikov. 1997. Cell. 91:47&#226;&#8364;&#8220;57).  members Mcd1p/Scc1p, Smc1p, Smc3p, Scc3p/Irr1p                            link cohesion condensation general <br /> (Guacci, V., D. Koshland,  Strunnikov. 1997. Cell.                      property yeast chromosomes. <br /> 91:47&#226;&#8364;&#8220;57; Michaelis, C., R. Ciosk, K. Nasmyth. <br /> 1997. Cell. 91:35&#226;&#8364;&#8220;45; Toth, , R. Ciosk, F. Uhlmann, M.                      Key words: mitosis &#226;&#8364;&#162; chromosomes &#226;&#8364;&#162; cohesion &#226;&#8364;&#162;  <br /> Galova,  Schleiffer, K. Nasmyth. 1999. Genes                            densation &#226;&#8364;&#162; yeast <br /> Dev. 13:307&#226;&#8364;&#8220;319). Pds5p binds centromeric arm <br />  <br />  <br />  <br /> Introduction replication, duplicated chromatids (sister chro-                    rescent situ hybridization (FISH)1 indicated matids) associated entire length.                      mammals yeast, sister chromatids associated mitosis, association cohesion sister chro-                     lengths time replication anaphase <br /> matids maintained sisters condense, attach                    (Wilson, 1925; Selig et al., 1992; Guacci et al., 1993, 1994). mitotic spindle bipolar orientation, congress                    Mammalian yeast chromosomes undergo cell cy- metaphase plate. metaphase anaphase                          cle&#226;&#8364;&#8220;dependent condensation (Wilson, 1925; Lawrence et <br /> transition, sister chromatid cohesion dissolved synchro-                   al., 1988; Trask et al., 1989; Guacci et al., 1994). simi- <br /> nously chromosomes sisters segregate oppo-                      larity yeast mammalian chromosomes established <br /> site poles microtubule-dependent movement. Finally,                       budding yeast model study sister chromatid segregated sister chromatids decondense.                                  cohesion condensation (Guacci et al., 1993, 1994).    Eukaryotic chromosomes large dynamic structures,                       method green fluorescent protein (GFP)&#226;&#8364;&#8220;tagging chro- <br /> making daunting task identify components                       mosomal loci vivo enhanced yeast sponsible cohesion condensation. Insights                    (Straight et al., 1996; Michaelis et al., 1997). use timing, distribution, mechanism sister chromatid                       FISH- GFP-tagged loci cytological tools combined <br /> cohesion provided studies mamma-                         powerful genetic molecular tools available lian yeast cells. Classical cytological analyses fluo-                yeast led isolation mitotic regulators                                                                               chromosomal structural proteins (Guacci et al., 1993, 1994, <br />  <br /> Address correspondence Vincent Guacci, Fox Chase Cancer Center, <br /> Basic Science Division, Room W462, 7701 Burholme Avenue, Philadel-            1 <br />                                                                                 Abbreviations used paper: ChIP, chromatin immunoprecipitation; <br /> phia, PA 19111. Tel.: (215) 728-5632. Fax: (215) 728-3616. E-mail:            FISH, fluorescent situ hybridization; GFP, green fluorescent protein; <br /> va_guacci@fccc.edu                                                            HA, hemagglutinin. <br />  <br />  <br />  <br />  <br /> &#239;&#163;&#169; Rockefeller University Press, 0021-9525/2000/10/613/14 $5.00 Journal Cell Biology, Volume 151, Number 3, October 30, 2000 613&#226;&#8364;&#8220;626 <br /> http://www.jcb.org/cgi/content 151/3/613                                 613 <br />  1997b; Strunnikov et al., 1995; Funabiki et al., 1996; Yama-       al., 1999). addition roles cohesion  <br /> moto et al., 1996a,b; Michaelis et al., 1997; Furuya et al.,       densation, members Smc protein family 1998; Skibbens et al., 1999; Toth et al., 1999; Ciosk et al.,      implicated dosage compensation, recombination, 2000; Tanaka et al., 2000).                                        transcriptional silencing (Chuang et al., 1994; Jessberger et <br />    Chromosomal proteins essential sister chromatid          al., 1996; Donze et al., 1999). prominent characteristic <br /> hesion identified yeast vertebrates               Smc protein family members possess large <br /> (Guacci et al., 1997b; Michaelis et al., 1997; Furuya et al.,      coiled-coil domains span length   <br /> 1998; Losada et al., 1998; Skibbens et al., 1999; Toth et al.,     viewed Koshland Strunnikov, 1996). 1999; Ciosk et al., 2000; Tanaka et al., 2000). budding         broad range chromosomal functions conse- <br /> yeast, cohesion factors Mcd1p  known             quence specific interactions variety proteins Scc1p), Smc1p, Smc3p, Ctf7p  known Eco1p), Scc2p,          Smc members coiled-coil domain. <br /> Scc3p  known Irr1p), Scc4p (Kurlandzka et al.,            cohesin complex proteins molecular <br /> 1995; Guacci et al., 1997b; Michaelis et al., 1997; Skibbens et    glue mediates sister chromatid cohesion mito- <br /> al., 1999; Toth et al., 1999; Ciosk et al., 2000). factors   sis.  sister chromatid cohesion complex functionally divided mediate cohesion       highly regulated process cohesin complex  e., form molecular &#226;&#8364;&#339;glue&#226;&#8364;?) required      story (reviewed Koshland Guacci, 2000). establish cohesion glue.                establishment cohesion requires load- <br />    lines evidence indicate Mcd1p, Smc1p,           ing cohesin complex chromosomes, likely <br /> Smc3p, Scc3p molecular glue.             coupling DNA replication. identification proteins form soluble complex termed               budding yeast Trf4p DNA polymerase required hesin, localized chromosomes early S phase           sister chromatid cohesion supports connection  <br /> (Losada et al., 1998). Second, Mcd1p required cohe-         tween cohesion replication (Wang et al., 2000). Fur- <br /> sion cell viability soon S phase, indicat-     thermore, Drosophila MEI-S322 protein loads ing essential role time cohesion established          centromeric regions chromosomes prometaphase (Guacci et al., 1997b; Uhlmann Nasmyth, 1998). Fur-            thought function maintenance cohesion thermore, Mcd1p essential cohesion mitosis, dem-         centromere (Kerrebrock et al., 1995; Moore et al., <br /> onstrating required maintain cohesion (Guacci        1998).  region-specific cohesion factors et al., 1997b).  functional site cohesion           load chromosomes DNA replication. Fi- <br /> mapped yeast CEN DNA vivo (Megee                     nally, Mcd1p required condensation cohe- <br /> Koshland, 1999). Subsequently, cohesin complex members             sion (Guacci et al., 1997b).  continued Mcd1p Smc1p shown localize yeast CEN                screen identify additional components required DNA, providing direct link cohesin complex           sister chromatid cohesion help elucidate mecha- functional site cohesion (Blat Kleckner, 1999;        nisms sister chromatid cohesion condensation. <br /> Megee et al., 1999; Tanaka et al., 1999).                          identified new budding yeast protein, <br />    lines evidence indicate Ctf7p, Scc2p,       Pds5p (precocious dissociation sister chromatids). <br /> Scc4p required establishment cohesion.              Pds5p 1,277 amino acids shares homology  required cohesion cell viability dur-      proteins fungi (Aspergillus nidulans {BimD6} ing soon S phase, required S <br /> phase (Skibbens et al., 1999; Toth et al., 1999; Ciosk et al., <br /> 2000). Second, Scc2p Scc4p required loading            Table  Yeast Strains Used Study cohesin complex chromosomes, colo-             Strain                                  Genotype <br /> calize complex (Toth et al., 1999; Ciosk et al., <br />                                                                    VG958-7C              Mata pds5-3 trp1 leu2 bar1 gal1 <br /> 2000).  genetic interactions Ctf7p                                                                    VG982-3A              Mat&#226;?&#163; trp1 leu2 ura3 bar1 gal1 <br /> Pol30p (PCNA), known replication protein, Ctf18p,            VG982-7B              Mat&#226;?&#163; trp1 leu2 bar1 gal1 replication factor C homologue, indicate cohesion           VG982-6A              Mata trp1 ura3 bar1 gal1 replication intrinsically linked (Skibbens et al.,         VG986-5B              Mata pds5-1 trp1 ura3 bar1 gal1 <br /> 1999). Taken  Scc2p, Scc4p, Ctf7p function         VG987-5C              Mata pds5-2 trp1 ura3 bar1 gal1 <br /> load proteins physically mediate cohesion link        VG988-1C              Mata pds5-3 trp1 ura3 bar1 gal1 establishment cohesion DNA replication.                VG1328-2C             Mat&#226;?&#163; cdc15-2 trp1 ura3 gal1 <br />    Chromosomal proteins essential chromosome               VG2043-8C             Mata trp1 lys2-801 bar1 gal1 <br /> densation identified yeast Xenopus           VG2049-3A             Mata MCD1-GFP ura3 bar1 gal1 <br /> (Hirano Mitchison, 1994; Saitoh et al., 1994; Saka et          VG2050-1A             Mata MCD1-6HA ura3 bar1 gal1 <br />                                                                    VG2061-1D             Mat&#226;?&#163; MCD1-GFP cdc15-2 ura3 leu2 gal1 <br /> al., 1994; Strunnikov et al., 1995; Hirano et al., 1997; Sutani <br />                                                                    VG2063-5C             Mata PDS5-GFP trp1 ura3 lys2-801 bar1 gal1 Yanagida, 1997; Lavoie et al., 2000; Ouspenski et al.,         VG2064-2A             Mata PDS5-6HA trp1 ura3 lys2-801 bar1 gal1 <br /> 2000). budding yeast, Smc2p Brn1p required              VG2069-1B             Mata PDS5-GFP cdc15-2 trp1 ura3 gal1 chromosome condensation (Strunnikov et al., 1995;              VG2072-2D             Mata mcd1-1 PDS5-6HA trp1 ura3 bar1 gal1 <br /> Lavoie et al., 2000; Ouspenski et al., 2000). Homologues        VG2073-7D             Mata pds5-1 MCD1-6HA leu2 ura3 bar1 gal1 <br /> Smc2p, Smc4p, Barren pro-              VG2074-6C             Mata pds5-2 MCD1-6HA leu2 bar1 gal1 <br /> teins form soluble complex termed condensin (Hirano et           VG2073-7D             Mata pds5-3 MCD1-6HA leu2 ura3 bar1 gal1 <br /> al., 1997; Sutani et al., 1999). mechanism        VG1325                Mata trp1 leu2 ura3 ade2 bar1 gal1 <br /> densation determined, condensin com-                                    Mat&#226;?&#163; trp1 leu2 ura3 ade3 bar1?gal1 can1 <br /> plex function induce supercoils DNA facili-           VG1359                Mata pds5::URA3 trp1 leu2 ura3 ade2 bar1 galI <br />                                                                                          Mat&#226;?&#163; trp1 leu2 ura3 ade3 bar1?gal1 can1 <br /> tate condensation (Kimura Hirano, 1997; Kimura et <br />  <br />  Journal Cell Biology, Volume 151, 2000                      614 <br />  Sordaria {Spo76}) putative proteins Schizo-                      virtually spores germinated attempted cell divisions <br /> saccharomyces pombe humans (Denison et al., 1993;                           dying. Finally, strain VG1359 transformed plasmid <br />                                                                                pTH10, centromere plasmid bearing PDS5 andTRP1, sporulated, van Heemst et al., 1999). BimD6 Spo76 proteins                         dissected. Spore viability increased 65%. 10 tetrads dissected, 1 essential mitotic meiotic chromosome segrega-                      4:0 (viable:inviable), 4 3:1, 5 2:2. Viable Ura&#226;&#171;&#185; spores tion, respectively (Denison et al., 1993; van Heemst et al.,                   obtained Trp&#226;&#171;&#185;, indicating spores containing 1999). demonstrate Pds5p essential chromo-                       pds5::URA3 allele grew contained plasmid pTH10. data <br />                                                                                demonstrate PDS5 essential gene. <br /> somal protein required chromosome segregation sister chromatid cohesion mitosis.  Pds5p                      COOH-terminal Tagging Pds5p Mcd1p <br /> functional homology BimD6 Spo76.   localization Pds5p chromosomes depen-                       Strains constructed contain COOH-terminally tagged <br />                                                                                Pds5p (Pds5p-6HA Pds5p-GFP) Mcd1p (Mcd1p-6HA Mcd1p- <br /> dent cohesin complex member Mcd1p. Finally,                          GFP) control endogenous promoters sole source Pds5p required chromosome conden-                        Pds5p Mcd1p, respectively. Haploids tagged Pds5p sation mitosis, confirming sister chromatid                     follows: BamHI site generated <span id='am-1' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> PDS5 <br /> hesion condensation linked.                                            ORF PCR 0.26-kb BamHI fragment bearing tandem  <br />                                                                                magglutinin (HA) epitopes 0.75-kb BamHI fragment bearing GFP                                                                                inserted form plasmid pVG264 pVG263, respectively. plasmids <br /> Materials Methods                                                          pRS406 backbone, PDS5 promoter, ORF tag 3&#226;&#172;&#732;UTR. <br />                                                                                Plasmids pVG264 pVG263 linearized AflII, transformed                                                                                haploid VG982-3A, plated Ura&#226;&#171;&#186; media create tandem inte- <br /> Reagents Media                                                             grants PDS5 genomic locus. Ura&#226;&#171;&#185; transformants streaked Reagents media described previously (Guacci et al., 1997b;            FOA-containing plates Ura&#226;&#171;&#186; cells (popouts) screened PCR Yamamoto et al., 1996b). Benomyl gift Dupont. Yeast strains         identify retaining tagged PDS5 alleles. crossed haploid listed Table                                                          VG2043-8C resulting diploids sporulated dissected gener- <br />                                                                                ate VG2064-2A (PDS5-6HA) VG2063-5C (PDS5-GFP). Haploid <br />                                                                                VG2063-5C crossed VG1328-2C resulting diploid sporu- <br /> Identification Mutants Exhibiting Mitotic Lethality                         lated dissected generate haploid VG2069-1B (cdc15-2 PDS5-GFP). collection temperature-sensitive mutants (Ts&#226;&#171;&#186;) patched               make haploids containing tagged Mcd1p, BamHI site gener- <br /> YEPD plates grown 23&#226;&#172;&#353;C. Cells replica plated media          ated PCR amino acid MCD1 ORF taining &#226;?&#163; factor (10&#226;&#171;&#186;8 M) benomyl (70 &#226;?&#174;g/ml) incubated        6HA GFP fragment inserted form plasmid pVG269 pVG270,  <br /> 23&#226;&#172;&#353;C cells arrested G1 mid-M phase, respectively. Plates    spectively. plasmids YIplac211 backbone, MCD1 pro- transferred 37&#226;&#172;&#353;C 3 h inactivate mutant gene product         moter, ORF tag, 3&#226;&#172;&#732;UTR. Plasmids pVG269 pVG270 cells arrested, replica plated YEPD incubated       linearized AgeI, transformed haploid VG982-3A. Cells 23&#226;&#172;&#353;C 48 h release cells arrest allow growth. Mutants        bearing MCD1-6HA MCD1-GFP alleles identified described <br /> showing enhanced lethality transient arrest mid-M phase com-          tagged Pds5p. Strains bearing tagged Pds5p Mcd1p grew rates <br /> pared G1 phase chosen study.                                     similar isogenic wild-type strains, indicating tag did com- <br />                                                                                promise protein function. <br /> Cloning PDS5 Gene <br />                                                                                Yeast Cell Culture Conditions mutants enhanced mitotic lethality precocious dissociation sister chromatids fell single complementation group based       Yeast cells grown YPD 23&#226;&#172;&#353;C mid-log phase treated failure grow 37&#226;&#172;&#353;C mated, renamed pds5-1, pds5-2,        follows. pds5-3. Strains crossed Ts&#226;&#171;&#186; mitotic lethality           Arrest G1, S Mid-M Phase. &#226;?&#163; factor (10&#226;&#171;&#186;8 M), hydroxyurea (HU; <br /> phenotypes segregated 2&#226;&#171;&#185;:2&#226;&#171;&#186; cosegregated. analysis pds5        100 mM), nocodazole (15 &#226;?&#174;g/ml) added arrest cells G1, S, phenotypes, original isolates backcrossed times wild-type         mid-M phase, respectively, described (Guacci et al., 1997b). <br /> strains generate haploids VG986-5B (pds5-1), VG987-5C (pds5-2),            Synchronous Populations Cycling Cells Released S Phase. <br /> VG988-1C (pds5-1). PDS5 gene cloned transforming haploid            Cells treated described (Guacci et al., 1997b), &#226;?&#163; factor <br /> VG958-7C (pds5-3) pRS314 (CEN ARS TRP1) vector-based yeast              (10&#226;&#171;&#186;8 M) added media release S phase. <br /> genomic library (provided P. Hieter, University British Columbia,           Synchronous Populations Mid-M Phase Cells Obtained Release <br /> Vancouver, BC, Canada). Transformed cells plated Trp&#226;&#171;&#186;             G1 Phase. Cells treated described (Guacci et al., 1997b). <br /> dia, incubated 22 h 23&#226;&#172;&#353;C, transferred 37&#226;&#172;&#353;C 3 d. clones grew 37&#226;&#172;&#353;C contained identical plasmids (8B). 5.8-kb ClaI       Fluorescence Situ Hybridization <br /> fragment plasmid 8B complemented Ts&#226;&#171;&#186; mitotic lethal <br />                                                                                FISH performed described described (Guacci et al., 1994, <br /> phenotypes pds5 mutants. ClaI insert ends sequenced                                                                                1997a). Probes used analysis cohesion condensation  <br /> contained ORF YMR076C chromosome XIII. <br />                                                                                scribed (Guacci et al., 1994, 1997a,b). <br />  <br /> PDS5 Essential Gene                                                      Chromosome Spreads <br /> Plasmid pVG177 contains 5.8-kb ClaI fragment bearing PDS5 vector          Chromosome spreads performed described (Michaelis et al., <br /> pRS314. URA3 gene 1.2-kb HindIII fragment (blunt ended using          1997). HA-tagged proteins detected using mouse anti&#226;&#8364;&#8220;HA antibody <br /> Klenow) inserted plasmid pVG177 BstEII site (blunt             16B12 (Babco) (1:2,500) Cy3-conjugated goat anti&#226;&#8364;&#8220;mouse antibody <br /> ended using Klenow) form plasmid, pVG199, URA3 inserted           (Jackson ImmunoResearch Laboratories) (1:3,000). amino acid 114 PDS5 ORF (pds5::URA3). Plasmid pVG199 digested ClaI transformed diploid VG1325. Ura&#226;&#171;&#185; trans- <br /> formants analyzed Southern blot identify clones pds5::         Chromatin Immunoprecipitation <br /> URA3 chromosome wild-type PDS5 homologue,              Chromatin immunoprecipitation (ChIP) PCR analysis  renamed diploid VG1359.                                                formed described (Megee et al., 1999). <br />    determine PDS5 essential gene, wild-type parent <br /> VG1325 (PDS5/ PDS5) VG1359 (pds5::URA3/PDS5) diploid cells                 Fluorescence Microscopy Flow Cytometry sporulated dissected. VG1325 (PDS5/ PDS5) 98% spore <br /> viability. contrast, VG1359 (pds5::URA3/PDS5) 47% spore          Indirect immunofluorescence flow cytometry described (Yama- <br /> viability. 29 tetrads dissected, 25 2:2 (viable:inviable) 4   moto et al., 1996a). Images collected using epi-fluorescence mi- <br /> 1:3 tetrads. viable spores Ura&#226;&#171;&#186; indicating contained    croscope (E800; Nikon) recorded digitally using Hamamatso cooled <br /> pds5::URA3 allele. Microscopic examination spores failed form       CCD camera processing software Phase 3 Imaging, al- <br /> colonies revealed 98% cell bodies, demonstrating          lowed image superimposition. <br />  <br />  <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation                 615 <br />  Results                                                          using &#226;?&#163; factor, hydroxyurea, nocodazole, respec- <br />                                                                  tively, incubated 2 h 37&#226;&#172;&#353;C arrested <br /> PDS5 Isolated Screen Designed Identify                Materials Methods). Aliquots cells Genes Encoding Proteins Important Sister                     washed free arresting agent plated YEPD Chromatid Cohesion                                               determine cell viability (Fig. 1 B). G1 phase, pds5 cells <br />                                                                  maintained viability shifted 37&#226;&#172;&#353;C. S phase, pds5 previously used cytogenetic screen isolate mutants       cells showed 1.5-fold decrease viability  <br /> defective sister chromatid cohesion identified pds1       cubated 37&#226;&#172;&#353;C.  mid-M phase, pds5 cells mcd1 mutants (Guacci et al., 1993, 1997b; Yamamoto           showed 3-fold decrease viability 23&#226;&#172;&#353;C 40-fold <br /> et al., 1996a,b). characterization       decrease 2% viability shifted 37&#226;&#172;&#353;C.  Pds5p <br /> pds5 (precocious dissociation sister chromatids),          function required G1 phase, essential mutant identified screen. brief, collection   mid-M phase. weak effect S phase pds5 cells indi- mutants temperature sensitive growth (Ts&#226;&#171;&#186;)           cates Pds5p function required screened identify exhibiting enhanced lethality af-     replication soon  Pds5p localizes chromo- <br /> ter transient arrest mid-M phase compared G1          somes early S phase  . <br /> phase. Mutants enhanced mitotic lethality screened FISH identify exhibiting precocious <br /> dissociation sister chromatids  . mu- <br />                                                                  PDS5 Essential Gene Required Proper <br /> tants formed complementation group named <br />                                                                  Chromosome Segregation <br /> pds5-1, pds5-2, pds5-3. mutants showed          PDS5 gene cloned complementation creased viability 30&#226;&#172;&#353;C growth 37&#226;&#172;&#353;C, indicating      Ts&#226;&#171;&#186; mitotic lethal phenotypes pds5 cells (Fig. 1 C). allele defective function high tempera-     Standard yeast genetic techniques used ture (Fig. 1 . 23&#226;&#172;&#353;C, pds5 allele         PDS5 essential gene (Materials Methods). growth defects pds5 cells form heterogeneously sized       mitotic lethality pds5 cells suggested function colonies &#226;&#181;&#8216;80% pds5 cells form colonies            Pds5p chromosome structure segregation. deter- <br /> YEPD plates, compared 100% wild-type             role Pds5p cell cycle progression chro- <br /> cells (data shown).                                          mosome segregation, wild-type pds5 haploid cells <br />   precisely assess stage cell cycle           grown mid-log phase 23&#226;&#172;&#353;C shifted 37&#226;&#172;&#353;C, Pds5p function essential, wild-type pds5 hap-       cell, spindle, DNA morphologies DNA <br /> loid cells arrested G1, S, mid-M phases        content scored. 23&#226;&#172;&#353;C, FACS&#194;&#174; (Becton Dickinson) <br />  <br />  <br />  <br />  <br />                                                                                               Figure 1. Characterization                                                                                               pds5 mutants.   Ts&#226;&#171;&#186; pheno- <br />                                                                                               type pds5 cells. Wild-type <br />                                                                                               (VG982-6A), pds5-1 (VG986- <br />                                                                                               5B), pds5-2 (VG987-5C),                                                                                               pds5-3 (VG988-1C) cells grown <br />                                                                                               23&#226;&#172;&#353;C YPD liquid                                                                                               plated 10-fold serial dilutions <br />                                                                                               YPD incubated 23&#226;&#172;&#353;, <br />                                                                                               30&#226;&#172;&#353;, 37&#226;&#172;&#353;C. (B) Cell cycle&#226;&#8364;&#8220; <br />                                                                                               dependent lethality pds5 <br />                                                                                               cells. Strains arrested <br />                                                                                               23&#226;&#172;&#353;C G1, S, mid-M <br />                                                                                               phase, incubated 37&#226;&#172;&#353;C                                                                                               arrested, plated                                                                                               YPD 23&#226;&#172;&#353;C determine  <br />                                                                                               cent viability (Materials                                                                                               Methods). Data                                                                                               independent experiments                                                                                               used generate error bars. (C) <br />                                                                                               Cloning PDS5 gene. <br />                                                                                               Strains transformed <br />                                                                                               CEN vector                                                                                               bearing PDS5 ORF. <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 151, 2000                    616 <br />                                                                                                 Figure 2. Cell cycle progression <br />                                                                                                wild-type pds5 mutant <br />                                                                                                cells.   DNA content cells. <br />                                                                                                Wild-type (VG982-6A)                                                                                                pds5-3 (VG988-1C) cells                                                                                                grown asynchronously 23&#226;&#172;&#353;C <br />                                                                                                shifted 37&#226;&#172;&#353;C 3 h,                                                                                                subjected FACS&#194;&#174; analy- <br />                                                                                                sis measure DNA content <br />                                                                                                (Materials Methods). (B) <br />                                                                                                Micrographs showing mi- <br />                                                                                                totic spindle cells 1 h  <br />                                                                                                lease S phase 37&#226;&#172;&#353;C. <br />                                                                                                Strains synchronously <br />                                                                                                released 37&#226;&#172;&#353;C S phase <br />                                                                                                arrest,  1 h, cells                                                                                                processed immunofluores- <br />                                                                                                cence. Chromosomal DNA <br />                                                                                                (gray) Tubulin (white). (C) <br />                                                                                                Quantitation cell DNA <br />                                                                                                morphologies release                                                                                                S phase 37&#226;&#172;&#353;C. Strains B <br />                                                                                                scored cell DNA <br />                                                                                                morphologies S phase ar- <br />                                                                                                rested cells 37&#226;&#172;&#353;C [S phase <br />                                                                                                (HU)] cells 2 h  <br />                                                                                                lease 37&#226;&#172;&#353;C (2 h S phase). <br />                                                                                                Data 400 cells                                                                                                independent experiments                                                                                                scored generate error bars. <br />  <br />  <br /> profiles pds5 cells similar wild-type cells (Fig. 2   tures wild-type pds5 cells grown mid-log .  3 6 h 37&#226;&#172;&#353;C, pds5 cultures      phase 23&#226;&#172;&#353;C, arrested S phase addition hydroxy- <br /> enriched cells 2C DNA content showed              <span id='am-8' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span>, transferred 37&#226;&#172;&#353;C 1 h pres- <br /> twofold increase frequency large budded cells          ence hydroxyurea inactivate mutant Pds5p S short partially elongated spindles (Fig. 2         phase (Materials Methods). Finally, cells  <br /> data shown). phenotypes indicate pds5 cells       leased S phase transferring YEPD media  <br /> accumulate G2 M phases cell cycle             taining &#226;?&#163; factor 37&#226;&#172;&#353;C. regimen allows cells grown 37&#226;&#172;&#353;C. result consistent defect        progress mitosis nonpermissive temperature <br /> chromosome structure delaying cell cycle progression             pds5, arrests cells G1 phase prevent mitosis.                                                 progression new cell cycle (Materials Meth- <br />   Pds5 mutant cells characterized using syn-        ods). S phase arrested cells 23&#226;&#172;&#353; 37&#226;&#172;&#353;C, wild- <br /> chronous populations cycling cells obtained          type pds5 cells 1C DNA content, large bud,  <br /> lease S phase arrest. Asynchronously growing cul-           divided DNA mass, short spindle (Fig. 2 C, data <br />  <br />  <br />  <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation   617 <br />                                                                                            Figure 3. Sister chromatid cohe- <br />                                                                                           sion wild-type pds5 cells. <br />                                                                                             Micrographs mid-M <br />                                                                                           phase cells subjected FISH <br />                                                                                           using chromosome XVI CEN- <br />                                                                                           distal probe distal (295 kb                                                                                           CEN16). Wild-type (VG982- <br />                                                                                           6A) pds5-2 (VG987-5D) <br />                                                                                           cells arrested G1 phase <br />                                                                                           [G1 (&#226;?&#163;F) 23&#226;&#172;&#353;C], shifted 37&#226;&#172;&#353;C, <br />                                                                                           released G1                                                                                           arrested mid-M phase {M   <br />                                                                                           codazole) 37&#226;&#172;&#353;C}. Mid-M cells <br />                                                                                           processed FISH. Chro- <br />                                                                                           mosomal DNA (gray) hy- <br />                                                                                           bridized probe (white). Bar, 5 <br />                                                                                           &#226;?&#174;m. (B) Quantitation FISH <br />                                                                                           G1 mid-M phase cells. <br />                                                                                           Wild-type (VG982-6A), pds5-1 <br />                                                                                           (VG986-5B), pds5-2 (VG987- <br />                                                                                           5C), pds5-3 (VG988-1C) <br />                                                                                           arrested subjected                                                                                           FISH described                                                                                            number FISH signals                                                                                           DNA mass determined                                                                                           plotted percentage total <br />                                                                                           nuclei. 200 nuclei G1                                                                                           mid-M cells scored gen- <br />                                                                                           erate data. (C) DNA content                                                                                           mid-M phase cells. Aliquots                                                                                           cells strains B fixed <br />                                                                                           processed flow cytome- <br />                                                                                           try analyze DNA content <br />                                                                                           mid-M phase&#226;&#8364;&#8220;arrested cells                                                                                           37&#226;&#172;&#353;C. <br />  <br />  shown). 1 h release S phase 37&#226;&#172;&#353;C,     Pds5p Required Sister Chromatid Cohesion <br /> wild-type pds5 cells completed DNA replication       assayed pds5 cells defective sister indicated 2C DNA content (data shown).      chromatid cohesion. Wild-type, pds5-1, pds5-2, pds5-3 <br /> wild-type cells completed chromosome segrega-       haploid strains arrested G1 phase 23&#226;&#172;&#353;C incuba- <br /> tion elongated spindle segregated      tion YEPD liquid containing &#226;?&#163; factor, trans- <br /> DNA masses (Fig. 2 B). contrast, pds5 cells    ferred 37&#226;&#172;&#353;C 1 h inactivate mutant Pds5p G1 <br /> stretched nuclear DNA mass partially elongated spin-     phase (Materials Methods). Cultures  <br /> dle, indicative chromosome segregation defect (Fig. 2   leased G1 phase arrest 37&#226;&#172;&#353;C transferring cells B). 2 h release, 75% wild-type cells unbud-    YEPD liquid-containing nocodazole arrest cells mid- <br /> ded, demonstrated exited mitosis     M phase 37&#226;&#172;&#353;C (Materials Methods). proto- <br /> arrested G1 phase (Fig. 2 C). contrast, 25%    col, cells nonpermissive temperature pds5-3 cells unbudded majority remained       pds5 mutation S, G2, mid-M phases, mitosis large budded cells anaphase   encompasses time sister chromatid cohesion stretched DNA mass telophase separated DNA        established maintained. Aliquots G1 mid-M <br /> masses (Fig. 2 C). 4 h, number unbudded         phase cells processed FISH using chromosome <br /> pds5-3 cells increased 63%. Similar results ob-      XVI centromere-distal probe assay cohesion site <br /> tained pds5-1 pds5-2 cells (data shown).        295 kb centromere (Materials Methods). data suggest pds5 inactivation causes mitotic   Consistent previous studies, wild-type cells <br /> defect disrupts chromosome segregation delays,      mid-M phase FISH signal nuclear DNA does prevent, exit mitosis.                     mass, demonstrating sister chromatids paired <br />  Journal Cell Biology, Volume 151, 2000                618 <br />                                                                          replication new bud formation (Fig. 3 C, data                                                                         shown).  precocious dissociation sister <br />                                                                         chromatids pds5 cells defect cell-cycle <br />                                                                         regulation, reflects potential role Pds5p sister <br />                                                                         chromatid cohesion. <br />                                                                           shown pds5 mutants defective <br />                                                                         cohesion, previous regimen does distinguish <br />                                                                         defect establishment mainte- <br />                                                                         nance cohesion. used different regimen distin- <br />                                                                         guish possibilities inactivating mu- <br />                                                                         tant Pds5p cohesion established. Wild-type <br />                                                                         pds5-3 haploid cells arrested G1 phase 23&#226;&#172;&#353;C <br />                                                                         using &#226;?&#163; factor, released G1 phase arrested                                                                         mid-M phase 23&#226;&#172;&#353;C treatment nocodazole,                                                                         transferred 37&#226;&#172;&#353;C inactivate mutant Pds5-3p <br />                                                                         mid-M phase (Materials Methods). Aliquots                                                                         mid-M phase cells 23&#226;&#172;&#353; 37&#226;&#172;&#353;C plated cell  <br />                                                                         bility processed FISH using centromere-distal <br />                                                                         probe chromosome XVI. expected, wild- <br />                                                                         type cells mid-M phase 23&#226;&#172;&#353; 37&#226;&#172;&#353;C viable                                                                         FISH signal (Fig. 4, B). Similarly,                                                                         pds5-3 cells mid-M phase 23&#226;&#172;&#353;C viable                                                                         FISH signal.  shift mid-M phase <br />                                                                         pds5-3 cells 37&#226;&#172;&#353;C, viability decreased number                                                                         cells FISH signals increased (Fig. 4, B). <br />                                                                         Similar results obtained using FISH probes cen- <br />                                                                         tromere-proximal regions chromosomes IV XVI <br /> Figure 4. Maintenance sister chromatid cohesion wild-type         (data shown). results demonstrate Pds5p pds5 cells arrested mid-M phase. Wild-type (VG982-6A)            function required maintain sister chromatid cohesion pds5-3 (VG988-1C) cells arrested G1 phase                mid-M phase centromere proximal distal chro- <br /> 23&#226;&#172;&#353;C, released G1 phase arrested mid-M             mosomal regions. <br /> phase 23&#226;&#172;&#353;C. Cells transferred 37&#226;&#172;&#353;C 1 h ar- <br /> rested. Aliquots mid-M phase cells 23&#226;&#172;&#353; 37&#226;&#172;&#353;C plated determine cell viability   processed FISH (B).           Pds5p Required Chromosome Condensation <br /> Cell viability. Mid-M phase cells 23&#226;&#172;&#353; 37&#226;&#172;&#353;C plated YEPD grown 3 d 23&#226;&#172;&#353;C, percentage cell viabil-         previously shown mcd1 mutant defec- <br /> ity calculated (Materials Methods). Data generated          tive chromosome condensation sister chro- independent experiments. (B) Sister chromatid cohe-            matid cohesion (Guacci et al., 1997b).  used <br /> sion. Mid-M phase cells 23&#226;&#172;&#353; 37&#226;&#172;&#353;C processed FISH         FISH determine pds5 mutants defec- <br /> using chromosome XVI centromere distal probe (cosmid                  tive condensation. Wild-type, pds5-1, pds5-2, pds5-3 <br /> 70912). number FISH signals DNA mass                 haploid cells arrested G1 phase 23&#226;&#172;&#353;C using &#226;?&#163; fac- <br /> quantitated described Fig. 3 B.                                   tor, incubated 37&#226;&#172;&#353;C 1 h, released G1 <br />                                                                         phase arrested mid-M phase 37&#226;&#172;&#353;C using nocoda- <br />                                                                         zole processed FISH (Materials Methods). <br /> (Fig. 3, B). small number DNA masses,                  examined chromosome condensation FISH signals detected low level                 rDNA locus, 500-kb block repetitive DNA. G1 <br /> precocious sister chromatid dissociation spurious               phase cells, amorphous rDNA FISH characteristic ground (Fig. 3 B). contrast, pds5 cells mid-M                decondensed chromosome detected, mid-M <br /> phase 37&#226;&#172;&#353;C FISH signals, indicating sister              phase haploid cells, FISH signal formed single line, <br /> chromatids precociously dissociated (Fig. 3, B).              characteristic condensed chromosome paired sis- second FISH signal mid-M phase pds5 cells                ter chromatids (Guacci et al., 1994). expected, wild- preexisting aneuploidy G1 phase cells              type cells arrested mid-M phase, FISH signal formed FISH signal (Fig. 3 B). FISH signals               single line edge DNA masses (Fig. 5 . detected nuclei G1 phase wild-type pds5               contrast, pds5-1, pds5-2, pds5-3 cells mid-M phase, <br /> cells likely spurious background. Similar                amorphous FISH signal reminiscent decondensed <br /> sults obtained using probes CEN-proximal                   rDNA detected (Fig. 5  data shown). gions chromosomes  IV, XVI (data shown).                   amorphous FISH signal expected sisters chroma-  Pds5p required sister chromatid cohesion                tids simply dissociated wild-type cells, preco- CEN-proximal distal chromosomal regions.                       ciously dissociated sisters detected line-like sig- <br />    Cells defective mitotic checkpoint precociously               nals (Guacci et al., 1997b). Furthermore, anaphase nuclei <br /> dissociate sister chromatids consequence            cycling wild-type cells, rDNA line-like sis- <br /> failure arrest response spindle damage (Straight et            ter chromatids separated segregated (Guacci et <br /> al., 1996; Yamamoto et al., 1996b).  pds5 mutants               al., 1994).  pds5 cells mid-M phase 37&#226;&#172;&#353;C have functional mitotic checkpoint nocodazole-                  rDNA morphology indicative defect condensation <br /> treated pds5 cells undergo new rounds DNA                     merely dissociation sister chromatids. <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation          619 <br />                                                                                               Figure 5. Analysis chromo- <br />                                                                                              condensation wild- <br />                                                                                              type pds5 cells. Wild- <br />                                                                                              type (VG982-6A) pds5-3 <br />                                                                                              (VG988-1C) cells synchro- <br />                                                                                              nously arrested mid-M                                                                                              37&#226;&#172;&#353;C, described Fig. 3  <br />                                                                                                Micrographs showing FISH <br />                                                                                              rDNA. Mid-M phase cells <br />                                                                                              37&#226;&#172;&#353;C subjected FISH using <br />                                                                                              rDNA probe. Chromosomal <br />                                                                                              DNA (PI) hybridized probe <br />                                                                                              (FITC) shown. Arrows indi- <br />                                                                                              cate line-like rDNA wild-type <br />                                                                                              cells. Bar, 5 &#226;?&#174;m. (B) Micro- <br />                                                                                              graphs showing FISH chro- <br />                                                                                              mosome VIII region. Mid-M <br />                                                                                              phase cells 37&#226;&#172;&#353;C subjected                                                                                              FISH using mixture                                                                                              chromosome VIII probes. Chro- <br />                                                                                              mosomal DNA (gray) hy- <br />                                                                                              bridized probe (white)                                                                                              shown. Bar, 5 &#226;?&#174;m. <br />  <br />  <br />    determined Pds5p required chro-      signals detected DNA masses (Fig. 5 B, mosome condensation unique chromosomal regions.             data shown). increased numbers FISH number spacing FISH signals cells hybrid-        signals expected precocious sister chromatid <br /> ized mixture chromosome VIII XVI probes           dissociation.  sister chromatids remained  <br /> provides qualitative measure chromosome condensa-         densed tight lines clusters FISH <br /> tion. haploid G1 phase cells hybridized chro-     signals, separated condensed sister <br /> mosome XVI probes, widely spaced FISH signals         chromatid. Instead, dispersed FISH signals remi- detected nucleus, mid-M phase hap-        niscent separated decondensed chromosomes <br /> loid cells short line-like FISH signal       (Fig. 5 B). Similar results obtained pds5-1 closely associated FISH signals detected (Guacci et al.,    pds5-2 cells (data shown). Taken  results <br /> 1994). patterns indicate change decon-          FISH using rDNA chromosome VIII XVI <br /> densed chromosome G1 phase condensed chromo-           probes indicate pds5 cells exhibit defects chromo- paired sister chromatids mid-M phase. Wild-       condensation sister chromatid cohesion. <br /> type pds5-3 haploid cells arrested mid-M <br /> phase cells 37&#226;&#172;&#353;C, described rDNA FISH, subjected FISH using mixture probes <br />                                                                Pds5p Localizes Chromosomes Cell chromosome VIII XVI (Materials                                                                Cycle&#226;&#8364;&#8220;dependent Manner Binding Methods). wild-type cells, single line-like FISH signal <br />                                                                Chromosomes Dependent Mcd1p Function closely associated FISH signals detected      defects mitotic chromosome structure detected nuclei (Fig. 5 B, data shown). contrast,    pds5 mutants consistent role Pds5p chro- <br /> pds5-3 cells, larger number widely spaced FISH       mosomal structural protein. determine Pds5p <br />  <br />  Journal Cell Biology, Volume 151, 2000                  620 <br />                                                                        Figure 6. Cell cycle&#226;&#8364;&#8220;dependent localization Pds5p Mcd1p <br />                                                                       chromosomes.   Localization Pds5p chromosomes. <br />                                                                       Strain 2064-2A (PDS5-6HA) grown 23&#226;&#172;&#353;C, ar- <br />                                                                       rested G1, S, mid-M phase treatment &#226;?&#163; factor, <br />                                                                       hydroxyurea, nocodazole, respectively. Aliquots cells                                                                       fixed processed chromosome spreads (Materials                                                                       Methods). Chromosomal DNA (DAPI) Pds5p (Pds5-6HAp) <br />                                                                       shown. *Anaphase cell asynchronous population. <br />                                                                       Bar, 5 &#226;?&#174;m. (B) Localization Mcd1p chromosomes. Strain <br />                                                                       VG2050-1A (MCD1-6HA) grown processed chro- <br /> mosome spreads described  Chromosomal DNA (DAPI) Mcd1p (Mcd1-6HAp) shown. *Anaphase cell asyn- <br /> chronous population. Bar, 5 &#226;?&#174;m. (C) Enlargement mid-M phase cells cultures B. Bar, 1.25 &#226;?&#174;m. (D) Binding Pds5p Mcd1p CEN3 LEU2 loci. Strains grown mid-log phase subjected ChIP using anti&#226;&#8364;&#8220;HA antibody <br /> C12A5, analyzed PCR presence CEN3, LEU2, ADE3 chromosomal loci (Materials Methods). (Total) <br /> PCR total chromatin equivalent 1% used ChIP; positive control PCR. (E) Assay determine Mcd1p function required Pds5p localization chromosomes. Wild-type (VG2064-2A) mcd1-1 mutant (VG2072-2D) strains containing Pds5p- <br /> 6HA arrested G1 phase 23&#226;&#172;&#353;C, transferred 37&#226;&#172;&#353;C, arrested mid-M phase 37&#226;&#172;&#353;C processed chromosome <br /> spreads (Materials Methods). Chromosomal DNA (DAPI) Pds5p (Pds5-6HAp) shown. Bar, 5 &#226;?&#174;m. (F) Assay determine Pds5p function required Mcd1p localization chromosomes. Wild-type (VG2050-1A) pds5 mutant ({VG2073-7D; <br /> pds5-1}, {VG2074-6C; pds5-2}, {VG2075-6D; pds5-3} strains bearing Mcd1p-6HA treated described E. Chromosomal <br /> DNA (DAPI) Mcd1p (Mcd1-6HAp) shown. Bar, 5 &#226;?&#174;m. <br />  <br />  <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation        621 <br />  binds chromosomes, asynchronous culture haploid          asynchronously 23&#226;&#172;&#353;C, Pds5p bound chromosomes strain VG2064-2A arrested G1, S, mid-M                 42% cells, similar wild-type, bound phases. strain contains PDS5-6HA allele          43% cells.  localization Pds5p chromo- <br /> trol PDS5 promoter sole source Pds5p,            somes dependent Mcd1p function. assessed enabled assess chromosomal localization        Mcd1p localization chromosomes depen- <br /> Pds5p using chromosome &#226;&#8364;&#339;spread&#226;&#8364;? technique (Materials            dent Pds5p function. Wild-type VG2050-1A (Mcd1p- Methods). G1 phase cells, little Pds5p detected        6HA) pds5 mutant strains VG2073-7D (pds5-1 Mcd1p- chromosomes (Fig. 6 . contrast, bright Pds5p sig-       6HA), VG2074-6C (pds5-2 Mcd1p-6HA) VG2075-6D <br /> nal detected S mid-M phase cells (Fig. 6          (pds5-3 Mcd1p-6HA) arrested mid-M phase . asynchronous cells, general types         37&#226;&#172;&#353;C processed chromosome spreads Mcd1p DNA masses, elongated dumbbell-shaped DNA masses                  described  Mcd1p showed strong chromosomal lo- late anaphase cells roughly spherical DNA                calization wild-type pds5 mutant strains, indi- <br /> masses cells stages cell cycle. Pds5p    cating Pds5p function required Mcd1p bind- absent elongated DNA masses late                  ing chromosomes (Fig. 6 F).  slight <br /> anaphase cells (Fig. 6 . spherical DNA masses, 42%          difference patterns Mcd1p staining prominent Pds5p staining, remaining lit-        punctate pds5 mutant strains wild-type cells. <br /> tle Pds5p staining, consistent cell cycle&#226;&#8364;&#8220;depen-     subtle difference consequence  <br /> dent chromosomal binding. Finally, cells arrested           densation defect pds5 mutants  alternatively,  <br /> telophase cdc15 mutation, little Pds5-6HAp          flect decrease Mcd1p binding specific sites. tected chromosomes (data shown). Mcd1p, like               conducting experiments determine possibility Pds5p, required sister chromatid cohesion        correct. results demonstrate global localiza- condensation (Guacci et al., 1997b).          tion Pds5p chromosomes dependent wild-type <br /> alyzed localization Mcd1p using haploid strain             Mcd1p function. contrast, bulk Mcd1p localization VG2050-1A, contains Mcd1-6HAp sole                   chromosomes does depend Pds5p function. <br /> source Mcd1p, compared Pds5p (Ma-             chromosome spreads revealed little <br /> terials Methods). Mcd1p bound chromosomes             Pds5p chromosomes G1 phase, late anaphase, cell cycle&#226;&#8364;&#8220;dependent manner indistinguishable        telophase cells. wanted determine Pds5p (Fig. 6 B). staining Pds5p Mcd1p           absence Pds5p cells exclusion appeared brighter chromosomes S mid-M         Pds5p chromatin. address question, asyn- <br /> phase. Interestingly, mid-M phase cells, Pds5p         chronous culture haploid strain VG2063-5C (PDS5- <br /> Mcd1p staining diffuse DAPI ap-             GFP) arrested G1, S, mid-M phases lo- <br /> peared line-like cells (Fig. 6 C). pattern sug-   calization Pds5p directly determined intact cells gestive staining chromosomal axis.        fluorescence microscopy (Materials Methods). Pds5- <br /> sults demonstrate Pds5p localizes chromosomes        GFPp detected single prominent spherical signal <br /> cell cycle&#226;&#8364;&#8220;dependent manner identical Mcd1p.           G1, S, mid-M phase cells (Fig. 7 . intensity    Pds5p Mcd1p localize chromosomes            Pds5p fluorescence lower G1 phase cells ei- <br /> stages cell cycle required cohesion       ther S mid-M phase cells. DNA cells condensation. suggests proteins           stained DAPI, Pds5p localize function binding chromosomal sites.            region DNA, indicating Pds5p nu- test possibility, compared binding Pds5p        clear (data shown). asynchronous culture hap- Mcd1p discrete chromosomal loci. Asynchro-           loid strain VG2069-1B (PDS5-GFP cdc15-2) grown nous cultures VG2064-2A (Pds5-6HAp) VG2050-                23&#226;&#172;&#353;C, transferred 37&#226;&#172;&#353;C 3 h arrest cells <br /> 1A (Mcd1-6HAp) subjected ChIP using anti&#226;&#8364;&#8220;HA               telophase cdc15 mutation (Pringle antibodies. PCR analysis ChIP revealed Pds5p          Hartwell, 1981). spherical Pds5-GFPp signals Mcd1p bound CEN3 DNA sequence near               observed opposite ends cell (Fig. 7 . LEU2 gene, protein bound region near           DNA cells stained DAPI, Pds5p localized ADE3 gene (Fig. 6 D, Megee et al., 1999).            region DNA masses opposite ends, <br /> Pds5p Mcd1p bind specific chromo-             indicating nuclear cells (data shown). <br /> somal loci, suggests function common           examined Mcd1p arresting haploid strain VG2049- <br /> chromosomal sites mediate sister chromatid cohesion.           3A (MCD1-GFP) G1, S, mid-M phase (Materials <br />    determined chromosomal binding             Methods). Mcd1-GFPp detected G1 phase Pds5p dependent Mcd1p function. Asynchro-               cells, formed single prominent round signal S nously growing cultures strains VG2072-2D (mcd1-1              mid-M phase cells (Fig. 7 B). haploid strain <br /> Pds5-6HAp) VG2064-2A (Pds5-6HAp) arrested                VG2061-1D (MCD1-GFP cdc15-2) arrested telo- G1 phase 23&#226;&#172;&#353;C, transferred 37&#226;&#172;&#353;C inactivate        phase described  faint Mcd1-GFPp signal <br /> mutant mcd1p, released G1 arrested           detected, indicating Mcd1p ab- <br /> mid-M phase 37&#226;&#172;&#353;C (Materials Methods). Mid-M                sent cells. Western blot analysis total protein phase cells 37&#226;&#172;&#353;C subjected chromosome spreads          cells bearing Pds5-6HAp Mcd1-6HAp confirmed <br /> cells detect Pds5p (Materials Methods). ex-             cell cycle&#226;&#8364;&#8220;dependent changes Pds5p Mcd1p lev- <br /> pected, Pds5p widely distributed chromosomes            els obtained GFP-tagged proteins (data wild-type cells (Fig. 6 E). contrast, Pds5p failed bind     shown). results demonstrate Pds5p Mcd1p chromosomal regions mcd1 mutant mid-M            bound chromosomes similar stages cell cy- <br /> phase 37&#226;&#172;&#353;C (Fig. 6 E). mcd1 mutant grown          cle.  Pds5p present nucleus stages <br />  <br />  Journal Cell Biology, Volume 151, 2000                     622 <br />                                                                       times sister chromatid cohesion exists. Furthermore, <br />                                                                      Pds5p required maintain cohesion late mid-M <br />                                                                      phase. contrast, Ctf7p, Scc2p, Scc4p, nec- <br />                                                                      essary establishment cohesion, required                                                                      sister chromatid cohesion cell viability S phase                                                                      mid-M phase (Skibbens et al., 1999; Toth et al., <br />                                                                      1999; Ciosk et al., 2000).  Pds5p proper- <br />                                                                      ties cohesin complex members Mcd1p, Smc1p, <br />                                                                      Smc3p, Scc3p. results demonstrate Pds5p                                                                      component molecular glue mediates sister <br />                                                                      chromatid cohesion maintenance cohesion <br />                                                                      requires cohesin complex proteins. <br />                                                                         relationship Pds5p cohesin <br />                                                                      complex?  shown Pds5p Mcd1p lo- <br />                                                                      calize chromosomes cell cycle stages. Inter- <br />                                                                      estingly, localization Pds5p Mcd1p mid-M <br />                                                                      phase cells tighter DAPI, suggesting                                                                      proteins core chromosomal axis <br />                                                                       study). Second, chromatin immunoprecipitation                                                                      Pds5p reveals binds chromosomal DNA sites                                                                      CEN3 near LEU2, near ADE3. binding <br />                                                                      pattern identical Mcd1p (Megee et al., 1999). <br />                                                                      addition, global localization Pds5p chromo- <br />                                                                      somes depends Mcd1p function, Mcd1p localiza- <br />                                                                      tion chromosomes independent Pds5p function. <br /> Figure 7. Localization Pds5p Mcd1p intact yeast cells.      mcd1 pds5 mutants synthetically lethal (V. <br /> Haploid strains VG2063-5C (PDS5-GFP) VG2049-3A <br />                                                                      Guacci, unpublished results).  bimD6 mu- <br /> (MCD1-GFP) grown mid-log phase 23&#226;&#172;&#353;C, ar- <br /> rested G1, S, mid-M phase. Haploid strains VG2069-1B <br />                                                                      tant suppressed mutation SudA gene,  <br /> (cdc15-1 PDS5-GFP) VG2061-1D (cdc15-2 MCD1-GFP)                  codes  nidulans homologues Pds5p Smc3p, grown mid-log phase 23&#226;&#172;&#353;C, arrested telo-      respectively (Holt  1996). results reveal <br /> phase incubation 37&#226;&#172;&#353;C 3 h. Arrested cells fixed   Pds5p cohesin complex proteins functionally localization GFP-tagged proteins determined intact cells   interact. Furthermore, fact Mcd1p binding fluorescence microscopy (Materials Methods). Localiza-        chromosomes independent Pds5p function suggests <br /> tion   Pds5p (B) Mcd1p cells different cell     cohesin complex serve load Pds5p cycle stages.                                                        chromosomal sites, enables Pds5p mediate sister <br />                                                                      chromatid cohesion. cell cycle, Mcd1p prominent levels           homologues Pds5p, BimD6 protein  <br /> times bound chromosomes. fact                 nidulans Spo76 protein Sordaria, Pds5p binds chromosomes times Mcd1p               shown essential mitotic meiotic chromosome <br /> present consistent data showing Pds5p binding            segregation, respectively. BimD6 mutant cells exhibited chromosomes dependent Mcd1p function.                       lethal defects mitotic chromosome segregation                                                                      fewer kinetochore microtubules wild-type cells (Den- <br />                                                                      ison et al., 1993). Spo76p localizes meiotic chromo- <br /> Discussion                                                           somes length, spo76-1 mutant arrests executed cytogenetic screen temperature                 meiosis exhibits defects sister chromatid cohe- <br /> sensitive mutants identify exhibited preco-            sion condensation (van Heemst et al., 1999). cious dissociation sister chromatids. screen led          mitosis, spo76-1 mutant transient defect identification pds1 mcd1 mutants (Yama-               sister chromatid cohesion condensation, ex- <br /> moto et al., 1996a,b; Guacci et al., 1997b). identi-         hibited growth abnormalities. results indicate <br /> fied characterized PDS5, gene identified              Spo76p serve prominent nonessen- screen. PDS5 essential gene vegetative                tial function mitosis spo76-1 allele growth. Pds5p required sister chromatid cohesion           normal mitotic function. Alternatively, centromere proximal centromere distal regions. Pds5p             Pds5p homologue Sordaria performs required chromosome condensation              essential mitotic chromosomal function.  Pds5p <br /> petitive unique chromosomal regions. Pds5 mutant                 homologues play important roles chromosome <br /> cells functional mitotic checkpoint, demon-             structure function eukaryotes. <br /> strates defects cohesion condensation              Previously, proposed cohesion condensa- failure cell cycle arrest. cell cycle stages         tion linked mcd1 mutants defects Pds5p required viability loaded chro-           cohesion condensation (Guacci et al., 1997b). fact <br /> mosomes provides insights role cohesion. Pds5p           pds5 alleles defects cohesion function important viability soon af-        condensation confirms link.  link ter S phase anaphase localizes chromosomes              cohesion condensation general property yeast cell cycle window, encompasses                 chromosomes.  Pds5p Mcd1p homologues <br />  <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation       623 <br />  eukaryotes, likely cohesion&#226;&#8364;&#8220;conden-         fundamental similarities structure sation link conserved. Support idea         mechanism cohesion condensation. <br /> comes Sordaria, spo76-1 mutant exhibited               Second, interactions cohesin condensin <br /> defects cohesion condensation (van Heemst et           complexes exist, missed. cell cy- <br /> al., 1999). Unlike budding yeast, mitotic chromo-           cle window condensin cohesin complex mem- <br /> somes Sordaria detected individual rod-like           bers interact short involve small chromosomes reminiscent eukaryotic chromo-                total protein. addition, putative interactions <br /> somes. conservation chromosome structure eu-             occur proteins bound chromo- <br /> karyotes surprising chromosomal proteins              somes. Finally, interactions soluble com- <br /> involved condensation conserved eu-              plexes weak preserved immu- <br /> karyotes. conservation likely reflects common          noprecipitations fractionations. fact, biochemical <br /> chromosomal architecture required execute struc-         evidence yeast supports interaction cohesin <br /> tural changes eukaryotic chromosomes undergo dur-             condensin complex members. Smc1p Smc2p, <br /> ing mitosis.                                                       members cohesin condensin complexes, respec- <br />    proposed cohesion condensation             tively, immunoprecipitate overex- distinct chromosomal processes soluble             pressed (Strunnikov et al., 1995). addition, condensin cohesin complexes yeast, Xenopus,                 yeast proteins, Trf4p Hec1p, coimmunoprecipitate mammals copurify temporally coexist               Smc1p Smc2p (Castano et al., 1996; Zheng <br /> chromosomes (Hirano et al., 1997; Losada et al., 1998;             et al., 1999). results interactions Schmiesing et al., 1998; Toth et al., 1999). Furthermore, im-      condensin cohesin members occur medi- <br /> munodepletion cohesin complex members interphase             ated common component(s). <br /> Xenopus egg extracts resulted cohesion defects                Finally, discrepancy simply reflect different <br /> chromosomes entered mitosis rod like chromosomes         aspects chromosome condensation monitored. formed, suggested condensation unaffected           explain cohesion&#226;&#8364;&#8220;condensation link, proposed (Losada et al., 1998). results contradict        Mcd1p-containing cohesin complexes cohesion site <br /> link cohesion condensation revealed analy-          associate cohesion site sis budding yeast Sordaria mutants.       chromosome (Guacci et al., 1997b). loop ways explain seemingly contradictory results.               intervening DNA chromosomal axis     chromosome structure funda-          shorten chromosomes. DNA adjacent cohe- <br /> mentally different yeast higher eukaryotes.         sion sites (looped regions) packaged, pre- <br /> lines evidence make unlikely. proteins required        sumably condensin complex.  fully condensed cohesion condensation conserved             mitotic chromosomes require interaction  <br /> eukaryotes (reviewed Koshland Strunnikov, 1996;             hesion proteins adjacent cohesion sites chro- <br /> Koshland Guacci, 2000). addition, cohesin complex           mosome condensin-mediated packaging inter- <br /> members present chromosomes similar cell cycle           vening DNA loops (Fig. 8 . association adjacent <br /> stages budding yeast, Xenopus, human cells.              cohesion sites occur loop packaging hesin complex members bound chromosomes                  packaging loops brought adjacent cohesion sites interphase remain chromosomes, albeit             close proximity. mechanism different, end <br /> lower levels, mitosis (Losada et al., 1998, 2000;           result, shortening chromosomal axis, Schmiesing et al., 1998;  Waizenegger, S. Hauf, J.M.          Mcd1p Pds5p function perturbed, inter- <br /> Peters, unpublished observations). similar pattern            actions adjacent cohesion sites chroma- <br /> seen budding yeast  . Budding yeast               tid altered result axial shortening <br /> mammalian chromosomes undergo cell cycle&#226;&#8364;&#8220;dependent                 (Fig. 8 B).  loops packaged condensation, yeast exhibits smaller level           condensin complex factors rod-shaped <br /> condensation (Guacci et al., 1994). Cohesin binding sites          chromosomes form longer  mapped large chromosomal regions                 mal metaphase chromosomes. used FISH assess budding yeast recent PCR-based chromosomal walk-               length spacing DNA sequences individual <br /> ing revealed sites occur 8&#226;&#8364;&#8220;9 kb non-       chromosome arms overall length centromeric regions (Blat Kleckner, 1999; Laloraya,            spacing increased pds5 mcd1 mutants  S., V. Guacci, D. Koshland, manuscript submitted           study Guacci et al., 1997b). contrast, formation <br /> publication). Cohesin binding sites mapped           rod-like chromosomes taken evidence  higher eukaryotes, likely sites      densation unaffected immunodepletion  widely spaced metaphase chromosomes appear              hesin complex Xenopus extracts (Losada et al., contain 50&#226;&#8364;&#8220;250-kb DNA loops (Earnshaw Laem-                 1998).  axial length individual chromo- <br /> mli, 1983; Gasser et al., 1986).  likely cohesin   somes measured, leaves open possibility <br /> binding sites distributed eukaryotic chromo-             axial shortening chromosomes actually  <br /> somes, different spacing budding yeast             turbed immunodepletion. Clearly, questions <br /> higher eukaryotes. decreased spacing  e., higher          answered cohesion&#226;&#8364;&#8220;condensation link <br /> density) cohesion sites proposed explain             fully understood. <br /> lower levels chromosome condensation budding                    Xenopus, cohesin complex members present yeast (Guacci et al., 1997b). summary, chromosomes              high levels chromosomes interphase, decrease budding yeast higher eukaryotes signifi-             low levels chromosomes mitosis (Losada et al., 1998, <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 151, 2000                      624 <br />                                                                      parts chromosomes metaphase anaphase <br />                                                                     transition, results dissolution cohesion <br />                                                                     (Michaelis et al., 1997; Uhlmann et al., 1999).                                                                     tempting speculate loss Pds5p occurs                                                                     metaphase anaphase transition crucial dis- <br />                                                                     solution cohesion.  chromosomes decon- <br />                                                                     dense late anaphase Pds5p Mcd1p  <br />                                                                     quired condensation  study; Guacci et al., 1994, <br />                                                                     1997b).  essential correlate precise <br />                                                                     time Pds5p Mcd1p depart chromosomes <br />                                                                     state chromosome condensation. Unfortu- <br />                                                                     nately, budding yeast, metaphase anaphase tran- <br />                                                                     sition easily detected individual chro- <br />                                                                     mosomes visible conventional microscopy. <br />                                                                     Yeast cells S phase midmitosis round <br />                                                                     DNA mass short spindle. Just metaphase                                                                     anaphase, centromere proximal sequences sister chro- <br />                                                                     matids rapidly reach opposite poles prominent <br />                                                                     elongation DNA mass spindle occurs yeast <br />                                                                     (Guacci et al., 1997a; Straight et al., 1997).                                                                     metaphase anaphase transition, cells <br />                                                                     elongated spindles DNA masses,                                                                     sure sister chromatids separated.                                                                     late anaphase cells Pds5p                                                                     Mcd1p detected chromosomes  study; <br /> Figure 8. Model chromosome condensation.   Condensa-          Michaelis et al., 1997; Uhlmann et al., 1999).  tion wild-type cells.   Decondensed chromosome sis-     known departure Pds5p Mcd1p <br /> ter chromatid cohesion. Chromosome condensation begins ei-       occurs precise time sister cohesion dissolved <br /> ther packaging DNA cohesion sites (middle left)        metaphase anaphase transition) oc- association adjacent cohesion sites (middle right).   curs later mitosis chromosomes decondense (late <br /> fully condensed mitotic chromosome   requires       anaphase telophase). precise timing deter- <br /> association adjacent cohesion sites packaging     mined departure Pds5p Mcd1p chro- <br /> DNA cohesion sites. (B) Condensation pds5 mcd1        mosomes, effect departure chromo- <br /> mutants.   Decondensed chromosome dissociated sister <br />                                                                     structure remains open question. <br /> chromatids. chromosomes undergo condensation, DNA cohesion sites packaged, adjacent         finding Pds5p molecular glue  <br /> cohesion sites associate.                                    sponsible maintenance sister chromatid cohe- <br />                                                                     sion demonstrates cohesin complex proteins <br />                                                                     represent mechanism mediates  <br /> 2000). Pds5p Mcd1p present high levels yeast          hesion. characterization Pds5p relation- <br /> chromosomes S mid-M phase  study; Michaelis             ship cohesin complex provide valuable insights <br /> et al., 1997). qualitative analysis Pds5p Mcd1p          cohesion maintained ultimately dissolved. <br /> levels chromosome spreads suggests levels            end, examining Pds5p proteins bound chromosomes decrease               new member cohesin complex, loads S mid-M phase. result                chromosomes complex added  <br /> firmed quantitative methods, suggests            wards. testing genetic interactions yeast Xenopus, chromosomal levels proteins              Pds5p cohesin condensin complex members mediate sister chromatid cohesion decrease mitosis.          yeast help dissect link cohesion  chromosomes condensed mitosis,               densation. <br /> crease cohesin complex correlates condensation.                                                                     thank P. Megee B. Lavoie comments manuscript. possible decrease required proper <br />                                                                        D. Koshland supported Howard Hughes Medical Institute, <br /> chromosome condensation. Yeast chromosomes undergo                  V. Guacci, T. Hartman, K. Stead supported National condensation mitosis mammalian chromo-             Institutes Health grant CA06927 Commonwealth Pennsyl- <br /> somes (Guacci et al., 1994).  possible     vania. dramatic decrease cohesin complex Xenopus <br />                                                                     Submitted: 28 June 2000 consequence higher level chromosome <br />                                                                     Revised: 25 September 2000 <br /> condensation occurs budding yeast.                          Accepted: 25 September 2000 <br />    Pds5p bound chromosomes mid-M phase cells <br /> (nocodazole arrested), sister cohesion present,                                                                     References absent chromosomes late anaphase, sisters separated. Similar results observed Mcd1p              Blat, Y., N. Kleckner. 1999. Cohesins bind preferential sites yeast <br />                                                                        chromosome III, differential regulation arms versus centric <br />  study; Michaelis et al., 1997; Uhlmann et al., 1999).         region. Cell. 98:249&#226;&#8364;&#8220;259. suggested Mcd1p/Scc1p cleaved               Castano, , P. Brzoska, B. Sadoff, H. Chen, M. Christman. 1996. Mitotic <br />  <br />  <br />  <br />  <br /> Hartman et al. Pds5p Required Cohesion Condensation      625 <br />     chromosome condensation rDNA requires TRF4 DNA topo-                 Moore, D.P., W. Page, T.T. Tang, W. Kerrebrock, T.L. Orr-Weaver. <br />    isomerase Saccharomyces cerevisiae. Genes Dev. 10:2564&#226;&#8364;&#8220;2576.                  1998. cohesion protein MEI-S332 localizes condensed meiotic Chuang, P.T., D.G. Albertson, B.J. Meyer. 1994. DPY-27: chromosome              mitotic centromeres sister chromatids separate. J. Cell Biol. 140:1003&#226;&#8364;&#8220; <br />    condensation protein homolog regulates C. elegans dosage compensa-            1012. <br />    tion association X chromosome. Cell. 79:459&#226;&#8364;&#8220;474.               Ouspenski, I., O  Cabello, B.R. Brinkley. 2000. Chromosome conden- <br /> Ciosk, R., M. Shirayama,  Shevchenko, T. Tanaka,  Toth,  Shevchenko,            sation factor brn1p required chromatid separation mitosis. Mol. <br />    K. Nasmyth. 2000. Cohesin&#226;&#8364;&#8482;s binding chromosomes depends sep-           Biol. Cell. 11:1305&#226;&#8364;&#8220;1313. <br />    arate complex consisting Scc2 Scc4 proteins. Mol. Cell. 5:243&#226;&#8364;&#8220;254.       Pringle, J.R., L.H. Hartwell. 1981. Saccharomyces cerevisiae cell cycle. <br /> Denison, S.H., E. Kafer, G.S.  1993. Mutation bimD gene          Molecular Biology Yeast Saccharomyces: Life Cycle  <br />    pergillus nidulans confers conditional mitotic block sensitivity DNA      heritance. J.N. Strathern, E.W. Jones, J.R. Broach, editors. Cold Spring <br />    damaging agents. Genetics. 134:1085&#226;&#8364;&#8220;1096.                                          Harbor Laboratory, Cold Spring Harbor, NY. 97&#226;&#8364;&#8220;142. <br /> Donze, D., C.R. Adams, J. Rine, R.T. Kamakaka. 1999. boundaries         Saitoh, N., G. Goldberg, E.R. Wood, W.C. Earnshaw. 1994. ScII:    silenced HMR domain Saccharomyces cerevisiae. Genes Dev. 13:698&#226;&#8364;&#8220;            abundant chromosome scaffold protein member family puta- <br />    708.                                                                               tive ATPases unusual predicted tertiary structure. J. Cell Biol. 127: <br /> Earnshaw, W.C., U.K. Laemmli. 1983. Architecture metaphase chromo-             303&#226;&#8364;&#8220;318. <br />    somes chromosome scaffolds. J. Cell Biol. 96:84&#226;&#8364;&#8220;93.                         Saka, Y., T. Sutani, Y. Yamashita, S. Saitoh, M. Takeuchi, Y. Nakaseko, M. <br /> Funabiki, H., H. Yamano, K. Kumada, K. Nagao, T. Hunt, M. Yanagida.               Yanagida. 1994. Fission yeast cut3 cut14, members ubiquitous <br />    1996. Cut2 proteolysis required sister chromatid separation fission         protein family, required chromosome condensation segregation <br />    yeast. Nature. 381:438&#226;&#8364;&#8220;441.                                                        mitosis. EMBO (Eur. Mol. Biol. Organ.) J. 13:4938&#226;&#8364;&#8220;4952. <br /> Furuya, K., K. Takahashi, M. Yanagida. 1998. Faithful anaphase ensured      Schmiesing, J , R. Ball, H.C. Gregson, J.M. Alderton, S. Zhou, K. <br />    Mis4, sister chromatid cohesion molecule required S phase          Yokomori. 1998. Identification distinct human SMC protein com- <br />    destroyed G1 phase. Genes Dev. 12:3408&#226;&#8364;&#8220;3418.                                    plexes involved mitotic chromosome dynamics. Proc. Natl. Acad. Sci. <br /> Gasser, S.M., T. Laroche, J. Falquet, E. Boy la Tour, U.K. Laemmli.            USA. 95:12906&#226;&#8364;&#8220;12911. <br />    1986. Metaphase chromosome structure. Involvement topoisomerase II.          Selig, S., K. Oku <br /> </body></html>