<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Dec. 2007, p. 8575&#226;&#8364;&#8220;8583                                                                               Vol. 189,  23 <br /> 0021-9193/07/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00653-07 <br /> Copyright &#194;&#169; 2007, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />      Reassessment Vivo Functions DNA Polymerase                   RNase H Bacterial Cell Growth&#228;&#338;&#164;&#226;&#8364;&#160; <br />  Sanae Fukushima,1 Mitsuhiro Itaya,2 Hiroaki Kato,1 Naotake Ogasawara,3 Hirofumi Yoshikawa1* <br />  Department Bioscience, Tokyo University Agriculture, Sakuragaoka 1-1-1, Setagaya-ku, Tokyo 156-8502, Japan1; Institute         Advanced Biosciences, Keio University, 403-1, Nipponkoku, Daihoji, Tsuruoka, Yamagata 997-0017, Japan2;             Department Bioinformatics Genomics, Graduate School Information Science, Nara Institute                            Science Technology, Takayama 8916-5, Ikoma, Nara 630-0101, Japan3 <br />                                             Received 26 April 2007/Accepted 18 September 2007 <br />  <br />              major factor removing RNA primers processing Okazaki fragments DNA polymerase <br />           (Pol . Pol thought remove RNA primers resulting gaps simultaneously. RNase H, <br />           encoded rnh genes, factor removing RNA primers, disagreement respect <br />           essentiality polA rnh genes. previous study, looked synthetic lethality           paralogs Bacillus subtilis detected essential doublet paralogs, including polA ypcP pair. YpcP <br />           consists 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain. current study, confirmed polA genes           Escherichia coli B. subtilis completely deleted. 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity <br />           encoded polA ypcP xni required growth B. subtilis E. coli.  5&#226;&#180;&#8226;-3&#226;&#180;&#8226; <br />           exonuclease activity Pol indispensable cyanobacterium Synechococcus elongatus. results <br />           suggest 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity essential organisms. success constructing B. subtilis <br />           strain lacked RNase H genes indicates enzymatic activity dispensable, wild <br />           type. Increasing 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease activity partially compensated defective phenotype RNase <br />           H-deficient mutant, suggesting cooperative functions enzyme systems. search distri- <br />           bution 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain 250 bacterial genomes resulted finding eubacteria, <br />           archaea, possess domain. <br />  <br />  <br />    Structural characterization biochemical studies                  determination RNA primer length 9 12 nucleotides <br /> eral prokaryotic DNA polymerase (Pol  PolA) estab-                      (20). Later progress revealed enzymes variety <br /> lished organization functional domains: N-ter-                  prokaryotic eukaryotic organisms classified <br /> minal domain associated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity,                   major families, type 1 type 2 RNase H (31). E. coli <br /> central domain mediates proofreading 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonu-                  RNase HI, encoded rnhA (17), belongs type 1  <br /> clease activity, C-terminal domain responsible                   zymes; E. coli RNase HII, encoded rnhB (14), Bacillus <br /> polymerase activity (33). combination activities                   subtilis RNase HII (rnhB) HIII (rnhC) (30) categorized <br /> derived terminal domains, called nick trans-                  type 2 enzymes. Biochemical characterization lation activity, Pol thought act simultaneously               enzymes suggested type 1 type 2 RNase H func- <br /> removing RNA primers filling resulting gaps. Evi-                   tionally related. <br /> dently, Escherichia coli polA1 mutant (7), lacks poly-                  Previous studies disagreed regarding essentiality merase activity 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity, able                polA rnh genes. polA(Ex1) mutant described <br /> grow, accumulated Okazaki fragments, prob-                    exhibited temperature-sensitive growth, indicating  <br /> ably inability gaps (32). addition,                  quirement 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease catalytic domain Pol  <br /> polA(Ex1) mutant, lacks 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity                     polymerase 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain <br /> 43&#194;&#176;C, accumulated Okazaki fragments grow                    required growth rich media (16). hand, high temperatures (23).                                                     E. coli genome analysis, complete knockout strain <br />    RNase H specifically cleaves RNA strand RNA/DNA                      polA grew normally LB medium obtained,  <br /> hybrids plays role removing RNA primers Okazaki                    vealing dispensability polA (28).  fragments (29), process ribonucleoti-                 descriptions molecular mechanism(s) involved <br /> des DNA-RNA junction sites. fact, isolation                 processing RNA primers polA mutants, double mutant polA rnh  encodes RNase H)                        studies involving work polA rnh double mutant possible detect RNA primers contributed                  described support initiation replication <br />                                                                                plasmids chromosomes.  rnhA                                                                                rnhB genes dispensable E. coli presence polA; <br />    * Corresponding author. Mailing address: Department Bioscience,          hand, rnhB rnhC genes B. subtilis <br /> Tokyo University Agriculture, Sakuragaoka 1-1-1 Setagaya-ku, Tokyo          manifested synthetic lethality (15). <br /> 156-8502, Japan. Phone: 81-3-5477-2758. Fax: 81-3-5477-2668. E-mail:              number microbial-genome-sequencing projects  <br /> hiyoshik@nodai.ac.jp. <br />    &#226;&#8364;&#160; Supplemental material article http://jb <br />                                                                                vealed bacterial genomes consist half singlet <br /> .asm.org/.                                                                     genes half paralogous genes (24). comprehensive <br />    &#228;&#338;&#164; <br />      Published ahead print 28 September 2007.                            study, looked synthetic lethality paralogs  <br />                                                                         8575 <br />  8576       FUKUSHIMA ET AL.                                                                                                                             J. BACTERIOL. <br />  <br />  <br /> B. subtilis, detected essential functions doublet para-                    PCR-generated fragment containing antibiotic gene marker (primers  logs (Y. Shiwa, S. Fukushima, H. Yoshikawa, unpublished                           -Rev). case E. coli, strain KM22 used host dis- <br />                                                                                       ruptants obtained homologous recombination provided &#226;?&#173; red func- <br /> data); polA ypcP pair. YpcP protein                         tion (27) PCR-generated fragments. Fragments containing antibiotic resis- <br /> categorized 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease (http://bacillus.genome.jp/);                      tance gene markers amplified pair 5&#226;&#172;&#732; 3&#226;&#172;&#732; primers 40 considered attempt resolve discrepancy                          bases upstream downstream target gene attached  Table S1 polA dispensable 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity                       supplemental material). PCR products directly used trans- <br />                                                                                       form KM22. templates antibiotic markers B. subtilis Tn554 essential function maturation Okazaki frag- <br />                                                                                       Staphylococcus aureus spectinomycin (spc) resistance; pC194 S. aureus <br /> ments. essentiality 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease function B.                       chloramphenicol resistance (cat); pMUTIN-T3CC (K. Asai, unpublished <br /> subtilis suggested Duigou et al. (9), reported                        data), derivative pMUTIN4 (42) erythromycin (erm) resistance; efficient transformation &#226;&#338;&#172;polA strain partially                      pBEST309 (13) Enterococcus faecalis tetracycline (tet) resistance. deleted ypcP gene pMUTIN disruptant DNA.                                 templates markers E. coli R100 plasmid tetracycline (tet) resis- <br />                                                                                       tance synthetic plasmid pHSG399 chloramphenicol (cat) resistance. <br /> tailed analysis reported  used complete disruptants                                                                                       Plasmid pUC4K (Pharmacia) used confer kanamycin (kan) resistance entire genes domains genes inducible gene                             Synechococcus. generate desired PCR construct, used recombinant <br /> expression                                                                     PCR method (12), secondary-stage PCR products generated <br />    addition rnh genes described                         piece primary PCR fragments. constructs gene disruptions genes code RNase H B. subtilis                               verified diagnostic PCR assay. PCR primers used listed                                                                                       Table S1 supplemental material. Pyrobest DNA polymerase ExTaq <br /> genome.  ypdQ, similar sequence type 1                          DNA polymerase (TaKaRa, Shiga, Japan) used PCR. <br /> enzyme genes, reported lack critical                        Construction strains inducible expression  B. subtilis <br /> amino acids active-site residues purified                         strain harboring ypcP control IPTG-inducible promoter protein exhibits RNase H activity (30). surprise,                        constructed follows. 321-bp region containing ribosomal binding se- <br />                                                                                       quences 5&#226;&#172;&#732; terminus ypcP amplified PCR primer pair <br /> additional RNase H gene, ypeP, B. subtilis <br />                                                                                        Table S1 supplemental material)  digestion HindIII genome reassignment open reading frame (S. Ish-                             BamHI, cloned HindIII/BamHI sites pMUTIN-T3CC. resulting <br /> ikawa, personal communication). current study,                        plasmid used transform B. subtilis strain 168 erythromycin resistance confirmed polA genes E. coli B. subtilis                         obtain strain NBS5014. Consequently, integrant, ypcP placed completely deleted.  demonstrated                             control IPTG-inducible spac promoter, promoter activity                                                                                       ypcP itself monitored &#226;?&#164;-galactosidase activity (41). Alternatively, <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity indispensable B. subtilis                    B. subtilis strain harboring polA, N-terminal region polA, ypcP E. coli cyanobacteria, Synechococcus                        control IPTG-inducible promoter Phyper-spank (derived pDR111a) <br /> elongatus strain PCC 7942, suggesting essential function                           amyE locus &#226;&#338;&#172;polA mutant, constructed. fragments  <br /> common organisms. Finally, constructed B. subtilis                        taining length polA, 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA, ypcP                                                                                       amplified PCR primer pairs cloned pDG148  strain depleted RNase H genes. findings indicate <br />                                                                                       digestion XhoI SalI HindIII, cloned SalI/HindIII sites RNase H enzymatic activity dispensable,                       pDR111a (gift M. Fujita, University Houston, Houston, TX) (5, 10). wild type. Based findings, discuss                      resulting plasmids used transform B. subtilis strain NBS182 obtain <br /> cooperation 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease RNase H functions                           strain NBS247-249. vivo.                                                                                 SOS induction assay. indicator SOS induction, expression yneA <br />                                                                                       analyzed transcriptionally fused lacZ reporter assay. control <br />                                                                                       experiments, strain YNEAd grown optical density 600 nm (OD600) <br />                                                                                       0.35 LB medium 37&#194;&#176;C treated 50 ng/ml <br />                         MATERIALS METHODS <br />                                                                                       mitomycin C 10 min. mutagen-treated cells harvested resus- <br />    Bacterial strains, plasmids, genetic technique. bacterial strains used     pended fresh medium, culture continued. indicated times, study listed Table 1. B. subtilis strains maintained Schaef-   aliquots withdrawn &#226;?&#164;-galactosidase activity measured. Simulta- <br /> fer&#226;&#8364;&#8482;s sporulation medium (37) transformed chromosome plasmid              neously, NBS218 grown absence mitomycin C; culture <br /> DNA according method Anagnostopoulos Spizizen (1). Selection,           reached OD600 withdrawn YNEAd, aliquots assayed required, erythromycin (1 &#226;?&#174;g/ml); kanamycin, chloram-             &#226;?&#164;-galactosidase activity. <br /> phenicol, neomycin (5 &#226;?&#174;g/ml); tetracycline (15 &#226;?&#174;g/ml); spectino-            Bioinformatics. genome sequences obtained GenomeNet <br /> mycin (100 &#226;?&#174;g/ml). necessary, genes overexpressed cloning            (http://www.genome.ad.jp/); domain structures predicted SMART multicopy plasmid pDG148 (40) derivative, pAN18,             program (http://smart.embl-heidelberg ; 25, 38). hidden Markov model <br /> contains multiple cloning sites pUC18 HindIII site pDG148         (HMM) profiles 3_5_exonuclease (PF01612) DNA_pol_A (PF00476) <br /> induced 1 mM IPTG (isopropyl-&#226;?&#164;-D-thiogalactopyranoside). lacZ ex-            obtained Pfam (http://www.sanger.ac.uk/Software/Pfam/). built pression experiment performed growing B. subtilis cells 2&#226;&#171;&#187; SG medium        custom profile HMM 5_3_exonuclease based multiple sequence align- <br /> (2) 37&#194;&#176;C, aliquots culture subjected &#226;?&#164;-galactosidase activity   ment 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain generated SMART program <br /> assay previously described (2).                                                    (SMART accession  SM00475). Motif searches genomes  <br />    E. coli K-12 strain KM22, derived AB1157 (27), grown LB                formed using HMMER program; e-value cutoff 0.01. <br /> medium supplemented 25 &#226;?&#174;g/ml kanamycin, necessary, 20 &#226;?&#174;g/ml <br /> chloramphenicol 25 &#226;?&#174;g/ml tetracycline added. Transformation carried described previously (27). <br />    cyanobacterium S. elongatus PCC 7942 grown BG-11 (6) supple- <br />                                                                                                                         RESULTS <br /> mented, necessary, 10 &#226;?&#174;g/ml kanamycin. Transformation Synecho- <br />                                                                                         5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease domain polA ypcP es- <br /> coccus according method Porter (34). designed primer <br /> sequences based information contained database closely          sential B. subtilis. genome B. subtilis, ypcP related strain S. elongatus PCC 6301 (http://cyano.genome.jp/).                       paralog polA; initial attempts obtain    Construction strains carrying various deletion mutations. Gene disruptant       double-knockout mutant genes failed.  <br /> strains B. subtilis S. elongatus PCC 7942 constructed homologous      constructed polA null mutant strain NBS5015, recombination PCR-generated fragments. primary PCR-generated frag- <br /> ments contained 600 1,000 bp upstream (using primers &#226;&#171;&#186;11                                                                                       ypcP placed IPTG-inducible promoter, shown <br /> &#226;&#171;&#186;12)  Table S1 supplemental material) downstream (primers &#226;&#171;&#186;23          Fig. 1A. NBS5015 grow medium containing &#226;&#171;&#186;24) sequences target gene, overlap end        IPTG; Fig. 1B shows growth defect IPTG depletion. <br />  VOL. 189, 2007                                                        REASSESSMENT Pol RNase H FUNCTIONS                 8577 <br />  <br />  <br />                                 TABLE 1. Genotype bacterial strains plasmids used study <br />         Strain plasmid                                            Genotype                                          Source <br />  <br /> B. subtilis strains <br />   168                                 trpC2                                                                    Laboratory stock <br />   NBS5004                             trpC2   polA::spc                                                        study <br />   NBS182                              trpC2   polA::erm                                                        study, <br />   POLAd                               trpC2   pMUTIN-polA erm                                                  22 <br />   NBS5014                             trpC2   pMUTIN-T3CC-ypcP erm                                             study <br />   NBS5015                             trpC2   polA::spc pMUTIN-T3CC-ypcP erm                                   study, NBS50143 <br />                                                                                                                  NBS5004 <br />   NBS4070                             trpC2   ypcP::spc                                                        study <br />   NBS183                              trpC2   polAC::cat                                                       study <br />   NBS184                              trpC2   polAN::spc                                                       study <br />   NBS185                              trpC2   polAC::cat ypcP::spc                                             study, NBS1833 <br />                                                                                                                  NBS4070 <br />   NBS202                              trpC2 rnhB::spc                                                          study <br />   BEST218 1A1plus                     rnhB21::neo                                                              15 <br />   NBS203                              trpC2 rnhB21::neo                                                        study, BEST218 <br />                                                                                                                   1A1 plus3168 <br />   NBS204                              trpC2   rnhC::cat                                                        study <br />   NBS205                              trpC2   ypdQ::spc                                                        study <br />   NBS206                              trpC2   ypeP::tet                                                        study <br />   NBS207                              trpC2   rnhB::spc rnhC::cat                                              study, NBS2043 <br />                                                                                                                   NBS202 <br />   NBS208                              trpC2 rnhB::spc ypeP::tet                                                study, NBS2063 <br />                                                                                                                   NBS202 <br />   NBS209                              trpC2 rnhC::cat ypeP::tet                                                study, NBS2043 <br />                                                                                                                   NBS206 <br />   NBS210                              trpC2 rnhB::spc rnhC::cat ypeP::tet                                      study, NBS2043 <br />                                                                                                                   NBS208 <br />   NBS216                              trpC2 rnhB21::neo rnhC::cat ypeP::tet ypdQ::spc                          study, NBS2043 <br />                                                                                                                   NBS224 <br />   NBS217                              trpC2 rnhB21::neo rnhC::cat ypeP::tet ypdQ::spc polA::erm                study, NBS1823 <br />                                                                                                                   NBS216 <br />   NBS218                              trpC2 rnhB::spc rnhC::cat ypeP::tet pMUTIN-yneA erm                      study, YNEAd3 <br />                                                                                                                   NBS210 <br />   NBS223                              trpC2 rnhB21::neo ypdQ::spc                                              study, NBS2053 <br />                                                                                                                   NBS203 <br />   NBS224                              trpC2 rnhB21::neo ypeP::tet ypdQ::spc                                    study, NBS2063 <br />                                                                                                                   NBS223 <br />   YNEAd                               trpC2 pMUTIN-yneA erm                                                    22 <br />   NBS247                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-polA spc::&#226;&#172;&#732;amyE                           study, pDR111a- <br />                                                                                                                   polA3NBS182 <br />   NBS248                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-polA-N spc::&#226;&#172;&#732;amyE                         study, pDR111a- <br />                                                                                                                  polAN3NBS182 <br />   NBS249                              trpC2 polA::erm amyE&#226;&#172;&#732;::pDR111a-ypcP spc::&#226;&#172;&#732;amyE                           study, pDR111a- <br />                                                                                                                  ypcP3NBS182 <br />  <br /> E. coli strains <br />   KM22                                argE3 4 leuB6 proA2 thr-1 ara-14 galK2 lacY1 mtl-1 xyl-5 thi-1       27 <br />                                         rpsL31 tsx-33 supE44 &#226;&#338;&#172;(recC ptr recB recD)::Plac-bet exo kan <br />   NBS198                              polA::cat                                                                study <br />   NBS200                              xni::tet                                                                 study <br />  <br /> S. elongatus PCC 7942 strains <br />    NBC100                             Wild type                                                                Laboratory stock <br />    NBC318                             &#226;&#338;&#172;polAC::kan                                                              study <br />  <br />  <br /> Plasmids <br />   pDG148                              bla   kan                                                                40 <br />   pAN18                               bla   kan                                                                study <br />   pMUTIN-T3CC                         bla   erm                                                                Kei Asai <br />   pDR111a                             bla   spc                                                                Masaya Fujita <br />  <br />  <br />  <br />   ypcP identified polA paralog, encoded          terminal domains PolA (designated &#226;&#338;&#172;polAC) &#226;&#338;&#172;ypcP <br /> small protein corresponded 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease           background. succeeded constructing strain NBS185. domain PolA; enzymatic activity appeared               results indicated ypcP 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> quired viability. verify  tried delete C-         domain PolA sufficed cell survival. <br />  8578       FUKUSHIMA ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />        TABLE 2. Test sensitivity B. subtilis strains                   treatment (data shown). contrast E. coli polA <br />                           mitomycin Ca                                              (43). noted global mobility shift assay   Strain                   Relevant genotype                  Surviving fraction    DNA microarray analysis identified LexA binding site                                                                                     upstream region B. subtilis polA (3). <br /> 168                            Wild type                             0.54 <br /> NBS5004                        &#226;&#338;&#172;polA                             2.2 &#226;&#171;&#187; 10&#226;&#171;&#186;8 <br />                                                                                        xni E. coli counterpart ypcP. E. coli ortholog NBS4070                        &#226;&#338;&#172;ypcP                                 0.27           ypcP xni; reported encode exonuclease <br />                                                                                       (http://genolist.pasteur.fr/Colibri/). Based results     Cells grown exponentially (optical density 600 nm, 0.4) 37&#194;&#176;C LB <br /> medium supplemented appropriate antibiotics. Mitomycin C sensitivity       B. subtilis, tested synthetic lethality polA tested plating serial dilutions culture LB LB supplemented       xni applied E. coli.  constructed polA xni 50 ng/ml mitomycin C.                                                          disruptant E. coli KM22 independently.  trans- <br />                                                                                     formed xni disruptant &#226;&#338;&#172;polA chromosomal DNA                                                                                     obtain double-knockout strain;  effort failed. <br />    Characterization polA ypcP mutants ex-                          phenotype polA disruptant constructed pression profiles. E. coli polA null mutant manifests tem-                      similar reported Joyce Grindley (16) perature-sensitive growth (16). analyzed growth pro-                         grew LB medium rendered competent <br /> files various B. subtilis polA mutants combination                   slow growth minimal medium. distinct <br /> ypcP mutant. shown Fig. 2B, entire polA disruptant                        transformant obtained, control experiments (&#226;&#338;&#172;polA) exhibited temperature-sensitive growth 56.5&#194;&#176;C                       transformation wild-type strain using DNA B. subtilis. hand, &#226;&#338;&#172;ypcP mutant able                     yielded transformants. example, <br /> grow temperature, wild type. Interestingly,                      used fragment containing polA::cat cassette polA mutant (&#226;&#338;&#172;polAN) lacked N-terminal                            transform KM22, obtained approximately 100 transfor- <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain temperature sensitive (Fig.                       mants 1 &#226;?&#174;g DNA. hand, used 2B, NBS184); true mutant lacking Klenow                       DNA fragment transform NBS200 (xni::tet), fragment (&#226;&#338;&#172;polAC; NBS183).  growth phenotype                           colonies appeared confirmed disrupted strain &#226;&#338;&#172;polAC &#226;&#338;&#172;ypcP                        polA xni genes.  recipient cells yielded wild type.  results demonstrated                    normal number (&#226;&#172;&#381;100) transformants genetic 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA necessary cell                         markers.  postulated transformants <br /> growth high temperatures.                                                        suppressors concluded polA xni exhibited <br />    ypcP gene encodes 296 <span id='am-2' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> residues                          synthetic lethality. results indicated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> 70% amino acid sequence homology 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease                         activities encoded polA xni essential domain Pol  described  lack polA 5&#226;&#172;&#732;-                           E. coli. <br /> terminal region gave rise temperature-sensitive cell growth,                        N-terminal domain cyanobacterial polA indispens- &#226;&#338;&#172;ypcP did  suggesting natures                       able. genome cyanobacterium S. elongatus PCC <br /> enzymes different. shown Fig. 3, expression                          7942 does encode YpcP ortholog.  exam- <br /> profiles genes similar. Note expression                    ined essentiality polA (Synpcc7942_0194). described <br /> level ypcP-lacZ &#226;&#338;&#172;polA background slightly higher                      Materials Methods, attempted construct  wild type; reflect compensation lack 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity. Consequently,  <br /> structed polA- ypcP-inducible strain amyE locus. shown Fig. 2C, &#226;&#338;&#172;polA background (NBS182), <br /> strains harboring polA N-terminal fragment suppressed temperature sensitivity (Fig. 2C, c d), did control strains <br /> carrying length polA (Fig. 2C, b). hand, <br /> ypcP-expressing strains (Fig. 2C, e f) unable sup- <br /> press phenotype &#226;&#338;&#172;polA mutant completely. Note ypcP-expressing strain partially suppressed tem- <br /> perature sensitivity temperature (Fig. 2C, e f). results confirmed temperature sensitivity lack 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain polA enzymes differed slightly nature. described  multicopy ypcP, polA, partially suppressed RNase H deficiency;  enzymes share func- <br /> tions, natures differ somewhat. <br />    function Pol repair enzyme SOS                         FIG. 1. Growth defect polA ypcP double mutant.   Scheme <br /> response. Presumably, function derived Klenow                      showing gene structures polA ypcP loci NBS5015. <br /> fragment Pol YpcP involved. indicated                         (B) Growth profiles strain NBS5015. NBS5015 streaked LB <br /> Table 2, &#226;&#338;&#172;polA strain showed profound sensitivity                        plates containing 0.01 mM IPTG incubated overnight 37&#194;&#176;C. LB <br />                                                                                     medium (open circles) (closed circles) 1 mM IPTG <br /> DNA-damaging agent mitomycin C; reaction &#226;&#338;&#172;ypcP                           inoculated diluted aliquots overnight culture (starting <br /> strain comparable wild type. addition,                    OD600 &#226;&#171;&#189; 0.008) incubated 37&#194;&#176;C. Growth monitored OD <br /> expression genes induced mitomycin C                       measurements. <br />  VOL. 189, 2007                                                          REASSESSMENT Pol RNase H FUNCTIONS                     8579 <br />  <br />  <br />  <br />  <br />    FIG. 2. Growth various polA ypcP mutants LB plates 37&#194;&#176;C   56.5&#194;&#176;C (B). Relevant genotypes shown: &#226;&#338;&#172;polA, polA deletion <br /> strain NBS5004; &#226;&#338;&#172;ypcP, ypcP deletion strain NBS4070; &#226;&#338;&#172;polAC, polA Klenow fragment deletion strain NBS183; &#226;&#338;&#172;polAN, polA 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease <br /> domain deletion strain NBS184; &#226;&#338;&#172;ypcP, &#226;&#338;&#172;polAC, Klenow fragment ypcP deletion strain NBS185. (C) Alternatively, polA deletion strain <br /> NBS182 derivatives examined LB plates containing 1 mM IPTG 56.5&#194;&#176;C. independent transformants   NBS247 <br /> (amyE::polA)  b), NBS248 (amyE::polAN) (c d), NBS249 (amyE::ypcP) (e f) shown. <br />  <br />  <br />  <br /> tire polA gene knockout strain 7942polA series primer           primers, process ribonucleotides C-terminal Klenow domain disruptant using                    DNA-RNA junction. genes encoding RNase H 7942polAc series primer  Table S1 supplemental                B. subtilis genome, e., rnhB, rnhC, ypdQ, ypeP, material). cyanobacteria contain multiple copies            identified. YpeP, similar RNase HI, newly <br /> chromosome, usually possible obtain disruptant colonies         identified; possessed known conserved residues involved essential gene.  successive patch-         catalysis E. coli RnhA, e., D10, E48, D70, H124, ings, nonessential genes completely replaced. Using        D134 (Fig. 4). RNase H gene disruptant exhibited criterion, C-terminal disruptant stably maintained,         normal growth phenotype, rnhC ypeP mutants PCR analysis confirmed complete replacement ge-               formed small colonies, complete medium. nome mutant allele. resulting strain named              reported rnhB rnhC genes confer syn- <br /> NBC318. hand, entire polA gene disruptant               thetic lethality cells B. subtilis, unlike E. maintained stably, PCR analysis colo-          coli, RNase H indispensable (15).  obtained B. <br /> nies revealed wild-type allele polA retained (data       subtilis strain lacked rnhB rnhC, shown). results demonstrated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucle-          colony size small cell morphology ase domain Synechococcus Pol essential growth.               filamentous.  disruptants RNase <br />    RNase H function dispensable B. subtilis. RNase H,              H genes able grow 37&#194;&#176;C (Table 3). Mutants lacking <br /> encoded rnh genes, factor removing RNA              rnhB rnhC showed temperature-sensitive growth                                                                          56.5&#194;&#176;C, strain NBS210, additionally lacked ypeP, <br />                                                                          exhibited cold-sensitive growth 22&#194;&#176;C. growth rates LB <br />                                                                          medium high- low-temperature-sensitive mutants                                                                          37&#194;&#176;C half wild type (data shown). <br />                                                                             filamentous phenotype RNase H-defective strains                                                                          SOS response. triple mutant NBS210                                                                          grown LB medium 37&#194;&#176;C, cells mid-log <br />                                                                          phase exhibited filamentous cell morphology (Fig. 5, pDG148). <br />                                                                          filamentous phenotype suppressed                                                                          depleted genes, rnhB, rnhC, ypeP, propagated                                                                          multicopy plasmid. hand, overexpression ypdQ <br />                                                                          effect morphology NBS210 (Fig. 5). <br />                                                                             RNase H deficiency thought result accumulation <br />                                                                          Okazaki fragments induce SOS response.                                                                          examine possibility, performed lacZ reporter assay                                                                          yneA. expression yneA control LexA    FIG. 3. Expression profiles &#226;?&#164;-galactosidase polA-lacZ     induced SOS response functions overlap ypcP-lacZ fusions. strain carrying lacZ fusions grown 2&#226;&#171;&#187;     SulA E. coli (18). shown Fig. 6A, yneA-lacZ SG medium, &#226;?&#164;-galactosidase activity assayed. time         constitutively expressed NBS218, NBS210 derivative car- <br /> transition exponential growth stationary phase designated    rying yneA-lacZ construct. level expression  <br /> time zero. Assays carried times yielded similar <br /> patterns; representative data shown. Closed circles, POLAd (polA- <br />                                                                          markably higher wild type, cellular <br /> lacZ); closed squares, NBS5014 (ypcP-lacZ); open squares, NBS5015        SOS response induced mitomycin C.  (ypcP-lacZ &#226;&#338;&#172;polA background).                                     yneA disruptant NBS210 (NBS218) ceased exhibit  8580       FUKUSHIMA ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Multiple alignments type 1 RNase H sequences. B. subtilis RNase HI homolog YpdQ novel RNase HI candidate YpeP aligned E. coli RNase HI (eRnhA) Halobacterium RNase H (hRnh). asterisks indicate conserved amino acid residues 65% identity. <br />  <br />  <br />  <br /> filamentous phenotype, consequently, impairment              findings E. coli cyanobacteria suggest cell division exhibited RNase H-defective strain             5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity Pol activity <br /> suppressed (Fig. 6B).                                                   encoded ypcP xni required bacterial growth. <br />    Partial complementation RNase H 5&#226;&#180;&#8226;-3&#226;&#180;&#8226; exonuclease                 assistance RNase H, excision RNA prim- <br /> activity. suggested RNase H Pol             ers vitro promoted (11). RNase H genes function synergistically processing RNA primers           reported essential B. subtilis (15).  Okazaki fragments (35).  deficiency RNase H             succeeded disrupting candidate genes RNase H, compensated enhanced 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease ac-              conclude RNase H necessary cell viability.  <br /> tivity. shown Fig. 5 (polAFL ypcP), filamentous            growth rates strains bearing RNase H <br /> phenotype strain NBS210 obviously suppressed                gene deletions, including rnhB rnhC, low, suggesting polA ypcP gene overexpressed. similar result           RNase H involved processing RNA primers. <br /> obtained gene encoding N-terminal domain               Based colony morphologies individual disruptants, <br /> Pol (Fig. 5, polAN). results indicate filamen-         rnhC appears prominent RNase <br /> tous morphology defective RNase H activity               H genes; rnhB rnhC double mutant exhibited temperature- overexpression 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domains          sensitive growth filamentous cell morphology.  complement rnh mutations.                                     phenotypes apparently gene <br />                                                                         disruptants, suggesting major functions RNase H <br />                                 DISCUSSION                              attributable genes. overexpression                                                                         newly identified ypeP suppressed filamentation rnhB <br />   discovery synthetic lethality pair doublet         rnhC ypeP mutant, ypdQ did  (21) <br /> paralogs, polA ypcP, confirmed previous results (9)             claimed YpeP lacks RNase H activity based struc- clearly demonstrated polymerase 5&#226;&#172;&#732;-3&#226;&#172;&#732; exo-            tural requirements, revised assignment open reading <br /> nuclease activity essential function B. subtilis cells.       frames careful alignment revealed conservation ac- <br />                                                                         tive amino acids (Fig. 4). results suggest YpeP  <br />  TABLE 3. Cold-sensitive temperature-sensitive phenotypes                various gene disruption strains <br />                                                       Growth ata: <br />   Strain                Relevant genotype <br />                                                22&#194;&#176;C     37&#194;&#176;C   56.5&#194;&#176;C <br />  <br /> 168           Wild type                         &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS5004       &#226;&#338;&#172;polA                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#186; <br /> NBS4070       &#226;&#338;&#172;ypcP                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS202        &#226;&#338;&#172;rnhB                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS204        &#226;&#338;&#172;rnhC                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS205        &#226;&#338;&#172;ypdQ                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS206        &#226;&#338;&#172;ypeP                             &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS208        &#226;&#338;&#172;rnhB &#226;&#338;&#172;ypeP                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185; <br /> NBS209        &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#185;     FIG. 5. Effects overexpression various genes filamen- <br /> NBS207        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC                       &#226;&#171;&#185;        &#226;&#171;&#185;          &#226;&#171;&#186;   tous phenotype B. subtilis triple-RNase H-deficient strain. NBS210 <br /> NBS210        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP                 &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   harboring pDG148 derivatives grown 37&#194;&#176;C LB medium <br /> NBS216        &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP &#226;&#338;&#172;ypdQ           &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   containing 5 &#226;?&#174;g/ml kanamycin 1 mM IPTG. Cell morphologies NBS217        &#226;&#338;&#172;polA &#226;&#338;&#172;rnhB &#226;&#338;&#172;rnhC &#226;&#338;&#172;ypeP &#226;&#338;&#172;ypdQ     &#226;&#171;&#186;        &#226;&#171;&#185;          &#226;&#171;&#186;   mid-log phase observed microscope. Genes cloned                                                                         pDG148 shown. FL N indicate length N-terminal <br />       &#226;&#171;&#185;, growth; &#226;&#171;&#186;, growth.                                           domains. Bar, 10 &#226;?&#174;m. <br />  VOL. 189, 2007                                                            REASSESSMENT Pol RNase H FUNCTIONS                     8581 <br />  <br />  <br />                                                                            mutant, accumulate Okazaki fragments fail survive (32). <br />                                                                            relationship SOS induction temperature sen- <br />                                                                            sitivity remains unknown;  results B. subtilis <br />                                                                            indicate inefficient processing RNA primers                                                                            lagging strand results recognition DNA damage leads <br />                                                                            SOS induction temperature sensitivity. <br />                                                                               Haemophilus influenzae, mutation RNase HI  <br />                                                                            creases mutation rates tetranucleotide repeats  <br />                                                                            sults phase variation delayed mutagenic Okazaki <br />    FIG. 6. SOS induction assay.   Kinetics yneA-lacZ induction.       fragment processing; additional deletion Klenow <br /> Aliquots YNEAd culture treated (closed squares)        domain Pol gives rise higher phase variation rates (open squares) mitomycin C withdrawn 20 (lane 1), 40 (lane <br /> 2), 60 (lane 3) min drug treatment assayed &#226;?&#164;-galac-      (4). results support hypothesis enzyme <br /> tosidase activity. Portions alternatively grown NBS218 culture,    systems cooperate processing Okazaki fragments. <br /> triple-RNase H-deficient strain, sampled (open circles)            dispensability RNase H function attributed scribed Materials Methods assayed &#226;?&#164;-galactosidase           complementation 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activities Pol activity. (B) Effects yneA deletion B. subtilis cell morphology. <br />                                                                            YpcP. filamentous phenotype induced deficient <br /> Cells exponentially growing cultures examined microscope.   168 (wild type). (b) Triple-RNase H-deficient strain       RNase H partially suppressed overexpression NBS210. (c) Triple-RNase H- yneA-deficient strain NBS218. Bar,         polA, N-terminal polA, ypcP. results sug- <br /> 10 &#226;?&#174;m.                                                                     gest enhanced 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity suppressed                                                                            RNase H deficiency,  enzymes appear                                                                            cooperate. RNase H activity YpdQ does support                      results indicated 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity <br /> earlier findings ypdQ did function B. subtilis (15, 30).       indispensable E. coli, cyanobacterial <br />    genes related processing RNA                    strain S. elongatus PCC 7942. reported primers Okazaki fragments, disrupted,                 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity Pol appears essential ypcP confer viability cells. expression pro-            bacteria, H. influenzae Streptococcus pneumoniae, <br /> files ypcP polA similar. strain lacking                lack ypcP ortholog (4, 8).  enzymatic <br /> N-terminal domain Pol (&#226;&#338;&#172;polAN) temperature sensi-                 activity essential eubacterial cells. tive, propagation ypcP gene amyE locus             finding B. subtilis ypcP E. coli ortholog, xni, led suppress temperature sensitivity polA mutant                look distribution types 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases <br /> (Fig. 2C), suggesting difference natures             bacteria. E. coli xni identified designated <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases. noted partial suppres-           exo (36), encoded enzyme previously reported sion observed Fig. 2C gene dosage effect              mainly 3&#226;&#172;&#732;-5&#226;&#172;&#732; single-stranded DNA exonuclease activity presence copies ypcP gene genome.              5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity (39). charac-  activity YpcP inactive high tem-              teristic nature RNA/DNA hybrids analyzed, perature, vitro experiments demonstrated               posit process substrate manner possesses 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease activity (J. Sekiguchi, personal               YpcP, classified genes based structural <br /> communication).  RNase H remove                   similarities. categorized genes possessing 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucle- <br /> RNA primers completely (30), temperature-sensitive                 ase domain types, e., polA type, ypcP type, phenotype &#226;&#338;&#172;polA strain elucidated. Al-                  type, consisting 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease domain RNase H disruptant identified temper-               polymerase domain 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease domain ature sensitive regardless Pol activity, strongly suspect          homologous polA (e-value &#226;&#172;? 0.01). total 250 RNase H function, addition Pol  required           bacterial genomes far sequenced classified efficient processing Okazaki fragments (35) high-                 classes, shown Fig. 7  Fig. S1 supplemental <br /> temperature conditions.                                                    material). B. subtilis E. coli belong class 4, al- <br />    expression profiles yneA-lacZ multiple-gene               3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease activity B. subtilis known disruptants, including rnhB rnhC, clearly demonstrated                 absent (9). Archaea (class 6) exceptional   SOS response taken place. E. coli, SOS                 stead 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonucleases, like eukaryotes, flap <br /> response leads sulA expression inhibits ring forma-               endonucleases. hand, eubacteria tion (26). B. subtilis, detailed mechanism different,        type 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease. 90% eubacterial enhanced activity YneA indirectly inhibits ring               genomes contain polA-type genes; 37% carry ypcP-type genes. <br /> mation (18, 19).  filamentous cell morphology                genomes type 3 genes (class 3). consequence depleted ring formation attributable              hand, 20 genomes ypcP-type genes (class 2) exclusively, activation LexA repressor accumulated                 indicating 3&#226;&#172;&#732;-5&#226;&#172;&#732; exonuclease polymer- <br /> processed Okazaki fragments. basic regulatory mechanism                ase domain essential. SOS response controlled complex circuitry                    used informatics approach look volving RecA LexA proteins. chromosomal DNA                    distribution repair-related proteins PolA, e., <br /> damaged replication inhibited, RecA activated              PolB IMS domain, includes Pol IV Pol V <br /> inducing signals LexA repressor degraded.                 Fig. S1 supplemental material). Using analysis, reported polA mutant, RNase H                    archaea possess DNA polymerase II  8582       FUKUSHIMA ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                        3. Au, N., E. Kuester-Schoeck, V. Mandava, L. E. Bothwell, S. P. Canny, K. <br />                                                                                           Chachu, S.  Colavito, S. N. Fuller, E. S. Groban, L.  Hensley, T. C. <br />                                                                                           O&#226;&#8364;&#8482;Brien,  Shah, J. T. Tierney, L. L. Tomm, T. M. O&#226;&#8364;&#8482;Gara,   Goranov, <br />                                                                                            D. Grossman, C. M. Lovett. 2005. Genetic composition Bacillus <br />                                                                                           subtilis SOS  J. Bacteriol. 187:7655&#226;&#8364;&#8220;7666. <br />                                                                                        4. Bayliss, C. D., W.  Sweetman, E. R. Moxon. 2005. Destabilization                                                                                           tetranucleotide repeats Haemophilus influenzae mutants lacking RnaseHI <br />                                                                                           Klenow domain PolI. Nucleic Acids Res. 33:400&#226;&#8364;&#8220;408. <br />                                                                                        5. Britton, R. , P. Eichenberger, J. E. Gonzalez-Pastor, P. Fawcett, R. <br />                                                                                           Monson, R. Losick,  D. Grossman. 2002. Genome-wide analysis                                                                                           stationary-phase sigma factor (sigma-H) regulon Bacillus subtilis. J. Bac- <br />                                                                                           teriol. 184:4881&#226;&#8364;&#8220;4890. <br />                                                                                        6. Castenholz, R. W. 1988. Culturing methods cyanobacteria. Methods <br />                                                                                           Enzymol. 167:68&#226;&#8364;&#8220;93. <br />                                                                                        7. Lucia, P., J. Cairns. 1969. Isolation E. coli strain                                                                                           mutation affecting DNA polymerase. Nature 224:1164&#226;&#8364;&#8220;1166. <br />                                                                                        8. Diaz, , S.  Lacks, P. Lopez. 1992. 5&#226;&#172;&#732; 3&#226;&#172;&#732; exonuclease activity <br />                                                                                           DNA polymerase essential Streptococcus pneumoniae. Mol. Mi- <br />    FIG. 7. Venn diagram bacterial species according type                     crobiol. 6:3009&#226;&#8364;&#8220;3019. <br /> 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease gene. total 250 bacterial genomes (228 eubac-                   9. Duigou, S., S. D. Ehrlich, P. Noirot, M. F. Noirot-Gros. 2005. DNA <br /> teria 22 archaea) sequenced <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> categorized                         polymerase acts translesion synthesis mediated Y-polymerases classes based gene structure encoding 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease.                      Bacillus subtilis. Mol. Microbiol. 57:678&#226;&#8364;&#8220;690. <br />                                                                                       10. Fujita, M., R. Losick. 2005. Evidence entry sporulation circle corresponds types 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease: PolA type, YpcP                   Bacillus subtilis governed gradual increase level activity type, (5&#226;&#172;&#732;-3&#226;&#172;&#732; EXO-DNA_POL) type  text). Class 1                        master regulator Spo0A. Genes Dev. 19:2236&#226;&#8364;&#8220;2244. <br /> consisted genomes containing PolA type. Class 2 consisted                 11. Funnell, B. E., T.  Baker,  Kornberg. 1986. Complete enzymatic genomes containing YpcP type. Class 3 consisted ge-                        replication plasmids containing origin Escherichia coli chromo- <br /> nomes containing type. Class 4 consisted genomes                         J. Biol. Chem. 261:5616&#226;&#8364;&#8220;5624. <br /> containing PolA type YpcP type. Class 5 consisted                      12. Higuchi, R. 1989. Using PCR engineer DNA, p. 61&#226;&#8364;&#8220;70. H.  Erlich <br /> genomes containing YpcP type type. Class 6 consisted                    (ed.), PCR technology. Stockton Press, New York, NY. genomes containing 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease genes.                                     13. Itaya, M. 1992. Construction novel tetracycline resistance gene cassette <br />                                                                                           useful marker Bacillus subtilis chromosome. Biosci. Biotechnol. <br />                                                                                           Biochem. 56:685&#226;&#8364;&#8220;686. <br />                                                                                       14. Itaya, M. 1990. Isolation characterization second RNase H (RNase <br /> seven species, simultaneously belong class 2 Fig. 7,                          HII) Escherichia coli K-12 encoded rnhB gene. Proc. Natl. Acad. <br />                                                                                           Sci. USA 87:8587&#226;&#8364;&#8220;8591. <br /> possess repair polymerase  Fig. S1 supplemental                         15. Itaya, M.,  Omori, S. Kanaya, R. J. Crouch, T. Tanaka, K. Kondo. <br /> material). species belong genera, Buchnera,                                 1999. Isolation RNase H genes essential growth Bacillus <br /> Wigglesworthia, Phytoplasma, Tropheryma, sym-                             subtilis 168. J. Bacteriol. 181:2118&#226;&#8364;&#8220;2123. <br />                                                                                       16. Joyce, C. M., N. D. Grindley. 1984. Method determining biotic infectious bacteria. known                               gene Escherichia coli essential: application polA gene. J. Bacteriol. bacteria sensitive DNA-damaging agents,                              158:636&#226;&#8364;&#8220;643. <br />                                                                                       17. Kanaya, S., R. J. Crouch. 1983. DNA sequence gene coding mitomycin C, require DNA repair func-                                Escherichia coli ribonuclease H. J. Biol. Chem. 258:1276&#226;&#8364;&#8220;1281. <br /> tions environments inhabit.                                       18. Kawai, Y., S. Moriya, N. Ogasawara. 2003. Identification protein, <br />    noted  archaea eubacteria developed distinct                              YneA, responsible cell division suppression SOS response                                                                                           Bacillus subtilis. Mol. Microbiol. 47:1113&#226;&#8364;&#8220;1122. <br /> types processing enzymes, e., flap endonuclease 5&#226;&#172;&#732;-3&#226;&#172;&#732;                        19. Kawai, Y., N. Ogasawara. 2006. Bacillus subtilis EzrA FtsL syner- <br /> exonuclease, respectively. weak homologies                                gistically regulate FtsZ ring dynamics cell division. Microbiology domains polymerase  hyperthermophilic                              152:1129&#226;&#8364;&#8220;1141. <br />                                                                                       20. Kitani, T., K. Yoda, T. Ogawa, T. Okazaki. 1985. Evidence discon- <br /> bacterium Aquifex aeolicus, belongs class 2, diverged                            tinuous DNA replication Escherichia coli primed approximately 10 close root phylogenetic tree. Aquifex                            12 residues RNA starting purine. J. Mol. Biol. 184:45&#226;&#8364;&#8220;52. <br /> possesses gene corresponding separate Klenow frag-                             21. Knizewski, L., K. Ginalski. 2005. Bacillus subtilis YkuK protein dis- <br />                                                                                           tantly related RNase H. FEMS Microbiol. Lett. 251:341&#226;&#8364;&#8220;346. <br /> ment, domains, e., 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease, evolved                      22. Kobayashi, K., S. D. Ehrlich,  Albertini, G. Amati, K. K. Andersen, M.  duplication combination unit                              Arnaud, K. Asai, S. Ashikaga, S. Aymerich, P. Bessieres, F. Boland, S. C. <br />                                                                                           Brignell, S. Bron, K. Bunai, J. Chapuis, L. C. Christiansen,  Danchin, M. <br /> yielded variety related enzyme patterns different                              Debarbouille, E. Dervyn, E. Deuerling, K. Devine, S. K. Devine, O. Dreesen, <br /> organisms.                                                                                J. Errington, S. Fillinger, S. J. Foster, Y. Fujita,  Galizzi, R. Gardan, C. <br />                                                                                           Eschevins, T. Fukushima, K. Haga, C. R. Harwood, M. Hecker, D. Hosoya, <br />                                                                                           M. F. Hullo, H. Kakeshita, D. Karamata, Y. Kasahara, F. Kawamura, K. <br />                           ACKNOWLEDGMENTS                                                 Koga, P. Koski, R. Kuwana, D. Imamura, M. Ishimaru, S. Ishikawa,  Ishio, <br />    grateful Naoto Ohtani Keio University helpful                         D. Le Coq,  Masson, C. Mauel, R. Meima, R. P. Mellado,  Moir, S. <br />                                                   <br /> </body></html>